
RESEARCH | REPORTS

**BIODEGRADATION**

# A bacterium that degrades and assimilates poly(ethylene terephthalate)

Shosuke Yoshida,$^{1,2*}$ Kazumi Hiraga,$^{1}$ Toshihiko Takehana,$^{3}$ Ikuo Taniguchi,$^{4}$ Hironao Yamaji,$^{1}$ Yasuhito Maeda,$^{5}$ Kiyotsuna Toyohara,$^{5}$ Kenji Miyamoto,$^{2†}$ Yoshiharu Kimura,$^{4}$ Kohei Oda$^{1†}$

Poly(ethylene terephthalate) (PET) is used extensively worldwide in plastic products, and its accumulation in the environment has become a global concern. Because the ability to enzymatically degrade PET has been thought to be limited to a few fungal species, biodegradation is not yet a viable remediation or recycling strategy. By screening natural microbial communities exposed to PET in the environment, we isolated a novel bacterium, *Ideonella sakaiensis* 201-F6, that is able to use PET as its major energy and carbon source. When grown on PET, this strain produces two enzymes capable of hydrolyzing PET and the reaction intermediate, mono(2-hydroxyethyl) terephthalic acid. Both enzymes are required to enzymatically convert PET efficiently into its two environmentally benign monomers, terephthalic acid and ethylene glycol.

Plastics with desirable properties such as durability, plasticity, and/or transparency have been industrially produced over the past century and widely incorporated into consumer products (1). Many of these products are remarkably persistent in the environment because of the absence or low activity of catabolic enzymes that can break down their plastic constituents. In particular, polyesters containing a high ratio of aromatic components, such as poly(ethylene terephthalate) (PET), are chemically inert, resulting in resistance to microbial degradation (2, 3). About 56 million tons of PET was produced worldwide in 2013 alone, prompting further industrial production of its monomers, terephthalic acid (TPA) and ethylene glycol (EG), both of which are derived from raw petroleum. Large quantities of PET have been introduced into the environment through its production and disposal, resulting in the accumulation of PET in ecosystems across the globe (4).

There are very few reports on the biological degradation of PET or its utilization to support microbial growth. Rare examples include members of the filamentous fungi *Fusarium oxysporum* and *F. solani*, which have been shown to grow on a mineral medium containing PET yarns [although no growth levels were specified (5, 6)]. Once identified, microorganisms with the enzymatic machinery needed to degrade PET could serve as an environmental remediation strategy as well as a degradation and/or fermentation platform for biological recycling of PET waste products.

We collected 250 PET debris-contaminated environmental samples including sediment, soil, wastewater, and activated sludge from a PET bottle recycling site (7). Using these samples, we screened for microorganisms that could use low-crystallinity (1.9%) PET film as the major carbon source for growth. One sediment sample contained a distinct microbial consortium that formed on the PET film upon culturing (Fig. 1A) and induced morphological change in the PET film (Fig. 1B). Microscopy revealed that the consortium on the film, termed “no. 46,” contained

---

1 Department of Applied Biology, Faculty of Textile Science, Kyoto Institute of Technology, Matsugasaki, Sakyo-ku, Kyoto 606-8585, Japan.
2 Department of Biosciences and Informatics, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama, Kanagawa 223-8522, Japan.
3 Life Science Materials Laboratory, ADEKA, 7-2-34 Higashiogu, Arakawa-ku, Tokyo 116-8553, Japan.
4 Department of Polymer Science, Faculty of Textile Science, Kyoto Institute of Technology, Matsugasaki, Sakyo-ku, Kyoto 606-8585, Japan.
5 Ecology-Related Material Group Innovation Research Institute, Teijin, Hinode-cho 2-1, Iwakuni, Yamaguchi 740-8511, Japan.
*Present address: Department of Polymer Chemistry, Graduate School of Engineering, Kyoto University, Nishikyo-ku, Kyoto 615-8530, Japan.
†Corresponding author. E-mail: kmiyamoto@bio.keio.ac.jp (K.M.); bika@kit.ac.jp (K.O.)

---

**Fig. 1. Microbial growth on PET.** The degradation of PET film (60 mg, 20 × 15 × 0.2 mm) by microbial consortium no. 46 at 30°C is shown in (A) to (C). The MLE (modified lettuce and egg) medium (10 mL) was changed biweekly. (A) Growth of no. 46 on PET film after 20 days. (B) SEM image of degraded PET film after 70 days. The inset shows intact PET film. Scale bar, 0.5 mm. (C) Time course of PET film degradation by no. 46. PET film degradation by *I*. *sakaiensis* 201-F6 at 30°C is shown in (D) to (H). The YSV (yeast extract–sodium carbonate–vitamins) medium was changed weekly. (D to F) SEM images of *I*. *sakaiensis* cells grown on PET film for 60 hours. Scale bars, 1 μm. Arrow heads in the left panel of (D) indicate contact points of cell appendages and the PET film surface. Magnifications are shown in the right panel. Arrows in (F) indicate appendages between the cell and the PET film surface. (G) SEM image of a degraded PET film surface after washing out adherent cells. The inset shows intact PET film. Scale bar, 1 μm. (H) Time course of PET film degradation by *I*. *sakaiensis*.

a mixture of bacteria, yeast-like cells, and protozoa, whereas the culture fluid was almost transparent (Fig. 1A). This consortium degraded the PET film surface (fig. S1) at a rate of 0.13 mg cm⁻² day⁻¹ at 30°C (Fig. 1C), and 75% of the degraded PET film carbon was catabolized into CO₂ at 28°C (fig. S2).

Using limiting dilutions of consortium no. 46 that were cultured with PET film to enrich for microorganisms that are nutritionally dependent on PET, we successfully isolated a bacterium capable of degrading and assimilating PET. The strain represents a novel species of the genus *Ideonella*, for which we propose the name *Ideonella sakaiensis* 201-F6 (deposited in the National Center for

Biotechnology Information taxonomy database under identifier 1547922). In addition to being found in the culture fluid, cells were observed on the film (Fig. 1D) and appeared to be connected to each other by appendages (Fig. 1E). Shorter appendages were observed between the cells and the film; these might assist in the delivery of secreted enzymes into the film (Fig. 1, D and F). The PET film was damaged extensively (Fig. 1G) and almost completely degraded after 6 weeks at 30°C (Fig. 1H). In the course of subculturing no. 46, we found a subconsortium that lost its PET degradation capability. This subconsortium lacked *I. sakaiensis* (fig. S3), indicating that *I. sakaiensis* is functionally involved in PET degradation.

There are currently few known examples of esterases, lipases, or cutinases that are capable of hydrolyzing PET (8, 9). To explore the genes involved in PET hydrolysis in *I. sakaiensis* 201-F6, we assembled a draft sequence of its genome (table S1). One identified open reading frame (ORF), ISF6_4831, encodes a putative lipase that shares 51% amino acid sequence identity and catalytic residues with a hydrolase from *Thermobifida fusca* (TfH) (fig. S4 and table S2) that exhibits PET-hydrolytic activity (10). We purified the corresponding recombinant *I. sakaiensis* proteins (fig. S5) and incubated them with PET film at 30°C for 18 hours. Prominent pitting developed on the film surface (Fig. 2A). Mono(2-hydroxyethyl)

![Figure 2](#fig-2)

**Fig. 2. ISF6_4831 protein is a PETase.** Effects of PETase on PET film are shown in (A) and (B). PET film (diameter, 6 mm) was incubated with 50 nM PETase in pH 7.0 buffer for 18 hours at 30°C. (A) SEM image of the treated PET film surface. The inset shows intact PET film. Scale bar, 5 μm. (B) High-performance liquid chromatography spectrum of the products released from the PET film. (C) Unrooted phylogenetic tree of known PET hydrolytic enzymes. The GenBank or Protein Data Bank accession numbers (with the organism source of protein in parentheses) are shown at the leaves. Bootstrap values are shown at the branch points. Scale bar, 0.1 amino acid substitutions per single site. (D) Substrate specificity of four phylogenetically distinct PET hydrolytic enzymes (b/a indicates the ratio of the values in the middle-left panel to those in the leftmost panel). All reactions were performed in pH 7.0 buffer at 30°C. PET film was incubated with 50 nM enzyme for 18 hours. (E) Activity of the PET hydrolytic enzymes for highly crystallized PET (hcPET). The hcPET (diameter, 6 mm) was incubated with 50 nM PETase or 200 nM TfH, LCC, or FsC in pH 9.0 bicine-NaOH buffer for 18 hours at 30°C. (F) Effect of temperature on enzymatic PET film hydrolysis. PET film (diameter, 6 mm) was incubated with 50 nM PETase or 200 nM TfH, LCC, or FsC in pH 9.0 bicine-NaOH buffer for 1 hour. For better detection of the released products in (E) and (F), the pH and enzyme concentrations were determined based on the results shown in figs. S6 and S7, respectively. Error bars in (D) to (F) indicate SE (*n* ≥ 3).

RESEARCH | REPORTS

Fig. 3. PET metabolism by *I. sakaiensis*. (A) Transcript levels of selected genes when grown on TPA-Na, PET film, or BHET, relative to those when grown on maltose (PCA, protocatechuic acid; ORF#, last four digits of the ORF number). Two-sided P values were derived from Baggerly's test of the differences between the means of two independent RNA sequencing experiments (*P* < 0.05; **P** < 0.01). Colors correspond to the steps in (B). (B) Predicted *I. sakaiensis* PET degradation pathway. The cellular localization of PETase and MHETase was predicted first (supplementary text, section S1). Extracellular PETase hydrolyzes PET to produce MHET (the major product) and TPA. MHETase, a predicted lipoprotein, hydrolyzes MHET to TPA and EG. TPA is incorporated through the TPA transporter (TPATP) (17) and catabolized by TPA 1,2-dioxygenase (TPADO), followed by 1,2-dihydroxy-3,5-cyclohexadiene-1,4-dicarboxylate dehydrogenase (DCDDH). The resultant PCA is ring-cleaved by PCA 3,4-dioxygenase (Pca34). The predicted TPA degradation pathway is further described in the supplementary text (section S2).

terephthalic acid (MHET) was the major product released by the recombinant protein, together with minor amounts of TPA and bis(2-hydroxyethyl) TPA (BHET) (Fig. 2B). These results suggest that the ISF6_4831 protein hydrolyzes PET. This protein also hydrolyzed BHET to yield MHET with no further decomposition.

We compared the activity of the ISF6_4831 protein with that of three evolutionarily divergent PET-hydrolytic enzymes identified from a phylogenetic tree that we constructed using published enzymes (Fig. 2C and table S2). We purified TfH from a thermophilic actinomycete (10), cutinase homolog from leaf-branch compost metagenome (LC cutinase, or LCC) (11), and *F. solani* cutinase (FsC) from a fungus (fig. S5) (12), and we measured their activities against *p*-nitrophenol-linked aliphatic esters (*p*NP-aliphatic esters), PET film, and BHET at 30°C and pH 7.0. For *p*NP-aliphatic esters, which are preferred by lipases and cutinases, the activity of the ISF6_4831 protein was lower than that of TfH, LCC, and FsC (Fig. 2D). The activity of the ISF6_4831 protein against the PET film, however, was 120, 5.5, and 88 times as high as that of TfH, LCC, and FsC, respectively. A similar trend was observed for BHET (Fig. 2D). The catalytic preference of the ISF6_4831 protein for PET film over *p*NP-aliphatic esters was also substantially higher than that of TfH, LCC, and FsC (380, 48, and 400 times as high on average, respectively) (Fig. 2D). Thus, the ISF6_4831 protein prefers PET to aliphatic esters, compared with the other enzymes, leading to its designation as a PET hydrolase (termed PETase).

PETase was also more active than TfH, LCC, and FsC against commercial bottle-derived PET, which is highly crystallized (Fig. 2E), even though the densely packed structure of highly crystallized

Table 1. Kinetic parameters of MHETase. The kinetic parameters were determined in pH 7.0 buffer at 30°C. Because the enzymatic PET hydrolysis involves a heterogeneous reaction, Michaelis-Menten kinetics were not applied to PETase. ND, not detected (activity was below the detection limit of the assay).

| Substrate                          | $k_{\text{cat}}$ (s⁻¹) | $K_{m}$ (μM) |
|------------------------------------|------------------------|---------------|
| MHET                              | 31 ± 0.8*              | 7.3 ± 0.6*    |
| PET film                          | ND                     |               |
| BHET                              | 0.10 ± 0.004†          |               |
| *p*NP-aliphatic esters (*p*NP-acetate, *p*NP-butyrate) | ND            |               |
| Aromatic esters (ethyl gallate, ethyl ferulate, chlorogenic acid hydrate) | ND           |               |

*Data are shown as means ± SEs based on a nonlinear regression model. †The reported $k_{\text{cat}}$ is the apparent $k_{\text{cat}}$ determined with 0.9 mM BHET. Shown is the mean ± SE from three independent experiments.

PET greatly reduces the enzymatic hydrolysis of its ester linkages (9, 13). PETase was somewhat heat-labile, but it was considerably more active against PET film at low temperatures than were TfH, LCC, and FsC (Fig. 2F). Enzymatic degradation of polyesters is controlled mainly by their chain mobility (14). Flexibility of the polyester chain decreases as the glass transition temperature increases (9). The glass transition temperature of PET is around 75°C, meaning that the polyester chain of PET is in a glassy state at the moderate temperatures appropriate for mesophilic enzyme reactions. The substrate specificity of PETase and its prominent hydrolytic activity for PET in a glassy state would be critical to sustaining the growth of *I. sakaiensis* on PET in most environments.

*I. sakaiensis* adheres to PET (Fig. 1, D to F) and secretes PETase to target this material. We compared the PET hydrolytic activity of PETase with that of the other three PET hydrolytic enzymes

sciencemag.org SCIENCE

or PET film (fig. S9 and table S3). The catabolic genes for TPA and the metabolite protocatechuic acid (PCA) were up-regulated dramatically when cells were cultured on TPA-Na, BHET, or PET film. This contrasted with genes for the catabolism of maltose (Fig. 3A), which involves a pathway distinct from the degradation of TPA and EG, indicating efficient metabolism of TPA by *I. sakaiensis*. The transcript level of the PETase-encoding gene during growth on PET film was the highest among all analyzed coding sequences (table S4), and it was 15, 31, and 41 times as high as when bacteria were grown on maltose, TPA-Na, and BHET, respectively. This suggests that the expression of PETase is induced by PET film itself and/or some degradation products other than TPA, EG, MHET, and BHET.

The expression levels of the PETase gene in the four different media were similar to those of another ORF, ISF6_0224 (fig. S10), indicating similar regulation. ISF6_0224 is located adjacent to the TPA degradation gene cluster (fig. S11). The ISF6_0224 protein sequence matches those of the tannase family, which is known to hydrolyze the ester linkage of aromatic compounds such as gallic acid esters, ferulic saccharides, and chlorogenic acids. The catalytic triad residues and two cysteine residues found only in this family (15) are completely conserved in the ISF6_0224 protein (fig. S12). Purified recombinant ISF6_0224 protein (fig. S5) efficiently hydrolyzed MHET with a turnover rate (*k*~cat~) of 31 ± 0.8 s^−1^ and a Michaelis constant (*K*~m~) of 7.3 ± 0.6 μM (Table 1), but it did not show any activity against PET, BHET, *p*NP-aliphatic esters, or typical aromatic ester compounds catalyzed by the tannase family enzymes (Table 1). ISF6_0224 is nonhomologous to six known MHET-hydrolytic enzymes that also hydrolyze PET and *p*NP-aliphatic esters (table S2). These results strongly suggest that the ISF6_0224 protein is responsible for the conversion of MHET to TPA and EG in *I. sakaiensis*. The enzyme was thus designated a MHET hydrolase (termed MHETase).

To determine how the metabolism of PET (Fig. 3B) evolved, we used the Integr8 fully sequenced genome database (16) to search for other organisms capable of metabolizing this compound. However, we were unable to find other organisms with a set of gene homologs of signature enzymes for PET metabolism (PETase, MHETase, TPA dioxygenase, and PCA dioxygenase) (fig. S13). However, among the 92 microorganisms with MHETase homolog(s), 33 had homologs of both TPA and PCA dioxygenases. This suggests that a genomic basis to support the metabolism of MHET analogs was established much earlier than when ancestral PETase proteins were incorporated into the pathway. PET enrichment in the sampling site and the enrichment culture potentially promoted the selection of a bacterium that might have obtained the necessary set of genes through lateral gene transfer. A limited number of mutations in a hydrolase, such as PET hydrolytic cutinase, that inherently targets the natural aliphatic polymer cutin may have resulted in enhanced selectivity for PET.


---

**REFERENCES AND NOTES**

1. V. Sinha, M. R. Patel, J. V. Patel, *J. Polym. Environ.* **18**, 8–25 (2010).
2. R. J. Müller, I. Kleeberg, W. D. Deckwer, *J. Biotechnol.* **86**, 87–95 (2001).
3. D. Kint, S. Munoz-Guerra, *Polym. Int.* **48**, 346–352 (1999).
4. L. Neufeld, F. Stassen, R. Sheppard, T. Gilman, Eds., *The New Plastics Economy: Rethinking the Future of Plastics* (World Economic Forum, 2016); www3.weforum.org/docs/WEF_The_New_Plastics_Economy.pdf.
5. T. Nimchua, H. Punnapayak, W. Zimmermann, *Biotechnol. J.* **2**, 361–364 (2007).
6. T. Nimchua, D. E. Eveleigh, U. Sangwatanaroj, H. Punnapayak, *J. Ind. Microbiol. Biotechnol.* **35**, 843–850 (2008).
7. Materials and methods are available as supplementary materials on Science Online.
8. D. Ribitsch et al., *Biocatalysis Biotransform.* **30**, 2–9 (2012).
9. W. Zimmermann, S. Billig, *Adv. Biochem. Eng. Biotechnol.* **125**, 97–120 (2010).
10. R. J. Müller, H. Schrader, J. Profe, K. Dresler, W. D. Deckwer, *Macromol. Rapid Commun.* **26**, 1400–1405 (2005).
11. S. Sulaiman et al., *Appl. Environ. Microbiol.* **78**, 1556–1562 (2012).
12. C. M. Silva et al., *J. Polym. Sci. A Polym. Chem.* **43**, 2448–2450 (2005).
13. M. A. M. E. Vertommen, V. A. Nierstrasz, M. Veer, M. M. C. G. Warmoeskerken, *J. Biotechnol.* **120**, 376–386 (2005).
14. E. Marten, R. J. Müller, W. D. Deckwer, *Polym. Degrad. Stabil.* **88**, 371–381 (2005).
15. K. Suzuki et al., *Proteins* **82**, 2857–2867 (2014).
16. P. Kersey et al., *Nucleic Acids Res.* **33**, D297–D302 (2005).
17. M. Hosaka et al., *Appl. Environ. Microbiol.* **79**, 6148–6155 (2013).


**ACKNOWLEDGMENTS**

We are grateful to Y. Horiuchi, M. Uemura, T. Kawai, K. Sasage, and S. Hase for research assistance. We thank D. Dodd, H. Atomi, T. Nakayama, and A. Wlodawer for comments on this manuscript. This study was supported by grants-in-aid for scientific research (24780078 and 26850053 to S.Y.) and the Noda Institute for Scientific Research (S.Y.). The reported nucleotide sequence data, including assembly and annotation, have been deposited in the DNA Data Bank of Japan, European Molecular Biology Laboratory, and GenBank databases under the accession numbers BBYR01000001 to BBYR01000227. All other data are reported in the supplementary materials. The reported strain *Ideonella sakaiensis* 201-F6^T^ was deposited at the National Institute of Technology and Evaluation Biological Resource Center as strain NBRC 110686^T^ and at Thailand Institute of Scientific and Technological Research as strain TISTR 2288^T^.


**SUPPLEMENTARY MATERIALS**

www.sciencemag.org/content/351/6278/1196/suppl/DC1

Materials and Methods  
Supplementary Text  
Figs. S1 to S14  
Tables S1 to S5  
References (18–39)

15 October 2015; accepted 29 January 2016  
10.1126/science.aad6359


---

**NEURODEVELOPMENT**

**CLK2 inhibition ameliorates autistic features associated with SHANK3 deficiency**

Michael Bidinosti,$^{1\ast}$ Paolo Botta,$^{3\ast}$ Sebastian Krüttner,$^{3}$ Catia C. Proença,$^{1}$ Natacha Stoehr,$^{1}$ Mario Bernhard,$^{1}$ Isabelle Fruh,$^{1}$ Matthias Mueller,$^{1}$ Debora Bonenfant,$^{2}$ Hans Voshol,$^{2}$ Walter Carbone,$^{1}$ Sarah J. Neal,$^{4}$ Stephanie M. McTighe,$^{4}$ Guglielmo Roma,$^{1}$ Ricardo E. Dolmetsch,$^{4}$ Jeffrey A. Porter,$^{1}$ Pico Caroni,$^{3}$ Tewis Bouwmeester,$^{1}$ Andreas Lüthi,$^{3}$ Ivan Galimberti$^{1\dagger}$

SH3 and multiple ankyrin repeat domains 3 (SHANK3) haploinsufficiency is causative for the neurological features of Phelan-McDermid syndrome (PMDS), including a high risk of autism spectrum disorder (ASD). We used unbiased, quantitative proteomics to identify changes in the phosphoproteome of Shank3-deficient neurons. Down-regulation of protein kinase B (PKB/Akt)—mammalian target of rapamycin complex 1 (mTORC1) signaling resulted from enhanced phosphorylation and activation of serine/threonine protein phosphatase 2A (PP2A) regulatory subunit, B56β, due to increased steady-state levels of its kinase, Cdc2-like kinase 2 (CLK2). Pharmacological and genetic activation of Akt or inhibition of CLK2 relieved synaptic deficits in Shank3-deficient and PMDS patient-derived neurons. CLK2 inhibition also restored normal sociability in a Shank3-deficient mouse model. Our study thereby provides a novel mechanistic and potentially therapeutic understanding of deregulated signaling downstream of Shank3 deficiency.

C hromosomal aberrations at 22q13 that delete or inactivate one SH3 and multiple ankyrin repeat domains 3 (SHANK3) allele are genetic hallmarks of Phelan-McDermid syndrome (PMDS). De novo mutations in SHANK3 are also associated with nonsyndromic autism spectrum disorder (ASD) and intellectual disability (1–4). Genetic ablation of *Shank3* in mice yields ASD-like behavioral phenotypes and synaptic dysfunction (5–10), the latter of which

Science

A bacterium that degrades and assimilates poly(ethylene terephthalate)

Shosuke Yoshida, Kazumi Hiraga, Toshihiko Takehana, Ikuo Taniguchi, Hironao Yamaji, Yasuhito Maeda, Kiyotsuna Toyohara, Kenji Miyamoto, Yoshiharu Kimura and Kohhei Oda

Science 351 (6278), 1196-1199.
DOI: 10.1126/science.aad6359

Some bacteria think plastic is fantastic
Bacteria isolated from outside a bottle-recycling facility can break down and metabolize plastic. The proliferation of plastics in consumer products, from bottles to clothing, has resulted in the release of countless tons of plastics into the environment. Yoshida et al. show how the biodegradation of plastics by specialized bacteria could be a viable bioremediation strategy (see the Perspective by Bornscheuer). The new species, *Ideonella sakaiensis*, breaks down the plastic by using two enzymes to hydrolyze PET and a primary reaction intermediate, eventually yielding basic building blocks for growth.

Science, this issue p. 1196; see also p. 1154

ARTICLE TOOLS
http://science.sciencemag.org/content/351/6278/1196

SUPPLEMENTARY MATERIALS
http://science.sciencemag.org/content/suppl/2016/03/09/351.6278.1196.DC1

RELATED CONTENT
http://science.sciencemag.org/content/sci/351/6278/1154.full
http://science.sciencemag.org/content/sci/353/6301/759.2.full
http://science.sciencemag.org/content/sci/353/6301/759.3.full

REFERENCES
This article cites 37 articles, 7 of which you can access for free
http://science.sciencemag.org/content/351/6278/1196#BIBL

PERMISSIONS
http://www.sciencemag.org/help/reprints-and-permissions

Use of this article is subject to the Terms of Service

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. 2017 © The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. The title Science is a registered trademark of AAAS.
