
Environmental Microbiology (2016) 00(00), 00–00

**Genetic and functional characterization of a novel meta-pathway for degradation of naringenin in Herbaspirillum seropedicae SmR1**

Anelis Maria Marin,¹ Jésus de la Torre,² Alfredo Ricardo Marques Oliveira,³ Andersson Barison,³ Leda Satie Chubatsu,¹ Rose Adele Monteiro,¹ Fabio de Oliveira Pedrosa,¹ Emanuel Maltempi de Souza,¹ Roseli Wassem,⁴ Estrella Duque² and Juan-Luis Ramos²,⁵*

¹ Department of Biochemistry and Molecular Biology, Nitrogen Fixation Group, UFPR, Brazil.

² Department of Environmental Protection, Estación Experimental del Zaidín CSIC, Spain.

³ Department of Chemistry, UFPR, Brazil.

⁴ Department of Genetics, UFPR, Brazil.

⁵ Department of Biotechnology, Abengoa Research, Spain.

### Summary

In this study, a random mutant library of *Herbaspirillum seropedicae* SmR1 was constructed by Tn5 insertion and a mutant incapable of utilizing naringenin as a carbon source was isolated. The Tn5 transposon was found to be inserted in the *fdeE* gene (Hsero_1007), which encodes a monooxygenase. Two other mutant strains in *fdeC* (Hsero_1005) and *fdeG* (Hsero_1009) genes coding for a dioxygenase and a putative cyclase, respectively, were obtained by site-directed mutagenesis and then characterized. Liquid Chromatography coupled to mass spectrometry (LC-MS)/MS analyses of culture supernatant from the *fdeE* mutant strain revealed that naringenin remained unaltered, suggesting that the FdeE protein is involved in the initial step of naringenin degradation. LC-MS/MS analyses of culture supernatants from the wild-type (SmR1) and FdeC deficient mutant suggested that in *H. seropedicae* SmR1 naringenin is first mono-oxygenated by the FdeE protein, to produce 5,7,8-trihydroxy-2-(4-hydroxyphenyl)-2,3-dihydro-4H-chromen-4-one, that is subsequently dioxygenated and cleaved at the A-ring by the FdeC dioxygenase, since the latter compound accumulates in the *fdeC* strain. After meta-cleavage of the A-ring, the subsequent metabolic steps generate oxaloacetic acid that is metabolized via the tricarboxylic acid cycle. This bacterium can also modify naringenin by attaching a glycosyl group to the B-ring or a methoxy group to the A-ring, leading to the generation of dead-end products.

### Introduction

Flavonoids are an important class of plant secondary metabolites that play a central role in microbial-plant interactions; they act as chemo-attractants, biocides, nod-gene inducers and modifiers of bacterial surface polysaccharides (Mulligan and Long, 1985; Brencic and Winans, 2005; Shaw et al., 2006; Reddy et al., 2007, Cesco et al., 2010). Several physiological functions have been attributed to flavonoids, including, electron acceptors, free-radical scavengers and metal ion-chelators (Das and Rosazza, 2006). Das and Rosazza (2006) also suggested that flavonoids are metabolized as a carbon source through oxidative pathways, while conjugative pathways, such as glycosylation, hydroxylation and sulfation, are associated with detoxification mechanisms (Ibrahim, 2000; Hosny et al., 2001; Das and Rosazza, 2006; Bartmanska et al., 2013). Several potential pathways have been described for metabolism of flavonoids by soil microorganisms, although in most cases the details of the degradation pathway are unknown. It was suggested that *Aspergillus flavus*, *Eubacterium ramulus* and *Bacillus subtilis* were capable of cleaving the C-ring of flavonols that contain a 2,3-double bond and a 3-hydroxyl group through the action of a quercetinase (dioxigenase) (Schneider and Blaut, 2000; Braune et al., 2001; Das and Rosazza, 2006; Schaab et al., 2006). In the anaerobic rumen, the degradation of naringenin by bacteria yields phloroglucinol and p-hydroxyphenylpropionic acid through C-ring cleavage (Cheng et al., 1971; Das and Rosazza, 2006). *Pseudomonas* sp. are capable of hydroxylation of catechin into catechin 8-hydroxylation and subsequent A-ring degradation (Das and Rosazza, 2006), while *P. putida* PML2 can cleave the apigenin C-ring into phloroglucinol and 3,4-dihydroxy cinnamic acid (Pillai and Swarup, 2002).

Accepted 20 March, 2016. *For correspondence.* E-mail juan.ramos@abengoa.com; Tel. +34 954 937 111.

© 2016 Society for Applied Microbiology and John Wiley & Sons Ltd

2 A. M. Marin *et al.*

Few studies have addressed flavonoid degradation by plant growth promoting rhizobacteria. The ability of rhizobacteria to metabolize flavonoids is likely advantageous as plants exude different flavonoids into the rhizosphere, these flavonoids could be used as a carbon source, chemo-attractant, allelopathic growth inhibitors of competitors or pathogen suppressors (Cesco *et al.*, 2010). There are a few reports which describe the degradation of flavonoids by rhizobia, most of which suggest that they are processed via C-ring cleavage (Rao *et al.*, 1991; Rao and Cooper, 1994; 1995). Cleaving of the C-ring of the isoflavone genistein produced, phloroglucinol and *p*-coumaric acid or phloroglucinol carboxylic acid and *p*-hydroxybenzoic acid or phenylacetic acid as the final products (Rao and Cooper, 1995), while metabolism of the flavonol quercetin produces phloroglucinol and protocatechuic acid (Rao *et al.*, 1991). However, the genes and enzymes involved in these processes have not been described.

Herbaspirillum seropedicae SmR1 is a diazotrophic rhizobacterium found associated epiphytically and endophytically with the roots of monocotyledonous flowering plants (*Poaceae*) and they have been shown to be able to penetrate the epidermis at the root elongation zone or lateral root cracks (Pedrosa *et al.*, 2011; Monteiro *et al.*, 2012). This microorganism is also capable of degrading naringenin (Marin *et al.*, 2013). A previous study showed that naringenin altered expression of genes encoding proteins involved in flagella, chemotaxis, and cell wall function and metabolism of aromatic compound (Tadra-Sfeir *et al.*, 2015). In this work, a catabolic pathway is proposed for naringenin metabolism in *H*. seropedicae based on the identification of several intermediates and physiological analyses of knock-out mutants.

### Results and discussion

#### fde genes are required for naringenin utilization as a carbon source

Our previous work (Marin *et al.*, 2013) indicated that the *fde* operon enables naringenin degradation in *H*. seropedicae SmR1. Here, we constructed and characterized mutant strains unable to utilize naringenin as a carbon source in order to identify the role of the *fde* genes in the naringenin degradation pathway.

An initial screen of 6912 random Tn5 mutants of *H*. seropedicae SmR1 allowed the identification of one mutant strain, AMMETn5, unable to grow with naringenin as the sole carbon source. Genomic DNA sequencing revealed that the Tn5 was inserted in the *fdeE* gene (HSERO_RS05045), which encodes a putative monooxygenase located in the *fde* operon (Marin *et al.*, 2013). This strain, AMMETn5, as well as AMM1 (containing a Km resistance cassette in the *fdeA* gene; Marin *et al.*, 2013), was unable to metabolize naringenin (Marin *et al.*, 2013 and Fig. 1).

LC-MS/MS analyses of the AMMETn5 culture supernatant showed that naringenin remained unaltered after 48 h of incubation and no naringenin derived metabolite was found (Supporting Information Fig. S1), while in the supernatant of wild-type SmR1 several compounds accumulated (Fig. 2 and Supporting Information Fig. S1; Table 2). The results clearly showed that insertional inactivation of *fdeE* affected the ability of *H*. seropedicae SmR1 to initiate degradation of naringenin. Blast analyses showed that FdeE shares homology with the FAD-binding domain of the salicylate 1-monooxygenase protein family (Marin *et al.*, 2013). Pérez-Pantoja *et al.* (2009) proposed that oxygenases initiate the activation of the aromatic rings in preparation for cleavage. This activation involves hydroxylation by the addition of one or two oxygen atoms to the substrates, which, under aerobic conditions, are catalysed by enzymatic complexes made of oxygenases and electron carrier proteins (Pérez-Pantoja *et al.*, 2009). The results presented here suggest that FdeE, together with FdeD, a putative non-haeme Rieske protein encoded by *fdeD* in the *fde* operon (Marin *et al.*, 2013), form a complex FdeED, that catalyses the initial hydroxylation of naringenin.

The AMMC mutant strain, carrying an in-frame deletion of most of the *fdeC* gene, was still able to metabolize naringenin, however, as shown by LC-MS analyses a metabolite accumulated in the culture supernatant (Supporting Information Fig. S2). The culture supernatant of this strain when grown in the presence of naringenin accumulated 5,7,8-trihydroxy-2-(4-hydroxyphenyl)-2,3-dihydro-4H-chromen-4-one (Supporting Information Figs S2B and S2C, M-H 287.0580 and 287.0573 Da, compound IIa in Fig. 3), a product of naringenin hydroxylation, and unmodified naringenin (M-H 271.0624 Da, Supporting

![Figure 1](attachment://image.png)

Fig. 1. Mutations in *fdeA* and *fdeE* genes affect the ability of *H*. seropedicae SmR1 to use naringenin as a carbon source. SmR1 (wild-type), AMM1 (*fdeA* polar), AMMETn5 (*fdeE*), AMMC (*fdeC*) and AMMG (*fdeG*) strains grown in NFbHPN medium containing 10 mM naringenin. Colony forming units (CFU) were determined at \( t = 0 \) and 48 h. The graph shows the increase in the viable cells. Data presented are the average of three independent assays.

A new meta-pathway for degradation of flavonoids 3

Fig. 2. Metabolites identified from naringenin degradation by *H. seropedicae* SmR1. Wild-type (SmR1) strain was grown in NFbHPN medium containing naringenin 10 mmol/l as the sole carbon source at 30°C.

A. Full chromatogram of the culture supernatant from SmR1 after 15 h of growth showing peaks identified by mass spectrometry and the $m/z$ of the each ion.

B and C. Show enlarged detail of the same chromatogram, $m/z$ and molecular formulae of the metabolites. These results are representative of three independent experiments.

analyses showed that the FdeC protein shares homology with extradiol dioxygenases (Marin et al., 2013). Dioxygenases catalyse ring fission between two carbon atoms with vicinal hydroxyl groups (an ortho cleavage mechanism; intradiol dioxygenases), or between two carbons adjacent to two hydroxyl groups (a meta cleavage mechanism, performed by extradiol dioxygenases) (Harwood and Parales, 1996; Andújar and Santero, 2003; Mahiuddin et al., 2012). FdeH is probably involved in the A-ring cleavage with FdeC. FdeH contains a cupin domain that is found in several protein families (Marin et al., 2013), including some dioxygenases (Dunwell et al., 2004) such as quercetin 2,3-dioxygenase of *Aspergillus japonicus*, which catalyses the oxidation of quercetin into 2-protocatechuylphloroglucinol carboxylic acid and CO (Fusetti et al., 2002; Dunwell et al., 2004). The results we obtained with mutant strains that had inactive *fdeE* and *fdeC* genes showed that FdeE, likely in complex with FdeD, first hydroxylated naringenin and the A-ring was subsequently cleaved by FdeC. These data were supported by the compounds identified using mass spectrometry: naringenin remained intact in the culture supernatant from the *fdeE* mutant, and compound IIa accumulated in the culture supernatant of *fdeC* mutant strain.

The *fdeG* gene encodes a putative cyclase (Marin et al., 2013). The strain AMMG, constructed by knock-out mutagenesis, contains an in-frame deletion of this gene but is still able to degrade naringenin. The LC-MS profile of the culture supernatant was similar to that of the wild-type. The only observed effect was that the mutation slowed down the degradation process compared to the wild-type strain (data not shown), suggesting that the FdeG protein was not essential for the degradation process.

Intermediates of H. seropedicae SmR1 naringenin degradation

To identify the intermediates and products of naringenin degradation formed by *H. seropedicae* SmR1, the supernatants from cultures grown in the presence of naringenin were analysed by LC-MS/MS and nuclear magnetic resonance (NMR). When SmR1 was cultivated in NFbHPN-malate medium (the usual carbon source) without naringenin only three peaks were observed in the

Information Fig. S2C, compound I in Fig. 3) suggesting that the FdeC protein is responsible for the fission of compound IIa in the naringenin degradation pathway. Blast

© 2016 Society for Applied Microbiology and John Wiley & Sons Ltd, *Environmental Microbiology*, **00**, 00–00

4 A. M. Marin et al.

ion: C₂₁H₂₁O₁₀  
m/z 433.1129 Da  

Compound IIb: Naringenin-7-O-glucoside

ion: C₁₅H₁₁O₅  
m/z 271.0608 Da  

Compound I: Naringenin

ion: C₁₆H₁₃O₆  
m/z 301.0710 Da  

Compound IIc: 3,5-dihydroxy-2-(4-hydroxyphenyl)-7-methoxy-2,3-dihydro-4H-chromen-4-one

Fig. 3. Proposed naringenin degradation pathway in *H. seropedicae* SmR1. Indicated ions were detected by LC-MS analyses. Compounds in orange are proposed, since they were not detected. Compounds producing ions *m/z* 179.0708 Da and *m/z* 161.0602 Da were confirmed by H¹-RMN. Genetic evidence supports the involvement of the enzymes in black; the activities of enzymes in red were proposed based on protein sequence analyses.

FdeDE  
(hydroxylation)

ion: C₁₅H₁₁O₆  
m/z 287.0580 Da  

Compound IIa: 5,7,8-trihydroxy-2-(4-hydroxyphenyl)-2,3-dihydro-4H-chromen-4-one

FdeCH  
(dioxygenation)

Compound III: ((2E,4E)-2,4-dihydroxy-4-[6-(4-hydroxyphenyl)]-2,4-dioxodihydro-2H-pyran-3(4H)-ylidene]but-2-enoic acid)

H₂O

FdeJ

ion: C₁₁H₉O₄  
m/z 205.0503 Da  

Compound IVa: 5-(3,4-dihydroxyphenyl)-3-oxovalero-8-lactone

H₂O

FdeB

ion: C₁₁H₁₁O₅  
m/z 223.0604 Da  

Compound V: 5-hydroxy-5-(4-hydroxyphenyl)-3-oxopentanoic acid

CO₂

(decarboxylation) FdeI

ion: C₁₀H₁₁O₃  
m/z 179.0713 Da  

Compound VI: 4-hydroxy-4-(4-hydroxyphenyl)butan-2-one

H₂O

FdeI

ion: C₁₀H₉O₂  
m/z 161.0604 Da  

Compound VII: (3E)-4-(4-hydroxyphenyl)but-3-en-2-one

Carbon Source

Compound IVb: oxaloacetic acid

Further degradation/ Dead-end product

chromatogram, but none of them were similar to those found in the supernatant of SmR1 culture grown in the presence of naringenin (Supporting Information Fig. S1). This finding suggests that the peaks shown in Supporting Information Fig. S1B and Fig. 2 are products of naringenin degradation. Small differences in the retention times were observed, but MS analyses showed that the molecular formulae and mass spectra were the same. The compounds eluted at 2.72, 4.02, 4.18, 5.15, 5.70 and 7.21 min were identified by MS analyses (Fig. 2A; Table 2), however, those eluted at 2.41, 3.35, 3.68, 4.61 and 4.73 min could not be identified. The peak eluted at 4.18 min (Fig. 2B) exhibited a molecular ion at 179.0713 Da (M-H) and a molecular formula C₁₀H₁₁O₃ (4-hydroxy-4-(4-hydroxyphe-

nyl)butan-2-one; compound VI in Fig. 3), whereas the peak eluted at 5.70 min (Fig. 2C) contained the molecular ions 161.0604 Da (M-H) and 287.0550 Da (M-H), with deduced molecular formulae C₁₀H₉O₂ ((3E)-4-(4-hydroxyphenyl)but-3-en-2-one; compound VII in Fig. 3) and C₁₅H₁₁O₆ (5,7,8-trihydroxy-2-(4-hydroxyphenyl)-2,3-dihydro-4H-chromen-4-one; compound IIa in Fig. 3) respectively. Compounds VI and VII (Fig. 3) were purified by preparative high performance liquid chromatography (HPLC), freeze-dried and analysed by NMR.

The ¹H NMR spectra (Supporting Information Tables S1 and S2) of compound VI (Fig. 3) (eluted at 4.18 min – Fig. 2B; Table 2) showed a spin system of a *p*-substituted aromatic compound at 7.13 (*d*, 8.6 Hz) and 6.65 ppm (*d*, 8.6
Hz), as well as a singlet at 2.08 ppm from a methyl group close to carbonyl and third spin system at 4.88 (ddd, 8.9, 4.6 and 4.5 Hz), 2.58 (dd, 15.0 and 4.6 Hz), 2.71 (dd, 15.0 and 8.9 Hz) and 5.14 ppm (d 4.5 Hz). The signal at 5.14 ppm had no direct correlation and was assigned to a hydroxyl group. The signal at 4.88 ppm was correlated with the oxygenated carbon at 68.5 ppm, and had long-range correlation with carbons at 53.0, 126.8 and 206.8 ppm of an aromatic ring and a carbonyl group. Both hydrogens at 2.58 and 2.71 ppm showed direct correlation with the same carbon at 53.0 ppm, revealing the presence of a methylene group, and long range correlation with the carbons at 68.5 and 206.8 ppm. Additionally, the aromatic hydrogen at 7.13 ppm showed direct correlation with the carbon at 126.8 ppm and long-range correlation with carbons 68.5, 126.8 and 156.1 ppm, supporting the structure of 4-hydroxy-4-(4-hydroxyphenyl)-butan-2-one. The hydroxyl group of the 4-hydroxyphenyl moiety was supported by hydrogens showing long-range correlation with carbon at 156.1 ppm. The ${}^{1}$H NMR spectra of compound VII (Fig. 3) eluted at 5.70 min (Fig. 2C) was consistent with $(3E)$-4-hydroxy-4-(4-hydroxyphenyl)-but-3-en-2-one (Fig. 3), although the hydrogens at 4.88, 2.58 and 2.71 ppm were replaced by the two hydrogens at 6.58 (d, 16.3 Hz) and 7.51 ppm (d, 16.3 Hz) revealing the presence of a trans double bond, suggesting that compound VII (Fig. 3) resulted from dehydration of 4-hydroxy-4-(4-hydroxyphenyl)-butan-2-one (Fig. 3). Hydrogens at 6.58 and 7.51 ppm showed direct correlation with the carbons at 124.5 and 143.7 ppm respectively. The hydrogen at 7.51 showed long-range correlation with aromatic carbon at 130.5 ppm and the carbonyl group at 197.4 ppm supporting the structure of $(3E)$-4-(4-hydroxyphenyl)but-3-en-2-one.

Mass spectrometry analyses of the peaks at 2.72, 4.02 and 5.15 min (Fig. 2) revealed molecular ions of 223.0604 Da (M-H $C_{11} H_{11} O_{5}$, 5-hydroxy-5-(4-hydroxyphenyl)-3-oxopentanoic acid; compound V in Fig. 3), 205.0503 Da (M-H $C_{11} H_{9} O_{4}$, 5-(3,4-dihydroxyphenyl)-3-oxovalero-$\delta$-lactone; compound IVa in Fig. 3), 433.1129 Da (M-H $C_{21} H_{21} O_{10}$, naringenin-7-O-glucoside; compound IIb in Fig. 3) respectively. The peak eluted at 7.21 min (Fig. 2C) showed two molecular ions of 301.0710 Da (M-H $C_{16} H_{13} O_{6}$, 3,5-dihydroxy-2-(4-hydroxyphenyl)-7-methoxy-2,3-dihydro-4$H$-chromen-4-one, compound IIc in Fig. 3) and 271.0608 Da (M-H $C_{15} H_{11} O_{5}$, naringenin, compound I in Fig. 3). MS/MS tandem analyses of the ions of 205.0503 Da (M-H), 287.0550 Da (M-H) and 301.0710 Da (M-H) did not show any ion product (Table 2), as optimum collision energy could not be determined.

Taken together the results revealed a novel pathway for naringenin degradation by *H*. seropedicae SmR1 (Fig. 3). In this pathway, naringenin degradation takes place via hydroxylation of the A-ring by FdeDE, which generates compound IIa (Fig. 3), followed by its meta-cleavage in a

A new meta-pathway for degradation of flavonoids 5 reaction involving FdeC and probably FdeH. The putative product of cleavage by FdeCH (compound III in Fig. 3) was not found in this work. Cleavage of compound III, resulting in compound IVa and IVb (oxaloacetic acid), is likely to be catalysed by FdeJ which shares similarity with NAD-dependent dehydrogenases. The oxaloacetic acid released in this step could be used as a carbon source, to support bacterial growth. Studies of catechin degradation in *Pseudomonas* sp. showed that degradation of taxifolin follows a similar pathway. Taxifolin is first hydroxylated in the A-ring followed by A-ring cleavage with an ortho-dioxygenase, producing oxaloacetic acid and 5-(3,4-dihydroxyphenyl)-4-hydroxy-3-oxovalero-$\delta$-lactone (Das and Rosazza, 2006). Compound IVa (Fig. 3) is hydrolysed, resulting in compound V (M-H 223.0604 Da $C_{11} H_{11} O_{5}$, 5-hydroxy-5-(4-hydroxyphenyl)-3-oxopentanoic acid). This reaction is probably catalysed by FdeB, an alpha/beta hydrolase superfamily protein (Marin et al., 2013). The proteins found in this superfamily catalyse the hydrolysis of C–C, C–N and C–O bonds for several substrates (Holmsquist, 2000). Compound V (5-hydroxy-5-(4-hydroxyphenyl)-3-oxopentanoic acid, Fig. 3) is probably decarboxylated by Fdel, a fumarylacetoacetate hydrolase superfamily protein, into VI (4-hydroxy-4-(4-hydroxynyl)butan-2-one), (Marin et al., 2013). This family of proteins include bifunctional enzymes such as 2-hydroxyhepta-2,4-diene-1,7-dioate (HHDD) isomerase/5-oxopent-3-ene-1,2,5-tricarboxylate (OPET) decarboxylase, involved in the isomerization of 5-carboxymethyl-1,2-hydroxymuconic acid to OPET, and its decarboxylation into HHDD in the homoprotocatechuate degradation pathway (Roper and Cooper, 1993). The next step involves dehydration of compound VI resulting in compound VII (M-H 161.0604 Da $C_{10} H_{9} O_{2}$, $(3E)$-4-(4-hydroxyphenyl)but-3-en-2-one – Fig. 3), a reaction also catalysed by Fdel. Compound VII may be slowly degraded in reactions involving FdeA and FdeG or it may be a dead-end product since it did not disappear completely from the supernatant even after 48 h of incubation.

Other metabolites which remained unaltered during culture growth were also identified in culture supernatants of *H*. seropedicae SmR1 grown in naringenin, these include, methylated, hydroxylated and glycosylated naringenin (Fig. 3), evident from the molecular ions of compound IIc (M-H 301.0710 Da $C_{16} H_{13} O_{6}$, 3,5-dihydroxy-2-(4-hydroxyphenyl)-7-methoxy-2,3-dihydro-4$H$-chromen-4-one) and compound IIb (M-H 433.1129 Da $C_{21} H_{21} O_{10}$, naringenin 7-O-glucoside). Fragmentation (tandem MS/MS) of 433.1129 Da (M-H) showed an ion product of 271.0606 Da (M-H) due to the loss of a glucosyl moiety, possibly glucose, however, this could be confirmed unequivocally by the analyses performed. *H*. seropedicae SmR1 appeared to be unable to further metabolize these naringenin derivatives since they remained in the culture medium. Flavonoid

© 2016 Society for Applied Microbiology and John Wiley & Sons Ltd, *Environmental Microbiology*, **00**, 00–00
Table 1. Strains and plasmids.

| Strains or plasmids | Genotype or relevant characteristics | References |
|--------------------|-------------------------------------|------------|
| H. seropedicae     |                                     |            |
| SmR1               | Sm<sup>R</sup>, Nif<sup>+</sup>      | Souza et al. (2000) |
| AMM1               | Site-specific homologous inactivation of *fdeA* gene by kanamycin cassette insertion; polar mutation; Km<sup>R</sup> | Marin et al. (2013) |
| AMMETn5            | Chromosomal inactivation of the *fdeE* (Hsero_RS05045) gene by Tn5 transposon insertion; polar mutation; Km<sup>R</sup> | This work |
| AMMC               | Site-specific homologous inactivation of the *fdeC* (HSERO_RS05035) gene by in-frame deletion; Sm<sup>R</sup> | This work |
| AMMG               | Site-specific homologous inactivation of the *fdeG* (HSERO_RS05055) gene by in-frame deletion; Sm<sup>R</sup> | This work |
| Escherichia coli   |                                     |            |
| CC118λpir          | Host of suicide vector pUT-KM        | Herrero et al. (1990) |
| S17.1              | Contains *tra* genes; Sm<sup>R</sup> | Simon et al. (1983) |
| Plasmids           |                                     |            |
| pUT-KM             | Source of mini-Tn5; RP4mob; Km<sup>R</sup> | de Lorenzo et al. (1990). |
| pRK600             | Provides the *tra* functions for mobilization of the pUT plasmid; Cm<sup>R</sup> | Keen et al. (1988) |
| pK18mobsacB        | RP4mob, Km<sup>R</sup>, sacB         | Schäfer et al. (1994) |
| pK18-fdeCdel       | pK18mobsacB vector containing an in-frame deletion in *fdeC* (Hsero_1005) | This work |
| pK18-fdeGdel       | pK18mobsacB vector containing an in-frame deletion in *fdeG* (Hsero_1009) | This work |

glycosylation can be performed by glycosyltransferases from several organisms (Mackenzie et al., 1997); the enzyme responsible for this activity in *H. seropedicae* was not identified. Glycosylation reactions are a way of enhancing the solubility of these compounds, as well as of decreasing their toxic effects (Paquette et al., 2003).

In this study, metabolites of naringenin degradation by *H. seropedicae* SmR1 were identified and a novel degradation pathway through A-ring cleavage was proposed (Fig. 3). Genetic analyses allowed us to infer the functions of some of the *fde* operon-encoded proteins: FdeE is involved in the first step of the naringenin catabolism pathway, since a FdeE mutant strain was unable to metabolize this flavonoid, and FdeC is involved in the A-ring cleavage. These initial steps are in agreement with other similar catabolic pathways, in which aromatic compounds are

prepared for cleavage (Arai et al., 2000; Perez-Pantoja et al., 2009). Compound VII remains in the culture supernatant, when *H. seropedicae* SmR1 are grown in the presence of naringenin, and as yet, no metabolic pathway has been described for the use of compound VII. As such, the bacterial growth observed when naringenin is used as a carbon source appears to be supported by energy obtained via oxaloacetic acid production.

Experimental procedures

Bacterial strains, culture media and plasmids

*Herbaspirillum seropedicae* strains (Table 1) were grown in NFbHPN medium (Klassen et al., 1997). The antibiotics streptomycin (80 μg/ml) and kanamycin (500 μg/ml) were used as required. When indicated 10 mmol/l naringenin (Sigma-

Table 2. Identification of naringenin metabolites in culture supernatant from *H. seropedicae* SmR1 via MS and MS/MS analyses.

| Retention time (min) | Ion Molecular formulae | Detected mass | Calculated mass | Error (ppm) | Fragmentation pattern (MS/MS) | Ion fragment molecular formulae | Detected mass | Calculated mass | Error (ppm) |
|----------------------|-----------------------|---------------|-----------------|-------------|-------------------------------|----------------------------------|---------------|-----------------|-------------|
| 4.02                 | C<sub>11</sub>H<sub>9</sub>O<sub>4</sub> | 205.0503        | 205.0500     | 1.0    | Not detected                | None                               | None          | None            | None        |
| 2.72                 | C<sub>11</sub>H<sub>11</sub>O<sub>5</sub> | 223.0604        | 223.0606     | -0.9   | Detected                    | C<sub>7</sub>H<sub>5</sub>O<sub>2</sub> | 121.0291      | 121.0290        | 0.8         |
| 5.70                 | C<sub>15</sub>H<sub>11</sub>O<sub>6</sub> | 287.0550        | 287.0555     | -2.1   | Not detected                | None                               | None          | None            | None        |
|                      |                       | 287.0580        |                 |             |                                   |                                  |               |                 |             |
|                      |                       | 287.0573        |                 |             |                                   |                                  |               |                 |             |
|                      | C<sub>10</sub>H<sub>9</sub>O<sub>2</sub> | 161.0604        | 161.0602     | 0.6    | Detected                    | C<sub>7</sub>H<sub>5</sub>O<sub>2</sub> | 121.0288      | 121.0290        | -1.7        |
|                      |                       |                 |                 |             |                                   | C<sub>8</sub>H<sub>5</sub>O         | 117.0337      | 117.0340        | -2.6        |
| 4.18                 | C<sub>10</sub>H<sub>11</sub>O<sub>3</sub> | 179.0713        | 179.0708     | 2.8    | Detected                    | C<sub>8</sub>H<sub>7</sub>O<sub>2</sub> | 135.0444      | 135.0446        | -1.5        |
| 5.15                 | C<sub>21</sub>H<sub>21</sub>O<sub>10</sub> | 433.1129        | 433.1134     | -1.4   | Detected                    | C<sub>15</sub>H<sub>11</sub>O<sub>5</sub> | 271.0606      | 271.0606        | 0           |
| 7.21                 | C<sub>16</sub>H<sub>13</sub>O<sub>6</sub> | 301.0710        | 301.0712     | -0.7   | Not detected                | None                               | None          | None            | None        |
|                      | C<sub>15</sub>H<sub>11</sub>O<sub>5</sub> | 271.0608        | 271.0606     | 0.7    | Detected                    | C<sub>7</sub>H<sub>3</sub>O<sub>4</sub> | 151.0029      | 151.0031        | -1.3        |
|                      |                       | 271.0624        |                 |             |                                   | C<sub>8</sub>H<sub>7</sub>O         | 119.0490      | 119.0487        | 2.5         |
|                      |                       |                 |                 |             |                                   | C<sub>6</sub>H<sub>3</sub>O<sub>2</sub> | 107.0131      | 107.0133        | -0.2        |
Aldrich) was used as carbon source in 2,2,4,4,6,8,8-heptamethylnonane. *Escherichia coli* strains (Table 1) were grown in Luria Bertani (LB) medium (Sambrook et al., 1989) containing streptomycin (25 μg/ml), chloramphenicol (30 μg/ml) and kanamycin (50 μg/ml), as required.

For growth assays, \(10^5\) cells of *H. seropedicae* strains SmR1 (wild-type), AMM1 (polar \(fdeA^{-}\)), AMMETn5 (\(fdeE^{-}\)), AMMC (\(fdeC^{-}\)) and AMMG (\(fdeG^{-}\)) were inoculated in NFbHPN medium containing 10 mmol/l naringenin. These strains were incubated in the growth medium for a period of 2–3 days at \(30^\circ\)C in an orbital shaker at 120 r.p.m. Culture samples were collected at the indicated times and serial dilutions were plated on NFbHPN-malate and incubated for 24–48 h at \(30^\circ\)C.

### Generation of AMMC and AMMG chromosomal knock-out mutants

AMMC and AMMG mutant strains were obtained by deleting the internal portions of *fdeC* and *fdeG* genes respectively. The deletions were carried out *in frame* in order to maintain the transcription of the downstream genes of the *fde* operon. The deleted genes were inserted in the pK18mobsacB vector (Schäfer et al., 1994); these plasmids were called pK18-fdeCdel and pK18-fdeGdel respectively (Table 1). Both plasmids were introduced in *H. seropedicae* SmR1 by conjugation with an *E. coli* S17.1 strain carrying the mutagenic plasmids. Two recombination events were necessary to obtain each mutant strain. An initial recombinant was selected in NFbHPN-malate medium containing kanamycin. For the selection of clones in which recombination step has taken place, cells were spread on 10% sucrose. The double crossover in final recombination resulted in AMMC and AMMG mutant strains that were confirmed by PCR (data not shown). Through mutagenesis, the central portions of both the FdeC wild-type protein (308 amino acids) and FdeG wild-type protein (271 amino acids) were deleted, resulting in proteins of 30 and 43 amino acids respectively.

### Random mutant library

Construction of the random mutant library was adapted from Molina-Henares et al. (2009). Triparental conjugation was performed to obtain mutant strains, in which *H. seropedicae* SmR1 was the recipient, *E. coli* CC118 λpir pUT-Km (carrying Tn5 mini-transposon) was the donor strain and HB101 contained the pRK600 helper plasmid. The mixture of bacteria was incubated at \(30^\circ\)C for 1 h, centrifuged and resuspended in 1 ml of NFbHPN-malate liquid medium. Aliquots of 100 μl were deposited onto filter membrane on NFbHPN-LA (3:1) medium, and incubated at \(30^\circ\)C for 24 h. The cells were then resuspended in NFbHPN-malate and 750 μl were plated on LA containing streptomycin (80 μg/ml) and kanamycin (500 μg/ml). After 2 days incubation at \(30^\circ\)C, the colonies were transferred to 96-well plates for phenotypic assays. A total of 6912 mutants were screened in 96-well plates for growth on NFbHPN medium containing 10 mmol/l naringenin as sole carbon source. After 3 days of incubation on a rotary shaker at \(30^\circ\)C, the plates were visually analysed and a mutant strain, named AMMETn5, that failed to grow on naringenin was isolated. Transposon insertion sites were determined by arbitrary primed PCR followed by sequencing, as described previously (Espinosa-Urgel et al., 2000; Molina-Henares et al., 2009). Comparison of the obtained sequences with genome databases was performed using BLAST (http://www.ncbi.nlm.nih.gov/blast/BL).

#### Analyses of naringenin degradation products

LC-MS/MS analyses from culture supernatant were performed using a Waters Acquity H class ultra performance liquid chromatography (UPLC) coupled to a high resolution WATERS LCT Premier XE Q-TOF mass spectrometry or a Shimadzu Prominence ultra fast liquid chromatography (UFLC) coupled with a high resolution Bruker micrOTOF-QII. The UPLC system was equipped with an Acquity HSS T3 (2.1 × 50 mm²; 1.8 μm) chromatographic column and a gradient system consisting of acetonitrile (ACN) and \(\mathrm{H}_2\mathrm{O}\), starting with 5% ACN (containing 0.1% trifluoroacetic acid) for 5 min and then increasing to 95% for 10 min at a flow rate of 0.3 ml/min. The column temperature was maintained at \(40^\circ\)C and the samples at \(4^\circ\)C. Electrospray was utilized for ionization (\(100^\circ\)C) and helium gas was used for collision at 20 l/h. The software MassLynx v4.1 from Waters was used for run and post-run analyses. The UFLC system was equipped with a Shim-pack XR-ODS (3.0 × 30 mm²; 2.2 μm) chromatographic column and a gradient system consisting of ACN and \(\mathrm{H}_2\mathrm{O}\)/formic acid (0.1%), starting with 10% ACN for 7 min and increasing to 60% over an additional 11 min at a flow rate of 0.4 ml/min. The column temperature was maintained at \(40^\circ\)C and the samples at \(4^\circ\)C. Electrospray ionization (\(180^\circ\)C) was used and nitrogen gas was used for collision at 5 l/h. The software HyStar was used for run analyses, and Compass v4.1 was utilized for post-run analyses (both from Bruker). All MS analyses were carried out in the negative ionization mode.

One-dimensional and two-dimensional NMR data were acquired at 303 K in MeOD-\(d_4\) on a Bruker Avance III 600 NMR spectrometer operating at 14.1 Tesla, observing \({}^1\mathrm{H}\) and \({}^{13}\mathrm{C}\) at 600.13 and 150.52 MHz respectively. The spectrometer was equipped with a 5 mm quadrinuclear inverse detection probe with z-gradient. One-bond and long-range \({}^1\mathrm{H}-{ }^{13}\mathrm{C}\) correlation from HSQC and HMBC NMR experiments were optimized for an average coupling constant \({}^1J_{(\mathrm{C},\mathrm{H})}\) and \({}^{\mathrm{LR}}J_{(\mathrm{C},\mathrm{H})}\) of 140 and 8 Hz respectively. All \({}^1\mathrm{H}\) and \({}^{13}\mathrm{C}\) NMR chemical shifts (\(\delta\)) are given in ppm related to the TMS signal at 0.00 ppm as an internal reference, and the coupling constants (\(J\)) in Hz.

### Acknowledgements

The authors would like to thank INCT-Fixação Biológica de Nitrogênio, CAPES, Fundação Araucária, Junta de Andalucía (CVI-3010, Plan Nacional BIO2010-17227, ST-FLOW), and UFPR for financial support and fellowships; we thank Juan Moliz Medina, José Miguel Ramos López and Yolanda Madrid Fernández from Granada University for help with several LC-MS analyses. We also thank Roseli Prado and Valter Antonio de Baura for technical support.

© 2016 Society for Applied Microbiology and John Wiley & Sons Ltd, *Environmental Microbiology*, **00**, 00–00
References

Andújar, E., and Santero, E. (2003) Site-directed mutagenesis of an extradiol dioxygenase involved in tetralin biodegradation identifies residues important for activity or substrate specificity. *Microbiology* **149**: 1559–1567.

Arai, H., Ohishi, T., Chang, M.Y., and Kudo, T. (2000) Arrangement and regulation of the genes for meta-pathway enzymes required for degradation of phenol in *Comamonas testosteroni* TA441. *Microbiology* **146**: 1707–1715.

Bartmanska, A., Tronina, T., and Huszcza, E. (2013) Microbial sulfation of 8-Prenylaringenin. *Z Naturforsch* **68**: 231–235.

Braune, A., Gütschow M., Engst, W., and Blaut, M. (2001) Degradation of quercetin and luteolin by *Eubacterium ramulus*. *Appl Environ Microbiol* **67**: 5558–5567.

Brencic, A., and Winans, S.C. (2005) Detection of and response to signals involved in host-microbe interactions by plant-associated bacteria. *Microbiol Mol Biol Rev* **69**: 155–194.

Cesco, S., Neumann, G., Tomasi, N., Pinton, R., and Weisskopf, L. (2010) Release of plant-borne flavonoids into the rhizosphere and their role in plant nutrition. *Plant Soil* **329**: 1–25.

Cheng, K.G., Krishnamurty, H.G., Jones, G.A., and Simpson, F.J. (1971) Identification of products produced by the anaerobic degradation of naringin by *Butyriribrio* sp. C3. *Can J Microbiol* **17**: 129–131.

Das, S., and Rosazza, J.P.N. (2006) Microbial and enzymatic transformation of flavonoids. *J Nat Prod* **69**: 499–508.

Dunwell, J.M., Purvis, A., and Khuri, S. (2004) Cupins: the most functionally diverse protein superfamily? *Phytochemistry* **65**: 7–17.

Espinosa-Urgel, M., Salido, A., and Ramos, J.L. (2000) Genetic analysis of functions involved in adhesion of *Pseudomonas putida* to seeds. *J Bacteriol* **182**: 2363–2369.

Fusetti, F., Schroter, K.H., Steiner, R.A., van Noort, P.I., Pijning, T., Rozeboom, H.J., et al. (2002) Crystal structure of the copper-containing quercetin 2,3-dioxygenase from *Aspergillus japonicus*. *Structure* **10**: 259–268.

Harwood, C.S., and Parales, R.E. (1996) The β-ketoadipate pathway and the biology of self-identity. *Annu Rev Microbiol* **50**: 553–590.

Herrero, M., De Lorenzo, V., and Timmis, K.N. (1990) Transposon vectors containing non-antibiotic resistance selection markers for cloning and stable chromosomal insertion of foreign genes in Gram-negative bacteria. *J Bacteriol* **172**: 6557–6567.

Holmsquist, M. (2000) Alpha/beta-hydrolase fold enzymes: structures, functions and mechanisms. *Curr Protein Pept Sci* **1**: 209–235.

Hosny, M., Dhar, K., and Rosazza, J.P.N. (2001) Hydroxylations and methylations of quercetin, fisetin and catechin by *Streptomyces griseus*. *J Nat Prod* **64**: 462–465.

Ibrahim, A.-R. S. (2000) Sulfation of naringenin by *Cunninghamella elegans*. *Phytochemistry* **53**: 209–212.

Keen, N.T., Tamaki, S., Kobayashi, D., and Trolinger, D. (1988) Improved broad-host-range plasmids for DNA cloning in Gram-negative bacteria. *Gene* **70**: 191–197.

Klassen, G., Pedrosa, F.O., Souza, E.M., Funayama, S., and Rigo, L.U. (1997) Effect of nitrogen compounds on nitro-nase activity in *Herbaspirillum seropedicae* SMR1. *Can J Microbiol* **43**: 887–891.

de Lorenzo, V., Herrero, M., Jakubzik, U., and Timmis, K.N. (1990) Mini-Tn5 transposon derivatives for insertion mutagenesis, promoter probing, and chromosomal insertion of cloned DNA in Gram-negative Eubacteria. *J Bacteriol* **172**: 6568–6572.

Mackenzie, P.I., Owens, I.S., Burchell, B., Bock, K.W., Bairoch, A., Belanger, A., et al. (1997) The UDP-glycosyltransferase gene superfamily: recommended nomenclature update based on evolutionary divergence. *Pharmacogenetics* **7**: 255–269.

Mahiuddin Md., Fakhruddin, A.N.M., and Abdullah-Al-Mahin (2012) Degradation of phenol via meta cleavage pathway by *Pseudomonas fluorescens* PU1. *ISRN Microbiol*. doi: 10.5402/2012/741820.

Marin, A.M., Souza, E.M., Pedrosa, F.O., Souza, L.M., Sassaki, G.L., Baura, V.A., et al. (2013) Naringenin degradation by the endophytic diazotroph *Herbaspirillum seropedicae* SmR1. *Microbiology* **159**: 167–175.

Molina-Henares, M.A., García-Salamanca, A., Molina-Henares, A.J., de la Torre, J., Herrera, M.C., Ramos, J.L., and Duque, E. (2009) Functional analysis of aromatic biosynthetic pathways in *Pseudomonas putida* KT2440. *Microb Biotechnol* **2**: 91–100.

Monteiro, R.A., Balsanelli, E., Wassem, R., Marin, M.A., Brusamarello-Santos, L.C.C., Schmidt, M.A., et al. (2012) *Herbaspirillum*-plant interactions: microscopical, histological and molecular aspects. *Plant Soil* **356**: 175–196.

Mulligan, J.Y., and Long, S.R. (1985) Induction of *Rhizobium meliloti nodC* expression by plant exudate requires *nod*. *Proc Natl Acad Sci USA* **83**: 6609–6613.

Paquette, S., Mollerb, B.L., and Bakb, S. (2003) On the origin of family 1 plant glycosyltransferases. *Phytochemistry* **62**: 399–413.

Pedrosa, F.O., Monteiro, R.A., Wassem, R., Cruz, L.M., Ayub, R.A., Colauto, N.B., et al. (2011) Genome of *Herbaspirillum seropedicae* SmR1, a specialized diazotrophic endophyte of tropical grasses. *PLoS Genet* **7**: e1002064. doi:10.1371/journal.pgen.1002064.

Pérez-Pantoja, D., González, B., and Pieper, D.H. (2009) Aerobic degradation of aromatic hydrocarbons. Berlin, Germany: Springer-Verlag.

Pillai, B.V.S., and Swarup, S. (2002) Elucidation of the flavonoid catabolism pathway in *Pseudomonas putida* PML2 by comparative metabolic profiling. *Appl Environ Microbiol* **68**: 143–151.

Rao, J.R., and Cooper, J.E. (1994) Rhizobia catabolize *nod* gene-inducing flavonoids via C-ring fission mechanism. *J Bacteriol* **176**: 5409–5413.

Rao, J.R., and Cooper, J.E. (1995) Soybean nodulating rhizobia modify *nod* genes inducers daidzein and genistein to yield aromatic products that can influence gene-inducing activity. *Mol Plant Microbe Interact* **8**: 855–862.

Rao, J.R., Sharma, N.D., Hamilton, J.T.G., Boyd, D.R., and Cooper, J.E. (1991) Biotransformation of the pentahydroxy flavone quercetin by *Rhizobium loti* and *Bradyrhizobium* strains (*Lotus*). *Appl Environ Microbiol* **57**: 1563–1565.

Reddy, P.M., Rendón-Anaya, M., del Río, M.D.S., and Khandual, S. (2007) Flavonoids as signaling molecules and regulators of root nodule development. *Dyn Soil Dyn Plant* **1**: 83–94.

Roper, D.I., and Cooper, R.A. (1993) Purification, nucleotide sequence and some properties of a bifunctional isomerase/ decarboxylase from the homoprotocatechuate degradative pathway of *Escherichia coli*. *Eur J Biochem* 217: 575–580.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) *Molecular cloning: a laboratory manual.* New York: Cold Spring Harbor.

Schäfer, A., Tauch, A., Jäger, W., Kalinowski, J., Thierbach, G., and Pühler, A. (1994) Small mobilizable multi-purpose cloning vector derived from *Escherichia coli* plasmids pK18 and pK19: selection of defined deletions in the chromosome of *Corynebacterium glutamicum*. *Gene* 145: 69–73.

Schaab, M.R., Barney, B.M., and Francisco, W.A. (2006) Kinetic and spectroscopic studies on the quercetin 2,3-dioxygenase from *Bacillus subtilis*. *Biochemistry* 45: 1009–1016.

Schneider, H., and Blaut, M. (2000) Anaerobic degradation of flavonoids by *Eubacterium ramulus*. *Arch Microbiol* 173: 71–75.

Shaw, L.J., Morris, P., and Hooker, J.E. (2006) Perception and modification of plant flavonoid signals by rhizosphere microorganisms. *Environ Microbiol* 8: 1867–1880.

Simon, R., Priefer, U., and Pühler A. (1983) A Broad Host Range Mobilization System for In Vivo Genetic Engineering: Transposon Mutagenesis in Gram Negative Bacteria. *Biotechnology* 1: 784–791.

Souza, E.M., Pedrosa, F.O., Rigo, L.U., Machado, H.B., and Yates, M.G. (2000) Expression of the *nifA* gene of *Herbaspirillum seropedicae*: role of NtrC and NifA binding-sites and of -24/-12 promoter element. *Microbiology* 146: 1407–1418.

Tadra-Sfeir, M.Z., Faoro, H., Camilios-Neto, D., Brusamarello-Santos, L., Balsanelli, E., Weiss, V., *et al.* (2015) Genome wide transcriptional profiling of *Herbaspirillum seropedicae* SmR1 grown in the presence of naringenin. *Front Microbiol* 6: article 491.


Supporting information

Additional Supporting Information may be found in the online version of this article at the publisher’s web-site:

Fig. S1. Mutation in *fdeE* blocks the first step of naringenin degradation.
A. Uninoculated NFbHPN medium containing naringenin (10 mmol/l) after a 48 h incubation at 30°C.
B. Culture supernatant from wild-type (SmR1) strain grown in NFbHPN in the presence of naringenin (10 mmol/l). Box indicates the metabolites produced by the wild-type after a 48 h incubation at 30°C.
C. Culture supernatant from AMMETn5 mutant strain grown in NFbHPN with naringenin (10 mmol/l) as carbon source after a 48 h incubation at 30°C. Arrows indicate naringenin (7.03 or 7.02 min). These results are representative of three independent experiments.

Fig. S2. Mutation in *fdeC* blocks naringenin degradation at the second step of degradation pathway. Mutant strain AMMC was grown in NFbHPN medium containing naringenin (10 mmol/l) as sole carbon source at 30°C.
A. Culture supernatant from AMMC strain after a 48 h incubation.
B. Identification of ion *m/z* 287.0580 Da (M-H), eluted at 9.5 min.
C. At 9.7 min, *m/z* 287.0573 Da (M-H) is co-eluted with residual naringenin *m/z* 271.0624 Da (M-H). Dotted arrow indicates the 5,7,8-trihydroxy-2-(4-hydroxyphenyl)-2,3-dihydro-4H-chromen-4-one ion structure. Black arrow indicates naringenin in the chromatogram profile (A) and in the naringenin MS profile (C). This is a representative assay of three independent experiments.

Table S1. Assignment of ${}^{1}\mathrm{H}$ and ${}^{13}\mathrm{C}$ NMR chemical shifts ($\delta$) of compounds VI.

Table S2. Assignment of ${}^{1}\mathrm{H}$ and ${}^{13}\mathrm{C}$ NMR chemical shifts ($\delta$) of compounds VII.
