
The myogenic electric organ of  
Sternopygus macrurus: a non-contractile  
tissue with a skeletal muscle  
transcriptome  

Matthew Pinch${}^{1,*}$, Robert Güth${}^{1,*}$, Manoj P. Samanta${}^{2}$, Alexander Chaidez${}^{1}$ and Graciela A. Unguez${}^{1}$  

${}^{1}$ Department of Biology, New Mexico State University, Las Cruces, NM, United States  
${}^{2}$ Systemix Institute, Redmond, WA, United States  
${}^{*}$ These authors contributed equally to this work.  

---

### ABSTRACT

In most electric fish species, the electric organ (EO) derives from striated muscle cells that suppress many muscle properties. In the gymnotiform *Sternopygus macrurus*, mature electrocytes, the current-producing cells of the EO, do not contain sarcomeres, yet they continue to make some cytoskeletal and sarcomeric proteins and the muscle transcription factors (MTFs) that induce their expression. In order to more comprehensively examine the transcriptional regulation of genes associated with the formation and maintenance of the contractile sarcomere complex, results from expression analysis using qRT-PCR were informed by deep RNA sequencing of transcriptomes and miRNA compositions of muscle and EO tissues from adult *S. macrurus*. Our data show that: (1) components associated with the homeostasis of the sarcomere and sarcomere-sarcolemma linkage were transcribed in EO at levels similar to those in muscle; (2) MTF families associated with activation of the skeletal muscle program were not differentially expressed between these tissues; and (3) a set of microRNAs that are implicated in regulation of the muscle phenotype are enriched in EO. These data support the development of a unique and highly specialized non-contractile electrogenic cell that emerges from a striated phenotype and further differentiates with little modification in its transcript composition. This comprehensive analysis of parallel mRNA and miRNA profiles is not only a foundation for functional studies aimed at identifying mechanisms underlying the transcription-independent myogenic program in *S. macrurus* EO, but also has important implications to many vertebrate cell types that independently activate or suppress specific features of the skeletal muscle program.

---

**Subjects** Aquaculture, Fisheries and Fish Science, Developmental Biology, Evolutionary Studies, Molecular Biology  
**Keywords** Transcriptome, Skeletal muscle, Electric organ, Electric fish, Myogenic regulatory factors, Myogenic transcriptome, Myogenic phenotype, Electrocyte phenotype  

---

### INTRODUCTION

Skeletal muscles are crucial for animals to perform behaviors that are necessary for survival in a changeable environment. These striated tissues are comprised of cells that,

---

**How to cite this article** Pinch et al. (2016), The myogenic electric organ of *Sternopygus macrurus*: a non-contractile tissue with a skeletal muscle transcriptome. PeerJ 4:e1828; DOI 10.7717/peerj.1828
although highly specialized for contraction and force generation, express a wide range of morphological, metabolic and structural properties. Characterizing different muscle cell phenotypes and determining how they are generated during development have been topics of intensive investigation ([Blais, 2005](https://doi.org/10.1083/jcb.200409076); [Wigmore & Evans, 2002](https://doi.org/10.1083/jcb.200204059); [Musumeci et al., 2015](https://doi.org/10.1016/j.ydbio.2015.03.010)). Since the phenotypic properties of fully differentiated muscle fibers are not fixed, numerous studies have also focused on identifying the processes involved in remodeling muscle fibers during postnatal stages ([Gundersen, 2011](https://doi.org/10.1016/j.molmet.2011.02.001); [Gunning & Hardeman, 1991](https://doi.org/10.1016/0167-4889(91)90011-8); [Schiaffino & Reggiani, 2011](https://doi.org/10.1016/j.ydbio.2011.03.017)). Cell signaling pathways and key regulatory factors associated with inducing not only the skeletal muscle phenotype, but specific muscle fiber type properties have also been discovered ([Buckingham & Rigby, 2014](https://doi.org/10.1016/j.cell.2014.01.013); [Schiaffino & Reggiani, 2011](https://doi.org/10.1016/j.ydbio.2011.03.017); [McCarthy, 2011](https://doi.org/10.1016/j.jmb.2011.03.017)). Together, data from these studies have led to our current understanding that changes in environmental and intrinsic factors can affect gene transcription, or expression of genes, including turning genes completely on or off, or simply adjusting the levels of transcripts produced, and as a result influence muscle cell fate decisions throughout the life of the organism. Unfortunately, the existing evidence gives little insight into the coordination of subsets of muscle protein systems such as the genes associated with the different regions of the sarcomere, the structure that fully defines the adult skeletal muscle phenotype, and genes involved in sarcomere homeostasis. Studies on vertebrates wherein regulation of subsets of muscle genes are independently activated or repressed, and can be studied with high precision, are likely to further our understanding of some of these basic processes in muscle biology.

Electric fishes represent an excellent experimental model to address these unresolved questions. These vertebrate teleosts native to South America and Africa are unique in that they possess an electric organ (EO) that is specialized for the production of an electrical field outside the body ([Bennett, 1971](https://doi.org/10.1016/S0022-0981(71)80001-8); [Bass, 1986](https://doi.org/10.1016/0022-0981(86)90001-9)). This electric field is essential for navigation, communication and mate selection behaviors. In all electric fish species known to date, the current producing cells of the EO, i.e., electrocytes, derive from skeletal muscle cells ([Bennett, 1971](https://doi.org/10.1016/S0022-0981(71)80001-8); [Fox & Richardson, 1978](https://doi.org/10.1016/0022-0981(78)90001-8); [Fox & Richardson, 1979](https://doi.org/10.1016/0022-0981(79)90001-8)). Fully differentiated electrocytes, however, do not down-regulate the muscle program completely. In the gymnotiform *Sternopygus macrurus*, mature electrocytes are multinucleated target cells of cholinergic neurons like skeletal muscle fibers ([Unguez & Zakon, 1998](https://doi.org/10.1016/S0022-0981(98)90001-8); [Cuellar, Kim & Unguez, 2006](https://doi.org/10.1016/j.ydbio.2006.01.017)). Electrocytes also contain muscle proteins like desmin, titin, α-actin and α-actinin ([Cuellar, Kim & Unguez, 2006](https://doi.org/10.1016/j.ydbio.2006.01.017); [Kim et al., 2008](https://doi.org/10.1016/j.ydbio.2008.01.017)). However, unlike their precursor muscle cells, mature electrocytes do not contain sarcomeres or T-tubules ([Unguez & Zakon, 1998](https://doi.org/10.1016/S0022-0981(98)90001-8)). Interestingly, although fully differentiated electrocytes in *S. macrurus* transcribe some sarcomeric genes including slow and fast myosin heavy chain isoforms, and troponin-T, they do not contain the corresponding proteins ([Cuellar, Kim & Unguez, 2006](https://doi.org/10.1016/j.ydbio.2006.01.017)). Even more, members of the MyoD family of transcription factors (MTFs), i.e., *myoD*, *myogenin*, *myf5* and *myf6* (*mrf4*, *herculin*), are transcribed at levels that are similar to, or even higher, than those found in their skeletal muscle precursors ([Kim et al., 2004](https://doi.org/10.1016/j.ydbio.2004.01.017); [Kim et al., 2008](https://doi.org/10.1016/j.ydbio.2008.01.017)). Moreover, these *S. macrurus*-specific MTFs can activate the myogenic program in mouse non-muscle cells following forced expression ([Kim et al., 2009](https://doi.org/10.1016/j.ydbio.2009.01.017)). That mature electrocytes in *S. macrurus* contain MTFs is intriguing given that expression of MTFs in other vertebratesis linked to the activation of the striated muscle program (Buckingham & Rigby, 2014).

Hence, unlike mammalian skeletal muscles, MTF-independent transcriptional events are likely to influence the homeostasis of muscle-specific properties of mature electrocytes in *S. macrurus* after differentiation raising an argument for determining the extent to which transcriptional regulation of myogenesis is conserved across different vertebrate species.

In the present study we extended our analyses of the regulation of sarcomere gene expression in *S. macrurus* electrocytes to include sets of genes associated with different regions of the sarcomere, i.e., the costamere, Z-disk and I-A-M band, protein degradation pathways including the ubiquitin-proteasome system (UPS) and the autophagy pathway, and transcription factors known to regulate the myogenic phenotype other than the MyoD family. In addition, we also performed deep RNA sequencing (RNA-seq) to more efficiently identify genes that are differentially expressed in muscle and electrocytes, and facilitate our investigations aimed at elucidating gene networks involved in modifying the myogenic program in electrocytes.

Our current data showed that all genes known to make up the mammalian sarcomere are transcribed in the mature EO of *S. macrurus*, and these were expressed at levels similar to those detected in skeletal muscle. Our expression analyses also established that genes belonging to protein degradation pathways in striated muscle were expressed very similarly in mRNA quantity in both muscle and EO tissues arguing against an elevated breakdown of sarcomeric proteins taking place in electrocytes. Moreover, our RNA-seq data revealed that the mature EO maintains a transcriptome profile that is over ninety percent comparable to that of its precursor striated muscle. Thus, mature electrocytes in *S. macrurus* represent myogenically derived cells that became highly specialized non-contractile electrogenic cells without changing their striated muscle transcriptome. Just as our transcript profile shows that virtually all transcripts associated with the sarcomere are similarly expressed in muscle and EO, we found that myomirs miR-1/133/206 are similarly expressed in both tissues. However, we identified a set of microRNAs associated with the regulation of contractile muscle phenotype including miR-30a/193b/365 that are enriched in EO. Our transcript expression analysis coupled with the differential expression of microRNAs described here provides new scenarios by which transcriptional and post-transcriptional events may regulate specific contractile properties in a vertebrate teleost.

## MATERIALS AND METHODS

*Sternopygus macrurus* were obtained commercially from Ornamental Fish (Miami, FL). Adult fish of undetermined sex and 30–50 cm in length were housed individually in 15-to 20-gallon aerated aquaria maintained at 25–28°C and fed live red worms (Armstrong's Cricket Farm, West Monroe, LA) three times weekly. All fish used in this study had been in the aquaria for a minimum of one year. All fish were anesthetized in 2-phenoxyethanol (1.0 mL/L), and tail tissues were harvested for mRNA and microRNA (miRNA) detection and quantification using quantitative RT-PCR (qRT-PCR), deep RNA sequencing (RNA-seq), and in situ hybridization. In *S. macrurus*, there is a well-defined anatomical compartmentalization of skeletal muscle and EO tissues (Unguez & Zakon, 1998), which greatlyfacilitates their dissection and isolation with little to no tissue cross-contamination. Ventral muscle and EO tissues were excised, immediately immersed in RNAlater™ (Ambion, Austin, TX, USA) and stored at $-80^{\circ} \mathrm{C}$ until RNA extraction for subsequent mRNA analysis with RNA-seq $(n=1$ fish), qRT-PCR $(n=4$ fish), and miRNA analysis with qRT-PCR $(n=3$ fish). In a fourth group of adult control fish $(n=2)$, the distal tail was amputated and frozen in isopentane cooled in liquid nitrogen for *in situ* hybridization processing. Following tissue and tail dissections, fish were euthanized by an overdose in 2-phenoxyethanol $(1.0 \mathrm{mL} / \mathrm{L})$. All animal experimental procedures strictly conformed to the American Physiological Society Animal Care Guidelines and were approved by the Institutional Animal Care and Use Committee at New Mexico State University (Approval #2014-044).

### Determination of expression levels of mRNAs and miRNAs associated with sarcomeric structures in skeletal muscle and electric organ

#### Approach: mRNA and miRNA analysis using qRT-PCR

Total RNA was extracted from muscle and EO of adult control fish using TRIzol® (Life Technologies). Concentration $(\mathrm{ng} / \mu \mathrm{L})$ and purity $(260 / 280)$ measurements of total RNA from each muscle and EO sample were taken with a NanoDrop Lite spectrophotometer (Thermo Scientific). Total RNA was treated with DNase, and cDNA synthesis was performed as per manufacturer’s instructions (SuperScript First-Strand Synthesis System kit; Invitrogen). To ensure the absence of genomic DNA in subsequent RNA analyses, we produced a set of RNA samples that were not reverse transcribed (no-RT samples) by substituting the reverse transcriptase enzyme with nuclease-free water. cDNA and no-RT samples were stored at $-20^{\circ} \mathrm{C}$. Gene specific primers for mRNA and quantification (Table 1) were designed with Primer3Plus ([Koressaar & Remm, 2007](https://doi.org/10.1093/bioinformatics/btm254); [Untergasser et al., 2012](https://doi.org/10.1093/bioinformatics/bts134)) and evaluated with the program NetPrimer ([Premier Biosoft, 2015](https://www.premierbiosoft.com/netprimer/)). All primer pairs were first tested with qualitative PCR using Taq polymerase before qRT-PCR was performed. To validate amplicon size and ensure the absence of non-specific amplicons, qualitative PCR products were visualized on agarose gels stained with SYBR® Safe (Invitrogen). Annealing temperatures and primer concentrations were optimized for each primer pair.

qRT-PCR was performed using Perfecta SYBR® Green Fastmix (Quanta Biosciences, Inc., Gaithersburg, MD) and PCR reactions were performed in triplicate (cDNA samples) or single (no-RT samples) in 96-well plates with clear caps on a DNA Engine Opticon® 2 system (Bio-Rad, Hercules, CA). PCR amplification was carried out using a profile that consisted of an initial $30 \mathrm{~s}$ denaturation at $95^{\circ} \mathrm{C}$ followed by 40 cycles of $1 \mathrm{~s}$ denaturation at $95^{\circ} \mathrm{C}, 15 \mathrm{~s}$ primer annealing at optimized temperature depending on the primers (Table 1), and $30 \mathrm{~s}$ extension at $72^{\circ} \mathrm{C}$. For every PCR reaction, the fluorescence signal was measured at the end of the extension period. The PCR reaction was immediately followed by melting curve analysis from $65^{\circ} \mathrm{C}$ to $90^{\circ} \mathrm{C}$ in $0.2^{\circ} \mathrm{C}$ steps with a holding time of $2 \mathrm{~s}$ per step. Data was collected using the Opticon Monitor™ 3.1.32 software (Bio-Rad, Hercules, CA, USA).

To determine primer pair amplification efficiency, raw fluorescence data were analyzed using the real-time PCR Miner tool ([Zhao & Fernald, 2005](https://doi.org/10.1093/nar/gki406)). Raw $C_{t}$ values for each
Table 1 Oligonucleotide primers used for quantitative RT-PCR. Sequences are shown for sense and antisense primers, and the annealing temperature ($T_m$) is given for each primer pair.

| Gene     | Sense primer                          | Antisense primer                        | $T_m$ (°C) |
|----------|---------------------------------------|-----------------------------------------|------------|
| rps11    | TACC CAGA ATG AAAGGG CGT AT           | CAT GTT CTT GTG GCCT CTTC TCG           | 59         |
| snrpb    | AAG ATCA AGCCC AAAATTCCA              | CCT GAGG TGT CA TTACCT GCT G            | 59         |
| hsp70    | CGAT TTCT ACACCT CCAT CACC A          | ATG CTCT TGT TGAG TTCT TTGCC            | 58         |
| hsp90a2  | AAGG ACA ACCAGA AGCAC ATCT            | TCCT CTT CAT CTT CAGGG AGC              | 60         |
| plec     | ATCC TCCA AATCCT CTG TTCGC            | GATA TGT CTGG TGT CAAT GAGT CC          | 60         |
| ank2     | GATC TT CGCAT GGCT ATT GTG            | TCTT CTC ACCCA ACTT CTC TCGG            | 60         |
| dmd      | AGC ACC TACT GA TCC AGCA CTACT        | CTG CCT GTAG TTT CCT GTT CTC C          | 58         |
| cryaba   | CATC ACCT CCT CACT CTCC TCG           | TGACT TGT CCT CTT GAG TAAT GGG          | 58         |
| actn2    | GCTT GCTG CTG TGG T GCC               | GTG TTC ACA ATG TCCT CGGC               | 58         |
| myoz3    | AAGG ATGT GAT GGAG GAGC               | TCTT TCAG AGGA CCAGT GTA CCC            | 62         |
| myom1    | TTTC AAAGAA AGACG CAG GTGT            | ACAAC GGAT GTGG AGAGG ATGA              | 57         |
| ckma     | AACAG TCGT GGT GAGC GTAGG             | GAAC ACTCC GCAA CAGAGG                  | 58         |
| ttnb     | ATGT GCC TGAGG ATGAC GG               | CAAA CACTT CTGGT CTAGT TGG AG           | 60         |
| myh10    | GAGC AGG AGG AGT ACC AGCG             | TGGAA CTT GGT GTGT GAG CC               | 58         |
| mylk4    | CGG ACAT CTCT GACG AGGC               | GGAG CAT GTGT GGAGT GGC                | 62         |
| myh-f    | AAAGG TACA AAAGT CTT GAAT GCC AGT     | AGC CTG AGT CAT AGT CACT AGT TTG        | 60         |
| myh-s    | AGAGA GCTT GAAA ATG AGGT G            | TTC ATCGT GTAC TTT CTT GGAA             | 60         |
| myf5     | AACGC CATCC AGT ACATT GAG AG          | AAT CTG AGAC CAGA CAGG ACTGC            | 60         |
| myod     | ATATT CCGT TCA ACAT CACCT CT          | AGAAG ACAC CTGC CTC TGT G              | 58         |
| myog     | CGC AGTG CCAT ACAGT ACAT CG           | TCTG AGC CTGT GAT GCT GTCT AC           | 62         |
| myf6     | CCGAA ACGG GAT GTG ACAG               | CAGA GTGG CAGC CTT ACGC                 | 60         |
| sox6     | TTCC ACGG CAGC AAG AGC                | CCTC ATC ACCG CAT CCTG                 | 60         |
| eyal     | CCAC CTCT GTAG CAGA CGGGC             | TGT ACT GTGT CTG ACC ATAC GCC           | 60         |
| ube2a    | CCCG TCCG AGAA CAAT ATCA              | CAGGG CTGT TAGG ATTGGC                 | 58         |
| psmb1    | TGGT GACT GTTT GACG CTG AC            | CGC CTG CTT TGTA AGT GTCC               | 58         |
| psmc2    | AAGA GGAGG AGAA AGACG AGG             | GCTAA ATCCC ATA ACGC AGG AG             | 60         |
| psmc5    | GGTG TCCCT GAT GAT GGT GG             | CAGA CACT CGG ATG AAG GTGC              | 58         |
| psmc6    | ATTG ACGGG CGT TTG AAGG               | TCCA ACA CAT ACCG AGGGC                 | 58         |
| becn1    | ACGT GGACA AGGG CAAG ATC              | CCCAT TTG AGG TTTGG TGAGC               | 58         |
| atg12    | CTGTT ACCG ATAC AGG AGA CACT GG       | AGAA AGCGG GAGA TAA ACTGGG              | 60         |
| gabarapa | CGACG ACAGA TTTCC ACGG               | ATTG TTG ACGA AGA AGA GAGT GC            | 60         |

transcript were uploaded into the RefFinder online tool (Xie et al., 2012) in order to identify appropriate reference genes for normalization. This tool uses four different programs for ranking potential internal reference genes—BestKeeper (Pfaffl et al., 2004), NormFinder (Andersen, Jensen & Orntoft, 2004), geNorm (Vandesompele et al., 2002), and the Comparative Delta Ct Method (Silver et al., 2006)—to determine a single ranking of the most appropriate genes to use as internal reference genes. RefFinder assumes a perfect primer pair amplification efficiency for each qRT-PCR reaction when ranking the expression stability of each transcript. However, to more accurately rank the most stably expressed genes, we had to consider the amplification efficiency for each primer pair (De Spiegelaere et al., 2015). We then re-analyzed our qRT-PCR data using BestKeeper,

Table 2 TaqMan assay information used for miRNA quantitative RT-PCR.

| Assay name       | Context sequence                          |
|------------------|------------------------------------------|
| hsa-miR-206      | UGGAAUGUAAGGAAGUGUGUGGG                  |
| hsa-miR-365      | UAAUGCCCCCUAAAAAUCCUUAAU                |
| ssa-miR-193-3p   | AACUGGGCCCUGC AAAAGUCCCGCU              |
| smac-miR-30a     | UGUAACACAUCUUUUGACUGAAAGC               |
| ola-miR-30d      | UGUAACACAUCUCCC CGACUGGAAGC             |
| hsa-miR-1        | UGGAAUGUAAAAGAAGUAUGUAU                 |
| hsa-miR-499      | UUAAGACUUUGCAGUGAUGUUUU                 |
| hsa-miR-205      | UCCUUCAUUUCCACCGGAGUCUG                 |

NormFinder, and geNorm using the calculated primer pair amplification efficiencies for each gene. Results from the latter analyses confirmed the ranking of *rps11* (a 40S ribosomal subunit protein) and *snrpb* (a riboprotein component of the spliceosome) as the two most-stably expressed genes. $C_t$ values for all 29 transcripts associated with sarcomere homeostasis were imported into Microsoft Excel and analyzed using calculated primer pair efficiency adjustment and normalized against *rps11* and *snrpb* to derive fold expression differences between muscle and EO.

Quantification of miRNAs was performed using TaqMan® miRNA qPCR analysis (Applied Biosystems). Total RNA isolated from muscle and EO of adult control fish ($n = 3$) was reverse transcribed using miRNA-specific primers following the instructions outlined in the TaqMan® instruction manual. Primers for all qRT-PCR reactions for all miRNAs except for miR-30a were ordered from conserved pre-designed sets commercially available from TaqMan® (Table 2). The miR-30a reaction was specially designed by TaqMan® using an S. *macrurus*-specific miR-30a sequence (Table 2). All reverse transcription reactions were performed immediately prior to qPCR reactions.

Quantification of miRNA expression was performed using the reaction conditions specified by TaqMan® in 96-well plates with clear caps on a DNA Engine Opticon® 2 system (Bio-Rad, Hercules, CA, USA). All cDNA reactions were performed in triplicate, while no-RT reactions were performed on single samples. Data was collected using the Opticon Monitor™ 3.1.32 software (Bio-Rad, Hercules, CA, USA). Raw fluorescence data were analyzed using the real-time PCR miner tool (*Zhao & Fernald*, 2005) as described above. Based on its high miRNA expression stability according to the raw $C_t$ values using the RefFinder tool (*Xie et al.*, 2012), miR-206 was used for miRNA normalization. $C_t$ values for all miRNAs were imported into Excel, analyzed as using calculated primer pair efficiencies, and normalized against miR-206 to derive fold-change differences between skeletal muscle and EO.

**Approach: mRNA and miRNA analysis using RNA-seq Illumina sequencing**

RNA preparation. Skeletal muscle and EO tissues from one adult fish were blotted dry, weighed, and immediately frozen in liquid nitrogen. Tissues were then chopped and pulverized separately in liquid nitrogen using a mortar and pestle, re-suspended in TRIzol® reagent (Life Technologies) and their total RNA extracted following

Table 3 Summary of transcriptome sequencing of muscle and electric organ of Sternopygus macrurus.

(A) Library statistics

| Illumina paired end library | Read pairs (each pair counted as 1) | Read size |
|----------------------------|-------------------------------------|-----------|
| 1. S. macrurus EO           | 175,605,480                        | 2×101nt   |
| 2. S. macrurus muscle       | 171,612,181                        | 2×101nt   |
| 3. 5-day Spinal Transection EO | 161,775,709                        | 2×101nt   |
| 4. 5-day Spinal Transection muscle | 152,159,414                        | 2×101nt   |
| 5. 2-day Spinal Transection EO | 158,340,025                        | 2×101nt   |
| 6. 2-day Spinal Transection muscle | 157,943,982                        | 2×101nt   |

(B) Assembly statistics

| Assembler                          | Assembled transcripts (#)                     | Mean size          | N 50 size    |
|------------------------------------|----------------------------------------------|--------------------|--------------|
| Trinity assembly of 1, 2 (from Gallant et al., 2014) | 326,623 transcripts                      | 1,287bp           | 3,045bp      |
| SOAPdenovo-trans of all six libraries | 642,455 transcripts (61,532 scaffolds; 580,923 additional contigs) | 2,273bp—scaffolds, 468bp—all | 1,358bp      |

manufacturer's instructions. Residual DNA was removed from total RNA samples by treating with DNase I, Amplification Grade (Invitrogen) prior to purification using phenol:chloroform:isoamyl alcohol extraction followed by isopropanol precipitation. cDNA libraries were constructed from purified RNA using the Illumina TruSeq RNA Sample Preparation (v.2) kit (San Diego, CA, USA). Libraries were sequenced using 100bp paired-end reads (2×100bp) on an Illumina HiSeq 2000 at the Biotechnology Center of the University of Wisconsin in Madison, WI. An overview of the subsequent sequencing and assembly steps performed is outlined in Fig. 1.

Transcriptome assembly. An initial de novo assembly of 694,435,322 reads from EO and muscle libraries (Table 3A) was performed using the Trinity assembler (Grabherr et al., 2011) and first reported in Gallant et al., (2014). Briefly, for that assembly, high quality reads were obtained by removing adaptor sequences, duplication sequences, ambiguous reads and low quality reads. Short read libraries from muscle and EO tissues were combined and quality control and filtering was performed using the FASTX-Toolkit (Cold Spring Harbor Labs, 2010) as well as scythe (Buffalo, 2014) and sickle (Joshi & Fass, 2011). Those reads were assembled using Trinity with default options. This assembly produced 326,623 sequence contigs representing 221,914 subcomponents. Of these sequences, 163,477 were at least 500bp length and 63,408 were at least 2,000bp in length. The average sequence length was 1,287bp (Table 3B).

In this study, we performed an additional assembly of 1,954,873,582 reads from EO and muscle libraries plus four libraries taken from spinally transected fish. The four additional libraries used to assemble this new reference library consisted of skeletal muscle and EO tissues taken from spinally transected fish (R Guth et al., 2014, unpublished data). These four additional libraries were processed at the same time using the same methods as the control libraries we are describing. Illumina libraries were re-assembled using

2 Tissue Samples:
S. macrurus

Skeletal Muscle
Electric Organ

Illumina
Sequencing:
100 NT Base
Paired End

Reference
Assembly/Annotation:

TRINITY - 2 Libraries
SOAPdenovo - 6 Libraries

BLASTN/BLASTP:
(E. electricus/D. rerio)

Mapping/Transcript
Quantification:
BOWTIE;
RSEM

Normalization:
EBseq

Filtering

Sample
Preparation
and Illumina
HiSeq2000
RNA-seq

Transcriptome
assembly &
Assignment of
gene orthologs

Calculation of
expression
levels

Figure 1 Overview of steps performed to sequence, annotate and analyze transcriptomes of muscle
and electric organ of Sternopygus macrurus using Illumina HiSeq2000. Paired-end Illumina sequencing was performed on mRNA isolated from ventral muscle and electric organ of S. macrurus (n = 1). De novo reference transcriptomes were assembled and annotated using TRINITY (2 libraries) and SOAPdenovo (6 libraries), and confirmed by BLASTing against D. rerio. TRINITY and SOAPdenovo assemblies were combined to form one common reference transcriptome. Contigs were mapped to the reference using BOWTIE and quantified with RSEM. Muscle and electric organ transcript counts were normalized by library size using the Ebseq package in R. Finally, the annotated transcripts were filtered to eliminate any reads that contained less than 100 copies in both muscle and electric organ. Additional differential expression analysis using DESeq to validate our filtered results was also performed.

SOAPdenovo-trans (Table 3A), an assembler designed specifically for very large RNA-seq
libraries with fast execution time and high optimization memory performance (Xie et al., 2014). The assembly was performed at kmer size K = 31. This assembly produced 61,532 scaffolds and 580,923 additional unassembled contigs. Of these sequences, 117,622 were at least 500 bp length and 28,883 were at least 2,000 bp in length. The average sequence length was 468 bp for all assembled transcripts and 2,273 bp for scaffolds only (Table 3B).

Assignment of gene orthologs. S. macrurus transcripts assembled from Trinity and
SOAPdenovo-trans were compared separately against all genes from Electrophorus electricus
and *Danio rerio* using BLAST (blastn, e-value: 1e-5). Subsequently, all *S. macrurus* matches with each *E. electricus* gene were ranked from highest to lowest BLAST score. A similar but separate table was generated from comparisons between *S. macrurus* mRNA sequences and *D. rerio* genes. An *S. macrurus* transcript that matched one-to-one a pair of genes from *E. electricus* and *D. rerio* was selected as the *S. macrurus* ortholog for that gene. When more than one *S. macrurus* transcript matched a pair, the mRNA sequence with highest match based on CLUSTAL alignment with *E. electricus* and *D. rerio* was selected as the *S. macrurus* ortholog and used for analysis. In some cases when multiple *S. macrurus* transcripts had similar CLUSTAL-based sequence match to those of *E. electricus* and *D. rerio*, an average expression level was taken for each tissue type and an EO/muscle expression ratio was calculated from those average values. Manual confirmation of all *S. macrurus* transcripts discussed in this study was also performed.

**Calculation of expression levels.** Short reads from the muscle and EO libraries were individually mapped to the reference transcriptome assembly using bowtie (*Langmead et al., 2009*) (v.0.12.8) with default parameters. Read counting and ambiguity resolution was performed using RNA-Seq by Expectation Maximization (RSEM) (*Li & Dewey, 2011*) (v.1.2.3). Read counts were subsequently normalized for library size using the geometric mean method from EBSeq (*Leng et al., 2013*). The final data set comprised of 14,500 unique transcripts was generated after normalization and removal of sequences with fewer than 100 copies in both EO and skeletal muscle to minimize inclusion of genes likely to be expressed in cell types other muscle cells and erythrocytes. A 4-fold difference in the expression of a transcript between muscle and EO was designated as being differentially expressed. Additionally, we quantified transcript numbers in each sample using Kallisto (*Bray et al., 2015*), a recently published efficient RSEM alternative, and computed differential gene expression between samples using DESeq2 (*Love, Huber & Anders, 2014*).

**High-throughput sequencing of miRNAs in muscle and EO of *S. macrurus*.** Total RNA for miRNA Illumina RNA-seq was isolated from the same muscle and EO samples that were used for the transcriptome sequencing described above (*n* = 1). The processing and analysis of the whole miRNA RNA-seq libraries has been described previously (*Gallant et al., 2014; Traeger et al., 2015*).

**Assignment of transcripts into functional pathways.** Of a total of 14,500 transcripts assembled, 12,508 were annotated with EntrezGene IDs for *D. rerio* orthologs. *D. rerio* EntrezGene IDs were obtained from the *D. rerio* genes database (Zv9) using the ID converter tool available at the BioMart Central Portal (*Smedley et al., 2015*). Differentially expressed genes were assigned color codes (EO > SM: red, EO < SM: blue, not differentially expressed: yellow) and *S. macrurus* ortholog EntrezGene IDs and corresponding color codes were submitted to KEGG Search&Color pathway analysis (*Kanehisa & Goto, 2000*) and searched against *D. rerio*.
Statistical analysis of qRT-PCR studies

Normalized EO and muscle Ct values were grouped by tissue type (EO and muscle) and a paired student’s *t*-test was used to determine which genes were differentially expressed. To minimize potential false-positives, *p*-values were adjusted using the Benjamini–Hochberg method in R (*R Core Team*, 2016). Samples with an adjusted *p*-value < 0.05 were considered to be differentially expressed. Transcript and miRNA qPCR data were analyzed separately.

Prediction of transcript-miRNA interactions

3′ untranslated regions (3′ UTRs) from sarcomeric genes were isolated from their assembled transcript contigs and entered individually into the RNAhybrid web tool (*Rehmsmeier et al.*, 2004) along with the *S. macrurus*-specific miRNA sequences for miR-365, miR-193b and miR-30a. Target predictions were performed using default alignment and energetic parameters with the exception of number of results to return, which were increased to 10 in order to ensure that all target sites for the same miRNA in each 3′ UTR would be identified if multiple target sites were present. In order to identify conservation of the predicted miRNA target sites in *S. macrurus* across other teleost species, miRNA target prediction was performed on 3′ UTRs and miRNAs of *D. rerio* using the TargetScan Fish release 6.2 web tool (*Lewis, Burge & Bartel*, 2005; *Ulitsky et al.*, 2012).

Sarcomeric mRNA expression in skeletal muscle and electric organ tissues using *in situ* hybridization

Serial tail cross-sections (16 μm thick) of control tails (*n* = 2) were cut in a cryostat and mounted on pre-chilled SuperFrost charged glass slides (VWR, Chester, PA). Prehybridization, hybridization, and washes were carried out per manufacturer’s instructions (GeneDetect, Bradenton, FL). Briefly, tissue sections were fixed with 4% paraformaldehyde in 0.1 M PB (pH 7.4) at 4 °C for 15 min, washed three times for 5 min with 0.1 M PB, incubated with 100% ethanol for 5 min, and then air dried for 5 min at room temperature (RT). Oligonucleotide riboprobes (3′-DIG-labeled; GeneDetect) for fast myosin heavy chain (*myh-f*) and myosin light chain (*myl*) as well as for sense-strand negative control probes (*Table 4*) were diluted in hybridization buffer (20X SSC, 200 mg/ml dextran sulfate, 0.5X formamide, 10 mg/ml polyA, 10 mg/ml ssDNA, 10 mg/ml tRNA, 1 M DTT, 50X Denhardt’s) at 300 ng/ml and 350 ng/ml, respectively, slides were coverslipped and placed in a humidified hybridization chamber at optimized hybridization temperatures for each probe pair overnight for 18–21 hrs. All washes were performed at 55 °C unless otherwise noted and the hybridized probe was detected using the DIG Nucleic Acid Detection Kit (Roche, Germany). Tissue sections were blocked for 30 min in blocking solution (1:10 dilution; blocking reagent in maleic acid buffer (0.1 M maleic acid, 0.15 M NaCl, pH 7.5) at RT. The anti-digoxigenin-AP antibody was added (1:200 dilution) and slides were placed at 4 °C overnight. The slides were washed three times for 5 min with tris buffered saline (pH 7.5) and then equilibrated for 5 min with detection buffer (0.1 M Tris-HCl, 0.1 M NaCl, pH 9.5) at RT. The color substrate solution (200 μl of NBT/BCIP in 10 ml of detection buffer) was added and slides were placed in the dark overnight at RT. Images were captured with a Zeiss Axio Imager.Z1 epifluorescence microscope connected

Table 4 Oligonucleotide probes used for in situ hybridization. Sequences are shown for antisense (a) and sense (s) negative control probes, and the hybridization temperature (°C). See text for abbreviations of genes.

| Gene     | Probe Sequence 5′–3′                                                                                          | °C |
|----------|----------------------------------------------------------------------------------------------------------------|----|
| myh-fast | a CCTTTCCTGCTTCTCTGGCTCTGGCTTGAGCTTGGACCTGGGACTCAGCGATGTCCGGCAGTCC TGAGCTT 58<br>CCTCCATCTCATGCTGCACCTT CCTGAACCTGGACAGGTGAGTTGGGCCTGTCC TCA GCCATCCTCAGCCTGT<br>CTCTTGTAGGCTTTC ACTTTCAGTTGCAGCTTGTGCCACAAGACCTGCAGTCTTATCACATTCTTCTTGTCTT CCT<br>CAGTCTGGTAGGTAAGCTCCTTC ACTCTCCTTT CATATTTGCGCACTCCTTTAACAGCTTCAGCACTGGCTCTGCTCAGCTTCAAACCTCACC TCAGCTCACGCACCCCTAGACTCCAGTTTCTGGAGCTGCT<br>s GAAAGAAGCAGCTCCAGAAA ACTGGAGTCTAGGGTGC GTGAGCTGGGAGGGTGAGGTTGAAGCTGAGCAGAGAC 58<br>GCAGTGCTGAAGCTGTTAAAGGAGTGCGCAAATATGAAA GGAGAGTGAAAGGAGCTTACCTACCAGACTGAGG<br>AAGACAAGAAGAATGTGATAAGACTGCAGGATCTTGTGGACAAGCTGCAACTGAAAGTGAA AGCCTACAAGAG<br>ACAGGCTGAGGATGCTGAGGAGCAGGCCAACACTCACCTGTCCAGGAGGTGCAGCATGAGATGG<br>AGGAAGCTCAGGAGCGTGCCGACATCGCTGAGTCCCAGGTCAACAAAGCTCAGAGCCAAGAGCAGAGAAGCA<br>GGAAAGGGAAAGCGGCC |
| myl      | a AGCACCTTGAAAGAGGACAGGATGACATCCTCAGGGTCCAGCACCCTTCAGCTTCTCTCCGAACATGGTGAGG 58<br>AAGACGGTGAAAGTTGATGGGGGCCAGGGGCCTCTTGATCATGGGCCTCCAGCTCCTCATTC TACATTCAGCT<br>GGCCCATAGAGGCCAGCACGTCCCTAAGGTCGTCC TTGCTGATGATACCATCTCTGTTCTGGTCAATGATTGT<br>GAAAGCCTCTTTGTCTCCTGAA<br>s CCAGATT CAGGAGACAAAGAGGCTTTCACAATCATTGACCAGAACAGAGATGGTATCATCAGCAAGGACGACC 58<br>T TAGGGACGTGCTGGGCCTCATGGGCCAGCTGAATGTGAAAGAATGAGGAGC TGGGAGGCCATGATCAAGGAGG<br>CCCC TGGGCCCATCAACTTCACC GTCTTCCCTCACCATGTT CGGAGAGAAGCTGAAGGGTGCTGACCCTGAGGAT<br>GTCATCCTGTCC TCTTTCAAGGGTGCTGGATCCA |

A  Sarcomeric Structure

B Transcription Factors  C Protein Degradation

Figure 2 qRT-PCR analysis of 29 transcripts associated with the expression of sarcomeric genes. Plots illustrating fold-change differences between EO and muscle in *S. macrurus* (*n* = 4). (A) Fold-change differences of 15 transcripts associated with sarcomere formation and structure. (B) Fold-change differences of six transcription factors associated with regulating the expression of sarcomeric genes. (C) Fold-change differences of eight genes involved in protein degradation pathways. Data are shown as fold-changes ± standard error, graphed on a logarithmic scale with values >1 indicating increased expression in EO relative to muscle and values <1 indicating decreased expression in EO relative to muscle. Data points marked with asterisks are significantly differentially expressed between EO and muscle (adjusted *p*-value < 0.05).

70-kd protein (*hsp70*) and heat shock protein 90, alpha (cytosolic), class A member 1, tandem duplicate 2*hsp90aa1.2*), [Hellerschmied & Clausen, (2014)] were also detected in muscle and EO at similar levels ([Fig. 2A] and [Table 5A]).

The Illumina high-throughput sequencing of transcriptomes from the skeletal muscle and EO of one adult fish provided us with a complete profile of the sarcomere genes expressed in these myogenic tissues beyond those studied using qRT-PCR. These data showed that all genes known to make up the mammalian sarcomere ([Ehler & Gautel, 2008]; [Treves et al., 2009]) are transcribed in the fully differentiated EO of *S. macrurus*. Specifically, we identified 112 mammalian orthologs that compose the I-A-M band ([Fig. 4A]), Z-disk region ([Fig. 4B]), and costamere ([Fig. 4C]). We also found the EO to contain 96 of 112 sarcomeric gene transcripts at levels that were similar to those found in muscle ([Fig. 4]). At least eight different sarcomeric myosin heavy chain (MHC) mRNAs corresponding to different muscle cell types were detected and since their expression levels did not differ between tissue types, we represented this similarity in MHC isoform mRNAs and quantity

myl (anti)
![Image](image1.png)

myl (sense)
![Image](image2.png)

myh-f (anti)
![Image](image3.png)

myh-f (sense)
![Image](image4.png)

---

**Figure 3** *In situ* hybridization of muscle and electrocytes using probes against myosin light chain and fast-myosin heavy chain. Portions of transverse cryosections (16 μm-thick) taken from tails of control fish and processed for *in situ* hybridization. Antisense (anti) DIG-labeled RNA riboprobes specific to S. macrurus myosin light chain (*myl*) and sarcomeric fast myosin heavy chain (*myh-f*) transcripts were used for detection of these mRNAs in skeletal muscle fibers and electrocytes (EC). Sense strand DIG-labeled riboprobes against myosin light chain (*myl*) and sarcomeric fast myosin heavy chain (*myh-f*) were used as negative controls. Melanin is the dark brown pigment found around muscle fibers and electrocytes (arrow heads). Abbreviations: SM, skeletal muscle fibers; EC, electrocyte. Scale bar, 100 μm.

by providing only one general category for sarcomeric “myosin heavy chain” in **Fig. 4**. Of the 16 genes that were not similarly expressed in muscle and EO, three were detected in the EO at levels lower than 4-fold compared to muscle—the glycolytic enzyme beta-enolase (*eno3*) (EO: 22,939; muscle: 166,518 copies), *actn2* (EO: 6,055; muscle: 30,011 copies) and *atp1a1a.,2* (EO: 311; muscle: 3,235 copies). *eno3* is localized to the M-band and predominant in fast-twitch fibers (**Foucault et al., 1999**) (**Fig. 4A**). The thirteen genes that were more than 4-fold upregulated in EO are associated with the actin cytoskeleton that promotes cell attachment and these include: *desmin* (*desmb*), *ank2*, *cryaba*, N-cadherin (*cdh4*, *cdh8*, *cdh12*), and alpha-actinin-2-associated LIM protein (ALP; *pdlim3a*) (**Clark et al., 2002**) (**Fig. 4C**). DEseq analysis supported the 4-fold cut-off we established, as all three down-regulated genes were called by DESeq too, and only three up-regulated genes *ank2* (*p = 0.12*), *cdh4* (*p = 0.07*), and *cryaba* (*p = 0.07*) were not called as differentially expressed (*p ≤ 0.05*) by DESeq.

Table 5 Comparison of fold change differences of 29 transcripts between EO and muscle measured by qRT-PCR and RNA-seq. qRT-PCR measurements were normalized against the geometric means of two internal reference genes: rps11, and snrpb. For all qRT-PCR ratios, \( n = 4 \). Statistical analysis of the qRT-PCR dataset identified five significantly differentially expressed transcripts (\( p \leq 0.05 \), designated ‘*’). All RNA-seq ratios (\( n = 1 \)) were considered to be differentially expressed if the difference in transcript expression was more than 4-fold different between EO and muscle.†—To date, we have not resolved the identity of all fast-twitch and slow-twitch myosin heavy chain isoforms transcribed in muscle and EO of S. macrurus and therefore we did not include RNAseq data for myh-f and myh-s in this table.

(A) Sarcomeric structure

|         | hsp70   | hsp90a2 | plec    | ank2    | dmd     | cryaba  | actn2   | myoz3   | myoml   | ckma    | ttnB    | myh10   | mylk4   | myh-f   | myh-s   |
|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|
| Fold-change | 2.37    | 0.41    | 2.28    | 5.89*   | 1.26    | 1.03    | 0.77    | 0.72    | 0.35    | 0.38    | 0.66    | 3.88    | 0.90    | 0.22    | 0.02    |
| qRT-PCR Standard error | 0.16    | 0.05    | 0.10    | 0.34    | 0.02    | 0.12    | 0.14    | 0.01    | 0.02    | 0.05    | 0.57    | 0.07    | 0.03    | 0.01    |
| P-value | 0.08    | 0.18    | 0.10    | 0.04    | 0.38    | 0.99    | 0.65    | 0.38    | 0.08    | 0.14    | 0.66    | 0.08    | 0.99    | 0.26    | 0.09    |
| RNA-seq Fold-change | 3.54    | 0.70    | 3.61    | 4.75    | 2.24    | 6.46    | 0.20    | 1.50    | 0.62    | 0.70    | 12.19   | 0.33    | †       | †       |

(B) MTFs

|         | myf5    | myoD    | myog    | myf6    | sox6    | eyal    |
|---------|---------|---------|---------|---------|---------|---------|
| Fold-change | 0.51    | 0.97    | 0.65    | 1.07    | 3.30    | 2.30*   |
| qRT-PCR Standard error | 0.04    | 0.10    | 0.07    | 0.05    | 0.24    | 0.12    |
| P-value | 0.18    | 0.99    | 0.08    | 0.96    | 0.11    | 0.05    |
| RNA-seq Fold-change | 0.67    | 1.68    | 1.21    | 2.39    | 5.54    | 6.81    |

(C) Protein degradation

|         | ube2a   | psmb1   | psmc2   | psmc5   | psmc6   | becn1   | atg12   | gabarapa |
|---------|---------|---------|---------|---------|---------|---------|---------|----------|
| Fold-change | 1.82    | 1.00    | 0.98    | 0.92    | 1.29    | 1.86    | 1.00    | 1.13     |
| qRT-PCR Standard error | 0.05    | 0.03    | 0.07    | 0.05    | 0.09    | 0.07    | 0.04     | 0.03     |
| P-value | 0.12    | 0.99    | 0.99    | 0.72    | 0.38    | 0.10    | 0.99    | 0.64     |
| RNA-seq Fold-change | 2.35    | 1.40    | 1.66    | 1.69    | 1.84    | 2.89    | 2.34     | 2.22     |

A

B

C

Figure 4 Model of the contractile apparatus and the relative expression levels of transcripts representing sarcomeric components in muscle and EO. (A) I-band, A-band, and M-line regions of the sarcomere; (B) sarcomere Z-disk region; and (C) cytoskeletal filament linkages at the sarcolemma of striated muscle. All sarcomeric genes correspond to those listed for mammalian sarcomeres in *Ehler & Gautel (2008)*. Transcripts labeled in red, yellow and blue represent transcripts expressed at levels >4-fold higher in EO than muscle, similar in both EO and muscle, and <4-fold lower in EO than muscle, respectively. To date, we have not resolved the identity of all myosin heavy chain isoforms transcribed in muscle and EO of *S. macrurus* and thus, all identified isoforms have been included in one category, labeled “Myosin Heavy Chain” in this figure.

---

**Myogenic transcription factors are not downregulated in the mature electric organ of S. macrurus**

Previous expression studies in *S. macrurus* showed that the adult EO continues to transcribe the myogenic transcription factors (MTFs) *myf5*, *myod*, *myogenin (myog)* and *myf6 (mrf4)* (*Kim et al.*, 2004; *Kim et al.*, 2008) and moreover, these *S. macrurus*-specific MTFs are capable of inducing the myogenic phenotype in mammalian non-muscle cells *in vitro* (*Kim et al.*, 2009). In this study, we expanded our expression analysis of MYFs with important roles in myogenesis. Specifically, the expression levels of *srf*, *prdm1*, *six1*, *sox6*, *eyal1*, *fox03*, *hdac*, and members of the MEF2 (*mef2ca*, *mef2cb*, *mef2d*) and NFAT (*nfatc1*, *nfatc3*, *nfatc4*) families of transcription factors were studied using qRT-PCR and/or RNA-seq (Figs. 2B, 6 and Table 5B). In general, our data showed that all transcription factors are transcribed in adult EO tissue at levels similar (<4-fold difference) to those found in skeletal muscle. Only

sox6 and *eya1* were upregulated (<4-fold difference) in EO relative to skeletal muscle in the RNA-seq study, and only *eya1* was significantly upregulated (*p* < 0.05) in our qRT-PCR study (Figs. 2B, 6 and Table 5B). In addition, our RNA-seq and qRT-PCR data corroborated our previous findings showing that the four members of the MyoD family of MTFs are not down-regulated in mature EO tissue despite its lack of sarcomeric structures. DESeq corroborated results that we observed in our qRT-PCR analysis, and the 4-fold cut-off that we imposed on our transcriptome data by calling all six transcription factors analyzed by qRT-PCR as similarly expressed. This corroborates our qRT-PCR data in five of six cases with *eya1* as the only exception, being called significantly upregulated by qRT-PCR (*p* = 0.05), and not significantly different by DESeq (*p* = 0.07).

### Components of protein degradation pathways are not differentially expressed in muscle and EO of *S. macrurus*

Since our qRT-PCR and RNA-seq data showed that all sarcomere components and key myogenic transcription factors that regulate many of these contractile genes are transcribed in adult EO at similar, or even higher levels than those found in skeletal muscle, we examined some of the key genes associated with protein degradation pathways including the ubiquitin-proteasome system (UPS) and the autophagy pathway (Fig. 2C and Table 5C). Based on qRT-PCR, expression levels of all UPS genes (*ube2a*, *psmb1*, *psmc2*, *psmc5*, *psmc6*) were not significantly different between muscle and EO tissues (Fig. 2C and Table 5C). Similarly, genes in the autophagy pathway such as *becn1*, *atg12*, and *gabarapa* were not significantly differentially expressed between muscle and EO (Fig. 2C and Table 5C). The expression levels detected by qRT-PCR were corroborated by the RNA-seq survey (Table 5C). In addition to all genes in both the UPS and autophagy pathways (Figs. S1–S3), the RNA-seq survey also identified other known genes reported to be associated with protein turn-over including *capn3*, *trim63* (MuRF1) and *fbxo32* (Atrogin-1) (*Duguez, Bartoli & Richard, 2006; Witt et al., 2008; Bonaldo & Sandri, 2013*)—all transcripts with expression levels similar between EO and skeletal muscle (Fig. 6). Hence, based on the mRNA expression of components belonging to protein degradation pathways, these data imply that post-translational regulation of sarcomeric proteins might not play a major role in maintaining a partial muscle phenotype in the adult EO of *S. macrurus*. DESeq analysis supported the similar expression of proteolytic genes that we observed, with all eight genes detected in qRT-PCR also being called not significant (*p* > 0.05) in DESeq.

### A set of microRNAs are predicted to regulate sarcomeric components in the EO of *S. macrurus*

The absence of any indication that mature electrocytes downregulate the expression of genes that code for sarcomere structures at the mRNA level while retaining the expression of only a subset of its contractile proteins is suggestive of the involvement of post-transcriptional events in regulating the skeletal muscle program in the EO. This led us to perform an Illumina sequencing of mature miRNAs isolated from muscle and EO of *S. macrurus*. Our miRNA sequencing and expression analysis revealed 155 conserved miRNAs with known functions, all of which were expressed in both EO and muscle. Four miRNAs associated with the skeletal muscle phenotype were detected in EO: miR-133, miR-1, miR-206, and

Table 6 Comparison of select miRNA fold-changes between EO and muscle across both TaqMan® qRT-PCR and Illumina RNA-seq platforms. TaqMan® qRT-PCR measurements were normalized against miR-206. For all qRT-PCR ratios, $n = 3$. Statistical analysis of the qRT-PCR dataset identified one significantly differentially expressed miRNA ($p \leq 0.05$; designated ‘*’). All RNA-seq ratios ($n = 1$) were considered to be differentially expressed if the difference in miRNA expression was more than 4-fold different between EO and muscle.

| miR-365 | miR-193b | miR-30a | miR-30d | miR-1 | miR-499 | miR-205 |
|---------|----------|---------|---------|-------|---------|---------|
| Fold-change | 9.70* | 7.40 | 4.31 | 2.35 | 0.99 | 0.58 | 0.26 |
| Standard error | 0.31 | 0.53 | 0.37 | 0.15 | 0.09 | 0.02 | 0.006 |
| $P$-value | 0.04 | 0.06 | 0.06 | 0.06 | 0.92 | 0.07 | 0.16 |
| RNA-seq | Fold-change | 14.79 | 8.88 | 21.82 | 1.32 | 1.52 | 0.49 | 0.09 |

miR-499. Their expression levels were similar in both muscle and EO tissues (Table 6). Three miRNAs that were upregulated in EO relative to muscle included miR-365 (~15 fold), miR-193b (~9 fold), and miR-30a (~22 fold) (Table 6). miR-365 and miR-193b have been implicated in the loss of sarcomeric gene expression during their induction of adipogenesis in C2C12 cells (Sun et al., 2011), while miR-30a is involved in early muscle fate-determination (Soleimani et al., 2012; Ketley et al., 2013; Guess et al., 2015).

We confirmed the RNA-seq miRNA profiles using TaqMan® qRT-PCR for a total of eight miRNAs including the three highly upregulated miRNAs (miR-365, miR-193b, miR-30a), three muscle-specific miRNAs or myomirs (miR-1, miR-206, miR-499), one that was similarly expressed in both muscle and EO tissues (miR-30d), and one that was highly downregulated in EO (miR-205). In general, the relative trends and differences in miRNA expression detected by RNA-seq were confirmed by our qRT-PCR data (Fig. 5 and Table 6). In EO, miR-365 (9.7-fold; $p = 0.04$) and miR-193b (7.4-fold; $p = 0.06$) were highly upregulated while the three myomirs and miR-30d were detected in similar amounts in both EO and skeletal muscle and miR-205 levels were lower in EO than in muscle (Fig. 5 and Table 6). Discrepancies obtained between RNA-seq and qRT-PCR methodologies were related to the magnitude, but not direction, of the expression levels detected in muscle and EO. For example, miR-30a was upregulated about 22-fold in EO according to RNA-seq, but only upregulated about 4.3-fold based on qRT-PCR (Fig. 5 and Table 6). It was not unexpected to observe some variation in the data using these two detection platforms given the biological differences between fish sampled and a sample size of one used for the RNA-seq miRNA survey.

Following confirmation of the expression levels of miRNAs upregulated in EO that play roles in regulating the muscle phenotype, we determined whether they also contained sequences complementary to the 3′ UTRs of sarcomeric transcripts, which might suggest their potential to regulate their translation. Based on the analysis using S. macrurus-specific 3′ UTR sequences, miRNA sequences on the RNAhybrid miRNA target prediction program, D. rerio 3′ UTR sequences, and miRNA sequences on the TargetScan Fish release 6.2 program, we found that the highly upregulated miR-30, miR-365, and miR-193b in mature EO had conserved target sites in several genes associated with the sarcomere: calmodulin 2b (miR-30 and miR-365), I-A-M band-associated tropomodulin 1 (miR-193b), and M-line associated myomesin (miR-30) (Fig. 6).

qRT-PCR analysis of select miRNAs

![Image Description](image.png)

Figure 5 TaqMan® qRT-PCR analysis of seven miRNAs in EO and muscle of *S. macrurus*. Plots illustrating fold-change differences between EO and muscle in *S. macrurus* (*n* = 3). Data are shown as fold-changes ± standard error, graphed on a logarithmic scale with values >1 indicating increased expression in EO relative to muscle and values <1 indicating decreased expression in EO relative to muscle. Data points marked with asterisks are significantly differentially expressed between EO and muscle (adjusted *p*-value < 0.05).

## DISCUSSION

The analyses presented here describe the molecular expression of a subset of contractile genes in the mature EO and skeletal muscle tissues of *S. macrurus*. These data expand previous findings by increasing our transcript characterization of genes associated with sarcomere homeostasis using qRT-PCR. Our deep RNA sequencing of transcriptomes and miRNA compositions of muscle and EO tissues from an adult fish informed our qRT-PCR analysis on expression of sarcomere-associated genes, allowed the identification of additional genes that might be differentially expressed in muscle and electrocytes, and facilitated our investigations aimed at elucidating gene networks involved in modifying the myogenic program in electrocytes.

Our current transcript analysis revealed that electrocytes and muscle fibers express all components that make up a sarcomere in mammalian striated muscle fibers (*Ehler & Gautel*, 2008; *Treves et al.*, 2009). Most unexpected was the finding that electrocytes retained the expression of all these genes at levels similar to, or even higher than, those detected in myofibers. Examination of transcription factors known to regulate sarcomere gene expression also showed no differences, in either quality or quantity, between muscle and EO (Fig. 6). Given the role of degradation pathways in the homeostasis of molecular composition, we then predicted genes associated with these pathways to be significantly upregulated in EO, but were unable to find changes in the expression levels of key subsets of genes associated with the ubiquitin-proteasome system (UPS) and the autophagy pathway (Fig. 6; Figs. S1–S3). These data do not support protein degradation as a key mechanism by

PeerJ

---

**Id1-4**

**MyoD**  
**Myog**  
**Myf6**

**E12/E47**

**Myf5**

**Srf**

**Prdm1**

**Sox6**

**FoxO3**

**Six1**

**Eya1**

**Hdac (class II)**

**Mef2C,D**

**Nfatc1-4**

**Transcriptional Regulation**

**Sarcomeric genes**

**Tropomyosin (tpm1)**  

**calm1b**  
**calmodulin**  
**calm2b**  

**miR-30; miR-365**  

**AMP-deaminase (ampd1)**  

**MURF-1 (TRIM63)**  

**Nebulin (neb)**  

**Troponin**  
**tnnc2**  
**tnni2b,2**  

**CARP**  
**(ankrd)**  

**Titin**  
**ttnta**  
**ttnb**  

**Actin**  
**(ACTC1)**  

**Obscurin**  
**(OBSCN)**  

**p94/Calpain-3**  
**(Capn3)**  

**β-enolase**  
**(eno3)**  

**Myomesin (M-Protein)**  
**(MYOM)**  

**miR-30**  

**MM-creatine kinase**  
**(ckma)**  

**miR-193**  

**Z-disk**  

**Tropomodulin**  
**tmox1**  
**tmox2**  
**tmox4**  

**MyBP-C (mybpc2b)**  

**Myosin heavy chain**

**Sarcomeric genes**

**Protein Degradation**  

**trim63**

**fbxo32**

**Autophagy**

**Ubiquitin/Proteasome**

---

**Figure 6** Mechanisms of control of sarcomeric gene expression in skeletal muscle and electric organ of *S. macrurus*. Our qRT-PCR and transcriptome analysis of the expression of sarcomeric genes in muscle and EO of *S. macrurus* demonstrates that all transcription factors associated with maintenance of sarcomeric gene expression are similarly expressed, or even upregulated in the EO of *S. macrurus* relative to muscle (Top). Concomitantly, virtually all sarcomeric genes are transcribed at similar levels in both EO and muscle (middle), suggesting that transcriptional regulation is not the main driver of the loss of sarcomeres in EO. Additionally, genes associated with protein degradation are similarly expressed in EO and muscle (bottom), suggesting that protein degradation should not differ between both tissues. Finally, we identified a set of upregulated miRNAs and predicted a set of targets within the sarcomere (middle) that provide evidence of an important role of post-transcriptional regulation in controlling the partial muscle phenotype observed in the EO of *S. macrurus*. Names highlighted in red represent transcripts that were >4-fold upregulated in EO relative to muscle; Names highlighted in yellow represent transcripts that are similarly expressed in EO and muscle; Names highlighted in blue represent transcripts that were >4-fold upregulated in muscle relative to EO. miRNA target predictions with dashed lines represent targets predicted in *D. rerio* only using TargetScan Fish v. 6.2 (*Lewis, Burge & Bartel, 2005; Ulitsky et al., 2012*); miRNA target predictions with solid lines represent conserved targets in both *D. rerio* and *S. macrurus* as predicted by TargetScan Fish v. 6.2 (*Lewis, Burge & Bartel, 2005; Ulitsky et al., 2012*) and RNAhybrid (*Rehmsmeier et al., 2004; Krüger & Rehmsmeier, 2006*).

which mature electrocytes selectively repress various components of the contractile muscle program. Of interest, analysis of the EO transcriptome revealed that only 2% of the 14,500 transcripts comprising our reference transcriptome generated from muscle and EO tissues were downregulated, and 8% were upregulated in mature electrocytes in comparison to muscle fibers. Upon examination of this set of differentially expressed genes, we were

---

Pinch et al. (2016), PeerJ, DOI 10.7717/peerj.1828

unable to identify genes with a known link to the synthesis, maintenance or degradation of the sarcomere complex. These data (Fig. 6; Figs. S1–S3) combined with previous studies demonstrating partial expression of sarcomeric proteins in *S. macrurus* EO (*Unguez & Zakon*, 1998; *Cuellar, Kim & Unguez*, 2006), can most parsimoniously be interpreted to mean that the transcription of a striated muscle program in the absence of a contractile phenotype requires electrocytes to depend on post-transcriptional events to maintain an incomplete muscle phenotype.

Little is known about the direct contribution of post-transcriptional events in the maintenance and plasticity of muscle genes after cell differentiation is complete. Few studies have shown that the 3′-untranslated regions (3′ UTR) of some mRNAs that encode sarcomeric MHCs and synaptic proteins in muscle cells are targeted for translational regulation (*Kiri & Goldspink*, 2002; *Vracar-Grabar & Russell*, 2004; *Deschênes-Furry et al.*, 2005). Noncoding RNAs that interact with 3′ UTRs, specifically miRNAs, are emerging in recent years as a new layer of regulators of gene expression. In fact, some miRNAs have been characterized to be skeletal-muscle specific based on their effects on proliferation, differentiation, and determination of different muscle fiber types, as well as on muscle mass homeostasis in aging muscle (*Chen et al.*, 2006; *Rao et al.*, 2006; *Flynt et al.*, 2007; *Zacharewicz et al.*, 2014; *Guess et al.*, 2015; *Shi et al.*, 2015). The wide phylogenetic distribution of miRNAs across the animal taxa and their conserved functional translational repression of target mRNAs through binding to 3′ UTR sequences (*Ambros*, 2004; *Bartel & Chen*, 2004) suggests to us that miRNA-mediated post-transcriptional regulation may also be pivotal in the development and maintenance of the EO tissue in *S. macrurus* (Figs. 6 and 7A). Concurrent with this idea is the detection of similar amounts of fast tropomyosin (*tpma*) mRNA levels in both muscle and EO tissues (Fig. 4A) accompanied with a significantly lower detection of *tpma* protein in mature electrocytes compared to muscle fibers using immunolabeling and Western blots (R Güth & GA Unguez, 2016, unpublished data). Interestingly, the *S. macrurus* 3′ UTR of *tpma* showed little similarity to that of *D. rerio* with only one predicted miRNA target site in common to both teleosts according to TargetScan Fish version 6.2 (*Lewis, Burge & Bartel*, 2005; *Ulitsky et al.*, 2012) (Fig. 7B). Uncovering the functional roles of miRNAs that are upregulated (and downregulated) in the EO will help our understanding of how miRNAs work as part of a molecular toolkit involved in the modification of the muscle program to give rise to a noncontractile electrogenic tissue as the EO.

It is possible that *S. macrurus* may have evolved a unique group of miRNAs that control the expression of sarcomeric genes in EO. To this end, the existence of EO specific miRNAs was recently confirmed in a closely related gymnotiform, *Electrophorus electricus* (*Traeger et al.*, 2015). Future studies will investigate whether or not *S. macrurus* contains tissue-specific miRNAs that target species-specific regulatory elements leading to the differential protein expression patterns between muscle and EO in *S. macrurus*.

Elucidation of post-transcriptional mechanisms in the formation and maintenance of electrocytes will not only have an impact on our understanding of events underlying the plasticity of skeletal muscle cells into non-contractile cells in *S. macrurus* but may also contribute to furthering our understanding of the regulation of the muscle program in

A

Skeletal Muscle

Electrocyte

Muscle Phenotype

Transcriptional Regulation

Post-Transcriptional Regulation

MRFs
nfatc
mef2

Exosome
Protein Degradation Pathways
IncRNAs/microRNAs

B

D. rerio tpma 3'UTR

0 …… 240 …… 300 …… 540 …… 620 …… 1,776

S. macrurus tpma 3'UTR

0 …… 240 …… 300 …… 600 …… 680 …… 1,118

Figure 7 Overview of expression of transcripts associated with regulation of muscle gene expression in muscle and EO of S. macrurus. (A) Skeletal muscle cells are elongated, multi-nucleated, and contain structured sarcomeres. Electrocytes are large, cigar-shaped, multi-nucleated and do not contain sarcomeres, but express almost all transcripts that code for sarcomeric proteins at similar levels as muscle. In addition, the expression of muscle-specific transcription factors and protein-degradation genes is similar in both tissues, indicating that the down-regulation of the muscle phenotype in EO is not predominantly controlled at the transcriptional level or by protein degradation. These data suggest that post-transcriptional regulation of gene expression by non-coding RNAs (long non-coding RNAs, microRNAs) may play an important role in the repression of muscle gene expression in the EO of S. macrurus. (B) Sequence alignment of the 3' UTRs of a fast tropomyosin transcript (tpma; see Fig. 3 for transcript expression ratios) from S. macrurus and D. rerio reveal very little conservation (conserved areas indicated by bold lines with narrow vertical lines between them), only one predicted miRNA target site for miRs-129/722 (indicated as a blue box) was identified in the conserved regions, indicating the possibility of many species-specific regulatory elements that may control the differential protein expression pattern between muscle and EO in S. macrurus. Future studies in S. macrurus will explore the role played by miRNAs in regulating the partial muscle phenotype of mature electrocytes. D. rerio tpma 3' UTR sequence and predicted miRNA target were accessed from TargetScan Fish v. 6.2 (Lewis, Burge & Bartel, 2005; Ulitsky et al., 2012).

other cell types in vertebrates. For example, muscle transcription factors are found in almost all types of neoplastic rhabdomyosarcoma cells, which maintain a very limited myogenic phenotype (Tonin et al., 1991; Wijnaendts et al., 1994; Frascella & Rosolen, 1998). Similarly, muscle transcription factors expression is associated with an incomplete muscle phenotype in fully differentiated cells like Purkinje fibers of the cardiac conductive system

Pinch et al. (2016), PeerJ, DOI 10.7717/peerj.1828
(Takebayashi-Suzuki et al., 2001), myoid cells of the thymus (Grounds, Garrett & Beilharz, 1992; Kornstein, Asher & Fuchs, 1995), and myofibroblasts from liver, kidney, and lung tissues (Walker, Guerrero & Leinwand, 2001).

Although our data indicate that a high level of transcriptional repression of the myogenic program is not a requisite for the emergence and maintenance of the EO phenotype in *S. macrurus*, this molecular strategy appears to diverge from that used by other electric fish species in forming electrocytes from striated muscle cells. A recent study by Gallant et al. (2014) found that unlike *S. macrurus*, two other South American gymnotiforms (*E. electricus* and *Eigenmannia virescens*) and one African mormyrid (*Brienomyrus brachyistius*) strongly repressed the myogenic program at the transcriptional level. A deeper analysis of the expression of transcriptionally upregulated genes in muscle compared to all three EOs of *E. electricus* using cluster analysis (Gallant et al., 2014) identified isomers of many sarcomeric genes. When compared to the expression of their homologs in *S. macrurus*, virtually all were similarly expressed between muscle and EO in *S. macrurus*. This adds more evidence to support our observation that *S. macrurus* utilizes a different mechanism to regulate contractile gene expression compared to other related electric fish species. A similar study by Lamanna and colleagues (2015) also reported a notable downregulation of structural genes associated with the sarcomere complex and MTFs known to activate them in the mature EOs of two additional mormyrids (*Campylomormyrus compressirostris* and *Campylomormyrus tshokwe*). These observations are intriguing in view of the evolutionary relationships between these electric fish species. Despite sharing a common gymnotiform ancestry, the contractile muscle gene expression profiles in EOs of *E. electricus* and *S. macrurus* differ considerably. In contrast, the EO of the gymnotiform *E. electricus* and the distantly related mormyrids *B. brachyistius*, *C. compressirostris* and *C. tshokwe* exhibit a more similar downregulation of genes associated with the contractile muscle phenotype. Given their similar transcription-dependent downregulation of MTFs and their target muscle genes, a model of convergent evolution of EOs may be at work in electric fish species with the exception of *S. macrurus* (Gallant et al., 2014). Moreover, that mature electrocytes in *S. macrurus* represent myogenically derived cells that became highly specialized non-contractile electrogenic cells without changing their striated muscle transcriptome provides new scenarios by which to elucidate how transcriptional and post-transcriptional events may regulate specific contractile properties in a vertebrate.

## CONCLUSIONS

The present data showed that while mature electrocytes differentiate into highly specialized electrogenic cells with a morphology and function that significantly contrast those of their muscle precursors (Unguez & Zakon, 1998), they do so with relatively little modification of their myogenic transcriptome. Ultrastructural and biochemical studies have failed to detect many of the sarcomeric proteins in electrocytes (Unguez & Zakon, 1998; Güth, Pinch & Unguez, 2013). However, our expression analysis showed that electrocytes continue to transcribe genes responsible for the force generation capacity of striated muscle including those that encode proteins of all sarcomeric regions, the excitation-contraction coupling
system, and the transcriptional regulators and degradation pathways known to regulate
the manifestation of many of these muscle-specific genes. These data underscore the
complexity of the multi-level molecular processes that must exist to generate and maintain
the fully functional skeletal muscle phenotype. What is more, these data provide compelling
evidence corroborating an erythrocyte differentiation process in which downregulation of
the muscle program takes place downstream of the transcription process. Specifically, our
findings support a model of gene expression regulation in which select skeletal muscle
modules or gene subsets can be adapted to give rise to a current-producing cell that no
longer contracts. Whether the mismatch of transcript and protein expression of muscle
genes in EO is dependent on processes that regulate mRNA transport out of the nucleus,
mRNA processing, mRNA sorting, mRNA repression, or mRNA decay ([Von Roretz et al., 2011](https://doi.org/10.1038/nature09746); [Neguembor, Jothi & Gabellini, 2014](https://doi.org/10.1038/ng.2951)) is an exciting next step to investigate. This
comprehensive analysis of parallel mRNA and miRNA profiles can inform studies on the
evolution of other muscle-derived non-contractile cells such as heater organs ([Block, 1986](https://doi.org/10.1016/0022-0981(86)90118-8))
and bioluminescent tissues ([Johnston & Herring, 1985](https://doi.org/10.1016/0022-0981(85)90106-8)) in other teleosts, and the various
muscle-like cells in vertebrates including myofibroblasts and Purkinje fibers of the cardiac
conduction system.

## ACKNOWLEDGEMENTS

The authors are very indebted to Michael McDowell for the *in situ* data, Heather Sneed for
assistance with analysis of gene expression data, and Chiann-Ling C Yeh for artistic work
of figures for sarcomere.

### ADDITIONAL INFORMATION AND DECLARATIONS

#### Funding

The following grants funded this work: NIH SCORE Grant 1SC1GM092297-01A1 (GAU),
NSF Grant CNS-1248109 (GAU), and Howard Hughes Medical Institute Science Education
Grant 5200693. The funders had no role in study design, data collection and analysis,
decision to publish, or preparation of the manuscript.

#### Grant Disclosures

The following grant information was disclosed by the authors:
- NIH SCORE: 1SC1GM092297-01A1.
- NSF: CNS-1248109.
- Howard Hughes Medical Institute Science Education: 5200693.

#### Competing Interests

The authors declare there are no competing interests.

#### Author Contributions

- Matthew Pinch performed the experiments, analyzed the data, wrote the paper, prepared
  figures and/or tables, reviewed drafts of the paper.
- Robert Guth and Alexander Chaidez performed the experiments, analyzed the data, prepared figures and/or tables, reviewed drafts of the paper.
- Manoj P. Samanta conceived and designed the experiments, performed the experiments, analyzed the data, contributed reagents/materials/analysis tools, wrote the paper, prepared figures and/or tables, reviewed drafts of the paper.
- Graciela A. Unguez conceived and designed the experiments, analyzed the data, contributed reagents/materials/analysis tools, wrote the paper, reviewed drafts of the paper.

# Animal Ethics

The following information was supplied relating to ethical approvals (i.e., approving body and any reference numbers):

New Mexico State University Institutional Animal Care and Use Committee, 2014-044.

# DNA Deposition

The following information was supplied regarding the deposition of DNA sequences:

All Illumina Hi-Seq data described in this study have been deposited in the NCBI BioProject Database (accession: PRJNA248545).

# Supplemental Information

Supplemental information for this article can be found online at [http://dx.doi.org/10.7717/peerj.1828#supplemental-information](http://dx.doi.org/10.7717/peerj.1828#supplemental-information).

# REFERENCES

Ambros V. 2004. The functions of animal microRNAs. Nature 431(7006): 350–355. DOI [10.1038/nature02871](https://doi.org/10.1038/nature02871).

Andersen CL, Jensen JL, Ørntoft TF. 2004. Normalization of real-time quantitative reverse transcription-PCR data: a model-based variance estimation approach to identify genes suited for normalization, applied to bladder and colon cancer datasets. Cancer Research 64(15): 5245–5250. DOI [10.1158/0008-5472.CAN-04-0496](https://doi.org/10.1158/0008-5472.CAN-04-0496).

Bartel P, Chen CZ. 2004. Micromanagers of gene expression: the potentially widespread influence of metazoan microRNAs. Nature Reviews. Genetics 5(5): 396–400.

Bass AH. 1986. Electric organs revisited. In: Bullock TH, ed. Electroreception. New York: Wiley, 19–100.

Bennett MVL. 1971. Electric organs. In: Hoar WS, Randall DJ, eds. Fish physiology. New York: Academic Press, 347–491.

Blais A. 2005. An initial blueprint for myogenic differentiation. Genes & Development 19(5): 553–569. DOI [10.1101/gad.1281105](https://doi.org/10.1101/gad.1281105).

Block BA. 1986. Structure of the brain and eye heater tissue in marlins, sailfish, and spearfishes. Journal of Morphology 190(2): 169–189. DOI [10.1002/jmor.1051900203](https://doi.org/10.1002/jmor.1051900203).

Bonaldo P, Sandri M. 2013. Cellular and molecular mechanisms of muscle atrophy. Disease Models & Mechanisms 6(1): 25–39. DOI [10.1242/dmm.010389](https://doi.org/10.1242/dmm.010389).
Bray N, Pimentel H, Melsted P, Pachter L. 2015. Near-optimal RNA-seq quantification. [q-bio.QM] ArXiv preprint. arXiv:1505.02710v2.

Buckingham M, Rigby PWJ. 2014. Gene regulatory networks and transcriptional mechanisms that control myogenesis. Developmental Cell 28(3): 225–238 DOI 10.1016/j.devcel.2013.12.020.

Buffalo V. 2014. Scythe. Available at https://github.com/vsbuffalo/scythe (accessed 2 July).

Chen JF, Mandel EM, Thomson JM, Wu Q, Callis TE, Hammond SM, Conlon FL, Wang DZ. 2006. The role of microRNA-1 and microRNA-133 in skeletal muscle proliferation and differentiation. Nature Genetics 38(2): 228–233 DOI 10.1038/ngl725.

Clark KA, McElhinny AS, Beckerle MC, Gregorio CC. 2002. Striated muscle cytoarchitecture: an intricate web of form and function. Annual Review of Cell and Developmental Biology 18(January): 637–706 DOI 10.1146/annurev.cellbio.18.012502.105840.

Cold Spring Harbor Labs. 2010. FASTX-Toolkit. Available at http://hannonlab.cshl.edu/fastx_toolkit/index.html (accessed 2 July).

Cuellar H, Kim JA, Unguez GA. 2006. Evidence of post-transcriptional regulation in the maintenance of a partial muscle phenotype by electrogenic cells of S. Macrurus. FASEB Journal 20(14): E1856–E1865 DOI 10.1096/fj.06-6474fje.

Deschênes-Furry J, Bélanger G, Mwanjewe J, Lunde JA, Parks RJ, Perrone-Bizzozero N, Jasmin BJ. 2005. The RNA-binding protein HuR binds to acetylcholinesterase transcripts and regulates their expression in differentiating skeletal muscle cells. The Journal of Biological Chemistry 280(27): 25361–25368 DOI 10.1074/jbc.M410929200.

De Spiegelaere W, Dern-Wieloch J, Weigel R, Schumacher V, Schorle H, Nettersheim D, Bergmann M, Brehm R, Kliesch S, Vandekerckhove L, Fink C. 2015. Reference gene validation for RT-qPCR, a note on different available software packages. PLoS ONE 10(3): e0122515 DOI 10.1371/journal.pone.0122515.

Duguez S, Bartoli M, Richard I. 2006. Calpain 3: a key regulator of the sarcomere? FEBS Journal 273(15): 3427–3436 DOI 10.1111/j.1742-4658.2006.05351.x.

Ehler E, Gautel M. 2008. The sarcomere and sarcomerogenesis. Advances in Experimental Medicine and Biology 642(January): 1–14 DOI 10.1007/978-0-387-84847-1_1.

Flynt AS, Li N, Thatcher EJ, Solnica-Krezel L, Patton JG. 2007. Zebrafish miR-214 modulates hedgehog signaling to specify muscle cell fate. Nature Genetics 39(2): 259–263 DOI 10.1038/ngl953.

Foucault G, Vacher M, Merkulova T, Keller A, Arrio-Dupont M. 1999. Presence of enolase in the M-band of skeletal muscle and possible indirect interaction with the cytosolic muscle isoform of creatine kinase. The Biochemical Journal 338(Pt 1 February): 115–121 DOI 10.1042/bj3380115.

Fox GQ, Richardson GP. 1978. The developmental morphology of Torpedo marmorata: electric organ-myogenic phase. Journal of Comparative Neurology 179:677–698 DOI 10.1002/cne.901790313.

Fox GQ, Richardson GP. 1979. The developmental morphology of Torpedo marmorata: electric organ-electrogenic phase. Journal of Comparative Neurology 185:293–216 DOI 10.1002/cne.901850205.

Frascella E, Rosolen A. 1998. Detection of the myoD1 transcript in rhabdomyosarcoma cell lines and tumor samples by reverse transcription polymerase chain reaction. The American Journal of Pathology 152(2): 577–583.

Gallant JR, Traeger LL, Volkening JD, Moffett H, Chen PH, Novina CD, Phillips GN, Anand R, Wells GB, Pinch M, Güth R, Unguez GA, Albert JS, Zakon HH, Samanta MP, Sussman MR. 2014. Genomic basis for the convergent evolution of electric organs. Science 344(6191): 1522–1525 [DOI: 10.1126/science.1254432](https://doi.org/10.1126/science.1254432).

Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, Adiconis X, Fan L, Raychowdhury R, Zeng Q, Chen Z, Mauceli E, Hacohen N, Gnirke A, Rhind N, Di Palma F, Birren BW, Nusbaum C, Lindblad-Toh K, Friedman N, Regev A. 2011. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Nature Biotechnology 29(7): 644–652 [DOI: 10.1038/nbt.1883](https://doi.org/10.1038/nbt.1883).

Grounds MD, Garrett KL, Beilharz MW. 1992. The transcription of myoD1 and myogenin genes in thymic cells in vivo. Experimental Cell Research 198(2): 357–361 [DOI: 10.1016/0014-4827(92)90391-K](https://doi.org/10.1016/0014-4827(92)90391-K).

Guess MG, Barthel KKB, Harrison BC, Leslie A Leinwand. 2015. miR-30 family microRNAs regulate myogenic differentiation and provide negative feedback on the microRNA pathway. PLoS ONE 10(2): e0118229 [DOI: 10.1371/journal.pone.0118229](https://doi.org/10.1371/journal.pone.0118229).

Gundersen K. 2011. Excitation-transcription coupling in skeletal muscle: the molecular pathways of exercise. Biological Reviews 86(3): 564–600 [DOI: 10.1111/j.1469-185X.2010.00161.x](https://doi.org/10.1111/j.1469-185X.2010.00161.x).

Gunning P, Hardeman E. 1991. Multiple mechanisms regulate muscle fiber diversity. The FASEB Journal 5: 3064–3070.

Güth R, Pinch M, Unguez GA. 2013. Mechanisms of muscle gene regulation in the electric organ of sternopygus macrurus. The Journal of Experimental Biology 216(Pt 13): 2469–2477 [DOI: 10.1242/jeb.082404](https://doi.org/10.1242/jeb.082404).

Hellerschmied D, Clausen T. 2014. Myosin chaperones. Current Opinion in Structural Biology 25(100): 9–15 [DOI: 10.1016/j.sbi.2013.11.002](https://doi.org/10.1016/j.sbi.2013.11.002).

Johnston IA, Herring PJ. 1985. The transformation of muscle into bioluminescent tissue in the fish benthalebella infans zagmayer. Proceedings of the Royal Society B: Biological Sciences 225(1239): 213–218 [DOI: 10.1098/rspb.1985.0059](https://doi.org/10.1098/rspb.1985.0059).

Joshi NA, Fass JN. 2011. Sickle: a sliding-window, adaptive, quality-based trimming tool for FASTQ files. (Version 1.33). Available at [https://github.com/najoshi/sickle](https://github.com/najoshi/sickle) (accessed 2 July).

Kanehisa M, Goto S. 2000. KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Research 28(1): 27–30 [DOI: 10.1093/nar/28.1.27](https://doi.org/10.1093/nar/28.1.27).

Ketley A, Warren A, Holmes E, Gering M, Aziz Aboobaker A, David Brook J. 2013. The miR-30 microRNA family targets smoothened to regulate hedgehog signaling in zebrafish early muscle development. PLoS ONE 8(6): e65170 [DOI: 10.1371/journal.pone.0065170](https://doi.org/10.1371/journal.pone.0065170).

Kim HJ, Güth R, Jonsson CB, Unguez GA. 2009. S. Macrurus myogenic regulatory factors (MRFs) induce mammalian skeletal muscle differentiation; evidence for

functional conservation of MRFs. The International Journal of Developmental Biology 53(7):993–1002 DOI 10.1387/ijdb.082672hk.

Kim JA, Jonsson CB, Calderone T, Unguez GA. 2004. Transcription of myoD and myogenin in the non-contractile electrogenic cells of the weakly electric fish, sternopygus macrurus. Development Genes and Evolution 214(8):380–392 DOI 10.1007/s00427-004-0421-5.

Kim JA, Laney C, Curry J, Unguez GA. 2008. Expression of myogenic regulatory factors in the muscle-derived electric organ of sternopygus macrurus. The Journal of Experimental Biology 211(Pt 13):2172–2184 DOI 10.1242/jeb.016592.

Kiri A, Goldspink G. 2002. RNA-protein interactions of the 3′ untranslated regions of myosin heavy chain transcripts. Journal of Muscle Research and Cell Motility 23(2):119–129 DOI 10.1023/A:1020211729728.

Koressaar T, Remm M. 2007. Enhancements and modifications of primer design program primer3. Bioinformatics 23(10):1289–1291 DOI 10.1093/bioinformatics/btm091.

Kornstein MJ, Asher O, Fuchs S. 1995. Acetylcholine receptor alpha-subunit and myogenin mRNAs in thymus and thymomas. The American Journal of Pathology 146(6):1320–1324.

Krüger J, Rehmsmeier M. 2006. RNAhybrid: microRNA target prediction easy, fast and flexible. Nucleic Acids Research 34(suppl 2):W451–W454 DOI 10.1093/nar/gkl243.

Lamanna F, Kirschbaum F, Waurick I, Dieterich C, Tiedemann R. 2015. Cross-tissue and cross-species analysis of gene expression in skeletal muscle and electric organ of african weakly-electric fish (Teleostei; Mormyridae). BMC Genomics 16(1):668 DOI 10.1186/s12864-015-1858-9.

Langmead B, Trapnell C, Pop M, Salzberg SL. 2009. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biology 10(3): Article R25 DOI 10.1186/gb-2009-10-3-r25.

Leng N, Dawson JA, Thomson JA, Ruotti V, Rissman AI, Smits BMG, Haag JD, Gould MN, Stewart RM, Kendziorski C. 2013. EBSeq: an empirical bayes hierarchical model for inference in RNA-seq experiments. Bioinformatics 29(8):1035–1043 DOI 10.1093/bioinformatics/btt087.

Lewis BP, Burge CB, Bartel DP. 2005. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets. Cell 120(1):15–20 DOI 10.1016/j.cell.2004.12.035.

Li B, Dewey CN. 2011. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinformatics 12(January):323 DOI 10.1186/1471-2105-12-323.

Love MI, Huber W, Anders S. 2014. Moderated estimation of fold change and dispersion for RNA-Seq data with DESeq2. Genome Biology 15(12):Article 550 DOI 10.1186/s13059-014-0550-8.

McCarthy JJ. 2011. The myomiR network in skeletal muscle plasticity. Exercise and Sport Sciences Reviews 39(3):150–154 DOI 10.1097/JES.0b013e31821c01e1.

Musumeci G, Castrogiovanni P, Coleman R, Szychlinska MA, Salvatorelli L, Parenti R, Magro G, Imbesi R. 2015. Somitogenesis: from somite to skeletal muscle. Acta Histochemica 4–5:313–328 [DOI: 10.1016/j.acthis.2015.02.011].

Neguembor MV, Jothi M, Gabellini D. 2014. Long noncoding RNAs, emerging players in muscle differentiation and disease. Skeletal Muscle 4(1): Article 8 [DOI: 10.1186/2044-5040-4-8].

Pfaffl MW, Tichopad A, Prgomet C, Neuvians TP. 2004. Determination of stable housekeeping genes, differentially regulated target genes and sample integrity: bestkeeper-excel-based tool using pair-wise correlations. Biotechnology Letters 26(6):509–515 [DOI: 10.1023/B:BILE.0000019559.84305.47].

Premier Biosoft. 2015. NetPrimer. Available at <http://www.premierbiosoft.com/netprimer/> (accessed 2 July).

R Core Team. 2016. R: a language and environment for statistical computing. Vienna: R Foundation for Statistical Computing. Available at <https://www.R-project.org>.

Rao PK, Kumar RM, Farkhondeh M, Baskerville S, Lodish HF, He G, Gu S, Li S, Zhou X, Lam TW, Li Y, Xu X, Wong GK, Wang J. 2006. Myogenic factors that regulate expression of muscle-specific microRNAs. Proceedings of the National Academy of Sciences of the United States of America 103(23):8721–8726 [DOI: 10.1073/pnas.0602831103].

Rehmsmeier M, Steffen P, Höchsmann M, Giegerich R. 2004. Fast and effective prediction of microRNA/target duplexes. RNA 10:1507–1517 [DOI: 10.1261/rna.5248604].

Schiaffino S, Reggiani C. 2011. Fiber types in mammalian skeletal muscles. Physiological Reviews 91(4):1447–1531 [DOI: 10.1152/physrev.00031.2010].

Shi L, Zhou B, Li P, Schinckel AP, Liang T, Wang H, Li H, Fu L, Chu Q, Huang R. 2015. MicroRNA-128 targets myostatin at coding domain sequence to regulate myoblasts in skeletal muscle development. Cellular Signalling 27(9):1895–1904 [DOI: 10.1016/j.cellsig.2015.05.001].

Silver N, Best S, Jiang J, Thein SL. 2006. Selection of housekeeping genes for gene expression studies in human reticulocytes using real-time PCR. BMC Molecular Biology 7(January):33 [DOI: 10.1186/1471-2199-7-33].

Smedley D, Haider S, Durinck S, Pandini L, Provero P, Allen J, Arnaiz O, Awedh MH, Baldock R, Barbiera G, Bardou P, Beck T, Blake A, Bonierbale M, Brookes AJ, Bucci G, Buetti I, Burge S, Cabau C, Carlson JW, Chelala C, Chrysostomou C, Cittaro D, Collin O, Cordova R, Cutts RJ, Dassi E, Di Genova A, Djari A, Esposito A, Estrella H, Eyras E, Fernandez-Banet J, Forbes S, Free RC, Fujisawa T, Gadaleta E, Garcia-Manteiga JM, Goodstein D, Gray K, Guerra-Assunção JA, Haggarty B, Han DJ, Han BW, Harris T, Harshbarger J, Hastings RK, Hayes RD, Hoede C, Hu S, Hu ZL, Hutchins L, Kan Z, Kawaji H, Keliet A, Kerhornou A, Kim S, Kinsella R, Klopp C, Kong L, Lawson D, Lazarevic D, Lee JH, Letellier T, Li CY, Lio P, Liu CJ, Luo J, Maass A, Mariette J, Maurel T, Merella S, Mohamed AM, Moreews F, Nabihoudine I, Ndegwa N, Noirot C, Perez-Llamas C, Primig M, Quattrone A, Quesneville H, Rambaldi D, Reecy J, Riba M, Rosanoff S, Saddiq AA, Salas E, Sallou O, Shepherd R, Simon R, Sperling L, Spooner W, Staines DM, Steinbach

D, Stone K, Stupka E, Teague JW, Dayem Ullah AZ, Wang J, Ware D, Wong-Erasmus M, Youens-Clark K, Zadissa A, Zhang SJ, Kasprzyk A. 2015. The bioMart community portal: an innovative alternative to large, centralized data repositories. Nucleic Acids Research 43(W1): W589–W598 [DOI 10.1093/nar/gkv350].

Soleimani VD, Yin H, Jahani-Asl A, Ming H, Kockx CEM, Van Ijcken WFJ, Grosveld F, Rudnicki MA. 2012. Snail regulates myoD binding-site occupancy to direct enhancer switching and differentiation-specific transcription in myogenesis. Molecular Cell 47(3): 457–468 [DOI 10.1016/j.molcel.2012.05.046].

Sun L, Xie H, Mori MA, Alexander R, Yuan B, Hattangadi SM, Liu Q, Kahn CR, Lodish HF. 2011. Mir193b-365 is essential for brown fat differentiation. Nature Cell Biology 13(8): 958–965 [DOI 10.1038/ncb2286].

Takebayashi-Suzuki K, Pauliks LB, Eltsefon Y, Mikawa T. 2001. Purkinje fibers of the avian heart express a myogenic transcription factor program distinct from cardiac and skeletal muscle. Developmental Biology 234(2): 390–401 [DOI 10.1006/dbio.2001.0270].

Tonin PN, Scrable H, Shimada H, Cavenee WK. 1991. Muscle-specific gene expression in rhabdomyosarcomas and stages of human fetal skeletal muscle development. Cancer Research 51(19): 5100–5106.

Traeger LL, Volkening JD, Moffett H, Gallant JR, Chen P-H, Novina CD, Phillips Jr GN, Anand R, Wells GB, Pinch M, Güth R, Unguez GA, Albert JS, Zakon H, Sussman MR, Samanta MP. 2015. Unique patterns of transcript and miRNA expression in the South American strong voltage electric eel (electrophorus electricus). BMC Genomics 16(1): 243 [DOI 10.1186/s12864-015-1288-8].

Treves S, Vukcevic M, Maj M, Thurnheer R, Mosca B, Zorzato F. 2009. Minor sarcoplasmic reticulum membrane components that modulate excitation-contraction coupling in striated muscles. The Journal of Physiology 587(Pt 13): 3071–3079 [DOI 10.1113/jphysiol.2009.171876].

Ulitsky I, Shkumatava A, Jan CH, Subtelny AO, Koppstein D, Bell GW, Sive H, Bartel DP. 2012. Extensive alternative polyadenylation during zebrafish development. Genome Research 22(10): 2054–2066 [DOI 10.1101/gr.139733.112].

Unguez GA, Zakon HH. 1998. Phenotypic conversion of distinct muscle fiber populations to erythrocytes in a weakly electric fish. The Journal of Comparative Neurology 399(1): 20–34 [DOI 10.1002/(SICI)1096-9861(19980914)399:1<20::AID-CNE2>3.0.CO;2-C].

Untergasser A, Cutcutache I, Koressaar T, Ye J, Faircloth BC, Remm M, Rozen SG. 2012. Primer3—new capabilities and interfaces. Nucleic Acids Research 40(15): e115 [DOI 10.1093/nar/gks596].

Vandesompele J, De Preter K, Pattyn F, Poppe B, Van Roy N, De Paepe A, Speleman F. 2002. Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome Biology 3(7): research0034.1-research 0034.11 [DOI 10.1186/gb-2002-3-7-research0034].

Von Roretz C, Beauchamp P, Di Marco S, Gallouzi IE. 2011. HuR and myogenesis: being in the right place at the right time. *Biochimica et Biophysica Acta* 1813(9):1663–1667 [DOI 10.1016/j.bbamcr.2011.01.036](https://doi.org/10.1016/j.bbamcr.2011.01.036).

Vracar-Grabar M, Russell B. 2004. Creatine kinase is an alpha myosin heavy chain 3′ UTR mRNA binding protein. *Journal of Muscle Research and Cell Motility* 25(4–5):397–404 [DOI 10.1007/s10974-004-1141-1](https://doi.org/10.1007/s10974-004-1141-1).

Walker GA, Guerrero IA, Leinwand LA. 2001. Myofibroblasts: molecular crossdressers. *Current Topics in Developmental Biology* 51(January):91–107 [DOI 10.1016/S0070-2153(01)51003-0](https://doi.org/10.1016/S0070-2153(01)51003-0).

Wigmore PM, Evans DJ. 2002. Molecular and cellular mechanisms involved in the generation of fiber diversity during myogenesis. *International Review of Cytology* 216:175–232 [DOI 10.1016/S0074-7696(02)16006-2](https://doi.org/10.1016/S0074-7696(02)16006-2).

Wijnaendts LCD, Van Der Linden JC, Van Unnik AJM, Delemarre JFM, Voute PA, Meijer CJLM. 1994. The expression pattern of contractile and intermediate filament proteins in developing skeletal muscle and rhabdomyosarcoma of childhood: diagnostic and prognostic utility. *The Journal of Pathology* 174(4):283–292 [DOI 10.1002/path.1711740408](https://doi.org/10.1002/path.1711740408).

Witt CC, Witt SH, Lerche S, Labeit D, Back W, Labeit S. 2008. Cooperative control of striated muscle mass and metabolism by MuRF1 and MuRF2. *The EMBO Journal* 27(2):350–360 [DOI 10.1038/sj.emboj.7601952](https://doi.org/10.1038/sj.emboj.7601952).

Xie Y, Wu G, Tang J, Luo R, Patterson J, Liu S, Huang W, He G, Gu S, Li S, Zhou X, Lam TW, Li Y, Xu X, Wong GK, Wang J. 2014. SOAPdenovo-Trans: de novo transcriptome assembly with short RNA-seq reads. *Bioinformatics* 30(12):1660–1666 [DOI 10.1093/bioinformatics/btu077](https://doi.org/10.1093/bioinformatics/btu077).

Xie F, Xiao P, Chen D, Xu L, Zhang B. 2012. miRDeepFinder: a MiRNA analysis tool for deep sequencing of plant small RNAs. *Plant Molecular Biology* 80(1):75–84 [DOI 10.1007/s11103-012-9885-2](https://doi.org/10.1007/s11103-012-9885-2).

Zacharewicz E, Della Gatta P, Reynolds J, Garnham A, Crowley T, Russell AP, Lamon S. 2014. Identification of microRNAs linked to regulators of muscle protein synthesis and regeneration in young and old skeletal muscle. *PLoS ONE* 9(12):e114009 [DOI 10.1371/journal.pone.0114009](https://doi.org/10.1371/journal.pone.0114009).

Zhao S, Fernald RD. 2005. Comprehensive algorithm for quantitative real-time polymerase chain reaction. *Journal of Computational Biology: A Journal of Computational Molecular Cell Biology* 12(8):1047–1064 [DOI 10.1089/cmb.2005.12.1047](https://doi.org/10.1089/cmb.2005.12.1047).
