
Cell Reports

HCMV vCXCL1 Binds Several Chemokine Receptors and Preferentially Attracts Neutrophils over NK Cells by Interacting with CXCR2

Graphical Abstract
![](image.png)

Authors
Rachel Yamin, Laura S.M. Lecker, Yiska Weisblum, Alon Vitenshtein, Vu Thuy Khanh Le-Trilling, Dana G. Wolf, Ofer Mandelboim

Correspondence
oferm@ekmd.huji.ac.il

In Brief
Viral CXCL1 (vCXCL1) is a chemokine produced following infection with human cytomegalovirus. Yamin et al. show here that vCXCL1 binds to three chemokine receptors: CXCR1, CXCR2, and CX3CR1 and that neutrophils migrate faster and more efficiently than NK cells toward vCXCL1 through binding to CXCR2 expressed by neutrophils only.

Highlights
- vCXCL1 binds the chemokine receptor CX3CR1 expressed on CD56<sup>Dim</sup> CD16<sup>Pos</sup> NK cells
- vCXCL1 binds CXCR2 on neutrophils and CXCR1 on both NK cells and neutrophils
- Neutrophils migrate faster and more efficiently through the binding of vCXCL1 to CXCR2
- HCMV uses vCXCL1 to attract neutrophils and orchestrate its dissemination in the body

Yamin et al., 2016, Cell Reports 15, 1542–1553  
May 17, 2016 © 2016 The Author(s)  
http://dx.doi.org/10.1016/j.celrep.2016.04.042

OPEN ACCESS
CellPress

**HCMV vCXCL1 Binds Several Chemokine Receptors and Preferentially Attracts Neutrophils over NK Cells by Interacting with CXCR2**

Rachel Yamin,¹ Laura S.M. Lecker,² Yiska Weisblum,³ Alon Vitenshtein,⁴ Vu Thuy Khanh Le-Trilling,⁵ Dana G. Wolf,³ and Ofer Mandelboim¹,*  

¹The Lautenberg Center for General and Tumor Immunology, The BioMedical Research Institute Israel Canada of the Faculty of Medicine (IMRIC), The Hebrew University Hadassah Medical School, Jerusalem 91120, Israel  
²German Cancer Research Center, DKFZ, 69120 Heidelberg, Germany  
³Clinical Virology Unit, Hadassah Hebrew University Medical Center and Department of Biochemistry and Chanock Center for Virology, IMRIC, Jerusalem 91120, Israel  
⁴Institute for Virology of the University Hospital Essen, University Duisburg-Essen, 45147 Essen, Germany  
*Correspondence: oferm@ekmd.huji.ac.il  
http://dx.doi.org/10.1016/j.celrep.2016.04.042

---

### SUMMARY

HCMV is a highly sophisticated virus that has developed various mechanisms for immune evasion and viral dissemination throughout the body (partially mediated by neutrophils). NK cells play an important role in elimination of HCMV-infected cells. Both neutrophils and NK cells utilize similar sets of chemokine receptors to traffic, to and from, various organs. However, the mechanisms by which HCMV attracts neutrophils and not NK cells are largely unknown. Here, we show a unique viral protein, vCXCL1, which targets three chemokine receptors: CXCR1 and CXCR2 expressed on neutrophils and CXCR1 and CX3CR1 expressed on NK cells. Although vCXCL1 attracted both cell types, neutrophils migrated faster and more efficiently than NK cells through the binding of CXCR2. Therefore, we propose that HCMV has developed vCXCL1 to orchestrate its rapid systemic dissemination through preferential attraction of neutrophils and uses alternative mechanisms to counteract the later attraction of NK cells.

---

### INTRODUCTION

Human cytomegalovirus (HCMV) belongs to the beta herpesvirus subfamily (Jain et al., 2011; Stern-Ginossar et al., 2012). Following primary infection, HCMV establishes latent infection, similarly to other members of the herpesvirus family (Goodrum et al., 2012; Slobedman et al., 2010). Occasionally, HCMV can undergo re-activation and initiate productive infection in which new infectious progeny are produced. HCMV infection is mostly asymptomatic in immune-competent individuals, whereas in immune-compromised individuals (such as transplant recipients) or in congenitally infected neonates, it can cause substantial morbidity and mortality (Griffiths, 2012; Slobedman et al., 2010).

Like other viruses of the herpesvirus family, HCMV has developed various sophisticated mechanisms to evade the innate and adaptive immune responses (Jackson et al., 2011). In particular, HCMV has developed several mechanisms to evade and subvert the activity of natural killer (NK) cells (Brizić et al., 2014; Fielding et al., 2014; Halenius et al., 2015; Lisnić et al., 2015; Seidel et al., 2015; Slavuljica et al., 2011; Stern-Ginossar et al., 2007; Wilkinson et al., 2008).

NK cells belong to the innate immune system (Fehniger et al., 2003) and are best known for their important role in controlling tumor development and viral infections (Cheent and Khakoo, 2009; Cooper et al., 2001). Mature NK cells in the blood can be divided into two subpopulations based on the expression of CD56 and CD16. The CD56Dim NK cells express intermediate levels of the CD56 adhesion molecule, express CD16, and constitute around 90% of the NK cells, whereas CD56Bright NK cells that express high levels of CD56 lack the CD16 receptor and constitute approximately 10% of the peripheral blood NK cells (Cooper et al., 2001; Fehniger et al., 2003). These two NK cell populations are also functionally distinct: CD56Dim CD16Pos NK cells are highly cytotoxic, whereas the CD56Bright CD16Neg NK cells are poorly cytotoxic and are mainly responsible for cytokine secretion (Cooper et al., 2001).

Both NK cell populations express adhesion molecules and chemokine receptors that enable them to circulate in the blood and migrate into secondary lymphoid organs or inflamed tissues (Carrega and Ferlazzo, 2012; Hanna et al., 2003; Santoni et al., 2007). There is inconsistency in the literature regarding the chemokine receptor expression profile on the cell surface of the two NK cell populations. To our knowledge, most publications agree that freshly isolated CD56Dim CD16Pos NK cells express high levels of CXCR1 and CX3CR1; low levels of CXCR2, CXCR4, and CCR5 (which might differ between donors); and no detectable levels of CCR1–4 or CCR6 (Berahovich et al., 2006; Hanna et al., 2003; Ingjerdingen et al., 2001; Morohashi et al., 1995; Robertson, 2002; Yoneda et al., 2000). Hence, this NK population migrates mainly toward fractalkine (Fck/CX3CL1; CX3CR1 ligand), stromal-cell-derived factor 1 alpha (SDF1α/CXCL12; CXCR4 ligand), and
interleukin-8 (IL-8) (CXCL8; CXCR1 ligand). In contrast, CD56<sup>Bright</sup>CD16<sup>Neg</sup> NK cells express mostly CCR5, CCR7, and CXCR3 and low levels of CXCR4 and CX3CR1 and do not express CCR1–4, CCR6, CXCR1, or CXCR2 ([Campbell et al., 2001](https://doi.org/10.1084/jem.194.11.1621); [Hanna et al., 2003](https://doi.org/10.1084/jem.20021874)). Migration of this NK cell subtype is mostly stimulated by CXCL10 and CXCL11 (CXCR3 ligands), SDF1α (CXCR4 ligand), and CCL19 and CCL21 (CCR7 ligands; [Campbell et al., 2001](https://doi.org/10.1084/jem.194.11.1621); [Hanna et al., 2003](https://doi.org/10.1084/jem.20021874); [Morohashi et al., 1995](https://doi.org/10.1084/jem.182.6.1857); [Yoneda et al., 2000](https://doi.org/10.1084/jem.192.11.1609)).

Emerging evidence has demonstrated that NK cells cross-talk with other immune cell types such as neutrophils, which also belong to the innate immune system ([Scapini and Cassatella, 2014](https://doi.org/10.1182/blood-2013-10-533695)). Neutrophils are polymorphonuclear leukocytes and are the first leukocytes to appear on site during the initiation of an inflammatory process. CXCR1, CXCR2, and CXCR4 are the main chemokine receptors expressed by human neutrophils ([Eash et al., 2009](https://doi.org/10.1182/blood-2008-07-170048); [Kolaczkowska and Kubes, 2013](https://doi.org/10.1016/j.immuni.2013.02.005); [Petering et al., 1999](https://doi.org/10.1084/jem.189.11.1897)). SDF1α expression by stromal cells is responsible for the retention signal that keeps the neutrophils in the bone marrow ([Eash et al., 2009](https://doi.org/10.1182/blood-2008-07-170048), [2010](https://doi.org/10.1182/blood-2009-10-249982)). IL-8 (CXCR1 and CXCR2 ligand), on the other hand, is involved in recruitment and activation of neutrophils during inflammation ([Hammond et al., 1995](https://doi.org/10.1084/jem.182.6.1857)).

HCMV has developed several mechanisms to avoid innate immune cell attack, one of which is viral mimicry of chemokines ([Alcami, 2003](https://doi.org/10.1016/S0167-5699(03)00127-X); [McSharry et al., 2012](https://doi.org/10.1128/JVI.06488-11)). HCMV encodes one gene, UL128, which exhibits sequence similarity to CC chemokines. UL128 participates in HCMV entry into endothelial and epithelial cells and promotes proliferation of peripheral blood mononuclear cells (PBMCs) by activating the MAPK/ERK (mitogen-activated protein kinase/extracellular-signal-regulated kinase)-signaling pathway ([Zheng et al., 2012](https://doi.org/10.1084/jem.20111998)). In addition, HCMV encodes two genes, UL146 and UL147, with sequence similarity to CXC chemokines ([Penfold et al., 1999](https://doi.org/10.1084/jem.189.11.1897)). Previous papers showed that viral CXC-ligand 1 (vCXCL1; UL146 gene product) induces calcium mobilization and chemotaxis of human neutrophils. However, contradicting results were obtained regarding the identity of the receptors participating in the neutrophil migration by vCXCL1. Some papers show that the migration is mediated by both CXCR1 and CXCR2, whereas others claim that CXCR2 alone is responsible for neutrophil migration through vCXCL1 ([Heo et al., 2015](https://doi.org/10.1016/j.jbiotec.2015.02.010); [Lütichau, 2010](https://doi.org/10.1016/j.jbiotec.2010.01.010); [Penfold et al., 1999](https://doi.org/10.1084/jem.189.11.1897)). Although vCXCL2 (UL147 gene product) was discovered to be a CXC chemokine quite some time ago, little is known about its function or ability to bind to chemokine receptors on human immune cells ([Penfold et al., 1999](https://doi.org/10.1084/jem.189.11.1897)).

In our previous paper, we demonstrated that Kaposi’s sarcoma-associated herpesvirus (KSHV) encodes a viral chemokine, vMIP-II, which binds freshly isolated CD56<sup>Dim</sup> CD16<sup>Pos</sup> NK cells via CX3CR1 and IL-2-activated NK cells through CCR5. Furthermore, we showed that this binding inhibits NK cell migration in vitro ([Yamin et al., 2013](https://doi.org/10.1128/JVI.02948-12)). Because HCMV is known to be a master of immune evasion, we wanted to investigate whether the HCMV-derived viral chemokines will also have a similar effect on NK cells. Here, we show that vCXCL1, one of the viral chemokines encoded by HCMV, functions as an agonist and induces NK cell and neutrophil migration. We demonstrated that vCXCL1 binds CX3CR1. We show that neutrophils migrated faster and more efficiently than NK cells, and we propose that HCMV has

developed vCXCL1 to orchestrate its rapid systemic dissemination through preferential attraction of neutrophils and uses alternative mechanisms to counteract the later secondary attraction of NK cells.

## RESULTS

### vCXCL1-Ig Binds CD56<sup>Dim</sup> CD16<sup>Pos</sup> Naive NK Cells

To test whether vCXCL1 interacts with NK cells, we cloned it from cDNA derived from urine of a congenitally infected newborn (most resemble the Merlin HCMV strain; [Figure S1](#figS1); [Experimental Procedures](#secExpProcedures)) and fused it to the Fc domain of human immunoglobulin G1 (IgG1). The construct was stably expressed in 293T cells, and the corresponding protein was purified using a protein G column and used to stain freshly isolated NK cells or IL-2-activated NK cells. As shown in [Figure 1](#fig1), vCXCL1-Ig binds CD56<sup>Dim</sup> CD16<sup>Pos</sup> naive NK cells ([Figure 1A](#fig1), red circles; figure shows vCXCL1 staining relative to CD56 only), but not the CD56<sup>Bright</sup> CD16<sup>Neg</sup> NK cell population ([Figure 1A](#fig1), black circles; staining of several donors is summarized in [Figure 1B](#fig1)). IL-2-activated NK cells were not recognized by vCXCL1-Ig ([Figure 1C](#fig1); staining of several donors is summarized in [Figure 1D](#fig1)). No staining was detected when a control-Ig protein was used ([Figure 1](#fig1)).

### CD56<sup>Dim</sup> CD16<sup>Pos</sup> NK Cells Mainly Express Two Chemokine Receptors: CXCR1 and CX3CR1

There are mixed reports regarding the precise identity of the chemokine receptors that interact with vCXCL1 ([Lütichau, 2010](https://doi.org/10.1016/j.jbiotec.2010.01.010); [Penfold et al., 1999](https://doi.org/10.1084/jem.189.11.1897)). To determine the identity of the chemokine receptor(s) expressed on NK cells, which may interact with vCXCL1, we initially tested which chemokine receptors are expressed on the CD56<sup>Dim</sup> CD16<sup>Pos</sup> NK cells by double staining of freshly isolated NK cells with anti-CD56 in tandem with anti-CCR1, CCR2, CCR3, CCR5, CXCR1, CXCR2, CXCR4, or CX3CR1 monoclonal antibodies (mAbs). As can be seen in [Figure 2A](#fig2) and summarized in [Table S1A](#tabS1), CXCR1 and CX3CR1 were the only receptors exhibiting expression patterns similar to that of vCXCL1-Ig staining of freshly isolated NK cells (compare [Figures 2A](#fig2) and [1A](#fig1)). Both CXCR1 and CX3CR1 were expressed on the entire CD56<sup>Dim</sup> CD16<sup>Pos</sup>, but not on the CD56<sup>Bright</sup> CD16<sup>Neg</sup> NK cell population and were completely downregulated following NK cell activation ([Figure 2B](#fig2); [Table S1C](#tabS1)). Other chemokine receptors that we tested showed low or no expression on the CD56<sup>Dim</sup> CD16<sup>Pos</sup> NK cell population ([Figure 2A](#fig2); [Table S1](#tabS1)). The expression of two other receptors, CCR5 and CXCR4, were donor-dependent ([Table S1B](#tabS1)). Thus, we considered the possibility that CXCR1 and CX3CR1 might interact with vCXCL1 on naive NK cells.

### rvCXCL1 Induces Migration of Naive NK Cells via CXCR1 and CX3CR1

We have previously demonstrated that the KSHV-encoded chemokine vMIP-II blocks NK cell migration ([Yamin et al., 2013](https://doi.org/10.1128/JVI.02948-12)). To investigate whether vCXCL1 also antagonizes the migration of NK cells, we performed transwell migration assay in the presence of increasing concentrations of recombinant vCXCL1 (rvCXCL1), derived from a Toledo strain ([Figure S1](#figS1)). Surprisingly, and in contrast to what we observed with the KSHV-derived

Cell Reports 15, 1542–1553, May 17, 2016 1543
Figure 1. vCXCL1-Ig Binds CD56<sup>Dim</sup> CD16<sup>Pos</sup> Naive NK Cells

(A and C) Dot plot FACS staining of freshly isolated naive NK cells (A) and IL-2-activated NK cells (C) stained with vCXCL1-Ig or with control-Ig together with anti-CD56. The percentages of various populations are indicated inside the dot plots. Figure shows one representative staining out of more than four performed.

(B and D) Summary data of inter-donor variability in vCXCL1-Ig binding to naive NK cells (B) or IL-2-activated NK cells (D). Table shows both mean fluorescence intensity (MFI) and the percentage of the positive cells.

| Donor# | Control-Ig MFI | vCXCL1-Ig MFI | Control-Ig positive cells (%) | vCXCL1-Ig positive cells (%) |
|--------|-----------------|---------------|-------------------------------|------------------------------|
| Donor #1 | 2.57 | 32.49 | 3.07% | 62.34% |
| Donor #2 | 2.92 | 19.81 | 1.99% | 75.86% |
| Donor #3 | 3.02 | 20.54 | 1.84% | 62.52% |
| AVG±SD | 2.8±0.2 | 24.3±7.1 | 2.3±0.7 | 66.9±7.7 |
| P-value | 0.034 |  |  | 0.004 |

| Donor# | Control-Ig MFI | vCXCL1-Ig MFI | Control-Ig positive cells (%) | vCXCL1-Ig positive cells (%) |
|--------|-----------------|---------------|-------------------------------|------------------------------|
| Donor #1 | 2.81 | 2.84 | 0.3 | 3.45 |
| Donor #2 | 2.44 | 2.69 | 0.93 | 1.09 |
| Donor #3 | 2.25 | 2.91 | 3.13 | 4.15 |
| AVG±SD | 2.5±0.28 | 2.9±0.22 | 0.54±0.33 | 2.9±1.6 |
| P-value | 0.14 |  |  | 0.12 |

vMIP-II chemokine (Yamin et al., 2013), a significant, dose-dependent migration of NK cells was detected toward rvCXCL1 (Figure 3A). The migration efficiency was similar to NK cell migration toward rhFck and rhIL8; both are well-known chemo-attractants of NK cells (Figure 3A) and could be blocked by usage of neutralizing monoclonal antibody against vCXCL1 (Figure 3B). Thus, we concluded that vCXCL1 interacts with the naive CD56<sup>Dim</sup> CD16<sup>Pos</sup> NK cell population, possibly via CXCR1, CX3CR1, or both, and that this interaction probably leads to NK cell migration.

It is well established that binding of chemokines to their receptors can cause rapid internalization of the receptor due to agonist-dependent phosphorylation of the C-terminal tail of the G protein-coupled receptor (Allen et al., 2007; Neel et al., 2005). Therefore, chemokine receptor internalization is used as readout for interaction with an appropriate ligand (Allen et al., 2007; Neel et al., 2005). To investigate the ability of vCXCL1 to bind CXCR1 and CX3CR1 on NK cells, we performed internalization assays in which we incubated freshly isolated NK cells with two doses (0.1 μg or 1 μg) of rvCXCL1 and then stained them with anti-CX3CR1 or anti-CXCR1 (Figure 4A; summarized in Table S2). For controls, we used recombinant human IL-8 and fractalkine (rhIL8 and rhFck, the natural ligands of CXCR1 and CX3CR1, respectively). Incubation of NK cells with 1 μg of rvCXCL1 resulted in a downregulation of both CX3CR1 and CXCR1 (Figure 4A, green histograms), indicating that vCXCL1 interacts with both receptors. The vCXCL1-mediated internalization was significant (~25% and ~40% reduction in CX3CR1 and CXCR1 cell surface expression, respectively) and repeated with similar outcome between the different donors (Table S2). Incubation of rhFck with NK cells led to a complete downregulation of CX3CR1, but not

CXCR1 (Figure 4A, blue histograms; summarized in Table S2), whereas incubation of rhIL8 resulted in a complete downregulation of CXCR1, but not CX3CR1 (Figure 4A, red histograms; summarized in Table S2). The downregulation of CX3CR1 and CXCR1 by their corresponding natural ligands was complete and observed using a low quantity (0.1 μg) of the appropriate chemokines (Figure 4A; summarized in Table S2). Thus, we concluded that vCXCL1 interacts directly with CX3CR1 and CXCR1 and that these interactions are probably of low affinity.

It was previously shown that vCXCL1 interacts with CXCR1, but not with CX3CR1 (Lüttichau, 2010). Thus, to further demonstrate that vCXCL1 directly interacts with CX3CR1, we stably expressed CX3CR1 in 293T cells and used these 293T-CX3CR1-transfected cells to assay for vCXCL1-Ig binding. As can be seen, we observed that vCXCL1-Ig binds preferentially to the 293T-CX3CR1 transfected cells (black empty histogram) as compared to 293T parental cells (gray empty histogram; Figure 4B).

To further demonstrate that vCXCL1 directly interacts with CX3CR1 and CXCR1 and that this interaction is functional, we conducted transwell migration assays. In these assays, we pre-incubated freshly isolated NK cells with rvCXCL1, rhFck, or rhIL8 and then tested the cells' migration toward rvCXCL1. As shown in Figure 4C, rvCXCL1 caused NK cell migration and this migration was inhibited by around 40% when NK cells were pre-incubated with rvCXCL1, rhFck, or rhIL8. Approximately 80% inhibition of migration was observed. The chemokine-mediated inhibition of migration was specific because no effect was detected when recombinant human chemerin (rhChemerin; ChemR23 ligand) was used. Thus, vCXCL1 binds CX3CR1 and CXCR1 on NK cells and attracts NK cells.

Figure 2. CD56<sup>Dim</sup> CD16<sup>Pos</sup> Naive NK Cells Mainly Express Two Chemokine Receptors: CXCR1 and CX3CR1

(A and B) Dot plot FACS staining of freshly isolated naive NK cells (A) and IL-2-activated NK cells (B) double stained with anti-CD56 mAb together with specific antibodies against each of the following chemokine receptors: CCR1; CCR2; CCR3; CCR5; CXCR1; CXCR2; CXCR4; and CX3CR1 (indicated on the x axes). The percentages of various populations are indicated inside the dot plots. Figure shows one representative staining out of more than four performed for each receptor. See also Table S1.

### rvCXCL1 Induces Migration of Neutrophils via CXCR1 and CXCR2

(Fielding et al., 2014; Seidel et al., 2015; Slavuljica et al., 2011; Stern-Ginossar et al., 2007; Wilkinson et al., 2008). These observations together with the results presented here, in which the binding of vCXCL1 to CX3CR1 and CXCR1 on NK cells is

S3B). Interestingly, a pronounced downregulation (~75% reduction; Table S3B) of CXCR2 was observed even when neutrophils were incubated with only 0.1 μg of rvCXCL1 (Figure 5C, green histograms). Thus, we conclude that, of the three vCXCL1-binding chemokine receptors: CXCR2; CX3CR1; and CXCR1, vCXCL1 binds most efficiently to CXCR2.

To test whether vCXCL1 attracts neutrophils via CXCR1 and CXCR2, we performed a migration assay in which we pre-incubated the neutrophils with rhFck, rhIL8, or rvCXCL1 and then tested their migration toward rvCXCL1. Pre-incubation with rhFck showed no effect on neutrophil migration and was therefore set as 100% (Figure 5D). In contrast, pre-incubation of neutrophils either with rhIL8 or rvCXCL1 prevented neutrophil migration toward rvCXCL1 (Figure 5D).

### Neutrophils React Faster and More Efficiently to rvCXCL1 in Comparison to NK Cells

As rvCXCL1 seemed to bind CXCR2 with the highest affinity, and because neutrophil migration toward vCXCL1 was more efficient as compared to that of NK cells, we speculated that vCXCL1 attracts neutrophils faster than NK cells. To test this hypothesis, we simultaneously tested the migration kinetics of neutrophils and NK cells derived from the same donor toward rvCXCL1, rhIL8, or rhFck. As can be seen, neutrophils indeed migrated faster and more efficiently than NK cells toward rvCXCL1 and rhIL8 (Figure 6A). In contrast, neutrophils did not migrate toward rhFck (as they do not express CX3CR1), whereas NK cells did (Figure 6A). To further assess the preferential migration of neutrophils over NK cells toward rvCXCL1, we mixed neutrophils and NK cells together in the upper chamber of a transwell (Figure 6B) and examined their migration toward rvCXCL1, following 30 min and 3 hr. As can be seen in Figure 6C, 40% of the neutrophils, which were placed in the upper chamber together with NK cells, migrated toward rvCXCL1 after 30 min, whereas at this time point, little or no NK cell migration was observed (Figure 6C, left panel). Following 3 hr of co-incubation, NK cell migration toward rvCXCL1 was detected (Figure 6C, black columns, right panel); however, the neutrophil migration was much more pronounced (Figure 6C, gray columns, right panel). Thus, the neutrophil migration in response to vCXCL1 is faster and more efficient.

### vCXCL1 Secretion following HCMV Infection Primarily Attracts Neutrophils

Finally, we wanted to test whether vCXCL1 secreted during HCMV infection will attract neutrophils and NK cells. We initially wanted to determine the vCXCL1 concentration present in the supernatants of the infected cells. Unfortunately, the mAb that recognized vCXCL1 that was developed against the vCXCL1 Toledo strain was unable to recognize the vCXCL1 of the AD169 strain (probably because vCXCL1 is a highly polymorphic protein and substantially different between AD169 and Toledo strains; Figure S1). Therefore, to determine the concentration of vCXCL1 in the supernatant of the infected cells, we developed a calibration assay (described in detail in the Experimental Procedures section) that is based on CXCR2 internalization. The estimated vCXCL1 concentration in the supernatant was more than 150 ng/ml (blue square, Figure 7A), a concentration that is

Figure 4. RvCXCL1 Induces Naive NK Cell Migration via CX3CR1 and CXCR1

(A) Freshly isolated NK cells were incubated with and without 0.1 μg and 1 μg (indicated on the left side of the figure) of rhIL8 (red histograms, left), rhFck (blue histograms, middle), or rvCXCL1 (green histograms, right), at 37°C for 1 hr. Next, cells were stained with anti-CX3CR1 (two upper rows) or with anti-CXCR1 (two lower rows). Open gray histograms show the staining of the chemokine receptors on untreated cells. Filled gray histograms show staining of the untreated NK cells with an isotype control. The backgrounds of the treated cells were similar to the untreated cells and are not shown in the figure. See also Table S2.

(B) Binding of vCXCL1-lg to 293T-CX3CR1 transfectant (black open histogram) or to the 293T parental cells (gray open histogram). Filled gray histogram is the staining of the control fusion protein (control-lg) on the 293T-CX3CR1-transfected cells. Staining of the parental 293T cells with control-lg was similar to the 293T-CX3CR1-transfected cells and is not shown in the figure.

(C) Freshly isolated NK cells were incubated for 1 hr with and without the proteins indicated on the x axis. RvCXCL1 was placed in the bottom chamber, and migration was performed for 3 hr at 37°C. Migrating cells were counted by FACS. NK cell migration toward rvCXCL1 without pre-blocking was set as 100%, and the results are presented as % of migration. *p < 0.05. NS, not significant. Figure shows one representative experiment out of three performed.

sufficient to induce NK cell (Figure 3A) and neutrophil migration (Figure 5B).

To test whether vCXCL1 is responsible for the NK cell and neutrophil migration, we generated an HCMV deletion mutant lacking UL146 (HCMV ΔUL146) on AD169 VarL strain background. We verified that the mutant growth kinetics following infection was similar to that of the wild-type virus (Figure 7B). Next, we infected human foreskin fibroblast (HFF) either with the wild-type (WT) HCMV strain or with HCMV ΔUL146 at a MOI of 1; supernatants from the infected cells were collected 72 hr postinfection, because it was previously published that vUL146 is expressed late in the virus life cycle (Penfold et al., 1999). Migration assays were performed in parallel with freshly isolated neutrophils and NK cells toward supernatants collected from mock-infected, WT HCMV-infected, or HCMV ΔUL146-infected cells. Both neutrophils and NK cells migrated toward supernatants obtained from WT HCMV-infected cells; however, neutrophil migration was strongly induced compared to NK cell migration (Figures 7C and 7D). Importantly, in the absence of vCXCL1 (using supernatants from HCMV ΔUL146-infected cells), the neutrophil migration was almost completely abolished (Figure 7C), indicating that, following HCMV infection, neutrophil migration is primarily UL146 dependent. In contrast, only a modest (15%) but a statistically significant reduction in NK cell migration was observed in the absence of vCXCL1 (Figure 7D).

To further strengthen these observations, we pre-incubated neutrophils or NK cells with rvCXCL1 and with the key chemokines that induce downregulation of the appropriate receptors (rhIL8 in the case of neutrophils that leads to the downregulation of CXCR1 and CXCR2 and a combination of rhFck and rhIL8 that leads to the downregulation of CXCR1 and CX3CR1, in the case of NK cells). In agreement with what is shown with supernatant from HCMV ΔUL146-infected cells, incubation of neutrophils with rhIL8 completely abolished their migration

Figure 5. RvCXCL1 Induces Neutrophil Migration via CXCR1 and CXCR2

(A) Freshly isolated neutrophils were stained with mAb against CD16 and CEACAM1 (left dot plot). The double positive fraction (red square) was stained with specific antibodies against the chemokine receptors CXCR1, CXCR2, and CX3CR1.

(B) Transwell migration assays were performed using freshly isolated neutrophils toward the recombinant proteins indicated on the x axis (rhFck, rhIL8, or rvCXCL1). The number of migrating cells was quantified by FACS, following a 30-min incubation period, at 37°C. Percent of migrating cells out of total input cells was calculated. Data are presented as mean ± SEM (n = 6).

(C) Freshly isolated neutrophils were incubated with and without 0.1 μg and 1 μg (indicated in the left of the figure) of rhIL8 (red histograms, left) or rvCXCL1 (green histograms, right) for 10 min at 37°C, followed by staining with anti-CXCR1 (two upper rows) or anti-CXCR2 (two lower rows). Open gray histograms show chemokine receptors staining of the untreated cells. Filled gray histograms represent staining of the untreated cells with an isotype control. The backgrounds of the treated cells were similar to the untreated cells and are not shown in the figure.

(D) Freshly isolated neutrophils were incubated at 37°C for 10 min with or without the proteins indicated on the x axis. RvCXCL1 was placed in the bottom chamber, and the migrating neutrophils were counted using FACS following 30 min incubation at 37°C. Neutrophil migration toward rvCXCL1 after pre-blocking with rhFck was set as 100%, and the results are presented as % of migration. *p < 0.05. **p < 0.005. Figure shows one representative experiment out of three performed.

See also Table S3.

toward the supernatant from WT-HCMV-infected cells (Figure 7E). Incubation of NK cells with rhFck and rhIL8 together led to a partial reduction in the migration (Figure 7F). Thus, we concluded that, during HCMV infection, neutrophil migration is dependent on UL146, whereas NK cell migration does not depend exclusively on UL146 and might involve other receptors beside CXCR1 and CX3CR1.

DISCUSSION

Here, we show that HCMV-encoded vCXCL1 binds to NK cells and not only to neutrophils. Furthermore, we discovered, using three different assays (receptor internalization, direct binding, and functional experiments) that CX3CR1 interacts with vCXCL1. This result is in disagreement with a previous

Figure 6. Neutrophils Migrate Faster and More Efficiently Than NK Cells in Response to rvCXCL1

(A) A transwell migration assay was performed using rvCXCL1, rhIL8, or rhFck as the chemoattractant with either freshly isolated neutrophils (gray triangles) or NK cells (black squares) placed in the upper chamber for 10, 20, and 30 min at 37°C. Migration of untreated neutrophils and NK cells at the beginning of the experiment was set as 1, and the results are presented as FI. Figure shows one representative experiment out of two performed.

(B) Diagram that describes competitive transwell migration assay in which NK cells and neutrophils were incubated together at the upper chamber and their ability to migrate toward rvCXCL1 (lower chamber) is determined.

(C) Competitive transwell migration assays described in (B) was performed for 30 min (left panel) and 3 hr (right panel). Percent of migrating cells out of total input cells was calculated separately for neutrophils (gray) and NK cells (black). Data are presented as mean ± SEM (n = 3). *p < 0.05. **p < 0.005. ***p < 0.0005.

study, which demonstrated vCXCL1 binding only to CXCR1 and CXCR2, but not CX3CR1 ([Lütitchau, 2010](https://doi.org/10.1016/j.jhhm.2010.04.001)).

We show that vCXCL1 binds two chemokine receptors on CD56<sup>Dim</sup> CD16<sup>Pos</sup> NK cells, CX3CR1 and CXCR1. Indeed, blocking of these two receptors by their natural ligands, Fck and IL-8 (respectively), abolished the migration of NK cells toward the viral chemokine almost completely. Following cytokine stimulation, NK cells become highly cytotoxic and the expression profile of killer and chemokine receptors changes ([Hamann et al., 2011](https://doi.org/10.1016/j.jhhm.2010.04.001); [Inngjerdingen et al., 2001](https://doi.org/10.1016/S0196-3518(01)00233-9); [Sechler et al., 2004](https://doi.org/10.1016/j.jhhm.2010.04.001); [Vitale et al., 1998](https://doi.org/10.1016/S0196-3518(01)00233-9)). Among these receptors are also CXCR1 and CX3CR1, which disappear from the cell surface following IL-2 activation. Therefore, it is not surprising that vCXCL1 does not bind IL-2-activated NK cells.

Because vCXCL1 is a viral protein, the migration of NK cells toward vCXCL1 was surprising and raises the question as to why should the virus develop a viral chemokine to attract NK cells that can potentially eliminate the virus. HCMV is highly adapted to its human host and developed numerous mechanisms in order to escape recognition by NK cells ([Wilkinson et al., 2008](https://doi.org/10.1016/j.jhhm.2010.04.001)). For example, UL18, a viral MHC class I homolog protein, is involved in these immune escape mechanisms through efficient binding to the inhibitory receptor ILT2, leading to reduced NK cell activity ([Cosman et al., 1997](https://doi.org/10.1016/j.jhhm.2010.04.001); [Prod'homme et al., 2008](https://doi.org/10.1016/j.jhhm.2010.04.001)).

Another strategy to avoid NK cell recognition is downregulation of the stress-induced ligands. UL16, for example, binds the stress-induced ligands MICB, ULBP1, and ULBP2 and sequesters them in the endoplasmic reticulum, hampering NK activation ([Welte et al., 2003](https://doi.org/10.1016/j.jhhm.2010.04.001)). US9 is a viral glycoprotein that targets a specific allele of MICA and illustrates the co-evolution of HCMV with its human host ([Seidel et al., 2015](https://doi.org/10.1016/j.jhhm.2010.04.001)). Furthermore, HCMV uses not only viral proteins but also viral encoded microRNAs (miRNAs). HCMV-miR-UL112, for instance, downregulates the stress-induced ligand MICB during viral infection and therefore leads to reduced killing of the infected cells by NK cells ([Stern-Ginossar et al., 2007](https://doi.org/10.1016/j.jhhm.2010.04.001)). Thus, we assumed the HCMV can deal with NK cells and that vCXCL1 is produced by the virus for other purposes.

As previously shown, vCXCL1 does not only bind NK cells but also neutrophils, via the chemokine receptors CXCR1 and CXCR2 ([Heo et al., 2015](https://doi.org/10.1016/j.jhhm.2010.04.001); [Lütitchau, 2010](https://doi.org/10.1016/j.jhhm.2010.04.001)). These two receptors bind the same ligand, IL-8, and therefore, as we show here, usage of rhIL8 alone was enough to abolish neutrophil migration toward vCXCL1.

As binding of rvCXCL1 led to a partial internalization of CXCR1 and to a significantly higher internalization of CXCR2, we concluded that vCXCL1 binds CXCR2 with higher affinity compared to CXCR1. This also explains why neutrophils migrate

Figure 7. Reduced Neutrophil and NK Cell Migration in the Absence of UL146 during HCMV Infection

(A) Freshly isolated neutrophils were incubated with increasing concentrations (indicated in the x axis) of rvCXCL1 (gray circles) for 30 min at 37°C, followed by staining with anti-CXCR2. MFI of CXCR2 expression without blocking was set on 100%, and the residual CXCR2 expression was calculated. Estimated levels of vCXCL1 following infection of HFF cells with WT HCMV are shown as blue square on the graph.

(B) HFFs were infected (MOI of 0.5) with WT HCMV (blue circles) or ΔUL146 (red circles), and supernatants containing progeny viruses were harvested at the indicated hours postinfection (x axis). The plaque-forming units (PFU) were determined using a standard plaque assay on HFF monolayers.

(C and D) HFF cells were infected with WT HCMV or with HCMV ΔUL146 at a MOI of 1. Three days postinfection, cell supernatants were collected and used for transwell migration assays with either neutrophils (C) or NK cells (D). Neutrophils and NK cell migration toward supernatants from mock-infected HFF was set as 1, and the results are presented as FI.

(E and F) Freshly isolated neutrophils were incubated at 37°C for 30 min with or without the proteins indicated on the x axis. Transwell migration assays were performed with neutrophils (E) or NK cells (F) toward supernatant from mock-infected or WT-HCMV-infected HFFs. Neutrophils and NK cell migration toward supernatants from mock-infected cells was set as 1, and the results are presented as FI. *p < 0.05. **p < 0.005. ***p < 0.0005.

(G) HCMV-infected endothelial cells secrete vCXCL1 (1) and recruit both neutrophils and NK cells to the infection site (2). Migration of NK cells is dependent on CXCR1 and CX3CR1 receptors, whereas neutrophil migration is dependent on CXCR2 and CXCR1. Neutrophils migrate faster and more efficiently in comparison to NK cells due to their CXCR2 receptor (3). Therefore, neutrophils reach the infection site early and can disseminate the virus while proceeding to travel throughout the body (4). This enables the virus to maintain a pool of HCMV-infected cells. NK cells that migrate toward vCXCL1 will get to the infection site at a later time point (5) and will be subverted by HCMV-immune evasion tactics.

toward vCXCL1 more efficiently in comparison to NK cells (fold increase of 15 in neutrophils versus 2 in NK cells), as CXCR2 is not expressed by NK cells. In that respect, neutrophils not only migrate faster toward vCXCL1, but they also do it more efficiently.

vCXCL1 is a highly polymorphic protein (Figure S1). In this study, we used three variants of the protein: vCXCL1-Ig derived from a Merlin strain; rvCXCL1 derived from a Toledo strain; and

the native protein from the AD169VarL strain. It is interesting to note that, although these strains share low percentage of identity (Figure S1), they consistently showed the same binding patterns and effects, both on neutrophils and on NK cells, indicating that, despite sequence variations, the primary function of this protein appears to be conserved.

During infection, vCXCL1 is expressed as a late protein in the virus life cycle and is released from the infected cells. HCMV
deleted for the UL146 gene (which encodes vCXCL1) exhibits low ability to induce migration of neutrophils. Unlike neutrophils, NK cells still migrated in the absence of the UL146 gene, although the migration was lower than in the presence of UL146. This phenomenon might occur if NK cells are migrating toward other factors and not only vCXCL1 following HCMV infection. The identity of these factors is currently unknown, but we hypothesize that these factors bind other receptors than CXCR1 and CX3CR1, because blocking of these receptors with rhFck and rhIL8 did not abrogate NK cell migration.

The fact that UL146 is a late gene might offer an additional explanation to the apparent contradiction in HCMV inducing NK cell migration: at this time point in the viral life cycle, infectious progeny are being released, making it the perfect timing for attracting and spreading through neutrophils. At the same time, NK cells that are also attracted will only arrive at a late time point in the life cycle, giving the previously mentioned viral immune evasion mechanisms time to act and rendering NK cells inefficient at this late stage.

Taking these results together, we suggest a model (Figure 7G) in which HCMV-infected cells secrete vCXCL1 in order to attract neutrophils. Neutrophils can serve as a “Trojan horse” carrier of HCMV that facilitates the systemic spread of the virus throughout the body (Figure 7G). NK cells are also attracted to the infected cells. However, their migration is slower and less efficient than neutrophils (Figure 7G). Thus, when NK cells finally arrive at the site of infection, the virus is already distributed by neutrophils and the infected cells can counterattack the NK cells due to the numerous immune evasion mechanisms they have developed (Wilkinson et al., 2008). The vCXCL1’s preferential and selective targeting of neutrophils is an elegant example of the highly evolved nature of HCMV subversion mechanisms as it has been developed to walk a very thin line of harnessing the immune cells to its own benefit without self-elimination.

### EXPERIMENTAL PROCEDURES

The experiments in this paper were conducted under the oversight of the Helsinki institutional review board.

#### Ethics Statement
NK cells and neutrophils that were used in this study were obtained from the blood of healthy volunteers. The institutional Helsinki committee of Hadassah approved the study (Helsinki number 0030-12-HMO). All subjects provided a written informed consent.

#### NK and Neutrophil Isolation and Generation of IL-2-Activated Bulk NK Cell Cultures
PBMCs were purified from heparinized blood by centrifugation on Lymphoprep (StemCells Technologies). NK cells were isolated using the EasySep human NK cell enrichment kit (StemCells Technologies). Activated NK lines were generated by culturing isolated NK cells together with irradiated feeder cells (2.5×10⁴ allogeneic PBMCs from two donors and 5×10³ RPMI-8866 cells in each well) and 20 μg/ml PHA (Roche). Both PBMCs and RPMI-8866 cells were irradiated in 6,000 rad prior to seeding in 96-well U-bottom plate. The cultures were maintained in DMEM:F-12 Nutrient Mix (70:30), 10% human serum (Sigma-Aldrich), 2 mM glutamine (Biological Industries [BI]), 1 mM sodium pyruvate (BI), 1× nonessential amino acids (BI), 100 U/ml penicillin (BI), 0.1 mg/ml streptomycin (BI), and 500 U/ml rhIL-2 (PeproTech). NK cells were stained with anti-CD56-phycoerythrin (BD Biosciences) and anti-CD3-allophycocyanin (BioLegend) to confirm NK purity after isolation and following activation. Neutrophils isolation

---

was performed as previously described (Shpacovitch et al., 2011). Neutrophils were stained with anti-CD16-FITC (BioLegend) and anti-CD66a/c/e-PE (BioLegend). See also Supplemental Experimental Procedures.

### Migration Assay and Recombinant Proteins
NK cells (0.5×10⁶; 100 μl) were placed in the upper well of a transwell filter (Corning; diameter, 6.5 mm; pore size, 5 μm; 24-well cells clusters). Filters were then plated in bottom wells containing 600 μl migration medium (RPMI 1640 with 1% fetal calf serum [FCS]) supplemented with either rvCXCL1 (620-CM-025), rhIL8 (208-IL-050), rhFck (365-FR-025), or rhChemerin (2324-CM-025; obtained from R&D Systems), as indicated in each figure. At least three wells were used for each chemokine. After 3 hr of NK cell incubation, or 30 min of neutrophil incubation, at 37°C and 5% CO2, the upper chambers were removed and cells in the bottom chamber were collected and counted using a flow cytometer. For blocking of NK cell migration, the cells were incubated with the indicated recombinant chemokines for 1 hr at 4°C and then loaded into the upper chamber of a transwell plate. Blocking of neutrophil migration was performed for 10 min at 4°C. Competitive migration assays were performed with a mixture of NK cells and neutrophils (0.1×10⁶ each in 100 μl total) at the upper well of a transwell filter. Migration percentage was calculated by dividing the number of the migrating cells by total input cells. Migration fold increase (FI) was calculated by dividing the number of cells migrating in the presence of chemokines by those migrating toward medium only (control). In the blocking experiments, migration percentage was calculated by setting the migration without blocking to 100% and dividing the migration after blocking by this number.

### Internalization Experiments
NK cells were incubated for 1 hr at 37°C, whereas neutrophils were incubated for 10 min at 37°C, with the indicated recombinant chemokines, and then stained with anti-CX3CR1 or anti-CXCR1 mAb.

### Determination of vCXCL1 Concentration in the Supernatant of Infected Cells
Freshly isolated neutrophils were incubated with increasing concentrations of rvCXCL1 for 30 min at 37°C, followed by staining with anti-CXCR2 mAb. MFI of CXCR2 expression without blocking was set on 100%, and the percentages of residual CXCR2 expression were calculated. The estimated levels of vCXCL1 in the infected cell supernatants were calculated following infection of HFF cells with WT HCMV and then incubation of neutrophils with supernatant collected from the infected cells 3 days postinfection. Staining with anti-CXCR2 mAb was followed. Residual CXCR2 expression percentages were calculated, and the levels of vCXCL1 in the supernatant were estimated.

### HCMV Infections
The HCMV virus strains AD169-WT and the ΔUL146 (AD169 background) were used in this study. Both viruses were propagated in HFF cells as previously described (Brumeau et al., 1996). HFF cells were infected at MOI of 1. Supernatant was collected from the infected cells 72 hr postinfection and then used for migration assays. See also Supplemental Experimental Procedures.

### Statistical Analysis
Student’s t test was used to determine statistical significance. p value of less than 0.05 was considered significant. NS, not significant; *p < 0.05; **p < 0.005; or ***p < 0.0005.

### SUPPLEMENTAL INFORMATION
Supplemental Information includes Supplemental Experimental Procedures, one figure, and three tables and can be found with this article online at http://dx.doi.org/10.1016/j.celrep.2016.04.042.

### AUTHOR CONTRIBUTIONS
R.Y. designed and performed the experiments; L.S.M.L. and Y.W. conducted some experiments and provided critical reagents; and A.V. and V.T.K.L.-T. provided critical reagents. D.G.W. provided experimental advice and critical reagents. O.M. supervised the project.
ACKNOWLEDGMENTS

This study was supported by the European Research Council under the European Union’s Seventh Framework Programme (FP/2007-2013)/ERC Grant Agreement number 320473-BacNK. Further support came from the I-CORE Program of the Planning and Budgeting Committee and the Israel Science Foundation and by the I-Core on Chromatin and RNA in Gene Regulation, the GIF foundation, the Lewis family foundation, the ICRF professorship grant, the Israeli Science Foundation, the Helmholtz Israel grant, and the Rosetrees Trust (all to O.M.). This work was also supported by grants from the Israel Science Foundation (grant number 275/13); the Israeli Ministry of Health (grant number 10998); the European Union Seventh Framework Programme 562 FP7/2012-2016 (grant agreement number 316655); and the Samueli Institute Brain Mind & Healing Center, under a contract from the US Army Medical Research and Materiel Command (USAMRAA award no. W81XWH-10-1-0938), within the TATRC portfolio of Integrative Medicine Research. O.M. is a Crown Professor of Molecular Immunology.

Received: September 13, 2015  
Revised: February 25, 2016  
Accepted: April 5, 2016  
Published: May 5, 2016  

REFERENCES

Alcami, A. (2003). Viral mimicry of cytokines, chemokines and their receptors. Nat. Rev. Immunol. 3, 36–50.

Allen, S.J., Crown, S.E., and Handel, T.M. (2007). Chemokine: receptor structure, interactions, and antagonism. Annu. Rev. Immunol. 25, 787–820.

Berahovich, R.D., Lai, N.L., Wei, Z., Lanier, L.L., and Schall, T.J. (2006). Evidence for NK cell subsets based on chemokine receptor expression. J. Immunol. 177, 7833–7840.

Brizić, I., Lenac Roviš, T., Krmpotić, A., and Jonjić, S. (2014). MCMV avoidance of recognition and control by NK cells. Semin. Immunopathol. 36, 641–650.

Brumeanu, T.-D., Bot, A., Bona, C.A., Dehazya, P., Wolf, I., and Zaghouani, H. (1996). Engineering of doubly antigenized immunoglobulins expressing T and B viral epitopes. Immunotechnology 2, 85–95.

Campbell, J.J., Qin, S., Unutmaz, D., Soler, D., Murphy, K.E., Hodge, M.R., Wu, L., and Butcher, E.C. (2001). Unique subpopulations of CD56+ NK and NK-T peripheral blood lymphocytes identified by chemokine receptor expression repertoire. J. Immunol. 166, 6477–6482.

Carrega, P., and Ferlazzo, G. (2012). Natural killer cell distribution and trafficking in human tissues. Front. Immunol. 3, 347.

Cheent, K., and Khakoo, S.I. (2009). Natural killer cells: integrating diversity with function. Immunology 126, 449–457.

Cooper, M.A., Fehniger, T.A., and Caligiuri, M.A. (2001). The biology of human natural killer-cell subsets. Trends Immunol. 22, 633–640.

Cosman, D., Fanger, N., Borges, L., Kubin, M., Chin, W., Peterson, L., and Hsu, M.L. (1997). A novel immunoglobulin superfamily receptor for cellular and viral MHC class I molecules. Immunity 7, 273–282.

Eash, K.J., Means, J.M., White, D.W., and Link, D.C. (2009). CXCR4 is a key regulator of neutrophil release from the bone marrow under basal and stress granulopoiesis conditions. Blood 113, 4711–4719.

Eash, K.J., Greenbaum, A.M., Gopalan, P.K., and Link, D.C. (2010). CXCR2 and CXCR4 antagonistically regulate neutrophil trafficking from murine bone marrow. J. Clin. Invest. 120, 2423–2431.

Fehniger, T.A., Cooper, M.A., Nuovo, G.J., Cella, M., Facchetti, F., Colonna, M., and Caligiuri, M.A. (2003). CD56bright natural killer cells are present in human lymph nodes and are activated by T cell-derived IL-2: a potential new link between adaptive and innate immunity. Blood 101, 3052–3057.

Fielding, C.A., Aicheler, R., Stanton, R.J., Wang, E.C., Han, S., Seirafian, S., Davies, J., McSharry, B.P., Weekes, M.P., Antrobus, P.R., et al. (2014). Two novel human cytomegalovirus NK cell evasion functions target MICA for lysosomal degradation. PLoS Pathog. 10, e1004058.

Goodrum, F., Caviness, K., and Zagallo, P. (2012). Human cytomegalovirus persistence. Cell. Microbiol. 14, 644–655.

Griffiths, P.D. (2012). Burden of disease associated with human cytomegalovirus and prospects for elimination by universal immunisation. Lancet Infect. Dis. 12, 790–798.

Halenius, A., Gerke, C., and Hengel, H. (2015). Classical and non-classical MHC I molecule manipulation by human cytomegalovirus: so many targets—but how many arrows in the quiver? Cell. Mol. Immunol. 12, 139–153.

Hamann, I., Unterwalder, N., Cardona, A.E., Meisel, C., Zipp, F., Ransohoff, R.M., and Infante-Duarte, C. (2011). Analyses of phenotypic and functional characteristics of CX3CR1-expressing natural killer cells. Immunology 133, 62–73.

Hammond, M.E., Lapointe, G.R., Feucht, P.H., Hit, S., Gallegos, C.A., Gordon, C.A., Giedlin, M.A., Mullenbach, G., and Tekamp-Olson, P. (1995). IL-8 induces neutrophil chemotaxis predominantly via type I IL-8 receptors. J. Immunol. 155, 1428–1433.

Hanna, J., Wald, O., Goldman-Wohl, D., Prus, D., Markel, G., Gazit, R., Katz, G., Haimov-Kochman, R., Fuji, N., Yagel, S., et al. (2003). CXCL12 expression by invasive trophoblasts induces the specific migration of CD16- human natural killer cells. Blood 102, 1569–1577.

Heo, J., Dogra, P., Masi, T.J., Pitt, E.A., de Kruijf, P., Smit, M.J., and Sparer, T.E. (2015). Novel human cytomegalovirus viral chemokines, vCXCL-1s, display functional selectivity for neutrophil signaling and function. J. Immunol. 195, 227–236.

Inngjerdingen, M., Damaj, B., and Maghazachi, A.A. (2001). Expression and regulation of chemokine receptors in human natural killer cells. Blood 97, 367–375.

Jackson, S.E., Mason, G.M., and Wills, M.R. (2011). Human cytomegalovirus immunity and immune evasion. Virus Res. 157, 151–160.

Jain, M., Duggal, S., and Chugh, T.D. (2011). Cytomegalovirus infection in non-immunosuppressed critically ill patients. J. Infect. Dev. Ctries. 5, 571–579.

Kolaczkowska, E., and Kubes, P. (2013). Neutrophil recruitment and function in health and inflammation. Nat. Rev. Immunol. 13, 159–175.

Lisnić, B., Lisnić, V.J., and Jonjić, S. (2015). NK cell interplay with cytomegaloviruses. Curr. Opin. Virol. 15, 9–18.

Lütticau, H.R. (2010). The cytomegalovirus UL146 gene product vCXCL1 targets both CXCR1 and CXCR2 as an agonist. J. Biol. Chem. 285, 9137–9146.

McSharry, B.P., Avdic, S., and Slobedman, B. (2012). Human cytomegalovirus encoded homologs of cytokines, chemokines and their receptors: roles in immunomodulation. Viruses 4, 2448–2470.

Morohashi, H., Miyawaki, T., Nomura, H., Kuno, K., Murakami, S., Matsushima, K., and Mukaida, N. (1995). Expression of both types of human interleukin-8 receptors on mature neutrophils, monocytes, and natural killer cells. J. Leukoc. Biol. 57, 180–187.

Neel, N.F., Schutyser, E., Sai, J., Fan, G.H., and Richmond, A. (2005). Chemokine receptor internalization and intracellular trafficking. Cytokine Growth Factor Rev. 16, 637–658.

Penfold, M.E., Dairaghi, D.J., Duke, G.M., Saederup, N., Mocarski, E.S., Kemble, G.W., and Schall, T.J. (1999). Cytomegalovirus encodes a potent alpha chemokine. Proc. Natl. Acad. Sci. USA 96, 9839–9844.

Petering, H., Götzé, O., Kimmig, D., Smolarski, R., Kapp, A., and Elsner, J. (1999). The biologic role of interleukin-8: functional analysis and expression of CXCR1 and CXCR2 on human eosinophils. Blood 93, 694–702.

Prod’homme, V., Griffin, C., Aicheler, R.J., Wang, E.C., McSharry, B.P., Rickards, C.R., Stanton, R.J., Borysiewicz, L.K., López-Botet, M., Wilkinson, G.W., and Tomasec, P. (2007). The human cytomegalovirus MHC class I homolog UL18 inhibits LIR-1+ but activates LIR-1- NK cells. J. Immunol. 178, 4473–4481.

Robertson, M.J. (2002). Role of chemokines in the biology of natural killer cells. J. Leukoc. Biol. 71, 173–183.

Santoni, A., Zingoni, A., Cerboni, C., and Gismondi, A. (2007). Natural killer (NK) cells from killers to regulators: distinct features between peripheral blood and decidual NK cells. Am. J. Reprod. Immunol. 58, 280–288.

Scapini, P., and Cassatella, M.A. (2014). Social networking of human neutrophils within the immune system. Blood 124, 710–719.

Sechler, J.M., Garlic, J., Grivel, J.C., and Murphy, P.M. (2004). IL-15 alters expression and function of the chemokine receptor CX3CR1 in human NK cells. Cell. Immunol. 230, 99–108.

Seidel, E., Le, V.T.K., Bar-On, Y., Tsukerman, P., Enk, J., Yamin, R., Stein, N., Schmiedel, D., Oiknine Djian, E., Weisblum, Y., et al. (2015). Dynamic co-evolution of host and pathogen: HCMV downregulates the prevalent allele MICA*008 to escape elimination by NK cells. Cell Rep. 10, 968–982.

Shpacovitch, V.M., Feld, M., Holzinger, D., Kido, M., Hollenberg, M.D., Levi-Schaffer, F., Vergnolle, N., Ludwig, S., Roth, J., Luger, T., and Steinhoff, M. (2011). Role of proteinase-activated receptor-2 in anti-bacterial and immunomodulatory effects of interferon-γ on human neutrophils and monocytes. Immunology 133, 329–339.

Slavuljica, I., Krmpotić, A., and Jonjić, S. (2011). Manipulation of NKG2D ligands by cytomegaloviruses: impact on innate and adaptive immune response. Front. Immunol. 2, 85.

Slobedman, B., Cao, J.Z., Avdic, S., Webster, B., McAllery, S., Cheung, A.K., Tan, J.C., and Abendroth, A. (2010). Human cytomegalovirus latent infection and associated viral gene expression. Future Microbiol. 5, 883–900.

Stern-Ginossar, N., Elefant, N., Zimmermann, A., Wolf, D.G., Saleh, N., Biton, M., Horwitz, E., Prokocimer, Z., Prichard, M., Hahn, G., et al. (2007). Host immune system gene targeting by a viral miRNA. Science 317, 376–381.

Stern-Ginossar, N., Weisburd, B., Michalski, A., Le, V.T., Hein, M.Y., Huang, S.X., Ma, M., Shen, B., Qian, S.B., Hengel, H., et al. (2012). Decoding human cytomegalovirus. Science 338, 1088–1093.

Vitale, M., Bottino, C., Sivori, S., Sanseverino, L., Castriconi, R., Marcenaro, E., Augugliaro, R., Moretta, L., and Moretta, A. (1998). NKp44, a novel triggering surface molecule specifically expressed by activated natural killer cells, is involved in non-major histocompatibility complex-restricted tumor cell lysis. J. Exp. Med. 187, 2065–2072.

Welte, S.A., Sinzger, C., Lutz, S.Z., Singh-Jasuja, H., Sampaio, K.L., Eknigk, U., Rammensee, H.G., and Steinle, A. (2003). Selective intracellular retention of virally induced NKG2D ligands by the human cytomegalovirus UL16 glycoprotein. Eur. J. Immunol. 33, 194–203.

Wilkinson, G.W., Tomasec, P., Stanton, R.J., Armstrong, M., Prod’homme, V., Aicheler, R., McSharry, B.P., Rickards, C.R., Cochrane, D., Llewellyn-Lacey, S., et al. (2008). Modulation of natural killer cells by human cytomegalovirus. J. Clin. Virol. 41, 206–212.

Yamin, R., Kaynan, N.S., Glasner, A., Vitenshtein, A., Tsukerman, P., Bauman, Y., Ophir, Y., Elias, S., Bar-On, Y., Gur, C., and Mandelboim, O. (2013). The viral KSHV chemokine vMIP-II inhibits the migration of Naive and activated human NK cells by antagonizing two distinct chemokine receptors. PLoS Pathog. 9, e1003568.

Yoneda, O., Imai, T., Goda, S., Inoue, H., Yamauchi, A., Okazaki, T., Imai, H., Yoshie, O., Bloom, E.T., Domae, N., and Umehara, H. (2000). Fractalkine-mediated endothelial cell injury by NK cells. J. Immunol. 164, 4055–4062.

Zheng, Q., Tao, R., Gao, H., Xu, J., Shang, S., and Zhao, N. (2012). HCMV-encoded UL128 enhances TNF-α and IL-6 expression and promotes PBMC proliferation through the MAPK/ERK pathway in vitro. Viral Immunol. 25, 98–105.
