
Developmental Cell

The Autophagy Machinery Controls Cell Death Switching between Apoptosis and Necroptosis

### Graphical Abstract

| MAP3K7 Intact                                                                 | MAP3K7 Loss                                                                 |
|-------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
| **Death Receptor**                                                            | **Death Receptor**                                                            |
| ![Diagram](#)                                                                 | ![Diagram](#)                                                                 |
| DISC                                                                          | Increased Interaction                                                        |
| TRADD                                                                         | MLKL                                                                          |
| RIPK1                                                                         | RIPK1                                                                         |
| TAB2/3                                                                        | RIPK3                                                                         |
| MAP3K7                                                                        | p62                                                                           |
| NF-κB Signaling                                                               | p62 Loss                                                                      |
| MAPK Signaling                                                                | FADD                                                                          |
|                                                                               | RIPK1                                                                         |
|                                                                               | Caspase 8                                                                     |
|                                                                               | ↓                                                                             |
|                                                                               | Apoptosis                                                                     |
|                                                                               | ↓                                                                             |
|                                                                               | Necroptosis                                                                   |

### Authors

Megan L. Goodall, Brent E. Fitzwalter, Shadi Zahedi, ..., Michael Morgan, Scott D. Cramer, Andrew Thorburn

### Correspondence

andrew.thorburn@ucdenver.edu

### In Brief

Goodall et al. show that in the context of Map3k7 loss, the autophagy machinery has a scaffolding role in modulating the mode of cell death between necroptosis and apoptosis. p62/SQSTM1 recruits RIPK1 and mediates necrosome assembly in association with autophagic machinery for TRAIL-induced necroptosis. Blocking this recruitment results in apoptosis.

### Highlights

- MAP3K7 loss sensitizes cells to TRAIL-induced necrototic cell death
- The necrosome associates with autophagy machinery
- Autophagy inhibition promotes or inhibits cell death depending on the blocked step
- Autophagy machinery can serve as a scaffold to modulate mode of programmed cell death

---

Goodall et al., 2016, Developmental Cell 37, 337–349  
May 23, 2016 © 2016 Elsevier Inc.  
http://dx.doi.org/10.1016/j.devcel.2016.04.018

Developmental Cell
Article

# The Autophagy Machinery Controls Cell Death Switching between Apoptosis and Necroptosis

Megan L. Goodall,$^{1}$ Brent E. Fitzwalter,$^{1}$ Shadi Zahedi,$^{2}$ Min Wu,$^{1,3}$ Diego Rodriguez,$^{4}$ Jean M. Mulcahy-Levy,$^{2}$ Douglas R. Green,$^{5}$ Michael Morgan,$^{1}$ Scott D. Cramer,$^{1,5}$ and Andrew Thorburn$^{1,5,*}$

1 Department of Pharmacology  
2 Department of Pediatrics  
University of Colorado Denver, Aurora, CO 80045, USA  
3 Department of Molecular and Cellular Oncology, MD Anderson Cancer Center, Houston, TX 77030, USA  
4 Department of Immunology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA  
5 Co-senior author  
*Correspondence: andrew.thorburn@ucdenver.edu  
http://dx.doi.org/10.1016/j.devcel.2016.04.018

## SUMMARY

Although autophagy controls cell death and survival, underlying mechanisms are poorly understood, and it is unknown whether autophagy affects only whether or not cells die or also controls other aspects of programmed cell death. MAP3K7 is a tumor suppressor gene associated with poor disease-free survival in prostate cancer. Here, we report that Map3k7 deletion in mouse prostate cells sensitizes to cell death by TRAIL (TNF-related apoptosis-inducing ligand). Surprisingly, this death occurs primarily through necroptosis, not apoptosis, due to assembly of the necrosome in association with the autophagy machinery, mediated by p62/SQSTM1 recruitment of RIPK1. The mechanism of cell death switches to apoptosis if p62-dependent recruitment of the necrosome to the autophagy machinery is blocked. These data show that the autophagy machinery can control the mechanism of programmed cell death by serving as a scaffold rather than by degrading cargo.

## INTRODUCTION

Macroautophagy (autophagy) maintains cellular homeostasis by catabolizing organelles and proteins to generate nutrients and macromolecular precursors (Kroemer et al., 2010). Autophagy occurs at a basal level in all cells and is upregulated in response to diverse stressors (e.g., starvation, oxidative stress, drug treatment, ER stress), playing a critical role in development and disease (Green and Levine, 2014; Levine and Kroemer, 2008; Mizushima and Levine, 2010). Autophagy can both promote and inhibit cell death under different cellular contexts, and several mechanistic links between autophagy and apoptosis have been elucidated (Fitzwalter and Thorburn, 2015; Rubinstein and Kimchi, 2012). For example, autophagy promotes apoptosis by Fas Ligand/CD95 because of its ability to degrade a negative regulator of CD95 signaling (Gump et al., 2014) but it can protect against tumor necrosis factor-related apoptosis-inducing ligand (TRAIL)-induced apoptosis by controlling the levels of a pro-apoptotic member of the BCL family (Thorburn et al., 2014). During developmental cell death, similar mechanisms whereby components of the apoptosis machinery are degraded by autophagy have also been identified (Nezis et al., 2010). Very little is known about how autophagy regulates other forms of programmed cell death (Galluzzi et al., 2015), such as necroptosis.

Necroptosis is best understood in response to tumor necrosis factor α (TNF-α) and requires a cytosolic complex, known as the necrosome, which is formed by the serine/threonine receptor interacting protein 3 (RIPK3) in complex with RIPK1, FADD, and caspase-8 (Han et al., 2011; Vandenabeele et al., 2010). Mixed lineage kinase domain-like protein (MLKL) is recruited to the necrosome, and phosphorylated MLKL mediates plasma membrane lysis to induce necroptosis (Cai et al., 2014; Sun et al., 2012; Zhao et al., 2012). TNF-α can also stimulate other secondary complexes to activate nuclear factor κB (NF-κB) or, via the death-inducing signaling complex, promote apoptosis. All of these complexes can involve RIPK1, and the balance of activities within them is believed to control caspase-dependent and -independent cell death (Arslan and Scheidererit, 2011; Fuchs and Steller, 2015). For instance, repression of the necroptotic pathway by apoptotic regulators, such as FADD and caspase-8, is essential for proper mammalian development (Kaiser et al., 2011; Oberst et al., 2011; Zhang et al., 2011). The importance of this balance of different modes of programmed cell death was elegantly shown by the finding that genetic ablation of Ripk1 in mice causes postnatal lethality that is only rescued with loss of both Ripk3 and either caspase-8 or Fadd (Dillon et al., 2014). This is likely due to the fact that RIPK1, which directly regulates caspase-8 activity in some circumstances (Bertrand et al., 2008; Dondelinger et al., 2013; Morgan et al., 2009; Wang et al., 2008), has also been shown to both positively and negatively regulate RIPK3 oligomerization and necroptosis (Dannappel et al., 2014; Orozco et al., 2014).

Necroptosis is associated with inflammatory disease (Linkermann and Green, 2014; Pasparakis and Vandenabeele, 2015) and is important in the response to bacterial and viral infection (Cho et al., 2009). For instance, mice with deletions in Ripk3 or Mlkl are protected from inflammatory pancreatitis (He et al., 2009; Wu et al., 2013). A role for necroptosis in cancer is suggested because expression of RIPK3 is commonly silenced in cancers, making most cancer cells unable to undergo

necroptosis even though they are still capable of activating apoptosis (Koo et al., 2015). This suggests that necroptosis may be specifically selected against during tumor evolution, perhaps because factors that activate adaptive anti-tumor immunity are preferentially released by induction of necroptosis rather than apoptosis of tumor cells (Yatim et al., 2015).

MAP3K7 (also known as transforming growth factor β-activated kinase 1, TAK1) is a serine/threonine protein kinase responsible for activating NF-κB signaling and mitogen-activated protein kinases (MAPK) downstream of death receptors. MAP3K7 is recruited to death receptor complexes through its interaction with RIPK1. Loss of MAP3K7 leads to hypersensitivity to cell death in response to TNF-α (Arslan and Scheidererit, 2011; Dondelinger et al., 2013; Lamothe et al., 2013; Morioka et al., 2014; Vanlangenakker et al., 2011) and TRAIL (Choo et al., 2006; Lluis et al., 2010; Morioka et al., 2009), but the underlying mechanisms are incompletely understood. Interestingly, deletion of the MAP3K7 gene occurs in 30%–40% of prostate tumors (both hemizygous and homozygous deletions are observed). Prostate cancer is the most commonly diagnosed cancer in the United States in men and the second leading cause of male cancer-related deaths, affecting approximately 10% of patients diagnosed (Siegel et al., 2014). Deletion or low mRNA expression of MAP3K7 defines an aggressive subtype of prostate cancer with poor disease-free survival (Liu et al., 2007; Rodrigues et al., 2015; Wu et al., 2012). Thus, finding a way to kill tumor cells that have lost expression of MAP3K7 might improve treatment of a lethal subset of prostate cancer. Consistent with previous reports, we show here that a mouse cell model of Map3k7 null tumors is hypersensitive to cell death in response to TRAIL. Surprisingly however, although TRAIL normally kills by caspase-dependent apoptosis (Johnstone et al., 2008), prostate cells lacking MAP3K7 are capable of switching between apoptosis and necroptosis in response to TRAIL treatment, and we found that TRAIL-induced necroptosis depends on the ability of RIPK1 to interact with p62/SQSTM1 and associate with the autophagosome machinery. These data define a mechanism by which the autophagy machinery controls switching between different types of programmed cell death in a regulated manner by serving as a scaffold for efficient formation and activation of the necrosome.

**RESULTS**

**Loss of Map3k7 Sensitizes to TRAIL-Induced Death**

Loss of MAP3K7 increases sensitivity to TRAIL in human cancer cells (Morioka et al., 2009). To characterize this sensitivity in an isogenic system that avoids complications with human cancer cells, which are often selectively resistant to necroptosis (Koo et al., 2015), we tested matched mouse prostate epithelial cells (MPECs) with either intact floxed or Cre-deleted Map3k7 (Wu et al., 2012) for their response to chemotherapeutics commonly used in prostate cancer (docetaxel, paclitaxel, and carboplatin) in addition to TNF-α and TRAIL (Figure 1A). Loss of Map3k7 increased cell death by TNF-α and TRAIL but not the other drugs. To assess the clinical relevance of TRAIL and TNF-α sensitivity to MAP3K7 loss in human prostate cancers, we analyzed publicly available prostate cancer copy-number alteration datasets (Baca et al., 2013; Cerami et al., 2012; Gao et al., 2013; Taylor et al., 2010). (The results shown here are in part based upon data generated by the Cancer Genome Atlas [TCGA] Research Network: <http://cancergenome.nih.gov/>). MAP3K7 (Figure 1B, red bars) was homozygously co-deleted with TRAIL receptors (TNFRSF10A and B, Figure 1B, blue and green bars, respectively) in 5% of prostate cancers, but not with other death receptors such as TNFRSF1A and B or FAS/CD95 (Figures 1B–1D). Furthermore, homozygous loss of TRAIL receptors occurs at the highest frequency in prostate cancers (Figure S1A). The loss of TRAIL receptors in conjunction with a specific gene, MAP3K7, whose loss selectively sensitizes cells to TRAIL, suggests that prostate tumors where MAP3K7 is lost are under selective pressure to circumvent TRAIL-induced cell death. Therefore, we focused on TRAIL rather than TNF-α for further experiments.

**TRAIL-Induced Cell Death Involves RIPK1-Dependent Necroptosis and Apoptosis**

TRAIL typically kills cells by canonical apoptosis that can be completely prevented by caspase inhibition. However, in Map3k7-null MPECs, the pan-caspase inhibitor Q-VD-OPh (QVD) failed to rescue cell viability after TRAIL treatment (Figures 2A–2C and S2A). TRAIL-induced death was, however, blocked by the RIPK1 kinase inhibitor necrostatin (Figures 2A–2C) or by knockdown of Ripk1 (Figures 2C and S2E). TRAIL-induced necroptosis was suggested in Map3k7 null cells by fast, temporally coincident annexin V and propidium iodide (PI) staining assessed by flow cytometry and live-cell imaging on an IncuCyte with coincident staining starting about 4 hr after TRAIL treatment (Figures 2B, S2B, and S2C), with little caspase 3/7 activity (Figure S2D). In contrast, staurosporine, which induces apoptosis, demonstrated annexin V positivity first (apoptosis), but little dual PI positivity (necroptosis) at early time points (Figures 2B and S2B, gray circles). Consistent with TRAIL-induced death being primarily by necroptosis, QVD did not significantly alter the amount of annexin V and PI co-staining, but, in the presence of necrostatin-1, the few cells that did die were largely annexin V-positive and PI-negative, suggestive of apoptosis (Figure S2B). Knockdown of Ripk3 or Mlkl by short-hairpin RNA (shRNA) as well as the RIPK3 inhibitor GSK′872 were all unable to rescue cell viability on their own; however, each of these necroptosis-specific inhibitions did rescue cell viability when combined with QVD (Figures 2D–2F and S2E). These data indicate that in Map3k7 null cells, TRAIL causes switching between necroptosis and apoptosis, both involving RIPK1. Necroptosis is the predominant mode of death in the absence of intervention, but the mechanism of cell death switches to apoptosis if the necrosome is targeted by removing MLKL or by removing or inhibiting RIPK3.

**Necroptosis Depends on the Autophagy Machinery**

Since we previously found that TRAIL-induced apoptosis is inhibited by autophagy (Thorburn et al., 2014), we tested how autophagy affects TRAIL-induced death in the Map3k7 null cells. Basal and induced autophagic flux was functional in the floxed and null Map3k7 cell lines as shown by immunoblotting for LC3-II accumulation after inhibition of the lysosome and by quantitative flow cytometry (Gump et al., 2014; Gump and Thorburn, 2014) of cells expressing mCherry-GFP-LC3-II (Figure S2F).

A □ Map3k7lox/lox
B
■ Map3k7-/- p=0.01

p=<0.01

N.S.
N.S.
N.S.
N.S.

p=<0.01

18
16
14
12
10
8
6
4
2
0

TCGA MSKCC Broad/Cornell
n=333 (Taylor et al) (Baca et al)
n=216 n=57

MAP3K7Del
TNFRSF10ADel
TNFRSF10BDel
TNFRSF1ADel
TNFRSF1BDel
FASDel

C

| Gene A | Gene B | p-Value | Log Odds Ratio | Association | Significant |
|--------|--------|---------|----------------|-------------|-------------|
| MAP3K7 | TNFRSF10A | <0.001 | 1.447 | Tendency towards co-occurrence | Yes |
| MAP3K7 | TNFRSF10B | <0.001 | 1.447 | Tendency towards co-occurrence | Yes |
| MAP3K7 | TNFRSF1A | 0.137 | 1.486 | Tendency towards co-occurrence | No |
| MAP3K7 | FAS | 0.572 | 0.026 | Tendency towards co-occurrence | No |
| TNFRSF10A | TNFRSF10B | <0.001 | >3 | Tendency towards co-occurrence | Yes |

D

MAP3K7 14%
TNFRSF10A 16%
TNFRSF10B 16%
TNFRSF1A 1%
TNFRSF1B 0%
FAS 8%

Homozygous | No Deletion
Deletion

Figure 1. MAP3K7 Loss Correlates with TRAIL Receptor Loss, but Not Other Death Receptor Loss, in Prostate Cancer

(A) Cell viability of MAP3K7 floxed (white bars) and null (black bars) MPECs treated with chemotherapeutics for 24 hr. Data represent mean ± SD. N.S., not significant.

(B) Frequency of homozygous deletions of MAP3K7, TNFRSF10A, TNFRSF10B, TNFRSF1A, TNFRSF1B, and FAS in three prostate cancer datasets.

(C) Co-occurrence and mutual exclusivity between MAP3K7, TNFRSF10A, TNFRSF10B, TNFRSF1A, TNFRSF1B, and FAS.

(D) Oncoprint of MAP3K7, TNFRSF10A, TNFRSF10B, TNFRSF1A, TNFRSF1B, and FAS (TCGA, n = 333). Each bar represents a patient sample with 150 samples; the additional 183 had no changes to these genes. See also Figure S1.

Inhibition of autophagosome fusion with the lysosome using either chloroquine (CQ) or bafilomycin A1 (Baf) caused more TRAIL-induced death, consistent with a protective effect of basal autophagy (Figure 3A). However, inhibition of early-to-mid stages of autophagy with the phosphoinositide 3-kinase inhibitor wortmannin or knockdown of Atg5, Atg7, or Beclin1 led to increased cell viability (Figures 3A, S3A, and S3B). Rescue in cell viability was not observed with knockdown of Ulk1 or FIP200 (Figure S3B). However, neither of these gene products is essential for basal autophagy, and FIP200 does not change recruitment of components of the autophagy machinery to earlier structures (Itakura and Mizushima, 2011). No rescue in cell viability was observed with knockdown of Rubicon, suggesting that autophagy rather than LC3-associated phagocytosis is responsible for these effects (Martinez et al., 2015) (Figure S3B). These data suggest that it is not the autophagy-mediated degradation of cargo that is important in determining the response to TRAIL treatment, because if this were the case one would expect a similar effect on cell death regardless of which stage of autophagy was inhibited. An alternative hypothesis to explain these results is that the autophagy machinery provides a scaffold for necroptosis signaling. To test this, we looked to see whether

Developmental Cell 37, 337–349, May 23, 2016 339

Figure 2. Necroptotic Cell Death Is Ripk1 Dependent

(A) Cell viability of Map3k7 floxed (white bars) and null (black bars) MPECs treated with TRAIL, QVD, and necrostatin-1 for 24 hr. Data represent mean ± SD.

(B) Map3k7 null MPECs treated with TRAIL, staurosporine, QVD, and necrostatin-1 and imaged in the presence of annexin V and PI on an IncuCyte in four fields of view every 4 hr for 24 hr, with representative images at 4 hr and scale bars representing 30 μm. Co-localization of annexin V and PI was determined using CellProfiler and the Pearson correlation coefficient (r) was determined. See also Figure S2.

(C) Cell viability of Map3k7 null MPECs treated with TRAIL, QVD, and necrostatin-1 for 24 hr with Ripk1 shRNA (black bars). Data represent mean ± SD.

(D) Cell viability of Map3k7 null MPECs treated with TRAIL, QVD, GSK'872, and necrostatin-1 for 24 hr. Data represent mean ± SD.

(E) Cell viability of Map3k7 null MPECs treated with TRAIL, QVD, and necrostatin-1 for 24 hr with Ripk3 shRNA (black bars). Data represent mean ± SD.

(F) Cell viability of Map3k7 null MPECs treated with TRAIL, QVD, and necrostatin-1 for 24 hr with Mkl1 shRNA (black bars). Data represent mean ± SD.
autophagy proteins are associated with necrosome proteins after TRAIL treatment (Figures 3B, S3C, and S3D). As expected, the necrosome proteins RIPK3 and MLKL were co-immunoprecipitated with RIPK1, with more association in the Map3k7 null cells after treatment with TRAIL (Figures 3B, S3C, and S3D). Proteins involved in early-to-mid stages of autophagy including ATG5/ATG12, ATG7, and p62/SQSTM1 also co-immunoprecipitated with RIPK1, while proteins that are important for earlier signaling in the autophagy process (BECN-1 and ULK1) and those involved in later steps when autophagosomes fuse with lysosomes (STX17 and LAMP) did not (Figure 3B). In situ complex formation of the necrosome together with autophagy proteins was demonstrated by proximity ligation assay (PLA) (Figures 3C–3F), which gives a positive signal when antibodies recognizing two proteins of interest are within ~40 nm of one another, strongly suggesting interaction. Knockdown of the target proteins confirmed antibody specificity. We performed PLAs that were quantitated by a blinded observer to test associations between the autophagosome-associated protein LC3 and either RIPK1 or active Ser345-phosphorylated MLKL (Rodriguez et al., 2016). Both LC3-RIPK1 and LC3-phospho-MLKL interactions were detected and both were increased by TRAIL treatment. We expanded this approach to develop a dual PLA (Figures 3E, 3F, S3E–S3G, and S7) to simultaneously test interactions for both ATG5-RIPK1 and ATG5-MLKL in the same cells using an antibody pair targeting ATG5 and RIPK1 and a separate antibody pair targeting ATG5 and MLKL (Figure S3E). Dual PLAs were performed with the initial PLA (interaction of ATG5 and RIPK1) in red, and the second PLA (interaction of ATG5 and MLKL) in green. Both PLAs were more robust in Map3k7 null cells than in floxed cells (Figure S3G), indicating increased basal interactions in the null cells. Consistent with the co-immunoprecipitations, PLA signals were increased upon treatment with TRAIL (Figures 3C–3F and S3). Necrostatin-1 treatment abrogated association of RIPK1 and MLKL with ATG5 (Figures 3E and 3F), indicating a requirement for RIPK1 kinase activity for complex formation. The PLA experiments therefore show that necrosome proteins associate with the autophagy machinery and that this is potentiated by TRAIL, leading to activation of MLKL in association with autophagy proteins. To test whether these complexes were forming on an autophagosomal membrane, we performed immunogold transmission electron microscopy (TEM) with antibodies targeted to ATG5 and RIPK1 (Figures 3G, 3H, and S3H), whereby a small gold particle (5 nm) indicates ATG5 protein and a large gold particle (15 nm) indicates RIPK1 protein. Small and large particles were observed close together; these complexes and the number of immunogold signals in the complexes increased upon stimulation with TRAIL and were found close to double-membrane structures, consistent with complex formation on autophagosomal structures.

### Efficient Necrosome Activation, but Not Formation, Depends on ATG5

To test whether autophagy proteins are required for efficient necrosome formation and activation, we pulled down RIPK1 and determined its association with MLKL and p62 after knockdown of Atg5 (Figure 4A). No significant change in RIPK1 association with p62 was observed with knockdown of Atg5, but a subtle reduction in RIPK1 association with MLKL was observed. To test for interaction in situ, we performed PLAs using both MLKL and phospho-MLKL with p62 after Atg5 knockdown (Figures 4B, 4C, and S4). An increase in signal was observed in both PLAs with MLKL and phospho-MLKL upon TRAIL treatment, indicating an increase in both complex formation and activation. Knockdown of Atg5 slightly reduced the signal in the PLA-detecting interaction between p62 and total MLKL but completely eliminated the TRAIL-induced p62-phospho-MLKL PLA signal. An immunoblot confirmed loss of phospho-MLKL in TRAIL-treated cells with Atg5 knockdown (Figure S4). These data suggest that ATG5 is more important for TRAIL-induced necrosome activation than it is for necrosome formation in the Map3k7 null cells.

### Binding of RIPK1 to p62 and Subsequent Recruitment to Autophagy Machinery Determines Whether Cells Die by Necroptosis or Apoptosis

RIPK1 interacts directly with p62 (Sanz et al., 1999), suggesting a mechanism by which necrosome components could be recruited to the autophagosome. To test this, we immunoprecipitated RIPK1 after knockdown of p62/Sqstm1 and found a significant reduction in ATG5 association with RIPK1 and MLKL (Figure 5A). To test whether these interactions were disrupted in situ, we performed dual PLA. This showed a decrease in association of ATG5 with RIPK1 (red) and MLKL (green) following p62/Sqstm1 knockdown, even in the presence of TRAIL (Figures 5B and 5C). Immunogold TEM following p62/Sqstm1 knockdown showed a reduction in ATG5-RIPK1 complex formation, size, and localization to membranes (Figures 5D, 5E, and S5). Surprisingly however, knockdown of p62/Sqstm1 did not significantly increase cell viability upon TRAIL treatment. Instead, cell death was now completely rescued by QVD (Figure 6A), suggesting that removal of p62/SQSTM1 caused the cells to be only able to activate apoptosis and not necroptosis. Switching of the cell death mechanism from necroptosis to apoptosis upon p62/Sqstm1 knockdown was confirmed by temporal analysis of annexin V and PI staining. Upon p62/Sqstm1 knockdown, a decrease in PI staining and its co-localization with annexin V was observed with QVD treatment in TRAIL-treated cells (Figures 6B and S6). These data lead to two conclusions. First, p62/SQSTM1 is needed for RIPK1 to localize to the autophagy machinery and allow for efficient assembly and activation of the necrosome. Second, it is this recruitment that causes the TRAIL-treated cells to die by necroptosis rather than apoptosis. To test this, we tested for rescue of necroptosis by mutant p62 molecules lacking the ZZ domain required for RIPK1 interaction, or lacking the LIR domain required for LC3 interaction but not required for recruitment of p62 to autophagosomal structures (Itakura and Mizushima, 2011). An shRNA targeting the 3′ UTR of p62/Sqstm1 reduced endogenous p62, and constructs expressing wild-type p62, the ΔZZ mutant, or the ΔLIR mutant were added back (Figure 6C). Removal of p62 allowed for rescue by QVD, but addition of the wild-type p62 or ΔLIR mutant did not. The ΔZZ p62 construct mimicked complete p62 loss in that QVD alone rescued cell viability. These results show that it is p62-mediated recruitment of RIPK1 to the autophagy machinery that results in activation of necroptosis rather than apoptosis upon TRAIL treatment.
A

|  | NT sh |
| --- | --- |
| ATG7 |  |
| ATG5 |  |
| β-ACTIN |  |

![Image](https://i.imgur.com/placeholder.png)

B

| IP: RIPK1 | Input |
| --- | --- |
| TRAIL: - + |  |
| Necrosome |  |
| RIPK1 |  |
| RIPK3 |  |
| MLKL |  |
| Autophagosome |  |
| Present |  |
| ATG7 |  |
| ATG5/12 |  |
| p62 |  |
| ULK1 |  |
| BECN1 |  |
| STX17 |  |
| LAMP2 |  |
| Absent |  |

C

| Control | TRAIL |
| --- | --- |
| PLA: RIPK1:LC3 |  |
| PLA: pMLKL:LC3 |  |

D

| RIPK1:LC3 PLA | p=0.05 |
| --- | --- |
| Number of Puncta/Cell |  |
| pMLKL:LC3 PLA | p=<0.05 |
| Number of Puncta/Cell |  |

E

| Control | TRAIL |
| --- | --- |
| Double PLA: ATG5/RIPK1 (red): ATG5/MLKL (green) |  |

F

|  |  |
| --- | --- |
| ATG5:RIPK1 PLA |  |
| ATG5:MLKL PLA |  |
| Number of Puncta/Cell |  |
| p=<0.05 |  |
| p=0.07 |  |
| p=<0.01 |  |
| Control | TRAIL + Necrostatin |

G

| Control | TRAIL |
| --- | --- |
| Immuno-gold TEM |  |

H

|  |  |
| --- | --- |
| Control |  |
| 10 ng/mL TRAIL |  |
| Average ATG5:RIPK1 Complexes/Image |  |
| p=<0.001 |  |
| p=<0.001 |  |
| p=<0.001 |  |
| Total | Near | Large |
| Complexes Membrane Complex |  |

(legend on next page)

Figure 4. Atg5 Is Required for Necrosome Activation

(A) Immunoblots of ATG5, p62, and MLKL after a co-immunoprecipitation of RIPK1 in Map3k7 null MPECs with knockdown of Atg5, with and without TRAIL treatment.

(B) PLA of p62 with MLKL and pMLKL in Map3k7 null MPECs treated with and without TRAIL with knockdown of Atg5. Scale bars represent 20 μm. See also Figure S4.

(C) Quantification of (B). Data represent mean ± SD. N.S., not significant.

DISCUSSION

Regulation of cell death is critical in development and disease. However, while it has been recognized for many years that the decision whether or not a cell should die has critical biological consequences, until recently we have had a much poorer understanding of the importance of how a cell dies and how this is controlled. For instance, in mice the loss of cell death

Figure 3. Autophagy Machinery Localizes with Necrosome Proteins

(A) Cell viability of Map3k7 null MPECs treated with TRAIL, necrostatin-1, CQ, BafA1, or wortmannin for 24 hr or with Atg5 and Atg7 shRNA. Immunoblots show knockdown of Atg5 and Atg7 with β-actin as loading control. Data represent mean ± SD.

(B) Immunoblots of necrosome proteins and autophagy proteins, ATG7, ATG5/12, p62, BECLIN-1, ULK1, STX17, and LAMP2, after a co-immunoprecipitation of RIPK1 in Map3k7 null MPECs with and without TRAIL treatment.

(C) Proximity ligation assay (PLA) of p62 with MLKL and phospho-MLKL in Map3k7 null MPECs treated with and without TRAIL. Scale bars represent 20 μm.

(D) Quantification of (C). Data represent mean ± SD.

(E) Dual PLA of ATG5 with RIPK1 (red) and ATG5 with MLKL (green) in Map3k7 null MPECs treated with TRAIL or necrostatin. Scale bars represent 20 μm.

(F) Quantification of (D). Data represent mean ± SD.

(G) Immunogold TEM in Map3k7 null MPECs treated with and without TRAIL. Large gold particles indicated by black arrows recognize antibodies to RIPK1, and small gold particles indicated by white arrows recognize antibodies to ATG5. Scale bars represent 100 nm.

(H) Quantification of (G). Data represent mean ± SD.

See also Figure S3.

Figure 5. p62 Is Required for Ripk1 Association with Autophagosome

(A) Immunoblots of ATG5, p62, and MLKL after a co-immunoprecipitation of RIPK1 in Map3k7 null MPECs with knockdown of p62/Sqstm1, with and without TRAIL treatment.

(B) Dual PLA of ATG5 with RIPK1 (red) and ATG5 with MLKL (green) in Map3k7 null MPECs with knockdown of p62 treated with and without TRAIL. Scale bars represent 20 μm.

(C) Quantification of (B). Data represent mean ± SD.

(D) Immunogold TEM in Map3k7 null MPECs with p62/Sqstm1 shRNA treated with and without TRAIL. Large gold particles indicated by black arrows recognize antibodies to RIPK1, and small gold particles indicated by white arrows recognize antibodies to ATG5. Scale bars represent 100 nm.

(E) Quantification of (D). Data represent mean ± SD. N.S., not significant.

See also Figure S5.

Figure 6. p62 Regulates the Switch between Apoptotic and Necroptotic Cell Death

(A) Cell viability of Map3k7 null MPECs treated with TRAIL, QVD, and necrostatin-1 with knockdown of p62/Sqstm1 (white bars). Data represent mean ± SD.

(B) Map3k7 null MPECs treated with TRAIL, QVD, and necrostatin-1 with p62/Sqstm1 knockdown and imaged in the presence of annexin V and PI on an IncuCyte in four fields of view every 4 hr for 24 hr. Representative images at 4-hr time point with scale bars representing 100 μm. Co-localization of annexin V and PI was determined using CellProfiler and the Pearson correlation coefficient (r) was determined. See also Figure S6.

(C) Cell viability of Map3k7 null MPECs treated with TRAIL, QVD, and necrostatin-1 with knockdown of p62/Sqstm1 and addition of either wild-type (WT) p62, mutant p62 lacking the ZZ domain (ΔZZ), or mutant p62 of the LIR domain (ΔLIR). Data represent mean ± SD. N.S., not significant.
and proteins such as LC3 that are associated with more mature autophagosome structures) this complex is retained as the autophagosome matures, allowing increased necrosome activation. Complexes we observed under electron microscopy often appeared on quite mature structures, consistent with this idea.

Other anti-cancer agents may also activate necroptosis involving autophagy ([Basit et al., 2013](https://doi.org/10.1038/onc.2013.45); [Kharaziha et al., 2015](https://doi.org/10.1038/onc.2015.17)). Importantly, however, the mechanism here applies under specific circumstances that suggest a potential for applying this knowledge in cancer treatment in cells that have lost a specific tumor suppressor and in response to a death stimulus, TRAIL, which can be used clinically ([Lemke et al., 2014](https://doi.org/10.1038/nm.3698)) and is itself an important endogenous tumor and metastasis suppressor ([Johnstone et al., 2008](https://doi.org/10.1038/onc.2008.21)). Our data indicate that MAP3K7 loss sensitizes the cells to undergo necroptosis because the components of the necrosome (RIPK1, RIPK3, and MLKL) associate together even in the absence of exogenous signaling; however, TRAIL receptor signaling and association with the autophagy machinery is needed for the necrosome to be efficiently activated, leading to phosphorylation of MLKL. MAP3K7 loss defines an aggressive subtype of prostate cancer, and sensitivity to TRAIL-induced cell death by necroptosis that could be further regulated using clinically available autophagy inhibitors such as chloroquine may expedite a new strategy for treating such tumors. Figure 1 suggests that about half of patients whose tumors will have homozygous loss of the MAP3K7 gene retain TRAIL receptor expression and could therefore potentially benefit from such an approach. With over 200,000 new prostate cancer cases a year in the United States, this may amount to more than 10,000 of such patients. In addition, by activating necroptosis this approach might better activate an anti-tumor immune response ([Yatim et al., 2015](https://doi.org/10.1038/onc.2015.21)). Because these effects are mediated by the autophagy machinery serving as a scaffold rather than by degrading cellular material, inhibition of autophagy before or after formation of autophagosomes can have opposing effects, showing that it is important not only if the autophagy pathway is inhibited but also where the pathway is inhibited. With autophagy inhibitors already in clinical trials for cancer treatment ([Rebecca and Amaravadi, 2015](https://doi.org/10.1038/onc.2015.21)) and suggestions of benefit in some patients ([Levy et al., 2014](https://doi.org/10.1038/onc.2014.21)), it is increasingly important to understand how autophagy modulates both the amount and type of tumor cell death, especially as new inhibitors targeting different steps in the autophagy pathway ([Egan et al., 2015](https://doi.org/10.1038/onc.2015.21); [Goodall et al., 2014](https://doi.org/10.1038/onc.2014.21); [McAfee et al., 2012](https://doi.org/10.1038/onc.2012.21)) could have opposing effects on these mechanisms.

### EXPERIMENTAL PROCEDURES

#### Reagents and Cell Lines
MPECs with floxed and Cre-deleted Map3k7 ([Wu et al., 2012](https://doi.org/10.1038/onc.2012.21)) were maintained and cultured in DMEM/F12 (Gibco 11320) with fraction 5 BSA (Sigma), cholera toxin (List Biologicals 101B), bovine pituitary extract (Hammond Cell Tech 1078-NZ), gentamicin, insulin (Sigma), vitamin E, transferrin, trace elements, EGF (Collaborative Research), and 1% fetal bovine serum at 37°C with 5% CO₂.

#### Dataset Analyses
Array comparative genomic hybridization data and clinical data for the MSKCC ([Taylor et al., 2010](https://doi.org/10.1038/onc.2010.21)), TCGA ([http://cancergenome.nih.gov/](http://cancergenome.nih.gov/)), and Broad/Core-nell-2 ([Baca et al., 2013](https://doi.org/10.1038/onc.2013.21)) prostate cancer datasets were downloaded from the cBioPortal for Cancer Genomics ([Cerami et al., 2012](https://doi.org/10.1038/onc.2012.21); [Gao et al., 2013](https://doi.org/10.1038/onc.2013.21)). Data were processed as described in the source publications.

#### Cell Viability Assays
Cells were seeded at 2,000 cells per well in 96-well plates (Corning) and incubated overnight, or 500 cells per well with shRNA for 48 hr for knockdown experiments. Cells were treated in a dose response to TRAIL (R&D Systems 375-TEC), 20 μM QVD (R&D Systems OPH00101), 30 μM necrostatin-1 (Enzo Life Sciences BML-AP309-0020), 40 μM CQ (Sigma C6628), 100 nM bafilomycin A1 (Sigma B1793), 100 nM wortmannin (Sigma W1628), or 0.5 nM staurosporine (Sigma S5921) for 24 hr. All experiments were performed at least three times in triplicate, and the proportion of cells per treatment group was normalized to control wells. Lactate dehydrogenase (LDH) release was quantitated using the Cytoscan-LDH Cytotoxicity Assay Kit (G-Biosciences 786-210) according to manufacturer’s instructions, and absorbance was read at 490 nm using a Benchmark Plus microplate spectrophotometer (Bio-Rad). Viable cells were measured using the Cell Titer-Glo luminescent cell viability assay (Promega G7572) following the manufacturer’s protocol, and luminescence was read using a Modulus Microplate reader.

#### Immunoblotting
Cells were seeded at 150,000 cells per well and treated with 1 ng/ml and 100 ng/ml TRAIL, 20 μM QVD, 30 μM necrostatin-1, 10 nM Rad001, or 40 μM CQ for 24 hr or starved with Earle’s Balanced Salt Solution (EBSS) (Sigma E2888) for 4 hr prior to collection. Cell lysates were harvested after treatments and time points indicated using RIPA buffer (4 M NaCl, 0.01% Nonidet P-40, 2% sodium deoxycholate, 10% SDS, 1 M Tris, 500 mM NaF, 0.5 M EDTA, dH₂O) with phosphatase inhibitors (Roche 04693116001). Membranes were blocked in 5% BSA or 5% milk in 1 × Tris-buffered saline with Tween 20, depending on antibody, and probed with primary antibodies at manufacturers’ recommended concentrations. Anti-β-actin (Sigma A5541) was used as the protein loading control. For a list of antibodies, see [Supplemental Experimental Procedures](https://doi.org/10.1038/onc.2016.21).

#### shRNA Infections
A pLK0.1 vector system was utilized for RNAi of necrosome- and autophagy-related proteins and were acquired from the Functional Genomics Facility at the University of Colorado Cancer Center. Lentiviruses were prepared according to protocols published at the RNAi Consortium web page ([http://www.broadinstitute.org/rnai/trc/lib](http://www.broadinstitute.org/rnai/trc/lib)). Cells were plated at 20,000 cells per well in six-well plates and transduced with lentivirus using 8 μg/ml polybrene. After 48–72 hr, cells were plated for assays following cell counts listed. In 96-well assays, knockdown was done in the plate at 500 cells per well for 48 hr. RNAi Consortium numbers for shRNAs are listed in [Supplemental Experimental Procedures](https://doi.org/10.1038/onc.2016.21).

#### Co-immunoprecipitations
MPEC cells were cultured at 1 × 10⁶ in 15-cm plates with and without shRNA for 72 hr at 37°C with 5% CO₂. A higher concentration of TRAIL was used for shorter time points to increase the speed of necrosome formation. Cells were treated with 500 ng/ml TRAIL for 2 hr and lysates harvested in immunoprecipitation (IP) buffer (20 mM Tris-HCl [pH 7.5], 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1% NP-40, 1% sodium deoxycholate, and 1 mM β-glycerophosphate) with phosphatase inhibitors. Lysates were spiked with 8 μg of RIPK1 antibody (BD Transduction Laboratories 610459) and incubated overnight at 4°C. Protein A/G agarose beads (Santa Cruz Biotechnology sc-2003) were added and incubated for 2 hr and washed with IP buffer at least three times. Samples were run on a polyacrylamide gel and immunoblotted.

#### Proximity Ligation Assays
Cells were seeded at 10,000 cells per well in an 8-well chamber coverslide (Millipore PEZGS0816) and incubated overnight at 37°C in 5% CO₂. Cells were treated with TRAIL for 2 hr and fixed with 3.7% formaldehyde. The PLA pair of monoclonal rabbit ATG5 (Abcam ab108327) and mouse RIPK1 (BD Transduction 610459), PLA pair of polyclonal ATG5 (Novus NB110-53818) and MLKL (Millipore MABC604), PLA pair p62/Sqstm1 rabbit (Abgent AP21183B) and rat MLKL, or mouse phospho-Ser345-MLKL ([Rodriguez et al., 2016](https://doi.org/10.1038/onc.2016.21)), and LC3 (Novus Biologicals NB100-2220) were incubated overnight at optimized dilutions based on immunofluorescence, and the PLA was performed according to the manufacturer’s directions (Sigma)

346 Developmental Cell 37, 337–349, May 23, 2016DUO92008, DUO92002, DUO92004) and signals quantitated by a blinded observer. For the dual PLA, see [Supplemental Experimental Procedures](http://dx.doi.org/10.1016/j.devcel.2016.04.018).

### Flow Cytometry

MPEC cells (both floxed and Cre-deleted) constitutively expressing mCherry-GFP-LC3 were seeded at \(2 \times 10^5\) in a six-well plate and incubated overnight. Cells were exposed to control medium, Rad001, or BafA1 (for baseline inhibited autophagy) for 24 hr or EBSS starvation medium for 4 hr prior to collection. Flow data were acquired on a Gallios561 and analyzed using Summit v5.1 (Beckman Coulter). Autophagic flux was determined by the ratio mCherry/GFP. For annexin V/PI, caspase 3/7, or reactive oxygen species as-says, see [Supplemental Experimental Procedures](http://dx.doi.org/10.1016/j.devcel.2016.04.018).

### IncuCyte Cell Death Assay

MPEC cells were seeded at 500 cells per well in a 96-well plate (Costar). Cells were cultured at \(37^\circ\) and \(5\% \mathrm{CO}_2\) and monitored using an IncuCyte Zoom (Essen BioScience). Cells were exposed to TRAIL, QVD, or necrostatin-1 for 24 hr in the presence of annexin V and PI, and images were captured at 4-hr intervals from four separate regions per well using a \(10 \times\) objective. Each experiment was done in triplicate and co-localization was determined by exported green and red masks in CellProfiler (Broad Institute, [www.cellprofiler.org](http://www.cellprofiler.org)).

### Immunogold Transmission Electron Microscopy

MPEC cells were seeded in 10-cm plates and incubated overnight at \(37^\circ \mathrm{C}\) in \(5 \% \mathrm{CO}_2\) and treated as described, then trypsinized, washed, pelleted, and re-suspended in \(0.5 \%\) glutaraldehyde and \(4 \%\) formaldehyde fixatives in \(0.1 \mathrm{M}\) cacodylate buffer. Cell pellets were embedded in \(2 \%\) agarose, post-fixed in osmium tetroxide, and dehydrated with an acetone series. Cell samples were infiltrated and embedded in LR White resin and polymerized at \(60^\circ \mathrm{C}\) for 24 hr. Ultrathin sections of \(70 \mathrm{~nm}\) were generated and placed on nickel-coated grids, then incubated for 1 hr in \(5 \% \mathrm{BSA}+5 \%\) goat serum in \(1 \times\) PBS, washed, and incubated in primary ATG5 (rabbit) and RIPK1 (mouse) antibodies (listed above) overnight at \(4^\circ \mathrm{C}\). Grids were washed and incubated in secondary goat-anti-rabbit \(6-\mathrm{nm}\) gold antibody (Electron Microscopy Sciences, 25103) and goat-anti-mouse \(15-\mathrm{nm}\) gold antibody (Electron Microscopy Sciences 25132) for 1 hr. Sections were examined using a transmission electron microscope at the Electron Microscopy Center, University of Colorado Denver. Autophagosomal structures were identified as double-membrane structures. Images were quantified using Nikon NIS Elements software and thresholded excluding anything outside of \(0.6-1\) circularity and larger than \(20 \mathrm{~nm}\) in size. Regions of interest (ROIs) were drawn around anything containing two or more gold particles for quantification. Accounting for gold particle SDs, small gold particles (ATG5) were considered anything from 4 to \(8 \mathrm{~nm}\) and large gold particles (RIPK1) anything from 10 to \(18 \mathrm{~nm}\). Large complexes were defined as containing more than three particles, with at least one of each size. Complexes near a membrane were considered those within the ROI centered around the membrane.

### p62 Constructs

Human p62 cDNA was inserted into the pEGFP-C1 vector from Clontech, and site-directed mutagenesis was performed to delete the ZZ domain of p62. Primers used were \(5^{\prime}-\mathrm{GCACCCCAATGTGATCCACCAA}\) and \(5^{\prime}-\mathrm{GAATGC}\) GAGCTTGGTGATCAC. Mutations to the LIR domain were D335/336/337A based on Itakura and Mizushima (2011). Primers used were \(5^{\prime}-\mathrm{GACA}\) GATGGGTCCAGGCATCAGCTCCTCCTGAACAGT, \(5^{\prime}-\mathrm{ACTGTTCAGGAGGA}\) GCTGATGCCCTGGACCCATCTGTC, \(5^{\prime}-\mathrm{GATGGGTCCAGGCAGCAGCTCCTC}\) CTGAA, and \(5^{\prime}-\mathrm{TTCAGGAGGAGCTGCTGCCCTGGACCCATC}\).

### SUPPLEMENTAL INFORMATION

Supplemental Information includes Supplemental Experimental Procedures and seven figures and can be found with this article online at [http://dx.doi.org/10.1016/j.devcel.2016.04.018](http://dx.doi.org/10.1016/j.devcel.2016.04.018).

### AUTHOR CONTRIBUTIONS

M.L.G., S.C., and A.T. conceived the project. M.L.G. designed and executed most of the experiments and analyzed data. A.T., M.M., and S.C. contributed

to experimental design. B.F., S.Z., M.W., and J.M.L. performed some experiments and helped with analysis. D.R. and D.G. provided essential reagents. M.L.G. and A.T. wrote the paper with contributions from all of the authors.

### ACKNOWLEDGMENTS

We thank Jenny Mae Samson for help with some experiments and Alicia Pastor from the Electron Microscopy core at Michigan State University for help with Immunogold TEM. Supported by NIH grants RO1CA150925, RO1CA190170, RO1CA199741, R21CA187354, and 1F32CA196080, and Shared Resources supported by P30CA046934.

Received: October 24, 2015  
Revised: March 29, 2016  
Accepted: April 25, 2016  
Published: May 23, 2016  

### REFERENCES

Arslan, S.C., and Scheidererit, C. (2011). The prevalence of TNFalpha-induced necrosis over apoptosis is determined by TAK1-RIP1 interplay. PLoS One 6, e26069.

Baca, S.C., Prandi, D., Lawrence, M.S., Mosquera, J.M., Romanel, A., Drier, Y., Park, K., Kitabayashi, N., MacDonald, T.Y., Ghandi, M., et al. (2013). Punctuated evolution of prostate cancer genomes. Cell 153, 666–677.

Basit, F., Cristofanon, S., and Fulda, S. (2013). Obatoclax (GX15-070) triggers necroptosis by promoting the assembly of the necrosome on autophagosomal membranes. Cell Death Differ. 20, 1161–1173.

Bertrand, M.J., Milutinovic, S., Dickson, K.M., Ho, W.C., Boudreault, A., Durkin, J., Gillard, J.W., Jaquith, J.B., Morris, S.J., and Barker, P.A. (2008). cIAP1 and cIAP2 facilitate cancer cell survival by functioning as E3 ligases that promote RIP1 ubiquitination. Mol. Cell 30, 689–700.

Gai, Z., Jitkaew, S., Zhao, J., Chiang, H.C., Choksi, S., Liu, J., Ward, Y., Wu, L.G., and Liu, Z.G. (2014). Plasma membrane translocation of trimerized MLKL protein is required for TNF-induced necroptosis. Nat. Cell Biol. 16, 55–65.

Cerami, E., Gao, J., Dogrusoz, U., Gross, B.E., Sumer, S.O., Aksoy, B.A., Jacobsen, A., Byrne, C.J., Heuer, M.L., Larsson, E., et al. (2012). The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov. 2, 401–404.

Cho, Y.S., Challa, S., Moquin, D., Genga, R., Ray, T.D., Guildford, M., and Chan, F.K. (2009). Phosphorylation-driven assembly of the RIP1-RIP3 complex regulates programmed necrosis and virus-induced inflammation. Cell 137, 1112–1123.

Choo, M.K., Kawasaki, N., Singhirunnusorn, P., Koizumi, K., Sato, S., Akira, S., Saiki, I., and Sakurai, H. (2006). Blockade of transforming growth factor-beta-activated kinase 1 activity enhances TRAIL-induced apoptosis through activation of a caspase cascade. Mol. Cancer Ther. 5, 2970–2976.

Dannappel, M., Vlantis, K., Kumari, S., Polykratis, A., Kim, C., Wachsmuth, L., Eftychi, C., Lin, J., Corona, T., Hermance, N., et al. (2014). RIPK1 maintains epithelial homeostasis by inhibiting apoptosis and necroptosis. Nature 513, 90–94.

Dillon, C.P., Weinlich, R., Rodriguez, D.A., Cripps, J.G., Quarato, G., Gurung, P., Verbist, K.C., Brewer, T.L., Llambi, F., Gong, Y.N., et al. (2014). RIPK1 blocks early postnatal lethality mediated by caspase-8 and RIPK3. Cell 157, 1189–1202.

Dondelinger, Y., Aguileta, M.A., Goossens, V., Dubuisson, C., Grootjans, S., Dejardin, E., Vandebabeele, P., and Bertrand, M.J. (2013). RIPK3 contributes to TNFR1-mediated RIPK1 kinase-dependent apoptosis in conditions of cIAP1/2 depletion or TAK1 kinase inhibition. Cell Death Differ. 20, 1381–1392.

Egan, D.F., Chun, M.G., Vamos, M., Zou, H., Rong, J., Miller, C.J., Lou, H.J., Raveendra-Panickar, D., Yang, C.C., Sheffler, D.J., et al. (2015). Small molecule inhibition of the autophagy kinase ULK1 and identification of ULK1 substrates. Mol. Cell 59, 285–297.
Fitzwalter, B.E., and Thorburn, A. (2015). Recent insights into cell death and autophagy. FEBS J. 282, 4279–4288.

Fuchs, Y., and Steller, H. (2015). Live to die another way: modes of programmed cell death and the signals emanating from dying cells. Nat. Rev. Mol. Cell Biol. 16, 329–344.

Galluzzi, L., Bravo-San Pedro, J.M., Vitale, I., Aaronson, S.A., Abrams, J.M., Adam, D., Alnemri, E.S., Altucci, L., Andrews, D., Annicchiarico-Petruzzelli, M., et al. (2015). Essential versus accessory aspects of cell death: recommendations of the NCCD 2015. Cell Death Differ. 22, 58–73.

Gao, J., Aksoy, B.A., Dogrusoz, U., Dresdner, G., Gross, B., Sumer, S.O., Sun, Y., Jacobsen, A., Sinha, R., Larsson, E., et al. (2013). Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Sci. Signal. 6, pl1.

Goodall, M.L., Wang, T., Martin, K.R., Kortus, M.G., Kauffman, A.L., Trent, J.M., Gately, S., and MacKeigan, J.P. (2014). Development of potent autophagy inhibitors that sensitize oncogenic BRAF V600E mutant melanoma tumor cells to vemurafenib. Autophagy 10, 1120–1136.

Green, D.R., and Levine, B. (2014). To be or not to be? How selective autophagy and cell death govern cell fate. Cell 157, 65–75.

Gump, J.M., and Thorburn, A. (2014). Sorting cells for basal and induced autophagic flux by quantitative ratiometric flow cytometry. Autophagy 10, 1327–1334.

Gump, J.M., Staskiewicz, L., Morgan, M.J., Bamberg, A., Riches, D.W., and Thorburn, A. (2014). Autophagy variation within a cell population determines cell fate through selective degradation of Fap-1. Nat. Cell Biol. 16, 47–54.

Han, J., Zhong, C.Q., and Zhang, D.W. (2011). Programmed necrosis: backup to and competitor with apoptosis in the immune system. Nat. Immunol. 12, 1143–1149.

He, S., Wang, L., Miao, L., Wang, T., Du, F., Zhao, L., and Wang, X. (2009). Receptor interacting protein kinase-3 determines cellular necrotic response to TNF-alpha. Cell 137, 1100–1111.

Itakura, E., and Mizushima, N. (2011). p62 Targeting to the autophagosome formation site requires self-oligomerization but not LC3 binding. J. Cell Biol. 192, 17–27.

Johnstone, R.W., Frew, A.J., and Smyth, M.J. (2008). The TRAIL apoptotic pathway in cancer onset, progression and therapy. Nat. Rev. Cancer 8, 782–798.

Kaiser, W.J., Upton, J.W., Long, A.B., Livingston-Rosanoff, D., Daley-Bauer, L.P., Hakem, R., Caspary, T., and Mocarski, E.S. (2011). RIP3 mediates the embryonic lethality of caspase-8-deficient mice. Nature 471, 368–372.

Kharaziha, P., Chioureas, D., Baltatzis, G., Fonseca, P., Rodriguez, P., Gogvadze, V., Lennartsson, L., Bjorklund, A.C., Zhivotovsky, B., Grander, D., et al. (2015). Sorafenib-induced defective autophagy promotes cell death by necroptosis. Oncotarget 6, 37066–37082.

Koo, G.B., Morgan, M.J., Lee, D.G., Kim, W.J., Yoon, J.H., Koo, J.S., Kim, S.I., Kim, S.J., Son, M.K., Hong, S.S., et al. (2015). Methylation-dependent loss of RIP3 expression in cancer represses programmed necrosis in response to chemotherapeutics. Cell Res. 25, 707–725.

Kroemer, G., Marino, G., and Levine, B. (2010). Autophagy and the integrated stress response. Mol. Cell 40, 280–293.

Lakhani, S.A., Masud, A., Kuida, K., Porter, G.A., Jr., Booth, C.J., Mehal, W.Z., Inayat, I., and Flavell, R.A. (2006). Caspases 3 and 7: key mediators of mitochondrial events of apoptosis. Science 311, 847–851.

Lamothe, B., Lai, Y., Xie, M., Schneider, M.D., and Darnay, B.G. (2013). TAK1 is essential for osteoclast differentiation and is an important modulator of cell death by apoptosis and necroptosis. Mol. Cell Biol. 33, 582–595.

Lemke, J., von Karstedt, S., Zinggrebe, J., and Walczak, H. (2014). Getting TRAIL back on track for cancer therapy. Cell Death Differ. 21, 1350–1364.

Levine, B., and Kroemer, G. (2008). Autophagy in the pathogenesis of disease. Cell 132, 27–42.

Levy, J.M., Thompson, J.C., Griesinger, A.M., Amani, V., Donson, A.M., Birks, D.K., Morgan, M.J., Mirsky, D.M., Handler, M.H., Foreman, N.K., et al. (2014). Autophagy inhibition improves chemosensitivity in BRAF(V600E) brain tumors. Cancer Discov. 4, 773–780.


Lindsten, T., Ross, A.J., King, A., Zong, W.X., Rathmell, J.C., Shiels, H.A., Ulrich, E., Waymire, K.G., Mahar, P., Frauworth, K., et al. (2000). The combined functions of proapoptotic Bcl-2 family members bak and bax are essential for normal development of multiple tissues. Mol. Cell 6, 1389–1399.

Linkermann, A., and Green, D.R. (2014). Necroptosis. N. Engl. J. Med. 370, 455–465.

Liu, W., Chang, B.L., Cramer, S., Koty, P.P., Li, T., Sun, J., Turner, A.R., Von Kap-Herr, C., Bobby, P., Rao, J., et al. (2007). Deletion of a small consensus region at 6q15, including the MAP3K7 gene, is significantly associated with high-grade prostate cancers. Clin. Cancer Res. 13, 5028–5033.

Lluis, J.M., Nachbur, U., Cook, W.D., Gentle, I.E., Moujalled, D., Moulin, M., Wong, W.W., Khan, N., Chau, D., Callus, B.A., et al. (2010). TAK1 is required for survival of mouse fibroblasts treated with TRAIL, and does so by NF-kappaB dependent induction of cFLIPL. PLoS One 5, e8620.

Martinez, J., Malireddi, R.K., Lu, Q., Cunha, L.D., Pelletier, S., Gingras, S., Orchard, R., Guan, J.L., Tan, H., Peng, J., et al. (2015). Molecular characterization of LC3-associated phagocytosis reveals distinct roles for Rubicon, NOX2 and autophagy proteins. Nat. Cell Biol. 17, 893–906.

McAfee, Q., Zhang, Z., Samanta, A., Levi, S.M., Ma, X.H., Piao, S., Lynch, J.P., Uehara, T., Sepulveda, A.R., Davis, L.E., et al. (2012). Autophagy inhibitor Lys05 has single-agent antitumor activity and reproduces the phenotype of a genetic autophagy deficiency. Proc. Natl. Acad. Sci. USA 109, 8253–8258.

Mizushima, N., and Levine, B. (2010). Autophagy in mammalian development and differentiation. Nat. Cell Biol. 12, 823–830.

Morgan, M.J., Kim, Y.S., and Liu, Z.G. (2009). Membrane-bound Fas ligand requires RIP1 for efficient activation of caspase-8 within the death-inducing signaling complex. J. Immunol. 183, 3278–3284.

Morioka, S., Omori, E., Kajino, T., Kajino-Sakamoto, R., Matsumoto, K., and Ninomiya-Tsuji, J. (2009). TAK1 kinase determines TRAIL sensitivity by modulating reactive oxygen species and clAP. Oncogene 28, 2257–2265.

Morioka, S., Broglie, P., Omori, E., Ikeda, Y., Takaesu, G., Matsumoto, K., and Ninomiya-Tsuji, J. (2014). TAK1 kinase switches cell fate from apoptosis to necrosis following TNF stimulation. J. Cell Biol. 204, 607–623.

Nezis, I.P., Shravage, B.V., Sagona, A.P., Johansen, T., Baehrecke, E.H., and Stenmark, H. (2010). Autophagy as a trigger for cell death: autophagic degradation of inhibitor of apoptosis dBruce controls DNA fragmentation during late oogenesis in Drosophila. Autophagy 6, 1214–1215.

Oberst, A., Dillon, C.P., Weinlich, R., McCormick, L.L., Fitzgerald, P., Pop, C., Hakem, R., Salvesen, G.S., and Green, D.R. (2011). Catalytic activity of the caspase-8-FLIP(L) complex inhibits RIPK3-dependent necrosis. Nature 471, 363–367.

Orozco, S., Yatim, N., Werner, M.R., Tran, H., Gunja, S.Y., Tait, S.W., Albert, M.L., Green, D.R., and Oberst, A. (2014). RIPK1 both positively and negatively regulates RIPK3 oligomerization and necroptosis. Cell Death Differ. 21, 1511–1521.

Pasparakis, M., and Vandenabeele, P. (2015). Necroptosis and its role in inflammation. Nature 517, 311–320.

Rebecca, V.W., and Amaravadi, R.K. (2015). Emerging strategies to effectively target autophagy in cancer. Oncogene 35, 1–11.

Rodrigues, L.U., Rider, L., Nieto, C., Romero, L., Karimpour-Fard, A., Loda, M., Lucia, M.S., Wu, M., Shi, L., Cimic, A., et al. (2015). Coordinate loss of MAP3K7 and CHD1 promotes aggressive prostate cancer. Cancer Res. 75, 1021–1034.

Rodriguez, D.A., Weinlich, R., Brown, S., Guy, C., Fitzgerald, P., Dillon, C.P., Oberst, A., Quarato, G., Low, J., Cripps, J.G., et al. (2016). Characterization of RIPK3-mediated phosphorylation of the activation loop of MLKL during necroptosis. Cell Death Differ. 23, 76–88.

Rubinstein, A.D., and Kimchi, A. (2012). Life in the balance—a mechanistic view of the crosstalk between autophagy and apoptosis. J. Cell Sci. 125, 5259–5268.

Sanz, L., Sanchez, P., Lallena, M.J., Diaz-Meco, M.T., and Moscat, J. (1999). The interaction of p62 with RIP links the atypical PKCs to NF-kappaB activation. EMBO J. 18, 3044–3053.

Siegel, R., Ma, J., Zou, Z., and Jemal, A. (2014). Cancer statistics, 2014. CA Cancer J. Clin. 64, 9–29.

Sun, L., Wang, H., Wang, Z., He, S., Chen, S., Liao, D., Wang, L., Yan, J., Liu, W., Lei, X., et al. (2012). Mixed lineage kinase domain-like protein mediates necrosis signaling downstream of RIP3 kinase. Cell 148, 213–227.

Taylor, B.S., Schultz, N., Hieronymus, H., Gopalan, A., Xiao, Y., Carver, B.S., Arora, V.K., Kaushik, P., Cerami, E., Reva, B., et al. (2010). Integrative genomic profiling of human prostate cancer. Cancer Cell 18, 11–22.

Thorburn, J., Andrysik, Z., Staskiewicz, L., Gump, J., Maycotte, P., Oberst, A., Green, D.R., Espinosa, J.M., and Thorburn, A. (2014). Autophagy controls the kinetics and extent of mitochondrial apoptosis by regulating PUMA levels. Cell Rep. 7, 45–52.

Vandenabeele, P., Declercq, W., Van Herreweghe, F., and Vanden Berghe, T. (2010). The role of the kinases RIP1 and RIP3 in TNF-induced necrosis. Sci. Signal. 3, re4.

Vanlangenakker, N., Vanden Berghe, T., Bogaert, P., Laukens, B., Zobel, K., Deshayes, K., Vucic, D., Fulda, S., Vandenabeele, P., and Bertrand, M.J. (2011). cIAP1 and TAK1 protect cells from TNF-induced necrosis by preventing RIP1/RIP3-dependent reactive oxygen species production. Cell Death Differ. 18, 656–665.

Wang, L., Du, F., and Wang, X. (2008). TNF-alpha induces two distinct caspase-8 activation pathways. Cell 133, 693–703.

Wu, M., Shi, L., Cimic, A., Romero, L., Sui, G., Lees, C.J., Cline, J.M., Seals, D.F., Sirintrapun, J.S., McCoy, T.P., et al. (2012). Suppression of Tak1 promotes prostate tumorigenesis. Cancer Res. 72, 2833–2843.

Wu, J., Huang, Z., Ren, J., Zhang, Z., He, P., Li, Y., Ma, J., Chen, W., Zhang, Y., Zhou, X., et al. (2013). MLKL knockout mice demonstrate the indispensable role of MLKL in necroptosis. Cell Res. 23, 994–1006.

Yatim, N., Jusforgues-Saklani, H., Orozco, S., Schulz, O., Barreira da Silva, R., Reis, E.S.C., Green, D.R., Oberst, A., and Albert, M.L. (2015). RIPK1 and NF-kappaB signaling in dying cells determines cross-priming of CD8+ T cells. Science 350, 328–334.

Zhang, H., Zhou, X., McQuade, T., Li, J., Chan, F.K., and Zhang, J. (2011). Functional complementation between FADD and RIP1 in embryos and lymphocytes. Nature 471, 373–376.

Zhao, J., Jitkaew, S., Cai, Z., Choksi, S., Li, Q., Luo, J., and Liu, Z.G. (2012). Mixed lineage kinase domain-like is a key receptor interacting protein 3 downstream component of TNF-induced necrosis. Proc. Natl. Acad. Sci. USA 109, 5322–5327.
