
TICB 1256 No. of Pages 12 ARTICLE IN PRESS

Trends in Cell Biology

Review

An evolutionary perspective on the necroptotic pathway

Yves Dondelinger,$^{1,2}$ Paco Hulpiau,$^{1,2}$ Yvan Saeys,$^{1,3}$ Mathieu J.M. Bertrand,$^{1,2}$ and Peter Vandenabeele$^{1,2,*}$

Throughout the animal kingdom, innate immune receptors protect the organism from microbial intruders by activating pathways that mediate inflammation and pathogen clearance. Necroptosis contributes to the innate immune response by killing pathogen-infected cells and by alerting the immune system through the release of danger signals. Components of the necroptotic signaling axis – TIR-domain-containing adapter-inducing interferon-β (TRIF), Z-DNA sensor DAI, receptor-interacting kinase (RIPK)1, RIPK3 and mixed-lineage kinase domain-like protein (MLKL) – are therefore expected to be found in all animals. However, a phylogenetic analysis reveals that the necroptotic axis, except for RIPK1, is poorly conserved in the animal kingdom, suggesting that alternative mechanisms regulate necroptosis in these species or that necroptosis would apparently be absent. These findings question the universal role of necroptosis during innate immunity in the animal kingdom.

### Necroptosis Contributes to Innate Immunity

Necrosis has long been considered an accidental form of cell death induced by physicochemical damage. It has, however, become clear that necrosis can also occur, in many forms, as a genetically programmed mode of cell death called regulated necrosis [1]. Morphologically, regulated necrosis is characterized by cell and organelle swelling, which ultimately culminates in plasma membrane rupture. Of the different subsets of regulated necrosis, the molecular mechanisms of necroptosis are the best understood at the moment. Necroptosis induction requires RIPK3 and MLKL, and is tightly controlled by phosphorylation/dephosphorylation, ubiquitylation/deubiquitylation, and proteolytic events [2].

Necroptosis can be activated downstream of several death receptors (DRs), including tumor necrosis factor receptor (TNFR)1, Fas, TNF-related apoptosis-inducing ligand receptor (TRAILR)1, and TRAILR2 [3]. Additionally, necroptosis can be triggered by stimulation of specific pathogen recognition receptors (PRRs), including TLR3, TLR4, and the Z-DNA sensor DAI (also known as ZBP1) [4–7]. As a result of the importance of these receptors in the innate immune response against various microbial pathogens, programmed cell death is believed to play a key role in the killing and removal of pathogen-infected cells [8,9]. However, necroptosis is not the default pathway activated downstream of these DRs and PRRs. In most cases, binding of the ligand to its respective receptor triggers induction of proinflammatory mediators via activation of the mitogen-activated protein kinase (MAPK) and nuclear factor (NF)-κB pathways and/or induces apoptosis. The apoptotic cellular suicide pathway prevents further pathogen replication and results in the clearance of the infected cells from the organism by macrophages. Some microbes have evolved to evade from this clearance mechanism by acquiring genes encoding caspase inhibitors that can block the apoptotic cell death machinery [8]. In conditions of caspase-8 suppression, the necroptotic pathway can be activated to counteract the pathogen-mediated apoptosis inhibition. In this way, necroptosis functions as a back-up cell death program that is

**Trends**

Necroptosis is believed to contribute to innate immunity in humans and mice by killing pathogen-infected cells and by alerting the immune system through the release of danger signals

RIPK3 and MLKL form the core machinery of necroptotic cell death in mammalian cells

During necroptosis, RIPK3 phosphorylates and activates MLKL

MLKL activation results in its recruitment and oligomerization at phosphatidylinositol phosphates in the plasma membrane leading to plasma membrane rupture and cell death

It is unclear when the necroptotic cell death pathway emerged and whether it acts as a universal cell death program in the animal kingdom

1 VIB Inflammation Research Center, Technologiepark 927, Zwijnaarde-Ghent, 9052, Belgium  
2 Department of Biomedical Molecular Biology, Ghent University, Technologiepark 927, Zwijnaarde-Ghent, 9052, Belgium  
3 Department of Respiratory Medicine, Ghent University Hospital, Ghent, Belgium  

*Correspondence:  
Peter.Vandenabeele@irc.vib-ugent.be  
(P. Vandenabeele).

Trends in Cell Biology, Month Year, Vol. xx, No. yy  
http://dx.doi.org/10.1016/j.tcb.2016.06.004  
© 2016 Elsevier Ltd. All rights reserved.

activated when apoptosis induction fails [8]. Plasma membrane rupture of necroptotic cells leads to the release of damage-associated molecular patterns (DAMPs) in the surrounding tissue, which may act as a danger signal that triggers inflammation, resulting in the attraction of different types of immune cells to the site of infection [10]. The cooperative action between necroptosis and the innate immune system results in the elimination of these microbes from the organism.

At present, all evidence indicates that RIPK3 and MLKL form the core of the necroptotic machinery as both proteins are indispensable for necroptosis induction downstream of the above mentioned DRs and PRRs [5,11–13]. However, certain pathway-specific adaptor proteins are required to transmit the necroptotic signal from the receptors to RIPK3 and MLKL. Both RIPK3 kinase activity and RIPK3 RHIM domain are required for necroptosis induction [11,12]. The adaptor protein RIPK1 is an essential protein downstream of DRs, while TRIF is required downstream of Toll-like receptor (TLR)3/4 (Figure 1) [3,5]. RIPK1 and TRIF contain a RHIM domain that allows RIPK3 activation via RHIM-mediated interactions [6,7,11,14,15]. The Z-DNA sensor DAI also contains two RHIM-like domains, which allows it to directly activate RIPK3 in the absence of RIPK1 or TRIF. All these intrinsically different pathways are able to converge on RIPK3 activation via RHIM-mediated interactions (Figure 1). Active RIPK3 then interacts with the kinase-like domain of MLKL, allowing phosphorylation of MLKL by RIPK3 [16]. RIPK3 phosphorylates MLKL in its activation loop, which induces a conformational change in MLKL that puts it into its active state [16–19]. Active MLKL then, either directly or indirectly, destabilizes plasma membrane integrity leading to cell swelling and membrane rupture (Figure 1) [20–24].

Throughout the animal kingdom, PRRs play crucial roles in the recognition and removal of pathogens by activating innate immune responses [25,26]. Similarly, members of the DR family and their ligands have been described from Cnidarians to mammals and are reported to contribute to the inflammatory response and to immune cell homeostasis, although it is not clear whether these functions are completely conserved during evolution [27,28]. Our current understanding of the molecular mechanisms of necroptosis mainly originates from studies performed with samples of human and mouse origin, and it remains unclear whether necroptosis also occurs in more evolutionary distant species. Previous studies have reported the extent to which regulated necrosis occurs in some model systems, such as *Caenorhabditis elegans*, *Dictyostelium discoideum*, *Saccharomyces cerevisiae*, and *Drosophila melanogaster*, but they were approached from a more descriptive and morphological point of view, rather than through the identification of the molecular players involved [29,30]. To gain a better understanding of the conservation of the necroptotic pathways in the animal kingdom, we performed a phylogenetic analysis to identify RIPK1, TRIF, DAI, RIPK3, and MLKL homologs (Figure 2). We found that several components of the necroptotic pathway are surprisingly poorly conserved within the animal kingdom, which suggests that RIPK3-MLKL-mediated necroptosis, as we know it in humans and mice, cannot be a universal cell death pathway.

### Ubiquitously Expressed RIPK1 is a Master Regulator of Cell Death and Inflammation Pathways Downstream of DRs

Members of the TNF superfamily mediate a broad spectrum of cellular responses including survival, proliferation, differentiation, and cell death [31]. Several receptors of the TNF superfamily, namely the DRs, contain a cytoplasmic death domain (DD) that allows them to induce programmed cell death [31]. Upon ligand binding to a DR, large structural rearrangements in the receptor induce the exposition of the DD, allowing recruitment of other DD-containing proteins, including RIPK1, via DD-DD interactions to initiate assembly of large protein complexes [32]. In most cells, DR activation can lead to either cell survival or apoptosis depending on the cellular context. As mentioned earlier, inhibition of the apoptotic pathway often results in the activation of a cryptic necroptotic pathway as an alternative cell death route [3,11–13,33]. Although RIPK1 is dispensable for apoptosis induction downstream of TRAILR1/2, FasR, and in some cases

Trends in Cell Biology

Cell swelling resulting in  
plasma membrane rupture  

Activation of Na²⁺ and  
Ca²⁺ channels?  

Direct pore formation?  

TNF          FasL         TRAIL  
Cytoplasm  
RHIM —— RHIM  
domain       domain  

open active  
conformation  

MLKL  

RHIM —— RHIM  
domain       domain  

Viral z-DNA  
DAI  

Figure 1. Overview of Different Pathways Leading to Necroptosis. Necroptosis is activated downstream of death receptors such as TNFR1, Fas, and TRAILR1/2 or pathogen recognition receptors including TLR3/4 and Z-DNA sensor DAI. The pro-death signals from different receptors converge on RIPK3 and MLKL activation by RHIM-mediated interactions via receptor-specific adaptor proteins in mammalian cells. Phosphorylation of MLKL by RIPK3 induces conformational changes in MLKL leading to MLKL membrane translocation and plasma membrane rupture. Abbreviations: MLKL, mixed-lineage kinase domain-like protein; RIPK, receptor-interacting kinase; TLR, Toll-like receptor; TNF, tumor necrosis factor; TNFR1, TNF receptor 1; TRAILR, TNF-related apoptosis-inducing ligand receptor; TRIF, TIR-domain-containing adapter-inducing interferon-β.

Trends in Cell Biology

**Figure 2. Presence of RIPK1, TRIF, DAI, RIPK3, and MLKL Homologs in the Animal Kingdom.** A phylogenetic analysis was performed by analyzing the publicly available genome assemblies of the indicated species for the presence or absence of RIPK1, TRIF, Z-DNA sensor DAI, RIPK3 and MLKL homologs and by utilizing the BLASTP algorithm against the predicted proteomes of these species. The presence or absence of these homologs was then validated by a BLAST search of conserved sequences against the genome assemblies in combination with an evaluation of the completeness of the genomic context in the ENSEMBL, NCBI, and UCSC genome browsers. If all above-mentioned analyses were negative, a gene was considered absent. The genomic assembly of platypus did not contain a RIPK3 gene at the time of analysis, but there are gaps in the contigs near the RIPK3 locus rendering the result inconclusive. The Echinodermata, Hemichordata, Cephalochordata and Urochordata contain RIPK-like proteins, but they are quite divergent from Vertebrata RIPK1/3 making it unclear whether they are true RIPK1/3 homologs. From left to right, species in each clade from which the genomes were investigated: (1) Cnidaria: *Nematostella vectensis* (sea anemone), *Hydra vulgaris* (fresh-water polyp), (2) Ecdysozoa: *Drosophila melanogaster* (fruit fly), *Caenorhabditis elegans* (roundworm), (3) Lophotrochozoa: *Aplysia californica* (sea hare), *Lottia gigantea* (owl limpet), *Helobdella robusta* (leech), *Capitella teleta* (polychaete worm); Non-vertebrate deuterostomes (4) Echinodermata: *Strongylocentrotus purpuratus* (purple sea urchin), (5) Hemichordata: *Saccoglossus kowalevskii* (acorn worm), (6) Cephalochordata: *Branchiostoma floridae* (amphioxus), (7) Urochordata: *Ciona intestinalis* (sea squirt); Non-mammalian vertebrates (8) Cyclostomata: *Petromyzon marinus* (lamprey), (9) Fish: *Danio rerio* (zebrafish), *Fugu rubripes* (fugu), *Clupea harengus* (herring), *Lepisosteus oculatus* (spotted gar), *Callorhinchus milli* (elephant shark), *Latimeria chalumnae* (coelacanth), (10) Amphibia: *Xenopus tropicalis* (frog), (11) Reptilia: *Anolis carolinensis* (lizard), *Chrysemys picta* (turtle), *Alligator mississippiensis* (American alligator), (12) Aves: *Gallus gallus* (chicken), *Taeniopygia guttata* (zebra finch), *Cuculus canorus* (cuckoo), *Melopsittacus undulatus* (budgerigar); Mammalian genomes (13) Prototheria: *Ornithorhynchus anatinus* (platypus), (14) Metatheria: *Monodelphis domestica* (opossum), *Macropus eugenii* (wallaby), *Sarcophilus harrisii* (Tasmanian devil) (15) Afrotheria: *Loxodonta africana* (elephant), *Procavia capensis* (rock hyrax) (16) Xenarthra: *Dasypus novemcinctus* (armadillo), *Choloepus hoffmanni* (two-toed sloth) (17) Eulipotyphla: *Sorex araneus* (shrew), *Erinaceus europaeus* (hedgehog), (18) Chiroptera: *Pteropus vampyrus* (megabat), *Myotis lucifugus* (microbat), (19) Cetartiodactyla: *Sus scrofa* (pig), *Balaenoptera acutorostrata* (whale), (20) Perissodactyla: *Equus caballus* (horse), (21) Carnivora: *Felis catus* (cat), *Canis lupus familiaris* (dog), *Mustela putorius furo* (ferret), (22) Rodentia: *Mus musculus* (mouse), (23) Lagomorpha: *Ochotona princeps* (pika), (24) Scandentia: *Tupaia belangeri* (tree shrew), (25) Dermoptera: *Galeopterus variegatus* (flying lemur), (26) Primates: *Homo sapiens* (human), *Pan troglodytes* (chimp). Abbreviations: MLKL, mixed-lineage kinase domain-like protein; RIPK, receptor-interacting kinase; TRIF, TIR-domain-containing adapter-inducing interferon-β.

TNFR1, it is required for necroptosis induction downstream of all of these receptors [3,34,35]. At present, it is not clear how caspase inhibition switches from an apoptotic response to a necroptotic one, but it might depend on the inhibition of caspase 8-dependent RIPK1, RIPK3, and CYLD cleavage [36–40]. Regardless of the exact mechanism, necroptosis induction downstream of different death receptors converges on RIPK1 kinase activation (Figure 1).

RIPK1 is composed of an N-terminal kinase domain linked by a largely unstructured intermediate domain to a C-terminal DD [41] (Figure 2). The RHIM domain of RIPK1 is localized at the most C-terminal part of its intermediate domain [42]. Both the RHIM domain and kinase activity of RIPK1 are required for necroptosis induction [3,11]. As discussed later, the RHIM domain probably acts as a protein-protein aggregation seed inducing RIPK3 oligomerization and activation. Although well established for the past 15 years, the precise function of RIPK1 kinase activity in cell death

induction remains an enigma, and necroptotic RIPK1 substrates have not yet been identified [3]. RIPK1 kinase activity also regulates apoptosis in certain conditions [35,43–45], suggesting that the same RIPK1 substrate may control both necroptosis and apoptosis. It is tempting to speculate that RIPK1 itself is its main substrate and that RIPK1 autophosphorylation creates an open conformation allowing the interaction with pro-death molecules such as FADD (Fas-associated DD) and RIPK3. Multiple autophosphorylation sites have been identified, but none have been reported to significantly influence RIPK1-dependent cell death pathways [46,47]. Alternatively, RIPK1 may phosphorylate different substrates in the apoptotic and necroptotic pathway depending on the cellular context.

Reflective of its key role as an adaptor molecule transmitting a death signal downstream of DRs, RIPK1 homologs can be found within the entire Craniata clade (Figure 2). Intriguingly, genes encoding proteins with similar domain architecture to RIPK1 can be found, and often in multiple copies, in genomes of the Echinodermata and Hemichordata phyla, as well as Cephalochordata and Urochordata subphyla. However, the amino acid sequences of these proteins are substantially different from the sequence of the true RIPK1 orthologs found in the Craniata clade (Figure 3). Without functional data, it is difficult to estimate whether they represent true RIPK1 orthologs. At the moment, it is unclear whether necroptosis also occurs downstream of DRs in the more distantly related Deuterostomia. The omnipresence of RIPK1 in Deuterostomia is probably related to its function in modulating NF-κB and MAPK signaling pathways in addition to its regulatory role in both apoptosis and necroptosis [2].

RIPK1 is not expressed in the clades of the Cnidaria and Protostomia, including the phyla of Mollusca, Annelida, Arthropoda, and Nematoda. However, all Protostomia, except nematodes, express IMD (immunodeficiency) or IMD-like proteins (Figure 2). IMD plays a crucial role downstream of the peptidoglycan receptor PGRP-LC in *Drosophila*, thereby contributing to host defense against both Gram-negative and Gram-positive bacteria [48–50]. IMD is considered a RIPK1/2 homolog because its C-terminal DD shows homology with the DD and CARD of mammalian RIPK1 and RIPK2 respectively. In addition to sequence homology, both IMD and the PGRP-LC pathway in *Drosophila* show a high degree of functional similarity with RIPK1 and the TNFR1 pathway, and with RIPK2 and the nucleotide-binding oligomerization domain (NOD) 1/2 pathways in mammalian cells [48,51]. NOD1/2 are innate immune intracellular receptors activated by bacterial peptidoglycans. In all of these pathways, IMD proteins, RIPK1, and RIPK2 play central roles in the induction of the innate immune response. The conjugation of ubiquitin chains by members of the inhibitor of apoptosis protein (IAP) family creates a scaffold for the downstream activation of the MAPK and NF-κB signaling pathways [51]. In addition, all three proteins are capable of inducing cell death downstream of these receptors [3,52,53]. In the case of RIPK1, cell death induction relies on its kinase activity [3,35,45]. However, IMD lacks the N-terminal kinase domain characteristic for Craniata RIPK1. Intriguingly, the IMD-like protein CAPTEDRAFT 219991 from the Annelid *Capitella teleta* contains an N-terminal kinase domain that seems unrelated to the Craniata RIPK1 kinase domain (Figure 3). This finding indicates that the fusion of an N-terminal kinase domain to a C-terminal DD, as present in RIPK1, could have occurred on multiple occasions during evolution.

It is interesting to note that many molecular events are highly similar between the mammalian TNF and NOD1/2 pathways and the *Drosophila* IMD pathway [48,51]. In addition, DR pathways seem to be present from cnidarians to mammals and show a remarkable extent of functional conservation [28]. These findings suggest that these cell death pathways, and the proteins regulating them, must have emerged from a common ancestor early in evolution and diverged into the many different yet functionally related pathways we know today.

Trends in Cell Biology

**Key:**
- RIPK1 homologs
- RIPK3 homologs
- MLKL homologs

![Phylogenetic Tree](image.png)

Figure 3. Phylogenetic Tree of RIPK1, RIPK3, and MLKL Homologs. The kinase domains of the putative MLKL, RIPK3, or RIPK1 homologs were aligned via the CLC Main Workbench program (v7.5.1) and the data are presented as unrooted radial tree created by neighbor-joining. While it is evident that sea urchin (*Strongylocentrotus purpuratus*), acorn worm (*Saccoglossus kowalevskii*), lancelet (*Branchiostoma belcheri tsingtauense*) and sea squirt (*Ciona intestinalis*) contain potentially different RIPK1 and RIPK3 homologs, it is difficult, without functional data, to estimate whether they represent true RIPK1 and RIPK3 orthologs. The *Ciona intestinalis* RIPK1-and RIPK3-like proteins cluster together and not with their Gnathostomata counterparts. Similar clustering is observed among other more distantly related RIPK1 molecules. More functional evidence is needed to catalogue these RIPK1-and RIPK3-like proteins as true RIPK1 or RIPK3 homologs.

Abbreviations: MLKL, mixed-lineage kinase domain-like protein; RIPK, receptor-interacting kinase.

---

### RHIM-Mediated Interactions Evolved to Activate Necroptosis in Mammals

RIPK3 is a serine/threonine kinase consisting of an N-terminal kinase domain and a C-terminal disordered region in which a RHIM domain resides (Figure 2) [42,54,55]. In humans and mice, both RIPK3 kinase activity and the RHIM domain are required for necroptosis induced by various receptors [11–14]. The molecular mechanisms leading to RIPK3 kinase activation are not clear, although it has sometimes been stated that RIPK3 is activated through RIPK1-mediated phosphorylation [11]. However, it is important to note that direct RIPK3 phosphorylation by RIPK1 has not been demonstrated to be a prerequisite for RIPK3-mediated necroptosis. Furthermore, RIPK1 and its kinase activity are not required for necroptosis induction downstream of PRRs TLR3/4 and the cytosolic Z-DNA sensor DAI, making it clear that RIPK1-mediated RIPK3

6 Trends in Cell Biology, Month Year, Vol. xx, No. yy

phosphorylation, or RIPK3 phosphorylation in general, cannot be a universal theme in RIPK3 activation (Figure 1) [5–7]. Instead, activation of RIPK1 kinase activity may represent a switch from the scaffold pro-survival role of RIPK1 to its kinase-dependent pro-death function, which can promote necroptosis as well as apoptosis [2]. A potential hypothesis is that RIPK1 kinase activity is important for RIPK1 autophosphorylation, which may lead to conformational changes allowing RIPK1 to interact with and activate RIPK3 via RHIM-RHIM interactions. Indeed, all adaptor/sensor proteins reported to transmit the necroptotic signal to RIPK3 (RIPK1, TRIF, and DAI) contain at least one RHIM domain, suggesting that RHIM-mediated interactions are crucial in the regulation of RIPK3 activation.

The RHIM domain was initially discovered as a protein interaction domain required for the interaction between RIPK1 and RIPK3 [42]. Since then, two additional mammalian proteins, DAI and TRIF, were found to contain RHIM domains [56,57]. Although RHIM-mediated interactions are crucial for necroptosis induction, the molecular basis for these interactions is not well understood. The minimal interaction domain has to contain a stretch of about 70 amino acids in which a crucial conserved core motif of 12 amino acids resides [42]. Mutagenesis of four consecutive amino acids to alanine or one amino acid to aspartate within this core region abrogates RHIM-RHIM interaction and, as a consequence, necroptosis induction [42,58]. The RHIM domain is mainly disordered, except for the core region that has a propensity to form β strands [58]. RHIM-mediated interactions appear to drive RIPK1-RIPK3 oligomerization, leading to the formation of amyloid-like filamentous structures during TNF-induced necroptosis [58]. Most of the knowledge on how RHIM-mediated interactions regulate necroptosis originates from dimerization experiments [59–61]. In those experiments, dimerization is induced artificially by using fusion proteins containing portions of RIPK1 or RIPK3 linked to a domain that can oligomerize upon binding to an exogenously added ligand. Forced heterodimerization of RIPK1-RIPK3 kinase domains induced necroptosis only when the RIPK3 portion contained a functional RHIM domain capable of recruiting additional RIPK3 proteins [61]. Dimerization of RHIM-deficient RIPK3 kinase domains is also sufficient to induce necroptosis, indicating that RIPK3 kinase domain dimers might represent the smallest functional unit able to initiate the necroptotic signal. The RHIM domain would therefore be important for RIPK3 recruitment and dimerization [61], which allows for autophosphorylation on Ser227 in human cells (Ser232 in mice) and subsequent MLKL recruitment [61]. Together, these findings suggest that RHIM-mediated higher order structures form when RIPK3 proteins are brought in close proximity, allowing for downstream activation of RIPK3 and MLKL (Figure 1). Most of the data are based on the interaction between the RHIM domains of RIPK1 and RIPK3. It would be interesting to validate this model by characterizing DAI-RIPK3 and TRIF-RIPK3 RHIM interactions and investigating their contribution to necroptosis induction.

Our phylogenetic analysis reveals that TRIF emerged after the bifurcation between Cyclostomata and Gnathostomata, which includes all vertebrates except Cyclostomata. By contrast, DAI appeared later, namely in the Amphibia class. Intriguingly, both TRIF and DAI surfaced during the same evolutionary time frame as RIPK3 (Figure 2), suggesting a mechanism of coevolution between sensor/adaptor (DAI/TRIF) and the recruited kinase (RIPK3). Furthermore, DAI apparently was lost on multiple occasions during evolution, as both the Aves class (chicken, zebra finch, and cuckoo) and the Marsupialia infraclass (opossum, wallaby, and Tasmanian devil) do not express DAI. Interestingly, this finding again coincides with the absence of RIPK3 in these two cases, and it might be explained by the redundancy of DAI and other interferon-inducing DNA sensors in conditions where DAI is unable to additionally induce necroptosis due to the absence of RIPK3 [62–64]. TRIF, in contrast, is essential for TLR3/4-induced NF-κB and interferon regulatory factor 3 activation, which explains its ubiquitous expression in the Gnathostomata clade [65].

Trends in Cell Biology

Current RHIM domain recognition algorithms fail to recognize RHIM domains in all RIPK1, RIPK3, DAI, and TRIF homologs in the Craniata clade, suggesting that these algorithms may be too strict to identify more distantly related RHIM domains. Indeed, a recent article indicated the presence of more divergent RHIM-like domains in *Drosophila* (including the *Drosophila* RIPK1 ortholog IMD) and *Branchiostoma* proteins [66]. The authors linked the mammalian RHIM domain to the prion-forming portion of the HET-s protein of the fungus *Podospora anserina* [66]. The two different variants of the HET-s protein assemble into amyloid-like structures to induce fungal cell death during heterokaryon incompatibility [67, 68], indicating that the role of RHIM-like interactions is probably not limited to mammalian necroptosis regulation. Instead, they likely have a much broader role in innate immunity by modulating cell death and self/non-self-discrimination processes in Eukaryotes.

**The Components of the Necroptotic Core Machinery, RIPK3 and MLKL, Are Surprisingly Poorly Conserved**

Together with serine/threonine kinase RIPK3, MLKL forms the core of the necroptotic machinery and both are, as a consequence, indispensable for necroptosis induction in mice and humans downstream of PRRs and DRs [5, 11–13, 16, 69, 70]. MLKL is composed of an N-terminal 4-helical-bundle domain (4HBD) linked by a brace region to a C-terminal pseudokinase domain (Figure 2) [17]. During necroptosis, the RIPK3 kinase domain interacts with the pseudokinase domain of MLKL, resulting in RIPK3-mediated phosphorylation of the activation loop of MLKL [16, 17, 19, 71]. It is currently not clear how RIPK3-mediated phosphorylation activates the killing function of MLKL. Structure-function analysis identified the 4HBD of MLKL to be sufficient for necroptosis induction [18, 21–23]. A potential model is that RIPK3 phosphorylation of MLKL induces a conformational change allowing MLKL to adopt an open, active conformation and thereby exposing clusters of conserved amino acids present at the surface of the MLKL 4HBD [17, 18, 20, 22, 23, 72]. These conserved residues appear to be involved in the translocation and oligomerization of MLKL at membranes, including the plasma membrane [18, 22, 72]. In addition, recombinant MLKL binds phosphatidylinositol phosphates and cardiolipin [22, 23]. Together, these results suggest a model in which clusters of conserved amino acids present in the 4HBD of MLKL allows its translocation to negatively charged components of membranes. At these membranes, MLKL most likely oligomerizes into higher-order complexes of unknown stoichiometry [18, 20–23]. Oligomerized MLKL was suggested to activate sodium or calcium channels in the plasma membrane, thereby disrupting cellular ion homeostasis and resulting in osmotic cell swelling [20, 21]. MLKL was also suggested to directly induce plasma membrane permeabilization; a model supported by the fact that recombinant MLKL can induce leakage from phosphatidylinositol phosphate-containing liposomes [22, 23, 73, 74]. Recently, it was proposed that MLKL could form a cation selective channel in the plasma membrane thereby unifying both models [24].

In accordance with a crucial role for necroptosis in the innate immune response, almost ubiquitous expression of MLKL was found in the Deuterostomia superphylum (Figure 2). Although the origin of the *MLKL* gene remains unclear, it seems to have appeared after the bifurcation between Protostomia (Annelida, Mollusca, Arthropoda, and Nematoda) and Deuterostomia (Echinodermata, Hemichordata, and Chordata) as all investigated lineages from the latter and none of the first express MLKL (Figure 2). In contrast, the *RIPK3* gene only appears in the Craniata clade, and not in Cephalochordata and Urochordata (Figure 2). *Ciona intestinalis*, belonging to the Urochordata subphylum, expresses a RIPK3-like molecule, but it is unclear whether it represents a true RIPK3 ortholog (Figure 3). Intriguingly, the fungus *Chaetomium globosum* contains a protein with a HELo-like domain, HELLP, which displays sequence homology with MLKL [75]. HELLP translocates to the cell membrane to induce fungal cell death in a process similar to the Het-s system in *Podospora anserina* [75]. As there are no MLKL-like proteins identified in the Protostomia clade, it is unclear whether fungal HELLP and

Trends in Cell Biology

Deuterostomia MLKL shared a common ancestor (which did not leave any traces in the Protostomia clade) or whether this is an example of convergent evolution.

A detailed analysis of the presence of RIPK3 and MLKL in Craniata reveals a more complex situation. Indeed, both genes were lost in some specific Craniata clades and other discrete species during evolution. Within the Mammalia class, the whole Carnivora order appears to lack the MLKL gene. Absence of the MLKL gene is observed in the domestic dog (*Canis lupus familiaris*), the domestic cat (*Felis catus*), and the ferret (*Mustela putorius furo*), but cannot be entirely validated in other Carnivora order members, including the giant panda (*Ailuropoda melanoleuca*), due to the incomplete genomic assemblies around the MLKL locus. Despite this, it is tempting to speculate that a microbe-rich and virus-containing diet of raw meat may have caused evolutionary counter-selection against necroptosis in the Carnivora order. The abundance of microbes and viruses in their diet may activate the necroptotic cell death pathway via DR and PRR signaling, resulting in uncontrolled inflammatory responses. Deregulated necroptosis may be detrimental for the host organism and could have driven selection against the presence of MLKL. However, it is interesting to note that in the class of the Reptilia, other carnivorous species such as the American alligator (*Alligator mississippiensis*) still contain the MLKL gene, suggesting that other factors besides diet may have contributed to the counter selection of the MLKL gene and necroptosis in general. In line with this idea, the complete class of Aves apparently lacks the RIPK3 gene, while the infraclass of Marsupialia (opossum, Tasmanian devil, and wallaby) lacks both RIPK3 and MLKL genes, although there are still gaps surrounding the RIPK3 and MLKL loci in the incomplete genomic assemblies of the Tasmanian devil and the wallaby. A possible physiological phenomenon that may have promoted counter selection against elements of the necroptosis pathway in these species is torpor. Many animals such as birds and mammals drastically reduce energy expenditure during times of cold exposure, food shortage, or drought, by temporarily abandoning optimal body temperature by a process called torpor (seasonal torpor in case of hibernation, daily torpor at night). A state of torpor might represent a cold ischemia condition, which could be followed by reperfusion when temperature and metabolism are restored [76]. In this respect, necroptosis has been shown to contribute to damage induced by ischemia-reperfusion injury [77]. The possible absence of RIPK3 and MLKL genes, and necroptosis as a consequence, in specific clades and species might, therefore, represent an evolutionary adaptation to the physiological phenomenon of torpor at the cost of the beneficial role that necroptosis plays in innate immunity. Contagious cancer transmission is rare in Mammalia, but can occur by biting leading to the Tasmanian devil facial tumor disease and by sexual intercourse resulting in canine transmissible venereal tumors [78]. Intriguingly, both species lack the core necroptotic machinery. Although purely speculative, it is conceivable that besides inbreeding and limited genetic diversity between individuals in a population, the absence of the necroptosis machinery in cancer cells may lead to more aggressive tumors escaping the immune surveillance system and facilitating contagious cancer transmission.

Our phylogenetic analysis reveals that certain species of fish, including the zebrafish (*Danio rerio*), the fugu (*Takifugu rubripes*), and the tetraodon (*Tetraodon nigroviridis*), also lack MLKL. Another interesting observation is that specific species, including the African clawed frog (*Xenopus (Silurana) tropicalis*), the lancelet, the acorn worm (*Saccoglossus kowalevskii*), and the sea urchin (*Strongylocentrotus purpuratus*) contain two copies of MLKL (Figure 3). With the exception of the African clawed frog (*Xenopus (Silurana) tropicalis*), MLKL duplication correlates well with the absence of RIPK3 in the genomes, suggesting the existence of alternative modes of MLKL activation in these RIPK3-lacking species. Together, these results indicate that RIPK3 and MLKL appeared early, namely after the bifurcation of Protostomia and Deuterostomia, but that RIPK3 and MLKL must have been lost later in specific lineages over the course of evolution.

Trends in Cell Biology

According to the essential role of RIPK3 and MLKL as the core necroptotic execution machinery in humans and mice (Figure 1), it will be important to determine whether cells from species lacking RIPK3 and/or MLKL maintain the ability to undergo necroptosis following activation of the pathways known to drive necroptosis in humans and mice. It is conceivable that other, yet unidentified proteins are able to contribute to necroptosis in the absence of RIPK3 and/or MLKL. Since some species still express RIPK3 but not MLKL, a compensatory protein may have taken over MLKL-dependent induction of necroptosis. In support of this hypothesis, RIPK3, independent of MLKL, promotes myocardial necroptosis in mice by CaMKII activation [79]. Alternatively, the presence of RIPK3 but not MLKL might reflect the fact that RIPK3 is a multifunctional protein that plays additional roles in pathways unrelated to necroptosis [80]. By contrast, birds express MLKL but not RIPK3, which suggests that other kinases may compensate for the loss of RIPK3 by directly activating MLKL. Interestingly, MLKL activity can be activated by phosphorylation in a RIPK3-independent manner, suggesting that additional, RIPK3-independent MLKL activation mechanisms exist in mammalian cells [81]. In further support of this idea, induced ectopic expression of human or chicken MLKL 4HBD was unable to induce necroptosis in murine fibroblasts, while expression of their mouse or frog counterparts were able to kill. By contrast, recombinant MLKL proteins derived from all of these different species were able to induce liposome leakage, indicating that poorly conserved unidentified proteins exist that must regulate MLKL activity [74]. Another argument in favor of the presence of other crucial downstream necroptotic factors comes from the observation that there is a large discrepancy in timing between MLKL phosphorylation/oligomerization and plasma membrane rupture in certain cellular models [81].

Taken together, this phylogenetic analysis questions the existence of a universal RIPK3- and MLKL-dependent necroptotic execution mechanism in the animal kingdom, and as a consequence the universal role of necroptosis in innate immunity. It would be interesting to test whether these nonmammalian, RIPK3- and/or MLKL-deficient cells are still able to induce necroptosis upon stimulation with known necroptotic triggers. If so, inhibitor, knockdown, or knockout screening studies in these cells may lead to the identification of the missing regulatory necroptotic proteins, which also might contribute to necroptosis induction in mammalian cells.

### Concluding Remarks

Necroptosis, which critically relies on the activity of both RIPK3 and MLKL, contributes to innate immunity as a pathogen clearance mechanism and it may also aid in alerting the immune system through the release of DAMPs. However, the appearance of necroptosis came at a price. Unwarranted activation of necroptosis may contribute to the pathogenesis of a broad spectrum of inflammatory diseases [2]. It appears that the necroptotic pathway emerged after the bifurcation of the Protostomia and the Deuterostomia superphyla, since almost all Deuterostomia and no Protostomia express the essential necroptotic execution protein MLKL. Intriguingly, this coincides with the presence of a kinase domain attached to the death domain of RIPK1-like/RIPK1 proteins in Deuterostomia. Indeed, RIPK1, RIPK3, and MLKL are regulated by several phosphorylation events. However, the precise sequence of events and mechanisms regulating necroptosis are still awaiting further investigation (see Outstanding Questions). Moreover, the absence of RIPK3 and/or MLKL in some clades and species suggests that during evolution, counter-selection against necroptosis may have occurred. The reason for counter-selection might be infection-associated events (e.g., Carnivora may be particularly prone to infections due to their diet) or evolutionary advantages afforded by torpor. Alternatively, it is conceivable that the structurally related paralogous MLKL-like proteins may compensate for the absence of MLKL, that other MLKL activation pathways exist, or that other regulated necrotic pathways function as backup systems for necroptosis. Regardless, the phylogenetic analysis presented in this review indicates that necroptosis sensu stricto in these more distantly related phyla cannot be initiated from the classical kinase-regulated RIPK3-MLKL necroptotic

---

**Outstanding Questions**

- Is MLKL activity the final step in the necroptotic signaling cascade and does it directly permeabilize the plasma membrane? If so, what is the precise mechanism of MLKL-mediated plasma membrane permeabilization?
- What is the identity of the additional factors that regulate necroptotic cell death at the level of RIPK3 and MLKL?
- Are RIPK3- and/or MLKL-deficient species unable to induce necroptosis? Do these species rely more on alternative programmed cell death pathways to combat pathogen infection? Or do they have alternative ways to activate necroptosis?
- What could have been the selective force for the absence of RIPK3 and/or MLKL in certain animal species and even whole classes and orders?
- Is the physiological relevance of necroptosis limited to pathogen infection or does it also contribute to human disease?

10 Trends in Cell Biology, Month Year, Vol. xx, No. yy

axis present in human and mouse cells. Altogether, this analysis questions the universal role of a RIPK3-MLKL-dependent necroptotic pathway in innate immunity in the animal kingdom and it may suggest that other forms of regulated necrosis could function as back-up systems in these species.

## Acknowledgments

The authors would like to thank Vojo DERETIC for bringing up the issue of contagious cancer in Tasmanian devil. Research in the Vandenabeele unit is supported by Belgian grants (Interuniversity Attraction Poles, IAP 7/32), Flemish grants (Research Foundation Flanders: FWO G.0875.11, FWO G.0973.11, FWO G.0A45.12N, FWO G.0787.13N, FWO G.0E04.16N), Methusalem grant (BOF09/01M00709 and BOF16/MET_V/007), a grant from the ‘Foundation against Cancer’ (2012-188) and grants from VIB. Research in the Bertrand group is supported by IAP 7/32, FWO grants G.0172.12N and G.013715N. Yvan Saeys is an ISAC Marylou Ingram Scholar.

## References

1. Vanden Berghe, T. *et al.* (2014) Regulated necrosis: the expanding network of non-apoptotic cell death pathways. *Nat. Rev. Mol. Cell Biol.* 15, 135–147
2. Pasparakis, M. and Vandenabeele, P. (2015) Necroptosis and its role in inflammation. *Nature* 517, 311–320
3. Holler, N. *et al.* (2000) Fas triggers an alternative, caspase-8-independent cell death pathway using the kinase RIP as effector molecule. *Nat. Immunol.* 1, 489–495
4. Kalai, M. *et al.* (2002) Tipping the balance between necrosis and apoptosis in human and murine cells treated with interferon and dsRNA. *Cell Death Differ.* 9, 981–994
5. Kaiser, W.J. *et al.* (2013) Toll-like receptor 3-mediated necrosis via TRIF, RIP3, and MLKL. *J. Biol. Chem.* 288, 31268–31279
6. Upton, J.W. *et al.* (2012) DAI/ZBP1/DLM-1 complexes with RIP3 to mediate virus-induced programmed necrosis that is targeted by murine cytomegalovirus vIRA. *Cell Host Microbe* 11, 290–297
7. Upton, J.W. *et al.* (2010) Virus inhibition of RIP3-dependent necrosis. *Cell Host Microbe* 7, 302–313
8. Kaiser, W.J. *et al.* (2013) Viral modulation of programmed necrosis. *Curr. Opin. Virol.* 3, 296–306
9. Sridharan, H. and Upton, J.W. (2014) Programmed necrosis in microbial pathogenesis. *Trends Microbiol.* 22, 199–207
10. Kaczmarek, A. *et al.* (2013) Necroptosis: the release of damage-associated molecular patterns and its physiological relevance. *Immunity* 38, 209–223
11. Cho, Y.S. *et al.* (2009) Phosphorylation-driven assembly of the RIP1-RIP3 complex regulates programmed necrosis and virus-induced inflammation. *Cell* 137, 1112–1123
12. He, S. *et al.* (2009) Receptor interacting protein kinase-3 determines cellular necrotic response to TNF-alpha. *Cell* 137, 1100–1111
13. Zhang, D.W. *et al.* (2009) RIP3, an energy metabolism regulator that switches TNF-induced cell death from apoptosis to necrosis. *Science* 325, 332–336
14. He, S. *et al.* (2011) Toll-like receptors activate programmed necrosis in macrophages through a receptor-interacting kinase-3-mediated pathway. *Proc. Natl. Acad. Sci. U.S.A.* 108, 20054–20059
15. Kaiser, W.J. *et al.* (2008) Receptor-interacting protein homotypic interaction motif-dependent control of NF-kappa B activation via the DNA-dependent activator of IFN regulatory factors. *J. Immunol.* 181, 6427–6434
16. Sun, L. *et al.* (2012) Mixed lineage kinase domain-like protein mediates necrosis signaling downstream of RIP3 kinase. *Cell* 148, 213–227
17. Murphy, J.M. *et al.* (2013) The pseudokinase MLKL mediates necroptosis via a molecular switch mechanism. *Immunity* 39, 443–453
18. Hildebrand, J.M. *et al.* (2014) Activation of the pseudokinase MLKL unleashes the four-helix bundle domain to induce membrane localization and necroptotic cell death. *Proc. Natl. Acad. Sci. U.S.A.* 111, 15072–15077
19. Rodriguez, D.A. *et al.* (2016) Characterization of RIPK3-mediated phosphorylation of the activation loop of MLKL during necroptosis. *Cell Death Differ.* 23, 76–88
20. Cai, Z. *et al.* (2014) Plasma membrane translocation of trimerized MLKL protein is required for TNF-induced necroptosis. *Nat. Cell Biol.* 16, 55–65
21. Chen, X. *et al.* (2014) Translocation of mixed lineage kinase domain-like protein to plasma membrane leads to necrotic cell death. *Cell Res.* 24, 105–121
22. Donderlinger, Y. *et al.* (2014) MLKL compromises plasma membrane integrity by binding to phosphatidylinositol phosphates. *Cell Rep.* 7, 971–981
23. Wang, H. *et al.* (2014) Mixed lineage kinase domain-like protein MLKL causes necrotic membrane disruption upon phosphorylation by RIP3. *Mol. Cell* 54, 133–146
24. Xia, B. *et al.* (2016) MLKL forms cation channels. *Cell Res.* 26, 517–528
25. Buchmann, K. (2014) Evolution of innate immunity: clues from invertebrates via fish to mammals. *Front. Immunol.* 5, 459
26. Kimbrell, D.A. and Beutler, B. (2001) The evolution and genetics of innate immunity. *Nat. Rev. Genet.* 2, 256–267
27. Wiens, G.D. and Glenney, G.W. (2011) Origin and evolution of TNF and TNF receptor superfamilies. *Dev. Comp. Immunol.* 35, 1324–1335
28. Quistad, S.D. *et al.* (2014) Evolution of TNF-induced apoptosis reveals 550 My of functional conservation. *Proc. Natl. Acad. Sci. U.S.A.* 111, 9567–9572
29. Galluzzi, L. *et al.* (2011) Programmed necrosis from molecules to health and disease. *Int. Rev. Cell Mol. Biol.* 289, 1–35
30. McCall, K. (2010) Genetic control of necrosis - another type of programmed cell death. *Curr. Opin. Cell Biol.* 22, 882–888
31. Aggarwal, B.B. (2003) Signalling pathways of the TNF superfamily: a double-edged sword. *Nat. Rev. Immunol.* 3, 745–756
32. Kersse, K. *et al.* (2011) The death-fold superfamily of homotypic interaction motifs. *Trends Biochem. Sci.* 36, 541–552
33. Vercammen, D. *et al.* (1998) Dual signaling of the Fas receptor: initiation of both apoptotic and necrotic cell death pathways. *J. Exp. Med.* 188, 919–930
34. Jouan-Lanhouet, S. *et al.* (2012) TRAIL induces necroptosis involving RIPK1/RIPK3-dependent PARP-1 activation. *Cell Death Differ.* 19, 2003–2014
35. Wang, L. *et al.* (2008) TNF-alpha induces two distinct caspase-8 activation pathways. *Cell* 133, 693–703
36. O'Donnell, M.A. *et al.* (2011) Caspase 8 inhibits programmed necrosis by processing CYLD. *Nat. Cell Biol.* 13, 1437–1442
37. Feng, S. *et al.* (2007) Cleavage of RIP3 inactivates its caspase-independent apoptosis pathway by removal of kinase domain. *Cell. Signal.* 19, 2056–2067
38. Kaiser, W.J. *et al.* (2011) RIP3 mediates the embryonic lethality of caspase-8-deficient mice. *Nature* 471, 368–372
39. Oberst, A. *et al.* (2011) Catalytic activity of the caspase-8-FLIP(L) complex inhibits RIPK3-dependent necrosis. *Nature* 471, 363–367
40. Lin, Y. *et al.* (1999) Cleavage of the death domain kinase RIP by caspase-8 prompts TNF-induced apoptosis. *Genes Dev.* 13, 2514–2526

Trends in Cell Biology

41. Stanger, B.Z. *et al.* (1995) RIP: a novel protein containing a death domain that interacts with Fas/APO-1 (CD95) in yeast and causes cell death. *Cell* 81, 513–523  
42. Sun, X. *et al.* (2002) Identification of a novel homotypic interaction motif required for the phosphorylation of receptor-interacting protein (RIP) by RIP3. *J. Biol. Chem.* 277, 9505–9511  
43. Biton, S. and Ashkenazi, A. (2011) NEMO and RIP1 control cell fate in response to extensive DNA damage via TNF-alpha feedforward signaling. *Cell* 145, 92–103  
44. Dondelinger, Y. *et al.* (2013) RIPK3 contributes to TNFR1-mediated RIPK1 kinase-dependent apoptosis in conditions of cIAP1/2 depletion or TAK1 kinase inhibition. *Cell Death Differ.* 20, 1381–1392  
45. Dondelinger, Y. *et al.* (2015) NF-kappaB-independent role of IKKalpha/IKKbeta in preventing RIPK1 kinase-dependent apoptotic and necrototic cell death during TNF signaling. *Mol. Cell* 60, 63–76  
46. Degterev, A. *et al.* (2008) Identification of RIP1 kinase as a specific cellular target of necrostatins. *Nat. Chem. Biol.* 4, 313–321  
47. McQuade, T. *et al.* (2013) Positive and negative phosphorylation regulates RIP1- and RIP3-induced programmed necrosis. *Biochem.* J. 456, 409–415  
48. Myllymaki, H. *et al.* (2014) The Drosophila imd signaling pathway. *J. Immunol.* 192, 3455–3462  
49. Kleino, A. and Silverman, N. (2014) The Drosophila IMD pathway in the activation of the humoral immune response. *Dev. Comp. Immunol.* 42, 25–35  
50. Buchon, N. *et al.* (2014) Immunity in Drosophila melanogaster - from microbial recognition to whole-organism physiology. *Nat. Rev. Immunol.* 14, 796–810  
51. Vandenabeele, P. and Bertrand, M.J. (2012) The role of the IAP E3 ubiquitin ligases in regulating pattern-recognition receptor signaling. *Nat. Rev. Immunol.* 12, 833–844  
52. Georgel, P. *et al.* (2001) Drosophila immune deficiency (IMD) is a death domain protein that activates antibacterial defense and can promote apoptosis. *Dev. Cell* 1, 503–514  
53. McCarthy, J.V. *et al.* (1998) RIP2 is a novel NF-kappaB-activating and cell death-inducing kinase. *J. Biol. Chem.* 273, 16968–16975  
54. Sun, X. *et al.* (1999) RIP3, a novel apoptosis-inducing kinase. *J. Biol. Chem.* 274, 16871–16875  
55. Yu, P.W. *et al.* (1999) Identification of RIP3, a RIP-like kinase that activates apoptosis and NFkappaB. *Curr. Biol.* 9, 539–542  
56. Kaiser, W.J. and Offermann, M.K. (2005) Apoptosis induced by the toll-like receptor adaptor TRIF is dependent on its receptor interacting protein homotypic interaction motif. *J. Immunol.* 174, 4942–4952  
57. Rebsamen, M. *et al.* (2009) DAI/ZBP1 recruits RIP1 and RIP3 through RIP homotypic interaction motifs to activate NF-kappaB. *EMBO Rep.* 10, 916–922  
58. Li, J. *et al.* (2012) The RIP1/RIP3 necrosome forms a functional amyloid signaling complex required for programmed necrosis. *Cell* 150, 339–350  
59. Cook, W.D. *et al.* (2014) RIPK1- and RIPK3-induced cell death mode is determined by target availability. *Cell Death Differ.* 21, 1600–1612  
60. Orozco, S. *et al.* (2014) RIPK1 both positively and negatively regulates RIPK3 oligomerization and necroptosis. *Cell Death Differ.* 21, 1511–1521  

61. Wu, X.N. *et al.* (2014) Distinct roles of RIP1-RIP3 hetero- and RIP3-RIP3 homo-interaction in mediating necroptosis. *Cell Death Differ.* 21, 1709–1720  
62. Takaoka, A. *et al.* (2007) DAI (DLM-1/ZBP1) is a cytosolic DNA sensor and an activator of innate immune response. *Nature* 448, 501–505  
63. Ishii, K.J. *et al.* (2008) TANK-binding kinase-1 delineates innate and adaptive immune responses to DNA vaccines. *Nature* 451, 725–729  
64. Wu, J. and Chen, Z.J. (2014) Innate immune sensing and signaling of cytosolic nucleic acids. *Annu. Rev. Immunol.* 32, 461–488  
65. Kawasaki, T. and Kawai, T. (2014) Toll-like receptor signaling pathways. *Front. Immunol.* 5, 461  
66. Kajava, A.V. *et al.* (2014) Evolutionary link between metazoan RHIM motif and prion-forming domain of fungal heterokaryon incompatibility factor HET-s/HET-s. *Sci. Rep.* 4, 7436  
67. Wasmer, C. *et al.* (2008) Amyloid fibrils of the HET-s(218-289) prion form a beta solenoid with a triangular hydrophobic core. *Science* 319, 1523–1526  
68. Saupe, S.J. (2011) The [Het-s] prion of *Podospora anserina* and its role in heterokaryon incompatibility. *Semin. Cell Dev. Biol.* 22, 460–468  
69. Wu, J. *et al.* (2013) MLKL knockout mice demonstrate the indispensable role of MLKL in necroptosis. *Cell Res.* 23, 994–1006  
70. Zhao, J. *et al.* (2012) Mixed lineage kinase domain-like is a key receptor interacting protein 3 downstream component of TNF-induced necrosis. *Proc. Natl. Acad. Sci. U.S.A.* 109, 5322–5327  
71. Xie, T. *et al.* (2013) Structural insights into RIP3-mediated necroptotic signaling. *Cell Rep.* 5, 70–78  
72. Quarato, G. *et al.* (2016) Sequential engagement of distinct MLKL phosphatidylinositol-binding sites executes necroptosis. *Mol. Cell* 61, 589–601  
73. Su, L. *et al.* (2014) A plug release mechanism for membrane permeation by MLKL. *Structure* 22, 1489–1500  
74. Tanzer, M.C. *et al.* (2016) Evolutionary divergence of the necroptosis effector MLKL. *Cell Death Differ.* 23, 1185–1197  
75. Daskalov, A. *et al.* (2016) Identification of a novel cell death-inducing domain reveals that fungal amyloid-controlled programmed cell death is related to necroptosis. *Proc. Natl. Acad. Sci. U.S.A.* 113, 2720–2725  
76. Bogren, L.K. *et al.* (2014) Resistance to systemic inflammation and multi organ damage after global ischemia/reperfusion in the arctic ground squirrel. *PloS One* 9, e94225  
77. Linkermann, A. *et al.* (2013) Necroptosis in immunity and ischemia-reperfusion injury. *Am. J. Transplant.* 13, 2797–2804  
78. Murchison, E.P. (2008) Clonally transmissible cancers in dogs and Tasmanian devils. *Oncogene* 27 (Suppl 2), S19–S30  
79. Zhang, T. *et al.* (2016) CaMKII is a RIP3 substrate mediating ischemia- and oxidative stress-induced myocardial necroptosis. *Nat. Med.* 22, 175–182  
80. Silke, J. *et al.* (2015) The diverse role of RIP kinases in necroptosis and inflammation. *Nat. Immunol.* 16, 689–697  
81. Tanzer, M.C. *et al.* (2015) Necroptosis signalling is tuned by phosphorylation of MLKL residues outside the pseudokinase domain activation loop. *Biochem. J.* 471, 255–265  
