
LETTER

Toremifene interacts with and destabilizes the Ebola virus glycoprotein

Yuguang Zhao${}^{1*}$, Jingshan Ren${}^{1*}$, Karl Harlos${}^{1}$, Daniel M. Jones${}^{1}$, Antra Zeltina${}^{1}$, Thomas A. Bowden${}^{1}$, Sergi Padilla-Parra${}^{1,2}$, Elizabeth E. Fry${}^{1}$ & David I. Stuart${}^{1,3}$

Ebola viruses (EBOVs) are responsible for repeated outbreaks of fatal infections, including the recent deadly epidemic in West Africa. There are currently no approved therapeutic drugs or vaccines for the disease. EBOV has a membrane envelope decorated by trimers of a glycoprotein (GP, cleaved by furin to form GP1 and GP2 subunits), which is solely responsible for host cell attachment, endosomal entry and membrane fusion ${}^{1-7}$. GP is thus a primary target for the development of antiviral drugs. Here we report the first, to our knowledge, unliganded structure of EBOV GP, and high-resolution complexes of GP with the anticancer drug toremifene and the painkiller ibuprofen. The high-resolution apo structure gives a more complete and accurate picture of the molecule, and allows conformational changes introduced by antibody and receptor binding to be deciphered ${}^{8-10}$. Unexpectedly, both toremifene and ibuprofen bind in a cavity between the attachment (GP1) and fusion (GP2) subunits at the entrance to a large tunnel that links with equivalent tunnels from the other monomers of the trimer at the three-fold axis. Protein-drug interactions with both GP1 and GP2 are predominately hydrophobic. Residues lining the binding site are highly conserved among filoviruses except Marburg virus (MARV), suggesting that MARV may not bind these drugs. Thermal shift assays show up to a 14°C decrease in the protein melting temperature after toremifene binding, while ibuprofen has only a marginal effect and is a less potent inhibitor. These results suggest that inhibitor binding destabilizes GP and triggers premature release of GP2, thereby preventing fusion between the viral and endosome membranes. Thus, these complex structures reveal the mechanism of inhibition and may guide the development of more powerful anti-EBOV drugs.

The recent outbreak of EBOV in West Africa, the worst of more than 30 in the past 40 years, comprised more than 28,000 cases and over 11,000 deaths ${}^{11}$. In the urgent need to find therapeutics, many small compounds and existing Food and Drug Administration (FDA)-approved drugs have been screened *in vitro* or *in silico* (for example, ibuprofen was suggested by docking experiments ${}^{12}$) to find lead compounds for drug development or repurpose drugs for the treatment of EBOV disease ${}^{13-16}$. Among these, a set of selective oestrogen receptor modulators (SERMs) stand out as potential inhibitors from *in vitro* and *in vivo* studies ${}^{14}$; however, their mechanism of action remains largely unknown. Using recombinant EBOV GP we tested whether nine such compounds could directly bind by a thermal-shift assay (Methods). The results show that toremifene in particular markedly decreases the melting temperature ($T_{\mathrm{m}}$) of EBOV GP, by up to 14°C at 100 μM (Fig. 1). This contrasts with the action of inhibitors on most protein targets, which tend to increase stability ${}^{17}$, although destabilization has been reported before ${}^{18}$. Benzotropine ${}^{19}$, the G-protein-coupled receptor (GPCR) antagonist, also decreases the $T_{\mathrm{m}}$ of GP by 4°C, while other compounds, including ibuprofen, showed $T_{\mathrm{m}}$ shifts of less than 2°C (Fig. 1, Extended Data Fig. 1). The destabilization effects of toremifene and ibuprofen are both pH and concentration dependent (Fig. 1). The binding constants ($K_{\mathrm{d}}$ values) determined by this assay are 16 μM for toremifene and 6 mM for ibuprofen (Extended

![Figure 1](#fig1)

**Figure 1 | Summary of thermal-shift assays.** **a,** The effects of toremifene and ibuprofen on the melting temperature of EBOV GP at different pHs. The raw fluorescence traces are shown in Extended Data Fig. 1. The protein melting temperature at pH 5.2 at which the crystals were grown is taken as the reference point. **b,** The melting temperatures of EBOV GP at different concentrations of toremifene or ibuprofen, at pH 5.2. Data are mean ± s.d. ($n = 3$).

---

${}^{1}$Division of Structural Biology, University of Oxford, Wellcome Trust Centre for Human Genetics, Headington, Oxford OX3 7BN, UK. ${}^{2}$Cellular Imaging Core, University of Oxford, Wellcome Trust Centre for Human Genetics, Headington, Oxford OX3 7BN. ${}^{3}$Diamond Light Source Ltd, Harwell Science & Innovation Campus, Didcot OX11 0DE, UK.

*These authors contributed equally to this work.

RESEARCH LETTER

Figure 2 | Overall structure. **a**, Cartoon diagram of EBOV GP monomer; GP1 is in blue, GP2 is in red, and the glycan cap is in cyan. Secondary structural elements named as described previously<sup>8</sup>. Disulfide bonds shown as orange sticks, glycans in grey. The mucin domain omitted in our construct is shown as a yellow oval. FL, fusion loop. The C-terminal inserted foldon trimerization domain is disordered. **b**, The biological trimer viewed perpendicular to the threefold axis with one monomer coloured as in **a** and the second and third faded for clarity. **c**, The trimer viewed along the threefold, towards the viral membrane. **d**, Close up of the inhibitor binding site. **e**, Close up of the glycan cap and receptor binding site. Areas shown in **d** and **e** are indicated in panel **a**. In **d** and **e** antigenic sites are coloured grey and the receptor binding site yellow.

Data Fig. 1). In a mouse model<sup>14</sup>, toremifene appears to be even more potent (half-maximum inhibitory concentration (IC<sub>50</sub>) ~1 μM).

The crystal structure of unliganded EBOV GP was determined at 2.2 Å resolution, with good R-factors and stereochemistry (Extended Data Table 1). Three copies each of the GP1 and GP2 subunits (Fig. 2a) form the biological trimer around the crystallographic three-fold axis (Fig. 2b, c). This structure, although crystallized at pH 5.2, represents the pre-fusion state of the molecule, with the GP1 receptor-binding site blocked by a glycan cap (Fig. 2e). GP1 is predominantly composed of β-strands, forming a large semi-circular groove at the centre of the subunit that clamps the α3 helix and β19–β20 strands of GP2 (Fig. 2d). The glycan cap is removed in the late endosome by cathepsin B/L to expose the receptor Niemann–Pick disease type C1 (NPC1)-binding site<sup>9,20,21</sup>. GP2 catalyses membrane fusion and contains N-terminal (α3 and α4) and C-terminal (α5) heptad repeats linked by a CX<sub>6</sub>CC motif (residues 601–609, Fig. 2a). The C-terminal heptad repeat, disordered in all previously published GP structures<sup>8–10</sup>, contributes to the trimer interface in our structure (Fig. 2b, c) and contains N618, which is glycosylated as predicted. The well-ordered CX<sub>6</sub>CC motif forms intrasubunit (C601–C608), and intersubunit (C53–C609) disulfide bonds (Fig. 3). Mutation of any of these cysteine residues renders the virus incapable of entering host cells<sup>22</sup>. In the fusion process, GP2 undergoes conformational changes in which α5 refolds onto a helix coalesced from α3 and α4 to form a six-helix bundle<sup>23</sup> (Extended Data Fig. 2). In our pre-fusion structure, the hydrophobic GP2 fusion loop (residues 511–554) (Fig. 2a) projects into a neighbouring monomer and is stabilized in a shallow depression surrounded by loops β4–β5 and β10–β11, and α3. Apart from residues 521–526, which have very loose interactions with the rest of the protein, the fusion loop in this pH 5.2 apo GP is very similar to that in the KZ52 Fab complex crystallized at pH 8.3 (Extended Data Fig. 3). This is in contrast to the large conformational changes reported for the isolated fusion loop at different pHs (ref. 24), suggesting that GP1 maintains GP2 in the pre-fusion state until their separation triggered by receptor binding.

In total, 319 out of 394 Cα atoms in our apo GP structure match with the GP–Fab complex<sup>8</sup>, with a root mean squared deviation (r.m.s.d.) value of 1.1 Å (Fig. 3a–d), however, there are marked differences, beyond the C-terminal heptad repeat and CX<sub>6</sub>CC motif. The β1–β2 hairpin interacts directly with the KZ52 Fab and is pushed 6 Å inwards in the Fab complex (Fig. 3c). The glycan cap is better ordered in the apo structure, revealing an extra strand, β18′, inserted between β17 and β18, overlapping β18 in the Fab complex but running in the opposite direction (Fig. 3b). Several cross-reactive neutralizing monoclonal antibodies from EBOV survivors bind to the cap<sup>25</sup>, suggesting that this conserved epitope is important for antibody-mediated clearance of the virus. Our structure defines this conformational epitope.

In the apo GP structure (excluding the glycan cap), 230 Cα atoms overlay with the GP in the NPC1 receptor complex<sup>9</sup>, with an r.m.s.d. of 0.9 Å (Fig. 3e). NPC1 binding draws helix α1 approximately 2 Å towards the receptor, causing the preceding 3<sub>10</sub>-helix α1′ to unwind, disrupting interactions with α3 of GP2, as described previously<sup>9</sup> (Fig. 3f). These structural changes also break hydrogen bonds from α1′ to the amide groups of residues K510 and N512, disordering the N-terminal residues 502–507 of GP2. The structural differences continue to the other side of α3. In the NPC1-bound structure, the α3 helix starts two residues earlier and the β1–β2 hairpin bends inwards, adopting a conformation similar to that in the KZ52 Fab complex (Fig. 3c, g). In addition, there is a large tunnel between neighbouring monomers (Fig. 4, Extended Data Fig. 4), whose hydrophobic entrance is formed by surrounding residues from the β1–β2 hairpin, β3, β6 and β13 of GP1, and α3, β19 and β20 of GP2. Residues 192–195 (with sequence DFFS, and named DFF lid thereafter) form a tight turn with F193 and F194 plugging the entrance of the tunnel and making tight

Data Table 1). Both inhibitors have well-defined electron density and bind at the same site at the entrance of the large tunnel by expelling the DFF lid (Fig. 4 and Extended Data Figs 4, 5, 7). In addition to the tunnel entrance residues already mentioned, the tunnel is lined by residues from the N-terminal loop, the β1–β2 hairpin, β2–β3 loop of GP1, and α3 and α4 of GP2 from a neighbouring monomer, and interconnected with the other two tunnels in the trimer at the three-fold axis (Fig. 4b and Extended Data Fig. 4). Y517 makes dominant interactions with toremifene by contacting all three phenyl rings (Fig. 4c). In addition, phenyl ring A of toremifene is fully buried and interacts with V66, L68, L515 and L558, ring B with L186, and ring C with V66 and A101. The ethyl chloride group interacts with L184, L186, M548 and L558, while the dimethylethanamine group points towards the main tunnel and is surrounded by polar/charged residues, including R64, E100, T519, T520 and D522 (Fig. 4c).

Ibuprofen is bound with its isobutyl group partially overlapping the ethyl chloride group of the toremifene but closer to L554. However, its phenyl ring does not overlap any of the rings of toremifene (Extended Data Fig. 5), but makes extensive interactions with M548 (Fig. 4d). The propanoic acid moiety is orientated to make a hydrogen bond to the side chain of R64 and hydrophobic interactions with Y517. Remarkably, ibuprofen was initially suggested to interact with EBOV GP by *in silico* screening, and predicted to dock in a pocket of the mucin domain¹². A racemic mixture of ibuprofen was used for all experiments, however, we note that the S-isomer (which is also active as a painkiller) binds preferentially.

The flexible region, 521–526, of the fusion loop is stabilized in the two inhibitor-bound structures, but in different conformations compared to apo GP. The most notable conformation changes induced by toremifene are at side chains of M548 and L554, and M548 by ibuprofen (Fig. 4). The residues involved in inhibitor binding are highly conserved across filoviruses, with the exception of MARV (Extended Data Fig. 8), where the DFF lid and its preceding loop are replaced by a helix, and V66 and A101 are substituted by M50 and E85, respectively, partially blocking the binding site³⁰.

The SERMs tamoxifen, 4-hydroxytamoxifen and clomiphene are less potent inhibitors, despite their chemical similarity to toremifene¹⁴,¹⁵. Compared to the ethyl chloride group of toremifene, the corresponding ethyl group in tamoxifen and chlorine in clomiphene are expected to make weaker interactions with L184, L186, M548 and L558. A partially bound 4-hydroxytamoxifen structure obtained by crystal soaking

Figure 3 | Structure comparisons. **a**, Structure of the apo GP monomer compared with the GP of the KZ52 Fab complex. **b**–**d**, Details of the structural differences at the glycan cap (**b**), β1–β2 hairpin (**c**) and the CX₆CC motif (**d**). The apo GP is coloured as in Fig. 1a, the GP in complex with Fab grey. **e**, Comparison of apo GP with GP from the GP–receptor complex shown in same style as **a**. **f**, **g**, close up views of the major structural differences at α1′ and α1 helices (**f**), and β1–β2 hairpin (**g**).

interactions with β19 and β20 (Fig. 4a and Extended Data Fig. 5a). This structure may also hold the putative cleavage site residue 190 in position (Extended Data Fig. 5a)—in the endosome the glycan cap is cleaved to free the receptor-binding site of GP1 (refs 25–29), which also exposes the entrance of the tunnel. Receptor binding is proposed to trigger unwinding of GP2 from GP1 and subsequently lead to membrane fusion²³. The above structural changes resulting from receptor binding probably all contribute to weaken GP1–GP2 interactions. Both α1′ and α1 are shielded by residues 287–293 and the N563 glycan (which is resistant to enzymatic deglycosylation), perhaps preventing premature release of GP2 (Extended Data Fig. 6). Structures of GP–toremifene and GP–ibuprofen complexes were obtained by crystal soaking and refined to 2.7 Å resolution (Extended

Figure 4 | Inhibitor-binding site. **a**, Details of the inhibitor-biding site in the apo GP. The backbone is shown as ribbons with GP1 in blue and GP2 in red, side chains as grey sticks. **b**, Tunnels of the GP trimer viewed along the three-fold axis towards the viral membrane. Toremifenes bound at the entrances of the tunnels are shown as yellow sticks. **c**, **d**, Details of protein–inhibitor interactions of the GP–toremifene complex (**c**) and GP–ibuprofen complex (**d**). Toremifene is shown as yellow sticks, ibuprofen as cyan sticks. Protein main chains are shown as ribbons and side chains as sticks (GP1 blue, GP2 red). Side chains in the apo structure with large conformational changes are shown as thinner grey sticks.
(RESEARCH LETTER)

(data not shown) shows the hydroxyl group makes close contacts with G67, shifting the whole inhibitor ~1.0 Å towards the solvent, weakening ring-stacking interactions with Y517 and having no contacts from the ethyl group to L184, L186 and L558 compared to toremifene. Our crystallographic results are in line with the inhibition data\(^{14-16}\) and our thermal-shift assay (Extended Data Fig. 1e, f). If toremifene and ibuprofen inhibit viral infection by causing premature conversion of GP to the post-fusion conformation or blocking receptor binding, we would expect them to abolish viral fusion. This was confirmed by measuring their effect on the fusion of EBOV GP pseudoviruses as judged by a β-lactamase reporter assay (Extended Data Fig. 9). Benzotriptine, which decreases the \(T_{\mathrm{m}}\) of GP by 4°C, could not be soaked in our crystals, and needs further investigation (Extended Data Fig. 1f).

Our results pinpoint the binding site of toremifene and ibuprofen on the surface of the GP and reveal that they decrease the stability of the viral GP, and prevent viral fusion. The binding site is different to that predicted for ibuprofen\(^{12}\), and the information on the binding modes of these compounds and the spare volume at the binding cavity can guide the design of more potent compounds. Finally, our readily grown well-diffracting crystals are suitable for fragment-based screening for different classes of binders for the development of new inhibitors to combat EBOV infection.

**Online Content** Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.

Received 16 March; accepted 26 May 2016.  
Published online 29 June 2016.

1. Takada, A. *et al.* A system for functional analysis of Ebola virus glycoprotein. *Proc. Natl Acad. Sci. USA* **94**, 14764–14769 (1997).
2. Hacke, M. *et al.* Inhibition of Ebola virus glycoprotein-mediated cytotoxicity by targeting its transmembrane domain and cholesterol. *Nat. Commun.* **6**, 7688 (2015).
3. Aleksandrowicz, P. *et al.* Ebola virus enters host cells by macropinocytosis and clathrin-mediated endocytosis. *J. Infect. Dis.* **204** (suppl. 3), S957–S967 (2011).
4. Carette, J. E. *et al.* Ebola virus entry requires the cholesterol transporter Niemann-Pick C1. *Nature* **477**, 340–343 (2011).
5. Côté, M. *et al.* Small molecule inhibitors reveal Niemann-Pick C1 is essential for Ebola virus infection. *Nature* **477**, 344–348 (2011).
6. Nanbo, A. *et al.* Ebolavirus is internalized into host cells *via* macropinocytosis in a viral glycoprotein-dependent manner. *PLoS Pathog.* **6**, e1001121 (2010).
7. Saeed, M. F., Kolokoltsov, A. A., Albrecht, T. & Davey, R. A. Cellular entry of Ebola virus involves uptake by a macropinocytosis-like mechanism and subsequent trafficking through early and late endosomes. *PLoS Pathog.* **6**, e1001110 (2010).
8. Lee, J. E. *et al.* Structure of the Ebola virus glycoprotein bound to an antibody from a human survivor. *Nature* **454**, 177–182 (2008).
9. Wang, H. *et al.* Ebola viral glycoprotein bound to its endosomal receptor Niemann-Pick C1. *Cell* **164**, 258–268 (2016).
10. Dias, J. M. *et al.* A shared structural solution for neutralizing ebolaviruses. *Nat. Struct. Mol. Biol.* **18**, 1424–1427 (2011).
11. World Health Organization. *Ebola Situation Reports*; http://www.who.int/csr/disease/ebola/situation-reports/en/ (2016).
12. Veljkovic, V. *et al.* In silico analysis suggests repurposing of ibuprofen for prevention and treatment of EBOLA virus disease. *F1000Res.* **4**, 104 (2015).
13. Edwards, M. R. *et al.* High-throughput minigenome system for identifying small-molecule inhibitors of Ebola virus replication. *ACS Infect. Dis.* **1**, 380–387 (2015).
14. Johansen, L. M. *et al.* FDA-approved selective estrogen receptor modulators inhibit Ebola virus infection. *Sci. Transl. Med.* **5**, 190ra79 (2013).
15. Johansen, L. M. *et al.* A screen of approved drugs and molecular probes identifies therapeutics with anti-Ebola virus activity. *Sci. Transl. Med.* **7**, 290ra89 (2015).
16. Kouznetsova, J. *et al.* Identification of 53 compounds that block Ebola virus-like particle entry via a repurposing screen of approved drugs. *Emerg. Microbes Infect.* **3**, e84 (2014).
17. Lea, W. A. & Simeonov, A. Differential scanning fluorometry signatures as indicators of enzyme inhibitor mode of action: case study of glutathione S-transferase. *PLoS One* **7**, e36219 (2012).
18. Cimperman, P. *et al.* A quantitative model of thermal stabilization and destabilization of proteins by ligands. *Biophys. J.* **95**, 3222–3231 (2008).
19. Cheng, H. *et al.* Inhibition of Ebola and Marburg virus entry by G protein-coupled receptor antagonists. *J. Virol.* **89**, 9932–9938 (2015).
20. Brecher, M. *et al.* Cathepsin cleavage potentiates the Ebola virus glycoprotein to undergo a subsequent fusion-relevant conformational change. *J. Virol.* **86**, 364–372 (2012).
21. Zhao, Y., Ren, J., Harlos, K. & Stuart, D. I. Structure of glycosylated NPC1 luminal domain C reveals insights into NPC2 and Ebola virus interactions. *FEBS Lett.* **590**, 605–612 (2016).
22. Jeffers, S. A., Sanders, D. A. & Sanchez, A. Covalent modifications of the ebola virus glycoprotein. *J. Virol.* **76**, 12463–12472 (2002).
23. Weissenhorn, W., Carfí, A., Lee, K. H., Skehel, J. J. & Wiley, D. C. Crystal structure of the Ebola virus membrane fusion subunit, GP2, from the envelope glycoprotein ectodomain. *Mol. Cell* **2**, 605–616 (1998).
24. Gregory, S. M. *et al.* Structure and function of the complete internal fusion loop from Ebolavirus glycoprotein 2. *Proc. Natl Acad. Sci. USA* **108**, 11211–11216 (2011).
25. Flyak, A. I. *et al.* Cross-reactive and potent neutralizing antibody responses in human survivors of natural Ebolavirus infection. *Cell* **164**, 392–405 (2016).
26. Chandran, K., Sullivan, N. J., Felbor, U., Whelan, S. P. & Cunningham, J. M. Endosomal proteolysis of the Ebola virus glycoprotein is necessary for infection. *Science* **308**, 1643–1645 (2005).
27. Dube, D. *et al.* The primed ebolavirus glycoprotein (19-kilodalton GP1,2): sequence and residues critical for host cell binding. *J. Virol.* **83**, 2883–2891 (2009).
28. Hood, C. L. *et al.* Biochemical and structural characterization of cathepsin L-processed Ebola virus glycoprotein: implications for viral entry and immunogenicity. *J. Virol.* **84**, 2972–2982 (2010).
29. Schornberg, K. *et al.* Role of endosomal cathepsins in entry mediated by the Ebola virus glycoprotein. *J. Virol.* **80**, 4174–4178 (2006).
30. Hashiguchi, T. *et al.* Structural basis for Marburg virus neutralization by a cross-reactive human antibody. *Cell* **160**, 904–912 (2015).

**Acknowledgements** We thank Diamond scientists at I02 and I03 for assistance with data collection, T. S. Walter for help with crystallization and thermal-shift essay. Y.Z. was supported by the Biostruct-X project (283570) funded by the EU seventh Framework Programme (FP7), J.R. by the Wellcome Trust, and D.I.S., E.E.F. and K.H. by the UK Medical Research Council (MR/N00065X/1). This is a contribution from the UK Instruct Centre. The Wellcome Trust Centre for Human Genetics is supported by the Wellcome Trust (grant 090532/Z/09/Z). A.Z. is supported by a Marie Curie Fellowship (658363), T.A.B. is supported by the MRC (MR/L009528/1). SP-P is funded by a Nuffield Department of Medicine Leadership Fellowship. D.I.S. is a Jenner Investigator.

**Author Contributions** Y.Z., J.R. and D.I.S designed the project. Y.Z. made the protein and grew the crystals together with J.R., collected X-ray data and determined the structures. K.H. helped with crystal mounting and data collection. D.M.J. and S.P. carried out cell imaging experiments. A.Z. and T.A.B. provided the cDNA. Y.Z., J.R., E.E.F. and D.I.S. analysed the results and wrote the manuscript in discussions with all authors.

**Author Information** The atomic coordinates and structure factors have been deposited with the RCSB Protein Data Bank under accession codes 5JQ3 (native GP), 5JQ7 (GP-toremifene) and 5JQB (GP-ibuprofen). Reprints and permissions information is available at www.nature.com/reprints. The authors declare competing financial interests: details are available in the online version of the paper. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to D.I.S. (dave@strubi.ox.ac.uk).

**Reviewer Information** Nature thanks E. Saphire, W. Weissenhorn and the other anonymous reviewer(s) for their contribution to the peer review of this work.
METHODS

No statistical methods were used to predetermine sample size. The experiments were not randomized, and investigators were not blinded to allocation during experiments and outcome assessment.

Protein cloning, expression and purification. Zaire EBOV (strain Mayinga-76) glycoprotein extracellular domain DNA was synthesized (UniProt entry KB-Q05320). The expression construct GPΔ contains two directly linked sections, amino acids 32–312 and 464–632, with a T42A mutation to eliminate N40 glycosylation. At the N terminus of the protein, the four amino acids ETGR were added from the expression vector pNeosec<sup>31</sup>. At the C terminus, a foldon trimerization sequence from the bacteriophage T4 fibritin and a 6× His tag were added with the sequence: GSGYIPEAPRDGQAYVRKDGEWVLLSTFLGTHHHHHH. The endotoxin free pNeosec-GPΔ plasmid was transiently transfected into the human embryonic kidney HEK293T (ATCC CRL11268) cells with polyethylenimine (PEI, molecular mass 25kDa, Sigma). To inhibit the formation of complex glycosylation, the mannosidase inhibitor kifunensine (Cayman Chemical) was added to a final concentration of 5 μM. After 5 days of transfection, the conditioned media was collected, dialysed against PBS and incubated with talon beads (Takara Bio Europe SAS) at 15°C for 1 h with gentle shaking. The beads were collected and washed with PBS plus 5–10 mM imidazole. The protein was eluted with 200 mM imidazole in PBS and further purified by size exclusion chromatography with a Superdex 200 HiLoad 16/600 column (GE Healthcare) and a buffer of 10 mM MES, pH 5.2, 150 mM NaCl.

Thermal-shift assay. 10 μl of solution containing 2 μM glycosylated EBOV GP protein, buffered by the addition of 10 μl of 850 mM sodium malonate at the desired pH, was mixed with 10 μl of 15× SYPRO Orange dye (Thermo Fisher Scientific), along with 10 μl of 50 μM compound in 10% DMSO or just 10% DMSO. The mixture was made up to a total volume of 50 μl. Samples were placed in a semi skirted 96-well PCR plate (4 Titude), sealed and heated in an Mx3005p qPCR machine (Stratagene, Agilent Technologies) from 24.5 to 98.5°C at a rate of 1°C min<sup>−1</sup>. Fluorescence changes were monitored with excitation and emission wavelengths at 492 and 610 nm, respectively. Reference wells, that is, solutions without chemical, but with the same amount of DMSO, were used to compare the melting temperature (*T*<sub>m</sub>). Experiments were carried out in triplicate. Compounds tested included toremifene, tamoxifen, 4-hydroxytamoxifen, anastrozole, benzotropine, clomipramine, ibuprofen, diacylglycerol kinase inhibitor and benzodiazepine. The SERMs endoxifen and clomiphene could not be tested since they bind to SYPRO Orange directly.

Ebola pseudovirus production and titration. HIV-1-derived pseudoviruses expressing the Ebola virus envelope glycoproteins (EBOV pseudoparticle, EBOVpp) were produced as described previously<sup>32</sup>. HEK-293T cells in T175 flasks were transfected with 2 μg pR8ΔEnv, 2 μg BlaM-Vpr, 1 μg pcREV and 3 μg of plexm-EBOV_GP plasmids (containing Zaire EBOV GP residues 1–676 under control of β-actin/CMV chimaeric promoter). After 8 h of transfection, the medium was replaced by fresh DMEM with 10% FBS. Viral supernatants were collected and concentrated using the Lenti-X Concentrator (Clontech). Virus titres were determined by infecting TZM-bl cells (PTA-5659, no mycoplasma contamination) with a serial dilution of concentrated pseudovirus followed by a β-Gal assay. Since the TZM-bl cells contain a β-Gal expression cassette with an HIV-1-induced promoter infected cells can be identified through hydrolysis of X-gal<sup>33</sup>.

BlaM assay and analysis. The β-lactamase assay<sup>33</sup> was applied to assess EBOVpp fusion. 24 h before the assay TZM-bl cells were plated at 4 × 10<sup>4</sup> cells per well in black clear-bottomed 96-well plates. On the day of assay, cells were cooled on ice before the addition of EBOVpp (MOI 0.5), then centrifuged at 2,100g for 30 min at 4°C. Viral supernatants were removed and cells washed with PBS. Then, 100 μl of DMEM plus 10% FBS containing toremifene (15 μM or 1.5 μM), ibuprofen (150 μM or 15 μM), or the same amount of solvent for the above compounds (5% DMSO) was added to each well before placing in a 37°C, CO<sub>2</sub> incubator to initiate viral entry. After 120 min, cells were loaded with 1× CCF2-AM from the LiveBLAzer FRET—B/G Loading Kit (Life Technologies) and incubated at room temperature in the dark for 2 h. After CCF2-AM removal, cells were washed with PBS and fixed with 2% PFA before viewing. Cells were excited using a 405 nm continuous laser (Leica) and the emission spectra between 430 and 560 nm were recorded pixel by pixel (512 × 512) using a Leica SP8 X-SMD microscope with a 20× objective. The ratio of blue emission (440–480 nm, cleaved CCF2-AM) to green (500–540 nm, uncleaved CCF2-AM) was then calculated pixel by pixel using a customized macro<sup>34</sup> for ImageJ (http://imagej.nih.gov/ij/) with ten different observation fields for each condition. A blue/green threshold (fusion threshold) was set using HIV<sub>NoEnv</sub> virions containing Vpr-BlaM as a background control to provide a fusion detection limit that corresponded to 0.75 ± 0.05 BlaM ratio. The fusion threshold was calculated recovering the signal (blue/green intensity ratio) coming from individual cells plus 2× s.d. from ~200 cells in the observation field. This threshold was then applied to all conditions. Cells above the threshold were pseudocoloured in red and cells below the threshold pseudocoloured in blue. ‘Red’ cells were then compared with blue cells (non-fusogenic) as an accurate measure of fusion in different conditions.

Crystallization and inhibitor soaking. For crystallization, the GP was treated with endo-β-acetylglucosaminidase F1 at 37°C for 1 h, further purified with size-exclusion chromatography and concentrated to 10–12 mg ml<sup>−1</sup>. Crystallization screen experiments were carried out using the nanolitre sitting-drop vapour diffusion method in 96-well plates as previously described<sup>35</sup>. Crystals were initially obtained from Hampton Research PEGRx 1 screen, condition 37 containing 10% (w/v) PEG 6,000 and 0.1 M sodium citrate tribasic dihydrate (pH 5.0). The best crystals were grown in the optimized condition containing 9% (w/v) PEG 6,000 and 0.1 M sodium citrate tribasic dihydrate at pH 5.2.

To obtain GP–inhibitor complexes, crystal-soaking experiments were performed. Crystal-soaking solutions were prepared by first dissolving the inhibitors in 100% DMSO and then diluting with 15% (w/v) PEG 6,000 and 0.1 M sodium citrate tribasic dihydrate (pH 5.0) to a final DMSO concentration of 10%. The inhibitor concentration was typically from 1 to 10 mM depending on solubility. Crystals were soaked in the above solutions for between 20 min and 5 h. Crystals soaked for longer normally diffracted less well and the crystals from which the GP–toremifene and GP–ibuprofen complex structures were obtained were only soaked for 20 min.

Data collection and structure determination. Crystals were cryo-protected using solutions containing 75% crystallization liquor (or inhibitor soaking solution) and 25% (v/v) glycerol and frozen in liquid nitrogen before data collection. All data were collected from frozen crystals at 100 K. Data were acquired as 0.1° images on PILATUS 6M detectors at Diamond Light Source, using beamline I03 for native data (exposure time 0.1 s per frame, beam size 80 × 20 μm and 30% beam transmission), and I02 for inhibitor soaked crystals (exposure time 0.05 s per frame, beam size 90 × 25 μm and 40% beam transmission). Diffraction images were indexed, integrated and scaled with the automated data processing program Xia2-3dii<sup>36</sup>. The native data set was collected from four crystals to 2.23 Å resolution with 58-fold redundancy. A total of 7 inhibitors were soaked, including toremifene, tamoxifen, 4-hydroxytamoxifen, raloxifene, clomiphene, ibuprofen and benzitropine, and diffraction data were collected with resolutions ranging from 3.5 to 2.3 Å.

The crystals belong to space group *R*32 with unit cell dimensions *a* = *b* = 114.0 Å and *c* = 307.0 Å approximately. The apo structure was determined by molecular replacement with MOLREP<sup>37</sup> using the GP structure of the GP-KZ52 Fab complex (PDB ID, 3CSY) as a search model. There is one GP molecule in the crystal asymmetric unit. The biological trimer is formed by a crystallographic threefold axis. Structure refinement used REFMAC<sup>38</sup> and models were rebuilt with COOT<sup>39</sup>. The apo structure was refined to 2.23 Å resolution with an *R*<sub>work</sub> value of 0.223 (*R*<sub>free</sub> = 0.251) and good stereochemistry. Close examination of the data from inhibitor-soaked crystals showed that only toremifene and ibuprofen were fully bound with GP, and structures were refined to resolutions of 2.69 Å and 2.68 Å, respectively. 4-hydroxytamoxifen was only bound with partial occupancy (data not shown). Data collection and structure refinement statistics are given in Extended Data Table 1. Structural comparisons used SHP<sup>40</sup>, figures were prepared with PyMOL<sup>41</sup>.

---

31. Zhao, Y., Ren, J., Padilla-Parra, S., Fry, E. E. & Stuart, D. I. Lysosome sorting of β-glucocerebrosidase by LIMP-2 is targeted by the mannose 6-phosphate receptor. *Nat. Commun.* **5**, 4321 (2014).

32. Desai, T. M. *et al*. IFITM3 restricts influenza A virus entry by blocking the formation of fusion pores following virus-endosome hemifusion. *PLoS Pathog.* **10**, e1004048 (2014).

33. Cavrois, M., De Noronha, C. & Greene, W. C. A sensitive and specific enzyme-based assay detecting HIV-1 virion fusion in primary T lymphocytes. *Nat. Biotechnol.* **20**, 1151–1154 (2002).

34. Jones, D. M. & Padilla-Parra, S. Imaging real-time HIV-1 virion fusion with FRET-based biosensors. *Sci. Rep.* **5**, 13449 (2015).

35. Walter, T. S. *et al*. A procedure for setting up high-throughput nanolitre crystallization experiments. Crystallization workflow for initial screening, automated storage, imaging and optimization. *Acta Crystallogr. D* **61**, 651–657 (2005).

36. Winter, G., Lobley, C. M. & Prince, S. M. Decision making in xia2. *Acta Crystallogr. D* **69**, 1260–1273 (2013).

37. Vagin, A. & Teplyakov, A. Molecular replacement with MOLREP. *Acta Crystallogr. D* **66**, 22–25 (2010).

38. Winn, M. D., Murshudov, G. N. & Papiz, M. Z. Macromolecular TLS refinement in REFMAC at moderate resolutions. *Methods Enzymol.* **374**, 300–321 (2003).

39. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. *Acta Crystallogr. D* **60**, 2126–2132 (2004).

40. Stuart, D. I., Levine, M., Muirhead, H. & Stammers, D. K. Crystal structure of cat muscle pyruvate kinase at a resolution of 2.6 Å. *J. Mol. Biol.* **134**, 109–142 (1979).

41. DeLano, W. L. & Lam, J. W. PyMOL: A communications tool for computational models. *Abstr. Pap. Am. Chem. Soc.* **230**, U1371–U1372 (2005).

RESEARCH LETTER

Extended Data Figure 1 | Thermal-shift essay. Representative thermal melt curves of EBOV GP with 10 μM compounds and 2% DMSO.

a–d, Melting curves of EBOV GP with toremifene, ibuprofen or protein alone at pH 5.0, 6.0, 7.0 and 8.0, respectively. e, Small effects of SERM inhibitors tamoxifen, 4-hydroxytamoxifen and raloxifen on the melting temperature of EBOV GP shown at pH 5.2. f, Melt curves of EBOV GP with diglycerol kinase inhibitor, anastrozole and benzotropine mesylate at pH 5.2. g, h, Shifts in melting temperature (ΔTm °C in absolute value) were plotted against different concentrations of toremifene (g) or ibuprofen (h) at pH 5.2. Data are mean ± s.d. (n = 4). The affinity constant Kd is calculated by a ligand binding 1:1 saturation fitting with the SigmaPlot version 13 (Systat Software Inc.).

© 2016 Macmillan Publishers Limited. All rights reserved

a

S—S
53 108 135

SP | Glycan cap | Mucin | FL NHR | CHR | TM |
1 32 312 464 632 676

S—S
511 556

S—S
121 147

190 Putative
cathepsin cleavage

Furin cleavage
501-502

S—S
601 608

GPA:
32 312 464 632

GP1 GP2

b

FL

β20
β19
α3

α4

CX₆CC

α5

c

FL

β20
β19
α3

α4

CX₆CC

α5

Extended Data Figure 2 | Structural organization of EBOV GP and GP2 structure. a, Scheme showing the structural organization of EBOV GP. FL, fusion loop; NHR and CHR, N- and C-terminal heptad repeats; SP, signal peptide; TM, transmembrane helix. The GPΔ construct used for structure determination is made by deleting residues 313–463 of the GP mucin domain and residues 633–676. Residue 312 is directly linked to 464.

A foldon trimerization peptide and a 6× His tag are added at the C terminus. b, The GP2 trimer in the prefusion state (current structure). The trimer is shown as cartoon representation with the monomers coloured in red, green and blue, respectively. Disulfide bonds are shown as orange sticks. c, The six-helix bundle of GP2 in the post-fusion state.

RESEARCH LETTER

Extended Data Figure 3 | The fusion loop. **a**, The fusion loop that connects β19 and β20 of GP2 projects onto a shallow depression on the surface of a neighbouring monomer. The fusion loop is shown as a red coil with side chains drawn as grey sticks, the neighbouring monomer is shown in semi-transparent surface representation. **b**, Comparison of the fusion loop in the apo GP (red and grey) obtained at pH 5.2 with that in the KZ52 Fab complex (cyan) obtained at pH 8.3.

© 2016 Macmillan Publishers Limited. All rights reserved

Extended Data Figure 4 | Pockets and tunnels in EBOV GP trimer.
a, The several small pockets and three large tunnels in the GP trimer shown as grey surfaces. Protein backbones are drawn as ribbons and coloured as in Fig. 2 of the main text. A toremifene is bound at the

entrance of each large tunnel and shown as yellow sticks. b, Close up view of a tunnel. Each tunnel is bordered by secondary structure elements from two neighbouring monomers.

© 2016 Macmillan Publishers Limited. All rights reserved

RESEARCH LETTER

Extended Data Figure 5 | The inhibitor-binding site. **a**, The DFF lid (residues 192–194, blue coil for main chain and sticks for side chains) nestles at the entrance of the large tunnel in the apo structure. The rest of the protein is shown as an electrostatic surface. The putative cathepsin cleavage site at residue 190 is indicated by an arrow. **b**, **c**, Toremifene (yellow sticks in **b**) and ibuprofen (cyan sticks in **c**) bind at the same site by expelling the DFF lid. In both panels, the inhibitor bound structure is shown in blue (GP1) and red (GP2), the apo GP in grey. **d**, Comparing the binding modes of toremifene and ibuprofen. The toremifene-bound structure is shown in blue and red, the ibuprofen bound structure in grey.

Extended Data Figure 6 | The environment of α1′ and α1 helices. The surfaces of α1′ and α1 helices, which undergo large conformational changes upon receptor binding, are protected by the 287–293 loop from the glycan cap domain and the N563 glycan from GP2 in the apo GP. The glycan is modelled as Man9GlcNAc2.

RESEARCH LETTER

a  
![Chemical structure of toremifene](chemical_structure_toremifene.png)  

b  
![Chemical structure of ibuprofen](chemical_structure_ibuprofen.png)  

c  
![Electron density map for toremifene](electron_density_map_toremifene.png)  

d  
![Electron density map for ibuprofen](electron_density_map_ibuprofen.png)  

Extended Data Figure 7 | Chemical structures and electron density maps. **a**, **b**, The chemical structures of toremifene (**a**) and ibuprofen (**b**). **c**, **d**, $|F_{\mathrm{o}}-F_{\mathrm{c}}|$ omit electron density maps for toremifene (**c**) and ibuprofen (**d**) contoured at $3\sigma$.

$\beta 1$ $\beta 2$ $\beta 3$ $\alpha 1'$

35 40 45 60 65 70

Zaire... LGV**I**HNSTLQV ..... T**N**QLRSVGLNLEG  
Bundibugyo... LGV**V**HNNTLQV ..... T**S**QLKSVGLNLEG  
Tai_Forest... LGV**V**HNNTLQV ..... T**S**QLKSVGLNLEG  
Sudan... LGV**V**TNSTLEV ..... T**D**QLKSVGLNLEG  
Reston... LGI**V**TNSTLKA ..... T**S**QLKSVGLNLEG  
Cuevavirus... LGL**L**GNNSTIQ ..... T**D**QLQAIGLLLEG  
Marburg... VLE**I**ASNS·QP ..... T**E**DVHLMGFTLSG  

$\beta 5$ $\beta 6$ $\beta 13$

95 100 180 185 190

Zaire... PKVVNYEA**GEW** ..... VVAF**LI**LPQ A KKD  
Bundibugyo... PKVVNYEA**GEW** ..... VVAF**LI**LPE T KKD  
Tai_Forest... PKVVNYEA**GEW** ..... VI AF**LI**LPK A RKD  
Sudan... PQVVSYEA**GEW** ..... VI AF**LI**LAK P KET  
Reston... PKVVSYEA**GEW** ..... VVAF**LI**LSE P KK H  
Cuevavirus... PKIVGYTA**GEW** ..... TI AF**LI**VPK D APR  
Marburg... PKNVEYTE**GE**E ..... NIAAMIVNK T VHR  

$\beta 19$ $\alpha 3$

515 520 550 555

Zaire... NPN**L**HYWTTQD ..... LMHNQDG**LI**CGG LR  
Bundibugyo... NPN**L**HYWTTQD ..... IMHNQNG**LI**CGG LR  
Tai_Forest... NPN**L**HYWTALD ..... IMENQNG**LI**CGG LR  
Sudan... NPN**L**HYWTAQE ..... LMHNQN**ALV**CGG LR  
Reston... NPD**L**YYW TAVD ..... VMHNQNG**LI**CGG LR  
Cuevavirus... NPN**L**RYWTSRE ..... IMEHQNT**IV**CGG LR  
Marburg... DAELR**I**WSVQE ..... LIKNQN**NLV**CR LR  

Extended Data Figure 8 | Sequence alignment of filovirus GPs. Amino acid sequence alignment of 7 filovirus GPs around the inhibitor-binding site. The amino acids that form contacts with toremifene or ibuprofen are coloured in green. Numbering corresponds to the full length Zaire EBOV GP, conserved residues are shown in a red background. Secondary structure elements are labelled on the top.

RESEARCH LETTER

a
EBOVpp + DMSO
NoEnv + DMSO
EBOVpp +
15 μM Toremifene

EBOVpp +
1.5 μM Toremifene
EBOVpp +
150 μM Ibuprofen
EBOVpp +
15 μM Ibuprofen

b

ns

% Red Cells (Fusion)

EBOVpp + DMSO
NoEnv + DMSO
EBOVpp + 15 μM Toremifene
EBOVpp + 1.5 μM Toremifene
EBOVpp + 150 μM Ibuprofen
EBOVpp + 15 μM Ibuprofen

Extended Data Figure 9 | Toremifene and ibuprofen inhibit fusion of Ebola GP pseudovirus particles. a, CCF2-loaded TZM-bl cells were exposed to EBOV pseudoparticles (EBOVpp) or control particles lacking envelope proteins (NoENV) at 4°C to synchronise binding and receptor engagement before fusion was initiated by shifting cells to 37°C in the presence of toremifene (15 μM and 1.5 μM), ibuprofen (150 μM and 15 μM), or just the solvent (5% DMSO). After 2 h incubation, cells were loaded with the CCF2-AM FRET biosensor, fixed and the ratio of blue (440–480 nm, cleaved CCF2-AM) to green (500–540 nm, uncleaved CCF2-AM) fluorescence measured. Cells are pseudocoloured according to this ratio: blue represents no fusion, red represents fusion. Scale bar: 80 μm. b, The percentage of fusogenic cells (red versus blue) was calculated taking the average max value coming from the negative control as a threshold for fusion, data are means ± s.d. (n = 10). *P ≤ 0.05, ***P ≤ 0.001 (unpaired t-test, compared to the EBOV plus DMSO control). ns, not significant (P > 0.05). Error bars represent s.d.

Extended Data Table 1 | Data collection and refinement statistics

| Data collection | Native GP | GP-toremifene | GP-ibuprofen |
|------------------|------------|---------------|--------------|
| Space group     | $R32$      |               |              |
| Cell dimensions  |            |               |              |
| $a, b, c (\AA)$ | 114.3, 114.3, 307.4 | 113.5, 113.5, 306.9 | 113.8, 113.8, 306.2 |
| $\alpha, \beta, \gamma (^{\circ})$ | 90, 90, 120 | 90, 90, 120 | 90, 90, 120 |
| Resolution ($\AA$) | 94.2–2.23 (2.29–2.23)* | 51.2–2.69 (2.76–2.69) | 82.8–2.68 (2.75–2.68) |
| $R_{\text{merge}}$ | 0.204(---) | 0.079(---) | 0.143(---) |
| $I/\sigma I$ | 17.4(1.3) | 20.0(1.9) | 14.7(1.5) |
| Completeness (%) | 100(100) | 99.9(100) | 99.9(100) |
| Redundancy | 57.8(15.4) | 9.8(8.6) | 9.8(8.3) |

| Refinement |            |               |              |
|------------|------------|---------------|--------------|
| Resolution ($\AA$) | 94.2–2.23 | 51.2–2.68 | 82.8–2.68 |
| No. reflections | 36035/1865 | 20449/1090 | 20734/1107 |
| $R_{\text{work}} / R_{\text{free}}$ | 0.226/0.241 | 0.201/0.245 | 0.199/0.235 |
| No. atoms |            |               |              |
| Protein   | 3129       | 3034          | 3025         |
| Ligand/glycan/ion | 143        | 151           | 140          |
| Water     | 119        | 53            | 73           |
| $B$-factors |            |               |              |
| Protein   | 39         | 62            | 55           |
| Ligand/glycan/ion | 99         | 115           | 96           |
| Water     | 54         | 70            | 55           |
| R.m.s. deviations |            |               |              |
| Bond lengths ($\AA$) | 0.007    | 0.007         | 0.007        |
| Bond angles ($^{\circ}$) | 1.3      | 1.3           | 1.3          |

*Values in parentheses are for highest-resolution shell.
