
Review

# Measles Virus Hemagglutinin Protein Epitopes: The Basis of Antigenic Stability

Maino Tahara ${}^{1}$, Jean-Philippe Bürckert ${}^{2}$, Kazuhiko Kanou ${}^{3}$, Katsumi Maenaka ${}^{4}$, Claude P. Muller ${}^{2}$ and Makoto Takeda ${}^{1,*}$

${}^{1}$ Department of Virology 3, National Institute of Infectious Diseases, Musashimurayama 208-0011, Tokyo, Japan; maino@nih.go.jp  
${}^{2}$ Department of Infection and Immunity, Luxembourg Institute of Health, Esch-sur-Alzette L-4354, Luxembourg; Jean-Philippe.Buerckert@lih.lu (J.-P.B.); Claude.Muller@lih.lu (C.P.M.)  
${}^{3}$ Infectious Disease Surveillance Center, National Institute of Infectious Diseases, Shinjuku-ku 162-8640, Tokyo, Japan; kanou@nih.go.jp  
${}^{4}$ Laboratory of Biomolecular Science, Faculty of Pharmaceutical Sciences, Hokkaido University, Sapporo 060-0812, Hokkaido, Japan; maenaka@pharm.hokudai.ac.jp  

* Correspondence: mtakeda@nih.go.jp; Tel.: +81-42-848-7060

Academic Editor: Richard K. Plemper  
Received: 15 June 2016; Accepted: 20 July 2016; Published: 2 August 2016

Abstract: Globally eliminating measles using available vaccines is biologically feasible because the measles virus (MV) hemagglutinin (H) protein is antigenically stable. The H protein is responsible for receptor binding, and is the main target of neutralizing antibodies. The immunodominant epitope, known as the hemagglutinating and noose epitope, is located near the receptor-binding site (RBS). The RBS also contains an immunodominant epitope. Loss of receptor binding correlates with an escape from the neutralization by antibodies that target the epitope at RBS. Another neutralizing epitope is located near RBS and is shielded by an N-linked sugar in certain genotype strains. However, human sera from vaccines and measles patients neutralized all MV strains with similar efficiencies, regardless of the N-linked sugar modification or mutations at these epitopes. Two other major epitopes exist at a distance from RBS. One has an unstructured flexible domain with a linear neutralizing epitope. When MV-H forms a tetramer (dimer of dimers), these epitopes may form the dimer-dimer interface, and one of the two epitopes may also interact with the F protein. The neutralization mechanisms of antibodies that recognize these epitopes may involve inhibiting the H-F interaction or blocking the fusion cascade after MV-H binds to its receptors.

Keywords: measles virus; hemagglutinin protein; epitope

---

## 1. Introduction

Measles has been a leading cause of childhood death in the past, and a significant number of measles-related deaths are still reported, mainly in developing countries. However, progress in measles control suggests that the disease can potentially be eradicated globally by vaccination [1,2]. All available data show that the measles virus (MV) is serologically monotypic. Therefore, measles vaccines that are derived from strains isolated more than 50 years ago are still highly effective against all the currently circulating MV strains. The antigenic stability of MV is one of the major factors that make global measles eradication feasible [3]. MV is classified in the genus *Morbillivirus* in the family *Paramyxoviridae*. Two types of glycoprotein spikes, the hemagglutinin (H) and fusion (F) proteins, protrude from the viral envelope. Although both the H and F proteins are the targets of neutralizing antibodies, the humoral immune response is mainly directed against the H protein [4–6]. The function of the H protein is to bind to a cellular receptor on the target cells. Signaling lymphocyte activation

Viruses 2016, 8, 216; doi:10.3390/v8080216  
www.mdpi.com/journal/viruses

molecule (SLAM) expressed on immune cells and nectin-4 expressed on epithelial cells are receptors for MV [7–10]. In addition to SLAM and nectin-4, vaccine strains and certain laboratory strains of MV have been adapted to use CD46 as a receptor [11]. When the H protein binds to a receptor, conformational changes are triggered in the F protein, which mediate the fusion between the viral envelope and the host cell plasma membrane. The functional and physiological interactions between the H and F proteins are critical for triggering membrane fusion [12].

To date, many studies using different sets of monoclonal antibodies (MAbs) have determined the antigenic epitopes of the H protein. In general, MV is antigenically highly stable, but minor antigenic variations are caused by specific amino acid changes in the H protein [2,13–17]. The crystal structures of the H protein complexed with SLAM [18], nectin-4 [19], and CD46 [20] have been determined, so the antigenic epitopes or specific amino acids can be mapped onto these structures. Recent studies have also provided extensive insight into the mechanism of membrane fusion triggered by the interaction between the H and F proteins [21]. The aim of this review is to integrate the currently available data and additional data (provided in this review) to organize and maximize our understanding of the H protein epitopes and the molecular basis of the antigenic stability of MV.

In this review, the major epitopes of the H protein are classified into five types: hemagglutinating and noose epitope (HNE), receptor-binding epitope (RBE), sugar-shielded epitope (SSE), neutralizing epitope (NE), and loop epitope (LE). This review summarizes the details of the individual epitopes, including their locations and known and predicted functions, and clarifies why the MV-H protein remains antigenically stable.

## 2. Overall Structure of the H Protein

The head domain of the MV-H protein consists of a six-bladed β-propeller fold (β1–β6 sheets) (Figure 1A–C) [22]. When viewed from the top, the overall structure of the head domain is square, and there is an enlarged pocket in the center (Figure 1B,D) [22]. The N-linked sugar at amino position 215 (N215-sugar) is thought to shield the pocket [22]. The H protein forms a homodimer linked by an interchain disulfide bond between the cysteine residues at position 154. In the homodimer, the two H protein monomers interact with each other in a highly tilted position [22]. The dimer of the H protein is further assembled into a tetrameric structure by forming a dimer of dimers [18]. An X-ray crystal structural analysis has demonstrated two crystal forms (form I and form II) of the head domain of the MV-H protein [18]. The protein may form a different type of tetramer if it has a stalk region, and may exist in several different forms during the process of receptor binding and the subsequent triggering of membrane fusion.

Figure 1. Top view of MV-H and the locations of HNE, NE, and LE. (A) Ribbon diagram. MV-H has a six-bladed β-propeller fold (β1–β6). β-Sheets 1, 2, 3, 4, 5, and 6 are shown in blue, red, orange, yellow, light green, and green, respectively; (B) Schematic diagram. Light green, blue, and green ovals indicate locations of HNE, NE, and LE, respectively. Black solid and dashed lines indicate the overall structure of the H protein; (C,D) Locations of HNE, NE, and LE. The amino acids demonstrated or suggested to constitute a portion of an epitope are shown in colors: residues on β-sheets 1, 2, 3, 4, 5, and 6 are shown in blue, red, orange, yellow, light green, and green, respectively. Light green, blue, and green dashed-line circles indicate locations of HNE, NE, and LE, respectively. (C) Cartoon model. (D) Surface presentation and MAbs that recognize HNE, NE, and LE; (E) Enlarged view of HNE. Stick models indicate two cysteine residues, and the dashed red line indicates the disulfide bond between the cysteines.

## Hemagglutinating and Noose Epitope (HNE)

Amino acids 379–400 form an immunodominant epitope. This epitope is known as HNE because three cysteines in the epitope form a surface-exposed loop (light green in Figure 1C,D) [23,24]. The disulfide-constrained surface-exposed loop is often described as the “noose” motif [25]. MAb-BH6, -BH21, and -BH216, which show both hemagglutination-inhibiting (HI) and -neutralizing activities, were generated after immunization with MV [23]. These antibodies recognize the conformation of HNE, and the cysteine loop formed by the disulfide bond between the cysteines at positions 386 and 394 is critical for its conformation (Figure 1E) [23]. The C-terminal loop is also important for its recognition by MAb-BH6, -BH21, and -BH216 (Figure 1E) [23]. The binding of ~40% of human serum anti-MV antibodies in convalescent measles patients and vaccines is blocked by MAb-BH6 [26]. Although the amino acid sequence of this region is poorly conserved among the H proteins of other morbilliviruses, the two cysteines are highly conserved. Therefore, the disulfide bond may be important for the function or conformational stability of the H protein. Several previously described epitopes correspond to HNE: I recognized by MAb-B5 and MAb-E81 [27,28], E3 recognized by MAb-cl48 [29], D recognized by MAb-K71 [30], D/E recognized by MAb-NC32 [30], and F recognized by MAb-L77 [30] (Figure 1D and Table 1).

HNE is well conserved among MV strains, but certain genotype H1 strains are not neutralized by MAb-BH6 or -BH216 because they have a proline-to-leucine substitution at position 397 (P397L) [13,31]. Similarly, Japanese H1 strains with the P397L mutation are not neutralized by MAb-E81 [32]. The proline at position 397 is probably critical for MV neutralization by these MAbs. However, MAb-BH216 still binds to an HNE peptide consisting of amino acid region 379–400, even when it contains the P397L mutation [24]. These observations suggest that there are some structural differences between the HNE peptide and the native form of HNE. The sera of vaccinees efficiently neutralize these MV strains, despite the P397L mutation [13]. The roles of HNE are currently unknown, but it is probably structurally or functionally important for the H protein. A mutant that escaped neutralization by MAb-B5 and -E81 showed an attenuated capacity for replication because it had a mutation in the HNE (Q391R) [28].

Table 1. Relationships between epitopes and MAbs.

| Amino Acids¹ | MAb          | Epitope Name |
|--------------|--------------|--------------|
| 235         | E185         | iv           |
| 233–240     | BH1          |              |
| 235, 244–250| BH47, BH59, BH103, BH129 | NE        |
| 302         | E39          | v            |
| 310         | BH38         | LE           | E4        |
| 310         | BH141        | LE           | E4        |
| 309–318     | I-29         | LE           | I         | E4       | IA      | 1       |
| n.d.²       | I-12         | LE           | I         |          | IA      | 1       |
| 311         | E103         | LE           | vi        |
| 377–378     | L77          | F            |
| 391         | B5           | I            |
| 391         | E81          | I            |
| 379–400     | BH6, BH21, BH216 | HNE        |
| 395, 398    | cl48         | E3           |
| n.d.        | K71          | D            |
| n.d.        | NC32         | D/E          |
| 473–477     | E128         | SSE          | II        |
| 491         | 16-CD11      | SSE          | II        | II       | 2       |
| n.d.        | BH99         | SSE          |
| 488         | BH97         |              |
| 483         | 2F4          | RBE          | vii       |
| 505, 541, 543, 533 | 2F4   | RBE          | vii       |
| 505, 506    | 80-II-B2     | RBE          |
| 533         | cl55         | RBE          |
| 532, 533    | 16-DE6       | RBE          | III       | E2       | 3       |
| 547, 546    | 20H6         | RBE          |
| 552         | I-41         | RBE          | III       | E2       | IIIB    | 3       |
| 187         | I-44         | RBE          | IV        | IV       | 4       |
| 190         | 2F4          | RBE          | vii       |
| 190–200, 571–579 | BH26    |              |

References³

This review [23] [27] [28] [44] [29] [30] [36] [45]

[53] [40]

1. Amino acids demonstrated or suggested to constitute a portion of an epitope; ²n.d., not determined;
3. Corresponds to the epitope name.

Table 2. Neutralizing titers and hemagglutination-inhibiting (HI) titers of MAbs.

| Cell | Neutralizing Titer | HI |
|------|--------------------|----|
|      | B95a               |    |
|      | SLAM               |    |
| Receptor |                  |    |
| Virus<sup>1</sup> | A | D3 | D3/Q391R | II-18 | Nectin-4 | Vero | RBC<sup>3</sup> |
|        |                   |    |         | A     | D3     | D3/Q391R | CD46 | A |
| MAb<sup>2</sup> | E81 | 2560 | 5120 | <40 | 81,920 | 81,920 | <1280 | 163,840 | 40,960 |
|        | B5  | 81,920 | 20,480 | <1280 | 81,920 | 10,240 | <1280 | 163,840 | 20,480 |
|        | E128 | 5120 | 40 | n.d.<sup>4</sup> | 163,840 | <2560 | n.d. | 655,360 | 40,960 |
|        | E103 | 2560 | 10,240 | 20,480 | 10,240 | 20,480 | 20,480 | 2560 |
|        | E185 | 640 | 40 | n.d. | n.d. | n.d. | 2560 | 2560 |

<sup>1</sup> Viruses cited in this table have been reported previously [28]; <sup>2</sup> Undiluted ascites containing MAb; <sup>3</sup> Red blood cells of African green monkey; <sup>4</sup> n.d., not determined.

Viruses 2016, 8, 216

4. Receptor-Binding Epitope (RBE)

RBS is located on the lateral side of the H protein propeller fold structure (Figure 2) [18]. A phenylalanine at position 552 interacts with SLAM [18], and a mutation at this residue causes MV to escape neutralization by MAb-I-41 (Figure 2D) [34,35]. The aspartic acids at positions 505 and 507 also interact with SLAM [18], and a mutation at position 505 or 506 allows MV to escape neutralization by MAb-80-II-B2 (Figure 2D) [5,36]. The CAM-70 vaccine strain is not neutralized by MAb-80-II-B2 because it has an amino acid substitution at position 505, and it uses SLAM less efficiently than other strains [37]. A mutant resistant to neutralization by MAb-16-DE6 has acquired a mutation at amino acid position 533, one of the residues that interact with SLAM (Figure 2D) [34,35]. A mutation at position 533 also abolishes neutralization by MAb-cl55 (Figure 2D) [38], and MV escapes neutralization by MAb-20H6 when the H protein contains mutations at positions 546 and 547 (Figure 2D) [38]. The residue at position 546 modulates the binding ability of the H protein to CD46 [11] and nectin-4 [39]. These data suggest that RBS contains a major epitope (designated “the receptor-binding epitope” [RBE]), and a mutagenesis analysis using recombinant MV provided further evidence of this. When MV was mutated at a nectin-4-interacting residue (F483A, Y543S, or Y541S), it escaped neutralization by MAb-2F4 (Figure 2D), but lost the ability to use nectin-4 as a receptor [40,41]. Similarly, MV escaped neutralization by MAb-2F4 when it mutated at a SLAM-interacting residue of the H protein (R533A or D505S) (Figure 2D), but it lost its ability to use SLAM as a receptor [40].

One study has suggested that the epitope recognized by MAb-BH26 is the most important immunodominant epitope because MAb-BH26 inhibits the binding of ~60% of human serum antibodies in convalescent measles patients and vaccines [26]. The binding site of MAb-BH26 is predicted to be in the amino acid region 571–579 or 190–200 [5]. Both these regions constitute portions of the β6 sheet (Figure 2C), and the residues at positions 191–195 are located particularly close to RBS and interact directly with SLAM [18]. An amino acid mutation at position 190 reduces the neutralization activity of MAb-2F4 [32]. MAb-BH26 preferentially blocks MV infection via SLAM [5]. These data are consistent with the notion that the predicted BH26-recognizing region (amino acids 190–200) interacts directly with SLAM (Figure 2E) [18].

There is a linear epitope in the region defined by residues 188–199 and a potential glycosylation site at position 187 (Asn-Cys-Ser at positions 187–189) [35]. Variants that are not neutralized by MAb-I-44 have acquired an S189P or S189L mutation [35]. Therefore, the epitope recognized by MAb-I-44 may involve the N187-sugar or the linear epitope 188–199, which interacts with SLAM. A mutant protein containing N187S also lacks N187-sugar, but the mutant is neutralized efficiently by MAb-I-44 [42]. Therefore, MAb-I-44 probably recognizes amino acid sequences that include the SLAM-interacting linear epitope, rather than N187-sugar. However, biochemical analyses have indicated that MAb-I-44 only partially inhibits the H-SLAM interaction [43]. These observations suggest that N189-sugar partially shields the epitope recognized by MAb-I-44.

Various MAbs recognize RBE (Table 1). Several previously described epitopes correspond to RBE: vii recognized by MAb-2F4 [28]; III [44], E2 [29], 3 [45], and IIIB [36] recognized by MAb-16-DE6 and MAb-I-41 (Table 1). These data suggest that the capacity to escape the humoral immune response is incompatible with the functional integrity of the MV-H protein.

5. Epitope Shielded by N416-Sugar (Sugar-Shielded Epitope, SSE)

The H protein is glycosylated at four amino acid positions, 168, 187, 200, and 215 [42]. Position 238 is also a potential N-linked glycosylation site, but it is not used [42]. These N-linked sugars are expected to shield a wide area of the MV-H protein, limiting the areas targeted by the humoral immune response (Figure 3) [22]. The major epitopes, including HNE and RBE, are largely unshielded by these sugars (Figure 3) [22]. Certain genotypes have acquired an additional N-linked glycosylation site at position 416 (N416-sugar) [17,46,47]. It is evident that the N416-sugar shields a specific epitope(s) (SSE) that is recognized by MAbs E128 and BH99 [28,48] (Table 1). SSE recognized by MAb-E128 contains the amino acid region 473–477 (Figure 3G) [28], so SSE may constitute a portion of the CD46-binding
site [5,28,49]. The amino acids or the region recognized by MAb-BH99 are unknown, but MAb-16-CD11 competes for BH99 binding, and the epitope recognized by MAb-16CD11 contains an amino acid residue at position 491 (Figure 3G) [35]. Structural analysis data suggest that SSE does not overlap the SLAM- or nectin-4-binding site. However, MAb-E128 inhibits the SLAM-H interaction [28]. The binding of MAb-E128 also partially competes with that of MAb-2F4, which binds to the central region of RBS (Figure 2D) [40]. These findings are consistent with the observation that SSE is not RBS, but is located near RBS. A phylogenetic analysis suggested that an ancestral strain of the D3, D4, D5, D7, D8, D9, and D11 genotypes acquired the N416-sugar [50]. There are 24 genotypes [51], but only 11 have been detected in the last decade. Among these, the D4, D5, D7, D8, D9, and D11 genotype strains have N416-sugar, whereas the G3, H1 B2, B3, D6 genotype strains lack this sugar. Certain H1 genotype strains are not neutralized by MAb-E128, even though they lack the N416-sugar [32], so it is likely that they have acquired mutations in the epitope, rather than the N416-sugar shield. Both N416-sugar-possessing and -lacking strains are currently circulating worldwide. Furthermore, human sera obtained from vaccine recipients and measles patients neutralize different genotype strains with similar efficiencies, regardless of the N416-sugar modification (Table 3) [17,52]. Therefore, the advantage of N416-sugar is unclear.

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 3. Dimer structure of the MV-H protein and models of N-linked sugar modifications. (A,E) Red arrows in (A) and (E) indicate the direction of sight for (B–D) and (F–H), respectively. (B,F) Schematic diagram. Light green, blue, and green ovals indicate locations of HNE, NE, and LE, respectively. Black solid lines and dashed lines indicate the overall structure of the H protein. Magenta and cyan dashed lines indicate locations of N416-sugar and SLAM, respectively. (C,G) Surface presentation models. Amino acids demonstrated or suggested to constitute a portion of an epitope are shown in colors: residues on β-sheets 1, 2, 3, 4, 5, and 6 are shown in blue, red, orange, yellow, light green, and green, respectively. (D,H) Surface presentation models with N-linked sugar modifications constructed with GlyProt. N-linked sugars are shown with sphere models. (H) Upper structure has N416-sugar and lower structure lacks N416-sugar.
Table 3. Neutralizing titers of human sera from five vaccine recipients and four measles patients ${}^{1}$.

| Virus ${}^{2}$ | MAb ${}^{3}$ | Vaccines | Measles Patients |
|-----------------|--------------|----------|------------------|
|                 | E128         | #v9      | #v12             | #v14 | #v27 | #p1 | #p2 | #p3 | #p4 |
| A               | 163,840      | 80       | 640              | 160  | 80   | 40  | 40  | 320 | 5120 |
| D3              | <2560        | 160      | 640              | 160  | 80   | 320 | 160 | 640 | 20,480 |

${}^{1}$ Data in II-18 cells; ${}^{2}$ Viruses used in this table have been reported previously [28]; ${}^{3}$ Undiluted ascites containing MAb.

---

## 6. Neutralizing Epitope (NE) and BH1-Binding Epitope

Neutralizing antibodies often recognize conformational epitopes, but linear epitopes can also be targeted. MAb-BH47, -BH59, -BH103, and -BH129 recognize a linear epitope consisting of amino acids 244–250 (blue in Figure 1C,D) [53], which is designated 'the neutralizing epitope' (NE) [53]. The amino acid region 233–240 is recognized by MAb-BH1, and this region probably forms a surface-exposed domain together with NE, although the amino acid region at 240–247 has not been visualized in H protein structures [18–20,22,28], and may be an unstructured flexible region. An amino acid mutation in the BH1-binding region (E235G) severely reduces the neutralizing activities of MAb-BH47, -BH59, and -BH129, suggesting a close interaction between NE and the BH1-binding region (Table 4). MAbs recognizing NE efficiently neutralize MV infection, but show little or very low HI activity [53]. The data suggest that these MAbs do not block CD46 binding. In fact, NE is located distantly from RBS (Figure 3C). MAb-E185, which is predicted to recognize the BH1-binding region only weakly or moderately, inhibits both hemagglutination and MV infection (Table 2) [28]. MAb-BH1 binds to peptides containing the amino acid region 233–240, but binds poorly to the native form of the H protein [53]. Therefore, the BH1-binding region may not be readily accessible in the native form of the H protein. Although MAb-BH47, -BH59, and -BH129 have little or very low HI activity, they efficiently inhibit hemolysis [53]. These observations suggest that MAb-BH47, -BH59, and -BH129 inhibit the process of membrane fusion after MV-H binds to its receptor. NE is probably involved in the formation of the tetramer. In the form I tetramer, NE constitutes a portion of the dimer-dimer interface (Figure 4) [18]. Therefore, it is possible that MAbs that recognize NE inhibit the triggering of the fusion process by interfering with the arrangement of the tetramer. However, NE may not be an immunodominant epitope because MAb-BH47 only weakly inhibits the binding of human serum antibodies in convalescent measles patients and vaccinees, compared with other MAbs such as BH26, BH6, and BH38, which recognize other epitopes [26]. Certain MV genotype H1 strains are not neutralized by MAb-BH47, which recognizes NE [13].

---

Table 4. Neutralizing titers of MAbs that recognize NE ${}^{1}$.

| Virus ${}^{2}$ | A     | A/E235G | D3    |
|-----------------|-------|---------|-------|
| MAb ${}^{3}$    | BH47  | 81,920  | <640  |
|                 | BH59  | 20,480  | <160  |
|                 | BH129 | 640     | <160  |

${}^{1}$ Data in Vero/hNectin4 cells; ${}^{2}$ Viruses used in this table have been reported previously [28]; ${}^{3}$ Undiluted ascites containing MAb.

Figure 4. Tetrameric structure of MV-H (form I). (A–D) Four H protein molecules are shown in gray, light gray, purple, and light purple. SLAM is shown in cyan. (A) Schematic diagram of the formation of the dimer of dimers. Light green regions indicate the interface between the monomers in the dimer. Pink regions indicate the interface between dimers in the dimer of dimers. Pink regions of the purple and light gray MV-H molecules form the interface, whereas those of the gray and light purple MV-H molecules are exposed outside the dimer of dimers (tetramer). (B) Schematic diagram of the dimer of dimers (form I). Red arrow indicates the direction of sight for C and D. (C,D) Surface presentation models show MV-H, and ribbon diagrams show SLAM. (C) Light green regions indicate the interface between the monomers forming the dimer. Pink regions indicate the interface between the dimers forming the dimer of dimers. (D) Same angle as C. Amino acids demonstrated or suggested to constitute a portion of an epitope are shown in colors: residues on β-sheets 1, 2, 3, 4, 5, and 6 are shown in blue, red, orange, yellow, light green, and green, respectively.

## 7. Epitope on the Loop Protruding from the β2 Sheet (Loop Epitope; LE)

MAb-I-29 recognizes the amino acid region 309–318 and neutralizes MV [33]. A large loop protruding from the β2 sheet constitutes the epitope of this MAb, and is referred to as “the loop epitope” (LE) in this review (green in Figure 1C,D). MAb-BH38, -BH141, and -E103 recognize LE (Figure 1D), and previously described epitopes I [44], E4 [29], IA [36], 1 [45], and vi [28] correspond to this epitope (Table 1). MAb-I-29 shows little HI activity, suggesting that it does not inhibit CD46 binding, although it shows strong hemolysis-inhibiting activity [33]. This failure to inhibit receptor binding is consistent with the observation that LE is located at a position distant from RBS (Figure 2B). MAb-E103 also does not inhibit the H-SLAM interaction [28]. However, it shows some HI activity (Table 2). Therefore, MAb-E103 may inhibit the CD46 interaction. However, it also displays neutralizing activity by interfering with the H-F interaction [28]. The head domain of the H protein is situated on a stalk region, which has been shown to interact with the F protein and to play a key role in triggering fusion (Figure 5C,D) [54,55]. The crystal structure of the H protein (form I) [18] suggests that in two of the four H protein molecules (gray and light purple molecules in Figure 5), LE is located at the bottom of the tetramer. Therefore, LE may interact directly with the F protein in a higher-order structure of the H-F oligomeric protein (Figure 5D) [54,56,57]. Furthermore, LE in the other two H molecules (light gray and purple molecules in Figure 5) interacts with NE at the dimer-dimer interface of the tetramer [18] (Figure 4). Therefore, MAb-I-29 and -BH38, which recognize LE, interfere with the dimer-dimer interaction of the H tetramer [58]. These two interacting roles may generate the structural constraints against changes in LE. Supporting this, a mutant viral strain with a mutation in LE (Q311R) replicates less efficiently than the parental strain [28].

Figure 5. Predicted positional relationship between the MV-H tetramer (form I) and MV-F.

(A–D) Four H protein molecules are shown in gray, light gray, purple, and light purple. SLAM is shown in cyan. (C,D) MV-F is shown in light brown. (A,C) Schematic diagrams of the MV-H tetramer (form I) and MV-F. Red arrows in A and C indicate the direction of sight in B and D, respectively. (B,D) Surface presentation models of MV-H. Ribbon diagrams show SLAM (cyan) and MV-F (light brown). The structural model of MV-F has been reported previously [59].

## 8. Materials and Methods

### 8.1. Cells

II-18 and B95a cells were maintained in RPMI medium (Invitrogen, Carlsbad, CA, USA) supplemented with 7.5% fetal calf serum (FCS). Vero and Vero/hNectin4 cells [60] (Vero cells constitutively expressing human SLAM and nectin-4, respectively) were maintained in DMEM (Gibco, Carlsbad, CA, USA) supplemented with 7.5% FCS.

### 8.2. Viruses

All MV strains used in the present study were recombinant viruses expressing Renilla luciferase [61]. The “D3” virus has the genome of the wild-type IC323 strain (genotype D3) [28,62]. The “A” virus has the genome of the wild-type IC323 strain, except for the H gene, which was derived from the Edmonston vaccine strain (genotype A) [28]. The D3/Q391R virus is a mutant “D3” virus with the Q391R mutation in the H protein [28]. The A/E235G virus is a mutant “A” virus with the E235G mutation in the H protein [28].

### 8.3. MAbs and Human Sera

Mouse MAb-E81, -B5, -E128, -E103, and -E185 (mouse ascites) have been reported previously [28]. Mouse MAb-BH47, -BH59, and -BH129 (mouse ascites) have also been reported previously [53]. Sera from patients suffering from acute measles were collected in Tokyo in 2002. Vaccinated sera were collected from one-year-old children who had received one dose of a combined measles/mumps/

rubella vaccine. This study was performed with the approval of the Ethics Committee of the National Institute of Infectious Diseases, Japan.

### 8.4. Neutralizing Assay

The details of the method have been reported previously [28]. Briefly, 2000 PFU of recombinant MV was incubated with serially diluted MAbs or human sera, then inoculated into the culture medium for II-18, B95a, Vero, or Vero/hNectin4 cells. At two days post-infection, the luciferase activity in the cells was measured. The neutralizing titer was indicated by the maximum dilution causing a >50% reduction in luciferase activity.

### 8.5. HI Assay

MV antigen (4 hemagglutinating units in a volume of 25 μL) was added to 25 μL of serially twofold-diluted MAbs in U-bottom 96-well microplates. Phosphate-buffered saline (PBS; pH 7.2) containing 0.1% bovine serum albumin and 0.01% gelatin was used for dilution. Each well received 50 μL of a 0.5% suspension of African green monkey erythrocytes. Plates were shaken and incubated for 1.5 h at 37°C. The highest dilution of each MAb that completely inhibited hemagglutination was considered the HI titer of that MAb.

### 8.6. Structures of H and F Proteins

The crystal structures of the H protein unbound to its receptors [22] and when complexed with SLAM [18], nectin-4 [19], or CD46 have been reported previously [20]. The model structure of the MV-F protein was constructed based on the previously reported crystal structure of the F protein of parainfluenza virus type 5 (PDB: 2B9B) [59]. Sugar modifications on the H protein were modeled with GlyProt (http://www.glycosciences.de/modeling/glyprot/php/main.php). Figures were generated with PyMOL (http://www.pymol.org).

## 9. Conclusions

In this review, we have reorganized large amounts of data from various papers concerning the MV-H protein epitopes, together with additional data. The MV-H protein epitopes can be roughly clustered into five distinct epitopes. The first is HNE; the second comprises NE and the BH1-binding epitope; the third is LE and is located on the loop protruding from the β2 sheet; and the fourth is RBE. The exact location of the fifth epitope, SSE, is still unclear, but it is located near RBS and is shielded by the N416-sugar in the specific genotype strains currently circulating.

Among these epitopes, the function of RBE is very clear. The epitope is part of a region that must bind to two proteinaceous receptors, SLAM and nectin-4. Therefore, it is not shielded by sugars and must be subjected to strong structural constraints. Significantly, amino acid changes that allow MV to escape neutralization cause a loss of its receptor-binding activity [40]. The antibodies in polyclonal sera probably target various regions of this epitope, and it is the most important immunodominant epitope recognized by human sera [26]. Therefore, the RBE may contribute greatly to the antigenic stability of MV. The roles of NE and LE are still unclear, but it is interesting that they interact to form the dimer-dimer interface in a certain form of the MV-H tetramer. LE may also interact with the F protein. These epitopes may be involved in a step(s) in the membrane fusion cascade, mediated co-operatively by the H and F proteins. The structural and functional roles in the protein-protein interactions may constitute structural constraints against change to these epitopes. HNE is the best-known epitope of the MV-H protein, although its function is still unclear. Human sera recognize this epitope as an immunodominant epitope [26], and it must have an important role because its overall structure is conserved among the morbilliviruses. The epitope may be critical for the structural stability of the MV-H protein or it may function in the interactions between the multiple units of the H protein dimers and F protein trimers.

Currently used MV vaccines were generated from MV strains isolated more than 50 years ago. Nonetheless, they are still highly effective against the MV strains circulating at present. Sera obtained from convalescent measles patients and vaccines neutralize different MV strains, including vaccine strains and currently circulating strains, with similar efficiencies. This fact and our knowledge of the H protein epitopes strongly suggest that MV cannot undergo large antigenic changes, so the global elimination of MV should be feasible with the tools at hand. This comprehensive review of the epitopes of the MV-H protein will contribute to the global efforts to this end.

Acknowledgments: This research was partly supported by the Research Program on Emerging and Re-emerging Infectious Diseases of the Japan Agency for Medical Research and Development, AMED.

Author Contributions: Makoto Takeda, Maino Tahara, Katsumi Maenaka and Claude P. Muller conceived and designed the experiments; Maino Tahara, Jean-Philippe Bürckert and Kazuhiko Kanou performed the experiments; Makoto Takeda, Maino Tahara, Jean-Philippe Bürckert, Kazuhiko Kanou, Katsumi Maenaka and Claude P. Muller analyzed the data; Makoto Takeda, Maino Tahara, Jean-Philippe Bürckert and Claude P. Muller contributed reagents/materials; Makoto Takeda and Maino Tahara wrote the paper.

Conflicts of Interest: The authors declare no conflict of interest.

References

1. Tahara, M.; Bürckert, J.-P.; Kanou, K.; Maenaka, K.; Muller, C.P.; Takeda, M. Measles Virus Hemagglutinin Protein Epitopes: The Basis of Antigenic Stability. *Viruses* **2016**, *8*, 216. [CrossRef] [PubMed]
2. Bellini, W.J.; Rota, P.A. Biological feasibility of measles eradication. *Virus Res.* **2011**, *162*, 72–79. [CrossRef] [PubMed]
3. Strebel, P.M.; Cochi, S.L.; Hoekstra, E.; Rota, P.A.; Featherstone, D.; Bellini, W.J.; Katz, S.L. A world without measles. *J. Infect. Dis.* **2011**, *204*, 1–3. [CrossRef] [PubMed]
4. De Swart, R.L.; Yuksel, S.; Osterhaus, A.D. Relative contributions of measles virus hemagglutinin- and fusion protein-specific serum antibodies to virus neutralization. *J. Virol.* **2005**, *79*, 11547–11551. [CrossRef] [PubMed]
5. Bouche, F.B.; Ertl, O.T.; Muller, C.P. Neutralizing B cell response in measles. *Viral Immunol.* **2002**, *15*, 451–471. [CrossRef] [PubMed]
6. De Swart, R.L.; Yuksel, S.; Langerij, C.N.; Muller, C.P.; Osterhaus, A.D. Depletion of measles virus glycoprotein-specific antibodies from human sera reveals genotype-specific neutralizing antibodies. *J. Gen. Virol.* **2009**, *90*, 2982–2989. [CrossRef] [PubMed]
7. Tatsuo, H.; Ono, N.; Tanaka, K.; Yanagi, Y. SLAM (CDw150) is a cellular receptor for measles virus. *Nature* **2000**, *406*, 893–897. [PubMed]
8. Noyce, R.S.; Bondre, D.G.; Ha, M.N.; Lin, L.T.; Sisson, G.; Tsao, M.S.; Richardson, C.D. Tumor cell marker PVRL4 (nectin 4) is an epithelial cell receptor for measles virus. *PLoS Pathog.* **2011**, *7*, e1002240. [CrossRef] [PubMed]
9. Muhlebach, M.D.; Mateo, M.; Sinn, P.L.; Prüfer, S.; Uhlig, K.M.; Leonard, V.H.; Navaratnarajah, C.K.; Frenzke, M.; Wong, X.X.; Sawatsky, B.; et al. Adherens junction protein nectin-4 is the epithelial receptor for measles virus. *Nature* **2011**, *480*, 530–533. [CrossRef] [PubMed]
10. Takeda, M.; Tahara, M.; Nagata, N.; Seki, F. Wild-type measles virus is intrinsically dual-tropic. *Front. Microbiol.* **2011**. [CrossRef] [PubMed]
11. Yanagi, Y.; Takeda, M.; Ohno, S.; Hashiguchi, T. Measles virus receptors. *Curr. Top. Microbiol. Immunol.* **2009**, *329*, 13–30. [PubMed]
12. Jardetzky, T.S.; Lamb, R.A. Activation of paramyxovirus membrane fusion and virus entry. *Curr. Opin. Virol.* **2014**, *5*, 24–33. [CrossRef] [PubMed]
13. Finsterbusch, T.; Wolbert, A.; Deitemeier, I.; Meyer, K.; Mosquera, M.M.; Mankertz, A.; Santibanez, S. Measles viruses of genotype H1 evade recognition by vaccine-induced neutralizing antibodies targeting the linear haemagglutinin noose epitope. *J. Gen. Virol.* **2009**, *90*, 2739–2745. [CrossRef] [PubMed]
14. Shi, J.; Zheng, J.; Huang, H.; Hu, Y.; Bian, J.; Xu, D.; Li, F. Measles incidence rate and a phylogenetic study of contemporary genotype H1 measles strains in China: Is an improved measles vaccine needed? *Virus Genes* **2011**, *43*, 319–326. [CrossRef] [PubMed]

15. Tamin, A.; Rota, P.A.; Wang, Z.D.; Heath, J.L.; Anderson, L.J.; Bellini, W.J. Antigenic analysis of current wild type and vaccine strains of measles virus. *J. Infect. Dis.* 1994, **170**, 795–801. [CrossRef](https://doi.org/10.1093/infdis/170.4.795) [PubMed](https://pubmed.ncbi.nlm.nih.gov/7926918)

16. Kuhne, M.; Brown, D.W.; Jin, L. Genetic variability of measles virus in acute and persistent infections. *Infect. Genet. Evol.* 2006, **6**, 269–276. [CrossRef](https://doi.org/10.1016/j.meegid.2005.10.003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16298602)

17. Santibanez, S.; Niewiesk, S.; Heider, A.; Schneider-Schaulies, J.; Berbers, G.A.; Zimmermann, A.; Halenius, A.; Wolbert, A.; Deitmeier, I.; Tischer, A.; et al. Probing neutralizing-antibody responses against emerging measles viruses (MVs): Immune selection of MV by H protein-specific antibodies? *J. Gen. Virol.* 2005, **86**, 365–374. [CrossRef](https://doi.org/10.1099/vir.0.80663-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15722539)

18. Hashiguchi, T.; Ose, T.; Kubota, M.; Maita, N.; Kamishikiryo, J.; Maenaka, K.; Yanagi, Y. Structure of the measles virus hemagglutinin bound to its cellular receptor SLAM. *Nat. Struct. Mol. Biol.* 2011, **18**, 135–141. [CrossRef](https://doi.org/10.1038/nsmb.1966) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21186355)

19. Zhang, X.; Lu, G.; Qi, J.; Li, Y.; He, Y.; Xu, X.; Shi, J.; Zhang, C.W.; Yan, J.; Gao, G.F. Structure of measles virus hemagglutinin bound to its epithelial receptor nectin-4. *Nat. Struct. Mol. Biol.* 2012, **20**, 67–72. [CrossRef](https://doi.org/10.1038/nsmb.2225) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22968026)

20. Santiago, C.; Celma, M.L.; Stehle, T.; Casasnovas, J.M. Structure of the measles virus hemagglutinin bound to the CD46 receptor. *Nat. Struct. Mol. Biol.* 2010, **17**, 124–129. [CrossRef](https://doi.org/10.1038/nsmb.1739) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20062054)

21. Bose, S.; Jardetzky, T.S.; Lamb, R.A. Timing is everything: Fine-tuned molecular machines orchestrate paramyxovirus entry. *Virology* 2015, **479**, 518–531. [CrossRef](https://doi.org/10.1016/j.virol.2015.03.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25843386)

22. Hashiguchi, T.; Kajikawa, M.; Maita, N.; Takeda, M.; Kuroki, K.; Sasaki, K.; Kohda, D.; Yanagi, Y.; Maenaka, K. Crystal structure of measles virus hemagglutinin provides insight into effective vaccines. *Proc. Natl. Acad. Sci. USA* 2007, **104**, 19535–19540. [CrossRef](https://doi.org/10.1073/pnas.0705161104) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17901208)

23. Ziegler, D.; Fournier, P.; Berbers, G.A.; Steuer, H.; Wiesmuller, K.H.; Fleckenstein, B.; Schneider, F.; Jung, G.; King, C.C.; Muller, C.P. Protection against measles virus encephalitis by monoclonal antibodies binding to a cystine loop domain of the H protein mimicked by peptides which are not recognized by maternal antibodies. *J. Gen. Virol.* 1996, **77**, 2479–2489. [CrossRef](https://doi.org/10.1099/0022-1317-77-10-2479) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8830757)

24. Putz, M.M.; Hoebelke, J.; Ammerlaan, W.; Schneider, S.; Muller, C.P. Functional fine-mapping and molecular modeling of a conserved loop epitope of the measles virus hemagglutinin protein. *Eur. J. Biochem. FEBS* 2003, **270**, 1515–1527. [CrossRef](https://doi.org/10.1046/j.1432-1033.2003.03468.x)

25. Lapthorn, A.J.; Janes, R.W.; Isaacs, N.W.; Wallace, B.A. Cystine nooses and protein specificity. *Nat. Struct. Biol.* 1995, **2**, 266–268. [CrossRef](https://doi.org/10.1038/nsb0495-266) [PubMed](https://pubmed.ncbi.nlm.nih.gov/7620057)

26. Ertl, O.T.; Wenz, D.C.; Bouche, F.B.; Berbers, G.A.; Muller, C.P. Immunodominant domains of the Measles virus hemagglutinin protein eliciting a neutralizing human B cell response. *Arch. Virol.* 2003, **148**, 2195–2206. [CrossRef](https://doi.org/10.1007/s00705-003-0140-7) [PubMed](https://pubmed.ncbi.nlm.nih.gov/14502258)

27. Sato, T.A.; Fukuda, A.; Sugiura, A. Characterization of major structural proteins of measles virus with monoclonal antibodies. *J. Gen. Virol.* 1985, **66**, 1397–1409. [CrossRef](https://doi.org/10.1099/0022-1317-66-7-1397) [PubMed](https://pubmed.ncbi.nlm.nih.gov/3893893)

28. Tahara, M.; Ito, Y.; Brindley, M.A.; Ma, X.; He, J.; Xu, S.; Fukuhara, H.; Sakai, K.; Komase, K.; Rota, P.A.; Plemper, R.K.; Maenaka, K.; Takeda, M. Functional and structural characterization of neutralizing epitopes of measles virus hemagglutinin protein. *J. Virol.* 2013, **87**, 666–675. [CrossRef](https://doi.org/10.1128/JVI.02085-12) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23115200)

29. Lech, P.J.; Tobin, G.J.; Bushnell, R.; Gutschenritter, E.; Pham, L.D.; Nace, R.; Verhoeeyen, E.; Cosset, F.L.; Muller, C.P.; Russell, S.J.; et al. Epitope dampening monotypic measles virus hemagglutinin glycoprotein results in resistance to cocktail of monoclonal antibodies. *PLoS ONE* 2013, **8**, e52306. [CrossRef](https://doi.org/10.1371/journal.pone.0052306) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23383102)

30. Liebert, U.G.; Flanagan, S.G.; Loffler, S.; Baczko, K.; ter Meulen, V.; Rima, B.K. Antigenic determinants of measles virus hemagglutinin associated with neurovirulence. *J. Virol.* 1994, **68**, 1486–1493. [PubMed](https://pubmed.ncbi.nlm.nih.gov/8107208)

31. Xu, S.; Zhang, Y.; Zhu, Z.; Liu, C.; Mao, N.; Ji, Y.; Wang, H.; Jiang, X.; Li, C.; Tang, W.; et al. Genetic characterization of the hemagglutinin genes of wild-type measles virus circulating in China, 1993–2009. *PLoS ONE* 2013, **8**, e73374. [CrossRef](https://doi.org/10.1371/journal.pone.0073374) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24023758)

32. Tahara, M.; Yasui, Y.; Minagawa, H.; Komase, K.; Takeda, M. Characterization of measles virus strains in Japan. Unpublished work.

33. Makela, M.J.; Salmi, A.A.; Norrby, E.; Wild, T.F. Monoclonal antibodies against measles virus haemagglutinin react with synthetic peptides. *Scand. J. Immunol.* 1989, **30**, 225–231. [CrossRef](https://doi.org/10.1111/j.1365-3083.1989.tb02488.x) [PubMed](https://pubmed.ncbi.nlm.nih.gov/2553493)

34. Lech, P.J.; Pappoe, R.; Nakamura, T.; Tobin, G.J.; Nara, P.L.; Russell, S.J. Antibody neutralization of retargeted measles viruses. *Virology* 2014, **454**, 237–246. [CrossRef](https://doi.org/10.1016/j.virol.2014.02.007) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24582848)

35. Hu, A.; Sheshberadaran, H.; Norrby, E.; Kovamees, J. Molecular characterization of epitopes on the measles virus hemagglutinin protein. *Virology* **1993**, *192*, 351–354. [CrossRef](https://doi.org/10.1016/0042-6822(93)90276-8) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8398036)

36. Hummel, K.B.; Bellini, W.J. Localization of monoclonal antibody epitopes and functional domains in the hemagglutinin protein of measles virus. *J. Virol.* **1995**, *69*, 1913–1916. [PubMed](https://pubmed.ncbi.nlm.nih.gov/7883850)

37. Kato, S.; Ohgimoto, S.; Sharma, L.B.; Kurazono, S.; Ayata, M.; Komase, K.; Takeda, M.; Takeuchi, K.; Ihara, T.; Ogura, H. Reduced ability of hemagglutinin of the CAM-70 measles virus vaccine strain to use receptors CD46 and SLAM. *Vaccine* **2009**, *27*, 3838–3848. [CrossRef](https://doi.org/10.1016/j.vaccine.2009.03.039) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19327586)

38. Masse, N.; Ainouze, M.; Neel, B.; Wild, T.F.; Buckland, R.; Langedijk, J.P. Measles virus (MV) hemagglutinin: Evidence that attachment sites for MV receptors SLAM and CD46 overlap on the globular head. *J. Virol.* **2004**, *78*, 9051–9063. [CrossRef](https://doi.org/10.1128/JVI.78.17.9051-9063.2004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15308725)

39. Seki, F.; Yamada, K.; Nakatsu, Y.; Okamura, K.; Yanagi, Y.; Nakayama, T.; Komase, K.; Takeda, M. The SI strain of measles virus derived from a patient with subacute sclerosing panencephalitis possesses typical genome alterations and unique amino acid changes that modulate receptor specificity and reduce membrane fusion activity. *J. Virol.* **2011**, *85*, 11871–11882. [CrossRef](https://doi.org/10.1128/JVI.05348-11) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21849448)

40. Tahara, M.; Ohno, S.; Sakai, K.; Ito, Y.; Fukuhara, H.; Komase, K.; Brindley, M.A.; Rota, P.A.; Plemper, R.K.; Maenaka, K.; et al. The receptor-binding site of the measles virus hemagglutinin protein itself constitutes a conserved neutralizing epitope. *J. Virol.* **2013**, *87*, 3583–3586. [CrossRef](https://doi.org/10.1128/JVI.02914-12) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23363788)

41. Seki, F.; Someya, K.; Komase, K.; Takeda, M. A chicken homologue of nectin-4 functions as a measles virus receptor. *Vaccine* **2016**, *34*, 7–12. [CrossRef](https://doi.org/10.1016/j.vaccine.2015.11.045) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26626858)

42. Hu, A.; Cattaneo, R.; Schwartz, S.; Norrby, E. Role of N-linked oligosaccharide chains in the processing and antigenicity of measles virus haemagglutinin protein. *J. Gen. Virol.* **1994**, *75*, 1043–1052. [CrossRef](https://doi.org/10.1099/0022-1317-75-5-1043) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8188950)

43. Santiago, C.; Bjorling, E.; Stehle, T.; Casasnovas, J.M. Distinct kinetics for binding of the CD46 and SLAM receptors to overlapping sites in the measles virus hemagglutinin protein. *J. Biol. Chem.* **2002**, *277*, 32294–32301. [CrossRef](https://doi.org/10.1074/jbc.M202178200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11967297)

44. Sheshberadaran, H.; Norrby, E. Characterization of epitopes on the measles virus hemagglutinin. *Virology* **1986**, *152*, 58–65. [CrossRef](https://doi.org/10.1016/0042-6822(86)90368-9)

45. Sheshberadaran, H.; Payne, L.G. Protein antigen-monoclonal antibody contact sites investigated by limited proteolysis of monoclonal antibody-bound antigen: Protein “footprinting”. *Proc. Natl. Acad. Sci. USA* **1988**, *85*, 1–5. [CrossRef](https://doi.org/10.1073/pnas.85.1.1) [PubMed](https://pubmed.ncbi.nlm.nih.gov/3271303)

46. Sakata, H.; Kobune, F.; Sato, T.A.; Tanabayashi, K.; Yamada, A.; Sugiura, A. Variation in field isolates of measles virus during an 8-year period in Japan. *Microbiol. Immunol.* **1993**, *37*, 233–237. [CrossRef](https://doi.org/10.1111/j.1348-0421.1993.tb01829.x) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8398036)

47. Rota, J.S.; Hummel, K.B.; Rota, P.A.; Bellini, W.J. Genetic variability of the glycoprotein genes of current wild-type measles isolates. *Virology* **1992**, *188*, 135–142. [CrossRef](https://doi.org/10.1016/0042-6822(92)90488-8)

48. Levy, C.; Amirache, F.; Costa, C.; Frecha, C.; Muller, C.P.; Kweder, H.; Buckland, R.; Cosset, F.L.; Verhoeyen, E. Lentiviral vectors displaying modified measles virus gp overcome pre-existing immunity in in vivo-like transduction of human T and B cells. *Mol. Ther.* **2012**, *20*, 1699–1712. [CrossRef](https://doi.org/10.1038/mt.2012.87) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22549423)

49. Patterson, J.B.; Scheiflinger, F.; Manchester, M.; Yilma, T.; Oldstone, M.B. Structural and functional studies of the measles virus hemagglutinin: Identification of a novel site required for CD46 interaction. *Virology* **1999**, *256*, 142–151. [CrossRef](https://doi.org/10.1006/viro.1999.9668) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10074556)

50. Kimura, H.; Saitoh, M.; Kobayashi, M.; Ishii, H.; Saraya, T.; Kurai, D.; Tsukagoshi, H.; Shirabe, K.; Nishina, A.; Kozawa, K.; et al. Molecular evolution of haemagglutinin (H) gene in measles virus. *Sci. Rep.* **2015**. [CrossRef](https://doi.org/10.1038/srep12808) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26205995)

51. World Health Organization. Measles virus nomenclature update: 2012. *Wkly. Epidemiol. Rec.* **2012**, *87*, 73–81.

52. Nagai, M.; Xin, J.Y.; Yoshida, N.; Miyata, A.; Fujino, M.; Ihara, T.; Yoshikawa, T.; Asano, Y.; Nakayama, T. Modified adult measles in outbreaks in Japan, 2007–2008. *J. Med. Virol.* **2009**, *81*, 1094–1101. [CrossRef](https://doi.org/10.1002/jmv.21516) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19452402)

53. Fournier, P.; Brons, N.H.; Berbers, G.A.; Wiesmuller, K.H.; Fleckenstein, B.T.; Schneider, F.; Jung, G.; Muller, C.P. Antibodies to a new linear site at the topographical or functional interface between the haemagglutinin and fusion proteins protect against measles encephalitis. *J. Gen. Virol.* **1997**, *78*, 1295–1302. [CrossRef](https://doi.org/10.1099/0022-1317-78-6-1295) [PubMed](https://pubmed.ncbi.nlm.nih.gov/9184483)

54. Paal, T.; Brindley, M.A.; St Clair, C.; Prussia, A.; Gaus, D.; Krumm, S.A.; Snyder, J.P.; Plemper, R.K. Probing the spatial organization of measles virus fusion complexes. *J. Virol.* **2009**, *83*, 10480–10493. [CrossRef](https://doi.org/10.1128/JVI.00939-09) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19648268)

55. Apte-Sengupta, S.; Negi, S.; Leonard, V.H.; Oezguen, N.; Navaratnarajah, C.K.; Braun, W.; Cattaneo, R. Base of the measles virus fusion trimer head receives the signal that triggers membrane fusion. *J. Biol. Chem.* **2012**, *287*, 33026–33035. [CrossRef](https://doi.org/10.1074/jbc.M112.392080) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22815501)

56. Brindley, M.A.; Plemper, R.K. Blue native PAGE and biomolecular complementation reveal a tetrameric or higher-order oligomer organization of the physiological measles virus attachment protein H. *J. Virol.* **2010**, *84*, 12174–12184. [CrossRef](https://doi.org/10.1128/JVI.00323-10) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20479128)

57. Ader, N.; Brindley, M.A.; Avila, M.; Origgi, F.C.; Langedijk, J.P.; Orvell, C.; Vandevelde, M.; Zurbriggen, A.; Plemper, R.K.; Plattet, P. Structural rearrangements of the central region of the morbillivirus attachment protein stalk domain trigger F protein refolding for membrane fusion. *J. Biol. Chem.* **2012**, *287*, 16324–16334. [CrossRef](https://doi.org/10.1074/jbc.M112.349155) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22427638)

58. Nakashima, M.; Shirogane, Y.; Hashiguchi, T.; Yanagi, Y. Mutations in the putative dimer-dimer interfaces of the measles virus hemagglutinin head domain affect membrane fusion triggering. *J. Biol. Chem.* **2013**, *288*, 8085–8091. [CrossRef](https://doi.org/10.1074/jbc.M112.440887) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23358805)

59. Ayata, M.; Tanaka, M.; Kameoka, K.; Kuwamura, M.; Takeuchi, K.; Takeda, M.; Kanou, K.; Ogura, H. Amino acid substitutions in the heptad repeat A and C regions of the F protein responsible for neurovirulence of measles virus Osaka-1 strain from a patient with subacute sclerosing panencephalitis. *Virology* **2016**, *487*, 141–149. [CrossRef](https://doi.org/10.1016/j.virol.2015.10.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26549378)

60. Otsuki, N.; Sekizuka, T.; Seki, F.; Sakai, K.; Kubota, T.; Nakatsu, Y.; Chen, S.; Fukuhara, H.; Maenaka, K.; Yamaguchi, R.; et al. Canine distemper virus with the intact C protein has the potential to replicate in human epithelial cells by using human nectin4 as a receptor. *Virology* **2013**, *435*, 485–492. [CrossRef](https://doi.org/10.1016/j.virol.2012.10.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23123520)

61. Takeda, M.; Ohno, S.; Tahara, M.; Takeuchi, H.; Shirogane, Y.; Ohmura, H.; Nakamura, T.; Yanagi, Y. Measles viruses possessing the polymerase protein genes of the Edmonston vaccine strain exhibit attenuated gene expression and growth in cultured cells and SLAM knock-in mice. *J. Virol.* **2008**, *82*, 11979–11984. [CrossRef](https://doi.org/10.1128/JVI.01362-08) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18757878)

62. Takeda, M.; Takeuchi, K.; Miyajima, N.; Kobune, F.; Ami, Y.; Nagata, N.; Suzaki, Y.; Nagai, Y.; Tashiro, M. Recovery of pathogenic measles virus from cloned cDNA. *J. Virol.* **2000**, *74*, 6643–6647. [CrossRef](https://doi.org/10.1128/JVI.74.15.6643-6647.2000) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10888645)
