
ARTICLE IN PRESS

BBAMCB-58032; No. of pages: 10; 4C: 2, 7

Biochimica et Biophysica Acta xxx (2016) xxx–xxx

---

Contents lists available at ScienceDirect

**Biochimica et Biophysica Acta**

journal homepage: www.elsevier.com/locate/bbalip

---

# Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology

Alexander J. Kastaniotis ${ }^{a,*}$, Kaija J. Autio ${ }^{b}$, Juha M. Kerätär ${ }^{a}$, Geoffray Monteuuis ${ }^{a}$, Anne M. Mäkelä ${ }^{a}$, Remya R. Nair ${ }^{a}$, Laura P. Pietikäinen ${ }^{a}$, Antonina Shvetsova ${ }^{a}$, Zhijun Chen ${ }^{b}$, J. Kalervo Hiltunen ${ }^{a,b,*}$

${ }^{a}$ Faculty of Biochemistry and Molecular Medicine, Biocenter Oulu, University of Oulu, Oulu, Finland  
${ }^{b}$ State Key Laboratory of Supramolecular Structure and Materials and Institute of Theoretical Chemistry, Jilin University, 2699 Qianjin Street, Changchun 130012, PR China

---

## Article Info

### Article history:
- Received 15 May 2016
- Received in revised form 20 July 2016
- Accepted 17 August 2016
- Available online xxxx

### Keywords:
- Fatty acid synthesis
- Thioesters
- Metabolic compartmentalization
- Lipoic acid
- Respiration
- Metabolic interaction
- Lipids
- Pathways

---

## Abstract

Mitochondria and fatty acids are tightly connected to a multiplicity of cellular processes that go far beyond mitochondrial fatty acid metabolism. In line with this view, there is hardly any common metabolic disorder that is not associated with disturbed mitochondrial lipid handling. Among other aspects of mitochondrial lipid metabolism, apparently all eukaryotes are capable of carrying out *de novo* fatty acid synthesis (FAS) in this cellular compartment in an acyl carrier protein (ACP)-dependent manner. The dual localization of FAS in eukaryotic cells raises the questions why eukaryotes have maintained the FAS in mitochondria in addition to the “classic” cytoplasmic FAS and what the products are that cannot be substituted by delivery of fatty acids of extramitochondrial origin. The current evidence indicates that mitochondrial FAS is essential for cellular respiration and mitochondrial biogenesis. Although both β-oxidation and FAS utilize thioester chemistry, CoA acts as acyl-group carrier in the breakdown pathway whereas ACP assumes this role in the synthetic direction. This arrangement metabolically separates these two pathways running towards opposite directions and prevents futile cycling. A role of this pathway in mitochondrial metabolic sensing has recently been proposed. This article is part of a Special Issue entitled: Lipids of Mitochondria edited by Guenther Daum.

© 2016 Elsevier B.V. All rights reserved.

---

## 1. Introduction

The metabolism of living organisms is exposed to a large variety of fatty acids originating either from exogenous sources or arising from endogenous synthesis. Depending on the cellular compartment, these lipid carboxylic acids play multiple roles in cellular processes. Concerning mitochondria, fatty acids serve as energy sources, membrane constituents, molecules for post-translational modifications of proteins and as signaling factors. The largest fraction of fatty acid metabolism proceeds through fatty acyl thioesters, which are more reactive than their oxyster counterparts such as in sterolesters, phospholipids or triacylglycerols [1]. In many organisms, similar fatty acid processing reactions are frequently found to occur in several different cellular compartments. Pieces of evidence provided from many sources indicate that these reactions are not mere metabolic redundancies, but are either functionally complementary to each other or serve completely different metabolic roles. Among the many examples illustrating this observation are mitochondrial and peroxisomal β-oxidation pathways, synthesis of 3-hydroxymethylglutaryl-CoA for the generation of cholesterol in the cytosol or for ketogenesis in liver mitochondria, duplicity of fatty acid synthesis (FAS) in cytosol and mitochondria, presence of multiple acyl-CoA synthetases, acyl-CoA thioesterases and fatty acid binding proteins in various subcellular compartments.

As a recent addition to the plethora of mitochondrial processes, it has been recognized that these organelles are able to synthesize fatty acids in an acyl carrier protein (ACP)-dependent manner (mitochondrial fatty acid synthesis, mtFAS) [2–5]. Yeast strains deficient in mtFAS function display failure in mitochondrial RNA processing, highly diminished cytochromes of the mitochondrial respiratory chain and loss of lipoic acid. Consequently, these cells are unable to respire and can grow only on fermentable carbon source. Concerning mammals, the total knock-out of mtFAS pathway components in mice leads to embryonic lethality (our unpublished observations), and an induced conditional knockout of the fatty acid synthetic Mcat, the gene encoding the mitochondrial malonyl-CoA:ACP transacylase in adult mice, results in severe defects resembling phenotypes of other mitochondrial dysfunction models [6]. The dual localization of FAS in eukaryotic cells raises the question why eukaryotes have maintained the FAS in mitochondria in addition to the “classic” cytoplasmic FAS (FASt). It is noteworthy that functional cytoplasmic fatty acid synthesis or fatty acids added externally to the culture medium cannot rescue the respiratory deficient

http://dx.doi.org/10.1016/j.bbalip.2016.08.011

1388-1981/© 2016 Elsevier B.V. All rights reserved.

---

Please cite this article as: A.J. Kastaniotis, et al., Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology, Biochim. Biophys. Acta (2016), http://dx.doi.org/10.1016/j.bbalip.2016.08.011

ARTICLE IN PRESS

A.J. Kastaniotis et al. / Biochimica et Biophysica Acta xxx (2016) xxx–xxx

phenotype in yeast. This review summarizes our current understanding on mitochondrial FAS and its relations to other mitochondrial processes and physiology.

## 2. Components of mitochondrial fatty acid synthesis

Although the first component of mtFAS – a fungal mitochondrial acyl carrier protein (ACP) [7] – was identified more than a quarter century ago, mtFAS has remained to be perhaps one of the most puzzling aspects of mitochondrial lipid metabolism. While the pathway is highly conserved in eukaryotes in general, there are more or less subtle inter-species variations in the composition of the enzymes involved. Questions about some of the components involved in mtFAS in mammals as well as the overall physiologically relevant spectrum of products of this pathway persist. Nevertheless, the data produced and collected over the past three decades has allowed the identification of surprising links of mtFAS to the regulation of mitochondrial enzyme activities, RNA processing and control of mitochondrial gene expression. The mtFAS pathway (Fig. 1A) closely resembles a bacterial type II FAS system (FASII). In contrast to the eukaryotic cytosolic FASI machinery, where the required enzymatic activities are harbored by one or two polypeptides constituting multifunctional proteins carrying out all the steps of FAS, the FASII route proceeds by the action of individual polypeptides catalyzing the separate reactions [8]. Plant chloroplasts, the photosynthetic endosymbiont cousins of mitochondria, also follow a FASII mode, and it is not unreasonable to regard this phenomenon as corroborative evidence for the prokaryotic heritage of these organelles.

Despite the clearly prokaryotic origin of mtFAS, the systems found in yeasts and mammals appear to be mosaics, for which the components were recruited from different sources. Several of the proteins involved in mtFAS (Table 1) have been drafted straight from the bacterial pathway. The S. cerevisiae/human (mammalian) malonyl-CoA:ACP transferase (Mct1/MCAT) and the 3-ketoacyl-ACP synthase (KAS) (Cem1/OXSM) are highly similar to the enzymes acting in the Escherichia coli FASII system and can therefore be unequivocally identified as orthologs [9–11]. KAS enzymes and domains of multifunctional FASI polypeptides share a common architecture [12] that is also likely to make them susceptible to the same inhibitors. It has been shown previously that OXSM binds the drug cerulenin [11] and, like mammalian FAS, the yeast FASI complex containing a KAS domain [13] and the yeast mitochondrial Cem1 KAS [14], are inhibited by this compound. The speculation that OXSM is also affected by other KAS inhibitors in vivo was confirmed recently by Chen et al. [15] who showed that treatment of HEK293Tcells with the FAS inhibitor 4-methylene-2-octyl-5-oxotetrahydrofuran-3-carboxylic acid (C75) resulted in cellular defects consistent with mtFAS dysfunction. As suggested previously [11], it is likely that other KAS-inhibitors like platensimycin [16] and the FASII specific inhibitor thiolactomycin [17] will have similar effects.

Other components of mammalian mtFAS show subtle or strong divergence from the bacterial pathways. The mitochondrial ACP (Acp1 in Saccharomyces cerevisiae/NDUFAB1 in humans) is highly similar to prokaryotic ACPs. In addition to existing in a soluble pool, it has acquired a new role as a structural component of respiratory complex I in many non-plant eukaryotes [18–21]. Interestingly, the observation that mitochondrial ACP in Arabidopsis thaliana does not appear to be associated

![Figure 1](#fig1)

**Fig. 1.** Organization of fatty acid synthesis and β-oxidation in mitochondria. (A) Human enzymes/proteins of mtFAS in symbols are labeled in blue font, the corresponding or alternative S. cerevisiae enzymes are in red font. The red arrows indicate the influence of the functional mtFAS on assembly of mitochondrial respiratory chain and mitochondrial RNA processing as demonstrated using yeast as a model organism. (*) It is likely that insertion of sulfurs in the C6 and C8 positions of the octanoyl group by LIAS/Lip5 occurs on the target proteins (DLAT/Lat1 of PDH, DLST/Kgd2 of KGD, GCSH/Gcv3 of GCS and DBT of branched chain dehydrogenase in mammals) after transfer of the octanoyl moiety rather than on octanoyl-ACP. For discussion, see the text, Section 3. The abbreviations for the names of enzymes are as given in the Table 2. (B) Mitochondrial β-oxidation pathway is organized into a membrane associated and a matrix set of enzymes. Although not shown in the Figure, the matrix set of the enzymes is apparently organized into a multienzyme complex [123,124]. The abbreviations: CACT, carnitine acyl-carnitine translocase; CPT1, carnitine palmitoyltransferase 1 A or B; CPT2, carnitine palmitoyltransferase 2; ECHS1, enoyl-CoA hydratase 1, (crotonase); HADH, 3S-hydroxyacyl-CoA dehydrogenase; MCAD, medium-chain acyl-CoA dehydrogenase; MCKAT, medium-chain 3-ketoacyl-CoA thiolase; MTP, mitochondrial trifunctional protein containing long-chain 3-hydroxyacyl-CoA dehydrogenase and enoyl-CoA hydratase activities in the α-subunit and long-chain 3-ketoacyl-CoA thiolase activity in the β-subunit; SCAD, short-chain acyl-CoA dehydrogenase; VLCAD, very long-chain acyl-CoA dehydrogenase.

Please cite this article as: A.J. Kastaniotis, et al., Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology, Biochim. Biophys. Acta (2016), http://dx.doi.org/10.1016/j.bbalip.2016.08.011

ARTICLE IN PRESS

A.J. Kastaniotis et al. / Biochimica et Biophysica Acta xxx (2016) xxx–xxx 3

Table 1  
Components of the mitochondrial fatty acid and lipoic acid biosynthesis pathways.

| Name | Gene | EC number | Reference |
| --- | --- | --- | --- |
| Acetyl-CoA carboxylase | – | HFA1 | [25,28] |
| Malonyl-CoA synthetase | ACSF3 | – | [36] |
| Phosphopantethenyl–ACP transferase | AASDHPPT | PPT2 | [23,115] |
| Acyl Carrier Protein | NDUFAB1 | ACP1 | [3,39] |
| Malonyl-CoA::ACP transferase | MCAT | MCT1 | [10,116] |
| 3-Ketoacyl-ACP synthase | OXSM | CEM1 | [9,14] |
| 3-Oxoacyl-ACP reductase | KAR1 (SDH17B8 and CBR4) | OAR1 | [10,43] |
| 3-Hydroxyacyl-ACP dehydratase | HsHTD2* (RPP14/HsHTD2 transcript) | HTD2 | [44] |
| 2-Enoyl-ACP reductase | MECR | ETR1 | [4,117] |
| Lipoyltransferase 1 | LIPT1 | LIP3 | [72,73] |
| Lipoyltransferase 2 | LIPT2 | LIP2 | [71] |
| Lipoic acid synthetase | LIAS | LIP5 | [59,69] |
| Glycine cleavage system protein H | GCSH | GCV3 | [118,119] |
| NFU1 iron-sulfur cluster scaffold | NFU1 | NFU1 | [120–122] |

Gene names and protein functions are according to NCBI, the EC number is according to BRENDA enzyme database http://www.brenda-enzymes.org/.

with complex I [22] raises the possibility that a role of ACP in eukaryotic complex I structure/function is not universal. Of note, S. cerevisiae mitochondria do not contain a respiratory complex I but instead use non-coupling NADH dehydrogenases, and hence cannot serve as a model to investigate the role of ACP in complex I activity.

The yeast mitochondrial 4-phosphopantetheinyl transferase (PPTase) (Ppt2) required in the production of phosphopantetheinylated holo-ACP, also bears high similarity to bacterial acpS-like enzymes with this role [23]. A gene encoding such a PPTase in mammals could not be identified. However, an enzyme homologous to yeast Lys5 α-amino adipate reductase-PPTase (AASDHPPT) has been found to accept mammalian apo-ACP as a substrate, in addition to other phosphopantetheinylation targets [24]. This protein has not been reported to be localized to mitochondria, leaving an open question on where the conversion of apo- to holo-ACP takes place in mammals.

The S. cerevisiae Hfa1 acetyl-CoA carboxylase (ACC) harbors all activities required for malonyl-CoA synthesis on one polypeptide [25], while the bacterial counterparts assemble from several distinct subunits [8]. Sequence analysis indicates that this yeast ACC has been borrowed from the cytosolic FAS pathway. Following a whole genome duplication event that yielded in an initially tetraploid ancestor of modern day baker's yeast [26,27], the gene encoding the S. cerevisiae mitochondrial ACC has apparently diverged from an ancestral form that initially coded for two variants of the enzyme. Based on comparative sequence evidence, it has been postulated that, translation of a cytosolic isoform of ACC was initiated in the ancestral fungal gene by a canonical AUG start codon. In addition, a mitochondrial variant was synthesized from a non-AUG translation initiation codon further upstream that yielded in the addition of an in-frame N-terminal extension with the function of a mitochondrial targeting sequence to the ACC [28]. Sequence analysis and complementation studies presented in this paper indicated that this arrangement can still be found in the yeast Kluyveromyces lactis as well as other yeasts.

The use of non-AUG translation initiation codons in yeast has been documented for artificial constructs and for several aminoacyl-tRNA synthetases [29–33]. As in the case of the ancestral fungal ACC, cytosolic and mitochondrial isoforms of these enzymes are produced by alternative initiation of translation from canonical or non-AUG start codons, respectively. A remarkable feature of the HFA1 gene encoding S. cerevisiae mitochondrial ACC is the fact that it is the only known polypeptide-encoding cistron in this yeast that does not produce a functional protein when translated from the first in-frame AUG codon [25]. It has been pointed out in two separate publications that HFA1 appears to be a gene undergoing accelerated evolution compared to ACC1 encoding the duplicate cytosolic counterpart in S. cerevisiae [27,34]. In their report, Turunen and coworkers highlighted that the isoelectric point of the Hfa1 ACC (8.15) is remarkably higher than the pl of its cytosolic counterpart (6.22), with the most drastic change observed in the carboxyltransferase domain of the protein (8.7 in Hfa1, 5.45 in Acc1). This difference may indicate an adaptation of Hfa1 to the pH environment of the mitochondrial matrix, which is more basic than the cytosol. Suomi et al. also raised the intriguing question on the localization of biotin protein ligase/holocarboxylase synthase (Bpl1), the enzyme required for biotin attachment to Hfa1 and Acc1 [28]. Comparative genomic analyses indicated that the BPL1 ORF in several fungal genomes may be preceded by a sequence encoding a cryptic, non-AUG initiated mitochondrial targeting sequence, like K. lactis ACC and the ancestral form of HFA1/ACC1.

No mitochondrial ACC has been described in mammals. However, propionyl-CoA carboxylase, a mitochondrial enzyme in mammals can also catalyze carboxylation of acetyl-CoA in vitro [35], although with low catalytic efficiency. As an alternative, it has been suggested instead that mammalian ACSF3 malonyl-CoA synthetase can act as the supplier of the starting substrate of mtFAS [36]. The malonate required for this reaction may be imported into mammalian mitochondria by the carnitine shuttle process [37]. Unlike in the case recently described for the mitochondrially localized isoform of A. thaliana AEE18 malonyl-CoA synthetase [38], the RNAi-mediated knockdown of ACSF3 expression does not affect protein lipoylation in mammals, in contrast to the lipoylation phenotypes produced by inactivation of other mammalian mtFAS components [6,15,39]. This apparent lack of any effect of ACSF3 depletion on the abundance of the mtFAS products indicates that the quest for identification of the source for mitochondrial malonyl-CoA may not yet be over. This issue is also not yet entirely settled in plants. Abolishing the mitochondrial AEE18 isoform of ACSF3 does not completely abrogate lipoylation in A. thaliana [38], and mitochondrially localized ACC isoforms have been described in monocot plants [40,41].

Yeast Oar1 mitochondrial 3-ketoacyl-ACP reductase (KAR) is highly similar to bacterial FASII ketoreductases [10,42]. The mammalian enzyme, however, displays an interesting twist in its quaternary structure. Human mitochondrial 3-ketoacyl-ACP reductase is composed as a dimer of dimers (α₂β₂), containing two 17β-hydroxysteroid dehydrogenase type 8 (HSD17B8; α-subunit) polypeptides, and two molecules of carbonyl reductase type 4 (CBR4; β-subunit) [42]. Both proteins have high similarity to Oar1 and the E. coli FASII ketoreductase FabG. When CBR4 and HSD17B8 are co-expressed in E. coli, the proteins exclusively co-purify as α₂β₂ tetrameric complex, and both proteins need to be expressed simultaneously in yeast to rescue respiratory deficiency of the oar1 mutant [42]. As elaborated on further below, the presence of HSD17B8 in this arrangement links mtFAS to β-oxidation in mammalian mitochondria [43], perhaps indicating a need to coordinate these pathways in organelles where fatty acid catabolic and anabolic processes run in parallel. The mammalian mitochondrial KAR has recently been subjected to increased structural and functional scrutiny [43].

Please cite this article as: A.J. Kastaniotis, et al., Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology, Biochim. Biophys. Acta (2016), http://dx.doi.org/10.1016/j.bbalip.2016.08.011

ARTICLE IN PRESS

A.J. Kastaniotis et al. / Biochimica et Biophysica Acta xxx (2016) xxx–xxx

Both the HSD17B8 and CBR4 components of human KAR (HsKAR) are highly similar to each other in structure and belong to the short chain dehydrogenase/reductase (SDR) protein superfamily. The $\alpha_{2}\beta_{2}$ heterotetrameric arrangement of the polypeptides sets this enzyme apart from other KARs. Studies have now shown that the $\beta$-subunit is the catalytic component required for the NADPH-dependent KAR reaction in this complex. Mutating the active site of CBR4 will abolish the activity *in vitro* and interfere with complementation of the respiratory deficient phenotype of the yeast *oar1Δ* strain [43]. The corresponding mutations in the $\alpha$-subunit have no or very little effect on activity and complementation power of HsKAR. Likewise, mutations in residues of CBR4 predicted to play a role in interaction with ACP negatively affect yeast mutant growth on non-fermentable carbon sources, while changes in corresponding positions in HSD17B8 do not interfere with growth of the yeast mutant. With respect to the KAR activity of HsKAR, HSD17B8 appears to act as a scaffold that is required for solubility of CBR4 and its function. However, the HSD17B8 subunits of KAR are not catalytically inactive [43]. It was shown that the $\alpha$-subunit of HsKAR is capable of converting 3R-hydroxyacyl-CoA into 3-ketoacyl-CoA. The collected data indicated that the $\alpha_{2}\beta_{2}$ tetramer has two binding sites for each NAD(H) and NADP(H), with low and high affinity binding for each of the cofactors.

The mitochondrial 3-hydroxyacyl thioester ($-$ACP) dehydratases (HTDs) contain the hydratase-2 fingerprint and accordingly show similarity to prokaryotic acyl dehydratases, eukaryotic dehydratase domains in FASI and hydratase-2 domains of peroxisomal multifunctional $\beta$-oxidation enzymes (Fox2 in yeast and MFE-2 in mammals) [44,45]. While the mammalian HTDs and the *E. coli* FabA and FabZ proteins fall into the same class of maoC-type dehydratases, yeast Htd2 groups in a separate clade, showing similarity to highly conserved, not yet characterized proteobacterial or fungal enzymes [45]. Human HTD (HsHTD2) is part of an evolutionary branch of dehydratases epitomized by bacterial PhaJ 3-hydroxybutyryl-CoA dehydratase, an enzyme with a role in polyhydroxyalkanoate biosynthesis [45]. The transcript encoding HsHTD2 displays a curious structure: this sequence is harbored as a second ORF on a bicistronic mRNA, on which it is preceded by an ORF encoding the Rpp14 subunit of mammalian nuclear RNase P, literally linking RNA processing and mtFAS. While the RPP14-encoding part of this gene is interrupted by several introns, the entire HsHTD2 ORF is contained within a single exon. This arrangement, including the general intron - exon structure - has been conserved in the vertebrate lineage down to tunicates, where a bicistronic transcript can be found harboring HTD2, preceded by an ORF encoding a protein of unknown function [45]. The HTD2 ORF is not a pseudogene, as it has been reported to be translated and regulated by oxygen and glucose levels in the cellular environment [46]. The characterization of HsHtd2 as mammalian mtFAS dehydratase also allowed for the subsequent isolation of the mtFAS HTD in the parasite *Trypanosoma brucei* [47], which had not previously been possible [48]. The RPP14-HsHTD2 bicistronic arrangement has been used recently as a piece of evidence challenging the “one mRNA – one protein” dogma for eukaryotic gene expression [49].

The apparent strict conservation of the bicistronic gene structure for nearly half a billion years may indicate strong evolutionary selection for this arrangement [45]. Complementation of the yeast *htd2* mutant by the human cDNA construct also indicates that the mechanism of translation of alternative ORFs in eukaryotes may be conserved [49].

Mitochondrial 2-enoyl-thioester ($-$ACP) reductase (ETRs) in eukaryotes also represents a deviation from the canonical bacterial FASII pathway. Prokaryotes use enzymes with a short chain alcohol dehydrogenase/reductase (SDR) fold in this reaction, exemplified by the *E. coli* fabI ETR [8]. The eukaryotic mtFAS system relies on a medium chain alcohol dehydrogenase/reductase (MDR) – type ETR (Etr1/MECR) [4,48,50–52]. SDR and MDR enzymes are only very distantly related. In their divergence from the common ancestor, MDRs appear to have acquired an additional GroES-type domain, which is one of the features that distinguish these proteins from SDRs [53]. ME CR enzymes constitute a separate group in the MDR superfamily [54]. Despite Etr1 and FabI belonging to different protein superfamilies, expression of a mitochondrially localized form of bacterial fabI in the *S. cerevisiae etr1Δ* strain rescues the respiratory deficient phenotype in these cells, indicating that it is the lack of the ETR activity that is responsible for this defect [4].

## 3. Comments on products of mitochondrial fatty acid synthesis

The bulk of fatty acids synthesized in the cell is produced by the cytosolic FASI pathway [3,21,55], and the products of mtFAS have not yet unambiguously settled. A well-established product of mtFAS is octanoyl-ACP, the precursor for lipoic acid synthesis in mitochondria (Fig. 2). This was first shown by investigating $^{14}$C malonate labeling of mitochondria prepared from pea leaves [56], where the radioactive label was found to accumulate on ACP and the H protein of the glycine cleavage system (GCS). Lipoic acid levels have subsequently been found to be negatively affected or completely abolished by mtFAS disruption in all eukaryotic organisms studied thus far [3,6,15,39,42,48]. Lipoic acid has previously been regarded as a vitamin for humans, but accumulating evidence indicates that eukaryotes harbor an entire set of lipoic acid synthesis and attachment enzymes (Table 1) and are capable of *de novo* generation of lipoic acid. This compound is an enzyme cofactor essential for the function of pyruvate dehydrogenase (PDH), $\alpha$-ketoglutarate dehydrogenase (KGD), GCS and – not present in yeast – branched chain dehydrogenase (BCD) (for review see [57]).

![Diagram](attachment://diagram.png)

**Fig. 2.** From acetyl-group to lipoic acid. Using acetyl-ACP as a primer, the mtFAS carries out the chain elongation generating octanoyl-ACP from where the octanoyl group is transferred to lysine residue of target protein and on which the insertion of sulfur occurs. R1 = thioester bond to phosphopantetheine in ACP; R2 = amide bond to lysine residue of the target protein. The primary target is apparently the H-protein (Gsv3) of the GCS in yeast from where the octanoyl/lipoyl group is transferred to the E2 domains of PDH and KGD. *As mentioned in the legend to Fig. 2, lipoylation is likely to occur on the target proteins. Lipoylation on Gcv3/GCSH and transfers of the lipoyl moiety to E2 subunits by transamidation similar to the reaction catalyzed b *B. subtilis* lipL is likely, but has not been formally demonstrated.*

Please cite this article as: A.J. Kastaniotis, et al., Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology, Biochim. Biophys. Acta (2016), http://dx.doi.org/10.1016/j.bbalip.2016.08.011

ARTICLE IN PRESS

A.J. Kastaniotis et al. / Biochimica et Biophysica Acta xxx (2016) xxx–xxx 5

Lipoic acid is found covalently attached to conserved lysine residues of the lipoyl domains of E2 subunits of PDH, KGD and BCD, the accessory PDH protein Pdx1/PDHX and the H protein of GCS. Perhaps due to the comparatively late discovery of this pathway in eukaryotes, many aspects of the processes involved in lipoylation are still not well understood. A comprehensive overview of lipoylation processes, with a main focus on prokaryotic lipoylation and the corresponding structural enzymology, was recently published [58].

An interesting aspect that has still not been properly addressed is the usage of dietary lipoic acid in eukaryotic mitochondria. There have been reports showing data consistent with lipoic acid uptake in yeast [59] and mammalian cells [15], but it has not been convincingly shown that dietary lipoic acid ends up on mitochondrial ketoacid dehydrogenase complexes. According to Smith et al. [6], expression of the enzymes that have been proposed to be required for utilization of lipoic acid of non-mitochondrial origin is restricted to liver in mouse. This narrow expression pattern sheds doubt on the possibility of general use of dietary lipoic acid in mitochondrial enzyme complex modification. Furthermore, there is no evidence that administered radioactively labeled lipoic acid will be attached to proteins in mammals whereas it is quickly reduced and targeted to β-oxidation for chain-shortening [60–62]. Nutritional lipoic acid as an antioxidant is reviewed in several recent articles [63–67] and this aspect of lipoic acid metabolism will not be further discussed here.

There are four components that have been shown to be required for lipoylation in S. cerevisiae: lipoic acid synthase Lip5 [59] (LIAS in mammals [68,69], also called LASY [70]), Lip2 [71] and Lip3 [72] lipoyltransferases (LIPT2 and LIPT1 in mammals, respectively; [73–76]) and Gcv3 (GCSH, the H protein of GCS). All of these proteins have high similarity to *E. coli* lipoylation enzymes. The most significant structural difference from the bacterial proteins occurs in Lip3/LIPT1 [72,75], which lack the adenylation domain of the bacterial lplA lipoyl ligase [77]. However, a separate enzyme capable of activating lipoic acid using GTP was found in bovine liver [78] although it is not clear if this protein acts as lipoic acid activating enzyme in a physiological context. Deletion of *LIP5*, *LIP2* and *GCV3* all lead to complete loss of protein lipoylation in yeast [59,71,72], while inactivation of Lip3 leaves lipoylation of Gcv3 intact [72]. It was shown that GCS function is not required for the lipoylation process, as deletion of other genes encoding components of GCS have no effect on the lipoic acid content of yeast cells. The complete loss of lipoylation in the *gcv3Δ* strain could be either explained by Lip2-lipoylated Gcv3 serving as an intermediate substrate for transfer to other proteins [72], as a regulator of lipoylation of PDH or KGD [79], or as a component of the Lip3-dependent lipoylation apparatus. It has been recently argued that Lip3 may be indeed an amidotransferase [58] like the *Bacillus subtilis* lipL homolog, which has been shown to be capable of transferring of octanoic acid from the H protein of bacterial GCS to the E2 subunits of the dehydrogenases [80]. However, the physiological substrate of eukaryotic Lip3/LIPT1 transferases is not entirely clear. Bovine LIPT1 has been shown to be able to transfer lipoyl-AMP to the H protein of mammalian GCS, but is also capable of using octanoyl-AMP for this reaction [73]. New evidence suggested that, unlike its *E. coli* ortholog LplA, which preferentially uses free lipoic acid as a substrate for adenylation and transfer to target proteins, yeast Lip3 apparently much prefers CoA-activated octanoic acid as the substrate in an *in vivo* experiment in *E. coli* [79].

Initially, the function of mtFAS in generation of the octanoic acid precursor alone was thought to sufficiently explain the respiratory deficiency of mtFAS-defective cells. It was recently shown, however, that the phenotypes of yeast strains with disruption of the mtFAS and lipoic acid synthesis pathways are not entirely congruent [81]. Strains deficient in Etr1 or Htd2 exhibit markedly reduced levels of several mitochondrially encoded proteins, of which Cox1 exhibited the strongest decrease. This defect, as well as the mitochondrial mRNA splicing deficiency in yeast, is much more pronounced in the mtFAS mutants than in lipoic acid synthesis or attachment deficient strains [81].

Moreover, a suppressor allele specifically ameliorating the respiratory defect in yeast mtFAS null mutants [44,82] restored steady state levels of mitochondrially encoded proteins but did not improve the complete lack of lipoylation in these cells [81]. As an additional piece of evidence arguing against the lack of lipoic acid as the determining component of respiratory deficiency, it was demonstrated that overexpression of the translational activator/respiratory complex assembly factor Mss51 [83] improved expression of Cox1. Furthermore, expression of the mss51<sup>F199I</sup> heme-independent mutant variant completely re-established Cox1 levels and partially restored respiratory growth. The evidence presented by Kursu et al. points to a role of mtFAS in the control of post-transcriptional regulation of mitochondrial gene expression in yeast. As mtFAS and the tricarboxylic acid cycle share acetyl-CoA as a substrate, it was proposed that mtFAS may act as part of a regulatory circuit adapting mitochondrial biogenesis to intramitochondrial acetyl-CoA availability [81]. In line with the presented evidence, a physiological function of fatty acid products of mtFAS distinct from octanoic/lipoic acid was postulated (Fig. 1A).

All the mtFAS enzymes characterized thus far accept longer substrates than C6/C8 *in vitro* [12]. Remarkably, the substrate binding site in Etr1 is structured in a manner that is predicted to result in pausing synthesis at the octanoyl-ACP intermediate before proceeding to further extension of the acyl chain [70] allowing the generation of C14 to C16 products. It has been documented that the pathway is capable of synthesizing fatty acids of this length [84], but the fate of these products remains largely a mystery. In bovine heart mitochondria, the respiratory complex I-associated form of ACP (the SDAP subunits/NDUFAB1) has been shown to carry a myristoyl group [85], but here this lipid is attached to an N-terminal serine [86,87] instead of being attached to phosphopantetheine.

A supposed role for mtFAS products as structural components of mitochondrial membranes has been repeatedly dismissed [3,55,81] but cannot be entirely ruled out. However, no mitochondrial transacylase utilizing acyl-ACP as an acyl-group donor has been identified this far. Mitochondria contain several thioesterases that work on acyl-CoA esters with a preference towards acyl chain length, but there are no reports describing whether acyl-ACP also could serve as their substrate. Therefore, it has also remained unsettled whether fatty acids synthesized by mtFAS are released from ACP in their free form in mitochondria.

## 4. Lipoylation defects and human diseases

The first patient suffering from symptoms caused by defects in lipoylation was described in 2011 [88]. The patient presented atypical nonketotic hyperglycinemia, accompanied by neonatal onset of epilepsy, muscular dystonia and lactic acidosis. The cause of the latter symptom could be tracked down to severe hypolipoylation of PDH to levels undetectable by anti-lipoic acid Western blotting, and a concomitant reduction of the PDH subunit protein levels. Likewise, no lipoylated DLST/KGD E2 protein could be detected. A defect in the lipoic acid synthase, resulting from homozygous presence of a missense mutant allele of the LIAS gene inherited from consanguineous parents was determined as the cause of these symptoms. The nucleic acid replacement led to a change of a conserved arginine to a histidine moiety in LIAS expressed in the patient. The severely affected patient lived into early childhood but died from a respiratory tract infection at the age of four years. Non-functionality of the LIAS variant in the patient was demonstrated by the poor ability of the patient allele to complement a lipoic acid synthase deficient *E. coli* strain. Remarkably, the patient exhibited reduced respiratory chain function, reminiscent of the phenotype of yeast mutants with lipoylation defects [59,81]. To date, at least three additional LIAS defect cases have been identified [89–91].

Three families with members carrying compound heterozygous mutant alleles of *LIPT1* leading to defective LIPT1 lipoyl transferase have been reported [92–94]. The patients displayed the hallmark PDH

Please cite this article as: A.J. Kastaniotis, et al., Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology, Biochim. Biophys. Acta (2016), http://dx.doi.org/10.1016/j.bbalip.2016.08.011

ARTICLE IN PRESS

A.J. Kastaniotis et al. / Biochimica et Biophysica Acta xxx (2016) xxx–xxx

defect and associated KGD dysfunction mirroring the phenotype of the yeast model *lip3Δ* strain, PDH and KGD E2 subunits were severely hypolipoylated, while GCS function and H protein lipoylation was unaffected. Unlike in the *LIAS* patient case, respiratory chain activities appeared to be within normal parameters. Depending on the type of the mutations, the severity ranged from neonatal death to viable patients displaying encephalopathy and pulmonary hypertension to Leigh syndrome. Interestingly, lipoylation of branched chain dehydrogenase in the latter patient did not seem affected [92].

Several mutations in iron-sulfur (Fe—S) cluster biogenesis factors have been found to lead to lipoylation defects [90]. This is due to the fact that LIAS is an Fe—S cluster protein that requires this posttranslational modification to function. Mutations in *BOLA3*, [95] *IBA57* [89] and *GLRX5* [96] results in general Fe—S cluster maturation failure in addition to hyperlipoylation phenotypes. In contrast, a mutation in the *NFU1* gene very specifically results in a lipoylation defect, associated with a succinate dehydrogenase deficiency [97]. Patients with mutations in *NFU1* exhibit phenotypes similar to individuals with *LIAS* defects.

## 5. At the interface of fatty acid synthesis and β-oxidation in mitochondria

With the exception of some fungi like *Candida tropicalis* or *S. cerevisiae* whose mitochondria are devoid of a β-oxidation pathway [98], the cells of many fungal species appear to harbor a complete set of enzymes required for the execution of mitochondrial fatty acid oxidation [98,99], similar to the mitochondria of metazoans. As all these species apparently also have functional mitochondrial fatty acid synthesis systems, they harbor two fatty acid metabolizing pathways operating in opposite directions in the same cellular compartment. In both cases, the metabolized acyl-groups are attached via a thioester bond to a phosphopantetheine which acts as the prosthetic group of either ACP (synthetic pathway) or CoA (catabolic pathways). This arrangement keeps these pathways metabolically separated. Although the mtFAS enzymes appear to accept also acyl-CoA esters as substrates in vitro, acyl groups are linked to ACP in their natural substrates. In line with this view, mutations in the ACP-binding region of human KAR1 do not have any appreciable consequence for the activity with CoA-linked substrates in vitro, but they have severe effects on the ability of the enzyme to rescue respiratory growth of the *oar1Δ* mutant [43]. Cofactor specificities of redox reactions in mitochondrial FAS and β-oxidation represent the second factor separating these pathways. Namely, both MECR/Etr1 and KAR (CBR4)/Oar1 catalyze the reductive reactions of the synthesis in an NADPH-dependent manner, while acyl-CoA dehydrogenases and 3S-hydroxyacyl-CoA dehydrogenases utilize NAD⁺ when catalyzing oxidative reactions of mitochondrial β-oxidation. Both NADPH and NAD⁺ concentrations are in millimolar ranges under conditions prevailing within respiring mitochondria whereas NADP⁺ and NADH concentrations are about two orders of magnitude lower [100–102], driving FAS towards synthesis and β-oxidation towards acyl-chain shortening [12].

The properties of the KAR1 subunits reveal an unexpected link between fatty acid synthesis and breakdown pathways [43]. Polyunsaturated fatty acids (PUFAs), which have been frequently discussed as health-promoting macronutrients in mammals, experience continuous turnover independent of their function. Chemically, naturally occurring unsaturated fatty acids have the vast majority of their double bonds in the Z-configuration, and typically double bonds in PUFAs locate alternatingly in odd and even numbered positions separated by a methyl group. As the only unsaturated intermediate of the main β-oxidation spiral is 2E-enoyl-CoA (Fig. 1B) wherefrom fatty acid breakdown proceeds via 3S-hydroxyacyl-CoA to 3-ketoacyl-CoA, the completion of chain shortening of polyunsaturated enoyl-CoA esters requires additional tools. The cellular instruments providing the required functions are the auxiliary enzymes Δ3,Δ2-enoyl-CoA isomerase (ECI), Δ3,5-

Δ2,4-dienoyl-CoA isomerase (DECI) and 2,4-dienoyl-CoA reductase (DECR)(For review, see [103]). However, if Z-PUFAs escape the auxiliary enzyme pathways, 2Z-enoyl-CoA metabolites will arise. Enoyl-CoA hydratase 1 (crotonase, ECHS1), an enzyme of the main β-oxidation pathway, accepts both 2E- and 2Z-enoyl-CoA substrates and converts these metabolites to 3S- and 3R-hydroxyacyl-CoA esters, respectively [104]. In mammalian mitochondria, the KAR1 α-subunit HSD17B8 accepts 3R-hydroxyacyl-CoA substrates and oxidizes them to 3-ketacyl-CoA esters in an NAD-dependent manner [43]. Thus if ECHS1 and HSD17B8 work in tandem, they allow for an alternative route to the auxiliary enzyme pathways for the breakdown of Z-PUFA enoyl-esters (Fig. 3).

## 6. Physiology and regulation of mtFAS

Eukaryotic cells deficient in mtFAS display a pleiotropic phenotype. *S. cerevisiae* strains with mtFAS defects are respiratory deficient. Mitochondrial morphology in these cells was reported to be abnormal [4, 44], while overexpression of Etr1 or its *C. tropicalis* ortholog in *S. cerevisiae* results in a dramatic enlargement of the mitochondrial compartment [4,44]. This effect is phenocopied in a mouse *Mecr* overexpression model [105], where the increased *Mecr* level also leads to cardiomyopathy. Cytochrome spectra analyses of yeast strains with loss-of-function mutations in mtFAS components show absence of cytochromes containing mitochondrially encoded protein subunits [9, 81]. In addition, a mild heme deficiency and a defect in mitochondrial mRNA splicing and processing of the mitochondrially encoded RPM1 RNA subunit of mitochondrial RNase P has been reported in yeast [81, 106]. The knockout of the *ACP1* gene in yeast results in rapid loss of mitochondrial DNA [3]. In *Neurospora crassa* and mammalian cultured cells, depletion of ACP leads to respiratory I assembly defects, respiratory deficiency and loss of lipoylation [39,107]. The inactivation of the *Mecr* gene in mice causes embryonic lethality (unpublished), and mice subjected to the induced knockout of the *Mcat* gene show severe signs of premature aging and energy disequilibrium [6]. Treatment of HEK293T cells with the KAS inhibitor C75 leads to reduced mitochondrial membrane potential, increased production of reactive oxygen species and reduced cell survival [15].

It has recently been shown independently by two groups that MECR plays a role in gene regulation in mammals. On one hand, Parl et al. [108] demonstrated that overexpression of MECR results in activation of gene expression via the PPARα and PPARγ pathway and leads in increased binding of PPARs to target DNAs. Remarkably, MECR appears to regulate also its own expression via the PPAR pathway. Parl et al. favor a model implying an mtFAS product acting as a signal, but do not rule out a physical role of MECR as co-activator. Kim et al. [109] on the other hand described a non-mitochondrial isoform of MECR in rat which may be conserved in other mammals (our observation). This protein was demonstrated to physically associate with PPARα using several different assays. Expression of the non-mitochondrial isoform of MECR was also shown to result in increased expression of PPARα target genes. MECR was originally identified as a nuclear receptor binding factor (Nrbf-1) in rat [110], as was yeast Etr1 [111], but in light of the clear role of these enzymes in mtFAS, a possible extramitochondrial role of these proteins have remained unexplored. More research of this aspect of MECR/Etr1 is necessary, as it was reported that overexpression of MECR has interesting consequences on the abundance of bioactive lipids [55]. The siRNA mediated knockdown of ACP reported in the same publication resulted in many reciprocal changes, indicating that part of this response may indeed be mediated by an mtFAS product rather than the MECR protein. However, results from this experimental setup, using an comparative approach of metabolomics analysis of the effects of overexpression of one component of mtFAS and a knockdown of a different protein involved in mtFAS should be interpreted with caution, as both MECR and ACP appear to have different additional functions not directly associated with mtFAS.

---

Please cite this article as: A.J. Kastaniotis, et al., Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology, Biochim. Biophys. Acta (2016), http://dx.doi.org/10.1016/j.bbalip.2016.08.011

ARTICLE IN PRESS

A.J. Kastaniotis et al. / Biochimica et Biophysica Acta xxx (2016) xxx–xxx

**PUFA**


    5Z
     ↓
   XAD
   2E, 5Z
     ↓
   ECI
   3E, 5Z
     ↓
   DECI
   2E, 4E
     ↓
   DECR
   3E
     ↓
   ECI
   2E-enoyl-CoA
     ↓
   HADH
   3-ketoacyl-CoA
     ↓
   Acetyl-CoA
   Acyl-CoA



    4Z
     ↓
   XAD
   2E, 4Z
     ↓
   DECR
   3E
     ↓
   ECI
   2E-enoyl-CoA
     ↓
   HADH
   3-ketoacyl-CoA
     ↓
   Acetyl-CoA
   Acyl-CoA



    3Z
     ↓
   ECI
   3R-OH
     ↓
   HSD17B8
   3-ketoacyl-CoA
     ↓
   Acetyl-CoA
   Acyl-CoA



    2Z
     ↓
   ECHS1
   3R-OH
     ↓
   HSD17B8
   3-ketoacyl-CoA
     ↓
   Acetyl-CoA
   Acyl-CoA


**B**

1) 2E-enoyl-CoA → 3S-hydroxyacyl-CoA  
   H₂O  
   ECHS1  

2Z-enoyl-CoA → 3R-hydroxyacyl-CoA  
   H₂O  
   ECHS1  

2) 3R-hydroxyacyl-CoA → 3-ketoacyl-CoA  
   NAD⁺ → NADH + H⁺  
   HSD17B8  

Fig. 3. Schematic presentation of possible pathways metabolizing enoyl-CoA esters with Z-double bonds in Z-configuration into metabolites accepted as substrates by enzymes of the main mitochondrial β-oxidation spiral. (A) Mitochondrial β-oxidation of PUFAs with double bonds in Z-configuration generates enoyl-CoA metabolites with Z-double bonds in positions 5, 4, 3 or 2. These metabolites can be transformed into 2E-enoyl-CoAs in pathways catalyzed by the auxiliary enzymes of β-oxidation as visualized in the Figure. For review of the auxiliary enzymes see the reference [103]. The abbreviations: XAD, acyl-CoA dehydrogenase, chain length preference not specified; ECI, Δ3,Δ2-enoyl-CoA isomerase; DECI, Δ3,Δ5-Δ2,Δ4-dienoyl-CoA isomerase; DECR, Δ2,Δ4-dienoyl-CoA reductase; HADH, 3S-hydroxyacyl-CoA dehydrogenase; ECHS1, 2-enoyl-CoA hydratase 1 (crotonase); HSD17B8, 17β-hydroxysteroid dehydrogenase type 8. ECl is referring to both Δ3,Δ2-enoyl-CoA isomerase 1 and 2. 3R-OH is referring to 3R-hydroxyacyl-CoA esters, and the numbers in front of E and Z give the positions of double bonds in carbon chain of enoyl-CoA esters. (B) Reactions catalyzed by ECHS1 and HSD17B8.

Currently we do not know much about the regulation of the mtFAS enzymes apart from the autoregulation of MECR in mammals. One possible mechanism is posttranscriptional modifications of mtFAS proteins. A recent study by Doulias et al. showed that nitric oxide regulates mitochondrial fatty acid metabolism through reversible protein S-nitrosylation [112]. Their work focused on mitochondrial very long chain acyl-CoA dehydrogenase whose catalytic activity was diminished upon S-nitrosylation. It is note worthy that, almost all enzymes of mtFAS can be found on the list of proteins targeted to nitrosylation, but the role of this modification on the flux through mtFAS pathway waits for further studies. Because protein acetylation is the most frequent posttranslational protein modification and about 63% of mitochondrial proteins contain lysine N-ε-acetylation sites, we shall also consider this protein modification as a potential mechanism to modify the rate of fatty acid synthesis in mitochondria.

### 7. Concluding and future aspects

Indirect evidence for a role of mtFAS-generated fatty acids other than octanoic acid is mounting. However, many aspects of the mechanisms that would explain how these mtFAS products act in mitochondrial biogenesis have remained in the dark. To unequivocally demonstrate that fatty acids of mtFAS origin are really covalently attached to mitochondrial proteins involved in mitochondrial biogenesis, evidence of such post-translational modifications needs to be provided. As the fatty acid output of mtFAS appears to be rather low, this will remain a difficult task even if the hypothesis is correct. Equally important would be the demonstration of fatty acyl transferases in mitochondria that can carry out such modifications. The yeast models have been of tremendous help for our understanding of the pathway, and it is remarkable how well the mechanisms in yeast appear to be conserved in mammals. Although yeast mitochondria differ in various aspects from their metazoan counterparts, it can be expected that many more features of lipoylation and mtFAS found in yeast are also going to have close counterparts in humans.

Most of the mitochondrial proteins, among them mtFAS enzymes, can be subjected to posttranslation modifications. Detailed characterization and comparison of non-nitrosylated - nitrosylated or acylated - deacylated enzymes have shown that the modification will change functional properties of the proteins in terms of kinetics or stability. These observations give intriguing hints at the physiological significance of the modification cycles, and thus a new aspect of the regulation of mitochondrial fatty acid metabolism has been generated.

Currently, genetically modified mouse models for mtFAS pathway defects are still scarce, very likely because the complete knockout appears to result in embryonic lethality. Because of the great variance in fatty acid metabolizing activities between tissues, tissue specific conditional knock out/knock down mouse models will provide valuable information for years to come. Novel techniques for specific genome editing present convenient means to test human disease-causing mutations in experimental animals. Currently, the analysis of gene regulation in eukaryotes has moved to a phase of elucidating the roles of epigenetics, DNA modifications and DNA binding protein complexes to control transcription. It can also be assumed that research on noncoding RNAs will bring new and unexpected novel information on mitochondrial fatty acid metabolism as a part of cellular homeostasis.

There are still many questions concerning mtFAS and lipoylation that need to be addressed. The proposed use of malonate of cytosolic origin in mammalian mtFAS [36] does not seem entirely consistent with the metabolic sensing model proposed for yeast mtFAS function by Kursu et al. [81]. Nevertheless, the ACSF3-mediated generation of malonyl-CoA may yet be part of another feedback mechanism, integrating events surrounding β-oxidation with control of respiration. The question on the subcellular localization of biotin protein ligase (Bpl1) [28] and mammalian PPTase still needs resolution. While several disease cases involving defects in the human lipoylation pathway have been reported to date [90], thus far no patients with mutations in mtFAS components have been identified. In light of the multitude of proteins involved in mtFAS, such types of disease must exist. The rising awareness of the existence of mtFAS in the research and medical communities, in concert with next generation sequencing technologies, will bring these cases to light in the near future.

The idea of using FAS-inhibitors in the treatment of cancer or as novel antibiotics should also be subjected to close scrutiny. InhA, the

Please cite this article as: A.J. Kastaniotis, et al., Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology, Biochim. Biophys. Acta (2016), http://dx.doi.org/10.1016/j.bbalip.2016.08.011

target of the drug isoniazid used in the treatment of infections with *Mycobacterium tuberculosis*, is an ETR of the SDH type superfamily of reductases like *E. coli* fabI and shares sensitivity to triclosan with this protein [4,81,113]. The human/yeast MDR-type MECR/Etr1 enzymes are much less sensitive to this drug, and hence similar inhibitors specific for SDH-type ETRs may be comparatively safe. Other types of FAS inhibitors targeting more conserved proteins/domains should be approached with much more caution. They may be usable as antibiotics, if applied for short term therapies in a similar manner as for example inhibitors of prokaryotic protein synthesis. For longer term regimes of treatment as e.g. common in some kinds of cancer therapy, the effect of FAS inhibitors on mitochondrial biogenesis in humans needs to be studied to avoid subsequent problems due to diminished mitochondrial function.

Lastly, the intriguing specificity of the NUF1 factor involved in the maturation of LIAS and succinate dehydrogenase [97] poses questions on the reason for this narrow phenotype. Lipoylation defects lead to succinate dehydrogenase deficiency in yeast [81]. Hence, if the yeast model again holds true also in mammals it is possible that the NFU1 patient mutation defect is primarily through lack of lipoylation. As lipoic acid synthase is a unique enzyme in which a Fe—S cluster is destroyed during the lipoylation process and apparently needs to be rejuvenated after each round of sulfur insertion to octanoic acid under the use of S-adenosylmethionine [114], it is tempting to speculate that NFU1 may be such a Fe—S cluster restoration factor.

### Transparency document

The **Transparency document** associated with this article can be found, in the online version.

### Acknowledgements

The original research in our laboratories has been supported by grants from the Academy of Finland, the Sigrid Juselius Foundation, AFM-telethon (No. 18885), and NSFC (No. 21372097).

### References

[1] P. Bhaumik, M.K. Koski, T. Glumoff, J.K. Hiltunen, R.K. Wierenga, Structural biology of the thioester-dependent degradation and synthesis of fatty acids, Curr. Opin. Struct. Biol. 15 (2005) 621–628.

[2] S. Mikolajczyk, S. Brody, De novo fatty acid synthesis mediated by acyl-carrier protein in *Neurospora crassa* mitochondria, Eur. J. Biochem. 187 (1990) 431–437.

[3] S. Brody, C. Oh, U. Hoja, E. Schweizer, Mitochondrial acyl carrier protein is involved in lipoic acid synthesis in *Saccharomyces cerevisiae*, FEBS Lett. 408 (1997) 217–220.

[4] J.M. Torkko, K.T. Koivuranta, I.J. Miinalainen, A.I. Yagi, W. Schmitz, A.J. Kastaniotis, T.T. Airenne, A. Gurvitz, K.J. Hiltunen, *Candida tropicalis* Etr1p and *Saccharomyces cerevisiae* Ybr026p (*Mrf1′p*), 2-enoyl thioester reductases essential for mitochondrial respiratory competence, Mol. Cell. Biol. 21 (2001) 6243–6253.

[5] J.K. Hiltunen, M.S. Schonauer, K.J. Autio, T.M. Mittelmeier, A.J. Kastaniotis, C.L. Dieckmann, Mitochondrial fatty acid synthesis type II: more than just fatty acids, J. Biol. Chem. 284 (2009) 9011–9015.

[6] S. Smith, A. Witkowski, A. Moghul, Y. Yoshinaga, M. Nefedov, P. de Jong, D. Feng, L. Fong, Y. Tu, Y. Hu, S.G. Young, T. Pham, C. Cheung, S.M. Katzman, M.D. Brand, C.L. Quinlan, M. Fens, F. Kuypers, S. Misquitta, S.M. Griffey, S. Tran, A. Gharib, J. Knudsen, H.K. Hannibal-Bach, G. Wang, S. Larkin, J. Thweatt, S. Pasta, Compromised mitochondrial fatty acid synthesis in transgenic mice results in defective protein lipoylation and energy disequilibrium, PLoS One 7 (2012), e47196.

[7] S. Brody, S. Mikolajczyk, *Neurospora* mitochondria contain an acyl-carrier protein, Eur. J. Biochem. 173 (1988) 353–359.

[8] S.W. White, J. Zheng, Y.M. Zhang, Rock, The structural biology of type II fatty acid biosynthesis, Annu. Rev. Biochem. 74 (2005) 791–831.

[9] A. Harington, C.J. Herbert, B. Tung, G.S. Getz, P.P. Slonimski, Identification of a new nuclear gene (*CEM1*) encoding a protein homologous to a beta-keto-acetyl synthase which is essential for mitochondrial respiration in *Saccharomyces cerevisiae*, Mol. Microbiol. 9 (1993) 545–555.

[10] R. Schneider, B. Brors, F. Burger, S. Camrath, H. Weiss, Two genes of the putative mitochondrial fatty acid synthase in the genome of *Saccharomyces cerevisiae*, Curr. Genet. 32 (1997) 384–388.

[11] C.E. Christensen, B.B. Kragelund, P. von Wettstein-Knowles, A. Henriksen, Structure of the human beta-ketoacyl [ACP] synthase from the mitochondrial type II fatty acid synthase, Protein Sci. 16 (2007) 261–272.

[12] J.K. Hiltunen, Z. Chen, A.M. Haapalainen, R.K. Wierenga, A.J. Kastaniotis, Mitochondrial fatty acid synthesis—an adopted set of enzymes making a pathway of major importance for the cellular metabolism, Prog. Lipid Res. 49 (2010) 27–45.

[13] H. Funabashi, A. Kawaguchi, H. Tomoda, S. Omura, S. Okuda, S. Iwasaki, Binding site of cerulenin in fatty acid synthetase, J. Biochem. 105 (1989) 751–755.

[14] L. Zhang, A.K. Joshi, J. Hofmann, E. Schweizer, S. Smith, Cloning, expression, and characterization of the human mitochondrial beta-ketoacyl synthase. Complementation of the yeast *CEM1* knock-out strain, J. Biol. Chem. 280 (2005) 12422–12429.

[15] C. Chen, X. Han, X. Zou, Y. Li, L. Yang, K. Cao, J. Xu, J. Long, J. Liu, Z. Feng, 4-Methylen-2-octyl-5-oxotetrahydrofuran-3-carboxylic acid (C75), an inhibitor of fatty acid synthase, suppresses the mitochondrial fatty acid synthesis pathway and impairs mitochondrial function, J. Biol. Chem. 289 (2014) 17184–17194.

[16] J. Wang, S.M. Soisson, K. Young, W. Shoop, S. Kodali, A. Galgoci, R. Painter, G. Parthasarathy, Y.S. Tang, R. Cummings, S. Ha, K. Dorso, M. Motyl, H. Jayasuriya, J. Ondeyka, K. Herath, C. Zhang, L. Hernandez, J. Allocco, A. Basilio, J.R. Tormo, O. Genilloud, F. Vicente, F. Pelaez, L. Colwell, S.H. Lee, B. Michael, T. Felcetto, C. Gill, L.L. Silver, J.D. Hermes, K. Bartizal, J. Barrett, D. Schmatz, J.W. Becker, D. Cully, S.B. Singh, Platensimycin is a selective FabF inhibitor with potent antibiotic properties, Nature 441 (2006) 358–361.

[17] S. Jackowski, C.M. Murphy, J.E. Cronan Jr., C.O. Rock, Acetoacetyl-acyl carrier protein synthase. A target for the antibiotic thiolutomycin, J. Biol. Chem. 264 (1989) 7624–7629.

[18] M.J. Runswick, I.M. Fearnley, J.M. Skehel, J.E. Walker, Presence of an acyl carrier protein in NADH:ubiquinone oxidoreductase from bovine heart mitochondria, FEBS Lett. 286 (1991) 121–124.

[19] U. Sackmann, R. Zensen, D. Rohlen, U. Jahnke, H. Weiss, The acyl-carrier protein in *Neurospora crassa* mitochondria is a subunit of NADH:ubiquinone reductase (complex I), Eur. J. Biochem. 200 (1991) 463–469.

[20] R. Zensen, H. Husmann, R. Schneider, T. Peine, H. Weiss, De novo synthesis and desaturation of fatty acids at the mitochondrial acyl-carrier protein, a subunit of NADH:ubiquinone oxidoreductase in *Neurospora crassa*, FEBS Lett. 310 (1992) 179–181.

[21] K. Dobrynin, A. Abdrakhmanova, S. Richers, C. Hunte, S. Kerscher, U. Brandt, Characterization of two different acyl carrier proteins in complex I from *Yarrowia lipolytica*, Biochim. Biophys. Acta 1797 (2010) 152–159.

[22] E.H. Meyer, J.L. Heazlewood, A.H. Millar, Mitochondrial acyl carrier proteins in *Arabidopsis thaliana* are predominantly soluble matrix proteins and none can be confirmed as subunits of respiratory complex I, Plant Mol. Biol. 64 (2007) 319–327.

[23] H.P. Stuible, S. Meier, C. Wagner, E. Hannappel, E. Schweizer, A novel phosphopantetheine:protein transferase activating yeast mitochondrial acyl carrier protein, J. Biol. Chem. 273 (1998) 22334–22339.

[24] A.K. Joshi, L. Zhang, V.S. Rangan, S. Smith, Cloning, expression, and characterization of a human 4′-phosphopantetheinyl transferase with broad substrate specificity, J. Biol. Chem. 278 (2003) 33142–33149.

[25] U. Hoja, S. Marthol, J. Hofmann, S. Stegner, R. Schulz, S. Meier, E. Greiner, E. Schweizer, *HFA1* encoding an organelle-specific acetyl-CoA carboxylase controls mitochondrial fatty acid synthesis in *Saccharomyces cerevisiae*, J. Biol. Chem. 279 (2004) 21779–21786.

[26] K.H. Wolfe, D.C. Shields, Molecular evidence for an ancient duplication of the entire yeast genome, Nature 387 (1997) 708–713.

[27] M. Kellis, B.W. Birren, E.S. Lander, Proof and evolutionary analysis of ancient genome duplication in the yeast *Saccharomyces cerevisiae*, Nature 428 (2004) 617–624.

[28] F. Suomi, K.E. Menger, G. Monteuuis, U. Naumann, V.A. Kursu, A. Shvetsova, A.J. Kastaniotis, Expression and evolution of the non-canonically translated yeast mitochondrial acetyl-CoA carboxylase *Hfa1p*, PLoS One 9 (2014), e114738.

[29] R.S. Zitomer, D.A. Walthall, B.C. Rymond, C.P. Hollenberg, *Saccharomyces cerevisiae* ribosomes recognize non-AUG initiation codons, Mol. Cell. Biol. 4 (1984) 1191–1197.

[30] G. Natsoulis, F. Hilger, G.R. Fink, The *HTS1* gene encodes both the cytoplasmic and mitochondrial histidine tRNA synthetases of *S. cerevisiae*, Cell 46 (1986) 235–243.

[31] B. Chatton, P. Walter, J.P. Ebel, F. Lacroute, F. Fasiolo, The yeast *VAS1* gene encodes both mitochondrial and cytoplasmic valyl-tRNA synthetases, J. Biol. Chem. 263 (1988) 52–57.

[32] H.L. Tang, L.S. Yeh, N.K. Chen, T. Ripmaster, P. Schimmel, C.C. Wang, Translation of a yeast mitochondrial tRNA synthetase initiated at redundant non-AUG codons, J. Biol. Chem. 279 (2004) 49656–49663.

[33] K.J. Chang, C.C. Wang, Translation initiation from a naturally occurring non-AUG codon in *Saccharomyces cerevisiae*, J. Biol. Chem. 279 (2004) 13778–13785.

[34] O. Turunen, R. Seelke, J. Macosko, In silico evidence for functional specialization after genome duplication in yeast, FEMS Yeast Res. 9 (2009) 16–31.

[35] H.R. Scholte, I.E. Luyt-Houwen, M.L. Dubelaar, W.C. Hulsmann, The source of malonyl-CoA in rat heart. The calcium paradox releases acetyl-CoA carboxylase and not propionyl-CoA carboxylase, FEBS Lett. 198 (1986) 47–50.

[36] A. Witkowski, J. Thweatt, S. Smith, Mammalian ACSF3 protein is a malonyl-CoA synthetase that supplies the chain extender units for mitochondrial fatty acid synthesis, J. Biol. Chem. 286 (2011) 33729–33736.

[37] L.L. Bieber, Carnitine, Annu. Rev. Biochem. 57 (1988) 261–283.

[38] X. Guan, B.J. Nikolau, *AAE13* encodes a dual-localized malonyl-CoA synthetase that is crucial for mitochondrial fatty acid biosynthesis, Plant J. 85 (2016) 581–593.

[39] D. Feng, A. Witkowski, S. Smith, Down-regulation of mitochondrial acyl carrier protein in mammalian cells compromises protein lipoylation and respiratory complex I and results in cell death, J. Biol. Chem. 284 (2009) 11436–11445.

[40] J.L. Heazlewood, K.A. Howell, J. Whelan, A.H. Millar, Towards an analysis of the rice mitochondrial proteome, Plant Physiol. 132 (2003) 230–242.


Please cite this article as: A.J. Kastaniotis, et al., Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology, Biochim. Biophys. Acta (2016), http://dx.doi.org/10.1016/j.bbalip.2016.08.011

[41] M. Focke, E. Gieringer, S. Schwan, L. Jansch, S. Binder, H.P. Braun, Fatty acid biosynthesis in mitochondria of grasses: malonyl-coenzyme A is generated by a mitochondrial-localized acetyl-coenzyme A carboxylase, Plant Physiol. 133 (2003) 875–884.

[42] Z. Chen, A.J. Kastaniotis, I.J. Miinalainen, V. Rajaram, R.K. Wierenga, J.K. Hiltunen, 17beta-hydroxysteroid dehydrogenase type 8 and carbonyl reductase type 4 assemble as a ketoacyl reductase of human mitochondrial FAS, FASEB J. 23 (2009) 3682–3691.

[43] R. Venkatesan, S.K. Sah-Teli, L.O. Awoniyi, G. Jiang, P. Prus, A.J. Kastaniotis, J.K. Hiltunen, R.K. Wierenga, Z. Chen, Insights into mitochondrial fatty acid synthesis from the structure of heterotetrameric 3-ketoacyl-ACP reductase/3R-hydroxyacyl-CoA dehydrogenase, Nat. Commun. 5 (2014) 4805.

[44] A.J. Kastaniotis, K.J. Autio, R.T. Sormunen, J.K. Hiltunen, Htd2p/Yhr067p is a yeast 3-hydroxyacyl-ACP dehydratase essential for mitochondrial function and morphology, Mol. Microbiol. 53 (2004) 1407–1421.

[45] K.J. Autio, A.J. Kastaniotis, H. Pospiech, I.J. Miinalainen, M.S. Schonauer, C.L. Dieckmann, J.K. Hiltunen, An ancient genetic link between vertebrate mitochondrial fatty acid synthesis and RNA processing, FASEB J. 22 (2008) 569–578.

[46] D.E. Andreev, P.B. O'Connor, A.V. Zhdanov, R.I. Dmitriev, I.N. Shatsky, D.B. Papkovsky, P.V. Baranov, Oxygen and glucose deprivation induces widespread alterations in mRNA translation within 20 minutes, Genome Biol. 16 (2015) (90–015-0651-z).

[47] K.J. Autio, J.L. Guler, A.J. Kastaniotis, P.T. Englund, J.K. Hiltunen, The 3-hydroxyacyl-ACP dehydratase of mitochondrial fatty acid synthesis in Trypanosoma brucei, FEBS Lett. 582 (2008) 729–733.

[48] J.L. Stephens, S.H. Lee, K.S. Paul, P.T. Englund, Mitochondrial fatty acid synthesis in Trypanosoma brucei, J. Biol. Chem. 282 (2007) 4427–4436.

[49] H. Mouilleron, V. Delcourt, X. Roucou, Death of a dogma: eukaryotic mRNAs can code for more than one protein, Nucleic Acids Res. 44 (2016) 14–23.

[50] T.T. Airenne, J.M. Torkko, S. Van den plas, R.T. Sormunen, A.J. Kastaniotis, R.K. Wierenga, J.K. Hiltunen, Structure-function analysis of enoyl thioester reductase involved in mitochondrial maintenance, J. Mol. Biol. 327 (2003) 47–59.

[51] I.J. Miinalainen, Z.J. Chen, J.M. Torkko, P.L. Pirila, R.T. Sormunen, U. Bergmann, Y.M. Qin, J.K. Hiltunen, Characterization of 2-enoyl thioester reductase from mammals. An ortholog of YBR026p/MRF1'p of the yeast mitochondrial fatty acid synthesis type II, J. Biol. Chem. 278 (2003) 20154–20161.

[52] J.M. Torkko, K.T. Koivuranta, A.J. Kastaniotis, T.T. Airenne, T. Glumoff, M. Ilves, A. Hartig, A. Gurvitz, J.K. Hiltunen, Candida tropicalis expresses two mitochondrial 2-enoyl thioester reductases that are able to form both homodimers and heterodimers, J. Biol. Chem. 278 (2003) 41213–41220.

[53] H. Jornvall, J. Hedlund, T. Bergman, U. Oppermann, B. Persson, Superfamilies SDR and MDR: from early ancestry to present forms. Emergence of three lines, a Zn-metalloenzyme, and distinct variabilities, Biochem. Biophys. Res. Commun. 396 (2010) 125–130.

[54] B. Persson, J. Hedlund, H. Jornvall, Medium- and short-chain dehydrogenase/reductase gene and protein families : the MDR superfamily, Cell. Mol. Life Sci. 65 (2008) 3879–3894.

[55] H.B. Clay, A.K. Parl, S.L. Mitchell, L. Singh, L.N. Bell, D.G. Murdock, Altering the mitochondrial fatty acid synthesis (mtFASII) pathway modulates cellular metabolic states and bioactive lipid profiles as revealed by metabolomic profiling, PLoS One 11 (2016), e0151171.

[56] H. Wada, D. Shintani, J. Ohlrogge, Why do mitochondria synthesize fatty acids? Evidence for involvement in lipoic acid production, Proc. Natl. Acad. Sci. U. S. A. 94 (1997) 1591–1596.

[57] R.N. Perham, Swinging arms and swinging domains in multifunctional enzymes: catalytic machines for multistep reactions, Annu. Rev. Biochem. 69 (2000) 961–1004.

[58] J.E. Cronan, Assembly of lipoic acid on its cognate enzymes: an extraordinary and essential biosynthetic pathway, Microbiol. Mol. Biol. Rev. 80 (2016) 429–450.

[59] P. Sulo, N.C. Martin, Isolation and characterization of LIP5. A lipoate biosynthetic locus of Saccharomyces cerevisiae, J. Biol. Chem. 268 (1993) 17634–17639.

[60] G.P. Biewenga, G.R. Haenen, A. Bast, The pharmacology of the antioxidant lipoic acid, Gen. Pharmacol. 29 (1997) 315–331.

[61] J. Bustamante, J.K. Lodge, L. Marcocci, H.J. Tritschler, L. Packer, B.H. Rihn, Alpha-lipoic acid in liver metabolism and disease, Free Radic. Biol. Med. 24 (1998) 1023–1039.

[62] K.P. Shay, R.F. Moreau, E.J. Smith, A.R. Smith, T.M. Hagen, Alpha-lipoic acid as a dietary supplement: molecular mechanisms and therapeutic potential, Biochim. Biophys. Acta 1790 (2009) 1149–1160.

[63] A. Goraca, H. Huk-Kolega, A. Piechota, P. Klejniewska, E. Ciejka, B. Skibska, Lipoic acid - biological activity and therapeutic potential, Pharmacol. Rep. 63 (2011) 849–858.

[64] S. Park, U. Karunakaran, N.H. Jeoung, J.H. Jeon, I.K. Lee, Physiological effect and therapeutic application of alpha lipoic acid, Curr. Med. Chem. 21 (2014) 3636–3645.

[65] M.B. Gomes, C.A. Negrato, Alpha-lipoic acid as a pleiotropic compound with potential therapeutic use in diabetes and other chronic diseases, Diabetol. Metab. Syndr. 6 (2014) (80–5996–6-80. eCollection 2014).

[66] D.Q. Jiang, M.X. Li, Y.J. Ma, Y. Wang, Y. Wang, Efficacy and safety of prostaglandin E1 plus lipoic acid combination therapy versus monotherapy for patients with diabetic peripheral neuropathy, J. Clin. Neurosci. 27 (2016) 8–16.

[67] A.E. Huerta, P.L. Prieto-Hontoria, N. Sainz, J.A. Martinez, M.J. Moreno-Aliaga, Supplementation with alpha-lipoic acid alone or in combination with eicosapentaenoic acid modulates the inflammatory status of healthy overweight or obese women consuming an energy-restricted diet, J. Nutr. (2016).

[68] T. Morikawa, R. Yasuno, H. Wada, Do mammalian cells synthesize lipoic acid? Identification of a mouse cDNA encoding a lipoic acid synthase located in mitochondria, FEBS Lett. 498 (2001) 16–21.

[69] X. Yi, N. Maeda, Endogenous production of lipoic acid is essential for mouse development, Mol. Cell. Biol. 25 (2005) 8387–8392.

[70] I. Padmalayam, S. Hasham, U. Saxena, S. Pillarisetti, Lipoic acid synthase (LASY): a novel role in inflammation, mitochondrial function, and insulin resistance, Diabetes 58 (2009) 600–608.

[71] M.E. Marvin, P.H. Williams, A.M. Cashmore, The isolation and characterisation of a Saccharomyces cerevisiae gene (LIP2) involved in the attachment of lipoic acid groups to mitochondrial enzymes, FEMS Microbiol. Lett. 199 (2001) 131–136.

[72] M.S. Schonauer, A.J. Kastaniotis, V.A. Kursu, J.K. Hiltunen, C.L. Dieckmann, Lipoic acid synthesis and attachment in yeast mitochondria, J. Biol. Chem. 284 (2009) 23234–23242.

[73] K. Fujiwara, K. Okamura-Ikeda, Y. Motokawa, Cloning and expression of a cDNA encoding bovine lipoyltransferase, J. Biol. Chem. 272 (1997) 31974–31978.

[74] K. Fujiwara, M. Suzuki, Y. Okumachi, K. Okamura-Ikeda, T. Fujiwara, E. Takahashi, Y. Motokawa, Molecular cloning, structural characterization and chromosomal localization of human lipoyltransferase gene, Eur. J. Biochem. 260 (1999) 761–767.

[75] K. Fujiwara, H. Hosaka, M. Matsuda, K. Okamura-Ikeda, Y. Motokawa, M. Suzuki, A. Nakagawa, H. Taniguchi, Crystal structure of bovine lipoyltransferase in complex with lipoyl-AMP, J. Mol. Biol. 371 (2007) 222–234.

[76] S.L. Hendrickson, J.A. Lautenberger, L.W. Chinn, M. Malasky, E. Sezgin, L.A. Kingsley, J.J. Goedert, G.D. Kirk, E.D. Gomperts, S.P. Buchbinder, J.L. Troyer, S.J. O'Brien, Genetic variants in nuclear-encoded mitochondrial genes influence AIDS progression, PLoS One 5 (2010), e12862.

[77] D.E. Green, T.W. Morris, J.E. Green, J.E. Cronan Jr., J.R. Guest, Purification and properties of the lipoate protein ligase of Escherichia coli, Biochem. J. 309 (Pt 3) (1995) 853–862.

[78] K. Fujiwara, S. Takeuchi, K. Okamura-Ikeda, Y. Motokawa, Purification, characterization, and cDNA cloning of lipoate-activating enzyme from bovine liver, J. Biol. Chem. 276 (2001) 28819–28823.

[79] F.A. Hermes, J.E. Cronan, The role of the Saccharomyces cerevisiae lipoate protein ligase homologue, Lip3, in lipoic acid synthesis, Yeast 30 (2013) 415–427.

[80] Q.H. Christensen, N. Martin, M.C. Mansilla, D. de Mendoza, J.E. Cronan, A novel amidotransferase required for lipoic acid cofactor assembly in Bacillus subtilis, Mol. Microbiol. 80 (2011) 350–363.

[81] V.A. Kursu, L.P. Pietikainen, F. Fontanesi, M.J. Altonen, F. Suomi, R. Raghavan Nair, M.S. Schonauer, C.L. Dieckmann, A. Barrientos, J.K. Hiltunen, A.J. Kastaniotis, Defects in mitochondrial fatty acid synthesis result in failure of multiple aspects of mitochondrial biogenesis in Saccharomyces cerevisiae, Mol. Microbiol. 90 (2013) 824–840.

[82] A. Harington, E. Schwarz, P.P. Slonimski, C.J. Herbert, Subcellular relocation of a long-chain fatty acid CoA ligase by a suppressor mutation alleviates a respiration deficiency in Saccharomyces cerevisiae, EMBO J. 13 (1994) 5531–5538.

[83] F. Fontanesi, I.C. Soto, D. Horn, A. Barrientos, Mss51 and Ssc1 facilitate translational regulation of cytochrome c oxidase biogenesis, Mol. Cell. Biol. 30 (2010) 245–259.

[84] S. Herzig, E. Raemy, S. Montessuit, J.L. Veuthey, N. Zamboni, B. Westermann, E.R. Kunji, J.C. Martinou, Identification and functional expression of the mitochondrial pyruvate carrier, Science 337 (2012) 93–96.

[85] J. Carroll, I.M. Fearnley, R.J. Shannon, J. Hirst, J.E. Walker, Analysis of the subunit composition of complex I from bovine heart mitochondria, Mol. Cell. Proteomics 2 (2003) 117–126.

[86] J. Hirst, J. Carroll, I.M. Fearnley, R.J. Shannon, J.E. Walker, The nuclear encoded subunits of complex I from bovine heart mitochondria, Biochim. Biophys. Acta 1604 (2003) 135–150.

[87] K.R. Vinuthkumar, J. Zhu, J. Hirst, Architecture of mammalian respiratory complex I, Nature 515 (2014) 80–84.

[88] J.A. Mayr, F.A. Zimmermann, C. Fauth, C. Bergheim, D. Meierhofer, D. Radmayr, J. Zschocke, J. Koch, W. Sperl, Lipoic acid synthetase deficiency causes neonatal-onset epilepsy, defective mitochondrial energy metabolism, and glycine elevation, Am. J. Hum. Genet. 89 (2011) 792–797.

[89] N. Ajit Bolar, A.V. Vanlander, C. Wilbrecht, N. Van der Aa, J. Smet, B. De Paepe, G. Vandeweyer, F. Kooy, F. Eyskens, E. De Latter, G. Delanghe, P. Govaert, J.G. Leroy, B. Loeys, R. Lill, L. Van Laer, R. Van Coster, Mutation of the iron-sulfur cluster assembly gene IBA57 causes severe myopathy and encephalopathy, Hum. Mol. Genet. 22 (2013) 2590–2602.

[90] J.A. Mayr, R.G. Feichtinger, F. Tort, A. Ribes, W. Sperl, Lipoic acid biosynthesis defects, J. Inherit. Metab. Dis. 37 (2014) 553–563.

[91] Y. Tsurusaki, R. Tanaka, S. Shimada, K. Shimojima, M. Shiina, M. Nakashima, H. Saitsu, N. Miyake, K. Ogata, T. Yamamoto, N. Matsumoto, Novel compound heterozygous LIAS mutations cause glycine encephalopathy, J. Hum. Genet. 60 (2015) 631–635.

[92] Y. Soreze, A. Boutron, F. Habarou, C. Barnerias, L. Nonnenmacher, H. Delpech, A. Mamoune, D. Chretien, L. Hubert, C. Bole-Feyson, P. Nitschke, I. Correia, C. Sardet, N. Boddaert, Y. Hamel, A. Delahodde, C. Ottolenghi, P. de Lonlay, Mutations in human lipoyltransferase gene LIPT1 cause a Leigh disease with secondary deficiency for pyruvate and alpha-ketoglutarate dehydrogenase, Orphanet J. Rare Dis. 8 (2013) (192–1172–8-192).

[93] F. Tort, X. Ferrer-Cortes, M. Thio, A. Navarro-Sastre, L. Matalonga, E. Quintana, N. Bujan, A. Arias, J. Garcia-Villoria, C. Acquaviva, C. Vianey-Saban, R. Artuch, A. Garcia-Cazorla, P. Briones, A. Ribes, Mutations in the lipoyltransferase LIPT1 gene cause a fatal disease associated with a specific lipoylation defect of the 2-ketoacid dehydrogenase complexes, Hum. Mol. Genet. 23 (2014) 1907–1915.

[94] V. Tache, L. Bivina, S. White, J. Gregg, J. Deignan, S.A. Boyadjiev, F.R. Poulain, Lipoyltransferase 1 gene defect resulting in fatal lactic acidosis in two siblings, Case Rep. Obstet. Gynecol. 2016 (2016) 6520148.

[95] J.M. Cameron, A. Janer, V. Levandovskiy, N. Mackay, T.A. Rouault, W.H. Tong, I. Ogilvie, E.A. Shoubridge, B.H. Robinson, Mutations in iron-sulfur cluster scaffold

Please cite this article as: A.J. Kastaniotis, et al., Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology, Biochim. Biophys. Acta (2016), http://dx.doi.org/10.1016/j.bbalip.2016.08.011

genes NFU1 and BOLA3 cause a fatal deficiency of multiple respiratory chain and 2-oxoacid dehydrogenase enzymes, Am. J. Hum. Genet. 89 (2011) 486–495.

[96] P.R. Baker II, M.W. Friederich, M.A. Swanson, T. Shaikh, K. Bhattacharya, G.H. Scharer, J. Aicher, G. Creadon-Swindell, E. Geiger, K.N. MacLean, W.T. Lee, C. Deshpande, M.L. Freckmann, L.Y. Shih, M. Wasserstein, M.B. Rasmussen, A.M. Lund, P. Procopis, J.M. Cameron, B.H. Robinson, G.K. Brown, R.M. Brown, A.G. Compton, C.L. Dieckmann, R. Collard, C.R. Coughlin II, E. Spector, M.F. Wempe, J.L. Van Hove, Variant non ketotic hyperglycinemia is caused by mutations in LIAS, BOLA3 and the novel gene GLRX5, Brain 137 (2014) 366–379.

[97] A. Navarro-Sastre, F. Tort, O. Stehling, M.A. Uzarska, J.A. Arranz, M. Del Toro, M.T. Labayru, J. Landa, A. Font, J. Garcia-Villoria, B. Merinero, M. Ugarte, L.G. Gutierrez-Solana, J. Campistol, A. Garcia-Cazorla, J. Vaquerizo, E. Riudor, P. Briones, O. Elpeleg, A. Ribes, R. Lill, A fatal mitochondrial disease is associated with defective NFU1 function in the maturation of a subset of mitochondrial Fe-S proteins, Am. J. Hum. Genet. 89 (2011) 656–667.

[98] Y.Q. Shen, G. Burger, Plasticity of a key metabolic pathway in fungi, Funct. Integr. Genomics 9 (2009) 145–151.

[99] R.N. Patkar, M. Ramos-Pamplona, A.P. Gupta, Y. Fan, N.I. Naqvi, Mitochondrial beta-oxidation regulates organellar integrity and is necessary for conidial germination and invasive growth in Magnaporthe oryzae, Mol. Microbiol. 86 (2012) 1345–1363.

[100] J.B. Jackson, Proton translocation by transhydrogenase, FEBS Lett. 545 (2003) 18–24.

[101] H. Yang, T. Yang, J.A. Baur, E. Perez, T. Matsui, J.J. Carmona, D.W. Lamming, N.C. Souza-Pinto, V.A. Bohr, A. Rosenzweig, R. de Cabo, A.A. Sauve, D.A. Sinclair, Nutrient-sensitive mitochondrial NAD+ levels dictate cell survival, Cell 130 (2007) 1095–1107.

[102] R.J. Mailloux, M.E. Harper, Glucose regulates enzymatic sources of mitochondrial NADPH in skeletal muscle cells; a novel role for glucose-6-phosphate dehydrogenase, FASEB J. 24 (2010) 2495–2506.

[103] J.K. Hiltunen, Y. Qin, Beta-oxidation - strategies for the metabolism of a wide variety of acyl-CoA esters, Biochim. Biophys. Acta 1484 (2000) 117–128.

[104] J.R. STERN, A. DEL CAMPILLO, Enzymes of fatty acid metabolism. II. Properties of crystalline crotonase, J. Biol. Chem. 218 (1956) 985–1002.

[105] Z. Chen, H. Leskinen, E. Liimatta, R.T. Sormunen, I.J. Miinalainen, I.E. Hassinen, J.K. Hiltunen, Myocardial overexpression of Mecr, a gene of mitochondrial FAS II leads to cardiac dysfunction in mouse, PLoS One 4 (2009), e5589.

[106] M.S. Schonauer, A.J. Kastaniotis, J.K. Hiltunen, C.L. Dieckmann, Intersection of RNA processing and the type II fatty acid synthesis pathway in yeast mitochondria, Mol. Cell. Biol. 28 (2008) 6646–6657.

[107] R. Schneider, M. Massow, T. Lisowsky, H. Weiss, Different respiratory-defective phenotypes of Neurospora crassa and Saccharomyces cerevisiae after inactivation of the gene encoding the mitochondrial acyl carrier protein, Curr. Genet. 29 (1995) 10–17.

[108] A. Parl, S.L. Mitchell, H.B. Clay, S. Reiss, Z. Li, D.G. Murdock, The mitochondrial fatty acid synthesis (mtFASII) pathway is capable of mediating nuclear-mitochondrial cross talk through the PPAR system of transcriptional activation, Biochem. Biophys. Res. Commun. 441 (2013) 418–424.

[109] D.G. Kim, J.C. Yoo, E. Kim, Y.S. Lee, O.V. Yarishkin, Y. da Lee, K.H. Lee, S.G. Hong, E.M. Hwang, J.Y. Park, A novel cytosolic isoform of mitochondrial trans-2-enoyl-CoA

reductase enhances peroxisome proliferator-activated receptor alpha activity, Endocrinol. Metab. (Seoul) 29 (2014) 185–194.

[110] N. Masuda, H. Yasumo, T. Furusawa, T. Tsukamoto, H. Sadano, T. Osumi, Nuclear receptor binding factor-1 (NRBF-1), a protein interacting with a wide spectrum of nuclear hormone receptors, Gene 221 (1998) 225–233.

[111] M. Yamazoe, K. Shirahige, M.B. Rashid, Y. Kaneko, T. Nakayama, N. Ogasawara, H. Yoshikawa, A protein which binds preferentially to single-stranded core sequence of autonomously replicating sequence is essential for respiratory function in mitochondria of Saccharomyces cerevisiae, J. Biol. Chem. 269 (1994) 15244–15252.

[112] P.T. Doulias, M. Tenopoulou, J.L. Greene, K. Raju, H. Ischiropoulos, Nitric oxide regulates mitochondrial fatty acid metabolism through reversible protein S-nitrosylation, Sci. Signal. 6 (2013) (rs1).

[113] A. Gurvitz, Triclosan inhibition of mycobacterial InhA in Saccharomyces cerevisiae: yeast mitochondria as a novel platform for in vivo antimycolate assays, Lett. Appl. Microbiol. 50 (2010) 399–405.

[114] R.M. Cicchillo, S.J. Booker, Mechanistic investigations of lipoic acid biosynthesis in Escherichia coli: both sulfur atoms in lipoic acid are contributed by the same lipoyl synthase polypeptide, J. Am. Chem. Soc. 127 (2005) 2860–2861.

[115] V. Praphanphoj, K.A. Sacksteder, S.J. Gould, G.H. Thomas, M.T. Geraghty, Identification of the alpha-aminoadipic semialdehyde dehydrogenase-phosphopantetheinyl transferase gene, the human ortholog of the yeast LYS5 gene, Mol. Genet. Metab. 72 (2001) 336–342.

[116] L. Zhang, A.K. Joshi, S. Smith, Cloning, expression, characterization, and interaction of two components of a human mitochondrial fatty acid synthase. Malonyltransferase and acyl carrier protein, J. Biol. Chem. 278 (2003) 40067–40074.

[117] Z.J. Chen, R. Pudas, S. Sharma, O.S. Smart, A.H. Juffer, J.K. Hiltunen, R.K. Wierenga, A.M. Haapalainen, Structural enzymological studies of 2-enoyl thioester reductase of the human mitochondrial FAS II pathway: new insights into its substrate recognition properties, J. Mol. Biol. 379 (2008) 830–844.

[118] L. Nagarajan, R.K. Storms, Molecular characterization of GCV3, the Saccharomyces cerevisiae gene coding for the glycine cleavage system hydrogen carrier protein, J. Biol. Chem. 272 (1997) 4444–4450.

[119] A. Zay, F.Y. Choy, C. Patrick, G. Sinclair, Glycine cleavage enzyme complex: molecular cloning and expression of the H-protein cDNA from cultured human skin fibroblasts, Biochem. Cell Biol. 89 (2011) 299–307.

[120] B. Schilke, C. Voisine, H. Beinert, E. Craig, Evidence for a conserved system for iron metabolism in the mitochondria of Saccharomyces cerevisiae, Proc. Natl. Acad. Sci. U. S. A. 96 (1999) 10206–10211.

[121] Y. Liu, W. Qi, J.A. Cowan, Iron-sulfur cluster biosynthesis: functional characterization of the N- and C-terminal domains of human NFU, Biochemistry 48 (2009) 973–980.

[122] Y. Liu, J.A. Cowan, Iron-sulfur cluster biosynthesis: characterization of a molten globule domain in human NFU, Biochemistry 48 (2009) 7512–7518.

[123] Y. Wang, A.W. Mohsen, S.J. Mihalik, E.S. Goetzman, J. Vockley, Evidence for physical association of mitochondrial fatty acid oxidation and oxidative phosphorylation complexes, J. Biol. Chem. 285 (2010) 29834–29841.

[124] E.S. Goetzman, Modeling disorders of fatty acid metabolism in the mouse, Prog. Mol. Biol. Transl. Sci. 100 (2011) 389–417.
