
ARTICLE IN PRESS

Biochimie xxx (2016) 1–7

Contents lists available at ScienceDirect

Biochimie

journal homepage: www.elsevier.com/locate/biochi

Review

Transcriptional regulation of the uncoupling protein-1 gene

Francesc Villarroya ${ }^{a, b, c, *}$ , Marion Peyrou ${ }^{a, b, c}$ , Marta Giralt ${ }^{a, b, c}$

${ }^{a}$ Department of Biochemistry and Molecular Biomedicine, Institut de Biomedicina (IBUB), University of Barcelona, Barcelona, Catalonia, Spain  
${ }^{b}$ CIBER Fisiopatología de la Obesidad y Nutrición, Instituto de Salud Carlos III, Spain  
${ }^{c}$ Institut de Recerca Pediàtrica Sant Joan de Déu, Barcelona, Catalonia, Spain

---

**A R T I C L E I N F O**

Article history:  
Received 13 July 2016  
Accepted 25 September 2016  
Available online xxx  

Keywords:  
Uncoupling protein-1  
Brown adipose tissue  
Beige adipose tissue  
Transcription  

---

**A B S T R A C T**

Regulated transcription of the uncoupling protein-1 ($U C P 1$) gene, and subsequent $U C P 1$ protein synthesis, is a hallmark of the acquisition of the differentiated, thermogenically competent status of brown and beige/brite adipocytes, as well as of the responsiveness of brown and beige/brite adipocytes to adaptive regulation of thermogenic activity. The $5^{\prime}$ non-coding region of the $U C P 1$ gene contains regulatory elements that confer tissue specificity, differentiation dependence, and neuro-hormonal regulation to $U C P 1$ gene transcription. Two main regions—a distal enhancer and a proximal promoter region—mediate transcriptional regulation through interactions with a plethora of transcription factors, including nuclear hormone receptors and cAMP-responsive transcription factors. Co-regulators, such as PGC-$1 \alpha$, play a pivotal role in the concerted regulation of $U C P 1$ gene transcription. Multiple interactions of transcription factors and co-regulators at the promoter region of the $U C P 1$ gene result in local chromatin remodeling, leading to activation and increased accessibility of RNA polymerase II and subsequent gene transcription. Moreover, a commonly occurring A-to-G polymorphism in close proximity to the $U C P 1$ gene enhancer influences the extent of $U C P 1$ gene transcription. Notably, it has been reported that specific aspects of obesity and associated metabolic diseases are associated with human population variability at this site. On another front, the unique properties of the $U C P 1$ promoter region have been exploited to develop brown adipose tissue-specific gene delivery tools for experimental purposes.

© 2016 Elsevier B.V. and Société Française de Biochimie et Biologie Moléculaire (SFBBM). All rights reserved.

---

**Contents**

1. Introduction ..... 00
2. Regulatory structure of $U C P 1$ gene transcription ..... 00
3. Analysis of $U C P 1$ gene transcription: a tool for the identification of new molecular actors in the control of brown and beige/brite cell differentiation and overall energy metabolism ..... 00
4. Chromatin remodeling and epigenetic control of $U C P 1$ gene transcription ..... 00
5. Polymorphisms of the human $U C P 1$ promoter and disease ..... 00
6. Transcriptional properties of the $U C P 1$ gene promoter: a transgenic engineering tool ..... 00
7. Conclusions ..... 00

---

Abbreviation list: ATF-2, activating transcription factor-2; BAT, brown adipose tissue; C/EBP, CCAAT-enhancer binding protein; CREB, cAMP-responsive element binding protein; JMJD1A, Jumonji domain-containing 1A; LXR, liver X receptor; LSD1, lysine-specific histone demethylase-1; PPAR, peroxisome-proliferator activated receptor; PGC-$1 \alpha$, proliferator-activated receptor-$\gamma$ coactivator 1-$\alpha$; PRDM16, PR domain-containing protein-16; RAR, retinoic acid receptor; RXR, retinoid X receptor; RIP140, receptor-interacting protein 140; TR, thyroid hormone receptor; UCP1, uncoupling protein-1; Zfp516, zinc finger protein 516.

* Corresponding author. Departament de Bioquímica i Biomedicina Molecular, Universitat de Barcelona, Avda Diagonal 643, E-08028, Barcelona, Spain.  
E-mail address: fvillarroya@ub.edu (F. Villarroya).

http://dx.doi.org/10.1016/j.biochi.2016.09.017  
0300-9084/© 2016 Elsevier B.V. and Société Française de Biochimie et Biologie Moléculaire (SFBBM). All rights reserved.

Please cite this article in press as: F. Villarroya, et al., Transcriptional regulation of the uncoupling protein-1 gene, Biochimie (2016), http://dx.doi.org/10.1016/j.biochi.2016.09.017

ARTICLE IN PRESS

F. Villarroya et al. / Biochimie xxx (2016) 1–7

Disclosure of potential conflicts of interest ..... 00  
Acknowledgments ..... 00  
References ..... 00  

---

### 1. Introduction

Relatively soon after the discovery of uncoupling protein-1 (UCP1) as the key protein that confers specific thermogenic properties on brown adipocyte mitochondria, major elements of the transcriptional regulation of the UCP1 gene were established. In the 1980s, before PCR-based technologies were available, the basic features of UCP1 gene structure and transcriptional control were established using classical molecular biology techniques. Essentially, physiological inducers of thermogenesis in brown adipose tissue (BAT) and previously identified increases in the amount of UCP1 protein that follow were found to be associated with marked changes in UCP1 gene transcript levels. After indirect techniques demonstrated that UCP1 mRNA levels increased in response to thermogenic stimuli [1], the cloning of UCP1 cDNA probes in rodents [2–4] allowed UCP1 mRNA levels to be measured by quantitative assessment of Northern blot assays. Data from the Ricquier laboratory at CNRS (France) and the Kozak laboratory at the Jackson Laboratory (USA) demonstrated that UCP1 mRNA transcripts were present only in BAT and their abundance was strongly induced in response to cold and β-adrenergic stimulation in rodents [4,5]. Measurement of nascent UCP1 transcripts using run-on assays directly demonstrated the transcriptional regulation of the UCP1 gene in response to thermogenic stimuli [6]. The first cDNA probe for human UCP1 was subsequently isolated, allowing human UCP1 mRNA to be monitored in humans and establishing its specific expression in BAT in humans as well as rodents [7]. The complete sequence and structure of the UCP1 gene from rats, mice and humans [8–10] was determined, including the sequence of a large part of the 5′ non-coding region. Thanks to these studies, it has become possible to undertake analyses of the molecular mechanisms underlying the strong transcriptional regulation of the UCP1 gene, which is essential for modulating the amounts of UCP1 protein generated in response to physiological inducers and/or repressors, and to assess this regulation in relation to tissue specificity and the brown adipocyte differentiation processes. Moreover, the ease with which specific transcripts can be accurately measured using quantitative real-time PCR techniques has made the quantification of UCP1 mRNA levels a routine part of the characterization of BAT in distinct experimental settings.

After several decades of research, we have come to recognize three major features of UCP1 protein expression in relation to the biology of BAT that ultimately rely on regulation of the UCP1 gene:  
a) UCP1 protein is present only in brown adipocytes, a broad definition that includes currently termed beige/brite adipocytes (brown adipocyte-like cells that appear in white fat depots under some circumstances), and not in other cell types; b) UCP1 is expressed in association with the differentiation state of brown adipocytes; and c) UCP1 expression is quantitatively regulated in response to exogenous, distinct signals in accordance with the physiological requirements for active thermogenesis in BAT. These three aspects of the control of UCP1 protein levels are achieved mainly through transcriptional regulation of the UCP1 gene, and are monitored in most studies through quantification of UCP1 mRNA levels as an indicator of the intensity of UCP1 gene transcription.

In this review, we summarize current knowledge regarding the molecular mechanisms that control UCP1 gene transcription in rodents and humans and the influence of polymorphisms at the 5′ non-coding regulatory region of the UCP1 gene in human diseases. We also address utilization of the unique properties of UCP1 gene transcription for experimental and biotechnology research.

---

### 2. Regulatory structure of UCP1 gene transcription

The molecular basis of the transcriptional control of the UCP1 gene has been extensively studied using mouse and rat UCP1 genes, as well as the human gene. Transcriptional control relies on regulatory regions in the 5′ non-coding region of the gene; notably, the cis-acting structure of this regulation is largely shared by rodent and human genes. A recent review by Bonet et al. described most of the regulatory elements, transcription factors, and co-regulators known to be involved in detail [11], and an updated schematic representation is shown in Fig. 1. These features include a proximal regulatory region, immediately upstream of the transcription start site, that contains a CREB (cAMP response element-binding protein) binding site, which mediates a positive transcriptional response to cAMP [12,13] and a negative response to AP2 (c-Jun/c-Fos) complexes [13], and a C/EBP (CCAAT-enhancer-binding protein) binding site capable of responding to C/EBPα and C/EBPβ [14]. Recently, the transcription factor Zfp516 (zinc finger protein-516) has been shown to bind to the proximal region of the UCP1 gene promoter to play a role in cold-induced UCP1 gene expression [15].

In addition to the proximal region, there is a strong enhancer region approximately 2.5 kb upstream of the transcription-initiation site in rodents (3.9 kb in the human gene) [12,16,17]. This region is strongly involved in determining BAT-specific transcription of the UCP1 gene [12,17]. It contains a cluster of response elements for nuclear hormone receptors that bind PPARα/RXR (peroxisome proliferator-activated receptor-α/retinoid X receptor), PPARγ/RXR, RAR/RXR and thyroid receptor (TR)/RXR heterodimers, providing responsiveness to ligands of PPARα and PPARγ (e.g., naturally occurring fatty acid derivatives, specific drugs such as fibrates and thiazolidinediones) [18–20], retinoic acids [21,22], and thyroid hormones [23,24]. Moreover, an ATF2 (activating transcription factor-2)-binding site that provides cAMP-dependent responsiveness to UCP1 gene transcription is also present in the UCP1 promoter [25,26]. Transcriptional coregulators, such as PPAR-γ coactivator 1-α (PGC-1α), bind and regulate PPAR/RXR heterodimers and possibly other nuclear receptor dimer complexes (see below).

These regulatory structures confer biological responsiveness of the UCP1 gene to brown adipocyte differentiation and environmentally mediated stimuli, and contribute to BAT-specific expression of the UCP1 gene. Regulation of the UCP1 gene by the proximal C/EBP-binding sites and distal PPARγ-binding sites couple UCP1 expression with the brown adipogenic differentiation process, whereas Jun/Fos binding contribute to maintenance of the repressed state of the UCP1 gene in undifferentiated cells. Transcriptional regulation of the UCP1 gene by cAMP-mediated signaling, elicited ultimately by noradrenergic action, is provided by protein kinase A (PKA)-mediated phosphorylation of the CREB transcription factor at the proximal UCP1 promoter region and p38 MAP kinase-mediated phosphorylation of ATF2 at the upstream enhancer region [26,27]. In fact, rapid phosphorylation of these

---

Please cite this article in press as: F. Villarroya, et al., Transcriptional regulation of the uncoupling protein-1 gene, Biochimie (2016), http://dx.doi.org/10.1016/j.biochi.2016.09.017

ARTICLE IN PRESS

F. Villarroya et al. / Biochimie xxx (2016) 1–7

Distal enhancer

![Diagram](attachment:diagram.png)

Proximal region

Fig. 1. Schematic representation of the two main regulatory regions of UCP1 gene transcription and the main ligands and signaling mediators of regulation. The figure integrates the similar structure of regulatory elements and protein factors in the UCP1 gene promoter from humans and rodents. Activating factors are shown in light grey and repressors in dark grey. ATF-2, activating transcription factor -2; C/EBP, CCAAT-enhancer binding protein; CREB, cAMP-responsive element binding protein; LXR, liver X receptor; LSD1, lysine-specific demethylase-1; PPAR, peroxisome-proliferator activated receptor; PGC-1α, proliferator-activated receptor-γ coactivator 1-α; PRDM16, PR domain-containing protein-16; RAR, retinoic acid receptor; RXR, retinoid X receptor; RIP140, receptor-interacting protein 140; TR, thyroid hormone receptor; Zfp516, zinc finger protein 516.

transcription factors in response to the respective kinases constitutes a first, fast mechanism of regulation. But there is a second layer of regulation at the protein synthesis level involving the induction of PGC-1α expression by cAMP [26,27], by activators of PPARγ [28] and PPARα [29], and by retinoids [28]. Similarly, triiodothyronine (T3) directly activates transcription of the UCP1 gene via a TR binding site in the enhancer region, but the capacity of cAMP-driven signals to induce the expression of thyroxine 5′-deiodinase in brown adipocytes, and therefore to provide active intracellular production of T3, contributes to UCP1 gene transcriptional regulation [30].

In addition to these positive regulatory elements, repressors of UCP1 gene transcription have also been identified. RIP140 (receptor-interacting protein 140), a corepressor reported to interact with nuclear receptors that bind the UCP1 gene enhancer region, is believed to contribute to the maintenance of UCP1 transcriptional repression in non-differentiated brown pre-adipocytes [31,32]. LXR (liver X receptor), an oxysterol-binding nuclear hormone receptor, also acts as a repressor of UCP1 gene transcription, in this case by interacting with the PPARγ binding site at the UCP1 gene distal enhancer and recruiting the RIP140 repressor [33]; however, the physiological significance of this action is not entirely clear. Other transcriptional repressors of UCP1 gene transcription include the retinoblastoma (Rb) protein, and the transcription factors E2F1 and HoxC8 (homeobox C8) [11], but whether their molecular actions on the UCP1 gene transcription machinery are direct or indirect is not entirely clear. There are recent indications that the unliganded vitamin D receptor (VDR) acts as negative regulator of human UCP1 gene transcription by interacting with a VDR-binding site at the 5′ non-coding region of the gene [34].

Finally, it is worth mentioning that there is some evidence that

transcriptional regulation of UCP1 is determined by cis-acting regions other than the aforementioned proximal and distal regulatory regions. Indications of BAT-specific DNA-binding factors at approximately –0.6 kb relative to the transcription start site have been reported [35]. Recently, Rosen and collaborators reported that the transcription factor IRF4 (interferon regulatory factor-4) exerts strong action on UCP1 gene transcription in response to cold through binding to a site about 1.2 kb upstream of the transcription-initiation site [36].

3. Analysis of UCP1 gene transcription: a tool for the identification of new molecular actors in the control of brown and beige/brite cell differentiation and overall energy metabolism

Analyzing the molecular mechanisms of UCP1 gene transcriptional regulation has been useful not only for revealing how UCP1 protein expression is controlled, but also as a tool for identifying molecular actors relevant to the overall biology of BAT—and beyond. Possibly the best example of this is the identification of the transcriptional coactivator PGC-1α. Puigserver and collaborators, searching for a transcriptional regulator of the UCP1 gene that was specifically induced by cold, identified PGC-1α as a transcriptional coactivator of PPARγ that acts at the UCP1 gene promoter [37]. At present, we know that PGC-1α is essential for cAMP-mediated induction of UCP1 gene transcription, as its expression is upregulated by cAMP-dependent, p38 MAP kinase-mediated stimuli and it interacts with several nuclear hormone receptors in addition to PPARγ at the UCP1 gene distal enhancer [38]. The identification of PGC-1α as a UCP1 transcription regulator has led to the recognition of a wider role for this transcriptional coregulator in other

Please cite this article in press as: F. Villarroya, et al., Transcriptional regulation of the uncoupling protein-1 gene, Biochimie (2016), http://dx.doi.org/10.1016/j.biochi.2016.09.017

essential processes that are linked to brown adipocyte differentiation, from mitochondrial biogenesis [39] to the acquisition of an insulin-sensitive cell status [40]. The strength of PGC-1α as driver of brown adipocyte differentiation is evidenced by experiments indicating that ectopic over-expression of PGC-1α leads white adipocytes to acquire a brown adipocyte phenotype [41]. Moreover, PGC-1α has been found to play an essential role in processes related to energy metabolism in other tissues, from promoting the oxidative potential of muscle cells [42] to controlling hepatic gluconeogenesis [43].

Other transcription factors and coregulators of UCP1 gene transcription shared by other genes associated with the differentiated brown or beige/brite adipocytes have recently been recognized. Among them, PRDM16 (PR domain-containing protein-16) has been found to induce UCP1 transcription through interactions with transcription factors (C/EBPβ, PPARγ, PPARα, Zfp516) and coregulators (PGC-1α) [15,29,44–46]. PRDM16 is considered a master controller of brown and beige/brite cell identity [47]. Its identification has proved essential for the recognition of specific molecular interactions that confer a specific brown/beige differentiation drive to the general adipogenic transcriptional machinery. In fact, both PPARγ and C/EBPα and -β orchestrate the differentiation program in white and brown adipose lineages [48]. It has been shown that PRDM16 and EBF2 (early B cell factor-2) recruit PPARγ specifically to the UCP1 gene and other BAT-selective genes [44,49]. EBF2 also recruits PPARγ to the promoter/enhancer region of PRDM16 itself and synergistically activates its transcription [49]. Other factors have also been reported to modulate PRDM16 activity and/or expression positively, such as PPARγ and PPARα agonists [29,50], sirtuin 1 [51] and bone morphogenetic protein-7 [52], or negatively (e.g., miR-133 [53] and Zfp423 [54]), thus amplifying PRDM16-dependent brown and beige/brite differentiation processes, including the induction of UCP1 gene transcription. Like PGC-1α, PRDM16 functions extend beyond controlling the expression of genes associated with brown and beige/brite differentiation and function; it is also known to regulate cardiac development and hematopoietic and neural stem cell homeostasis [47].

### 4. Chromatin remodeling and epigenetic control of UCP1 gene transcription

As noted above, UCP1 gene transcription is controlled by environmental signals (e.g., cold, diet), mediated by neurohormonal and hormonal signaling, as well as through developmental and cell differentiation-associated processes. As part of the developmental program of BAT morphogenesis, UCP1 gene transcription is switched on a few days before birth in rodents [55,56] and months before birth in humans [57] in the absence of environmental stressors. During differentiation, UCP1 gene transcription is induced in association with acquisition of the terminally differentiated brown and beige/brite cell phenotype. In recent years, a role for chromatin structure and epigenetic modifications in controlling transcription of the UCP1 gene in response to environmental and differentiation-related signals has been unveiled. By 2007, Parker and collaborators had already reported that the UCP1 transcriptional repressor RIP140 inhibits UCP1 gene transcription by acting as a scaffold to promote the assembly of DNA and histone methyltransferases on the UCP1 gene, leading to the methylation of CpG islands in the promoter region and altering its histone methylation status [32]. An association between BAT-specific transcription of the UCP1 gene and reduced methylation of CpG regions at the enhancer region of the UCP1 gene has also been reported [58].

Alterations in chromatin structure at the UCP1 promoter appear to mediate the interplay between tissue-specific and hormonally regulated UCP1 gene transcription. Accordingly, the UCP1 gene promoter region in non-UCP1-expressing white adipocytes is characterized by the presence of H3K9 dimethylation marks, indicative of repressed transcription; in contrast, H3K4 trimethylated marks, indicative of activation, are enriched in BAT, especially in response to cold [58]. Nardilysin, a novel histone-binding protein that specifically recognizes dimethyl H3K4, has recently been demonstrated to repress UCP1 gene transcription by interacting with PGC-1α at the UCP1 gene enhancer region; however, it is not known whether this repressive effect is related to dimethyl H3K4 binding capacity [59].

It has also been found that β-adrenergic stimulation of brown adipocytes decreases H3K27 trimethylation (a chromatin mark indicative of repressed transcription) and increases H3K27 acetylation (a chromatin mark indicative of increased transcription) at the UCP1 gene through mechanisms involving recruitment of the histone demethylase UTX (ubiquitously transcribed tetratricopeptide repeat on chromosome X) [60]. Moreover, the H3K9 demethylase, JMJD1A (Jumonji domain-containing 1A), appears to control long-range chromatin interactions at the UCP1 gene as well as responsiveness to intracellular signaling involved in regulation of UCP1 gene transcription [61]. In fact, protein kinase-A, a major intracellular mediator of thermogenic activation in brown adipocytes, phosphorylates JMJD1 and promotes nucleosome remodeling and binding of PPARγ to the UCP1 promoter [61]. Lysine-specific demethylase 1 (LSD1) has also been reported to contribute to H3K9 demethylation of the UCP1 promoter in response to thermogenic activation [46]. Moreover, β-adrenergic stimuli cause dissociation of histone deacetylase-1 (HADC1) from the UCP1 gene promoter and favor transcription-promoting H3K27 acetylation [60,62].

Recent studies have identified a role for the Mediator complex, which is central to the control of chromatin architecture, in the control of UCP1 gene expression [63,64]. Mediator is a complex that bridges enhancer regions and their associated transcription factor complexes with RNA Pol II and the transcriptional machinery at gene promoters. It has been proposed that, after PRDM16 interacts with regulatory regions of the UCP1 gene (and other genes preferentially expressed in BAT) through C/EBPβ and PPARγ, as indicated above, PRDM16 recruits MED1 (a component of the Mediator complex) to bridge enhancer-bound transcription factors with RNA Pol II—the general transcription machinery. Harms et al. [63] proposed that this process occurs sequentially. According to their model, the first step in this regulation involves PGC-1α-mediated recruitment of the histone acetylase p300 and PRDM16 binding at the enhancer region of UCP1, followed by acetylation of histones at the UCP1 promoter region and chromatin remodeling. In the second step, Mediator serves to facilitate the formation and/or function of the transcription pre-initiation complex [64].

### 5. Polymorphisms of the human UCP1 promoter and disease

Although several genetic variants of the UCP1 gene have been found in human populations, the most prevalent and extensively studied is one found at position –3826 relative to the TATA box in the UCP1 5′ non-coding region that appears to impact transcriptional regulation of the UCP1 gene. This polymorphism, which consists of an A to G point mutation, occurs in very close proximity to the distal enhancer region of the human UCP1 gene that controls gene transcription, as described above [65]. This polymorphism has been found to influence the extent of UCP1 gene expression in visceral adipose tissue from obese patients such that, in the presence of the G allele, UCP1 gene expression is reduced [66]. It was also recently demonstrated that the –3826 A to G variant (A-3826G) decreases the promoter activity of the UCP1 gene [67]. Since its discovery, dozens of studies have shown an association of this polymorphic variant in the 5′ region of the UCP1 gene with obesity

and associated metabolic diseases in populations of distinct ages, gender, and ethnicity. In 2000, M. Del Mar Gonzalez-Barroso and colleagues summarized the close to 20 studies that had been performed to that date, and concluded that the A-3826G polymorphism is not a major contributor to obesity development; instead, they suggested that the presence of the minor variant allele is related to an increased propensity toward weight gain over time [68]. A further summary of nearly 30 studies linking the A-3825G polymorphism with obesity and/or associated diseases (diabetes, dyslipidemia) reported in 2010 reached similar conclusions [69]. Since that time, multiple additional studies have either claimed a positive association of the A-3826G polymorphism with obesity or distinct types of associated co-morbidities (diabetes, dyslipidemia) [70,71] or dismissed this type of association [72,73]. The current consensus appears to be that A-3826G is not associated with overt obesity, but may have some association with certain aspects of the weight gain phenotype and associated diseases, especially type II diabetes, in particular populations.

Several studies have reported a synergistic effect of the A-3826G UCP1 promoter polymorphism and the missense mutation, Trp64Arg, in the β3-adrenergic receptor—the main mediator of the cellular responsiveness of brown and beige/brite adipocytes to thermogenic activation—on weight gain in obesity [74]. As for single analyses of the UCP1 gene promoter, subsequent studies have not been unanimous, ranging from reports of synergistic or additive associations of both polymorphisms with low metabolic rate in obese patients [75], resistance to weight loss [76,77] and visceral fat accumulation [78] or hyperlipidemia [79] to reports of no effects [80].

Recently, Saito and co-workers reported the first study that related the A-3826G UCP1 gene promoter polymorphism with human BAT activity, as assessed using the recently developed FDG-PET/CT scanning technology [81]. Their results indicated an impact of this UCP1 gene polymorphism on adiposity-related parameters in old, but not young, individuals, and also showed cumulative effects of the UCP1 G allele and β3-adrenergic receptor Arg-containing mutant on age-related accumulation of visceral fat and decreases in BAT activity.

In addition to A-3826G, other polymorphisms in the 5′ non-coding region of the UCP1 gene have been reported. Rousset et al. reported the existence of a polymorphic C to T point mutation at –3737, lying to the ATF binding site in the UCP1 gene enhancer [82], although it was not found to result in detectable changes in UCP1 transcriptional activity. A–1766G polymorphism that tended to be associated with increased body weight in Korean subjects has also been reported [83,84].

Finally, it is worth noting that genome-wide association studies have not identified UCP1 among genes for which common variants are associated with adiposity [85,86] or type II diabetes [87].

### 6. Transcriptional properties of the UCP1 gene promoter: a transgenic engineering tool

The properties of the UCP1 gene promoter region have been exploited for transgenesis-based experimentation, taking advantage of the unique cell-specific expression properties of UCP1 gene transcription in brown adipocytes. Among the main applications, one that should be highlighted is a mouse transgenic model in which the transcriptional regulatory region of the UCP1 gene driving expression of diphtheria toxin was used to genetically ablate BAT [88]. This model allowed the role of brown fat in energy expenditure and protection against obesity to be established.

The brown fat-specific transcriptional properties of the UCP1 promoter have also been used in the loxP DNA-recombination system, which permits the use of UCP1 promoter (BAT-specific)-driven expression of Cre recombinase to catalyze an intramolecular recombination reaction that results in deletion of a LoxP-flanked gene of interest. This has made it possible to ablate distinct candidate genes that are thought to be involved in brown adipocyte biology so as to explore their function specifically in BAT. Examples include using Cre recombinase driven by an 8-kb region of the mouse UCP1 gene promoter to achieve brown fat-specific deletion of the insulin receptor [89] or the hypoxia-inducible factor HIF-1α [90] to establish a role for these proteins in BAT. The specific roles of IRF4 and LSD1 as intracellular regulators of BAT function have recently been established with the help of this type of approach [36,46].

The properties of the UCP1 transcription regulatory region and its developmentally programmed behavior have also been used in experiments designed to trace cell lineage and development of brown adipocytes. Casteilla and collaborators [91] used this type of approach to drive the expression of a dormant LacZ gene by a Cre recombinase-expressing transgene under the control of a 4.5-kb non-coding region of the rat UCP1 gene. This approach allowed the identification of cells that had expressed UCP1 at any time during development and supported the conclusion that white adipocytes do not develop from brown adipocytes during embryogenesis. Recently, Wolfum and collaborators generated transgenic mice in which a GFP (green fluorescent protein) reporter or tamoxifen-regulated Cre recombinase were expressed under the control of the UCP1 promoter, thereby allowing brown and beige/brite differentiation processes and inter-conversion to be traced in relation to the white adipocyte cell lineage in several experimental settings [92]. A reporter system based on luciferase activity and green fluorescent protein (GFP) expression under the control of a 4.2-kb region of the human UCP1 promoter has been developed recently to monitor differentiation of precursor cells into brown and beige/brite adipocytes in vitro and in vivo [93].

It should also be noted that rat UCP1 promoter-based constructs have been used successfully to confer BAT-specific expression through adeno-associated viral vector-mediated gene transfer in vivo in adult mice [94], opening a further perspective on the use of the specificity of the UCP1 gene transcription-control region to achieve BAT-specific gene delivery for experimental purposes.

Given the unique features of the UCP1 gene as a qualitative marker of the brown and beige/brite differentiated cell, the transcription regulatory region of the UCP1 gene has become a peerless tool for driving brown (including beige/brite) adipocyte-specific gene expression or gene invalidation.

### 7. Conclusions

Transcriptional regulation of the UCP1 gene appears to be a key event in the process of acquiring the thermogenic properties and activity of brown and beige/brite adipocytes and the subsequent healthy metabolic effects associated with brown adipose tissue activity. The molecular machinery that controls UCP1 gene transcription is shared by genes encoding other proteins involved in BAT thermogenesis; however, cell-specific UCP1 gene transcription is the most stringently restricted to differentiated brown and beige/brite adipocytes. Moreover, the machinery controlling UCP1 gene transcription integrates mechanisms involved in responding to multiple signals (noradrenergic and others) that determine the thermogenic response of BAT to environmental changes. UCP1 gene transcription constitutes not only a qualitative marker of brown and beige/brite cell identity, but is also a powerful tool for genetic engineering and other experimental approaches designed to specifically target BAT. Although the available information on the regulation of UCP1 gene transcription is rather extensive, there are still unresolved issues that warrant further research.

importance of transcriptional regulation of the UCP1 gene should not dismiss the existence of relevant post-transcriptional mechanisms contributing to the regulation of UCP1 protein synthesis, as recently reported [95,96]. Moreover, the use of UCP1 mRNA abundance measurement as indicator of the extent of thermogenic activation of brown or beige adipose depots should be taken with caution because, as discussed by Cannon et al. [97], total UCP1 content in adipose depots is the most reliable indicator of thermogenic capacity. Improving our understanding of the transcriptional regulation of the UCP1 gene may provide invaluable insight into basic biological processes, such as adipogenic cell differentiation, as well as the pathophysiological events associated with variable BAT activity.

**Disclosure of potential conflicts of interest**

The authors indicate no potential conflicts of interest.

**Acknowledgments**

Our research was supported by grants SAF2014-55725 (Ministerio de Ciencia e Innovación) and PI14/00063 (Instituto de Salud Carlos III, Spain), co-financed by the European Regional Development Fund (ERDF).

**References**

[1] F. Bouillaud, D. Ricquier, G. Mory, et al., Increased level of mRNA for the uncoupling protein in brown adipose tissue of rats during thermogenesis induced by cold exposure or norepinephrine infusion, J. Biol. Chem. 259 (1984) 11583–11586.

[2] D. Ricquier, G. Mory, F. Bouillaud, et al., Rapid increase of mitochondrial uncoupling protein and its mRNA in stimulated brown adipose tissue. Use of a cDNA probe, FEBS Lett. 178 (1984) 240–244.

[3] F. Bouillaud, D. Ricquier, J. Thibault, et al., Molecular approach to thermogenesis in brown adipose tissue: cDNA cloning of the mitochondrial uncoupling protein, Proc. Natl. Acad. Sci. U. S. A. 82 (1985) 445–448.

[4] A. Jacobsson, U. Stadler, M.A. Glotzer, et al., Mitochondrial uncoupling protein from mouse brown fat. Molecular cloning, genetic mapping and mRNA expression, J. Biol. Chem. 260 (1985) 16250–16254.

[5] D. Ricquier, F. Bouillaud, P. Toumelin, et al., Expression of uncoupling protein mRNA in thermogenic or weakly thermogenic brown adipose tissue. Evidence for a rapid beta-adrenoreceptor-mediated and transcriptionally regulated step during activation of thermogenesis, J. Biol. Chem. 261 (1986) 13905–13910.

[6] A.C. Bianco, X.Y. Sheng, J.E. Silva, Triiodothyronine amplifies norepinephrine stimulation of uncoupling protein gene transcription by a mechanism not requiring protein synthesis, J. Biol. Chem. 263 (1988) 18168–18243.

[7] F. Bouillaud, F. Villarroya, E. Hentz, et al., Detection of brown adipose tissue uncoupling protein mRNA in adult patients by a human genomic probe, Clin. Sci. 75 (1988) 21–27.

[8] B.B. Boyer, L.P. Kozak, The mitochondrial uncoupling protein gene in brown fat: correlation between DNase I hypersensitivity and expression in transgenic mice, Mol. Cell. Biol. 11 (1991) 4147–4156.

[9] F. Bouillaud, S. Raimbault, D. Ricquier, The gene for rat uncoupling protein: complete sequence, structure of primary transcript and evolutionary relationships between exons, Biochem. Biophys. Res. Commun. 157 (1988) 783–792.

[10] A.M. Cassard, F. Bouillaud, M.G. Mattei, et al., Human uncoupling protein gene: structure, comparison with rat gene, and assignment to the long arm of chromosome 4, J. Cell. Biochem. 43 (1990) 255–264.

[11] M.L. Bonet, P. Oliver, A. Palou, Pharmacological and nutritional agents promoting browning of white adipose tissue, Biochim. Biophys. Acta 1831 (2013) 969–985.

[12] U.C. Kozak, J. Kopecky, J. Teisinger, et al., An upstream enhancer regulating brown-fat-specific expression of the mitochondrial uncoupling protein gene, Mol. Cell. Biol. 14 (1994) 59–67.

[13] P. Yubero, M.J. Barberá, R. Alvarez, et al., Dominant negative regulation by c-Jun of transcription of the uncoupling protein-1 gene through a proximal cAMP-regulatory element: a mechanism for repressing basal and norepinephrine-induced expression of the gene before brown adipocyte differentiation, Mol. Endocrinol. 12 (1998) 1023–1037.

[14] P. Yubero, C. Manchado, A.M. Cassard-Doulcier, et al., CCAAT/enhancer binding proteins alpha and beta are transcriptional activators of the brown fat uncoupling protein gene promoter, Biochem. Biophys. Res. Commun. 198 (1994) 653–659.

[15] J. Dempersmier, A. Sambeat, O. Gulyaeva, et al., Cold-inducible Zfp516

activates UCP1 transcription to promote browning of white fat and development of brown fat, Mol. Cell 57 (2015) 235–246.

[16] A.M. Cassard-Doulcier, C. Gelly, N. Fox, et al., Tissue-specific and beta-adrenergic regulation of the mitochondrial uncoupling protein gene: control by cis-acting elements in the 5′-flanking region, Mol. Endocrinol. 7 (1993) 497–506.

[17] A.M. Cassard-Doulcier, C. Gelly, F. Bouillaud, et al., 211-bp enhancer of the rat uncoupling protein-1 (UCP-1) gene controls specific and regulated expression in brown adipose tissue, Biochem. J. 333 (1998) 243–246.

[18] M.J. Barbera, A. Schluter, N. Pedraza, et al., Peroxisome proliferator-activated receptor alpha activates transcription of the brown fat uncoupling protein-1 gene. A link between regulation of the thermogenic and lipid oxidation pathways in the brown fat cell, J. Biol. Chem. 276 (2001) 1486–1493.

[19] I.B. Sears, M.A. MacGinnitie, L.G. Kovacs, et al., Differentiation-dependent expression of the brown adipocyte uncoupling protein gene: regulation by peroxisome proliferatoractivated receptor gamma, Mol. Cell. Biol. 16 (1996) 3410–3419.

[20] F. Villarroya, R. Iglesias, M. Giralt, PPARs in the control of uncoupling proteins gene expression, PPAR Res. 2007 (2007) 74364.

[21] R. Alvarez, J. de Andres, P. Yubero, et al., A novel regulatory pathway of brown fat thermogenesis. Retinoic acid is a transcriptional activator of the mitochondrial uncoupling protein gene, J. Biol. Chem. 270 (1995) 5666–5673.

[22] M. Larose, A.M. Cassard-Doulcier, C. Fleury, et al., Essential cis-acting elements in rat uncoupling protein gene are in an enhancer containing a complex retinoic acid response domain, J. Biol. Chem. 271 (1996) 31533–31542.

[23] A.M. Cassard-Doulcier, M. Larose, J.C. Matamala, et al., In vitro interactions between nuclear proteins and uncoupling protein gene promoter reveal several putative transactivating factors including Ets1, retinoid X receptor, thyroid hormone receptor, and a CACCC box- binding protein, J. Biol. Chem. 269 (1994) 24335–24342.

[24] R. Rabelo, A. Schifman, A. Rubio, et al., Delineation of thyroid hormone-responsive sequences within a critical enhancer in the rat uncoupling protein gene, Endocrinology 136 (1995) 1003–1013.

[25] W. Cao, A.V. Medvedev, K.W. Daniel, et al., beta-Adrenergic activation of p38 MAP kinase in adipocytes: cAMP induction of the uncoupling protein 1 (UCP1) gene requires p38 MAP kinase, J. Biol. Chem. 276 (2001) 27077–27082.

[26] W. Cao, K.W. Daniel, J. Robidoux, et al., p38 mitogen-activated protein kinase is the central regulator of cyclic AMP-dependent transcription of the brown fat uncoupling protein 1 gene, Mol. Cell. Biol. 24 (2004) 3057–3067.

[27] J. Robidoux, W. Cao, H. Quan, et al., Selective activation of mitogen-activated protein (MAP) kinase kinase 3 and p38alpha MAP kinase is essential for cyclic AMP-dependent UCP1 expression in adipocytes, Mol. Cell. Biol. 25 (2005) 5466–5479.

[28] E. Hondares, O. Mora, P. Yubero, et al., Thiazolidinediones and rexinoids induce peroxisome proliferator-activated receptor-coactivator (PGC)-1alpha gene transcription: an autoregulatory loop controls PGC-1alpha expression in adipocytes via peroxisome proliferator-activated receptor-gamma coactivation, Endocrinology 147 (2006) 2829–2838.

[29] E. Hondares, M. Rosell, J. Díaz-Delfín, et al., Peroxisome proliferator-activated receptor α (PPARα) induces PPARγ coactivator 1α (PGC-1α) gene expression and contributes to thermogenic activation of brown fat: involvement of PRDM16, J. Biol. Chem. 286 (2011) 43112–43122.

[30] J.E. Silva, R. Rabelo, Regulation of the uncoupling protein gene expression, Eur. J. Endocrinol. 136 (1997) 251–264.

[31] G. Leonardsson, J.H. Steel, M. Christian, et al., Nuclear receptor corepressor RIP140 regulates fat accumulation, Proc. Natl. Acad. Sci. U. S. A. 101 (2004) 8437–8442.

[32] E. Kiskinis, M. Hallberg, M. Christian, et al., RIP140 directs histone and DNA methylation to silence Ucp1 expression in white adipocytes, EMBO J. 26 (2007) 4831–4871.

[33] H. Wang, Y. Zhang, E. Yehuda-Shnaidman, et al., Liver X receptor is a transcriptional repressor of the uncoupling protein 1 gene and the brown fat phenotype, Mol. Cell Biol. 28 (2008) 2187–2200.

[34] P.J. Malloy, B.J. Feldman, Cell-autonomous regulation of brown fat identity gene UCP1 by unliganded vitamin D receptor, Mol. Endocrinol. 27 (2013) 1632–1642.

[35] P. Yubero, O. Viñas, R. Iglesias, et al., Identification of tissue-specific protein binding domains in the 5′-proximal regulatory region of the rat mitochondrial brown fat uncoupling protein gene, Biochem. Biophys. Res. Commun. 204 (1994) 867–873.

[36] X. Kong, A. Banks, T. Liu, et al., IRF4 is a key thermogenic transcriptional partner of PGC-1α, Cell 158 (2014) 69–83.

[37] P. Puigserver, Z. Wu, C.W. Park, et al., A cold-inducible coactivator of nuclear receptors linked to adaptive thermogenesis, Cell 92 (1998) 829–839.

[38] P. Puigserver, B.M. Spiegelman, Peroxisome proliferator-activated receptor-gamma coactivator 1 alpha (PGC-1 alpha): transcriptional coactivator and metabolic regulator, Endocr. Rev. 24 (2003) 78–90.

[39] Z. Wu, P. Puigserver, U. Andersson, et al., Mechanisms controlling mitochondrial biogenesis and respiration through the thermogenic coactivator PGC-1, Cell 98 (1999) 115–139.

[40] M. Uldry, W. Yang, J. St-Pierre, et al., Complementary action of the PGC-1 coactivators in mitochondrial biogenesis and brown fat differentiation, Cell Metab. 3 (2006) 333–341.

[41] C. Tiraby, G. Tavernier, C. Lefort, et al., Acquisition of brown fat cell features by human white adipocytes, J. Biol. Chem. 278 (2003) 33370–33376.

Please cite this article in press as: F. Villarroya, et al., Transcriptional regulation of the uncoupling protein-1 gene, Biochimie (2016), http://dx.doi.org/10.1016/j.biochi.2016.09.017

ARTICLE IN PRESS

F. Villarroya et al. / Biochimie xxx (2016) 1–7

[42] J. Lin, H. Wu, P.T. Tarr, et al., Transcriptional co-activator PGC-1 alpha drives the formation of slow-twitch muscle fibres, Nature 418 (2002) 797–801.

[43] J.C. Yoon, P. Puigserver, G. Chen, et al., Control of hepatic gluconeogenesis through the transcriptional coactivator PGC-1, Nature 413 (2001) 131–139.

[44] P. Seale, S. Kajimura, W. Yang, et al., Transcriptional control of brown fat determination by PRDM16, Cell. Metab. 6 (2007) 38–54.

[45] S. Kajimura, P. Seale, K. Kubota, et al., Initiation of myoblast to brown fat switch by a PRDM16-C/EBP-β transcriptional complex, Nature 460 (2009) 1154–1158.

[46] A. Sambeat, O. Gulyaeva, J. Dempersmier, et al., LSD1 interacts with Zfp516 to promote UCP1 transcription and brown fat program, Cell Rep. 15 (11) (2016 Jun 14) 2536–2549.

[47] J. Chi, P. Cohen, The multifaceted roles of PRDM16: adipose biology and beyond, Trends Endocrinol. Metab. 27 (1) (2016) 11–23.

[48] M. Giralt, F. Villarroya, White, brown, beige/brite: different adipose cells for different functions? Endocrinology 154 (2013) 2992–3000.

[49] S. Rajakumari, J. Wu, J. Ishibashi, et al., EBF2 determines and maintains brown adipocyte identity, Cell Metab. 17 (2013) 562–574.

[50] H. Ohno, K. Shinoda, B.M. Spiegelman, et al., PPARγ agonists induce a white-to-brown fat conversion through stabilization of PRDM16 protein, Cell Metab. 15 (2012) 395–404.

[51] L. Qiang, L. Wang, N. Kon, et al., Brown remodeling of white adipose tissue by SirT1-dependent deacetylation of Pparγ, Cell 150 (2012) 620–632.

[52] Y.H. Tseng, E. Kokkotou, T.J. Schulz, et al., New role of bone morphogenetic protein 7 in brown adipogenesis and energy expenditure, Nature 454 (2008) 1000–1004.

[53] M. Trajkovski, K. Ahmed, C.C. Esau, et al., MyomiR-133 regulates brown fat differentiation through Prdm16, Nat. Cell Biol. 14 (2012) 1330–1335.

[54] M. Shao, J. Ishibashi, C.M. Kusminski, et al., Zfp423 maintains white adipocyte identity through suppression of the beige cell thermogenic gene program, Cell Metab 23 (2016) 1167–1184.

[55] J. Houstek, J. Kopecký, Z. Rychter, et al., Uncoupling protein in embryonic brown adipose tissue—existence of nonthermogenic and thermogenic mitochondria, Biochim. Biophys. Acta 935 (1988) 19–25.

[56] M. Giralt, I. Martin, R. Iglesias, et al., Ontogeny and perinatal modulation of gene expression in rat brown adipose tissue. Unaltered iodothyronine 5'-deiodinase activity is necessary for the response to environmental temperature at birth, Eur. J. Biochem. 193 (1990) 297–302.

[57] M.E. Lean, W.P. James, G. Jennings, et al., Brown adipose tissue uncoupling protein content in human infants, children and adults, Clin. Sci. 71 (1986) 291–297.

[58] A. Shore, A. Karamitri, P. Kemp, et al., Role of Ucp1 enhancer methylation and chromatin remodelling in the control of Ucp1 expression in murine adipose tissue, Diabetologia 53 (2010) 1164–1173.

[59] Y. Hiraoka, T. Matsuoka, M. Ohno, et al., Critical roles of nardilysin in the maintenance of body temperature homeostasis, Nat. Commun. 5 (2014) 3224.

[60] L. Zha, F. Li, R. Wu, et al., The histone demethylase UTX promotes brown adipocyte thermogenic program via coordinated regulation of H3K27 demethylation and acetylation, J. Biol. Chem. 290 (2015) 25151–25214.

[61] Y. Abe, R. Rozqie, Y. Matsumura, et al., JMJD1A is a signal-sensing scaffold that regulates acute chromatin dynamics via SWI/SNF association for thermogenesis, Nat. Commun. 6 (2015) 7052.

[62] F. Li, R. Wu, X. Cui, et al., Histone deacetylase 1 (HDAC1) negatively regulates thermogenic program in brown adipocytes via coordinated regulation of histone H3 lysine 27 (H3K27) deacetylation and methylation, J. Biol. Chem. 291 (2016) 4523–4559.

[63] M.J. Harms, H.W. Lim, Y. Ho, et al., PRDM16 binds MED1 and controls chromatin architecture to determine a brown fat transcriptional program, Genes Dev. 29 (2015) 298–307.

[64] S. Iida, W. Chen, T. Nakadai, et al., PRDM16 enhances nuclear receptor-dependent transcription of the brown fat-specific Ucp1 gene through interactions with Mediator subunit MED1, Genes Dev. 29 (2015) 308–329.

[65] A.M. Cassard-Doulcier, F. Bouillaud, M. Chagnon, et al., The Bcl I polymorphism of the human uncoupling protein (ucp) gene is due to a point mutation in the 5'-flanking region, Int. J. Obes. Relat. Metab. Disord. 20 (1996) 278–279.

[66] H. Esterbauer, H. Oberkofler, Y.M. Liu, et al., Uncoupling protein-1 mRNA expression in obese human subjects: the role of sequence variations at the uncoupling protein-1 gene locus, J. Lipid. Res. 39 (1998) 834–844.

[67] G. Rose, P. Crocco, P. D'Aquila, et al., Two variants located in the upstream enhancer region of human UCP1 gene affect gene expression and are correlated with human longevity, Exp. Gerontol. 46 (2011) 897–904.

[68] M. Del Mar Gonzalez-Barroso, D. Ricquier, A.M. Cassard-Doulcier, The human uncoupling protein-1 gene (UCP1): present status and perspectives in obesity research, Obes. Rev. 1 (2000) 61–72.

[69] J.J. Jia, Y.B. Tian, Z.H. Cao, et al., The polymorphisms of UCP1 genes associated with fat metabolism, obesity and diabetes, Mol. Biol. Rep. 37 (2010) 1513–1535.

[70] C.F. Nicoletti, A.P. de Oliveira, M.J. Brochado, et al., UCP1 -3826 A>G polymorphism affects weight, fat mass, and risk of type 2 diabetes mellitus in grade III obese patients, Nutrition 32 (2016) 83–91.

[71] R. Bracale, G. Labruna, C. Finelli, et al., The absence of polymorphisms in

[72] L.A. Brondani, T.S. Assmann, B.M. de Souza, et al., Meta-analysis reveals the association of common variants in the uncoupling protein (UCP) 1-3 genes with body mass index variability, PLoS One 9 (2014) e96411.

[73] B.M. de Souza, L.A. Brondani, A.P. Bouças, et al., Associations between UCP1 -3826A/G, UCP2 -866G/A, Ala55Val and Ins/Del, and UCP3 -55C/T polymorphisms and susceptibility to type 2 diabetes mellitus: case-control study and meta-analysis, PLoS One 8 (2013) e54259.

[74] K. Clement, J. Ruiz, A.M. Cassard-Doulcier, et al., Additive effect of A → G (-3826) variant of the uncoupling protein gene and the Trp64Arg mutation of the beta 3-adrenergic receptor gene on weight gain in morbid obesity, Int. J. Obes. Relat. Metab. Disord. 20 (1996) 1062–1066.

[75] R. Valve, S. Heikkinen, A. Rissanen, et al., Synergistic effect of polymorphisms in uncoupling protein 1 and beta3-adrenergic receptor genes on basal metabolic rate in obese Finns, Diabetologia 41 (1998) 357–361.

[76] M. Fogelholm, R. Valve, K. Kukkonen-Harjula, et al., Additive effects of the mutations in the beta3-adrenergic receptor and uncoupling protein-1 genes on weight loss and weight maintenance in finnish women, J. Clin. Endocrinol. Metab. 83 (1998) 4246–4250.

[77] A. Kogure, T. Yoshida, N. Sakane, et al., Synergic effect of polymorphisms in uncoupling protein 1 and beta3-adrenergic receptor genes on weight loss in obese Japanese, Diabetologia 41 (1998) 1399.

[78] K. Tsunekawa, Y. Yanagawa, T. Aoki, et al., Association between accumulation of visceral fat and the combination of β3 adrenergic receptor Trp64Arg, β2 adrenergic receptor Arg16Gly and uncoupling protein 1 -3826A>G polymorphisms detected by Smart Amplification Process 2, Endocr. J. 58 (2011) 1079–1165.

[79] Y. Chen, X. Wang, Z. Shen, et al., Effect of the beta-3 adrenergic receptor Trp64Arg and uncoupling protein 1-3826 A>G genotypes on lipid and apolipoprotein levels in overweight/obese and non-obese Chinese subjects, Lipids Health Dis. 14 (2015) 34.

[80] T. Hayakawa, Y. Nagai, M. Taniguchi, et al., Phenotypic characterization of the beta3-adrenergic receptor mutation and the uncoupling protein 1 polymorphism in Japanese men, Metabolism 48 (1999) 636–640.

[81] T. Yoneshiro, T. Ogawa, N. Okamoto, et al., Impact of UCP1 and β3AR gene polymorphisms on age-related changes in brown adipose tissue and adiposity in humans, Int. J. Obes. 37 (2013) 993–1001.

[82] S. Rousset, M. del Mar Gonzalez-Barroso, C. Gelly, et al., A new polymorphic site located in the human UCP1 gene controls the in vitro binding of CREB-like factor, Int. J. Obes. Relat. Metab. Disord. 26 (2002) 735–738.

[83] K.S. Kim, D.Y. Cho, Y.J. Kim, et al., The finding of new genetic polymorphism of UCP-1 A-1766G and its effects on body fat accumulation, Biochim. Biophys. Acta 1741 (2005) 149–155.

[84] H.D. Shin, K.S. Kim, M.H. Cha, et al., The effects of UCP-1 polymorphisms on obesity phenotypes among Korean female subjects, Biochem. Biophys. Res. Commun. 335 (2005) 624–630.

[85] J.S. El-Sayed Moustafa, P. Froguel, From obesity genetics to the future of personalized obesity therapy, Nat. Rev. Endocrinol. 9 (2013) 402–413.

[86] S. Hägg, A. Ganna, S.W. Van Der Laan, et al., Gene-based meta-analysis of genome-wide association studies implicates new loci involved in obesity, Hum. Mol. Genet. 24 (2015) 6849–6860.

[87] K.J. Gaulton, T. Ferreira, Y. Lee, et al., Genetic fine mapping and genomic annotation defines causal mechanisms at type 2 diabetes susceptibility loci, Nat. Genet. 47 (2015) 1415–1425.

[88] B.B. Lowell, V. S-Susulic, A. Hamann, et al., Development of obesity in transgenic mice after genetic ablation of brown adipose tissue, Nature 366 (1993) 740–742.

[89] C. Guerra, P. Navarro, A.M. Valverde, et al., Brown adipose tissue-specific insulin receptor knockout shows diabetic phenotype without insulin resistance, J. Clin. Investig. 108 (2001) 1205–1213.

[90] J. Krishnan, C. Danzer, T. Simka, et al., Dietary obesity-associated Hif1α activation in adipocytes restricts fatty acid oxidation and energy expenditure via suppression of the Sirt2-NAD+ system, Genes Dev. 26 (2012) 259–270.

[91] K. Moulin, N. Truel, M. André, et al., Emergence during development of the white-adipocyte cell phenotype is independent of the brown-adipocyte cell phenotype, Biochem. J. 356 (2001) 659–664.

[92] M. Rosenwald, A. Perdikari, T. Rülicke, et al., Bi-directional interconversion of brite and white adipocytes, Nat. Cell Biol. 15 (2013) 659–667.

[93] R. Xue, M.D. Lynes, J.M. Dreyfuss, et al., Clonal analyses and gene profiling identify genetic biomarkers of the thermogenic potential of human brown and white preadipocytes, Nat. Med. 21 (2015) 760–768.

[94] V. Jimenez, S. Muñoz, E. Casana, et al., In vivo adeno-associated viral vector-mediated genetic engineering of white and brown adipose tissue in adult mice, Diabetes 62 (2013) 4012–4022.

[95] J.I. Odegaard, M.W. Lee, Y. Sogawa, et al., Perinatal licensing of thermogenesis by IL-33 and ST2, Cell 166 (2016) 841–854.

[96] N. Dai, L. Zhao, D. Wrighting, IGF2BP2/IMP2-Deficient mice resist obesity through enhanced translation of Ucp1 mRNA and Other mRNAs encoding mitochondrial proteins, Cell Metab. 21 (2015) 609–621.

[97] J. Nedergaard, B. Cannon, UCP1 mRNA does not produce heat, Biochim. Biophys. Acta 1831 (2013) 943–949.


Please cite this article in press as: F. Villarroya, et al., Transcriptional regulation of the uncoupling protein-1 gene, Biochimie (2016), http://dx.doi.org/10.1016/j.biochi.2016.09.017
