
TIMI 1372 No. of Pages 11 ARTICLE IN PRESS

Trends in Microbiology

Opinion

A Structural and Mathematical Modeling Analysis of the Likelihood of Antibody-Dependent Enhancement in Influenza

Boopathy Ramakrishnan,$^{1}$ Karthik Viswanathan,$^{1}$ Kannan Tharakaraman,$^{2}$ Vlado Dančík,$^{3}$ Rahul Raman,$^{2}$ Gregory J. Babcock,$^{1}$ Zachary Shriver,$^{1}$ and Ram Sasisekharan$^{2,*}$

Broadly neutralizing monoclonal antibodies (bNAbs) for viral infections, such as HIV, respiratory syncytial virus (RSV), and influenza, are increasingly entering clinical development. For influenza, most neutralizing antibodies target influenza virus hemagglutinin. These bNAbs represent an emerging, promising modality for treatment and prophylaxis of influenza due to their multiple mechanisms of antiviral action and generally safe profile. Preclinical work in other viral diseases, such as dengue, has demonstrated the potential for antibody-based therapies to enhance viral uptake, leading to enhanced viremia and worsening of disease. This phenomenon is referred to as antibody-dependent enhancement (ADE). In the context of influenza, ADE has been used to explain several preclinical and clinical phenomena. Using structural and viral kinetics modeling, we assess the role of ADE in the treatment of influenza with a bNAb.

Antibodies as Antivirals and the Development of Antibody-Dependent Enhancement

Recent developments in antibody discovery and protein engineering have led to the identification of highly potent, broadly neutralizing antibodies to a number of infectious agents. Several of these, including monoclonal antibodies against influenza and HIV, have entered clinical development and are in phase I or phase II (e.g., NCT02603952, NCT02293863, NCT02468115, NCT02568215, and NCT02588586; [https://clinicaltrials.gov/](https://clinicaltrials.gov/)). Recent antiviral data reported for these antibodies has generated much excitement, for example, data on 3BNC117, targeting the CD4-binding site of gp120, suggests that the antibody has higher efficacy in humans than was necessarily predicted from in vitro or in vivo studies because of effector functions, including the ability to clear infected cells [1,2]. Given the intriguing, albeit early, data on the antiviral activity of bNAbs, there is the potential for these agents to become important complements to existing, small-molecule antivirals.

For influenza, virus-specific antibodies play a pivotal role in preventing and controlling viral infection. Several studies have examined the dynamics of the humoral immune response upon

Trends

The discovery and characterization of broadly neutralizing antibodies to RNA viruses, including influenza and HIV, has led to clinical trials of new immunotherapeutics that hold significant promise.

Several of these, such as 3BNC117 (HIV), MHAA4549A, and VIS410 (influenza), have demonstrated early clinical success through provoking a significant antiviral response.

The possibility of antibody-dependent enhancement, known for other viral diseases, where subneutralizing levels of antibody can exacerbate disease, is an area of active concern for regulatory approval of at least some antiviral immunotherapies.

Recent data on the mode of binding and the mechanisms of action of these broadly neutralizing antibodies shed light on the relative possibility of antibody-dependent enhancement, especially for influenza.

$^{1}$Visterra, Inc. One Kendall Square, Suite B3301, Cambridge, MA 02139, USA  
$^{2}$Department of Biological Engineering, Massachusetts Institute of Technology, 77 Massachusetts Avenue, Cambridge, MA 01890, USA  

Trends in Microbiology, Month Year, Vol. xx, No. yy http://dx.doi.org/10.1016/j.tim.2016.09.003  
© 2016 Published by Elsevier Ltd.

infection or vaccination using genetic and/or structural tools [3–9], largely focusing on antibodies to influenza hemagglutinin (HA), a surface protein, which is generally associated with a protective response [10]. Notably, although antibodies to the other predominant surface protein, neuraminidase, are also produced, these antibodies, in isolation, do not prevent viral infection but may participate in effector-mediated neutralization. These studies have identified three general properties of the virus-neutralizing humoral response to hemagglutinin: (i) the antibody response within an individual is dictated by the history of previous exposures and/or vaccinations (i.e., original antigenic sin); (ii) the majority of the antibody response is focused to the head of the influenza HA protein; and (iii) these antibodies tend to be strain-specific due to the variability in amino acid sequence of the HA head, leading to the potential for seasonal reinfection due to alterations in virus structure through mutation and genetic reassortment.

HA is synthesized as a single polypeptide (HA0) which folds into a prefusion conformation and exists as a homotrimer. Maturation of the HA trimer occurs upon its cleavage into HA1 and HA2 subunits by host proteases. HA1 and HA2 subunits do not separate but instead remain as a stable complex with a disulfide bond linking the two subunits. The virus receptor-binding site and the host membrane fusion peptide are located in the HA1 and HA2 subunits, respectively. As such, HA1 plays an important role in virus-host attachment, while HA2 plays an important role during virus-host membrane fusion in the endosome. In the prefusion conformation, most of the HA2 subunit is buried by the HA1 subunit. The head domain of the HA molecule, which is comprised of a large portion of HA1, is often the target of neutralizing antibodies. These head-directed antibodies neutralize virus by blocking attachment to the host cell. By contrast, antibodies to the stem region of the HA molecule, primarily located within the HA2 subunit, are less frequently produced and generally act to prevent viral fusion through preventing the necessary conformational changes required for the virus and host membrane to fuse in the endosome [11,12]. In addition to direct neutralization, stem-binding antibodies play an important role in clearing the infected host cells by recruiting effector molecules, such as complement or innate immune cells [13,14]. Despite regions of the stem being highly conserved among various subtypes of influenza A, neutralizing antibodies targeting this region represent a minor fraction of the total humoral immune response. Furthermore, administration of novel HA stem immunogens clearly demonstrates that a humoral response to this region is protective [15,16]. In fact, a recent study confirmed that even non-neutralizing antibodies, induced by immunization, can be protective in animal models [17].

Recently, several broadly neutralizing antibodies against the stem of influenza HA have been reported, including against group 1 of influenza A [18], group 2 of influenza A [19], and against both group 1 and group 2 [20–22]. These broadly neutralizing antibodies, either individually or in combination, have been able to treat influenza infection and/or provide prophylactic protection in various animal models. To date, the family of stem-directed broadly neutralizing antibodies has been found to employ divergent germlines, including the VH~1-69~ germline as well as others, and targeting different, but overlapping epitopes (Figure 1). Early discovery of C179 [23] was followed over a decade later by the identification of other stem-binding antibodies, including F10 [24], CR6261/CR8020 [18,19], CR9114 [21], FI6 [20], 39.29 [25], 12D1 [26], 6F12 [27], and VIS410 [22]. Notably, these antibodies have differentiated properties, including breadth of coverage and the ability to elicit effector functions through binding Fcγ receptors [17,28].

There has been significant conversation within scientific and regulatory bodies on the potential risks associated with the use of antibodies in the treatment of influenza; discussions at the Pan American Health Organization 2014 meeting on monoclonal antibodies highlighted the controversy, with a special session on this topic being convened at the annual meeting of the International Society for Influenza and Other Respiratory Virus Diseases (2015, Austin, Texas). While antibodies against influenza have demonstrated the ability to mitigate lethal influenza

2 Trends in Microbiology, Month Year, Vol. xx, No. yy

TIMI 1372 No. of Pages 11 ARTICLE IN PRESS

Trends in Microbiology

CR6261    39.29    FI6    F10    CR8020

![Image Description](image.png)

Figure 1. Surface Representation of HA Homotrimer Molecule. Each monomer is shown in green, cyan, or light green, illustrating the neutralizing epitopes (shown in red) of the corresponding stem-binding broadly neutralizing antibodies. The epitopes were identified using the PISA server based on antibody cocystal structures with the HA molecule.

infection in animals, the clinical use of monoclonal antibodies for the prevention and/or treatment of acute influenza has raised the question of whether binding by a therapeutic antibody could potentially exacerbate disease either by enhancing virus uptake and propagation or by promoting fusion in the endosome [29]. In the former scenario, engagement of Fcγ receptors at the cell surface by antibody-bound virus would lead to uptake of the virus; however, in the event that the virus is not neutralized (either because of low affinity binding or through targeting a non-neutralizing epitope), antibody binding would result in virus escape and hence, enhanced infection. Alternatively, enhancement could occur after virus binding to the cognate cellular receptor (sialic acid in the case of influenza) and internalization by stem-binding antibodies that enhance fusion of the virus with the endosome, and result in enhanced infection.

This so-called antibody-dependent enhancement (ADE) of viral infection has been invoked for many viruses, including dengue [30,31], respiratory syncytial virus (RSV) [32,33], influenza [34,35] and others [36–38], to describe multiple clinical phenomena. For example, in the case of dengue, ADE is invoked to explain the fact that previously exposed individuals who are infected with a heterologous serotype are at increased risk for severe disease. At a mechanistic level, enhanced disease in the presence of non-neutralizing antibodies has been explained by antibody binding promoting enhanced virus output, resulting from the attachment of immune complexes to Fcγl and lla receptors, uptake of such complexes, and resulting infection [39,40]. In the case of dengue, animal models are able to replicate this phenomenon [41,42]. Conversely, in the case of RSV, immune enhancement was observed upon vaccination with formalin-inactivated virus which resulted in enhanced disease during breakthrough infections. In contrast to dengue, the target cells for RSV are pulmonary epithelial cells and not Fcγ-bearing cells. In this case, it is thought that non-neutralizing antibodies to the F (and potentially G) protein, elicited through vaccination, resulted in stimulation of RSV-primed Th2 polarized cells, immune complex formation, and increased cell/tissue damage [43]. This article discusses the link between ADE and influenza and offers perspectives on the likelihood of ADE with different anti-influenza antibodies using a mathematical modeling approach.

### ADE and Influenza

ADE associated with the influenza A virus was first described by Ochiai and colleagues [44], where they characterized the antibody-mediated growth of influenza A virus in macrophages using rabbit serum isolated from animals that had been exposed to influenza virus. Later studies with serum from animals infected with various influenza A viruses showed that the subtype

TIMI 1372 No. of Pages 11 ARTICLE IN PRESS

Trends in Microbiology

cross-reactive, non-neutralizing antibodies to both HA and neuraminidase molecules contributed to ADE in vitro [45]. This analysis was further elaborated in pigs, where enhanced disease was found to be caused by various influenza vaccination protocols (i.e., vaccine-associated enhanced respiratory disease, VAERD) [34]. More specifically, this study demonstrated that when pigs were vaccinated with an inactivated H1N2 virus and then challenged with 2009 H1N1, the pathological findings of the resulting infection exhibited enhanced pulmonary lesions. Additional analysis indicated that the increased pathology was the result of VAERD. VAERD in humans has been suspected [46], where the seasonal 2008–09 trivalent inactivated influenza vaccination was found to potentially be associated with increased risk of illness associated with the pandemic 2009 H1N1 virus. Notably, the observations of disease enhancement were in response to immunization or polyclonal sera.

Given these observations, both within influenza and with other viruses, could a monoclonal antibody-based solution for the treatment of influenza elicit a VAERD-like response through promotion of enhanced disease? To address this question, one has to consider the different potential mechanisms that have been described to cause enhancement. One potential mechanism, as in dengue, would be that antibodies targeting influenza HA could potentially mediate enhanced uptake of virus into Fcγ-expressing cells and, if non-neutralizing, could lead to escape of the virus from the endosome. Endosomal escape would result in either productive infection in cells that do not normally sustain influenza infection and/or promote additional infection in susceptible cell populations. Alternatively, antibodies targeting certain epitopes on the HA stem could be directly fusogenic, by stabilizing the presentation of fusion peptides and facilitating the low pH conformational change in HA that results in fusion of the viral and host cell membrane. Finally, a third possibility, as has been invoked for RSV, the presence of non-neutralizing antibodies could form immune complexes that lead to polarization of the immune response to a proinflammatory, pathogenic response, resulting in enhanced disease.

**HA2-Binding Antibodies, Depending on Epitope, Can Enhance Fusion**

The broadly neutralizing anti-influenza HA antibodies in clinical development target the stem region and inhibit virus-host endosomal membrane fusion. This class of antibodies does not inhibit virus-host receptor attachment. Since such antibodies target the fusogenic potential of the virus, it is important to assess the ability of stem-binding antibodies to directly enhance infection.

A previous study molecularly characterized the antibodies associated with VAERD in pigs in an effort to address the underlying mechanisms of VAERD [35]. While the sera from H1N2-vaccinated pigs could enhance pH1N1 infection of MDCK cells, the sera did not inhibit agglutination of red blood cells (RBCs) by H1N1 virus. This result indicates that the sera had little or no cross-reactive binding antibodies that blocked HA-glycan receptor interactions. Through additional studies they found this serum to be capable of enhancing H1N1 infection in MDCK cells. Fractionation of the serum antibodies identified that a polyclonal pool of antibodies (pAbs) bound epitopes mapping to amino acids 32 to 77 of HA2 as responsible for enhancing virus infection. Given the lack of Fcγ receptors on MDCK cells and the mapping of activity to HA2-directed pAb, the ability of the pAb to enhance RBC hemolysis and the enhancement of infection was attributed to an Fcγ receptor-independent pathway, where the polyclonal response directly promotes virus membrane fusion activity that, in turn, enhances replication and ultimately increased lung pathology.

Structure-based analysis of this phenomenon can enable us to further elucidate the mechanism by which enhancement may be occurring. As a first step, mapping the recognized epitope on HA2 is essential to describe its function. This is particularly important since HA2 exists in two different conformational forms—the prefusion conformation on the extracellular virus and the postfusion conformation inside the host cell endosome. Structure-based analysis indicates that

4 Trends in Microbiology, Month Year, Vol. xx, No. yy

residues 58–76, within the 32–77 epitope, are completely buried by the HA1 subunit in the prefusion conformation and exposed only in the postfusion conformation, suggesting interaction of the polyclonal response to the postfusion conformation of HA2 (Figure 2). Indeed, the recombinant HA2 subunit, such as the one used in this study to fractionate the enhancing polyclonal fraction [35], is known to exist only in the postfusion conformation when it is expressed as either membrane-bound or in a soluble form [47].

In the endosome, when HA undergoes conformational change, the first step of this process is the separation of the HA1–HA2 interface. The disulfide bond anchor keeps the two subunits attached, but the interface separation allows for the dramatic conformational change within the HA2 subunit. Because it is membrane bound, the distal fusion peptide is initially prevented from reaching and interacting with the host membrane to initiate membrane fusion (Figure 3). Cryoelectron tomography of influenza at neutral and acidic pH confirms this analysis [48]. At neutral pH, HA is found in an ordered array at the surface of the virus. Conversely, at acidic pH, the density associated with HA is fused and individual trimers cannot be resolved. To initiate a successful membrane fusion, then, HA molecules at this point must act cooperatively with their fusion peptides to re-orient themselves towards the host membrane, enabling membrane fusion (Figure 3). At the same time, the low pH form of HA is more susceptible to proteolysis, resulting in cleavage and release of HA1, thereby enhancing the ability of HA2 to cooperatively undergo membrane fusion. In this model, profusogenic antibodies, if present, would bind and crosslink HA2 molecules, enabling them to order and orient towards the host membrane and enhance viral fusion efficiency.

In direct contrast to the enhancing antibodies described above, neutralizing, stem-binding anti-influenza A antibodies bind to a distinct set of epitopes on the prefusion conformation of HA with

(A)                                                                                               (B)
![Image Description](#)

HA2 in prefusion conformation                                                                 HA2 in postfusion conformation

Trends in Microbiology

Figure 2. Epitopes for Antibodies to HA2 in Prefusion and Postfusion Conformation. Epitope mapping by phage display had shown that the anti-HA2 antibodies bind to residues 32–76 of the HA2 molecule [35]. These residues are shown in red on one molecule of the homotrimer in the (A) prefusion and (B) postfusion conformations of the HA2 crystal structure. The fusion loop residues are shown in black on the prefusion form of HA. In the postfusion state, the fusion peptide is not observed in the crystal structure and is therefore shown as a schematic (black line). In the prefusion conformation, most of the HA2 subunit residues are covered by the HA1 subunit; for clarity, one HA2 subunit is shown in the insert. More specifically, the HA2 subunit residues 58–76 are covered by the HA1 subunit (shown in orange); therefore, these residues are not available for antibody binding. Conversely, in the postfusion conformation, in addition to the fusion peptide, the stem residues 32 to 76 of HA2 subunit are found on the surface and are available for antibody binding.

TIMI 1372 No. of Pages 11 ARTICLE IN PRESS

Trends in Microbiology

![](image.png)

Figure 3. Influenza Fusion with the Host Endosomal Membrane. On the virus surface HA molecules in their prefusion conformation are found as an ordered array of molecules, oriented to bind to the host cell surface receptor molecules. In the prefusion conformation (shown in state A), the HA2 subunit exists in a metastable conformation, anchored by the receptor-bound HA1 subunit, and is ready to undergo conformational changes (this is referred to as the spring-loaded state). Upon virus internalization into the endosome of the host cell, HA undergoes a significant conformational change where the HA1 subunit first separates from the HA2 subunit (shown in state B) allowing the fusion peptide to be deployed towards the host endosomal membrane to initiate viral-host membrane fusion (shown in state C). To achieve successful host infection, both the prefusion and postfusion conformations are essential; for example, lack of a protease cleavage within HA0 inhibiting formation of HA1 and HA2 prevents viral fusion. Interestingly, when HA1 and HA2 subunit proteins are expressed separately, only the HA1 molecule can be folded into its prefusion conformation; the HA2 molecule, expressed either with the viral transmembrane domain or only as a soluble domain (residues 376–481 HA0 numbering or 1–331 HA2 numbering), folds only into its postfusion conformation.

their binding centered on the hydrophobic cleft found adjacent to the exposed A-helix of HA2 [49]. Antibody binding prevents viral infection of host cells by inhibiting the conformational change of HA in the endosome required for fusion of viral and host membranes. Although these antibodies do not inhibit virus entry, their binding prevents HA from undergoing the conformational change in the endosome. Taken together, the fact that neutralizing antibodies bind to a region of HA2 exposed in the prefusion conformation that is distinct from the epitope known to be fusogenic indicates that these antibodies are highly unlikely to elicit ADE by promoting fusogenic activity.

### Fcγ-Mediated Uptake of Influenza Virus

As an alternative to direct stabilization of the fusogenic state of HA, a monoclonal antibody could elicit enhanced viremia through immune complex formation and uptake through Fcγ receptors, such as is observed for dengue virus. Of note, influenza virus primarily targets sialic acid receptors present on epithelial cells of the respiratory tract, rather than Fcγ-bearing cells of the adaptive and innate immune system [50]. Several cell-culture-based studies have shown that innate immune cells, such as macrophages and dendritic cells, are generally not supportive of productive influenza infection for most seasonal and low-pathogenic strains but may support infection of some highly pathogenic strains [50]. Furthermore, quantitative estimates indicate that such events are unlikely during the natural course of influenza infection. Quantification of receptor presentation indicates that there are approximately \(4 \times 10^8\) target cells for influenza virus [51], each containing \(10^4-10^5\) sialic acid receptors [52]. The engagement of virus HA with cell-surface sialic acid is polyvalent with an apparent Kd of 0.4–0.5 nM [53]. This is in comparison to monovalent binding of antibody (IgG\(_{1,2,\text{and } 4}\)) to FcγRI, which binds with a maximal dissociation constant of 15 nM [54], approximately 30-fold weaker, and where lung monocytes (total number of \(2.3 \times 10^7\) cells [55]) have approximately \(1.3 \times 10^4\) FcγRI receptors on their surface

6 Trends in Microbiology, Month Year, Vol. xx, No. yy

TIMI 1372 No. of Pages 11 ARTICLE IN PRESS

Trends in Microbiology

[56]. Therefore, based on absolute number of receptors (~100-fold lower for Fcγ receptors compared to sialic acid receptors) and their relative affinity/avidity, influenza binding and endosomal uptake is expected to largely occur as a function of HA binding to its cognate receptor form of sialic acid (if present) as opposed to engagement through Fcγ receptors.

A further distinction can be made based on the particular epitope bound by the antibody. Most antibodies generated from a humoral response to vaccination or infection are directed to the HA head, which serve to neutralize virus by preventing attachment through the predominant sialic acid pathway. Binding of such antibodies could facilitate virus entry through the FcγR pathway; in this case, such antibodies do not have any mechanism to stop viral infection in the endosome [13]. Such a phenomenon would be largely unapparent with a neutralizing, head-directed antibody but would become more apparent in the event that the antibody is non-neutralizing. For example, HA1-directed antibodies that do not target the major antigenic sites surrounding the receptor binding site but rather the ‘side’ of the head are broadly binding but non-neutralizing [57].

By contrast, stem-directed antibodies that bind to the prefusion form and prevent HA conformational changes would not influence cell attachment through the ‘normal’ sialic acid pathway; hence internalization through the FcγR binding would be predicted to be minimal. In essence, this pathway would be outcompeted by the more prevalent, higher-avidity interaction between the receptor-binding site on HA1 and human sialic acid receptors. Then, in the event that such a stem-directed antibody did not neutralize the virus (i.e., did not prevent virus–cell fusion), active viral replication would occur to the same extent as if there were no antibody present.

# Mathematical Model to Assess the Impact of ADE on Viral Replication Kinetics

The above observations suggest that, from a structural and biochemical binding perspective, it is unlikely that ADE occurs for antibodies that target the prefusion conformation of HA stem. However, to quantitatively assess ADE and its effect on viral replication with these antibodies, we adapted the recently developed mathematical models for viral kinetics in influenza infection [58–61]. The basic kinetic parameters of influenza infection have been measured or approximated previously (Table 1). We extended the target cell-limited differential equation model by Baccam *et al.* [58] by explicitly modelling the three different, mutually exclusive, modes of infection: infection by free virus (*l*₁), infection by antibody-bound virus via SA receptor (*l*₂), and infection by antibody-bound virus via Fcγ receptor (*l*₃) (Box 1). The infections by the three mechanisms are expressed by the second-order terms, β₁TV, β₂TU, and β₃TU, where βᵢ is the rate of infection, T is the number of target cells, V is the number of viral particles not affected by antibodies, and U is the number of virus particles inhibited by the antibody. The infected cells then become either virus-producing cells (P) at an intrinsic rate *k*ᵢ (independent of antibody binding) or are cleared at rate *c*ᵢ, *i* = 1, 2, 3. The virus-producing infected cells produce new viral particles at rate *p* and are cleared at rate δ. The free virus is bound by antibodies at rate α or is cleared at rate c. The antibody-bound virus is cleared at rate *c*₀. The depletion of free and antibody-bound virus due to infection of target cells is explicitly modelled by terms β₁′TV, β₂′TU, and β₃′TU[61]. When modelling head-binding antibody, we assume 50% efficiency of infection (β₂ = 0.5 × β₁) and no clearance of infected cells (*c*₁ = *c*₂ = *c*₃ = 0). When modelling stem-binding antibody, we assume 100% efficiency of infection (β₂ = β₁) and 50% clearance of infected cells (*c*₂ = *k*₂, *c*₃ = *k*₃). When modeling ADE, we assume the same infection rate (β₃ = β₁). In contrast to Li and Handel [62], we do not explicitly model the immune response and the amount of antibodies; such effects are indirectly captured by the relative values of α and β₁, β₂, β₃. Further, we do not include the contribution of antibody-dependent cellular cytotoxicity (ADCC) or complement-dependent cytotoxicity (CDC) in mediating clearance of infected cells. Figure 4 shows the results from the model when antibody is administered 1 day post infection (1 dpi). The results indicated

TIMI 1372 No. of Pages 11 ARTICLE IN PRESS

Trends in Microbiology

Table 1. Numerical Values of the Model Parameters$^a$

| Parameter | Description | No antibody | Head-binding antibody | Stem-binding antibody |
|-----------|-------------|--------------|------------------------|-----------------------|
| $\beta_1$ | Rate of SA-receptor infection by free virus | $3.2 \times 10^{-5}$ | $3.2 \times 10^{-5}$ | $3.2 \times 10^{-5}$ |
| $\beta_2$ | Rate of SA-receptor infection by antibody-bound virus | — | $1.6 \times 10^{-5}$ | $3.2 \times 10^{-5}$ |
| $\beta_3$ | Rate of Fc$\gamma$-receptor infection by antibody-bound virus | — | $3.2 \times 10^{-5}$ | $3.2 \times 10^{-5}$ |
| $\beta'_1$ | Rate of free-virus consumption by binding to target cells | $8 \times 10^{-5}$ | $8 \times 10^{-5}$ | $8 \times 10^{5}$ |
| $\beta'_2$ | Rate of antibody-bound virus consumption by binding to target cells via SA receptor | — | $4 \times 10^{-5}$ | $8 \times 10^{-5}$ |
| $\beta'_3$ | Rate of antibody-bound virus consumption by binding to target cells via Fc$\gamma$ receptor | — | $8 \times 10^{-5}$ | $8 \times 10^{-5}$ |
| $k_1$ | Rate of transition from free-virus infected to virus-producing cells | $4.0$ | $4.0$ | $4.0$ |
| $k_2$ | Rate of transition from cells infected by antibody-bound virus via SA receptor to virus-producing cells | — | $4.0$ | $4.0$ |
| $k_3$ | Rate of transition from cells infected by antibody-bound virus via Fc$\gamma$ receptor to virus-producing cells | — | $4.0$ | $4.0$ |
| $c_1$ | Clearance rate of cells infected by free virus | $0$ | $0$ | $0$ |
| $c_2$ | Clearance rate of cells infected by antibody-bound virus via SA receptor | — | $0$ | $4.0$ |
| $c_3$ | Clearance rate of cells infected by antibody-bound virus via Fc$\gamma$ receptor | — | $0$ | $4.0$ |
| $\delta$ | Clearance of virus-producing cells | $3.45$ | $3.45$ | $3.45$ |
| $p$ | Rate of new virus production | $2.04 \times 10^{2}$ | $2.04 \times 10^{2}$ | $2.04 \times 10^{2}$ |
| $\alpha$ | Rate of antibody binding | — | $1.28 \times 10^{5}$ | $1.28 \times 10^{5}$ |
| $c$ | Clearance rate of free virus | $3.72 \times 10^{4}$ | $3.72 \times 10^{4}$ | $3.72 \times 10^{4}$ |
| $c_0$ | Clearance rate of antibody-bound virus | — | $3.72 \times 10^{4}$ | $3.72 \times 10^{4}$ |

$^a$The influenza virus infection model values of $\beta_1$ and $k_1$ were taken from the cell-limited model with delay by Baccam et al. [58], the virus consumption rate is set at $\beta'_1 = 2.5 \times \beta_1$, and values for $p$, $\delta$, and $c$ are optimized to match the Baccam et al. model. To model the role of antibody, the antibody concentration was assumed to be such that it results in a 10:1 ratio of antibody-bound vs. free virus, which resulted in $\alpha = 1.28 \times 10^{5}$, values for $\beta_1$, $\beta'_1$, $k_1$, $\delta$, $p$, and $c$ the same as in the no-antibody model, $\beta'_2 = 2.5 \times \beta_2$, $\beta'_3 = 2.5 \times \beta_3$, $k_2 = k_3 = k_1$, and $c_0 = c$. For head-binding antibody, $\beta_2 = 0.5 \times \beta_1$ and $c_1 = c_2 = c_3 = 0)$. For stem-binding antibody, $\beta_2 = \beta_1$ and $c_2 = k_2$, $c_3 = k_3$. When modeling ADE, $\beta_3 = \beta_1$ was assumed.

that, in a target cell-limited model, stem-binding neutralizing antibodies pose a lower risk of increased viral load due to ADE as compared to head-binding neutralizing antibodies (1.6-fold vs. 2.2-fold increase in area under the curve (AUC)). Taken together, the mathematical model quantitatively supports the structural and biochemical rationale that stem-binding antibodies, which are able to bind the prefusion form of HA and prevent its conformational change, are unlikely to induce significant ADE through Fc$\gamma$-mediated uptake.

## Antibody Binding Leading to Enhanced Pathology

Finally, antibody binding and immune complex formation is known in certain limited cases to enhance pathology. For example, this has been hypothesized to be the case in the context of formalin-inactivated RSV, where vaccination led to enhanced pulmonary disease upon subsequent infection, likely through production of non-neutralizing antibodies that promoted a nonproductive, damaging immune response [63,64]. In the case of RSV, a Th1-driven response tends to promote cell-mediated immunity important for protection against the virus. Conversely, a Th2 response can be associated with eosinophilia, goblet cell hyperplasia, mucus

TIMI 1372 No. of Pages 11 ARTICLE IN PRESS

Trends in Microbiology

Box 1. Influenza Virus Model in the Presence of Antibody

T is the number of uninfected target cells, $I_1$, $I_2$, and $I_3$ are the number of infected (but not virus-producing) cells, P is the number of cells actively producing new viruses, V is the free-virus load, and U is the antibody-bound virus load. We distinguish three modes of infection, $I_1$ – target cells infected by free virus via sialic-acid receptor (not bound by antibody), $I_2$ – target cells infected by antibody-bound virus via sialic-acid receptor, and $I_3$ – target cells infected by antibody-bound virus via Fcγ receptor (ADE).

\[
\frac{dT}{dt} = -\beta_1 TV - \beta_2 TU - \beta_3 TU
\]

\[
\frac{dI_1}{dt} = \beta_1 TV - k_1 I_1 - c_1 I_1
\]

\[
\frac{dI_2}{dt} = \beta_2 TU - k_2 I_2 - c_2 I_2
\]

\[
\frac{dI_3}{dt} = \beta_3 TU - k_3 I_3 - c_3 I_3
\]

\[
\frac{dP}{dt} = k_1 I_1 + k_2 I_2 + k_3 I_3 - \delta P
\]

\[
\frac{dV}{dt} = pP - \beta'_1 TV - \alpha V - cV
\]

\[
\frac{dU}{dt} = \alpha V - \beta'_2 TU - \beta'_3 TU - c_0 U
\]

![Graph](attachment:graph.png)

Key:
- Target cell-limited model from Baccam et al.
- Model with no antibodies
- Head binding antibody, no ADE
- Stem binding antibody, no ADE

| Without ADE | With ADE |
|-------------|----------|
| Head antibody | 88233    | 189902   |
| Stem antibody | 53186    | 85383    |
| No antibody   | 608916   |          |

Trends in Microbiology

Figure 4. Comparison of Virus Infection Models with Head- and Stem-Binding Antibodies. (Gray) Target cell-limited model with delayed virus production (Baccam et al.) [58]. (Black) Model with no antibodies ($\alpha = \beta_2 = \beta_3 = 0$) and explicitly modeled virus consumption (Cao et al.) [61]; parameters $V_0$, $p$, and $c$ optimized to match Baccam model. (Red) Head-binding antibody added at 1 day postinfection (dpi) ($\beta_2 = 0.5 \times \beta_1$, $\beta_3 = \beta_1$, $c_2 = c_3 = 0$). (Pink) Head-binding antibody added at 1 dpi without accounting for ADE ($\beta_3 = 0$). (Blue) Stem-binding antibody added at 1 dpi ($\beta_2 = \beta_3 = \beta_1$, $c_2 = k_2$, $c_3 = k_3$). (Cyan) Stem-binding antibody added at 1 dpi without accounting for ADE ($\beta_3 = 0$).

overproduction, and airway hypersensitivity. In contrast to natural infection that is resolved, the formalin-inactivated RSV vaccine induced a Th2-biased response, likely a consequence of inefficient induction of interferon-γ (IFN-γ)-secreting NK cells and CD8⁺ T lymphocytes [65,66]. Indeed, Th2 polarization of the immune response in RSV infection is a hallmark of more severe disease [43], and several viral gene products, including soluble G protein, promote such a response [67]. This is in contrast to influenza infection, where influenza immunity is largely driven by a Th1 response [68], with the virus suppressing efficient recruitment of Th2 cells [69]. Thus, while such an immune-modulating mechanism appears to be the case for RSV, and potentially other viruses, no information suggests that this is true for influenza. In addition, examination of the natural humoral response in animals administered a stem-directed antibody indicates that there is no detectable difference between infected animals which clear the virus naturally compared to those which are infected and then treated with a stem-directed antibody [70]. Taken together, these data argue that immune complex formation is likely not a substantial mechanism for disease enhancement for influenza.

### Concluding Remarks

In summary, the available evidence, and the known or hypothesized mechanisms that have been advanced by which antibodies (primarily non-neutralizing) can enhance viral replication and/or disease, do not *a priori* support the presence of a substantial risk of ADE for neutralizing, stem-directed antibodies. To verify these findings requires, of course, clinical testing (see Outstanding Questions). However, by bringing together multiple observations that have been made for stem-directed antibodies, largely discovered within the past 8 years, with what we know about influenza infection, propagation, and pathogenesis, it is unlikely that such antibodies would enhance infection by mechanisms that have been previously posited.

### Acknowledgments

This work was supported in part NIH Merit Award (R37 GM057073-13) and NIH grant (RO1 AI111395-03).

### References

1. Lu, C.L. *et al.* (2016) Enhanced clearance of HIV-1-infected cells by broadly neutralizing antibodies against HIV-1 in vivo. *Science* 352, 1001–1004
2. Schoofs, T. *et al.* (2016) HIV-1 therapy with monoclonal antibody 3BNC117 elicits host immune responses against HIV-1. *Science* 352, 997–1001
3. Laserson, U. *et al.* (2014) High-resolution antibody dynamics of vaccine-induced immune responses. *Proc. Natl. Acad. Sci. U.S.A.* 111, 4928–4933
4. Lingwood, D. *et al.* (2012) Structural and genetic basis for development of broadly neutralizing influenza antibodies. *Nature* 489, 566–570
5. Pappas, L. *et al.* (2014) Rapid development of broadly influenza neutralizing antibodies through redundant mutations. *Nature* 516, 418–422
6. Sobolev, O. *et al.* (2016) Adjuvanted influenza-H1N1 vaccination reveals lymphoid signatures of age-dependent early responses and of clinical adverse events. *Nat. Immunol.* 17, 204–213
7. Kreijtz, J.H. *et al.* (2011) Immune responses to influenza virus infection. *Virus Res.* 162, 19–30
8. Andrews, S.F. *et al.* (2015) Immune history profoundly affects broadly protective B cell responses to influenza. *Science Transl. Med.* 7, 316ra192
9. Schmidt, A.G. *et al.* (2015) Immunogenic stimulus for germline precursors of antibodies that engage the influenza hemagglutinin receptor-binding site. *Cell Rep.* 13, 2842–2850
10. Wang, T.T. *et al.* (2010) Vaccination with a synthetic peptide from the influenza virus hemagglutinin provides protection against distinct viral subtypes. *Proc. Natl. Acad. Sci. U.S.A.* 107, 18979–18984
11. Brandenburg, B. *et al.* (2013) Mechanisms of hemagglutinin targeted influenza virus neutralization. *PLoS ONE* 8, e80034
12. Otterstrom, J.J. *et al.* (2014) Relating influenza virus membrane fusion kinetics to stoichiometry of neutralizing antibodies at the single-particle level. *Proc. Natl. Acad. Sci. U.S.A.* 111, E5143–E5148
13. DiLillo, D.J. *et al.* (2016) Broadly neutralizing anti-influenza antibodies require Fc receptor engagement for in vivo protection. *J. Clin. Invest.* 126, 605–610
14. DiLillo, D.J. *et al.* (2014) Broadly neutralizing hemagglutinin stalk-specific antibodies require FcgammaR interactions for protection against influenza virus in vivo. *Nat. Med.* 20, 143–151
15. Impagliazzo, A. *et al.* (2015) A stable trimeric influenza hemagglutinin stem as a broadly protective immunogen. *Science* 349, 1301–1306
16. Yassine, H.M. *et al.* (2015) Hemagglutinin-stem nanoparticles generate heterosubtypic influenza protection. *Nat. Med.* 21, 1065–1070
17. Henry Dunand, C.J. *et al.* (2016) Both neutralizing and non-neutralizing human H7N9 influenza vaccine-induced monoclonal antibodies confer protection. *Cell Host Microbe* 19, 800–813
18. Ekiert, D.C. *et al.* (2009) Antibody recognition of a highly conserved influenza virus epitope. *Science* 324, 246–251
19. Ekiert, D.C. *et al.* (2011) A highly conserved neutralizing epitope on group 2 influenza A viruses. *Science* 333, 843–850
20. Corti, D. *et al.* (2011) A neutralizing antibody selected from plasma cells that binds to group 1 and group 2 influenza A hemagglutinins. *Science* 333, 850–856
21. Dreyfus, C. *et al.* (2012) Highly conserved protective epitopes on influenza B viruses. *Science* 337, 1343–1348
22. Tharakaraman, K. *et al.* (2015) A broadly neutralizing human monoclonal antibody is effective against H7N9. *Proc. Natl. Acad. Sci. U.S.A.* 112, 10890–10895

---

**Outstanding Questions**

In preclinical studies, there is evidence that broadly neutralizing antibodies can act synergistically with small-molecule antivirals. Will this increase the safety margin associated with the use of monoclonals in treating viral diseases?

Ultimately, issues of clinical safety and efficacy will require testing in influenza patients, many of whom are (partially) immunocompromised. What role will immune status of patients play on both the safety and the efficacy of broadly neutralizing antibodies?

In influenza, are non-neutralizing antibodies that target the head of HA one of the key determinants of vaccine-associated enhanced respiratory disease?

Intrinsically, antibody-dependent enhancement is characterized by a dysregulated inflammatory response where the antibody's engagement with Fcγ receptors suppresses innate immunity. What is the role of intrinsic antibody-dependent enhancement in influenza pathology?

How does the format of influenza vaccine influence the nature of antibody response? How precisely are postfusion conformation-specific antibodies produced?

What kind of sialylated glycans (linkage of sialic acid, number of monosaccharides), the receptor for influenza hemagglutinin, exist on Fcγ receptor-bearing cells?

23. Okuno, Y. *et al.* (1993) A common neutralizing epitope conserved between the hemagglutinins of influenza A virus H1 and H2 strains. *J. Virol.* 67, 2552–2558

24. Sui, J. *et al.* (2009) Structural and functional bases for broad-spectrum neutralization of avian and human influenza A viruses. *Nat. Struct. Mol. Biol.* 16, 265–273

25. Nakamura, G. *et al.* (2013) An in vivo human-plasmablast enrichment technique allows rapid identification of therapeutic influenza A antibodies. *Cell Host Microbe* 14, 93–103

26. Wang, T.T. *et al.* (2010) Broadly protective monoclonal antibodies against H3 influenza viruses following sequential immunization with different hemagglutinins. *PLoS pathog.* 6, e1000796

27. Tan, G.S. *et al.* (2012) A pan-H1 anti-hemagglutinin monoclonal antibody with potent broad-spectrum efficacy in vivo. *J. Virol.* 86, 6179–6188

28. Tharakaraman, K. *et al.* (2014) Broadly neutralizing influenza hemagglutinin stem-specific antibody CR8020 targets residues that are prone to escape due to host selection pressure. *Cell Host Microbe* 15, 644–651

29. Jang, Y.H. and Seong, B.L. (2014) Options and obstacles for designing a universal influenza vaccine. *Viruses* 6, 3159–3180

30. Whitehead, S.S. *et al.* (2007) Prospects for a dengue virus vaccine. *Nat. Rev. Microbiol.* 5, 518–528

31. Halstead, S.B. (1979) In vivo enhancement of dengue virus infection in rhesus monkeys by passively transferred antibody. *J. Infect. Dis.* 140, 527–533

32. Ponnuraj, E.M. *et al.* (2003) Antibody-dependent enhancement, a possible mechanism in augmented pulmonary disease of respiratory syncytial virus in the Bonnet monkey model. *J. Infect. Dis.* 187, 1257–1263

33. Osiowy, C. *et al.* (1994) Antibody-dependent enhancement of respiratory syncytial virus infection by sera from young infants. *Clin. Diagnost. Lab. Immunol.* 1, 670–677

34. Gauger, P.C. *et al.* (2011) Enhanced pneumonia and disease in pigs vaccinated with an inactivated human-like (delta-cluster) H1N2 vaccine and challenged with pandemic 2009 H1N1 influenza virus. *Vaccine* 29, 2712–2719

35. Khurana, S. *et al.* (2013) Vaccine-induced anti-HA2 antibodies promote virus fusion and enhance influenza virus respiratory disease. *Science Transl. Med.* 5, 200ra114

36. Gould, E.A. and Buckley, A. (1989) Antibody-dependent enhancement of yellow fever and Japanese encephalitis virus neurovirulence. *J. Gen Virol.* 70, 1605–1608

37. Gould, E.A. *et al.* (1987) Immune enhancement of yellow fever virus neurovirulence for mice: studies of mechanisms involved. *J. Gen. Virol.* 68, 3105–3112

38. Phillipotts, R.J. *et al.* (1985) Antibody-dependent enhancement of tick-borne encephalitis virus infectivity. *J. Gen. Virol.* 66, 1831–1837

39. Ubol, S. and Halstead, S.B. (2010) How innate immune mechanisms contribute to antibody-enhanced viral infections. *Clin. Vaccine Immunol.* 17, 1829–1835

40. Halstead, S.B. (1982) Immune enhancement of viral infection. *Prog. Allergy* 31, 301–364

41. de Alwis, R. *et al.* (2014) Dengue viruses are enhanced by distinct populations of serotype cross-reactive antibodies in human immune sera. *PLoS pathog.* 10, e1004386

42. Zellweger, R.M. and Shresta, S. (2014) Mouse models to study dengue virus immunology and pathogenesis. *Front. Immunol.* 5, 151

43. Caballero, M.T. *et al.* (2015) TLR4 genotype and environmental LPS mediate RSV bronchiolitis through Th2 polarization. *J. Clin. Invest.* 125, 571–582

44. Ochiai, H. *et al.* (1988) Antibody-mediated growth of influenza A NWS virus in macrophagelike cell line P388D1. *J. Virol.* 62, 20–26

45. Ochiai, H. *et al.* (1992) Infection enhancement of influenza A NWS virus in primary murine macrophages by anti-hemagglutinin monoclonal antibody. *J. Med. Virol.* 36, 217–221

46. Skowronski, D.M. *et al.* (2010) Association between the 2008-09 seasonal influenza vaccine and pandemic H1N1 illness during Spring-Summer 2009: four observational studies from Canada. *PLoS Med.* 7, e1000258

47. Swalley, S.E. *et al.* (2004) Full-length influenza hemagglutinin HA2 refolds into the trimeric low-pH-induced conformation. *Biochemistry* 43, 5902–5911

48. Fontana, J. *et al.* (2012) Structural changes in Influenza virus at low pH characterized by cryo-electron tomography. *J. Virol.* 86, 2919–2929

49. Wyrzucki, A. *et al.* (2014) Alternative recognition of the conserved stem epitope in influenza A virus hemagglutinin by a VH3-30-encoded heterosubtypic antibody. *J. Virol.* 88, 7083–7092

50. Short, K.R. *et al.* (2012) The fate of influenza A virus after infection of human macrophages and dendritic cells. *J. Gen. Virol.* 93, 2315–2325

51. Sidorenko, Y. and Reichl, U. (2004) Structured model of influenza virus replication in MDCK cells. *Biotechnol. Bioeng.* 88, 1–14

52. Nunes-Correia, I. *et al.* (1999) Interactions of influenza virus with cultured cells: detailed kinetic modeling of binding and endocytosis. *Biochemistry* 38, 1095–1101

53. Fei, Y. *et al.* (2015) Characterization of receptor binding profiles of influenza a viruses using an ellipsometry-based label-free glycan microarray assay platform. *Biomolecules* 5, 1480–1498

54. Bruhns, P. (2012) Properties of mouse and human IgG receptors and their contribution to disease models. *Blood* 119, 5640–5649

55. Crapo, J.D. *et al.* (1982) Cell number and cell characteristics of the normal human lung. *Am. Rev. Resp. Dis.* 126, 332–337

56. Antal-Szalmas, P. *et al.* (1997) Quantitation of surface CD14 on human monocytes and neutrophils. *J. Leukocyte Biol.* 61, 721–728

57. Thornburg, N.J. *et al.* (2016) H7N9 influenza virus neutralizing antibodies that possess few somatic mutations. *J. Clin. Invest.* 126, 1482–1494

58. Baccam, P. *et al.* (2006) Kinetics of influenza A virus infection in humans. *J. Virol.* 80, 7590–7599

59. Boianelli, A. *et al.* (2015) Modeling Influenza Virus Infection: A Roadmap for Influenza Research. *Viruses* 7, 5274–5304

60. Dobrovolny, H.M. *et al.* (2013) Assessing mathematical models of influenza infections using features of the immune response. *PLoS ONE* 8, e57088

61. Cao, P. *et al.* (2015) Innate immunity and the inter-exposure interval determine the dynamics of secondary influenza virus infection and explain observed viral hierarchies. *PLoS Comput. Biol.* 11, e1004334

62. Li, Y. and Handel, A. (2014) Modeling inoculum dose dependent patterns of acute virus infections. *J. Theor. Biol.* 347, 63–73

63. Collins, P.L. and Graham, B.S. (2008) Viral and host factors in human respiratory syncytial virus pathogenesis. *J. Virol.* 82, 2040–2055

64. Connors, M. *et al.* (1994) Enhanced pulmonary histopathology induced by respiratory syncytial virus (RSV) challenge of formalin-inactivated RSV-immunized BALB/c mice is abrogated by depletion of interleukin-4 (IL-4) and IL-10. *J. Virol.* 68, 5321–5325

65. Hussell, T. *et al.* (1997) CD8 + T cells control Th2-driven pathology during pulmonary respiratory syncytial virus infection. *Eur. J. Immunol.* 27, 3341–3349

66. Johnson, T.R. *et al.* (2004) Respiratory syncytial virus (RSV) G glycoprotein is not necessary for vaccine-enhanced disease induced by immunization with formalin-inactivated RSV. *J. Virol.* 78, 6024–6032

67. Boyoglu-Barnum, S. *et al.* (2014) Prophylaxis with a respiratory syncytial virus (RSV) anti-G protein monoclonal antibody shifts the adaptive immune response to RSV rA2-line19F infection from Th2 to Th1 in BALB/c mice. *J. Virol.* 88, 10569–10583

68. Deng, N. *et al.* (2014) Cytokine diversity in the Th1-dominated human anti-influenza response caused by variable cytokine expression by Th1 cells, and a minor population of uncommitted IL-2 + IFNgamma- Thpp cells. *PLoS ONE* 9, e95986

69. Wohlleben, G. *et al.* (2003) Influenza A virus infection inhibits the efficient recruitment of Th2 cells into the airways and the development of airway eosinophilia. *J. Immunol.* 170, 4601–4611

70. Baranovich, T. *et al.* (2016) The hemagglutinin stem-binding monoclonal antibody VIS410 controls influenza virus-induced acute respiratory distress syndrome. *Antimicrob. Agents Chemother.* 60, 2118–2131
