
Review
# Host–Pathogen Interactions in Measles Virus Replication and Anti-Viral Immunity

Yanliang Jiang, Yali Qin * and Mingzhou Chen *

State Key Laboratory of Virology and Modern Virology Research Center, College of Life Sciences, Wuhan University, Wuhan 430072, China; 2013202040037@whu.edu.cn

* Correspondence: yqin@whu.edu.cn (Y.Q.); chenmz@whu.edu.cn (M.C.)

Academic Editor: Richard Plemper

Received: 31 August 2016; Accepted: 7 November 2016; Published: 16 November 2016

Abstract: The measles virus (MeV) is a contagious pathogenic RNA virus of the family Paramyxoviridae, genus Morbillivirus, that can cause serious symptoms and even fetal complications. Here, we summarize current molecular advances in MeV research, and emphasize the connection between host cells and MeV replication. Although measles has reemerged recently, the potential for its eradication is promising with significant progress in our understanding of the molecular mechanisms of its replication and host-pathogen interactions.

Keywords: measles virus; paramyxoviruses; viral replication; host factors

---

## 1. Introduction

Measles, also known as rubeola or morbilli, is a contagious infection caused by measles virus (MeV), an RNA virus of the genus Morbillivirus within the family Paramyxoviridae [1]. Species of this genus also include the canine distemper virus in canines, rinderpest virus in cattle, and morbilliviruses which infect other animals [2]. Humans are the natural hosts of MeV, and no animal reservoirs are known to exist [3].

The virion of MeV is a pleomorphic or spherical particle with a diameter ranging from 120 to 1000 nm and has two major structural components: one is the helical ribonucleoprotein (RNP) core formed by the association of the nucleoprotein (N), phosphoprotein (P) and large protein (L) with the viral genome, the other is the cellular membrane-derived lipid envelope surrounding the RNP core [1,4,5]. The active RNP complex is responsible for initiating primary transcription after cell entry as well as counteracting the host interferon (IFN) signaling pathway [5–7]. The MeV RNP is bound by the matrix protein (M), and then covered by the lipid envelope containing two spike glycoproteins, F and H proteins, which are primarily responsible for membrane fusion and receptor attachment, respectively [8].

For measles, symptoms usually develop after an incubation period of 7–14 days and last 7–10 days. They typically include fever, cough, coryza, conjunctivitis, enanthema (Koplik spots) on the oral mucosa, and a maculopapular rash. Apart from the typical symptoms, a specific feature of measles is a long-lasting immunosuppression due to the loss of immune memory B and T cells [9]. As a consequence, patients can encounter complications, especially in the setting of malnutrition in developing countries, ranging from bacterial superinfections, pneumonia, and diarrhea to postinfectious encephalomyelitis (PIE), or a sub-acute sclerosing panencephalitis (SSPE), which can manifest even several years after recovery [10–12]. Vaccination is highly effective and has resulted in a huge decrease in measles-related mortality [4,12,13]. Measles, however, has not been eliminated and has even reemerged in developed countries owing to a low vaccine coverage rate; therefore, more efforts are needed for a worldwide elimination of MeV [14,15].

The aim of this review is to outline recent progress in the MeV research that is critical to understanding MeV replication and the virus–host biology. Here, we emphasize the findings related to the MeV genome, function of viral proteins, replication cycle, and the involvement of host cell factors that play key roles in the MeV replication cycle.

## 2. Genome and Lifecycle

### 2.1. Genome

Currently, 24 MeV genotypes compiled in eight clades (A–H) have been recognized by sequencing 450 nucleotides (nt) that code for the C-terminal 150 amino acids of the N gene [16,17]. However, cross-neutralization with strain-specific antisera revealed only one serotype [18]. Following the rule of six, the RNA genome of MeV is 15,894 bp in length and is tightly encapsidated by the helically arranged N protein to form a helical nucleocapsid (NC) containing N-RNA that is also observed in other Paramyxoviridae members [19,20]. The genome begins with a 52 nt non-coding region known as the leader and ends with a 37 nt non-coding region known as the trailer, both of which are essential for the transcription and replication of the genome [21]. The organization of the MeV genome is similar to that of most other members of the Paramyxoviridae: there are six genes coding for eight viral proteins arranged as 3′-N,P,V,C,M,F,H,L-5′, each flanked by gene-end and gene-start sequences [1,22].

The first gene codes for N protein. The conserved N-terminal N core (about 400 amino acids) constitutes the core region of N protein whereas the remaining C-terminal N tail (about 100 amino acids) is intrinsically disordered and interacts with the matrix protein and the C-terminal domain of phosphoprotein [19,23,24]. Moreover, the structural flexibility of the disordered N tail is important for interactions between the N tail and multiple cellular proteins, including the 70 KDa heat shock protein (Hsp72), eukaryotic translation initiation factor 3 (eIF3-p40) and interferon regulatory factor 3 (IRF-3) [25–27].

The second gene codes for three proteins—P, V, and C—via an RNA editing process and an alternative reading frame [28]. The P protein binds to newly synthesized N to form a soluble N<sup>0</sup>-P complex, thus preventing N from binding to cellular RNAs, and the N<sup>0</sup>-P complex is used as a substrate for the specific encapsidation of viral RNA [29,30]. In addition, the P protein tethers the polymerase onto and progresses along the N-RNA template by binding to the NC [31–34].

The main function of the V and C proteins is to suppress the host innate immune response by interfering with IFN signaling pathways [1]. These proteins also function as virulence factors in that they are indispensable for virus infection in vivo [35]. For the related Sendai virus, the C protein even enhances the release of the M protein in a manner dependent on the endosomal sorting complexes required for transport (ESCRT) pathway [36].

The third gene codes for the M protein, which is a hydrophobic protein. Although M is not a membrane protein, it associates with membranes, probably through its hydrophobic surface [37]. It also binds RNPs, associates with the cytoplasmic tails of F and H proteins and modulates cell fusion [24,38]. In addition, it acts as an inhibitor of viral polymerase activity, affecting both mRNA transcription and genome replication [24,39]. Thus, the M protein plays a crucial role in many stages of the viral lifecycle.

The fourth and fifth genes code for envelope-associated spike glycoproteins indicated in membrane fusion and receptor recognition, that are discussed below in the “Assembly and Egress” subsection later in this article. The last gene codes for the RNA-dependent RNA polymerase (RdRP), which is believed to possess all catalytic functions required for RNA synthesis, including ribonucleotide polymerization, capping and methylation, and polyadenylation [40,41]. The L, N, and P proteins associate with the viral RNA to form the active RNP complex that initiates primary transcription after cell entry [42,43].

2.2. Cell Entry

The initial binding of MeV to the cell surface is mediated by the tetrameric H protein via interaction with cell surface receptors, which triggers the conformational change of the trimeric F protein and then, the membrane fusion and the delivery of the viral RNP core into the cytoplasm [44,45]. Similar to the H proteins of the Morbillivirus, the H protein of MeV cannot bind sialic acid and lacks neuraminidase activity; thus, it is named H not HN [1,46,47]. The primary receptors for wild type MeV strains are CD150/SLAM and nectin-4/PVRL4 [48,49], and some laboratory-adapted and vaccine strains bind to CD46 as well. In addition, the F protein is critical for the fusion of infected cells with neighboring cells, which eventually results in multinucleated cell formation, termed “syncytia”, which is a hallmark of MeV and many other paramyxoviruses [8,50–52].

2.3. Transcription and Replication

MeV shares the gene order and transcription strategy that are fundamental characteristics of all other paramyxoviruses [22,53]. Following cell entry, the genomic RNPs are released into the cytosol and the encapsidated viral RNA serves as a template of the RdRP complex for both transcription and replication [5]. Transcription begins at the 3′ end of the genome and viral genes are transcribed in the 3′ to 5′ direction with a sequential “stop-start” mechanism. MeV shares the gene order and transcription strategy that are fundamental characteristics of all other paramyxoviruses [22,53]. Newly synthesized viral mRNAs are translated to viral proteins by using the host translation machinery. The negative-strand genome is also used to synthesize a positive-strand anti-genome, which is a complementary copy of the entire genome that produces more genomes via the same viral RNA polymerase. During replication, the newly synthesized genomic RNA is tightly wrapped with the N protein to provide a helical template for viral transcription and replication [32,54]. Although the mechanism of the switch from transcription to replication remains unclear, evidence suggests that the accumulation of N proteins is critical for it [42].

2.4. Assembly and Egress

The assembly of the M protein, the RNP complex, and the glycoproteins at selected sites on the plasma membranes of infected cells lead to the formation of fully infectious MeV particles, which is a result of coordinated interactions between viral components as well as between viral and cellular factors [45,55]. The C-terminal domain of the N protein has been proved to be essential for the interaction with the M protein by yeast two-hybrid binding assay and co-immunoprecipitation in mammalian cells [24], and mutations or deletions in the M gene block the transport of RNP complex to the plasma membrane during infection, further supporting the crucial role of M protein in incorporating the RNP complex into virions [56,57]. In addition, the M protein can assemble to form higher structure, and binds cellular membranes and cellular factors as well [45,55,58,59]. Consequently, it is generally considered the key driver of paramyxovirus particle assembly and budding. For many paramyxoviruses, including MeV, M protein expressed in the absence of other viral proteins is sufficient to form virus-like particles [37,56]. While many viruses take advantage of the cellular ESCRT machinery during egress, the budding of MeV particles is driven primarily by the M protein, which has been demonstrated to be ESCRT-independent [60,61].

Taken together, the aforementioned findings suggest that, for MeV, the F and H proteins assemble intracellularly prior to receptor binding and are co-transported to the plasma membrane. The M protein associates with the RNP complex in the cytoplasm and then carries it to the plasma membrane, where the assembly with F and H proteins occurs. Fully infectious virions are then released from the host cell in an ESCRT-independent manner.

Viruses 2016, 8, 308

3. Interaction between MeV and Cellular Factors

3.1. Host Factors Involved in MeV Replication

3.1.1. Host Factors Involved in MeV Entry

    As an obligate intracellular parasite, MeV interacts with numerous cellular molecules to manipulate cellular processes and to subvert anti-viral responses for its replication. At the entry level, at least three cellular receptors have been identified for wild type strains and laboratory-adapted strains. The primary receptor for wild type strains is CD150/SLAM, the expression of which is limited to activated T and B cells, macrophages and dendritic cells [48,62]. CD150/SLAM supports the transport of the infection to lymphocytes and results in temporary loss of immunity to other pathogens, which accounts for MeV-induced immune suppression [63]. The epithelial cell receptor nectin-4/PVRL4 was identified later using a comparative microarray approach [49,64]. In addition to the two aforementioned receptors, the laboratory-adapted and vaccine strains use CD46, which is expressed on nearly all nucleated cells [65,66]. Consequently, using CD46 as an additional receptor results in a tropism alteration of MeV. Despite the lymphocyte and epithelia cell tropism of wild type MeV strains, MeV may also infect other types of cells. For example, PIE and SSPE are complications of MeV infection that result from the infection of neurons [67,68]. Thus far, the mechanism accounting for MeV neuronal infection and transport is unclear.

3.1.2. Host Factors Involved in MeV RNA Synthesis and Assembly

    Many putative host factors involved in MeV RNA synthesis have been identified via yeast two-hybrid, co-immunoprecipitation and some proteomic approaches. Some findings have suggested that the inducible Hsp72 can directly modulate RNA synthesis of several mammalian RNA and DNA viruses, including MeV [69,70]. During RNA synthesis, P protein binds to two conserved hydrophobic domains on the C-terminal N tail to tether the RdRP to the ribonucleocapsid. By competing with P protein for binding to these domains, Hsp72 would loosen the binding between RdRP and the ribonucleocapsid so that RdRP can move to the next N tail, which sustains the RdRP processivity, resulting in increased genome transcription, replication, and virulence [25,71]. In addition, another host factor, SHC binding and spindle associated 1 (SHCBP1), was found to interact with both the C and the P proteins, and they did not compete for the binding [72]. The results indicated that the C protein interacted with the RNP complex through SHCBP1 and thereby modulated viral RNA synthesis. Without the SHCBP1-binding site, however, the C protein retained the regulatory ability, suggesting that other cellular factors might have similar functions as SHCBP1. Besides just exploiting the regulation on itself, MeV inhibits the translation of cellular mRNA via the interaction between the p40 subunit of eIF3 and the N protein [26].

    Other host factors that interact with MeV proteins include several kinases, such as casein kinase II that phosphorylates P protein and some unidentified kinases that phosphorylate N and P proteins [73,74]. Phosphorylation of these two proteins has totally different functions. Phosphorylation of P protein at S86 and S151 downregulates viral transcriptional activity, whereas MeV RNA synthesis is increased upon the phosphorylation of serine residues 479 and 510 in N protein, as confirmed [75] in a mini-genome expression system [76,77].

    Many viruses take advantage of the cellular trafficking system during their replication. Cytoskeletal tubulin has been shown to be an essential component for the transcription and replication of Sendai virus and vesicular stomatitis virus [78,79]. In MeV, tubulin also acts as a positive factor that it may be a subunit of the RdRP complex, resulting in the subsequent RNA synthesis [80]. Moreover, the MeV RNP complex is transported in a microtubule-dependent manner associated with recycling endosomes containing Ras-related protein Rab-11A [81]. Together with microtubules, another cytoskeleton component, actin, is essential for the reproduction of MeV as well, especially for budding. Experimental data revealed that the accumulation of RNP and defects in the maturation and release of infectious

MeV particles were connected with the disruption of actin filaments and that actin filaments were packaged within the virions, suggesting a close association between actin filaments and MeV assembly and budding [75, 79, 82]. These host factors interacting with MeV during its lifecycle are summarized in Figure 1.

![Diagram](attachment:diagram.png)

Figure 1. Schematic of host factors involved in the measles virus (MeV) lifecycle. The attachment and entry of MeV is mediated by H and F proteins, associated with cellular receptors CD150, nectin-4 for wild-type strains and CD46 for attenuated strains, respectively. During viral RNA synthesis, heat shock protein 72 (Hsp72) interacts with the N protein to sustain the RdRP processivity. The C protein interacts with the RNP complex through SHC binding and spindle associated 1 (SHCBP1) and modulates viral RNA synthesis. The N protein can also bind to eukaryotic translation initiation factor 3 (eIF3-p40) to inhibit the translation of cellular mRNA. There are some kinases that phosphorylate N and P proteins, including the casein kinase II (CKII). As for assembly and budding, the RNP complex is transported to the plasma membrane driven by the M protein, and the process is dependent on actin filaments and microtubules associated with Ras-related protein Rab11A.

### 3.2. Host Factors Involved in Anti-MeV Innate Immune Responses

#### 3.2.1. Host Factors Involved in IFN Response

The host has innate immunity, in particular the IFN system, to sense and protect it from MeV infection. However, MeV has also evolved multifaceted strategies to antagonize the immune attack.

The innate immune responses are activated by the sensing of pathogen-associated molecular patterns (PAMPs) via pattern recognition receptors (PRRs), including retinoic-acid inducible gene (RIG)-I-like receptors (RLRs), Toll-like receptors (TLRs), and nucleotide-binding oligomerization domain (NOD)-like receptors (NLRs), that play an essential role in MeV detection [83, 84]. The host cell can recognize the H, N proteins, and RNA of MeV [7, 83, 85]. For example, the interaction of H protein with Toll-like receptor 2 (TLR2) triggers the production of interleukin 6 (IL-6) and the expression of SLAM/CD150 on the cell surface, resulting in immune activation and effective spreading of MeV [85]. Following cell entry, the viral RNA is released into the cytoplasm and recognized by two key members of the RLR family: RNA helicase-like RIG-I and melanoma differentiation-associated protein 5 (MDA5) [83, 84, 86].

MeV has evolved multifaceted strategies to counteract viral RNA sensing. Like V proteins of other related paramyxoviruses, the V protein of MeV interacts with MDA5 and RIG-I-like receptor 2 (LGP2) at the same time, and the latter has been reported as both a coactivator of MDA5 and a negative regulator of both RIG-I and MDA5 [87–89]. By interacting with phosphoprotein phosphatase 1 α and γ (PP1α/γ), V protein can also prevent PP1-mediated activation of MDA5 [90]. Moreover, P protein suppresses the TLR4 signaling via the activated transcription of a ubiquitin-modifying enzyme A20, which negatively regulates the nuclear factor-kappa B (NF-κB) [91,92].

Sensing of MeV infection activates signaling cascades resulting in the activation of the transcription factors NF-κB and IRF-3/7, leading to the production of type I IFN-α/β and proinflammatory cytokines [84,93,94]. To antagonize the induction of IFN, MeV has evolved multiple mechanisms. The V protein binds to IkB kinase α (IKKα) to downregulate the phosphorylation of IRF-7, and it also inhibits the transcriptional activities of IRF3 and IRF7 by interacting with them [95,96]. Moreover, V protein interacts with and suppresses NLR family member NLRP3 inflammasome-mediated IL-1β secretion [97]. The P, V, and C proteins can bind and inhibit NF-κB-dependent gene expression, especially the binding of the V protein with NF-κB subunits p65 [98]. Despite the P, V, and C proteins-induced suppression of NF-κB signaling, MeV-induced activation of NF-κB pathway, which was crucial to its replication, was reported to be regulated by sphingosine kinase 1 (SK1) [99]. The data revealed that overexpression of SK1 promoted the replication of MeV. However, the inhibition of SK1 suppressed both the MeV replication and the MeV-induced NF-κB signaling, suggesting the dual role of NF-κB and the pro-viral role of SK1 in MeV replication. The mechanism of how the NF-κB signaling promotes MV replication needs to be further explored, such as the exact stage of the MeV lifecycle where SK1 initiates the regulation. The C protein impairs the activation of the protein kinase regulated by RNA (PKR) via regulating the production of defective copyback double-stranded viral RNA [100]. In addition, C protein of wild-type MeV can also counteract the promoter of IFN-β in the nucleus without inhibiting the activation of IRF-3. However, the C protein of vaccine strains cannot localize into the nucleus, suggesting that altered C protein intracellular localization of vaccine strains contributes to its attenuation [101]. On the contrary, the N protein interacts and activates IRF-3, which stimulates IFN-β production [7]. However, a direct interaction between IRF-3 and N could not be confirmed experimentally by using purified recombinant proteins and yeast two hybrid assay: this finding led to the hypothesis of an indirect binding requiring a specific cellular context [27].

MeV is capable of interfering with both IFN synthesis and the signal transduction pathway mediated by the release of IFN, thus allowing the successful escape of MeV from the innate immune system. Secreted IFN-α/β activates Janus kinase/signal transducers and activators of transcription (JAK/STAT) signaling in surrounding noninfected cells via binding to the type-I IFN receptors (IFNAR), thus triggering the expression of numerous IFN-stimulated genes (ISGs), some of which encode anti-viral proteins [102,103]. To inhibit the establishment of an antiviral state in non-infected cells, MeV has evolved a variety of evasive adaptations to counteract the JAK/STAT signalling pathway. For example, N and V proteins of MeV cause the defective nuclear redistribution of activated STAT [104,105]. The N-terminal and C-terminal regions of V protein interact with JAK-1, STAT-1 and STAT-2, which independently impair the signal transduction [106]. P protein inhibits the phosphorylation and nuclear translocation of STAT-1 [107,108], while C protein prevents the dimerization of phosphorylated STAT-1 [109]. Of note, the IFN-stimulated gene ADAR1 inhibits the activation of PKR and IRF-3, suggesting its function as a suppressor of MeV-mediated IFN-β production, which ultimately promotes MeV infection [110,111]. Furthermore, the activation of PKR can also be suppressed by the C protein of MeV [112,113]. The interactions involved in MeV-stimulated IFN induction and signaling are summarized in Figure 2.

Viruses 2016, 8, 308

Figure 2. A detailed overview of host factors involved in MeV-stimulated interferon (IFN) induction and signaling. The C and V proteins inhibit the activation of the viral RNA sensors retinoic-acid inducible gene I (RIG-I), melanoma differentiation-associated protein 5 (MDA5) and RIG-I-like receptor 2 (LGP2) and the following IFN responses. Furthermore, V protein inhibits the activation and function of interferon regulatory transcription factor 3 (IRF-3) and IRF-7 via interaction with them. The P, V, and C proteins bind subunits of nuclear factor-kappa B (NF-κB) and repress the production of IFN-β and cytokine. In addition, the interaction between P protein and the negative regulator A20 also leads to the repression of NF-κB. In addition, sphingosine kinase 1 (SK1) interacts with TNF receptor-associated factor 2 (TRAF2) and enhances tumor necrosis factor (TNF)-induced activation of NF-κB signaling. In the nucleus, C protein represses IFN-β transcription without inhibiting the activation of IRF-3. On the contrary, N protein positively activates the IRF-3 and IFN-β production. The interferon inducible protein ADAR1 negatively regulates the activation of protein kinase regulated by RNA (PKR) and the IRF-3, suppressing MeV-mediated IFN-β production. To interfere with the IFN signaling, C protein blocks the dimerization of phosphorylated signal transducer and activator of transcription 1 (STAT-1). P and V protein inhibit the phosphorylation and nuclear translocation of STAT-1. In addition, N protein can prevent the migration of STAT-1 and STAT-2 into nucleus. The IFN signaling triggers the expression of many IFN-inducible proteins. The arrows indicate the activation of IFN responses while the T-ended arrows indicate the repression by MeV. JAK-1, Janus kinase 1; TYK-2, tyrosine kinase 2; ISG IFN-stimulated gene; IRF, interferon regulatory factor; IKK, IκB kinase; TLR, Toll-like receptor; ADAR1, adenosine deaminase acting on RNA 1; MAVS, mitochondrial antiviral-signaling protein; OAS, 2′-5′-oligoadenylate synthetase; IRAK, interleukin-1 receptor-associated kinase 1.

### 3.2.2. Host Factors Involved in Stress Granule Formation

In addition to the synthesis of IFN and the subsequent signaling cascades, another response of host cells to infection of several paramyxoviruses, including MeV, is the formation of stress granules (SGs), which are cytoplasmic aggregates containing various translationally stalled mRNAs, 40S ribosomes, and RNA-binding proteins [112,114,115]. In general, the activation of IFN system, especially the activation of PKR, results in the phosphorylation and inactivation eIF2α, leading to the formation of SGs [115]. By binding with the eIF3-p40, the N protein inhibits the translation of cellular mRNA, which

may consequently promote SG formation [26]. Experiments with cells stably deficient either in PKR or adenosine deaminase acting on RNA 1 (ADAR1) showed that the formation of SG was induced in a PKR-dependent manner by MeV infection; however, SG formation was suppressed by ADAR1 [112].

Taken together, the aforementioned findings suggest that ADAR1 acts as a suppressor of host innate immune responses, including IFN-β production and SG formation induced by MeV infection. Of particular note, SG cannot form in cells infected by wild-type MeV; however, a mutant lacking C protein expression is a robust inducer of SG formation, which is consistent with the suppression of the activation of PKR by the C protein [102]. Given that SG formation cannot be induced by wild-type MeV infection, MeV may benefit by interfering with SG formation, suggesting an antiviral effect of SGs during MeV infection.

### 3.2.3. Host Factors Involved in Autophagy

Autophagy is a highly conserved mechanism that mediates the dysfunctional cytoplasmic components for lysosomal degradation and recycling, whose hallmark is the formation of the double-membraned autophagosomes that subsequently fuse with lysosomes to form autolysosomes for degradation. Autophagy not only maintains cellular and tissue homeostasis but also regulates innate immune responses against intracellular virus invasion [116]. Consequently, viruses have developed various mechanisms to escape or even hijack the autophagy machinery for their own benefit. For MeV, the infection can induce successive proviral autophagy signaling via different pathways, which ultimately promotes the formation of infectious viral particles [117].

Both virulent and attenuated strains induce a late and sustained autophagy wave, which is initiated after viral replication and relies on the expression of C protein of MeV. In addition, the expression of C protein alone can also induce an autophagic signaling in an immunity-related GTPase M (IRGM)-dependent manner in HeLa cells [118]. The late autophagy signaling can be sustained overtime as a result of the formation of syncytia [117]. Moreover, the extensive syncytia formation mediated by the expression of both H and F proteins in cells expressing one of the cellular receptors is sufficient to induce autophagy. Therefore, the expression of C protein is not necessary to induce autophagy in syncytia [119].

As for attenuated strains being able to bind CD46, the infection can induce two waves of autophagy, including an early but transient autophagy wave via the engagement of CD46-Cyt-1 in a Golgi-associated PDZ and coiled-coil motif-containing protein (GOPC)-dependent pathway and the late autophagy wave [120]. Research findings have suggested that the involvement of CD46 in the autophagic digestion of MeV peptides and the subsequent presentation by MHC-II may explain the acquisition of protective immunity via vaccine strains of MeV infection [121].

Of note, MeV induces de novo formation of autophagosomes, and such autophagosomes mature into autolysosomes, whereas some paramyxoviruses make use of autophagy machinery to replicate but inhibit autophagosome maturation by blocking the fusion of autophagosomes with lysosomes [120,122].

Apart from bulk autophagy, the Edmonston-MeV strain also exploits the selective mitophagy in non-small cell lung cancer (NSCLC) cells, which enhances its replication as well [123]. The mitophagy occurs upon Edmonston-MeV infection via recognition of damaged mitochondria by the autophagic receptor SQSTM1/p62, the devouring of these mitochondria by autophagosomes, and the subsequent degradation by lysosomes [123]. By clearing damaged mitochondria before they release cytochrome c, mitophagy prevents the beclin 1-mediated activation of Bid or the degradation of active caspase-8 and then inhibits apoptosis, resulting in mitigated cell death and the consequent oncolysis in NSCLC induced by Edmonston-MeV [124]. More important than the inhibition of apoptosis, though, is the attenuation of the RIG-I/MDA5-mediated signaling pathway via the SQSTM1/p62-mediated degradation of mitochondrion-tethered mitochondrial antiviral signaling protein (MAVS), suggesting a novel mechanism of Edmonston-MeV to weaken the innate immune responses [123]. An overview of host factors involved in MeV-inducible stress granule formation and autophagy is depicted in Figure 3.

Viruses 2016, 8, 308

Figure 3. A diagram of host factors involved in MeV inducible stress granule formation and autophagy. The C protein suppresses the activation of PKR, so a mutant MeV lacking C protein expression rather than the wild-type MeV induces the stress granule (SG) formation. The N protein binds to the eIF3-p40 and then promotes SG formation. As a suppressor of PKR, ADAR1 suppresses the PKR-dependent SG formation. The infection of Edmonston-MeV strain initiates an early but transient autophagy wave via the engagement of CD46-Cyt-1 in a Golgi-associated PDZ and coiled-coil motif-containing protein (GOPC)-dependent pathway. Moreover, the expression of C protein can induce an autophagic signaling in an immunity-related GTPase M (IRGM)-dependent manner. The expression of H and F proteins in cells expressing one of the cellular receptors is also sufficient to induce autophagy. The Edmonston-MeV strain also exploits the selective mitophagy via recognition of damaged mitochondria by autophagic receptor SQSTM1, resulting in the degradation of mitochondrion-tethered mitochondrial antiviral-signaling protein (MAVS) and subsequent attenuation of RIG-I/MDA5-mediated responses. The arrows indicate the activation of innate immune responses while the T-ended arrows indicate the repression by MeV.

## 4. Conclusions

In this review, we provide a summary of current achievements in MeV research and review the experimental efforts aimed at elucidating the molecular mechanisms of MeV replication and host–pathogen interactions. We focus on recent insights related to the MeV genome, the function of viral proteins, the replication cycle, and the involvement of host cell factors during the MeV lifecycle, with an emphasis on cellular factors mediating MeV-stimulated innate immune responses that play key roles in the MeV replication cycle.

The elucidation of the mechanisms of MeV infection has provided valuable information on viral replication and countermeasures to mitigate cellular innate immune responses, which will ultimately provide new targets for antiviral therapy against MeV. However, many details of the mechanisms remain unclear; therefore, further investigation is needed to provide a clearer and more comprehensive understanding of the aforementioned aspects.

Acknowledgments: This work was supported by a grant from the China Natural Science Foundation (grant 81471939, 81271816 and 31630086).

Author Contributions: Yanliang Jiang drafted the manuscript. All authors corrected, edited and approved the text.

Conflicts of Interest: The authors declare no conflict of interest.

# References

1. Chesney, R.C.M.M. Measles Virus. In *Encyclopedia of Virology*, 3rd ed.; Academic Press: Cambridge, MA, USA, 2008; pp. 285–291.
2. King, A.M.Q.; Adams, M.J.; Carstens, E.B.; Lefkowitz, E.J. Genus Morbillivirus. In *Virus Taxonomy: Ninth Report of the International Committee on Taxonomy of Viruses*; Elsevier/Academic Press: London, UK, 2011; pp. 680–681.
3. Goodson, J.L.; Seward, J.F. Measles 50 Years after Use of Measles Vaccine. *Infect. Dis. Clin. N. Am.* 2015, 29, 725–743. [CrossRef] [PubMed]
4. Holzmann, H.; Hengel, H.; Tenbusch, M.; Doerr, H.W. Eradication of measles: Remaining challenges. *Med. Microbiol. Immunol.* 2016, 205, 201–208. [CrossRef] [PubMed]
5. Cox, R.; Plemper, R.K. The paramyxovirus polymerase complex as a target for next-generation anti-paramyxovirus therapeutics. *Front. Microbiol.* 2015, 6, 459. [CrossRef] [PubMed]
6. Galinski, M.S. Paramyxoviridae: Transcription and replication. *Adv. Virus Res.* 1991, 39, 129–162. [PubMed]
7. tenOever, B.R.; Servant, M.J.; Grandvaux, N.; Lin, R.; Hiscott, J. Recognition of the measles virus nucleocapsid as a mechanism of IRF-3 activation. *J. Virol.* 2002, 76, 3659–3669. [CrossRef] [PubMed]
8. Plattet, P.; Alves, L.; Herren, M.; Aguilar, H.C. Measles virus fusion orotein: Structure, function and inhibition. *Viruses* 2016, 8, 112. [CrossRef] [PubMed]
9. Griffin, D.E.; Lin, W.H.; Pan, C.H. Measles virus, immune control, and persistence. *FEMS Microbiol. Rev.* 2012, 36, 649–662. [CrossRef] [PubMed]
10. Naim, H.Y. Measles virus. *Hum. Vaccines Immunother.* 2015, 11, 21–26. [CrossRef] [PubMed]
11. Perry, R.T.; Halsey, N.A. The clinical significance of measles: A review. *J. Infect. Dis.* 2004, 189 (Suppl. S1), S4–S16. [CrossRef] [PubMed]
12. Simons, E.; Ferrari, M.; Fricks, J.; Wannemuehler, K.; Anand, A.; Burton, A.; Strebel, P. Assessment of the 2010 global measles mortality reduction goal: Results from a model of surveillance data. *Lancet* 2012, 379, 2173–2178. [CrossRef]
13. Van den Ent, M.M.; Brown, D.W.; Hoekstra, E.J.; Christie, A.; Cochi, S.L. Measles mortality reduction contributes substantially to reduction of all cause mortality among children less than five years of age, 1990–2008. *J. Infect. Dis.* 2011, 204 (Suppl. S1), S18–S23. [CrossRef] [PubMed]
14. Abad, C.L.; Safdar, N. The reemergence of measles. *Curr. Infect. Dis. Rep.* 2015, 17, 51. [CrossRef] [PubMed]
15. Caseris, M.; Burdet, C.; Lepeule, R.; Houhou, N.; Yeni, P.; Yazdanpanah, Y.; Joly, V. An update on measles. *Rev. Med. Interne* 2015, 36, 339–345. [CrossRef] [PubMed]
16. Rota, P.A.; Brown, K.; Mankertz, A.; Santibanez, S.; Shulga, S.; Muller, C.P.; Hubschen, J.M.; Siqueira, M.; Beirnes, J.; Ahmed, H.; et al. Global distribution of measles genotypes and measles molecular epidemiology. *J. Infect. Dis.* 2011, 204 (Suppl. S1), S514–S523. [CrossRef] [PubMed]
17. Rota, P.A.; Bellini, W.J. Update on the global distribution of genotypes of wild type measles viruses. *J. Infect. Dis.* 2003, 187 (Suppl. S1), S270–S276. [CrossRef] [PubMed]
18. Riddell, M.A.; Rota, J.S.; Rota, P.A. Review of the temporal and geographical distribution of measles virus genotypes in the prevaccine and postvaccine eras. *Virol. J.* 2005, 2, 87. [CrossRef] [PubMed]
19. Bourhis, J.M.; Receveur-Brechot, V.; Oglesbee, M.; Zhang, X.; Buccellato, M.; Darbon, H.; Canard, B.; Finet, S.; Longhi, S. The intrinsically disordered C-terminal domain of the measles virus nucleoprotein interacts with the C-terminal domain of the phosphoprotein via two distinct sites and remains predominantly unfolded. *Protein Sci.* 2005, 14, 1975–1992. [CrossRef] [PubMed]
20. Schoehn, G.; Mavakis, M.; Albertini, A.; Wade, R.; Hoenger, A.; Ruigrok, R.W. The 12 Å structure of trypsin-treated measles virus N-RNA. *J. Mol. Biol.* 2004, 339, 301–312. [CrossRef] [PubMed]
21. Horikami, S.M.; Moyer, S.A. Structure, transcription, and replication of measles virus. *Curr. Top. Microbiol. Immunol.* 1995, 191, 35–50. [PubMed]

22. Dutch, R.E. Paramyxoviruses. In *Encyclopedia of Virology*, 3rd ed.; Academic Press: Cambridge, MA, USA, 2008; pp. 52–57.

23. Blocquel, D.; Habchi, J.; Costanzo, S.; Doizy, A.; Oglesbee, M.; Longhi, S. Interaction between the C-terminal domains of measles virus nucleoprotein and phosphoprotein: A tight complex implying one binding site. *Protein Sci.* **2012**, *21*, 1577–1585. [CrossRef](https://doi.org/10.1002/pro.2046) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22492766)

24. Iwasaki, M.; Takeda, M.; Shirogane, Y.; Nakatsu, Y.; Nakamura, T.; Yanagi, Y. The matrix protein of measles virus regulates viral RNA synthesis and assembly by interacting with the nucleocapsid protein. *J. Virol.* **2009**, *83*, 10374–10383. [CrossRef](https://doi.org/10.1128/JVI.00788-09) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19605485)

25. Zhang, X.; Bourhis, J.M.; Longhi, S.; Carsillo, T.; Buccellato, M.; Morin, B.; Canard, B.; Oglesbee, M. Hsp72 recognizes a P binding motif in the measles virus N protein C-terminus. *Virology* **2005**, *337*, 162–174. [CrossRef](https://doi.org/10.1016/j.virol.2005.03.033) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15862365)

26. Sato, H.; Masuda, M.; Kanai, M.; Tsukiyama-Kohara, K.; Yoneda, M.; Kai, C. Measles virus N protein inhibits host translation by binding to eIF3-p40. *J. Virol.* **2007**, *81*, 11569–11576. [CrossRef](https://doi.org/10.1128/JVI.00985-07) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17709396)

27. Colombo, M.; Bourhis, J.M.; Chamontin, C.; Soriano, C.; Villet, S.; Costanzo, S.; Couturier, M.; Belle, V.; Fournel, A.; Darbon, H.; et al. The interaction between the measles virus nucleoprotein and the Interferon Regulator Factor 3 relies on a specific cellular environment. *Virol. J.* **2009**, *6*, 59. [CrossRef](https://doi.org/10.1186/1743-422X-6-59) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19473510)

28. Bellini, W.J.; Englund, G.; Rozenblatt, S.; Arnheiter, H.; Richardson, C.D. Measles virus P gene codes for two proteins. *J. Virol.* **1985**, *53*, 908–919. [PubMed](https://pubmed.ncbi.nlm.nih.gov/2982570)

29. Spehner, D.; Drillien, R.; Howley, P.M. The assembly of the measles virus nucleoprotein into nucleocapsid-like particles is modulated by the phosphoprotein. *Virology* **1997**, *232*, 260–268. [CrossRef](https://doi.org/10.1006/viro.1997.8560) [PubMed](https://pubmed.ncbi.nlm.nih.gov/9195755)

30. Guryanov, S.G.; Liljeroos, L.; Kasaragod, P.; Kajander, T.; Butcher, S.J. Crystal Structure of the Measles Virus Nucleoprotein Core in Complex with an N-terminal Region of Phosphoprotein. *J. Virol.* **2016**, *90*, 2849–2857. [CrossRef](https://doi.org/10.1128/JVI.02546-15) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26797954)

31. Krumm, S.A.; Takeda, M.; Plemper, R.K. The measles virus nucleocapsid protein tail domain is dispensable for viral polymerase recruitment and activity. *J. Biol. Chem.* **2013**, *288*, 29943–29953. [CrossRef](https://doi.org/10.1074/jbc.M113.486763) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23836922)

32. Brunel, J.; Chopy, D.; Dosnon, M.; Bloyet, L.M.; Devaux, P.; Urzua, E.; Cattaneo, R.; Longhi, S.; Gerlier, D. Sequence of events in measles virus replication: Role of phosphoprotein-nucleocapsid interactions. *J. Virol.* **2014**, *88*, 10851–10863. [CrossRef](https://doi.org/10.1128/JVI.00988-14) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24920723)

33. Bourhis, J.M.; Canard, B.; Longhi, S. Structural disorder within the replicative complex of measles virus: Functional implications. *Virology* **2006**, *344*, 94–110. [CrossRef](https://doi.org/10.1016/j.virol.2005.09.029) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16213651)

34. Zhang, S.; Chen, L.; Zhang, G.; Yan, Q.; Yang, X.; Ding, B.; Tang, Q.; Sun, S.; Hu, Z.; Chen, M. An amino acid of human parainfluenza virus type 3 nucleoprotein is critical for template function and cytoplasmic inclusion body formation. *J. Virol.* **2013**, *87*, 12457–12470. [CrossRef](https://doi.org/10.1128/JVI.00988-13) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23760258)

35. Patterson, J.B.; Thomas, D.; Lewicki, H.; Billeter, M.A.; Oldstone, M.B. V and C proteins of measles virus function as virulence factors in vivo. *Virology* **2000**, *267*, 80–89. [CrossRef](https://doi.org/10.1006/viro.1999.0120) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10623578)

36. Devaux, P.; Cattaneo, R. Measles virus phosphoprotein gene products: Conformational flexibility of the P/V protein amino-terminal domain and C protein infectivity factor function. *J. Virol.* **2004**, *78*, 11632–11640. [CrossRef](https://doi.org/10.1128/JVI.78.21.11632-11640.2004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15479780)

37. Pohl, C.; Duprex, W.P.; Krohne, G.; Rima, B.K.; Schneider-Schaulies, S. Measles virus M and F proteins associate with detergent-resistant membrane fractions and promote formation of virus-like particles. *J. Gen. Virol.* **2007**, *88 Pt 4*, 1243–1250. [CrossRef](https://doi.org/10.1099/vir.0.82571-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17344536)

38. Tahara, M.; Takeda, M.; Yanagi, Y. Altered interaction of the matrix protein with the cytoplasmic tail of hemagglutinin modulates measles virus growth by affecting virus assembly and cell-cell fusion. *J. Virol.* **2007**, *81*, 6827–6836. [CrossRef](https://doi.org/10.1128/JVI.00114-07) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17428796)

39. Reuter, T.; Weissbrich, B.; Schneider-Schaulies, S.; Schneider-Schaulies, J. RNA interference with measles virus N, P, and L mRNAs efficiently prevents and with matrix protein mRNA enhances viral transcription. *J. Virol.* **2006**, *80*, 5951–5957. [CrossRef](https://doi.org/10.1128/JVI.02538-05) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16699038)

40. Ogino, T.; Banerjee, A.K. Unconventional mechanism of mRNA capping by the RNA-dependent RNA polymerase of vesicular stomatitis virus. *Mol. Cell* **2007**, *25*, 85–97. [CrossRef](https://doi.org/10.1016/j.molcel.2006.11.025) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17218278)

41. Ogino, T.; Kobayashi, M.; Iwama, M.; Mizumoto, K. Sendai virus RNA-dependent RNA polymerase L protein catalyzes cap methylation of virus-specific mRNA. *J. Biol. Chem.* 2005, **280**, 4429–4435. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

42. Plumet, S.; Duprex, W.P.; Gerlier, D. Dynamics of viral RNA synthesis during measles virus infection. *J. Virol.* 2005, **79**, 6900–6908. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

43. Rima, B.K.; Duprex, W.P. The measles virus replication cycle. *Curr. Top. Microbiol. Immunol.* 2009, **329**, 77–102. [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

44. Brindley, M.A.; Takeda, M.; Plattet, P.; Plemper, R.K. Triggering the measles virus membrane fusion machinery. *Proc. Natl. Acad. Sci. USA* 2012, **109**, E3018–E3027. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

45. El Najjar, F.; Schmitt, A.P.; Dutch, R.E. Paramyxovirus glycoprotein incorporation, assembly and budding: A three way dance for infectious particle production. *Viruses* 2014, **6**, 3019–3054. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

46. Colf, L.A.; Juo, Z.S.; Garcia, K.C. Structure of the measles virus hemagglutinin. *Nat. Struct. Mol. Biol.* 2007, **14**, 1227–1228. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

47. Hashiguchi, T.; Kajikawa, M.; Maita, N.; Takeda, M.; Kuroki, K.; Sasaki, K.; Kohda, D.; Yanagi, Y.; Maenaka, K. Crystal structure of measles virus hemagglutinin provides insight into effective vaccines. *Proc. Natl. Acad. Sci. USA* 2007, **104**, 19535–19540. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

48. Tatsuo, H.; Ono, N.; Tanaka, K.; Yanagi, Y. SLAM (CDw150) is a cellular receptor for measles virus. *Nature* 2000, **406**, 893–897. [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

49. Muhlebach, M.D.; Mateo, M.; Sinn, P.L.; Prüfer, S.; Uhlig, K.M.; Leonard, V.H.; Navaratnarajah, C.K.; Frenzke, M.; Wong, X.X.; Sawatsky, B.; et al. Adherens junction protein nectin-4 is the epithelial receptor for measles virus. *Nature* 2011, **480**, 530–533. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

50. Rünkler, N.; Dietzel, E.; Moll, M.; Klenk, H.D.; Maisner, A. Glycoprotein targeting signals influence the distribution of measles virus envelope proteins and virus spread in lymphocytes. *J. Gen. Virol.* 2008, **89 Pt 3**, 687–696. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

51. Muhlebach, M.D.; Leonard, V.H.; Cattaneo, R. The measles virus fusion protein transmembrane region modulates availability of an active glycoprotein complex and fusion efficiency. *J. Virol.* 2008, **82**, 11437–11445. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

52. Xu, K.; Chan, Y.P.; Bradel-Tretheway, B.; Akyol-Ataman, Z.; Zhu, Y.; Dutta, S.; Yan, L.; Feng, Y.; Wang, L.F.; Skiñiotis, G.; et al. Crystal structure of the pre-fusion Nipah virus fusion glycoprotein reveals a novel hexamer-of-trimers assembly. *PLoS Pathog.* 2015, **11**, e1005322. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

53. Whelan, S.P.; Barr, J.N.; Wertz, G.W. Transcription and replication of nonsegmented negative-strand RNA viruses. *Curr. Top. Microbiol. Immunol.* 2004, **283**, 61–119. [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

54. Gutsche, I.; Desfossés, A.; Effantin, G.; Ling, W.L.; Haupt, M.; Ruigrok, R.W.H.; Sachse, C.; Schoehn, G. Near-atomic cryo-EM structure of the helical measles virus nucleocapsid. *Science* 2015, **348**, 704. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

55. Harrison, M.S.; Sakaguchi, T.; Schmitt, A.P. Paramyxovirus assembly and budding: Building particles that transmit infections. *Int. J. Biochem. Cell Biol.* 2010, **42**, 1416–1429. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

56. Rünkler, N.; Pohl, C.; Schneider-Schaulies, S.; Klenk, H.D.; Maisner, A. Measles virus nucleocapsid transport to the plasma membrane requires stable expression and surface accumulation of the viral matrix protein. *Cell. Microbiol.* 2007, **9**, 1203–1214. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

57. Zhang, G.; Zhong, Y.; Qin, Y.; Chen, M. Interaction of human parainfluenza virus type 3 nucleoprotein with matrix protein mediates internal viral protein assembly. *J. Virol.* 2016, **90**, 2306–2315. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

58. Takimoto, T.; Portner, A. Molecular mechanism of paramyxovirus budding. *Virus Res.* 2004, **106**, 133–145. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

59. Liljeroos, L.; Butcher, S.J. Matrix proteins as centralized organizers of negative-sense RNA virions. *Front. Biosci.* 2013, **18**, 696–715. [CrossRef](https://doi.org/CrossRef)

60. Votteler, J.; Sundquist, W.I. Virus budding and the ESCRT pathway. *Cell Host Microbe* 2013, **14**, 232–241. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

61. Salditt, A.; Koethe, S.; Pohl, C.; Harms, H.; Kolesnikova, L.; Becker, S.; Schneider-Schaulies, S. Measles virus M protein-driven particle production does not involve the endosomal sorting complex required for transport (ESCRT) system. *J. Gen. Virol.* 2010, **91 Pt 6**, 1464–1472. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

62. Hsu, E.C.; Iorio, C.; Sarangi, F.; Khine, A.A.; Richardson, C.D. CDw150(SLAM) is a receptor for a lymphotropic strain of measles virus and may account for the immunosuppressive properties of this virus. *Virology* **2001**, *279*, 9–21. [CrossRef](https://doi.org/10.1006/viro.2000.0834) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11430882)

63. De Vries, R.D.; McQuaid, S.; van Amerongen, G.; Yuksel, S.; Verburgh, R.J.; Osterhaus, A.D.; Duprex, W.P.; de Swart, R.L. Measles immune suppression: Lessons from the macaque model. *PLoS Pathog.* **2012**, *8*, e1002885. [CrossRef](https://doi.org/10.1371/journal.ppat.1002885) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22916026)

64. Noyce, R.S.; Richardson, C.D. Nectin 4 is the epithelial cell receptor for measles virus. *Trends Microbiol.* **2012**, *20*, 429–439. [CrossRef](https://doi.org/10.1016/j.tim.2012.06.006) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22748773)

65. Dorig, R.E.; Marcil, A.; Chopra, A.; Richardson, C.D. The human CD46 molecule is a receptor for measles virus (Edmonston strain). *Cell* **1993**, *75*, 295–305. [CrossRef](https://doi.org/10.1016/0092-8674(93)90383-2)

66. Naniche, D.; Varior-Krishnan, G.; Cervoni, F.; Wild, T.F.; Rossi, B.; Rabourdin-Combe, C.; Gerlier, D. Human membrane cofactor protein (CD46) acts as a cellular receptor for measles virus. *J. Virol.* **1993**, *67*, 6025–6032. [PubMed](https://pubmed.ncbi.nlm.nih.gov/8371376)

67. Young, V.A.; Rall, G.F. Making it to the synapse: Measles virus spread in and among neurons. *Curr. Top. Microbiol. Immunol.* **2009**, *330*, 3–30. [PubMed](https://pubmed.ncbi.nlm.nih.gov/19216530)

68. Laksono, B.M.; de Vries, R.D.; McQuaid, S.; Duprex, W.P.; de Swart, R.L. Measles virus host invasion and pathogenesis. *Viruses* **2016**, *8*, 210. [CrossRef](https://doi.org/10.3390/v8060160) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27258296)

69. Lyupina, Y.V.; Dmitrieva, S.B.; Timokhova, A.V.; Beljelarskaya, S.N.; Zatsepina, O.G.; Evgen'ev, M.B.; Mikhailov, V.S. An important role of the heat shock response in infected cells for replication of baculoviruses. *Virology* **2010**, *406*, 336–341. [CrossRef](https://doi.org/10.1016/j.virol.2010.06.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20599805)

70. Chen, Y.J.; Chen, Y.H.; Chow, L.P.; Tsai, Y.H.; Chen, P.H.; Huang, C.Y.; Chen, W.T.; Hwang, L.H. Heat shock protein 72 is associated with the hepatitis C virus replicase complex and enhances viral RNA replication. *J. Biol. Chem.* **2010**, *285*, 28183–28190. [CrossRef](https://doi.org/10.1074/jbc.M110.135717) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20581223)

71. Carsillo, T.; Traylor, Z.; Choi, C.; Niewiesk, S.; Oglesbee, M. Hsp72, a host determinant of measles virus neurovirulence. *J. Virol.* **2006**, *80*, 11031–11039. [CrossRef](https://doi.org/10.1128/JVI.00846-06) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16912020)

72. Ito, M.; Iwasaki, M.; Takeda, M.; Nakamura, T.; Yanagi, Y.; Ohno, S. Measles virus nonstructural C protein modulates viral RNA polymerase activity by interacting with host protein SHCBP1. *J. Virol.* **2013**, *87*, 9633–9642. [CrossRef](https://doi.org/10.1128/JVI.00554-13) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23645687)

73. Das, T.; Schuster, A.; Schneider-Schaulies, S.; Banerjee, A.K. Involvement of cellular casein kinase II in the phosphorylation of measles virus P protein: Identification of phosphorylation sites. *Virology* **1995**, *211*, 218–226. [CrossRef](https://doi.org/10.1006/viro.1995.1357) [PubMed](https://pubmed.ncbi.nlm.nih.gov/7618091)

74. Sugai, A.; Sato, H.; Hagiwara, K.; Kozuka-Hata, H.; Oyama, M.; Yoneda, M.; Kai, C. Newly identified minor phosphorylation site threonine-279 of measles virus nucleoprotein is a prerequisite for nucleocapsid formation. *J. Virol.* **2014**, *88*, 1140–1149. [CrossRef](https://doi.org/10.1128/JVI.02297-13) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24259417)

75. Wakimoto, H.; Shimodo, M.; Satoh, Y.; Kitagawa, Y.; Takeuchi, K.; Gotoh, B.; Itoh, M. F-actin modulates measles virus cell-cell fusion and assembly by altering the interaction between the matrix protein and the cytoplasmic tail of hemagglutinin. *J. Virol.* **2013**, *87*, 1974–1984. [CrossRef](https://doi.org/10.1128/JVI.02245-12) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23175884)

76. Sugai, A.; Sato, H.; Yoneda, M.; Kai, C. Phosphorylation of measles virus phosphoprotein at S86 and/or S151 downregulates viral transcriptional activity. *FEBS Lett.* **2012**, *586*, 3900–3907. [CrossRef](https://doi.org/10.1016/j.febslet.2012.09.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22982256)

77. Sugai, A.; Sato, H.; Yoneda, M.; Kai, C. Phosphorylation of measles virus nucleoprotein affects viral growth by changing gene expression and genomic RNA stability. *J. Virol.* **2013**, *87*, 11684–11692. [CrossRef](https://doi.org/10.1128/JVI.01148-13) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23824866)

78. Moyer, S.A.; Baker, S.C.; Lessard, J.L. Tubulin: A factor necessary for the synthesis of both Sendai virus and vesicular stomatitis virus RNAs. *Proc. Natl. Acad. Sci. USA* **1986**, *83*, 5405–5409. [CrossRef](https://doi.org/10.1073/pnas.83.14.5405) [PubMed](https://pubmed.ncbi.nlm.nih.gov/3016508)

79. Berghall, H.; Wallen, C.; Hyypia, T.; Vainionpaa, R. Role of cytoskeleton components in measles virus replication. *Arch. Virol.* **2004**, *149*, 891–901. [CrossRef](https://doi.org/10.1007/s00705-003-0238-2) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15021182)

80. Moyer, S.A.; Baker, S.C.; Horikami, S.M. Host cell proteins required for measles virus reproduction. *J. Gen. Virol.* **1990**, *71 Pt 4*, 775–783. [CrossRef](https://doi.org/10.1099/0022-1317-71-4-775) [PubMed](https://pubmed.ncbi.nlm.nih.gov/2111953)

81. Nakatsu, Y.; Ma, X.; Seki, F.; Suzuki, T.; Iwasaki, M.; Yanagi, Y.; Komase, K.; Takeda, M. Intracellular transport of the measles virus ribonucleoprotein complex is mediated by Rab11A-positive recycling endosomes and drives virus release from the apical membrane of polarized epithelial cells. *J. Virol.* **2013**, *87*, 4683–4693. [CrossRef](https://doi.org/10.1128/JVI.02575-12) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23468481)

82. Dietzel, E.; Kolesnikova, L.; Maisner, A. Actin filaments disruption and stabilization affect measles virus maturation by different mechanisms. *Virol. J.* 2013, *10*, 249. [CrossRef](https://doi.org/10.1186/1743-422X-10-249) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24252103)

83. Sánchez David, R.Y.; Combredet, C.; Sismeiro, O.; Dillies, M.A.; Jagla, B.; Coppee, J.Y.; Mura, M.; Guerbois Galla, M.; Despres, P.; Tangy, F.; et al. Comparative analysis of viral RNA signatures on different RIG-I-like receptors. *eLife* 2016, *5*, e11275. [CrossRef](https://doi.org/10.7554/eLife.11275) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26986437)

84. Ikegame, S.; Takeda, M.; Ohno, S.; Nakatsu, Y.; Nakanishi, Y.; Yanagi, Y. Both RIG-I and MDA5 RNA helicases contribute to the induction of alpha/beta interferon in measles virus-infected human cells. *J. Virol.* 2010, *84*, 372–379. [CrossRef](https://doi.org/10.1128/JVI.01978-09) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20007263)

85. Bieback, K.; Lien, E.; Klagge, I.M.; Avota, E.; Schneider-Schaulies, J.; Duprex, W.P.; Wagner, H.; Kirschning, C.J.; Ter Meulen, V.; Schneider-Schaulies, S. Hemagglutinin protein of wild-type measles virus activates toll-like receptor 2 signaling. *J. Virol.* 2002, *76*, 8729–8736. [CrossRef](https://doi.org/10.1128/JVI.76.17.8729-8736.2002) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12149363)

86. Runge, S.; Sparrer, K.M.; Lassig, C.; Hembach, K.; Baum, A.; Garcia-Sastre, A.; Soding, J.; Conzelmann, K.K.; Hopfner, K.P. In vivo ligands of MDA5 and RIG-I in measles virus-infected cells. *PLoS Pathog.* 2014, *10*, e1004081. [CrossRef](https://doi.org/10.1371/journal.ppat.1004081) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24675929)

87. Takeuchi, O.; Akira, S. Pattern recognition receptors and inflammation. *Cell* 2010, *140*, 805–820. [CrossRef](https://doi.org/10.1016/j.cell.2010.01.058) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20211134)

88. Rodriguez, K.R.; Horvath, C.M. Paramyxovirus V protein interaction with the antiviral sensor LGP2 disrupts MDA5 signaling enhancement but is not relevant to LGP2-mediated RLR signaling inhibition. *J. Virol.* 2014, *88*, 8180–8188. [CrossRef](https://doi.org/10.1128/JVI.00488-14) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24711456)

89. Uchikawa, E.; Lethier, M.; Malet, H.; Brunel, J.; Gerlier, D.; Cusack, S. Structural analysis of dsRNA binding to anti-viral pattern recognition receptors LGP2 and MDA5. *Mol. Cell* 2016, *62*, 586–602. [CrossRef](https://doi.org/10.1016/j.molcel.2016.03.023) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27156472)

90. Davis, M.E.; Wang, M.K.; Rennick, L.J.; Full, F.; Gableske, S.; Mesman, A.W.; Gringhuis, S.I.; Geijtenbeek, T.B.; Duprex, W.P.; Gack, M.U. Antagonism of the phosphatase PP1 by the measles virus V protein is required for innate immune escape of MDA5. *Cell Host Microbe* 2014, *16*, 19–30. [CrossRef](https://doi.org/10.1016/j.chom.2014.05.016) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25015998)

91. Yokota, S.; Okabayashi, T.; Yokosawa, N.; Fujii, N. Measles virus P protein suppresses Toll-like receptor signal through up-regulation of ubiquitin-modifying enzyme A20. *FASEB J.* 2008, *22*, 74–83. [CrossRef](https://doi.org/10.1096/fj.07-8960com) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17884963)

92. Li, L.; Hailey, D.W.; Soetandyo, N.; Li, W.; Lippincott-Schwartz, J.; Shu, H.B.; Ye, Y. Localization of A20 to a lysosome-associated compartment and its role in NFkappaB signaling. *Biochim. Biophys. Acta* 2008, *1783*, 1140–1149. [CrossRef](https://doi.org/10.1016/j.bbamcr.2008.03.008) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18417268)

93. Creagh, E.M.; O’Neill, L.A. TLRs, NLRs and RLRs: A trinity of pathogen sensors that co-operate in innate immunity. *Trends Immunol.* 2006, *27*, 352–357. [CrossRef](https://doi.org/10.1016/j.it.2006.05.009) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16713955)

94. Xu, L.G.; Wang, Y.Y.; Han, K.J.; Li, L.Y.; Zhai, Z.; Shu, H.B. VISA is an adapter protein required for virus-triggered IFN-beta signaling. *Mol. Cell* 2005, *19*, 727–740. [CrossRef](https://doi.org/10.1016/j.molcel.2005.07.008) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16102392)

95. Pfaller, C.K.; Conzelmann, K.K. Measles virus V protein is a decoy substrate for IkappaB kinase alpha and prevents Toll-like receptor 7/9-mediated interferon induction. *J. Virol.* 2008, *82*, 12365–12373. [CrossRef](https://doi.org/10.1128/JVI.00948-08) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18614716)

96. Irie, T.; Kiyotani, K.; Igarashi, T.; Yoshida, A.; Sakaguchi, T. Inhibition of interferon regulatory factor 3 activation by paramyxovirus V protein. *J. Virol.* 2012, *86*, 7136–7145. [CrossRef](https://doi.org/10.1128/JVI.00069-12) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22491457)

97. Komune, N.; Ichinohe, T.; Ito, M.; Yanagi, Y. Measles virus V protein inhibits NLRP3 inflammasome-mediated interleukin-1beta secretion. *J. Virol.* 2011, *85*, 13019–13026. [CrossRef](https://doi.org/10.1128/JVI.05379-11) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21813788)

98. Schuhmann, K.M.; Pfaller, C.K.; Conzelmann, K.K. The measles virus V protein binds to p65 (RelA) to suppress NF-kappaB activity. *J. Virol.* 2011, *85*, 3162–3171. [CrossRef](https://doi.org/10.1128/JVI.01975-10) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21248083)

99. Vijayan, M.; Seo, Y.J.; Pritzl, C.J.; Squires, S.A.; Alexander, S.; Hahm, B. Sphingosine kinase 1 regulates measles virus replication. *Virology* 2014, *450–451*, 55–63. [CrossRef](https://doi.org/10.1016/j.virol.2014.01.022) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24508859)

100. Pfaller, C.K.; Radeke, M.J.; Cattaneo, R.; Samuel, C.E. Measles virus C protein impairs production of defective copyback double-stranded viral RNA and activation of protein kinase R. *J. Virol.* 2014, *88*, 456–468. [CrossRef](https://doi.org/10.1128/JVI.02188-13) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24198409)

101. Sparrer, K.M.; Pfaller, C.K.; Conzelmann, K.K. Measles virus C protein interferes with Beta interferon transcription in the nucleus. *J. Virol.* 2012, *86*, 796–805. [CrossRef](https://doi.org/10.1128/JVI.06386-11) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22025688)

102. Fleming, S.B. Viral Inhibition of the IFN-Induced JAK/STAT Signalling Pathway: Development of Live Attenuated Vaccines by Mutation of Viral-Encoded IFN-Antagonists. *Vaccines* **2016**, *4*, 23. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

103. Pitini, V.; Arrigo, C.; Altavilla, G. How cells respond to interferons. *J. Clin. Oncol.* **2010**, *28*, e439. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

104. Takayama, I.; Sato, H.; Watanabe, A.; Omi-Furutani, M.; Sugai, A.; Kanki, K.; Yoneda, M.; Kai, C. The nucleocapsid protein of measles virus blocks host interferon response. *Virology* **2012**, *424*, 45–55. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

105. Palosaari, H.; Parisien, J.P.; Rodriguez, J.J.; Ulane, C.M.; Horvath, C.M. STAT protein interference and suppression of cytokine signal transduction by measles virus V protein. *J. Virol.* **2003**, *77*, 7635–7644. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

106. Caignard, G.; Bourai, M.; Jacob, Y.; Tangy, F.; Vidalain, P.O. Inhibition of IFN-α/β signaling by two discrete peptides within measles virus V protein that specifically bind STAT1 and STAT2. *Virology* **2009**, *383*, 112–120. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

107. Devaux, P.; von Messling, V.; Songsungthong, W.; Springfeld, C.; Cattaneo, R. Tyrosine 110 in the measles virus phosphoprotein is required to block STAT1 phosphorylation. *Virology* **2007**, *360*, 72–83. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

108. Devaux, P.; Hudacek, A.W.; Hodge, G.; Reyes-del Valle, J.; McChesney, M.B.; Cattaneo, R. A recombinant measles virus unable to antagonize STAT1 function cannot control inflammation and is attenuated in Rhesus monkeys. *J. Virol.* **2011**, *85*, 348–356. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

109. Yokota, S.-I.; Okabayashi, T.; Fujii, N. Measles virus C protein suppresses gamma-activated factor formation and virus-induced cell growth arrest. *Virology* **2011**, *414*, 74–82. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

110. Li, Z.; Okonski, K.M.; Samuel, C.E. Adenosine deaminase acting on RNA 1 (ADAR1) suppresses the induction of interferon by measles virus. *J. Virol.* **2012**, *86*, 3787–3794. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

111. Pfaller, C.K.; Li, Z.; George, C.X.; Samuel, C.E. Protein kinase PKR and RNA adenosine deaminase ADAR1: New roles for old players as modulators of the interferon response. *Curr. Opin. Immunol.* **2011**, *23*, 573–582. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

112. Okonski, K.M.; Samuel, C.E. Stress granule formation induced by measles virus is protein kinase PKR dependent and impaired by RNA adenosine deaminase ADAR1. *J. Virol.* **2013**, *87*, 756–766. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

113. Nakatsu, Y.; Takeda, M.; Ohno, S.; Shirogane, Y.; Iwasaki, M.; Yanagi, Y. Measles virus circumvents the host interferon response by different actions of the C and V proteins. *J. Virol.* **2008**, *82*, 8296–8306. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

114. Takeuchi, K.; Komatsu, T.; Kitagawa, Y.; Sada, K.; Gotoh, B. Sendai virus C protein plays a role in restricting PKR activation by limiting the generation of intracellular double-stranded RNA. *J. Virol.* **2008**, *82*, 10102–10110. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

115. Kedersha, N.; Anderson, P. Stress granules: Sites of mRNA triage that regulate mRNA stability and translatability. *Biochem. Soc. Trans.* **2002**, *30 Pt 6*, 963–969. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

116. Jordan, T.X.; Randall, G. Manipulation or capitulation: Virus interactions with autophagy. *Microbes Infect.* **2012**, *14*, 126–139. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

117. Richetta, C.; Gregoire, I.P.; Verlhac, P.; Azocar, O.; Baguet, J.; Flacher, M.; Tangy, F.; Rabourdin-Combe, C.; Faure, M. Sustained autophagy contributes to measles virus infectivity. *PLoS Pathog.* **2013**, *9*, e1003599. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

118. Gregoire, I.P.; Richetta, C.; Meyniel-Schicklin, L.; Borel, S.; Pradezynski, F.; Diaz, O.; Deloire, A.; Azocar, O.; Baguet, J.; Le Breton, M.; et al. IRGM is a common target of RNA viruses that subvert the autophagy network. *PLoS Pathog.* **2011**, *7*, e1002422. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

119. Delpeut, S.; Rudd, P.A.; Labonte, P.; von Messling, V. Membrane fusion-mediated autophagy induction enhances morbillivirus cell-to-cell spread. *J. Virol.* **2012**, *86*, 8527–8535. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

120. Joubert, P.E.; Meiffren, G.; Gregoire, I.P.; Pontini, G.; Richetta, C.; Flacher, M.; Azocar, O.; Vidalain, P.O.; Vidal, M.; Lotteau, V.; et al. Autophagy induction by the pathogen receptor CD46. *Cell Host Microbe* **2009**, *6*, 354–366. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

121. Rivailler, P.; Trescol-Biemont, M.C.; Gimenez, C.; Rabourdin-Combe, C.; Horvat, B. Enhanced MHC class II-restricted presentation of measles virus (MV) hemagglutinin in transgenic mice expressing human MV receptor CD46. *Eur. J. Immunol.* **1998**, *28*, 1301–1314. [CrossRef](https://doi.org/CrossRef)

122. Ding, B.; Zhang, G.; Yang, X.; Zhang, S.; Chen, L.; Yan, Q.; Xu, M.; Banerjee, A.K.; Chen, M. Phosphoprotein of human parainfluenza virus type 3 blocks autophagosome-lysosome fusion to increase virus production. *Cell Host Microbe* **2014**, *15*, 564–577. [CrossRef](https://doi.org/10.1016/j.chom.2014.04.018) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24905588)

123. Xia, M.; Gonzalez, P.; Li, C.; Meng, G.; Jiang, A.; Wang, H.; Gao, Q.; Debatin, K.M.; Beltinger, C.; Wei, J. Mitophagy enhances oncolytic measles virus replication by mitigating DDX58/RIG-I-like receptor signaling. *J. Virol.* **2014**, *88*, 5152–5164. [CrossRef](https://doi.org/10.1128/JVI.03458-13) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24671966)

124. Xia, M.; Meng, G.; Jiang, A.; Chen, A.; Dahlhaus, M.; Gonzalez, P.; Beltinger, C.; Wei, J. Mitophagy switches cell death from apoptosis to necrosis in NSCLC cells treated with oncolytic measles virus. *Oncotarget* **2014**, *5*, 3907–3918. [CrossRef](https://doi.org/10.18632/oncotarget.1887) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24569182)
