
CHAPTER FOUR

# Amyotrophic Lateral Sclerosis Pathogenesis Converges on Defects in Protein Homeostasis Associated with TDP-43 Mislocalization and Proteasome-Mediated Degradation Overload

G. Lin${}^{*,†,1}$, D. Mao${}^{†,‡,1}$, H.J. Bellen${}^{*,†,§,2}$

* Baylor College of Medicine, Houston, TX, United States  
† Jan and Dan Duncan Neurological Research Institute, Texas Children’s Hospital, Houston, TX, United States  
‡ Program in Developmental Biology, Baylor College of Medicine, Houston, TX, United States  
§ Howard Hughes Medical Institute, Baylor College of Medicine, Houston, TX, United States  
$^2$ Corresponding author: e-mail address: hbellen@bcm.edu

## Contents

1. Introduction                                                                                          112
2. Current Challenges in ALS Research                                                                   114
3. RNA Processing, Splicing, RNA Foci, and Protein Aggregation                                           129
   - 3.1 TDP-43                                                                                         129
   - 3.2 FUS                                                                                            132
   - 3.3 C9orf72                                                                                        136
4. Proteostasis Deficiency in ALS                                                                       140
   - 4.1 VCP                                                                                            140
   - 4.2 UBQLN 2                                                                                        144
   - 4.3 VAPB                                                                                           148
5. SOD-1 and Proteinopathy in ALS                                                                       151
6. Prion-Like Protein Toxicity and ALS                                                                  154
7. Conclusion and Future Perspective                                                                     155
Acknowledgments                                                                                         156
References                                                                                              156

## Abstract

Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disorder that affects upper and/or lower motor neurons. It usually affects people between the ages of 40–70. The average life expectancy is about 3–5 years after diagnosis and there is no

1 These authors contributed equally to this work.

Current Topics in Developmental Biology, Volume 121  
ISSN 0070-2153  
http://dx.doi.org/10.1016/bs.ctdb.2016.07.004  
© 2017 Elsevier Inc. All rights reserved.

effective cure available. Identification of variants in more than 20 different loci has provided insight into the pathogenic molecular mechanisms mediating disease pathogenesis. In this review, we focus on seven ALS-causing genes: TDP-43, FUS, C9orf72, VCP, UBQLN2, VAPB and SOD-1, which encompass about 90% of the variants causing familial ALS. We examine the biological functions of these genes to assess how these pathogenic variants contribute to ALS pathogenesis by integrating findings from studies in *Drosophila melanogaster* and mammals. Additionally, we highlight the functional and genetic connections between these loci. Altogether, this review reveals that the majority of biological studies converge on defects in proteostasis due to the mislocalization of TDP-43 and/or altering the function of specific proteins mediating or modulating proteasomal degradation.

---

### 1. INTRODUCTION

Amyotrophic lateral sclerosis (ALS), also known as Lou Gehrig’s disease, is a fatal neurodegenerative disorder characterized by progressive degeneration of upper and lower motor neurons with a prevalence of 4–6 per 100,000 in the population worldwide ([Majoor-Krakauer, Willems, & Hofman, 2003](https://doi.org/10.1002/mds.10585)) and a lifelong risk of 1 in 600 individuals ([Gonzalez-Perez et al., 2012](https://doi.org/10.1016/j.neurobiolaging.2012.03.010)). The incidence of this disease increases with age and is higher in males than females ([Alonso, Logroscino, Jick, & Hernan, 2009](https://doi.org/10.1002/mds.22623); [Fang et al., 2009](https://doi.org/10.1002/mds.22623)). ALS was first described by Charles Bell in 1824 and Jean-Martin Charcot first linked the clinical symptoms with the neurological pathology in 1869 ([Rowland & Shneider, 2001](https://doi.org/10.1002/mds.10585)). Charcot introduced the term amyotrophic lateral sclerosis, where amyotrophic means “no muscle nourishment” in Greek ([Rowland & Shneider, 2001](https://doi.org/10.1002/mds.10585)). The most characteristic clinical feature of ALS is the progressive motor weakness and atrophy ([Peters, Ghasemi, & Brown, 2015](https://doi.org/10.1016/j.neurobiolaging.2012.03.010)).

There are two types of ALS, familial ALS (FALS) and sporadic ALS (SALS) ([Gros-Louis, Gaspar, & Rouleau, 2006](https://doi.org/10.1002/mds.22623)). About 10% of patients with a positive family history are classified as FALS, whereas the remainder are SALS ([Mitchell & Borasio, 2007](https://doi.org/10.1002/mds.22623)). Pathologically, protein and RNA aggregates are widely observed in motor neurons of postmortem ALS brain and spinal cord ([Peters, Ghasemi, et al., 2015](https://doi.org/10.1016/j.neurobiolaging.2012.03.010)). These aggregates usually contain ubiquitin and TDP-43 ([Blokhus, Groen, Koppers, van den Berg, & Pasterkamp, 2013](https://doi.org/10.1002/mds.22623)). Hence, defects in RNA metabolism and an imbalance in proteostasis are thought to play an important role in ALS pathogenesis. However, it is unclear how an imbalance in proteostasis develops and
how this relates to ALS pathogenesis. The genes identified in FALS provide an opportunity to dissect the pathogenic mechanisms.

In this review, we focus on seven major ALS-causing genes: TARDBP (TDP-43), FUS, C9orf72, VCP, UBQLN2, VAPB, and SOD-1. *Drosophila* has been used to study the mechanisms associated with the function of these loci in neurodegenerative diseases. Table 1 summarizes techniques that are frequently used in *Drosophila* to define phenotypes associated with neurodegeneration. Please refer to the references therein for a detailed description of these methods. All the genes discussed in this review, except C9orf72, have *Drosophila* homologs and have been explored in this model organism. For simplicity, we use the human nomenclature rather than the fly

**Table 1 Assays Frequently Used to Characterize Neurodegenerative Defects in *Drosophila***

| Defects          | Tissue               | Assay                          | References                              |
|------------------|----------------------|--------------------------------|-----------------------------------------|
| Survival         | Whole animal         | Life span                      | Linford, Bilgir, Ro, and Pletcher (2013) |
|                  |                      | Lethal phase                   |                                         |
|                  |                      | Eclosion rate                  |                                         |
| Neuronal defects | Photoreceptor        | Transmission electron microscopy | Wolff (2011)                           |
|                  | morphology           | Immunofluorescence             | Walther and Pichaud (2006)              |
|                  | Photoreceptor        | Electroretinogram              | Dolph, Nair, and Raghu (2011)           |
|                  | function             |                                |                                         |
| Neuromuscular    | Giant fiber assay    |                                | Allen and Godenschwege (2010)           |
| junction (NMJ)   | (Adult)              |                                |                                         |
|                  | NMJ morphology       |                                | Andlauer and Sigrist (2012a, 2012b, 2012c) |
|                  | (Larva)              |                                |                                         |
|                  | NMJ electrophysiology |                                | Imlach and McCabe (2009)                |
|                  | (Larva)              |                                |                                         |
| Locomotor        | Behavioral assays    | Negative geotaxis assay (Adult) | Ali, Escala, Ruan, and Zhai (2011)      |
| defects          |                      | Flight assay (Adult)           | Babcock and Ganetzky (2014)             |
|                  |                      | Crawling assay                 | Nichols, Becnel, and Pandey (2012)      |
|                  |                      | (Larva)                        |                                         |nomenclature (TDP-43 in flies is TAR DNA-binding protein-43 homolog (TBPH); FUS is cabeza (*caz*); VCP is Transitional endoplasmic reticulum ATPase (TER94); UBQLN2 is Ubiquilin (Ubqn); and VAPB is VAMP-associated protein-33-1 (*Vap-33-1*)). Findings from vertebrate and *Drosophila* studies suggest that variants in these seven loci cause progressive defects in protein homeostasis that converge on the inability of neurons to cope with protein degradation. We propose a model in which four factors contribute to the proteinopathy. First, in C9orf72-related ALS, the presence of a hexanucleotide repeat expansion (HRE), (GGGGCC)*n*, in the noncoding intron of the C9orf72 gene ([DeJesus-Hernandez et al., 2011](https://doi.org/10.1016/j.neuron.2011.09.011); [Renton et al., 2011](https://doi.org/10.1016/j.neuron.2011.09.010)) causes the accumulation of abortive transcripts that form foci containing RNAs and proteins ([Donnelly et al., 2013](https://doi.org/10.1016/j.cell.2013.08.016); [Haeusler et al., 2014](https://doi.org/10.1016/j.cell.2014.05.019)). In addition, dipeptide proteins repeat (DPRs) synthesized via an ATG-independent translation mechanism has been proposed to mediate HRE toxicity. Moreover, the HRE-containing transcripts or the arginine-containing DPRs recruit various proteins, including proteins required for nuclear import and export ([Freibaum et al., 2015](https://doi.org/10.1016/j.cell.2015.01.036); [Zhang, Donnelly, et al., 2015](https://doi.org/10.1016/j.cell.2015.01.037)). These in turn lead to cytoplasmic mislocalization of TDP-43 ([Chew et al., 2015](https://doi.org/10.1016/j.cell.2015.01.038); [Cooper-Knock et al., 2015](https://doi.org/10.1016/j.cell.2015.01.039); [Zhang, Donnelly, et al., 2015](https://doi.org/10.1016/j.cell.2015.01.037)). Second, the aggregated and mislocalized TDP-43 lead to the inclusion of cryptic exons in primary mRNA transcripts ([Ling, Pletnikova, Troncoso, & Wong, 2015](https://doi.org/10.1016/j.neuron.2015.01.018)), thereby producing aberrant mRNA and truncated proteins ([Ling et al., 2015](https://doi.org/10.1016/j.neuron.2015.01.018)). Third, these aberrant proteins and aggregates are typically removed by proteasome degradation or autophagy ([Wong & Cuervo, 2010](https://doi.org/10.1016/j.jmb.2010.01.010)). However, ALS-causing variants in genes implicated in proteasome degradation like VCP, UBQLN2, and VAPB, impair this mechanism and cause ALS ([Deng et al., 2011](https://doi.org/10.1016/j.cell.2011.09.012); [Johnson et al., 2010](https://doi.org/10.1016/j.cell.2010.05.039); [Nishimura et al., 2004](https://doi.org/10.1038/ng1358)). Finally, the accumulation of RNA and protein aggregates lead to progressive degenerative phenotypes ([Blokhus et al., 2013](https://doi.org/10.1016/j.neuron.2013.01.010); [Ling, Polymenidou, & Cleveland, 2013](https://doi.org/10.1016/j.neuron.2013.01.011)). Table 2 summarizes many of the key papers discussed in this review. We realize that this model is an oversimplification of the available data, but it provides a useful framework to integrate many data sets that will be expanded upon below.

---

### 2. CURRENT CHALLENGES IN ALS RESEARCH

In the past two decades, many of the breakthroughs in ALS research are associated with the identification of novel ALS pathogenic variants and the characterization of the respective genes ([Ajroud-Driss & Siddique, 2015](https://doi.org/10.1016/j.neurobiolaging.2015.01.010)).
Table 2 Key Papers Discussed in This Review

| Gene | Genetic Manipulation | TDP-43 Phenotype | Proteostasis | Species | References |
|------|----------------------|------------------|--------------|---------|------------|
|      | FTD and ALS patients | Identification of hyperphosphorylated TDP-43 aggregates | Hyperphosphorylated TDP-43 aggregates | Human | Arai et al. (2010), Hasegawa et al. (2008), and Neumann et al. (2009) |
|      |                      | Identification of hyperphosphorylated, ubiquitinated, and cleaved 25-kDa TDP-43 | N.D. | Human | Neumann et al. (2006) |
| TDP-43 | TDP-43 deleted mouse embryonic stem cells; ALS patients | Loss of TDP-43 causes the incorporation of cryptic exons into mRNAs, disrupts mRNA translation, and may lead to protein aggregation | N.D. | Human; mammalian cell line | Ling et al. (2015) |
|      | Ectopic expression of wild-type TDP-43 | Nuclear localization | No protein aggregation | Mammalian cell line | Zhang et al. (2009), Arai et al. (2010), and Yang et al. (2010) |
|      |                      | Formation of cytosolic dTDP-43-positive aggregates | Protein aggregates | Drosophila | Lin, Cheng, and Shen (2011) |

Continued

Table 2 Key Papers Discussed in This Review—cont'd

| Gene | Genetic Manipulation | TDP-43 Phenotype | Proteostasis | Species | References |
|------|-----------------------|------------------|--------------|---------|------------|
|      | Ectopic expression of C-terminal disease-related 25-kDa or 35-kDa fragments of TDP-43 | Cytoplasmic localized TDP-43 | Protein aggregations | Mammalian cell line | Zhang et al. (2009), Arai et al. (2010), and Yang et al. (2010) |
| TDP-43 | Ectopic expression of wild-type, ΔNLS, disease related (G287S, G348C, N390D, A315T, or A382T), or RNA-binding deficient TDP-43 | Only ΔNLS leads to a cytoplasmic localization of TDP-43 | N.D. | Mammalian cell line and Drosophila | Voigt et al. (2010) |
|      | Ectopic expression of wild-type or disease-related (A315T) TDP-43 | TDP-43andA315T aggregate in axons of larval eye imaginal discs | N.D. | Drosophila | Estes et al. (2011) |
|      | Ectopic expression of disease-related (A315T) TDP-43 | Neurons with ubiquitin pathology show loss of nuclear TDP-43 staining | Ubiquitin-positive aggregations in frontal cortex and spinal motor neurons | Mouse | Wegorzewska, Bell, Cairns, Miller, and Baloh (2009) |

| Ectopic expression of TDP-43 with Ser 409/410 mutated to phosphomimetic aspartic acids | Reduces TDP-43-positive aggregates | Reduces TDP-43-positive aggregates | Mammalian cell line | Brady, Meng, Zheng, Mao, and Hu (2011) and Li, Yeh, Chiu, Tang, and Tu (2011) |
| --- | --- | --- | --- | --- |
| TDP-43 Overexpression of a chaperone, CG 14207 | Reduces TDP-43-positive aggregates and TDP-43 toxicity | Reduces TDP-43-positive aggregates and TDP-43 toxicity | Drosophila | Gregory, Barros, Meehan, Dobson, and Luheshi (2012) |
| Overexpression of a chaperone, Hsp 104 | Suppresses TDP-43 toxicity, aggregation, and mislocalization | Suppresses TDP-43 toxicity, aggregation, and mislocalization | Yeast | Jackrel et al. (2014) |
| FALS patient or mammalian cell lines ectopically express wild-type or disease-related FUS(R521G) | Wild-type FUS localized to nucleus, whereas cytoplasmic localization is observed in disease-related (R521G) FUS | Cytoplasmic aggregates | Human and mammalian cell line | Kwiatkowski et al. (2009) and Vance et al. (2009) |
| Ectopic expression of wild-type or disease-related FUS(R518K) | Wild-type FUS is localized to nucleus, whereas cytoplasmic localization is observed in disease-related (R518K) FUS; FUS genetically interacts with TDP-43 in a disease mutation-dependent manner to cause eye degeneration | N.D. | Drosophila | Lanson et al. (2011) |

Table 2 Key Papers Discussed in This Review—cont'd

| Gene | Genetic Manipulation | TDP-43 Phenotype | Proteostasis | Species | References |
|------|-----------------------|------------------|--------------|---------|------------|
| FUS<sup>b</sup> | Ectopic expression of wild-type or phosphorylation deficient FUS | DNA damage induces FUS phosphorylation and cytoplasmic localization FUS | N.D. | Mammalian cell lines | Deng, Holler, et al. (2014) |
|  | Ectopic expression of wild-type or phosphorylation deficient FUS | C-terminal phosphorylation on Tyr 526 impairs FUS nucleotransport machinery | N.D. | Mammalian cell lines | Darovic et al. (2015) |
|  | Ectopic expression of wild-type or argine methylation deficient FUS | Argine methylation of FUS is required for its interaction with transportin 1 and nuclear transport | N.D. | Mammalian cell lines; Drosophila | Dormann et al. (2012), Scaramuzzino et al. (2013), and Jackel et al. (2015) |
|  | shRNA knockdown of FUS | Misspliced Tau protein and formation of Tau aggregates | N.D. | Mammalian cell lines | Orozco et al. (2012) |
|  | Motorneuron expression of wild-type or disease-related FUS (R518KorR521C) | Wild-type FUS localized to nucleus, whereas cytoplasmic localization is observed in disease-related (R518KorR521C) FUS | N.D. | Drosophila | Daigle et al. (2013) |

| FALS patients | N.D. | Formation of RNA foci in nuclei | Human | DeJesus-Hernandez et al. (2011) |
| --- | --- | --- | --- | --- |
| Cytoplasmic TDP-43 aggregates | Cytoplasmic TDP-43 aggregates | Human | Stewart et al. (2012) and Rohrer et al. (2015) |
| Cytoplasmic TDP-43, p62 and Ubiquitin-2-positive aggregates | Cytoplasmic TDP-43, p62, and Ubiquitin-2-positive aggregates | Human | Fratta et al. (2013) |
| CNS samples from C9orf72ALS patients | TDP-43 mislocalization | Formation of RNA foci | Human | Cooper-Knock et al. (2015) |
| N.D. | Detection of DPR aggregates in the cerebellum of C9orf72 ALS patients | Human | Ash et al. (2013), Mori et al. (2013), and Zuet al. (2013) |
| CNS samples from C9orf72ALS patients | N.D. | DPR aggregates are rarely present in the spinal cords or motor neurons of C9orf72 ALS patients | Human | Gomez-Deza et al. (2015) |
| Neuronal and glial knockout of C9orf72 | No obvious TDP-43 mislocalization | No protein aggregation | Mouse | Koppers et al. (2015) |
| iPSC-derived from C9orf72ALS patients | N.D. | Intranuclear RNA foci that contain HRE and HRE-binding protein | Human | Donnelly et al. (2013) |

Table 2 Key Papers Discussed in This Review—cont'd

| Gene          | Genetic Manipulation                                                                 | TDP-43 Phenotype                          | Proteostasis                                                                                              | Species         | References                     |
|---------------|-------------------------------------------------------------------------------------|------------------------------------------|----------------------------------------------------------------------------------------------------------|-----------------|--------------------------------|
| N.D.          | HRE forms                                                                           | G-quadruplex that impairs RNA polymerase II activity and leads to the accumulation of short incomplete transcripts that bind nucleolar proteins to form aggregates | Human           | Haeusler et al. (2014)        |
| C9orf72       | Transgenic Drosophila expressing HRE; iPSC-derived from C9orf72 ALS patients            | Mislocalization of TDP-43 in the cytoplasm is observed in mutant Drosophila as well as iPSCs | Formation of RanGAP 1 and ubiquitin-containing aggregates in the cytoplasm                                  | Drosophila and Human | Zhang, Donnelly, et al. (2015) |
|               | N.D.                                                                                |                                           | Impairment of nuclear import machinery and formation of RNA foci in the nucleus                             | Drosophila and Human | Freibaum et al. (2015)        |
|               | Transgenic Drosophila expressing HRE                                                 | N.D.                                     | Formation of DPR aggregates correlate with neurodegeneration                                                | Drosophila      | Mizielinska et al. (2014), Freibaum et al. (2015), and Tran et al. (2015) |
|               | Transgenic mice expressing HRE                                                      | Phosphorylated TDP-43 cytoplasmic inclusions are observed                                      | Formation of intranuclear RNA foci and DPR aggregates                                                     | Mouse            | Chew et al. (2015)            |
|               | Transgenic yeast expressing HRE                                                     | N.D.                                     | Formation of arginine-rich DPRs are toxic in yeast                                                         | Yeast           | Jovicic et al. (2015)         |

| IBMPFD patients with VCP variant R155H (brain) | Accumulations of insoluble, phosphorylated TDP-43 | Accumulations of TDP-43 colocalized with ubiquitin | Human | Neumann et al. (2007) |
| --- | --- | --- | --- | --- |
| IBMPFD patients with VCP variant R155H N387H (muscle) | TDP-43 is present as large inclusions in muscle cytoplasm | TDP-43 inclusions colocalize with ubiquitin | Human | Weihl et al. (2008) |
| IBMPFD patients with VCP variant R155H (brain) | TDP-43 redistributes to cytoplasmic inclusions from the nucleus | N.D. | Human | Johnson et al. (2010) |
| Frontal cortex of IBMPFD patients | TDP-43 redistributes to cytoplasm and some nuclei show dense inclusions | N.D. | Human | Ritson et al. (2010) |
| Cold-sensitive cdc 48-1 (yeast ortholog of VCP) mutant strain | N.D. | Required in ubiquitin-proteasome degradation | Yeast | Dai and Li (2001) |
| siRNA knockdown of VCP | N.D. | Accumulation of high-molecular-weight conjugates of ubiquitin | Drosophila S2 and mammalian cell line | Wojcik, Yano, and DeMartino (2004) |

Table 2 Key Papers Discussed in This Review—cont'd

| Gene | Genetic Manipulation | TDP-43 Phenotype | Proteostasis | Species | References |
|------|-----------------------|------------------|--------------|---------|------------|
|      | siRNA knockdown or    | TDP-43 mislocalizes to cytoplasm | Protein aggregation caused by autophagy degradation defect | Mouse skeletal muscle and mammalian cell line | Ju et al. (2009) |
|      | ectopic expression of IBMPFD mutant VCP (R155H, A232E, or E587Q) |  |  |  |  |
|      | siRNA knockdown of VCP and expression of dominant-negative or disease-associated VCP variants (R155HorA232E) | N.D. | Accumulation of immature autophagic vesicles | Mammalian cell line | Tresse et al. (2010) |
| VCP  | Transgenic mice expressing VCP/p 97 mutations (R155HorA232E) | Cytoplasmic accumulation of TDP-43 | TDP-43 accumulations are also ubiquitin positive | Mouse | Custer, Neumann, Lu, Wright, and Taylor (2010) |
|      | Mouse primary cortical neurons expressing mutant VCP (R155HorA232E) | Cytoplasmic accumulation of TDP-43 | N.D. | Mouse | Ritson et al. (2010) |
|      | Knock-in mice expressing VCP/p 97 variant R155H | Cytoplasmic accumulation of TDP-43 | Ubiquitin-positive inclusion bodies, and increased LC3-II staining | Mouse | Nalbandian et al. (2013) |
|      |  | TDP-43 positive cytosolic inclusions | Ubiquitin and SQSTM1/p62 aggregates | Mouse | Yin et al. (2012) |

| Myoblasts derived from patients with IBMPFD carrying R155HorR155S mutations | N.D. | Accumulation of numerous, large LAMP-1-and LAMP-2-positive vacuoles and accumulation of LC3-II | Human | Tresse et al. (2010) |
| --- | --- | --- | --- | --- |
| Expression of human Cdc48 variant (D592N) | N.D. | Impaired 20S proteasome binding and proteolytic communication | Thermoplasma acidophilum | Barthelme, Jauregui, and Sauer (2015) |
| Patients with UBQLN2 variant (P497L) | TDP-43-immunoreactive inclusions | Ubiquilin-2-positive inclusions | Human | Fahed et al. (2014) |
| Patients with UBQLN2 variants (P497HP506T) | Inclusions that are positive for TDP-43 | Inclusions positive for Ubiquilin-2, ubiquitin, and p62 | Human | Deng et al. (2011) |
| Patient with UBQLN2 (T487I) variant | Frequent colocalization of TDP-43 with Ubiquilin 2 | Colocalization of Ubiquilin-2 with ubiquitin in all neuronal inclusions | Human | Williams et al. (2012) |
| Endogenous Ubiquilin | N.D. | Ubiquilin is present in autophagosomes and binds to the autophagosome marker LC3 | Mouse and mammalian cell line | Rothenberg et al. (2010) |
| siRNA knockdown | N.D. | Stabilizes ERAD substrates | Mammalian cell line | Kim, Kim, Yoon, and Lim et al. (2008) and Lim et al. (2009) |

Table 2 Key Papers Discussed in This Review—cont'd

| Gene          | Genetic Manipulation                     | TDP-43 Phenotype | Proteostasis                                      | Species       | References                          |
|---------------|------------------------------------------|------------------|---------------------------------------------------|---------------|--------------------------------------|
|               | siRNA knockdown of UBQLN1 or UBQLN 2     | N.D.             | Inhibit autophagosome formation and delay the delivery of autophagosomes to lysosomes | Mammalian cell line | N'Diaye et al. (2009) and Rothenberg et al. (2010) |
|               | siRNA knockdown of UBQLN                 | N.D.             | Induction of the ER stress reporter, accumulation of polyubiquited proteins | C. elegans    | Lim et al. (2009)                   |
| UBQLN 1/2     | Overexpression or siRNA knockdown of UBQLN2 | N.D.             | Accumulation of ERAD substrates                    | Mammalian cell line | Xia et al. (2014)                  |
|               | Ectopic expression of Flag-hUbiquilin-1 (E54D) | Reduces the level of endogenous TDP-43 | N.D.             | Mammalian cell line | Gonzalez-Perez et al. (2012)        |
|               | Coexpression of Flag-hUbiquilin-1 (E54D) and TDP-43-EGFP | Redistributes TDP-43 to the cytoplasm | N.D.             | Mammalian cell line | Gonzalez-Perez et al. (2012)        |
|               | Ectopic expression of UAS-HA-hUBQLN1 (E54D) | Reduces the level of endogenous TDP-43 | N.D.             | Drosophila      | Gonzalez-Perez et al. (2012)        |

| Coexpression of WT or variant (P497H) Ubiquilin-2 with C-terminal fragment of TDP 43 | TDP 43 inclusions are colocalized with wild-type and mutant (P497H) Ubiquilin-2 | N.D. | Mammalian cell line | Deng et al. (2011) |
| --- | --- | --- | --- | --- |
| Ectopic expression of UBQLN 2 variants (P497HorP506T) | N.D. | Ubiquitin-mediated impairment of proteasomal degradation | Mammalian cell line | Deng et al. (2011) |
| Ectopic expression of UBQLN variants (P497H, P497S, P506T, P509S, orP525S) | N.D. | Weaker binding to the S5a subunit of the proteasome | Mammalian cell line | Chang and Monteiro (2015) |
| UBQLN 1/2 Overexpression of UBQLN-1/2 | N.D. | Suppress starvation-induced cell death | Mammalian cell line | N'Diaye et al. (2009) |
| Overexpression of UBQLN-1/2 | N.D. | Reduce starvation-induced cell death caused by overexpression of presenilin-2 proteins | Mammalian cell line | Rothenberg et al. (2010) |
| Overexpression of UBQLN-1 | Potentiates TDP-43 aggregate formation in the cytoplasm | Recruits TDP-43 to detergent-resistant cytoplasmic aggregates that colocalize with autophagosomal markers | Mammalian cell line | Kim et al. (2009) |

Table 2 Key Papers Discussed in This Review—cont'd

| Gene       | Genetic Manipulation                                                                 | TDP-43 Phenotype                                                                                     | Proteostasis                                                                                          | Species               | References                                      |
|------------|-------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|-----------------------|--------------------------------------------------|
| UBQLN 1/2  | Coexpression of HA-TDP-43 and Myc-Ubiquilin-1                                         | Physically interacts with TDP-43 through the C-terminal UBA domain of Ubiquilin-1                      | N.D.                                                                                                 | Yeast and mammalian cell line | Kim et al. (2009)                              |
|            | Coexpression of human UBQLN and TDP-43                                               | Reduced eclosion rate as well as shortened lifespan caused by TDP-43 overexpression                  | N.D.                                                                                                 | Drosophila            | Hanson, Kim, Wassarman, and Tibbetts (2010)     |
| VAPB       | VAPB null mutation                                                                  | N.D.                                                                                                | Aberrant ER expansion, accumulation of membrane proteins on the ER and alternatively spliced XBP-1         | Drosophila            | Moustaqim-Barrette et al. (2013)                |
|            | Ectopic expression of VAPB (mammalian P56S or Drosophila P58S) variant                 | N.D.                                                                                                | Formation of protein aggregates, ER stress, and traps endogenous wild-type VAPB proteins                   | Drosophila and mammalian cell line | Teuling et al. (2007) and Tsuda et al. (2008) |
|            | VAPB(P56S) transgenic mice                                                          | N.D.                                                                                                | Mutant VAPB proteins accumulate in large puncta and significant increase in ER stress                    | Mouse                  | Aliaga et al. (2013)                            |

| Knock-in mice carrying the VAPB(P56S) variant | N.D. | Accumulation of ubiquitinated proteins in cytoplasmic inclusions that colocalize with autophagosome markers | Mouse | Larroquette et al. (2015) |
| --- | --- | --- | --- | --- |
| SOD-1 patients | N.D. | SOD-1 patients show elevated ER stress | Human | Atkin et al. (2008) and Ilieva et al. (2007) |
| Transgenic mouse lines expressing disease-related SOD-1(G93A; G85R, orG37R) | N.D. | Formation of SOD-1 mutant containing protein aggregates in all transgenic lines | Mouse | Bruijn et al. (1997, 1998), Cleveland (1999) (review), Gurney et al. (1994), Julien and Kriz (2006) (review), and Wong et al. (1995) |
| Transgenic Drosophila lines expressing disease-related SOD-1 variants (A4VorG85R) | N.D. | Accumulation of SOD-1 aggregates with age in motor neurons | Drosophila | Watson, Lagow, Xu, Zhang, and Bonini (2008) |
| Ectopic expression of wild-type or disease-related SOD-1(A4V; G85R, orG93A) | N.D. | Mutant but not wild-type SOD-1 forms in cultured cells promote ER stress | Mammalian cell lines | Nishitoh et al. (2008), Oh, Shin, Yuan, and Kang (2008), and Tobisawa et al. (2003) |
| Transgenic mouse lines expressing disease-related SOD-1(G93A orG85R) | N.D. | Transgenic mice expressing mutant SOD-1 exhibits elevated ER stress | Mouse | Atkin et al. (2006), Kikuchi et al. (2006), Saxena, Cabuy, and Caroni (2009), Tobisawa et al. (2003), and Wate et al. (2005) |

Table 2 Key Papers Discussed in This Review—cont'd

| Gene | Genetic Manipulation | TDP-43 Phenotype | Proteostasis | Species | References |
|------|-----------------------|------------------|--------------|---------|------------|
| SOD-1 | Transgenic mouse lines expressing disease-related SOD-1 (G85R) | N.D. | Remove one copy of PERK in SOD-1 (G85R) mice accelerates aggregate formation, shorten lifespan, and enhances degeneration phenotypes | Mouse | Wang, Popko, and Roos (2011) |
|      | Transgenic mouse lines expressing disease-related SOD-1 (G85R) | N.D. | Genetic or pharmacological activation of PERK ameliorates mutant SOD-1-induced ALS phenotypes | Mouse | Wang, Popko, and Roos (2014), Das et al. (2015), and Wang, Popko, Tixier, and Roos (2014) |
|      |                      | N.D. | Inactivation of the IRE1 pathway reduces the accumulation of mutant SOD-1 in spinal cord, prolongs lifespan and alleviates degeneration phenotypes | Mouse | Hetz et al. (2009) |

a Not determined.

b Since FUS and TDP-43 may function in the same pathway, we summarize FUS phenotypes in this section.

However, most of the isolated genes are poorly characterized, and relatively little is known about their in vivo function. Yeast as well as animal models like mice, flies, and worms are valuable to help define the function of these proteins. These studies are typically based on loss-of-function alleles as well as expression of mutant alleles in the null or wild-type background. Results from these studies are very important as they may affect the design of therapeutic strategies. For example, if the disease-causing variant results in loss of function, then an overexpression strategy may be advised; but if the variant results in gain of function, then suppressing gene function may be advised. In addition to examine individual gene functions, animal models also allow the elucidation of genetic interactions between disease-causing genes in controlled experiments. Delineating the genetic interactions between different disease-causing genes allows elucidation of the pathogenic mechanisms. Here, we will summarize the salient features that contributed to the model outlined earlier and describe some of the contribution of *Drosophila* research to the field. We only cover seven of the ALS-causing loci because they cover the vast majority of the FALS cases and because they have been studied to some extent in *Drosophila*.

---

### 3. RNA PROCESSING, SPLICING, RNA FOCI, AND PROTEIN AGGREGATION

A general pathological feature of neurodegenerative diseases is the presence of insoluble protein aggregates that are often abnormally phosphorylated and ubiquitinated (Ling et al., 2013). A fundamental question is how the formation of these insoluble protein aggregates contributes to the pathogenesis of ALS. Interestingly, several of the ALS genes identified so far encode DNA/RNA-binding proteins, including TDP-43, FUS, and hnRNPs (Peters, Ghasemi, et al., 2015). More recently, HRE in *C9orf72* was found to cause RNA foci and affect RNA metabolism (Donnelly et al., 2013; Haeusler et al., 2014). Therefore, the formation of protein aggregates abolishing normal RNA splicing may play an important role in ALS pathogenesis. In this section, we will discuss the connection between protein aggregates, RNA metabolism, and ALS pathogenesis with a special emphasis on TDP-43, FUS, and *C9orf72*.

#### 3.1 TDP-43

TAR DNA-binding protein (TDP-43) is a 43 kDa RNA/DNA-binding protein encoded by *TARDBP* gene that is evolutionarily conserved
(Buratti et al., 2001; Ou, Wu, Harrich, Garcia-Martinez, & Gaynor, 1995). TDP-43 is often found in pathological inclusions in spinal cord motor neurons, hippocampal, and frontal cortex neurons as well as glial cells in SALS and FALS patients that are not caused by SOD-1 variants (Blokhus et al., 2013). Variants in TARDBP (also known as ALS10) that affect TDP-43 have been reported in ~4% of FALS and a few SALS cases (Renton, Chio, & Traynor, 2014). TDP-43 is normally localized to the nucleus but in ALS patients, the protein is present in the cytoplasm and excluded from the nucleus (Neumann et al., 2006). These observations suggest that TDP-43 plays an important role in the development of ALS.

Loss of TDP-43 in flies causes semilethality (Feiguin et al., 2009). Motor neuronal expression of either human or *Drosophila* TDP-43 in the TDP-43 null mutant prolongs life span, suggesting a functional conservation of TDP-43 between *Drosophila* and human (Feiguin et al., 2009). In addition, loss of TDP-43 in fly larvae reduced motility and decreased boutons at the neuromuscular junctions (NMJs) (Feiguin et al., 2009). Moreover, neuronal overexpression of the wild-type human transgene causes a decrease in bouton and branch number associated with protein aggregates (Li et al., 2010), suggesting that loss and gain of TDP-43 function affects NMJ morphology.

In 90% of all ALS patients, TDP-43 is absent from the nucleus and present in ubiquitinated cytoplasmic aggregates (Ling et al., 2013). Ectopic expression of a C-terminal fragment TDP-43 in cell lines causes protein aggregations (Zhang, Gui, et al., 2009), suggesting that TDP-43 is physically prone to aggregate formation. Moreover, overexpression of the human wild-type or mutant TDP-43 in *Drosophila* or mice leads to locomotor defects (Estes et al., 2011; Li et al., 2010; Lin et al., 2011; Voigt et al., 2010; Wegorzewska et al., 2009). These data indicate that TDP-43 aggregates may play an important role in ALS pathology. A predominant marker of TDP-43 aggregates is the presence of hyperphosphorylated TDP-43 as well as the presence of the N-terminal truncated 25 kDa TDP-43 (will be referred to as TDP-25) in cytoplasmic inclusions (Arai et al., 2010; Neumann et al., 2006). TDP-43 is hyperphosphorylated at several serine residues, including S409/410 (Hasegawa et al., 2008; Neumann et al., 2009). Mutating the S409/410 to phosphomimetic aspartic acid residues greatly reduces protein aggregation (Brady et al., 2011; Li et al., 2011), suggesting that the hyperphosphorylation on these two sites alleviates TDP-43 proteinopathy. TDP-25 forms cytoplasmic protein aggregates when expressed in the cell lines (Yang et al., 2010). Similarly, expressionof TDP-25 in *Drosophila* eyes causes a rough eye phenotype suggesting that the toxicity is associated with TDP-25 (Gregory et al., 2012). However, its toxicity is milder than the wild-type human TDP-43 (Gregory et al., 2012). Moreover, the authors show that TDP-25 is more susceptible to protein degradation (Gregory et al., 2012). This may contribute to the reduction of its toxicity. Furthermore, coexpression of chaperones in yeast as well as *Drosophila* suppresses TDP-43 aggregation, cytoplasmic localization, and toxicity (Estes et al., 2011; Gregory et al., 2012; Jackrel et al., 2014), suggesting that promoting protein refolding and thereby reducing the burden of proteasomal degradation is beneficial to ALS.

To assess the correlation between TDP-43 nuclear localization and degeneration, Miguel, Frebourg, Campion, and Lecourtois (2011) expressed wild type or mutant human TDP-43 that lacks either the nuclear localization sequence (NLS) or nuclear export sequence (NES) in *Drosophila* eyes and examined their ability to cause protein aggregations and eye degeneration. Only mutant TDP-43 lacking NLS caused severe eye degeneration. Interestingly, human TDP-43 proteins expressed in adult fly neurons are abnormally phosphorylated on the disease-specific sites, Ser409/Ser410 (Miguel et al., 2011). This study highlights the importance of TDP-43 mislocalization in the induction of TDP-43 proteinopathy. Note that TDP-43 aggregation is not commonly observed in mouse models of TDP-43 proteinopathy (Choksi et al., 2014; Gendron & Petrucelli, 2011). Hence, *Drosophila* provides a platform to study TDP-43 proteinopathy.

TDP-43 is a member of the heterogeneous nuclear ribonucleoprotein (hnRNP) family of proteins (Buratti & Baralle, 2008) and affects pre-mRNA splicing, transcription, mRNA stability, and mRNA transport (Buratti & Baralle, 2008). TDP-43 and other members of the hnRNP family bind RNA, but the RNA-binding specificity of hnRNPs varies substantially, ranging from nonspecific to highly defined RNA sequences. TDP-43 specifically binds to a long UG-repeat sequence in RNA (Ayala et al., 2005; Buratti et al., 2001) via two RNA recognition motifs (Lukavsky et al., 2013) and regulates alternative splicing of mRNA including several human disease-related genes (Bose, Wang, Hung, Tarn, & Shen, 2008; Buratti & Baralle, 2001; Buratti et al., 2001; Mercado, Ayala, Romano, Buratti, & Baralle, 2005). In an in vitro splicing assay, the *Drosophila* TDP-43 shows comparable ability to recognize and splice an UG-rich substrate with human TDP-43 (Ayala et al., 2005), suggesting that the pre-mRNA splicing activity of TDP-43 is conserved in *Drosophila*. Neuronal expression of a human TDP-43 lacking RNA-binding activity in *Drosophila* reverses life span
reductions and locomotor defects caused by the expression of human TDP-43 harboring ALS-linked pathogenic variants (Voigt et al., 2010), indicating that the RNA-binding activity of TDP-43 may be important for ALS pathology.

Cytoplasmic inclusion and nuclear exclusion of TDP-43 are a hallmark of ALS. Studies in cells suggest that TDP-43 is an important regulator of pre-mRNA splicing. However, how TDP-43 leads to the formation of aggregates in ALS patients and how its pre-mRNA splicing activity is involved in ALS pathogenesis remains to be established. A recent publication by Ling et al. (2015) provides an important hint on how these two phenomena maybe interconnected. By using CreER-inducible TDP-43 knockout mouse embryonic stem cells and high-resolution RNA-sequencing (RNA-seq) technology, Ling et al. (2015) found that nonconserved cryptic exons were not spliced out with loss of TDP-43. The incorporation of cryptic exons in mature mRNA disrupts translation and produces numerous aberrant proteins (Fig. 1A). Interestingly, suppression of the incorporation of cryptic exons prevents cell death in TDP-43 knockout cells. These data suggest that the chronic production of aberrant proteins may overload the protein degradation machinery.

These findings raise the question whether there is a few or a group of conserved TDP-43-binding exons that are responsible for the disease. Interestingly, while all TDP-43 targets share the UG-rich repeat consistent with its binding preference in vitro, TDP-43 is associated with a subset of cryptic exons that is totally different in mice vs human (Ling et al., 2015). Taken together, these data indicate that there is probably not a specific TDP-43 target responsible for the degenerative phenotypes. Rather, loss of TDP-43 may affect numerous genes containing UG-rich repeats and the incorporation of cryptic exons produces numerous aberrant proteins, eventually leading to the formation of protein aggregates that overload the protein removal machinery.

## 3.2 FUS

FUS (fused in sarcoma) was first reported to be an oncoprotein in liposarcomas associated with a *t*(12; 16) chromosomal translocation (Crozat, Aman, Mandahl, & Ron, 1993; Rabbits, Forster, Larson, & Nathan, 1993). It is a member of a protein family containing a DNA/RNA-binding domain (Tan & Manley, 2009). FUS regulates RNA processing, pre-mRNA splicing and mRNA trafficking (Deng, Gao, & Jankovic, 2014). Variants in *FUS* (also known as *ALS6*) lead to the formation

ALS and Proteostasis

A
1 2 X 3 4
TDP-43
TDP-43 suppresses
the incorporation of cryptic exon
↓
1 2 3 4
Mature protein
Depletion of TDP-43
from nucleus
↓
1 2 X 3 4
TDP-43 suppresses
the incorporation of cryptic exon
↓
1 2 X 3 4
Aberrant protein
Chaperones
↓
Overload the protein refolding and
degradation machinery
↓
Degeneration

B
5' UTR of C9orf72
Exon 1a Exon 1b Exon 2 →
ATG
C9orf72 HRE
HRE interferes with
transcription
→
Abortive HRE-containing RNA
RAN translation
{G R}ₙ or {P R}ₙ
DPRs
Protein/RNA
aggregates
↓
Nuclear transport
defect
↓
Depletion of TDP-43
from nucleus

Fig. 1 A model for the involvement of RNA metabolism in the formation of protein/RNA aggregates and degeneration. (A) Variants in TDP-43 or C9orf72 cause TDP-43 exclusion from the nucleus. This disrupts the function of TDP-43 to suppress the incorporation of the cryptic exon to mRNAs and leads to the synthesis of misfolded proteins that overload the protein refolding and degradation machinery. This results in a chronic accumulation of protein aggregates and degeneration. The enhancement of chaperone function may relieve protein aggregation and suppress degeneration. (B) The insertion of HREs in the 5' noncoding intron of C9orf72 gene interfere with transcription and lead to the generation of HRE-containing abortive RNAs. These aberrant RNAs result in the formation of protein/RNA aggregates or DPRs via RAN translation which in turn cause defects in the nuclear transport machinery and promote the accumulation of nuclear proteins including TDP-43 in the cytoplasm.
of FUS-positive inclusions in motor neurons in 4–5% of all FALS (Kwiatkowski et al., 2009; Vance et al., 2009) and 1% of SALS patients (Belzil et al., 2009; Deng, Gao, et al., 2014; Rademakers et al., 2010). The homolog of FUS in *Drosophila* corresponds to *caz* (Stolow & Haynes, 1995). Loss of FUS in *Drosophila* causes reduced eclosion rates, a shorter life span, and defects in locomotion (Wang, Brent, Tomlinson, Shneider, & McCabe, 2011; Xia et al., 2012). Neuronal expression of fly or human FUS cDNA rescues these defects, suggesting a conservation of function (Wang, Brent, et al., 2011). Mice with a genetic knockout of FUS die 16 hours after birth (Hicks et al., 2000), but in an independent knockout line with a different genetic background is viable with only a reduction in body weight and loss of fertility in males (Kuroda et al., 2000). Nevertheless, neither of them displays obvious neurological defects, suggesting that *Drosophila* is a better model to study FUS-related ALS.

The pathology associated with FUS pathogenic variants is characterized by the presence of FUS immunoreactive inclusion in the cytoplasm of neurons and glia (Deng, Gao, et al., 2014). In addition to ALS, FUS-containing inclusions have also been detected in disorders including frontotemporal dementia (FTD) and polyglutamine diseases, such as Huntington’s chorea, spinocerebellar ataxia types 1–3 and dentatorubral-pallidoluysian atrophy (Deng, Gao, et al., 2014). FUS contains a NLS and is predominantly localized to the nucleus. Expression of the ALS-linked variants affecting the NLS (R518K, R521C, R521H, R524S, and R525L) in *Drosophila* eyes causes age- and dosage-dependent degenerations (Lanson et al., 2011). Interestingly, removal of the nuclear export sequence (NES) from these mutants rescues the eye degeneration phenotypes (Lanson et al., 2011). These findings support the argument that cytoplasmic localization of FUS plays an important role in the development of the degenerative phenotypes.

Similar to TDP-43, several FUS modifications, including phosphorylation and methylation, have been associated with the FUS-containing inclusions. Phosphorylation of the N-terminus of FUS by the DNA-dependent protein kinase (DNA-PK) contributes to the formation of inclusions (Deng, Holler, et al., 2014) and phosphorylation of the C-terminal tyrosine 526 impairs its nuclear import (Darovic et al., 2015). Arginine methylation is a common posttranslational modification modulating its affinity to transportin 1 and controls nuclear transport (Dormann & Haass, 2013). FUS is methylated at a site near the NLS by arginine methyltransferase 1 and 8 – (PRMT1 and PRMT8) (Dormann et al., 2012; Scaramuzzino et al., 2013). In *Drosophila* photoreceptors, a reduction of PRMT protein, Dart1, leads toa decrease in methylation of the ectopically expressed human FUS P525L and promotes degeneration caused by FUS P525L (Jackel et al., 2015). This finding suggests that arginine methylation of FUS promotes the toxicity of ALS-linked FUS variants.

FUS dysfunction alters the splicing of several genes encoding proteins forming pathological inclusions, suggesting that the pathogenic variants create stresses on protein degradation. For example, loss of FUS causes the aggregation of mis-spliced Tau protein in neurons (Orozco & Edbauer, 2013; Orozco et al., 2012). Indeed, ectopic expression of human FUS carrying ALS-causing variants (R518K or R521C) in *Drosophila* causes degeneration in eyes, brains, and motor neurons (Daigle et al., 2013). The introduction of point mutations that disrupt the RNA-binding ability of these FUS transgene constructs prevents the incorporation of FUS in these inclusions and reduces degeneration (Daigle et al., 2013). These data suggest that FUS RNA-binding activity may contribute to the formation of protein aggregates and cause degeneration. In addition, FUS is also present in the Drosha complex, an RNase III enzyme required for microRNA biogenesis (Gregory et al., 2004), suggesting that loss of FUS might disrupt microRNA biogenesis and contribute to the pathogenesis of ALS6. Unlike TDP-43 which shows a high binding specificity to the UG-rich sequence, FUS shows little to no binding preference to specific RNA sequences (Wang, Schwartz, & Cech, 2015). Hence, FUS dysfunction may result in a broader pathology than TDP-43 dysfunction.

TDP-43 and FUS not only share similar functions in pre-mRNA splicing, but are also physically and functionally interconnected, as suggested by genetic and biochemical studies in *Drosophila* and cell culture (Wang, Brent, et al., 2011). Neuronal expression of FUS in a TDP-43 null background suppresses lethality, locomotion defects, and the reduced bouton number caused by loss of TDP-43 (Wang, Brent, et al., 2011). Moreover, coexpression of either wild type or the ALS-causing TDP-43 variant (M337V) enhances eye degeneration caused by either wild type or variant (R521H) FUS (Lanson et al., 2011). These findings collectively argue that FUS and TDP-43 may function in the same pathway.

In summary, the formation of cytoplasmic FUS aggregates is likely to be important in the pathogenesis of FUS variant-linked ALS. The role of FUS in the pathogenesis may resemble that of TDP-43. Loss of FUS may cause aberrant splicing of its target genes that eventually leads to a burden of aberrant proteins that are ineffectively removed in neurons and cause degeneration.
3.3 C9orf72

The HRE, (GGGGCC)ₙ, in the noncoding region of *C9orf72* was recently linked to ALS and FTD (DeJesus-Hernandez et al., 2011; Renton et al., 2011). It accounts for 40% of FALS and 8% of SALS (DeJesus-Hernandez et al., 2011; Majounie et al., 2012; Renton et al., 2011). Cytoplasmic aggregates that contain TDP-43, FUS, and Tau are a common observation in motor neurons in *C9orf72*-associated ALS cases (Fratta et al., 2013; Rohrer et al., 2015; Stewart et al., 2012), implicating that there is an important connection between *C9orf72* HRE and protein cytoplasmic aggregates in ALS pathophysiology. In addition to ALS, HRE insertion in the *C9orf72* gene has also been widely associated with FTD. FTD is a type of dementia that results in a progressive degeneration of the frontal and/or temporal lobes of the brain. Although ALS and FTD are distinct disorders, accumulating evidence suggest that they may share common neurodegenerative pathways and may be part of a spectrum. Indeed, TDP-43 or FUS-containing protein aggregates are usually observed in FTD. In addition, ALS-causing genetic loci such as *C9orf72* and VCP have also been connected with FTD (Ferrari, Kapogiannis, Huey, & Momeni, 2013).

Three models may underlie the pathogenesis of *C9orf72*-mediated ALS: (1) HRE interferes with *C9orf72* protein level and loss *C9orf72* protein mediates pathogenesis. (2) HRE forms a toxic RNA structure that binds and sequesters RNA-binding proteins (RBPs). (3) HRE is translated into toxic DPRs via repeat-associated and non-ATG translation (RAN translation, or RANT). Both models (2) and (3) lead to protein aggregates that deplete regulators of nucleocytoplasmic transport and contribute to degeneration. The data related to *C9orf72* loss of expression in pathogenesis is ambiguous. Most studies agree that *C9orf72* mRNA and protein expression is lower in brain tissue of C9-ALS patients or in induced pluripotent stem (iPS) neurons derived from C9-ALS/FTD patients (Ciura et al., 2013; Donnelly et al., 2013; Waite et al., 2014). Moreover, loss of the Caenorhabditis elegans *C9orf72* ortholog, *alfa-1*, causes age-dependent locomotion defects resulting in paralysis as well as the specific degeneration of GABAergic motor neurons (Therrien, Rouleau, Dion, & Parker, 2013). In addition, *C9orf72* knockdown in zebrafish embryos causes axon degeneration of motor neurons (Ciura et al., 2013). However, in a rare homozygous *C9orf72* patient, the clinical or pathological features are not worse than the dominant cases (Fratta et al., 2013). Whole body knockout of *C9orf72* causes a robust immunological defect, but no motor defects were

observed (Atanasio et al., 2016; Jiang et al., 2016; Sudria-Lopez et al., 2016). In addition, selective deletion of C9orf72 in neurons and glia in mice causes reduced body weight but does not induce motor neuron loss, or affect survival, and no mislocalization of TDP-43 was observed (Koppers et al., 2015). Hence, loss of C9orf72 may not play a key role in ALS pathogenesis.

C9-ALS/FTD patients exhibit RNA foci in the motor cortex and spinal cord (Donnelly et al., 2013). Antisense oligonucleotides targeting HRE suppress HRE RNA foci-induced cytotoxicity in C9orf72 iPSC neurons (Donnelly et al., 2013), suggesting that HRE RNA underlies the pathogenesis of HRE-linked ALS. The HRE RNA or the single-stranded DNA formed during transcription has a high tendency to form a G-quadruplex, a stack of hydrogen-bonded guanine tetramers (Haeusler et al., 2014). During transcription, the single-stranded G-quadruplex DNA impairs RNA polymerase II elongation and leads to the accumulation of abortive transcripts (the RNA G-quadruplexes). The abortive transcripts bind and sequester RBPs (Donnelly et al., 2013), which further lead to aggregation of ribonucleoproteins and nucleolar stress (Haeusler et al., 2014). Nineteen RBPs were identified that exhibit a higher binding affinity to HREs in a protein-array analysis (Donnelly et al., 2013); among them RanGAP1 (Drosophila: RanGAP) was identified as a suppressor of HRE-mediated toxicity from an RNAi screen in Drosophila using eye degeneration as a phenotypic readout (Zhang, Donnelly, et al., 2015). RanGAP (Ran GTPase-activating protein) is a master regulator of nucleocytoplasmic transport, suggesting that this process is involved in HRE-induced pathogenesis. Indeed, either knockdown of Exportin proteins or overexpression of Importin proteins that promotes nuclear import suppresses HRE-mediated eye degeneration. In addition, RanGAP/RanGAP1 was observed to interact and colocalize with RNA foci in Drosophila S2 cells and/or iPSCs derived from C9orf72-linked ALS patients. Interestingly, feeding the HRE-expressing flies antisense oligonucleotides or small inhibitors that target HRE suppresses nucleocytoplasmic transport defects as well as the degenerative phenotypes in Drosophila eyes, providing further support that nucleocytoplasmic transport mediates HRE-induced toxicity. Moreover, TDP-43 is observed to be mislocalized to the cytoplasm in HRE-expressing flies, iPSC neurons, transgenic mice that express HRE (Chew et al., 2015; Zhang, Donnelly, et al., 2015) as well as tissue samples from C9orf72 patients (Cooper-Knock et al., 2015), suggesting that TDP-43 mislocalization may play a role in C9orf72-linked ALS. Taken together, these data support a model in which the HRE binds and recruits RBPs such as RanGAP to
the RNA foci, leading to their mislocalization and loss of function. These defects in nucleocytoplasmic transportation cause cytoplasmic mis-localization and accumulation of nuclear proteins including TDP-43 that contributes to the pathogenesis.

The third model argues that DPR expressed by the repeat-associated non-ATG (RAN) translation of HRE causes neuronal toxicity ([Cleary & Ranum, 2013; Zu et al., 2011]). RAN translation can occur in all three reading frames independent of an ATG start codon, generating six different types of DPRs in total. However, since poly(GP) can be translated from either direction, only five types, poly-GA, GP, GR, PA, and PR, are generated. These DPRs are highly prone to aggregate formation and have been shown to accumulate in the cerebellum of *C9orf72* patients ([Ash et al., 2013; Mori et al., 2013; Zu et al., 2013]). However, recent observations suggest that DPRs are rarely present in the spinal cords or motor neurons of *C9orf72*-linked ALS patients ([Gomez-Deza et al., 2015]). Hence, the neurotoxicity of the DPRs is still under debate. In *Drosophila*, ectopic expression of HRE in *Drosophila* eyes under control of the GAL4/UAS system causes degeneration ([Mizielinska et al., 2014]). Introduction of stop codons that block RAN translation in all six reading frames without affecting the circular dichromatic property of the HRE RNA suppresses the HRE-induced eye degeneration, suggesting that DPRs, but not the HRE RNA, are toxic. Moreover, [Mizielinska et al. (2014)] generated “protein-only” constructs using alternative codons to individually express four major forms of DPRs, poly(GR), poly(PR), poly(PA), or poly(GA) encoded by the HRE. Only the arginine-containing DPRs, poly(GR) and poly(PR), caused eye degeneration, indicating that some DPRs but not the G-quadruplex RNA lead to eye degeneration ([Mizielinska et al., 2014]). [Freibaum et al. (2015)] confirmed these results in an independent line of HRE-expressing transgenic flies. Moreover, they found that expression of HRE cause locomotion defects and reduced bouton numbers at NMJs in larvae ([Freibaum et al., 2015]). They further performed a genetic modifier screen that covers ~80% of the *Drosophila* genome to identify genetic loci whose partial loss of function modified the HRE-induced eye degeneration. They discovered that loss of proteins that play a role in nuclear import either enhance or suppress HRE-induced eye degeneration, suggesting that nuclear import is compromised. Moreover, they identify a nuclear export protein as a suppressor of HRE-induced eye degeneration, suggesting that nuclear export is also important. In agreement with this, nuclear RNA is accumulated in mammalian cell lines and in iPSCs derived from C9 patients. In addition,Freibbaum et al. (2015) also observed morphological disruption of nuclear envelope and RNA foci in the nuclei of *Drosophila* salivary glands. As low levels of DPRs were observed in this study, the DPRs may be involved in C9orf72 toxicity. The DPR toxicity model is further supported recently by Jovicic et al. (2015). They reported that arginine-rich DPRs, expressed via alternative codons that express the DPR but do not form G-quadruplex, are toxic in yeast. Through genome-wide complementary screens, they also found an enrichment of modifiers functioning in nucleocytoplasmic transport (Jovicic et al., 2015). Moreover, an independent study in which HRE is expressed in *Drosophila* supports the DPR-toxicity model (Tran et al., 2015). Taken together, these studies converge on an effect of HRE toxicity via disrupting nucleocytoplasmic transport (Freibbaum et al., 2015; Jovicic et al., 2015; Zhang, Donnelly, et al., 2015). However, it is still not obvious whether the toxicity of HRE is derived from the G-quadruplex, DPRs, or both. Zhang, Donnelly, et al. (2015) and Zhang, Gui, et al. (2015) did not detect DPRs, poly(GR) or poly(GP), in HRE-expressing flies, arguing that G-quadruplex is toxic. Freibbaum et al. (2015) observed DPRs as well as RNA foci in HRE-expressing flies. They support the DPR toxicity model but do not rule out an involvement of G-quadruplex toxicity. Both Jovicic et al. (2015) and Tran et al. (2015) argued that it is the DPR not G-quadruplex that cause toxicity in yeast or fly, respectively.

The above findings suggest that C9orf72 loss of expression may not play an important role in ALS, indicating that the first model is unlikely. Rather, the HRE seems to be a key player in C9orf72 pathogenesis. The second model argues that HRE sequences disrupt RNA polymerase activity, leading to the formation of toxic RNA foci containing short abortive transcripts (Fig. 1B). These HRE-containing transcripts lead to the formation of protein aggregates containing proteins such as RanGAP that disrupts nuclear import machinery. One likely target is TDP-43. The aberrantly localized TDP-43 causes abnormal incorporation of cryptic exons and leads to a proteinopathy. The third model argues that arginine-containing DPRs synthesized via RAN translation leads to degeneration. The expression of the arginine-containing DPRs causes impairment in nuclear import and export and may contribute to the accumulation of protein in the cytoplasm and the formation of nuclear RNA foci, respectively. The latter are probably not toxic. It is important to note that mouse models that carry an HRE have been generated. The sense, antisense RNA foci as well as DPRs were observed in neuronal tissues including motor cortex and spinal cord (O’Rourke et al., 2015; Peters, Cabrera, et al., 2015). However, no obvious
behavioral and histological defects that indicate neurodegeneration has been observed in these mice after a 2-year period. Interestingly, the authors did not observe a TDP-43 cytoplasmic mislocalization. These data suggest that the role of the RNA foci and DPRs in C9orf72-linked degeneration is still not resolved.

---

### 4. PROTEOSTASIS DEFICIENCY IN ALS

Protein aggregation and formation of intracellular inclusions are central features of many neurodegenerative diseases. It is interesting that at least three genes that have been implicated in protein quality control at the ER are linked to ALS. These genes correspond to VCP (Johnson et al., 2010), UBQLN2 (Deng et al., 2011), and VAPB (Nishimura et al., 2004).

Accumulation of unfolded or misfolded proteins in the ER lead to ER stress as well as unfolded protein response (UPR) (Ron & Walter, 2007). UPR activation attenuates protein translation, promotes protein degradation, as well as upregulates the expression of chaperones that are targeted to the ER to refold proteins. If the proteins cannot be properly folded, the ER-associated protein degradation (ERAD) pathway is activated (Vembar & Brodsky, 2008). Misfolded proteins are ubiquitinated and degraded by the proteasome machinery or autophagy. Defects in ERAD lead to protein aggregations that are often observed in many neurodegenerative diseases, including ALS (Ling et al., 2013). Indeed, one of the most common observations in the cell body of motor neurons is protein aggregation in the cytoplasm (Al-Chalabi et al., 2012). Hence, a better understanding of mechanisms affecting proteostasis will provide better insight into ALS pathology.

#### 4.1 VCP

Valosin-containing protein (VCP)/p97 belongs to the hexameric AAA (ATPase associated with diverse cellular activities) protein family. Proteins of this family generally consume energy from ATP hydrolysis to structurally remodel their targets (Erzberger & Berger, 2006). VCP is conserved across many species and named Cdc48 in yeast and Ter94 in flies, respectively. Variants in VCP were first found to associate with IBMPFD (inclusion body myopathy associated with Paget disease of bone (PDB) and FTD), a type of inclusion body myopathy (Watts et al., 2004). IBMPFD is a dominant autosomal progressive disorder (Kimonis et al., 2000). It is characterized by an adult-onset proximal and distal muscle weakness, early onset PDB,
and FTD (Kovach et al., 2001). Moreover, TDP-43 and ubiquitin-positive cytoplasmic inclusions in brains and muscles are observed in IBMPFD patients regardless of the affected residues of VCP or clinical symptoms (Ju et al., 2009; Neumann et al., 2007; Weihl et al., 2008). Interestingly, variants in the same protein domain of VCP can cause IBMPFD as well as FALS (Johnson et al., 2010). Variants in *VCP* are responsible for 2% of FALS (also known as ALS14) cases and are inherited in a dominant manner (Johnson et al., 2010; Miller et al., 2012). Moreover, TDP-43-positive cytoplasmic inclusions are observed in the spinal cord in these patients (Johnson et al., 2010). In this section, we will discuss how variants in *VCP* cause TDP-43 proteinopathy and contribute to degeneration.

Loss of VCP in *Drosophila* is cell lethal (Leon & McKearin, 1999) and loss of VCP in mice leads to lethality prior to implantation (Muller, Deinhardt, Rosewell, Warren, & Shima, 2007). VCP has been shown to affect numerous cellular processes including membrane fusion (Latterich, Frohlich, & Schekman, 1995; Rabouille, Levine, Peters, & Warren, 1995), cell cycle progression (Cao, Nakajima, Meyer, & Zheng, 2003), and transcriptional activation (Rape et al., 2001). In addition, VCP has also been shown to regulate several proteostasis pathways including proteasome degradation (Dai & Li, 2001), ERAD (Jarosch, Geiss-Friedlander, Meusser, Walter, & Sommer, 2002; Ye, Meyer, & Rapoport, 2001), and autophagy (Ju et al., 2009). VCP consists of an N-terminal regulatory domain, two ATPase domains (D1 and D2), and a C-terminal HbYX motif (Brunger & DeLaBarre, 2003; DeLaBarre & Brunger, 2003; Stolz, Hilt, Buchberger, & Wolf, 2011). The N-terminal domain is responsible for the recognition of substrates. This interaction is achieved either through a direct binding to polyubiquitin chains of substrates (Dai & Li, 2001) or through cofactors that mediate the interaction with polyubiquitinated targets (Meyer, Wang, & Warren, 2002). The D1 and D2 ATPase domains hydrolyze ATP to generate energy for VCP to function as a segregase to structurally remodel ubiquitinated proteins. This facilitates the extraction of aberrant proteins from ER, plasma membranes, or protein aggregates for proteasome degradation (Buchberger, Schindelin, & Hanzelmann, 2015). The HbYX motif together with a pore-2 loop at the bottom of the D2 domain mediates the docking of aberrant proteins into the proteasome and facilitates the transfer of unfolded protein to the proteasome for degradation (Barthelme & Sauer, 2012, 2013). Variants in the pore-2 loop of VCP impair 20S proteasome binding and have been linked to FALS (Abramzon et al., 2012; Barthelme et al., 2015; Johnson et al., 2010). These data suggest that VCP plays a role to remodel
aberrant proteins and promote the degradation of the aberrant proteins via the proteasome system.

Interestingly, most of the variants identified in ALS or IBMPFD patients localize to the N-terminal domain or the interface between N-terminal domain and D1 domain, suggesting that substrate interaction plays a key role in the pathogenesis of VCP-linked diseases, including ALS. To date, more than 30 cofactors have been identified and most of them bind to the N-terminal domain of VCP (Barthelme & Sauer, 2016). Loss of VCP in yeast causes accumulation of ubiquitinated proteins (Dai & Li, 2001). Similarly, in *Drosophila* and human cells, suppression of VCP expression by RNAi leads to an upregulation of ubiquitinated proteins (Wojcik et al., 2004) supporting the observation from in vitro assays that VCP loss of function causes defects in proteostasis. The ectopic expression of mutant VCP with disease variants in cell lines disrupts ERAD and leads to accumulation of ubiquitinated proteins, indicating a defect in proteostasis which resembles the defect in patients with VCP variants (Ritson et al., 2010). Knock-in of the VCP(R155H/+) variant in mice, which account for 50% of IBMPFD cases, causes an accumulation of cytoplasmic TDP-43 and ubiquitin-positive inclusions (Custer et al., 2010; Nalbandian et al., 2013; Yin et al., 2012). Hence, both loss of VCP and the expression of R155H VCP variant lead to similar proteostasis defects, suggesting that VCP loss of function may be a key. Indeed, VCP variants have been shown to decrease protein stability and/or impair VCP binding with cofactors (Zhang, Gui, et al., 2015). In summary, disease-associated VCP variants may impair the interaction with cofactors that regulate degradation of ubiquitinated proteins, which in turn leads to the formation of protein aggregates that are likely to play a role in the degeneration.

The relationship between VCP and TDP-43 has been explored in *Drosophila* (Ritson et al., 2010). Ectopic expression of mutant VCPs that contain disease-causing variants, R152H or A229E, in *Drosophila* eyes leads to a severe photoreceptor loss when compared to wild-type VCP expression, which only causes a subtle defect (Ritson et al., 2010). Expression of these mutant VCPs in the *Drosophila* central nervous system (CNS) leads to a reduced life span. Moreover, loss of TDP-43 suppresses mutant VCP-induced photoreceptor loss, whereas coexpression of wild-type TDP-43 with mutant VCP enhances photoreceptor loss, suggesting that phenotypes mediated by expression of pathogenic VCP are mediated through TDP-43 in *Drosophila*. Interestingly, TDP-43 without the NES has a minor effect on VCP-mediated degenerative phenotypes, while TDP-43 without a NLS

enhances VCP-mediated photoreceptor loss (Ritson et al., 2010). These observations suggest that the nuclear exclusion of TDP-43 and/or the formation of TDP-43 cytoplasmic inclusions play a key role in the pathogenesis of VCP-related diseases. In addition, the redistribution of TDP-43 is also observed in cultured primary neurons or brain cortex expressing VCP (R155H) (Nalbandian et al., 2013; Ritson et al., 2010). Consistently, knock-in of the VCP (R155H/+) variant in mice causes motor neuron loss and muscle denervation at 24–27 months. Furthermore, there is also an accumulation of cytoplasmic TDP-43 and ubiquitin-positive inclusions in hippocampi as well as spinal cords in VCP(R155H/+) mice (Nalbandian et al., 2013; Yin et al., 2012). These data suggest that TDP-43 mislocalization and TDP-43 pathology play a key role in the pathogenesis of VCP-linked ALS.

VCP has also been shown to interact genetically with FUS in *Drosophila*. Reduced levels of FUS in *Drosophila* lead to a rough-eye phenotype, locomotion defects, as well as a reduction of the bouton numbers and branch length in NMJ (Azuma et al., 2014). Overexpression of VCP suppresses these phenotypes while loss of VCP exacerbates the photoreceptor defects (Azuma et al., 2014). In addition, overexpression of VCP promotes FUS nuclear localization, suggesting that VCP promotes nuclear translocation of FUS or suppresses FUS degradation (Azuma et al., 2014).

Although most data relate VCP to defects in the proteasomal degradation pathway, the accumulation of LC3 and p62 has also been observed in patients with VCP R155H variant, suggesting defects in autophagy (Ju et al., 2009; Tresse et al., 2010). Both LC3 and p62 are key components of autophagy (Xie & Klionsky, 2007). Indeed, both loss of VCP and expression of VCP carrying disease-causing variants impair autophagosome maturation in cultured cells, indicating that VCP regulates autophagy pathway (Ju et al., 2009; Tresse et al., 2010). In *Drosophila*, HDAC6, a microtubule-associated deacetylase that binds to polyubiquitated proteins and VCP, has been shown to mediate a compensatory autophagic degradation upon proteasome inhibition (Pandey, Batlevi, Baehrecke, & Taylor, 2007; Pandey, Nie, et al., 2007; Seigneurin-Berny et al., 2001). Overexpression of VCP (R155H) disrupts autophagic degradation but not proteasome degradation in cultured cells (Tresse et al., 2010). HDAC6 may modulate the role of VCP between these two functions. Interestingly, suppression of autophagy in cultured cells causes cytoplasmic TDP-43 accumulation (Ju et al., 2009), suggesting that cytoplasmic mislocalized TDP-43 can be degraded through autophagy.

In summary, VCP extracts proteins from membranes and promotes protein refolding and degradation via the proteasomal and/or autophagic degradation pathway (Fig. 2A). Protein cofactors such as HDAC6 play an important role in modulating this process. Variants in VCP may disrupt its binding to cofactors and block aberrant protein refolding and degradation, leading to an accumulation of misfolded proteins. Moreover, VCP loss of function impairs nuclear localization of TDP-43, leading to the accumulation of TDP-43 in cytoplasm which in turn promotes the formation of protein aggregates, possibly further exacerbating the proteostasis defects (Fig. 2B).

## 4.2 UBQLN2

UBQLN2 belongs to an evolutionarily conserved protein family that functions in proteasomal degradation (Mah, Perry, Smith, & Monteiro, 2000; Wu, Wang, Zheleznyak, & Brown, 1999). There are four homologs in mammals, UBQLN1–4, and a single homolog in *Drosophila*. Human Ubiquilin-1 and -2 share 79% homology and are conserved in domain structure, indicating that they might share a similar molecular function. Ubiquilin-1 is expressed ubiquitously, whereas Ubiquilin-2 is mainly expressed in muscles and nervous system (Wu et al., 1999; Zhang & Saunders, 2009). Variants in *UBQLN1* have been linked to Alzheimer’s disease (AD) and Brown-Vialetto-Van Laere syndrome (BVVLS) (Bertram et al., 2005; Gonzalez-Perez et al., 2012). BVVLS is a rare neurological disorder characterized by a progressive loss of hearing and lower or upper motor neuron degeneration. It is also associated with TDP-43-containing protein aggregates in cultured neurons that express the mutant protein (Gonzalez-Perez et al., 2012; Sathasivam, 2008). Ubiquilin-1 has also been shown to bind to polyubiquitinated presenilin-1, a protease that regulates amyloid formation in AD, and targets it to the proteasome (Haapasalo et al., 2010). Variants in *UBQLN2* cause a dominantly inherited, X-linked form of ALS (also known as ALS15) and FTD (Deng et al., 2011; Fahed et al., 2014; Gellera et al., 2013; Synofzik et al., 2012; Williams et al., 2012). Expression of disease-causing variants of Ubiquilin-2 in cell lines disrupts proteasomal degradation and causes accumulation of ubiquitinated substrates (Deng et al., 2011). Hence, defects in proteostasis play a role in the pathogenesis of diseases associated with *UBQLN* variants.

Ubiquilin-1 and -2 are cytosolic proteins that control proteasomal degradation and ERAD (Walters, Kleijnen, Goh, Wagner, & Howley, 2002; Zhang & Saunders, 2009). Ubiquilin-2 contains an N-terminal UBL

ALS and Proteostasis

A
Misfolded protein
ER lumen
Derlin
HDAC6
ATP
VCP
ADP + Pi
cofactor
UBA
UBL
Ubiquilins
Autophagosome
Proteasome
Cytoplasm

C
ER lumen
VAPB
VAPB P56S
MSP
MSBP56S
ER stress
Cytoplasm

B
TDP-43
VCP
TDP-43
Nucleus
TDP-43
UBA
UBL
?
Proteasome Autophagosome Protein/RNA aggregates
Cytoplasm

Fig. 2 A model for the involvement of the proteasome degradation system in the formation of protein aggregates and degeneration. (A) VCP facilitates the unfolding and translocation of misfolded proteins from membranes or protein aggregates and consumes ATP. The translocation machinery (e.g., Derlin) is required for extracting proteins from the membrane or the ER lumen. Cofactors, such as HDAC6, modulate VCP function in protein degradation through proteasomes or autophagosomes. UBQLNs bind to ubiquitinated proteins from the ER or cytoplasm through their UBA domain. This facilitates the delivery of target proteins to proteasome for degradation. (B) In addition to their function to regulate protein degradation through the proteasome degradation system, VCP and Ubiquilins also play a role to control TDP-43 localization. VCP can affect TDP-43 localization to the nucleus. UBQLNs bind to cytoplasmic TDP-43 and deliver it to (Continued)
(ubiquitin-like) domain followed by STI1-like motifs, PXX repeats, and a C-terminal UBA (ubiquitin-associated) domain. The UBL domain mediates the interaction of Ubiquilins with the proteasome while the UBA domain mediates its interaction with polyubiquitinated proteins (Kleijnen, Alarcon, & Howley, 2003; Walters et al., 2002). This domain organization is characteristic of proteins that are involved in facilitating the presentation of polyubiquitinated proteins to the proteasomes for degradation (Elsasser & Finley, 2005; Miller & Gordon, 2005). Hence, it has been proposed that variants of Ubiquilin-1 and -2 may disrupt its binding to the proteasome or polyubiquitinated proteins. This in turn may impair the shuttling of polyubiquitinated proteins to the proteasomal degradation machinery and causes a chronic accumulation of protein aggregates. However, most of the disease-causing variants identified in patients localize to the PXX repeats, of which the molecular function is poorly characterized (Deng et al., 2011; Fahed et al., 2014; Gellera et al., 2013). Ectopic expression of Ubiquilin-2 containing human variants in the PXX repeats in a neuronal cell line leads to an aggregation of a proteasome reporter substrate in the cytoplasm (Deng et al., 2011). These disease mutants show a reduced binding affinity to the proteasome, whereas their interaction with polyubiquitinated proteins is not affected (Chang & Monteiro, 2015). The data suggest that the PXX repeats may specifically affect the interaction of Ubiquilin with the proteasome. Hence, the ALS-causing variants in UBQLN2 may disrupt this interaction and impair the delivery of polyubiquitinated proteins to the proteasome and therefore causes accumulation of ubiquitinated proteins.

The molecular mechanism related to how Ubiquilins control ERAD has been investigated in cultured cells. Overexpression of Ubiquilins promotes the degradation of ERAD substrates whereas loss of Ubiquilins or overexpression of mutant forms leads to substrate accumulation (Kim et al., 2008; Xia et al., 2014), indicating that Ubiquilins are required for ERAD. It is still unclear how Ubiquilins regulate ERAD and current observations suggest that Ubiquilins may bind to ER stress response proteins like Erasin

Fig. 2—Cont'd either the proteasomes or autophagosomes for degradation. Moreover, UBQLNs also transfer TDP-43 to aggregates. Collectively, these mechanisms lead to TDP-43 nuclear exclusion and cause the incorporation of cryptic exon sequences in mRNAs. This results in the accumulation of aberrant proteins. (C) VAPB is localized to the ER through its C-terminal transmembrane domain. The N-terminal MSP domain is localized to the cytoplasm. The disease-causing variant (P56S) disrupts the proper function of VAPB and leads to the formation of protein aggregates, traps endogenous wild-type VAPB proteins, induces ER stress and degeneration.(Lim et al., 2009), Herp (Kim et al., 2008), or UBXD8 (Xia et al., 2014), to facilitate the recognition of misfolded ER proteins for proteasome degradation. Erasin is an ER membrane-associated protein that functions as a platform to recruit VCP and Ubiquilin to facilitate ERAD (Liang et al., 2006; Lim et al., 2009). Suppression of the expression of Erasin or Ubiquilin causes an accumulation of ERAD substrates (Lim et al., 2009). In C. elegans, loss of Erasin or Ubiquilin leads to the accumulation of polyubiquitied proteins, activation of ER stress, and a reduced life span (Lim et al., 2009), providing in vivo evidence that Erasin and Ubiquilin play a key role in ERAD. Moreover, Erasin binds to Derlin, an ER membrane protein that may form a channel that allows the export of misfolded proteins or extract misfolded protein from the ER in a VCP-dependent manner (Lilley & Ploegh, 2004; Mehnert, Sommer, & Jarosch, 2014). Hence, Erasin has been proposed to recruit Ubiquilin and promotes the shuttling of misfolded proteins to the proteasomal degradation machinery (Lim et al., 2009). In addition to Erasin, Herp, and UBXD8 have also been reported to bind to Ubiquilin (Kim et al., 2008; Xia et al., 2014). Disrupting the interaction between Ubiquilin and Herp or UBXD8 reduces ERAD (Kim et al., 2008; Xia et al., 2014). Pathogenic variants of UBQLN2 impair its interaction with UBXD8 and disrupt ERAD (Xia et al., 2014). Hence, Ubiquilin dysfunction may impair ERAD, leading to the accumulation of misfolded proteins that in turn contribute to neuronal degeneration in ALS.

In addition to the proteasomal degradation machinery and ERAD, evidence from cultured cell lines suggests that Ubiquilins may also play a role in autophagy (N'Diaye et al., 2009; Rothenberg et al., 2010). Ubiquilin-1 and -2 have been shown to localize to autophagosomes in HeLa cells as well as mouse brain and liver tissue. Loss of Ubiquilins reduces autophagosome formation and maturation (Rothenberg et al., 2010). Overexpression of Ubiquilin-1 or -2 suppresses starvation-induced cell death, and removing key proteins of the autophagy machinery, ATG5 and ATG7, abolish this protective effect (N'Diaye et al., 2009). These findings indicate that Ubiquilin may promote autophagy to suppress starvation-induced cell death. Moreover, Ubiquilin-1 physically interacts with LC3, a key component in autophagosomes. Deleting the UBA domain of Ubiquilin-1 abolishes this interaction as well as the protective effect in starvation-induced cell death (N'Diaye et al., 2009). Therefore, it is proposed that Ubiquilins may function in the delivery of ubiquitinated cargo to autophagosome for degradation (Fig. 2A). However, these data have not yet been validated in vivo and the mechanism is not understood.
UBQLN2 also interacts with TDP-43. Cytoplasmic inclusions that contain TDP-43 are often observed in ALS patients with UBQLN2 variants (Fahed et al., 2014; Williams et al., 2012) and Ubiquilin-2-positive aggregates in other ALS patient samples also colocalize with TDP-43 inclusions (Deng et al., 2011; Williams et al., 2012). In addition, Ubiquilin-1 physically interacts with TDP-43 in yeast and the C-terminal UBA domain of Ubiquilin-1 is necessary and sufficient for this interaction (Kim et al., 2009). Overexpression of Ubiquilin-1 in HeLa cells potentiates TDP-43 aggregate formation in the cytoplasm (Kim et al., 2009), suggesting that Ubiquilin-1 promotes TDP-43 aggregation. Moreover, the reduced eclosion rate as well as shortened life span caused by TDP-43 overexpression is exacerbated upon coexpressing human Ubiquilin in Drosophila (Hanson et al., 2010), suggesting that Ubiquilins enhance TDP-43 toxicity. In addition, the same lab has also documented that overexpression of Ubiquilin-1 reduces the detergent soluble form of TDP-43 in Drosophila and in cultured cells, consistent with the hypothesis that insoluble TDP-43 is toxic (Hanson et al., 2010). Furthermore, overexpression of Ubiquilin-1 with disease variant (E54D) in both Drosophila as well as HEK293 cells also reduces the soluble form of TDP-43 and redistributes TDP-43 from the nucleus to the cytoplasm (Gonzalez-Perez et al., 2012). Given that Ubiquilin physically interacts with TDP-43, it is proposed that Ubiquilin modulates the level of soluble TDP-43 either by coaggregation with TDP-43 or by delivering it to proteasome or autophagosome for degradation (Fig. 2B). Hence, defects in Ubiquilin pathway disturb the proper function of TDP-43 in the nucleus and may lead to the neurodegeneration.

## 4.3 VAPB

VAPB (VAMP-associated protein B) belongs to the VAMP-associated protein family. It is highly conserved across species and named *Vap-33-1* in *Drosophila*, *Vpr-1* in *C. elegans*, and *Scs2p* in *Saccharomyces cerevisiae* (Nikawa, Murakami, Esumi, & Hosaka, 1995; Pennetta, Hiesinger, Fabian-Fine, Meinertzhagen, & Bellen, 2002; Weir, Klip, & Trimble, 1998). We will use VAPB for all species. There are three homologs of VAPs in mammals, VAPA, VAPB, and VAPC (Nishimura, Hayashi, Inada, & Tanaka, 1999). Both VAPA and VAPB contain an N-terminal major sperm proteins (MSP) domain that shows homology to MSP in *C. elegans*, a coiled-coil domain, and a C-terminal transmembrane (TM) domain (Nishimura et al., 1999). Multiple variants in VAPB (P56S, T46I, A145V, S160Δ,
and V234I) have been identified in patients with ALS (also known as ALS8) or spinal muscular atrophy (Chen et al., 2010; Kabashi et al., 2013; Nishimura et al., 2004; van Blitterswijk et al., 2012). Among them, the P56S variant is by far the most prevalent form that cosegregates with disease in ALS patients.

VAPB plays a role in several cellular processes and functions both cell autonomously and nonautonomously. The cell autonomous function of VAPs (VAPA and VAPB) has been associated with its localization to the ER (Kagiwada, Hosaka, Murata, Nikawa, & Takatsuki, 1998; Tsuda et al., 2008). Both VAPA and VAPB are required for the transfer of oxysterols and ceramides from the ER to Golgi via interactions with lipid-binding proteins, such as OSBP (oxysterol-binding protein) and CERT (ceramide transfer protein) (Kawano, Kumagai, Nishijima, & Hanada, 2006; Loewen, Roy, & Levine, 2003; Sanhueza et al., 2015). The disease-causing variant, P56S, disrupts these interactions in *Drosophila* and may also do so in mammalian cells (Moustaqim-Barrette et al., 2013; Teuling et al., 2007) and loss of VAPs affects the structure of ER and Golgi (Moustaqim-Barrette et al., 2013; Peretti, Dahan, Shimoni, Hirschberg, & Lev, 2008). In *Drosophila*, overexpression of specific OSBPs in neuronal tissues suppresses the early lethality and locomotor defect in *VAPB* (P58S) (equivalent to the P56S in human) mutant flies (Moustaqim-Barrette et al., 2013). These observations suggest that promoting lipid transfer can partially suppress the phenotypes associated with *VAPB* (P58S) in flies.

VAPB has also been shown to function in a cell nonautonomous fashion, as the MSP domain of VAPB is cleaved and secreted from cells (Charng, Yamamoto, & Bellen, 2014; Tsuda et al., 2008). In *Drosophila* wing discs, ectopically expressing VAPB (P58S) results in protein aggregation and failure of MSP secretion (Tsuda et al., 2008). In *C. elegans*, secreted MSP binds to the Ephrin A4 receptor as well as Lar-like protein–tyrosine–phosphatase (CLR) and Roundabout (Robo) present on muscles (Han et al., 2012). The binding of MSP to these receptors is required to retain cytoskeletal organization in muscles as well as proper mitochondrial morphology, which relies on actin and myosins (Han et al., 2012). Loss of VAPB in the nervous system causes a severe muscle mitochondria defect in flies and worms. Secreted MSP is also detected in human serum and cerebrospinal fluid (Deidda et al., 2014; Tsuda et al., 2008), and the MSP level is reduced in patients with SALS (Deidda et al., 2014), indicating that loss of MSP secretion may play a role in ALS.
Loss of VAPB in *Drosophila* leads to pupal lethality. Introducing one copy of wild-type VAPB or VAPB (P58S) in VAPB null flies rescues the lethality (Moustaqim-Barrette et al., 2013). However, VAPB (P58S) flies exhibit severely reduced life span with age-dependent climbing defects and reduced flying ability (Moustaqim-Barrette et al., 2013). Heterozygous VAPB (P58S/+) flies have a normal life span as well as locomotion (Moustaqim-Barrette et al., 2013), indicating that the P58S allele is a partial loss of function. Interestingly, the VAPB protein level is reduced in motor neurons derived from fibroblasts of ALS patients carrying the VAPB (P56S) variant (Mitne-Neto et al., 2011). Similarly, the VAPB level is decreased in spinal cords of SOD-1 (G93A) transgenic mice as well as SALS patients (Anagnostou et al., 2010; Teuling et al., 2007). These data suggest that the VAPB (P56S) is a partial loss-of-function variant and that loss of VAPB may play a role in FALS and SALS.

VAPB knockout mice are viable and display a mild defect in motor activity starting at 18 months of age (Kabashi et al., 2013). Homozygous VAPB (P56S) knock-in mice start to develop defects at 9–11 months age, including cytoplasmic inclusions, locomotion defects as well as muscle denervation (Larroquette et al., 2015). The VAPB (P56S/+) heterozygous mice display similar but milder phenotypes (Larroquette et al., 2015), consistent with the dominant inheritance pattern of VAPB (P56S) (Nishimura et al., 2004). The data indicate that the VAPB (P56S) variant causes both a loss of function and/or a dominant-negative effect. The later may be due to the observation that the mutant VAPB protein preferentially binds to wild-type protein and this hypothesis is supported by in vivo data (Teuling et al., 2007). This hypothesis is also consistent with the observations in transgenic mice (Aliaga et al., 2013; Tudor et al., 2010) and *Drosophila* expressing various VAPB variants (Chen et al., 2010; Forrest et al., 2013; Sanhueza, Zechini, Gillespie, & Pennetta, 2013; Tsuda et al., 2008).

The most striking phenotypes associated with VAPB studies are defects in ER quality control and protein aggregation. Studies in mammalian cells and *Drosophila* have shown that VAPB is associated with ER. In *Drosophila*, loss of VAPB causes aberrant ER expansion, accumulation of membrane proteins on the ER, and alternatively spliced XBP-1, which are typically associated with ER stress (Moustaqim-Barrette et al., 2013; Tsuda et al., 2008). Similarly, expression of VAPB (P58S) also leads to formation of protein aggregates, ER stress, and traps endogenous wild-type VAPB proteins (Teuling et al., 2007; Tsuda et al., 2008). Knock-in mice carrying the VAPB (P56S) variant develop an accumulation of ubiquitinated proteins in
cytoplasmic inclusions in motor neurons prior to the onset of the defects in locomotion. Those aggregates also partially colocalize with autophagosome markers (Larroquette et al., 2015). These data suggest that defects in VAPB function lead to ER stress and protein aggregation. Mutant VAPB proteins accumulate in large puncta in both corticospinal motor neurons and spinal motor neurons in VAPB (P56S) transgenic mice (Aliaga et al., 2013). VAPB (P56S) leads to a significant increase in ER stress and proapoptotic factors expression (Aliaga et al., 2013). Therefore, VAPB (P56S) protein accumulates and the failure of protein aggregate clearance may affect neuronal survival.

In summary, VAPB has pleiotropic functions in ER quality control, lipid transfer from ER to Golgi, and mitochondrial dynamics in muscles (Chang et al., 2014). Disease-causing variants in VAPB disrupt lipid transfer, cause protein aggregations, and trap wild-type VAPB protein. The accumulation of VAPB aggregates correlates with ER stress (Fig. 2C). However, the induced ER stress is not sufficient to clear all the aggregates, which may eventually lead to neuronal degeneration. These results indicate that a defect in protein clearance combined with reduced MSP secretion may play a significant role in ALS pathology.

### 5. SOD-1 AND PROTEINOPATHY IN ALS

Copper zinc superoxide dismutase, SOD-1, is a ubiquitously expressed and highly conserved enzyme that plays a role in scavenging superoxide radicals. It was the first gene to be associated with FALS (Rosen et al., 1993) and is present in 10–20% of all FALS cases (also known as ALS1) (Renton et al., 2014). Two major hypotheses may underlie SOD-1 toxicity: (1) a loss of dismutase activity and/or (2) toxicity caused by protein aggregation.

A reduction of dismutase activity and an increase in superoxides were detected in tissue extracts from ALS SOD-1 patients (Bowling et al., 1995; Cohen, Kohen, Lavon, Abramsky, & Steiner, 1996; Deng et al., 1993; Moumen, Nouvelot, Duval, Lechevalier, & Viader, 1997; Rosen et al., 1993). It was proposed that some ALS-related SOD-1 variants disrupt dimerization and reduce its dismutase activity. This in turn leads to the elevation of oxidative stress that causes protein misfolding and aggregation and contributes to degeneration. The dismutase activity of SOD-1 decreases with age in ALS patients but not in normal controls (Fiszman, Borodinsky, Ricart, Sanz, & Sica, 1999). In contrast, loss of dismutaseactivity does not correlate with age of onset or aggressiveness of the disease in SOD-1 patients (Fiszman et al., 1999; Ratovitski et al., 1999) and many question a role for the enzymatic function in the pathogenesis of the disease.

Interestingly, loss of SOD-1 in *Drosophila* causes a mild (~10%) reduction of life span, a severe loss of fertility, hypersensitivity to oxidative stress (Phillips, Campbell, Michaud, Charbonneau, & Hilliker, 1989), and retinal degeneration (Phillips et al., 1995). In mice, five-independent SOD-1 knockout mice have been established (Ho et al., 1998; Huang et al., 1997; Matzuk, Dionne, Guo, Kumar, & Lebovitz, 1998; Reaume et al., 1996; Yoshida, Maulik, Engelman, Ho, & Das, 2000) and all these mutants exhibit mild defects, including a progressive loss of locomotion, denervation at NMJs, and muscle atrophy (Reaume et al., 1996; Saccon, Bunton-Stasyshyn, Fisher, & Fratta, 2013), symptoms that are all typically associated with ALS. In addition, an increased vulnerability to oxidative stress has also been observed in two mouse models (Ho et al., 1998; Reaume et al., 1996). However, no obvious motor neuron loss or accumulation of ubiquitinated protein inclusions has been documented in these mice. The above data suggest that loss of dismutase activity in ALS-linked SOD-1 variants may play a role in the pathogenesis of the disease. This loss may create a sensitized background to oxidative stress which may promote the accumulation of misfolded proteins and protein aggregates. However, the lack of motor neuron death and paralysis suggests that the SOD-1 loss of dismutase activity may not cause ALS, but may promote the pathogenesis.

The formation of SOD-1-containing insoluble inclusions in the affected motor neurons is a hallmark of SOD-1-related FALS cases and has been considered a cause of motor neuron death (Bruijn et al., 1998). SOD-1 mutants that carry human ALS-linked variants are susceptible to partial unfolding and prone to the formation of aggregates (Tiwari & Hayward, 2005). Transgenic mice overexpressing ALS-related SOD-1 variants exhibit large cytoplasmic SOD-1- and ubiquitin-positive inclusions in motor neurons, a shorter life span, hind limb paralysis, muscle atrophy, motor neuron loss, and axon degeneration (Bruijn et al., 1997; Cleveland, 1999; Gurney et al., 1994; Julien & Kriz, 2006; Wong et al., 1995). However, the dismutase activity is either unchanged relative to the number of copies or even elevated in at least two transgenic strains, SOD-1 (G93A) and SOD-1 (G37R) (Bruijn et al., 1997; Deng et al., 2006; Saccon et al., 2013; Subramaniam et al., 2002), suggesting that the loss of dismutase activity discussed previously may not be implicated. In *Drosophila*, overexpression of human wild type or ALS-linked (A4V, G85R) SOD-1 in motor neurons causes a
progressive climbing defect and abnormal synaptic transmission (Watson et al., 2008). However, overexpression of *Drosophila* wild-type SOD-1 does not cause any defects (Watson et al., 2008), suggesting that a gain of the dismutase activity does not cause neuronal toxicity. In addition, Watson et al. (2008) found that the degenerative phenotypes are accompanied by the focal accumulation of human wild-type and mutant SOD-1 proteins in motor neurons (Watson et al., 2008), suggesting that chronic accumulation of misfolded SOD-1 in the ubiquitin-positive inclusions contributes to the degeneration. This model is supported by the fact that infusion of oligonucleotides complementary to human SOD-1 mRNA prolongs survival in rats expressing human SOD-1 (G93A) (Smith et al., 2006). Clinical trials that involve the introduction of these oligonucleotides to SOD-1 FALS patients are under evaluation (Miller et al., 2013). Interestingly, TDP-43 proteinopathy is absent from SOD-1–positive inclusions in SOD-1–related ALS cases (Blokhus et al., 2013; Mackenzie et al., 2007), implicating that SOD-1 inclusions affect degeneration via a distinct molecular pathway.

Extensive datasets suggest that ER stress may play an important role in SOD-1 FALS. Indeed, overexpression of the mutant but not wild-type SOD-1 forms in cultured cells promotes ER stress (Nishitoh et al., 2008; Oh et al., 2008; Tobisawa et al., 2003). Similarly, transgenic mice that express mutant SOD-1 (Atkin et al., 2006; Kikuchi et al., 2006; Saxena et al., 2009; Tobisawa et al., 2003; Wate et al., 2005) and motor neurons of SOD-1 patients exhibit an elevated level of ER stress (Atkin et al., 2008; Ilieva et al., 2007). The ER stress responses promote the expression of chaperones that are targeted to the ER to refold misfolded proteins (Gardner, Pincus, Gotthardt, Gallagher, & Walter, 2013; Ron & Walter, 2007). The ER stress response is activated through three sensor pathways, IRE1, PERK, or ATF6, which activate distinct downstream effectors that promote chaperone expression to facilitate protein refolding (Ron & Walter, 2007).

To assess the role of the PERK pathway, Wang, Popko, et al. (2011) removed one copy of PERK in SOD-1 (G85R) mice and noted that this accelerated aggregate formation, shortened life span, and enhanced the degeneration phenotypes (Wang, Popko, et al., 2011). These data suggest that activation of the PERK pathway provides a neuroprotective effect. Indeed, removal of one copy of GADD34 (growth arrest and DNA damage-inducible protein), a downstream suppressor of PERK, enhanced PERK signaling, and reduced aggregate formation, prolonged life span, and alleviated the degenerative phenotypes (Wang, Popko, et al., 2014). Similarly,
Guanabenz or Sephin1, drugs which inhibit GADD34, and hence promote PERK signaling, ameliorated the mutant SOD-1-induced ALS phenotypes in mice (Das et al., 2015; Wang, Popko, et al., 2014), further suggesting that activation of PERK is beneficial in ALS. However, a neuronal knockout of XBP-1 in SOD-1 (G85R), a downstream component of the IRE1 pathway, reduced the accumulation of mutant SOD-1 in spinal cord, prolonged life span, and alleviated degeneration phenotypes (Hetz et al., 2009). Hence, activation of the PERK1 pathway as well as inactivation of the IRE1 pathway is beneficial. The latter data were attributed to the observation that basal autophagy activity is enhanced in these mice and may compensate for the ERAD defect (Hetz et al., 2009). In summary, variants in SOD-1 cause misfolding of the protein and the formation of protein aggregates. However, activation of the ER stress and promotion of ERAD and/or basal autophagy may be beneficial to the progression of the disease.

### 6. PRION-LIKE PROTEIN TOXICITY AND ALS

The general concept of prion diseases is that an infectious misfolded protein, such as a misfolded prion isoform, PrP<sup>sc</sup>, forms aggregates through self-aggregation and propagates through binding to wild type or normal protein, such as prion (PrP<sup>c</sup>), to produce a misfolded, pathogenic version (Colby & Prusiner, 2011; Prusiner, 1982). In addition, the misfolded protein has the capacity to be transmitted from one cell to another. Therefore, the disease can spread from the site of onset to surrounding tissues. ALS shows a number of similarities with prion diseases. (1) All ALS patients have misfolded protein-containing aggregates in their affected neurons (Blokhus et al., 2013). (2) In cultured neuronal cells, misfolded forms of SOD-1 or TDP-43 can self-propagate within the cells and be transmitted to neighboring cells via cellular connections (Johnson et al., 2009; Munch, O’Brien, & Bertolotti, 2011). (3) Studies of tissue samples from postmortem ALS patients suggest that lower motor neuron degeneration is a focal process that gradually spreads to distal sites (Lee & Kim, 2015; Ravits, Laurie, Fan, & Moore, 2007), similar to the spreading of the prion diseases. (4) ALS-causing genes, including TDP43, FUS, and hnRNPA1 have a prion-like domain (Kim et al., 2013), which may contribute to the spread.

SOD-1 aggregates in patients or mice exhibit a fibrillary structure enriched in β-sheets, a feature characteristic for a template to transform a normal protein into a misfolded protein which elongates into fibrils(Furukawa, Kaneko, Yamanaka, O’Halloran, & Nukina, 2008). This hypothesis has been supported experimentally as protein extracts from spinal cords of SOD-1 (G93A) mice are able to induce wild-type SOD-1 to form amyloid-like fibrils (Chia et al., 2010; Grad et al., 2011). Moreover, in vitro data suggest that aggregates composed of mutated SOD-1 can be efficiently transferred from a sick cell to a healthy cell in culture (Munch et al., 2011). These data argue that ALS-related SOD-1 mutants might use a prion-like mechanism to spread the disease.

TDP-43 harbors a predicted prion-like domain (Lee & Kim, 2015). The predicted prion domain of TDP-43 contains a glycine-rich domain in the C-terminal region (Lee & Kim, 2015). Expression of the full-length human wild-type and ALS mutant TDP-43 in *Drosophila* eyes causes an eye degeneration (Estes et al., 2011; Hanson et al., 2010; Li et al., 2010; Ritson et al., 2010). Interestingly, the severity of eye degeneration correlates with the level of a truncated 25 kDa C-terminal fragment that contains the prion-like domain (Ritson et al., 2010) and the 25 kDa C-terminal domain of TDP-43 is present in the cytoplasmic inclusions of ALS patients (Neumann et al., 2006). These data argue that there is a relationship between the formation of TDP-43-containing aggregates, the prion-like mechanism, and the degeneration phenotype.

The predicted prion-like domain of FUS is localized at the very N-terminal part that is SYGQ-rich (Lee & Kim, 2015), required for FUS self-assembly, and formation of cytoplasmic FUS-containing inclusions (Yang, Gal, Chen, & Zhu, 2014). In *Drosophila*, ectopic expression of either wild-type or human ALS-linked variants in FUS in the eyes causes degeneration (Chen et al., 2011; Lanson et al., 2011). However, the role of the prion-like domain alone in the induction of degeneration in *Drosophila* has not been tested. Finally, another prion-like domain containing RBP, hnRNPA1, has recently been associated with ALS and contains a prion-like domain (Alberti, Halfmann, King, Kapila, & Lindquist, 2009; Kim et al., 2013). The above data argue that a prion-like disease propagation mechanism may play a role in the pathogenesis of FALS and SALS.

### 7. CONCLUSION AND FUTURE PERSPECTIVE

In summary, we propose that variants in seven genes (*TARDBP, FUS, C9orf72, VCP, UBQLN2, VAPB,* and *SOD-1*) that cause ALS converge on chronic defects in RNA and protein homeostasis. Mutations in these proteins lead to the aberrant localization and/or degradation of TDP-43.
Variants in TDP-43 cause the exclusion of TDP-43 from the nucleus and lead to the incorporation of cryptic exons in the mRNAs of thousands of genes, causing the formation of misfolded proteins that overload the protein refolding and degradation machinery and leads to protein aggregates. Similarly, FUS variants have been shown to cause aberrant splicing of numerous genes. The presence of the HRE in C9orf72 causes protein aggregates that impair nucleocytoplasmic transport, promoting TDP-43 mislocalization to the cytoplasm, which in turn promotes TDP-43 proteinopathy. This model is further supported by the observation that variants in VCP or UBQLNs that affect protein refolding, ubiquitination, and degradation via the proteasome degradation pathway also lead to mislocalization of TDP-43. Finally, variants in VAPB and SOD-1 also cause protein aggregates, induce ER stress, and affect ERAD function even though TDP-43 pathology may not play a role in the latter.

# ACKNOWLEDGMENTS

We thank Karen Schulze, Ke Zhang, Shinya Yamamoto, Mumine Senturk, and Hsiao-Tuan Chao for their critical reading of the manuscript. This work was supported by Target ALS, the Robert A. and Renee E. Belfer Family Foundation, and the Huffington Foundation. H.J.B. is an Investigator of the Howard Hughes Medical Institute.

# REFERENCES

Abramzon, Y., Johnson, J. O., Scholz, S. W., Taylor, J. P., Brunetti, M., Calvo, A., et al. (2012). Valosin-containing protein (VCP) mutations in sporadic amyotrophic lateral sclerosis. *Neurobiology of Aging*, 33(9), 2231.e1–2231.e6.

Ajroud-Driss, S., & Siddique, T. (2015). Sporadic and hereditary amyotrophic lateral sclerosis (ALS). *Biochimica et Biophysica Acta*, 1852(4), 679–684.

Alberti, S., Halfmann, R., King, O., Kapila, A., & Lindquist, S. (2009). A systematic survey identifies prions and illuminates sequence features of prionogenic proteins. *Cell*, 137(1), 146–158.

Al-Chalabi, A., Jones, A., Troakes, C., King, A., Al-Sarraj, S., & van den Berg, L. H. (2012). The genetics and neuropathology of amyotrophic lateral sclerosis. *Acta Neuropathologica*, 124(3), 339–352.

Ali, Y. O., Escala, W., Ruan, K., & Zhai, R. G. (2011). Assaying locomotor, learning, and memory deficits in Drosophila models of neurodegeneration. *Journal of Visualized Experiments*, (49), pii: 2504.

Aliaga, L., Lai, C., Yu, J., Chub, N., Shim, H., Sun, L., et al. (2013). Amyotrophic lateral sclerosis-related VAPB P56S mutation differentially affects the function and survival of corticospinal and spinal motor neurons. *Human Molecular Genetics*, 22(21), 4293–4305.

Allen, M. J., & Godenschwege, T. A. (2010). Electrophysiological recordings from the Drosophila giant fiber system (GFS). *Cold Spring Harbor Protocols*. 2010(7). http://dx.doi.org/10.1101/pdb.prot5453. pdb prot5453.

Alonso, A., Logroscino, G., Jick, S. S., & Hernan, M. A. (2009). Incidence and lifetime risk of motor neuron disease in the United Kingdom: A population-based study. *European Journal of Neurology*, 16(6), 745–751.

Anagnostou, G., Akbar, M. T., Paul, P., Angelinetta, C., Steiner, T. J., & de Belleroche, J. (2010). Vesicle associated membrane protein B (VAPB) is decreased in ALS spinal cord. *Neurobiology of Aging*, 31(6), 969–985.

Andlauer, T. F., & Sigrist, S. J. (2012a). In vivo imaging of Drosophila larval neuromuscular junctions to study synapse assembly. *Cold Spring Harbor Protocols*, 2012(4), 407–413.

Andlauer, T. F., & Sigrist, S. J. (2012b). In vivo imaging of the Drosophila larval neuromuscular junction. *Cold Spring Harbor Protocols*, 2012(4), 481–489.

Andlauer, T. F., & Sigrist, S. J. (2012c). Quantitative analysis of Drosophila larval neuromuscular junction morphology. *Cold Spring Harbor Protocols*, 2012(4), 490–493.

Arai, T., Hasegawa, M., Nonoka, T., Kametani, F., Yamashita, M., Hosokawa, M., et al. (2010). Phosphorylated and cleaved TDP-43 in ALS, FTLD and other neurodegenerative disorders and in cellular models of TDP-43 proteinopathy. *Neuropathology*, 30(2), 170–181.

Ash, P. E., Bieniek, K. F., Gendron, T. F., Caulfield, T., Lin, W. L., Dejesus-Hernandez, M., et al. (2013). Unconventional translation of C9ORF72 GGGGCC expansion generates insoluble polypeptides specific to c9FTD/ALS. *Neuron*, 77(4), 639–646.

Atanasio, A., Decman, V., White, D., Ramos, M., Ikiz, B., Lee, H. C., et al. (2016). C9orf72 ablation causes immune dysregulation characterized by leukocyte expansion, autoantibody production, and glomerulonephropathy in mice. *Scientific Reports*, 6, 23204.

Atkin, J. D., Farg, M. A., Turner, B. J., Tomas, D., Lysaght, J. A., Nunan, J., et al. (2006). Induction of the unfolded protein response in familial amyotrophic lateral sclerosis and association of protein-disulfide isomerase with superoxide dismutase 1. *Journal of Biological Chemistry*, 281(40), 30152–30165.

Atkin, J. D., Farg, M. A., Walker, A. K., McLean, C., Tomas, D., & Horne, M. K. (2008). Endoplasmic reticulum stress and induction of the unfolded protein response in human sporadic amyotrophic lateral sclerosis. *Neurobiology of Disease*, 30(3), 400–407.

Ayala, Y. M., Pantano, S., D'Ambrogio, A., Buratti, E., Brindisi, A., Marchetti, C., et al. (2005). Human, Drosophila, and C. elegans TDP43: Nucleic acid binding properties and splicing regulatory function. *Journal of Molecular Biology*, 348(3), 575–588.

Azuma, Y., Tokuda, T., Shimamura, M., Kyotani, A., Sasayama, H., Yoshida, T., et al. (2014). Identification of ter94, Drosophila VCP, as a strong modulator of motor neuron degeneration induced by knockdown of Caz, Drosophila FUS. *Human Molecular Genetics*, 23(13), 3467–3480.

Babcock, D. T., & Ganetzky, B. (2014). An improved method for accurate and rapid measurement of flight performance in Drosophila. *Journal of Visualized Experiments*, 84, e51223.

Barthelme, D., Jauregui, R., & Sauer, R. T. (2015). An ALS disease mutation in Cdc48/p97 impairs 20S proteasome binding and proteolytic communication. *Protein Science*, 24(9), 1521–1527.

Barthelme, D., & Sauer, R. T. (2012). Identification of the Cdc48*20S proteasome as an ancient AAA+ proteolytic machine. *Science*, 337(6096), 843–846.

Barthelme, D., & Sauer, R. T. (2013). Bipartite determinants mediate an evolutionarily conserved interaction between Cdc48 and the 20S peptidase. *Proceedings of the National Academy of Sciences of the United States of America*, 110(9), 3327–3332.

Barthelme, D., & Sauer, R. T. (2016). Origin and functional evolution of the Cdc48/p97/VCP AAA+ protein unfolding and remodeling machine. *Journal of Molecular Biology*, 428(9 Pt B), 1861–1869.

Belzil, V. V., Valdmanis, P. N., Dion, P. A., Daoud, H., Kabashi, E., Noreau, A., et al. (2009). Mutations in FUS cause FALS and SALS in French and French Canadian populations. *Neurology*, 73(15), 1176–1179.

Bertram, L., Hiltunen, M., Parkinson, M., Ingelsson, M., Lange, C., Ramasamy, K., et al. (2005). Family-based association between Alzheimer’s disease and variants in UBQLN1. *The New England Journal of Medicine*, 352(9), 884–894.

Blokhuis, A. M., Groen, E. J., Koppers, M., van den Berg, L. H., & Pasterkamp, R. J. (2013). Protein aggregation in amyotrophic lateral sclerosis. *Acta Neuropathologica*, 125(6), 777–794.

Bose, J. K., Wang, I. F., Hung, L., Tarn, W. Y., & Shen, C. K. (2008). TDP-43 overexpression enhances exon 7 inclusion during the survival of motor neuron pre-mRNA splicing. *Journal of Biological Chemistry*, 283(43), 28852–28859.

Bowling, A. C., Barkowski, E. E., McKenna-Yasek, D., Sapp, P., Horvitz, H. R., Beal, M. F., et al. (1995). Superoxide dismutase concentration and activity in familial amyotrophic lateral sclerosis. *Journal of Neurochemistry*, 64(5), 2366–2369.

Brady, O. A., Meng, P., Zheng, Y., Mao, Y., & Hu, F. (2011). Regulation of TDP-43 aggregation by phosphorylation and p62/SQSTM1. *Journal of Neurochemistry*, 116(2), 248–259.

Bruijn, L. I., Becher, M. W., Lee, M. K., Anderson, K. L., Jenkins, N. A., Copeland, N. G., et al. (1997). ALS-linked SOD1 mutant G85R mediates damage to astrocytes and promotes rapidly progressive disease with SOD1-containing inclusions. *Neuron*, 18(2), 327–338.

Bruijn, L. I., Houseweart, M. K., Kato, S., Anderson, K. L., Anderson, S. D., Ohama, E., et al. (1998). Aggregation and motor neuron toxicity of an ALS-linked SOD1 mutant independent from wild-type SOD1. *Science*, 281(5384), 1851–1854.

Brunger, A. T., & DeLaBarre, B. (2003). NSF and p97/VCP: Similar at first, different at last. *FEBS Letters*, 555(1), 126–133.

Buchberger, A., Schindelin, H., & Hanzelmann, P. (2015). Control of p97 function by cofactor binding. *FEBS Letters*, 589(19 Pt. A), 2578–2589.

Buratti, E., & Baralle, F. E. (2001). Characterization and functional implications of the RNA binding properties of nuclear factor TDP-43, a novel splicing regulator of CFTR exon 9. *Journal of Biological Chemistry*, 276(39), 36337–36343.

Buratti, E., & Baralle, F. E. (2008). Multiple roles of TDP-43 in gene expression, splicing regulation, and human disease. *Frontiers in Bioscience*, 13, 867–878.

Buratti, E., Dork, T., Zuccato, E., Pagani, F., Romano, M., & Baralle, F. E. (2001). Nuclear factor TDP-43 and SR proteins promote in vitro and in vivo CFTR exon 9 skipping. *EMBO Journal*, 20(7), 1774–1784. http://dx.doi.org/10.1093/emboj/20.7.1774.

Cao, K., Nakajima, R., Meyer, H. H., & Zheng, Y. (2003). The AAA-ATPase Cdc48/p97 regulates spindle disassembly at the end of mitosis. *Cell*, 115(3), 355–367.

Chang, L., & Monteiro, M. J. (2015). Defective proteasome delivery of polyubiquitinated proteins by ubiquilin-2 proteins containing ALS mutations. *PLoS One*, 10(6), e0130162.

Charng, W. L., Yamamoto, S., & Bellen, H. J. (2014). Shared mechanisms between Drosophila peripheral nervous system development and human neurodegenerative diseases. *Current Opinion in Neurobiology*, 27, 158–164.

Chen, H. J., Anagnostou, G., Chai, A., Withers, J., Morris, A., Adhikaree, J., et al. (2010). Characterization of the properties of a novel mutation in VAPB in familial amyotrophic lateral sclerosis. *Journal of Biological Chemistry*, 285(51), 40266–40281.

Chen, Y., Yang, M., Deng, J., Chen, X., Ye, Y., Zhu, L., et al. (2011). Expression of human FUS protein in Drosophila leads to progressive neurodegeneration. *Protein & Cell*, 2(6), 477–486.

Chew, J., Gendron, T. F., Prudencio, M., Sasaguri, H., Zhang, Y. J., Castanedes-Casey, M., et al. (2015). Neurodegeneration. C9ORF72 repeat expansions in mice cause TDP-43 pathology, neuronal loss, and behavioral deficits. *Science*, 348(6239), 1151–1154.

Chia, R., Tattum, M. H., Jones, S., Collinge, J., Fisher, E. M., & Jackson, G. S. (2010). Superoxide dismutase 1 and tgSOD1 mouse spinal cord seed fibrils, suggesting a propagative cell death mechanism in amyotrophic lateral sclerosis. *PLoS One*, 5(5), e10627.

Choksi, D. K., Roy, B., Chatterjee, S., Yusuff, T., Bakhoum, M. F., Sengupta, U., et al. (2014). TDP-43 phosphorylation by casein kinase Iepsilon promotes oligomerization and enhances toxicity in vivo. *Human Molecular Genetics*, 23(4), 1025–1035.

Ciura, S., Lattante, S., Le Ber, I., Latouche, M., Tostivint, H., Brice, A., et al. (2013). Loss of function of C9orf72 causes motor deficits in a zebrafish model of amyotrophic lateral sclerosis. *Annals of Neurology*, 74(2), 180–187.

Cleary, J. D., & Ranum, L. P. (2013). Repeat-associated non-ATG (RAN) translation in neurological disease. *Human Molecular Genetics*, 22(R1), R45–R51.

Cleveland, D. W. (1999). From Charcot to SOD1: Mechanisms of selective motor neuron death in ALS. *Neuron*, 24(3), 515–520.

Cohen, O., Kohen, R., Lavon, E., Abramsky, O., & Steiner, I. (1996). Serum Cu/Zn superoxide dismutase activity is reduced in sporadic amyotrophic lateral sclerosis patients. *Journal of the Neurological Sciences*, 143(1–2), 118–120.

Colby, D. W., & Prusiner, S. B. (2011). Prions. *Cold Spring Harbor Perspectives in Biology*, 3(1), a006833.

Cooper-Knock, J., Higginbottom, A., Stopford, M. J., Highley, J. R., Ince, P. G., Wharton, S. B., et al. (2015). Antisense RNA foci in the motor neurons of C9ORF72-ALS patients are associated with TDP-43 proteinopathy. *Acta Neuropathologica*, 130(1), 63–75.

Crozat, A., Aman, P., Mandahl, N., & Ron, D. (1993). Fusion of CHOP to a novel RNA-binding protein in human myxoid liposarcoma. *Nature*, 363(6430), 640–644.

Custer, S. K., Neumann, M., Lu, H., Wright, A. C., & Taylor, J. P. (2010). Transgenic mice expressing mutant forms VCP/p97 recapitulate the full spectrum of IBMPFD including degeneration in muscle, brain and bone. *Human Molecular Genetics*, 19(9), 1741–1755.

Dai, R. M., & Li, C. C. (2001). Valosin-containing protein is a multi-ubiquitin chain-targeting factor required in ubiquitin-proteasome degradation. *Nature Cell Biology*, 3(8), 740–744.

Daigle, J. G., Lanson, N. A., Jr., Smith, R. B., Casci, I., Maltare, A., Monaghan, J., et al. (2013). RNA-binding ability of FUS regulates neurodegeneration, cytoplasmic mislocalization and incorporation into stress granules associated with FUS carrying ALS-linked mutations. *Human Molecular Genetics*, 22(6), 1193–1205.

Darovic, S., Prpar Mihevc, S., Zupunski, V., Guncar, G., Stalekar, M., Lee, Y. B., et al. (2015). Phosphorylation of C-terminal tyrosine residue 526 in FUS impairs its nuclear import. *Journal of Cell Science*, 128(22), 4151–4159.

Das, I., Krzyzosiak, A., Schneider, K., Wrabetz, L., D’Antonio, M., Barry, N., et al. (2015). Preventing proteostasis diseases by selective inhibition of a phosphatase regulatory subunit. *Science*, 348(6231), 239–242.

Deidda, I., Galizzi, G., Passantino, R., Cascio, C., Russo, D., Colletti, T., et al. (2014). Expression of vesicle-associated membrane-protein-associated protein B cleavage products in peripheral blood leukocytes and cerebrospinal fluid of patients with sporadic amyotrophic lateral sclerosis. *European Journal of Neurology*, 21(3), 478–485.

DeJesus-Hernandez, M., Mackenzie, I. R., Boeve, B. F., Boxer, A. L., Baker, M., Rutherford, N. J., et al. (2011). Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS. *Neuron*, 72(2), 245–256.

DeLaBarre, B., & Brünger, A. T. (2003). Complete structure of p97/valosin-containing protein reveals communication between nucleotide domains. *Nature Structural Biology*, 10(10), 856–863.

Deng, H. X., Chen, W., Hong, S. T., Boycott, K. M., Gorrie, G. H., Siddique, N., et al. (2011). Mutations in UBQLN2 cause dominant X-linked juvenile and adult-onset ALS and ALS/dementia. *Nature*, 477(7363), 211–215.

Deng, H., Gao, K., & Jankovic, J. (2014). The role of FUS gene variants in neurodegenerative diseases. *Nature Reviews. Neurology*, 10(6), 337–348.

Deng, H. X., Hentati, A., Tainer, J. A., Iqbal, Z., Cayabyab, A., Hung, W. Y., et al. (1993). Amyotrophic lateral sclerosis and structural defects in Cu, Zn superoxide dismutase. *Science*, 261(5124), 1047–1051.

Deng, Q., Holler, C. J., Taylor, G., Hudson, K. F., Watkins, W., Gearing, M., et al. (2014). FUS is phosphorylated by DNA-PK and accumulates in the cytoplasm after DNA damage. *Journal of Neuroscience*, 34(23), 7802–7813.

Deng, H. X., Shi, Y., Furukawa, Y., Zhai, H., Fu, R., Liu, E., et al. (2006). Conversion to the amyotrophic lateral sclerosis phenotype is associated with intermolecular linked insoluble aggregates of SOD1 in mitochondria. *Proceedings of the National Academy of Sciences of the United States of America*, 103(18), 7142–7147.

Dolph, P., Nair, A., & Raghu, P. (2011). Electroretinogram recordings of Drosophila. *Cold Spring Harbor Protocols*, 2011(1), pdb.prot5549. http://dx.doi.org/10.1101/pdb.prot5549.

Donnelly, C. J., Zhang, P. W., Pham, J. T., Haeusler, A. R., Mistry, N. A., Videnisky, S., et al. (2013). RNA toxicity from the ALS/FTD C9ORF72 expansion is mitigated by antisense intervention. *Neuron*, 80(2), 415–428.

Dormann, D., & Haass, C. (2013). Fused in sarcoma (FUS): An oncogene goes awry in neurodegeneration. *Molecular and Cellular Neurosciences*, 56, 475–486.

Dormann, D., Madl, T., Valori, C. F., Bentmann, E., Tahirovic, S., Abou-Ajram, C., et al. (2012). Arginine methylation next to the PY-NLS modulates transportin binding and nuclear import of FUS. *EMBO Journal*, 31(22), 4258–4275.

Elsasser, S., & Finley, D. (2005). Delivery of ubiquitinated substrates to protein-unfolding machines. *Nature Cell Biology*, 7(8), 742–749.

Erzberger, J. P., & Berger, J. M. (2006). Evolutionary relationships and structural mechanisms of AAA+ proteins. *Annual Review of Biophysics and Biomolecular Structure*, 35, 93–114.

Estes, P. S., Boehringer, A., Zwick, R., Tang, J. E., Grigsby, B., & Zarnescu, D. C. (2011). Wild-type and A315T mutant TDP-43 exert differential neurotoxicity in a Drosophila model of ALS. *Human Molecular Genetics*, 20(12), 2308–2321.

Fahed, A. C., McDonough, B., Gouvion, C. M., Newell, K. L., Dure, L. S., Bebin, M., et al. (2014). UBQLN2 mutation causing heterogeneous X-linked dominant neurodegeneration. *Annals of Neurology*, 75(5), 793–798.

Fang, F., Valdimarsdottir, U., Bellocchio, R., Ronnevi, L. O., Sparen, P., Fall, K., et al. (2009). Amyotrophic lateral sclerosis in Sweden, 1991–2005. *Archives of Neurology*, 66(4), 515–519.

Feiguin, F., Godena, V. K., Romano, G., D'Ambrogio, A., Klima, R., & Baralle, F. E. (2009). Depletion of TDP-43 affects Drosophila motoneurons terminal synapsis and locomotive behavior. *FEBS Letters*, 583(10), 1586–1592.

Ferrari, R., Kapogiannis, D., Huey, E. D., & Momeni, P. (2013). FTD and ALS: A tale of two diseases. *Current Alzheimer Research*, 8(3), 273–294.

Fiszman, M. L., Borodinsky, L. N., Ricart, K. C., Sanz, O. P., & Sica, R. E. (1999). Cu/Zn superoxide dismutase activity at different ages in sporadic amyotrophic lateral sclerosis. *Journal of the Neurological Sciences*, 162(1), 34–37.

Forrest, S., Chai, A., Sanhueza, M., Marescotti, M., Parry, K., Georgiev, A., et al. (2013). Increased levels of phosphoinositides cause neurodegeneration in a Drosophila model of amyotrophic lateral sclerosis. *Human Molecular Genetics*, 22(13), 2689–2704.

Fratta, P., Poulter, M., Lashley, T., Rohrer, J. D., Polke, J. M., Beck, J., et al. (2013). Homozygosity for the C9orf72 GGGGCC repeat expansion in frontotemporal dementia. *Acta Neuropathologica*, 126(3), 401–409.

Freibaum, B. D., Lu, Y., Lopez-Gonzalez, R., Kim, N. C., Almeida, S., Lee, K. H., et al. (2015). GGGGCC repeat expansion in C9orf72 compromises nucleocytoplasmic transport. *Nature*, 525(7567), 129–133.

Furukawa, Y., Kaneko, K., Yamanaka, K., O’Halloran, T. V., & Nukina, N. (2008). Complete loss of post-translational modifications triggers fibrillar aggregation of SOD1 in the

familial form of amyotrophic lateral sclerosis. *Journal of Biological Chemistry*, 283(35), 24167–24176.

Gardner, B. M., Pincus, D., Gotthardt, K., Gallagher, C. M., & Walter, P. (2013). Endoplasmic reticulum stress sensing in the unfolded protein response. *Cold Spring Harbor Perspectives in Biology*, 5(3), a013169.

Gellera, C., Tilocca, C., Del Bo, R., Corrado, L., Pensato, V., Agostini, J., et al. (2013). Ubiquilin 2 mutations in Italian patients with amyotrophic lateral sclerosis and frontotemporal dementia. *Journal of Neurology, Neurosurgery, and Psychiatry*, 84(2), 183–187.

Gendron, T. F., & Petrucelli, L. (2011). Rodent models of TDP-43 proteinopathy: Investigating the mechanisms of TDP-43-mediated neurodegeneration. *Journal of Molecular Neuroscience*, 45(3), 486–499.

Gomez-Deza, J., Lee, Y. B., Troakes, C., Nolan, M., Al-Sarraj, S., Gallo, J. M., et al. (2015). Dipeptide repeat protein inclusions are rare in the spinal cord and almost absent from motor neurons in C9ORF72 mutant amyotrophic lateral sclerosis and are unlikely to cause their degeneration. *Acta Neuropathologica Communications*, 3, 38.

Gonzalez-Perez, P., Lu, Y., Chian, R. J., Sapp, P. C., Tanzi, R. E., Bertram, L., et al. (2012). Association of UBQLN1 mutation with Brown-Vialetto-Van Laere syndrome but not typical ALS. *Neurobiology of Disease*, 48(3), 391–398.

Grad, L. I., Guest, W. C., Yanai, A., Pokrishevsky, E., O’Neill, M. A., Gibbs, E., et al. (2011). Intermolecular transmission of superoxide dismutase 1 misfolding in living cells. *Proceedings of the National Academy of Sciences of the United States of America*, 108(39), 16398–16403.

Gregory, J. M., Barros, T. P., Meehan, S., Dobson, C. M., & Luheshi, L. M. (2012). The aggregation and neurotoxicity of TDP-43 and its ALS-associated 25 kDa fragment are differentially affected by molecular chaperones in Drosophila. *PLoS One*, 7(2), e31899.

Gregory, R. I., Yan, K. P., Amuthan, G., Chendrimada, T., Doratotaj, B., Cooch, N., et al. (2004). The microprocessor complex mediates the genesis of microRNAs. *Nature*, 432(7014), 235–240.

Gros-Louis, F., Gaspar, C., & Rouleau, G. A. (2006). Genetics of familial and sporadic amyotrophic lateral sclerosis. *Biochimica et Biophysica Acta*, 1762(11–12), 956–972.

Gurney, M. E., Pu, H., Chiu, A. Y., Dal Canto, M. C., Polchow, C. Y., Alexander, D. D., et al. (1994). Motor neuron degeneration in mice that express a human Cu, Zn superoxide dismutase mutation. *Science*, 264(5166), 1772–1775.

Haapasalo, A., Viswanathan, J., Bertram, L., Soininen, H., Tanzi, R. E., & Hiltunen, M. (2010). Emerging role of Alzheimer’s disease-associated ubiquilin-1 in protein aggregation. *Biochemical Society Transactions*, 38(Pt 1), 150–155.

Haeusler, A. R., Donnelly, C. J., Periz, G., Simko, E. A., Shaw, P. G., Kim, M. S., et al. (2014). C9orf72 nucleotide repeat structures initiate molecular cascades of disease. *Nature*, 507(7491), 195–200.

Han, S. M., Tsuda, H., Yang, Y., Vibbert, J., Cottee, P., Lee, S. J., et al. (2012). Secreted VAPB/ALS8 major sperm protein domains modulate mitochondrial localization and morphology via growth cone guidance receptors. *Developmental Cell*, 22(2), 348–362.

Hanson, K. A., Kim, S. H., Wasserman, D. A., & Tibbetts, R. S. (2010). Ubiquilin modifies TDP-43 toxicity in a Drosophila model of amyotrophic lateral sclerosis (ALS). *Journal of Biological Chemistry*, 285(15), 11068–11072.

Hasegawa, M., Arai, T., Nonaka, T., Kametani, F., Yoshida, M., Hashizume, Y., et al. (2008). Phosphorylated TDP-43 in frontotemporal lobar degeneration and amyotrophic lateral sclerosis. *Annals of Neurology*, 64(1), 60–70.

Hetz, C., Thielen, P., Matus, S., Nassif, M., Court, F., Kiffin, R., et al. (2009). XBP-1 deficiency in the nervous system protects against amyotrophic lateral sclerosis by increasing autophagy. *Genes & Development*, 23(19), 2294–2306.

Hicks, G. G., Singh, N., Nashabi, A., Mai, S., Bozek, G., Klewes, L., et al. (2000). Fus deficiency in mice results in defective B-lymphocyte development and activation, high levels of chromosomal instability and perinatal death. *Nature Genetics*, 24(2), 175–179.

Ho, Y. S., Gargano, M., Cao, J., Bronson, R. T., Heimler, I., & Hutz, R. J. (1998). Reduced fertility in female mice lacking copper-zinc superoxide dismutase. *Journal of Biological Chemistry*, 273(13), 7765–7769.

Huang, T. T., Yasunami, M., Carlson, E. J., Gillespie, A. M., Reaume, A. G., Hoffman, E. K., et al. (1997). Superoxide-mediated cytotoxicity in superoxide dismutase-deficient fetal fibroblasts. *Archives of Biochemistry and Biophysics*, 344(2), 424–432.

Ilieva, E. V., Ayala, V., Jove, M., Dalfó, E., Cacabelos, D., Povedano, M., et al. (2007). Oxidative and endoplasmic reticulum stress interplay in sporadic amyotrophic lateral sclerosis. *Brain*, 130(Pt. 12), 3111–3123.

Imlach, W., & McCabe, B. D. (2009). Electrophysiological methods for recording synaptic potentials from the NMJ of Drosophila larvae. *Journal of Visualized Experiments*, 24(24), pii: 1109.

Jackel, S., Summerer, A. K., Thommes, C. M., Pan, X., Voigt, A., Schulz, J. B., et al. (2015). Nuclear import factor transportin and arginine methyltransferase 1 modify FUS neurotoxicity in Drosophila. *Neurobiology of Disease*, 74, 76–88.

Jackrel, M. E., DeSantis, M. E., Martinez, B. A., Castellano, L. M., Stewart, R. M., Caldwell, K. A., et al. (2014). Potentiated Hsp104 variants antagonize diverse proteotoxic misfolding events. *Cell*, 156(1–2), 170–182.

Jarosch, E., Geiss-Friedlander, R., Meusser, B., Walter, J., & Sommer, T. (2002). Protein dislocation from the endoplasmic reticulum—Pulling out the suspect. *Traffic*, 3(8), 530–536.

Jiang, J., Zhu, Q., Gendron, T. F., Saberi, S., McAlonis-Downes, M., Seelman, A., et al. (2016). Gain of toxicity from ALS/FTD-linked repeat expansions in C9ORF72 is alleviated by antisense oligonucleotides targeting GGGGCC-containing RNAs. *Neuron*, 90(3), 535–550.

Johnson, J. O., Mandrioli, J., Benatar, M., Abramzon, Y., Van Deerlin, V. M., Trojanowski, J. Q., et al. (2010). Exome sequencing reveals VCP mutations as a cause of familial ALS. *Neuron*, 68(5), 857–864.

Johnson, B. S., Snade, D., Lee, J. J., McCaffery, J. M., Shorter, J., & Gitler, A. D. (2009). TDP-43 is intrinsically aggregation-prone, and amyotrophic lateral sclerosis-linked mutations accelerate aggregation and increase toxicity. *Journal of Biological Chemistry*, 284(30), 20329–20339.

Jovicic, A., Mertens, J., Boeynaems, S., Bogaert, E., Chai, N., Yamada, S. B., et al. (2015). Modifiers of C9orf72 dipeptide repeat toxicity connect nucleocytoplasmic transport defects to FTD/ALS. *Nature Neuroscience*, 18(9), 1226–1229.

Ju, J. S., Fuentealba, R. A., Miller, S. E., Jackson, E., Piwnica-Worms, D., Baloh, R. H., et al. (2009). Valosin-containing protein (VCP) is required for autophagy and is disrupted in VCP disease. *Journal of Cell Biology*, 187(6), 875–888.

Julien, J. P., & Kriz, J. (2006). Transgenic mouse models of amyotrophic lateral sclerosis. *Biochimica et Biophysica Acta*, 1762(11–12), 1013–1024.

Kabashi, E., El Oussini, H., Bercier, V., Gros-Louis, F., Valdmanis, P. N., McDermid, J., et al. (2013). Investigating the contribution of VAPB/ALS8 loss of function in amyotrophic lateral sclerosis. *Human Molecular Genetics*, 22(12), 2350–2360.

Kagiwada, S., Hosaka, K., Murata, M., Nikawa, J., & Takatsuki, A. (1998). The Saccharomyces cerevisiae SCS2 gene product, a homolog of a synaptobrevin-associated protein, is an integral membrane protein of the endoplasmic reticulum and is required for inositol metabolism. *Journal of Bacteriology*, 180(7), 1700–1708.

Kawano, M., Kumagai, K., Nishijima, M., & Hanada, K. (2006). Efficient trafficking of ceramide from the endoplasmic reticulum to the Golgi apparatus requires a VAMP-associated protein-interacting FFAT motif of CERT. *Journal of Biological Chemistry*, 281(40), 30279–30288.

Kikuchi, H., Almer, G., Yamashita, S., Guegan, C., Nagai, M., Xu, Z., et al. (2006). Spinal cord endoplasmic reticulum stress associated with a microsomal accumulation of mutant superoxide dismutase-1 in an ALS model. *Proceedings of the National Academy of Sciences of the United States of America*, 103(15), 6025–6030.

Kim, H. J., Kim, N. C., Wang, Y. D., Scarborough, E. A., Moore, J., Diaz, Z., et al. (2013). Mutations in prion-like domains in hnRNPA2B1 and hnRNPA1 cause multisystem proteinopathy and ALS. *Nature*, 495(7442), 467–473.

Kim, T. Y., Kim, E., Yoon, S. K., & Yoon, J. B. (2008). Herp enhances ER-associated protein degradation by recruiting ubiquilins. *Biochemical and Biophysical Research Communications*, 369(2), 741–746.

Kim, S. H., Shi, Y., Hanson, K. A., Williams, L. M., Sakasai, R., Bowler, M. J., et al. (2009). Potentiation of amyotrophic lateral sclerosis (ALS)-associated TDP-43 aggregation by the proteasome-targeting factor, ubiquilin 1. *Journal of Biological Chemistry*, 284(12), 8083–8092.

Kimonis, V. E., Kovach, M. J., Waggoner, B., Leal, S., Salam, A., Rimer, L., et al. (2000). Clinical and molecular studies in a unique family with autosomal dominant limb-girdle muscular dystrophy and Paget disease of bone. *Genetics in Medicine*, 2(4), 232–241.

Kleijnen, M. F., Alarcon, R. M., & Howley, P. M. (2003). The ubiquitin-associated domain of hPLIC-2 interacts with the proteasome. *Molecular Biology of the Cell*, 14(9), 3868–3875.

Koppers, M., Blokhuis, A. M., Westeneng, H. J., Terpstra, M. L., Zundel, C. A., Vieira de Sa, R., et al. (2015). C9orf72 ablation in mice does not cause motor neuron degeneration or motor deficits. *Annals of Neurology*, 78(3), 426–438.

Kovach, M. J., Waggoner, B., Leal, S. M., Gelber, D., Khardori, R., Levenstien, M. A., et al. (2001). Clinical delineation and localization to chromosome 9p13.3-p12 of a unique dominant disorder in four families: Hereditary inclusion body myopathy, Paget disease of bone, and frontotemporal dementia. *Molecular Genetics and Metabolism*, 74(4), 458–475.

Kuroda, M., Sok, J., Webb, L., Baechtold, H., Urano, F., Yin, Y., et al. (2000). Male sterility and enhanced radiation sensitivity in TLS(-/-) mice. *EMBO Journal*, 19(3), 453–462.

Kwiatkowski, T. J., Jr., Bosco, D. A., Leclerc, A. L., Tamrazian, E., Vanderburg, C. R., Russ, C., et al. (2009). Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis. *Science*, 323(5918), 1205–1208.

Lanson, N. A., Jr., Maltare, A., King, H., Smith, R., Kim, J. H., Taylor, J. P., et al. (2011). A Drosophila model of FUS-related neurodegeneration reveals genetic interaction between FUS and TDP-43. *Human Molecular Genetics*, 20(13), 2510–2523.

Larroquette, F., Seto, L., Gaub, P. L., Kamal, B., Wallis, D., Lariviere, R., et al. (2015). Vapb/amyotrophic lateral sclerosis 8 knock-in mice display slowly progressive motor behavior defects accompanying ER stress and autophagic response. *Human Molecular Genetics*, 24(22), 6515–6529.

Latterich, M., Frohlich, K. U., & Schekman, R. (1995). Membrane fusion and the cell cycle: Cdc48p participates in the fusion of ER membranes. *Cell*, 82(6), 885–893.

Lee, S., & Kim, H. J. (2015). Prion-like mechanism in amyotrophic lateral sclerosis: Are protein aggregates the key? *Experimental Neurobiology*, 24(1), 1–7.

Leon, A., & McKearin, D. (1999). Identification of TER94, an AAA ATPase protein, as a Bam-dependent component of the Drosophila fusome. *Molecular Biology of the Cell*, 10(11), 3825–3834.

Li, Y., Ray, P., Rao, E. J., Shi, C., Guo, W., Chen, X., et al. (2010). A Drosophila model for TDP-43 proteinopathy. *Proceedings of the National Academy of Sciences of the United States of America*, 107(7), 3169–3174.

Li, H. Y., Yeh, P. A., Chiu, H. C., Tang, C. Y., & Tu, B. P. (2011). Hyperphosphorylation as a defense mechanism to reduce TDP-43 aggregation. *PLoS One*, 6(8), e23075.

Liang, J., Yin, C., Doong, H., Fang, S., Peterhoff, C., Nixon, R. A., et al. (2006). Characterization of erasin (UBXD2): A new ER protein that promotes ER-associated protein degradation. *Journal of Cell Science*, 119(Pt. 19), 4011–4024.

Lilley, B. N., & Ploegh, H. L. (2004). A membrane protein required for dislocation of misfolded proteins from the ER. *Nature*, 429(6994), 834–840.

Lim, P. J., Danner, R., Liang, J., Doong, H., Harman, C., Srinivasan, D., et al. (2009). Ubiquilin and p97/VCP bind erasin, forming a complex involved in ERAD. *Journal of Cell Biology*, 187(2), 201–217.

Lin, M. J., Cheng, C. W., & Shen, C. K. (2011). Neuronal function and dysfunction of Drosophila dTDP. *PLoS One*, 6(6), e20371.

Linford, N. J., Bilgir, C., Ro, J., & Pletcher, S. D. (2013). Measurement of lifespan in Drosophila melanogaster. *Journal of Visualized Experiments*, (71), pii: 50068.

Ling, J. P., Pletnikova, O., Troncoso, J. C., & Wong, P. C. (2015). TDP-43 repression of nonconserved cryptic exons is compromised in ALS-FTD. *Science*, 349(6248), 650–655.

Ling, S. C., Polymenidou, M., & Cleveland, D. W. (2013). Converging mechanisms in ALS and FTD: Disrupted RNA and protein homeostasis. *Neuron*, 79(3), 416–438.

Loewen, C. J., Roy, A., & Levine, T. P. (2003). A conserved ER targeting motif in three families of lipid binding proteins and in Opi1p binds VAP. *EMBO Journal*, 22(9), 2025–2035.

Lukavsky, P. J., Daujotyte, D., Tollervey, J. R., Ule, J., Stuani, C., Buratti, E., et al. (2013). Molecular basis of UG-rich RNA recognition by the human splicing factor TDP-43. *Nature Structural & Molecular Biology*, 20(12), 1443–1449.

Mackenzie, I. R., Bigio, E. H., Ince, P. G., Geser, F., Neumann, M., Cairns, N. J., et al. (2007). Pathological TDP-43 distinguishes sporadic amyotrophic lateral sclerosis from amyotrophic lateral sclerosis with SOD1 mutations. *Annals of Neurology*, 61(5), 427–434.

Mah, A. L., Perry, G., Smith, M. A., & Monteiro, M. J. (2000). Identification of ubiquilin, a novel presenilin interactor that increases presenilin protein accumulation. *Journal of Cell Biology*, 151(4), 847–862.

Majoor-Krakauer, D., Willem, P. J., & Hofman, A. (2003). Genetic epidemiology of amyotrophic lateral sclerosis. *Clinical Genetics*, 63(2), 83–101.

Majounie, E., Renton, A. E., Mok, K., Dopper, E. G., Waite, A., Rollinson, S., et al. (2012). Frequency of the C9orf72 hexanucleotide repeat expansion in patients with amyotrophic lateral sclerosis and frontotemporal dementia: A cross-sectional study. *Lancet Neurology*, 11(4), 323–330.

Matzuk, M. M., Dionne, L., Guo, Q., Kumar, T. R., & Lebovitz, R. M. (1998). Ovarian function in superoxide dismutase 1 and 2 knockout mice. *Endocrinology*, 139(9), 4008–4011.

Mehnert, M., Sommer, T., & Jarosch, E. (2014). Der1 promotes movement of misfolded proteins through the endoplasmic reticulum membrane. *Nature Cell Biology*, 16(1), 77–86.

Mercado, P. A., Ayala, Y. M., Romano, M., Buratti, E., & Baralle, F. E. (2005). Depletion of TDP 43 overrides the need for exonic and intronic splicing enhancers in the human apoA-II gene. *Nucleic Acids Research*, 33(18), 6000–6010.

Meyer, H. H., Wang, Y., & Warren, G. (2002). Direct binding of ubiquitin conjugates by the mammalian p97 adaptor complexes, p47 and Ufd1-Npl4. *EMBO Journal*, 21(21), 5645–5652.

Miguel, L., Frebourg, T., Campion, D., & Lecourtois, M. (2011). Both cytoplasmic and nuclear accumulations of the protein are neurotoxic in Drosophila models of TDP-43 proteinopathies. *Neurobiology of Disease*, 41(2), 398–406.

Miller, J., & Gordon, C. (2005). The regulation of proteasome degradation by multi-ubiquitin chain binding proteins. *FEBS Letters*, 579(15), 3224–3230.

Miller, T. M., Pestronk, A., David, W., Rothstein, J., Simpson, E., Appel, S. H., et al. (2013). An antisense oligonucleotide against SOD1 delivered intrathecally for patients with SOD1 familial amyotrophic lateral sclerosis: A phase 1, randomised, first-in-man study. *Lancet Neurology*, 12(5), 435–442.

Miller, J. W., Smith, B. N., Topp, S. D., Al-Chalabi, A., Shaw, C. E., & Vance, C. (2012). Mutation analysis of VCP in British familial and sporadic amyotrophic lateral sclerosis patients. *Neurobiology of Aging*, 33(11), 2721 e2721–2722.

Mitchell, J. D., & Borasio, G. D. (2007). Amyotrophic lateral sclerosis. *Lancet*, 369(9578), 2031–2041.

Mitne-Neto, M., Machado-Costa, M., Marchetto, M. C., Bengtson, M. H., Joazeiro, C. A., Tsuda, H., et al. (2011). Downregulation of VAPB expression in motor neurons derived from induced pluripotent stem cells of ALS8 patients. *Human Molecular Genetics*, 20(18), 3642–3652.

Mizielinska, S., Gronke, S., Niccoli, T., Ridler, C. E., Clayton, E. L., Devoy, A., et al. (2014). C9orf72 repeat expansions cause neurodegeneration in Drosophila through arginine-rich proteins. *Science*, 345(6201), 1192–1194.

Mori, K., Weng, S. M., Arzberger, T., May, S., Rentzsch, K., Kremmer, E., et al. (2013). The C9orf72 GGGGCC repeat is translated into aggregating dipeptide-repeat proteins in FTLD/ALS. *Science*, 339(6125), 1335–1338.

Moumen, R., Nouvelot, A., Duval, D., Lechevalier, B., & Viader, F. (1997). Plasma superoxide dismutase and glutathione peroxidase activity in sporadic amyotrophic lateral sclerosis. *Journal of the Neurological Sciences*, 151(1), 35–39.

Moustaqim-Barrette, A., Lin, Y. Q., Pradhan, S., Neely, G. G., Bellen, H. J., & Tsuda, H. (2013). The amyotrophic lateral sclerosis 8 protein, VAP, is required for ER protein quality control. *Human Molecular Genetics*, 23(8), 1975–1989.

Muller, J. M., Deinhardt, K., Rosewell, I., Warren, G., & Shima, D. T. (2007). Targeted deletion of p97 (VCP/CDC48) in mouse results in early embryonic lethality. *Biochemical and Biophysical Research Communications*, 354(2), 459–465.

Munch, C., O’Brien, J., & Bertolotti, A. (2011). Prion-like propagation of mutant superoxide dismutase-1 misfolding in neuronal cells. *Proceedings of the National Academy of Sciences of the United States of America*, 108(9), 3548–3553.

Nalbandian, A., Llewellyn, K. J., Badadani, M., Yin, H. Z., Nguyen, C., Katheria, V., et al. (2013). A progressive translational mouse model of human valosin-containing protein disease: The VCP(R155H/+) mouse. *Muscle & Nerve*, 47(2), 260–270.

N’Diaye, E. N., Kajihara, K. K., Hsieh, I., Morisaki, H., Debnath, J., & Brown, E. J. (2009). PLIC proteins or ubiquilins regulate autophagy-dependent cell survival during nutrient starvation. *EMBO Reports*, 10(2), 173–179.

Neumann, M., Kwong, L. K., Lee, E. B., Kremmer, E., Flatley, A., Xu, Y., et al. (2009). Phosphorylation of S409/410 of TDP-43 is a consistent feature in all sporadic and familial forms of TDP-43 proteinopathies. *Acta Neuropathologica*, 117(2), 137–149.

Neumann, M., Mackenzie, I. R., Cairns, N. J., Boyer, P. J., Markesbery, W. R., Smith, C. D., et al. (2007). TDP-43 in the ubiquitin pathology of frontotemporal dementia with VCP gene mutations. *Journal of Neuropathology and Experimental Neurology*, 66(2), 152–157.

Neumann, M., Sampathu, D. M., Kwong, L. K., Truax, A. C., Micsenyi, M. C., Chou, T. T., et al. (2006). Ubiquitinated TDP-43 in frontotemporal lobar degeneration and amyotrophic lateral sclerosis. *Science*, 314(5796), 130–133.

Nichols, C. D., Becnel, J., & Pandey, U. B. (2012). Methods to assay Drosophila behavior. *Journal of Visualized Experiments*, (61), pii: 3795.

Nikawa, J., Murakami, A., Esumi, E., & Hosaka, K. (1995). Cloning and sequence of the SCS2 gene, which can suppress the defect of INO1 expression in an inositol auxotrophic mutant of Saccharomyces cerevisiae. *Journal of Biochemistry*, 118(1), 39–45.

Nishimura, Y., Hayashi, M., Inada, H., & Tanaka, T. (1999). Molecular cloning and characterization of mammalian homologues of vesicle-associated membrane protein-associated (VAMP-associated) proteins. *Biochemical and Biophysical Research Communications*, 254(1), 21–26.

Nishimura, A. L., Mitne-Neto, M., Silva, H. C., Richieri-Costa, A., Middleton, S., Cascio, D., et al. (2004). A mutation in the vesicle-trafficking protein VAPB causes late-onset spinal muscular atrophy and amyotrophic lateral sclerosis. *American Journal of Human Genetics*, 75(5), 822–831.

Nishitoh, H., Kadowaki, H., Nagai, A., Maruyama, T., Yokota, T., Fukutomi, H., et al. (2008). ALS-linked mutant SOD1 induces ER stress- and ASK1-dependent motor neuron death by targeting Derlin-1. *Genes & Development*, 22(11), 1451–1464.

Oh, Y. K., Shin, K. S., Yuan, J., & Kang, S. J. (2008). Superoxide dismutase 1 mutants related to amyotrophic lateral sclerosis induce endoplasmic stress in neuro2a cells. *Journal of Neurochemistry*, 104(4), 993–1005.

O’Rourke, J. G., Bogdanik, L., Muhammad, A. K., Gendron, T. F., Kim, K. J., Austin, A., et al. (2015). C9orf72 BAC transgenic mice display typical pathologic features of ALS/FTD. *Neuron*, 88(5), 892–901.

Orozco, D., & Edbauer, D. (2013). FUS-mediated alternative splicing in the nervous system: Consequences for ALS and FTLD. *Journal of Molecular Medicine (Berlin, Germany)*, 91(12), 1343–1354.

Orozco, D., Tahirovic, S., Rentzsch, K., Schwenk, B. M., Haass, C., & Edbauer, D. (2012). Loss of fused in sarcoma (FUS) promotes pathological Tau splicing. *EMBO Reports*, 13(8), 759–764.

Ou, S. H., Wu, F., Harrich, D., Garcia-Martinez, L. F., & Gaynor, R. B. (1995). Cloning and characterization of a novel cellular protein, TDP-43, that binds to human immunodeficiency virus type 1 TAR DNA sequence motifs. *Journal of Virology*, 69(6), 3584–3596.

Pandey, U. B., Batlevi, Y., Baehrecke, E. H., & Taylor, J. P. (2007). HDAC6 at the intersection of autophagy, the ubiquitin-proteasome system and neurodegeneration. *Autophagy*, 3(6), 643–645.

Pandey, U. B., Nie, Z., Batlevi, Y., McCray, B. A., Ritson, G. P., Nedelsky, N. B., et al. (2007). HDAC6 rescues neurodegeneration and provides an essential link between autophagy and the UPS. *Nature*, 447(7146), 859–863.

Pennetta, G., Hiesinger, P. R., Fabian-Fine, R., Meinertzhagen, I. A., & Bellen, H. J. (2002). Drosophila VAP-33A directs bouton formation at neuromuscular junctions in a dosage-dependent manner. *Neuron*, 35(2), 291–306.

Peretti, D., Dahan, N., Shimoni, E., Hirschberg, K., & Lev, S. (2008). Coordinated lipid transfer between the endoplasmic reticulum and the Golgi complex requires the VAP proteins and is essential for Golgi-mediated transport. *Molecular Biology of the Cell*, 19(9), 3871–3884.

Peters, O. M., Cabrera, G. T., Tran, H., Gendron, T. F., McKeon, J. E., Metterville, J., et al. (2015). Human C9ORF72 hexanucleotide expansion reproduces RNA foci and dipeptide repeat proteins but not neurodegeneration in BAC transgenic mice. *Neuron*, 88(5), 902–909.

Peters, O. M., Ghasemi, M., & Brown, R. H., Jr. (2015). Emerging mechanisms of molecular pathology in ALS. *Journal of Clinical Investigation*, 125(5), 1767–1779.

Phillips, J. P., Campbell, S. D., Michaud, D., Charbonneau, M., & Hilliker, A. J. (1989). Null mutation of copper/zinc superoxide dismutase in Drosophila confers hypersensitivity to

paraquat and reduced longevity. *Proceedings of the National Academy of Sciences of the United States of America*, 86(8), 2761–2765.

Phillips, J. P., Tainer, J. A., Getzoff, E. D., Boulian ne, G. L., Kirby, K., & Hilliker, A. J. (1995). Subunit-destabilizing mutations in Drosophila copper/zinc superoxide dismutase: Neuropathology and a model of dimer dysequilibrium. *Proceedings of the National Academy of Sciences of the United States of America*, 92(19), 8574–8578.

Prusiner, S. B. (1982). Novel proteinaceous infectious particles cause scrapie. *Science*, 216(4542), 136–144.

Rabbitts, T. H., Forster, A., Larson, R., & Nathan, P. (1993). Fusion of the dominant negative transcription regulator CHOP with a novel gene FUS by translocation t(12;16) in malignant liposarcoma. *Nature Genetics*, 4(2), 175–180.

Rabouille, C., Levine, T. P., Peters, J. M., & Warren, G. (1995). An NSF-like ATPase, p97, and NSF mediate cisternal regrowth from mitotic Golgi fragments. *Cell*, 82(6), 905–914.

Rademakers, R., Stewart, H., Dejesus-Hernandez, M., Krieger, C., Graff-Radford, N., Fabros, M., et al. (2010). Fus gene mutations in familial and sporadic amyotrophic lateral sclerosis. *Muscle & Nerve*, 42(2), 170–176.

Rape, M., Hoppe, T., Gorr, I., Kalocay, M., Richly, H., & Jentsch, S. (2001). Mobilization of processed, membrane-tethered SPT23 transcription factor by CDC48(UFD1/NPL4), a ubiquitin-selective chaperone. *Cell*, 107(5), 667–677.

Ratovitski, T., Corson, L. B., Strain, J., Wong, P., Cleveland, D. W., Culotta, V. C., et al. (1999). Variation in the biochemical/biophysical properties of mutant superoxide dismutase 1 enzymes and the rate of disease progression in familial amyotrophic lateral sclerosis kindreds. *Human Molecular Genetics*, 8(8), 1451–1460.

Ravits, J., Laurie, P., Fan, Y., & Moore, D. H. (2007). Implications of ALS focality: Rostral-caudal distribution of lower motor neuron loss postmortem. *Neurology*, 68(19), 1576–1582.

Reaume, A. G., Elliott, J. L., Hoffman, E. K., Kowall, N. W., Ferrante, R. J., Siwek, D. F., et al. (1996). Motor neurons in Cu/Zn superoxide dismutase-deficient mice develop normally but exhibit enhanced cell death after axonal injury. *Nature Genetics*, 13(1), 43–47.

Renton, A. E., Chio, A., & Traynor, B. J. (2014). State of play in amyotrophic lateral sclerosis genetics. *Nature Neuroscience*, 17(1), 17–23.

Renton, A. E., Majounie, E., Waite, A., Simon-Sanchez, J., Rollinson, S., Gibbs, J. R., et al. (2011). A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD. *Neuron*, 72(2), 257–268.

Ritson, G. P., Custer, S. K., Freibaum, B. D., Guinto, J. B., Geffel, D., Moore, J., et al. (2010). TDP-43 mediates degeneration in a novel Drosophila model of disease caused by mutations in VCP/p97. *Journal of Neuroscience*, 30(22), 7729–7739.

Rohrer, J. D., Isaacs, A. M., Mizielinska, S., Mead, S., Lashley, T., Wray, S., et al. (2015). C9orf72 expansions in frontotemporal dementia and amyotrophic lateral sclerosis. *Lancet Neurology*, 14(3), 291–301.

Ron, D., & Walter, P. (2007). Signal integration in the endoplasmic reticulum unfolded protein response. *Nature Reviews. Molecular Cell Biology*, 8(7), 519–529.

Rosen, D. R., Siddique, T., Patterson, D., Figlewicz, D. A., Sapp, P., Hentati, A., et al. (1993). Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis. *Nature*, 362(6415), 59–62.

Rothenberg, C., Srinivasan, D., Mah, L., Kaushik, S., Peterhoff, C. M., Ugolino, J., et al. (2010). Ubiquilin functions in autophagy and is degraded by chaperone-mediated autophagy. *Human Molecular Genetics*, 19(16), 3219–3232.

Rowland, L. P., & Shneider, N. A. (2001). Amyotrophic lateral sclerosis. *The New England Journal of Medicine*, 344(22), 1688–1700.

Saccon, R. A., Bunton-Stasyshyn, R. K., Fisher, E. M., & Fratta, P. (2013). Is SOD1 loss of function involved in amyotrophic lateral sclerosis? *Brain*, 136(Pt. 8), 2342–2358.

Sanhueza, M., Chai, A., Smith, C., McCray, B. A., Simpson, T. I., Taylor, J. P., et al. (2015). Network analyses reveal novel aspects of ALS pathogenesis. *PLoS Genetics*, 11(3), e1005107.

Sanhueza, M., Zechini, L., Gillespie, T., & Pennetta, G. (2013). Gain-of-function mutations in the ALS8 causative gene VAPB have detrimental effects on neurons and muscles. *Biology Open*, 3(1), 59–71.

Sathasivam, S. (2008). Brown-Vialetto-Van Laere syndrome. *Orphanet Journal of Rare Diseases*, 3, 9.

Saxena, S., Cabuy, E., & Caroni, P. (2009). A role for motoneuron subtype-selective ER stress in disease manifestations of FALS mice. *Nature Neuroscience*, 12(5), 627–636.

Scaramuzzino, C., Monaghan, J., Milioto, C., Lanson, N. A., Jr., Maltare, A., Aggarwal, T., et al. (2013). Protein arginine methyltransferase 1 and 8 interact with FUS to modify its sub-cellular distribution and toxicity in vitro and in vivo. *PLoS One*, 8(4), e61576.

Seigneurin-Berny, D., Verdel, A., Curtet, S., Lemercier, C., Garin, J., Rousseaux, S., et al. (2001). Identification of components of the murine histone deacetylase 6 complex: Link between acetylation and ubiquitination signaling pathways. *Molecular and Cellular Biology*, 21(23), 8035–8044.

Smith, R. A., Miller, T. M., Yamanaka, K., Monia, B. P., Condon, T. P., Hung, G., et al. (2006). Antisense oligonucleotide therapy for neurodegenerative disease. *Journal of Clinical Investigation*, 116(8), 2290–2296.

Stewart, H., Rutherford, N. J., Briemberg, H., Krieger, C., Cashman, N., Fabros, M., et al. (2012). Clinical and pathological features of amyotrophic lateral sclerosis caused by mutation in the C9ORF72 gene on chromosome 9p. *Acta Neuropathologica*, 123(3), 409–417.

Stolow, D. T., & Haynes, S. R. (1995). Cabeza, a Drosophila gene encoding a novel RNA binding protein, shares homology with EWS and TLS, two genes involved in human sarcoma formation. *Nucleic Acids Research*, 23(5), 835–843.

Stolz, A., Hilt, W., Buchberger, A., & Wolf, D. H. (2011). Cdc48: A power machine in protein degradation. *Trends in Biochemical Sciences*, 36(10), 515–523.

Subramaniam, J. R., Lyons, W. E., Liu, J., Bartnikas, T. B., Rothstein, J., Price, D. L., et al. (2002). Mutant SOD1 causes motor neuron disease independent of copper chaperone-mediated copper loading. *Nature Neuroscience*, 5(4), 301–307.

Sudria-Lopez, E., Koppers, M., de Wit, M., van der Meer, C., Westeneng, H. J., Zundel, C. A., et al. (2016). Full ablation of C9orf72 in mice causes immune system-related pathology and neoplastic events but no motor neuron defects. *Acta Neuropathologica*, 132(1), 145–147.

Synofzik, M., Maetzler, W., Grehl, T., Prudlo, J., Vom Hagen, J. M., Haack, T., et al. (2012). Screening in ALS and FTD patients reveals 3 novel UBQLN2 mutations outside the PXX domain and a pure FTD phenotype. *Neurobiology of Aging*, 33, 2949 e2913–2947.

Tan, A. Y., & Manley, J. L. (2009). The TET family of proteins: Functions and roles in disease. *Journal of Molecular Cell Biology*, 1(2), 82–92.

Teuling, E., Ahmed, S., Haasdijk, E., Demmers, J., Steinmetz, M. O., Akhmanova, A., et al. (2007). Motor neuron disease-associated mutant vesicle-associated membrane protein-associated protein (VAP) B recruits wild-type VAPs into endoplasmic reticulum-derived tubular aggregates. *Journal of Neuroscience*, 27(36), 9801–9815.

Therrien, M., Rouleau, G. A., Dion, P. A., & Parker, J. A. (2013). Deletion of C9ORF72 results in motor neuron degeneration and stress sensitivity in C. elegans. *PLoS One*, 8(12), e83450.

Tiwari, A., & Hayward, L. J. (2005). Mutant SOD1 instability: Implications for toxicity in amyotrophic lateral sclerosis. *Neurodegenerative Diseases*, 2(3–4), 115–127.

Tobisawa, S., Hozumi, Y., Arawaka, S., Koyama, S., Wada, M., Nagai, M., et al. (2003). Mutant SOD1 linked to familial amyotrophic lateral sclerosis, but not wild-type SOD1, induces ER stress in COS7 cells and transgenic mice. *Biochemical and Biophysical Research Communications*, 303(2), 496–503.

Tran, H., Almeida, S., Moore, J., Gendron, T. F., Chalasani, U., Lu, Y., et al. (2015). Differential toxicity of nuclear RNA foci versus dipeptide repeat proteins in a Drosophila model of C9ORF72 FTD/ALS. *Neuron*, 87(6), 1207–1214.

Tresse, E., Salomons, F. A., Vesa, J., Bott, L. C., Kimonis, V., Yao, T. P., et al. (2010). VCP/p97 is essential for maturation of ubiquitin-containing autophagosomes and this function is impaired by mutations that cause IBMPFD. *Autophagy*, 6(2), 217–227.

Tsuda, H., Han, S. M., Yang, Y., Tong, C., Lin, Y. Q., Mohan, K., et al. (2008). The amyotrophic lateral sclerosis 8 protein VAPB is cleaved, secreted, and acts as a ligand for Eph receptors. *Cell*, 133(6), 963–977.

Tudor, E. L., Galtrey, C. M., Perkinton, M. S., Lau, K. F., De Vos, K. J., Mitchell, J. C., et al. (2010). Amyotrophic lateral sclerosis mutant vesicle-associated membrane protein-associated protein-B transgenic mice develop TAR-DNA-binding protein-43 pathology. *Neuroscience*, 167(3), 774–785.

van Blitterswijk, M., van Es, M. A., Koppers, M., van Rheenen, W., Medic, J., Schelhaas, H. J., et al. (2012). VAPB and C9orf72 mutations in 1 familial amyotrophic lateral sclerosis patient. *Neurobiology of Aging*, 33(12), 2950 e2951–2954.

Vance, C., Rogelj, B., Hortobagyi, T., De Vos, K. J., Nishimura, A. L., Sreedharan, J., et al. (2009). Mutations in FUS, an RNA processing protein, cause familial amyotrophic lateral sclerosis type 6. *Science*, 323(5918), 1208–1211.

Vembar, S. S., & Brodsky, J. L. (2008). One step at a time: Endoplasmic reticulum-associated degradation. *Nature Reviews. Molecular Cell Biology*, 9(12), 944–957.

Voigt, A., Herholz, D., Fiesel, F. C., Kaur, K., Muller, D., Karsten, P., et al. (2010). TDP-43-mediated neuron loss in vivo requires RNA-binding activity. *PLoS One*, 5(8), e12247.

Waite, A. J., Baumer, D., East, S., Neal, J., Morris, H. R., Ansorge, O., et al. (2014). Reduced C9orf72 protein levels in frontal cortex of amyotrophic lateral sclerosis and frontotemporal degeneration brain with the C9ORF72 hexanucleotide repeat expansion. *Neurobiology of Aging*, 35(7), 1779.e5–1779.e13.

Walters, K. J., Kleijnen, M. F., Goh, A. M., Wagner, G., & Howley, P. M. (2002). Structural studies of the interaction between ubiquitin family proteins and proteasome subunit S5a. *Biochemistry*, 41(6), 1767–1777.

Walther, R. F., & Pichaud, F. (2006). Immunofluorescent staining and imaging of the pupal and adult Drosophila visual system. *Nature Protocols*, 1(6), 2635–2642.

Wang, J. W., Brent, J. R., Tomlinson, A., Shneider, N. A., & McCabe, B. D. (2011). The ALS-associated proteins FUS and TDP-43 function together to affect Drosophila locomotion and life span. *Journal of Clinical Investigation*, 121(10), 4118–4126.

Wang, L., Popko, B., & Roos, R. P. (2011). The unfolded protein response in familial amyotrophic lateral sclerosis. *Human Molecular Genetics*, 20(5), 1008–1015.

Wang, L., Popko, B., & Roos, R. P. (2014a). An enhanced integrated stress response ameliorates mutant SOD1-induced ALS. *Human Molecular Genetics*, 23(10), 2629–2638.

Wang, L., Popko, B., Tixier, E., & Roos, R. P. (2014b). Guanabenz, which enhances the unfolded protein response, ameliorates mutant SOD1-induced amyotrophic lateral sclerosis. *Neurobiology of Disease*, 71, 317–324.

Wang, X., Schwartz, J. C., & Cech, T. R. (2015). Nucleic acid-binding specificity of human FUS protein. *Nucleic Acids Research*, 43(15), 7535–7543.

Wate, R., Ito, H., Zhang, J. H., Ohnishi, S., Nakano, S., & Kusaka, H. (2005). Expression of an endoplasmic reticulum-resident chaperone, glucose-regulated stress protein 78, in the spinal cord of a mouse model of amyotrophic lateral sclerosis. *Acta Neuropathologica*, 110(6), 557–562.

Watson, M. R., Lagow, R. D., Xu, K., Zhang, B., & Bonini, N. M. (2008). A drosophila model for amyotrophic lateral sclerosis reveals motor neuron damage by human SOD1. *Journal of Biological Chemistry*, 283(36), 24972–24981.

Watts, G. D., Wymer, J., Kovach, M. J., Mehta, S. G., Mumm, S., Darvish, D., et al. (2004). Inclusion body myopathy associated with Paget disease of bone and frontotemporal

dementia is caused by mutant valosin-containing protein. Nature Genetics, 36(4), 377–381.

Wegorzewska, I., Bell, S., Cairns, N. J., Miller, T. M., & Baloh, R. H. (2009). TDP-43 mutant transgenic mice develop features of ALS and frontotemporal lobar degeneration. Proceedings of the National Academy of Sciences of the United States of America, 106(44), 18809–18814.

Weihl, C. C., Temiz, P., Miller, S. E., Watts, G., Smith, C., Forman, M., et al. (2008). TDP-43 accumulation in inclusion body myopathy muscle suggests a common pathogenic mechanism with frontotemporal dementia. Journal of Neurology, Neurosurgery, and Psychiatry, 79(10), 1186–1189.

Weir, M. L., Klip, A., & Trimble, W. S. (1998). Identification of a human homologue of the vesicle-associated membrane protein (VAMP)-associated protein of 33 kDa (VAP-33): A broadly expressed protein that binds to VAMP. Biochemical Journal, 333(Pt. 2), 247–251.

Williams, K. L., Warrach, S. T., Yang, S., Solski, J. A., Fernando, R., Rouleau, G. A., et al. (2012). UBQLN2/ubiquilin 2 mutation and pathology in familial amyotrophic lateral sclerosis. Neurobiology of Aging, 33(10), 2527.e3–2527.10.

Wojcik, C., Yano, M., & DeMartino, G. N. (2004). RNA interference of valosin-containing protein (VCP/p97) reveals multiple cellular roles linked to ubiquitin/proteasome-dependent proteolysis. Journal of Cell Science, 117(Pt. 2), 281–292.

Wolff, T. (2011). Preparation of Drosophila eye specimens for transmission electron microscopy. Cold Spring Harbor Protocols, 2011(11), 1386–1388.

Wong, E., & Cuervo, A. M. (2010). Integration of clearance mechanisms: The proteasome and autophagy. Cold Spring Harbor Perspectives in Biology, 2(12), a006734.

Wong, P. C., Pardo, C. A., Borchelt, D. R., Lee, M. K., Copeland, N. G., Jenkins, N. A., et al. (1995). An adverse property of a familial ALS-linked SOD1 mutation causes motor neuron disease characterized by vacuolar degeneration of mitochondria. Neuron, 14(6), 1105–1116.

Wu, A. L., Wang, J., Zheleznyak, A., & Brown, E. J. (1999). Ubiquitin-related proteins regulate interaction of vimentin intermediate filaments with the plasma membrane. Molecular Cell, 4(4), 619–625.

Xia, R., Liu, Y., Yang, L., Gal, J., Zhu, H., & Jia, J. (2012). Motor neuron apoptosis and neuromuscular junction perturbation are prominent features in a Drosophila model of Fus-mediated ALS. Molecular Neurodegeneration, 7, 10.

Xia, Y., Yan, L. H., Huang, B., Liu, M., Liu, X., & Huang, C. (2014). Pathogenic mutation of UBQLN2 impairs its interaction with UBXD8 and disrupts endoplasmic reticulum-associated protein degradation. Journal of Neurochemistry, 129(1), 99–106.

Xie, Z., & Klionsky, D. J. (2007). Autophagosome formation: Core machinery and adaptations. Nature Cell Biology, 9(10), 1102–1109.

Yang, L., Gal, J., Chen, J., & Zhu, H. (2014). Self-assembled FUS binds active chromatin and regulates gene transcription. Proceedings of the National Academy of Sciences of the United States of America, 111(50), 17809–17814.

Yang, C., Tan, W., Whittle, C., Qiu, L., Cao, L., Akbarian, S., et al. (2010). The C-terminal TDP-43 fragments have a high aggregation propensity and harm neurons by a dominant-negative mechanism. PLoS One, 5(12), e15878.

Ye, Y., Meyer, H. H., & Rapoport, T. A. (2001). The AAA ATPase Cdc48/p97 and its partners transport proteins from the ER into the cytosol. Nature, 414(6864), 652–656.

Yin, H. Z., Nalbandian, A., Hsu, C. I., Li, S., Llewellyn, K. J., Mozaffar, T., et al. (2012). Slow development of ALS-like spinal cord pathology in mutant valosin-containing protein gene knock-in mice. Cell Death & Disease, 3, e374.

Yoshida, T., Maulik, N., Engelman, R. M., Ho, Y. S., & Das, D. K. (2000). Targeted disruption of the mouse Sod I gene makes the hearts vulnerable to ischemic reperfusion injury. Circulation Research, 86(3), 264–269.

Zhang, K., Donnelly, C. J., Haeusler, A. R., Grima, J. C., Machamer, J. B., Steinwald, P., et al. (2015). The C9orf72 repeat expansion disrupts nucleocytoplasmic transport. *Nature*, 525(7567), 56–61.

Zhang, X., Gui, L., Zhang, X., Bulfer, S. L., Sanghez, V., Wong, D. E., et al. (2015). Altered cofactor regulation with disease-associated p97/VCP mutations. *Proceedings of the National Academy of Sciences of the United States of America*, 112(14), E1705–E1714.

Zhang, C., & Saunders, A. J. (2009). An emerging role for Ubiquilin 1 in regulating protein quality control system and in disease pathogenesis. *Discovery Medicine*, 8(40), 18–22.

Zhang, Y. J., Xu, Y. F., Cook, C., Gendron, T. F., Roettges, P., Link, C. D., et al. (2009). Aberrant cleavage of TDP-43 enhances aggregation and cellular toxicity. *Proceedings of the National Academy of Sciences of the United States of America*, 106(18), 7607–7612.

Zu, T., Gibbens, B., Doty, N. S., Gomes-Pereira, M., Huguet, A., Stone, M. D., et al. (2011). Non-ATG-initiated translation directed by microsatellite expansions. *Proceedings of the National Academy of Sciences of the United States of America*, 108(1), 260–265.

Zu, T., Liu, Y., Banez-Coronel, M., Reid, T., Pletnikova, O., Lewis, J., et al. (2013). RAN proteins and RNA foci from antisense transcripts in C9ORF72 ALS and frontotemporal dementia. *Proceedings of the National Academy of Sciences of the United States of America*, 110(51), E4968–E4977.
