
Oakes et al. Molecular Brain (2017) 10:5  
DOI 10.1186/s13041-017-0287-x

**Molecular Brain**

---

### REVIEW Open Access

**TBK1: a new player in ALS linking autophagy and neuroinflammation**

James A. Oakes<sup>1,2</sup>, Maria C. Davies<sup>2</sup> and Mark O. Collins<sup>2*</sup>

---

#### Abstract

Amyotrophic lateral sclerosis (ALS) is an adult-onset neurodegenerative disorder affecting motor neurons, resulting in progressive muscle weakness and death by respiratory failure. Protein and RNA aggregates are a hallmark of ALS pathology and are thought to contribute to ALS by impairing axonal transport. Mutations in several genes known to contribute to ALS result in deposition of their protein products as aggregates; these include TARDBP, C9ORF72, and SOD1. In motor neurons, this can disrupt transport of mitochondria to areas of metabolic need, resulting in damage to cells and can elicit a neuroinflammatory response leading to further neuronal damage. Recently, eight independent human genetics studies have uncovered a link between TANK-binding kinase 1 (TBK1) mutations and ALS. TBK1 belongs to the IKK-kinase family of kinases that are involved in innate immunity signaling pathways; specifically, TBK1 is an inducer of type-1 interferons. TBK1 also has a major role in autophagy and mitophagy, chiefly the phosphorylation of autophagy adaptors. Several other ALS genes are also involved in autophagy, including p62 and OPTN. TBK1 is required for efficient cargo recruitment in autophagy; mutations in TBK1 may result in impaired autophagy and contribute to the accumulation of protein aggregates and ALS pathology. In this review, we focus on the role of TBK1 in autophagy and the contributions of this process to the pathophysiology of ALS.

#### Keywords: TBK1, Amyotrophic lateral sclerosis, ALS, Motor neuron disease, Autophagy, Mitophagy, Neuroinflammation, Signaling, Frontotemporal dementia, FTD

---

#### Introduction

ALS is a multifactorial disorder with diverse genetic and environmental components [1]. The median incidence rate of ALS is approximately 2/100000 [2], of which 5–10% are familial (fALS). The mean age of onset of ALS is between 50 and 65 years with death occurring on average 2–3 years post-onset, due to respiratory failure [3]. Riluzole is currently the only drug available for treatment of ALS [4]; its modest therapeutic value highlights the imminent need for novel ALS treatments. Environmental factors linked to ALS include smoking and exposure to chemicals metals and radiation [5]. These factors are thought to contribute to ALS by a number of mechanisms; an important common factor is increased oxidative stress in neurons. Other mechanisms include damage to mitochondria, neurotoxicity and increased glutamate excitotoxicity by heavy metals (particularly lead) and cigarette smoke [3]. Mutations in more than 40 genes are thought to contribute to the disease. Amongst the most prevalent causes are SOD1, C9ORF72, TARDBP, FUS, OPTN and the most recently identified ALS gene TANK-binding kinase 1 (TBK1) [1, 6, 7].

##### TBK1 as an ALS gene

Genetic alterations in TBK1 (also known as NAK or T2K) were initially associated with diseases with known neuroinflammatory components, including two forms of glaucoma: primary open angle glaucoma and normal tension glaucoma. Increased TBK1 copy number resulting in a gain of function is recognized as a rare cause of these diseases [8, 9]. Heterozygous loss of function TBK1 mutations are also associated with herpes simplex encephalitis in childhood and it has been suggested that this is due to decreased activity in TLR3 mediated immunity [10]. TBK1 was first identified as an ALS gene by two independent studies. Cirulli et al. [6] performed exome sequencing on 2869 ALS patients and 6405 controls of

* Correspondence: mark.collins@sheffield.ac.uk  
<sup>2</sup>Department of Biomedical Science, University of Sheffield, Firth Court, Western Bank, Sheffield S10 2TN, UK  
Full list of author information is available at the end of the article

© The Author(s). 2017 Open Access This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies to the data made available in this article, unless otherwise stated.

genetically European ethnicity and analyzed variants using a number of inheritance models. This study confirmed several previously identified ALS genes and identified TBK1 as a novel ALS gene [6]. Freischmidt et al. [7] performed exome sequencing and a targeted mutation screen using high resolution melting curve analysis which identified TBK1 as an ALS gene in a Swedish population. Subsequently, a study of Australian fALS patients identified a novel TBK1 mutation in a family of Chinese origin, the first TBK1 mutation found in an Asian ALS patient [11]. This study did not find any TBK1 mutations in patients of European ancestry and concluded that TBK1 mutations are rare in Australian fALS patients [11]. More recently, TBK1 mutations have been found to be a rare cause of ALS in Taiwanese and Chinese populations [12, 13], as well as in Sardinian ALS patients [14]. Frontotemporal dementia (FTD) is a neurodegenerative disorder closely linked to ALS and many patients present with both conditions. In a study of ALS and FTD patients in a French population, TBK1 mutations occurred more frequently in patients with FTD-ALS comorbidity (10.8%) than in those with ALS alone (0.5%) [15–17]. Several other studies have identified TBK1 mutations to be a major cause of FTD either concurrent with or without ALS [12, 14].

These human genetic studies identified nonsense, frameshift, missense and deletion mutations in both sporadic and familial ALS cases (and ALS-FTD/FTD) dispersed throughout the TBK1 protein sequence (Fig. 1a). Nonsense and frameshift mutations cause major disruption to TBK1 and may decrease its expression at both the mRNA and protein level [7, 17], implying that TBK1 haploinsufficiency contributes to the development of ALS in these cases. However, the contribution of missense mutations and single amino acid deletions are more subtle, as they may or may not confer either loss of function or reduction in function. These variants are also less likely to cause a decrease in expression suggesting haploinsufficiency is not to blame in these cases. Elucidation of the functional effects of these variants will contribute to our understanding of ALS pathogenesis as well as TBK1 function as discussed below.

Molecular pathology of ALS

ALS involves the premature loss of upper and lower motor neurons, with degeneration of these neurons in the spinal cord, brainstem and motor cortex [3]. This leads to weakening and atrophy of muscles and eventually paralysis [1, 3]. At the molecular level, numerous factors affect ALS pathology including RNA and protein aggregates, mitochondrial dysfunction and neuroinflammation.

Protein and RNA aggregates

The presence of protein and RNA aggregates in the cytoplasm of motor neurons is the primary hallmark of

A

| KD | ULD | CCD1 | CCD2 |
| --- | --- | --- | --- |
| 1-307 | 308-383 | 407-657 | 658-713 |

B

![Image Description](image_url)

Fig. 1a Primary structure of TBK1. TBK1 is 729 amino acids in length and comprises four main domains: a kinase domain (KD), ubiquitin-like domain (ULD) and two coiled-coil domains (CCD1 and CCD2). Locations of TBK1 mutations associated with ALS and ALS-FTD are indicated by vertical lines: Black—frame shift mutations, orange—nonsense mutations, green—Missense mutations, blue—in-frame deletions, purple—splice mutations and yellow—insertion mutations. b Tertiary structure of TBK1. The region highlighted in red represents the kinase domain, the turquoise region represents the ubiquitin-like domain and the yellow region represents coiled-coil domain 1. Coiled-coil domain 2 is not shown in this model. Diagram produced in PyMOL using crystal structure data published by Larabi et al. 2013 [40] accessed via protein databank (accession number: 4IWO). (RCSB Protein Data Bank, 2016)

ALS. The most common protein inclusion in ALS is TDP-43 (encoded by TARDBP). TARDBP mutations are a cause of both fALS and sporadic ALS (sALS); however, mutations are not a requirement for formation of aggregates, which may be formed from wild-type proteins. TDP-43 is an RNA-binding protein normally found in the nucleus where it has roles in gene expression and RNA processing. However, cytoplasmic aggregates of TDP-43 are frequently observed in ALS and these are often ubiquitinated [1].

C9ORF72, which has roles in stress granule formation, microglial function and autophagy [18–20], contains a hexanucleotide GGGGCC repeat in its first intron; expansion of this repeat is the most frequent cause of fALS [21]. In healthy individuals, the repeat is <30 copies in length, however, in ALS patients the number of repeats may be expanded to hundreds of copies. C9ORF72 expanded repeats result in three types of inclusion: intranuclear RNA foci, cytoplasmic C9ORF72 aggregates, (which may colocalize with p62 and TDP-43), and dipeptide repeat proteins produced by RAN-translation of the expanded region [1, 22]. These abnormal protein aggregates are thought to be the mechanism by which

C9ORF72 expanded repeats contribute to ALS. C9ORF72 transcription in heterozygous 311005021Rik (C9ORF72 mouse ortholog) knock-in C57BL/6 mice is highest in neurons of the hippocampus, cerebellum, cortex, brainstem nuclei and striatum, all of which are known to degenerate in ALS and FTD [23]. This suggests that C9ORF72 repeat expansion related pathology would be most potent in these regions, providing some insight into the cell-specificity of neuronal degeneration in C9ORF72-ALS.

SOD1 was the first ALS gene identified and it is widely studied due to the availability of the SOD1-G93A mouse model of ALS [24]. Mutations in SOD1 produce an unstable protein which is deposited in the cytoplasm; oligomerisation of unstable SOD1 leads to aggregate formation [25]. FUS is another RNA binding protein in which mutations can result in the formation of cytoplasmic aggregates. It is a component of stress granules and may form p62 and TDP-43 positive aggregates [1]. TDP-43 positive protein inclusions, as well as C9ORF72 repeat expansion pathologies, are also involved in FTD and ALS-FTD.

The mechanism by which these inclusions contribute to ALS and FTD pathogenesis is as yet undefined, though they are thought to impair axonal transport [3]. It is also unclear as to whether the inclusions themselves are cytotoxic or if they are secondary to another primary pathology [1, 22]. It is postulated that impaired autophagy could be a contributing factor to the accumulation of cytoplasmic aggregates [26]. TBK1 has important roles in autophagy [27] and it seems likely that it is by this mechanism that TBK1 mutations contribute to ALS.

### Mitochondrial dysfunction

Mitochondria have important roles in cellular respiration, calcium buffering and apoptosis. Neurons are particularly sensitive to mitochondrial dysfunction given their high metabolic rate [28] and the presence of abnormal or dysfunctional mitochondria in neurons is thought to be a contributing factor in ALS. The presence of mutant SOD1 in the cytoplasm of motor neurons plays a major role in mitochondrial dysfunction resulting in impaired ATP production, impaired calcium buffering and early apoptosis of neuronal cells [29–31]. Calcium is critical to the correct functioning of motor neurons as it has major roles in metabolism, development, and synaptic transmission. Early apoptosis is brought about by the interaction of SOD1 and mitochondrial apoptotic machinery, this process directly contributes to motor neuron degeneration [32].

Transport of substances along the length of motor neurons is crucial for normal function and disruption of axonal transport of mitochondria has been observed in both ALS patients and animal models [33, 34].

Mitochondria are transported along motor neurons to areas of greatest metabolic need and calcium regulation; disruption of this process leads to reduced ATP availability and dysregulation of calcium levels resulting in neuron damage. Disruption of axonal transport also potentiates the accumulation of protein and RNA aggregates discussed above, this leads to further impairment of axonal transport resulting in motor neuron degeneration [3]. Damage to motor neurons as a result of protein aggregate accumulation and mitochondrial dysfunction leads to secondary non-neuronal causes of motor neuron degeneration such as neuroinflammation [1].

#### Neuroinflammation

Glial cells are the principle innate immune cell of the CNS and pathology associated with these cells is referred to as neuroinflammation [35], a hallmark of ALS. Glial cells primarily express the Toll-like receptors TLR3 and TLR4 [36]; ligand binding results in activation and migration of glial cells towards sites of damage where they dispose of damaged cells through phagocytosis. A by-product of this process is the production of neurotoxic molecules such as pro-inflammatory cytokines and reactive oxygen species. These molecules may cause further neuronal damage leading to further glial cell activation resulting in a positive feedback loop of neuroinflammation [35]. TBK1 is involved in the innate immune response by regulating the production of IFNα and IFNβ. Ligand binding of TLR3 or TLR4 results in recruitment of adaptor proteins TRIF and TRAM. TRIF interacts with TRAF3 resulting in activation of TBK1/IKKi hetero- or homo-dimers which in turn phosphorylate IRF-3 and IRF-7 allowing the formation of homodimers. IRF-3/IRF-7 homodimers are transported to the nucleus where they act as transcription factors for IFNα and IFNβ [37]. TBK1 is also involved in TLR independent antiviral signaling involving intracellular RIG-I-like receptors (RLRs). Ligand binding of RLRs leads to TBK1 activation through a complex pathway involving the outer mitochondrial membrane (OMM) protein MAVS and the adaptor protein STING [38].

T cells have been observed in spinal cord lesions of ALS patients and are thought to play a role in the regulation of neuroinflammation. CD4+ cells stabilize microglial activation, decrease pro-inflammatory cytokines and increase growth factor IGF-1 suggesting that T cells play a protective role in ALS [35]. Migration of T cells from the lymph nodes is impaired in TBK1 knockout mice [39], potentially resulting in decreased T cell number in the CNS. This may increase the damage caused by neuroinflammation by removing the protective regulation by T cells.

TBK1 structure and function

**Protein structure and regulation**

TBK1 contains four domains: a serine/threonine kinase domain (KD) (residues 1–307) located at its N-terminal, a ubiquitin-like domain (ULD) (residues 308–383) and two coiled-coil domains, coiled-coil domain 1 (CCD1) (residues 407–657) and coiled-coil domain 2 (CCD2) (residues 658–713). http://www.uniprot.org/uniprot/Q9UHD2 [40]. (Fig. 1). CCD1 is also referred to as a scaffold dimerization domain (SDD) [7, 40, 41]. TBK1 may form a homodimer or a heterodimer with IKKi; formation of a homodimer is primarily mediated by interactions between the two CCD1 domains however, the KDs and ULDs also interact with the adjacent molecule. The interacting residues forming the dimer are conserved and are required for activation, a process that requires TBK1 dimerisation [42].

The KD is comprised of two lobes termed N-terminal and C-terminal lobes, with the active site situated between these two lobes. The KD also contains an activation loop (Leu 164–Gly 199) which includes Ser 172, phosphorylation of which brings about TBK1 activation. In the inactive form, the activation loop protrudes away from the KD towards the C-lobe of the KD of the other TBK1 protein comprising the dimer. A conserved residue, Glu 55, is also displaced from the active site. It is speculated that TBK1 dimers are able to autophosphorylate through the interaction of adjacent KDs and activation loops. Phosphorylation of Ser 172 results in a conformational change of the activation loop. The activation loop retracts towards and interacts with its own KD allowing substrate binding. The conserved Glu 55 residue is also rotated into a position where it can form a salt bridge with Lys 38. Other than this, conformational changes as a result of phosphorylation are limited [40, 43].

Poly-ubiquitination of Lys 30 and Lys 401 is a requirement for activation of TBK1 and a multistep mechanism of TBK1 activation beginning with poly-ubiquitination of Lys 30 and Lys 401 followed by phosphorylation of Ser 172 has been suggested [42]. This results in a conformational change of the active site to allow substrate binding. Along with the KD, the ULD is also important for the kinase activity of TBK1; deletion of the ULD results in a loss of kinase activity [44]. Sequence alignment of the TBK1 ULD with similar human ULDs and ULDs of TBK1 from other species has identified many structurally important conserved residues. Three residues, Leu 316, Ile 353, and Val 382, are suspected to be involved in protein-protein interactions [41]. These residues form a hydrophobic patch homologous to the hydrophobic patch of ubiquitin (Leu 316, Ile 44, and Val 70) [41]. Ubiquitin interacts with its binding partners via this hydrophobic patch, indicating that TBK1 may also interact with proteins via this structure. The hydrophobic patch is also thought to be involved in interactions between

the TBK1 ULD and CCD1. Comparison of the surface charge of various regions of ULDs of IKK-family kinases reveals that it differs between family members and these differences may determine substrate specificity [41]. Mutations around the hydrophobic patch have been demonstrated to prevent activation of downstream molecules of TBK1 [41].

TBK1 is regulated by adaptor proteins that control its localization, activation and participation in different signaling pathways [45, 46]. TBK1 possesses the ability to robustly autophosphorylate, and therefore requires strict regulation. This, alongside the fact that TBK1 plays roles in many pathways including induction of interferons and autophagy, means that the subcellular localization of TBK1 may contribute to its regulation as well as its signaling specificity [47]. NAP1, TANK, and Sintbad are adaptor proteins that bind to the CCD2 domain of TBK1. These adaptor proteins bind in a mutually exclusive manner and the differing subcellular localization of these adaptors may determine the pathway in which TBK1 will participate [46]. This is supported by findings that TBK1 activated in autophagy does not result in activation of its downstream targets in the innate immunity signaling, this suggests that there is limited crosstalk between the different TBK1 pathways [48]. NAP1 and Sintbad are localized diffusely throughout the cytoplasm whereas TANK is punctate in the perinuclear region, this has led to suggestions that binding of TANK results in the induction of IFNα and IFNβ whereas binding of NAP1 or Sintbad is more important for autophagy [49]. A network of 30 proteins interacting with one or more of these adaptor proteins and/or TBK1/IKKi has been established [46]. This all suggests that adaptor binding to TBK1 plays a key role in its activation and function within the cell.

**TBK1 mutations**

We collated 92 TBK1 mutations identified in patients with ALS, ALS-FTD or FTD from eight human genetics studies (Table 1, Additional file 1: Table S1 and Fig. 1). 88 of these mutations were identified in ALS patients (ALS and ALS with FTD) of which 27 are potential loss of function variants (nonsense, splice site and frameshift

**Table 1 Breakdown of TBK1 mutations by disease type and protein domains**

| Disease | Domain       | Ubiquitin-like domain | CCD1    | CCD2    |
|---------|--------------|-----------------------|---------|---------|
|         | Kinase domain |                       |          |          |
| ALS     | 32 (84%)     | 11 (100%)             | 29 (71%) | 2 (25%)  |
| ALS-FTD | 2 (5%)       | 0 (0%)                | 9 (22%)  | 4 (50%)  |
| FTD     | 4 (11%)      | 0 (0%)                | 3 (7%)   | 2 (25%)  |
| Total no. cases | 38 | 11 | 41 | 8 |

mutations) and 16 are in-frame insertions/deletions. Freischmidt et al. identified 8 heterozygous loss of function variants, of which 7 resulted in the loss of expression of TBK1 and therefore were reported as causative via haploinsufficiency [7]. The loss of function variant that was expressed (p.690-713del) contained a deletion in the CCD2 domain that prevented binding of OPTN, indicating that this may be sufficient to cause ALS/FTD [7].

The majority of ALS-associated TBK1 variants (45) identified to date are missense mutations (Fig. 2, Additional file 1: Table S1) of unknown pathogenicity. The functional relevance of these variants is less obvious than nonsense or frameshift mutations, which lead to loss of expression of TBK1. Missense mutations may be pathogenic if the site is crucial to the function or stability of the protein. Mutations in the KD may affect phosphorylation of substrates whereas mutations within the ULD may affect recruitment to ubiquitinated proteins and organelles. CCD1 is important for TBK1 dimerisation and mutations here could affect this process, which is required for TBK1 activation [42]. CCD2 domain mutations could interfere with adaptor binding and TBK1 activation [46].

Mutations in the KD and CCD1 account for a greater proportion of disease cases than mutations in the ULD or CCD2 (Table 1, Additional file 1: Table S1) but when domain size is taken into account, mutations are evenly distributed across all domains of TBK1 (Fig. 1). KD, ULD and CCD1 mutations occur more frequently in ALS cases whereas CCD2 mutations seem more likely to result in ALS-FTD or FTD. However, larger numbers of patients with TBK1 mutations are needed to confirm any potential domain-specific associations. Using tools to predict the functional effects of mutations, the largest proportion of missense mutations that are probably damaging, occur in the kinase domain (Additional file 1: Table S1). However, biochemical analysis of variants located in each domain of TBK1 revealed functional deficits in TBK1 in four out of five cases, indicating that many missense mutations have the potential to be pathogenic [7]. Determining which TBK1 missense mutations cause selective loss of function of TBK1 could shed light on the functions of TBK1 most relevant to disease.

### Autophagy and the role of TBK1

#### Autophagy
Autophagy is a process by which ubiquitinated proteins and damaged organelles are degraded and recycled. Abnormal protein aggregates are a hallmark of ALS pathology, in addition to this, mutations in several genes involved autophagy have been associated with ALS including SQSTM1 (encodes p62), SOD1, OPTN, VCP, UBQLN2 and most recently TBK1. This suggests that disruption of autophagy is important in ALS pathophysiology [1, 50].

Autophagy begins with the formation of an immature membrane structure called a phagophore in response to signaling initiated by phosphorylation of the ULK1-ATG13-FIP200 complex [51]. This is normally inhibited by mTORC1, however, the action of mTORC1 can, in turn, be inhibited by AMPK [26]. The activated ULK1-ATG13-FIP200 complex triggers movement of another complex containing beclin1 and PI3K CIII towards a phagophore. This complex mediates elongation of the phagophore membrane and envelopment of proteins or organelles marked for degradation, resulting in the formation of a double-membrane bound autophagosome. The process of autophagosome formation is also mediated by two interlinked control systems, the Atg5-Atg12 conjugation system [52] and the microtubule-associated protein 1A/1B LC3 conjugation system [53]. The Atg5-Atg12 conjugation system results in the formation of a complex of Atg5-Atg12-Atg16. This complex allows conjugation of LC3 with phosphatidyl-ethanolamine to produce LC3-II [50], which then binds to the surface of a phagophore where it has roles in elongation and cargo recruitment [26]. The autophagosome is transported along a microtubule to a lysosome-rich area. The autophagosome fuses with a lysosome forming an autophagolysosome, the contents of which are digested [50] (Fig. 3).

Autophagy adaptors enhance disposal of cargo, such as ubiquitinated proteins and damaged mitochondria, by linking them with autophagosome-associated proteins such as LC3-II [50]. Many of the ALS-associated inclusions described previously contain p62 and ubiquitin. Ubiquitination of proteins marks them for degradation; autophagy adaptors recruit ubiquitinated proteins to the phagophore by linking ubiquitin and LC3-II. p62 and

Oakes et al. Molecular Brain (2017) 10:5

---

**Fig. 3 Autophagolysosome formation and maturation.** Autophagy involves the formation of an isolation membrane. This membrane undergoes elongation, forming a phagophore and a portion of the cytoplasm is separated to form an autophagosome. This involves the ULK1-ATG13-FIP200, Beclin1-PI3K CIII and Atg5-Atg12-Atg16 complexes. The autophagosome fuses with a lysosome to form an autophagolysosome, allowing digestive enzymes to degrade any material in the vesicle. TBK1 binds and phosphorylates autophagy receptors OPTN and p62; they bind to ubiquitin residues on target cargo and to LC3-II. This enhances the ability of the receptors to bind ubiquitinated residues on target cargo and LC3-II. This all allows ubiquitinated cargo to be recruited to the phagophore for degradation; the receptors act as adaptors that link cargo to autophagic machinery.

---

OPTN are autophagy adaptors and mutations in these proteins result in impaired autophagy and are recognized causes of ALS [54, 55]. Importantly, p62, OPTN, and another autophagy adaptor NDP52 are all regulated by phosphorylation by TBK1 [48].

The presence of abnormal protein inclusions in the brain is a common feature of ALS and neurons can respond to this accumulation by upregulating autophagy [56]; if autophagy is disrupted, this may contribute to neurodegeneration and the pathology of ALS. This hypothesis is supported by the loss of autophagy in the CNS of Atg7⁻/⁻ and Atg5⁻/⁻ mice (two genes essential for autophagy), resulting in neurodegenerative disease [57, 58]. Loss of neurons, increased apoptosis and increased protein aggregation were evident when autophagy was decreased [57, 58]. The fact that several ALS genes are involved in autophagy and the knowledge that formation of protein aggregates contribute to ALS pathogenesis, suggests that disruption of autophagy is of importance to the development of ALS. The various roles of TBK1 in autophagy, especially adaptor phosphorylation, suggests that this may also be an important process in the contribution of TBK1 mutations to ALS.

TBK1 also plays a role in innate immunity, chiefly the induction of type-I interferons, alongside its aforementioned role in autophagy. These roles seem to be largely distinct and only limited crosstalk is evident. Interferons, induced by TBK1, lead to the transcription of interferon-inducible genes, amongst the targets of type-I interferons is ISG15, which has been linked to autophagy [59, 60]. ISG15 interacts with p62 and HDAC6, a protein with roles in cargo aggregation and autophagolysosome formation [61]. Overexpression of ISG15 results in enhanced aggregate formation and autophagy [60]. ISG15 can also suppress autophagy by binding beclin1 and PI3K CIII [59]; proteins important in phagophore elongation [26]. Loss of TBK1 decreases ISG15 and therefore removes its suppression of autophagy. In addition to this, enhanced aggregate formation was only observed with overexpression of ISG15 [60]. These factors suggest that ISG15 related mechanisms are probably not important in the contribution of mutant TBK1 to ALS; however, this highlights an interesting link between two TBK1 pathways.

**Autophagy, TBK1 and ALS pathogenesis**

TBK1 plays key roles in autophagy, including the phosphorylation of a number of autophagy adaptors including p62, OPTN and NDP52 [48], enhancing their ability to link LC3-II and ubiquitinated cargo (Fig. 3) [26]. Phosphorylation of OPTN and p62 involves interaction with
the CCD2 domain of TBK1 [15]. Several TBK1 mutations identified in ALS patients cause protein truncation resulting in loss of CCD2 [7], decreasing the ability of TBK1 to phosphorylate these molecules. In addition to this, some TBK1 mutations result in decreased mRNA and protein levels [7, 17], which may decrease activation of autophagy adaptors, resulting in decreased autophagy and accumulation of protein aggregates in motor neurons.

TBK1 has also been implicated in autophagosome maturation. In TBK1 knockdown cells, autophagosome formation was not disrupted whereas maturation of autophagosomes into autophagolysosomes was inhibited [27]. Rab8b is an upstream regulator of TBK1 and they both colocalize with LC3 on autophagosomes; this interaction is proposed as a mechanism by which TBK1 is involved in autophagosome maturation [27]. Maturation of autophagosomes involves transport to a lysosome-rich area via microtubules, this is dependent on the action of the motor protein dynein [62]. TBK1 regulates microtubule dynamics in mitosis and the cytoplasmic levels of dynein [63]. Disruption of microtubule transport due to loss of TBK1 could contribute to ALS by the impaired maturation of autophagosomes into autophagolysosomes.

Several studies have attempted to characterize neuropathological features associated with TBK1-ALS and FTD, though overall the number of individuals examined is still low. An ALS patient carrying a TBK1 mutation has been found with both TDP-43 positive and p62 positive inclusions in motor neurons as well as TDP-43 inclusions in the cortex [64]. An FTD patient from the same study was found to have TDP-43 inclusions in various brain regions as well as cytoplasmic p62 and ubiquitin-positive inclusions in glial cells [64]. Another study found TDP-43 positive inclusions in three out of five TBK1-FTD patients [17] and another found both TDP-43 and p62 positive inclusions in various brain regions of a TBK1-FTD/ALS patient [7]. These findings, particularly of p62 and ubiquitin-positive inclusions, provide further indications that TBK1 mutations may contribute to ALS through impaired autophagy.

### Mitophagy, TBK1 and ALS pathogenesis

Recent studies have identified important roles for TBK1 and other ALS genes in autophagy of mitochondria, specifically known as mitophagy [48, 65]. In the PINK1-Parkin pathway, PINK1 is able to detect damaged mitochondria by crossing the outer mitochondrial membrane (OMM). If the mitochondrion is healthy, PINK1 passes through the OMM and is degraded on the inner mitochondrial membrane (IMM). However, if the mitochondrion is damaged PINK1 is retained on the OMM where it accumulates and phosphorylates ubiquitin chains on several OMM proteins resulting in binding of autophagy adaptors [48]. PINK1 concurrently recruits and phosphorylates Parkin,

resulting in its activation. Parkin is an E3 ubiquitin ligase that constructs ubiquitin chains on OMM proteins, which are then phosphorylated by PINK1, allowing further adaptor binding [65]. TBK1 can then be activated by a mechanism dependent on Parkin, OPTN, NDP52 and OPTN-ubiquitin binding ability [48]. Activated TBK1 can phosphorylate NDP52, OPTN, and p62, greatly enhancing their ability to link ubiquitin and LC3-II [65]. Phosphorylation of OPTN enhances its ubiquitin binding ability and its role in TBK1 activation resulting in a positive feedback mechanism of TBK1 and OPTN activation [48, 65]. NDP52 and OPTN are able to bring about mitophagy by linking ubiquitinated OMM proteins and LC3 proteins on phagophores resulting in engulfment and digestion of mitochondria [26].

The role of p62 in mitophagy is controversial; several studies have reported that p62 is not essential for mitophagy but plays a role in the aggregation of ubiquitinated mitochondria [48, 65]. On the other hand, it has also been reported that activation of p62 is required for efficient mitophagy [66]. TBK1 is required for efficient recruitment of autophagy adaptors and efficient mitophagy, and TBK1 is essential for mitophagy via OPTN [48, 65]. Loss of TBK1 function would result in impaired mitophagy and accumulation of defective mitochondria, which may contribute to ALS by disrupting axonal transport that occurs in ALS [67].

Recent studies have added further weight to the argument that TBK1 mutations contribute to ALS through impaired autophagy/mitophagy. The E696K missense mutation identified in ALS patients [7, 17] occurs in the TBK1 CCD2 domain that interacts with adaptor proteins such as OPTN. This mutation disrupts two hydrogen bonds with H52 and K55 on the N-terminal domain of OPTN, which in turn disrupts the OPTN-TBK1 complex. Co-localization of TBK1 with OPTN is also reduced following the E696K mutation, with co-immunoprecipitation studies showing that the mutation almost completely abolishes the interaction between the two proteins [68]. Whilst wild-type TBK1 localizes to damaged mitochondria, the E696K mutant exhibited severely reduced colocalization with damaged mitochondria [69]. This suggests that this process is dependent on OPTN, due to the loss of TBK1-OPTN binding in the E696K mutant. This potentially provides evidence that defective mitophagy is a pathogenic mechanism in TBK1-ALS.

### Tissue specificity and importance of age in TBK1-ALS

Autophagy can play one of two roles in the cell; to degrade proteins, regardless of cell stress, via continuous operation at low levels (basal autophagy) or to supply amino acids for cell survival during poor environmental conditions (adaptive autophagy) [70]. Basal autophagy in neuronal cells is crucial to their integrity. Most cells can
dilute damaging agents through cell division, whereas the post-mitotic nature of neuronal cells means that they require autophagy to remove toxic proteins [71]. This implies a high level of basal autophagy in these cells, an idea supported by the high autophagic efficiency suggested by the rarity of autophagic vacuoles in healthy neurons [72]. Given the high level of autophagic activity required for neuronal maintenance, it is not surprising that these tissues are particularly susceptible to damage through disruption of autophagy.

TBK1 is normally expressed diffusely in the cytoplasm at moderate levels in all tissues but is expressed at a much higher level in neuronal cells of the cerebral cortex, hippocampus and lateral ventricle [73]. Moderate TBK1 levels are also seen in the glial cells of the cerebral cortex, Purkinje cells and granular layer cells in the cerebellum [73]. ALS is characterized by loss of spinal, cerebellar, hippocampal, brain stem and cortical motor neurons [23], as well as a loss of pyramidal neurons in the primary motor cortex. The fact that TBK1 expression is high in neurons of the CNS which are then lost during ALS, may suggest that loss of function TBK1 mutations may have a greater effect in these cells. TBK1 is also highly expressed in various other tissues, including the lungs, endocrine tissues and skin [73]. This begs the question as to why the major effects of ALS are only observed in nervous tissue. This may be explained by a threshold for development of pathology that may not be reached in these other tissues.

Advanced age appears to be an important factor in the development of ALS and this may be due to a number of factors. A reduction in the number of motor neurons is a normal part of aging and this can contribute to sarcopenia, the age-related loss of muscle mass [74]. In addition to this, the uptake of heavy metals into spinal cord neurons is also increased with advancing age [74]; exposure to heavy metals is a risk factor of ALS due to increased glutamate excitotoxicity [3]. Recovery of motor function is decreased in an aged ALS mouse model when compared to their younger counterparts [75]. Finally, the rate of autophagy declines with age leading to a reduction of the ability of cells to remove protein aggregates [76], or indeed damaged mitochondria which may be particularly detrimental to neurons. These factors may contribute to the tissue specificity of TBK1 involvement in ALS, regardless of its widespread expression. It is unlikely that defects in TBK1 related pathways suddenly appear at some point in the course of advancing age but instead the threshold required to cause disease is lowered to the point where these defects become pathological. In other words, age-related factors such as sarcopenia, heavy metal accumulation, and impaired neuronal recovery may facilitate the progression of ALS due to defective TBK1 signaling.


Future outlook

TBK1 is involved in a variety of ALS-relevant pathways such as autophagy and neuroinflammation. TBK1 function, pathological findings in ALS and known pathogenic mechanisms of ALS point towards autophagy as the major contribution of TBK1 mutations to ALS. Defective autophagy may lead to the accumulation of protein aggregates, autophagosomes and damaged mitochondria in motor neurons. This may all result in impaired axonal transport of molecules and organelles, such as mitochondria, which are crucial for neuron function and survival. Given the varied and crucial roles of TBK1 in autophagy and mitophagy, it seems that these mechanisms may be paramount in the contribution of TBK1 to ALS pathogenesis. Impaired mitochondrial function and transport may result in neuronal damage by the mechanisms discussed above. Neuronal damage may trigger innate responses by cells surrounding neurons leading to neuroinflammation, another important mechanism in ALS pathogenesis. Further dissection of TBK1 signaling pathways in neurons will help further our understanding of the contribution of TBK1 mutations to ALS. Given a large number of downstream targets of TBK1 that have been identified, relatively few have been investigated thoroughly. Studies comparing the pathological findings in different ALS genotypes may also aid our understanding of how different genes/pathways contribute to ALS.


Additional file

**Additional file 1:** Dataset of TBK1 mutations identified in ALS/FTD patients compiled from the literature. (PDF 478 kb)


Abbreviations

ALS: Amyotrophic lateral sclerosis; AMPK: AMP-activated protein kinase; Atg(5,12,13,16): Autophagy-related (5,12,13,16); ATP: Adenosine triphosphate; C9ORF72: Chromosome 9 open reading frame 72; CCD(1,2): Coiled-coil domain; CNS: Central nervous system; fALS: Familial amyotrophic lateral sclerosis; FIP200: Focal adhesion kinase family interacting protein of 200 kD; FTD: Frontotemporal dementia; FUS: Fused in sarcoma; HDAC6: Histone deacetylase 6; IFN(α,β): Interferon; IFN: Interferon; IGF-1: Insulin-like growth factor 1; IKK(α,β,ε/i): IkB kinase; IMM: Inner mitochondrial membrane; IRF(3,7): Interferon regulatory factor; ISG15: Interferon-stimulated gene 15; IkB: Inhibitor of kappa B; KD: Kinase domain; LC3: Microtubule-associated protein 1A/1B-light chain 3; MAVS: Mitochondrial antiviral signalling protein; mTORC1: Mammalian transport of rapamycin complex 1; NAK: NFκB activating kinase; NAP1: Nucleosome assembly protein 1; NDP52: Nuclear dot protein 52 kDa; NFκB: Nuclear factor kappa-light-chain-enhancer of activated B cells; OMM: Outer mitochondrial membrane; OPTN: Optineurin; PI3K CIII: Phosphoinositide 3-kinase class III; PINK1: Phosphatase and tensin homolog (PTEN)-induced putative kinase 1.; RAN-translation: Repeat-associated non-ATG translation; RLR: Rig-I-like receptor; sALS: Sporadic amyotrophic lateral sclerosis; SDD: Scaffold dimerisation domain; SOD1: Superoxide dismutase 1; SQSTM1: Sequestosome-1; STING: Stimulator of interferon genes; T2K: TRAF2 kinase; TANK: TRAF family member-associated NF-kappa-B activator; TARDBP/ TDP-43: Transactive response DNA binding protein 43; TBK1: TANK-binding kinase 1; TLR(3,4): Toll-like receptor; TRAF(2,3): TNF receptor-associated factor; TRAM: Translocating chain-associating membrane protein; TRIF: TIR-domain-containing adapter-inducing interferon-β; UBQLN2: Ubiquilin-2; ULD: Ubiquitin-like domain; ULK1: Unc-51 Like Autophagy Activating Kinase 1; VCP: Valosin-containing protein

Oakes et al. Molecular Brain (2017) 10:5

### Acknowledgements
Not applicable.

### Funding
This work was supported by a research grant (R/144823-11-1) from the Royal Society.

### Availability of data and materials
Not applicable.

### Authors' contributions
JAO, MCD, and MOC wrote the manuscript. All authors read and approved the final manuscript.

### Competing interests
The authors declare that they have no competing interests.

### Consent for publication
Not applicable.

### Ethics approval and consent to participate
As this study did not involve any animal or human participants, human data or human tissue, ethical committee approval is not required.

### Author details
¹School of Medicine, University of Sheffield, Sheffield, UK. ²Department of Biomedical Science, University of Sheffield, Firth Court, Western Bank, Sheffield S10 2TN, UK.

Received: 21 June 2016 Accepted: 24 January 2017  
Published online: 02 February 2017

### References
1. Peters OM, Ghasemi M, Brown Jr RH. Emerging mechanisms of molecular pathology in ALS. J Clin Invest. 2015;125(5):1767–79.
2. Chio A, Logroscino G, Traynor BJ, Collins J, Simeone JC, Goldstein LA, White LA. Global epidemiology of amyotrophic lateral sclerosis: a systematic review of the published literature. Neuroepidemiology. 2013;41(2):118–30.
3. Zarei S, Carr K, Reiley L, Diaz K, Guerra O, Altamirano PF, Pagani W, Lodin D, Orozco G, Chinea A. A comprehensive review of amyotrophic lateral sclerosis. Surg Neurol Int. 2015;6:171.
4. Miller RG, Mitchell JD, Moore DH. Riluzole for amyotrophic lateral sclerosis (ALS)/motor neuron disease (MND). Cochrane Database Syst Rev. 2012;3: Cd001447.
5. Sutedja NA, Veldink JH, Fischer K, Kromhout H, Heederik D, Huisman MH, Wokke JH, van den Berg LH. Exposure to chemicals and metals and risk of amyotrophic lateral sclerosis: a systematic review. Amyotroph Lateral Scler. 2009;10(5–6):302–9.
6. Cirulli ET, Lasseigne BN, Petrovski S, Sapp PC, Dion PA, Leblond CS, Couthouis J, Lu YF, Wang Q, Krueger BJ, et al. Exome sequencing in amyotrophic lateral sclerosis identifies risk genes and pathways. Science. 2015;347(6229):1436–41.
7. Freischmidt A, Wieland T, Richter B, Ruf W, Schaeffer V, Muller K, Marroquin N, Nordin F, Hubers A, Weydt P, et al. Haploinsufficiency of TBK1 causes familial ALS and fronto-temporal dementia. Nat Neurosci. 2015;18(5):631–6.
8. Ritch R, Darbro B, Menon G, Khanna CL, Solivan-Timpe F, Roos BR, Sarfarzi M, Kawase K, Yamamoto T, Robin AL, et al. TBK1 gene duplication and normal-tension glaucoma. JAMA Ophthalmol. 2014;132(5):544–8.
9. Awadalla MS, Fingert JH, Roos BE, Chen S, Holmes R, Graham SL, Chehadé M, Galanopoulos A, Ridge B, Souzeau E, et al. Copy number variations of TBK1 in Australian patients with primary open-angle glaucoma. Am J Ophthalmol. 2015;159(1):124–30.e121.
10. Herman M, Ciancanelli M, Ou YH, Lorenzo L, Klaudel-Dreszler M, Pauwels E, Sancho-Shimizu V, Perez de Diego R, Abhyankar A, Israelsson E, et al. Heterozygous TBK1 mutations impair TLR3 immunity and underlie herpes simplex encephalitis of childhood. J Exp Med. 2012;209(9):1567–82.
11. Williams KL, McCann EP, Fifita JA, Zhang K, Duncan EL, Leo PJ, Marshall M, Rowe DB, Nicholson GA, Blair IP. Novel TBK1 truncating mutation in a familial amyotrophic lateral sclerosis patient of Chinese origin. Neurobiol Aging. 2015;36(12):3334.e3331–3335.

12. Tsai PC, Liu YC, Lin KP, Liu YT, Liao YC, Hsiao CT, Soong BW, Yip PK, Lee YC. Mutational analysis of TBK1 in Taiwanese patients with amyotrophic lateral sclerosis. Neurobiol Aging. 2016;40:191.e11–6.
13. Shu S, Li XL, Liu Q, Liu F, Cui B, Liu MS, Cui LY, Li XG, Zhang X. Screening of the TBK1 gene in familial and sporadic amyotrophic lateral sclerosis patients of Chinese origin. Amyotroph Lateral Scler Frontotemporal Degener. 2016;17(7–8):605–607.
14. Borghero G, Pugliatti M, Marrosu F, Marrosu MG, Murru MR, Floris G, Cannas A, Occhineri P, Cau TB, Loi D, et al. TBK1 is associated with ALS and ALS-FTD in Sardinian patients. Neurobiol Aging. 2016;43:180.e181–185.
15. Le Ber I, De Septenville A, Millecamps S, Camuzat A, Caroppo P, Couratier P, Blanc F, Lacomblez L, Sellal F, Fleury MC, et al. TBK1 mutation frequencies in French frontotemporal dementia and amyotrophic lateral sclerosis cohorts. Neurobiol Aging. 2015;36(11):3116.e3115–3118.
16. Gijselinck I, Van Mossevelde S, van der Zee J, Sieben A, Philtjens S, Heeman B, Engelborghs S, Vandenbulcke M, De Baets G, Baumer V, et al. Loss of TBK1 is a frequent cause of frontotemporal dementia in a Belgian cohort. Neurology. 2015;85(24):2116–25.
17. Pottier C, Bieniek KF, Finch N, van de Vorst M, Baker M, Perkerson R, Brown P, Ravenscroft T, van Blitterswijk M, Nicholson AM, et al. Whole-genome sequencing reveals important role for TBK1 and OPTN mutations in frontotemporal lobar degeneration without motor neuron disease. Acta Neuropathol. 2015;130(1):77–92.
18. Maharjan N, Kunzli C, Butheyl K, Saxena S. C9ORF72 Regulates stress granule formation and its deficiency impairs stress granule assembly, hypersensitizing cells to stress. Mol Neurobiol. 2016. doi:10.1007/s12035-016-9850-1.
19. O'Rourke JG, Bogdanik L, Yanez A, Lall D, Wolf AJ, Muhammad AK, Ho R, Carmona S, Vit JP, Zarrow J, et al. C9orf72 is required for proper macrophage and microglial function in mice. Science. 2016;351(6279):1324–9.
20. Sullivan PM, Zhou X, Robins AM, Paushter DH, Kim D, Smolkal MB, Hu F. The ALS/FTLD associated protein C9orf72 associates with SMCR8 and WDR41 to regulate the autophagy-lysosome pathway. Acta Neuropathol Commun. 2016;4(1):51.
21. DeJesus-Hernandez M, Mackenzie IR, Boeve BF, Boxer AL, Baker M, Rutherford NJ, Nicholson AM, Finch NA, Flynn H, Adamson J, et al. Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS. Neuron. 2011;72(2):245–56.
22. Lee YB, Chen HJ, Peres JN, Gomez-Deza J, Attig J, Stalekar M, Troakes C, Nishimura AL, Scotter EL, Vance C, et al. Hexanucleotide repeats in ALS/FTD form length-dependent RNA foci, sequester RNA binding proteins, and are neurotoxic. Cell Rep. 2013;5(5):1178–86.
23. Suzuki N, Marof AM, Merkle FT, Koszka K, Into A, Armstrong I, Moccia R, Davis-Dusenbery BN, Eggan K. The mouse C9ORF72 ortholog is enriched in neurons known to degenerate in ALS and FTD. Nat Neurosci. 2013;16(12):1725–7.
24. Rosen DR, Siddique T, Patterson D, Figlewicz DA, Sapp P, Hentati A, Donaldson D, Goto J, O'Regan JP, Deng HX, et al. Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis. Nature. 1993;362(6415):59–62.
25. Lindberg MJ, Bystrom R, Boknas N, Andersen PM, Oliveberg M. Systematically perturbed folding patterns of amyotrophic lateral sclerosis (ALS)-associated SOD1 mutants. Proc Natl Acad Sci U S A. 2005;102(28):9754–9.
26. Kiriyama Y, Nochi H. The Function of Autophagy in Neurodegenerative Diseases. Int J Mol Sci. 2015;16(11):26797–812.
27. Pilli M, Arko-Mensah J, Ponpuak M, Roberts E, Master S, Mandell MA, Dupont N, Ornatowski W, Jiang S, Bradfute SB, et al. TBK-1 promotes autophagy-mediated antimicrobial defense by controlling autophagosome maturation. Immunity. 2012;37(2):223–34.
28. Kann O, Kovacs R. Mitochondria and neuronal activity. Am J Physiol Cell Physiol. 2007;292(2):C641–657.
29. Bowling AC, Schulz JB, Brown Jr RH, Beal MF. Superoxide dismutase activity, oxidative damage, and mitochondrial energy metabolism in familial and sporadic amyotrophic lateral sclerosis. J Neurochem. 1993;61(6):2322–5.
30. Kong J, Xu Z. Massive mitochondrial degeneration in motor neurons triggers the onset of amyotrophic lateral sclerosis in mice expressing a mutant SOD1. J Neurosci. 1998;18(9):3241–50.
31. Liu J, Lillo C, Jonsson PA, Vande Velde C, Ward CM, Miller TM, Subramaniam JR, Rothstein JD, Marklund S, Andersen PM, et al. Toxicity of familial ALS-linked SOD1 mutants from selective recruitment to spinal mitochondria. Neuron. 2004;43(1):5–17.
32. Mattson MP. Apoptosis in neurodegenerative disorders. Nat Rev Mol Cell Biol. 2000;1(2):120–9.

33. Morotz GM, De Vos KJ, Vagnoni A, Ackerley S, Shaw CE, Miller CC. Amyotrophic lateral sclerosis-associated mutant VAPBP56S perturbs calcium homeostasis to disrupt axonal transport of mitochondria. Hum Mol Genet. 2012;21(9):1979–88.
34. Magrane J, Cortez C, Gan WB, Manfredi G. Abnormal mitochondrial transport and morphology are common pathological denominators in SOD1 and TDP43 ALS mouse models. Hum Mol Genet. 2014;23(6):1413–24.
35. Komine O, Yamanaka K. Neuroinflammation in motor neuron disease. Nagoya J Med Sci. 2015;77(4):537–49.
36. Carpentier PA, Duncan DS, Miller SD. Glial toll-like receptor signaling in central nervous system infection and autoimmunity. Brain Behav Immun. 2008;22(2):140–7.
37. Kawai T, Akira S. Signaling to NF-kappaB by Toll-like receptors. Trends Mol Med. 2007;13(11):460–9.
38. Cui J, Chen Y, Wang HY, Wang RF. Mechanisms and pathways of innate immune activation and regulation in health and cancer. Hum Vaccin Immunother. 2014;10(11):3270–85.
39. Yu J, Zhou X, Chang M, Nakaya M, Chang JH, Xiao Y, Lindsey JW, Dorta-Estremera S, Cao W, Zal A, et al. Regulation of T-cell activation and migration by the kinase TBK1 during neuroinflammation. Nat Commun. 2015;6:6074.
40. Larabi A, Devos JM, Ng SL, Nanao MH, Round A, Maniatis T, Panne D. Crystal structure and mechanism of activation of TANK-binding kinase 1. Cell Rep. 2013;3(3):734–46.
41. Li J, Miyahira A, Sun J, Liu Y, Cheng G, Liang H. Crystal structure of the ubiquitin-like domain of human TBK1. Protein Cell. 2012;3(5):383–91.
42. Tu D, Zhu Z, Zhou AY, Yun CH, Lee KE, Toms AV, Li Y, Dunn GP, Chan E, Thai T, et al. Structure and ubiquitination-dependent activation of TANK-binding kinase 1. Cell Rep. 2013;3(3):747–58.
43. Ma X, Helgason E, Phung QT, Quan CL, Iyer RS, Lee MW, Bowman KK, Starovasnik MA, Dueber EC. Molecular basis of Tank-binding kinase 1 activation by transautophosphorylation. Proc Natl Acad Sci USA. 2012;109(24):9378–83.
44. Ikeda F, Hecker CM, Rozenknop A, Nordmeier RD, Rogov V, Hofmann K, Akira S, Dotsch V, Dikic I. Involvement of the ubiquitin-like domain of TBK1/IKK-i kinases in regulation of IFN-inducible genes. EMBO J. 2007;26(14):3451–62.
45. Kim JY, Welsh EA, Oguz U, Fang B, Bai Y, Kinose F, Bronk C, Remsing Rix LL, Beg AA, Rix U, et al. Dissection of TBK1 signaling via phosphoproteomics in lung cancer cells. Proc Natl Acad Sci USA. 2013;110(30):12414–9.
46. Goncalves A, Burckstummer T, Dixit E, Scheicher R, Gorna MW, Karayel E, Sugar C, Stukalov A, Berg T, Kralovics R, et al. Functional dissection of the TBK1 molecular network. PLoS One. 2011;6(9):e23971.
47. Helgason E, Phung QT, Dueber EC. Recent insights into the complexity of Tank-binding kinase 1 signaling networks: the emerging role of cellular localization in the activation and substrate specificity of TBK1. FEBS Lett. 2013;587(8):1230–7.
48. Heo JM, Ordureau A, Paulo JA, Rinehart J, Harper JW. The PINK1-PARKIN Mitochondrial Ubiquitylation Pathway Drives a Program of OPTN/NDP52 Recruitment and TBK1 Activation to Promote Mitophagy. Mol Cell. 2015;60(1):7–20.
49. Thurston TL, Ryzhakov G, Bloor S, von Muhlinen N, Randow F. The TBK1 adaptor and autophagy receptor NDP52 restricts the proliferation of ubiquitin-coated bacteria. Nat Immunol. 2009;10(11):1215–21.
50. Lee JK, Shin JH, Lee JE, Choi EJ. Role of autophagy in the pathogenesis of amyotrophic lateral sclerosis. Biochim Biophys Acta. 2015;1852(11):2517–24.
51. Jung CH, Jun CB, Ro SH, Kim YM, Otto NM, Cao J, Kundu M, Kim DH. ULK-Atg13-FIP200 complexes mediate mTOR signaling to the autophagy machinery. Mol Biol Cell. 2009;20(7):1992–2003.
52. Mizushima N, Noda T, Yoshimori T, Tanaka Y, Ishii T, George MD, Klionsky DJ, Ohsumi M, Ohsumi Y. A protein conjugation system essential for autophagy. Nature. 1998;395(6700):395–8.
53. Ichimura Y, Kirisako T, Takao T, Satomi Y, Shimonishi Y, Ishihara N, Mizushima N, Tanida I, Kominami E, Ohsumi M, et al. A ubiquitin-like system mediates protein lipidation. Nature. 2000;408(6811):488–92.
54. Fecto F, Yan J, Vemula SP, Liu E, Yang Y, Chen W, Zheng JG, Shi Y, Siddique N, Arrat H, et al. SQSTM1 mutations in familial and sporadic amyotrophic lateral sclerosis. Arch Neurol. 2011;68(11):1440–6.
55. Maruyama H, Morino H, Ito H, Izumi Y, Kato H, Watanabe Y, Kinoshita Y, Kamada M, Nodera H, Suzuki H, et al. Mutations of optineurin in amyotrophic lateral sclerosis. Nature. 2010;465(7295):223–6.
56. Morimoto N, Nagai M, Ohta Y, Miyazaki K, Kurata T, Morimoto M, Murakami T, Takehisa Y, Ikeda Y, Kamiya T, et al. Increased autophagy in transgenic mice with a G93A mutant SOD1 gene. Brain Res. 2007;1167:112–7.
57. Komatsu M, Waguri S, Chiba T, Murata S, Iwata J, Tanida I, Ueno T, Koike M, Uchiyama Y, Kominami E, et al. Loss of autophagy in the central nervous system causes neurodegeneration in mice. Nature. 2006;441(7095):880–4.
58. Hara T, Nakamura K, Matsui M, Yamamoto A, Nakahara Y, Suzuki-Migishima R, Yokoyama M, Mishima K, Saito I, Okano H, et al. Suppression of basal autophagy in neural cells causes neurodegenerative disease in mice. Nature. 2006;441(7095):885–9.
59. Xu D, Zhang T, Xiao J, Zhu K, Wei R, Wu Z, Meng H, Li Y, Yuan J. Modification of BECN1 by ISG15 plays a crucial role in autophagy regulation by type I IFN/ interferon. Autophagy. 2015;11(4):617–28.
60. Nakashima H, Nguyen T, Goins WF, Chiocca EA. Interferon-stimulated gene 15 (ISG15) and ISG15-linked proteins can associate with members of the selective autophagic process, histone deacetylase 6 (HDAC6) and SQSTM1/p62. J Biol Chem. 2015;290(3):1485–95.
61. Lee JY, Koga H, Kawaguchi Y, Tang W, Wong E, Gao YS, Pandey UB, Kaushik S, Tresse E, Lu J, et al. HDAC6 controls autophagosome maturation essential for ubiquitin-selective quality-control autophagy. EMBO J. 2010;29(5):969–80.
62. Kimura S, Noda T, Yoshimori T. Dynein-dependent movement of autophagosomes mediates efficient encounters with lysosomes. Cell Struct Funct. 2008;33(1):109–22.
63. Pillai S, Nguyen J, Johnson J, Haura E, Coppola D, Chellappan S. Tank binding kinase 1 is a centrosome-associated kinase necessary for microtubule dynamics and mitosis. Nat Commun. 2015;6:10072.
64. Van Mossevelde S, van der Zee J, Gijselinck I, Engelborghs S, Sieben A, Van Langenhove T, De Bleecker J, Baets J, Vandenbulcke M, Van Laere K, et al. Clinical features of TBK1 carriers compared with C9orf72, GRN and non-mutation carriers in a Belgian cohort. Brain. 2015.
65. Lazarou M, Sliter DA, Kane LA, Sarraf SA, Wang C, Burman JL, Sideris DP, Fogel AI, Youle RJ. The ubiquitin kinase PINK1 recruits autophagy receptors to induce mitophagy. Nature. 2015;524(7565):309–14.
66. Matsumoto G, Shimogori T, Hattori N, Nukina N. TBK1 controls autophagosomal engulfment of polyubiquitinated mitochondria through p62/SQSTM1 phosphorylation. Hum Mol Genet. 2015;24(15):4429–42.
67. Sasaki S, Iwata M. Impairment of fast axonal transport in the proximal axons of anterior horn neurons in amyotrophic lateral sclerosis. Neurology. 1996;47(2):535–40.
68. Li F, Xie X, Wang Y, Liu J, Cheng X, Guo Y, Gong Y, Hu S, Pan L. Structural insights into the interaction and disease mechanism of neurodegenerative disease-associated optineurin and TBK1 proteins. Nat Commun. 2016;7:12708.
69. Richter B, Sliter DA, Herhaus L, Stolz A, Wang C, Beli P, Zaffagnini G, Wild P, Martens S, Wagner SA, et al. Phosphorylation of OPTN by TBK1 enhances its binding to Ub chains and promotes selective autophagy of damaged mitochondria. Proc Natl Acad Sci USA. 2016;113(15):4039–44.
70. Sasaki S. Autophagy in spinal cord motor neurons in sporadic amyotrophic lateral sclerosis. J Neuropathol Exp Neurol. 2011;70(5):349–59.
71. Son JH, Shim JH, Kim KH, Ha JY, Han JY. Neuronal autophagy and neurodegenerative diseases. Exp Mol Med. 2012;44(2):89–98.
72. Boland B, Nixon RA. Neuronal macroautophagy: from development to degeneration. Mol Aspects Med. 2006;27(5–6):503–19.
73. Uhlen M, Fagerberg L, Hallstrom BM, Lindskog C, Oksvold P, Mardinoglu A, Sivertsson A, Kampf C, Sjostedt E, Asplund A, et al. Proteomics. Tissue-based map of the human proteome. Science. 2015;347(6220):1260419.
74. Roubenoff R. Sarcopenia and its implications for the elderly. Eur J Clin Nutr. 2000;54 Suppl 3:S40–47.
75. Spiller KJ, Restrepo CR, Khan T, Stieber AM, Kwong LK, Trojanowski JQ, Lee VM. Progression of motor neuron disease is accelerated and the ability to recover is compromised with advanced age in rNLS8 mice. Acta Neuropathol Commun. 2016;4(1):105.
76. Zhang C, Cuervo AM. Restoration of chaperone-mediated autophagy in aging liver improves cellular maintenance and hepatic function. Nat Med. 2008;14(9):959–65.
