
Neuron
Review

# Autophagy and Neurodegeneration: Pathogenic Mechanisms and Therapeutic Opportunities

Fiona M. Menzies,$^{1,3,4}$ Angeleen Fleming,$^{1,3}$ Andrea Caricasole,$^{2,3}$ Carla F. Bento,$^{1,3}$ Stephen P. Andrews,$^{2,3}$ Avraham Ashkenazi,$^{1,3}$ Jens Füllgrabe,$^{1,3}$ Anne Jackson,$^{1,3}$ Maria Jimenez Sanchez,$^{1,3}$ Cansu Karabiyik,$^{1,3}$ Floriana Licitra,$^{1,3}$ Ana Lopez Ramirez,$^{1,3}$ Mariana Pavel,$^{1,3}$ Claudia Puri,$^{1,3}$ Maurizio Renna,$^{1,3}$ Thomas Ricketts,$^{1,3}$ Lars Schlotawa,$^{1,3}$ Mariella Vicinanza,$^{1,3}$ Hyeran Won,$^{1,3}$ Ye Zhu,$^{1,3}$ John Skidmore,$^{2,3}$ and David C. Rubinsztein$^{1,5,*}$

$^{1}$Department of Medical Genetics, Cambridge Institute for Medical Research, Cambridge Biomedical Campus, Wellcome Trust/MRC Building, Cambridge Biomedical Campus, Hills Road, Cambridge CB2 0XY, UK  
$^{2}$Alzheimer’s Research UK Cambridge Drug Discovery Institute, University of Cambridge, Cambridge Biomedical Campus, Hills Road, Cambridge CB2 0AH, UK  
$^{3}$Co-first author  
$^{4}$Present address: Eli Lilly and Company Limited, Erl Wood Manor, Sunninghill Road, Windlesham, Surrey GU20 6PH, UK  
$^{5}$Lead Contact  
*Correspondence: dcr1000@cam.ac.uk  
http://dx.doi.org/10.1016/j.neuron.2017.01.022

Autophagy is a conserved pathway that delivers cytoplasmic contents to the lysosome for degradation. Here we consider its roles in neuronal health and disease. We review evidence from mouse knockout studies demonstrating the normal functions of autophagy as a protective factor against neurodegeneration associated with intracytoplasmic aggregate-prone protein accumulation as well as other roles, including in neuronal stem cell differentiation. We then describe how autophagy may be affected in a range of neurodegenerative diseases. Finally, we describe how autophagy upregulation may be a therapeutic strategy in a wide range of neurodegenerative conditions and consider possible pathways and druggable targets that may be suitable for this objective.

## Autophagy Cell Biology

Macroautophagy (henceforth called autophagy) is a major intracellular protein degradation pathway whereby cytoplasmic contents are delivered, by double-membraned vesicles called autophagosomes, to the lysosome for degradation. It should be differentiated from other pathways that will not be considered in this review, like chaperone-mediated autophagy and microautophagy, where substrates are directly translocated into the lysosome without vesicular transport. The first morphologically characteristic structure in autophagy is the double-membraned, cup-shaped autophagosome precursor, called the phagophore, that engulfs substrates as its edges extend. After the phagophore edges close to form a vesicle, the completed autophagosomes traffic along microtubules to enable autophagosome-lysosome fusion, which leads to the degradation of the autophagic contents (Figure 1). Autophagy is regulated by a series of proteins defined as autophagy-related (ATG) proteins.

Autophagy was initially characterized as a bulk and non-selective degradation pathway induced by nutrient deprivation. However, more recent studies made clear that autophagy also contributes to intracellular homeostasis in non-starved cells by degrading cargo material such as aggregate-prone proteins, including those causing many neurodegenerative conditions (aggrephagy), damaged mitochondria (mitophagy), excess peroxisomes (pexophagy), and invading pathogens (xenophagy) (Stolz et al., 2014). In the classic example, aggregates of aberrantly folded proteins are tagged with ubiquitin chains that are recognized by ubiquitin-binding domain-containing receptors such as Sequestosome 1 (SQSTM1)/p62 (Bjørkøy et al., 2005), neighbor of BRCA1 gene 1 (NBR1) (Kirkin et al., 2009), optineurin

(OPTN) (Wild et al., 2011), Tax1 binding protein 1 (TAX1BP1), nuclear dot protein 52 (NDP52)/CALCOCO2 ((Newman et al., 2012), TOLL-interacting protein (TOLLIP) (Lu et al., 2014), and 26S proteasome regulatory subunit (RPN10) (Marshall et al., 2015; Khaminets et al., 2016; Stolz et al., 2014). These receptors also contain LC3-interacting region (LIR) motifs in their sequences that can recognize the key autophagosome-associated protein 1 light chain 3 (LC3). Thus, these receptors serve as a bridge between ubiquitinated cargo and autophagosomes and enhance the incorporation of cargo into autophagosomes for subsequent lysosomal degradation (Bjørkøy et al., 2005; Pankiv et al., 2007).

This concept of selective autophagy has now been extended with the description of precision autophagy. This is defined by the involvement of receptors with the ability to recognize LC3 family member proteins, their specific substrates, and also, importantly, the capacity to act as platforms for the assembly of other core ATG factors (Kimura et al., 2015). To date, receptors identified in this class come from the tripartite motif-containing (TRIM) family, a diverse group of proteins associated with the degradation of targets such as innate immunity signaling molecules (reviewed by Kimura et al., 2016). It seems likely that there are many other molecules that play such roles in autophagy, and identifying them will be key for our understanding of autophagy in non-starved cells.

### Key Autophagy Regulators

Autophagy activation in response to the primordial stimuli of nutrient deprivation and/or low cellular energy levels is mediated by signaling pathways that converge on ULK1/2 (mammalian homologs of the C. elegans uncoordinated-51 kinase) (Figure 2).

Neuron
Review

![Diagram of the Mammalian Autophagy Pathway](#)

Figure 1. Overview of the Mammalian Autophagy Pathway

Nutrient depletion, growth factor deprivation, and low energy are well established autophagy inducers. These culminate in mTORC1 inhibition and AMPK activation, which, in turn, positively regulate the ULK1 complex through a series of phosphorylation events. Induction of the ULK1 complex subsequently activates the VSP34 complex, which leads to PI3P synthesis in pre-autophagosomal structures. The membranes of these structures appear to have multiple sources, such as the endoplasmic reticulum, Golgi apparatus and trans-Golgi network, plasma membrane, endosomal compartment, and mitochondria. PI3P defines the LC3 lipidation sites for autophagosome precursors by assisting in the recruitment of the ATG12-ATG5-ATG16L1 complex. This complex is essential for the conjugation of LC3-I to PE in membranes, which sustains membrane elongation and engulfment of a variety of substrates (e.g., protein aggregates, infectious agents, and damaged mitochondria) that end up being degraded in the lysosome after fusion with autophagosomes. Conditions characterized by mTORC1 inactivation are also accompanied by translocation of TFEB to the nucleus and TFEB activation, leading to the transcription of many autophagy and lysosomal genes, which ensures synthesis of key components of the pathway and efficient autophagy-dependent degradation. AMPK, AMP-dependent protein kinase; mTORC1, mammalian target of rapamycin complex 1; PE, phosphatidylethanolamine; PI3P, phosphatidylinositol 3-phosphate; TFEB, transcription factor EB; ULK, mammalian homologs of the C. elegans uncoordinated-51 kinase.

ULK1/2 forms a complex with ATG13, ATG101, and focal adhesion kinase family-interacting protein of 200 kDa (FIP200). Nutrients and growth factor availability and levels of AMP/ATP (which reflect the energetic status of the cell) are sensed by mammalian target of rapamycin complex 1 (mTORC1) and AMP-dependent protein kinase (AMPK), respectively, which, in turn, oppositely regulate the ULK1/2 complex through a series of phosphorylation events. For instance, activation of AMPK by allosteric binding of AMP and phosphorylation of Thr172 promotes autophagy by directly activating ULK1 through phosphorylation of Ser317 and Ser77 under glucose deprivation (Kim et al., 2011) or Ser555 under amino acid starvation and mitophagy (Egan et al., 2011). On the other hand, in medium replete with amino acids (sensed by the Rag-Ragulator complex) and growth factors (that signal by receptor tyrosine kinases and the phosphatidylinositol 3-kinase (PI3K)/AKT pathway), mTORC1 is activated and inhibits autophagy by binding to the ULK1 complex (via Raptor-ULK1 interaction) and by phosphorylating both ATG13 and ULK1 (at Ser 757), thereby suppressing ULK1 kinase activity and preventing the interaction between ULK1 and AMPK (Ganley et al., 2009; Hosokawa et al., 2009; Jung et al., 2009; Kim et al., 2011; Laplante and Sabatini, 2012).

Activation of the ULK complex is required for the recruitment of the class III PI3K VPS34 to the phagophore initiation sites where VPS34 generates phosphatidylinositol 3-phosphate (PI3P) while in a complex with VPS15, ATG14, and Beclin 1. The exact functions of PI3P in autophagy are still unclear. However, it appears to aid the recruitment of tryptophan-aspartic acid (WD) repeat domain phosphoinositide-interacting (WIPI) proteins to the phagophore membrane, which, in turn, controls the recruitment of crucial downstream autophagic proteins (e.g., ATG16L1 by WIPI2) that dictate where the phagophores form (Dooley et al., 2014).

The ATG12 and ATG8/LC3 ubiquitin-like conjugation systems are then required for sustaining the expansion of the phagophore. In the first system, ATG12 is conjugated to ATG5 in a reaction that involves ATG7 and ATG10 (E1-like and E2-like enzymes, respectively), and the resulting complex binds non-covalently to

1016 Neuron 93, March 8, 2017

closure of the fully formed autophagosome (Bento et al., 2016b). This late stage, where the outer and the inner membranes of the pre-autophagosomal structure become separate entities, is poorly understood, but ATG2, in combination with WIPI1, appears to regulate autophagosome closure (Velikkakath et al., 2012), a process likely involving membrane fission/scission-type events akin to the genesis of multi-vesicular bodies by endosomal sorting complexes required for transport (ESCRT)-mediated membrane budding (Knorr et al., 2015).

The source of autophagosome membranes is an area of active investigation. The endoplasmic reticulum (ER), Golgi and trans-Golgi network, mitochondria, plasma membrane, and endosomal compartments have all been suggested as possible sources of phagophore membranes (Bento et al., 2016b). The ER emerged as one of the possible sources of membranes for pre-autophagosomes, not only because isolation membranes were observed cradled within a subdomain of the ER and interconnected with it (Axe et al., 2008; Hayashi-Nishino et al., 2009) but also because ATG14- and ATG5-positive isolation membranes were found in close proximity to a subdomain of the ER in contact with mitochondria during starvation (Hamasaki et al., 2013). Post-Golgi tubulo-vesicular compartments undergoing remodelling and homotypic fusion (Guo et al., 2012; Orsi et al., 2012; Young et al., 2006) and the ER-Golgi intermediate compartment (ERGIC) have also been considered as pre-autophagosomal membrane sources (Ge et al., 2013, 2014). The ERGIC was specifically observed to bud LC3 lipidation-active vesicles that may enable autophagosome biogenesis and expansion (Ge et al., 2013, 2014).

The plasma membrane and the endocytic compartments are also suggested as membrane sources for early autophagosomal precursor structures. Clathrin-dependent endocytosis has been implicated in this process by delivering ATG16L1 and mATG9 to recycling endosomes (via different routes and involving VAMP3-dependent membrane fusion events), which leads to the formation of early autophagosomal structures and mature autophagosomes (Moreau et al., 2011; Puri et al., 2013; Ravikumar et al., 2010). Overexpression of the recycling endosome proteins TBC1D14 (a Rab11 effector) or PX-containing SNX18 were shown to induce accumulation of mATG9 and ATG16L1, respectively, along with other autophagic proteins (e.g., ULK1 and LC3) in this compartment (Knævelsrud et al., 2013; Lamb et al., 2016; Longatti et al., 2012), reinforcing the possible role of the recycling endosome as an important phagophore membrane source.

**Transcriptional Regulation of Autophagy**

In addition to phosphorylation, discussed above, autophagy is also regulated at the transcriptional level. mTORC1, apart from regulating the ULK1/2 complex, also connects the lysosome nutrient-sensing (LYNUS) machinery to the transcriptional regulation of autophagy genes via transcription factor EB (TFEB) (Settembre et al., 2012). In resting cells, phosphorylation of TFEB by active mTORC1 induces TFEB binding to 14-3-3 proteins and, therefore, TFEB retention in the cytosol (Roczniak-Ferguson et al., 2012; Settembre et al., 2011). However, under starvation and consequent mTORC1 inactivation, TFEB is no longer phosphorylated and translocates to the nucleus, where it binds to coordinated lysosomal expression and regulation (CLEAR) consensus sequences in promoters of target genes and induces

Neuron 93, March 8, 2017 1017
their transcription. Among these genes, many are directly related to the lysosome and autophagy (e.g., lysosomal hydrolases, vacuolar-type H+-ATPase (v-ATPase) subunits, and ATG proteins), and, thus, TFEB appears to co-ordinately regulate the expression of many of the key genes required for autophagy/lysosome function ([Sardiello et al., 2009](https://doi.org/10.1038/nature07952); [Settembre et al., 2011](https://doi.org/10.1038/nsmb.2064)). Zinc-finger protein with KRAB and SCAN domains 3 (ZKSCAN3) is a transcriptional repressor of autophagy that appears to oppose TFEB. ZKSCAN3 silencing enhances autophagosome and lysosomal biogenesis, and mTORC1 inhibition induces its accumulation in the cytosol ([Chauhan et al., 2013](https://doi.org/10.1038/cdd.2013.12)). More than 20 other transcription factors have now been linked to transcriptional regulation of autophagy following a wide range of stimuli ([Füllgrabe et al., 2014](https://doi.org/10.1016/j.cell.2014.01.038)). For instance, microphthalmia-associated transcription factor (MITF) (which belongs to the same family of proteins as TFEB) ([Martina et al., 2014](https://doi.org/10.1016/j.cub.2014.01.050)), p53 ([Kenzelmann Broz and Attardi, 2013](https://doi.org/10.1016/j.molcel.2013.02.018)) and forkhead box O3 (FOXO3) ([Mammucari et al., 2007](https://doi.org/10.1038/onc.2007.39)) have all been shown to trans-activate ATG genes.

### Autophagy Physiology and the Nervous System

The mammalian nervous system requires autophagy to maintain its normal functions and homeostasis. Because ubiquitous deletion of core autophagy genes results in neonatal and embryonic lethality, multiple nervous system-specific knockout mouse models have been generated to allow analyses of the roles of autophagy in neuronal function. A Nestin-Cre promoter that switches on embryonically in neuronal precursor cells has been used to excise floxed alleles of Atg5 and Atg7. This results in autophagy deficiency in neuronal cells and glia, accompanied by the accumulation of intra-neuronal aggregates ([Hara et al., 2006](https://doi.org/10.1016/j.neuron.2006.04.017); [Komatsu et al., 2006, 2007a](https://doi.org/10.1016/j.cell.2006.02.029)). The accumulation of these aggregates in otherwise normal mice suggests that autophagy plays a key role in removing aggregate-prone proteins, supporting earlier studies in cell culture models of Huntington’s disease ([Ravikumar et al., 2002](https://doi.org/10.1038/ng934)). Subsequent Nestin-Cre models have been generated targeting FIP200 and Wipi4 as well as a double knockout for Ulk1/2 ([Joo et al., 2016](https://doi.org/10.1016/j.cell.2016.01.038); [Liang et al., 2010](https://doi.org/10.1016/j.cell.2010.06.016); [Orosco et al., 2014](https://doi.org/10.1016/j.cell.2014.03.027)). The result in all models is reduced survival and early-onset, progressive neurodegeneration across broad areas of the brain. However, the nature of pathology varies according to the gene targeted; e.g., progressive spongiosis is seen in FIP200 Nestin-Cre mice but not the Atg5 and Atg7 Nestin-Cre models. The variability observed between models will likely be accounted for by the stages at which autophagy is disrupted as well as non-autophagy functions for each target.

To further delineate the function of autophagy in different neuronal types, targeted knockouts of Atg5 and Atg7 in Purkinje neurons of the cerebellum ([Komatsu et al., 2006](https://doi.org/10.1016/j.cell.2006.02.029); [Nishiyama et al., 2007](https://doi.org/10.1016/j.neuron.2007.05.020)), Atg7 in agouti-related peptide (AgRP) neurons of the hypothalamus ([Kaushik et al., 2011](https://doi.org/10.1016/j.cell.2011.03.023)), as well as Atg5 and Atg7 in rhodopsin neurons of the retina ([Chen et al., 2013](https://doi.org/10.1016/j.cell.2013.03.020); [Zhou et al., 2015](https://doi.org/10.1016/j.cell.2015.02.035)) have been developed. Deletion of Atg5 and Atg7 results in cell-autonomous Purkinje neuron degeneration. The earliest signs of homeostatic disruption in these mice are Purkinje axonal swellings, which are observed prior to progressive dystrophy and degeneration of the axon ([Komatsu et al., 2007b](https://doi.org/10.1016/j.cell.2007.05.020); [Nishiyama et al., 2007](https://doi.org/10.1016/j.neuron.2007.05.020)). These findings support a key role of autophagy in axon homeostasis and highlight an early sign of neuronal dysfunction. Studies from cell culture systems have led to the proposal that different neuronal types may have varying capacities to degrade autophagy substrates, such as mutant polyglutamine proteins, and that this correlates with the sensitivity of that cell type to the toxicity of aggregate-prone proteins ([Tsvetkov et al., 2013](https://doi.org/10.1016/j.cell.2013.03.020)).

In the peripheral nervous system (PNS), a specific role for autophagy in Schwann cell (SC) function has been identified during myelination and re-myelination. In the early postnatal period, when myelination is occurring, autophagy is involved in the maturation of SCs, particularly in the reduction of abaxonal cytoplasm volume. In conditional knockouts where ATG7 is absent in SCs, the abaxonal cytoplasm remains abundant, and, consequently, small fiber hypermyelination is observed ([Jang et al., 2015](https://doi.org/10.1016/j.cell.2015.01.048)). Conversely, autophagy upregulation via rapamycin treatment during the early post-natal period leads to a greater reduction in abaxonal cytoplasm within the SCs. After nerve injury, myelin debris is cleared by SCs through a newly described form of selective autophagy named myelinophagy ([Gomez-Sanchez et al., 2015](https://doi.org/10.1016/j.cell.2015.01.048)), and this debris clearance is delayed in Atg7-SC conditional knockouts ([Jang et al., 2016](https://doi.org/10.1016/j.cell.2016.01.038)).

In addition to protecting against neurodegeneration, autophagy also regulates neurogenesis. An important role for autophagy in sustaining the post-natal pool of neural stem cells (NSCs) has recently emerged. In the adult brain, NSCs within the sub-ventricular zone (SVZ) of the lateral ventricle wall and subgranular zone (SGZ) of the dentate gyrus can produce new functional neurons in response to physiological and pathological stimuli ([Gage, 2000](https://doi.org/10.1038/35017500)). Ablation of FIP200 results in a reduced number of NSCs in the SVZ and the dentate gyrus, affecting the self-renewal capacity of NSCs ([Wang et al., 2013a](https://doi.org/10.1016/j.cell.2013.03.020)). Key autophagy proteins, including AMBRA1 and Beclin 1, are highly expressed in the SVZ of the adult mouse brain, and heterozygosity for Becn1 results in decreased cell proliferation in vitro ([Yazdankhah et al., 2014](https://doi.org/10.1016/j.cell.2014.03.027)). Depletion of ATG5 in adult NSCs in the dentate gyrus also leads to reduced survival and neuronal maturation ([Xi et al., 2016](https://doi.org/10.1016/j.cell.2016.01.038)). In all of these studies, the depletion of the NSC pool could be attributed to apoptosis ([Wang et al., 2013a; Xi et al., 2016; Yazdankhah et al., 2014](https://doi.org/10.1016/j.cell.2013.03.020)). However, although apoptosis is likely responsible for reduced survival of autophagy-defective NSCs, it is unclear whether NSC differentiation and neuronal maturation are apoptosis-dependent ([Xi et al., 2016; Yazdankhah et al., 2014](https://doi.org/10.1016/j.cell.2016.01.038)). It will be interesting to examine whether potential beneficial effects of increasing basal autophagy act via changes in NSCs, and this is likely to be an exciting topic for future research. In addition to effects on the stem cell pool, autophagy is required for normal neural progenitor cell proliferation and differentiation, as revealed by studies in ubiquitous ATG16L1 hypomorph mice and knockdown of ATG5 in vivo in embryonic brains ([Lv et al., 2014; Wu et al., 2016](https://doi.org/10.1016/j.cell.2014.03.027)).

Although impairment of autophagy is deleterious, autophagy upregulation appears to be protective in many normal contexts. Studies in *C. elegans* have shown a clear association between increased longevity and constitutive autophagy, which may be mediated through multiple different but overlapping mechanisms, including nutrient restriction ([Hansen et al., 2008](https://doi.org/10.1016/j.cell.2008.02.038)), altered mitosis ([Ghavidel et al., 2015](https://doi.org/10.1016/j.cell.2015.01.048)), and mitochondrial turnover ([Palikaras et al., 2015](https://doi.org/10.1016/j.cell.2015.01.048)). Similarly, pan-neuronal overexpression of

1018 Neuron 93, March 8, 2017
Atg8a (Simonsen et al., 2008) or AMPK (Ulgherait et al., 2014) in Drosophila resulted in an extended lifespan. Overexpression of ATG5 in mice induces autophagy and extends the lifespan and is associated with anti-aging phenotypes including leanness, increased insulin sensitivity, and improved motor function (Pyo et al., 2013). In Huntington’s disease (HD), expansion of the polyglutamine tract in the huntingtin (HTT) protein causes disease with a polyglutamine length-dependent severity. In mice where the polyglutamine expansion was removed from the endogenous Htt-coding gene (delta-Q mice), an increase in autophagosome biosynthesis was observed, and this was also associated with a significantly longer lifespan. (Zheng et al., 2010). Together, these studies indicate a beneficial effect of enhancing basal autophagy during aging.

### Neurodegenerative Diseases in which Autophagy Is Impaired

As outlined above, there is increasing evidence for the physiological importance of autophagy in neuronal health, raising the possibility that autophagy dysfunction may play a role in neurodegenerative diseases. Circumstantially, this is further supported by the major pathological phenotype of most late-onset neurodegenerative diseases, the presence of intraneuronal aggregates of misfolded proteins, which are substrates for autophagic degradation (Menzies et al., 2015a; Ravikumar et al., 2002). In some forms of disease, these aggregate-prone proteins are the result of specific mutations (such as in Huntington’s disease). However, in the vast majority of diseases, the underlying reason for the presence of aggregates is unknown.

For the most part, neurodegenerative diseases do not follow simple, monogenic inheritance patterns. However, in all major neurodegenerative diseases, a subset of cases are associated with inherited genetic mutations. These familial forms of disease allow an insight into the potential mechanisms of pathogenesis. Identification of disease-associated genes and investigations into their functions reveal that many affect autophagy. The absolute contribution of autophagy dysfunction to disease progression has yet to be established. Given that many neurodegenerative diseases are late-onset, it is possible that small alterations in the turnover of proteins will have cumulative effects that manifest later in life.

The autophagy pathway is complex, with multiple steps and modes of regulation. This makes identifying potentially minor perturbations in the pathway difficult. Therefore, in some cases, the evidence for autophagy involvement in neurodegenerative disease pathogenesis appears controversial. This evidence is discussed in a disease-specific manner below (Figure 3).

#### Alzheimer’s Disease

The pathological hallmarks of Alzheimer’s disease (AD) are intracellular tau tangles and extracellular amyloid β (Aβ) plaques. Aβ is formed from amyloid precursor protein (APP) by two cleavage events. There is a complex interplay between Aβ and autophagy. Aβ may be degraded by autophagy, and upregulation of autophagy has been shown to reduce Aβ levels in a number of systems (Boland et al., 2008; Spilman et al., 2010; Tian et al., 2011; Vingtdeux et al., 2011). However, Aβ may also be generated in autophagosomes, which appear to contain both APP and Presenilin-1 (PS-1), an enzyme involved in the cleavage of

APP to Aβ (Boland et al., 2008; Yu et al., 2005). Furthermore, autophagy may play a role in the secretion of Aβ into the extracellular space, where it forms plaques, because deletion of ATG7 in APP transgenic mice results in less Aβ extracellular secretion and plaque formation, contrary to what one may expect if autophagy simply degraded Aβ (Nilsson et al., 2013).

Genetic studies have implicated phosphatidylinositol-binding clathrin assembly protein (PICALM) in Alzheimer’s disease (Harold et al., 2009; Jun et al., 2010), and changes in the level of this protein have been reported in the brains of patients with Alzheimer’s (Ando et al., 2013, 2016). PICALM is a clathrin adaptor protein required for the endocytosis of soluble NSF attachment protein receptors (SNAREs), and loss of this function has been demonstrated to inhibit autophagy at multiple steps, including early autophagosome formation and maturation of autophagosomes (Moreau et al., 2014). Of course, altered trafficking of these SNAREs is also likely to affect other vesicle trafficking pathways, and, therefore, as with all of the disease mutations discussed in this review, it is yet to be established whether or how much PICALM-mediated autophagy perturbation contributes to disease. However, a second mechanism through which PICALM function may affect autophagy has also been proposed. Through its action in a complex with assembly polypeptide 2 (AP2), PICALM may act as an autophagy receptor that is able to interact with LC3 and target APP into autophagosomes (Tian et al., 2013).

Mutations in PS-1 cause familial AD, and there is a great deal of evidence demonstrating that these mutations change the way the APP protein is processed (for example, Citron et al., 1997). However, other functions of PS-1 may also contribute to disease pathogenesis. PS-1 has been shown to function as an ER chaperone for the V₀a₁ subunit of the lysosomal v-ATPase. Familial AD-associated mutations in PS-1 result in decreased maturation of the lysosomal v-ATPase and, thus, increased lysosomal pH (Coffey et al., 2014; Lee et al., 2010; Wolfe et al., 2013), which would be predicted to reduce autophagosome clearance.

Autophagy may also be affected by reduced levels of Beclin 1 mRNA and protein, which have been reported in AD brains (Pickford et al., 2008; Small et al., 2005). Similarly, Beclin 1 protein levels are thought to be decreased because of caspase 3 cleavage, which is activated in the brains of AD patients (Rohn et al., 2011).

#### Tauopathies

Tau accumulation into intracellular tangles is one of the hallmark pathologies of AD and is also seen in a group of neuronal disorders termed tauopathies, which include progressive supranuclear palsy (PSP), corticobasal degeneration (CBD) and frontotemporal dementias (FTDs) (Lee et al., 2001). Hyperphosphorylated tau co-localizes with LC3-positive vesicles and the autophagy cargo-receptor SQSTM1/p62 in CBD and PSP patients (Piras et al., 2016). Moreover, aberrant tau appears to disrupt axonal vesicle transport by impairing the dynein-dynactin complex, increasing the number of autophagosomes and contributing to tau-induced toxicity in FTDs and AD (Butzlaff et al., 2015; Majid et al., 2014). Tau has also been shown to bind lysosomal membranes and perturb lysosomal permeability in vitro and in a mouse model of AD (Collin et al., 2014; Wang et al., 2009). Defective

Impaired autophagosome formation
PICALM (Alzheimer's)
α-syn, VPS35 (Parkinson's)
Htt (Huntington's)
C9ORF72 (ALS)
SPG49 (HSP)
Malin, Laforin (Lafora disease)
GJB1 (CMT1)
ATG5 (childhood ataxia)
WDR45 (SENDA/BPAN)

Disrupted lysosomal function
PS-1 (Alzheimer's)
ATP13A2, SYT11, GBA, α-syn, VPS35 (Parkinson's)
SPG11, SPG15 (HSP)
dynamin 2, Rab7, VCP (CMT2)
NPC1 (Niemann-Pick Type C disease)
SNX14 (hereditary cerebellar ataxia)
lysosomal storage disorders

Atg5.Atg12/Atg16L1 complex

Secretion
Aβ (Alzheimer's)

lysosome

autolysosome

phagophore
LC3-II
autophagosome

Disruption of cargo recognition
PICALM (Alzheimer's)
Htt (Huntington's)
SQSTM1/p62, OPTN, ubiquitin-2,
TBK1 (ALS)

Autophagic cargo
Aβ (Alzheimer's)
Tau (Alzheimer's and tauopathies)

Inhibited autolysosome formation
PICALM (Alzheimer's)
Tau (Alzheimer's and tauopathies)
CHMP2B (ALS)
DYNC1H1 (CMT2)
VPS11 (leukencephaly)
lysosomal storage disorders

Figure 3. Intersections between Autophagy and Disease-Associated Genes
An increasing number of genes associated with neurodegenerative diseases have now been implicated in autophagy function. These genes act at a number of different steps throughout the autophagic process, from early steps of autophagosome formation through autolysosome formation. Their proposed sites of action are indicated, along with the neurodegenerative disease with which they are associated.

lyosomal membrane integrity was also found in AD patients (Perez et al., 2015), and, more recently, increased levels of the lyosomal components lyosomal-associated membrane protein 1 (LAMP1) and cathepsin D were reported in CBD and PSP patients (Piras et al., 2016).

**Parkinson’s Disease**
Compelling evidence supporting a role for dysfunctional autophagy as a causative factor in neurodegenerative disease comes from the studies of mitophagy in Parkinson’s disease (PD). Autosomal recessive forms of early-onset PD are associated with mutations in genes encoding phosphatase and tensin homolog-induced putative kinase 1 (PINK1) (Valente et al., 2004) and the E3 ubiquitin ligase Parkin (Kitada et al., 1998). Elegant studies have demonstrated that PINK1 and Parkin act in the same pathway to promote mitophagy. PINK1 is stabilized on the outer membrane of damaged mitochondria, leading to the recruitment and activation of the E3 ubiquitin ligase Parkin on these mitochondria and, ultimately, their sequestration into autophagosomes (Matsuda et al., 2010; Narendra et al., 2008). This disease-associated mitophagy signaling pathway has been the subject of recent comprehensive reviews (Pickrell and Youle, 2015; Nguyen et al., 2016).

Several models have been generated to elucidate the function of these mitophagy effectors in the nervous system. Although no gross behavioral changes were found in mice where Parkin was deleted (Goldberg et al., 2003; Itier et al., 2003; Perez and Palmiter, 2005; Von Coelln et al., 2004), the activity of striatal neuron mitochondria was impaired (Palacino et al., 2004). A deficit in evoked dopamine release response and striatal synaptic plasticity in the striatum was also observed (Kitada et al., 2009). Similarly, deletion of Pink1 resulted in impaired respiration in striatal mitochondria and increased sensitivity to oxidative stress (Gauthier et al., 2008). Furthermore, dopaminergic neurons derived from Pink1 or DJ-1 knockout mice also showed defective morphology and reduced activity (Shim et al., 2011). Indeed, loss of endogenous Parkin synergizes with a loss of DNA polymerase γ that results in increased mitochondrial mutations and, in turn, in dopaminergic neuron damage (Pickrell et al., 2015). The use of a recently generated reporter mouse to assess in vivo mitophagy will shed light on how mitophagy flux is

1020 Neuron 93, March 8, 2017
modulated in response to different genetic and pharmacological perturbations (Sun et al., 2015).

PD is characterized by loss of dopaminergic neurons in the substantia nigra and, typically, by the presence of $\alpha$-synuclein ($\alpha$-syn) inclusions. These inclusions have been shown to affect autophagy function (Tanik et al., 2013). In cells with $\alpha$-syn inclusions, although lysosomal function appears normal, autophagosome maturation and fusion with the lysosomes are decreased, resulting in a decrease in protein degradation. It is interesting to note that this alteration in vesicle trafficking does not seem to result from a non-specific physical blockade of the axons by the inclusions but, rather, a specific inhibition of endocytic and autophagic vesicles (Volpicelli-Daley et al., 2014). Independent of the formation of inclusions, increased levels of $\alpha$-syn (as found in disease) has been shown to impair autophagy because $\alpha$-syn overexpression in cell and mouse models led to mislocalization of mATG9 (Winslow et al., 2010). Similarly, abnormal trafficking of mATG9 has been shown to be caused by the vacuolar protein sorting-associated protein 35 (VPS35) D620N mutation, which causes an autosomal dominant form of PD. VPS35 is a component of the retromer complex, which recruits the actin nucleation-promoting WASP and Scar homolog (WASH) complex to endosomes. D620N VPS35 prevents this recruitment and, thus, causes mATG9 mislocalization and autophagy impairment (Zavodszky et al., 2014).

The most common genetic risk factor for PD is heterozygosity for mutations in the lysosomal enzyme glucocerebrosidase (GBA). These mutations have been reported in up to 31.3% of Ashkenazi Jewish patients with Parkinson’s disease and up to 9.4% in patients of other ethnic origins (Sidransky and Lopez, 2012). Homozygous GBA mutations cause the lysosomal storage disorder Gaucher disease (reviewed in Michel et al., 2016), where loss of GBA leads to the accumulation of its substrate glucosylceramide within lysosomes and, thus, to autophagy impairment because of lysosomal dysfunction. In PD patients without GBA mutations, the levels and activity of the enzyme are decreased in brain areas with increased $\alpha$-syn levels in early disease stages (Mazzulli et al., 2011; Murphy et al., 2014).

Loss-of-function mutations in the P-type ATPase ATP13A2 (also known as PARK9) cause familial Kufor-Rakeb syndrome, characterized by early-onset Parkinsonism (Ramirez et al., 2006). Lysosomal ATPases are required for the maintenance of lysosomal pH and, therefore, the activity of lysosomal proteases. Mutations in ATP13A2 impair these processes and lead to an increase in the number of autophagic vesicles, which are unable to fuse with lysosomes (Dehay et al., 2012). Impaired lysosomal degradative capacity in cells with ATP13A2 mutations leads to an accumulation of $\alpha$-synuclein, which may contribute to the toxicity of ATP13A2 mutations (Usenovic et al., 2012). Recent studies show that ATP13A2 depletion leads to a decrease in the levels of another PD-associated gene, synaptotagmin 11 (SYT11), and that it is this SYT11 decrease that causes lysosomal dysfunction and impaired autophagosome degradation resulting from loss of ATP13A2 activity. Overexpression of SYT11 in ATP13A2 knockdown cells is able to rescue the autophagy defects seen in these cells, demonstrating that they act in the same pathway (Bento et al., 2016a).

### Polyglutamine Disorders

Nine neurological disorders are caused by mutant proteins with expanded glutamine repeats (polyQ), including HD, various spinocerebellar ataxias (SCAs), and spinal and bulbar muscular atrophy (SBMA). Autophagy perturbation is observed in several of these diseases. At the transcriptional level, the mutant polyQ androgen receptor (which causes SBMA) interacts with TFEB, a potent autophagy gene inducer, and inhibits TFEB transactivation (Cortes et al., 2014). A transgenic mouse model of SCA3 showed low levels of sirtuin 1 (Cunha-Santos et al., 2016), an enzyme that normally deacetylates several autophagy proteins to promote autophagy (Huang et al., 2015; Lee et al., 2008). Additionally, brains from mutant SCA3 transgenic mice have low levels of Parkin (Durcan et al., 2011) and Beclin 1 (Nascimento-Ferreira et al., 2011). Beclin 1 appears to be sequestered into aggregates in the brains of mouse models of HD and SCA7 (Alves et al., 2014; Shibata et al., 2006).

The most investigated polyQ disease protein is huntingtin, mutated in HD. Mutant huntingtin impairs efficient cargo recognition by autophagosomes (Martinez-Vicente et al., 2010), whereas wild-type huntingtin serves as a scaffold for the recruitment of several autophagy proteins during selective autophagy of targets, such as mitochondria and protein aggregates (Ochaba et al., 2014; Rui et al., 2015). Furthermore, a role for huntingtin in axonal transport of autophagosomes has been suggested by live-cell imaging studies in striatal neurons. Following either loss of huntingtin or expression of the mutant protein, a decrease autophagosome transport and subsequent inhibition of substrate degradation were observed (Wong and Holzbaur, 2014a). Mutant huntingtin also interacts with and inactivates Rhes, a protein selectively expressed in the striatum, the brain region selectively affected in HD. Rhes is required for autophagy because it interacts with Beclin 1 and reduces the inhibitory interaction of Bcl-2 with Beclin 1 (Mealer et al., 2014). Another important autophagy negative regulator, mTOR, is sequestered to aggregates in HD and SCA7 brains (Alves et al., 2014; Ravikumar et al., 2004).

In addition to potential mechanistic roles in autophagy, it has also been proposed that mutant huntingtin may accumulate because of the loss of an acetylation signal that would normally target it for degradation by this pathway (Jeong et al., 2009). Overall, mutant huntingtin is involved in different pathways that modulate autophagy—some toxic and others that may be protective. The overall outcome in terms of autophagy activity may depend on the ratio of soluble to aggregated mutant protein.

### Amyotrophic Lateral Sclerosis

Amyotrophic lateral sclerosis (ALS) is associated with the accumulation of misfolded proteins such as superoxide dismutase 1 (SOD1), TAR DNA-binding protein 43 (TDP-43), fused in sarcoma/translocated in sarcoma (FUS), and/or C9ORF72 (reviewed in Blokhuis et al., 2013). It is most commonly a sporadic disease, but approximately 5%–10% of cases are familial. A range of genes have been associated with the disease, but recently it has become apparent that a number of these genes encode for proteins that act as autophagy receptors, like SQSTM1/p62 (Fecto et al., 2011), OPTN (Maruyama et al., 2010), and Ubiquilin 2 (Williams et al., 2012), which enhancethe incorporation of autophagy substrates into autophagosomes via interactions with LC3, as discussed above ([Khaminets et al., 2016](https://doi.org/10.1016/j.cell.2015.12.048)). Mutations in SQSTM1/p62 have been associated with disrupted autophagic degradation of mutant SOD1 and TDP-43 ([Gal et al., 2009](https://doi.org/10.1074/jbc.M808868200); [Teyssou et al., 2013](https://doi.org/10.1016/j.jmb.2013.01.013)). ALS-causing mutations in the LC3 binding region of SQSTM1/p62 have been shown to impair its recruitment into autophagosomes ([Goode et al., 2016](https://doi.org/10.1016/j.neuron.2016.01.028)). However, mutations have been identified throughout the length of the protein, and it is not clear how these mutations affect SQSTM1/p62 function. Similarly, disease-causing mutations in OPTN decrease autophagy and protein clearance ([Shen et al., 2015](https://doi.org/10.1016/j.celrep.2015.03.025)) and also inhibit the ability of OPTN to recruit LC3 to damaged mitochondria and induce mitophagy ([Wong and Holzbaur, 2014b](https://doi.org/10.1016/j.tcb.2014.02.004)). OPTN also functions to bind LC3 and Myosin VI, which is required for the trafficking of autophagosomes ([Tumbarello et al., 2012](https://doi.org/10.1016/j.molcel.2012.03.018)); the majority of ALS-associated mutations in OPTN are located in the myosin VI-binding domain and show loss of this binding ([Shen et al., 2015](https://doi.org/10.1016/j.celrep.2015.03.025); [Sundaramoorthy et al., 2015](https://doi.org/10.1016/j.neuron.2015.03.025)). Recently, mutations in TANK-binding kinase 1 (TBK1) have been associated with ALS ([Cirulli et al., 2015](https://doi.org/10.1016/j.cell.2015.05.029); [Freischmidt et al., 2015](https://doi.org/10.1016/j.cell.2015.05.028)); TBK1 has been shown to phosphorylate OPTN to promote efficient mitophagy and SQSTM1/p62 to allow autophagosome maturation ([Moore and Holzbaur, 2016](https://doi.org/10.1016/j.tcb.2016.01.001); [Pilli et al., 2012](https://doi.org/10.1016/j.cell.2012.01.039)). Disease-associated mutations in this kinase have been shown to reduce binding of TBK1 to OPTN and, thus, decrease the clearance of dysfunctional mitochondria ([Richter et al., 2016](https://doi.org/10.1016/j.cell.2016.01.039)).

Alongside autophagy receptor proteins, other ALS-associated genes have been identified as having a role in autophagy. Disruption of the ESCRT machinery through mutations in charged multivesicular body protein 2B (CHMP2B) have been associated with both ALS and frontotemporal dementia, a neurodegenerative disease thought to be on a spectrum with ALS. Disease-associated mutations in CHMP2B impair autolysosome formation and lead to the accumulation of protein aggregates ([Filimonenko et al., 2010](https://doi.org/10.1016/j.cell.2010.03.023); [Lee et al., 2007](https://doi.org/10.1016/j.cell.2007.03.045)).

The most common cause of ALS is a hexanucleotide repeat expansion in the C9ORF72 gene ([DeJesus-Hernandez et al., 2011](https://doi.org/10.1016/j.cell.2011.09.013); [Renton et al., 2011](https://doi.org/10.1016/j.cell.2011.09.014)). These mutations may cause disease through a number of different mechanisms, including toxic gain of function from ATG-independent translation of the RNA repeats and loss of normal protein function ([Todd and Petrucelli, 2016](https://doi.org/10.1016/j.neuron.2016.01.027)). A role has been suggested for C9ORF72 in endocytic vesicle trafficking and autophagy ([Farg et al., 2014](https://doi.org/10.1016/j.cell.2014.03.038)). Recently, C9ORF72 was found to form a complex with Smith-Magenis syndrome chromosome region, candidate 8 (SMCR8) and WD repeat domain 41 (WDR41) ([Sellier et al., 2016](https://doi.org/10.1016/j.cell.2016.01.028); [Sullivan et al., 2016](https://doi.org/10.1016/j.cell.2016.01.029)). This complex acts as a guanosine diphosphate (GDP)/guanosine triphosphate (GTP) exchange factor (GEF) for the activation of Rab8 and Rab39, which are involved in the formation or maturation of autophagosomes ([Pilli et al., 2012](https://doi.org/10.1016/j.cell.2012.01.039); [Seto et al., 2013](https://doi.org/10.1016/j.cell.2013.01.039)), and interacts with the autophagy receptors SQSTM1/p62 and OPTN ([Sellier et al., 2016](https://doi.org/10.1016/j.cell.2016.01.028)). Furthermore, C9ORF72 can interact with ULK1, a key kinase in the control of autophagosome formation, and this interaction mediates the translocation of the ULK1 autophagy initiation complex to the phagophore via RAB1a ([Webster et al., 2016](https://doi.org/10.1016/j.cell.2016.01.030)). Understanding the role of these genes in autophagy may be beginning to provide a link in

the pathogenic mechanism between ALS genes; for example, Rab8 has been identified as a modifier of toxicity in a *Drosophila* mutant CHMP2B model ([West et al., 2015](https://doi.org/10.1016/j.cell.2015.01.039)). Finally, another Rab GEF, Alsín (ALS2), has been associated with ALS ([Yang et al., 2001](https://doi.org/10.1016/S0896-6273(01)00268-8)). Alsín is an activator of Rab5 ([Topp et al., 2004](https://doi.org/10.1016/j.cell.2004.03.028)), which has been shown previously to regulate autophagy ([Otomo et al., 2011](https://doi.org/10.1016/j.cell.2011.03.028); [Ravikumar et al., 2008](https://doi.org/10.1016/j.cell.2008.03.028)).

### Hereditary Spastic Paraplegia

Hereditary spastic paraplegias (HSPs) are a heterogeneous group of neurodegenerative diseases characterized by length-dependent degeneration of upper motor neuron axons resulting in progressive lower limb spasticity. To date, more than 70 distinct loci (SPG1–72) and 50 spastic paraplegia genes have been identified; the type of HSP is designated by the locus with which it is associated (i.e., SPG1–56) ([Fink, 2013](https://doi.org/10.1016/j.neurobiolaging.2012.09.001)). SPG11, encoding spatacsin, is the most prevalent gene mutated in autosomal recessive HSP. Spatacsin knockout mice have a reduced number of lysosomes in Purkinje cells and impaired autophagic lysosome reformation in neurons ([Varga et al., 2015](https://doi.org/10.1016/j.cell.2015.01.039)). SPG11 and SPG15 (encoding spastizin) are both similar in their symptoms and in their cellular mechanisms; loss or mutation of either gene disrupts autophagosome maturation and lysosomal biogenesis ([Chang et al., 2014](https://doi.org/10.1016/j.cell.2014.03.038); [Renvoisé et al., 2014](https://doi.org/10.1016/j.cell.2014.03.039)). Spastizin also interacts with the Beclin 1-UVRAG (UV radiation resistance-associated gene)-Rubicon complex required for autophagosome maturation ([Vantaggiato et al., 2013](https://doi.org/10.1016/j.cell.2013.01.039)).

A rare form of HSP is caused by mutations in tectonin β-propeller-containing protein 2 (TECPR2), encoded by SPG49. The loss of TECPR2 results in a reduction in the levels of LC3 along with a reduction in the lipidation of LC3 ([Oz-Levi et al., 2012](https://doi.org/10.1016/j.cell.2012.01.039)). TECPR2 contains a LIR, and its interaction with LC3 is required for maintenance of functional ER exit sites ([Stadel et al., 2015](https://doi.org/10.1016/j.cell.2015.01.039)). In patient fibroblasts with TECPR2 mutations, ER exit is inhibited, and this loss is associated with decreased autophagosome formation.

### Lafora Disease

Lafora disease is a progressive neurodegenerative disease that manifests with myoclonus epilepsy caused by loss-of-function mutations in either malin or laforin, which form a complex. A characteristic feature of the disease is the formation of inclusions of insoluble forms of glycogen called polyglucosan bodies (or lafora bodies). Because malin and laforin have been implicated in glycogen metabolism, it was originally assumed that perturbations in this function cause Lafora disease, but there is also clear evidence to support a key role for dysfunctional autophagy in this disease. Knockout mice for either gene show decreased LC3-II levels, and this is thought to result from increased mTOR activity in the case of mutations in laforin ([Aguado et al., 2010](https://doi.org/10.1016/j.cell.2010.01.039); [Garyali et al., 2014](https://doi.org/10.1016/j.cell.2014.01.039)), although the mTOR involvement is more controversial in the case of malin mutations ([Criado et al., 2012](https://doi.org/10.1016/j.cell.2012.01.039); [Garyali et al., 2014](https://doi.org/10.1016/j.cell.2014.01.039)).

### Charcot-Marie-Tooth Disease

Charcot-Marie-Tooth (CMT) diseases are hereditary motor and sensory peripheral neuropathies caused by inherited mutations in genes encoding either myelin-related (CMT type 1, CMT1) or axonal (CMT type 2, CMT2) proteins ([Bird, 1993a](https://doi.org/10.1016/j.cell.1993.01.039)). CMT type 1 X-linked disease, one of the most common neurological disorders, is caused by loss-of-function mutations in the gene GJB1encoding the gap junction protein connexin 32 ([Bird, 1993b; Xie et al., 2016]). Recent studies reported that connexin 32 depletion reduces autophagosome biogenesis not via its gap junction function but, rather, through directly interacting with autophagy core proteins (VPS34, BECN1, and ATG16L1) to sequester them at the plasma membrane ([Bejarano et al., 2014]). Upon starvation-induced autophagy, ATG14 localizes with a complex containing connexins and several autophagy proteins, facilitating the recruitment of ATG9 and formation of autophagosomes required for the degradation of connexins ([Bejarano et al., 2014]). However, these data have not yet been validated in neurons or in the context of CMT1.

The genes associated with the axonal CMT type 2 disease include those encoding the dynein-dynactin motor protein complex, also known to cause distal spinal and bulbar muscular atrophy ([Puls et al., 2005; Punetha et al., 2015]). Many of these mutations affect trafficking pathways relevant to autophagy. DYNC1H1 (dynein heavy chain 1) mutations, found in some familial cases of CMT2, directly affect neuronal retrograde axonal transport and phenocopy disease pathogenesis in rodent models ([Weedon et al., 2011]). Additionally, dynein mutations impair autophagosome-lysosome fusion by reducing the retrograde trafficking of autophagosomes to the microtubule-organizing center where lysosomes are clustered. This causes reduced autophagic clearance and accumulation of the aggregate-prone proteins ([Jahreiss et al., 2008; Ravikumar et al., 2005]). Other supportive evidence shows that mutations in the p150 subunit of dynactin cause motor neuron degeneration and result in protein aggregation ([Levy et al., 2006]). Additional genes affected in some cases of CMT2 encode endocytic-regulatory proteins such as dynamin. Both knockin mice bearing the dynamin 2 mutations ([Durieux et al., 2012]) and *Drosophila* models (*Shi* ortholog loss of function) ([Fang et al., 2016]) showed accumulation of immature autophagosome structures because of defects in autolysosome acidification rather than affecting the fusion step per se. Mutations in genes encoding proteins involved in endosomal maturation, such as RAB7, lead to impaired autophagic flux ([Ganley et al., 2011; Spinosa et al., 2008]). A recently characterized CMT2 mutation in valosin-containing protein (VCP) leads to a similar phenotype: accumulation of immature autophagosome and impaired autophagic flux ([Gonzalez et al., 2014]).

**Lysosomal Diseases**

Lysosomal diseases are rare inherited disorders with variable phenotypes. They represent the most common cause of neurodegeneration in childhood but can also result in neurological impairment in adults ([Poupetová et al., 2010; Wraith, 2002]). Most lysosomal storage disorders are caused by loss of function of specific lysosomal hydrolases, leading to the accumulation of the substrates of these enzymes and accumulation of general autophagic substrates because of impaired autophagosome-lysosome fusion ([Ballabio and Gieselmann, 2009; Platt et al., 2012; Settembre et al., 2008]).

There is increasing evidence that changes in membrane lipid composition as a result of lysosomal dysfunction contribute to lysosome fusion defects. In mouse models of mucopolysaccharidosis type III (Sanfilippo syndrome), defects in the breakdown of heparin sulfate cause an altered membrane lipid composition, with SNARE protein redistribution resulting in impaired autophagosome-lysosome fusion and a block in autophagy ([Fraldi et al., 2010; Settembre et al., 2008]). The accumulation of the glycosphingolipid psychosine in Krabbe disease, caused by a defect of β-galactocerebrosidase, alters membrane lipid composition ([Hawkins-Salsbury et al., 2013]). In Niemann-Pick type A disease, where mutations in the gene encoding acid sphingomyelinase cause accumulation of sphingomyelin, defects of mATG9 trafficking and autophagosome closure have also been observed ([Corcelle-Termeau et al., 2016]). Sphingomyelin storage also leads to lysosomal membrane permeabilization, thereby liberating cathepsins into the cytosol ([Serrano-Puebla and Boya, 2016]).

Apart from mutations in genes encoding lysosomal hydrolases, defects in posttranslational modifications, impaired trafficking of lysosomal enzymes, or defective acidification also result in lysosomal dysfunction ([Colacurcio and Nixon, 2016; Hirst et al., 2015; Kyttälä et al., 2005; Morimoto et al., 1989; Tiede et al., 2005]). One example is multiple sulfatase deficiency, in which a failed posttranslational modification of sulfatases by an ER-resident enzyme abrogates the function of a whole group of lysosomal hydrolases ([Dierks et al., 2009]).

Beside this, impaired lysosomal structure, regeneration, fusion, and signaling also contribute to lysosomal malfunction ([Blanz et al., 2010; Chang et al., 2014; Cortes et al., 2014; Endo et al., 2015; Yu et al., 2010]). For example, Niemann-Pick disease type C, caused by loss of NPC1 function, leads to impaired Ca²⁺ homeostasis and incorrect cholesterol trafficking with accumulation of unesterified cholesterol and glycosphingolipids in lysosomes and late endosomes, disrupting their fusion ([Lloyd-Evans et al., 2008; Lloyd-Evans and Platt, 2010; Pacheco and Lieberman, 2008]).

Recently, the group of diseases associated with lysosomal dysfunction has expanded. Mutations in SNX14, a sorting nexin phosphoinositol binding protein localized on the late endosome and lysosomal membrane and involved in cargo sorting upon endocytosis, were found in patients with hereditary cerebellar ataxia. Autophagosome clearance was slowed in patient cells, suggesting lysosome-autophagosome dysfunction ([Akizu et al., 2015]). A specific role for autophagosome-lysosome fusion in lysosomal disease pathology is underlined by the discovery of a missense mutation in VPS11 in patients with a rare form of leukoencephalopathy. VPS11 is a member of the homotypic fusion and protein sorting (HOPS) and class C core vacuole/endosome tethering (CORVET) complexes, and mutations lead to impaired autophagy. A zebrafish VPS11 mutant (*vps11(plt)*) shows a reduction in CNS myelination and extensive neuronal death in the hindbrain and midbrain ([Zhang et al., 2016]).

**Core Autophagy Genes Implicated in Neurodegenerative Disease**

As outlined above, mutations in what might be termed “autophagy accessory” genes have been identified in a number of diseases, but diseases have also been identified that result from mutations in core autophagy genes. Recently, an E122D ATG5 mutation was identified in two siblings with childhood ataxia, characterized by lack of coordination and cerebellar hypoplasia ([Kim et al., 2016]). The E122D mutation weakens binding of ATG5 to ATG12, resulting in decreased autophagosome formation and reduced autophagy flux.Another autophagy gene involved in disease is WDR45, a gene coding for the protein WIPI4, one of the four WIPI proteins ([Proikas-Cezanne et al., 2004](https://doi.org/10.1038/nrm1461)). WIPI proteins are key autophagic lipid sensors that facilitate autophagosome maturation by bridging PI3P production and LC3 lipidation in mammalian cells ([Lamb et al., 2013](https://doi.org/10.1038/nsmb.2579)). De novo mutations in WDR45 cause static encephalopathy of childhood with neurodegeneration in adulthood (SENDA) ([Haack et al., 2012](https://doi.org/10.1038/ng.1127); [Saitsu et al., 2013](https://doi.org/10.1038/ng.2676)), also called β-propeller protein-associated neurodegeneration (BPAN). In lymphoblastoid cell lines derived from BPAN patients, mutant WDR45 is less stable and degraded, resulting in decreased protein levels compared with unaffected individuals. These cells showed an impairment in autophagy flux and an accumulation of LC3-positive autophagosome membranes ([Saitsu et al., 2013](https://doi.org/10.1038/ng.2676)). CNS-specific WDR45 knockout mice exhibit some aspects of the BPAN phenotype: poor motor coordination, impaired learning and memory, and extensive axon swelling with numerous axon spheroids, along with autophagy perturbation in the form of accumulation of SQSTM1/p62 and ubiquitin-positive aggregates ([Zhao et al., 2015](https://doi.org/10.1016/j.neuron.2015.02.028)).

### Autophagy Therapeutics for Neurodegenerative Diseases

The neuropathology of protein misfolding diseases is intimately linked with the propensity of specific proteins to misfold and self-associate, giving rise to an array of oligomers and aggregates. Because many of these proteins cause disease primarily via toxic gain-of function mechanisms, one way of combating disease is to reduce the levels of such proteins ([Ciechanover and Kwon, 2015](https://doi.org/10.1038/nature14540)). Furthermore, most aggregate-prone, neurodegenerative disease-associated proteins have been shown to be autophagy substrates, including mutant huntingtin, α-synuclein, and tau ([Berger et al., 2006](https://doi.org/10.1016/j.cell.2006.04.035); [Ravikumar et al., 2002](https://doi.org/10.1016/S0896-6273(02)00659-X), [2004](https://doi.org/10.1016/j.neuron.2004.04.010); [Webb et al., 2003](https://doi.org/10.1016/S0896-6273(03)00089-3); [Rubinsztein et al., 2012](https://doi.org/10.1038/nature11483)). Importantly, induction of autophagy reduces the levels of both the soluble and aggregated species in such models and is associated with beneficial effects. It is likely that the autophagic process is capturing small oligomeric species and not the very large aggregates visible by light microscopy—the reduction of the large aggregates after autophagy induction may reflect a reduced input of the smaller species into the aggregates and the flux of the mutant proteins on and off the aggregates ([Berger et al., 2006](https://doi.org/10.1016/j.cell.2006.04.035); [Ravikumar et al., 2002](https://doi.org/10.1016/S0896-6273(02)00659-X), [2004](https://doi.org/10.1016/j.neuron.2004.04.010); [Webb et al., 2003](https://doi.org/10.1016/S0896-6273(03)00089-3)).

Many small molecules employed to induce autophagy have an impact on cellular effectors with pleiotropic effects, such as mTOR and AMPK ([Fleming et al., 2011](https://doi.org/10.1016/j.cell.2011.07.015); [Levine et al., 2015](https://doi.org/10.1016/j.cell.2015.05.032)). Nevertheless, in many cases, the effects of such autophagy modulators on the levels of the substrates have been shown to be autophagy-dependent (e.g., [Williams et al., 2008](https://doi.org/10.1016/j.cub.2008.01.018)). Table S1 lists key autophagy enhancing molecules that have been validated in vivo, their mechanism of action, biological targets, and CNS penetration profiles.

Autophagy inducers can be classified into two main groups: those acting via mTOR-dependent or mTOR-independent targets. mTOR inhibitors are either ATP-competitive inhibitors (e.g., Torin1 and related compounds) or non-ATP-competitive inhibitors (e.g., rapamycin and rapalogs) ([Kim and Guan, 2015](https://doi.org/10.1016/j.molcel.2015.03.016)). Because of their inhibition of mTORC1, mTORC2, and, in some cases, PI3K activities, chronic dosing of ATP-competitive inhibitors of mTOR activity in animals presents significant toxicity issues ([Kim and Guan, 2015](https://doi.org/10.1016/j.molcel.2015.03.016)), and, for this reason, they are generally unsuitable for studies in mouse models of neurodegeneration, which typically require chronic dosing. Although mTORC1 inhibits autophagy, mTORC2 is required for autophagy, and the autophagy inhibition resulting from prolonged exposure to such compounds may explain at least some of their toxicity in vivo ([Renna et al., 2013](https://doi.org/10.1016/j.cell.2013.02.029)). Rapamycin and rapalogs have relatively safer profiles because of their non-ATP competitive mode of action and apparent selectivity for mTORC1 ([Wander et al., 2011](https://doi.org/10.1016/j.cell.2011.07.024)), allowing everolimus (a rapalog; [Lebowhl et al., 2013](https://doi.org/10.1016/j.cell.2013.02.029)) to be recently approved by the Food and Drug Administration (FDA) for the treatment of tuberous sclerosis. Rapamycin and its analogs have shown benefits as autophagy inducers in animal models of AD, PD, FTD, HD, and prion protein (PrP) ([Cortes et al., 2012](https://doi.org/10.1016/j.neurobiolaging.2011.12.008); [Jiang et al., 2014](https://doi.org/10.1016/j.neurobiolaging.2013.12.010); [Mengies et al., 2010](https://doi.org/10.1016/j.neurobiolaging.2009.09.008); [Ozcelik et al., 2013](https://doi.org/10.1016/j.neurobiolaging.2012.12.010); [Ravikumar et al., 2004](https://doi.org/10.1016/j.neuron.2004.04.010); [Sarkar et al., 2008](https://doi.org/10.1016/j.neuron.2008.02.022); [Spilman et al., 2010](https://doi.org/10.1016/j.neurobiolaging.2009.09.008); [Wang et al., 2013b](https://doi.org/10.1016/j.neurobiolaging.2012.12.010)). The therapeutic efficacy of rapamycin in SOD1 ALS mouse models is complex, with some studies reporting detrimental effects ([Zhang et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.09.010)) and others reporting beneficial effects only after uncoupling of its immunomodulatory function ([Staats et al., 2013](https://doi.org/10.1016/j.neurobiolaging.2012.12.010)).

Many mTOR-independent autophagy activators signal via AMPK. Trehalose, a widely studied autophagy inducer in neurodegeneration models ([Sarkar et al., 2007](https://doi.org/10.1016/j.neuron.2007.02.008)), has recently been characterized as acting via AMPK activation ([DeBosch et al., 2016](https://doi.org/10.1016/j.neuron.2016.02.028)). The molecular targets of this disaccharide have been proposed to be GLUT proteins, a family of glucose transporters whose inhibition by trehalose results in AMPK activation ([DeBosch et al., 2016](https://doi.org/10.1016/j.neuron.2016.02.028)). Trehalose dosing in mice has shown therapeutic effects, concomitant with autophagy induction, in an impressive variety of mouse models of neurodegeneration, including AD, PD, FTD, HD, SCA17, PrP, ALS, and oculopharyngeal muscular dystrophy (OPMD) ([Aguib et al., 2009](https://doi.org/10.1016/j.neurobiolaging.2008.09.008); [Castillo et al., 2013](https://doi.org/10.1016/j.neurobiolaging.2012.12.010); [Chen et al., 2015](https://doi.org/10.1016/j.neurobiolaging.2014.12.010); [Davies et al., 2006](https://doi.org/10.1016/j.neurobiolaging.2005.09.008); [Du et al., 2013](https://doi.org/10.1016/j.neurobiolaging.2012.12.010); [Li et al., 2015](https://doi.org/10.1016/j.neurobiolaging.2014.12.010); [Rodríguez-Navarro et al., 2010](https://doi.org/10.1016/j.neurobiolaging.2009.09.008); [Schaeffer and Goedert, 2012](https://doi.org/10.1016/j.neurobiolaging.2011.12.010); [Schaeffer et al., 2012](https://doi.org/10.1016/j.neurobiolaging.2011.12.010); [Tanaka et al., 2004](https://doi.org/10.1016/j.neurobiolaging.2003.09.008); [Tanji et al., 2015](https://doi.org/10.1016/j.neurobiolaging.2014.12.010); [Zhang et al., 2014](https://doi.org/10.1016/j.neurobiolaging.2013.12.010)).

Metformin is another AMPK-dependent autophagy inducer that has shown beneficial effects in animal models of neurodegeneration. Metformin has shown efficacy in animal models of AD, HD, and Lafora disease ([Berthier et al., 2016](https://doi.org/10.1016/j.neurobiolaging.2015.12.010); [Ma et al., 2007](https://doi.org/10.1016/j.neurobiolaging.2006.09.008); [Son et al., 2016](https://doi.org/10.1016/j.neurobiolaging.2015.12.010)) but showed no efficacy in ALS models ([Kaneb et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.09.008)). Berberine, which showed efficacy in an HD model ([Jiang et al., 2015](https://doi.org/10.1016/j.neurobiolaging.2014.12.010)), may also induce autophagy via AMPK activation ([Yu et al., 2014](https://doi.org/10.1016/j.neurobiolaging.2013.12.010)). Methylene blue, a reported anti-aggregant and AMPK activator ([Shin et al., 2014](https://doi.org/10.1016/j.neurobiolaging.2013.12.010)), was shown to induce autophagy and ameliorate the disease phenotype in FTD models ([Congdon et al., 2012](https://doi.org/10.1016/j.neurobiolaging.2011.12.010); [Hochgräfe et al., 2015](https://doi.org/10.1016/j.neurobiolaging.2014.12.010); [Melis et al., 2015](https://doi.org/10.1016/j.neurobiolaging.2014.12.010)). Nilotinib, a c-Abl kinase inhibitor and AMPK activator ([Yu et al., 2013](https://doi.org/10.1016/j.neurobiolaging.2012.12.010)), was shown to induce autophagy and ameliorate the disease phenotype in PD mice ([Hebron et al., 2014](https://doi.org/10.1016/j.neurobiolaging.2013.12.010); [Lonskaya et al., 2015](https://doi.org/10.1016/j.neurobiolaging.2014.12.010)).

Finally, a growing number of autophagy inducers are being characterized that may act on other pathways, including compounds acting on the modulation of cyclic AMP (cAMP)/inositol triphosphate (IP₃), such as rilmenidine, clonidine, minoxidil,and verapamil, which have been shown to ameliorate phenotypes in animal models of HD ([Rose et al., 2010](https://doi.org/10.1038/nm.2145); [Williams et al., 2008](https://doi.org/10.1074/jbc.M709679200); Table S1).

In addition to the small molecules with in vivo validation reviewed above, there are other molecules that have been shown to be active in cell-based models but not yet in vivo. These are presented, with their mechanism of action and any available evidence for potential CNS penetration, in Table S2.

### Non-Small-Molecule Approaches

In addition to small-molecule therapeutics, a number of other strategies for autophagy enhancement have been adopted. [Shoji-Kawata et al. (2013)](https://doi.org/10.1038/emboj.2013.10) have reported the development of a Tat-Beclin 1 peptide that consists of the HIV-1 Tat protein transduction domain linked to a modified 18 amino acid sequence based on the residues 267–284 of Beclin 1. This region was identified as being required for the autophagy-inducing effects of Beclin 1. The Tat-Beclin 1 peptide was found to increase autophagic flux via the canonical pathway using a range of cell models and to decrease the number of small huntingtin aggregates in HeLa cells. It was also shown that Tat-Beclin 1 increased GFP-LC3 dots in a range of tissues in a transgenic mouse model ([Shoji-Kawata et al., 2013](https://doi.org/10.1038/emboj.2013.10)).

A number of mouse studies exploring the potential of gene therapy have been reported. For example, delivery of the TFEB gene ameliorates signs of disease in a PD model based on adeno-associated virus (AAV) vector-mediated overexpression of human wild-type α-synuclein in the rat midbrain ([Decressac et al., 2013](https://doi.org/10.1016/j.neuron.2013.02.018)), and, in a transgenic mouse model overexpressing tau, TFEB was able to decrease the levels of pathological tau and improve cognitive performance and synaptic function ([Polito et al., 2014](https://doi.org/10.1016/j.neuron.2014.03.025)). Beneficial effects have also been reported with Beclin 1 overexpression in an α-synuclein model of PD/Lewy body diseases ([Spencer et al., 2009](https://doi.org/10.1016/j.neuron.2009.03.013)) and in a model of Machado-Joseph disease ([Nascimento-Ferreira et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.08.018)). Parkin overexpression has beneficial effects in an AD model ([Khandelwal et al., 2011](https://doi.org/10.1016/j.neuron.2011.05.022)).

Another possible candidate target for such approaches is calpain. Knockdown of calpain in a Huntington’s disease *Drosophila* model (RNAi) results in upregulation of autophagy, reduction in huntingtin aggregate number, and increases cell survival and is also protective against tau toxicity in HD and tauopathy models, respectively ([Menzies et al., 2015b](https://doi.org/10.1016/j.cell.2015.03.027)). In mice, neuronal overexpression of calpastatin (CAST), an endogenous inhibitor of calpain, increased autophagy and confers protection when crossed to HD mice, improving aggregate burden, behavioral measures of degeneration, and long-term survival ([Menzies et al., 2015b](https://doi.org/10.1016/j.cell.2015.03.027)). These CAST transgenic mice were compared with wild-type littermates, and no deficits were found by SmithKline Beecham, Harwell, Imperial College, Royal London Hospital phenotype assessment (SHIRPA) behavioral testing or in survival, suggesting that such a strategy may be well tolerated.

### Considerations for Autophagy as a Therapeutic Strategy in Neurodegeneration

Taken together, the evidence from pharmacological efficacy mouse studies suggests that autophagy activation has significant therapeutic potential across a wide range of neurodegenerative diseases. Although the majority of studies of autophagy-modulating approaches in neurodegenerative disease models have reported beneficial effects on disease signs, this is not universally true ([Hernandez et al., 2012](https://doi.org/10.1016/j.neuron.2012.03.014)). For example, in the SOD1 G93A mutant model of ALS, treatment with rapamycin decreased the lifespan and increased motor neuron degeneration ([Zhang et al., 2011](https://doi.org/10.1016/j.neuron.2011.05.018)). However, rapamycin is protective in such mice lacking mature lymphocytes, suggesting that the immunosuppressive effects of this drug (unrelated to autophagy) may be deleterious in this disease and may counterbalance the protective effects of autophagy induction ([Staats et al., 2013](https://doi.org/10.1016/j.neuron.2013.03.018)). Indeed, an alternative mTOR-independent autophagy inducer, trehalose, is protective in this SOD1 ALS model ([Zhang et al., 2014](https://doi.org/10.1016/j.neuron.2014.03.025)).

Similarly, the time point in disease at which autophagy is induced may be critical. For example, many studies have demonstrated the protective effect of autophagy upregulation on α-synuclein toxicity (e.g., [Decressac et al., 2013](https://doi.org/10.1016/j.neuron.2013.02.018); [Hebron et al., 2014](https://doi.org/10.1016/j.neuron.2014.03.025); [Spencer et al., 2009](https://doi.org/10.1016/j.neuron.2009.03.013)). However, at least in tissue culture, α-synuclein aggregate-containing cells appear to have impaired autophagosome maturation, and autophagy upregulation may actually be toxic under these conditions ([Tanik et al., 2013](https://doi.org/10.1016/j.neuron.2013.03.018)). Likewise, the point of intervention in the autophagy pathway might be an important consideration. For example, where autophagosome clearance or degradation is impaired, upstream induction of autophagy may lead to a largely unproductive accumulation of autophagosomes, possibly leading to toxic consequences; this may be avoided by more directly targeting lysosomal function or using approaches that enhance the entire pathway, including increasing lysosomal capacity (for example, with TFEB, as discussed above). Thus, one should ideally understand autophagy dynamics in diseases one is aiming to treat to best tailor the therapeutic approach to the most suitable diseases. Further developments in autophagy-based therapeutics for neurodegeneration must await the advent of sharper pharmacological tools for autophagy induction, with more selective molecular targets within the autophagic process, and of higher resolution readouts for different steps of the autophagic flux with which autophagy dysfunction in different neurodegenerative diseases can be profiled.

### Conclusions

In this review, we have highlighted how autophagy defects at various stages of the pathway may be seen in diverse neurodegenerative conditions and that autophagy upregulation may be a promising therapeutic strategy for some diseases. However, the roles of autophagy in health and disease may be more complex and not simply cell-autonomous. For example, autophagy affects inflammatory and immune processes, which are increasingly being implicated in various neurodegenerative diseases ([Rubinsztein et al., 2015](https://doi.org/10.1016/j.neuron.2015.01.039)). Future studies aiming to understand non-cell-autonomous effects of autophagy in the nervous system and its relevance to neuronal-glial interactions are likely to be informative.

From a therapeutic perspective, we believe that autophagy is a promising target mechanism. In many cases, it enhances the removal of the primary toxic entity causing disease (e.g., mutant tau or mutant huntingtin) and, thus, targets such diseases at their roots. Autophagy upregulation has additional protective effectsby reducing the susceptibility to pro-death insults (Boya et al., 2005; Ravikumar et al., 2006). One benefit of autophagy upregulation as a therapeutic approach is that one need not require constitutive activation of the pathway because a pulsatile strategy (like periodically taking out the rubbish) may be sufficient to have efficacy, especially when employed over a long time period. This would have major benefits from a drug toxicity perspective.

**SUPPLEMENTAL INFORMATION**

Supplemental Information includes two tables and can be found with this article online at http://dx.doi.org/10.1016/j.neuron.2017.01.022.

**AUTHOR CONTRIBUTIONS**

F.M.M., A.F., A.C., C.F.B., S.P.A., A.A., J.F., A.J., M.J.S., C.K., F.L., A.L.R., M.P., C.P., M.R., T.R., L.S., M.V., H.W., Y.Z., J.S., and D.C.R. all contributed to the research and drafting of the article. The different sections were coordinated by F.M.M. and D.C.R. All authors reviewed final manuscript. Funding acquisition: D.C.R., A.A., and J.F.

**ACKNOWLEDGMENTS**

D.C.R. is grateful for funding from the Tau Consortium, Alzheimer’s Research UK, Wellcome Trust (Principal Research Fellowship 095317/Z/11/Z), a Wellcome Trust Strategic Grant to the Cambridge Institute for Medical Research (100140/Z/12/Z), NIHR Biomedical Research Unit in Dementia at Addenbrooke’s Hospital, The Rosetrees Trust, FEBS (long-term fellowships to A.A. and J.F.), Addenbrooke’s Charitable Trust, Sims Scholarship, and the James Baird Fund (R.F.). F.M.M. is currently an employee of Eli Lilly and Co.

**REFERENCES**

Aguado, C., Sarkar, S., Korolchuk, V.I., Criado, O., Vernia, S., Boya, P., Sanz, P., de Córdoba, S.R., Knecht, E., and Rubinsztein, D.C. (2010). Laforin, the most common protein mutated in Lafora disease, regulates autophagy. Hum. Mol. Genet. 19, 2867–2876.

Aguib, Y., Heiseke, A., Gilch, S., Riemer, C., Baier, M., Schätzl, H.M., and Ertmer, A. (2009). Autophagy induction by trehalose counteracts cellular prion infection. Autophagy 5, 361–369.

Akizu, N., Cantagrel, V., Zaki, M.S., Al-Gazali, L., Wang, X., Rosti, R.O., Diko-glu, E., Gelot, A.B., Rosti, B., Vaux, K.K., et al. (2015). Biallelic mutations in SNX14 cause a syndromic form of cerebellar atrophy and lysosome-autophagosome dysfunction. Nat. Genet. 47, 528–534.

Alves, S., Cormier-Dequaire, F., Marinello, M., Marais, T., Muriel, M.P., Beaumatin, F., Charbonnier-Beaupel, F., Tahiri, K., Seilhean, D., El Hachimi, K., et al. (2014). The autophagy/lysosome pathway is impaired in SCA7 patients and SCA7 knock-in mice. Acta Neuropathol. 128, 705–722.

Ando, K., Brion, J.P., Stygelbout, V., Suain, V., Authelet, M., Dedecker, R., Chanut, A., Lacor, P., Lavaur, J., Sazdovitch, V., et al. (2013). Clathrin adaptor CALM/PICALM is associated with neurofibrillary tangles and is cleaved in Alzheimer’s brains. Acta Neuropathol. 125, 861–878.

Ando, K., Tomimura, K., Sazdovitch, V., Suain, V., Yilmaz, Z., Authelet, M., Ndjim, M., Vergara, C., Belkouch, M., Potier, M.C., et al. (2016). Level of PIC-ALM, a key component of clathrin-mediated endocytosis, is correlated with levels of phosphotau and autophagy-related proteins and is associated with tau inclusions in AD, PSP and Pick disease. Neurobiol. Dis. 94, 32–43.

Axe, E.L., Walker, S.A., Manifava, M., Chandra, P., Roderick, H.L., Habermann, A., Griffiths, G., and Ktistakis, N.T. (2008). Autophagosome formation from membrane compartments enriched in phosphatidylinositol 3-phosphate and dynamically connected to the endoplasmic reticulum. J. Cell Biol. 182, 685–701.

Ballabio, A., and Gieselmann, V. (2009). Lysosomal disorders: from storage to cellular damage. Biochim. Biophys. Acta 1793, 684–696.

Bejarano, E., Yuste, A., Patel, B., Stout, R.F., Jr., Spray, D.C., and Cuervo, A.M. (2014). Connexins modulate autophagosome biogenesis. Nat. Cell Biol. 16, 401–414.

Bento, C.F., Ashkenazi, A., Jimenez-Sanchez, M., and Rubinsztein, D.C. (2016a). The Parkinson’s disease-associated genes ATP13A2 and SYT11 regulate autophagy via a common pathway. Nat. Commun. 7, 11803.

Bento, C.F., Renna, M., Ghislat, G., Puri, C., Ashkenazi, A., Vicinanza, M., Menzies, F.M., and Rubinsztein, D.C. (2016b). Mammalian Autophagy: How Does It Work? Annu. Rev. Biochem. 85, 685–713.

Berger, Z., Ravikumar, B., Menzies, F.M., Oroz, L.G., Underwood, B.R., Pangalos, M.N., Schmitt, I., Wullner, U., Evert, B.O., O’Kane, C.J., and Rubinsztein, D.C. (2006). Rapamycin alleviates toxicity of different aggregate-prone proteins. Hum. Mol. Genet. 15, 433–442.

Berthier, A., Payá, M., García-Cabrero, A.M., Ballester, M.I., Heredia, M., Ser-ratosa, J.M., Sánchez, M.P., and Sanz, P. (2016). Pharmacological Interventions to Ameliorate Neuropathological Symptoms in a Mouse Model of Lafora Disease. Mol. Neurobiol. 53, 1296–1309.

Bird, T.D. (1993a). Charcot-Marie-Tooth Hereditary Neuropathy Overview. In GeneReviews(R), R.A. Pagon, M.P. Adam, H.H. Ardinger, S.E. Wallace, A. Amemiya, L.J.H. Bean, T.D. Bird, C.T. Fong, H.C. Mefford, R.J.H. Smith, and K. Stephens, eds. (Seattle, WA: University of Washington).

Bird, T.D. (1993b). Charcot-Marie-Tooth Neuropathy Type 1. In GeneRe-views(R), R.A. Pagon, M.P. Adam, H.H. Ardinger, S.E. Wallace, A. Amemiya, L.J.H. Bean, T.D. Bird, C.T. Fong, H.C. Mefford, R.J.H. Smith, and K. Ste-phen, eds. (Seattle, WA: University of Washington).

Bjørkøy, G., Lamark, T., Brech, A., Outzen, H., Perander, M., Overvatn, A., Stenmark, H., and Johansen, T. (2005). p62/SQSTM1 forms protein aggre-gates degraded by autophagy and has a protective effect on huntingtin-induced cell death. J. Cell Biol. 171, 603–614.

Blanz, J., Groth, J., Zachos, C., Wehling, C., Saftig, P., and Schwake, M. (2010). Disease-causing mutations within the lysosomal integral membrane protein type 2 (LIMP-2) reveal the nature of binding to its ligand beta-glucocer-ebosidase. Hum. Mol. Genet. 19, 563–572.

Blokhuis, A.M., Groen, E.J., Koppers, M., van den Berg, L.H., and Pasterkamp, R.J. (2013). Protein aggregation in amyotrophic lateral sclerosis. Acta Neuro-pathol. 125, 777–794.

Boland, B., Kumar, A., Lee, S., Platt, F.M., Wegiel, J., Yu, W.H., and Nixon, R.A. (2008). Autophagy induction and autophagosome clearance in neurons: relationship to autophagic pathology in Alzheimer’s disease. J. Neurosci. 28, 6926–6937.

Boya, P., González-Polo, R.A., Casares, N., Perfettini, J.L., Dessen, P., Larochette, N., Métivier, D., Meley, D., Souqueire, S., Yoshimori, T., et al. (2005). Inhi-bition of macroautophagy triggers apoptosis. Mol. Cell. Biol. 25, 1025–1040.

Butzlaff, M., Hannan, S.B., Karsten, P., Lenz, S., Ng, J., Voßfeldt, H., Prüßing, K., Pflanz, R., Schulz, J.B., Rasse, T., and Voigt, A. (2015). Impaired retrograde transport by the Dynein/Dynactin complex contributes to Tau-induced toxicity. Hum. Mol. Genet. 24, 3623–3637.

Castillo, K., Nassif, M., Valenzuela, V., Rojas, F., Matus, S., Mercado, G., Court, F.A., van Zundert, B., and Hetz, C. (2013). Trehalose delays the progression of amyotrophic lateral sclerosis by enhancing autophagy in motoneurons. Autophagy 9, 1308–1320.

Chang, J., Lee, S., and Blackstone, C. (2014). Spastic paraplegia proteins spastizin and spatacsin mediate autophagic lysosome reformation. J. Clin. Invest. 124, 5249–5262.

Chauhan, S., Goodwin, J.G., Chauhan, S., Manyam, G., Wang, J., Kamat, A.M., and Boyd, D.D. (2013). ZKSCAN3 is a master transcriptional repressor of autophagy. Mol. Cell 50, 16–28.

Chen, Y., Sawada, O., Kohno, H., Le, Y.Z., Subauste, C., Maeda, T., and Maeda, A. (2013). Autophagy protects the retina from light-induced degeneration. J. Biol. Chem. 288, 7506–7518.

Chen, L., Xie, Z., Turkson, S., and Zhuang, X. (2015). A53T human α-synuclein overexpression in transgenic mice induces pervasive mitochondria macroautophagy defects preceding dopamine neuron degeneration. J. Neurosci. 35, 890–905.

1026 Neuron 93, March 8, 2017
Ciechanover, A., and Kwon, Y.T. (2015). Degradation of misfolded proteins in neurodegenerative diseases: therapeutic targets and strategies. Exp Mol Med. 47, e147.

Cirulli, E.T., Lasseigne, B.N., Petrovski, S., Sapp, P.C., Dion, P.A., Leblond, C.S., Couthouis, J., Lu, Y.F., Wang, Q., Krueger, B.J., et al.; FALS Sequencing Consortium (2015). Exome sequencing in amyotrophic lateral sclerosis identifies risk genes and pathways. Science 347, 1436–1441.

Citron, M., Westaway, D., Xia, W., Carlson, G., Diehl, T., Levesque, G., Johnson-Wood, K., Lee, M., Seubert, P., Davis, A., et al. (1997). Mutant presenilins of Alzheimer’s disease increase production of 42-residue amyloid beta-protein in both transfected cells and transgenic mice. Nat. Med. 3, 67–72.

Coffey, E.E., Beckel, J.M., Laties, A.M., and Mitchell, C.H. (2014). Lysosomal alkalization and dysfunction in human fibroblasts with the Alzheimer’s disease-linked presenilin 1 A246E mutation can be reversed with cAMP. Neuroscience 263, 111–124.

Colacurcio, D.J., and Nixon, R.A. (2016). Disorders of lysosomal acidification—the emerging role of v-ATPase in aging and neurodegenerative disease. Ageing Res. Rev. 32, 75–88.

Collin, L., Bohrmann, B., Göpfert, U., Oroszlan-Szovik, K., Ozmen, L., and Grüninger, F. (2014). Neuronal uptake of tau/pS422 antibody and reduced progression of tau pathology in a mouse model of Alzheimer’s disease. Brain 137, 2834–2846.

Congdon, E.E., Wu, J.W., Myeku, N., Figueroa, Y.H., Herman, M., Marinec, P.S., Gestwicki, J.E., Dickey, C.A., Yu, W.H., and Duff, K.E. (2012). Methylthioninium chloride (methylene blue) induces autophagy and attenuates tauopathy in vitro and in vivo. Autophagy 8, 609–622.

Corcelle-Termeau, E., Vindeløv, S.D., Hämalistö, S., Mograbi, B., Keldsbo, A., Bräsen, J.H., Favaro, E., Adam, D., Szyniarowski, P., Hofman, P., et al. (2016). Excess sphingomyelin disturbs ATG9A trafficking and autophagosome closure. Autophagy 12, 833–849.

Cortes, C.J., Qin, K., Cook, J., Solanki, A., and Mastrianni, J.A. (2012). Rapamycin delays disease onset and prevents PrP plaque deposition in a mouse model of Gerstmann-Sträussler-Scheinker disease. J. Neurosci. 32, 12396–12405.

Cortes, C.J., Miranda, H.C., Frankowski, H., Batlevi, Y., Young, J.E., Le, A., Ivanov, N., Sopher, B.L., Carromeu, C., Muotri, A.R., et al. (2014). Polyglutamine-expanded androgen receptor interferes with TFEB to elicit autophagy defects in SBMA. Nat. Neurosci. 17, 1180–1189.

Criado, O., Aguado, C., Gayarre, J., Duran-Trio, L., Garcia-Cabrero, A.M., Vera, S., San Millán, B., Heredia, M., Romá-Mateo, C., Mouron, S., et al. (2012). Lafora bodies and neurological defects in malin-deficient mice correlate with impaired autophagy. Hum. Mol. Genet. 21, 1521–1533.

Cunha-Santos, J., Duarte-Neves, J., Carmona, V., Guarente, L., Pereira de Almeida, L., and Cavadas, C. (2016). Caloric restriction blocks neuropathology and motor deficits in Machado-Joseph disease mouse models through SIRT1 pathway. Nat. Commun. 7, 11445.

Davies, J.E., Sarkar, S., and Rubinsztein, D.C. (2006). Trehalose reduces aggregate formation and delays pathology in a transgenic mouse model of oculopharyngeal muscular dystrophy. Hum. Mol. Genet. 15, 23–31.

DeBosch, B.J., Heitmeier, M.R., Mayer, A.L., Higgins, C.B., Crowley, J.R., Kraft, T.E., Chi, M., Newberry, E.P., Chen, Z., Finck, B.N., et al. (2016). Trehalose inhibits solute carrier 2A (SLC2A) proteins to induce autophagy and prevent hepatic steatosis. Sci. Signal. 9, ra21.

Decressac, M., Mattsson, B., Weikop, P., Lundblad, M., Jakobsson, J., and Björklund, A. (2013). TFEB-mediated autophagy rescues midbrain dopamine neurons from α-synuclein toxicity. Proc. Natl. Acad. Sci. USA 110, E1817–E1826.

Dehay, B., Ramirez, A., Martinez-Vicente, M., Perier, C., Cannon, M.H., Dou dokoff, E., Vital, A., Vila, M., Klein, C., and Bezard, E. (2012). Loss of P-type ATPase ATP13A2/PARK9 function induces general lysosomal deficiency and leads to Parkinson disease neurodegeneration. Proc. Natl. Acad. Sci. USA 109, 9611–9616.

DeJesus-Hernandez, M., Mackenzie, I.R., Boeve, B.F., Boxer, A.L., Baker, M., Rutherford, N.J., Nicholson, A.M., Finch, N.A., Flynn, H., Adamson, J., et al. (2011). Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS. Neuron 72, 245–256.


Dierks, T., Schlotawa, L., Frese, M.A., Radhakrishnan, K., von Figura, K., and Schmidt, B. (2009). Molecular basis of multiple sulfatase deficiency, mucolipidosis II/III and Niemann-Pick C1 disease—Lysosomal storage disorders caused by defects of non-lysosomal proteins. Biochim. Biophys. Acta 1793, 710–725.

Dooley, H.C., Razi, M., Polson, H.E., Girardin, S.E., Wilson, M.I., and Tooze, S.A. (2014). WIPI2 links LC3 conjugation with PI3P, autophagosome formation, and pathogen clearance by recruiting Atg12-5-16L1. Mol. Cell 55, 238–252.

Du, J., Liang, Y., Xu, F., Sun, B., and Wang, Z. (2013). Trehalose rescues Alzheimer’s disease phenotypes in APP/PS1 transgenic mice. J. Pharm. Pharmacol. 65, 1753–1756.

Durcan, T.M., Kontogiannea, M., Thorarinsdottir, T., Fallon, L., Williams, A.J., Djarmati, A., Fantaneau, T., Paulson, H.L., and Fon, E.A. (2011). The Machado-Joseph disease-associated mutant form of ataxin-3 regulates parkin ubiquitination and stability. Hum. Mol. Genet. 20, 141–154.

Durieux, A.C., Vassilopoulos, S., Lainé, J., Fraysse, B., Briñas, L., Prudhon, B., Castells, J., Freyssenet, D., Bonne, G., Guicheney, P., and Bitoun, M. (2012). A centronuclear myopathy-dynamin 2 mutation impairs autophagy in mice. Traffic 13, 869–879.

Egan, D., Kim, J., Shaw, R.J., and Guan, K.L. (2011). The autophagy initiating kinase ULK1 is regulated via opposing phosphorylation by AMPK and mTOR. Autophagy 7, 643–644.

Endo, Y., Furuta, A., and Nishino, I. (2015). Danon disease: a phenotypic expression of LAMP-2 deficiency. Acta Neuropathol. 129, 391–398.

Fang, X., Zhou, J., Liu, W., Duan, X., Gala, U., Sandoval, H., Jaiswal, M., and Tong, C. (2016). Dynamin Regulates Autophagy by Modulating Lysosomal Function. J. Genet. Genomics 43, 77–86.

Farg, M.A., Sundaramoorthy, V., Sultana, J.M., Yang, S., Atkinson, R.A., Levine, V., Halloran, M.A., Gleeson, P.A., Blair, I.P., Soo, K.Y., et al. (2014). C9ORF72, implicated in amyotrophic lateral sclerosis and frontotemporal dementia, regulates endosomal trafficking. Hum. Mol. Genet. 23, 3579–3595.

Fecto, F., Yan, J., Vemula, S.P., Liu, E., Yang, Y., Chen, W., Zheng, J.G., Shi, Y., Siddique, N., Arrat, H., et al. (2011). SQSTM1 mutations in familial and sporadic amyotrophic lateral sclerosis. Arch. Neurol. 68, 1440–1446.

Filimonenko, M., Isakson, P., Finley, K.D., Anderson, M., Jeong, H., Melia, T.J., Bartlett, B.J., Myers, K.M., Birkeland, H.C., Lamark, T., et al. (2010). The selective macroautophagic degradation of aggregated proteins requires the PI3P-binding protein Alfy. Mol. Cell 38, 265–279.

Fink, J.K. (2013). Hereditary spastic paraplegia: clinico-pathologic features and emerging molecular mechanisms. Acta Neuropathol. 126, 307–328.

Fleming, A., Noda, T., Yoshimori, T., and Rubinsztein, D.C. (2011). Chemical modulators of autophagy as biological probes and potential therapeutics. Nat. Chem. Biol. 7, 9–17.

Fraldi, A., Annunziata, F., Lombardi, A., Kaiser, H.J., Medina, D.L., Spampanato, C., Fedele, A.O., Polishchuk, R., Sorrentino, N.C., Simons, K., and Ballabio, A. (2010). Lysosomal fusion and SNARE function are impaired by cholesterol accumulation in lysosomal storage disorders. EMBO J. 29, 3607–3620.

Freischmidt, A., Wieland, T., Richter, B., Ruf, W., Schaeffer, V., Müller, K., Marroquin, N., Nordin, F., Hübers, A., Weydt, P., et al. (2015). Haploinsufficiency of TBK1 causes familial ALS and fronto-temporal dementia. Nat. Neurosci. 18, 631–636.

Füllgrabe, J., Klionsky, D.J., and Joseph, B. (2014). The return of the nucleus: transcriptional and epigenetic control of autophagy. Nat. Rev. Mol. Cell Biol. 15, 65–74.

Gage, F.H. (2000). Mammalian neural stem cells. Science 287, 1433–1438.

Gal, J., Ström, A.L., Kwinter, D.M., Kilty, R., Zhang, J., Shi, P., Fu, W., Wooten, M.W., and Zhu, H. (2009). Sequestosome 1/p62 links familial ALS mutant SOD1 to LC3 via an ubiquitin-independent mechanism. J. Neurochem. 111, 1062–1073.

Ganley, I.G., Lam, H., Wang, J., Ding, X., Chen, S., and Jiang, X. (2009). ULK1.ATG13.FIP200 complex mediates mTOR signaling and is essential for autophagy. J. Biol. Chem. 284, 12297–12305.

Ganley, I.G., Wong, P.M., Gammoh, N., and Jiang, X. (2011). Distinct autophagolysosomal fusion mechanism revealed by thapsigargin-induced autophagy arrest. Mol. Cell 42, 731–743.

Garyali, P., Segvich, D.M., DePaoli-Roach, A.A., and Roach, P.J. (2014). Protein degradation and quality control in cells from laforin and malin knockout mice. J. Biol. Chem. 289, 20606–20614.

Gautier, C.A., Kitada, T., and Shen, J. (2008). Loss of PINK1 causes mitochondrial functional defects and increased sensitivity to oxidative stress. Proc. Natl. Acad. Sci. USA 105, 11364–11369.

Ge, L., Melville, D., Zhang, M., and Schekman, R. (2013). The ER-Golgi intermediate compartment is a key membrane source for the LC3 lipidation step of autophagosome biogenesis. eLife 2, e00947.

Ge, L., Zhang, M., and Schekman, R. (2014). Phosphatidylinositol 3-kinase and COPII generate LC3 lipidation vesicles from the ER-Golgi intermediate compartment. eLife 3, e04135.

Ghavidel, A., Baxi, K., Ignatchenko, V., Prusinkiewicz, M., Arnason, T.G., Kislinger, T., Carvalho, C.E., and Harkness, T.A. (2015). A Genome Scale Screen for Mutants with Delayed Exit from Mitosis: Ire1-Independent Induction of Autophagy Integrates ER Homeostasis into Mitotic Lifespan. PLoS Genet. 11, e1005429.

Goldberg, M.S., Fleming, S.M., Palacino, J.J., Cepeda, C., Lam, H.A., Bhatnagar, A., Meloni, E.G., Wu, N., Ackerson, L.C., Klapstein, G.J., et al. (2003). Parkin-deficient mice exhibit nigrostriatal deficits but not loss of dopaminergic neurons. J. Biol. Chem. 278, 43628–43635.

Gomez-Sanchez, J.A., Carty, L., Iruarrizaga-Lejarreta, M., Palomo-Irigoyen, M., Varela-Rey, M., Griffith, M., Hantke, J., Macias-Camara, N., Azkargorta, M., Aurrekoetxea, I., et al. (2015). Schwann cell autophagy, myelinophagy, initiates myelin clearance from injured nerves. J. Cell Biol. 210, 153–168.

Gonzalez, M.A., Feely, S.M., Speziani, F., Strickland, A.V., Danzi, M., Bacon, C., Lee, Y., Chou, T.F., Blanton, S.H., Weihl, C.C., et al. (2014). A novel mutation in VCP causes Charcot-Marie-Tooth Type 2 disease. Brain 137, 2897–2902.

Goode, A., Butler, K., Long, J., Cavey, J., Scott, D., Shaw, B., Sollenberger, J., Gell, C., Johansen, T., Oldham, N.J., et al. (2016). Defective recognition of LC3B by mutant SQSTM1/p62 implicates impairment of autophagy as a pathogenic mechanism in ALS-FTLD. Autophagy 12, 1094–1104.

Guo, Y., Chang, C., Huang, R., Liu, B., Bao, L., and Liu, W. (2012). AP1 is essential for generation of autophagosomes from the trans-Golgi network. J. Cell Sci. 125, 1706–1715.

Haack, T.B., Hogarth, P., Kruer, M.C., Gregory, A., Wieland, T., Schwarzmayr, T., Graf, E., Sanford, L., Meyer, E., Kara, E., et al. (2012). Exome sequencing reveals de novo WDR45 mutations causing a phenotypically distinct, X-linked dominant form of NBIA. Am. J. Hum. Genet. 91, 1144–1149.

Hamasaki, M., Furuta, N., Matsuda, A., Nezu, A., Yamamoto, A., Fujita, N., Oomori, H., Noda, T., Haraguchi, T., Hiraoka, Y., et al. (2013). Autophagosomes form at ER-mitochondria contact sites. Nature 495, 389–393.

Hansen, M., Chandra, A., Mitic, L.L., Onken, B., Driscoll, M., and Kenyon, C. (2008). A role for autophagy in the extension of lifespan by dietary restriction in C. elegans. PLoS Genet. 4, e24.

Hara, T., Nakamura, K., Matsui, M., Yamamoto, A., Nakahara, Y., Suzuki-Migishima, R., Yokoyama, M., Mishima, K., Saito, I., Okano, H., and Mizushima, N. (2006). Suppression of basal autophagy in neural cells causes neurodegenerative disease in mice. Nature 441, 885–889.

Harold, D., Abraham, R., Hollingworth, P., Sims, R., Gerrish, A., Hamshere, M.L., Pahwa, J.S., Moskvina, V., Dowzell, K., Williams, A., et al. (2009). Genome-wide association study identifies variants at CLU and PICALM associated with Alzheimer’s disease. Nat. Genet. 41, 1088–1093.

Hawkins-Salsbury, J.A., Parameswar, A.R., Jiang, X., Schlesinger, P.H., Bongarzone, E., Ory, D.S., Demchenko, A.V., and Sands, M.S. (2013). Psychosine, the cytotoxic sphingolipid that accumulates in globoid cell leukodystrophy, alters membrane architecture. J. Lipid Res. 54, 3303–3311.

Hayashi-Nishino, M., Fujita, N., Noda, T., Yamaguchi, A., Yoshimori, T., and Yamamoto, A. (2009). A subdomain of the endoplasmic reticulum forms a cradle for autophagosome formation. Nat. Cell Biol. 11, 1433–1437.


Hebron, M.L., Lonskaya, I., Olopade, P., Selby, S.T., Pagan, F., and Moussa, C.E. (2014). Tyrosine Kinase Inhibition Regulates Early Systemic Immune Changes and Modulates the Neuroimmune Response in α-Synucleinopathy. J. Clin. Cell. Immunol. 5, 259.

Hernandez, D., Torres, C.A., Setlik, W., Cebrián, C., Mosharov, E.V., Tang, G., Cheng, H.C., Kholodilov, N., Yarygina, O., Burke, R.E., et al. (2012). Regulation of presynaptic neurotransmission by macroautophagy. Neuron 74, 277–284.

Hirst, J., Edgar, J.R., Esteves, T., Darios, F., Madeo, M., Chang, J., Roda, R.H., Dürr, A., Anheim, M., Gellera, C., et al. (2015). Loss of AP-5 results in accumulation of aberrant endolysosomes: defining a new type of lysosomal storage disease. Hum. Mol. Genet. 24, 4984–4996.

Hochgräfe, K., Sydow, A., Matenia, D., Cadinu, D., Könen, S., Petrova, O., Pickhardt, M., Goll, P., Morellini, F., Mandelkow, E., and Mandelkow, E.M. (2015). Preventive methylene blue treatment preserves cognition in mice expressing full-length pro-aggregant human Tau. Acta Neuropathol. Commun. 3, 25.

Hosokawa, N., Hara, T., Kaizuka, T., Kishi, C., Takamura, A., Miura, Y., Iemura, S., Natsume, T., Takehana, K., Yamada, N., et al. (2009). Nutrient-dependent mTORC1 association with the ULK1-Atg13-FIP200 complex required for autophagy. Mol. Biol. Cell 20, 1981–1991.

Huang, R., Xu, Y., Wan, W., Shou, X., Qian, J., You, Z., Liu, B., Chang, C., Zhou, T., Lippincott-Schwartz, J., and Liu, W. (2015). Deacetylation of nuclear LC3 drives autophagy initiation under starvation. Mol. Cell 57, 456–466.

Itier, J.M., Ibanez, P., Mena, M.A., Abbas, N., Cohen-Salmon, C., Bohme, G.A., Laville, M., Pratt, J., Corti, O., Pradier, L., et al. (2003). Parkin gene inactivation alters behaviour and dopamine neurotransmission in the mouse. Hum. Mol. Genet. 12, 2277–2291.

Jahreiss, L., Menzies, F.M., and Rubinsztein, D.C. (2008). The itinerary of autophagosomes: from peripheral formation to kiss-and-run fusion with lysosomes. Traffic 9, 574–587.

Jang, S.Y., Shin, Y.K., Park, S.Y., Park, J.Y., Rha, S.H., Kim, J.K., Lee, H.J., and Park, H.T. (2015). Autophagy is involved in the reduction of myelinating Schwann cell cytoplasm during myelin maturation of the peripheral nerve. PLoS ONE 10, e0116624.

Jang, S.Y., Shin, Y.K., Park, S.Y., Park, J.Y., Lee, H.J., Yoo, Y.H., Kim, J.K., and Park, H.T. (2016). Autophagic myelin destruction by Schwann cells during Wallerian degeneration and segmental demyelination. Glia 64, 730–742.

Jeong, H., Then, F., Melia, T.J., Jr., Mazzulli, J.R., Cui, L., Savas, J.N., Voisin, C., Paganetti, P., Tanese, N., Hart, A.C., et al. (2009). Acetylation targets mutant huntingtin to autophagosomes for degradation. Cell 137, 60–72.

Jiang, T., Yu, J.T., Zhu, X.C., Zhang, Q.Q., Cao, L., Wang, H.F., Tan, M.S., Gao, Q., Qin, H., Zhang, Y.D., and Tan, L. (2014). Temsirolimus attenuates tauopathy in vitro and in vivo by targeting tau hyperphosphorylation and autophagic clearance. Neuropharmacology 85, 121–130.

Jiang, W., Wei, W., Gaertig, M.A., and Li, X.J. (2015). Therapeutic effect of barberine on Huntington’s disease transgenic mouse model. PLoS One. 10, e0134142.

Joo, J.H., Wang, B., Frankel, E., Ge, L., Xu, L., Iyengar, R., Li-Harms, X., Wright, C., Shaw, T.I., Lindsten, T., et al. (2016). The Noncanonical Role of ULK/ATG1 in ER-to-Golgi Trafficking Is Essential for Cellular Homeostasis. Mol. Cell 62, 491–506.

Jun, G., Naj, A.C., Beecham, G.W., Wang, L.S., Buros, J., Gallins, P.J., Buxbaum, J.D., Ertekin-Taner, N., Fallin, M.D., Friedland, R., et al.; Alzheimer’s Disease Genetics Consortium (2010). Meta-analysis confirms CR1, CLU, and PICALM as alzheimer disease risk loci and reveals interactions with APOE genotypes. Arch. Neurol. 67, 1473–1484.

Jung, C.H., Jun, C.B., Ro, S.H., Kim, Y.M., Otto, N.M., Cao, J., Kundu, M., and Kim, D.H. (2009). ULK-Atg13-FIP200 complexes mediate mTOR signaling to the autophagy machinery. Mol. Biol. Cell 20, 1992–2003.

Kaneb, H.M., Sharp, P.S., Rahmani-Kondori, N., and Wells, D.J. (2011). Metformin treatment has no beneficial effect in a dose-response survival study in the SOD1(G93A) mouse model of ALS and is harmful in female mice. PLoS ONE 6, e24189.

Kaushik, S., Rodriguez-Navarro, J.A., Arias, E., Kiffin, R., Sahu, S., Schwartz, G.J., Cuervo, A.M., and Singh, R. (2011). Autophagy in hypothalamic AgRP neurons regulates food intake and energy balance. Cell Metab. 14, 173–183.

Kenzelmann Broz, D., and Attardi, L.D. (2013). TRP53 activates a global autophagy program to promote tumor suppression. Autophagy 9, 1440–1442.

Khaminets, A., Behl, C., and Dikic, I. (2016). Ubiquitin-Dependent And Independent Signals In Selective Autophagy. Trends Cell Biol. 26, 6–16.

Khandelwal, P.J., Herman, A.M., Hoe, H.S., Rebeck, G.W., and Moussa, C.E. (2011). Parkin mediates beclin-dependent autophagic clearance of defective mitochondria and ubiquitinated Abeta in AD models. Hum. Mol. Genet. 20, 2091–2102.

Kim, Y.C., and Guan, K.L. (2015). mTOR: a pharmacologic target for autophagy regulation. J. Clin. Invest. 125, 25–32.

Kim, J., Kundu, M., Viollet, B., and Guan, K.L. (2011). AMPK and mTOR regulate autophagy through direct phosphorylation of Ulk1. Nat. Cell Biol. 13, 132–141.

Kim, M., Sandford, E., Gatica, D., Qiu, Y., Liu, X., Zheng, Y., Schulman, B.A., Xu, J., Semple, I., Ro, S.H., et al. (2016). Mutation in ATG5 reduces autophagy and leads to ataxia with developmental delay. eLife 5.

Kimura, T., Jain, A., Choi, S.W., Mandell, M.A., Schroder, K., Johansen, T., and Deretic, V. (2015). TRIM-mediated precision autophagy targets cytoplasmic regulators of innate immunity. J. Cell Biol. 210, 973–989.

Kimura, T., Mandell, M., and Deretic, V. (2016). Precision autophagy directed by receptor regulators - emerging examples within the TRIM family. J. Cell Sci. 129, 881–891.

Kirkin, V., Lamark, T., Sou, Y.S., Bjørkøy, G., Nunn, J.L., Bruun, J.A., Shvets, E., McEwan, D.G., Clausen, T.H., Wild, P., et al. (2009). A role for NBR1 in autophagosomal degradation of ubiquitinated substrates. Mol. Cell 33, 505–516.

Kitada, T., Asakawa, S., Hattori, N., Matsumine, H., Yamamura, Y., Minoshima, S., Yokochi, M., Mizuno, Y., and Shimizu, N. (1998). Mutations in the parkin gene cause autosomal recessive juvenile parkinsonism. Nature 392, 605–608.

Kitada, T., Pisani, A., Karouani, M., Haburcak, M., Martella, G., Tscherter, A., Pla- tania, P., Wu, B., Pothos, E.N., and Shen, J. (2009). Impaired dopamine release and synaptic plasticity in the striatum of parkin-/- mice. J. Neurochem. 110, 613–621.

Knævelsrud, H., Søreng, K., Raiborg, C., Håberg, K., Rasmuson, F., Brech, A., Liestøl, K., Rusten, T.E., Stenmark, H., Neufeld, T.P., et al. (2013). Membrane remodeling by the PX-BAR protein SNX18 promotes autophagosome formation. J. Cell Biol. 202, 331–349.

Knorr, R.L., Lipowsky, R., and Dimova, R. (2015). Autophagosome closure requires membrane scission. Autophagy 11, 2134–2137.

Komatsu, M., Waguri, S., Chiba, T., Murata, S., Iwata, J., Tanida, I., Ueno, T., Koike, M., Uchiyama, Y., Kominami, E., and Tanaka, K. (2006). Loss of autophagy in the central nervous system causes neurodegeneration in mice. Nature 441, 880–884.

Komatsu, M., Waguri, S., Koike, M., Sou, Y.S., Ueno, T., Hara, T., Mizushima, N., Iwata, J., Ezaki, J., Murata, S., et al. (2007a). Homeostatic levels of p62 control cytoplasmic inclusion body formation in autophagy-deficient mice. Cell 131, 1149–1163.

Komatsu, M., Wang, Q.J., Holstein, G.R., Friedrich, V.L., Jr., Iwata, J., Kominami, E., Chait, B.T., Tanaka, K., and Yue, Z. (2007b). Essential role for autophagy protein Atg7 in the maintenance of axonal homeostasis and the prevention of axonal degeneration. Proc. Natl. Acad. Sci. USA 104, 14489–14494.

Kyttälä, A., Yliannala, K., Schu, P., Jalanko, A., and Luzio, J.P. (2005). AP-1 and AP-3 facilitate lysosomal targeting of Batten disease protein CLN3 via its dileucine motif. J. Biol. Chem. 280, 10277–10283.

Lamb, C.A., Yoshimori, T., and Tooze, S.A. (2013). The autophagosome: origins unknown, biogenesis complex. Nat. Rev. Mol. Cell Biol. 14, 759–774.

Lamb, C.A., Nühlen, S., Judith, D., Frith, D., Snijders, A.P., Behrends, C., and Tooze, S.A. (2016). TBC1D14 regulates autophagy via the TRAPP complex and ATG9 traffic. EMBO J. 35, 281–301.

Laplante, M., and Sabatini, D.M. (2012). mTOR signaling in growth control and disease. Cell 149, 274–293.

Lebwohl, D., Anak, O., Sahmoud, T., Klimovsky, J., Elmroth, I., Haas, T., Pos-luszny, J., Saletan, S., and Berg, W. (2013). Development of everolimus, a novel oral mTOR inhibitor, across a spectrum of diseases. Ann. N Y Acad. Sci. 1291, 14–32.

Lee, V.M., Goedert, M., and Trojanowski, J.Q. (2001). Neurodegenerative tauopathies. Annu. Rev. Neurosci. 24, 1121–1159.

Lee, J.A., Beigneux, A., Ahmad, S.T., Young, S.G., and Gao, F.B. (2007). ESCRT-III dysfunction causes autophagosome accumulation and neurodegeneration. Curr. Biol. 17, 1561–1567.

Lee, I.H., Cao, L., Mostoslavsky, R., Lombard, D.B., Liu, J., Bruns, N.E., Tso-kos, M., Alt, F.W., and Finkel, T. (2008). A role for the NAD-dependent deacetylase Sirt1 in the regulation of autophagy. Proc. Natl. Acad. Sci. USA 105, 3374–3379.

Lee, J.H., Yu, W.H., Kumar, A., Lee, S., Mohan, P.S., Peterhoff, C.M., Wolfe, D.M., Martinez-Vicente, M., Massey, A.C., Sovak, G., et al. (2010). Lysosomal proteolysis and autophagy require presenilin 1 and are disrupted by Alzheimer-related PS1 mutations. Cell 141, 1146–1158.

Levine, B., Packer, M., and Codogno, P. (2015). Development of autophagy inducers in clinical medicine. J. Clin. Invest. 125, 14–24.

Levy, J.R., Sumner, C.J., Caviston, J.P., Tokito, M.K., Ranganathan, S., Ligon, L.A., Wallace, K.E., LaMonte, B.H., Harrison, G.G., Puls, I., et al. (2006). A motor neuron disease-associated mutation in p150Glued perturbs dynactin function and induces protein aggregation. J. Cell Biol. 172, 733–745.

Li, Y., Guo, Y., Wang, X., Yu, X., Duan, W., Hong, K., Wang, J., Han, H., and Li, C. (2015). Trehalose decreases mutant SOD1 expression and alleviates motor deficiency in early but not end-stage amyotrophic lateral sclerosis in a SOD1-G93A mouse model. Neuroscience 298, 12–25.

Liang, C.C., Wang, C., Peng, X., Gan, B., and Guan, J.L. (2010). Neural-specific deletion of FLIP200 leads to cerebellar degeneration caused by increased neuronal death and axon degeneration. J. Biol. Chem. 285, 3499–3509.

Lloyd-Evans, E., and Platt, F.M. (2010). Lipids on trial: the search for the offending metabolite in Niemann-Pick type C disease. Traffic 11, 419–428.

Lloyd-Evans, E., Morgan, A.J., He, X., Smith, D.A., Elliot-Smith, E., Silcence, D.J., Churchill, G.C., Schuchman, E.H., Galione, A., and Platt, F.M. (2008). Niemann-Pick disease type C1 is a sphingosine storage disease that causes deregulation of lysosomal calcium. Nat. Med. 14, 1247–1255.

Longatti, A., Lamb, C.A., Razi, M., Yoshimura, S., Barr, F.A., and Tooze, S.A. (2012). TBC1D14 regulates autophagosome formation via Rab11- and ULK1-positive recycling endosomes. J. Cell Biol. 197, 659–675.

Lonskaya, I., Hebron, M.L., Selby, S.T., Turner, R.S., and Moussa, C.E. (2015). Nilotinib and bosutinib modulate pre-plaque alterations of blood immune markers and neuro-inflammation in Alzheimer’s disease models. Neurosci- ence 304, 316–327.

Lu, K., Psakhye, I., and Jentsch, S. (2014). Autophagic clearance of polyQ proteins mediated by ubiquitin-Atg8 adaptors of the conserved CUET protein family. Cell 158, 549–563.

Lv, X., Jiang, H., Li, B., Liang, Q., Wang, S., Zhao, Q., and Jiao, J. (2014). The crucial role of Atg5 in cortical neurogenesis during early brain development. Sci. Rep. 4, 6010.

Ma, T.C., Buescher, J.L., Oatis, B., Funk, J.A., Nash, A.J., Carrier, R.L., and Hoyt, K.R. (2007). Metformin therapy in a transgenic mouse model of Huntington’s disease. Neurosci. Lett. 411, 98–103.

Majid, T., Ali, Y.O., Venkitaramani, D.V., Jang, M.K., Lu, H.C., and Pautler, R.G. (2014). In vivo axonal transport deficits in a mouse model of fronto-temporal dementia. Neuroimage Clin. 4, 711–717.

Mammucari, C., Milan, G., Romanello, V., Masiero, E., Rudolf, R., Del Piccolo, P., Burden, S.J., Di Lisi, R., Sandri, C., Zhao, J., et al. (2007). FoxO3 controls autophagy in skeletal muscle in vivo. Cell Metab. 6, 458–471.

Marshall, R.S., Li, F., Gemperline, D.C., Book, A.J., and Vierstra, R.D. (2015). Autophagic Degradation of the 26S Proteasome Is Mediated by the Dual ATG8/Ubiquitin Receptor RPN10 in Arabidopsis. Mol. Cell 58, 1053–1066.

Martina, J.A., Diab, H.I., Li, H., and Puertollano, R. (2014). Novel roles for the MiTF/TFE family of transcription factors in organelle biogenesis, nutrient sensing, and energy homeostasis. Cell. Mol. Life Sci. 71, 2483–2497.

Martinez-Vicente, M., Talloczy, Z., Wong, E., Tang, G., Koga, H., Kaushik, S., de Vries, R., Arias, E., Harris, S., Sulzer, D., and Cuervo, A.M. (2010). Cargo recognition failure is responsible for inefficient autophagy in Huntington’s disease. Nat. Neurosci. 13, 567–576.

Maruyama, H., Morino, H., Ito, H., Izumi, Y., Kato, H., Watanabe, Y., Kinoshita, Y., Kamada, M., Nodera, H., Suzuki, H., et al. (2010). Mutations of optineurin in amyotrophic lateral sclerosis. Nature 465, 223–226.

Matsuda, N., Sato, S., Shiba, K., Okatsu, K., Saisho, K., Gautier, C.A., Sou, Y.S., Saiki, S., Kawajiri, S., Sato, F., et al. (2010). PINK1 stabilized by mitochondrial depolarization recruits Parkin to damaged mitochondria and activates latent Parkin for mitophagy. J. Cell Biol. 189, 211–221.

Mazzulli, J.R., Xu, Y.H., Sun, Y., Knight, A.L., McLean, P.J., Caldwell, G.A., Sidransky, E., Grabowski, G.A., and Krainc, D. (2011). Gaucher disease glucocerebrosidase and α-synuclein form a bidirectional pathogenic loop in synucleinopathies. Cell 146, 37–52.

Mealer, R.G., Murray, A.J., Shahani, N., Subramaniam, S., and Snyder, S.H. (2014). Rhes, a striatal-selective protein implicated in Huntington disease, binds beclin-1 and activates autophagy. J. Biol. Chem. 289, 3547–3554.

Melis, V., Magbagbeolu, M., Rickard, J.E., Horsley, D., Davidson, K., Harrington, K.A., Goatman, K., Goatman, E.A., Deiana, S., Close, S.P., et al. (2015). Effects of oxidized and reduced forms of methylthioninium in two transgenic mouse tauopathy models. Behav. Pharmacol. 26, 353–368.

Menzies, F.M., Huebener, J., Renna, M., Bonin, M., Riess, O., and Rubinsztein, D.C. (2010). Autophagy induction reduces mutant ataxin-3 levels and toxicity in a mouse model of spinocerebellar ataxia type 3. Brain 133, 93–104.

Menzies, F.M., Fleming, A., and Rubinsztein, D.C. (2015a). Compromised autophagy and neurodegenerative diseases. Nat. Rev. Neurosci. 16, 345–357.

Menzies, F.M., Garcia-Arencibia, M., Imarisio, S., O’Sullivan, N.C., Ricketts, T., Kent, B.A., Rao, M.V., Lam, W., Green-Thompson, Z.W., Nixon, R.A., et al. (2015b). Calpain inhibition mediates autophagy-dependent protection against polyglutamine toxicity. Cell Death Differ. 22, 433–444.

Michel, P.P., Hirsch, E.C., and Hunot, S. (2016). Understanding Dopaminergic Cell Death Pathways in Parkinson Disease. Neuron 90, 675–691.

Moore, A.S., and Holzbaur, E.L. (2016). Dynamic recruitment and activation of ALS-associated TBK1 with its target optineurin are required for efficient mitophagy. Proc. Natl. Acad. Sci. USA 113, E3349–E3358.

Moreau, K., Ravikumar, B., Renna, M., Puri, C., and Rubinsztein, D.C. (2011). Autophagosome precursor maturation requires homotypic fusion. Cell 146, 303–317.

Moreau, K., Fleming, A., Imarisio, S., Lopez Ramirez, A., Mercer, J.L., Jimenez-Sanchez, M., Bento, C.F., Puri, C., Zavodszky, E., Siddiqi, F., et al. (2014). PICALM modulates autophagy activity and tau accumulation. Nat. Commun. 5, 4998.

Morimoto, S., Martin, B.M., Yamamoto, Y., Kretz, K.A., O’Brien, J.S., and Kishimoto, Y. (1989). Saposin A: second cerebrosidase activator protein. Proc. Natl. Acad. Sci. USA 86, 3389–3393.

Murphy, K.E., Gysbers, A.M., Abbott, S.K., Tayebi, N., Kim, W.S., Sidransky, E., Cooper, A., Garner, B., and Halliday, G.M. (2014). Reduced glucocerebrosidase is associated with increased α-synuclein in sporadic Parkinson’s disease. Brain 137, 834–848.

Narendra, D., Tanaka, A., Suen, D.F., and Youle, R.J. (2008). Parkin is recruited selectively to impaired mitochondria and promotes their autophagy. J. Cell Biol. 183, 795–803.

Nascimento-Ferreira, I., Santos-Ferreira, T., Sousa-Ferreira, L., Auregan, G., Onofre, I., Alves, S., Dufour, N., Colomer Gould, V.F., Koeppen, A., Déglon, N., and Pereira de Almeida, L. (2011). Overexpression of the autophagic

beclin-1 protein clears mutant ataxin-3 and alleviates Machado-Joseph disease. Brain 134, 1400–1415.

Newman, A.C., Scholefield, C.L., Kemp, A.J., Newman, M., McIver, E.G., Kalam, A., and Wilkinson, S. (2012). TBK1 kinase addiction in lung cancer cells is mediated via autophagy of Tax1bp1/Ndp52 and non-canonical NF-κB signaling. PLoS ONE 7, e50672.

Nguyen, T.N., Padman, B.S., and Lazarou, M. (2016). Deciphering the Molecular Signals of PINK1/Parkin Mitophagy. Trends Cell Biol. 26, 733–744.

Nilsson, P., Loganathan, K., Sekiguchi, M., Matsuba, Y., Hui, K., Tsubuki, S., Tanaka, M., Iwata, N., Saito, T., and Saito, T.C. (2013). Aβ secretion and plaque formation depend on autophagy. Cell Rep. 5, 61–69.

Nishiyama, J., Miura, E., Mizushima, N., Watanabe, M., and Yuzaki, M. (2007). Aberrant membranes and double-membrane structures accumulate in the axons of Atg5-null Purkinje cells before neuronal death. Autophagy 3, 591–596.

Ochaba, J., Lukacsovich, T., Csikos, G., Zheng, S., Margulis, J., Salazar, L., Mao, K., Lau, A.L., Yeung, S.Y., Humbert, S., et al. (2014). Potential function for the Huntingtin protein as a scaffold for selective autophagy. Proc. Natl. Acad. Sci. USA 111, 16889–16894.

Orosco, L.A., Ross, A.P., Cates, S.L., Scott, S.E., Wu, D., Sohn, J., Pleasure, D., Pleasure, S.J., Adamopoulos, I.E., and Zarbalis, K.S. (2014). Loss of Wdfy3 in mice alters cerebral cortical neurogenesis reflecting aspects of the autism pathology. Nat. Commun. 5, 4692.

Orsi, A., Razi, M., Dooley, H.C., Robinson, D., Weston, A.E., Collinson, L.M., and Tooze, S.A. (2012). Dynamic and transient interactions of Atg9 with autophagosomes, but not membrane integration, are required for autophagy. Mol. Biol. Cell 23, 1860–1873.

Otomo, A., Kunita, R., Suzuki-Utsunomiya, K., Ikeda, J.E., and Hadano, S. (2011). Defective relocation of ALS2/alsin missense mutants to Rac1-induced macropinosomes accounts for loss of their cellular function and leads to disturbed amphisome formation. FEBS Lett. 585, 730–736.

Oz-Levi, D., Ben-Zeev, B., Ruzzo, E.K., Hitomi, Y., Gelman, A., Pelak, K., Anikster, Y., Reznik-Wolf, H., Bar-Joseph, I., Olender, T., et al. (2012). Mutation in TECPR2 reveals a role for autophagy in hereditary spastic paraparesis. Am. J. Hum. Genet. 91, 1065–1072.

Ozcnelik, S., Fraser, G., Castets, P., Schaeffer, V., Skachokova, Z., Breu, K., Clavaguera, F., Sinneich, M., Kappos, L., Goedert, M., et al. (2013). Rapamycin attenuates the progression of tau pathology in P301S tau transgenic mice. PLoS ONE 8, e62459.

Pacheco, C.D., and Lieberman, A.P. (2008). The pathogenesis of Niemann-Pick type C disease: a role for autophagy? Expert Rev. Mol. Med. 10, e26.

Palacino, J.J., Sagi, D., Goldberg, M.S., Krauss, S., Motz, C., Wacker, M., Klose, J., and Shen, J. (2004). Mitochondrial dysfunction and oxidative damage in parkin-deficient mice. J. Biol. Chem. 279, 18614–18622.

Palikaras, K., Lionaki, E., and Tavernarakis, N. (2015). Coordination of mitophagy and mitochondrial biogenesis during ageing in C. elegans. Nature 521, 525–528.

Pankiv, S., Clausen, T.H., Lamark, T., Brech, A., Bruun, J.A., Outzen, H., Øvervatn, A., Bjørkøy, G., and Johansen, T. (2007). p62/SQSTM1 binds directly to Atg8/LC3 to facilitate degradation of ubiquitinated protein aggregates by autophagy. J. Biol. Chem. 282, 24131–24145.

Perez, F.A., and Palmiter, R.D. (2005). Parkin-deficient mice are not a robust model of parkinsonism. Proc. Natl. Acad. Sci. USA 102, 2174–2179.

Perez, S.E., He, B., Nadeem, M., Wuu, J., Ginsberg, S.D., Ikonomovic, M.D., and Mufson, E.J. (2015). Hippocampal endosomal, lysosomal, and autophagic dysregulation in mild cognitive impairment: correlation with aβ and tau pathology. J. Neuropathol. Exp. Neurol. 74, 345–358.

Pickford, F., Masliah, E., Britschgi, M., Lucin, K., Narasimhan, R., Jaeger, P.A., Small, S., Spencer, B., Rockenstein, E., Levine, B., and Wyss-Coray, T. (2008). The autophagy-related protein beclin 1 shows reduced expression in early Alzheimer disease and regulates amyloid beta accumulation in mice. J. Clin. Invest. 118, 2190–2199.

Pickrell, A.M., and Youle, R.J. (2015). The roles of PINK1, parkin, and mitochondrial fidelity in Parkinson’s disease. Neuron 85, 257–273.

Pickrell, A.M., Huang, C.H., Kennedy, S.R., Ordureau, A., Sideris, D.P., Hoekstra, J.G., Harper, J.W., and Youle, R.J. (2015). Endogenous Parkin Preserves Dopaminergic Substantia Nigral Neurons following Mitochondrial DNA Mutagenic Stress. Neuron 87, 371–381.

Pilli, M., Arko-Mensah, J., Ponpuak, M., Roberts, E., Master, S., Mandell, M.A., Dupont, N., Ornatowski, W., Jiang, S., Bradfute, S.B., et al. (2012). TBK-1 promotes autophagy-mediated antimicrobial defense by controlling autophagosome maturation. Immunity 37, 223–234.

Piras, A., Collin, L., Grüninger, F., Graff, C., and Rönnbäck, A. (2016). Autophagic and lysosomal defects in human tauopathies: analysis of post-mortem brain from patients with familial Alzheimer disease, corticobasal degeneration and progressive supranuclear palsy. Acta Neuropathol. Commun. 4, 22.

Platt, F.M., Boland, B., and van der Spoel, A.C. (2012). The cell biology of disease: lysosomal storage disorders: the cellular impact of lysosomal dysfunction. J. Cell Biol. 199, 723–734.

Polito, V.A., Li, H., Martini-Stoica, H., Wang, B., Yang, L., Xu, Y., Swartzlander, D.B., Palmieri, M., di Ronza, A., Lee, V.M., et al. (2014). Selective clearance of aberrant tau proteins and rescue of neurotoxicity by transcription factor EB. EMBO Mol. Med. 6, 1142–1160.

Poupetová, H., Ledvinová, J., Berná, L., Dvoráková, L., Kozich, V., and Elleder, M. (2010). The birth prevalence of lysosomal storage disorders in the Czech Republic: comparison with data in different populations. J. Inherit. Metab. Dis. 33, 387–396.

Proikas-Cezanne, T., Waddell, S., Gaugel, A., Frickey, T., Lupas, A., and Nordheim, A. (2004). WIPI-1alpha (WIPI49), a member of the novel 7-bladed WIPI protein family, is aberrantly expressed in human cancer and is linked to starvation-induced autophagy. Oncogene 23, 9314–9325.

Puls, I., Oh, S.J., Sumner, C.J., Wallace, K.E., Floeter, M.K., Mann, E.A., Kennedy, W.R., Wendelschafer-Crabb, G., Vortmeyer, A., Powers, R., et al. (2005). Distal spinal and bulbar muscular atrophy caused by dynactin mutation. Ann. Neurol. 57, 687–694.

Punetha, J., Monges, S., Franchi, M.E., Hoffman, E.P., Cirak, S., and Tesi-Rocha, C. (2015). Exome Sequencing Identifies DYNC1H1 Variant Associated With Vertebral Abnormality and Spinal Muscular Atrophy With Lower Extremity Predominance. Pediatr. Neurol. 52, 239–244.

Puri, C., Renna, M., Bento, C.F., Moreau, K., and Rubinsztein, D.C. (2013). Diverse autophagosomal membrane sources coalesce in recycling endosomes. Cell 154, 1285–1299.

Pyo, J.O., Yoo, S.M., Ahn, H.H., Nah, J., Hong, S.H., Kam, T.I., Jung, S., and Jung, Y.K. (2013). Overexpression of Atg5 in mice activates autophagy and extends lifespan. Nat. Commun. 4, 2300.

Ramirez, A., Heimbach, A., Gründemann, J., Stiller, B., Hampshire, D., Cid, L.P., Goebel, I., Mubaidin, A.F., Wriekat, A.L., Roeper, J., et al. (2006). Hereditary parkinsonism with dementia is caused by mutations in ATP13A2, encoding a lysosomal type 5 P-type ATPase. Nat. Genet. 38, 1184–1191.

Ravikumar, B., Duden, R., and Rubinsztein, D.C. (2002). Aggregate-prone proteins with polyglutamine and polyalanine expansions are degraded by autophagy. Hum. Mol. Genet. 11, 1107–1117.

Ravikumar, B., Vacher, C., Berger, Z., Davies, J.E., Luo, S., Oroz, L.G., Scaravilli, F., Easton, D.F., Duden, R., O’Kane, C.J., and Rubinsztein, D.C. (2004). Inhibition of mTOR induces autophagy and reduces toxicity of polyglutamine expansions in fly and mouse models of Huntington disease. Nat. Genet. 36, 585–595.

Ravikumar, B., Acevedo-Arozena, A., Imarisio, S., Berger, Z., Vacher, C., O’Kane, C.J., Brown, S.D., and Rubinsztein, D.C. (2005). Dynein mutations impair autophagic clearance of aggregate-prone proteins. Nat. Genet. 37, 771–776.

Ravikumar, B., Berger, Z., Vacher, C., O’Kane, C.J., and Rubinsztein, D.C. (2006). Rapamycin pre-treatment protects against apoptosis. Hum. Mol. Genet. 15, 1209–1216.

Ravikumar, B., Imarisio, S., Sarkar, S., O’Kane, C.J., and Rubinsztein, D.C. (2008). Rab5 modulates aggregation and toxicity of mutant huntingtin through macroautophagy in cell and fly models of Huntington disease. J. Cell Sci. 121, 1649–1660.

Ravikumar, B., Moreau, K., Jahreiss, L., Puri, C., and Rubinsztein, D.C. (2010). Plasma membrane contributes to the formation of pre-autophagosomal structures. Nat. Cell Biol. 12, 747–757.

Renna, M., Bento, C.F., Fleming, A., Menzies, F.M., Siddiqi, F.H., Ravikumar, B., Puri, C., Garcia-Arencibia, M., Sadiq, O., Corrochano, S., et al. (2013). IGF-1 receptor antagonism inhibits autophagy. Hum. Mol. Genet. 22, 4528–4544.

Renton, A.E., Majounie, E., Waite, A., Simón-Sánchez, J., Rollinson, S., Gibbs, J.R., Schymick, J.C., Laaksovirta, H., van Swieten, J.C., Myllykangas, L., et al.; ITALSGEN Consortium (2011). A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD. Neuron 72, 257–268.

Renvoisé, B., Chang, J., Singh, R., Yonekawa, S., FitzGibbon, E.J., Mankodi, A., Vanderver, A., Schindler, A., Toro, C., Gahl, W.A., et al. (2014). Lysosomal abnormalities in hereditary spastic paraplegia types SPG15 and SPG11. Ann. Clin. Transl. Neurol. 1, 379–389.

Richter, B., Sliter, D.A., Herhaus, L., Stolz, A., Wang, C., Beli, P., Zaffagnini, G., Wild, P., Martens, S., Wagner, S.A., et al. (2016). Phosphorylation of OPTN by TBK1 enhances its binding to Ub chains and promotes selective autophagy of damaged mitochondria. Proc. Natl. Acad. Sci. USA 113, 4039–4044.

Rocznik-Ferguson, A., Petit, C.S., Froehlich, F., Qian, S., Ky, J., Angarola, B., Walther, T.C., and Ferguson, S.M. (2012). The transcription factor TFEB links mTORC1 signaling to transcriptional control of lysosome homeostasis. Sci. Signal. 5, ra42.

Rodríguez-Navarro, J.A., Rodríguez, L., Casarejos, M.J., Solano, R.M., Gómez, A., Peruchio, J., Cuervo, A.M., García de Yébenes, J., and Mena, M.A. (2010). Trehalose ameliorates dopaminergic and tau pathology in parkin deleted/tau overexpressing mice through autophagy activation. Neurobiol. Dis. 39, 423–438.

Rohn, T.T., Wirawan, E., Brown, R.J., Harris, J.R., Masliah, E., and Vandena-beele, P. (2011). Depletion of Beclin-1 due to proteolytic cleavage by caspases in the Alzheimer’s disease brain. Neurobiol. Dis. 43, 68–78.

Rose, C., Menzies, F.M., Renna, M., Acevedo-Arozena, A., Corrochano, S., Sadiq, O., Brown, S.D., and Rubinsztein, D.C. (2010). Rilmenidine attenuates toxicity of polyglutamine expansions in a mouse model of Huntington’s disease. Hum. Mol. Genet. 19, 2144–2153.

Rubinsztein, D.C., Codogno, P., and Levine, B. (2012). Autophagy modulation as a potential therapeutic target for diverse diseases. Nat. Rev. Drug Discov. 11, 709–730.

Rubinsztein, D.C., Bento, C.F., and Deretic, V. (2015). Therapeutic targeting of autophagy in neurodegenerative and infectious diseases. J. Exp. Med. 212, 979–990.

Rui, Y.N., Xu, Z., Patel, B., Chen, Z., Chen, D., Tito, A., David, G., Sun, Y., Stimming, E.F., Bellen, H.J., et al. (2015). Huntingtin functions as a scaffold for selective macroautophagy. Nat. Cell Biol. 17, 262–275.

Saitsu, H., Nishimura, T., Muramatsu, K., Kodera, H., Kumada, S., Sugai, K., Kasai-Yoshida, E., Sawaura, N., Nishida, H., Hoshino, A., et al. (2013). De novo mutations in the autophagy gene WDR45 cause static encephalopathy of childhood with neurodegeneration in adulthood. Nat. Genet. 45, 445–449.

Sardiello, M., Palmieri, M., di Ronza, A., Medina, D.L., Valenza, M., Gennarino, V.A., Di Malta, C., Donaudy, F., Embrione, V., Polishchuk, R.S., et al. (2009). A gene network regulating lysosomal biogenesis and function. Science 325, 473–477.

Sarkar, S., Davies, J.E., Huang, Z., Tunnacliffe, A., and Rubinsztein, D.C. (2007). Trehalose, a novel mTOR-independent autophagy enhancer, accelerates the clearance of mutant huntingtin and alpha-synuclein. J. Biol. Chem. 282, 5641–5652.

Sarkar, S., Krishna, G., Imarisio, S., Saiki, S., O’Kane, C.J., and Rubinsztein, D.C. (2008). A rational mechanism for combination treatment of Huntington’s disease using lithium and rapamycin. Hum. Mol. Genet. 17, 170–178.

Schaeffer, V., and Goedert, M. (2012). Stimulation of autophagy is neuroprotective in a mouse model of human tauopathy. Autophagy 8, 1686–1687.

Schaeffer, V., Lavenir, I., Ozcelik, S., Tolnay, M., Winkler, D.T., and Goedert, M. (2012). Stimulation of autophagy reduces neurodegeneration in a mouse model of human tauopathy. Brain 135, 2169–2177.

Sellier, C., Campanari, M.L., Julie Corbier, C., Gaucherot, A., Kolb-Cheynel, I., Oulad-Abdelghani, M., Ruffenach, F., Page, A., Ciura, S., Kabashi, E., and Charlet-Berguerand, N. (2016). Loss of C9ORF72 impairs autophagy and synergizes with polyQ Ataxin-2 to induce motor neuron dysfunction and cell death. EMBO J. 35, 1276–1297.

Serrano-Puebla, A., and Boya, P. (2016). Lysosomal membrane permeabilization in cell death: new evidence and implications for health and disease. Ann. N Y Acad. Sci. 1371, 30–44.

Seto, S., Sugaya, K., Tsujimura, K., Nagata, T., Horii, T., and Koide, Y. (2013). Rab39a interacts with phosphatidylinositol 3-kinase and negatively regulates autophagy induced by lipopolysaccharide stimulation in macrophages. PLoS ONE 8, e83324.

Settembre, C., Fraldi, A., Jahreiss, L., Spampanato, C., Venturi, C., Medina, D., de Pablo, R., Tacchetti, C., Rubinsztein, D.C., and Ballabio, A. (2008). A block of autophagy in lysosomal storage disorders. Hum. Mol. Genet. 17, 119–129.

Settembre, C., Di Malta, C., Polito, V.A., Garcia Arencibia, M., Vetrini, F., Erdin, S., Erdin, S.U., Huynh, T., Medina, D., Colella, P., et al. (2011). TFEB links autophagy to lysosomal biogenesis. Science 332, 1429–1433.

Settembre, C., Zoncu, R., Medina, D.L., Vetrini, F., Erdin, S., Erdin, S., Huynh, T., Ferron, M., Karsenty, G., Vellard, M.C., et al. (2012). A lysosome-to-nucleus signalling mechanism senses and regulates the lysosome via mTOR and TFEB. EMBO J. 31, 1095–1108.

Shen, W.C., Li, H.Y., Chen, G.C., Chern, Y., and Tu, P.H. (2015). Mutations in the ubiquitin-binding domain of OPTN/optineurin interfere with autophagy-mediated degradation of misfolded proteins by a dominant-negative mechanism. Autophagy 11, 685–700.

Shibata, M., Lu, T., Furuya, T., Degterev, A., Mizushima, N., Yoshimori, T., MacDonald, M., Yankner, B., and Yuan, J. (2006). Regulation of intracellular accumulation of mutant Huntingtin by Beclin 1. J. Biol. Chem. 281, 14474–14485.

Shim, J.H., Yoon, S.H., Kim, K.H., Han, J.Y., Ha, J.Y., Hyun, D.H., Paek, S.H., Kang, U.J., Zhuang, X., and Son, J.H. (2011). The antioxidant Trolox helps recovery from the familial Parkinson’s disease-specific mitochondrial deficits caused by PINK1- and DJ-1-deficiency in dopaminergic neuronal cells. Mitochondrion 11, 707–715.

Shin, S.Y., Kim, T.H., Wu, H., Choi, Y.H., and Kim, S.G. (2014). SIRT1 activation by methylene blue, a repurposed drug, leads to AMPK-mediated inhibition of steatosis and steatohepatitis. Eur. J. Pharmacol. 727, 115–124.

Shoji-Kawata, S., Sumpter, R., Leveno, M., Campbell, G.R., Zou, Z., Kinch, L., Wilkins, A.D., Sun, Q., Pallauf, K., MacDuff, D., et al. (2013). Identification of a candidate therapeutic autophagy-inducing peptide. Nature 494, 201–206.

Sidransky, E., and Lopez, G. (2012). The link between the GBA gene and parkinsonism. Lancet Neurol. 11, 986–998.

Simonsen, A., Cumming, R.C., Brech, A., Isakson, P., Schubert, D.R., and Finley, K.D. (2008). Promoting basal levels of autophagy in the nervous system enhances longevity and oxidative resistance in adult Drosophila. Autophagy 4, 176–184.

Small, S.A., Kent, K., Pierce, A., Leung, C., Kang, M.S., Okada, H., Honig, L., Vonsattel, J.P., and Kim, T.W. (2005). Model-guided microarray implicates the retromer complex in Alzheimer’s disease. Ann. Neurol. 58, 909–919.

Son, S.M., Shin, H.J., Byun, J., Kook, S.Y., Moon, M., Chang, Y.J., and Mook-Jung, I. (2016). Metformin Facilitates Amyloid-β Generation by β- and γ-Secretases via Autophagy Activation. J. Alzheimers Dis. 51, 1197–1208.

Spencer, B., Potkar, R., Trejo, M., Rockenstein, E., Patrick, C., Gindi, R., Adame, A., Wyss-Coray, T., and Masliah, E. (2009). Beclin 1 gene transfer activates autophagy and ameliorates the neurodegenerative pathology in alpha-synuclein models of Parkinson’s and Lewy body diseases. J. Neurosci. 29, 13578–13588.

Spilman, P., Podlutskaya, N., Hart, M.J., Debnath, J., Gorostiza, O., Bredesen, D., Richardson, A., Strong, R., and Galvan, V. (2010). Inhibition of mTOR by rapamycin abolishes cognitive deficits and reduces amyloid-beta levels in a mouse model of Alzheimer’s disease. PLoS ONE 5, e9979.

Spinosa, M.R., Progida, C., De Luca, A., Colucci, A.M., Alifano, P., and Bucci, C. (2008). Functional characterization of Rab7 mutant proteins associated with Charcot-Marie-Tooth type 2B disease. J. Neurosci. 28, 1640–1648.

Staats, K.A., Hernandez, S., Schönefeldt, S., Bento-Abreu, A., Dooley, J., Van Damme, P., Liston, A., Robberecht, W., and Van Den Bosch, L. (2013). Rapamycin increases survival in ALS mice lacking mature lymphocytes. Mol. Neurodegener. 8, 31.

Stadel, D., Millarte, V., Tillmann, K.D., Huber, J., Tamin-Yecheskel, B.C., Akutsu, M., Demishtein, A., Ben-Zeev, B., Anikster, Y., Perez, F., et al. (2015). TECPR2 Cooperates with LC3C to Regulate COPII-Dependent ER Export. Mol. Cell 60, 89–104.

Stolz, A., Ernst, A., and Dikic, I. (2014). Cargo recognition and trafficking in selective autophagy. Nat. Cell Biol. 16, 495–501.

Sullivan, P.M., Zhou, X., Robins, A.M., Paushter, D.H., Kim, D., Smolkka, M.B., and Hu, F. (2016). The ALS/FTLD associated protein C9orf72 associates with SMCR8 and WDR41 to regulate the autophagy-lysosome pathway. Acta Neuropathol. Commun. 4, 51.

Sun, N., Yun, J., Liu, J., Malide, D., Liu, C., Rovira, I.I., Holmström, K.M., Ferguson, M.M., Yoo, Y.H., Combs, C.A., and Finkel, T. (2015). Measuring In Vivo Mitophagy. Mol. Cell 60, 685–696.

Sundaramoorthy, V., Walker, A.K., Tan, V., Fifita, J.A., Mccann, E.P., Williams, K.L., Blair, I.P., Guillemin, G.J., Farg, M.A., and Atkin, J.D. (2015). Defects in optineurin- and myosin VI-mediated cellular trafficking in amyotrophic lateral sclerosis. Hum. Mol. Genet. 24, 3830–3846.

Tanaka, M., Machida, Y., Niu, S., Ikeda, T., Jana, N.R., Doi, H., Kurosawa, M., Nekooki, M., and Nukina, N. (2004). Trehalose alleviates polyglutamine-mediated pathology in a mouse model of Huntington disease. Nat. Med. 10, 148–154.

Tanik, S.A., Schultheiss, C.E., Volpicelli-Daley, L.A., Brunden, K.R., and Lee, V.M. (2013). Lewy body-like α-synuclein aggregates resist degradation and impair macroautophagy. J. Biol. Chem. 288, 15194–15210.

Tanji, K., Miki, Y., Maruyama, A., Mimura, J., Matsumiya, T., Mori, F., Imaizumi, T., Itoh, K., and Wakabayashi, K. (2015). Trehalose intake induces chaperone molecules along with autophagy in a mouse model of Lewy body disease. Biochem. Biophys. Res. Commun. 465, 746–752.

Teyssou, E., Takeda, T., Lebon, V., Boillée, S., Doukouré, B., Bataillon, G., Sazdovitch, V., Cazeneuve, C., Meininger, V., LeGuern, E., et al. (2013). Mutations in SQSTM1 encoding p62 in amyotrophic lateral sclerosis: genetics and neuropathology. Acta Neuropathol. 125, 511–522.

Tian, Y., Bustos, V., Flajolet, M., and Greengard, P. (2011). A small-molecule enhancer of autophagy decreases levels of Abeta and APP-CTF via Atg5-dependent autophagy pathway. FASEB J. 25, 1934–1942.

Tian, Y., Chang, J.C., Fan, E.Y., Flajolet, M., and Greengard, P. (2013). Adaptor complex AP2/PICALM, through interaction with LC3, targets Alzheimer’s APP-CTF for terminal degradation via autophagy. Proc. Natl. Acad. Sci. USA 110, 17071–17076.

Tiede, S., Storch, S., Lübke, T., Henrissat, B., Bargal, R., Raas-Rothschild, A., and Braulke, T. (2005). Mucolipidosis II is caused by mutations in GNPTA encoding the alpha/beta GlcNAc-1-phosphotransferase. Nat. Med. 11, 1109–1112.

Todd, T.W., and Petrucelli, L. (2016). Insights into the pathogenic mechanisms of Chromosome 9 open reading frame 72 (C9orf72) repeat expansions. J. Neurochem. 138 (Suppl 1), 145–162.

Topp, J.D., Gray, N.W., Gerard, R.D., and Horazdovsky, B.F. (2004). Alsin is a Rab5 and Rac1 guanine nucleotide exchange factor. J. Biol. Chem. 279, 24612–24623.

Tsvetkov, A.S., Arrasate, M., Barnada, S., Ando, D.M., Sharma, P., Shaby, B.A., and Finkbeiner, S. (2013). Proteostasis of polyglutamine varies among neurons and predicts neurodegeneration. Nat. Chem. Biol. 9, 586–592.

Tumbarello, D.A., Waxse, B.J., Arden, S.D., Bright, N.A., Kendrick-Jones, J., and Buss, F. (2012). Autophagy receptors link myosin VI to autophagosomes to mediate Tom1-dependent autophagosome maturation and fusion with the lysosome. Nat. Cell Biol. 14, 1024–1035.

Ulgherait, M., Rana, A., Rera, M., Graniel, J., and Walker, D.W. (2014). AMPK modulates tissue and organismal aging in a non-cell-autonomous manner. Cell Rep. 8, 1767–1780.

Usenovic, M., Tresse, E., Mazzulli, J.R., Taylor, J.P., and Krainc, D. (2012). Deficiency of ATP13A2 leads to lysosomal dysfunction, α-synuclein accumulation, and neurotoxicity. J. Neurosci. 32, 4240–4246.

Valente, E.M., Abou-Sleiman, P.M., Caputo, V., Muqit, M.M., Harvey, K., Gispert, S., Ali, Z., Del Turco, D., Bentivoglio, A.R., Healy, D.G., et al. (2004). Hereditary early-onset Parkinson’s disease caused by mutations in PINK1. Science 304, 1158–1160.

Vantaggiato, C., Crimella, C., Airoldi, G., Polishchuk, R., Bonato, S., Brighina, E., Scarlato, M., Musumeci, O., Toscano, A., Martinuzzi, A., et al. (2013). Defective autophagy in spastizin mutated patients with hereditary spastic paraparesis type 15. Brain 136, 3119–3139.

Varga, R.E., Khundadze, M., Damme, M., Nietzsche, S., Hoffmann, B., Stauber, T., Koch, N., Hennings, J.C., Franzka, P., Huebner, A.K., et al. (2015). In Vivo Evidence for Lysosome Depletion and Impaired Autophagic Clearance in Hereditary Spastic Paraplegia Type SPG11. PLoS Genet. 11, e1005454.

Velikkakath, A.K., Nishimura, T., Oita, E., Ishihara, N., and Mizushima, N. (2012). Mammalian Atg2 proteins are essential for autophagosome formation and important for regulation of size and distribution of lipid droplets. Mol. Biol. Cell 23, 896–909.

Vingtdeux, V., Chandakkar, P., Zhao, H., d’Abramo, C., Davies, P., and Marambaud, P. (2011). Novel synthetic small-molecule activators of AMPK as enhancers of autophagy and amyloid-β peptide degradation. FASEB J. 25, 219–231.

Volpicelli-Daley, L.A., Gamble, K.L., Schultheiss, C.E., Riddle, D.M., West, A.B., and Lee, V.M. (2014). Formation of α-synuclein Lewy neurite-like aggregates in axons impedes the transport of distinct endosomes. Mol. Biol. Cell 25, 4010–4023.

Von Coelln, R., Thomas, B., Savitt, J.M., Lim, K.L., Sasaki, M., Hess, E.J., Dawson, V.L., and Dawson, T.M. (2004). Loss of locus coeruleus neurons and reduced startle in parkin null mice. Proc. Natl. Acad. Sci. USA 101, 10744–10749.

Wander, S.A., Hennessy, B.T., and Slingerland, J.M. (2011). Next-generation mTOR inhibitors in clinical oncology: how pathway complexity informs therapeutic strategy. J. Clin. Invest. 121, 1231–1241.

Wang, Y., Martinez-Vicente, M., Krüger, U., Kaushik, S., Wong, E., Mandelkow, E.M., Cuervo, A.M., and Mandelkow, E. (2009). Tau fragmentation, aggregation and clearance: the dual role of lysosomal processing. Hum. Mol. Genet. 18, 4153–4170.

Wang, C., Liang, C.C., Bian, Z.C., Zhu, Y., and Guan, J.L. (2013a). FIP200 is required for maintenance and differentiation of postnatal neural stem cells. Nat. Neurosci. 16, 532–542.

Wang, I.F., Tsai, K.J., and Shen, C.K. (2013b). Autophagy activation ameliorates neuronal pathogenesis of FTLD-U mice: a new light for treatment of TARDBP/TDP-43 proteinopathies. Autophagy 9, 239–240.

Webb, J.L., Ravikumar, B., Atkins, J., Skepper, J.N., and Rubinsztein, D.C. (2003). Alpha-Synuclein is degraded by both autophagy and the proteasome. J. Biol. Chem. 278, 25009–25013.

Webster, C.P., Smith, E.F., Bauer, C.S., Moller, A., Hautbergue, G.M., Ferriuolo, L., Myszczynska, M.A., Higginsbottom, A., Walsh, M.J., Whitworth, A.J., et al. (2016). The C9orf72 protein interacts with Rab1a and the ULK1 complex to regulate initiation of autophagy. EMBO J. 35, 1656–1676.

Weedon, M.N., Hastings, R., Caswell, R., Xie, W., Paszkiewicz, K., Antoniadi, T., Williams, M., King, C., Greenhalgh, L., Newbury-Ecob, R., and Ellard, S. (2011). Exome sequencing identifies a DYNC1H1 mutation in a large pedigree with dominant axonal Charcot-Marie-Tooth disease. Am. J. Hum. Genet. 89, 308–312.

West, R.J., Lu, Y., Marie, B., Gao, F.B., and Sweeney, S.T. (2015). Rab8, POSH, and TAK1 regulate synaptic growth in a Drosophila model of frontotemporal dementia. J. Cell Biol. 208, 931–947.

Wild, P., Farhan, H., McEwan, D.G., Wagner, S., Rogov, V.V., Brady, N.R., Richter, B., Korac, J., Waidmann, O., Choudhary, C., et al. (2011). Phosphorylation of the autophagy receptor optineurin restricts Salmonella growth. Science 333, 228–233.

Williams, A., Sarkar, S., Cuddon, P., Ttofi, E.K., Saiki, S., Siddiqi, F.H., Jahreiss, L., Fleming, A., Pask, D., Goldsmith, P., et al. (2008). Novel targets for Huntington’s disease in an mTOR-independent autophagy pathway. Nat. Chem. Biol. 4, 295–305.

Williams, K.L., Warrach, S.T., Yang, S., Solski, J.A., Fernando, R., Rouleau, G.A., Nicholson, G.A., and Blair, I.P. (2012). UBQLN2/ubiquilin 2 mutation and pathology in familial amyotrophic lateral sclerosis. Neurobiol. Aging 33, e2523–2510.

Winslow, A.R., Chen, C.W., Corrochano, S., Acevedo-Arozena, A., Gordon, D.E., Peden, A.A., Lichtenberg, M., Menzies, F.M., Ravikumar, B., Imarisio, S., et al. (2010). α-Synuclein impairs macroautophagy: implications for Parkinson’s disease. J. Cell Biol. 190, 1023–1037.

Wolfe, D.M., Lee, J.H., Kumar, A., Lee, S., Orenstein, S.J., and Nixon, R.A. (2013). Autophagy failure in Alzheimer’s disease and the role of defective lysosomal acidification. Eur. J. Neurosci. 37, 1949–1961.

Wong, Y.C., and Holzbaur, E.L. (2014a). The regulation of autophagosome dynamics by huntingtin and HAP1 is disrupted by expression of mutant huntingtin, leading to defective cargo degradation. J. Neurosci. 34, 1293–1305.

Wong, Y.C., and Holzbaur, E.L. (2014b). Optineurin is an autophagy receptor for damaged mitochondria in parkin-mediated mitophagy that is disrupted by an ALS-linked mutation. Proc. Natl. Acad. Sci. USA 111, E4439–E4448.

Wraith, J.E. (2002). Lysosomal disorders. Semin. Neonatol 7, 75–83.

Wu, X., Fleming, A., Ricketts, T., Pavel, M., Virgin, H., Menzies, F.M., and Rubinsztein, D.C. (2016). Autophagy regulates Notch degradation and modulates stem cell development and neurogenesis. Nat. Commun. 7, 10533.

Xi, Y., Dhaliwal, J.S., Ceizar, M., Vaculik, M., Kumar, K.L., and Lagace, D.C. (2016). Knockout of Atg5 delays the maturation and reduces the survival of adult-generated neurons in the hippocampus. Cell Death Dis. 7, e2127.

Xie, C., Zhou, X., Zhu, D., Liu, W., Wang, X., Yang, H., Li, Z., Hao, Y., Zhang, G.X., and Guan, Y. (2016). CNS involvement in CMTX1 caused by a novel connexin 32 mutation: a 6-year follow-up in neuroimaging and nerve conduction. Neurol. Sci. 37, 1063–1070.

Yang, Y., Hentati, A., Deng, H.X., Dabbagh, O., Sasaki, T., Hirano, M., Hung, W.Y., Ouahchi, K., Yan, J., Azim, A.C., et al. (2001). The gene encoding alsin, a protein with three guanine-nucleotide exchange factor domains, is mutated in a form of recessive amyotrophic lateral sclerosis. Nat. Genet. 29, 160–165.

Yazdankhah, M., Farioli-Vecchioli, S., Tonchev, A.B., Stoykova, A., and Cecconi, F. (2014). The autophagy regulators Ambra1 and Beclin 1 are required for adult neurogenesis in the brain subventricular zone. Cell Death Dis. 5, e1403.

Young, A.R., Chan, E.Y., Hu, X.W., Köchl, R., Crawshaw, S.G., High, S., Hailey, D.W., Lippincott-Schwartz, J., and Tooze, S.A. (2006). Starvation and ULK1-dependent cycling of mammalian Atg9 between the TGN and endosomes. J. Cell Sci. 119, 3888–3900.

Yu, W.H., Cuervo, A.M., Kumar, A., Peterhoff, C.M., Schmidt, S.D., Lee, J.H., Mohan, P.S., Mercken, M., Farmery, M.R., Tjernberg, L.O., et al. (2005). Macroautophagy—a novel Beta-amyloid peptide-generating pathway activated in Alzheimer’s disease. J. Cell Biol. 171, 87–98.

Yu, L., McPhee, C.K., Zheng, L., Mardones, G.A., Rong, Y., Peng, J., Mi, N., Zhao, Y., Liu, Z., Wan, F., et al. (2010). Termination of autophagy and reformation of lysosomes regulated by mTOR. Nature 465, 942–946.

Yu, H.C., Lin, C.S., Tai, W.T., Liu, C.Y., Shiau, C.W., and Chen, K.F. (2013). Nilotinib induces autophagy in hepatocellular carcinoma through AMPK activation. J. Biol. Chem. 288, 18249–18259.

Yu, R., Zhang, Z.Q., Wang, B., Jiang, H.X., Cheng, L., and Shen, L.M. (2014). Berberine-induced apoptotic and autophagic death of HepG2 cells requires AMPK activation. Cancer Cell Int. 14, 49.

Zavodszky, E., Seaman, M.N., Moreau, K., Jimenez-Sanchez, M., Breusegem, S.Y., Harbour, M.E., and Rubinsztein, D.C. (2014). Mutation in VPS35

associated with Parkinson’s disease impairs WASH complex association and inhibits autophagy. Nat. Commun. 5, 3828.

Zhang, X., Li, L., Chen, S., Yang, D., Wang, Y., Zhang, X., Wang, Z., and Le, W. (2011). Rapamycin treatment augments motor neuron degeneration in SOD1(G93A) mouse model of amyotrophic lateral sclerosis. Autophagy 7, 412–425.

Zhang, X., Chen, S., Song, L., Tang, Y., Shen, Y., Jia, L., and Le, W. (2014). MTOR-independent, autophagic enhancer trehalose prolongs motor neuron survival and ameliorates the autophagic flux defect in a mouse model of amyotrophic lateral sclerosis. Autophagy 10, 588–602.

Zhang, J., Lachance, V., Schaffner, A., Li, X., Fedick, A., Kaye, L.E., Liao, J., Rosenfeld, J., Yachelevich, N., Chu, M.L., et al. (2016). A Founder Mutation

in VPS11 Causes an Autosomal Recessive Leukoencephalopathy Linked to Autophagic Defects. PLoS Genet. 12, e1005848.

Zhao, Y.G., Sun, L., Miao, G., Ji, C., Zhao, H., Sun, H., Miao, L., Yoshii, S.R., Mizushima, N., Wang, X., and Zhang, H. (2015). The autophagy gene Wdr45/Wipi4 regulates learning and memory function and axonal homeostasis. Autophagy 11, 881–890.

Zheng, S., Clabough, E.B., Sarkar, S., Futter, M., Rubinsztein, D.C., and Zeitlin, S.O. (2010). Deletion of the huntingtin polyglutamine stretch enhances neuronal autophagy and longevity in mice. PLoS Genet. 6, e1000838.

Zhou, Z., Doggett, T.A., Sene, A., Apte, R.S., and Ferguson, T.A. (2015). Autophagy supports survival and phototransduction protein levels in rod photoreceptors. Cell Death Differ. 22, 488–498.
