
Veterinary Quarterly

ISSN: 0165-2176 (Print) 1875-5941 (Online) Journal homepage: http://www.tandfonline.com/loi/tveq20

# Ebola virus – epidemiology, diagnosis and control: threat to humans, lessons learnt and preparedness plans - an update on its 40 year's journey

Raj Kumar Singh, Kuldeep Dhama, Yashpal Singh Malik, Muthannan Andavar Ramakrishnan, Kumaragurubaran Karthik, Rekha Khandia, Ruchi Tiwari, Ashok Munjal, Mani Saminathan, Swati Sachan, Perumal Arumugam Desingu, Jobin Jose Kattoor, Hafiz M.N. Iqbal & Sunil Kumar Joshi

To cite this article: Raj Kumar Singh, Kuldeep Dhama, Yashpal Singh Malik, Muthannan Andavar Ramakrishnan, Kumaragurubaran Karthik, Rekha Khandia, Ruchi Tiwari, Ashok Munjal, Mani Saminathan, Swati Sachan, Perumal Arumugam Desingu, Jobin Jose Kattoor, Hafiz M.N. Iqbal & Sunil Kumar Joshi (2017): Ebola virus – epidemiology, diagnosis and control: threat to humans, lessons learnt and preparedness plans - an update on its 40 year's journey, Veterinary Quarterly, DOI: 10.1080/01652176.2017.1309474

To link to this article: http://dx.doi.org/10.1080/01652176.2017.1309474

© 2017 The Author(s). Published by Informa UK Limited, trading as Taylor & Francis Group

Accepted author version posted online: 20 Mar 2017.

Submit your article to this journal

View related articles

View Crossmark data

Full Terms & Conditions of access and use can be found at
http://www.tandfonline.com/action/journalInformation?journalCode=tveq20

Download by: [80.82.77.83]
Date: 21 March 2017, At: 03:30

Publisher: Taylor & Francis
Journal: Veterinary Quarterly
DOI: https://doi.org/10.1080/01652176.2017.1309474

TVEQ Review article

# Ebola Virus—Epidemiology, diagnosis and control: Threat to humans, lessons learnt and preparedness plans - An update on its 40 year's journey.

Raj Kumar Singh<sup>a</sup>, Kuldeep Dhama<sup>b*</sup>, Yashpal Singh Malik<sup>c*</sup>, Muthannan Andavar Ramakrishnan<sup>d</sup>, Kumaragurubaran Karthik<sup>e</sup>, Rekha Khandia<sup>f</sup>, Ruchi Tiwari<sup>g</sup>, Ashok Munjal<sup>f</sup>, Mani Saminathan<sup>b</sup>, Swati Sachan<sup>h</sup>, Perumal Arumugam Desingu<sup>b</sup>, Jobin Jose Kattoor<sup>c</sup>, Hafiz M.N. Iqbal<sup>i</sup>, Sunil Kumar Joshi<sup>j</sup>

<sup>a</sup>ICAR-Indian Veterinary Research Institute, Izatnagar, Bareilly, 243 122, Uttar Pradesh, India;  
<sup>b</sup>Division of Pathology, ICAR-Indian Veterinary Research Institute, Izatnagar, Bareilly, 243 122, Uttar Pradesh, India; <sup>c</sup>Division of Biological Standardization, ICAR-Indian Veterinary Research Institute, Izatnagar, Bareilly, 243 122, Uttar Pradesh, India; <sup>d</sup>Division of Virology, ICAR-Indian Veterinary Research Institute, Mukteswar Campus, Uttarakhand. 263 138, India; <sup>e</sup>Divison of Bacteriology and Mycology, ICAR-Indian Veterinary Research Institute, Izatnagar, Bareilly, 243 122, Uttar Pradesh, India; <sup>f</sup>Department of Biochemistry and Genetics, Barkatullah University, Bhopal-462026, Madhya Pradesh, India; <sup>g</sup>Department of Veterinary Microbiology and Immunology, College of Veterinary Sciences, Deen Dayal Upadhyay Pashu Chikitsa Vigyan Vishwavidyalay Evum Go-Anusandhan Sansthan (DUVASU), Mathura, India –281001, India; <sup>h</sup>Immunology Section, ICAR-Indian Veterinary Research Institute, Izatnagar, Bareilly, 243 122, Uttar Pradesh, India; <sup>i</sup>School of Engineering and Science, Tecnologico de Monterrey, Campus Monterrey, Ave. Eugenio Garza Sada 2501, Monterrey, N.L., CP 64849, Mexico; <sup>j</sup>Cellular Immunology Lab, Frank Reidy Research center for Bioelectrics, School of Medical Diagnostics & Translational Sciences, Old Dominion University, Norfolk, VA 20508, USA

*Corresponding authors: kdhama@rediffmail.com & malikyps@gmail.com

## Abstract

Ebola virus (EBOV) is an extremely contagious pathogen and causes lethal hemorrhagic fever disease in man and animals with high fatality rates. The recently occurred Ebola virus disease (EVD) outbreaks in the West African countries have categorized it as an international health concern. For the virus maintenance and transmission, the non-human primates and reservoir hosts like fruit bats have played a vital role. Furthermore, the virus may also get transferred through contaminated biological fluids like breast milk, tears, urine, semen, blood, etc. For curbing the disease timely, we need effective therapeutics/prophylactics, however, in the absence of any approved vaccine, timely diagnosis and monitoring of EBOV remains of utmost importance for restraining the EVD incidences. The technologically advanced vaccines like a viral-vectored vaccine, DNA vaccine and Virus-like particles (VLPs) are underway for testing against EBOV. In the absence of any effective control measure, the adaptation of high standards of biosecurity measures, strict sanitary and hygienic practices, strengthening of surveillance and monitoring systems, imposing appropriate quarantine checks and vigilance on trade, transport, and movement of visitors from EVD endemic countries remains the answer of choice for tackling the EBOV spread. Herein, we converse with the current scenario of EBOV giving due emphasis on animal and veterinary perspectives along with advances in diagnosis and control strategies to be adopted, lessons learned from the recent outbreaks and the global preparedness plans. To retrieve the evolutionary information, we have analyzed a total of 56 genome sequences of various EBOV species submitted between 1976 and 2016 in public databases.

Keywords: Ebola virus; Ebola hemorrhagic fever; public health; transmission; carriers; epidemiology; diagnosis; vaccines; treatment; control; preparedness

1. **Introduction**

   The entry of Ebola virus disease (EVD) was first marked in the year 1976 (Vogel and Viale 2014) and since then it is a highly contagious virus infection leading to the death of large numbers of humans (Bausch & Schwarz 2014; Muyembe-Tamfum et al. 2012; Bellizzi 2014; Dhama et al. 2015a). The disease is zoonotic, and evidence suggests the involvement of fruit bats as the main reservoir (Towner et al. 2008; Towner et al. 2009; Spengler et al. 2016; Hassanin et al. 2016; Judson et al. 2016). Although it affects both human and non-human primates, the major outbreaks have been reported in humans, mostly in Western Africa (Weingart et al. 2013; Feldmann & Feldmann 2014; Shrivastava et al. 2015a, 2015b). The Ebola virus (EBOV) has been listed as category A agent warranting biosafety containment level 4 (BSL4) for the handling of suspected samples. The disease symptoms start with fever, pain in stomach with vomiting and diarrhea, bleeding, intravascular coagulation, multisystem organ failure, and hypovolemic shock and in terminal cases, the patient succumbs to death (Mendoza et al. 2016). The virus spreads from diseased to normal persons through direct contact and infected materials. This virus tends to persist in body leading to further complications such as loss of vision (Shantha et al. 2017), hearing (Hebert et al. 2017), etc. along with transmission through semen from male to female (Hartmann et al. 2017) and to fetus at the time of pregnancy (Friedrich 2016; Wiwanitkit 2016).

   A major outbreak of EBOV, which occurred in 2014 in Western African countries, transpired to several other countries in a short span of time, culminating it as an alarming situation world over. The spread of EBOV was so rapid that family members and the community around infected people suffered sternly (Kuhn et al. 2010; Weingart et al. 2013; Gatherer 2014). Of note, affected young children showed comparatively shorter incubation period and rapid course of disease with high mortality (Team 2015). Owing to global trade and tourism, there is every possible opportunity that EBOV can spread to other continents where it may lead to massive outbreaks. The epidemics which occurred over the past few years should serve as an eye-opener to the world so that everyone is well prepared for the next pandemic if any occurs. In this direction, it is important that the nature of the agent, its evolution pattern, and pathway of spread must be understood at a subtle level. To deal with such a highly spreading disease we must be equipped with rapid state-of-art diagnostics which are useful at the bedside of the patient, ultimately benefiting in opting appropriate control and treatment measures. Likewise, there is an urgent need to have effective and adequate vaccines to provide protection to the population at high risk. Therapeutic strategies must be evaluated for their high potential.

   Currently, there are five distinct EOBV species which have been described, and four of them (Zaire ebolavirus, Sudan Ebolavirus, Tai Forest ebolavirus, and Bundibugyo ebolavirus) causes disease in humans, whereas the Reston ebolavirus is known to cause disease in non-human primates solely. We here converse comprehensively over the developments ensued in understanding the etiopathology of EVD, ecology, prevention and control strategies along with various therapeutic alternatives.

2. **The virus and its genome**

   Ebola virus (EBOV) belongs to the order Mononegavirale (single-stranded, non-segmented, negative sense RNA virus) of the family Filoviridae, genus Ebolavirus. Other members of the family include Marburgvirus and Cuevavirus, of which Marburgvirus has also been implicated in causing hemorrhagic diseases similar to EBOV, and both of these are filamentous shape viruses. The genus Marburgvirus consists of a species Marburg
marburgvirus with two viruses viz., Marburg virus (MARV), and Ravn virus (RAVV). The genus Cuevavirus consists of a single species—Lloviu cuevavirus with Lloviu virus (LLOV). The EBOV was first isolated in 1976 near Zaire valley in the Democratic Republic of Congo (Zaire) rooting its name from the place of isolation (Kuhn et al. 2010), initially named as an Ebola-like virus which was later changed into Ebola virus (EBOV) in the year 2002. The genus Ebolavirus consists of 5 species viz., 1. Zaire ebolavirus (Zaire virus-ZEBOV), 2. Sudan ebolavirus (Sudan virus-SUDV), 3. Reston ebolavirus (Reston virus-RESTV), 4. Tai Forest ebolavirus (Tai Forest virus-TAFV), and 5. Bundibugyo ebolavirus (Bundibugyo virus-BDBV) and Côte d'Ivoire ebolavirus (CIEBOV) (Bausch & Schwarz 2014; Kuhn et al. 2013; Bukreyev et al. 2014; Kuhn et al. 2014a; Kuhn et al. 2014b; http://www.ictvonline.org/virusTaxonomy.asp). The genetic diversity among species in the genus Ebolavirus ranges between 25 and 35% (Grard et al. 2011). Among the 5 species, ZEBOV and SUDV are extremely deadly, of which the former is most dangerous with more than 90% lethality (Bellizzi 2014; Toit 2014; Safari et al. 2015). ZEBOV was frequently documented in central Africa and has caused the major outbreaks recently in Western African countries like Nigeria, Liberia, Guinea, Senegal, and Sierra Leone (Baize et al. 2014).

As shown in Figure 1, structurally the virions of filovirus show pleomorphism which includes long filaments (Latin-filum means thread), shaped like a "6", a "U", or a circle. Viral filaments are 80 nm in diameter with up to 14,000 nm in length and are enveloped in a lipid membrane (http://www.cdc.gov/vhf/virus-families/filoviridae.html). The genome of EBOV is 19 kb in length, consists of seven genes which are arranged as -3" (leader)-NP-VP35-VP40-GP-VP30-VP24-L-(trailer) 5" (Volchkov et al. 1999). These seven open reading frames (ORFs) encode structural nucleoprotein (NP), envelope glycoprotein (GP) and matrix proteins like viral proteins (VP24) (membrane-associated protein) and VP40. Except for GP, each of these genes encodes a single protein product. Apart from structural proteins, L (RNA polymerase), VP30 and VP35 (both polymerase matrix proteins) correspond to the major non-structural protein. These proteins exhibit different roles in virus pathogenesis like VP24 inhibits interferon signaling, VP35 is an antagonist of interferon, whereas VP40 has a role in budding and release of the virus (Feldmann & Geisbert 2011). VP40 has been recently found in the extracellular region like exosome that can affect the immune system of the host (Pleet et al. 2017). The GP gene codes for three different versions of GP viz. GP (contains GP1 attachment protein and the GP2 fusion/entry protein), soluble form (sGP-is produced from the unedited RNA transcript) and small soluble form (ssGP-is a truncated version of sGP). Most of these proteins have multiple functions. NP facilitate the genomic RNA encapsidation to form RNP complex (RNA+NP++L), which plays an essential role in virus replication. The GP and VP40 are essential components of the viral envelope. GP has a role in virus attachment, entry, causing cell rounding, down-regulating host surface proteins, causes inflammation, cell damage and cytotoxicity (Casillas et al. 2003; Geisbert et al. 2009; Olival et al. 2013). The nucleocapsid consists of the viral RNA complexed with five proteins viz., NP, VP24, VP30 and VP35 and L (Sanchez et al. 1993; Lee & Saphire 2009; Bharat et al. 2012; Dzubańska et al. 2014; Basler 2014; Brauburger et al. 2014; Gallaher & Garry 2015; Dong et al. 2015; Jun et al. 2015). Unlike influenza A virus, the rate of genetic change is very slow in EBOV, but this virus diverged several thousand years ago (Suzuki & Gojobori 1997; Taylor et al. 2011).

### 2.1. Evolutionary picture of Ebolaviruses

To retrieve the evolutionary information, we have analyzed a total of 56 genome sequences of various EBOV species submitted between 1976 and 2016 in public database, wherein the Zaire, Sudan, Reston, Tai and Bundi EBOVs formed distinct clades (Figure 2). The ZEBOV strains organized into two distinct sub-clades, sub-clade 1 comprising of the isolates from Gabon and the Democratic Republic of Congo and sub-clade 2 clubed SierraLeone and Liberia isolates (Figure 2). The evolutionary divergence between the species was estimated using the Maximum Composite Likelihood model (Tamura et al. 2004). The number of transversional substitutions per site from the average of overall sequence pairs between groups was obtained. The rate variation among sites was modeled with a gamma distribution (shape parameter = 1). Codon positions included were 1st+2nd+3rd+Noncoding. There was a total of 17999 positions in the final dataset. Evolutionary analyses were conducted in MEGA 6 (Tamura et al. 2013). Further, the analysis revealed that the Tai and Bundi EBOVs remain closely related to 17.8% mean group diversity; the ZEBOV had the mean group diversity of 26.1% and 26.8%, respectively with Tai and Bundi EBOV. The SUDV outgroup from ZEBOV, Tai and Bundi EBOVs revealed mean group diversity of 36.3%, 37.4%, and 37.6%, respectively. Similarly, RESTV outgroup from Zaire, Tai and Bundi EBOVs came up with mean group diversity of 34.1%, 35.1%, and 34.8%, respectively. Likewise, the mean group diversity between Sudan and Reston EBOVs was found to be 36.6%.

### 2.2. Epidemiology, Transmission, and Spread

It is evident that since the first report of EVD, the number of ratified cases has increased several folds. The causality cases of EVD almost doubled in Africa in comparison to the earlier reported cases of the last four decades (6,458 deaths out of 12,299 cases; Muyembe-Tamfum et al. 2012; Tambo et al. 2014; Bellizzi 2014) and the cases reported between 2014-2016 (12,922 deaths out of 31,079 cases) (CDC 2015) (Table 1). The 2014-15 outbreak was assumed to be the biggest epidemic in the history of this disease (Gatherer 2014; Zhang & Wang 2014; Elstona et al. 2017). Though several strains of EBOV have been identified in the past, the 2014-15 outbreak affecting mainly Western African countries (Guinea, Sierra Leone, Liberia, Senegal, and Nigeria) was confirmed due to ZEBOV (Bellizzi 2014). This strain was the first EBOV which caused the historic 1976 Ebola case affecting a middle age school teacher in the Democratic Republic of the Congo (Johnson 1978). Identifying the severity and out-of-control situation about EVD, the World Health Organization declared it as a “Public Health Emergency of International Concern” (PHEIC). Owing to its high virulence and fast transmission capability, it is categorized under ‘class A’ bio-weapon organism (Balmith et al. 2016) and thus making it essential to have real-time monitoring of EVD cases and to look into any connectivity among the cases. For this, use of smart mobile phones reporting of EVD cases in West Africa was encouraged by epidemiologists as an effective technique. Use of social network analysis in this way will certainly help in understanding disease spread in a better way (Kangbai 2016). The case fatality rate observed in Ebola-affected cases in different countries varied between 25–100%. The countries affected during 1976-2016 are depicted in Figure 3 and a historical timeline of EBOV is presented in Figure 4. The disease outbreaks caused by different EBOV species are given in Table 1 and chronological data of EBOV cases and fatality since its first incidence is summarized in Table 2.
2.2.1. Zaire ebolavirus (ZEBOV)

The first known Ebola outbreak occurred in 1976 in Sudan (later the causative agent was classified under Sudan virus-SUDV) and Democratic Republic of the Congo (DRC; previously known as Zaire) (Report of an International Commission 1978; Report of a WHO/International Study Team 1978, Figure 3, Figure 4). In the DRC outbreak (between September and October 1976), 318 cases of acute viral hemorrhagic fever (VHF) with 280 deaths (case fatality rate-CFR 88%) occurred in and around the Yambuku region. The duration of the clinical disease was about one week with non-specific symptoms for the initial 4 days followed by a severe sore throat, maculopapular rash, abdominal pain and bleeding from multiple sites - mainly from the gastrointestinal tract. The causative agent could be isolated from 8 cases using Vero cell cultures and was named as Ebola virus (Report of an International Commission 1978). The index case (the first identified case) was a 44-year-old male instructor who has been treated for presumptive malaria at the Yambuku hospital, and from there the subsequent outbreaks emerged (WHO 1978b).

The first EBOV was isolated in 1976 (isolate E718) from the blood sample of a 42-year-old Belgian nursing sister (patient No. 718) who was working at Mission Hospital, Yambuku, DRC. The sister (Myriam Louise Ecran) was one of the first victims of Ebola hemorrhagic fever (EHF) and died on 30 September 1976 (Kuhn et al. 2014; CDC 2015; www.bbc.com/news/magazine-28262541). Researchers decided to name the virus to the nearby river—the Ebola, rather than the village name to avoid stigma to that village. In Belgian, the river named as l'Ebola (the indigenous Ngbandi name is Legbala) means “white water” or “pure water” (Report of an International Commission 1978; Report of a WHO/International Study Team 1978). Since that time, the name Ebola has been used (http://www.nature.com/scitable/blog/viruses101/the_scientist_who_discovered_ebola). The EBOV was initially categorized as “Marburg-like” due to the morphological resemblance (Johnson et al. 1977). Initially, it was presumed that the DRC and Sudan outbreaks were caused by the same strain of the virus (the distance between Yambuku and Nzara is about 800 km) but, later it was realized that there is involvement of two distinct species viz., Zaire Ebola virus (Ebola virus-ZEBOV) and Sudan ebolavirus (Sudan virus-SUDV), respectively (McCormick et al. 1983; Pourrut et al. 2005; Muyembe-Tamfum et al. 2012).

The subsequent outbreak of EBOV occurred in June 1977 in Tanda (325 km from Yambuku), DRC and a 9-year-old girl died with typical clinical signs of EHF. No secondary cases were identified (Heymann et al. 1980). Later, a less extensive form of EBOV occurred, between July and October 1979, in Nzara and Yambio (25 km away from Nzara), wherein 22 deaths occurred out of 34 cases with CFR of 65% (Baron et al. 1983).

After 15 years of pause, three EBOV outbreaks occurred in northeastern Gabon between 1994 and 1997. The first epidemic (fall 1994) occurred in Mekouka along with co-infection of yellow fever virus. In this epidemic, 49 cases of EVD, with 29 deaths (CFR 59%) occurred. A second epidemic (spring 1996) in Mayibout caused 31 cases with 21 deaths (CFR 68%) and of note a chimpanzee was suspected as the index case. The third epidemic (fall 1996) occurred in Booué and caused 60 cases with 45 deaths (CFR 75%). Several chimpanzees died in this area, and EBOV antigen could be detected from one of the skin samples by immuno-staining (Georges et al. 1999). Partial GP gene sequence analysis revealed that the causative agent behind all these three outbreaks was EBOV (Georges-Courbot et al. 1997). Additionally, this outbreak also led to the death of a nurse in South Africa which was transmitted from an EBOV-infected Gabonese physician working in Libreville and traveled to Johannesburg for treatment (WHO 1996; Pourrut et al. 2005).

Afterwards, one of the largest EBOV outbreaks occurred in the city of Kikwit, DRC (~500 km southeast of Kinshasa), between January and July 1995 with 255 deaths out of 315 cases (CFR 81%). Of note, a 42-year-old male charcoal worker was identified as index case who had acquired infection from some natural reservoir such as bats, as no great apes were found in that region (Khan et al. 1999; Guimard et al. 1999).

In Taï Forest, Côte d'Ivoire, between 1996 and 1997, autopsy samples from several species of vertebrates including bats, rodents, insectivores, monkeys, and birds were screened, and anti-EBOV IgG could be detected only from a Colobe bai monkey (red colobus, Colobus badius) (Pourrut et al. 2005). During ecological studies in the Central African Republic (CAR) in 1998, 242 vertebrates include bats, rodents and insectivores were captured and organs tested for the presence of EBOV nucleic acid by GP and L genes based RT-PCR. Seven animals including six mice (Mus setulosus and Praomys spp.) and a shrew (Sylvisorex ollula) were found harboring EBOV (Morvan et al. 1999).

Between October 2001 and December 2003, at least five EBOV outbreaks occurred in the border areas of northeast Gabon and northwest DRC with 313 cases and 265 deaths (CFR 85%). The first outbreak (October 2001 to May 2002) occurred in both Gabon and DRC; the outbreak occurred as multiple independent epidemic chains with 92 cases and 70 deaths (CFR 77%). Epidemiologic investigations suggest the possible involvement of carcasses of wild animals (bushmeat) including duikers, chimpanzees, and gorillas in the index patients. In January 2002, in Franceville, a single case was reported in the south of Gabon (Nkoghe et al. 2005a) and the remaining four outbreaks occurred only in DRC.

In a later outbreak (Entsiami - January 2002 to June 2002), a total of 30 cases with 25 deaths (CFR 83%) occurred; a gorilla and a duiker were suspected to be the source for the index cases. In the following outbreak (Oloba - May to June 2002), 13 cases with 12 deaths (CFR 92%) occurred, and a chimpanzee was identified as the source for index case. In another outbreak (Mbomo and Kéllé - December 2002 to April 2003) 143 cases with 129 deaths (CFR 90%) occurred; gorillas and duikers were suspected to be the source of infection for the index cases. The last outbreak (Mbanza and Mbomo - November 2003 and December 2003) caused 35 cases with 29 deaths (CFR 83%). However, the source of infection could not be traced (http://www.who.int/wer/2003/en/wer7826.pdf; Formenty et al. 2003; Leroy et al. 2004; Rouquet et al. 2005; Pourrut et al. 2005; Nkoghe et al. 2005b). Sequencing analysis of GP gene of EBOV revealed that the above five human outbreaks originated from distinct animal sources and viral strains (Leroy et al. 2004). These Gabon-DRC cross-border outbreaks were marked by large wildlife epizootics associated with over 80% mortality especially in great apes (Leroy et al. 2004; Rouquet et al. 2005; Bermejo et al. 2006; Lahm et al. 2007; Grard et al. 2011).

After the first outbreak in the forest zone of Gabon-DRC, an Animal Mortality Monitoring Network (AMMN) was created. The analysis of the samples recovered between August 2001 and June 2003 revealed that 10 gorillas, 3 chimpanzees, and 1 duiker were positive for EBOV infection (Rouquet et al. 2005). Another group of researchers from Centre International de Recherches Médicales de Franceville (CIRMF), Gabon analyzed a total of 34 carcasses in the same period, and 14 (10 gorillas, 3 chimpanzees, and 1 duiker) were found positive for EBOV infection (Pourrut et al. 2005).

In the seroprevalence study of EBOV in 20 species of non-human primates between 1985 and 2000 in Cameroon, Gabon, and the DRC, the wild chimpanzees showed 12.9% positivity. Also, few other monkey species (5 drills, one baboon, one mandrill and one Cercopithecus sp.) were also positive indicating the complexity of the EBOV circulation and the possibility of involvement of more reservoir species (Leroy et al. 2004; Pourrut et al. 2005).
To identify the viral reservoir in the multiple outbreaks of Ebola in Gabon and DRC between 2001 and 2005, the investigation was carried out from samples of over 1000 small vertebrates. Anti-EBOV IgG, as well as EBOV-specific RNA sequences, were detected from three different bat species (Hypsignathus monstrosus, Epomops franqueti, and Myonycteris torquata). The sequence confirmation of EBOV indicates bats might serve as a natural reservoir of EBOV (Leroy et al. 2005; Pourrut et al. 2009). In a large-scale serological survey from these three bat species which were captured between 2003 and 2006 in Gabon and DRC 5% EBOV infection positivity was shown, supporting their potential reservoir status (Pourrut et al. 2007; Pourrut et al. 2009). In a further extension of this study, samples were analyzed for EBOV from nine species of bats which were sampled between 2003 and 2008 and among these, 4% of the total population sampled belonged to six species (Epomops franqueti, Hypsignathus monstrosus, Myonycteris torquata, Micropterus pusillus, Mops condylurus and Rousettus aegyptiacus) which revealed anti-EBOV antibodies (Pourrut et al. 2009).

The EBOV reemerged in the Luebo region of DRC in 2007 and 2008, causing two successive outbreaks. The 2007 outbreak caused 264 cases with 187 deaths (CFR 71%) and re-occurred one year later with 32 cases with 15 deaths (CFR 47%). The suspected source of the Luebo 2007 outbreak was fruit bats (Hypsignathus monstrosus and Epomops franqueti). The index source of the 2008 outbreak could not be traced out. The analysis of whole-genome sequence results of viruses from these two outbreaks confirms them almost identical (Grard et al. 2011).

Following this, the EBOV outbreak occurred in multiple villages in the DRC between July and November 2014 in Équateur province with a total of 69 cases with 49 deaths (CFR 71%). A pregnant woman was the index case, and monkey meat was the suspected source of the infection. A complete genome sequence from this outbreak shared 99.2 and 96.8% identities with EBOVs of 1995 outbreak in the DRC and current outbreak in West Africa, respectively (Maganga et al. 2014).

The recent EBOV epidemic burst in West Africa started in Guinea in late 2013 and spread to several countries within a few months period (Cenciarelli et al. 2015). As of April 13, 2016, a total of 31,013 cases with 12,872 deaths (CFR 41.5%) have been reported. The widespread transmission occurred in three countries viz., Liberia, Sierra Leone, Guinea (28,616 cases with 11,310 deaths - CFR 40%). Very few travel-related cases (36 cases and 15 deaths) have been reported in seven countries viz., Italy, Mali, Nigeria, Senegal, Spain, the United Kingdom, and the United States of America) (http://www.cdc.gov/vhf/ebola/outbreaks/2014-west-africa/case-counts.html). The suspected index case for these outbreaks was a two-year-old child (http://www.who.int/csr/disease/ebola/ebola-6-months/guinea/en/). The whole genome sequencing of 99 samples from 78 affected individuals in Sierra Leone suggested the involvement of the West African variant of EBOV in the current epidemic. The suspected source of the outbreak was from an unknown animal reservoir. The Sierra Leone outbreak originated from the introduction of two EBOV variants lineages (Gire et al. 2014).

Considering the highest risk to the national security and public health concern, the US-NIAID classified the EBOV under Category A Priority Pathogens (https://www.niaid.nih.gov/topics/biodefenserelated/biodefense/pages/cata.aspx). Laboratory-acquired EBOV infection also had been reported from Russia on two occasions. It had emerged that in 1996, a female lab technician carrying out the high-risk work, cut herself when she drew blood from a horse that had been infected with EVD and died quickly. In the second incident, a Russian lab worker died in 2004 while working with EBOV-infected guinea pigs (https://www.washingtonpost.com/national/health-science/2014/10/23/ce409716-5945-11e4-b812-38518ae74c67_story.html);
http://www.theaustralian.com.au/news/world/the-times/russians-died-in-ebola-weapons-lab/news-tory/e426b0bd0064da5fe8c78e4d485c162c).

2.2.2. Sudan ebolavirus (Sudan virus-SUDV)

Sudan virus–SUDV (Sudan ebolavirus) and Ebola virus–EBOV (Zaire ebolavirus) are related regarding clinical disease but vary in serology, pathogenic potential, and virological properties (Richman et al. 1983). In Sudan, between June and November 1976, a large outbreak of EHF occurred in Nzara, Maridi, and the surrounding area and caused 284 cases with 151 deaths (CFR 53%). The clinical disease observed in this outbreak was associated with high mortality (CFR 53%) and a prolonged recovery period in survivors. A cotton factory worker in Nzara was identified as the index case (Report of a WHO/International Study Team 1978). The specimens collected during the Sudan and DRC outbreaks were sent to high-security laboratories in England (Microbiological Research Establishment, Porton Down), Belgium (Institute of Tropical Medicine, Antwerp), and the United States of America (CDC, Atlanta) for isolation and identification of the agent. All three laboratories isolated a virus that morphologically resembled Marburg virus but was serologically distinct (Bowen et al. 1977; Johnson et al. 1977; Emond et al. 1977). In the UK, one of the investigators accidentally pricked his thumb while handling the samples and developed an illness (Emond et al. 1977).

Between July and October 1979, SUDV outbreak occurred in Nzara and Yambio of southern Sudan and caused 34 cases with 22 deaths (CFR 65%). A 45-year-old man who had been employed in the Nzara textile factory was the index case (Baron et al. 1983), and the possible source of infection was bats (Pourrut et al. 2005).

An outbreak of VHF occurred between October 2000 and January 2012 from Gulu district (borders Sudan), Uganda, and 425 infected cases were reported with 224 deaths (CFR 53%). A concurrent outbreak of SUDV and measles occurred in Yambio County of southern Sudan between April and June 2004. A total of 17 VHF cases and 7 deaths (CFR 41%), were reported in Yambio payam. The CDC confirmed the presence of SUDV in this outbreak (WHO 2005). The index case was a radio technician, and the source of the outbreak was a baboon (Papio anubis) (WHO 2005). During 2008, SUDV outbreak recurred in Kaluamba with 37 cases and 16 deaths (CFR 44%). An 18-year-old girl appeared to be an index case, but the source was not known (Muyembe-Tamfum et al. 2012; Rewar & Mirdha 2014).

In May 2011, a 12-year-old girl from Luwero, Uganda, was admitted to the Hospital with symptoms of VHF and died within 3 hours of admission. Evidence of infection with SUDV was confirmed by RT-PCR, antigen-detection ELISA, and virus isolation. Analysis of complete genome sequencing of the above virus had 99.3% identity to the Gulu SUDV-2000 virus. The circumstantial evidence suggests that a bat was the source of infection (Shoemaker et al. 2012).

In order to detect VHFs in Uganda, a surveillance program was initiated in 2010 by the CDC, Atlanta, USA, in collaboration with the Uganda Virus Research Institute (UVRI) and the Uganda Ministry of Health (UMoH) (MacNeil et al. 2011). Samples collected from the outbreak of Kibaale, Uganda between July and August 2012, yielded 11 laboratory confirmed cases (SUDV infection) with 4 deaths (CFR 36%). Complete genome analysis of the above 4 clinical samples showed ~99.9% sequence identity with each other and ~99.2% identity with the SUDV Nakisimata (Uganda-2011), and SUDV Gulu (Uganda-2000) isolates (Albariño et al. 2013). In the same surveillance, the samples collected from an outbreak in the relatively close districts of Luwero, Jinja, and Nakasongola, Uganda in November 2012 yielded 6 laboratory confirmed cases with 3 deaths (CFR 50%). Sequence analysis of NP gene from three serum samples revealed that they are nearly ~100% identical. Involvement of bats was suspected for its transmission (Albariño et al. 2013).

2.2.3. Reston ebolavirus (Reston virus - RESTV)

In October 1989, 100 cynomolgus monkeys (*Macaca fascicularis*) were transported from Manila, the Philippines to Reston, Virginia and placed in a quarantine facility. During the quarantine, numerous macaques died, due to concomitant infection of Ebola-related filovirus (as per the recent classification - Reston virus or RESTV) and by simian hemorrhagic fever virus (SHFV - an arterivirus) (CDC 1989). RESTV could be isolated at least from five monkeys (Jahrling et al. 1990). This was considered as the first known EBOV outbreak that occurred outside of Africa as well as in non-human primates (Miranda & Miranda 2011).

Between November 1989 and March 1990, active infection of RESTV occurred in seven shipments of cynomolgus monkeys and further transmission to monkeys in quarantine facilities and many of them died. Four animal handlers at a quarantine facility had anti-RESTV antibodies (CDC 1989; Jahrling et al. 1990; CDC 1990a). Among these four cases, viremia developed in one person on days 9, 10 and 11 post-inoculation (WHO 2009). CDC has screened ~2200 serum samples from different monkey species from a variety of settings, and ~10% had antibodies against at least one of four filovirus test antigens (EBOV, SUDV, RESTV, and Marburg virus) (CDC 1990b).

Studies were initiated to document transmission at export facilities located in the Philippines. At one export facility, within 3 months' period (between March and May 1990), out of 403 monkeys, 161 (40%) died from RESTV as confirmed in these animals. The clinical manifestations observed in the outbreak included diarrhea (50%), respiratory illness (34%), and hemorrhage (1%) (Hayes et al. 1992). However, three workers in the animal facility developed antibodies against RESTV without any clinical signs (Miranda et al. 1991). The source of RESTV was not known (Rollin et al. 1999).

The next outbreak occurred in 1992. A batch of 55 cynomolgus monkeys was imported from a monkey-breeding company in the Philippines into Italy in March 1992. Four monkeys died within a month, and viruses isolated from three monkeys were confirmed as filoviruses by electron microscopy. They were shown to be antigenically related to EBOV by indirect immunofluorescence assay (WHO 1992). Although two people developed virus-specific IgG antibodies, clinical infections not observed (Miranda & Miranda 2011).

In March 1996, an imported cynomolgus monkey that was held in a quarantine facility in Texas developed nonspecific signs such as anorexia and lethargy and died after 3 days. A second monkey also showed similar kind of symptoms, and it was euthanized. RESTV was isolated from both the animals. No viral antigen or RESTV-specific antibodies could be detected from animal handlers. Also, during the laboratory investigation, as in 1989 and 1990, SHFV could have infected some of the animals (Rollin et al. 1999). GP gene of RESTV isolated from the first animal had 98.9% nucleotide identity with the 1989 RESTV (Sanchez et al. 1999). In a USA-Philippine Joint Investigation at the monkey export facility in the Philippines it revealed that 14 out of 21 animal houses had infected monkeys. Anti-RESTV Antibodies and virus nucleic acid could be detected from 3 out of 1732, and 132 of 1011 monkeys, respectively. Seroprevalence on animal handlers from monkey facilities and people with occupational exposure to pigs in USA, Italy, and the Philippines, from 1989 to 2009 indicated that 9 (2%) out of 662 persons had detectable RSTV-specific IgG antibodies (Rollin et al. 1999; WHO 2009; Miranda & Miranda 2011).

In the Philippines, during 2008–2009, serum samples were collected from 141 wild-caught bats (17 species) and at different locations 7 out of 16 serum samples from *R. amplexicaudatus* bats had anti-RESTV antibodies. These antibody-positive bats were captured near the RESTV positive cynomolgus monkeys and swine suggesting that bats are a possible natural reservoir of RESTV (Taniguchi et al. 2011). In the Philippines, between July 2007 and June 2008, there were multiple outbreaks of a respiratory and abortion disease
syndrome in swine. The clinical signs resembled an infection caused by a highly pathogenic porcine reproductive and respiratory syndrome virus (PRRSV—an arterivirus), also referred to as “blue ear disease”. The diagnostic investigation carried out at two USA laboratories (APHIS and FADDL), revealed concomitant infection of pigs with PRRSV and RESTV. Viral genomes from three samples revealed a very high inter-isolate divergence (3.93%) as compared to the original 1989 RESTV (2.5%). Anti-RESTV antibodies could be detected in 6 out of 141 people who worked on pig farms or with swine products (WHO 2009; Barrette et al. 2009).

In a multi-institutional study on bats in the Philippines, RESTV RNA was detected in oropharyngeal swabs taken from *Miniopterus schreibersii*, and sequencing of three samples showed high nucleotide identity with a pig isolate from Bulacan province. Also, four sera showed the presence of anti-EBOV antibodies i.e. three from *Acerodon jubatus*, and one from *Pteropus vampyrus* (possibly infected with EBOV and RESTV), also suggesting that bats could be a natural reservoir for RESTV (Jayme et al. 2015).

In China, between February and September 2011, 137 PRRSV confirmed pig spleen samples were screened for the presence of RESTV, and 4 samples were found positive. The partial sequencing of L gene from four samples showed 95.1 to 97.2% nucleotide identities with each other and 96.1–98.9% identity with two RESTV variants of domestic pigs and cynomolgus macaques from the Philippines (Pan et al. 2014).

### 2.2.4. Tai Forest ebolavirus (Tai Forest virus - TAFV) and Bundibugyo ebolavirus (Bundibugyo virus – BDBV)

There was a single non-fatal human case of Tai Forest virus (TAFV—previously known as Côte d'Ivoire ebolavirus). In 1994, a Swiss ethnologist became infected by TAFV after conducting an autopsy examination on a wild chimpanzee in the Tai Forest, Cote-d'Ivoire (Le Guenno et al. 1995).

The first outbreak of VHF due to Bundibugyo virus—BDBV (*Bundibugyo ebolavirus*) occurred in Uganda between August and December 2007 and infested 131 cases with 42 deaths (CFR 32%) (MacNeil et al. 2011). The entire genome sequencing of BDBV and the TAFV were carried out by the Next Generation Sequencing, and the analysis revealed that the BDBV differs significantly from the other four Ebola virus species (Towner et al. 2008). Another outbreak of BDBV occurred in Isiro, DRC between June and November 2012 with 36 cases with 13 deaths (CFR 36%). The full genome sequence analysis revealed that the new BDBV was ~98.6% identical to those of the original BDBV-2007 isolate (Albariño et al. 2013).

Human outbreaks of EVD are hypothesized to be originated from direct contact with an infected animal or its body fluids. The human to human transmission occurred through the direct contact with blood from infected patients or other body fluids (Report of an International Commission 1978; Baron et al. 1983; Dowell et al. 1999; Muyembe-Tamfum et al. 1999; Roels et al. 1999; Francesconi et al. 2003; Muyembe-Tamfum et al. 2012; Lawrence et al. 2017). The major risk factors associated with virus transmission chains are gathering funerals of Ebola positive patients, close contact with family members of infected patients, and treating patients without adequate personal protective measures (Okware et al. 2002). People visiting or taking care of infected persons are at high risk of Ebola infections (Muyembe-Tamfum et al. 2012). Recently, a post-recovery sexual transmission of EBOV via semen has been documented in the West African outbreak (Mate et al. 2015). Food can contribute to EBOV transmission, particularly through harvesting of bush meat (meat of wildlife) (Mam et al. 2015). Transmission through fomites in a clinical setting is unlikely (Bausch et al. 2007). The experimental aerosol transmission has been demonstrated between monkeys (Johnson et al. 1995; Jaax et al. 1995; Reed et al. 2011) and from pigs to monkeys

10(Weingartl et al. 2012). Similarly, airborne transmission among humans is hypothetical (Baron et al. 1983; Dowell et al. 1999).

### 3. Host range, carriers, and reservoirs

EBOV causes acute hemorrhagic fever in human and non-human primates like cynomolgus (*Macaca fascicularis*) and rhesus monkeys (*Macaca rhesus*), African green monkey (*Cercopithecus aethiops*) and baboons (*Papio hamadryas*). Inoculation of infected material to laboratory animals had resulted in non-lethal febrile disease, and subsequent passages of splenic material in animals increases the virulence leading to the death of laboratory animals. Among mice, the newborn is more sensitive than adult ones. Stray dogs in Africa eating animals died due to EBOV remained asymptomatic. Furthermore, a survey in 2005 showed that around 30% of dogs had seropositivity for EBOV without any clinical signs (Allela et al. 2005; Osterholm et al. 2015). Pigs have been shown to acquire natural EBOV infection and can transmit EBOV to humans (Osterholm et al. 2015) and hence alarming the food safety and animal health officials (Feldmann & Feldmann 2014). However, limited information is available to predict or prove the role of other livestock species, like cattle, horse, sheep, etc. being a reservoir of EBOV (Mann et al. 2015). Apart from the transmission of the virus from reservoir animals, contaminated plant food products may also act as a source of infection to the susceptible population (Mann et al. 2015).

Researchers predict that there may be a reservoir host for EVD, as it has a re-emerging pattern and hypothesize that three fruit bat species viz. *Epomops franqueti*, *Hypsignathus monstrosus*, and *Myonycteris torquata* are the important reservoir (Leroy et al. 2009). The African fruit bats have been suggested to be the natural reservoirs for EBOV (Zaire) and Marburg virus (Leroy et al. 2005; Pourrut et al. 2009; Barrette et al. 2009). In the Philippines, anti-RESTV antibody could be detected from bats which were captured near to the area where RESTV infections in cynomolgus monkeys and swine had been detected, suggesting that bats could be the possible natural reservoir (Taniguchi et al. 2011; Jayme et al. 2015). Finding of EBOV antibodies in bats from Bangladesh increased the speculation of the bat as a reservoir (Olival et al. 2013). In humans, there is no age and sex difference in susceptibility to EVD, and additionally, there is no report about humans to act as a reservoir. Recently, a study conducted to find any difference in the disease pathology between male and female man showed both had similar risk rate, yet females have higher survival rate after developing the disease (Team 2016). The virus can persist in the semen of men for a period of several months (~18 months) after recovery and can be transmitted to the female (Heeney 2015; Mackay & Arden 2015; Fallah et al. 2016; Bausch and Crozier 2016) which suggests that man may be a reservoir for the EBOV. To tackle this, WHO has recommended testing of semen by RT-PCR from EVD survivors for three months from onset of the disease along with counseling to encourage safe sexual practices until their semen is tested negative twice (Purpura et al. 2016; Check 2016). Death among chimpanzees and gorillas was always interposed with outbreaks in man, dead non-human primate's contact and also eating of fruit bats (Muyembe-Tamfum et al. 2012). Direct contact with secretions from infected persons and organ transplantation seems to be another important means of transmission. Body secretions from non-human primates also act as a potential source of infection to man (Feldmann & Feldmann 2014). Mucous membranes seem to be the major portal of entry of the organism. Aerosol infection in non-human primates raises concern about the chances of airborne infection in humans (Olival et al. 2013). Research findings show that the dried virus particle can travel longer distance compared to the virus in fluid form. EBOV needs a liquid medium for its survival. Hence, its transmission can't be airborne. Studies reported that due to the increase in population and also increased movement of man and animals there is difference in the disease dynamics of recent outbreaks of EVD as compared to earlier outbreaks (Pigott et al. 2014).
3.1. Role of animals in transmission of EBOVs

The possible source and mode of transmission of EBOV for each outbreak are described earlier in the section on epidemiology. In this section, the role of animals in the transmission of EBOV is highlighted. Regarding the multiple outbreaks of EBOV between 2001 and 2003 in Gabon and DRC, there is a link to handling/eating of bush meat. The majority of outbreaks started with the history of handling of infected wild animal carcasses, including gorillas, chimpanzees, duikers (small antelopes), and possibly monkeys by the index cases (WHO 2003; Leroy et al. 2004).

In 1994, in the Taï National Park, Côte d'Ivoire an Ebola outbreak occurred among chimpanzees (Formenty et al. 1999a), and an ethologist was infected with TAFV while conducting a necropsy on a wild chimpanzee (Formenty et al. 1999b). Investigation of a 2007 EBOV outbreak suggested that the possible source was fruit bats (Leroy et al. 2009). Since anti-EBOV antibodies and viral RNA have been detected in bats, there is a strong belief in the research community that fruit bats could be a primary natural reservoir for EBOV (Pourrut et al. 2009). In a large-scale survey conducted at Gabon and DRC between 2003 and 2008, anti-EBOV antibodies could be detected from six bat species (Pourrut et al. 2009).

Multiple outbreaks of RESTV occurred between 1989 and 1996 in captive monkeys in the Philippines, the United States of America, and Italy. During these outbreaks, three animal handlers developed anti-RESTV antibodies without clinical symptoms. One became infected during necropsy on a monkey as evidenced by viremia and seroconversion. An additional five monkey handlers in the Philippines developed anti-RESTV antibodies without significant illness (Miranda & Miranda 2011). Serological studies on animal handlers from monkey facilities and people with occupational exposure to pigs in the USA, Italy, and the Philippines, from 1989 to 2009 indicated that 9 (2%) out of 662 persons had detectable RSTV-specific IgG antibodies (Rollin et al. 1999; WHO 2009; Miranda & Miranda 2011).

The RESTV infection was also identified in the Philippines (2007-2008) and China (2011) along with co-infection of PRRSV (Barrette et al. 2009; Pan et al. 2014). In the Philippines, six workers in contact with infected pigs had anti-RESTV IgG antibodies, suggesting that pigs can transmit RESTV to humans. Transmission from pigs may be due to direct contact with body fluids and respiratory routes (WHO 2009). Serum samples from wild-caught bats during 2008–2009 in the Philippines revealed anti-RESTV antibodies in *R. amplexicaudatus* bats. These antibody positive bats were captured near to the area where REST infections were detected in cynomolgus monkeys and swine (Taniguchi et al. 2011). In another study, RESTV RNA was detected in oropharangeal swabs taken from *Miniopterus schreibersii*, and the presence of the virus had also been identified in additional sympatric taxa (*M. australis*, *Cynopterus brachyotis*, and *Chaerephon plicata*) and additional locations (Puning Cave). Also, the presence of anti-RESTV antibodies was confirmed from *Acerodon jubatus* and *Pteropus vampyrus* suggesting that RESTV infection is widespread in bats (Jayme et al. 2015).

In a seroprevalence study conducted in the dog population during 2001-2002 outbreaks in Gabon, anti-EBOV virus antibodies could be detected from a significant number of samples which were collected from the human outbreak area. However, antibodies were detected in 2 out of 102 dog samples which were collected from the non-exposed country (Allela et al. 2005). Neither Ebola-like illness nor virus could be identified from dogs, and the validity of the immune assay for canine samples has been questioned (Osterholm et al. 2015). During ecological studies in the Central African Republic (CAR) in 1998, EBOV RNA could be detected from mice (*Mus setulosus* and *Praomys* spp.) and a shrew (*Sylvisorex ollula*) (Morvan et al. 1999).

3.2. Role of plants and food materials in transmission of EBOVs

Apart from the transmission of the virus from reservoir animals and contaminated meat, plant food products may also act as a source of infection to the susceptible population (Mann et al. 2015). A two-year-old child was identified as index case who was playing with mangoes partially eaten and dropped by bats (Coen and Henk, 2015; Mann et al. 2015). There are also reports of cases of Ebola infection and diseases acquired by eating fruits half-eaten by bats (Leroy et al. 2007). This may be due to the shedding of virus in the saliva of bats (Amman et al. 2015). It is evident from the study conducted in which plants experimentally inoculated with the virus supported the replication of virus with or without the development of visible lesions (Swanepoel et al. 1996; Okware et al. 2002). The first victim of 2007 outbreak in the DRC was known to purchase fresh bat meat before developing the disease (Mann et al. 2015). But contradictory opinion occurs regarding the role of bushmeat as a source of infection (Mufunda et al. 2016). There is only indirect evidence available in this regard, and more needs to be investigated to confirm whether plants, plant products, and other food material can contribute towards spreading the disease. If this is so, then it is a serious concern as it would make the disease control more difficult. Furthermore, there is also fear that it might act as a weapon for bioterrorism (Maras & Miranda 2016). Capacity to transmit the infection through food products depends on the ability of the virus to survive in the atmosphere for a certain period. It has been found that EBOV can survive up to a period of three weeks at a lower temperature in the atmosphere (Piercy et al. 2010).

4. **Ebola Virus Disease (EVD)**

EVD has an acute clinical outcome with signs like nausea, vomiting, diarrhea, pain in the abdomen, muscles, and head; there is also the loss of appetite and enlargement of lymph nodes (Feldmann & Feldmann 2014; Olival et al. 2013). The incubation period ranges from 1 to 10 days, while exceptions may reach up to 21 days. Hemorrhages and sudden rise in temperature are the two common clinical signs noticed and other signs of chest pain, difficulty in breathing and swallowing. Blood may be noticed in feces and urine; there may also be coagulation problem during bruises or damage to blood vessels (Jahromi and Mood, 2015). Bleeding from nostrils, gums, vaginal mucosa, and gastrointestinal (GI) tract have been a common sign in 40-50% of the cases, and maculopapular rashes can be noticed in 50% of cases. Bleeding begins within 7 days from the start of initial clinical symptoms, and it may be noticed both internally as well as subcutaneously. Hemorrhages may range from petechiae, purpura, and ecchymoses; hematomas can also occur due to bleeding in the subcutaneous tissue. Death can occur within 7-16 days from the start of clinical symptoms and occurs due to several organ dysfunctions and severe bleeding (Takada & Kawaoka 2001; Hoenen et al. 2006). Patients who survive from EVD will exhibit organ swelling, orchitis, and pain in the joints. An overview of EBOV transmission, pathogenesis and clinical signs is depicted in Figure 5.

The major route of infection is through the mucosa or skin from where it reaches the macrophages, monocytes, and dendritic cells, leading to spread of the virus to regional lymph nodes, liver, and spleen. Macrophages and monocytes stimulated by EBOV release cytokine storm thereby damaging tissues and blood vessels (Olival et al. 2013). Death occurs due to blood loss and/or coagulation (Casillas et al. 2003). Coagulopathy occurs due to thrombocytopenia, loss of anticoagulant protein C, destruction of clotting factors and also due to the destruction of fibrin. Damage to blood vessels causes disseminated intravascular coagulation as well as renal failure (Nyamathi et al. 2003; Geisbert et al. 2003). Antibodies developed against EBOV binds with the complement Clq and reaches to the binding sites on dendritic cells and macrophages, leading to damage of these cells.

Lesions of EVD include extensive hemorrhages of the mucosa, necrosis of different organs like liver, kidney, testes and ovaries. Necrotic foci with inflammatory cells can be noticed in hepatic lobules, and there may be multinucleated syncytia formation in the hepatic
cells. Necrosis of red pulp and fibrin deposition are the characteristic lesions noticed in the spleen. Splenic macrophages reveal big, acidophilic particles in their cytoplasm which are similar to intracytoplasmic inclusion bodies. GI tract shows mononuclear infiltration into the submucosa and lamina propria. Mild emphysema, edema in the terminal alveoli and stasis of blood can be noticed in the lung parenchyma (Wyers et al. 1999).

The clinical features observed in different EBOV and SUDV outbreaks were almost similar. The disease caused by EBOV and SUDV is known as “Ebola virus disease” (EVD) – the term "Ebola hemorrhagic fever" (EHF) was previously used. In EBOV outbreak in Zaire in 1976, the major clinical manifestations observed included a severe sore throat, maculopapular rash, intractable abdominal pain, and bleeding from multiple sites principally the gastrointestinal tract (malena, haematemesis, mouth/gingival, vaginal epistaxis, injection sites/scarification) (Report of an International Commission 1978). More detailed clinical manifestations were reported in SUDV outbreaks in Sudan in 1976. In this outbreak, EHV was a unique clinical disease with a CFR (53%) and a prolonged recovery period in those who survived. The clinical signs observed were fever, headache, and pains in joint and muscle, diarrhea, vomiting, chest pain, pain and dryness of the throat, rash and hemorrhages (Report of a WHO/International Study Team 1978). In contrast to the above outbreaks, clinical manifestations observed in 2013–2016 West African outbreaks differed mainly due to less hemorrhage (less than 5% of patients before death). There were 4 clinical phases: (i) an early febrile phase (onset 0–3 days) – fever (up to 40 °C), malaise, fatigue, body aches etc., (ii) gastrointestinal phase (onset 3–10 days) – epigastric pain, nausea, vomiting, diarrhea, persistent fever, asthenia, headache, conjunctival injection, chest pain, abdominal pain, arthralgias, myalgias, hiccups, delirium etc., (iii) shock or recovery phase (onset 7–12 days) diminished consciousness or coma, rapid thready pulse, oliguria, anuria and tachypnea, and (iv) late complications phase (onset ≥10 days) associated with gastrointestinal hemorrhage, secondary infections, meningoencephalitis, and persistent neurocognitive abnormalities (Chertow et al. 2014; Chertow et al. 2016). The convalescent state is complicated by uveitis and viral shedding through ocular discharge (Chancellor et al. 2016; Varkey et al. 2015). In a study conducted by the WHO on EVD in West Africa, the most common symptoms observed included fever, fatigue, vomiting, diarrhea, loss of appetite, headache, abdominal pain and hemorrhagic symptoms (WHO Ebola Response Team 2014, 2016).

### 5. Post-Ebola Syndrome

Of note, thousands of patients have survived during the Ebola outbreaks and have shown symptoms that persisted or developed after hospital discharge, which could be due to the persistence of the virus in the immune privileged sites, such as brain and retina of the host (Fallah et al. 2016). Defective interfering particles of the virus might be responsible for virus persistence and chronic condition due to EBOV infection as observed under in vitro condition in cell culture condition (Calain et al. 2016). To note, this virus persists in the body longer than suspected by the scientists. However, there is a paucity of sequelae of data regarding post-Ebola syndrome (PES). In an SUDV outbreak in Uganda in 2000, 60 out of 257 people survived and the PES complications observed included abdominal pain, loss of vision and hearing, impotence, bleeding, psychological problems, and general weakness (Wendo 2001; Shantha et al. 2016). During 1995 EBOV outbreak in Kikwit DRC, the clinical signs observed among 19 survivors were arthralgia, ocular disease, parotitis, unilateral orchitis, hearing loss or tinnitus and pericarditis (Kibadi et al. 1999; Kibadi et al. 1999). In the same outbreak, 29 Ebola convalescents and 152 household contacts (HHCs) were monitored for up to 21 months. Arthralgias and myalgia were the most commonly reported symptoms among convalescents. The other clinical signs observed during the first six months of follow-up were abdominal pain, extreme fatigue, and anorexia more frequently than did HHCs, whereas
fever, headache, diarrhea, dyspnea, hiccups, and hemorrhage were the same in both groups (Figure 5, Rowe et al. 1999).

In a recent outbreak occurred in Guinea in 2015, a survey was conducted on 105 EVD survivors using a standard data collection form revealing anorexia and arthralgia frequently (Qureshi et al. 2015). Other clinical signs reported in different outbreaks included pain, weakness, hearing difficulty, and mental disturbances (Report of a WHO/International Study Team 1978; Okware et al. 2002). Recently, a very systematic cross-sectional survey of the symptoms of all survivors was conducted from the Ebola Treatment Unit (ETU) at Freetown, Sierra Leone. The unit treated 88 EVD confirmed cases between December 1, 2014, and March 31, 2015, of whom 44 survived. The symptoms observed in the survivors included musculoskeletal pain (70%), headache (48%), and ocular problems (14%). A total of 117 separate complaints were reported, 31 patients (70%) had musculoskeletal pain, 21 (48%) had headaches, and 6 (14%) had ocular problems. Twenty-six (59%) of the 44 survivors reported other symptoms like cough, abdominal pain, chest pain, itching, insomnia, fever, loss of appetite, labored speech, epigastric pain, rash, weight loss, hiccups, increased appetite, chest pain, sneezing, diarrhea, vomiting, left-sided weakness with facial nerve palsy, breathlessness, rash, dry flaky skin, earache, fever blister/cold sore, left scrotal swelling, nasal congestion, and tremors (Scott et al. 2016).

## 6. Pathogenesis

The EBOV viral proteins are involved in various stages of the pathogenesis (Figure 5), viz. VP35 is essential for ribonucleoprotein complex (RNP) production, important for viral replication and transcription, and VP35 blocks the signaling of type I interferon (Muhlberger et al. 1999; Basler et al. 2000; Basler et al. 2003). By antagonizing the production of interferon, host cellular IFN-α/β response obstruction and inhibition of antiviral protein production as a result of dsRNA of the virus are the mechanisms involved in pathogenesis caused by VP35 protein (Basler et al. 2003; Wong et al. 2014a). VP40 matrix protein is involved in budding from the host cell and also involved in the formation of the virus-like particles (VLPs) (Harty et al. 2000; Jasenosky et al. 2001; Noda et al. 2002). Envelope GP and other viral proteins may have a role to inhibit immune mechanism thereby increasing the virulence of the virus (Takada & Kawaoka 2001). Lately, scientists have tried to elucidate the exact molecular pathogenesis of this virus (Shi & Shen 2013) where they found that caspase 8, FADD-like apoptosis regulator (CFLAR), dystroglycan 1 (DAG1) and tissue factor pathway inhibitor (TFPI) were inhibited in human umbilical vein endothelial cells expressing EBOV GP.

The virus enters through mucosal surfaces, cut or lesions of the skin, or direct transfer. Cells like monocytes, macrophages, dendritic cells, hepatocytes and endothelial cells are the targets of EBOV (Feldmann et al. 1996; El Sayed et al. 2016). Epithelial, endothelial fibroblasts, hepatocytes, and adrenal gland cells are also infected (Olejnik et al. 2011). Following infection, mononucleated cells are the preferred by EBOV for replication in initial stages and a rapid viremia is generated (Olejnik et al. 2011). EBOV destructs lymphocytes and monocytes and induces cytokine storm and coagulation anomalies. Also enhanced activation of B and T cell is evident with a higher magnitude of inflammatory cytokines (McElroy et al. 2015). Evidence suggests the pathological manifestations by EBOV via releasing immune mediators which are more critical. There is a rapid multiplication of virus inside the host, and it escapes the host defense mechanism. A major target of EBOV is antigen presenting cells mainly CD16+ monocytes which are not activated in an acute episode of EVD (Lüdtke et al. 2016). Both virus-encoded and host proteins are responsible for pathogenesis, releasing inflammatory cytokines like tumor necrosis factor alpha (TNF-α), IFN-α, IFN-γ, interleukin-2 (IL-2), and IL-10 that are responsible for massive hemorrhage and death (Villinger et al. 1999). This strongly activates the immune system, and mainly
interferon-related genes are upregulated (such as ISG15, OAS1, etc.) (Caballero et al. 2016). Antibodies can be detected after 6 days' post infection and can be detected up to 90 days. Reports show that IgG response can be detected for around 400 days in animals while it can persist around 10 years in human sera (Ksiazek et al. 1999). Though antibodies can be detected in the serum against glycoproteins, they turned out to be not protective against infection and also failed to inhibit replication of the virus in cell culture. Passive administration of antibodies to animals prevented clinical manifestations but has not prevented death which hypothesizes that immunity is not mainly through antibodies but could also be cell-mediated. There is no report till to date regarding the re-infection of EBOV in a person who suffered earlier infection, showing that there is lifelong immunity against EBOV.

During the clinical course, the predominant clinico-pathological findings included hypoalbuminemia, hyponatremia, hypokalemia, hypocalcemia, and hypomagnesemia. Aminotransferase activity peaked at a median of 9 days after the onset of illness (Uyeki et al. 2016).

## 7. Experimental Animal Inoculation

Several experimental studies have been conducted on the animal-to-animal EBOV transmission using different animal species. One of the studies demonstrated transmission of EBOV from inoculated rhesus monkeys to control monkeys caged in the same room. They postulate the possible mode of transmission through aerosol, oral, or conjunctival exposure to virus-laden droplets (Jaax et al. 1995). However, there is a possibility for transmission through spitting and throwing feces by the monkeys (http://www.cdc.gov/vhf/ebola/transmission/human-transmission.html) or through routine animal husbandry practices (Osterholm et al. 2015). In another study, the transmission of EBOV from infected pigs to un inoculated caged macaques was demonstrated when they were housed together (Weingartl et al. 2012). The route of inoculation also might play an important role in the transmission as in an experiment when monkeys were inoculated with EBOV via intramuscular route; no transmission occurred in uninoculated control monkeys (Alimonti et al. 2014).

Few studies have examined the role of route of exposures in transmission. One such study demonstrates that EBOV infection through oral, conjunctival or intramuscular routes can cause the illness in rhesus monkeys (Jaax et al. 1996). In another study, aerogenic route of infection was demonstrated with different doses of antigen (Johnson et al. 1995). Another study investigated the replication, pathogenicity, shedding, and transmission efficacy of EBOV in pigs, wherein, following mucosal exposure, EBOV replicated in pigs, mainly in the respiratory tract, and induced severe lung lesions (Kobinger et al. 2011).

## 8. Clinical and Laboratory Diagnosis: A complex Issue!

During several Ebola outbreaks since 1976, EVD has emerged as a direct challenge to manage (CDC 2015; Mattia et al. 2016; Racsa et al. 2016). If detected early, it gives good prognosis and lives can be saved (Okware et al. 2015). The diagnostic and therapeutic advances following the most recent outbreak have improved prognosis to some extent (Butler 2014; Gilbert 2015).

In a September 2014 statement, the WHO said, “The Ebola epidemic ravaging parts of West Africa is the most severe acute public health emergency seen in modern times. Never before in recorded history has a biosafety level four pathogen infected so many people so quickly, over such a broad geographical area, for so long” (WHO 2014). This grim situation triggered a search for reliable and sensitive laboratory tests to identify Ebola and diagnose EVD as early as possible after exposure (Ayukekbong 2016; Uyeki et al. 2016). Historically, Dr. Ngoy Mushola recorded the first clinical description of the EVD in his clinical log during the Zaire outbreak: “The illness is characterized by a high temperature of about 39°C,
hematemesis, diarrhea with blood, retrosternal abdominal pain, prostration with 'heavy' articulations, and rapid evolution of death after a mean of three days" (Olupot 2015; Shah et al. 2015). At present WHO/CDC define that "any illness with onset of fever and no response to treatment for the usual causes of fever in the area, along with at least one of the following signs: bloody diarrhea, bleeding from gums, bleeding into skin (purpura), and bleeding into the eyes and urine", for a suspected Ebola case.

To proceed for diagnosis, first of all, patients' history such as travel, occupation, and exposure to wild animals must be recorded. The most difficult task in EVD exposure is to specifically and differentially diagnose the EVD at the onset of disease symptoms. The EVD infected individuals are presented clinically as very sick with muscular and abdominal pain, cramps and hiccups with vomiting and diarrhea, severe headache, weakness followed by bruising and petechial bleeding. With the severity of disease, mucosal and frank bleeding occurs in the form of epistaxis and bloody stool. The EVD symptoms are quite similar with several other infectious diseases such as Typhoid fever, Lassa fever, Meningococcemia, Malaria, Influenza, Measles, Marburg virus disease, Shigellosis, Leptospirosis, Yellow fever, fulminant viral Hepatitis and Travelers' diarrhea (Hartman et al. 2010; Uyeki et al. 2016) and thus needs to be differentially diagnosed. The CDC/WHO recommend for immediate clinical risk analysis for EBOV infection in any person with the symptoms described above. Currently, there is no specific treatment existing except that an early diagnosis of EBOV will ensure fast supportive care before the development of irreversible shock and necessary preventive measures and patient isolation can be instituted. Since major Ebola outbreak in 2014, researchers are tirelessly working to develop a specific test to confirm Ebola exposure before and after the onset of clinical symptoms. This initiative is also critical as most of the body fluids are highly infectious due to the persistent presence of Ebola for a long time. The abrasions on the skin and exposed mucosa are the most likely route of infection after direct contact of EBOV-containing body fluids from an infected person. The WHO/CDC characterized blood, feces, and vomit as the most infectious body fluids. Furthermore, EBOV also been detected in saliva (Formenty et al. 2006; Spengler et al. 2015), breast milk (Nordenstedt et al. 2016), aqueous humor (Varkey et al. 2015), semen, vaginal fluid and urine (Deen et al. 2015; Fischer et al. 2016; Pettitt et al. 2016; Thorson et al. 2016; Chughtai et al. 2016).

After the EVD outbreak in 2014, there is a critical urgency for the availability of robust diagnostic and prognostic tools for infectious diseases, particularly, in the resource-poor areas (Ghani et al. 2015). Diagnostic measures must be employed as soon as clinical symptoms become evident. In the early stage of infection, the presence of EBOV-antigen must be confirmed, followed by antibody detection at the late stage of infection (Olival et al. 2013; Okeke et al. 2014). In the year 1979, the first report on the quantitative measurement of antibodies against EBOV in human sera from N.W. -Zaire was published (Van der Groen & Pattyn 1979). Subsequently, an immunofluorescence focus assay (Truant et al. 1983) and formalin-fixed tissue specimens by enzyme treatment (Kurata et al. 1983) were developed to detect EBOV. Since EBOV equally infect primates, an enzyme immunoassay (EIA) was employed (Ksiazek et al. 1992) for the detection of EBOV in the infected primate tissues. For rapid diagnosis (within 30 minutes), EBOV-antigen detection based on the immune-filtration technique was also developed with limited detection of the Zaire and Sudan species only (Lucht et al. 2007). Ultimately, the confirmatory diagnosis relies on the isolation of virus on Vero or Vero E6 cell lines (Steele et al. 2001; Sanchez 2007; Makino & Kawaoka 2009). However, handling of infected samples and isolation of virus essentially requires BSL 4 infrastructure due to very high infectivity and mortality of lab personnel. Various other diagnostic platforms include electron microscopy, immunohistochemistry and IgG and IgM antigen capture enzyme-linked immunosorbent assay (ELISA) has also been evaluated (Mishra 2014). Serum, plasma, and whole blood have been used with good results
for detection of EBOV using antigen capture ELISA by using a semi-synthetic repertoire of llama single-domain antibodies (Sherwood et al. 2007).

Reverse transcription-polymerase chain reaction (RT-PCR) seems to be a good diagnostic tool for detection of the virus (Drosten et al. 2002; Towner et al. 2004). Given the complex nature of EVD and rapid transmission, it is essential to deploy highly sensitive and specific EBOV detection assays. A reverse transcription loop-mediated isothermal amplification (RT-LAMP) has been developed to detect EBOV within 26 minutes (Kurosaki et al. 2007) without RT-PCR instruments. Recently, RT-LAMP for detecting EBOV directly from blood has been described for the GP gene of EBOV. It detects EBOV within 40 minutes with high sensitivity of detection limit to identify \(2.8 \times 10^2\) pfu/test and \(1 \times 10^3\) pfu/test of EBOV-kikwit and EBOV-makona, respectively (Benzine et al. 2016). Further, using DNA-intercalating dye SYBR green based or TaqMan probe based real time RT-PCR, highly sensitive detection of EBOV antigen is possible (Huang et al. 2012; Liu et al. 2012). A multiplex real-time PCR has also been developed to detect Ebola and Marburg viruses in a single step (Yang et al. 2012). Similarly, several companies around the world are marketing lateral flow like kits that can detect EBOV antigens. Corgenix (Broomfield, Co, USA) and VedaLab (Alenson, France) have developed kits which use a small amount of sample (blood, plasma or urine) from infected persons (Baker 2014).

ELISA tests can be used for detection of antibodies of EBOV though this is not promising as antibodies develop only in the late phase of the infection and a chance for a person to escape the initial phase without death is less (Ksiazek et al. 1999; Mishra 2014). Similarly, there is cross-reaction of antibodies with other pathogens. Therefore, antibody detection tests are less effective for EBOV infection. Specific detection of EBOV in clinical laboratories is currently being carried out largely through the analysis of the virus's nucleic acid (genetic material), using commercial or in-house tests (Dhama et al. 2015a). Although these Nucleic acid tests (NATs) are highly specific they require well-established laboratories and fully trained personnel due to their complex nature. Also, the turn-around time of the final results may take more than 12–24 hours. During Guinea Ebola outbreak, patient’s swab samples were used to detect EBOV by a rapid reverse transcription recombinase polymerase amplification (RPA) assay (EBOV-RT-RPA). This highly efficient assay produced results in 30–60 minutes (Faye et al. 2015; Dedkov et al. 2016). A new test comparable to ELISA is developed by using Fe₃O₄ magnetic nanoparticle (MNP) as a nanzyme probe (Nanzyme-strip) to specifically detect the EBOV-glycoprotein as low as 1 ng/mL and is 100-fold more sensitive than the standard strip method (Duan et al. 2015). The WHO/CDC has officially approved (Feb 19, 2015), the *ReEBOV™ Antigen Rapid Test Kit (Corgenix, Broomfield, Co, USA) as eligible for procurement and to be used in Ebola-affected countries (Dhillon et al. 2015; Flint et al. 2015). The ReEBOV™ Antigen Rapid Test is an immune-chromatographic dipstick immunoassay point-of-care test based on detection of the Ebola protein (VP40 antigen) rather than nucleic acid and can provide results within 15 minutes. The ReEBOV™ Antigen Rapid Test kit can correctly identify about 92% of Ebola infected patients and 85% of those not infected with the virus compared to other available NAT and currently being used in the field (RealStar® Filovirus Screen RT-PCR Kit 1.0, Altona Diagnostics GmbH, Hamburg, Germany). Despite the fact that ReEBOV™ Antigen Rapid Test kit is less accurate, it is easy to perform and does not require electricity. It can, therefore, be used at lower healthcare facilities or in mobile units for patients in remote areas (Broadhurst et al. 2015). It is always better to confirm the data from ReEBOV™ antigen Rapid Test Kit by testing a new blood sample using an approved Ebola NAT. To test the sensitivity and versatility, comparative studies on 11 different procedures has been performed for currently available RT-PCR assays (Nouvellet et al. 2015; Cherpillod et al. 2016). However, the detection limit of ReEBOV™ test is not sufficiently sensitive to identify all the EVOB-suspected cases. Another significant obstacle to the use of these RT-PCR tests is to

18detect novel filovirus species and lineages which have emerged due to rapid mutation rates and genetic diversity of RNA viruses. More recently, a real time RT-PCR assay was developed as a point-of-care test that does not require RNA extraction and thus the entire process completes within 1.5 hours (Zhang et al. 2017).

The chronology of diagnostic tests developed to detect EBOV in various situations has been summarized in Table 3 and the diagnostic kits developed for EBOV detection are presented in Table 4.

In recent times, new nucleic acid sequencing technologies [referred to as ‘next-generation’ sequencing (NGS)] have evolved which provide more efficient and specific viral diagnostics. The technology of NGS provides rapid high-throughput DNA sequence data in large volumes. The NGS has been successfully tested for its efficiency during the 2014 Ebola outbreak in Guinea and Sierra Leone (Gire et al. 2014). This study was able to identify the virus sequence differences and similarities to previous outbreak variants and to elaborate its emergence from the natural reservoir. Further, data helped the researchers to understand how EBOV has moved across African nations during the months of the outbreak (Gire et al. 2014). A new approach for direct detection of EBOV-infection has been tested. This approach is amplification-free and based on the Optofluidic analysis (Cai et al. 2015). Recently, a research team from Marburg, Germany (Krahling et al. 2016), has developed a Zaire Ebola virus (ZEBOV)-specific enzyme-linked immunosorbent assay (ELISA) using inactivated ZEBOV-Makona virus isolate. The ZEBOV-ELISA is highly specific and sensitive with superior reproducibility. This newly developed test is suitable both for the detection of the ZEBOV surface glycoprotein in vaccinated individuals and for detection of specific antibody responses directed against different ZEBOV proteins in EVD patients (Krahling et al. 2016). The possibility of EBOV-encoded miRNA-like fragment as a specific biomarker for early EVD confirmation has also been explored (Chen et al. 2016). The high potential of the recent advances in the field of diagnosis including molecular tools, LAMP, lateral flow assay, biosensors, biochips, microarrays, recombinant protein and nanotechnology based detection methodologies must be exploited to their full perspectives for rapid detection and monitoring of EBOV infections (Kurosaki et al. 2007; Belak et al., 2009; Dhama et al. 2014; Benzine et al. 2016).

### 9. Global EBOV Monitoring and Disease Surveillance: Did We Learned New Lessons?

Due to social conflicts, globalization and climate change, intercontinental migration of humans has been rampant. The uncontrolled movements and demographic transitions demand a serious preparedness for unseen lethal epidemics of uncommon viral diseases. It is no more an exotic infection for any country but has taken the nature of a global problem (Arwady et al. 2015). Thus, surveillance of emerging infectious diseases is extremely vital for the early identification and preparedness of public health threats. Fortunately, particularly after the 2014 outbreak, several new rapid molecular diagnostic assays have become available for precise surveillance of highly infectious diseases in real time manner (Benowitz et al. 2014; Curran et al. 2016).

The last EBOV outbreak in West Africa is a serious warning to the public health system and awakening to the communities for better preparedness to combat such large-scale epidemic. More than 12,000 deaths were reported during 2014 EBOV-epidemics. The EBOV and related viruses are extremely lethal and managing such diseases are beyond our capacity if not prepared (Woolhouse et al. 2015; Walker & Whitty 2015; Jacobsen et al. 2016). New technologies have the capacity to simulate and predict the future epidemics and can direct the public health authorities to respond in a swifter way to infectious outbreaks of all kinds. A multidisciplinary approach has the potential to manage the spread of pathogens and reduce

19the morbidity and mortality. These novel tools can be harnessed for mathematical modeling, risk mapping, phylogenetic analysis, pathogen genome sequencing and rapid on-site diagnosis (Woolhouse et al. 2015). During 2001–2003, a study in Gabon and DRC recommended a method for predicting and possibly prevent human Ebola outbreaks by determining a correlation of wild animal mortality with human Ebola outbreaks (Rouquet et al. 2005). To monitor the disease outbreak effectively, it should be considered to gather data from contaminated raw food products derived from bushmeat and plants also (Mann et al. 2015).

In recent times, internet-based surveillance tools and epidemic simulation based prediction can efficiently facilitate the potential risk assessment and outbreak detection in infectious disease emergencies which can prevent future infectious diseases threats (Christaki 2015). The widespread transmission that occurred in the three most affected countries (Guinea, Liberia, and Sierra Leone) in West Africa and the epidemic’s designation as a “public health emergency of international concern (PHEIC)” insisted the formulation of specific guidelines to efficiently manage the EBOV epidemic. In 2014 CDC issued interim U.S.A. Guidance for “Monitoring and Movement of Persons with Potential Ebola Virus Exposure” to guide CDC staff and public health partners engaged in the response. The guidance provided by public health authorities and collaborative agencies with recommendations for monitoring travelers who may be potentially exposed to Ebola and therefore authorities can enforce the restriction in movement. In humans, EBOV spreads through direct contact with infected body fluids (blood, vomitus, feces, semen, vaginal fluid, and urine) with broken skin or exposed mucous membranes, and possibly through sexual contact. Because of the risk for importing disease from infected travelers, in October 2014 the CDC recommended that “all travelers to the United States of America from Ebola-affected countries receive enhanced entry screening and post-arrival active monitoring for Ebola signs or symptoms until 21 days after their departure from an Ebola-affected country” (CDC 2014, 2015). On February 19, 2016, the CDC-guidance expired (CDC 2016) as Guinea was declared free of EBOV transmission (after more than 45 days had passed since the last EBOV case reported and the human-to-human transmission ended).

New technology platforms are now being employed vigorously for monitoring and surveillance of EBOV (Cross et al. 2016). During 2014 EBOV outbreak, genomic surveillance was performed using ‘next-generation’ sequencing [NGS] which helped in identifying the EBOV origin and transmission (Gire et al. 2014). Further, several start-ups are seriously designing the efficient and portable devices for EBOV genomic surveillance. In this line, a novel portable Nanopore DNA sequencing instrument (Pennisi 2016; Quick et al. 2016) was developed and used in EBOV genome sequencing. The Nanopore real-time genomic surveillance system was transported in standard airline luggage to Guinea and used for real-time genomic surveillance of the ongoing epidemic during April 2015. Once received an Ebola-positive sample, this portable system produced results in less than 24h and genomic sequencing within 15–60 minutes. Such innovations warrant the use of genomic surveillance in resource-limited settings in a real-time manner. Rapid monitoring of potential outbreaks, using such portable sequencer, is now a reality in remote areas (Quick et al. 2016). Further, a handy, portable and compact (13.5cm × 8.5cm × 2.5cm) infection monitoring system was recently introduced to monitor mass gathering places such as airports (Sun et al. 2015). This highly mobile and hand-held system can rapidly screen the non-contact vital signs using two integrated sensors. One sensor which is a 24-GHz microwave radar can measure the heart and respiration rates and; another sensor is a thermopile array for capturing facial temperature. After collecting vital signs data, the system detects infected individuals using a linear discriminant function (LDA). Such devices can be very useful in providing real-time updates and essential resources for managing current Ebola outbreak with the help of Global Ebola

20
Response Monitoring and Mapping System (WHO) and United Nations Mission for Ebola Emergency Response (UNMEER) (Sun et al. 2015).

The global revolution of the internet and social media are a very powerful tool in disease surveillance. Short message service (SMS)-based system has been established to facilitate the active surveillance of individuals who are either returning from EVD-affected countries or in contacts with EBOV-exposed local confirmed cases (Tracey et al. 2015). This SMS-system collects the information on symptoms and temperature twice daily and communicates to the public health authorities (Tracey et al. 2015). During the August 2015 epidemic of EBOV in Sierra Leone, toll-free, nationwide Ebola call centers were established by Sierra Leone's Emergency Operations Center. This “alert” system proved to be very effective in providing health education about EBOV to the callers and boosting public reporting to public health officials about possible Ebola cases and deaths (Miller et al. 2015). Based on a study in Southern Sierra Leone, the effectiveness of the cell phone messaging technology has been proved to be highly effective as a rapid communication tool for community epidemic surveillance from peripheral health care facilities to higher levels (Jia & Mohamed 2015). One recent report suggested that the utility of Hybrid Information Network (HIN) in facilitating the quicker public health measures may be possible, by integrating both hierarchical and formalized command control-driven and community-based, or ad hoc emerging networks, which can efficiently employ social media to facilitate the robust preparedness for the potential outbreak and improving the capability of EBOV-identification and diagnosis (Hossain et al. 2016).

A community-based effort in Liberia has been very successful to defeat the EVD in recent times (Shrivastava et al. 2016). These coordinated efforts involved strong leadership and clear vision by the policy maker by declaring the disease as a public health priority to all the concerned departments; actively involving the community leaders by developing a sense of need among them; increasing the level of faith and neutralizing any rumors about the disease among the local people - by building see-through walls in treatment centers so that people can watch the proceeding or developing mechanism to respond promptly to transport calls or by sensitizing the health professionals to build confidence among local residents; strengthening of the treatment facilities (by creating medical units, treatment centers, logistics support) and laboratory network by establishing five dedicated Ebola laboratories; courageous attitude of the local volunteers; enormous support from the international community; and well-coordinated national and international response (Shrivastava et al. 2016). In response to the 2014 EVD-outbreak, a common international code, ICD-10-CM code “A98.4” (International Classification of Diseases -10th Version-Clinical Modification) has been implemented in all-in all healthcare institutions for Ebola bio-surveillance specifically. This healthcare coding system will facilitate real-time tracking of EBD-outbreaks (Chabra 2015). Since EBOV is a mutating RNA virus, a rapid monitoring system using aptameric sensing technology has been proposed for pandemic surveillance (Acquah et al. 2015). Computer simulation and mathematic modeling have been proposed to quantify the West Africa Ebola epidemic (Chretien et al. 2015). International Society of Travel Medicine (ISTM) and CDC collectively operate a global GeoSentinel surveillance network of travel and tropical medicine clinicians that collects de-identified demographic, diagnostic, and travel information on ill travelers who cross international borders. GeoSentinel is composed of 63 clinical sites in 29 countries on 6 continents. Through the efforts of GeoSentinel Surveillance Network, differential diagnosis of EVD in travelers arriving from Liberia, Sierra Leone or Guinea was conducted to track the outbreak (Boggild et al. 2015). A similar approach “Surveillance and Outbreak Response Management System (SORMAS)” has also been suggested to support the control of the EVD-outbreak (Fähnrich et al. 2015). In USA, to understand and address the doubts and concerns of the citizens’ life twitter chat was
conducted thus such platforms can be very effective in future to collect and disseminate the information in future at the time of such outbreaks (Lazard et al. 2015).

Finally, the lessons learned from such epidemic with high mortality emphasize that the efficient surveillance system and coordinated global efforts involving every social and political leader can be a best possible scenario to prevent such deadly viral disease (Carney & Weber 2015). Though CDC/WHO have officially declared the end of the Ebola outbreak, the residual risks of Ebola reintroduction or re-emergence are still haunting the human population worldwide. According to the Ebola Situation Report - 16 March 2016, the WHO has strongly recommended the implementation of superior EBOV-monitoring systems in Guinea, Liberia, and Sierra Leone to alert authorities for the EVD-clinical cases or EVD-related death. Developed rich nations must effectively implement a high-quality, robust emergency response system, and provide the unconditional support to the establishment of sustainable health-care services in West Africa (Quaglio et al. 2016). Only in this way we together can avert the next humanitarian crisis due to EBOV or related viral diseases.

### 10. Prevention and control

Currently, there is no robust and potent vaccine to control this lethal infection leaving prevention to be the best way to escape from infection. Monitoring, surveillance, isolating infected persons and treating them separately, travel to the endemic area during outbreaks should be discouraged, are the limited and effective ways to prevent further spread of infection (Tambo et al. 2014). For EVD-outbreak prevention, a new organization named as “United Nations Mission for Ebola Emergency Response (UNMEER)” has been created during the 2014 outbreak of EBOV in Africa.

Various strategies like reporting of EVD outbreak, proper treatment to infected persons, proper burial of dead persons, disinfection of the areas and proper planning to prevent further spread are the measures taken to control EVD at national and international level (Millman et al. 2016). Hospitals, where infected personals are treated, should also be disinfected along with sterilization of all instruments that come in contact with the infected persons. Both incineration and autoclave can easily destroy the virus. Fecal wastes should also be deactivated to destroy the virus. Places, where there are no proper facilities for incineration or autoclave should pack the materials in a leak proof triple layered container and these should be transported to other places for sterilization and disposal.

There is an urgent need for creating awareness among the public about this deadly disease so that spread of Ebola can be prevented (Salman et al. 2017). Proper monitoring at airports and public places where there are all possible chances for spread of infection is of utmost importance (Brown et al. 2014; Read et al. 2015). People who fall ill should be kept under the scanner for at least 21 days to rule out the infection. Health care workers and doctors are also at higher risks as they come in direct contact with the infected persons. Hence they should wear all protective wears like gowns, gloves, face mask, goggles and respirators and also should follow barrier nursing. Collected samples for diagnosis should be transported under high precaution and handled only in the BSL4 lab. Climate change has a huge role in the dissemination of various diseases, and EBOV can also be transmitted to non-human primates due to movement of bats as a result of climate change (Dhama et al. 2013a). Since bushmeat has been found to be the potential and important source of many epidemics in the past, it is important to monitor their illegal export to the naïve countries (Mann et al. 2015).

### 11. Vaccines

Although EBOV was discovered in 1976 until now, there is no effective vaccine or post-exposure treatment available. After the recent outbreaks, the EVD management plan relies majorly on supportive therapies. Since 2014 EBOV-outbreaks, the scientific
community is working tirelessly to develop immunization regimen to control virus spread (Geisbert & Jahrling 2003). The prime-boost approach of DNA vaccine followed by adenovirus vectored ZEBOV GP has resulted in good cellular and humoral response in cynomolgus macaques during experimental studies. DNA vaccines are also in phase I of clinical trials. The glycoprotein of EBOV has been a good target for eliciting humoral immunity and surface protein targeted human monoclonal antibody has provided a ray of hope for a better vaccine (Nyamathi et al. 2003). These DNA vaccines contain glycoprotein (GP) and Nucleoprotein (NP) of any of the 5 strains of EBOV. It is administered intramuscularly and followed by booster it generated an immune response in 70% of volunteers with poor T cell response. DNA vaccination may be further improved by vaccinating through electroporation, injecting higher quantities, or prime boosting with other antigen delivery platforms (Sarwar et al. 2015). The problem in developing a peptide-based vaccine to protect from this infection is due to the vast similarity that exists between it and human peptides. So, it is necessary to identify certain peptides that are unique and conserved in EBOV to develop peptide-based subunit vaccines (Kanduc 2016). Experiments with lab animals like mice and guinea pigs revealed that DNA vaccine protected them against challenge with wild type Mayinga 1976 strain of ZEBOV (Gulland 2014).

Recombinant virus vectors are a good choice of antigen delivery as they induce the cell-mediated immune response, due to intracellular expression and processing of antigen. Of note, Ad5 serotype is a popular adenovirus-based vector used for such purposes due to ease in genetic manipulations and to grow to high titers. In this approach, an El region of adenovirus is replaced with GP of Ebola, and in non-human primates, it offered 100% protection in a single dose. However, pre-existing neutralizing antibodies limits the utility of Ad vectors (Ledgerwood et al. 2010). Immunization in individuals having anti-Ad5 antibodies before vaccination showed reduced cellular and humoral response (Zhu et al. 2015). Adenovirus-based vaccine (cAdVaxE) is a bivalent vaccine against GP of ZEBOV and SEBOV which has protected 100% of mice against challenge (Wang et al. 2006). Another virus used include Vesicular stomatitis virus (VSV) which is an enveloped single-stranded RNA virus. The recombinant VSV is produced by replacing GP of VSV with GP of EBOV. When recombinant VSV-based filovirus vaccine is used in prime-boost approach, it can give good protection. The VSV vectored vaccine expressing GP of filovirus has protected non-human primates against 3 species of filovirus and Marburg virus (Mire et al. 2013). Furthermore, it has entered in phase III efficacy trial in Guinea. It conferred protection in non-human primates using different routes of immunization like mucosal, oral or intramuscular (Geisbert et al. 2009) with advantages like transient viremia and self-limited febrile illness (Sridhar, 2015). Likewise, the recombinant human parainfluenza virus 3 (HPIV-3), a single-stranded, negative-sense RNA virus, has been used as a vaccine candidate owing to its ability to accommodate several GPs in the viral backbone (Skiadopoulos et al. 2002). Intranasal immunization with EBOV GP expressed with replication-competent HPIV 3 showed better protection against challenge with a higher parental dose of EBOV (Yang et al. 2008). Other vectors like cytomegalovirus (CMV) based vaccine protected mice against EBOV, and this has the potential to be used against EVD in humans (Tsuda et al. 2015).

EBOV-virus like particles (VLPs) produced by expressing GP and VP40 in mammalian cells can also stimulate protective immune response making better vaccine options (Cazares et al. 2016). GP based fusion vaccines developed by EBOV GP fused with Fc fragment of human IgG1 adjuvanted with poly-ICLC has shown to induce a strong humoral immune response and protect guinea pigs against lethal EBOV challenge (Konduru et al. 2016). Subunit vaccine contained GP protein expressed in Autographa californica multicapsid nucleopolyhedrovirus, a baculovirus expression system in SF9 insect cells. Infected cells are lysed and adjuvanted with Matrix-M, a phospholipid constituent of synthetic cholesterol and saponin. This vaccine generated higher cellular and humoral activity
(Lovgren et al. 2011; Martins et al. 2016). With the use of reverse genetics, defective EBOV without VP30 was found 100% effective in protecting guinea pigs and mice (Hoenen et al. 2012). Replication-defective vaccine by deleting VP30 has also been considered as an effective vaccination strategy to prevent the infection (Saphire 2015).

Although WHO reported two vaccine candidates, namely chimpanzee adenovirus-based, and VSV-based EBOV vaccines, as are under phase III clinical trials, there is a need to search for a safe and effective vaccine (Gulland 2014; Cohen 2014). Heterologous VSV-and Ad5-vectored Ebola vaccine, GamEvac-Combi was used in a Phase I and II trial which induced both humoral and cellular immunity in all the 84 volunteers (Dolzhikova et al. 2017). Presently, seven vaccines have entered clinical trials (Chappell & Watterson 2017). Scientists from the University of Texas, USA developed a new EBOV inhalant vaccine that provides 67-100% protection in non-human primates. Most recently, a recombinant VSV-Zaire ebolavirus vaccine (rVSV-ZEBOV) is being tested under “Sierra Leone Trial to Introduce a Vaccine against Ebola (STRIVE)” program. This promising candidate vaccine, which is in phase 2/3 human trial, is the result of collaborative efforts between the College of Medicine and Allied Health Sciences (COMAHS), University of Sierra Leone, the Sierra Leone Ministry of Health and Sanitation (MoHS), and the CDC. This new vaccine is based on Zaire ebolavirus immunogen and can only protect against Zaire ebolavirus, the major agent behind EVD in West Africa (CDC-STRIVE 2015; Folayan et al. 2016). Progress and advances in the area vaccines and vaccination comprising of edible vaccine, peptide vaccine, DNA vaccine, immunomics based vaccine and others need to be optimally utilized for designing and developing effective vaccines along with exploiting immunomodulatory approaches employing novel adjuvants and vaccine delivery systems for improving the efficacy of the EBOV vaccines (Dhama et al. 2013b; Delany et al. 2014; Sarwar et al. 2015; Singh et al. 2015).

## 12. Treatment

The recommended symptomatic treatments include anti-inflammatory drugs for fever and pain, and intravenous fluids to maintain body osmotic balance. As the kinome analysis revealed the involvement of mitogen-activated protein kinase (MAPK), phosphoinositide 3-kinases (PI3Ks), and calcium/calmodulin kinases in EBOV infectivity, the PI3Ks inhibitor LY294002 and CAMK2 inhibitor KN-93 were tested and found effective in reducing EBOV infection in Vero E6 cells. The p38 MAP kinase has a role in activating the signaling cascade leading to produce interleukins and TNFα. The p38 MAPK inhibitor like SB202190 helps in ameliorating EBOV mediated cytokine storm in human monocyte-derived dendritic cells (Johnson et al. 2014). Artemisinin, an antimalarial drug is also beneficial regarding reducing the acute inflammatory response (Ho et al. 2014). FTY720 is a drug used to treat patients with autoimmune disorders and prevents transplant rejection. Its sphingosine analog AAL-R helped in limiting the inflammation in mice challenged with H1N1 influenza virus (Walsh et al. 2011).

As of now, to kill EBOV, we don't have a specific antiviral drug. Many antiviral drugs are being tested. Therefore, the current treatment relies only on supportive therapy to revive. Recently, it has been discovered that antibodies can effectively block the entry of this virus so it can be employed as therapy for control of Ebola infection in humans (Corti et al. 2016). There is a search for a new drug to control lethality caused by this virus (Choi & Croyle 2013; Kilgore et al. 2015). Clomiphene and Toremifene are estrogen receptor drugs which were found to inhibit EVD infection in mice. These drugs inhibit virus entry and also inhibit EBOV fusion to host cells (Johansen et al. 2013). Ion channel blocking agents like amiodarone were reported to inhibit entry of EBOV into the cells (Gehring et al. 2014). These drugs combined with antiviral drugs can be candidates for the treatment of EVD in

24
places where medical facilities are less. Recently, a group of researchers has tested the anti-
flu drug Favipiravir in EVD treatment (Nagata et al. 2015; Sissoko et al. 2016) which
indicated its use in patients with medium to high viremia. However, it turned out to be not
effective in the case of very high viremia (Sissoko et al. 2016). Convalescent whole blood
(CWB) and convalescent plasma (CP) therapy intends to transfuse the blood/plasma from
diseases survivors and helps in neutralizing and reduce virus load, till the time patients own
immune system can destroy the invader (Van Griensven et al. 2015). Up to 10ml/kg BW of
CP can be obtained on 15 days’ interval from a plasma donor. It has longer shelf-life.
However, CP usage is superior to CWB as their might be a problem of the immune response
against leukocyte antigens and febrile transfusion reactions with the risk of bloodborne
pathogen transmission (Erhabor and Adias, 2011). WHO also recommended the use of whole
blood products and convalescent serum (Gulland 2014) during critical conditions. Before
transfusion, exhaustive testing of plasma is required for residual EBOV RNA, and other
potential viruses like HIV, HBV, HCV, and syphilis. Purified IgG from survivors given to
challenged non-human primates 2 days’ post infection, prevented death, showing the
importance of antibody preparations (Dye et al. 2012). RNAi has been a promising
technology that can be used for the treatment of EBOV (Warren et al. 2010). A pyrazine
carboxamide derivative T-705 (Favipiravir) is in the last phase of the clinical trial and has
yielded satisfactory results in a mouse model against EBOV. Favipiravir neither inhibits the
DNA or RNA synthesis (inhibit RNA-dependent RNA polymerase) in cell lines of
mammalian origin, nor it is toxic to cells (Oestereich et al. 2014). Brincidofovir is a cidofovir
analogue conjugated with lipid and also possess antiviral activity. It has activity against
dsDNA viruses but how it prevents Ebola replication is still a question (Florescu et al. 2014;
Dunning et al., 2016). The broad spectrum anti-viral agent alisporivir which inhibits the host
protein cyclophilin A (CypA) seems to have not much effect on Ebola strains (Makona and
Mayinga) as this virus does not require CypA for replication (Chiramel et al. 2016).
BCX 4430, viral RNA polymerase inhibitor was protective against EBOV in mice
(Fauci 2014; Warren et al. 2014). A protein named double-stranded RNA binding protein 76
(DRBP 76) interacted with viral proteins, with RNAs and with PKR, an interferon-induced
antiviral kinase and also inhibited polymerase function of EBOV (Shabman et al. 2011). The
siRNA technology targeting L-protein of RNA polymerase was used to inhibit EBOV
replication hence can be used as a post-exposure treatment to EBOV infection (Geisbert et al.
2010). The siRNAs and phosphorodiamidate morpholino oligomers, PMO aimed at ZEBOV
RNA polymerase L-protein prevented EVD in non-human primates (Warren et al. 2010;
Geisbert et al. 2010). PMOs tend to target VP 35 and VP 24 that protects from the disease
(Warren et al. 2015). Two new drugs based on antisense PMOs (AVI-6002, AVI-6003) and
lipid nanoparticles (LNP)/siRNA (TKM-Ebola) are in phase I clinical trials. Encapsulation
with LNP gives an effective drug delivery, but they need several dose regimens to achieve
efficacy (Choi & Croyle 2013; Thi et al. 2016). The miRNA technology has also shown good
results during in vitro trials (Sheng et al. 2014). Understanding the mechanism of virus entry
through two-pore channels paved the way in testing certain molecules such as Tetrandrin that
inhibits or halts virus entry into the cells and prevents the disease (Sakurai et al. 2015).
Meanwhile searching for new drugs, some of the drugs have been repurposed for therapy
(Table 5). Chloroquine and its structural analogs including hydroxychloroquine, pamaquine,
primaquine, and plasmaquine, etc. are antimalarial drugs. These are also considered as a
lysosomotropic agent that prevents endosomal/lysosomal acidification. Based on literature data,
such endosomal/lysosomal acidification inhibition by potential therapeutic agents such as
chloroquine are highly effective against viral infections and associated pathologies such as of
EBOV. Intraperitoneal administration of chloroquine protected 100% of monkeys challenged
with EBOV with a dose of 90mg/kg BW, 4h before infection (Olsen et al. 2013). Of note, in
Liberia when the mass administration of the antimalarial drug was adopted, reduction in fever
25

status was observed during Ebola outbreak (Kuehne et al. 2016). Rosuvastatin, atorvastatin, and pravastatin are the statins which reduce inflammatory action and are known to reduce C-reactive protein (CRP) and TNFα. It also hampers cholesterol supported EBOV membrane biosynthesis (Haque et al. 2015). EBOV infection leads to overexpression of the procoagulant tissue factor and endothelial cells also participate in coagulation disharmony characteristic of Ebola infection. In macaque model recombinant nematode anticoagulant protein c2 (rNAPc2) improved survival and provide an additional therapeutic regimen which can be adapted along with several other therapies (Geisbert et al. 2003). Suramin which is used as an anti-trypanosomal agent can also be used as an anti-viral agent to treat EVD and Chikungunya due to its competitive anti-heparin activity. It also can interfere with viral replication and viral entry into the cell (Henß et al. 2016). In the face of unavailability of any suitable treatment procedures for curing the affected persons of this dreaded disease, use of passive immunization (neutralizing antibodies) protocol was opted by transferring the sera from healthy persons which recovered from EVD (Saphire 2013). But it is not sufficient alone to give 100% protection after the onset of viremia (Mire et al. 2016). Additionally, ZMapp which possess humanized mouse antibodies showed promising results in non-human primates (Wong et al. 2014b; Zhang et al. 2014) and consequently, on August 12, 2014, the WHO approved the use of ZMapp for the treatment of EVD. These mAbs are expressed in the glycomodified *Nicotiana benthamiana* and found to be carrying better anti-EBOV efficiency in animal models (Zhang et al. 2014). Bispecific Trojan-horse antibodies that can broadly neutralize all known filoviruses were found to protect mice from multiple EBOV infection (Wec et al. 2016). Recently, attempts have been made to evaluate the anti-Ebola potential of human scFvs against VP40 envelope protein of EBOV that plays an important role in its life cycle. These antibodies were found to block the Ebola virus-like particles in hepatic cells which are transduced with pseudo-Lentivirus particles (Teimoori et al. 2016). This suggests that these transbodies might be effective against EBOV as well interfering with their replication in the cell. Also, it has been found that there exists a conserved guanine-rich sequence within the L gene of EBOV that can form quadruplex RNA and it can be targeted by cationic porphyrin TmPyP4 that leads to inhibition of L gene expression (Wang et al. 2016).

Progress is on the way towards developing effective therapeutics as some lead has been obtained by identifying the weak spots in EBOV structure. Looking into the demand of ZMapp and existing limited manufacturing facilities, an alternative approach such as the use of the plant as bioreactors for the bulk production of ZMaps, have been proposed. Certain host factors are also responsible for virus replication and survival. To identify them insertional mutagenesis can be employed which is a high throughput method where host genes are disrupted. This helps in identifying those genes required for virus replication and can be used as a drug target in future (Cheng et al. 2016). This gives direction to further identification of viral targets or host targets which could be used to control or inhibit this virus disease in the patient. Molecular docking studies of EBOV glycoproteins have been carried out recently which could provide insights into drug designing (Ahmad et al. 2016; M Alam El-Din et al. 2016). Advances in designing effective therapeutic agents and drugs by utilizing emerging and upcoming options viz., cytokines, recombinant proteins, RNAi technology, TLR, avian egg antibodies, nano-medicine, immunomodulators, probiotics herbs, need to be explored to the desired potential to counter EBOV (Geisbert et al. 2010; Dhama et al. 2013c, 2015b; Malik et al. 2013; Gao & Yin 2014; Arntzen, 2015; Streatfield et al. 2015; Yao et al. 2015; Cross et al. 2016; Iqbal et al. 2016).

## Conclusions and future perspectives

Ebola virus disease is one of the most common examples to refer emerging or re-emerging zoonoses globally. Nonetheless, the disease is known since 1976, its severity, exceptional infectious or contagious nature, acquiring pandemic status during 2014-2015, has

raised numerous questions before the health professionals and policy makers. The emergence of Ebola as a threat has cautioned all of us to get ready with arsenals for contending Ebola-like diseases through multidisciplinary and collaborative approaches overcoming the national boundaries. This is the right time to tackle the EVD emergency through the development and use of state-of-art tools and techniques befitting for confirmatory diagnosis, developing internationally suited surveillance, monitoring, and networking systems as well as identifying the animal reservoirs. The urgency for developing and use of both, the safe and efficacious vaccine and affordable anti-viral drugs are desperately needed. Kinome, transcriptome and exome analysis may be helpful in identifying the genomic targets up or down-regulated and also insight into siRNA and miRNA generated during infection helping in developing efficacious therapeutic regimens. This is an apt time to think of for “One Health” concept and come out with a strategy to safeguard human, animal, and environment. Realizing the eventual impact of the threat imposed by EVD, all pinpointed efforts must be focused on achieving the goal of preventing and controlling the existence of EVD timely. The Ebola epidemic has been a humanitarian crisis in the three worst affected countries in West Africa. Although WHO has declared the end of Ebola, there is still a potential risk of sporadic transmission of EBOV which could be largely due to residual virus persistence in some survivors in West Africa (Brainard et al. 2016). The WHO is urging public health authorities to remain on high alert and ready to respond in any emergency (Stamm 2015). Further, strong surveillance and emergency response capacity need to be upheld, while care, screening, and counseling also need to be provided for survivors.

Acknowledgments  
All the authors acknowledge the support from their respective institutions and universities.

Competing interest  
Authors declare that there is no conflict of interests that could arise.

References  
1. Acquah C, Danquah MK, Agyei D, Moy CK, Sidhu A, Ongkudon CM. 2015. Deploying aptameric sensing technology for rapid pandemic monitoring. Crit Rev Biotechnol. 18:1-13.  
2. Ahmad N, Farman A, Badshah SL, Ur Rahman A, Ur Rashid H, Khan K. 2016. Molecular modeling, simulation and docking study of ebola virus glycoprotein. J Mol Graph Model. 19(72): 266-271.  
3. Albarino CG, Shoemaker T, Khristova ML, Wamala JF, Muyembe JJ, Balinandi S, Tumusiime A, Campbell S, Cannon D, Gibbons A, Bergeron E, Bird B, Dodd K, Spiropoulou C, Erickson BR, Guerrero L, Knust B, Nichol ST, Rollin PE, Stroher U. 2013. Genomic analysis of filoviruses associated with four viral hemorrhagic fever outbreaks in Uganda and the Democratic Republic of the Congo in 2012. Virology. 442:97–100.  
4. Alimonti J, Leung A, Jones S, Gren J, Qiu X, Fernando L, Balcewich B, Wong G, Ströher U, Grolla A, Strong J, Kobinger G. 2014. Evaluation of transmission risks associated with in vivo replication of several high containment pathogens in a biosafety level 4 laboratory. Sci Rep. 4: 5824.  
5. Allela L, Boury O, Pouillot R, Délicat A, Yab P, Kumulungui B, Rouquet P, Gonzalez JP, Leroy EM. 2005. Ebola virus antibody prevalence in dogs and human risk. Emerg Infect Dis. 11: 385-390.  
6. Amman BR, Jones ME, Sealy TK, Uebelhoer LS, Schuh AJ, Bird BH, Coleman-MacCray JD, Martin BE, Nichol ST, Towner JS. 2015. Oral shedding of Marburg virus in experimentally infected Egyptian fruit bats (Rousettus aegyptiacus). J Wildl Dis. 51(1): 113-124.  
7. Arntzen C. 2015. Plant-made pharmaceuticals: From “Edible Vaccines” to Ebola therapeutics. Plant Biotechnol. 13: 1013–1016.  
8. Arwady MA, Bawo L, Hunter JC, Massaquoi M, Matanock A, Dahn B, Ayscue P, Nyenshaw T, Forrester JD, Hensley LE, Monroe B, Schoepp RJ, Chen TH, Schaecher KE, George T, Rouse E, Schafer IJ, Pillai SK, De Cock KM. 2015. Evolution of Ebola virus disease from exotic infection to global health priority, Liberia, mid-2014. Emerg Infect Dis. 21(4): 578.

9. Ayukekbong JA. 2016. The 2014-2015 Ebola saga: lessons for the future. J Epidemiol Community Health. 70(1): 1–2.
10. Baca JT, Severns V, Lovato D, Branch DW, Larson RS. 2015. Rapid detection of Ebola virus with a reagent-free, point-of-care biosensor. Sensors (Basel). 15(4): 8605–8614.
11. Baize S, Pannetier D, Oestereich L, Rieger T, Koivogui L, Magassouba N, Soropogui B, Sow MS, Keita S, DeClerck H, Tiffany A, Dominguez G, Loua M, Traore A, Kolie M, Malano ER, Heleze E, Bocquin A, Mely S, Raoul H, Caro V, Cadar D, Gabriel M, Pahlmann M, Tappe D, Schmidt-Chanasit J, Impouma B, Diallo AK, Formenty P, Van Herp M, Gunther S. 2014. Emergence of Zaire Ebola virus disease in Guinea - Preliminary report. N Engl J Med. 371: 1418–1425.
12. Baker A. 2014. The race to diagnose. In Time Magazine. Cape Town, USA. 28–29.
13. Balmith M, Faya M, Soliman ME. 2016. Ebola virus: A gap in drug design and discovery - experimental and computational perspective. Chem Biol Drug Des. In press.
14. Baron RC, McCormick JB, Zubeir OA. 1983. Ebola virus disease in southern Sudan: hospital dissemination and intrafamilial spread. Bull World Health Organ. 61: 997–1003.
15. Barrette RW, Metwally SA, Rowland JM, Xu L, Zaki SR, Nichol ST, Rollin PE, Towner JS, Shieh W-J, Batten B, Sealy TK, Carrillo C, Moran KE, Bracht AJ, Mayr GA, Sirios-Cruz M, Catbagan DP, Lautner EA, Ksiazek TG, White WR, McIntosh MT. 2009. Discovery of swine as a host for the Reston ebolavirus. Science. 325: 204–206.
16. Basler CF, Mikulasova A, Martinez-Sobrido L, Paragas J, Mühlberger E, Bray M, Klenk HD, Palese P, Garcia-Sastre A. 2003. The ebolavirus VP 35 protein inhibits activation of interferon regulatory factor 3. J Virol. 77: 7945–7956.
17. Basler CF, Wang X, Mühlberger E, Volchkov V, Paragas J, Klenk HD, Garcia-Sastre A, Palese P. 2000. The Ebola virus VP 35 protein functions as a type I IFN antagonist. Proc Natl Acad Sci USA. 97: 12289–12294.
18. Basler CF. 2014. Portrait of a killer: genome of the 2014 EBOV outbreak strain. Cell Host Microbe. 16: 419–421.
19. Bausch DG, Crozier I. 2016. The Liberia men's health screening program for Ebola virus: win-win-win for survivor, scientist, and public health. Lancet Glob Health. 4: e672–e673.
20. Bausch DG, Schwarz L. 2014. Outbreak of Ebola Virus Disease in Guinea: Where Ecology Meets Economy. PLoS Negl Trop Dis. 8: e3056.
21. Bausch DG, Towner JS, Dowell SF, Kaducu F, Lukwiya M, Sanchez A, Nichol ST, Ksiazek TG, Rollin PE. 2007. Assessment of the risk of Ebola virus transmission from bodily fluids and fomites. J Infect Dis. 196 Suppl 2: S142–147.
22. Belak S, Thoren P, Le Blanc N, Viljoen G. 2009. Advances in viral disease diagnostic and molecular epidemiological technologies. Expert Rev Mol Diagn. 9(4): 367–381.
23. Bellizzi S. 2014. The current Ebola outbreak: old and new contexts. J Infect Dev Ctries. 8: 1378–1380.
24. Benowitz I, Ackelsberg J, Balter SE, Baumgartner JC, Dentinger C, Fine AD, Harper SA, Jones LE, Laraque F, Lee EH, Merizalde G, Yacisin KA, Varma JK, Layton MC. 2014. Centers for Disease Control and Prevention (CDC). Surveillance and preparedness for Ebola virus disease -- New York City, 2014. MMWR Morb Mortal Wkly Rep. 63(41): 934–936.
25. Benzine JW, Brown KM, Agans KN, Godiska R, Mire CE, Gowda K, Converse B, Geisbert TW, Mead DA, Chander Y. 2016. Molecular diagnostic field test for point-of-care detection of Ebola virus directly from blood. J Infect Dis. In press.
26. Bermejo M, Rodriguez-Teijeiro JD, Illera G, Barroso A, Vila C, Walsh PD. 2006. Ebola outbreak killed 5000 gorillas. Science. 314: 1564.
27. Bharat TAM, Noda T, Riches JD, Kraehling V, Kolesnikova L, Becker S, Kawaoka Y, Briggs JAG. 2012. Structural dissection of Ebola virus and its assembly determinants using cryo-electron tomography. Proc Natl Acad Sci USA. 109: 4275–4280.
28. Boggild AK, Esposito DH, Kozarsky PE, Ansdell V, Beeching NJ, Campion D, Castelli F, Caumes E, Chappuis F, Cramer JP, Gkrania-Klotsas E, Grobusch MP, Hagmann SH, Hynes NA, Lim PL, López-Vélez R, Malvy DJ, Mendelson M, Parola P, Sotir MJ, Wu HM, Hamer DH. 2015. GeoSentinel Surveillance Network. Differential diagnosis of illness in travelers arriving from Sierra Leone, Liberia, or Guinea: a cross-sectional study from the GeoSentinel Surveillance Network. Ann Intern Med. 162 (11): 757–764.
29. Boisen ML, Oottamasathien D, Jones AB, Millett MM, Nelson DS, Bornholdt ZA, Fusco ML, Abelson DM, Oda S, Hartnett JN, Rowland MM, Heinrich ML, Akdag M, Goba A, Momoh M, Fullah M, Baimba F, Gbakie M, Safa S, Fonnie R, Kanneh L, Cross RW, Geisbert JB, Geisbert TW, Kulakosky PC, Grant DS, Shaffer JG, Schieffelin JS, Wilson RB, Saphire EO, Branco LM, Garry RF, Khan SH, Pitts KR, Viral Hemorrhagic Fever Consortium. 2015. Development of Prototype Filovirus Recombinant Antigen Immunoassays. J Infect Dis. 212(Suppl 2): S359–367.

30. Borisovich IV, Mikhaĭlov VV, Potryvaeva NV, MalinkinIuN, Kirillov AP, Krasnianskiĭ VP, Markov VI, Makhlaĭ AA, Lebedinskaia EV. 1996. Development of the immunoenzyme test-system for detection of Ebola virus antigen. Vopr Virusol. 41(5): 232-234.

31. Bowen ET, Lloyd G, Harris WJ, Platt GS, Baskerville A, Vella EE. 1977. Viral haemorrhagic fever in southern Sudan and northern Zaire. Preliminary studies on the aetiological agent. Lancet. 1: 571–573.

32. Brainard J, Pond K, Hooper L, Edmunds K, Hunter P. 2016. Presence and Persistence of Ebola or Marburg Virus in Patients and Survivors: A Rapid Systematic Review. PLoS Negl Trop Dis. 10(2): e0004475.

33. Brauburger K, Boehmann Y, Tsuda Y, Hoenen T, Olejnik J, Schümann M, Ebihara H, Mühlberger E. 2014. Analysis of the highly diverse gene borders in Ebola virus reveals a distinct mechanism of transcriptional regulation. J Virol. 88: 12558–12571.

34. Broadhurst MJ, Kelly JD, Miller A, Semper A, Bailey D, Groppelli E, Simpson A, Brooks T, Hula S, Nyoni W, Sankoh AB, Kanu S, Jalloh A, Ton Q, Sarchet N, George P, Perkins MD, Wonderly B, Murray M, Pollock NR. 2015. ReEBOV Antigen Rapid Test kit for point-of-care and laboratory-based testing for Ebola virus disease: a field validation study. Lancet. 386(9996): 867–874.

35. Brown CM, Aranas AE, Benenson GA, Brunette G, Cetron M, Chen T, Cohen NJ, Diaz P, Haber Y, Hale CR, Holton K, Kohl K, Lee AW, Palumbo GJ, Pearson K, Phares CR, Alvarado-Ramy F, Roohi S, Rotz LD, Tappero J, Washburn FM, Watkins J, Pesik N. 2014. Airport exit and entry screening for Ebola—August-November 10, 2014. MMWR Morb Mortal Wkly Rep. 63: 1163–1167.

36. Bukreyev AA, Chandran K, Dolnik O, Dye JM, Ebihara H, Leroy EM, Mühlberger E, Netesov SV, Patterson JL, Paweska JT, Saphire EO, Smither SJ, Takada A, Towner JS, Volchkov VE, Warren TK, Kuhn JH. 2014. Discussions and decisions of the 2012–2014 International Committee on Taxonomy of Viruses (ICTV) Filoviridae Study Group, January 2012–June 2013. Arch Virol. 159: 821–830.

37. Butler D. 2014. Ebola experts seek to expand testing. Nature. 516(7530): 154–155.

38. Caballero IS, Honko AN, Gire SK, Winnicki SM, Melé M, Gerhardinger C, Lin AE, Rinn JL, Sabeti PC, Hensley LE, Connor JH. 2016. In vivo Ebola virus infection leads to a strong innate response in circulating immune cells. BMC genomics. 17: 707.

39. Cai H, Parks JW, Wall TA, Stott MA, Stambaugh A, Alfson K, Griffiths A, Mathies RA, Carrion R, Patterson JL, Hawkins AR, Schmidt H. 2015. Optofluidic analysis system for amplification-free, direct detection of Ebola infection. Sci Rep. 5: 14494.

40. Calain P, Roux L, Kolakofsky D. 2016. Defective interfering genomes and Ebola virus persistence. The Lancet. 388: 659–660.

41. Calza L, Trapani F, Bartoletti M, Manfredi R, Colangeli V, Borderi M, Grossi G, Motta R, Viale P. 2012. Statin therapy decreases serum levels of high-sensitivity C-reactive protein and tumor necrosis factor α in HIV-infected patients treated with ritonavir-boosted protease inhibitors. HIV Clin Trials. 13: 153–161.

42. Carette JE, Raaben M, Wong AC, Herbert AS, Obernosterer G, Mulherkar N, Kuehne AI, Kranzusch PJ, Griffin AM, Ruthel G, Dal Cin P, Dye JM, Whelan SP, Chandran K, Brummelkamp TR. 2011. Ebola virus entry requires the cholesterol transporter Niemann-Pick Cl. Nature 477: 340–343.

43. Carney TJ, Weber DJ. 2015. Public Health Intelligence: Learning From the Ebola Crisis. Am J Public Health. 105(9): 1740–1744.

44. Carra JH, Martins KA, Schokman RD, Robinson CG, Steffens JT, Bavari S. 2015. A thermostable, chromatographically purified Ebola nano-VLP vaccine. J Transl Med. 13: 228.

45. Casillas AM, Nyamathi AM, Sosa A, Wilder CL, Sands H. 2003. A current review of Ebola virus: pathogenesis, clinical presentation, and diagnostic assessment. Biol Res Nurs. 4: 268–275.

46. Cazares LH, Ward MD, Brueggemann EE, Kenny T, Demond P, Mahone CR, Martins KAO, Nuss JE, Glaros T, Bavari S. 2016. Development of a liquid chromatography high resolution mass spectrometry method for the quantitation of viral envelope glycoprotein in Ebola virus-like particle vaccine preparations. Clin Proteomics. 13: 18.

47. CDC. 1989. Ebola virus infection in imported primates--Virginia, 1989. MMWR Morb Mortal Wkly Rep. 38: 831–832, 837–838.

48. CDC. 1990a. Epidemiologic Notes and Reports Update: filovirus infection in animal handlers. MMWR Morb Mortal Wkly Rep. 39: 221.

49. CDC. 1990b. Update: filovirus infections among persons with occupational exposure to nonhuman primates. MMWR Morb Mortal Wkly Rep. 39: 266–267; 273.

50. CDC. 2015. Ebola Outbreak in West Africa - Case Counts. As of April 11, 2015. Updated April 13, 2015. Available: http://www.cdc.gov/vhf/ebola/outbreaks/2014-west-africa/case-counts.html. Accessed on April 15, 2015.

51. CDC. 2015. Etymologia. Ebola. Emerg Infect Dis. 21: 1905.

52. CDC. 2015. Sierra Leone Trial to Introduce a Vaccine against Ebola (STRIVE) Q&A. Retrieved September 1st 2015.

53. Cenciarelli O, Pietropaoli S, Malizia A, Carestia M, D'Amico F, Sassolini A, Di Giovanni D, Rea S, Gabbarini V, Tamburrini A, Palombi L, Bellecci C, Gaudio P. 2015. Ebola virus disease 2013-2014 outbreak in West Africa: an analysis of the epidemic spread and response. Int J Microbiol. 2015:769121.
54. Cenedella RJ. 2009. Cholesterol synthesis inhibitor U18666A and the role of sterol metabolism and trafficking in numerous pathophysiological processes. Lipids. 44:477–487.
55. Centers for Disease Control and Prevention. Ebolavirus Ecology [Internet]. 1 Aug 2014. http://www.cdc.gov/vhf/ebola/resources/virus-ecology.html
56. Chabra S. 2015. International Classification of Diseases, 10th Revision, coding for prematurity: need for standardized nomenclature. Health Care Manag (Frederick). 34(2):123–127.
57. Chancellor JR, Padmanabhan SP, Greenough TC, Sacra R, Ellison III RT, Madoff LC, Droms RJ, Hinkle DM, Asdourian GK, Finberg RW, Stroher U, Uyeki TM, Ceron OM. 2016. Uveitis and systemic inflammatory markers in convalescent phase of Ebola virus disease. Emerg Infect Dis. 22:295–297.
58. Chappell KJ, Watterson. 2017. Fighting Ebola: A Window for Vaccine Re-evaluation? PLoS Pathog. 13(1):e1006037.doi:10.1371/journal.ppat.1006037.eCollection 2017.
59. Check HE. 2016. Ebola virus lingers longer than scientists thought. Nature. 537:291–292.
60. Chen Z, Liang H, Chen X, Ke Y, Zhou Z, Yang M, Zen K, Yang R, Liu C, Zhang CY. 2016. An Ebola virus-encoded microRNA-like fragment serves as a biomarker for early diagnosis of Ebola virus disease. Cell Res. 26(3):380–383.
61. Cheng F, Murray JL, Zhao J, Sheng J, Zhao Z, Rubin DH. 2016. Systems biology-based investigation of cellular antiviral drug targets identified by gene-trap insertional mutagenesis. PLoS Comput Biol. 12:e1005074.
62. Cherpillod P, Schibler M, Vieille G, Cordey S, Mamin A, Vetter P, Kaiser L. 2016. Ebola virus disease diagnosis by real-time RT-PCR: A comparative study of 11 different procedures. J Clin Virol. 77:9–14.
63. Chertow DS, Kleine C, Edwards JK, Scaini R, Giuliani R, Sprecher A. 2014. Ebola virus disease in West Africa--clinical manifestations and management. N Engl J Med. 371:2054–2057.
64. Chertow DS, Nath A, Suffredini AF, Danner RL, Reich DS, Bishop RJ, Childs RW, Arai AE, Palmore TN, Lane HC, Fauci AS, Davey RT. 2016. Severe Meningoencephalitis in a Case of Ebola Virus Disease: A Case Report. Ann Intern Med. 165(4):301–304.
65. Chiramel AI, Banadyga L, Dougherty JD, Falzarano D, Martellaro C, Brees D, Taylor RT, Ebihara H, Best SM. 2016. Alisporivir has limited antiviral effects against Ebola virus strains Makona and Mayinga. J Infect Dis. 241.
66. Choi JH, Croyle MA. 2013. Emerging targets and novel approaches to Ebola virus prophylaxis and treatment. BioDrugs. 27:565583.
67. Choi WY, Hong KJ, Hong JE, Lee WJ. 2015. Progress of vaccine and drug development for Ebola preparedness. Clin Exp Vaccine Res. 4:11–16
68. Chretien JP, Riley S, George DB. 2015. Mathematical modeling of the West Africa Ebola epidemic. eLife. 4:e09186.
69. Christaki E. 2015. New technologies in predicting, preventing and controlling emerging infectious diseases. Virulence. 6(6):558–565.
70. Chughtai AA, Barnes M, Macintyre CR. 2016. Persistence of Ebola virus in various body fluids during convalescence: evidence and implications for disease transmission and control. Epidemiol Infect. 25:1–9.
71. Coen A, Henk M. How the virus came into this world. Die Zeit. 2015.http://www.zeit.de/feature/ebola-afrika-virus.
72. Cohen J. 2014. Infectious Disease. Ebola vaccines racing forward at record pace. Science. 345:1228–1229.
73. Corti D, Misasi J, Mulangu S, Stanley DA, Kanekiyo M, Wollen S, Plöquin A, Doria-Rose NA, Staupé RP, Bailey M, Shi W, Choe M, Marcus H, Thompson EA, Caiggi A, Silacci C, Fernandez-Rodriguez B, Perez L, Sallusto F, Vanzetta F, Agatic G, Cameroni E, Kisalu N, Gordon I, Ledgerwood JE, Mascola JR, Graham BS, Muyembe-Tamfum JJ, Trefry JC, Lanzavecchia A, Sullivan NJ. 2016. Protective monotherapy against lethal Ebola virus infection by a potently neutralizing antibody. Science. 351(6279):1339–1342.
74. Cross RW, Boisen ML, Millett MM, Nelson DS, Oottamasathien D, Hartnett JN, Jones AB, Goba A, Momoh M, Fullah M, Bornholdt ZA. 2016. Analytical Validation of the ReEBOV Antigen Rapid Test for Point-of-Care Diagnosis of Ebola Virus Infection. J Infect Dis. In press.
75. Curran KG, Gibson JJ, Marke D, Caulker V, Bomeh J, Redd JT, Bunga S, Brunkard J, Kilmarx PH. 2016. Cluster of Ebola Virus Disease Linked to a Single Funeral-Moyamba District, Sierra Leone, 2014. MMWR Morb Mortal Wkly Rep. 65(8):202–205.
76. Dedkov VG, Magassouba NF, Safonova MV, Deviatkin AA, Dolgova AS, Pyankov OV, Sergeev AA, Utkin DV, Odinokov GN, Safronov VA, Agafonov AP, Maleev VV, Shipulin GA. 2016. Development and evaluation of a real-time RT-PCR assay for the detection of Ebola virus (Zaire) during an Ebola outbreak in Guinea in 2014–2015. J Virol Methods. 228:26–30.


77. Deen GF, Knust B, Broutet N, Sesay FR, Formenty P, Ross C, Thorson AE, Massaquoi TA, Marrinan JE, Ervin E, Jambai A, McDonald SL, Bernstein K, Wurie AH, Dumbuya MS, Abad N, Idriss B, Wi T, Bennett SD, Davies T, Ebrahim FK, Meites E, Naidoo D, Smith S, Banerjee A, Erickson BR, Brault A, Durski KN, Winter J, Sealy T, Nichol ST, Lamunu M, Ströher U, Morgan O, Sahr F. 2015. Ebola RNA Persistence in Semen of Ebola Virus Disease Survivors - Preliminary Report. N Engl J Med. 52(5):714-721.

78. Delany I, Rappuoli R, De Gregorio E. 2014. Vaccines for the 21st century. EMBO Mol Med 6:708-720.

79. Dhama K, Tiwari R, Chakraborty S, Kumar A, Karikalan M, Singh R, Rai RB. 2013a. Global warming and emerging infectious diseases of animals and humans: current scenario, challenges, solutions and future perspectives – a review. Int J Curr Res. 5:1942-1958.

80. Dhama K, Wani MY, Deb R, Karthik K, Tiwari R, Barathidasan R, Kumar A, Mahima, Verma AK, Singh SD. 2013b. Plant based oral vaccines for human and animal pathogens-a new era of prophylaxis: current and future prospective. J. Exp. Biol. Agri. Sci. 1(1): 1-12.

81. Dhama K, Chakraborty S, Mahima, Wani MY, Verma AK, Deb R, Tiwari R, Kapoor S 2013c. Novel and emerging therapies safeguarding health of humans and their companion animals: a review. Pak J Biol Sci. 16(3):101-111.

82. Dhama K, Karthik K, Chakraborty S, Tiwari R, Kapoor S, Kumar A, Thomas P. 2014. Loop-mediated isothermal amplification of DNA (LAMP) – a new diagnostic tool lights the world of diagnosis of Animal and Human Pathogens: A review. Pak J Biol Sci. 17(2): 151-166.

83. Dhama K, Malik YS, Malik SV, Singh RK. 2015a. Ebola from emergence to epidemic: the virus and the disease, global preparedness and perspectives. J Infect Dev Ctries. 9(5): 441-455.

84. Dhama K, Saminathan M, Jacob SS, Singh M, Karthik K, Amarpal, Tiwari R, Sunkara LT, Malik YS, Singh RK. 2015b. Effect of immunomodulation and immunomodulatory agents on health with some bioactive principles, modes of action and potent biomedical applications. Int J Pharmacol. 11(4): 253-290.

85. Dhillon RS, Srikrishna D, Garry RF, Chowell G. 2015. Ebola control: rapid diagnostic testing. Lancet Infect Dis. 15: 147-48.

86. Dolzhikova IV, Zubkova OV, Tukhvatulin AI, Dzharullaeva AS, Tukhvatulina NM, Shcheblyakov DV, Shmarov MM, Tokarskaya EA, Simakova YV, Egorova DA, Scherbinin DN, Tutykhina IL, Lysenko AA, Kostarnoy AV, Gancheva PG, Ozharovskaya TA, Belugin BV, Kolobukhina LV, Pantyukhov VB, Syromyatnikova SI, Shatokhina IV, Sizikova TV, Rumyantseva IG, Andrus AF, Boyarskaya NV, Voytyuk AN, Babira VF, Volchkina SV, Kutaev DA, Bel'skih AN, Zhdanov KV, Zakharenko SM, Borisovich SV, Logunov DY, Naroditsky BS, Gintsburg AL. 2017. Safety and immunogenicity of GamEvac-Combi, a heterologous VSV-and Ad5-vectored Ebola vaccine: An open phase I/II trial in healthy adults in Russia. Hum Vaccin Immunother. 2:1-8. doi: 10.1080/21645515.2016.1238535.

87. Dong S, Yang P, Li G, Liu B, Wang W, Liu X, Xia B, Yang C, Lou Z, Guo Y, Rao Z. 2015. Insight into the Ebola virus nucleocapsid assembly mechanism: crystal structure of Ebola virus nucleoprotein core domain at 1.8 Å resolution. Protein Cell. 6:351-362.

88. Dowell SF, Mukunu R, Ksiazek TG, Khan AS, Rollin PE, Peters CJ. 1999. Transmission of Ebola hemorrhagic fever: a study of risk factors in family members, Kikwit, Democratic Republic of the Congo, 1995. Commission de Lutte contre les Epidémies à Kikwit. J Infect Dis. 179 (Suppl 1): S87-91.

89. Drosten C, Göttig S, Schilling S, Asper M, Panning M, Schmitz H, Günther S. 2002. Rapid detection and quantification of RNA of Ebola and Marburg viruses, Lassa virus, Crimean-Congo hemorrhagic fever virus, Rift Valley fever virus, dengue virus, and yellow fever virus by real-time reverse transcription-PCR. J Clin Microbiol. 40:2323-2330.

90. Drosten C, Göttig S, Schilling S, Asper M, Panning M, Schmitz H, Günther S. 2002. Rapid detection and quantification of RNA of Ebola and Marburg viruses, Lassa virus, Crimean-Congo hemorrhagic fever virus, Rift Valley fever virus, dengue virus, and yellow fever virus by real-time reverse transcription-PCR. J Clin Microbiol. 40(7): 2323-2330.

91. Duan D, Fan K, Zhang D, Tan S, Liang M, Liu Y, Zhang J, Zhang P, Liu W, Qiu X, Kobinger GP, Gao GF, Yan X. 2015. Nanzyme-strip for rapid local diagnosis of Ebola. Biosens Bioelectron. 74:134-141.

92. Dunning J, Kennedy SB, Antierens A, Whitehead J, Ciglenecki I, Carson G, Kanapathipillai R, Castle L, Howell-Jones R, Pardinaz-Solis R, Grove J, Scott J, Lang T, Olliaro P, Horby PW, RAPIDE-BCV trial team. 2016. Experimental treatment of Ebola virus disease with brincidofovir. PloS one. 11: e0162199.

93. Dye JM, Herbert AS, Kuehne AI, Barth JF, Muhammad MA, Zak SE, Ortiz RA, Prugar LI and. Pratt WD. (2012). Postexposure antibody prophylaxis pro-tects nonhuman primates from filovirus disease. Proc Natl Acad Sci. 109: 5034-5039.

94. Elston JWT, Cartwright C, Ndumbi P, Wright J. 2017. The health impact of the 2014–15 Ebola outbreak. Public Health 143: 60-70.

95. El Sayed SM, Abdelrahman AA, Ozbak HA, Hemeg HA, Kheyami AM, Rezk N, El-Ghoul MB, Nabo MM, Fathy YM. 2016. Updates in diagnosis and management of Ebola hemorrhagic fever. J Res Med Sci. 18(21): 84.

96. Emond RT, Evans B, Bowen ET, Lloyd G. 1977. A case of Ebola virus infection. BMJ. 2:541–544.

97. Erhabor O, Adias TC. 2011. From whole blood to component therapy: the economic, supply/demand need for implementation of component therapy in sub-Saharan Africa. Transfus Clin Biol. 18:516–526.

98. Fähnrich C, Denecke K, Adeoye OO, Benzler J, Claus H, Kirchner G, Mall S, Richter R, Schapranow MP, Schwarz N, Tom-Aba D, Uflacker M, Poggensee G, Krause G. 2015. Surveillance and Outbreak Response Management System (SORMAS) to support the control of the Ebola virus disease outbreak in West Africa. Euro Surveill. 20(12): 21071.

99. Fallah MP, Skrip LA, Dahn BT, Nyenswah TG, Flomo H, Glayweon M, Lorseh TL, Kaler SG, Higgs ES, Galvani AP. 2016. Pregnancy outcomes in Liberian women who conceived after recovery from Ebola virus disease. Lancet Glob Health. 4:e678-e679.

100. Falzarano D, Geisbert TW, and Feldmann H. 2011. Progress in filovirus vaccine development: evaluating the potential for clinical use. Expert Rev Vaccines. 10(1): 63–77.

101. Fauci AS. 2014. Ebola—underscoring the global disparities in health care resources. The New Eng J Med. 371:1084-1086.

102. Faye O, Faye O, Soropogui B, Patel P, El Wahed AA, Loucoubar C, Fall G, Kiory D, Magassouba N, Keita S, Kondé MK, Diallo AA, Koivogui L, Karlberg H, Mirazimi A, Nentwich O, Piepenburg O, Niedrig M, Weidmann M, Sall AA. 2015. Development and deployment of a rapid recombinase polymerase amplification Ebola virus detection assay in Guinea in 2015. Euro Surveill. 20(44). doi: 10.2807/1560-7917.ES.2015.20.44.30053. Erratum in: Euro Surveill. 2016; 21(4). doi: 10.2807/1560-7917.ES.2016.21.4.30121.

103. Felsenstein J. 1985. Confidence limits on phylogenies: An approach using the bootstrap. Evolution 39: 783-791.

104. Florescu DF, Kalil AC, Hewlett AL, Schuh AJ, Stroher U, Uyeki TM, Smith PW. 2015. Administration of brincidofovir and convalescent plasma in a patient with Ebola virus disease. Clin Infect Dis. 61(6):969-73.

105. Florescu DF, Keck MA. 2014. Development of CMX 001 (Brincidofovir) for the treatment of serious diseases or conditions caused by dsDNA viruses. Expert Rev Anti Infect Ther. 12:1171–1178.

106. Feldmann FL, Feldmann H. 2014. Ebola: facing a new transboundary animal disease? Dev Biol (Basel). 135:201-209.

107. Feldmann H, Bugany H, Mahner F, Klenk HD, Drenckhahn D, Schnittler HJ. 1996. Filovirus-induced endothelial leakage triggered by infected monocytes/macrophages. J Virol. 70: 2208-2214.

108. Feldmann H, Geisbert TW. 2011. Ebola haemorrhagic fever. Lancet. 377:849-862.

109. Fischer RJ, Judson S, Miazgowicz K, Bushmaker T, Munster VJ. 2016. Ebola Virus Persistence in Semen Ex Vivo. Emerg Infect Dis. 22(2): 289-291.

110. Flint M, Goodman CH, Bearden S, Blau DM, Amman BR, Basile AJ, Belser JA, Bergeron É, Bowen MD, Brault AC, Campbell S, Chakrabarti AK, Dodd KA, Erickson BR, Freeman MM, Gibbons A, Guerrero LW, Klena JD, Lash RR, Lo MK, McMullan LK, Momoh G, Massally JL, Goba A, Paddock CD, Priestley RA, Pyle M, Rayfield M, Russell BJ, Salzer JS, Sanchez AJ, Schuh AJ, Sealy TK, Steinau M, Stoddard RA, Taboy C, Turnsek M, Wang D, Zemtsova GE, Zivcec M, Spiropoulou CF, Ströher U, Towner JS, Nichol ST, Bird BH. 2015. Ebola Virus Diagnostics: The US Centers for Disease Control and Prevention Laboratory in Sierra Leone, August 2014 to March 2015. J Infect Dis. 212 (Suppl 2):S350-358.

111. Folayan MO, Yakubu A, Haire B, Peterson K. 2016. Ebola vaccine development plan: ethics, concerns and proposed measures. BMC Medical Ethics. 17:10. DOI: 10.1186/s12910-016-0094-4.

112. Formenty P, Boesch C, Wyers M, Steiner C, Donati F, Dind F, Walker F, Le Guenno B. 1999a. Ebola virus outbreak among wild chimpanzees living in a rain forest of Côte d'Ivoire. J Infect Dis. 179 (Suppl 1):S120–126.

113. Formenty P, Hatz C, Le Guenno B, Stoll A, Rogenmoser P, Widmer A. 1999b. Human infection due to Ebola virus, subtype Côte d'Ivoire: clinical and biologic presentation. J Infect Dis. (179 Suppl 1): S48–53.

114. Formenty P, Leroy EM, Epelboin A, Libama F, Lenzi M, Sudeck H, Yaba P, Allarangar Y, Boumandouki P, Nkounkou VB, Drosten C, Grolla A, Feldmann H, Roth C. 2006. Detection of Ebola virus in oral fluid specimens during outbreaks of Ebola virus hemorrhagic fever in the Republic of Congo. Clin Infect Dis. 42:1521–1526.

115. Formenty P, Libama F, Epelboin A, Allarangar Y, Leroy E, Moudzeo H, Tarangonia P, Molamou A, Lenzi M, Ait-Ikhlef K, Hewlett B, Roth C, Grein T. 2003. Outbreak of Ebola hemorrhagic fever in the Republic of the Congo, 2003: a new strategy?. Med Trop (Mars). 63:291–295.

116. Francesconi P, Yoti Z, Declich S, Onek PA, Fabiani M, Olang J, Andraghetti R, Rollin PE, Opira C, Greco D, Salmaso S. 2003. Ebola hemorrhagic fever transmission and risk factors of contacts, Uganda. Emerg Infect Dis. 9:1430–1437.
117. Friedrich MJ. 2016. Clinical Sequelae of Ebola. JAMA. 315:647-647.
118. Gallaher WR, Garry RF. 2015. Modeling of the Ebola virus delta peptide reveals a potential lytic sequence motif. Viruses. 7:285–305.
119. Gao J, Yin L. 2014. Drug development for controlling Ebola epidemic—A race against time. Drug Discov Ther. 8(5):229-231.
120. Gatherer D. 2014. The 2014 Ebola virus disease outbreak in West Africa. J Gen Virol. 95:1619-1624.
121. Gehring G, Rohrmann K, Atencio N, Mittler E, Becker S, Dahlmann F, Pohlmann S, Vondran FW, David S, Manns MP, Ciesek S, von Hahn T. 2014. The clinically approved drugs amiodarone, dronedarone and verapamil inhibit filovirus cell entry. J Antimicrob Chemother. 69:2123-2131.
122. Geisbert T, Hensley L, Larsen T, Young H, Reed D, Geisbert J, Scott D, Kagan E, Jahrling P, Davis K. 2003. Pathogenesis of Ebola Hemorrhagic Fever in Cynomolgus Macaques. American J Pathol. 163:2347-2370.
123. Geisbert TW, Geisbert JB, Leung A, Daddario-DiCaprio KM, Hensley LE, Grolla A, Feldmann H. 2009. Single-injection vaccine protects nonhuman primates against infection with Marburg virus and three species of Ebola virus. J Virol. 83:7296-7304.
124. Geisbert TW, Geisbert JB, Leung A, Daddario-DiCaprio KM, Hensley LE, Grolla A, Feldmann H. 2009. Single-injection vaccine protects nonhuman primates against infection with Marburg virus and three species of Ebola virus. J Virol. 83:7296-7304.
125. Geisbert TW, Hensley LE, Jahrling PB, Larsen T, Geisbert JB, Paragas JJ, Young HA, Fredeking TM, Rote WE, Vlasuk GP. 2003. Treatment of Ebola virus infection with a recombinant inhibitor of factor VIIa/tissue factor: a study in rhesus monkeys. Lancet. 13:1953–1958.
126. Geisbert TW, Jahrling PB. 2003. Towards a vaccine against Ebola virus. Expert Rev Vaccines. 2:777-789.
127. Geisbert TW, Lee AC, Robbins M, Geisbert JB, Honko AN, Sood V, Johnson JC, de Jong S, Tavakoli I, Judge A, Hensley LE, Maclachlan I. 2010. Post exposure protection of non-human primates against a lethal Ebola virus challenge with RNA interference: A proof-of-concept study. Lancet. 375:1896-1905.
128. Georges AJ, Leroy EM, Renaut AA, Benissan CT, Nabias RJ, Ngoc MT, Obiang PI, Lepage JP, Bertherat EJ, Bénoni DD, Wickings EJ, Amblard JP, Lansoud-Soukate JM, Milleliri JM, Baize S, Georges-Courbot MC. 1999. Ebola hemorrhagic fever outbreaks in Gabon, 1994-1997: epidemiologic and health control issues. J Infect Dis. 179 (Suppl 1): S65–75.
129. Georges-Courbot MC, Sanchez A, Lu CY, Baize S, Leroy E, Lansout-Soukate J, Tévi-Bénissan C, Georges AJ, Trappier SG, Zaki SR, Swanepoel R, Lemann PA, Rollin PE, Peters CJ, Nichol ST, Ksiazek TG. 1997. Isolation and phylogenetic characterization of Ebola viruses causing different outbreaks in Gabon. Emerg Infect Dis. 3:59–62.
130. Ghani AC, Burgess DH, Reynolds A, Rousseau C. 2015. Expanding the role of diagnostic and prognostic tools for infectious diseases in resource-poor settings. Nature. 528(7580):S50-52.
131. Gibb TR, Norwood DA Jr, Woollen N, Henchal EA. 2001. Development and evaluation of a fluorogenic 5' nuclease assay to detect and differentiate between Ebola virus subtypes Zaire and Sudan. J Clin Microbiol. 39(11):4125-4130.
132. Gilbert GL. 2015. Laboratory testing in management of patients with suspected Ebolavirus disease: infection control and safety. Pathology. 47(5):400-402.
133. Gire SK, Goba A, Andersen KG, Sealfon RSG, Park DJ, Kanneh L, Jalloh S, Momoh M, Fullah M, Dudas G, Wohl S, Moses LM, Yozwiak NL, Winnicki S, Matranga CB, Malboeuf CM, Qu J, Gladden AD, Schaffner SF, Yang X, Jiang PP, Nekoui M, Colubri A, Coomber MR, Fonnie M, Moigboi A, Gbakie M, Kamara FK, Tucker V, Konuwa E, Saffa S, Sellu J, Jalloh AA, Kovoma A, Koninga J, Mustapha I, Kargbo K, Foday M, Yillah M, Kanneh F, Robert W, Massally JL, Chapman SB, Bochicchio J, Murphy C, Nusbaum C, Young S, Birren BW, Grant DS, Scheiffelin JS, Lander ES, Happi C, Gevao SM, Gnirke A, Rambaut A, Garry RF, Khan SH, Sabeti PC. 2014. Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science. 345:1369–1372.
134. Goeijenbier M, van Kampen JJA, Reusken CBEM, Koopmans MPG, van Gorp ECM. 2014. Ebola virus disease: a review on epidemiology, symptoms, treatment and pathogenesis. Neth J Med. 72:442–448.
135. Grard G, Biek R, Tamfum J-JM, Fair J, Wolfe N, Formenty P, Paweska J, Leroy E. 2011. Emergence of divergent Zaire ebola virus strains in Democratic Republic of the Congo in 2007 and 2008. J Infect Dis. 204(Suppl 3): S776–784.
136. Guimard Y, Bwaka MA, Colebunders R, Calain P, Massamba M, De Roo A, Mupapa KD, Kibadi K, Kuvula KJ, Ndaberey DE, Katwiki KR, Mapanda BB, Nkuku OB, Fleerackers Y, Vandend Ende NE,


Kipasa MA. 1999. Organization of patient care during the Ebola hemorrhagic fever epidemic in Kikwit, Democratic Republic of the Congo, 1995. J Infect Dis. 179 (Suppl 1): S268–273.

137. Gulland A. 2014. First Ebola treatment is approved by WHO. British Med J. 349: g5539.

138. Haque A, Didier Hober, Joel Blondiaux. 2015. Addressing Therapeutic Options for Ebola Virus Infection in Current and Future Outbreaks. Antimicrobial Agents and Chemotherapy. 59(10): 5892-5902.

139. Hartman AL, Towner JS, Nichol ST. 2010. Ebola and marburg hemorrhagic fever. Clin Lab Med. 30: 161-177.

140. Hartmann RK, Grünweller A, Strecker T. 2017. Persistence of Ebola virus in the semen of male survivors. J Public Health Emerg 1: 23.

141. Harty RN, Brown ME, Wang G, Huibregtse J, Hayes FP. 2000. A PPxY motif within the VP 40 protein of Ebola virus interacts physically and functionally with a ubiquitin ligase: implications for filovirus budding. Proc Natl Acad Sci USA. 97: 13871-13876.

142. Hassanin A, Nesi N, Marin J, Kadjo B, Pourrut X, Leroy É, Gembu GC, Musaba Akawa P, Ngoagouni C, Nakouné E, Ruedi M, Tshikung D, Pongombo Shongo C, Bonillo C. 2016. Comparative phylogeography of African fruit bats (Chiroptera, Pteropodidae) provide new insights into the outbreak of Ebola virus disease in West Africa, 2014-2016. CR Biol. 339(11-12): 517-528.

143. Hayes CG, Burans JP, Ksiazek TG, Del Rosario RA, Miranda ME, Manaloto CR, Barrientos AB, Robles CG, Dayrit MM, Peters CJ. 1992. Outbreak of fatal illness among captive macaques in the Philippines caused by an Ebola-related filovirus. Am J Trop Med Hyg. 46: 664-671.

144. Heald AE, Iversen PL, Saoud JB, Sazani P, Charleston JS, Axtelle T, Wong M, Smith WB, Vutikullird A, Kaye E. 2014. Safety and pharmacokinetic profiles of phosphorodiamidate morpholino oligomers with activity against Ebola virus and Marburg virus: results of two single ascending-dose studies. Antimicrob Agents Chemother. 58: 6639-6647.

145. Hebert EH, Bah MO, Etard JF, Sow MS, Resnikoff S, Fardeau C, Tourea A, Ouendeno AN, Sagno IC, March L, Izard S, Lama PL, Barry M, Delaporte E. 2017. Ocular complications in survivors of the Ebola outbreak in Guinea. American Journal of Ophthalmology 175: 114-121.

146. Heeney JL. 2015. Ebola: Hidden reservoirs. Nature. 527(7579): 453-455.

147. Henß L, Beck S, Weidner T, Biedenkopf N, Sliva K, Weber C, Becker S, Schnierle BS. 2016. Suramin is a potent inhibitor of Chikungunya and Ebola virus cell entry. Virology Journal. 13: 149.

148. Heymann DL, Weisfeld JS, Webb PA, Johnson KM, Cairns T, Berquist H. 1980. Ebola hemorrhagic fever: Tandala, Zaire, 1977-1978. J Infect Dis. 142: 372-376.

149. Ho WE, Peh HY, Chan TK, Wong WS. 2014. Artemisinins: pharmacological actions beyond anti-malarial. Pharmacol The. 142: 126-139.

150. Hoenen T, Groseth A, Falzarano D, Feldmann H. 2006. Ebola virus: unravelling pathogenesis to combat a deadly disease. Trends Mol Med. 12: 206-215.

151. Hoenen T, Groseth A, Feldmann H. 2012. Current Ebola vaccines. Expert Opin Biol Ther. 12: 859-872.

152. Hoenen T, Groseth A, Rosenke K, Fischer RJ, Hoenen A, Judson SD Martellaro C, Falzarano D, Marzi A, Squires RB, Wollenberg KR, de Wit E, Prescott J, Safronetz D, van Doremalen N, Bushmaker T, Feldmann F, McNally K, Bolay FK, Fields B, Sealy T, Rayfield M, Nichol ST, Zoon KC, Massaquoi M, Munster VJ, Feldmann H. 2016. Nanopore Sequencing as a Rapidly Deployable Ebola Outbreak Tool. Emerg Infect Dis. 22(2): 331-334.

153. Hossain L, Kam D, Kong F, Wigand RT, Bossomaier T. 2016. Social media in Ebola outbreak. Epidemiol Infect. 4: 1-8.

154. Houssin T, Cramer J, Grojsman R, Bellahsene L, Colas G, Moulet H, Minnella W, Panne-tier C, Leberre M, Plecis A, Chen Y. 2016. Ultrafast, sensitive and large-volume on-chip real-time PCR for the molecular diagnosis of bacterial and viral infections. Lab Chip. 16: 1401-1411.

155. http://www.nature.com/scitable/blog/viruses101/the_scientist_who_discovered_ebola

156. http://www.theaustralian.com.au/news/world/the-times/russians-died-in-ebola-weapons-lab/news-story/e426b0bd0064da5fe8c78e4d485c162c

157. http://www.who.int/csr/disease/ebola/ebola-6-months/guinea/en/

158. http://www.who.int/wer/2003/en/wer7826.pdf

159. https://www.washingtonpost.com/national/health-science/2014/10/23/ce409716-5945-11e4-b812-38518ae74c67_story.html

160. Huang Y, Wei H, Wang Y, Shi Z, Raoul H, Yuan Z. 2012. Rapid detection of filoviruses by real-time TaqMan polymerase chain reaction assays. Virol Sin. 27: 273-277.

161. Huang Y, Zhu Y, Yang M, Zhang Z, Song D, Yuan Z. 2014. Nucleoprotein-based indirect enzyme-linked immunosorbent assay (indirect ELISA) for detecting antibodies specific to Ebola virus and Marburg virus. Virol Sin. 29(6): 372-380.

162. Iqbal HM, Villalba A, Khandia R, Munjal A, Dhama K. (2016). Recent trends in nanotechnology-based drugs and formulations for targeted therapeutic delivery. Recent Pat Inflamm Allergy Drug Discov. 2016 Dec 13. [Epub ahead of print]
163. Iversen PL, Warren TK, Wells JB, Garza NL, Mourich DV, Welch LS, Panchal RG, Bavari S. 2012. Discovery and early development of AVI-7537 and AVI-7288 for the treatment of Ebola virus and Marburg virus infections. Viruses 4:2806–2830.
164. Jääskeläinen AJ, Moilanen K, Aaltoinen K, Putkuri N, Sironen T, Kallio-Kokko H, Vapalahti O. 2015. Development and evaluation of a real-time EBOV-L-RT-qPCR for detection of Zaire ebolavirus. J Clin Virol. 67:56–58.
165. Jaax N, Jahrling P, Geisbert T, Geisbert J, Steele K, McKee K, Nagley D, Johnson E, Jaax G, Peters C. 1995. Transmission of Ebola virus (Zaire strain) to uninfected control monkeys in a biocontainment laboratory. Lancet. 346:1669–1671.
166. Jaax N, Jahrling P, Geisbert T, Geisbert J, Steele K, McKee K, Nagley D, Johnson E, Jaax G, Peters C. 1995. Transmission of Ebola virus (Zaire strain) to uninfected control monkeys in a biocontainment laboratory. Lancet. 346:1669–1671.
167. Jaax NK, Davis KJ, Geisbert TJ, Vogel P, Jaax GP, Topper M, Jahrling PB. 1996. Lethal experimental infection of rhesus monkeys with Ebola-Zaire (Mayinga) virus by the oral and conjunctival route of exposure. Arch Pathol Lab Med. 120:140–155.
168. Jacobsen KH, Aguirre AA, Bailey CL, Baranova AV, Crooks AT, Croitoru A, Delamater PL, Gupta J, Kehn-Hall K, Narayanan A, Pierobon M, Rowan KE, Schwebach JR, Seshaiyer P, Sklar ew DM, Stefanidis A, Agouris P. 2016. Lessons from the Ebola Outbreak: Action Items for Emerging Infectious Disease Preparedness and Response. Ecohealth. 13(1):200–212.
169. Jahrling PB, Geisbert TW, Dalgard DW, Johnson ED, Ksiazek TG, Hall WC, Peters CJ. 1990. Preliminary report: isolation of Ebola virus from monkeys imported to USA. Lancet. 335:502–505.
170. Jahrling PB, Joan B. Geisbert JB, Swearingen JR, Larsen T and Thomas W. Geisbert TW. 2007. Ebola hemorrhagic fever: evaluation of passive immunotherapy in nonhuman primates. J Infect Dis. 196 (Suppl.), S400–S403.
171. Jahromi MK, Mood BS. 2015. Ebola Hemorrhagic Fever. Int J Infect., 2(1): e22401.
172. Japtok L, Kleuser B. 2009. The role of sphingosine-1-phosphate receptor modulators in the prevention of transplant rejection and autoimmune diseases. Curr Opin Invest Drugs. 10:1183–1194.
173. Jasenosky LD, Neumann G, Lukashevich I, Kawaoka Y. 2001. Ebola virus VP40-induced particle formation and association with the lipid bilayer. J Virol. 75:5205–5214.
174. Jayme SI, Field HE, de Jong C, Olival KJ, Marsh G, Tagtag AM, Hughes T, Bucad AC, Barr J, Azul RR, Lilia MR, Foord A, Yu M, Cruz MS, Santos JJ, Lim TM, Benigno CC, Epstein JH, Wang L, Daszak P, Newman SN. 2015. Molecular evidence of Ebola Reston virus infection in Philippine bats. Virol J. 12:107.DOI:10.1186/s12985-015-0331-3.
175. Jensen van Vuren P, Grobbelaar A, Storm N, Conteh O, Konneh K, Kamara A, Sanne I, Paweska JT. 2016. Comparative evaluation of the diagnostic performance of the Prototype Cepheid GeneXpert Ebola Assay. J Clin Microbiol. 54(2): 359–367.
176. Jia K, Mohamed K. 2015. Evaluating the use of cell phone messaging for community Ebola syndromic surveillance in high risked settings in Southern Sierra Leone. Afr Health Sci. 15(3): 797–802.
177. Johansen LM, Brannan JM, Delos SE, Shoemaker CJ, Stossel A, Lear C, Hoffstrom B.G., Dewald LE, Schornberg KL, Scully C, Lehar J, Hensley LE, White JM, Olinger GG. 2013. FDA-approved selective estrogen receptor modulators inhibit Ebola virus infection. Sci Transl Med. 5: 190ra79.
178. Johnson E, Jaax N, White J, Jahrling P. 1995. Lethal experimental infections of rhesus monkeys by aerosolized Ebola virus. Int J Exp Pathol. 76:227–236.
179. Johnson JC, Martinez O, Honko AN, Hensley LE, Olinger GG, Basler CF. 2014. Pyridyl imidazole inhibitors of p38MAP kinase impair viral dendritic cells. Antiviral Res. 107:102–109.
180. Johnson KM, Lange JV, Webb PA, Murphy FA. 1977. Isolation and partial characterisation of a new virus causing acute haemorrhagic fever in Zaire. Lancet. 1:569–571.
181. Johnson KM. 1978. Ebola haemorrhagic fever in Zaire, 1976. Bull World Health Organ. 56:271–293.
182. Jones SM, Feldmann H, Ströher U, Geisbert JB, Fernando L, Grolla A, Klene HD, Sullivan NJ, Volchkov VE, Fritz EA, Daddario KM, Hensley LE, Jahrling PB, Geisbert TW. 2005. Live attenuated recombinant vaccine protects nonhuman primates against Ebola and Marburg viruses. Nat Med. 11(7):786–90.
183. Judson SD, Fischer R, Judson A, Munster VJ. 2016. Ecological contexts of index cases and spillover events of different Ebolaviruses. PLoS Pathog. 12(8):e1005780.
184. Jun SR, Leuze MR, Nookaew I, Uberbacher EC, Land M, Zhang Q, Wanchai V, Chai J, Nielsen M, Trolle T, Lund O, Buzzard GS, Pedersen TD, Wassenaar TM, Ussery DW 2015. Ebolavirus comparative genomics. FEMS Microbiol Rev. 39:764–778.
185. Kanduc D. 2016. Peptides for anti-Ebolavirus vaccines. Curr drug discov technol. In press.
35

186. Kangbai JB. 2016. Social network analysis and modeling of cellphone-based syndromic surveillance data for Ebola in Sierra Leone. Asian Pac J Trop Med. 9:851-855.
187. Khan AS, Tshioko FK, Heymann DL, Le Guenno B, Nabeth P, Kerstiëns B, Fleerackers Y, Kilmarx PH, Rodier GR, Nkuku O, Rollin PE, Sanchez A, Zaki SR, Swanepoel R, Tomori O, Nichol ST, Peters CJ, Muyembe-Tamfum JJ, Ksiazek TG. 1999. The reemergence of Ebola hemorrhagic fever, Democratic Republic of the Congo, 1995. Commission de Lutte contre les Epidémies à Kikwit. J Infect Dis. 179 (Suppl 1): S76–86.
188. Kibadi K, Mupapa K, Kuvula K, Massamba M, Ndaberey D, Muyembe-Tamfum JJ, Bwaka MA, De Roo A, Colebunders R. 1999. Late ophthalmologic manifestations in survivors of the 1995 Ebola virus epidemic in Kikwit, Democratic Republic of the Congo. J Infect Dis. 179 (Suppl 1): S13–14.
189. Kilgore PE, Grabenstein JD, Salim AM, Rybak M. 2015. Treatment of Ebola virus disease. Pharmacotherapy. 35: 43-53.
190. Kobinger GP, Leung A, Neufeld J, Richardson JS, Falzarano D, Smith G, Tierney K, Patel A, Weingartl HM. 2011. Replication, pathogenicity, shedding, and transmission of Zaire ebolavirus in pigs. J Infect Dis. 204: 200–208.
191. Konduru K, Shurtleff AC, Bradfute SB, Nakamura S, Bavari S, Kaplan G. 2016. Ebolavirus glycoprotein Fc fusion protein protects guinea pigs against lethal challenge. PLOS ONE. 11: e0162446.
192. Kost GJ, Ferguson WJ, Hoe J, Truong AT, Banpavichit A, Kongpila S. 2015. The Ebola Spatial Care Path™: Accelerating point-of-care diagnosis, decision making, and community resilience in outbreaks. Am J Disaster Med. 10(2): 121-43.
193. Krähling V, Becker D, Rohde C, Eickmann M, Eroğlu Y, Herwig A, Kerber R, Kowalski K, Vergara-Alert J, Becker S. 2016. European Mobile Laboratory consortium. Development of an antibody capture ELISA using inactivated Ebola Zaire Makona virus. Med Microbiol Immunol. 205(2): 173-183.
194. Kratz T, Roddy P, Tshomba Oloma A, Jeffs B, Pou Ciruelo D, de la Rosa O, Borchert M. 2015. Ebola virus disease outbreak in Isiro, Democratic Republic of the Congo, 2012: Signs and symptoms, management and outcomes. PLoS ONE. 10: e0129333.
195. Ksiazek TG, Rollin PE, Jahrling PB, Johnson E, Dalgard DW, Peters CJ. 1992. Enzyme immunosorbent assay for Ebola virus antigens in tissues of infected primates. J Clin Microbiol. 30(4): 947-950.
196. Ksiazek TG, Rollin PE, Williams AJ, Bressler DS, Martin ML, Swanepoel R, Burt FJ, Leman PA, Khan AS, Rowe AK, Mukunu R, Sanchez A, Peters CJ. 1999. Clinical virology of Ebola hemorrhagic fever (EHF): virus, virus antigen, and IgG and IgM antibody findings among EHF patients in Kikwit, Democratic Republic of the Congo, 1995. J Infect Dis. 179 (Suppl 1): S177-187.
197. Ksiazek TG, West CP, Rollin PE, Jahrling PB, Peters CJ. 1999. ELISA for the detection of antibodies to Ebola viruses. J Infect Dis. 179: S192-S198.
198. Kuehne A, Tiffany A, Lasry E, Janssens M, Besse C, Okonta C, Larbi K, Pah AC, Danis K, Porten K. 2016. Impact and lessons learned from mass drug administrations of malaria chemoprevention during the Ebola outbreak in Monrovia, Liberia, 2014. PloS one. 11: e0161311.
199. Kuhn JH, Becker S, Ebihara H, Geisbert TW, Johnson KM, Kawaoka Y, Lipkin WI, Negredo AI, Netesov SV, Nichol ST, Palacios G, Peters CJ, Tenorio A, Volchkov VE, Jahrling PB. 2010. Proposal for a revised taxonomy of the family Filoviridae: Classification, names of taxa and viruses, and virus abbreviations. Arch Virol. 155: 2083-2103.
200. Kuhn JH, Bao Y, Bavari S, Becker S, Bradfute S, Brister JR, Bukreyev AA, Cai Y, Chandran K, Davey RA, Dolnik O, Dye JM, Enterlein S, Gonzalez JP, Formenty P, Freiberg AN, Hensley LE, Honko AN, Ignatyev GM, Jahrling PB, Johnson KM, Klenk HD, Kobinger G, Lackemeyer MG, Leroy EM, Lever MS, Lofts LL, Mühlberger E, Netesov SV, Olinger GG, Palacios G, Patterson JL, Paweska JT, Pitt L, Radoshitzky SR, Ryabchikova EI, Saphire EO, Shestopalov AM, Smither SJ, Sullivan NJ, Swanepoel R, Takada A, Towner JS, van der Groen G, Volchkov VE, Wahl-Jensen V, Warren TK, Warfield KL, Weidmann M, Nichol ST. 2013. Virus nomenclature below the species level: a standardized nomenclature for laboratory animal-adapted strains and variants of viruses assigned to the family Filoviridae. Arch Virol. 158: 1425-1432.
201. Kuhn JH, Andersen KG, Baize S, Bào Y, Bavari S, Berthet N, Blinkova O, Brister JR, Clawson AN, Fair J, Gabriel M, Garry RF, Gire SK, Goba A, Gonzalez J, Günther S, Happi CT, Jahrling PB, Kapetshi J, Kobinger G, Kugelman JR, Leroy EM, Maganga GD, Mbala PK, Moses LM, Muyembe-Tamfum J, N'Faly M, Nichol ST, Omilabu SA, Palacios G, Park DJ, Paweska JT, Radoshitzky SR, Rossi CA, Sabeti PC., Schieffelin JS., Schoepp RJ., Sealfon R., Swanepoel R., Towner JS., Wada J., Wauquier N., Yozwiak NL and Formenty P. 2014a. Nomenclature- and database-compatible names for the two Ebola virus variants that emerged in Guinea and the Democratic Republic of the Congo in 2014. Viruses. 6:4760-4799.
202. Kuhn JH, Bào Y, Bavari S, Becker S, Brauburger K, Rodney Brister J, Bukreyev AA, Cai Y, Chandran K, Davey RA, Dolnik O, Dye JM, Enterlein S, Gonzalez JP, Formenty P, Freiberg AN, Hensley

LE, Hoenen T, Honko AN, Ignatyev GM, Jahrling PB, Johnson KM, Klenk HD, Kobinger
G, Lackemeyer MG, Leroy EM, Lever MS, Mühlberger E, Netesov SV, Olinger GG, Palacios
G, Patterson JL, Paweska JT, Pitt L, Radoshitzky SR, Ryabchikova EI, Saphire EO, Shestopalov
AM, Smithers SJ, Sullivan NJ, Swanepoel R, Takada A, Towner JS, van der Groen G, Volchkov
VE, Volchkova VA, Wahl-Jensen V, Warren TK, Warfield KL, Weidmann M, Nichol ST. 2014b. Virus nomenclature below the species level: a standardized nomenclature for filovirus strains and variants rescued from cDNA. Arch Virol. 159:1229–1237.
203. Kurata T, Hondo R, Sato S, Oda A, Aoyama Y, McCormick JB. 1983. Detection of viral antigens in formalin-fixed specimens by enzyme treatment. Ann N Y Acad Sci. 420:192-207.
204. Kurosaki Y, Magassouba N, Oloniniyi OK, Cherif MS, Sakabe S, Takada A, Hirayama K, Yasuda J. 2016. Development and Evaluation of Reverse Transcription-Loop-Mediated Isothermal Amplification (RT-LAMP) Assay Coupled with a Portable Device for Rapid Diagnosis of Ebola Virus Disease in Guinea. PLoS Negl Trop Dis. 10(2): e0004472.
205. Kurosaki Y, Takada A, Ebihara H, Grolla A, Kamoc N, Feldmann H, Kawaoka Y, Yasuda J. 2007. Rapid and simple detection of Ebola virus by reverse transcription-loop-mediated isothermal amplification. J Virol Methods. 141:78-83.
206. Lahm SA, Kombila M, Swanepoel R, Barnes RFW. 2007. Morbidity and mortality of wild animals in relation to outbreaks of Ebola haemorrhagic fever in Gabon, 1994-2003. Trans R Soc Trop Med Hyg. 101:64-78.
207. Lawrence P, Danet N, Reynard O, Volchkova V, Volchkov V. 2017. Human transmission of Ebola virus. Current Opinion in Virology 22:51-58.
208. Lazard AJ, Scheinfeld E, Bernhardt JM, Wilcox GB, Suran M. 2015. Detecting themes of public concern: A text mining analysis of the Centers for Disease Control and Prevention's Ebola live Twitter chat. Am J Infect Control. 43(10): 1109-1111.
209. Le Guenno B, Formenty P, Formentry P, Wyers M, Gounon P, Walker F, Boesch C. 1995. Isolation and partial characterisation of a new strain of Ebola virus. Lancet. 345:1271-1274.
210. Ledgerwood JE, Costner P, Desai N, Holman L, Enama ME, Yamshchikov G, Mulangu S, Hu Z, Andrews CA, Sheets RA, Koup RA, Roederer M, Bailer R, Mascola JR, Pau MG, Sullivan NJ, Goudsmit J, Nabel GJ, Graham BS. 2010. A replication defective recombinant AD5 vaccine expressing Ebola virus GP is safe and immunogenic in healthy adults. Vaccine 29:304-313.
211. Lee JE, Saphire EO. 2009. Ebolavirus glycoprotein structure and mechanism of entry. Future Virol. 4:621-635.
212. Leroy E, Epelboin A, Mondonge V, Pourrut X, Gonzalez JP, Muyembe-Tamfum JJ, Formenty P. 2009. Human Ebola outbreak resulting from direct exposure to fruit bats in Luebo, Democratic Republic of Congo, 2007. Vector Borne Zoonotic Dis. 9:723-728.
213. Leroy E, Gonzalez JP, Pourrut X. 2007. Ebolavirus and other filoviruses. In Wildlife and Emerging Zoonotic Diseases: The Biology, Circumstances and Consequences of Cross-Species Transmission (pp. 363-387). Springer Berlin Heidelberg.
214. Leroy EM, Baize S, Lu CY, McCormick JB, Georges AJ, Georges-Courbot MC, Lansoud-Soukate J, Fisher-Hoch SP. 2000. Diagnosis of Ebola haemorrhagic fever by RT-PCR in an epidemic setting. J Med Virol. 60(4): 463-467.
215. Leroy EM, Kumulungui B, Pourrut X, Rouquet P, Hassanin A, Yaba P, Delicat A, Paweska JT, Gonzalez J-P, Swanepoel R. 2005. Fruit bats as reservoirs of Ebola virus. Nature. 438:575-576.
216. Leroy EM, Rouquet P, Formenty P, Souquière S, Kilbourne A, Froment J-M, Bermejo M, Smit S, Karesh W, Swanepoel R, Zaki SR, Rollin PE. 2004. Multiple Ebola virus transmission events and rapid decline of central African wildlife. Science. 303:387-390.
217. Leski TA, Ansumana R, Taitt CR, Lamin JM, Bangura U, Lahai J, Mbayo G, Kanneh MB, Bawo B, Bockarie AS, Scullion M, Phillips CL, Horner CP, Jacobsen KH, Stenger DA. 2015. Use of the FilmArray System for Detection of Zaire ebolavirus in a Small Hospital in Bo, Sierra Leone. J Clin Microbiol. 53(7): 2368-2370.
218. Liu L, Sun Y, Kargbo B, Zhang C, Feng H, Lu H, Liu W, Wang C, Hu Y, Deng Y, Jiang J, Kang X, Yang H, Jiang Y, Yang Y, Kargbo D, Qian J, Chen W. 2014. Detection of Zaire Ebola virus by real-time reverse transcription-polymerase chain reaction, Sierra Leone, 2014. J Virol Methods. 222:62-65.
219. Liu Y, Shi ZX, Ma YK, Wang HT, Wang ZY, Shao DH, Wei JC, Wang SH, Li BB, Wang SM, Liu XH, Ma ZY. 2012. Development of SYBR Green I real-time RT-PCR for the detection of Ebola virus. Bing Du Xue Bao. 28:567-571.
220. Lovgren Bengtsson K, Morein B, Osterhaus AD. 2011. ISCOM technology-based Matrix M adjuvant: success in future vaccines relies on formulation. Expert Rev Vaccines, 10(4): 401-403
221. Lucht A, Formenty P, Feldmann H, Gotz M, Leroy E, Bataboukila P, Grolla A, Feldmann F, Wittmann T, Campbell P, Atsangandoko C, Boumandoki P, Finke EJ, Miethe P, Becker S, Grunow R. 2007.
37

Development of an immunofiltration-based antigen-detection assay for rapid diagnosis of Ebola virus infection. J Infect Dis. 196(Suppl 2): S184-192.

222. Lucht A, Grunow R, Otterbein C, Möller P, Feldmann H, Becker S. 2004. Production of monoclonal antibodies and development of an antigen capture ELISA directed against the envelope glycoprotein GP of Ebola virus. Med Microbiol Immunol. 193(4): 181-7.

223. Ludtke A, Ruibal P, Becker-Ziaja B, Rottstegge M, Wozniak DM, Cabeza-Cabrero M., Thorenz A, Weller R, Kerber R, Idoyaga J, Magassouba NF, Gabriel M, Gunther S, Oestereich L, Munoz-Fontela C. 2016. Ebola Virus disease is characterized by poor activation and reduced levels of circulating CD16+ monocytes. J Infect Dis. 260.

224. M Alam El-Din H, A Loutfy S, Fathy N, H Elberry M, M Mayla A, Kassem S, Naqvi A. 2016. Molecular docking based screening of compounds against VP40 from Ebola virus. Bioinformation. 12(3): 192-196.

225. Mackay IM, Arden KE. 2015. Ebola virus in the semen of convalescent men. Lancet Infect Dis. 15(2): 149-150.

226. MacNeil A, Farnon EC, Morgan OW, Gould P, Boehmer TK, Blaney DD, Wiersma P, Tappero JW, Nichol ST, Ksiazek TG, Rollin PE. 2011. Filovirus outbreak detection and surveillance: lessons from Bundibugyo. J Infect Dis. 204(Suppl 3): S761–767.

227. Madrid PB, Chopra S, Manger ID, Gilfillan L, Keepers TR, Shurtleff AC, Green CE, Iyer LV, Dilks HH, Davey RA, Kolokoltsov AA, Carrion R, Jr, Patterson JL, Bavari S, Panchal RG, Warren TK, Wells JB, Moos WH, Burke RL, Tanga MJ. 2013. A systematic screen of FDA-approved drugs for inhibitors of biological threat agents. PLoS One. 8: e60579.

228. Maganga GD, Kapetshi J, Berthet N, Kebela Ilunga B, Kabange F, Mbala Kingebeni P, Mondonge V, Muyembe J-JT, Bertherat E, Briand S, Cabore J, Epelboin A, Formenty P, Kobinger G, Gonzalez-Angulo L, Labouba I, Okwo-Bele J, Dye C, Leroy EM. 2014. Ebola virus disease in the Democratic Republic of Congo. N Engl J Med. 371: 2083–2091.

229. Makino A, Kawaoka Y. 2009. Generation of vero cells expressing ebola virus glycoprotein. J Vet Med Sci. 71(4): 505-507.

230. Malik YS, Sharma K, Jeena LM, Kumar N, Sircar S, Rajak KK, Dhama K. 2013. Toll-like receptors: the innate immune receptors with ingenious anti-viral paradigm. South Asian J Exp Biol 3: 207-213.

231. Mann E, Streng S, Bergeron J, Kircher A. 2015. A Review of the Role of Food and the Food System in the Transmission and Spread of Ebolavirus. PLoS Negl Trop Dis. 9: e0004160.

232. Maras MH, Miranda MD. 2016. The weaponization of Ebola: A new risk in the wake of an outbreak?. Comp Strategy. 35(1): 72-79.

233. Martin P, Laupland KB, Frost EH, Valiquette L. 2015. Laboratory diagnosis of Ebola virus disease. Intensive Care Med. 41(5): 895-898.

234. Martins KA, Jahrling PB, Bavari S, Kuhn JH. 2016. Ebola Virus Disease Candidate Vaccines Under Evaluation in Clinical Trials. Expert Rev Vaccines. 15(9): 1101-12.

235. Mate SE, Kugelman JR, Nyenswah TG, Ladner JT, Wiley MR, Cordier-Lassalle T, Christie A, Schroth GP, Gross SM, Davies-Wayne GJ, Shinde SA, Murugan R, Sieh SB, Badio M, Fakoli L, Taweh F, de Wit E, van Doremalen N, Munster VJ, Pettitt J, Prieto K, Humrighouse BW, Ströher U, DiClaro JW, Hensley LE, Schoepp RJ, Safronetz D, Fair J, Kuhn JH, Blackley DJ, Laney AS, Williams DE, Lo T, Gasasira A, Ch.B., Nichol ST, Formenty P, Kateh FN, De Cock KM, Bolay F, Sanchez-Lockhart M, and Palacios G. 2015. Molecular Evidence of Sexual Transmission of Ebola Virus. N Engl J Med. 373: 2448–2454.

236. Mattia JG, Vandy MJ, Chang JC, Platt DE, Dierberg K, Bausch DG, Brooks T, Conteh S, Crozier I, Fowler RA, Kamara AP, Kang C, Mahadevan S, Mansaray Y, Marcell L, McKay G, O'Dempsey T, Parris V, Pinto R, Rangel A, Salam AP, Shantha J, Wolfman V, Yeh S, Chan AK, Mishra S. 2016. Early clinical sequelae of Ebola virus disease in Sierra Leone: a cross-sectional study. Lancet Infect Dis. 16(3): 331-338.

237. McCormick JB, Bauer SP, Elliott LH, Webb PA, Johnson KM. 1983. Biologic differences between strains of Ebola virus from Zaire and Sudan. J Infect Dis. 147: 264–267.

238. McElroy AK, Akondy RS, Davis CW, Ellebedy AH, Mehta AK, Kraft CS, Lyon GM, Ribner BS, Varkey J, Sidney J, Sette A, Campbell S, Ströher U, Damon I, Nichol ST, Spiropoulou CF, Ahmed R. 2015. Human Ebola virus infection results in substantial immune activation. Proc Natl Acad Sci. 112: 4719–4724.

239. McElroy AK, Erickson BR, Flietstra TD, Rollin PE, Nichol ST, Towner JS, Spiropoulou CF. 2014. Ebola hemorrhagic Fever: novel biomarker correlates of clinical outcome. J Infect Dis. 210(4): 558-566.

240. Mendoza EJ, Qiu X, Kobinger GP. 2015. Progression of Ebola Therapeutics During the 2014-Outbreak. Trends Mol Med. 22(2): 164–173

241. Miller LA, Stanger E, Senesi RG, DeLuca N, Dietz P, Hausman L, Kilmarx PH, Mermin J. 2015. Centers for Disease Control and Prevention. Use of a nationwide call center for Ebola response and

monitoring during a 3-day house-to-house campaign - Sierra Leone, September 2014. MMWR Morb Mortal Wkly Rep. 64(1): 28-29.

242. Millman AJ, Chamany S, Guthartz S, Thihalolipavan S, Porter M, Schroeder A, Vora NM, Varma JK, Starr D. 2014. Active monitoring of travelers arriving from ebola-affected countries - New York City, October 2014-April 2015. MMWR Morb Mortal Wkly Rep. 65(3): 51-54.

243. Miranda ME, White ME, Dayrit MM, Hayes CG, Ksiazek TG, Burans JP. 1991. Seropidemiological study of filovirus related to Ebola in the Philippines. Lancet. 337: 425-426.

244. Miranda MEG, Miranda NLJ. 2011. Reston ebolavirus in humans and animals in the Philippines: a review. J Infect Dis. 204 (Suppl 3): S757–760.

245. Mire CE, Geisbert JB, Agans KN, Thi EP, Lee AC, Fenton KA, Geisbert TW. 2016. Passive immunotherapy: assessment of convalescent serum against Ebola virus Makona infection in nonhuman primates. J Infect Dis. 333.

246. Mire CE, Geisbert JB, Marzi A, Agans KN, Feldmann H, Geisbert TW. 2013. Vesicular stomatitis virus-based vaccines protect nonhuman primates against Bundibugyo ebolavirus. PLoS Negl Trop Dis. 7: e2600.

247. Mishra B. 2014. The threat of Ebola: An update. Indian J Med Microbiol. 32: 364-370.

248. Morvan JM, Deubel V, Gounon P, Nakoune E, Barrière P, Murri S, Perpète O, Selekon B, Coudrier D, Gautier-Hion A, Colyn M, Volehkov V. 1999. Identification of Ebola virus sequences present as RNA or DNA in organs of terrestrial small mammals of the Central African Republic. Microbes Infect. 1: 1193–1201.

249. Mufunda J, Ndambakuwa Y, Muno-dawafa D, Kobie A. 2016. Is a total ban on business and consumption of bush-meat a sustainable end game for eb-ola outbreak in West Africa: but why now. Public Health Open J. 1(1): 4-7.

250. Muhlberger E, Weik M, Volchkov VE, Klenk HD, Becker S. 1999. Comparison of the transcription and replication strategies of Marburg virus and Ebola virus by using artificial replication systems. J Virol. 73: 2333-2342.

251. Muyembe-Tamfum JJ, Kipasa M, Kiyungu C, Colebunders R. 1999. Ebola outbreak in Kikwit, Democratic Republic of the Congo: discovery and control measures. J Infect Dis. 179 (Suppl 1): S259–262.

252. Muyembe-Tamfum JJ, Mulangu S, Masumu J, Kayembe JM, Kemp A, Paweska JT. 2012. Ebola virus outbreaks in Africa: past and present. Onderstepoort J Vet Res. 79: 451.

253. Nagata T, Lefor AK, Hasegawa M, Ishii M. 2015. Favipiravir: a new medication for the Ebola virus disease pandemic. Disaster Med Public Health Prep. 9(1): 79-81.

254. Nei M, Kumar S. 2000. Molecular Evolution and Phylogenetics. Oxford University Press, New York.

255. Nkoghe D, Formenty P, Leroy EM, Nnegue S, Edou SYO, Ba JI, Allarangar Y, Cabore J, Bachy C, Andraghetti R, de Benoist AC, Galanis E, Rose A, Bausch D, Reynolds M, Rollin P, Choueibou C, Shongo R, Gergonne B, Koné LM, Yada A, Roth C, Mve MT. 2005b. Multiple Ebola virus haemorrhagic fever outbreaks in Gabon, from October 2001 to April 2002. Bull Soc Pathol Exot. 98: 224–229.

256. Nkoghe D, Nnegue S, Mve MT, Formenty P, Thompson G, Iba Ba J, Okome Nkoumou M, Leroy E. 2005a. Isolated case of haemorrhagic fever observed in Gabon during the 2002 outbreak of Ebola but distant from epidemic zones. Médecine Trop Rev Corps Santé Colon. 65: 349–354.

257. Noda T, Sagara H, Suzuki E, Takada A, Kida H, Kawaoka Y. 2002. Ebola virus VP40 drives the formation of virus-like filamentous particles along with GP. J Virol. 76: 4855-4865.

258. Nordenstedt H, Bah EI, de la Vega MA, Barry M, N'Faly M, Barry M, Crahay B, Decroo T, Van Herp M, Engelbeen B. 2016. Ebola virus in breast milk in an ebola virus-positive mother with twin babies, Guinea, 2015. Emerg Infect Dis. 22(4): 759-760.

259. Nouvellet P, Garske T, Mills HL, Nedjati-Gilani G, Hinsley W, Blake IM, Van Kerkhove MD, Cori A, Dorigatti I, Jombart T, Riley S, Fraser C, Donnelly CA, Ferguson NM. 2015. The role of rapid diagnostics in managing Ebola epidemics. Nature. 528(7580): S109-116.

260. Nyamathi AM, Fahey JL, Sands H, Casillas AM. 2003. Ebola virus: immune mechanisms of protection and vaccine development. Biol Res Nurs. 4: 276-281.

261. Oestereich L, Ludtke A, Wurr S, Rieger T, Muñoz-Fontela C, Gunther S. 2014. Successful treatment of advanced Ebola virus infection with T-705 (favipiravir) in a small animal model. Antiviral Res. 105: 17-21.

262. Okeke IN, Manning RS, Pfeiffer T. 2014. Diagnostic schemes for reducing epidemic size of african viral hemorrhagic fever outbreaks. J Infect Dev Ctries. 8: 1148-1159.

263. Okware SI, Omaswa F, Talisuna A, Amandua J, Amone J, Onek P, Opio A, Warnala J, Lubwama J, Luswa L, Kagwa P, Tylleskar T. 2015. Managing Ebola from rural to urban slum settings: experiences from Uganda. Afr Health Sci. 15(1): 312-321.

264. Okware SI, Omaswa FG, Zaramba S, Opio A, Lutwama JJ, Kamugisha J, Rwaguma EB, Kagwa P, Lamunu M. 2002. An outbreak of Ebola in Uganda. Trop Med Int Health. 7:1068–1075.
265. Okware SI, Omaswa FG, Zaramba S, Opio A, Lutwama JJ, Kamugisha J, Rwaguma EB, Kagwa P, Lamunu M. 2002. An outbreak of Ebola in Uganda. Trop Med Int Health. 7:1068–1075.
266. Olejnik J, Ryabchikova E, Corley RB, Muhlberger E. 2011. Intracellular events and cell fate in filovirus infection. Viruses 3:1501–1531.
267. Olival KJ, Islam A, Yu M, Anthony SJ, Epstein JH, Khan SA, Khan SU, Crameri G, Wang LF, Lipkin WI, Luby SP, Daszak P. 2013 Ebola virus antibodies in fruit bats, Bangladesh. Emerg Infect Dis. 19: 270.
268. Olsen NJ, Schleich MA, Karp DR. 2013. Multifaceted effects of hydroxychloroquine in human disease. Semin Arthritis Rheum. 43:264–272.
269. Olupot-Olupot P. 2015. Ebola in children: epidemiology, clinical features, diagnosis and outcomes. Pediatr Infect Dis J. 34(3): 314-316.
270. Osterholm MT, Moore KA, Kelley NS, Brosseau LM, Wong G, Murphy FA, Peters CJ, LeDuc JW, Russell PK, Van Herp M, Kapetshi J, Muyembe J-JT, Ilunga BK, Strong JE, Grolla A, Wolz A, Kargbo B, Kargbo DK, Formenty P, Sanders DA, Kobinger GP. 2015. Transmission of Ebola viruses: what we know and what we do not know. MBio. 6: e00137.
271. Pan Y, Zhang W, Cui L, Hua X, Wang M, Zeng Q. 2014. Reston virus in domestic pigs in China. Arch Virol. 159:1129–1132.
272. Pang Z, Li A, Li J, Qu J, He C, Zhang S, Li C, Zhang Q, Liang M, Li D. 2014. Comprehensive multiplex one-step real-time TaqMan qRT-PCR assays for detection and quantification of hemorrhagic fever viruses. PLoS One. 9(4): e95635.
273. Pennisi E. Genomics. 2016. Pocket DNA sequencers make real-time diagnostics a reality. Science. 351(6275):800-801.
274. Pettitt J, Higgs ES, Adams RD, Jahrling PB, Hensley LE. 2016. Use of existing diagnostic reverse-transcription polymerase chain reaction assays for detection of ebola virus RNA in semen. J Infect Dis. 213(8):1237-1239.
275. Piercy TJ, Smithers SJ, Steward JA, Eastaugh L, Lever MS. 2010. The survival of filoviruses in liquids, on solid substrates and in a dynamic aerosol. J Appl Microbiol. 109(5): 1531-1539.
276. Pigott DM, Golding N, Mylne A, Huang Z, Henry AJ, Weiss DJ, Brady OJ, Kraemer MU, Smith DL, Moyes CL, Bhatt S, Gething PW, Horby PW, Bogoch II, Brownstein JS, Mekaru SR, Tatem AJ, Khan K, Hay SI. 2014. Mapping the zoonotic niche of Ebola virus disease in Africa. Elife. 7: e04395.
277. Pinsky BA, Sahoo MK, Sandlund J, Kleman M, Kulkarni M, Grufman P, Nygren M, Kwiatkowski R, Baron EJ, Tenover F, Denison B, Higuchi R, Van Atta R, Beer NR, Carrillo AC, Naraghi-Arani P, Mire CE, Ranadheera C, Grolla A, Lagerqvist N, Persing DH. 2015. Analytical Performance Characteristics of the Cepheid GeneXpert Ebola Assay for the Detection of Ebola Virus. PLoS One. 10(11): e0142216.
278. Pleet ML, DeMarino C, Lepene B, Aman MJ, Kashanchi F. 2017. The Role of Exosomal VP40 in Ebola Virus Disease. DNA Cell Biol. 2017 Feb 8. doi: 10.1089/dna.2017.3639.
279. Pourrut X, Délicat A, Rollin PE, Ksiazek TG, Gonzalez JP, Leroy EM. 2007. Spatial and temporal patterns of Zaire ebolavirus antibody prevalence in the possible reservoir bat species. J Infect Dis. 196 (Suppl 2): S176–183.
280. Pourrut X, Kumulungui B, Wittmann T, Moussavou G, Délicat A, Yaba P, Nkoghe D, Gonzalez JP, Leroy EM. 2005. The natural history of Ebola virus in Africa. Microbes Infect. 7:1005–1014.
281. Pourrut X, Souris M, Towner JS, Rollin PE, Nichol ST, Gonzalez J-P, Leroy E. 2009. Large serological survey showing cocirculation of Ebola and Marburg viruses in Gabonese bat populations, and a high seroprevalence of both viruses in Rousettus aegyptiacus. BMC Infect Dis. 9: 159.
282. Purpura LJ, Soka M, Baller A, White S, Rogers E, Choi MJ, Mahmoud N, Wasunna C, Massaquoi M, Vanderende K, Kollie J, Dweh S, Bemah P, Christie A, Ladele V, Subah O, Pillai S, Mugisha M, Kpaka J, Nichol S, Ströher U, Abad N, Mettee-Zarecki S, Bailey JA, Rollin P, Marston B, Nyenswah T, Gasasira A, Knust B, Williams D. 2016. Implementation of a national semen testing and counseling program for male Ebola survivors - Liberia, 2015-2016. In press.
283. Pushko P, Bray M, Ludwig GV, Parker M, Schmaljohn A, Sanchez A, Jahrling PB, Smith JF. 2000. Recombinant RNA replicons derived from attenuated Venezuelan equine encephalitis virus protect guinea pigs and mice from Ebola hemorrhagic fever virus. Vaccine 2000; 19:142e53
284. Pushko P, Bray M, Ludwig GV, Parker M, Schmaljohn A, Sanchez A, Jahrling PB, Smith JF. 2003. Ebola virus-like particles protect from lethal Ebola virus infection. Proc Natl Acad Sci. 100:15889e94.
285. Qiu X, Fernando L, Alimonti JB, Melito PL, Feldmann F, Dick D, Ströher U, Feldmann H, Jones SM. 2009. Mucosal immunization of cynomolgus macaques with the VSVDeltaG/ZEBOVGP vaccine stimulates strong ebola GP-specific immune responses. PLoS One. 4(5): e5547.
286. Quaglio G, Goerens C, Putoto G, Rübig P, Lafaye P, Karapiperis T, Dario C, Delaunois P, Zachariah R. 2016. Ebola: lessons learned and future challenges for Europe. Lancet Infect Dis. 16(2): 259-263.

287. Quick J, Loman NJ, Duraffour S, Simpson JT, Severi E, Cowley L, Bore JA, Koundouno R, Dudas G, Mikhail A, Ouédraogo N, Afrough B, Bah A, Baum JH, Becker-Ziaja B, Boettcher JP, Cabeza-Cabrero M, Camino-Sánchez Á, Carter LL, Doerrbecker J, Enkirch T, García-Dorival I, Hetzelt N, Hinzmann J, Holm T, Kafetzopoulou LE, Koropogui M, Kosgey A, Kuisma E, Logue CH, Mazzarelli A, Meisel S, Mertens M, Michel J, Ngabo D, Nitzsche K, Pallasch E, Patrono LV, Portmann J, Repits JG, Rickett NY, Sachse A, Singethan K, Vitoriano I, Yemanaberhan RL, Zekeng EG, Racine T, Bello A, Sall AA, Faye O, Faye O, Magassouba N, Williams CV, Amburgey V, Winona L, Davis E, Gerlach J, Washington F, Monteil V, Jourdain M, Bererd M, Camara A, Somlare H, Camara A, Gerard M, Bado G, Baillet B, Delaune D, Nebie KY, Diarra A, Savane Y, Pallawo RB, Gutierrez GJ, Milhano N, Roger I, Williams CJ, Yattara F, Lewandowski K, Taylor J, Rachwal P, Turner DJ, Pollakis G, Hiscox JA, Matthews DA, O'Shea MK, Johnston AM, Wilson D, Hutley E, Smit E, Di Caro A, Wölfel R, Stoecker K, Fleischmann E, Gabriel M, Weller SA, Koivogui L, Diallo B, Keita S, Rambaut A, Formenty P, Günther S, Carroll MW. 2016. Real-time, portable genome sequencing for Ebola surveillance. Nature. 530(7589):228-232.

288. Qureshi AI, Chughtai M, Loua TO, Pe Kolie J, Camara HFS, Ishfaq MF, N'Dour CT, Beavogui K. 2015. Study of Ebola virus disease survivors in Guinea. Clin Infect Dis. 61:1035–1042.

289. Racsa LD, Kraft CS, Olinger GG, Hensley LE. 2016. Viral Hemorrhagic fever diagnostics. Clin Infect Dis. 62(2):214-219.

290. Rajiah K, San KP, Jiun TW, May TA, Neng YC, Seng HK, Soon LJ, Pazooki N. 2015. Prevalence and Current Approaches of Ebola Virus Disease in ASEAN Countries. J Clin Diagn Res. 2015.9(9):LE01-6. doi:10.7860/JCDR/2015/13364.6429. Epub Sep 1.

291. Read JM, Diggle PJ, Chirombo J, Solomon T, Baylis M. 2015. Effectiveness of screening for Ebola at airports. The Lancet. 385(9962):23-24.

292. Reed DS, Lackemeyer MG, Garza NL, Sullivan LJ, Nichols DK. 2011. Aerosol exposure to Zaire ebolavirus in three nonhuman primate species: differences in disease course and clinical pathology. Microbes Infect. 13:930–936.

293. Report of a WHO/International Study Team. 1978. Ebola haemorrhagic fever in Sudan, 1976. Bull World Health Organ. 56:247–270.

294. Report of an International Commission. 1978. Ebola haemorrhagic fever in Zaire, 1976. Bull World Health Organ. 56:271–293.

295. Rewar S, Mirdha D. 2014. Transmission of ebola virus disease: an overview. Ann Glob Health. 80:444–451.

296. Richman DD, Cleveland PH, McCormick JB, Johnson KM. 1983. Antigenic analysis of strains of Ebola virus: identification of two Ebola virus serotypes. J Infect Dis. 147:268–271.

297. Roels TH, Bloom AS, Buffington J, Muhungu GL, Mac Kenzie WR, Khan AS, Ndambi R, Noah DL, Rolka HR, Peters CJ, Ksiazek TG. 1999. Ebola hemorrhagic fever, Kikwit, Democratic Republic of the Congo, 1995: risk factors for patients without a reported exposure. J Infect Dis. 179(Suppl 1):S92–97.

298. Rollin PE, Williams RJ, Bressler DS, Pearson S, Cottingham M, Pucak G, Sanchez A, Trappier SG, Peters RL, Greer PW, Zaki S, Demarcus T, Hendricks K, Kelley M, Simpson D, Geisbert TW, Jahrling PB, Peters CJ, Ksiazek TG. 1999. Ebola (subtype Reston) virus among quarantined nonhuman primates recently imported from the Philippines to the United States. J Infect Dis. 179(Suppl 1):S108–114.

299. Rouquet P, Froment J-M, Bermejo M, Kilbourn A, Karesh W, Reed P, Kumulungui B, Yaba P, Délicat A, Rollin PE, Leroy EM. 2005. Wild animal mortality monitoring and human Ebola outbreaks, Gabon and Republic of Congo, 2001-2003. Emerg Infect Dis. 11(2):283–290.

300. Rowe AK, Bertolli J, Khan AS, Mukunu R, Muyembe-Tamfum JJ, Bressler D, Williams AJ, Peters CJ, Rodriguez L, Feldmann H, Nichol ST, Rollin PE, Ksiazek TG. 1999. Clinical, virologic, and immunologic follow-up of convalescent Ebola hemorrhagic fever patients and their household contacts, Kikwit, Democratic Republic of the Congo. Commission de Lutte contre les Epidémies à Kikwit. J Infect Dis. 179(Suppl 1):S28–35.

301. Safari S, Baratloo A, Rouhipour A, Ghelichkhani P, Mamood Yousefifard M. 2015. Ebola Hemorrhagic Fever as a Public Health Emergency of International Concern; a Review Article. Emerg (Tehran). 3(1):3–7.

302. Saitou N, Nei M. 1987. The neighbor-joining method: A new method for reconstructing phylogenetic trees. Mol Biol Evolut. 4:406-425.

303. Sakurai Y, Kolokoltsov AA, Chen CC, Tidwell MW, Bauta WE, Klugbauer N, Grimm C, Wahl-Schott C, Biel M, Davey RA. 2015. Two-pore channels control Ebola virus host cell entry and are drug targets for disease treatment. Science. 347(6225):995-998.

304. Salman M, Shehzadi N, Hussain K, Saleem F, Khan MT, Asif N, Yousaf M, Rafique M, Bedar R, Tariq S, Abu Bakar U, Syed Sulaiman SA. 2017. Knowledge of Ebola virus disease among a university population: A cross-sectional study. Am J Infect Control. 2017 Feb 1; 45(2):e23-e25.

305. Sanchez A, Kiley MP, Holloway BP, Auperin DD. 1993. Sequence analysis of the Ebola virus genome: organization, genetic elements, and comparison with the genome of Marburg virus. Virus Res. 29: 215–240.

306. Sanchez A, Ksiazek TG, Rollin PE, Miranda ME, Trappier SG, Khan AS, Peters CJ, Nichol ST. 1999. Detection and molecular characterization of Ebola viruses causing disease in human and nonhuman primates. J Infect Dis. 179 (Suppl 1): S164–169.

307. Sanchez A. 2007. Analysis of filovirus entry into vero E6 cells, using inhibitors of endocytosis, endosomal acidification, structural integrity, and cathepsin (B and L) activity. J Infect Dis. 196 (Suppl 2): S251-258.

308. Saphire EO. 2013. An update on the use of antibodies against the filoviruses. Immunotherapy. 5: 1221–1233.

309. Saphire EO. 2015. New Advances in the Effort against Ebola. Cell Host Microbe. 17(5): 545–547.

310. Sarwar UN, Costner P, Enama ME, Berkowitz N, Hu Z, Hendel CS, Sitar S, Plummer S, Mulangu S, Bailer RT, Koup RA, Mascola JR, Nabel GJ, Sullivan NJ, Graham BS, Ledgerwood JE. 2015. Safety and immunogenicity of DNA vaccines encoding Ebolavirus and Marburgvirus wild-type glycoproteins in a phase I clinical trial. J Infect Dis. 211: 549–557

311. Scott JT, Sesay FR, Massaquoi TA, Idriss BR, Sahr F, Semple MG. 2016. Post-Ebola Syndrome, Sierra Leone. Emerg Infect Dis. 22: 641–646.

312. Shabman RS, Leung DW, Johnson J, Glennon N, Gulcicek EE, Stone KL, Leung L, Hensley L, Amarasinghe GK, Basler CF. 2011. DRBP 76 associates with Ebola virus VP 35 and suppresses viral polymerase function. J Infect Dis. 204: S911-S918.

313. Shah NR, Sharma S, Waters L, Arango C, Scuderi C. 2015. Ebola virus disease: an update on diagnosis, treatment, and prevention. Consultant. 55(12): 988-994.

314. Shantha JG, Crozier I, Varkey JB, Kraft CS, Lyon GM, Mehta AK, Carlson RD, Hill CE, Kumar G, Debiec MR, Patel PS, Olsen TW, Nussenblatt RB, Martin DF, Stroher U, Uyeki TM, Ribner BS, Smith JR, Yeh S. 2016. Long-term Management of Panuveitis and Iris Heterochromia in an Ebola Survivor. Ophthalmology 123(12): 2626–2628.e2.

315. Shantha JG, Crozier I, Hayek BR, Bruce BB, MPH, Gargu C, Brown J, Fankhauser J, Yeh S. 2017. Ophthalmic manifestations and causes of vision impairment in Ebola virus disease survivors in Monrovia, Liberia. Ophthalmology 124(2): 170–177.

316. Sheng M, Zhong Y, Chen Y, Du J, Ju X, Zhao C, Zhang G, Zhang L, Liu K, Yang N, Xie P, Li D, Zhang MQ, Jiang C. 2014. Hsa-miR-1246, hsa-miR-320a and hsa-miR-196b-5p inhibitors can reduce the cytotoxicity of Ebola virus glycoprotein in vitro. Sci China Life Sci. 57: 959-972.

317. Sherwood LJ, Osborn LE, Carrion R Jr, Patterson JL, Hayhurst A. 2007. Rapid assembly of sensitive antigen-capture assays for Marburg virus, using in vitro selection of llama single-domain antibodies, at biosafety level 4. J Infect Dis. 196 (Suppl 2): S213-219.

318. Shi M, Shen YQ. 2013. Research progress of the molecule mechanisms of Ebola virus infection of cells. Bing Du Xue Bao. 29: 71-75.

319. Shoemaker T, MacNeil A, Balinandi S, Campbell S, Wamala JF, McMullan LK, Downing R, Lutwama J, Mbidde E, Stroher U, Rollin PE, Nichol ST. 2012. Reemerging Sudan Ebola virus disease in Uganda, 2011. Emerg Infect Dis. 18: 1480–1483.

320. Shrivastava SR, Shrivastava PS, Ramasamy J. 2015a. Ebola disease: An international public health emergency. Asian Pacific J Trop Dis. 5: 253-262.

321. Shrivastava SR, Shrivastava PS, Ramasamy J. 2015b. Lessons learnt from the 2014 Ebola outbreak in West-Africa. J Res Med Sci. 20(1): 107-108.

322. Shrivastava SR, Shrivastava PS, Ramasamy J. 2016. Ebola-free Liberia: Scrutinizing the efforts of public health sector and international agencies. Int J Prev Med. 7: 1.

323. Singh RK, Badasara SK, Dhama K, Malik YPS. 2015. Progress and prospects in vaccine research. Chapter in National Workshop on “Current Trends and Future Research Challenges in Vaccines and Adjuvants”. Organized at ICAR Indian Veterinary Research Institute, Izatnagar 243122, Bareilly, Uttar Pradesh, India during 19-20 November 2015, pp: 1-19.

324. Sissoko D, Laouenan C, Folkesson E, M'Lebing AB, Beavogui AH, Baize S, Camara AM, Maes P, Shepherd S, Danel C, Carazo S, Conde MN, Gala JL, Colin G, Savini H, Bore JA, Le Marcis F, Koundouno FR, Petitjean F, Lamah MC, Diederich S, Tounkara A, Poelart G, Berbain E, Dindart JM, Duraffour S, Lefevre A, Leno T, Peyrouset O, Irenge L, Bangoura N, Palich R, Hinzmann J, Kraus A, Barry TS, Berette S, Bongono A, Camara MS, Chanfreau Munoz V, Doumbouya L, Souley Harouna, Kighoma PM, Koundouno FR, René Lolamou, Loua CM, Massala V, Moumouni K, Provost C, Samake N, Sekou C, Soumah A, Arnould I, Komano MS, Gustin L, Berutto C, Camara D, Camara FS, Colpaert J, Delamou L, Jansson L, Kourouma E, Loua M, Malme K, Manfrin E, Maomou A, Milinouno A, Ombelet S, Sidiboun AY, Verreckt I, Yombouno P, Bocquin A, Carbonnelle C, Carmoi T, Frange P, Mely S,

Nguyen VK, Pannetier D, Taburet AM, Treluyer JM, Kolie J, Moh R, Gonzalez MC, Kuisma E, Liedigk B, Ngabo D, Rudolf M, Thom R, Kerber R, Gabriel M, Di Caro A, Wölfel R, Badir J, Bentahir M, Deccache Y, Dumont C, Durant JF, El Bakkouri K, Gasasira Uwamahoro M, Smits B, Toufik N, Van Cauwenberge S, Ezzedine K, D'Ortenzio E, Pizarro L, Etienne A, Guedj J, Fizet A, Barte de Sainte Fare E, Murgue B, Tran-Minh T, Rapp C, Piguet P, Poncin M, Draguez B, Allafort Duverger T, Barbe S, Baret G, Defourny I, Carroll M, Raoul H, Augier A, Eholie SP, Yazdanpanah Y, Levy-Marchal C, Antierrens A, Van Herp M, Günther S, de Lamballerie X, Keïta S, Mentre F, Anglaret X, Malvy D. JIKI Study Group. 2016. Experimental Treatment with Favipiravir for Ebola Virus Disease (the JIKI Trial): A Historically Controlled, Single-Arm Proof-of-Concept Trial in Guinea. PLoS Med. 13(3): e1001967.

325. Skiadopoulos MH, Surman SR, Riggs JM, Orvell C, Collins PL, Murphy BR. 2002. Evaluation of the replication and immunogenicity of recombinant human parainfluenza virus type 3 vectors expressing up to three foreign glycoproteins. Virology. 297(1): 136-152.

326. Smithers SJ, Eastaugh LS, Steward JA, Nelson M, Lenk RP, Lever MS. 2014. Post-exposure efficacy of oral T-705 (favipiravir) against inhalational Ebola virus infection in a mouse model. Antiviral Res. 104:153-155.

327. Spengler JR, Chakrabarti AK, Coleman-McCray JD, Martin BE, Nichol ST, Spiropoulou CF, Bird BH. 2015. Utility of Oral Swab Sampling for Ebola Virus Detection in Guinea Pig Model. Emerg Infect Dis. 21(10):1816-1819.

328. Spengler JR, Ervin ED, Towner JS, Rollin P, Nichol S. 2016. Comments to Author Perspectives on West Africa Ebola Virus Disease Outbreak, 2013–2016. Emerg Infect Dis. 22(6): 956-963.

329. Sridhar S. 2015. Clinical development of Ebola vaccines. Ther Adv Vaccines. 3(5-6): 125–138.

330. Stamm LV. 2015. Ebola virus disease: Rapid diagnosis and timely case reporting are critical to the early response for outbreak control. Am J Trop Med Hyg. 93(3): 438-440.

331. Steele K, Crise B, Kuehne A, Kell W. 2001. Ebola virus glycoprotein demonstrates differential cellular localization in infected cell types of nonhuman primates and Guinea Pigs. Arch Pathol Lab Med. 125:625-630.

332. Stephens KW, Hutchins RJ, Dauphin LA. 2010. Cross-platform evaluation of commercial real-time reverse transcription PCR master mix kits using a quantitative 5′ nuclease assay for Ebola virus. Mol Cell Probes. 24:370-375.

333. Streatfield SJ, Kushner N, Yusibov V. 2015. Plant-produced candidate countermeasures against emerging and reemerging infections and bioterror agents. Plant Biotechnol J. 13(8): 1136-1159.

334. Sullivan NJ, Martin JE, Graham BS, Nabel GJ. 2009. Correlates of protective immunity for Ebola vaccines: implications for regulatory approval by the animal rule. Nat Rev Microbiol. 7:393-400.

335. Sun G, Miyata K, Matsuoka A, Zhao Z, Iwakami S, Kim S, Matsui T. 2015. A compact and hand-held infection-screening system for use in rapid medical inspection at airport quarantine stations: system design and preliminary validation. J Med Eng Technol. 39(3): 185-190.

336. Suzuki Y, Gojobori T. 1997. The origin and evolution of Ebola and Marburg viruses. Mol Biol Evol. 14: 800-806.

337. Swanepoel R, Leman PA, Burt FJ, Zachariades NA, Braack LE, Ksiazek TG, Rollin PE, Zaki SR, Peters CJ. 1996. Experimental inoculation of plants and animals with Ebola virus. Emerg Infect Dis. 2(4): 321.

338. Takada A, Kawaoka Y. 2001. The pathogenesis of Ebola hemorrhagic fever. Trends Microbiol. 9: 506-511.

339. Tambo E, Ugwu EC, Ngogang JY. 2014. Need of surveillance response systems to combat Ebola outbreaks and other emerging infectious diseases in African countries. Infect Dis Pov. 3: 29.

340. Tamura K, Nei M, Kumar S. 2004. Prospects for inferring very large phylogenies by using the neighbor-joining method. Proceedings of the National Academy of Sciences (USA) 101: 11030-11035.

341. Tamura K, Stecher G, Peterson D, Filipski A, Kumar S. 2013. MEGA6: Molecular Evolutionary Genetics Analysis version 6.0. Mol Biol Evolut. 30: 2725-2729.

342. Taniguchi S, Watanabe S, Masangkay JS, Omatsu T, Ikegami T, Alviola P, Ueda N, Iha K, Fujii H, Ishii Y, Mizutani T, Fukushi S, Saijo M, Kurane I, Kyuwa S, Akashi H, Yoshikawa Y, Morikawa S. 2011. Reston ebolavirus antibodies in bats, the Philippines. Emerg Infect Dis. 17: 1559-1560.

343. Taylor DJ, Dittmar K, Ballinger MJ, Bruenn JA. 2011. Evolutionary maintenance of filovirus-like genes in bat genomes. BMC Evol Biol. 11: 336.

344. Team WER. 2015. Ebola virus disease among children in West Africa. N Engl J Med. 372: 1274-1277.

345. Team WER. 2016. Ebola Virus Disease among Male and Female Persons in West Africa. N Engl J Med. 374: 96-98.

346. Teimoori S, Seesuay W, Jittavisuthikul S, Chaisri U, Sookrung N, Densumite J, Saelim N, Chulanetra M, Maneewatch S, Chaicumpa W. 2016. Human transbodies to VP 40 inhibit cellular egress of Ebola virus-like particles. Biochem Bioph Res Co. In press.

347. Thi EP, Mire CE, Lee AC, Geisbert JB, Zhou JZ, Agans KN, Snead NM, Deer DJ, Barnard TR, Fenton KA, MacLachlan I, Geisbert TW. 2015. Lipid nanoparticle siRNA treatment of Ebola-virus-Makona-infected nonhuman primates. Nature. 521: 362-365.

348. Thorson A, Formenty P, Lofthouse C, Broutet N. 2016. Systematic review of the literature on viral persistence and sexual transmission from recovered Ebola survivors: evidence and recommendations. BMJ Open. 6(1): e008859.

349. Thorson A, Formenty P, Lofthouse C, Broutet N. 2016. Systematic review of the literature on viral persistence and sexual transmission from recovered Ebola survivors: evidence and recommendations. BMJ Open. 6(1): e008859.

350. Toit AD. 2014. Ebola virus in West Africa. Nature Rev Microbiol. 12: 312.

351. Towner JS, Pourrut X, Albarino CG, Nkogue CN, Bird BH, Grard G, Ksiazek TG, Gonzalez JP, Nichol ST, Leroy EM. 2007. Marburg virus infection detected in a common African bat. PLoS One. 2(8): e764.

352. Towner JS, Rollin PE, Bausch DG, Sanchez A, Crary SM, Vincent M, Lee WF, Spiropoulou CF, Ksiazek TG, Lukwiya M, Kaducu F, Downing R, Nichol ST. 2004. Rapid diagnosis of Ebola hemorrhagic fever by reverse transcription-PCR in an outbreak setting and assessment of patient viral load as a predictor of outcome. J Virol. 78: 4330-4341.

353. Towner JS, Sealy TK, Khristova ML, Albariño CG, Conlan S, Reeder SA, Quan P-L, Lipkin WI, Downing R, Tappero JW, Okware S, Lutwama J, Bakamutumaho B, Kayiwa J, Comer JA, Rollin PE, Ksiazek TG, Nichol ST. 2008. Newly discovered ebola virus associated with hemorrhagic fever outbreak in Uganda. PLoS Pathog. 4: e1000212.

354. Tracey LE, Regan AK, Armstrong PK, Dowse GK, Effler PV. 2015. Ebola Tracks: an automated SMS system for monitoring persons potentially exposed to Ebola virus disease. Euro Surveill. 20(1): 20999.

355. Truant AL, Regnery RL, Kiley MP. 1983. Development of an immunofluorescence focus assay for Ebola virus. J Clin Microbiol. 18(2): 416-419.

356. Tsang MK, Ye W, Wang G, Li J, Yang M, Hao J. 2016. Ultrasensitive detection of ebola virus oligonucleotide based on upconversion nanoprobe/nanoporous membrane system. ACS Nano. 10(1): 598-605.

357. Tsuda Y, Parkins CJ, Caposio P, Feldmann F, Botto S, Ball S, Messaoudi I, Cicin-Sain L, Feldmann H, Jarvis MA. 2015. A cytomegalovirus-based vaccine provides long-lasting protection against lethal Ebola virus challenge after a single dose. Vaccine. 33: 2261-2266.

358. Uyeki TM, Mehta AK, Davey RT Jr, Liddell AM, Wolf T, Vetter P, Schmiedel S, Grünewald T, Jacobs M, Arribas JR, Evans L, Hewlett AL, Brantsaeter AB, Ippolito G, Rapp C, Hoepelman AI, Gutman J. 2016. Working Group of the U.S.-european clinical network on clinical management of ebola virus disease patients in the U.S. and Europe. Clinical management of ebola virus disease in the United States and Europe. N Engl J Med. 374(7): 636-646.

359. Van der Groen G, Pattyn SR. 1979. Measurement of antibodies to Ebola virus in human sera from N.W.-Zaire. Ann Soc Belg Med Trop. 59(1): 87-92.

360. van Griensven J, De Weiggheleire A, Delamou A, Smith PG, Edwards T, Vandekerckhove P, Bah EI, Colebunders R, Herve I, Lazaygues C, Haba N, Lynen L. 2015. The use of Ebola convalescent plasma to treat Ebola virus disease in resource-constrained settings: a perspective from the field. Clin. Infect. Dis. 62: 69-74.

361. Varkey JB, Shantha JG, Crozier I, Kraft CS, Lyon GM, Mehta AK, Kumar G, Smith JR, Kainulainen MH, Whitmer S, Ströher U, Uyeki TM, Ribner BS, Yeh S. 2015. Persistence of ebola virus in ocular fluid during convalescence. N Engl J Med. 372: 2423.

362. Villinger F, Rollin PE, Brar SS, Chikkala NF, Winter J, Sundstrom J, Zaki SR, Swanepoel R, Ansari A, Peters CJ. 1999. Markedly elevated levels of interferon (IFN)-gamma, IFN-alpha, interleukin (IL)-2, IL-10, and tumor necrosis factor-alpha associated with fatal Ebola virus infection. J Infect Dis. 179: S188-S191.

363. Vogel WH, and Viale PH. 2014. What You Need to Know About the Ebola Virus. J Adv Pract Oncol. 5: 471-473

364. Volchkov VE, Volchkova VA, Chepurnov AA, Blinov VM, Dolnik O, Netesov SV, Feldmann H. 1999. Characterization of the L gene and 5'trailer region of Ebola virus. J Gen Virol. 80(2): 355-362.

365. Walker NF, Whitty CJ. 2015. Tackling emerging infections: clinical and public health lessons from the West African Ebola virus disease outbreak, 2014-2015. Clin Med (Lond). 15(5): 457-460.

366. Walsh KB, Teijaro JR, Wilker PR, Jatzek A, Fremgen DM, Das SC, Watanabe T, Hatta M, Shinya K, Suresh M, Kawaoka Y, Rosen H, Oldstone MB. 2011. Suppression of cytokine storm with a sphingosine analog provides protection against pathogenic influenza virus. Proc Natl Sci. 108: 12018-12023.

367. Wang D, Raja NU, Trubey CM, Juompan LY, Luo M, Woraratanadharm J, Deitz SB, Yu H, Swain BM, Moore KM, Pratt WD, Hart MK, Dong JY. 2006. Development of a cAdVax-based bivalent Ebola virus

vaccine that induces immune responses against both the Sudan and Zaire species of Ebola virus. J Virol. 80:2738-2746.

368. Wang S, Zang L, Feng M, Liang Z, Wang S, Zheng S, Zhang L, Jiang Z, Chen D. 1997. Transmission electron microscopic study of the hemorrhagic spots in patients with epidemic hemorrhagic fever in the early stage. Ultrastruct Pathol. 21(3):281-287.

369. Wang SR, Zhang QY, Wang JQ, Ge XY, Song YY, Wang YF, Li XD, Fu BS, Xu GH, Shu B, Gong P, Zhang B, Tian T, Zhou X. 2016. Chemical Targeting of a G-Quadruplex RNA in the Ebola Virus L Gene. Cell Chem Biol. In press.

370. Warren TK, Warfield KL, Wells J, Swenson DL, Donner KS, Van Tongeren SA, Garza NL, Dong L, Mourich DV, Crumley S, Nichols DK, Iversen PL, Bavari S. 2010. Advanced antisense therapies for postexposure protection against lethal filovirus infections. Nat Med. 16:991-994.

371. Warren TK, Wells J, Panchal RG, Stuthman KS, Garza NL, Van Tongeren SA, Dong L, Retterer CJ, Eaton BP, Pegoraro G, Honnold S, Bantia S, Kotian P, Chen X, Taubenheim BR, Welch LS, Minning DM, Babu YS, Sheridan WP, Bavari S. 2014. Protection against filovirus diseases by a novel broad-spectrum nucleoside analogue BCX 4430. Nature. 508:402-405.

372. Wec AZ, Nyakatura EK., Herbert AS, Howell KA, Holtsberg FW, Bakken RR, Mittler E, Christin JR, Shulenin S, Jangra RK, Bharrhan S, Kuehne AI, Bornholdt ZA, Flyak AI, Saphire EO, Crowe JE, Aman MJ, Dye JM, Lai JR, Chandran K. 2016. A “Trojan horse” bispecific antibody strategy for broad protection against ebolaviruses. Science. 3267.

373. Weingart HM, Embury-Hyatt C, Nfon C, Leung A, Smith G, Kobinger G. 2012. Transmission of Ebola virus from pigs to non-human primates. Sci Rep. 2:811.

374. Weingart HM, Nfon C, Kobinger G. 2013. Review of Ebola virus infections in domestic animals. Dev Biol (Basel). 135:211-218.

375. Weller SA, Bailey D, Matthews S, Lumley S, Sweed A, Ready D, Eltringham G, Richards J, Vipond R, Lukaszewski R, Payne PM, Aarons E, Simpson AJ, Hutley EJ, Brooks T. 2016. Evaluation of the Biofire Film Array BioThreat-E Test (v 2.5) for rapid identification of ebola virus disease in heat-treated blood samples obtained in Sierra Leone and the United Kingdom. J Clin Microbiol. 54(1):114-119.

376. Wendo C. 2001. Caring for the survivors of Uganda’s Ebola epidemic one year on. Lancet. 358:1350.

377. WHO. 1992. Viral haemorrhagic fever in imported monkeys. Wkly Epidemiol Rec. 67:142–143.

378. WHO. 1996. Ebola haemorrhagic fever - South Africa. Wkly Epidemiol Rec. 71:359.

379. WHO. 2003. Outbreak(s) of Ebola haemorrhagic fever, Congo and Gabon, October 2001-July 2002. Wkly Epidemiol Rec. 78:223–228.

380. WHO. 2005. Outbreak of Ebola haemorrhagic fever in Yambio, south Sudan, April - June 2004. Wkly Epidemiol Rec. 80:370–375.

381. WHO. 2009. WHO experts consultation on Ebola Reston pathogenicity in humans [Internet]. [cited 2016 Mar 17]. Available from: http://apps.who.int/iris/handle/10665/130162.

382. WHO Ebola Response Team. 2014. Ebola virus disease in West Africa-the first 9 months of the epidemic and forward projections. N Engl J Med. 371:1481–1495.

383. WHO Ebola Response Team. 2016. Ebola Virus Disease among Male and Female Persons in West Africa. N Engl J Med. 374:96–98.

384. WHO. 2014. WHO emergency quality assessment mechanism for EVD IVDS. Public report. Product: RealStar® Filovirus Screen RT-PCR Kit 1.0.

385. WHO. 2015. Public Report for ReEBOV™ Antigen Rapid Test Kit (EA 0011-011-00).

386. Warren TK, Whitehouse CA, Wells J, Welch L, Heald AE, Charleston JS, Sazani P, Reid SP, Iversen PL, Bavari S. 2015. A single phosphorodiamidate morpholino oligomer targeting VP24 protects rhesus monkeys against lethal Ebola virus infection. M Bio. 6(1): e02344-14.

387. Wiwanitkit V. 2016. Ebola and pregnancy. Obstetric Medicine: The Medicine of Pregnancy. 9:50-50.

388. Wolf T, Kann G, Becker S, Stephan C, Brodt H, de Leuw P, Grunewald T, Vogl T, Kempf VAJ, Keppeler OT, Zacharowski K. 2015. Severe Ebola virus disease with vascular leakage and multiorgan failure: treatment of a patient in intensive care. Lancet. 385(9976):1428-35. doi: 10.1016/S0140-6736(14)62384-9. Epub 2014 Dec 19.

389. Wong G, Kobinger GP, Qiu X. 2014a. Characterization of host immune responses in Ebola virus infections. Expert Rev Clin Immunol. 10:781-790.

390. Wong G, Qiu X, Olinger GG, Kobinger GP. 2014b. Post-exposure therapy of filovirus infections. Trends Microbiol. 22:456.

391. Woolhouse ME, Rambaut A, Kellam P. 2015. Lessons from Ebola: Improving infectious disease surveillance to inform outbreak management. Sci Transl Med. 7(307):307rv5.

392. www.bbc.com/news/magazine-28262541

393. Wyers M, Formenty P, Cherel Y, Guigand L, Fernandez B, Boesch C, Le Guenno B. 1999. Histopathological and immunohistochemical studies of lesions associated with Ebola virus in a naturally infected chimpanzee. J Infect Dis. 179: S54-S59.

394. Varkey JB, Shantha JG, Crozier I, Kraft CS, Lyon GM, Mehta AK, Kumar G, Smith JR, Kainulainen MH, Whitmer S, Ströher U, Uyeki TM, Ribner BS, Yeh S. 2015. Persistence of Ebola virus in ocular fluid during convalescence. N Engl J Med. 372(25): 2423-2427.

395. Yang L, Sanchez A, Ward JM, Murphy BR, Collins PL, Bukreyev A. 2008. A paramyxovirus-vectored intranasal vaccine against Ebola virus is immunogenic in vector-immune animals. Virology. 377: 255-264.

396. Yang Y, Bai L, Hu KX, Yang ZH, Hu JP, Wang J. 2012. Multiplex real-time PCR method for rapid detection of Marburg virus and Ebola virus. Zhonghua Shi Yan He Lin Chuang Bing Du Xue Za Zhi. 26: 313-315.

397. Yao J, Weng Y, Dickey A, Wang KY (2015). Plants as factories for human pharmaceuticals: Applications and challenges. Int J Mol Sci. 16(12): 28549-65.

398. Ye L, Yang C. 2015. Development of vaccines for prevention of Ebola virus infection. Microbes Infect. 17(2): 98-108.

399. Zaki SR, Shieh WJ, Greer PW, Goldsmith CS, Ferebee T, Katshitshi J, Tshioko FK, Bwaka MA, Swanepoel R, Calain P, Khan AS, Lloyd E, Rollin PE, Ksiazek TG, Peters CJ. 1999. A novel immunohistochemical assay for the detection of Ebola virus in skin: implications for diagnosis, spread, and surveillance of Ebola hemorrhagic fever. Commission de Lutte contre les Epidémies à Kikwit. J Infect Dis. 179(Suppl 1): S36-47.

400. Zhang L, Wang H. 2014. Forty years of the war against Ebola. J Zhejiang Univ Sci B. 15: 761-765.

401. Zhang Y, Li D, Jin X, Huang Z. 2014. Fighting Ebola with ZMapp: spotlight on plant-made antibody. Sci China Life Sci. 57: 987-988.

402. Zhang XA, Li S, Ching J, Feng HY, Yang K, Dolinger DL, Zhang LD, Zhang PH, Liu W, Cao WC. 2017. A sensitive and specific point-of-care detection assay for Zaire Ebola virus. Emerg Microbes Infect. 6(1): e5. doi: 10.1038/emi.2016.134.

Legends of Figures

-ve sense RNA  
genome  

80 nm in  
diameter  

14,000 nm in length  

GP2 fusion/entry protein  

Fusion and  
entry  

Attachment  

GP1- attachment protein  

19 kb in length  

3'HO  
NP  
VP35  
VP40  
GP/sGP  
VP30  
VP24  
L  
P-5'  

Structural  
nucleoprotein  

Non-structural  
protein  

Matrix protein  

Envelope  
glycoprotein  

Non-structural  
protein  

Matrix protein  

RNA polymerase  

Aids in RNA  
complex formation  

VP 35-antagonist of interferon  

Host cell  

Essential for viral  
replication  

Nucleus  

VP24 inhibits interferon signaling  

Involved in attachment,  
entry, causing cell  
rounding, down regulating  
host surface proteins,  
causes inflammation, cell  
damage and cytotoxicity  

√ Long filaments, shaped  
like a "6", a "U", or a  
circle  

√ Enveloped in lipid  
membrane  

√ Single-stranded, non-  
segmented, negative  
sense RNA virus  

√ Genome consist of  
seven genes  

√ sGP - is produced from  
the unedited RNA  
transcript  

√ ssGP - is a truncated  
version of sGP  

Figure 1: Structure of Ebola virus and its genome. Ebola virus possesses negative sense  
RNA genome with exceptionally 14000 nm length with 3'nucleoprotein and 5’RNA  
polymerase end.

100 KC 242797/Zaire ebolavirus/H.sapiens-tc/GAB/1996/10ba
100 KC 242795/Zaire ebolavirus/H.sapiens-tc/GAB/1996/1Mbie
100 KC 242798/Zaire ebolavirus/H.sapiens-tc/GAB/1996/1lkot
100 KP 271020/Zaire ebolavirus/H.sapiens-wt/COD/2014/Lomela-Lokolia 19
100 KP 271018/Zaire ebolavirus /H.sapiens-wt/COD/2014/Lomela-Lokolia 16
100 KU 182909/Ebola virus/H.sapiens-tc/COD/1995/Kikwit-9510622
100 KU 182907/Ebola virus/H.sapiens-tc/COD/1995/Kikwit-9510627
100 KU 182902/Ebola virus/H.sapiens-tc/COD/1995/Kikwit-9510624
51 KC 242791/Zaire ebolavirus/H.sapiens-tc/COD/1977/Bonduni
KM 655246/Zaire ebolavirus/H.sapiens-tc/COD/1976/Yambuku-Ecran
AF 086833/Ebola virus-Mayinga Zaire 1976
00 KC 242801/Zaire ebolavirus /H.sapiens-tc/COD/1976/deRoo ver
KR 063671/Ebola virus/H.sapiens-tc/COD/1976/Yambuku-Mayinga
AY 142960/Zaire Ebola virus -Mayinga subtype
100 KC 242800/Zaire ebolavirus/H.sapiens-tc/GAB/2002/Ilembe
KC 242790/Zaire ebolavirus/H.sapiens-tc/COD/2007/5 Luebo
KC 242789/Zaire ebolavirus/H.sapiens-tc/COD/2007/4 Luebo
KC 242786/Zaire ebolavirus/H.sapiens-tc/COD/2007/1 Luebo
KT 357860/Zaire ebolavirus/DML 14077/SLe/Western Urban/20150630
KR 025228/Zaire ebolavirus/H.sapiens-wt/GBR/2015/Makona-UK 3
KM 233112/Zaire ebolavirus/H.sapiens-wt/SLE/2014/Makona-G 3851
94 KT 357850/Zaire ebolavirus/DML 24853/SLe/Kono/20150206
KM 233118/Zaire ebolavirus/H.sapiens-wt/SLE/2014/Makona-NM 042.3
KM 233058/Zaire ebolavirus/H.sapiens-wt/SLE/2014/Makona-G 3750.1
58 KR 781609/Zaire ebolavirus /H.sapiens-rec/LBR/2014/Makona-L2014ZsG
KR 781608/Zaire ebolavirus /H.sapiens-rec/LBR/2014/Makona-L2014
KR 074999/Zaire ebolavirus /H.sapiens-wt/LBR/2014/Makona-Liberia-DQE 5
KR 074997/Zaire ebolavirus /H.sapiens-wt/LBR/2015/Makona-Liberia-DQE 3
100 NC 014372/Tai Forest ebolavirus /H.sapiens-tc/CIV/1994/Pauleoula-CI
100 FJ 217162/Cote d'ivoire ebolavirus
100 KU 182910/Tai Forest ebolavirus /H.sapiens-tc/CIV/1994/Tai Forest-CDC 807212
100 KC 545395/Bundibugyo ebolavirus/EboBund-122 2012
100 KC 545393/Bundibugyo ebolavirus/EboBund-112 2012
100 KC 545396/Bundibugyo ebolavirus /EboBund-14 2012
100 KU 182911/Bundibugyo virus/H.sapiens-tc/UGA/2007/Bundibugyo-200706291
100 KR 063673/Bundibugyo virus/H.sapiens-tc/UGA/2007/Butalya-811250
100 FJ 217161 Bundibugyo ebolavirus
FJ 621584/Reston ebolavirus/Reston 08-C
JX 477165/Reston ebolavirus isolate RESTV/Sus-wt/PHL/2009/09AFarmA
100 AF 522874/Reston Ebola-Pennsylvania
100 NC 004161/Reston ebolavirus /M.fascicularis-tc/USA/1989/Philippines 89-Pennsylvania
100 AB 050936/Reston ebolavirus
100 JX 477166/eston ebolavirus/M.fascicularis-tc/PHL-USA/1996/Ferlite Philippines/Alice TX
67 FJ 968794/Sudan ebolavirus-Boniface
100 KT 878488/Sudan ebolavirus /C.porcellus-lab/SSD/1976/Nzara-Boneface
100 KT 750754/Sudan ebolavirus
EU 338380/Sudan ebolavirus isolate EBOV-S-2004
100 KC 545392/Sudan ebolavirus isolate EboSud-682 2012
100 KC 545389.1| Sudan ebolavirus EboSud-602 2012
100 KC 545390/Sudan ebolavirus isolate EboSud-603 2012
100 KC 589025/Sudan ebolavirus EboSud-639
65 JN 638998/Sudan ebolavirus -Nakisamata
95 KR 063670/Sudan ebolavirus /H.sapiens-tc/UGA/2000/Gulu-808892
96 KU 182912/Sudan ebolavirus/H.sapiens-tc/SDN/2000/Gulu-200011676
100 NC 006432.1| Sudan ebolavirus /H.sapiens-tc/UGA/2000/Gulu-808892
94 AY 729654/Sudan ebolavirus strain Gulu

Figure 2: Evolutionary relationships of Ebola virus complete genomes. The evolutionary history was inferred using the Neighbor-Joining method (Saitou and Nei 1987). The optimal tree with the sum of branch length = 1.06917633 is shown. The percentage of replicate trees in which the associated taxa clustered together in the bootstrap test (1000 replicates) are

shown next to the branches (Felsenstein 1985). The tree is drawn to scale, with branch lengths in the same units as those of the evolutionary distances used to infer the phylogenetic tree. The evolutionary distances were computed using the p-distance method (Nei and Kumar 2000) and are in the units of the number of base differences per site. The analysis involved 56 nucleotide sequences. Codon positions included were 1st+2nd+3rd+Noncoding. All positions containing gaps and missing data were eliminated. There was a total of 17999 positions in the final dataset. Evolutionary analyses were conducted in MEGA 6 (Tamura et al. 2013).

![Map](https://example.com/map_image.png)

Figure 3: Ebola-affected countries on the world map. The coloured area depicts Ebola presence from 1976 to 2016.

1976-Zaire- Zaire ebolavirus
Number of outbreaks by Tai forest virus- 1
Number of Outbreaks in Uganda- 5

South Sudan-
Sudan virus
England- Sudan virus
Zaire- Zaire ebola
virus

1. Most number of outbreaks by
a ebolavirus- Zaire ebolavirus
2. First ebolavirus recorded
outside Africa- Reston
ebolavirus (Philippines).
3. Maximum lethality- Zaire
ebolavirus
4. Most human cases recorded
to be affected- Last outbreak
in 2014

Gabon- Zaire ebola
Republic of Congo- Zaire ebola
Republic of Congo- Zaire ebola
Republic of Congo- Zaire ebola
South Sudan- Sudan virus
Russia- Zaire ebola

Democratic Republic of the Congo- Zaire ebola
Number of outbreaks by Bundibugyo virus- 2

Uganda- Sudan virus
Democratic Republic of the
Congo- Bundibugyo virus
2012
2011
2012-2013
Uganda- Sudan virus
Guinea, Liberia, Sierra,
Leone, Nigeria- Zaire
ebola
WHO terminated the
Public Health
Emergency of
International Concern
(PHEIC) in Western
Africa

EBOLA

Number of outbreaks in Democratic Republic of the Congo- 6
Number of outbreaks in Sudan -3

Figure 5. Transmission, pathogenesis and clinical signs of Ebola virus.

- **Blood vessel**
  - Vascular damage
  - Leakage of blood vessel
  - Incubation period: 1 to 21 days

- **Fibrin clot**
  - Massive blood loss
  - Death due to hypovolemic shock/thrombosis

- **Spleen**
  - Intracytoplasmic inclusion bodies
  - Spread of the virus to regional lymph nodes, liver, and spleen

- **Adrenal gland**
  - Necrosis of adrenal cortical cells leading to impaired steroid production

- **Liver**
  - Necrosis of hepatocytes leads to decreased level of clotting factors

- **Retina**
  - Persistence of virus in immune privileged site

- **Brain**

- **Regional lymph node**

- **Monkeys**
  - Transmission of virus from non-human primates

- **Breach in the skin**
  - Consumption of infected mango

- **Intravascular coagulation leads to renal failure**
  - Virus reaches macrophages, monocytes, and dendritic cells

- **Dendritic cell**

- **Stomach pain**

- **Ebola entry through broken skin/ mucous membranes/blood**
  - Transmission from bats

- **Mango half eaten by bat**

- **Fever and headache**
  - Joint pain and muscle pain
  - Vomiting and diarrhea
  - Chest pain, difficulty in breathing and swallowing

- **Post-Ebola syndrome**: abdominal pains, loss of vision and hearing, impotence, bleeding, psychological problems, and general weakness

- **Others signs**: organ swelling, orchitis, several organ dysfunction

Table 1: Outbreaks of Ebolavirus genus between 1976 and March 2016

| Ebola subtype | Number of Transmission Events | Reported human cases | Reported deaths (%) | Remarks |
|---------------|--------------------------------|----------------------|---------------------|---------|
| Recent outbreaks (since 2014) | | | | |
| Ebolavirus | multiple countries mainly three West African countries viz., Guinea, Liberia, Sierra Leone | 28652 | 11325 (40) | Data between March 2014 and 27 March 2016. Seven countries (Italy, Mali, Nigeria, Senegal, Spain, the United Kingdom, and the United States of America) have previously reported a case or cases imported from a country with widespread and intense transmission. |
| Ebolavirus | 1 | DRC | 66 | 49 (74) | August-November 2014; the outbreak was unrelated to the outbreak of Ebola in West Africa. |
| Outbreaks between 1976 and 2013 | | | | |
| Bundibugyovirus | 2 | DRC, Uganda | 185 | 50 (27) | |
| Restonvirus | 7 | Philippines, USA, Italy | 13 | 0 (0) | 4 outbreaks in monkeys only; One outbreak in pigs with antibody detection in 6 workers; two outbreaks in moneys with 7 people developed antibodies. |
| Sudanvirus | 8 | Uganda, Uganda, England | 779 | 412 (53) | |
| Tai Forestvirus | 1 | Côte d'Ivoire (Ivory Coast) | 1 | 0 (0) | |
| Zaire virus | 15 | DRC, Russia, Gabon, South Africa | 1383 | 1086 (79) | |
| Total | | | 31079 | 12922 | |

Data sources: http://www.cdc.gov/vhf/ebola/outbreaks/history/chronology.html; http://apps.who.int/ebola/current-situation/ebola-situation-report-30-march-2016

| Years | Country | Cases reported | Deaths | Case Fatality Rate |
|-------|---------|---------------|--------|-------------------|
| 1976  | Zaire   | 318           | 280    | 88%               |
|       | Sudan   | 284           | 151    | 53.1%             |
|       | England | 1             | 0      | 0%                |
| 1977  | Zaire   | 1             | 1      | 100%              |
| 1979  | Sudan   | 34            | 22     | 64.7%             |
| 1989- | Philippines | 3         | 0      | 0%                |
| 1990  |         |               |        |                   |
| 1990  | USA     | 4             | 0      | 0%                |
| 1994  | Gabon   | 52            | 31     | 59.6%             |
|       | Côte d'Ivoire | 1       | 0      | 0%                |
|       | Ivory Coast |           |        |                   |
| 1995  | Democratic Republic of Congo | 315 | 250    | 71.4%             |
| 1996  | Gabon   | 37            | 21     | 56.7%             |
| 1996- | Gabon   | 60            | 45     | 75%               |
| 1997  |         |               |        |                   |
| 1996  | South Africa | 2       | 1      | 50%               |
| 2000- | Uganda  | 425           | 224    | 52.7%             |
| 2001  |         |               |        |                   |
| 2001- | Gabon   | 65            | 53     | 81.5%             |
| 2002  | Democratic Republic of Congo | 57 | 43     | 75.4%             |
| 2002- | Democratic Republic of Congo | 143 | 128    | 89.5%             |
| 2003  |         |               |        |                   |
| 2003  | Democratic Republic of Congo | 35 | 29     | 82.8%             |
| 2004  | Sudan   | 17            | 7      | 41.1%             |
|       | Russia  | 1             | 1      | 100%              |
| 2007  | Democratic Republic of Congo | 264 | 187    | 70.8%             |
| 2007- | Uganda  | 149           | 37     | 24.8%             |
| 2008  |         |               |        |                   |
| 2008  | Philippines | 6       | 0      | 0%                |
| 2008- | Democratic Republic of Congo | 32 | 15     | 46.8%             |
| 2009  |         |               |        |                   |
| 2011  | Uganda  | 1             | 1      | 100%              |
| 2012  | Uganda  | 11            | 4      | 36.3%             |
|       | Democratic Republic of the Congo | 36 | 13     | 36.1%             |
| 2012- | Uganda  | 6             | 3      | 50%               |
| 2013  |         |               |        |                   |
| 2014- | United States of America | 4 | 1      | 25%               |
| 2016  | Spain   | 1             | 0      | 0%                |

United Kingdom    1   0   0%
Italy            1   0   0%
Guinea          3814 2544 66.7%
Sierra Leone    14124 3956 28%
Senegal         1   0   0%
Mali            9   6   66.6%
Liberia        10678 4810 45%
Nigeria         20  8   40%
Total          31013 12872 41.5%

*Data compiled from WHO situation reports apps.who.int/ebola/ebola-situation-reports?m=20141121 and CDC updates on Ebola virus https://www.cdc.gov/vhf/ebola/outbreaks/2014-west-africa/previous-updates.html

| S.No. | Technique(s)                                                                 | Input molecule (starting material)                                      | Target to be used for detection | Advantage(s)                                                                                   | Reference/Year          |
|-------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------|-----------------------------------|---------------------------------------------------------------------------------------------|-------------------------|
| 1.    | Formalin-fixed tissue-specimens examination after enzyme treatment.            | Formalin fixed tissue slides                                              | --                                | Advantages are minimized due to arrival of sophisticated and efficient diagnostic techniques. | Kurata et al., 1983      |
| 2.    | Immunofluorescence focus assay.                                              | Serum                                                                      | Ebola antigen                     |                                                                                             | Truant et al., 1983      |
| 3.    | Enzyme Immunosorbent Assay for Primates.                                      |                                                                           |                                   |                                                                                             | Ksiazek et al., 1992     |
| 4.    | Immuno-enzyme Test System.                                                   |                                                                           |                                   |                                                                                             | Borisевич et al., 1996  |
| 5.    | Transmission electron microscopic study of the hemorrhagic spots.             | Squamous epithelial cells and the capillary endothelial cells on mucous membrane of mucous membrane of soft palate | Direct virus visualization        | Visualization of virus in the enlarged Golgi apparatus.                                     | Wang et al., 1997        |
| 6.    | Immuno-histochemical and electron microscopic examination techniques.          | Tissue                                                                     | Viral antigens and particles directly visualized | For safe, sensitive, and specific detection of Ebola virus in formalin-fixed skin.           | Zaki et al., 1999        |
| 7.    | Diagnosis by RT-PCR in an epidemic setting.                                   | Peripheral blood mononuclear cells (PBMC)                                 | Nucleic acid                      | PCR based diagnostic assay.                                                                  | Leroy et al., 2000       |
| 8.    | Development and evaluation of a fluorogenic 5' nuclease assay.                | Cultured virus in Vero cells                                              | Glycoprotein (GP) antigens         | Detect and differentiate between Zaire and Sudan subtypes.                                  | Gibb et al., 2001        |
| 9.    | 5' nuclease probe                                                            | Serum                                                                      | Nucleic acid                      | Detection limit: 9to16RNA                                                                    | Drosten et al., 2002     |

|   | detection by real-time reverse transcription-PCR. |   |   | copies per reaction. |   |
|---|--------------------------------------------------|---|---|----------------------|---|
| 10. | Antigen capture ELISA directed against the envelope glycoprotein of Ebola virus. | Serum | Glycoprotein (GP) antigens | Sensitivity up to ~10³ plaque-forming units/ml. Able to detect even SDS-inactivated virus. | Lucht et al., 2004 |
| 11. | Rapid diagnosis of Ebola hemorrhagic fever by reverse transcription-PCR in an outbreak setting and assessment of patient viral load as a predictor of outcome. | Serum, blood or plasma | Nucleoprotein (NP) | One-step fluorogenic Q-RT-PCR assay | Towner et al., 2004 |
| 12. | Immune-filtration based antigen-detection assay for rapid detection (Only Zaire and Sudan strain specific). | Serum | Matrix protein (VP40) | Sensitivity similar to ELISA and rapid assay without need of the higher technical skill. | Lucht et al., 2007 |
| 13. | Reverse transcription-loop-mediated isothermal amplification assay (Zaire strain specific). | Vero cells cultured virus | NP | Portable amplification based platform. Negative for other febrile disease viruses such as Lassa virus, Rift Valley fever virus, Yellow fever virus. | Kurosaki et al., 2007 |
| 14. | Quantitative 5'nuclease assay for NP/- | Vero cells cultured virus | NP | Evaluation of commercial master mixes for sensitivity and reaction time. | Stephens et al., 2010 |
| 15. | SYBR Green I real-time RT-PCR. | Cell cultured virus | NP | No cross reactivity with Marburg, Dengue, Xinjiang hemorrhagic fever virus, Japanese encephalitis virus, | Liu et al., 2012; 2015 |

|   |   |   |   | Influenza virus (H1N1 and H3N2) and Porcine reproductive and respiratory syndrome viruses. |   |
|---|---|---|---|---|---|
| 16. | Multiplex RT-PCR for simultaneous detection of Marburg virus and Ebola virus. | Cell cultured virus | Nucleic acid | Sensitive with ability to test two hemorrhagic viruses in a single assay. | Yang et al., 2012 |
| 17. | Indirect ELISA for detecting Abs specific to Ebola & Marburg viruses. | Cell culture supernatant | NP | Very sensitive assay. | Huang et al., 2014 |
| 18. | Novel biomarker soluble CD40 ligand (sCD40L) correlates of EVD clinical outcome. | Serum | Different 26 cytokines | Simultaneous detection of 26 proinflammatory cytokines. Increased soluble CD40 ligand (sCD40L) level is indicator of diseases severity. | McElroy et al., 2014 |
| 19. | Comprehensive multiplex one-step TaqMan qRT-PCR assay. | Cell cultured virus | Nucleic acid | In a single qRT-PCR assay, 28 hemorrhagic viruses may be detected. | Pang et al., 2014 |
| 20. | Aptamericsensing technology for rapid pandemic monitoring. | Serum | Protein | Aptamers are bio-affinity probes with improved pathogen detection ability. | Acquah et al., 2015 |
| 21. | GeoSentinel Surveillance Network. Differential diagnosis of illness in travelers arriving from Sierra Leone, Liberia, or Guinea. | General health surveillance of physical illness | Patients | Based on clinical data collected on GeoSentinel Global Surveillance Network platform ( 57 specialized travel and tropical medicine clinics). | Boggild et al., 2015 |
| 22. | Antigen-capture ELISA and lateral flow immunodiagnostic. | Serum | Matrix VP 40 and glycoprotein (GP) antigens | Sensitivity sufficient for clinical diagnosis. | Boisen et al., 2015 |
| 23. | Optofluidicanalysis | Serum | Nucleic acid | Chip-scale approach | Cai et al., 2015 |

|   | system for amplification-free, direct detection of Ebola. |   |   | Single nucleic acid detection in 10min. |   |
|---|-------------------------------------------------------|---|---|----------------------------------------|---|
| 24. | Fe3O 4 magnetic nanoparticle based immuno-chromatographic strip (Nanopzyme-strip) | Serum | Glycoprotein (GP) antigens | Comparable to ELISA, but faster and simpler. | Duan et al., 2015 |
| 25. | Rapid recombinase polymerase amplification Ebola virus detection assay. | Serum and swab | Nucleic acid | Assay can combine extraction and detection in a single step. | Faye et al., 2015 |
| 26. | Real-time EBOV-L-RT-qPCR for detection of Zaire Ebola virus. | Serum or blood | L gene | No false positives with serum or EDTA bloods samples. | Jääskeläinen et al., 2015 |
| 27. | The Ebola Spatial Care Path™ (SCP): Accelerating point-of-care testing (POCT). | Whole blood | Platelet count, white blood cell count, granulocyte count and percentage, lymphocyte/monocyte count and percentage, prothrombin time, virus-specific IgM and IgG, hemoglobin and hematocrit | Ebola molecular diagnostics kits for rapid POCT. The reader / processor / cartridges in sealed, disposable pouches containing the biological and chemical reagents. | Kost et al., 2015 |
| 28. | Ultrafast, sensitive and large-volume on-chip real-time PCR. | (General pathogen detection) |   | Ultrafast RT-PCR system using microfluidic thermalization can detect Ebola in 7.5min. | Houssin et al., 2016 |
| 29. | Prototype Cepheid | Stored serum and GP and NP gene |   | No cross-reactivity. | Pinsky et al., 2015; |
|   | GeneXpert Ebola Assay. | plasma samples |   | Less input sample volume. | Jensen et al., 2016 |
| --- | --- | --- | --- | --- | --- |
| 30. | RT-Loop-Mediated Isothermal Amplification (RT-LAMP) Assay | Cultured virus | Untranscribed trailer region or nucleoprotein coding region of the viral RNA | Higher sensitivity to nanoscale. No cross-reactivity to other hemorrhagic fever viruses and arboviruses. | Kurosaki et al., 2016 |
| 31. | Upconversion Nanoprobe / Nanoporous Membrane System. | Inactivated Ebola virus samples | Ultrasensitive Detection of Ebola Virus Oligonucleotide | Detection up to pico-mole level. Low cost, rapid, and ultrasensitive detection. | Tsang et al., 2016 |
| 32. | Biofire Film-Array BioThreat-E Test (v2.5). | EDTA-blood, serum or plasma | -- | Whole blood maybe used. | Leski et al., 2015; Weller et al., 2016 |
| 33. | Zaire Ebola virus (ZEBOV) -specific enzyme-linked immunosorbent assay (ELISA) using inactivated ZEBOV isolate Makona | Serum | Glycoprotein (GP) antigens | Simple and cost effective. | Krahl ing et al., 2016 |
| 34. | RT-LAMP | Blood | Untranscribed trailer region or nucleoprotein coding region of the viral RNA. | Whole blood used to detect GP gene of EBOV High sensitivity Result within 40 minutes | Benzine et al. 2016; Kurosaki et al., 2016 |
| 35. | ReEBOV Antigen Rapid Test (ReEBOV RDT) | Whole blood or serum | VP antigen (anti-EBOV VP 40) | Highly specific Point of care test for EVD | Cross et al. 2016 |
| S.No. | Kit Name                                      | Type of the Test                                   | Input Material(s)                                                                 | Company name, Country                          |
|-------|-----------------------------------------------|---------------------------------------------------|---------------------------------------------------------------------------------|----------------------------------------------|
| 1     | -                                             | Antigen detection: LAMP                           | Serum, plasma, amniotic or synovial fluid                                    | Lucigen, USA                                  |
| 2     | ABICAP-rapid-Ebola test                       | Antigen detection-Antibody Immuno-Column for Analytical Purpose | Whole blood, serum, urine, stool, tears and saliva as well as in environmental samples such as water, air and soil | Senova GmbH, Germany                         |
| 3     | AccuPower EBOV Quantitative RT-PCR Kit (EBO-1111A) | Nucleic acid detection: RT-PCR                     | Serum or plasma                                                                  | Bioneer, Korea                                |
| 4     | eZYSCREEN kit                                 | Antigen detection                                 | Blood or serum                                                                   | French Atomic Energy Commission (CEA), France |
| 5     | Life river TM Ebola Virus (EBOV) RealTime RT-PCR Kit | Real-time reverse transcription PCR               | Blood, serum, plasma (non-heparin anticoagulant)                              | Shanghai ZJ BioTech Co., Ltd., China          |
| 6     | LightMix Modular Ebolavirus Zaire              | Real-time reverse transcription PCR                | Whole Blood                                                                     | Tib-molbiol/Roche, Germany                   |
| 7     | Lipsgene SEBOV/ZEBOV Kit                      | Real-time reverse transcription PCR                | Blood, serum or plasma                                                           | Lipsdiag, Germany                            |
| 8     | OraQuick® Ebola Rapid Antigen Test Kit         | Antigen detection                                 | Cadaveric oral fluid/whole blood                                               | OraSure Technologies, Inc, USA               |
| 9     | Rapid Response Ebola Virus Test Cassette      | Antigen detection chromatographic immunoassay      | Whole Blood/Serum/ Plasma                                                       | BTN X Inc., Canada                            |
| 10    | RealStar Filovirus Screen RT-PCR Kit 1.0      | Real-time reverse transcription PCR                | EDTA plasma                                                                      | Altona Diagnostics, Germany                  |

| 11 | ReEBOV Antigen Rapid Test Kit | Antigen detection: Immuno-chromatographic dipstick immunoassay | Blood/serum/plasma | Corgenix, USA |
| --- | --- | --- | --- | --- |
| 12 | SD Q Line Ebola Zaire Ag | Antigen detection-Lateral flow chromatographic immunoassay | Blood or serum | SD Biosensor Inc., GyeongGi-do, Republic of Korea |
| 13 | Xpert® Ebola Test | Real-time reverse transcription PCR in self-contained cartridge system- | Whole blood EDTA or buccal swab specimens | Cepheid AB, Sweden |

| Type of Therapy | Name of Molecule/modality | Modus Operandis | Model Organism (onto which the test has been conducted) | Company | References |
|------------------|--------------------------|-----------------|----------------------------------------------------|-----------|-------------|
| siRNA therapy    | TKM-Ebola                | Cocktail of three siRNAs targeting L, VP24andVP 35 genes | monkeys                          | Tekmira, Canada | Geisbert et al., 2010 |
| Antiviral drug   | AVI-7537                 | Through gene silencing | mice and monkeys | Sarepta Therapeutics | Warren et al., 2010; Iversen et al., 2012; Heald et al., 2014 |
|                  | Favipiravir(T-705), a pyrazinecarboxamide derivative | Inhibit viral RNA-dependent RNA polymerase activity and induce lethal RNA mutations | mice | Toyama Chemical, Japan | Smith er et al., 2014; Oester eich et al., 2014 |
|                  | Brincidofovir(CMX 001)    | Improved bioavailability and activity against dsDNA viruses | Human (but not for ebola specific) | Chimerix of Durham, North Carolina | Florescu et al., 2015 |
| ZMapp           | Combination of humanized monoclonal antibodies(c13C6+c2G4 and C4G7 | Conformational epitopes of GP 2 or GP trimer stem region are targeted | macaques | Mapp Biopharmaceutical, San Diego, California | Haque et al., 2015; Kilgore et al., 2015 |
| Nucleotide analogue | BCX 4430 | nucleoside RNA polymerase inhibitor, incorporate into growing RNA and lead to premature termination | mice | BioCryst Pharmaceuticals, Inc. Durham, North Carolina | Warren et al., 2014 |
| Induction of Niemann-Pick Cl mutant phenotype | U18666A/ clomiphene/to remiphene induce cholesterol accumulation in | Membrane fusion and virus escape is prevented | In vitro | A chemical supplied by several vendors across the world | Carette et al., 2011 |

|  | lysosomes and inhibit ebola virus infection |  |  |  |
| --- | --- | --- | --- | --- |
| Repurposed Drug | FTY 720 (FDA-approved drug for multiple sclerosis) | Prevent cytokine storm and sepsis, reduce inflammation | Human | Sigma/Novartis | Jap tok and, Kleuser, 2009; Cross and Naismith, 2014 |
| FX 06 (Anticoagulant therapy) | fibrin-derived peptide help in vascular leakage | Human | MChE-F4 Pharma, Vienna, Austria | Wolf et al., 2015 |
| Rosuvastatin, Atorvastatin, and Pravastatin (Lipid lowering drugs) | Reduction of cholesterol content affect the biosynthesis machinery in Ebola virus | Human | By several vendors across the world | Calza et al., 2012; Haque et al., 2015 |
| Chloroquine, Hydroxychloroquine (Antimalarial/lupus erythematosus/rheumatoid arthritis treatment with its anti-inflammatory properties) | Prevent endosome acidification and Ebola virus is trapped inside endosome | Human | By several vendors across the world | Madrid et al., 2013; Olsen et al., 2013; Kuehne et al. 2016 |
| rNAPc 2 (Recombinant nematode anticoagulant protein c 2) | modulation of coagulation factors | primates | ARCA biopharma, Colorado | Geisbert et al., 2003 |
| Suramin (anti-trypanosomal agent) | Anti-heparin activity, interfere with viral replication and viral entry into the cell | In vitro | Germanin, Bayer | HenB et al. 2016 |
| Convalescent plasma (CP) therapy | Plasma for Ebola survivors | Providing highly specific antibodies and replenish lost serum components | Rhesus macaques | NA | Jahrling et al., 2007; Dye et al., 2012; Mendoza et al., 2016 |
| Viral vectored vaccines | recombinant chimpanzee adenovirus serotype 3 vector | GP gene | Human | NA | Choi et al., 2015 |

|  | Recombinanat Vesicular stomatitis virus | GP/NP | cynomolgus macaques | NA | Jones et al., 2005; Quiet al., 2009; Falzarano et al., 2011 |
| --- | --- | --- | --- | --- | --- |
|  | Venezuelan equine encephalitis virus (VEEV) replicons |  | mice and guinea pigs | NA | Pushko et al., 2004 |
| DNA vaccine | inserted NP/GP | Antigen presentation by both MHC-I and II | guinea pigs or chimpanzees | NA | Sullivan et al., 2009; Hoenen et al., 2012 |
| Virus-like particles (VLPs) | DNA vectors expressing GP and VP 40 proteins in human 293T cells | Morphologically similar to virus and provoke protective immune response | mice | NA | Schmaljohn et al., 2003, YeL, Yang, 2014; Carra et al., 2015 |
| Trojan horse antibodies | Bispecific antibody | Specific for NCP1orGP receptor binding site coupled to mAb against conserved surface exposed GP epitope | mice | NA | We c et al. 2016 |
| Human transbodies | Cell penetrable Human ScFv against VP 40 | Inhibit egress of EBOV VLPs from hepatocytes | In vitro | NA | Tei moo ri et al. 2016 |
| mPyP 4 | Cationic porphyrin | Inhibition of L gene of EBOV | In vitro | NA | Wang et al. 2016 |
