
A Journal of the Gesellschaft Deutscher Chemiker
Angewandte
International Edition
Chemie
www.angewandte.org

Accepted Article

Title: Sulfation of Human Cytomegalovirus Protein UL22A Enhances Binding to the Chemokine RANTES

Authors: Richard James Payne, Xiaoyi Wang, Julie Sanchez, and Martin Stone

This manuscript has been accepted after peer review and appears as an Accepted Article online prior to editing, proofing, and formal publication of the final Version of Record (VoR). This work is currently citable by using the Digital Object Identifier (DOI) given below. The VoR will be published online in Early View as soon as possible and may be different to this Accepted Article as a result of editing. Readers should obtain the VoR from the journal website shown below when it is published to ensure accuracy of information. The authors are responsible for the content of this Accepted Article.

To be cited as:
Angew. Chem. Int. Ed. 10.1002/anie.201703059
Angew. Chem. 10.1002/ange.201703059

Link to VoR:
http://dx.doi.org/10.1002/anie.201703059
http://dx.doi.org/10.1002/ange.201703059

WILEY-VCH

Angewandte Chemie International Edition 10.1002/anie.201703059

COMMUNICATION

Sulfation of Human Cytomegalovirus Protein UL22A Enhances Binding to the Chemokine RANTES

Xiaoyi Wang,[a] Julie Sanchez,[b] Martin J. Stone[b] and Richard J. Payne*[a]

Abstract: UL22A is an 82 amino acid chemokine binding protein produced by human cytomegalovirus that likely assists the virus in dampening the host anti-viral response. Herein, we propose that UL22A is sulfated on two tyrosine residues and test this hypothesis through the chemical synthesis of a small library of differentially sulfated protein variants. The (sulfo)proteins were efficiently prepared using a novel β-selenoleucine motif to facilitate one-pot ligation-deselenization chemistry. Tyrosine sulfation of UL22A proved critical for RANTES binding, with the doubly sulfated variant exhibiting 2.5 orders of magnitude improvement in binding compared to the unmodified protein.

Chemokines are chemoattractant cytokines produced at the site of infection or injury. They are secreted into the vasculature where they activate chemokine receptors located on the surface of circulating leukocytes, inducing leukocyte chemotaxis along the chemotactic gradient, a critical component of the inflammatory response.[1] Chemokines are classified into two major (CC and CXC) and two minor (XC and CX₃C) families based on conserved sequence motifs but they may also be subdivided according to the functional activity that they orchestrate. Inflammatory chemokines are upregulated in response to tissue damage, whereas homeostatic chemokines direct leukocyte migration during normal immune surveillance as well as development.

Considering this exquisite control of inflammatory and immune responses within mammalian systems, it is perhaps not surprising that pathogens have evolved mechanisms to evade the host immune response coordinated by chemokines.[2] Indeed, it is now known that a number of mammalian viruses, including poxviruses and herpesviruses interfere with chemokine-receptor signaling through the production of chemokine mimics, chemokine receptor mimics or through the expression of chemokine binding proteins (CKBPs).[3] CKBPs are soluble proteins that do not possess structural homology to chemokines or their receptors and are capable of modulating signaling through the chemokine receptor axis by directly binding to chemokines.

A CKBP from human cytomegalovirus (HCMV) - called UL22A (or UL21.5) – has been shown to selectively bind to CC

chemokine ligand 5 (CCL5) – also called RANTES (Regulated on Activation, Normal T Cell Expressed and Secreted).[5] RANTES is an inflammatory chemokine that signals through the receptors CCR1, CCR3 and CCR5 and plays a critical role in the recruitment of a range of leukocytes, including T cells, basophils, eosinophils and macrophages into inflammatory sites. Together with T-cell secreted cytokines such as IL-2 and IFN-γ, RANTES also induces the activation and proliferation of natural killer cells to generate C-C chemokine-activated killer cells. It has been demonstrated that the mRNA encoding the 82 residue UL22A protein is packaged into virions and it is hypothesized that the protein is expressed and secreted by HCMV to modulate the host anti-viral response mediated through RANTES signaling as depicted schematically in Figure 1.

![Figure 1](#fig1)

Figure 1. Schematic illustration showing: expression, processing and secretion of UL22A (blue) by an HCMV-infected cell and UL22A binding to the chemokine RANTES (red), thus inhibiting RANTES ability to bind and activate chemokine receptors (black) and thereby inhibiting downstream leukocyte chemotaxis.

The selective RANTES binding exhibited by UL22A has a number of potential therapeutic applications. More specifically, given that the overexpression of RANTES results in excessive leukocyte recruitment and is associated with pathologies of numerous inflammatory diseases including asthma, arthritis, atherosclerosis, renal disease, Alzheimer’s disease and a variety of cancers,[6] CKBPs such as UL22A could serve a beneficial role in the control of inflammatory processes driven by this chemokine. Moreover, the ability to selectively suppress the response of a single chemokine would have important implications for anti-inflammatory agents with safer therapeutic profiles.

---

[a] X. Wang, Prof. R.J. Payne  
School of Chemistry, The University of Sydney  
Sydney, NSW 2006, Australia  
E-mail: richard.payne@sydney.edu.au  
Homepage: http://sydney.edu.au/science/chemistry/~payne/index.html  

[b] J. Sanchez, A/Prof M. J. Stone  
Department of Biochemistry and Molecular Biology  
Monash University  
Melbourne, VIC 3800, Australia  

Supporting information for this article is given via a link at the end of the document

This article is protected by copyright. All rights reserved.

Angewandte Chemie International Edition 10.1002/anie.201703059

COMMUNICATION

![Diagram]

Figure 2. Schematic of UL22A sequence showing the acidic sequence of amino acids (D60-D70) surrounding two Tyr residues (Y65 and Y69) that are predicted to be sulfated and the V71-L72 junction (orange and dotted line) selected for ligation-based assembly. NB: The sequence is numbered from amino acid 21 due to cleavage of a signal peptide (MARRLWILSLLAVTLTVALA) to form mature UL22A(21-103). [7]

Given our interest in chemokine-chemokine receptor interactions, we were intrigued by the binding selectivity of UL22A for a single chemokine (RANTES). Indeed, this selectivity has not been observed for chemokine binding proteins from other organisms and, in general, chemokines themselves do not signal through a single receptor axis. Upon inspection of the amino acid sequence of UL22A, we noted two tyrosine (Tyr) residues (Y65 and Y69 in Figure 2) that were embedded within an acidic sequence possessing six acidic aspartate and glutamate amino acids from residues 60-70 in the cysteine-free protein. Given the importance of sulfation of the N-terminus of chemokine receptors for tight binding to cognate chemokines, through their positively charged N-terminal extracellular domain, [8] we predicted that sulfation of Tyr65 and Tyr69 of UL22A would occur in HCMV-infected cells *in vivo* and would lead to a substantial improvement in binding to the chemokine RANTES. Indeed, it has been hypothesized that HCMV proteins (as well as other animal virus proteins) are capable of being modified in the host cell trans-Golgi apparatus during replication [9] and UL22A has already been predicted to be glycosylated. [5] Owing to the enzymatic nature of sulfation [10], it is likely that UL22A would be produced as a heterogeneous mixture of four isoforms; unsulfated 1, monosulfated on Tyr65 2, monosulfated on Tyr69 3 and disulfated (on both Tyr65 and Tyr69) 4 (Figure 2). To test this hypothesis, we set out to assess the effect of Tyr sulfation on Y65 and Y69 of UL22A on chemokine binding. Towards this end, we report herein the development of an efficient one-pot chemical synthesis of homogeneously sulfated variants of UL22A, together with binding studies with RANTES. We demonstrate that sulfation has a dramatic effect on the affinity of UL22A for the chemokine and provides new insights for the development of peptides to target chemokine-modulated diseases in the future.

In order to access homogeneously modified UL22A proteins, we turned to the use of Fmoc-solid-phase peptide synthesis (Fmoc-SPPS) in combination with peptide ligation methodology. [11] Given the absence of Cys residues in UL22A and a dearth of suitably placed Ala residues to facilitate native chemical ligation [12] or ligation-desulfurization chemistry, [11c],[13] respectively, we chose to disconnect the protein into two fragments; UL22A(21-71) 5-8 possessing differential sulfation patterns at the two putative Tyr sulfation sites and functionalized as a C-terminal selenoester, and UL22A(72-103) 9 bearing an N-terminal β-selenoleucine [(β)-SeLeu] (Scheme 1). Owing to the high acid lability of the tyrosine sulfate ester moiety, the functionality must be protected during SPPS. As such, we proposed the use of the acid stable neopentyl ester protecting group on the sulfated Tyr residues in 5-8 which could be removed following assembly of the protein. [14] We envisaged assembling the fragments via a novel one-pot (β)-SeLeu-mediated ligation-deselenization, inspired from the selenocystine-selenoester methodology recently reported. [15] Given the rapid reaction kinetics of the ligation methodology at selenocystine we anticipated that the ligation would proceed smoothly, despite the sterically encumbered Val-Leu ligation junction selected (that would be challenging to construct efficiently under a traditional native chemical ligation regime [16]). Our synthetic endeavours began with the preparation of suitably protected (β)-SeLeu 10, which was achieved in 8 steps as a single diastereomer from commercially available Garner's aldehyde in good overall yield (see Supporting Information for synthetic details and characterization). Synthesis of (sulfo)peptide selenoester fragments 5-8 was achieved on 2-chlorotrityl chloride resin via standard Fmoc-SPPS methods as we have reported previously [15b] (see Supporting Information for synthetic details). Sulfated tyrosine residues were incorporated using the commercially available neopentyl-protected sulfotyrosine cassette [Fmoc-Tyr(OSO₃CH₂C(CH₃)₃)-OH]. [14] Gratifyingly, following reverse-phase HPLC purification each of the differentially sulfated 51-mer peptide selenoesters were isolated in excellent yield (19-26%) based on the original resin loading (102 en bloc steps). Selenopeptide dimer fragment 9 was also synthesized via Fmoc-SPPS incorporating the (β)-SeLeu building block 10 on the N-terminus. Following acidolytic side chain deprotection (including deprotection of the PMB ether on the (β)-SeLeu) and cleavage from the resin dimeric peptide 9 was isolated in a 30% yield.

traditional native chemical ligation pathway. In our hands the use of a thiol-based reductant e.g. DTT led to slow ligation rates and a complicated product mixture due to the generation of selenyl-sulfides. As such, we opted for the use of the water soluble phosphine reductant *tris-carboxyethylphosphine* (TCEP), in combination with the radical scavengers diphenyl diselenide (DPDS) [15a] or ascorbic acid [17] to hinder the TCEP-promoted deselenization of the β-seleno moiety from 9. [18] Gratifyingly, optimal conditions using TCEP (1.5 eq. with respect to 9) and DPDS (50 mM) were found that provided the desired ligation products (general structures I and II, Scheme 1) with minimal deselenization of starting selenopeptide 9 (<10%) in trial ligation reactions with unsulfated selenoester 5 (see Supporting Information for details).

Figure 3. A) crude UPLC trace of the ligation between sulfopeptide selenoester 7 and selenopeptide 9 followed by *in situ* deselenization. * corresponds to buffer salts and TCEP=Se; B) UPLC trace of purified sulfated UL22A 3 (sulfated on Y69); C) ESI mass spectrum of sulfated UL22A 3. # denotes molecular ion with the loss of SO₃ on the mass spectrometer; D) MALDI-TOF spectrum of sulfated UL22A 3.

With optimized conditions in hand, 9 was ligated with the four differentially sulfated peptide selenoesters 5-8 under the additive ligation conditions (6 M Gn·HCl, 0.1 M Na₂HPO₄, pH = 6.2-6.5, 1.5 eq TCEP, 50 mM DPDS). After only 1 h each of the ligation reactions had reached completion (as judged by UPLC-MS analysis). The facile nature of this reaction was particularly impressive given the sterically encumbered nature of the Val-Leu ligation junction. As was the case with the parent ligation reaction at selenocysteine [15a] two products were generated, the symmetrical diselenide product I and the asymmetric diselenide product II, that were formed in a ratio of ca. 2:3 (see Supporting Information). In addition, the ligation conditions led to partial deprotection of the neopentyl protecting groups on the sulfotyrosine residues. This product mixture was inconsequential for the efficiency of the protein synthesis: Following *in situ* treatment with TCEP and DTT for 16 h, complete deselenization of the β-Se-Leu moiety into native Leu occurred, as well as deprotection of the neopentyl protecting groups (see Figure 3A for crude UPLC trace of one-pot ligation-deselenization reaction to afford sulfated UL22A 3). Following reverse-phase HPLC purification the library of differentially sulfated UL22A proteins 1-4 were isolated in excellent yield (51-58% over the two steps). It should be noted that 0.1 vol% formic acid was used in the HPLC

This article is protected by copyright. All rights reserved.

Angewandte Chemie International Edition 10.1002/anie.201703059

COMMUNICATION

elents (see Supporting Information). As such, fractions containing the desired sulfoproteins (determined by UPLC-MS) were lyophilized immediately to prevent loss of the acid labile sulfate ester modifications following purification. Exemplar data for purified monosulfated UL22A 3 are shown in Figure 3B-D (see Supporting Information for analytical data for synthetic UL22A proteins 1, 2 and 4).

With homogeneously sulfated UL22A proteins 1-4 in hand, we next tested our hypothesis that tyrosine sulfation of the UL22A protein would enhance binding affinity for RANTES. This was accomplished using a competitive fluorescence anisotropy assay in which a fluorescent sulfopeptide derived from the CCR3 receptor (FI-R3D) was displaced from RANTES by increasing concentrations of UL22A, thereby causing a reduction in the fluorescence anisotropy of the sulfopeptide (see Figure 4 and Supporting Information). [19] Fitting the displacement data to a competitive inhibition equation indicated that the concentration of unsulfated UL22A 1 required for 50% displacement of the fluorescent sulfopeptide (IC₅₀) was 4.9 ± 0.5 μM (Table 1). Similar displacement of the fluorescent competitor was achieved with much lower concentrations of sY65-monosulfated UL22A 2 (IC₅₀ = 166 ± 13 nM), sY69-monosulfated UL22A 3 (131 ± 10 nM) or sY65sY69-doubly sulfated UL22A 4 (12.6 ± 1.2 nM). It should be noted that a previous study has reported the “apparent Kᵈ” between RANTES and UL22A (expressed in mammalian cells) to be 0.32 nM; [5] the higher IC₅₀ value in the current assay results from the high chemokine concentration (100 nM) required for the competitive fluorescence assay. Our binding data demonstrate that addition of a single sulfate group to either tyrosine residue enhances binding affinity by ~30 to ~40-fold and sulfation of both Tyr residues enhances affinity ~400-fold. We next interrogated the binding affinity of the differentially sulfated UL22A proteins against monocyte chemoattractant protein-1 (MCP-1/CCL2), a chemokine involved in regulating migration and infiltration of monocytes and macrophages. While sulfation also enhanced the binding of UL22A to MCP-1, all forms of UL22A bound ~10-fold more weakly to MCP-1 than to RANTES, consistent with the previous report that UL22A is a selective binder of RANTES [5] (Table 1 and Supporting Information). Importantly, these data support our hypothesis and demonstrate for the first time that sulfation of UL22A enhances chemokine binding affinity. Future work in our laboratories will seek to identify whether this phenomenon is common across other CKBP.

In summary, we have described an efficient one-pot chemical synthesis of a library of four homogeneously sulfated chemokine binding UL22A proteins from human cytomegalovirus (HCMV) using a novel β-selenoleucine-mediated peptide ligation reaction followed by deselenization chemistry. Competitive fluorescence anisotropy binding assays using the synthetic UL22A (sulfo)proteins indicated that sulfation of tyrosine residues substantially enhanced binding affinity for the chemokine RANTES, with more than two orders of magnitude improvement in affinity for the doubly-sulfated UL22A compared to the unsulfated counterpart. We were also able to show that unmodified UL22A and sulfated UL22A showed an order of magnitude selectivity for RANTES binding over the chemokine

MCP-1. Interestingly, enhancements in chemokine-binding affinity have been observed upon sulfation of specific Tyr residues in peptides derived from N-terminal extracellular domains of chemokine receptors. [14c, 19-20] Although future studies will be required to determine whether UL22A structurally mimics the interactions of chemokine receptors with their cognate chemokines, this work has demonstrated the crucial role played by tyrosine sulfation in chemokine binding by this viral chemokine inhibitor. Furthermore, the sulfoproteins produced in this study also provide a valuable resource for future development of antibody reagents for the detection of site-specific tyrosine sulfate modifications of viral proteins *in vivo*.

![Figure 4](https://i.imgur.com/yourimage.png)

Figure 4. Binding of synthetic UL22A (sulfo)proteins 1-4 to human RANTES. Fluorescence anisotropy binding curves show that sulfation of UL22A at Tyr65 and/or Tyr69 strongly improves binding affinity for RANTES. Dose-response curves for binding against RANTES by increasing concentration of different UL22A isoforms with FI-R3D as a probe (displacement of FI-R3D from RANTES using purified UL22A isoforms) and error bars representing S.E.M. values.

Table 1. IC₅₀ values for binding of UL22A (sulfo)proteins 1-4 to human RANTES and MCP-1. Errors are standard error of the mean of experiments performed in triplicate.

| UL22A isoforms | IC₅₀ (nM) RANTES | IC₅₀ (nM) MCP-1 |
|----------------|------------------|------------------|
| unsulfated (1) | 4900 ± 500       | 19100 ± 5300     |
| sY65 (2)       | 166 ± 13         | 1900 ± 200       |
| sY69 (3)       | 131 ± 10         | 1900 ± 200       |
| sY65sY69 (4)   | 12.6 ± 1.2       | 130 ± 20         |

Acknowledgements

The authors would like to thank the University of Sydney (X.W.) and Monash University (J.S.) for the provision of PhD scholarships and the Australian Research Council for funding this research (Discovery Project DP160101324 and Future Fellowship FT130100150 to R.J.P.).

This article is protected by copyright. All rights reserved.

Angewandte Chemie International Edition

COMMUNICATION

Keywords: sulfation • leucine • peptide ligation • chemokine binding proteins • UL22A

[1] M. Baggolini, Nature 1998, 392, 565-568.

[2] a) J. Felix, S. N. Savvides, Nat. Rev. Immunol. 2017, 17, 112-129; b) M. L. Epperson, C. A. Lee, D. H. Fremont, Immunol. Rev. 2012, 250, 199-215.

[3] a) B. T. Seet, G. McFadden, J. Leukocyte Biol. 2002, 72, 24-34; b) H. Heidarieh, B. Hernaez, A. Alcami, Virus Res. 2015, 209, 67-75; c) M. J. Stone, J. A. Hayward, C. Huang, Z. E Huma, J. Sanchez, Int. J. Mol. Sci. 2017, 18, 342, DOI:10.3390/ijms18020342

[4] V. Gonzalez-Motos, K. A. Kropp, A. Viejo-Borbolla, Cytokine Growth Factor Rev. 2016, 30, 71-80.

[5] D. Wang, W. Bresnahan, T. Shenk, Proc. Natl. Acad. Sci. U. S. A. 2004, 101, 16642-16647.

[6] a) T. J. Schall, J. Jongstra, B. J. Dyer, J. Jorgensen, C. Clayberger, M. M. Davis, A. M. Krensky, J. Immunol. 1988, 141, 1018-1025; b) V. Appay, S. L. Rowland-Jones, Trends Immunol. 2001, 22, 83-87; c) D. Aldinucci, A. Colombatti, Mediators Inflammation 2014, 2014, article no. 292376.

[7] M. Hsu, C. T. Rauch, M. J. Gerhart, A. Kaykas, D. Cosman, J. Gen.Virol. 1999, 80, 437-440.

[8] a) M. J. Stone, R. J. Payne, Acc. Chem. Res. 2015, 48, 2251-2261; b) X. Liu, L. R. Malins, M. Roche, J. Sterjovski, R. Duncan, M. L. Garcia, N. C. Barnes, D. A. Anderson, M. J. Stone, P. R. Gorry, R. J. Payne, ACS Chem. Biol. 2014, 9, 2074-2081; c) J. P. Ludeman, M. J. Stone, Br. J. Pharmacol. 2014, 171, 1167-1179; C. J. Millard, J. P. Ludeman, M. Canals, J. L. Bridgford, M. G. Hinds, D. J. Clayton, A. Christopoulos, R. J. Payne, M. J. Stone, Structure 2014, 22, 1571-1581

[9] H. C. Lin, K. Tsai, B. L. Chang, J. Liu, M. Young, W. Hsu, S. Louie, H. B. Nicholas, G. L. Rosenquist, Biochem. Biophys. Res. Commun. 2003, 312, 1154-1158.

[10] W. B. Huttner, Nature 1982, 299, 273-276.

[11] a) S. B. H. Kent, Chem.Soc. Rev. 2009, 38, 338-351; b) R. J. Payne, C.-H. Wong, Chem. Commun. 2010, 46, 21-43; c) L. R. Malins, R. J. Payne, Curr. Opin. Chem. Biol. 2014, 22, 70-78; d) T. J. Harmand, C. E. Murar, J. W. Bode, Curr. Opin. Chem. Biol. 2014, 22, 115-121; e) S. Bondalapati, M. Jbara, A. Brik, Nature Chem. 2016, 8, 407-418.

[12] P. E. Dawson, T. W. Muir, I. Clark-Lewis, S. B. Kent, Science 1994, 266, 776-779.

[13] a) C. P. R. Hackenberger, D. Schwarzer, Angew. Chem. Int. Ed. 2008, 47, 10030-10074; Angew. Chem. 2011, 120, 10182-10228; b) P. E. Dawson, Isr. J. Chem. 2011, 51, 862-867; c) B. Premdjee, R. J. Payne, in Chemical Ligation: Tools for Biomolecule Synthesis and Modification (Wiley) 2017, 161-222.

[14] a) L. S. Simpson, T. S. Widlanski, J. Am. Chem. Soc. 2006, 128, 1605-1610; b) R. E. Thompson, X. Liu, J. Ripoll-Rozada, N. Alonso-Garcia, B. L. Parker, P. J. B. Pereira, R. J. Payne, Nature Chemistry 2017, DOI: 10.1038/NCHEM.2744; c) J. H. Tan, J. P. Ludeman, J. Wedderburn, M. Canals, P. Hall, S. J. Butler, D. Taleski, A. Christopoulos, M. J. Hickey, R. J. Payne, J. Biol. Chem. 2013, 288, 10024-10034.

[15] a) N. J. Mitchell, L. R. Malins, X. Liu, R. E. Thompson, B. Chan, L. Radom, R. J. Payne, J. Am. Chem. Soc. 2015, 137, 14011-14014; b) N. J. Mitchell, S. S. Kulkarni, L. R. Malins, S. Wang, R. J. Payne, Chem. Eur. J. 2017, 23, 946-952.

[16] a) Z. Harpaz, P. Siman, K. Kumar, A. Brik, ChemBioChem 2010, 11, 1232-1235; b) S. Shang, Z. Tan, S. J. Danishefsky, Proc. Natl. Acad. Sci. U. S. A. 2011, 108, 5986-5989; c) Z. Tan, S. Shang, S. J. Danishefsky, Angew. Chem. Int. Ed. 2010, 49, 9500-9503; Angew. Chem. 2010, 122, 9690-9693.

[17] P. S. Reddy, S. Dery, N. Metanis, Angew. Chem. Int. Ed. 2016, 55, 992-995; Angew. Chem. 2016, 128, 1004-1007.

[18] a) L. R. Malins, N. J. Mitchell, S. McGowan, R. J. Payne, Angew. Chem. Int. Ed. 2015, 54, 12716-12721; Angew. Chem. 2015, 127, 12907-12912; b) L. R. Malins, N. J. Mitchell, R. J. Payne, J. Pept. Sci. 2014, 20, 64-77.

[19] J. P. Ludeman, M. Nazari-Robati, B. L. Wilkinson, C. Huang, R. J. Payne, M. J. Stone, Org. Biomol. Chem. 2015, 13, 2162-2169.

[20] J. Z. Zhu, C. J. Millard, J. P. Ludeman, L. S. Simpson, D. J. Clayton, R. J. Payne, T. S. Widlanski, M. J. Stone, Biochemistry 2011, 50, 1524-1534.

Angewandte Chemie International Edition

COMMUNICATION

Modifications Matter: Human
cytomegalovirus produces a
chemokine binding protein called
UL22A that we have predicted to be
post-translationally sulfated on two
tyrosine residues. An efficient one-pot
chemical synthesis of a library of
homogeneously sulfated UL22A
proteins is described using a novel β-
selenoleucine-mediated
peptide ligation reaction followed by
deselenization chemistry. Binding
experiments unequivocally showed
that sulfation of tyrosine residues
substantially enhanced binding affinity
for the chemokine RANTES.

Xiaoyi Wang, Julie Sanchez, Martin J.
Stone, Richard J. Payne*

Page No. – Page No.
Sulfation of Human Cytomegalovirus
Protein UL22A Enhances Binding to
the Chemokine RANTES

This article is protected by copyright. All rights reserved.
