
ARTICLE IN PRESS

DCC YJMBI-65429; No. of pages: 25; 4C:

![](https://via.placeholder.com/100x100)

**Proteasome Structure and Assembly**

Lauren Budenholzer†, Chin Leng Cheng†, Yanjie Li† and Mark Hochstrasser

*Department of Molecular Biophysics & Biochemistry, Yale University, 266 Whitney Avenue, New Haven, CT 06520, USA*

Correspondence to Mark Hochstrasser: [mark.hochstrasser@yale.edu](mailto:mark.hochstrasser@yale.edu)  
[http://dx.doi.org/10.1016/j.jmb.2017.05.027](http://dx.doi.org/10.1016/j.jmb.2017.05.027)  

Edited by Andreas Martin

## Abstract

The eukaryotic 26S proteasome is a large multisubunit complex that degrades the majority of proteins in the cell under normal conditions. The 26S proteasome can be divided into two subcomplexes: the 19S regulatory particle and the 20S core particle. Most substrates are first covalently modified by ubiquitin, which then directs them to the proteasome. The function of the regulatory particle is to recognize, unfold, deubiquitylate, and translocate substrates into the core particle, which contains the proteolytic sites of the proteasome. Given the abundance and subunit complexity of the proteasome, the assembly of this ~2.5 MDa complex must be carefully orchestrated to ensure its correct formation. In recent years, significant progress has been made in the understanding of proteasome assembly, structure, and function. Technical advances in cryo-electron microscopy have resulted in a series of atomic cryo-electron microscopy structures of both human and yeast 26S proteasomes. These structures have illuminated new intricacies and dynamics of the proteasome. In this review, we focus on the mechanisms of proteasome assembly, particularly in light of recent structural information.

© 2017 Elsevier Ltd. All rights reserved.

---

## Introduction

Protein degradation is an essential facet of cellular function. In eukaryotes, most regulated protein degradation is performed by the ubiquitin-proteasome system [1–4]. The concerted action of a series of enzymes activates and affixes one or more ubiquitin moieties to a target protein [1,2]. Attachment of ubiquitin or ubiquitin polymers to proteins marks them as substrates of the proteasome [1,5–7]. The proteasome is a large, multimeric protease complex that binds, deubiquitylates, and unfolds its substrates prior to completing their degradation [3].

Proteasomes are found in eukaryotes, archaea, and some bacteria. For full activity, proteasomal activators associate with one or both ends of the proteasome core particle (CP) [3]. Known eukaryotic activators include the 19S regulatory particle (RP), proteasome activator PA28, and PA200/Blm10; hybrid proteasomes with different activators on either end of the CP cylinder have also been observed [8]. In this review, we will focus on the eukaryotic RP-activated proteasome, that is, the 26S proteasome (Fig. 1). Under certain conditions, the RP can be split into two large subcomplexes, called the base and lid [9]. The base and lid are major assembly intermediates that form en route to full proteasome formation [3]. Functionally, the major role of the lid is to coordinate substrate deubiquitylation with the unfolding and translocation of polypeptide substrates by the base. The base includes a heterohexameric ring of AAA ATPases (Rpt subunits) that is responsible for these latter activities.

Interestingly, both the CP and base require extrinsic assembly factors or chaperones for their efficient assembly, but the lid apparently does not [3,10,11]. Five assembly chaperones have been identified for base formation, while three factors are well established as important CP assembly chaperones. In the following sections, we will discuss recent advances in our understanding of the structure of each of these proteasomal complexes, and we will focus principally on their assembly mechanisms.

0022-2836/© 2017 Elsevier Ltd. All rights reserved. J Mol Biol (2017) xx, xxx–xxx

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), [http://dx.doi.org/10.1016/j.jmb.2017.05.027](http://dx.doi.org/10.1016/j.jmb.2017.05.027)

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

a)  
![](image1.png)

b)  
![](image2.png)

c)  
![](image3.png)

Fig. 1. The structure of the 26S proteasome. The CP is in gray with α-rings in dark gray and β-rings in light gray. The Rpt ATPase ring subunits of the RP base are colored light brown, while the Rpn1, Rpn2, and Rpn13 base subunits are pink. Individual lid subunits are labeled and depicted in cyan, and Rpn10 is blue. The CP subunits β2 and β7 are depicted in copper and yellow, respectively, to highlight their C-terminal tails that extend to neighboring subunits. The positions of individual CP and base subunits are indicated in Figs. 2 and 4 due to space limitations in Fig. 1. Figures were generated from PDB 3JCP.

The 20S CP

The dyad-symmetric CP is a highly conserved, barrel-shaped complex composed of four stacked heptameric rings [8]. Narrow substrate entry channels are created by the two outer rings, which are each formed by seven α-subunits. The two inner rings create an internal chamber that houses the proteolytic active sites responsible for protein cleavage; these rings are each formed by seven β-subunits. CP α-subunits include highly conserved N-terminal extensions that are absent from β-subunits. These N termini form a gate that controls substrate passage through the central α-ring channel [12]. Archaeal and bacterial CPs usually have a single type of α-subunit and β-subunit, each present in 14 copies in each particle. Thus, these proteasomes have 14 active sites arrayed within their central chambers.

In eukaryotes, seven distinct α-subunit paralogs form each heptameric outer ring and seven distinct β-subunit paralogs form each inner ring. The N-terminal peptide extension of the eukaryotic α3 subunit is the key contributor to the α-ring gate [12]. This gate is most often closed in the absence of activators [13]. Only three of the seven eukaryotic β-subunits (β1, β2, and β5) retain an intact active site, so each eukaryotic CP has six proteolytic active sites. The proteasome is a threonine protease in which the active-site Thr residue is at the N terminus of the β-subunit. These subunits are activated following autocatalytic processing of an N-terminal propeptide [14,15]. Recent biochemical and structural analyses indicate that propeptide autocleavage and substrate proteolysis utilize closely related mechanisms [16]. The Thr residue is part of a conserved Thr-Lys-Asp catalytic triad, which functions similarly in both processes [16].

Overview of CP assembly

CP biogenesis in eukaryotes is assisted by many factors to help ensure proper assembly. These factors include the intrinsic self-assembly properties of subunits, the N- and C-terminal extensions of specific subunits, and dedicated assembly chaperones [8]. Assembly is less complex for the simpler prokaryotic proteasomes, and only some assembly features are shared among bacterial, archaeal, and eukaryotic CPs. We will discuss these in turn.

Bacterial CP assembly

Proteasomes are found in the bacterial orders Actinomycetales and Nitrospirales [17]. However, a very recent bioinformatics analysis has uncovered proteasome-related genes in additional bacteria as well [18]. Proteasome assembly has been examined only in Actinobacteria to date. Based on in vitro analysis, actinobacterial CP assembly does not require prior formation of an α-ring (Fig. 2ai). Instead, individual α-

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

a)

(i)  
![Diagram](https://i.imgur.com/1234567.png)

(ii)  
![Diagram](https://i.imgur.com/8901234.png)

b)  
![Diagram](https://i.imgur.com/5678901.png)

mature  
proteasome  

preholo-  
proteasome  

half-proteasome  

Ump1  

Pba1-Pba2  

Pba3-Pba4  

Fig. 2. Models for CP assembly pathways. (a) Two proposed assembly pathways are the (i) α-ring-independent (or α-β heterodimer-initiated) pathway and (ii) α-ring-dependent pathway. (b) Assembly pathway of the constitutive CP in eukaryotes. The α-ring is assembled with the help of two pairs of chaperone proteins: Pba3–Pba4 and Pba1–Pba2. β2 and Ump1 then associate with the α-ring. The association of β3 and β4 is accompanied by the dissociation of Pba3–Pba4. Then, β5, β6, and β1 are incorporated sequentially, followed by β7. The insertion of β7 leads to the dimerization of two half-proteasomes to form the preholoproteasome. In this complex, Pba1–Pba2 and Ump1 are still associated. In the last step, Pba1–Pba2 dissociates from the CP, the propeptides of the β-subunits undergo autoprocessing to activate the CP, and Ump1 is degraded, forming the mature CP.

and β-subunits form α-β heterodimers, which oligomerize to form a double-ring half-proteasome, followed by dimerization of half-mers [19–21]. Upon dimerization, the N-terminal propeptides of the β-subunits are autocatalytically removed and the N-terminal catalytic threonine nucleophiles are activated. Bacterial α- or β-subunits alone cannot self-assemble into homoheptameric ring structures due to the limited α-α and β-β contact surfaces between subunits [21,22].

Instead, the N-terminal propeptides of β-subunits make significant contact with α-subunits and augment α-α contacts, facilitating α-β heterodimer multimerization during half-proteasome formation [19–21]. In some species, such as *Mycobacterium tuberculosis*, the β propeptides protrude from the β face of the half-proteasome and inhibit the completion of assembly [19]. To date, no proteasome assembly chaperones have been identified in bacteria.

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

# ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

## Archaeal CP assembly

Archaeal proteasomes are similar to bacterial proteasomes in that they often contain only one type each of α- and β-subunits, although some have highly similar variants of the subunit, which likely reflect recent gene duplications. It has been proposed that the archaeal CP assembles in an α-ring-dependent manner (Fig. 2aii), based on detailed *in vitro* studies of the assembly of the *Thermoplasma acidophilum* CP [23]. When *T. acidophilum* α- and β-subunits are coexpressed in *Escherichia coli*, the α-ring appears to form first, serving as a template for the attachment of β-subunits, leading to the formation of half-proteasomes that dimerize to form an active CP. The short N-terminal propeptides of these archaeal β-subunits have no obvious effect on CP assembly *in vitro* [23]. It is possible that in archaeal cells, the β propeptides prevent N-terminal acetylation and the consequent irreversible inactivation of the catalytic Thr1 residues prior to assembly, as is true for eukaryotic active-site-bearing β-subunits [24].

The conserved N-terminal helices in archaeal α-subunits are required for the formation of the α-ring gate [14]. They are also essential for α-ring self-assembly, at least *in vitro* [14]. Given the compositional simplicity of archaeal proteasomes, it was not anticipated that trans-acting factors would be required for assembly. Nevertheless, proteins related to the Pba1–Pba2 (PAC1–PAC2) proteasome assembly chaperones of eukaryotes (see below) have been found in diverse archaeal species, and evidence that they promote CP assembly has been found from *in vitro* studies [25].

Both archaeal and bacterial CPs have similar structures [8]. A lingering question has been why the assembly of archaeal proteasomes seems to depend on the initial formation of a full α-ring heptamer, while bacterial proteasome assembly is α-ring-independent. This puzzle may have been resolved based on the recent report of a second potential pathway of archaeal proteasome assembly [26]. This pathway is α-ring-independent, similar to that of actinobacteria (Fig. 2ai). For the archaeon *Methanococcus maripaludis*, the formation of an α-ring is not required for efficient assembly of the CP when both *M. maripaludis* α- and β-subunits are coexpressed in *E. coli*. This conclusion is based on an *M. maripaludis* α-subunit mutant that cannot form a free α-ring but is still able to form heterodimers with coexpressed wild-type *M. maripaludis* β-subunits, leading to the formation of functional archaeal CPs [26]. The existence of two archaeal CP assembly pathways suggests that proteasome-containing bacterial species, which are relatively rare, may have acquired proteasome genes from ancient archaea but retained only the α-ring-independent assembly pathway [26]. Since proteasome architecture is conserved in all three domains of life, it is possible that CP assembly without prior α-ring formation also occurs in eukaryotes, but this has not yet been documented.

## Eukaryotic CP assembly

### Overview

Not surprisingly, eukaryotic CP assembly is more complicated than that in bacteria or archaea since eukaryotes express at least 14 different paralogous CP subunits [8]. Most evidence to date strongly supports a chaperone-assisted, α-ring-dependent assembly mechanism (Fig. 2b), similar to the first CP assembly pathway described for archaea (Fig. 2aii). Two pairs of heterodimeric chaperone proteins, yeast Pba1–Pba2 (PAC1–PAC2 in humans) and Pba3–Pba4 (human PAC3–PAC4), are thought to promote α-ring assembly, while an additional chaperone protein, Ump1 (hUMP1, POMP, or proteasemblin in humans), functions in proper incorporation of β-subunits and half-mer dimerization. The order of β-subunit incorporation varies somewhat for different proteasomal isoforms and possibly different species. β7 is always the last subunit to incorporate; its insertion into the β-ring completes half-mer formation, which is closely coupled to dimerization of two half-mers. The β-subunit precursors in the resulting “preholoproteasome” are then autoprocessed, with the released propeptides being degraded along with Ump1 to yield a mature, active 20S proteasome (CP).

### Assembly of the α-ring

In eukaryotes, the α-ring has been proposed to act as an assembly template based on the observation that both *Trypanosoma brucei* α5 [27] and human α7 [28] can self-assemble into homoheptameric rings. Whether either heteroheptameric or simpler α-rings are common assembly intermediates *in vivo* is still not clear. Isolated human α7 subunits can form double heptameric ring structures [28,29], similar to the half-proteasome structure. These would be expected to be dead-end complexes, but the stacked α7 double-ring readily disassembles upon introduction of α6 subunits [29]. This suggests that proteasome assembly may involve disassembly of non-native oligomers. Additionally, the *Saccharomyces cerevisiae* α4 subunit can form high-molecular-weight oligomers *in vivo*; when examined using site-specific disulfide crosslinking and SDS gel analysis, an α4 laddering pattern is revealed, suggesting that these structures are α4 rings [30]. The same α4 laddering pattern was observed in mammalian cells [31]. Despite the apparent existence of α4 rings in both yeast and mammals, their physiological contribution to proteasome assembly remains unclear.

As noted above, eukaryotic CP assembly is generally thought to be α-ring-dependent. Support for this comes from the detection of an α-ring complex in mammalian cell lysates [32]. All seven different α-subunits and the dimeric CP assembly chaperone PAC1–PAC2 are present in this complex. The association of PAC1–PAC2 suggests that it may function in assisting α-ring assembly [32–34]. PAC3–PAC4 (Pba3–Pba4) also appears to aid assembly in the early stages of CP formation [31, 32, 34–37]. The exact sequence of α-subunit additions during the formation of an α-ring is still unknown.

### Assembly chaperone Pba1–Pba2 (PAC1–PAC2)

Human PAC1 and PAC2 form a heterodimer and stabilize each other *in vivo* [32]. In mammalian cells, knockdown of PAC1, PAC2, or both using short interfering RNA leads to slow cell growth, reduced levels of α-rings, and increased levels of misassembled α-ring dimers. This suggests that PAC1 and PAC2 help assemble the α-ring at least in part by preventing α-rings from dimerizing, maintaining the α-ring in a state capable of β-subunit incorporation [32]. Like PAC1 and PAC2, yeast Pba1 and Pba2 function together as a heterodimer [33, 34]. In contrast to PAC1–PAC2, Pba1–Pba2 is metabolically stable [32]. Deletion of *PBA1* and/or *PBA2* results in no obvious growth defect; however, either deletion can exacerbate the growth defect observed in certain proteasome mutants [33, 34]. Additionally, Pba1–Pba2 was found to be exclusively associated with CP assembly intermediates *in vivo*, consistent with a role of Pba1–Pba2 in proteasome assembly [33].

Structures of three specific yeast CP assembly intermediates bearing Pba1–Pba2 have been characterized: the “15S intermediate” (Ump1, Pba1–Pba2, all α- and all β-subunits except β7), the “preholoproteasome” (immature full CP plus Ump1 and Pba1–Pba2) [38], and a reconstituted Pba1–Pba2–CP complex [39] (Fig. 3b). The first two structures were derived from negative-stain electron microscopy (EM), while the third was determined by X-ray crystallography. In the 15S intermediate (Fig. 3b, top left), Pba1–Pba2 is partially embedded in the central cavity of the α-ring, where the diameter of the pore is larger than that of a mature CP. Pba1–Pba2 can be seen interacting with all alpha subunits except α1 and α5 and helping to maintain the integrity of the α-ring [38]. The second state of Pba1–Pba2 is shown in the negative-stain EM structure of the preholoproteasome (Fig. 3b, top right), where β7 has inserted and the two CP half-mers have come together. Pba1–Pba2 is shifted out of the central cavity and located away from the center toward α5. Compared to the 15S structure, the Pba2–α7 interaction disappears in the preholoproteasome, while Pba1 interacts with the preholoproteasome at the α5/α6 interface. The disappearance and appearance of these interactions suggest that Pba1–Pba2 has distinct roles during different stages of CP assembly.

The third complex containing Pba1–Pba2 that was captured was a crystal structure of a reconstituted Pba1–Pba2–CP complex (Fig. 3b, bottom). It represents a snapshot of the interaction between Pba1–Pba2 and mature CP, presumably when Pba1–Pba2 is poised for release. Pba1–Pba2 has been shown to dissociate easily from mature CP at physiological salt concentrations [39]. Both Pba1 and Pba2 have a C-terminal HbYX (hydrophobic-tyrosine-any amino acid) motif [25], which is found in several proteasome activators and is required for the activators to interact with the α pocket formed by two adjacent α-subunits. In this crystal structure, the C terminus of Pba1 binds to the α5/α6 pocket and the C terminus of Pba2 to the α6/α7 pocket. Similar to the preholoproteasome, the gate of the α-ring is closed in the Pba1–Pba2–CP complex [39].

Pba1–Pba2 binds to the immature CP 3 orders magnitude more tightly than it does to mature CP [40]. This affinity switch is likely influenced by both the HbYX motifs of Pba1 and Pba2 and allosteric communication from β-subunit active sites to the outer surface of the α-ring [25, 40]. Tight binding of Pba1–Pba2 to CP precursors prevents their association with the RP, potentially ensuring RP–CP interaction only when the CP is properly processed to its mature form.

PbaA and PbaB were identified as archaeal relatives of Pba1 and Pba2, respectively [25]. Since archaeal proteasome subunits can form CPs spontaneously without dedicated chaperones, the physiological roles of these proteins remain to be determined. Studies have shown that PbaA and PbaB have distinctive functions in different species. In certain species, PbaA, but not PbaB, interacts with immature archaeal proteasomes at α pockets using a C-terminal HbYX motif [25]. In other species, PbaB forms a homotetramer and acts as an ATP-independent proteasome activator [41], while PbaA forms a homopentamer that is incapable of binding to the mature CP [42].

### Assembly chaperone Pba3–Pba4 (PAC3–PAC4)

Similar to PAC1 and PAC2, human PAC3 and PAC4 form a heterodimer [34]. Knockdown of PAC3 with small interfering RNA in mammalian cells leads to free α-subunit accumulation and proteasome assembly defects, supporting the hypothesis that PAC3 is involved in α-ring assembly [43]. In addition, knockdown of PAC3 leads to accumulation of proteasomal subparticles that lack the α4 subunit; these complexes appear to be aberrant dead-end assembly species [31].

Yeast Pba3 and Pba4 also form a heterodimer and interact most strongly with α5 subunits early in proteasome assembly [35, 36]. Deletion of *PBA3*

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

a)

![Image](image1.png)

b)

![Image](image2.png)

Fig. 3. Two pairs of CP assembly chaperone proteins. (a) Pba3–Pba4 associates with α5 and makes contacts with α4 and α6. The steric clash of Pba3–Pba4 with β4 and β5 is shown. Yeast Pba3–Pba4–α5 X-ray structure was modeled into the full CP structure by superimposing α5 from the two structures. The view is from the axial channel of the CP. α4: light brown; α5: cyan; α6: in dark gray; β4: purple; β5: light blue; Pba3: yellow; Pba4: magenta. The black circle in the inset highlights the steric conflict of Pba3–Pba4 with β4 and β5. Figures were generated from Pba3–Pba4–α5 (PDB ID: 2Z5C) and yeast CP (PDB ID: 5CZ4) structures. (b) Different positions of Pba1–Pba2 during CP assembly. Top left: Negative-stain EM structure of the 15S intermediate. Pba1–Pba2 locates in the widened pore of the α-ring. The figure was generated from EMD-2656 using Chimera. Top right: Negative-stain EM structure of the preholoproteasome. Pba1–Pba2 sits on top of the α-ring. The figure was generated using EMD-2658. Bottom left: X-ray crystal structure of a reconstituted Pba1–Pba2–20S complex. Bottom right: Highlight of the insertion of the HbYX motifs of Pba1 and Pba2 into CP α-ring pockets. The C termini of Pba1 and Pba2 are marked with arrows (PDB ID: 4G4S).

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

and/or PBA4 results in the accumulation of polyubiquitinated proteins, decreased proteasome activity, and accumulation of CP assembly intermediates, supporting the role of Pba3–Pba4 as a proteasome assembly chaperone.

In budding yeast, α3 is the only nonessential CP subunit, and lack of α3 leads to the formation of an alternative proteasome isoform where a second α4 subunit incorporates at the α3 position. These particles, which are functional, have been called “α4-α4” proteasomes [44]. The ability to form α4-α4 proteasomes appears to be conserved from yeast to humans [31, 44]. Overexpression of PAC3 reduces the formation of this isoform, and conversely, PAC3 knockdown enhances its levels [31]. This suggests that PAC3 plays an important role in regulating the assembly of the α4-α4 proteasome. Similarly, in yeast pba3 or pba4 null mutants, ~20–50% of CP particles are of the α4-α4 type [35]. In yeast, the absence of the Pba3–Pba4 assembly chaperone also results in the accumulation of a dead-end complex that is composed of seven α-subunits plus β2, β3, and β4, but two copies of α2 are present and no α4 is incorporated [36, 37]. Thus, Pba3–Pba4 and PAC3–PAC4 may guide the correct positioning of α2, α3, and α4 subunits [35] and prevent off-pathway complex formation [36, 37].

Crystal structures of Pba3–Pba4 and a Pba3–Pba4–α5 complex reveal that the chaperone heterodimer structurally resembles individual α- and β-subunits of the CP [36]. However, the interaction of α5 with Pba3–Pba4 in the ternary complex is different from how α5 binds its CP neighbors α4, α6, and β5. When modeled into a full 20S proteasome structure by aligning α5 in the two complexes, Pba3–Pba4 binds α5 at a position near the axial channel of the α-ring (Fig. 3a). Pba4 interacts with α4 to promote the incorporation of the latter next to α5, explaining why α4 is missing in the dead-end complex in pba4 knockouts [37]. Furthermore, the location of Pba3–Pba4 with respect to α5 would clash sterically with the β4 subunit in the mature CP; therefore, Pba3–Pba4 must dissociate from α5 prior to β4 incorporation during proteasome assembly [37]. This accounts for the absence of Pba3–Pba4 in the “13S intermediate” (all α-subunits, β2, β3, β4, Pba1–Pba2, and Ump1) [33], the earliest CP intermediate detected in yeast so far, indicating that Pba3–Pba4 contributes to proteasome assembly before this intermediate forms.

### Incorporation of β-subunits

Based on single gene knockdowns in cultured mammalian cells, each β-subunit appears to be added sequentially to a preformed α-ring to form a half-proteasome [45]. Among the seven β-subunits, β2 is the first to be added to the α-ring, followed by β3, β4, β5, β6, β1, and finally β7 [45]. N-terminal propeptides and C-terminal tails of several β-sub-

units act as “intramolecular chaperones” to promote the process [45]. A dedicated extrinsic chaperone, Ump1, assists in assembly as well. The contributions of these factors are described below.

As the first subunit to join the α-ring, β2 uses its unusually long C-terminal tail to help dictate the directionality and specificity of proteasome assembly. The β2 C-terminal tail, which is essential for viability, wraps around β3 and makes contact with β4 [24, 45, 46] in the mature CP (Fig. 1). β3 and β4 are thought to incorporate sequentially after β2. It should be noted, however, that the experiments performed could not distinguish whether single subunits or small oligomers are added; the β2 and β3 subunits, for example, could potentially dimerize prior to associating with the α-ring.

The three active-site-bearing subunits (β1, β2, and β5) all have N-terminal propeptides that are autocatalytically removed near the end of CP assembly. These propeptides all play some role in subunit processing and assembly of the CP [24]. The most prominent CP intramolecular chaperone is the N-terminal propeptide of β5 (β5pro), which is 75 residues in length in yeast [15] and is essential for proteasome biogenesis in both yeast and humans [15, 45]. β5pro appears to function with Ump1 to promote propeptide autocleavage during proteasome maturation, and it facilitates dimerization of two half-proteasomes, similar to the dimerization function of the C-terminal tail of β7 [15, 47]. A detailed structure-function analysis of β5pro further clarified its contribution to CP assembly [47]. The N-terminal half of β5pro is poorly conserved and is not essential for viability, although substantial defects in proteasome maturation are observed when this half of β5pro is absent. Sequences closer to the cleavage site are more conserved and contribute to proteasome assembly downstream of 13S intermediate formation. Sequences immediately adjacent to the autoprocessing site are not critical for assembly but contribute strongly to autocleavage [47]. In mammalian cells, β5pro is required for β6 incorporation but is not necessary for its own correct placement [45, 48]. The N-terminal propeptides of β6, β1, β2, and β7 are not essential for proteasome assembly in yeast [24, 33, 49]; the same is true for mammalian cells except for the β2 propeptide, which is essential for β3 incorporation [45].

#### Dimerization of half-proteasomes

Because β7 incorporation and half-mer dimerization are very tightly coupled temporally, it had been unclear which occurred first. The entry point of β7 was clarified by a recent study in yeast [47]. In a strain that retards CP assembly, an intermediate of a size consistent with a half-proteasome was identified and found to contain the unprocessed precursor β7 subunit. The results are most consistent with β7 incorporating to form half-proteasomes prior to dimerization rather than dimerization preceding β7

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

insertion. β7 possesses a unique long C-terminal tail that promotes half-mer dimerization [33,46,47,50]. This tail extends into the interface between β1 and β2 on the opposite β-ring and clamps the two half-proteasomes together (Fig. 1). Deletion of this region of β7 causes the accumulation of proteasome assembly intermediates and active-site autoproces-sing defects [33,46]. Based on negative-stain EM structures of 15S and preholoproteasome complexes, the incorporation of β7 appears to cause the movements of β-subunits into their ultimate positions, which lead to changes in the positioning of α-subunits as well [38]. Together, these movements result in the final formation of the gate of the α-ring and a shift upward in the location of Pba1–Pba2 [38].

Assembly chaperone Ump1 (hUmp1)

In addition to the aforementioned intra- and inter-molecular chaperones, the dedicated CP chaperone Ump1 promotes the precise assembly of β-subunits [51]. Ump1 is an intrinsically disordered protein and was the first proteasome chaperone protein discovered [38,51–53]. In yeast, Ump1 acts early in the incorporation of β-subunits into the CP. It associates with the CP intermediates along with or shortly after the first β-subunits (β2, β3, and β4) are added to the α-ring [33]. In human cells, hUMP1 is essential for the recruitment of β2 to the α-ring, a role that is not shared with yeast Ump1 [45]. A negative-stain EM structure of the yeast 15S CP assembly intermediate purified via Ump1, together with crosslinking data, gives some hints regarding the interactions between Ump1 and the 15S complex [38]. Ump1 loops around the inner chamber of the complex, making contacts with α1, α4, β4, and β6. The N terminus of Ump1 is located near the interface between β6 and the incoming β7, possibly serving as a detector to sense the arrival of β7. The N-terminal region of Ump1 is expected to be in the vicinity of β5pro in the 15S complex, potentially permitting a direct interaction between these proteins. However, the precise nature of how Ump1 and β5pro interact with one another remains unclear [38].

Alternative CP isoforms

In mammalian cells, several CP variants have been described in which the active subunits of the “constitutive” 20S proteasome have been replaced by inducible or tissue-specific paralogs. The “immunoproteasome” is the best studied of these variants [54]. It has three β-subunit substitutions; β1i, β2i, and β5i replace β1, β2, and β5, respectively. These alternative subunits are ~60% identical to the subunits they replace. The immunoproteasome alters the population of peptides generated for major histocompatibility complex class I antigen presentation.

Notably, the assembly pathway of such alternative proteasomes differs from that of constitutive CPs.

Both β1i and β2i are assembled onto the α-ring ahead of other β-subunits, initiating β-ring assembly [54]. After incorporation of β3, β5i is incorporated prior to β4, unlike the order of β-subunit addition in constitutive proteasome assembly. The propeptides of the individual immunosubunits help drive their selective incorporation specifically into immunoproteasomes [55,56]. Most immunoproteasomes bear all three βi subunits in each β-ring, but some are assembled as mixed proteasomes containing both constitutive β and βi subunits [57]. This implies some flexibility in the order of subunit addition during proteasome assembly. It is also expected to further enhance peptide diversity for antigen presentation.

The thymus-specific proteasome (thymoproteasome) is another alternative CP isoform, assembled exclusively with β1i, β2i, and β5t at the expected positions in the β-ring even in the presence of β5i [54]. Similar to β5i, β5t is incorporated after β3 incorporation but independently of β4 insertion. Interestingly, the propeptide of β5t is what drives its early incorporation, whereas the mature domain of β5i is responsible for its ability to insert prior to β4 addition [54]. The thymoproteasome plays an essential role in the positive selection of immune CD8⁺ T cells [58].

In addition to the β-subunit-substituted proteasomes, there are CPs that contain alternative α-subunits. The assembly process of α-subunit-substituted proteasomes is not as clear as that of the β-subunit-substituted ones. As introduced above, the α4-α4 proteasome is formed under conditions that include the deletion of α3, overexpression of α4, deletion of *PBA3* (*PAC3*) or *PBA4* (*PAC4*), or environmental stimuli such as oxidative stress [30,31,35]. The exact assembly pathway of the α4-α4 proteasomes is unclear. One clear difference from constitutive proteasome assembly is that the α4-α4 CP does not require the assembly chaperone Pba3–Pba4 (*PAC3*–*PAC4*).

Several other α-subunit-substituted proteasomes have been documented. These include spermatoproteasomes, which contain an alternative α4 subunit, α4s, which is synthesized exclusively in male germ cells after their differentiation into spermatocytes [59,60], and testis-specific proteasomes, containing an alternative α6 subunit (α6T), which is essential for fertility in *Drosophila* [61,62]. Selective assembly of α-subunit-substituted alternative CPs must distinguish alternative α-subunits from their constitutive counterparts. The mechanisms by which this occurs are still unknown.

Other factors assisting CP assembly

Many other factors may have an impact on CP assembly. Comprehensive proteomic analyses of co- and post-translational modifications of proteasome subunits have revealed a plethora of modifications, both constitutive and transient [63]. At least some of these modifications may modulate proteasome

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

assembly. The only example of this so far is from the archaeon *Haloferax volcanii*. Specifically, changes in N<sup>α</sup>-acetylation of the *H. volcanii* α1 subunit alter the rate or the level of α-ring assembly [64]. Additional trans-acting factors can also contribute. Proteins in the transmembrane recognition complex in mammals/ guided entry of tail-anchored proteins in yeast pathway [65] and a zygote-specific proteasome assembly chaperone in mouse [66] were shown to participate in eukaryotic CP assembly.

Yeast Blm10 (human PA200), a large dome-shaped Huntingtin, elongation factor 3 (*EF3*), protein phosphatase 2A (PP2A), TOR1(HEAT)-repeat protein, has been reported to be associated with CP assembly intermediates [33,50,67,68], suggesting a potential role of Blm10 in modulating CP assembly. However, Blm10 also associates with mature proteasomes and has apparent functions beyond CP assembly [69–73]. Yeast Fub1 (human PI31), a proline-rich protein [74], has been reported to be associated with several CP subunits; loss of Fub1 exacerbates defects caused by mutations in proteasome subunits or assembly chaperones [75,76]. These data suggest a possible role of Fub1 in modulating proteasome assembly. However, human PI31 has no effect on proteasome content and function [77]. A study on the *Drosophila melanogaster* PI31 did implicate DmPI31 in promoting RP assembly [78]. More studies will be required to establish the contributions of Blm10/PA200 and Fub1/PI31 to proteasome assembly. Given the various types of CPs, more proteins that modulate proteasome assembly are likely to exist.

### The 19S RP

The RP appears to be unique to eukaryotes and highly conserved among them; only a modest degree of subunit variability has been documented. Moreover, 19 subunits comprise the basic RP core in yeast, but a wide range of proteasome-interacting proteins are also known [79]. Its two main components, the nine-subunit base and nine-subunit lid, form a conformationally dynamic complex that binds an additional subunit, Rpn10. Rpn10 is one of several ubiquitin receptors in the RP, and it also stabilizes the interaction between lid and base. In the following sections, we review structural aspects of the RP and its assembly.

### The RP Base

#### Base composition, structure, and function

The base is composed of six paralogous AAA-ATPases termed Rpt1–Rpt6 and three non-ATPases, Rpn1, Rpn2, and Rpn13. Every Rpt subunit has an N-terminal α helix followed by an oligonucleotide/

oligosaccharide-binding (OB) fold, a large AAA-ATPase domain, and a C-terminal helical domain [80,81]. The six Rpt subunits associate in three pairs (Rpt1–Rpt2, Rpt3–Rpt6, Rpt4–Rpt5), which arrange into a ring to form a trimer of dimers in the order Rpt1-Rpt2-Rpt6-Rpt3-Rpt4-Rpt5 (Fig. 4a). This order was initially determined via disulfide crosslinking [82] and was validated by numerous cryo-EM structures of the 26S proteasome. Each Rpt subunit makes extensive interactions with its Rpt heterodimer partner, but the specificity of these pairwise interactions is believed to be largely dictated by the coiled coils (CCs) formed by their N-terminal helices [80,81].

#### AAA-ATPase structure and conformational changes

Cryo-EM structures revealed that the Rpt subunit domains arrange in a two-tiered ring configuration (Fig. 4a) [83,84]. The upper ring is made from the OB folds of the Rpt subunits, from which the N-terminal CC domains radiate upward and outward [83,84]. The lower, larger ring is composed of the large ATPase and C-terminal helical domains of these subunits. The CC of Rpt1–Rpt2 interacts with Rpn1, while the CC of Rpt3–Rpt6 contacts Rpn2 [83,84]. By contrast, the Rpt4–Rpt5 CC extends significantly from the body of the base without interaction with other proteasome subunits in the basal s1 state (see below) [83]. The three CCs also differ in length, with the Rpt3–Rpt6 CC being the longest, followed by Rpt4–Rpt5 and Rpt1–Rpt2 [83,84]. These data all suggest that the CCs have distinct functions within the RP.

The unstructured C-terminal tails of specific Rpt subunits bind to pockets between adjacent α-subunits of the CP, and one or more of these interactions help trigger gate opening of the CP, as described earlier. Gate opening allows the translocation of unfolded substrates into the proteolytic chamber for degradation [85]. The C-terminal tails of Rpt2, Rpt3, and Rpt5 contain the conserved HbYX motif thought to be important for CP gate opening, although recent cryo-EM data on the 26S proteasome indicate that the binding of these three tails is not sufficient for opening [86,87].

Initial subnanometer-resolution cryo-EM structures of the proteasome revealed that the Rpt ATPase ring is tilted at an angle of ~10° relative to the CP α-ring surface and toward the lid [88]. Additionally, the ATPase domains of the Rpt subunits in the ring adopt a spiral staircase-like configuration in the order of Rpt3-Rpt4-Rpt5-Rpt1-Rpt2 from top to bottom relative to the CP, with Rpt6 at about the same height as Rpt5 and bridging the vertical displacement between the highest (Rpt3) and lowest (Rpt2) subunits [84,88,89]. When the ring binds to a substrate, the conformation of the ATPase ring transitions to a more planar configuration with alignment of the RP ATPase and CP central channels [90].

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

a) Side View  
b) Rpn2  
c) Rpn10 (vWA domain)-ubiquitin complex  

Fig. 4. Structures of the Rpt heterohexameric ring, Rpn2, and an Rpn10-ubiquitin conjugate. (a) Top and side views of the *Saccharomyces cerevisiae* Rpt hexameric ring. The side view reveals that the ring arranges into two roughly coaxial rings or spirals. The upper ring of OB folds follows the CCs, extending upward, which join pairs of ATPases. The lower tier consists of the AAA-ATPase and C-terminal domains. The top view shows the narrow central pore of the Rpt ring through which substrates are threaded (PDB ID: 5MP9). (b) Crystal structure of yeast Rpn2. The structure on the left illustrates the N-terminal, C-terminal, and toroidal domains of Rpn2. The structure on the right highlights a single repeat (in cyan) of the 11 PC repeats of the toroidal domain (PDB ID: 4ADY). (c) Crystal structure of yeast Rpn10 vWA domain in complex with ubiquitin. The isopeptide bond formed between ubiquitin-G76 and Rpn10-K84 residues is boxed (PDB ID: 5LN1).

The ability to reconstitute the full base by co-expression of yeast subunits in *E. coli* has enabled the determination of the distinct roles of each Rpt subunit [90]. Mutational analyses of the ATP-hydrolyzing activity of the Rpt subunits have indicated that subunits located at the top of the ring, such as Rpt3 and Rpt4, contribute more to binding and translocation of substrates compared to subunits positioned lower in the ring, such as Rpt1 and Rpt2 [90].

Published cryo-EM structures of the yeast proteasome also indicate that the base is dynamic and exists in multiple conformational states (see Fig. 9 for similar models of the human proteasome). Several of these conformations have been captured using large samples of proteasomes incubated with various nucleotides and nucleotide analogs; the imaged particles were subsequently subjected to deep classification to tease out different conformations. Four distinct conformations have been elucidated thus far: the substrate-accepting (s1), substrate-commitment (s2), substrate-processing (s3), and the recently identified gate opening of CP (s4) states [87,91]. A major feature of the s1 state is the structural heterogeneity of Rpn1; this is likely related to Rpn1's role in recruiting various extrinsic ubiquitin receptors or ubiquitinated substrates [91]. Rpn1 in this state is also bound to the CC of Rpt1–Rpt2 [91]. The base conformations in s1 and s2 show several subtle differences. In the s2 conformation, the ATPase ring is less tilted than s1 relative to the CP rings [91]. The N-terminal CCs of the Rpt subunits show the most changes between s1 and s2: the Rpt4–Rpt5 CC engages the ubiquitin receptor Rpn10, the Rpt3–Rpt6 CC interacts with Rpn2 and the lid, and according to a recent report, the Rpt1–Rpt2 CC develops a kink [87,91].

In the s3 conformation, the ATPase ring is further flattened relative to s2 and is virtually planar relative to the CP [91]. This change allows the pore of the ATPase ring to align with the (still occluded) substrate channel of the CP [87,91]. The HbYX motifs of Rpt2, Rpt3, and Rpt5 are bound to α-subunit pockets on the CP in all four conformations, indicating that the binding of these HbYX motifs is not sufficient to open the CP gate since gate opening is not observed in s1–s3 [87]. In s4, the density of the C-terminal tail of Rpt6 is detected in the pocket between the α3 and α4 subunits, suggesting that Rpt6 tail binding may be necessary for the opening of the CP gate, although further investigation is required to confirm this [87].

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

Non-ATPase subunits of the base and ubiquitin receptors

The base Rpn1 and Rpn2 subunits are the two largest subunits of the proteasome. The central regions of both proteins contain 11 proteasome/cyclosome (PC) repeats [92]. Each repeat is roughly 35–40 residues in length and consists of a pair of antiparallel α helices [67]. A crystal structure of Rpn2 revealed a central domain in which the PC repeats form a toroidal structure (Fig. 4b) [92]. This central region is flanked by an N-terminal domain consisting of stacked α helices, forming a rod-like structure, and a globular β-sandwich C-terminal domain [92]. Similarly, the PC repeats of Rpn1 have also been predicted to form a toroid-shaped structure [108]. The PC repeats of both Rpn1 and Rpn2 serve as scaffolds, potentially for the assembly of RP subunits and also to allow docking and facilitate the delivery of substrates to the Rpt ring for unfolding and translocation [92].

Both Rpn1 and Rpn2 also recognize ubiquitin-like domains (UBLs) in proteins such as Rad23, Dsk2, and Rpn13 [93,94]. Rad23 and Dsk2, which also contain ubiquitin-binding ubiquitin associated domains, are not constitutive elements of the proteasome but rather function as extrinsic ubiquitin receptors that bind ubiquitinated substrates and deliver them to the proteasome for degradation via binding to Rpn1 [93,94]. Another UBL-ubiquitin associated protein, Ddi1, has also been identified as an extrinsic ubiquitin receptor [94]. Interestingly, a recent report suggests that Rpn1 itself functions as a ubiquitin receptor [95]. A specific site on the Rpn1 toroid was found to bind both ubiquitin and UBL proteins, specifically Rad23, whereas a different site on the toroid binds to the UBL domain of the deubiquitinase (DUB) Ubp6 [95]. Thus, the predicted Rpn1 toroid acts as a multipurpose scaffold that binds not only to substrates directly via ubiquitin binding or indirectly via extrinsic ubiquitin receptors but also to the DUB Ubp6 [95,96].

In addition to binding Rpn1, Ubp6 also interacts with the Rpt1 ATPase subunit, and these interactions stabilize the substrate-committed (s2) conformation of the proteasome. This may enable the engaged substrate to be translocated into the Rpt ring for unfolding [97,98]. These interactions also place Ubp6 in close proximity to the intrinsic proteasomal DUB Rpn11, likely allowing these two DUBs to coordinate substrate deubiquitination [97,98]. The binding of ubiquitin-bound Ubp6 is also thought to prevent interference from additional substrates by ensuring that the proteasome remains in the s2 conformation until the engaged substrate is completely translocated and has its polyubiquitin chains removed [97].

The base subunit Rpn13 is a ubiquitin receptor that binds to substrates via a conserved pleckstrin-like receptor for ubiquitin (PRU) domain in its N-terminal region [99,100]. This domain consists of four antiparallel β strands packed against two antiparallel β strands [99,100]. The structure of human Rpn13 in complex with ubiquitin revealed that Rpn13 binds ubiquitin via residues on loops between the three β strands of the PRU domain [99]. The PRU domain is also responsible for binding Rpn2, whereas a distinct C-terminal domain, not present in yeast Rpn13, binds to the DUB Uch37 [100,101]. Rpn13 docks onto the proteasome by interaction with the N-terminal region of Rpn2 [101]. When human Rpn13 is not bound to the proteasome, its Uch37-binding domain and ubiquitin-binding domain (UBD) bind one another, rendering the UBD inaccessible [101]. Rpn13 binding to Rpn2 releases this interaction and activates Rpn13, allowing Rpn13 to bind ubiquitinated substrates [101].

Rpn10 was the first identified intrinsic ubiquitin receptor of the proteasome [102,103]. It bears an N-terminal von Willebrand type A factor (vWA) domain, which interacts with proteasome subunits, and a C-terminal ubiquitin-interacting motif (UIM) domain [104]. Mutation of the vWA domain promotes dissociation of the lid and base subcomplexes, suggesting a role for Rpn10 in stabilizing their interaction. Interestingly, a hydrophobic patch on the vWA domain has been recently shown also to bind ubiquitin, revealing a second UBD on Rpn10, which promotes ubiquitination of the subunit at a specific lysine, Lys84 (Fig. 4c) [104]. Monoubiquitination of Rpn10 inhibits its ability to recruit substrates for degradation, therefore decreasing proteasome activity [105]. The modification of Lys84 of the subunit was shown to promote its release from the proteasome [104]. Monoubiquitination of Rpn10 decreases under stress conditions such as heat or cold shock, suggesting that this modification is part of a physiological mechanism to regulate proteasome activity [105].

Overview of base assembly

The six eukaryotic Rpt subunits have high sequence and structural homology to one another (Fig. 4a). Ordered assembly of these subunits into the base complex is ensured by a set of dedicated assembly chaperones [11,106–110] (Fig. 5). The Nas2, Nas6, Rpn14, and Hsm3 chaperones all interact with the C-terminal helical domains of their respective Rpt subunits (Nas2-Rpt5, Nas6-Rpt3, Rpn14-Rpt6, and Hsm3-Rpt1) [11,106–109]. By contrast, the Adc17 chaperone binds to the N-terminal CC-OB domain of Rpt6 [110]. None of these assembly factors is essential for viability, but loss of specific combinations of them is lethal under proteotoxic stress conditions. The details of base assembly are still poorly understood. Assembly may follow multiple routes, which could also vary among species. We will discuss base assembly primarily from analyses done with the yeast

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

Nas6-Rpt3CTD complex
Rpn14

![Image](image1.png)

Nas2NTD-Rpt5ATPase-CTD complex
Hsm3-Rpt1CTD complex

![Image](image2.png)

Fig. 5. Crystal structures of base assembly chaperones alone or in complex with their respective Rpt partners. All chaperones and Rpt subunits depicted are from *Saccharomyces cerevisiae* with the exception of the Rpt5ATPase-CTD. The Rpt5ATPase-CTD construct consists of the ATPase domain from *Pyrococcus furiosus* PAN (an archaeal homolog of Rpt subunits) followed by the C-terminal peptide from *S. cerevisiae* Rpt5. Regions highlighted in red on Nas6, Rpn14, and Hsm3 represent a single structural repeat of the chaperones. PDB IDs: 2DZN (Nas6-Rpt3CTD), 3WHL (Nas2NTD-Rpt5ATPase-CTD), 3ACP (Rpn14), and 4A3V (Hsm3-Rpt1CTD).

model system and compare these results with base assembly in mammalian cells (Fig. 6).

One of the earliest base assembly events is thought to be the formation of Rpt dimer pairs: Rpt1–Rpt2, Rpt3–Rpt6, and Rpt4–Rpt5. In yeast, two assembly chaperones, Adc17 and Hsm3, have

been shown to facilitate Rpt dimerization. Adc17 functions to bring together Rpt3 and Rpt6 and likely dissociates early during base assembly since it has not been found in larger subcomplexes [110]. In contrast, Hsm3 provides a scaffold to assist in the formation of the Rpt1–2 heterodimer (Rpn1 is also

a)
![Image](image3.png)

b)
![Image](image4.png)

Fig. 6. Models of proteasome base assembly in (a) yeast and (b) human. The models are based primarily on the characterization of complexes isolated from cells and analyzed by mass spectrometry. The ordering of events is therefore speculative, and distinct assembly pathways remain possible. Rpt1–Rpt6 subunits are numbered 1–6 in the figure. Potential differences can be seen in the association order of Rpt dimer complexes in *Saccharomyces cerevisiae* and human, but both pathways may occur in both species.

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

present in the complex and contacts Hsm3) and only dissociates after the formation of the full RP complex but before incorporation into the 26S proteasome [107,111].

It is predicted that the Rpt subunits form subassemblies consisting of the heterodimer ATPase pairs and their respective chaperones en route to the formation of the mature base: Hsm3-Rpt1-Rpt2-Rpn1 (Hsm3 module), Nas2-Rpt5-Rpt4 (Nas2 module), and Nas6-Rpt3-Rpt6-Rpn14 (Nas6-Rpn14 module) [11,82,106,107]. The Hsm3 module and Nas2 modules have been detected in yeast, whereas the Nas6-Rpn14 module has only been detected in mammalian cells to date [11,82,106,107]. In yeast, the Nas2 and (presumptive) Nas6-Rpn14 modules are able to associate, forming an intermediate-sized complex with all components of these modules and Rpn2 and Rpn13 [62,112]. Nas2 must dissociate before or during the binding of the Hsm3 module to this intermediate, as a complex that contains both Hsm3 and Nas2 has never been observed [82]. Hsm3 has been detected in full base complexes, whereas Nas2 has not [82].

Mammalian base assembly might proceed slightly differently from that in yeast. In particular, in mammalian cells, the Hsm3 and Nas6-Rpn14 modules have been observed to form a complex prior to binding of the Nas2 module [11]. Rpn2 and Rpn13 are also thought to bind to the complex containing the Hsm3 and Nas6-Rpn14 modules in mammals. It is still unclear whether Rpn2/Rpn13 or the Nas2 module binds first to this complex [11,106,112]. For both yeast and mammalian cells, most—but not all—data suggest that Rpn10 and the lid associate only with fully formed base complexes, completing the assembly of the RP [11,82,106,107,113].

### Potential role of the CP as a template for base assembly

An interesting question that remains in the understanding of base assembly is whether the CP can serve as a “template” for the assembly of base subcomplexes. The CP-templating hypothesis was first put forth when it was discovered that base intermediates accumulate if CP assembly is compromised; this was seen in mutants lacking CP assembly chaperones or the α3 CP subunit [35]. In a subsequent study, accumulation of base intermediates, instead of free and fully assembled base or RP, was detected in yeast strains containing mutations in the C termini of Rpt subunits that specifically bind to the α-ring pockets [114]. These observations suggested that the anchoring of the Rpt tails into the CP α-ring pockets could promote the correct placement of base subunits during RP assembly. Consistent with the CP-templating hypothesis, a report showed that Rpt1–α4 and Rpt2–α4 fusions in yeast do not disrupt the formation of full 26S proteasomes [115]. While this suggests that

the base could potentially assemble on the α-ring surface of the CP, assembly without the subunit fusions might not normally favor this route. To date, a complex of the CP with a partially assembled base complex has not been reported. Such intermediates would be expected if the CP truly acts as a base assembly template. It has been suggested that intermediates of this kind might be difficult to capture due to their unstable nature and rapid association of base modules with the CP template [116].

A CP-independent base assembly pathway has been inferred primarily from the ability to reconstitute the full yeast base complex in the absence of CP in *E. coli* [89]. This functional base complex was reconstituted by co-expressing all six Rpt subunits, Rpn1, Rpn2, Rpn13, and four base assembly chaperones Hsm3, Nas2, Nas6, and Rpn14 [89]. Although full base complex can be assembled independently of CP *in vitro*, it does not necessarily rule out a CP-templating mechanism for enhancing base assembly. CP-dependent and -independent assembly pathways are not mutually exclusive. However, further experiments will be required to test the CP template model.

#### Base assembly chaperones

##### Nas2 (p27 in human)

Nas2 contains a C-terminal post synaptic density protein (PSD95), Drosophila disc large tumor suppressor (*Dlg1*), zonula occludens-1 protein (*zo-1*) domain that binds to the HbYX motif of Rpt5 [106,107,117]. This interaction is thought to prevent premature proteasome association and activation since the Rpt5 HbYX motif contributes to CP association and opening of the CP gate, as discussed earlier [85]. A crystal structure of the N-terminal domain of Nas2 in complex with the C-terminal helical domain of Rpt5 revealed a second function for the chaperone (Fig. 5). This region of Rpt5 normally binds Rpt1, its neighbor in the ATPase ring [118]. Nas2 binding would therefore block the incorporation of Rpt1 (as part of the Hsm3 module) or any incorrect modules. This might limit the assembly of non-native ATPase hexamers, while Nas2-post synaptic density protein (PSD95), Drosophila disc large tumor suppressor (*Dlg1*), zonula occludens-1 protein (*zo-1*) binding of the Rpt5 HbYX peptide would block CP association with incorrect base assemblies [118]. These results are fully consistent with earlier biochemical data, which predicted that Nas2 dissociates immediately before or during the binding of the Hsm3 module [82]. It is likely that the correct positioning of Rpt1 next to Rpt5 results in the release of Nas2.

In summary, Nas2 likely has two roles: preventing premature association of base intermediates with the CP and ensuring the proper joining of base assembly modules to form the correctly arranged base subcomplex.

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

Hsm3 (S5b in human)

Crystal structures of Hsm3 revealed a curved structure that consists of 11 HEAT repeats, where each repeat consists of 2 antiparallel α helices (Fig. 5) [111,119,120]. The central region of the concave surface of Hsm3 binds to the C-terminal helical domain of Rpt1 [111,119,120]. Mutations of hydrophobic residues at this interface are strongly detrimental to binding and cause proteasome assembly defects *in vivo* [119]. The C-terminal domain of Hsm3 also interacts with the AAA-ATPase domain of Rpt2 [111,119]. Hence, Hsm3 is thought to act as a scaffold to bridge Rpt1 and Rpt2 [111,119]. Additionally, yeast two-hybrid and *in vitro* binding analyses showed that Hsm3 weakly interacts with Rpn1, further highlighting the importance of Hsm3 in scaffolding assembly of the Hsm3-Rpt1-Rpt2-Rpn1 module [111].

Nas6 (p28/gankyrin in human)

A crystal structure of Nas6 bound to the C-terminal domain of Rpt3 revealed that Nas6 consists of 7 ankyrin repeats; each repeat is formed by ~30 residues arranged in 2 antiparallel helices followed by a β-hairpin loop (Fig. 5) [121]. Similar to the Hsm3 HEAT repeats, the Nas6 ankyrin repeats create a curved structure whose concave surface binds to the C-terminal α-helical bundle of Rpt3 via extensive van der Waals interactions and complementary charged patches [121]. Interestingly, when Nas6 is bound to Rpt3, it is predicted to clash sterically with the CP and block the interaction of the base and CP. This could prevent premature or potentially incorrect docking of the RP complex to the CP [108]. When the free base docks onto the CP, the C-terminal tail of Rpt6 appears to bind to the pocket between the CP α2 and α3 subunits, while Nas6 remains bound to Rpt3. This Rpt6 binding to the CP may then promote the release of Nas6 [122]. Specifically, Rpt6 tail-CP binding is thought to promote Rpt3 tail insertion into a CP α-ring pocket, causing a steric clash between Nas6 and the CP and Nas6 dissociation [122]. A recent report showed that Nas6 also prevents the association of the base and lid but specifically when ATP hydrolysis is inhibited, suggesting a switch in Nas6 positioning that may coordinate proper assembly of the full 26S proteasome [116]. A role for Nas6 in lid–base joining had been predicted from *in vivo* studies [107].

Rpn14 (PAAF1 in human)

A high-resolution crystal structure of Rpn14 confirmed that this protein contains a C-terminal WD40-repeat domain with a seven-bladed β-propeller fold in which each blade consists of four antiparallel β-strands (Fig. 5) [123]. Rpn14 binds with high affinity to a four-helix bundle in the C-terminal domain of Rpt6

[124]. NMR analyses of the C-terminal domain of Rpt6 revealed that it assumes two dynamic conformational states: a partially unfolded state and an ordered four-helix bundle [124]. The switch between these states is thought to allow Rpn14 binding and release from the C-terminal domain of Rpt6 [124]. Despite the available structural and biochemical data on Rpn14, its exact role in base assembly remains unclear.

Adc17

Adc17 is a recently discovered base assembly chaperone that was identified via a suppressor screen with a yeast *rpt6* temperature-sensitive mutant [110]. No sequence homolog of Adc17 in vertebrates has been identified. Notably, Adc17 is the only base proteasome assembly chaperone identified to date that interacts with the N-terminal region of an Rpt subunit [110]. The predicted role of Adc17 in Rpt3–Rpt6 heterodimer formation is supported by *in vitro* binding assays, which revealed that Adc17 binds both Rpt6 alone and in combination with Rpt3, in the latter case forming a ternary protein complex, but it cannot interact with Rpt3 alone [110]. Additionally, mutation of a conserved residue in a predicted Adc17 CC abolished its ability to bind Rpt6 [110]. These results suggest that Adc17 first binds Rpt6 and subsequently assists the dimerization of Rpt6 with Rpt3 [110], potentially facilitating the formation of the CC between the ATPase N-terminal helices. Interestingly, Adc17 does not appear to interact with the full Nas6-Rpt3-Rpt6-Rpn14 module; therefore, it probably dissociates shortly after Rpt3–Rpt6 dimerization [110].

The RP Lid

Lid composition

The lid has nine subunits: Rpn3, 5, 6, 7, 8, 9, 11, 12, and 15 (Sem1/DSS1; Fig. 7) and shows significant sequence and structural similarity to both the COP9 signalosome and eIF3 [9,125–127]. Each of these complexes contains multiple subunits bearing a proteasome, COP9 signalosome, e/F3 (PCI) fold. In the 26S proteasome, the DUB activity of the lid Rpn11 subunit cleaves ubiquitin chains from proteasomal substrates. Similarly, the COP9 signalosome removes the UBL Nedd8 from substrate proteins, principally cullins. The eIF3 complex is involved in the initiation of protein translation (a UBL-cleaving activity has not been found). These complexes can even share subunits. In yeast, Rpn5 is both a proteasome lid subunit and a component of the COP9 signalosome [128,129].

Two of the lid proteins, Rpn11 and Rpn8, adopt an Mpr1/Pad1 N-terminal (MPN) fold. Rpn11 is a JAMM metalloprotease; Rpn8 is inactive but

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

Rpn8/11  
heterodimer  

+  

LP3 Module 1 LP2 Lid  

Fig. 7. Hierarchical assembly pathway model for the RP lid. In the model, the lid assembles along a two-pronged pathway in which the subcomplexes LP3 and Module 1 are formed first and then associate to form LP2. The final step in assembly occurs when Rpn12 joins LP2 to form the mature lid. The full lid, but not LP2, is able to bind the free base. PCI subunits (gray) are labeled with the yeast Rpn subunit numbers.

heterodimerizes with Rpn11, yielding a complex with modest DUB activity [130,131]. Sem1 (Rpn15) is a small intrinsically disordered protein that is also part of several other nonproteasomal protein complexes [132]. In the proteasome lid, Sem1 binds both Rpn3 and Rpn7 [133–136]. The remaining six lid subunits have the PCI fold, which consists of an extended helical domain followed by a conserved winged-helix motif and C-terminal α helix [137]. The winged-helix domains associate into a horseshoe shape with the N-terminal helical domains extending like fingers from the horseshoe. The C-terminal helices of the PCI subunits and the two MPN subunits associate into a complex helical bundle [130].

Lid assembly

There have been significant advances in recent years toward a better understanding of lid assembly. Importantly, both the lid [88,133,138,139] and the entire RP [140] can be reconstituted by recombinant yeast protein expression in *E. coli*. Additionally, the majority of lid subunits can be tagged with green fluorescent protein or other epitopes and show no apparent phenotypic defects [139], although some subunits are sensitive to tag location [140]. These tools have greatly expanded our ability to study the assembly and structure of the complex.

Lid assembly appears to proceed through an ordered series of subcomplexes that ultimately yield the mature lid (Fig. 7) [138]. Lid formation does not depend on the presence of a wild-type RP base or CP. Many of the early assembly steps are still undefined, but the formation of the C-terminal helical bundle from the MPN and PCI lid subunits seems to dictate the order of later steps in assembly [138,141]. Rpn8 and Rpn11, as noted, can form a heterodimer. Rpn5, Rpn6, and Rpn9 join the dimer to form “Module 1”. In a second branch of the assembly pathway, Sem1 brings Rpn3 and Rpn7 together to form lid particle 3 (LP3). Module 1 then joins with LP3 to form LP2, which contains all lid subunits except Rpn12 [138,141,142]. Rpn12 comes in at the end of the pathway, inserting its C-terminal helix into an extended groove formed by the nearly completed helical bundle. Incorporation of the terminal helix of Rpn12 into the bundle triggers extensive conformational changes that facilitate the association of the lid with the base to complete RP formation; LP2, in contrast, has little apparent affinity for the base [140].

There is evidence, however, that certain lid subassemblies can associate with base-CP complexes. In yeast strains expressing a mutant Rpn11 subunit with a truncated C terminus, Module 1 accumulates, and incomplete proteasomes can be isolated with this subcomplex bound to base-CP complexes [142]. Addition of the missing C-terminal Rpn11 fragment, either in cells or to extracts, allows the restoration of full proteasome formation. The extent to which wild-type proteasomes follow such a pathway remains to be determined.

Structure and conformational changes in the lid

Many of the advances in our understanding of the lid and its assembly have been aided by recent higher

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

Fig. 8. Structure of the recombinant RP lid. (a) The high-resolution cryo-EM structure of free lid (PDB ID 3JCK). As in Fig. 7, Rpn8 is shown in purple, Rpn11 in pink, and Sem1 in gold. The six PCI proteins are shown in cyan. (b) The Rpn8–Rpn11 heterodimer (PDB ID 4O8X). Rpn11 (light gray), which harbors the DUB activity of the lid, is shown in the context of the Rpn8–11 heterodimer. Active-site residues are shown in light pink, and the Zn²⁺ in yellow. The zinc atom is not depicted to scale for clarity. The Rpn11 active site is easily accessible. (c) The active site of Rpn11 is depicted in the context of the full lid. The inset shows the interactions among Rpn11 (pink), Rpn8 (purple), and Rpn5 (cyan), blocking the Rpn11 active site. Again, the zinc atom is not depicted to scale for clarity.

resolution structural studies. Cryo-EM reconstructions of the recombinant lid and the full 26S proteasome reveal large-scale and local conformational shifts during assembly and substrate processing [125]. Substantial conformational changes have also been documented by a combination of negative-stain EM and chemical crosslinking mass spectrometry [140]. In addition, structures of subunits or subcomplexes of the lid have also been derived using X-ray crystallography [3]. Here, we discuss aspects of lid assembly and function in the context of these new structural data.

[138,141]. The globular cores of Rpn8 and Rpn11 interact through two regions of primarily α-helical structure [130,131]. One of these interfaces consists of a four-helix bundle, in which one helix from each subunit extends to interact with the MPN domain of the other protein, anchoring the heterodimer [130,131]. Later in lid assembly, there is a rigid-body rotation of the Rpn8/Rpn11 heterodimer during the transition from LP2 to lid; this is followed by additional large conformational changes when the lid is incorporated into the full proteasome [125,140].

### Structural insight into activation of the Rpn11 DUB

Rpn11 is at a position in the RP where it can recognize and deubiquitinate polyubiquitinated substrates as they are channeled into the central pore of the ATPase ring [83]. An insertion sequence in Rpn11, Ins-2, helps ensure the proper positioning of Rpn11 over the ATPase channel via interactions with the base subunit Rpn2 [131]. In other DUBs, the Ins-2 segment is important for maintaining enzyme structural integrity [143]. High-resolution structural

The Rpn8/Rpn11 heterodimer

The Rpn8/Rpn11 heterodimer, which bears the major DUB activity of the proteasome, is thought to be an important early intermediate in the assembly of the lid [141]. The C-terminal helices of Rpn8 are of particular importance due to their central location in the lid helical bundle; without them, lid assembly is severely impaired both *in vivo* and when the recombinant lid subunits are co-expressed in *E. coli*

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

analyses have clarified additional details regarding the mechanism of Rpn11 action and its inhibition when not incorporated into proteasomes [125, 131]. The active-site zinc is tetrahedrally coordinated by His109, His111, Asp122, and a water molecule; the latter interacts with nearby Asn275 of Rpn5 [125, 130] (Fig. 8b and c). The Rpn5-Asn275 side chain stabilizes the tetrahedral coordination of the Zn²⁺ ion, inhibiting Rpn11 catalytic activity. Additional interactions between the insertion motif Ins-1 of Rpn11 and Rpn5 and between Rpn8 and Rpn9 further stabilize the inhibited state in the isolated lid. When the lid incorporates into the proteasome, the Rpn8–Rpn11 heterodimer rotates ~90° away from Rpn5, relieving these inhibitory effects and steric interference caused by Rpn5 helix-13 [125, 131].

These results are in good accord with available biochemical data: free lid exhibits fivefold less DUB activity than the purified Rpn8/Rpn11 heterodimer, and complexes with mutations in regions of Rpn5 and Rpn8 that either limit access to or prevent the proper formation of the Rpn11 active site exhibit increased DUB activity in free lid particles. The majority of the data suggest that Rpn11 DUB activity decreases as the lid proceeds along its assembly pathway. One notable exception seems to be when assembly halts at Module 1 in certain lid assembly mutants; here, Module 1 shows increased DUB activity in comparison to free Rpn11 [142]. As noted above, this complex has also been reported to interact with the base, unlike other lid assembly precursors [142]; whether DUB activity in this context is relevant *in vivo* is uncertain. In addition to other inhibitory mechanisms, helix-13 of Rpn5 has been shown to interact with the catalytic groove of Rpn11, contributing to the steric inhibition of Rpn11 DUB activity in the free lid (Fig. 8c) [125].

The structural data also suggest an explanation for why Rpn11 can cleave polyubiquitin chains linked to virtually any substrate: the DUB interacts with ubiquitinated substrates only through contacts to the ubiquitin molecule that is distal to the cleaved isopeptide bond. Comparisons of Rpn11 active-site accessibility in the proteasome between different conformational states also revealed that the active site is occluded in the resting state of the proteasome but moves to a position over the ATPase outer-ring channel upon proteasome engagement with substrate [4, 91, 144]. This allows the placement of the bond between the substrate and ubiquitin chain into the unobstructed Rpn11 active site, as the substrate is threading through the ATPase pore, leading to the amputation of the ubiquitin chain and continued substrate translocation.

Sem1 (DSS1/Rpn15)

Sem1 functions as a molecular tether that facilitates the formation of the lid assembly intermediate known as LP3 [133, 145]. Sem1, together with Rpn3 and Rpn7, comprises LP3. Two short segments in Sem1 connected by a disordered linker hold the other two subunits together. Later in assembly when LP3 and Module 1 have joined, this tethering function of Sem1 is no longer required [133], although the short polypeptide has additional post-assembly functions in the proteasome as well [146]. Because most of Sem1 is intrinsically disordered, only a few short segments have been visualized in any of the 26S proteasome cryo-EM structures. Interestingly, Sem1 has also been reported to bind to ubiquitin [134], but whether it has this function in the context of the proteasome is unclear because the ubiquitin and RP-subunit-binding sites in Sem1 overlap.

PCI subunits and the role of Rpn12 in RP assembly

In the lid, the horseshoe-like structure formed by the winged-helix domains of the six PCI proteins cradles the MPN core of Rpn8/Rpn11 (Figs. 7 and 8a) [4, 88, 91, 133, 144]. The lid undergoes substantial conformational changes upon formation of the 26S proteasome, where it is splayed onto the side of the Rpt3/Rpt6 base sector and makes contact with the CP as well [125]. The lid also rotates and translates its position on the proteasome during the proteolytic cycle [83]. These movements may coordinate the displacement of substrates from their original binding sites in the RP to positions allowing their unfolding by the base ATPases and deubiquitination by Rpn11.

In what is likely the most common pathway of lid assembly, subunits assemble first into the eight-subunit LP2 intermediate and then bind the remaining PCI subunit, Rpn12. This completes lid formation and establishes competence for base binding [138, 140]. LP2 is unable to bind to the base. Large-scale and more local conformational changes are triggered by LP2 binding to Rpn12 [140]. The key to these changes is the integration of the C-terminal helix of Rpn12 into the nascent lid helical bundle. This is thought to create an “open” state of the lid that can associate stably with the base. These conformational changes are necessary to relieve steric clashes that prevent LP2 from interacting with the base and forming the RP [125, 140]. When the lid integrates into the RP and 26S proteasome, it undergoes significant conformational changes again. The entire PCI horseshoe compresses, and Rpn3, Rpn7, and Rpn12 rotate toward the base [125].

Regulation of Proteasome Expression

Proteasome homeostasis is essential to ensuring normal cell growth and viability. Specific cell signaling pathways are responsible for regulating the expression of proteasome subunits and assembly chaperones. The best-characterized homeostatic mechanism

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

described to date centers on yeast Rpn4, a zinc-finger transcription factor, which is part of a proteasome-dependent negative feedback loop [147, 148]. The first indication that such a mechanism might exist came from studies of CP mutants with severely impaired activity; massive accumulation of proteasome subunits was observed [15]. Notably, the proteasome degrades Rpn4 with a half-life of ~2 min [147]. Thus, cellular Rpn4 levels are inversely related to proteasome activity.

The transcriptional targets of Rpn4 are genes with a proteasome-associated control element (PACE) in their promoters; this 9-nt sequence or slightly different variants are found upstream of almost all proteasomal subunit genes [149]. When proteasome activity is compromised, such as during proteotoxic stress, Rpn4 becomes more stable and accumulates [148]. This increase in Rpn4 levels leads in turn to increased rates of proteasomal gene transcription and formation of more proteasomes [147–149]. Restoration of proteolytic capacity then leads to enhanced Rpn4 degradation, returning proteasome gene transcription to its basal rate.

PACE sequences are not found in the promoter regions of most yeast proteasome assembly chaperones, although several have PACE-like sequences [150]. It was speculated that some assembly chaperones need not be expressed at high levels, particularly as they are utilized repeatedly [150]. Alternatively, it is possible that a different pathway regulates the levels of assembly chaperones. This turns out to be the case, as it was recently discovered that the evolutionarily conserved TORC1 pathway regulates levels of the RP assembly chaperones (RACs) in both human and yeast [151].

TORC1 is a highly conserved protein kinase involved in many signaling pathways and is a major regulator of cell growth [152]. A mitogen-activated protein kinase Mpk1, found downstream in one of the TORC1 pathways, induces the expression of RACs under specific stress conditions [151]. TORC1 is an inhibitor of Mpk1 [151]. Upon stress, TORC1 is inhibited, allowing Mpk1 to be activated and subsequently increasing the expression of RACs [151]. The exact mechanism of RAC up-regulation remains to be determined, but interestingly, the data are most consistent with translational control. The TORC1 pathway does not control the levels of all proteasome assembly chaperones [151]. Levels of the CP assembly chaperones Pba1 and Pba2 were not altered by MPK1 deletion, indicating that they are likely regulated in a different manner [151], possibly via the Rpn4 feedback loop [150]. Further work is required to clarify the links between Rpn4 and TORC1 regulation of proteasome assembly.

In humans, the functional equivalents of Rpn4 are the Nrf1 [153–155] and Nrf2 [156–159] transcription factors. Nrf1 regulates basal levels of proteasome subunits and mediates proteasome gene up-

regulation in response to short-term, low-dose proteasome inhibition, whereas Nrf2 induces proteasome gene transcription during oxidative stress, but not during proteasome inhibition. Nrf1 and Nrf2 both contain a unique leucine-zipper domain and are members of the Cap'n'Collar transcription factor family. Like Rpn4, both Nrf1 and Nrf2 are rapidly degraded by the proteasome with half-lives of 12–13 min [154, 160]. Under normal conditions, Nrf1 is associated with the endoplasmic reticulum [161], whereas Nrf2 localizes in the cytoplasm and at mitochondria [162].

When proteasomes are inhibited, the N-terminal transmembrane region of Nrf1 is proteolytically processed, and active Nrf1 is released from the endoplasmic reticulum and translocated into the nucleus to regulate gene expression [155, 163]. Similarly, in response to oxidative stress, Nrf2 translocates from the cytoplasm to the nucleus where it activates its target genes [159]. Both Nrf1 and Nrf2 recognize the antioxidant response element in the regulatory regions of their target genes [164]. It is speculated that Nrf1 and Nrf2 up-regulate proteasome genes through recognition of upstream antioxidant response element sequences. Ubiquitin-specific protease 15 has been suggested to activate Nrf1 function in the nucleus by stabilizing Nrf1 through its deubiquitination [165]. Furthermore, activation of mTORC1 signaling stimulates Nrf1 activity, leading to increased levels of proteasomes [166]. However, the connection between Nrf1/Nrf2 and TORC1-mediated regulation of proteasome assembly chaperones remains unknown.

### Perspectives

Over the past few years, significant progress has been made toward understanding the assembly mechanisms of the proteasome. Despite this, there are still many questions that remain. Regarding the CP, we still do not know how each α-subunit incorporates at the correct position and whether there is an α-ring-independent assembly pathway in eukaryotes. We also do not know yet the physiological roles of the alternative proteasomes containing α-subunit substitutions.

It remains possible that there are more assembly chaperones left to discover for both the CP and the RP base and potentially even the lid. In the future, it will be important to document proteasomal conformational changes upon binding and hydrolysis of ATP by specific Rpt subunits not only to link these changes with substrate unfolding, deubiquitination, and degradation (Fig. 9) but also to determine how the ATPase cycle might be linked to proteasome assembly. The differences between base assembly models for mammalian and yeast proteasomes remain to be further explored, and the question

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

![image](https://i.imgur.com/1234567.png)

Fig. 9. Model of substrate degradation by the proteasome based on cryo-EM structures of the human proteasome [167]. Four conformations of the proteasome, named $S_{A}-S_{D}$, are depicted. Figures were made based on PDB entries 5T0G, 5T0H, 5T0I, and 5T0J using PyMol to show the cutaway surface of each state.

of whether the CP can function as a template for  
RP base ATPase arrangement *in vivo* is still  
unanswered. With respect to the lid, the structures  
of most of the known assembly intermediates are not  
known. In the case of LP2, high-resolution structures  
may pinpoint the exact reasons that it cannot bind  
the base and may give hints to how the binding of  
the Rpn12 C-terminal peptide triggers the conformational changes that accompany the completion of  
lid assembly and enable base association. Given the  
tools now available, we can expect rapid advances  
in addressing these issues.

**Abbreviations used:**  
CP, core particle; RP, regulatory particle; EM, electron  
microscopy; HEAT, Huntingtin, elongation factor 3 (EF3),  
protein phosphatase 2A (PP2A), TOR1 repeat; OB,  
oligonucleotide/oligosaccharide-binding; CC, coiled coil;  
PC, proteasome/cyclosome; UBL, ubiquitin-like domain; DUB, deubiquitinase; PRU,  
pleckstrin-like receptor for ubiquitin; UBD, ubiquitin-binding domain; vWA, von Willebrand type A factor; PCI,  
proteasome, COP9 signalosome, e/F3; MPN, Mpr1/Pad1  
N-terminal; LP3, lid particle 3; PACE, proteasome-associated control element; RAC, RP assembly chaperone.

---

**References**

[1] Y. Saeki, K. Tanaka, Assembly and function of the  
proteasome Ubiquitin Family Modifiers and the Proteasome,  
in: R. Jürgen Dohmen, Martin Scheffner (Eds.), Edition: 1  
Chapter: 22, 832, Humana Press, 2012.  

[2] M. Hochstrasser, Ubiquitin-dependent protein degradation,  
Annu. Rev. Genet. 30 (1996) 405–439, <http://dx.doi.org/10.1146/annurev.genet.30.1.405>.  

[3] R.J. Tomko Jr., M. Hochstrasser, Molecular architecture  
and assembly of the eukaryotic proteasome, Annu. Rev.  
Biochem. 82 (2013) 415–445, <http://dx.doi.org/10.1146/annurev-biochem-060410-150257>.  

[4] M. Wehmer, E. Sakata, Recent advances in the structural  
biology of the 26S proteasome, Int. J. Biochem. Cell Biol. 79  
(2016) 437–442, <http://dx.doi.org/10.1016/j.biocel.2016.08.008>.  

[5] Y. Lu, B.-H. Lee, R.W. King, D. Finley, M.W. Kirschner,  
Substrate degradation by the proteasome: a single-molecule kinetic analysis, Science 348 (2015) 1,250,834,  
<http://dx.doi.org/10.1126/science.1,250,834>.  

[6] V. Vittal, M.D. Stewart, P.S. Brzovic, R.E. Klevit, Regulating  
the regulators: recent revelations in the control of E3  
ubiquitin ligases, J. Biol. Chem. 290 (2015) 21,244–21,251,  
<http://dx.doi.org/10.1074/jbc.R115.675165>.

---

**Acknowledgments**

We thank Judy Ronau for her critical reading of the  
manuscript and many helpful suggestions for the  
figures. Work on the proteasome in our group has  
been supported by NIH grants GM046904 and  
GM083050 to M.H. L.B. was funded in part by an  
award from the NSF Graduate Research Fellowships  
Program.

Received 14 March 2017;  
Received in revised form 22 May 2017;  
Accepted 30 May 2017  
Available online xxxx

**Keywords:**  
proteasome;  
ubiquitin proteasome system;  
proteasome assembly;  
protein degradation  

These authors contributed equally to this work.

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), <http://dx.doi.org/10.1016/j.jmb.2017.05.027>

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

[7] Y. Saeki, Ubiquitin recognition by the proteasome, J. Biochem. mvw091, 161 (2017) 113–124, http://dx.doi.org/10.1093/jb/mvw091.

[8] M.J. Kunjappu, M. Hochstrasser, Assembly of the 20S proteasome, Biochim. Biophys. Acta 1843 (2014) 2–12, http://dx.doi.org/10.1016/j.bbamcr.2013.03.008.

[9] M.H. Glickman, D.M. Rubin, V.A. Fried, D. Finley, The regulatory particle of the *Saccharomyces cerevisiae* proteasome, Mol. Cell. Biol. 18 (1998) 3149–3162.

[10] E. Isono, K. Nishihara, Y. Saeki, H. Yashiroda, N. Kamata, L. Ge, et al., The assembly pathway of the 19S regulatory particle of the yeast 26S proteasome, Mol. Biol. Cell 18 (2007) 569–580, http://dx.doi.org/10.1091/mbc.E06-07-0635.

[11] T. Kaneko, J. Hamazaki, S.-I. Iemura, K. Sasaki, K. Furuyama, T. Natsume, et al., Assembly pathway of the mammalian proteasome base subcomplex is mediated by multiple specific chaperones, Cell 137 (2009) 914–925, http://dx.doi.org/10.1016/j.cell.2009.05.008.

[12] M. Groll, M. Bajorek, A. Köhler, L. Moroder, D.M. Rubin, R. Huber, et al., A gated channel into the proteasome core particle, Nat. Struct. Mol. Biol. 7 (2000) 1062–1067, http://dx.doi.org/10.1038/80992.

[13] P.A. Osmulski, M. Hochstrasser, M. Gaczynska, A tetrahedral transition state at the active sites of the 20S proteasome is coupled to opening of the alpha-ring channel, Structure 17 (2009) 1137–1147, http://dx.doi.org/10.1016/j.str.2009.06.011.

[14] J. Lowe, D. Stock, B. Jap, W. Baumeister, R. Huber, Crystal structure of the 20S proteasome from the archaeon *T. acidophilum* at 3.4 Å resolution, Science 268 (1995) 533–539.

[15] P. Chen, M. Hochstrasser, Autocatalytic subunit processing couples active site formation in the 20S proteasome to completion of assembly, Cell 86 (1996) 961–972.

[16] E.M. Huber, W. Heinemeyer, X. Li, C.S. Arendt, M. Hochstrasser, M. Groll, A unified mechanism for proteolysis and autocatalytic activation in the 20S proteasome, Nat. Commun. 7 (2016) 10,900, http://dx.doi.org/10.1038/ncomms10900.

[17] S.H. Becker, K.H. Darwin, Bacterial proteasomes: mechanistic and functional insights, Microbiol. Mol. Biol. Rev. 81 (2017) e00036-16, http://dx.doi.org/10.1128/MMBR.00036-16.

[18] G. Lehmann, R.G. Udasin, I. Livneh, A. Ciechanover, Identification of UBact, a ubiquitin-like protein, along with other homologous components of a conjugation system and the proteasome in different gram-negative bacteria, Biochem. Biophys. Res. Commun. 483 (2017) 946–950, http://dx.doi.org/10.1016/j.bbrc.2017.01.037.

[19] D. Li, H. Li, T. Wang, H. Pan, G. Lin, H. Li, Structural basis for the assembly and gate closure mechanisms of the *Mycobacterium tuberculosis* 20S proteasome, EMBO J. 29 (2010) 2037–2047, http://dx.doi.org/10.1038/emboj.2010.95.

[20] G. Lin, G. Hu, C. Tsu, Y.Z. Kunes, H. Li, L. Dick, et al., *Mycobacterium tuberculosis* prcBA genes encode a gated proteasome with broad oligopeptide specificity, Mol. Microbiol. 59 (2006) 1405–1416, http://dx.doi.org/10.1111/j.1365-2958.2005.05035.x.

[21] F. Zühl, E. Seemüller, R. Golbik, W. Baumeister, Dissecting the assembly pathway of the 20S proteasome, FEBS Lett. 418 (1997) 189–194, http://dx.doi.org/10.1016/S0014-5793(97)01370-7.

[22] Y.D. Kwon, I. Nagy, P.D. Adams, W. Baumeister, B.K. Jap, Crystal structures of the *Rhodococcus* proteasome with and

without its pro-peptides: implications for the role of the pro-peptide in proteasome assembly, J. Mol. Biol. 335 (2004) 233–245.

[23] P. Zwickl, J. Kleinz, W. Baumeister, Critical elements in proteasome assembly, Nat. Struct. Mol. Biol. 1 (1994) 765–770, http://dx.doi.org/10.1038/nsb1194-765.

[24] C.S. Arendt, M. Hochstrasser, Eukaryotic 20S proteasome catalytic subunit propeptides prevent active site inactivation by N-terminal acetylation and promote particle assembly, EMBO J. 18 (1999) 3575–3585, http://dx.doi.org/10.1093/emboj/18.13.3575.

[25] A.R. Kusmierczyk, M.J. Kunjappu, R.Y. Kim, M. Hochstrasser, A conserved 20S proteasome assembly factor requires a C-terminal HbYX motif for proteasomal precursor binding, Nat. Struct. Mol. Biol. 18 (2011) 622–629, http://dx.doi.org/10.1038/nsmb.2027.

[26] D. Panfair, A. Ramamurthy, A.R. Kusmierczyk, Alpha-ring independent assembly of the 20S proteasome, Sci. Rep. 5 (2015) 13,130, http://dx.doi.org/10.1038/srep13130.

[27] Y. Yao, C.R. Toth, L. HUANG, M.L. Wong, P. Dias, A.L. Burlingame, et al., alpha5 subunit in *Trypanosoma brucei* proteasome can self-assemble to form a cylinder of four stacked heptamer rings, Biochem. J. 344 (1999) 349–358, http://dx.doi.org/10.1042/bj3440349.

[28] W.L. Gerards, J. Enzlin, M. Häner, I.L. Hendriks, U. Aebi, H. Bloemendal, et al., The human alpha-type proteasomal subunit HsC8 forms a double ringlike structure, but does not assemble into proteasome-like particles with the beta-type subunits HsDelta or HsBPROS26, J. Biol. Chem. 272 (1997) 10,080–10,086, http://dx.doi.org/10.1074/jbc.272.15.10080.

[29] K. Ishii, M. Noda, H. Yagi, R. Thammaporn, S. Seetaha, T. Satoh, et al., Disassembly of the self-assembled, double-ring structure of proteasome α7 homo-tetradecamer by α6, Sci. Rep. 5 (2015) 18,167, http://dx.doi.org/10.1038/srep18167.

[30] L.J. Hammack, A.R. Kusmierczyk, Assembly of proteasome subunits into non-canonical complexes *in vivo*, Biochem. Biophys. Res. Commun. 482 (2016) 164–169, http://dx.doi.org/10.1016/j.bbrc.2016.11.024.

[31] A. Padmanabhan, S.A.-T. Vuong, M. Hochstrasser, Assembly of an evolutionarily conserved alternative proteasome isoform in human cells, Cell Rep. 14 (2016) 2962–2974, http://dx.doi.org/10.1016/j.celrep.2016.02.068.

[32] Y. Hirano, K.B. Hendil, H. Yashiroda, S.-I. Iemura, R. Nagane, Y. Hioki, et al., A heterodimeric complex that promotes the assembly of mammalian 20S proteasomes, Nature 437 (2005) 1381–1385, http://dx.doi.org/10.1038/nature04106.

[33] X. Li, A.R. Kusmierczyk, P. Wong, A. Emili, M. Hochstrasser, Beta-subunit appendages promote 20S proteasome assembly by overcoming an Ump1-dependent checkpoint, EMBO J. 26 (2007) 2339–2349, http://dx.doi.org/10.1038/sj.emboj.7601681.

[34] B. Le Tallec, M.-B. Barrault, R. Courbeyrette, R. Guérois, M.-C. Marsolier-Kergoat, A. Peyroche, 20S proteasome assembly is orchestrated by two distinct pairs of chaperones in yeast and in mammals, Mol. Cell 27 (2007) 660–674, http://dx.doi.org/10.1016/j.molcel.2007.06.025.

[35] A.R. Kusmierczyk, M.J. Kunjappu, M. Funakoshi, M. Hochstrasser, A multimeric assembly factor controls the formation of alternative 20S proteasomes, Nat. Struct. Mol. Biol. 15 (2008) 237–244, http://dx.doi.org/10.1038/nsmb.1389.

[36] H. Yashiroda, T. Mizushima, K. Okamoto, T. Kameyama, H. Hayashi, T. Kishimoto, et al., Crystal structure of a chaperone complex that contributes to the assembly of

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

yeast 20S proteasomes, Nat. Struct. Mol. Biol. 15 (2008) 228–236, http://dx.doi.org/10.1038/nsmb.1386.

[37] K. Takagi, Y. Saeki, H. Yashiroda, H. Yagi, A. Kaiho, S. Murata, et al., Pba3–Pba4 heterodimer acts as a molecular matchmaker in proteasome α-ring formation, Biochem. Biophys. Res. Commun. 450 (2014) 1110–1114, http://dx.doi.org/10.1016/j.bbrc.2014.06.119.

[38] M. Kock, M.M. Nunes, M. Hemann, S. Kube, R. Jürgen Dohmen, F. Herzog, et al., Proteasome assembly from 15S precursors involves major conformational changes and recycling of the Pba1–Pba2 chaperone, Nat. Commun. 6 (2015) 6123, http://dx.doi.org/10.1038/ncomms7123.

[39] B.M. Stadtmueller, E. Kish-Trier, K. Ferrell, C.N. Petersen, H. Robinson, D.G. Myszka, et al., Structure of a proteasome Pba1–Pba2 complex: implications for proteasome assembly, activation, and biological function, J. Biol. Chem. 287 (2012) 37,371–37,382, http://dx.doi.org/10.1074/jbc.M112.367003.

[40] P.S. Wani, M.A. Rowland, A. Ondracek, E.J. Deeds, J. Roelofs, Maturation of the proteasome core particle induces an affinity switch that controls regulatory particle association, Nat. Commun. 6 (2015) 6384, http://dx.doi.org/10.1038/ncomms7384.

[41] K. Kumoi, T. Satoh, K. Murata, T. Hiromoto, T. Mizushima, Y. Kamiya, et al., An archaeal homolog of proteasome assembly factor functions as a proteasome activator, PLoS One 8 (2013) e60294, http://dx.doi.org/10.1371/journal.pone.0060294.

[42] A. Sikdar, T. Satoh, M. Kawasaki, K. Kato, Crystal structure of archaeal homolog of proteasome-assembly chaperone PbaA, Biochem. Biophys. Res. Commun. 453 (2014) 493–497.

[43] Y. Hirano, H. Hayashi, S.-I. Iemura, K.B. Hendil, S.-I. Niwa, T. Kishimoto, et al., Cooperation of multiple chaperones required for the assembly of mammalian 20S proteasomes, Mol. Cell 24 (2006) 977–984, http://dx.doi.org/10.1016/j.molcel.2006.11.015.

[44] I. Velichutina, P.L. Connerly, C.S. Arendt, X. Li, M. Hochstrasser, Plasticity in eucaryotic 20S proteasome ring assembly revealed by a subunit deletion in yeast, EMBO J. 23 (2004) 500–510, http://dx.doi.org/10.1038/sj.emboj.7600059.

[45] Y. Hirano, T. Kaneko, K. Okamoto, M. Bai, H. Yashiroda, K. Furuyama, et al., Dissecting β-ring assembly pathway of the mammalian 20S proteasome, EMBO J. 27 (2008) 2204–2213, http://dx.doi.org/10.1038/emboj.2008.148.

[46] P.C. Ramos, A.J. Marques, M.K. London, R.J. Dohmen, Role of C-terminal extensions of subunits beta2 and beta7 in assembly and activity of eukaryotic proteasomes, J. Biol. Chem. 279 (2004) 14,323–14,330, http://dx.doi.org/10.1074/jbc.M308757200.

[47] X. Li, Y. Li, C.S. Arendt, M. Hochstrasser, Distinct elements in the proteasomal β5 subunit propeptide required for autocatalytic processing and proteasome assembly, J. Biol. Chem. 291 (2016) 1991–2003, http://dx.doi.org/10.1074/jbc.M115.677047.

[48] E. Witt, D. Zantopf, M. Schmidt, R. Kraft, P.M. Kloetzel, E. Krüger, Characterisation of the newly identified human Ump1 homologue POMP and analysis of LMP7(beta 5i) incorporation into 20S proteasomes, J. Mol. Biol. 301 (2000) 1–9, http://dx.doi.org/10.1006/jmbi.2000.3959.

[49] S. Jäger, M. Groll, R. Huber, D.H. Wolf, W. Heinemeyer, Proteasome beta-type subunits: unequal roles of propeptides in core particle maturation and a hierarchy of active

site function, J. Mol. Biol. 291 (1999) 997–1013, http://dx.doi.org/10.1006/jmbi.1999.2995.

[50] A.J. Marques, C. Glanemann, P.C. Ramos, R.J. Dohmen, The C-terminal extension of the beta7 subunit and activator complexes stabilize nascent 20S proteasomes and promote their maturation, J. Biol. Chem. 282 (2007) 34,869–34,876, http://dx.doi.org/10.1074/jbc.M705836200.

[51] P.C. Ramos, J. Höckendorff, E.S. Johnson, A. Varshavsky, R.J. Dohmen, Ump1p is required for proper maturation of the 20S proteasome and becomes its substrate upon completion of the assembly, Cell 92 (1998) 489–499.

[52] B. Sá-Moura, A.M. Simões, J. Fraga, H. Fernandes, I.A. Abreu, H.M. Botelho, et al., Biochemical and biophysical characterization of recombinant yeast proteasome maturation factor ump1, Comput. Struct. Biotechnol. J. 7 (2013) e201304006–e201304009, http://dx.doi.org/10.5936/csbj.201304006.

[53] Y. Uekusa, K. Okawa, M. Yagi-Utsumi, O. Serve, Y. Nakagawa, T. Mizushima, et al., Backbone ¹H, ¹³C and ¹⁵N assignments of yeast Ump1, an intrinsically disordered protein that functions as a proteasome assembly chaperone, Biomol. NMR Assign. 8 (2014) 383–386, http://dx.doi.org/10.1007/s12104-013-9523-1.

[54] M. Bai, X. Zhao, K. Sahara, Y. Ohte, Y. Hirano, T. Kaneko, et al., Assembly mechanisms of specialized core particles of the proteasome, Biomol. Ther. 4 (2014) 662–677, http://dx.doi.org/10.3390/biom4030662.

[55] M. De, K. Jayarapu, L. Elenich, J.J. Monaco, R.A. Colbert, T.A. Griffin, Beta 2 subunit propeptides influence cooperative proteasome assembly, J. Biol. Chem. 278 (2003) 6153–6159, http://dx.doi.org/10.1074/jbc.M209292200.

[56] D.J. Kingsbury, T.A. Griffin, R.A. Colbert, Novel propeptide function in 20S proteasome assembly influences beta subunit composition, J. Biol. Chem. 275 (2000) 24,156–24,162, http://dx.doi.org/10.1074/jbc.M001742200.

[57] I.E. Kammerl, S. Meiners, Proteasome function shapes innate and adaptive immune responses, Am. J. Phys. Lung Cell. Mol. Phys. 311 (2016) L328–L336, http://dx.doi.org/10.1152/ajplung.00156.2016.

[58] S. Murata, Y. Takahama, K. Tanaka, Thymoproteasome: probable role in generating positively selecting peptides, Curr. Opin. Immunol. 20 (2008) 192–196, http://dx.doi.org/10.1016/j.coi.2008.03.002.

[59] M.-X. Qian, Y. Pang, C.H. Liu, K. Haratake, B.-Y. Du, D.-Y. Ji, et al., Acetylation-mediated proteasomal degradation of core histones during DNA repair and spermatogenesis, Cell 153 (2013) 1012–1024, http://dx.doi.org/10.1016/j.cell.2013.04.032.

[60] H. Uechi, J. Hamazaki, S. Murata, Characterization of the testis-specific proteasome subunit α4s in mammals, J. Biol. Chem. 289 (2014) 12,365–12,374, http://dx.doi.org/10.1074/jbc.M114.558866.

[61] L. Zhong, J.M. Belote, The testis-specific proteasome subunit Prosalpha6T of *D. melanogaster* is required for individualization and nuclear maturation during spermatogenesis, Development 134 (2007) 3517–3525, http://dx.doi.org/10.1242/dev.004770.

[62] J.M. Belote, M. Miller, K.A. Smyth, Evolutionary conservation of a testes-specific proteasome subunit gene in *Drosophila*, Gene 215 (1998) 93–100.

[63] J. Kikuchi, Y. Iwafune, T. Akiyama, A. Okayama, H. Nakamura, N. Arakawa, et al., Co- and post-translational modifications of the 26S proteasome in yeast, Proteomics 10 (2010) 2769–2779, http://dx.doi.org/10.1002/pmic.200900283.


Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

[64] M.A. Humbard, G. Zhou, J.A. Maupin-Furlow, The N-terminal penultimate residue of 20S proteasome alpha1 influences its N(alpha) acetylation and protein levels as well as growth rate and stress responses of *Haloferax volcanii*, J. Bacteriol. 191 (2009) 3794–3803, [http://dx.doi.org/10.1128/JB.00090-09](http://dx.doi.org/10.1128/JB.00090-09).

[65] T. Akahane, K. Sahara, H. Yashiroda, K. Tanaka, S. Murata, Involvement of Bag6 and the TRC pathway in proteasome assembly, Nat. Commun. 4 (2013) 2234, [http://dx.doi.org/10.1038/ncomms3234](http://dx.doi.org/10.1038/ncomms3234).

[66] S.-W. Shin, N. Shimizu, M. Tokoro, S. Nishikawa, Y. Hatanaka, M. Anzai, et al., Mouse zygote-specific proteasome chaperone important for maternal-to-zygotic transition, Biol. Open 2 (2013) 170–182, [http://dx.doi.org/10.1242/bio.20123020](http://dx.doi.org/10.1242/bio.20123020).

[67] A.F. Savulescu, M.H. Glickman, Proteasome activator 200: the HEAT is on…, Mol. Cell. Proteomics 10 (2011) R110.006890–1–R110.006890–8, [http://dx.doi.org/10.1074/mcp.R110.006890](http://dx.doi.org/10.1074/mcp.R110.006890).

[68] M. Fehlker, P. Wendler, A. Lehmann, C. Enenkel, Blm3 is part of nascent proteasomes and is involved in a late stage of nuclear proteasome assembly, EMBO Rep. 4 (2003) 959–963, [http://dx.doi.org/10.1038/sj.emboj.embor938](http://dx.doi.org/10.1038/sj.emboj.embor938).

[69] T. Dange, D. Smith, T. Noy, P.C. Rommel, L. Jurzitza, R.J.B. Cordero, et al., Blm10 protein promotes proteasomal substrate turnover by an active gating mechanism, J. Biol. Chem. 286 (2011) 42,830–42,839, [http://dx.doi.org/10.1074/jbc.M111.300178](http://dx.doi.org/10.1074/jbc.M111.300178).

[70] M.H. Weberruss, A.F. Savulescu, J. Jando, T. Bissinger, A. Harel, M.H. Glickman, et al., Blm10 facilitates nuclear import of proteasome core particles, EMBO J. 32 (2013) 2697–2707, [http://dx.doi.org/10.1038/emboj.2013.192](http://dx.doi.org/10.1038/emboj.2013.192).

[71] A. Lehmann, K. Jechow, C. Enenkel, Blm10 binds to pre-activated proteasome core particles with open gate conformation, EMBO Rep. 9 (2008) 1237–1243, [http://dx.doi.org/10.1038/embor.2008.190](http://dx.doi.org/10.1038/embor.2008.190).

[72] M. Schmidt, W. Haas, B. Crosas, P.G. Santamaria, S.P. Gygi, T. Walz, et al., The HEAT repeat protein Blm10 regulates the yeast proteasome by capping the core particle, Nat. Struct. Mol. Biol. 12 (2005) 294–303, [http://dx.doi.org/10.1038/nsmb914](http://dx.doi.org/10.1038/nsmb914).

[73] K. Sadre-Bazzaz, F.G. Whitby, H. Robinson, T. Formosa, C.P. Hill, Structure of a Blm10 complex reveals common mechanisms for proteasome binding and gate opening, Mol. Cell 37 (2010) 728–735, [http://dx.doi.org/10.1016/j.molcel.2010.02.002](http://dx.doi.org/10.1016/j.molcel.2010.02.002).

[74] S.L. McCutchen-Maloney, K. Matsuda, N. Shimbara, D.D. Binns, K. Tanaka, C.A. Slaughter, et al., cDNA cloning, expression, and functional characterization of PI31, a proline-rich inhibitor of the proteasome, J. Biol. Chem. 275 (2000) 18,557–18,565, [http://dx.doi.org/10.1074/jbc.M001697200](http://dx.doi.org/10.1074/jbc.M001697200).

[75] H. Yashiroda, Y. Toda, S. Otsu, K. Takagi, T. Mizushima, S. Murata, N-terminal α7 deletion of the proteasome 20S core particle substitutes for yeast PI31 function, Mol. Cell. Biol. 35 (2014) 151–152, [http://dx.doi.org/10.1128/MCB.00582-14](http://dx.doi.org/10.1128/MCB.00582-14) (MCB.00582–14).

[76] A. Hatanaka, B. Chen, J.-Q. Sun, Y. Mano, M. Funakoshi, H. Kobayashi, et al., Fub1p, a novel protein isolated by boundary screening, binds the proteasome complex, Genes Genet. Syst. 86 (2011) 305–314, [http://dx.doi.org/10.1266/ggs.86.305](http://dx.doi.org/10.1266/ggs.86.305).

[77] X. Li, D. Thompson, B. Kumar, G.N. DeMartino, Molecular and cellular roles of PI31 (PSMF1) protein in regulation of proteasome function, J. Biol. Chem. 289 (2014)

17,392–17,405, [http://dx.doi.org/10.1074/jbc.M114.561183](http://dx.doi.org/10.1074/jbc.M114.561183).

[78] M. Bader, S. Benjamin, O.L. Wapinski, D.M. Smith, A.L. Goldberg, H. Steller, A conserved F box regulatory complex controls proteasome activity in *Drosophila*, Cell 145 (2011) 371–382.

[79] C. Guerrero, C. Tagwerker, P. Kaiser, L. HUANG, An integrated mass spectrometry-based proteomic approach: quantitative analysis of tandem affinity-purified *in vivo* cross-linked protein complexes (QTAX) to decipher the 26S proteasome-interacting network, Mol. Cell. Proteomics 5 (2006) 366–378, [http://dx.doi.org/10.1074/mcp.M500303-MCP200](http://dx.doi.org/10.1074/mcp.M500303-MCP200).

[80] S. Djuranovic, M.D. Hartmann, M. Habeck, A. Ursinus, P. Zwickl, J. Martin, et al., Structure and activity of the N-terminal substrate recognition domains in proteasomal ATPases, Mol. Cell 34 (2009) 580–590, [http://dx.doi.org/10.1016/j.molcel.2009.04.030](http://dx.doi.org/10.1016/j.molcel.2009.04.030).

[81] F. Zhang, M. Hu, G. Tian, P. Zhang, D. Finley, P.D. Jeffrey, et al., Structural insights into the regulatory particle of the proteasome from *Methanocaldococcus jannaschii*, Mol. Cell 34 (2009) 473–484, [http://dx.doi.org/10.1016/j.molcel.2009.04.021](http://dx.doi.org/10.1016/j.molcel.2009.04.021).

[82] R.J. Tomko, M. Funakoshi, K. Schneider, J. Wang, M. Hochstrasser, Heterohexameric ring arrangement of the eukaryotic proteasomal ATPases: implications for proteasome structure and assembly, Mol. Cell 38 (2010) 393–403, [http://dx.doi.org/10.1016/j.molcel.2010.02.035](http://dx.doi.org/10.1016/j.molcel.2010.02.035).

[83] B. Luan, X. Huang, J. Wu, Z. Mei, Y. Wang, X. Xue, et al., Structure of an endogenous yeast 26S proteasome reveals two major conformational states, Proc. Natl. Acad. Sci. 201601561, 113 (10) (2016) 2642–2647, [http://dx.doi.org/10.1073/pnas.1601561113](http://dx.doi.org/10.1073/pnas.1601561113).

[84] F. Beck, P. Unverdorben, S. Bohn, A. Schweitzer, G. Pfeifer, E. Sakata, et al., Near-atomic resolution structural model of the yeast 26S proteasome, Proc. Natl. Acad. Sci. 109 (2012) 14,870–14,875, [http://dx.doi.org/10.1073/pnas.1213333109](http://dx.doi.org/10.1073/pnas.1213333109).

[85] D.M. Smith, S.-C. Chang, S. Park, D. Finley, Y. Cheng, A.L. Goldberg, Docking of the proteasomal ATPases' carboxyl termini in the 20S proteasome's alpha ring opens the gate for substrate entry, Mol. Cell 27 (2007) 731–744, [http://dx.doi.org/10.1016/j.molcel.2007.06.033](http://dx.doi.org/10.1016/j.molcel.2007.06.033).

[86] Z. Ding, Z. Fu, C. Xu, Y. Wang, Y. Wang, Y. Wang, et al., High-resolution cryo-EM structure of the proteasome in complex with ADP-AlFx, Cell Res. 27 (2017) 373–385, [http://dx.doi.org/10.1038/cr.2017.12](http://dx.doi.org/10.1038/cr.2017.12).

[87] M. Wehmer, T. Rudack, F. Beck, A. Aufderheide, G. Pfeifer, J.M. Plitzko, et al., Structural insights into the functional cycle of the ATPase module of the 26S proteasome, Proc. Natl. Acad. Sci. 201621129, 114 (6) (2017) 1305–1310, [http://dx.doi.org/10.1073/pnas.1621129114](http://dx.doi.org/10.1073/pnas.1621129114).

[88] G.C. Lander, E. Estrin, M.E. Matyskiela, C. Bashore, E. Nogales, A. Martin, Complete subunit architecture of the proteasome regulatory particle, Nature 482 (2012) 186–191, [http://dx.doi.org/10.1038/nature10774](http://dx.doi.org/10.1038/nature10774).

[89] R. Beckwith, E. Worden, E. Estrin, A. Martin, Reconstitution of the 26S proteasome reveals functional asymmetries in its heterohexameric AAA+ unfoldase, Biophys. J. 106 (2014) 468a, [http://dx.doi.org/10.1016/j.bpj.2013.11.2649](http://dx.doi.org/10.1016/j.bpj.2013.11.2649).

[90] R. Beckwith, E. Estrin, E.J. Worden, A. Martin, Reconstitution of the 26S proteasome reveals functional asymmetries in its AAA+ unfoldase, Nat. Struct. Mol. Biol. 20 (2013) 1164–1172, [http://dx.doi.org/10.1038/nsmb.2659](http://dx.doi.org/10.1038/nsmb.2659).

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), [http://dx.doi.org/10.1016/j.jmb.2017.05.027](http://dx.doi.org/10.1016/j.jmb.2017.05.027)

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly 23

[91] P. Unverdorben, F. Beck, P. Śledź, A. Schweitzer, G. Pfeifer, J.M. Plitzko, et al., Deep classification of a large cryo-EM dataset defines the conformational landscape of the 26S proteasome, Proc. Natl. Acad. Sci. 111 (2014) 5544–5549, http://dx.doi.org/10.1073/pnas.1403409111.

[92] J. He, K. Kulkarni, P.C.A. da Fonseca, D. Krutauz, M.H. Glickman, D. Barford, et al., The structure of the 26S proteasome subunit Rpn2 reveals its PC repeat domain as a closed toroid of two concentric α-helical rings, Structure 20 (2012) 513–521, http://dx.doi.org/10.1016/j.str.2011.12.015.

[93] S. Elsasser, R.R. Gali, M. Schwickart, C.N. Larsen, D.S. Leggett, B. Müller, et al., Proteasome subunit Rpn1 binds ubiquitin-like protein domains, Nat. Cell Biol. 4 (2002) 725–730, http://dx.doi.org/10.1038/ncb845.

[94] Y. Saeki, A. Saitoh, A. Toh-e, H. Yokosawa, Ubiquitin-like proteins and Rpn10 play cooperative roles in ubiquitin-dependent proteolysis, Biochem. Biophys. Res. Commun. 293 (2002) 986–992, http://dx.doi.org/10.1016/S0006-291X(02)00340-6.

[95] Y. Shi, X. Chen, S. Elsasser, B.B. Stocks, G. Tian, B.-H. Lee, et al., Rpn1 provides adjacent receptor sites for substrate binding and deubiquitination by the proteasome, Science 351 (2016) aad9421–aad9421, http://dx.doi.org/10.1126/science.aad9421.

[96] D.S. Leggett, J. Hanna, A. Borodovsky, B. Crosas, M. Schmidt, R.T. Baker, et al., Multiple associated proteins regulate proteasome structure and function, Mol. Cell 10 (2002) 495–507.

[97] C. Bashore, C.M. Dambacher, E.A. Goodall, M.E. Matyskiela, G.C. Lander, A. Martin, Ubp6 deubiquitinase controls conformational dynamics and substrate degradation of the 26S proteasome, Nat. Struct. Mol. Biol. 22 (2015) 712–719, http://dx.doi.org/10.1038/nsmb.3075.

[98] A. Aufderheide, F. Beck, F. Stengel, M. Hartwig, A. Schweitzer, G. Pfeifer, et al., Structural characterization of the interaction of Ubp6 with the 26S proteasome, Proc. Natl. Acad. Sci. 201510449, 112 (28) (2015) 8626–8631, http://dx.doi.org/10.1073/pnas.1510449112.

[99] P. Schreiner, X. Chen, K. Husnjak, L. Randles, N. Zhang, S. Elsasser, et al., Ubiquitin docking at the proteasome through a novel pleckstrin-homology domain interaction, Nature 453 (2008) 548–552, http://dx.doi.org/10.1038/nature06924.

[100] K. Husnjak, S. Elsasser, N. Zhang, X. Chen, L. Randles, Y. Shi, et al., Proteasome subunit Rpn13 is a novel ubiquitin receptor, Nature 453 (2008) 481–488, http://dx.doi.org/10.1038/nature06926.

[101] X. Chen, B.-H. Lee, D. Finley, K.J. Walters, Structure of proteasome ubiquitin receptor hRpn13 and its activation by the scaffolding protein hRpn2, Mol. Cell 38 (2010) 404–415, http://dx.doi.org/10.1016/j.molcel.2010.04.019.

[102] S. Elsasser, D. Chandler-Militello, B. Müller, J. Hanna, D. Finley, Rad23 and Rpn10 serve as alternative ubiquitin receptors for the proteasome, J. Biol. Chem. 279 (2004) 26,817–26,822, http://dx.doi.org/10.1074/jbc.M404020200.

[103] Q. Deveraux, V. Ustrell, C. Pickart, M. Rechsteiner, A 26S protease subunit that binds ubiquitin conjugates, J. Biol. Chem. 269 (1994) 7059–7061.

[104] T. Keren-Kaplan, L. Zeev Peters, O. Levin-Kravets, I. Attali, O. Kleifeld, N. Shohat, et al., Structure of ubiquitylated-Rpn10 provides insight into its autoregulation mechanism, Nat. Commun. 7 (2016) 12,960, http://dx.doi.org/10.1038/ncomms12960.

[105] M. Isasa, E.J. Katz, W. Kim, V. Yugo, S. González, D.S. Kirkpatrick, et al., Monoubiquitination of RPN10 regulates substrate recruitment to the proteasome, Mol. Cell 38 (2010) 733–745, http://dx.doi.org/10.1016/j.molcel.2010.05.001.

[106] Y. Saeki, A. Toh-e, T. Kudo, H. Kawamura, K. Tanaka, Multiple proteasome-interacting proteins assist the assembly of the yeast 19S regulatory particle, Cell 137 (2009) 900–913, http://dx.doi.org/10.1016/j.cell.2009.05.005.

[107] M. Funakoshi, R.J. Tomko, H. Kobayashi, M. Hochstrasser, Multiple assembly chaperones govern biogenesis of the proteasome regulatory particle base, Cell 137 (2009) 887–899, http://dx.doi.org/10.1016/j.cell.2009.04.061.

[108] J. Roelofs, S. Park, W. Haas, G. Tian, F.E. McAllister, Y. Huo, et al., Chaperone-mediated pathway of proteasome regulatory particle assembly, Nature 459 (2009) 861–865, http://dx.doi.org/10.1038/nature08063.

[109] B. Le Tallec, M.-B. Barrault, R. Guérois, T. Carré, A. Peyroche, Hsm3/S5b participates in the assembly pathway of the 19S regulatory particle of the proteasome, Mol. Cell 33 (2009) 389–399, http://dx.doi.org/10.1016/j.molcel.2009.01.010.

[110] A. Hanssum, Z. Zhong, A. Rousseau, A. Krzyzosiak, A. Sigurdardottir, A. Bertolotti, An inducible chaperone adapts proteasome assembly to stress, Mol. Cell 55 (2014) 566–577, http://dx.doi.org/10.1016/j.molcel.2014.06.017.

[111] M.-B. Barrault, N. Richet, C. Godard, B. Murciano, B. Le Tallec, E. Rousseau, et al., Dual functions of the Hsm3 protein in chaperoning and scaffolding regulatory particle subunits during the proteasome assembly, Proc. Natl. Acad. Sci. 109 (2012) E1001–E1010, http://dx.doi.org/10.1073/pnas.1116538109.

[112] R.J. Tomko, M. Hochstrasser, Order of the proteasomal ATPases and eukaryotic proteasome assembly, Cell Biochem. Biophys. 60 (2011) 13–20, http://dx.doi.org/10.1007/s12013-011-9178-4.

[113] D. Thompson, K. Hakala, G.N. DeMartino, Subcomplexes of PA700, the 19S regulator of the 26S proteasome, reveal relative roles of AAA subunits in 26S proteasome assembly and activation and ATPase activity, J. Biol. Chem. 284 (2009) 24,891–24,903, http://dx.doi.org/10.1074/jbc.M109.023218.

[114] S. Park, J. Roelofs, W. Kim, J. Robert, M. Schmidt, S.P. Gygi, et al., Hexameric assembly of the proteasomal ATPases is templated through their C termini, Nature 459 (2009) 866–870, http://dx.doi.org/10.1038/nature08065.

[115] C.-G. Pack, H. Yukii, A. Toh-e, T. Kudo, H. Tsuchiya, A. Kaiho, et al., Quantitative live-cell imaging reveals spatio-temporal dynamics and cytoplasmic assembly of the 26S proteasome, Nat. Commun. 5 (2014) 3396, http://dx.doi.org/10.1038/ncomms4396.

[116] F. Li, G. Tian, D. Langager, V. Sokolova, D. Finley, S. Park, Nucleotide-dependent switch in proteasome assembly mediated by the Nas6 chaperone, Proc. Natl. Acad. Sci. 114 (2017) 1548–1553, http://dx.doi.org/10.1073/pnas.1612922114.

[117] S.Y.-C. Lee, A. De la Mota-Peynado, J. Roelofs, Loss of Rpt5 protein interactions with the core particle and Nas2 protein causes the formation of faulty proteasomes that are inhibited by Ecm29 protein, J. Biol. Chem. 286 (2011) 36,641–36,651, http://dx.doi.org/10.1074/jbc.M111.280875.

[118] T. Satoh, Y. Saeki, T. Hiromoto, Y.-H. Wang, Y. Uekusa, H. Yagi, et al., Structural basis for proteasome formation controlled by an assembly chaperone nas2, Structure 22 (2014) 731–743, http://dx.doi.org/10.1016/j.str.2014.02.014.

Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly

[119] K. Takagi, S. Kim, H. Yukii, M. Ueno, R. Morishita, Y. Endo, et al., Structural basis for specific recognition of Rpt1p, an ATPase subunit of 26S proteasome, by proteasome-dedicated chaperone Hsm3p, J. Biol. Chem. 287 (2012) 12,172–12,182, http://dx.doi.org/10.1074/jbc.M112.345876.

[120] S. Park, X. Li, H.M. Kim, C.R. Singh, G. Tian, M.A. Hoyt, et al., Reconfiguration of the proteasome during chaperone-mediated assembly, Nature 497 (2013) 512–516, http://dx.doi.org/10.1038/nature12123.

[121] Y. Nakamura, T. Umehara, A. Tanaka, M. Horikoshi, B. Padmanabhan, S. Yokoyama, Structural basis for the recognition between the regulatory particles Nas6 and Rpt3 of the yeast 26S proteasome, Biochem. Biophys. Res. Commun. 359 (2007) 503–509, http://dx.doi.org/10.1016/j.bbrc.2007.05.138.

[122] V. Sokolova, F. Li, G. Polovin, S. Park, Proteasome activation is mediated via a functional switch of the Rpt6 C-terminal tail following chaperone-dependent assembly, Sci. Rep. 5 (2015) 14,909, http://dx.doi.org/10.1038/srep14909.

[123] S. Kim, Y. Saeki, K. Fukunaga, A. Suzuki, K. Takagi, T. Yamane, et al., Crystal structure of yeast rpn14, a chaperone of the 19S regulatory particle of the proteasome, J. Biol. Chem. 285 (2010) 15,159–15,166, http://dx.doi.org/10.1074/jbc.M110.104042.

[124] A. Ehlinger, S. Park, A. Fahmy, J.W. Lary, J.L. Cole, D. Finley, et al., Conformational dynamics of the Rpt6 ATPase in proteasome assembly and Rpn14 binding, Structure 21 (2013) 753–765, http://dx.doi.org/10.1016/j.str.2013.02.021.

[125] C.M. Dambacher, E.J. Worden, M.A. Herzik, A. Martin, G.C. Lander, Atomic structure of the 26S proteasome lid reveals the mechanism of deubiquitinase inhibition, eLife Sci. 5 (2016) e13027, http://dx.doi.org/10.7554/eLife.13027.

[126] J.P. Erzberger, F. Stengel, R. Pellarin, S. Zhang, T. Schaefer, C.H.S. Aylett, et al., Molecular architecture of the 40S·eIF1·eIF3 translation initiation complex, Cell 158 (2014) 1123–1135, http://dx.doi.org/10.1016/j.cell.2014.07.044.

[127] G.M. Lingaraju, R.D. Bunker, S. Cavadini, D. Hess, U. Hassiepen, M. Renatus, et al., Crystal structure of the human COP9 signalosome, Nature 512 (2014) 161–165, http://dx.doi.org/10.1038/nature13566.

[128] Z. Yu, O. Kleifeld, A. Lande-Atir, M. Bsoul, M. Kleiman, D. Krutauz, et al., Dual function of Rpn5 in two PCI complexes, the 26S proteasome and COP9 signalosome, Mol. Biol. Cell 22 (2011) 911–920, http://dx.doi.org/10.1091/mbc.E10-08-0655.

[129] C. Meister, M.K. Gulko, A.M. Köhler, G.H. Braus, The devil is in the details: comparison between COP9 signalosome (CSN) and the LID of the 26S proteasome, Curr. Genet. 62 (2016) 129–136, http://dx.doi.org/10.1007/s00294-015-0525-7.

[130] G.R. Pathare, I. Nagy, P. Śledź, D.J. Anderson, H.-J. Zhou, E. Pardon, et al., Crystal structure of the proteasomal deubiquitylation module Rpn8-Rpn11, Proc. Natl. Acad. Sci. 111 (2014) 2984–2989, http://dx.doi.org/10.1073/pnas.1400546111.

[131] E.J. Worden, C. Padovani, A. Martin, Structure of the Rpn11-Rpn8 dimer reveals mechanisms of substrate deubiquitination during proteasomal degradation, Nat. Struct. Mol. Biol. 21 (2014) 220–227, http://dx.doi.org/10.1038/nsmb.2771.

[132] B.B. Kragelund, S.M. Schenstrøm, C.A. Rebula, V.G. Panse, R. Hartmann-Petersen, DSS1/Sem1, a multifunctional and intrinsically disordered protein, Trends Biochem. Sci. 41 (5) (2016) 446–459, http://dx.doi.org/10.1016/j.tibs.2016.02.004.

[133] R.J. Tomko, M. Hochstrasser, The intrinsically disordered Sem1 protein functions as a molecular tether during proteasome lid biogenesis, Mol. Cell 53 (2014) 433–443, http://dx.doi.org/10.1016/j.molcel.2013.12.009.

[134] K. Paraskevopoulos, F. Kriegenburg, M.H. Tatham, H.I. Rösner, B. Medina, I.B. Larsen, et al., Dss1 is a 26S proteasome ubiquitin receptor, Mol. Cell 56 (3) (2014) 453–461, http://dx.doi.org/10.1016/j.molcel.2014.09.008.

[135] K. Gudmundsdottir, C.J. Lord, A. Ashworth, The proteasome is involved in determining differential utilization of double-strand break repair pathways, Oncogene 26 (2007) 7601–7606, http://dx.doi.org/10.1038/sj.onc.1210579.

[136] S.-J. Wei, J.G. Williams, H. Dang, T.A. Darden, B.L. Betz, M.M. Humble, et al., Identification of a specific motif of the DSS1 protein required for proteasome interaction and p53 protein degradation, J. Mol. Biol. 383 (2008) 693–712, http://dx.doi.org/10.1016/j.jmb.2008.08.044.

[137] A.M. Ellisdon, M. Stewart, Structural biology of the PCI-protein fold, BioArchitecture 2 (2012) 118–123, http://dx.doi.org/10.4161/bioa.21131.

[138] R.J. Tomko, M. Hochstrasser, Incorporation of the Rpn12 subunit couples completion of proteasome regulatory particle lid assembly to lid–base joining, Mol. Cell 44 (2011) 907–917, http://dx.doi.org/10.1016/j.molcel.2011.11.020.

[139] K. Fukunaga, T. Kudo, A. Toh-e, K. Tanaka, Y. Saeki, Dissection of the assembly pathway of the proteasome lid in Saccharomyces cerevisiae, Biochem. Biophys. Res. Commun. 396 (2010) 1048–1053, http://dx.doi.org/10.1016/j.bbrc.2010.05.061.

[140] R.J. Tomko Jr, D.W. Taylor, Z.A. Chen, H.-W. Wang, J. Rappsilber, M. Hochstrasser, A single α helix drives extensive remodeling of the proteasome lid and completion of regulatory particle assembly, 163 (2015) 432–444. doi: http://dx.doi.org/10.1016/j.cell.2015.09.022.

[141] E. Estrin, J.R. Lopez-Blanco, P. Chacón, A. Martin, Formation of an intricate helical bundle dictates the assembly of the 26S proteasome lid, Structure 21 (2013) 1624–1635, http://dx.doi.org/10.1016/j.str.2013.06.023.

[142] Z. Yu, N. Livnat-Levanon, O. Kleifeld, W. Mansour, M.A. Nakasone, C.A. Castaneda, et al., Base-CP proteasome can serve as a platform for stepwise lid formation, Biosci. Rep. 35 (2015) 1–14, http://dx.doi.org/10.1042/BSR20140173.

[143] J.A. Ronau, J.F. Beckmann, M. Hochstrasser, Substrate specificity of the ubiquitin and Ubl proteases, Cell Res. 26 (2016) 441–456, http://dx.doi.org/10.1038/cr.2016.38.

[144] A. Schweitzer, A. Aufderheide, T. Rudack, F. Beck, G. Pfeifer, J.M. Plitzko, et al., Structure of the human 26S proteasome at a resolution of 3.9 Å, Proc. Natl. Acad. Sci. 113 (2016) 7816–7821, http://dx.doi.org/10.1073/pnas.1608050113.

[145] S. Bohn, E. Sakata, F. Beck, G.R. Pathare, J. Schnitger, I. Nagy, et al., Localization of the regulatory particle subunit Sem1 in the 26S proteasome, Biochem. Biophys. Res. Commun. 435 (2013) 250–254, http://dx.doi.org/10.1016/j.bbrc.2013.04.069.

[146] T. Sone, Y. Saeki, A. Toh-e, H. Yokosawa, Sem1p is a novel subunit of the 26S proteasome from Saccharomyces cerevisiae, J. Biol. Chem. 279 (2004) 28,807–28,816, http://dx.doi.org/10.1074/jbc.M403165200.

ARTICLE IN PRESS

Review: Proteasome Structure and Assembly 25

[147] Y. Xie, A. Varshavsky, RPN4 is a ligand, substrate, and transcriptional regulator of the 26S proteasome: a negative feedback circuit, Proc. Natl. Acad. Sci. U. S. A. 98 (2001) 3056–3061, http://dx.doi.org/10.1073/pnas.071022298.

[148] D. Ju, L. Wang, X. Mao, Y. Xie, Homeostatic regulation of the proteasome via an Rpn4-dependent feedback circuit, Biochem. Biophys. Res. Commun. 321 (2004) 51–57, http://dx.doi.org/10.1016/j.bbrc.2004.06.105.

[149] G. Mannhaupt, R. Schnall, V. Karpov, I. Vetter, H. Feldmann, Rpn4p acts as a transcription factor by binding to PACE, a nonamer box found upstream of 26S proteasomal and other genes in yeast, FEBS Lett. 450 (1999) 27–34.

[150] R. Shirozu, H. Yashiroda, S. Murata, Identification of minimum Rpn4-responsive elements in genes related to proteasome functions, FEBS Lett. 589 (2015) 933–940, http://dx.doi.org/10.1016/j.febslet.2015.02.025.

[151] A. Rousseau, A. Bertolotti, An evolutionarily conserved pathway controls proteasome homeostasis, Nature 536 (2016) 184–189, http://dx.doi.org/10.1038/nature18943.

[152] M. Laplante, D.M. Sabatini, mTOR signaling in growth control and disease, Cell 149 (2012) 274–293, http://dx.doi.org/10.1016/j.cell.2012.03.017.

[153] S.K. Radhakrishnan, C.S. Lee, P. Young, A. Beskow, J.Y. Chan, R.J. Deshaies, Transcription factor Nrf1 mediates the proteasome recovery pathway after proteasome inhibition in mammalian cells, Mol. Cell 38 (2010) 17–28, http://dx.doi.org/10.1016/j.molcel.2010.02.029.

[154] J. Steffen, M. Seeger, A. Koch, E. Krüger, Proteasomal degradation is transcriptionally controlled by TCF11 via an ERAD-dependent feedback loop, Mol. Cell 40 (2010) 147–158, http://dx.doi.org/10.1016/j.molcel.2010.09.012.

[155] Z. Sha, A.L. Goldberg, Proteasome-mediated processing of Nrf1 is essential for coordinate induction of all proteasome subunits and p97, Curr. Biol. 24 (2014) 1573–1583, http://dx.doi.org/10.1016/j.cub.2014.06.004.

[156] D.C. KRAFT, C.C. DEOCARIS, R. WADHWA, S.I.S. RATTAN, Preincubation with the proteasome inhibitor MG-132 enhances proteasome activity via the Nrf2 transcription factor in aging human skin fibroblasts, Ann. N. Y. Acad. Sci. 1067 (2006) 420–424, http://dx.doi.org/10.1196/annals.1354.060.

[157] A.M. Pickering, R.A. Linder, H. Zhang, H.J. Forman, K.J.A. Davies, Nrf2-dependent induction of proteasome and Pa28αβ regulator are required for adaptation to oxidative stress, J. Biol. Chem. 287 (2012) 10,021–10,031, http://dx.doi.org/10.1074/jbc.M111.277145.

[158] S. Kapeta, N. Chondrogianni, E.S. Gonos, Nuclear erythroid factor 2-mediated proteasome activation delays senescence in human fibroblasts, J. Biol. Chem. 285 (2010) 8171–8184, http://dx.doi.org/10.1074/jbc.M109.031575.

[159] M.-K. Kwak, N. Wakabayashi, J.L. Greenlaw, M. Yamamoto, T.W. Kensler, Antioxidants enhance mammalian proteasome expression through the Keap1-Nrf2 signaling pathway, Mol. Cell. Biol. 23 (2003) 8786–8794, http://dx.doi.org/10.1128/MCB.23.23.8786-8794.2003.

[160] D. Stewart, E. Killeen, R. Naquin, S. Alam, J. Alam, Degradation of transcription factor Nrf2 via the ubiquitin-proteasome pathway and stabilization by cadmium, J. Biol. Chem. 278 (2003) 2396–2402, http://dx.doi.org/10.1074/jbc.M209195200.

[161] W. Wang, J.Y. Chan, Nrf1 is targeted to the endoplasmic reticulum membrane by an N-terminal transmembrane domain. Inhibition of nuclear translocation and transacting function, J. Biol. Chem. 281 (2006) 19,676–19,687, http://dx.doi.org/10.1074/jbc.M602802200.

[162] S.-C. Lo, M. Hannink, PGAM5 tethers a ternary complex containing Keap1 and Nrf2 to mitochondria, Exp. Cell Res. 314 (2008) 1789–1803, http://dx.doi.org/10.1016/j.yexcr.2008.02.014.

[163] S.K. Radhakrishnan, W. den Besten, R.J. Deshaies, p97-dependent retrotranslocation and proteolytic processing govern formation of active Nrf1 upon proteasome inhibition, eLife Sci. 3 (2014) e01856, http://dx.doi.org/10.7554/eLife.01856.

[164] M. Ohtsuji, F. Katsuoka, A. Kobayashi, H. Aburatani, J.D. Hayes, M. Yamamoto, Nrf1 and Nrf2 play distinct roles in activation of antioxidant response element-dependent genes, J. Biol. Chem. 283 (2008) 33,554–33,562, http://dx.doi.org/10.1074/jbc.M804597200.

[165] K. Fukagai, T. Waku, A.M.M.A. Chowdhury, K. Kubo, M. Matsumoto, H. Kato, et al., USP15 stabilizes the transcription factor Nrf1 in the nucleus, promoting the proteasome gene expression, Biochem. Biophys. Res. Commun. 478 (2016) 363–370, http://dx.doi.org/10.1016/j.bbrc.2016.07.045.

[166] Y. Zhang, B.D. Manning, mTORC1 signaling activates NRF1 to increase cellular proteasome levels, Cell Cycle 14 (2015) 2011–2017, http://dx.doi.org/10.1080/15384101.2015.1044188.

[167] S. Chen, J. Wu, Y. Lu, Y.-B. Ma, B.-H. Lee, Z. Yu, et al., Structural basis for dynamic regulation of the human 26S proteasome, Proc. Natl. Acad. Sci. 113 (2016) 12,991–12,996, http://dx.doi.org/10.1073/pnas.1614614113.


Please cite this article as: L. Budenholzer, et al., Proteasome Structure and Assembly, J. Mol. Biol. (2017), http://dx.doi.org/10.1016/j.jmb.2017.05.027
