
REVIEW

# Fungal dysbiosis: immunity and interactions at mucosal barriers

**Iliyan D. Iliev**${}^{1-3}$ **and Irina Leonardi**${}^{1,3}$

**Abstract |** Fungi and mammals share a co-evolutionary history and are involved in a complex web of interactions. Studies focused on commensal bacteria suggest that pathological changes in the microbiota, historically known as dysbiosis, are at the root of many inflammatory diseases of non-infectious origin. However, the importance of dysbiosis in the fungal community — the mycobiota — was only recently acknowledged to have a pathological role, as novel findings have suggested that mycobiota disruption can have detrimental effects on host immunity. Fungal dysbiosis and homeostasis are dynamic processes that are probably more common than actual fungal infections, and therefore constantly shape the immune response. In this Review, we summarize specific mycobiota patterns that are associated with fungal dysbiosis, and discuss how mucosal immunity has evolved to distinguish fungal infections from dysbiosis and how it responds to these different conditions. We propose that gut microbiota dysbiosis is a collective feature of complex interactions between prokaryotic and eukaryotic microbial communities that can affect immunity and that can influence health and disease.

---

### Symbiosis

An ecological relationship between different species that persistently live in close contact. It includes relationships such as mutualism, parasitism and commensalism.

---

Fungi are omnipresent in our living environment, are an integral part of all ecosystems${}^{1}$, and have had a central role in the evolution of life${}^{2}$. Molecular evidence suggests that animals and fungi have co-evolved since diverging from plants more than 1 billion years ago${}^{3}$. This has provided abundant opportunities for fungi to greatly influence the evolution of animals${}^{3}$ and their immune systems${}^{4,5}$. The immune system has been crucial in establishing a close relationship between the host, bacteria and fungi by ‘keeping the peace’ at the mammalian barrier surfaces that are highly colonized by microorganisms. Although this relationship often results in symbiosis (BOX 1), it can also lead to diseases that have a devastating socioeconomic impact${}^{6-11}$.

The rapid development of deep-sequencing and computational technologies has provided great opportunities to explore the structure and functionality of the microbial communities that are associated with our body surfaces. Extensive microbial data generated worldwide have enabled the scientific community to uncover the functions of the human microbiota${}^{12}$. Bacterial communities have been the primary focus of these sequencing efforts, and fewer data are currently available that characterize the fungal microbiota — the mycobiota. Nevertheless, recent studies have revealed that host-associated fungal populations are also dynamic and responsive to environmental and pathophysiological changes${}^{9,13-17}$.

A century of studies that focused on commensal bacteria in the gut have suggested that dysbiosis${}^{18}$ is at the root of several diseases of complex aetiology and that involve genetic polymorphisms, immune mechanisms and the microbiota${}^{19-21}$. Although the term dysbiosis is imperfect, it describes conditions that are distinct from infection and it is widely used to describe altered bacterial communities as both a cause and a consequence of pathologies (BOX 2). Nevertheless, infection and dysbiosis can both prelude each other — either homeostasis can be followed by dysbiosis and then infection, or homeostasis can be followed by infection and then lead to dysbiosis — in a dynamic continuum. Several studies have indicated that a similar process involving fungal communities — fungal dysbiosis — could affect the host mycobiota${}^{9,16,17,21-24}$ (FIG. 1). As fungi are common inhabitants of all barrier surfaces, changes in fungal communities might have substantial effects on the host immune responses.

In this Review, we discuss the mycobiota in homeostasis and dysbiosis, how mucosal immunity distinguishes between fungal infections and fungal dysbiosis, and how mucosal immunity responds to these conditions. We define the specific mycobiota patterns that are associated with fungal dysbiosis and changes in mycobiota diversity at several body sites. Finally, we discuss the impact of fungal dysbiosis on

---

**1 Gastroenterology and Hepatology Division, Joan and Sanford I. Weill Department of Medicine, Weill Cornell Medicine, New York, NY 10021, USA.**

**2 Department of Microbiology and Immunology, Weill Cornell Medicine, New York, NY 10065, USA.**

**3 The Jill Roberts Institute for Research in Inflammatory Bowel Disease, Weill Cornell Medicine, New York, NY 10021, USA.**

**Correspondence to I.D.I. iliev@med.cornell.edu**

**doi:**10.1038/nri.2017.55

**Published online 12 Jun 2017**

REVIEWS

Box 1 | Ecological relationships between fungi and their hosts

There are numerous examples of oppositional and symbiotic relationships between fungi and animals. Whereas some fungi, such as members of the Pleurotaceae family, have become predators and use elaborate strategies to catch and to digest nematodes that live in the soil, other fungi, such as Harposporium spp. and Ophiocordyceps spp., have adapted to a parasitic lifestyle<sup>128</sup>. Ophiocordyceps spp. have gone as far as manipulating host behaviour to ensure their own survival and dispersal<sup>128</sup>. Other fungi, such as Blastomyces dermatitidis and some Aspergillus species, have developed mechanisms that mimic mammalian regulatory molecules and suppress host immunity<sup>129,130</sup>. The skin and the mucosal surfaces of animals and humans are rich in nutrients and provide temperate environments for commensal fungal symbionts to thrive. Malassezia spp. are well adapted to sebaceous microenvironments and are abundantly present on the skin<sup>9,60</sup>. The feet are home to a diverse fungal community that is dominated by saprophytic fungi that feed on dead cells, which are shed continuously at this skin site<sup>9</sup>. Symbiotic relationships between fungi and their hosts are well established in the gastrointestinal tracts of different orders of animals, from insects to mammals<sup>10,11</sup>. Whereas some fungal symbionts are crucial for processing indigestible lignocellulose, which is associated with a herbivorous diet, other symbionts produce enzymes that are necessary for the neutralization of toxic dietary compounds<sup>11</sup>. The host, conversely, provides the commensal mycobiota with an optimal environment to grow, feed and procreate. In some cases, the fungal symbionts are even vertically transmitted through generations<sup>131</sup>, suggesting that, similar to bacterial symbiosis, some fungal commensals might be inherited from the mother. Recently, high-throughput sequencing approaches revealed that mouse and human intestines are home to a diverse fungal community that might have an important role in host physiology and immunity<sup>13,15–17,21</sup>.

(BOX 2), we further define the specific conditions that are associated with fungal community disruption in the context of inflammatory diseases, host immunity and drug-triggered microbial alterations.

### Mycobiota in the oral cavity

The oral cavity is one of the first mucosal sites where the asymptomatic carriage of fungi was described<sup>31</sup>. Diversity in the oral mycobiota is low and it is dominated by members of the Ascomycota phylum, mainly Candida spp.<sup>25,32</sup>. Culture-independent methods have revealed that most fungal species in the oral cavity are Fusarium spp. or members of a species belonging to the Saccharomycetaceae family (such as Candida albicans, Candida dubliniensis, Candida rugosa, Candida pararugosa, Saccharomyces cerevisiae, Hanseniaspora uvarum and Pichia spp.)<sup>25,32</sup>. Dysbiosis in the oral mucosa is poorly characterized and most studies have focused on the overgrowth of C. albicans upon immunosuppression and diabetes<sup>32</sup> (FIG. 1b). The first culture-independent assessment of the oral mycobiota in patients with HIV revealed that, in addition to the overgrowth of C. albicans, the presence of other species belonging to the Ascomycota phylum, including Dothideomycetes spp. and Leotiomyctes spp., was also increased, whereas the presence of Tremellomycetes spp. was reduced<sup>32</sup>. Of note, this study did not observe any alteration in the bacterial microbiota, indicating that the observed fungal dysbiosis was probably mediated by a direct effect of the compromised immune system and not by an indirect effect mediated by changes in the bacterial microbiota.

### Mycobiota at steady state and in dysbiosis

Fungi have traditionally been studied using culture-dependent methods. Only recently have advances in deep-sequencing technologies provided opportunities to unveil the complexity of the fungal communities that colonize mammalian barrier surfaces. Most approaches have relied on sequencing the fungal internal transcribed spacer (ITS) ITS1 and ITS2 regions and using computational algorithms to align operational taxonomic units (OTUs) to respective fungal databases<sup>9,16,25,26</sup>. Whereas fungal identification remains challenging in environmental and soil samples owing to enormous fungal diversity and the presence of many unknown fungal species<sup>27</sup>, the development of several databases designed to specifically target host-associated fungal communities has made it possible to identify more than 80% of the fungal ITS sequences derived from faecal or mucosal samples<sup>9,28–30</sup>. Despite this progress, each step of this identification approach can contribute to biases<sup>26,28</sup> and can complicate the comparison of mycobiota sequencing results from different studies. Additional bias might be introduced by therapeutic and dietary interventions that can contribute to further changes in fungal communities<sup>15</sup>. Despite these confounding factors, several studies have revealed similarities in how fungal communities adapt to environmental perturbations, and these are summarized below. Keeping in mind the limitations that the term dysbiosis imposes

### Dysbiosis

A generalized term indicating changes in the composition of the microbiota that is caused by multiple factors. It can be either a cause or a consequence of disease. The term is undergoing revision in light of recent advances in microbiome science.

### Internal transcribed spacer (ITS)

A sequence in the fungal genome positioned between the 18S and 5.8S (ITS1) or the 5.8S and 28S (ITS2) fungal rDNA that is widely used in mycobiome next-generation sequencing. The variability of the ITS regions enables them to be used to classify fungal genera and species.

### Operational taxonomic units (OTUs)

Clusters of marker gene sequences (for example, 16S rRNA or internal transcribed spacer) based on sequence similarity used for taxonomy-independent community analysis.

### Mycobiota in the gastrointestinal tract

The fungal community in the lower gastrointestinal tract has higher richness and is more diverse than that in the upper gastrointestinal tract. A recent review of 36 published gut mycobiota articles concluded that only 15 of the 267 identified species were detected in more than 5 studies<sup>33</sup>. Among these 15 species, 13 can grow at 37°C and thus have the potential to permanently inhabit the intestinal niche<sup>22,34</sup>. These species belong to the genera Candida, Saccharomyces, Aspergillus, Cryptococcus, Malassezia, Cladosporium, Galactomyces and Trichosporon<sup>33</sup>. Nevertheless, commensalism is not the only way in which fungi can influence host immunity. For example, although Histoplasma spp., Blastomyces spp. and Coccidioides spp. cannot colonize the mucosal surfaces, they can cause severe infections in the lung<sup>35</sup>. Similarly, non-commensal fungi that are present in the diet, the air or other environmental sources, might also activate immune responses in the gut<sup>15,24</sup> (FIG. 1b).

The high fungal variability observed in different studies has been interpreted as a sign of the temporal instability of the intestinal mycobiota<sup>13,34</sup>. However, most of these studies relied on the sampling of faecal material. Although additional data are needed to draw definitive conclusions, a recent study of mucosa-associated fungi suggests that a more stable community is associated with the gut mucosa<sup>36</sup> and that this can be perturbed during intestinal disease<sup>37</sup>. These results are interesting, as fungi that are in close proximity to the intestinal mucosa might

2 | ADVANCE ONLINE PUBLICATION

© 2017 Macmillan Publishers Limited, part of Springer Nature. All rights reserved.

Richness  
The number of different species represented in an ecological community.

Commensalism  
A relationship between two species through which one organism benefits from the other without affecting it.

Inflammatory bowel disease (IBD). A relapsing and remitting condition of complex aetiology. It is characterized by inflammation of the lower digestive tract with possible extra-intestinal manifestations. The most common types of IBD are Crohn’s disease and ulcerative colitis.

have an increased potential to interact locally with the intestinal epithelium and the mucosal immune system.

It has been known for a long time that inflammatory conditions that target the lower gastrointestinal tract can promote bacterial dysbiosis, and this has highlighted the important contribution of bacteria to the aetiology of inflammatory bowel disease (IBD)³⁸,³⁹. In light of these findings, several studies have investigated fungal dysbiosis in patients with IBD and have identified a general increase in the fungal burden in the colonic mucosa during both Crohn’s disease²¹,²² and ulcerative colitis⁴⁰. Consistent with these results, increases in fungal burden and pronounced fungal translocation occur in the mucosa during the chronic stage of colitis⁴¹, as well as during the acute stage of colitis in mice deficient in *Clec7a* (which encodes the fungal receptor dectin 1 (see below))¹⁶. Together, these studies suggest that genetic factors, extensive tissue damage and the presence of an inflammatory environment can cause fungal dysbiosis in the gut.

Several fungal taxa seem to be consistently altered during chronic intestinal inflammation. Patients with IBD have an increased occurrence and abundance of cultivable *C. albicans* in the faeces, and this has been further confirmed by culture-independent studies¹⁷,²¹–²³. In patients with Crohn’s disease, the presence of *C. albicans* and *Candida parapsilosis* increased in the inflamed mucosa³⁷. Increased abundance of *Candida tropicalis* in Crohn’s disease correlated with inflammation and increased levels of anti-*S. cerevisiae* antibody (ASCA)²³. Consistently, intestinal inflammation is sufficient to promote *Candida* spp. colonization and increased abundance in the mouse gut¹⁶,⁴¹. Although it is clear that *Candida* spp. increase during intestinal disease (FIG. 1a), the effect of the expansion on other gut fungi remains unclear. Some studies have suggested that the presence of species of the phylum Ascomycota decreases in favour of an increase in species of the phylum Basidiomycota¹⁷,³⁷, whereas another study reported a general increase in five species of the phylum Ascomycota in IBD²¹.

Clone library approaches based on 18S rDNA denaturing gradient gel electrophoresis (DGGE) show that patients with Crohn’s disease have increased fungal diversity and richness in the faeces and the mucosa, compared with healthy individuals and with individuals who have other forms of intestinal inflammation, such as patients with infectious colitis³⁷,⁴⁰. By contrast, ITS-sequencing approaches showed either no changes in the mycobiota diversity in faecal samples¹⁷ and colonic mucosa²² during Crohn’s disease, or a reduction in fungal diversity in a paediatric IBD cohort that mainly included patients with Crohn’s disease⁴². A further study showed decreased species richness in the faeces of patients with Crohn’s disease and their healthy relatives compared with unrelated healthy controls, but no change in fungal diversity was reported²³. Interestingly, one study showed that patients with active ulcerative colitis have a clear reduction in fungal diversity¹⁷. The differences in the gut mycobiota observed among different studies suggest that, in addition to inflammation, other factors, such as immunosuppressive therapy, disease type (Crohn’s disease or ulcerative colitis) and disease-specific diet might influence the structure of the mycobiota in patients with IBD.

Patients with Crohn’s disease have increased levels of antifungal antibodies, which show that an immune response to intestinal fungi is elicited during IBD⁴³. These systemic IgG and IgA ASCA immunoglobulins are widely recognized as reliable diagnostic markers for Crohn’s disease and are good predictors of the disease course⁴³. Recent studies have shown that *C. albicans* can act as an immunogen for ASCA production⁴⁴ and that ASCAs cross-react with cell wall mannans isolated from other yeasts, including commensal *Candida* species⁴⁵. Despite this link with intestinal fungi, the exact mechanism behind ASCA generation and the role of ASCAs in antifungal immunity are not yet understood.

The frequent co-occurrence of gastrointestinal symptoms and neurodevelopmental disorders has uncovered the presence of bacterial dysbiosis in autism-spectrum disorders (ASDs), schizophrenia and other neurodevelopmental disorders⁴⁶,⁴⁷. Patients with ASD and Rett syndrome were recently reported to have altered intestinal mycobiota⁴⁷,⁴⁸; however, whether fungi also contribute to the development of gastrointestinal symptoms in these patient groups is currently unknown.

**Box 2 | Uncoupling infection from dysbiosis**

Early in the 16th century, Girolamo Fracastoro proposed that minute invisible seeds — seminaria — transmit diseases from person to person¹³². This revolutionary theory of the microbial origin of disease was proven approximately 100 years later with the development of the microscope by Antony van Leeuwenhoek¹³³. It took another 200 years for the theory to fully ripen. In the second half of the 19th century, Louis Pasteur and Friedrich Jakob Henle independently published their ideas of the microbial origin of diseases¹³⁴,¹³⁵. The idea was further developed by Robert Koch¹¹², who eventually developed guidelines known as Koch’s postulates that were intended to move the field forwards from observations to evidence-based causative relationships between microorganisms and disease¹³⁶. Over the years, Koch’s postulates have undergone several modifications to incorporate acute and chronic diseases with diverse aetiologies¹³⁷ and to include advances led by genomics-based approaches in clinical microbiology and virology¹³⁸,¹³⁹. Although Koch’s postulates framed the idea of the microbial origin of infectious diseases, they cannot explain situations in which microorganisms indirectly shape a pro-inflammatory or an anti-inflammatory environment to affect immunity. During the second half of the 19th century, while working on causes of longevity, Ilya Ilyich Metchnikov theorized that intestinal microorganisms can “auto-intoxicate and auto-infect” their host¹⁸. Metchnikov and colleagues Stamen Grigorov and Léon Massol proposed that lactic acid-producing bacteria, which they called *Bulgarian bacillus*, can decelerate this process¹⁸. Together with the concept of beneficial bacteria, Metchnikov coined the term dysbiosis (initially known as dysbacteriosis) to define the pathological changes of the intestinal microflora¹⁸. However, dysbiosis is a general and wide definition, which can be applied to either cause or effect. Some have criticized these shortcomings¹⁴⁰ or have attempted to define dysbiosis within several conditions¹⁴¹. Similar to the modifications that Koch’s postulates underwent over time, the concept of dysbiosis will probably evolve to reflect the modern advances in microbiome science.

**Mycobiota in the lung, vagina and skin**

Disease-mediated fungal dysbiosis has been described at other barrier sites that have a lower fungal burden than the gut. The healthy lung has long been considered an aseptic organ and bacterial colonization was thought to occur only during disease²⁰. Similarly, the fungal burden is generally low in healthy individuals whose lung mycobiota seems to mostly comprise environmental fungi or fungi disseminating from the oral cavity,

REVIEWs

a

| Genus |  |
| --- | --- |
| Aspergillus |  |
| Alternaria |  |
| Candida |  |
| Capnodiales |  |
| Cladosporium |  |
| Cryptococcus |  |
| Debaryomyces |  |
| Epicoccum |  |
| Eremothecium |  |
| Malassezia |  |
| Penicillium |  |
| Pichia |  |
| Saccharomyces |  |
| Trichosporon |  |
| Other |  |

| Phylum |  |
| --- | --- |
| Ascomycota |  |
| Basidiomycota |  |
| Other |  |

Healthy mycobiota

Dysbiotic mycobiota

Mouth

Lung

Skin (AF)

Gut

Vagina

Skin (HF)

Wound (HF)

b Factors contributing to dysbiosis

In the mouth, HIV-mediated immunodeficiency causes severe dysbiosis, which correlates with decreased numbers of CD4+ T cells

Cystic fibrosis causes severe physiological changes in the lung accompanied by the outgrowth of opportunistic pathogens. Candida spp. can adapt to escape *Pseudomonas aeruginosa*-mediated inhibition of the yeast-to-hypha transition

Primary immunodeficiencies that disrupt the T~H~17 pathway, such as STAT3 mutation, lead to fungal dysbiosis

In the gut, dysbiosis is induced by antibiotic-mediated depletion of bacteria, genetic defects in antifungal immunity pathways, changes in diet, antifungal drugs and inflammation

Several factors can lead to increased growth of *Candida* spp. in the vagina, including glycogen and oestrogen produced during pregnancy, HIV-mediated immunodeficiency and depletion of lactobacilli

Temporally stable dysbiotic fungal communities occupy chronic wounds and can interfere with the skin-healing process

Figure 1 | The mycobiota in health and dysbiosis. **a** | During homeostasis, diverse fungal communities reside on all human barrier surfaces, such as the mouth<sup>32</sup>, lung<sup>53</sup>, skin<sup>9</sup>, gut<sup>14,17,22</sup> and vagina<sup>57</sup> (left side). The pie charts represent the relative abundance of the observed taxa at the phylum and genus levels (inner and outer circles, respectively). Of note, the data for the vagina are estimates that are based on culture-dependent studies, due to a lack of sequencing-based studies related to disease conditions (indicated with an asterisk). ‘Other’ refers to sequences with <5% relative abundance. During disease states, these fungal communities are perturbed (right side). Dysbiotic fungal communities are observed in the oral cavity<sup>32</sup> and the vagina<sup>58</sup> in individuals with HIV; in the lungs of individuals with cystic fibrosis<sup>51</sup>; on the skin of individuals with primary immunodeficiency<sup>61</sup> and chronic wounds<sup>63</sup>; and in the gut of patients with Crohn’s disease<sup>14,17,22</sup>. **b** | Factors contributing to fungal dysbiosis at different barrier surfaces. AF, antecubital fossa; HF, hind foot; STAT3, signal transducer and activator of transcription 3.

including species belonging to the genera *Cladosporium*, *Aspergillus*, *Penicillium* and *Candida*<sup>25,32,49</sup>. By contrast, more stable fungal communities can colonize the lung when its physiology is altered (FIG. 1b). In most patients with cystic fibrosis, for example, the fungal burden in the lung seems to be increased, whereas alpha diversity is reduced and this is correlated with disease severity<sup>50,51</sup>. Despite the high inter-individual variability of mycobiota composition, fungal genera such as *Candida* and *Aspergillus* have been linked to morbidity in individuals with cystic fibrosis<sup>50,51</sup>. Of note, once established, lung dysbiotic fungal communities seem to stabilize and persist even in the presence of antibiotic therapy or immunosuppressant therapy<sup>51,52</sup>. Fungi have also been linked to the severity of asthma<sup>20,53</sup>; however, it is unclear whether the effect is local or is influenced by the intestinal mycobiota, as suggested by experimental studies<sup>24,54,55</sup>.

Alpha diversity

A biodiversity measure of the mean species diversity within an individual environmental habitat.

Vaginal candidiasis is one of the most common mucosal fungal infections in humans. However, fungal communities are present in healthy women without active infections. Similar to the oral mucosa and mycobiota, the vaginal mycobiota has low diversity and the vaginal mucosa is mainly colonized by *C. albicans*, *Candida glabrata*, *Candida krusei*, *C. parapsilosis* and *S. cerevisiae*<sup>56–58</sup>. Several factors, such as antibiotic use, pregnancy, HIV infection and recurrent vulvovaginal candidiasis, have been associated with alterations of the vaginal fungal community structure<sup>56,58,59</sup> (FIG. 1b). *Candida* spp. colonization of the vagina increases by up to 40% during pregnancy, owing to increased oestrogen levels in and glycogen production by vaginal epithelial cells, and this might have important consequences for pregnant women, including preterm delivery<sup>56</sup>.

In contrast to the other surfaces of the body, the skin is cold, dry and has a higher availability of lipids and

proteins (such as keratin), which constitute the major nutrients for the colonizing fungal flora, including lipophilic species such as *Propionibacterium* spp. and *Malassezia* spp. [8,60]. The skin microbiota shows temporal stability and can be influenced by primary immunodeficiencies [60–62]. When skin integrity is compromised, fungi rapidly populate the wound site. A recent study demonstrated that fungi are present in 80% of chronic wounds in patients with type 1 or type 2 diabetes [63] (FIG. 1b). Despite the high interpersonal variability of the wound mycobiota, this study showed an association of *Candida* spp., *Trichosporon* spp. and *Rhodotorula* spp. with non-healing wounds and amputation, and temporally unstable fungal and bacterial communities were associated with a positive healing outcome [63]. Therefore, in addition to bacteria, the specific signature of a wound fungal community can have a negative outcome on the skin-healing process.

Together, these studies suggest that disease-induced dysbiotic fungal communities are often well adapted to changes that are induced by pathological conditions and can contribute to the disease course. Further studies are required in treatment-naive individuals, including paediatric patients and newly diagnosed patients, to provide crucial insights into a purely disease-driven dysbiosis. Mouse studies are an essential tool to analyse the effect of fungal dysbiosis under such conditions and have already provided some evidence for fungal involvement in several pathologies (discussed below).

### Influence of age, gender, diet and environment

Several studies have shown a high variability of the mycobiota between different individuals, which complicates the distinction between healthy and dysbiotic fungal communities. This variability highlights the need for a stricter definition of dysbiosis (BOX 2) and the need for experimental studies that define the consequences of an altered mycobiota. For example, the composition of the mycobiota in the gastrointestinal tract is influenced substantially by diet, gender, age and geographical location [64]. *Aspergillus* spp. and *Tremellomycetes* spp. are more abundant in male individuals, whereas *Candida* spp. are more abundant in females, possibly owing to these fungi spreading from the vagina [64]. Genera such as *Aspergillus*, *Tremellomycetes* and *Penicillium* are abundant in infants, whereas the overall diversity of the fungal community decreases with age [42,64]. Similarly, the diversity of the skin mycobiota decreases with age, probably owing to increasingly sebaceous skin that favours colonization by *Malassezia* species [65].

Diet can influence not only bacterial but also fungal communities, as food is a constant source of fungal species that are associated with vegetables, fruits and dairy products [15]. Individuals who ate a controlled animal-derived diet that was rich in cheese had an increased fungal burden in the gut, and fungal species that were detected in the faeces of these individuals corresponded to those isolated from the food that they had consumed [15,34]. Diet might be one of the factors that contributes to the differences in the fungal community that are observed between different ethnicities [21].

Together, these studies suggest that dietary fungal intake contributes substantially to the gut mycobiome and that specific dietary nutrients can sustain specific fungal community members (FIG. 1b). In support of this, a recent study reported that *Candida* spp. were positively associated with carbohydrates and were negatively associated with saturated fatty acids, whereas *Aspergillus* spp. were negatively correlated with dietary short-chain fatty acids [14].

#### Mucosal immunity to fungi

The immune system has evolved to tolerate fungi and to respond to them upon injury or infection, and the presence of fungi at mammalian barrier surfaces might constantly shape immunity. Mucosal immunity to fungi has generally been explored in the context of fungal infections [8,35,66–72] (FIG. 2a). However, very little is known about the role of fungi in influencing immunity during the steady state or during dysbiosis (FIG. 2b). We discuss below the mechanisms of antifungal immunity, using the lower gastrointestinal tract as a model barrier surface where, despite their high abundance, fungi rarely cause infection but fungal dysbiosis occurs frequently [16,17,21–24].

#### The role of cell-mediated immunity

The cell types that are involved in innate and adaptive immune responses to fungi during infection have been well characterized for several barrier surfaces, such as the oral mucosa, the lungs, the vaginal mucosa and the skin [66,68,69,72–76] (FIG. 2a). Innate immune cells are important for the clearance of fungi and for the initiation of adaptive immune responses during fungal infections [8,35,66–72]. Neutrophils, CCR2<sup>+</sup>Ly6C<sup>+</sup> monocytes, CX3CR1<sup>+</sup> mononuclear phagocytes, CD11b<sup>+</sup>CD103<sup>+</sup> dendritic cells, natural killer cells and epithelial cells have important functions in antifungal immunity by having a role in the recognition, phagocytosis and killing of fungi, as well as through their indirect activation by cell–cell crosstalk [66,68,69,72–72]. The activation of specific cell types or cell–cell crosstalk is further dependent on the specific barrier surface that is affected by the fungal infection (FIG. 2a). Whereas some fungi, such as *Candida* spp., are widely present and can infect several barrier sites, the distribution of other fungi is tissue-dependent [29]. As a central hub of mucosal immunity, the gastrointestinal tract is naturally equipped with cellular machinery to recognize and to interact with the microbiota [79]. The intestine harbours several subsets of phagocytes, which are known to respond to bacterial infections or to fluctuations in the bacterial communities [79]. Although phagocytic cell populations probably sense mycobiota in the intestine, their role during fungal dysbiosis remains unknown.

In addition to cellular innate and adaptive immunity, humoral immune mechanisms such as the complement system and antifungal antibodies have an important role in antifungal immunity [80] and might be involved in shaping the mycobiota at the mucosal surfaces. Innate immune cells are further equipped with a range of receptors to sense and to interact with fungi [5,29].

REVIEWs

a Fungal infection

Oral cavity

AMP

Candida spp.

Lung

Aspergillus spp. Blastomyces spp. Histoplasma spp. Cryptococcus spp.

AMP

IL-1β, CXCL2, TNF

Macrophage

NK cell

IL-6, IL-12

Monocyte

Pentraxin 3, TNF

DC

Neutrophil

T<sub>H</sub>17 cell

ROS

T<sub>H</sub>1 cell

Chemokines

IL-17, IL-22

NK cell

Neutrophil

IL-17

γδ T cell

IL-6, IL-12

IL-1β, TNF

IL-17

T<sub>H</sub>17 cell

γδ T cell

ILC3

Monocyte

CCR2

Skin

Langerhans

cell

Candida spp.

Malassezia spp.

Vagina

AMP

Candida spp.

IL-1β, IL-6, IL-23

T<sub>H</sub>17 cell

γδ T cell

T<sub>H</sub>1 cell

Chemokines

IL-17, IL-22

DC

Macrophage

IL-1β, IL-12

IL-23

Neutrophil

ROS

ILC3

IDO1+

DC

iT<sub>reg</sub> cell

IL-10, TGFβ

Figure 2 | Examples of mucosal immune responses to fungal infection and dysbiosis.

The host barrier surfaces are inhabited by mycobiota. Sites such as the skin, lung, oral cavity and the vagina are prone to fungal infections and fungal dysbiosis when host immunity is compromised, whereas the gastrointestinal tract is resistant to fungal infections but susceptible to fungal dysbiosis. Host mucosal immunity has evolved to distinguish fungal infections from fungal dysbiosis. **a** | Fungal infection leads to breaches in epithelial surfaces, which in turn leads to the rapid infiltration of neutrophils and monocytes at the site of infection. The production of cytokines and chemokines by epithelial and innate immune cells leads to further recruitment of immune cells such as monocytes, neutrophils and different T cell subsets. Resident phagocytes such as macrophages and dendritic cells (DCs) recognize and process fungal antigens and promote fungal-specific T cell responses and polarization. Group 3 innate lymphoid cells (ILC3s) can respond directly to fungi or to the inflammatory environment by producing IL-22 and IL-17. Neutrophils infiltrate and facilitate rapid fungal killing by producing reactive oxygen species (ROS). Epithelial cells produce antimicrobial peptides (AMPs) that directly affect fungal survival. Site-dependent features define mucosal immunity upon fungal infection at different barrier surfaces. **b** | By contrast, fungal dysbiosis is characterized by alterations in the mycobiota that might be both the cause and the consequence of changes in the tissue environment or in the intestinal lumen. Fungal dysbiosis can influence both local and systemic immunity through several mechanisms, including the modulation of the cytokine milieu, the activation of different cell types and the release of metabolites. Fungal dysbiosis in the gut can influence immunity at distant sites, such as the lung, and can contribute to allergy. Intestinal inflammation and breaches of the intestinal epithelial barrier caused by non-fungal triggers can lead to direct exposure to fungal antigens derived from the intestinal lumen and to the development of systemic IgG and IgA anti-Saccharomyces cerevisiae antibodies (ASCA). Mucosal immunity to gut fungi during dysbiosis is poorly explored and only a few molecules have been studied in this context. At the barrier surfaces, fungi and bacteria can affect each other; thus, microbiota dysbiosis is probably a collective feature of the complex crosstalk between the fungal and bacterial microbiota and the host. CCR2, CC-chemokine receptor 2; CXCL2, CXC-chemokine ligand 2; IDO1, indoleamine 2,3-dioxygenase 1; IFNγ, interferon-γ; iT<sub>reg</sub>, induced T regulatory; NK, natural killer; PGE2, prostaglandin E2; TNF, tumour necrosis factor; T<sub>H</sub>, Thelper.

b Fungal dysbiosis

Lung

M2

macrophage

DC

Eosinophil

Expansion

of cell

types

Modulation of

cytokine milieu

Other

extraintestinal

effects?

T<sub>H</sub>2 cell

PGE2

Other mediators?

Disease-induced

epithelial cell

damage

AMP

Altered

mycobiota

Altered

bacteria

Intestine

?

Cellular networks

that sense and/or

respond to fungi

ASCA

6 | ADVANCE ONLINE PUBLICATION

© 2017 Macmillan Publishers Limited, part of Springer Nature. All rights reserved.
www.nature.com/nri

The role of C-type lectin receptors

Genetic evidence and experimental studies have underlined a central role for C-type lectin receptors (CLRs) in antifungal immunity, whereas Toll-like receptors (TLRs) and NOD-like receptors (NLRs) mostly have a secondary role^{5,81}. CLRs, such as dectin 1, dectin 2, dectin 3, macrophage-inducible C-type lectin (MINCLE; also known as CLEC4E) and the mannose receptor, recognize several molecules that are present in the fungal cell wall^{5,81}. Spleen tyrosine kinase (SYK) is the primary signal transduction molecule that is used by several of these receptors, and its activation requires an immunoreceptor tyrosine-based activation motif (ITAM) or ITAM-like motif present within the receptor tail (in the case of dectin 1) or within an associated Fc receptor common γ-chain (FcRγ) adaptor molecule (in the case of dectin 2, dectin 3 and MINCLE)^{5,80,81}. Upon receptor engagement, ITAM motifs are phosphorylated by SRC family kinases, leading to the recruitment and phosphorylation of SYK and to the activation of a downstream signalling cascade, including phospholipase Cγ2 (PLCγ2), protein kinase Cδ (PKCδ), the caspase recruitment domain-containing protein 9 (CARD9)–BCL-10–mucosa-associated lymphoid tissue lymphoma translocation protein 1 (MALT1) complex and inhibitor of κB (IkB) kinase (FIG. 3). The mannose receptor is an exception and does not activate this pathway^{5,81}.

The role of dectin 1. A receptor for fungal β-glucans, dectin 1 has been studied widely as a crucial receptor for phagocytosis and killing of fungi and the cytokine response to them^{82}. Mouse studies have revealed a protective role for dectin 1 in systemic candidiasis^{82} and in the control of fungi at several mucosal surfaces, such as the skin^{69}, the oral mucosa^{67} and the lungs^{72}. The role of dectin 1 in protection against *Candida* spp. infection seems to depend on the specific fungal strain^{83}, suggesting that some fungi can mask their cell wall β-glucans and can thus escape recognition. In humans, a mutation in *CLEC7A*, the gene encoding dectin 1, leads to low dectin 1 expression and impaired cytokine production upon fungal re-stimulation^{70}. As a result, patients carrying this mutation often suffer from recurrent vulvovaginal infections and onychomycosis, partly owing to defective IL-17 production in response to fungi^{70}.

Dectin 1 might also control antifungal immunity during fungal dysbiosis. We have shown that dectin 1-deficient mice developed more severe colitis than wild-type mice, which was accompanied by mycobiota changes and the overgrowth of opportunistic species from genera such as *Candida* and *Trichosporon*^{16}. This phenotype was dependent on the inability of *Clec7a⁻/⁻* mice to control *Candida* spp. during intestinal inflammation, and treatment with the antifungal drug fluconazole ameliorated colitis^{16,84}. *In situ* examination of mouse colons revealed the absence of hyphal growth in the lower gastrointestinal tract during *Candida tropicalis* colonization of wild-type mice and mice with defects in antifungal immunity^{16} (I.L. and I.D.I., unpublished observations). By contrast, hyphal growth is commonly observed during *Candida* spp. infections at other mucosal sites^{66,69}, and factors involved in hyphal elongation or yeast-to-hypha transition have been shown to aid either fungal invasion (which is mediated by extent of cell elongation 1 (*ECE1*))^{85} or commensalism (which is mediated by enhanced filamentous growth protein 1 (*Efg1*))^{86}. This suggests that colitis in *Clec7a⁻/⁻* mice is not aggravated by a typical fungal infection, but that other aspects of host–fungal interactions that affect the mycobiota might be involved. Consistent with results from these mouse models, patients carrying a two-marker *CLEC7A* haplotype develop severe refractory ulcerative colitis, which supports a possible role for dectin 1 in the severity^{16} of, but not the susceptibility to, intestinal disease^{87}. Furthermore, dectin 1 deficiency is also associated with increased colonization of the gut by *Candida* spp. and the development of graft-versus-host disease in patients who have received transplants^{88,89}. Together, these studies suggest that impaired antifungal immunity, *Candida* spp. overgrowth and fungal dysbiosis might be involved in several intestinal diseases.

The role of dectin 2. Dectin 2 (encoded by *Clec6a*) has been implicated in the induction of T helper 17 (*T~H~*17) responses and protection from systemic candidiasis^{90}; however, its role in mucosal immunity is less well-explored. Dectin 2-mediated immunity might be important for the generation of fungi-specific *T~H~*17 responses to *Fonsecaea pedrosoi* and the resolution of skin infections in a mouse model of chromoblastomycosis^{91}. Immunity to the skin commensal *Malassezia* spp. is also partially controlled by dectin 2 (REF. 92). In the lung, dectin 2 might be involved in the recognition of fungal and mite allergens^{93}, but *Clec6a*-knockout mice do not develop more severe lung inflammation than wild-type mice upon infection with fungi such as *Cryptococcus neoformans*^{94}. The role of dectin 2 in human antifungal mucosal defence is even less clear. A recent study described a polymorphism in *CLEC6A* that is associated with increased susceptibility to pulmonary cryptococcosis^{95}. However, it is uncertain whether dectin 2 is directly responsible for this phenotype.

The role of other CLRs. Other CLRs, such as MINCLE and dectin 3, which induce SYK–CARD9–BCL-10–MALT1-dependent signalling, are also involved in mucosal immunity. Dectin 3 induces the expression of MINCLE upon specific stimulation^{96} and forms heterodimers with dectin 2 to induce signalling, presumably through FcRγ coupling^{97} (FIG. 3), suggesting that these receptors are expressed in an interdependent manner. Dectin 3- and MINCLE-deficient mice are both susceptible to *Candida* spp. systemic infections^{81,90,97}; however, it is less clear whether these receptors are directly involved in mucosal immunity to fungi. MINCLE, for example, can recognize a lipophilic component of *Malassezia* spp. cell wall *in vitro*^{92}, but whether such recognition of commensal *Malassezia* spp. occurs in the skin has not been determined.

Dectin 3, which is encoded by *Clec4d*, has recently been shown to control intestinal inflammation in mice. *Clec4d⁻/⁻* mice developed more severe colitis than wild-type mice, which was associated with an increased *C. tropicalis* burden and was efficiently treated with

Onychomycosis  
A fungal infection of the fingernails or toenails.

Chromoblastomycosis  
A chronic localized infection of the skin and subcutaneous tissue caused by pigmented fungi that contain sclerotic bodies.

REVIEWs

**Phaeohyphomycosis**
A heterogeneous group of fungal infections that are characterized by the presence of pigmented fungal cells.

**Deep-seated dermatophytosis**
A fungal infection of deep keratinized tissue (including skin, hair and claws).

**CLR signalling**

![Diagram](attachment:CLR_signalling_diagram.png)

Figure 3 | **C-type lectin receptor recognition and signalling.** Fungal polysaccharides are recognized by C-type lectin receptors (CLRs) such as macrophage-inducible C-type lectin (MINCLE), dectin 1, dectin 2 and dectin 3, resulting in the activation of spleen tyrosine kinase (SYK). The E3 ubiquitin ligase casitas B-lineage lymphoma-b (CBLB), which ubiquitylates SYK, regulates antifungal immune responses downstream of dectin 1 and dectin 2. CLRs trigger phagocytosis and respiratory burst (production of reactive oxygen species (ROS)) through the SYK-dependent activation of phospholipase Cy2 (PLCy2), which, in turn, activates NADPH phagocyte oxidase. Engagement of protein kinase Cδ (PKCδ) followed by activation of the CARD9–BCL-10–MALT1 complex leads to nuclear factor-κB (NF-κB) activation, caspase 1 activity and the production of cytokines and other mediators that are crucial for host defence. Dectin–SYK induction also promotes IRF5-dependent interferon-β (IFNβ) production. CD23 is a newly identified CLR that is upregulated upon dectin 1 activation and that leads to the production of ROS. The signalling events downstream of CD23 and the mannose receptor (MR) remain unknown (indicated by a question mark). ASC, apoptosis-associated speck-like protein containing a CARD; CARD9, caspase recruitment domain-containing protein 9; COX2, cyclooxygenase 2; FcRγ, Fc receptor common γ-chain; IRF5, interferon-regulatory factor 5; MALT1, mucosa-associated lymphoid tissue lymphoma translocation protein 1; NFAT, nuclear factor of activated T cells; NLRP3, NOD, LRR and Pyrin domain-containing protein 3; PGE2, prostaglandin E2; TNF, tumour necrosis factor.

fluconazole⁹⁸. Decreased numbers of T<sub>H</sub>17 cells, impaired phagocytosis by dectin 3-deficient macrophages and a defective intestinal epithelial cell barrier in these mice can be partly responsible for *C. tropicalis* overgrowth. However, the mechanisms behind increased susceptibility to colitis remain unclear, as levels of several pro-inflammatory cytokines (including tumour necrosis factor) were reduced in the colons of *Clec4d*⁻/⁻ mice⁹⁸. Although the structure of the microbiota was not assessed in this study, augmented fungal burden and the increased relative abundance of *C. tropicalis* in the intestines of *Clec4d*⁻/⁻ mice indicate changes to the fungal community. Finally, the relevance of dectin 3 to human intestinal disease remains uncertain owing to a lack of reports on the genetic associations between *CLEC4D* polymorphisms and IBD.

**The role of CARD9**

CARD9 occupies a key position downstream of several antifungal receptors, such as the CLRs⁸¹ (FIG. 3). As such, CARD9 is crucial for the induction of cytokine production and the triggering of cellular immunity to fungi at several mucosal sites⁸,⁷¹,⁹⁹,¹⁰⁰. Thus, biallelic mutations in *CARD9* have important consequences for the control of mucosal and systemic fungal infections⁸,⁷¹,¹⁰¹. Although CARD9 might be involved in signalling through receptors that sense stimuli other than fungi, mice and humans that are deficient in CARD9 seem to be specifically susceptible to fungal infections⁷¹,⁹⁹–¹⁰¹. CARD9-deficient patients often develop chronic mucocutaneous candidiasis that is characterized by persistent and recurrent *Candida* spp. infections of the mouth, skin, vagina and other mucosal surfaces⁸,⁷¹. CARD9 mutations can also lead to severe onychomycosis, phaeohyphomycosis and deep-seated dermatophytosis, affecting deeper subcutaneous tissue⁸,¹⁰². Fungal infections in these patients are caused by *Trichophyton* spp. and *Phialophora* spp. and can expand to affect the lymph nodes and bones⁸,¹⁰²,¹⁰³. These rare but severe effects of CARD9 deficiency suggest a central role for this adaptor protein in controlling mucosal and systemic immunity to fungal infections in humans.

In addition, CARD9 has been implicated in IBD. A nonsynonymous single nucleotide polymorphism in *CARD9* has been detected with high frequency in both Caucasian (at a frequency of 53%) and African (at a frequency of 25%) populations¹⁰⁴,¹⁰⁵. This polymorphism is strongly associated with the risk of developing Crohn’s disease and ulcerative colitis¹⁰⁶. Using experimental models of colitis, it was shown that CARD9 signalling can be protective against fungi that encounter the intestinal mucosa during intestinal barrier damage¹⁰⁷. *Card9*⁻/⁻ mice had increased antifungal antibodies in serum, and colitis could be partially ameliorated by antifungal treatment, suggesting fungal involvement in the observed phenotype¹⁰⁷. Furthermore, both the fungal and the bacterial gut communities in *Card9*⁻/⁻ mice were altered¹⁰⁸. Interestingly, *Candida* spp. and *Aspergillus* spp. were barely detectable in *Card9*⁻/⁻ mice, whereas the *Sporobolomyces* genus was the most prevalent group of fungi¹⁰⁸, suggesting that *C. tropicalis*, which expands in dectin 1- and dectin 3-deficient animals¹⁶,⁹⁸, was not the main driver of dysbiosis during CARD9 deficiency. Consistently, human mycobiota studies failed to establish a positive correlation between CARD9 polymorphism and *Candida* spp. overgrowth in patients with IBD¹⁷. Finally, apart from the the CARD9 polymorphism reported above¹⁰⁴,¹⁰⁵, no other CARD9 deficiencies in humans have been associated with a predisposition to IBD.

Although some CARD9 mutations are accompanied by a defect in the production of IL-17 (REF. 71), other mutations do not affect IL-17-mediated immunity¹⁰⁹. Mechanistic studies in mice have demonstrated that a subtle tuning and timing of the CLRs–CARD9–IL-17-mediated response to fungal infections might be taking place⁷⁴,¹⁰⁰. Although CARD9 has a clear role in adaptive T<sub>H</sub>17 cell responses to oral *C. albicans* infection,
it was not required for IL-17-dependent innate immunity in this model<sup>100</sup>. In addition, a role of CARD9 in the late stages of mucosal immunity to fungi has been described during *Aspergillus fumigatus* lung infection<sup>74</sup>. Of note, CARD9 has a redundant role in the induction of T<sub>H</sub>17-mediated immunity during systemic infection with other *Candida* species, such as *C. tropicalis*<sup>110</sup>. Whether CARD9 can influence T<sub>H</sub>17 cell responses during mucosal infection with other non-*Candida* opportunistic fungi remains to be elucidated.

### The role of IL-17 and IL-22

Although more studies are needed to define CARD9-dependent and CARD9-independent regulation of mucosal responses to fungi that are mediated by IL-17, the role of this cytokine during mucosal fungal infections has been determined by multiple human genetic studies and mouse experimental studies, which have been summarized in several recent reviews<sup>80,81,111</sup>. Despite the excellent body of experimental work, it is still unclear whether the IL-17 pathway has an effect on commensal fungal communities. A proof-of-concept clinical study exploring IL-17A blockade with secukinumab in patients with Crohn’s disease identified an unexpectedly higher rate of fungal infections among the treated individuals<sup>112</sup>. This adverse outcome correlated with augmented intestinal pathology and suggests a possible role of IL-17 in controlling fungal communities; IL-17 can mediate the production of antimicrobial peptides (AMPs) by epithelial cells, which has been shown to clear *Candida* spp. infection in the oral mucosa<sup>78</sup>. Similar to IL-17, IL-22 acts on epithelial cells as a potent inducer of AMPs and is protective against mucosal *Candida* spp. and *Aspergillus* spp. infections<sup>66,68,72,73</sup>. These barrier-enhancing properties might explain the indispensable role of IL-17 and IL-22 in the protection against fungal infection at the mucosal surfaces and provide a strong rationale for exploring the role of these pathways in relation to fungal communities in the gut.

### Other contributors

Several other molecules, including inflammasome proteins such as NLRP3 and NLRC4, which activate caspase 1 and can process pro-IL-1β into its biologically active form, as well as IL-1β itself, have been explored in the context of mucosal immunity to fungal infections<sup>67,80,113</sup>. NOD, LRR and Pyrin domain-containing protein 3 (NLRP3) and NLR family CARD-containing protein 4 (NLRC4) inflammasomes are protective in mouse models of vaginal *Candida* spp. infection<sup>67,80,114</sup>. Furthermore, genetic polymorphisms in NLRP3 are associated with susceptibility to recurrent vulvovaginal candidiasis, increased IL-1β production and hyper-inflammation<sup>115,116</sup>. Sensing of fungi by NLRP3 inflammasomes seems to be morphotype dependent, with preferential activation by *Candida* spp. hyphae<sup>80</sup>. However, whether these molecules are involved in the control of the mycobiota during the steady state or upon dysbiosis remains unclear.

Despite our growing knowledge of antifungal immunity and increasing numbers of available antifungal drugs, a rapid rise in fungal infections has prompted the search for novel immunomodulatory strategies to induce immunity to fungi. A recent study has demonstrated that the inhibition of JUN N-terminal kinase 1 (JNK1; also known as MAPK8) promotes the expression of CD23 (a recently discovered CLR), the production of nitric oxide, and a potent antifungal effect both *in vitro* and *in vivo*<sup>117</sup>. The E3 ubiquitin ligase casitas B-lineage lymphoma-b (CBLB), which ubiquitylates SYK, was shown to regulate antifungal immune responses downstream of dectin 1 and dectin 2 (REFS 118, 119) (FIG. 3). Genetic deletion or peptide-based inhibition of CBLB promoted a strong antifungal response, enhanced fungal killing and protected mice during systemic and cutaneous *C. albicans* infections<sup>118</sup>. Because of their position downstream of several antifungal receptors, CBLB or JNK1 blockade might be a therapeutic avenue to pursue in individuals that have an increased risk of fungal infections and gastrointestinal complications, such as patients who are immunosuppressed or who are infected with HIV. Finally, several other CLRs, chemokine receptors and pathways have been studied in the context of antifungal immunity; however, their role in mucosal immunity to fungi has not been explored.

#### Interplay between bacteria and fungi

Fungi and bacteria share similar niches on the mucosal surfaces and can undoubtedly influence each other. Interactions between fungi and bacteria can occur directly through physical contact and secreted molecules, or indirectly through the alteration of the host immune response. Several studies have attempted to identify interkingdom and intrakingdom correlations between fungi and bacteria using next-generation sequencing<sup>14,17,23</sup>. However, this approach needs to be experimentally validated, as correlations between relative abundances are at high risk of identifying false associations<sup>120</sup>.

Both clinical studies<sup>21</sup> and mouse models<sup>13,54</sup> suggest that fungal colonization is modulated by bacterial communities. Short-term antibiotic treatment is sufficient to promote persistent *Candida* spp. colonization in the mouse gut<sup>13,54</sup>, with immunological effects at distant sites such as the lung<sup>54,55</sup>. Although antibiotic treatment rarely promotes the outgrowth of *Candida* spp. in the oral mucosa<sup>121</sup>, antibiotic-induced vaginal candidiasis is relatively common, suggesting a mucosal site-specific nature of bacterial–fungal interplay. Lactobacilli-induced reduction of pH or the production of a bacteriocin-like compound can inhibit the growth of *Candida* spp. and are both essential barriers to the colonization of the vaginal mucosa by several fungal species<sup>31,122</sup>. In addition to lactobacilli, other bacterial species, including *Pseudomonas* and *Burkholderia*, as well as bacterial metabolites such as butyrate, can inhibit *Candida* yeast-to-hypha transition — a process required for the formation of resistant biofilms of fungi and bacteria<sup>123,124</sup>. However, fungal communities can adapt to the selective pressure exerted by the host and their bacterial neighbours. Analysis of the relatively stable fungal communities in the cystic fibrosis lung, suggests that *Candida* spp. clones were resistant to the filamentation-repressive effects of

Secukinumab  
A human monoclonal antibody that binds to IL-17A and inhibits its functions.
REVIEWS

Pseudomonas aeruginosa through a mutation in the repressive gene NRG1 (REF. 51). Although the formation of biofilms might contribute to the morbidity of both oral and vaginal candidiasis^{32,124}, such biofilms have not been detected in the gut^{16,125}, which is probably owing to the inability of *Candida* to switch to hyphal growth in this environment. According to a recent study, *C. albicans* adopts a third GUT (gastrointestinally induced transition) morphology in the intestines that cannot induce systemic disease and that is metabolically accustomed to the intestinal environment^{125}.

In addition to *Candida* spp., other fungal species can compete for establishment in an environmental niche. Compounds that are secreted by *Pichia* spp. can inhibit the *in vitro* growth of *Candida* spp., *Aspergillus* spp. and *Fusarium* spp., the formation of biofilms by *Candida* spp., and can reduce fungal burden in a mouse model of oral candidiasis^{32}. Similarly, bacteria can indirectly affect fungal colonization through the modulation of the host immune response. For example, in mice, *Bacteroides thetaiotaomicron* prevents *C. albicans* colonization of the gut via the HIF1α-mediated production of the antimicrobial peptide CRAMP^{126}.

The host immune response to fungi might, in turn, indirectly influence bacteria in the gut. A recent study demonstrated that, in the absence of commensal *Candida* spp., *Clec7a⁻/⁻* mice are protected from colitis through a mechanism that involves the intestinal bacteria^{84}. The authors found that, in the absence of *Candida* spp., antimicrobial peptides targeting Gram-positive bacteria were reduced in the colons of *Clec7a⁻/⁻* mice, leading to an increase in commensal *Lactobacillus murinus* that induced regulatory T cell expansion and protection from colitis. However, upon colonization with *C. tropicalis*, this potentially beneficial effect was lost and *Clec7a⁻/⁻* mice developed more severe intestinal inflammation^{84}. Similarly, *Card9⁻/⁻* mice that are susceptible to fungi^{99,107} also carry altered populations of tryptophan-metabolizing bacteria, including reduced levels of lactobacilli^{108}. This correlated with decreased levels of aryl hydrocarbon receptor (AHR) ligands and defective *Il22*, *Reg3g* and *Reg3b* expression in the colons of *Card9⁻/⁻* mice that could be rescued by supplementation with lactobacilli^{108}. The protective role of lactobacilli through the production of AHR ligands and the stimulation of IL-22 release by ILCs has been previously described as a mechanism that promotes *Candida* spp. colonization resistance in the stomach^{127}. Whether fungal and bacterial communities influence each other during these cases of apparent interkingdom dysbiosis in a genetically susceptible host remains unclear. Nevertheless, such studies suggest a more complex microbial interkingdom relationship and possibly a dual role for antifungal immunity pathways in the control of bacterial and fungal populations in the gut.

Finally, the specific disruption of the healthy gut fungal community can also have an adverse effect on host health. We and others^{24,55} have recently demonstrated that prolonged antifungal treatment of healthy wild-type mice exacerbated immune-mediated disease in several experimental models of colitis and lung allergy, leading to the expansion of neutrophils, monocytes, T~H~1 cells and T~H~17 cells (which was shown in a colitis model), or eosinophils, T~H~2 cells and IgE-producing B cells (which was shown in a lung allergy model (FIG. 2b))^{24,55}. Analysis of the mycobiota in mice treated with fluconazole and amphotericin B revealed dramatic changes in the composition and diversity of the gut mycobiota, suggesting the induction of gut fungal community dysbiosis^{24}. Oral supplementation with *Aspergillus amstelodami*, *Epicoccum nigrum* and *Wallemia sebi*, which expanded during fungal dysbiosis, was sufficient to recapitulate the detrimental effects of antifungal drugs on inflammatory disease. In addition to the mycobiota, drug-induced fungal dysbiosis may affect bacterial community structure and genera, such as *Bacteroides*, *Clostridium* and *Lactobacillus*, which are also decreased during fungal dysbiosis^{25}.

### Concluding remarks

An increasing number of studies are detailing the complexity of antifungal immune responses, but most evidence focuses on fungal infections. Increasing evidence indicates that human barrier surfaces harbour diverse communities of fungi that cohabit with the host throughout its life. Although opportunistic fungi within those communities can cause pathologies in susceptible individuals, most host–fungal interactions at the body surfaces are non-infectious. This raises several questions: do commensal fungi, similar to bacteria, contribute to shaping and maintaining immune homeostasis? Which pathways and intercellular networks are involved in sensing commensal fungi during the steady state? The characterization of immune responses in the presence of dysbiotic and steady-state fungal communities might provide key answers. Future studies on the crosstalk between the mycobiota, fungal metabolites, epithelium and mucosa-resident immune cells will propel the field towards understanding how the mycobiota interacts with host immunity at barrier surfaces. Understanding the mechanisms behind fungal dysbiosis might provide diagnostic and therapeutic targets by identifying fungi that could be or that should not be targeted during inflammatory conditions.

Interkingdom microbiome interactions involving fungi can also indirectly modulate immunity by affecting microbiome function and metabolism. The application of metagenomics, metatranscriptomics and metabolomics holds promise for identifying the mediators of interkingdom interactions; however, the use of experimental approaches to validate the mediators remains crucial. The fast-growing field of antifungal immunity has entered a new age in which a focus on complex microbial communities rather than a single fungal species will shed light on the role of the mycobiota in homeostasis and inflammation. Collectively, an increasing body of evidence suggests crosstalk between gut fungi, bacteria and the host mucosal immunity. Disruption of this network can contribute to interkingdom microbial community alterations with detrimental consequences to the host and thus positions fungi in the crossfire between mucosal immunity and commensal bacteria.

1. Peay, K. G., Kennedy, P. G. & Talbot, J. M. Dimensions of biodiversity in the Earth mycobiome. *Nat. Rev. Microbiol.* **14**, 434–447 (2016).
2. Remy, W., Taylor, T. N., Hass, H. & Kerp, H. Four hundred-million-year-old vesicular arbuscular mycorrhizae. *Proc. Natl Acad. Sci. USA* **91**, 11841–11843 (1994).
3. Wainright, P. O., Hinkle, G., Sogin, M. L. & Stickel, S. K. Monophyletic origins of the metazoa: an evolutionary link with fungi. *Science* **260**, 340–342 (1993).
4. Casadevall, A. Fungi and the rise of mammals. *PLoS Pathog.* **8**, e1002808 (2012).
5. Brown, G. D. & Gordon, S. Fungal beta-glucans and mammalian immunity. *Immunity* **19**, 311–315 (2003).
6. Fisher, M. C. *et al.* Emerging fungal threats to animal, plant and ecosystem health. *Nature* **484**, 186–194 (2012).
7. Brown, G. D. *et al.* Hidden killers: human fungal infections. *Sci. Transl Med.* **4**, 165rv13 (2012).
8. Lanternier, F. *et al.* Deep dermatophytosis and inherited CARD9 deficiency. *N. Engl. J. Med.* **369**, 1704–1714 (2013).
9. Findley, K. *et al.* Topographic diversity of fungal and bacterial communities in human skin. *Nature* **498**, 367–370 (2013). This paper describes the presence and diversity of *Malassezia* species in the human skin and is by far the most comprehensive culture-independent study of the skin mycobiome.
10. Brune, A. Symbiotic digestion of lignocellulose in termite guts. *Nat. Rev. Microbiol.* **12**, 168–180 (2014).
11. Solomon, K. V. *et al.* Early-branching gut fungi possess a large, comprehensive array of biomass-degrading enzymes. *Science* **351**, 1192–1195 (2016).
12. Belkaid, Y. & Harrison, O. J. Homeostatic immunity and the microbiota. *Immunity* **46**, 562–576 (2017).
13. Dollive, S. *et al.* Fungi of the murine gut: episodic variation and proliferation during antibiotic treatment. *PLoS ONE* **8**, e71806 (2013).
14. Hoffmann, C. *et al.* Archaea and fungi of the human gut microbiome: correlations with diet and bacterial residents. *PLoS ONE* **8**, e66019 (2013).
15. David, L. A. *et al.* Diet rapidly and reproducibly alters the human gut microbiota. *Nature* **505**, 559–563 (2014). This study shows that the human gut microbiota, including fungi, bacteria and viruses, changes rapidly in response to a diet consisting entirely of either animal products or plant products.
16. Ilié, I. D. *et al.* Interactions between commensal fungi and the C-type lectin receptor dectin-1 influence colitis. *Science* **336**, 1314–1317 (2012).
17. Sokol, H. *et al.* Fungal microbiota dysbiosis in IBD. *Gut* **66**, 1039–1048 (2016). This study used next-generation sequencing to simultaneously characterize the faecal fungal and bacterial microbiota in patients with ulcerative colitis and Crohn’s disease.
18. Metchnikoff, E. *The Prolongation of Life: Optimistic Studies* 161–183 (G. P. Putnam’s Sons, 1908).
19. Hsiao, E. Y. *et al.* Microbiota modulate behavioral and physiological abnormalities associated with neurodevelopmental disorders. *Cell* **155**, 1451–1463 (2013).
20. Marsland, B. J. & Gollwitzer, E. S. Host-microorganism interactions in lung diseases. *Nat. Rev. Immunol.* **14**, 827–835 (2014).
21. Lewis, J. D. *et al.* Inflammation, antibiotics, and diet as environmental stressors of the gut microbiota in pediatric Crohn’s disease. *Cell Host Microbe* **18**, 489–500 (2015). This article shows that inflammation, antibiotics and diet independently affect the gut microbiota in patients with Crohn’s disease, and identifies an association between antibiotic use and increased fungal load.
22. Liguori, G. *et al.* Fungal dysbiosis in mucosa-associated microbiota of Crohn’s disease patients. *J. Crohns Colitis* **10**, 296–305 (2016).
23. Hoarau, G. *et al.* Bacteriome and mycobiome interactions underscore microbial dysbiosis in familial Crohn’s disease. *mBio* **7**, e01250-16 (2016).
24. Wheeler, M. L. *et al.* Immunological consequences of intestinal fungal dysbiosis. *Cell Host Microbe* **19**, 865–873 (2016). This study shows that targeted fungal community dysbiosis in the mouse gut can have local and systemic effects on immunity and inflammation.
25. Ghannoum, M. A. *et al.* Characterization of the oral fungal microbiome (mycobiome) in healthy individuals. *PLoS Pathog.* **6**, e1000713 (2010).
26. Bittinger, K. *et al.* Improved characterization of medically relevant fungi in the human respiratory tract using next-generation sequencing. *Genome Biol.* **15**, 487 (2014).
27. Lindahl, B. D. *et al.* Fungal community analysis by high-throughput sequencing of amplified markers — a user’s guide. *New Phytol.* **199**, 288–299 (2013).
28. Tang, J., Iliev, I. D., Brown, J., Underhill, D. M. & Funari, V. A. Mycobiome: approaches to analysis of intestinal fungi. *J. Immunol. Methods* **421**, 112–121 (2015).
29. Underhill, D. M. & Iliev, I. D. The mycobiota: interactions between commensal fungi and the host immune system. *Nat. Rev. Immunol.* **14**, 405–416 (2014).
30. Motooka, D. *et al.* Fungal ITS1 deep-sequencing strategies to reconstruct the composition of a 26-species community and evaluation of the gut mycobiota of healthy Japanese individuals. *Front. Microbiol.* **8**, 238 (2017).
31. Krasner, R. I., Young, G. & Yudkofsky, P. L. Interactions of oral strains of *Candida albicans* and lactobacilli. *J. Bacteriol.* **72**, 525–529 (1956).
32. Mukherjee, P. K. *et al.* Oral mycobiome analysis of HIV-infected patients: identification of *Pichia* as an antagonist of opportunistic fungi. *PLoS Pathog.* **10**, e1003996 (2014).
33. Hallen-Adams, H. E. & Suhr, M. J. Fungi in the healthy human gastrointestinal tract. *Virulence* **8**, 352–358 (2017).
34. Suhr, M. J., Banjara, N. & Hallen-Adams, H. E. Sequence-based methods for detecting and evaluating the human gut mycobiome. *Lett. Appl. Microbiol.* **62**, 209–215 (2016).
35. Cutler, J. E., Deepe, G. S. Jr & Klein, B. S. Advances in combating fungal diseases: vaccines on the threshold. *Nat. Rev. Microbiol.* **5**, 13–28 (2007).
36. Luan, C. *et al.* Dysbiosis of fungal microbiota in the intestinal mucosa of patients with colorectal adenomas. *Sci. Rep.* **5**, 7980 (2015).
37. Li, Q. *et al.* Dysbiosis of gut fungal microbiota is associated with mucosal inflammation in Crohn’s disease. *J. Clin. Gastroenterol.* **48**, 513–523 (2014).
38. Kirsner, J. B. Historical origins of current IBD concepts. *World J. Gastroenterol.* **7**, 175–184 (2001).
39. Tamboli, C. P., Neut, C., Desreumaux, P. & Colombel, J. F. Dysbiosis in inflammatory bowel disease. *Gut* **53**, 1–4 (2004).
40. Ott, S. J. *et al.* Fungi and inflammatory bowel diseases: alterations of composition and diversity. *Scand. J. Gastroenterol.* **43**, 831–841 (2008).
41. Oiu, X. *et al.* Changes in the composition of intestinal fungi and their role in mice with dextran sulfate sodium-induced colitis. *Sci. Rep.* **5**, 10416 (2015).
42. Chehoud, C. *et al.* Fungal signature in the gut microbiota of pediatric patients with inflammatory bowel disease. *Inflamm. Bowel Dis.* **21**, 1948–1956 (2015).
43. Israeli, E. *et al.* Anti-*Saccharomyces cerevisiae* and antineutrophil cytoplasmic antibodies as predictors of inflammatory bowel disease. *Gut* **54**, 1232–1236 (2005).
44. Standaert-Vitse, A. *et al.* *Candida albicans* colonization and ASCA in familial Crohn’s disease. *Am. J. Gastroenterol.* **104**, 1745–1753 (2009).
45. Schaffer, T. *et al.* Anti-*Saccharomyces cerevisiae* mannan antibodies (ASCA) of Crohn’s patients crossreact with mannan from other yeast strains, and murine ASCA IgM can be experimentally induced with *Candida albicans*. *Inflamm. Bowel Dis.* **13**, 1339–1346 (2007).
46. Sharon, G., Sampson, T. R., Geschwind, D. H. & Mazmanian, S. K. The central nervous system and the gut microbiome. *Cell* **167**, 915–932 (2016).
47. Strati, F. *et al.* Altered gut microbiota in Rett syndrome. *Microbiome* **4**, 41 (2016).
48. Strati, F. *et al.* New evidences on the altered gut microbiota in autism spectrum disorders. *Microbiome* **5**, 24 (2017).
49. Charlson, E. S. *et al.* Lung-enriched organisms and aberrant bacterial and fungal respiratory microbiota after lung transplant. *Am. J. Respir. Crit. Care Med.* **186**, 536–545 (2012).
50. Delhaes, L. *et al.* The airway microbiota in cystic fibrosis: a complex fungal and bacterial community — implications for therapeutic management. *PLoS ONE* **7**, e36313 (2012).
51. Kim, S. H. *et al.* Global analysis of the fungal microbiome in cystic fibrosis patients reveals loss of function of the transcriptional repressor Nrg1 as a mechanism of pathogen adaptation. *PLoS Pathog.* **11**, e1005308 (2015). This paper shows the presence of stable fungal communities in the lungs of patients with cystic fibrosis and identifies mutations in the transcriptional repressor *NRG1* in *C. albicans* that seem to be a common mechanism of adaptation.
52. Willger, S. D. *et al.* Characterization and quantification of the fungal microbiome in serial samples from individuals with cystic fibrosis. *Microbiome* **2**, 40 (2014).
53. van Woerden, H. C. *et al.* Differences in fungi present in induced sputum samples from asthma patients and non-atopic controls: a community based case control study. *BMC Infect. Dis.* **13**, 69 (2013).
54. Noverr, M. C., Noggle, R. M., Toews, G. B. & Huffnagle, G. B. Role of antibiotics and fungal microbiota in driving pulmonary allergic responses. *Infect. Immun.* **72**, 4996–5003 (2004).
55. Kim, Y. G. *et al.* Gut dysbiosis promotes M2 macrophage polarization and allergic airway inflammation via fungi-induced PGE2. *Cell Host Microbe* **15**, 95–102 (2014).
56. Farr, A. *et al.* Effect of asymptomatic vaginal colonization with *Candida albicans* on pregnancy outcome. *Acta Obstet. Gynecol. Scand.* **94**, 989–996 (2015).
57. Drell, T. *et al.* Characterization of the vaginal micro- and mycobiome in asymptomatic reproductive-age Estonian women. *PLoS ONE* **8**, e54379 (2013).
58. Merenstein, D. *et al.* Colonization by *Candida* species of the oral and vaginal mucosa in HIV-infected and noninfected women. *AIDS Res. Hum. Retroviruses* **29**, 30–34 (2013).
59. Guo, R. *et al.* Increased diversity of fungal flora in the vagina of patients with recurrent vaginal candidiasis and allergic rhinitis. *Microb. Ecol.* **64**, 918–927 (2012).
60. Oh, J. *et al.* Temporal stability of the human skin microbiome. *Cell* **165**, 854–866 (2016).
61. Oh, J. *et al.* The altered landscape of the human skin microbiome in patients with primary immunodeficiencies. *Genome Res.* **23**, 2103–2114 (2013).
62. Smekens, S. P. *et al.* Skin microbiome imbalance in patients with STAT1/STAT3 defects impairs innate host defense responses. *J. Innate Immun.* **6**, 253–262 (2014).
63. Kalan, L. *et al.* Redefining the chronic-wound microbiome: fungal communities are prevalent, dynamic, and associated with delayed healing. *mBio* **7**, e01058-16 (2016). This study shows that fungal communities in diabetic foot ulcers are associated with poor outcomes and longer healing times.
64. Strati, F. *et al.* Age and gender affect the composition of fungal population of the human gastrointestinal tract. *Front. Microbiol.* **7**, 1227 (2016).
65. Jo, J. H. *et al.* Diverse human skin fungal communities in children converge in adulthood. *J. Invest. Dermatol.* **136**, 2356–2363 (2016).
66. Conti, H. R. *et al.* Th17 cells and IL-17 receptor signaling are essential for mucosal host defense against oral candidiasis. *J. Exp. Med.* **206**, 299–311 (2009).
67. Hise, A. G. *et al.* An essential role for the NLRP3 inflammasome in host defense against the human fungal pathogen *Candida albicans*. *Cell Host Microbe* **5**, 487–497 (2009).
68. De Luca, A. *et al.* IL-22 and IDO1 affect immunity and tolerance to murine and human vaginal candidiasis. *PLoS Pathog.* **9**, e1003486 (2013).
69. Kashem, S. W. *et al.* *Candida albicans* morphology and dendritic cell subsets determine T helper cell differentiation. *Immunity* **42**, 356–366 (2015).
70. Ferwerda, B. *et al.* Human dectin-1 deficiency and mucocutaneous fungal infections. *N. Engl. J. Med.* **361**, 1760–1767 (2009).
71. Glocker, E. O. *et al.* A homozygous CARD9 mutation in a family with susceptibility to fungal infections. *N. Engl. J. Med.* **361**, 1727–1735 (2009).
72. Gessner, M. A. *et al.* Dectin-1-dependent interleukin-22 contributes to early innate lung defense against *Aspergillus fumigatus*. *Infect. Immun.* **80**, 410–417 (2012).
73. Puel, A. *et al.* Autoantibodies against IL-17A, IL-17F, and IL-22 in patients with chronic mucocutaneous candidiasis and autoimmune polyendocrine syndrome type I. *J. Exp. Med.* **207**, 291–297 (2010).

REVIEWS

74. Jhingran, A. *et al.* Compartment-specific and sequential role of MyD88 and CARD9 in chemokine induction and innate defense during respiratory fungal infection. *PLoS Pathog.* **11**, e1004589 (2015).

75. Rivera, A. *et al.* Dectin-1 diversifies *Aspergillus fumigatus*-specific T cell responses by inhibiting T helper type 1 CD4 T cell differentiation. *J. Exp. Med.* **208**, 369–381 (2011).

76. Trautwein-Weidner, K. *et al.* Antigen-specific Th17 cells are primed by distinct and complementary dendritic cell subsets in oropharyngeal candidiasis. *PLoS Pathog.* **11**, e1005164 (2015).

77. Lionakis, M. S. *et al.* CX3CR1-dependent renal macrophage survival promotes *Candida* control and host survival. *J. Clin. Invest.* **123**, 5035–5051 (2013).

78. Conti, H. R. *et al.* IL-17 receptor signaling in oral epithelial cells is critical for protection against oropharyngeal candidiasis. *Cell Host Microbe* **20**, 606–617 (2016).

79. Mowat, A. M. & Agace, W. W. Regional specialization within the intestinal immune system. *Nat. Rev. Immunol.* **14**, 667–685 (2014).

80. Netea, M. G., Joosten, L. A., van der Meer, J. W., Kullberg, B. J. & van de Veerdonk, F. L. Immune defence against *Candida* fungal infections. *Nat. Rev. Immunol.* **15**, 630–642 (2015).

81. Plato, A., Hardison, S. E. & Brown, G. D. Pattern recognition receptors in antifungal immunity. *Semin. Immunopathol.* **37**, 97–106 (2015).

82. Taylor, P. R. *et al.* Dectin-1 is required for beta-glucan recognition and control of fungal infection. *Nat. Immunol.* **8**, 31–38 (2007).

83. Marakalala, M. J. *et al.* Differential adaptation of *Candida albicans* in vivo modulates immune recognition by dectin-1. *PLoS Pathog.* **9**, e1003315 (2013).

84. Tang, C. *et al.* Inhibition of dectin-1 signaling ameliorates colitis by inducing *Lactobacillus*-mediated regulatory T cell expansion in the intestine. *Cell Host Microbe* **18**, 183–197 (2015).

85. Moyes, D. L. *et al.* Candidalysin is a fungal peptide toxin critical for mucosal infection. *Nature* **532**, 64–68 (2016). This study identifies a *C. albicans* toxin that directly disrupts the epithelial layer, triggers a danger response and leads to epithelial activation.

86. Pierce, J. V. & Kumamoto, C. A. Variation in *Candida albicans* EFG1 expression enables host-dependent changes in colonizing fungal populations. *mBio* **3**, e00117-12 (2012).

87. de Vries, H. S. *et al.* Genetic association analysis of the functional c.714T>G polymorphism and mucosal expression of dectin-1 in inflammatory bowel disease. *PLoS ONE* **4**, e7818 (2009).

88. Plantinga, T. S. *et al.* Early stop polymorphism in human DECTIN-1 is associated with increased *Candida* colonization in hematopoietic stem cell transplant recipients. *Clin. Infect. Dis.* **49**, 724–732 (2009).

89. van der Velden, W. J. *et al.* Role of the mycobiome in human acute graft-versus-host disease. *Biol. Blood Marrow Transplant.* **19**, 329–332 (2013).

90. Saijo, S. *et al.* Dectin-2 recognition of alpha-mannans and induction of Th17 cell differentiation is essential for host defense against *Candida albicans*. *Immunity* **32**, 681–691 (2010).

91. Wuthrich, M. *et al.* Fonsecaea pedrosoi-induced Th17-cell differentiation in mice is fostered by dectin-2 and suppressed by mincle recognition. *Eur. J. Immunol.* **45**, 2542–2552 (2015).

92. Ishikawa, T. *et al.* Identification of distinct ligands for the C-type lectin receptors mincle and dectin-2 in the pathogenic fungus Malassezia. *Cell Host Microbe* **13**, 477–488 (2013).

93. Barrett, N. A., Maekawa, A., Rahman, O. M., Austen, K. F. & Kanaoka, Y. Dectin-2 recognition of house dust mite triggers cysteinyl leukotriene generation by dendritic cells. *J. Immunol.* **182**, 1119–1128 (2009).

94. Nakamura, Y. *et al.* Dectin-2 deficiency promotes Th2 response and mucin production in the lungs after pulmonary infection with *Cryptococcus neoformans*. *Infect. Immun.* **83**, 671–681 (2015).

95. Hu, X. P. *et al.* Dectin-2 polymorphism associated with pulmonary cryptococcosis in HIV-uninfected Chinese patients. *Med. Mycol.* **53**, 810–816 (2015).

96. Zhao, X. Q. *et al.* C-Type lectin receptor dectin-3 mediates trehalose 6,6′-dimycolate (TDM)-induced mincle expression through CARD9/Bcl10/MALT1-dependent nuclear factor (NF)-kappaB activation. *J. Biol. Chem.* **289**, 30052–30062 (2014).

97. Zhu, L. L. *et al.* C-Type lectin receptors dectin-3 and dectin-2 form a heterodimeric pattern-recognition receptor for host defense against fungal infection. *Immunity* **39**, 324–334 (2013).

98. Wang, T. *et al.* Dectin-3 deficiency promotes colitis development due to impaired antifungal innate immune responses in the gut. *PLoS Pathog.* **12**, e1005662 (2016).

99. LeibundGut-Landmann, S. *et al.* Syk- and CARD9-dependent coupling of innate immunity to the induction of T helper cells that produce interleukin 17. *Nat. Immunol.* **8**, 630–638 (2007).

100. Bishu, S. *et al.* The adaptor CARD9 is required for adaptive but not innate immunity to oral mucosal *Candida albicans* infections. *Infect. Immun.* **82**, 1173–1180 (2014).

101. Drummond, R. A. *et al.* CARD9-dependent neutrophil recruitment protects against fungal invasion of the central nervous system. *PLoS Pathog.* **11**, e1005293 (2015).

102. Wang, X. *et al.* CARD9 mutations linked to subcutaneous phaeohyphomycosis and TH17 cell deficiencies. *J. Allergy Clin. Immunol.* **133**, 905–908.e3 (2014).

103. Lanternier, F. *et al.* Inherited CARD9 deficiency in otherwise healthy children and adults with *Candida* species-induced meningoencephalitis, colitis, or both. *J. Allergy Clin. Immunol.* **135**, 1558–1568.e2 (2015).

104. International HapMap Consortium *et al.* A second generation human haplotype map of over 3.1 million SNPs. *Nature* **449**, 851–861 (2007).

105. Frazer, K. A. *et al.* A second generation human haplotype map of over 3.1 million SNPs. *Nature* **449**, 851–861 (2007).

106. Khor, B., Gardet, A. & Xavier, R. J. Genetics and pathogenesis of inflammatory bowel disease. *Nature* **474**, 307–317 (2011).

107. Sokol, H. *et al.* Card9 mediates intestinal epithelial cell restitution, T-helper 17 responses, and control of bacterial infection in mice. *Gastroenterology* **145**, 591–601.e3 (2013).

108. Lamas, B. *et al.* CARD9 impacts colitis by altering gut microbiota metabolism of tryptophan into aryl hydrocarbon receptor ligands. *Nat. Med.* **22**, 598–605 (2016).

109. Drummond, R. A. & Lionakis, M. S. Mechanistic insights into the role of C-type lectin receptor/CARD9 signaling in human antifungal immunity. *Front. Cell. Infect. Microbiol.* **6**, 39 (2016).

110. Whibley, N. *et al.* Delinking CARD9 and IL-17: CARD9 protects against *Candida tropicalis* infection through a TNF-α-dependent, IL-17-independent mechanism. *J. Immunol.* **195**, 3781–3792 (2015).

111. Conti, H. R. & Gaffen, S. L. IL-17-mediated immunity to the opportunistic fungal pathogen *Candida albicans*. *J. Immunol.* **195**, 780–788 (2015).

112. Hueber, W. *et al.* Secukinumab, a human anti-IL-17A monoclonal antibody, for moderate to severe Crohn’s disease: unexpected results of a randomised, double-blind placebo-controlled trial. *Gut* **61**, 1693–1700 (2012).

113. Borghi, M. *et al.* Pathogenic NLRP3 inflammasome activity during *Candida* infection is negatively regulated by IL-22 via activation of NLRC4 and IL-1Ra. *Cell Host Microbe* **18**, 198–209 (2015).

114. Tomalka, J. *et al.* A novel role for the NLRC4 inflammasome in mucosal defenses against the fungal pathogen *Candida albicans*. *PLoS Pathog.* **7**, e1002379 (2011).

115. Lev-Sagie, A. *et al.* Polymorphism in a gene coding for the inflammasome component NALP3 and recurrent vulvovaginal candidiasis in women with vulvar vestibulitis syndrome. *Am J. Obstet. Gynecol.* **200**, 303.e1–303.e6 (2009).

116. Jaeger, M. *et al.* Association of a variable number tandem repeat in the NLRP3 gene in women with susceptibility to RVVC. *Eur. J. Clin. Microbiol. Infect. Dis.* **35**, 797–801 (2016).

117. Zhao, X. *et al.* JNK1 negatively controls antifungal innate immunity by suppressing CD23 expression. *Nat. Med.* **23**, 337–346 (2017).

118. Wirnsberger, G. *et al.* Inhibition of CBLB protects from lethal *Candida albicans* sepsis. *Nat. Med.* **22**, 915–923 (2016).

119. Xiao, Y. *et al.* Targeting CBLB as a potential therapeutic approach for disseminated candidiasis. *Nat. Med.* **22**, 906–914 (2016).

120. Friedman, J. & Alm, E. J. Inferring correlation networks from genomic survey data. *PLoS Comput. Biol.* **8**, e1002687 (2012).

121. Wolfson, S. A. Black hairy tongue associated with penicillin therapy. *J. Am. Med. Assoc.* **140**, 1206–1208 (1949).

122. Boris, S. & Barbes, C. Role played by lactobacilli in controlling the population of vaginal pathogens. *Microbes Infect.* **2**, 543–546 (2000).

123. Noverr, M. C. & Huffnagle, G. B. Regulation of *Candida albicans* morphogenesis by fatty acid metabolites. *Infect. Immun.* **72**, 6206–6210 (2004).

124. Lynch, A. S. & Robertson, G. T. Bacterial and fungal biofilm infections. *Annu. Rev. Med.* **59**, 415–428 (2008).

125. Pande, K., Chen, C. & Noble, S. M. Passage through the mammalian gut triggers a phenotypic switch that promotes *Candida albicans* commensalism. *Nat. Genet.* **45**, 1088–1091 (2013). This study shows that in the gut, *C. albicans* adopts a commensal form induced through downregulation of virulence factors and induction of metabolic functions.

126. Fan, D. *et al.* Activation of HIF-1α and LL-37 by commensal bacteria inhibits *Candida albicans* colonization. *Nat. Med.* **21**, 808–814 (2015). This work shows that commensal bacteria can confer resistance to *C. albicans* by promoting the HIF1α-mediated expression of the antimicrobial peptide LL-37.

127. Zelante, T. *et al.* Tryptophan catabolites from microbiota engage aryl hydrocarbon receptor and balance mucosal reactivity via interleukin-22. *Immunity* **39**, 372–385 (2013).

128. de Bekker, C. *et al.* Species-specific ant brain manipulation by a specialized fungal parasite. *BMC Evol. Biol.* **14**, 166 (2014).

129. Lin, C. H. *et al.* Identification of a major epitope by anti-interferon-gamma autoantibodies in patients with mycobacterial disease. *Nat. Med.* **22**, 994–1001 (2016).

130. Sterkel, A. K. *et al.* Fungal mimicry of a mammalian aminepeptidase disables innate immunity and promotes pathogenicity. *Cell Host Microbe* **19**, 361–374 (2016). This study shows that the pathogenic yeast *Blastomyces dermatitidis* produces a close mimic of a mammalian ectopeptidase, which interferes with innate antifungal responses in the lung.

131. Poulsen, M. & Boomsma, J. J. Mutualistic fungi control crop diversity in fungus-growing ants. *Science* **307**, 741–744 (2005).

132. Nelson, K. E. & Williams, C. M. *Infectious Disease Epidemiology: Theory and Practice* 3rd edn (Jones & Bartlett Learning, 2014).

133. Lane, N. The unseen world: reflections on Leeuwenhoek (1677) ‘Concerning little animals’. *Phil. Trans. R. Soc. B Biol. Sci.* **370**, 20140344 (2015).

134. Pasteur, L. & Illo, J. Pasteur and rabies: an interview of 1882. *Med. Hist.* **40**, 373–377 (1996).

135. Henle, J. *On Miasmata and Contagia* (Johns Hopkins Press, 1938).

136. Koch, R. An address on bacteriological research. *Br. Med. J.* **2**, 380–383 (1890).

137. Evans, A. S. Causation and disease: the Henle–Koch postulates revisited. *Yale J. Biol. Med.* **49**, 175–195 (1976).

138. Fredricks, D. N. & Relman, D. A. Sequence-based identification of microbial pathogens: a reconsideration of Koch’s postulates. *Clin. Microbiol. Rev.* **9**, 18–33 (1996).

139. Byrd, A. L. & Segre, J. A. Infectious disease. Adapting Koch’s postulates. *Science* **351**, 224–226 (2016).

140. Olesen, S. W. & Alm, E. J. Dysbiosis is not an answer. *Nat. Microbiol.* **1**, 16228 (2016).

141. Petersen, C. & Round, J. L. Defining dysbiosis and its influence on host immunity and disease. *Cell. Microbiol.* **16**, 1024–1033 (2014).


Acknowledgements
The authors would like to thank members of the Iliiev laboratory and the New York Host-Mycobiota Group for helpful suggestions related to the manuscript. This work was funded by the US National Institutes of Health (grants DK098310 and AI123819 to I.D.I.), Kenneth Rainin Foundation (Innovator and Breakthrough awards to I.D.I.), Swiss National Science Foundation (fellowship P2ZHP3_164850 to I.L.) and support from the Jill Roberts Institute for Research in IBD. The authors apologize to all the contributors to this field whose work could not be cited owing to space limitations.


Competing interests statement
The authors declare no competing interests.


Publisher’s note
Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.
