
Plant Physiology Preview. Published on July 28, 2017, as DOI:10.1104/pp.17.00501

Short title: 16α-Hydroxylase in Glycoalkaloid Biosynthesis (45 characters and spaces)

Corresponding author details:

Correspondence should be addressed to Masaharu Mizutani;

Address: Graduate School of Agricultural Science, Kobe University, Rokkoudai 1-1, Nada-ku, Kobe, Hyogo 657-8501, Japan.

TEL, +81-78-803-5885; Fax, +81-78-803-5885; E-mail, mizutani@gold.kobe-u.ac.jp

Article Title: A Dioxygenase Catalyzes Steroid 16α-Hydroxylation in Steroidal Glycoalkaloid Biosynthesis

Masaru Nakayasu<sup>a</sup>, Naoyuki Umemoto<sup>b,c</sup>, Kiyoshi Ohyama<sup>d</sup>, Yoshinori Fujimoto<sup>d</sup>, Hyoung Jae Lee<sup>a</sup>, Bunta Watanabe<sup>e</sup>, Toshiya Muranaka<sup>f</sup>, Kazuki Saito<sup>b,g</sup>, Yukihiro Sugimoto<sup>a</sup>, and Masaharu Mizutani<sup>a*</sup>

<sup>a</sup>Graduate School of Agricultural Science, Kobe University, Rokkoudai 1-1, Nada-ku, Kobe, Hyogo 657-8501, Japan.

Tel: +81-78-803-5885; email: nakayasumasaru@yahoo.co.jp

<sup>b</sup>RIKEN Center for Sustainable Resource Science, Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan.

<sup>c</sup>Central Laboratories for Key Technologies, Kirin Co., Ltd. Fukuura 1-13-5, Kanazawa-ku,

Yokohama, Kanagawa 236-0004, Japan.

d Department of Chemistry and Materials Science, Tokyo Institute of Technology, Ookayama

2-12-1, Meguro-ku, Tokyo 152-8551, Japan.

e Institute for Chemical Research, Kyoto University, Gokasho, Uji, Kyoto 611-0011, Japan.

f Department of Biotechnology, Graduate School of Engineering, Osaka University,

Yamadaoka 2-1, Suita, Osaka 565-0871, Japan.

g Graduate School of Pharmaceutical Sciences, Chiba University, Inohana 1-8-1, Chuo-ku,

Chiba 260-8675, Japan.

One sentence summary;

The 2-oxoglutarate-dependent dioxygenase 16DOX catalyzes steroid 16α-hydroxylation

in the steroidal glycoalkaloid (SGA) pathway and is a suitable target for controlling

toxic SGA levels in potato. (197 characters and spaces < 200 characters)

Footnotes;

List of author contributions

M.N., N.U., K.O., K.S., and M.M. designed the research; M.M., Y.F., T.M., K.S., and Y.S.

supervised the experiments; M.N., N.U., K.O., H.L., and B.W. performed the research; and

M.N., N.U., and M.M. wrote the article.

Funding information

This work was supported by the Program for Promotion of Basic and Applied Researches for

Innovations in Bio-oriented Industry (BRAIN), Japan, by the Scientific Technique Research

47 Promotion Program for Agriculture, Forestry, Fisheries and Food Industry, Japan, and by the
48 Cross-ministerial Strategic Innovation Promotion Program (SIP), Japan.
49 
50 *Address correspondence to mizutani@gold.kobe-u.ac.jp
51 The author responsible for distribution of materials integral to the findings presented in this
52 article in accordance with the policy described in the Instructions for Authors
53 (www.plantphysiol.org) is: Masaharu Mizutani (mizutani@gold.kobe-u.ac.jp).
54 
55 
Downloaded from on July 30, 2017 - Published by www.plantphysiol.org
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

Abstract

Steroidal glycoalkaloids (SGAs) are toxic specialized metabolites that are found in Solanaceae. Potato (Solanum tuberosum) contains the SGAs α-solanine and α-chaconine, while tomato (S. lycopersicum) contains α-tomatine, all of which are biosynthesized from cholesterol. However, although two cytochrome P450 monooxygenases that catalyze the 22- and 26-hydroxylation of cholesterol have been identified, the 16-hydroxylase remains unknown. Feeding with deuterium-labeled cholesterol indicated that the 16α- and 16β-hydrogen atoms of cholesterol were eliminated to form α-solanine and α-chaconine in potato, while only the 16α-hydrogen atom was eliminated in α-tomatine biosynthesis, suggesting that a single oxidation at C-16 takes place during tomato SGA biosynthesis while a two-step oxidation occurs in potato. Here we show that a 2-oxoglutarate-dependent dioxygenase (2OGD), designated as 16DOX, is involved in SGA biosynthesis. We found that the transcript of potato 16DOX (St16DOX) was expressed at high levels in the tuber sprouts, where large amounts of SGAs are accumulated. Biochemical analysis of the recombinant St16DOX protein revealed that St16DOX catalyzes the 16α-hydroxylation of hydroxycholesterols and that (22S)-22,26-dihydroxycholesterol was the best substrate among the nine compounds tested. St16DOX-silenced potato plants contained significantly lower levels of SGAs and a detailed metabolite analysis revealed that they accumulated the glycosides of (22S)-22,26-dihydroxycholesterol. Analysis of the tomato 16DOX (Sl16DOX) gene gave essentially the same results. These findings clearly indicate that 16DOX is a steroid 16α-hydroxylase that functions in the SGA biosynthetic pathway. Furthermore, St16DOX-silencing did not affect potato tuber yield, indicating that 16DOX may be a suitable target for controlling toxic SGA levels in potato. (249 words < 250)

79
80

3

Downloaded from on July 30, 2017 - Published by www.plantphysiol.org  
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

Introduction

Steroidal glycoalkaloids (SGAs) are typically found in plants in the family Solanaceae (Harrison, 1990; Helmut, 1998; Petersen et al., 1993). In particular, potato (*Solanum tuberosum*) is known to contain the SGAs α-solanine and α-chaconine, while tomato (*S. lycopersicum*) contains α-tomatine, which are toxic to fungi, bacteria, insects, animals, and humans (Friedman, 2002, 2006). The mechanisms of toxicity include the disruption of membranes and the inhibition of acetylcholine esterase activity (Roddick, 1989). α-Solanine and α-chaconine occur in most tissues of potato plants, but are present at particularly high levels in floral and tuber sprout tissues (Kozukue and Mizuno, 1985, 1989; Ginzberg et al., 2009). Therefore, since these potato SGAs are toxic and cause a bitter taste, controlling their content in potato tubers is an important focus of potato breeding (Friedman, 2006). Similarly, α-tomatine exists in all tomato plant tissues, but is richly accumulated in the leaves and immature fruit, with its content decreasing during fruit ripening (Friedman, 2002).

SGAs are composed of C₂₇ steroids with an oligosaccharide at the hydroxy group at the C-3 position (Friedman, 2002, 2006). In α-solanine and α-chaconine in potato, the oligosaccharides solatriose and chacotriose are attached to the C-3 hydroxy group of solanidine, which is a solanidane-type aglycone; and similarly, in α-tomatine in tomato, lycoctetraose is linked to the C-3 hydroxy group of tomatidine, which is spirosolane-type aglycone. These SGAs are biosynthesized from cholesterol (Sawai et al., 2014), which is subsequently modified via oxidation at the C-16, C-22, and C-26 positions, amination at the C-26 position, and glycosylation at the C-3 hydroxy group (Friedman, 2002; Ginzberg et al., 2009; Petersen et al., 1993; Ohyama et al., 2013). Cytochrome P450 monooxygenases (CYPs) are likely involved in the oxidative modification of cholesterol, while a transaminase catalyzes

Figure 1 (Nakayasu)

![Diagram](#)

Figure 1. The putative biosynthetic pathway for SGAs in potato and tomato. Thick solid arrow indicates the reaction step characterized in this work. Thin solid arrows indicate the reaction steps reported by Umemoto et al. (2016). White arrows represent multiple reaction stages.

the amination at C-26, and UDP-dependent glycosyltransferases (UGTs) also function in the glycosylation at the C-3 hydroxy group (Fig. 1). Some UGTs have now been identified as the

enzymes that are involved in the glycosylation steps for SGA biosynthesis (Moehs CP et al., 1997; McCue KF et al., 2005, 2006, 2007; Itkin et al., 2011). Itkin et al. (2013) reported that several genes that are involved in the SGA biosynthetic pathways exist as gene clusters on chromosomes 7 and 12 in the genomes of tomato and potato. In addition, sterol side chain reductase 2 (SSR2) has been identified as a gene that is committed to cholesterol biosynthesis and involved in SGA production (Sawai et al., 2014). However, the genes that are involved in the later steps of the SGA biosynthetic pathways remain to be completely elucidated.

CYPs are heme-thiolate membrane proteins that are generally bound to the cytoplasmic surface of the endoplasmic reticulum and are known to be involved in the oxidation steps during the metabolism of various plant natural products. A number of CYPs have been reported to catalyze the oxidation of triterpene skeletons, such as oleanane-, lupine-, ursane- and dammarane-types (Moses et al., 2014; Seki et al., 2015). In addition, several CYP genes have been identified as being responsible for the oxidation of the steroid backbone in the biosynthesis and catabolism of brassinosteroids, which are a plant steroid hormones (Ohnishi et al., 2009). Therefore, we previously investigated SGA biosynthesis in potato and tomato by selecting several CYP genes that show high expression in SGA-rich organs as candidate oxidases of cholesterol using the public expressed sequence tag (EST) databases of potato and tomato. This showed that two CYPs (PGA1/CYP72A208 and PGA2/CYP72A188) are involved in the hydroxylation of cholesterol at the C-26 and C-22 positions (Umemoto et al., 2016). In addition, another CYP (PGA3/GAME4/CYP88B1) was also identified as being involved in SGA biosynthesis, but its catalytic function remains unknown (Umemoto and Sasaki, 2013; Itkin et al., 2013). We were unable to find a C-16 hydroxylase among the candidate CYPs, however.
The 2-oxoglutarate dependent dioxygenase (2OGD) superfamily is the second largest enzyme family, following the CYP superfamily, in the plant genome (Kawai et al., 2014). 2OGDs are non-heme iron-containing proteins that localize in the cytosol as soluble proteins. Many 2OGDs play crucial roles in the oxidation steps in the biosynthesis of diverse specialized metabolites such as flavonoids and phytohormones (Kawai et al., 2014). However, unlike CYPs, no 2OGDs have been identified as being involved in the biosynthesis of triterpenoids and steroids to date. Here, we characterized the 2OGD gene named 16DOX and found that it is involved in SGA biosynthesis. The 16DOX gene was co-expressed with the previously identified SGA biosynthetic genes in potato and tomato, and the 16DOX protein was found to catalyze the hydroxylation of cholesterol at the C-16α position. Furthermore, 16DOX-silencing in transgenic potato plants led to significantly reduced endogenous SGA levels and suppression of potato tuber sprouting, which are similar to the phenotypes observed in PGA1- and PGA2-silenced plants (Umemoto et al., 2016). Thus, 16DOX is the first 2OGD to be identified as functioning as a steroid 16α-hydroxylase involved in SGA biosynthesis.
146

6

Downloaded from on July 30, 2017 - Published by www.plantphysiol.org  
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

RESULTS

SGA analysis of potato shoots and tomato seedlings fed with stable isotope-labeled compounds

To examine the fates of hydrogen atoms at the C-16 position during SGA biosynthesis in potato shoots and tomato seedlings, we conducted feeding experiments using three kinds of stable isotope-labeled compounds: [15,15,16α,17α-²H₄] cholesterol, [15,15,16β,17α-²H₄] cholesterol, and [15,15,17α-²H₃] cholesterol. The accumulated SGAs in the harvested shoots and seedlings were extracted and analyzed by liquid chromatography-mass spectrometry (LC-MS) (Fig. 2). In the case of potato shoots, labeled α-solanine and α-chaconine were detected by monitoring the m/z 871 and 855 ions [M+H]⁺, respectively (Fig. 2A-F), while non-labeled α-solanine and α-chaconine were detected by monitoring the m/z 868 and 852 ions [M+H]⁺, respectively, in shoots that were fed with the three labeled cholesterols. These results indicate that the 15- and 17-hydrogens of cholesterol were retained but both hydrogens at the 16α and 16β positions were eliminated in the process of potato SGA biosynthesis. In the case of tomato seedlings, labeled α-tomatine was detected by monitoring the m/z 1037 ion [M+H]⁺ (Fig. 2G and H), while non-labeled α-tomatine was detected by monitoring the m/z 1034 ion [M+H]⁺ in seedlings that were fed with [15,15,16α,17α-²H₄] cholesterol and [15,15,17α-²H₃] cholesterol. By contrast, when [15,15,16β,17α-²H₄] cholesterol was administered, labeled α-tomatine was detected by monitoring the m/z 1038 ion [M+H]⁺ (Fig. 2I), which is one mass higher than the results obtained with [15,15,16β,17α-²H₄] cholesterol and [15,15,17α-²H₃] cholesterol. These results indicate that 16β-hydrogen was retained but 16α-hydrogen was eliminated in tomato SGA biosynthesis. These results are consistent with the previous observation reported by Canonica

Figure 2 (Nakayasu)

![Image](image.png)

Figure 2. LC-MS analysis of SGAs upon feeding stable isotope-labeled compounds to *in vitro*-grown potato shoots and tomato seedlings. The two traces on the left are mass-chromatograms for the indicated $m/z$ ions. The right mass spectrum was obtained at the retention time indicated by the arrow. Plants were fed with [15,15,17$\alpha$-$^{2}$H$_{3}$]cholesterol (A-C), [15,15,16$\alpha$, 17$\alpha$-$^{2}$H$_{4}$]cholesterol (D-F), or [15,15,16$\beta$,17$\alpha$-$^{2}$H$_{4}$]cholesterol (G-I). (A, D, and G) LC-MS analysis of $\alpha$-chaconine in potato shoots. (B, E, and H) LC-MS analysis of $\alpha$-solanine in potato shoots. (C, F, and I) LC-MS analysis of $\alpha$-tomatine in tomato seedlings.

et al. (1977). Taken together, these findings suggest that oxidation (dehydrogenation) occur at both the C-16$\alpha$ and C-16$\beta$ positions in potato, and at the C-16$\alpha$ position in tomato during

SGA biosynthesis.

Identification of the candidate 16DOX gene in potato and tomato

To identify the candidate genes that are involved in the oxidative modification of cholesterol for SGA biosynthesis, we surveyed several enzyme superfamilies that are included in the EST databases of potato from the DFCI Plant Gene Indices (http://compbio.dfci.harvard.edu/tgi/plant.html) and the tomato databases from MiBASE (http://www.pgb.kazusa.or.jp/mibase/) and Sol Genomics (http://solgenomics.net). We initially selected CYP genes based on the correlation between the read numbers of the EST contigs and the tissue specific accumulation of SGAs in potato and tomato. This led to the identification of the CYPs PGA1 (Sotub06g021140, CYP72A208) and PGA2 (Sotub07g016580, CYP72A188) as a steroid 26-hydroxylase and a steroid 22-hydroxylase, respectively, for potato SGA biosynthesis (Umemoto et al., 2016). However, we were unable to identify a 16-hydroxylase among the candidate CYPs. The 2OGD superfamily is also associated with the oxygenation reactions of various specialized metabolites (Kawai et al., 2014). Therefore, we surveyed the unigenes encoding 2OGDs from the potato database as a second focus.

A total of 256 2OGD transcripts were extracted by Basic Local Alignment Search Tool X (BLASTX) search of nucleotide sequences of all transcript sequences obtained from Spud DB potato genomics resource (http://solanaceae.plantbiology.msu.edu/) against protein sequences of 130 2OGDs from Arabidopsis (Kawai et al., 2014). One of the 2OGD transcripts (PGSC0003DMT400030676) showed the highest FPKM value in tuber sprout among the 256 2OGD transcripts in RNA-Seq Gene Expression Data obtained from Spud DB (Supplemental

Figure 3 (Nakayasu)

![Image](#)

Figure 3. Quantitative RT-PCR analysis of the expression patterns of SGA biosynthetic genes in various organs of potato plants. Transcript levels of SGA biosynthetic genes are shown relative to that of *EF1a* as an internal reference gene. Bars indicate one standard deviation from the mean (*n* = 3).

Table S1), and we selected the transcript as a candidate gene encoding a 16-hydroxylase, which was designated as *St16DOX*. Quantitative reverse transcription polymerase chain

Downloaded from on July 30, 2017 - Published by www.plantphysiol.org  
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

reaction (RT-PCR) analysis showed that potato St16DOX was highly expressed in the tuber
sprouts (Fig. 3), where large amounts of SGAs are accumulated (Friedman and Dao, 1992;
Smith et al., 1996; Milner et al., 2011). BLAST searches against the genome databases of
Solanum species (Sol Genomics Network: http://solgenomics.net) showed that St16DOX is
identical to Sotub07g016570 in the Potato ITAG protein database and shows 93% amino acid
identity to tomato Solyc07g043420 (designated as Sl16DOX) in the Tomato ITAG protein
database. Tomato Sl16DOX was highly expressed in the flowers, which accumulate large
amounts of α-tomatine (Supplemental Fig. S1). These results suggest that 16DOX genes are a
candidate for 16-hydroxylase in SGA biosynthesis (Fig. 3 and Supplemental Fig. S1).

In a recent phylogenetic classification of the plant 2OGD superfamily (Kawai et al.,
2014), 16DOXs were found to belong to clade DOXC41 which includes hyoscyamine
6β-hydroxylase (H6H), which is involved in the biosynthesis of the tropane alkaloid
scopolamine (Matsuda et al., 1991). The 16DOXs share around 44% amino acid sequence
identity to H6H in Hyoscyamus niger and contain several sequence motifs that are highly
conserved among 2OGDs (Supplemental Fig. S2). The St16DOX and Sl16DOX proteins
contain the Fe(II)-binding motif His-X-Asp-Xn-His (His-217, Asp-219, and His-272 in
16DOX), which is conserved in the 2OGD superfamily (Bugg, 2003; Lukacin and Britsch,
1997; Wilmouth et al., 2002). The Arg-X-Ser motif (Arg-282 and Ser-284), which binds to
the C-5 carboxy group of 2-oxoglutarate, is also conserved in the 16DOX proteins (Lukacin et
al., 2000; Wilmouth et al., 2002).

In vitro functional analysis of the recombinant 16DOX protein

To investigate the catalytic functions of 16DOX, recombinant St16DOX protein

Figure 4 (Nakayasu)

A

B

Figure 4. GC-MS analysis of the reaction products from the recombinant St16DOX and Sl16DOX proteins with (22S)-22-hydroxycholesterol as a substrate. (A) The extracted ion chromatogram targeting ion \( m/z \) 173 of the reaction products and the authentic compounds. (B) Mass spectrum of each peak shown in (A) at a retention time of 20.0 min.

was prepared with a bacterial expression system in *Escherichia coli* and an in vitro enzyme assay was performed with (22S)-22-hydroxycholesterol as a substrate. The reaction products

222 were analyzed using gas chromatography-mass spectrometry (GC-MS). St16DOX
223 metabolized (22S)-22-hydroxycholesterol to a product with a retention time of 20.0 min and a
224 major mass fragment ion at m/z 173 (Fig. 4). This product was identical to the authentic
225 compound (22S)-16α,22-dihydroxycholesterol in terms of both the retention time and the
226 mass spectrum, but different from the authentic compound (22S)-16β,22-dihydroxycholesterol,
227 indicating that St16DOX catalyzes the 16-hydroxylation of (22S)-22-hydroxycholesterol
228 specifically in the 16α-configuration. The catalytic activity of tomato Sl16DOX was
229 consistent with that of St16DOX (Fig. 4).
230 We next determined the substrate specificity of St16DOX toward cholesterol and
231 several oxygenated cholesterols (Fig. 5). St16DOX showed the highest activity toward
232 (22S,25S)-22,26-dihydroxycholesterol, reaching a rate that was approximately 50 times higher
233 than that with (22S)-22-hydroxycholesterol (Fig. 5 and Supplemental Fig. S3). LC-MS
234 analysis showed a fragment ion at m/z 435 in the product, which corresponds with the
235 deduced parent ion of 16,22,26-trihydroxycholesterol (Supplemental Fig. S4). St16DOX
236 weakly metabolized (22R)-22-hydroxycholesterol to a new product (Supplemental Fig. S5),
237 but the structure of the metabolite was unknown. By contrast, the assays with the other
238 substrates did not give any product peaks. In terms of kinetic parameters, the Michaelis
239 constant (Km value) for St16DOX toward (22S,25S)-22,26-dihydroxycholesterol was
240 determined to be 4.19 ± 0.17 μM (Supplemental Fig. S6). These results strongly suggest that
241 St16DOX functions as a 16α-hydroxylase of (22S,25S)-22,26-dihydroxycholesterol in SGA
242 biosynthesis.
243
244 SGA analysis of St16DOX-silenced transgenic potato plants.

Figure 5 (Nakayasu)

Substrate
(22S,25S)-22,26-dihydroxycholesterol
(22S)-22-hydroxycholesterol
(22R)-22-hydroxycholesterol
Cholesterol
22-oxocholesterol
(25S)-26-hydroxycholesterol
(25R)-26-hydroxycholesterol
(22S,25RS)-22-hydroxy-26-oxocholesterol
(22R,25S)-22-hydroxy-26-oxocholesterol

Relative activity
0     50    100(%)

Figure 5. Relative activities of recombinant St16DOX toward cholesterol and several oxygenated cholesterols. N.D. indicates not detected.

To confirm the contribution of St16DOX to SGA biosynthesis in potato, we transformed potato plants with an RNA interference vector to create St16DOX-silenced

Figure 6 (Nakayasu)

![Image](#)

Figure 6. SGA contents and yields of tubers from *St16DOX*-silenced transgenic potato plants. (A) Quantification RT-PCR analysis of *St16DOX* transcript levels in the *in vitro*-grown shoots of *St16DOX*-silenced plants. (B) LC-MS analysis of SGA levels in the *in vitro*-grown shoots of *St16DOX*-silenced plants. (C and D) LC-MS analysis of SGA levels in the peel (C) and cortex (D) of harvested tubers from *St16DOX*-silenced plants. (E) Yields of tubers from *St16DOX*-silenced plants. Bars indicate one standard deviation from the mean (*n* = 3). FW, fresh weight; NT, non-transgenic control plants; #15, #16, #28, #39, and #41, independent transgenic lines.

transgenic potato plants. Among 41 lines of *St16DOX*-silenced transgenic plants, the *in vitro*-grown shoots of five independent lines (#15, #16, #28, #39, and #41; Fig. 6) had

significantly lower St16DOX transcript levels than the control (Fig. 6A) and consistently had  
a much lower SGA content (Fig. 6B). All of the silenced lines grew normally in a greenhouse,  
and the potato tubers were harvested. The SGA contents in the tuber peel and cortex of the  
five silenced lines were significantly lower than the control (Fig. 6C and D), indicating that  
St16DOX is involved in SGA biosynthesis in potato. Similarly, the SGA contents in  
Sl16DOX-silenced transgenic tomato plants were also severely reduced (Supplemental Fig.  
S7).

### Phenotype of St16DOX-silenced transgenic potato plants

The St16DOX-silenced transgenic potato plants had similar tuber yields to the  
control (Fig. 6E), however, the tuber sprouts of these plants stopped the elongation under dark  
and light conditions. Although sprout initiation seemed to be normal, but the sprouts did not  
grow even after more than three months at 20°C or for a one year at 4°C after the cessation  
of usual dormancy in the control plants (Supplemental Fig. S8A). Interestingly, tiny sprouts  
could start to grow after the tubers were planted in soil (Supplemental Fig. S8B), but not in  
water under light or dark conditions. In addition, after placing separated tops of the tiny  
sprouts on tissue culture media, the sprouts could grow (Supplemental Fig. S8C). The  
phenotype is completely same to those of PGA1- and PGA2-silenced potato plants (Umemoto  
et al., 2016). PGA1- and PGA2-silenced potato plants did not bloom and are sterile. Unlike to  
PGA1- and PGA2-silenced potato plants, the St16DOX-silenced transgenic potato plants  
bloomed normally. The Sl16DOX-silenced transgenic tomato plants are confirmed to be fertile  
and did not demonstrate any outward difference compared with the control.

272 Endogenous metabolite analysis in St16DOX-silenced transgenic potato plants.
273 To examine the effects of St16DOX gene silencing on the endogenous metabolites,
274 we analyzed changes in steroidal compounds in the St16DOX-silenced plants using LC-MS.
275 This analysis detected seven distinctive peaks that were presumed to be steroidal saponins,
276 which had retention times of 17.44, 17.94, 20.02, 20.56, 23.36, 24.05, and 28.45 min, and
277 gave putative masses of parental ions at m/z 1236.0, 1220.0, 1074.0, 1058.0, 911.8, 895.8, and
278 603.7, respectively (Supplemental Fig. S9). These peaks had a major mass fragment ion at m/z
279 383.6, which corresponds to [dihydroxycholesterol – 2H₂O + H⁺]⁺, and therefore were
280 assumed to be the glycosides (1–5 saccharides) of dihydroxycholesterol.
281 To determine the aglycone structure of these steroidal glycosides that were
282 accumulated in the St16DOX-silenced plants, we extracted the endogenous metabolites and
283 hydrolyzed them with 1N HCl, and then analyzed the trimethylsilylated metabolites using
284 GC-MS. Compared with non-transgenic plants, the St16DOX-silenced plants had three new
285 peaks in the extracted ion chromatogram targeting the ion m/z 171 (Fig. 7A). One of these
286 peaks had a retention time of 23.8 min, and was identical to the authentic
287 (22S, 25S)-22,26-dihydroxycholesterol in terms of both the retention time and the mass
288 spectrum (Fig. 7A and B). The other two peaks, which had retention times of 19.1 min and
289 19.3 min, were identical to the artifacts that occur during the processes of acid hydrolysis and
290 trimethylsilylated derivatization of authentic (22S, 25S)-22,26-dihydroxycholesterol (Fig. 7A).
291 These results indicate that St16DOX gene silencing results in the accumulation of the
292 glycosides of (22S, 25S)-22,26-dihydroxycholesterol, which was determined to be the best
293 substrate for St16DOX in the in vitro assay. Similarly, Sl16DOX-silenced transgenic tomato
294 plants also accumulated the glycosides of (22S, 25S)-22,26-dihydroxycholesterol

Figure 7 (Nakayasu)

![Image](#)

Figure 7. GC-MS analysis of the accumulated compounds in leaves of St16DOX-silenced transgenic potato plants. (A) The extracted ion chromatogram targeting ion \( m/z \) 171 of the accumulated compounds and the authentic compound. (B) Mass spectrum of each peak shown in (A) at a retention time of 23.8 min. NT, non-transgenic control line; #15, a transgenic line.

(Supplemental Fig. S10).

297

19

Downloaded from on July 30, 2017 - Published by www.plantphysiol.org  
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

298

20

Downloaded from on July 30, 2017 - Published by www.plantphysiol.org  
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

DISCUSSION

16DOX is a dioxygenase that shows steroid 16α-hydroxylase activity in SGA biosynthesis

SGA biosynthesis in potato and tomato is thought supposed to require the oxidation of cholesterol at the C-16, C-22, and C-26 positions, followed by transamination and glycosylation (Petersen et al., 1993; Friedman, 2002; Ginzberg et al., 2009). We recently identified two CYPs (PGA1 and PGA2) that are involved in SGA biosynthesis, which catalyze the hydroxylation of cholesterol at C-26 and C-22, respectively (Umemoto et al., 2016). In the present study, we investigated the role of a 2OGD gene named 16DOX, which belongs to clade DOXC41. An in vitro functional analysis of the recombinant 16DOX protein confirmed that 16DOX catalyzes hydroxylation of steroids at C-16α (Fig. 4). In addition, 16DOX-silenced transgenic plants contained significantly lower amounts of SGAs (Fig. 6B-D). These results clearly demonstrate that 16DOX is a steroid 16α-hydroxylase that is involved in SGA biosynthesis in Solanum species. Although many CYPs from various organisms have been characterized as catalyzing the oxidative modification of steroid compounds, this is the first 2OGD to be identified as functioning in the oxidation of steroids.

Reaction orders during steroidal alkaloid biosynthesis in Solanum and Veratrum species

In this study, we found that 16DOX-silenced transgenic plants accumulated the glycosides of dihydroxycholesterol (Supplemental Fig. S8). Similarly, Itkin et al. (2013) also reported that transgenic tomato plants in which GAME11, a gene coding a putative dioxygenase, was silenced by virus-induced gene silencing accumulated dihydroxycholesterol saponins of an unknown structure. This GAME11 corresponds to Sl16DOX and their results were consistent with our observations. In this study, we determined the aglycone structure of

the glycoside product as (22S,25S)-22,26-dihydroxycholesterol (Fig. 7) and, furthermore,
functional characterization revealed that this was the best substrate for the recombinant
St16DOX (Fig. 5). Therefore, we conclude that 16DOX catalyzes the C-16 hydroxylation of
(22S,25S)-22,26-dihydroxycholesterol, which is biosynthesized via the C-22 and C-26
hydroxylation of cholesterol by PGA2 and PGA1, respectively.
Veratrum species are known to contain the steroidal alkaloid cycloamine, which
exhibits potent pharmacological activities, and Veratrum alkaloids are proposed to be
biosynthesized from cholesterol via the predicted intermediates verazine and solanidine
(Kaneko et al., 1976, 1977). Therefore, since solanidine is a common precursor for the
biosynthesis of SGA as well as Veratrum alkaloids, verazine has also been proposed as a
precursor in potato SGA biosynthesis (Freidman, 2006). Recently, Augustin et al. (2015)
reported the functional characteristic of the four genes (CYP90B27, CYP94N1, GABAT1,
and CYP90G1) that catalyze the first six reactions in the production of verazine from
cholesterol in V. californicum. In this pathway, oxidation and transamination at C-26,
subsequent oxidation at C-22, and F-ring closure occurred in the absence of the hydroxylation
at C-16 (Augustin et al., 2015), and previous phytochemical studies have also supported this
reaction order (Kaneko et al., 1970, 1975, 1976, 1977). By contrast, our study demonstrated
that 16DOX hydroxylated (22S,25S)-22,26-dihydroxycholesterol at C-16 (Fig. 5) and this was
supported by the observation that the glycosides of 22,26-dihydroxycholesterol accumulated
in 16DOX-silenced transgenic plants (Supplemental Fig. S9) whereas verazine-type
compounds were not detected. These different reaction orders may be explained by the
independent origin of the biosynthetic genes in Solanum and Veratrum species (Fig. 8). Potato
PGA2 (CYP72A188) and PGA1 (CYP72A208) belong to the CYP72 family, and differ from

Figure 8. (Nakayasu)

< Solanum species >
![Chemical Structure](chemical_structure_1.png)
cholesterol

< Veratum species >
![Chemical Structure](chemical_structure_2.png)
cholesterol

Figure 8. The putative reaction orders in the biosynthetic pathways of steroidal alkaloids in Solanum and Veratum species. Thick arrows indicate the reaction steps that were characterized in this work. Black-filled arrow indicates the multiple reaction stages reported by Umemoto et al. (2016) and Augustin et al. (2015). White arrows represent unclear multiple reaction stages.

Veratrum CYP90B27 and CYP94N1, which catalyze C-22 hydroxylation and two steps of C-26 oxidation to form C-26 aldehyde, which can be transaminated by GABAT1 (Augustin et

The identification of a C-22 oxidase in *Solanum* species and a C-16 hydroxylase in *Veratrum* species will provide further support for the different reaction orders and origins of the steroloidal alkaloid pathways in these genera.

16DOX catalyzes C-16 hydroxylation specifically in the 16α-configuration

Feeding with the three labeled cholesterols indicated that the 16α-hydrogen of cholesterol was eliminated but the 16β-hydrogen was retained in tomato SGA biosynthesis, (Fig. 2G-I), which supported the finding that 16DOX catalyzes C-16 hydroxylation specifically in the 16α-configuration. However, the structure of the final product α-tomatine is not consistent with this, as the oxygen atom in the E-ring of α-tomatine is in the 16β-configuration and 16α-hydrogen is also present. Curiously, our tracer experiments suggested that the 16α-hydrogen of α-tomatine is derived from the 16β-hydrogen of cholesterol. In the proposed biosynthetic pathway of α-tomatine (Itkin et al., 2013), 16,22,26-trihydroxycholesterol is likely re-oxidized at C-22, followed by E-ring closure to form furostanol-type aglycone, in which the oxygen atom is in the 16β-configuration. Therefore, this apparent discrepancy may be explained by the process of E-ring closure, during which the inversion of 16β-hydrogen to the C-16α position occurs, although a 2nd step of C-22 oxidase and a E-ring cyclase that are involved in tomato SGA biosynthesis have not yet been characterized.

Our tracer experiments also suggested that both the 16α- and 16β-hydrogen atoms of cholesterol were eliminated during potato SGA biosynthesis while St16DOX catalyzes only 16α-hydroxylation (Fig. 2A-F). These results suggest that loss of 16β-hydrogen atom is catalyzed by an additional unknown C-16 oxidase and that this oxidation step is specifically

occurred in potato SGA biosynthesis. In addition, 16α-hydrogen was subsequently added to

form solanidine in potato SGA biosynthesis (Fig. 2A-F), suggesting that an unknown C-16

reductase which introduces 16α-hydrogen may be involved in the final stage of solanidine

biosynthesis. Taken together, these findings indicate that further investigation is required to

elucidate the mechanism by which the E- and F-rings of spirosoiane and solanidane are

formed during SGA biosynthesis.

Co-expression of 16DOX with SGA biosynthetic genes

Quantitative RT-PCR analysis in various tissues of potato showed that the 16DOX

gene was co-expressed with the PGA1 and PGA2 genes as well as the UGT genes that are

responsible for the glycosylation steps in SGA biosynthesis (Fig. 3). Itkin et al. (2013)

reported that SGA biosynthetic genes including GAME11 (which corresponds to 16DOX)

exist as gene clusters on chromosomes 7 and 12 of tomato and potato. Such physical

clustering of functionally related genes may enable the coordinated regulation of gene

expression at the chromatin level (Field et al., 2011). Previously, Sl16DOX was reported as

the pistil-expressed dioxygenase that has been variously named TPP1, GAD2, SPP2, and

H6H-like protein in tomato (Milligan and Gasser, 1995; Jacobsen and Olszewski, 1996;

Sylviane, 1999; Nakane, 2003). The findings of these studies were consistent with the present

results that Sl16DOX showed high expression levels in flowers (Fig. 3). Recently, Thagun et

al. (2016) reported that the overexpression and suppression of jasmonate-responsive ERF

(JRE) genes (JRE3, JRE4, and JRE5) in tomato significantly influenced SGA accumulation

and expression of the genes encoding the SGA biosynthetic enzymes, including those

involved in the upstream mevalonate pathway. JRE4 was found to bind a GCC box-like

element that is enriched in the promoters of JRE-regulated genes and also controlled the

coordinated expression of the SGA biosynthetic genes. Similarly, Cárdenas et al. (2016)

revealed that GAME9 (which is identical to JRE4) in tomato and potato regulates the

expression of the SGA biosynthetic genes by directly or indirectly acting on the promoters of

downstream target genes. Thus, co-expression analysis of the SGA biosynthetic genes that are

regulated by JRE4/GAME9 will help to identify novel genes that are involved in the

formation of the E- and F-rings of spirosolane and solanidane, including a C-16 oxidase in

potato.

### Phenotype of 16DOX-silenced transgenic plants

16DOX-silenced transgenic potato plants contained significantly lower amounts of

α-solanine and α-chaconine, and the silenced lines grew normally in a greenhouse with

comparable yields of tubers to the wild type. Interestingly, 16DOX-silenced transgenic potato

plants did not sprout (Supplemental Fig. S8A), but the sprouts could grow by planting the

tubers in soil or placing the excised sprout tips on tissue culture media (Supplemental Fig.

S8B and C). These phenotypes are similar to those of PGA1- and PGA2-silenced plants,

except that latter also exhibited abnormal flowers (Umemoto et al., 2016). By contrast,

GAME4/PGA3- or SSR2-silenced transgenic potato plants were phenotypically identical to

control plants, despite containing much lower SGA contents (Itkin et al., 2013; Umemoto and

Sasaki, 2013; Sawai et al., 2014). These findings demonstrate that the amounts of SGA are

unrelated to sprouting morphogenesis, and so 16DOX-silenced plants may accumulate

unknown compounds that inhibit sprouting and can be removed by planting the tubers in soil

(Umemoto et al., 2016). We found that 16DOX-silenced transgenic plants accumulated

416 dihydroxycholesterol and/or its glycosides (Supplemental Fig. S9), and PGA1-
and
417 PGA2-silenced plants were also found to accumulate 22- or 26-hydroxycholesterols and/or
418 their glycosides (Umemoto et al., 2016). Therefore, it is possible that the suppression of
419 sprouting morphogenesis is caused by the accumulation of hydroxylated cholesterol or its
420 glycosides in the sprout clusters of the silenced potatoes.
421 Our observations demonstrate that it may be possible to breed SGA-free potato, as
422 reported by Umemoto et al. (2016). 16DOX is a valuable target for the development of a
423 loss-of-function mutant because it occurs as a single copy gene in the genome of potato and
424 tomato. Such a loss-of-function mutant for 16DOX could be obtained thought mutation
425 breeding using targeting induced local lesions in genomes (TILLING; Elias et al., 2009) and
426 genome editing techniques (Sawai et al., 2014; Nicolia et al., 2015).
427
Downloaded from on July 30, 2017 - Published by www.plantphysiol.org
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

MATERIALS AND METHODS

Chemicals

Authentic samples of α-solanine, α-chaconine, α-tomatine,

(22R)-22-hydroxycholesterol, and (22S)-22-hydroxycholesterol were purchased from Sigma-Aldrich. Cholesterol was purchased from Tama Biochemical Co. Authentic compounds of (25R)-26-hydroxycholesterol, (25S)-26-hydroxycholesterol, (22S)-16α,22-dihydroxycholesterol, (22S)-16β,22-dihydroxycholesterol, (22S,25S)-22,26-dihydroxycholesterol, (22R,25S)-22-hydroxy-26-oxocholesterol, (22S,25RS)-22-hydroxy-26-oxocholesterol, [15,15,16α,17-²H₄]-cholesterol, and [15, 15, 17-²H₃]-cholesterol were synthesized as described in the Supplemental Materials and Methods.

RNA extraction and reverse transcription

Total RNA extraction was performed using the RNeasy Plant Mini Kit (QIAGEN, Hilden, Germany) and the RNase-Free DNase Set (QIAGEN). Total RNAs of potato and tomato were prepared from the leaves, flowers, tuber peels, stems, roots, stolons and tuber sprouts of *S. tuberosum* cv. Sassy, and the leaves, flowers, mature green fruits, yellow fruits, orange fruits, and red fruits of *S. lycopersicum* cv. Micro-Tom, respectively. The extracted total RNAs of potato and tomato were used to synthesize the first strand cDNAs using the SuperScript® First-Strand Synthesis System for RT-PCR (Life Technologies) and the Transcriptor First Strand cDNA Synthesis Kit (TOYOBO), respectively.

Cloning of 16DOX cDNAs

The cDNA fragments that contained the open reading frame of each 16DOX gene were amplified by RT-PCR with primers 1 and 2 for St16DOX, and primers 3 and 4 for Sl16DOX, respectively (Supplemental Table S2), which were designed from the potato and tomato unigene sequences (Sotub07g016570 in the Potato ITAG protein database and Solyc07g043420 in the Tomato ITAG protein database, respectively). The PCR products were cloned into the pENTR/D-TOPO plasmid (Life Technologies).

Expression of the recombinant 16DOX protein in *E. coli*

The coding sequences for each 16DOX gene were amplified from the pENTR/D-TOPO plasmid using primers 5 and 6 for St16DOX, and primers 7 and 8 for Sl16DOX (Supplemental Table S2), which contained restriction sites. The amplified DNA fragments were ligated into the pMD19 vector (TaKaRa) and digested with BamHI and SalI. The DNA fragments were then ligated into the BamHI-SalI sites of pGEX4T-1. *E. coli* strain BLR (DE3) (Clontech) transformed with the constructed plasmid was grown at 37°C in lysogeny broth (LB) with 50 μg/ml ampicillin until its OD~600~ reached 0.5. Recombinant protein expression was induced by adding 0.1 mM isopropyl β-D-1-thiogalactopyranoside (IPTG) and was continued for 20 h at 18°C. The culture was then centrifuged at 3,500 rpm for 30 min at 4°C and the cell pellets were resuspended in 5 ml of cold sonication buffer containing 50 mM sodium phosphate (pH 7.4), 300 mM NaCl, and 20% (v/v) glycerol. The solution was then sonicated three times for 30 sec each on ice using a Bandelin Sonopuls HD 2070 ultrasonic homogenizer type MS 73 (Sigma) at a sound intensity of 200 W/cm² and centrifuged at 15,000 rpm for 10 min at 4°C. The GST-tagged proteins present in the supernatant were purified using GST Spin Trap columns (GE Healthcare) according to the

474 manufacturer's instructions. After two column washes, the adsorbed proteins were eluted
475 twice in 200 μl of a solution, containing 50 mM Tris-HCl (pH 8.0), 20 mM reduced
476 glutathione and 20% (v/v) glycerol, and the elution was mixed. The concentration of the
477 purified proteins was determined by the Bradford system. The purified recombinant proteins
478 were visualized by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE).
479 The proteins were revealed by staining the gel with Coomassie brilliant blue R-250 and were
480 then used for further analyzes.
481
482 **In vitro enzyme activity assay**
483 An in vitro enzyme activity assay was performed using 100 μl of reaction mixture that
484 consisted of 100 mM Bistris-HCl (pH 7.2), 5 mM 2-ketoglutaric acid, 10 mM sodium
485 ascorbate, 0.2 mM FeSO₄, 25 μM (22S)-22-hydroxycholesterol as a substrate, and each of the
486 purified recombinant 16DOX proteins of potato and tomato as an enzyme. The reaction was
487 initiated by the addition of the enzyme being tested and was carried out at 30°C for 3 h. The
488 reaction was then stopped by the addition of 100 μl ethyl acetate, followed by the addition of
489 0.2 μg 25-hydroxycholesterol in 100% ethanol as an internal standard. The reaction products
490 were extracted three times with an equal volume of ethyl acetate, and the organic phase was
491 collected and evaporated. The residue was trimethylsilylated with the TMS-HT Kit (Tokyo
492 Chemical Industry Co., Ltd.) at 80°C for 30 min.
493 GC-MS analysis of the reaction products was performed as previously described by
494 Seki et al. (2008) with minor modifications. GC-MS was conducted using a GC-MS-QP 2010
495 Ultra (Shimadzu) with a DB-5MS (30 m × 0.25 mm, 0.25 μm film thickness; J&W Scientific)
496 capillary column. The injection temperature was 250°C and the following column temperature

30
Downloaded from on July 30, 2017 - Published by www.plantphysiol.org
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

program was used: 80°C for 1 min, followed by a rise to 300°C at a rate of 20°C/min and a
hold at 300°C for 20 min. The carrier gas was He and the flow rate was 1.0 ml/min. The
interface temperature was 300°C, with a splitless injection.

Biochemical analysis of recombinant St16DOX

To determine the substrate specificity of St16DOX, its activity was assayed using 25
μM cholesterol, (22R) and (22S)-22-hydroxycholesterols, 22-oxocholesterol, (25R) and
(25S)-26-hydroxycholesterols, (22S,25S)-22,26-dihydroxycholesterol, (22R,25S), and
(22S,25RS)-22-hydroxy-26-oxocholesterols. Each reaction was carried out at 30°C for 3 h.
Extraction and GC-MS analysis of the reaction product were performed as described above.
In addition, the reaction product for (22S,25S)-22,26-dihydroxycholesterol was also analyzed
by LC-MS by extracting and evaporating the product as described above, and dissolving the
residue in 200 μl ethanol. LC-MS analysis was performed using a system consisting of an
ACQUITY UPLC H-Class System (Waters, Milford, MA, USA) and an SQ Detector 2
(Waters), and data acquisition and analysis were performed using MassLynx 4.1 software
(Waters). Each sample (5 μl) was injected into an ACQUITY UPLC BEH C-18
chromatographic column (50 × 2.1 mm, 1.7 μm; Waters), in which the column temperature
was set at 40°C and the flow rate was set at 0.2 ml/min. The mobile phases were water with
0.1% (v/v) formic acid (A) and acetonitrile (B), using a gradient elution of 10%-90% B at
0-30 min, 90%-100% B at 30-40 min and 100% B at 40-46 min (0-30 min and 30-40 min,
linear gradient). The mass spectra were obtained in positive electrospray ionization (ESI),
with a capillary voltage of 3 kV and a sample cone voltage of 60 V. MS scan mode with a
mass range of m/z 250-1400 was used.

Next, we determined the kinetic parameters of recombinant St16DOX in triplicate assays. The activity was assayed using (22S,25S)-22,26-dihydroxycholesterol at a concentration ranging from 1 to 50 μM. The reaction was carried out at 30°C for 30 min. Extraction and GC-MS analysis of the reaction product were performed as described above. Kinetic parameters were determined by non-linear regression using the ANEMONA program (Hernandez and Ruiz, 1998).

Generation of transformation vectors, plant transformation, and growth conditions

A 370-bp fragment of St16DOX cDNA was PCR amplified using primers 9 and 10 (Supplemental Table S2), which contained restriction sites. An RNAi binary vector that targeted the St16DOX gene (pKT258) was constructed from the binary vector pKT11 (Umemoto et al., 2001) by locating two 370-bp fragments of St16DOX in opposite directions that interposed the third intron of the Arabidopsis thaliana At4gl4210 gene under the control of the cauliflower mosaic virus 35S (CaMV35S) promoter in the T-DNA region as previously described (Umemoto et al., 2016). pKT258 was then electroporated into Agrobacterium tumefaciens GV3110mp90. Potatoes (S. tuberosum cv. Sassy) were transformed using Agrobacterium GV3110mp90 cells with pKT258, as previously reported (Monnma, 1990). In vitro-grown plants were cultured at 20°C under a 16h light/8h dark cycle, following which 41 transformants were individually selected by genomic PCR of in vitro-grown shoots using primers 11 and 12 (Supplemental Table S2), which targeted the kanamycin resistance gene in the T-DNA region that was integrated into the potato genome. Quantitative RT-PCR analysis of St16DOX was performed using primers 13 and 14 (Supplemental Table S2) as described in the Real-time quantitative RT-PCR analysis section. Total RNA was then prepared from the

stems of five independent lines of *in vitro*-cultured plants: #5, #16, #28, #39, and #41. Primers

15 and 16 (Supplemental Table S2), which targeted the potato elongation factor 1a gene

(EF1α) (Nicot et al., 2005), were used as a control. Tomato plants (*S. lycopersicum* cv.

Micro-Tom) were also transformed using *Agrobacterium* GV3110 mp90 cells with pKT258,

as previously reported (Sun et al., 2006)

LC-MS analysis of SGAs in 16DOX-silenced transgenic plants

The SGAs that were contained in 16DOX-silenced transgenic plants were extracted

and quantified as previously described (Umemoto et al., 2016). LC-MS analysis of the

steroidal compounds accumulated in the 16DOX-silenced plants was performed as described

in the Biochemical analysis of recombinant St16DOX section with minor modifications:

LC-MS was conducted with an ACQUITY UPLC HSS T3 chromatographic column (100 ×

2.1 mm, 1.8 μm; Waters); water with 0.1% (v/v) formic acid (A) and acetonitrile (B) were

used as the mobile phases, using a gradient elution of 10%B at 0–2 min and 10%–55%B at

2–32 min (linear gradient); and MS scan mode was used with a mass range of *m/z* 300–1250.

GC-MS analysis of steroids in 16DOX-silenced transgenic plants

The steroid compounds that accumulated in the *in vitro*-grown shoots of

16DOX-silenced transgenic potato plants and the leaves of 16DOX-silenced transgenic tomato

plants were extracted and analyzed using a similar method to that previously described

(Ohyama et al., 2013). The extracted residue was trimethylsilylated with

N-methyl-N-trimethylsilyl trifluoroacetamide (Sigma-Aldrich) at 80°C for 30 min. GC-MS

analysis was performed as described above with minor modifications: a DB-1MS (30 m ×

0.25 mm, 0.25 μm film thickness; J&W Scientific) capillary column was used.

Real-time quantitative RT-PCR analysis

Quantitative RT-PCR was performed with a LightCycler® Nano (Roche) using THUNDERBIRD™ SYBR® qPCR Mix (TOYOBO) with the following primers sets: 17 and 18 for St16DOX, 19 and 20 for PGA1, 21 and 22 for PGA2, 23 and 24 for SGT1, 25 and 26 for SGT3 and 15 and 16 for EF1a (Nicot et al., 2005) using the cDNA of various potato tissues as templates; and 17 and 18 for Sl16DOX, 27 and 28 for PGA1-homolog, 29 and 30 for PGA2-homolog, 31 and 32 for GAME1, and 33 and 34 for ubiquitin using the cDNA of various tomato tissues as templates (Supplemental Table S2). Cycling was carried out at 95°C for 10 min, 45 cycles at 95°C for 10 sec, 60°C for 10 sec, and 72°C for 15 sec for amplification, followed by holding at 95°C for 30 sec and ramping up from 60°C to 95°C at 0.1°C/sec to perform a melting curve analysis. Three biological replicates were analyzed in duplicate. The gene expression levels were normalized against the values obtained for the EF1a and ubiquitin genes, which were used as an internal reference in potato and tomato, respectively. Data acquisition and analysis were performed using LightCycler® Nano software (Roche).

Tracer experiments using stable isotope-labeled compounds

Tracer experiments were performed according to a previously reported method (Ohyama et al., 2013) with minor modifications: each stable isotope-labeled compound (1.0 mg) was dissolved in acetone (25 μl)–Tween 80 (25 μl). In vitro-grown potato (S. tuberosum cv. Sassy) shoots and tomato (S. lycopersicum cv. Micro-Tom) seedlings were prepared for

tracer experiments and fed with the stable isotope-labeled compounds
[15, 15, 16α, 17-²H₄] cholesterol, [15, 15, 16β, 17-²H₄] cholesterol, and [15, 15, 17-²H₃] cholesterol.
After 7 days, the SGAs that accumulated in the harvested potato shoots and tomato seedlings
were extracted and analyzed by LC-MS as described above.

Accession Numbers
St16DOX, LC222743; Sl16DOX, LC222744

Supplemental Data
Supplemental Fig. S1. Quantitative RT-PCR analysis of the expression pattern of SGA biosynthetic genes in various organs of tomato.
Supplemental Fig. S2. Amino acid sequence alignment of St16DOX, Sl16DOX, and hyoscyamine 6β-hydroxylase from Hyoscyamus niger (HnH6H).
Supplemental Fig. S3. GC-MS analysis of the reaction products obtained from the recombinant St16DOX protein with (22S, 25S) -22, 26-dihydroxycholesterol as a substrate.
Supplemental Fig. S4. LC-MS analysis of the reaction products from the recombinant St16DOX protein with (22S, 25S) -22, 26-dihydroxycholesterol as a substrate.
Supplemental Fig. S5. GC-MS analysis of the reaction products from the recombinant 16DOX proteins with (22R) -22-hydroxycholesterol as a substrate.
Supplemental Fig. S6. Kinetic analysis of the recombinant St16DOX with (22S, 25S) -22, 26-dihydroxycholesterol.
Supplemental Fig. S7. LC-MS analysis of α-tomatine levels in the leaves of Sl16DOX-silenced transgenic tomato plants.

612 Supplemental Fig. S8. Phenotypes of St16DOX-silenced transgenic potato plants.
613 Supplemental Fig. S9. LC-MS analysis of the accumulated compounds in the leaves of
614 St16DOX-silenced transgenic potato plants.
615 Supplemental Fig. S10. GC-MS analysis of the accumulated compounds in leaves of
616 Sl16DOX-silenced transgenic tomato plants.
617 Supplemental Table S1. The list of a total of 2562OGD transcripts extracted from Spud DB
618 potato genomics resource.
619 Supplemental Table S2. The oligonucleotides that were used in the current study.
620 
621 
622 The following materials are available in the online version of this article.
623 Supplemental Fig. S1. Quantitative RT-PCR analysis of the expression pattern of SGA
624 biosynthetic genes in various organs of tomato. Transcript levels of the SGA biosynthetic
625 genes are shown relative to that of the ubiquitin gene, which was used as an internal reference.
626 Bars indicate one standard deviation from the mean (n = 3).
627 
628 Supplemental Fig. S2. Amino acid sequence alignment of St16DOX, Sl16DOX, and
629 hyoscyamine 6β-hydroxylase from Hyoscyamus niger (HnH6H). Multiple sequence
630 alignment was performed using the ClustalW multiple alignment analysis tool of BioEdit.
631 Identical and similar amino acid residues are shaded in black and gray, respectively.
632 Conserved Fe(II)-binding motifs among the 2OGD superfamily are indicated with solid
633 arrows. Conserved motifs binding to the C-5 carboxy group of 2-oxoglutarate in the 2OGD
634 superfamily are indicated with dashed arrows.

Supplemental Fig. S3. GC-MS analysis of the reaction products obtained from the recombinant St16DOX protein with (22S,25S)-22,26-dihydroxycholesterol as a substrate. (A) The extracted ion chromatograms targeting ion m/z 99 of the reaction products. (B) Mass spectra of the substrate and the product peaks from St16DOX shown in (A) with retention times of 23.8 min and 25.8 min, respectively.

Supplemental Fig. S4. LC-MS analysis of the reaction products from the recombinant St16DOX protein with (22S,25S)-22,26-dihydroxycholesterol as a substrate. (A) Total ion chromatograms of the reaction products. (B) Mass spectra of the substrate and the product peaks from St16DOX shown in (A) with retention times of 18.16 min and 15.16 min, respectively.

Supplemental Fig. S5. GC-MS analysis of the reaction products from the recombinant 16DOX proteins with (22R)-22-hydroxycholesterol as a substrate. (A) The extracted ion chromatograms targeting ion m/z 171 of the reaction products. (B) Mass spectrum of the product peak from St16DOX shown in (A) with a retention time of 21.8 min.

Supplemental Fig. S6. Kinetic analysis of the recombinant St16DOX with (22S,25S)-22,26-dihydroxycholesterol. Enzyme activities were measured with substrate concentrations up to 50 μM (22S,25S)-22,26-dihydroxycholesterol. A Michaelis–Menten curve (featuring a \(K_m\) value of 4.19 μM) was fitted to the values obtained. Kinetic parameters were determined by non-linear regression with ANEMONA (Hernandez and Ruiz, 1998).

Supplemental Fig. S7. LC-MS analysis of α-tomatine levels in the leaves of Sl16DOX-silenced transgenic tomato plants. Bars indicate one standard deviation from the mean (n = 3). FW, fresh weight; NT, non-transgenic control line; #9, #17, #29, and #32, independent transgenic lines.

Supplemental Fig. S8. Phenotypes of St16DOX-silenced transgenic potato plants. (A) Sprouted St16DOX-silenced (#28 and #41, left and middle) and control (right) potato plants 18 weeks after the cessation of plant dormancy in the control. (B) Sprouted St16DOX-silenced potato (#28) planted into soil. (C) In vitro growth of the sprout tips on tissue culture media—sprout tips were cut from tubers of the St16DOX-silenced plant (#28) and placed on tissue culture media without plant hormones. (D) Flowers of an St16DOX-silenced plant (#16).

Supplemental Fig. S9. LC-MS analysis of the accumulated compounds in the leaves of St16DOX-silenced transgenic potato plants. (A) Total ion chromatogram of the accumulated compounds. (B) Mass spectra of the peaks shown in (A) with retention times of 17.44 min, 17.94 min, 20.02 min, 20.56 min, 23.36 min, 24.05 min, and 28.45 min.

Supplemental Fig. S10. GC-MS analysis of the accumulated compounds in leaves of Sl16DOX-silenced transgenic tomato plants. (A) The extracted ion chromatograms targeting ion m/z 171 of the accumulated compounds and the authentic compound. (B) Mass spectra of the peaks shown in (A) with a retention time of 23.8 min. NT, Non-transgenic control line;

#17, a transgenic line.

Supplemental Table S1. The list of a total of 256 2OGD transcripts extracted from Spud DB potato genomics resource.

Supplemental Table S2. The oligonucleotides that were used in the current study.

ACKNOWLEDGMENTS

We thank Masako Otsuka, Noriko Yasuno and Ryo Negoya for technical assistance. Parts of the experimental measurements were carried out using the Bruker and JEOL 600 MHz NMR spectrometers, as well as the JEOL JMS-700 mass spectrometer in the Joint Usage/Research Center (JURC) at the Institute for Chemical Research, Kyoto University. The synthetic research was supported by 2nd Seiken Academic Incentive Fund provided by Research Instituted for Production Development, Kyoto, Japan.

Figure legends

Fig. 1. The putative biosynthetic pathway for SGAs in potato and tomato. Thick solid arrow indicates the reaction step characterized in this work. Thin solid arrows indicate the reaction steps reported by Umemoto et al. (2016). White arrows represent multiple reaction stages.

Fig. 2. LC-MS analysis of SGAs upon feeding stable isotope-labeled compounds to in vitro-grown potato shoots and tomato seedlings. The two traces on the left are

mass-chromatograms for the indicated \( m/z \) ions. The right mass spectrum was obtained at the

retention time indicated by the arrow. Plants were fed with \([15, 15, 17\alpha-2H_3]\) cholesterol (A-C),

\([15, 15, 16\alpha, 17\alpha-2H_4]\) cholesterol (D-F), or \([15, 15, 16\beta, 17\alpha-2H_4]\) cholesterol (G-I). (A, D, and

G) LC-MS analysis of \(\alpha\)-chaconine in potato shoots. (B, E, and H) LC-MS analysis of

\(\alpha\)-solanine in potato shoots. (C, F, and I) LC-MS analysis of \(\alpha\)-tomatine in tomato seedlings.

Fig. 3. Quantitative RT-PCR analysis of the expression patterns of SGA biosynthetic

genes in various organs of potato plants. Transcript levels of SGA biosynthetic genes are

shown relative to that of \(EF1\alpha\) as an internal reference gene. Bars indicate one standard

deviation from the mean (\(n = 3\)).

Fig. 4. GC-MS analysis of the reaction products from the recombinant St16DOX and

Sl16DOX proteins with (22S)-22-hydroxycholesterol as a substrate. (A) The extracted ion

chromatogram targeting ion \(m/z\) 173 of the reaction products and the authentic compounds.

(B) Mass spectrum of each peak shown in (A) at a retention time of 20.0 min.

Fig. 5. Relative activities of recombinant St16DOX toward cholesterol and several

oxygenated cholesterols. N.D. indicates not detected.

Fig. 6. SGA contents and yields of tubers from St16DOX-silenced transgenic potato

plants. (A) Quantification RT-PCR analysis of St16DOX transcript levels in the in

vitro-grown shoots of St16DOX-silenced plants. (B) LC-MS analysis of SGA levels in the in

vitro-grown shoots of St16DOX-silenced plants. (C and D) LC-MS analysis of SGA levels in

the peel (C) and cortex (D) of harvested tubers from St16DOX-silenced plants. (E) Yields of

tubers from St16DOX-silenced plants. Bars indicate one standard deviation from the mean (n

= 3). FW, fresh weight; NT, non-transgenic control plants; #15, #16, #28, #39, and #41,

independent transgenic lines.

Fig. 7. GC-MS analysis of the accumulated compounds in leaves of St16DOX-silenced

transgenic potato plants. (A) The extracted ion chromatogram targeting ion m/z 171 of the

accumulated compounds and the authentic compound. (B) Mass spectrum of each peak shown

in (A) at a retention time of 23.8 min. NT, non-transgenic control line; #15, a transgenic line.

Fig. 8. The putative reaction orders in the biosynthetic pathways of steroidal alkaloids in

Solanum and Veratrum species. Thick arrows indicate the reaction steps that were

characterized in this work. Black-filled arrow indicates the multiple reaction stages reported

by Umemoto et al. (2016) and Augustin et al. (2015). White arrows represent unclear multiple

reaction stages.
# Parsed Citations

Augustin MM, Ruzicka DR, Shukla AK, Augustin JM, Starks CM, O'Neil-Johnson M, McKain MR, Evans BS, Barrett MD, Smithson A, et al (2015) Elucidating steroid alkaloid biosynthesis in Veratrum californicum: production of verazine in Sf9 cells. Plant J 82: 991-1003  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Bugg TDH (2003) Dioxygenase enzymes: catalytic mechanisms and chemical models. Tetrahedron 59: 7075-7101  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Canonica L, Ronchetti F, Russo G (1977) Fate of the 16β-hydrogen atom of cholesterol in the biosynthesis of tomatidine and solanidine. J Chem Soc Chem Commun: 286-287  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Cárdenas PD, Sonawane PD, Pollier J, Vanden Bossche R, Dewangan V, Weithorn E, Tal L, Meir S, Rogachev I, Malitsky S, et al (2016) GAME9 regulates the biosynthesis of steroidal alkaloids and upstream isoprenoids in the plant mevalonate pathway. Nat Commun 7: 10654  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Elias R, Till BJ, Mba C, Al-Safadi B (2009) Optimizing TILLING and Ecotilling techniques for potato (Solanum tuberosum L). BMC Res Notes 2: 141  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Field B, Fiston-Lavier AS, Kemen A, Geisler K, Quesneville H, Osbourn AE (2011) Formation of plant metabolic gene clusters within dynamic chromosomal regions. Proc Natl Acad Sci USA 108: 16116-16121  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Friedman M (2002) Tomato Glycoalkaloids: Role in the Plant and in the Diet. J Agric Food Chem 50: 5751-5780  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Friedman M (2006) Potato glycoalkaloids and metabolites: roles in the plant and in the diet. J Agric Food Chem 54: 8655-8681  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Friedman M, Dao L (1992) Distribution of glycoalkaloids in potato plants and commercial potato products. J. Agric. Food Chem 40: 419-423  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Ginzberg I, Tokuhisa J, Veilleux R (2009) Potato Steroidal Glycoalkaloids: Biosynthesis and Genetic Manipulation. Potato Res 52: 1-15  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Harrison DM (1990) Steroidal alkaloids. Nat Prod Rep 7: 139-147  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Helmut R (1998) Solanum steroid alkaloids - an update. Alkaloids: Chem Biol Perspect 12: 103-185  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Hernandez A, Ruiz MT (1998) An EXCEL template for calculation of enzyme kinetics parameters by non-linear regression. Bioinformatics 14: 227-228  
Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Downloaded from on July 30, 2017 - Published by www.plantphysiol.org  
Copyright © 2017 American Society of Plant Biologists. All rights reserved.Itkin M, Heinig U, Tzfadia O, Bhide AJ, Shinde B, Cardenas PD, Bocobza SE, Unger T, Malitsky S, Finkers R, et al (2013) Biosynthesis of antinutritional alkaloids in solanaceous crops is mediated by clustered genes. Science 341: 175-179

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Itkin M, Rogachev I, Alkan N, Rosenberg T, Malitsky S, Masini L, Meir S, Iijima Y, Aoki K, de Vos R, et al (2011) GLYCOALKALOID METABOLISM1 is required for steroidal alkaloid glycosylation and prevention of phytoxicity in tomato. Plant Cell 23: 4507-4525

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Jacobsen SE, Olszewski NE (1996) Gibberellins regulate the abundance of RNAs with sequence similarity to proteinase inhibitors, dioxygenases and dehydrogenases. Planta 198: 78-86

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Kawai Y, Ono E, Mizutani M (2014) Evolution and diversity of the 2-oxoglutarate-dependent dioxygenase superfamily in plants. Plant J 78: 328-343

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Kaneko K, Mitsuhashi H, Hirayama K, Yoshida N (1970) Dormantinol, a possible precursor in solanidine biosynthesis, from budding Veratrum grandiflorum. Phytochemistry 9: 2489-2495

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Kaneko K, Seto H, Motoki C, Mitsuhashi H (1975) Biosynthesis of rubijervine in Veratrum grandiflorum. Phytochemistry 14: 1295-1301

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Kaneko K, Watanabe M, Mitsuhashi H (1976) Origin of nitrogen in the biosynthesis of solanidine by Veratrum grandiflorum. Phytochemistry 15: 1391-1393

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Kaneko K, Watanabe M, Mitsuhashi H (1977) Dormantinol, a possible precursor in solanidine biosynthesis, from budding Veratrum grandiflorum. Phytochemistry 16: 1247-1251

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Kozukue N, Mizuno S (1985) Studies on Glycoalkaloids of Potatoes (Part II) Analysis of Glycoalkaloid Content in Potato Tissues and Tubers. J Jpn Soc Hortic Sci (Suppl) 54: 496-497

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Kozukue N, Mizuno S (1989) Studies on Glycoalkaloids of Potatoes (Part IV) Changes of Glycoalkaloid Content in Four Parts of a Sprouted Potato Tuber and in Potato Tubers during Storage. J Jpn Soc Hortic Sci 58:231-235

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Lantin S, O'Brien M, Matton DP (1999) Pollination, wounding and jasmonate treatments induce the expression of a developmentally regulated pistil dioxygenase at a distance, in the ovary, in the wild potato Solanum chacoense Bitt. Plant Mol Biol 41: 371-386

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Lukacin R, Britsch L (1997) Identification of strictly conserved histidine and arginine residues as part of the active site in Petunia hybrida flavanone 3β-hydroxylase. Eur J Biochem 249: 748-757

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Lukacin R, Gröning I, Pieper U, Matern U (2000) Site-directed mutagenesis of the active serine290 in flavanone 3β-hydroxylase from Petunia hybrida. Eur J Biochem 267: 853-860

Pubmed: [Author and Title](Author and Title)  
CrossRef: [Author and Title](Author and Title)  
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Matsuda J, Okabe S, Hashimoto T, Yamada Y (1991) Molecular cloning of hyoscyamine 6β-hydroxylase, a 2-oxoglutarate-dependent dioxygenase, from cultured roots of Hyoscyamus niger. J Biochem 266: 9460-9464

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

McCue KF, Allen PV, Shepherd LV, Blake A, Maccree MM, Rockhold DR, Novy RG, Stewart D, Davies HV, Belknap WR (2007) Potato glycosyltransferase, the terminal step in triose side-chain biosynthesis. Phytochemistry 68: 327-334

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

McCue KF, Allen PV, Shepherd LV, Blake A, Whitworth J, Maccree MM, Rockhold DR, Stewart D, Davies HV, Belknap WR (2006) The primary in vivo steroidal alkaloid glucosyltransferase from potato. Phytochemistry 67: 1590-1597

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

McCue KF, Shepherd LVT, Allen PV, Maccree MM, Rockhold DR, Corsini DL, Davies HV, Belknap WR (2005) Metabolic compensation of steroidal glycoalkaloid biosynthesis in transgenic potato tubers: using reverse genetics to confirm the in vivo enzyme function of a steroidal alkaloid galactosyltransferase. Plant Sci 168: 267-273

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Milligan SB, Gasser CS (1995) Nature and regulation of pistil-expressed genes in tomato. Plant Mol Biol 28: 691-711

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Milner SE, Brunton NP, Jones PW, O'Brien NM, Collins SG, Maguire AR (2011) Bioactivities of glycoalkaloids and their aglycones from Solanum species. J. Agric. Food Chem 59: 3454-3484

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Moehs CP, Allen PV, Friedman M, Belknap WR (1997) Cloning and expression of solanidine UDP-glucose glucosyltransferase from potato. Plant J 11: 227-236

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Momma T (1990) Recent study for genetic engineering of soybean glycinin gene. Plant tissue culture lett 7: 57-63

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Moses T, Papadopoulou KK, Osbourn A (2014) Metabolic and functional diversity of saponins, biosynthetic intermediates and semi-synthetic derivatives. Crit Rev Biochem Mol Biol 49: 439-462

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Nakane E, Kawakita K, Doke N, Yoshioka H (2003) Elicitation of primary and secondary metabolism during defense in the potato. J Gen Plant Pathol 69: 378-384

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Nicot N, Hausman JF, Hoffmann L, Evers D (2005) Housekeeping gene selection for real-time RT-PCR normalization in potato during biotic and abiotic stress. J Exp Bot 56: 2907-2914

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Nicolia A, Proux-Wéra E, Åhman I, Onkokesung N, Andersson M, Andreasson E, Zhu LH (2015) Targeted gene mutation in tetraploid potato through transient TALEN expression in protoplasts. J Biotechnol 204: 17-24

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Ohnishi T, Yokota T, Mizutani M (2009) Insights into the function and evolution of P450s in plant steroid metabolism. Phytochemistry 70: 1918-1929

Downloaded from on July 30, 2017 - Published by www.plantphysiol.org
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Ohyama K, Okawa A, Moriuchi Y, Fujimoto Y (2013) Biosynthesis of steroidal alkaloids in Solanaceae plants: involvement of an aldehyde intermediate during C-26 amination. Phytochemistry 89: 26-31

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Petersen HW, Mølgaard P, Nyman U, Olesen CE (1993) Chemotaxonomy of the tuber-bearing Solanum species, subsection Potatoe (Solanaceae). Biochem Syst Ecol 21: 629-644

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Roddick JG (1989) The acetylcholinesterase-inhibitory activity of steroidal glycoalkaloids and their aglycones. Phytochemistry 28: 2631-2634

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Sawai S, Ohyama K, Yasumoto S, Seki H, Sakuma T, Yamamoto T, Takebayashi Y, Kojima M, Sakakibara H, Aoki T, et al (2014) Sterol Side Chain Reductase 2 Is a Key Enzyme in the Biosynthesis of Cholesterol, the Common Precursor of Toxic Steroidal Glycoalkaloids in Potato. Plant Cell 26: 3763-3774

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Seki H, Ohyama K, Sawai S, Mizutani M, Ohnishi T, Sudo H, Akashi T, Aoki T, Saito K, Muranaka T (2008) Licorice ß-amyrin 11-oxidase, a cytochrome P450 with a key role in the biosynthesis of the triterpene sweetener glycyrrhizin. Proc Natl Acad Sci USA 105: 14204-14209

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Seki H, Tamura K, Muranaka T (2015) P450s and UGTs: Key Players in the Structural Diversity of Triterpenoid Saponins. Plant Cell Physiol 56: 1463-1471

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Smith DB, Roddick JG, Jones JL (1996) Potato glycoalkaloids: some unanswered questions. Trends in food science & technology 7: 126-131

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Sun HJ, Uchii S, Watanabe S, Ezura H (2006) A highly efficient transformation protocol for Micro-Tom, a model cultivar for tomato functional genomics. Plant Cell Physiol 47: 426-431

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Thagun C, Imanishi S, Kudo T, Nakabayashi R, Ohyama K, Mori T, Kawamoto K, Nakamura Y, Katayama M, Nonaka S, et al (2016) Jasmonate-Responsive ERF Transcription Factors Regulate Steroidal Glycoalkaloid Biosynthesis in Tomato. Plant Cell Physiol 57: 961-975

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Umemoto N, Nakayasu M, Ohyama K, Yotsu-Yamashita M, Mizutani M, Seki H, Saito K, Muranaka T (2016) Two Cytochrome P450 Monooxygenases Catalyze Early Hydroxylation Steps in the Potato Steroid Glycoalkaloid Biosynthetic Pathway. Plant Physiol 171: 2458-2467

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Umemoto N, Sasaki K (2013). Protein having glycoalkaloid biosynthetic enzyme activity and gene encoding the same. US Patent Application No. 20130167271 A1

Pubmed: [Author and Title](Author and Title)
CrossRef: [Author and Title](Author and Title)
Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Umemoto N, Tsukahara M, Yoshioka M (2001) JP Patent Publication (Kokai) 2001-161373 A

Downloaded from on July 30, 2017 - Published by www.plantphysiol.org  
Copyright © 2017 American Society of Plant Biologists. All rights reserved.

Varma KR, Wickramasinghe JAF, Caspi E (1969) Biosynthesis of plant sterols. IX. The mode of oxygenation at carbon atom 26 in the formation of sapogenins from cholesterol. J Biol Chem 244: 3951-3957

Pubmed: [Author and Title](Author and Title)

CrossRef: [Author and Title](Author and Title)

Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)

Wilmouth RC, Turnbull JJ, Welford RWD, Clifton IJ, Prescott AG, Schofield CJ (2002) Structure and mechanism of anthocyanidin synthase from Arabidopsis thaliana. Structure 10: 93-103

Pubmed: [Author and Title](Author and Title)

CrossRef: [Author and Title](Author and Title)

Google Scholar: [Author Only](Author Only) [Title Only](Title Only) [Author and Title](Author and Title)
