
International Edition: DOI: 10.1002/anie.201705489  
German Edition: DOI: 10.1002/ange.201705489  

### Enzymatic Synthesis of Psilocybin

**Janis Fricke**, **Felix Blei**, and **Dirk Hoffmeister**

#### Abstract: Psilocybin is the psychotropic tryptamine-derived natural product of *Psilocybe* carpophores, the so-called “magic mushrooms”. Although its structure has been known for 60 years, the enzymatic basis of its biosynthesis has remained obscure. We characterized four psilocybin biosynthesis enzymes, namely i) PsiD, which represents a new class of fungal L-tryptophan decarboxylases, ii) PsiK, which catalyzes the phosphotransfer step, iii) the methyltransferase PsiM, catalyzing iterative N-methyl transfer as the terminal biosynthetic step, and iv) PsiH, a monooxygenase. In a combined PsiD/PsiK/PsiM reaction, psilocybin was synthesized enzymatically in a step-economic route from 4-hydroxy-L-tryptophan. Given the renewed pharmaceutical interest in psilocybin, our results may lay the foundation for its biotechnological production.

Numerous mushrooms of the genus *Psilocybe* produce psychotropically active natural products that profoundly change perception when ingested. For centuries, Central American cultures considered these mushrooms divine and have used them for spiritual purposes. More recently, the carpophores have been used as recreational drugs (colloquially dubbed “magic mushrooms”). The pharmacological effects are caused by modified tryptamines,[1] with psilocybin (1, Scheme 1) being the major chemical constituent of these fungi.[2] This prodrug-like natural product becomes rapidly dephosphorylated upon oral ingestion to yield the actual psychotropic agent psilocin (2), which primarily acts as a partial agonist on the 5HT$_{2A}$ receptor in the human central nervous system.[1] Compound 1 has attracted pharmaceutical attention as clinical studies showed a positive trend in the treatment of existential anxiety with advanced-stage cancer patients and for nicotine addiction.[3] Studies on the clinical use of 1 against depression are ongoing.[4]

Hofmann and co-workers elucidated the structures of 1 and 2 in 1959.[2a] Later, the demethylated analogues baeocystin (3) and norbaeocystin (4) and the trimethylated variant aeruginascin (5) were described (Scheme 1).[5] The order of biosynthetic events leading to 1, with its unique 4-phosphoryloxy group at the indole framework, was published in 1968, based on $^{14}$C and $^3$H radiotracer labeling.[6] It was proposed that L-tryptophan (6) first undergoes decarboxylation to yield tryptamine (7), which is followed by successive N,N-dimethylation, C4 hydroxylation, and 4-O-phosphorylation.

Herein, we report on the enzymes for the biosynthesis of 1 in *P. cubensis*, which include an L-tryptophan decarboxylase, a kinase, an S-adenosyl-L-methionine (SAM)-dependent N-methyltransferase, and a monooxygenase. Our results reveal that the biosynthesis does not follow the above order. Using heterologously produced enzymes and 4-hydroxy-L-tryptophan (8) as the substrate, we reconstituted the biosynthetic pathway of 1 in vitro. We identified a new class of fungal L-tryptophan decarboxylases, and provide evidence that N,N-dimethylation is the final step. We therefore present a refined biosynthetic pathway for 1.

The genomes of *P. cubensis* and *P. cyanescens* were sequenced. In fungi, genes encoding a particular biosynthesis are usually co-localized in contiguous gene clusters, in most cases around genes encoding polyketide synthases, peptide synthetases, or terpene cyclases.[7] The biosynthesis of compound 1 does not require such genes. Also, it was elusive whether a gene for an aromatic L-amino acid decarboxylase, such as CsTDC,[8] was present in the cluster as the respective enzyme activity is required for primary metabolism as well. Instead, only genes for a methyltransferase, a hydroxylase, and a kinase are expected for the biosynthesis of 1. In either genome, a locus (*P. cubensis*: 21.8 kb; *P. cyanescens*: 25 kb) was identified that included the expected genes (hereafter referred to as *psiM*, *psiH*, and *psiK*; Figure 1 and the Supporting Information, Table S1), alongside a putative decarboxylase gene (*psiD*). Genes for major facilitator family transporters (*psiT1* and *psiT2*) and a helix loop helix (HLH) domain transcriptional regulator (*psiR*) were also found.

---

[*] J. Fricke, [+] F. Blei, [+] Prof. Dr. D. Hoffmeister  
Department Pharmaceutical Microbiology at the Hans Knöll Institute  
Friedrich-Schiller-Universität  
Beutenbergstrasse 11a, 07745 Jena (Germany)  
E-mail: dirk.hoffmeister@leibniz-hki.de  

[+] These authors contributed equally to this work.  

Supporting information for this article can be found under:  
https://doi.org/10.1002/anie.201705489.

Figure 1. Map of the syntenic loci (psi) for the biosynthesis of 1 in *P. cubensis* (I) and *P. cyanescens* (II). Genes relevant for enzymatic synthesis are labeled in bold and are color-coded. The clusters include genes for a kinase (*psiK*, red), a methyltransferase (*psiM*, green), tryptophan decarboxylase (*psiD*, blue), and a P₄₅₀ monooxygenase (*psiH*, orange). Furthermore, two major-facilitator-type transporters (PsiT1 and PsiT2) and a putative transcriptional regulator (PsiR) are encoded. Hypothetical genes are shown in light gray. Introns are not shown.

Most basidiomycetes are not amenable to genetic manipulation, which precludes reverse genetic strategies or gene silencing.[9] Therefore, we followed an in vitro approach to provide evidence that these loci govern the biosynthesis of compound 1. Given the high degree of similarity of the predicted enzymes encoded in *P. cubensis* and *P. cyanescens* (between 78 and 85% identical amino acids), we chose *P. cubensis* as our model. The cDNAs of *psiD*, *psiK*, and *psiM* were cloned to create the expression plasmids pJF24, pJF23, and pFB13. They were used to individually transform *Escherichia coli* KRX and produce N-terminal hexahistidine fusion proteins, which were purified by metal affinity chromatography (Figure S1).

PsiD is a 49.6 kDa enzyme (439 amino acids in its native form, calculated pI 5.3) that belongs to the PLP-independent phosphatidylserine decarboxylase family (E.C. 4.1.1.65) and is most similar to hypothetical proteins of other basidiomycetes, such as from *Fibulorhizoctonia* sp. (GenBank accession: KZP09902.1, 60% identical amino acids) and *Moniliophthora roreri* (XP_007857499.1, 52% identical amino acids). PsiD is not related to PLP-dependent aromatic L-amino acid decarboxylases (E.C. 4.1.1.28, Figure S2)[8,10] but related to phosphatidylserine decarboxylases.[11] LC-MS analyses showed product formation when 6 was added as a substrate (Figure 2A). Surprisingly, compound 8 was accepted as a substrate by PsiD as well. Fractions with the typical indole UV/Vis spectrum appeared at retention times of \(t_{\mathrm{R}} = 14.0\) min in the reaction with 6 and \(t_{\mathrm{R}} = 9.8\) min with 8. These signals corresponded to the masses of 7 and 4-hydroxytryptamine (9; Figure 2A and Table S2). Therefore, we identified PsiD as a plausible enzyme for the biosynthesis of 1 that belongs to a class of decarboxylases for which 6 has previously not been described as a substrate and that is distinct from other fungal and plant aromatic amino acid decarboxylases. Given the turnover of 8, a step-economic three-enzyme in vitro synthesis of 1 appeared possible.

We next investigated the putative SAM-dependent methyltransferase PsiM (34.5 kDa, 309 amino acids). It is similar to other hypothetical basidiomycete methyltransferases, such as those from *Heterobasidion irregulare* (XP_009549744.1, 52% identical amino acids) and *Dichomitus squalens*

Figure 2. Chromatographic analysis of product formation by the enzymes for the biosynthesis of 1 in vitro. Chromatograms were recorded at \(\lambda = 280\) nm. A) Reactions catalyzed by PsiD. Top trace (overlay of individual chromatograms): Authentic standards of 6–9. Traces a and b: Reaction with 6 as the substrate and negative control (heat-inactivated enzyme), respectively. Traces c and d: Reaction with 8 and negative control. B) Reactions with PsiM. Top traces: Authentic standards of 6–10. Traces a and b: Reaction with 6 and negative control, respectively. Traces c and d: Reaction with 7 and negative control. Traces e and f: Reaction with 8 and negative control. Traces g and h: Reaction with 9 and negative control. C) Reactions with PsiK. Top traces: Authentic standards of 1, 2, 8, and 9. Traces a and b: Reaction with 8 and negative control, respectively. Traces c and d: Reaction with 9 and negative control. Traces e and f: Reaction with 2 and negative control. The signals at 7.7 min occurring both in the negative controls and enzyme-containing reactions are caused by 1,4,5-oxadithiepane, which is formed owing to the presence of β-mercaptoethanol. HRMS spectra for HPLC signals of the compounds given in parentheses are shown on the right.

(XP_007367644.1, 50% identical amino acids). PsiM is a class I methyltransferase featuring a Rossmann fold, with the sequence G-V-D-I-G-T-G-A-S representing a plausible fold core.[12] Based on the proposed order of biosynthesis events towards 1,[6] PsiM was expected to catalyze iterative methyl transfer to the amino group of 7 to yield N,N-dimethyltryptamine (10). Mechanistic considerations made 6 unlikely as a substrate as the decarboxylation step requires a primary amine for Schiff base formation. Assays containing SAM and 6, 7, 8, or 9 were analyzed by LC-MS. However, mono- or dimethylated products were not detected (Figure 2B), and the use of 9 as the substrate resulted only in product traces. A combined PsiD/PsiM assay did not result in methylated products either (Figure 3). Therefore, we

identical in retention time, UV/Vis, and mass spectra to an authentic sample of 1 (Figure 3, trace g; Figure 4). A subsequent scaled-up assay with 15 μmol (3.3 mg) of 8 as the starting material yielded 3.9 μmol of 1.

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

**Figure 3.** LC-MS analysis of coupled enzymatic assays. Chromatograms were recorded at λ = 280 nm. Top trace: Authentic standards of 1–3 and 6–10 (overlay of chromatograms). Trace a: The coupled PsiD/PsiM reaction with 6 as the substrate (trace b: negative control) and 7 as the product. Trace c: The coupled PsiD/PsiM reaction with 8 as the substrate and 9 as the product (trace d: negative control). Trace e: The coupled PsiD/PsiK reaction with 8 as the substrate and formation of 4 (trace f: negative control). Trace g: The combined PsiD/PsiK/PsiM assay (trace h: negative control) with the formation of 1, 3, and 4 in vitro. The signal at t<sub>R</sub> = 4.7 min is due to S-adenosyl-L-homocysteine.

hypothesized that the phosphoryloxy group is a structural prerequisite for PsiM substrates, and we proceeded with an investigation of the phototransfer step.

The putative kinase PsiK (40.4 kDa, 362 amino acids) falls into the 5-methylthioribose family of small-molecule kinases. Highest similarity was found to hypothetical enzymes of *Fibulorhizoctonia* sp. (KZP10121.1, 41% identical amino acids) and *Daedalea quercina* (KZT 74976.1, 31% identical amino acids). Pure PsiK converted 2 into 1, as evident by LC-MS analysis (Figure 2C and Table S2). Comparison with an authentic psilocybin standard showed identical retention times. When 8 was added as substrate, product formation was not detected. This result indicated rejection of this compound by PsiK and that a decarboxylated substrate is required. When 9 was offered as substrate, the formation of 4 was observed (Figure 2C). To confirm the requirement for a decarboxylated substrate and to simultaneously establish another step in the enzymatic synthesis, a coupled PsiD/PsiK assay with 8 as the substrate was run, which resulted in the formation of 4, as shown by LC-MS analysis (Figure 3 and Table S2). We therefore describe PsiK as the dedicated kinase that catalyzes the 4-O-phosphorylation step in the biosynthesis of 1.

PsiK is one of very few biochemically characterized natural product kinases, among them those for paeninodine and calycin biosynthesis.[13] To show methyltransferase activity and to add the third step to our enzymatic approach for the synthesis of 1, a triple PsiD/PsiK/PsiM assay was subsequently set up, using 8 as the substrate. LC-MS analysis unequivocally confirmed the formation of a product that was

**Figure 4.** Production of 1 i) according to the in vitro enzymatic route from 8 and ii) biosynthetically in *P. cubensis* starting from 6. The dashed arrow from 1 to 2 symbolizes hypothetical intracellular dephosphorylation.

Minor amounts of 3 and 4 were detected as well. Therefore, we concluded that compound 4 is the main PsiM substrate, which is produced by PsiD and PsiK and then progressively N,N-dimethylated to yield 1. Our results therefore imply that 2 may not be a true substrate towards 1. Given that PsiK readily turns over 2 into 1, we propose that this activity represents a protective mechanism to re-phosphorylate the unstable compound 2 to the stable 1 in the case of intracellular ester cleavage (Figure 4).

Iterative methyl transfer to small molecules by enzymes has been investigated previously, and RemG was shown to catalyze geminal C-dimethylation during resistomycin assembly.[14] Furthermore, the mycobacterial methyltransferase EgtD progressively catalyzes N-trimethylation of the L-histidine amino group during ergothioneine biosynthesis, and the enzyme structure has also been solved.[15] However, EgtD and PsiM are not closely related as the former is a member of the methyltransferase family 33, whereas the latter falls into family 10. Similarly, PsiM is unrelated to mammalian indolethylamine-N-methyltransferase.[16]

In an earlier study,[17] it had already been speculated that compound 3 might be a precursor of 1, which has now been confirmed by our present work. In addition, it was hypothesized[5a,6] that *P. cubensis* may hydroxylate early, that is, in a mammalian serotonin-like order of biosynthetic events.[18] Given the PsiD flexibility towards 8, it appeared plausible that decarboxylation and 4-hydroxylation may be interchangeable steps. To elucidate the hydroxylation event, the P<sub>450</sub> monooxygenase PsiH was produced heterologously in *Aspergillus niger* tJF02.05. The conversion of 6 or 7, added to

the culture, into 8 or 9, respectively, was investigated by LC-MS (Figure S3). Whereas compound 8 was not detected, conversion of 7 into 9 was observed. As the hydroxylation of N-alkylated tryptamines has been demonstrated already,[17] compound 10 may serve as a PsiH substrate that would yield 2 as an intermediate. The fungus, however, precludes this route by the strict specificity of PsiM. We have therefore demonstrated that the biosynthesis of 1 is a virtually linear process (Figure 4), initiated by PsiD-catalyzed decarboxylation as a gateway step. Compound 2 occurs as a minor species in *Psilocybe* carpophores, and is probably due to intracellular dephosphorylation that is not fully compensated by PsiK. Nonenzymatic conversion of 1 into 2 during isolation may additionally increase the apparent concentration of 2.

Our findings set the stage for the heterologous production of psilocybin (1) with engineered microbial hosts in a controlled environment for pharmaceutical purposes, should the rediscovered pharmaceutical value lead to increased demands.

### Acknowledgements

We thank J. Greßler, J. Wick (both Friedrich-Schiller-Universität Jena), and A. Perner (Hans-Knöll-Institut Jena) for cDNA synthesis, providing psilocybin, and recording high-resolution mass spectra, respectively. D.H. is authorized to handle compounds that are controlled by the Narcotics Act of the Federal Republic of Germany. This work was supported by the Deutsche Forschungsgemeinschaft (DFG grant HO2515/7-1 to D.H.).

### Conflict of interest

The authors declare no conflict of interest.

**Keywords:** biosynthesis · enzymes · kinases · natural products · psilocybin

**How to cite:** Angew. Chem. Int. Ed. **2017**, 56, 12352–12355  
Angew. Chem. **2017**, 129, 12524–12527

---

[1] a) D. E. Nichols, *Pharmacol. Ther.* **2004**, 101, 131–181; b) F. Hasler, U. Grimberg, M. A. Benz, T. Huber, F. X. Vollenweider, *Psychopharmacology* **2004**, 172, 145–156.  
[2] a) A. Hofmann, R. Heim, A. Brack, H. Kobel, A. Frey, H. Ott, T. Petrzilka, F. Troxler, *Helv. Chim. Acta* **1959**, 42, 1557–1572; b) A. Y. Leung, A. H. Smith, A. G. Paul, *J. Pharm. Sci.* **1965**, 54, 1576–1579.  
[3] a) C. S. Grob, A. L. Danforth, G. S. Chopra, M. Hagerty, C. R. McKay, A. L. Halberstadt, G. R. Greer, *Arch. Gen. Psychiatry* **2011**, 68, 71–78; b) M. W. Johnson, A. Garcia-Romeu, M. P. Cosimano, R. R. Griffiths, *J. Psychopharmacol.* **2014**, 28, 983–992.  
[4] A. Mahapatra, R. Gupta, *Ther. Adv. Psychopharmacol.* **2017**, 7, 54–56.  
[5] a) A. Y. Leung, A. G. Paul, *J. Pharm. Sci.* **1968**, 57, 1667–1671; b) D. B. Repke, D. T. Leslie, *J. Pharm. Sci.* **1977**, 66, 113–114; c) N. Jensen, J. Gartz, H. Laatsch, *Planta Med.* **2006**, 72, 665–666.  
[6] a) S. Agurell, J. L. Nilsson, *Acta Chem. Scand.* **1968**, 22, 1210–1218; b) S. Agurell, J. L. Nilsson, *Tetrahedron Lett.* **1968**, 9, 1063–1064.  
[7] D. Hoffmeister, N. P. Keller, *Nat. Prod. Rep.* **2007**, 24, 393–416.  
[8] D. Kalb, J. Gressler, D. Hoffmeister, *ChemBioChem* **2016**, 17, 132–136.  
[9] M. Stadler, D. Hoffmeister, *Front. Microbiol.* **2015**, 6, 127.  
[10] M. P. Torrens-Spence, M. Lazear, R. von Guggenberg, H. Ding, J. Li, *Phytochemistry* **2014**, 106, 37–43.  
[11] a) D. Rontein, W. I. Liu, D. R. Voelker, A. D. Hanson, *Plant Physiol.* **2003**, 132, 1678–1687; b) I. Schuiki, G. Daum, *IUBMB Life* **2009**, 61, 151–162.  
[12] a) R. M. Kagan, S. Clarke, *Arch. Biochem. Biophys.* **1994**, 310, 417–427; b) P. Z. Kozbial, A. R. Mushegian, *BMC Struct. Biol.* **2005**, 5, 19.  
[13] a) S. Zhu, C. D. Fage, J. D. Hegemann, D. Yan, M. A. Marahiel, *J. Biol. Chem.* **2016**, 291, 13662–13678; b) T. Wakimoto, Y. Egami, Y. Nakashima, Y. Wakimoto, T. Mori, T. Awakawa, T. Ito, H. Kenmoku, Y. Asakawa, J. Piel, I. Abe, *Nat. Chem. Biol.* **2014**, 10, 648–655.  
[14] K. Ishida, K. Fritzsch, C. Hertweck, *J. Am. Chem. Soc.* **2007**, 129, 12648–12649.  
[15] a) A. Vit, L. Misson, W. Blankenfeldt, F. P. Seebeck, *ChemBioChem* **2015**, 16, 119–125.  
[16] a) J. Axelrod, *Science* **1961**, 134, 343; b) M. A. Thompson, R. M. Weinshilboum, *J. Biol. Chem.* **1998**, 273, 34502–34510.  
[17] J. Gartz, *J. Basic Microbiol.* **1989**, 29, 347–352.  
[18] W. Lovenberg, H. Weissbach, S. Udenfriend, *J. Biol. Chem.* **1962**, 237, 89–93.

Manuscript received: May 29, 2017  
Accepted manuscript online: August 1, 2017  
Version of record online: August 25, 2017
