
Molecular Microbiology (2017) 106(3), 439–451

doi:10.1111/mmi.13776

First published online 8 September 2017

# A novel lipoate attachment enzyme is shared by Plasmodium and Chlamydia species

Gustavo A. Afanador,$^{1\dagger}$ Alfredo J. Guerra,$^{1}$ Russell P. Swift,$^{1}$ Ryan E. Rodriguez,$^{1\ddagger}$ David Bartee,$^{2}$ Krista A. Matthews,$^{1\dagger}$ Arne Schön,$^{3}$ Ernesto Freire,$^{3}$ Caren L. Freel Meyers$^{2}$ and Sean T. Prigge$^{1*}$

$^{1}$Department of Molecular Microbiology and Immunology, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA.

$^{2}$Department of Pharmacology and Molecular Sciences, Johns Hopkins School of Medicine, Baltimore, MD, USA.

$^{3}$Department of Biology, The Johns Hopkins University, Baltimore, MD, USA.

## Summary

Lipoate is an essential cofactor for enzymes that are important for central metabolism and other processes. In malaria parasites, scavenged lipoate from the human host is required for survival. The *Plasmodium falciparum* mitochondrion contains two enzymes (*PfLipL1* and *PfLipL2*) that are responsible for activating mitochondrial proteins through the covalent attachment of lipoate (lipoylation). Lipoylation occurs via a novel redox-gated mechanism that remains poorly understood. We show that *PfLipL1* functions as a redox switch that determines which downstream proteins will be activated. Based on the lipoate redox state, *PfLipL1* either functions as a canonical lipoate ligase or as a lipoate activating enzyme which works in conjunction with *PfLipL2*. We demonstrate that *PfLipL2* is a lipoyltransferase and is a member of a novel clade of lipoate attachment enzymes. We show that a *LipL2* enzyme from *Chlamydia trachomatis* has similar activity, demonstrating conservation between intracellular pathogens from different phylogenetic kingdoms and supporting the hypothesis that an early ancestor of malaria parasites once contained a chlamydial endosymbiont. Redox-dependent lipoylation may regulate processes such as central metabolism and oxidative defense pathways.

## Introduction

*Plasmodium falciparum* (*Pf*) is an apicomplexan parasite and the causative agent of the most severe form of human malaria (Okiro *et al.*, 2009; Mudhune *et al.*, 2011; Okiro *et al.*, 2011). During their lifecycle, these parasites cycle between sexual and asexual stages in the mosquito vector and human host respectively. Within the human host, malaria parasites first infect the liver followed by a cyclical infection of red blood cells. The latter is the asexual reproductive cycle which corresponds to the symptomatic phase of malaria, with the majority of malaria drugs targeting the blood stage for treatment and cure (Delves *et al.*, 2012). However, recent reports indicate an alarming increase in resistance to currently available drugs, creating a need to identify new drug targets in the parasite (Fidock *et al.*, 2008; Okiro *et al.*, 2011; Dondorp *et al.*, 2011; Phy *et al.*, 2012).

Lipoate metabolism is essential for malaria parasite survival (Allary *et al.*, 2007; Storm, 2012). Lipoate is an essential cofactor for aerobic metabolism in oxidative decarboxylation reactions of 2-oxoacid complexes (comprised of three subunits named E1, E2 and E3) and for the glycine cleavage system (GCV) (Spalding and Prigge, 2010). Lipoate is attached by dedicated enzymes to a conserved lysine in the E2 subunits or the H-protein of the GCV (Spalding and Prigge, 2010). In *Pf*, lipoate metabolism is partitioned between two organelles, namely the apicoplast (a non-photosynthetic plastid organelle) and the mitochondrion (Foth *et al.*, 2003; Wrenger and Müller, 2004; Günther *et al.*, 2005; Allary *et al.*, 2007). Studies in *Pf* and rodent models of malaria demonstrated that the apicoplast harbors the machinery responsible for the biosynthesis of lipoate, which is dispensable in the blood stage but essential for the parasite’s progression from the liver to the blood stage (Foth *et al.*, 2003; Falkard *et al.*, 2013). By contrast, the

Accepted 22 August, 2017. *For correspondence. E-mail sprigge2@jhu.edu; Tel. 443 287 4822; Fax 410 955 0105. Present addresses: $^{\dagger}$Department of Biochemistry, University of Texas Southwestern Medical Center, Dallas, TX, USA $^{\ddagger}$Department of Chemistry and Life Science, United States Military Academy, West Point, New York, USA

© 2017 John Wiley & Sons Ltd

Host Lipoate

![Diagram](attachment:diagram.png)

**Fig. 1.** Mechanism of mitochondrial lipoate metabolism in *P. falciparum*. In vitro data suggest that there are two routes to lipoylate the different substrates in the parasite mitochondrion. In the first route (top) LipL1 specifically lipoylates the H-protein in a two-step ligation reaction using the oxidized form of lipoate. LipL1 possesses a second function as lipoate activating enzyme to generate the conjugate, dihydrolipoyl-AMP under reducing conditions (bottom). The conjugate is then used by a second enzyme, LipL2, that functions as a lipoyl-AMP:Nε-lysine lipoyltransferase to specifically lipoylate BCDH and KDH.

mitochondrion relies entirely on lipoate scavenged from the host (Allary et al., 2007). This pathway has been shown to be essential for parasite survival during both the liver and blood stages (Allary et al., 2007; Storm, 2012; Deschermeier et al., 2012; Falkard et al., 2013). Furthermore, treatment with lipoate analogs, 8-bromooctanoate (8-BrO) or 6,8-dichlorooctanoate (6,8-diClO), causes parasite cell growth inhibition likely due to inhibition of the activity of the mitochondrial lipoylation substrate (Allary et al., 2007; Afanador et al., 2014).

Three lipoylated substrates have been localized to the mitochondrion: the branched chain dehydrogenase (BCDH) (PF3D7_0303700), the α-ketoglutarate dehydrogenase (KDH) (PF3D7_1320800), and the H-protein (PF3D7_1132900) of the GCV (Spalding et al., 2010; Afanador et al., 2014). These three mitochondrial proteins are lipoylated through two distinct routes involving the proteins PfLipL1 (PF3D7_1314600) and PfLipL2 (PF3D7_0923600). PfLipL1 is the only lipoate ligase in the mitochondrion and is solely responsible for lipoylation of the H-protein (Afanador et al., 2014). BCDH and KDH are lipoylated through a second mechanism that requires both PfLipL1 and PfLipL2 (Afanador et al., 2014). These two lipoylation pathways have a strong dependence on the lipoate redox state since lipoylation of the BCDH and KDH requires fully reduced lipoate (dihydrolipoate), while lipoylation of the H-protein does not (Afanador et al., 2014). This redox-dependent mechanism of controlling lipoate metabolism has not been described in other organisms. Although PfLipL1 and PfLipL2 are both required for protein lipoylation in the mitochondrion, the mechanistic role that each protein plays remains poorly understood. Furthermore, the nature of the molecular switch that determines which lipoylation route proceeds (and therefore which parasite proteins will be activated through lipoylation) has remained elusive in previous studies.

Herein we use recombinant lipoate attachment enzymes and the lipoylation domains (LD) of substrate proteins to understand how proteins are lipoylated in malaria parasites. We show that lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub> in the presence of chemically synthesized lipoyl-AMP and PfLipL2 is independent of PfLipL1. This is consistent with the idea that the lipoylation of the BCDH<sub>LD</sub> and KDH<sub>LD</sub> proceeds through a sequential mechanism in which PfLipL1 activates lipoate with ATP to form lipoyl-AMP, and PfLipL2 acts as a lipoyltransferase to lipoylate the BCDH<sub>LD</sub> and KDH<sub>LD</sub> utilizing the lipoyl-AMP as a substrate (Fig. 1). This work also shows that PfLipL2 is not redox sensitive, but rather it is PfLipL1 that recognizes the redox state of lipoate and acts as a molecular switch defining which substrate proteins will be lipoylated downstream. Lastly, PfLipL2 represents a new family of lipoate attachment enzymes that is conserved across kingdoms, including other *Plasmodium* species and the bacterial pathogen *Chlamydia trachomatis*.

### Results

PfLipL2 is an enzyme with lipoyl-AMP:Nε-lysine lipoyltransferase activity

Lipoylation of Pf mitochondrial substrate proteins proceeds through two distinct pathways. PfLipL1 is sufficient to lipoylate the H-protein, whereas both PfLipL1 and PfLipL2 are required to lipoylate the BCDH<sub>LD</sub> and KDH<sub>LD</sub> (Afanador et al., 2014). Additionally, only PfLipL1 is capable of forming the intermediate lipoyl-AMP. Thus, the lipoylation of the BCDH<sub>LD</sub> and KDH<sub>LD</sub> likely occurs

Fig. 2. LipL1 exclusively lipoylates the H-protein *in vitro* and LipL2 has lipoyltransferase activity for BCDH<sub>LD</sub> and KDH<sub>LD</sub>.

A. LipL1 can use lipoyl-AMP as a substrate for the H-protein lipoylation reaction (lane 1), but not for lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub> (lanes 2 and 3).

B. Alanine substituted LipL1<sub>K160A</sub> cannot use lipoyl-AMP as substrate to lipoylate the H-protein (lane 1) nor is it able to lipoylate the BCDH<sub>LD</sub> and KDH<sub>LD</sub> in the absence of LipL2 (lanes 3 and 5); this mutant does not prevent lipoylation of the BCDH<sub>LD</sub> and KDH<sub>LD</sub> in the presence of LipL2 (lanes 2 and 4).

C. LipL2 acts as a lipoyltransferase by using lipoyl-AMP as substrate to lipoylate the BCDH<sub>LD</sub> and KDH<sub>LD</sub> but cannot lipoylate the H-protein. Note a minor population of E. coli E2-KDH found in purified PfLipL2.

through a coupled reaction where PfLipL1 forms lipoyl-AMP and PfLipL2 acts either as a lipoyltransferase or as an effector protein. To better understand the transfer reaction we chemically synthesized the lipoyl-AMP intermediate and probed the activity of each enzyme with the different Pf mitochondrial substrates. First, we tested the transfer activity of PfLipL1 against each substrate (Fig. 2A). PfLipL1 lipoylates the H-protein (Fig. 2A, lane 1) but does not lipoylate the BCDH<sub>LD</sub> and KDH<sub>LD</sub> in the presence of lipoyl-AMP (Fig. 2A, lanes 2–3). This is consistent with the previous observation that PfLipL1 is unable to lipoylate the BCDH<sub>LD</sub> or KDH<sub>LD</sub> in a ligation reaction (Afanador et al., 2014).

Next, we probed the role of PfLipL1 in the coupled lipoylation reaction of BCDH<sub>LD</sub> and KDH<sub>LD</sub> by using a catalytically inactive mutant (PfLipL1<sub>K160A</sub>). PfLipL1<sub>K160A</sub> has been shown to prevent the lipoylation of all substrates and is unable to synthesize the intermediate lipoyl-AMP (Afanador et al., 2014). To test the activity of PfLipL1<sub>K160A</sub> in a lipoate transfer reaction, we incubated PfLipL1<sub>K160A</sub> and H-protein with lipoyl-AMP. As shown in Fig. 2B, this mutant is unable to catalyze the transfer of lipoate to the H-protein. Additionally, we tested PfLipL1<sub>K160A</sub> in the presence of PfLipL2, BCDH<sub>LD</sub>, KDH<sub>LD</sub>, and lipoyl-AMP. This mutant does not prevent the lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub> (Fig. 2B, lanes 2 & 4) suggesting a possible catalytic role by PfLipL2. Taken together, these results are consistent with the idea that the role of PfLipL1 in the combined reaction is to produce lipoyl-AMP and that PfLipL2 plays a catalytic role in the lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub>.

To determine whether PfLipL2 activity requires an interaction with PfLipL1, we tested PfLipL2 in the presence of lipoyl-AMP alone. As shown in Fig. 2C, PfLipL2 alone is unable to lipoylate the H-protein (lane 1); however, PfLipL2 can recognize and lipoylate BCDH<sub>LD</sub> or KDH<sub>LD</sub> in the presence of lipoyl-AMP (lanes 2 and 3). These results indicate that PfLipL2 acts independently as a lipoyl-AMP:N<sup>ε</sup>-lysine lipoyltransferase with substrates

BCDH<sub>LD</sub> and KDH<sub>LD</sub>. Thus, PfLipL1 and PfLipL2 appear to catalyze successive enzymatic steps with PfLipL1 responsible for the formation of lipoyl-AMP and PfLipL2 responsible for transferring the lipoyl moiety from lipoyl-AMP to BCDH<sub>LD</sub> and KDH<sub>LD</sub>.

PfLipL1 is responsible for redox-dependent lipoylation

Previous work showed that reduced lipoate is a requirement for the lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub> (Afanador et al., 2014). Since PfLipL2 catalyzed the attachment of lipoate, we hypothesized that PfLipL2 only recognizes the reduced state of lipoyl-AMP. To test this, we incubated PfLipL2 with lipoyl-AMP in the presence or absence of different reducing agents (DTT, THP and TCEP) and tested the lipoyl transfer reaction with each of the mitochondrial substrate proteins. As shown in Fig. 3A (lanes 1–4), PfLipL2 does not transfer lipoate to the H-protein under any redox conditions, consistent with previous observations that H-protein is only lipoylated by PfLipL1. Surprisingly, both BCDH<sub>LD</sub> and KDH<sub>LD</sub> were lipoylated under all conditions tested implying that PfLipL2 is not redox sensitive (Fig. 3A, lanes 5–12).

We next wanted to determine the redox state of lipoate (or lipoyl-AMP) under the reaction conditions used in our lipoylation assays. To accomplish this, we developed a method to measure the reduction of lipoate using the mitochondrial Pf dihydrolipoamide dehydrogenase (mLipDH or E3 subunit, PF3D7_1232200). PfmLipDH is a flavin dependent enzyme that can catalyze the oxidation of dihydrolipoamide with the concomitant reduction of NAD<sup>+</sup> to NADH (McMillan et al., 2004). Recombinant PfmLipDH was expressed and purified, and the production of NADH was monitored by absorbance at 340 nm in the presence of lipoate and various reducing agents. At pH 7.5, TCEP rapidly reduces lipoate while DTT displays little reduction activity (Supporting Information Fig. S1). At pH 9.3, corresponding to the

LipL2 + LA-AMP

| H-protein | KDH<sub>LD</sub> | BCDH<sub>LD</sub> |
| --- | --- | --- |
| 50 |  |  |
| 37 |  |  |

| 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| DTT | - | + | - | - | + | - | - | + | - |  |  |
| THP | - | + | - | - | + | - | - | + | - |  |  |
| TCEP | - | + | - | - | + | - | - | + | - |  |  |

PfLipL1 + LA + ATP

| H-protein/EcKDH | 50 | H-protein |
| --- | --- | --- |
| α-LA |  |  |

| 1 | 2 | 3 | 4 |
| --- | --- | --- | --- |
| DTT | - | + | - |  |
| TCEP | - | + | - |  |
| THP | - | + | - |  |

Fig. 3. Redox dependence of LipL1 and LipL2 lipoylation activity in vitro. A. PfLipL2 does not lipoylate the H-protein under any redox conditions (lanes 1–4). The BCDH<sub>LD</sub> and KDH<sub>LD</sub> are lipoylated independent of the reducing potential (lanes 5–8 and 9–12). Thus, LipL2 is likely not responsible for the redox sensitivity previously observed in the combined reaction.  
B. Addition of TCEP and THP, a TCEP analog, (lanes 3 and 4) but not DTT (lane 2) completely abrogates H-protein lipoylation activity by LipL1.

pKa of DTT (Lukesh et al., 2012), the amount of reduced DTT increases slightly, presumably due to the increased prevalence of the active thiolate form of DTT at higher pH. These results are consistent with the observation that TCEP, but not DTT, is an effective reducing agent for lipoate (Burns et al., 1991).

To test the hypothesis that PfLipL1 acts as a sensor for the redox state of lipoate, we incubated lipoate with a strong reducing agent (TCEP or THP) for a 5-min period prior to a ligation reaction to ensure full reduction. We then proceeded to add PfLipL1 to test its ability to lipoylate the H-protein with dihydrolipoate. As seen in Fig. 3B, PfLipL1 is unable to lipoylate the H-protein with dihydrolipoate, but can modify it with lipoate (treated with DTT or no reducing agent). Taken together these observations implicate PfLipL1 as a redox switch determining the fate of scavenged lipoate in the parasite mitochondrion.

and 6,8-dichlorooctanoyl-AMP (6,8-diClO-AMP)]. Additionally, we characterized the PfLipL1-ligand interaction by isothermal titration calorimetry when the conditions were appropriate, and the results are summarized in Table 1. In the absence of reducing agent, both lipoate and lipoyl-AMP bind tightly to PfLipL1 with relative binding affinities (K<sub>d</sub>) of 0.130 μM and 0.020 μM respectively. By contrast, PfLipL1 has a K<sub>d</sub> of 94 μM and 5.8 μM for dihydrolipoate and dihydrolipoyl-AMP respectively. This accounts for a 723-fold weaker affinity between the oxidized and reduced forms of lipoate and a 290-fold drop in affinity between the oxidized and reduced forms of lipoyl-AMP. Furthermore, the reduced state mimics 6,8-diClO and 6,8-diClO-AMP bind with K<sub>d</sub> values of 13 μM and 0.09 μM respectively. These two analogs are virtually unaffected by the presence of reducing agent which results in a K<sub>d</sub> of 18 μM and 0.25 μM for 6,8-diClO and 6,8-diClO-AMP respectively. This is equivalent to only a 1.4-fold and 2.8-fold drop in affinity in the presence of reducing agent. PfLipL1 binds tightly to lipoyl-AMP (oxidized form) and is able to modify the H-protein in a canonical two-step ligation reaction. However, under reducing conditions, the affinity of PfLipL1 for dihydrolipoyl-AMP (reduced form) shifts 290 fold from 0.02 μM to 5.8 μM. The weaker binding affinity of dihydrolipoyl-AMP may enhance direct transfer of the conjugate to PfLipL2, or facilitate release of the conjugate, making it available for subsequent binding to PfLipL2.

TCEP reduces the affinity of PfLipL1 for lipoate and lipoyl-AMP

To further characterize the redox sensitivity of PfLipL1, we used a thermal shift assay (Afanador et al., 2013) to extract binding constants from the stabilization of PfLipL1 upon binding of lipoate, the intermediate lipoyl-AMP, and dichloro analogs of these compounds that mimic the reduced state of lipoate [6,8-dichlorooctanoate (6,8-diClO)]

Table 1. Binding affinities of Lipoate, 6,8-diClO, and adenylated substrates for PfLipL1 under reducing and non-reducing conditions determined by thermal shift assay (TSA) and isothermal titration calorimetry (ITC).

| Substrate | Concentration in TSA (μM) | K<sub>d</sub> TSA (μM) | Fold change | K<sub>d</sub> ITC (μM) |
| --- | --- | --- | --- | --- |
| Lipoate | 10 | 0.130 ± 0.045 |  | 0.4 ± 0.1 |
| Lipoate + TCEP | 200 | 94 ± 30 | 723 | n.d. |
| Lipoyl-AMP | 10 | 0.020 ± 0.005 |  | 0.065 ± 0.005 |
| Lipoyl-AMP + TCEP | 20 | 5.8 ± 0.4 | 290 | n.d. |
| 6,8-diClO | 100 | 13 ± 4 |  | n.d. |
| 6,8-diClO + TCEP | 100 | 18 ± 1 | 1.4 | n.d. |
| 6,8-diClO-AMP | 10 | 0.09 ± 0.03 |  | 0.10 ± 0.03 |
| 6,8-diClO-AMP + TCEP | 10 | 0.25 ± 0.02 | 2.8 | 0.5 ± 0.2 |

Table 2. Sequence identity of relevant lipoate ligases and lipoate attachment enzymes.

| PfLipL1 | PfLipL2 | PbLipL2 | EcLplA | BtLipT | CtLipL2 |
|---------|---------|---------|--------|--------|----------|
| PfLipL1 | –       | 13%     | 34%    | 27%    | 16%      |
| PfLipL2 | 13%     | –       | 46%    | 10%    | 29%      |

Plasmodium falciparum (Pf); Plasmodium berghei (Pb); Escherichia coli lipoate ligase A (EcLplA); Bos taurus lipoyl-AMP:Nε-lysine lipoyltransferase (BtLipT).

An active site lysine is essential for PfLipL2 activity

Previous studies functionally characterized PfLipL1 including its key catalytic residue (K160) (Afanador et al., 2014). In most characterized lipoylation systems, a conserved lysine residue in the active site facilitates transfer of the lipoyl moiety from lipoyl-AMP (Fujiwara et al., 2005, 2007, 2010). However, sequence alignment failed to identify any essential catalytic residue in PfLipL2. In contrast to PfLipL1, PfLipL2 shares low sequence identity to Escherichia coli lipoate ligase (EcLplA) and Bos taurus lipoyl-AMP:Nε-lysine lipoyltransferase (BtLipT) (Table 2) and does not have an apparent conserved lysine residue when aligned to these

or other characterized enzymes. In order to identify an essential residue in PfLipL2, we generated a homology model of PfLipL2 based on the BtLipT structure (PDB: 2E5A) using the program i-Tasser (Fig. 4A) (Zhang, 2008). Despite poor alignment in a pairwise sequence alignment of BtLipT and PfLipL2, the resulting homology model of PfLipL2 contains a lysine residue that is positioned in space near the active site lysine (K135) of BtLipT (Fig. 4A, inset). This lysine residue in PfLipL2 (K219) is found downstream of conserved regions in lipoate attachment enzymes (vide infra). We proceeded to express and purify an alanine mutant at position 219 (PfLipL2_K219A) and tested the recombinant protein. In a coupled reaction with PfLipL1, PfLipL2_K219A prevents the lipoylation of BCDH_LD and KDH_LD (Fig. 4B). As predicted, PfLipL2_K219A is unable to use lipoyl-AMP as a substrate (Fig. 4C), implying that K219 is involved in the lipoyltransferase reaction. Thus, PfLipL2 acts as an lipoyl-AMP:Nε-lysine lipoyltransferase with an activity analogous to that of BtLipT (Fujiwara et al., 1994, 2007). These results are consistent with the model for lipoylation in P. falciparum mitochondrion summarized in Fig. 1.

Fig. 4. LipL2 acts as an N-lysine lipoyltransferase.

A. i-Tasser generated model of LipL2 (cyan) based on BtLipT (PDB: 2E5A) overlaid with the crystal structure of BtLipT (purple) (Zhang, 2008). Catalytic lysine residues and lipoyl-AMP are shown as stick representation and labeled accordingly.

B. Mutational analysis of LipL2 shows that Lys219 but not Cys93 (Afanador et al., 2014) or Cys276 (Afanador et al., 2014) is essential for LipL2 lipoyltransferase activity.

C. LipL2_K219A is unable to use lipoyl-AMP to lipoylate the BCDH_LD and KDH_LD.

B

| PfLipL1 + LA + ATP |
|---------------------|
| KDHL_D              | BCDHL_D            |
|                     |                    |
| 50                  |                    |
| 37                  |                    |
|                     |                    |
| 1                   | 2                  | 3                  | 4                  | 5                  | 6                  | 7                  | 8                  |
| PfLipL2             | +                  | -                  | +                  | -                  |                    |                    |
| PfLipL2_C93A        | -                  | +                  | -                  | +                  | -                  |                    |
| PfLipL2_C276A       | -                  | +                  | -                  | +                  |                    |                    |
| PfLipL2_K219A       | -                  | +                  | -                  | +                  |                    |                    |
| THP                 | +                  |                    |                    |                    |                    |                    |

C

| LA-AMP |
|--------|
| KDHL_D | BCDHL_D |
| 50     |         |
| 37     |         |
| 1      | 2       | 3       | 4       |
| PfLipL2 | +       | -       | +       | -       |
| PfLipL2_K219A | -       | +       | -       | +       |
| TCEP    | +       | +       | +       | +       |

Fig. 5. LipL2 transferase activity is conserved in a variety of organisms.

A. PblipL2 is unable to lipoylate any substrate proteins in the absence of lipoyl-AMP conjugate. Addition of lipoyl-AMP results in lipoylation of KDHLD and BCDHLD in the presence of strong reducing agents.

B. Putative Ct substrate proteins can be lipoylated in E. coli, demonstrating that they contain lipoylation domains.

C. CtlipL2 can lipoylate CtkDHLD and CtBCDHLD either in the presence or absence of strong reducing conditions. However, Cth-protein and CtpDHLD are not lipoylated under the same conditions.

D. CtlipL2 is unable to catalyze the lipoylation of any substrates in the absence of lipoyl-AMP conjugate, analogous to what is observed for PfLipL2.

LipL2-like activity is present in human pathogens across kingdoms

We next probed the conservation of LipL2-like enzymes in other organisms. Searching for conserved sequence motifs we found a unique pair of motifs that define LipL2-like enzymes. Pattern 1, known as the 'lipoate-binding loop' ([RK]-R-x-[GST]-G-G-[GAQ]-[ACT]-[VI]), is conserved in all lipoate attachment enzymes (Fujiwara et al., 2010). A second region called the 'adenylate-binding loop' is typically characteristic of lipoate attachment enzymes (including PfLipL1, BtLipT and HsLipT), but is not present in LipL2-like enzymes. However, we identified a second conserved pattern (K-x(2)-G-[NSA]-[AS]-[QEL]-x(27,31)-P-x(3)-P-x-[HY]-R-x(2)-R-x-H-x(2)-F-[VILM]) in LipL2-like enzymes located downstream of the 'lipoate-binding loop' (Supporting Information Fig. S2). Using these conserved patterns we identified a family of LipL2-like enzymes across a broad spectrum of organisms including an enzyme in Plasmodium berghei and Chlamydia trachomatis (Supporting Information Table S1).

We cloned, purified, and tested the activity of the putative LipL2-like enzymes from P. berghei (PbLipL2) and C. trachomatis (CtlipL2) for lipoyl-AMP:Nε-lysine

parasite congeners (Fig. 5C). Finally, CtLipL2 lipoate ligase activity was probed in the presence of lipoate and ATP under different reducing conditions. Consistent with what was reported in the literature (Ramaswamy and Maurelli, 2010), CtLipL2 is not a lipoate ligase and is unable to lipoylate any Ct substrate under any of the conditions tested (Fig. 5D). Taken together these data show that CtLipL2 is a lipoyltransferase with an activity similar to what we observed for PfLipL2 and PbLipL2.

**Discussion**

To understand the role of PfLipL1 in the coupled PfLipL1/PfLipL2 lipoylation reaction we generated an inactive mutant, PfLipL1<sub>K160A</sub>. PfLipL1<sub>K160A</sub> is unable to lipoylate any substrate enzyme in an ATP-dependent reaction presumably due to the inability to form the lipoyl-AMP conjugate (Afanador et al., 2014). A similar result was observed in a K133A LplA mutant in E. coli (Fujiwara et al., 2010). Even in the presence of chemically synthesized lipoyl-AMP, PfLipL1<sub>K160A</sub> is unable to catalyze the transfer reaction with the H-protein (Fig. 2B, lane 1), highlighting the importance of this Lys residue in both ligation and transfer reactions. As expected, PfLipL1<sub>K160A</sub> was unable to modify either the BCDH or KDH in a transfer reaction but did not prevent their lipoylation when PfLipL2 was present in the reaction (Fig. 2B). Thus, PfLipL2 is sufficient to modify the BCDH and KDH when lipoyl-AMP is supplied (Fig. 2C). These observations are consistent with the characterization of PfLipL1 as the only canonical lipoate ligase in the parasite mitochondrion with a mechanism of action analogous to what was observed for the E. coli lipoate ligase (Fujiwara et al., 2010). However, unlike in E. coli, where the lipoate ligase is sufficient to modify all canonical substrates in a ligation reaction, PfLipL1 can only modify the H-protein, and not the BCDH or KDH.

The mechanism for the lipoylation of the BCDH and KDH is more similar to the bovine system in which it has been reported that a lipoate activating enzyme activates lipoate with GTP, and a lipoyl-GMP:N<sup>ε</sup>-lysine lipoyltransferase (BtLipT) then transfers the lipoyl moiety from the formed conjugate (lipoyl-GMP) to the substrates (Fujiwara et al., 1994, 2001, 2007). We have previously shown that PfLipL1, and not PfLipL2, is capable of forming the intermediate, lipoyl-AMP (Afanador et al., 2014). The data presented in this work implicate PfLipL2 as a lipoyl-AMP:N<sup>ε</sup>-lysine lipoyltransferase which lipoylates the BCDH and KDH, but with several differences compared to any other described system. First, LipL1 has dual activity as the lipoate ligase required for H-protein lipoylation and as a lipoate-activating enzyme catalyzing the formation of the lipoyl-

AMP conjugate. In mammalian systems, the latter role has been reported to be catalyzed by a medium chain acyl-CoA synthetase (Fujiwara et al., 2001). Second, mammalian systems use GTP to activate lipoate (Fujiwara et al., 2001), whereas Pf uses ATP exclusively (Afanador et al., 2014). Finally and perhaps most strikingly, is the difference in the dependence on the redox state of lipoate. In the mammalian system, lipoate activation and subsequent transfer by BtLipT proceeds in the presence of DTT which is insufficient to reduce lipoate (Fujiwara et al., 1994). By contrast, Pf lipoylation of BCDH and KDH requires the reduction of lipoate for the reaction to proceed and does not occur in the presence of DTT (Afanador et al., 2014).

Redox dependent lipoylation in P. falciparum mitochondria may affect central metabolism through the coordinated activation of BCDH and KDH. Work by other groups has shown that BCDH functions as a pyruvate dehydrogenase generating acetyl-CoA in the mitochondrion, and that KDH has the expected enzymatic activity of an alpha-ketoglutarate dehydrogenase (Cobbold et al., 2013; Oppenheim et al., 2014; Ke et al., 2015). These proteins are both components of the tricarboxylic acid (TCA) cycle, raising the possibility that protein lipoylation regulates the activity of this pathway. TCA cycle activity is thought to be critical for parasite survival during different life cycle stages, such as the mosquito stage (Ke et al., 2015; Srivastava et al., 2016), suggesting a critical role for lipoylation during these stages of parasite development. Additionally, lipoylation of BCDH may be important for generating acetyl-CoA for acetylation reactions in other subcellular compartments of the parasite, such as the cytosol and nucleus. A requirement for acetyl-CoA in these compartments may explain the toxicity of the lipoate analog 6,8-diClO which appears to be covalently attached to BCDH and KDH in treated parasites (Afanador et al., 2014). Since KDH and other TCA cycle enzymes are dispensable in blood stage P. falciparum parasites (Ke et al., 2015), inhibition of BCDH is presumably responsible for blood stage parasite growth inhibition by 6,8-diClO.

The role of the H-protein is more difficult to discern because it is unlikely to function as part of a glycine cleavage system due to the lack of an identifiable P-protein in the parasite genome (Salcedo et al., 2005; Nickel et al., 2006; Spalding and Prigge, 2010). An intriguing possibility is that the H-protein could function as part of an oxidative stress pathway analogous to those observed in T. vaginalis and M. tuberculosis (Bryk et al., 2002; Nývltová et al., 2016). A similar mechanism involving the antioxidant properties of lipoate may be important for malaria parasites (Bunik, 2003; Müller, 2004). In this context, the redox-dependent lipoylation of the H-protein that we observe in vitro may make sense

© 2017 John Wiley & Sons Ltd, *Molecular Microbiology*, **106**, 439–451

LipB

![Diagram](attachment:diagram.png)

Fig. 6. Cofactor transferase protein superfamily. A phylogenetic tree from Pfam entry PF03099 was generated using Dendroscope based on Christensen et al. (Huson et al., 2007; Finn et al., 2010; Christensen et al., 2011). (Left) The blue clade contains EcLipB and other octanoyl transferases, the green clade contains EcBirA and other biotin-protein ligases, and the red clade contains EcLplA and other lipoate attachment enzymes. (Right) The lipoate attachment enzyme clade has been expanded and colored based on characterized enzyme activities as follows: sequences similar to *E. coli* LplA lipoate ligase (Morris et al., 1995) are colored red, sequences similar to *Listeria monocytogenes* LipL amidotransferase (Christensen et al., 2011) are colored purple, sequences similar to *Bacillus subtilis* LipM octanoyltransferase (Christensen and Cronan, 2010) are colored blue, sequences similar to *Bos taurus* LipT mammalian lipoyltransferase (Fujiwara et al., 1994) are colored turquoise, and LipL2-like activity is colored green. Representative member proteins are highlighted by a circle of the corresponding color and labeled accordingly.

since H-protein lipoylation would tend to increase under conditions of elevated oxidative stress. This scenario is not hard to imagine since the glutathione redox potential in the mitochondria of blood stage *P. falciparum* parasites is –328 mV (Mohring et al., 2017), a value similar to that of lipoate.

Orthologs of *Pf*LipL1 and *Pf*LipL2 are found in several other organisms, suggesting that lipoylation mechanisms may be conserved. We tested the activity of a rodent malaria parasite (*Plasmodium berghei*) LipL2 with the *Pf* substrates and found that this enzyme has lipoyltransferase activity and lipoylates the same proteins (BCDH and KDH, but not the H-protein) as *Pf*LipL2 (Fig. 5A). Additionally, we studied a LipL2 ortholog from the intracellular pathogen *Chlamydia trachomatis*. Similar to *Plasmodium* species, *C. trachomatis* possesses a LipL1-like enzyme which has been shown to be an active ligase, but no enzymatic activity was described for *Ct*LipL2 (Ramaswamy and Maurelli, 2010). We now show that *Ct*LipL2 behaves as a lipoyltransferase with its cognate substrates and that it does not possess any ligation activity *in vitro* (Fig. 5C,D). *Pf*LipL2 and other LipL2-like enzymes are members of the cofactor transferase family (PF03099) and appear to form a distinct clade of lipoate attachment enzymes found in prokaryotic and eukaryotic species (Fig. 6) (Finn et al., 2010; Christensen et al., 2011). LipL2 orthologs found in these species may be evidence of horizontal gene transfer from an ancient chlamydial endosymbiotic event with a primary photosynthetic eukaryote (Huang and Gogarten, 2007).

The *Pf*LipL1/*Pf*LipL2 reaction proceeds in a sequential order, however, it remains unclear if a direct interaction between the two enzymes needs to take place or if dihydrolipoyl-AMP is released from *Pf*LipL1 due to the low binding affinity. We decided to address this question by determining whether there is a specific interaction between LipL1 and LipL2 enzymes consistent with a direct transfer mechanism (Supporting Information Fig. S4). We found that *Pf*LipL1 and *Pb*LipL2 can function together to lipoylate both *Pf*KDH<sub>LD</sub> and *Pf*BCDH<sub>LD</sub>. Similarly, we found that *Pf*LipL1 and *Ct*LipL2 can lipoylate both *Ct*KDH<sub>LD</sub> and *Ct*BCDH<sub>LD</sub>. The ability of *Pf*LipL1 to function with parasite or bacterial LipL2 enzymes suggests that there is not a specific protein-protein interaction between LipL1 and LipL2 enzymes. These results are consistent with the release of dihydrolipoyl-AMP from LipL1 followed by subsequent binding to LipL2. It is also possible that despite the low sequence identity between *Pf*LipL2 and *Ct*LipL2 (22%) there is some conserved sequence and structure motif responsible for cross-species recognition.

In the current study, we have expanded the understanding of mitochondrial lipoylation in *Pf*. The parasite mitochondrion contains two lipoate attachment enzymes (*Pf*LipL1 and *Pf*LipL2) and three lipoate requiring substrate enzymes (H-protein, BCDH and KDH) that are lipoylated through two mutually exclusive routes gated by
the redox potential. The BCDH and KDH are lipoylated with lipoyl-AMP as the *PfLipL2* substrate. Contrary to previously proposed models, *PfLipL2* is not a lipoate ligase but rather acts as a lipoyl-AMP:Nε-lysine lipoyltransferase. Additionally, we show that *PfLipL2* activity is not sensitive to redox conditions, but rather that *PfLipL1* appears to be the redox switch that determines the fate of scavenged lipoate in the mitochondrion. Lastly, *PfLipL2* is part of a new family of attachment enzymes that is conserved across Kingdoms. We show that *PbLipL2* and *CtLipL2* not only act as lipoyltransferases, but also seem to have the same conserved substrate specificity. Given the recent work describing a lipoate-dependent redox system involving the H-protein of *T. vaginalis* (Nývltová et al., 2016), it is an intriguing possibility that the redox sensitivity described herein may be part of a larger regulatory mechanism that is linked to the redox state of parasite mitochondria.

### Experimental procedures

#### Materials

The reducing agents used in this study were obtained from Fluka (TCEP; tris(2-carboxyethyl)phosphine), Calbiochem (THP; tris(hydroxypropyl)phosphine), and Sigma-Aldrich (DTT; dithiothreitol). Antibodies were purchased from Calbiochem (rabbit α-LA polyclonal antibodies) and GE Healthcare (donkey α-Rabbit IgG HRP conjugate). Specific rabbit polyclonal antibodies (*α-LA₈₉₁*) were generated and purified as described below. The following chemicals were purchased from Sigma-Aldrich: ATP, NAD⁺ and *R*-LA. The compound 6,8-diClO was synthesized as described in reference (Afanador et al., 2014). The adenylated conjugates 6,8-diClO-AMP and lipoyl-AMP were synthesized as described below.

#### Synthesis of lipoyl-AMP and 6,8-dichlorooctanoyl-AMP

Procedures for synthesis of Lipoic Acid-Adenosine Monophosphate Mixed Anhydrides were based on Reed et al. with modifications (Reed et al., 1958). All reagents were obtained from commercial suppliers and used without further purification unless stated. Acetonitrile was distilled after drying on CaH₂ then stored over 3Å molecular sieves. Yields of all reactions refer to the purified products. ¹H spectra were acquired on a Bruker Avance III 500 spectrometer operating at 500 MHz. Chemical shift values are reported as δ (ppm) relative to DMSO at δ 2.50 ppm. The purity of synthesized compounds was analyzed by HPLC (Beckman Gold Nouveau System Gold) on a C₁₈ column (Grace Alltima 3 μm C₁₈ analytical Rocket® column, 53 mm × 7 mm) using triethylammonium acetate buffer (50 mM, pH 6) and methanol as eluent, flow rate 3 mL/min, and detection at 260 nm.

(R)-5-(1,2-dithiolan-3-yl)pentanoic ((2R,3S,4R,5R)-5-(6-amino-9H-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl phosphoric) anhydride monosodium salt (Lipoyl-AMP, LA-AMP). *R*-Lipoic acid (0.500 g, 2.42 mmol) was dissolved in acetonitrile (2.5 mL) and added to a solution of dicyclohexylcarbodiimide (0.250 g, 1.21 mmol) in acetonitrile (0.9 mL) and the mixture was stirred at ambient temperature for 30 min. The resulting suspension was then transferred to a conical tube and centrifuged (5 min, 4000 rpm) to pellet the solid. The supernatant containing the *R*-lipoic acid anhydride was added to a solution of adenosine monophosphate monohydrate (0.442 g, 1.21 mmol) in pyridine (3.68 mL) and water (7.81 mL) at 0°C. After stirring the mixture at 0°C for 50 min, water (10 mL) was added and the solution was transferred to a separatory funnel where it was washed with ice-cold diethyl ether (2 × 10 mL) and then ice-cold chloroform (2 × 10 mL). The aqueous layer was collected and acetone was added to precipitate residual adenosine monophosphate, which was removed by centrifugation (5 min, 4000 rpm). The supernatant was then condensed under reduced pressure and redissolved in a minimum volume of water before being loaded onto a C₁₈ solid phase extraction column, washed with water (10 mL), and eluted with acetonitrile. Condensation *in vacuo* yielded the desired product in 96% purity (0.006 g, 0.01 mmol, 1% yield). ¹H NMR (500 MHz, DMSO-d₆) δ 8.57 (s, 1H), 8.31 (s, 1H), 5.94 (d, *J* = 5.62 Hz, 1H), 4.57 (t, *J* = 5.38 Hz, 1H), 4.18 (dd, *J* = 3.42, 4.65 Hz, 1H), 4.04–4.15 (m, 10H), 3.53 – 3.64 (m, 1H), 3.13–3.22 (m, 1H), 3.05–3.13 (m, 1H), 2.35–2.44 (m, 1H), 2.36 (t, *J* = 7.21 Hz, 2H), 2.20 (t, *J* = 7.21 Hz, 1H), 1.80–1.91 (m, 1H), 1.58–1.69 (m, 1H), 1.44–1.58 (m, 4H), 1.28–1.41 (m, 2H).

6,8-Dichlorooctanoic ((2R,3S,4R,5R)-5-(6-amino-9H-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl phosphoric) anhydride monosodium salt (6,8-dichlorooctanoyl-AMP, diCl-AMP). 6,8-Dichlorooctanoic acid (0.300 g, 1.41 mmol) was dissolved in dichloromethane (2.9 mL) and dicyclohexylcarbodiimide (0.145 g, 0.704 mmol) was added. After stirring for 30 min, the solid was removed by filtration and washed with dichloromethane. The filtrate was collected and condensed under reduced pressure. The resulting oil was dissolved in pyridine (1.7 mL), cooled to 0°C and then added to a solution of adenosine monophosphate monohydrate (0.257 g, 0.704 mmol) in pyridine (1.7 mL) and water (2.2 mL) at 0°C. The solution was stirred at 0°C for 60 min. Water (10 mL) was added, and the solution was transferred to a separatory funnel where it was washed with ice-cold diethyl ether (2 × 10 mL) and then ice-cold chloroform (2 × 10 mL). The aqueous layer was collected and condensed under reduced pressure. The resulting residue was dissolved in a minimum volume of saturated aqueous NaHCO₃ and loaded on a C₁₈ solid phase extraction column and washed with water (10 mL) and then eluted with acetonitrile. Condensation *in vacuo* yielded the desired product in 97% purity (0.005 g, 0.01 mmol, 1% yield). ¹H NMR (500 MHz, DMSO-d₆) δ 8.44 (s, 1H), 8.14 (s, 1H), 7.24 (s, 2H), 5.91 (d, *J* = 6.13 Hz, 1H), 5.44 (d, *J* = 4.87 Hz, 1H), 5.29 (d, *J* = 3.77 Hz, 1H), 4.59 (q, *J* = 4.70 Hz, 1H), 4.10–4.18 (m, 2H), 4.01 (q, *J* = 3.30 Hz, 1H), 3.86–3.95 (m, 2H),

© 2017 John Wiley & Sons Ltd, *Molecular Microbiology*, **106**, 439–451
3.67–3.81 (m, 2H), 2.31 (t, $ J = 7.07 \, \text{Hz} $, 2H), 2.12–2.20 (m, 1H), 2.00–2.09 (m, 1H), 1.72–1.81 (m, 1H), 1.61–1.70 (m, 1H), 1.42–1.54 (m, 3H), 1.31–1.42 (m, 1H).

### Protein expression and purification

Cloning, expression and purification of the lipoylation domains of $ PfBCDH $ (PF3D7_0303700) and $ PfKDH $ (PF3D7_1320800), the H-protein (PF3D7_1132900), as well as $ LipL1 $ (PF3D7_1314600) and $ LipL2 $ (PF3D7_0923600) were previously described (Afanador et al., 2014). The genes encoding the lipoylation enzymes were cloned into the expression plasmid pMALcHT, which encodes a maltose binding protein (MBP) followed by a linker region composed of a tobacco etch virus (TEV) protease cut site and a six histidine affinity tag (Muench et al., 2003). The plasmid pMALcHT-$ PfLipL1 $(Afanador et al., 2014) was mutated by site-directed mutagenesis to generate pMALcHT-$ PfLipL1_{K160A} $ using the primers listed in Supporting Information Table S2. This plasmid was transformed into BL21-Star (DE3) cells (Invitrogen) and co-transformed with the pRIL plasmid isolated from BL21-CodonPlus-RIL cells (Agilent) and the plasmid pKM586 encoding the TEV protease, as described (Muench et al., 2003). The resulting transformed cells produced $ PfLipL1_{K160A} $ fused to an amino-terminal six histidine-tag. For protein expression, cells were grown to mid-log phase and protein expression was induced by the addition of 0.4 mM IPTG. Cells were harvested after growth for 10 h at $ 20^\circ \text{C} $. Recombinant $ PfLipL1_{K160A} $ was purified by metal chelate chromatography followed by cation exchange chromatography.

Plasmid pMALcHT-$ PfLipL2 $, encoding a codon harmonized variant (Afanador et al., 2014), was mutated by site-directed mutagenesis using primers listed in Supporting Information Table S2 to generate pMALcHT-$ LipL2_{K219A} $. Plasmid pMALcHT-$ PfLipL2_{K219A} $ was transformed into BL21-Star (DE3) and co-transformed with the pRIL plasmid. These cells produce $ LipL2 $ fused to an amino-terminal MBP. $ LipL2_{K219A} $ was purified using Amylose resin (New England Biolabs #E8021).

$ PbLipL2 $ (PBANKA_0824500) was cloned from *Plasmodium berghei* ANKA genomic DNA (a generous gift from the Sinnis Lab) into pMALcHT using primers $ PbLipL2.fwd $ and $ PbLipL2.rev $ listed in Supporting Information Table S2. Similar to $ LipL2 $ constructs, pMALcHT-$ PbLipL2 $ was transformed into BL21-Star (DE3) and co-transformed with the pRIL plasmid. These cells produced $ PbLipL2 $ fused to an amino-terminal MBP, which was purified using Amylose resin.

$ CtLipL2 $ (AAC68100) was cloned from $ C. trachomatis $ Serovar D strain UW-3/Cx gDNA (purchased from ATCC) into plasmid pMALcHT using primers $ CtLipL2.fwd $ and $ CtLipL2.rev $ listed in Supporting Information Table S2. pMALcHT-$ CtLipL2 $ was transformed in BL21-Star (DE3), co-transformed with the pRIL plasmid, and expressed for 10 hours at $ 20^\circ \text{C} $. The MBP-tagged $ CtLipL2 $ was purified by affinity chromatography (MBPTrap, GE Healthcare Life Sciences).

$ CtHprotein $ (AAC67875), $ CtBCDH $ (AAC67997) and $ CtPDH $ (AAC67840) were cloned from $ C. trachomatis $

Serovar D strain UW-3/Cx genomic DNA (purchased from ATCC) into plasmid pGEXTK using primers $ CtHprotein.fwd $ and $ CtHprotein.rev $, $ CtBCDH.fwd $ and $ CtBCDH.rev $, and $ CtPDH.fwd $ and $ CtPDH.rev $ listed in Supporting Information Table S2. Expression plasmid pGEXTK is a modified version of pGEXT (Delli-Bovi et al., 2010) containing a kanamycin selection cassette. Codon-optimized $ CtKDH $ (synthesized by GeneArt) was digested with $ BamHI $ and $ SalI $ and inserted into pGEXTK to yield pGEXTK-$ CtKDH $. All plasmids were transformed into BL21-Star (DE3) along with the pRIL plasmid. $ CtHprotein $ and $ CtBCDH $ were expressed for 3 h at $ 37^\circ \text{C} $ while $ CtPDH $ and $ CtKDH $ were expressed for 10 h at $ 20^\circ \text{C} $. The resulting GST-fusion proteins were purified by affinity chromatography (GSTrap, GE Healthcare Life Sciences).

#### Lipoylation assays

Purified recombinant enzymes (1 μM) were incubated in reaction buffer (100 mM Na$^+$/K$^+$ phosphate buffer, 150 mM NaCl at pH 7.5) containing (unless otherwise stated) 2 mM ATP, 2 mM MgCl$_2$, 5 mM TCEP, 200 μM R-lipoic acid and 10 μM apo-protein substrate. The reactions were incubated at $ 37^\circ \text{C} $ for 1 h and quenched with the addition of gel loading buffer. Proteins were then resolved by SDS-PAGE and transferred to nitrocellulose membrane for Western blotting. Membranes were blocked with 5% milk in PBS for 30 minutes, and probed with 1:5000 rabbit polyclonal $ \alpha $-LA$_{891} $ (see below) for 1 hour in 1% milk/PBS at room temperature. The membrane was then washed with PBS three times and probed with 1:5000 donkey $ \alpha $-Rabbit IgG horseradish peroxidase (HRP) secondary antibody (GE Healthcare) in 1% milk/PBS overnight at $ 4^\circ \text{C} $. Proteins were visualized using enhanced chemiluminescence (ECL) western substrate (Pierce) by exposure to film.

All lipoylation assays were conducted similarly using the same protein concentrations and base buffer. Where indicated, ATP was omitted from the reaction and 5 mM of DTT, 5 mM THP or water was used instead of TCEP. When noted, R-lipoic acid was substituted with 1 or 10 μM R-lipoyl-AMP as the sole lipoate source, with no ATP added.

#### Cell-based lipoylation assay

$ E. coli $ BL21 cells were transformed with the pGEXTK-$ CtHprotein $, pGEXTK-$ CtBCDH $, pGEXTK-$ CtKDH $ and pGEXTK-$ CtPDH $ plasmids and grown in LB media in the presence of kanamycin. For the cell-based assay, 20 mL cultures were grown to mid-log phase at $ 37^\circ \text{C} $ and protein expression was induced with 0.4 mM IPTG for 10 hours at $ 20^\circ \text{C} $. Cells were harvested by centrifugation and resuspended with 0.5 mL of buffer containing 20 mM HEPES, 100 mM NaCl at pH 7.5 and lysed by sonication. Cell lysates were clarified by centrifugation at 16000 g. The supernatants were collected, and the proteins were resolved by SDS-PAGE. Lipoylated proteins were visualized by Western blot as described for the lipoylation assays.

Thermal shift assay

Thermal shift assays were performed using a previously described method with minor modifications as follows (Afanador *et al.*, 2013). Real-time PCR tube strips (Eppendorf) were used to hold 31 μL mixtures containing final concentrations of 1 μM LipL1, 10 μM of ligand (*R*-LA, *R*-Lipoyl-AMP or 6,8-diCIO) diluted in DMSO. The reactions were set up with a 29 μL mixture containing LipL1, buffer (100 mM Na/K Phosphate pH 7.5, 150 mM NaCl and 2 mM MgCl₂) and Sypro Orange (Sigma, Product Number S-5692 at a final concentration of 5X) to which 1 μL of DMSO (or ligand) and 1 μL of water (or DTT or TCEP diluted in water at a final concentration of 5 mM) were added. The reaction mixture was incubated in a RT-PCR machine (Applied Biosystems, Step One Plus Real-Time PCR System) for 2 min at 20°C followed by 0.2°C increases in the temperature every 10 s until a final temperature of 80°C was reached. During the thermal scan, fluorescence was monitored using a predefined TAMRA filter in which an increase in Sypro Orange fluorescence was observed upon thermal denaturation of LipL1. The derivative of the fluorescence curve was used to determine the melting temperature (*T*ₘ). The initial *T*ₘ in the absence of ligand, but in the presence of DMSO, served as the baseline temperature (*T*ₒ) for determining temperature shifts (Δ*T*ₘ). All measurements were made with three technical replicates on three separate occasions.

All binding affinities were calculated as described previously in the literature (Lo *et al.*, 2004; Afanador *et al.*, 2013), employing the Eq. (1):

\[
K_{\mathrm{d}(T_{\mathrm{m}})} = \frac{\left[\mathrm{L}_{T_{\mathrm{m}}}\right]}{\exp \left\{\frac{-\Delta H}{R} \left(\frac{1}{T_{\mathrm{m}}} - \frac{1}{T_{\mathrm{o}}}\right) + \frac{\Delta C_{\mathrm{p}}}{R} \left[\ln \left(\frac{T_{\mathrm{m}}}{T_{\mathrm{o}}}\right) + \frac{T_{\mathrm{o}}}{T_{\mathrm{m}}} - 1\right]\right\}}
\]

Here *T*ₒ represents the melting temperature of *Pf*LipL1 in the absence of ligand, *T*ₘ represents the melting temperature of *Pf*LipL1 in the presence of ligand, R is the universal gas constant, Δ*H* represents the enthalpy of protein unfolding, Δ*C*ₚ represents the change in the heat capacity upon protein unfolding, and [L<sub>*T*ₘ</sub>] represents the concentration of the free ligand at the melting temperature. Both Δ*H* and Δ*C*ₚ were determined by Differential Scanning Calorimetry (DSC). A sample of 0.5 mg/mL *Pf*LipL1 was scanned from 10 to 65°C at 1°C/min using a VP-DSC microcalorimeter from MicroCal/Malvern Instruments LLC (Northampton, MA). The change in heat capacity, Δ*C*ₚ was 5 kcal/mol·K as estimated from the difference between the baselines for the denatured and native states. The unfolding enthalpy (Δ*H*) obtained by integration of the peak, was 124.5 kcal/mol. The dissociation constant was calculated at 37°C using the Eq. (2):

\[
K_{\mathrm{d}(T)} = \frac{K_{\mathrm{d}(T_{\mathrm{m}})}}{\exp \left(\frac{-\Delta H_{\mathrm{L}(T)}}{R} \left(\frac{1}{T} - \frac{1}{T_{\mathrm{m}}}\right)\right)}
\]

Here, Δ*H*<sub>L(*T*)</sub> is the van’t Hoff enthalpy of binding at temperature *T* and estimated to be −15 kcal/mol.


Production and validation of rabbit α-LA<sub>891</sub> antibodies

*R*-lipoic acid was conjugated to keyhole limpet hemocyanin using the Imject EDC mcKLH Spin Kit (Pierce). Antibodies to the conjugate were generated in rabbits using the standard protocol of the custom antibody service Cocalico Biologicals (Reamstown, PA). Specific antibodies were purified from serum using a holo-H-protein affinity column. To make the affinity column, a 1 mL NHS-activated HP column (GE Healthcare) was activated with 1 mM HCl according to the manufacturer’s instructions. Immediately after activation, 5 mg of *E. coli* H-protein in 4 mL of reaction buffer (100 mM NaHCO₃, 500 mM NaCl pH 8.3) was pumped through the column at a constant rate of 0.1 mL/min. H-protein was circulated through the column for 1 h at room temperature at which point the release of N-hydroxysuccinimide reaction product was quantified by absorbance at 260 nm (*ε* = 8600 M⁻¹ cm⁻¹). The column was subsequently washed and blocked according to the manufacturer’s directions. Rabbit antiserum (10 mL) was then circulated over the affinity column at room temperature for 4 h at 0.25 mL/min. Nonspecifically bound proteins were washed from the column with 10 mL of PBS. The bound antibodies were then eluted from the column in 4 mL of 50 mM glycine pH 1.9. A total of 1.44 mg of specific IgG was concentrated to 1 mg/mL and stored at −80°C in storage buffer (PBS, 50% glycerol, 0.02% NaN₃). The specific polyclonal antibodies were designated α-LA<sub>891</sub>. To test this new reagent, we generated lipoylated *Pf*H-protein using LipL1, *R*-LA and ATP in a lipoate ligase reaction, as described above. The resulting reaction mixture was then used to compare α-LA<sub>891</sub> to commercially available α-LA (Calbiochem) by Western blot (Supporting Information Fig. S3).


Sequence identity and alignments

The EuPathDB (Aurrecoechea *et al.*, 2010), including PlasmoDB, ToxoDB and PiroplasmaDB, and the Wellcome Trust Sanger Institute for *Eimeria tenella* were used for BLAST homology search of LipL2-like enzymes (Ling *et al.*, 2007). Amino acid sequence identity was calculated by using the MUSCLE algorithm (Edgar, 2004a, 2004b), and sequence alignments were performed with the programs CLUSTALW2 and ESPRIPT2 (Gouet *et al.*, 1999; Thompson *et al.*, 2002). The superfamily phylogenetic tree was obtained from Pfam entry PF03099 and displayed using Dendroscope by distance methods using bootstrap sampling with 100 replicates (Huson *et al.*, 2007).


Acknowledgements

We would like to thank Dr. Melanie J. Shears who provided insight and expertise in the preparation of this manuscript and the laboratory of Photini Sinnis for providing *Plasmodium berghei* gDNA. This work was supported by the National Institutes of Health (R56 AI065853 and R01 AI125534 to STP, F32 AI110028 to AJG, R01 GM084998 to CLFM, AI099704 to DB, and P01 GM056550 to EF), the National Science Foundation (MCB-1157506 to EF), and the Johns Hopkins Malaria Research Institute and the Bloomberg Family Foundation. The authors declare that they have no conflict of interest.


© 2017 John Wiley & Sons Ltd, *Molecular Microbiology*, **106**, 439–451

References

Afanador, G.A., Matthews, K.A., Bartee, D., Gisselberg, J.E., Walters, M.S., Freel Meyers, C.L., and Prigge, S.T. (2014) Redox-dependent lipoylation of mitochondrial proteins in *Plasmodium falciparum*. Mol Microbiol 94: 156–171.

Afanador, G.A., Muench, S.P., McPhillie, M., Fomovska, A., Schön, A., Zhou, Y., et al. (2013) Discrimination of potent inhibitors of Toxoplasma gondii enoyl-acyl carrier protein reductase by a thermal shift assay. Biochemistry 52: 9155–9166.

Allary, M., Lu, J.Z., Zhu, L., and Prigge, S.T. (2007) Scavenging of the cofactor lipoate is essential for the survival of the malaria parasite *Plasmodium falciparum*. Mol Microbiol 63: 1331–1344.

Aurrecoechea, C., Brestelli, J., Brunk, B.P., Fischer, S., Gajria, B., Gao, X., et al. (2010) EuPathDB: a portal to eukaryotic pathogen databases. Nucleic Acids Res 38: D415–D419.

Bryk, R., Lima, C.D., Erdjument-Bromage, H., Tempst, P., and Nathan, C. (2002) Metabolic enzymes of mycobacteria linked to antioxidant defense by a thioredoxin-like protein. Science 295: 1073–1077.

Bunik, V.I. (2003) 2-Oxo acid dehydrogenase complexes in redox regulation. Eur J Biochem 270: 1036–1042.

Burns, J.A., Butler, J.C., Moran, J., and Whitesides, G.M. (1991) Selective reduction of disulfides by tris(2-carboxyethyl)phosphine. J Org Chem 56: 2648–2650.

Christensen, Q.H., and Cronan, J.E. (2010) Lipoic acid synthesis: a new family of octanoyltransferases generally annotated as lipoate protein ligases. Biochemistry 49: 10024–10036.

Christensen, Q.H., Hagar, J.A., O’Riordan, M.X.D., and Cronan, J.E. (2011) A complex lipoate utilization pathway in *Listeria monocytogenes*. J Biol Chem 286: 31447–31456.

Cobbold, S.A., Vaughan, A.M., Lewis, I.A., Painter, H.J., Camargo, N., Perlman, D.H., et al. (2013) Kinetic flux profiling elucidates two independent acetyl-CoA biosynthetic pathways in *Plasmodium falciparum*. J Biol Chem 288: 36338–36350.

Delli-Bovi, T.A., Spalding, M.D., and Prigge, S.T. (2010) Overexpression of biotin synthase and biotin ligase is required for efficient generation of sulfur-35 labeled biotin in *E. coli*. BMC Biotechnol 10: 73.

Delves, M., Plouffe, D., Scheurer, C., Meister, S., Wittlin, S., Winzeler, E.A., et al. (2012) The activities of current antimalarial drugs on the life cycle stages of *Plasmodium*: a comparative study with human and rodent parasites. PLoS Med 9: e1001169.

Deschermeier, C., Hecht, L.-S., Bach, F., Rützel, K., Stanway, R.R., Nagel, A., et al. (2012) Mitochondrial lipoic acid scavenging is essential for *Plasmodium berghei* liver stage development. Cell Microbiol 14: 416–430.

Dondorp, A.M., Fairhurst, R.M., Slutsker, L., Macarthur, J.R., Breman, J.G., Guerin, P.J., et al. (2011) The threat of artemisinin-resistant malaria. N Engl J Med 365: 1073–1075.

Edgar, R.C. (2004a) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res 32: 1792–1797.

Edgar, R.C. (2004b) MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics 5: 113.

Falkard, B., Kumar, T.R.S., Hecht, L.-S., Matthews, K.A., Henrich, P.P., Gulati, S., et al. (2013) A key role for lipoic acid synthesis during *Plasmodium* liver stage development. Cell Microbiol 15: 1585–1604.

Fidock, D.A., Eastman, R.T., Ward, S.A., and Meshnick, S.R. (2008) Recent highlights in antimalarial drug resistance and chemotherapy research. Trends Parasitol 24: 537–544.

Finn, R.D., Mistry, J., Tate, J., Coggill, P., Heger, A., Pollington, J.E., et al. (2010) The Pfam protein families database. Nucleic Acids Res 38: D211–D222.

Foth, B.J., Ralph, S.A., Tonkin, C.J., Struck, N.S., Fraunholz, M., Roos, D.S., et al. (2003) Dissecting apicoplast targeting in the malaria parasite *Plasmodium falciparum*. Science 299: 705–708.

Fujiwara, K., Hosaka, H., Matsuda, M., Okamura-Ikeda, K., Motokawa, Y., Suzuki, M., et al. (2007) Crystal structure of bovine lipoyltransferase in complex with lipoyl-AMP. J Mol Biol 371: 222–234.

Fujiwara, K., Maita, N., Hosaka, H., Okamura-Ikeda, K., Nakagawa, A., and Taniguchi, H. (2010) Global conformational change associated with the two-step reaction catalyzed by *Escherichia coli* lipoate-protein ligase A. J Biol Chem 285: 9971–9980.

Fujiwara, K., Okamura-Ikeda, K., and Motokawa, Y. (1994) Purification and characterization of lipoyl-AMP:N epsilon-lysine lipoyltransferase from bovine liver mitochondria. J Biol Chem 269: 16605–16609.

Fujiwara, K., Takeuchi, S., Okamura-Ikeda, K., and Motokawa, Y. (2001) Purification, characterization, and cDNA cloning of lipoate-activating enzyme from bovine liver. J Biol Chem 276: 28819–28823.

Fujiwara, K., Toma, S., Okamura-Ikeda, K., Motokawa, Y., Nakagawa, A., and Taniguchi, H. (2005) Crystal structure of lipoate-protein ligase A from *Escherichia coli*. Determination of the lipoic acid-binding site. J Biol Chem 280: 33645–33651.

Gouet, P., Courcelle, E., Stuart, D.I., and Métoz, F. (1999) ESPript: analysis of multiple sequence alignments in PostScript. Bioinformatics 15: 305–308.

Günther, S., McMillan, P.J., Wallace, L.J.M., and Müller, S. (2005) *Plasmodium falciparum* possesses organelle-specific alpha-keto acid dehydrogenase complexes and lipoylation pathways. Biochem Soc Trans 33: 977–980.

Huang, J., and Gogarten, J.P. (2007) Did an ancient chlamydial endosymbiosis facilitate the establishment of primary plastids? Genome Biol 8: R99.

Huson, D.H., Richter, D.C., Rausch, C., Dezulian, T., Franz, M., and Rupp, R. (2007) Dendroscope: an interactive viewer for large phylogenetic trees. BMC Bioinformatics 8: 460.

Ke, H., Lewis, I.A., Morrisey, J.M., McLean, K.J., Ganesan, S.M., Painter, H.J., et al. (2015) Genetic investigation of tricarboxylic acid metabolism during the *Plasmodium falciparum* life cycle. Cell Rep 11: 164–174.

Ling, K.-H., Rajandream, M.-A., Rivailler, P., Ivens, A., Yap, S.-J., Madeira, A.M.B.N., et al. (2007) Sequencing and analysis of chromosome 1 of *Eimeria tenella* reveals a unique segmental organization. Genome Res 17: 311–319.
Lo, M.-C., Aulabaugh, A., Jin, G., Cowling, R., Bard, J., Malamas, M., and Ellestad, G. (2004) Evaluation of fluorescence-based thermal shift assays for hit identification in drug discovery. *Anal Biochem* 332: 153–159.

Lukesh, J.C., III, Palte, M.J., and Raines, R.T. (2012) A potent, versatile disulfide-reducing agent from aspartic acid. *J Am Chem Soc* 134: 4057–4059.

McMillan, P.J., Stimmler, L.M., Foth, B.J., McFadden, G.I., and Müller, S. (2004) The human malaria parasite *Plasmodium falciparum* possesses two distinct dihydrolipoamide dehydrogenases. *Mol Microbiol* 55: 27–38.

Mohring, F., Rahbari, M., Zechmann, B., Rahlfs, S., Przyborski, J.M., Meyer, A.J., and Becker, K. (2017) Determination of glutathione redox potential and pH value in subcellular compartments of malaria parasites. *Free Radic Biol Med* 104: 104–117.

Morris, T.W., Reed, K.E., and Cronan, J.E. (1995) Lipoic acid metabolism in *Escherichia coli*: the *lplA* and *lipB* genes define redundant pathways for ligation of lipoyl groups to apoprotein. *J Bacteriol* 177: 1–10.

Mudhune, S.A., Okiro, E.A., Noor, A.M., Zurovac, D., Juma, E., Ochola, S.A., and Snow, R.W. (2011) The clinical burden of malaria in Nairobi: a historical review and contemporary audit. *Malar J* 10: 138.

Muench, S.P., Rafferty, J.B., McLeod, R., Rice, D.W., and Prigge, S.T. (2003) Expression, purification and crystallization of the *Plasmodium falciparum* enoyl reductase. *Acta Crystallogr Sect D Biol Crystallogr* 59: 1246–1248.

Müller, S. (2004) Redox and antioxidant systems of the malaria parasite *Plasmodium falciparum*. *Mol Microbiol* 53: 1291–1305.

Nickel, C., Rahlfs, S., Deponte, M., Koncarevic, S., and Becker, K. (2006) Thioredoxin networks in the malarial parasite *Plasmodium falciparum*. *Antioxid Redox Signal* 8: 1227–1239.

Nývltová, E., Smutná, T., Tachezy, J., and Hrdý, I. (2016) OsmC and incomplete glycine decarboxylase complex mediate reductive detoxification of peroxides in hydrogenosomes of *Trichomonas vaginalis*. *Mol Biochem Parasitol* 206: 29–38.

Okiro, E.A., Al-Taiar, A., Reyburn, H., Idro, R., Berkley, J.A., and Snow, R.W. (2009) Age patterns of severe paediatric malaria and their relationship to *Plasmodium falciparum* transmission intensity. *Malar J* 8: 4.

Okiro, E.A., Bitira, D., Mbabazi, G., Mpimbaza, A., Alegana, V.A., Talisuna, A.O., and Snow, R.W. (2011) Increasing malaria hospital admissions in Uganda between 1999 and 2009. *BMC Med* 9: 37.

Oppenheim, R.D., Creek, D.J., MacRae, J.I., Modrzynska, K.K., Pino, P., Limenitakis, J., et al. (2014) BCKDH: the

missing link in apicomplexan mitochondrial metabolism is required for full virulence of *Toxoplasma gondii* and *Plasmodium berghei*. *PLoS Pathog* 10: e1004263.

Phyo, A.P., Nkhoma, S., Stepniewska, K., Ashley, E.A., Nair, S., McGready, R., et al. (2012) Emergence of artemisinin-resistant malaria on the western border of Thailand: a longitudinal study. *Lancet* 379: 1960–1966.

Ramaswamy, A.V., and Maurelli, A.T. (2010) Chlamydia trachomatis serovar L2 can utilize exogenous lipoic acid through the action of the lipoic acid ligase LplA1. *J Bacteriol* 192: 6172–6181.

Reed, L.J., Koike, M., Levitch, M.E., and Leach, F.R. (1958) Studies on the nature and reactions of protein-bound lipoic acid. *J Biol Chem* 232: 143–158.

Salcedo, E., Sims, P.F.G., and Hyde, J.E. (2005) A glycine cleavage complex as part of the folate one-carbon metabolism of *Plasmodium falciparum*. *Trends Parasitol* 21: 406–411.

Spalding, M.D., Allary, M., Gallagher, J.R., and Prigge, S.T. (2010) Validation of a modified method for Bxb1 mycobacteriophage integrase-mediated recombination in *Plasmodium falciparum* by localization of the H-protein of the glycine cleavage complex to the mitochondrion. *Mol Biochem Parasitol* 172: 156–160.

Spalding, M.D., and Prigge, S.T. (2010) Lipoic acid metabolism in microbial pathogens. *Microbiol Mol Biol Rev* 74: 200–228.

Srivastava, A., Philip, N., Hughes, K.R., Georgiou, K., MacRae, J.I., Barrett, M.P., et al. (2016) Stage-specific changes in plasmodium metabolism required for differentiation and adaptation to different host and vector environments. *PLoS Pathog* 12: e1006094.

Storm, J. (2012) Lipoic acid metabolism of plasmodium - a suitable drug target. *Curr Pharm Des* 18: 3480–3489.

Thompson, J.D., Gibson, T.J., and Higgins, D.G. (2002) Multiple sequence alignment using ClustalW and ClustalX. *Curr Protoc Bioinformatics* Chapter 2: Unit 2.3–2.3.22.

Wrenger, C., and Müller, S. (2004) The human malaria parasite *Plasmodium falciparum* has distinct organelle-specific lipoylation pathways. *Mol Microbiol* 53: 103–113.

Zhang, Y. (2008) I-TASSER server for protein 3D structure prediction. *BMC Bioinformatics* 9: 40.

**Supporting information**

Additional supporting information may be found in the online version of this article at the publisher’s web-site.