
Review
# Immune Evasion Strategies during Chronic Hepatitis B and C Virus Infection

Ana Maria Ortega-Prieto and Marcus Dorner*

* Section of Virology, Department of Medicine, Imperial College London, London W2 1PG, UK;
a.ortega-prieto@imperial.ac.uk
* Correspondence: m.dorner@imperial.ac.uk; Tel.: +44-207-594-5037

Academic Editors: Angela L. Rasmussen and Juliet Morrison  
Received: 28 July 2017; Accepted: 30 August 2017; Published: 1 September 2017

Abstract: Both hepatitis B virus (HBV) and hepatitis C virus (HCV) infections are a major global healthcare problem with more than 240 million and 70 million infected, respectively. Both viruses persist within the liver and result in progressive liver disease, resulting in liver fibrosis, cirrhosis and hepatocellular carcinoma. Strikingly, this pathogenesis is largely driven by immune responses, unable to clear an established infection, rather than by the viral pathogens themselves. Even though disease progression is very similar in both infections, HBV and HCV have evolved distinct mechanisms, by which they ensure persistence within the host. Whereas HCV utilizes a cloak-and-dagger approach, disguising itself as a lipid-like particle and immediately crippling essential pattern-recognition pathways, HBV has long been considered a “stealth” virus, due to the complete absence of innate immune responses during infection. Recent developments and access to improved model systems, however, revealed that even though it is among the smallest human-tropic viruses, HBV may, in addition to evading host responses, employ subtle immune evasion mechanisms directed at ensuring viral persistence in the absence of host responses. In this review, we compare the different strategies of both viruses to ensure viral persistence by actively interfering with viral recognition and innate immune responses.

Keywords: Hepatitis B virus; Hepatitis C virus; innate immunity; adaptive immunity; immune evasion

---

## 1. Introduction

Chronic viral hepatitis (CVH) infections are a major global public health concern with more than 240 million and 70 million people infected with hepatitis B virus (HBV) and hepatitis C virus (HCV), respectively [1,2]. Both viruses persist within infected hepatocytes in the liver, where they eventually result in the development of liver fibrosis, cirrhosis and hepatocellular carcinoma [3]. Despite the similarity in disease progression and target cells, HCV and HBV have developed complex but nearly opposing strategies of ensuring persistence within infected individuals. While HCV infection usually results in vigorous innate immune activation during infection, the clinical symptoms of acute infection are often subclinical and may go unnoticed [4,5]. In contrast, acute HBV infection is usually easily detectable through the accompanying symptoms [6]. However, HBV itself seems to engage far fewer innate and inflammatory pathways, compared to HCV infection.

Both, acute and chronic infection by HCV and HBV may eventually result in the development of a life-long, chronic infection in 80% and 5% of adults, respectively. However, HBV infection during childhood may result in chronic infection in as many as 90% of infected, suggesting a strong contribution of immune maturation on the development of persistent infection [7]. In the case of spontaneous clearance of HCV or HBV infection, a potent adaptive immune response is clearing infected hepatocytes, resulting in temporary liver injury, as determined by elevated levels of alanine aminotransferases (ALT) [8]. Viral antigen subsequently is completely eliminated and, once cleared

results in life-long, sterilizing immunity. In the case of HCV infection, spontaneous clearance has been strongly linked with innate immune activation and interferon responses [9].

Both, HBV and HCV infect hepatocytes of the liver via receptor-mediated endocytosis, where they initiate replication. During the initial interaction with their host cell, both, HBV and HCV are exposed to a number of pattern recognition receptors (PRR), which form the first line of defense against infections.

HCV infection is restricted to the cytoplasm, where it reorganizes parts of the endoplasmic reticulum to form the membranous web and the replicase compartment [5]. Ranging from the initial translation of the HCV polyprotein and RNA replication to viral assembly of virions on lipid droplets, HCV RNA and proteins are constantly prone to triggering innate cellular immune responses, especially the uncapped 5' internal ribosomal entry site and its extensive secondary structure, resulting in the formation of double-stranded RNA intermediates are potent pathogen-associated molecular patterns (PAMP). To circumvent cellular sensing, HCV has evolved several intricate mechanisms, by which it prevents the host from initiating interferon responses.

In contrast, HBV capsids are directly shuttling into the nucleus of infected cells before disassembly and release of the relaxed-circular (rc) HBV DNA genome [8]. During subsequent complex, and thus far poorly understood mechanisms, host factors are recruited in order to remove the covalently-bound HBV reverse transcriptase from rcDNA and repair the incomplete (+)-strand, resulting in covalently closed, circular (ccc)DNA, which forms the basis of all de novo produced HBV. Unique to HBV, its cccDNA genome is recruiting host-derived histones, forming, what is commonly known as HBV minichromosome. All viral transcripts are processed by host polymerase II, capped and polyadenylated before translation. One of these transcripts, the overlength pgRNA, is encapsidated prior to reverse transcription, effectively hiding or shielding most of the HBV life-cycle from immune detection.

Whereas HCV replication during acute and chronic infection is uniformly recognized by the adaptive immune system, HBV infection is divided into distinct clinical stages, which are associated with varying degrees of immune activation and subsequent liver injury [10]. Early after infection, HBV replicates nearly uncontrolled within the infected liver (termed HBeAg-positive infection or “immune tolerant” phase), resulting in up to 80% of hepatocytes becoming infected and serum HBV DNA titers of more than 20,000 IU/mL [11]. Immune activation and liver injury are negligible during this stage of infection; however, more evidence is gathering that HBV actively subverts innate immunity during this phase of infection. Once the immune system recognizes HBV infection (termed HBeAg-positive hepatitis or “immune reactive” phase), which in many cases occurs only years after infection, a strong CD8 T cell response results in viral suppression accompanied by marked liver injury [11]. To date, it remains poorly defined, as to what triggers the transition from HBeAg-positive infection to hepatitis. This phase of immune clearance is followed by seroconversion from HBeAg positive to anti-HBe antibodies. This is often accompanied by particular mutations within the HBV pre-core or basal core promoter, resulting in reduced or absent HBeAg secretion [12]. During this phase, referred to as HBeAg-negative infection or “inactive carrier” state, serum HBV DNA levels are below 2000 IU/mL and persistently normal ALT levels. Following the establishment of HBeAg-negative infection, HBV may periodically reactivate (termed HBeAg-negative hepatitis), resulting in sporadic flares of serum HBV DNA elevations accompanied by ALT level elevations [13].

Common to both, HBV and HCV infection, is a further subversion of adaptive immune responses during chronic infection [8]. Both pathogens result in a terminally exhausted T cell phenotype, with most HBV- or HCV-specific CD8 T cells being anergic and unable to mount an effective polyfunctional response [14]. In terms of humoral responses, HBV and HCV have evolved different means of subverting B cell responses. Despite the potent ability of HBsAg to elicit neutralizing antibody responses, HBV replication produces an enormous quantity of sub-viral particles, which compete with the infectious virions for antibody binding, thus incapacitating any antibody-based neutralization. HCV infection on the other hand generally does not result in the development of sterilizing antibody-based immunity, which is due to HCV virions mimicking very low-density

lipoproteins (VLDL) to evade antibody targeting and virus spreading through tight junctions between two hepatocytes, where they are immune to antibody neutralization.

Here, we give an overview of how HBV and HCV interact with the hosts' innate and adaptive immune system and the mechanisms evolved in order to ensure persistence.

## 2. Host Responses to HBV and HCV Infection

### 2.1. Cell-Intrinsic Innate Immune Responses

Cell-intrinsic innate immune responses are among the first line of defense against incoming infectious diseases. Pattern-recognition receptors (PRR) located on the cell membrane, intracellular vesicles or within the cytoplasm recognize conserved pathogen-associated molecular patterns (PAMP), including foreign proteins, lipids and nucleic acids, resulting in the initiation of signal transduction pathways, culminating in the production of pro-inflammatory cytokines and interferons (IFN) [15] (Figure 1). Three distinct classes of IFN can be produced by different cell types, type I (IFNα, IFNβ), type II (IFNγ) and type III (IFNλ1, IFNλ2, IFNλ3, IFNλ4) [16,17]. Once produced, they are secreted and bind to their cognate receptors (IFNAR1/2 for type I IFN, IFNGR for type II IFN and IL28RA/IL10R2 for type III IFN) in an autocrine or paracrine manner and trigger the JAK/STAT signal transduction pathway. For type I and III IFN, this, in turn, results in the up-regulation of hundreds of interferon-stimulated genes (ISG), which exert antiviral functions [18]. These ISG can be broadly classified into several categories. In the case of HCV, these include ISG with direct antiviral function, including inhibitors of HCV entry, translation, replication and release as well as ISG, which are part of the active sensing and transcriptional activation system or negative regulators of innate immune activation (Figure 2 and Table 1). There are numerous ISGs that have been identified playing a role during the HCV viral life cycle, more extended information can be found in the review published by Bartenschlager and colleagues [19]. Expression of these ISG is mainly aimed at restricting the spread of viral infections to allow for the induction of cellular innate and adaptive immune responses. In cases, where viral replication cannot be controlled by the antiviral effector function of ISG, they can additionally result in cell death of infected cells, in order to restrict viral spread.

![Figure 1](attachment://image.png)

Figure 1. Recognition of hepatitis B virus (HBV) and hepatitis C virus (HCV) by pattern-recognition receptors and immune evasion strategies. Innate immune signaling pathways involved in the recognition of HBV and HCV infection and the subsequent induction of interferon and interferon-stimulated genes. Several viral proteins of HCV (red) and HBV (green) interfere at various stages with these pathways.

Figure 2. The lifecycle of HCV and HBV and interferon effector molecules targeting different steps in the viral infection process. HCV (left) and HBV (right) life cycles including pattern recognition receptor targets (blue) and interferon-stimulated genes (yellow) affecting viral entry, replication and egress.

Table 1. Interferon-stimulated genes with activity against HBV and HCV.

| Interferon-Stimulated Gene | ISG Class                          | Effect on HCV                              | Effect on HBV                              | Reference       |
|---------------------------|-------------------------------------|--------------------------------------------|--------------------------------------------|-----------------|
| ADAP2                     | Direct antiviral                   | Inhibits entry                             | -                                          | [20]            |
| ADAR                      | Direct antiviral                   | RNA editing                                | -                                          | [21]            |
| APOBEC 1, 3A, 3B, 3G, 3F  | Direct antiviral                   | -                                          | cccDNA/HBV DNA editing                    | [22–27]         |
| AID                       | Direct antiviral                   | -                                          | cccDNA editing                             | [28–30]         |
| CH25H                     | Unknown                            | Inhibits replication                       | Inhibits entry                             | [31–34]         |
| DDIT4                     | Unknown                            | Inhibits infection                         | -                                          | [21]            |
| DDX 58 (RIG-I)            | RNA sensor                         | Enhanced viral sensing                     | Inhibits replication                       | [21, 35–37]     |
| DDX60                     | RIG-I enhancer                     | Inhibits infection                         | -                                          | [21, 38, 39]    |
| DDX60L                    | Unknown                            | Inhibits replication                       | -                                          | [40]            |
| EIF2AK2 (PKR)             | RNA sensor                         | Inhibition of translation                  | Inhibition of replication                  | [41–44]         |
| GBP1                      | Unknown                            | Inhibits infection                         | -                                          | [45]            |
| IFI44L                    | Unknown                            | Inhibits infection                         | -                                          | [21]            |
| IFI6                      | Unknown                            | Inhibits HCV entry                         | -                                          | [45, 46]        |
| IFI27                     | Unknown                            | Inhibits infection                         | -                                          | [21]            |
| IFIT1                     | Direct antiviral                   | Sequesters viral nucleic acids             | -                                          | [45, 47]        |
| IFIT3                     | MAVS/TBK1 enhancer                 | Inhibits infection                         | -                                          | [48]            |
| IFITM1                    | Direct antiviral                   | Inhibits entry and replication             | Inhibition of replication                  | [48–51]         |
| IFITM3                    | Direct antiviral                   | Inhibits translation                       | -                                          | [52]            |
| IRF1                      | ISG inducer                        | Inhibits infection                         | Inhibition of replication                  | [21, 44, 53]    |
| IRF2                      | Unknown                            | Inhibits infection                         | -                                          | [21]            |
| IRF7                      | IFN inducer                        | Inhibits infection                         | -                                          | [21]            |
| IRF9                      | IFN inducer                        | Inhibits infection                         | -                                          | [54]            |
| ISG12                     | Ubiquitin-dependent degradation     | Viral protein degradation                  | -                                          | [45]            |
| ISG20                     | Direct antiviral                   | Cleaves RNA genome                        | Inhibits encapsidation, Cleaves RNA genome | [42, 55, 56]    |
| ISG56                     | Unknown                            | Inhibits replication                       | -                                          | [49]            |
| MAP3K14                   | Unknown                            | Inhibits infection                         | -                                          | [21]            |
| MDA5                      | RNA sensor                         | Inhibits infection                         | -                                          | [21, 57]        |
| MOV10                     | Unknown                            | Inhibits infection                         | -                                          | [21]            |
| MS4A4A                    | Unknown                            | Inhibits infection                         | -                                          | [21]            |
| MX1                       | ISG inducer                        | Inhibits infection                         | Inhibits replication                       | [45, 58, 59]    |
| MYD88                     | IFN inducer                        | -                                          | RNA degradation                            | [60]            |
| NOS2                      | Unknown                            | -                                          | Inhibits replication                       | [48, 61]        |
| NT5C3                     | Unknown                            | Inhibits infection                         | -                                          | [21]            |
| OAS1                      | RNase I activator                   | Inhibits infection                         | -                                          | [45, 62]        |
| OASL                      | Unknown                            | Inhibits replication                       | -                                          | [21, 63]        |
| PLSCR1                    | Unknown                            | Inhibits infection                         | Inhibits replication                       | [48, 64]        |
| RNASEL                    | Direct antiviral                   | Cleaves RNA genome                        | Inhibition of replication                  | [44, 48]        |
| R2AD2                     | Direct antiviral                   | Inhibits replication                       | -                                          | [48]            |

| Interferon-Stimulated Gene | ISG Class                     | Effect on HCV                 | Effect on HBV                  | Reference |
|---------------------------|-------------------------------|------------------------------|-------------------------------|-----------|
| STING                     | IFN inducer                   | -                            | Inhibits HBV assembly         | [65]      |
| SOCS3                     | IFN negative regulator        | Inhibits replication         | -                             | [66, 67]  |
| STAT1/3                   | IFN inducer                   | Inhibits replication         | Inhibits transcription         | [68–70]   |
| BST2 (Tetherin)           | Direct antiviral              | Inhibits HCV secretion       | Inhibits HBV secretion         | [71–73]   |
| TLR3                      | RNA sensor                    | Inhibits infection            | Inhibits replication           | [74–76]   |
| TLR7                      | RNA sensor                    | Inhibits infection            | Inhibits replication           | [77–79]   |
| TLR8                      | RNA sensor                    | Inhibits infection            | Inhibits replication           | [78, 79]  |
| TRIM14                    | Unknown                       | Degradation of NS5a          | -                             | [48, 80]  |
| TRIM12a                   | Unknown                       | Inhibits infection            | -                             | [21]      |
| TRIM22                     | Unknown                       | Ubiquitination of NS5a       | Inhibits transcription         | [81, 82]  |
| TRIM41                    | E3 ubiquitin ligase activator | -                            | Inhibits transcription         | [83]      |
| Viperin                   | Direct antiviral              | Inhibits replication         | -                             | [84]      |
| ZAP                       | Unknown                       | -                            | Inhibits replication           | [85, 86]  |

In contrast to HCV infection, HBV infection does not elicit notable innate immune activation through pro-inflammatory cytokines and IFN. Remarkably, early studies in chimpanzees and humans have demonstrated that HBV infection does not induce any gene expression signature characteristic for IFN or ISG, resulting in HBV to be known as “stealth” virus [87–89]. This is further supported by a recent study based on the use of in vitro models and liver chimeric mice, which has shown an absence of IFN responses upon stable HBV replication. Additionally, it was observed that cells previously infected with HBV and stimulated with inducers of IFN signaling successfully mounting a downstream immune response in the presence of active HBV infection. This supports that HBV does not actively evade immune recognition [90]. It, however, remains poorly understood as to why host innate immune responses fail to recognize HBV infection or what mechanism ultimately results in their activation later during infection. Nevertheless, several ISGs have been implicated in the potential control of HBV infection at various steps in its lifecycle. Recently, cholesterol-25-hydroxylase (CH25H) was identified as an ISG inhibiting HBV entry through its receptor, Na⁺-taurocholate cotransporting polypeptide (NTCP) [31]. Once HBV cccDNA is formed, members of the nuclear deaminase family, including apolipoprotein B mRNA editing enzyme, catalytic polypeptide-like (APOBEC) 3A, APOBEC3G, and AID have been demonstrated to deaminate cccDNA and target it for degradation [22, 23, 28]. cccDNA itself, due to its chromatinized structure, is additionally regulated through epigenetic mechanisms, most notably through STAT1 and STAT3, which directly bind to cccDNA and regulate its transcriptional activity [68]. Tripartite motif-containing (TRIM) 22 is an additional ISG, which has recently been demonstrated to interfere with the transcriptional activity of HBV cccDNA through direct binding to the HBV core promoter [81]. Other TRIM family members have also been demonstrated to restrict HBV transcription, including TRIM41, which transactivates an E3 ubiquitin ligase to inhibit HBV enhancer elements I and II [83]. The PRR RIG-I has been shown to have a dual antiviral effect on HBV pgRNA. In addition to sensing pgRNA and inducing expression of type III IFN, it counteracts the interaction of HBV polymerase with the 5′-ε region, suppressing viral replication [35]. ISGs additionally act on a posttranscriptional level on HBV pgRNA, including the zinc finger antiviral protein (ZAP), which binds to HBV RNAs and decays it through its ZRE sequence domains [85]. Also, myeloid differentiation primary response protein 88 (MyD88) is able to accelerate the decay of viral pgRNA [60] and myxoma resistance protein 1 (MxA) inhibits the encapsidation of pgRNA [58, 91] (Figure 2 and Table 1).

Notably, type I IFNs, especially IFNα, have been extensively used in the past in the treatment of chronic HCV and HBV infection [92]. Even though the systemic administration of IFNα is associated with significant side effects and results in generally poor response rates (e.g., fewer than 50% therapeutic success in HCV infection), it is to date the sole therapeutic intervention, which may result in a functional cure of HBV infection through HBsAg seroconversion [93]. Analysis of genetic factors associated with HCV chronicity have revealed the strong association of IFN, specifically type III IFN with not only treatment response but also with the spontaneous clearance of HCV

infection. In particular, human BDCA3+ DCs, mainly allocated in the liver, are able to sense HCV in a CD81-, endosome-, and TRIF-dependent manner and generate high quantities of IL-28B/IFN-λ3. This particular subset of dendritic cells has a superior response to HCV in patients with the major genotype (TT) for the SNP rs8099917 [94]. Genome-wide association studies (GWAS) furthermore revealed that a number of single nucleotide polymorphisms within the locus of type III IFNs are strongly associated with HCV persistence [95–97]. Later, it was identified that one of the SNPs, variant ss469415590, is forming a thus far unknown IFN, IFNL4, which is resulting in poor treatment outcome and little spontaneous clearance [98], indicating that the role of IFN in viral infections is more complex than originally anticipated. Furthermore, an association of the presence or absences of IFNL4 and other viral infections, including HBV, have still to be confirmed.

### 2.2. Innate Immune Responses

Cell-based innate immune responses against HBV and HCV have been extensively studied in the past years. These include Natural Killer (NK) cell, NKT cells, monocyte/macrophage, dendritic cells as well as specialized myeloid subpopulations (e.g., myeloid-derived suppressor cells) (Figure 3).

![Diagram](attachment:diagram.png)

**Figure 3.** Cell based innate and adaptive immune responses to HBV and HCV infection. The role of innate immune cells (Kupffer cells, myeloid-derived suppressor cells (MDSC), NK cells and Dendritic cells) and adaptive immune cells (B cells, CD4 T cells, CD8 T cells and regulatory T cells) on HBV and HCV infection and mechanisms devised by HCV (red) and HBV (green) to subvert these.

#### 2.2.1. NK Cell Responses in Chronic Viral Hepatitis

NK cells, which can be divided into CD56<sup>bright</sup>, IFNγ-producing, and Th1-priming, and CD56<sup>dim</sup>, fully mature and highly cytotoxic cell subsets play an important role in early innate immune responses [99]. Since their activation and NK cell-mediated killing is generally independent of prior

sensitization, their activation is tightly regulated. This involves a plethora of killer immunoglobulin-like receptors (KIR) the lectin-like receptors [100]. NK cell activation, in general, is driven by “missing self” antigen and NK cells are inhibited by the presence of major histocompatibility complex (MHC) class I molecules on target cells, preventing them from killing. Upon loss of MHC class I, which is often down-regulated during viral infections, NK cells kill the target cell via direct cytotoxicity. Both, KIR and MHC genes exhibit significant genetic diversity, resulting in KIR predominantly recognizing specific subgroups of MHC class I alleles. Furthermore, KIR may be inhibitory or activating, furthermore adding complexity to NK cell-mediated killing [101]. NK cells represent a major cell population in the liver, where they are enriched to approximately 30% of liver-resident lymphocytes [102].

Although in vitro data suggest that NK cells can effectively kill HCV-infected replicon cells, it also has been demonstrated that direct contact of NK cells with hepatocytes may impair their killing capabilities and their IFNγ response by thus far poorly understood mechanisms [103,104]. Furthermore, it has been suggested that hepatocytes are relatively resistant to perforin- and granzyme-mediated cytotoxicity [105]. During chronic HCV infection, NK cells remain in a state of chronic activation, thus contributing to the progression of liver fibrosis in HCV infection [106].

Elegant experiments in the HBV-transgenic mouse model have identified NK1.1⁺CD3⁻ NK cells as the major constituent of inflammatory infiltrates in the liver [107]. This large number of NK cells in HBV infection could contribute to the liver injury, which is observed during HBV infection. Additionally, hepatocyte apoptosis has been implicated as the main event in the initiation of hepatic inflammation and HBV-infected apoptotic cells result in an immunogenic rather than tolerogenic immune response [108]. In particular, TRAIL, which is expressed specifically on NK cells in the liver of HBV-infected patients, has been reported to correlate with liver damage [109]. TRAIL is particularly highly expressed on CD56bright NK cells, which are preferentially expanded in the liver [110]. Another important NK cell-mediated hepatocyte apoptosis pathway is initiated by Fas, which plays a particularly important role in DC-activated NK cells [111]. Finally, NK cells are a potent source of the pro-inflammatory cytokine TNF, which is greatly enhanced during chronic HBV infection and has been shown to inhibit HBV replication via disruption of capsid integrity [112,113].

### 2.2.2. NKT Cell Responses in Chronic Viral Hepatitis

NKT cells, which are greatly enriched in the liver, bridge the gap between innate and adaptive immune responses. They are T cells, which, in contrast to conventional T cells, recognize lipid-based antigen via the MHC-like molecule CD1d [114]. Once activated, NKT cells are able to secrete a diverse range of cytokines, which contributes to the activation of other immune cells. NKT cells are divided into two distinct families, type I “invariant” NKT cells, which harbor the Vα24-Jα18 invariant TCR α chain, recognizing α-galactosylceramide (α-GalCer) and type II “non-invariant” NKT cells, which express a range of TCR and bind various lipid antigen [114]. Both these subclasses of NKT cells have been shown to be activated during HBV infection, resulting in them expressing IFNγ [115].

### 2.2.3. Kupffer Cell Responses in Chronic Viral Hepatitis

Kupffer cells (KCs), the liver-resident macrophage type, which usually plays an essential role in regulating liver immune functionality and tolerance, are long-lived and abundant in the liver, constituting approximately 15–20% of the total liver cell population [116]. Their roles range from regulating the tolerogenic environment in the liver, suppressing T cell activation to orchestrating immune responses. KCs are professional antigen-sensing cells and express a variety of PRR, rendering them capable of secreting large amounts of reactive oxygen species (ROS), type I IFN and pro-inflammatory cytokines [117].

In contrast to HBV DNA replication, which is occurring hidden within capsids, HCV replication generates a plethora of pathogen-associated molecular patterns (PAMP). To date, the role of KCs as host cells for HCV in addition to hepatocytes remains controversial [117]. If infected, KCs could play an important role in recognizing early HCV infection [117]. During chronic HCV infection, KCs exhibit

massively enriched MHC class II expression levels, which has been associated with the presence of IFNγ, which may likely be produced by NK cells [118]. Usually, this MHC class II expression facilitates priming of CD4 T cells, however, KCs have the ability to cross-present viral antigen to CD8 T cells, presenting an effective means for KCs to prime CD8 T cell responses in the infected liver [119]. A number of studies, however, indicate aberrant KC functions in HBV- and HCV-infected individuals, which may be a result of HBV-mediated TLR2 down-regulation on KCs leading to reduced immune surveillance [118, 120, 121] (Figure 2). Additionally, KCs have been shown to secrete reduced amounts of TNFα in the presence of HBV antigen, which has previously been demonstrated to affect HBV replication by disrupting capsid integrity through activation of NFκB [113, 122]. Recently, it has been shown that exposure of primary monocyte-derived macrophages to high titers of HBV induced expression of inflammatory cytokines. In contrast, these cells are unable to sense HBV, when low titers were used [90].

### 2.2.4. The Role of Myeloid-Derived Suppressor Cells in Chronic Viral Hepatitis

Recently, bone marrow-derived myeloid-derived suppressor cells (MDSC) have been reported to play a fundamental role in regulating liver injury during chronic HBV as well as HCV infection [123]. Specifically, granulocytic MDSCs have been demonstrated to suppress T cell-mediated immunopathology in HBV by depriving T cells of L-arginine [124]. MDSC have also been reported to be induced during chronic HCV infection and exert their immunomodulatory role through similar mechanisms [123].

### 2.2.5. Dendritic Cells in Chronic Viral Hepatitis

Dendritic cells (DCs), which are the main drivers of priming adaptive immune responses to infections, are among the most professional antigen-presenting cells (APC) [125]. They constantly take up foreign antigen and have the potential to present through MHC class I and II, thus priming both, CD4 and CD8 T cell responses. Additionally, through co-stimulatory molecules, including CD80 (B7.1) and CD86 (B7.2) and high-level secretions of IL-12, they activate naïve CD4 and CD8 T cells [126]. DCs comprise at least two main subpopulations with additional subclasses. Myeloid DC, although representing the dominant subpopulation of DCs in the blood, only make up 0.5% of the total PBMC. They constitute of two distinct populations, expressing either CD1c (BDCA-1) or CD141 (BDCA-3) as well as HLA-DR/CD11c/CD1c or CD11c/CD141. mDCs localize to tissue, where they internalize antigen and prime T cell responses [127]. Plasmacytoid DCs comprise an even smaller population of PBMC, averaging about 0.2%. They are expressing CD303 (BDCA-2) or CD123 on lineage negative, CD11c negative HLA-DR-expressing cells. Even though they are able to present antigen, their role is predominantly the production of IFNα in response to the sensing of pathogens by PAMPs [127]. In the liver however, DCs are accompanied by liver sinusoidal endothelial cells (LSEC), which also express CD54, CD80, CD86 and MHC class I and II, thus functioning as APC for CD4 and CD8 T cells as well [128]. Under normal conditions, liver-resident DCs have an immature phenotype, as indicated by low-level expression of MHC class I and II as well as CD86. In liver biopsies of HCV-infected patients, this phenotype is shifted towards a more mature phenotype [129] and in vitro, DCs have been shown to mature in the presence of GM-CSF and HBV antigen [130], thus suggesting a functionally active phenotype during infection. However, data on maturation ability of DCs in the context of HCV infection are still conflicting and further work is required to clearly dissect the role of DCs in chronic HCV infection [131–134]. In chronic HBV infection, pDCs have been shown to differ significantly from those of healthy donors, in that TLR9-mediated IFNα production is impaired in chronic HBV infection [135]. The mechanism for this is however largely unclear.

### 2.2.6. Extracellular Vesicles in HBV and HCV Innate Immune Responses

Exosomes are cell-derived vesicles, which have been shown to carry various molecular constituents of their origin cell. Through membrane vesicle transport, exosomes can transfer proteins,
lipids and nucleic acids between different cell types. Recently, it has been demonstrated that HBV- and HCV-infected hepatocytes interact with and stimulate DCs via secretion of viral nucleic acid-containing exosomes [136, 137]. These exosomes have been demonstrated to transfer IFN-induced antiviral activity to neighboring cells, bypassing the requirement for JAK/STAT signal transduction by directly containing ISG [137, 138]. Furthermore, exosomes can transfer infectious HBV and HCV genomes to neighboring cells, resulting in receptor-independent entry [139–142]. In addition to being secreted from infected hepatocytes, exosomes derived from macrophages during HCV infection were shown to contain microRNA (miRNA)-29 family members, which inhibits HCV replication [143]. Even though it has not yet been shown for HBV or HCV infection, virions themselves can act as transport vesicles for immune activators, including cGAMP, which induces IFN responses in cells following entry [144].

### 2.3. Adaptive Immune Responses

Additionally, cell-based adaptive immune responses against HBV and HCV have been investigated for years. These include CD4 and CD8 T-cells, regulatory T-cells and B-cells (Figure 3).

#### 2.3.1. T Cell Responses to HBV and HCV

The outcome of both, HBV and HCV infection is guided by a combination of genetic factors and virus-specific immune responses. Early studies in chimpanzees and observational studies in humans have identified virus-specific CD4 and CD8 T cell responses as important mechanisms for protection against persistence, even though they are delayed in most cases, probably due to the tolerogenic environment of the liver.

Specifically, the magnitude and breadth of CD4 and CD8 T cell response seems to be a key determinant for spontaneous clearance of HCV infection. During the progression from acute to chronic HCV infection, antigen-specific CD8 T cells are recruited to the liver and nearly disappear from peripheral circulation. Similarly, broadly reactive CD4 T cells are predominantly detectable during acute infection, whereas their numbers decline during chronic infection. Both, CD4 and CD8 T cell responses are deterministic of whether an acute infection is resolved, or whether it progresses to chronic infection. In vivo studies in chimpanzees demonstrated a clear association of CD4 T cell help in maintaining functional CD8 T cell responses, since depletion of CD4 T cells resulted in a decline of antigen-specific CD8 T cell numbers and their functionality.

In chronic infection, the HCV-specific CD4 and CD8 T cells are characterized by high expression of several exhaustion markers, including T cell immunoglobulin and mucin domain (Tim-) 3, cytotoxic T lymphocyte-associated antigen-(CTLA) 4 and 2B4, which restricts their polyfunctionality, ability to proliferate and degranulate in response to cognate antigen. This includes key effector cytokine production, including interleukin (IL)-2, interferon (IFN)-γ, TNF-α and the degranulation marker CD107a and prevents terminal differentiation to long-lived, CD127-positive memory T cells [145]. Recently, it has been shown in human and chimpanzee studies that this exhausted phenotype is not reverted by treatment with immune checkpoint inhibitors, including antibodies directed against PD-1 [146–149]. This is a clear result from the prolonged exposure to high levels of viral antigen since CD8 T cells with antigen-specificity for a mutated HCV epitope exhibit lower expression levels of PD-1 and increased levels of responsiveness.

In addition to these classical effector T cell populations, CD4⁺CD25⁺FoxP3⁺ regulatory T cells (TREG) have been shown to be enriched in HBV- and HCV-infected patients and actively suppress immune responses to HBV and HCV [150]. This feature of TREG is extending beyond active infection since even TREG from patients following spontaneous clearance are able to suppress HBV-specific CD8 T cell responses. Additionally, HBcAg-specific TREG secreting IL-10 contributes to the suppression of IFNγ in CD4 T cell populations and depletion of TREG restores IFNγ secretion ex vivo [151, 152]. This partial restoration of HBcAg-specific T cell responses has also been observed in HBV-infected patients on antiviral treatment, where Adefovir- and Entecavir-treated patients exhibited a decrease in circulating TREG upon viral load suppression [153, 154]. In HCV infection, TREG have been shown to be
enriched and able to suppress the secretion of IFNγ from HCV-specific CD8T cells, indicating a role for T<sub>REG</sub> in the development of chronic infection [155].

Strikingly, this exhausted T cell phenotype is also conserved in tissue-resident T cells (T<sub>RM</sub>), which have recently been shown to be enriched in the liver of patients with chronic HBV infection. These cells, which are characterized by high levels of expression of CD69 and presence or absence of αEβ7 integrin (CD103), consist of a heterogeneous population of memory T cells, including mucosal-associated invariant T (MAIT) cells and γδT cells. Especially the population of CD69<sup>+</sup>CD103<sup>+</sup>T<sub>RM</sub> cells is greatly increased in the liver of HBV-infected patients, accounting for more than 20% of total memory CD8T cells, compared to 10% in uninfected donors. Even though this population expresses high levels of PD-1, they readily produce IFNγ, IL-2, and TNFα upon restimulation [156].

### 2.3.2. B Cell Responses in HBV and HCV

Similar to the T cell responses, B cell and humoral responses to infection with HCV are delayed, averagely peaking 6–8 weeks following detection of HCV RNA levels in serum. In contrast to T cell responses, which correlate with spontaneous clearance, there is no association with the development of a robust antibody response and clearance of HCV, since in humans who spontaneously clear infection; antibody responses are often delayed and low in titer. Even though similar to other infectious diseases, including HIV, HCV sometimes elicits the development of broadly neutralizing antibodies (bnAb), these usually are not protective [157]. Even though it has been shown, that these bnAbs are able to control infection in vitro, very limited evidence for the establishment of sterilizing immunity is available [158]. Thus, HCV is almost uniformly able to re-infect patients upon spontaneous or treatment-induced viral clearance.

In contrast, humoral immune responses to HBV infection as well as HBV immunization are predominantly protective and elicit sterilizing immunity. Especially in birth dose vaccinations of HBV, studies have shown 90% of immunized children to exhibit high levels of protective antibodies and that antibody protection against HBV infection is long-lived at over 20 years [159,160]. Only 25% of immunized exhibited antibody levels deemed below protective levels after 22 years of initial vaccination [161]. During natural HBV infection, antibody-mediated protection has been shown to be sterilizing, if production rates of HBsAg-containing sub-viral particles are low, thus facilitating neutralization of Dane particles, or if anti-HBV antibody production is fast and of high affinity [161].

## 3. Immune Evasion Strategies of HCV

### 3.1. Viral Proteins and Their Role in Immune Evasion

With HCV replication restricted to extranuclear compartments, recognition of the uncapped HCV RNA genome or partially double-stranded RNA regions via RIG-I or MDA5 are a constant threat to the virus [57,162,163]. To circumvent the RIG-I/MDA5-initiated induction of interferon regulatory factor (IRF)3 and subsequent production of IFN, the HCV protease NS3/4a has evolved to exert a dual function. In addition to processing and cleaving the HCV polyprotein, NS3/4a rapidly cleaves the adapter protein MAVS, which is essential for relaying RIG-I/MDA5-initiated signal transduction as well as TRIF, which is a downstream signaling mediator of Toll-like receptor (TLR)3, which recognizes double-stranded RNA within endosomal compartments [164,165]. This, in turn, prevents the expression of type I and III IFN [166]. Additionally, NS3 itself was shown to bind tank binding kinase (TBK)1, thus preventing engagement of IRF3 and expression of IFN (Figure 1).

To circumvent the response to paracrine IFN, HCV core has evolved to antagonize and attenuate the JAK/STAT signaling pathway by directly interacting with STAT1. This binding prevents its phosphorylation and thus, activation. Furthermore, HCV core is able to specifically up-regulate SOCS3, thus down-regulating JAK/STAT activity [167]. Moreover, the NS4b protein of HCV efficiently targets STING-mediated TBK1 activation, suppressing the induction of type I IFN [168]. Most notably, HCV has evolved an elaborate structure to bypass host innate immune sensing. Since HCV RNA replication

is a prime accumulation of PAMPs for RNA sensors, including RIG-I and MDA5, the complete HCV replicase complex is enveloped in a unique structure termed the membranous web. This structure, which is formed by viral nonstructural proteins from the endoplasmic reticulum serves as a shield against host responses. It has been recently shown that each of the double-membrane vesicles of the membranous web is sealed by a nuclear pore, which restricts entry of PRR and prevents early signaling, which would result in interferon production [169] (Figure 1).

Other HCV proteins have evolved antagonizing functions against late ISG, including NS5a, which can bind to 2′,5′-OAS, blocking its antiviral effect and reducing ISG expression levels [170]. NS5a has also been implicated in directly binding to the TLR adaptor MyD88 to inhibit TLR-mediated innate immune evasion as well as to protein kinase (PK)R [171]. PKR has been shown can be inhibited by the envelope protein E2 [172] (Figure 1).

### 3.2. Viral Variability and Immune Evasion

The decline in T cell responses, as well as the overall exhausted phenotype of T cells in chronic HCV infection, is thus far poorly understood. Models of HCV kinetics suggest that up to \(10^{12}\) virions are being produced daily [173,174]. HCV NS5b as RNA-dependent RNA polymerase without proofreading capability results in an extensive accumulation of viral quasi-species with a mutation rate of \(10^{-3}\) nucleotides per year [174]. Even though many of these are not functional, HCV has developed a remarkable ability to introduce mutations into various structural and non-structural proteins, which results in replication-competent virus [174]. Mutation of MHC class I and II restricted T cell epitopes may affect the outcome of infection by delaying clearance of HCV-infected cells and may greatly contribute to loss of functionality in T cell responses directed against HCV [175]. Even though this impaired T cell receptor engagement facilitates immune escape, there should be sufficiently novel T cell repertoire to account for these novel antigens. However, it was shown that a substitution in an amino acid of a variant/A*0201 complex results in unaltered binding affinity with significantly impaired ability to expand primed T cells and poor activation of previously primed antigen-specific T cells [176,177].

### 3.3. Evasion of Humoral Immune Responses

The HCV life cycle, in contrast to other viruses, is characterized by two distinct viral egress pathways. In addition to the release of viral particles into the extracellular space, HCV particles can directly cross tight junctions, infecting neighboring cells without the need to diffuse. This, in turn, prevents HCV virions being exposed to antibodies, which may neutralize infectivity. To date, the exact mechanisms of this cell-to-cell spread remain poorly understood, however, cellular lipids have been shown to play an important role during this process [178]. A second mechanism, by which HCV efficiently evades antibody neutralization, is via shifting glycosylation patterns on the E2 glycoprotein [179].

### 3.4. Viral Mimicry of Cellular Lipids

In contrast to other members of the Flaviviridae, HCV infectious particles are neither uniform in size nor symmetric in architecture. HCV has evolved assembly and maturation steps in close proximity to cellular lipid droplets. The majority of the infectious HCV particle is comprised of host cellular lipids, including apolipoproteins (Apo)E, A1 and B. Only very few viral antigen and glycoproteins are exposed on this so-called lipoviral particle, potentially to limit exposure of viral antigen to the immune system. Even though ApoE incorporation into HCV particles prevents antibody recognition and enhances infectivity [180], it has been demonstrated that antibodies against ApoE can successfully neutralize HCV infectivity, furthermore underlining the importance of host cell-derived lipids in the HCV life cycle.

Vaccines 2017, 5, 24

---

## 4. Immune Evasion Strategies of HBV

### 4.1. Viral Proteins and Their Role in Immune Evasion

Encoding for only eight proteins on six mRNA transcripts, most HBV proteins have evolved ways to impair immune recognition. Initially, to protect the rcDNA from PRR recognition, the HBV capsid does not disintegrate following fusion with the endosome but is actively imported into the nucleus. In contrast to HCV, all coding nucleic acids of HBV are capped and polyadenylated, preventing their recognition by RIG-I or MDA5. However, regions of the HBV pgRNA are double-stranded and may interact with dsRNA sensors. To counteract this, HBV is inducing parkin and the linear ubiquitin assembly complex (LUBAC), which in turn attenuates signal transduction through MAVS [181]. At least two distinct tolerogenic mechanisms can be ascribed to HBeAg and HBsAg. HBeAg, which is dispensable for viral replication, is secreted from infected cells and has been demonstrated to cross the placenta, leading to neonatal tolerance, suppression of antibody and T cell responses to HBcAg, with which it shares sequence homology. In contrast, HBsAg, which comprises the glycoproteins S, M, and L of HBV, is secreted from infected cells as infectious Dane particle as well as sub-viral particles. These sub-viral particles are produced in vast excess and are a potent tolerogen. These two secreted viral factors are produced to such high levels that they mask the corresponding antibodies through exhaustive binding. Additionally, both HBeAg and HBsAg have been implicated in impairing TLR signaling initiation. This is orchestrated by HBeAg or the corresponding intracellular p22 form binding directly to MyD88 and TIRAP and interfering with TLR2 signaling. HBsAg has been demonstrated to inhibit TLR2 and c-Jun N-terminal protein kinase (JNK), thus preventing the production of interleukin (IL)-12 [182]. Another important role for HBsAg lies in the subversion of TLR3 signaling since it was demonstrated that HBV elicits type I IFN responses in the absence of HBsAg. Additionally, HBV polymerase actively inhibits both importin α5 and protein kinase C-δ, thus preventing activated STAT1/2 heterodimers to enter the nucleus to transactivate ISG expression [183]. This points to an active interference with most PRR responsible for recognizing HBV infection. Moreover, this IFN induction may contribute to the immune control of HBV infection upon successful treatment-induced HBsAg seroconversion. Both HBsAg and HBeAg furthermore inhibit the interaction of major vault protein (MVP) with MyD88 in infected cells, thus preventing type I IFN responses [184] as well as inhibiting the action of the ISG tetherin, which usually prevents virion budding [185] (Figure 1).

In addition to these extracellular immune evasion mechanisms, HBV proteins have also been shown to directly interact with host innate immune activation mechanisms. The HBV polymerase, in addition to functioning as reverse transcriptase for pregenomic (pg) RNA, has been shown to directly interfere with PRR signaling via interaction with DDX3 and subsequent inhibition of TBK1 [186]. Additionally, HBp is able to directly interfere with STING-mediated induction of DNA sensing within infected cells [187] and prevent the activation of IKKs through interaction with HSP90β [188]. Furthermore, HBVp masks the antigenic step of reverse transcription by delaying reverse transcription until the HBV pol/pgRNA complex is successfully encapsidated (Figure 1).

A novel immune evasion pathway by HBV has been recently described, which exploits the targeting of interferon regulatory factor (IRF) 3 and 7 nuclear translocation via the NFκB essential modulator (NEMO) by the RUN domain Beclin-1 interacting cysteine-rich-containing (RUBICON) protein, which is driven by HBx [189] (Figure 1).

The HBx protein has been associated to a plethora of biological functions, including competitive inhibition of MAVS-, TRIF-, and IRF3-mediated dsRNA recognition through their proteasomal degradation [190]. Additionally, HBx has been implicated in the evasion of transcriptional repression of HBV cccDNA via the structural maintenance of chromosome 5/6 complex (Smc5/6) by targeting it for degradation [191] as well as in inhibiting the transcription of TRIM22, an ISG with anti-HBV capacity [192]. However, many of the HBx-associated mechanisms of innate immune evasion stem from overexpression studies. The HBV proteins core and pre-core can inhibit the expression of MxA in a transcriptional level based on performed in vitro experiments [193] (Figure 1).

Vaccines 2017, 5, 24

4.2. Viral Mimicry of HBV cccDNA as Minichromosome

An important step within the HBV life cycle is the conversion of HBV rcDNA to cccDNA, whereby the covalently linked HBV polymerase, as well as an RNA linker, are removed and the incomplete (−)-strand of HBV is repaired [194]. During this process, the HBV genome is organized on host-derived histones [195]. Transcriptional regulation of HBV has been shown to be regulated on an epigenetic level similar to gene regulation on mammalian genes [196]. However, the HBx and HBc proteins play a role in shaping the transcriptional landscape of the HBV genome [197,198]. It has been demonstrated that host factors, including the SETDB1 histone methyltransferase, actively repress HBV transcriptional activity by decreasing histone H3 acetylation and increasing H3K9 tri-methylation [199]. This process is however rapidly reversed by HBx binding to cccDNA and inducing complete transcriptional activation [199]. This host mimicry in combination with HBV transcripts being indistinguishable to host mRNA due to the capped structure and polyadenylation are the main reasons for the extensive stability of HBV cccDNA and absence of innate immune sensing [194]. Even though several cellular mechanisms have been proposed, which could limit the number of cccDNA molecules, most notably the deaminase function of APOBEC3 family members, their role in actually affecting cccDNA quantity in vivo is still highly controversial [200–205] (Figure 2).

4.3. Evasion of Innate and Adaptive Immune Responses

In addition to shaping the innate immune activation landscape of HBV-infected hepatocytes, HBV has furthermore been shown to affect a number of other, liver-resident cell types. HBeAg, which is secreted by HBV-infected hepatocytes, is impairing TLR2 expression levels in KCs, thus suppressing their ability to respond to cell-wall components, lipoteichoic acid, and lipoproteins [121,206]. Furthermore, macrophages exposed to HBsAg and HBcAg selectively inhibit TLR2 activation, limiting IL-12 secretion and contribute to CD8 T cell exhaustion via a direct interaction between HBcAg and TLR2, respectively [120,182]. An additional interaction between HBcAg and gClqR further suppresses TLR4-induced IL-12 production [207]. These mechanisms have been shown to be particularly important for the induction of tolerance in vertical transmissions of HBV, where maternally derived HBeAg enhances PD-L1 expression in macrophages of offspring to suppress the induction of CD8 T cell responses [208].

NK cells, which can take up HBV nucleic acids through exosomes, exhibit an impaired phenotype in HBV infection, since IFNγ production, cytotoxic activity, proliferation and responsiveness to exogenous stimuli is reduced, compared to healthy donors [139]. Even though it has been shown that exosomes from HBV-infected hepatocytes contain rcDNA, sub-genomic HBV RNA and HBV proteins, the exact mechanism for this are still unclear. Another feature of the interaction of HBV with NK cells is their general frequency, activation and cytokine production, which is significantly reduced in HBV-infected patients compared to healthy controls with especially the anti-inflammatory cytokine IL-10 being elevated [209,210]. Additionally, NK cells of HBV-infected patients are characterized by elevated expression levels of inhibitory receptors and reduced expression of activating receptors [211–213]. This could explain the altered responses of NK cells in HBV infection.

Another method of evading immune detection of HBV is the selective loss of HBeAg through the accumulation of mutations within the pre-core or basal core promoter [214]. This loss of HBeAg, which usually occurs during the inactive carrier phase of infection, removes one of the key targets for the immune response, effectively concealing HBV from immune recognition [12].

5. Conclusions

HBV and HCV as hepatotropic viruses have adapted efficiently to survive within the liver, where the highly tolerogenic environment further facilitates viral persistence. This sets HBV and HCV apart from other infectious diseases, which persist in less immune-privileged sites within the body. Despite the overall similar disease progression, where liver cirrhosis and hepatocellular carcinoma often define the endpoint of infection, HBV and HCV have evolved distinct means of evading or subverting the

host response. Overall, HBV is more immune silent compared to HCV, simply by the reduced number of PAMP present in the viral genome. Most of the HBV life cycle remains hidden from the host by physically locating it within assembled viral capsids. Additionally, HBV has evolved all its transcripts to resemble cellular mRNAs, further reducing the likelihood of immune recognition. HCV, on the other hand, encodes a multitude of PAMPs, ranging from its uncapped 5' internal ribosomal entry site (IRES) structure, partially double-stranded RNA genome segments and lack of polyadenylation signal. These features would suggest, that HBV actively interferes with immune activation, rather than remaining hidden from immune detection. However, this is still controversial and evidence for both hypotheses have been presented. However, since HBsAg, which is also used as HBV vaccine, is highly immunogenic [215–217], the nearly complete absence of immune responses during early HBV infection still remains elusive. Further study of the interaction of HBV and the immune system in physiological model systems are therefore required.

To counteract the inevitable activation of cytoplasmic RNA sensors, which have specialized in recognizing these PAMP, HCV has adopted more direct means of immune evasion. This includes a crippling of the RNA-mediated IFN responses by disrupting signaling through TLR3, RIG-I, and MDA5 via its protease. Replication of HCV RNA, which is another foreign process in cells, is hidden from the host cell by enveloping itself in the membranous web, further underlining the plethora of changes, which HCV induces in infected cells to hide from immune recognition. In contrast, steps during the HBV life cycle, which carry a significant risk of detection by the immune system, are well hidden within the viral capsid. However, this concealment is virally encoded and does not require extensive restructuring of infected cells. Nevertheless, both viruses encode proteins, which the host may recognize as being foreign. Over the past years, it has been more and more acknowledged that HBV, in addition to HCV, modulates early events during infection in order to restrict this from happening. This has gradually shifted the view on HBV from a pathogen, which interacts very limited with the innate immune system, to one, which expertly subverts recognition. Additionally, the HBV has been shown to interact with a wide range of other immune cells to maintain the initial immunotolerant state after infection.

Even though several interferon effector molecules have been described to affect infection of HBV or HCV, many of the studies have been performed using overexpression of ISG. However, endogenous IFN-mediated induction of many ISG is resulting in much more subtle expression changes. Thus, it remains an open question as to how much these ISG contribute to viral clearance over simply delaying virus replication until adaptive immune responses can be mounted.

Once chronic infection is established, both, HBV and HCV interfere with adaptive T cell responses, resulting in the gradual emergence of exhausted T cell phenotypes. Even though current interventional studies are aimed at breaking T cell tolerance, the exact mechanisms, by which T cell exhaustion occurs and is maintained, require further investigation. In an era of direct acting antiviral drugs able to eradicate HCV from the liver, studies are needed to evaluate the long-term impact of HCV antigen removal on T cell exhaustion. This can potentially result in important insights, which may be translatable to HBV infection.

To conclude, multiple pathways, by which HBV and HCV evade innate and adaptive immune recognition have been described. Nevertheless, important open questions in regards to why HBV infection does not induce early innate immune activation, the detailed mechanisms of action of ISG against HCV or HBV, whether HBV cccDNA can be exposed to immune recognition, the inability of HCV to induce sterilizing immunity and the mechanism of HBV- and HCV-associated T cell exhaustion still requires further investigation.

Acknowledgments: A.M.O.P. and M.D. are funded through starting grant from the European Research Council (ERC-StG-2015-637304). M.D. is funded through a Wellcome Trust New Investigator award (104771/Z/14/Z) and from funding through the Imperial NIHR Biomedical Research Centre.

Author Contributions: A.M.O.P. and M.D. conceived and wrote this manuscript.

Conflicts of Interest: The authors declare no conflict of interest.

References

1. Lampertico, P.; Agarwal, K.; Berg, T.; Buti, M.; Janssen, H.L.A.; Papatheodoridis, G.; Zoulim, F.; Tacke, F.; European Association for the Study of the Liver. EASL 2017 clinical practice guidelines on the management of hepatitis B virus infection. *J. Hepatol.* **2017**. [CrossRef] [PubMed]
2. Petruzziello, A.; Marigliano, S.; Loquercio, G.; Cacciapuoti, C. Hepatitis C virus (HCV) genotypes distribution: An epidemiological up-date in europe. *Infect. Agent. Cancer* **2016**, *11*, 53. [CrossRef] [PubMed]
3. Lin, M.V.; King, L.Y.; Chung, R.T. Hepatitis C virus-associated cancer. *Annu. Rev. Pathol.* **2015**, *10*, 345–370. [CrossRef] [PubMed]
4. Sharma, S.A.; Feld, J.J. Acute hepatitis C: Management in the rapidly evolving world of HCV. *Curr. Gastroenterol. Rep.* **2014**, *16*, 371. [CrossRef] [PubMed]
5. Dustin, L.B.; Rice, C.M. Flying under the radar: The immunobiology of hepatitis C. *Annu. Rev. Immunol.* **2007**, *25*, 71–99. [CrossRef] [PubMed]
6. Jindal, A.; Kumar, M.; Sarin, S.K. Management of acute hepatitis B and reactivation of hepatitis B. *Liver Int.* **2013**. [CrossRef] [PubMed]
7. Fattovich, G.; Bortolotti, F.; Donato, F. Natural history of chronic hepatitis B: Special emphasis on disease progression and prognostic factors. *J. Hepatol.* **2008**, *48*, 335–352. [CrossRef] [PubMed]
8. Guidotti, L.G.; Chisari, F.V. Immunobiology and pathogenesis of viral hepatitis. *Annu. Rev. Pathol.* **2006**, *1*, 23–61. [CrossRef] [PubMed]
9. Thomas, D.L.; Thio, C.L.; Martin, M.P.; Qi, Y.; Ge, D.; O’Huigin, C.; Kidd, J.; Kidd, K.; Khakoo, S.I.; Alexander, G.; et al. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. *Nature* **2009**, *461*, 798–801. [CrossRef] [PubMed]
10. McMahon, B.J. The natural history of chronic hepatitis B virus infection. *Hepatology* **2009**, *49*, S45–S55. [CrossRef] [PubMed]
11. McMahon, B.J. Natural history of chronic hepatitis B. *Clin. Liver Dis.* **2010**, *14*, 381–396. [CrossRef] [PubMed]
12. Hadziyannis, S.J.; Papatheodoridis, G.V. Hepatitis B e antigen-negative chronic hepatitis B: Natural history and treatment. *Semin. Liver Dis.* **2006**, *26*, 130–141. [CrossRef] [PubMed]
13. Wu, J.F.; Chang, M.H. Natural history of chronic hepatitis B virus infection from infancy to adult life—The mechanism of inflammation triggering and long-term impacts. *J. Biomed. Sci.* **2015**, *22*, 92. [CrossRef] [PubMed]
14. Hakim, M.S.; Spaan, M.; Janssen, H.L.; Boonstra, A. Inhibitory receptor molecules in chronic hepatitis B and C infections: Novel targets for immunotherapy? *Rev. Med. Virol.* **2014**, *24*, 125–138. [CrossRef] [PubMed]
15. Fensterl, V.; Chattopadhyay, S.; Sen, G.C. No love lost between viruses and interferons. *Annu. Rev. Virol.* **2015**, *2*, 549–572. [CrossRef] [PubMed]
16. Snell, L.M.; McGaha, T.L.; Brooks, D.G. Type I interferon in chronic virus infection and cancer. *Trends Immunol.* **2017**. [CrossRef] [PubMed]
17. Lazear, H.M.; Nice, T.J.; Diamond, M.S. Interferon-lambda: Immune functions at barrier surfaces and beyond. *Immunity* **2015**, *43*, 15–28. [CrossRef] [PubMed]
18. Schoggins, J.W. Interferon-stimulated genes: Roles in viral pathogenesis. *Curr. Opin. Virol.* **2014**, *6*, 40–46. [CrossRef] [PubMed]
19. Metz, P.; Reuter, A.; Bender, S.; Bartenschlager, R. Interferon-stimulated genes and their role in controlling hepatitis C virus. *J. Hepatol.* **2013**, *59*, 1331–1341. [CrossRef] [PubMed]
20. Shu, Q.; Lennemann, N.J.; Sarkar, S.N.; Sadovsky, Y.; Coyne, C.B. Adap2 is an interferon stimulated gene that restricts RNA virus entry. *PLoS Pathog.* **2015**, *11*, e1005150. [CrossRef] [PubMed]
21. Schoggins, J.W.; Wilson, S.J.; Panis, M.; Murphy, M.Y.; Jones, C.T.; Bieniasz, P.; Rice, C.M. A diverse range of gene products are effectors of the type I interferon antiviral response. *Nature* **2011**, *472*, 481–485. [CrossRef] [PubMed]
22. Chowdhury, S.; Kitamura, K.; Simadu, M.; Koura, M.; Muramatsu, M. Concerted action of activation-induced cytidine deaminase and uracil-DNA glycosylase reduces covalently closed circular DNA of duck hepatitis B virus. *FEBS Lett.* **2013**, *587*, 3148–3152. [CrossRef] [PubMed]
23. Lucifora, J.; Xia, Y.; Reisinger, F.; Zhang, K.; Stadler, D.; Cheng, X.; Sprinzl, M.F.; Koppensteiner, H.; Makowska, Z.; Volz, T.; et al. Specific and nonhepatotoxic degradation of nuclear hepatitis B virus cccDNA. *Science* **2014**, *343*, 1221–1228. [CrossRef] [PubMed]

24. Luo, X.; Huang, Y.; Chen, Y.; Tu, Z.; Hu, J.; Tavis, J.E.; Huang, A.; Hu, Y. Association of hepatitis B virus covalently closed circular DNA and human APOBEC3b in hepatitis B virus-related hepatocellular carcinoma. *PLoS ONE* **2016**, *11*, e0157708. [CrossRef](https://doi.org/10.1371/journal.pone.0157708) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27391168)

25. Renard, M.; Henry, M.; Guetard, D.; Vartanian, J.P.; Wain-Hobson, S. APOBEC1 and APOBEC3 cytidine deaminases as restriction factors for hepadnaviral genomes in non-humans in vivo. *J. Mol. Biol.* **2010**, *400*, 323–334. [CrossRef](https://doi.org/10.1016/j.jmb.2010.04.063) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20447351)

26. Gonzalez, M.C.; Suspene, R.; Henry, M.; Guetard, D.; Wain-Hobson, S.; Vartanian, J.P. Human APOBEC1 cytidine deaminase edits HBV DNA. *Retrovirology* **2009**, *6*, 96. [CrossRef](https://doi.org/10.1186/1742-4690-6-96) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19958508)

27. Suspene, R.; Guetard, D.; Henry, M.; Sommer, P.; Wain-Hobson, S.; Vartanian, J.P. Extensive editing of both hepatitis B virus DNA strands by APOBEC3 cytidine deaminases in vitro and in vivo. *Proc. Natl. Acad. Sci. USA* **2005**, *102*, 8321–8326. [CrossRef](https://doi.org/10.1073/pnas.0500153102) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15867158)

28. Qiao, Y.; Han, X.; Guan, G.; Wu, N.; Sun, J.; Pak, V.; Liang, G. TGF-beta triggers HBV cccDNA degradation through AID-dependent deamination. *FEBS Lett.* **2016**, *590*, 419–427. [CrossRef](https://doi.org/10.1002/1873-3468.12208) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26852837)

29. Que, L.; Liu, G.; Kitamura, K.; Wakae, K.; Li, Y.; Nishitsuji, H.; Ujino, S.; Shimotohno, K.; Muramatsu, M. Molecular characterization of AID-mediated reduction of hepatitis B virus transcripts. *Virology* **2017**, *510*, 281–288. [CrossRef](https://doi.org/10.1016/j.virol.2017.09.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28947176)

30. Liang, G.; Kitamura, K.; Wang, Z.; Liu, G.; Chowdhury, S.; Fu, W.; Koura, M.; Wakae, K.; Honjo, T.; Muramatsu, M. RNA editing of hepatitis B virus transcripts by activation-induced cytidine deaminase. *Proc. Natl. Acad. Sci. USA* **2013**, *110*, 2246–2251. [CrossRef](https://doi.org/10.1073/pnas.1219756110) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23382226)

31. Iwamoto, M.; Watashi, K.; Tsukuda, S.; Aly, H.H.; Fukasawa, M.; Fujimoto, A.; Suzuki, R.; Aizaki, H.; Ito, T.; Koiwai, O.; et al. Evaluation and identification of hepatitis B virus entry inhibitors using HepG2 cells overexpressing a membrane transporter NTCP. *Biochem. Biophys. Res. Commun.* **2014**, *443*, 808–813. [CrossRef](https://doi.org/10.1016/j.bbrc.2014.01.071) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24486857)

32. Chen, Y.; Wang, S.; Yi, Z.; Tian, H.; Aliyari, R.; Li, Y.; Chen, G.; Liu, P.; Zhong, J.; Chen, X.; et al. Interferon-inducible cholesterol-25-hydroxylase inhibits hepatitis C virus replication via distinct mechanisms. *Sci. Rep.* **2014**, *4*, 7242. [CrossRef](https://doi.org/10.1038/srep07242) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25487410)

33. Anggakusuma; Romero-Brey, I.; Berger, C.; Colpitts, C.C.; Boldanova, T.; Engelmann, M.; Todt, D.; Perin, P.M.; Behrendt, P.; Vondran, F.W.; et al. Interferon-inducible cholesterol-25-hydroxylase restricts hepatitis C virus replication through blockage of membranous web formation. *Hepatology* **2015**, *62*, 702–714.

34. Liu, S.Y.; Aliyari, R.; Chikere, K.; Li, G.; Marsden, M.D.; Smith, J.K.; Pernet, O.; Guo, H.; Nusbaum, R.; Zack, J.A.; et al. Interferon-inducible cholesterol-25-hydroxylase broadly inhibits viral entry by production of 25-hydroxycholesterol. *Immunity* **2013**, *38*, 92–105. [CrossRef](https://doi.org/10.1016/j.immuni.2012.11.014) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333352)

35. Sato, S.; Li, K.; Kameyama, T.; Hayashi, T.; Ishida, Y.; Murakami, S.; Watanabe, T.; Iijima, S.; Sakurai, Y.; Watashi, K.; et al. The RNA sensor RIG-I dually functions as an innate sensor and direct antiviral factor for hepatitis B virus. *Immunity* **2015**, *42*, 123–132. [CrossRef](https://doi.org/10.1016/j.immuni.2014.12.013) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25620650)

36. Sumpter, R., Jr.; Loo, Y.M.; Foy, E.; Li, K.; Yoneyama, M.; Fujita, T.; Lemon, S.M.; Gale, M., Jr. Regulating intracellular antiviral defense and permissiveness to hepatitis C virus RNA replication through a cellular rna helicase, RIG-I. *J. Virol.* **2005**, *79*, 2689–2699. [CrossRef](https://doi.org/10.1128/JVI.79.5.2689-2699.2005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15709017)

37. Chen, X.; Qian, Y.; Yan, F.; Tu, J.; Yang, X.; Xing, Y.; Chen, Z. 5′-triphosphate-siRNA activates RIG-I-dependent type I interferon production and enhances inhibition of hepatitis B virus replication in HepG2.2.15 cells. *Eur. J. Pharmacol.* **2013**, *721*, 86–95. [CrossRef](https://doi.org/10.1016/j.ejphar.2013.10.020) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24140861)

38. Miyashita, M.; Oshiumi, H.; Matsumoto, M.; Seya, T. Ddx60, a dxed/h box helicase, is a novel antiviral factor promoting RIG-I-like receptor-mediated signaling. *Mol. Cell Biol.* **2011**, *31*, 3802–3819. [CrossRef](https://doi.org/10.1128/MCB.05498-11) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21622923)

39. Oshiumi, H.; Miyashita, M.; Okamoto, M.; Morioka, Y.; Okabe, M.; Matsumoto, M.; Seya, T. Ddx60 is involved in RIG-I-dependent and independent antiviral responses, and its function is attenuated by virus-induced EGFR activation. *Cell Rep.* **2015**, *11*, 1193–1207. [CrossRef](https://doi.org/10.1016/j.celrep.2015.04.035) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25981368)

40. Grunvogel, O.; Esser-Nobis, K.; Reustle, A.; Schult, P.; Muller, B.; Metz, P.; Trippler, M.; Windisch, M.P.; Frese, M.; Binder, M.; et al. Ddx60l is an interferon-stimulated gene product restricting hepatitis C virus replication in cell culture. *J. Virol.* **2015**, *89*, 10548–10568. [CrossRef](https://doi.org/10.1128/JVI.01099-15) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26202998)

41. Kang, J.I.; Kwon, S.N.; Park, S.H.; Kim, Y.K.; Choi, S.Y.; Kim, J.P.; Ahn, B.Y. PKR protein kinase is activated by hepatitis C virus and inhibits viral replication through translational control. *Virus Res.* **2009**, *142*, 51–56. [CrossRef](https://doi.org/10.1016/j.virusres.2009.01.013) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19233225)

42. Jiang, D.; Guo, H.; Xu, C.; Chang, J.; Gu, B.; Wang, L.; Block, T.M.; Guo, J.T. Identification of three interferon-inducible cellular enzymes that inhibit the replication of hepatitis C virus. *J. Virol.* 2008, 82, 1665–1678. [CrossRef] [PubMed]

43. Park, I.H.; Baek, K.W.; Cho, E.Y.; Ahn, B.Y. PKR-dependent mechanisms of interferon-alpha for inhibiting hepatitis B virus replication. *Mol. Cells* 2011, 32, 167–172. [CrossRef] [PubMed]

44. Guidotti, L.G.; Morris, A.; Mendez, H.; Koch, R.; Silverman, R.H.; Williams, B.R.; Chisari, F.V. Interferon-regulated pathways that control hepatitis B virus replication in transgenic mice. *J. Virol.* 2002, 76, 2617–2621. [CrossRef] [PubMed]

45. Itsui, Y.; Sakamoto, N.; Kurosaki, M.; Kanazawa, N.; Tanabe, Y.; Koyama, T.; Takeda, Y.; Nakagawa, M.; Kakinuma, S.; Sekine, Y.; et al. Expressional screening of interferon-stimulated genes for antiviral activity against hepatitis C virus replication. *J. Viral. Hepat.* 2006, 13, 690–700. [CrossRef] [PubMed]

46. Meyer, K.; Kwon, Y.C.; Liu, S.; Hagedorn, C.H.; Ray, R.B.; Ray, R. Interferon-alpha inducible protein 6 impairs EGFR activation by CD81 and inhibits hepatitis C virus infection. *Sci. Rep.* 2015, 5, 9012. [CrossRef] [PubMed]

47. Zhou, X.; Michal, J.J.; Zhang, L.; Ding, B.; Lunney, J.K.; Liu, B.; Jiang, Z. Interferon induced IFIT family genes in host antiviral defense. *Int. J. Biol. Sci.* 2013, 9, 200–208. [CrossRef] [PubMed]

48. Metz, P.; Dazert, E.; Ruggieri, A.; Mazur, J.; Kaderali, L.; Kaul, A.; Zeuge, U.; Windisch, M.P.; Trippler, M.; Lohmann, V.; et al. Identification of type I and type II interferon-induced effectors controlling hepatitis C virus replication. *Hepatology* 2012, 56, 2082–2093. [CrossRef] [PubMed]

49. Raychoudhuri, A.; Shrivastava, S.; Steele, R.; Kim, H.; Ray, R.; Ray, R.B. ISG56 and IFITM1 proteins inhibit hepatitis C virus replication. *J. Virol.* 2011, 85, 12881–12889. [CrossRef] [PubMed]

50. Wilkins, C.; Woodward, J.; Lau, D.T.; Barnes, A.; Joyce, M.; McFarlane, N.; McKeating, J.A.; Tyrrell, D.L.; Gale, M., Jr. IFITM1 is a tight junction protein that inhibits hepatitis C virus entry. *Hepatology* 2013, 57, 461–469. [CrossRef] [PubMed]

51. Pei, R.; Qin, B.; Zhang, X.; Zhu, W.; Kemper, T.; Ma, Z.; Trippler, M.; Schlaak, J.; Chen, X.; Lu, M. Interferon-induced proteins with tetratricopeptide repeats 1 and 2 are cellular factors that limit hepatitis B virus replication. *J. Innate. Immun.* 2014, 6, 182–191. [CrossRef] [PubMed]

52. Yao, L.; Dong, H.; Zhu, H.; Nelson, D.; Liu, C.; Lambiasi, L.; Li, X. Identification of the IFITM3 gene as an inhibitor of hepatitis C viral translation in a stable stat1 cell line. *J. Viral. Hepat.* 2011, 18, e523–e529. [CrossRef] [PubMed]

53. Kanazawa, N.; Kurosaki, M.; Sakamoto, N.; Enomoto, N.; Itsui, Y.; Yamashiro, T.; Tanabe, Y.; Maekawa, S.; Nakagawa, M.; Chen, C.H.; et al. Regulation of hepatitis C virus replication by interferon regulatory factor 1. *J. Virol.* 2004, 78, 9713–9720. [CrossRef] [PubMed]

54. Bobardt, M.; Hopkins, S.; Baugh, J.; Chatterji, U.; Hernandez, F.; Hiscott, J.; Sluder, A.; Lin, K.; Gallay, P.A. HCV NS5a and IRF9 compete for cypa binding. *J. Hepatol.* 2013, 58, 16–23. [CrossRef] [PubMed]

55. Liu, Y.; Nie, H.; Mao, R.; Mitra, B.; Cai, D.; Yan, R.; Guo, J.T.; Block, T.M.; Mechti, N.; Guo, H. Interferon-inducible ribonuclease ISG20 inhibits hepatitis B virus replication through directly binding to the epsilon stem-loop structure of viral RNA. *PLoS Pathog.* 2017, 13, e1006296. [CrossRef] [PubMed]

56. Leong, C.R.; Funami, K.; Oshiumi, H.; Mengao, D.; Takaki, H.; Matsumoto, M.; Aly, H.H.; Watashi, K.; Chayama, K.; Seya, T. Interferon-stimulated gene of 20 kda protein (ISG20) degrades RNA of hepatitis B virus to impede the replication of HBV in vitro and in vivo. *Oncotarget* 2016, 7, 68179–68193. [CrossRef] [PubMed]

57. Du, X.; Pan, T.; Xu, J.; Zhang, Y.; Song, W.; Yi, Z.; Yuan, Z. Hepatitis C virus replicative double-stranded RNA is a potent interferon inducer that triggers interferon production through MDA5. *J. Gen. Virol.* 2016, 97, 2868–2882. [CrossRef] [PubMed]

58. Li, N.; Zhang, L.; Chen, L.; Feng, W.; Xu, Y.; Chen, F.; Liu, X.; Chen, Z.; Liu, W. Mxa inhibits hepatitis B virus replication by interaction with hepatitis B core antigen. *Hepatology* 2012, 56, 803–811. [CrossRef] [PubMed]

59. Shi, X.; Jiao, B.; Chen, Y.; Li, S.; Chen, L. Mxa is a positive regulator of type I ifn signaling in HCV infection. *J. Med. Virol.* 2017. [CrossRef] [PubMed]

60. Li, J.; Lin, S.; Chen, Q.; Peng, L.; Zhai, J.; Liu, Y.; Yuan, Z. Inhibition of hepatitis B virus replication by MYD88 involves accelerated degradation of pregenomic RNA and nuclear retention of pre-s/s RNAs. *J. Virol.* 2010, 84, 6387–6399. [CrossRef] [PubMed]

61. Chang, W.W.; Su, I.J.; Lai, M.D.; Chang, W.T.; Huang, W.; Lei, H.Y. The role of inducible nitric oxide synthase in a murine acute hepatitis B virus (HBV) infection model induced by hydrodynamics-based in vivo transfection of HBV-DNA. *J. Hepatol.* 2003, **39**, 834–842. [CrossRef]

62. Naganuma, A.; Nozaki, A.; Tanaka, T.; Sugiyama, K.; Takagi, H.; Mori, M.; Shimotohno, K.; Kato, N. Activation of the interferon-inducible 2′-5′-oligoadenylate synthetase gene by hepatitis C virus core protein. *J. Virol.* 2000, **74**, 8744–8750. [CrossRef] [PubMed]

63. Ishibashi, M.; Wakita, T.; Esumi, M. 2′,5′-oligoadenylate synthetase-like gene highly induced by hepatitis C virus infection in human liver is inhibitory to viral replication in vitro. *Biochem. Biophys. Res. Commun.* 2010, **392**, 397–402. [CrossRef] [PubMed]

64. Yang, J.; Zhu, X.; Liu, J.; Ding, X.; Han, M.; Hu, W.; Wang, X.; Zhou, Z.; Wang, S. Inhibition of hepatitis B virus replication by phospholipid scramblase 1 in vitro and in vivo. *Antiviral. Res.* 2012, **94**, 9–17. [CrossRef] [PubMed]

65. Dansako, H.; Ueda, Y.; Okumura, N.; Satoh, S.; Sugiyama, M.; Mizokami, M.; Ikeda, M.; Kato, N. The cyclic GMP-AMP synthetase-sting signaling pathway is required for both the innate immune response against HBV and the suppression of HBV assembly. *FEBS J.* 2016, **283**, 144–156. [CrossRef] [PubMed]

66. Collins, A.S.; Ahmed, S.; Napoletano, S.; Schroeder, M.; Johnston, J.A.; Hegarty, J.E.; O'Farrelly, C.; Stevenson, N.J. Hepatitis C virus (HCV)-induced suppressor of cytokine signaling (SOCS) 3 regulates proinflammatory TNF-alpha responses. *J. Leukoc. Biol.* 2014, **96**, 255–263. [CrossRef] [PubMed]

67. Shao, R.X.; Zhang, L.; Peng, L.F.; Sun, E.; Chung, W.J.; Jang, J.Y.; Tsai, W.L.; Hyppolite, G.; Chung, R.T. Suppressor of cytokine signaling 3 suppresses hepatitis C virus replication in an mTOR-dependent manner. *J. Virol.* 2010, **84**, 6060–6069. [CrossRef] [PubMed]

68. Belloni, L.; Allweiss, L.; Guerrieri, F.; Pediconi, N.; Volz, T.; Pollicino, T.; Petersen, J.; Raimondo, G.; Dandri, M.; Levrero, M. IFN-alpha inhibits HBV transcription and replication in cell culture and in humanized mice by targeting the epigenetic regulation of the nuclear cccDNA minichromosome. *J. Clin. Investig.* 2012, **122**, 529–537. [CrossRef] [PubMed]

69. Waris, G.; Turkson, J.; Hassanein, T.; Siddiqui, A. Hepatitis C virus (HCV) constitutively activates STAT-3 via oxidative stress: Role of STAT-3 in HCV replication. *J. Virol.* 2005, **79**, 1569–1580. [CrossRef] [PubMed]

70. Yamauchi, S.; Takeuchi, K.; Chihara, K.; Honjoh, C.; Kato, Y.; Yoshiki, H.; Hotta, H.; Sada, K. STAT1 is essential for the inhibition of hepatitis C virus replication by interferon-lambda but not by interferon-alpha. *Sci. Rep.* 2016, **6**, 38336. [CrossRef] [PubMed]

71. Yan, R.; Zhao, X.; Cai, D.; Liu, Y.; Block, T.M.; Guo, J.T.; Guo, H. The interferon-inducible protein tetherin inhibits hepatitis B virus virion secretion. *J. Virol.* 2015, **89**, 9200–9212. [CrossRef] [PubMed]

72. Amet, T.; Byrd, D.; Hu, N.; Sun, Q.; Li, F.; Zhao, Y.; Hu, S.; Grantham, A.; Yu, Q. Bst-2 expression in human hepatocytes is inducible by all three types of interferons and restricts production of hepatitis C virus. *Curr. Mol. Med.* 2014, **14**, 349–360. [CrossRef] [PubMed]

73. Dafa-Berger, A.; Kuzmina, A.; Fassler, M.; Yitzhak-Asraf, H.; Shemer-Avni, Y.; Taube, R. Modulation of hepatitis C virus release by the interferon-induced protein BST-2/tetherin. *Virology* 2012, **428**, 98–111. [CrossRef] [PubMed]

74. Wang, B.; Trippler, M.; Pei, R.; Lu, M.; Broering, R.; Gerken, G.; Schlaak, J.F. Toll-like receptor activated human and murine hepatic stellate cells are potent regulators of hepatitis C virus replication. *J. Hepatol.* 2009, **51**, 1037–1045. [CrossRef] [PubMed]

75. Thomas, E.; Gonzalez, V.D.; Li, Q.; Modi, A.A.; Chen, W.; Noureddin, M.; Rotman, Y.; Liang, T.J. HCV infection induces a unique hepatic innate immune response associated with robust production of type III interferons. *Gastroenterology* 2012, **142**, 978–988. [CrossRef] [PubMed]

76. Wu, J.; Huang, S.; Zhao, X.; Chen, M.; Lin, Y.; Xia, Y.; Sun, C.; Yang, X.; Wang, J.; Guo, Y.; et al. Poly(I:C) treatment leads to interferon-dependent clearance of hepatitis B virus in a hydrodynamic injection mouse model. *J. Virol.* 2014, **88**, 10421–10431. [CrossRef] [PubMed]

77. Negash, A.A.; Ramos, H.J.; Crochet, N.; Lau, D.T.; Doehle, B.; Papic, N.; Delker, D.A.; Jo, J.; Bertoletti, A.; Hagedorn, C.H.; et al. IL-1beta production through the NLRP3 inflammasome by hepatic macrophages links hepatitis C virus infection with liver inflammation and disease. *PLoS Pathog.* 2013, **9**, e1003330. [CrossRef] [PubMed]

78. Lee, J.; Tian, Y.; Chan, S.T.; Kim, J.Y.; Cho, C.; Ou, J.H. TNF-alpha induced by hepatitis C virus via TLR7 and TLR8 in hepatocytes supports interferon signaling via an autocrine mechanism. *PLoS Pathog.* 2015, **11**, e1004937. [CrossRef](https://doi.org/10.1371/journal.ppat.1004937) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26505858)

79. Isogawa, M.; Robek, M.D.; Furuichi, Y.; Chisari, F.V. Toll-like receptor signaling inhibits hepatitis B virus replication in vivo. *J. Virol.* 2005, **79**, 7269–7272. [CrossRef](https://doi.org/10.1128/JVI.79.11.7269-7272.2005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15890903)

80. Wang, S.; Chen, Y.; Li, C.; Wu, Y.; Guo, L.; Peng, C.; Huang, Y.; Cheng, G.; Qin, F.X. TRIM14 inhibits hepatitis C virus infection by spry domain-dependent targeted degradation of the viral NS5a protein. *Sci. Rep.* 2016, **6**, 32336. [CrossRef](https://doi.org/10.1038/srep32336) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27581526)

81. Gao, B.; Duan, Z.; Xu, W.; Xiong, S. Tripartite motif-containing 22 inhibits the activity of hepatitis B virus core promoter, which is dependent on nuclear-located ring domain. *Hepatology* 2009, **50**, 424–433. [CrossRef](https://doi.org/10.1002/hep.22956) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19551885)

82. Yang, C.; Zhao, X.; Sun, D.; Yang, L.; Chong, C.; Pan, Y.; Chi, X.; Gao, Y.; Wang, M.; Shi, X.; et al. Interferon alpha (IFNalpha)-induced TRIM22 interrupts HCV replication by ubiquitinating NS5a. *Cell Mol. Immunol.* 2016, **13**, 94–102. [CrossRef](https://doi.org/10.1038/cmi.2015.46) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26108588)

83. Zhang, S.; Guo, J.T.; Wu, J.Z.; Yang, G. Identification and characterization of multiple TRIM proteins that inhibit hepatitis B virus transcription. *PLoS ONE* 2013, **8**, e70001. [CrossRef](https://doi.org/10.1371/journal.pone.0070001) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23840660)

84. Helbig, K.J.; Eyre, N.S.; Yip, E.; Narayana, S.; Li, K.; Fiches, G.; McCartney, E.M.; Jangra, R.K.; Lemon, S.M.; Beard, M.R. The antiviral protein viperin inhibits hepatitis C virus replication via interaction with nonstructural protein 5a. *Hepatology* 2011, **54**, 1506–1517. [CrossRef](https://doi.org/10.1002/hep.24515) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21732280)

85. Mao, R.; Nie, H.; Cai, D.; Zhang, J.; Liu, H.; Yan, R.; Cuconati, A.; Block, T.M.; Guo, J.T.; Guo, H. Inhibition of hepatitis B virus replication by the host zinc finger antiviral protein. *PLoS Pathog.* 2013, **9**, e1003494. [CrossRef](https://doi.org/10.1371/journal.ppat.1003494) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23874212)

86. Chen, E.Q.; Dai, J.; Bai, L.; Tang, H. The efficacy of zinc finger antiviral protein against hepatitis B virus transcription and replication in transgenic mouse model. *Virol. J.* 2015, **12**, 25. [CrossRef](https://doi.org/10.1186/s12985-015-0268-5) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25637178)

87. Wieland, S.; Thimme, R.; Purcell, R.H.; Chisari, F.V. Genomic analysis of the host response to hepatitis B virus infection. *Proc. Natl. Acad. Sci. USA* 2004, **101**, 6669–6674. [CrossRef](https://doi.org/10.1073/pnas.0401910101) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15118104)

88. Lebosse, F.; Testoni, B.; Fresquet, J.; Facchetti, F.; Galmozzi, E.; Fournier, M.; Hervieu, V.; Berthillon, P.; Berby, F.; Bordes, I.; et al. Intrahepatic innate immune response pathways are downregulated in untreated chronic hepatitis B. *J. Hepatol.* 2017, **66**, 897–909. [CrossRef](https://doi.org/10.1016/j.jhep.2016.11.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27988198)

89. Luangsay, S.; Gruffaz, M.; Isorce, N.; Testoni, B.; Michelet, M.; Faure-Dupuy, S.; Maadadi, S.; Ait-Goughoulte, M.; Parent, R.; Rivoire, M.; et al. Early inhibition of hepatocyte innate responses by hepatitis B virus. *J. Hepatol.* 2015, **63**, 1314–1322. [CrossRef](https://doi.org/10.1016/j.jhep.2015.07.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26231878)

90. Cheng, X.; Xia, Y.; Serti, E.; Block, P.D.; Chung, M.; Chayama, K.; Rehermann, B.; Liang, T.J. Hepatitis B virus evades innate immunity of hepatocytes but activates cytokine production by macrophages. *Hepatology* 2017, **65**, 1088–1101. [CrossRef](https://doi.org/10.1002/hep.28998) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27780188)

91. Gordien, E.; Rosmorduc, O.; Peltekian, C.; Garreau, F.; Brechot, C.; Kremsdorf, D. Inhibition of hepatitis B virus replication by the interferon-inducible MXA protein. *J. Virol.* 2001, **75**, 2684–2691. [CrossRef](https://doi.org/10.1128/JVI.75.6.2684-2691.2001) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11238937)

92. Degasperi, E.; Vigano, M.; Aghemo, A.; Lampertico, P.; Colombo, M. PEGIFN-alpha2a for the treatment of chronic hepatitis B and C: A 10-year history. *Expert Rev. Anti-Infect. Ther.* 2013, **11**, 459–474. [CrossRef](https://doi.org/10.1586/eri.13.20) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23551154)

93. Trepo, C. A brief history of hepatitis milestones. *Liver Int.* 2014, **34** (Suppl. S1), 29–37. [CrossRef](https://doi.org/10.1111/liv.12440) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24933654)

94. Yoshio, S.; Kanto, T.; Kuroda, S.; Matsubara, T.; Higashitani, K.; Kakita, N.; Ishida, H.; Hiramatsu, N.; Nagano, H.; Sugiyama, M.; et al. Human blood dendritic cell antigen 3 (BDCA3)(+) dendritic cells are a potent producer of interferon-lambda in response to hepatitis C virus. *Hepatology* 2013, **57**, 1705–1715. [CrossRef](https://doi.org/10.1002/hep.26189) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23255336)

95. Suppiah, V.; Moldovan, M.; Ahlenstiel, G.; Berg, T.; Weltman, M.; Abate, M.L.; Bassendine, M.; Spengler, U.; Dore, G.J.; Powell, E.; et al. IL28b is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. *Nat. Genet.* 2009, **41**, 1100–1104. [CrossRef](https://doi.org/10.1038/ng.447) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759530)

96. Tanaka, Y.; Nishida, N.; Sugiyama, M.; Kurosaki, M.; Matsuura, K.; Sakamoto, N.; Nakagawa, M.; Korenaga, M.; Hino, K.; Hige, S.; et al. Genome-wide association of IL28b with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. *Nat. Genet.* 2009, **41**, 1105–1109. [CrossRef](https://doi.org/10.1038/ng.449) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759531)

97. Ge, D.; Fellay, J.; Thompson, A.J.; Simon, J.S.; Shianna, K.V.; Urban, T.J.; Heinzen, E.L.; Qiu, P.; Bertelsen, A.H.; Muir, A.J.; et al. Genetic variation in IL28b predicts hepatitis C treatment-induced viral clearance. *Nature* 2009, *461*, 399–401. [CrossRef](https://doi.org/10.1038/nature08481) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573)

98. Prokunina-Olsson, L.; Muchmore, B.; Tang, W.; Pfeiffer, R.M.; Park, H.; Dickensheets, H.; Hergott, D.; Porter-Gill, P.; Mumy, A.; Kohaar, I.; et al. A variant upstream of IFNL3 (IL28b) creating a new interferon gene IFNL4 is associated with impaired clearance of hepatitis C virus. *Nat. Genet.* 2013, *45*, 164–171. [CrossRef](https://doi.org/10.1038/ng.2521) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23302470)

99. Sun, J.C.; Lanier, L.L. NK cell development, homeostasis and function: Parallels with CD8(+) t cells. *Nat. Rev. Immunol.* 2011, *11*, 645–657. [CrossRef](https://doi.org/10.1038/nri3035) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21822276)

100. Parham, P.; Moffett, A. Variable NK cell receptors and their MHC class I ligands in immunity, reproduction and human evolution. *Nat. Rev. Immunol.* 2013, *13*, 133–144. [CrossRef](https://doi.org/10.1038/nri3370) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23416828)

101. Lanier, L.L. NK cell receptors. *Annu. Rev. Immunol.* 1998, *16*, 359–393. [CrossRef](https://doi.org/10.1146/annurev.immunol.16.1.359) [PubMed](https://pubmed.ncbi.nlm.nih.gov/9597134)

102. Lysakova-Devine, T.; O’Farrelly, C. Tissue-specific NK cell populations and their origin. *J. Leukoc. Biol.* 2014, *96*, 981–990. [CrossRef](https://doi.org/10.1189/jlb.0713418) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24907996)

103. Larkin, J.; Bost, A.; Glass, J.I.; Tan, S.L. Cytokine-activated natural killer cells exert direct killing of hepatoma cells harboring hepatitis C virus replicons. *J. Interferon Cytokine Res.* 2006, *26*, 854–865. [CrossRef](https://doi.org/10.1089/jir.2006.26.854) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17034107)

104. Yang, C.M.; Yoon, J.C.; Park, J.H.; Lee, J.M. Hepatitis C virus impairs natural killer cell activity via viral serine protease NS3. *PLoS ONE* 2017, *12*, e0175793. [CrossRef](https://doi.org/10.1371/journal.pone.0175793) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28441708)

105. Kafrouni, M.I.; Brown, G.R.; Thiele, D.L. Virally infected hepatocytes are resistant to perforin-dependent CTL effector mechanisms. *J. Immunol.* 2001, *167*, 1566–1574. [CrossRef](https://doi.org/10.4049/jimmunol.167.3.1566) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11459888)

106. Holder, K.A.; Russell, R.S.; Grant, M.D. Natural killer cell function and dysfunction in hepatitis C virus infection. *Biomed. Res. Int.* 2014, *2014*, 903764. [CrossRef](https://doi.org/10.1155/2014/903764) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25147728)

107. Kakimi, K.; Guidotti, L.G.; Koezuka, Y.; Chisari, F.V. Natural killer T cell activation inhibits hepatitis B virus replication in vivo. *J. Exp. Med.* 2000, *192*, 921–930. [CrossRef](https://doi.org/10.1084/jem.192.7.921) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11015438)

108. Dunn, C.; Brunetto, M.; Reynolds, G.; Christophides, T.; Kennedy, P.T.; Lampertico, P.; Das, A.; Lopes, A.R.; Borrow, P.; Williams, K.; et al. Cytokines induced during chronic hepatitis B virus infection promote a pathway for NK cell-mediated liver damage. *J. Exp. Med.* 2007, *204*, 667–680. [CrossRef](https://doi.org/10.1084/jem.20061893) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17339434)

109. Peppa, D.; Gill, U.S.; Reynolds, G.; Easom, N.J.; Pallett, L.J.; Schurich, A.; Micco, L.; Nebbia, G.; Singh, H.D.; Adams, D.H.; et al. Up-regulation of a death receptor renders antiviral T cells susceptible to NK cell-mediated deletion. *J. Exp. Med.* 2013, *210*, 99–114. [CrossRef](https://doi.org/10.1084/jem.20121448) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23288688)

110. Maini, M.K.; Peppa, D. NK cells: A double-edged sword in chronic hepatitis B virus infection. *Front. Immunol.* 2013, *4*, 57. [CrossRef](https://doi.org/10.3389/fimmu.2013.00057) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23596418)

111. Zheng, Q.; Zhu, Y.Y.; Chen, J.; Ye, Y.B.; Li, J.Y.; Liu, Y.R.; Hu, M.L.; Zheng, Y.C.; Jiang, J.J. Activated natural killer cells accelerate liver damage in patients with chronic hepatitis B virus infection. *Clin. Exp. Immunol.* 2015, *180*, 499–508. [CrossRef](https://doi.org/10.1111/cei.12583) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25752770)

112. Mizuhara, H.; O’Neill, E.; Seki, N.; Ogawa, T.; Kusunoki, C.; Otsuka, K.; Satoh, S.; Niwa, M.; Senoh, H.; Fujiwara, H. T cell activation-associated hepatic injury: Mediation by tumor necrosis factors and protection by interleukin 6. *J. Exp. Med.* 1994, *179*, 1529–1537. [CrossRef](https://doi.org/10.1084/jem.179.4.1529) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8183915)

113. Biermer, M.; Puro, R.; Schneider, R.J. Tumor necrosis factor alpha inhibition of hepatitis B virus replication involves disruption of capsid integrity through activation of NF-kappab. *J. Virol.* 2003, *77*, 4033–4042. [CrossRef](https://doi.org/10.1128/JVI.77.7.4033-4042.2003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12634358)

114. Godfrey, D.I.; MacDonald, H.R.; Kronenberg, M.; Smyth, M.J.; Van Kaer, L. NKT cells: What’s in a name? *Nat. Rev. Immunol.* 2004, *4*, 231–237. [CrossRef](https://doi.org/10.1038/nri1309) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15039760)

115. Zeissig, S.; Murata, K.; Sweet, L.; Publicover, J.; Hu, Z.; Kaser, A.; Bosse, E.; Iqbal, J.; Hussain, M.M.; Balschun, K.; et al. Hepatitis B virus-induced lipid alterations contribute to natural killer T cell-dependent protective immunity. *Nat. Med.* 2012, *18*, 1060–1068. [CrossRef](https://doi.org/10.1038/nm.2789) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22673022)

116. Krenkel, O.; Tacke, F. Liver macrophages in tissue homeostasis and disease. *Nat. Rev. Immunol.* 2017, *17*, 306–321. [CrossRef](https://doi.org/10.1038/nri.2017.18) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28345626)

117. Boltjes, A.; Movita, D.; Boonstra, A.; Woltman, A.M. The role of kupffer cells in hepatitis B and hepatitis C virus infections. *J. Hepatol.* 2014, *61*, 660–671. [CrossRef](https://doi.org/10.1016/j.jhep.2014.05.036) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24931382)

118. Burgio, V.L.; Ballardin, G.; Artini, M.; Caratozzolo, M.; Bianchi, F.B.; Levrero, M. Expression of co-stimulatory molecules by kupffer cells in chronic hepatitis of hepatitis C virus etiology. *Hepatology* 1998, *27*, 1600–1606. [CrossRef](https://doi.org/10.1002/hep.510270520) [PubMed](https://pubmed.ncbi.nlm.nih.gov/9570885)

119. Ebrahimkhani, M.R.; Mohar, I.; Crispe, I.N. Cross-presentation of antigen by diverse subsets of murine liver cells. *Hepatology* **2011**, *54*, 1379–1387. [CrossRef](https://doi.org/10.1002/hep.24160) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21374663)

120. Li, M.; Sun, R.; Xu, L.; Yin, W.; Chen, Y.; Zheng, X.; Lian, Z.; Wei, H.; Tian, Z. Kupffer cells support hepatitis B virus-mediated CD8+ T cell exhaustion via hepatitis b core antigen-TLR2 interactions in mice. *J. Immunol.* **2015**, *195*, 3100–3109. [CrossRef](https://doi.org/10.4049/jimmunol.1500016) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26130795)

121. Visvanathan, K.; Skinner, N.A.; Thompson, A.J.; Riordan, S.M.; Sozzi, V.; Edwards, R.; Rodgers, S.; Kurtovic, J.; Chang, J.; Lewin, S.; et al. Regulation of toll-like receptor-2 expression in chronic hepatitis B by the precore protein. *Hepatology* **2007**, *45*, 102–110. [CrossRef](https://doi.org/10.1002/hep.21463) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17187350)

122. Jia, B.; Guo, M.; Li, G.; Yu, D.; Zhang, X.; Lan, K.; Deng, Q. Hepatitis B virus core protein sensitizes hepatocytes to tumor necrosis factor-induced apoptosis by suppression of the phosphorylation of mitogen-activated protein kinase kinase 7. *J. Virol.* **2015**, *89*, 2041–2051. [CrossRef](https://doi.org/10.1128/JVI.02539-14) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25428798)

123. Goh, C.C.; Roggerson, K.M.; Lee, H.C.; Golden-Mason, L.; Rosen, H.R.; Hahn, Y.S. Hepatitis C virus-induced myeloid-derived suppressor cells suppress NK cell IFN-gamma production by altering cellular metabolism via arginase-1. *J. Immunol.* **2016**, *196*, 2283–2292. [CrossRef](https://doi.org/10.4049/jimmunol.1502018) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26921315)

124. Pallett, L.J.; Gill, U.S.; Quaglia, A.; Sinclair, L.V.; Jover-Cobos, M.; Schurich, A.; Singh, K.P.; Thomas, N.; Das, A.; Chen, A.; et al. Metabolic regulation of hepatitis B immunopathology by myeloid-derived suppressor cells. *Nat. Med.* **2015**, *21*, 591–600. [CrossRef](https://doi.org/10.1038/nm.3828) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25788097)

125. Steinman, R.M. The dendritic cell system and its role in immunogenicity. *Annu. Rev. Immunol.* **1991**, *9*, 271–296. [CrossRef](https://doi.org/10.1146/annurev.iy.09.040191.001415) [PubMed](https://pubmed.ncbi.nlm.nih.gov/1907458)

126. Heath, W.R.; Carbone, F.R. Dendritic cell subsets in primary and secondary T cell responses at body surfaces. *Nat. Immunol.* **2009**, *10*, 1237–1244. [CrossRef](https://doi.org/10.1038/ni.1765) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749750)

127. Kadowaki, N. The divergence and interplay between PDC and MDC in humans. *Front. Biosci.* **2009**, *14*, 808–817. [CrossRef](https://doi.org/10.2741/3296)

128. Schildberg, F.A.; Hegenbarth, S.I.; Schumak, B.; Scholz, K.; Limmer, A.; Knolle, P.A. Liver sinusoidal endothelial cells veto CD8 T cell activation by antigen-presenting dendritic cells. *Eur. J. Immunol.* **2008**, *38*, 957–967. [CrossRef](https://doi.org/10.1002/eji.200737826) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18338500)

129. Velazquez, V.M.; Hon, H.; Ibegbu, C.; Knechtle, S.J.; Kirk, A.D.; Grakoui, A. Hepatic enrichment and activation of myeloid dendritic cells during chronic hepatitis C virus infection. *Hepatology* **2012**, *56*, 2071–2081. [CrossRef](https://doi.org/10.1002/hep.25865) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22714586)

130. Jan, R.H.; Lin, Y.L.; Chen, C.J.; Lin, T.Y.; Hsu, Y.C.; Chen, I.K.; Chiang, B.L. Hepatitis B virus surface antigen can activate human monocyte-derived dendritic cells by nuclear factor kappa b and p38 mitogen-activated protein kinase mediated signaling. *Microbiol. Immunol.* **2012**, *56*, 719–727. [CrossRef](https://doi.org/10.1111/j.1348-0421.2012.00471.x) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22905963)

131. Leone, P.; Di Tacchio, M.; Berardi, S.; Santantonio, T.; Fasano, M.; Ferrone, S.; Vacca, A.; Dammacco, F.; Racanelli, V. Dendritic cell maturation in HCV infection: Altered regulation of MHC class I antigen processing-presenting machinery. *J. Hepatol.* **2014**, *61*, 242–251. [CrossRef](https://doi.org/10.1016/j.jhep.2014.03.033) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24768893)

132. Zhou, Y.; Lukes, Y.; Anderson, J.; Fileta, B.; Reinhardt, B.; Sjogren, M. Hepatitis C virus E2 envelope protein induces dendritic cell maturation. *J. Viral. Hepat.* **2007**, *14*, 849–858. [CrossRef](https://doi.org/10.1111/j.1365-2893.2007.00850.x) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17635570)

133. Sarobe, P.; Lasarte, J.J.; Zabaleta, A.; Arribillaga, L.; Arina, A.; Melero, I.; Borras-Cuesta, F.; Prieto, J. Hepatitis C virus structural proteins impair dendritic cell maturation and inhibit in vivo induction of cellular immune responses. *J. Virol.* **2003**, *77*, 10862–10871. [CrossRef](https://doi.org/10.1128/JVI.77.20.10862-10871.2003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/14512550)

134. Saito, K.; Ait-Goughoulte, M.; Truscott, S.M.; Meyer, K.; Blazevic, A.; Abate, G.; Ray, R.B.; Hoft, D.F.; Ray, R. Hepatitis C virus inhibits cell surface expression of HLA-DR, prevents dendritic cell maturation, and induces interleukin-10 production. *J. Virol.* **2008**, *82*, 3320–3328. [CrossRef](https://doi.org/10.1128/JVI.02155-07) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18216158)

135. Vincent, I.E.; Zannetti, C.; Lucifora, J.; Norder, H.; Protzer, U.; Hainaut, P.; Zoulim, F.; Tommasino, M.; Trepo, C.; Hasan, U.; et al. Hepatitis B virus impairs TLR9 expression and function in plasmacytoid dendritic cells. *PLoS ONE* **2011**, *6*, e26315. [CrossRef](https://doi.org/10.1371/journal.pone.0026315) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22046342)

136. Dreux, M.; Garaigorta, U.; Boyd, B.; Decembre, E.; Chung, J.; Whitten-Bauer, C.; Wieland, S.; Chisari, F.V. Short-range exosomal transfer of viral RNA from infected cells to plasmacytoid dendritic cells triggers innate immunity. *Cell Host Microbe.* **2012**, *12*, 558–570. [CrossRef](https://doi.org/10.1016/j.chom.2012.09.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23040536)

137. Li, J.; Liu, K.; Liu, Y.; Xu, Y.; Zhang, F.; Yang, H.; Liu, J.; Pan, T.; Chen, J.; Wu, M.; et al. Exosomes mediate the cell-to-cell transmission of IFN-alpha-induced antiviral activity. *Nat. Immunol.* **2013**, *14*, 793–803. [CrossRef](https://doi.org/10.1038/ni.2615) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23749196)

Vaccines 2017, 5, 24

138. Zhu, X.; He, Z.; Yuan, J.; Wen, W.; Huang, X.; Hu, Y.; Lin, C.; Pan, J.; Li, R.; Deng, H.; et al. IFITM3-containing exosome as a novel mediator for anti-viral response in dengue virus infection. *Cell Microbiol.* 2015, 17, 105–118. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

139. Yang, Y.; Han, Q.; Hou, Z.; Zhang, C.; Tian, Z.; Zhang, J. Exosomes mediate hepatitis B virus (HBV) transmission and NK-cell dysfunction. *Cell Mol. Immunol.* 2017, 14, 465–475. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

140. Longatti, A.; Boyd, B.; Chisari, F.V. Virion-independent transfer of replication-competent hepatitis C virus RNA between permissive cells. *J. Virol.* 2015, 89, 2956–2961. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

141. Cosset, F.L.; Dreux, M. HCV transmission by hepatic exosomes establishes a productive infection. *J. Hepatol.* 2014, 60, 674–675. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

142. Bukong, T.N.; Momen-Heravi, F.; Kodys, K.; Bala, S.; Szabo, G. Exosomes from hepatitis C infected patients transmit HCV infection and contain replication competent viral RNA in complex with ago2-miR122-HSP90. *PLoS Pathog.* 2014, 10, e1004424. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

143. Zhou, Y.; Wang, X.; Sun, L.; Zhou, L.; Ma, T.C.; Song, L.; Wu, J.G.; Li, J.L.; Ho, W.Z. Toll-like receptor 3-activated macrophages confer anti-HCV activity to hepatocytes through exosomes. *FASEB J.* 2016, 30, 4132–4140. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

144. Bridgeman, A.; Maelfait, J.; Davenport, T.; Partridge, T.; Peng, Y.; Mayer, A.; Dong, T.; Kaever, V.; Borrow, P.; Rehwinkel, J. Viruses transfer the antiviral second messenger cGAMP between cells. *Science* 2015, 349, 1228–1232. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

145. McKinney, E.F.; Smith, K.G. T cell exhaustion and immune-mediated disease—the potential for therapeutic exhaustion. *Curr. Opin. Immunol.* 2016, 43, 74–80. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

146. Bengsch, B.; Martin, B.; Thimme, R. Restoration of HBV-specific CD8+ T cell function by PD-1 blockade in inactive carrier patients is linked to T cell differentiation. *J. Hepatol.* 2014, 61, 1212–1219. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

147. Tzeng, H.T.; Tsai, H.F.; Liao, H.J.; Lin, Y.J.; Chen, L.; Chen, P.J.; Hsu, P.N. PD-1 blockage reverses immune dysfunction and hepatitis B viral persistence in a mouse animal model. *PLoS ONE* 2012, 7, e39179. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

148. Fisicaro, P.; Valdatta, C.; Massari, M.; Loggi, E.; Biasini, E.; Sacchelli, L.; Cavallo, M.C.; Silini, E.M.; Andreone, P.; Missale, G.; et al. Antiviral intrahepatic T-cell responses can be restored by blocking programmed death-1 pathway in chronic hepatitis B. *Gastroenterology* 2010. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

149. Maier, H.; Isogawa, M.; Freeman, G.J.; Chisari, F.V. PD-1:PD-L1 interactions contribute to the functional suppression of virus-specific CD8+ T lymphocytes in the liver. *J. Immunol.* 2007, 178, 2714–2720. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

150. Franzese, O.; Kennedy, P.T.; Gehring, A.J.; Gotto, J.; Williams, R.; Maini, M.K.; Bertoletti, A. Modulation of the CD8+-T-cell response by CD4+ CD25+ regulatory T cells in patients with hepatitis B virus infection. *J. Virol.* 2005, 79, 3322–3328. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

151. Kondo, Y.; Kobayashi, K.; Ueno, Y.; Shiina, M.; Niitsuma, H.; Kanno, N.; Kobayashi, T.; Shimosegawa, T. Mechanism of T cell hyporesponsiveness to hbcag is associated with regulatory T cells in chronic hepatitis B. *World J. Gastroenterol.* 2006, 12, 4310–4317. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

152. Stoop, J.N.; van der Molen, R.G.; Baan, C.C.; van der Laan, L.J.; Kuipers, E.J.; Kusters, J.G.; Janssen, H.L. Regulatory T cells contribute to the impaired immune response in patients with chronic hepatitis B virus infection. *Hepatology* 2005, 41, 771–778. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

153. Stoop, J.N.; van der Molen, R.G.; Kuipers, E.J.; Kusters, J.G.; Janssen, H.L. Inhibition of viral replication reduces regulatory T cells and enhances the antiviral immune response in chronic hepatitis B. *Virology* 2007, 361, 141–148. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

154. Kondo, Y.; Ueno, Y.; Kobayashi, K.; Kakazu, E.; Shiina, M.; Inoue, J.; Tamai, K.; Wakui, Y.; Tanaka, Y.; Ninomiya, M.; et al. Hepatitis B virus replication could enhance regulatory T cell activity by producing soluble heat shock protein 60 from hepatocytes. *J. Infect. Dis.* 2010, 202, 202–213. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

155. Sugimoto, K.; Ikeda, F.; Stadanlick, J.; Nunes, F.A.; Alter, H.J.; Chang, K.M. Suppression of HCV-specific t cells without differential hierarchy demonstrated ex vivo in persistent HCV infection. *Hepatology* 2003, 38, 1437–1448. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

156. Pallett, L.J.; Davies, J.; Colbeck, E.J.; Robertson, F.; Hansi, N.; Easom, N.J.W.; Burton, A.R.; Stegmann, K.A.; Schurich, A.; Swadling, L.; et al. IL-2high tissue-resident T cells in the human liver: Sentinels for hepatotropic infection. *J. Exp. Med.* 2017, 214, 1567–1580. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

157. Giang, E.; Dorner, M.; Prentoe, J.C.; Dreux, M.; Evans, M.J.; Bukh, J.; Rice, C.M.; Ploss, A.; Burton, D.R.; Law, M. Human broadly neutralizing antibodies to the envelope glycoprotein complex of hepatitis C virus. *Proc. Natl. Acad. Sci. USA* **2012**, *109*, 6205–6210. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

158. De Jong, Y.P.; Dorner, M.; Mommersteeg, M.C.; Xiao, J.W.; Balazs, A.B.; Robbins, J.B.; Winer, B.Y.; Gerges, S.; Vega, K.; Labitt, R.N.; et al. Broadly neutralizing antibodies abrogate established hepatitis C virus infection. *Sci. Transl. Med.* **2014**. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

159. Williams, I.T.; Goldstein, S.T.; Tufa, J.; Tauillii, S.; Margolis, H.S.; Mahoney, F.J. Long term antibody response to hepatitis B vaccination beginning at birth and to subsequent booster vaccination. *Pediatr. Infect. Dis. J.* **2003**, *22*, 157–163. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

160. Hudu, S.A.; Malik, Y.A.; Niazlin, M.T.; Harmal, N.S.; Adnan, A.; Alshrari, A.S.; Sekawi, Z. Antibody and immune memory persistence post infant hepatitis B vaccination. *Patient Preference Adherence* **2013**, *7*, 981–986. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

161. Ciupe, S.M.; Ribeiro, R.M.; Perelson, A.S. Antibody responses during hepatitis B viral infection. *PLoS Comput. Biol.* **2014**, *10*, e1003730. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

162. Wang, Y.; Li, J.; Wang, X.; Zhou, Y.; Zhang, T.; Ho, W. HCV dsRNA-activated macrophages inhibit HCV replication in hepatocytes. *Hepat. Mon.* **2015**, *15*, e29282. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

163. Cao, X.; Ding, Q.; Lu, J.; Tao, W.; Huang, B.; Zhao, Y.; Niu, J.; Liu, Y.J.; Zhong, J. MDA5 plays a critical role in interferon response during hepatitis C virus infection. *J. Hepatol.* **2015**, *62*, 771–778. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

164. Li, K.; Foy, E.; Ferreon, J.C.; Nakamura, M.; Ferreon, A.C.; Ikeda, M.; Ray, S.C.; Gale, M., Jr.; Lemon, S.M. Immune evasion by hepatitis C virus NS3/4a protease-mediated cleavage of the toll-like receptor 3 adaptor protein TRIF. *Proc. Natl. Acad. Sci. USA* **2005**, *102*, 2992–2997. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

165. Li, X.D.; Sun, L.; Seth, R.B.; Pineda, G.; Chen, Z.J. Hepatitis C virus protease NS3/4a cleaves mitochondrial antiviral signaling protein off the mitochondria to evade innate immunity. *Proc. Natl. Acad. Sci. USA* **2005**, *102*, 17717–17722. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

166. Okamoto, M.; Oshiumi, H.; Azuma, M.; Kato, N.; Matsumoto, M.; Seya, T. IPS-1 is essential for type III IFN production by hepatocytes and dendritic cells in response to hepatitis C virus infection. *J. Immunol.* **2014**, *192*, 2770–2777. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

167. Yao, Z.Q.; Waggoner, S.N.; Cruise, M.W.; Hall, C.; Xie, X.; Oldach, D.W.; Hahn, Y.S. SOCS1 and SOCS3 are targeted by hepatitis C virus core/GC1qr ligation to inhibit T-cell function. *J. Virol.* **2005**, *79*, 15417–15429. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

168. Ding, Q.; Cao, X.; Lu, J.; Huang, B.; Liu, Y.J.; Kato, N.; Shu, H.B.; Zhong, J. Hepatitis C virus NS4b blocks the interaction of sting and TBK1 to evade host innate immunity. *J. Hepatol.* **2013**, *59*, 52–58. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

169. Neufeldt, C.J.; Joyce, M.A.; Van Buuren, N.; Levin, A.; Kirkegaard, K.; Gale, M., Jr.; Tyrrell, D.L.; Wozniak, R.W. The hepatitis C virus-induced membranous web and associated nuclear transport machinery limit access of pattern recognition receptors to viral replication sites. *PLoS Pathog.* **2016**, *12*, e1005428. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

170. Han, J.Q.; Barton, D.J. Activation and evasion of the antiviral 2′-5′ oligoadenylate synthetase/ribonuclease l pathway by hepatitis C virus mRNA. *RNA* **2002**, *8*, 512–525. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

171. Tsutsumi, T.; Okushin, K.; Enooku, K.; Fujinaga, H.; Moriya, K.; Yotsuyanagi, H.; Aizaki, H.; Suzuki, T.; Matsuura, Y.; Koike, K. Nonstructural 5a protein of hepatitis C virus interferes with toll-like receptor signaling and suppresses the interferon response in mouse liver. *PLoS ONE* **2017**, *12*, e0170461. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

172. Taylor, D.R.; Shi, S.T.; Romano, P.R.; Barber, G.N.; Lai, M.M. Inhibition of the interferon-inducible protein kinase PKR by HCV E2 protein. *Science* **1999**, *285*, 107–110. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

173. Pierce, B.G.; Keck, Z.Y.; Fung, S.K. Viral evasion and challenges of hepatitis C virus vaccine development. *Curr. Opin. Virol.* **2016**, *20*, 55–63. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

174. Echeverria, N.; Moratorio, G.; Cristina, J.; Moreno, P. Hepatitis C virus genetic variability and evolution. *World J. Hepatol.* **2015**, *7*, 831–845. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

175. Sagnelli, E.; Argentini, C.; Genovese, D.; Pisaturo, M.; Coppola, N.; Taffon, S.; Sagnelli, C.; Rapicetta, M. Virological and epitope evolution of HCV infection from acute hepatitis C to subsequent episodes of HCV-related acute liver cell necrosis. *Infection* **2009**, *37*, 344–348. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

176. Neveu, B.; Debeaupuis, E.; Echasserieau, K.; le Moullac-Vaidye, B.; Gassin, M.; Jegou, L.; Decalf, J.; Albert, M.; Ferry, N.; Gournay, J.; et al. Selection of high-avidity CD8 T cells correlates with control of hepatitis C virus infection. *Hepatology* **2008**, *48*, 713–722. [CrossRef](https://doi.org/10.1002/hep.22059) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18311898)

177. Ohno, S.; Moriya, O.; Yoshimoto, T.; Hayashi, H.; Akatsuka, T.; Matsui, M. Immunogenic variation between multiple HLA-A*0201-restricted, hepatitis C virus-derived epitopes for cytotoxic T lymphocytes. *Viral. Immunol.* **2006**, *19*, 458–467. [CrossRef](https://doi.org/10.1089/vim.2006.19.458) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17140110)

178. Gondar, V.; Molina-Jimenez, F.; Hishiki, T.; Garcia-Buey, L.; Koutsoudakis, G.; Shimotohno, K.; Benedicto, I.; Majano, P.L. Apolipoprotein E, but not apolipoprotein B, is essential for efficient cell-to-cell transmission of hepatitis C virus. *J. Virol.* **2015**, *89*, 9962–9973. [CrossRef](https://doi.org/10.1128/JVI.01095-15) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26150420)

179. Pantua, H.; Diao, J.; Ultsch, M.; Hazen, M.; Mathieu, M.; McCutcheon, K.; Takeda, K.; Date, S.; Cheung, T.K.; Phung, Q.; et al. Glycan shifting on hepatitis C virus (HCV) E2 glycoprotein is a mechanism for escape from broadly neutralizing antibodies. *J. Mol. Biol.* **2013**, *425*, 1899–1914. [CrossRef](https://doi.org/10.1016/j.jmb.2013.02.014) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23454288)

180. Bankwitz, D.; Doepke, M.; Hueging, K.; Weller, R.; Bruening, J.; Behrendt, P.; Lee, J.Y.; Vondran, F.W.R.; Manns, M.P.; Bartenschlager, R.; et al. Maturation of secreted HCV particles by incorporation of secreted apoe protects from antibodies by enhancing infectivity. *J. Hepatol.* **2017**, *66*, 1035–1045. [CrossRef](https://doi.org/10.1016/j.jhep.2017.01.011) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28216373)

181. Khan, M.; Syed, G.H.; Kim, S.J.; Siddiqui, A. Hepatitis B virus-induced parkin-dependent recruitment of linear ubiquitin assembly complex (LUBAC) to mitochondria and attenuation of innate immunity. *PLoS Pathog.* **2016**, *12*, e1005693. [CrossRef](https://doi.org/10.1371/journal.ppat.1005693) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27145200)

182. Wang, S.; Chen, Z.; Hu, C.; Qian, F.; Cheng, Y.; Wu, M.; Shi, B.; Chen, J.; Hu, Y.; Yuan, Z. Hepatitis B virus surface antigen selectively inhibits TLR2 ligand-induced IL-12 production in monocytes/macrophages by interfering with JNK activation. *J. Immunol.* **2013**, *190*, 5142–5151. [CrossRef](https://doi.org/10.4049/jimmunol.1202848) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23547203)

183. Chen, J.; Wu, M.; Zhang, X.; Zhang, W.; Zhang, Z.; Chen, L.; He, J.; Zheng, Y.; Chen, C.; Wang, F.; et al. Hepatitis B virus polymerase impairs interferon-alpha-induced STAT activation through inhibition of importin-alpha5 and protein kinase C-delta. *Hepatology* **2013**, *57*, 470–482. [CrossRef](https://doi.org/10.1002/hep.26099) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22991182)

184. Liu, S.; Peng, N.; Xie, J.; Hao, Q.; Zhang, M.; Zhang, Y.; Xia, Z.; Xu, G.; Zhao, F.; Wang, Q.; et al. Human hepatitis B virus surface and E antigens inhibit major vault protein signaling in interferon induction pathways. *J. Hepatol.* **2015**, *62*, 1015–1023. [CrossRef](https://doi.org/10.1016/j.jhep.2014.11.018) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25460326)

185. Miyakawa, K.; Matsunaga, S.; Watashi, K.; Sugiyama, M.; Kimura, H.; Yamamoto, N.; Mizokami, M.; Wakita, T.; Ryo, A. Molecular dissection of HBV evasion from restriction factor tetherin: A new perspective for antiviral cell therapy. *Oncotarget* **2015**, *6*, 21840–21852. [CrossRef](https://doi.org/10.18632/oncotarget.4388) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26158256)

186. Wang, H.; Ryu, W.S. Hepatitis B virus polymerase blocks pattern recognition receptor signaling via interaction with DDX3: Implications for immune evasion. *PLoS Pathog.* **2010**, *6*, e1000986. [CrossRef](https://doi.org/10.1371/journal.ppat.1000986) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20628558)

187. Liu, Y.; Li, J.; Chen, J.; Li, Y.; Wang, W.; Du, X.; Song, W.; Zhang, W.; Lin, L.; Yuan, Z. Hepatitis B virus polymerase disrupts k63-linked ubiquitination of sting to block innate cytosolic DNA-sensing pathways. *J. Virol.* **2015**, *89*, 2287–2300. [CrossRef](https://doi.org/10.1128/JVI.02945-14) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25480786)

188. Liu, D.; Wu, A.; Cui, L.; Hao, R.; Wang, Y.; He, J.; Guo, D. Hepatitis B virus polymerase suppresses nf-kappab signaling by inhibiting the activity of IKKs via interaction with HSP90beta. *PLoS ONE* **2014**, *9*, e91658.

189. Wan, Y.; Cao, W.; Han, T.; Ren, S.; Feng, J.; Chen, T.; Wang, J.; Broering, R.; Lu, M.; Zhu, Y. Inducible rubicon facilitates viral replication by antagonizing interferon production. *Cell Mol. Immunol.* **2017**, *14*, 607–620. [CrossRef](https://doi.org/10.1038/cmi.2016.10) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27558405)

190. Jiang, J.; Tang, H. Mechanism of inhibiting type i interferon induction by hepatitis B virus x protein. *Protein Cell* **2010**, *1*, 1106–1117. [CrossRef](https://doi.org/10.1007/s13238-010-0099-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21125030)

191. Niu, C.; Livingston, C.M.; Li, L.; Beran, R.K.; Daffis, S.; Ramakrishnan, D.; Burdette, D.; Peiser, L.; Salas, E.; Ramos, H.; et al. The SMC5/6 complex restricts hbv when localized to ND10 without inducing an innate immune response and is counteracted by the HBV x protein shortly after infection. *PLoS ONE* **2017**, *12*, e0169648. [CrossRef](https://doi.org/10.1371/journal.pone.0169648) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28125727)

192. Lim, K.H.; Park, E.S.; Kim, D.H.; Cho, K.C.; Kim, K.P.; Park, Y.K.; Ahn, S.H.; Park, S.H.; Kim, K.H.; Kim, C.W.; et al. Suppression of interferon-mediated anti-HBV response by single CpG methylation in the 5′-UTR of TRIM22. *Gut* **2017**, *66*, 1035–1045. [CrossRef](https://doi.org/10.1136/gutjnl-2016-312127) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27256722)

193. Fernandez, M.; Quiroga, J.A.; Carreno, V. Hepatitis B virus downregulates the human interferon-inducible mxa promoter through direct interaction of precore/core proteins. *J. Gen. Virol.* **2003**, *84*, 2073–2082. [CrossRef](https://doi.org/10.1099/vir.0.19211-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12810858)

194. Seeger, C.; Mason, W.S. Molecular biology of hepatitis B virus infection. *Virology* **2015**, *479–480*, 672–686. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

195. Bock, C.T.; Schranz, P.; Schroder, C.H.; Zentgraf, H. Hepatitis B virus genome is organized into nucleosomes in the nucleus of the infected cell. *Virus Genes* **1994**, *8*, 215–229. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

196. Locarnini, S.; Zoulim, F. Molecular genetics of HBV infection. *Antivir. Ther.* **2010**, *15* (Suppl. S3), 3–14. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

197. Guo, Y.H.; Li, Y.N.; Zhao, J.R.; Zhang, J.; Yan, Z. HBC binds to the CpG islands of HBV cccDNA and promotes an epigenetic permissive state. *Epigenetics* **2011**, *6*, 720–726. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

198. Belloni, L.; Pollicino, T.; De Nicola, F.; Guerrieri, F.; Raffa, G.; Fanciulli, M.; Raimondo, G.; Levrero, M. Nuclear HBX binds the HBV minichromosome and modifies the epigenetic regulation of cccDNA function. *Proc. Natl. Acad. Sci. USA* **2009**, *106*, 19975–19979. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

199. Riviere, L.; Gerossier, L.; Ducroux, A.; Dion, S.; Deng, Q.; Michel, M.L.; Buendia, M.A.; Hantz, O.; Neveut, C. Hbx relieves chromatin-mediated transcriptional repression of hepatitis B viral cccdna involving setdb1 histone methyltransferase. *J. Hepatol.* **2015**, *63*, 1093–1102. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

200. Meier, M.A.; Suslov, A.; Ketterer, S.; Heim, M.H.; Wieland, S.F. Hepatitis B virus covalently closed circular DNA homeostasis is independent of the lymphotoxin pathway during chronic HBV infection. *J. Viral. Hepat.* **2017**. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

201. Bouzidi, M.S.; Caval, V.; Suspene, R.; Hallez, C.; Pineau, P.; Wain-Hobson, S.; Vartanian, J.P. APOBEC3de antagonizes hepatitis B virus restriction factors APOBEC3f and APOBEC3g. *J. Mol. Biol.* **2016**, *428*, 3514–3528. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

202. Vartanian, J.P.; Henry, M.; Marchio, A.; Suspene, R.; Aynaud, M.M.; Guetard, D.; Cervantes-Gonzalez, M.; Battiston, C.; Mazzaferro, V.; Pineau, P.; et al. Massive APOBEC3 editing of hepatitis B viral DNA in cirrhosis. *PLoS Pathog.* **2010**, *6*, e1000928. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

203. Kock, J.; Blum, H.E. Hypermutation of hepatitis B virus genomes by APOBEC3g, APOBEC3c and APOBEC3h. *J. Gen. Virol.* **2008**, *89*, 1184–1191. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

204. Baumert, T.F.; Rosler, C.; Malim, M.H.; von Weizsacker, F. Hepatitis B virus DNA is subject to extensive editing by the human deaminase APOBEC3c. *Hepatology* **2007**, *46*, 682–689. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

205. Nguyen, D.H.; Gummuluru, S.; Hu, J. Deamination-independent inhibition of hepatitis B virus reverse transcription by APOBEC3g. *J. Virol.* **2007**, *81*, 4465–4472. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

206. Cooper, A.; Tal, G.; Lider, O.; Shaul, Y. Cytokine induction by the hepatitis B virus capsid in macrophages is facilitated by membrane heparan sulfate and involves TLR2. *J. Immunol.* **2005**, *175*, 3165–3176. [CrossRef](https://doi.org/CrossRef)

207. Purvina, M.; Hoste, A.; Rossignol, J.M.; Lagaudriere-Gesbert, C. Human hepatitis B viral e antigen and its precursor p20 inhibit T lymphocyte proliferation. *Biochem. Biophys. Res. Commun.* **2012**, *417*, 1310–1315. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

208. Tian, Y.; Kuo, C.F.; Akbari, O.; Ou, J.H. Maternal-derived hepatitis B virus E antigen alters macrophage function in offspring to drive viral persistence after vertical transmission. *Immunity* **2016**, *44*, 1204–1214. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

209. Peppa, D.; Micco, L.; Javaid, A.; Kennedy, P.T.; Schurich, A.; Dunn, C.; Pallant, C.; Ellis, G.; Khanna, P.; Dusheiko, G.; et al. Blockade of immunosuppressive cytokines restores NK cell antiviral function in chronic hepatitis B virus infection. *PLoS Pathog.* **2010**, *6*, e1001227. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

210. Tjwa, E.T.; van Oord, G.W.; Hegmans, J.P.; Janssen, H.L.; Woltman, A.M. Viral load reduction improves activation and function of natural killer cells in patients with chronic hepatitis B. *J. Hepatol.* **2011**, *54*, 209–218. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

211. Oliviero, B.; Varchetta, S.; Paudice, E.; Michelone, G.; Zaramella, M.; Mavilio, D.; De Filippi, F.; Bruno, S.; Mondelli, M.U. Natural killer cell functional dichotomy in chronic hepatitis B and chronic hepatitis C virus infections. *Gastroenterology* **2009**. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

212. Ju, Y.; Hou, N.; Meng, J.; Wang, X.; Zhang, D.; Liu, Y.; Zhu, F.; Zhang, L.; Sun, W.; et al. T cell immunoglobulin- and mucin-domain-containing molecule-3 (TIM-3) mediates natural killer cell suppression in chronic hepatitis B. *J. Hepatol.* **2010**, *52*, 322–329. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

213. Bonorino, P.; Ramzan, M.; Camous, X.; Dufeu-Duchesne, T.; Thelu, M.A.; Sturm, N.; Dariz, A.; Guillermet, C.; Pernollet, M.; Zarski, J.P.; et al. Fine characterization of intrahepatic NK cells expressing natural killer receptors in chronic hepatitis B and C. *J. Hepatol.* **2009**, *51*, 458–467. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

214. Tong, S.; Li, J.; Wands, J.R.; Wen, Y.M. Hepatitis B virus genetic variants: Biological properties and clinical implications. *Emerg. Microbes. Infect.* 2013, 2, e10. [CrossRef](https://doi.org/10.7717/peerj.10) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

215. Shouval, D. Hepatitis B vaccines. *J. Hepatol.* 2003, 39 (Suppl. 1), S70–S76. [CrossRef](https://doi.org/10.1016/S0168-8278(03)00116-0)

216. Poovorawan, Y.; Theamboonlers, A.; Sanpavat, S.; Chumdermpadetsuk, S.; Safary, A.; Vandepapeliere, P. Long-term antibody persistence after booster vaccination with combined tetravalent diphtheria tetanus, whole-cell bordetella pertussis and hepatitis B vaccine in healthy infants. *Ann. Trop. Paediatr.* 1997, 17, 301–308. [CrossRef](https://doi.org/10.1179/027249397X00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000