
Molecular Mechanisms Used by Salmonella to Evade the Immune System

Joaquín Bernal-Bayard${}^{1*}$ and Francisco Ramos-Morales${}^{2}$

${}^{1}$Département de Microbiologie, Institut Pasteur, Paris, France.  
${}^{2}$Departamento de Genética, Universidad de Sevilla, Seville, Spain.

*Correspondence: jbernal@pasteur.fr  

https://doi.org/10.21775/cimb.025.133

---

### Abstract

Human and animal pathogens are able to circumvent, at least temporarily, the sophisticated immune defences of their hosts. Several serovars of the Gram-negative bacterium *Salmonella enterica* have been used as models for the study of pathogen–host interactions. In this review we discuss the strategies used by *Salmonella* to evade or manipulate three levels of host immune defences: physical barriers, innate immunity and adaptive immunity. During its passage through the digestive system, *Salmonella* has to face the acidic pH of the stomach, bile and antimicrobial peptides in the intestine, as well as the competition with resident microbiota. After host cell invasion, *Salmonella* manipulates inflammatory pathways and the autophagy process. Finally, *Salmonella* evades the adaptive immune system by interacting with dendritic cells, and T and B lymphocytes. Mechanisms allowing the establishment of persistent infections are also discussed.

---

### Introduction

*Salmonella* spp. are pathogenic Gram-negative, rod-shaped, motile bacteria belonging to the family Enterobacteriaceae. The genus *Salmonella* includes the species *S. bongori* and *S. enterica*. *S. enterica* is further divided into several subspecies and more than 2500 serovars (Tindall et al., 2005). *Salmonella* can cause gastroenteritis, typhoid fever, abortion, and bacteraemia, depending on the serovar and the host. For instance, *S. enterica* subspecies *enterica* serovar Typhi (*S. Typhi*) is responsible for typhoid fever in humans, whereas *S. enterica* subspecies *enterica* serovar Typhimurium (*S. Typhimurium*) causes gastrointestinal inflammation in humans and a systemic typhoid-like disease in mice (Garai et al., 2012). *Salmonella* is a facultative intracellular pathogen that usually resides in a modified phagolysosome known as *Salmonella*-containing vacuole (SCV) during host infection. However, recent data indicate that this facultative intracellular pathogen has a distinct bimodal lifestyle in epithelial
cells, where, there are subpopulations of vacuolar and cytosolic Salmonella (Knodler et al., 2010). Target cells include M cells, gut epithelial cells, dendritic cells (DCs), macrophages, monocytes, neutrophils, B cells, and T cells. During its passage through the host, Salmonella must resist or evade multiple levels of immune defence. Many virulence genes contribute to the survival of these bacteria inside the host. Some of them are clustered in horizontally acquired genomic regions known as Salmonella pathogenicity islands (SPIs) (Gerlach and Hensel, 2007). The largest and best known are SPI1, which is present in all the members of the genus, and SPI2, which is present in *S.* enterica but not in *S.* bongori. These SPIs encode two distinct type III secretion systems, type III secretion system 1 (T3SS1) and type III secretion system 2 (T3SS2), respectively (Galán and Curtiss, 1989; Ochman et al., 1996; Shea et al., 1996). These systems are able to inject directly into the eukaryotic host cells a number of bacterial proteins known as effectors (Ramos-Morales, 2012a). Collectively, these virulence factors contribute to invade host cells, interfere with host cellular functions, subvert immunity, establish an intracellular niche, and promote pathogen proliferation.

In this review we summarize the mechanisms of immunity that pathogens in general have to face and the specific strategies that *S.* enterica uses to deal with the different levels of defence of the host.

---

### Overview of mechanisms of immune defence

Humans, like any other animal are confronted every day to a myriad of microorganisms. However, these potentially infectious agents do not cause disease regularly due to a series of defence mechanisms that have been shaped over millions of years of evolution.

Three levels of immune defence can be recognized (Delves et al., 2011; Kenneth, 2011):  
(i) physical barriers, (ii) the innate immune system and (iii) the adaptive immune system.

#### Physical barriers

The best way to avoid infections is by keeping potentially infectious agents out of the body. Physical barriers are very effective for this purpose. They are represented by the skin and other epithelia covering the gastrointestinal, respiratory and urogenital tracts. These surfaces provide, not only mechanical, but also chemical and microbiological barriers.

The primary mechanical protection common to all epithelia consists of epithelial cells linked by tight junctions. The internal epithelia, or mucosal epithelia, are covered with mucous secretions. Mucus blocks the adherence of bacteria to epithelial cells. Microorganisms trapped within it can be removed by the movement of cilia, coughing or sneezing. In the intestinal mucosa, mucus is secreted by specialized goblet cells and creates a physical barrier against microbial pathogens (Liévin-Le Moal and Servin, 2006). Other general mechanical factors that contribute to this first level of defence are the flow of air or other fluids, like the tears in the eyes.

In addition to these physical obstacles, a chemical barrier to infections is provided by substances that kill microbes or that inhibit their growth. These chemical substances include a group of antibacterial enzymes present in body fluids, like lactoperoxidase in milk, lysozyme and secretory phospholipase A2 in tears, nasal secretions and saliva. Lysozyme breaks the peptidoglycan (PG) in the bacterial cell wall of Gram-positive and Gram-negative bacteria. Phospholipase A2 kills bacteria by hydrolysing phospholipids in the cell membrane. Antimicrobial peptides, like defensins, cathelicidins and histatins, represent another importantgroup of antimicrobial agents associated to epithelia. All of them are amphipathic peptides activated by proteolysis. Defensins and cathelicidins disrupt membranes and are toxic to many microorganisms, whereas histatins act against pathogenic fungi. Additional chemical barriers that are found by potential food pathogens include the acidity of the stomach pH and the bile in the intestine.

Microbiota is the microbial community composed of bacteria, protozoa, archaea, viruses and fungi that reside in different body niches. This microbiota competes with pathogenic microorganisms for nutrients and attachment sites. Therefore, the normal microbiota that is present in the intestine provides a microbiological physical defence against potential microbial pathogens that, like Salmonella, usually invade through the gut. In addition, some microbiota-produced metabolites affect the growth and virulence of pathogens. These metabolites include short-chain fatty acids, succinate, mucin O-glycans, molecular hydrogen, secondary bile acids, and the quorum sensing autoinducer AI-2 (Vogt et al., 2015).

## Innate immune defence

The second level of defence is the innate immune response. This response occurs immediately after entry of a foreign agent into the body, but its effectiveness does not improve upon secondary encounter with the same agent. Microorganisms are recognized and killed by two main classes of phagocytic cells: polymorphonuclear neutrophils and macrophages. These cells carry pattern recognition receptors (PRRs) on their cytoplasmic membranes that adhere to pathogen-associated molecular patterns (PAMPs) on the microbe surface. PRRs include Toll-like receptors (TLRs), C-type lectin receptors (CTLRs), NOD-like receptors (NLRs), RIG-like receptors (RLRs) and scavenger receptors. PAMPs that activate TLRs include lipopolysaccharide (LPS), PG, lipoproteins, flagellin and other pathogen-derived ligands. Binding of a TLR to its ligand leads to activation of nuclear factor κB (NF-κB) and several members of the interferon-regulated factor (IRF) family of transcription factors. CTLRs are very diverse transmembrane proteins. RLRs are found in the cytoplasm and activate NF-κB and IRF3/4 to induce antiviral type I interferons in response to double-stranded RNA. Scavenger receptors recognize modified low-density proteins, LPS and other ligands, and some of them can cooperate with TLRs (Murshid et al., 2016). NLRs are soluble cytosolic proteins that usually are composed of three domains: an N-terminal domain that recruits proteases or kinases, a central oligomerization domain and C-terminal leucine-rich repeats (LRRs) that recognize PAMPs. NLRs exist in an autoinhibited conformation that is released upon binding to a PAMP, allowing oligomerization and recruitment of different host ligands depending on the particular NLR (Kim et al., 2016). The NLRs NOD1 and NOD2 induce autophagy to remove pathogens by recruiting ATG16L1 to the plasma membrane at the site of bacterial entry. Autophagy is a conserved intracellular degradation pathway that involves the formation of double-membrane vesicles, known as autophagosomes, which deliver cytosolic components to the lysosome for degradation. Microbe-induced autophagy, or xenophagy, is important to restrict the growth of a number of intracellular pathogens, including S. Typhimurium (Gomes and Dikic, 2014). NOD1 and NOD2 activate NF-κB and mitogen-activated protein kinase (MAPK) signalling pathways, whereas NLRP2 and NLRP4 are negative regulators of the NF-κB pathway. Some NLRs, as well as absent in melanoma 2-like receptors (ALRs) and pyrin, act as inflammasome sensors (Sharma and Kanneganti, 2016). The inflammasome is mainly activated in myeloid cells as an essential component of innate immunity that promotes caspase-1-induced conversion of

Curr. Issues Mol. Biol. Vol. 25procytokines into active IL-1β and IL-18. It can result in an inflammatory form of cell death known as pyroptosis. Inflammasomes are cytosolic protein complexes that usually contain a specific sensor, an adaptor molecule (ASC) and pro-caspase-1. Canonical inflammasomes contain NLRP1, NLRP3, NLRC4, AIM2 or pyrin as sensors. NLRP3 is also involved in a non-canonical inflammasome that activates pro-caspase-11 in response to Gram-negative bacteria. This inflammasome is primed by recognition of LPS through a TLR and type I interferon signalling (Rathinam and Fitzgerald, 2016).

The general consequences of binding to PRRs include the activation of macrophages and neutrophils to increase their phagocytic activity, and the release of cytokines and chemokines that amplify the immune response. Once inside the phagocytic cell, the microorganism can be killed by reactive oxygen or nitrogen intermediates, by preformed antimicrobials like the defensins mentioned above or a neutral protease. This takes place with the assistance of other factors like low pH, lysozyme and lactoferrin. Other important cells in the innate immune system are natural killer cells, which identify host cells expressing abnormal patterns of proteins and induce apoptosis of these cells.

In addition to the cellular components of innate immunity, the complement system and other soluble circulating defensive proteins integrate the humoral innate immunity. The complement system is a multicomponent enzyme cascade that facilitates phagocytosis and lysis of microorganisms. This system can be activated by the lectin, classical or alternative pathways. All of the aforementioned generate a C3 convertase that leaves C3b bound to the microbial surface and releases C3a. C3b acts as an opsonin that increases the ability of phagocytes to ingest bacteria. C3a, together with C5a, recruit phagocytic cells to the site of infection and promote inflammation. Complement activation also leads to the formation of a membrane-attack complex that causes cell lysis. In addition, complement activation together with products of activated mast cells promotes inflammation, a response that leads to local swelling, redness, pain and temperature elevation.

### Adaptive immune response

Unlike innate responses, acquired or adaptive immunity is able to recognize specific antigens and develop a response against these pathogens. In contrast to innate immunity, adaptive immunity is different in each individual of a species and responds to specific antigenic challenges, displaying then a flexible spectrum of action, as it is able to recognize millions of different antigenic molecules. The two major varieties of lymphocytes, B-lymphocytes and T-lymphocytes, are essential to generate humoral and cellular immunity, respectively. B-lymphocytes are able to make antibodies, specific antigen-recognition molecules, known as immunoglobulins, which deal with extracellular infections. T-lymphocytes are involved in the control of intracellular infections. Their receptors recognize processed antigens in association with molecules of the major histocompatibility complex (MHC). T cells are classified in two categories: CD4 (helpers) and CD8 (cytotoxic) T cells, which are activated upon recognition of an antigen presented by major histocompatibility complex class II (MHC-II) and major histocompatibility complex class I (MHC-I), respectively by the TCR. While MHC-I is present on most cell surfaces, MHC-II is expressed on antigen-presenting cells (APC).

A connection between the innate and adaptive immune systems at the level of cellular immunity is provided by DCs, which are relevant APCs. Stimulation of DCs by PAMPs triggers their maturation. Mature DCs present antigens to T cells via MHC molecules andalso provide co-stimulatory signals via B7 family ligands, promoting the activation of naive T cells. The interconnection between innate and adaptive immunity is also important for humoral immunity since the classical pathway to activate the complement requires immunoglobulin M (IgM) or immunoglobulin G (IgG).

# Dealing with physical barriers

## Acidic pH

Bacteria belonging to the genus *Salmonella* usually invade their hosts through oral ingestion of contaminated food or water. Lysozyme found in saliva is not a significant barrier for these pathogens since, like many other Gram-negative bacteria, they are capable of resisting it thanks to their outer membrane shield around the PG layer (Van Kesteren et al., 1942; Masschalck and Michiels, 2003). Therefore, the first real challenge that these bacteria have to face upon ingestion is the acidic pH of the stomach of the host (Ramos-Morales, 2012b) due to gastric secretions and hydrochloric acid. The pH value of the stomach is variable: a pH of 1.3 was observed in healthy humans in the fasted state and of 4.9 after meal ingestion (Russell et al., 1993). Values of 3.9 and 4.0 were measured in rats and mice, respectively, in the fasted state (McConnell et al., 2008). This low gastric pH can quickly kill enteric bacteria, but different bacterial species have developed mechanisms to survive during passage through the stomach. When compared with *E. coli*, that resists extreme acid stress and can survive at pH 2 for hours, *S. enterica* is much less acid resistant (Koutsoumanis and Sofos, 2004; Lin et al., 1995). However, *S. Typhimurium* possesses at least three mechanisms of acid tolerance response (ATR) (Audia et al., 2001): (i) the exponential phase ATR is induced at pH 4.5–5.8 and allows subsequent survival at pH 3; (ii) the RpoS dependent stationary phase ATR is pH independent and is part of a general stress response; (iii) the RpoS independent stationary phase ATR is induced by exposure to pH 4.5 and provides longer tolerance to pH 3 than the RpoS dependent system. These tolerance systems require acid shock proteins whose synthesis is regulated by RpoS, the iron regulatory protein Fur or the two-component systems PhoQ/PhoP or EnvZ/OmpR (Audia et al., 2001; Foster, 1993). Small RNAs (sRNAs) have recently been identified as major regulators of stress response networks. For instance, the sRNA DsrA influences the ATR and virulence in *S. Typhimurium* (Ryan et al., 2016).

Bacteria that are able to survive the passage through the stomach must face new challenges in the intestine before establishing an intracellular niche. This includes mucus, bile, antimicrobial peptides and the resident microbiota (Fig. 6.1).

## Mucus

Epithelial cells in the human gut are covered with mucous secretions that protect the epithelium from the environment. The mucus barrier is composed of at least two glycan layers, which in turn incorporate multiple layers of glycoproteins (mucin) and complex oligosaccharides (glycocalyx) that protect cells from the local environment and infection. Microbes are able to use the host glycan as nutrients that regulate bacterial infection and virulence (Marcobal et al., 2013). Intestinal intracellular pathogens have to penetrate the mucin layers and subsequently gain access to the cell membrane. In order to achieve this, *Salmonella* uses a set of glycosyl hydrolases (GHs) that are able to degrade glycans. Recently, Arabyan et al. (2016) investigated the specific enzymes used by *Salmonella* to degrade the glycan. They

Curr. Issues Mol. Biol. Vol. 25
Salmonella ingestion

Physical barriers

Salmonella strategies

Acidic pH → Acid tolerance response

Bacterial envelope: LPS, ECA, PG
Eflux pumps
DNA repair
Adaptation
LPS modification

Antimicrobial peptides → Binding, efflux, degradation

Intestinal microbiota → Carbon sources: fucose, ethanolamine
Electron acceptors: nitrate, tetrathionate

Inflammation

Salmonella proliferation

Figure 6.1 Schematic representation of Salmonella strategies to circumvent host physical barriers. After ingestion of contaminated food or water, Salmonella crosses the stomach where it is able to resist to acidic pH activating an acid tolerance response. Then Salmonella reaches the intestine where it has to face different chemical challenges, like the bile produced in the liver or antimicrobial peptides secreted by Paneth cells in the gut. In addition to that, Salmonella has to compete with the resident intestinal microbiota for essential nutrients. Interestingly, Salmonella triggers an inflammation response in the gut that offers different nutrients or adhesion receptor sites that can be exploited only by pathogens.

identified two specific glycan-degrading enzymes in Salmonella, NanH and MalS, as new virulence factors. These enzymes are expressed during in vitro infection of human colonic epithelial cells (Caco-2) and degrade the glycocalyx layer produced by these cells. The authors also analysed the host response to glycan degradation and they observed induced expression of host genes that hydrolyse mannose, fucose, and N-acetylneuraminic acid (Neu5Ac) from the glycan; as well as sialyltransferases, suggesting host glycan remodelling during infection (Arabyan et al., 2016).

Bile

Bile is a secretory and excretory fluid produced in the liver whose composition varies depending on the individual’s diet (Coleman, 1987). Secretory functions include: delivery to the intestinal tract of bile salts to aid fat digestion and absorption/secretion of polymeric IgA to prevent infection in the biliary and upper intestinal tracts. In addition, bile is the excretory vehicle of liver-derived metabolites of potentially toxic materials. Bile contains inorganic ions, bile salts, lipids (phosphatidylcholine, cholesterol) and proteins (plasma proteins, liver-specific proteins, polymeric IgA). It is a bactericidal agent that disrupts
bacterial cell membranes due to the detergent activity of bile acids (Begley et al., 2005), causes DNA damage (Prieto et al., 2004, 2006) and alters the conformation of proteins. Salmonella, however, resists bile through different mechanisms. First, the bacterial cell envelope acts as a barrier to membrane-active agents like bile salts. Two components of the outer membrane are important for bile resistance: the LPS (Crawford et al., 2012; Gunn, 2000) and the enterobacterial common antigen (ECA) (Ramos-Morales et al., 2003). Recently, it has been shown that exposure to the bile salt sodium deoxycholate is associated with changes in the structure of the second layer of the cell envelope, the PG, and that PG remodelling contributes to bile resistance (Hernández et al., 2015). Once bile enters the cell, additional mechanisms are activated. Efflux pumps mediate the export of bile salts from the bacterial cytoplasm to the outside medium and are responsible for resistance to many different toxic compounds (Nishino et al., 2006). AcrAB-TolC is the best characterized efflux system in enteric bacteria (Ma et al., 1993, 1995; Nikaido et al., 2008; Zgurskaya and Nikaido, 1999). AcrB is an inner-membrane protein, TolC is the outer membrane component and AcrA is located in the periplasm. The genes encoding these proteins are regulated by the transcriptional regulator RamA, which is synthesized in response to bile (Baucheron et al., 2014). In addition, bile-induced DNA damages can be repaired by Dam-directed mismatch repair and by base excision repair, and the impairment of DNA replication rescued by SOS-dependent translation DNA replication and RecBCD-dependent recombinational repair (Prieto et al., 2006). Finally, Salmonella is also able to adapt to grow in extremely high bile concentrations if previously exposed to sublethal doses. This is a transitory state that involves multiple changes in gene expression (Prouty et al., 2004). The RpoS-dependent general stress response plays a crucial role in the adaptation of Salmonella to bile (Hernández et al., 2012).

### Antimicrobial peptides

Different families of antimicrobial peptides protect the intestine. Defensins and cathelicidins are of special interest in the context of gastrointestinal infections (Wehkamp et al., 2007). They are produced in the intestinal epithelium either constitutively or in an inducible manner. For instance, Paneth cells of the small intestine express the most abundant, constitutively expressed defensins. Human cathelicidin is expressed by epithelial cells in the stomach, small bowel and colon. In this context, butyrate, a short-chain fatty acid produced in the colon by bacterial fermentation of dietary fibres, has been shown to induce expression of the human cathelicidin gene. Salmonella is able to sense sublethal concentrations of antimicrobial peptides and induce several mechanisms of resistance that involve modification of the LPS or sequestering, efflux and proteolytic degradation of antimicrobial peptides (Matamouros and Miller, 2015). Regulatory proteins involved in this process include the alternative sigma factor RpoE and the two-component or phosphorelay systems PhoQ/PhoP, PmrA/PmrB and Rcs. PhoQ is an inner-membrane histidine kinase that senses antimicrobial peptides (Bader et al., 2003, 2005) and phosphorylates the response regulator PhoP. Activation of PhoP prevents dephosphorylation of PmrA (Kato and Groisman, 2004). This protein directly controls expression of genes involved in LPS modifications necessary for resistance to antimicrobial peptides. The Rcs system is a phosphotransfer cascade that responds to antimicrobial peptides through RcsF (Farris et al., 2010). This system controls the synthesis of colanic acid capsule and the transcription of other genes necessary for resistance (Detweiler et al., 2003). Modifications of the bacterial surface that contribute to antimicrobial peptide resistance include:

Curr. Issues Mol. Biol. Vol. 25
1 Regulation of O-antigen length by the PmrA/PmrB system (Farizano et al., 2012; Pescaretti et al., 2011).
2 Modification of the anionic phosphate groups in the core and lipid A regions of the LPS by addition of aminoarabinose and phosphoethanolamine. PmrA is involved in the regulation of these modifications as well as in the inhibition of the activity of LpxT, which is responsible for increasing the negative charge of LPS by adding an additional phosphate. Together these changes lead to a reduction of the negative charge of the outer membrane and of the electrostatic interactions with antimicrobial peptides (Gunn et al., 1998; Herrera et al., 2010; Jones et al., 2008a; Lee et al., 2004; Zhou et al., 2001b).
3 Decrease in the fluidity of the outer membrane by the incorporation, catalysed by PagP, of an additional palmitate to lipid A and to phosphatidylglycerol (Bishop et al., 2000; Dalebroux et al., 2014; Guo et al., 1998).

Additional resistance mechanisms involve binding, efflux or degradation of antimicrobial peptides. The two PhoP-regulated proteins Mig-14 and VirK are suggested to bind antimicrobial peptides to prevent their penetration into Salmonella (Brodsky et al., 2005). The sapABCDF operon encodes an ABC transporter system involved in resistance to antimicrobial peptides (Parra-Lopez et al., 1993). It has been proposed that the SapABCDF transporter system functions as a complex with SapG and SapJ to mediate both peptide and K⁺ transport (Parra-Lopez et al., 1994). Finally, PgtE is a post-transcriptionally regulated component of the PhoQ/PhoP regulon with protease activity against alpha-helical cationic antimicrobial peptides (Guina et al., 2000).

### Intestinal microbiota

Before invading appropriate host cells and becoming an intracellular pathogen, Salmonella has to compete with the resident intestinal microbiota. The lumen of the human gastrointestinal tract is rich in nutrients and provides an appropriate niche for a diverse community of bacteria (Bäckhed et al., 2005). They increase in number from the stomach to the colon, where they reach concentrations around 10¹² per gram of luminal content. Most colonic bacteria are members of the Firmicutes and Bacteroidetes phyla, whereas Proteobacteria, Actinobacteria, Fusobacteria and Verrucomicrobia are less abundant (Eckburg et al., 2005). This microbiota helps to prevent infection by providing a physical barrier for the attachment of bacterial pathogens to the mucosal surfaces and competing with pathogens for essential nutrients (van der Waaij et al., 1971). Salmonella, however, can exploit the host innate immune response and its own metabolic capacities to overcome, at least temporarily, this barrier (Khan, 2014). First, the initial growth of Salmonella in the gut is powered by Hyb hydrogenases that enable the bacteria to use hydrogen produced by the resident microbiota as a central intermediate of metabolism (Maier et al., 2013). Using a mouse colitis model, it has been shown that inflammatory host responses triggered by S. Typhimurium virulence factors (T3SS effectors) shift the competition in favour of the pathogen (Stecher et al., 2007). The mechanisms leading to this result are not completely understood and different explanations have been proposed: (i) inflammation involves release of antibacterial factors that may kill or delay the growth of specific members of the microbiota that would normally inhibit Salmonella growth; and (ii) the inflamed intestine could offer different
nutrients or adhesion receptor sites that can be exploited only by pathogens. In particular, reactive oxygen species generated by neutrophils during inflammation can react with endogenous thiosulfate to form tetrathionate. The *ttrRSBCA* locus, located in SPI2, confers to *S.* Typhimurium the ability to use tetrathionate as a terminal electron acceptor in anaerobic respiration (Winter *et al.*, 2010a). This confers a growth advantage because, unlike most bacteria in the gut, *S.* Typhimurium can use ethanolamine as a carbon source in the presence of tetrathionate as a respiratory electron acceptor (Thiennimitr *et al.*, 2011). It has also been suggested that the ability of *S.* Typhimurium to degrade fucose and anaerobically degrade 1,2-propanediol, probably using tetrathionate as a terminal electron acceptor, provides an advantage to this pathogen during competition with the microbiota (Staib and Fuchs, 2015). In addition, some *S.* Typhimurium isolates carry the gene *sopE* in a prophage. The product of this gene increases the severity of intestinal inflammation and drives the host to generate an additional terminal electron acceptor, nitrate, that suppresses genes responsible for utilization of energetically inferior electron acceptors such as tetrathionate (Lopez *et al.*, 2012). Stecher *et al.* (2012) described in an interesting study the discovery of a mechanism driving efficient conjugation between Enterobacteriaceae (*S.* Typhimurium and *E.* coli) in the host's intestine. Using a mouse colitis model, the authors observed parallel blooms of pathogenic *S.* Typhimurium and resident commensal *E.* coli. This phenomenon, probably due to dysbiosis of the anaerobic microbiota caused by Salmonella infection, led *E.* coli to account for >80% of the total intestinal bacteria. In this scenario, the authors observed conjugative horizontal gene transfer efficiencies of 100%. This suggests that bacterial blooms occur after Salmonella infection fuels the reassortment of genetic material between different Enterobacteriaceae. This further implies that infected patients might enhance the spread of plasmid-encoded fitness, virulence- and antibiotic resistance determinants (Stecher *et al.*, 2012).

---

### Evasion of the innate immune system

Salmonellae that are able to deal with the aforementioned physical barriers can invade the gut epithelium, preferentially in the distal ileum, using several routes. They usually invade microfold cells (M cells) that overlie Peyer’s patches, but they can also induce their own uptake by epithelial cells or be engulfed by CD18⁺ phagocytes, probably monocytes or dendritic cells (Watson and Holden, 2010). In a systemic infection, Salmonella can disseminate to internal organs [mesenteric lymph nodes (MLNs), liver and spleen]. Access to other cell types, like macrophages, could be a consequence of cell death induction (Fink and Cookson, 2007) and subsequent phagocytosis. Both intracellularly and during the transient extracellular passage from one host cell to another, Salmonella has to face and resist components of the innate immune system.

Salmonella finds cationic antimicrobial peptides not only in the intestinal lumen but also inside macrophages. Resistance mechanisms against this innate defence were described in the previous section. Interestingly, the PhoQ/PhoP two-component system, that sense these antimicrobial peptides, are also additively activated by intracellular signals (acidic pH and divalent cation limitation). However, the presence of antimicrobial peptides could be the most relevant signal for systemic virulence (Hicks *et al.*, 2015).
Manipulation of inflammatory pathways

Relevant PRRs for the recognition of Salmonella PAMPs are members of the TLR family, associated to host membranes, and members of the NLR family, that recognize the presence of PAMPs in the cytosol. This recognition leads to the activation of the NF-κB pathway (O’Dea and Hoffmann, 2010). Specific Salmonella virulence factors that enhance NF-κB activity are the T3SS1 effector SopE, which is recognized by NOD1 (Keestra et al., 2013), and the T3SS2 effector SrfA, that promotes NF-κB activation by binding to TOLLIP, an inhibitor of IRAK1 (Lei et al., 2016). This proinflammatory pathway can also be induced by the T3SS needle proteins PrgI and SsaG through TLR2 and TLR4 (Jessen et al., 2014). In addition, SopE acts as a GTP exchange factor (GEF) for Rho family GTPases Cdc42 and Rac1, leading to activation of the c-Jun N-terminal kinase (JNK) pathway. Three different Salmonella proteins have been involved in inflammasome activation: (i) flagellin, an inducer of the NLRC4 inflammasome injected through T3SS1; (ii) PrgJ, a component of the basal body inner rod of T3SS1, also detected by NLRC4 (Miao et al., 2010a); (iii) SopE contributes to inflammasome activation through its GEF activity, although the specific inflammasome sensor has not been determined (Hoffmann et al., 2010; Müller et al., 2009). NLRP3 has been suggested to have also an important role in innate immune defence against S. Typhimurium, since mice lacking both NLRP3 and NLRC4 are significantly more susceptible to infection (Broz et al., 2010). It has been recently shown that NLRP3 associates with NLRC4 in macrophages infected with S. Typhimurium or transfected with flagellin, revealing an unexpected overlap between two distinct inflammasomes (Qu et al., 2016). S. Typhimurium and other Gram-negative bacterial pathogens could induce a non-canonical inflammasome involving cytosolic recognition of LPS and activation of caspase-11 in mice and caspase-4 in humans (Broz and Monack, 2013a; Broz et al., 2012; Casson et al., 2015; Storek and Monack, 2015). This activation has been observed in the context of infections with mutants that aberrantly enter the cytosol (Aachoui et al., 2013) after lysis of the SCV dependent on IFN-induced GTPases (Meunier et al., 2014). However, wild-type S. Typhimurium can also lyse their nascent vacuole following invasion of tissue culture epithelial cells and enter the cytosol where it can eventually hyper-replicate (Birmingham et al., 2006; Knodler et al., 2014). Using a semi-quantitative single-cell analysis, Malik-Kale et al., showed that although cytosolic hyper-replication occurs in less than 20% of infected epithelial cells, it accounts for the majority of net intracellular replication (Malik-Kale et al., 2012). Interestingly, the transcriptional profiling of both intracellular bacterial subpopulations are different and it is known that cytosolic Salmonella are induced for T3SS1 and flagellated, whereas vacuolar bacteria are T3SS2-induced (Knodler et al., 2010). Eventually, cytosolic hyper-replication leads to epithelial cell death via pyroptosis. This results in cell lysis, proinflammatory cytokine release and escape of the cytosolic bacteria into the extracellular space, providing a potential mechanism of dissemination (Knodler, 2015).

Anti-inflammatory response to Salmonella infection

Although an initial inflammatory response is beneficial for Salmonella to outcompete the microbiota, as the intracellular infection progresses, the pathogen still has mechanisms to inhibit inflammatory pathways (Fig. 6.2). Down-regulation of flagellin expression in systemic sites is a way to prevent NLRC4 inflammasome activation (Cummings et al., 2006; Miao et al., 2010b; Winter et al., 2010b). Control of bacterial production of the tricarboxylic acid (TCA) cycle metabolite citrate may contribute to evade NLRP3 inflammasome

Molecular Mechanisms Used by Salmonella | 143

![Diagram](#)

- Ubiquitin
- T3SS1 effector
- T3SS2 effector
- T3SS1/2 effector
- Host proteins

Pro-inflammatory pathways
Anti-inflammatory pathways

Figure 6.2 Schematic illustration of mechanisms of *Salmonella* to manipulate the host immune system by injecting effector proteins through two T3SS. *Salmonella* triggers its own internalization into the host cell in a T3SS1-dependent manner. Following internalization, *Salmonella* establishes its intracellular niche in a modified phagosome, called *Salmonella*-containing vacuole (SCV), where the pathogen manages to survive and proliferate. Biogenesis and maturation of the SCV depends on the activity of T3SS 1 and 2 effectors that manipulate the normal endocytic pathway to avoid the fusion and degradation by a lysosome. The actions of the T3SS proteins effectors on the inflammatory pathways to evade the host immune system are schematically represented.

activation in macrophages (Wynosky-Dolfi et al., 2014). In addition, many T3SS effectors have an anti-inflammatory role. Both the T3SS1 effector AvrA and the T3SS2 effector SseL deubiquitinate IkBα and inhibit its degradation (Le Negrate et al., 2008; Ye et al., 2007). Binding of IkBα to NF-κB prevents its translocation to the nucleus. AvrA also inhibits JNK-induced signalling (Du and Galán, 2009; Jones et al., 2008b; Wu et al., 2012). GogA, GtgA and PipA are members of the same family of proteases that redundantly cleave specific members of the NF-κB family, including RelA (p65) and RelB (Sun et al., 2016). SspH1 localizes to the mammalian nucleus and inhibits NF-κB-dependent gene expression through interaction with protein kinase 1 (PKN1) (Haraga et al., 2006). The effector SptP consists of two domains. The N-terminal domain acts as a GAP for Cdc42 and Rac1 that mediates reversion of the effects of SopE on the actin cytoskeleton (Fu and Galán, 1999). The C-terminal domain possesses tyrosine phosphatase activity, which is (Kaniga et al., 1996) involved in reversing the initial activation of the MAPK ERK, triggered by *Salmonella* infection (Murli et al., 2001). SptP inhibits this MAPK pathway through inhibition of Raf1 activation in a process that involves both SptP activities (Lin et al., 2003). This down-modulation of ERK activation could explain the contribution of SptP, together with SspH1, towards the inhibition of IL-8 production after invasion of intestinal epithelial cells (Haraga and Miller, 2003). Another effector involved in reduction of the production of inflammatory cytokines through the inactivation of ERK is SpvC. This effector is encoded in a virulence plasmid present in some *S.* enterica serovars and can be secreted by T3SS1 and T3SS2. SpvC inactivates ERK by irreversibly removing phosphate from a threonine residue in the conserved activation loop motif TXY using its phosphothreonine lyase activity

Curr. Issues Mol. Biol. Vol. 25
(Mazurkiewicz et al., 2008). Interestingly, during the interaction of Salmonella with the plant model Arabidopsis, SpvC interacts with and dephosphorylates activated MAPK6 to inhibit defence signalling (Neumann et al., 2014). This suggests that some Salmonella effector proteins could have a conserved function during proliferation in different hosts. GogB is another anti-inflammatory effector that limits NF-κB activation by targeting the host SCF E3 ubiquitin ligase and inhibiting IkBa degradation (Pilar et al., 2012). SseK1, an effector protein that can be secreted by T3SS1 and T3SS2 (Baisón-Olmo et al., 2015), has a N-acetylglucosamine transferase activity that modifies the TNF-α receptor TNFR1 and its adaptor TRADD, preventing TNF-α mediated activation of NF-κB (Li et al., 2013).

An important mediator of the anti-inflammatory effects of Salmonella is haem-oxygenase 1 (HO-1). HO-1 is an anti-stress regulatory molecule. It catabolises the degradation of haem into biliverdin with the production of free iron and CO. Different studies have shown that HO-1 can be induced by a wide variety of bacteria. This is likely due to the increase in oxidative stress and inflammation (reviewed in Blancou et al., 2011). Onyiah et al. (2013) using a colitis mouse model, showed that the enteric microbiota induces expression of HO-1 in mice and zebrafish. HO-1-derived CO enhances macrophage bactericidal activity and bacterial clearance *in vivo*. Similarly, others works have shown that an absence of HO-1 in humans and mice leads to chronic inflammation (Chung et al., 2008; Yachie et al., 1999), although the exact mechanisms remains unclear. One hypothesis emerged from the experiments performed by Choi and Lee, who showed a positive feedback of the system in which CO stimulates the synthesis of the anti-inflammatory cytokine interleukin 10 (IL-10) by macrophages (Otterbein et al., 2000). IL-10 induces the expression of HO-1 potentially leading to a self-amplification of the anti-inflammatory effect (Lee and Chau, 2002). Others propose that HO-1 controls early innate immune response of S. Typhimurium-infected macrophages. In this study, a reduced survival of intracellular Salmonella in cells with low HO-1 expression was observed, probably due to the limited iron availability and to the activation of pro-inflammatory pathways that occurs in the absence of this enzyme (Mitterstiller et al., 2016). On the other hand, Salmonella-infected macrophages with strongly up-regulated HO-1, provide a survival niche for Salmonella (Nix et al., 2007). Thus, the induction of HO-1 expression has been suggested to be a strategy of the pathogen, given its harmful effects in the early control of Salmonella infection.

Another example of the immunosuppressive role of HO-1 was described by Riquelme et al. (2015), who suggest HO-1 prevents T-cell-mediated inflammatory disease by producing CO and impairing DC immunogenicity. CO has been shown to impair the mitochondrial function in DCs by reducing both the mitochondrial membrane potential and ATP production. This gas is able to impair the endo-lysosomal antigen trafficking route, reducing the ability of DCs to prime naive T cells. The role of HO-1 in immunity has been also studied in DCs, where CO is the molecule with the most prominent immunosuppressive capacity over their function (Rémy et al., 2009; Tardif et al., 2013). Contrary to this, other authors, like Zaki et al., suggest that HO-1 contributes to improved control of Salmonella replication within macrophages by exerting a cytoprotective effect (Zaki et al., 2009). Further studies are necessary to clarify the dual role of HO-1 in infections.

## Autophagy

The relationship of Salmonella with autophagy is also of interest. During S. Typhimurium infection, T3SS1-mediated damage of the SCV allows ubiquitin and galectin-8 to targetbacteria inside the affected SCVs or free in the cytoplasm (Birmingham and Brumell, 2006; Birmingham *et al.*, 2006; Thurston *et al.*, 2012). These signals recruit essential anti-bacterial autophagy components including several cargo receptors, the kinase TBK1 and WIPI2, that restrict *Salmonella* proliferation (Radtke *et al.*, 2007; Thurston *et al.*, 2016, 2009, 2012). Autophagy has also a role in sealing damaged endosomal membranes, allowing T3SS2 induction inside the SCV (Kreibich *et al.*, 2015; Owen and Casanova, 2015). In epithelial cells, *Salmonella* induces the formation of ubiquitinated aggregates near the SCV in a T3SS2-dependent manner. These aggregates are recognized by the autophagy machinery. The T3SS2 effector SseL is a deubiquitinase whose activity lowers autophagic flux and favours intracellular *Salmonella* replication (Mesquita *et al.*, 2012). In a recent study that used a fibroblast-infected model, López-Montero *et al.* (2016) showed that *S.* Typhimurium develops membranous aggregates connected to the phagosome where the bacteria controls its progeny and establishes an intracellular persistent infection. In addition, T3SS2 mediates active suppression of autophagic signalling in macrophages through recruitment of focal adhesion kinase (FAK) to SCVs and activation of the Akt-mTORC1 signalling pathway (Owen *et al.*, 2014). The inhibition of autophagy by *Salmonella* also prevents the induction of a protective cytokine response mediated by interferon beta (IFN-β) (Owen *et al.*, 2016). Two recent studies show that SpvB, a T3SS2 effector with ADP-ribosyltransferase activity that prevents polymerization of G-actin into F-actin filaments (Hochmann *et al.*, 2006; Lesnick *et al.*, 2001; Margarit *et al.*, 2006; Tezcan-Merdol *et al.*, 2001), can inhibit autophagosome formation and increase inflammatory injury in zebrafish intestine (Li *et al.*, 2016) mammalian cells and mice (Chu *et al.*, 2016). This inhibition occurs at the early stage of autophagy via depolymerization of actin filaments.

## The complement system

Finally, mechanisms to evade or interfere with the complement system have evolved in *Salmonella*. The human pathogen *S.* Typhi expresses the Vi capsular polysaccharide during transit through the intestine (Tran *et al.*, 2010). This polysaccharide has anti-inflammatory properties, (Sharma and Qadri, 2004) and also inhibits complement deposition (Looney and Steigbigel, 1986; Wilson *et al.*, 2011). This, thanks to a lack of hydroxyl groups available for ester formation with C3b (Heyns and Kiessling, 1967). Evolution of these immune evasive properties of *S.* Typhi led to inactivation of the *fepE* gene, encoding a regulator of very-long O-antigen chains (Crawford *et al.*, 2013). The surface protease PgtE of *S.* Typhimurium affects complement activity by cleaving C3b, C4b, C5, B and H complement components (Ramu *et al.*, 2007; Riva *et al.*, 2015). Although factors B and H have opposite effects on complement activation, the overall effect of PgtE favours protection of *Salmonella* against the host immune system, since less C3-derived fragments accumulated on *Salmonella* and the association with neutrophils was reduced (Riva *et al.*, 2015).

---

## Mechanisms to escape adaptive immunity

In addition to mechanisms developed by *Salmonella* to escape from innate immunity, this intracellular pathogen has acquired different mechanisms to evade different adaptive immunity mechanisms.
Interactions of *Salmonella enterica* with dendritic cells

Innate and adaptive immunities are linked by DCs. Therefore, the activation of an effective adaptive immune response relies on efficient DCs capable of priming of naive T cells. This is one of the most relevant immune evasion strategies for *Salmonella*. This pathogen has developed molecular mechanisms to prevent the presentation of bacterial antigens that prime naive T cells and therefore avoid the initiation of the adaptive immunity.

In general, virulent strains of *Salmonella* are capable of modulating phagocytosis by DCs, avoiding lysosomal degradation and preventing antigen presentation to T cells. Here, we summarize the mechanisms developed by *Salmonella* to escape from acquired immunity. It is well known that *Salmonella* is able to induce phagocytosis on epithelial cells by active translocation of T3SS1 effectors which modulate the actin cytoskeleton. This results in internalization of *Salmonella* into the epithelial cell (Scherer et al., 2000; Steele-Mortimer et al., 2002; Zhou et al., 2001a). In contrast, the uptake of *Salmonella* by DCs is tightly regulated by effector proteins of T3SS1 (Bueno et al., 2010), being able to control the amount of bacteria that enter into the cells, avoiding then a massive immune response that would restrict *Salmonella* replication and dissemination. Additionally, Riquelme et al. showed that IgG-opsonized *Salmonella* are recognized by different receptors (FcγRIII receptors and others that have not been identified) expressed on DCs surfaces. These immune complexes are internalized and degraded by the lysosomal route, restoring its capacity to present *Salmonella* antigens to T cells and initiate an adaptive immunity response. However, the exact molecular mechanism with which IgG interferes with the secretion of *Salmonella* virulent effectors or impair its capacity to evade capture in DCs remains unclear (Riquelme et al., 2012).

After internalization, establishing a systemic infection depends on the pathogen's capacity to survive inside the host cells and to evade the immune response. In that sense, there are many studies on *Salmonella* virulence proteins that contribute to intracellular survival in DCs and dissemination in the host (Albaghdadi et al., 2009). One of the most relevant strategies developed by *Salmonella* to survive and replicate intracellularly is the disruption of the host endocytic trafficking machinery to avoid its lysosomal degradation within the SCV. The ability to survive in DCs is dependent on SPI2 effectors (Albaghdadi et al., 2009; Halici et al., 2008; Jantsch et al., 2003), which are injected to DCs cytosol from the SCV through a T3SS2. In fact, T3SS2 mutants show a reduced capacity to survive inside DCs and are attenuated in mice (Tobar et al., 2006). However, there are several examples of T3SS1 effectors that participate in avoiding lysosomal fusion. Among these SPI1 effectors, SopB and SopE were found to play a role in SCV maturation (Hernandez et al., 2004; Mukherjee et al., 2001). SopB is a phosphatase that mediates phosphatidylinositol 3-phosphate production in the SCV, through Rab5 recruitment to the SCV and its effector Vps34 (Mallo et al., 2008). Another example is SopE, which acts as a Rab5-specific exchange factor and mediates the recruitment of Rab5 in the GTP form in the SCV (Mukherjee et al., 2001). There are also examples of T3SS2 effectors that impair trafficking of endocytic cargo to lysosomes. The effector SopD2 directly binds and inhibits the host GTPase Rab7, a regulatory switch central to endocytic trafficking and phagosome–lysosome fusion. Consequently, this limits Rab7 interaction with its dynein- and kinesin-binding effectors RILP and FYCO1. This in turn disrupts the regulation of microtubule motors (D’Costa et al., 2015). SifA is another SPI2 effector required for bacterial survival inside DCs that alters lysosomal function
(Beuzón et al., 2000; Boucrot et al., 2003; Petrovska et al., 2004). McGourty et al. showed that SifA prevents the delivery of hydrolytic enzymes to the SCV by inhibiting Rab9-dependent retrograde trafficking of mannose-6-phosphate receptors (MPRs). This requires binding of SifA to its host cell target SKIP. Translocated SifA forms a stable complex with SKIP and Rab9 in infected cells. Sequestration of Rab9 by SifA-SKIP accounts for the effect of SifA on MPR transport and lysosome function (McGourty et al., 2012). There are other effectors of the T3SS2 that are involved in DC intracellular survival of Salmonella like SseJ, SseF, SspH2 and PipB2. Strains lacking these effectors show reduced survival inside DCs but the specific role of each of these virulence proteins remains unclear (Halici et al., 2008).

One of the suggested strategies used by Salmonella to avoid antigen presentation is inhibition of lysosomal degradation (Tobar et al., 2006, 2004). There are studies that suggest that the PhoQ/PhoP regulatory system is involved in this escape mechanism. This because Salmonella strains with mutations in the PhoQ/PhoP system are not able to escape from lysosomal degradation and therefore do not interfere with antigen processing and presentation (Niedergang et al., 2000). Other studies report that Salmonella regulates the expression of MHC class II antigens by polyubiquitination of HLA-DR. (Lapaque et al., 2009). These mechanisms rely on virulence factors encoded in SPI2. T3SS2 effectors like SifA, SlrP, SspH2, PipB2 and SopD have been shown to participate in this evasion process since they are required for the inhibition of MHC-II-dependent antigen presentation in DCs (Halici et al., 2008). The ability of Salmonella to prevent antigen presentation by DCs seems to depend exclusively on SPI2-related proteins, since a mutant strain lacking the T3SS1 is still able to avoid antigen presentation by DCs (Bueno et al., 2010). Importantly, this feature seems to be host specific and restricted to serovar Typhimurium, since other serovars like *S.* enteritidis and *S.* Typhi are not able to interfere with this function in murine DCs (Bueno et al., 2008).

*S.* Typhimurium can interfere with the host’s immune response during oral infection of mice through selective killing of CD8α⁺ DCs in a process that is dependent on MyD88 and TNFR1 (Sundquist and Wick, 2009). It has been recently shown that the pR~ST98~ plasmid of *S.* Typhi may also influence maturation, survival and cytokine production of DCs, preventing activation of T-cell-mediated immunity against antigens derived from this pathogen (Wei et al., 2015).

Another strategy used by virulent Salmonella is based on down-regulation of flagellin, a target of the innate and adaptive immune responses during infection. This strategy consists in preventing T-cell activation by the active reduction of the availability of bacterial antigens for presentation to T cells (Alaniz et al., 2006). Some data suggest that flagellin is differentially expressed by Salmonella populations infecting Peyer’s patches (Cummings et al., 2006). Furthermore, intracellular Salmonella can make flagellin unavailable for antigen processing by DCs (Alaniz et al., 2006). Similar studies showed that during low-dose Salmonella infections, the microbe evades activation of flagellin-specific CD4 T cells (Srinivasan et al., 2004).

## Direct interactions of Salmonella enterica with T cells

Examples of *S.* Typhimurium directly blocking T-cell proliferation are prevalent in the literature. This inhibition was observed in both CD4⁺ and CD8⁺ T cells and was due to a down-regulation of TCR expression β-chain, which interferes with the first step in T-cell clonal expansion (van der Velden et al., 2005, 2008). The authors of the aforementioned study also identified the enzyme L-asparaginase II as responsible for this inhibitory effect.

Curr. Issues Mol. Biol. Vol. 25The activity of this enzyme causes depletion of exogenous L-asparagine, leading to down-regulation of TCR-β expression, suppression of T-cell blastogenesis, blockade of cytokine production, and, ultimately, inhibition of T-cell proliferation (Kullas et al., 2012). In agreement with this, the cytolytic capacities of CD8⁺ T cells have been shown to be initiated by TCR interactions (Lewinsohn et al., 2011).

The phagosomal lifestyle of S. Typhimurium seems to be responsible for delayed MHC-I dependent antigen presentation and delayed expansion and contraction of the CD8⁺ T-cell response. Interestingly, even memory CD8⁺ T cells failed to undergo rapid expansion in response to infection with S. Typhimurium expressing the model antigen OVA (Luu et al., 2006).

Additionally, Salmonella is able to induce apoptosis of Ag-specific CD4 T cells (Srinivasan et al., 2009). This is driven by SPI-2 virulence genes. In vivo studies by Ertelt et al. suggested that Salmonella undergoes, in a SPI-2 dependent manner, a selective culling of activated CD4⁺ T-cell subsets, which re-shapes the repertoire of antigen-specific T cells that persist later after infection (Ertelt et al., 2011).

## Non-cognate activation of T cells

In addition to conventional activation by cognate peptide–MHC complexes presented by DCs, T cells can be indirectly stimulated (bystander stimulation) by Salmonella. This alternative pathway involves the well-characterized TLRs and NLRs, as well as other less known PRRs. It results in production of inflammatory cytokines that prime T cells at the site of infection (Broz and Monack, 2013b). Indeed, Salmonella has been proposed as a particularly appropriate model pathogen for study of non-cognate CD4 T-cell responses based on (i) strength of the Th1 response during infection, (ii) the requirement for CD4 T cells in bacterial clearance and (iii) the well-characterized inflammatory response to conserved molecular patterns induced by Salmonella infection (O'Donnell and McSorley, 2014).

## B lymphocytes' role

B lymphocytes have critical roles as positive and negative regulators of immunity. For instance, B cells have important contributions in acquired immunity against Salmonella infections, being essential for immunity during secondary challenge (McSorley and Jenkins, 2000). Several studies describe antibody-dependent (Mastroeni et al., 1993; McSorley and Jenkins, 2000; Mittrücker et al., 2000) and antibody-independent Salmonella-specific B cell responses (Barr et al., 2010; Mastroeni et al., 2000; Ugrinic et al., 2003). These studies suggest that B cells can work as APCs and have a key role in the production of inflammatory cytokines during Salmonella infection. The immunosuppressive role of B lymphocytes, classically associated with B cell-derived IL-10, is related to the regulation of autoimmune disease. This IL-10 also increases susceptibility to pathogens (Fillatreau, 2011; Fillatreau et al., 2008). In a recent study, Shen et al. described interleukin 35 (IL-35)-producing B cells as novel key players in the negative regulation of immunity (Shen et al., 2014). The authors showed that mice whose B cells did not express IL-35 displayed improved resistance to S. Typhimurium infection. The increased immunity found in these mice was associated with higher activation of macrophages and T cells, as well as an increased stimulatory function of B cells as APCs. The data presented in this work demonstrate that B cells can inhibit antimicrobial immunity through production of IL-35 (Shen et al., 2014).One of the strategies developed by Salmonella to evade the action of B cells is to delay the formation of the germinal centre, which is necessary to produce high-affinity antibodies against a specific pathogen (Cunningham et al., 2007). It has been shown that during Salmonella infection germinal centre formation and affinity maturation are delayed until the second month of infection. This differs greatly with conventional 7 day functional germinal centre formation. Furthermore, high-affinity antibodies start to be produced during the second week after immunization.

## Persistence

One important aspect of typhoid Salmonella is its capacity to establish chronic infections. After adequate treatment the majority of patients who present acute typhoid fever recover; however, a significant percentage of patients (1–6%) develop a chronic infection (Levine et al., 1982; Monack et al., 2004a). These patients become asymptomatic chronic carriers and sporadically shed bacteria in their stool, allowing the pathogen to close its life cycle and to be transmitted to new hosts (Monack et al., 2004b). Despite the importance of Salmonella chronic infections, relatively little is known about the host immune response or virulence mechanisms that characterize long-term systemic infections. Since the majority of chronically infected patients are asymptomatic, the identification of potential targets is challenging (Shpargel et al., 1985; Sinnott and Teall, 1987). However, it is interesting to note that gut persistent infections are Salmonella pathogenicity island 1 (SPI-1) and Salmonella pathogenicity island 2 (SPI-2) dependent (Lam et al., 2014). One study suggested that the human carrier state may be associated to an ineffective immune response (Thompson et al., 2009). In this section we describe some recent advances with respect to S. Typhi and S. Typhimurium persistent host colonization.

## Biofilm formation

Biofilm development is an important component of bacterial survival and a common source of persistent infections (Costerton et al., 1999). Salmonella are able to form biofilm in both biotic and abiotic surfaces. Moreover, there is a clear correlation between chronic infection and bacterial biofilm formation on the surface of gallstones (Gonzalez-Escobedo et al., 2011; Prouty et al., 2002). Epidemiological studies have revealed a strong association between chronic carriers of S. Typhi and gallstones (Schiøler et al., 1983). Although, 90% of chronically affected individuals have gallstones (Karaki and Matsubara, 1984) or other biliary track pathologies, this is not required to develop a carrier state (Monack et al., 2004b). Salmonella biofilms have been found on gallstones in both S. Typhi patients and a mouse model of chronic S. Typhimurium infection (Crawford et al., 2010), suggesting both serovars share this biofilm strategy to survive in their host. Interestingly, bile has been shown to enhance S. Typhi biofilm formation (Gonzalez-Escobedo et al., 2011). Consistently, S. Typhi exhibits specific binding to cholesterol-coated surfaces, like gallstones (Crawford et al., 2008). FliC and OmpC are two proteins that were described to have a key role on initial adhesion to cholesterol-coated surfaces (Crawford et al., 2010). A hallmark of biofilm formation is the self-production of an extracellular matrix that ensures the integrity of the biofilm. The biofilm matrix is composed of EPS (extracellular polymeric substances) and water. Some components of biofilm EPS like cellulose, colonic acid and O antigen capsule are crucial for S. Typhi persistence and biofilm development (Crawford et al., 2008; Prouty

Curr. Issues Mol. Biol. Vol. 25and Gunn, 2003; Prouty et al., 2002). Some studies show that bile present in the gallbladder induces the production of O-antigen, which facilitates S. Typhi biofilm formation on human gallstones (Crawford et al., 2008; Hall-Stoodley and Stoodley, 2009).

# Persistence in immune cells

The study of the molecular mechanisms underlying persistent infections is limited by the lack of appropriate animal models that resist challenges using virulent strains of Salmonella. A suitable mice model was established in 2004 to study S. Typhimurium persistent infections (Monack et al., 2004b). In this model, virulent S. Typhimurium strains cause long-term chronic infections. The persistent bacteria can then be monitored within macrophages in the MSNs up to 1 year post inoculation (Monack et al., 2004b). In addition to that, *in vitro* assays showed that hemophagocytic macrophages might represent a survival niche for Salmonella (Nix et al., 2007). Another study showed that persistent bacteria are sequestered within macrophages in systemic tissues, where they do not replicate and seem to be in a dormant-like state. This supports the idea of macrophages as a reservoir for persistent bacteria (Helaine et al., 2010). Another study published recently showed that S. Typhimurium preferentially associates with anti-inflammatory/M2 macrophages at later stages of infection in mice (Eisele et al., 2013). This subset of macrophages does not express many of the defence mechanisms needed to eliminate invading microbes. Moreover, there are pathogens that have developed strategies to induce polarization of cells towards the M2 phenotype as a virulence mechanism (Jensen et al., 2011).

In addition to macrophages, it has been shown that Salmonella remains for a long period of time within plasma cells, bone marrow B cell precursors, and all B cell subsets from the spleen (Castro-Eguiluz et al., 2009; López-Medina et al., 2015a). The ability of Salmonella to persist within B cells suggests that it developed a CD8⁺ T-cell response evasion mechanism. In fact, Salmonella infection results in the enhanced expression of PD-L1 in B cells (López-Medina et al., 2015a). PD-L1 is a member of the programmed death-1 (PD-1)/programmed death-ligands (PD-Ls) pathway that can turn off or reduce TCR signalling by recruiting phosphatases like SHP-1 and SHP-2 (Chemnitz et al., 2004; Freeman et al., 2000; Sheppard et al., 2004). The activation of the PD-1:PD-Ls axis leads to impairment of CD8 T cells during Salmonella infection (López-Medina et al., 2015b).

# T3SS effectors and persistence

In order to shed light on the molecular mechanisms involved in persistence, Monack's group performed a genome-wide screen to identify Salmonella genes required for persistent infection of the mice (Lawley et al., 2006). The screen revealed that SPI1 was necessary to maintain a persistent infection for at least 1 month post challenge. The authors confirmed that SPI1 effectors SipB, SipC, and SipD, which have a role on invasion or translocation of other effectors, contribute to establish a persistent infection. Similarly, another study showed that SPI1 is required for persistently infected mice to transmit S. Typhimurium to naive cage-mates (Lawley et al., 2008). Data from the genome screen mentioned above also showed that the SPI2 effector SseI was required for maintaining a long-term systemic infection. A follow-up study showed that SseI modulates macrophages and DCs by directly binding to IQGAP1, an important regulator of host cell migration (McLaughlin et al., 2009). So, SseI blocks somehow the migration of macrophages and DCs interfering with the host immune system to clear systemic bacteria.

Curr. Issues Mol. Biol. Vol. 25
Other elements that have been shown to contribute to persistent Salmonella infection are those that protect against host-derived antimicrobial peptides. Examples of these factors are PgtE, a modifier of the bacterial outer surface membrane (Lawley et al., 2006), and Mig-14, which provides resistance against anti-microbial peptides like VirK, RcsC, and YdeI (Erickson and Detweiler, 2006), both of them regulated by the PhoP/PhoQ system (Bader et al., 2005).

# Persistence and metabolism

Recently, a pioneering study described how bacteria modulate the host cell metabolism to create an appropriate environment for long-term infection (Eisele et al., 2013). Firstly, the authors observed that PPARδ, which is a transcriptional factor involved in fatty acid metabolism, is up-regulated in Salmonella infected M2 macrophages. This situation produces a shift in the metabolic state of the cell that leads to an increase in the level of glucose available to bacteria. Moreover, the authors show that pharmacological activation of PPARδ increases glucose availability and enhances bacterial replication in macrophages, while Salmonella fails to persist in macrophage lines lacking PPARδ. All together, these data support the idea that Salmonella has evolved to survive long periods of time in M2 macrophages by harnessing the unique metabolism of these cells (Eisele et al., 2013).

Another relevant topic concerns genetic adaptation of bacteria during a persistent infection. A recent study describes the acquisition of adaptive mutations that facilitate persistence and survival in the host over the course of an infection (Søndberg and Jelsbak, 2016). In this work, the authors used a model of mice infected chronically with S. Typhimurium. They observed that bacteria acquire distinct single nucleotide polymorphisms (SNPs) in known regulators of metabolic and virulence genes. One such difference, the kdgR-SNP was confirmed to confer selective advantage during chronic infections and constitutes a true patho-adaptive mutation. Thus, the results provide evidence for rapid genetic adaptation to the host of S. Typhimurium during persistent infection (Søndberg and Jelsbak, 2016).

A non-mutational way for Salmonella to cause persistent infections even after antibiotic exposure is through the production of persister cells. These are rare cells, produced by all bacterial populations, that transiently become multidrug tolerant. It is thought this could be due to a state of low metabolic activity that leads to dormant or slow-growing cells (Bigger, 1944; Helaine and Kugelberg, 2014; Maisonneuve and Gerdes, 2014). Salmonella persisters are part of a non-replicating population formed immediately after uptake by macrophages and are induced by vacuolar acidification and nutritional deprivation (Helaine et al., 2014). Furthermore, the majority of the 14 toxin–antitoxin (TA) modules contributed to intracellular persister formation. A more comprehensive analysis of S. Typhimurium TA modules identified multiple toxins with anti-proliferative activity. A selected group of the corresponding TA modules were important for survival of intracellular bacteria inside fibroblasts (Lobato-Márquez et al., 2015). One of these toxins, TacT, is an acetyltransferase that blocks the primary amine group of amino acids on charged tRNA molecules, thereby inhibiting translation and promoting persister formation (Cheverton et al., 2016).

# Transmission

During systemic salmonellosis, the life cycle completed when the pathogenic bacteria leave the former host to infect a naive one. This highlights the main issue with persistent infections: the constant shedding of virulent bacteria on faeces allows for propagation and maintenance
within a population. In a chronic infection scenario, Salmonella can persist in MLNs, bone marrow, and gallbladder, and periodically discharge Salmonella from the gallbladder to the small intestine in bile. The molecular mechanisms that control transmission of bacteria from host reservoirs are poorly understood. However, an interesting study proposed that host transmission of S. Typhimurium is controlled by virulence factors and indigenous intestinal microbiota. The authors described a model in which persistently infected mice provide a natural mode of S. Typhimurium transmission. In this context, a subgroup of mice (30%), called 'supershedders' was found to shed high levels of S. Typhimurium in their stools, leading to rapid transmission of the infection to naive hosts (Lawley et al., 2008). The authors showed that the development of the supershedder phenotype depends on SPI1 and SPI2. Moreover, treatment with antibiotics induced the supershedder state in mice, suggesting that the indigenous intestinal microbiota has a role in controlling this phenotype.

---

### Concluding remarks

We have described recent advances concerning the strategies used by Salmonella to evade both innate and adaptive immune responses. Most of these evasion mechanisms are directly related to T3SS1 and 2 and to the different effectors injected by them (Table 6.1). In the last two decades more than 40 effectors have been identified, and the biochemical activity and host targets for some of them have been successfully characterized. However, the precise role of many of them and their contribution to the relationship between Salmonella and its hosts at the cellular and systemic level remain unknown. A better understanding of the specific functions of these effector proteins will allow us to unravel the complex network of action of these molecules and to decipher the underlying mechanisms that enable Salmonella to evade the host immune system.

For instance, we have to define the role of T3SS effectors in establishing different intracellular lifestyles and decipher which signals enable Salmonella to survive intracellularly versus proliferate and escape, depending on the cellular type.

Another challenge is to improve our understanding of Salmonella control of the fusion of SCV to the lysosome. The identification of new and appropriate SCV markers and the development of new imaging technologies will improve the detection of intracellular Salmonella and ultimately help us to answer these questions.

One of the most significant challenges concerns host specificity achieved by different Salmonella serovars. There is an obvious shortage of appropriate models. For example, there is no suitable animal model for the typhoid human pathogen S. Typhi. Therefore, in vivo studies for this serovar are deficient. Researchers tend to use the S. Typhimurium-infected mouse model to study systemic typhoid-like disease. However, extrapolation of results obtained in S. Typhimurium to S. Typhi is not always possible. The development of new animal and cellular models are slowly helping us surmount this lack of appropriate tools. One example is the recent work of Fresnay et al., who established a controlled human infection model with wt S. Typhi using cutting-edge multichromatic flow cytometry to analyse the pre-challenge immunological status and its correlation with the subsequent clinical outcome (Fresnay et al., 2016). Specifically, the authors investigated the relationship between S. Typhi-specific CD8⁺ T-cell responses before exposure to wt S. Typhi and clinical outcome, i.e. whether the participants who were challenged developed disease or not. They observed higher baseline levels of multi functional S. Typhi-specific CD8⁺ T in patients who did not

Curr. Issues Mol. Biol. Vol. 25
Table 6.1 Role of T3SS effectors in Salmonella evasion of the immune system

| Effector | T3SS | Interaction with the immune system | References |
|----------|------|------------------------------------|------------|
| AvrA     | 1    | Inhibits NF-κB and JNK pro-inflammatory pathways | Collier-Hyams et al. (2002), Jones et al. (2008b) |
| CigR     | 2    | Involved in biofilm formation | Yin et al. (2016) |
| GogA     | 2    | Inhibits inflammation by targeting NF-κB signalling | Sun et al. (2016) |
| GogB     | 2    | Limits inflammation preventing NF-κB | Pilar et al. (2012) |
| GtgA     | 2    | Inhibits inflammation by targeting NF-κB signalling | Sun et al. (2016) |
| GtgE     | 1/2  | Counters a Rab32-dependent host defence pathway | Spanò et al. (2016) |
| PipA     |      | Inhibits inflammation by targeting NF-κB signalling | Sun et al. (2016) |
| PipB     | 2    | Role in iNOS production in chicken oviduct epithelial cells | Li et al. (2009) |
| PrgJ     | ?    | Activates caspase-1 through NLRC4 | Miao et al. (2010) |
| PipB2    | 1/2  | Inhibits dendritic cell migration | McLaughlin et al. (2014) |
| SifA     | 2    | Inhibits dendritic cell migration | McLaughlin et al. (2014) |
| SipA/SspA | 1    | Elicits accumulation of PERP to the apical surface of colonic epithelial cells that leads to inflammatory responses | Hallstrom et al. (2015) |
| SipB     | 1    | Contributes to activation and release of IL-18 | Dreher et al. (2002) |
| SipC/SspC | 1    | Elicits accumulation of PERP to the apical surface of colonic epithelial cells that leads to inflammatory responses | Hallstrom and McCormick (2016) |
| SipD     | 1    | Triggers cell death in macrophages | Arizmendi et al. (2016) |
| SlrP     | 1/2  | Inhibits dendritic cell migration | McLaughlin et al. (2014) |
| SopA     | 1    | Stimulates inflammation targeting TRIM56 and TRIM65 | Kamanova et al. (2016) |
| SopB     | 1    | Inhibits production of mitochondrial superoxide ROS Activates MAPK and NF-κB signalling through stimulation of Rho-family GTPases | Bruno et al. (2009), Ruan et al. (2016) |
| SopD     | 1/2? | Contributes to PMN migration and fluid secretion in bovine intestine | Zhang et al. (2002) |
| SopD2    | 2    | Counters a Rab32-dependent host defence pathway | Spanò et al. (2016) |
| SopE     | 1    | Induces caspase-1 dependent pro-inflammatory responses. Activates MAPK and NF-κB signalling through stimulation of Rho-family GTPases | Bruno et al. (2009), Hoffmann et al. (2010), Müller et al. (2009) |
| SopE2    | 1    | Activates MAPK and NF-κB signalling through stimulation of Rho-family GTPases | Bruno et al. (2009) |
| SpiC/SsaB | 2    | Inhibits dendritic cell migration | McLaughlin et al. (2014) |
| SptP     | 1    | Inhibits p38MAPK signalling. Suppresses degranulation of local mast cells | Choi et al. (2013), Tenor et al. (2004) |

Table 6.1 Continued

| Effector | T3SS | Interaction with the immune system                                                                 | References                                                                 |
|----------|------|-----------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
| SpvB     | 2    | Contributes to gut inflammation through a T3SS2-dependent pathway                                | Käppeli et al. (2011)                                                      |
| SpvC     | 1/2  | Anti-inflammatory in mice. Interferes with immunity in plants                                   | Haneda et al. (2012); Neumann et al. (2014)                                 |
| SpvD     | 1/2  | Inhibits nuclear transport of NF-κB p65                                                        | Rolhion et al. (2016)                                                     |
| SseF     | 2    | Inhibits dendritic cell migration                                                             | McLaughlin et al. (2014)                                                  |
| SseI/SrfH | 2    | Modulates cell migration of macrophages and dendritic cells                                    | McLaughlin et al. (2009), Worley et al. (2006)                             |
| SseK1    | 1/2  | Interferes with NF-κB activation                                                              | Li et al. (2013)                                                          |
| SseK3    | 2    | Interferes with NF-κB activation                                                              | Yang et al. (2015)                                                        |
| SseL     | 2    | Inhibits autophagy                                                                            | Mesquita et al. (2012)                                                    |
| SspH1    | 1/2  | Inhibits NF-κB-dependent gene expression                                                      | Haraga and Miller (2003), Haraga et al. (2006)                             |
| SspH2    | 2    | Enhances SG1-dependent NLR-mediated immunity; inhibits dendritic cell migration                 | Bhavsar et al. (2013), McLaughlin et al. (2014)                           |
| SteA     | 1/2  | Represses genes involved in immune responses                                                   | Cardenal-Muñoz et al. (2014)                                              |
| SteB     |      | Biofilm formation                                                                             | Dong et al. (2008), McGhee et al. (2009)                                  |

develop the disease. Their observations also indicate that S. Typhi-specific CD8⁺T migrated not only to mucosal sites, but also to secondary lymphoid tissues. These localized events, delay disease onset. They also suggested that co-production of MIP-1β with other cytokines is a key component in protection against S. Typhi (Fresnay et al., 2016). This type of studies is key to identify an early selection of novel vaccine candidates for further evaluation in clinical trials. These models are also necessary to understand the role of certain T3SS effectors that may only be relevant in certain cell types or animal species.

Advances in techniques for single cell analysis allow us to study bacterial subpopulations that have different expression profiles of effectors and other virulence factors.

Finally, a more precise understanding of the interactions between Salmonella and the host immune system could help us design new approaches to modify the progression of Salmonella infections, strengthening key components of the immune response.

### Acknowledgements

We thank Sebastián Aguilar for useful discussions and critical reading of the manuscript. Work in the laboratory of FR-M is supported by Ministerio de Economía y Competitividad of Spain and the European Regional Development Fund (grants SAF 2013-46229-R and SAF 2016-75365-R).

References

Aachoui, Y., Leaf, I.A., Hagar, J.A., Fontana, M.F., Campos, C.G., Zak, D.E., Tan, M.H., Cotter, P.A., Vance, R.E., Aderem, A., *et al.* (2013). Caspase-11 protects against bacteria that escape the vacuole. Science 339, 975–978. https://doi.org/10.1126/science.1230751.

Alaniz, R.C., Cummings, L.A., Bergman, M.A., Rassoulian-Barrett, S.L., and Cookson, B.T. (2006). Salmonella Typhimurium coordinately regulates FliC location and reduces dendritic cell activation and antigen presentation to CD4+ T cells. J. Immunol. 177, 3983–3993.

Albaghdadi, H., Robinson, N., Finlay, B., Krishnan, L., and Sad, S. (2009). Selectively reduced intracellular proliferation of Salmonella enterica serovar Typhimurium within APCs limits antigen presentation and development of a rapid CD8 T cell response. J. Immunol. 183, 3778–3787.

Arabyan, N., Park, D., Foutouhi, S., Weis, A.M., Huang, B.C., Williams, C.C., Desai, P., Shah, J., Jeannotte, R., Kong, N., *et al.* (2016). Salmonella degrades the host glycocalyx leading to altered infection and glycan remodeling. Sci. Rep. 6, 29525. https://doi.org/10.1038/srep29525.

Arizmendi, O., Picking, W.D., and Picking, W.L. (2016). Macrophage apoptosis triggered by IpaD from Shigella flexneri. Infect. Immun. 84, 1857–1865. https://doi.org/10.1128/IAI.01483-15.

Audia, J.P., Webb, C.C., and Foster, J.W. (2001). Breaking through the acid barrier: an orchestrated response to proton stress by enteric bacteria. Int. J. Med. Microbiol. 291, 97–106.

Bäckhed, F., Ley, R.E., Sonnenburg, J.L., Peterson, D.A., and Gordon, J.I. (2005). Host-bacterial mutualism in the human intestine. Science 307, 1915–1920.

Bader, M.W., Navarre, W.W., Shiau, W., Nikaido, H., Frye, J.G., McClelland, M., Fang, F.C., and Miller, S.I. (2003). Regulation of Salmonella Typhimurium virulence gene expression by cationic antimicrobial peptides. Mol. Microbiol. 50, 219–230.

Bader, M.W., Sanowar, S., Daley, M.E., Schneider, A.R., Cho, U., Xu, W., Klevit, R.E., Le Moual, H., and Miller, S.I. (2005). Recognition of antimicrobial peptides by a bacterial sensor kinase. Cell 122, 461–472.

Baisón-Olmo, F., Galindo-Moreno, M., and Ramos-Morales, F. (2015). Host cell type-dependent translocation and PhoP-mediated positive regulation of the effector SseK1 of Salmonella enterica. Front. Microbiol. 6, 396.

Barr, T.A., Brown, S., Mastroeni, P., and Gray, D. (2010). TLR and B cell receptor signals to B cells differentially program primary and memory Th1 responses to Salmonella enterica. J. Immunol. 185, 2783–2789. https://doi.org/10.4049/jimmunol.1001431.

Baucheron, S., Nishino, K., Monchaux, I., Canepa, S., Maurel, M.-C., Coste, F., Roussel, A., Cloeckaert, A., and Giraud, E. (2014). Bile-mediated activation of the acrAB and tolC multidrug efflux genes occurs mainly through transcriptional derepression of ramA in Salmonella enterica serovar Typhimurium. J. Antimicrob. Chemother. 69, 2400–2406.

Begley, M., Gahan, C.G., and Hill, C. (2005). The interaction between bacteria and bile. FEMS Microbiol. Rev. 29, 625–651.

Beuzón, C.R., Méresse, S., Unsworth, K.E., Ruíz-Albert, J., Garvis, S., Waterman, S.R., Ryder, T.A., Boucrot, E., and Holden, D.W. (2000). Salmonella maintains the integrity of its intracellular vacuole through the action of SifA. EMBO J. 19, 3235–3249. https://doi.org/10.1093/emboj/19.13.3235.

Bhavsar, A.P., Brown, N.F., Stoepel, J., Wiermer, M., Martin, D.D., Hsu, K.J., Imami, K., Ross, C.J., Hayden, M.R., Foster, L.J., *et al.* (2013). The Salmonella type III effector SspH2 specifically exploits the NLR co-chaperone activity of SGT1 to subvert immunity. PLOS Pathog. 9, e1003518. https://doi.org/10.1371/journal.ppat.1003518.

Bigger, J. (1944). Treatment of staphylococcal infections with penicillin by intermittent sterilisation. Lancet 244, 497–500.

Birmingham, C.L., and Brumell, J.H. (2006). Autophagy recognizes intracellular Salmonella enterica serovar Typhimurium in damaged vacuoles. Autophagy 2, 156–158.

Birmingham, C.L., Smith, A.C., Bakowski, M.A., Yoshimori, T., and Brumell, J.H. (2006). Autophagy controls Salmonella infection in response to damage to the Salmonella-containing vacuole. J. Biol. Chem. 281, 11374–11383.

Bishop, R.E., Gibbons, H.S., Guina, T., Trent, M.S., Miller, S.I., and Raetz, C.R. (2000). Transfer of palmitate from phospholipids to lipid A in outer membranes of gram-negative bacteria. EMBO J. 19, 5071–5080. https://doi.org/10.1093/emboj/19.19.5071.

Blancou, P., Tardif, V., Simon, T., Rémy, S., Carreño, L., Kalergis, A., and Anegon, I. (2011). Immunoregulatory properties of heme oxygenase-1. Methods Mol. Biol. 677, 247–268. https://doi.org/10.1007/978-1-60761-869-0_18.

Boucrot, E., Beuzón, C.R., Holden, D.W., Gorvel, J.P., and Méresse, S. (2003). Salmonella Typhimurium SifA effector protein requires its membrane-anchoring C-terminal hexapeptide for its biological function. J. Biol. Chem. 278, 14196–14202.

Brodsky, I.E., Ghori, N., Falkow, S., and Monack, D. (2005). Mig-14 is an inner membrane-associated protein that promotes Salmonella Typhimurium resistance to CRAMP, survival within activated macrophages and persistent infection. Mol. Microbiol. 55, 954–972.

Broz, P., and Monack, D.M. (2013a). Noncanonical inflammasomes: caspase-11 activation and effector mechanisms. PLOS Pathog. 9, e1003144. https://doi.org/10.1371/journal.ppat.1003144.

Broz, P., and Monack, D.M. (2013b). Newly described pattern recognition receptors team up against intracellular pathogens. Nat. Rev. Immunol. 13, 551–565. https://doi.org/10.1038/nri3479.

Broz, P., Newton, K., Lamkanfi, M., Mariathasan, S., Dixit, V.M., and Monack, D.M. (2010). Redundant roles for inflammasome receptors NLRP3 and NLRC4 in host defense against Salmonella. J. Exp. Med. 207, 1745–1755. https://doi.org/10.1084/jem.20100257.

Broz, P., Ruby, T., Belhocine, K., Bouley, D.M., Kayagaki, N., Dixit, V.M., and Monack, D.M. (2012). Caspase-11 increases susceptibility to Salmonella infection in the absence of caspase-1. Nature 490, 288–291. https://doi.org/10.1038/nature11419.

Bruno, V.M., Hannemann, S., Lara-Tejero, M., Flavell, R.A., Kleinstein, S.H., Galán, J.E., H., S., Bruno, V.M., and Hannemann, S. (2009). Salmonella Typhimurium type III secretion effectors stimulate innate immune responses in cultured epithelial cells. PLOS Pathog. 5, e1000538.

Bueno, S.M., González, P.A., Carreño, L.J., Tobar, J.A., Mora, G.C., Pereda, C.J., Salazar-Onfray, F., and Kalergis, A.M. (2008). The capacity of Salmonella to survive inside dendritic cells and prevent antigen presentation to T cells is host specific. Immunology 124, 522–533.

Bueno, S.M., Wozniak, A., Leiva, E.D., Riquelme, S.A., Carreño, L.J., Hardt, W.-D., Riedel, C.A., and Kalergis, A.M. (2010). Salmonella pathogenicity island 1 differentially modulates bacterial entry to dendritic and non-phagocytic cells. Immunology 130, 273–287.

Cardenal-Muñoz, E., Gutiérrez, G., and Ramos-Morales, F. (2014). Global impact of Salmonella type III secretion effector SteA on host cells. Biochem. Biophys. Res. Commun. 449, 419–424. https://doi.org/10.1016/j.bbrc.2014.05.056.

Casson, C.N., Yu, J., Reyes, V.M., Taschuk, F.O., Yadav, A., Copenhagen, A.M., Nguyen, H.T., Collman, R.G., and Shin, S. (2015). Human caspase-4 mediates noncanonical inflammasome activation against gram-negative bacterial pathogens. Proc. Natl. Acad. Sci. U. S. A. 112, 6688–6693.

Castro-Eguiluz, D., Pelayo, R., Rosales-Garcia, V., Rosales-Reyes, R., Alpuche-Aranda, C., and Ortiz-Navarrete, V. (2009). B cell precursors are targets for Salmonella infection. Microb. Pathog. 47, 52–56. https://doi.org/10.1016/j.micpath.2009.04.005.

Chemnitz, J.M., Parry, R.V., Nichols, K.E., June, C.H., and Riley, J.L. (2004). SHP-1 and SHP-2 associate with immunoreceptor tyrosine-based switch motif of programmed death 1 upon primary human T cell stimulation, but only receptor ligation prevents T cell activation. J. Immunol. 173, 945–954.

Cheverton, A.M., Gollan, B., Przydadz, M., Wong, C.T., Mylona, A., Hare, S.A., and Helaine, S. (2016). A Salmonella toxin promotes persister formation through acetylation of tRNA. Mol. Cell 63, 86–96. https://doi.org/10.1016/j.molcel.2016.05.002.

Choi, H.W., Brooking-Dixon, R., Neupane, S., Lee, C.-J., Miao, E.A., Staats, H.F., and Abraham, S.N. (2013). Salmonella Typhimurium impedes innate immunity with a mast-cell-suppressing protein tyrosine phosphatase, SptP. Immunity 39, 1108–1120.

Chu, Y., Gao, S., Wang, T., Yan, J., Xu, G., Li, Y., Niu, H., Huang, R., and Wu, S. (2016). A novel contribution of *spvB* to pathogenesis of Salmonella Typhimurium by inhibiting autophagy in host cells. Oncotarget 7, 8295–8309. https://doi.org/10.18632/oncotarget.6989.

Chung, S.W., Liu, X., Macias, A.A., Baron, R.M., and Perrella, M.A. (2008). Heme oxygenase-1-derived carbon monoxide enhances the host defense response to microbial sepsis in mice. J. Clin. Invest. 118, 239–247. https://doi.org/10.1172/JCI32730.

Coleman, R. (1987). Biochemistry of bile secretion. Biochem. J. 244, 249–261.

Collier-Hyams, L.S., Zeng, H., Sun, J., Tomlinson, A.D., Bao, Z.Q., Chen, H., Madara, J.L., Orth, K., and Neish, A.S. (2002). Cutting edge: Salmonella AvrA effector inhibits the key proinflammatory, anti-apoptotic NF-kappa B pathway. J. Immunol. 169, 2846–2850.

Costerton, J.W., Stewart, P.S., and Greenberg, E.P. (1999). Bacterial biofilms: a common cause of persistent infections. Science 284, 1318–1322.

Crawford, R.W., Gibson, D.L., Kay, W.W., and Gunn, J.S. (2008). Identification of a bile-Induced exopolysaccharide required for Salmonella biofilm formation on gallstone surfaces. Infect. Immun. 76, 5341–5349.

Crawford, R.W., Rosales-Reyes, R., Ramírez-Aguilar, Mde. L., Chapa-Azuela, O., Alpuche-Aranda, C., and Gunn, J.S. (2010). Gallstones play a significant role in Salmonella spp. gallbladder colonization and carriage. Proc. Natl. Acad. Sci. U.S.A. 107, 4353–4358. https://doi.org/10.1073/pnas.1000862107.

Crawford, R.W., Keestra, A.M., Winter, S.E., Xavier, M.N., Tsolis, R.M., Tolstikov, V., and Bäumler, A.J. (2012). Very long O-antigen chains enhance fitness during Salmonella-induced colitis by increasing bile resistance. PLOS Pathog. 8, e1002918. https://doi.org/10.1371/journal.ppat.1002918.

Crawford, R.W., Wangdi, T., Spees, A.M., Xavier, M.N., Tsolis, R.M., and Bäumler, A.J. (2013). Loss of very-long O-antigen chains optimizes capsule-mediated immune evasion by Salmonella enterica serovar Typhi. MBio 4, 1–8.

Cummings, L.A., Wilkerson, W.D., Bergsbaken, T., and Cookson, B.T. (2006). In vivo, fliC expression by Salmonella enterica serovar Typhimurium is heterogeneous, regulated by ClpX, and anatomically restricted. Mol. Microbiol. 61, 795–809.

Cunningham, A.F., Gaspal, F., Serre, K., Mohr, E., Henderson, I.R., Scott-Tucker, A., Kenny, S.M., Khan, M., Toellner, K.-M., Lane, P.J.L., et al. (2007). Salmonella induces a switched antibody response without germinal centers that impedes the extracellular spread of infection. J. Immunol. 178, 6200–6207.

D’Costa, V.M., Braun, V., Landekic, M., Shi, R., Proteau, A., McDonald, L., Cygler, M., Grinstein, S., and Brumell, J.H. (2015). Salmonella disrupts host endocytic trafficking by SopD2-mediated inhibition of Rab7. Cell Rep. 12, 1508–1518.

Dalebroux, Z.D., Matamouros, S., Whittington, D., Bishop, R.E., and Miller, S.I. (2014). PhoPQ regulates acidic glycerophospholipid content of the Salmonella Typhimurium outer membrane. Proc. Natl. Acad. Sci. U. S. A. 111, 1963–1968.

Delves, P.J., Martin, S.J., Burton, D.R., and Roitt, I.M. (2011). Roitt’s essential immunology.

Detweiler, C.S., Monack, D.M., Brodsky, I.E., Mathew, H., and Falkow, S. (2003). virK, somA and rcsC are important for systemic Salmonella enterica serovar Typhimurium infection and cationic peptide resistance. Mol. Microbiol. 48, 385–400.

Dong, H., Zhang, X., Pan, Z., Peng, D., and Liu, X. (2008). Identification of genes for biofilm formation in a Salmonella enteritidis strain by transposon mutagenesis. Wei Sheng Wu Xue Bao 48, 869–873.

Dreher, D., Kok, M., Obregon, C., Kiama, S.G., Gehr, P., and Nicod, L.P. (2002). Salmonella virulence factor SipB induces activation and release of IL-18 in human dendritic cells. J. Leukoc. Biol. 72, 743–751.

Du, F., and Galán, J.E. (2009). Selective inhibition of type III secretion activated signaling by the Salmonella effector AvrA. PLOS Pathog. 5, e1000595. https://doi.org/10.1371/journal.ppat.1000595.

Eckburg, P.B., Bik, E.M., Bernstein, C.N., Purdom, E., Dethlefsen, L., Sargent, M., Gill, S.R., Nelson, K.E., and Relman, D.A. (2005). Diversity of the human intestinal microbial flora. Science 308, 1635–1638.

Eisele, N.A., Ruby, T., Jacobson, A., Manzanillo, P.S., Cox, J.S., Lam, L., Mukundan, L., Chawla, A., and Monack, D.M. (2013). Salmonella require the fatty acid regulator PPARδ for the establishment of a metabolic environment essential for long-term persistence. Cell Host Microbe 14, 171–182. https://doi.org/10.1016/j.chom.2013.07.010.

Erickson, K.D., and Detweiler, C.S. (2006). The Rcs phosphorelay system is specific to enteric pathogens/commensals and activates ydeI, a gene important for persistent Salmonella infection of mice. Mol. Microbiol. 62, 883–894.

Ertelt, J.M., Johans, T.M., Mysz, M.A., Nanton, M.R., Rowe, J.H., Aguilera, M.N., and Way, S.S. (2011). Selective culling of high avidity antigen-specific CD4+ T cells after virulent Salmonella infection. Immunology 134, 487–497.

Farizano, J.V., Pescaretti, M. de las M., López, F.E., Hsu, F.-F., and Delgado, M.A. (2012). The PmrAB system-inducing conditions control both lipid A remodeling and O-antigen length distribution, influencing the Salmonella Typhimurium-host interactions. J. Biol. Chem. 287, 38778–38789.

Farris, C., Sanowar, S., Bader, M.W., Pfuetzner, R., and Miller, S.I. (2010). Antimicrobial peptides activate the Rcs regulon through the outer membrane lipoprotein RcsF. J. Bacteriol. 192, 4894–4903. https://doi.org/10.1128/JB.00505-10.

Fillatreau, S. (2011). Novel regulatory functions for Toll-like receptor-activated B cells during intracellular bacterial infection. Immunol. Rev. 240, 52–71. https://doi.org/10.1111/j.1600-065X.2010.00991.x.

Fillatreau, S., Gray, D., and Anderton, S.M. (2008). Not always the bad guys: B cells as regulators of autoimmune pathology. Nat. Rev. Immunol. 8, 391–397. https://doi.org/10.1038/nri2315.

Fink, S.L., and Cookson, B.T. (2007). Pyroptosis and host cell death responses during Salmonella infection. Cell. Microbiol. 9, 2562–2570.

Foster, J.W. (1993). The acid tolerance response of Salmonella Typhimurium involves transient synthesis of key acid shock proteins. J. Bacteriol. 175, 1981–1987.

Freeman, G.J., Long, A.J., Iwai, Y., Bourque, K., Chernova, T., Nishimura, H., Fitz, L.J., Malenkovich, N., Okazaki, T., Byrne, M.C., et al. (2000). Engagement of the PD-1 immunoinhibitory receptor by a novel B7 family member leads to negative regulation of lymphocyte activation. J. Exp. Med. 192, 1027–1034.

Fresnay, S., McArthur, M.A., Magder, L., Darton, T.C., Jones, C., Waddington, C.S., Blohmke, C.J., Angus, B., Levine, M.M., Pollard, A.J., et al. (2016). Salmonella Typhi-specific multifunctional CD8+ T cells play a dominant role in protection from typhoid fever in humans. J. Transl. Med. 14, 62. https://doi.org/10.1186/s12967-016-0819-7.

Fu, Y., and Galán, J.E. (1999). A Salmonella protein antagonizes Rac-1 and Cdc42 to mediate host-cell recovery after bacterial invasion. Nature 401, 293–297. https://doi.org/10.1038/45829.

Galán, J.E., and Curtiss, R. (1989). Cloning and molecular characterization of genes whose products allow Salmonella Typhimurium to penetrate tissue culture cells. Proc. Natl. Acad. Sci. U. S.A. 86, 6383–6387.

Garai, P., Gnanadhas, D.P., and Chakravortty, D. (2012). Salmonella enterica serovars Typhimurium and Typhi as model organisms: Revealing paradigm of host-pathogen interactions. Virulence 3, 377–388.

Gerlach, R.G., and Hensel, M. (2007). Salmonella pathogenicity islands in host specificity, host pathogen-interactions and antibiotics resistance of Salmonella enterica. Berliner Und Münchener Tierärztliche Wochenschrift 120, 317–327.

Gomes, L.C., and Dikic, I. (2014). Autophagy in antimicrobial immunity. Mol. Cell 54, 224–233. https://doi.org/10.1016/j.molcel.2014.03.009.

Gonzalez-Escobedo, G., Marshall, J.M., and Gunn, J.S. (2011). Chronic and acute infection of the gall bladder by Salmonella Typhi: understanding the carrier state. Nat. Rev. Microbiol. 9, 9–14. https://doi.org/10.1038/nrmicro2490.

Guina, T., Yi, E.C., Wang, H., Hackett, M., and Miller, S.I. (2000). A PhoP-regulated outer membrane protease of Salmonella enterica serovar Typhimurium promotes resistance to alpha-helical antimicrobial peptides. J. Bacteriol. 182, 4077–4086.

Gunn, J.S. (2000). Mechanisms of bacterial resistance and response to bile. Microbes Infect. 2, 907–913.

Gunn, J.S., Lim, K.B., Krueger, J., Kim, K., Guo, L., Hackett, M., and Miller, S.I. (1998). PmrA-PmrB-regulated genes necessary for 4-aminoarabinose lipid A modification and polymyxin resistance. Mol. Microbiol. 27, 1171–1182.

Guo, L., Lim, K.B., Poduje, C.M., Daniel, M., Gunn, J.S., Hackett, M., and Miller, S.I. (1998). Lipid A acylation and bacterial resistance against vertebrate antimicrobial peptides. Cell 95, 189–198.

Halici, S., Zenk, S.F., Jantsch, J., and Hensel, M. (2008). Functional analysis of the Salmonella pathogenicity island 2-mediated inhibition of antigen presentation in dendritic cells. Infect. Immun. 76, 4924–4933.

Hall-Stoodley, L., and Stoodley, P. (2009). Evolving concepts in biofilm infections. Cell. Microbiol. 11, 1034–1043. https://doi.org/10.1111/j.1462-5822.2009.01323.x.

Hallstrom, K.N., and McCormick, B.A. (2016). The type three secreted effector SipC regulates the trafficking of PERP during Salmonella infection. Gut Microbes 7, 136–145. https://doi.org/10.1080/19490976.2015.1128626.

Hallstrom, K.N., Srikanth, C.V., Agbor, T.A., Dumont, C.M., Peters, K.N., Paraoan, L., Casanova, J.E., Boll, E.J., and McCormick, B.A. (2015). PERP, a host tetraspanning membrane protein, is required for Salmonella-induced inflammation. Cell. Microbiol. 17, 843–859. https://doi.org/10.1111/cmi.12406.

Haneda, T., Ishii, Y., Shimizu, H., Ohshima, K., Ida, N., Danbara, H., and Okada, N. (2012). Salmonella type III effector SpvC, a phosphothreonine lyase, contributes to reduction in inflammatory response during intestinal phase of infection. Cell. Microbiol. 14, 485–499. https://doi.org/10.1111/j.1462-5822.2011.01733.x.

Haraga, A., and Miller, S.I. (2003). A Salmonella enterica serovar Typhimurium translocated leucine-rich repeat effector protein inhibits NF-kappa B-dependent gene expression. Infect. Immun. 71, 4052–4058.

Haraga, A., and Miller, S.I. (2006). A Salmonella type III secretion effector interacts with the mammalian serine/threonine protein kinase PKN1. Cell. Microbiol. 8, 837–846.

Helaine, S., and Kugelberg, E. (2014). Bacterial persisters: formation, eradication, and experimental systems. Trends Microbiol. 22, 417–424. https://doi.org/10.1016/j.tim.2014.03.008.

Helaine, S., Thompson, J.A., Watson, K.G., Liu, M., Boyle, C., and Holden, D.W. (2010). Dynamics of intracellular bacterial replication at the single cell level. Proc. Natl. Acad. Sci. U.S.A. 107, 3746–3751. https://doi.org/10.1073/pnas.1000041107.

Helaine, S., Cheverton, A.M., Watson, K.G., Faure, L.M., Matthews, S.A., and Holden, D.W. (2014). Internalization of Salmonella by macrophages induces formation of nonreplicating persisters. Science 343, 204–208. https://doi.org/10.1126/science.1244705.

Hernandez, L.D., Hueffer, K., Wenk, M.R., and Galán, J.E. (2004). Salmonella modulates vesicular traffic by altering phosphoinositide metabolism. Science 30, 1805–1807.

Hernández, S.B., Cota, I., Ducret, A., Aussel, L., and Casadesús, J. (2012). Adaptation and preadaptation of Salmonella enterica to Bile. PLOS Genet. 8, e1002459. https://doi.org/10.1371/journal.pgen.1002459.

Hernández, S.B., Cava, F., Pucciarelli, M.G., García-Del Portillo, F., de Pedro, M.A., and Casadesús, J. (2015). Bile-induced peptidoglycan remodelling in Salmonella enterica. Environ. Microbiol. 17, 1081–1089. https://doi.org/10.1111/1462-2920.12491.

Herrera, C.M., Hankins, J.V., and Trent, M.S. (2010). Activation of PmrA inhibits LpxT-dependent phosphorylation of lipid A promoting resistance to antimicrobial peptides. Mol. Microbiol. 76, 1444–1460. https://doi.org/10.1111/j.1365-2958.2010.07150.x.

Heyns, K., and Kiessling, G. (1967). Strukturaufklärung des Vi-antigens aus Citrobacter freundii (E. coli) 5396/3. Carbohydr. Res. 3, 340–353.

Hicks, K.G., Delbecq, S.P., Sancho-Vaello, E., Blanc, M.P., Dove, K.K., Prost, L.R., Daley, M.E., Zeth, K., Klevit, R.E., and Miller, S.I. (2015). Acidic pH and divalent cation sensing by PhoQ are dispensable for systemic salmonellae virulence. Elife 4, e06792. https://doi.org/10.7554/eLife.06792.

Hochmann, H., Pust, S., von Figura, G., Aktories, K., and Barth, H. (2006). Salmonella enterica SpvB ADP-ribosylates actin at position arginine-177-characterization of the catalytic domain within the SpvB protein and a comparison to binary clostridial actin-ADP-ribosylating toxins. Biochemistry 45, 1271–1277.

Hoffmann, C., Galle, M., Dilling, S., Käppeli, R., Müller, A.J., Songhet, P., Beyaert, R., and Hardt, W.D. (2010). In macrophages, caspase-1 activation by SopE and the type III secretion system-1 of S. Typhimurium can proceed in the absence of flagellin. PLOS ONE 5, e12477. https://doi.org/10.1371/journal.pone.0012477.

Jantsch, J., Cheminay, C., Chakravortty, D., Lindig, T., Hein, J., and Hensel, M. (2003). Intracellular activities of Salmonella enterica in murine dendritic cells. Cell. Microbiol. 5, 933–945.

Jensen, K.D.C., Wang, Y., Wojno, E.D.T., Shastri, A.J., Hu, K., Cornel, L., Boedec, E., Ong, Y.C., Chien, Y.H., Hunter, C.A., et al. (2011). Toxoplasma polymorphic effectors determine macrophage polarization and intestinal inflammation. Cell Host Microbe 9, 472–483.

Jessen, D.L., Osei-Owusu, P., Toosky, M., Roughead, W., Bradley, D.S., and Nilles, M.L. (2014). Type III secretion needle proteins induce cell signaling and cytokine secretion via Toll-like receptors. Infect. Immun. 82, 2300–2309. https://doi.org/10.1128/IAI.01705-14.

Jones, J.W., Shaffer, S.A., Ernst, R.K., Goodlett, D.R., and Turecek, F. (2008). Determination of pyrophosphorylated forms of lipid A in Gram-negative bacteria using a multivariated mass spectrometric approach. Proc. Natl. Acad. Sci. U.S.A. 105, 12742–12747. https://doi.org/10.1073/pnas.0800445105.

Jones, R.M., Wu, H., Wentworth, C., Luo, L., Collier-Hyams, L., and Neish, A.S. (2008). Salmonella AvrA coordinates suppression of host immune and apoptotic defenses via JNK pathway blockade. Cell Host Microbe 3, 233–244. https://doi.org/10.1016/j.chom.2008.02.016.

Kamanova, J., Sun, H., Lara-Tejero, M., and Galán, J.E. (2016). The Salmonella effector protein SopA modulates innate immune responses by targeting TRIM E3 ligase family members. PLOS Pathog. 12, e1005552. https://doi.org/10.1371/journal.ppat.1005552.

Kaniga, K., Uralil, J., Bliska, J.B., and Galán, J.E. (1996). A secreted protein tyrosine phosphatase with modular effector domains in the bacterial pathogen Salmonella Typhimurium. Mol. Microbiol. 21, 633–641.

Käppeli, R., Kaiser, P., Stecher, B., and Hardt, W.D. (2011). Roles of spvB and spvC in S. Typhimurium colitis via the alternative pathway. Int. J. Med. Microbiol. 301, 117–124. https://doi.org/10.1016/j.ijmm.2010.08.017.

Karaki, K., and Matsubara, Y. (1984). Surgical treatment of chronic biliary typhoid and paratyphoid carriers. Nihon Shokakibyo Gakkai Zasshi 81, 2978–2985.

Kato, A., and Groisman, E.A. (2004). Connecting two-component regulatory systems by a protein that protects a response regulator from dephosphorylation by its cognate sensor. Genes Dev. 18, 2302–2313. https://doi.org/10.1101/gad.1230804.

Keestra, A.M., Winter, M.G., Auburger, J.J., Frässle, S.P., Xavier, M.N., Winter, S.E., Kim, A., Poon, V., Ravesloot, M.M., Waldenmaier, J.F., et al. (2013). Manipulation of small Rho GTPases is a pathogen-induced process detected by NOD1. Nature 496, 233–237. https://doi.org/10.1038/nature12025.

Kenneth, M. (2011). Innate Immunity: The First Lines of Defense. Janeway’s Immunobiol. 37–73.

Khan, C.M. (2014). The dynamic interactions between Salmonella and the microbiota, within the challenging niche of the gastrointestinal tract. Int. Sch. Res. Notices 2014, 846049. https://doi.org/10.1155/2014/846049.

Kim, Y.K., Shin, J.-S., Nahm, M.H., Janeway, C., Medzhitov, R., Lemaitre, B., Nicolas, E., Michaut, L., Reichhart, J., Hoffmann, J., et al. (2016). NOD-Like receptors in infection, immunity, and diseases. Yonsei Med. J. 57, 5.

Knodler, L.A. (2015). *Salmonella enterica*: living a double life in epithelial cells. Curr. Opin. Microbiol. 23, 23–31. https://doi.org/10.1016/j.mib.2014.10.010.

Knodler, L.A., Vallance, B.A., Celli, J., Winfree, S., Hansen, B., Montero, M., and Steele-Mortimer, O. (2010). Dissemination of invasive *Salmonella* via bacterial-induced extrusion of mucosal epithelia. Proc. Natl. Acad. Sci. U.S.A. 107, 17733–17738. https://doi.org/10.1073/pnas.1006098107.

Knodler, L.A., Nair, V., Steele-Mortimer, O., Voth, D., Heinzen, R., Vergne, I., Chua, J., Singh, S., Deretic, V., Celli, J., et al. (2014). Quantitative assessment of cytosolic *Salmonella* in epithelial cells. PLOS ONE 9, e84681.

Koutsoumanis, K.P., and Sofos, J.N. (2004). Comparative acid stress response of *Listeria monocytogenes*, *Escherichia coli* O157:H7 and *Salmonella Typhimurium* after habituation at different pH conditions. Lett. Appl. Microbiol. 38, 321–326.

Kreibich, S., Emmenlauer, M., Fredlund, J., Rämö, P., Münz, C., Dehio, C., Enninga, J., and Hardt, W.D. (2015). Autophagy proteins promote repair of endosomal membranes damaged by the *Salmonella* Type Three Secretion System 1. Cell Host Microbe 18, 527–537. https://doi.org/10.1016/j.chom.2015.10.015.

Kullas, A.L., McClelland, M., Yang, H.J., Tam, J.W., Torres, A., Porwollik, S., Mena, P., McPhee, J.B., Bogomolnaya, L., Andrews-Polymenis, H., et al. (2012). L-asparaginase II produced by *Salmonella Typhimurium* inhibits T cell responses and mediates virulence. Cell Host Microbe 12, 791–798. https://doi.org/10.1016/j.chom.2012.10.018.

Lam, L.H., Monack, D.M., Parry, C., Hien, T., Dougan, G., White, N., Farrar, J., Levin, M., Black, R., Lanata, C., et al. (2014). Intraspecies competition for niches in the distal gut dictate transmission during persistent *Salmonella* infection. PLOS Pathog. 10, e1004527.

Lapaque, N., Hutchinson, J.L., Jones, D.C., Méresse, S., Holden, D.W., Trowsdale, J., and Kelly, A.P. (2009). *Salmonella* regulates polyubiquitination and surface expression of MHC class II antigens. Proc. Natl. Acad. Sci. U.S.A. 106, 14052–14057. https://doi.org/10.1073/pnas.0906735106.

Lawley, T.D., Chan, K., Thompson, L.J., Kim, C.C., Govoni, G.R., and Monack, D.M. (2006). Genome-wide screen for *Salmonella* genes required for long-term systemic infection of the mouse. PLOS Pathog. 2, e11. https://doi.org/10.1371/journal.ppat.0020011.

Lawley, T.D., Bouley, D.M., Hoy, Y.E., Gerke, C., Relman, D.A., and Monack, D.M. (2008). Host transmission of *Salmonella enterica* serovar Typhimurium is controlled by virulence factors and indigenous intestinal microbiota. Infect. Immun. 76, 403–416.

Le Negrate, G., Faustin, B., Welsh, K., Loeffler, M., Krajewska, M., Hasegawa, P., Mukherjee, S., Orth, K., Krajewski, S., Godzik, A., et al. (2008). *Salmonella* secreted factor L deubiquitinase of *Salmonella Typhimurium* inhibits NF-kappaB, suppresses IkappaBalpha ubiquitination and modulates innate immune responses. J. Immunol. 180, 5045–5056.

Lee, T.S., and Chau, L.Y. (2002). Heme oxygenase-1 mediates the anti-inflammatory effect of interleukin-10 in mice. Nat. Med. 8, 240–246. https://doi.org/10.1038/nm0302-240.

Lee, H., Hsu, F.-F., Turk, J., and Groisman, E.A. (2004). The PmrA-regulated *pmrC* gene mediates phosphoethanolamine modification of lipid A and polymyxin resistance in *Salmonella enterica*. J. Bacteriol. 186, 4124–4133.

Lei, L., Wang, W., Xia, C., and Liu, F. (2016). *Salmonella* virulence factor SsrAB Regulated factor modulates inflammatory responses by enhancing the activation of NF-kB signaling pathway. J. Immunol. 196, 792–802.

Lesnick, M.L., Reiner, N.E., Fierer, J., and Guiney, D.G. (2001). The *Salmonella spvB* virulence gene encodes an enzyme that ADP-ribosylates actin and destabilizes the cytoskeleton of eukaryotic cells. Mol. Microbiol. 39, 1464–1470.

Levine, M.M., Black, R.E., and Lanata, C. (1982). Precise estimation of the numbers of chronic carriers of *Salmonella Typhi* in Santiago, Chile, an endemic area. J. Infect. Dis. 146, 724–726.

Lewinsohn, D.A., Gold, M.C., and Lewinsohn, D.M. (2011). Views of immunology: effector T cells. Immunol. Rev. 240, 25–39. https://doi.org/10.1111/j.1600-065X.2010.00997.x.

Li, S., Zhang, M.Z., Yan, L., Lillehoj, H., Pace, L.W., and Zhang, S. (2009). Induction of CXC chemokine messenger-RNA expression in chicken oviduct epithelial cells by Salmonella enterica serovar Enteritidis via the type three secretion system-1. Avian Dis. 53, 396–404.

Li, S., Zhang, L., Yao, Q., Li, L., Dong, N., Rong, J., Gao, W., Ding, X., Sun, L., Chen, X., et al. (2013). Pathogen blocks host death receptor signalling by arginine GlcNAcylation of death domains. Nature 501, 242–246. https://doi.org/10.1038/nature12436.

Li, Y., Wang, T., Gao, S., Xu, G., Niu, H., Huang, R., and Wu, S. (2016). Salmonella plasmid virulence gene *spvB* enhances bacterial virulence by inhibiting autophagy in a zebrafish infection model. Fish Shellfish Immunol. 49, 252–259.

Liévin-Le Moal, V., and Servin, A.L. (2006). The front line of enteric host defense against unwelcome intrusion of harmful microorganisms: mucins, antimicrobial peptides, and microbiota. Clin. Microbiol. Rev. 19, 315–337.

Lin, J., Lee, I.S., Frey, J., Slonczewski, J.L., and Foster, J.W. (1995). Comparative analysis of extreme acid survival in Salmonella Typhimurium, Shigella flexneri, and Escherichia coli. J. Bacteriol. 177, 4097–4104.

Lin, S.L., Le, T.X., and Cowen, D.S. (2003). SptP, a Salmonella Typhimurium type III-secreted protein, inhibits the mitogen-activated protein kinase pathway by inhibiting Raf activation. Cell. Microbiol. 5, 267–275.

Lobato-Márquez, D., Moreno-Córdoba, I., Figueroa, V., Díaz-Orejas, R., and García-del Portillo, F. (2015). Distinct type I and type II toxin-antitoxin modules control Salmonella lifestyle inside eukaryotic cells. Sci. Rep. 5, 9374.

Looney, R.J., and Steigbigel, R.T. (1986). Role of the Vi antigen of Salmonella Typhi in resistance to host defense in vitro. J. Lab. Clin. Med. 108, 506–516.

Lopez, C.A., Winter, S.E., Rivera-Chávez, F., Xavier, M.N., Poon, V., Nuccio, S.-P., Tsolis, R.M., and Bäumler, A.J. (2012). Phage-mediated acquisition of a type III secreted effector protein boosts growth of Salmonella by nitrate respiration. MBio 3.

López-Medina, M., Pérez-López, A., Alpuche-Aranda, C., and Ortiz-Navarrete, V. (2015a). Salmonella induces PD-L1 expression in B cells. Immunol. Lett. 167, 131–140. https://doi.org/10.1016/j.imlet.2015.08.004.

López-Medina, M., Carrillo-Martín, I., Leyva-Rangel, J., Alpuche-Aranda, C., and Ortiz-Navarrete, V. (2015b). Salmonella impairs CD8 T cell response through PD-1: PD-L axis. Immunobiology 220, 1369–1380. https://doi.org/10.1016/j.jimbio.2015.07.005.

López-Montero, N., Ramos-Marquès, E., Risco, C., and García-Del Portillo, F. (2016). Intracellular Salmonella induces aggrephagy of host endomembranes in persistent infections. Autophagy 12, 1–16.

Luu, R.A., Gurnani, K., Dudani, R., Kammara, R., van Faassen, H., Sirard, J.C., Krishnan, L., and Sad, S. (2006). Delayed expansion and contraction of CD8+ T cell response during infection with virulent Salmonella Typhimurium. J. Immunol. 177, 1516–1525.

Ma, D., Cook, D.N., Alberti, M., Pon, N.G., Nikaido, H., and Hearst, J.E. (1993). Molecular cloning and characterization of *acrA* and *acrE* genes of Escherichia coli. J. Bacteriol. 175, 6299–6313.

Ma, D., Cook, D.N., Alberti, M., Pon, N.G., Nikaido, H., and Hearst, J.E. (1995). Genes *acrA* and *acrB* encode a stress-induced efflux system of Escherichia coli. Mol. Microbiol. 16, 45–55.

Maier, L., Vyas, R., Cordova, C.D., Lindsay, H., Schmidt, T.S., Brugiroux, S., Periaswamy, B., Bauer, R., Sturm, A., Schreiber, F., et al. (2013). Microbiota-derived hydrogen fuels Salmonella Typhimurium invasion of the gut ecosystem. Cell Host Microbe 14, 641–651. https://doi.org/10.1016/j.chom.2013.11.002.

Maisonneuve, E., and Gerdes, K. (2014). Molecular mechanisms underlying bacterial persisters. Cell 157, 539–548. https://doi.org/10.1016/j.cell.2014.02.050.

Malik-Kale, P., Winfree, S., Steele-Mortimer, O., Marlovits, T., Stebbins, C., Steele-Mortimer, O., Beuzon, C., Salcedo, S., Holden, D., Birmingham, C., et al. (2012). The bimodal lifestyle of intracellular Salmonella in epithelial cells: replication in the cytosol obscures defects in vacuolar replication. PLOS ONE 7, e38732.

Mallo, G.V., Espina, M., Smith, A.C., Terebiznik, M.R., Alemán, A., Finlay, B.B., Rameh, L.E., Grinstein, S., and Brumell, J.H. (2008). SopB promotes phosphatidylinositol 3-phosphate formation on Salmonella vacuoles by recruiting Rab5 and Vps34. J. Cell Biol. 182, 741–752. https://doi.org/10.1083/jcb.200804131.

Marcobal, A., Southwick, A.M., Earle, K.A., and Sonnenburg, J.L. (2013). A refined palate: bacterial consumption of host glycans in the gut. Glycobiology 23, 1038–1046. https://doi.org/10.1093/glycob/cwt040.

Margarit, S.M., Davidson, W., Frego, L., and Stebbins, C.E. (2006). A steric antagonism of actin polymerization by a Salmonella virulence protein. Structure 14, 1219–1229.

Masschalck, B., and Michiels, C.W. (2003). Antimicrobial properties of lysozyme in relation to foodborne vegetative bacteria. Crit. Rev. Microbiol. 29, 191–214. https://doi.org/10.1080/713610448.

Mastroeni, P., Villarreal-Ramos, B., and Hormaeche, C.E. (1993). Adoptive transfer of immunity to oral challenge with virulent salmonellae in innately susceptible BALB/c mice requires both immune serum and T cells. Infect. Immun. 61, 3981–3984.

Mastroeni, P., Simmons, C., Fowler, R., Hormaeche, C.E., and Dougan, G. (2000). Igh-6⁻/⁻ (B-cell-deficient) mice fail to mount solid acquired resistance to oral challenge with virulent Salmonella enterica serovar Typhimurium and show impaired Th1 T-cell responses to Salmonella antigens. Infect. Immun. 68, 46–53.

Matamouros, S., and Miller, S.I. (2015). S. Typhimurium strategies to resist killing by cationic antimicrobial peptides. Biochim. Biophys. Acta 1848, 3021–3025.

Mazurkiewicz, P., Thomas, J., Thompson, J.A., Liu, M., Arbibe, L., Sansonetti, P., and Holden, D.W. (2008). SpvC is a Salmonella effector with phosphothreonine lyase activity on host mitogen-activated protein kinases. Mol. Microbiol. 67, 1371–1383. https://doi.org/10.1111/j.1365-2958.2008.06134.x.

McConnell, E.L., Basit, A.W., and Murdan, S. (2008). Measurements of rat and mouse gastrointestinal pH, fluid and lymphoid tissue, and implications for in-vivo experiments. J. Pharm. Pharmacol. 60, 63–70. https://doi.org/10.1211/jpp.60.1.0008.

McGhie, E.J., Brawn, L.C., Hume, P.J., Humphreys, D., and Koronakis, V. (2009). Salmonella takes control: effector-driven manipulation of the host. Curr. Opin. Microbiol. 12, 117–124. https://doi.org/10.1016/j.mib.2008.12.001.

McGourty, K., Thurston, T.L., Matthews, S.A., Pinaud, L., Mota, L.J., and Holden, D.W. (2012). Salmonella Inhibits Retrograde Trafficking of Mannose-6-Phosphate Receptors and Lysosome Function. Science (80-.). 338.

McLaughlin, L.M., Govoni, G.R., Gerke, C., Gopinath, S., Peng, K., Laidlaw, G., Chien, Y.H., Jeong, H.W., Li, Z., Brown, M.D., *et al.* (2009). The Salmonella SPI2 effector SseI mediates long-term systemic infection by modulating host cell migration. PLOS Pathog. 5, e1000671. https://doi.org/10.1371/journal.ppat.1000671.

McLaughlin, L.M., Xu, H., Carden, S.E., Fisher, S., Reyes, M., Heilshorn, S.C., and Monack, D.M. (2014). A microfluidic-based genetic screen to identify microbial virulence factors that inhibit dendritic cell migration. Integr. Biol. (Camb). 6, 438–449.

McSorley, S.J., and Jenkins, M.K. (2000). Antibody is required for protection against virulent but not attenuated Salmonella enterica serovar Typhimurium. Infect. Immun. 68, 3344–3348.

Mesquita, F.S., Thomas, M., Sachse, M., Santos, A.J., Figueira, R., and Holden, D.W. (2012). The Salmonella deubiquitinase SseL inhibits selective autophagy of cytosolic aggregates. PLOS Pathog. 8, e1002743. https://doi.org/10.1371/journal.ppat.1002743.

Meunier, E., Dick, M.S., Dreier, R.F., Schürmann, N., Kenzelmann Broz, D., Warming, S., Roose-Girma, M., Bumann, D., Kayagaki, N., Takeda, K., *et al.* (2014). Caspase-11 activation requires lysis of pathogen-containing vacuoles by IFN-induced GTPases. Nature 509, 366–370. https://doi.org/10.1038/nature13157.

Miao, E.A., Mao, D.P., Yudkovsky, N., Bonneau, R., Lorang, C.G., Warren, S.E., Leaf, I.A., and Aderem, A. (2010a). Innate immune detection of the type III secretion apparatus through the NLRC4 inflammasome. Proc. Natl. Acad. Sci. U.S.A. 107, 3076–3080. https://doi.org/10.1073/pnas.0913087107.

Miao, E.A., Leaf, I.A., Treuting, P.M., Mao, D.P., Dors, M., Sarkar, A., Warren, S.E., Wewers, M.D., and Aderem, A. (2010b). Caspase-1-induced pyroptosis is an innate immune effector mechanism against intracellular bacteria. Nat. Immunol. 11, 1136–1142.

Mitterstiller, A.M., Haschka, D., Dichtl, S., Nairz, M., Demetz, E., Talasz, H., Soares, M.P., Einwallner, E., Esterbauer, H., Fang, F.C., *et al.* (2016). Heme oxygenase 1 controls early innate immune response of macrophages to Salmonella Typhimurium infection. Cell. Microbiol. 18, 1374–1389. https://doi.org/10.1111/cmi.12578.

Mittrücker, H.W., Raupach, B., Köhler, A., and Kaufmann, S.H. (2000). Cutting edge: role of B lymphocytes in protective immunity against Salmonella Typhimurium infection. J. Immunol. 164, 1648–1652.

Monack, D.M., Mueller, A., and Falkow, S. (2004a). Persistent bacterial infections: the interface of the pathogen and the host immune system. Nat. Rev. Microbiol. 2, 747–765. https://doi.org/10.1038/nrmicro955.

Monack, D.M., Bouley, D.M., and Falkow, S. (2004b). Salmonella Typhimurium persists within macrophages in the mesenteric lymph nodes of chronically infected Nramp1+/+ mice and can be reactivated by IFNgamma neutralization. J. Exp. Med. 199, 231–241.

Mukherjee, K., Parashuraman, S., Raje, M., and Mukhopadhyay, A. (2001). SopE acts as an Rab5-specific nucleotide exchange factor and recruits non-prenylated Rab5 on Salmonella-containing phagosomes to promote fusion with early endosomes. J. Biol. Chem. 276, 23607–23615.

Müller, A.J., Hoffmann, C., Galle, M., Van Den Broeke, A., Heikenwalder, M., Falter, L., Misselwitz, B., Kremer, M., Beyaert, R., and Hardt, W.D. (2009). The S. Typhimurium effector SopE induces caspase-1 activation in stromal cells to initiate gut inflammation. Cell Host Microbe 6, 125–136. https://doi.org/10.1016/j.chom.2009.07.007.

Murli, S., Watson, R.O., and Galán, J.E. (2001). Role of tyrosine kinases and the tyrosine phosphatase SptP in the interaction of Salmonella with host cells. Cell. Microbiol. 3, 795–810.

Murshid, A., Borges, T.J., Lang, B.J., and Calderwood, S.K. (2016). The Scavenger Receptor SREC-I Cooperates with Toll-Like Receptors to Trigger Inflammatory Innate Immune Responses. Front. Immunol. 7, 226. https://doi.org/10.3389/fimmu.2016.00226.

Neumann, C., Fraiture, M., Hernández-Reyes, C., Akum, F.N., Virlogeux-Payant, I., Chen, Y., Pateyron, S., Colcombet, J., Kogel, K.-H., Hirt, H., et al. (2014). The Salmonella effector protein SpvC, a phosphothreonine lyase is functional in plant cells. Front. Microbiol. 5, 548.

Niedergang, F., Sirard, J.C., Blanc, C.T., and Kraehenbuhl, J.P. (2000). Entry and survival of Salmonella Typhimurium in dendritic cells and presentation of recombinant antigens do not require macrophage-specific virulence factors. Proc. Natl. Acad. Sci. U.S.A. 97, 14650–14655.

Nikaido, E., Yamaguchi, A., and Nishino, K. (2008). AcrAB multidrug efflux pump regulation in Salmonella enterica serovar Typhimurium by RamA in response to environmental signals. J. Biol. Chem. 283, 24245–24253. https://doi.org/10.1074/jbc.M804544200.

Nishino, K., Latifi, T., and Groisman, E.A. (2006). Virulence and drug resistance roles of multidrug efflux systems of Salmonella enterica serovar Typhimurium. Mol. Microbiol. 59, 126–141.

Nix, R.N., Altschuler, S.E., Henson, P.M., and Detweiler, C.S. (2007). Hemophagocytic macrophages harbor Salmonella enterica during persistent infection. PLOS Pathog. 3, e193.

O’Dea, E., and Hoffmann, A. (2010). The regulatory logic of the NF-kappaB signaling system. Cold Spring Harb. Perspect. Biol. 2, a000216.

O’Donnell, H., and McSorley, S.J. (2014). Salmonella as a model for non-cognate Th1 cell stimulation. Front. Immunol. 5, 1–13.

Ochman, H., Soncini, F.C., Solomon, F., and Groisman, E.A. (1996). Identification of a pathogenicity island required for Salmonella survival in host cells. Proc. Natl. Acad. Sci. U.S.A. 93, 7800–7804.

Onyiah, J.C., Sheikh, S.Z., Maharshak, N., Steinbach, E.C., Russo, S.M., Kobayashi, T., Mackey, L.C., Hansen, J.J., Moeser, A.J., Rawls, J.F., et al. (2013). Carbon monoxide and heme oxygenase-1 prevent intestinal inflammation in mice by promoting bacterial clearance. Gastroenterology 144, 789–798.

Otterbein, L.E., Bach, F.H., Alam, J., Soares, M., Tao Lu, H., Wysk, M., Davis, R.J., Flavell, R.A., and Choi, A.M. (2000). Carbon monoxide has anti-inflammatory effects involving the mitogen-activated protein kinase pathway. Nat. Med. 6, 422–428. https://doi.org/10.1038/74680.

Owen, K.A., and Casanova, J.E. (2015). Salmonella manipulates autophagy to ‘serve and protect’. Cell Host Microbe 18, 517–519. https://doi.org/10.1016/j.chom.2015.10.020.

Owen, K.A., Meyer, C.B., Bouton, A.H., and Casanova, J.E. (2014). Activation of focal adhesion kinase by Salmonella suppresses autophagy via an Akt/mTOR signaling pathway and promotes bacterial survival in macrophages. PLOS Pathog. 10, e1004159.

Owen, K.A., Anderson, C.J., and Casanova, J.E. (2016). Salmonella suppresses the TRIF-dependent type I interferon response in macrophages. MBio 7, e02051–15. https://doi.org/10.1128/mBio.02051-15.

Parra-Lopez, C., Baer, M.T., and Groisman, E.A. (1993). Molecular genetic analysis of a locus required for resistance to antimicrobial peptides in Salmonella Typhimurium. EMBO J. 12, 4053–4062.

Parra-Lopez, C., Lin, R., Aspedon, A., and Groisman, E.A. (1994). A Salmonella protein that is required for resistance to antimicrobial peptides and transport of potassium. EMBO J. 13, 3964–3972.

Pescaretti, M. de L.M., López, F.E., Morero, R.D., and Delgado, M.A. (2011). The PmrA/PmrB regulatory system controls the expression of the wzzfepE gene involved in the O-antigen synthesis of Salmonella enterica serovar Typhimurium. Microbiology 157, 2515–2521.

Petrovska, L., Aspinall, R.J., Barber, L., Clare, S., Simmons, C.P., Stratford, R., Khan, S.A., Lemoine, N.R., Frankel, G., Holden, D.W., et al. (2004). Salmonella enterica serovar Typhimurium interaction with dendritic cells: Impact of the sifA gene. Cell. Microbiol. 6, 1071–1084.

Pilar, A.V.C., Reid-Yu, S. a, Cooper, C. a, Mulder, D.T., and Coombes, B.K. (2012). GogB is an anti-inflammatory effector that limits tissue damage during Salmonella infection through interaction with human FBXO22 and Skp1. PLOS Pathog. 8, e1002773.

Prieto, A.I., Ramos-Morales, F., and Casadesús, J. (2004). Bile-induced DNA damage in *Salmonella enterica*. Genetics 168, 1787–1794.

Prieto, A.I., Ramos-Morales, F., and Casadesús, J. (2006). Repair of DNA damage induced by bile salts in *Salmonella enterica*. Genetics 174, 575–584.

Prouty, A.M., and Gunn, J.S. (2003). Comparative analysis of *Salmonella enterica* serovar Typhimurium biofilm formation on gallstones and on glass. Infect Immun 71, 7154–7158.

Prouty, A.M., Schewesinger, W.H., and Gunn, J.S. (2002). Biofilm formation and interaction with the surfaces of gallstones by *Salmonella* spp. Infect. Immun. 70, 2640–2649.

Prouty, A.M., Brodsky, I.E., Manos, J., Belas, R., Falkow, S., and Gunn, J.S. (2004). Transcriptional regulation of *Salmonella enterica* serovar Typhimurium genes by bile. FEMS Immunol. Med. Microbiol. 41, 177–185. https://doi.org/10.1016/j.femsim.2004.03.002.

Qu, Y., Misaghi, S., Newton, K., Maltzman, A., Izrael-Tomasevic, A., Arnott, D., and Dixit, V.M. (2016). NLRP3 recruitment by NLRC4 during *Salmonella* infection. J. Exp. Med. 213, 877–885. https://doi.org/10.1084/jem.20132234.

Radtke, A.L., Delbridge, L.M., Balachandran, S., Barber, G.N., and O’Riordan, M.X. (2007). TBK1 protects vacuolar integrity during intracellular bacterial infection. PLOS Pathog. 3, e29.

Ramos-Morales, F. (2012a). Impact of *Salmonella enterica* type III secretion system effectors on the eukaryotic host cell. ISRN Cell Biol. 2012, 1–36.

Ramos-Morales, F. (2012b). Acidic pH: enemy or ally for enteric bacteria? Virulence 3, 103–106. https://doi.org/10.4161/viru.19382.

Ramos-Morales, F., Prieto, A.I., Beuzón, C.R., Holden, D.W., and Casadesús, J. (2003). Role for *Salmonella enterica* enterobacterial common antigen in bile resistance and virulence. J. Bacteriol. 185, 5328–5332.

Ramu, P., Tanskanen, R., Holmberg, M., Lähteenmäki, K., Korhonen, T.K., and Meri, S. (2007). The surface protease PgtE of *Salmonella enterica* affects complement activity by proteolytically cleaving C3b, C4b and C5. FEBS Lett. 581, 1716–1720.

Rathinam, V.A., and Fitzgerald, K.A. (2016). Inflammasome complexes: emerging mechanisms and effector functions. Cell 165, 792–800. https://doi.org/10.1016/j.cell.2016.03.046.

Rémy, S., Blancou, P., Tesson, L., Tardif, V., Brion, R., Royer, P.J., Motterlini, R., Foresti, R., Painchaud, M., Pogu, S., *et al.* (2009). Carbon monoxide inhibits TLR-induced dendritic cell immunogenicity. J. Immunol. 182, 1877–1884. https://doi.org/10.4049/jimmunol.0802436.

Riquelme, S.A., Bueno, S.M., and Kalergis, A.M. (2012). IgG keeps virulent *Salmonella* from evading dendritic cell uptake. Immunology 136, 291–305. https://doi.org/10.1111/j.1365-2567.2012.03578.x.

Riquelme, S.A., Pogu, J., Anegon, I., Bueno, S.M., and Kalergis, A.M. (2015). Carbon monoxide impairs mitochondria-dependent endosomal maturation and antigen presentation in dendritic cells. Eur. J. Immunol. 45, 3269–3288.

Riva, R., Korhonen, T.K., and Meri, S. (2015). The outer membrane protease PgtE of *Salmonella enterica* interferes with the alternative complement pathway by cleaving factors B and H. Front. Microbiol. 6, 1–9.

Rolhion, N., Furniss, R.C., Grabe, G., Ryan, A., Liu, M., Matthews, S.A., and Holden, D.W. (2016). Inhibition of nuclear transport of NF-κB p65 by the *Salmonella* type III secretion system effector SpvD. PLOS Pathog. 12, e1005653. https://doi.org/10.1371/journal.ppat.1005653.

Ruan, H., Zhang, Z., Tian, L., Wang, S., Hu, S., and Qiao, J.J. (2016). The *Salmonella* effector SopB prevents ROS-induced apoptosis of epithelial cells by retarding TRAF6 recruitment to mitochondria. Biochem. Biophys. Res. Commun. 478, 618–623. https://doi.org/10.1016/j.bbrc.2016.07.116.

Russell, T.L., Berardi, R.R., Barnett, J.L., Dermentzoglou, L.C., Jarvenpaa, K.M., Schmaltz, S.P., and Dressman, J.B. (1993). Upper gastrointestinal pH in seventy-nine healthy, elderly, North American men and women. Pharm. Res. 10, 187–196.

Ryan, D., Ojha, U.K., Jaiswal, S., Padhi, C., and Suar, M. (2016). The small RNA DsrA influences the acid tolerance response and virulence of *Salmonella enterica* serovar Typhimurium. Front. Microbiol. 7, 599. https://doi.org/10.3389/fmicb.2016.00599.

Scherer, C.A., Cooper, E., and Miller, S.I. (2000). The *Salmonella* type III secretion translocon protein SspC is inserted into the epithelial cell plasma membrane upon infection. Mol. Microbiol. 37, 1133–1145.

Schiøler, H., Christiansen, E.D., Høybye, G., Rasmussen, S.N., and Greibe, J. (1983). Biliary calculi in chronic *Salmonella* carriers and healthy controls: a controlled study. Scand. J. Infect. Dis. 15, 17–19.

Sharma, A., and Qadri, A. (2004). Vi polysaccharide of *Salmonella Typhi* targets the prohibitin family of molecules in intestinal epithelial cells and suppresses early inflammatory responses. Proc. Natl. Acad. Sci. U.S.A. 101, 17492–17497.

Sharma, D., and Kanneganti, T.D. (2016). The cell biology of inflammasomes: Mechanisms of inflammasome activation and regulation. J. Cell Biol. 213, 617–629. https://doi.org/10.1083/jcb.201602089.

Shea, J.E., Hensel, M., Gleeson, C., and Holden, D.W. (1996). Identification of a virulence locus encoding a second type III secretion system in Salmonella Typhimurium. Proc. Natl. Acad. Sci. U.S.A. 93, 2593–2597.

Shen, P., Roch, T., Lampropoulou, V., O’Connor, R.A., Stervbo, U., Hilgenberg, E., Ries, S., Dang, V.D., Jaime, Y., Daridon, C., *et al.* (2014). IL-35-producing B cells are critical regulators of immunity during autoimmune and infectious diseases. Nature 507, 366–370. https://doi.org/10.1038/nature12979.

Sheppard, K.A., Fitz, L.J., Lee, J.M., Benander, C., George, J.A., Wooters, J., Qiu, Y., Jussif, J.M., Carter, L.L., Wood, C.R., *et al.* (2004). PD-1 inhibits T-cell receptor induced phosphorylation of the ZAP70/CD3zeta signalosome and downstream signaling to PKCtheta. FEBS Lett. 574, 37–41. https://doi.org/10.1016/j.febslet.2004.07.083.

Shpargel, J.S., Berardi, R.S., and Lenz, D. (1985). Salmonella Typhi carrier state 52 years after illness with typhoid fever: a case study. Am. J. Infect. Control 13, 122–123.

Sinnott, C.R., and Teall, A.J. (1987). Persistent gallbladder carriage of Salmonella Typhi. Lancet 1, 976.

Søndberg, E., and Jelsbak, L. (2016). Salmonella Typhimurium undergoes distinct genetic adaption during chronic infections of mice. BMC Microbiol. 16, 30.

Spanò, S., Gao, X., Hannemann, S., Lara-Tejero, M., and Galán, J.E. (2016). A bacterial pathogen targets a host Rab-family GTPase defense pathway with a GAP. Cell Host Microbe 19, 216–226. https://doi.org/10.1016/j.chom.2016.01.004.

Srinivasan, A., Foley, J., Ravindran, R., and McSorley, S.J. (2004). Low-dose Salmonella infection evades activation of flagellin-specific CD4 T cells. J. Immunol. 173, 4091–4099.

Srinivasan, A., Nanton, M., Griffin, A., and McSorley, S.J. (2009). Culling of activated CD4 T cells during typhoid is driven by Salmonella virulence genes. J. Immunol. 182, 7838–7845. https://doi.org/10.4049/jimmunol.0900382.

Staib, L., and Fuchs, T.M. (2015). Regulation of fucose and 1,2-propanediol utilization by Salmonella enterica serovar Typhimurium. Front. Microbiol. 6, 1116. https://doi.org/10.3389/fmicb.2015.01116.

Stecher, B., Robbiani, R., Walker, A.W., Westendorf, A.M., Barthel, M., Kremer, M., Chaffron, S., Macpherson, A.J., Buer, J., Parkhill, J., *et al.* (2007). Salmonella enterica serovar Typhimurium exploits inflammation to compete with the intestinal microbiota. PLOS Biol. 5, 2177–2189.

Stecher, B., Denzler, R., Maier, L., Bernet, F., Sanders, M.J., Pickard, D.J., Barthel, M., Westendorf, A.M., Krogfelt, K.A., Walker, A.W., *et al.* (2012). Gut inflammation can boost horizontal gene transfer between pathogenic and commensal Enterobacteriaceae. Proc. Natl. Acad. Sci. U.S.A. 109, 1269–1274. https://doi.org/10.1073/pnas.1113246109.

Steele-Mortimer, O., Brumell, J.H., Knodler, L.A., Méresse, S., Lopez, A., and Finlay, B.B. (2002). The invasion-associated type III secretion system of Salmonella enterica serovar Typhimurium is necessary for intracellular proliferation and vacuole biogenesis in epithelial cells. Cell. Microbiol. 4, 43–54.

Storek, K.M., and Monack, D.M. (2015). Bacterial recognition pathways that lead to inflammasome activation. Immunol. Rev. 265, 112–129. https://doi.org/10.1111/imr.12289.

Sun, H., Kamanova, J., Lara-Tejero, M., and Galán, J.E. (2016). A family of Salmonella type III secretion effector proteins selectively targets the NF-κB signaling pathway to preserve host homeostasis. PLOS Pathog. 12, e1005484. https://doi.org/10.1371/journal.ppat.1005484.

Sundquist, M., and Wick, M.J. (2009). Salmonella induces death of CD8α+ dendritic cells but not CD11c(int)CD11b(+) inflammatory cells in vivo via MyD88 and TNFR1. J. Leukoc. Biol. 85, 225–234. https://doi.org/10.1189/jlb.0708413.

Tardif, V., Riquelme, S.A., Remy, S., Carreño, L.J., Cortés, C.M., Simon, T., Hill, M., Louvet, C., Riedel, C.A., Blancou, P., *et al.* (2013). Carbon monoxide decreases endosome-lysosome fusion and inhibits soluble antigen presentation by dendritic cells to T cells. Eur. J. Immunol. 43, 2832–2844.

Tenor, J.L., McCormick, B.A., Ausubel, F.M., and Aballay, A. (2004). Caenorhabditis elegans-based screen identifies Salmonella virulence factors required for conserved host-pathogen interactions. Curr. Biol. 14, 1018–1024.

Tezcan-Merdol, D., Nyman, T., Lindberg, U., Haag, F., Koch-Nolte, F., and Rhen, M. (2001). Actin is ADP-ribosylated by the Salmonella enterica virulence-associated protein SpvB. Mol. Microbiol. 39, 606–619.

Thiennimitr, P., Winter, S.E., Winter, M.G., Xavier, M.N., Tolstikov, V., Huseby, D.L., Sterzenbach, T., Tsolis, R.M., Roth, J.R., and Bäumler, A.J. (2011). Intestinal inflammation allows Salmonella to use ethanolamine to compete with the microbiota. Proc. Natl. Acad. Sci. U.S.A. 108, 17480–17485. https://doi.org/10.1073/pnas.1107857108.

Thompson, L.J., Dunstan, S.J., Dolecek, C., Perkins, T., House, D., Dougan, G., Nguyen, T.D.H., Tran, T.H.T.P. La, Doan, C. Du, Le, T.P., et al. (2009). Transcriptional response in the peripheral blood of patients infected with Salmonella enterica serovar Typhi. Proc. Natl. Acad. Sci. U.S.A. 106, 22433–22438.

Thurston, T.L., Boyle, K.B., Allen, M., Ravenhill, B.J., Karpiyevich, M., Bloor, S., Kaul, A., Noad, J., Foeglein, A., Matthews, S.A., et al. (2016). Recruitment of TBK1 to cytosol-invading Salmonella induces WIPI2-dependent antibacterial autophagy. EMBO J. 182, e201694491.

Thurston, T.L., Ryzhakov, G., Bloor, S., von Muhlenen, N., and Randow, F. (2009). The TBK1 adaptor and autophagy receptor NDP52 restricts the proliferation of ubiquitin-coated bacteria. Nat. Immunol. 10, 1215–1221. https://doi.org/10.1038/ni.1800.

Thurston, T.L., Wandel, M.P., von Muhlenen, N., Foeglein, A., and Randow, F. (2012). Galectin 8 targets damaged vesicles for autophagy to defend cells against bacterial invasion. Nature 482, 414–418. https://doi.org/10.1038/nature10744.

Tindall, B.J., Grimont, P.A.D., Garrity, G.M., and Euzéby, J.P. (2005). Nomenclature and taxonomy of the genus *Salmonella*. Int. J. Syst. Evol. Microbiol. 55, 521–524.

Tobar, J.A., Carreño, L.J., Bueno, S.M., González, P.A., Mora, J.E., Quezada, S.A., and Kalergis, A.M. (2006). Virulent *Salmonella enterica* serovar Typhimurium evades adaptive immunity by preventing dendritic cells from activating T cells. Infect. Immun. 74, 6438–6448.

Tobar, J.A., González, P.A., and Kalergis, A.M. (2004). *Salmonella* escape from antigen presentation can be overcome by targeting bacteria to Fc gamma receptors on dendritic cells. J. Immunol. 173, 4058–4065.

Tran, Q.T., Gomez, G., Khare, S., Lawhon, S.D., Raffatellu, M., Bäumler, A.J., Ajithdoss, D., Dhavala, S., and Adams, L.G. (2010). The *Salmonella enterica* serotype Typhi Vi capsular antigen is expressed after the bacterium enters the ileal mucosa. Infect. Immun. 78, 527–535.

Ugrinovic, S., Ménager, N., Goh, N., and Mastroeni, P. (2003). Characterization and development of T-Cell immune responses in B-cell-deficient (*Igh-6(-/-)*) mice with *Salmonella enterica* serovar Typhimurium infection. Infect. Immun. 71, 6808–6819.

van der Velden, A.W., Copass, M.K., and Starnbach, M.N. (2005). *Salmonella* inhibit T cell proliferation by a direct, contact-dependent immunosuppressive effect. Proc. Natl. Acad. Sci. U.S.A. 102, 17769–17774.

van der Velden, A.W., Dougherty, J.T., and Starnbach, M.N. (2008). Down-modulation of TCR expression by *Salmonella enterica* serovar Typhimurium. J. Immunol. 180, 5569–5574.

Van Kesteren, M., Bibby, B.G., and Berry, G.P. (1942). Studies on the antibacterial factors of human saliva. J. Bacteriol. 43, 573–583.

Vogt, S.L., Peña-Díaz, J., and Finlay, B.B. (2015). Chemical communication in the gut: Effects of microbiota-generated metabolites on gastrointestinal bacterial pathogens. Anaerobe 34, 106–115.

van der Waaij, D., Berghuis-de Vries, J.M., and Lekkerkerk Lekkerkerk-v (1971). Colonization resistance of the digestive tract in conventional and antibiotic-treated mice. J. Hyg. (Lond). 69, 405–411.

Watson, K.G., and Holden, D.W. (2010). Dynamics of growth and dissemination of *Salmonella in vivo*. Cell. Microbiol. 12, 1389–1397. https://doi.org/10.1111/j.1462-5822.2010.01511.x.

Wehkamp, J., Schauber, J., and Stange, E.F. (2007). Defensins and cathelicidins in gastrointestinal infections. Curr. Opin. Gastroenterol. 23, 32–38. https://doi.org/10.1097/MOG.0b013e32801182c2.

Wei, L., Jin, Q., Chu, Y., Wu, S., and Huang, R. (2015). Suppression of dendritic cell and T-cell activation by the pRST⁹⁸ *Salmonella* plasmid. Mol. Med. Rep. 11, 2306–2314. https://doi.org/10.3892/mmr.2014.2919.

Wilson, R.P., Winter, S.E., Spees, A.M., Winter, M.G., Nishimori, J.H., Sanchez, J.F., Nuccio, S.-P., Crawford, R.W., Tükel, Ç., and Bäumler, A.J. (2011). The Vi capsular polysaccharide prevents complement receptor 3-mediated clearance of *Salmonella enterica* serotype Typhi. Infect. Immun. 79, 830–837.

Winter, S.E., Thiennimitr, P., Winter, M.G., Butler, B.P., Huseby, D.L., Crawford, R.W., Russell, J.M., Bevins, C.L., Adams, L.G., Tsolis, R.M., *et al.* (2010a). Gut inflammation provides a respiratory electron acceptor for *Salmonella*. Nature 467, 426–429. https://doi.org/10.1038/nature09415.

Winter, S.E., Winter, M.G., Godinez, I., Yang, H.-J., Rüssmann, H., Andrews-Polymenis, H.L., and Bäumler, A.J. (2010b). A rapid change in virulence gene expression during the transition from the intestinal lumen into tissue promotes systemic dissemination of *Salmonella*. PLOS Pathog. 6, e1001060.

Worley, M.J., Nieman, G.S., Geddes, K., and Heffron, F. (2006). *Salmonella* Typhimurium disseminates within its host by manipulating the motility of infected cells. Proc. Natl. Acad. Sci. U.S.A. 103, 17915–17920.

Wu, H., Jones, R.M., and Neish, A.S. (2012). The *Salmonella* effector AvrA mediates bacterial intracellular survival during infection *in vivo*. Cell. Microbiol. 14, 28–39. https://doi.org/10.1111/j.1462-5822.2011.01694.x.

Wynosky-Dolfi, M.A., Snyder, A.G., Philip, N.H., Doonan, P.J., Poffenberger, M.C., Avizonis, D., Zwack, E.E., Riblett, A.M., Hu, B., Strowig, T., *et al.* (2014). Oxidative metabolism enables Salmonella evasion of the NLRP3 inflammasome. J. Exp. Med. 211, 653–668. https://doi.org/10.1084/jem.20130627.

Yachie, A., Niida, Y., Wada, T., Igarashi, N., Kaneda, H., Toma, T., Ohta, K., Kasahara, Y., and Koizumi, S. (1999). Oxidative stress causes enhanced endothelial cell injury in human heme oxygenase-1 deficiency. J. Clin. Invest. 103, 129–135. https://doi.org/10.1172/JCI4165.

Yang, Z., Soderholm, A., Lung, T.W., Giogha, C., Hill, M.M., Brown, N.F., Hartland, E., and Teasdale, R.D. (2015). SseK3 is a Salmonella effector that binds TRIM32 and modulates the host’s NF-κB signalling activity. PLOS ONE 10, e0138529. https://doi.org/10.1371/journal.pone.0138529.

Ye, Z., Petrof, E.O., Boone, D., Claud, E.C., and Sun, J. (2007). Salmonella effector AvrA regulation of colonic epithelial cell inflammation by deubiquitination. Am. J. Pathol. 171, 882–892.

Yin, J., Xia, J., Tao, M., Xu, L., Li, Q., Geng, S., and Jiao, X. (2016). Construction and characterization of a cigR deletion mutant of Salmonella enterica serovar Pullorum. Avian Pathol. 45, 569–575. https://doi.org/10.1080/03079457.2016.1187708.

Zaki, M.H., Fujii, S., Okamoto, T., Islam, S., Khan, S., Ahmed, K.A., Sawa, T., and Akaike, T. (2009). Cytoprotective function of heme oxygenase 1 induced by a nitrated cyclic nucleotide formed during murine salmonellosis. J. Immunol. 182, 3746–3756. https://doi.org/10.4049/jimmunol.0803363.

Zgurskaya, H.I., and Nikaido, H. (1999). Bypassing the periplasm: reconstitution of the AcrAB multidrug efflux pump of Escherichia coli. Proc. Natl. Acad. Sci. U.S.A. 96, 7190–7195.

Zhang, S., Santos, R.L., Tsolis, R.M., Stender, S., Hardt, W.-D., Bäumler, A.J., and Adams, L.G. (2002). The Salmonella enterica serotype Typhimurium effector proteins SipA, SopA, SopB, SopD, and SopE2 act in concert to induce diarrhea in calves. Infect. Immun. 70, 3843–3855.

Zhou, D., Chen, L.M., Hernandez, L., Shears, S.B., and Galán, J.E. (2001a). A Salmonella inositol polyphosphatase acts in conjunction with other bacterial effectors to promote host cell actin cytoskeleton rearrangements and bacterial internalization. Mol. Microbiol. 39, 248–259.

Zhou, Z., Ribeiro, A.A., Lin, S., Cotter, R.J., Miller, S.I., and Raetz, C.R. (2001b). Lipid A modifications in polymyxin-resistant Salmonella Typhimurium: PMRA-dependent 4-amino-4-deoxy-L-arabinose, and phosphoethanolamine incorporation. J. Biol. Chem. 276, 43111–43121.
