
# Cryo-EM structure of the bacteriophage T4 isometric head at 3.3-Å resolution and its relevance to the assembly of icosahedral viruses

Zhenguo Chen${}^{a,1,2}$, Lei Sun${}^{a,1,2}$, Zhihong Zhang${}^{b,1,3}$, Andrei Fokine${}^{a}$, Victor Padilla-Sanchez${}^{b}$, Dorit Hanein${}^{c}$, Wen Jiang${}^{a}$, Michael G. Rossmann${}^{a,4}$, and Venigalla B. Rao${}^{b,4}$

${}^{a}$Department of Biological Sciences, Purdue University, West Lafayette, IN 47907; ${}^{b}$Department of Biology, The Catholic University of America, Washington, DC 20064; and ${}^{c}$Bioinformatics and Structural Biology Program, Sanford-Burnham-Prebys Medical Discovery Institute, La Jolla, CA 92037

Edited by Ming Luo, Georgia State University, Atlanta, GA, and accepted by Editorial Board Member Peter Palese August 18, 2017 (received for review May 22, 2017)

The 3.3-Å cryo-EM structure of the 860-Å-diameter isometric mutant bacteriophage T4 capsid has been determined. WT T4 has a prolate capsid characterized by triangulation numbers (T numbers) $T_{\text{end}} = 13$ for end caps and $T_{\text{mid}} = 20$ for midsection. A mutation in the major capsid protein, gp23, produced T=13 icosahedral capsids. The capsid is stabilized by 660 copies of the outer capsid protein, Soc, which clamp adjacent gp23 hexamers. The occupancies of Soc molecules are proportional to the size of the angle between the planes of adjacent hexameric capsomers. The angle between adjacent hexameric capsomers is greatest around the fivefold vertices, where there is the largest deviation from a planar hexagonal array. Thus, the Soc molecules reinforce the structure where there is the greatest strain in the gp23 hexagonal lattice. Mutations that change the angles between adjacent capsomers affect the positions of the pentameric vertices, resulting in different triangulation numbers in bacteriophage T4. The analysis of the T4 mutant head assembly gives guidance to how other icosahedral viruses reproducibly assemble into capsids with a predetermined T number, although the influence of scaffolding proteins is also important.

bacteriophage T4 | isometric head | virus capsid assembly | triangulation numbers | capsid decoration proteins

Tailed bacteriophages are widely distributed in nature and probably are the most abundant organisms on the planet (1, 2). Phage capsids are extremely stable and able to sustain high internal pressure exerted by their tightly packed genomes (3, 4). Bacteriophage T4 belongs to the Myoviridae family of phages (5, 6). Its 172-kbp DNA genome is packaged to near-crystalline density, resulting in an internal capsid pressure of ~20 atm (7–9). T4 serves as an excellent model to elucidate the mechanisms of virus assembly and DNA packaging (10). T4 also provides a valuable biotechnology platform for diverse applications such as vaccine design and gene therapy (11–14).

The T4 capsid, or head, is a 1,200-Å-long and 860-Å-wide prolate icosahedron (15, 16). The capsid has a unique “portal” vertex to which a ~1,400-Å-long contractile tail is attached (17, 18). The tail is terminated by a baseplate (19–21) to which are bound six long tail fibers (18). During infection, the tail binds to an *Escherichia coli* cell and creates a conduit for injection of the phage DNA into the bacterial cytoplasm (22). The internal pressure then drives the DNA out of the capsid, into the cell, through the tail (3).

The T4 head, tail, and fibers assemble by separate pathways (5). They then join together to form the infectious virion. The capsid assembly proceeds via formation of a precursor, called a prohead, which contains an inner scaffolding core and an outer shell (15). The prohead formation is initiated by the dodecameric protein gene product (gp) 20, which makes the unique portal vertex of the prohead (23). Gp20 nucleates the assembly of the inner scaffolding core made by the major core protein, gp22, the prohead protease gp21 (24), and other proteins. The major capsid protein gp23 coassembles with the core to form the outer shell of the prohead. Pentamers of the vertex protein, gp24, form the other 11 vertices of the prohead (25). The gp23 and gp24 proteins have approximately 22% sequence identity and likely have evolved from a primordial gene via gene duplication.

When the prohead assembly has been completed, the gp21 protease cleaves the inner core proteins (5, 15). Most of the cleavage products escape from the prohead to create space for DNA. In addition, the protease cleaves off 65 N-terminal residues from the major capsid protein, gp23, and 10 N-terminal residues from the vertex protein, gp24, producing gp23* (48.7 kDa) and gp24*, respectively. The genomic DNA is then packaged into the empty prohead through the portal vertex by an ATP-driven packaging motor containing five subunits of the motor protein, gp17 (9). The gp23*-gp24* shell of the prohead undergoes large conformational changes, resulting in the expansion of the capsid volume by ~50%. In vivo, the expansion

### Significance

The WT bacteriophage T4 head is prolate with icosahedral T=13 ends closing the central cylindrical section. The structure of a T=13 icosahedral head assembly, in which the major capsid protein of T4 phage had a single mutation at a residue in the interface between neighboring subunits, has been determined to 3.3-Å resolution. The structure shows the extensive network of interactions between the major capsid proteins gp23 and the vertex proteins gp24 and the stabilizing Soc proteins. We show that the angle between hexameric and pentameric capsomers helps to determine the triangulation number (T number) and suggests how other viruses assemble consistently with a predetermined T number.

Author contributions: M.G.R. and V.B.R. designed research; Z.C., Z.Z., and V.P.-S. performed research; W.J. contributed new reagents/analytic tools; D.H. enabled some of the data collection; L.S. and A.F. analyzed data; Z.C., L.S., A.F., M.G.R., and V.B.R. wrote the paper; and M.G.R. and V.B.R. supervised research.

The authors declare no conflict of interest.

This article is a PNAS Direct Submission. M.L. is a guest editor invited by the Editorial Board.

Data deposition: The cryo-EM map of the isometric T4 capsid has been deposited in the Electron Microscopy Data Bank (ID code **EMD-8661**). The atomic coordinates have been deposited in the Protein Data Bank, [www.pdb.org](http://www.pdb.org) (PDB ID code **5VF3**).

1. Z.C., L.S. and Z.Z. contributed equally to this work.
2. Present addresses: The Fifth People’s Hospital of Shanghai, Fudan University, Shanghai 201100, People’s Republic of China; and Institute of Biomedical Sciences, Fudan University, Shanghai 200032, People’s Republic of China.
3. Present address: Huaihai Institute of Technology, Lianyungang 222005, People’s Republic of China.
4. To whom correspondence may be addressed. Email: mr@purdue.edu or rao@cua.edu.

This article contains supporting information online at [www.pnas.org/lookup/suppl/doi:10.1073/pnas.1708483114/-/DCSupplemental](http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1708483114/-/DCSupplemental).
happens during DNA packaging. However, these processes can be decoupled (26). The outer surface of the expanded capsid is decorated by the small outer capsid (Soc) protein and the highly immunogenic outer capsid (Hoc) protein (27). Soc helps to stabilize the capsid against extremes of pH and temperature, whereas Hoc probably assists in finding and infecting the bacterial host (28, 29). After the completion of genome packaging, the portal vertex is sealed by the assembly of gp13–gp14 complex (i.e., “neck”), which also creates the attachment site for the independently assembled phage tail (30).

The structure of the mature T4 head was previously studied by using cryo-EM (16, 23, 31, 32) (Fig. 1 A and C). The best resolution that had been achieved for the prolate head was approximately 10 Å (23). These earlier studies showed that the major capsid protein gp23* is organized into a hexagonal lattice characterized by triangulation numbers \(T_{\text{cap}} = 13\) laevo for the icosahedral caps and \(T_{\text{mid}} = 20\) for the elongated midsection (16). The Hoc monomers bind at the centers of the gp23* hexameric capsomers, whereas Soc molecules bind at the interface between adjacent gp23* hexamers and clamp neighboring capsomers together (29). The T4 head contains 155 hexamers of gp23*, 12 copies of gp20 (61 kDa) at the special portal vertex, 11 pentamers of gp24* (44 kDa) at the remaining fivefold vertices, 155 copies of Hoc (40.4 kDa), and 870 copies of Soc (9.1 kDa).

The structure of the T4 vertex protein gp24 was determined by using X-ray crystallography (25). The structure of gp24 has a fold similar to the capsid protein of phage HK97 (33). The HK97 fold is a building block for the capsids of tailed dsDNA phages (34) as well as for herpesviruses (35). The crystal structures of Soc (29) and Hoc (28) from T4-like phages have also been determined, although the Hoc structure did not include the C-terminal domain, which binds to the capsid surface. In addition, an atomic-resolution cryo-EM structure of the dodecameric T4 portal protein, gp20, has been determined (23). However, the structure of the major capsid protein gp23 had not been determined before the present study because gp23 tends to aggregate when produced recombinantly, making it difficult to crystallize.

Mutations in the T4 prohead proteins can change the length of the capsid, producing mixtures of prolate WT capsids, icosahedral “isometric” or “petite” capsids, intermediate-length heads, and/or “giant” heads of various lengths (15, 36–38). In this study, we used a mutant T4 phage that produces predominantly empty isometric heads. These isometric particles have icosahedral symmetry except for the unique portal vertex occupied by the dodecameric portal protein gp20. The gp23* protein in these isometric heads forms a hexagonal lattice characterized by the triangulation number \(T = 13\) laevo. Here we describe the cryo-EM structure of the isometric T4 head at 3.3-Å resolution. The electron density map established the atomic-resolution structures of the gp23*, gp24*, and Soc molecules in the capsid environment. Analysis of interactions between these subunits shows how a highly stable virus capsid can be assembled and how it can

![Figure 1](#fig1)

**Fig. 1.** Structure of the T4 prolate (A and C) and isometric (B and D) heads. Gp23*, gp24*, Hoc, and Soc are colored blue, magenta, yellow, and white, respectively. Note that Soc is absent around the gp24* pentamers. In C and D, the gp23* protein has been removed for clarity.withstand internal pressure caused by the tightly packaged viral genome. The structure also suggests the factors that determine the T<sub>end</sub> and T<sub>mid</sub> values that describe the quasi-symmetry of virus capsids.

# Results and Discussion

## Structure Determination

Of several T4 phage mutants, the 17amNG178.23pt21–34c mutant produced the largest fraction (approximately 80%) of isometric capsids (Fig. 2A). This T4 mutant contained two mutations. The first of these mutations was A275T in gp23 that resulted in the production of isometric capsids together with approximately 20% of prolate and intermediate-length heads. The second of these mutations was an amber mutation in the packaging motor protein gp17 that resulted in the production of DNA-free “empty” capsids when the phage was grown on a non-suppressor *E. coli* strain. The isometric capsids produced by this double-mutant phage were assembled via the initial formation of isometric proheads, followed by proteolytic cleavage of the prohead’s inner scaffolding core, of gp23, and of gp24. The cleaved proheads remained empty because they could not initiate DNA packaging as a result of the mutation in gene 17. The protein shells of the empty proheads expanded spontaneously (at 37 °C), resulting in mature empty heads (Fig. 2A). These were purified by CsCl gradient centrifugation and ion-exchange chromatography (Materials and Methods).

Samples of the purified heads were frozen onto Lacey carbon EM grids and examined with a Titan Krios electron microscope (FEI). Three different data sets were collected. An initial cryo-EM dataset included 40,000 particles recorded on film and resulted in 3D reconstruction of approximately 5.2-Å resolution.

A second cryo-EM data set was collected with the FEI Falcon II direct detector (pixel size of 1.38 Å) and included approximately 25,000 particles that generated a map with a resolution of 3.9 Å. The third data set was acquired on a Titan Krios electron microscope equipped with a K2 Summit detector (Gatan), used in the “super-resolution” mode, resulting in a pixel size of 0.81 Å. A total of 19,510 isometric particles were boxed from 3,500 images. Nonreferenced 2D classification was performed with the Relion program (39) to select 17,954 particles. The jspr program (40) was used for generation of the initial model and refinement of the particle orientations, assuming icosahedral symmetry. The final 3D cryo-EM reconstruction had an overall resolution of 3.3 Å, based on the “gold-standard” Fourier shell correlation (FSC) using the FSC = 0.143 criterion (41) (Fig. 2B). The map showed continuous density for the polypeptide chains and recognizable side-chain densities (Fig. 2C).

The atomic models of the gp23*, gp24*, and Soc molecules were built by using the COOT program (42) and refined with Phenix software (43) (*SI Appendix*, Table S1). The density for the Hoc molecules was weak and of insufficient quality for building an atomic model.

## Structure of the T4 Isometric Head

The T4 isometric capsid shell consists of 120 hexameric capsomers of gp23*, 11 pentameric capsomers of gp24*, and 1 dodecamer of gp20 (Fig. 1B and D). During the icosahedral reconstruction, 11 pentameric vertices occupied by gp24* were averaged with the special portal vertex occupied by gp20. Thus, the quality of the reconstruction was slightly diminished, especially at the vertices. However, the cryo-EM density in the vertex regions of the map was easily interpretable

![Figure 2](https://i.imgur.com/yourimage.png)

Fig. 2. Cryo-EM reconstruction of the T4 isometric head. (A) EM image. (B) FSC curve. (C) Electron density of two selected regions fitted with the refined structure.

Chen et al.  
PNAS Early Edition | 3 of 10in terms of the gp24* structure, and showed no significant influence from the portal protein at the special vertex.

The outer and inner surfaces of the capsid were found to be negatively charged. This implies that the dsDNA genome cannot bind easily to the inner surface of the virus, making it possible for the DNA to be ejected from the virus while infecting a host cell. In addition, the charged outer surface can prevent aggregation of the virus particles. The surfaces of the gp24* pentamers are more negatively charged than those of gp23* hexamers (Fig. 3). This might be to further decrease the chance of the genome getting stuck in the vertices while being ejected.

The decoration protein, Soc, forms a nearly continuous surface ridge that encircles each of the gp23* hexamers except at the interface between gp23* hexamers and gp24* pentamers (Fig. 1D). Clearly, the gp23*-gp24* interface is structurally different from the gp23*-gp23* interface, and is inappropriate for binding Soc. In addition, each hexamer contains one Hoc molecule at its center. Thus, the stoichiometry of the isometric T4 head is 720 copies of gp23*, 660 copies of Soc, 120 copies of Hoc, 55 copies of gp24*, and 12 copies of gp20. Each icosahedral asymmetric unit consists of 12 gp23* molecules (2 hexamers), 1 gp24* molecule, 11 Soc molecules, and 2 Hoc molecules (Fig. 4).

Gp24* Structure. The gp24* structure (25) has a polypeptide fold that is similar to the structure of the bacteriophage HK97 capsid protein (33) (Fig. 5). Furthermore, gp24* assembles into pentameric capsomers, similar to the HK97 capsomers (Fig. 6A and D). The gp24* molecule has a peripheral domain (P domain), which forms the periphery of the capsomers, and an axial domain (A domain) located near the central fivefold axes of the capsomers (Figs. 5D and 6C). The HK97 capsid protein contains an extended loop (E loop), which is covalently crosslinked to a P domain of another capsid protein subunit, belonging to a neighboring capsomer. However, the T4 gp24* proteins, unlike the HK97 capsid proteins, contain an additional insertion domain (I domain), which is located on the outer capsid surface,

forming the characteristic surface bumps (Fig. 6A). The insertion domain is connected to the rest of the protein structure via long linkers (I-domain linkers), which topologically correspond to the E loop in HK97. In the gp24 crystal structure determined previously, these linkers were not resolved because of their flexibility (Fig. 5C). However, the structure of these I-domain linkers is well-resolved in the present cryo-EM map. The insertion domain is bound to a neighboring gp24* subunit in the same pentameric capsomer, thus stabilizing the capsomer structure, a feature seen only in the T4 capsid thus far. The linkers interact with a neighboring gp24* molecule (Fig. 6C and E). In addition, one of the linkers interacts with the N-terminal region of a gp23* subunit from one capsomer and the P domain of a gp23* subunit from another capsomer (SI Appendix, Fig. S1). These noncovalent interactions stabilize the structure and occur near a similar location where the covalent intersubunit crosslinking occurs in HK97 (33).

The cryo-EM structure of pentameric gp24* in the T4 isometric capsid, analyzed here, has large conformational changes compared with the crystal structure of the monomeric, uncleaved gp24 (Fig. 5 C and D). A total of 246 of 427 residues can be aligned with an rmsd of 3.3 Å between Ca atoms. The first 10 residues of gp24 are missing in the cryo-EM structure as a result of proteolytic cleavage during capsid assembly. In the crystal structure, these 10 amino-terminal residues form a helix. Residues 11–36 form an elongated N-arm in the cryo-EM structure, whereas they form a helix in the crystal structure. In the cryo-EM structure, the N-arm of gp24* interacts with two gp23* molecules, thereby stabilizing the capsid. In addition, a part of the N-arm (residues 26–30) is involved in a parallel β-structure interaction with an I-domain linker of an adjacent gp24* subunit (Fig. 6E). In the mature capsid, the N terminus of gp24* (residue 11) is located on the outer surface of the capsid. However, in the prohead, the N terminus is required to be exposed to the maturation protease, which acts from within the prohead (24).

In the cryo-EM structure of the pentameric gp24*, the insertion domain is rotated and repositioned relative to its position in the crystal structure of the monomeric gp24. In addition the “spine” helix (residues 194–221) in the P domain of the gp24 crystal structure is bent, but it is straight in the cryo-EM structure of gp24* (Fig. 5C). This results in displacing the neighboring small helix (residues 181–190) and the P loop (residues 363–372) by approximately 7 Å.

The assembly of phage heads includes a maturation step, induced by a proteolytic event, resulting in an expansion of the head as it is filled with the DNA genome. The structure of gp24 before cleavage has greatest similarity to the HK97 prohead (44) (SI Appendix, Table S2), whereas the structure of the cleaved gp24* has the greatest similarity to the HK97 mature head (33, 45) (SI Appendix, Table S2). This suggests that the crystal structure of monomeric gp24 and the cryo-EM structure of gp24* correspond to the immature and mature states of the T4 head, respectively.

Gp23* Structure. As anticipated, the structure of gp23* is similar to that of gp24* (rmsd of 2.6–3.0 Å; Fig. 5E and F and SI Appendix, Table S3). Like gp24*, gp23* has an insertion domain in addition to the P and A domains of the HK97 fold (33). The insertion domain of gp23* is also connected via long linker polypeptides, which are topologically equivalent to the E-loop of HK97 (Fig. 5E). Gp23* does not include the 65 N-terminal residues of gp23 because these residues were proteolytically cleaved off during capsid assembly. Unlike HK97 and gp24*, the N-terminal region of gp23* contains a compact region, which we here refer to as the “N-fist” (residues 66–93; Fig. 5E), followed by a long N-arm (residues 94–110), similar to HK97. The N terminus of gp23* is located on the outer surface of the gp23* shell. However, in the immature prohead the gp23 cleavage site is located inside the prohead, where it is accessible to the prohead protease. Hence, presumably, the N-terminal region of

Fig. 3. The molecular surfaces of gp23 and gp24 colored according to electrostatic potential. The color ranges from red, corresponding to a potential of −10 kT/e−, to blue, corresponding to a potential of +10 kT/e−.

4 of 10 | www.pnas.org/cgi/doi/10.1073/pnas.1708483114 Chen et al.
Fig. 4. Organization of the T4 isometric capsid. (A) Schematic representation of a part of the surface of the T4 head. Three asymmetric units are highlighted in blue, green, and brown. The gp24* vertex pentameric capsomers are shown as pentagons in magenta. Gp23* capsomers are shown as hexagons. The red triangles represent the icosahedral threefold axes. The angles between adjacent capsomeres are shown in degrees on a yellow background. (B) An enlargement of the blue asymmetric unit shown in A. The asymmetric unit consists of 12 gp23* molecules (2 hexamers), 1 gp24* molecule, 11 Soc molecules (red), and 2 Hoc molecules (shown diagrammatically as a dotted circle).

gp23 adopts a different conformation in the prohead than that observed for gp23* in the mature capsid.

The icosahedral asymmetric unit of the cryo-EM structure contains two gp23* hexamers giving a total of 12 independent gp23* subunits (Fig. 4). The rmsd between corresponding Co atoms between pairs of subunits varies from 0.2 to 1.7 Å (*SI Appendix*, Fig. S2 and Table S4). The gp23* subunit A (Fig. 4) facing the gp24* pentamer has the greatest difference to all other subunits. The subunits within the hexamer closest to the pentamer (subunits A–F) have greater structural variability than the subunits within the other hexamer of the asymmetric unit (subunits G–L). Furthermore, the hexamer closest to the pentamer has greater similarity between the subunits related by twofold symmetry than those related by sixfold symmetry (Fig. 4 and *SI Appendix*, Table S4).

The differences among the gp23* subunits are in the N-terminal region (residues 66–110), the region that includes the insertion domain and the I-domain linkers (residues 129–252), and in the small helix within the P domain (residues 272–282; *SI Appendix*, Fig. S2). As these regions are involved in the intersubunit interactions (Fig. 6D), these differences are probably necessary for the subunits to adapt to slightly different environments in the capsid.

Each gp23* subunit interacts with four other subunits within its own hexamer and five subunits in neighboring capsomers. (Fig. 6D and *SI Appendix*, Fig. S3). Like in the case of gp24*, the insertion domain and the I-domain linkers make extensive contacts with a neighboring subunit from the same capsomer, thus stabilizing the capsomer structure. The I-domain linkers also interact with the N-fists of the gp23* subunits from other capsomers. In addition, one of the I-domain linkers interacts with the P-loop of another subunit located in a neighboring capsomer. Similar structural motifs (P-loop and E-loop) in HK97 are involved in the chainmail covalent cross-linking of subunits (33). The N-arm of gp23* interacts with an I-domain linker of a neighboring subunit belonging to the same capsomer. A part of the N-arm (residues 98–100) is involved in a parallel β-structure interaction with an I-domain linker of an adjacent gp23* subunit (Fig. 6F). The N-fist of each gp23* interacts with four different subunits, two from the same capsomer and two from another capsomer (*SI Appendix*, Fig. S3). In addition, the intercapsomer interactions are stabilized by attractive electrostatic forces between the P domains of gp23* subunits located in neighboring capsomers. The negatively charged small helix from one P domain (residues 272–282) interacts with a positively charged β-sheet region of another P domain. In particular, Glu<sup>273</sup> from the small P-domain helix interacts with Lys<sup>474</sup> and Arg<sup>476</sup> from another gp23* subunit. Electrostatic interactions between similar regions were also observed in the mature and immature heads of HK97 (33, 46). Collectively, a large network of intra- and intercapsomer interactions described here produce a highly stable capsid structure.

Soc Structure. The 9-kDa Soc protein attaches to the outer surface of the T4 head during the final stage of maturation (27). The cryo-EM structure shows that each Soc molecule interacts with three gp23* subunits, which belong to two different hexameric capsomers (Fig. 4B and *SI Appendix*, Fig. S4). A Soc binding site is made by the N-fist of one gp23* subunit, an I-domain linker from a second subunit belonging to the same capsomer, and an I-domain linker from another subunit belonging

A HK97 prohead II
Domain A

Domain P
E-loop

B HK97 head II
Domain A

N-arm

Domain P
E-Loop

C Crystal structure of T4 gp24
Domain A

Domain I

I-domain linker

Doman P

D Cryo-EM structure of T4 gp24*
Domain A

N-arm
Domain I

Domain P
I-domain linker

E Cryo-EM structure of T4 gp23*
Domain A

N-fist
N-arm
Domain I

Domain P

F T4 gp23* vs T4 gp24*
Domain A

N-arm
Domain I

Domain P
I-domain Linker

Fig. 5. Comparison of the T4 gp23 and gp24 monomer structures with HK97 capsid protein. (A and B) The immature and mature HK97 capsid structures. (C) The crystal structure of gp24. (D) The cryo-EM structure of gp24* reported here. (E) The cryo-EM structure of gp23* reported here. (F) The superposition of gp23* and gp24*.

to a different capsomer. Furthermore, the intercapsomer interactions are stabilized by trimeric interactions of Soc molecules at the quasi-threefold axes that relate three adjacent gp23* capsomers. Reinforcement by 200 such Soc triple clamps and 60 Soc monomers near the pentameric vertices that form an external, nearly continuous, cage (Fig. 1D) allows the T4 phage to survive under extreme conditions such as pH 11.

In the isometric capsid studied here, Soc binding sites were not fully occupied, and the occupancies of different sites (as determined in the structure refinement procedure) varied from approximately 0.6 to 0.4 ([SI Appendix](https://www.pnas.org/content/suppl/2018/01/10/1708483114.DCSupplemental), Table S5). The difference in the Soc binding was the result of small conformational changes in the gp23* I-domain linkers, which create part of the Soc binding sites. The Soc occupancy correlated with the angles (Fig. 4A) between the normals of the planes of adjacent hexameric capsomers (Fig. 7 and [SI Appendix](https://www.pnas.org/content/suppl/2018/01/10/1708483114.DCSupplemental), Table S5). The smaller the angle (i.e., the more nearly the neighboring hexamers are coplanar), the lower is the occupancy of the corresponding

A gp24* pentamer
B gp23* hexamer

Domain I
Domain I

C
D

N-arm
Domain P
N-fist
Domain P Domain I

E
F

I-domain
linker
P-loop
A26
A30
Domain A
A98
A100
I-domain
linker
P-loop


Fig. 6. Gp24 pentamer and gp23 hexamer structures. (A and B) Side views of the gp24 pentamer and the gp23 hexamer. The I domains of each subunit are located on the outer capsid surface forming the characteristic surface bumps. (C and D) Top views of gp24 pentamer and gp23 hexamer (view from the outside of the capsid shell). (E) Magnification of C shows that a part of the N-arm (residues 26–30) is involved in a parallel β-structure interaction with an I-domain linker of an adjacent gp24* subunit. (F) Magnification of D shows that a part of the N-arm (residues 98–100) is involved in a parallel β-structure interaction with an I-domain linker of an adjacent gp23* subunit.

Soc molecules. Thus, Soc provides the greatest reinforcement near the vertices, where there is probably the greatest strain in the gp23* hexagonal lattice. Earlier work had also shown that the stability of Soc binding to the gp23* lattice varied with local curvature (15, 47).

Hoc Density. The crystal structure of a Hoc molecule had been determined previously (28), although the structure was missing the capsid binding C-terminal domain 4 (48). The first three Ig-like domains form an elongated but not entirely straight structure. As there are six possible orientations about the hexamer axis with which a Hoc monomer can bind to the center of a gp23* hexamer, the cryo-EM map represents an averaged structure of the Hoc molecule. Moreover, the elongated Hoc molecule is likely to be flexible. As a result, the Hoc density is difficult to interpret.

The asymmetric unit of the isometric head contains two Hoc densities, located at the centers of the two independent gp23* hexamers (Fig. 4). The Hoc density at the center of the hexamer close to the icosahedral threefold axis was less interpretable compared with the Hoc density in the hexamer located close to the fivefold axis. Because the gp23* hexamer located near the fivefold vertices was less sixfold-symmetric compared with the other hexamer, there are probably preferred orientations for Hoc binding to this hexamer, which in turn results in the partially interpretable Hoc density. The interpretable Hoc features were an α-helix and two short β-strands, consistent with the secondary structure prediction for the C-terminal domain of Hoc (28).

Mutations That Determine Length of the Capsid. A variety of mutations in gp23 can alter the WT prolate structure of the head to isometric and giant heads (15, 36–38). The T4 isometric head used in the present study had an A275T mutation in gp23, near the quasi-threefold axis. This mutation is located in the negatively charged small helix of the gp23* P domain, which is involved in electrostatic interactions with the P domain from another capsomer (*SI Appendix*, Fig. S5). Although the side chain of this residue does not interact directly with the neighboring capsomer, the mutation probably modifies the position of the interacting helix, which in turn affects the intercapsomer interaction and angle. Another group of mutations, similar to A275T, that change the length of the head, are A268V, D285V, D287V, and D287N (*SI Appendix*, Figs. S5 and S6). All these are located in a relatively small region of gp23*, close to the short P-domain helix, and therefore may affect intercapsomer interactions.

Mutations A66D, A94V, and G97S also change the prolate capsid length. All these are located in the N-terminal region of gp23. Residue 66 is the N-terminal residue of gp23*, whereas residues 94 and 97 are located in the N-arm of gp23* (*SI Appendix*, Fig. S6). In the prohead, the N-terminal region of gp23 is located inside the capsid and probably interacts with the inner scaffolding core. Therefore, the gp23–core interactions play a role in the head length determination.

The third cluster of mutations that change the capsid length are T457A, R460C, and G461S. These sites are also located near the quasi-threefold axes, relating adjacent capsomers (*SI Appendix*, Figs. S4 and S5). In this quasi-threefold region, P-loops belonging to the

mass from their mean plane through the two independent hexamers per icosahedral asymmetric unit of the T4 isometric head are given in *SI Appendix*, Table S6. No distance was greater than 0.8 Å. Furthermore, superposition of one hexamer onto the other showed an rmsd of 1.5 Å between equivalent Cα atoms, with only 1% of Cα atoms deviating by more than 4 Å, which is probably not significant considering the 3.3-Å resolution of the cryo-EM map. If the hexamers and pentamers are planar, the angle between hexamers or between pentamers and hexamers will be determined by the T number. For instance, the angle between adjacent fivefold axes in an icosahedron is 63.44°. Thus, for a virus with T=1 symmetry, the angle between the normals of two adjacent pentamers has to be 63.44°. Hence, the amino acids along the periphery of a pentamer of a T=1 virus must bind two pentamers together so as to produce an angle of 63.44° between the pentamers. Larger T numbers require the introduction of hexamers into the array forming the capsid. Thus, the curvature between pentamers will be spread over the angular difference between adjacent pentamers and hexamers and between adjacent hexamers. For instance, in the present case of a T=13 capsid (Fig. 4A), the angular difference between the “blue pentamer” and the “green pentamer” is given roughly by the sum of the angles between the blue pentamer and the blue ABCDEF hexamer (29°), the blue ABCDEF hexamer and blue GHIJKL hexamers (6°), the blue GHIJKL hexamer and the green ABCDEF hexamer (18°; a sum of 53° so far), and also possibly between the green ABCDEF hexamer and the green pentamer (29°), which would give a total of 82°. This is not exactly the 63.44° required for icosahedral symmetry because the individual rotations are not exactly about a common axis of rotation and because the individual hexamers and pentamers do not necessarily have exact five- or sixfold symmetry.

On strictly geometric grounds, the angles between the normal to a pentamer and each of the normals to the surrounding hexamers have to be 37.62° to create a pentagonal hole in a hexagonal array. Furthermore, also on strictly geometric grounds, the angle between the normals of the adjacent hexamers surrounding a pentamer has to be 41.81°. These angles are fixed for any icosahedral assembly of pentamers and hexamers. The observed angles were 29° and 33°, respectively (Fig. 4A), for the isometric T4 assembly. The deviation of these angles from the theoretical values of 37.62° and 41.81°, respectively, must be the result of a lack of exact planarity and a lack of exact rotational symmetry.

Any mutation that alters the T number must have the property to change the angles between capsomers and therefore must occur at the periphery of the gp23* hexamers (*SI Appendix*, Fig. S6). Furthermore, mutations that change the T number would be more easily accommodated if the mutated amino acid were hydrophobic and therefore independent of the exact hydrophobic environment. In contrast, mutations of polar residues would require more accurate positioning with respect to the surrounding polar environment. Thus, for an amino acid to achieve a variety of similar but different contacts, as would be the case if the mutation changed the T number, the mutated amino acid would more likely have to be hydrophobic rather than polar. Hence, mutations that increase the variety of polymorphic particles might tend to increase the hydrophobicity of an amino acid at the interface between capsomers. This is roughly consistent with the observed variety of T4 head length-varying mutations described here.

## Conclusions

We report the 3.3-Å resolution structure of the ~50-MDa isometric bacteriophage T4 capsid that has a diameter of approximately 860 Å. This is one of the largest structures determined to date to near-atomic resolution. The major capsid protein, gp23*, has a poly-peptide fold similar to the phage HK97 capsid protein. However, unlike the HK97 capsid that is stabilized by covalent crosslinking of
subunits (33), the T4 capsid shows intricate noncovalent interactions among the subunits that, together, generate a very stable virus capsid. The T4 major capsid protein subunits are “glued” together by the I domains that make a network of interactions with the A and P domains of other gp23* subunits in the same hexameric capsomer. The I-domain is flanked by long linkers, which are topologically equivalent to the elongated “E” loop of the HK97 fold. The I-domain linkers are involved in extensive intra- and intercapsomer interactions that stabilize the capsid. The intercapsomer interactions are reinforced by electrostatic forces between the P domains of gp23* subunits from adjacent capsomers. Furthermore, the N-terminal regions of gp23* interact extensively with four neighboring subunits belonging to two different capsomers. These stabilizing interactions allow the capsid to withstand the internal pressure from the tightly packed DNA.

The capsid is further stabilized by Soc molecules, which form an external cage of 200 trimers, each clamping three adjacent capsomers. The Soc protein has the highest affinity for their binding sites located near pentameric vertices where there is the greatest strain on the capsids’ integrity, in part because of the bigger angle between the planes of adjacent capsomers.

Mutations that change the capsid length are mostly located in the capsomer’s periphery and therefore affect the angle between the planes of adjacent capsomers. In turn, this affects the positions where pentameric holes are created within the hexagonal array of hexamers, resulting in different triangulation numbers. Many of the mutations result in greater hydrophobicity of the binding interface, thereby probably imparting flexibility to support various environments at the intercapsomer interfaces as required by different T numbers.

The architecture of T4 capsid illustrates how an extremely stable virus capsid can be constructed, yet with considerable plasticity to alter the size of the capsid. Thus, relatively little change in the subunit structure can assemble stable structures of different size. The same principles will be relevant when considering the assembly of very small viruses such as ΦX174 (52) and very large viruses such as adenovirus (57–59), PBCV1 (55), and even Mimivirus (56).

## Materials and Methods

### Preparation of the Isometric Mutant Phage
Stocks of 17amNG178.23pt21–34c mutant phage were prepared on the amber suppressor *E. coli* strain B40. Construction of this mutant was described previously (31). Isometric phage can accommodate only >65% of phage T4 genome, hence it cannot productively infect *E. coli* and produce a plaque. Therefore, there is strong selection pressure to accumulate WT revertants. To minimize this, the 17amNG178.23pt21–34c phage stocks were prepared starting from a minute plaque produced on the amber suppressor *E. coli* strain B40. Large stock was then amplified by subculturing the minute plaque no more than twice to minimize the accumulation of revertants. Consequently, the plaque-forming titer of the phage stock is quite low, 1–3 × 10¹⁰ pfu/mL, but the number of virus particles as determined by SDS/PAGE is approximately 10 times higher. Heads produced from such stocks contained ~80% isometric empty capsid particles.

### Purification of Isometric Capsids
*E. coli* P301 (suppressor-minus) cells were grown at 37 °C on 1:1 mixture of Luria broth and M9 minimal medium supplemented with amino acids, infected with 17amNG178.23pt21–34c phage at a multiplicity of infection (MOI) of 2.5, and superinfected at the same MOI 7 min later. Infection was allowed to continue for another 20 min. The cells were pelleted by centrifugation at 7,000 rpm (4,300 × g) for 12 min. The pellet was then resuspended in a buffer containing 50 mM Tris·HCl, pH 7.5, 5 mM MgCl₂, 200 mM NaCl, and 150 mM imidazole. These conditions produced ~90% expanded empty proheads, of which ~80% were isometric. However, most of these heads were tailed by nonspecific binding of neck and tail proteins. High concentration of salt and imidazole in the buffer dissociated the tails and generated tailless empty proheads.

The crude extract of tailless isometric capsids prepared as described earlier was treated with DNasel (20 μg/mL) to digest the phage DNA. Several drops of chloroform were added to lyse the cells, and the sample was incubated at 37 °C for 30 min. The lysate was then centrifuged at 7,000 rpm (4,300 × g) for 10 min, and the pellet containing cell debris was discarded. The supernatant containing the heads was centrifuged at 17,000 rpm (34,500 × g) for 45 min.

The pellet was resuspended in a buffer containing 50 mM Tris·HCl, pH 7.5, and 5 mM MgCl₂, and layered on a 5-mL cesium chloride gradient. The gradients were centrifuged at 40,000 rpm for 1 h in a SWTi55 rotor. The proheads banded near approximately half the length of the gradient tube. The top portion of the band, which predominantly contained isometric proheads, was collected and dialyzed overnight against 10 mM Tris·HCl, pH 7.5, 50 mM NaCl, and 5 mM MgCl₂. The proheads were then purified by chromatography by using two ion-exchange columns, first on a DEAE-Sepharose column and second on a Mono-Q FPLC column. In each case, a linear 50–300-mM NaCl gradient was used to separate the expanded (i.e., mature) isometric capsids from any unexpanded capsids and contaminating DNA. The peak fractions containing the expanded particles were concentrated by Amicon membrane filter (30K cutoff) to approximately 10¹⁰ particles per microliter and stored at –80 °C.

### Preparation of Isometric Heads for Cryo-EM
Purified T4 isometric head sample was frozen by using a Gatan CP3 cryo-plunger. A 2.5-μL sample was applied onto the glow-discharged Lacey carbon grid with 3-nm-thick supporting carbon film, with a blotting time of 7 s with humidity maintained at greater than 85%. The grids were stored in liquid nitrogen for future use.

### Cryo-EM Imaging, Data Processing, and Resolution Assessment
An initial cryo-EM dataset was collected by using films installed on a Titan Krios microscope (FEI) at Purdue University. Approximately 40,000 particles were boxed for reconstruction from 1,512 films with pixel size of 1.1 Å per pixel. A reconstruction was calculated by using the program EMAN (60, 61), obtaining a resolution of 5.2 Å. The backbone structures of gp23* and gp24* were built with the guide of the previous gp24 crystal structure and gp23 predicted model.

A second cryo-EM data set was collected with a Falcon II direct detector installed on a Titan Krios microscope (FEI) at Sanford-Burnham-Prebys Medical Discovery Institute. Approximately 4,000 images were collected with a pixel size of 1.38 Å per pixel. Approximately 25,000 particles were boxed. Each image was composed of seven frames, with a total dose rate of approximately five electrons per square Ångström. The resulting reconstruction was calculated by using jspr (40) and had a resolution of 3.9 Å.

The final cryo-EM data set was collected by using the Purdue University Titan Krios microscope equipped with a Gatan K2 Summit detector. Approximately 3,500 images were collected by using a magnification of 180,000×, with pixel size of 0.81 Å at superresolution mode. Each image was composed of 45 frames. Each frame had an exposure time of 0.2 s and a dose rate of approximately three electrons per Ångström.

Motion correction was applied to each image (62). Averaged images were produced from all frames in the stack. The contrast transfer function correction was applied to each motion-corrected image. These images were used for boxing the particles. A total of 19,510 particles were boxed. A reference-free, 2D classification program implanted in the RELION software (39) was used to classify the boxed particles by a reciprocal space maximum-likelihood method. The 2D classification selected 17,954 particles for further data processing. The program jspr (40) was used for producing an initial model and further 3D refinement. The jspr program was also used to correct the anisotropic magnification of images produced by the microscope (63). To refine the 3D density map, the same particles were reboxed from the averaged images by using only frames 3–22 while excluding the initial 2 frames with large movements and the last 23 frames with greater radiation damage. The program jspr (40) was used for the refinement. The dataset of reboxed particles was split into two halves before the start of the refinement. The low-resolution model was then refined in each half dataset independently. The refinement stopped when there was no further improvement of the FSC. The FSC was calculated from the two independent reconstructions. The resolution of the reconstruction was determined by the point at which the FSC was 0.143. Both datasets were combined for calculation of the final map. A temperature factor of –100 Å² and a low-pass filter to 3.3 Å was applied for calculation of the final 3D map.

### Atomic Model Building and Refinement
The map showed continuous density for the main chain and distinct side-chain densities for bulky amino acid residues. Based on this map, the atomic models of the gp23*, gp24*, and Soc proteins within one asymmetric unit were built by using the COOT graphical program (42). The structure was refined in real space against the experimental electron density by using the phenix.real_space_refine program (64) from Phenix software (43) ([SI Appendix, Table S1](https://www.pnas.org/content/suppl/2023/01/10/2217600120.DCSupplemental)). The occupancies of the 11 Soc molecules located in the asymmetric unit were refined in reciprocal space by using the phenix.refine program (43).

Figures were prepared by using the programs Chimera (65) and PyMOL ([www.pymol.org](http://www.pymol.org)).

Chen et al.
ACKNOWLEDGMENTS. We thank Valorie Bowman for help with the data collection at Purdue University, Mark Swift and Niels Volkmann for help with data collection at Sanford-Burnham-Prebys Medical Discovery Institute, and Wei-Chun Tang for help with DNA sequencing of T4 phage mutants at The Catholic

1. Hatfull GF, Hendrix RW (2011) Bacteriophages and their genomes. Curr Opin Virol 1:298–303.
2. Hendrix RW (2003) Bacteriophage genomics. Curr Opin Microbiol 6:506–511.
3. Evilevitch A, Lavelle L, Knobler CM, Raspaud E, Gelbart WM (2003) Osmotic pressure inhibition of DNA ejection from phage. Proc Natl Acad Sci USA 100:9292–9295.
4. Tzlil S, Kindt JT, Gelbart WM, Ben-Shaul A (2003) Forces and pressures in DNA pack- aging and release from viral capsids. Biophys J 84:1616–1627.
5. Leiman PG, Kanamaru S, Mesyanzhinov VV, Arisaka F, Rossmann MG (2003) Structure and morphogenesis of bacteriophage T4. Cell Mol Life Sci 60:2356–2370.
6. Yap ML, Rossmann MG (2014) Structure and function of bacteriophage T4. Future Microbiol 9:1319–1327.
7. Liu S, et al. (2014) A viral packaging motor varies its DNA rotation and step size to preserve subunit coordination as the capsid fills. Cell 157:702–713.
8. Berndsen ZT, Keller N, Smith DE (2015) Continuous allosteric regulation of a viral packaging motor by a sensor that detects the density and conformation of packaged DNA. Biophys J 108:315–324.
9. Sun S, et al. (2008) The structure of the phage T4 DNA packaging motor suggests a mechanism dependent on electrostatic forces. Cell 135:1251–1262.
10. Rao VB, Feiss M (2015) Mechanisms of DNA packaging by large double-stranded DNA viruses. Annu Rev Virol 2:351–378.
11. Sathaliyawala T, et al. (2006) Assembly of human immunodeficiency virus (HIV) an- tigens on bacteriophage T4: A novel in vitro approach to construct multicomponent HIV vaccines. J Virol 80:7688–7698.
12. Shivachandra SB, et al. (2007) Multicomponent anthrax toxin display and delivery using bacteriophage T4. Vaccine 25:1225–1235.
13. Tao P, Mahalingam M, Rao VB (2016) Highly effective soluble and bacteriophage T4 nanoparticle plague vaccines against Yersinia pestis. Methods Mol Biol 1403: 499–518.
14. Tao P, et al. (2013) In vitro and in vivo delivery of genes and proteins using the bacteriophage T4 DNA packaging machine. Proc Natl Acad Sci USA 110:5846–5851.
15. Black LW, Showe MK, Steven AC (1994) Morphogenesis of the T4 head. Molecular Biology of Bacteriophage T4, ed Karam JD (Am Soc Microbiol, Washington, DC), pp 218–258.
16. Fokine A, et al. (2004) Molecular architecture of the prolate head of bacteriophage T4. Proc Natl Acad Sci USA 101:6003–6008.
17. Kostyuchenko VA, et al. (2005) The tail structure of bacteriophage T4 and its mech- anism of contraction. Nat Struct Mol Biol 12:810–813.
18. Leiman PG, et al. (2010) Morphogenesis of the T4 tail and tail fibers. Virol J 7:355.
19. Kostyuchenko VA, et al. (2003) Three-dimensional structure of bacteriophage T4 baseplate. Nat Struct Biol 10:688–693.
20. Taylor NM, et al. (2016) Structure of the T4 baseplate and its function in triggering sheath contraction. Nature 533:346–352.
21. Yap ML, et al. (2016) Role of bacteriophage T4 baseplate in regulating assembly and infection. Proc Natl Acad Sci USA 113:2654–2659.
22. Leiman PG, Chipman PR, Kostyuchenko VA, Mesyanzhinov VV, Rossmann MG (2004) Three-dimensional rearrangement of proteins in the tail of bacteriophage T4 on in- fection of its host. Cell 118:419–429.
23. Sun L, et al. (2015) Cryo-EM structure of the bacteriophage T4 portal protein assembly at near-atomic resolution. Nat Commun 6:7548.
24. Fokine A, Rossmann MG (2016) Common evolutionary origin of procapsid proteases, phage tail tubes, and tubes of bacterial type VI secretion systems. Structure 24: 1928–1935.
25. Fokine A, et al. (2005) Structural and functional similarities between the capsid pro- teins of bacteriophages T4 and HK97 point to a common ancestry. Proc Natl Acad Sci USA 102:7163–7168.
26. Rao VB, Black LW (1985) DNA packaging of bacteriophage T4 proheads in vitro. Ev- idence that prohead expansion is not coupled to DNA packaging. J Mol Biol 185: 565–578.
27. Ishii T, Yanagida M (1977) The two dispensable structural proteins (soc and hoc) of the T4 phage capsid; their purification and properties, isolation and characterization of the defective mutants, and their binding with the defective heads in vitro. J Mol Biol 109:487–514.
28. Fokine A, et al. (2011) Structure of the three N-terminal immunoglobulin domains of the highly immunogenic outer capsid protein from a T4-like bacteriophage. J Virol 85: 8141–8148.
29. Qin L, Fokine A, O’Donnell E, Rao VB, Rossmann MG (2010) Structure of the small outer capsid protein, Soc: A clamp for stabilizing capsids of T4-like phages. J Mol Biol 395:728–741.
30. Fokine A, et al. (2013) The molecular architecture of the bacteriophage T4 neck. J Mol Biol 425:1731–1744.
31. Iwasaki K, et al. (2000) Molecular architecture of bacteriophage T4 capsid: Vertex structure and bimodal binding of the stabilizing accessory protein, Soc. Virology 271: 321–333.
32. Olson NH, Gingery M, Eiserling FA, Baker TS (2001) The structure of isometric capsids of bacteriophage T4. Virology 279:385–391.

University of America. This work was supported by National Institutes of Health (NIH) Grant AI081726 (to V.B.R. and M.G.R.). NIH Grants S10 OD012372 and P01 GM098412-S1 (to D.H.) funded the purchase of the Titan Krios and Falcon II direct detection imaging device at Sanford-Burnham-Prebys Medical Discovery Institute.

33. Wikoff WR, et al. (2000) Topologically linked protein rings in the bacteriophage HK97 capsid. Science 289:2129–2133.
34. Fokine A, Rossmann MG (2014) Molecular architecture of tailed double-stranded DNA phages. Bacteriophage 4:e28281.
35. Baker ML, Jiang W, Rixon FJ, Chiu W (2005) Common ancestry of herpesviruses and tailed DNA bacteriophages. J Virol 79:14967–14970.
36. Doermann AH, Pao A, Jackson P (1987) Genetic control of capsid length in bacte- riophage T4: Clustering of ptg mutations in gene 23. J Virol 61:2823–2827.
37. Lane T, Eiserling F (1990) Genetic control of capsid length in bacteriophage T4. VII. A model of length regulation based on DNA size. J Struct Biol 104:9–23.
38. Mooney DT, Stockard J, Parker ML, Doermann AH (1987) Genetic control of capsid length in bacteriophage T4: DNA sequence analysis of petite and petite/giant mu- tants. J Virol 61:2828–2834.
39. Scheres SH (2012) RELION: Implementation of a Bayesian approach to cryo-EM structure determination. J Struct Biol 180:519–530.
40. Guo F, Jiang W (2014) Single particle cryo-electron microscopy and 3-D reconstruction of viruses. Methods Mol Biol 1117:401–443.
41. Henderson R, et al. (2012) Outcome of the first electron microscopy validation task force meeting. Structure 20:205–214.
42. Emsley P, Lohkamp B, Scott WG, Cowtan K (2010) Features and development of Coot. Acta Crystallogr D Biol Crystallogr 66:486–501.
43. Adams PD, et al. (2010) PHENIX: A comprehensive Python-based system for macro- molecular structure solution. Acta Crystallogr D Biol Crystallogr 66:213–221.
44. Gertsman I, et al. (2009) An unexpected twist in viral capsid maturation. Nature 458: 646–650.
45. Helgstrand C, et al. (2003) The refined structure of a protein catenane: The HK97 bacteriophage capsid at 3.44 Å resolution. J Mol Biol 334:885–899.
46. Gertsman I, Fu CY, Huang R, Komives EA, Johnson JE (2010) Critical salt bridges guide capsid assembly, stability, and maturation behavior in bacteriophage HK97. Mol Cell Proteomics 9:1752–1763.
47. Aebi U, et al. (1977) Capsid fine structure of T-even bacteriophages. Binding and localization of two dispensable capsid proteins into the P23* surface lattice. J Mol Biol 110:687–698.
48. Sathaliyawala T, et al. (2010) Functional analysis of the highly antigenic outer capsid protein, Hoc, a virus decoration protein from T4-like bacteriophages. Mol Microbiol 77:444–455.
49. Johnson K, Condie B, Mooney DT, Doermann AH (1992) Mutations that eliminate the requirement for the vertex protein in bacteriophage T4 capsid assembly. J Mol Biol 224:601–611.
50. Caspar DL, Klug A (1962) Physical principles in the construction of regular viruses. Cold Spring Harb Symp Quant Biol 27:1–24.
51. Tsao J, et al. (1991) The three-dimensional structure of canine parvovirus and its functional implications. Science 251:1456–1464.
52. McKenna R, et al. (1992) Atomic structure of single-stranded DNA bacteriophage phi X174 and its functional implications. Nature 355:137–143.
53. Yan X, Olson NH, Van Etten JL, Baker TS (1998) Cryo-electron microscopy and image re- construction of PBCV-1, an algal virus with T=169 lattice symmetry. Electron Microscopy, 1998: Proceedings of the 14th International Congress on Electron Microscopy, Cancún (Mexico), eds Calderón Benavides HA, Yacamán MJ (Institute of Physics, Bristol, UK), Vol 4, pp 775–776.
54. Nandhagopal N, et al. (2002) The structure and evolution of the major capsid protein of a large, lipid-containing DNA virus. Proc Natl Acad Sci USA 99:14758–14763.
55. Zhang X, et al. (2011) Three-dimensional structure and function of the Paramecium bursaria chlorella virus capsid. Proc Natl Acad Sci USA 108:14837–14842.
56. Xiao C, et al. (2009) Structural studies of the giant mimivirus. PLoS Biol 7:e92.
57. Liu H, et al. (2010) Atomic structure of human adenovirus by cryo-EM reveals inter- actions among protein networks. Science 329:1038–1043.
58. Reddy VS, Natchiar SK, Stewart PL, Nemerow GR (2010) Crystal structure of human adenovirus at 3.5 Å resolution. Science 329:1071–1075.
59. Rux JJ, Burnett RM (2004) Adenovirus structure. Hum Gene Ther 15:1167–1176.
60. Ludtke SJ (2010) 3-D structures of macromolecules using single-particle analysis in EMAN. Methods Mol Biol 673:157–173.
61. Ludtke SJ, Baldwin PR, Chiu W (1999) EMAN: Semiautomated software for high- resolution single-particle reconstructions. J Struct Biol 128:82–97.
62. Li X, et al. (2013) Electron counting and beam-induced motion correction enable near- atomic-resolution single-particle cryo-EM. Nat Methods 10:584–590.
63. Yu G, et al. (2016) An algorithm for estimation and correction of anisotropic mag- nification distortion of cryo-EM images without need of pre-calibration. J Struct Biol 195:207–215.
64. Afonine PV, Headd JJ, Terwilliger TC, Adams PD (2013) New tool: phenix. real_space_refine. Computational Crystallography Newsletter 4:43–44.
65. Pettersen EF, et al. (2004) UCSF Chimera—a visualization system for exploratory re- search and analysis. J Comput Chem 25:1605–1612.
