
Glycoconj J
DOI 10.1007/s10719-017-9799-y

---

**ORIGINAL ARTICLE**

---

### Identification and biochemical characterization of WbwB, a novel UDP-Gal: Neu5Ac-R α1,4-galactosyltransferase from the intestinal pathogen *Escherichia coli* serotype O104

Diana Czuchry${}^{1}$ · Walter A. Szarek${}^{2}$ · Inka Brockhausen${}^{1}$

---

Received: 1 August 2017 / Revised: 11 September 2017 / Accepted: 13 September 2017

© Springer Science+Business Media, LLC 2017

---

#### Abstract
The intestinal pathogen *Escherichia coli* serotype O104:H4 (ECO104) can cause bloody diarrhea and haemolytic uremic syndrome. The ECO104 O antigen has the unique repeating unit structure [4Galα1–4Neu5,7,9Ac₃α2–3Galβ1–3GalNAcβ1-], which includes the mammalian sialyl-T antigen as an internal structure. Previously, we identified WbwC from ECO104 as the β3Gal-transferase that synthesizes the T antigen, and showed that α3-sialyl-transferase WbwA transfers sialic acid to the T antigen. Here we identify the *wbwB* gene product as a unique α1,4-Gal-transferase WbwB that transfers Gal from UDP-Gal to the terminal sialic acid residue of Neu5Acα2–3Galβ1–3GalNAcα-diphosphate-lipid acceptor. NMR analysis of the WbwB enzyme reaction product indicated that Galα1-4Neu5Acα2–3Galβ1–3GalNAcα-diphosphate-lipid was synthesized. WbwB from ECO104 has a unique acceptor specificity for terminal sialic acid as well as the diphosphate group in the acceptor. The characterization studies showed that WbwB does not require divalent metal ion as a cofactor. Mutagenesis identified Lys243 within an RKR motif and both Glu315 and Glu323 of the fourth EX₇E motif as essential for the activity. WbwB is the final glycosyltransferase in the biosynthesis pathway of the ECO104 antigen repeating unit. This work contributes to knowledge of the biosynthesis of bacterial virulence factors.

#### Keywords
O antigen · Galactosyltransferase · E.coli O104 · Biochemical identification of WbwB · NMR of reaction product

---

### Introduction

The human pathogen *Escherichia coli* (*E. coli*) O104:H4 (ECO104) is a Shiga toxin-producing *E.coli* (STEC) or enterohaemorrhagic *E.coli* (EHEC) that can cause bloody diarrhea and lead to hemolytic uremic syndrome and kidney failure. A massive outbreak of food-borne illness caused by *E. coli* O104:H4 in Germany in 2011 resulted in more than 3000 people infected and more than 50 deaths [1–3]. The pathogenic, potentially fatal and multi-antibiotic resistant nature of *E. coli* O104:H4 and other STECs requires specific prevention and treatment methods. Vaccines are not yet available for this pathogen but could be based on the structure of the serotype-specific O104 antigen.

O antigens of Gram negative bacteria are involved in interactions with the environment and are considered to be virulence factors [4]. It is thus important to understand their biosynthesis and to identify new targets for treatment and technology to synthesize vaccines. The biosynthesis of O antigens is still poorly understood due to the difficulty in finding appropriate substrates for the enzymes involved. The ECO104 antigen has the repeating unit [4Galα1–4Neu5,7,9Ac₃α2–3Galβ1–3GalNAcβ1-]n [5], which includes mimics of the human T and sialyl-T antigens. The repeating units are thought to be assembled on an undecaprenol-phosphate (P-Und) lipid carrier by sequentially acting specific glycosyltransferases (GTs) [6].

The ECO104 antigen biosynthesis gene cluster [7] contains the GT genes *wbwC*, *wbwA* and *wbwB*. In addition, the *wckD* gene has sequence similarity to the gene encoding sialate O-

acetyltransferase NeuD and may be responsible for the 7-, 9-O-acetylation of sialic acid. The *nnaB*, *nnaC* and *nnaA* genes are thought to be involved in the synthesis of CMP-Neu5Ac. The presence of a translocase gene *wzx* and polymerase gene *wzy* suggests that the synthesis of the ECO104 antigen follows the polymerase-dependent pathway. In this pathway (Fig. 1), the O antigen repeating unit is synthesized by the sequential action of GTs on the inner leaflet of the inner membrane, utilizing the membrane-bound lipid carrier P-Und [11]. The nucleotide donor substrates required for these reactions are synthesized in the cytosolic compartment. The first reaction of the ECO104 repeating unit synthesis is the reversible transfer of GlcNAc-phosphate from UDP-GlcNAc to P-Und by an analog of WecA [12] to form GlcNAc-diphosphate undecaprenol (GlcNAc-PP-Und). In ECO104 and several related strains, GlcNAc-PP-Und may be converted to GalNAc-PP-Und by a 4-epimerase that is identical to the enzyme identified in *E. coli* O157 and does not act on UDP-GlcNAc [8]. The repeating unit of ECO104 is then assembled on GalNAc-PP-Und. We have previously synthesized a GalNAc-PP-Und analog, GalNAcα-PO₃-PO₃-(CH₂)₁₁-O-phenyl (GalNAc-PP-PhU) that was used as the acceptor substrate for β1,3-Gal-transferase WbwC [9] to form the mammalian cancer-associated T antigen. Subsequently, sialic acid (Neu5Ac) is transferred by α2,3-sialyltransferase WbwA to form the sialyl-T antigen, Neu5Aca2–3Galβ1–3GalNAca-R [10]. WbwC and WbwA have distinct specificities for the nucleotide donors and sugar moieties of the acceptor substrates and were

both shown to require the pyrophosphate group in the acceptor substrate. According to the structure of the O antigen [5], the final GT in the ECO104 pathway is expected to synthesize the unusual linkage of Galα1–4Neu5Ac; therefore, due to sequence similarity to other retaining GTs, WbwB had been proposed to be the final α1,4-Gal-transferase [7] but had not been biochemically characterized.

After the repeating unit has been synthesized, the oligosaccharide-PP-Und is translocated to the periplasmic leaflet by Wzx (Fig. 1) [13]. The multiple membrane-spanning polymerase Wzy then inserts the new repeating unit at the reducing end of the growing O antigen chain [11]. Wzy polymerases appear to be O antigen-specific and often invert the anomeric linkage of the reducing end sugar moiety of the repeating unit [13, 14]. Thus, in the polymerized ECO104 antigen the original α-linkage of GalNAcα-PP-Und is inverted to form the GalNAcβ-linkage. Finally, the O antigen polymer is ligated to the core oligosaccharide linked to lipid A by ligase WaaL [15] to complete the lipopolysaccharide (LPS) which is then transported by the Lpt complex to the outer surface of the cell [16], exposing the O antigen to the environment.

In this paper we describe the biochemical function of WbwB as an α1,4-Gal-transferase that transfers a single Gal residue to sialic acid termini. WbwB has a unique specificity for UDP-Gal, for a terminal sialic acid residue and the diphosphate-lipid moiety of the acceptor substrate. This work

![Diagram](attachment:diagram.png)

Fig. 1 Proposed pathway in the biosynthesis of the ECO104 antigen repeating unit. The repeating unit of *E.coli* O104:H4 (ECO104) is assembled at the inner face of the cytoplasmic membrane on an undecaprenol-phosphate (P-Und) lipid intermediate. The first reaction in the pathway of ECO104 synthesis is the transfer of GlcNAc-phosphate from UDP-GlcNAc to P-Und by GlcNAc-1-P transferase WecA. ECO104 has a 4-epimerase identical to that of *E.coli* O157 [8] that converts GlcNAc-PP-Und to GalNAc-PP-Und, which is the acceptor substrate for β1,3-Gal-transferase WbwC that synthesizes the T antigen [9]. The α2,3-sialyltransferase WbwA then transfers sialic acid from CMP-Neu5Ac to Gal to form the sialyl-T antigen [10]. Finally, α1,4-Gal-transferase WbwB (this work) adds a Gal residue to sialic acid. All of these glycosyltransferases require the presence of phosphate(s) in the acceptor intermediates. The flippase Wzx translocates the repeating unit to the periplasm for further processing of the O antigen-linked lipopolysaccharide in the polymerase pathway. It is not yet known at what stage O-acetyltransferase WckD adds the 7- and 9-acetyl groups to sialic acid

confirms that WbwB is the final GT to add the fourth and last sugar in the ECO 104 repeating unit biosynthesis pathway.

**Materials and methods**

**Materials**

All reagents were purchased from Sigma-Aldrich-Millipore unless stated otherwise. Radioactive nucleotide sugars were obtained from American Radiolabeled Chemicals.

CMP-Neu5Ac was from EMD Millipore, and UDP-Gal was a kind gift from Beat Ernst, University of Basel, Switzerland. Antibiotics, premixed Lennox-Luria Broth and isopropylthiogalactoside (IPTG) were obtained from Bioshop. Ni²⁺-NTA Sepharose was from Qiagen. C₁₈ Sep-Pak (short) cartridges were from Waters. The synthesis of GalNAcα-PP-PhU was described before [9] and Galβ1-3GalNAcα-PP-PhU was synthesized from GalNAcα-PP-PhU with WbwC [9]. Neu5Acα2-3Galβ1-3GalNAcα-Bn was synthesized from Galβ1-3GalNAcα-Bn with human α3-sialyltransferase ST3Gal1, and the WbwB acceptor Neu5Acα2-3Galβ1-3GalNAcα-PP-PhU was synthesized with WbwA as described [10]. Other sialyl-oligosaccharides were kindly donated by K.L. Matta (State University of New York). Neu5Acα2-(4-methyl)umbelliferyl was from Sigma.

**Synthesis of the wbwB gene and expression of the WbwB protein**

A pRSETA plasmid containing the wbwB gene and ampicillin resistance gene was obtained from GeneArts (Life Technologies). The WbwB protein has a His₆ tag at the amino terminus. The sequence of the WbwB protein without the His₆ tag is shown in Fig. 2. The plasmid was transformed into BL210-DE3-Codon Plus bacteria containing a chloramphenicol resistance gene, grown for 16 h on a plate containing Luria-Bertani (LB) agar with 100 μg/ml ampicillin and 50 μg/ml chloramphenicol. A single colony was picked and

grown in 5 ml LB broth with 100 μg/ml ampicillin and 50 μg/ ml chloramphenicol. Bacteria were transferred to 500 ml of LB containing 100 μg/ml of ampicillin and 50 μg/ml chlor-amphenicol at 37°C with 200 rpm shaking until the optical density at 600 nm was 0.6. The expression of His-tagged protein was induced with 1 mM IPTG for 12 h at 25°C at 200 rpm shaking and was confirmed by Sodium dodecylsulfate polyacrylamide gel electrophoresis (SDS-PAGE) and Western blotting. Bacterial cells were harvested by centrifugation at 3500×g for 10 min. The pellet was resuspended in 10 ml phosphate-buffered saline pH 7.4 (PBS) with 20% glycerol and frozen in aliquots at −20°C for activity assays.

**Expression of SiaD constructs**

The polymerizing sialyltransferase SiaD derived from Neisseria meningitides NmW-135 is the only known enzyme other than WbwB that adds Gal to sialic acid. Three plasmids containing SiaD constructs with C-terminal His-tags were kindly provided by R. Gerardy-Schahn, Medizinische Hochschule, Hannover, Germany. These plasmids included genes of a) the full length enzyme with sialyltransferase domain at the carboxy-terminus and Gal-transferase domain at the amino-terminus, b) Gal-transferase domain (amino acids 1 to 398), and c) full length enzyme with a S972A mutation that inactivates the sialyltransferase activity as described [17, 18]. The plasmids were transformed into BL21-DE3-Codon Plus bacteria containing a chloramphenicol resistance gene, grown for 16 h on a plate containing LB agar with 100 μg/ml ampicillin and 50 μg/ml chloramphenicol. A single colony was picked and grown in 5 ml LB broth with 100 μg/ml ampicillin and 50 μg/ml chlor-amphenicol. Bacteria were transferred to 500 ml of LB containing 100 μg/ml of ampicillin and 50 μg/ml chlor-amphenicol at 37°C with 200 rpm shaking until the optical density at 600 nm was 0.6. The expression of SiaD proteins was induced with 1 mM IPTG for 12 h at 25°C at 200 rpm shaking and was confirmed by SDS-PAGE and Western blotting.

1 mknilfinna lsgtggarvi lnlakslrrr ghkvallldr vdnihykidd gieiyvrdkf  
61 kikkvnpti nknndsqafft nqsnsvffnf iklyakkls ffyflkapce myafkkfint  
121 kkfdainnn vyvnvdriyf esklsnnyyv nfhnspieif srrefstllp lskifrnvkl  
181 lsvskgiade lltlkgfndk eiitiynpfn ftel **egnsri qcegniplpk efivtvstlt**  
241 **drkrvdrtik ampkiiireyg eidlliigeg** qlrndlqnlv kelniekvh flgfqtnpyy  
301 finkaql lil ssdseglptv iieslilgtp vlstdcptgp neilvnwgde slvsltktsd  
361 ennicnelae kairllckky dkkyvkhksd lqrfdekkis tsweilf  

Fig. 2 WbwB amino acid sequence without the polyhistidine tag. The major glycosyltransferase (GT) domain of WbwB from E.coli O104:H4 spans from amino acids 225 to 351 (Blue). WbwB has five EX₇E motifs that are underlined; the fourth EX₇E sequence (E315 to E323) is conserved. Potentially important and mutated amino acids within the GT domain are shown in red. Mutants R242A, K243A, R244A, D313A, E315A and E323A were studied

Mutations

Six single point mutations of WbwB were performed by Mutagenex Inc. Single amino acids were replaced by alanine: R242A, K243A, R244A, D313A, E315A and E323A (Fig. 2). The pRSETA plasmids containing the mutated genes and wild type WbwB plasmid were each transformed into competent BL21 bacteria cells, and grown overnight and expressed as described above. Mutant proteins were purified and monitored as described below.

Purification, SDS-PAGE and Western blot

Proteins were isolated using Ni²⁺-NTA Sepharose affinity chromatography and identified as follows. The bacterial pellet was resuspended in 10 ml of lysis buffer (50 mM sodium phosphate, 5% glycerol, 300 mM NaCl) and was lysed by sonication 3 times for 15 s with 1 min on ice in between. Lysates were centrifuged at 48,000×g for 20 min at 4 °C to precipitate membrane components. The soluble cellular components were applied to a column of 5 ml pre-equilibrated Ni²⁺-NTA Sepharose. The column was washed with 10 column volumes of wash buffer (50 mM sodium phosphate, 300 mM NaCl). WbwB was eluted with 10 ml of elution buffer (150 mM imidazole, 50 mM sodium phosphate, 300 mM NaCl). Expression and purification was monitored by 12% SDS PAGE at 120 V and measuring enzyme activity. The eluted fractions containing WbwB were pooled and dialyzed overnight with 50 mM of sodium phosphate and 1 mM dithiothreitol (DTT). For Western blots, proteins were transferred overnight to a nitrocellulose membrane at 4 °C and 30 V. Blocking was performed by placing the membrane in 5% skim milk powder in TBST (Tris-buffered saline, 0.1% Tween 20) for 1 h with shaking. The membrane was washed 3 times with TBST for 10 min each, and then incubated overnight at 4 °C with the primary antibody monoclonal mouse anti-His IgG (Cell Biolabs) in a 1:5000 dilution in 5% milk in TBST with shaking. The membrane was washed 3 times with TBST for 10 min and incubated for 1 h at rt. with secondary antibody, 1:10,000 goat anti-mouse IgG (Santa Cruz Biotechnologies) in TBST with 5% milk. The membrane was washed 3 times with TBST for 10 min. One ml of each of the two Clarity ECL Substrates (Bio-Rad) was added to the membrane for 5 min. The film was exposed to the membrane for 1 min, developed and bands were visualized using an imager. Purified enzymes were stored at -80 °C.

Glycosyltransferase assays

To prepare bacterial homogenates for glycosyltransferase assays, BL21 bacteria containing WbwB in 20% glycerol/PBS were thawed and isolated by centrifugation. Bacteria were disrupted by sonication in 50 mM sucrose for 15 s (3 times)

with 1 min intervals on ice. Initial WbwB Gal-transferase assays were carried out with homogenates. Standard Gal-transferase assay mixtures contained in a total volume of 40 μl: 10 μl bacterial homogenate (2 mg protein/ml), 5 mM MnCl₂, 0.125 M Tris-HCl, pH 7, 1 mM UDP-[6-³H]Gal (2900 cpm/nmol), and 0.01 mM Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU prepared as described below [9, 10]. Mixtures were incubated for 20 min at 37 °C. Negative controls lacked the acceptor substrate. The assay mixtures were applied to short C₁₈ Sep-Pak columns (Waters) equilibrated in water. The reaction product was eluted with 3 ml MeOH and quantified by scintillation counting. Enzyme product was also separated by high pressure liquid chromatography (HPLC) using a C₁₈ column and acetonitrile-water mixtures as the mobile phase. Assays were carried out in at least duplicate determinations with <10% difference between assays. Sialyltransferase assays using WbwA and ST3Gal I, and WbwC Gal-transferase assays were carried out as described before [9, 10]. SiaD constructs [17, 18] were assayed for Sialyl- and Gal-transferase activities using Galα1–4Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU and Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU, respectively, as acceptors.

Production of large-scale WbwB enzyme product

The non-radioactive enzyme product for WbwB was prepared from GalNAcα-PP-PhU in a one-pot reaction with WbwC, WbwA and WbwB. WbwC synthesized Galβ1–3GalNAcα-PP-PhU, which was converted to Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU by WbwA. WbwB then synthesized Galα1–4Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU. The reaction mixture contained in a total volume of 28 ml: 7 ml of WbwC homogenate (28 mg protein) in 50 mM sucrose, 7 ml of WbwA homogenate (28 mg protein) in 50 mM sucrose, 7 ml WbwB homogenate in 50 mM sucrose (28 mg protein), 7 μmol GalNAcα-PP-PhU (the relatively low amount ensured total conversion), 20 μmol CMP-Neu5Ac, 28 μmol UDP-Gal, 0.14 μmol MnCl₂ and 3.5 μmol 2-(N-morpholino)ethanesulfonic acid (MES) buffer, pH 7. After incubation for 1 h at 37 °C, 30 ml water were added, the mixture was centrifuged at 12000×g and the supernatant was subjected to purification on 60 C₁₈ Sep-Pak columns. Hydrophilic compounds were eluted from each column with 4 ml water. Fractions (3 ml) eluted with MeOH were pooled, and MeOH was removed by rotor evaporation followed by lyophilization. One ml of water was added, and insoluble residue was removed by centrifugation in a microcentrifuge at 10500×g. Parallel small scale radioactive WbwB assays that utilized non-radioactive WbwA product Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU and radioactive UDP-Gal served to quantify the amount of WbwB product, to obtain a standard compound for HPLC and to obtain substrate for galactosidase digestion.

Analysis of WbwB reaction product

Non-radioactive WbwB reaction product was analyzed for its molecular mass and purity by electrospray ionization mass spectrometry (ESI-MS) in the negative-ion mode [10]. For linkage analysis by nuclear magnetic resonance (NMR), WbwB product was evaporated 3 times and dissolved in D₂O and analyzed by NMR using a 600 MHz Avance-600 Bruker spectrometer using XWINNMR software [9]. To determine the anomeric Gal linkage synthesized by WbwB, radioactive WbwB product was subjected to digestion by green coffee bean and *Aspergillus niger* α-galactosidases and Jack bean and bovine testicular β-galactosidase as described before [9]. After galactosidase treatment for 30 min, mixtures were separated by C₁₈ Sep-Pak columns and the ratios of undigested WbwB reaction product (eluted with MeOH) and free cleaved Gal (eluted with water) were determined by scintillation counting.

To test if the repeating unit made by WbwB had the antigenicity of ECO104, polyclonal goat antibody to ECO104 extract (KPL) was used according to the manufacturer's instructions. Fifty nmoles of reaction products of WbwB, WbwA and WbwC, as well as standards with related structures and heat-killed ECO104 cells (KPL) as a positive control, were treated with anti-ECO104 antibody at rt. overnight, followed by treatment with Horse radish peroxidase-labeled secondary anti-goat IgG (KPL) for 2 h. Ten % of the mixtures (50 μl) were spotted directly onto nitrocellulose. The remainder of compounds was applied to C18 Sep-Pak, eluted with MeOH, dried, resuspended in 50 μl water and spotted onto nitrocellulose membrane. Spots were visualized with Westone (iNtRON Biotechnology) and exposure on x-ray film.

Results

Sequence analysis and structure prediction of WbwB

WbwB has been classified into the large GT4 family of Carbohydrate-active Enzymes (CAZy) with more than 83,000 entries. GT4 enzymes are predicted to have a GT-B fold consisting of two βαβ Rossmann-like domains, connected by a linker region. In contrast to inverting enzymes, the characterized GT4 enzymes do not require a divalent metal ion for activity and do not have conserved DxD motifs [19]. Uniprot has identified two GT-like domains in WbwB. Amino acids 15 to 210 of WbwB comprise the first GT-like domain and amino acids 225 to 351 comprise the second GT domain, which has a much higher homology to other, mostly uncharacterized GTs and may contain the nucleotide sugar binding domain.

WbwB has five EX₇E motifs in the Carboxy-terminal domain with yet unknown function. PglJ, an α1,4-GalNAc transferase involved in the assembly of the N-linked oligosaccharide in *Campylobacter jejuni*, has 54% sequence identity with the domain of WbwB comprising amino acids 225 to 351 (Uniprot). This PglJ was shown to transfer GalNAc from UDP-GalNAc in α1–4 linkage to another GalNAc residue, linked to Bacillosamine-PP-Und [20]. PglJ from *Hafnia alvei* with 54% sequence identity with amino acids 225 to 351 of WbwB is thought to transfer a GalNAc residue in α1–4 linkage directly to bacillosamine-PP-Und. This enzyme also has an RKR motif and a DXEX₇E sequence in a similar position to that of WbwB. This suggests that the RKR motif may possibly be involved in binding the diphosphate of the acceptor.

WbwB shares up to 28% sequence identity with other GT4 enzymes (Table 1) that have been shown to have retaining activity. The Gal-transferase domain (residues 1 to 398) of the bifunctional, alternating α1,4-Gal-transferase/α2,6-sialyltransferase SiaD NmW-135, shown to have α4-Gal-transferase activity [17], has only 20% sequence identity with WbwB. It is striking that many of the GT4 enzymes, including SiaD, contain a conserved EX₇E motif in the C-terminal domain.

In order to estimate the overall fold and structure of WbwB, the sequence was analyzed by PHYRE2 homology modeling tool [21]. The program chose sucrose-P-synthase SpsA (Ho) from *Halothermothrix orenii* as the major template with 100% confidence and 99% coverage [22]. In SpsA (Ho), Arg270 and Lys275 appear to be involved in binding UDP-Glc. Lys 275 corresponds to Lys243 of WbwB, which could bind phosphates of UDP-Gal [23]. The second PHYRE2 homology match was the crystal structure of MshA from *Corynebacterium glutamicum* with UDP and UDP-GlcNAc and this structure provided information of key residues in the donor binding site. In the MshA structure complexed with UDP solved by Vetting et al. [24] the Glu324 binds the ribose moiety of UDP and Arg231 and Lys236 interact with the UDP pyrophosphate. Sequence alignment showed that Lys236 and Glu324 of MshA aligned with Lys243 and Glu323 of WbwB. Thus, according to PHYRE2, one or more of the three residues Arg242, Lys243 and Glu323 in WbwB may be involved in UDP-Gal binding.

Enzyme characterization

WbwB was highly active in the bacterial homogenate in 50 mM sucrose up to 500 nmol/h/mg with 0.01 mM Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU as the acceptor substrate. WbwB was also purified on Ni²⁺-NTA Sepharose and eluted with 150 mM imidazole. The SDS PAGE analysis showed a prominent band at 48 kDa in the eluted fractions. The total activity of the purified WbwB enzyme fractions was about 25% of the total activity in the cell lysate.

The addition of 5 mM ethylenediaminetetraacetic acid (EDTA), MnCl₂, MgCl₂ or Co-acetate to the assay mixtures

Table 1 Comparison of amino acid sequences of GT 4 family enzymes

| Protein          | Species       | Activity                  | Sequence ID (%) | Accession number |
|------------------|---------------|---------------------------|-----------------|------------------|
| WbwB             | ECO 104       | α4Gal-T                   | 100             | Q93NP7           |
| WbwB             | Np            | ?                         | 28              | B9L6V8           |
| PglJ             | Cj            | α4GalNAc-T                | 54<sup>a</sup>   | Q0P9C7           |
| PglJ             | Ha            | α4GalNAc-T                | 54<sup>a</sup>   | A0A172X015       |
| WaaB             | EC K-12       | α6Gal-T                   | 27              | O68206           |
| WasC             | As            | α6Gal-T                   | 22              | C0JKQ0           |
| WcfP             | Bf            | αGal-T                    | 22              | Q93QV9           |
| BshA             | Ba            | αGlcNAc-T                 | 22              | Q81ST7           |
| WalW             | Ea            | GT                        | 21              | D4HUA3           |
| SiaD GalT domain | NmW-135       | α4Gal-T (polymerizing)    | 20              | B5WYL4           |
| WbnH             | ECO 86        | α3GalNAc-T                | 20              | P0DMP6           |
| WbdH             | ECO 111       | α3Gal-T                   | 18              | C8UCZ0           |
| WaaG             | EC K12        | α3Glc-T                   | 17              | P25740           |
| PimB             | Cg            | αMan-T                    | 16              | Q8NNK8           |
| PimA             | Ms            | Man-T                     | 15              | A0QWG6           |
| MshA             | Cg            | αGlcNAc-T                 | 15              | Q8NTA6           |
| WsaF             | Gs            | β2L-Rha-T                 | 15              | Q7BG50           |
| SpsA             | Ho            | sucrose-P synthase        | 12              | B2CCB8           |

Sequences of CAZy GT 4 enzymes were compared using UniProt. As, Aeromonas salmonicida; Ba, Bacillus anthracis; Bf, Bacteroides fragilis; Cg, Corynebacterium glutamicum; Cj, Campylobacter jejuni; Ea, Erwinia amylovora; EC, E.coli; Gs, Geobacillus stearothermophilus; Ha, Hafnia alvei; Ho, Halothermothrix orenii; Ms., Mycobacterium smegmatis; Nm, Neisseria meningitides; Np, Nautilia profundicola; Sa, Staphylococcus aureus; -T, -transferase

<sup>a</sup> Sequence was compared to the GT domain of WbwB (amino acids 225 to 351). All of these glycosyltransferases are retaining enzymes and have EX<sub>7</sub>E sequences

using purified WbwB gave similar activities showing that metal ion cofactors were not required for activity as expected. However, the addition of 5 mM Zn-acetate abolished the enzyme activity, suggesting that Zn may interfere with the enzyme protein structure and activity. A broad pH optimum of WbwB activity was found between pH 6 and 7.

Substrate specificity of WbwB

The nucleotide donor specificity of purified WbwB was tested by replacing 1 mM UDP-Gal with UDP-Glc, UDP-GalNAc,

UDP-GlcNAc or CMP-Neu5Ac in the standard assay. Only UDP-Gal served as a donor substrate with an apparent K<sub>M</sub> of 0.24 mM, with all other nucleotide sugars yielding < 5% of the activity with UDP-Gal. This shows a distinct specificity of WbwB for UDP-Gal.

The acceptor substrate specificity of purified WbwB was determined with a number of sialic acid-terminating synthetic compounds, including the WbwA product, Neu5Aca 2-3Galβ1–3GalNAca-PP-PhU. The biosynthetic intermediates, GalNAc-PP-PhU and Galβ1–3GalNAc-PP-PhU, were also tested (Table 2). The compounds contained a hydrophobic aglycone

Table 2 WbwB acceptor specificity

| Acceptor substrate                                      | Concentration (mM) | Activity (%) |
|--------------------------------------------------------|--------------------|--------------|
| Neu5Aca 2–3Galβ1–3GalNAca-PP-PhU (WbwA product)         | 0.01               | 100          |
| Galβ1–3GalNAca-PP-PhU                                   | 0.1                | <1           |
| GalNAca-PP-PhU                                         | 0.1                | <1           |
| Neu5Aca 2–3Galβ1–3GalNAca-Bn                            | 0.125              | <1           |
| Neu5Aca 2–3Galβ1–3GlcNAcβ-Bn                            | 2                  | <1           |
| Neu5Aca 2–3Galβ1–4GlcNAcβ-Bn                            | 0.75               | <1           |
| Neu5Aca 2–6Galβ1–3GlcNAcβ-Bn                            | 2                  | <1           |
| Neu5Aca 2-(4-methyl)umbelliferyl                        | 2                  | <5           |

WbwB assays were carried as described in the **Materials and methods** section, using different oligosaccharide derivatives as acceptor substrate and purified WbwB. Product was isolated using the C<sub>18</sub>Sep-Pak method

group (benzyl, umbelliferyl or phenyl-undecyl) that can bind to C18-Sep-Pak in spite of the charge of sialic acid or phosphates [10]. The only effective substrate was Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU. The apparent KM value for this acceptor was 25 μM with an apparent Vmax value of 6170 nmol/h/mg.

The presence of sialic acid in the acceptor substrate was essential for WbwB activity. Galβ1–3GalNAcα-PP-PhU and GalNAcα-PP-PhU did not serve as acceptor substrates, even at a 10 to 20-fold higher concentration. However, the Neu5Acα2–3Galβ1–3GalNAcα-sequence alone was not sufficient in the acceptor, since Neu5Acα2–3Galβ1–3GalNAcα-Bn at 12.5 to 25 times higher concentration than Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU did not show any activity. The activities with other α2–3- or α2–6-linked sialyl-oligosaccharides, up to 200 times the concentration of the WbwA product (Table 2), were less than 5% of the activity of the positive control. Using these sialyl-containing compounds in the assays, mass spectrometry analyses of the assay mixtures in the ESI-negative ion mode showed the presence of acceptors but did not reveal Gal-transferase products. This suggests that WbwB is specific not only for the terminal sialic acid but also for the diphosphate in the acceptor.

The SiaD constructs containing the Gal-transferase domain were expressed and bacterial homogenates tested for the specificity towards 0.1 mM WbwA product as acceptor in standard WbwB Gal-transferase assays. In contrast to WbwB, the Gal-transferase activities of SiaD constructs were not able to add Gal to Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU, having <1% of the activity

of WbwB. This indicates that the two α1,4-Gal-transferases (SiaD and WbwB) have distinct acceptor specificities. No sialyltransferase activity of full length SiaD towards the WbwB product Galα1–4Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU was detected, suggesting that the SiaD sialyltransferase is specific for its acceptor substrate [17, 18].

### Analysis of WbwB reaction product

The WbwB reaction product was analyzed by ESI-MS in the negative ion mode, which revealed a peak at m/z 1241.4 for [M-H]⁻ and peaks at m/z 1263.4 for [M-H + Na]⁻, and m/z 1285.4 for [M-H + 2Na]⁻ (Fig. 3). This showed that one Gal residue was added to Neu5Acα2–3Galβ1–3GalNAc-PP-PhU.

Galactosidase digestions of radioactive WbwB reaction product were carried out in order to determine the anomeric linkage of the newly added Gal residue. The positive control substrates, Galβ-p-nitrophenyl for β-galactosidase and Gala-p-nitrophenyl for α-galactosidase yielded high activities with the respective enzymes. However, WbwB product was not a suitable substrate for either β- or α-galactosidases, possibly because α-galactosidases do not efficiently cleave the Gala1-4-linked residue when it is linked to sialic acid.

Previously, we showed by dot blots that anti-T antibodies bound Galβ1–3GalNAcα-PP-PhU [9]. Here, we used a similar procedure with anti-ECO104 antibodies in a dot blot to test for reactivity with the newly synthesized non-radioactive WbwB reaction product. Spots of GalNAc-PP-PhU, WbwA product Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU, WbwB product Gal-Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU and

**Fig. 3** Analysis of WbwB reaction product by electrospray ionization (ESI) mass spectrometry in the negative ion mode. The mass peaks confirmed the structure Gal-Neu5Ac-GalNAc-PO₃-PO₃-(CH₂)₁₁-O-phenyl. The peak at m/z 1241 is for [M-H]⁻ and peaks at m/z 1263 for [M-H + Na]⁻, and at m/z 1285 for [M-H + 2Na]⁻.
ECO 104 positive control showed intense staining before C₁₈-Sep-Pak separation. After Sep-Pak extraction, only the positive control showed staining. Thus WbwB product was not recognized by the anti-ECO 104 antibody. This could mean that the O104 single repeating unit is too small or the structure inappropriate as an antigen, or that the 7-, 9-O-acetyl groups of sialic acid were required for binding.

In order to determine the structure of the WbwB reaction product and the sugar linkage of the newly added Gal residue, about 1 μmole of WbwB product was dissolved in D₂O and subjected to 1D and 2D NMR spectral analysis. The Proton chemical shifts of WbwB product were obtained from the correlation spectroscopy (COSY) and total correlated spectroscopy (TOCSY), and from the comparison to the spectra of WbwC product Galβ1–3GalNAcα-PP-PhU and WbwA product Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU (Table 3)

[9, 10]. Heteronuclear single quantum coherence spectroscopy (HSQC) was used to determine ¹³C shifts but no significant differences were found between WbwA and WbwB products. The H-1 of the newly added Gal¹ residue of WbwB product (Fig. 4) showed a chemical shift of 5.01 ppm with a coupling constant of 3.1 Hz, indicative of Gal¹ in α-linkage. The proton shifts of the Galβ1–3GalNAcα residues, and alkyl and phenyl residues in the WbwB product did not change significantly compared to the substrate [10]. However, the chemical shift of the equatorial proton H-3e of sialic acid changed significantly from 2.67 ppm in the substrate to 2.81 ppm in the reaction product, while the H-4 shift changed from 3.58 to 3.64 ppm and H-5 from 3.75 to 3.96 ppm. This indicated that Gal¹ was linked to sialic acid. Nuclear Overhauser Effect spectroscopy (NOESY) showed cross peaks between H-1 of the newly added Gal¹ (5.01 ppm) and the equatorial H-3e of Neu5Ac (2.81 ppm) (Fig. 4). Other NOEs were seen between Gal¹ H-1 and Gal¹ H-2 (3.70 ppm), between equatorial Neu5Ac H-3e (2.81 ppm) and axial Neu5Ac H-3a (1.66 ppm), and between GalNAc H-1 (5.43 ppm) and H-2 (4.31 ppm). The H-1 proton of Gal¹ is therefore close in space to the equatorial H-3 proton of Neu5Ac indicating the Gal¹–4Neu5Ac linkage. This confirms the structure of the WbwB reaction product as Galα1–4Neu5Acα2–3Galβ1–3GalNAcα-PP-PhU.

### Table 3 600 MHz NMR, chemical shifts (ppm) and coupling constants J (Hz) for the WbwB reaction product

| Residue | Chemical shift in ppm (Coupling constants in Hz) of WbwB, WbwA, WbwC reaction products |
|---------|--------------------------------------------------------------------------------------------|
|         | WbwB                              | WbwA                              | WbwC                              |
| **GalNAcα** |                                   |                                   |                                   |
| H-1     | 5.43                              | 5.45 (3.3, 6.7)                   | 5.45 (3.7, 7.2)                   |
| H-2     | 4.31                              | 4.31                              | 4.32                              |
| H-3     | 3.99                              | 3.98                              | 4.00                              |
| H-4eq   | 4.18                              | 4.18                              | 4.19                              |
| **Gal²β1–3** |                                   |                                   |                                   |
| H-1     | 4.47 (6.7)                        | 4.47 (7.8)                        | 4.41 (8.3)                        |
| H-2     | 3.44                              | 3.44                              | 3.45                              |
| H-3     | 4.00                              | 3.98                              | 3.55                              |
| H-4     | 3.87                              | 3.86                              | 3.82                              |
| **Neu5Acα2–3** |                               |                                   |                                   |
| H-3ax   | 1.66                              | 1.70 (12.5)                       | –                                 |
| H-3eq   | 2.81 (12.5, 3.0)                  | 2.67 (4.5, 12.5)                  | –                                 |
| H-4     | 3.64                              | 3.58                              | –                                 |
| H-5     | 3.96                              | 3.75                              | –                                 |
| **Gal¹α1–4** |                               |                                   |                                   |
| H-1     | 5.01 (3.1)                        | –                                 | –                                 |
| H-2     | 3.70                              | –                                 | –                                 |
| H-3     | 3.62                              | –                                 | –                                 |
| H-4     | 3.88                              | –                                 | –                                 |
| P-O-CH₂ | 3.85                              | 3.85                              | 3.85                              |

The structure of the WbwB reaction product (WbwB) was determined by COSY, TOCSY, HSQC, ROESY and NOESY as Galα1–4Neu5Acα2–3Galβ1–3GalNAcα-PO₃-PO₃-(CH₂)₁₁-O-phenyl. The chemical shifts of the acceptor substrate (WbwA) Neu5Acα2–3Galβ1–3GalNAcα-PO₃-PO₃-(CH₂)₁₁-O-phenyl [10] and the precursor substrate Galβ1–3GalNAcα-PO₃-PO₃-(CH₂)₁₁-O-phenyl (WbwC) are also shown [9]. Major proton chemical shifts are seen at H-3 eq, H-4 and H-5 of Neu5Ac in the WbwB product

#### Analysis of mutant WbwB proteins

The DXEX₇E sequence of WbwB includes possibly a DxD-related motif and a conserved EX₇E motif. The structure modeling analysis also suggested that the fourth EX₇E motif as well as Asp 313, Glu 315 and Glu 323 residues are possibly near the substrate binding site. The RKR sequence could be involved in binding the phosphates of UDP-Gal or the acceptor substrate [24]. We therefore analyzed 6 mutants: R242A, K243A, R244A, D313A, E315A and E323A. All mutants were expressed as seen by SDS-PAGE and Western blots (data not shown). Both the total cell lysates and the purified enzymes eluted from the Ni-NTA column were assayed. Compared with the activity of wild type WbwB, the D313A mutant was fully active in the total cell lysate and in the purified form (Fig. 5). This rules out that D313-X-E315 represents an essential DXD motif variant. Mutants of the first and last Glu residues of the fourth, conserved EX₇E motif, E315A, and E323A, were inactive, although a low residual activity was seen in the E323A mutant. This suggests that both Glu residues are essential for activity and at least one of these residues may represent a catalytic base. The RKR sequence was also shown to be important. Mutation of the central Lys 243 residue to Ala eliminated Gal-transferase activity, suggesting that Lys 243 is an essential residue in WbwB. Mutation of the adjacent Arg 242 to Ala reduced the activity by 75 to 84%. The R244A mutant
Glycoconj J

Inka-WbwB-product.14.1.2rr — NOESY mix=.4

Fig. 4 Analysis of WbwB reaction product by 600 MHz NMR. WbwB product Galα1-4Neu5Acα2-3Galβ1-3GalNAcα-PO₃-PO₃-(CH₂)₁₁-O-Phenyl was synthesized as described in the Materials and methods section, dissolved in D₂O, and analyzed by 1D and 2D experiments. The structure is shown at the top. The NOESY spectrum at the bottom shows an NOE between H-1 of Gal¹ (5.01 ppm) and Neu5Ac H-3e (2.81 ppm) that indicates the addition of Gal¹ to the 4-OH of Neu5Ac in α1-4 linkage by WbwB. Other NOEs were seen between Gal¹ H-1 (5.01 ppm) and Gal¹ H-2 (3.70 ppm), between Neu5Ac H-3e (2.81 ppm) and Neu5Ac H-3a (1.66 ppm), and between GalNAc H-1 (5.43 ppm) and H-2 (4.31 ppm)

was active in the total cell lysate but not after purification which may indicate that the protein was unstable. These Arg residues, therefore, may contribute to the activity by maintenance of the protein structure.

utilize a nucleotide sugar donor substrate, and also accommodate the negatively charged sialic acid of the acceptor substrate. In addition, WbwB must also bind the diphosphate group of the acceptor.

The lipid moiety in the natural acceptor in bacteria is undecaprenol that anchors the intermediate in the membrane *in vivo*. Phenyl-undecyl and other related lipids [25–30] serve as an appropriate lipid moiety in *in vitro* assays of bacterial GTs, and the structure and length of the hydrophobic chain can be variable. Several GTs have been characterized involved in bacterial polysaccharide synthesis that add the second sugar residue to GlcNAc/GalNAc-diphosphate-lipid and all of these require the diphosphate in the acceptor substrate [20, 25, 28–34]. Other GTs that add the second and subsequent sugars are active with oligosaccharide substrates that do not contain phosphates [26, 27, 35–38]. It is therefore surprising that all of the GTs that extend the O104 repeating unit require the diphosphate in the substrate. WbwB has relatively high activity at a very low acceptor substrate concentrations. In bacteria, this high activity can ensure the completion of the repeating unit before translocation to the periplasm where the polymerase acts, presumably with high specificity for the completed unit.

The importance of the EX₇E motifs found in many retaining GTs has been studied [39–44] but structural analysis and mutations of these retaining GTs have not yet provided a consistent role for the first or the second Glu residue. The function of the Glu residues may vary in the individual proteins from structural roles to binding of the nucleotide sugar. In WbwB, clearly both Glu residues of the conserved, fourth EX₇E motif within the C-terminal GT domain are essential for the activity of the enzyme. At least one of the Glu residues may provide a catalytic base that could deprotonate the 4-hydroxyl of sialic acid, which then becomes a nucleophile resulting in cleavage of Gal from UDP-Gal. However, the mechanism of this retaining GT and the roles of the other four EX₇E motifs remain to be determined.

This work establishes the enzymes and the sequence of sugar addition to form the repeating unit of ECO104. Work is in progress to characterize the putative *O*-acetyltransferase WckD that may be responsible for the conversion of Neu5Ac to the 7- and 9-*O*-acetylated form. This final reaction in the assembly pathway of the repeating unit may take place in the cytoplasm where acetyl-CoA is used as a donor substrate. This transfer of the acetyl groups may be required for the polymerization by a structure-specific Wzy polymerase in ECO104, and for the recognition by anti-ECO104 antibodies.

Using the bacterial enzymes in the ECO104 pathway we can provide the technology basis for the chemo-enzymatic synthesis of the O104 antigen that may be developed into a vaccine. The polymerization of the repeating unit by polymerase Wzy may be required to synthesize a vaccine candidate. It is hoped that this knowledge provides options for antibacterial targets for this dangerous food pathogen.

Acknowledgements Mass spectrometry was carried out by J. Wang, Department of Chemistry, Queen’s University. The authors thank F. Sauriol, Department of Chemistry, Queen’s University for NMR analyses and J.S. Schutzbach and J. Allingham for helpful advice. We are grateful to R.Gerardy-Schahn for plasmids encoding SiaD constructs. This work was funded by the Canadian Institutes of Health Research and the Natural Sciences and Engineering Council of Canada.

Compliance with ethical standards

Conflict of interest The authors declare they have no conflict of interest.

Ethical approval This article does not contain any studies with human participants or animals performed by any of the authors.

References

1. Muniesa, M., Hammeri, J., Hertwig, S., Appel, B., Brussow, H.: Shiga toxin-producing *Escherichia coli* O104:H4: a new challenge for microbiology. Appl. Environ. Microbiol. **78**, 4065–4073 (2012)
2. Trachtman, H.: Escherichia Coli O104:H4 outbreak in Germany. New Engl. J. Med. **366**, 76 (2012)
3. Qin, J., Cui, Y., Zhao, X., Rohde, H., Liang, T., Wolters, M., Li, D., Belmar Campos, C., Christner, M., Song, Y., Yang, R.: Identification of the Shiga toxin-producing *Escherichia coli* O104:H4 strain responsible for a food poisoning outbreak in Germany by PCR. J. Clin. Microbiol. **49**, 3439–3440 (2011)
4. Bengoeachea, J.A., Najdenski, H., Skurnik, M.: Lipopolysaccharide O antigen status of *Yersinia enterocolitica* O:8 is essential for virulence and absence of O antigen effects the expression of the other *Yersinia* virulence factors. Mol. Microbiol. **52**, 451–469 (2004)
5. Gamian, A., Romanowska, E., Ulrich, J., Defaye, J.: The structure of the sialic acid-containing *Escherichia Coli* O-specific polysaccharide and its linkage to the core region in lipopolysaccharide. Carbohydr. Res. **236**, 195–208 (1992)
6. Brockhausen, I.: Crossroads between bacterial and mammalian glycosyltransferases. Front. Immunol. **5**, 1–21 (2014)
7. Wang, L., Briggs, C.E., Rothemund, D., Fratamico, P., Luchansky, J.B., Reeves, P.R.: Sequence of the *E. coli* O104 antigen gene cluster and identification of O104 specific genes. Gene. **270**, 231–236 (2001)
8. Rush, J.S., Alaimo, C., Robbiani, R., Wacker, M., Waechter, C.J.: A novel epimerase that converts GlcNAc-P-P-undecaprenol to GalNAc-P-P-undecaprenol in *Escherichia Coli* O157. J. Biol. Chem. **285**, 1671–1680 (2010)
9. Wang, S., Czuchry, D., Liu, B., Vinnikova, A.N., Gao, Y., Vlahakis, J.Z., Szarek, W.A., Wang, L., Feng, L., Brockhausen, I.: Characterization of two UDP-Gal:GalNAc-diphosphate-lipid 1,3-galactosyltransferases WbwC from *Escherichia Coli* serotypes O104 and O5. J. Bacteriol. **196**, 3122–3133 (2014)
10. Czuchry, D., Desormeaux, P., Stuart, M., Jarvis, D., Matta, K.L., Szarek, W.A., Brockhausen, I.: Identification and biochemical characterization of the novel alpha2,3-sialyltransferase WbwA from the pathogenic *Escherichia coli* serotype O104. J. Bacteriol. **197**, 3760–3768 (2015)
11. Whitfield, C., Trent, M.S.: Biosynthesis and export of bacterial lipopolysaccharides. Annu. Rev. Biochem. **83**, 99–128 (2014)
12. Lehrer, J., Vigeant, K.A., Tatar, L.D., Valvano, M.A.: Functional characterization and membrane topology of *Escherichia Coli* WecA, a sugar-phosphate transferase initiating the biosynthesis of Enterobacterial common antigen and O-antigen lipopolysaccharide. J. Bacteriol. **189**, 2618–2628 (2007)
13. Islam, S.T., Lam, J.S.: Synthesis of bacterial polysaccharides via the Wzx/Wzy-dependent pathway. Can. J. Microbiol. **60**, 697–716 (2014)
14. Islam, S.T., Huszczyński, S.M., Nugent, T., Gold, A.C., Lam, J.S.: Conserved-residue mutations in Wzy affect O-antigen polymerization and Wzz-mediated chain-length regulation in *Pseudomonas aeruginosa* PAO1. Sci. Report. **3**, 3441 (2013)
15. Han, W., Wu, B., Zhao, G., Woodward, R., Pettit, N., Cai, L., Thon, V., Wang, P.G.: Defining function of lipopolysaccharide O-antigen ligase WaaL using chemoenzymatically synthesized substrates. J. Biochem. **287**, 5357–5365 (2012)
16. Putker, F., Bos, M.P., Tommassen, J.: Transport of lipopolysaccharide to the gram-negative bacterial cell surface. FEMS Microbiol. Rev. **39**, 985–1002 (2015)
17. Romanow, A., Haselhorst, T., Stummeyer, K., Claus, H., Bethe, A., Mühlenhoff, M., Vogel, U., von Itzstein, M., Gerardy-Schahn, R.: Biochemical and biophysical characterization of the Sialyl-/Hexosyltransferase synthesizing the meningococcal serogroup W135 Heteropolysaccharide capsule. J. Biol. Chem. **288**, 11718–11730 (2013)
18. Romanow, A., Keys, T.G., Stummeyer, K., Freiberer, F., Henrissat, B., Gerardy-Schahn, R.: Dissection of hexosyl-and sialyltransferase domains in the bifunctional capsule polymerase from *Neisseria meningitidis* W and Y defines a new sialyltransferase family. J. Biol. Chem. **49**, 33945–33957 (2014)
19. Ruane, K.M., Davies, G.J., Matinez-Fleites, C.: Crystal structure of a family GT4 glycosyltransferase from *Bacillus anthracis* ORF BA 1558. Proteins. **73**, 784–787 (2008)
20. Glover, K.J., Weerapana, E., Imperiali, B.: In vitro assembly of the undecaprenylpyrophosphate-linked heptasaccharide for prokaryotic N-linked glycosylation. Proc. Natl. Acad. Sci. **102**, 14255–11425 (2005)
21. Kelley, L.A., Sternberg, M.J.E.: Protein structure prediction on the web: a case study using the Phyre server. Nat. Protoc. **4**, 363–371 (2009)
22. Momma, M., Fujimoto, Z.: Interdomain disulfide bridge in the Rice granule bound starch synthase I catalytic domain as elucidated by X-ray structure analysis. Biosci. Biotechnol. Biochem. **76**, 1591–1595 (2012)
23. Chua, T.K., Bujnicki, J.M., Tan, T.-C., Huynh, F., Patel, B.K., Sivaraman, J.: The structure of sucrose phosphate synthase from *Halothermothrix orenii* reveals its mechanism of action and binding mode. Plant Cell. **20**, 1059–1072 (2008)
24. Vetting, M.W., Frantom, P.A., Blanchard, J.S.: Structural and enzymatic analysis of MshA from *Corynebacterium glutamicum* substrate-assisted catalysis. J. Biochem. **283**, 15834–15844 (2008)
25. Xu, C., Liu, B., Hu, B., Han, Y., Feng, L., Allingham, J., Szarek, W.A., Wang, L., Brockhausen, I.: Biochemical characterization of UDP-Gal:GlcNAc-pyrophosphate-lipid beta1,4-galactosyltransferase WfeD, a new enzyme from *Shigella boydii* type 14 that catalyzes the second step in O-antigen repeating-unit synthesis. J. Bacteriol. **193**, 449–459 (2011)
26. Yi, W., Yao, Q., Zhang, Y., Motari, E., Lin, S., Wang, P.G.: The wbnH gene of *Escherichia Coli* O86:H2 encodes an alpha-1,3-N-acetylgalactosaminyl transferase involved in the O-repeating unit biosynthesis. Biochem. Biophys. Res. Commun. **344**, 631–639 (2006)
27. Yi, W., Shao, J., Zhu, L., Li, M., Singh, M., Lu, Y., Lin, S., Li, H., Ryu, K., Shen, J., Guo, H., Yao, Q., Bush, C.A., Wang, P.G.: *Escherichia Coli* O86 O-antigen biosynthetic gene cluster and step-wise enzymatic synthesis of human blood group B antigen Tetrasaccharide. J. Am. Chem. Soc. **127**, 2040–2041 (2005)
28. Riley, J., Xu, C., Brockhausen, I.: Synthesis of acceptor substrate analogs for the study of glycosyltransferases involved in the second step of the biosynthesis of O antigen repeating units. Carbohydr. Res. **345**, 586–597 (2010)

29. Montoya-Peleaz, P., Riley, J.G., Szarek, W.A., Valvano, M.A., Schutzbach, J.S., Brockhausen, I.: Identification of a UDP-Gal: GlcNAc-R galactosyltransferase activity in Escherichia Coli VW187. Bioorg. Med. Chem. Lett. 15, 1205–1211 (2005)
30. Riley, J.G., Menggad, M., Montoya-Peleaz, P., Szarek, W.A., Marolda, C.L., Valvano, M.A., Schutzbach, J.S., Brockhausen, J.: The wbbD gene of E. coli strain VW187 (O7:K1) encodes a UDP-Gal: GlcNAc(alpha)-pyrophosphate-R (beta) 1,3-galactosyltransferase involved in the biosynthesis of O7-specific lipopolysaccharide. Glycobiology. 15, 605–613 (2005)
31. Brockhausen, I., Hu, B., Liu, B., Lau, K., Szarek, W.A., Wang, L., Feng, L.: Characterization of two UDP-GlcNAc:beta1,3-glucosyltransferases from the *Escherichia coli* serotypes O56 and O152. J. Bacteriol. 190, 4922–4932 (2008)
32. Wang, S., Hao, Y., Lam, J.S., Vlahakis, J.Z., Szarek, W.A., Vinnikova, A., Veselovsky, V.V., Brockhausen, I.: Biosynthesis of the Common Polysaccharide Antigen of *Pseudomonas aeruginosa* PAO1: Characterization and role of GDP-D-rhamnose: GlcNAc/GalNAc-diphosphate-lipid alpha1,3-D-rhamnosyltransferase WbpZ. J. Bacteriol. 197, 2012–2019 (2015)
33. Gao, Y., Liu, B., Strum, S., Schutzbach, J.S., Druzhinina, T.N., Utkina, N.S., Torgov, V.I., Danilov, L.L., Veselovsky, V.V., Vlahakis, J.Z., Szarek, W.A., Wang, L., Brockhausen, I.: Biochemical characterization of WbdN, a beta1,3-glucosyltransferase involved in O-antigen synthesis in enterohemorrhagic *Escherichia Coli* O157. Glycobiology. 22, 1092–1102 (2012)
34. Chen, C., Liu, B., Xu, Y., Utkina, N., Zhou, D., Danilov, L., Torgov, V., Veselovsky, V., Feng, L.: Biochemical characterization of the novel α-1,3-galactosyltransferase WclR from *Escherichia Coli* O3. Carbohydr. Res. 430, 36–43 (2016)
35. Li, M., Liu, X.-W., Shao, J., Shen, J., Jia, Q., Yi, W., Song, J.K., Woodward, R., Chow, C.S., Wang, P.G.: Characterization of a novel R1,2-Fucosyltransferase of *Escherichia Coli* O128:B12 and functional investigation of its common motif. Biochemistry. 47, 378–387 (2008)
36. Liu, X.-W., Xia, C., Lei Li, L., Guan, W.-Y., Pettit, N., Zhang, H.-C., Chen, M., Wang, P.G.: Characterization and synthetic application of a novel beta 1,3-galactosyltransferase from *Escherichia Coli* O55:H7. Bioorg. Med. Chem. 17, 4910–4915 (2009)
37. Li, M., Shen, J., Liu, X., Shao, J., Yi, W., Chow, C.S., Wang, P.G.: Identification of a new R1,2-Fucosyltransferase involved in O-antigen biosynthesis of *Escherichia Coli* O86:B7 and formation of H-type 3 blood group antigen. Biochemistry. 47, 11590–11597 (2008)
38. Yi, W., Perali, R.S., Eguchi, H., Motari, E., Woodward, R., Wang, P.G.: Characterization of a bacterial beta-1,3-galactosyltransferase with application in the synthesis of tumor-associated T-antigen mimics. Biochemistry. 47, 1241–1248 (2008)
39. Cid, E., Gomis, R.R., Geremia, R.A., Guinovart, J.J., Ferrer, J.C.: Identification of two essential glutamic acid residues in glycogen synthase. J. Biol. Chem. 275, 33614–33621 (2000)
40. Kostova, Z., Yan, B.C., Vainauskas, S., Schwartz, R., Menon, A.K., Orlean, P.: Comparative importance in vivo of conserved glutamate residues in the EX7E motif retaining glycosyltransferase Gpi3p, the UDP-GlcNAc-binding subunit of the first enzyme in glycosylphosphatidylinositol assembly. Eur. J. Biochem. 270, 4507–4514 (2003)
41. Absmanner, B., Schmeiser, V., Kaempf, M., Lehle, L.: Biochemical characterization, membrane association and identification of amino acids essential for the function of Alg11 from *Saccharomyces cerevisiae*, an α1,2-mannosyltransferase catalysing two sequential glycosylation steps in the formation of the lipid-linked core oligosaccharide. Biochem. J. 426, 205–217 (2010)
42. Sobhanifar, S., Worrall, L.J., Gruninger, R.J., Wasney, G.A., Blaukopf, M., Baumann, L., Lameignere, E., Solomonson, M., Brown, E.D., Withers, S.G., Strynadka, N.C.J.: Structure and mechanism of *Staphylococcus aureus* TarM, the wall teichoic acid α-glycosyltransferase. PNAS. 112, E576–E585 (2015)
43. Greenfield, L.K., Richards, M.R., Vinogradov, E., Wakarchuk, W.W., Lowary, T.L., Whitfield, C.: Domain Organization of the Polymerizing Mannosyltransferases Involved in synthesis of the *Escherichia Coli* O8 and O9a lipopolysaccharide O-antigens. J. Biol. Chem. 287, 38135–38149 (2012)
44. Ghafoor, A., Jordens, Z., Rehma, B.H.A.: Role of PelF in Pel polysaccharide biosynthesis in *pseudomonas aeruginosa*. Appl. Environ. Microbiol. 79, 2968–2978 (2013)
