
# Structural studies of Chikungunya virus maturation

Moh Lan Yap${}^{a,b}$, Thomas Klose${}^{a}$, Akane Urakami${}^{c}$, S. Saif Hasan${}^{a}$, Wataru Akahata${}^{c}$, and Michael G. Rossmann${}^{a,1}$

${}^{a}$Department of Biological Sciences, Purdue University, West Lafayette, IN 47907; ${}^{b}$Department of Biological Science, Faculty of Science, Universiti Tunku Abdul Rahman, 31900 Kampar, Perak, Malaysia; and ${}^{c}$VLP Therapeutics, Gaithersburg, MD 20878

Edited by Robert M. Stroud, University of California, San Francisco, California, and approved November 10, 2017 (received for review July 25, 2017)

Cleavage of the alphavirus precursor glycoprotein p62 into the E2 and E3 glycoproteins before assembly with the nucleocapsid is the key to producing fusion-competent mature spikes on alphaviruses. Here we present a cryo-EM, 6.8-Å resolution structure of an “immature” Chikungunya virus in which the cleavage site has been mutated to inhibit proteolysis. The spikes in the immature virus have a larger radius and are less compact than in the mature virus. Furthermore, domains B on the E2 glycoproteins have less freedom of movement in the immature virus, keeping the fusion loops protected under domain B. In addition, the nucleocapsid of the immature virus is more compact than in the mature virus, protecting a conserved ribosome-binding site in the capsid protein from exposure. These differences suggest that the posttranslational processing of the spikes and nucleocapsid is necessary to produce infectious virus.

alphavirus | Chikungunya virus | maturation | cryo-electron microscopy | conformational changes

Chikungunya virus (CHIKV) is a mosquito-borne virus, which was first reported in Tanzania in 1952 (1) and later emerged as an epidemic in the French Reunion Island in 2005 (2). In the past decade, CHIKV has spread to more than 40 countries across Africa, Asia, and Europe, causing over a million infections in the Americas alone since 2014 (3). Among the symptoms of the disease are rash, myalgia, high fever, and, typically, severe arthritis (4).

CHIKV is a member of the alphavirus genus in the Togaviridae family (5). Other closely related and well-studied alphaviruses are Semliki Forest virus (SFV), Ross River virus (RRV), Sindbis virus (SINV), and Venezuelan Equine Encephalitis virus (VEEV). Alphaviruses are spherical enveloped viruses with an ~700-Å diameter and a $T = 4$ quasi-icosahedral symmetry. The genome of alphaviruses is an ~12-kb positive-sensed single-stranded RNA molecule encoding four nonstructural proteins (nsP1–4), which are required for virus replication, and five structural proteins (capsid protein C, glycoproteins E1, E2, E3, and 6K) (6). The structural proteins are synthesized as a long polyprotein, which is then posttranslationally cleaved into C, E1, 6K, and p62. A total of 240 copies of the C protein associate with a newly synthesized genomic RNA molecule to form a nucleocapsid in the host cell’s cytoplasm (7). The glycoproteins E1 and p62 interact to form heterodimers that subsequently trimerize into a viral spike in the endoplasmic reticulum (ER). The glycoprotein p62 is then cleaved into E2 and E3 by cellular furin during its transportation from the acidic environment of the Golgi and early endosomes to the neutral pH environment of the cell surface, releasing E3 (Movie S1). Virus budding occurs at the cell membrane where the nucleocapsid is enveloped by the glycoproteins E1–E2 on the plasma lipid membrane. The protein 6K facilitates particle morphogenesis (8–10), but its position in the particle remains to be verified.

Alpha- and flaviviruses (11) have many similarities. Their glycoprotein exteriors have icosahedral symmetry and surround a lipid membrane that, in turn, surrounds their RNA genome, which is associated with the capsid protein. A major difference between alpha- (12) and flaviviruses (13) is the maturation process. Flaviviruses are assembled as “immature” noninfectious particles in the ER of the host cell that are then proteolytically modified to produce infectious viruses on leaving the host cell. However, alphavirus components are proteolytically modified before assembly into mature viruses on the plasma membrane. In addition, a regular, icosahedral capsid shell is observed only in alphaviruses. During infection, a conserved sequence on the N-terminal regions of the capsid proteins binds to the host cell’s 60S ribosomal subunits, initiating the dissociation of the nucleocapsid and the release of the RNA from the nucleocapsid (14). This ribosome-binding site (RBS) is buried during nucleocapsid assembly but is exposed at the end of the maturation process (15, 16).

In alphaviruses, there are 20 trimeric spikes located on the icosahedral threefold axes and another 60 trimeric spikes in general positions that obey $T = 4$ quasi-symmetry (17–19). Glycoprotein E1 is involved in cell fusion (20), and glycoprotein E2 interacts with host receptors (21) whereas glycoprotein E3 facilitates E1-p62 heterodimerization and prevents the exposure of the E1 fusion loops from premature fusogenic activation (22, 23). Cryo-EM studies have shown that E3 remains associated with the mature virus of SFV (24), RRV (18), and VEEV (25). However, SINV (26, 27) and CHIKV (28) release E3 after budding.

### Significance

Chikungunya virus (CHIKV) belongs to the alphavirus family, the members of which have enveloped icosahedral capsids. The maturation process of alphaviruses involves proteolysis of some of the structural proteins before assembling with nucleocapsids to produce mature virions. We mutated the proteolytic cleavage site on E2 envelope protein, which is necessary in initiating the maturation process. Noninfectious virus-like particles (VLP) equivalent to “immature” fusion incompetent particles were produced to study the immature conformation of CHIKV. We describe the 6.8-Å resolution electron microscopy structure of “immature” CHIK VLPs. Structural differences between the mature and immature VLPs show that posttranslational processing of the envelope proteins and nucleocapsid is necessary to allow exposure of the fusion loop on glycoprotein E1 to produce an infectious virus.

Author contributions: M.L.Y. and M.G.R. designed research; M.L.Y., T.K., A.U., and W.A. performed research; M.L.Y. and S.S.H. analyzed data; and M.L.Y. and M.G.R. wrote the paper.

Conflict of interest statement: M.L.Y., T.K., S.S.H., and M.G.R. declare no competing financial interests. A.U. is an employee of VLP Therapeutics, and W.A. is an officer and shareholder of VLP Therapeutics.

This article is a PNAS Direct Submission.

Published under the PNAS license.

Data deposition: The final immature Chikungunya VLP electron density map was deposited in the Electron Microscopy Data Bank, [https://www.emdatabank.org](https://www.emdatabank.org) (accession code EMD-8734), and structure coordinates have been deposited in the Protein Data Bank, [www.rcsb.org/pdb](http://www.rcsb.org/pdb) (PDB ID code 5VU2).

${}^{1}$To whom correspondence should be addressed. Email: mr@purdue.edu.

This article contains supporting information online at [www.pnas.org/lookup/suppl/doi:10.1073/pnas.1713166114/-/DCSupplemental](http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1713166114/-/DCSupplemental).
Here, we report the structure of immature CHIKV, which was determined using virus-like particles (VLPs) with mutations at the furin cleavage site on p62. The E3 remained associated with the E2, mimicking the precursor p62 in its immature conformation. A crystal structure of the E1-p62 heterodimer [Protein Data Bank (PDB) ID code 3N40 (29)] was fitted into the cryo-EM electron density map of immature CHIKV VLPs to examine the interactions of E1 and p62 with each other in the immature virus. A previous report showed that alphaviruses can be assembled in a partially mature, replication-competent state (25). Hence, the structure described here represents an intermediate structure of CHIKV during the assembly and maturation process. We showed that there are significant conformational differences between the mature and immature viruses, including the nucleocapsid, the transmembrane helices, and the cellular attachment sites on E2. The presence of E3 in the immature virus stabilized domain B on E2, protecting the fusion peptide on E1 from becoming exposed and fusogenic.

**Results and Discussion**

**Cryo-EM Structure of Immature CHIKV.** The cryo-EM density map of immature CHIK VLPs attained a 6.8-Å resolution (Fig. 1A). The virions had a diameter of 660 Å and, like mature virions, have $T = 4$ icosahedral symmetry. Central cross-sections of the reconstruction showed that the immature virion (Fig. 1C) has a nucleocapsid, enveloped by a plasma membrane and an outermost layer of glycoproteins. Unlike flaviviruses, alphaviruses, including CHIKV, have a well-ordered icosahedral nucleocapsid within the membrane envelope (Fig. 1B).

Immature CHIKV virions, like mature CHIKV virions, have spike-like features (Fig. 2A) on their surface. Intraspike contacts are formed between the three E2 molecules that form a spike. The glycoprotein E1 wraps around E2 and contributes to interspike interactions. Furthermore, E3 is located at the periphery of the E2 molecules (Fig. 2A). The trimeric immature spikes, although organized with $T = 4$ quasi-symmetry, are similar to mature CHIKV spikes, but are less compact with a hole along their threefold axes, resulting in a bigger spike radius. The spikes are more densely packed on the surface of immature CHIKV, resulting in smaller holes along the icosahedral twofold (i2) and icosahedral fivefold (i5) symmetry axes and smaller separation between the spikes, in comparison with mature CHIKV (Fig. 2B). Thus, the spikes undergo a structural rearrangement during maturation.

**Glycoprotein Spikes.** As described in the crystal structure of E1-p62 (29), E1 has three beta-sheet-rich domains, namely domains I, II, and III. A fusion loop is located at the tip of domain II. E2 consists of three Ig-like domains (A, B, and C) and a long beta-ribbon (domain D) connecting domain B to C. Domain D interacts extensively with E3. The E1 fusion loop is sandwiched between domains A and B of E2.

The crystal structure of E1-p62 (PDB ID code 3N40) (29) was fitted into the cryo-EM electron density map of immature

![Figure 1](#fig1)

**Fig. 1.** Cryo-EM reconstruction of immature CHIK virus-like particle. (A) Three-dimensional cryo-EM map of immature CHIKV, viewed down an icosahedral twofold axis. An icosahedral asymmetric unit is marked by a black triangle. Icosahedral symmetry elements are shown as black-filled pentagon, triangles, and ellipse. Four unique subunits in an asymmetric unit are shown in white numbers. (B) Internal capsid protein shell of the immature CHIKV. (C) Central cross-sections of the immature CHIKV viewed down an icosahedral twofold axis. Components of the virus are shown in different colors as indicated in the figure. (D) Enlarged view of the region outlined by the black rectangle in C. Fitting of an E1-p62-C structure is shown in the cryo-EM map of immature CHIKV.
part of the capsid protein starts at about residue 109 (VEEV) (25) and residue 97 (SINV) (19), presumably due to interactions with the genomic RNA. This ordered part has a helical structure in VEEV. In the virus, the positively charged region of the capsid proteins is associated with the negatively charged RNA. The external glycoproteins and the internal capsid protein shell are associated with the E1 and E2 cytoplasmic tail binding to the capsid protein (34). This interaction has been shown to be important in virus budding and fusion (35).

The model of the E1 and E2 transmembrane (TM) domains and capsid protein as observed in mature CHIKV (28) was fitted into the cryo-EM map of immature CHIKV (Fig. 1D). There are a number of differences between the structure of immature and mature CHIKV. These include a small change in orientation and location of the TM helices (Fig. 3A and Fig. S1), resulting in a more compact nucleocapsid in the immature CHIKV (Fig. 2B). The additionally ordered amino terminal region of the capsid protein in the virus is a part of the RBS (Fig. 3B). This fragment consists of the residues 98–112 (KPGRRERMCMKIEND) of the capsid protein, which are the conserved RBS in alphaviruses (14).

The life cycle of alphaviruses can be described as starting with the mature virus in which the B domain of E2 is only loosely associated with the underlying domain II of the E1 glycoprotein. Thus, after virus recognition of a cell, the virus is enclosed into the low pH environment of an endosome. This causes the formation of trimeric fusogenic spikes resulting in the fusion of the viral and cellular membranes. As a result, the nucleocapsid is exposed to the cell’s cytoplasm containing ribosomes at low pH. The RBS on the icosahedral capsid of the mature virus is then available to bind to ribosomes. The association of ribosomes with the nucleocapsid causes the viral capsid to disintegrate (14) while guiding the genome to a ribosome. The replicated genome then associates with newly synthesized capsid proteins to make new nucleocapsids. These are transported to the plasma membrane where they will associate with mature E1–E2 to form new mature particles that bud out from the membrane. In some cases, the E3 glycoprotein, although cleaved from E2, will remain on the particle.

In the present case, the cleavage site has been mutated resulting in no cleavage of p62 but producing the assembly of immature particles. The diameter of the nucleocapsids in these immature particles is about 20 Å smaller than in the mature particles, making the particles more compact and less likely to expose the RBS. When these immature particles bind to a cell surface the (E1p62)₃, trimeric spikes cannot fuse with the cell membrane because the E3 glycoprotein stops the exposure of the fusion loop. Observations of the immature particles indicate that furin cleavage of glycoprotein spikes followed by their association with the preformed nucleocapsid is required to produce fusion- and replication-competent particles.

Table 1. Average density height of the densities at the atomic positions (sumf) on fitting of the atomic structure of CHIKV heterodimer E1-p62 into the immature cryo-EM density map

| Protein | Protein domain | $T=4$ fitting | Independent molecule fitting |
|---------|----------------|---------------|-------------------------------|
| E1      | I              | 14.2          | 15.3                         |
|         | II             | 14.2          | 16.0                         |
|         | III            | 15.6          | 16.0                         |
| E2      | A              | 11.1          | 13.2                         |
|         | B              | 10.0          | 11.9                         |
|         | C              | 16.3          | 18.8                         |
|         | D              | 13.9          | 14.1                         |
| E3      |                | 12.9          | 13.1                         |
| Average |                | 13.5          | 14.8                         |

Yap et al.

Fig. 3. Comparisons of the E1-E2-C structure in the immature and mature CHIKV. (A) Superposition of the E1-p62-C structure in the immature conformation (magenta) to E1-E2-C structure in the mature conformation (cyan). The E3 molecule in p62 is in blue. The capsid protein has a similar orientation in both immature and mature conformations. However, the TM helices have a different orientation and location in the immature form compared with the mature form. (B) Density of the capsid protein. The ordered structure of the capsid protein consists of residues 113–261. Additional density seen only in the cryo-EM reconstruction of the virus (outlined with black dashes) belongs to the N-terminal region of the capsid protein. This region is the RBS (residues 98–112). The hydrophobic pocket of the capsid protein is indicated by a red star.

Conclusion. During the maturation process of alphaviruses, cleavage of p62 into E2 and E3 exposes the fusion loop on E1 and arranges the glycoprotein spikes into a mature conformation. Association of the mature spikes with the preassembled nucleocapsid expands the nucleocapsid to a less compact form. This step exposes the RBS on the capsid protein, thus priming the nucleocapsid to be disassembled upon release into the host-cell cytoplasm during the next infection cycle. The events described here for VLPs would correspond to the release of the genome into a host cell after virus entry and may be similar to the mechanism of genome release in many other viruses.

Materials and Methods

Production and Purification of Immature CHIK VLP. The coding sequence for the CHIKV strain 37997 structural proteins, C-E3-E2-6K-E1, was synthesized by the Blue Heron Company and cloned into a pUC119-derived vector under the control of a human cytomegalovirus early immediate promoter. The expression plasmid for the furin cleavage-resistant CHIKV VLP was generated by mutating amino acids 61–64 in E3 to Ser-Gly-Gly-Gly-Ser, using the QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent Technologies). The VLPs were produced in FreeStyle 293-F cells (Thermo Fisher Scientific). The cells were transfected with VLP-expressing plasmids by polyethylenimine reagent (Polysciences). Four days after transfection, the cell culture supernatant was harvested and clarified by centrifugation and filtration through a 0.45-μm polyethersulfone (PES) membrane. The VLPs secreted in the culture supernatant were collected by using OptiPrep Density Gradient Medium (Sigma-Aldrich), as described previously (36), and further purified by Hiprep 16/60 Sephacryl S-500 HR column (GE Healthcare Life Sciences). The eluates containing purified VLPs were concentrated by Amicon Ultra-15 centrifugal filter units (EMD Millipore) and filtered with a 0.20-μm PES membrane.

Electron Microscopy and 3D Reconstruction. Aliquots of a 2.5-μL sample at 3 mg/mL concentration were loaded on glow-discharged C-Flat grids (CF-2/2–4C). These grids were blotted for 5 s and flash-frozen in liquid ethane using a Gatan CP3 plunge freezer. The grids were viewed using the FEI Titan Krios electron microscope operated at 300 kV. Images were recorded with a Gatan K2 Summit detector calibrated to have a magnification of 38,461, yielding a pixel size of 0.65 Å. A total dose of 36 e⁻/Å² and an exposure time of 7.6 s were used to collect 38 movie frames. Fully automated data collection was implemented using Leginon (37). The MotionCorr software (38) was used to correct the beam-induced motion. A total of 5,325 images were collected, and 76,806 particles were boxed using the EMAN2 package (39). Contrast transfer function parameters were estimated using CTFFIND3 (40). The 2D classification was performed using RELION (41), and the 3D reconstruction was performed using the JSPR software (42). The final electron density map was reconstructed using 72,944 particles and was estimated to have a resolution of 6.8 Å based on the gold-standard Fourier shell correlation (FSC) criterion of 0.143 (43) (Fig. 4). The map was deposited in the Electron Microscopy Data Bank ([www.emdatabank.org](http://www.emdatabank.org)) with Electron Microscopy Data Bank accession code EMD-8734, and structure coordinates have been deposited with the PDB (PDB ID code 5VU2).

Data Analysis and Figure Preparation. The crystal structure of E1-p62 (PDB ID code 3N40) and models of the E1 and E2 TM domains and capsid protein (PDB ID code 3J2W) were fit into the cryo-EM map using the EMfit program (30) to maximize the average density height (sumf value) at all atomic positions. The model was fit as a rigid body using the $T = 4$ quasi-symmetry and also was fit independently as a rigid body into four unique positions in an asymmetric unit. All figures were prepared using Chimera (44).

ACKNOWLEDGMENTS. We thank Sheryl Kelly and Yingyuan Sun for help in the preparation of this manuscript and technical support, respectively, and the Purdue Cryo-EM Facility for instrument access and technical support. The work was funded by NIH Grant AI095366 (to M.G.R.). Part of this work was supported by VLP Therapeutics.
1. Robinson MC (1955) An epidemic of virus disease in Southern Province, Tanganyika Territory, in 1952-1953. *Trans R Soc Trop Med Hyg* 49:28–32.
2. Schuffenecker I, et al. (2006) Genome microevolution of chikungunya viruses causing the Indian Ocean outbreak. *PLoS Med* 3:e263.
3. Centers for Disease Control and Prevention (2017) NOWCAST: Chikungunya in the Americas. Available at https://www.cdc.gov/chikungunya/modeling/index.html. Accessed April 11, 2017.
4. Ryman KD, Klimstra WB (2008) Host responses to alphavirus infection. *Immunol Rev* 225:27–45.
5. Kuhn RJ (2007) *Togaviridae: The viruses and their replication*. Fields Virology, eds Knipe DM, Howley PM (Lippincott Williams & Wilkins, Philadelphia), 5th Ed, pp 1001–1022.
6. Strauss JH, Strauss EG (1994) The alphaviruses: Gene expression, replication, and evolution. *Microbiol Rev* 58:491–562.
7. Melancon P, Garoff H (1987) Processing of the Semliki Forest virus structural polyprotein: Role of the capsid protease. *J Virol* 61:1301–1309.
8. Gaedigk-Nitschko K, Schlesinger MJ (1990) The Sindbis virus 6K protein can be detected in virions and is acylated with fatty acids. *Virology* 175:274–281.
9. Gaedigk-Nitschko K, Schlesinger MJ (1991) Site-directed mutations in Sindbis virus E2 glycoprotein’s cytoplasmic domain and the 6K protein lead to similar defects in virus assembly and budding. *Virology* 183:206–214.
10. Sanz MA, Carrasco L (2001) Sindbis virus variant with a deletion in the 6K gene shows defects in glycoprotein processing and trafficking: Lack of complementation by a wild-type 6K gene in trans. *J Virol* 75:7778–7784.
11. Schmaljohn AL, McClain D (1996) Alphaviruses (*Togaviridae*) and Flaviviruses (*Flaviviridae*). Medical Microbiology, ed Baron S (University of Texas Medical Branch at Galveston, Galveston, TX), 4th Ed.
12. Jose J, Snyder JE, Kuhn RJ (2009) A structural and functional perspective of alphavirus replication and assembly. *Future Microbiol* 4:837–856.
13. Mukhopadhyay S, Kuhn RJ, Rossmann MG (2005) A structural perspective of the flavivirus life cycle. *Nat Rev Microbiol* 3:13–22.
14. Wengler G, Würkner D, Wengler G (1992) Identification of a sequence element in the alphavirus core protein which mediates interaction of cores with ribosomes and the disassembly of cores. *Virology* 191:880–888.
15. Wengler G, Wengler G, Boege U, Wahn K (1984) Establishment and analysis of a system which allows assembly and disassembly of alphavirus core-like particles under physiological conditions in vitro. *Virology* 132:401–412.
16. Wengler G (1987) The mode of assembly of alphavirus cores implies a mechanism for the disassembly of the cores in the early stages of infection. Brief review. *Arch Virol* 94:1–14.
17. Vogel RH, Provencher SW, von Bonsdorff CH, Adrian M, Dubochet J (1986) Envelope structure of Semliki Forest virus reconstructed from cryo-electron micrographs. *Nature* 320:533–535.
18. Cheng RH, et al. (1995) Nucleocapsid and glycoprotein organization in an enveloped virus. *Cell* 80:621–630.
19. Mukhopadhyay S, et al. (2006) Mapping the structure and function of the E1 and E2 glycoproteins in alphaviruses. *Structure* 14:63–73.
20. Lescar J, et al. (2001) The fusion glycoprotein shell of Semliki Forest virus: An icosahedral assembly primed for fusogenic activation at endosomal pH. *Cell* 105:137–148.
21. Smith TJ, et al. (1995) Putative receptor binding sites on alphaviruses as visualized by cryoelectron microscopy. *Proc Natl Acad Sci USA* 92:10648–10652.
22. Mulvey M, Brown DT (1995) Involvement of the molecular chaperone BiP in maturation of Sindbis virus envelope glycoproteins. *J Virol* 69:1621–1627.
23. Carleton M, Lee H, Mulvey M, Brown DT (1997) Role of glycoprotein PE2 in formation and maturation of the Sindbis virus spike. *J Virol* 71:1558–1566.
24. Mancini EJ, Clarke M, Gowen BE, Rutten T, Fuller SD (2000) Cryo-electron microscopy reveals the functional organization of an enveloped virus, Semliki Forest virus. *Mol Cell* 5:255–266.
25. Zhang R, et al. (2011) 4.4 Å cryo-EM structure of an enveloped alphavirus Venezuelan equine encephalitis virus. *EMBO J* 30:3854–3863.
26. Paredes AM, et al. (1993) Three-dimensional structure of a membrane-containing virus. *Proc Natl Acad Sci USA* 90:9095–9099.
27. Paredes AM, et al. (1998) Structural localization of the E3 glycoprotein in attenuated Sindbis virus mutants. *J Virol* 72:1534–1541.
28. Sun S, et al. (2013) Structural analyses at pseudo atomic resolution of Chikungunya virus and antibodies show mechanisms of neutralization. *Elife* 2:e00435.
29. Voss JE, et al. (2010) Glycoprotein organization of Chikungunya virus particles revealed by X-ray crystallography. *Nature* 468:709–712.
30. Rossmann MG, Bernal R, Pletnev SV (2001) Combining electron microscopic with x-ray crystallographic structures. *J Struct Biol* 136:190–200.
31. Lee S, et al. (1996) Identification of a protein binding site on the surface of the alphavirus nucleocapsid and its implication in virus assembly. *Structure* 4:531–541.
32. Choi HK, et al. (1991) Structure of Sindbis virus core protein reveals a chymotrypsin-like serine proteinase and the organization of the virion. *Nature* 354:37–43.
33. Choi HK, Lu G, Lee S, Wengler G, Rossmann MG (1997) Structure of Semliki Forest virus core protein. *Proteins* 27:345–359.
34. Owen KE, Kuhn RJ (1997) Alphavirus budding is dependent on the interaction between the nucleocapsid and hydrophobic amino acids on the cytoplasmic domain of the E2 envelope glycoprotein. *Virology* 230:187–196.
35. Tang J, et al. (2011) Molecular links between the E2 envelope glycoprotein and nucleocapsid core in Sindbis virus. *J Mol Biol* 414:442–459.
36. Akahata W, et al. (2010) A virus-like particle vaccine for epidemic Chikungunya virus protects nonhuman primates against infection. *Nat Med* 16:334–338.
37. Suloway C, et al. (2005) Automated molecular microscopy: The new Legion system. *J Struct Biol* 151:41–60.
38. Li X, et al. (2013) Electron counting and beam-induced motion correction enable near-atomic-resolution single-particle cryo-EM. *Nat Methods* 10:584–590.
39. Tang G, et al. (2007) EMAN2: An extensible image processing suite for electron microscopy. *J Struct Biol* 157:38–46.
40. Mindell JA, Grigorieff N (2003) Accurate determination of local defocus and specimen tilt in electron microscopy. *J Struct Biol* 142:334–347.
41. Scheres SH (2012) A Bayesian view on cryo-EM structure determination. *J Mol Biol* 415:406–418.
42. Guo F, Jiang W (2014) Single particle cryo-electron microscopy and 3-D reconstruction of viruses. *Methods Mol Biol* 1117:401–443.
43. Scheres SH, Chen S (2012) Prevention of overfitting in cryo-EM structure determination. *Nat Methods* 9:853–854.
44. Pettersen EF, et al. (2004) UCSF Chimera: A visualization system for exploratory research and analysis. *J Comput Chem* 25:1605–1612.