
Cell Host & Microbe

Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in Drosophila Respectively Control Local and Systemic Immune Responses to Microbiota

### Graphical Abstract

![Graphical Abstract Image](image_url)

### Authors

Bernard Charroux, Florence Capo,  
C. Léopold Kurz, Sabine Peslier,  
Delphine Chaduli,  
Annelise Viallat-lieutaud, Julien Royet

#### Correspondence

bernard.charroux@univ-amu.fr (B.C.),  
julien.royet@univ-amu.fr (J.R.)

### In Brief

Since microbiota-derived peptidoglycan activates NF-κB signaling in the gut and remote organs, its levels need to be controlled *in vivo*. Charroux et al. show how, by generating both secreted and cytosolic peptidoglycan-cleaving enzymes, *Drosophila* uncouple the modulation of local versus systemic responses to a single gut-bacteria-derived product.

### Highlights

- The PGRL-LB locus encodes cytosolic and secreted peptidoglycan (PGN)-cleaving enzymes
- Cytosolic isoforms regulate PGN-dependent NF-κB signaling inside enterocytes
- The secreted isoform cleaves PGN in the gut, preventing its diffusion to remote tissues
- Microbiota-derived PGN has detrimental effects on remote tissues and organs

Charroux et al., 2018, Cell Host & Microbe 23, 1–14  
February 14, 2018 © 2017 Elsevier Inc.  
https://doi.org/10.1016/j.chom.2017.12.007

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in Drosophila Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

**Cell Host & Microbe**

**Article**

# Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota

Bernard Charroux,$^{1,*}$ Florence Capo,$^{1}$ C. Léopold Kurz,$^{1}$ Sabine Peslier,$^{1}$ Delphine Chaduli,$^{1}$ Annelise Viallat-lieutaud,$^{1,2}$ and Julien Royet$^{1,2,*}$

$^{1}$Aix Marseille Université, CNRS, IBDM, Marseille, France  
$^{2}$Lead Contact  

*Correspondence:* bernard.charroux@univ-amu.fr (B.C.), julien.royet@univ-amu.fr (J.R.)  
https://doi.org/10.1016/j.chom.2017.12.007

## SUMMARY

Gut-associated bacteria produce metabolites that both have a local influence on the intestinal tract and act at a distance on remote organs. In *Drosophila*, bacteria-derived peptidoglycan (PGN) displays such a dual role. PGN triggers local antimicrobial peptide production by enterocytes; it also activates systemic immune responses in fat-body cells and modulates fly behavior by acting on neurons. How these responses to a single microbiota-derived compound are simultaneously coordinated is not understood. We show here that the PGRP-LB locus generates both cytosolic and secreted PGN-cleaving enzymes. Through genetic analysis, we demonstrate that the cytosolic PGRP-LB isoforms cell-autonomously control the intensity of NF-κB activation in enterocytes, whereas the secreted isoform prevents massive and detrimental gut-derived PGN dissemination throughout the organism. This study explains how *Drosophila* are able to uncouple the modulation of local versus systemic responses to a single gut-bacteria-derived product by using isoform-specific enzymes.

## INTRODUCTION

The collection of microorganisms that live within eukaryotes and are called microbiota provides crucial signals for the development and homeostasis of the intestinal tract ([Honda and Littman, 2016](https://doi.org/10.1038/nrmicro3850); [Rooks and Garrett, 2016](https://doi.org/10.1038/nrmicro3849); [Sommer and Backhed, 2013](https://doi.org/10.1038/nrmicro2974)). The metabolites and compounds produced by these bacteria can also influence the formation and the functionality of tissues and organs that are not in direct contact with the bacterial community, such as the bone marrow or the brain ([Grigg and Sonnenberg, 2017](https://doi.org/10.1038/nrmicro.2017.10); [Sharon et al., 2016](https://doi.org/10.1038/nature17625)). How these distinct functions are orchestrated at the molecular level is not well understood. *Drosophila* has recently emerged as a powerful model to study the interactions between gut-associated bacteria and the host ([Buchon et al., 2013a](https://doi.org/10.1016/j.cell.2013.01.044); [Erkosaar et al., 2013](https://doi.org/10.1016/j.cub.2013.01.044); [Ma et al., 2015](https://doi.org/10.1016/j.cell.2015.01.044)). Two bacteria-derived molecules, uracil and peptidoglycan (PGN), act as activators of host signaling cascades in the *Drosophila* gut ([Bosco-Drayon et al., 2012](https://doi.org/10.1016/j.cell.2012.01.044); [Lee et al., 2013](https://doi.org/10.1016/j.cell.2013.01.044), [2015](https://doi.org/10.1016/j.cell.2015.01.044); [Neyen et al., 2012](https://doi.org/10.1016/j.cell.2012.01.044)). Uracil produced by pathogens and pathobionts, but not by symbionts, activates the DUOX-dependent NADPH oxidases that regulate the production of reactive oxygen species in enterocytes, and hence shapes the bacterial communities ([Lee et al., 2013](https://doi.org/10.1016/j.cell.2013.01.044)). In contrast, PGN is released from most gut bacteria regardless of whether they are symbionts, pathogens, or pathobionts. Recognition of PGN by enterocytes is mediated by either the cytosolic PGN recognition receptor-LE (PGRP-LE) or the membrane-associated PGRP-LC receptors ([Bosco-Drayon et al., 2012](https://doi.org/10.1016/j.cell.2012.01.044)). PGN binding to these receptors triggers IMD/NF-κB pathway activation and antimicrobial peptide (AMP) production by enterocytes ([Bosco-Drayon et al., 2012](https://doi.org/10.1016/j.cell.2012.01.044); [Neyen et al., 2012](https://doi.org/10.1016/j.cell.2012.01.044)). Since normal conventional gut environments where symbionts are dominant create a PGN-rich condition that could potentially induce constitutive IMD-dependent AMP production, different negative regulators suppress such activation ([Lee and Ferrandon, 2011](https://doi.org/10.1016/j.cell.2011.01.044)). These regulators include the secreted PGN-degrading enzyme PGRP-LB, which by cleaving PGN, limits IMD pathway activation ([Paredes et al., 2011](https://doi.org/10.1016/j.cell.2011.01.044); [Zaidman-Rémy et al., 2006](https://doi.org/10.1016/j.cell.2006.01.044)). Mutant flies with a defect in one of these negative regulators show classical features of intestinal dysbiosis such as constitutive IMD pathway activation (high AMP production), modification of the gut commensal community, gut cell apoptosis, and early host death ([Guo et al., 2014](https://doi.org/10.1016/j.cell.2014.01.044); [Paredes et al., 2011](https://doi.org/10.1016/j.cell.2011.01.044); [Ryu et al., 2008](https://doi.org/10.1016/j.cell.2008.01.044); [Zaidman-Rémy et al., 2006](https://doi.org/10.1016/j.cell.2006.01.044)). In addition to activating the local immune response in enterocytes, gut-bacteria-derived PGN can translocate in the hemolymph (insect blood) that bathes most larval and adult organs and tissues ([Basset et al., 2000](https://doi.org/10.1016/j.cell.2000.01.044); [Vodovar et al., 2005](https://doi.org/10.1016/j.cell.2005.01.044)). By interacting with the PGRP-LC receptor, PGN present in the hemolymph can activate the NF-κB pathway in fat-body cells ([Choe et al., 2005](https://doi.org/10.1016/j.cell.2005.01.044); [Gottar et al., 2002](https://doi.org/10.1016/j.cell.2002.01.044); [Neyen et al., 2016](https://doi.org/10.1016/j.cell.2016.01.044); [Ramet et al., 2002](https://doi.org/10.1016/j.cell.2002.01.044)). It is, however, likely that circulating PGN has an impact on other tissues and organs. Consistently, we have recently shown that circulating PGN can also modulate fly behavior by interfering with the activity of some octopaminergic neurons that control ovipositioning ([Kurz et al., 2017](https://doi.org/10.1016/j.cell.2017.01.044)).

In the present study, we show that in addition to the previously characterized secreted PGRP-LB isoform, the PGRP-LB locus codes for cytosolic isoforms. Through genetic experiments, we demonstrate that these non-secreted PGN-cleaving enzymes present in enterocytes regulate the intensity of NF-κB signaling

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

in these cells in a cell-autonomous manner. Our data further demonstrate that the main function of secreted PGRP-LB is to prevent the PGN released by gut-associated bacteria from reaching the hemolymph where it would activate the systemic immune response and have detrimental consequences such as organ wasting and fat-body metabolism dysfunction. We finally show that intracytoplasmic PGN-dependent NF-κB signaling in cells that are neither enterocytes nor fat body is the triggering event of these non-autonomous fat-body metabolic disorders. In conclusion, this study highlights how, by generating different enzymes from a single locus, *Drosophila* is able to simultaneously but independently control the local and the systemic consequences of the constant production of PGN, a single and universal metabolite released by gut-associated bacteria.

**RESULTS**

**The PGRP-LB Locus Encodes Extracellular and Cytosolic PGN-Degrading Enzymes**

Gene model prediction based on expressed sequence tag sequencing suggests the existence of three PGRP-LB transcripts generated by alternative splicing ([http://flybase.org/reports/FBgn0037906.html](http://flybase.org/reports/FBgn0037906.html)) (*Figure S1A*). The PGRP-LB<sup>PA</sup>, PGRP-LB<sup>PC</sup>, and PGRP-LB<sup>PD</sup> mRNAs are transcribed from distinct promoters and encode the PGRP-LB<sup>PA</sup>, PGRP-LB<sup>PC</sup>, and PGRP-LB<sup>PD</sup> isoforms, respectively (*Figures S1B* and *S1C*). Previous work has shown that PGRP-LB<sup>PC</sup> is a secreted amidase that transforms the PGN released by extracellular bacteria into muropeptides unable to activate the NF-κB/IMD pathway (*Zaidman-Rémy et al.*, 2006). The role of the PGRP-LB<sup>PA</sup> and PGRP-LB<sup>PD</sup> isoforms remains largely uncharacterized (*Kurz et al.*, 2017). The PGRP-LB<sup>PA</sup> protein contains the amidase domain only, while PGRP-LB<sup>PD</sup> possesses an additional N-terminal sequence with no obvious similarity to other protein motifs (*Figure S1B*). Since predictive software did not reveal the presence of hydrophobic or subcellular localization domains, we hypothesized that these amidases might be cytosolic (*Figure S1C*). To precisely characterize their subcellular localization, we generated upstream activating sequence (UAS) transgenic lines in which the EGFP coding sequence was fused in-frame with each coding sequence of the PGRP-LB isoforms. When expressed in third-instar larvae salivary glands, the PGRP-LB::EGFP fusion proteins (which are all fully functional, see below) showed distinct subcellular distributions (*Figures 1A* and *1B*). The PGRP-LB<sup>PC</sup>::EGFP protein was detected in a cytoplasmic punctuated pattern and was absent from the plasma membrane. Double staining with a fluorescent fusion to the glue cargo protein Sgs3 (Sgs-3::DsRed) demonstrated that the PGRP-LB<sup>PC</sup>::EGFP protein localized inside the secretory granules (*Figure 1A*) (*Burgess et al.*, 2012). The PGRP-LB<sup>PA</sup>::EGFP and PGRP-LB<sup>PD</sup>::EGFP proteins were distributed in a pattern complementary to that of PGRP-LB<sup>PC</sup>::EGFP. They were detected in the cytosol (but excluded from the secretory granules), at the plasma membrane and in the nucleus (*Figure 1A*). Antibody staining performed without membrane permeabilization further demonstrated that PGRP-LB<sup>PA</sup>::EGFP and PGRP-LB<sup>PD</sup>::EGFP reside inside the cell (*Figure S1D*). Consistently, PGRP-LB<sup>PC</sup>::EGFP, but neither PGRP-LB<sup>PA</sup>::EGFP nor PGRP-LB<sup>PD</sup>::EGFP, was secreted into the lumen of salivary glands

(*Figure 1*B). These data demonstrate that the PGRP-LB locus can generate at least three isoforms, one extracellular (PGRP-LB<sup>PC</sup>) and two cytosolic (PGRP-LB<sup>PA</sup> and PGRP-LB<sup>PD</sup>).

**PGRP-LB Controls Both Gut Response to Bacteria and PGN Dissemination to Fat Body**

To explore the function of the different PGRP-LB isoforms, we characterized the consequences of PGRP-LB inactivation on the fly response to infection by the Gram-negative bacteria *Erwinia carotovora carotovora* (*Ecc*). The PGRP-LB<sup>4</sup> allele that removes the amidase domain shared by all isoforms and hence corresponds to a complete loss-of-function allele, was used for these experiments (*Figure S1A*) (*Paredes et al.*, 2011). Using the Dipterericin-Cherry (Dipt-Cherry) reporter as an in vivo readout for IMD pathway activation, we showed that septic injury induced similar IMD pathway activation levels in wild-type flies and in PGRP-LB<sup>4</sup> mutants (*Figure 2A*). In contrast, infection by oral feeding with the same bacteria species induced a much stronger IMD activation in PGRP-LB<sup>4</sup> adults than in controls (*Figure 2A*). qRT-PCR quantification on dissected tissues showed a respective 34- and 216-fold increase of *Dipt* mRNA levels in midguts and fat bodies of orally infected PGRP-LB<sup>4</sup> mutants compared with controls (*Figure 2B*). A much weaker phenotype due to the lack of PGRP-LB function was observed when flies were infected by septic injury (3.5- and 2-fold in midguts and fat bodies, respectively) (*Figure 2B*). Confocal imaging of dissected guts and fat bodies of per-os-infected flies confirmed these results and further illustrated the upregulation of the IMD signaling in PGRP-LB<sup>4</sup> mutant guts (*Figure 2C*) (*Buchon and Osman*, 2015). Whereas Dipt-Cherry was mainly activated in the R2 region of *Ecc*-fed control midguts, stronger activation in the R2 domain and ectopic activation in the R1 and R4 domains were observed in PGRP-LB<sup>4</sup> mutant guts (*Figures 2C* and *2D*). On the other hand, Dipt-Cherry activation was observed in PGRP-LB<sup>4</sup> mutant fat bodies but not in wild-type (*Figure 2C*). The different outcomes associated with PGRP-LB inactivation are dependent on the mode of infection used, highlighting the essential role that PGRP-LB plays in the gut (*Paredes et al.*, 2011). Our data speak for a dual role of PGRP-LB in regulating the local response to gut bacteria and in preventing dissemination of microbiota-derived PGN to remote tissues of the host. Oral *Ecc* infection was thereafter used as the mode of infection to dissect the precise role of the different isoforms in the control of PGN-dependent NF-κB activation in flies.

**PGRP-LB Is Expressed in Multiple Cell Types of the Midgut and the Abdomen**

In order to get insight into the tissues that express the different PGRP-LB proteins, we cloned respectively 3, 2.8, and 3.8 kb of genomic DNA upstream of the predicted transcription start site of each isoform upstream of the Gal4 coding sequence. We named the corresponding transgenic lines *pLB1<sup>Gal4</sup>*, *pLB2<sup>Gal4</sup>*, and *pLB3<sup>Gal4</sup>*, respectively (*Figure S1A*; *Kurz et al.*, 2017). When combined with UAS-mCD8::ChRFP, these Gal4 lines showed specific expression domains. In the abdomen, *pLB1<sup>Gal4</sup>* labeled pericardial cells, while *pLB2<sup>Gal4</sup>* was uniformly expressed in the fat body and *pLB3<sup>Gal4</sup>* drove the expression of UAS-mCD8::ChRFP in pericardial cells and in oenocytes

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

A

| PGRP-LB<sup>PA</sup>::EGFP | PGRP-LB<sup>PD</sup>::EGFP | PGRP-LB<sup>PC</sup>::EGFP |
|---------------------------|---------------------------|---------------------------|
| ![Image](image1)          | ![Image](image2)          | ![Image](image3)          |

| Sgs3::DsRed              | Sgs3::DsRed              | Sgs3::DsRed              |
|---------------------------|---------------------------|---------------------------|
| ![Image](image4)          | ![Image](image5)          | ![Image](image6)          |

| Merged                   | Merged                   | Merged                   |
|---------------------------|---------------------------|---------------------------|
| ![Image](image7)          | ![Image](image8)          | ![Image](image9)          |

50 μm

B

| PGRP-LB<sup>PA</sup>::EGFP | PGRP-LB<sup>PD</sup>::EGFP | PGRP-LB<sup>PC</sup>::EGFP |
|---------------------------|---------------------------|---------------------------|
| ![Image](image10)         | ![Image](image11)         | ![Image](image12)         |

lumen

Figure 1. The PGRP-LB<sup>PA</sup> and PGRP-LB<sup>PD</sup> Isoforms Are Cytosolic

(A) Salivary glands of Sgs3::DsRed; Da<sup>Gal4</sup>/UAS-PGRP-LB<sup>PA</sup>::EGFP, Sgs3::DsRed; Da<sup>Gal4</sup>/UAS-PGRP-LB<sup>PD</sup>::EGFP, and Sgs3::DsRed; Da<sup>Gal4</sup>/UAS-PGRP-LB<sup>PC</sup>::EGFP third-instar larvae. Insets correspond to higher magnification. PGRP-LB<sup>PC</sup>::EGFP, but neither PGRP-LB<sup>PA</sup>::EGFP nor PGRP-LB<sup>PD</sup>::EGFP, co-localizes with the secretory granule marker Sgs3::DsRed.

(B) Salivary glands of Da<sup>Gal4</sup>/UAS-PGRP-LB<sup>PA</sup>::EGFP, Da<sup>Gal4</sup>/UAS-PGRP-LB<sup>PD</sup>::EGFP, and Da<sup>Gal4</sup>/UAS-PGRP-LB<sup>PC</sup>::EGFP third-instar larvae. The GFP signal is found in the salivary gland lumen only for the PGRP-LB<sup>PC</sup>::EGFP isoform. See also Figure S1.

Cell Host & Microbe 23, 1–14, February 14, 2018 3

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

**Figure 2. PGRP-LB Downregulates IMD Pathway Activation after Oral Infection**

IMD pathway activation is monitored with either the *Diptericin* reporter (*Dipt-Cherry*) (A and C) or by measuring transcription of the IMD pathway target gene *Diptericin* using qRT-PCR (B). Control (*w<sup>1118</sup>; *Dipt-Cherry<sup>C1</sup>*) or *PGRP-LB<sup>A</sup>* (*w<sup>1118</sup>; *Dipt-Cherry<sup>C1</sup>, PGRP-LB<sup>A</sup>*) adult mutants are infected by pricking the thorax with an Ecc-contaminated needle (Injury) or by Ecc ingestion (Feeding).

(legend continued on next page)

4 Cell Host & Microbe 23, 1–14, February 14, 2018

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

(Figure S2A). In the anterior midgut, $pLB1^{Gal4}$ was expressed in both the enterocytes and the intestinal stem cells (ISCs); $pLB2^{Gal4}$ in the visceral muscles, in few enterocytes, and in ISCs; and $pLB3^{Gal4}$ exclusively in visceral muscles. These data indicate that the PGRP-LB locus is expressed in various cell types both in the midgut and in the abdomen and suggest that the different PGRP-LB isoforms might regulate the immune response in a cell- and/or tissue-specific manner.

**Extracellular and Cytosolic Isoforms Play Specific Roles in Regulating the Immune Response**

The above results demonstrate that PGRP-LB plays at least two roles. It buffers the gut response to microbiota and prevents the diffusion of bacteria-derived PGN to the whole body. To test if these functions are performed by specific isoforms, we performed rescue experiments of the $PGRP-LB^4$ null allele using Gal4 drivers with restricted and well-characterized expression patterns (Figure S2B) (Chatterjee et al., 2016). Only when expressed in enterocytes with the $Mex^{Gal4}$ driver did all $PGRP-LB$ isoforms rescue the gut $Dipt$ overexpression phenotype observed in $PGRP-LB^4$ mutants (Figures 3A, 3B, and S2C). Overexpression of $PGRP-LB^PA$ and $PGRP-LB^PD$ in visceral muscles ($Mef2^{Gal4}$) or in oenocytes ($oeno^{Gal4}$) did not rescue the gut immune response. In contrast, $PGRP-LB^PC$ isoform overexpression in visceral muscles partially rescued enterocyte $Dipt$ expression (Figure 5A). Using the same genetic combination, we tested the ability of the different isoforms to prevent fat-body immune activation by gut bacteria. Whereas $PGRP-LB^PC$ was able to rescue the fat-body phenotype when ectopically expressed in any of the tested tissues, the $PGRP-LB^PA$ and $PGRP-LB^PD$ isoforms were unable to do so (Figures 3A, 3B, and S2D). These data indicate that the extracellular isoform produced either within the gut lumen (via the enterocyte Gal4 driver) or by cells contacting the hemolymph (via the muscle, fat body, and oenocyte drivers) is able to degrade circulating PGN and hence to prevent its access to fat-body cells. They also demonstrate that while intracellular $PGRP-LB^PA$ and $PGRP-LB^PD$ are able to repress NF-$\kappa$B activation in enterocytes, they cannot prevent PGN-mediated activation of the IMD pathway in fat-body cells. This was further confirmed by expressing the different isoforms directly in fat-body cells using the $r4^{Gal4}$ driver. Only the $PGRP-LB^PC$ isoform was able to rescue the $PGRP-LB^4$ phenotype, although all $PGRP-LB::EGFP$-tagged proteins were detected in fat-body cells (Figure 3B).

**Cytosolic Isoforms Regulate Gut IMD Signaling Cell-Autonomously**

The rescue experiments suggested that cytosolic amidases are controlling enterocyte responses to PGN produced by gut-associated bacteria. Knowing that enterocyte NF-$\kappa$B activation is mainly dependent on intracellular detection of PGN via the PGRP-LE sensor, it is tempting to propose that $PGRP-LB^PA$ and $PGRP-LB^PD$ directly regulate PGN levels inside the enterocytes (Bosco-Drayon et al., 2012). To test this hypothesis, we generated clones of enterocytes mutant for the $PGRP-LB^4$ allele. In contrast to $PGRP-LB^4$ mutant guts, which show IMD target reporter $AttacinD-Cherry$ ($AttD-Cherry$) overexpression, the $PGRP-LB^4$ mutant clones observed in all gut regions (R1, R2, and R4) had $AttD-Cherry$ expression levels comparable to that of neighboring wild-type cells (Figure 4A). This could indicate that all isoforms act non-cell-autonomously and that mutant clones are therefore rescued by amidases secreted by neighboring cells. Alternatively, the $PGRP-LB^PC$ secreted amidase produced by non-mutant cells adjacent to the clones could be sufficient to degrade all lumen PGN and therefore mask a cell-autonomous function of $PGRP-LB^PA$ and $PGRP-LB^PD$. To distinguish between these hypotheses, we undertook a gain-of-function approach and tested whether clones of cells overexpressing the $PGRP-LB^PA$ or $PGRP-LB^PD$ isoforms could rescue $Dipt-Cherry$ overexpression seen in a $PGRP-LB^4$ mutant background in a cell-autonomous manner (Bosch et al., 2015). Ectopic expression of either $PGRP-LB^PA$ or $PGRP-LB^PD$ was indeed sufficient to rescue the $PGRP-LB^4$ mutant phenotype (Figures 4B and S3). The sharp border of the clones and the almost fully complementary pattern between the GFP signal marking the clone border and the Cherry labeling cells in which the IMD pathway was activated strongly suggested that the $PGRP-LB$ cytosolic isoforms can repress IMD pathway activation in a cell-autonomous manner in enterocytes. Cells in which $PGRP-LB^PA$ and $PGRP-LB^PD$ expression did not prevent IMD pathway activation (yellow in the merged channel) are likely to correspond to cells in which NF-$\kappa$B activation is PGRP-LC dependent and is therefore not expected to be sensitive to modulation of intracytoplasmic

proventriculus (see Figure 2D) rely on the transmembrane signaling PGRP-LC receptor to sense extracellular PGN (Figure 3C). If such extracellular PGN sensing mechanisms, and hence IMD downstream signaling, is expected to be tunable by the secreted amidase $PGRP-LB^PC$, we hypothesized that it should be unaffected by the activity of the cytosolic ones. Consistently, overexpression of $PGRP-LB^PC$, but not of $PGRP-LB^PA$ or $PGRP-LB^PD$, fully prevented IMD pathway activation in PGRP-LC-dependent proventriculus of Ecc-infected adult guts (Figure 3C). Altogether these data indicate that $PGRP-LB^PC$ has two roles: one local, to regulate IMD signaling in PGRP-LC-dependent enterocytes, and another to prevent PGN diffusion from the gut lumen to remote organs.

**The PGRP-LB^PC Isoform Regulates PGRP-LC-Dependent NF-$\kappa$B Activation in the Gut**

We have previously shown that, depending on their localization along the gut anteroposterior axis, enterocytes use specific sensors to detect PGN (Bosco-Drayon et al., 2012). While enterocytes in the posterior midgut detect PGN intracellularly via PGRP-LE, enterocytes in the anterior domains such as the

(A and B) $PGRP-LB$ inactivation has a much stronger effect on IMD pathway activation in guts and fat bodies of orally infected flies than in flies contaminated with a needle. (B) mRNA level in non-infected flies was set to 1, and values obtained with indicated genotypes were expressed as a fold of this value. Histograms correspond to the mean value ± SD of three experiments. *p < 0.05, **p < 0.01; Kruskal-Wallis test.

(C) Confocal images of adult gut and fat body showing the consequences of $PGRP-LB$ inactivation on $Dipt-Cherry$ expression after Ecc oral infection.

(D) Schematic representation of the *Drosophila* adult gut. The five midgut regions (R1 to R5) defined in (Buchon et al., 2013b) are indicated. c, crop; os, esophagus; pv, proventriculus; mt, Malpighian tubules.

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

A

|  |  |  |
| --- | --- | --- |
| **Midgut** | **Fat body** |  |
| **Mex<sup>Gal4</sup>** | **oeno<sup>Gal4</sup>** | **Mef2<sup>Gal4</sup>** |
| ![](https://i.imgur.com/yourimage1.png) | ![](https://i.imgur.com/yourimage2.png) | ![](https://i.imgur.com/yourimage3.png) |
|  |  |  |
| 1: Control | 3: PGRP-LB<sup>A</sup>+X<sup>Gal4</sup>>PGRP-LB<sup>PA</sup>::EGFP |  |
| 2: PGRP-LB<sup>A</sup> | 4: PGRP-LB<sup>A</sup>+X<sup>Gal4</sup>>PGRP-LB<sup>PC</sup>::EGFP |  |
|  | 5: PGRP-LB<sup>A</sup>+X<sup>Gal4</sup>>PGRP-LB<sup>PD</sup>::EGFP |  |

B

C

| Midgut | Fat body | Proventriculus |
| --- | --- | --- |
| ![](https://i.imgur.com/yourimage4.png) | ![](https://i.imgur.com/yourimage5.png) | ![](https://i.imgur.com/yourimage6.png) |

Figure 3. PGRP-LB Isoforms Play Specific Roles in Regulating Local and Systemic Immune Responses

Ecc infection triggered Dipt expression in adult midgut and fat body of PGRP-LB<sup>A</sup> mutant rescued by expressing the PGRP-LB isoforms in specific tissues. (A and B) Flies of the following genotypes are used: control (*w<sup>1118</sup>; ; Dipt-Cherry<sup>C1</sup>), PGRP-LB<sup>A</sup>(*w<sup>1118</sup>; ; Dipt-Cherry<sup>C1</sup>, PGRP-LB<sup>A</sup>), PGRP-LB<sup>A</sup> expressing UAS-PGRP- LB<sup>PA</sup>::EGFP or UAS-PGRP-LB<sup>PC</sup>::EGFP or UAS-PGRP-LB<sup>PD</sup>::EGFP under the control of the indicated Gal4 line.

(A) When expressed in enterocytes (Mex<sup>Gal4</sup>), all isoforms are able to rescue gut Dipt overexpression observed in PGRP-LB<sup>A</sup> mutant. None of the isoforms can fully rescue Dipt overexpression when expressed in either the oenocytes (oeno<sup>Gal4</sup>) or the muscles (Mef2<sup>Gal4</sup>). When expressed with Mex<sup>Gal4</sup>, oeno<sup>Gal4</sup>, or

(legend continued on next page)

6 Cell Host & Microbe 23, 1–14, February 14, 2018

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

PGN levels ([Bosco-Drayon et al., 2012](https://doi.org/10.1016/j.chom.2017.12.007); [Neyen et al., 2012](https://doi.org/10.1016/j.chom.2017.12.007)). The fact that cytosolic enzymes are able to cell-autonomously regulate IMD pathway activation suggests that, in such cells, PGN is sensed intracellularly and hence via PGRP-LE and not PGRP-LC. This is fully consistent with data showing that increased *Dipt* expression observed in *PGRP-LB<sup>4</sup>* mutant guts (in which the PGRP-LC-dependent proventriculus has been removed) is more efficiently suppressed by concomitant inactivation of PGRP-LE than by that of PGRP-LC and that the opposite is obtained for fat-body cells, in which IMD activation is mediated by the PGRP-LC membrane receptor ([Figures S4A](https://doi.org/10.1016/j.chom.2017.12.007) and S4B).

### The Amidase Activity Is Required for Both PGRP-LB Functions

Amidases such as PGRP-LB or PGRP-SC bind and cleave PGN at specific sites ([Mellroth and Steiner, 2006](https://doi.org/10.1016/j.chom.2017.12.007); [Zaidman-Rémy et al., 2011](https://doi.org/10.1016/j.chom.2017.12.007)). Our rescue experiments that are performed by overexpressing PGRP-LB isoforms could reflect titration of the PGN rather than its direct degradation. To circumvent this concern, we genetically engineered mutant versions of PGRP-LB isoforms by replacing the conserved zinc ligand cysteine by a serine, a substitution sufficient to kill the amidase activity of both the T7 lysozyme and the PGRP-SC1B proteins without affecting their binding properties to PGN ([Mellroth et al., 2003](https://doi.org/10.1016/j.chom.2017.12.007)). In contrast to the non-mutated form, amidase-dead isoforms were no longer able to rescue the excessive production of *Dipt* observed in *PGRP-LB<sup>4</sup>* mutant guts ([Figure S4C](https://doi.org/10.1016/j.chom.2017.12.007)). In addition, when expressed under the control of either *Mef2<sup>Gal4</sup>* or *oeno<sup>Gal4</sup>* drivers, the amidase-dead PGRP-LB<sup>C</sup>::EGFP protein did not rescue the excessive fat-body response of the *PGRP-LB<sup>4</sup>* mutants ([Figure S4D](https://doi.org/10.1016/j.chom.2017.12.007)). These data demonstrate that the amidase activity is required in all isoforms to regulate NF-κB pathway activation both in fat-body cells and in enterocytes. They further suggest that PGRP-LB<sup>A</sup> and PGRP-LB<sup>D</sup> act by reducing the level of available PGN to be detected by PGRP-LE inside the enterocytes.

amidase domain shared by all isoforms and does not possess a unique coding exon, it was not possible to specifically inactivate this isoform via targeted mutagenesis. However, we generated CRISPR-specific alleles for *PGRP-LB<sup>C</sup>* and *PGRP-LB<sup>D</sup>* ([Figures S5A–S5C](https://doi.org/10.1016/j.chom.2017.12.007) and S1C and [Kurz et al., 2017](https://doi.org/10.1016/j.chom.2017.12.007)). We tested the ability of *PGRP-LB<sup>D</sup>*- and *PGRP-LB<sup>C</sup>*-specific mutant alleles to activate the local and systemic immune responses following *per os Ecc* infection. As expected from previous rescue experiments, systemic IMD pathway activation was not increased in *PGRP-LB<sup>D</sup>* CRISPR mutant alleles ([Figure 5A](https://doi.org/10.1016/j.chom.2017.12.007)). Similarly, no effects were observed in the gut response, suggesting that PGRP-LB<sup>A</sup> and PGRP-LB<sup>D</sup> may function redundantly. In contrast, *PGRP-LB<sup>C</sup>*-specific mutants show upregulation of IMD target gene expression in the fat body following *Ecc* feeding ([Figures 5B–5D](https://doi.org/10.1016/j.chom.2017.12.007)). Such mutant flies also present a weak upregulation of NF-κB target gene activation in the midgut compared with *PGRP-LB<sup>4</sup>* mutants ([Figure 5B](https://doi.org/10.1016/j.chom.2017.12.007)). It is expected from the above results that this upregulation corresponds to IMD signaling activated in PGRP-LC-dependent enterocytes such as the one from the proventriculus ([Figures 3C](https://doi.org/10.1016/j.chom.2017.12.007) and S4A). Altogether, these results indicate that IMD pathway activation in the gut is controlled by two mechanisms. First, by the cytosolic PGRP-LB<sup>A</sup> and PGRP-LB<sup>D</sup> isoforms that regulate IMD signaling in PGRP-LE-dependent enterocytes and second, by PGRP-LB<sup>C</sup>, which buffers PGRP-LC-dependent activation. They also indicate that PGRP-LB<sup>C</sup> is playing an essential role in preventing microbiota-derived PGN from reaching the hemolymph and coming into contact with all the tissues that are bathed in it.

### Deleterious Effects of *Ecc* Infection Are Due to PGRP-LE Dependent NF-κB Pathway Activation

Chronic oral *Ecc* infection of *PGRP-LB<sup>4</sup>* mutant has been shown to reduce the *Drosophila* lifespan ([Paredes et al., 2011](https://doi.org/10.1016/j.chom.2017.12.007)). We confirmed here these data and showed that constant *Ecc* feeding is also associated with physiological disorders such as lipid droplet depletion, autophagy induction in fat body, and organ wasting ([Figures 6A](https://doi.org/10.1016/j.chom.2017.12.007), 6B, S6A, and S6B). Although these phenotypes secondary to a prolonged contact with bacteria have been attributed to the deleterious effects of constant IMD pathway activation, the tissue(s) and mechanisms involved have not been identified ([Bosco-Drayon et al., 2012](https://doi.org/10.1016/j.chom.2017.12.007); [Paredes et al., 2011](https://doi.org/10.1016/j.chom.2017.12.007)). These phenotypes could result from a dysfunction of the gut consecutive to a permanent NF-κB signaling in enterocytes (local effect) and/or from an excess of circulating PGN in

### Gut-Borne PGRP-LB<sup>C</sup> Prevents Microbiota-Derived PGN from Reaching the Hemolymph

The above results suggest that cytosolic and extracellular amidases play specific roles in regulating NF-κB-dependent responses to gut-associated bacteria. To precisely determine the respective contribution of each isoform via a loss-of-function approach, we generated isoform-specific mutants using the CRISPR-Cas9 technology. Since PGRP-LB<sup>A</sup> consists of the

*Mef2<sup>Gal4</sup>*, the PGRP-LB<sup>C</sup>::EGFP isoform, but not the cytosolic UAS-PGRP-LB<sup>A</sup>::EGFP or UAS-PGRP-LB<sup>D</sup>::EGFP proteins, can rescue the systemic overexpression of *Dipt* observed in *PGRP-LB<sup>4</sup>* mutant. The mRNA level in control infected flies was set to 1 and values obtained with indicated genotypes were expressed as a fold of this value. Histograms correspond to the mean value ± SD of three experiments. *p* < 0.05, **p** < 0.01; Kruskal-Wallis test; ns, not significant. (B) Confocal images of the midgut R1 region and the fat body of control (*w<sup>1118</sup>*; *Dipt-Cherry<sup>C1</sup>*), *PGRP-LB<sup>4</sup>* (*w<sup>1118</sup>*; *Dipt-Cherry<sup>C1</sup>*, *PGRP-LB<sup>4</sup>*), and *PGRP-LB<sup>4</sup>* flies overexpressing the PGRP-LB isoforms in enterocytes (*w<sup>1118</sup>*; *Mex<sup>Gal4</sup>/UAS-PGRP-LB<sup>A</sup>::EGFP* or *UAS-PGRP-LB<sup>C</sup>::EGFP* or *UAS-PGRP-LB<sup>D</sup>::EGFP*; *Dipt-Cherry<sup>C1</sup>*, *PGRP-LB<sup>4</sup>*) or in fat-body cells (*w<sup>1118</sup>*; +/*UAS-PGRP-LB<sup>A</sup>::EGFP* or *UAS-PGRP-LB<sup>C</sup>::EGFP* or *UAS-PGRP-LB<sup>D</sup>::EGFP*; *Dipt-Cherry<sup>C1</sup>*, *PGRP-LB<sup>4</sup>/r4<sup>Gal4</sup>*, *PGRP-LB<sup>4</sup>*) 24 hr after *Ecc* feeding. Overexpression of either PGRP-LB<sup>A</sup>::EGFP, PGRP-LB<sup>C</sup>::EGFP, or PGRP-LB<sup>D</sup>::EGFP in enterocytes rescues the exacerbated *Dipt-Cherry* activation of *PGRP-LB<sup>4</sup>* mutant gut. When expressed by fat-body cells (*r4<sup>Gal4</sup>*), PGRP-LB<sup>C</sup>::EGFP, but not PGRP-LB<sup>A</sup>::EGFP or PGRP-LB<sup>D</sup>::EGFP, suppresses *Dipt-Cherry* overactivation in the fat body. (C) *Diptericin* expression (visualized with the *Dipt-Cherry* reporter) in proventriculus of *Ecc* orally infected adults. *Dipt-Cherry* expression (labels with an asterisk, the more anterior staining being constitutive and infection independent) is fully PGRP-LC dependent. Indeed, *Dipt-Cherry* expression observed in wild-type and exacerbated in *PGRP-LB* mutant is abolished by inactivating PGRP-LC but unaffected by PGRP-LE removal. Ectopic expression of *Dipt-Cherry* observed in *PGRP-LB* mutant proventriculus is fully suppressed by the ubiquitous expression of PGRP-LB<sup>C</sup>::EGFP but not of PGRP-LB<sup>A</sup>::EGFP or PGRP-LB<sup>D</sup>::EGFP. See also [Figure S2](https://doi.org/10.1016/j.chom.2017.12.007).

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

the hemolymph that will directly or indirectly affect peripheral organs such as the fat body and the ovaries (systemic effect). To get insight into these issues, we tested which of the local or systemic NF-κB responses was deleterious to chronically infected *PGRP-LB*<sup>Δ</sup> flies. We took advantages of the specific action of the different PGRP-LB isoforms to downregulate NF-κB signaling locally in the gut and/or systemically in peripheral organs. We found that when expressed in enterocytes, the intracytoplasmic isoforms could not rescue the lifespan nor the fat-body defects of Ecc-infected *PGRP-LB*<sup>Δ</sup> mutant (Figures 6B and S6C), although they were able to downregulate the immune response in enterocytes (Figures 3A and 3B). On the other hand, these phenotypes were prevented by expressing PGRP-LB<sup>PC</sup> in fat-body cells and in enterocytes (Figures 6A, 6B, and S6C). Knowing that fat-body expression of PGRP-LB<sup>PC</sup> suppresses the excessive systemic fat-body immune response, but not the gut response (Figures 3B and 6A), these results suggest that constant activation of IMD signaling in peripheral organs, but not in the intestinal tract, is the primary cause of premature death of Ecc-fed *PGRP-LB*<sup>Δ</sup> flies. This hypothesis was further validated by experiments showing that ubiquitous expression via the Da<sup>Gal4</sup> driver of PGRP-LB<sup>PA</sup> and PGRP-LB<sup>PD</sup> strongly rescued lipid droplet depletion, organ wasting, and longevity but not the ectopic activation of Dipt-Cherry in fat-body cells (Figures 6A and 6B). These results demonstrate that excessive intracellular PGN-dependent signaling in tissues other than fat body is responsible for physiological disorders associated with chronic Ecc oral infection.

All together, these data indicate that the main function of the secreted PGRP-LB<sup>PC</sup> isoform is to prevent diffusion of gut-bacteria-derived PGN to remote tissues. If such control is impaired, the level of PGN in the hemolymph increases, leading to various physiological consequences. First, it induces, in a PGRP-LC-dependent manner, AMP production in fat-body cells. Second, it leads to excessive intracytoplasmic PGN-dependent signaling in cells that remain to be identified. Likely mediated by the intracytoplasmic PGRP-LE sensor, this exacerbated NF-κB signaling perturbs the insect physiology and fitness. Some of these consequences are organ wasting and non-autonomous modifications in fat-body metabolism. The role of the PGRP-LB<sup>PA</sup> and PGRP-LB<sup>PD</sup> intracytoplasmic amidases is to buffer the PGN level inside these cells to prevent them from overacting to fluctuation of PGN levels that takes place in the insect blood (Figure 7).

## DISCUSSION

Our data show that the PGRP-LB locus can produce cytosolic and extracellular proteins that both buffer the effects of gut-bacteria-derived PGN. The two intracytoplasmic enzymes act locally in a cell-autonomous manner to control the level of PGN-dependent NF-κB activation in enterocytes in response to gut bacteria. These data reinforce the model proposed, but not yet fully demonstrated, in which gut lumen PGN can enter the cytosol

LB<sup>Δ</sup> mutant gut orally infected with Ecc. Whereas gain-of-function clones (marked by EGFP and indicated by the white dashed line) of EGFP have no effect on Dipt-Cherry expression, clones overexpressing PGRP-LB<sup>PA</sup>::EGFP or PGRP-LB<sup>PD</sup>::EGFP repress Dipt-Cherry expression in a cell-autonomous manner in the midgut R1 region. See also Figure S3.

---

**Figure 4. The Cytosolic PGRP-LB Isoforms Repress IMD Pathway Activation Cell-Autonomously in Enterocytes**

Confocal images of the midgut R1, R2, and R4 regions of adult flies 24 hr after Ecc feeding.

(A) Flies of the following genotypes are used: control (*w*<sup>1118</sup>;; AttD-Cherry/+) or *PGRP-LB*<sup>Δ</sup> (*w*<sup>1118</sup>;; AttD-Cherry/+; *PGRP-LB*<sup>Δ</sup>) or *Marcm*<sup>PGRP-LBΔ</sup> (yw, hsflp, tubGal4, UAS-GFP/+; AttD-Cherry/+; FRT82B, *PGRP-LB*<sup>Δ</sup>/FRT82B, tub<sup>GaI80</sup>). Upper images: AttD-Cherry is overexpressed in the R1 region of *PGRP-LB*<sup>Δ</sup> gut orally infected with Ecc. Lower images: Clones mutant for *PGRP-LB*<sup>Δ</sup> (marked by the expression of GFP) in the R1, R2, and R4 regions do not overexpress AttD-Cherry compared with neighboring wild-type cells.

(B) Flies of the following genotypes are used: control (*w*<sup>1118</sup>;; Dipt-Cherry<sup>C1</sup>) or *PGRP-LB*<sup>Δ</sup> (*w*<sup>1118</sup>;; Dipt-Cherry<sup>C1</sup>, *PGRP-LB*<sup>Δ</sup>) or *PGRP-LB*<sup>Δ</sup> flies over-expressing EGFP, the PGRP-LB<sup>PA</sup>::EGFP or the PGRP-LB<sup>PD</sup>::EGFP isoforms in clones (*CoinFLPGal4*/+; hsflp/UAS-EGFP or UAS-PGRP-LB<sup>PA</sup>::EGFP or UAS-PGRP-LB<sup>PD</sup>::EGFP; Dipt-Cherry<sup>C1</sup>, *PGRP-LB*<sup>Δ</sup>/Dipt-Cherry<sup>C1</sup>, *PGRP-* LB<sup>Δ</sup>). Upper images: Dipt-Cherry is overexpressed in the R1 region of *PGRP-*

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in Drosophila Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

**A**

| Midgut | Fat body |
| --- | --- |
|  |  |
| ns | ns |
| ns | ns |
| **ns** | **ns** |
| **\*\*** | **\*\*** |
| **\*\*** | **\*\*** |
| **\*\*** | **\*\*** |
|  |  |
| 60 | 300 |
| 50 | 247.4 |
| 40 | 200 |
| 43.0 | 100 |
| 30 | 1.0 |
| 20 | 0.6 7.1 0.6 0.2 |
| 10 |  |
| 0 |  |
| 1 2 3 4 5 6 | 1 2 3 4 5 6 |

**B**

| Midgut | Fat body |
| --- | --- |
|  |  |
| \* | \*\* |
| \* | \*\* |
| **\*\*** | **\*\*** |
| **\*\*** | **\*\*** |
| **\*\*** | **\*\*** |
|  |  |
| 100 | 300 |
| 91.7 | 209 241 |
| 80 | 127 124 |
| 60 | 98 |
| 40 |  |
| 20 |  |
| 10 |  |
| 0 |  |
| 1 2 3 4 5 6 | 1 2 3 4 5 6 |

**C**

| Midgut | Fat body |
| --- | --- |
|  |  |
| ns | ns |
| ns | ns |
| **\*\*** | **\*\*** |
| **\*\*** | **\*\*** |
|  |  |
| 60 | 300 |
| 48. | 228. |
| 40 | 200 |
| 8 | 150 |
| 20 | 100 |
| 0 | 50 |
| 1 2 3 4 | 1 2 3 4 |
| + *Mex<sup>Gal4</sup>* > *PGRP-LB<sup>PC</sup>::EGFP* | + *r4<sup>Gal4</sup>* > *PGRP-LB<sup>PC</sup>::EGFP* |

**D**

| Midgut | Fat body | Midgut | Fat body |
| --- | --- | --- | --- |
|  |  |  |  |
| R1 | R1 |  |  |
| 100 µm | 20 µm | 100 µm | 20 µm |
| Dipt-Cherry | Dipt-Cherry | Dipt-Cherry | Dipt-Cherry |

**Figure 5. CRISPR-Specific Inactivation of PGRP-LB<sup>PC</sup> Promotes Excessive Systemic Immune Response to Gut Infection**

*Dipt* mRNA levels in dissected midguts and fat bodies of *PGRP-LB<sup>PD</sup>* and *PGRP-LB<sup>PC</sup>* CRISPR-specific mutants. Flies of the following genotypes are used: control (*w<sup>1118</sup>*) or *PGRP-LB<sup>A</sup>* (*w<sup>1118</sup>; *PGRP-LB<sup>A</sup>*) or *PGRP-LB<sup>PD*Z10e</sup>* (*w<sup>1118</sup>; *PGRP-LB<sup>PD*Z10e</sup>*) or *PGRP-LB<sup>PD*Z19b</sup>* (*w<sup>1118</sup>; *PGRP-LB<sup>PD*Z19b</sup>*) or *PGRP-LB<sup>A</sup>* (*w<sup>1118</sup>; *PGRP-LB<sup>A</sup>*) or *PGRP-LB<sup>PD*Z10e</sup>* (*w<sup>1118</sup>; *PGRP-LB<sup>PD*Z10e</sup>*) or *PGRP-LB<sup>PD*Z19b</sup>* (*w<sup>1118</sup>; *PGRP-LB<sup>PD*Z19b</sup>*) or *PGRP-LB<sup>A</sup>* (*w<sup>1118</sup>; *PGRP-LB<sup>A</sup>*) or *PGRP-LB<sup>PC*6A</sup>* (*w<sup>1118</sup>; *PGRP-LB<sup>PC*6A</sup>*) or *PGRP-LB<sup>PC*10A</sup>* (*w<sup>1118</sup>; *PGRP-LB<sup>PC*10A</sup>*)

(legend continued on next page)

of enterocytes to mediate the activation of PGRP-LE, which shares some characteristics with mammalian Nod2 (Bosco-Drayon et al., 2012; Kaneko et al., 2006; Kurata, 2010; Neyen et al., 2012; Philpott et al., 2014) (Figure 7). Our data suggest that by cleaving PGN within the enterocytes, PGRP-LB<sup>PA</sup> and PGRP-LB<sup>PD</sup> prevent constant NF-κB activation in these cells. The fact that PGRP-LB<sup>PD</sup> single mutants are not showing overexpression of AMP production in enterocytes suggests that PGRP-LB<sup>PA</sup> and PGRP-LB<sup>PD</sup> are at least partially redundant for this function. Alternatively, it could be that PGRP-LB<sup>PA</sup>, for which no mutant could be generated, is playing a preponderant role in the gut. The fact that ectopic expression of either PGRP-LB<sup>PA</sup> or PGRP-LB<sup>PD</sup> does not suppress IMD activation in some enterocytes and in the proventriculus is consistent with our previous data demonstrating heterogeneity in the mode of PGN sensing in enterocytes (Bosco-Drayon et al., 2012). While in most enterocytes, PGN is sensed intracytoplasmically via PGRP-LE, some gut cells detect PGN at the membrane via PGRP-LC. In the latter, it is expected that degrading PGN within the cells will not prevent the NF-κB activation that depends on extracellular PGN detection by PGRP-LC. This is exactly what we observed for the proventriculus cells whose IMD activation can be tuned down only by extra- and not by intracytosolic amidases.

The fact that ubiquitous, but not gut-only, overexpression of cytosolic amidases can counteract the effect of chronic Ecc infection demonstrates that PGN sensing within the cytosol occurs in cells other than enterocytes. This result is in agreement with recent work showing that a PGRP-LE-dependent PGN sensing in octopaminergic neurons is important to adapt female egg-laying behavior to their infectious status (Kurz et al., 2017). However, since PGRP-LB<sup>PA</sup> or PGRP-LB<sup>PD</sup> ectopic expression in neurons via Elav<sup>Gal4</sup> did not prevent the deleterious effects of chronic Ecc infection (Figure S6D), it appears that constant IMD pathway activation in these cells is not the cause of Ecc-induced lethality. One of the future challenges will be to identify such cells. Our inability to rescue lethality of Ecc chronically infected flies with Gal4 drivers specific for muscles (Mef2<sup>Gal4</sup>) and hemocytes (Hml<sup>Gal4</sup>) suggests that these cells are not directly implicated, although additive effects are possible (Figure S6D). It should also be noted that the deleterious effect on ovaries of an oral Ecc chronic infection that we report here is quite different from the ones that we described earlier after an acute bacterial infection (Kurz et al., 2017). Whereas this previous study demonstrated that an acute PGN sensing via octopaminergic neurons leads to an accumulation of stage 14 oocytes

with no sign of cell death, a constant PGN-mediated NF-κB signaling following Ecc chronic infection induces severe cell death phenotypes in midstage follicles (Figure S7). These phenotypes were fully rescued by pan expression of any of the PGRP-LB isoforms (Figure S7).

It should also be noted that PGRPs protein has been shown to regulate interactions between insects and bacteria that are not extracellular. For example, the mutualistic relationships between the tsetse fly and its obligate symbiont Wigglesworthia or between the weevil Sitophilus zeamais and its symbiont are regulated by PGRP-LB proteins (Anselme et al., 2006; Wang and Aksoy, 2012; Wang et al., 2009). Since these symbionts are intracellular, it is likely that the PGRP-LB intracytoplasmic isoforms are playing a major role in regulating this dialogue. Similarly, since PGRP-LE-mediated detection of PGN controls the growth of the intracellular pathogen Listeria monocytogenes in hemocytes (Yano et al., 2008), it would be interesting to test whether the interaction between the PGN of Listeria and PGRP-LE is fine-tuned by PGRP-LB<sup>PA</sup> and/or PGRP-LB<sup>PD</sup> in Drosophila blood cells.

Our data also show that by digesting PGN in the gut lumen, the secreted PGRP-LB isoform prevents its dissemination in the rest of the body. If this study does not reveal the precise mechanisms by which microbiota-derived PGN crosses the gut, it does, however, show that if the PGN has to penetrate inside enterocytes to reach the hemolymph, it is invisible to cytosolic amidase. Indeed, forced gut expression of PGRP-LB<sup>PD</sup> or PGRP-LB<sup>PA</sup> had no impact on the ability of gut-borne PGN to reach the hemolymph. Although not better characterized in its detailed mechanisms, evidence exists for a systemic regulation of non-gut cells by microbiota-derived PGN beyond Drosophila and namely in mammals (Bouskra et al., 2008). Clarke et al. showed that gut PGN is able to translocate across the mouse gut mucosa, enter the circulation, and reach the bone marrow where it primes neutrophils in a Nod1-dependent manner (Clarke et al., 2010). Additional work in mice showed that bacterial PGN derived from the commensal gut microbiota translocates into the brain where it is sensed by specific pattern recognition receptors (PRRs) of the innate immune system (Arentsen et al., 2017). Mice harboring mutations in one of the mammalian PGRP orthologs (Pglyrp2) present changes in social behavior similar to mice with manipulated microbiota. These findings suggest that the central activation of PRR by PGN could be one of the signals of the communication between the gut microbiota and the developing brain and highlight the importance of PGN as a microbiota-derived signaling

or PGRP-LB<sup>PD*10A</sup> (w<sup>1118</sup>; PGRP-LB<sup>PD*10A</sup>) or PGRP-LB<sup>PD*6A</sup>/PGRP-LB<sup>A</sup> (w<sup>1118</sup>; PGRP-LB<sup>PD*6A</sup>/PGRP-LB<sup>A</sup>) or PGRP-LB<sup>PD*10A</sup>/PGRP-LB<sup>A</sup> (w<sup>1118</sup>; PGRP-LB<sup>PD*10A</sup>/PGRP-LB<sup>A</sup>).
(A) In contrast to PGRP-LB<sup>A</sup> null mutants, PGRP-LB<sup>PD</sup> CRISPR mutants do not show IMD pathway overactivation in gut or in fat body after Ecc oral infection.
(B) Whereas PGRP-LB<sup>PC</sup> CRISPR mutants display only mild Dipt activation in the gut, they phenocopy PGRP-LB<sup>A</sup> null mutants as far as systemic immune response is concerned.
(C) Midgut and fat-body Dipt overexpression in PGRP-LB<sup>PC*6A</sup>/PGRP-LB<sup>A</sup> and PGRP-LB<sup>PC*10A</sup>/PGRP-LB<sup>A</sup> mutants is fully rescued by providing PGRP-LB<sup>PC</sup>::EGF in the midgut (via Mex<sup>Gal4</sup>) and in the fat body (via r4<sup>Gal4</sup>), respectively. mRNA level in control infected flies was set to 1 and values obtained with indicated genotypes were expressed as a fold of this value. Histograms correspond to the mean value ± SD of three experiments. ns, not significantly different from one another; *p < 0.05, **p < 0.01; Kruskal-Wallis test.
(D) Confocal images of the R1 midgut region and the fat body of adult flies mutant for PGRP-LB<sup>PC</sup> showing Dipt-Cherry expression after Ecc oral infection. Flies of the following genotypes are used: PGRP-LB<sup>PC*6A</sup> (w<sup>1118</sup>; Dipt-Cherry<sup>C1</sup>, PGRP-LB<sup>PC*6A</sup>) and PGRP-LB<sup>PC*10A</sup> (w<sup>1118</sup>; Dipt-Cherry<sup>C1</sup>, PGRP-LB<sup>PC*10A</sup>). Overexpression of Dipt-Cherry is observed in the fat body of the PGRP-LB<sup>PC</sup> CRISPR mutants compared with control flies (compare with Figure 3B). See also Figures S4 and S5.

10 Cell Host & Microbe 23, 1–14, February 14, 2018

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

A

Fat body | Ovaries
--- | ---
Control | 
Lipids | Dipt-Cherry | Merged | 
|  |  | 
PGRP-LB<sup>A</sup> | 
Lipids | Dipt-Cherry | Merged | 
20 μm |  | 500 μm | 

8 days of Ecc feeding

Fat body | Ovaries | Fat body | Ovaries
--- | --- | --- | ---
r4Gal<sub>4</sub>, PGRP-LB<sup>D</sup> +PGRP-LB<sup>C</sup> +PGRP-LB<sup>A</sup> |  | DaGal<sub>4</sub>, PGRP-LB<sup>A</sup> +PGRP-LB<sup>D</sup> +PGRP-LB<sup>C</sup> | 
Lipids | Dipt-Cherry | Merged | Lipids | Dipt-Cherry | Merged | 
Ov. | Ov. |  | Ov. | 
Lipids | Dipt-Cherry | Merged | Lipids | Dipt-Cherry | Merged | 
Ov. | Ov. |  | Ov. | 
20 μm |  | 500 μm | 20 μm |  | 500 μm | 

10 days of Ecc feeding | 8 days of Ecc feeding

B

Mex<sup>Gal4</sup>

![Graph](https://i.imgur.com/placeholder.png)

r4Gal4

![Graph](https://i.imgur.com/placeholder.png)

Da<sup>Gal4</sup>

![Graph](https://i.imgur.com/placeholder.png)

Control  
PGRP-LB<sup>A</sup>  
PGRP-LB<sup>A</sup> + X<sup>Gal4</sup> > PGRP-LB<sup>PA</sup>::EGFP  
PGRP-LB<sup>A</sup> + X<sup>Gal4</sup> > PGRP-LB<sup>PC</sup>::EGFP  
PGRP-LB<sup>A</sup> + X<sup>Gal4</sup> > PGRP-LB<sup>PD</sup>::EGFP  

Figure 6. Systemic, but Not Local, Protection Is Crucial to Resist Chronic Ecc Ingestion

(A) Confocal images of fat body and phase-contrast pictures of ovaries from adult flies chronically infected with Ecc. For confocal images, lipid droplets are stained with Bodipy 493/803 (green) and expression of Dipt-Cherry (red) is shown. For the phase-contrast pictures, each ovary (Ov.) is delimited with a white dashed line.

(B) Survival analysis of flies orally infected with Ecc reveals the essential role of PGRP-LB<sup>PA</sup> and PGRP-LB<sup>PD</sup> in preventing deleterious effects of the constant microbiota-derived PGN. (A and B) Flies of the following genotypes were used: control (*w*<sup>1118</sup>; ; Dipt-Cherry<sup>C1</sup>) or PGRP-LB<sup>A</sup>(*w*<sup>1118</sup>; ; Dipt-Cherry<sup>C1</sup>, PGRP-LB<sup>A</sup>) or PGRP-LB<sup>A</sup> expressing UAS-PGRP-LB<sup>PA</sup>::EGFP or UAS-PGRP-LB<sup>PC</sup>::EGFP or UAS-PGRP-LB<sup>PD</sup>::EGFP under the control of the indicated Gal4 line. See also Figure S6.

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

---

**Figure 7. Model of PGRP-LB Isoforms' Role and Mode of Action**

In wild-type flies, PGN released by gut-associated bacteria is digested in the lumen by the secreted PGRP-LB<sup>PC</sup> isoform. If some of this PGN is not digested by this scavenging enzyme or if the enzyme digestive capacity is overwhelmed by the amount of PGN, accumulating muropeptides can activate the NF-κB pathway in enterocytes. In most anterior midgut domains, such as the proventriculus, PGN is sensed at the plasma membrane via the PGRP-LC receptor. In more posterior domains, such as the posterior midgut, PGN is detected by the cytosolic PGRP-LE. By cleaving the PGN in the extracellular space, the secreted amidase prevents constitutive activation in PGRP-LC-dependent cells. The level of PGN signaling is also controlled inside the enterocytes by two cytosolic PGRP-LB isoforms (PGRP-LB<sup>PA</sup> and PGRP-LB<sup>PD</sup>) that could function redundantly. This double mechanism of PGN elimination in the extracellular lumen and in the enterocytes prevents constitutive NF-κB activation in response to microbiota. The ability of the secreted PGRP-LB<sup>PC</sup> isoform to cleave PGN in the gut lumen has also systemic consequences. In the absence of PGRP-LB<sup>PC</sup>, PGN that accumulates in the lumen crosses the gut epithelium, by unknown mechanisms, to *in fine* reach the hemolymph in which most of the adult tissues and organs are bathed. The presence of PGN in the hemolymph has important consequences. By binding to PGRP-LC expressed at the membrane of the fat-body cells, it induces production of AMP. Excess of PGN also triggers lipid droplet reduction and autophagy in fat cells and organ wasting. These metabolic consequences are due to intracellular PGN-dependent NF-κB pathway activation in cells that are not enterocytes or fat-body cells and remain to be identified. It has recently been shown that by activating the NF-κB pathway in the octopaminergic neurons, PGN adapts egg-laying behavior to the female (Kurz et al., 2017). The mechanisms by which PGN enters the cell for later PGRP-LE-mediated detection remain unknown.

metabolite. It would be important to identify, in mammals, PGN-modifying molecules, which, like in *Drosophila*, could regulate the PGN capacity to signal locally or systemically.

---

### STAR★METHODS

Detailed methods are provided in the online version of this paper and include the following:

- **KEY RESOURCES TABLE**
- **CONTACT FOR REAGENT AND RESOURCE SHARING**
- **EXPERIMENTAL MODEL AND SUBJECT DETAILS**

---

### METHOD DETAILS

- Drosophila Culture Media
- Drosophila Genetics and Analysis
- Generation of Transgenic Flies
- Confocal Imaging
- Electronic Microscopy
- Lysotracker and Bodipy Staining
- Quantitative Real-Time PCR
- Infection of Adults by Ecc-1
- Survival Tests with Bacterial Infection

---

12 Cell Host & Microbe 23, 1–14, February 14, 2018

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

- **QUANTIFICATION AND STATISTICAL ANALYSIS**
  - Replication
  - Statistical Analysis

**SUPPLEMENTAL INFORMATION**

Supplemental Information includes seven figures and one table and can be found with this article online at [https://doi.org/10.1016/j.chom.2017.12.007](https://doi.org/10.1016/j.chom.2017.12.007).

**ACKNOWLEDGMENTS**

We thank Ambra Masuzzo and Marie Meister for comments on the manuscript. This work was supported by CNRS, Equipe FRM to Julien Royet “Equipe FRM DEQ20140329541,” ANR-11-LABX-0054 (Investissements d’Avenir–Labex INFORM).

**AUTHOR CONTRIBUTIONS**

Conceptualization, B.C. and J.R.; Investigation, B.C., F.C., C.L.K., S.P., D.C., and A.V.; Writing – Original Draft, B.C. and J.R.; Writing – Review & Editing, B.C. and J.R.; Funding Acquisition, J.R.

Received: June 6, 2017  
Revised: October 10, 2017  
Accepted: December 14, 2017  
Published: February 1, 2018

**REFERENCES**

Chatterjee, A., Roy, D., Patnaik, E., and Nongthomba, U. (2016). Muscles provide protection during microbial infection by activating innate immune response pathways in *Drosophila* and zebrafish. Dis. Model. Mech. 9, 697–705.

Chen, Y., Wang, Z., Wang, P., Li, D., Zhou, J., and Wu, S. (2014). CYLD negatively regulates Hippo signaling by limiting Hpo phosphorylation in *Drosophila*. Biochem. Biophys. Res. Commun. 452, 808–812.

Choe, K.M., Lee, H., and Anderson, K.V. (2005). *Drosophila* peptidoglycan recognition protein LC (PGRP-LC) acts as a signal-transducing innate immune receptor. Proc. Natl. Acad. Sci. USA 102, 1122–1126.

Clarke, T.B., Davis, K.M., Lysenko, E.S., Zhou, A.Y., Yu, Y., and Weiser, J.N. (2010). Recognition of peptidoglycan from the microbiota by Nod1 enhances systemic innate immunity. Nat. Med. 16, 228–231.

Erkosar, B., Storelli, G., Defaye, A., and Leulier, F. (2013). Host-intestinal microbiota mutualism: “learning on the fly”. Cell Host Microbe 13, 8–14.

Gottar, M., Gobert, V., Michel, T., Belvin, M., Duyk, G., Hoffmann, J.A., Ferrandon, D., and Royet, J. (2002). The *Drosophila* immune response against Gram-negative bacteria is mediated by a peptidoglycan recognition protein. Nature 416, 640–644.

Grigg, J.B., and Sonnenberg, G.F. (2017). Host-microbiota interactions shape local and systemic inflammatory diseases. J. Immunol. 198, 564–571.

Guo, L., Karpac, J., Tran, S.L., and Jasper, H. (2014). PGRP-SC2 promotes gut immune homeostasis to limit commensal dysbiosis and extend lifespan. Cell 156, 109–122.

Honda, K., and Littman, D.R. (2016). The microbiota in adaptive immune homeostasis and disease. Nature 535, 75–84.

Anselme, C., Vallier, A., Balmand, S., Fauvarque, M.O., and Heddi, A. (2006). Host PGRP gene expression and bacterial release in endosymbiosis of the weevil Sitophilus zeamais. Appl. Environ. Microbiol. 72, 6766–6772.

Arentsen, T., Qian, Y., Gkotzis, S., Femenia, T., Wang, T., Udekwu, K., Forssberg, H., and Diaz Heijtz, R. (2017). The bacterial peptidoglycan-sensing molecule Pglyrp2 modulates brain development and behavior. Mol. Psychiatry 22, 257–266.

Basset, A., Khush, R.S., Braun, A., Gardan, L., Boccard, F., Hoffmann, J.A., and Lemaitre, B. (2000). The phytopathogenic bacteria Erwinia carotovora infects *Drosophila* and activates an immune response. Proc. Natl. Acad. Sci. USA 97, 3376–3381.

Bosch, J.A., Tran, N.H., and Hariharan, I.K. (2015). CoinFLP: a system for efficient mosaic screening and for visualizing clonal boundaries in *Drosophila*. Development 142, 597–606.

Bosco-Drayon, V., Poidevin, M., Boneca, I.G., Narbonne-Reveau, K., Royet, J., and Charroux, B. (2012). Peptidoglycan sensing by the receptor PGRP-LE in the *Drosophila* gut induces immune responses to infectious bacteria and tolerance to microbiota. Cell Host Microbe 12, 153–165.

Bouskra, D., Brezillon, C., Berard, M., Werts, C., Varona, R., Boneca, I.G., and Eberl, G. (2008). Lymphoid tissue genesis induced by commensals through NOD1 regulates intestinal homeostasis. Nature 456, 507–510.

Buchon, N., Broderick, N.A., and Lemaitre, B. (2013a). Gut homeostasis in a microbial world: insights from *Drosophila melanogaster*. Nat. Rev. Microbiol. 11, 615–626.

Buchon, N., and Osman, D. (2015). All for one and one for all: regionalization of the *Drosophila* intestine. Insect Biochem. Mol. Biol. 67, 2–8.

Buchon, N., Osman, D., David, F.P., Fang, H.Y., Boquete, J.P., Deplancke, B., and Lemaitre, B. (2013b). Morphological and molecular characterization of adult midgut compartmentalization in *Drosophila*. Cell Rep. 3, 1725–1738.

Burgess, J., Del Bel, L.M., Ma, C.I., BarylkO, B., Polevoy, G., Rollins, J., Albanesi, J.P., Kramer, H., and Brill, J.A. (2012). Type II phosphatidylinositol 4-kinase regulates trafficking of secretory granule proteins in *Drosophila*. Development 139, 3040–3050.

Charroux, B., and Royet, J. (2009). Elimination of plasmatocytes by targeted apoptosis reveals their role in multiple aspects of the *Drosophila* immune response. Proc. Natl. Acad. Sci. USA 106, 9797–9802.

Kaneko, T., Yano, T., Aggarwal, K., Lim, J.H., Ueda, K., Oshima, Y., Peach, C., Erturk-Hasdemir, D., Goldman, W.E., Oh, B.H., et al. (2006). PGRP-LC and PGRP-LE have essential yet distinct functions in the *Drosophila* immune response to monomeric DAP-type peptidoglycan. Nat. Immunol. 7, 715–723.

Kurata, S. (2010). Extracellular and intracellular pathogen recognition by *Drosophila* PGRP-LE and PGRP-LC. Int. Immunol. 22, 143–148.

Kurz, C.L., Charroux, B., Chaduli, D., ViaIlat-Lieutaud, A., and Royet, J. (2017). Peptidoglycan sensing by octopaminergic neurons modulates *Drosophila* oviposition. Elife 6, e21937.

Lee, K.A., Kim, B., Bhin, J., Kim, D.H., You, H., Kim, E.K., Kim, S.H., Ryu, J.H., Hwang, D., and Lee, W.J. (2015). Bacterial uracil modulates *Drosophila* DUOX-dependent gut immunity via Hedgehog-induced signaling endosomes. Cell Host Microbe 17, 191–204.

Lee, K.A., Kim, S.H., Kim, E.K., Ha, E.M., You, H., Kim, B., Kim, M.J., Kwon, Y., Ryu, J.H., and Lee, W.J. (2013). Bacterial-derived uracil as a modulator of mucosal immunity and gut-microbe homeostasis in *Drosophila*. Cell 153, 797–811.

Lee, K.Z., and Ferrandon, D. (2011). Negative regulation of immune responses on the fly. EMBO J. 30, 988–990.

Ma, D., Storelli, G., Mitchell, M., and Leulier, F. (2015). Studying host-microbiota mutualism in *Drosophila*: harnessing the power of gnotobiotic flies. Biomed. J. 38, 285–293.

Mellroth, P., Karlsson, J., and Steiner, H. (2003). A scavenger function for a *Drosophila* peptidoglycan recognition protein. J. Biol. Chem. 278, 7059–7064.

Mellroth, P., and Steiner, H. (2006). PGRP-SB1: an N-acetylmuramoyl L-alanine amidase with antibacterial activity. Biochem. Biophys. Res. Commun. 350, 994–999.

Neyen, C., Poidevin, M., Roussel, A., and Lemaitre, B. (2012). Tissue- and ligand-specific sensing of gram-negative infection in *Drosophila* by PGRP-LC isoforms and PGRP-LE. J. Immunol. 189, 1886–1897.

Neyen, C., Runchel, C., Schupfer, F., Meier, P., and Lemaitre, B. (2016). The regulatory isoform rPGRP-LC induces immune resolution via endosomal degradation of receptors. Nat. Immunol. 17, 1150–1158.

Paredes, J.C., Welchman, D.P., Poidevin, M., and Lemaitre, B. (2011). Negative regulation by amidase PGRPs shapes the *Drosophila* antibacterial response and protects the fly from innocuous infection. Immunity 35, 770–779.

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

Philpott, D.J., Sorbara, M.T., Robertson, S.J., Croitoru, K., and Girardin, S.E. (2014). NOD proteins: regulators of inflammation in health and disease. Nat. Rev. Immunol. 14, 9–23.

Ramat, M., Manfruelli, P., Pearson, A., Mathey-Prevot, B., and Ezekowitz, R.A. (2002). Functional genomic analysis of phagocytosis and identification of a *Drosophila* receptor for E. coli. Nature 416, 644–648.

Rooks, M.G., and Garrett, W.S. (2016). Gut microbiota, metabolites and host immunity. Nat. Rev. Immunol. 16, 341–352.

Ryu, J.H., Kim, S.H., Lee, H.Y., Bai, J.Y., Nam, Y.D., Bae, J.W., Lee, D.G., Shin, S.C., Ha, E.M., and Lee, W.J. (2008). Innate immune homeostasis by the homeobox gene caudal and commensal-gut mutualism in *Drosophila*. Science 319, 777–782.

Sharon, G., Sampson, T.R., Geschwind, D.H., and Mazmanian, S.K. (2016). The central nervous system and the gut microbiome. Cell 167, 915–932.

Sommer, F., and Backhed, F. (2013). The gut microbiota—masters of host development and physiology. Nat. Rev. Microbiol. 11, 227–238.

Takehana, A., Yano, T., Mita, S., Kotani, A., Oshima, Y., and Kurata, S. (2004). Peptidoglycan recognition protein (PGRP)-LE and PGRP-LC act synergistically in *Drosophila* immunity. EMBO J. 23, 4690–4700.

Tavignot, R., Chaduli, D., Djitte, F., Charroux, B., and Royet, J. (2017). Inhibition of a NF-kappaB/Diap1 pathway by PGRP-LF is required for proper apoptosis during *Drosophila* development. PLoS Genet. 13, e1006569.

Vodovar, N., Vinals, M., Liehl, P., Basset, A., Degrouard, J., Spellman, P., Boccard, F., and Lemaitre, B. (2005). *Drosophila* host defense after oral infection by an entomopathogenic *Pseudomonas* species. Proc. Natl. Acad. Sci. USA 102, 11414–11419.

Wang, J., and Aksoy, S. (2012). PGRP-LB is a maternally transmitted immune milk protein that influences symbiosis and parasitism in tsetse’s offspring. Proc. Natl. Acad. Sci. USA 109, 10552–10557.

Wang, J., Wu, Y., Yang, G., and Aksoy, S. (2009). Interactions between mutualist Wigglesworthia and tsetse peptidoglycan recognition protein (PGRP-LB) influence trypanosome transmission. Proc. Natl. Acad. Sci. USA 106, 12133–12138.

Yano, T., Mita, S., Ohmori, H., Oshima, Y., Fujimoto, Y., Ueda, R., Takada, H., Goldman, W.E., Fukase, K., Silverman, N., et al. (2008). Autophagic control of listeria through intracellular innate immune recognition in *Drosophila*. Nat. Immunol. 9, 908–916.

Zaidman-Rémy, A., Hervé, M., Poidevin, M., Pili-Floury, S., Kim, M.S., Blanot, D., Oh, B.H., Ueda, R., Mengin-Lecreulx, D., and Lemaitre, B. (2006). The *Drosophila* amidase PGRP-LB modulates the immune response to bacterial infection. Immunity 24, 463–473.

Zaidman-Rémy, A., Poidevin, M., Hervé, M., Welchman, D.P., Paredes, J.C., Fahlander, C., Steiner, H., Mengin-Lecreulx, D., and Lemaitre, B. (2011). *Drosophila* immunity: analysis of PGRP-SB1 expression, enzymatic activity and function. PLoS One 6, e17231.

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in Drosophila Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

STAR★METHODS

KEY RESOURCES TABLE

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
| --- | --- | --- |
| Bacterial and Virus Strains |  |  |
| Ecc, Erwinia carotovora carotovora 152141 | Basset et al., 2000 | N/A |
| Chemicals, Peptides, and Recombinant Proteins |  |  |
| Agar | VWR | Cat# 20768.361 |
| Yeast extract | VWR | Cat# 24979.413 |
| Methylparaben sodium salt | MERCK | Cat# 106756 |
| Propionic acid | CARLOERBA | Cat# 409553 |
| Ampicillin | Euromedex | Cat# EU 0400-B |
| Kanamycin | Fisher | Cat# BP 906-5 |
| Tetracyclin | Sigma-Aldrich | Cat# 87128 |
| Erythromycin | Sigma-Aldrich | Cat# E 5389 |
| Bodipy 493/503 | Invitrogen | Cat# D 3922 |
| Lysotracker Red DND-99 | Invitrogen | Cat# L 7528 |
| Paraformaldehyde | Electron Microscopy Sciences | Cat# 15714-S |
| Glutaraldehyde | Sigma | Cat# G-6257 |
| Dulbecco's phosphate buffer saline | EUROBIO | Cat# CSOPBS 01-08 |
| Triton-X-100 | Euromedex | Cat# 2000-B |
| Sodium cacodylate | Electron Microscopy Sciences | Cat# 12310 |
| Osmium tetroxide | Electron Microscopy Sciences | Cat# 19170 |
| Sucrose | Normapur | Cat# 27480-294 |
| Vectashield | Vector Laboratories | Cat# H 1200 |
| Critical Commercial Assays |  |  |
| RNeasy Mini Kit | QIAGEN | Cat# 74106 |
| SuperScript III | Invitrogen | Cat# 18080-044 |
| FastStart Universal SYBR Green Master | Sigma-Aldrich | Cat# 04913914001 |
| Experimental Models: Organisms/Strains |  |  |
| Drosophila melanogaster w[*]; P{w[+mW.hs]=GAL4-da.G32}UH1, Sb[1]/TM6B, Tb[1] | Bloomington Drosophila Stock Center | BDSC_55851 |
| Drosophila melanogaster y[1] w[*]; P{w[+mC]=GAL4-Mef2.R}3 | Bloomington Drosophila Stock Center | BDSC_27390 |
| Drosophila melanogaster w[*]; P{w[+mC]=GAL4-r4} | Chen et al., 2014 | N/A |
| Drosophila melanogaster w[*]; P{w[+mC]=GAL4-Mex} | Chen et al., 2014 | N/A |
| Drosophila melanogaster w[*]; P{w[+mC]=GAL4.oeno}4M | Bloomington Drosophila Stock Center | BDSC_65405 |
| Drosophila melanogaster w[*]; P{Sgs3-DsRed} | Burgess et al., 2012 | N/A |
| Drosophila melanogaster w[*]; P{w[+mC]=UAS-2xEGFP}AH2 | Bloomington Drosophila Stock Center | BDSC_6874 |
| Drosophila melanogaster w[*]; P{w[+mC]=UAS-mCD8.ChRFP}2 | Bloomington Drosophila Stock Center | BDSC_27391 |
| Drosophila melanogaster w[*]; P{AttacinD-Cherry}2 | Tavignot et al., 2017 | N/A |
| Drosophila melanogaster w[*]; P{Diptericin-Cherry^1}3 | Charroux and Royet, 2009 | N/A |
| Drosophila melanogaster w[*]; P{Diptericin-Cherry^2}2 | B.C. and J.R., unpublished | N/A |
| Drosophila melanogaster w[*] P{y[+t7.7]} w[+mC]=CoinFLP-GAL4}attP3 | Bloomington Drosophila Stock Center | BDSC_59269 |
| Drosophila melanogaster w[1118]; P{ry[+t7.2]=70FLP}10 | Bloomington Drosophila Stock Center | BDSC_6938 |

(Continued on next page)

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

**Continued**

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
| --- | --- | --- |
| *Drosophila melanogaster* P{ry[+t7.2]=neoFRT}82B , PGRP-LB<sup>4</sup> | This study | N/A |
| *Drosophila melanogaster* y[1] w[*]; PGRP-LE<sup>112</sup> | Takehana et al., 2004 | N/A |
| *Drosophila melanogaster* w[*]; PGRP-LC<sup>E12</sup> | Gottar et al., 2002 | N/A |
| *Drosophila melanogaster* w[*]; PGRP-LB<sup>4</sup> | Paredes et al., 2011 | N/A |
| *Drosophila melanogaster* w[*]; PGRP-LB<sup>PD*Z10</sup> | Kurz et al., 2017 | N/A |
| *Drosophila melanogaster* w[*]; PGRP-LB<sup>PD*Z10b</sup> | Kurz et al., 2017 | N/A |
| *Drosophila melanogaster* w[*]; PGRP-LB<sup>PC*6A</sup> | This study | N/A |
| *Drosophila melanogaster* w[*]; PGRP-LB<sup>PC*10A</sup> | This study | N/A |
| *Drosophila melanogaster* w[*]; P{w[+mC]= UAS-PGRP-LB<sup>PA</sup>::EGFP}2 | Kurz et al., 2017 | N/A |
| *Drosophila melanogaster* w[*]; P{w[+mC]= UAS-PGRP-LB<sup>PC</sup>::EGFP}2 | Kurz et al., 2017 | N/A |
| *Drosophila melanogaster* w[*]; P{w[+mC]= UAS-PGRP-LB<sup>PD</sup>::EGFP}2 | Kurz et al., 2017 | N/A |
| *Drosophila melanogaster* w[*]; | This study | N/A |
| P{w[+mC]= UAS-PGRP-LB<sup>PA-C166S</sup>::EGFP}2 |  |  |
| *Drosophila melanogaster* w[*]; | This study | N/A |
| P{w[+mC]= UAS-PGRP-LB<sup>PC-C177S</sup>::EGFP }2 |  |  |
| *Drosophila melanogaster* w[*]; | This study | N/A |
| P{w[+mC]= UAS-PGRP-LB<sup>PD-C200S</sup>::EGFP }2 |  |  |
| *Drosophila melanogaster* w[*]; P{w[+mC]=GAL4-pLB1}2 | Kurz et al., 2017 | N/A |
| *Drosophila melanogaster* w[*]; P{w[+mC]=GAL4-pLB2}2 | Kurz et al., 2017 | N/A |
| *Drosophila melanogaster* w[*]; P{w[+mC]=GAL4-pLB3}2 | Kurz et al., 2017 | N/A |

**Oligonucleotides**

|  |  |  |
| --- | --- | --- |
| Dipteracin forward Q rtPCR GCTGCGCAATCGCTTCTACT | Eurofins Genomics | N/A |
| Dipteracin reverse Q rtPCR TGGTGGAGTGGGCTTCATG | Eurofins Genomics | N/A |
| rp49 forward Q rtPCR CGCTTCAAGGGACAGTATCTG | Eurofins Genomics | N/A |
| rp49 reverse Q rtPCR AAACGCCGGTTCTGCATGA | Eurofins Genomics | N/A |
| PGRP-LB forward Q rtPCR GCTCATGCACCCAAGTACAA | Eurofins Genomics | N/A |
| PGRP-LB reverse Q rtPCR GGGTCAATGTAGCCCTTGAA | Eurofins Genomics | N/A |
| PGRP-LB<sup>PA</sup> forward ATGCAGCAGGCGAATCTGG | Eurofins Genomics | N/A |
| PGRP-LB<sup>PA</sup> reverse GACCTTGGGCGCAGCTG | Eurofins Genomics | N/A |
| PGRP-LB<sup>PC</sup> forward ATGACAGCACTGGGATTAG | Eurofins Genomics | N/A |
| PGRP-LB<sup>PC</sup> reverse GACCTTGGGCGCAGCTG | Eurofins Genomics | N/A |

Additional oligonucleotides see **Table S1**

**Software and Algorithms**

|  |  |  |
| --- | --- | --- |
| GraphPad Prism 6 | GraphPad software | https://www.graphpad.com/scientific-software/prism/ |
| SignalP 4.1 Server | Center of Biological Sequence, Technical University of Denmark | http://www.cbs.dtu.dk/services/SignalP/ |

**CONTACT FOR REAGENT AND RESOURCE SHARING**

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact Julien Royet ([julien.royet@univ-amu.fr](mailto:julien.royet@univ-amu.fr)).

**EXPERIMENTAL MODEL AND SUBJECT DETAILS**

**Animals**

**Drosophila**

Female flies were used for all experiments. Flies were grown at 25°C on a yeast/cornmeal medium in 12h/12h light/dark cycle controlled incubators. The full list of genotypes of the flies used can be found in the **Key Resources Table**.

**Microbe Strain**

The bacterial strain used was *Erwinia carotovora carotovora*<sup>15 2141</sup> (Ecc) cultured in in Luria-Bertani medium at 30°C overnight.

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

# METHOD DETAILS

## Drosophila Culture Media
For 1 liter of food, 8.2 g of agar, 80 g of cornmeal flour (Westhove, Farigel maize H1) and 80 g of yeast extract were cooked for 10 min in boiling water. 5.2 g of Methylparaben sodium salt and 4 mL of 99% propionic acid were added when the food had cooled down. For antibiotic (ATB) treatment, standard medium was supplemented with Ampicillin, Kanamycin, Tetracyclin and Erythromycin at 50 μg/ml final concentrations.

## Drosophila Genetics and Analysis
To generate the $PGRP-LB^{PC}$ CRISPR alleles, plasmid encoding two synthetic guide RNAs (sgRNAs) was injected (100 ng/μl) into preblastoderm embryos of nosPhiC31; attP40 flies. sgRNAs (sgRNA1: GTGCTGTGCATGTTGCTGCTT and sgRNA2: TTAGTTCTGC TATCGATGAT) were designed using the online CRISPR design tool ([http://crispor.tefor.net](http://crispor.tefor.net)) to target RC specific exon of $PGRP-$ $LB$ and were cloned into the pCFD4-U6:1-U6:3 vector (Addgene #49411;). Males transgenic flies were crossed with Cas9 nickase transgenic females flies. This strategy reduces off-target activity. We used the $y^1$, $w^{1118}$ $P(3xP3-EGFP$, vasa-cas9D10A) attP18 fly line. G0 founder flies were crossed with $w^{1118}; MKRS/TM6b$ flies then the F1 progeny was crossed to Dipt-Cherry${}^{C1}$ $PGRP-LB^{\Delta}$ flies and screened for enhanced fat body activation of Dipt-Cherry after Ecc feeding

For $PGRP-LB^{\Delta}$ mutant clones assay, 4-6 days old mated females raised and aged in ATB containing media were heat shocked for 30 minutes at 37°C to induce clones, then the flies were kept at 25°C during 2-3 days in antibiotic media. Adult flies were next transferred into non-ATB media during 24h before being infected by Ecc at 25°C for 48h. Adult flies with the following genotype were used: $yw hsflp tubGal4 UAS-GFP/+; AttD-Cherry/+; FRT82B PGRP-LB^{\Delta}/FRT82B tubGal^{80}$.

For UAS-$PGRP-LB^{isoform}$ overexpressing clones in $PGRP-LB^{\Delta}$ mutant background, 5-8 days old mated females were raised and aged in presence of ATB at 25°C. Adult flies were then transferred into non-ATB media for 24h before the infection by Ecc for 48h. No heat shock was required for clone induction. Flies of the following genotype were used: CoinFLP${}^{Gal4}$/+; hsFLP/UAS-$PGRP-LB^{PA::EGFP}$ or UAS-$PGRP-LB^{PD::EGFP}$; Dipt-Cherry${}^{C1}$ $PGRP-LB^{\Delta}$/Dipt-Cherry${}^{C1}$ $PGRP-LB^{\Delta}$.

## Generation of Transgenic Flies
We used a two-step PCR approach with overlapping primers to generate the UAS-$PGRP-LB^{PA-C160S::EGFP}$, UAS-$PGRP-$ $LB^{PC-C177S::EGFP}$ and UAS-$PGRP-LB^{PD-C200S::EGFP}$ constructs. To do so we first amplified two overlapping DNA fragments of the coding sequence of each $PGRP-LB$ isoform using a set of overlapping and modified primers ($PGRP-LB^{PA}$ forward or $PGRP-$ $LB^{PC}$ forward or $PGRP-LB^{PD}$ forward with $PGRP-LB$ reverse Cysteine to Serine and $PGRP-LB$ forward Cysteine to Serine with either $PGRP-LB^{PA}$ reverse or $PGRP-LB^{PC}$ reverse or $PGRP-LB^{PD}$ reverse) such that the Cysteine coding sequence TGT was replaced by the Serine coding sequence TCT. The two PCRs products obtained were then mixed together and used as DNA template in a second PCR reaction aimed to amplify the full coding sequence of each isoform, with primers containing AttB1 or AttB2 sequences ($PGRP-LB^{PA}$ forward AttB1 with $PGRP-LB^{PA}$ reverse AttB2, $PGRP-LB^{PC}$ forward AttB1 with $PGRP-LB^{PC}$ reverse AttB2 and $PGRP-$ $LB^{PD}$ forward AttB1 with $PGRP-LB^{PD}$ reverse AttB2) for further cloning in pTWG using the Gateway technology (Invitrogen).

## Confocal Imaging
Larval or adult tissues were dissected in PBS, fixed for 20 min in 4% paraformaldehyde on ice and rinse 3 times in PBT (PBS + 0.1% Triton X-100). The tissues were mounted in Vectashield (Vector Laboratories) fluorescent mounting medium, with or without DAPI. Images were captured with a LSM 780 Zeiss confocal microscope and processed using Adobe Photoshop.

## Electronic Microscopy
For electron microscopic sections, adult abdominal carcasses were dissected and fixed at RT in 4% PFA and 2% glutaraldehyde in 0.12 M sodium cacodylate buffer at pH 7.4 for 1 hr. The carcasses were then washed 3610 min in 0.12 M sodium cacodylate buffer, post-fixed in 2% OsO4 in 0.12 M sodium cacodylate buffer for 1 hr and washed again 3 X 10 min. Samples were subsequently dehydrated through series of ethanol gradients and infiltrated with propylene oxide, embedded in epoxy resin (Fluka, Sigma) and polymerized at 80°C. Ultrathin (80 nm) plastic sections were cut using a Leica UltraCut microtome with a diamond Diatome knife and post-stained with 2% uranyl acetate, followed by treatment with Reynolds’lead citrate, and stabilized for transmission EM by carbon coating. Examination was performed with a Zeiss Leo 912 microscope at 100 kV. Images were captured using a Gatan 792 Bioscan camera using Digital Micrograph software.

## Lysotracker and Bodipy Staining
For Lysotracker and Bodipy staining, adult tissues were dissected in PBS, fixed for 20 min in 4% paraformaldehyde on ice, rinse 3 times in PBS without detergent and stained with Lysotracker™ Red DND-99 and/or Bodipy™ 493/503 at 1:1000 in PBS for 30 min.

## Quantitative Real-Time PCR
RNA from whole dissected organs (n=12) was extracted with RNeasy Mini Kit. Three hundred nanograms of total-RNA was then reverse transcribed in 10 ul reaction volume using the Superscript III enzyme (Invitrogen) and random hexamer primers. Quantitative real-time PCR was performed on a CFX96 Real-Time PCR Detection System (BIO-RAD) in 96-well plates using the FastStart

Please cite this article in press as: Charroux et al., Cytosolic and Secreted Peptidoglycan-Degrading Enzymes in *Drosophila* Respectively Control Local and Systemic Immune Responses to Microbiota, Cell Host & Microbe (2017), https://doi.org/10.1016/j.chom.2017.12.007

Universal SYBR Green Master (Sigma-Aldrich). The amount of mRNA detected was normalized to control rp49 mRNA values. Normalized data was used to quantify the relative levels of a given mRNA according to cycling threshold analysis (ΔCt). All datasets were organized and analyzed in Microsoft Excel 2013.

**Infection of Adults by Ecc-1**

Bacterial cultures were centrifuged at 2500 g for 15 min at RT and resuspended in fresh Luria-Bertani medium. Cells were serially diluted in PBS and their concentration was determined by optical density (OD) measurement at 600 nm. For oral infection, 4-6 days female were raised at 25°C in presence of ATB in the food. 24h before the infection, female flies were transferred in vial without ATB and then placed in a fly vial with *Ecc* contaminated food. This food solution was obtained by mixing a pellet of an overnight culture of bacteria *Ecc* (OD = 200) with a solution of 5% sucrose (50/50) and added to a filter disk that completely covered the agar surface of the fly vial. Septic injuries were performed by pricking adult females with a thin needle dipped into a solution of *Ecc* OD=200.

**Survival Tests with Bacterial Infection**

For oral infections, adult flies were transferred every 2 days in a fresh vial in which 150 microliters of a fresh solution of *Ecc* (OD =200) 5% sucrose (50/50) has been deposited.

**QUANTIFICATION AND STATISTICAL ANALYSIS**

**Replication**

For survival curves, at least three samples of 20 flies for each genotype were used for a given survival experiment. At least three independent survival experiments were performed to test one hypothesis. For qRT-PCR, at least 15 flies of a given genotype was used. At least, three independent experiments were performed for each condition.
- No Strategy for randomization and/or stratification was applied
- No Inclusion and exclusion criteria of any data or subjects was used.

**Statistical Analysis**

The Prism software (GraphPad) was used for statistical analyses. Our datasets were tested for normality using the D’Agostino-Pearson omnibus test and none of them passed the test (except the data of Figure 2B). Consequently, we used the unpaired Mann-Whitney or Kruskal-Wallis tests. Details on statistical analysis can be found in the figure legends.

e4 Cell Host & Microbe 23, 1–14.e1–e4, February 14, 2018
