
ARTICLE

doi:10.1038/nature25462

Cryo–EM shows how dynactin recruits two dyneins for faster movement

Linas Urnavicius${}^{1*}$, Clinton K. Lau${}^{1*}$, Mohamed M. Elshenawy${}^{2}$, Edgar Morales–Rios${}^{3}$, Carina Motz${}^{1†}$, Ahmet Yildiz${}^{2,4}$ & Andrew P. Carter${}^{1}$

---

Dynein and its cofactor dynactin form a highly processive microtubule motor in the presence of an activating adaptor, such as BICD2. Different adaptors link dynein and dynactin to distinct cargoes. Here we use electron microscopy and single-molecule studies to show that adaptors can recruit a second dynein to dynactin. Whereas BICD2 is biased towards recruiting a single dynein, the adaptors BICDR1 and HOOK3 predominantly recruit two dyneins. We find that the shift towards a double dynein complex increases both the force and speed of the microtubule motor. Our 3.5 Å resolution cryo-electron microscopy reconstruction of a dynein tail–dynactin–BICDR1 complex reveals how dynactin can act as a scaffold to coordinate two dyneins side-by-side. Our work provides a structural basis for understanding how diverse adaptors recruit different numbers of dyneins and regulate the motile properties of the dynein–dynactin transport machine.

Cytoplasmic dynein-1 (dynein) is the main transporter of cargoes towards the minus ends of microtubules in animal cells${}^{1}$. These cargoes move at a range of speeds${}^{2}$ and vary in size from large organelles${}^{3}$ to small individual proteins${}^{4}$. Dynein is activated to form a highly processive motor by binding its cofactor dynactin and a cargo adaptor, such as BICD2 (bicaudal D homologue 2)${}^{5,6}$. Dynein contains two motor domains joined by a tail region, whereas dynactin is built around a short actin-like filament, capped at its pointed and barbed ends and decorated with a shoulder${}^{6-9}$. A previous 8 Å resolution cryo-electron microscopy (cryo-EM) structure showed how a coiled coil in BICD2 recruits the tail of dynein to the filament of dynactin${}^{8}$. Other adaptors that activate dynein and link it to different cargoes have been identified${}^{5,10,11}$. These activating adaptors also contain long coiled coils; however, the sequence similarity between them is low${}^{12-15}$ and it is unclear whether they engage dynein and/or dynactin in the same way as BICD2 does. There is also evidence that certain adaptors—such as BICDR1${}^{14}$ (BICD related-1, also known as BICDL1) and HOOK3${}^{5,10,11}$—drive faster movement of dynein towards the minus ends of microtubules when compared with BICD2, although the mechanism underpinning this increased speed is not currently understood.

### Dynactin can recruit two dyneins

We determined the cryo-EM structures of two previously unsolved dynein–dynactin–adaptor complexes. BICDR1, like BICD2, binds RAB6 vesicles${}^{16}$, whereas HOOK3 links dynein and dynactin to early endosomes${}^{17,18}$. We determined 7 Å resolution maps of both the dynein tail–dynactin–BICDR1 complex (hereafter termed TDR) and the dynein tail–dynactin–HOOK3 complex (hereafter termed TDH), which we compare to the previously determined structure of the dynein tail–dynactin–BICD2 complex (hereafter termed TDB)${}^{8}$ (Fig. 1a, Extended Data Fig. 1a–d, Extended Data Table 1).

The coiled coils of all three adaptors run along the length of the dynactin filament (Fig. 1a). However, in contrast to previous predictions${}^{13}$, each adaptor makes different interactions. BICD2 and BICDR1 diverge in their path and relative rotation (Fig. 1b). HOOK3 follows yet another route over the surface of dynactin (Fig. 1c). TDH also shows an extra coiled-coil density near the pointed end of dynactin (Fig. 1c) and extra globular density towards its barbed end (Extended Data Fig. 1e, f). The identity of the second coiled coil is unclear, but the globular density probably corresponds to the N-terminal Hook domain, which is required for HOOK3 to activate dynein and dynactin${}^{11,19}$.

The most notable feature of TDR and TDH is the presence of two dynein tails (Fig. 1a). The tail of the first dynein (dynein-A) binds in an equivalent position to the dynein tail in TDB${}^{8}$ and to the full-length dynein in the dynein–dynactin–BICD2 complex (hereafter termed DDB)${}^{9}$. The second dynein (dynein-B) binds next to dynein-A near the barbed end of dynactin.

#### Adaptors determine dynein recruitment

We determined whether BICD2, BICDR1 and HOOK3 recruited different numbers of dyneins in moving dynein–dynactin complexes. We mixed dyneins that had been labelled with tetramethylrhodamine (TMR) or Alexa Fluor 647, and used single-molecule fluorescence microscopy to measure the frequency at which the two dyes colocalized on microtubules (Fig. 2a, b). In the presence of dynactin and BICD2, $13 \pm 1\%$ (s.e.m.) of processive complexes were labelled with both dyes; this was significantly higher ($P < 0.0001$) than the colocalization observed for the dynein-only control ($2.1 \pm 0.3\%$). Using BICDR1 or HOOK3 as an adaptor led to colocalization percentages of $31 \pm 2\%$ and $34 \pm 1\%$, respectively (Fig. 2b). After correction for complexes that were double-labelled with the same colour, we estimate that $26\%$ of BICD2 complexes contained two dyneins, compared to $61\%$ for BICDR1 and $67\%$ for HOOK3. We conclude that the majority of motile complexes that contain BICD2 have one dynein, whereas both BICDR1 and HOOK3 preferentially recruit two.

Although both this study and previous work${}^{6-9}$ are consistent with BICD2 predominantly recruiting one dynein, its ability to recruit a second was unanticipated. To verify this observation, we applied a mixture of BICD2, dynein tail and dynactin onto grids for negative-stain electron microscopy analysis (Fig. 2c). In agreement with our single-molecule data, 3D classification of adaptor complexes showed that $17 \pm 1\%$ of BICD2 complexes contained two dyneins;

---

${}^{1}$Medical Research Council Laboratory of Molecular Biology, Division of Structural Studies, Francis Crick Avenue, Cambridge CB2 0QH, UK. ${}^{2}$Department of Molecular and Cell Biology, University of California at Berkeley, Berkeley, California 94720, USA. ${}^{3}$Department of Biochemistry, CINVESTAV, México D.F., Mexico. ${}^{4}$Physics Department, University of California at Berkeley, Berkeley, California 94720, USA. †Present address: Department Chemie, Technische Universität München (TUM), Lichtenbergstrasse 4, 85747, Garching, Germany. *These authors contributed equally to this work.

202 | NATURE | VOL 554 | 8 FEBRUARY 2018

© 2018 Macmillan Publishers Limited, part of Springer Nature. All rights reserved.

a
Dynein motor domain
Dynein tail
Adaptor
Dynactin
BICDR1
Pointed end
Filament
b
Adaptor
Shoulder
Pointed end
Filament
c
TDH extra coiled coil

Figure 1 | Dynactin can recruit two dyneins. a, Sub-7 Å cryo-EM maps of TDR and TDH complexes, coloured according to their components. TDB complex (Electron Microscopy Data Bank (EMDB) code: EMD-2862) is included for comparison. b, Surface representation molecular models of BICDR1 and BICD2 on dynactin show the divergent paths of the coiled coils. c, Comparison of HOOK3 and BICD2 on dynactin.

69 ± 4% contained only one dynein, and the rest were ambiguous. The ability of BICD2 to bind two dyneins also agrees with a cryo-electron tomography reconstruction of microtubule-bound DDB²⁰. Negative-stain electron microscopy of BICDR1 and HOOK3 complexes showed that 94 ± 2% and 88 ± 1% of these complexes, respectively, contained two dyneins (Fig. 2c). This suggests an even higher degree of second dynein recruitment than is indicated by our single-molecule data. Our data suggest that the number of dyneins bound to dynactin can be controlled by the identity of the adaptor.

**Two dyneins increase force and speed**

We next investigated how the recruitment of different numbers of motors affects the motile properties of the dynein–dynactin complex. We used an optical trap to measure the stall force of DDB, dynein–dynactin–BICDR1 (hereafter termed DDR) and dynein-dynactin–HOOK3 (hereafter termed DDH) (Fig. 3a, b). Similar to our previous measurements²¹, the stall force of DDB is 3.7 ± 0.2 pN, which is significantly lower (*P* < 0.0001) than the stall force of the plus-end-directed motor kinesin-1 (5.7 ± 0.2 pN)²². By comparison, the stall force of DDR is 6.5 ± 0.3 pN and that of DDH is 4.9 ± 0.2 pN (Fig. 3b), which suggests that recruiting higher numbers of dyneins to dynactin increases force

a
DDB
DDR
DDH
b
Colocalization (%)
40
***
NS
***
10 s
0
10
20
30
Dynein DDB
DDR
DDH
5 µm
c
TDB
TDR
TDH
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynactin
Dynact
RESEARCH ARTICLE

**Figure 3 | Two dyneins increase force and speed of dynein–dynactin.**

a, Traces showing typical stalls of beads driven by single DDB, DDR, DDH or kinesin-1. Experiments were independently repeated four, six, five and four times for DDB, DDR, DDH and kinesin-1, respectively. Arrowheads denote motor detachment from the microtubule after the stall. b, Scatter plots showing stall force distributions ($n = 54$ (DDB), $n = 53$ (DDR), $n = 118$ (DDH), $n = 83$ (kinesin-1)). c, Mean speeds of DDB, DDR and DDH ($n = 3,343$ (DDB), $n = 3,162$ (DDR), $n = 3,744$ (DDH)). d, Schematic depicting experimental design for TMR-tail–Alexa Fluor 647-dynein experiment. e, Dynein-only complexes move significantly faster than tail–dynein complexes ($n = 1,004$ (dynein-only), $n = 939$ (tail–dynein)). In b, c and e, horizontal lines represent mean ± s.e.m., ****$P < 0.0001$ (ANOVA with Tukey’s test for b, c; unpaired two-sided t-test for e).

moving complexes that contained only full-length dynein ('dynein-only') with those that contained one tail and one active dynein ('tail-dynein'). As expected, dynein-only complexes moved at a similar speed ($1.25 \pm 0.04\,\mu$m s${}^{-1}$, Fig. 3e) to DDR ($1.22 \pm 0.05\,\mu$m s${}^{-1}$, Extended Data Fig. 2d). However, tail–dynein complexes moved significantly more slowly ($0.84 \pm 0.03\,\mu$m s${}^{-1}$, $P < 0.0001$, Fig. 3e, Extended Data Fig. 2e) than either DDR or dynein-only complexes. This suggests that the presence of a second dynein increases the velocity of dynein–dynactin complexes.

We propose that the increase in speed on the recruitment of two dyneins is linked to the way in which dynactin recruits them side-by-side (Fig. 1a). This may restrict the inherent sideways and backwards movements of the motor domains${}^{28}$ and cause the complex to take a more direct and faster route along the microtubule. Other dynein regulators, such as LIS1 (otherwise known as PAFAH1B1), have been reported to increase the speed of dynein–dynactin complexes${}^{29,30}$ and could act by increasing motor copy number. For LIS1, however, quantitative fluorescence measurements suggest that this is not the case${}^{29}$. The velocity of BICD2 complexes containing both fluorophores and therefore two dyneins ($1.08 \pm 0.03\,\mu$m s${}^{-1}$, Extended Data Fig. 2f) was significantly faster than the average DDB velocity ($P < 0.0001$), but not as fast as DDR. This suggests that, in addition to recruiting two dyneins, certain adaptors also affect speed through small differences in how they recruit the motors to dynactin.

**The dynein–dynactin–BICDR1 structure**

To investigate how dynactin recruits two dyneins, we collected data sufficient to determine the TDR structure to an overall resolution of 3.5 Å (Extended Data Fig. 3, Extended Data Table 1). To improve the tail density, we performed multiple rounds of particle signal subtraction, focused 3D classification and refinement on regions that moved as rigid blocks, which improved the definition of the blocks at each iteration (Extended Data Fig. 4). This produced a set of 3.4 Å maps that covered the entire length of the dynein tail (Extended Data Table 1).

**Figure 4 | Structure of the dynein HC and architecture of the TDR complex.**

a, Consensus molecular model of one dynein HC, complete with IC and LIC. HC is coloured according to helical bundle number. b, Assembled model of the TDR complex, showing the arrangement of dynein-A (cartoon) and dynein-B (cartoon) on BICDR1 (cartoon) and dynactin (surface). The HC NDD and dynein LC ROBL1 of dynein-B are labelled.

Previous low-resolution structures showed that the dynein tail comprises two heavy chains (HCs), which consist of a series of helical bundles held together by an N-terminal dimerization domain (NDD)${}^{8,9}$. Each HC binds one intermediate chain (IC) and one light intermediate chain (LIC)${}^{7,9,31}$. The IC N-terminal regions are held together by the dynein light chains, ROBL1 (otherwise known as DYNLRB1), LC8 (otherwise known as DYNLL1) and TCTEX1 (also known as DYNLT1)${}^{32,33}$. We used our high-resolution maps to build an atomic model of the dynein tail. We *de novo* traced helical bundles 1–6 of the HC and the WD40 domain of the IC (Fig. 4a, Extended Data Figs 5a, b, 6a, Extended Data Table 1). We also placed helices for part of helical bundle 7 and rebuilt homology models for the LIC${}^{31}$ and ROBL1${}^{32}$ (Fig. 4a, Extended Data Figs 5c, 6b, c). Our structure reveals that the IC WD40 domain makes extensive contacts with HC bundles 4 and 5, and that its central cavity is filled by a loop-helix from bundle 4 (Extended Data Fig. 6a). By contrast, the LIC globular domain interacts only with two helices from bundle 6. The tight binding of the LIC to the HC${}^{34}$ is the result of its N- and C termini, which span out from the globular domain and form an integral part of HC bundles 5 and 7, respectively (Extended Data Figs 5c, 6b).

We assembled and refined a model of the whole TDR complex (Fig. 4b, Extended Data Table 1, Supplementary Video 1) into our 3.5 Å map. We used a previous dynactin structure${}^{8}$ and a model of the BICDR1 coiled-coil region. For each dynein, we fit in two copies each of HC, IC and LIC, one ROBL1 dimer and a new 1.9 Å crystal structure of the human NDD (Extended Data Fig. 6d, e, Extended Data Table 2).

**Structural basis of two-dynein recruitment**

Our TDR structure shows the two dyneins binding to grooves on the surface of dynactin that are formed by its $\beta$-actin subunit and the three actin-related protein 1 (ARP1, also known as ACTR1A) subunits ARP1F, ARP1D and ARP1B (Fig. 4b). The two dynein-A chains,

Figure 5 | Interactions recruiting two dyneins to the TDR complex.

a, Dynein HC (dynein-A, blue; dynein-B, pink) interactions with dynactin subunits (green). Contact residues on dynactin are shown as yellow spheres. For each HC, helix α6 is highlighted (dynein-A, dark blue; dynein-B, dark pink). b, A2 makes extensive interactions with B1. Interaction sites are shown as yellow and red spheres. c, Interactions of A1 and A2, and the first dynein-B chain, B1, all bind in a similar way, by making contacts to both sides of their respective grooves. Although the precise interactions overlap, they are all slightly different from one another (Fig. 5a). The second dynein-B chain, B2, binds between ARP1B and the barbed-end capping protein CAPZβ (CAPZβ is also known as CAPZB) and is rotated by 90° along its long axis, relative to the other dynein chains.

The two dyneins also make extensive interactions with one another. These consist of the IC WD40 domain of A2 binding the HC of B1; direct HC-to-HC interactions; and contacts between the A2 LIC and the HC and IC of B1 (Fig. 5b). These contacts are highly conserved across higher eukaryotes (Extended Data Fig. 7a). They contribute to a cascade of interactions (Supplementary Video 2) between the four dynein chains that include contacts between the IC WD40 domain of each chain and the neighbouring HC (Extended Data Fig. 7b). This network of connections stabilizes the binding of the second dynein and ensures all four HCs are held in a rigid orientation with respect to one another. This is likely to keep the dynein motor domains properly aligned and may be important for the increase in speed when two dyneins are recruited to the dynactin scaffold.

Our structure reveals the key role BICDR1 has in recruiting two dyneins to dynactin. Dynein-A binds the adaptor in three places: its A1 chain uses a single site on helical bundle 2, whereas its A2 chain binds via two sites (Fig. 5c). One of these sites also involves helical bundle 2, and the other uses helical bundle 5. Recruitment of dynein-B depends only on its B1 chain, which also uses sites on bundles 2 and 5. The first of these sites contacts the adaptor coiled coil in a position opposite the binding site of A2 (Fig. 5c). The second site does not directly contact the coiled coil, but instead touches density that packs against it (Fig. 5c, Extended Data Fig. 7c, d). Although the identity of this region

dynein-A (top) and B1 (bottom) with BICDR1. Interaction sites marked in dark blue and red. Extra density from A2 LIC mediates the connection between B1 and BICDR1. d, Negative-stain electron microscopy reconstructions of DDB containing two dyneins (top) or dynein-A only (bottom), sliced to highlight BICD2. Arrows depict alternative positions of BICD2 at the barbed end of dynactin.

is uncertain, there is a weak density connecting it to the LIC, which suggests that it corresponds to the flexible LIC C terminus (Extended Data Fig. 5c, 7d). This region of LIC has previously been shown to interact with BICD2 and HOOK3¹⁹,³¹.

### Adaptor position controls dynein number

All three dynein–dynactin–adaptor complexes recruit dynein-A in a similar way, despite the differences in the positions of the adaptors themselves (Extended Data Fig. 8a). In TDR and TDH, dynein-B can contact the adaptor because the BICDR1 and HOOK3 N termini follow downward paths, stabilized by interactions with CAPZβ. By contrast, in TDB no contact site for dynein-B is available because the adaptor is shifted upward towards the shoulder domain to contact ARP1A (Fig. 1b, Extended Data Fig. 8b).

We investigated which structural changes allow BICD2 to recruit a second dynein (Fig. 2). We combined our negative-stain electron microscopy TDB datasets (Fig. 2c) to determine structures of sufficient quality to resolve the position of the coiled coil. We found that TDB with two dynein tails has BICD2 in a lower position, similar to BICDR1 and HOOK3 and different from its position in single-dynein-bound TDB (Fig. 5d). Our data suggest that a switch in the position in the N terminus of the adaptor is sufficient to account for dynein-B recruitment.

In conclusion, we show that dynactin can act as a natural scaffold to line up two dyneins in close proximity. This arrangement results in a dynein–dynactin complex that moves faster and can produce larger forces when compared with complexes containing a single dynein. Our observations provide a mechanism by which cargo can control the output of the dynein–dynactin machine via the identity of its activating adaptor.

Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.

Received 31 August; accepted 18 December 2017.

1. Roberts, A. J., Kon, T., Knight, P. J., Sutoh, K. & Burgess, S. A. Functions and mechanics of dynein motor proteins. Nat. Rev. Mol. Cell Biol. **14**, 713–726 (2013).
2. Klinman, E. & Holzbaur, E. L. F. Comparative analysis of axonal transport markers in primary mammalian neurons. Methods Cell Biol. **131**, 409–424 (2016).
3. Burkhardt, J. K., Echeverri, C. J., Nilsson, T. & Vallee, R. B. Overexpression of the dynamitin (p50) subunit of the dynactin complex disrupts dynein-dependent maintenance of membrane organelle distribution. J. Cell Biol. **139**, 469–484 (1997).
4. Ben-Yaakov, K. *et al.* Axonal transcription factors signal retrogradely in lesioned peripheral nerve. EMBO J. **31**, 1350–1363 (2012).
5. McKenney, R. J., Huynh, W., Tanenbaum, M. E., Bhabha, G. & Vale, R. D. Activation of cytoplasmic dynein motility by dynactin-cargo adapter complexes. Science **345**, 337–341 (2014).
6. Schlager, M. A., Hoang, H. T., Urnavicius, L., Bullock, S. L. & Carter, A. P. *In vitro* reconstitution of a highly processive recombinant human dynein complex. EMBO J. **33**, 1855–1868 (2014).
7. Chowdhury, S., Ketcham, S. A., Schroer, T. A. & Lander, G. C. Structural organization of the dynein-dynactin complex bound to microtubules. Nat. Struct. Mol. Biol. **22**, 345–347 (2015).
8. Urnavicius, L. *et al.* The structure of the dynactin complex and its interaction with dynein. Science **347**, 1441–1446 (2015).
9. Zhang, K. *et al.* Cryo-EM reveals how human cytoplasmic dynein is auto-inhibited and activated. Cell **169**, 1303–1314.e18 (2017).
10. Redwine, W. B. *et al.* The human cytoplasmic dynein interactome reveals novel activators of motility. eLife **6**, e28257 (2017).
11. Olenick, M. A., Tokito, M., Boczkowska, M., Dominguez, R. & Holzbaur, E. L. Hook adaptors induce unidirectional processive motility by enhancing the dynein-dynactin interaction. J. Biol. Chem. **291**, 18239–18251 (2016).
12. Gama, J. B. *et al.* Molecular mechanism of dynein recruitment to kinetochores by the Rod-Zw10-Zwilch complex and Spindly. J. Cell Biol. **216**, 943–960 (2017).
13. Zheng, W. Probing the energetics of dynactin filament assembly and the binding of cargo adaptor proteins using molecular dynamics simulation and electrostatics-based structural modeling. Biochemistry **56**, 313–323 (2017).
14. Schlager, M. A. *et al.* Bicaudal d family adaptor proteins control the velocity of Dynein-based movements. Cell Reports **8**, 1248–1256 (2014).
15. Cianfrocco, M. A., DeSantis, M. E., Leschziner, A. E. & Reck-Peterson, S. L. Mechanism and regulation of cytoplasmic dynein. Annu. Rev. Cell Dev. Biol. **31**, 83–108 (2015).
16. Schlager, M. A. *et al.* Pericentrosomal targeting of Rab6 secretory vesicles by Bicaudal-D-related protein 1 (BICDR-1) regulates neuritogenesis. EMBO J. **29**, 1637–1651 (2010).
17. Bielska, E. *et al.* Hook is an adapter that coordinates kinesin-3 and dynein cargo attachment on early endosomes. J. Cell Biol. **204**, 989–1007 (2014).
18. Zhang, J., Qiu, R., Arst, H. N., Peñalva, M. A. & Xiang, X. HookA is a novel dynein-early endosome linker critical for cargo movement *in vivo*. J. Cell Biol. **204**, 1009–1026 (2014).
19. Schroeder, C. M. & Vale, R. D. Assembly and activation of dynein-dynactin by the cargo adaptor protein Hook3. J. Cell Biol. **214**, 309–318 (2016).
20. Grotjahn, D. A. *et al.* Cryo-electron tomography reveals that dynactin recruits a team of dyneins for processive motility. Nat. Struct. Mol. Biol. http://dx.doi.org/10.1038/s41594-018-0027-7 (2018).
21. Belyy, V. *et al.* The mammalian dynein-dynactin complex is a strong opponent to kinesin in a tug-of-war competition. Nat. Cell Biol. **18**, 1018–1024 (2016).

22. Svoboda, K., Schmidt, C. F., Schnapp, B. J. & Block, S. M. Direct observation of kinesin stepping by optical trapping interferometry. Nature **365**, 721–727 (1993).
23. Rai, A. K., Rai, A., Ramaiya, A. J., Jha, R. & Mallik, R. Molecular adaptations allow dynein to generate large collective forces inside cells. Cell **152**, 172–182 (2013).
24. Rai, A. *et al.* Dynein clusters into lipid microdomains on phagosomes to drive rapid transport toward lysosomes. Cell **164**, 722–734 (2016).
25. Hendricks, A. G., Holzbaur, E. L. F. & Goldman, Y. E. Force measurements on cargoes in living cells reveal collective dynamics of microtubule motors. Proc. Natl Acad. Sci. USA **109**, 18447–18452 (2012).
26. Driller-Colangelo, A. R., Chau, K. W. L., Morgan, J. M. & Derr, N. D. Cargo rigidity affects the sensitivity of dynein ensembles to individual motor pausing. Cytoskeleton (Hoboken) **73**, 693–702 (2016).
27. Derr, N. D. *et al.* Tug-of-war in motor protein ensembles revealed with a programmable DNA origami scaffold. Science **338**, 662–665 (2012).
28. Reck-Peterson, S. L. *et al.* Single-molecule analysis of dynein processivity and stepping behavior. Cell **126**, 335–348 (2006).
29. Baumbach, J. *et al.* Lissencephaly-1 is a context-dependent regulator of the human dynein complex. eLife **6**, e21768 (2017).
30. Gutierrez, P. A., Ackermann, B. E., Vershinin, M. & McKenney, R. J. Differential effects of the dynein-regulatory factor Lissencephaly-1 on processive dynein-dynactin motility. J. Biol. Chem. **292**, 12245–12255 (2017).
31. Schroeder, C. M., Ostrem, J. M. L., Hertz, N. T. & Vale, R. D. A Ras-like domain in the light intermediate chain bridges the dynein motor to a cargo-binding region. eLife **3**, e03351 (2014).
32. Hall, J., Song, Y., Karplus, P. A. & Barbar, E. The crystal structure of dynein intermediate chain-light chain roadblock complex gives new insights into dynein assembly. J. Biol. Chem. **285**, 22566–22575 (2010).
33. Williams, J. C. *et al.* Structural and thermodynamic characterization of a cytoplasmic dynein light chain-intermediate chain complex. Proc. Natl Acad. Sci. USA **104**, 10028–10033 (2007).
34. King, S. J., Bonilla, M., Rodgers, M. E. & Schroer, T. A. Subunit organization in cytoplasmic dynein subcomplexes. Protein Sci. **11**, 1239–1250 (2002).

**Supplementary Information** is available in the online version of the paper.

**Acknowledgements** We thank S. Scheres, X. Bai, K. Vinothkumar and R. Leiro for cryo-EM advice; S. Chen, G. McMullan, C. Savva, G. Cannone, J. Grimmett and T. Darling for technical support; S. Bullock for SNAPf-dynein (1-1074-GST); M. Yu for crystallography support and the European Synchrotron Radiation Facility (beamline ID29) for data collection; T. Croll for model building; S. Bullock, L. Passmore, S. Lacey and H. Foster for manuscript comments; and G. Lander for discussions. This work was funded by Wellcome Trust (WT100387) and MRC grants (MC_UP_A025_1011) to A.P.C.; and NIH (GM094522) and NSF (MCB-1055017, MCB-1617028) grants to A.Y.

**Author Contributions** L.U. performed all cryo-EM work on TDR, and C.K.L. performed all cryo-EM work on TDH. L.U., C.K.L., M.M.E and A.P.C. performed single-molecule experiments. L.U. performed negative-stain electron microscopy. M.M.E. and A.Y. performed optical trapping. E.M.-R. determined the NDD structure. C.M. made dynein (1-1455). A.P.C., L.U. and C.K.L. built and refined the TDR model and prepared the manuscript.

**Author Information** Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Publisher’s note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Correspondence and requests for materials should be addressed to A.P.C. (cartera@mrc-lmb.cam.ac.uk).

METHODS

Cloning. The following adaptors were codon-optimized for expression in Sf9 cells (Epoch Life Science): mouse BICDR1 (BICDL1), human HOOK3 residues 1–522 and mouse BICD2 residues 1–400. Adaptors were subcloned into pOmniBac- and pACEBac1-derived vectors for expression in Sf9 cells⁶. Tags were added for purification (a His₆-ZZ tag with a TEV protease cleavage site (TEV), or a PreScission protease site (Psc) followed by a 2× strep affinity tag) or protein labelling (GFP or SNAPf (NEB)). The following constructs were generated: pOmniBac-His₆-ZZ-TEV-SNAPf-BICD2¹⁻⁴⁰⁰, pOmniBac-His₆-ZZ-TEV-BICDR1, pOmniBac-His₆-ZZ-TEV-SNAPf-BICDR1, pOmniBac-His₆-ZZ-TEV-BICDR1-SNAPf, pOmniBac-His₆-ZZ-TEV-BICDR1-GFP, pACEBac1-HOOK3¹⁻⁵²²-SNAPf-Psc-2×strep and pACEBac1-HOOK3¹⁻⁵²²-GFP-Psc-2×strep.

We generated a new dynein tail construct containing residues 1–1,455 of the human dynein HC. The fragment of the Sf9-codon-optimized DYNC1H1 gene was cloned into a pACEBac1 vector contain an N-terminal His₆-ZZ-TEV tag and fused to pDyn2 (containing genes for human IC2C, LIC2, TCTEX1, LC8 and ROBL1) as described⁶.

Protein purification. Dynactin was purified from pig brains using the large-scale SP sepharose protocol⁸. Wild-type human dynein and a mutant open dynein⁹ were expressed and purified using baculovirus as described⁶. The two dynein tail constructs (HC¹,⁴⁵⁵ and SNAPf–HC¹⁻¹,⁰⁷⁴-GST) were purified as described⁸.

His₆-ZZ-TEV-tagged adaptor constructs were purified as described⁶. C-terminal Psc-strep-tagged constructs were purified as follows: pellets from 500 ml of Sf9 cell culture were resuspended in 50 ml of lysis buffer (30 mM Hepes-KOH pH 7.2, 50 mM KAc, 2 mM MgAc, 1 mM EGTA, 10% (v/v) glycerol, 1 mM DTT) plus one complete-EDTA protease-inhibitor tablet (Roche) and 1 mM PMSF. Cells were dounced (30–40 strokes) and the lysate clarified in a Ti70 rotor (Beckman Coulter) at 30,000 r.c.f. for 20 min at 4°C before loading onto a 1-ml Streptap HP column (GE Healthcare) pre-equilibrated in lysis buffer. The column was washed with 20 column volumes of lysis buffer and bound protein eluted using 7 column volumes of lysis buffer plus 3 mM β-des thiobiotin. Protein-containing fractions were concentrated to approximately 5 mg ml⁻¹ in 30-kDa cut-off Amicon centrifugal filters (Merck Millipore) and gel-filtered using a Superose-6 10/300 column (GE Healthcare) pre-equilibrated in buffer containing 25 mM Hepes-KOH pH 7.2, 150 mM KCl, 1 mM MgCl₂, 5 mM DTT.

A C-terminal GFP-tagged truncated human kinesin-1 (K560–GFP) was prepared as described³⁵.

N-terminal dimerization domain crystallization. Residues 1–201 of DYNC1H1 were expressed using a modified pRSET(A) plasmid³⁶. Seleno-methionine-labelled NDD was expressed in a SoluBL21 *Escherichia coli* strain as described³⁷. It was purified from 2 l of culture using a 5-ml HisTrap column (GE Healthcare). Fractions were pooled, concentrated in a 30-kDa Amicon and applied to a Superdex200 10/300 gel filtration column (GE Healthcare) equilibrated with buffer containing 50 mM Tris-HCl pH 7.4, 150 mM KAc, 10 mM β-mercapto ethanol, 2 mM MgAc, 1 mM EGTA, 10% (v/v) glycerol and inhibitor tablets (1 tablet: 100 ml, complete-EDTA free). The NDD peak was concentrated to 10 mg ml⁻¹. For protein crystallization, 2 μl of protein was mixed with 2 μl precipitant (100 mM NaAc, pH 5.5, 10% (v/v) glycerol, 50 mM CaAc, 20% PEG 2,000 MME). Crystals were grown at 18°C by hanging drop for 48 h, collected with microloops (Mitegen), dipped into mother liquor containing an extra 15% (v/v) glycerol and flash-frozen in liquid nitrogen. Single-wavelength anomalous diffraction data were collected at ID29 beamline at ESRF, and integrated and scaled by the EDNA auto pipeline³⁸. The structure was solved in PHENIX³⁹, built in COOT⁴⁰ and refined using REFMAC⁴¹.

Cryo-EM of TDR. Cryo-EM grids for TDR (dynein tail (HC¹,⁴⁵⁵)-dynactin-BICDR1) were prepared similarly to TDB⁸, though no cross-linker was used. Protein concentrations were chosen to give densely packed particles (approximately 100 per image). Micrographs were recorded using FEI Titan Krios equipped with Falcon II detector (300 kV, 32 frames, 2-s exposure, 1.34 Å per pixel, 52 e⁻ per Å²) using automated data collection (EPU, FEI). Seven images were collected per hole (26,906 images in total, over 11 separate sessions). Drift correction was performed using MotionCor2⁴² and contrast transfer function (CTF) parameters estimated using CTFFIND3_130307⁴³. Subsequent processing used Relion v.2.0⁴⁴ unless otherwise stated.

Micrographs were first manually examined to remove images with a large amount of contamination, very low number of particles (<15), substantial uncorrected drift, a large astigmatism, extreme defocus values (<1 μm or >6 μm) and/or abnormal Fourier patterns. We selected 23,945 micrographs that had good signal to at least 8 Å. For the first dataset, a small set of particles was manually picked from 8× binned micrographs and subjected to reference-free 2D classification. A selection of 2D-class averages that represented a range of different size and shape particles present in micrographs (not just TDR complexes) was selected, centred using the Relion shift_com function, low-pass filtered to 50 Å and used as references for autopicking all binned micrographs using Relion v.1.4⁴⁵. For optimal particle picking of other datasets, we used 2D classes obtained from multiple sessions. We also used a value of 1.1 for the ‘maximum standard deviation of the background noise’ setting to ensure we picked all good TDR particles with high contrast. This value resulted in substantial levels of contaminants but these were removed by extensive manual screening, as indicated below. Initially, particles with a high ‘autopick figure of merit’ values were screened to remove false positives. Then, the particles were cleaned by several cycles of 2D classification. At each cycle, the only particles that were discarded were those that were obviously dynein tail or contamination. In addition, particles with high ‘log likelihood contribution’ values were manually screened for remaining false positives and particles containing impurities with a very strong signal. After several cycles of 2D classification, particles corresponding to free dynactin in its dominant view were discarded. To do this, we first subjected particles from these classes to an additional round of 2D classification with the ‘image alignment’ setting turned off, in order to recover any TDR particles. Some of the dynactin classes of other-than-dominant view could not be easily distinguished from the TDR complex. Therefore, particles from all other dynactin-like classes were combined with all TDR particles and subjected to 3D refinement with the previous TDB structure⁸ low-pass filtered to 60 Å as a reference. The output translational information from this 3D refinement was used to obtain more accurate coordinates of the particles in individual micrographs (script from R. Fernandez-Leiro). These coordinates were used to extract re-centred particles from 8-binned micrographs, which were manually cleaned as described above. After this cleaning, screened particles were subjected to another round of 3D refinement followed by 3D classification, to improve the separation of dynactin and TDR particles. In all the steps described above (picking, 2D classification, and 3D refinement and classification), the option to ‘ignore CTFs until first peak’ was turned on. Translational information from the 3D classification was used as above to extract re-centred TDR particles from unbinned micrographs. The 3D refinement using the unbinned particles from the first dataset yielded a 6.5 Å resolution map, and the 3D refinement using all 11 datasets yielded a 3.5 Å resolution map.

3D classification (see above) revealed movement of the dynein tails, which resulted in a lower resolution for these parts of the map. As a result, we conducted focused 3D classification and 3D refinement of the dynein tail as described⁴⁶, except that we used multiple rounds of mask optimization. First, we generated overlapping binary masks covering the N- and C-terminal densities of all four dynein chains and used the particle subtraction function in Relion to subtract the density outside these regions from the raw TDR particles. Next, 3D refinement was used to align these ‘subtracted particles’ on the basis of the remaining density. Then we used 3D classification with no alignments to investigate which parts of the structure moved as a rigid block. We then made new masks around the rigid block and repeated all the steps of particle subtraction, 3D refinement and 3D classification. This process was repeated several times to obtain the optimal mask.

In the case of the N-terminal region of the tail, the optimal mask was used for a final round of particle subtraction and 3D refinement, which resulted in a 3.4 Å map. To further improve the densities for the IC WD40 domain, we performed local sub-volume averaging with Chimera 1.10⁴⁷. Similarly, for the C-terminal region of the dynein tail we performed a round of particle subtraction and 3D refinement using the optimal mask. We then used 3D classification with no alignment to identify the most homogenous particles for different local regions. For HC residues 500–927, we performed a 3D classification using the whole of the optimal mask. For ROBL1 or LIC, we performed 3D classification using the local mass around their respective regions. In all cases, selected particles were refined using the whole optimal mask. All density maps were corrected for the modulation transfer function of the detector, and then sharpened by applying a negative *B* factor that was estimated either using automated procedures within Relion or by manually set parameters.

Model building and refinement. Building was performed in COOT and refinement in PHENIX. The dynein HC residues 201–710 from dynein-B1 and the IC from dynein-A2 were *de novo* built and refined into the ‘N-terminal tail’ map guided by maps generated by local sub-volume averaging, with improved density for flexible loops. HC residues 500–927 from dynein-A2 were built and refined into the ‘C-terminal tail’ map. A ‘LIC-mask’ map was used to model secondary structure elements for HC residues 927–1,057 and to rebuild a Phyre2⁴⁸ homology model for human LIC2 (both dynein-A2). A ‘Robl-mask’ map was used to rebuild a homology model for the ROBL1–IC-extended-N termini complex and identify its interactions with the dynein-A2 IC WD40 domain. The separately built components were combined to generate a consensus model for dynein-A2.

The structure of TDR was assembled and real-space refined into an overall TDR map that was not sharpened, and was filtered to 8 Å resolution. We used four copies of the HC–IC–LIC consensus model, two copies of the NDD, a model of dynactin⁸ and a stretch of coiled coil for BICDR1. The combined model was fit

into the 3.5 Å overall map. Sections of dynactin, including the CAPZαβ dimer and the N terminus of subunit p50, were rebuilt. Corrections were made to secondary structure elements in the pointed end and shoulder domains. An approximate model for the BICDR1 coiled coil (residues 105–392) was generated by placing helices into density, and assigning registry on the basis of fitting residues Phe159 and Trp166 into the bulky density in the core of the coiled coil. Regions of the TDR model in weak density were set to zero occupancy for refinement into the overall 3.5 Å map. Segments of the final model were re-refined against the N-terminal and C-terminal maps (Extended Data Table 2).

Cryo-EM of TDH. To assemble the TDH complex, dynein tail, dynactin and HOOK3<sup>1–522</sup>–SNAPf were mixed in a 2:1:20 molar ratio and incubated on ice for 15 min. The sample was cross-linked to increase the amount of complex formed by the addition of 0.0125% (v/v) glutaraldehyde (Sigma-Aldrich) at room temperature for 15 min before quenching with 200 mM Tris pH 7.4 (final concentration). The sample was gel-filtered using a TSKgel G4000SW<sub>XL</sub> (TOSOH Bioscience) equilibrated in 25 mM Hepes-KOH pH 7.2, 150 mM KCl, 1 mM MgCl<sub>2</sub>, 0.1 mM Mg.ATP, 5 mM DTT. The TDH complex was concentrated in a 100-kDa cut-off Amicon at 1,500 r.c.f. to 0.1–0.2 mg ml<sup>−1</sup>. Three microlitres of the TDH sample were applied to freshly glow-discharged Quantifoil R/2 300-square-mesh copper grids covered with a thin carbon support. Samples were incubated on grids on a FEI Vitroblot IV for 45 s and blotted for 3–4.5 s at 100% humidity and 4 °C, and then plunged into liquid ethane.

Micrographs were recorded using automated data collection (FEI EPU) on an FEI Titan Krios with a FEI Falcon III detector in linear mode: 300 kV; 59 frames during a 1.5 s exposure; 1.42 Å per pixel; 45 e<sup>−</sup> per Å<sup>2</sup>; and 5 images per hole. Correction of inter-frame movement for each pixel and dose-weighting was performed using MotionCor2. CTF parameters were estimated using GCTF<sup>49</sup>. A reference set of 2D classes was generated using Relion v.2.0 from a small set of particles picked by the EMAN2<sup>50</sup> Swarm boxing tool. Gautomatch (http://www.mrc-lmb.cam.ac.uk/kzhang/) was used to pick particles from all micrographs (4 × binned) using this reference set. Relion v.2.0 was used for 2D classification to clean the autopicked particles. An 8.2 Å resolution TDB structure<sup>8</sup>, low-pass filtered to 60 Å, was used as an initial model for a first round of 3D refinement. The dataset was cleaned by 3D classification using the output from 3D refinement as a reference. The cleaned particles were re-extracted from unbinned micrographs and used for a final round of 3D refinement, yielding a 6.7 Å map.

Negative stain electron microscopy analysis. The dynein tail–dynactin–adaptor triple complexes were assembled by mixing 1.7 μl of 1 mg ml<sup>−1</sup> dynein tail (HC<sup>1,455</sup>), 1 μl of 1.35 mg ml<sup>−1</sup> dynactin and 2 μl of 1.3 mg ml<sup>−1</sup> cargo adaptor (BICD2<sup>1–400</sup>–GFP, HOOK3<sup>1–522</sup>–SNAPf and BICDR1–GFP). The samples were incubated on ice for 15 min before diluting eightfold for preparation of negative stain grids as described<sup>6</sup>. Two replicate samples were made on separate days. Between 200 and 400 micrographs of each sample were recorded using FEI EPU on a FEI Falcon II direct electron detector, fitted to a FEI F20 electron microscope operated at 200 kV: 1-s exposure; 2.08 Å per pixel; 20 e<sup>−1</sup> per Å<sup>2</sup>; 0.8–1.2 μm underfocus. A small set of particles was picked from 4 × binned micrographs using the EMAN2 Swarm boxing tool. Subsequent processing was done by Relion v.2.0 unless otherwise stated. Particles were extracted and subjected to reference-free 2D classification. From two to five 2D-class averages of triple complex were centred using a shift_com command, low-pass filtered to 50 Å resolution and used as references for automated particle picking within Relion v.1.4 of all 4 × binned micrographs. Autopicked particles were extracted, split in half and subjected to 2D classification as above. Ten 2D-class averages representing different particle orientations were selected and used for another round of autopicking. This third round of auto-picking was used to obtain the optimal particle selection (fewest missed particles) and best centring.

The resulting particles were subjected to three rounds of 2D classification to identify good complex particles (particle numbers for TDB: 6,382, 7,648, 5,534, and 6,163; TDH: 3,430 and 3,958; TDR: 1,713, 1,861, 5,782 and 5,388). For the best 3D classification, these sets of particles were first subjected to 3D refinement using the cryo-EM structure of dynactin (EMDB code: EMD-2856), low-pass filtered to 60 Å, as a reference. 3D classification was carried out using the map from the 3D refinement, as well as the orientation and rotational assignment for the particles. 3D classification was carried out by setting the regularization parameter *T* to 2 and gradually adjusting image alignment sampling: 10 iterations with 15° angular sampling interval, offset search range set to 8 pixels and step to 2 pixels; 25 iterations with an angular sampling interval of 7.5°, offset search range set to 5 pixels and step to 1 pixel. The gradual reduction in sampling yielded the best classification. It was followed by 15 iterations with an angular sampling interval of 3.7°, offset search range set to 3 pixels and step to 0.5 pixels, which yielded 3D classes with density maps of sufficient quality to identify the number of dynein tails bound to dynactin. The fraction of complexes containing one or two dyneins was calculated from the number of particles assigned to each class.

To determine the path of BICD2, all particles of the complexes from both TDB (BICD2<sup>1–400</sup>–GFP) datasets were combined (42,823 particles). Binned particles from these complexes were subjected to 3D refinement as described above. These coordinate files were used to re-centre—as described above—and extract particles from unbinned micrographs, with CTF parameters that were determined using CTFFIND3_130307. Extracted particles were subjected to 3D refinement, followed by 3D classification. In both steps, the CTF correction was set to ignore CTFs until the first peak. Particles from 3D classes with dynein-A only, or with dynein-A and dynein-B, were separated and processed separately. Each set of particles was subjected to another round of 3D refinement and subsequent 3D classification (using 25 iterations: 7.5° angular sampling interval, offset search range of 5 pixels and step of 1 pixel) to obtain the best particles for each complex (13,278 and 14,070 particles for TDB with dynein-A only and with two dyneins, respectively). These were used in a final round of 3D refinement. Molecular models of dynactin, cargo adaptors and dynein tails were fitted into the density maps and used to colour different segments in Chimera.

Single molecule assays. SNAPf–dynein or SNAPf–tail complexes were labelled with TMR-Star or Alexa Fluor 647 (NEB) and purified separately, and the percentage labelling was quantified as previously described<sup>6</sup>. Single-molecule assays were conducted as previously described<sup>6</sup>, with slight modifications in order to optimize the number of moving complexes. Under these optimized conditions, DDB complexes moved faster than observed in previous publications<sup>5,6,10,11</sup>. All adaptor complexes were measured under identical conditions. The percentage of processive events was 56% for DDB, 76% for DDH and 75% for DDR.

Microtubules were made by mixing 26 μl of 5.2 mg ml<sup>−1</sup> unlabelled pig tubulin, 5 μl of 2 mg ml<sup>−1</sup> HiLyte Fluor 488 tubulin and 10 μl of 2 mg ml<sup>−1</sup> biotin tubulin (Cytoskeleton) in BRB80 buffer (80 mM PIPES pH 6.8, 1 mM MgCl<sub>2</sub>, 1 mM EGTA, 1 mM DTT). The mixture was incubated on ice for 5 min before adding 41 μl of polymerization buffer (2 × BRB80 buffer plus 20% (v/v) DMSO and 2 mM Mg.GTP). Microtubules were polymerized at 37°C for 30–60 min. The sample was diluted with 200 μl of microtubule buffer (BRB80 plus 40 μM paclitaxel). Excess tubulin was removed by pelleting (20,238 r.c.f., 8.5 min; at room temperature). The microtubule pellet was washed with 200 μl of microtubule buffer and re-pelleted as above. The microtubule pellet was re-suspended in 200 μl of microtubule buffer and stored at room temperature, and covered from light for at least half a day (to a maximum of three days) before use. Microtubules were visualized (see below) and if the density was too low, or the free tubulin concentration was too high, they were re-pelleted and re-suspended in a smaller volume of microtubule buffer.

Motility chambers were made by applying two strips of double sided-tape (Tesa) approximately 8–10 mm apart on a glass slide and placing a cleaned coverslip<sup>6</sup> on top. The glass surface was functionalized with 0.4 mg ml<sup>−1</sup> biotinylated poly(ε-lysine)-g-poly(ethylene-glycol) (SuSoS AG). The chamber was immediately washed with 40 μl of assay buffer (30 mM Hepes-KOH pH 7.2, 5 mM MgSO<sub>4</sub>, 1 mM EGTA, 1 mM DTT). Subsequently, 10 μl of 1 mg ml<sup>−1</sup> streptavidin (NEB) was flowed through and immediately washed with 40 μl of assay buffer. The chamber was then incubated with freshly diluted microtubules (typically 3 μl of microtubules and 10 μl of assay buffer). Microtubules were immediately washed out with assay buffer, followed by assay buffer supplemented with 1.25 mg ml<sup>−1</sup> α-casein (Sigma-Aldrich).

Dynein–dynactin–cargo-adaptor complexes were prepared by mixing 1 μl of 1 mg ml<sup>−1</sup> fluorescently labelled dynein, 1 μl of 1.35 mg ml<sup>−1</sup> dynactin and 2 μl of 1.3 mg ml<sup>−1</sup> cargo adaptor (SNAPf–BICD2<sup>1–400</sup>, HOOK3<sup>1–522</sup>–SNAPf and SNAPf–BICDR1). For experiments with SNAPf–tail, an additional 1 μl of 1 mg ml<sup>−1</sup> TMR-labelled tail was added. The complex was incubated on ice for 10–15 min, and then diluted with assay buffer to a final volume of 10 μl. One microlitre of complex solution was added to 19 μl of assay buffer containing 1.25 mg ml<sup>−1</sup> α-casein, an oxygen scavenging system (0.2 mg ml<sup>−1</sup> catalase (Calbiochem) and 1.5 mg ml<sup>−1</sup> glucose oxidase (Sigma-Aldrich)), 0.45% (w/v) glucose, 1% BME, 25 mM KCl and 5 mM Mg.ATP. Taxol was omitted from this buffer to reduce the non-specific background and non-moving events. The motility mix was flowed into the chamber, and washed with assay buffer supplemented with 1.25 mg ml<sup>−1</sup> α-casein for a second time. The sample was analysed immediately at 23 ± 1 °C with a total internal reflection fluorescence microscope, as previously described<sup>6</sup>. Colocalization and velocity were determined from the same datasets (4, 8, 9 and 7 chambers overall, for dynein-only, DDB, DDH and DDR, respectively). Data were collected on two different days. Tail and full-length dynein speed data were collected from three chambers.

Data were analysed using ImageJ<sup>51</sup>. Tif stacks were Z-projected to identify the paths of microtubules, segmented lines drawn along them and kymographs generated using the reslice function. Processive movements were defined as previously described<sup>6</sup>. Velocity was calculating using a pixel size of 105 nm and a frame rate of 236 ms per frame.

Colocalization data were collected using a DV2 beam splitter (Photometrics), which projected each channel (TMR and Alexa Fluor 647) onto a different half
of the image. Movie tif-stacks were split and kymographs generated for each channel. Kymographs were overlaid using the ‘Colour:Merge’ function to generate a composite image. The kymographs were manually scored for processive events that showed colocalization, followed by those events that only appeared in individual channels. Colocalization in the dynein-alone control chambers was determined for all microtubule binding events longer than 2 frames.

The fraction of complexes containing two dyneins, \( d \), was calculated from the fraction of total events with signal for TMR-only (\( R_{\text{obs}} \)), Alexa Fluor 647-only (\( G_{\text{obs}} \)) and colocalized signal (\( Y_{\text{obs}} \)). Colocalized events represent dynein–dynactin–adaptor complexes with two dyneins, and single-coloured events represent a mixture of both one- and two-dynein complexes. We can express this using the following three equations:

\[
R_{\text{obs}} = (s \times r) + (d \times r^2)
\]

\[
G_{\text{obs}} = (s \times r) + (d \times g^2)
\]

\[
Y_{\text{obs}} = d \times 2(r \times g)
\]

Where \( s \) is the fraction of complexes that contain one dynein, \( r \) is the fraction of TMR-labelled dynein and \( g \) is the fraction of Alexa Fluor 647-labelled dynein flowed into the chamber. These equations hold at high labelling efficiency—our dynein was >94% labelled for a dynein monomer, or >99.64% per dimer—where dynein can be labelled by either TMR or Alexa Fluor 647 (\( r + g = 1 \)). We can therefore solve these equations for \( d \) without knowing \( r \) or \( g \).

**Stall force measurements.** Eight-hundred-nanometre carboxy latex beads (Life Technologies) were functionalized with anti-GFP antibodies^{21}. Dynein, dynactin and cargo adaptor (BICD2^{1−400}–GFP, HOOK3^{1−522}–GFP and BICDR1–GFP) were mixed at a 1:5:20 molar ratio and incubated for 10 min on ice in dynein motility buffer (DMB: 30 mM HEPES pH 7.0, 5 mM MgSO₄, 1 mM EGTA) supplemented with 1 mM tris(2-carboxyethyl)phosphine (TCEP) and 1 mg ml⁻¹ BSA. The diluted mixture was incubated with the anti-GFP-beads for 10 min on ice. Cy5-labelled axonemes^{21} were introduced to the sample flow chamber, which was washed with 40 μl of DMB plus 1 mM TCEP and 500 μg ml⁻¹ casein. The protein–bead mixture was introduced to the chamber in imaging buffer (DMB with 1 mM TCEP, 500 μg ml⁻¹ casein, 2.5 mM protocatechuic acid, 35 μg ml⁻¹ protocatechuate-3,4-dioxygenase (PCD), 2 mM Mg.ATP). Kinesin was diluted in BRB80 supplemented with 1.5 mg ml⁻¹ casein and 2 mM DTT and mixed with anti-GFP beads for 10 min on ice. Kinesin-coated beads were introduced to the sample chamber in motility imaging solution (BRB80 supplemented with 2 mM DTT and 1.5 mg ml⁻¹ casein, 2.5 mM protocatechuic acid, 35 μg ml⁻¹ PCD, 2 mM Mg.ATP). For each experiment, the protein concentration was adjusted until less than 30% of the tested beads exhibited any binding and motility when brought in contact with axonemes. This ensured that 95% of the beads were driven by a single processive motor complex.

Trapping experiments were performed on a custom-built fully automated optical trap microscope setup^{21}. To generate stall force histograms, position data from trap recordings (5 kHz) were down-sampled to 500 Hz and then manually screened and selected. To qualify as a stall event, the position trace had to reach a plateau and remain stationary for at least 100 ms before full release from the microtubule (defined as a rapid greater-than-2 ms jump of at least 50 nm). For DDB, we measured \( n = 54 \) stalls from 14 beads in 4 independent experiments; for DDH, \( n = 118 \) stalls from 25 beads in 6 independent experiments; for DDR, \( n = 53 \) stalls from 17 beads in 5 independent experiments; and for kinesin, \( n = 83 \) stalls from 18 beads in 4 independent experiments.

**Statistical analysis.** Statistical analysis was performed using GraphPad Prism 7 (GraphPad). The statistical significance of difference in mean values was calculated

using one-way ANOVA with Tukey’s multiple comparisons test, or an unpaired \( t \)-test, as indicated for each experiment. No statistical methods were used to predetermine sample size. Velocity and colocalization data were randomized before analysis; other experiments were not randomized, and the investigators were not blinded to allocation during experiments and outcome assessment.

**Code availability.** Code used is available from A.P.C.

**Data availability.** Cryo-EM maps have been deposited in the EMDB under accession codes EMD-4168 (whole TDR complex), EMD-4169 (N-terminal tail), EMD-4170 (C-terminal tail/HCC terminus), EMD-4171 (LIC region), EMD-4172 (ROBL1 region) and EMD-4177 (TDH complex). Coordinates are available from the RCSB Protein Data Bank under accession codes 6F1T (whole TDR complex), 6F1U (N-terminal tail), 6F1V (C-terminal tail), 6F1Y (LIC region), 6F1Z (ROBL1 region), 6F38 (TDH complex) and 6F3A (TDB complex). Raw data are available from A.P.C.

35. Tomishige, M. & Vale, R. D. Controlling kinesin by reversible disulfide cross-linking. Identifying the motility-producing conformational change. *J. Cell Biol.* **151**, 1081–1092 (2000).

36. Dodd, R. B. *et al.* Solution structure of the Kaposi’s sarcoma-associated herpesvirus K3 N-terminal domain reveals a novel E2-binding C4HC3-type RING domain. *J. Biol. Chem.* **279**, 53840–53847 (2004).

37. Studier, F. W. Protein production by auto-induction in high density shaking cultures. *Protein Expr. Purif.* **41**, 207–234 (2005).

38. Incardona, M. F. *et al.* EDNA: a framework for plugin-based applications applied to X-ray experiment online data analysis. *J. Synchrotron Radiat.* **16**, 872–879 (2009).

39. Adams, P. D. *et al.* PHENIX: a comprehensive Python-based system for macromolecular structure solution. *Acta Crystallogr. D* **66**, 213–221 (2010).

40. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. *Acta Crystallogr. D* **60**, 2126–2132 (2004).

41. Murshudov, G. N. *et al.* REFMAC5 for the refinement of macromolecular crystal structures. *Acta Crystallogr. D* **67**, 355–367 (2011).

42. Zheng, S. Q. *et al.* MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. *Nat. Methods* **14**, 331–332 (2017).

43. Mindell, J. A. & Grigorieff, N. Accurate determination of local defocus and specimen tilt in electron microscopy. *J. Struct. Biol.* **142**, 334–347 (2003).

44. Kimanius, D., Forsberg, B. O., Scheres, S. H. & Lindahl, E. Accelerated cryo-EM structure determination with parallelisation using GPUs in RELION-2. *eLife* **5**, e18722 (2016).

45. Bharat, T. A. M., Russo, C. J., Löwe, J., Passmore, L. A. & Scheres, S. H. W. Advances in single-particle electron cryomicroscopy structure determination applied to sub-tomogram averaging. *Structure* **23**, 1743–1753 (2015).

46. Nguyen, T. H. D. *et al.* Cryo-EM structure of the yeast U4/U6.U5 tri-snRNP at 3.7 Å resolution. *Nature* **530**, 298–302 (2016).

47. Pettersen, E. F. *et al.* UCSF Chimera—a visualization system for exploratory research and analysis. *J. Comput. Chem.* **25**, 1605–1612 (2004).

48. Kelley, L. A., Mezulis, S., Yates, C. M., Wass, M. N. & Sternberg, M. J. The Phyre2 web portal for protein modeling, prediction and analysis. *Nat. Protocols* **10**, 845–858 (2015).

49. Zhang, K. Gctf: Real-time CTF determination and correction. *J. Struct. Biol.* **193**, 1–12 (2016).

50. Tang, G. *et al.* EMAN2: an extensible image processing suite for electron microscopy. *J. Struct. Biol.* **157**, 38–46 (2007).

51. Schneider, C. A., Rasband, W. S. & Eliceiri, K. W. NIH Image to ImageJ: 25 years of image analysis. *Nat. Methods* **9**, 671–675 (2012).

52. Kucukelbir, A., Sigworth, F. J. & Tagare, H. D. Quantifying the local resolution of cryo-EM density maps. *Nat. Methods* **11**, 63–65 (2014).

53. Drozdetskiy, A., Cole, C., Procter, J. & Barton, G. J. JPred4: a protein secondary structure prediction server. *Nucleic Acids Res.* **43**, W389–W394 (2015).

54. Robert, X. & Gouet, P. Deciphering key features in protein structures with the new ENDscript server. *Nucleic Acids Res.* **42**, W320–W324 (2014).

© 2018 Macmillan Publishers Limited, part of Springer Nature. All rights reserved.
RESEARCH ARTICLE

Extended Data Figure 1 | Single particle cryo-EM analysis of TDR and TDH.

a, Cryo-EM reconstruction of the TDR complex analysed by ResMap⁵² and showing resolution distribution from 4 to 12 Å.

b, The gold-standard Fourier shell correlation (FSC) curve of the 6.5 Å TDR map.

c, Cryo-EM reconstruction of the TDH complex, showing resolution distribution from 4 to 12 Å.

d, The gold-standard FSC curve of the 6.7 Å TDH map.

e, Cryo-EM density for TDR low-pass filtered to 6.7 Å resolution (coloured according to cartoon) and to 20 Å (transparent outline). Density at the N terminus of BICDR1 is boxed.

f, Cryo-EM density for TDH low-pass filtered to 6.7 Å (coloured according to cartoon) and to 20 Å (transparent outline) showing the putative Hook domain, an extension of the HOOK3 coiled coil ending in extra density near dynein-B (dashed box).

a
- DDB 5.2 µm R²=0.995
- DDH 10.5 µm R²=0.994
- DDR 10.6 µm R²=0.996

b
- DDB 0.86 ± 0.04 µm/s
- DDR 1.35 ± 0.04 µm/s

c
- DDB 0.86 ± 0.04 µm/s
- DDH 1.23 ± 0.04 µm/s

d
- mtDDB 0.64 ± 0.07 µm/s
- mtDDR 1.22 ± 0.05 µm/s

e
- tail-dynein 0.84 ± 0.04 µm/s
- dynein-only 1.25 ± 0.04 µm/s

f
- DDB 0.86 ± 0.04 µm/s
- DDB colocalizers 1.08 ± 0.05 µm/s

Extended Data Figure 2 | Single-molecule assay speed distributions.
a, A one-cumulative frequency distribution plot showing run-lengths of DDB, DDR and DDH, with fit to a one-phase exponential decay. The decay constant (run length) and R² value (least squares regression) of the fit are shown. We measured 785, 677 and 684 events for DDB, DDH and DDR, respectively, from microtubules of at least 20 µm in length from three chambers. b–f, Distribution of mean velocities of processive (unidirectional, minus-end-directed) events for DDB (n = 3,343) and DDR (n = 3,162) (b); DDB and DDH (n = 3,744) (c); active mutant dynein in complex with dynactin and BICD2 (mtDDB, n = 905) or BICDR1 (mtDDR, n = 1,183) (d); the colocalized mtDDR complexes containing both TMR-dynein tail and Alexa Fluor 647-full-length dynein (tail-dynein, n = 939) or Alexa Fluor 647-only complexes containing only full-length dynein (dynein-only, n = 1,004) (e); and all DDB complexes and complexes with both fluorophores, and hence two dyneins (colocalizers, n = 660) (f). Mean ± s.e.m. values were estimated by fitting the histograms to a Gaussian distribution (dashed lines).

RESEARCH ARTICLE

Extended Data Figure 3 | Single-particle cryo-EM analysis of TDR complex at 3.5 Å resolution. **a**, Micrograph of the TDR complex (representative of 26,906 micrographs). **b**, Typical 2D-class averages of TDR in different orientations. **c**, The overall density map of TDR was analysed by ResMap, showing a resolution distribution from 3 to 8 Å. **d**, The gold-standard FSC curve of the overall TDR map. **e**, Mesh representation of 3.4 Å resolution density map of α helices from dynein-B1 obtained by focused 3D classification and refinement. **f**, Sample density obtained by local sub-volume averaging, showing β strands from IC WD40.

© 2018 Macmillan Publishers Limited, part of Springer Nature. All rights reserved.

3.5 Å map of TDR complex  
205,611 selected particles  

Mask design  

N-term dynein tail  
Signal subtraction of  
projections from raw particles  
↓  
3D refinement  
↓  
3D classification  
with no alignments  

Optimal  
masks for  
N-terminus  
Aligning particles  
based on  
tail N-terminus  

Particles subtraction  
and 3D refinement  

3.4 Å map  
Building:  
HC 200-700aa  

Local sub-volume averaging  

Building:  
HC  
201-460aa  

Building:  
IC WD 40  
238-595aa  

C-term dynein tail  
Signal subtraction of  
projections from raw particles  
↓  
3D refinement  
↓  
3D classification  
with no alignments  

Optimal  
masks for  
C-terminus  
Aligning particles  
based on  
tail C-terminus  

Particles subtraction  
and 3D refinement  

HC C-term  
Robl  
LIC  

3D classification  
with no alignments  
↓  
3D refinement  

3.4 Å map  
86,030 final particles  

Mask on Robl  
3D classification  
with no alignments  
↓  
3D refinement  

3.4 Å map  
77,136 final particles  

Mask on LIC  
3D classification  
with no alignments  
↓  
3D refinement  

3.4 Å map  
113,987 final particles  

Building:  
HC 500-927aa  

Multiple cycles  
indicated best  
rigid body regions  
for optimal  
mask design  

Extended Data Figure 4 | Cryo-EM data procedures of TDR. Focused 3D classification and refinement procedures used in this study to improve density maps for dynein tails.

a
HC 1 MSEPGGGGGEDGSAGLEVS A V Q N VADSVLQKHLRKLVP L L LEDGGEAPA A LEALEEKSALEQMRKFLS 70
α1 α2 α3
90 110 130
71 DPQVHTVLVERSTLKEDVGDEGEEEKEFISYNINIDIHYGVKSNSLAFIKRTPVIDADKPVSSQLRVTLT 140
β1 β2 β3 β4
160 180 200
141 SEDSPYETLHSFISNAVAPFFKSYIRESGKADRDGDKMAPSVEKKIAELEMGLLHLQQNIEIPEISLPIH 210
α4 α5
230 250 270
211 PMITNVAKQCYERGEKPKVTDFGDKVEDPTFLNQLQSGVNWRIREIQKVTKLDRDPA SGTALOEISFWLN 280
α6 α7 α10b
300 320 340
281 LERALYRIQEKRSEPEVLLTLDILKHGKRFHATVSFDTDTGLKQALETVNDYNPLMKDFPLNDLLSATEL 350
α8 α9 α10a
370 390 410
351 DKIRQALVAIFTHLRKIRNTKYPIQRALRLVEAISRDLSSQLLKVLGTRKLMHVAYEEFEKVMVACFEVF 420
α11 α12 α17
440 460 480
421 QTWDDEYEKLQVLLRDIVKRKREENLKMVWRINPAHRKLQARLDQMRKFRRQHEQLRAVIVRVLRPQVTA 490
α13 α14 α15a α15b α16
510 530 550
491 VAQQNGGEVPEPQDMKVAEVLFDAADANAI EEVNLAYENVKEVDGLDSKEGTEAWEAAMKYDERIDRV 560
α18 α19a
650 670 690
631 QACKMSHVRDLPPVSGSIIWAKQIDRQLTAYMKRVEDVLGKGWENHVEGQKLKQDGDSFRMKLNTQEIFD 700
α19b α20
720 740 760
701 DWARKVQQRNLGVSGRIFTIESTVRGRGTGNVLKLKVNFLPEIITLSKEVRNLKWLGFRVPLAINKAHQ 770
α21 α22
790 810 830
771 ANQLYPFAISLIESVRTYERTCEKVEERNTISLLVAGLKKEVQALIAEGIALVWESYKLDPYVQRLAETV 840
α23 α24 α25
860 880 900
841 FNFQEKVDDLLIIEEKIDLEVRSELECTMYDHKTFEILNRVQKAVDDLNLHSYSNLPIWVNKLDMEIERI 910
α26 α27
930 950 970
911 LGVRLQAGLRAWTQVLLGQAEDKAEVDMTDAPQVSHKPGGEPKIKNVVHELRITNQVIYLNPPIIECRY 980
α28a α28b α29
1000 1020 1040
981 KLYQEMFAWKMVVLSLPRIQSQRYQGVHYELTEEEKFYRNALTRMPDPVALEESYSAVMGIVSEVEQY 1050
1070 1090 1110
1051 VKVWLQYQCLWDMQAENIYNRLGEDLNKWOALLVQIRKARGTFDNAETKKEFGPVVIDYGKVSKVNLKY 1120
1140 1160 1180
1121 DSWHKEVLSKEGQMLGSNTEFHSQISKSRQLEQHSVDTASTSDAVTFITYVQSLKKIKQFEKQVELY 1190
1210 1230 1250
1191 RNGQRILLEKQRFQFPSWLYIDNIEGEWGAFNDIMRRKDSAIOQQVANLOMKIVQEDRAVESRTTDL LTD 1260
1280 1300 1320
1261 WEKTKPVTGNLRPEEALQALTIEGKFGRLKDDREKCAKAEALELDTGLLSGSEERVQVALEELQDLK 1330
1350 1370 1390
1331 GVWSELSKVWEQIDQMKEQFWSVQPRKLRQNLDALLNQLKSFPARLRQYASYEFVQRLLLKGYMKINMLV 1400
1420 1440
1401 IELKSEALKDRHWKQLMKRLHVNWVVSELTLGQIWVDVLQKNEAIVKDVLVAQG 1455

Extended Data Figure 5 | Secondary structure diagram of dynein HC.
a, Secondary structure elements of dynein HC are matched against the primary sequence showing the NDD (purple) and the dynein helical bundles (blue; cyan; green; pale yellow; orange; red; pink).
b, Secondary structure elements of IC. Extended N-terminal regions are coloured purple and other elements are coloured according to the blade of

b
IC 1 MSDKSELKALERKKQRLAQIREEKKRKEEERKKETDQKKEAVAPVQEESDLEKKRREAEALLQSMGLT 70
α1 α2
90 110 130
71 PESPIVPPPMSPPSSKSVSTPSEAGSQDSGDGAVGSRRGPIKLGMAKITQVDFPPREIVTYTKETQTPVA 140
β1 β2
160 180 200
141 QPKEDEEEDDVVAKPPIEPEEEKTLKKDEENDSKAPPHELTEEKQQILHSEEFLSFFDHSTRIVERA 210
α3 α4 α5
230 250 270
211 LSEQINIFFDYSGRDLDEKEGEIQAGAKLSLN RQFFDERWSKHRVVSCLDWSSQYPELLVASYNNNEDAP 280
β3 β4 β5 β6 β7
300 320 340
281 HEPDGVALWNMKYKKTTPEYVFHCQSAVMSATFAKFHPNLVVGGTYSGQIVLWDNRSNKRTPVQRTPLS 350
β8 β9 β10 β11 β12 β13
370 390 410
351 AAAHTHPVYCVNVVGTQNAHNLISTDGTKICSWSLIMLSHPQDSMELVHKQSKAVAVTSMSFPVGDVNN 420
β14 β15 β16 β17 β18
440 460 480
421 FVVGSEEGSVYTACRHGSKAGISEMFEGHQGPITGIHCHA AVGAVDESHLFVTSSFDWTVKLWSTKNNKP 490
β19 β20 β21 β22 β23 β24
510 530 550
491 LYSFEDNAGYVYDVMWSPTHPALFACVDGMGRLDLWNLLNNDTEVPTASISVEGNPALNRVRWTHSGREIA 560
β25 β26 β27 β28 β29 β30 β31
580
561 VGDSEGQIVIYDVGGEQIAVPNDEWAREGRTLAEINANRADAEEAAATRIPA 612
β32 α6 β33 α7

C
LIC 1 MAPVGVEKKLLLGPNGPAVAAAGDLTSEEEEGQSLWSSILSEVSTRARSKLPSGKNILVFGEDGSGKTTL 70
α1 β1 α2
90 110 130
71 MTKLQGAEHGKKGRGLEYYLYLSVHDEDRDDHTRCNVWILDGDLYHKGLLKFAVSAESLPETLVIFVADMS 140
β2 α3 α4 β4
160 180 200
141 RPWTVMESLQKWASVLR E HIDKMKIPPEKMRELERKFVKDFQDYMEEP EEGCQGSPQRRGPLTSGSDEENV 210
α5 α6 α7
230 250 270
211 ALPLGDNVLTHNLGIPVLVCTKCDAVSVELEKEHDYRDEHLDFIQSHLRRFCLQYGAALIYTSVKEEKNL 280
β5 α8 α9 β6
300 320 340
281 DLLYKYIVHKTYGHFETTPALVVEKDAVFI PAGWDNEKKTAILHENFTTVKPEDAYEDFKPPVRKL VH 350
α10 β7 α11 α12
370 390 410
351 DKELAAEDEQVFLMKQQSLLAKQPATPTRASESPARGPSGPSRTQGRGGBPASVPSSSPGTSVKKPDNIK 420
α13
440 460 480
421 NNAASEGVLASEFNSLLSKKTGSPSGSPGAGGVQSTAKKSGQKTVLSNVQEELDRMTRKPDSMVTNSSTEN 490
α14 α15
491 EA
492

the WD40 domain to which they belong, except sheet β5, which associates with β30–32. c, Secondary structure elements of LIC, showing the globular domain helices and sheets (blue) and the two helices that pack against the HC (red). Jpred⁵³ secondary structure predictions of features not seen in the electron microscopy map are shown in grey.

Extended Data Figure 6 | Interactions between dynein subunits. **a**, The dynein HC (yellow) interacts with the IC WD40 domain (blue) using bundles 4 and 5, with a helical segment (red cartoon) sitting in the WD40 central cavity. Dynein-A2 is shown. Interacting residues are shown as sticks (bottom panel), with HC residues in red and IC residues in green. **b**, Density map and model showing how the LIC (density and cartoon, blue) N- and C-terminal regions extend from the globular domain and pack against the HC (density, coloured by bundle number). Dynein-A2 is shown. **c**, ROBL1 (cartoon, light and dark pink) makes contacts with the IC N-terminal helices (cartoon, light and dark blue), which mediate the interaction between ROBL1 and the IC WD40 (surface). **d**, Representative density from the 1.9 Å resolution NDD crystal structure. **e**, Cartoon model of the NDD showing one chain in rainbow spectrum.

a

| HC | 600 | 625 | 675 | 690 | 725 | 890 | 1020 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| H. sapiens | RP**HIRGAI**REY | KFKVQYP | WENHVE | GDSFRMKLN | RVRRGTGN | LNLHSY | FYRNALTRM |
| M. musculus | RP**HIRGAI**REY | KFKVQYP | WENHVE | GDSFRMKLN | RVRRGTGN | LNLHSY | FYRNALTRM |
| X. tropicalis | RP**HIRGAI**REY | KFKVQYP | WENHVE | GDSFRMKLN | RVRRGTGN | LNLHSY | FYRNALTRM |
| D. rerio | RP**HIRGAI**REY | KFKVQYP | WENHVE | GDSFRMKLN | RARGRTGN | LNLHSY | FYRNALTRM |
| D. melanogaster | RP**HIRGAI**REY | KFKVQYP | WETHIE | GDSFRAKLS | RSRIGRGN | LSLRQY | TYRNLLTKL |
| C. elegans | RP**IRGAI**YEY | RFTKAR. | WENVD | GDNFKVKLN | QVDGRM.. | LSLGNY | TYHNILNVN |
| U. maydis | RP**KVRGAV**Q**EY** | KFKAQYR | WELEYAE | SETFRRKLD | RAA...GN | LNLEGY | DYASLLTRF |
| A. nidulans | RP**KIRGAIG**EY | RFKQQYG | WHLTE | SNLFRKKLD | RAA...GN | LNLENY | CFADLPQH |
| S. cerevisiae | CP**RIKVKVLE**N | GLELL.. | WNDTLE | IVQLRKETN | .....NED | LFTKGL | TFNSLVVIKL |

| IC | 280 | 316 | 366 | 385 |
| --- | --- | --- | --- | --- |
| H. sapiens | APHEP | KFHPN | GTQNAH | SLDMLS**H** |
| M. musculus | APHEP | KFHPN | GTQNAH | SLDMLS**H** |
| X. tropicalis | APHEP | KFHPN | GTQNAH | SLDMLSQ |
| D. rerio | APHEP | QFHPN | GTQNAH | SLDMLST |
| D. melanogaster | SPNEP | KFNP**N** | GTQNAH | SLDMLSQ |
| C. elegans | IPNEP | RFHAH | GTKNAH | NVDNL**TQ** |
| U. maydis | NEDAP | PFHPN | GSANAN | MLDMLAR |
| A. nidulans | APHEP | PFHPN | GTQNAH | TVDMLSQ |
| S. cerevisiae | NASGR | RYHPE | SVQDQE | DLTVLRK |

| LIC | 275 | 305 |
| --- | --- | --- |
| H. sapiens | SVKEE | VVEKD**A**V |
| M. musculus | SVKEE | VVEKD**A**V |
| X. tropicalis | SVKEE | VVEKD**A**V |
| D. rerio | SVKEE | VVEKD**A**V |
| D. melanogaster | SVKED | VVEKD**A**V |
| C. elegans | SAKE**T** | VIERESI |
| U. maydis | SSVRA | TTDRDTL |
| A. nidulans | KPFHA | FIERDKI |
| S. cerevisiae | CEDHT | MVKRSEI |


b

- IC-A1
- HC-A2
- IC-A2
- HC-B1
- IC-B1
- HC-B2

c

- Higher contour
- Dynactin
- BICDR1
- Extra density
- HC-A2
- HC-B1
- LIC-A2
- LIC-B1

d

- Lower contour
- Dynactin
- Connection to LIC-A2
- C-terminus
- HC-A2
- LIC-A2
- LIC-B1

Extended Data Figure 7 | Dynein–dynein contacts and interactions at the BICDR1 N terminus.

a, Conservation diagram showing sequence similarity between A2 and B1 interacting residues. Residues coloured white with red background are completely conserved, whereas residues coloured red show sequence similarity at that position. Residues at each interaction site are numbered below the alignment (A2 residues in yellow circles, B1 residues in red circles). These numbers label the accompanying cartoon to show the dynein chains that constitute each interaction. Alignment generated by ESPript<sup>54</sup> (http://esprit.ibcp.fr).

b, Intermediate

chain interactions showing connections between the IC of A1 and the HC of A2; the IC of A2 and the HC of B1; and the IC of B1 and the HC of B2. Interacting sites on each IC are shown as yellow spheres; sites on each HC are shown as red spheres.

c, B1 (pink) contacts extra density (labelled, blue) adjacent to the BICDR1 coiled coil. The cartoon below shows the location of the area depicted (correspondingly coloured).

d, Weak density connects the extra density with the LIC A2 helix 13 (blue). A cartoon representation of the area depicted is shown below.

Extended Data Figure 8 | Comparison between different adaptors recruiting dynein. **a**, The TDR structure (left) is compared to models of TDH (middle) and TDB (right). Although the paths of BICDR1 (yellow), HOOK3 (magenta) and BICD2 (orange) vary along the surface of dynactin (green surface), dynein-A HC's (light blue) bind at the same sites in each complex. **b**, Zoomed-in views of the barbed end of dynactin show that BICD2 adopts an upwards position to contact ARP1A (grey), whereas BICDR1 and HOOK3 adopt lower positions to bind dynein-B using the region coloured in red. The BICD2–ARP1A interaction site is highlighted in purple.

Extended Data Table 1 | Cryo-EM data collection parameters of TDR and TDH structures and model refinement statistics of the 3.5 Å resolution TDR structure

| Data collection and processing | TDR_1 | TDR_2 | TDH |
|---|---|---|---|
| Voltage (kV) | 300 | 300 | 300 |
| Electron exposure (e−/Å²) | 52 | 52 | 45 |
| Pixel size (Å) | 1.34 | 1.34 | 1.42 |
| Number of sessions | 1 | 11 | 5 |
| Micrographs | 2,459 | 26,906 | 5,464 |
| Symmetry imposed | C1 | C1 | C1 |
| Final particle images (no.) | 12,420 | 205,611 | 23,407 |
| Map resolution (Å) | 6.5 | 3.5 | 6.7 |
| FSC threshold | 0.143 | 0.143 | 0.143 |

| Refinement | N-terminal tail | C-terminal tail | TDR | TDR(ordered) |
|---|---|---|---|---|
| Map | EMD-4169 | EMD-4170 | EMD-4168 | EMD-4168 |
| Map resolution (Å) | 3.4 | 3.4 | 8 | 3.5 |
| FSC threshold | 0.143 | 0.143 | 0.143 | 0.143 |
| Map sharpening B factor (Å²) | -50 | -30 | 0 | -70 |
| Map CC (around atoms) | 0.76 | 0.70 | 0.79 | 0.65 |
| Model composition |  |  |  |  |
| Non-hydrogen atoms | 28,871 | 5,164 | 92,789 | 90,725 |
| Protein residues | 3,555 | 628 | 13,982 | 13,567 |
| Ligands (ADP/ATP) | 0/0 | 0/0 | 9/1 | 9/1 |
| R.m.s. deviations |  |  |  |  |
| Bond lengths (Å) | 0.02 | 0.01 | 0.01 | 0.01 |
| Bond angles (°) | 1.59 | 1.93 | 1.60 | 1.60 |
| Validation |  |  |  |  |
| MolProbity score | 2.12 | 2.21 | 2.05 | 2.05 |
| Clashscore | 11.18 | 13.19 | 9.08 | 9.08 |
| Poor rotamers (%) | 0.29 | 0.36 | 0.44 | 0.44 |
| Ramachandran plot |  |  |  |  |
| Favored (%) | 89.83 | 88.75 | 89.13 | 89.17 |
| Disallowed (%) | 0.11 | 0.16 | 0.14 | 0.14 |
| Cβ deviations (%) | 0.00 | 0.00 | 0.00 | 0.00 |

The TDR_1 dataset is included in TDR_2 dataset. The N-terminal tail model consists of HC of A2 (residues 201–829), HC of B1 (residues 201–629), HC of B2 (residues 201–575), IC of A2 WD40 domain, BICDR1 (residues 132–210), ARP1B, ARP1D, ARP1F, CAPZα and CAPZβ. The C-terminal tail model consists of HC of A2 (residues 517–927) and HC of B1 (residues 453–702). The TDR (ordered) model consists of all parts of TDR for which density was seen.

Extended Data Table 2 | Crystal structure data collection parameters and model refinement statistics of the 1.9 Å resolution structure of the human dynein NDD

| NDD (PDB 5OWO) |
| --- |

**Data collection**

- Space group: P 2 2₁ 2₁
- Cell dimensions:
  - $a, b, c$ (Å): 50.5, 101.8, 176.19
  - $\alpha, \beta, \gamma$ (°): 90, 90, 90
- Resolution (Å): 50.45-1.86 (1.94-1.86) *
- $R_{\text{merge}}$: 0.291(3.939)
- $I / \sigma I$: 3.9(0.4)
- Completeness (%): 94.3(98.5)
- Redundancy: 3.1 (2.9)

**Refinement**

- Resolution (Å): 1.80
- No. reflections: 48901
- $R_{\text{work}} / R_{\text{free}}$: 27.09/29.24
- $B$-factor, from Wilson plot (Å): 25.7
- R.m.s. deviations:
  - Bond lengths (Å): 0.02
  - Bond angles (°): 2.15

*Values in parentheses are for the highest-resolution shell.

# Life Sciences Reporting Summary

Nature Research wishes to improve the reproducibility of the work that we publish. This form is intended for publication with all accepted life science papers and provides structure for consistency and transparency in reporting. Every life science submission will use this form; some list items might not apply to an individual manuscript, but all fields must be completed for clarity.

For further information on the points included in this form, see [Reporting Life Sciences Research](https://www.nature.com/nature-research/editorial-policies/reporting-summary). For further information on Nature Research policies, including our [data availability policy](https://www.nature.com/nature-research/editorial-policies/data-availability), see [Authors & Referees](https://www.nature.com/nature-research/editorial-policies/authors-referees) and the [Editorial Policy Checklist](https://www.nature.com/nature-research/editorial-policies/editorial-policy-checklist).

## Experimental design

1. **Sample size**
   - Describe how sample size was determined.
     > No sample-size calculation or pre-specification of sample size was used in our experimental design. In general, no fewer than fifteen individual molecules were used to make any conclusions. The sample size chosen for the velocity and colocalization data (>900) is larger than examples from previous publications (Schlager et al. 2014, EMBO J, n=331; McKenney et al. 2014, Science, n=>129). The sample size chosen for stall force experiments (>53) is in line with previous stall force studies (Belyy et al. Nature Cell Biol., n=50).

2. **Data exclusions**
   - Describe any data exclusions.
     > Processive dynein/dynactin complex movements in TIRF motility assays were defined as established in Schlager et al., 2014. Any beads that did not show clear motility or a clear stall (defined in Methods) are not included in data analysis.

3. **Replication**
   - Describe whether the experimental findings were reliably reproduced.
     > TIRF data were reliably reproduced from 6 replicates (3 replicates collected on two separate days). Optical trap data were reliably reproduced from at least 5 independent experiments. Estimation of the number of dyneins bound by negative stain EM were reliably reproduced from two repeats collected on two separate days.

4. **Randomization**
   - Describe how samples/organisms/participants were allocated into experimental groups.
     > For calculation of the gold standard FSC, cryo-EM particles were randomly split into two halves using the RELION software.

5. **Blinding**
   - Describe whether the investigators were blinded to group allocation during data collection and/or analysis.
     > Investigators analyzing colocalization and speed data were not told which protein sample each dataset belonged to.

Note: all studies involving animals and/or human research participants must disclose whether blinding and randomization were used.

6. Statistical parameters

For all figures and tables that use statistical methods, confirm that the following items are present in relevant figure legends (or in the Methods section if additional space is needed).

| n/a | Confirmed |
| --- | --- |
| ☐ | The exact sample size $(n)$ for each experimental group/condition, given as a discrete number and unit of measurement (animals, litters, cultures, etc.) |
| ☐ | A description of how samples were collected, noting whether measurements were taken from distinct samples or whether the same sample was measured repeatedly |
| ☐ | A statement indicating how many times each experiment was replicated |
| ☐ | The statistical test(s) used and whether they are one- or two-sided (note: only common tests should be described solely by name; more complex techniques should be described in the Methods section) |
| ☐ | A description of any assumptions or corrections, such as an adjustment for multiple comparisons |
| ☐ | The test results (e.g., $P$ values) given as exact values whenever possible and with confidence intervals noted |
| ☐ | A clear description of statistics including central tendency (e.g., median, mean) and variation (e.g., standard deviation, interquartile range) |
| ☐ | Clearly defined error bars |

See the web collection on **statistics for biologists** for further resources and guidance.

### Software

Policy information about availability of computer code

7. Software

Describe the software used to analyze the data in this study.

ImageJ (v. 1.51j) and Graphpad PRISM 7 were used to analyze TIRF data. Matlab was used to process and analyze optical trap data. Figures were plotted using Matlab and GraphPad PRISM. The following EM softwares were used: Relion 1.4, Relion 2.0, EMAN2, Motioncorr2, GCTF. Gautomatch, CTFFIND3. Structures were built and refined using Coot, PHENIX, Chimera and Pymol.

For manuscripts utilizing custom algorithms or software that are central to the paper but not yet described in the published literature, software must be made available to editors and reviewers upon request. We strongly encourage code deposition in a community repository (e.g., GitHub). Nature Methods **guidance for providing algorithms and software for publication** provides further information on this topic.

### Materials and reagents

Policy information about availability of materials

8. Materials availability

Indicate whether there are restrictions on availability of unique materials or if these materials are only available for distribution by a for-profit company.

No unique materials were used in this study.

9. Antibodies

Describe the antibodies used and how they were validated for use in the system under study (i.e., assay and species).

Carboxyl latex beads (Life Technologies) were coated with custom-made rabbit polyclonal anti-GFP antibodies (Covance Inc). Antibodies were purified through GFP affinity chromatography and validated by ELISA.

10. Eukaryotic cell lines

a. State the source of each eukaryotic cell line used.

Sf9 cells were used only for protein production, originally purchased from ThermoFisher Scientific (cat no. 11496015).

b. Describe the method of cell line authentication used.

No cell line authentication was used.

c. Report whether the cell lines were tested for mycoplasma contamination.

Cells were not tested for mycoplasma contamination.

d. If any of the cell lines used are listed in the database of commonly misidentified cell lines maintained by ICALC, provide a scientific rationale for their use.

No commonly misidentified cell lines were used.

- **Animals and human research participants**

Policy information about [studies involving animals](https://www.nature.com/nature-research/editorial-policies/animal-experiments); when reporting animal research, follow the [ARRIVE guidelines](https://www.nc3rs.org.uk/arrive-guidelines)

11. Description of research animals

Provide details on animals and/or animal-derived materials used in the study.

> No animals were used in this study.

Policy information about [studies involving human research participants](https://www.nature.com/nature-research/editorial-policies/human-research-participants)

12. Description of human research participants

Describe the covariate-relevant population characteristics of the human research participants.

> No human research participants were involved in this study.
