
Ann. Rev. Biochem. 1986. 55:167–193  
Copyright © 1986 by Annual Reviews Inc. All rights reserved

# LYSOSOMAL ENZYMES AND THEIR RECEPTORS

**Kurt von Figura and Andrej Hasilik**

Physiologisch-Chemisches Institut, Westfälische Wilhelms-Universität, Waldeyerstr.  
15, D-4400 Münster, West Germany

---

## CONTENTS

- PERSPECTIVES AND SUMMARY ..... 167
- BIOSYNTHESIS OF LYSOSOMAL ENZYMES ..... 168
  - Synthesis and Modifications in Endoplasmic Reticulum ..... 168
  - Transport to the Golgi ..... 170
  - Common Modifications in the Golgi ..... 171
  - Formation of Mannose 6-Phosphate Residues ..... 173
- MANNOSE 6-PHOSPHATE--DEPENDENT TRANSPORT IN VIVO ..... 178
- MANNOSE 6-PHOSPHATE-SPECIFIC RECEPTORS ..... 178
  - Two Distinct Receptors ..... 178
  - The Cation-Independent Receptor (MPR) ..... 179
- ITINERARIES OF LYSOSOMAL ENZYMES AND OF THE CATION-INDEPENDENT RECEPTOR (MPR) ..... 182
- MANNOSE 6-PHOSPHATE-INDEPENDENT TRANSPORT ..... 188

---

## PERSPECTIVES AND SUMMARY

Interest in lysosomes and lysosomal enzymes was stimulated by the existence of some 30 inherited lysosomal storage disorders in man. The enzyme defects involved in most of these disorders were identified in the 1970s; see review by Neufeld, Lim, & Shapiro in this series in 1975 (1). Presently, these mutations are being characterized at the level of DNA and RNA.

Targeting of lysosomal enzymes is part of the more general question: how do eukaryotic cells transport proteins synthesized in the rough endoplasmic reticulum to diverse destinations? Hickman & Neufeld discovered, in 1972, that the multiple deficiency of lysosomal enzymes in I-cell disease results from a

167
0066-4154/86/0701-0167$02.00

deficiency in a recognition marker that is common to lysosomal enzymes and required for targeting the enzymes to lysosomes (2). This observation provided the basis for many subsequent studies that eventually led to the identification of the recognition marker and its receptor. A 215-kd receptor, which recognizes mannose 6-phosphate residues in lysosomal enzymes, has been identified as an essential component of a system that in many cells allows for specific transport of lysosomal enzymes to lysosomes. It was originally identified as a cell-surface receptor binding exogenous lysosomal enzymes and mediating their transfer to lysosomes along the pathway of receptor-mediated endocytosis. We now know that this receptor functions also in transport of endogenous lysosomal enzymes and that its presence in organelles that constitute elements of the secretory pathway is important for that function. The combined application of biochemical and cytological methods has significantly contributed to the present knowledge of lysosomal enzyme transport. Further, the current application of recombinant DNA methods to the study of lysosomal enzymes and their receptors is expected to provide answers to many unresolved questions.

This review is limited to discussion of synthesis and transport of lysosomal enzymes in mammalian tissues. We focus on the processing of the oligosaccharides in lysosomal enzymes, on the mannose 6-phosphate-specific receptor (MPR), and on its role in the transport of lysosomal enzymes. The transport of lysosomal enzymes and the function of mannose 6-phosphate-specific receptors have been the subject of recent reviews (3–5).

## BIOSYNTHESIS OF LYSOSOMAL ENZYMES

### Synthesis and Modifications in Endoplasmic Reticulum

Most of the proteins that are localized partially or fully to the extracytosolic side of the endoplasmic reticulum, Golgi complex, lysosomes, and nuclear and plasma membranes have in common a signal sequence that directs the ribosomes engaged in their synthesis to the rough endoplasmic reticulum [reviewed in (6)]. This common signal sequence is a stretch of 15–30 mainly hydrophobic amino acids that is localized to the N-terminus. When protruding from the ribosome this stretch first forms a complex with a cytosolic ribonucleoprotein called signal recognition particle (7). Subsequently, this complex binds to a receptor at the surface of the rough endoplasmic reticulum that is called the SRP receptor or the docking protein (8, 9), and the nascent protein is transferred into the lumen of the rough endoplasmic reticulum. Most of the soluble proteins that are released into the endoplasmic reticulum lose the signal peptide before the synthesis of their polypeptide is completed.

A difference in size between polypeptides synthesized in vitro (nonglycosylated and possibly containing the signal peptide) and those synthesized in cultured cells treated with tunicamycin, an inhibitor of glycosylation (nongly-

cosylated products lacking cleavable signal peptide), is the commonly accepted evidence for the existence of a signal peptide in a protein. Using this technique, it has been shown that porcine cathepsin D (10), mouse cathepsin D, and rat β-glucuronidase (11), as well as α and β chains of human β-hexosaminidase (12), transiently contain signal peptides. Direct evidence for the presence of a signal sequence in porcine cathepsin D has been obtained by Erickson et al (13), who determined partial N-terminal sequences of porcine cathepsin D synthesized in vitro both in the presence and in the absence of membranes. The translation of cathepsin D is also regulated by the signal recognition particle (14).

As has been demonstrated for several (nonlysosomal) glycoproteins, a glucosylated oligosaccharide is transferred to certain asparagine residues in the polypeptide intruding into the lumen of the rough endoplasmic reticulum (15–17). Usually this transfer takes place prior to folding of the protein backbone. The oligosaccharide transferred to asparagine residues is pre-assembled in an “activated” form as a derivative of dolichol pyrophosphate [(18), see Figure 1]. The asparagine-linked oligosaccharides in lysosomal

![Diagram](ER)
(cis + mid-Golgi)
(trans-Golgi)

(a) (b) (c)

G₂
G₃
M² M²
M² M²
M³ M⁶
M⁴ Gn
B₄ Gn
P P
Dolichol

enzymes are subject to processing that follows the principles elucidated in the past decade for secretory and membrane glycoproteins. These have been reviewed in the previous issue of this series by Kornfeld & Kornfeld (18) and will be only briefly mentioned. We will focus on the reactions that have been studied in lysosomal enzymes and which in part are specific for them.

Processing of the oligosaccharide is initiated by “trimming” reactions. The removal of the first glucose residue is effected by glucosidase I, takes place within a few minutes of the transfer of the oligosaccharide, and may even precede the completion of the polypeptide synthesis (19). Removal of the two other glucoses by glucosidase II is a much slower process (20). Recently, removal of outer glucose residues within 1 min of synthesis has been observed in cathepsin D in human fibroblasts (21). The trimming in the rough endoplasmic reticulum also involves a specific α-mannosidase (22), and in general seems to yield octamannosyl chains. Specific removal of a mannose residue from the terminus of the middle branch is likely to facilitate further processing in the Golgi complex (23).

## Transport to the Golgi

Several cytological observations suggest that the transport is mediated by smooth vesicles formed in “transitional” elements of the endoplasmic reticulum (24, 25). The transit times from the reticulum to the Golgi complex vary among different products (26–29), and it is not known whether this results from characteristic interactions of the individual products with other components remaining in or leaving the reticulum. In the case of membrane-associated histocompatibility antigens, the transport depends on the availability of β₂-microglobulin (30). Lodish et al (26, 31) postulated that a receptor protein in the endoplasmic reticulum membrane regulates the selective transport of secretory proteins into transport vesicles en route to the Golgi. 1-Deoxynojirimycin, an inhibitor of the trimming glucosidases (32, 33), has been shown to inhibit secretion of α₁-proteinase inhibitor in rat hepatocytes (34) and HepG2 cells (31), and the transport of lysosomal enzymes into lysosomes in fibroblasts (21). In the presence of 1-deoxynojirimycin, these glycoproteins were retarded in the endoplasmic reticulum. Upon subcellular fractionation of cells treated with the drug, the retarded glycoproteins were found in the microsomal fraction and their carbohydrates did not show the characteristics of processing in the Golgi complex (21, 31).

In rat hepatocytes and HepG2 cells, the selectivity of the effect of 1-deoxynojirimycin was indicated by the fact that it did not inhibit the secretion of albumin (31, 34) or of the glycoprotein C3 and transferrin (31). In a mixed population of hybridoma cells, the drug inhibited the secretion of IgD and not of IgM (35). This differential inhibition corresponded well to a differential effect on the formation of complex oligosaccharides. In the case of the membrane-associated glycoprotein v-erbB, the transport to the plasma membrane

was not affected, although the processing was blocked (36). It appears that the inhibition of the processing in the presence of 1-deoxynojirimycin interferes with the transport of certain soluble glycoproteins, including lysosomal enzymes. The inhibition of transport of affected glycoproteins was incomplete and molecules that eventually reached their normal extracellular or lysosomal destinations contained at least some normally processed oligosaccharides (21, 34). Under normal conditions, the transport of proteins between organelles may be differentially influenced by interactions with other components of the system. A well-known example is the retention of β-glucuronidase in microsomal organelles containing a protein called egasyn (37).

## Common Modifications in the Golgi

TRANSPORT The Golgi complex is an elaborate membrane system, in which a number of modification reactions, in particular the synthesis of the carbohydrate portion of the various glycoconjugates, are accomplished [reviewed in (38)]. The complex consists of a stack of flat or fenestrated cisternae with associated tubules and vesicles with polar orientation. The so-called *cis* part receives the product of biosynthesis from the rough endoplasmic reticulum (24) and the other pole, the *trans* part, is marked in secretory cells by condensing vacuoles and secretory vesicles (25). Functionally, it is useful to consider three regions within the stack: *cis*, *mid*, and *trans*, as suggested by Griffith et al (39) and Rothman et al (40). Most studies on the intracellular transport deal with membrane proteins. The various parts of the Golgi complex, though not rigidly separated, are involved in different carbohydrate modification reactions (see below). Lipids, membrane-associated proteins and soluble proteins that are produced in the endoplasmic reticulum are subject to a vectorial (*cis* to *trans*) flow through the Golgi complex. The complex’s own constituents behave as a stationary phase and their relative distribution through the cisternae may be maintained through a counterflow resembling the distillation process (41). It has been pointed out by Slot & Geuze (42) that small vesicles with a large membrane/volume ratio may efficiently accomplish the transport of membrane components. Indeed, small vesicles in the vicinity of the Golgi complex are enriched in various receptors (42–44), and it should be of interest to test the possibility that some vesicles are enriched in the membrane constituents of various parts of the Golgi complex and serve to reflux these constituents between the neighboring organelles. As far as transport between the *cis* and *mid* parts of the Golgi complex is concerned, it has been suggested by Rothman and coworkers that it is the biosynthetic product that is passed over in small vesicles. This suggestion is based on studies of the transport of the membrane glycoprotein G of the vesicular stomatitis virus in an in vitro system, in which the budding of vesicles from Golgi cisternae was observed (45).

As judged from the modifications of the carbohydrates, lysosomal enzymes

can pass through all three parts of the complex. It is a matter of debate, however, whether their passage through the *trans*-Golgi is obligatory. This question is related to the localization of the sorting step and will be discussed below.

COMMON MODIFICATIONS Depending on the protein moiety, the outer-chain mannose residues (see Figure 1) in glycoproteins entering *cis*-Golgi are subjected to further trimming. This is accomplished by at least two α-mannosidases in an ordered sequence, and also involves a specific *N*-acetylglucosaminyl transferase. The first enzyme in this sequence, mannosidase I, is defined as an enzyme hydrolyzing high-mannose oligosaccharides to yield Man₅ GlcNAc₂ (46), and is an accepted marker for *cis*-Golgi (47). Its product may subsequently be processed by *N*-acetylglucosaminyl transferase I to yield GlcNAc Man₅ GlcNAc₂ (48, 49).

Recently, immunocytochemical evidence has been presented for localization of this enzyme in the mid-cisternae of the Golgi complex (50). Mannosidase II is highly specific for the product of *N*-acetylglucosaminyl transferase I and converts it to GlcNAc Man₃ GlcNAc₂. Mannosidase II seems to be rather broadly distributed through the Golgi complex, with a maximum activity in the middle portion of the complex (51). With the aid of specific antibodies directed to mannosidase II, a rather uniform distribution of the enzyme in the elements of the Golgi complex (52) has been firmly established. If mannosidase II does not act on its substrate, addition of galactose and sialic acid results in formation of hybrid oligosaccharides. The action of mannosidase II is prevented e.g. by the presence of the so-called bisecting *N*-acetylglucosamine linked to β-mannose (53, 54). The product of mannosidase II is the preferred substrate of *N*-acetylglucosaminyl transferase II. This reaction opens the pathway for the synthesis of an array of complex oligosaccharides with two or more antennas (18, 55). The final steps in this synthesis take place in *trans*-Golgi cisternae, where galactosyl transferase is localized as has been demonstrated by Roth & Berger (56).

In earlier studies, ample indirect evidence has been obtained of the presence of hybrid or complex oligosaccharides in lysosomal enzymes. This evidence is based on sensitivity to neuraminidase, binding to immobilized and cellular lectins, and carbohydrate analyses, in which fucose, galactose, and sialic acid were detected. Structural data on complex oligosaccharides in soluble lysosomal enzymes is available for β-glucuronidase from human spleen (57). This enzyme contains a small amount of complex oligosaccharides with two antennas, of which one is incomplete. Bi- and triantennary oligosaccharides comprising about 80% of the total oligosaccharides were found in human β-glucocerebrosidase (58). It should be pointed out that this hydrolase is an example of a membrane-associated lysosomal enzyme (59). The observation,

in lysosomal enzymes, of short oligosaccharides containing only four (57, 60) or just a single sugar residue (61), points to intralysosomal degradation. This may in part explain the low amounts of typical complex oligosaccharide structures found in lysosomal enzymes. As determined by lectin binding and resistance to endoglycosaminidase H, an enzyme cleaving most of the high-mannose and hybrid oligosaccharides (62), complex oligosaccharides have been found in metabolically labeled mouse β-glucuronidase (63), human fibroblasts cathepsin D, β-hexosaminidase and arylsulfatase B (64, 65), and in Chang liver α-galactosidase (66), which contains some tri- or tetraantennary oligosaccharides. In newly synthesized cathepsin D, oligosaccharides resistant to endoglycosaminidase H and containing galactose comprise almost 30% of the total oligosaccharide population (64, 67).

While the content of complex oligosaccharides in soluble lysosomal enzymes of normal fibroblasts is generally low, it is elevated in I-cell and mucolipidosis III fibroblasts, predominantly in their secretions (64, 68–71). This striking change results from a defect in these mutant cells in the phosphorylation of high-mannose oligosaccharides (see below).

Sialylated hybrid oligosaccharides have been found in cathepsin D and β-hexosaminidase of human fibroblasts and contained 5–10% of the radioactivity in anionic oligosaccharides released by endoglycosaminidase H (72). A group of hybrid oligosaccharides that contain phosphate in addition to sialic acid was characterized by Varki & Komfeld (73) and will be discussed below.

### Formation of Mannose 6-Phosphate Residues

REACTIONS Observations on the efficiency and saturability of the endocytosis of lysosomal enzymes led Hickman & Neufeld to propose, in 1972 (2), that lysosomal enzymes carry a specific recognition marker. By 1980, it became apparent from a series of observations contributed from several laboratories that the recognition marker in lysosomal enzymes is a carbohydrate (74), that it is related to mannose (75), sensitive to alkaline phosphatase, and probably represented by mannose 6-phosphate residues (76–79). Mannose 6-phosphate was identified in lysosomal enzymes as a component of oligosaccharides cleavable by endoglycosaminidase H (80–83). Phosphate has been found in the carbohydrate of all soluble lysosomal enzymes tested including β-glucuronidase (82), β-hexosaminidase, α-glucosidase and cathepsin D (83), α-L iduronidase (84), arylsulfatase A (85), arylsulfatase B (65), myeloperoxidase (86), acid phosphatase (87), and cathepsin C (88). Subsequent to the identification of the phosphorylated residue, it was observed that some of the phosphorylated oligosaccharides contain N-acetyl-glucosaminyl-1-phospho-6-mannose diester groups (72, 89–91). The anomeric configuration of the N-acetylglucosaminyl residue was found to be α (72, 89, 90, 92). Previously, phosphodiester compounds of sugars or sugar alcohols had been found in cell walls of various

microorganisms. In analogy to the synthesis of phosphomannan in yeast (93), two carbohydrate modification reactions, which form the phosphorylated recognition marker, were identified: transfer of N-acetylglucosaminyl 1-phosphate from the UDP-N-acetylglucosamine to the C-6 hydroxyl of a mannose residue, and hydrolysis of the covering N-acetylglucosamine residue (see Figure 2).

The enzyme catalyzing the first reaction, UDP-N-acetylglucosamine:lysosomal enzyme N-acetylglucosaminyl-1-phosphotransferase (referred to as transferase), was first demonstrated in membranes from rat liver, fibroblasts, and Chinese hamster ovary cells (94–96), and was partially purified from rat liver Golgi preparations (97). [β-³²P]UDP-N-acetylglucosamine is a suitable substrate for determination of transferase activity and can be prepared from [γ-³²P]ATP using commercially available enzymes (98). N-acetylglucosamine 1-phosphate can be transferred to the C-6 position of mannose residues in precursor and mature forms of lysosomal enzymes, high-mannose oligosaccharides, and α-methylmannoside (96, 97). However, lysosomal enzymes are phosphorylated at least 100 times more efficiently than the oligosaccharides or α-methylmannoside. Phosphorylation of high-mannose oligosaccharides in nonlysosomal glycoproteins is not detectable (97), or only barely so (96). The acceptor activity of lysosomal enzymes is destroyed by denaturation (96). When lysosomal enzymes are deglycosylated with endoglicosaminidase H, they become specific inhibitors of the N-acetylglucosaminyl-1-phosphate transfer to lysosomal enzymes (99). Apparently, lysosomal enzymes contain a unique, denaturable structure that is distinct from the acceptor oligosaccharide, common to all lysosomal enzymes, and recognized by the transferase. The dual recognition of lysosomal enzymes by the transferase is strongly supported by the failure of deglycosylated lysosomal enzymes to inhibit phosphorylation of α-methyl mannoside (99), and the characterization of a mutant that phosphorylates high-mannose oligosaccharides and α-methyl mannoside but not lysosomal enzymes (100).

Man-Lysosomal enzyme precursor


UDP-GlcNAc → UMP  
↓  
GlcNAc 1-(P)-6-Man-Lysosomal enzyme precursor  
↓  
H₂O → GlcNAc  
↓  
(P)-6-Man-Lysosomal enzyme precursor

Figure 2 Two-step biosynthesis of mannose 6-phosphate residues in lysosomal enzymes.

Primary structures of several lysosomal enzymes have been determined. No homologies are found when the primary sequences in the vicinity of the two glycosylation sites in porcine cathepsin D (101), one in rat cathepsin B (102), and four potential glycosylation sites found in the established partial sequence of human α-fucosidase (103) are compared. Therefore, the signal for the phosphorylation is not represented (in different species) by a common sequence adjacent to the glycosylation sites. Phosphorylation at different glycosylation sites within a single enzyme is random (63, 64). Thus, it is likely that the signal is present in the lysosomal enzyme in a single copy. Because the signal is sensitive to treatment with heat, sodium dodecylsulfate, or trypsin (99), it is probably part of the protein and dependent on the tertiary rather than primary structure.

In the cell, the precursor forms of the lysosomal enzymes serve as acceptors for the transferase, which is localized to the Golgi complex (104, 105). Within the Golgi complex, transferase activity decreases from *cis* to *trans* (51, 106, 107). At the time of phosphorylation, the bulk of oligosaccharides contain six to nine mannose residues (63). Analyses of mutants showed that phosphorylation of glucosylated high-mannose oligosaccharides is possible in branches that do not contain glucose (108), and that truncated oligosaccharides with only five mannose residues can be phosphorylated (108, 109). Mutants provide some indication of whether a single transferase is responsible for phosphorylation. In I-cell patients, in fibroblasts, and in all tissues examined, the transferase activity is absent (94, 110–113), and lysosomal enzymes do not contain phosphate (83, 85, 87, 114), pointing to the existence of a single transferase. In a milder form of the disease, mucolipidosis III, transferase activity (100, 111, 115) and phosphorylation of lysosomal enzymes (85, 116) are markedly reduced. The residual phosphorylation leads to the formation of oligosaccharides with one as well as with two phosphate groups (115). This further supports the concept of a single transferase.

The second enzyme in the specific pathway, N-acetylglucosamine 1-phosphodiester α-N-acetylglucosaminidase, has been partially purified from rat liver (117, 118) and human placenta (119). The enzyme is immunologically and catalytically distinct from the lysosomal α-N-acetylglucosaminidase. It hydrolyzes UDP-N-acetylglucosamine, but not the corresponding arylglycoside. The mechanism of hydrolysis is that of a glycosidase and not a phosphodiesterase (120). It fractionates with Golgi membranes (104, 117, 118) and in fractionated Golgi membranes, it is distributed at slightly higher densities than the transferase (51, 105–107), which suggests a mid-Golgi localization.

STRUCTURE AND DISTRIBUTION OF PHOSPHORYLATED OLIGOSACCHARIDES IN LYSOSOMAL ENZYMES In Figure 1, examples of the main types of oligosaccharides found in lysosomal enzymes are shown. The structures of the

phosphorylated oligosaccharides were studied in purified lysosomal enzymes as well as in total cellular or secreted glycoproteins. Only a minor portion of the oligosaccharides in lysosomal enzymes becomes phosphorylated. This portion does not exceed 30% in β-glucuronidase synthesized in mouse lymphoma cells (63). Only about 25% of the radioactive mannose was released as phosphorylated oligosaccharides from cathepsin D and β-hexosaminidase that were isolated from NH₄Cl-induced secretions of human fibroblasts and therefore not subjected to degradation in lysosomes (72). In lysosomal enzymes isolated from tissues, the portion of phosphorylated oligosaccharides is even lower. In a β-glucuronidase preparation from human spleen containing forms enriched in mannose 6-phosphate, phosphorylated oligosaccharides accounted for 10% of total oligosaccharides (57, 92), in β-glucuronidase from rat liver lysosomes phosphorylated oligosaccharides were not detectable (91), and in cathepsin D from porcine spleen they accounted for less than 4% of total oligosaccharides (121). Pulse-chase labeling studies indicate the presence of secondary modifications to the oligosaccharides in lysosomal enzymes including removal of blocking N-acetylglucosamine residues (63, 89) and removal of phosphate (63). Therefore, the amount of phosphorylated oligosaccharides and the relative amounts of the different species depend greatly on the source and subcellular location of a lysosomal enzyme.

In all studies, oligosaccharides with one phosphate group were found to be two to five times more frequent than oligosaccharides with two phosphate groups (63, 72, 73, 90, 92). Primarily, the phosphate is found in the diester form. In the cathepsin D and β-hexosaminidase preparations from human skin fibroblasts mentioned above (72), more than 80% of the phosphorylated oligosaccharides contained one or two covered phosphates. A similar portion was found in phosphorylated oligosaccharides in β-glucuronidase and the total cellular glycoproteins isolated from mouse lymphoma cells after a three-hour labeling period (89, 90). In the lysosomal enzymes isolated from tissues, phosphorylated oligosaccharides are found to contain either exclusively monoesters [cathepsin D from porcine spleen, (121)] exclusively diesters [microsomal β-glucuronidase from rat liver, (91)], or predominantly diesters [β-glucuronidase from human spleen, (92)]. Oligosaccharides that contain both a phosphomonoester and phosphodiester group represent in all preparations a minor species (73, 92). The underlying oligosaccharides contain four to nine mannose residues, but species with six to eight mannose residues are the most frequent forms (63, 72, 73, 89–92, 121).

Varki & Kornfeld (90) analyzed in detail the position of the phosphate groups in cellular glycoproteins of mouse lymphoma cells. According to their analysis, the mannose residues that can be phosphorylated are represented by the three residues at the nonreducing termini of branches a through c and the penultimate residues of branches a and c in the Man₈GlcNAc₂ oligosaccharide (shown in

Figure 1). This oligosaccharide is probably the predominant product of the endoplasmic reticulum α-mannosidase (18, 22). Phosphorylation of these five mannose residues is not random and phosphorylated mannoses are most commonly found in branches c and a. During the processing covering N-acetylglucosamine residues and outer nonphosphorylated mannoseresidues are removed. As a result, the oligosaccharides underlying phosphomonoesters are in general smaller than those carrying phosphodiesters, which suggests that the cleavage of the covering N-acetylglucosamine residue is followed by removal of mannoses (63, 73, 90). In oligosaccharides with two phosphate groups, the phosphates always occur on two different branches.

Recently, phosphorylated oligosaccharides that contain one or two sialic acid residues were identified by Varki & Komfeld (73) in P388D₁ macrophage-like cells. These hybrid oligosaccharides contain only a single phosphate as mono- or diester on branch c and lack a bisecting N-acetylglucosamine on the β-linked mannose. Hybrid oligosaccharides containing phosphodiester groups also were found in thyroglobulin secreted by malignant thyroid tissue (122).

Lysosomal enzymes usually contain two or more oligosaccharides per subunit. This holds true for human spleen β-glucuronidase (63, 57), cathepsin D, the α and β chain of β-hexosaminidase, and arylsulfatase B from human skin fibroblasts (64, 65). Any of the oligosaccharides in mouse β-glucuronidase (63) and human cathepsin D (64) may become phosphorylated. From the extent of phosphorylation, it appears that on the average one or less than one oligosaccharide per subunit is phosphorylated. The percentage of phosphate groups that becomes uncovered in the Golgi complex is unknown. The phosphate groups in less than 20% of the phosphorylated oligosaccharides in cathepsin D and β-hexosaminidase become uncovered (72), provided that NH₄Cl does not interfere with the action of phosphodiester α-N-acetylglucosaminidase. Under these conditions, less than 10% of cathepsin D and β-hexosaminidase subunits leave the Golgi complex with high-mannose oligosaccharides containing phosphomonoesters.

Analyses of the oligosaccharides in β-glucuronidase from human spleen (57, 92), mouse lymphoma cells (89), rat liver microsomes and lysosomes (91), and cathepsin D from porcine spleen (121, 123) indicate that neutral high-mannose structures are the prevailing oligosaccharides in these enzymes. The sizes of these oligosaccharides depend on the enzyme sources. In β-glucuronidase from human spleen and from mouse lymphoma and P388D₁ cells, the high-mannose oligosaccharides with nine and eight mannose residues are most frequent (57, 63, 89, 92), whereas in β-glucuronidase from rat liver lysosomes, forms with five mannose residues prevail (91). The presence of neutral high-mannose oligosaccharides depends largely on phosphorylation. This is indicated by the paucity of high-mannose oligosaccharides in lysosomal enzymes from I-cell fibroblasts (64, 68, 69).

178 VON FIGURA & HASILIK

# MANNOSE 6-PHOSPHATE-DEPENDENT TRANSPORT IN VIVO

It appears that mannose 6-phosphate residues participate in the transport of lysosomal enzymes in two physiologically significant ways. First, in certain cells, they are indispensable for targeting endogenous lysosomal enzymes to lysosomes. In 1972, Hickman & Neufeld (2) reported the important finding that lysosomal enzymes in I-cell fibroblasts lack the recognition marker required for receptor-mediated endocytosis. They proposed that the inability of I-cell fibroblasts to equip their enzymes with this recognition marker caused the intracellular deficiency and extracellular accumulation of many lysosomal enzymes that is observed in I-cell fibroblasts. The subsequent demonstration that the inability of I-cell fibroblasts to synthesize mannose 6-phosphate residues in lysosomal enzymes (83, 114) was the result of a deficiency in N-acetylglucosamine 1-phosphotransferase (94, 110) further identified mannose 6-phosphate residues as indispensable for transport of lysosomal enzymes in fibroblasts. The general importance of these residues is indicated by the excessive levels of lysosomal enzymes in all body fluids of I-cell patients and the morphological alterations in many tissues that result from deficiency in intracellular lysosomal enzymes (124).

The second function of mannose 6-phosphate residues is to mediate intercellular exchange of lysosomal enzymes. This function is exemplified in female carriers of Hunter disease. Hunter disease is an X-linked lysosomal disorder that is characterized by a deficiency in iduronate sulfatase. The female carriers have two populations of cells, one of which is deficient in iduronate sulfatase due to inactivation of the X-chromosome carrying the nonaffected gene. Yet, the phenotype of carriers as well as the morphology of their tissues is normal (125). This is explained by transfer of iduronate sulfatase from the normal to the deficient cell population. This phenomenon of cross-correction has been studied by Neufeld and coworkers (1) in fibroblasts and shown to depend on the mannose 6-phosphate-containing recognition marker mentioned above.

The functions of the phosphorylated recognition markers depend on their interaction with specific mannose 6-phosphate receptors. Therefore, the two functions also apply to at least one of the receptors, which will be discussed in the following section.

## MANNOSE 6-PHOSPHATE-SPECIFIC RECEPTORS

### Two Distinct Receptors

Two mannose 6-phosphate-specific receptors are known. These receptors can be differentiated by dependence on divalent cations. The cation-independent

receptor has been extensively characterized with regard to structure, biosynthesis, turnover, subcellular location, and function and will be discussed in detail below and be referred to as MPR. The existence of a second cation-dependent receptor is suggested by a recent report by Hoflack & Komfeld (126). These authors found that membranes from mouse P388D₁ macrophage-like cells bind lysosomal enzymes in a saturable and mannose-6 phosphate–dependent manner. More importantly, binding is strictly dependent on divalent cations. P388D₁ macrophages do not synthesize detectable amounts of the cation-independent receptor (127). In these cells, the transport of lysosomal enzymes to lysosomes is probably mediated by the cation-dependent receptors.

## The Cation-Independent Receptor (MPR)

### ISOLATION OF MPR
MPR is a membrane protein that can be solubilized in the presence of nonionic detergents without loss of its binding properties. The first purification of MPR was achieved by Sahagian et al (128). The authors isolated the MPR from bovine liver using the lysosomal enzyme β-galactosidase immobilized to Sepharose 4B. Because of easier accessibility, yeast phosphomannan and secretions from *Dictyostelium discoideum* became the preferred materials for preparation of affinity matrices (73, 129, 130). The receptors have been isolated from a variety of tissues and cells including bovine liver, human fibroblasts, rat hepatocytes, Chinese hamster ovary cells, and rat chondrosarcoma (128, 131). In several cells of human origin, including normal fibroblasts, HepG2, U937, and HL-60 cells, the receptor constitutes 0.1–0.5% of total membrane protein (our unpublished results).

### STRUCTURAL FEATURES
In SDS-polyacrylamide gel electrophoresis, MPR from different sources behaves as a glycoprotein with an \(M_r\) of about 215,000. It contains fucose and terminal sialic acid residues (127, 131), phosphoserine groups, and intrachain disulfide linkages (132). The latter are indicated by a decrease in electrophoretic mobility after reduction. The receptors span the membrane. A small portion of about 10 kd protrudes at the cytosolic side of the membrane and harbors the C-terminus of the receptor. The greater portion of the receptor polypeptide is exposed at the luminal side of organelles and the outer side of the plasma membrane. This portion contains the mannose 6-phosphate binding site(s) (133, 134).

### BIOSYNTHESIS AND TURNOVER OF MPR
The receptors undergo a series of posttranslational modifications, some of which cause minor changes in the mobility in SDS polyacrylamide electrophoresis (127, 132). The high-mannose oligosaccharides in the receptor are converted predominantly to complex-type structures. The processing of the oligosaccharides is remarkably slow and not complete even two to three hours after synthesis (127, 132). Phosphorylation of
the receptor on serine residues is restricted to the pool of mature receptors. The phosphate has a half-life about seven times shorter than the protein in MPR (132). Additional posttranslational modifications are indicated by sequential acquisition of immunoreactivity and of binding activity in Chinese hamster ovary cells (132). In fibroblasts and adherent Chinese hamster ovary cells, the half-life of the protein moiety ranges from 10 to 29 hours (132, 135, 136). The turnover is not affected if saturating amounts of exogenous ligands are added to cells or if endogenous ligands are lacking (135). Exposure to NH₄Cl or leupeptin, two inhibitors of lysosomal proteolysis, also has no significant effect (135). Sahagian (137) found under specific conditions that the medium of Chinese hamster ovary cells contained immunoreactive fragments, supposedly representing degradation products of MPR. The author proposed a nonlysosomal mechanism for receptor degradation that depends on binding of secreted lysosomal enzymes to MPR.

**BINDING SPECIFICITY** Isolated receptors incorporated into liposomes (130, 131, 138) or immobilized on various matrixes (73, 130, 131, 139) bind lysosomal enzymes in a saturable and mannose 6-phosphate–dependent manner. The binding characteristics of the purified receptors (73, 128, 130, 139) are comparable to receptors in isolated membranes (140–143) or at the cell surface (144–146). The lysosomal enzymes bind with \(K_{D}\)'s in the nanomolar range. The binding is competitively inhibited by mannose 6-phosphate (\(K_{i} = 0.05–2\) mM) and lysosomal enzymes (\(K_{i} = 2–40\) nM). It does not depend on divalent cations and drops precipitously between pH 6 and pH 5.

By studying the binding of oligosaccharides to immobilized receptors, Fischer et al (130) and Varki & Kornfeld (73) could show that phosphomonoester groups in oligosaccharides are essential for binding. Oligosaccharides with one phosphomonoester residue were retarded on receptor-substituted columns and oligosaccharides with two phosphates in monoester linkage bound firmly to the column. Neutral oligosaccharides or oligosaccharides with one or two phosphates in diester linkage to *N*-acetylglucosamine did not bind. The affinity to immobilized receptors compared well with the ability of oligosaccharides to become internalized (147, 148). The \(K_{\text{uptake}}\) of oligosaccharides with one or two phosphates in monoester linkage were \(3.2 \times 10^{-7}\) and \(3.9 \times 10^{-8}\) M (148). Neutral oligosaccharides and those with one phosphate in diester linkage were not internalized. The 10-fold lower \(K_{\text{uptake}}\) of oligosaccharides with two phosphates in monoester linkages argues for an extended binding site in the receptor. Earlier observations of Sly and coworkers (149, 150), who compared the uptake and inhibitor activity of phosphomannan fragments containing multiple phosphomonoesters to that of pentamannosyl monophosphate, suggested a cooperativity in binding. The polyvalent fragment was a potent inhibitor and was internalized, whereas the monovalent pentasaccharide wasnot internalized and was no more inhibitory than mannose 6-phosphate. Recognition of carbohydrates neighboring the mannose 6-phosphate residues contributes to binding. This is evident from the following three observations. First, the \(K_{\text{uptake}}\) of monophosphorylated oligosaccharides is 100-fold lower than the \(K_{i}\) of mannose 6-phosphate (148). Second, presence of outer mannose residues accessible to jack bean \(\alpha\)-mannosidase can lower the affinity of oligosaccharides with phosphomonoester groups for the receptor. Third, the positions of phosphomonoester groups on the oligosaccharides influence the affinity for the receptors (73). Gabel et al (151) analyzed the oligosaccharides in endogenous glycoproteins bound to the receptors in mouse lymphoma cells. The receptor-bound material was highly enriched in oligosaccharides bearing one or two phosphomonoesters, whereas phosphorylated oligosaccharides in nonreceptor-bound material were enriched in phosphodiesters.

Lysosomal enzymes from *Dictyostelium discoideum* have been widely used as tools to measure uptake via the MPR and to isolate the receptors. Their binding properties (145, 146, 152) are indistinguishable from those of lysosomal enzymes. Yet, the recognition of the slime mold enzymes by the receptors is resistant to exhaustive treatment with alkaline phosphatase (153). Gabel et al (154) recently identified these alkaline phosphatase-resistant groups as methylphosphomannose residues. Thus, mannose 6-phosphate residues covered with methyl groups are recognized by the MPR, whereas those covered with \(N\)-acetylglucosamine, as occurs in lysosomal enzymes of mammalian origin, do not bind to the MPR.

DISTRIBUTION OF MPR IN CELLULAR MEMBRANES Early studies on binding of lysosomal enzymes suggested that most of the receptor is localized to intracellular membranes (143). In subcellular fractionation studies, the receptor was found in the plasma membrane (128), in Golgi membranes (155), in coated vesicles (134, 156) and in endosomes (157), but not in lysosomes (132, 136). In various immunocytochemical investigations (158–161), the consensus finding is that MPR is present in the plasma membrane (including coated pits), in coated vesicles present in the vicinity of the plasma membrane, in organelles with tubular extensions that have been defined as CURL (162) [the vesicular elements of CURL are equivalent to endosomes (163) or receptosomes (164)], and in the Golgi area including the nearby coated vesicles, although in certain tissues it may be difficult to detect MPR in the plasma membrane (161). However, the findings with regard to the distribution of MPR in intracellular membranes are strikingly divergent. Thus, in rat hepatocytes and some other cells Brown & Farquhar (159) find MPR to be localized specifically to one or two cisternae in cis-Golgi, whereas Geuze and coworkers (160) observe MPR well distributed through all of the Golgi complex and the trans-Golgi reticulum in rat hepatocytes. In HepG2 cells, more MPR is present in the trans-Golgi
reticulum than in the Golgi itself (161). The binding of the specific antibodies used in these studies was visualized with an immunoperoxidase-conjugated second antibody (159) or with protein A-coated gold particles (160, 161). In these studies, little MPR was found in the endoplasmic reticulum. In contrast, in Chinese hamster ovary cells endoplasmic reticulum was reported to be rich in this receptor (158). Finally, extensive (44, 159) or limited (158, 160) labeling of MPR was observed in the membranes of lysosomal organelles. While it is presently impossible to reconcile these conflicting findings, it is possible that the use of different visualization techniques, the application of different criteria for the identification of the organelles, and the study of different cell types all may have contributed to the apparent differences in these observations.

# ITINERARIES OF LYSOSOMAL ENZYMES AND OF THE CATION-INDEPENDENT RECEPTOR (MPR)

Although the role of MPR in targeting endogenous lysosomal enzyme is firmly established, no conclusive answers are available about where receptors and ligands combine (binding site), where the receptor-ligand complexes are segregated from the secretory pathway, where the ligands dissociate from the receptors, where receptors and ligands are separated, and along which pathway receptors recycle.

## BINDING SITE

Since the binding property of a lysosomal enzyme depends on formation of, at minimum, one uncovered mannose 6-phosphate residue, its interaction with MPR would be possible at or trans to the location of the uncovering enzyme. Localization of MPR to probably all membranes of the Golgi complex and of the uncovering α-N-acetylglucosaminidase to mid-Golgi suggests that mid-Golgi is the most proximal site where the binding of lysosomal enzymes can occur.

## SEGREGATION SITE

The binding to receptors is thought to be followed by the selective packaging into vesicles. The purpose of this packaging is to separate newly synthesized lysosomal enzymes from other products that traverse the Golgi complex and are destined for secretion and certain cellular membranes. The most proximal site where segregation could occur is the binding site itself. The restriction of MPR to the cis-Golgi cisternae and neighboring membranes that Brown & Farquhar (159) observed led them to propose that the lysosomal enzyme-receptor complexes are segregated in the cis-Golgi into clathrin-coated vesicles. This hypothesis is difficult to reconcile with the presence of sialylated, phosphorylated hybrid and complex type oligosaccharides in lysosomal enzymes, which are bound to the receptors (151), en route to lysosomes (67), or present in lysosomes (64, 165). Considering the trans-Golgi localization of the terminal glycosyltransferases that add galactose and sialic acid

residues to hybrid and complex type oligosaccharides (41, 56), lysosomal enzymes are expected to pass the *trans*-Golgi cisternae. As processing appears more likely to occur in nonreceptor than in receptor-bound form, a significant fraction of lysosomal enzyme is likely not to bind before reaching *trans*-Golgi cisternae.

The prevalence of neutral high-mannose oligosaccharides in lysosomal enzymes (see above) also has been utilized as an argument to propose that segregation takes place predominantly in *cis*-Golgi. Lysosomal enzyme precursors secreted in the presence of NH₄Cl retain their neutral high-mannose oligosaccharides (72). These findings, together with the preferred occurrence of complex oligosaccharides in lysosomal enzymes from I-cell fibroblasts (see above), indicate that the exemption of neutral high-mannose oligosaccharides from processing to complex structures is due to the presence of phosphorylated oligosaccharides rather than to binding to MPR or to a bypass of *trans*-Golgi cisternae.

In double-labeling experiments with immunogold, Geuze and coworkers (160, 161) colocalized MPR, lysosomal enzymes, and albumin to the Golgi cisternae and the *trans*-Golgi reticulum. In HepG2 cells, albumin was colocalized with MPR and lysosomal enzymes even at the level of packaging in the coated buds of the *trans*-Golgi reticulum (161).

Theoretically, the most distal site at which segregation of lysosomal enzymes could occur is the plasma membrane. Available data on lysosomal enzyme transport neither contradict nor favor a pathway via the plasma membrane. Such a pathway has been suggested to involve transport of MPR-ligand complexes along the secretory route to the plasma membrane and internalization of the complexes along the pathway of receptor-mediated endocytosis. The pool of lysosomal enzymes bound to MPR detectable by immunofluorescence or immunogold at the plasma membrane of fibroblasts (166) or HepG2 cells (161) may represent the newly synthesized endogenous lysosomal enzymes moving via the plasma membrane. If transport via the plasma membrane constitutes a major pathway, the enzymes at the plasma membrane must be protected from displacement by mannose 6-phosphate (167). However, the lysosomal enzymes at the plasma membrane also may represent enzymes that are reinternalized following secretion. Although this secretion-recapture pathway (168) is of importance in certain conditions such as the Hunter-carrier, it appears to constitute only a minor pathway for delivery of lysosomal enzymes to lysosomes. If cells are grown in the presence of mannose 6-phosphate and related compounds, which block MPR-dependent uptake of lysosomal enzymes, little if any accumulation of lysosomal enzymes in the medium and no intracellular reduction were found (167, 169–171). A portion of the lysosomal enzymes present at the cell surface may function in the degradation of the components of the extracellular matrix (172). Further studies may well show that the location

of the segregation site depends on the cell type and that within a single cell segregation may be a smooth process occurring along an extended section of the secretory pathway.

COATED MEMBRANES AND VESICLES Ultrastructural studies localized MPR and lysosomal enzymes to coated membranes and coated vesicles neighboring the Golgi complex or *trans*-Golgi reticulum and at the plasma membrane. Willingham et al (173) identified the latter as being part of the endocytosis pathway for exogenous lysosomal enzymes. Highly purified preparations of coated vesicles from brain, liver, and human placenta contain lysosomal enzymes. In such preparations, the enzymes are enriched in precursor forms and at least in part bound to MPR (156, 174). Coated membranes immunoselected from fibroblasts, and analyzed for the lysosomal enzyme cathepsin D, contained exclusively the precursor forms of this enzyme (175). It was estimated that during transport the precursors of cathepsin D are associated with coated membranes only for a short period not exceeding a few minutes. This agrees with the view that coated vesicles rapidly lose their coats. The observation that precursors of cathepsin D contain oligosaccharides of the complex type (our unpublished results) supports the notion that coated membranes are involved in lysosomal transport distal to *trans*-Golgi.

DISSOCIATION OF MPR-LYSOSOMAL ENZYME COMPLEXES Like many other receptors involved in transport of ligands to lysosomal or prelysosomal structures, MPR participates in many rounds of transport. The amount of internalized ligand is far in excess of the number of MPR binding sites when the protein synthesis is blocked by cycloheximide (141, 144). Reutilization implies dissociation and separation of receptors and ligands and transport of unoccupied receptors back to the binding site. Receptor-ligand complexes dissociate below pH 5.7 (128, 141). In vivo the same mechanism appears to be utilized for dissociation of receptor-ligand complexes. This is illustrated by the effects of conditions that prevent acidification of intracellular organelles. (Acidification of organelles is reviewed by Helenius & Mellman in this volume.) When dissociation is inhibited, cells should rapidly become depleted in unoccupied receptors. As a consequence, receptor-dependent functions (i.e. sorting of endogenous and endocytosis of exogenous lysosomal enzymes) should be blocked. This is exactly what is observed in cells exposed to weak bases (83, 84, 116, 141, 169, 176, 177) and the ionophore monensin (177–179). Weak bases and ionophores raise the pH of acidic organelles above 6 (180). Among the acidic subcellular compartments (181–184), lysosomes, CURL, and coated vesicles are candidates for the dissociation site. In the laboratories of Robbins (185, 186) and Sly (187, 188), several mutants of Chinese hamster ovary cells were characterized that are defective in ATP-dependent acidification of endo-

sommes but not of lysosomes. Such mutations result in an enhancement in the secretion of lysosomal enzymes and a decrease in the endocytosis. These observations define the dissociation of receptor-enzyme complexes clearly as a prelysosomal event.

SEPARATION OF MPR AND LYSOSOMAL ENZYMES Reutilization of receptors implies that following dissociation, receptors and ligands are separated. Recent morphologic studies by Geuze and coworkers (43, 160, 161) shed some light on where receptors might be separated from lysosomal enzymes. A membrane reticulum containing tubular and vesicular structures extending in hepatocytes from the peripheral cytoplasm down to the *trans*-Golgi area was described by Geuze et al (162) as the compartment where, following endocytosis, the asialoglycoprotein receptors are separated from their ligands. The receptors concentrate in the tubular membranes. Free clathrin is frequently observed adjacent to these tubules, suggesting that coated structures mediate retrieval of receptors from the tubules. The ligands concentrate in the lumina of the smooth vesicular structures, which may develop into or fuse with secondary lysosomes. In rat hepatocytes, MPR colocalizes in CURL with the asialoglycoprotein receptor (43). In HepG2 cells, MPR accumulates in the tubules of CURL, while most of the lysosomal enzymes are found in the lumina of the smooth vesicular structures (161). Thus, CURL is a likely candidate as the site where the receptors are physically separated from lysosomal enzymes. The absence or paucity of receptors in lysosome-enriched fractions from Chinese hamster ovary cells (132) and fibroblasts (133) supports the view that separation of MPR and its ligands occurs prelysosomally.

CYCLING OF MPR The pathway(s) along which receptors are transported for reutilization are unknown. Models for transport of MPR have to take into account that the receptors functioning in transport of exogenous and endogenous lysosomal enzymes share at least one organelle, in which they mix. This is indicated by the inhibition of segregation and of endocytosis of lysosomal enzymes (189, 190), and the rapid tagging of all cellular receptors with antibodies (133, 137, 190) in cells exposed to antireceptor antibodies. CURL is the most likely candidate for the organelle where receptors coming from the Golgi and the plasma membrane mix. Separate pathways may exist for their return to the Golgi and the plasma membrane. We favor a simple view, in which all receptors are transported from the CURL-tubules to the Golgi complex, where they mix with newly synthesized receptors. Unoccupied MPR following the secretory route would replenish the pool of plasma membrane receptors. As discussed above, MPR occupied with endogenous ligands may follow the same pathway or diverge somewhere before reaching the plasma membrane. In the latter case, MPR ferrying endogenous ligands would gain access to CURL

along a route different from the pathway of receptor-mediated endocytosis, and mix in CURL with MPR coming from the cell surface.

Conditions perturbing the transport of MPR to or from CURL, as well as those inhibiting the dissociation of ligands, may be expected to change the subcellular distribution of MPR. Gonzalez-Noriega et al (141) observed a depletion of MPR binding sites at the cell surface with use of chloroquine-treated fibroblasts, and proposed this depletion resulted from inhibition of receptor recycling to the cell surface. When measured as antigenic sites, the number of cell surface receptors is also decreased. However, the uptake of iodinated anti-MPR antibodies is largely resistant to chloroquine and related weak bases (our unpublished results). This indicates that chloroquine alters the steady-state concentration in membranes, but allows for recycling of receptors. Since the ligands do not dissociate in cells exposed to weak bases, ligand-occupied and hence functionally inactive receptors may recycle in such cells. A significant portion of lysosomal enzymes internalized by Chinese hamster ovary cells is returned to the cell surface, indicating that ligand occupation per se does not prevent the backflow of receptors (4).

Farquhar and coworkers (44, 191, 192) proposed that the movements of MPR are induced upon its occupation with or dissociation from ligand. In cells that do not synthesize ligands for MPR (I-cell fibroblasts or hepatocytes exposed to tunicamycin), receptors accumulated in the cis-Golgi cisternae and in adjacent coated vesicles, whereas they were steadily removed from endosomes/lysosomes. An opposite distribution was observed in chloroquine-treated hepatocytes, where ligands cannot be separated from receptors. The authors proposed that MPR shuttles nonconstitutively with movement from the cis-Golgi to endosomes/lysosomes being triggered by occupation with ligand, and movement from endosomes/lysosomes to the cis-Golgi area by dissociation of ligands. At present, this attractive model is difficult to accommodate with the observation that cycloheximide (which inhibits the synthesis of endogenous ligands) is without apparent effect on receptor distribution in hepatocytes (193), and on uptake of antireceptor antibodies (190). Furthermore, cycloheximide does not affect the exchange of receptors between cell surfaces and intracellular membranes, and the kinetics of receptor exchange between intracellular membranes and cell surfaces are similar in I-cell and control fibroblasts and are not significantly affected by weak bases and monensin (our unpublished results). Thus, the ultrastructural and biochemical analyses indicate that dissipation of pH gradients within cells (e.g., by chloroquine or monensin) may alter the relative rates of the movement of the receptor to and from a compartment such that the overall cycling rate is not appreciably changed.

DELIVERY OF LYSOSOMAL ENZYMES INTO DENSE LYSOSOMES The route of the transport of lysosomal enzymes from CURL to lysosomes is not known.
It may involve packaging into specific transport vesicles, which fuse with existing lysosomes. A more likely possibility is a gradual transition of (light) CURL elements to (dense) lysosomes (163, 194). Vesicular elements of the CURL may lose their tubular, receptor-dense connections and detach from the continuous network of CURL. These vesicles, which may be considered as new (light) lysosomes, gradually acquire the high density characteristic of lysosomes. The traditional view that secondary lysosomes are the organelles in which lysosomal enzymes first encounter their substrates and initiate degradation has to be modified. Ligands that are internalized by receptor-mediated endocytosis are separated from their receptors in CURL (162, 193) and mix in the lumen of vesicular CURL elements with lysosomal enzymes. Since CURL is an acidic organelle (182), degradation may well be initiated therein. Some recent experiments designed to trace the compartments in which endocytosed ligands are subject to proteolysis indicate that the degradation is initiated in light organelles that cofractionate with endosomal markers [(195), and personal communication from S. Diment and P. Stahl]. At present, these organelles cannot unequivocally be identified with CURL (vesicles still connected with tubules and subject to membrane recycling) or with light organelles defined above as new lysosomes. New lysosomes should share many properties with CURL, such as composition of the fluid content, but the former should be distinguished by the lack of membrane components, which are subject to recycling. Since internalized lysosomal enzyme can induce degradation of material stored in secondary lysosomes (196), the possibility has to be considered that new lysosomes can fuse with existing secondary lysosomes.

Lysosomal enzymes are converted by a series of proteolytic steps into mature forms, which are typically represented by the forms isolated from tissues. For cathepsin D, the conversion is initiated in light organelles and completed in dense lysosomes (67). In the light organelles the precursor is converted into an intermediate independent of ATP-driven acidification (our unpublished results). Generation of mature forms from the intermediate depends on ATP-driven acidification and thiolproteinases and is localized in lysosomes (197). The processing is likely to be initiated in light organelles discontinuous with CURL, such as new (light) lysosomes according to the definition given above. This view is based on the absence of proteolytically processed forms of cathepsin D in the medium of cultured fibroblasts (169), in spite of the exocytosis of fluid from CURL (198).

For details of proteolytic maturation of lysosomal enzymes and their fate in lysosomes the reader is referred to recent reviews (37, 199).

MODEL FOR MPR-DEPENDENT TRANSPORT OF LYSOSOMAL ENZYMES In this section we propose a speculative model (Figure 3), which attempts to reduce the number of signals required for transport to a minimum. Nascent
lysosomal enzymes are sorted into the endoplasmic reticulum with the aid of amino-terminal signal peptides. A yet undefined structure common to all lysosomal enzymes serves as a signal for the phosphorylation by N-acetylglucosaminyl-1-phosphotransferase. This process, in turn, initiates the generation of mannose 6-phosphate residues, which serve as signals for the MPR. Binding to MPR somewhere in Golgi complex removes lysosomal enzymes from the fluid content that is destined for secretion. By utilizing the secretory route, the complexes can reach the plasma membrane, where a signal residing in MPR allows for collection in coated pits. Along the pathway of receptor-mediated endocytosis, the complexes are ferried to CURL, where receptors and ligands are separated. The same signal allowing for concentration in coated pits at the plasma membrane may be utilized in CURL for collecting the receptors into vesicles that move to the Golgi complex.

The proposed model might reflect a primitive pathway, from which more sophisticated pathways evolved. The latter may allow for a shorter path between the binding site and the segregation site and avoid transport via the plasma membrane. Necessarily, the evolution of these pathways would require the formation of additional signals. Functionally analogous signals have been postulated to direct transport of Golgi-derived products to distinct destinations, such as different domains of the plasma membrane (200, 201) or elements of either a constitutive or a regulated secretory pathway (202). Such signals may trigger packaging of receptor-ligand complexes into specific vesicles that mature into CURL or guide the return of receptors from CURL to either Golgi or plasma membrane. In Figure 3 such shorter pathways are indicated by dashed lines.

# MANNOSE 6-PHOSPHATE-INDEPENDENT TRANSPORT

Several lines of evidence indicate that lysosomal enzymes can be ferried in a manner that is independent of mannose 6-phosphate-specific receptors. In I-cell fibroblasts, variable residual amounts of lysosomal enzymes are found within the lysosomes (69). The fraction of newly synthesized enzyme targeted to lysosomes may be as high as 20–50% for α-glucosidase and cathepsin D (87,

Figure 3 A model of the transport of lysosomal enzymes. The thin lines refer to pathways of the enzymes and the bold lines to those of MPR. For further explanation see text.

169), whereas for other enzymes, such as β-hexosaminidase, α-L-iduronidase, and arylsulfatase A it is below the limit of detection (84, 87, 169, 203). The activity of acid phosphatase is normal in I-cell fibroblasts. This has to be attributed to secondary effects, since only one third of the newly synthesized acid phosphatase polypeptides are targeted to lysosomes (87). The normal activities in I-cell fibroblasts of β-glucocerebrosidase and acetyl CoA:α-glucosaminide N-acetyltransferase, two integral membrane enzymes, suggest that membrane proteins find their way independent of mannose 6-phosphate residues. This agrees with the absence of mannose 6-phosphate residues in membrane proteins of the lysosomal membrane (our unpublished results), including β-glucocerebrosidase (204).

In liver, spleen, kidney, and brain from I-cell patients, the activities of many lysosomal enzymes are normal, although these tissues are deficient in N-acetylg glucosaminyl 1-phosphotransferase (112, 113). As the residual lysosomal enzymes in I-cell fibroblasts, these enzymes must use mannose 6-phosphate-independent mechanisms for transport into lysosomes. Therefore, we must consider that mannose 6-phosphate-independent mechanisms also contribute to targeting of lysosomal enzymes in normal tissues.

**ACKNOWLEDGMENTS**

Thanks are due to Dr. J. Conary for critical reading of the manuscript and Mrs. R. Rumpff for typing the manuscript. Our research on lysosomal enzymes is sponsored by the Deutsche Forschungsgemeinschaft and Fonds der Chemischen Industrie.

**Literature Cited**

1. Neufeld, E. F., Lim, T. W., Shapiro, L. J. 1975. *Ann. Rev. Biochem.* 44:357–76
2. Hickman, S., Neufeld, E. F. 1972. *Biochem. Biophys. Res. Commun.* 49: 992–99
3. Creek, K. E., Sly, W. S. 1984. In *Lysosomes in Biology and Pathology*, ed. J. T. Dingle, R. T. Dean, W. Sly, pp. 63–82. Amsterdam Elsevier
4. Sahagian, G. G. 1985. In *Recent Research on Vertebrate Lectins*. Advanced Cell Biology Monographs, ed. B. Parent, K. Olden. New York: van Nostrand In press
5. Marchase, R. B., Koro, L. A., Hiller, A. M. 1984. In *The Receptors*, ed. P. M. Conn, pp. 261–311. Orlando: Academic
6. Walter, P., Gilmore, R., Blobel, G. 1984. *Cell* 38:5–8
7. Walter, P., Blobel, G. 1982. *Nature* 299:691–98
8. Gilmore, R., Blobel, G., Walter, P. 1982. *J. Cell Biol.* 95:463–69
9. Meyer, D. I., Krause, E., Dobberstein, B. 1982. *Nature* 297:647–50
10. Erickson, A. H., Blobel, G. 1979. *J. Biol. Chem.* 254:11771–74
11. Rosenfeld, M. G., Kreibich, G., Popov, D., Kato, K., Sabatini, D. D. 1982. *J. Cell Biol.* 93:135–43
12. Proia, R., Neufeld, E. F. 1982. *Proc. Natl. Acad. Sci. USA* 79:6360–64
13. Erickson, A. H., Conner, G. E., Blobel, G. 1981. *J. Biol. Chem.* 256:11224–31
14. Erickson, A. H., Walter, P., Blobel, G. 1983. *Biochem. Biophys. Res. Commun.* 115:275–80
15. Kiely, M., McKnight, G. S., Schimke, R. 1976. *J. Biol. Chem.* 251:5490–95
16. Bergman, L. W., Kuehl, W. M. 1978. *Biochemistry* 17:5174–80
17. Rothman, J. E., Katz, F. N., Lodish, H. F. 1978. *Cell* 15:1447–54
18. Kornfeld, R., Kornfeld, S. 1985. *Ann. Rev. Biochem.* 54:631–64
19. Atkinson, P. H., Lee, J. T. 1984. *J. Cell Biol.* 98:2245–49
20. Hubbard, S. C., Robbins, P. W. 1979. *J. Biol. Chem.* 254:4568–76

21. Lemansky, P., Gieselmann, V., Hasilik, A., von Figura, K. 1984. *J. Biol. Chem.* 259:10129–35
22. Bischoff, J., Kornfeld, R. 1983. *J. Biol. Chem.* 258:7907–10
23. Tabas, I., Kornfeld, S. 1979. *J. Biol. Chem.* 254:11655–63
24. Jamieson, J. D., Palade, G. E. 1968. *J. Cell Biol.* 39:589--603
25. Farquhar, M. G., Palade, G. E. 1981. *J. Cell Biol.* 91:77s–103s
26. Lodish, H. F., Kong, N., Snider, M., Strous, G. J. A. M. 1983. *Nature* 304:80–83
27. Fries, E., Gustafsson, L., Peterson, P. A. 1984. *EMBO J.* 3:147–52
28. Scheele, G., Tartakoff, A. 1985. *J. Biol. Chem.* 260:926–31
29. Gabel, C. A., Kornfeld, S. 1984. *J. Cell Biol.* 99:296–305
30. Sege, K., Rask, L., Peterson, P. A. 1981. *Biochemistry* 20:4523–30
31. Lodish, H. F., Kong, N. 1984. *J. Cell Biol.* 98:1720–29
32. Saunier, B., Kilker, R. D. Jr., Tkacz, J. S., Quaroni, A., Herscovics, A. 1982. *J. Biol. Chem.* 257:14155–61
33. Romero, P. A., Saunier, B., Herscovics, A. 1985. *Biochem. J.* 226:733–40
34. Gross, V., Andus, T., Tran-Thi, T.-A., Schwarz, R. T., Decker, K., Heinrich, P. C. 1983. *J. Biol. Chem.* 258:12203–9
35. Peyrieras, N., Bause, E., Legler, G., Vasilov, R., Claesson, L., et al. 1983. *EMBO J.* 2:823–32
36. Schmidt, J. A., Beug, H., Hayman, M. J. 1985. *EMBO J.* 4:105–12
37. Skudlarek, M. D., Novak, E., Swank, R. T. 1984. See Ref. 3, pp. 17–34
38. Tartakoff, A. M. 1983. *Int. Rev. Cytol.* 85:221–52
39. Griffith, G., Quinn, P., Warren, G. 1983. *J. Cell Biol.* 96:835–50
40. Rothman, J. E., Miller, R. L., Urbani, L. J. 1984. *J. Cell Biol.* 99:260–71
41. Rothman, J. E. 1981. *Science* 213:1212–19
42. Slot, J. W., Geuze, H. J. 1983. *J. Histochem. Cytochem.* 31:1049–56
43. Geuze, H. J., Slot, J. W., Strous, G. J. A. M., Peppard, J., von Figura, K., et al. 1984. *Cell* 37:195–204
44. Brown, W. J., Farquhar, M. G. 1984. *Proc. Natl. Acad. Sci. USA* 81:5135–39
45. Balch, W. E., Glick, B. S., Rothman, J. E. 1984. *Cell* 39:525–36
46. Tulsiani, D. R. P., Hubbard, S. C., Robbins, P. W., Touster, O. 1982. *J. Biol. Chem.* 257:3660–68
47. Dunphy, W. G., Rothman, J. E. 1983. *J. Cell Biol.* 97:270–75
48. Harpaz, N., Schachter, H. 1980. *J. Biol. Chem.* 255:4894–902
49. Oppenheimer, C. L., Hill, R. L. 1981. *J. Biol. Chem.* 256:799–804
50. Dunphy, W. G., Brands, R., Rothman, J. E. 1985. *Cell* 40:463–72
51. Goldberg, D., Kornfeld, S. 1983. *J. Biol. Chem.* 258:3159–65
52. Novikoff, P. M., Tulsiani, D. R. P., Touster, O., Yam, A., Novikoff, A. B. 1983. *Proc. Natl. Acad. Sci. USA* 80: 4364–68
53. Narasimhan, S. 1982. *J. Biol. Chem.* 257:10235–42
54. Allen, S. D., Tsai, D., Schachter, H. 1984. *J. Biol. Chem.* 259:6984–90
55. Schachter, H., Narasimhan, S., Gleeson, P., Vella, G. 1983. *Can. J. Biochem. Cell Biol.* 61:1049–66
56. Roth, J., Berger, E. 1982. *J. Cell Biol.* 92:223–29
57. Howard, D. R., Natowicz, M., Baenziger, J. U. 1982. *J. Biol. Chem.* 257: 10861–68
58. Takasaki, S., Murray, G. J., Furbish, F. S., Brady, R. O., Barranger, J. A., Kobata, A. 1984. *J. Biol. Chem.* 259: 10112–17
59. Erickson, A., Ginns, E. I., Barranger, J. A. 1985. *J. Biol. Chem.* In press
60. Taniguchi, T., Mizuochi, T., Towatari, T., Katunuma, N., Kobata, A. 1985. *J. Biochem.* 97:973–76
61. Takahashi, T., Dehdrami, A. H., Schmidt, P. G., Tang, J. 1984. *J. Biol. Chem.* 259:9874–82
62. Kobata, A. 1979. *Anal. Biochem.* 100:1–14
63. Goldberg, D. E., Kornfeld, S. 1981. *J. Biol. Chem.* 256:13060–67
64. Hasilik, A., von Figura, K. 1981. *Eur. J. Biochem.* 121:125–29
65. Steckel, F., Hasilik, A., von Figura, K. 1983. *J. Biol. Chem.* 258:14322–26
66. Sweeley, C. C. 1983. *Arch. Biochem. Biophys.* 224:186–95
67. Gieselmann, V., Pohlmann, R., Hasilik, A., von Figura, K. 1983. *J. Cell Biol.* 97:1–5
68. Kress, B., Freeze, H., Herd, J. K., Alhadeff, J. A., Miller, A. 1980. *J. Biol. Chem.* 255:955–61
69. Miller, A. L., Kress, B. C., Stein, R., Kinnon, C., Kem, H., et al. 1981. *J. Biol. Chem.* 256:9352–62
70. Vladutiu, G. D., Rattazzi, M. C. 1975. *Biochem. Biophys. Res. Commun.* 67: 956–64
71. Kress, B. C., Hirani, S., Freeze, H. H., Little, L., Miller, A. L. 1982. *Biochem. J.* 207:421–28
72. Hasilik, A., Klein, U., Waheed, A.,

Strecker, G., von Figura, K. 1980. *Proc. Natl. Acad. Sci. USA* 77:7074–78

73. Varki, A., Kornfeld, S. 1983. *J. Biol. Chem.* 258:2808–18

74. Hickman, S., Shapiro, L. J., Neufeld, E. F. 1974. *Biochem. Biophys. Res. Commun.* 57:55–61

75. Hieber, V., Distler, J., Myerowitz, R., Schmickel, R. D., Jourdan, G. W. 1976. *Biochem. Biophys. Res. Commun.* 73:710–17

76. Kaplan, A., Achord, D. T., Sly, W. S. 1977. *Proc. Natl. Acad. Sci. USA* 74: 2026–30

77. Kaplan, A., Fischer, D., Achord, D., Sly, W. S. 1977. *J. Clin. Invest.* 60: 1088–93

78. Sando, G. N., Neufeld, E. F. 1977. *Cell* 12:619–27

79. Ullrich, K., Mersmann, G., Weber, E., von Figura, K. 1978. *Biochem. J.* 170: 643–50

80. von Figura, K., Klein, U. 1979. *Eur. J. Biochem.* 94:347–54

81. Distler, J., Hieber, V., Sahagian, G., Schmickel, R., Jourdan, G. W. 1979. *Proc. Natl. Acad. Sci. USA* 76:4235–39

82. Natowicz, M. R., Chi, M. M.-Y., Lowry, O. H., Sly, W. S. 1979. *Proc. Natl. Acad. Sci. USA* 76:4322–26

83. Hasilik, A., Neufeld, E. F. 1980. *J. Biol. Chem.* 255:4946–50

84. Myerowitz, R., Neufeld, E. F. 1981. *J. Biol. Chem.* 256:3044–48

85. Waheed, A., Hasilik, A., Cantz, M., von Figura, K. 1982. *Eur. J. Biochem.* 122:119–23

86. Hasilik, A., Pohlmann, R., Olsen, R. L., von Figura, K. 1984. *EMBO J.* 3:267–76

87. Lemansky, P., Gieselmann, V., Hasilik, A., von Figura, K. 1985. *J. Biol. Chem.* 260:9023–30

88. Mainferme, F., Wattiaux, R., von Figura, K. 1985. *Eur. J. Biochem.* 153:211–16

89. Tabas, I., Kornfeld, S. 1980. *J. Biol. Chem.* 255:6633–39

90. Varki, A., Kornfeld, S. 1980. *J. Biol. Chem.* 1980. 255:10847–58

91. Mizuochi, T., Nishimura, J., Kato, K., Kobata, A. 1981. *Arch. Biochem. Biophys.* 209:298–303

92. Natowicz, M., Baenziger, J. U., Sly, W. S. 1982. *J. Biol. Chem.* 257:4412–20

93. Karson, E. M., Ballou, C. E. 1978. *J. Biol. Chem.* 253:6484–92

94. Hasilik, A., Waheed, A., von Figura, K. 1981. *Biochem. Biophys. Res. Commun.* 98:761–67

95. Reitman, M. L., Kornfeld, S. 1981. *J. Biol. Chem.* 256:4275–81

96. Reitman, M. L., Kornfeld, S. 1981. *J. Biol. Chem.* 256:11977–80

97. Waheed, A., Hasilik, A., von Figura, K. 1982. *J. Biol. Chem.* 257:12322–31

98. Lang, L., Kornfeld, S. 1984. *Anal. Biochem.* 140:264–69

99. Lang, L., Reitman, M., Tang, J., Roberts, R. M., Kornfeld, S. 1984. *J. Biol. Chem.* 259:14663–71

100. Varki, A., Reitman, M. L., Kornfeld, S. 1981. *Proc. Natl. Acad. Sci. USA* 78: 7773–77

101. Shewale, J. G., Tang, J. 1984. *Proc. Natl. Acad. Sci. USA* 81:3703–7

102. Takio, K., Towatari, T., Katunuma, N., Teller, D. C., Titani, K. 1983. *Proc. Natl. Acad. Sci. USA* 80:3660–70

103. de Wet, J. R., Fukushima, H., Dewji, N. N., Wilcox, E., O’Brien, J. S., Helinski, D. R. 1984. *DNA* 3:437–47

104. Waheed, A., Pohlmann, R., Hasilik, A., von Figura, K. 1981. *J. Biol. Chem.* 256: 4150–52

105. Deutscher, S. L., Creek, K. E., Merion, M., Hirschberg, C. B. 1983. *Proc. Natl. Acad. Sci. USA* 80:3938–42

106. Pohlmann, R., Waheed, A., Hasilik, A., von Figura, K. 1982. *J. Biol. Chem.* 257:5723–25

107. Morré, D. J., Minnifield, N., Creek, K. E., Navas, P. 1985. *Special FEBS Meet.*, p. 144 (Abstr.) Algarve, Portugal: printed in Dept. Zool. Univ. Coimbra

108. Gabel, C. A., Kornfeld, S. 1982. *J. Biol. Chem.* 257:10605–12

109. Stahl, J., Robbins, A. R., Krag, S. 1982. *Proc. Natl. Acad. Sci. USA* 79:2296– 300

110. Reitman, M. L., Varki, A., Kornfeld, S. 1981. *J. Clin. Invest.* 67:1574–79

111. Varki, A., Reitman, M. L., Vannier, A., Kornfeld, S., Grubb, J. H., Sly, W. S. 1982. *Am. J. Hum. Genet.* 34:717–29

112. Waheed, A., Pohlmann, R., Hasilik, A., von Figura, K., van Elsen, A., Leroy, J. G. 1982. *Biochem. Biophys. Res. Commun.* 105:1052–58

113. Owada, M., Neufeld, E. F. 1982. *Biochem. Biophys. Res. Commun.* 105: 814–20

114. Bach, G., Bargal, R., Cantz, M. 1979. *Biochem. Biophys. Res. Commun.* 91: 976–81

115. Waheed, A., Hasilik, A., Cantz, M., von Figura, K. 1982. *Hoppe Seyler’s Z. Physiol. Chem.* 363:169–78

116. Robey, P., Neufeld, E. F. 1982. *Arch. Biochem. Biophys.* 213:251–57

117. Varki, A., Kornfeld, S. 1980. *J. Biol. Chem.* 255:8398–401

118. Varki, A., Kornfeld, S. 1981. *J. Biol. Chem.* 256:9937–43

119. Waheed, A., Hasilik, A., von Figura, K. 1981. *J. Biol. Chem.* 256:5717–21

120. Varki, A., Sherman, N., Kornfeld, S.

192 VON FIGURA & HASILIK

121. Nakao, Y., Kozutsumi, Y., Kawasaki, T., Yamashina, I., van Halbeek, H., Vliegenthart, J. F. G. 1984. Arch. Biochem. Biophys. 229:43–54
122. Yamamoto, K., Tsuji, T., Tarutani, O., Osawa, T. 1985. Biochim. Biophys. Acta 838:84–91
123. Takahashi, T., Schmidt, P. G., Tang, J. 1983. J. Biol. Chem. 258:2819–30
124. Neufeld, E. F., McKusick, V. A. 1983. In The Metabolic Basis of Inherited Disease, ed. J. B. Stanbury, J. B. Wyngaarden, D. S. Frederickson, J. L. Goldstein, M. S. Brown, pp. 751–77. New York: McGraw-Hill. 5th ed.
125. McKusick, V. A., Neufeld, E. F. 1983. See Ref. 124, pp. 778–87
126. Hoflack, B., Komfeld, S. 1985. Proc. Natl. Acad. Sci. USA 82:4428–32
127. Goldberg, D. E., Gabel, C. A., Komfeld, S. 1983. J. Cell Biol. 97:1700–6
128. Sahagian, G. G., Distler, J., Jouridian, G. W. 1981. Proc. Natl. Acad. Sci. USA 78:4289–93
129. Sahagian, G. G., Distler, J. J., Jouridian, G. W. 1982. Methods Enzymol. 83:392–96
130. Fischer, H. D., Creek, K. E., Sly, W. S. 1982. J. Biol. Chem. 257:9938–43
131. Steiner, A. W., Rome, L. H. 1982. Arch. Biochem. Biophys. 214:681–87
132. Sahagian, G. G., Neufeld, E. F. 1983. J. Biol. Chem. 258:7121–28
133. von Figura, K., Gieselmann, V., Hasilik, A. 1985. Biochem. J. 225:543–47
134. Sahagian, G. G., Steer, C. J. 1985. J. Biol. Chem. 260:9838–42
135. Creek, K. E., Sly, W. S. 1983. Biochem. J. 214:353–60
136. von Figura, K., Gieselmann, V., Hasilik, A. 1984. EMBO J. 3:1281–86
137. Sahagian, G. G. 1984. Biol. Cell 51:207–14
138. Campbell, C. H., Miller, A. C., Rome, L. H. 1983. Biochem. J. 214:413–19
139. Talkad, V., Sly, W. S. 1983. J. Biol. Chem. 258:7345–51
140. Rome, L. H., Miller, J. 1980. Biochem. Biophys. Res. Commun. 92:986–93
141. Gonzalez-Noriega, A., Grubb, J. H., Talkad, V., Sly, W. S. 1980. J. Cell Biol. 85:839–52
142. Fischer, H. D., Gonzalez-Noriega, A., Sly, W. S. 1980. J. Biol. Chem. 255:5069–74
143. Fischer, H. D., Gonzalez-Noriega, A., Sly, W. S., Moré, J. D. 1980. J. Biol. Chem. 255:9608–15
144. Rome, L. H., Weissmann, B., Neufeld, E. F. 1979. Proc. Natl. Acad. Sci. USA 76:2331–34
145. Freeze, H. H., Yeh, R. Y., Miller, A. L. 1983. J. Biol. Chem. 258:8928–33
146. Shepherd, V. L., Freeze, H. H., Miller, A. L., Stahl, P. D. 1984. J. Biol. Chem. 259:2257–61
147. Creek, K. E., Sly, W. S. 1982. J. Biol. Chem. 257:9931–37
148. Natowicz, M., Hallett, D. W., Frier, C., Chi, M., Schlesinger, P. H., Baenziger, J. U. 1983. J. Cell Biol. 96:915–19
149. Fischer, H. D., Natowicz, M., Sly, W. S., Bretthauer, R. K. 1980. J. Cell Biol. 84:77–86
150. Kaplan, A., Fischer, D., Sly, W. S. 1978. J. Biol. Chem. 253:647–50
151. Gabel, C. A., Goldberg, D. E., Kornfeld, S. 1982. J. Cell Biol. 95:536–42
152. Freeze, H. H., Miller, A. L., Kaplan, A. 1980. J. Biol. Chem. 255:11081–84
153. Cladaras, M. H., Graham, T., Kaplan, A. 1983. Biochem. Biophys. Res. Commun. 116:541–46
154. Gabel, C. A., Costello, C. E., Reinhold, V. N., Kurz, L., Kornfeld, S. 1984. J. Biol. Chem. 259:13762–69
155. Brown, W. J., Farquhar, M. G. 1985. J. Cell Biol. 101:138a (Abstr.)
156. Campbell, C. H., Rome, L. H. 1983. J. Biol. Chem. 258:13347–52
157. Dickson, R. B., Beguinot, L., Hanover, J. A., Richert, N. D., Willingham, M. C., Pastan, I. 1983. Proc. Natl. Acad. Sci. USA 80:5335–39
158. Willingham, M. C., Pastan, I. H., Sahagian, G. G. 1983. J. Histochem. Cytochem. 31:1–11
159. Brown, W. J., Farquhar, M. G. 1984. Cell 36:295–307
160. Geuze, H. J., Slot, J. W., Strous, G. J. A. M., Hasilik, A., von Figura, K. 1984. J. Cell Biol. 98:2047–54
161. Geuze, H. J., Slot, J. W., Strous, G. J. A. M., Hasilik, A., von Figura, K. 1985. J. Cell Biol. In press
162. Geuze, H. J., Slot, J. W., Strous, G. J. A. M., Lodish, H. F., Schwartz, A. L. 1983. Cell 32:277–87
163. Helenius, A., Mellman, I., Wall, D., Hubbard, A. 1983. Trends Biochem. Sci. 8:245–50
164. Pastan, I., Willingham, M. C. 1983. Trends Biochem. Sci. 8:250–54
165. Vladutiu, G. D. 1983. Biochim. Biophys. Acta 760:363–70
166. von Figura, K., Voss, B. 1979. Exp. Cell Res. 121:267–76
167. von Figura, K., Weber, E. 1978. Biochem. J. 176:943–50
168. Neufeld, E. F., Sando, G. N., Garvin, A. J., Rome, L. H. 1977. J. Supramol. Struct. 6:95–101
169. Hasilik, A., Neufeld, E. F. 1980. J. Biol. Chem. 256:4937–45

170. Vladutiu, G. D., Rattazzi, M. 1979. *Clin. Invest.* 63:595–601
171. Sly, W. S., Natowicz, M., Gonzalez-Noriega, A., Grubb, J. H., Fischer, H. D. 1981. In *Lysosomes and Lysosomal Storage Diseases*, ed. J. W. Callahan, J. A. Lowden, pp. 131–43. New York: Raven
172. Roff, C. F., Wang, J. L. 1983. In *Glycoconjugates Proc. 7th Int. Symp. Glycoconjugates*, ed. M. A. Chester, D. Heinegard, A. Lundblad, S. Svensson, pp. 520–21. Lund: Rahms
173. Willingham, M. C., Pastan, I. H., Sahagian, G. G., Jourdian, G. W., Neufeld, E. F. 1981. *Proc. Natl. Acad. Sci. USA* 78:6967–71
174. Timmers, S., Zühsdorf, M., Robenek, H., Hasilik, A., von Figura, K. 1983. *Hoppe-Seyler's Z. Physiol. Chem.* 364: 1287–95
175. Schulze-Lohoff, E., Hasilik, A., von Figura, K. 1985. *J. Cell Biol.* 101:824–29
176. Sando, G. N., Titus-Dillon, P., Hall, C. W., Neufeld, E. F. 1979. *Exp. Cell Res.* 119:359–64
177. Merion, M., Sly, W. S. 1983. *J. Cell Biol.* 96:644–50
178. Pohlmann, R., Krüger, S., Hasilik, A., von Figura, K. 1984. *Biochem. J.* 217:649–58
179. Vladutiu, G. D., Rattazzi, M. G. 1980. *Biochem. J.* 192:813–20
180. Ohkuma, S., Poole, B. 1978. *Proc. Natl. Acad. Sci. USA* 75:3327–31
181. Schneider, D. L. 1981. *J. Biol. Chem.* 256:3858–64
182. Tycko, B., Maxfield, F. R. 1982. *Cell* 28:643–51
183. Galloway, C. J., Dean, G. E., Marsh, M., Rudnick, G., Mellman, I. 1983. *Proc. Natl. Acad. Sci. USA* 80:3334–38
184. Forgac, M., Cantley, L., Wiedenmann, B., Altstiel, L., Branton, D. 1983. *Proc. Natl. Acad. Sci. USA* 80:1300–3
185. Robbins, A. R., Peng, S. R., Marshall, J. L. 1983. *J. Cell Biol.* 96:1064–71
186. Robbins, A. R., Oliver, C., Bateman, J. L., Krag, S. S., Galloway, C. J., Mellman, I. 1984. *J. Cell Biol.* 99:1296–1308

187. Merion, M., Schlesinger, P., Brooks, R. M., Moehring, J. M., Moehring, T. J., Sly, W. S. 1983. *Proc. Natl. Acad. Sci. USA* 80:5315–19
188. Sly, W. S., Merion, M., Schlesinger, P., Moehring, J. M., Moehring, T. J. 1983. In *Protein Synthesis*, ed. A. K. Abraham, T. S. Eikhom, I. F. Pryme, pp. 239–51. Clifton, NJ: Humana
189. Creek, K. E., Grubb, J. H., Sly, W. S. 1983. *J. Cell Biol.* 97:253a (Abstr.)
190. Gartung, C., Braulke, T., Hasilik, A., von Figura, K. 1985. *EMBO J.* 4:1725–30
191. Brown, W. J., Constantinescu, E., Farquhar, M. G. 1984. *J. Cell Biol.* 99:320–26
192. Brown, W. J., Farquhar, M. G. 1984. *J. Cell Biol.* 99:231a (Abstr.)
193. Geuze, H. J., Slot, J. W., Strous, G. J., Luzio, J. P., Schwartz, A. L. 1984. *EMBO J.* 3:2677–85
194. Brown, M. S., Anderson, R. G. W., Goldstein, J. L. 1983. *Cell* 32:663–67
195. Berg, T., Ford, T., Kindberg, G., Blomhoff, R., Drevon, C. 1985. *Exp. Cell Res.* 156:570–74
196. Rome, L. H., Garvin, A. J., Alietta, M. M., Neufeld, E. F. 1979. *Cell* 17:143–53
197. Gieselmann, V., Hasilik, A., von Figura, K. 1985. *J. Biol. Chem.* 260:3215–20
198. Besterman, J. M., Airhart, J. A., Woodworth, R. C., Low, R. B. 1981. *J. Cell Biol.* 91:716–27
199. Hasilik, A., von Figura, K. 1984. See Ref. 3, pp. 3–16
200. Rindler, M. J., Ivanov, I. E., Plesken, H., Rodriguez-Boulan, E., Sabatini, D. D. 1984. *J. Cell Biol.* 98:1304–19
201. Matlin, K. S., Simons, K. 1984. *J. Cell Biol.* 98:2131–39
202. Gumbiner, B., Kelly, R. B. 1982. *Cell* 28:51–59
203. Waheed, A., Hasilik, A., von Figura, K. 1982. *Hoppe-Seyler's Z. Physiol. Chem.* 363:425–30
204. Murray, G. J., Jonsson, L. V., Sorrell, S. H., Ginns, E. I., Tager, J. M., et al. 1985. *Fed. Proc.* 44:709 (Abstr.)
