
# Structures of the prefusion form of measles virus fusion protein in complex with inhibitors

Takao Hashiguchi${}^{a,1,2}$, Yoshinari Fukuda${}^{a,1}$, Rei Matsuoka${}^{b,c}$, Daisuke Kuroda${}^{d}$, Marie Kubota${}^{a}$, Yuta Shirogane${}^{a}$, Shumpei Watanabe${}^{a}$, Kouhei Tsumoto${}^{d,e}$, Daisuke Kohda${}^{b}$, Richard Karl Plemper${}^{f}$, and Yusuke Yanagi${}^{a,2}$

${}^{a}$Department of Virology, Faculty of Medicine, Kyushu University, 812-8582 Fukuoka, Japan; ${}^{b}$Division of Structural Biology, Medical Institute of Bioregulation, Kyushu University, 812-8582 Fukuoka, Japan; ${}^{c}$Biostructural Mechanism Laboratory, RIKEN SPring-8 Center, 679-5148 Hyogo, Japan; ${}^{d}$Department of Bioengineering, School of Engineering, The University of Tokyo, 113-8656 Tokyo, Japan; ${}^{e}$Medical Proteomics Laboratory, The Institute of Medical Science, The University of Tokyo, 108-8639 Tokyo, Japan; and ${}^{f}$Institute for Biomedical Sciences, Georgia State University, Atlanta, GA 30303

Edited by Robert A. Lamb, HHMI and Northwestern University, Evanston, IL, and approved February 2, 2018 (received for review October 31, 2017)

Measles virus (MeV), a major cause of childhood morbidity and mortality, is highly immunotropic and one of the most contagious pathogens. MeV may establish, albeit rarely, persistent infection in the central nervous system, causing fatal and intractable neurodegenerative diseases such as subacute sclerosing panencephalitis and measles inclusion body encephalitis. Recent studies have suggested that particular substitutions in the MeV fusion (F) protein are involved in the pathogenesis by destabilizing the F protein and endowing it with hyperfusogenicity. Here we show the crystal structures of the prefusion MeV-F alone and in complex with the small compound AS-48 or a fusion inhibitor peptide. Notably, these independently developed inhibitors bind the same hydrophobic pocket located at the region connecting the head and stalk of MeV-F, where a number of substitutions in MeV isolates from neurodegenerative diseases are also localized. Since these inhibitors could suppress membrane fusion mediated by most of the hyperfusogenic MeV-F mutants, the development of more effective inhibitors based on the structures may be warranted to treat MeV-induced neurodegenerative diseases.

structure | fusion | measles virus | infection | neurodegenerative disease

the six-helix bundle (6-HB) is formed from two heptad-repeat (HR) domains, HR-A and HR-B. This is a driving force for causing membrane fusion at the cell surface (12–14).

Recent studies have implicated several substitutions in the F-protein ectodomain in the pathogenesis of SSPE and MIBE (15–18). These substitutions destabilize the metastable prefusion conformation of MeV-F, and confer the hyperfusogenic phenotype in SLAM- and nectin-4-expressing cells. More importantly, they even allow the mutant viruses to induce syncytia in cells lacking SLAM and nectin-4, and spread efficiently in human primary neuron cultures and the brains of hamsters and mice. Similarly destabilized, hyperfusogenic F-protein mutants have also been described in several other paramyxoviruses (19–22).

We here determined the structures of the prefusion MeV F protein (MeV-F) alone and bound to an inhibitor compound or an inhibitor peptide. The prefusion MeV-F structures, combined with its postfusion model, provide the structural basis for MeV-mediated membrane fusion, MeV neurovirulence, and inhibition by fusion inhibitors.

### Significance

Measles remains a major cause of childhood morbidity and mortality worldwide, and no licensed therapeutic agents are currently available. Measles virus (MeV) sometimes invades the central nervous system (CNS), causing neurodegenerative diseases several months/years after acute infection. Recently, MeV-induced encephalitis among human immunodeficiency virus (HIV)-infected children has become a great concern in high-HIV-prevalence countries. The tropism of MeV for the CNS is mediated by hyperfusogenic mutations in the MeV fusion (F) protein. Here we show the crystal structures of MeV-F alone and bound to inhibitors. The structures and accompanying cell-based fusion assays with inhibitors provide insight into the molecular mechanism for the inhibition of MeV-mediated fusion, which would help us conquer MeV-induced neurodegenerative diseases.

Author contributions: T.H., Y.F., and Y.Y. designed research; T.H., Y.F., R.M., D. Kuroda, Y.S., and S.W. performed research; T.H., Y.F., R.M., D. Kuroda, M.K., K.T., D. Kohda, R.K.P., and Y.Y. analyzed data; and T.H., Y.F., and Y.Y. wrote the paper.

The authors declare no conflict of interest.

This article is a PNAS Direct Submission.

Published under the [PNAS license](https://www.pnas.org/content/license).

Data deposition: The atomic coordinates for the structures reported in this paper have been deposited in the Protein Data Bank, [www.wwpdb.org](https://www.wwpdb.org) (MeV-F alone, MeV-F–AS-48, and MeV-F–FIP as PDB ID codes 5YXW, 5YZC, and 5YZD, respectively).

${}^{1}$T.H. and Y.F. contributed equally to this work.

${}^{2}$To whom correspondence may be addressed. Email: takaoh@virology.med.kyushu-u.ac.jp or yyanagi@virology.med.kyushu-u.ac.jp.

This article contains supporting information online at [www.pnas.org/lookup/suppl/doi:10.1073/pnas.1718957115/-/DCSupplemental](https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1718957115/-/DCSupplemental).

Results

Structure Determination and Overall Structures. For crystallization of MeV-F, its ectodomain (composed of the head and stalk) trimer was stabilized in the prefusion state by introducing cysteine substitutions into the stalk region, as the known trimeric foldons fibritin and GCNt (23, 24) did not assist its expression. The stabilized MeV-F ectodomain trimer was expressed in *Drosophila* S2 cells, and then crystallized alone and in complex with an MeV entry inhibitor compound, AS-48 (25), or a fusion inhibitor peptide (FIP), Z-D-Phe-Phe-Gly (Z, benzoyloxycarbonyl; D-Phe, D-form Phe) (25, 26). Diffractions to 2.78-, 2.33-, and 2.64-Å resolutions were obtained from single crystals of MeV-F alone, MeV-F–AS-48, and MeV-F–FIP, respectively. The initial experimental phases were determined by the native sulfur single-wavelength anomalous diffraction (S-SAD) phasing method. The structures were refined to an $R_{\text{work}}/R_{\text{free}}$ of 20.2/24.9%, 18.4/22.1%, and 19.1/23.2%, respectively (Fig. 1 and Table S1). The overall organization of the prefusion MeV-F is similar to those of the prefusion F proteins of other paramyxoviruses and a related pneumovirus, the respiratory syncytia virus (RSV) (Fig. 1 and Figs. S1 and S2 A–C).

MeV fusion inhibitors AS-48 and FIP were independently developed. Based on a modeled structure of MeV-F, the inhibitor compound OX-1 was initially designed such that it would bind the MeV-F microdomain near the FP (27), and its chemical improvement led to AS-48 with high stability and low cytotoxicity (25). FIP is a modified peptide originally designed based on the sequence of the $F_0$ cleavage site of Sendai virus (SeV), a paramyxovirus (26). This peptide did not inhibit SeV-mediated fusion but proved to be highly effective in inhibiting MeV entry and fusion, although it does not perfectly match the sequence of MeV-F. Binding of FIP has been shown to make MeV-F thermodynamically stable in its prefusion state, thereby inhibiting MeV-mediated membrane fusion (26). Interestingly, both AS-48 and FIP were found to bind a hydrophobic pocket located at the region connecting the head and stalk of MeV-F, far from the FP, with a stoichiometry of three ligands per MeV-F trimer (Fig. 1C). In crystal structures, the FP (G115 to H138) is located at the upper part of the head and tightly sandwiched between its own protomer and another protomer (Fig. S3). The results suggest that these inhibitors do not directly inhibit the structural change around the FP.

The hydrophobic pocket targeted by these inhibitors corresponds to the N-terminal part of HR-B (Fig. S4 A–C). As a comparison of the prefusion MeV-F structure with a model of its postfusion state shows (Fig. S4 A and B), HR-B is believed to undergo a large structural change during the transition from the prefusion to the postfusion state, interacting with HR-A. Binding of the inhibitors presumably increases the stability of the prefusion MeV-F trimer by bridging the protomers at the hydrophobic pocket, thereby preventing its conformational change to the postfusion form and the formation of 6-HB. The finding that AS-48 restores the intracellular transport and activity of a destabilized mutant MeV-F (28) also supports that the compound acts by stabilizing the prefusion conformation of MeV-F.

MeV-F–Inhibitor Interactions. Although AS-48 (a small-molecule compound) and FIP (a peptide) are chemically distinct, the two inhibitors bind MeV-F in a similar manner. The interactions are primarily hydrophobic, and mainly mediated by aromatic rings in the inhibitors and nonpolar residues of the hydrophobic pocket in MeV-F (Fig. 2). In the MeV-F–AS-48 complex (Fig. 2A and Fig. S4 C–E), the 2-carbamoyl-5-nitrophenyl group of AS-48 forms aromatic-stacking interactions with W311, P353, and P451 of MeV-F, in addition to a hydrophobic interaction with L370 and a hydrogen bond with T369. The benzene ring of AS-48 interacts with hydrophobic residues I452, V459, L463, and I467, together with carbon atoms of G460 and G464 (their main chains) and N462 (its side chain). The acetamide of AS-48 is surrounded by S352 and S453, but no polar interactions are found between the acetamide and the side chains of S352 and S453. The acetamide of AS-48 forms hydrogen bonds with the oxygen atom in the main chain of P451 and the nitrogen atom in the main chain of S453. In the MeV-F–FIP complex (Fig. 2B and Fig. S4 C–E), the benzylloxycarbonyl group and D-Phe of FIP are positioned with respect to MeV-F almost in the same manner as the 2-carbamoyl-5-nitrophenyl

![Figure 1](attachment://image.png)

Fig. 1. Crystal structures of the prefusion MeV-F trimer alone and bound to the inhibitors AS-48 or FIP. (A) Structure of the MeV-F trimer in its prefusion state viewed from the side (*Top*). Each protomer is shown as a ribbon (rainbow colors) or surface diagram (dark and light gray colors). Linear schematic of the MeV-F structure (*Bottom*). The residues visible in the crystal structure (positions 105 to 114 and 482 to 496 are invisible) are represented in the boxes colored in the same rainbow colors as the ribbon diagram of the crystal structure. Disulfide bonds are shown as black lines under the boxes, and glycans are represented as branches on top of the boxes. (B) Structure of the prefusion MeV-F protomer. N and C termini of the $F_1$ and $F_2$ subunits are shown. (C) Structures of the MeV-F trimer in complex with AS-48 (*Left*) or FIP (*Right*). The structures of the prefusion MeV-F trimer are shown as a ribbon diagram, and the inhibitors are shown as a sphere diagram. Color coding of inhibitors: magenta (AS-48) or cyan (FIP); carbon, blue; nitrogen, and red, oxygen. Inhibitory activities of AS-48 and FIP are shown as 50% inhibitory concentrations ($IC_{50}$s).

structures, FIP, but not AS-48, forms a hydrogen bond with E471 of MeV-F (both inhibitors interact with residues at positions 451 and 467) (Figs. 2 and 3C), and the side chain of D471 of CDV-F may be too short to interact with FIP.

**Hyperfusogenic MeV-F–Mediated Membrane Fusion.** The hyperfusogenic phenotype conferred by substitutions in the MeV-F ectodomain is critical for neurovirulence (15–17, 29). However, how these substitutions affect fusion activity is largely unknown. In the MeV-F structures, the residues that confer hyperfusogenicity on MeV-F after their substitutions are clustered in three specific regions (termed sites I, II, and III) (Fig. 4A). Site I is located close to the FP, and includes residues I87 and M94 (15, 30). Site II is situated at the interface of three protomers as well as at the interface of two intramolecular domains (DI and DIII in Fig. S3). S262 is the only residue at this site (15). Site III is the region connecting the head and stalk domains, and contains many residues: L354, L454, T461, N462, and N465 (15, 17, 31). Site III corresponds to HR-B, thought to form the 6-HB structure together with HR-A.

These three sites likely play an important role in maintaining the metastable prefusion structure and/or initiate the conformational changes following the triggering signal from the H protein. The MeV-F structure provides insight into the mechanism for the destabilization of the prefusion conformation by the substitutions. L354M, T461I, N462K, and N465K likely collide with residues in another protomer, whereas S262R and L454W appear to largely collide with residues in the same protomer. I87T and M94V are expected to lose the hydrophobic interaction with the residues and FP in the same protomer, respectively. These microenvironmental changes caused by the substitutions would lead to destabilization and hyperfusogenicity of these mutants.

Among the three sites, site III appears to be the most critical, as many positions are changed in virus isolates from patients with SSPE and MIBE (16). In fact, recombinant MeVs possessing the F protein with single substitutions at site III (e.g., L454W, T461I, and N462K) were shown to induce syncytia in cells lacking the known receptors and spread in human primary neuron cultures and the brains of hamsters and mice (15–18). Furthermore, site III almost overlaps with the binding site for AS-48 and FIP. These inhibitors presumably stabilize the metastable prefusion structure and prevent its conformational changes upon triggering (*Structure Determination and Overall Structures*). Thus, site III of MeV-F may represent a highly critical region regulating MeV-induced membrane fusion.

**Inhibition of Hyperfusogenic MeV-F–Mediated Membrane Fusion.** To further understand the mechanisms of fusion triggering and inhibition, the abilities of AS-48 and FIP to inhibit membrane fusion mediated by hyperfusogenic MeV-F mutants were examined. The wild-type (WT) and hyperfusogenic MeV-F were expressed, together with MeV-H, in cells, and average total sizes of syncytia in the presence of each inhibitor were quantified (Fig. 4 B–D and Fig. S7). Without the inhibitors, the WT MeV-F induced syncytia in Vero cells expressing SLAM or nectin-4 but not in Vero cells lacking these morbillivirus receptors. By contrast, hyperfusogenic F proteins induced syncytia even in Vero cells, and larger ones in Vero cells expressing SLAM or nectin-4. Remarkably, AS-48 and FIP inhibited cell fusion mediated by not only WT but all hyperfusogenic MeV-F proteins except the N462K mutant. Thus, both inhibitors acted on site I and II mutants as well as site III mutants. Substitutions at position 462 have been shown to occur as resistance to AS-48 (28), and FIP also failed to inhibit membrane fusion induced by the N462K mutant (Fig. 4 B–D). The structures of MeV-F–inhibitor complexes clearly revealed that both inhibitors directly interact with N462 of MeV-F (Fig. 2), identifying this residue as a mediator of primary resistance. The MeV-F–AS-48 cocystal structure indicates that another substituted residue, 367, found in an escape mutant (28), also functions through primary resistance.

**Cross-Reactive Inhibition of Morbillivirus F-Mediated Membrane Fusion.** We also examined the abilities of AS-48 and FIP to inhibit CDV- and RPV-induced membrane fusion. AS-48 efficiently inhibited membrane fusion mediated by CDV and RPV glycoproteins, whereas FIP suppressed membrane fusion induced by RPV but not CDV glycoproteins (Fig. 3A and Fig. S6). The sequence identity among MeV, RPV, and CDV F proteins mapped on the MeV-F structure shows that amino acid residues are highly conserved around the binding site for AS-48 and FIP, where only three positions, 451, 467, and 471, are different among the three viruses (Fig. 3B and C). The MeV and RPV F proteins differ only at position 467, and both inhibitors efficiently inhibited RPV-induced membrane fusion. By contrast, all three positions are different between MeV and CDV. In the MeV-F–inhibitor complex

Hashiguchi et al.

Fig. 3. Structural basis for inhibition of morbillivirus entry by inhibitors. (A) Inhibition of cell–cell fusion mediated by RPV (Left) or CDV (Right) in CHO cells expressing cow or dog SLAM, respectively. The cells were transfected with expression plasmids encoding RPV or CDV H and F proteins and incubated with or without inhibitors for 24 h. The average sizes of syncytia in the presence or absence of inhibitors were quantified by using the hybrid cell count system (Keyence). Data are the mean ± SD of the six images, and are representative of three independently performed experiments. (B) The identical amino acid residues among MeV, RPV, and CDV are mapped on the prefusion MeV-F structure and colored in red (Top). The different amino acid residues among MeV, RPV, and CDV at inhibitor-binding sites are summarized (Bottom). (C) Detailed views of inhibitor-binding sites.

**Discussion**

In this study, we determined the structures of the prefusion form of MeV-F. Although the known trimeric foldons fibritin and GCNt (23, 24) did not assist MeV-F expression, the introduction of cysteine substitutions into the stalk region allowed the prefusion MeV-F trimer to be stabilized and expressed. We previously engineered disulfide bridges to introduce intersubunit covalent links into the prefusion MeV-F, and demonstrated its surface expression as stable DTT-sensitive trimers and reversible inhibition of its fusion activity (32). Among the ~150 different versions of MeV-F we engineered for the present crystal structural study, the introduction of four cysteine substitutions around the C-terminal region of HR-B was the most suitable. The engineering of disulfide bonds to stabilize the prefusion conformation may have a broad application for determining the structures of unstable glycoproteins, as previously reported in RSV (33), HIV (34, 35) and Lassa virus (36).

The structures of the prefusion MeV-F revealed detailed molecular mechanisms by which the two inhibitors AS-48 and FIP inhibit membrane fusion mediated by MeV, CDV, and RPV F proteins. These two inhibitors bind the same hydrophobic pocket of MeV-F, engaging many common residues. This inhibitor-binding site and its vicinity (site III) are also critical for fusion triggering, and accommodate many substitutions that confer the hyperfusogenic phenotype on MeV-F. Although these inhibitors possess only intermediate antiviral potency in cultured cells, with a low-to-submicromolar range of 50% inhibitory concentrations, the structures should provide templates for the development of more effective entry inhibitors targeting this highly vulnerable site on MeV-F.

The understanding of the mechanisms by which inhibitors prevent virus-mediated membrane fusion is important for the

Fig. 4. Characterization of hyperfusogenic MeV-F mutants. (A) Amino acid residues (sphere diagrams) whose substitutions confer hyperfusogenicity on MeV-F are clustered in three sites. Each protomer is colored as in Fig. 2. (B–D) Inhibition of cell–cell fusion mediated by hyperfusogenic mutants in Vero/SLAM (B), Vero/nectin-4 (C), and Vero cells (D). The cells were transfected with the expression plasmid encoding WT or mutant F proteins, together with that encoding the WT H protein. At 24 h (Vero/SLAM and Vero/nectin-4) or 96 h (Vero) posttransfection, the cells were stained with a Giemsa solution. Syncytia were quantified as in Fig. 3. Data are the mean ± SD of the six images, and are representative of three independently performed experiments.

AS-48
![](https://i.imgur.com/1234567.png)

FIP
Z-D-Phe-Phe-Gly  
(Z: Benzylloxycarbonyl)
![](https://i.imgur.com/890abcde.png)

JNJ-2408068
![](https://i.imgur.com/fedcba98.png)

Arbidol
![](https://i.imgur.com/7654321.png)

toremifene
![](https://i.imgur.com/gfedcba.png)

Fig. 5. Comparison of the inhibitor-binding sites and the inhibition mechanisms among class I viral fusion proteins. (Top) Structures of inhibitors against MeV-F, RSV-F, Flu-HA, and EBOV-GP. (Middle) Structures of the class I viral fusion proteins bound to respective inhibitors. The trimeric subunits containing the FP/fusion loop are colored in yellow, blue, and green, and other subunits are in gray for all viral proteins. (Bottom) Inhibitor-binding sites and inhibition mechanisms of class I viral fusion proteins.

| Binding site | Inhibition mechanism |
|--------------|----------------------|
| F1           | far from FP          |
|              | stabilization        |

| F1           | close to FP          |
|              | stabilization        |

| HA1 and HA2  | far from FP          |
|              | stabilization        |

| GP1 and GP2  | close to FP          |
|              | destabilization      |

future development and improvement of antiviral drugs. To gain insight into the mode of action of fusion inhibitors, the structures of MeV-F-inhibitor complexes were compared with those of other class I viral fusion protein-inhibitor complexes: RSV F-JNJ-2408068 (37), influenza A virus (Flu) hemagglutinin (HA)-Arbidol (38), and ebolavirus (EBOV) glycoprotein (GP)-toremifene (39) (Fig. 5). All inhibitors commonly bind the hydrophobic sites of the respective viral proteins. While the inhibitors stabilize MeV-F, RSV-F, and Flu-HA proteins, toremifene destabilizes the EBOV-GP protein. The inhibitor-binding sites are near the FP in RSV-F and EBOV-GP proteins but far from it in MeV-F and Flu-HA proteins. Interestingly, Arbidol binds the region connecting the head and stalk of Flu HA, with a stoichiometry of three ligands per HA trimer. Thus, AS-48/FIP and Arbidol may inhibit membrane fusion in a similar manner, although their structures are totally different. In fact, FIP was reported to exhibit anti-Flu activity (26).

For the functional activation of paramyxovirus F proteins, the precursor F₀ must be cleaved into F₁ and F₂ subunits by cellular proteases, furin for MeV and parainfluenza virus 5 (PIV5) and cathepsin L for Nipah virus (NiV) and Hendra virus (HeV) (40). The atomic-resolution structure of the activated, cleaved form (F₁/F₂) has been reported for PIV5-F (41). The F₀ structure is determined for PIV5-F (24), NiV-F (42), and HeV-F (43). The F protein of the related pneumovirus RSV is cleaved at two sites by furin (40), and its structure is reported as the activated form (23). Compared with these F proteins, the structure of MeV-F shows higher similarity to those of NiV-F and HeV-F, despite their different protease requirements (Fig. S2 A–C).

Although the overall structure of MeV-F is similar to those of other paramyxoviruses and RSV, significant differences exist. The first difference is found in the alpha helix (α6) immediately following the FP. In the F proteins of PIV5, NiV, and HeV, this alpha helix is formed together with the C terminus of the FP (Fig. S2C). However, the α6-helix of MeV-F is split from the C terminus of the FP and turns outward into the solvent (Fig. S2C). The incompact configuration of the α6-helix of MeV-F may partly contribute to its instability as the prefusion form. The second difference is observed in the protease cleavage site. The C terminus of the cleaved MeV-F₂ has a short helix (α3) structure, which packs against the outside of the FP N terminus of MeV-F₁ (Fig. S2D). By contrast, the cleavage sites exhibit a loop structure in PIV5-F (41), NiV-F (42), and HeV-F (43), and RSV-F (33) is missing the corresponding helix or loop structure (Fig. S2D). The similar alpha-helical packing against the fusion loop was reported in the Marburg virus glycoprotein (44). The α3-helix of the cleaved MeV-F₂ may act to prevent an unnecessary conformational change, as proposed for the Marburg virus glycoprotein.

Importantly, the two MeV inhibitors, AS-48 and FIP, could suppress membrane fusion mediated by most of the destabilized, hyperfusogenic MeV-F mutants (Fig. 4 and Fig. S7). In addition to the mutations in the F-protein ectodomain, those causing the defect of the M protein or producing altered cytoplasmic tails of the F protein are often detected in MeVs from patients with SSPE and MIBE (45, 46). These mutations were also shown to endow MeV with hyperfusogenicity and facilitate MeV spread in the brains of genetically modified mice (47). In all these hyperfusogenic mutations, the lower threshold for triggering membrane fusion in the brain may lead to MeV-induced neurodegenerative diseases. The results indeed warrant the future development of fusion inhibitors to treat MeV-induced CNS infections. However, a recent study has reported several resistant mutants against AS-48 and/or FIP (48), like those against RSV entry inhibitors (49). The mutated positions are located around the AS-48– and FIP-binding sites found in the present study. Thus, the structure-based design and combined use of different inhibitors would be required to overcome the problem of potential resistance by hyperfusogenic MeVs from patients with SSPE and MIBE. Antiviral fusion inhibitors may also prove useful to treat unvaccinated individuals exposed to WT MeV in our current efforts to eliminate measles with vaccination.

**Materials and Methods**

AS-48 was synthesized as reported previously (25). FIP was obtained commercially (Virus Replication Inhibiting Peptide; Peptide Institute). Both inhibitors were verified to exhibit the reported properties (25, 26). Vero/SLAM (50), Vero/nectin-4 (51), and Vero cells were maintained in DMEM (Wako) supplemented with 10% FBS (Sigma), 0.075% NaHCO₃, 2 mM Hepes, and penicillin/streptomycin (Gibco) in a 37 °C humidified chamber with 5% CO₂. CHO/CowSLAM and CHO/DogSLAM cells (52) were maintained in RPMI medium supplemented with 10% FBS, 0.075% NaHCO₃, 2 mM Hepes, 1.7 mM L-proline, and penicillin/streptomycin in a 37 °C humidified chamber with 5% CO₂. Drosophila Schneider S2 cells were maintained in complete Schneider's medium (Lonza) supplemented with 10% (vol/vol) FBS in a 28 °C incubator. The expression, purification, crystallization, and structure determination of proteins were carried out as previously described (44, 53) with some modifications. Cell-cell fusions were quantified by using the hybrid cell count system (Keyence). More detailed information is provided in *SI Materials and Methods*.

**ACKNOWLEDGMENTS.** We thank the beamline staff of the Photon Factory and T. Yoshitomi for technical help. This study was supported by Ministry of Education, Culture, Sports, Science, and Technology KAKENHI Grants 17K19562 (to T.H.) and 24115005 (to Y.Y.), The Uehara Memorial Foundation (Y.Y.), Japan Agency for Medical Research and Development (AMED) J-PRIDE Grant JP17fm0208022h (to T.H.), Public Health Service Grants AI071002 (NIH/National Institute of Allergy and Infectious Diseases) and HD079327 (NIH/National Institute of Child Health and Development) (to R.K.P.), and the AMED Platform Project for Supporting Drug Discovery and Life Science Research (Basis for Supporting Innovative Drug Discovery and Life Science Research).
1. Griffin DE (2013) Measles virus. Fields Virology, eds Knipe DM, et al. (Lippincott Williams & Wilkins, Philadelphia), 6th Ed, Vol 1, pp 1042–1069.
2. Rima BK, Duprex WP (2006) Morbilliviruses and human disease. *J Pathol* 208:199–214.
3. Bellini WJ, et al. (2005) Subacute sclerosing panencephalitis: More cases of this fatal disease are prevented by measles immunization than was previously recognized. *J Infect Dis* 192:1686–1693.
4. Reuter D, Schneider-Schaulies J (2010) Measles virus infection of the CNS: Human disease, animal models, and approaches to therapy. *Med Microbiol Immunol (Berl)* 199:261–271.
5. Tatsuo H, Ono N, Tanaka K, Yanagi Y (2000) SLAM (CDw150) is a cellular receptor for measles virus. *Nature* 406:893–897.
6. Noyce RS, et al. (2011) Tumor cell marker PVRL4 (nectin 4) is an epithelial cell receptor for measles virus. *PLoS Pathog* 7:e1002240.
7. Mühlebach MD, et al. (2011) Adherens junction protein nectin-4 is the epithelial receptor for measles virus. *Nature* 480:530–533.
8. McQuaid S, Cosby SL (2002) An immunohistochemical study of the distribution of the measles virus receptors, CD46 and SLAM, in normal human tissues and subacute sclerosing panencephalitis. *Lab Invest* 82:403–409.
9. Reymond N, et al. (2001) Nectin4/PRR4, a new afadin-associated member of the nectin family that trans-interacts with nectin1/PRR1 through V domain interaction. *J Biol Chem* 276:43205–43215.
10. Hashiguchi T, et al. (2007) Crystal structure of measles virus hemagglutinin provides insight into effective vaccines. *Proc Natl Acad Sci USA* 104:19535–19540.
11. Hashiguchi T, et al. (2011) Structure of the measles virus hemagglutinin bound to its cellular receptor SLAM. *Nat Struct Mol Biol* 18:135–141.
12. Plemper RK, Brindley MA, Iorio RM (2011) Structural and mechanistic studies of measles virus illuminate paramyxovirus entry. *PLoS Pathog* 7:e1002058.
13. Plattet P, Alves L, Herren M, Aguilar HC (2016) Measles virus fusion protein: Structure, function and inhibition. *Viruses* 8:112.
14. Jardetzky TS, Lamb RA (2014) Activation of paramyxovirus membrane fusion and virus entry. *Curr Opin Virol* 5:24–33.
15. Watanabe S, et al. (2013) Mutant fusion proteins with enhanced fusion activity promote measles virus spread in human neuronal cells and brains of suckling hamsters. *J Virol* 87:2648–2659.
16. Watanabe S, et al. (2015) Measles virus mutants possessing the fusion protein with enhanced fusion activity spread effectively in neuronal cells, but not in other cells, without causing strong cytopathology. *J Virol* 89:2710–2717.
17. Jurgens EM, et al. (2015) Measles fusion machinery is dysregulated in neuro-pathogenic variants. *MBio* 6:e02528-14.
18. Ayata M, et al. (2010) The F gene of the Osaka-2 strain of measles virus derived from a case of subacute sclerosing panencephalitis is a major determinant of neurovirulence. *J Virol* 84:11189–11199.
19. Horvath CM, Lamb RA (1992) Studies on the fusion peptide of a paramyxovirus fusion glycoprotein: Roles of conserved residues in cell fusion. *J Virol* 66:2443–2455.
20. Sergel TA, McGinnes LW, Morrison TG (2000) A single amino acid change in the Newcastle disease virus fusion protein alters the requirement for HN protein in fusion. *J Virol* 74:5101–5107.
21. Terrier O, et al. (2008) Characterization of naturally occurring parainfluenza virus type 2 (hPIV-2) variants. *J Clin Virol* 43:86–92.
22. Plattet P, et al. (2007) Signal peptide and helical bundle domains of virulent canine distemper virus fusion protein restrict fusogenicity. *J Virol* 81:11413–11425.
23. McLellan JS, et al. (2013) Structure of RSV fusion glycoprotein trimer bound to a prefusion-specific neutralizing antibody. *Science* 340:1113–1117.
24. Yin H-S, Wen X, Paterson RG, Lamb RA, Jardetzky TS (2006) Structure of the parainfluenza virus 5F protein in its metastable, prefusion conformation. *Nature* 439:38–44.
25. Plemper RK, et al. (2005) Design of a small-molecule entry inhibitor with activity against primary measles virus strains. *Antimicrob Agents Chemother* 49:3755–3761.
26. Richardson CD, Scheid A, Choppin PW (1980) Specific inhibition of paramyxovirus and myxovirus replication by oligopeptides with amino acid sequences similar to those at the N-termini of the F1 or HA2 viral polypeptides. *Virology* 105:205–222.
27. Plemper RK, et al. (2004) A target site for template-based design of measles virus entry inhibitors. *Proc Natl Acad Sci USA* 101:5628–5633.
28. Doyle J, et al. (2006) Two domains that control prefusion stability and transport competence of the measles virus fusion protein. *J Virol* 80:1524–1536.
29. Shirogane Y, Watanabe S, Yanagi Y (2012) Cooperation between different RNA virus genomes produces a new phenotype. *Nat Commun* 3:1235.
30. Plemper RK, Compans RW (2003) Mutations in the putative HR-C region of the measles virus F2 glycoprotein modulate syncytium formation. *J Virol* 77:4181–4190.
31. Satoh Y, et al. (2017) A residue located at the junction of the head and stalk regions of measles virus fusion protein regulates membrane fusion by controlling conformational stability. *J Gen Virol* 98:143–154.
32. Lee JK, Prussia A, Snyder JP, Plemper RK (2007) Reversible inhibition of the fusion activity of measles virus F protein by an engineered intersubunit disulfide bridge. *J Virol* 81:8821–8826.
33. Stewart-Jones GB, et al. (2015) A cysteine zipper stabilizes a pre-fusion F glycoprotein vaccine for respiratory syncytial virus. *PLoS One* 10:e0128779.
34. Julien JP, et al. (2013) Crystal structure of a soluble cleaved HIV-1 envelope trimer. *Science* 342:1477–1483.
35. Pancera M, et al. (2014) Structure and immune recognition of trimeric pre-fusion HIV-1 Env. *Nature* 514:455–461.
36. Hastie KM, et al. (2017) Structural basis for antibody-mediated neutralization of Lassa virus. *Science* 356:923–928.
37. Battles MB, et al. (2016) Molecular mechanism of respiratory syncytial virus fusion inhibitors. *Nat Chem Biol* 12:87–93.
38. Kadam RU, Wilson IA (2017) Structural basis of influenza virus fusion inhibition by the antiviral drug Arbidol. *Proc Natl Acad Sci USA* 114:206–214.
39. Zhao Y, et al. (2016) Toremifene interacts with and destabilizes the Ebola virus glycoprotein. *Nature* 535:169–172.
40. Lamb RA, Parks GD (2013) Paramyxoviridae: The viruses and their replication. *Fields Virology*, eds Knipe DM, et al. (Lippincott Williams & Wilkins, Philadelphia), 6th Ed, Vol 1, pp 957–995.
41. Welch BD, et al. (2012) Structure of the cleavage-activated prefusion form of the parainfluenza virus 5 fusion protein. *Proc Natl Acad Sci USA* 109:16672–16677.
42. Xu K, et al. (2015) Crystal structure of the pre-fusion Nipah virus fusion glycoprotein reveals a novel hexamer-of-trimers assembly. *PLoS Pathog* 11:e1005322.
43. Wong JJ, Paterson RG, Lamb RA, Jardetzky TS (2016) Structure and stabilization of the Hendra virus F glycoprotein in its prefusion form. *Proc Natl Acad Sci USA* 113:1056–1061.
44. Hashiguchi T, et al. (2015) Structural basis for Marburg virus neutralization by a cross-reactive human antibody. *Cell* 160:904–912.
45. Cattaneo R, et al. (1988) Biased hypermutation and other genetic changes in defective measles viruses in human brain infections. *Cell* 55:255–265.
46. Schmid A, et al. (1992) Subacute sclerosing panencephalitis is typically characterized by alterations in the fusion protein cytoplasmic domain of the persisting measles virus. *Virology* 188:910–915.
47. Cathomen T, et al. (1998) A matrix-less measles virus is infectious and elicits extensive cell fusion: Consequences for propagation in the brain. *EMBO J* 17:3899–3908.
48. Ha MN, et al. (2017) Mutations in the fusion protein of measles virus that confer resistance to the membrane fusion inhibitors carbobenzoxy-D-Phe-L-Phe-Gly and 4-nitro-2-phenylacetyl amino-benzamide. *J Virol* 91:e01026-17.
49. Yan D, et al. (2014) Cross-resistance mechanism of respiratory syncytial virus against structurally diverse entry inhibitors. *Proc Natl Acad Sci USA* 111:E3441–E3449.
50. Ono N, et al. (2001) Measles viruses on throat swabs from measles patients use signaling lymphocytic activation molecule (CDw150) but not CD46 as a cellular receptor. *J Virol* 75:4399–4401.
51. Otsuki N, et al. (2013) Canine distemper virus with the intact C protein has the potential to replicate in human epithelial cells by using human nectin 4 as a receptor. *Virology* 435:485–492.
52. Tatsuo H, Ono N, Yanagi Y (2001) Morbilliviruses use signaling lymphocyte activation molecules (CD150) as cellular receptors. *J Virol* 75:5842–5850.
53. Kubota M, et al. (2016) Trisaccharide containing α2,3-linked sialic acid is a receptor for mumps virus. *Proc Natl Acad Sci USA* 113:11579–11584.
54. Takeda M, et al. (2000) Recovery of pathogenic measles virus from cloned cDNA. *J Virol* 74:6643–6647.
55. Takeda M, et al. (2005) Long untranslated regions of the measles virus M and F genes control virus replication and cytopathogenicity. *J Virol* 79:14346–14354.
56. Mucs D, Bryce RA (2013) The application of quantum mechanics in structure-based drug design. *Expert Opin Drug Discov* 8:263–276.
57. Kabsch W (2010) XDS. *Acta Crystallogr D Biol Crystallogr* 66:125–132.
58. Hendrickson WA, Teeter MM (1981) Structure of the hydrophobic protein crambin determined directly from the anomalous scattering of sulphur. *Nature* 290:107–113.
59. Terwilliger TC, et al. (2009) Decision-making in structure solution using Bayesian estimates of map quality: The PHENIX AutoSol wizard. *Acta Crystallogr D Biol Crystallogr* 65:582–601.
60. Adams PD, et al. (2010) PHENIX: A comprehensive Python-based system for macromolecular structure solution. *Acta Crystallogr D Biol Crystallogr* 66:213–221.
61. Emsley P, Cowtan K (2004) Coot: Model-building tools for molecular graphics. *Acta Crystallogr D Biol Crystallogr* 60:2126–2132.
62. Chaudhury S, Lyskov S, Gray JJ (2010) PyRosetta: A script-based interface for implementing molecular modeling algorithms using Rosetta. *Bioinformatics* 26:689–691.
63. Alford RF, et al. (2017) The Rosetta all-atom energy function for macromolecular modeling and design. *J Chem Theory Comput* 13:3031–3048.
64. Roberts E, Eargle J, Wright D, Luthy-Schulten Z (2006) MultiSeq: Unifying sequence and structure data for evolutionary analysis. *BMC Bioinformatics* 7:382.
65. Katoh K, Standley DM (2013) MAFFT multiple sequence alignment software version 7: Improvements in performance and usability. *Mol Biol Evol* 30:772–780.
66. O’Donoghue P, Luthy-Schulten Z (2005) Evolutionary profiles derived from the QR factorization of multiple structural alignments gives an economy of information. *J Mol Biol* 346:875–894.
67. Russell RB, Barton GJ (1992) Multiple protein sequence alignment from tertiary structure comparison: Assignment of global and residue confidence levels. *Proteins* 14:309–323.
68. Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. *Nucleic Acids Res* 22:4673–4680.