
Report

**Neuron**

**ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism**

### Graphical Abstract

| Mouse model | ATXN1-CIC complex status | Cerebellar pathology |
|-------------|--------------------------|----------------------|
| PolyQ-expanded ATXN1 | Toxic Gain of function | Aggregated ATXN1 |
| PolyQ-expanded ATXN1 unable to bind CIC | Loss of toxicity | No ATXN1-CIC |
| Absence of ATXN1-CIC complex | Loss of function | |

### Authors

Maxime W.C. Rousseaux,  
Tyler Tschumperlin, Hsiang-Chih Lu, ...,  
Roy V. Sillitoe, Huda Y. Zoghbi,  
Harry T. Orr  

### Correspondence

hzoghbi@bcm.edu (H.Y.Z.),  
orrx002@umn.edu (H.T.O.)  

### In Brief

Rousseaux, Tschumperlin, Lu, and colleagues show that formation of the ATXN1-CIC complex is critical for polyQ-expanded ATXN1-mediated toxicity. They find that this complex mediates its effects through a gain-of-function mechanism in the cerebellum of SCA1 mice and SCA1 patient-derived neurons.

### Highlights

- Toxicity of polyQ-expanded ATXN1 in the cerebellum requires interaction with CIC
- CIC and the ATXN1-CIC complex are dispensable for cerebellar function
- Expression profiling reveals gain of function of CIC upon ATXN1 polyQ expansion
- Genetic signatures are disrupted in neurons derived from SCA1 patients

---

Rousseaux et al., 2018, Neuron 97, 1–9  
March 21, 2018 © 2018 Elsevier Inc.  
https://doi.org/10.1016/j.neuron.2018.02.013

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

**Neuron**

**Report**

# ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism

Maxime W.C. Rousseaux,$^{1,2,11}$ Tyler Tschumperlin,$^{3,4,11}$ Hsiang-Chih Lu,$^{1,5,11}$ Elizabeth P. Lackey,$^{1,6,7}$ Vitaliy V. Bondar,$^{1,2}$ Ying-Wooi Wan,$^{1,2}$ Qiumin Tan,$^{1,2}$ Carolyn J. Adamski,$^{1,2}$ Jillian Friedrich,$^{3,4}$ Kirk Twaroski,$^{8}$ Weili Chen,$^{8}$ Jakub Tolar,$^{2}$ Christine Henzler,$^{9}$ Ajay Sharma,$^{1,2}$ Aleksandar Bajić,$^{1,2}$ Tao Lin,$^{1,7}$ Lisa Duvick,$^{3,4}$ Zhandong Liu,$^{1,8}$ Roy V. Sillitoe,$^{1,6,7}$ Huda Y. Zoghbi,$^{1,2,5,6,10,*}$ and Harry T. Orr$^{3,4,12,*}$

$^{1}$Jan and Dan Duncan Neurological Research Institute at Texas Children’s Hospital, Baylor College of Medicine, Houston, TX 77030, USA  
$^{2}$Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA  
$^{3}$Institute for Translational Neuroscience, University of Minnesota, Minneapolis, MN 55455, USA  
$^{4}$Department of Laboratory Medicine and Pathology, University of Minnesota, Minneapolis, MN 55455, USA  
$^{5}$Program in Developmental Biology, Baylor College of Medicine, Houston, TX 77030, USA  
$^{6}$Department of Neuroscience, Baylor College of Medicine, Houston, TX 77030, USA  
$^{7}$Department of Pathology and Immunology, Baylor College of Medicine, Houston, TX 77030, USA  
$^{8}$Department of Pediatrics, Stem Cell Institute, University of Minnesota, Minneapolis, MN 55455, USA  
$^{9}$RISS Bioinformatics, Minnesota Supercomputing Institute, University of Minnesota, Minneapolis, MN 55455, USA  
$^{10}$Howard Hughes Medical Institute, Houston, TX 77030, USA  
$^{11}$These authors contributed equally  
$^{12}$Lead Contact  

*Correspondence:* [hzoghbi@bcm.edu](mailto:hzoghbi@bcm.edu) (H.Y.Z.), [orrx002@umn.edu](mailto:orrx002@umn.edu) (H.T.O.)  
[https://doi.org/10.1016/j.neuron.2018.02.013](https://doi.org/10.1016/j.neuron.2018.02.013)

## SUMMARY

Polyglutamine (polyQ) diseases are caused by expansion of translated CAG repeats in distinct genes leading to altered protein function. In spinocerebellar ataxia type 1 (SCA1), a gain of function of polyQ-expanded ataxin-1 (ATXN1) contributes to cerebellar pathology. The extent to which cerebellar toxicity depends on its cognate partner capicua (CIC), versus other interactors, remains unclear. It is also not established whether loss of the ATXN1-CIC complex in the cerebellum contributes to disease pathogenesis. In this study, we exclusively disrupt the ATXN1-CIC interaction *in vivo* and show that it is at the crux of cerebellar toxicity in SCA1. Importantly, loss of CIC in the cerebellum does not cause ataxia or Purkinje cell degeneration. Expression profiling of these gain- and loss-of-function models, coupled with data from iPSC-derived neurons from SCA1 patients, supports a mechanism in which gain of function of the ATXN1-CIC complex is the major driver of toxicity.

## INTRODUCTION

Adult-onset neurodegenerative diseases are often characterized by the abnormal accumulation of disease-driving proteins. Mechanisms by which the accumulated protein causes pathology are unclear. Polyglutamine (polyQ) disorders represent monogenic proteinopathies, in which a polyQ-expanded tract in a protein causes neuronal demise. In spinocerebellar ataxia type 1 (SCA1), polyQ-expanded Ataxin-1 (ATXN1, encoded by *ATXN1*) leads to cerebellar and brain stem degeneration ([Orr et al., 1993](https://doi.org/10.1038/ng0193-15)). Individuals carrying uninterrupted CAG expansions in *ATXN1*, ranging from 39 to 82 repeats, present with ataxia, speaking and swallowing difficulties, and muscle weakness and succumb to disease within two decades of diagnosis. While there is a clear link between polyQ length and disease severity and onset ([Orr et al., 1993](https://doi.org/10.1038/ng0193-15)), detailed molecular mechanism(s) that underlie polyQ-expanded ATXN1 toxicity remain unknown.

Previous studies found that ATXN1 natively forms a transcriptional repressor complex with Capicua (CIC), as well as its paralog ATXN1L: the “ATXN1-CIC complex” ([Kim et al., 2013](https://doi.org/10.1016/j.cell.2013.02.030); [Lam et al., 2006](https://doi.org/10.1016/j.molcel.2006.01.030); [Lee et al., 2011](https://doi.org/10.1016/j.cell.2011.07.029)). The interaction of polyQ-expanded ATXN1 with CIC is believed to mediate some of the toxicity based on data showing that the ATXN1-CIC complex becomes hypermorphic upon the polyQ expansion in ATXN1 ([Fryer et al., 2011](https://doi.org/10.1016/j.cell.2011.07.029)). Furthermore, reducing CIC levels in an SCA1 mouse model rescues pathology and corrects the gain-of-function transcriptional phenotype. Loss-of-function studies of the ATXN1-CIC complex in the cortex and hypothalamus ([Lu et al., 2017](https://doi.org/10.1016/j.neuron.2017.03.020)), as well as the periphery ([Park et al., 2017](https://doi.org/10.1016/j.neuron.2017.03.020); [Simón-Carrasco et al., 2017](https://doi.org/10.1016/j.neuron.2017.03.020); [Tan et al., 2018](https://doi.org/10.1016/j.neuron.2018.01.020)), suggest that this complex is critical for normal cellular functions. Collectively, these findings lead to two unanswered questions. (1) Is cerebellar pathology exclusively driven by the ATXN1-CIC complex or are there other interactors and/or modes of toxicity that contribute to SCA1 pathogenesis? (2) Does loss of the ATXN1-CIC complex contribute to disease?

Studying the gain versus loss of function of ATXN1-CIC complex formation *in vivo* offers an opportunity to parse out a specific

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

mode of toxicity in SCA1. Specifically, whether other mechanisms—including RNA-mediated toxicity, repeat associated non-ATG (RAN) translation, and/or additional interacting partners—mediate cerebellar pathology in SCA1 remains unknown (Wojciechowska et al., 2014; Zoghbi and Orr, 2009). Given recent loss-of-function evidence suggesting that the ATXN1-CIC complex plays important developmental roles (Lu et al., 2017), formally testing its contribution to cerebellar function is critical for a better understanding of SCA1 pathogenesis and for the design of therapeutics.

**RESULTS**

**Mutations in the ATXN1-CIC Interface Disrupt the Complex In Vivo and Prevent SCA1 Pathogenesis**

We previously found that haploinsufficiency of CIC suppresses disease phenotypes in mice (Fryer et al., 2011). Mutating two crucial residues in ATXN1, valine 591 (V591A) and serine 602 (S602D), in its AXH domain was sufficient to abolish ATXN1-CIC complex formation in vitro (Kim et al., 2013). Mice that overexpress polyQ-expanded ATXN1 under the control of a Purkinje cell (PC)-specific promoter exhibit the cardinal features of SCA1, including motor performance deficits and PC degeneration (Burrig et al., 1995). We harnessed the information from structural studies to generate a new polyQ-expanded ATXN1 transgenic line that is CIC-interaction deficient (ATXN1[82Q] V591A;S602D; Figure S1A).

First, we asked whether the V591A; S602D mutation in the AXH domain of ATXN1 prevented its association with CIC in vivo. We immunoprecipitated CIC from wild-type (WT), ATXN1[82Q], and ATXN1[82Q]V591A;S602D mice and tested whether ATXN1 was complexed with CIC. CIC could bind endogenous mouse ATXN1 in all cases but could bind to polyQ-expanded ATXN1 only in the ATXN1[82Q] mice (Figure 1A). Previous work demonstrated that protein levels, localization, and developmental timing of ATXN1 transgene expression in PCs affect the severity of disease in adult mice (Burrig et al., 1995; Serra et al., 2006). We found that there were no differences in transgene expression (Figure 1B) and that these mice exhibited similar developmental timing of transgene transcription and nuclear localization (Figures S1E–S1G).

Next, we measured motor performance of ATXN1[82Q] V591A;S602D mice via the rotarod test. At 6, 12, and 20 weeks of age, ATXN1[82Q] mice were significantly impaired on the rotarod compared to WT animals (Figure 1C). In contrast, ATXN1[82Q]V591A;S602D were indistinguishable from WT mice at each age tested. We examined whether the observed motor performance rescue was due to improved cerebellar morphology. Consistent with previous reports, mice expressing ATXN1[82Q] showed significant loss of molecular layer thickness as early as 6 weeks, which progressively degenerated over time (Figure 1D; Burrig et al., 1995). Conversely, there was a striking absence of neurodegeneration in ATXN1 [82Q]V591A;S602D cerebella up to 52 weeks of age (Figure 1D). Thus, disrupting the interaction between ATXN1 [82Q] and CIC is sufficient to eliminate toxicity in the mouse cerebellum.

**Loss of CIC and a Corresponding Disruption of the ATXN1-CIC Complex in the Cerebellum Does Not Cause Neurodegenerative Phenotypes**

Previous work from our lab and others suggested that part of the cerebellar phenotypes occurring in SCA1 may be due to a partial loss of function of the ATXN1-CIC complex (Crespo-Barreto et al., 2010; Fryer et al., 2011). However, these studies are difficult to interpret due to the functionally redundant ATXN1 homolog ATXN1L that compensates for its loss in vivo (Lee et al., 2011). Due to the important roles of this protein complex in the developing forebrain (Lu et al., 2017) and the immune system (Park et al., 2017; Simón-Carrasco et al., 2017; Tan et al., 2018), its native cerebellar function remains clouded. To formally test whether a loss of function of the ATXN1-CIC complex accounts for part of the toxicity observed in SCA1, we deleted members of this complex from the mouse cerebellum during development using En1-Cre (Li et al., 2002). Since ATXN1 and ATXN1L are functionally redundant and their loss is phenocopied by CIC, we generated En1-Cre; Atxn1−/−; Atxn1flox/flox (“Cerebellar Atxn1-1l KO”; note: the Atxn1 allele is a null) and En1-Cre; Cicflox/flox (“Cerebellar Cic KO”) mouse lines to determine the effect of disrupting this complex in development. We examined the knockout efficiency of the ATXN1-CIC complex in these mice by western blot, immunofluorescence, and qPCR, confirming their effective deletion (Figure 2A; Figures S2A and S2B). Moreover, we noted that ATXN1-ATXN1L stabilized CIC, whereas CIC loss did not result in a dramatic destabilization of ATXN1 or ATXN1L.

SCA1 mice exhibit motor behavior deficits, neurodegeneration, and early lethality (Burrig et al., 1995; Watase et al., 2002), we therefore asked whether loss of these complex members would result in motor deficits reminiscent of SCA1. In stark contrast to SCA1 mice, loss of function of ATXN1-CIC did not cause SCA1-like motor phenotypes when tested with the open field and rotarod assays. In fact, the cerebellar CIC knockout mice were slightly hyperactive and stayed significantly longer on the rotarod (Figures 2B and 2C) and lived well over 1 year of age. Even at 1 year of age, ATXN1-CIC complex knockout mice demonstrated normal cerebellar morphology (Figure 2D; Figure S2C).

A possible explanation for the lack of phenotype in the cerebellar ATXN1-CIC complex knockout mice is developmental compensation. To test this, we disrupted this complex in adult mice. Given that loss of Atxn1/1l or Cic phenocopy each other in all paradigms tested (Lee et al., 2011; Lu et al., 2017), we focused on the adult loss of Cic. Adult cerebellar conditional knockouts of Cic were obtained using UBC-CreERT2 (Ruzankina et al., 2007) and designated UBC-CreERT2; Cicflox/flox [“Adult Cic KO”]. To excise the floxed Cic allele, we administered a series of tamoxifen injections to 8- to 12-week-old adult mice (Sztainberg et al., 2015). Examination of CIC protein levels by western blot confirmed that it was effectively knocked out (Figure 2A, right panel). These adult knockout mice had normal exploratory behavior in the open field when compared to controls and did not exhibit any motor impairment on the rotarod 8 weeks following the last injection (Figure 2B, right panels). Cerebellar histological analysis of UBC-CreERT2; Cicflox/flox mice compared to their littermates failed to reveal any changes in

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

$\square$ Control $\quad \square$ Pcp2-ATXN1[82Q] $\quad \square$ Pcp2-ATXN1[82Q]V591A;S602D

**A**  
IP - Cerebellum  

| IP: CIC | Input (0.2%) |
| --- | --- |
| ATXN1[82Q]-Atxn1 | CIC |

**B**  
qPCR - Cerebellum  

**C**  
Rotarod  

**D**  
IF-Calbindin  

Control $\quad$ Pcp2-ATXN1[82Q] $\quad$ Pcp2-ATXN1[82Q]V591A;S602D  

---

**Figure 1. ATXN1-CIC Complex Disruption Prevents SCA1-like Toxicity in Mice**

(A) Immunoprecipitation coupled to western blot reveals that CIC binds to ATXN1[82Q], but not ATXN1[82Q]V591A;S602D (n = 3), quantified on the right.

(B) qPCR analysis reveals similar transgene expression levels between ATXN1 lines (n = 5–7).

(C) Rotarod analysis of ATXN1[82Q], ATXN1[82Q]V591A;S602D, and control mice at 6, 12, and 20 weeks (n = 6–14).

(D) Representative calbindin immunofluorescence photomicrographs of ATXN1[82Q], ATXN1[82Q]V591A;S602D, and control mice at 52 weeks. Quantification of PC pathology at 3.5, 6, 12, 20, and 52 weeks is presented below (n = 3–7).

Data are presented as mean ± SEM; *p < 0.05, **p < 0.01, ***p < 0.0001, and ns p > 0.05.

See Figure S1.

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

$\square$ Ctrl $\quad \blacksquare$ Heterozygous $\quad \square$ Knockout

**A**

Cerebellar $Atxn1-Atxn1l/KO$

Cerebellar $Cic KO$

Adult $Cic KO$

**B**

Open Field Assay

**C**

Rotarod

**D**

Ctrl Het KO

Cerebellar $Atxn1-Atxn1l/KO$

Cerebellar $Cic KO$

Adult $Cic KO$

**Figure 2. Loss of the ATXN1-CIC Complex Does Not Impair Cerebellar Function**

(A) Western blot of cerebellar lysates from En1-Cre; $Atxn1^{-/-}$; $Atxn1f^{lox/flox}$ (left), En1-Cre; $Cic^{floxflox}$ (middle), and UBC-CreER; $Cic^{floxflox}$ (right) mice compared to their respective controls ($n = 3–7$).

(B) Total distance traveled in the open field assay for En1-Cre; $Atxn1^{-/-}$; $Atxn1f^{lox/flox}$ (left), En1-Cre; $Cic^{floxflox}$ (middle), and UBC-CreER; $Cic^{floxflox}$ (right) mice compared to their respective controls at 3 months ($n = 9–20$).

(C) Latency to fall in the rotarod assay for En1-Cre; $Atxn1^{-/-}$; $Atxn1f^{lox/flox}$ (left), En1-Cre; $Cic^{floxflox}$ (middle), and UBC-CreER; $Cic^{floxflox}$ (right) mice compared to their controls at 3 months ($n = 6–20$).

(D) Calbindin staining of cerebella from En1-Cre; $Atxn1^{-/-}$; $Atxn1f^{lox/flox}$ (top), En1-Cre; $Cic^{floxflox}$ (middle), and UBC-CreER; $Cic^{floxflox}$ (bottom) mice compared to their controls. Quantification of the molecular layer thickness is indicated on the right ($n = 3–4$).

Data are presented as mean ± SEM; *p < 0.05, **p < 0.01, ***p < 0.001, ****p < 0.0001, and ns p > 0.05.

See Figure S2.

morphology or PC number at 20 weeks post injection (Figure 2C, bottom panels), a time point when pathology can be clearly observed in mouse models of SCA1 ([Burright et al., 1995](https://doi.org/10.1016/S0896-6273(05)80130-0); [Wataste et al., 2002](https://doi.org/10.1016/S0896-6273(02)00673-8)). These data show that CIC loss in the adult cerebellum does not lead to hallmark cerebellar features of SCA1 pathology.

To assess for defects at the cellular and neurophysiological level in CIC KO mice, we further studied PC function in these

4 Neuron 97, 1–9, March 21, 2018

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

mice. We tested whether Cerebellar Cic KO mice exhibit PC spine or climbing fiber innervation defects as well as electrophysiological abnormalities. We found that spine density ([Figure 3](#fig3)A) and climbing fiber innervation ([Figure 3](#fig3)B) were largely unaffected from CIC disruption. Interestingly, Cerebellar Cic KO mice displayed an increase in PC firing pattern variability while maintaining normal firing frequencies at 2 months of age (P60; [Figure 3](#fig3)C, top panels), which were markedly increased at an earlier age (P20; [Figure 3](#fig3)C, bottom panels), suggestive of a mild neurodevelopmental phenotype. These findings contrast with those from SCA1 mice (and other models of ataxia), which show reduced firing frequencies that worsen together with behavior phenotypes ([Chopra and Shakkottai, 2014](#ref1); [Dell’Orco et al., 2015](#ref2)). Thus, loss of CIC in the cerebellum, both developmentally and in adulthood, leads to abnormal PC firing regularity but does not result in SCA1-like degenerative disease.

### Transcriptional Gain of Function of the ATXN1-CIC Complex in SCA1 Is Conserved between Mice and Humans

Given that the polyQ-expanded ATXN1-CIC complex acts in a gain-of-function manner, we asked what may be the primary mediators of such an effect. We performed transcriptomic analysis on ATXN1[82Q], ATXN1[82Q]V591A;S602D, and En1-Cre; $Cic^{floxFlox}$ mice (and control littermates). ATXN1[82Q] mice had marked gene expression changes (545, fold change ≥ 1.5, false discovery rate [FDR] < 0.05) in their cerebella, whereas the ATXN1[82Q]V591A;S602D mice exhibited very few expression changes (16, fold change ≥ 1.5, FDR < 0.05; [Figure 4](#fig4)A; [Figures S3](#figs3)B and S3D). In contrast, En1-Cre; $Cic^{floxFlox}$ mice exhibited few expression changes (136, fold change ≥ 1.5, FDR < 0.05), which did not significantly overlap with the ATXN1[82Q] mice ([Figure 4](#fig4)A; [Figures S3](#figs3)A–S3C). These molecular findings are consistent with our behavioral and anatomical findings that SCA1 is driven by a gain of function of the ATXN1-CIC complex.

To test whether this transcriptomic signature was preserved in other models of polyQ-mediated ATXN1 toxicity, we used a knockin mouse model of SCA1. $Atxn1^{154Q/2Q}$ mice express 154 glutamines in one $Atxn1$ allele ([Watase et al., 2002](#ref3)). We tested a subset of genes highly dysregulated in ATXN1[82Q] mice ([Ingram et al., 2016](#ref4)) and found that more than 60% were dysregulated in $Atxn1^{154Q/2Q}$ mice ([Figure 4](#fig4)B; [Figure S3](#figs3)E). To test whether these findings could be generalized to human SCA1, we generated neurons from SCA1 patients as well as their unaffected siblings ([Figure S4](#figs4)). Fibroblasts were obtained under oversight of the University of Minnesota Institutional Review Board. Genes tested—in particular, those involved in glutamatergic neurotransmission—were significantly downregulated in these neurons ([Figure 4](#fig4)C).

### DISCUSSION

A fundamental question in polyQ-expansion disorders involves the extent to which polyQ pathogenesis is driven by aspects of the native function of disease-causing protein compared to pathologic features such as protein inclusions, mRNA containing an expanded CAG tract, and, more recently, dipeptide repeat proteins generated from RAN translation. Within the polyQ native function, there is uncertainty regarding the relative importance of gain- and loss-of-function mechanisms to toxicity. In SCA1, there is evidence that polyQ expansion in ATXN1 causes toxicity through a gain-of-function mechanism ([Zoghbi and Orr, 2009](#ref5)). Whether it does so selectively through its native interactor CIC or whether it is toxic via other mechanisms is currently unknown. Moreover, whether loss of function also contributes to the cerebellar degeneration seen in SCA1 is unclear ([Crespo-Barreto et al., 2010](#ref6); [Fryer et al., 2011](#ref7); [Lee et al., 2011](#ref8)). In this study, we found that polyQ-expanded ATXN1 exerts cerebellar toxicity exclusively through its interaction with CIC and that its nuclear accumulation is not sufficient for its toxicity. These data are consistent with our recent results that suggest that CIC stabilizes the formation of toxic polyQ-expanded ATXN1 oligomers that correlate with disease onset and severity ([Lasagna-Reeves et al., 2015](#ref9)).

Another important outcome of the genetic studies pertains to excluding additional potential pathogenic mechanisms: specifically, the toxicity of RNA and RAN-translated peptides. The fact that mice carrying the ATXN1[82Q]V591A;S602D allele lack disease but have similar levels of CAG-expanded ATXN1 mRNA as ATXN1[82Q] mice that have PC degeneration argues against toxicity at the RNA level. This differs from what was suggested for SCA3 ([Li et al., 2008](#ref10)) and HD ([Bañez-Coronel et al., 2015](#ref11)).

Given that the ATXN1-CIC complex is critical for various developmental functions, we tested whether this complex is also important for cerebellar function. We found that developmental or adult ablation of CIC in the cerebellum did not result in neurodegenerative phenotypes. To study the role of the ATXN1-CIC complex in its native versus polyQ-expanded states, we performed mRNA profiling in these mouse models. We found that polyQ-expanded ATXN1 in PCs exhibits a large number of transcriptional changes that are conserved in an independent model of polyQ-expanded ATXN1 as well as in neurons derived from individuals with SCA1. Importantly, these changes are dramatically curtailed when polyQ-expanded ATXN1 cannot bind CIC, suggesting that its interaction with CIC mediates the bulk of these effects. Alternatively, when CIC was depleted from the cerebellum, very few transcriptional changes occurred, suggesting that the native role of the ATXN1-CIC complex in the cerebellum is more subtle than in other areas of the brain ([Lu et al., 2017](#ref12)) or the periphery ([Park et al., 2017](#ref13); [Simón-Carrasco et al., 2017](#ref14); [Tan et al., 2018](#ref15)). This is important as it hints to a mechanism for the selective sensitivity of the cerebellum to polyQ expansion.

This study supports three main conclusions: (1) formation of the ATXN1-CIC complex upon polyQ expansion is critical for cerebellar SCA1 phenotypes; (2) loss of the ATXN1-CIC complex does not impart ataxia or cerebellar toxicity; and (3) polyQ-expanded ATXN1 RNA does not contribute to SCA1 cerebellar toxicity. These findings share important parallels with another polyQ disorder: spinal-bulbar muscular atrophy (SBMA). In SBMA, polyQ expansions in the androgen receptor (AR) are thought to cause neuronal toxicity through a transcriptional gain-of-function mechanism ([La Spada, 2014](#ref16)). Moreover, loss of AR in humans leads to androgen insensitivity syndrome, a disorder that is clinically distinct from SBMA and does not present with motor neuron degeneration ([Hughes et al., 2012](#ref17)).

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

$\square$ $Cic^{flo x/+}$ or $Cic^{flo x/flo x}$ $\blacksquare$ En1-Cre; $Cic^{flo x/flo x}$

**A**

| $Cic^{flo x/+}$ | En1-Cre; $Cic^{flo x/flo x}$ |
| --- | --- |
| ![Image](image1) | ![Image](image2) |

**B**

| $Cic^{flo x/+}$ | En1-Cre; $Cic^{flo x/flo x}$ |
| --- | --- |
| ![Image](image3) | ![Image](image4) |

**C**

| $Cic^{flo x/+}$ | En1-Cre; $Cic^{flo x/flo x}$ |
| --- | --- |
| ![Image](image5) | ![Image](image6) |

---

**Figure 3. Mice with Hindbrain Ablation of CIC Fail to Manifest SCA1-like Degenerative Disease**

(A) Representative photomicrographs of Golgi-Cox-stained PCs in cerebella from En1-Cre; $Cic^{flo x/flo x}$ and controls. Quantification of PC soma area and spine number is on the right.

(B) Representative photomicrographs of cerebellar sections from En1-Cre; $Cic^{flo x/flo x}$ and control mice stained for Vglut2 and calbindin. Quantification of CF translocation and Vglut2-positive puncta on PCs is on the right. For (A) and (B), n = 9–39 neurons per condition from at least three mice per genotype.

(C) In vivo electrophysiological analysis of En1-Cre; $Cic^{flo x/flo x}$ and controls at 60 (P60, top) and 20 (P20, bottom) days of age, respectively. Representative traces are on the left; * denotes complex spike firing from a PC. PC firing frequency and coefficients of variation (CV and CV2) of the interspike interval are on the right (n = 12–25 neurons recorded from at least three animals/genotype).

Data are presented as mean ± SEM; *p < 0.05, **p < 0.01, ***p < 0.001, and ns p > 0.05.

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

**Figure 4. SCA1-Responsive Genes Are Conserved between Mice and Humans**

(A) Summary model and heatmap on log fold change from RNA sequencing (RNA-seq) profiles obtained from ATXN1[82Q] (n = 5), ATXN1[82Q] V591A;S602D (n = 5), and En1-Cre; Cic<sup>flox/flox</sup> (n = 3–4) compared to their respective control mice. Genes in the heatmap are those differentially expressed (FDR < 0.05, minimum fold change of ±1.5) in ATXN1[82Q] mice.

(B) qPCR validation of RNA-seq results in the *Atxn1*<sup>154Q/+</sup> mouse model of SCA1 (12 weeks old, affected) reveals decreases in genes consistent with that seen in the ATXN1[82Q] mice (n = 3–5).

(C) Neuron derivation from individuals with SCA1 (left). qPCR analysis of glutamate ion channel activity-related genes is quantified for three independent patient and three unaffected sibling cell lines (right). Each data point represents the average of four biological replicates. Data presented as mean ± SEM; *p < 0.05, **p < 0.01, ****p < 0.0001, and ns p > 0.05. See Figures S3 and S4.

---

Importantly, binding of the AR ligand androgen is required for polyQ-expanded, AR-mediated pathogenesis ([Katsuno et al., 2002](https://doi.org/10.1073/pnas.252616899)). While RAN translation of AR polyQ repeats and loss-of-function mechanisms have also been proposed as modes of toxicity ([Wojciechowska et al., 2014](https://doi.org/10.1016/j.cell.2014.03.027)), mounting evidence suggests that the nuclear entry of polyQ-expanded AR is necessary for disease pathogenesis—castration of male mice carrying the polyQ-expanded AR abrogates disease while not impairing the ability of RNA translation to occur ([Katsuno et al., 2002](https://doi.org/10.1073/pnas.252616899)). Future structure-guided approaches targeting interacting partners of ATXN1, AR, and other polyQ proteinopathies likely will shed insight into tractable mechanisms of toxicity.

These conclusions highlight the importance of studying both the loss and the gain of function of aggregation-prone disease-driving genes, particularly in the context of their native complexes. Our studies suggest that disrupting the ATXN1-CIC interaction, either pharmacologically or with a small peptide, will bear fruit for the ataxia in SCA1. Another option is the therapeutic targeting of total ATXN1 levels ([Keiser et al., 2015](https://doi.org/10.1016/j.celrep.2015.03.028); [Park et al., 2013](https://doi.org/10.1016/j.cell.2013.03.027)) given that ATXN1L could compensate for the native function of ATXN1 in other regions ([Lu et al., 2017](https://doi.org/10.1016/j.cell.2017.03.038)). Since the loss

---

**STAR★METHODS**

Detailed methods are provided in the online version of this paper and include the following:

- KEY RESOURCES TABLE
- CONTACT FOR REAGENT AND RESOURCE SHARING
- EXPERIMENTAL MODEL AND SUBJECT DETAILS
  - Mouse Models
  - iPSC Generation and iNeuron Culture
- METHOD DETAILS
  - Experimental Design
  - RNA Extraction and Measurement
  - Immunoprecipitation
  - Western Blot

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

- Behavioral Test
- Immunohistochemistry and Immunofluorescence
- Golgi Staining
- Tamoxifen Injections
- In Vivo Purkinje Cell Electrophysiology
- RNA Sequencing

### QUANTIFICATION AND STATISTICAL ANALYSIS
### DATA AND SOFTWARE AVAILABILITY

### SUPPLEMENTAL INFORMATION
Supplemental Information includes four figures and two tables and can be found with this article online at [https://doi.org/10.1016/j.neuron.2018.02.013](https://doi.org/10.1016/j.neuron.2018.02.013).

### ACKNOWLEDGMENTS
This work was supported by grants NIH/NINDS R37 NS022920, National Ataxia Foundation Pioneer Award, and Wallin Neuroscience Discovery Award to H.T.O.; NIH/NINDS R01 NS027699-17 and R37 NS027699 to H.Y.Z. and R01NS089664 and R01NS100874 to R.V.S.; CIHR Fellowship 201210MFE-290072-173743 and Parkinson's Foundation grant PF-JfA-1762 to M.W.C.R.; and NIH/NINDS F32 NS083091 to Q.T. H.Y.Z. is an investigator of the Howard Hughes Medical Institute. The authors thank M.A. Durham and L.A. Lavery for discussions and reading the manuscript, the iPSC, neurobehavioral, and neuropathological cores of the Jan and Dan Duncan Neurological Research Institute, and the University of Minnesota Genomics Center and Stem Cell Institute. The project used cores supported in part by BCM IDDRC grant U54HD083092 from the Eunice Kennedy Shriver NICHD. Content is solely the responsibility of authors and does not necessarily represent the official views of the Eunice Kennedy Shriver NICHD and the NIH.

### AUTHOR CONTRIBUTIONS
M.W.C.R., T.T., H.-C.L., H.Y.Z., and H.T.O. conceived the study, reviewed the data, and wrote the paper. M.W.C.R. and H.-C.L. performed biochemical, behavioral, and histological characterization of loss-of-function mouse mutants. T.T., J.F., and L.D. performed molecular, behavioral, and histological characterization of transgenic mouse lines. V.V.B. and C.J.A. performed biochemical analysis and generated reagents. M.W.C.R. and A.S. performed tamoxifen injections. E.P.L. and R.V.S. performed and analyzed the electrophysiological analysis. T.L. performed Golgi staining. M.W.C.R., Y.-W.W., J.F., Z.L., C.H., and Q.T. performed RNA-seq analysis and validation. K.T., W.C., and J.T. derived iPSCs from a SCA1 kindred. V.V.B., A.B., and M.W.C.R. performed SCA1 patient-derived neuron culture and analysis.

### DECLARATION OF INTERESTS
The authors declare no competing interests.

Received: February 9, 2017  
Revised: December 18, 2017  
Accepted: February 15, 2018  
Published: March 8, 2018

### REFERENCES
- Anders, S., Pyl, P.T., and Huber, W. (2015). HTSeq—a Python framework to work with high-throughput sequencing data. Bioinformatics 31, 166–169.
- Bañez-Coronel, M., Ayhan, F., Tarabochia, A.D., Zu, T., Perez, B.A., Tusi, S.K., Pletnikova, O., Borchelt, D.R., Ross, C.A., Margolis, R.L., et al. (2015). RAN translation in Huntington disease. Neuron 88, 667–677.
- Bowman, A.B., Lam, Y.C., Jafar-Nejad, P., Chen, H.K., Richman, R., Samaco, R.C., Fryer, J.D., Kahle, J.J., Orr, H.T., and Zoghbi, H.Y. (2007). Duplication of Ataxin-1B/Boat suppresses SCA1 neuropathy by decreasing incorporation of polyglutamine-expanded ATAXIN-1 into its native complexes. Nat. Genet. 39, 373–379.
- Burright, E.N., Clark, H.B., Servadio, A., Matilla, T., Feddersen, R.M., Yunis, W.S., Duvick, L.A., Zoghbi, H.Y., and Orr, H.T. (1995). SCA1 transgenic mice: a model for neurodegeneration caused by an expanded CAG trinucleotide repeat. Cell 82, 937–948.
- Chopra, R., and Shakkottai, V.G. (2014). Translating cerebellar Purkinje neuron physiology to progress in dominantly inherited ataxia. Future Neurol. 9, 187–196.
- Crespo-Barreto, J., Fryer, J.D., Shaw, C.A., Orr, H.T., and Zoghbi, H.Y. (2010). Partial loss of ataxin-1 function contributes to transcriptional dysregulation in spinocerebellar ataxia type 1 pathogenesis. PLoS Genet. 6, e1001021.
- Dell’Orco, J.M., Wasserman, A.H., Chopra, R., Ingram, M.A., Hu, Y.S., Singh, V., Wulff, H., Opal, P., Orr, H.T., and Shakkottai, V.G. (2015). Neuronal atrophy early in degenerative ataxia is a compensatory mechanism to regulate membrane excitability. J. Neurosci. 35, 11292–11307.
- Dobin, A., Davis, C.A., Schlesinger, F., Dreankow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M., and Gingeras, T.R. (2013). STAR: ultrafast universal RNA-seq aligner. Bioinform. 29, 15–21.
- Duvick, L., Barnes, J., Ebner, B., Agrawal, S., Andresen, M., Lim, J., Giesler, G.J., Zoghbi, H.Y., and Orr, H.T. (2010). SCA1-like disease in mice expressing wild-type ataxin-1 with a serine to aspartic acid replacement at residue 776. Neuron 67, 929–935.
- Fryer, J.D., Yu, P., Kang, H., Mandel-Brehm, C., Carter, A.N., Crespo-Barreto, J., Gao, Y., Flora, A., Shaw, C., Orr, H.T., and Zoghbi, H.Y. (2011). Exercise and genetic rescue of SCA1 via the transcriptional repressor Capicua. Science 334, 690–693.
- Holt, G.R., Softky, W.R., Koch, C., and Douglas, R.J. (1996). Comparison of discharge variability in vitro and in vivo in cat visual cortex neurons. J. Neurophysiol. 75, 1806–1814.
- Hughes, I.A., Werner, R., Bunch, T., and Hiort, O. (2012). Androgen insensitivity syndrome. Semin. Reprod. Med. 30, 432–442.
- Ingram, M., Wozniak, E.A.L., Duvick, L., Yang, R., Bergmann, P., Carson, R., O’Callaghan, B., Zoghbi, H.Y., Henzler, C., and Orr, H.T. (2016). Cerebellar transcriptome profiles of ATXN1 transgenic mice reveal SCA1 disease progression and protection pathways. Neuron 89, 1194–1207.
- Jafar-Nejad, P., Ward, C.S., Richman, R., Orr, H.T., and Zoghbi, H.Y. (2011). Regional rescue of spinocerebellar ataxia type 1 phenotypes by 14-3-3epsilon haploinsufficiency in mice underscores complex pathogenicity in neurodegeneration. Proc. Natl. Acad. Sci. USA 108, 2142–2147.
- Jiang, X., Chen, J., Bajić, A., Zhang, C., Song, X., Carroll, S.L., Cai, Z.L., Tang, M., Xue, M., Cheng, N., et al. (2017). Quantitative real-time imaging of glutathione. Nat. Commun. 8, 16087.
- Katsuno, M., Adachi, H., Kume, A., Li, M., Nakagomi, Y., Niwa, H., Sang, C., Kobayashi, Y., Doyu, M., and Sobue, G. (2002). Testosterone reduction prevents phenotypic expression in a transgenic mouse model of spinal and bulbar muscular atrophy. Neuron 35, 843–854.
- Keiser, M.S., Kordower, J.H., Gonzalez-Alegre, P., and Davidson, B.L. (2015). Broad distribution of ataxin 1 silencing in rhesus cerebella for spinocerebellar ataxia type 1 therapy. Brain 138, 3555–3566.
- Kim, E., Lu, H.C., Zoghbi, H.Y., and Song, J.J. (2013). Structural basis of protein complex formation and reconfiguration by polyglutamine disease protein Ataxin-1 and Capicua. Genes Dev. 27, 590–595.
- Kimmel, R.A., Turnbull, D.H., Blanquet, V., Wurst, W., Loomis, C.A., and Joyner, A.L. (2000). Two lineage boundaries coordinate vertebrate apical ectodermal ridge formation. Genes Dev. 14, 1377–1389.
- La Spada, A. (2014). Spinal and bulbar muscular atrophy. In GeneReviews(R), R.A. Pagon, M.P. Adam, H.H. Ardinger, S.E. Wallace, A. Amemiya, L.J.H. Bean, T.D. Bird, C.T. Fong, H.C. Mefford, and R.J.H. Smith, et al., eds. (University of Washington).
- Lam, Y.C., Bowman, A.B., Jafar-Nejad, P., Lim, J., Richman, R., Fryer, J.D., Hyun, E.D., Duvick, L.A., Orr, H.T., Botas, J., and Zoghbi, H.Y. (2006). ATAXIN-1 interacts with the repressor Capicua in its native complex to cause SCA1 neuropathology. Cell 127, 1335–1347.

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

Lasagna-Reeves, C.A., Rousseaux, M.W., Guerrero-Muñoz, M.J., Park, J., Jafar-Nejad, P., Richman, R., Lu, N., Sengupta, U., Litvinchuk, A., Orr, H.T., et al. (2015). A native interactor scaffolds and stabilizes toxic ATAXIN-1 oligomers in SCA1. eLife 4, 4.

Lee, Y., Fryer, J.D., Kang, H., Crespo-Barreto, J., Bowman, A.B., Gao, Y., Kahle, J.J., Hong, J.S., Kheradmand, F., Orr, H.T., et al. (2011). ATXN1 protein family and CIC regulate extracellular matrix remodeling and lung alveolarization. Dev. Cell 21, 746–757.

Li, J.Y., Lao, Z., and Joyner, A.L. (2002). Changing requirements for Gbx2 in development of the cerebellum and maintenance of the mid/hindbrain organizer. Neuron 36, 31–43.

Li, L.B., Yu, Z., Teng, X., and Bonini, N.M. (2008). RNA toxicity is a component of ataxin-3 degeneration in Drosophila. Nature 453, 1107–1111.

Love, M.I., Huber, W., and Anders, S. (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550.

Lu, H.C., Tan, Q., Rousseaux, M.W., Wang, W., Kim, J.Y., Richman, R., Wan, Y.W., Yeh, S.Y., Patel, J.M., Liu, X., et al. (2017). Disruption of the ATXN1-CIC complex causes a spectrum of neurobehavioral phenotypes in mice and humans. Nat. Genet. 49, 527–536.

Matilla, A., Roberson, E.D., Banfi, S., Morales, J., Armstrong, D.L., Burright, E.N., Orr, H.T., Sweat, J.D., Zoghbi, H.Y., and Matzuk, M.M. (1998). Mice lacking ataxin-1 display learning deficits and decreased hippocampal paired-pulse facilitation. J. Neurosci. 18, 5508–5516.

Orr, H.T., Chung, M.Y., Banfi, S., Kwiatkowski, T.J., Jr., Servadio, A., Beaudet, A.L., McCall, A.E., Duvick, L.A., Ranum, L.P., and Zoghbi, H.Y. (1993). Expansion of an unstable trinucleotide CAG repeat in spinocerebellar ataxia type 1. Nat. Genet. 4, 221–226.

Park, J., Al-Ramahi, I., Tan, Q., Mollema, N., Diaz-Garcia, J.R., Gallego-Flores, T., Lu, H.C., Lagalwar, S., Duvick, L., Kang, H., et al. (2013). RAS-MAPK-MSK1 pathway modulates ataxin 1 protein levels and toxicity in SCA1. Nature 498, 325–331.

Park, S., Lee, S., Lee, C.G., Park, G.Y., Hong, H., Lee, J.S., Kim, Y.M., Lee, S.B., Hwang, D., Choi, Y.S., et al. (2017). Capicua deficiency induces autoimmunity and promotes follicular helper T cell differentiation via derepression of ETV5. Nat. Commun. 8, 16037.

Rousseaux, M.W., de Haro, M., Lasagna-Reeves, C.A., De Maio, A., Park, J., Jafar-Nejad, P., Al-Ramahi, I., Sharma, A., See, L., Lu, N., et al. (2016). TRIM28 regulates the nuclear accumulation and toxicity of both alpha-synuclein and tau. eLife 5, 5.

Ruegsegger, C., Stucki, D.M., Steiner, S., Angliker, N., Radecke, J., Keller, E., Zuber, B., Rüegg, M.A., and Saxena, S. (2016). Impaired mTORC1-dependent

expression of Homer-3 influences SCA1 pathophysiology. Neuron 89, 129–146.

Ruzankina, Y., Pinzon-Guzman, C., Asare, A., Ong, T., Pontano, L., Cotsarelis, G., Zediak, V.P., Velez, M., Bhandoola, A., and Brown, E.J. (2007). Deletion of the developmentally essential gene ATR in adult mice leads to age-related phenotypes and stem cell loss. Cell Stem Cell 1, 113–126.

Serra, H.G., Duvick, L., Zu, T., Carlson, K., Stevens, S., Jorgensen, N., Lysholm, A., Burright, E., Zoghbi, H.Y., Clark, H.B., et al. (2006). RORalpha-mediated Purkinje cell development determines disease severity in adult SCA1 mice. Cell 127, 697–708.

Servadio, A., Koshy, B., Armstrong, D., Antalfyy, B., Orr, H.T., and Zoghbi, H.Y. (1995). Expression analysis of the ataxin-1 protein in tissues from normal and spinocerebellar ataxia type 1 individuals. Nat. Genet. 10, 94–98.

Simón-Carrasco, L., Graña, O., Salmón, M., Jacob, H.K.C., Gutierrez, A., Jiménez, G., Drosten, M., and Barbacid, M. (2017). Inactivation of Capicua in adult mice causes T-cell lymphoblastic lymphoma. Genes Dev. 31, 1456–1468.

Sztainberg, Y., Chen, H.M., Swann, J.W., Hao, S., Tang, B., Wu, Z., Tang, J., Wan, Y.W., Liu, Z., Rigo, F., and Zoghbi, H.Y. (2015). Reversal of phenotypes in MECP2 duplication mice using genetic rescue or antisense oligonucleotides. Nature 528, 123–126.

Tan, Q., Brunetti, L., Rousseaux, M.W.C., Lu, H.-C., Wan, Y.-W., Revelli, J.-P., Liu, Z., Goodell, M.A., and Zoghbi, H.Y. (2018). Loss of Capicua alters early T cell development and predisposes mice to T cell lymphoblastic leukemia/lymphoma. Proc. Natl. Acad. Sci. USA 115, E1511–E1519.

Tolar, J., Le Blanc, K., Keating, A., and Blazar, B.R. (2010). Concise review: hitting the right spot with mesenchymal stromal cells. Stem Cells 28, 1446–1455.

Watase, K., Weeber, E.J., Xu, B., Antalfyy, B., Yuva-Paylor, L., Hashimoto, K., Kano, M., Atkinson, R., Sun, Y., Armstrong, D.L., et al. (2002). A long CAG repeat in the mouse Sca1 locus replicates SCA1 features and reveals the impact of protein solubility on selective neurodegeneration. Neuron 34, 905–919.

White, J.J., and Sillitoe, R.V. (2017). Genetic silencing of olivocerebellar synapses causes dystonia-like behaviour in mice. Nat. Commun. 8, 14912.

Wojciechowska, M., Olejniczak, M., Galka-Marciniak, P., Jazurek, M., and Krzyzosiak, W.J. (2014). RAN translation and frameshifting as translational challenges at simple repeats of human neurodegenerative disorders. Nucleic Acids Res. 42, 11849–11864.

Zoghbi, H.Y., and Orr, H.T. (2009). Pathogenic mechanisms of a polyglutamine-mediated neurodegenerative disease, spinocerebellar ataxia type 1. J. Biol. Chem. 284, 7425–7429.

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

STAR★METHODS

KEY RESOURCES TABLE

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
| --- | --- | --- |
| Antibodies |  |  |
| Anti-Mouse; Human/Rabbit ATXN1 | In house; Servadio et al., 1995 | 11750; RRID: AB_2721278 |
| Anti-Mouse; Human/Rabbit ATXN1 | In house; Servadio et al., 1995 | 11NQ; RRID: AB_2721279 |
| Anti-Mouse/Mouse Calbindin (D-28K) | Swant | CB38; RRID: AB_10000340 |
| Anti-Mouse/Mouse Calbindin (D-28K) | Sigma | C9848; RRID: AB_2314067 |
| Anti-Mouse/Rabbit ATXN1L | In house; Bowman et al., 2007 | RRID: AB_2721280 |
| Anti-Mouse/Rabbit Capicua | In house; Lam et al., 2006 | RRID: AB_2721281 |
| Anti-Mouse/Rabbit Capicua | Abcam | ab123822; RRID: AB_10976499 |
| Anti-Mouse/Mouse GAPDH (6C5) | Advanced ImmunoChemical | 2-RGM2; RRID: AB_2721282 |
| Anti-Rabbit/Donkey IgG-Alexa Fluor 488 | Jackson ImmunoResearch | 711-545-152; RRID: AB_2313584 |
| Anti-Mouse; Human/Mouse ATXN1 | This paper | RRID: AB_2721283 |
| Anti-Mouse/Rabbit Vglut2 | Synaptic Systems | 135402; RRID: AB_2187539 |
| Anti-Mouse/Mouse Vinculin | Sigma-Aldrich | V9131; RRID: AB_477629 |
| Chemicals, Peptides, and Recombinant Proteins |  |  |
| TRIzol reagent | Thermo Fisher Scientific | 15596-026 |
| M-MLV Reverse Transcriptase | Thermo Fisher Scientific | 28025013 |
| Xpert Protease inhibitor cocktail (100×) | GenDepot | P3100-100 |
| Xpert Phosphatase inhibitor cocktail (100×) | GenDepot | P3200-020 |
| bFGF | PeproTech | AF-100-18B |
| EGF | PeproTech | AF-100-15 |
| BDNF | PeproTech | AF-450-02 |
| GDNF | PeproTech | AF-450-10 |
| NT-3 | PeproTech | AF-450-03 |
| Y-27632 | Selleckchem | S1049 |
| SB-431542 | Selleckchem | S1067 |
| Cycloamine | Selleckchem | S1146 |
| Dorsomorphin | Santa Cruz | sc-361173 |
| Dibutyryl-cAMP | Santa Cruz | sc-201567 |
| Accutase | Sigma-Aldrich | A6964 |
| Critical Commercial Assays |  |  |
| iTaq Universal SYBR Green Supermix | Bio-Rad | 1725124 |
| VECTASTAIN Elite ABC HRP Kit (Peroxidase, Mouse IgG) | Vector Laboratories | PK-6102 |
| iScript Reverse Transcription Supermix for RT-qPCR | Bio-Rad | 1708840 |
| Light Cycler 480 Probes Master kit | Roche | 04-707-494-001 |
| Quant-iT RiboGreen assay | Life Technologies | R11490 |
| RNeasy Mini Kit | QIAGEN | 74104 |
| PicoGreen | Thermo Fisher Scientific | P11496 |
| FD Rapid Golgi Stain Kit | FD Neurotechnologies | PK401 |
| CytoTune-iPS 2.0 kit | Invitrogen | A16517 |
| Deposited Data |  |  |
| RNA-seq data from En1-Cre; Cic<sup>flox/flox</sup> and control mice (cerebellum) | This paper | GEO: GSE108254 |
| RNA-seq data from PCP2-ATXN1[82Q], PCP2-ATXN1[82Q] V591A; S602D and FVB control mice (cerebellum) | This paper | GEO: GSE108255 |
| Entire RNA-seq dataset | This paper | GEO: GSE108256 |

(Continued on next page)

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

---

**Continued**

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
|---------------------|--------|------------|
| **Experimental Models: Cell Lines** |
| The generation of IPSC-derived neurons from an SCA1 kindred is described in detail in the STAR Methods. | This paper | Available upon request |
| **Experimental Models: Organisms/Strains** |
| Mouse: C57BL/6 | The Jackson Laboratory | Stock No: 005304; RRID: IMSR_JAX: 005304 |
| Mouse: FVB | The Jackson Laboratory | Stock No: 001800; RRID: IMSR_JAX: 001800 |
| Mouse: Atxn1<sup>null</sup> | Matilla et al., 1998 | RRID: MGI: 2181805 |
| Mouse: Atxn1<sup>flox</sup> | Lu et al., 2017 | RRID: MGI: 5903201 |
| Mouse: Cic<sup>flox</sup> | Lu et al., 2017 | RRID: MGI: 5903084 |
| Mouse: En1-Cre | Kimmel et al., 2000 | RRID: MGI: 3528251 |
| Mouse: UBC-Cre<sup>ERT2</sup>-B6.Cg-Tg(UBC-cre/ERT2)1Ejb/1J | The Jackson Laboratory | Stock No: 007001; RRID: IMSR_JAX: 007001 |
| Mouse: PCP2-ATXN1[82Q] | This paper | RRID: MGI: 6113813 |
| Mouse: PCP2-ATXN1[82Q]V591A;S602D | This paper | RRID: MGI: 6113692 |
| Mouse: Atxn1<sup>154Q/+</sup> | In house; Watase et al., 2002 | RRID: MGI: 2429435 |
| **Recombinant DNA** |
| PCP2-ATXN1[82Q] | In house; Burright et al., 1995 | N/A |
| PCP2-ATXN1[82Q]V591A;S602D | In house | N/A |
| **Oligonucleotides** |
| Oligonucleotides used in this study are listed in Table S2 | This paper | N/A |
| **Software and Algorithms** |
| STAR aligner-2.5.3a | Dobin et al., 2013 | https://github.com/alexdobin/STAR |
| HTSeq | Anders et al., 2015 | https://htseq.readthedocs.io/en/release_0.9.1/ |
| DESeq2 | Love et al., 2014 | https://bioconductor.org/packages/release/bioc/html/DESeq2.html |
| pheatmap | N/A | https://cran.r-project.org/web/packages/pheatmap/index.html |

---

**CONTACT FOR REAGENT AND RESOURCE SHARING**

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Harry T. Orr (orrx002@umn.edu).

---

**EXPERIMENTAL MODEL AND SUBJECT DETAILS**

**Mouse Models**

Atxn1<sup>null</sup>, Atxn1<sup>flox</sup>, Cic<sup>flox</sup> and Atxn1<sup>154Q/+</sup> mice were previously described (Lee et al., 2011; Lu et al., 2017; Matilla et al., 1998; Watase et al., 2002). En1-Cre mice were a gift from Dr. Alexandra Joyner (Kimmel et al., 2000). Ubc-Cre<sup>ERT2</sup> mice (B6.Cg-Tg(UBC-cre/ERT2)1Ejb/1J) were obtained from Jackson Laboratory (Stock number: 007001) (Ruzankina et al., 2007). Pcp2-ATXN1[82Q] V591A;S602D mice were generated by sequential site directed mutagenesis reactions (Quikchange II Site-Directed Mutagenesis kit #200521 Agilent) using the Pcp2-ATXN1[82Q] (B05) construct previously described (Burright et al., 1995). In addition, Pcp2-ATXN1[82Q] DNA was reinjected as the repeat in the previously described transgene in the B05 mouse had contracted to less than 70Q and had a diminished phenotype. Pro-nuclear injections were performed by the Mouse Genetics Laboratory, University of Minnesota. PCR and Southern blot analyses were used to identify transgene positive animals. For Pcp2-ATXN1[82Q], seven lines were generated with 3 lines expressing the transgene and two lines developed disease. In the case of Pcp2-ATXN1[82Q] V591A;S602D, four lines were generated and 2 lines had comparable high expression equivalent to the new Pcp2-ATXN1[82Q] line. Given the biochemical, histological, and behavioral similarities between the two lines (Figures S1B–S1D), we used “line B” of the ATXN1[82Q]V591A;S602D mice for detailed analyses.

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

All knockout and knockin mice were maintained on a C57BL/6 background and all transgenic mice were maintained on a FVB background. Genotyping was performed using standardized procedures using the primers outlined in the [Key Resources Table](https://www.cell.com/cms/10.1016/j.neuron.2018.02.013/figuresupplemental). All the studies were performed with the investigators blinded to the genotypes of the mice. Ages of mice are highlighted in the figures and figure legends. Briefly, for the knockout mice studies, 3-month-old mice were used for behavioral analysis and tissues were processed at 12 months of age. For the transgenic mice studies, behavioral and histological studies were performed at the times indicated (from 3.5 to 52 weeks of age) and RNA-seq was performed at 12 weeks of age. *Atxn1*<sup>154Q/+</sup> knockin mice tissues were collected at 12 weeks of age. Both males and females were used for the study, and all procedures were approved by the Institutional Animal Care and Use Committees of Baylor College of Medicine and the University of Minnesota.

### iPSC Generation and iNeuron Culture

Fibroblasts were derived from skin biopsies collected from individuals with SCA1 and their unaffected siblings after obtaining written, informed content as approved by the institutional Review Board of the Human Subjects Committee at the University of Minnesota. The age and gender of these individuals has not been reported to respect the wishes of anonymity of this kindred. iPSC cells were reprogrammed from fibroblasts ([Tolar et al., 2010](https://www.cell.com/cms/10.1016/j.neuron.2018.02.013/figuresupplemental)). Fibroblast cells were reprogramed with CytoTune-iPS 2.0 kit (Invitrogen) follow the manufacturer’s instructions. After iPSC cells became Sentai virus free, karyotype analysis was performed. The evaluation of iPSC cells with normal karyotypes were as described previously ([Tolar et al., 2010](https://www.cell.com/cms/10.1016/j.neuron.2018.02.013/figuresupplemental)). The experiments using mice for teratoma experiment were conducted after approval by the Institutional Animal Care and Research Committee at University of Minnesota.

NPC derivation from iPSCs was performed following an earlier protocol ([Jiang et al., 2017](https://www.cell.com/cms/10.1016/j.neuron.2018.02.013/figuresupplemental)). Briefly, thawed iPSCs were cultured for two weeks and then harvested with accutase (Sigma-Aldrich, A6964). Cells were then recovered from a pellet, diluted in culturing media (E8) with 10 μM ROCK inhibitor Y-27632 (Selleck Chemicals, S1049) and plated in 24-well Aggrewell800 plates with 2 million cells per well. On the next day, half of E8 media was changed without adding Y-27632. After 24 more hours induction was initiated by gently collecting aggregates and replacing E8 media with neuronal induction media (NIM) with 10 μM SB431542 (Selleck Chemicals, S1067). This culture was maintained on an orbital shaker with daily changes of three-fourths of media volume for three days. Next step was to collect aggregates and plate on culturing surface previously coated with 0.5 mg of base membrane protein (Cultrex BME RGF, 3433-010-01). At this point media was changed to neuronal proliferation media (NPM) and cultures were propagated with daily changes for three more days in the presence of 10 μM SB431542 and 1 μM dorsomorphin (Santa Cruz Biotechnology, sc-361173). Over the next three days aside of the dual SMAD inhibitors NPM media was supplemented with 1 μM cyclopamine (Selleck Chemicals, S1146) to promote progenitor dorsalization. Cultures were then for three more days in NPM without drugs and rosette selection reagent (Stem Cell Technologies, 05832) was used to lift rosettes 12 days from the start if induction procedure. From this point on NPCs were cultured on Cultrex coated plates in NPM and passaged once in 5–7 days with accutase. After doing IF to confirm NPCs identity, cells were cryo-preserved or induced into neurons by plating 2 million NPCs in Cultrex coated T-75 dishes in neuronal differentiation media (NDM). Neurons were differentiated for 16 with full media changes every other day while 1 μg/mL laminin (Cultrex Mouse Laminin I PathClear, 3400-010-02) was supplemented in media every four days. For final experiments neurons were passaged with trypsin, cells counted and plated at a density of 0.4 million cells per well in 24-well plates previously coated with base membrane protein. These cultures were further matured for one month in NDM with half media changes every 2–3 days and supplemented with laminin every four days.

### METHOD DETAILS

#### Experimental Design

Experiments were conducted so that replication of findings were often ensured by independent parties. For instance, two individuals have performed the behavioral analyses thus ensuring the validity of the findings. Moreover, the iterative nature of performing these analyses on multiple cohorts of mice controls for inter-cohort variability. Experimenters were kept blind to the genotypes of mice throughout the study and therefore randomization in data collection naturally occurred. Sample-size estimation was based on our (and others’) previous work with SCA1 animal models. For human neuron analysis, we used three independent lines of SCA1 patients and 3 sibling controls to ensure reproducibility in our findings. No exclusion criteria were set for these studies and therefore no animals were omitted from our analysis.

#### RNA Extraction and Measurement

qPCR of target genes was performed using intron spanning primers (when possible) as previously described ([Rousseaux et al., 2016](https://www.cell.com/cms/10.1016/j.neuron.2018.02.013/figuresupplemental)). Briefly, total RNA was isolated using TRIzol (Invitrogen) and 1.5 μg of RNA was reversed transcribed into cDNA using M-MLV Reverse Transcriptase. 75 ng of cDNA was used for each sample/primer pair and technical duplicates were used for each biological sample. S16 and *HPRT* were used as housekeeping genes. Samples were run on a CFX96 Real-time PCR detection system (Bio-Rad). Relative transcript levels were measured using ΔCt. For the *Pcp2-ATXN1[82Q]* and *Pcp2-ATXN1[82Q]V591A;S602D* expression quantification, 500ng of RNA was reversed transcribed into cDNA using iScript cDNA kit (Bio-Rad #172-5037) in triplicate synthesis reactions. cDNA was diluted 5-fold and 2 μL was used in a qPCR reaction with Light Cycler 480 Probes Master kit (Roche #04-707-494-001). *GAPDH*, *HPRT* and S16 were used as housekeeping genes. The complete list of primers used in this study is presented in the [Key Resources Table](https://www.cell.com/cms/10.1016/j.neuron.2018.02.013/figuresupplemental).

e3 Neuron 97, 1–9.e1–e5, March 21, 2018

Please cite this article in press as: Rousseaux et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

---

### Immunoprecipitation
Immunoprecipitation of CIC was performed as previously described ([Lu et al., 2017](https://doi.org/10.1016/j.neuron.2018.02.013)). Briefly, cerebella were lysed in extraction buffer (75 mM NaCl, 5 mM MgCl₂, 50 mM Tris pH 8.0, 0.5% Triton X-100, supplemented with fresh protease and phosphatase inhibitors). Spun cerebellar lysates were incubated with 3 μL (~1 mg/mL) of rabbit anti-CIC or normal rabbit IgG and incubated overnight at 4°C. Lysate-antibody mixture was then washed prior to being analyzed via western blot.

---

### Western Blot
Tissues were homogenized with RIPA buffer (50 mM Tris-HCl pH 7.6, 250 mM NaCl, 0.1% SDS, 0.5% Sodium deoxycholate, 1% Triton X-100) in the presence of protease inhibitor and phosphatase inhibitor (GenDEPOT). Western blotting was performed with NuPage 4%–12% gel with MES buffer and transferred to a nitrocellulose membrane. The membranes were blocked with 5% milk in TBST (25 mM Tris-HCl pH 8.0, 150 mM NaCl, 0.1% Tween-20). Primary antibodies were diluted in 2% BSA in TBST and incubated at 4°C overnight. The membranes were washed for three times in TBST, and incubated with secondary antibody in 5% milk in TBST before being developed and imaged using a GE ImageQuant LAS 4000. The primary antibodies used in this study are presented in the [Key Resources Table](https://doi.org/10.1016/j.neuron.2018.02.013).

---

### Behavioral Test
The open-field analysis ([Lu et al., 2017](https://doi.org/10.1016/j.neuron.2018.02.013)) and rotarod ([Burright et al., 1995; Park et al., 2013; Watase et al., 2002](https://doi.org/10.1016/j.neuron.2018.02.013)) were performed as previously described. For each test, mice were left to habituate in the testing room with ambient white noise for 30-60 min prior to testing. All behavioral analysis was performed by individuals blind to genotype and treatment.

---

### Immunohistochemistry and Immunofluorescence
Mice were deeply anesthetized and transcardially perfused with 1x PBS followed by 4% PFA in 1x PBS. The tissues were then treated with 70% ethanol for 24 hr, 95% ethanol overnight, 100% ethanol for 4 hr and chloroform overnight prior to being treated with paraffin for 4 hr twice. Paraffin embedded tissue blocks were sectioned at 8 μm thickness. H&E staining were performed using standard histological methodology. Immunohistochemistry for Calbindin was performed using rabbit anti-Calbindin D-28k primary antibody (Swant, CB-38) and detected using the VECTASTAIN Elite ABC Kit (Vector, PK-6200). Briefly, for Calbindin (C9848, 1:4,000), Vglut2 (Synaptic Systems 135 402, 1:2,000), ATXN1 (11NQ and 11750, 1:1,000) and CIC (CIC-418-4, 1:2,000) immunofluorescence, mouse hemi-brains were collected and post-fixed for 48h in 10% buffered formalin. Brains were then cryoprotected using 30% sucrose and frozen on dry ice. 40 μm free-floating sagittal cryosections were collected on a Microm Leica CM3050S cryostat and samples were processed as previously described ([Lu et al., 2017; Ruegsegger et al., 2016; Watase et al., 2002](https://doi.org/10.1016/j.neuron.2018.02.013)). Fluorescent imaging was performed on a Zeiss LSM880 Airyscan confocal microscope. Analysis for Climbing Fiber (Vglut2-positive) innervation of Purkinje cells (Calbindin-positive) was performed as previously described using IMARIS ([Ruegsegger et al., 2016](https://doi.org/10.1016/j.neuron.2018.02.013)).

For the *Pcp2-ATXN1[82Q]* and *Pcp2-ATXN1[82Q]V591A;S602D* transgenic animals, brains were cut on a vibratome and 50 μm free-floating sagittal sections were stained with mouse anti-Calbindin D-28k (Sigma, C9848) and rabbit ATXN1 (11750). Fluorescent images were scanned using an Olympus Fluoview 1000 IX2 inverted microscope and molecular layer measurements were done in the Fluoview Viewer software as previously described ([Duvick et al., 2010](https://doi.org/10.1016/j.neuron.2018.02.013)).

---

### Golgi Staining
Golgi-Cox staining was performed using the FD Rapid GolgiStain Kit (PK 401, FD Neuro Technologies inc). Brains were sectioned and stained at 50 μm (directly on slides). For each section, a z stack series was taken at 1 μm intervals (corresponding to approximately 50 pictures). Spine analysis was performed using Image J1.50i using the simple neurite tracer function to measure dendritic length and counting spines adjacent to the demarcated neurite through the Z-plane. Soma area was also calculated using Image J1.50i. Briefly, photomicrographs were converted to binary and the area of individual PCs was measured.

---

### Tamoxifen Injections
Tamoxifen injections were performed as previously described ([Sztainberg et al., 2015](https://doi.org/10.1016/j.neuron.2018.02.013)). Briefly, tamoxifen was dissolved in peanut oil (20 mg/mL) at 50°C, aliquoted and frozen until use. Starting at 8-12 weeks of age, tamoxifen or vehicle (peanut oil) was injected intraperitoneally at a dose of 100mg/kg, three times a week for four weeks. Mice were left to recover for at least two weeks before proceeding with behavioral, biochemical, and histological assessment.

---

### In Vivo Purkinje Cell Electrophysiology
Recordings were performed as previously described ([White and Sillitoe, 2017](https://doi.org/10.1016/j.neuron.2018.02.013)). Briefly, extracellular single-unit recordings were obtained *in vivo* using 5-15 MΩ glass electrodes backfilled with 0.9% (w/v) saline. Animals were anesthetized throughout recording sessions with a cocktail of 50 mg/kg ketamine and 0.5 mg/kg dexmedetomidine and 0.15%–0.25% isoflurane. Craniotomies were performed 6.5 mm posterior and 0-1.5 mm lateral of bregma with a diameter of 1-3mm. The signals were band-pass filtered at 0.3-13 kHz, amplified with an ELC-03XS amplifier (NPI), and digitized in real time in Spike2 (CED). Purkinje cell recordings were identified by the presence of both simple spikes and complex spikes, and isolated Purkinje cells were held for ~300 s. Spike sorting and analysis was performed with Spike2 and MATLAB scripts. Spike firing frequency (Hz = spike/s) and firing pattern variability were

Please cite this article in press as: Rousseau et al., ATXN1-CIC Complex Is the Primary Driver of Cerebellar Pathology in Spinocerebellar Ataxia Type 1 through a Gain-of-Function Mechanism, Neuron (2018), https://doi.org/10.1016/j.neuron.2018.02.013

calculated for each Purkinje cell. Average variability in firing pattern was quantified using the coefficient of variance (CV) of the inter-spike time interval (ISI) [CV = (standard deviation of ISIs) / (mean of ISIs)]. To measure rhythmicity during burst firing, CV2 of the ISI was calculated [CV2 = 2|ISIn+1 − ISIn| / (ISIn+1 + ISIn)] ([Holt et al., 1996](https://www.sciencedirect.com/science/article/pii/S0896627396800450)).

### RNA Sequencing

Total RNA was isolated from dissected cerebella using TRIzol Reagent (Life Technologies, Carlsbad, California) following the manufacturer’s protocols. Cerebella were homogenized using an RNase-Free Disposable Pellet Pestles in a motorized chuck. For RNA-sequencing, RNA was further purified to remove any organic carryover using the RNeasy Mini Kit (QIAGEN) following the manufacturer’s RNA Cleanup protocol.

Cerebellar RNA from three biological replicates for each genotype was isolated. Purified RNA was sent to the University of Minnesota Genomics Center for quality control, including quantification using fluorimetry (RiboGreen assay, Life Technologies) and RNA integrity assessed with capillary electrophoresis (Agilent BioAnalyzer 2100, Agilent Technologies) generating an RNA integrity number (RIN). All submitted samples had greater than 1ug total mass and RINs 7.9 or greater. Library creation was completed using oligo-dT purification of polyadenylated RNA, which was reverse transcribed to create cDNA. cDNA was fragmented, blunt-ended, and ligated to barcoded adaptors. Library was size selected to 320bp ± 5% to produce average inserts of approximately 200bp, and size distribution validated using capillary electrophoresis and quantified using fluorimetry (PicoGreen, Life Technologies) and Q-PCR. Libraries were then normalized, pooled, and sequenced on an Illumina HiSeq 2000 using a 125nt paired-end read strategy. Data were stored and maintained on University of Minnesota Supercomputing Institute (MSI) Servers.

Raw reads were first groomed by trimming the first 12bp from the 5′ ends with random base concentration, then mapped to UCSC mm10 using STAR aligner (STAR-2.5.3a). An average of ~87% in mappability was obtained. Then, expression level for each gene was estimated using Python program HTSeq ([Anders et al., 2015](https://academic.oup.com/bioinformatics/article/31/22/3676/248052)). Specifically, `htseq-count` function of HTSeq accumulated the number of aligned reads that falls under the exons of the gene. The obtained read counts were then used to perform gene analyses using the DESeq2 package ([Love et al., 2014](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8)) in the R environment using default parameters. Heatmap was generated using the pheatmap package ([https://cran.r-project.org/web/packages/pheatmap/index.html](https://cran.r-project.org/web/packages/pheatmap/index.html)) in the R environment.

### QUANTIFICATION AND STATISTICAL ANALYSIS

Experimenters were blind to genotypes and, in the case of adult knockouts, separate experimenters were used for the tamoxifen/vehicle injections versus the behavioral testing. Animals were given nondescript codes for analysis and experimenters were unblinded once the primary endpoint was reached. A minimum of three biological samples were used for each condition and samples were randomized when appropriate. All data presented are of mean + (or ± ) SEM. *, **, ***, **** and ns denote p < 0.05, p < 0.01, p < 0.001, p < 0.0001 and p > 0.05, respectively. Complete statistical analysis for all data points throughout the manuscript are presented as [Table S1](#table-s1).

### DATA AND SOFTWARE AVAILABILITY

Raw RNA sequencing files have been deposited to GEO: GSE108256.
