
Accepted Manuscript

Title: Autophagy as a Common Pathway in Amyotrophic Lateral Sclerosis

Authors: Dao K.H. Nguyen, Ravi Thombre, Jiou Wang

PII: S0304-3940(18)30261-1  
DOI: https://doi.org/10.1016/j.neulet.20
# Autophagy as a Common Pathway
## in Amyotrophic Lateral Sclerosis

Dao K. H. Nguyen${}^{1,2,a}$, Ravi Thombre${}^{1,2,a}$, Jiou Wang${}^{1,2*}$jiouw@jhu.edu

${}^{1}$ Department of Biochemistry and Molecular Biology, Bloomberg School of Public Health, Johns Hopkins University, Baltimore, MD, 21205, USA.

${}^{2}$ Department of Neuroscience, School of Medicine, Johns Hopkins University, Baltimore, MD, 21205, USA.

${}^{a}$These authors contributed equally to this work.

Running Title: Autophagy in ALS.

*Corresponding author.*

### Highlights

- > Many ALS-causing genetic factors share a common link to autophagy
- > Diverse mechanisms in ALS impair or overactivate autophagy
- > Autophagic activation or repression has variable effects in ALS models
- > The nodes of autophagy amenable to therapeutic intervention remain uncertain

### Abstract

Age-dependent neurodegenerative diseases are associated with a decline in protein quality control systems including autophagy. Amyotrophic lateral sclerosis (ALS) is a motor neuron degenerative disease of complex etiology with increasing connections to other neurodegenerative conditions such as frontotemporal dementia. Among the diverse genetic causes for ALS, a striking feature is the common connection to autophagy and its associated pathways. There is a recurring theme of protein misfolding as in other neurodegenerative diseases, but importantly there is a distinct common thread among ALS genes that connects them to the cascade of autophagy.

However, the roles of autophagy in ALS remain enigmatic and it is still unclear whether activation or inhibition of autophagy would be a reliable avenue to ameliorate the disease. The main evidence that links autophagy to different genetic forms of ALS is discussed.

KeyWords: ALS; alsin; autophagy; C9orf72; CHMP2B; dynactin; FIG4; FTD; FUS; optineurin; p62; profilin; SOD1; TBK1; TDP-43; VAPB; VCP; UBQLN2

Introduction

ALS is an age-dependent neurodegenerative disease characterized by progressive degeneration of motor neurons (MNs), with ~10% of all cases being familial. To date, a large number of genes have been linked to ALS, including Cu/Zn superoxide dismutase (SOD1) [1], TAR DNA-binding protein (TDP-43) [2, 3], fused in sarcoma (FUS) [1, 4], sequestosome 1 (SQSTM1/p62) [5, 6], optineurin (OPTN) [7], TANK binding kinase 1 (TBK1) [8], VAMP-associated protein B (VAPB) [9, 10], valosin-containing protein (VCP) [11], ubiquilin 2 (UBQLN2) [12], alsin (ALS2) [13, 14], charged multivesicular body protein 2B (CHMP2B) [15–17], dynactin (DCTN) [18], profilin 1 (PFN1) [19], factor-induced gene 4 (FIG4) [20], and a hexanucleotide repeat expansion in the gene C9orf72 [21, 22]. In recent years, increasing evidence has suggested that ALS and frontotemporal dementia (FTD) are two related diseases in a continuous clinical spectrum [23]. FTD is the second most common cause of dementia after Alzheimer's disease in patients younger than 65 [24]. Biochemical and genetic studies have identified the involvement of TDP-43, FUS, and C9orf72 in both ALS and FTD, as part of evidence linking these two neurodegenerative disorders.

The toxicity caused by protein misfolding or aggregation, or proteotoxicity, is a common theme in many forms of ALS. For example, ubiquitin-positive inclusions have been reported as an almost universal pathologic feature of ALS, including those linked to SOD1, TDP-43, FUS, and C9orf72 [25]. Not only has the proteinopathy of the abnormal proteins been identified as a major pathological feature in familial ALS (fALS), but the abnormal accumulation of the wild-type forms of some proteins, such as TDP-43, has also been observed in a large fraction of the patients with sporadic ALS (sALS) [2, 26]. However, the involvement of proteotoxicity in ALS is not limited to proteinaceous inclusions but is also affirmed by strong evidence of impairment in protein quality control systems, particularly autophagy, in ALS. Although proteotoxicity has been implicated in many neurodegenerative diseases, there are a multitude of ALS genes with functions related to

the autophagic system, including p62, optineurin, VCP, ubiquilin 2, and TBK1. The increasingly strong connection between ALS and autophagy suggests that there are shared mechanisms underlying different forms of ALS (Figure 1 & Table 1). In this review, we summarize recent findings concerning a few select ALS genes with regard to their connections to autophagy, as part of a discussion on our current understanding of the basic disease pathways.

### SOD1

Superoxide dismutase 1 (SOD1) is a key enzyme involved in quenching toxic superoxide radicals within the cell. More than 100 different mutations of SOD1 have been linked to fALS, with approximately 20% of fALS cases and 1% of sALS cases being attributable to SOD1 mutations [1, 27–29]. Although the mechanism by which mutant SOD1 (mSOD1) causes ALS is not fully understood, many studies suggest that the misfolding or aggregation of mSOD1 is a key event in the pathogenic process. The role of autophagy in mSOD1 clearance was initially implicated by the observation that autophagic inhibition resulted in the accumulation of insoluble SOD1 aggregates in MNs [30]. On the basis of this finding, dysregulations in protein quality control systems such as autophagy has been suspected in ALS pathophysiology associated with mSOD1.

Pathogenic mSOD1 is characterized by the misfolding and aggregation of the protein. Autophagy adaptor p62 interacts with polyubiquitinated misfolded mSOD1, and enhances the association between LC3-II and mSOD1, sequestering mSOD1 into protein inclusions [31–33]. Studies suggest that mSOD1 aggregation can affect multiple early steps in autophagy, with evidence from mSOD1 models pointing to the occurrence of hyperactive induction of autophagy [34]. Consistent with this evidence, levels of Ser757-phosphorylated ULK1, a target of mTOR, was significantly reduced in MNs of SOD1<sup>G85R</sup> mice, suggestive of AMPK activation and mTOR repression, resulting in enhanced initiation of autophagy [35, 36]. Further indicating autophagic induction, amplified proteolytic processing of LC3-I to LC3-II and increased LC3-II turnover was

observed in MN cell cultures expressing SOD1<sup>G93A</sup> [37]. Equivalent results were also seen in vivo, with upregulated LC3-II levels and autophagic vacuoles consistently being found in spinal cord and brainstem fractions of SOD1<sup>G93A</sup> mice [38–40]. Moreover, in spinal cords of SOD1<sup>G93A</sup> mice, there was persistent upregulation of the autophagy regulators TFEB (a coordinator of autophagic induction and lysosome biogenesis) [41], and beclin 1 (an initiator of autophagy involved in autophagosome production) [42], with a corresponding increase in LC3-II [43].

Autophagosome maturation and subsequent lysosomal fusion are downstream events in autophagy at which mSOD1 is also thought to exert its deleterious effects. Autophagosomes mature during retrograde transport and are shuttled by way of the dynein-dynactin complex, acidifying prior to fusion with lysosomes located in the soma [44]. In retrograde transport, processive movement of dynein along the microtubule enables transport of autophagosomes for endosomal or lysosomal fusion [45, 46]. It was found that in ALS-linked mSOD1-expressing neurons and transgenic mice, there was colocalization between dynein, and mSOD1. In mice, misfolded mSOD1 acquired a gain-of-interaction with dynein, overloading dynein-mediated retrograde axonal transport, impairing autophagosome transport [47]. Similar defects in retrograde transport were observed in another SOD1<sup>G93A</sup> murine model, where mSOD1 closely associated with the dynein-dynactin complex in MNs, sequestering dynein into mSOD1 aggregates and impairing retrograde transport efficiency at an early symptomatic stage [48]. Interestingly, it was shown in cells that overexpression of SNAPIN, a vesicle docking protein, can compete with mSOD1 for binding to dynein, thereby rescuing the effects of impaired retrograde transport via recruitment of dynein motors to late endosomes for transport [49]. Overall, mSOD1-induced dysfunctions in retrograde transport appear to contribute to a failure of late-stage fusion steps in autophagy.

Given the important role of autophagy in cleansing the cell of misfolded proteins, promoting autophagy in mSOD1 models has been thought to be beneficial [50]. To date, however, the assessment of autophagic enhancement in ALS SOD1<sup>G93A</sup> mice has uncovered conflicting

results. In several studies, when SOD1<sup>G93A</sup> mice were administered with autophagy inducer lithium carbonate, they experienced delayed disease progression and prolonged lifespan [51, 52], contrasting with later findings reporting worsened neuropathology in SOD1<sup>G93A</sup> mice treated with lithium [53]. Furthermore, when SOD1<sup>G93A</sup> mice were treated with autophagy enhancers including trehalose and resveratrol, animals displayed delayed symptom onset, prolonged survival, and diminished MN loss [35, 54–58]. However, when SOD1<sup>G93A</sup> mice were treated with another autophagy enhancer, rapamycin, they showed accelerated MN degeneration and shortened lifespan [59]. On the other hand, in several other studies, autophagic inhibition by n-butyldeneptalide (n-BP) has yielded positive effects in ameliorating MN pathology in mSOD1 models. Specifically, n-BP treatment in SOD1<sup>G93A</sup> mice attenuated MN loss by preventing the abnormal accumulation of autophagosomes in MNs, as evidenced by the inhibition of the expression of a number of proteins: LC3-II, p62, and beclin1, as well as other autophagy-related proteins (Atg1/5) [60, 61].

It appears that autophagic disturbances related to mSOD1 vary depending on the phases of disease development, further complicating the association between mSOD1 and autophagic dysregulation. In a recent study where the essential autophagy gene ATG7 was conditionally knocked out in MNs of SOD1<sup>G93A</sup> mice, autophagic inhibition resulted in an earlier onset of disease phenotypes, suggesting that autophagy fulfills a neuroprotective role at the early stage of disease. Later, however, inhibition of autophagy extended the lifespan of mice and slowed disease progression, suggesting that autophagy plays a deleterious role at an advanced disease state [62]. This stage-dependent action of autophagy has also been demonstrated in another study, which indicated that starvation-induced autophagy markedly reduced neurotoxic mSOD1 aggregation at an early pathological stage, yet dramatically increased mSOD1 inclusions at a later point of disease in SOD1<sup>G93A</sup> mice [63]. Further studies are needed to resolve the precise effects of autophagic modulation in mSOD1-associated ALS pathology.

TDP-43 and FUS

A subset of ALS-associated genes encodes RNA-binding proteins (RBPs), including TAR DNA-binding protein 43 (TDP-43) and fused in sarcoma (FUS) [64–66]. Mutations in each of these genes are accountable for about 3% of fALS cases and less than 1% of sALS cases [67]. TDP-43 mutations are also identified in approximately 2% of FTD cases [68]. The connection between these RBPs and autophagy is in part indicated by their shared tendency to exhibit misfolding and aggregation [69, 70] that burden the autophagic system. Also, loss-of-function of these proteins contributes to dysregulations in autophagy [71–73]. Notwithstanding the many advances that have been made from studies of these RBPs in ALS, how the gain-and loss-of-function mechanisms of these RBPs contribute to ALS pathogenesis remains a complicated area of investigation.

### Gain of toxicity

The gain of toxicity of RBPs is directly linked to their role as constituents of the ubiquitinated protein inclusions observed in most forms of ALS. To date, TDP-43-positive inclusion buildup has been described as one of the most prevalent pathological hallmarks of ALS [1, 4, 65, 74]. The protein structures of TDP-43 and FUS share characteristics of low sequence complexity domains (LCs) [75, 76], which are regions where the majority of ALS-related mutations reside [65, 76]. The link between the function of LCs and RBP-positive inclusion formation has been recently studied, where mutations in LCs of both TDP-43 and FUS was shown to enhance inclusion formation [69, 77, 78]. LCs was reported to promote the sequestration of soluble proteins into membrane-less protein-rich liquid droplets, a phase transition event that was proposed to be vital for their normal functions but also can render them prone to aggregation [75, 77, 78]. As part of a normal cellular stress response, TDP-43 and FUS are transiently recruited at stress granules (SGs) for translation repression. Under chronic stress, however, their prolonged cytosolic

deposition can disrupt the structural homeostasis of SGs, leading to the formation of persistent inclusions [79–81].

As a mechanism for maintaining protein homeostasis, autophagy is known as an important pathway for the clearance of TDP-43 and FUS-positive protein inclusions [82–84]. Several studies have demonstrated that pharmacological induction of autophagy enhances the TDP-43 turnover rate as well as the rate of FUS-positive SG's clearance, in certain cases rescuing the ALS-related pathology [82, 85–87]. Rapamycin treatment reduced cell death in neurons expressing FUSR521C, an ALS-linked mutation [83]. In Drosophila and mouse models of TDP-43 proteinopathy, rapamycin-induced autophagy enhanced the clearance of TDP-43 aggregates while lessening ALS/FTD-like symptoms [85, 86].

RBPs and their inclusions can influence autophagy beyond being autophagic substrates. Overexpression of soluble TDP-43 in the cytoplasm was suggested to disrupt the VAPB-PTPIP51 tether, loosening ER-mitochondrial contact, subsequently enabling the recruitment of the early autophagic membrane and inducing autophagy [88, 89]. Several mutations have been reported to cause direct disruption of autophagy initiation. Mutations in the C-terminal domains of TDP-43 and FUS inhibited ER-Golgi vesicle transport through depletion of the vesicle-trafficking regulator Rab1, a small GTPase required for phagophore formation [90–92]. Additional data indicated that accumulation of FUS mutants inhibited autophagy, while overexpression of Rab1 reversed this disruption [93].

Loss of function

TDP-43 and FUS are primarily localized to the nucleus, where they carry out important functions that include RNA splicing, transcriptional regulation, and the maintenance of genome integrity [72, 73, 94–96]. Both LC and RNA recognition motif domains are essential for their native functions. For instance, LCs allow these proteins to cluster at DNA breaks, forming scaffolds for
the repair complex [75, 96]. As part of the cell's normal stress response, temporary translocation

of TDP-43 to the cytoplasm can exert a protective effect through autophagic upregulation, as part

of adaptation to maintain cellular homeostasis under stress. Transient discharge of TDP-43 from

the nucleus was shown to relieve the repression of the FOXO transcription factors, inducers of

autophagy [97]. However, under conditions of prolonged cellular stress such as those present in

neurodegeneration, the protective effect of TDP-43 cytosolic sequestration is thought to be

compromised, contributing to the overburdening of protein quality control [97].

In the case of ALS, chronic cytoplasmic sequestration of TDP-43 at ribonucleoprotein

(RNP) granules constitutes a loss-of-function in mRNA stabilization [72], directly influencing the

expression of TDP-43's target genes, among which are important players in autophagy.

Nevertheless, the overall impact of this loss-of-function for autophagic clearance remains unclear.

In several instances, TDP-43 ablation inhibits the expression of positive regulators of autophagy.

TDP-43 depletion was shown to downregulate the expression of the autophagic factor Atg7, an

E1-like enzyme essential for the lipidation of LC3-I and autophagosome formation [98]. TDP-43

depletion also inhibited autophagosome-lysosome fusion by depleting dynactin, a motor protein

essential for retrograde transport that enables vesicular fusion events in the pathway [99].

However, in other cases, the loss of TDP-43 function was reported to downregulate expression

of mTOR, a negative regulator of autophagy. In particular, TDP-43 nuclear depletion resulted in

reduced expression of Raptor, an mTOR adaptor protein. Downregulation of Raptor inhibits the

phosphorylation of transcription factor TFEB, resulting in its nuclear translocation and a

corresponding increase in the expression of autophagy and lysosome biosynthesis genes [99].

These concurrent events ultimately lead to the accumulation of autophagic vesicles observed in

TDP-43 knockdown cells [99].

In sum, it is clear that the loss of TDP-43 function leads to a complex array of gene

expression dysregulations that result in various aberrations to the events of autophagy; however,

the combined effect of these changes on autophagic clearance remains uncertain.
9
# SQSTM1, OPTN, and TBK1

Macroautophagy was thought to be a nonselective process, until the identification of autophagy receptors, integral components of the autophagic machinery that enable selective recruitment of specific cargo for degradation [100]. Mutations in SQSTM1/p62 and OPTN, two well-defined autophagy receptors, and in their activating kinase, TBK1, have been identified in multiple forms of ALS [7, 8, 101]. Specifically, mutations of these multi-functional proteins has been implicated in the impairment of the clearance of neurotoxic aggregates, as well as the disruption of inflammation regulation, conventional hallmarks of ALS [8, 102, 103]. p62 and OPTN share two common domains that are important for their role as autophagy receptors, the ubiquitin-associated domain (UBA) and the LC3-interacting region (LIR). The UBA domain allows these receptors to bind ubiquitinated cargo marked for degradation, and the LIR domain interacts with LC3, allowing the recruitment of receptors and cargo to the phagophore [100, 104, 105]. Distinctly, p62 contains another domain, PB1, which enables the oligomerization of p62 that is essential for the function of this protein in autophagy [106].

## SQSTM1/p62

Mutations in sequestosome 1, or p62, account for approximately 1% of ALS cases [101, 107]. Furthermore, about 2% of FTD patients in a French cohort were reported to carry p62 mutations [108]. p62 is known to be involved not only in selective autophagy but also in degradation through the ubiquitin-proteasome system (UPS) via its UBA domain [109–111]. p62 was first suggested to play a role in neurodegeneration due its enrichment in misfolded proteinaceous inclusions in many neuronal disorders including Alzheimer's disease and Parkinson's disease [112]. Accordingly, p62 has been widely used as a pathological marker to study these disorders. Notably, p62-positive inclusions have also been detected consistently in C9orf72-related ALS (C9ALS), the most common form of ALS [113–115].
Mutations in p62 were most commonly identified in Paget disease of bone (PDB), a skeletal disorder, occurring at a frequency of 20-50% in familial and 5-10% in sporadic PDB cases [102] with the majority of PDB-associated p62 mutations residing in the UBA domain. Similarly, p62 missense mutations of UBA were also identified in patients with ALS-FTD [5, 108], suggesting that disruptions in the interactions between p62 and ubiquitinated cargo are shared between ALS-FTD and PDB. However, it remains unclear whether these shared mutations affect autophagy in both diseases. For example, expression of the p62<sup>P392L</sup> mutant impaired basal autophagy in murine osteoclasts, implicating this mutation to impeded autophagic clearance in PBD [102]. By comparison, the implications of p62 UBA domain mutations on autophagy in the context of CNS disorders such as ALS-FTD remains to be studied.

Beyond the UBA domain, additional relatively rare ALS-associated p62 mutations are diffusely expressed in other regions of p62, including the LIR domain and p62 promoter region [6, 101]. Assessment of mutations in the LIR domain of p62 have particularly important implications for their possible contribution to disturbed autophagy in ALS, as this domain enables interactions with LC3, a central component in the recruitment of substrates to the autophagic membrane. A recent study reported that the ALS-associated mutation L341V in the LIR domain resulted in significantly reduced affinity of p62 for LC3-II, hindering its functional involvement with autophagic vesicles [116]. Furthermore, mutations detected in the promoter region of p62 of ALS/FTD patients suggest that reduced p62 expression could play a role in the autophagy dysfunction found in this pathological spectrum [117, 118].

In addition to canonical interactions between p62 and ubiquitinated inclusions, p62 also associates with mSOD1, offering insights as to how p62 may contribute to SOD1-related ALS. Early evidence of this interaction includes the observation that p62 expression and accumulation correlate with an increase in aggregation of mutant (G93A), but not wildtype, SOD1 [106]. This association was later reported to be independent of the UBA domain, instead interacting via a distinct SOD1 mutant interacting region (SMIR) to enable sequestration of mSOD1 without thepresence of the ubiquitin signal [32]. In murine models, depletion of p62 was reported to shorten

the lifespan of SOD1<sup>H46R</sup> and SOD1<sup>G93A</sup> mice [119]. In the SOD1<sup>H46R</sup> model, p62 loss aggravated

ALS symptoms, provoking accelerated weight loss, motor dysfunction, and MN degeneration.

Furthermore, a link between p62 and alsin, a causative gene in ALS, has also been suggested.

Depletion of alsin exacerbated the aforementioned contribution of p62 depletion to the

neuropathology of SOD1<sup>H46R</sup> mice [119].

**Optineurin**

Optineurin (OPTN) is a protein involved in a diverse range of cellular processes, including

vesicular trafficking, maintenance of the Golgi, NF-κB modulation, and antibacterial signaling

[120]. In addition, it has a well-characterized role as an autophagy receptor. Genetic analyses

have uncovered OPTN mutations in cohorts of ALS patients, which account for approximately 3%

of fALS and about 1% of sALS [64], raising questions as to the potential associations between

OPTN dysfunction, autophagic impairment, and ALS neuropathology. Rare ALS-associated

pathogenic variants of OPTN were initially reported, including deletion of OPTN exon 5, aQ398X

nonsense mutation, and an E478G missense mutation [7]. Later, insertion R96L, nonsense

Q165X, and truncation (c.1320delA) mutations of OPTN were detected, expanding the mutational

spectrum associated with ALS [121]. Interestingly, the majority of deleterious OPTN mutations

are found in the UBA region and occur in a heterozygous manner, potentially implicating

haploinsufficiency in the case of a loss-of-function mechanism in pathogenesis [105]. Despite

these findings, however, definitive mechanistic causes accounting for OPTN dysregulation and

consequent autophagic impediments are yet to be identified.

Several pathways involving mutant OPTN-induced autophagic impairment have been

proposed. In cell cultures, wildtype OPTN colocalizes with httx1 Q103 and SOD1<sup>G93C</sup> inclusion

bodies, toxic protein models of Huntington's disease and ALS, respectively [122]. Deletion of

OPTN induces increases in both httx1 Q103 and SOD1<sup>G93C</sup> inclusions, hinting at the involvement

12of OPTN in regulating the turnover of aggregation-prone proteins [122]. Specifically, OPTN recognition of misfolded proteins is mediated by the C-terminal coiled-coiled UBA domain, which enables K63-linked polyubiquitin-mediated autophagy. Furthermore, OPTN UBA mutations, including ALS-associated E478G, reduce degradation of polyubiquitinated inclusion bodies [122, 123]. In another study, OPTN<sup>E478G</sup> overexpression resulted in perturbed autophagic degradation, as shown by enhanced levels of lipidated LC3-II, with possible impaired autophagosome transport or fusion events occurring [124]. The proposed model to account for these findings posits that OPTN UBA mutants interact with wildtype OPTN, forming hybrid wildtype-mutant OPTN complexes that repress autophagosomal maturation [124]. Further in vitro studies in neuronal cultures have expanded these findings by demonstrating that the ALS-linked OPTN mutants E478G and Q398X failed to bind myosin VI (MYO6), an essential protein involved in regulating the intracellular transport of autophagosomes and endocytic/secretory vesicles [125, 126]. Diminished OPTN-MYO6 binding resulted in the accumulation of immature autophagosomes and conditions of ER stress, Golgi fragmentation, and impaired secretory protein trafficking in mutant-expressing cells. In contrast, wildtype OPTN linked sequestered autophagosomes to endosomes via recruitment of MYO6 and its cargo adaptor TOM1, allowing for efficient autophagosome maturation [126]. These results have been further supported by analyses of spinal cord fractions from ALS patients, in which OPTN-MYO6 interactions were absent [126]. Thus, the compromised binding of mutant OPTN to MYO6 may represent a late-stage inhibition of autophagosome fusion with lysosomes or endosomes.

There is additional evidence to suggest OPTN's involvement at more upstream nodes of autophagic regulation. It was shown that Ypt1, the yeast homolog of mammalian Rab1a, a GTPase involved in autophagosome formation, is a binding partner of OPTN [91, 127]. The Rab1a-OPTN association was dependent upon the GTP-binding capacity of Rab1a, along with the Rab1a zinc finger domain of OPTN [127]. After siRNA-mediated silencing of Rab1a, autophagosome formation is inhibited in neuroblastoma cells, suggesting that intact Rab1a is a

13
necessary component of autophagosome formation undertaken by OPTN [127]. Given that previous findings in cells expressing ALS-associated OPTNE478G have indicated an abolition of vesicle and autophagosome formation [128], dysfunction in Rab1a binding may underlie this pathology.

TBK1

TBK1 is a key player in various cellular processes including innate immunity, nucleic acid sensing pathways, and autophagy [129, 130]. Regarding autophagy, TBK1 operates primarily by phosphorylating the autophagic adaptors p62 and OPTN. TBK1 comprises four distinct domains, an N-terminal serine/threonine kinase domain, two coiled-coiled domains (CCD1/2), and a ubiquitin-like domain [131]. TBK's modulatory properties in autophagy were initially shown in cultured cells, with TBK1 phosphorylation of the UBA domain of OPTN on Ser-177 enhancing LC3 binding affinity and promoting autophagic clearance of cytosolic Salmonella in HeLa cells [132]. It has also been reported that TBK1 coordinates the autophagic machinery via phosphorylating Ser-403 on the UBA domain of p62, enhancing the affinity of UBA for K48/K63-linked ubiquitin chains, resulting in the autophagic clearance of mycobacteria in macrophages [133].

In ALS patients, exome sequencing has revealed that novel genetic aberrations of TBK1, encompassing loss-of-function mutations, are causal in pathogenesis. About 4% of fALS patients carry loss-of-function TBK1 mutations, however, these mutations are relatively rare (<0.1%) in sALS cases. Within a Belgian cohort, TBK1 mutations also account for about 1% of FTD patients [8, 134, 135]. A particularly pathogenic variant results in a (p.690-713) deletion in the C-terminal CCD2 domain of TBK1, within vitro models exhibiting an inability of truncated TBK1 to associate with OPTN [134]. In addition, other loss-of-function mutations occurring in the serine/threonine kinase domain have been reported in a cohort of French ALS and FTD patients. The pathogenicity of these variants is considered to be a result of haploinsufficiency, since they are heterozygous.
missense in nature [136]. Given the essential catalytic role of the serine/threonine domain, mutations in this region may repress the activation of p62 and OPTN. Nevertheless, since the majority of ALS-associated TBK1 mutations are missense mutations with largely unknown domain-specific interactions, further analyses are necessary to deduce exactly how the constituents of the broad spectrum of mutations contribute to the neurodegenerative state.

Elucidating the mechanisms by which TBK1 regulates autophagy may provide insights into how dysregulations of the autophagic pathway contribute to ALS pathology. TBK1 potentially plays a role in autophagic initiation, since its kinase activity has recently been reported to be necessary for the proper functioning of the C9orf72 complex, whose interactions with conventional autophagic initiation factors, including ULK1 and mTOR, were also shown [137–139]. TBK1 activates C9orf72 activity via phosphorylation of its binding partner, SMCR8. In cells, depletion of SMCR8 alters autophagy, causing an accumulation of p62-positive aggregates, whereas mimicking TBK1 phosphorylation corrects this autophagic dysfunction [140]. Furthermore, in a glaucoma-associated variant of OPTN, M98K, premature retinal cell death has been attributed to amplified TBK1 activity when compared to controls. Specifically, TBK1 phosphorylation at Ser-177 of OPTN induces autophagosome formation, increasing autophagic flux and causing retinal cell death [141]. Mutations in TBK1 may also impede later autophagic events, such as dynein-dependent autophagosome maturation along microtubules. TBK1 phosphorylation of the microtubule-binding protein CEP170 is necessary for centrosomal localization and binding to the microtubule depolymerase Kif2b, which regulates the association of NuMA with dynein that is required for microtubule activity [142]. Hence, loss of TBK1 function may contribute to ALS by disrupting the microtubular retrograde transport of autophagosomes, preventing autolysosome formation [131]. In addition, TBK1 activity plays an important regulatory role in the autophagy of damaged mitochondria, since constitutive interactions of TBK1 with OPTN induces the retention of OPTN/TBK1 on damaged mitochondria and result in an enhanced binding capacity of the UBA

15domain of OPTN with polyubiquitin chains, promoting mitophagy in cells by facilitating LC3-mediated linkage with phagophores [143, 144].

In sum, mutations in p62, OPTN, and TBK1 significantly impair autophagy through various mechanisms, resulting in ALS-relevant pathological features such as the accumulation of autophagic substrates and damaged organelles in MNs. A comprehensive understanding of the interplay between TBK1 and its effectors p62 and OPTN in autophagy may allow for targeted approaches aimed towards correcting aberrations resulting from alterations in the functions of these proteins.

### VAPB

Mutations in vesicle-associated membrane protein-associated protein B (VAPB) are relatively rare, with a mutation frequency of less than 0.6% in ALS patients [145, 146]. VAPB and VAPA together comprise the VAP protein family of ER tail-anchored transmembrane proteins. VAPB is associated with the membrane surrounding the ER and operates as a receptor for cytosolic protein recruitment to the ER surface. Interacting with FFAT-motif-containing polypeptides, VAPB plays diverse roles, being involved in such cellular functions as membrane trafficking, lipid transport and metabolism, membrane contact site formation, and neurotransmitter release. A rare missense mutation (P56S) in VAPB has been found to cause autosomal dominant ALS in a Brazilian cohort [10], with P56S transgenic mice later recapitulating this ALS-like pathology [147]. Another pathogenic mutation of VAPB (T46I) has been found in individuals with fALS, with this variant being speculated to impede the interaction of VAPB with cytosolic proteins by inducing dysfunctions in lipid metabolism [9]. Accumulating evidence has linked VAPB to autophagic regulation, raising the prospect that dysfunctions in VAPB may potentially underlie autophagic impairment in ALS pathogenesis.

Interactions between the ER and mitochondria are involved in mediating a range of physiological processes, including autophagy. These interactions arise as a result of ER-residentVAPB closely associating with mitochondrial protein PTPIP51, enabling ER membrane recruitment to mitochondria [88]. Autophagosomes are known to form at this ER-mitochondria interface, with the pre-autophagosome marker ATG14 and autophagosome formation marker ATG5 becoming localized to this site [148]. It was recently reported that the VAPB-PTPIP51 tether regulates autophagosome formation. Overexpression of VAPB and PTPIP51 induces a tightening of the tether and concomitant impairment of autophagosome formation. Conversely, siRNA-mediated loss of VAPB or PTPIP51 loosens the tether, stimulating autophagosome formation [89]. Interestingly, after chemically induced autophagy by rapamycin and torin 1, VAPB and PTPIP51 overexpression results in a reduction in the autophagic substrates LC3 and DFCP1, yet no such effect has been reported for cells subjected to starvation-induced autophagy [89]. Thus, the effects of VAPB-PTPIP51 in modulating autophagy appear to be dependent on the particular signaling mechanism by which stimuli induce autophagy. In addition, another means by which VAPB participates in autophagosome formation has been identified from co-immunoprecipitation assays in cells, which exhibited binding interactions between VAPB and Rab3GAP1, a protein influencing autophagosome biogenesis [149, 150].

In various models assessing the effects of ALS-associated P56S, persistent intracellular inclusions and alterations in ER structure are observed, perhaps providing the trigger for downstream autophagic dysregulations. Cells overexpressing P56S displayed significant ER restructuring, accompanied by inclusions continuous with the ER [151]. In a further characterization of these results in mice, P56S induced rapid oligomerization and aggregation of mutant VAPB [152]. These effects can be explained by the buildup of inclusions positive for mutant VAPB exceeding the capacity for ER-associated degradation, suggesting that VAPB dysfunctions contribute to impairments in ER quality control, in turn, leading to conditions of ER stress. Another study employing P56SVAPB knock-in mice reported that ER stress in murine models was coupled with evidence of impaired autophagy preceding the development of ALS-like pathology. This was shown by the significant colocalization of early autophagosome markers p62

17
and LC3 in cytoplasmic inclusions, along with accumulation of VAPBP56S within the cytosol of MNs, indicating VAPB mislocalization from the ER to cytosolic regions [153]. There is therefore reason to speculate that perturbed autophagy may be the result of ER stress.

### VCP

Valosin-containing protein (VCP/p97) is an essential AAA+-ATPase involved in a range of cellular processes that include DNA replication and repair, cell cycle regulation, and protein clearance [154]. Regarding the latter, VCP is integral to ERAD, a process in which unwanted proteins are extracted from the ER for proteasomal degradation [155]. Expanding VCP's role in proteostasis, emerging findings now point to the involvement of VCP in autophagy [156, 157].

Mutations of VCP are causal in inclusion body myopathy, Paget's disease, and frontotemporal dementia (IBMPFD) [158]. Broadening the phenotypic spectrum of VCP mutations to ALS, whole-exome sequencing has revealed the existence of several VCP mutations (R155H, D592N, R191Q, R159G), which account for approximately 1-2% of fALS cases [11, 159]. Initial evidence suggesting VCP's involvement in autophagy has come from muscle lysates of IBMPFD patients expressing IBMPFD/ALS-associated R155H. Analysis of patient tissue revealed the accumulation of autophagosomes colocalized with p62 and LC3, coalescing at rimmed vacuoles [156]. Since then, additional studies in mouse models expressing pathogenic VCP variants have elicited similar observations [157, 160, 161]. Nevertheless, the precise mode of involvement of VCP in autophagy is a subject of debate.

There are several proposed mechanisms by which VCP interacts with the autophagic machinery. With regard to autophagic initiation, in neuronal cell cultures subjected to insulin withdrawal-induced induction of autophagy, VCP expression repressed the negative autophagy regulators calpain 2 and mTOR, while upregulating GSK-3β, a positive autophagic regulator [162]. Moreover, two direct targets of mTOR, EIF4EBP1 and RPS6KB1/p70S6, were found to be decreased in a murine VCP<sup>R155H</sup> model. Reduced mTOR activity was consistent with the

enhanced autophagic activation and autophagosome biogenesis apparent in the model [163].

Additional studies have suggested a role for VCP in downstream autophagosome maturation.

Proteomic analysis in IBMPFD murine muscle lysates and neuronal cells revealed that VCP loss in addition to presence of VCP<sup>R155H</sup> resulted in the failure of autophagosomes to mature into autolysosomes, causing accumulation of non-degradative autophagosomes, ubiquitinated aggregates, and p62-positive inclusions [156, 157]. Other evidence suggests that mutant VCP interferes with processes following autophagosome-lysosome fusion, since cells expressing ALS/IBMPFD-linked VCP<sup>R155H</sup> and VCP<sup>A232E</sup> mutations exhibit accumulations of irregular, enlarged autophagic vacuoles possessing acidified autophagosomes, implying that autophagic failure occurs at a later point, following acidification and delivery of lysosomal hydrolases [157].

Stress granule clearance represents yet another juncture in autophagy that VCP is suspected to influence. This possibility was recognized in yeast models in which the VCP ortholog Cdc48 promoted targeting of SGs for autophagic degradation [164]. These results have also been noted in HeLa cells, in which cell cultures expressing ALS/IBMPFD-associated VCP mutations were found to constitutively accumulate pathogenic TDP-43-positive SGs; in contrast, in wildtype cells, VCP regulated the amount of TDP-43-containing RNP complexes, facilitating their disassembly and targeting to autophagosomes [164].

Based on the assessment of ALS-related VCP dysfunctions, it is evident that autophagic impairment is a common theme underlying neuropathology. Extensive evidence suggests that VCP's role in autophagy is multifaceted and varied, necessitating clarification as to which autophagic processes are directly regulated by VCP and what the specific consequences of dysregulation are.

C9orf72

A GGGGCC hexanucleotide repeat expansion in C9orf72 is the most common cause of ALS [21, 22]. Notably, the frequency of C9orf72 repeat expansion varies between ALS cohorts,
with very high prevalence in European cohorts (~33% in fALS, ~5% in sALS) and lower prevalence in Asian cohorts (~2% in fALS, ~0.3% in sALS). In FTD patients, the frequency of C9orf72 repeat expansion also varies within a range of 3% to 12%, depending on the studied cohort [67, 165]. To date, there are three major mechanistic components that have been reported to contribute to the pathogenesis of C9ALS: (1) the loss or reduction in expression of the C9orf72 protein [21, 166], (2) the toxicity mediated by C9orf72 repeat expansion sense/antisense RNA transcripts [167, 168], and (3) the generation of repeat dipeptides from non-ATG translation of C9orf72 repeat expansion [169–171]. The sensitivity of patients' cells to inhibition of autophagy was exacerbated as a result of the presence of C9orf72 repeat expansion, although the mechanism is unknown [172]. Regarding the role of C9orf72 repeat expansion in autophagy, gain-of-function components (2) and (3) may exert indirect influences that perturb the autophagy-lysosomal pathway. By sequestering important RBPs such as TDP-43 in nuclear RNA foci and preventing nuclear import via disruption of nucleocytoplasmic transport, the native function of these proteins can be compromised and the autophagy-lysosomal pathway influenced as a result [79, 167, 173–175]. In this section, however, we will focus on the direct influence of the repeat expansion on autophagy via (1) or through C9orf72 protein reduction or depletion.

Since the discovery of C9orf72 repeat expansion, the transcript level of the C9orf72 protein has been recognized as being reduced in the frontal and cerebellar cortices of C9ALS patients [21, 166], with later studies reporting a reduction in the protein levels in patients' brains [176]. Homology analysis of the C9orf72 protein sequence revealed a DENN-like domain in its structure suggesting that C9orf72 may play a role as a guanine exchange factor (GEF) for Rab GTPases—key players in vesicular trafficking [177–179] that are extensively involved at different events of macroautophagy [92]. This role of C9orf72 in Rab-mediated events in the autophagy-lysosomal pathway was supported by the observation that C9orf72 interacts with Rab1, Rab5, Rab7, and Rab11, small GTPases known to be involved in autophagy [180, 181]. In addition, a role for C9orf72 in modulating filamentous actin (F-actin) assembly was recently suggested [182].F-actin is known to play central roles in the structure and dynamics of the cytoskeleton, the maintenance of which may be critical in the autophagy-lysosomal pathway [183–185], raising the possibility of another mechanism through which C9orf72 may affect autophagy.

In an interaction essential to its known functions, C9orf72 forms complexes with SMCR8, another DENN domain protein [137–140, 186]. Multiple studies have described the many events that this complex could potentially influence within the cascade of autophagy including autophagic induction, nutrient-sensing pathways, and lysosome biogenesis/clearance [137–139, 186].

Depletion of C9orf72 protein appears to inhibit autophagy initiation via the ULK1-mediated pathway [137, 187]. C9orf72 was reported to interact with the ULK1/FIP100/ATG13/ATG101 complex in multiple studies [137, 186–188]. Loss of C9orf72 expression may disrupt the induction of ULK1-mediated autophagy via two events: Firstly, C9orf72 was suggested to act as an effector protein for Rab1a, a GTPase known to be recruited for phagophore formation [92]. Through its interaction with the ULK1 complex, C9orf72 could regulate the Rab1a-dependent trafficking of ULK1 to the autophagic initiation site [187]. Secondly, C9orf72 may be essential for relaying signals from the ULK1 complex to promote downstream Rab-mediated phagophore maturation.

The C9orf72-SMCR8 complex was shown to act as a GEF toward Rab8a and Rab39b [140], with expression of phosphomimetic Rab39b being found to rescue the inhibition of autophagic induction caused by C9orf72 depletion. This finding suggested that Rab39b could act immediately downstream of ULK1, whose phosphorylation via C9orf72 is required for autophagy initiation [137, 140]. Notably, a recent study reported that the loss of SMCR8, C9orf72's binding partner, resulted in an upregulation of ULK1 expression, characterizing SMCR8 as a negative regulator of autophagy induction [188]. In the same study, however, SMCR8 also exhibited another independent function in promoting autophagosome maturation [188]. The multitude of roles in autophagy, exhibited by C9orf72 and SMCR8 that results in various disturbance to the events of the autophagy cascade further demonstrate the complexity in function of the C9orf72 complex.Autophagy is a crucial mechanism by which the cell adapts to its environment by managing and recycling its metabolism according to the availability of nutrients. The autophagy-lysosomal pathway, therefore, is directly influenced by nutrient-sensing signaling pathways, of which the mTOR-mediated pathway is a major regulator. Loss of C9orf72 function was reported to cause a defect in mTOR signaling pathway [138, 139]. In the absence of C9orf72, mTOR, a negative regulator of autophagy, was inactivated [139]. In two separate studies, it was observed that mTOR inactivation led to the nuclear translocation of transcription factor TFEB, a master inducer of autophagy and lysosome biogenesis genes, resulting in an increase in autophagic flux [137, 138]. Notably, this increase in autophagic flux was not observed in torin 1-induced autophagy [187], suggesting that there are two distinct pathways in starvation and in chemical-induced autophagy where C9orf72 may play different roles.

The potential role of C9orf72 in the lysosome pathway has been suggested by its localization to this organelle [180]. This localization was reported to be negatively regulated by amino acid availability, suggesting potentially unknown roles of the protein at the lysosome. Interestingly, inhibition of mTOR via torin 1 treatment did not result in the localization of C9orf72 to lysosomes, implicating that the lysosomal function of the protein is associated with starvation-induced, but not chemically-induced autophagy [139]. Furthermore, based on evidence from multiple studies, lysosome biogenesis has been suggested to be upregulated as a result of C9orf72 loss. Supporting this in cell models, ablation of C9orf72 resulted in swollen lysosomes that cluster near the nucleus [139]. Also, in liver lysates from C9orf72-knockout mice, a buildup of lysosomal markers, including LC3-I, LAMP1, and PSAP, was reported [138, 186]. In another study, aberrantly enlarged lysosomes were observed in bone marrow-derived macrophages and microglia from C9orf72-knockout mice [189]. Upregulation of TFEB, an inducer of lysosome biogenesis via inactivation of the mTOR pathway, has been suggested to contribute to the aforementioned abnormalities in lysosomal morphology and lysosomal marker levels in C9orf72-knockout models [138].

22
In summary, C9orf72's role in autophagy is multifaceted, and the ultimate consequence of C9orf72 downregulation related to the repeat expansion remains undefined. On one hand, C9orf72 has been shown to mediate autophagic induction in conjunction with the ULK1 complex, yet on the other hand, C9orf72 modulates mTOR and lysosomal functions. Together, the function of C9orf72 in the autophagy-lysosome pathways needs to be further studies.

## Other ALS genes

### UBQLN2

Ubiquilin 2 (UBQLN2) is part of the ubiquilin family of proteins involved in proteasome-mediated protein degradation via activity of ubiquitin-associated (UBA) and ubiquitin-like (UBL) domains. UBQLN2 may also operate as an autophagic receptor, with reported interactions between UBQLN2 and LC3-II [190]. Deletion of UBQLN2's UBL domain abrogates its association with LC3-II [191]. According to studies in French/French-Canadian populations, UBQLN2 mutations are relatively rare and mostly found in sALS patients (~0.4%) [192, 193]. UBQLN2 mutations were also found in about 0.3% of FTD cases in an extended Belgian cohort [194]. Most ALS/FTD-associated UBQLN2 mutations are clustered in UBQLN2's proline repeat region [12], although they are also found elsewhere [192, 195]. This then raises the question if autophagic dysfunction due to mutant UBQLN2 may contribute to ALS pathogenesis.

In vitro studies have conferred limited support for disturbed autophagic pathways by UBQLN2 mutations. Cells overexpressing mutant UBQLN2 showed diminished colocalization between UBQLN2 and ATG9/ATG16L1, essential factors in autophagosome-lysosome fusion [196]. Also, although wildtype UBQLN2 is found in endosomal vesicles colocalizing with p62 and autophagy initiator ULK1, ALS-linked UBQLN2 variant E478 abolished vesicle formation, inhibited endosomal trafficking, and increased inclusion body formation [128]. Similar findings have been recapitulated in animal models, where overexpression of ALS-associated UBQLN2P497H accelerated MN loss in rats via possible endosomal pathway impairment, evidenced by reduced
early endosome antigen 1 [197], a necessary component of autophagosome maturation [198].

Aberrant accumulation of autophagy substrates p62 and LC3-II are also present in neural tissue of rats overexpressing UBQLN2<sup>P497H</sup> [197, 199]. Despite these findings, the role of UBQLN2 in the context of autophagy remains controversial, and it is unclear by which precise mechanisms UBQLN2 may regulate autophagic flux.

### Als in

Mutations in ALS2, which encodes the protein alsin, have been identified in a rare juvenile-onset form of ALS [14]. Als in is constitutively expressed and is localized to the cytosolic side of the endosomal membrane [29, 200]. Its protein structure includes a Vps9 domain, which is crucial for its GEF role with regard to Rab5, a regulator of endosomal vesicle transport and fusion [200–204]. Als in-mediated endosome trafficking and vesicular fusion were evidenced to be important events within the autophagy cascade that particularly influence the maturation of autophagosomes and the formation of amphisomes, two events preceding fusion with the lysosome and vesicular clearance [205]. Consistently, loss of alsin was reported to exacerbate the neurotoxicity of protein aggregates in murine models of ALS, including SQSTM1/p62-null and mSOD1 mice models. Als in ablation led to elevated SOD1<sup>H46R</sup> protein buildup and toxicity by reducing autophagic clearance, which was a result of impaired endosomal trafficking and disrupted maturation of autophagosomes [206]. More recently, loss of alsin was also reported to intensify the effect of p62 deletion in SOD1<sup>H46R</sup> transgenic mice [119]. This modifier effect of alsin on p62 depletion further suggests that intact alsin-mediated endosomal functions are required for proper autophagic clearance.

### CHMP2B

Charged multivesicular protein 2B (CHMP2B) is a subunit of the endosomal sorting complex required for transport (ESCRT-III), whose mutations were identified in less than 1% of a large cohort of ALS patients in North England [17]. In autophagy, dissociation of ESCRT-III fromthe endosomal membranes generates multivesicular bodies (MVBs) that fuse with autophagosomes, producing amphisomes that later fuse with lysosomes [207, 208]. CHMP2B recycles ESCRT-III and mediates its dissociation via the recruitment of Vps 4, an ATPase that facilitates membrane abscission and MVB formation. Sequencing of CHMP2BinFTD patients has revealed a splice-site mutation that produces two aberrant transcripts (CHMP2B^Intron 5, CHMP2B^10) that result in C-terminal domain truncations and significant p62 and ubiquitin deposition in patient brains [15]. CHMP2B mutations (T104N, I29V, Q206H) have also been found in ALS cohorts, with patients' brains exhibiting immature autophagosome accumulation and upregulated LC3-II and p62 levels, implicating autophagic impairment as a common neuropathological pathway [16, 17].

Various studies of autophagic disruption in CHMP2B models have recapitulated the observations made in clinical ALS-FTD cases marked by CHMP2B dysfunctions. In cells, CHMP2B^Intron 5 overexpression inhibited autophagic clearance, inducing the formation of ubiquitin- and p62-positive inclusions [209, 210]. Similar results were reported in a CHMP2B^Intron 5 murine model [211]. C-terminal CHMBP2B truncations may underlie autophagic aberrations, given the role of the autoinhibitory C-terminal region in preventing self- or abnormal interactions with other ESCRT components [212]. Consistent with this explanation, truncated CHMP2B^Intron 5 in cells is avidly associated with mSnf 7-2, a key constituent of ESCRT-III, and together they form abnormal complexes that prevent ESCRT-III dissociation from the endosomal membranes and downstream autolysosomal fusion [184, 210]. In addition, in cell models, CHMP2B^Intron 5 displays impaired endosomal recruitment of Rab7, a GTPase required for endosome-lysosome fusion [213]. Similarly, another C-terminal truncation (ALS-associated CHMP2BT104N), accumulates in Rab5- and Rab7-positive endosomes, while also closely complexing with mSnf 7-2 in cortical neurons, indicating defective endocytic trafficking [212]. There is also evidence suggesting a more upstream consequence of CHMP2B mutations in autophagy, with experiments in a Drosophila model indicating that CHMP2B^Intron 5 inhibits autophagosome maturation by repressing binding

25
interactions between Vti1a and STX13, an endosomal SNARE involved in the maturation of phagophores into closed autophagosomes [214]. Thus, C-terminal CHMP2B truncations appear to exert deleterious effects that disrupt various steps in autophagy, primarily at the late-stage endosome-lysosome fusion. Further analysis is needed to decipher the role this disruption may play in ALS pathogenesis.

Dynactin

Dynactin (DCTN) is a protein found in complex with molecular motor dynein, both of which are required for retrograde transport of autophagic substrates. Mutations of DCTN are infrequent in fALS, but estimated to vary from 0.5–3% in different sALS cohorts [215]. Hinting at the involvement of dynactin mutations in the motor neuron disease, a G59S mutation in the largest subunit of dynactin, p150Glued, has been detected in patients with lower MN pathology, with patient neural tissue displaying abnormal dynein and dynactin inclusions and significant MN degeneration [18, 216]. Additional ALS-associated DCTN mutations include T1249I and M751T [217].

Neuropathological features of dynactin mutations have been associated with autophagic impairment resulting from repressed retrograde transport. The G59S mutation of p150Glued inhibits dynein-dynactin-mediated retrograde transport by reducing the affinity of p150Glued for microtubules and microtubule plus-end binding protein (EB1) in transfected MNs, thereby decreasing the activity of the dynein complex. This effect results in G59S aggregation, perhaps as the result of insufficient molecular chaperone recruitment [218]. Further expanding these findings, a study of G59S transgenic mice revealed that these mice exhibit dysfunctional intracellular vesicular trafficking, since G59S p150Glued impedes the targeting of cargo to its intended destination and causes inclusion formation (specifically, autophagosomes enriched in LC3-II and ubiquitin) [219]. Also supporting the role of dynactin in autophagy, DCTN knockdown in C. elegans markedly decreased the transport of autophagosomes and significantly reduced both the speed and distance of retrograde transport. These effects induce the accumulation of

immature autophagosomes and subsequent MN degeneration [220], modeling pathological features seen after DCTN silencing in ALS patients [220, 221].

Interestingly, studies employing various models have suggested that the autophagy-related pathological features of dynactin mutations may be ameliorated by autophagic enhancement. In G59S p150Glued transgenic mice, consumption of a low-protein diet largely alleviates MN pathology, as a result of diet-induced autophagic induction and balancing of autophagic flux, indicated by normalized levels of LC3-II, decreased immature autophagosome formation and reduced p62-positive inclusions [222]. Similarly, the autophagic inducer rapamycin attenuated neurodegeneration and locomotor dysfunction in cultured MNs [220]. Taken together, these data suggest that autophagic modulators may compensate for retrograde transport deficits brought on by dynactin mutations.

Profilin 1

Profilin 1 (PFN1) is an actin-binding protein that facilitates the binding and conversion of actin-ADP monomers into actin-ATP, enabling its addition to growing actin filaments. PFN1 also binds to phosphoinositides and polyproline-rich ligands [223]. Collectively, these binding interactions underscore PFN1's role in regulating cytoskeletal dynamics, membrane trafficking, and neuronal synaptic activity [224]. Emerging evidence has revealed a link between mutations of PFN1 and ALS, with whole-exome sequencing of individuals with fALS uncovering C71G, M114T, and G118V mutants of PFN1 [19]. Since then, other ALS-related pathogenic variants (R136W, T109M, A20T, Q139L) of PFN1 have been found [225–227]. Overall, mutations of PFN1 were found in 1–2% of fALS patients. In addition, they were also identified in less than 1% of FTD cases from an extended Flanders-Belgian cohort [194, 228]. Various models assessing the effects of mutant PFN1 have revealed interactions of PFN1 with autophagic substrates, raising the possibility that autophagic dysregulations may be at play in mutant PFN1-related neuropathology. In transfected cells and primary mouse neurons, mutant PFN1 aggregates are reported to be

positive for p62, ubiquitin, and TDP-43 [19]. Also, autophagy adaptor LC3 is sequestered in PFN1 aggregates, repressing the conversion of LC3-I to LC3-II in a potentially p62-dependent manner [229]. In transgenic mice expressing the PFN1-C71G mutant, which has been shown to possess a highly disordered non-native helical conformation [230], severe MN loss and elevated p62 levels are observed [231]. Although these studies based on overexpression models suggest a gain-of-function mechanism, the involvement of loss-of-function in disease, as suggested by the observation that mutant PFN1 displays lower levels of bound actin and inhibited axonal outgrowth would also be possible [19].

FIG4

Factor-induced gene 4 (FIG4) encodes the protein Fig4, the phosphatase that modifies phosphatidylinositol-3,5,-bisphosphate (PI(3,5)P2), thereby regulating the level of this important signaling lipid molecule of the endosomal pathway. Located on the cytosolic side of the endosomal membrane, (PI(3,5)P2) mediates retrograde trafficking of endosomes to the Golgi apparatus [232, 233]. Mutations in FIG4 have been identified in 1-3% of ALS patients from central Europe, thereby qualifying as contributing factors for ALS [20, 234]. These mutations are often deleterious, causing loss-of-function of the protein. Loss of Fig4 function is known to cause dysfunction in the activity of the PI(3,5)P2 biosynthesis complex, thereby resulting in a decrease in PI(3,5)P2 levels [235, 236]. PI(3,5)P2 depletion, in turn, results in enlarged endosomal vesicles and defects in the endo-lysosomal pathway [235, 237, 238].

There have been reports of Fig4 playing a regulatory role in the autophagy-lysosomal pathway. In Drosophila models, Fig4 ablation induced lysosomal enlargement and abnormal level of lysosomal markers, resulting in a shorter lifespan and mobility defects [206]. The effect of Fig4 depletion on autophagic clearance has been further substantiated in a mouse model, in which p62/SQSTM1 and LC3-II were observed to accumulate in neurons and astrocytes of FIG4-knockout mice. A defect in autolysosome clearance was also suggested by an abnormal

accumulation of LAMP-2-positive vesicles in cultured FIG4-knockout neurons and astrocytes

[239]. Together with other endosome-associated proteins such as CHMP2B and alsin, the linkage of Fig 4 to ALS further underscores the importance of endocytosis and autophagy in neurodegeneration.

## Perspective

The common theme of autophagy defects presents an opportunity to develop a broad-spectrum strategy that could be effective for many forms of ALS (Table 1). Here, we have reviewed the main evidence linking autophagy to ALS suggesting that autophagy is a stress response whose balance may be a key to pathogenesis and treatment. Elucidating a clear connection between apparent dysregulations in autophagy and ALS pathophysiology presents numerous challenges, given that ALS is governed by a wide array of causative genes. ALS-related mutations within a subset of autophagic genes deleteriously affect different junctures of autophagy, encompassing autophagic initiation, autophagosome maturation and transport, endosome trafficking, and autophagosome-lysosome fusion. Autophagy destabilization likewise can arise from mutations in genes not canonically involved in autophagy, by way of diminished autophagic gene expression, retrograde transport impairment, or aggregation-induced overburdening of autophagic clearance. Indeed, the precise sequence of events resulting in autophagic failure depends largely on the particular context of the disease state. Both excessive and inadequate autophagic flux have been connected to ALS pathology, and autophagy may either alleviate or exacerbate the disease process at different stages. Consequently, accurately defining the nature of autophagic defects and the cell's response to these alterations becomes increasingly important because it enables an integrated understanding of the autophagic dysregulation and proteostatic imbalance observed in ALS. Therefore, further studies are needed to provide a systematic analysis of the range of genetic players involved in autophagy and

associated with ALS. These efforts may not only resolve current gaps in the understanding of

autophagy in ALS but also suggest mechanism-based therapeutic measures.

Acknowledgements

We are grateful to Ernest Wang and Elizabeth Alexander for their contributions in the preparation of the manuscript. This work was supported by NIH (NS074324, NS089616), the Robert Packard Center for ALS Research at Johns Hopkins, the ALS Association, The Muscular Dystrophy Association, Department of Defense, and The Judith & Jean Page Adams Charitable Foundation.

Author Contributions

D.N., R.T., and J.W. wrote the paper. D.N. and R.T. designed the figure with suggestions from J.W.

References

[1] D.R. Rosen, T. Siddique, D. Patterson, D.A. Figlewicz, P. Sapp, A. Hentati, D. Donaldson, J. Goto, J.P. O'Regan, H.-X. Deng, Z. Rahmani, A. Krizus, D. McKenna-Yasek, A. Cayabyab, S.M. Gaston, R. Berger, R.E. Tanzi, J.J. Halperin, B. Herzfeldt, R. Van den Bergh, W.-Y. Hung, T. Bird, G. Deng, D.W. Mulder, C. Smyth, N.G. Laing, E. Soriano, M.A. Pericak-Vance, J. Haines, G.A. Rouleau, J.S. Gusella, H.R. Horvitz, R.H. Brown, Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis, Nature. 362 (1993) 59–62. doi:10.1038/362059a0.

[2] M. Neumann, D.M. Sampathu, L.K. Kwong, A.C. Truax, M.C. Micsenyi, T.T. Chou, J. Bruce, T. Schuck, M. Grossman, C.M. Clark, L.F. McCluskey, B.L. Miller, E. Masliah, I.R. Mackenzie, H. Feldman, W. Feiden, H. a Kretzschmar, J.Q. Trojanowski, V.M.-Y. Lee, Ubiquitinated TDP-43 in Frontotemporal Lobar Degeneration and Amyotrophic Lateral Sclerosis, Science (80-.). 314 (2006) 130–133. doi:10.1126/science.1134108.

[3] J. Sreedharan, I.P. Blair, V.B. Tripathi, X. Hu, C. Vance, B. Rogelj, S. Ackerley, J.C. Durnall, K.L. Williams, E. Buratti, F. Baralle, J. de Belleroche, J.D. Mitchell, P.N. Leigh, A. Al-Chalabi, C.C. Miller, G. Nicholson, C.E. Shaw, TDP-43 Mutations in Familial and Sporadic Amyotrophic Lateral Sclerosis, Science (80-.). 319 (2008) 1668–1672. doi:10.1126/science.1154584.

[4] T.J. Kwiatkowski, D.A. Bosco, A.L. LeClerc, E. Tamrazian, C.R. Vanderburg, C. Russ, A. Davis, J. Gilchrist, E.J. Kasarskis, T. Munsat, P. Valdmanis, G.A. Rouleau, B.A. Hosler, P. Cortelli, P.J. de Jong, Y. Yoshinaga, J.L. Haines, M.A. Pericak-Vance, J. Yan, N. Ticozzi, T. Siddique, D. McKenna-Yasek, P.C. Sapp, H.R. Horvitz, J.E. Landers, R.H. Brown, Mutations in the FUS/TLS Gene on Chromosome 16 Cause Familial Amyotrophic Lateral Sclerosis, Science (80-.). 323 (2009) 1205–1208. doi:10.1126/science.1166066.

[5] F. Fecto, T. Siddique, UBQLN2/P62 cellular recycling pathways in amyotrophic lateral sclerosis and frontotemporal dementia, Muscle and Nerve. 45 (2012) 157–162. doi:10.1002/mus.23278.

[6] E. Teyssou, T. Takeda, V. Lebon, S. Boillée, B. Doukouré, G. Bataillon, V. Sazdovitch, C. Cazeneuve, V. Meininge, E. Leguern, F. Salachas, D. Seilhean, S. Millecamps, Mutations in SQSTM1 encoding p62 in amyotrophic lateral sclerosis: Genetics and neuropathology, Acta Neuropathol. 125 (2013) 511–522. doi:10.1007/s00401-013-1090-0.

[7] H. Maruyama, H. Morino, H. Ito, Y. Izumi, H. Kato, Y. Watanabe, Y. Kinoshita, M. Kamada, H. Nodera, H. Suzuki, O. Komure, S. Matsuura, K. Kobatake, N. Morimoto, K. Abe, N. Suzuki, M. Aoki, A. Kawata, T. Hirai, T. Kato, K. Ogasawara, A. Hirano, T. Takumi, H. Kusaka, K. Hagiwara, R. Kaji, H. Kawakami, Mutations of optineurin in amyotrophic lateral sclerosis, Nature. 465 (2010) 223–226. doi:10.1038/nature08971.

[8] E.T. Cirulli, B.N. Lasseigne, S. Petrovski, P.C. Sapp, P.A. Dion, C.S. Leblond, J. Couthouis, Y.-F. Lu, Q. Wang, B.J. Krueger, Z. Ren, J. Keebler, Y. Han, S.E. Levy, B.E. Boone, J.R. Wimbish, L.L. Waite, A.L. Jones, J.P. Carulli, A.G. Day-Williams, J.F. Staropoli, W.W. Xin, A. Chesi, A.R. Raphael, D. McKenna-Yasek, J. Cady, J.M.B. Vianney de Jong, K.P. Kenna, B.N. Smith, S. Topp, J. Miller, A. Gkazi, FALS Sequencing Consortium, A. Al-Chalabi, L.H. van den Berg, J. Veldink, V.

Silani, N. Ticozzi, C.E. Shaw, R.H. Baloh, S. Appel, E. Simpson, C. Lagier-Tourenne, S.M. Pulst, S. Gibson, J.Q. Trojanowski, L. Elman, L. McCluskey, M. Grossman, N.A. Shneider, W.K. Chung, J.M. Ravits, J.D. Glass, K.B. Sims, V.M. Van Deerlin, T. Maniatis, S.D. Hayes, A. Ordureau, S. Swarup, J. Landers, F. Baas, A.S. Allen, R.S. Bedlack, J.W. Harper, A.D. Gitler, G.A. Rouleau, R. Brown, M.B. Harms, G.M. Cooper, T. Harris, R.M. Myers, D.B. Goldstein, Exome sequencing in amyotrophic lateral sclerosis identifies risk genes and pathways., Science. 347 (2015) 1436–41. doi:10.1126/science.aaa3650.

[9] H.-J. Chen, G. Anagnostou, A. Chai, J. Withers, A. Morris, J. Adhikaree, G. Pennetta, J.S. de Belleruche, Characterization of the properties of a novel mutation in VAPB in familial amyotrophic lateral sclerosis., J. Biol. Chem. 285 (2010) 40266–81. doi:10.1074/jbc.M110.161398.

[10] A.L. Nishimura, M. Mitne-Neto, H.C.A. Silva, A. Richieri-Costa, S. Middleton, D. Cascio, F. Kok, J.R.M. Oliveira, T. Gillingwater, J. Webb, P. Skehel, M. Zatz, A Mutation in the Vesicle-Trafficking Protein VAPB Causes Late-Onset Spinal Muscular Atrophy and Amyotrophic Lateral Sclerosis, Am. J. Hum. Genet. 75 (2004) 822–831. doi:10.1086/425287.

[11] J.O. Johnson, J. Mandrioli, M. Benatar, Y. Abramzon, V.M. Van Deerlin, J.Q. Trojanowski, J.R. Gibbs, M. Brunetti, S. Gronka, J. Wuu, J. Ding, L. McCluskey, M. Martinez-Lage, D. Falcone, D.G. Hernandez, S. Arepalli, S. Chong, J.C. Schymick, J. Rothstein, F. Landi, Y.D. Wang, A. Calvo, G. Mora, M. Sabatelli, M.R. Monsurrò, S. Battistini, F. Salvi, R. Spataro, P. Sola, G. Borghero, G. Galassi, S.W. Scholz, J.P. Taylor, G. Restagno, A. Chiò, B.J. Traynor, Exome Sequencing Reveals VCP Mutations as a Cause of Familial ALS, Neuron. 68 (2010) 857–864. doi:10.1016/j.neuron.2010.11.036.

[12] H.-X. Deng, W. Chen, S.-T. Hong, K.M. Boycott, G.H. Gorrie, N. Siddique, Y. Yang, F. Fecto, Y. Shi, H. Zhai, H. Jiang, M. Hirano, E. Rampersaud, G.H. Jansen, S. Donkervoort, E.H. Bigio, B.R. Brooks, K. Ajroud, R.L. Sufit, J.L. Haines, E. Mugnaini, M.A. Pericak-Vance, T. Siddique, Mutations in UBQLN2 cause dominant X-linked juvenile and adult-onset ALS and ALS/dementia, Nature. 477 (2011) 211–215. doi:10.1038/nature10353.

[13] S. Hadano, C.K. Hand, H. Osuga, Y. Yanagisawa, A. Otomo, R.S. Devon, N. Miyamoto, J. Showguchi-Miyata, Y. Okada, R. Singaraja, D. a Figlewicz, T. Kwiatkowski, B. a Hosler, T. Sagie, J. Skaug, J. Nasir, R.H. Brown, S.W. Scherer, G. a Rouleau, M.R. Hayden, J.E. Ikeda, A gene encoding a putative GTPase regulator is mutated in familial amyotrophic lateral sclerosis 2., Nat. Genet. 29 (2001) 166–173. doi:10.1038/ng1001-166.

[14] Y. Yang, A. Hentati, H.X. Deng, O. Dabbagh, T. Sasaki, M. Hirano, W.Y. Hung, K. Ouahchi, J. Yan, A.C. Azim, N. Cole, G. Gascon, A. Yagmour, M. Ben-Hamida, M. Pericak-Vance, F. Hentati, T. Siddique, The gene encoding alsin, a protein with three guanine-nucleotide exchange factor domains, is mutated in a form of recessive amyotrophic lateral sclerosis., Nat. Genet. 29 (2001) 160–165. doi:10.1038/ng1001-160.

[15] G. Skibinski, N.J. Parkinson, J.M. Brown, L. Chakrabarti, S.L. Lloyd, H. Hummerich, J.E. Nielsen, J.R. Hodges, M.G. Spillantini, T. Thusgaard, S. Brandner, A. Brun, M.N. Rossor, A. Gade, P. Johannsen, S.A. Sørensen, S. Gydesen, E.M. Fisher, J. Collinge, Mutations in the endosomal ESCRTIII-complex subunit CHMP2B in frontotemporal dementia, Nat. Genet. 37 (2005) 806–808. doi:10.1038/ng1609.

[16] N. Parkinson, P.G. Ince, M.O. Smith, R. Highley, G. Skibinski, P.M. Andersen, K.E. Morrison, H.S.

Pall, O. Hardiman, J. Collinge, P.J. Shaw, E.M.C. Fisher, ALS phenotypes with mutations in CHMP2B (charged multivesicular body protein 2B), Neurology. 67 (2006) 1074–1077. doi:10.1212/01.wnl.0000231510.89311.8b.

[17] L.E. Cox, L. Ferraiuolo, E.F. Goodall, P.R. Heath, A. Higginbottom, H. Mortiboys, H.C. Hollinger, J.A. Hartley, A. Brockington, C.E. Burness, K.E. Morrison, S.B. Wharton, A.J. Grierson, P.G. Ince, J. Kirby, P.J. Shaw, Mutations in CHMP2B in Lower Motor Neuron Predominant Amyotrophic Lateral Sclerosis (ALS), PLoS One. 5 (2010) e9872. doi:10.1371/journal.pone.0009872.

[18] I. Puls, C. Jonnakuty, B.H. LaMonte, E.L.F. Holzbaur, M. Tokito, E. Mann, M.K. Floeter, K. Bidus, D. Drayna, S.J. Oh, R.H. Brown, C.L. Ludlow, K.H. Fischbeck, Mutant dynactin in motor neuron disease, Nat. Genet. 33 (2003) 455–456. doi:10.1038/ng1123.

[19] C.-H. Wu, C. Fallini, N. Ticozzi, P.J. Keagle, P.C. Sapp, K. Piotrowska, P. Lowe, M. Koppers, D. McKenna-Yasek, D.M. Baron, J.E. Kost, P. Gonzalez-Perez, A.D. Fox, J. Adams, F. Taroni, C. Tiloca, A.L. Leclerc, S.C. Chafe, D. Mangroo, M.J. Moore, J.A. Zitzewitz, Z.-S. Xu, L.H. van den Berg, J.D. Glass, G. Siciliano, E.T. Cirulli, D.B. Goldstein, F. Salachas, V. Meininger, W. Rossoll, A. Ratti, C. Gellera, D.A. Bosco, G.J. Bassell, V. Silani, V.E. Drory, R.H. Brown Jr, J.E. Landers, Mutations in the profilin 1 gene cause familial amyotrophic lateral sclerosis, Nature. 488 (2012) 499–503. doi:10.1038/nature11280.

[20] C.Y. Chow, J.E. Landers, S.K. Bergren, P.C. Sapp, A.E. Grant, J.M. Jones, L. Everett, G.M. Lenk, D.M. McKenna-Yasek, L.S. Weisman, D. Figlewicz, R.H. Brown, M.H. Meisler, Deleterious Variants of FIG4, a Phosphoinositide Phosphatase, in Patients with ALS, Am. J. Hum. Genet. 84 (2009) 85–88. doi:10.1016/j.ajhg.2008.12.010.

[21] M. DeJesus-Hernandez, I.R. Mackenzie, B.F. Boeve, A.L. Boxer, M. Baker, N.J. Rutherford, A.M. Nicholson, N.C.A. Finch, H. Flynn, J. Adamson, N. Kouri, A. Wojtas, P. Sengdy, G.Y.R. Hsiung, A. Karydas, W.W. Seeley, K.A. Josephs, G. Coppola, D.H. Geschwind, Z.K. Wszolek, H. Feldman, D.S. Knopman, R.C. Petersen, B.L. Miller, D.W. Dickson, K.B. Boylan, N.R. Graff-Radford, R. Rademakers, Expanded GGGGCC Hexanucleotide Repeat in Noncoding Region of C9ORF72 Causes Chromosome 9p-Linked FTD and ALS, Neuron. 72 (2011) 245–256. doi:10.1016/j.neuron.2011.09.011.

[22] A.E. Renton, E. Majounie, A. Waite, J. Simón-Sánchez, S. Rollinson, J.R. Gibbs, J.C. Schymick, H. Laaksovirta, J.C. van Swieten, L. Myllykangas, H. Kalimo, A. Paetau, Y. Abramzon, A.M. Remes, A. Kaganovich, S.W. Scholz, J. Duckworth, J. Ding, D.W. Harmer, D.G. Hernandez, J.O. Johnson, K. Mok, M. Ryten, D. Trabzuni, R.J. Guerreiro, R.W. Orrell, J. Neal, A. Murray, J. Pearson, I.E. Jansen, D. Sondervan, H. Seelaar, D. Blake, K. Young, N. Halliwell, J.B. Callister, G. Toulson, A. Richardson, A. Gerhard, J. Snowden, D. Mann, D. Neary, M.A. Nalls, T. Peuralinna, L. Jansson, V.M. Isoviita, A.L. Kaivorinne, M. Hölttä-Vuori, E. Ilonen, R. Sulkava, M. Benatar, J. Wuu, A. Chiò, G. Restagno, G. Borghero, M. Sabatelli, D. Heckerman, E. Rogaeva, L. Zinman, J.D. Rothstein, M. Sendtner, C. Drepper, E.E. Eichler, C. Alkan, Z. Abdullaev, S.D. Pack, A. Dutra, E. Pak, J. Hardy, A. Singleton, N.M. Williams, P. Heutink, S. Pickering-Brown, H.R. Morris, P.J. Tienari, B.J. Traynor, A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p 21-linked ALS-FTD, Neuron. 72 (2011) 257–268. doi:10.1016/j.neuron.2011.09.010.

[23] T. van Langenhove, J. van der Zee, C. van Broeckhoven, The molecular basis of the frontotemporal lobar degeneration-amyotrophic lateral sclerosis spectrum, Ann. Med. 44

(2012) 817-828.doi:10.3109/07853890.2012.665471.

[24] Z. Arvanitakis, Update on frontotemporal dementia., Neurologist. 16 (2010) 16–22. doi:10.1097/NRL.0b013e3181b1d5c6.

[25] O.M. Peters, M. Ghasemi, R.H. Brown, Emerging mechanisms of molecular pathology in ALS, J. Clin. Invest. 125 (2015) 1767–1779. doi:10.1172/JCI71601.

[26] D.A. Bosco, G. Morfini, N.M. Karabacak, Y. Song, F. Gros-Louis, P. Pasinelli, H. Goolsby, B.A. Fontaine, N. Lemay, D. McKenna-Yasek, M.P. Frosch, J.N. Agar, J.-P. Julien, S.T. Brady, R.H. Brown, Wild-type and mutant SOD1 share an aberrant conformation and a common pathogenic pathway in ALS, Nat. Neurosci. 13 (2010) 1396–1403. doi:10.1038/nn.2660.

[27] S. Chen, P. Sayana, X. Zhang, W. Le, Genetics of amyotrophic lateral sclerosis: an update, Mol. Neurodegener. 8 (2013) 28. doi:10.1186/1750-1326-8-28.

[28] S.J. Kaur, S.R. McKeown, S. Rashid, Mutant SOD1 mediated pathogenesis of Amyotrophic Lateral Sclerosis, Gene. 577 (2016) 109–118. doi:10.1016/j.gene.2015.11.049.

[29] P. Pasinelli, R.H. Brown, Molecular biology of amyotrophic lateral sclerosis: insights from genetics, Nat. Rev. Neurosci. 7 (2006) 710–723. doi:10.1038/nrn1971.

[30] T. Kabuta, Y. Suzuki, K. Wada, Degradation of amyotrophic lateral sclerosis-linked mutant Cu, Zn-superoxide dismutase proteins by macroautophagy and the proteasome, J. Biol. Chem. 281 (2006) 30524–30533. doi:10.1074/jbc.M603337200.

[31] E. Shvets, E. Fass, R. Scherz-Shouval, Z. Elazar, The N-terminus and Phe52 residue of LC3 recruit p62/SQSTM1 into autophagosomes, J. Cell Sci. 121 (2008) 2685–2695. doi:10.1242/jcs.026005.

[32] J. Gal, A.-L. Ström, D.M. Kwinter, R. Kilty, J. Zhang, P. Shi, W. Fu, M.W. Wooten, H. Zhu, Sequestosome 1/p62 links familial ALS mutant SOD1 to LC3 via an ubiquitin-independent mechanism, J. Neurochem. 111 (2009) 1062–1073. doi:10.1111/j.1471-4159.2009.06388.x.

[33] F. Tian, N. Morimoto, W. Liu, Y. Ohta, K. Deguchi, K. Miyazaki, K. Abe, In vivo optical imaging of motor neuron autophagy in a mouse model of amyotrophic lateral sclerosis, Autophagy. 7 (2014) 985–992. doi:10.4161/auto.7.9.16012.

[34] C. Hetz, P. Thielen, S. Matus, M. Nassif, F. Court, R. Kiffin, G. Martinez, A.M. Cuervo, R.H. Brown, L.H. Glimcher, XBP-1 deficiency in the nervous system protects against amyotrophic lateral sclerosis by increasing autophagy, Genes Dev. 23 (2009) 2294–2306. doi:10.1101/gad.1830709.

[35] J. Kim, M. Kundu, B. Viollet, K.-L. Guan, AMPK and mTOR regulate autophagy through direct phosphorylation of Ulk1, Nat. Cell Biol. 13 (2011) 132–141. doi:10.1038/ncb2152.

[36] U. Bandyopadhyay, M. Nagy, W.A. Fenton, A.L. Horwich, Absence of lipofuscin in motor neurons of SOD1-linked ALS mice, Proc. Natl. Acad. Sci. 111 (2014) 11055–11060. doi:10.1073/pnas.1409314111.

[37] Y. Wei, Autophagic induction of amyotrophic lateral sclerosis linked Cu/Zn superoxide dismutase 1G93A mutant in NSC34 cells, Neural Regen. Res. 9 (2014) 16–24. doi:10.4103/1673-5374.125325.

[38] N. Morimoto, M. Nagai, Y. Ohta, K. Miyazaki, T. Kurata, M. Morimoto, T. Murakami, Y. Takehisa, Y. Ikeda, T. Kamiya, K. Abe, Increased autophagy in transgenic mice with a G93A mutant SOD1 gene, Brain Res. 1167 (2007) 112–117. doi:10.1016/j.brainres.2007.06.045.

[39] A. Li, X. Zhang, W. Le, Altered macroautophagy in the spinal cord of SOD1 mutant mice, Autophagy. 4 (2008) 290–293. doi:10.4161/auto.5524.

[40] T. An, P. Shi, W. Duan, S. Zhang, P. Yuan, Z. Li, D. Wu, Z. Xu, C. Li, Y. Guo, Oxidative stress and autophagic alteration in brainstem of SOD1-G93A mouse model of ALS, Mol. Neurobiol. 49 (2014) 1435–1448. doi:10.1007/s12035-013-8623-3.

[41] C. Settembre, A. Ballabio, TFEB regulates autophagy: An integrated coordination of cellular degradation and recycling processes, Autophagy. 7 (2011) 1379–1381. doi:10.4161/auto.7.11.17166.

[42] R. Kang, H.J. Zeh, M.T. Lotze, D. Tang, The Beclin 1 network regulates autophagy and apoptosis, Cell Death Differ. 18 (2011) 571–580. doi:10.1038/cdd.2010.191.

[43] Y. Chen, H. Liu, Y. Guan, Q. Wang, F. Zhou, L. Jie, J. Ju, L. Pu, H. Du, X. Wang, The altered autophagy mediated by TFEB in animal and cell models of amyotrophic lateral sclerosis, Am. J. Transl. Res. 7 (2015) 1574–1587.

[44] K. Katsumata, J. Nishiyama, T. Inoue, N. Mizushima, J. Takeda, M. Yuzaki, Dynein-and activity-dependent retrograde transport of autophagosomes in neuronal axons, Autophagy. 6 (2010) 378–385. doi:10.4161/auto.6.3.11262.

[45] J.A. Johnston, M.E. Illing, R.R. Kopito, Cytoplasmic dynein/dynactin mediates the assembly of aggresomes, Cell Motil. Cytoskeleton. 53 (2002) 26–38. doi:10.1002/cm.10057.

[46] A.-L. Ström, J. Gal, P. Shi, E.J. Kasarskis, L.J. Hayward, H. Zhu, Retrograde axonal transport and motor neuron disease., J. Neurochem. 106 (2008) 495–505. doi:10.1111/j.1471-4159.2008.05393.x.

[47] F. Zhang, A.L. Strom, K. Fukada, S. Lee, L.J. Hayward, H. Zhu, Interaction between familial Amyotrophic Lateral Sclerosis (ALS)-linked SOD1 mutants and the dynein complex, J. Biol. Chem. 282 (2007) 16691–16699. doi:10.1074/jbc.M609743200.

[48] L.G. Bilsland, E. Sahai, G. Kelly, M. Golding, L. Greensmith, G. Schiavo, Deficits in axonal transport precede ALS symptoms in vivo, Proc. Natl. Acad. Sci. 107 (2010) 20523–20528. doi:10.1073/pnas.1006869107.

[49] Y. Xie, B. Zhou, M.Y. Lin, S. Wang, K.D. Foust, Z.H. Sheng, Endolysosomal Deficits Augment Mitochondria Pathology in Spinal Motor Neurons of Asymptomatic fALS Mice, Neuron. 87 (2015) 355–371. doi:10.1016/j.neuron.2015.06.026.

[50] T. Hara, K. Nakamura, M. Matsui, A. Yamamoto, Y. Nakahara, R. Suzuki-Migishima, M. Yokoyama, K. Mishima, I. Saito, H. Okano, N. Mizushima, Suppression of basal autophagy in neural cells causes neurodegenerative disease in mice., Nature. 441 (2006) 885–9. doi:10.1038/nature04724.

[51] H.L. Feng, Y. Leng, C.H. Ma, J. Zhang, M. Ren, D.M. Chuang, Combined lithium and valproate treatment delays disease onset, reduces neurological deficits and prolongs survival in an amyotrophic lateral sclerosis mouse model, Neuroscience. 155 (2008) 567–572.

doi:10.1016/j.neuroscience.2008.06.040.

[52] F. Fornai, P. Longone, L. Cafaro, O. Kastsiuchenka, M. Ferrucci, M.L. Manca, G. Lazzeri, A. Spalloni, N. Bellio, P. Lenzi, N. Modugno, G. Siciliano, C. Isidoro, L. Murri, S. Ruggieri, A. Paparelli, Lithium delays progression of amyotrophic lateral sclerosis., Proc. Natl. Acad. Sci. U.S.A. 105 (2008) 2052–7. doi:10.1073/pnas.0708022105.

[53] C. Pizzasegola, I. Caron, C. Daleno, A. Ronchi, C. Minoia, M.T. Carrì, C. Bendotti, Treatment with lithium carbonate does not improve disease progression in two different strains of SOD1 mutant mice, Amyotroph. Lateral Scler. 10 (2009) 221–228. doi:10.1080/17482960902803440.

[54] S. Watanabe, O. Komine, F. Endo, K. Wakasugi, K. Yamanaka, Intracerebroventricular administration of Cystatin C ameliorates disease in SOD1-linked amyotrophic lateral sclerosis mice., J. Neurochem. (2017). doi:10.1111/jnc.14285.

[55] K. Castillo, M. Nassif, V. Venezuela, F. Rojas, S. Matus, G. Mercado, F.A. Court, B. van Zundert, C. Hetz, Trehalose delays the progression of amyotrophic lateral sclerosis by enhancing autophagy in motoneurons., Autophagy. 9 (2013) 1308–20. doi:10.4161/auto.25188.

[56] X. Zhang, S. Chen, L. Song, Y. Tang, Y. Shen, L. Jia, W. Le, MTOR-independent, autophagic enhancer trehalose prolongs motor neuron survival and ameliorates the autophagic flux defect in a mouse model of amyotrophic lateral sclerosis, Autophagy. 10 (2014) 588–602. doi:10.4161/auto.27710.

[57] J.K. Jeong, M.H. Moon, Y.J. Lee, J.W. Seol, S.Y. Park, Autophagy induced by the class III histone deacetylase Sirt1 prevents prion peptide neurotoxicity, Neurobiol. Aging. 34 (2013) 146–156. doi:10.1016/j.neurobiolaging.2012.04.002.

[58] R. Mancuso, J. del Valle, L. Modol, A. Martinez, A.B. Granado-Serrano, O. Ramirez-Núñez, M. Pallás, M. Portero-Otín, R. Osta, X. Navarro, Resveratrol Improves Motoneuron Function and Extends Survival in SOD1G93A ALS Mice, Neurotherapeutics. 11 (2014) 419–432. doi:10.1007/s13311-013-0253-y.

[59] X. Zhang, L. Li, S. Chen, D. Yang, Y. Wang, X. Zhang, Z. Wang, W. Le, Rapamycin treatment augments motor neuron degeneration in SOD1G93A mouse model of amyotrophic lateral sclerosis, Autophagy. 7 (2011) 412–425. doi:10.4161/auto.7.4.14541.

[60] K.-W. Hsueh, T.-W. Chiou, S.-F. Chiang, T. Yamashita, K. Abe, C.V. Borlongan, P.R. Sanberg, A. (Yu H. Huang, S.-Z. Lin, H.-J. Harn, Autophagic down-regulation in motor neurons remarkably prolongs the survival of ALS mice, Neuropharmacology. 108 (2016) 152–160. doi:10.1016/j.neuropharm.2016.03.035.

[61] Q.M. Zhou, J.J. Zhang, S. Li, S. Chen, W.D. Le, n-butylidenephthalide treatment prolongs lifespan and attenuates motor neuron loss in SOD1G93A mouse model of amyotrophic lateral sclerosis, CNS Neurosci. Ther. 23 (2017) 375–385. doi:10.1111/cns.12681.

[62] N.D. Rudnick, C.J. Griffey, P. Guarnieri, V. Gerbino, X. Wang, J.A. Piersaint, J.C. Tapia, M.M. Rich, T. Maniatis, Distinct roles for motor neuron autophagy early and late in the SOD1G93A mouse model of ALS, Proc. Natl. Acad. Sci. 114 (2017) E8294–E8303. doi:10.1073/pnas.1704294114.

[63] K. Zhang, P. Shi, T. An, Q. Wang, J. Wang, Z. Li, W. Duan, C. Li, Y. Guo, Food restriction-induced autophagy modulates degradation of mutant SOD1 in an amyotrophic lateral sclerosis mouse model, Brain Res. 1519 (2013) 112–119. doi:10.1016/j.brainres.2013.04.050.

[64] S.C. Ling, M. Polymenidou, D.W. Cleveland, Converging mechanisms in ALS and FTD: Disrupted RNA and protein homeostasis, Neuron. 79 (2013) 416–438. doi:10.1016/j.neuron.2013.07.033.

[65] C. Lagier-Tourenne, D.W. Cleveland, Rethinking ALS: The FUS about TDP-43, Cell. 136 (2009) 1001–1004. doi:10.1016/j.cell.2009.03.006.

[66] D. Ito, N. Suzuki, Conjoint pathologic cascades mediated by ALS/FTLD-U linked RNA-binding proteins TDP-43 and FUS, Neurology. 77 (2011) 1636–1643. doi:10.1212/WNL.0b013e3182343365.

[67] Z.-Y. Zou, Z.-R. Zhou, C.-H. Che, C.-Y. Liu, R.-L. He, H.-P. Huang, Genetic epidemiology of amyotrophic lateral sclerosis: a systematic review and meta-analysis, J. Neurol. Neurosurg. Psychiatry. 88 (2017) 540–549. doi:10.1136/jnnp-2016-315018.

[68] B. Borroni, S. Archetti, R. Del Bo, A. Papetti, E. Buratti, C. Bonvicini, C. Agosti, M. Cosseddu, M. Turla, D. Di Lorenzo, G. Pietro Comi, M. Gennarelli, A. Padovani, TARDBP Mutations in Frontotemporal Lobar Degeneration: Frequency, Clinical Features, and Disease Course, Rejuvenation Res. 13 (2010) 509–517. doi:10.1089/rej.2010.1017.

[69] B.S. Johnson, D. Snead, J.J. Lee, J.M. McCaffery, J. Shorter, A.D. Gitler, TDP-43 is intrinsically aggregation-prone, and amyotrophic lateral sclerosis-linked mutations accelerate aggregation and increase toxicity, J. Biol. Chem. 284 (2009) 20329–20339. doi:10.1074/jbc.M109.010264.

[70] Z. Sun, Z. Diaz, X. Fang, M.P. Hart, A. Chesi, J. Shorter, A.D. Gitler, Molecular determinants and genetic modifiers of aggregation and toxicity for the ALS disease protein FUS/TLS, PLoS Biol. 9 (2011). doi:10.1371/journal.pbio.1000614.

[71] F.C. Fiesel, P.J. Kahle, TDP-43 and FUS/TLS: Cellular functions and implications for neurodegeneration, FEBS J. 278 (2011) 3550–3568. doi:10.1111/j.1742-4658.2011.08258.x.

[72] C. Lagier-Tourenne, M. Polymenidou, D.W. Cleveland, TDP-43 and FUS/TLS: emerging roles in RNA processing and neurodegeneration, Hum. Mol. Genet. 19 (2010) R46–R64. doi:10.1093/hmg/ddq137.

[73] C. Lagier-Tourenne, M. Polymenidou, K.R. Hutt, A.Q. Vu, M. Baughn, S.C. Huelga, K.M. Clutario, S.-C. Ling, T.Y. Liang, C. Mazur, E. Wancewicz, A.S. Kim, A. Watt, S. Freier, G.G. Hicks, J.P. Donohue, L. Shiuve, C.F. Bennett, J. Ravits, D.W. Cleveland, G.W. Yeo, Divergent roles of ALS-linked proteins FUS/TLS and TDP-43 intersect in processing long pre-mRNAs, Nat. Neurosci. 15 (2012) 1488–1497. doi:10.1038/nn.3230.

[74] T. Arai, M. Hasegawa, H. Akiyama, K. Ikeda, T. Nonaka, H. Mori, D. Mann, K. Tsuchiya, M. Yoshida, Y. Hashizume, T. Oda, TDP-43 is a component of ubiquitin-positive tau-negative inclusions in frontotemporal lobar degeneration and amyotrophic lateral sclerosis, Biochem. Biophys. Res. Commun. 351 (2006) 602–611. doi:10.1016/j.bbrc.2006.10.093.

[75] Z.M. March, O.D. King, J. Shorter, Prion-like domains as epigenetic regulators, scaffolds for

subcellular organization, and drivers of neurodegenerative disease, Brain Res. 1647 (2016) 9-18.doi:10.1016/j.brainres.2016.02.037.

[76] M. Udan, R.H. Baloh, Implications of the prion-related Q/N domains in TDP-43 and FUS, Prion. 5 (2011) 1-5.doi:10.4161/pri.5.1.14265.

[77] A. Patel, H.O. Lee, L. Jawerth, S. Maharana, M. Jahnel, M.Y. Hein, S. Stoynov, J. Mahamid, S. Saha, T.M. Franzmann, A. Pozniakovski, I. Poser, N. Maghelli, L.A. Royer, M. Weigert, E.W. Myers, S. Grill, D. Drechsel, A.A. Hyman, S. Alberti, A Liquid-to-Solid Phase Transition of the ALS Protein FUS Accelerated by Disease Mutation, Cell. 162 (2015) 1066–1077. doi:10.1016/j.cell.2015.07.047.

[78] A. Molliex, J. Temirov, J. Lee, M. Coughlin, A.P. Kanagaraj, H.J. Kim, T. Mittag, J.P. Taylor, Phase Separation by Low Complexity Domains Promotes Stress Granule Assembly and Drives Pathological Fibrillization, Cell. 163 (2015) 123–133.doi:10.1016/j.cell.2015.09.015.

[79] Z. Monahan, F. Shewmaker, U.B. Pandey, Stress granules at the intersection of autophagy and ALS, Brain Res. 1649 (2016) 189–200.doi:10.1016/j.brainres.2016.05.022.

[80] M. Lo Bello, F. Di Fini, A. Notaro, R. Spataro, F.L. Conforti, V. La Bella, ALS-Related Mutant FUS Protein Is Mislocalized to Cytoplasm and Is Recruited into Stress Granules of Fibroblasts from Asymptomatic FUSP525L Mutation Carriers., Neurodegener. Dis. 17 (2017) 292–303. doi:10.1159/000480085.

[81] L. Liu-Yesucevitz, A. Bilgutay, Y.J. Zhang, T. Vanderwyde, A. Citro, T. Mehta, N. Zaarur, A. McKee, R. Bowser, M. Sherman, L. Petrucelli, B. Wolozin, Tar DNA binding protein-43 (TDP-43) associates with stress granules: Analysis of cultured cells and pathological brain tissue, PLoS One. 5 (2010).doi:10.1371/journal.pone.0013250.

[82] X. Wang, H. Fan, Z. Ying, B. Li, H. Wang, G. Wang, Degradation of TDP-43 and its pathogenic form by autophagy and the ubiquitin-proteasome system, Neurosci. Lett. 469 (2010) 112–116.doi:10.1016/j.neulet.2009.11.055.

[83] H.H. Ryu, M.H. Jun, K.J. Min, D.J. Jang, Y.S. Lee, H.K. Kim, J.A. Lee, Autophagy regulates amyotrophic lateral sclerosis-linked fused in sarcoma-positive stress granules in neurons, Neurobiol. Aging. 35 (2014) 2822–2831.doi:10.1016/j.neurobiolaging.2014.07.026.

[84] O.A. Brady, P. Meng, Y. Zheng, Y. Mao, F. Hu, Regulation of TDP-43 aggregation by phosphorylation and p62/SQSTM1, J. Neurochem. 116 (2011) 248–259.doi:10.1111/j.1471-4159.2010.07098.x.

[85] I.-F. Wang, B.-S. Guo, Y.-C. Liu, C.-C. Wu, C.-H. Yang, K.-J. Tsai, C.-K.J. Shen, Autophagy activators rescue and alleviate pathogenesis of a mouse model with proteinopathies of the TAR DNA-binding protein 43, Proc. Natl. Acad. Sci. 109 (2012) 15024–15029. doi:10.1073/pnas.1206362109.

[86] C.-W. Cheng, M.-J. Lin, C.-K.J. Shen, Rapamycin alleviates pathogenesis of a new Drosophila model of ALS-TDP, J. Neurogenet. 29 (2015) 59–68.doi:10.3109/01677063.2015.1077832.

[87] S.J. Barmada, A. Serio, A. Arjun, B. Bilican, A. Daub, D.M. Ando, A. Tsvetkov, M. Pleiss, X. Li, D. Peisach, C. Shaw, S. Chandran, S. Finkbeiner, Autophagy induction enhances TDP 43 turnover and survival in neuronal ALS models, Nat. Chem. Biol. 10 (2014) 677–685.

doi:10.1038/nchembio.1563.

[88] R. Stoica, K.J. De Vos, S. Paillusson, S. Mueller, R.M. Sancho, K.-F. Lau, G. Vizcay-Barrena, W.-L. Lin, Y.-F. Xu, J. Lewis, D.W. Dickson, L. Petrucelli, J.C. Mitchell, C.E. Shaw, C.C.J. Miller, ER-mitochondria associations are regulated by the VAPB-PTPIP51 interaction and are disrupted by ALS/FTD-associated TDP-43, Nat. Commun. 5 (2014) 3996. doi:10.1038/ncomms4996.

[89] P. Gomez-Suaga, S. Paillusson, R. Stoica, W. Noble, D.P. Hanger, C.C.J. Miller, The ER-Mitochondria Tethering Complex VAPB-PTPIP51 Regulates Autophagy, Curr. Biol. 27 (2017) 371–385. doi:10.1016/j.cub.2016.12.038.

[90] K.Y. Soo, M. Halloran, V. Sundaramoorthy, S. Parakh, R.P. Toth, K.A. Southam, C.A. McLean, P. Lock, A. King, M.A. Farg, J.D. Atkin, Rab1-dependent ER-Golgi transport dysfunction is a common pathogenic mechanism in SOD1, TDP-43 and FUS-associated ALS, Acta Neuropathol. 130 (2015) 679–697. doi:10.1007/s00401-015-1468-2.

[91] F.C.M. Zoppino, R.D. Militello, I. Slavin, C. Alvarez, M.I. Colombo, Autophagosome formation depends on the small GTPase rab1 and functional ER exit sites, Traffic. 11 (2010) 1246–1261. doi:10.1111/j.1600-0854.2010.01086.x.

[92] Z. Szatmári, M. Sass, The autophagic roles of Rab small GTPases and their upstream regulators: A review, Autophagy. 10 (2014) 1154–1166. doi:10.4161/auto.29395.

[93] K.Y. Soo, J. Sultana, A.E. King, R. Atkinson, S.T. Warraich, V. Sundaramoorthy, I. Blair, M.A. Farg, J.D. Atkin, ALS-associated mutant FUS inhibits macroautophagy which is restored by overexpression of Rab1., Cell Death Discov. 1 (2015) 15030. doi:10.1038/cddiscovery.2015.30.

[94] J.P. Ling, O. Pletnikova, J.C. Troncoso, P.C. Wong, TDP-43 repression of nonconserved cryptic exons is compromised in ALS-FTD, Science (80-) . 349 (2015) 650–655. doi:10.1126/science.aab0983.

[95] E.S. Arnold, S.-C. Ling, S.C. Huelga, C. Lagier-Tourenne, M. Polymenidou, D. Ditsworth, H.B. Kordasiewicz, M. McAlonis-Downes, O. Platoshyin, P.A. Parone, S. Da Cruz, K.M. Clutario, D. Swing, L. Tessarollo, M. Marsala, C.E. Shaw, G.W. Yeo, D.W. Cleveland, ALS-linked TDP-43 mutations produce aberrant RNA splicing and adult-onset motor neuron disease without aggregation or loss of nuclear TDP-43, Proc. Natl. Acad. Sci. 110 (2013) E736–E745. doi:10.1073/pnas.1222809110.

[96] A.S. Mastrocola, S.H. Kim, A.T. Trinh, L.A. Rodenkirch, R.S. Tibbetts, The RNA-binding protein fused in sarcoma (FUS) functions downstream of poly(ADP-ribose) polymerase (PARP) in response to DNA damage, J. Biol. Chem. 288 (2013) 24731–24741. doi:10.1074/jbc.M113.497974.

[97] T. Zhang, G. Baldie, G. Periz, J. Wang, RNA-Processing Protein TDP-43 Regulates FOXO-Dependent Protein Quality Control in Stress Response, PLoS Genet. 10 (2014) e1004693. doi:10.1371/journal.pgen.1004693.

[98] J.K. Bose, C.C. Huang, C.K.J. Shen, Regulation of autophagy by neuropathological protein TDP-43, J. Biol. Chem. 286 (2011) 44441–44448. doi:10.1074/jbc.M111.237115.

[99] A.-F. Imhaus, G. Duménil, The number of Neisseria meningitidis type IV pili determines host

cell interaction, EMBO J. 33 (2014) 1767–1783. doi:10.15252/embj.

[100] V. Majcher, A. Goode, V. James, R. Layfield, Autophagy receptor defects and ALS-FTLD, Mol. Cell. Neurosci. 66 (2015) 43–52. doi:10.1016/j.mcn.2015.01.002.

[101] F. Fecto, J. Yan, S.P. Vemula, E. Liu, Y. Yang, W. Chen, J.G. Zheng, Y. Shi, N. Siddique, H. Arrat, S. Donkervoort, S. Ajroud-Driss, R.L. Sufit, S.L. Heller, H.-X. Deng, T. Siddique, SQSTM1 mutations in familial and sporadic amyotrophic lateral sclerosis., Arch. Neurol. 68 (2011) 1440–6. doi:10.1001/archneurol.2011.250.

[102] S.L. Rea, V. Majcher, M.S. Searle, R. Layfield, SQSTM1 mutations – Bridging Paget disease of bone and ALS/FTLD, Exp. Cell Res. 325 (2014) 27–37. doi:10.1016/j.yexcr.2014.01.020.

[103] H. Maruyama, H. Kawakami, Optineurin and amyotrophic lateral sclerosis., Geriatr. Gerontol. Int. 13 (2013) 528–32. doi:10.1111/ggi.12022.

[104] S. Pankiv, T.H. Clausen, T. Lamark, A. Brech, J.A. Bruun, H. Outzen, A. Øvervatn, G. Bjørkøy, T. Johansen, p62/SQSTM1 binds directly to Atg8/LC3 to facilitate degradation of ubiquitinated protein aggregates by autophagy*[S], J. Biol. Chem. (2007). doi:10.1074/jbc.M702824200.

[105] K. Slowikka, L. Vereecke, G. van Loo, Cellular Functions of Optineurin in Health and Disease, Trends Immunol. 37 (2016) 621–633. doi:10.1016/j.it.2016.07.002.

[106] J. Gal, A.-L. Ström, R. Kilty, F. Zhang, H. Zhu, p62 Accumulates and Enhances Aggregate Formation in Model Systems of Familial Amyotrophic Lateral Sclerosis, J. Biol. Chem. 282 (2007) 11068–11077. doi:10.1074/jbc.M608787200.

[107] A.E. Renton, A. Chiò, B.J. Traynor, State of play in amyotrophic lateral sclerosis genetics, Nat. Neurosci. 17 (2014) 17–23. doi:10.1038/nn.3584.

[108] I. Le Ber, A. Camuzat, R. Guerreiro, K. Bouya-Ahmed, J. Bras, G. Nicolas, A. Gabelle, M. Didic, A. De Septenville, S. Millecamps, T. Lenglet, M. Latouche, E. Kabashi, D. Campion, D. Hannequin, J. Hardy, A. Brice, French Clinical and Genetic Research Network on FTD/FTD-ALS, SQSTM1 mutations in French patients with frontotemporal dementia or frontotemporal dementia with amyotrophic lateral sclerosis., JAMA Neurol. 70 (2013) 1403–10. doi:10.1001/jamaneurol.2013.3849.

[109] M.L. Seibenhener, J.R. Babu, T. Geetha, H.C. Wong, N.R. Krishna, M.W. Wooten, Sequestosome 1/p62 Is a Polyubiquitin Chain Binding Protein Involved in Ubiquitin Proteasome Degradation, Mol. Cell. Biol. (2004). doi:10.1128/MCB.24.18.8055-8068.2004.

[110] J.R. Babu, T. Geetha, M.W. Wooten, Sequestosome 1/p62 shuttles polyubiquitinated tau for proteasomal degradation, J. Neurochem. (2005). doi:10.1111/j.1471-4159.2005.03181.x.

[111] B. Ciani, R. Layfield, J.R. Cavey, P.W. Sheppard, M.S. Searle, Structure of the ubiquitin-associated domain of p62 (SQSTM1) and implications for mutations that cause Paget's disease of bone, J. Biol. Chem. 278 (2003) 37409–37412. doi:10.1074/jbc.M307416200.

[112] K. Zatloukal, C. Stumptner, A. Fuchs bichler, H. Heid, M. Schnoelzer, L. Kenner, R. Kleinert, M. Prinz, A. Aguzzi, H. Denk, p62 Is a Common Component of Cytoplasmic Inclusions in Protein Aggregation Diseases, Am. J. Pathol. 160 (2002) 255–263. doi:10.1016/S0002-9440(10)64369-6.

[113] A. King, S. Al-Sarraj, C. Troakes, B.N. Smith, S. Maekawa, M. Iovino, M.G. Spillantini, C.E. Shaw, Mixed tau, TDP-43 and p62 pathology in FTLD associated with a C9ORF72 repeat expansion and p.Ala239Thr MAPT (tau) variant, Acta Neuropathol. 125 (2013) 303–310. doi:10.1007/s00401-012-1050-0.

[114] C. Troakes, S. Maekawa, L. Wijesekera, B. Rogelj, L. Siklós, C. Bell, B. Smith, S. Newhouse, C. Vance, L. Johnson, T. Hortobágyi, A. Shatunov, A. Al-Chalabi, N. Leigh, C.E. Shaw, A. King, S. Al-Sarraj, An MND/ALS phenotype associated with C9orf72 repeat expansion: Abundant p62-positive, TDP-43-negative inclusions in cerebral cortex, hippocampus and cerebellum but without associated cognitive decline, Neuropathology. 32 (2012) 505–514. doi:10.1111/j.1440-1789.2011.01286.x.

[115] S.A. Andrew, K. Claire, T. Bradley, S. Satomi, C.E. Shaw, p62 positive, TDP-43 negative, neuronal cytoplasmic and intranuclear inclusions in the cerebellum and hippocampus define the pathology of C9orf72-linked FTLD and MND/ALS, Acta Neuropathol. (2011) 691–702. doi:10.1007/s00401-011-0911-2.

[116] A. Goode, K. Butler, J. Long, J. Cavey, D. Scott, B. Shaw, J. Sollenberger, C. Gell, T. Johansen, N.J. Oldham, M.S. Searle, R. Layfield, Defective recognition of LC3B by mutant SQSTM1/p62 implicates impairment of autophagy as a pathogenic mechanism in ALS-FTLD, Autophagy. 12 (2016) 1094–1104. doi:10.1080/15548627.2016.1170257.

[117] E. Rubino, I. Rainero, A. Chio, E. Rogaeva, D. Galimberti, P. Fenoglio, Y. Grinberg, G. Isaia, A. Calvo, S. Gentile, A.C. Bruni, P.H. St. George-Hyslop, E. Scarpini, S. Gallone, L. Pinessi, SQSTM1 mutations in frontotemporal lobar degeneration and amyotrophic lateral sclerosis, Neurology. 79 (2012) 1556–1562. doi:10.1212/WNL.0b013e31826e25df.

[118] Y. Du, M.C. Wooten, M.W. Wooten, Oxidative damage to the promoter region of SQSTM1/p62 is common to neurodegenerative disease, Neurobiol. Dis. 35 (2009) 302–310. doi:10.1016/j.nbd.2009.05.015.

[119] S. Hadano, S. Mitsui, L. Pan, A. Otomo, M. Kubo, K. Sato, S. Ono, W. Onodera, K. Abe, X.P. Chen, M. Koike, Y. Uchiyama, M. Aoki, E. Warabi, M. Yamamoto, T. Ishii, T. Yanagawa, H.F. Shang, F. Yoshii, Functional links between SQSTM1 and ALS2 in the pathogenesis of ALS: Cumulative impact on the protection against mutant SOD1-mediated motor dysfunction in mice, Hum. Mol. Genet. 25 (2015) 3321–3340. doi:10.1093/hmg/ddw180.

[120] D. Kachaner, P. Génin, E. Laplantine, R. Weil, Toward an integrative view of Optineurin functions, Cell Cycle. 11 (2012) 2808–2818. doi:10.4161/cc.20946.

[121] J.H. Weishaupt, S. Waibel, A. Birve, A.E. Volk, B. Mayer, T. Meyer, A.C. Ludolph, P.M. Andersen, A novel optineurin truncating mutation and three glaucoma-associated missense variants in patients with familial amyotrophic lateral sclerosis in Germany, Neurobiol. Aging. 34 (2013). doi:10.1016/j.neurobiolaging.2012.09.007.

[122] J. Korac, V. Schaeffer, I. Kovacevic, A.M. Clement, B. Jungblut, C. Behl, J. Terzic, I. Dikic, Ubiquitin-independent function of optineurin in autophagic clearance of protein aggregates, J. Cell Sci. 126 (2013) 580–592. doi:10.1242/jcs.114926.

[123] A. Khaminets, C. Behl, I. Dikic, Ubiquitin-Dependent And Independent Signals In Selective Autophagy, Trends Cell Biol. 26 (2016) 6–16. doi:10.1016/j.tcb.2015.08.010.

[124] W.C. Shen, H.Y. Li, G.C. Chen, Y. Chern, P.H. Tu, Mutations in the ubiquitin-binding domain of OPTN/optineurin interfere with autophagy-mediated degradation of misfolded proteins by a dominant-negative mechanism, Autophagy. 11 (2015) 685-700. doi:10.4161/auto.36098.

[125] A.J. Kruppa, J. Kendrick-Jones, F. Buss, Myosins, Actin and Autophagy, Traffic. 17 (2016) 878-890. doi:10.1111/tra.12410.

[126] V. Sundaramoorthy, A.K. Walker, V. Tan, J.A. Fifita, E.P. McCann, K.L. Williams, I.P. Blair, G.J. Guillemain, M.A. Farg, J.D. Atkin, Defects in optineurin-and myosin VI-mediated cellular trafficking in amyotrophic lateral sclerosis, Hum. Mol. Genet. 24 (2015) 3830-3846. doi:10.1093/hmg/ddv126.

[127] G.J. Song, H. Jeon, M. Seo, M. Jo, K. Suk, Interaction between optineurin and Rab1a regulates autophagosome formation in neuroblastoma cells, J. Neurosci. Res. (2017). doi:10.1002/jnr.24143.

[128] M. Osaka, D. Ito, T. Yagi, Y. Nihei, N. Suzuki, Evidence of a link between ubiquilin 2 and optineurin in amyotrophic lateral sclerosis, Hum. Mol. Genet. 24 (2015) 1617-1629. doi:10.1093/hmg/ddu575.

[129] C. Pedros, Y. Zhang, J.K. Hu, Y.S. Choi, A.J. Canonigo-Balancio, J.R. Yates, A. Altman, S. Crotty, K.F. Kong, A TRAF-like motif of the inducible costimulator ICOS controls development of germinal center TFH cells via the kinase TBK1, Nat. Immunol. 17 (2016) 825-833. doi:10.1038/ni.3463.

[130] Q. Zhang, F. Meng, S. Chen, S.W. Plouffe, S. Wu, S. Liu, X. Li, R. Zhou, J. Wang, B. Zhao, J. Liu, J. Qin, J. Zou, X.H. Feng, K.L. Guan, P. Xu, Hippo signalling governs cytosolic nucleic acid sensing through YAP/TAZ-mediated TBK1 blockade, Nat. Cell Biol. 19 (2017) 362-374. doi:10.1038/ncb3496.

[131] J.A. Oakes, M.C. Davies, M.O. Collins, TBK1: a new player in ALS linking autophagy and neuroinflammation, Mol. Brain. 10 (2017) 5. doi:10.1186/s13041-017-0287-x.

[132] P. Wild, H. Farhan, D.G. McEwan, S. Wagner, V.V. Rogov, N.R. Brady, B. Richter, J. Korac, O. Waidmann, C. Choudhary, V. Dotsch, D. Bumann, I. Dikic, Phosphorylation of the Autophagy Receptor Optineurin Restricts Salmonella Growth, Science. 333 (2011) 228-233. doi:10.1126/science.1205405.

[133] M. Pilli, J. Arko-Mensah, M. Ponpuak, E. Roberts, S. Master, M.A. Mandell, N. Dupont, W. Ornatowski, S. Jiang, S.B. Bradfute, J.A. Bruun, T.E. Hansen, T. Johansen, V. Deretic, TBK-1 Promotes Autophagy-Mediated Antimicrobial Defense by Controlling Autophagosome Maturation, Immunity. 37 (2012) 223-234. doi:10.1016/j.immuni.2012.04.015.

[134] A. Freischmidt, T. Wieland, B. Richter, W. Ruf, V. Schaeffer, K. Müller, N. Marroquin, F. Nordin, A. Hubers, P. Weydt, S. Pinto, R. Press, S. Millecamps, N. Molko, E. Bernard, C. Desnuelle, M.-H. Soriani, J. Dorst, E. Graf, U. Nordström, M.S. Feiler, S. Putz, T.M. Boeckers, T. Meyer, A.S. Winkler, J. Winkelmann, M. de Carvalho, D.R. Thal, M. Otto, T. Brännström, A.E. Volk, P. Kursula, K.M. Danzer, P. Lichtner, I. Dikic, T. Meitinger, A.C. Ludolph, T.M. Strom, P.M. Andersen, J.H. Weishaupt, Haploinsufficiency of TBK1 causes familial ALS and fronto-temporal dementia, Nat. Neurosci. 18 (2015) 631-636. doi:10.1038/nn.4000.

[135] I. Gijselinck, S. Van Mossevelde, J. van der Zee, A. Sieben, S. Philtjens, B. Heeman, S.

Engelborghs, M. Vandenbulcke, G. De Baets, V. Bäumer, I. Cuijt, M. Van den Broeck, K. Peeters, M. Mattheijssens, F. Rousseau, R. Vandenberghe, P. De Jonghe, P. Cras, P.P. De Deyn, J.-J. Martin, M. Cruts, C. Van Broeckhoven, BELNEU Consortium, Loss of TBK1 is a frequent cause of frontotemporal dementia in a Belgian cohort., Neurology. 85 (2015) 2116–25. doi:10.1212/WNL.0000000000002220.

[136] I. Le Ber, A. De Septenville, S. Millecamps, A. Camuzat, P. Caroppo, P. Couratier, F. Blanc, L. Lacomblez, F. Sellal, M.C. Fleury, V. Meininge, C. Cazeneuve, F. Clot, O. Flabeau, E. LeGuern, A. Brice, M. Didic, B. Dubois, V. Golfier, D. Hannequin, R. Levy, B.F. Michel, F. Pasquier, C. Thomas-Anterion, M. Puel, F. Salachas, M. Vercelletto, TBK1 mutation frequencies in French frontotemporal dementia and amyotrophic lateral sclerosis cohorts, Neurobiol. Aging. 36 (2015) 3116.e5–3116.e8. doi:10.1016/j.neurobiolaging.2015.08.009.

[137] M. Yang, C. Liang, K. Swaminathan, S. Herrlinger, F. Lai, R. Shiekhattar, J.-F. Chen, A C9ORF72/SMCR8-containing complex regulates ULK1 and plays a dual role in autophagy, Sci. Adv. 2 (2016) e1601167–e1601167. doi:10.1126/sciadv.1601167.

[138] J. Ugolino, Y.J. Ji, K. Conchina, J. Chu, R.S. Nirujogi, A. Pandey, N.R. Brady, A. Hamacher-Brady, J. Wang, Loss of C9orf72 Enhances Autophagic Activity via Deregulated mTOR and TFEB Signaling, PLoS Genet. 12 (2016). doi:10.1371/journal.pgen.1006443.

[139] J. Amick, A. Rocznia-Ferguson, S.M. Ferguson, C9orf72 binds SMCR8, localizes to lysosomes, and regulates mTORC1 signaling, Mol. Biol. Cell. 27 (2016) 3040–3051. doi:10.1091/mbc.E16-01-0003.

[140] C. Sellier, M. Campanari, C. Julie Corbier, A. Gaucherot, I. Kolb-Cheynel, M. Oulad-Abdelghani, F. Ruffenach, A. Page, S. Ciura, E. Kabashi, N. Charlet-Berguerand, Loss of C9ORF72 impairs autophagy and synergizes with polyQ Ataxin-2 to induce motor neuron dysfunction and cell death, EMBO J. 35 (2016) 1276–1297. doi:10.15252/emboj.201593350.

[141] K. Sirohi, A. Kumari, V. Radha, G. Swarup, A Glaucoma-Associated variant of optineurin, M98K, activates Tbk1 to enhance autophagosome formation and retinal cell death dependent on Ser 177 phosphorylation of optineurin, PLoS One. 10 (2015). doi:10.1371/journal.pone.0138289.

[142] S. Pillai, J. Nguyen, J. Johnson, E. Haura, D. Coppola, S. Chellappan, Tank binding kinase 1 is a centrosome-associated kinase necessary for microtubule dynamics and mitosis, Nat. Commun. 6 (2015) 10072. doi:10.1038/ncomms10072.

[143] J.M. Heo, A. Ordureau, J.A. Paulo, J. Rinehart, J.W. Harper, The PINK1-PARKIN Mitochondrial Ubiquitylation Pathway Drives a Program of OPTN/NDP52 Recruitment and TBK1 Activation to Promote Mitophagy, Mol. Cell. 60 (2015) 7–20. doi:10.1016/j.molcel.2015.08.016.

[144] B. Richter, D.A. Sliter, L. Herhaus, A. Stolz, C. Wang, P. Beli, G. Zaffagnini, P. Wild, S. Martens, S.A. Wagner, R.J. Youle, I. Dikic, Phosphorylation of OPTN by TBK1 enhances its binding to Ub chains and promotes selective autophagy of damaged mitochondria, Proc. Natl. Acad. Sci. 113 (2016) 4039–4044. doi:10.1073/pnas.1523926113.

[145] S. Millecamps, F. Salachas, C. Cazeneuve, P. Gordon, B. Bricka, A. Camuzat, L. Guillot-Noël, O. Russaouen, G. Bruneteau, P.-F. Pradat, N. Le Forestier, N. Vandenberghe, V. Danel-Brunaud, N. Guy, C. Thauvin-Robinet, L. Lacomblez, P. Couratier, D. Hannequin, D. Seilhean, I. Le Ber, P.

Corcia, W. Camu, A. Brice, G. Rouleau, E. LeGuern, V. Meininger, SOD1, ANG, VAPB, TARDBP, and FUS mutations in familial amyotrophic lateral sclerosis: genotype-phenotype correlations., J. Med. Genet. 47 (2010) 554–60. doi:10.1136/jmg.2010.077180.

[146] P.M. Andersen, A. Al-Chalabi, Clinical genetics of amyotrophic lateral sclerosis: What do we really know?, Nat. Rev. Neurol. 7 (2011) 603–615. doi:10.1038/nrneurol.2011.150.

[147] L. Aliaga, C. Lai, J. Yu, N. Chub, H. Shim, L. Sun, C. Xie, W.J. Yang, X. Lin, M.J. O'Donovan, H. Cai, Amyotrophic lateral sclerosis-related VAPBP56S mutation differentially affects the function and survival of corticospinal and spinal motor neurons, Hum. Mol. Genet. 22 (2013) 4293–4305. doi:10.1093/hmg/ddt279.

[148] M. Hamasaki, N. Furuta, A. Matsuda, A. Nezu, A. Yamamoto, N. Fujita, H. Oomori, T. Noda, T. Haraguchi, Y. Hiraoka, A. Amano, T. Yoshimori, Autophagosomes form at ER-mitochondria contact sites, Nature. 495 (2013) 389–393. doi:10.1038/nature11910.

[149] D. Hantan, Y. Yamamoto, T. Sakisaka, VAP-B binds to Rab3GAP1 at the ER: Its implication in nuclear envelope formation through the ER-Golgi intermediate compartment, Kobe J. Med. Sci. 60 (2014) E48–E56.

[150] N. Spang, A. Feldmann, H. Huesmann, F. Bekbulat, V. Schmitt, C. Hiebel, I. Koziollek-Drechsler, A.M. Clement, B. Moosmann, J. Jung, C. Behrends, I. Dikic, A. Kern, C. Behl, RAB3GAP1 and RAB3GAP2 modulate basal and rapamycin-induced autophagy, Autophagy. 10 (2014) 2297–2309. doi:10.4161/15548627.2014.994359.

[151] E. Fasana, M. Fossati, A. Ruggiano, S. Brambilla sca, C.C. Hoogenraad, F. Navone, M. Francolini, N. Borgese, A VAPB mutant linked to amyotrophic lateral sclerosis generates a novel form of organized smooth endoplasmic reticulum, FASEB J. 24 (2010) 1419–1430. doi:10.1096/fj.09-147850.

[152] M. Kuijpers, V. van Dis, E.D. Haasdijk, M. Harterink, K. Vocking, J. a Post, W. Scheper, C.C. Hoogenraad, D. Jaarsma, Amyotrophic lateral sclerosis (ALS)-associated VAPB-P56S inclusions represent an ER quality control compartment., Acta Neuropathol. Commun. 1 (2013) 24. doi:10.1186/2051-5960-1-24.

[153] F. Larroquette, L. Seto, P.L. Gaub, B. Kamal, D. Wallis, R. Larivière, J. Vallée, R. Robitaille, H. Tsuda, Vapb/Amyotrophic lateral sclerosis 8 knock-in mice display slowly progressive motor behavior defects accompanying ER stress and autophagic response, Hum. Mol. Genet. 24 (2015) 6515–6529. doi:10.1093/hmg/ddv360.

[154] A. Kakizuka, Roles of VCP in human neurodegenerative disorders., Biochem. Soc. Trans. 36 (2008) 105–108. doi:10.1042/BST0360105.

[155] B. DeLaBarre, J.C. Christianson, R.R. Kopito, A.T. Brunger, Central Pore Residues Mediate the p97/VCP Activity Required for ERAD, Mol. Cell. 22 (2006) 451–462. doi:10.1016/j.molcel.2006.03.036.

[156] J.S. Ju, R.A. Fuentealba, S.E. Miller, E. Jackson, D. Piwnica-Worms, R.H. Baloh, C.C. Weihl, Valosin-containing protein (VCP) is required for autophagy and is disrupted in VCP disease, J. Cell Biol. 187 (2009) 875–888. doi:10.1083/jcb.200908115.

[157] E. Tresse, F.A. Salomons, J. Vesa, L.C. Bott, V. Kimonis, T.P. Yao, N.P. Dantuma, J.P. Taylor,

VCP/p97 is essential for maturation of ubiquitin-containing autophagosomes and this function is impaired by mutations that cause IBMPFD, Autophagy. 6 (2010) 217–227. doi:10.4161/auto.6.2.11014.

[158] G.D.G. Watts, D. Thomasova, S.K. Ramdeen, E.C. Fulchiero, S.G. Mehta, D.A. Drachman, C.C. Weihl, Z. Jamrozik, H. Kwiecinski, A. Kaminska, V.E. Kimonis, Novel VCP mutations in inclusion body myopathy associated with Paget disease of bone and frontotemporal dementia, Clin. Genet. 72 (2007) 420–426. doi:10.1111/j.1399-0004.2007.00887.x.

[159] A.E. Renton, A. Chiò, B.J. Traynor, State of play in amyotrophic lateral sclerosis genetics, Nat. Neurosci. 17 (2013) 17–23. doi:10.1038/nn.3584.

[160] S.K. Custer, M. Neumann, H. Lu, A.C. Wright, J.P. Taylor, Transgenic mice expressing mutant forms VCP/p97 recapitulate the full spectrum of IBMPFD including degeneration in muscle, brain and bone, Hum. Mol. Genet. 19 (2010) 1741–1755. doi:10.1093/hmg/ddq050.

[161] A. Nalbandian, K.J. Llewellyn, M. Kitazawa, H.Z. Yin, M. Badadani, N. Khanlou, R. Edwards, C. Nguyen, J. Mukherjee, T. Mozaffar, G. Watts, J. Weiss, V.E. Kimonis, The Homozygote VCPR155H/R155H Mouse Model Exhibits Accelerated Human VCP-Associated Disease Pathology, PLoS One. 7 (2012). doi:10.1371/journal.pone.0046308.

[162] B.K. Yeo, C.J. Hong, K.M. Chung, H. Woo, K. Kim, S. Jung, E.-K. Kim, S.-W. Yu, Valosin-containing protein is a key mediator between autophagic cell death and apoptosis in adult hippocampal neural stem cells following insulin withdrawal, Mol. Brain. 9 (2016) 31. doi:10.1186/s13041-016-0212-8.

[163] J.K. Ching, C.C. Weihl, Rapamycin-induced autophagy aggravates pathology and weakness in a mouse model of VCP-associated myopathy, Autophagy. 9 (2013) 799–800. doi:10.4161/auto.23958.

[164] J.R. Buchan, R.-M. Kolaitis, J.P. Taylor, R. Parker, Eukaryotic Stress Granules Are Cleared by Autophagy and Cdc48/VCP Function, Cell. 153 (2013) 1461–1474. doi:10.1016/j.cell.2013.05.037.

[165] J.C. Fong, A.M. Karydas, J.S. Goldman, Genetic counseling for FTD/ALS caused by the C9ORF72 hexanucleotide expansion, Alzheimers. Res. Ther. 4 (2012) 27. doi:10.1186/alzrt130.

[166] V.V. Belzil, P.O. Bauer, M. Prudencio, T.F. Gendron, C.T. Stetler, I.K. Yan, L. Pregent, L. Daughrity, M.C. Baker, R. Rademakers, K. Boylan, T.C. Patel, D.W. Dickson, L. Petrucelli, Reduced C9orf72 gene expression in c9FTD/ALS is caused by histone trimethylation, an epigenetic event detectable in blood, Acta Neuropathol. 126 (2013) 895–905. doi:10.1007/s00401-013-1199-1.

[167] C.J. Donnelly, P.W. Zhang, J.T. Pham, A.R. Heusler, N.A. Mistry, S. Vidensky, E.L. Daley, E.M. Poth, B. Hoover, D.M. Fines, N. Maragakis, P.J. Tienari, L. Petrucelli, B.J. Traynor, J. Wang, F. Rigo, C.F. Bennett, S. Blackshaw, R. Sattler, J.D. Rothstein, RNA Toxicity from the ALS/FTD C9ORF72 Expansion Is Mitigated by Antisense Intervention, Neuron. 80 (2013) 415–428. doi:10.1016/j.neuron.2013.10.015.

[168] A.R. Haeusler, C.J. Donnelly, G. Periz, E.A.J. Simko, P.G. Shaw, M.-S. Kim, N.J. Maragakis, J.C. Troncoso, A. Pandey, R. Sattler, J.D. Rothstein, J. Wang, C9orf72 nucleotide repeat structures

initiate molecular cascades of disease, Nature. 507 (2014) 195–200.
doi:10.1038/nature13124.

[169] P.E.A. Ash, K.F. Bieniek, T.F. Gendron, T. Caulfield, W.L. Lin, M. DeJesus-Hernandez, M.M. Van Blitterswijk, K. Jansen-West, J.W. Paul, R. Rademakers, K.B. Boylan, D.W. Dickson, L. Petrucelli, Unconventional Translation of C9ORF72 GGGGCC Expansion Generates Insoluble Polypeptides Specific to c9FTD/ALS, Neuron. 77 (2013) 639–646.
doi:10.1016/j.neuron.2013.02.004.

[170] S. Mizielinska, S. Gronke, T. Niccoli, C.E. Ridler, E.L. Clayton, A. Devoy, T. Moens, F.E. Norona, I.O.C. Woollacott, J. Pietrzyk, K. Cleverley, A.J. Nicoll, S. Pickering-Brown, J. Dols, M. Cabecinha, O. Hendrich, P. Fratta, E.M.C. Fisher, L. Partridge, A.M. Isaacs, C9orf72 repeat expansions cause neurodegeneration in Drosophila through arginine-rich proteins, Science (80-) . 345 (2014) 1192–1194. doi:10.1126/science.1256800.

[171] Z. Tao, H. Wang, Q. Xia, K. Li, K. Li, X. Jiang, G. Xu, G. Wang, Z. Ying, Nucleolar stress and impaired stress granule formation contribute to C9orf72RAN translation-induced cytotoxicity, Hum. Mol. Genet. 24 (2015) 2426–2441. doi:10.1093/hmg/ddv005.

[172] S. Almeida, E. Gascon, H. Tran, H.J. Chou, T.F. Gendron, S. Degroot, A.R. Tapper, C. Sellier, N. Charlet-Berguerand, A. Karydas, W.W. Seeley, A.L. Boxer, L. Petrucelli, B.L. Miller, F.B. Gao, Modeling key pathological features of frontotemporal dementia with C9ORF72 repeat expansion in iPSC-derived human neurons, Acta Neuropathol. 126 (2013) 385–399.
doi:10.1007/s00401-013-1149-y.

[173] K. Zhang, C.J. Donnelly, A.R. Haeusler, J.C. Grima, J.B. Machamer, P. Steinwald, E.L. Daley, S.J. Miller, K.M. Cunningham, S. Vidensky, S. Gupta, M.A. Thomas, I. Hong, S.-L. Chiu, R.L. Huganir, L.W. Ostrow, M.J. Matunis, J. Wang, R. Sattler, T.E. Lloyd, J.D. Rothstein, The C9orf72 repeat expansion disrupts nucleocytoplasmic transport, Nature. 525 (2015) 56–61.
doi:10.1038/nature14973.

[174] A. Jovičić, J. Mertens, S. Boeynaems, E. Bogaert, N. Chai, S.B. Yamada, J.W. Paul, S. Sun, J.R. Herdy, G. Bieri, N.J. Kramer, F.H. Gage, L. Van Den Bosch, W. Robberecht, A.D. Gitler, Modifiers of C9orf72 dipeptide repeat toxicity connect nucleocytoplasmic transport defects to FTD/ALS, Nat. Neurosci. 18 (2015) 1226–1229. doi:10.1038/nn.4085.

[175] B.D. Freibaum, Y. Lu, R. Lopez-Gonzalez, N.C. Kim, S. Almeida, K.-H. Lee, N. Badders, M. Valentine, B.L. Miller, P.C. Wong, L. Petrucelli, H.J. Kim, F.-B. Gao, J.P. Taylor, GGGGCC repeat expansion in C9orf72 compromises nucleocytoplasmic transport, Nature. 525 (2015) 129–133. doi:10.1038/nature14974.

[176] A.J. Waite, D. Bäumer, S. East, J. Neal, H.R. Morris, O. Ansorge, D.J. Blake, Reduced C9orf72 protein levels in frontal cortex of amyotrophic lateral sclerosis and frontotemporal degeneration brain with the C90RF72 hexanucleotide repeat expansion, Neurobiol. Aging. 35 (2014) 1779.e5–1779.e13. doi:10.1016/j.neurobiolaging.2014.01.016.

[177] D. Zhang, L.M. Iyer, F. He, L. Aravind, Discovery of novel DENN proteins: Implications for the evolution of eukaryotic intracellular membrane structures and human disease, Front. Genet. 3 (2012). doi:10.3389/fgene.2012.00283.

[178] A.L. Marat, H. Dokainish, P.S. McPherson, DENN domain proteins: Regulators of Rab GTPases,

J. Biol. Chem. 286 (2011) 13791–13800. doi:10.1074/jbc.R110.217067.

[179] T.P. Levine, R.D. Daniels, A.T. Gatta, L.H. Wong, M.J. Hayes, The product of C9orf72, a gene strongly implicated in neurodegeneration, is structurally related to DENN Rab-GEFs, Bioinformatics. 29 (2013) 499–503. doi:10.1093/bioinformatics/bts725.

[180] M.A. Farg, V. Sundaramoorthy, J.M. Sultana, S. Yang, R.A.K. Atkinson, V. Levina, M.A. Halloran, P.A. Gleeson, I.P. Blair, K.Y. Soo, A.E. King, J.D. Atkin, C9ORF72, implicated in amyotrophic lateral sclerosis and frontotemporal dementia, regulates endosomal trafficking, Hum. Mol. Genet. 23 (2014) 3579–3595. doi:10.1093/hmg/ddu068.

[181] Y. Aoki, R. Manzano, Y. Lee, R. Dafinca, M. Aoki, A.G.L. Douglas, M.A. Varela, C. Sathyaprakash, J. Scaber, P. Barbagallo, P. Vader, I. Mäger, K. Ezzat, M.R. Turner, N. Ito, S. Gasco, N. Ohbayashi, S. El Andaloussi, S. Takeda, M. Fukuda, K. Talbot, M.J.A. Wood, C9orf72 and RAB7L1 regulate vesicle trafficking in amyotrophic lateral sclerosis and frontotemporal dementia, Brain. 140 (2017) 887–897. doi:10.1093/brain/awx024.

[182] R. Sivadasan, D. Hornburg, C. Drepper, N. Frank, S. Jablonka, A. Hansel, X. Lojewski, J. Sterneckert, A. Hermann, P.J. Shaw, P.G. Ince, M. Mann, F. Meissner, M. Sendtner, C9ORF72 interaction with cofilin modulates actin dynamics in motor neurons, Nat. Neurosci. 19 (2016) 1610–1618. doi:10.1038/nn.4407.

[183] M.O. Aguilera, W. Berón, M.I. Colombo, The actin cytoskeleton participates in the early events of autophagosome formation upon starvation induced autophagy, Autophagy. 8 (2012) 1590–1603. doi:10.4161/auto.21459.

[184] J.-Y. Lee, H. Koga, Y. Kawaguchi, W. Tang, E. Wong, Y.-S. Gao, U.B. Pandey, S. Kaushik, E. Tresse, J. Lu, J.P. Taylor, A.M. Cuervo, T.-P. Yao, HDAC6 controls autophagosome maturation essential for ubiquitin-selective quality-control autophagy, EMBO J. 29 (2010) 969–980. doi:10.1038/emboj.2009.405.

[185] C. Zhuo, Y. Ji, Z. Chen, K. Kitazato, Y. Xiang, M. Zhong, Q. Wang, Y. Pei, H. Ju, Y. Wang, Proteomics analysis of autophagy-deficient Atg7-/- MEFs reveals a close relationship between F-actin and autophagy, Biochem. Biophys. Res. Commun. 437 (2013) 482–488. doi:10.1016/j.bbrc.2013.06.111.

[186] P.M. Sullivan, X. Zhou, A.M. Robins, D.H. Paushter, D. Kim, M.B. Smolka, F. Hu, The ALS/FTLD associated protein C9orf72 associates with SMCR8 and WDR41 to regulate the autophagy-lysosome pathway, Acta Neuropathol. Commun. 4 (2016) 51. doi:10.1186/s40478-016-0324-5.

[187] C.P. Webster, E.F. Smith, C.S. Bauer, A. Moller, G.M. Hautbergue, L. Ferraiuolo, M.A. Myszczynska, A. Higginbottom, M.J. Walsh, A.J. Whitworth, B.K. Kaspar, K. Meyer, P.J. Shaw, A.J. Grierson, K.J. De Vos, The C9orf72 protein interacts with Rab1a and the ULK1 complex to regulate initiation of autophagy, EMBO J. 35 (2016) 1656–1676. doi:10.15252/embj.201694401.

[188] J. Jung, A. Nayak, V. Schaeffer, T. Starzetz, A.K. Kirsch, S. Müller, I. Dikic, M. Mittelbronn, C. Behrends, Multiplex image-based autophagy RNAi screening identifies SMCR8 as ULK1 kinase activity and gene expression regulator., Elife. 6 (2017). doi:10.7554/eLife.23063.

[189] J.G. O'Rourke, L. Bogdanik, A. Yanez, D. Lall, A.J. Wolf, A.K.M.G. Muhammad, R. Ho, S. Carmona,


J.P. Vit, J. Zarrow, K.J. Kim, S. Bell, M.B. Harms, T.M. Miller, C.A. Dangler, D.M. Underhill, H.S. Goodridge, C.M. Lutz, R.H. Baloh, C9orf72 is required for proper macrophage and microglial function in mice, Science. 351 (2016) 1324–1329. doi:10.1126/science.aaf1064.

[190] C. Rothenberg, D. Srinivasan, L. Mah, S. Kaushik, C.M. Peterhoff, J. Ugolino, S. Fang, A.M. Cuervo, R.A. Nixon, M.J. Monteiro, Ubiquilin functions in autophagy and is degraded by chaperone-mediated autophagy, Hum. Mol. Genet. 19 (2010) 3219–3232. doi:10.1093/hmg/ddq231.

[191] E.-N. N'Diaye, K.K. Kajiha ra, I. Hsieh, H. Morisaki, J. Debnath, E.J. Brown, PLIC proteins or ubiquilins regulate autophagy-dependent cell survival during nutrient starvation, EMBO Rep. 10 (2009) 173–179. doi:10.1038/embor.2008.238.

[192] H. Daoud, H. Suhail, A. Szuto, W. Camu, F. Salachas, V. Meining er, J.P. Bouchard, N. Dupr??, P.A. Dion, G.A. Rouleau, UBQLN2 mutations are rare in French and French-Canadian amyotrophic lateral sclerosis, Neurobiol. Aging. 33 (2012). doi:10.1016/j.neurobiolaging.2012.03.015.

[193] S. Millecamps, P. Corcia, C. Cazeneuve, S. Boillée, D. Seilhean, V. Danel-Brunaud, N. Vandenberghe, P.F. Pradat, N. Le Forestier, L. Lacomblez, G. Bruneteau, W. Camu, A. Brice, V. Meining er, E. LeGuern, F. Salachas, Mutations in UBQLN2 are rare in French amyotrophic lateral sclerosis, Neurobiol. Aging. 33 (2012) 839.e1–839.e3. doi:10.1016/j.neurobiolaging.2011.11.010.

[194] L. Dillen, T. Van Langenhove, S. Engelborghs, M. Vandenbulcke, S. Sarafov, I. Tournev, C. Merlin, P. Cras, R. Vandenberghe, P.P. De Deyn, A. Jordanova, M. Cruts, C. Van Broeckhoven, J. van der Zee, Explorative genetic study of <em>UBQLN2</em> and <em>PFN1</em> in an extended Flanders-Belgian cohort of frontotemporal lobar degeneration patients, Neurobiol. Aging. 34 (2017) 1711.e1–1711.e5. doi:10.1016/j.neurobiolaging.2012.12.007.

[195] M. Synofzik, W. Maetzler, T. Gre hl, J. Prudlo, J.M. vom Hagen, T. Haack, P. Rebassoo, M. Munz, L. Schöls, S. Biskup, Screening in ALS and FTD patients reveals 3 novel UBQLN2 mutations outside the PXX domain and a pure FTD phenotype, Neurobiol. Aging. 33 (2012). doi:10.1016/j.neurobiolaging.2012.07.002.

[196] M. Osaka, D. Ito, N. Suzuki, Disturbance of proteasomal and autophagic protein degradation pathways by amyotrophic lateral sclerosis-linked mutations in ubiquilin 2, Biochem. Biophys. Res. Commun. 472 (2016) 324–331. doi:10.1016/j.bbrc.2016.02.107.

[197] Q. Wu, M. Liu, C. Huang, X. Liu, B. Huang, N. Li, H. Zhou, X.G. Xia, Pathogenic Ubqln2 gains toxic properties to induce neuron death, Acta Neuropathol. 129 (2014) 417–428. doi:10.1007/s00401-014-1367-y.

[198] M. Razi, E.Y.W. Chan, S.A. Tooze, Early endosomes and endosomal coatomer are required for Autophagy, J. Cell Biol. 185 (2009) 305–321. doi:10.1083/jcb.200810098.

[199] B. Huang, Q. Wu, H. Zhou, C. Huang, X.G. Xia, Increased Ubqln2 expression causes neuron death in transgenic rats, J. Neurochem. (2016) 285–293. doi:10.1111/jnc.13748.

[200] J.D. Topp, N.W. Gray, R.D. Gerard, B.F. Horazdovsky, Alsin is a Rab5 and Rac1 guanine nucleotide exchange factor, J. Biol. Chem. 279 (2004) 24612–24623. doi:10.1074/jbc.M313504200.

[201] D.S. Carney, B.A. Davies, B.F. Horazdovsky, Vps 9 domain-containing proteins: activators of Rab5 GTPases from yeast to neurons, Trends Cell Biol. 16 (2006) 27–35. doi:10.1016/j.tcb.2005.11.001.

[202] A. Otomo, S. Hadano, T. Okada, H. Mizumura, R. Kunita, H. Nishijima, J. Showguchi-Miyata, Y. Yanagisawa, E. Kohiki, E. Suga, M. Yasuda, H. Osuga, T. Nishimoto, S. Narumiya, J.E. Ikeda, ALS2, a novel guanine nucleotide exchange factor for the small GTPase Rab5, is implicated in endosomal dynamics, Hum. Mol. Genet. 12 (2003) 1671–1687. doi:10.1093/hmg/ddg184.

[203] R.S. Devon, P.C. Orban, K. Gerrow, M. a Barbieri, C. Schwab, L.P. Cao, J.R. Helm, N. Bissada, R. Cruz-Aguado, T.-L. Davidson, J. Witmer, M. Metzler, C.K. Lam, W. Tetzlaff, E.M. Simpson, J.M. McCaffery, a E. El-Husseini, B.R. Leavitt, M.R. Hayden, Als2-deficient mice exhibit disturbances in endosome trafficking associated with motor behavioral abnormalities., Proc. Natl. Acad. Sci. U.S.A. 103 (2006) 9595–600. doi:10.1073/pnas.0510197103.

[204] S. Hadano, R. Kunita, A. Otomo, K. Suzuki-Utsunomiya, J.E. Ikeda, Molecular and cellular function of ALS2/alsin: Implication of membrane dynamics in neuronal development and degeneration, Neurochem. Int. 51 (2007) 74–84. doi:10.1016/j.neuint.2007.04.010.

[205] A. Otomo, R. Kunita, K. Suzuki-Utsunomiya, J.E. Ikeda, S. Hadano, Defective relocalization of ALS2/alsin missense mutants to Rac1-induced macropinosomes accounts for loss of their cellular function and leads to disturbed amphisome formation, FEBS Lett. 585 (2011) 730–736. doi:10.1016/j.febslet.2011.01.045.

[206] A. Kyotani, Y. Azuma, I. Yamamoto, H. Yoshida, I. Mizuta, T. Mizuno, M. Nakagawa, T. Tokuda, M. Yamaguchi, Knockdown of the Drosophila FIG4 induces deficient locomotive behavior, shortening of motor neuron, axonal targeting aberration, reduction of life span and defects in eye development, Exp. Neurol. 277 (2016) 86–95. doi:10.1016/j.expneurol.2015.12.011.

[207] T.E. Rusten, A. Simonsen, ESCRT functions in autophagy and associated disease, Cell Cycle. 7 (2008) 1166–1172. doi:10.4161/cc.7.9.5784.

[208] G. Bodon, R. Chassefeyre, K. Pernet-Gallay, N. Martinelli, G. Effantin, D.L. Hulsik, A. Belly, Y. Goldberg, C. Chatellard-Causse, B. Blot, G. Schoehn, W. Weissenhorn, R. Sadoul, Charged Multivesicular Body Protein 2B (CHMP2B) of the Endosomal Sorting Complex Required for Transport-III (ESCRT-III) polymerizes into helical structures deforming the plasma membrane, J. Biol. Chem. 286 (2011) 40276–40286. doi:10.1074/jbc.M111.283671.

[209] M. Filimonenko, S. Stuffers, C. Raiborg, A. Yamamoto, L. Malerød, E.M.C. Fisher, A. Isaacs, A. Brech, H. Stenmark, A. Simonsen, Functional multivesicular bodies are required for autophagic clearance of protein aggregates associated with neurodegenerative disease, J. Cell Biol. 179 (2007) 485–500. doi:10.1083/jcb.200702115.

[210] J.A. Lee, A. Beigneux, S.T. Ahmad, S.G. Young, F.B. Gao, ESCRT-III Dysfunction Causes Autophagosome Accumulation and Neurodegeneration, Curr. Biol. 17 (2007) 1561–1567. doi:10.1016/j.cub.2007.07.029.

[211] S. Ghazi-Noori, K.E. Froud, S. Mizielinska, C. Powell, M. Smidak, M. Fernandez De Marco, C. O'Malley, M. Farmer, N. Parkinson, E.M.C. Fisher, E.A. Asante, S. Brandner, J. Collinge, A.M. Isaacs, Progressive neuronal inclusion formation and axonal degeneration in CHMP2B mutant transgenic mice, Brain. 135 (2012) 819–832. doi:10.1093/brain/aws006.

[212] J.H. Han, H.H. Ryu, M.H. Jun, D.J. Jang, J.A. Lee, The functional analysis of the CHMP2B missense mutation associated with neurodegenerative diseases in the endo-lysosomal pathway, Biochem. Biophys. Res. Commun. 421 (2012) 544–549. doi:10.1016/j.bbrc.2012.04.041.

[213] H. Urwin, A. Authier, J.E. Nielsen, D. Metcalf, C. Powell, K. Froud, D.S. Malcolm, I. Holm, P. Johannsen, J. Brown, E.M.C. Fisher, J. van der Zee, M. Bruyland, C. Van Broeckhoven, J. Collinge, S. Brandner, C. Futter, A.M. Isaacs, Disruption of endocytic trafficking in frontotemporal dementia with CHMP2B mutations, Hum. Mol. Genet. 19 (2010) 2228–2238. doi:10.1093/hmg/ddq100.

[214] Y. Lu, Z. Zhang, D. Sun, S.T. Sweeney, F.B. Gao, Syntaxin 13, a genetic modifier of mutant CHMP2B in frontotemporal dementia, is required for autophagosome maturation, Mol. Cell. 52 (2013) 264–271. doi:10.1016/j.molcel.2013.08.041.

[215] X. Liu, L. Yang, L. Tang, L. Chen, X. Liu, D. Fan, DCTN1 gene analysis in Chinese patients with sporadic amyotrophic lateral sclerosis, PLoS One. 12 (2017) 1–8. doi:10.1371/journal.pone.0182572.

[216] I. Puls, S.J. Oh, C.J. Sumner, K.E. Wallace, M.K. Floeter, E.A. Mann, W.R. Kennedy, G. Wendelschafer-Crabb, A. Vortmeyer, R. Powers, K. Finnegan, E.L.F. Holzbaur, K.H. Fischbeck, C.L. Ludlow, Distal spinal and bulbar muscular atrophy caused by dynactin mutation, Ann. Neurol. 57 (2005) 687–694. doi:10.1002/ana.20468.

[217] C. Munch, R. Sedlmeier, T. Meyer, V. Homberg, A.D. Sperfeld, A. Kurt, J. Prudlo, G. Peraus, C.O. Hanemann, G. Stumm, A.C. Ludolph, Point mutations of the p150 subunit of dynactin (DCTN1) gene in ALS, Neurology. 63 (2004) 724–726. doi:10.1212/01.WNL.0000134608.83927.B1.

[218] J.R. Levy, C.J. Sumner, J.P. Caviston, M.K. Tokito, S. Ranganathan, L.A. Ligon, K.E. Wallace, B.H. LaMonte, G.G. Harmison, I. Puls, K.H. Fischbeck, E.L.F. Holzbaur, A motor neuron disease-associated mutation in p150Glued perturbs dynactin function and induces protein aggregation, J. Cell Biol. 172 (2006) 733–745. doi:10.1083/jcb.200511068.

[219] F.M. Laird, M.H. Farah, S. Ackerley, A. Hoke, N. Maragakis, J.D. Rothstein, J. Griffin, D.L. Price, L.J. Martin, P.C. Wong, Motor Neuron Disease Occurring in a Mutant Dynactin Mouse Model Is Characterized by Defects in Vesicular Trafficking, J. Neurosci. 28 (2008) 1997–2005. doi:10.1523/JNEUROSCI.4231-07.2008.

[220] K. Ikenaka, K. Kawai, M. Katsuno, Z. Huang, Y.-M. Jiang, Y. Iguchi, K. Kobayashi, T. Kimata, M. Waza, F. Tanaka, I. Mori, G. Sobue, dnc-1/dynactin 1 Knockdown Disrupts Transport of Autophagosomes and Induces Motor Neuron Degeneration, PLoS One. 8 (2013) e54511. doi:10.1371/journal.pone.0054511.

[221] Y.-M. Jiang, M. Yamamoto, F. Tanaka, S. Ishigaki, M. Katsuno, H. Adachi, J. Niwa, M. Doyu, M. Yoshida, Y. Hashizume, G. Sobue, Gene Expressions Specifically Detected in Motor Neurons (Dynactin 1, Early Growth Response 3, Acetyl-CoA Transporter, Death Receptor 5, and Cyclin C) Differentially Correlate to Pathologic Markers in Sporadic Amyotrophic Lateral Sclerosis, J. Neuropathol. Exp. Neurol. 66 (2007) 617–627. doi:10.1097/nen.0b013e318093ece3.

[222] D. Wiesner, J. Sinniger, A. Henriques, S. Dieterlé, H.P. Müller, V. Rasche, B. Ferger, S. Dirrig-


The OCR has been converted into Markdown format as requested.
Grosch, R. Soylu-Kucharz, A. Petersén, P. Walther, B. Linkus, J. Kassubek, P.C. Wong, A.C. Ludolph, L. Dupuis, Low dietary protein content alleviates motor symptoms in mice with mutant dynactin/dynein-mediated neurodegeneration, Hum. Mol. Genet. 24 (2015) 2228–2240. doi:10.1093/hmg/ddu741.

[223] B.C. Gibbon, L.E. Zonia, D.R. Kovar, P.J. Hussey, C.J. Staiger, Pollen profilin function depends on interaction with proline-rich motifs, Plant Cell. 10 (1998) 981–993. doi:10.1105/tpc.10.6.981.

[224] W. Witke, The role of profilin complexes in cell motility and other cellular processes, Trends Cell Biol. 14 (2004) 461–469. doi:10.1016/j.tcb.2004.07.003.

[225] Y. Chen, Z.-Z. Zheng, R. Huang, K. Chen, W. Song, B. Zhao, X. Chen, Y. Yang, L. Yuan, H.-F. Shang, PFN1 mutations are rare in Han Chinese populations with amyotrophic lateral sclerosis, Neurobiol. Aging. 34 (2013) 1922.e1–1922.e5. doi:10.1016/j.neurobiolaging.2013.01.013.

[226] C. Ingre, J.E. Landers, N. Rizik, A.E. Volk, C. Akimoto, A. Birve, A. Hübers, P.J. Keagle, K. Piotrowska, R. Press, P.M. Andersen, A.C. Ludolph, J.H. Weishaupt, A novel phosphorylation site mutation in profilin 1 revealed in a large screen of US, Nordic, and German amyotrophic lateral sclerosis/frontotemporal dementia cohorts, Neurobiol. Aging. 34 (2013) 1708.e1–1708.e6. doi:10.1016/j.neurobiolaging.2012.10.009.

[227] B.N. Smith, C. Vance, E.L. Scotter, C. Troakes, C.H. Wong, S. Topp, S. Maekawa, A. King, J.C. Mitchell, K. Lund, A. Al-Chalabi, N. Ticozzi, V. Silani, P. Sapp, R.H. Brown, J.E. Landers, S. Al-Sarraji, C.E. Shaw, Novel mutations support a role for Profilin 1 in the pathogenesis of ALS, Neurobiol. Aging. 36 (2015) 1602.e17–e27. doi:10.1016/j.neurobiolaging.2014.10.032.

[228] H. Daoud, S. Dobrzeniecka, W. Camu, V. Meininger, N. Dupré, P.A. Dion, G.A. Rouleau, Mutation analysis of PFN1 in familial amyotrophic lateral sclerosis patients, Neurobiol. Aging. 34 (2013) 1311.e1–1311.e2. doi:10.1016/j.neurobiolaging.2012.09.001.

[229] Y. Tanaka, T. Nonaka, G. Suzuki, F. Kametani, M. Hasegawa, Gain-of-function profilin 1 mutations linked to familial amyotrophic lateral sclerosis cause seed-dependent intracellular TDP-43 aggregation, Hum. Mol. Genet. 25 (2016) 1420–1433. doi:10.1093/hmg/ddw024.

[230] L. Lim, J. Kang, J. Song, ALS-causing profilin-1-mutant forms a non-native helical structure in membrane environments, Biochim. Biophys. Acta - Biomembr. 1859 (2017) 2161–2170. doi:10.1016/j.bbamem.2017.08.013.

[231] C. Yang, E.W. Danielson, T. Qiao, J. Metterville, R.H. Brown, J.E. Landers, Z. Xu, Mutant PFN1 causes ALS phenotypes and progressive motor neuron degeneration in mice by a gain of toxicity, Proc. Natl. Acad. Sci. 113 (2016) E6209–E6218. doi:10.1073/pnas.1605964113.

[232] D. Sbrissa, O.C. Ikonomov, Z. Fu, T. Ijuin, J. Gruenberg, T. Takenawa, A. Shisheva, Core protein machinery for mammalian phosphatidylinositol 3, 5-bisphosphate synthesis and turnover that regulates the progression of endosomal transport: Novel Sac phosphatase joins the ArPIKfyve-PIKfyve complex, J. Biol. Chem. 282 (2007) 23878–23891. doi:10.1074/jbc.M611678200.

[233] A.C. Rutherford, The mammalian phosphatidylinositol 3-phosphate 5-kinase (PIKfyve) regulates endosome-to-TGN retrograde transport, J. Cell Sci. 119 (2006) 3944–3957. doi:10.1242/jcs.03153.

[234] A. Osmanovic, I. Rangnau, A. Kosfeld, S. Abdulla, C. Janssen, B. Auber, P. Raab, M. Preller, S. Petri, R.G. Weber, FIG 4 variants in central European patients with amyotrophic lateral sclerosis: A whole-exome and targeted sequencing study, Eur. J. Hum. Genet. 25 (2017) 324–331. doi:10.1038/ejhg.2016.186.

[235] C.Y. Chow, Y. Zhang, J.J. Dowling, N. Jin, M. Adamska, K. Shiga, K. Szigeti, M.E. Shy, J. Li, X. Zhang, J.R. Lupski, L.S. Weisman, M.H. Meisler, Mutation of FIG4 causes neurodegeneration in the pale tremor mouse and patients with CMT4J, Nature. 448 (2007) 68–72. doi:10.1038/nature05876.

[236] R. Bharadwaj, K.M. Cunningham, K. Zhang, T.E. Lloyd, FIG4 regulates lysosome membrane homeostasis independent of phosphatase function, Hum. Mol. Genet. 25 (2016) 681–692. doi:10.1093/hmg/ddv505.

[237] A. Nishiyama, Brd4 Is Required for Recovery from Antimicrotubule Drug-induced Mitotic Arrest: Preservation of Acetylated Chromatin, Mol. Biol. Cell. 17 (2005) 814–823. doi:10.1091/mbc.E05-08-0729.

[238] S.A. Rudge, D.M. Anderson, S.D. Emr, Vacuole size control: regulation of PtdIns(3,5)P2 levels by the vacuole-associated Vac14-Fig4 complex, a PtdIns(3,5)P2-specific phosphatase., Mol. Biol. Cell. 15 (2004) 24–36. doi:10.1091/mbc.E03-05-0297.

[239] C.J. Ferguson, G.M. Lenk, M.H. Meisler, Defective autophagy in neurons and astrocytes from mice deficient in PI(3,5)P2, Hum. Mol. Genet. 18 (2009) 4868–4878. doi:10.1093/hmg/ddp460.

[240] J.A. Cassel, A.B. Reitz, Ubiquilin-2 (UBQLN2) binds with high affinity to the C-terminal region of TDP-43 and modulates TDP-43 levels in H4 cells: Characterization of inhibition by nucleic acids and 4-aminoquinolines, Biochim. Biophys. Acta - Proteins Proteomics. 1834 (2013) 964–971. doi:10.1016/j.bbapap.2013.03.020.

Figure Legend

Figure 1: Schematic representation of ALS-related genes and their modulatory effects on autophagy.

The spectrum of ALS-associated genes and their various modulatory effects upon autophagy entailing the stages of autophagy initiation, autophagosome maturation, and autophagosome-lysosome/endosome fusion are depicted herein. Proteins prone to aggregation are highlighted in yellow, those involved in autophagic initiation and autophagosome formation in red, those involved in endosome trafficking and the autophagosome-lysosome fusion in green, and those with less defined roles in autophagy are signified in white.

Fig-1

![Diagram](attachment:diagram.png)

55

Table 1: Summary of ALS genes involved in autophagy impairment

| Gene   | Role in Autophagy                                                                                                                                                                                                                                      | References         |
|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|
| SOD1   | Mutants induce autophagy through undefined mechanisms and interfere with autophagosome maturation by disrupting the microtubule-localized dynein-dynactin complex integral to retrograde transport.                                                                                     | [38–40, 44, 47, 49] |
| TDP-43 | Gains toxicity via the formation of proteinaceous inclusions, known substrates of autophagy. Loss of nuclear functions caused by chronic sequestration in RNP granules results in reduced expression of autophagic factors that function at various events within the autophagy-lysosomal pathway. | [88–90, 97–99]      |
| FUS    | Gains toxicity via formation of proteinaceous inclusions, known substrates of autophagy. Possible direct disruption of autophagy regulation.                                                                                                                | [83, 90, 93]        |
| P62    | Functions as an autophagy receptor. ALS-linked mutations may impede its association with ubiquitinated substrates or block its association with autophagic membranes via LC3-II.                                                                           | [32, 104]           |
| OPTN   | Autophagy adaptor that binds ubiquitinated cargo via UBA and LC3viaLIR; also regulates autophagosome formation and maturation.                                                                                                                              | [124–127]           |
| TBK1   | Phosphorylates the UBA domain of autophagy adaptors p62 and OPTN; ALS-associated mutants induce dysfunctions in C-terminal CCD2 and serine/threonine kinase domains that may inhibit p62/OPTN activation; also involved in autophagy initiation via phosphorylation of the C9orf72 binding partner SMCR8; phosphorylation of microtubule-binding protein CEP170 underscores involvement in autophagosome maturation. | [132, 134, 140–143] |
| VAPB   | ER resident that tethers with mitochondrial protein PTPIP51, regulating autophagosome formation; tightening of the tether represses autophagosome formation, while loosening stimulates formation; modulatory effects in autophagic induction vary depending on autophagic stimuli (chemical vs. starvation-induced); ALS-associated mutations induce oligomerization and aggregation of VAPB that overburdens ERAD, hastening ALS pathology. | [89, 149–152]       |
| VCP    | Precise roles are uncertain, but speculated to diversely affect autophagy; ALS-associated mutations reduce mTOR activity, aberrantly induce autophagy; mutations also impede autophagosome-autolysosome maturation, and potentially impair autophagic clearance following autophagosome-lysosome fusion; mutants repress TDP-43-containing RNP complex disassembly. | [156, 157, 163]      |
| C9orf72| A DENN domain protein, suggested to possess multifunctional roles in autophagy initiation, nutrient sensing, and lysosomal pathways.                                                                                                                     | [137–140, 186–189]  |

UBQLN 2 Recruits autophagosomes to polyubiquitinated aggregates via UBA domain interactions; mutants may hinder the endosomal pathway [128,190,191,240]

ALS 2 Functions as GEF of Rab 5, mediating endosome trafficking and fusion. Loss-of-function mutations lead to impeded maturation and clearance of autophagic vesicles. [13,119,205]

CHMP2B Carries out ESCRT-III dissociation from endosomal membranes, forming MVBs that transition to amphisomes, which fuse with lysosomes; toxic gain-of-function ALS-FTD-associated mutations are C-terminal truncations that disrupt late-stage endosome-lysosome fusion. [212-214]

DCTN Involved in retrograde transport of autophagic substrates; ALS-associated mutations exhibit impaired intracellular trafficking, immature autophagosome accumulation, and hastened MN degeneration. [218-221]

PFN 1 Regulates actin dynamics, membrane trafficking, and synaptic structure; ALS-associated gain-of-function mutants may disrupt TDP-43 binding partner interactions, inducing deleterious conformational changes. [229-231]

FIG 4 A required phosphatase for IP(3, 5) P2 biosynthesis. Loss of function mutations result in a reduction in IP(3, 5) P2 level, subsequently leading to enlarged endosomes and disrupted autolysosome clearance. [235,236,239]
