
Review

# Autophagy in Age-Associated Neurodegeneration

Athanassios Metaxakis ${}^{1,\dagger}$, Christina Ploumi ${}^{1,2,\dagger}$ and Nektarios Tavernarakis ${}^{1,2,*}$

${}^{1}$ Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas, Heraklion 70013, Crete, Greece; thanos_metaxakis@imbb.forth.gr (A.M.); ploumi@imbb.forth.gr (C.P.)

${}^{2}$ Department of Basic Sciences, Faculty of Medicine, University of Crete, Heraklion 70013, Crete, Greece

* Correspondence: tavernarakis@imbb.forth.gr; Tel.: +3028-1039-1069

† These authors have contributed equally to this work.

Received: 2 April 2018; Accepted: 3 May 2018; Published: 5 May 2018

---

**Abstract:** The elimination of abnormal and dysfunctional cellular constituents is an essential prerequisite for nerve cells to maintain their homeostasis and proper function. This is mainly achieved through autophagy, a process that eliminates abnormal and dysfunctional cellular components, including misfolded proteins and damaged organelles. Several studies suggest that age-related decline of autophagy impedes neuronal homeostasis and, subsequently, leads to the progression of neurodegenerative disorders due to the accumulation of toxic protein aggregates in neurons. Here, we discuss the involvement of autophagy perturbation in neurodegeneration and present evidence indicating that upregulation of autophagy holds potential for the development of therapeutic interventions towards confronting neurodegenerative diseases in humans.

**Keywords:** ageing; Alzheimer's disease; amyotrophic lateral sclerosis; autophagy; Huntington's disease; mitophagy; neurodegeneration; Parkinson's sisease; protein aggregation; treatment

---

## 1. Introduction

Autophagy is an evolutionarily conserved physiological process that facilitates decomposition of unnecessary or dysfunctional cytoplasmic components and organelles in the lysosome. Depending on the substrate subjected to selective degradation, autophagy can, among other types, be categorized in ER-phagy (endoplasmic reticulum), mitophagy (mitochondria), ribophagy (ribosomes), lysophagy (lysosomes) and pexophagy (peroxisomes). Three main types of autophagy have been identified: macroautophagy, microautophagy and chaperone-mediated autophagy. Macroautophagy, commonly referred to as autophagy, requires the formation of an autophagosome, a double-membraned vesicle, in which proteins mainly, but also other macromolecules and organelles, are trapped prior to their degradation in the lysosomes. Under normal conditions, autophagy is active at the basal level but under various stresses, such as starvation, accumulation of misfolded proteins or protein aggregates, organelles’ damage and infections, it is rapidly stimulated [1].

Autophagy is a major contributor to intracellular homeostasis, since it selectively degrades cellular components that are unnecessary or even deleterious for cell survival. In starved organisms, autophagy prolongs survival through providing them with necessary energy and dietary components via recycling intracellular proteins and organelles. Several genetic studies, from yeasts to mammals, have identified more than 30 evolutionarily conserved autophagy-related genes (ATG), necessary for autophagy [2,3]. These genes are sequentially activated and facilitate the initiation, formation and elongation of the autophagosome, as well as its fusion with lysosome, for the generation of the autolysosome. Eventually, its contents are degraded and recycled. In general, autophagy has an essential role in the elimination of unnecessary, defective or aggregated proteins and damaged organelles, as well as in differentiation, cognition and development [4–7]. Hence, autophagy is a well-orchestrated, evolutionarily conserved
mechanism through which organisms maintain intracellular homeostasis and especially proteostasis, and survive under adverse conditions [8].

Supportive of its role in homeostasis maintenance, a growing body of evidence from both human pathology studies and experiments with animal model systems has revealed a beneficial role of autophagy in organismal health. Several pathologies, including neurodegeneration, cancer, stroke and metabolic syndromes are associated with defects in the autophagic machinery [9–11]. Importantly, autophagy has been recognized as a major mechanism affecting healthspan and lifespan in animal model systems [12,13]. Proteostasis impairment through ageing, together with a simultaneous age-related decline in activity of ATGs, implicate autophagy and proteostasis in age-associated deterioration and disease [14,15].

Several reports find a strong association between ageing and downregulation or deregulation of autophagic activity [16,17]. Interestingly, autophagy is shown to be mainly regulated through metabolic pathways and food consumption that are shown to affect ageing and age-related diseases. Insulin/insulin-like growth factor-1 (IGF-1) signaling (IIS), the mammalian target of the rapamycin pathway (mTOR pathway), dietary restriction (DR) and starvation modulate both autophagy and longevity [18,19]. Importantly, genetic or pharmacological interventions that extend lifespan through a reduction of IIS, inhibition of mTOR, or a deacetylation of histones and DR also increase autophagy, while several studies suggest that their anti-ageing effects require autophagy. Moreover, several transcription factors involved in lifespan extension also contribute to the upregulation of ATG genes [20,21]. The above findings implicate autophagic impairment in the progression of age-related pathologies.

Loss of proteostasis is suggested to underlie several age-related neurodegenerative diseases, such as Alzheimer’s disease (AD) and Parkinson’s disease (PD), in which the gradual accumulation of protein aggregates causes neuronal impairment and death. The ratio of autophagosome formation to autophagic degradation controls autophagic flux, the deterioration of which causes neuronal cell death [22,23]. Neurons are post-mitotic cells, more easily affected by proteostasis impairment, since damaged proteins and organelles are not diminished by mitosis. Autophagy is important for the elimination of aggregated proteins and dysfunctional organelles. Hence, autophagy is a major proteostatic mechanism that could protect nerve cells from proteotoxicity. Indeed, autophagy impairment leads to neurodegenerative disorders [24,25] and the expression of several essential genes for autophagy are shown to be reduced in the brain through ageing, such as the Atg5, Atg7 and Beclin 1 genes [26]. Several studies show that autophagy decline in aged organisms might lead to neurodegeneration through impaired proteostasis and suggest genetic or pharmacological stimulation of autophagic components as putative therapies. However, overactivated autophagy is associated with exaggerated degradation of cellular components and neuronal cell death [27–29].

Here, we discuss the role of macroautophagy (herein referred to as autophagy) on the development of age-related neurodegeneration. We will present findings from research in animal model systems and humans supporting the assertion that dysfunctional autophagy and impaired proteostasis underlie, at least in part, the progression of the major neurodegenerative disorders. We also show evidence supporting the anti-ageing role of functional autophagy and we discuss the possible use of drugs enhancing autophagy for the treatment of age-related neurodegenerative diseases in humans.

## 2. General Autophagic Mechanisms

Autophagy is a tightly-regulated process, which can lead either to the bulk degradation of intracellular material in response to increased energy demands, or to the selective elimination of specific cytoplasmic components in response to damage or stress. Nutrient deprivation and/or other stress stimuli, such as the accumulation of protein aggregates, commence a signaling cascade, which results in the formation of the isolation membrane, known as the phagophore, and its enclosure in the final autophagosome form [30] (Figure 1). Transcription factor EB (TFEB) is considered to be the master regulator of autophagy, since it regulates the expression of genes required for autophagosomal and
lysosomal biogenesis [31]. Extensive studies in yeast have identified ATG proteins as the main factors that orchestrate the initiation and progression of autophagy. Most of these proteins are conserved in mammalian systems [32].

![Diagram](attachment:diagram.png)

Autophagy induction | Phagophore elongation & sequestration of cytoplasmic cargoes | Autophagosome maturation | Retrograde transport & autolysosome formation | Lysosomal digestion

Figure 1. General autophagic mechanisms. Under nutrient-rich conditions, the autophagic pathway remains inactive due to mTOR-dependent phosphorylation of ULK1. Nutrient deprivation or stress activate AMPK, which promotes autophagy initiation, through mTOR inhibition and concurrent ULK1 activation. Once active, the ULK1 complex is translocated to the pre-autophagosomal membrane structures (mainly derived by the endoplasmic reticulum (ER) or endosomal membranes), to initiate the formation of the phagophore. ULK1- and AMPK-dependent phosphorylation of Beclin 1 disrupt its interaction with anti-autophagic factors (like Bcl-2) and enhance the recruitment of the VPS34 complex. Subsequently, the active VPS34 complex acts as a PI3K to generate PI3P. The resulted PI3P interacts with WIPI proteins, which in turn promote the recruitment of autophagy-related gene (ATG) proteins, required for phagophore nucleation. The critical step for autophagosome formation is the conjugation of LC3/ATG8 to PE lipids, a reaction that requires the co-ordinated function of ATG7, ATG3 and ATG5-12-16 (acting as E1, E2 and E3 enzymes respectively). In parallel, ATG9 promotes autophagosome formation, as it provides lipids to the expanding phagophore. During expansion, specific adaptors sequester cytoplasmic material (like protein aggregates or damaged mitochondria) in the phagophore, by directly interacting with LC3-II. Scission events mediate the closure of the phagophore and the formation of the double-bilayered autophagosome. The newly-formed autophagosomes enter the microtubule-dependent retrograde transport to attach lysosomes. Autophagosome-lysosome attachment is aided by multiple proteins, including soluble NSF attachment protein receptors (SNAREs) and the scaffolding homotypic fusion and protein sorting (HOPS) complex. Following attachment, the outer autophagosomal membrane fuses with the single lysosomal membrane, resulting in the autolysosome. Efficient acidification (mediated by v-ATPase) activates the lysosomal hydrolases, which finally digest the sequestered cytoplasmic material.

Mammalian ULK1/2 (homologous to yeast Atg1 and C. elegans UNC-51) kinases pre-exist in the cytosol as a complex with ATG13 (homologous to yeast Atg13), focal adhesion kinase family interacting protein of 200kD (FIP200) and ATG101, and remain inactivated due to mTOR-dependent inhibition [33–35]. Upon starvation, the ULK1/2 complex is activated in an adenosine monophosphate-activated protein kinase (AMPK)-dependent manner and is specifically accumulated to endoplasmic reticulum (ER) membranes, possibly aiding in the formation of the pre-autophagosomal structure (PAS) [35]. In turn, the active ULK1/2 complex is crucial for the

recruitment and activation of VPS34, the vacuolar protein sorting 34 homolog, which, in complex with VPS15, ATG14L and Beclin 1, acts as a class III phosphoinositide 3-kinase (PI3K) to generate phosphatidylinositol 3-phosphate (PI3P). Particularly, ULK1 phosphorylates Beclin 1 at Ser14, and this phosphorylation is enhanced by ATG14L and ultraviolet (UV) radiation resistance-associated gene protein (UVRAG), promoting either autophagy initiation or autophagy maturation respectively [36]. In addition to ULK1, AMPK mediates its pro-autophagic effects through Beclin 1 phosphorylation at T388 [37]. Both Beclin 1 and ATGL14L are required for the efficient recruitment of VPS34 to the PAS. WD-repeat protein interacting with phosphoinositides (WIPI) proteins detect the newly synthesized PI3P and function as a platform for the assembly of downstream ATG proteins, responsible for the elongation of the phagophore [38–40].

Two ubiquitin-like conjugation events are required for phagophore expansion. Firstly, the E1-like ATG7 and the E2-like ATG10 mediate the conjugation of ATG12 to ATG5. The resulted complex is then associated with ATG16L, which is needed for the translocation of the entire complex to the phagophore. After ATG4-mediated cleavage, LC3/ATG8 is converted to LC3-I, which is finally conjugated to the lipid phosphatidylethanolamine (PE), giving LC3-II, in a process involving ATG7, ATG3 and ATG12-ATG5-ATG16L (acting as E1, E2 and E3 respectively) [41,42]. ATG9 is an important regulator of autophagosome maturation, as it provides lipid components to the expanding phagophore [43,44]. During elongation, adaptor proteins such as P62/SQSTM1 (phosphotyrosine-independent ligand for the Lck SH2 domain of 62 KDa/sequestosome), OPTN1 (optineurin), NBR1 (neighbour of BRCA1 Gene 1) and NDP52 (nuclear dot protein 52 kDa) mediate the selective sequestration of ubiquitinated cytoplasmic contents (including organelles and aggregated proteins), by directly interacting with LC3-II [45,46]. After substrate sequestration, the phagophore finally closes in a process involving membrane scission [47]. Persistence of unclosed autophagic structures in Atg2-deficient cells supports the assertion that ATG2 is required for phagophore closure [48].

Once sealed, the autophagosomes are associated with microtubules and transported to the perinuclear region, where the lysosomes have already accumulated, in response to stress [49]. Subsequently, the autophagosomes locate in close proximity with lysosomes in a process involving the coordinated function of tethering factors, the most important of which are the SNAREs (soluble NSF attachment protein receptors) and the HOPS (homotypic fusion and protein sorting) complex [50,51]. Alternatively, the newly-generated autophagosomes can fuse with late endosomes to form amphisomes, before being engaged to lysosomes [52,53]. Notably, ATG4 seems to participate in the removal of LC3-II from the outer autophagosomal membrane, prior to lysosome fusion [54]. Following that, the outer autophagosomal membrane is fused with the lysosome’s single membrane to generate the autolysosome, in which the inner autophagosomal membrane and the engulfed material are broken down by lysosomal acidic hydrolases. Activation of the proton pump vacuolar-type H⁺ ATPase (vATPase) is a critical independent step for lysosomal digestion [55].

## 3. Autophagy and Age-Related Neurodegeneration

Ageing is the greatest risk factor for the most prevalent neurodegenerative diseases, such as AD and PD. With an ever-increasing human population and life-expectancy, modern societies are urgently seeking treatments for neurodegenerative diseases, while the number of people affected by neurodegeneration will increase in the near future. For example, the prevalence of AD is estimated to be duplicated within the next 20 years [56]. For this reason, understanding the mechanisms underlying age-related neurodegeneration is of great importance.

Impaired proteostasis is a common feature of most neurodegenerative diseases. Intriguingly, several studies show an association between neurodegeneration and alterations in the autophagic machinery. For instance, neurons of patients with AD [57,58] and PD [59] have a profound accumulation of autophagosomes. Interestingly, accumulated autophagosomes co-exist with aggregated proteins. This indicates that autophagosome formation and autophagic degradation are imbalanced, as well as the inability of autophagic machinery to eliminate aggregates in a

coordinated way. Aggregated proteins can cause synaptic impairment, damage to organelles, and neuronal cell death. Other studies show that reduction of lysosomal acidification or decreased activity of lysosomal hydrolases is inhibitory for autophagic degradation and causes neurodegenerative diseases [6]. Also, inhibition of the microtubule-mediated autophagosomal and endosomal transport blocks the autophagosome–lysosome fusion that is associated with neurodegenerative diseases [60–62]. Indeed, recent studies directly link mutations in ATG genes with the development of neurodegenerative diseases such as AD, PD and amyotrophic lateral sclerosis (ALS) [16]. In addition, deletion of the essential autophagic genes ATG7 [25] and ATG5 [24] can sufficiently induce the formation of cytoplasmic inclusions and neurodegeneration in neurons.

In conclusion, experimental evidence indicates that autophagy impairment might underlie the progression of at least some neurodegenerative diseases. Here we will present findings from research in animal model systems supporting the association between autophagy function and progression of the most prevalent age-related neurodegenerative diseases affecting humans, PD, AD, Huntington’s disease (HD) and ALS.

### 3.1. Alzheimer's Disease (AD)

AD is the most frequent type of progressive dementia diagnosed in elderly people worldwide. AD-affected brains are characterized by degeneration and eventual loss of neurons of the cerebral cortex and certain sub-cortical regions. The neuropathology of AD has been linked to the aberrant extracellular deposition of amyloid beta (Aβ) plaques and the intracellular formation of hyperphosphorylated tau-comprised neurofibrillary tangles (NFTs) [63].

Aβ peptide results from sequential proteolytic cleavages of the amyloid precursor protein (APP), an integral membrane protein mainly concentrated in synaptic regions. Human APP can be processed either by α-secretase, which releases the soluble APPα fragment, or by a β-site amyloid precursor protein-cleaving enzyme (BACE1), which releases the sAPPβ fragment. Cleavage by α-secretase is non-amyloidogenic, since it occurs inside the Aβ domain of APP. On the contrary, the β-secretase-processed membrane-associated APP fragment can subsequently be cleaved by γ-secretase, leading to the release of the pathogenic, aggregation-prone Aβ peptide [64–66]. Stress-induced expression of BACE1 has been suggested to be the initial step for AD pathogenesis [67,68]. Thus, age-related organismal changes could contribute to altered BACE1 expression and subsequent Aβ accumulation in the aged brain.

Tau is a microtubule-associated protein, ubiquitously expressed in the central nervous system. It has a crucial regulatory role in microtubule assembly and stabilization. In AD-affected brains, and other related tauopathies, like frontotemporal dementias (FTDs), tau is hyperphosphorylated and it self-aggregates, forming the intracellular NFTs [69,70]. Inability of the hyperphosphorylated tau to associate with tubulin results in the gradual disruption of microtubule network [71].

Electron microscopy studies have confirmed the presence of increased autophagosomes in post mortem human brain tissues of AD patients [57,58]. The increased number of autophagosomes is not correlated with induced autophagy, as previously thought, but rather reflects blockage in autophagic flux [72]. Aβ peptide can be generated intracellularly inside autophagosomes, in which APP, BACE1 and γ-secretase reside. The Aβ-loading of autophagosomes could lead either to the extracellular secretion of Aβ or to its lysosomal degradation. Studies have shown that autophagy is required for efficient Aβ secretion, since loss of the Atg7 gene in APP transgenic mice results in reduced extracellular Aβ plaques and increased Aβ accumulation [73,74]. In healthy neurons, the newly formed autophagosomes follow the retrograde transport system along the axon to finally reach the soma, where the lysosomes are primarily located. Conversely, in AD pathology, the axonal vesicle transport is blocked, as shown in AD transgenic mice. In particular, Aβ oligomers interfere directly with amphisome transport by disrupting the dynein-Snapin complex [75]. Hyperphosphorylated tau seems to be a contributing factor for defective retrograde transport [76].

In addition to the impairment in autophagosome transport, defects in lysosomal function have also been implicated in AD pathogenesis. Interestingly, presenilin 1 (PS1), one of the four factors that constitute the γ-secretase complex, is required for efficient lysosomal acidification. In particular, PS1 regulates the maturation and transport of v-ATPase, which is essential for the acidification of the newly synthesized autolysosomes. Deficits in lysosomal acidification lead to the accumulation of Aβ-loaded autophagosomes [77]. The fact that mutations in PS1 have been associated with the onset of familial AD has raised the intriguing question as to how loss-of-function mutations in a protein required for Aβ generation can cause AD. Several studies have tried to give an explanation, providing contradictory results [78–81].

Overexpression of distinct Aβ peptides (Aβ42 and Aβ40) in *Drosophila* neurons uncovered an age-dependent neurodegeneration mechanism which likely involves disintegration of autolysosomal membranes, rather than defects in autophagosome maturation (transport and fusion). The authors suggested that the particular peptides can induce neurodegeneration due to leakage of lysosomal proteins from post-lysosomal vesicles [82].

Several studies have also implicated Beclin 1 deficiency in the pathophysiology of AD [83–85]. Specifically, neuronal expression levels of Beclin 1 drop physiologically during ageing but also in AD-affected brains [26,84,85]. Induced Caspase-3 activity in AD brain tissue is suggested to mediate the cleavage and the subsequent depletion of Beclin 1 [86]. A recent study showed that transgenic mice with a specific mutation in Beclin 1 (F121A) display constitutively induced autophagy in the brain (among other tissues) and significantly reduced Aβ accumulation. This mutation disrupts the interaction of Beclin 1 with its inhibitory partner, B-cell lymphoma 2 (Bcl-2) [87]. In addition to Beclin 1, several core autophagic genes in *Drosophila* (*Atg1*, *Atg8a* and *Atg18*), have been found to be downregulated in an age-dependent manner, leading to induced Aβ accumulation [88]. Nuclear receptor binding factor 2 (NRBF2), an important member of the PI3K complex, has been also shown to be significantly downregulated in the hippocampus of AD mouse models. In the particular study, they showed that NRBF2 positively regulates autophagy and is specifically involved in the autophagy-mediated degradation of APP C-terminal fragments, thus eliminating the generation of Aβ oligomers [89].

mTOR signaling is an additional autophagy-related pathway involved in AD pathology. Rat ovary cells, stably expressing a familial AD mutation, display increased mTOR signaling, as reflected by increased phosphorylation of its downstream targets, p70S6 kinase (p70S6K) and eukaryotic initiation factor 4E-binding protein 1 (4E-BP1) [90]. Subsequent studies have provided evidence that mTOR is also involved in tau-mediated pathogenesis in AD and related tauopathies. Increased mTOR activity induces tau expression and promotes its glycogen synthase kinase-3 (GSK3β)-dependent phosphorylation [91]. Additionally, mTOR promotes transportation of tau to exocytic vesicles, facilitating its secretion in the extracellular space, as shown in AD-affected brains and in human neuroblastoma cells [92].

### 3.2. Parkinson's Disease (PD)

PD is the second-most prevalent late-onset neurodegenerative disorder after AD [93]. Symptomatology includes tremors, muscular rigidity, akinesia, bradykinesia and postural instability [94]. These symptoms are caused by insufficient dopamine production in substantia nigra, due to the selective death of dopaminergic neurons [94–96]. Another characteristic of PD is the accumulation of ubiquitinated α-synuclein-containing inclusions within dopaminergic neurons. These inclusions are called Lewy bodies [96]. Accumulation of α-synuclein in the Lewy bodies is caused by impaired axonal transport [97].

There are two types of PD, the sporadic and the familial types, accounting for up to 90% and 10% of incidents, respectively [98]. In both cases, specific genetic mutations are involved in the disease progression. These genes are largely associated with the functionality of intracellular transport pathways that end up in the lysosome, hence implicating a putative involvement of autophagy in
the aetiology of the disease [99]. Indeed, several findings suggest a blockade of autophagy in PD. Post mortem examination in brain samples from patients with sporadic PD show accumulation of autophagosomes, accompanied with loss of lysosomal markers in dopaminergic neurons [100,101]. Also, TFEB, an essential inducer of autophagy and lysosomal biogenesis [162], is absent from nuclei of dopaminergic neurons of PD patients, where it exerts its action [63]. These findings strongly implicate reduced autophagy in the aetiology of PD.

Dysfunctional mitophagy has been suggested to be a major cause of PD. Mitophagy is an essential mitochondrial quality-control mechanism and selectively eliminates damaged mitochondria. As such, loss of function mutations in two genes that are essential for mitophagy, the phosphatase and tensin homolog (PTEN)-induced putative kinase 1 (*PINK1*) and *Parkin*, cause sporadic juvenile-onset and autosomal recessive PD, respectively [103–105]. Mutations in these genes are the leading cause of parkinsonism [106]. Specifically, PINK1 is a kinase with a mitochondrial targeting sequence and a serine/threonine kinase domain and can be processed by presenilin-associated rhomboid-like (PARL), a mitochondrial protease, under normal conditions [107,108]. PARL-mediated processing of PINK1 leads to its degradation by the ubiquitin-proteasome system [109]. Under adverse conditions, such as mitochondrial depolarization, the processing of PINK1 by PARL is inhibited, thus causing its accumulation on the outer mitochondrial membrane. Subsequently, PINK1 autophosphorylates and recruits Parkin to damaged mitochondria [110]. Parkin is an E3 ubiquitin ligase which becomes phosphorylated and activated, together with ubiquitin [111–113], by PINK1. Although Parkin is not essential for mitophagy initiation, it enhances mitophagy. PINK1 phosphorylates ubiquitin which ubiquitinates mitochondrial surface proteins, thus leading to the formation of the phagophore around mitochondria and their targeting for lysosomal degradation. Mutations in the PINK1- and Parkin-expressing genes abrogate the above procedure and lead to mitophagy impairment that characterizes patients with PD. In support of this, the activity of essential autophagy genes are associated with the development of PD-like phenotypes in both flies and mice [114].

### 3.3. Huntington's Disease (HD)

Another age-related neurodegenerative disease is HD. It is an autosomal-dominant neurodegenerative disease, characterized by the improper folding and neuronal aggregation of huntingtin (HTT) protein. Mutant HTT is generated by the cytosine-adenine-guanine (CAG) expansion, that encodes a polyglutamine (polyQ) at the N-terminus of HTT. Aggregation of HTT leads to neuronal death, causing movement disabilities, and mental and psychiatric pathologies [115–117]. Several mechanisms are implicated in HD-related neurodegeneration [118,119], due to the multifunctional role of HTT and its involvement in multiple cellular pathways [120]. Its structure is similar to three different autophagic proteins in yeast: Atg23, Vac8, and Atg11 [121,122] and it is essential for neuronal survival [123,124]. HTT has a regulatory role in autophagy through its interaction with ULK1 and p62 [125]. It induces autophagy by releasing ULK1 from mTOR, which inhibits the activity of the ULK1/2 complex. It also binds p62 and inhibits it from linking ubiquitinated substrates and LC3, thus leading to autophagy blockage.

In support of a role of HTT on autophagic mechanism, loss of polyQ in HTT stimulates neuronal autophagy [126], while recent data indicate that HTT enhances autophagy through acting as a scaffolding protein [121], thus facilitating substrate recognition through physical interaction with autophagic components [125]. Such a role of HTT is supported by experiments with cell lines, mouse models and cells derived from patients with HD, showing that expression of mutant HTT causes defects in cargo loading into autophagosomes [127]. Interestingly, the clearance of mutant HTT through pharmacological [128–130] and genetic [131] interventions attenuates neurodegeneration [128,130,131] and improves performance [128–130] in mouse models of HD. In conclusion, mutant HTT impairs autophagic function, while its autophagy-mediated clearance relieves neurons from toxicity.
3.4. Amyotrophic Lateral Sclerosis (ALS)

ALS is a fatal neurodegenerative disorder, characterized by the progressive loss of brain and spinal-cord motor neurons, and the subsequent atrophy of the voluntary muscles controlled by them. Although it was initially regarded as a juvenile disease, ALS seems to be age-related, since the incidence of disease is exaggerated with age [132, 133]. The majority of ALS incidents are sporadic (approximately 90%), while only 10% are linked to genetic inheritance [134]. The exact molecular mechanisms leading to ALS pathology remain largely enigmatic. Despite the common neuromuscular defects of ALS patients, multiple factors that may contribute to ALS progression have been identified. The persistence of insoluble protein aggregates in the degenerating motor neurons reflects probable deficiency in the protein quality-control mechanisms, as evident in common neurodegenerative diseases [135, 136].

Numerous studies have suggested the potential role of autophagy in ALS development [137–140]. Neuron-specific loss of Atg5 and Atg7 causes motor defects in mice, in the absence of any disease-related background, suggesting that impaired autophagosome formation may be the main contributing factor for the generation of toxic protein aggregates in ALS, among other neurodegenerative diseases [24, 25]. Interestingly, mRNA and protein levels of both TFEB and Beclin 1 were found significantly reduced in transgenic mice expressing a mutant form of human superoxide dismutase 1, commonly aggregated in familial ALS (fALS) cases [141]. Mutated forms of superoxide dismutase 1 (SOD1) interact with Beclin 1 and inhibit its pro-autophagic activity in ALS mouse models. However, the same study provided contradictory evidence for a negative role of Beclin 1 in ALS, since Beclin 1 haploinsufficiency protects against the neurotoxic effects in SOD1-mutated ALS mice [142].

Another ALS-related factor implicated in autophagy regulation is the transactive DNA-binding protein 43 (TDP-43), the aggregated form of which has been observed in inclusions of almost all familial and sporadic ALS (sALS) cases. Sequestration of TDP-43 in cytoplasmic inclusions has been suggested to inhibit its main role as a regulator of mRNA stability. Atg7 mRNA is one of the TDP-43 targets found to be destabilized in mouse neuroblastoma cells subjected to TDP-43 knockdown [143]. Additionally, a recent study showed that loss of TDP-43 induces TFEB nuclear translocation and at the same time disrupts the autolysosome formation in human neuroblastoma cells [144]. Although contradictory, these observations strongly suggest that TDP-43 has a physiological role in autophagy regulation, and that its aberrant function may contribute to the autophagy-dependent degeneration of motor neurons.

An hexanucleotide expansion in the intronic region of the chromosome 9 open reading frame 72 (C9orf72) is the most common cause of ALS. The particular repeat-expansion has been suggested to perturb C9orf72 function [145]. C9orf72 seems to be an important autophagy regulator, as it has been reported that it directly interacts with the small GTPases Rab7 and Rab11, which assist in endo-lysosomal trafficking [146]. Importantly, recent studies indicated that C9orf72 acts as a guanine nucleotide exchange factor (GEF) for the activation of its interacting GTPases [147]. In addition to endolysosomal trafficking, C9orf72 has also been involved in the regulation of autophagy initiation, by facilitating the interaction between the ULK1 complex and Rab1a, an event required for the translocation of the ULK1 complex to the phagophore [148]. The particular interaction with Rab1a is independent of C9orf72’s GEF activity [148, 149].

Mutations in the selective autophagy receptors p62 and optineurin (OPTN) have been also linked with rare ALS cases [150, 151]. Studies in motor neurons of ALS mouse models have shown that p62 can bind the mutant form of SOD1 in an ubiquitin-independent manner, leading to the sequestration of SOD1 aggregates in autophagosomes and their subsequent lysosomal degradation [152]. In addition to SOD1, p62 seems to be also involved in the autophagic elimination of TDP-43 aggregates [153]. Recent studies suggested that the L341V p62 missense mutation, identified in late-onset sALS patients, renders p62 unable to recognize and interact with LC3-II [154]. Interestingly loss of p62 in zebrafish led to motor neuron defects, similar to ALS symptoms [155]. Apart from p62, mutations in the ubiquitin-binding domain of OPTN were shown to inhibit autophagosome maturation, thus decreasing the elimination of HTT-positive inclusion bodies, as shown in transfected cell lines [156]. There have been characterized OPTN mutations which impair either its ubiquitin-binding activity

(E478G) or its interaction with LC3-II (F178A) [157]. ALS-associated mutations in OPTN and TBK1 (TANK-binding kinase), its activating kinase, are suggested to cause mitochondrial dysfunction, by inhibiting the efficient sequestration of damaged mitochondria to autophagosomes [158, 159]. Mutations in Valosin-containing protein (VCP) have also been identified in fALS cases [160]. VCP, alternatively known as p97, belongs to the AAA ATPase family and has been suggested to have a role in the selective degradation of ubiquitinated proteins through autophagy [161].

There have also been reported some ALS cases in which autophagosome trafficking is impaired. Specifically, mutations in dynein and dynactin strongly inhibit the transport of the autophagic vesicles in neuronal bodies of motor neurons, as shown in mice [162–165]. However, the ALS-related phenotypes observed in these cases could be a side effect, since dynein and dynactin regulate the transport of many different neuronal cargoes (including synaptic vesicles). In this regard, a recent study indicated that dynein is required for the formation and maintenance of neuromuscular junctions, since it mediates the transport of acetylcholine receptors and muscle-specific tyrosine kinase (MusK) [166]. Dynein has also been found to be sequestered in SOD1-positive aggregates formed in spinal cord and sciatic nerve motor neurons of ALS mouse models. Interestingly, mutant SOD1 aggregates hijack the axonal transport machinery to be transported to the soma and mediate their neurotoxic effects [167].

The involvement of the chaperone system in macroautophagy also seems to be important for the clearance of pathogenic aggregates in ALS. Increasing evidence supports the assertion that the co-chaperone Bcl-2-associated athanogene 3 (BAG3) in a complex with heatshock protein 70 (HSP70) and Heat Shock Protein Family B (Small) Member 8 (HSPB8) participates in the transfer of mutant SOD1 peptides to aggresomes, thus enhancing their autophagic removal [168–170]. Interestingly, BAG3 is associated with p62/SQSTM1, which mediates the interaction with the LC3-II-marked phagophores [168, 171]. Given that p62 is mutated in several ALS cases, as previously mentioned, the BAG3-HSP70-HSPB8 chaperone complex may be unable to stimulate the autophagic sequestration of pathogenic aggregates like those of mutant SOD1. Remarkably, the same chaperone complex has been also involved in the selective autophagic removal of mutant HTT, but as shown, its contribution is not sufficient [172, 173].

## 4. Autophagy and Ageing

Although there is no clear evidence regarding the mechanistic link between ageing and neurodegenerative diseases, studies suggest that mitochondrial DNA mutations and oxidative stress are both causative agents for ageing and neurodegeneration [174, 175]. Ageing is associated with proteostasis decline, reduced nutrient sensing, organelle and mitochondrial dysfunction, cellular senescence, and stem-cell exhaustion [176]. The main cellular mechanism maintaining proteostasis and mitochondrial dysfunction is autophagy. For this, it has been suggested that a decline in basal autophagy might underlie, at least in part, ageing [8]. Supportive evidence includes the downregulation of Atg5, Atg7, and Beclin 1 autophagy genes that has been observed in aged human brains [26], as also the increased mTOR activity, accompanied by a decline in ATG protein levels in brains of aged rodents [177, 178]. Such findings suggest decreased autophagic function in aged brains. In support, human skin fibroblasts of healthy individuals show age-related mitophagy decline [179, 180], as also mitochondrial biogenesis reduction [181–183]. Mitochondrial DNA (mtDNA) content is reduced in the frontal cortex in aged rats, while mtDNA deletion content is increased [184]. The above, together with other findings [185], suggest that mitochondrial biogenesis and mitophagy during ageing are impaired.

Several findings indicate that functional autophagy is necessary for healthspan and enhanced longevity. Combined genetic, pharmacological and longevity studies in animal models associate impaired autophagy with reduced healthspan and lifespan [186]. In yeast, a short lifespan is associated with autophagy defects and autophagy mutants do not respond to lifespan extension under dietary restriction [94, 187]. In *Caenorhabditis elegans*, mutations in essential ATGs cause lifespan shortening [95], while DAF-16/FOXO Controlled, germline Tumour affecting 1 (DCT-1) is important
for mitophagy and longevity maintenance under stress \[188\]. In *Drosophila melanogaster*, lowered expression of ATG1 protein kinase and sestrin1, a negative feedback regulator of mTOR, shortens lifespan and causes mitochondrial dysfunction \[189\]. Also, rapamycin (mTOR pathway inhibitor) extends lifespan through autophagy in flies \[190\]. In mice, loss of essential ATGs lead to the accretion of inclusion bodies containing ubiquitinated proteins, lipofuscin-containing lysosomes and disorganized mitochondria \[191\].

Autophagy is not only a prerequisite for healthspan and enhanced longevity, but its enhancement can sufficiently extend lifespan. In flies, neuronal upregulation of Parkin increases lifespan, reduces levels of protein aggregation during ageing and modulates mitochondrial activity \[114,192\]. Several more examples show that autophagy induction through pharmacological treatment can promote longevity. Pharmacological perturbation of mTOR (autophagy inhibitor) through rapamycin treatment increases lifespan in all organisms tested \[13\]. Caloric restriction (CR), another intervention that affects metabolism and extends lifespan, prevents the downregulation of several autophagy effectors in aged rodents \[178\]. Also, drugs that alter the acetylation state of proteins are shown to affect autophagy and longevity. As such, both resveratrol and spermidine enhance autophagy by inhibiting proteins’ acetylation. The natural polyphenol resveratrol is an autophagy inducer that promotes lifespan. It is a deacetylase activator with its main target being *Sirtuin 1* (SIRT1), the mammalian ortholog of yeast *Sir2*. SIRT1 is a lifespan-extending NAD(+)–dependent protein deacetylase \[193\]. Regarding the mechanism of autophagy activation, resveratrol has been suggested to induce autophagy by decreasing protein acetylation \[194\], but evidence suggests that its action might be mediated through Death-associated protein kinase 1 (DAPK1) \[195\] and through mTOR inhibition \[196\]. Another drug that inhibits protein acetylation, the polyamine spermidine, extends the lifespan of yeast, worms and flies through increased autophagy \[197\]. Interestingly, spermidine enhances healthspan and lifespan by inhibiting EP300 (E1A-binding protein p300) acetyltransferase \[198\].

In conclusion, functional autophagy is a prerequisite for healthspan and enhanced lifespan in experimental animals, with existing evidence suggesting that functional autophagy has anti-ageing effects on health for its role on proteostasis maintenance. Given that the major risk for the development of age-related neurodegeneration in humans is ageing, but also that aggregate-prone proteins are used as autophagy substrates, such as tau \[199\], α-synuclein \[200\], mutant HTT \[128\], and mutant ataxin 3 \[199\], the usage of autophagy-enhancing drugs could be effective for treatment of neurodegenerative disorders in aged individuals.

## 5. Autophagy as a Therapeutic Target for Neurodegenerative Disorders

Autophagy upregulation is shown to have a neuroprotective and anti-ageing role in animal models. Its usage as a putative treatment of neurodegeneration, even if it cannot counteract the aetiology of each disease, could potentially improve the symptomatology of neurodegeneration, since it is known to promote cell survival, improve mitochondrial function and oxidative stress resistance. Interestingly, it has been reported for certain neurodegenerative disorders that abnormal proteins impede autophagy. Hence, general autophagy upregulation might improve pathophysiology \[201–203\].

Several drugs known to enhance autophagy are shown to decrease the accumulation of protein aggregates and improve neurodegenerative pathologies. In AD mouse models, long-term rapamycin treatment ameliorates symptoms, reduces Aβ42 (a main component of the amyloid plaques found in AD-affected brains), and rescues cognitive deficits \[60,90\]. On the other hand, autophagy induction after the formation of mature plaques and tangles was not effective on AD-like pathology or cognitive deficits \[204\] and rapamycin treatment of flies expressing Aβ1-42 was found to shorten lifespan \[82\]. Hence, it appears that the benefit of autophagy induction is context-dependent \[205\].

Nevertheless, rapamycin reduces huntingtin accumulation and cell death in HD cell models \[128,206\]. In addition, it has been shown to reduce aberrant protein aggregation in two models of ALS \[207\]. Despite the beneficial role of rapamycin on autophagy induction, it is also an immunesuppressor and its chronic administration may cause unwanted side effects. For this, rapamycin analogues have been tested as alternatives to rapamycin. Carbamazepine (CBZ), an anti-epileptic drug that enhances autophagy by decreasing mTOR activity, has a similar role to rapamycin in transgenic mice models of AD. After three months of treatment with CBZ, cerebral amyloid plaque burden and Aβ42 levels were reduced, autophagy was enhanced, and AD pathologies were alleviated [208]. Another rapamycin analogue, temsirolimus, shows effects similar to rapamycin on AD models [209]. Finally, GTM-1, a novel mTOR-independent autophagy inducer, has been shown to ameliorate Aβ pathology in mice [210]. The small-molecule enhancer of rapamycin 28 (Smer28) has been also found to enhance the clearance of APP C-terminal fragments through stimulation of the Atg5-dependent autophagic degradation in cell lines and in rat primary neuronal cultures [211].

A promising drug for PD treatment is resveratrol, which induces autophagy through the AMPK/SIRT1 pathway, and has neuroprotective effects on a rodent model of PD [212]. A similar role has been observed for spermidine, another autophagy inducer [198]. Resveratrol is protective against dopamine toxicity for cells expressing mutant HTT. The relative mechanism is suggested to involve the rescue of ATG4-mediated autophagosome formation [213]. Another drug that is shown to affect autophagy through AMPK induction, Nilotinib, is shown to reverse loss of dopamine neurons and improve motor behaviour via autophagic degradation of α-synuclein in PD models [214,215]. Notably, it is also implicated in Aβ clearance [216]. Metformin is an additional AMPK-dependent agent that improves neurodegeneration in AD and HD mouse models [217,218]. The use of Berberine seems to be promising, since studies have reported its beneficial effects on AD, PD and HD mouse models [219–221]. Although its mechanism of action is not fully understood, it has been suggested that Berberine may promote autophagy through AMPK activation [222].

Trehalose, a natural disaccharide, is shown to reduce both accumulation of Aβ [223] and α-synuclein in vitro [224]. Moreover, it rescues dopaminergic activity in PD mouse models and delays pathology in HD mouse models [130,223,224]. Additionally, it has been shown to prolong lifespan in ALS mouse models [225]. Lithium is another promising drug for treatment of neurodegeneration, since it has been shown to facilitate the clearance of mutant α-synuclein in vitro [226]. Lithium treatment also suppresses tau protein in mice via GSK3β inhibition and reduces mutant huntingtin protein aggregates and cell death [227]. Moreover, lithium treatment improved neuronal survival and enhanced lifespan in an ALS model [228], but the beneficial effects seem to be sex-specific [229]. Finally, lithium also decreases the levels of inositol 3-phosphates (IP3), which inhibits autophagy [230]. Another inositol-lowering agent, Rilmenidine, is also shown to remove mutant huntingtin in cells [129,231,232]. Furthermore, mTOR-independent autophagy activators, such as calpastatin (calpain inhibitor), improve the removal of huntingtin and lower its toxicity in cellular HD models [233]. In this pathway, cAMP regulates IP3 levels, which in turn inhibit autophagy.

In an attempt to increase autophagosome formation directly, intracerebral delivery of Beclin 1 in PD mouse models ameliorates the neurodegenerative effects [200]. This highlights the importance of using Beclin 1 mimetics or other autophagy inducers for the treatment of neurodegenerative diseases. Interestingly, the use of cystatin B inhibitors is promising, since it ameliorates cognitive decline in AD mouse models [234]. Cystatin B inhibits lysosomal activity by inhibiting cathepsins’ (lysosomal hydrolases) function. So, cystatin B inhibitors enhance lysosomal activity and thus promote the effective clearance of autophagosomes. The aforementioned drugs and their targets are shown in Figure 2.
Cells 2018, 7, 37

Glucose
transporter

Growth factors

Ca²⁺

Trehalose
(AD, PD, HD)

• Lithium
(AD, ALS, PD)
GSK-3β

↓ATP

• Nilotinib (AD, PD)
• Metformin (AD, HD)
• Berberin
(AD, PD, HD)
AMPK

↑Ca²⁺

• Resveratrol (PD, HD)
• Spermidine (PD, AD)

mTOR

Calpain

cAMP

• Rapamycin
(AD, HD, ALS)
• Carbamazepine (AD)
• Temsirolimus (AD)
• Smer28 (AD)

ULK1

• Calpastatin (HD)

Autophagosome formation

Lysosomal digestion

TFEB
Epigenetic
changes (EP300)
Autophagosome
& Lysosome-
related genes

Nucleus

• Beclin 1 mimetics
(PD)
Beclin 1

Cathespins

• Spermidine
(PD, AD)
Caspase-3

• Cystatin B inhibitors (AD)

Figure 2. Putative pharmacological targets for autophagy enhancement. Selective pharmacological agents used to upregulate autophagy in model systems of neurodegenerative diseases.

## 6. Conclusions

Autophagy is an evolutionarily conserved mechanism that maintains intracellular homeostasis and promotes survival under adverse conditions. Impaired autophagy leads to the accumulation of intracellular toxic aggregates, harms healthspan, and decreases longevity. Recent evidence from research on humans but also on different animal model systems reveals a prominent anti-ageing role for autophagy. Specifically, autophagy is indispensable for the clearance of malfunctioning organelles, such as mitochondria, and aberrant protein aggregates that accumulate through ageing.

Several studies support the assertion that, at least in part, ageing is accelerated by the age-related autophagy decline that has been observed in several species, including humans. Also, functional autophagy is a prerequisite for lifespan extension achieved through different experimental interventions. Hence, autophagy decline might have a causal link to physiological deterioration observed in aged animals. Indeed, several studies suggest that functional autophagy delays ageing through maintenance of proteostasis.

Loss of proteostasis underlies the most prevalent neurodegenerative diseases in humans, such as AD and PD. Overall, increasing evidence indicates that deregulated autophagy plays a key role in the progression of neurodegeneration. Several studies on animal models of neurodegeneration reveal enhanced autophagy can decrease or even ameliorate intracellular accumulation of toxic aggregates and relative pathologies. These observations imply potential therapeutic strategies to prevent or treat neurodegenerative diseases in humans through pharmacological enhancement of autophagy. Regarding autophagy-enhancing agents, treatment dose and duration should be carefully chosen, as over-activation of autophagy could result in detrimental effects. Nevertheless, the existence of a large arsenal of autophagy-enhancing drugs is a valuable tool for the development of future treatments for neurodegenerative disorders.

Author Contributions: A.M. and C.P. summarized the literature, created the initial figures and wrote the paper. N.T. edited the paper, revised the figures and contributed to the writing.

Acknowledgments: A.M. is supported by a standard Marie Curie intra-European individual fellowship. C.P. is financially supported by the General Secretariat for Research and Technology (GSRT), the Hellenic Foundation for Research and Innovation (HFRI) [Scholarship code: 1324], and by grants from the European Research Council (ERC—GA695190—MANNA, ERC—GA737599—NeuronAgeScreen).

Conflicts of Interest: The authors declare no conflict of interest.

## References

1. Yorimitsu, T.; Klionsky, D.J. Autophagy: Molecular machinery for self-eating. *Cell Death Differ.* **2005**, *12* (Suppl. 2), 1542–1552. [CrossRef] [PubMed]
2. Tsukada, M.; Ohsumi, Y. Isolation and characterization of autophagy-defective mutants of saccharomyces cerevisiae. *FEBS Lett.* **1993**, *333*, 169–174. [CrossRef]
3. Harding, T.M.; Morano, K.A.; Scott, S.V.; Klionsky, D.J. Isolation and characterization of yeast mutants in the cytoplasm to vacuole protein targeting pathway. *J. Cell Biol.* **1995**, *131*, 591–602. [CrossRef] [PubMed]
4. Guan, J.L.; Simon, A.K.; Prescott, M.; Menendez, J.A.; Liu, F.; Wang, F.; Wang, C.; Wolvetang, E.; Vazquez-Martin, A.; Zhang, J. Autophagy in stem cells. *Autophagy* **2013**, *9*, 830–849. [CrossRef] [PubMed]
5. Mizushima, N.; Komatsu, M. Autophagy: Renovation of cells and tissues. *Cell* **2011**, *147*, 728–741. [CrossRef] [PubMed]
6. Shen, H.M.; Mizushima, N. At the end of the autophagic road: An emerging understanding of lysosomal functions in autophagy. *Trends Biochem. Sci.* **2014**, *39*, 61–71. [CrossRef] [PubMed]
7. Nikoletopoulou, V.; Sidiropoulou, K.; Kallergi, E.; Dalezios, Y.; Tavernarakis, N. Modulation of autophagy by bdnf underlies synaptic plasticity. *Cell Metab.* **2017**, *26*, 230–242.e5. [CrossRef] [PubMed]
8. Rubinsztein, D.C.; Marino, G.; Kroemer, G. Autophagy and aging. *Cell* **2011**, *146*, 682–695. [CrossRef] [PubMed]
9. Cuervo, A.M.; Macian, F. Autophagy and the immune function in aging. *Curr. Opin. Immunol.* **2014**, *29*, 97–104. [CrossRef] [PubMed]
10. Galluzzi, L.; Kroemer, G. Amino acid deprivation promotes intestinal homeostasis through autophagy. *Oncotarget* **2016**, *7*, 29877–29878. [CrossRef] [PubMed]
11. Wirawan, E.; Vanden Berghe, T.; Lippens, S.; Agostinis, P.; Vandenabeele, P. Autophagy: For better or for worse. *Cell Res.* **2012**, *22*, 43–61. [CrossRef] [PubMed]
12. Lionaki, E.; Markaki, M.; Tavernarakis, N. Autophagy and ageing: Insights from invertebrate model organisms. *Ageing Res. Rev.* **2013**, *12*, 413–428. [CrossRef] [PubMed]
13. Bjedov, I.; Partridge, L. A longer and healthier life with tor down-regulation: Genetics and drugs. *Biochem. Soc. Trans.* **2011**, *39*, 460–465. [CrossRef] [PubMed]
14. Cuervo, A.M. Autophagy and aging: Keeping that old broom working. *Trends Genet.* **2008**, *24*, 604–612. [CrossRef] [PubMed]
15. Koga, H.; Cuervo, A.M. Chaperone-mediated autophagy dysfunction in the pathogenesis of neurodegeneration. *Neurobiol. Dis.* **2011**, *43*, 29–37. [CrossRef] [PubMed]
16. Kiriyama, Y.; Nochi, H. The function of autophagy in neurodegenerative diseases. *Int. J. Mol. Sci.* **2015**, *16*, 26797–26812. [CrossRef] [PubMed]
17. Nakamura, S.; Yoshimori, T. Autophagy and longevity. *Mol. Cells* **2018**, *41*, 65–72. [PubMed]
18. Niccoli, T.; Partridge, L. Ageing as a risk factor for disease. *Curr. Biol.* **2012**, *22*, R74–R752. [CrossRef] [PubMed]
19. Yang, L.; Wang, H.; Liu, L.; Xie, A. The role of insulin/igf-1/pi3k/akt/gsk3beta signaling in parkinson’s disease dementia. *Front. Neurosci.* **2018**, *12*, 73. [CrossRef] [PubMed]
20. He, C.; Klionsky, D.J. Regulation mechanisms and signaling pathways of autophagy. *Annu. Rev. Genet.* **2009**, *43*, 67–93. [CrossRef] [PubMed]
21. Sun, L.; Zhao, M.; Liu, M.; Su, P.; Zhang, J.; Li, Y.; Yang, X.; Wu, Z. Suppression of foxo3a attenuates neurobehavioral deficits after traumatic brain injury through inhibiting neuronal autophagy. *Behav. Brain Res.* **2018**, *337*, 271–279. [CrossRef] [PubMed]
22. Chu, C.T. Autophagic stress in neuronal injury and disease. *J. Neuropathol. Exp. Neurol.* **2006**, *65*, 423–432. [CrossRef] [PubMed]

23. Zhang, X.J.; Chen, S.; Huang, K.X.; Le, W.D. Why should autophagic flux be assessed? *Acta Pharmacol. Sin.* 2013, 34, 595–599. [CrossRef](https://doi.org/10.1038/aps.2013.46) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23752428)

24. Hara, T.; Nakamura, K.; Matsui, M.; Yamamoto, A.; Nakahara, Y.; Suzuki-Migishima, R.; Yokoyama, M.; Mishima, K.; Saito, I.; Okano, H.; et al. Suppression of basal autophagy in neural cells causes neurodegenerative disease in mice. *Nature* 2006, 441, 885–889. [CrossRef](https://doi.org/10.1038/nature04723) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16625183)

25. Komatsu, M.; Waguri, S.; Chiba, T.; Murata, S.; Iwata, J.; Tanida, I.; Ueno, T.; Koike, M.; Uchiyama, Y.; Kominami, E.; et al. Loss of autophagy in the central nervous system causes neurodegeneration in mice. *Nature* 2006, 441, 880–884. [CrossRef](https://doi.org/10.1038/nature04723) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16625182)

26. Lipinski, M.M.; Zheng, B.; Lu, T.; Yan, Z.; Py, B.F.; Ng, A.; Xavier, R.J.; Li, C.; Yankner, B.A.; Scherzer, C.R.; et al. Genome-wide analysis reveals mechanisms modulating autophagy in normal brain aging and in Alzheimer’s disease. *Proc. Natl. Acad. Sci. USA* 2010, 107, 14164–14169. [CrossRef](https://doi.org/10.1073/pnas.1009470107) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20660704)

27. Button, R.W.; Luo, S.; Rubinsztein, D.C. Autophagic activity in neuronal cell death. *Neurosci. Bull.* 2015, 31, 382–394. [CrossRef](https://doi.org/10.1007/s12264-015-1505-6) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25877398)

28. Kroemer, G.; Levine, B. Autophagic cell death: The story of a misnomer. *Nat. Rev. Mol. Cell Biol.* 2008, 9, 1004–1010. [CrossRef](https://doi.org/10.1038/nrm2529) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794877)

29. Lee, J.A.; Gao, F.B. Inhibition of autophagy induction delays neuronal cell loss caused by dysfunctional escrt-iii in frontotemporal dementia. *J. Neurosci.* 2009, 29, 8506–8511. [CrossRef](https://doi.org/10.1523/JNEUROSCI.0928-09.2009) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19553457)

30. Kroemer, G.; Marino, G.; Levine, B. Autophagy and the integrated stress response. *Mol. Cell* 2010, 40, 280–293. [CrossRef](https://doi.org/10.1016/j.molcel.2010.10.003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21055989)

31. Napolitano, G.; Ballabio, A. Tfeb at a glance. *J. Cell Sci.* 2016, 129, 2475–2481. [CrossRef](https://doi.org/10.1242/jcs.183703) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27381596)

32. Mizushima, N.; Yoshimori, T.; Ohsumi, Y. The role of atg proteins in autophagosome formation. *Annu. Rev. Cell Dev. Biol.* 2011, 27, 107–132. [CrossRef](https://doi.org/10.1146/annurev-cellbio-092910-154005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21774830)

33. Karanasios, E.; Stapleton, E.; Manifava, M.; Kaizuka, T.; Mizushima, N.; Walker, S.A.; Ktistakis, N.T. Dynamic association of the ulk1 complex with omegasomes during autophagy induction. *J. Cell Sci.* 2013, 126, 5224–5238. [CrossRef](https://doi.org/10.1242/jcs.131056) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24009280)

34. Ganley, I.G.; Lam du, H.; Wang, J.; Ding, X.; Chen, S.; Jiang, X. Ulk1.Atg13.Fip200 complex mediates mtor signaling and is essential for autophagy. *J. Biol. Chem.* 2009, 284, 12297–12305. [CrossRef](https://doi.org/10.1074/jbc.M809585200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19240028)

35. Kim, J.; Kundu, M.; Viollet, B.; Guan, K.L. Ampk and mtor regulate autophagy through direct phosphorylation of ulk1. *Nat. Cell Biol.* 2011, 13, 132–141. [CrossRef](https://doi.org/10.1038/ncb2152) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21186357)

36. Russell, R.C.; Tian, Y.; Yuan, H.; Park, H.W.; Chang, Y.Y.; Kim, J.; Kim, H.; Neufeld, T.P.; Dillin, A.; Guan, K.L. Ulk1 induces autophagy by phosphorylating beclin-1 and activating vps34 lipid kinase. *Nat. Cell Biol.* 2013, 15, 741–750. [CrossRef](https://doi.org/10.1038/ncb2757) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23685596)

37. Zhang, D.; Wang, W.; Sun, X.; Xu, D.; Wang, C.; Zhang, Q.; Wang, H.; Luo, W.; Chen, Y.; Chen, H.; et al. Ampk regulates autophagy by phosphorylating becn1 at threonine 388. *Autophagy* 2016, 12, 1447–1459. [CrossRef](https://doi.org/10.1080/15548627.2016.1183188) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27197283)

38. Proikas-Cezanne, T.; Ruckerbauer, S.; Stierhof, Y.D.; Berg, C.; Nordheim, A. Human wipi-1 puncta-formation: A novel assay to assess mammalian autophagy. *FEBS Lett.* 2007, 581, 3396–3404. [CrossRef](https://doi.org/10.1016/j.febslet.2007.06.033) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17586542)

39. Polson, H.E.; de Lartigue, J.; Rigden, D.J.; Reedijk, M.; Urbe, S.; Clague, M.J.; Tooze, S.A. Mammalian atg18 (wipi2) localizes to omegasome-anchored phagophores and positively regulates lc3 lipidation. *Autophagy* 2010, 6, 506–522. [CrossRef](https://doi.org/10.4161/auto.6.4.11567) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20203289)

40. Bakula, D.; Muller, A.J.; Zuleger, T.; Takacs, Z.; Franz-Wachtel, M.; Thost, A.K.; Brigger, D.; Tschan, M.P.; Frickey, T.; Robenek, H.; et al. Wipi3 and wipi4 beta-propellers are scaffolds for lkb1-ampk-tsc signalling circuits in the control of autophagy. *Nat. Commun.* 2017, 8, 15637. [CrossRef](https://doi.org/10.1038/ncomms15637) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28555077)

41. Romanov, J.; Walczak, M.; Ibircu, I.; Schuchner, S.; Ogris, E.; Kraft, C.; Martens, S. Mechanism and functions of membrane binding by the atg5-atg12/atg16 complex during autophagosome formation. *EMBO J.* 2012, 31, 4304–4317. [CrossRef](https://doi.org/10.1038/emboj.2012.271) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22968376)

42. Ichimura, Y.; Kirisako, T.; Takao, T.; Satomi, Y.; Shimonishi, Y.; Ishihara, N.; Mizushima, N.; Tanida, I.; Kominami, E.; Ohsumi, M.; et al. A ubiquitin-like system mediates protein lipidation. *Nature* 2000, 408, 488–492. [PubMed](https://pubmed.ncbi.nlm.nih.gov/11100732)

43. Yamamoto, H.; Kakuta, S.; Watanabe, T.M.; Kitamura, A.; Sekito, T.; Kondo-Kakuta, C.; Ichikawa, R.; Kinjo, M.; Ohsumi, Y. Atg9 vesicles are an important membrane source during early steps of autophagosome formation. *J. Cell Biol.* 2012, 198, 219–233. [CrossRef](https://doi.org/10.1083/jcb.201204018) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22782990)

44. Zhou, C.; Ma, K.; Gao, R.; Mu, C.; Chen, L.; Liu, Q.; Luo, Q.; Feng, D.; Zhu, Y.; Chen, Q. Regulation of matg9 trafficking by src- and ulk1-mediated phosphorylation in basal and starvation-induced autophagy. *Cell Res.* 2017, 27, 184–201. [CrossRef] [PubMed]

45. Zaffagnini, G.; Martens, S. Mechanisms of selective autophagy. *J. Mol. Biol.* 2016, 428, 1714–1724. [CrossRef] [PubMed]

46. Deng, Z.; Purtell, K.; Lachance, V.; Wold, M.S.; Chen, S.; Yue, Z. Autophagy receptors and neurodegenerative diseases. *Trends Cell Biol.* 2017, 27, 491–504. [CrossRef] [PubMed]

47. Knorr, R.L.; Lipowsky, R.; Dimova, R. Autophagosome closure requires membrane scission. *Autophagy* 2015, 11, 2134–2137. [CrossRef] [PubMed]

48. Velikkakath, A.K.; Nishimura, T.; Oita, E.; Ishihara, N.; Mizushima, N. Mammalian atg2 proteins are essential for autophagosome formation and important for regulation of size and distribution of lipid droplets. *Mol. Biol. Cell* 2012, 23, 896–909. [CrossRef] [PubMed]

49. Mackeh, R.; Perdiz, D.; Lorin, S.; Codogno, P.; Pous, C. Autophagy and microtubules—New story, old players. *J. Cell Sci.* 2013, 126, 1071–1080. [CrossRef] [PubMed]

50. Wang, Y.; Li, L.; Hou, C.; Lai, Y.; Long, J.; Liu, J.; Zhong, Q.; Diao, J. Snare-mediated membrane fusion in autophagy. *Semin. Cell Dev. Biol.* 2016, 60, 97–104. [CrossRef] [PubMed]

51. Jiang, P.; Nishimura, T.; Sakamaki, Y.; Itakura, E.; Hatta, T.; Natsume, T.; Mizushima, N. The hops complex mediates autophagosome-lysosome fusion through interaction with syntaxin 17. *Mol. Biol. Cell* 2014, 25, 1327–1337. [CrossRef] [PubMed]

52. Berg, T.O.; Fengsrud, M.; Stromhaug, P.E.; Berg, T.; Seglen, P.O. Isolation and characterization of rat liver amphisomes. Evidence for fusion of autophagosomes with both early and late endosomes. *J. Biol. Chem.* 1998, 273, 21883–21892. [CrossRef] [PubMed]

53. Sanchez-Wandelmer, J.; Reggiori, F. Amphisomes: Out of the autophagosome shadow? *EMBO J.* 2013, 32, 3116–3118. [CrossRef] [PubMed]

54. Yu, Z.Q.; Ni, T.; Hong, B.; Wang, H.Y.; Jiang, F.J.; Zou, S.; Chen, Y.; Zheng, X.L.; Klionsky, D.J.; Liang, Y.; et al. Dual roles of atg8-pe deconjugation by atg4 in autophagy. *Autophagy* 2012, 8, 883–892. [CrossRef] [PubMed]

55. Mauvezin, C.; Nagy, P.; Juhasz, G.; Neufeld, T.P. Autophagosome-lysosome fusion is independent of v-atpase-mediated acidification. *Nat. Commun.* 2015, 6, 7007. [CrossRef] [PubMed]

56. Prince, M.; Bryce, R.; Albanese, E.; Wimo, A.; Ribeiro, W.; Ferri, C.P. The global prevalence of dementia: A systematic review and metaanalysis. *Alzheimers Dement.* 2013, 9, 63–75.e62. [CrossRef] [PubMed]

57. Nixon, R.A.; Wegiel, J.; Kumar, A.; Yu, W.H.; Peterhoff, C.; Cataldo, A.; Cuervo, A.M. Extensive involvement of autophagy in Alzheimer disease: An immuno-electron microscopy study. *J. Neuropathol. Exp. Neurol.* 2005, 64, 113–122. [CrossRef] [PubMed]

58. Yu, W.H.; Cuervo, A.M.; Kumar, A.; Peterhoff, C.M.; Schmidt, S.D.; Lee, J.H.; Mohan, P.S.; Mercken, M.; Farmery, M.R.; Tjernberg, L.O.; et al. Macroautophagy—a novel beta-amyloid peptide-generating pathway activated in Alzheimer’s disease. *J. Cell Biol.* 2005, 171, 87–98. [CrossRef] [PubMed]

59. Anglade, P.; Vyas, S.; Javoy-Agid, F.; Herrero, M.T.; Michel, P.P.; Marquez, J.; Mouatt-Prigent, A.; Ruberg, M.; Hirsch, E.C.; Agid, Y. Apoptosis and autophagy in nigral neurons of patients with parkinson’s disease. *Histol. Histopathol.* 1997, 12, 25–31. [PubMed]

60. Boland, B.; Kumar, A.; Lee, S.; Platt, F.M.; Wegiel, J.; Yu, W.H.; Nixon, R.A. Autophagy induction and autophagosome clearance in neurons: Relationship to autophagic pathology in Alzheimer’s disease. *J. Neurosci.* 2008, 28, 6926–6937. [CrossRef] [PubMed]

61. Webb, J.L.; Ravikumar, B.; Rubinsztein, D.C. Microtubule disruption inhibits autophagosome-lysosome fusion: Implications for studying the roles of aggresomes in polyglutamine diseases. *Int. J. Biochem. Cell Biol.* 2004, 36, 2541–2550. [CrossRef] [PubMed]

62. Nixon, R.A. Endosome function and dysfunction in Alzheimer’s disease and other neurodegenerative diseases. *Neurobiol. Aging* 2005, 26, 373–382. [CrossRef] [PubMed]

63. Wang, J.; Gu, B.J.; Masters, C.L.; Wang, Y.J. A systemic view of Alzheimer disease—Insights from amyloid-beta metabolism beyond the brain. *Nat. Rev. Neurol.* 2017, 13, 703. [CrossRef] [PubMed]

64. Chen, G.F.; Xu, T.H.; Yan, Y.; Zhou, Y.R.; Jiang, Y.; Melcher, K.; Xu, H.E. Amyloid beta: Structure, biology and structure-based therapeutic development. *Acta Pharmacol. Sin.* 2017, 38, 1205–1235. [CrossRef] [PubMed]

65. Yankner, B.A.; Lu, T. Amyloid beta-protein toxicity and the pathogenesis of Alzheimer disease. *J. Biol. Chem.* 2009, 284, 4755–4759. [CrossRef] [PubMed]

66. Bloom, G.S. Amyloid-beta and tau: The trigger and bullet in Alzheimer disease pathogenesis. *JAMA Neurol.* 2014, 71, 505–508. [CrossRef] [PubMed]

67. Vassar, R.; Kovacs, D.M.; Yan, R.; Wong, P.C. The beta-secretase enzyme bace in health and Alzheimer’s disease: Regulation, cell biology, function, and therapeutic potential. *J. Neurosci.* 2009, 29, 12787–12794. [CrossRef] [PubMed]

68. Velliquette, R.A.; O’Connor, T.; Vassar, R. Energy inhibition elevates beta-secretase levels and activity and is potentially amyloidogenic in app transgenic mice: Possible early events in Alzheimer’s disease pathogenesis. *J. Neurosci.* 2005, 25, 10874–10883. [CrossRef] [PubMed]

69. Alonso, A.C.; Li, B.; Grundke-Iqbal, I.; Iqbal, K. Mechanism of tau-induced neurodegeneration in Alzheimer disease and related tauopathies. *Curr. Alzheimer Res.* 2008, 5, 375–384. [CrossRef] [PubMed]

70. Iqbal, K.; Liu, F.; Gong, C.X.; Grundke-Iqbal, I. Tau in Alzheimer disease and related tauopathies. *Curr. Alzheimer Res.* 2010, 7, 656–664. [CrossRef] [PubMed]

71. Li, B.; Chohan, M.O.; Grundke-Iqbal, I.; Iqbal, K. Disruption of microtubule network by alzheimer abnormally hyperphosphorylated tau. *Acta Neuropathol.* 2007, 113, 501–511. [CrossRef] [PubMed]

72. Wolfe, D.M.; Lee, J.H.; Kumar, A.; Lee, S.; Orenstein, S.J.; Nixon, R.A. Autophagy failure in Alzheimer’s disease and the role of defective lysosomal acidification. *Eur. J. Neurosci.* 2013, 37, 1949–1961. [CrossRef] [PubMed]

73. Nilsson, P.; Loganathan, K.; Sekiguchi, M.; Matsuba, Y.; Hui, K.; Tsubuki, S.; Tanaka, M.; Iwata, N.; Saito, T.; Saito, T.C. Abeta secretion and plaque formation depend on autophagy. *Cell Rep.* 2013, 5, 61–69. [CrossRef] [PubMed]

74. Nilsson, P.; Sekiguchi, M.; Akagi, T.; Izumi, S.; Komori, T.; Hui, K.; Sorgjerd, K.; Tanaka, M.; Saito, T.; Iwata, N.; et al. Autophagy-related protein 7 deficiency in amyloid beta (abeta) precursor protein transgenic mice decreases abeta in the multivesicular bodies and induces abeta accumulation in the golgi. *Am. J. Pathol.* 2015, 185, 305–313. [CrossRef] [PubMed]

75. Tammineni, P.; Ye, X.; Feng, T.; Aikal, D.; Cai, Q. Impaired retrograde transport of axonal autophagosomes contributes to autophagic stress in Alzheimer’s disease neurons. *eLife* 2017, 6, e21776. [CrossRef] [PubMed]

76. Majid, T.; Ali, Y.O.; Venkitaramani, D.V.; Jang, M.K.; Lu, H.C.; Pautler, R.G. In vivo axonal transport deficits in a mouse model of fronto-temporal dementia. *NeuroImage Clin.* 2014, 4, 711–717. [CrossRef] [PubMed]

77. Lee, J.H.; Yu, W.H.; Kumar, A.; Lee, S.; Mohan, P.S.; Peterhoff, C.M.; Wolfe, D.M.; Martinez-Vicente, M.; Massey, A.C.; Sovak, G.; et al. Lysosomal proteolysis and autophagy require presenilin 1 and are disrupted by alzheimer-related ps1 mutations. *Cell* 2010, 141, 1146–1158. [CrossRef] [PubMed]

78. Kelleher, R.J., 3rd; Shen, J. Presenilin-1 mutations and Alzheimer’s disease. *Proc. Natl. Acad. Sci. USA* 2017, 114, 629–631. [CrossRef] [PubMed]

79. Veugelen, S.; Saito, T.; Saito, T.C.; Chavez-Gutierrez, L.; De Strooper, B. Familial Alzheimer’s disease mutations in presenilin generate amyloidogenic abeta peptide seeds. *Neuron* 2016, 90, 410–416. [CrossRef] [PubMed]

80. Xia, D.; Watanabe, H.; Wu, B.; Lee, S.H.; Li, Y.; Tsvetkov, E.; Bolshakov, V.Y.; Shen, J.; Kelleher, R.J., 3rd. Presenilin-1 knockin mice reveal loss-of-function mechanism for familial Alzheimer’s disease. *Neuron* 2015, 85, 967–981. [CrossRef] [PubMed]

81. Xia, D.; Kelleher, R.J., 3rd; Shen, J. Loss of abeta43 production caused by presenilin-1 mutations in the knockin mouse brain. *Neuron* 2016, 90, 417–422. [CrossRef] [PubMed]

82. Ling, D.; Song, H.J.; Garza, D.; Neufeld, T.P.; Salvaterra, P.M. Abeta42-induced neurodegeneration via an age-dependent autophagic-lysosomal injury in drosophila. *PLoS ONE* 2009, 4, e4201. [CrossRef] [PubMed]

83. Lucin, K.M.; O’Brien, C.E.; Bieri, G.; Czirr, E.; Mosher, K.I.; Abbey, R.J.; Mastroeni, D.F.; Rogers, J.; Spencer, B.; Masliah, E.; et al. Microglial beclin 1 regulates retromer trafficking and phagocytosis and is impaired in Alzheimer’s disease. *Neuron* 2013, 79, 873–886. [CrossRef] [PubMed]

84. Pickford, F.; Masliah, E.; Britschgi, M.; Lucin, K.; Narasimhan, R.; Jaeger, P.A.; Small, S.; Spencer, B.; Rockenstein, E.; Levine, B.; et al. The autophagy-related protein beclin 1 shows reduced expression in early Alzheimer disease and regulates amyloid beta accumulation in mice. *J. Clin. Investig.* 2008, 118, 2190–2199. [PubMed]

85. Jaeger, P.A.; Pickford, F.; Sun, C.H.; Lucin, K.M.; Masliah, E.; Wyss-Coray, T. Regulation of amyloid precursor protein processing by the beclin 1 complex. *PLoS ONE* **2010**, *5*, e11102. [CrossRef](https://doi.org/10.1371/journal.pone.0011102) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20689549)

86. Rohn, T.T.; Wirawan, E.; Brown, R.J.; Harris, J.R.; Masliah, E.; Vandenabeele, P. Depletion of beclin-1 due to proteolytic cleavage by caspases in the Alzheimer’s disease brain. *Neurobiol. Dis.* **2011**, *43*, 68–78. [CrossRef](https://doi.org/10.1016/j.nbd.2011.03.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21419408)

87. Rocchi, A.; Yamamoto, S.; Ting, T.; Fan, Y.; Sadleir, K.; Wang, Y.; Zhang, W.; Huang, S.; Levine, B.; Vassar, R.; et al. A *becn1* mutation mediates hyperactive autophagic sequestration of amyloid oligomers and improved cognition in Alzheimer’s disease. *PLoS Genet.* **2017**, *13*, e1006962. [CrossRef](https://doi.org/10.1371/journal.pgen.1006962) [PubMed](https://pubmed.ncbi.nlm.nih.gov/29073004)

88. Omata, Y.; Lim, Y.M.; Akao, Y.; Tsuda, L. Age-induced reduction of autophagy-related gene expression is associated with onset of Alzheimer’s disease. *Am. J. Neurodegener. Dis.* **2014**, *3*, 134–142. [PubMed](https://pubmed.ncbi.nlm.nih.gov/25364506)

89. Yang, C.; Cai, C.Z.; Song, J.X.; Tan, J.Q.; Durairajan, S.S.K.; Iyashwamy, A.; Wu, M.Y.; Chen, L.L.; Yue, Z.; Li, M.; et al. Nrbf2 is involved in the autophagic degradation process of app-ctfs in Alzheimer disease models. *Autophagy* **2017**, *13*, 2028–2040. [CrossRef](https://doi.org/10.1080/15548627.2017.1359477) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28719965)

90. Caccamo, A.; Majumder, S.; Richardson, A.; Strong, R.; Oddo, S. Molecular interplay between mammalian target of rapamycin (mTOR), amyloid-beta, and tau: Effects on cognitive impairments. *J. Biol. Chem.* **2010**, *285*, 13107–13120. [CrossRef](https://doi.org/10.1074/jbc.M110.106946) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20185686)

91. Caccamo, A.; Magri, A.; Medina, D.X.; Wisely, E.V.; Lopez-Aranda, M.F.; Silva, A.J.; Oddo, S. mTOR regulates tau phosphorylation and degradation: Implications for Alzheimer’s disease and other tauopathies. *Aging Cell* **2013**, *12*, 370–380. [CrossRef](https://doi.org/10.1111/acel.12067) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23406361)

92. Tang, Z.; Ioja, E.; Bereczki, E.; Hultenby, K.; Li, C.; Guan, Z.; Winblad, B.; Pei, J.J. mTOR mediates tau localization and secretion: Implication for Alzheimer’s disease. *Biochim. Biophys. Acta* **2015**, *1853*, 1646–1657. [CrossRef](https://doi.org/10.1016/j.bbamcr.2015.03.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25817896)

93. Elbaz, A.; Carcaillon, L.; Kab, S.; Moisan, F. Epidemiology of parkinson’s disease. *Rev. Neurol.* **2016**, *172*, 14–26. [CrossRef](https://doi.org/10.1016/j.neurol.2015.11.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26777975)

94. Jankovic, J. Parkinson’s disease: Clinical features and diagnosis. *J. Neurol. Neurosurg. Psychiatry* **2008**, *79*, 368–376. [CrossRef](https://doi.org/10.1136/jnnp.2007.131045) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18344392)

95. Brichta, L.; Greengard, P.; Flajolet, M. Advances in the pharmacological treatment of parkinson’s disease: Targeting neurotransmitter systems. *Trends Neurosci.* **2013**, *36*, 543–554. [CrossRef](https://doi.org/10.1016/j.tins.2013.07.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23916898)

96. Kalia, L.V.; Lang, A.E. Parkinson’s disease. *Lancet* **2015**, *386*, 896–912. [CrossRef](https://doi.org/10.1016/S0140-6736(14)61393-3)

97. De Vos, K.J.; Grierson, A.J.; Ackerley, S.; Miller, C.C. Role of axonal transport in neurodegenerative diseases. *Annu. Rev. Neurosci.* **2008**, *31*, 151–173. [CrossRef](https://doi.org/10.1146/annurev.neuro.31.060407.125645) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18558857)

98. Trimmer, P.A.; Bennett, J.P., Jr. The cybrid model of sporadic parkinson’s disease. *Exp. Neurol.* **2009**, *218*, 320–325. [CrossRef](https://doi.org/10.1016/j.expneurol.2009.04.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19375585)

99. Abeliovich, A.; Gitler, A.D. Defects in trafficking bridge parkinson’s disease pathology and genetics. *Nature* **2016**, *539*, 207–216. [CrossRef](https://doi.org/10.1038/nature20126) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27580033)

100. Dehay, B.; Bove, J.; Rodriguez-Muela, N.; Perier, C.; Recasens, A.; Boya, P.; Vila, M. Pathogenic lysosomal depletion in parkinson’s disease. *J. Neurosci.* **2010**, *30*, 12535–12544. [CrossRef](https://doi.org/10.1523/JNEUROSCI.2188-10.2010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20720116)

101. Chu, Y.; Dodiya, H.; Aebischer, P.; Olanow, C.W.; Kordower, J.H. Alterations in lysosomal and proteasomal markers in parkinson’s disease: Relationship to alpha-synuclein inclusions. *Neurobiol. Dis.* **2009**, *35*, 385–398. [CrossRef](https://doi.org/10.1016/j.nbd.2009.05.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19464086)

102. Settembre, C.; Di Malta, C.; Polito, V.A.; Garcia Arencibia, M.; Vetrini, F.; Erdin, S.; Erdin, S.U.; Huynh, T.; Medina, D.; Colella, P.; et al. Tfeb links autophagy to lysosomal biogenesis. *Science* **2011**, *332*, 1429–1433. [CrossRef](https://doi.org/10.1126/science.1203686) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21636772)

103. Whitworth, A.J.; Pallanck, L.J. Pink1/parkin mitophagy and neurodegeneration—what do we really know in vivo? *Curr. Opin. Genet. Dev.* **2017**, *44*, 47–53. [CrossRef](https://doi.org/10.1016/j.gde.2017.03.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28343976)

104. Valente, E.M.; Abou-Sleiman, P.M.; Caputo, V.; Muqit, M.M.; Harvey, K.; Gispert, S.; Ali, Z.; Del Turco, D.; Bentivoglio, A.R.; Healy, D.G.; et al. Hereditary early-onset parkinson’s disease caused by mutations in pink1. *Science* **2004**, *304*, 1158–1160. [CrossRef](https://doi.org/10.1126/science.1096284) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15155933)

105. Valente, E.M.; Brancati, F.; Caputo, V.; Graham, E.A.; Davis, M.B.; Ferraris, A.; Breteler, M.M.; Gasser, T.; Bonifati, V.; Bentivoglio, A.R.; et al. Park6 is a common cause of familial parkinsonism. *Neurol. Sci.* **2002**, *23* (Suppl. 2), S117–S118. [CrossRef](https://doi.org/10.1007/s100720200040) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12582622)

106. Hardy, J. Genetic analysis of pathways to parkinson disease. *Neuron* **2010**, *68*, 201–206. [CrossRef](https://doi.org/10.1016/j.neuron.2010.10.002) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20955936)

107. Jin, S.M.; Lazarou, M.; Wang, C.; Kane, L.A.; Narendra, D.P.; Youle, R.J. Mitochondrial membrane potential regulates pink1 import and proteolytic destabilization by parl. *J. Cell Biol.* 2010, **191**, 933–942. [CrossRef](https://doi.org/10.1083/jcb.201006059) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20805301)

108. Meissner, C.; Lorenz, H.; Weihofen, A.; Selkoe, D.J.; Lemberg, M.K. The mitochondrial intramembrane protease parl cleaves human pink1 to regulate pink1 trafficking. *J. Neurochem.* 2011, **117**, 856–867. [CrossRef](https://doi.org/10.1111/j.1471-4159.2011.07195.x) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21320177)

109. Yamano, K.; Youle, R.J. Pink1 is degraded through the n-end rule pathway. *Autophagy* 2013, **9**, 1758–1769. [CrossRef](https://doi.org/10.4161/auto.25887) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23884337)

110. Okatsu, K.; Oka, T.; Iguchi, M.; Imamura, K.; Kosako, H.; Tani, N.; Kimura, M.; Go, E.; Koyano, F.; Funayama, M.; et al. Pink1 autophosphorylation upon membrane potential dissipation is essential for parkin recruitment to damaged mitochondria. *Nat. Commun.* 2012, **3**, 1016. [CrossRef](https://doi.org/10.1038/ncomms2016) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22982999)

111. Kazlauskaite, A.; Kondapalli, C.; Gourlay, R.; Campbell, D.G.; Ritorto, M.S.; Hofmann, K.; Alessi, D.R.; Knebel, A.; Trost, M.; Muqit, M.M. Parkin is activated by pink1-dependent phosphorylation of ubiquitin at ser65. *Biochem. J.* 2014, **460**, 127–139. [CrossRef](https://doi.org/10.1042/BJ20140049) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24716696)

112. Koyano, F.; Okatsu, K.; Kosako, H.; Tamura, Y.; Go, E.; Kimura, M.; Kimura, Y.; Tsuchiya, H.; Yoshihara, H.; Hirokawa, T.; et al. Ubiquitin is phosphorylated by pink1 to activate parkin. *Nature* 2014, **510**, 162–166. [CrossRef](https://doi.org/10.1038/nature13175) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24828048)

113. Wauer, T.; Simicek, M.; Schubert, A.; Komander, D. Mechanism of phospho-ubiquitin-induced parkin activation. *Nature* 2015, **524**, 370–374. [CrossRef](https://doi.org/10.1038/nature14618) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26083755)

114. Knuppertz, L.; Osiewacz, H.D. Orchestrating the network of molecular pathways affecting aging: Role of nonselective autophagy and mitophagy. *Mech. Ageing Dev.* 2016, **153**, 30–40. [CrossRef](https://doi.org/10.1016/j.mad.2015.11.003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26616556)

115. Ross, C.A.; Tabrizi, S.J. Huntington’s disease: From molecular pathogenesis to clinical treatment. *Lancet Neurol.* 2011, **10**, 83–98. [CrossRef](https://doi.org/10.1016/S1474-4422(10)70245-3)

116. Dayalu, P.; Albin, R.L. Huntington disease: Pathogenesis and treatment. *Neurol. Clin.* 2015, **33**, 101–114. [CrossRef](https://doi.org/10.1016/j.ncl.2014.10.003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25439650)

117. The huntington’s disease collaborative research group. A novel gene containing a trinucleotide repeat that is expanded and unstable on huntington’s disease chromosomes. *Cell* 1993, **72**, 971–983.

118. Saudou, F.; Humbert, S. The biology of huntingtin. *Neuron* 2016, **89**, 910–926. [CrossRef](https://doi.org/10.1016/j.neuron.2016.01.013) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26868150)

119. Schwab, L.C.; Richetin, K.; Barker, R.A.; Deglon, N. Formation of hippocampal mhtt aggregates leads to impaired spatial memory, hippocampal activation and adult neurogenesis. *Neurobiol. Dis.* 2017, **102**, 105–112. [CrossRef](https://doi.org/10.1016/j.nbd.2017.03.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28314768)

120. Harjes, P.; Wanker, E.E. The hunt for huntingtin function: Interaction partners tell many different stories. *Trends Biochem. Sci.* 2003, **28**, 425–433. [CrossRef](https://doi.org/10.1016/S0968-0004(03)00140-7)

121. Ochaba, J.; Lukacsovich, T.; Csikos, G.; Zheng, S.; Margulis, J.; Salazar, L.; Mao, K.; Lau, A.L.; Yeung, S.Y.; Humbert, S.; et al. Potential function for the huntingtin protein as a scaffold for selective autophagy. *Proc. Natl. Acad. Sci. USA* 2014, **111**, 16889–16894. [CrossRef](https://doi.org/10.1073/pnas.1414914111) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25349413)

122. Steffan, J.S. Does huntingtin play a role in selective macroautophagy? *Cell Cycle* 2010, **9**, 3401–3413. [CrossRef](https://doi.org/10.4161/cc.9.17.12953) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20852406)

123. Dragatsis, I.; Levine, M.S.; Zeitlin, S. Inactivation of hdh in the brain and testis results in progressive neurodegeneration and sterility in mice. *Nat. Genet.* 2000, **26**, 300–306. [CrossRef](https://doi.org/10.1038/79944) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11017085)

124. Nasir, J.; Floresco, S.B.; O’Kusky, J.R.; Diewert, V.M.; Richman, J.M.; Zeisler, J.; Borowski, A.; Marth, J.D.; Phillips, A.G.; Hayden, M.R. Targeted disruption of the huntington’s disease gene results in embryonic lethality and behavioral and morphological changes in heterozygotes. *Cell* 1995, **81**, 811–823. [CrossRef](https://doi.org/10.1016/0092-8674(95)90544-6)

125. Rui, Y.N.; Xu, Z.; Patel, B.; Chen, Z.; Chen, D.; Tito, A.; David, G.; Sun, Y.; Stimming, E.F.; Bellen, H.J.; et al. Huntingtin functions as a scaffold for selective macroautophagy. *Nat. Cell Biol.* 2015, **17**, 262–275. [CrossRef](https://doi.org/10.1038/ncb3113) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25621688)

126. Zheng, S.; Clabough, E.B.; Sarkar, S.; Futter, M.; Rubinsztein, D.C.; Zeitlin, S.O. Deletion of the huntingtin polyglutamine stretch enhances neuronal autophagy and longevity in mice. *PLoS Genet.* 2010, **6**, e1000838. [CrossRef](https://doi.org/10.1371/journal.pgen.1000838) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20221250)

127. Martinez, J.E.; Vershinin, M.D.; Shubeita, G.T.; Gross, S.P. On the use of in vivo cargo velocity as a biophysical marker. *Biochem. Biophys. Res. Commun.* 2007, **353**, 835–840. [CrossRef](https://doi.org/10.1016/j.bbrc.2006.12.119) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17196386)

128. Ravikumar, B.; Vacher, C.; Berger, Z.; Davies, J.E.; Luo, S.; Oroz, L.G.; Scaravilli, F.; Easton, D.F.; Duden, R.; O’Kane, C.J.; et al. Inhibition of mtor induces autophagy and reduces toxicity of polyglutamine expansions in fly and mouse models of huntington disease. *Nat. Genet.* **2004**, *36*, 585–595. [CrossRef](https://doi.org/10.1038/ng1355) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15071553)

129. Rose, C.; Menzies, F.M.; Renna, M.; Acevedo-Arozena, A.; Corrochano, S.; Sadiq, O.; Brown, S.D.; Rubinsztein, D.C. Rilmenidine attenuates toxicity of polyglutamine expansions in a mouse model of huntington’s disease. *Hum. Mol. Genet.* **2010**, *19*, 2144–2153. [CrossRef](https://doi.org/10.1093/hmg/ddq098) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20215157)

130. Tanaka, M.; Machida, Y.; Niu, S.; Ikeda, T.; Jana, N.R.; Doi, H.; Kurosawa, M.; Nekooki, M.; Nukina, N. Trehalose alleviates polyglutamine-mediated pathology in a mouse model of huntington disease. *Nat. Med.* **2004**, *10*, 148–154. [CrossRef](https://doi.org/10.1038/nm980) [PubMed](https://pubmed.ncbi.nlm.nih.gov/14702087)

131. Tsunemi, T.; Ashe, T.D.; Morrison, B.E.; Soriano, K.R.; Au, J.; Roque, R.A.; Lazarowski, E.R.; Damian, V.A.; Masliah, E.; La Spada, A.R. Pgc-1alpha rescues huntington’s disease proteotoxicity by preventing oxidative stress and promoting tfeb function. *Sci. Transl. Med.* **2012**, *4*, 142ra197. [CrossRef](https://doi.org/10.1126/scitranslmed.3003852) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22764203)

132. Logroscino, G.; Traynor, B.J.; Hardiman, O.; Chio, A.; Mitchell, D.; Swingler, R.J.; Millul, A.; Benn, E.; Beghi, E. Incidence of amyotrophic lateral sclerosis in europe. *J. Neurol. Neurosurg. Psychiatry* **2010**, *81*, 385–390. [CrossRef](https://doi.org/10.1136/jnnp.2009.183076) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20080955)

133. Niccoli, T.; Partridge, L.; Isaacs, A.M. Ageing as a risk factor for als/ftd. *Hum. Mol. Genet.* **2017**, *26*, R105–R113. [CrossRef](https://doi.org/10.1093/hmg/ddx193) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28583957)

134. Ajroud-Driss, S.; Siddique, T. Sporadic and hereditary amyotrophic lateral sclerosis (als). *Biochim. Biophys. Acta* **2015**, *1852*, 679–684. [CrossRef](https://doi.org/10.1016/j.bbadis.2014.09.011) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25245848)

135. Wang, Q.; Johnson, J.L.; Agar, N.Y.; Agar, J.N. Protein aggregation and protein instability govern familial amyotrophic lateral sclerosis patient survival. *PLoS Biol.* **2008**, *6*, e170. [CrossRef](https://doi.org/10.1371/journal.pbio.0060170) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18597553)

136. Blokhuis, A.M.; Groen, E.J.; Koppers, M.; van den Berg, L.H.; Pasterkamp, R.J. Protein aggregation in amyotrophic lateral sclerosis. *Acta Neuropathol.* **2013**, *125*, 777–794. [CrossRef](https://doi.org/10.1007/s00401-013-1096-7) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23400585)

137. Tian, F.; Morimoto, N.; Liu, W.; Ohta, Y.; Deguchi, K.; Miyazaki, K.; Abe, K. In vivo optical imaging of motor neuron autophagy in a mouse model of amyotrophic lateral sclerosis. *Autophagy* **2011**, *7*, 985–992. [CrossRef](https://doi.org/10.4161/auto.7.8.15780) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21633207)

138. Sasaki, S. Autophagy in spinal cord motor neurons in sporadic amyotrophic lateral sclerosis. *J. Neuropathol. Exp. Neurol.* **2011**, *70*, 349–359. [CrossRef](https://doi.org/10.1097/NEN.0b013e318214c9f1) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21407066)

139. Chen, S.; Zhang, X.; Song, L.; Le, W. Autophagy dysregulation in amyotrophic lateral sclerosis. *Brain Pathol.* **2012**, *22*, 110–116. [CrossRef](https://doi.org/10.1111/j.1750-3639.2011.00518.x) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22004120)

140. Deng, Z.; Sheehan, P.; Chen, S.; Yue, Z. Is amyotrophic lateral sclerosis/frontotemporal dementia an autophagy disease? *Mol. Neurodegener.* **2017**, *12*, 90. [CrossRef](https://doi.org/10.1186/s13024-017-0209-2) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28780277)

141. Chen, Y.; Liu, H.; Guan, Y.; Wang, Q.; Zhou, F.; Jie, L.; Ju, J.; Pu, L.; Du, H.; Wang, X. The altered autophagy mediated by tfeb in animal and cell models of amyotrophic lateral sclerosis. *Am. J. Transl. Res.* **2015**, *7*, 1574–1587. [PubMed](https://pubmed.ncbi.nlm.nih.gov/26430422)

142. Nassif, M.; Valenzuela, V.; Rojas-Rivera, D.; Vidal, R.; Matus, S.; Castillo, K.; Fuentealba, Y.; Kroemer, G.; Levine, B.; Hetz, C. Pathogenic role of becn1/beclin 1 in the development of amyotrophic lateral sclerosis. *Autophagy* **2014**, *10*, 1256–1271. [CrossRef](https://doi.org/10.4161/auto.29057) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24867070)

143. Bose, J.K.; Huang, C.C.; Shen, C.K. Regulation of autophagy by neuropathological protein tdp-43. *J. Biol. Chem.* **2011**, *286*, 44441–44448. [CrossRef](https://doi.org/10.1074/jbc.M111.278159) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22003053)

144. Xia, Q.; Wang, H.; Hao, Z.; Fu, C.; Hu, Q.; Gao, F.; Ren, H.; Chen, D.; Han, J.; Ying, Z.; et al. Tdp-43 loss of function increases tfeb activity and blocks autophagosome-lysosome fusion. *EMBO J.* **2016**, *35*, 121–142. [CrossRef](https://doi.org/10.15252/embj.201592884) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26689987)

145. Mori, K.; Weng, S.M.; Arzberger, T.; May, S.; Rentzsch, K.; Kremmer, E.; Schmid, B.; Kretzschmar, H.A.; Cruts, M.; Van Broeckhoven, C.; et al. The c9orf72 ggggcc repeat is translated into aggregating dipeptide-repeat proteins in ftld/als. *Science* **2013**, *339*, 1335–1338. [CrossRef](https://doi.org/10.1126/science.1232927) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23416207)

146. Farg, M.A.; Sundaramoorthy, V.; Sultana, J.M.; Yang, S.; Atkinson, R.A.K.; Levina, V.; Halloran, M.A.; Gleeson, P.A.; Blair, I.P.; Soo, K.Y.; et al. C9orf72, implicated in amyotrophic lateral sclerosis and frontotemporal dementia, regulates endosomal trafficking. *Hum. Mol. Genet.* **2017**, *26*, 4093–4094. [CrossRef](https://doi.org/10.1093/hmg/ddx269) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28641886)

147. Corbier, C.; Sellier, C. C9orf72 is a gdp/gtp exchange factor for rab8 and rab39 and regulates autophagy. *Small GTPases* **2017**, *8*, 181–186. [CrossRef](https://doi.org/10.1080/21541248.2017.1321687) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28452415)

148. Webster, C.P.; Smith, E.F.; Bauer, C.S.; Moller, A.; Hautbergue, G.M.; Ferraiuolo, L.; Myszczynska, M.A.; Higginbottom, A.; Walsh, M.J.; Whitworth, A.J.; et al. The c9orf72 protein interacts with rab1a and the ulk1 complex to regulate initiation of autophagy. *EMBO J.* **2016**, *35*, 1656–1676. [CrossRef](https://doi.org/10.15252/embj.201593356) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27558864)

149. Webster, C.P.; Smith, E.F.; Grierson, A.J.; De Vos, K.J. C9orf72 plays a central role in rab gtpase-dependent regulation of autophagy. *Small GTPases* **2016**, *1–10*. [CrossRef](https://doi.org/10.1080/21541248.2016.1192827) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27411286)

150. Fecto, F.; Yan, J.; Vemula, S.P.; Liu, E.; Yang, Y.; Chen, W.; Zheng, J.G.; Shi, Y.; Siddique, N.; Arrat, H.; et al. Sqstm1 mutations in familial and sporadic amyotrophic lateral sclerosis. *Arch. Neurol.* **2011**, *68*, 1440–1446. [CrossRef](https://doi.org/10.1001/archneurol.2011.192) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21747045)

151. Maruyama, H.; Morino, H.; Ito, H.; Izumi, Y.; Kato, H.; Watanabe, Y.; Kinoshita, Y.; Kamada, M.; Nodera, H.; Suzuki, H.; et al. Mutations of optineurin in amyotrophic lateral sclerosis. *Nature* **2010**, *465*, 223–226. [CrossRef](https://doi.org/10.1038/nature09020) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20445625)

152. Gal, J.; Strom, A.L.; Kwinter, D.M.; Kilty, R.; Zhang, J.; Shi, P.; Fu, W.; Wooten, M.W.; Zhu, H. Sequestosome 1/p62 links familial als mutant sod1 to lc3 via an ubiquitin-independent mechanism. *J. Neurochem.* **2009**, *111*, 1062–1073. [CrossRef](https://doi.org/10.1111/j.1471-4159.2009.06334.x) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19735306)

153. Brady, O.A.; Meng, P.; Zheng, Y.; Mao, Y.; Hu, F. Regulation of tdp-43 aggregation by phosphorylation and p62/sqstm1. *J. Neurochem.* **2011**, *116*, 248–259. [CrossRef](https://doi.org/10.1111/j.1471-4159.2010.07100.x) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21118546)

154. Goode, A.; Butler, K.; Long, J.; Cavey, J.; Scott, D.; Shaw, B.; Sollenberger, J.; Gell, C.; Johansen, T.; Oldham, N.J.; et al. Defective recognition of lc3b by mutant sqstm1/p62 implicates impairment of autophagy as a pathogenic mechanism in als-ftld. *Autophagy* **2016**, *12*, 1094–1104. [CrossRef](https://doi.org/10.1080/15548627.2016.1158850) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26987012)

155. Lattante, S.; de Calbiac, H.; Le Ber, I.; Brice, A.; Ciura, S.; Kabashi, E. Sqstm1 knock-down causes a locomotor phenotype ameliorated by rapamycin in a zebrafish model of als/ftld. *Hum. Mol. Genet.* **2015**, *24*, 1682–1690. [CrossRef](https://doi.org/10.1093/hmg/ddv013) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25653309)

156. Shen, W.C.; Li, H.Y.; Chen, G.C.; Chern, Y.; Tu, P.H. Mutations in the ubiquitin-binding domain of optn/optineurin interfere with autophagy-mediated degradation of misfolded proteins by a dominant-negative mechanism. *Autophagy* **2015**, *11*, 685–700. [CrossRef](https://doi.org/10.4161/auto.35057) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25601983)

157. Wong, Y.C.; Holzbaur, E.L. Temporal dynamics of park2/parkin and optn/optineurin recruitment during the mitophagy of damaged mitochondria. *Autophagy* **2015**, *11*, 422–424. [CrossRef](https://doi.org/10.1080/15548627.2015.1005275) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25668313)

158. Moore, A.S.; Holzbaur, E.L. Dynamic recruitment and activation of als-associated tbkl with its target optineurin are required for efficient mitophagy. *Proc. Natl. Sci. USA* **2016**, *113*, E3349–E3358. [CrossRef](https://doi.org/10.1073/pnas.1525843113) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27122538)

159. Wong, Y.C.; Holzbaur, E.L. Optineurin is an autophagy receptor for damaged mitochondria in parkin-mediated mitophagy that is disrupted by an als-linked mutation. *Proc. Natl. Sci. USA* **2014**, *111*, E4439–E4448. [CrossRef](https://doi.org/10.1073/pnas.1410880111) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25122760)

160. Johnson, J.O.; Mandrioli, J.; Benatar, M.; Abramzon, Y.; Van Deerlin, V.M.; Trojanowski, J.Q.; Gibbs, J.R.; Brunetti, M.; Gronka, S.; Wuu, J.; et al. Exome sequencing reveals vcp mutations as a cause of familial als. *Neuron* **2010**, *68*, 857–864. [CrossRef](https://doi.org/10.1016/j.neuron.2010.11.034) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21145000)

161. Tresse, E.; Salomons, F.A.; Vesa, J.; Bott, L.C.; Kimonis, V.; Yao, T.P.; Dantuma, N.P.; Taylor, J.P. Vcp/p97 is essential for maturation of ubiquitin-containing autophagosomes and this function is impaired by mutations that cause ibmpfd. *Autophagy* **2010**, *6*, 217–227. [CrossRef](https://doi.org/10.4161/auto.6.2.11019) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20118710)

162. Hafezparast, M.; Klocke, R.; Ruhrberg, C.; Marquardt, A.; Ahmad-Annuar, A.; Bowen, S.; Lalli, G.; Witherden, A.S.; Hummerich, H.; Nicholson, S.; et al. Mutations in dynein link motor neuron degeneration to defects in retrograde transport. *Science* **2003**, *300*, 808–812. [CrossRef](https://doi.org/10.1126/science.1083160) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12679256)

163. Ilieva, H.S.; Yamanaka, K.; Malkmus, S.; Kakinohana, O.; Yaksh, T.; Marsala, M.; Cleveland, D.W. Mutant dynein (loa) triggers proprioceptive axon loss that extends survival only in the sod1 als model with highest motor neuron death. *Proc. Natl. Sci. USA* **2008**, *105*, 12599–12604. [CrossRef](https://doi.org/10.1073/pnas.0805825105) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18809922)

164. Puls, I.; Jonnakuty, C.; LaMonte, B.H.; Holzbaur, E.L.; Tokito, M.; Mann, E.; Floeter, M.K.; Bidus, K.; Drayna, D.; Oh, S.J.; et al. Mutant dynactin in motor neuron disease. *Nat. Genet.* **2003**, *33*, 455–456. [CrossRef](https://doi.org/10.1038/ng1131) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12660812)

165. LaMonte, B.H.; Wallace, K.E.; Holloway, B.A.; Shelly, S.S.; Ascano, J.; Tokito, M.; Van Winkle, T.; Howland, D.S.; Holzbaur, E.L. Disruption of dynein/dynactin inhibits axonal transport in motor neurons causing late-onset progressive degeneration. *Neuron* **2002**, *34*, 715–727. [CrossRef](https://doi.org/10.1016/S0896-6273(02)00680-8)

166. Vilmont, V.; Cadot, B.; Vezin, E.; Le Grand, F.; Gomes, E.R. Dynein disruption perturbs post-synaptic components and contributes to impaired musk clustering at the nmj: Implication in als. *Sci. Rep.* 2016, 6, 27804. [CrossRef](https://doi.org/10.1038/srep27804) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27353297)

167. Zhang, F.; Strom, A.L.; Fukada, K.; Lee, S.; Hayward, L.J.; Zhu, H. Interaction between familial amyotrophic lateral sclerosis (als)-linked sod1 mutants and the dynein complex. *J. Biol. Chem.* 2007, 282, 16691–16699. [CrossRef](https://doi.org/10.1074/jbc.M702188200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17449390)

168. Gamerding, M.; Kaya, A.M.; Wolfrum, U.; Clement, A.M.; Behl, C. Bag3 mediates chaperone-based aggresome-targeting and selective autophagy of misfolded proteins. *EMBO Rep.* 2011, 12, 149–156. [CrossRef](https://doi.org/10.1038/embor.2010.186) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21150906)

169. Crippa, V.; Sau, D.; Rusmini, P.; Boncoraglio, A.; Onesto, E.; Bolzoni, E.; Galbiati, M.; Fontana, E.; Marino, M.; Carra, S.; et al. The small heat shock protein b8 (hspb8) promotes autophagic removal of misfolded proteins involved in amyotrophic lateral sclerosis (als). *Hum. Mol. Genet.* 2010, 19, 3440–3456. [CrossRef](https://doi.org/10.1093/hmg/ddq253) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20530456)

170. Crippa, V.; Boncoraglio, A.; Galbiati, M.; Aggarwal, T.; Rusmini, P.; Giorgetti, E.; Cristofani, R.; Carra, S.; Pennuto, M.; Poletti, A. Differential autophagy power in the spinal cord and muscle of transgenic als mice. *Front. Cell. Neurosci.* 2013, 7, 234. [CrossRef](https://doi.org/10.3389/fncel.2013.00234) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24348354)

171. Gamerding, M.; Hajieva, P.; Kaya, A.M.; Wolfrum, U.; Hartl, F.U.; Behl, C. Protein quality control during aging involves recruitment of the macroautophagy pathway by bag3. *EMBO J.* 2009, 28, 889–901. [CrossRef](https://doi.org/10.1038/emboj.2009.10) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19204752)

172. Carra, S.; Seguin, S.J.; Lambert, H.; Landry, J. Hspb8 chaperone activity toward poly(q)-containing proteins depends on its association with bag3, a stimulator of macroautophagy. *J. Biol. Chem.* 2008, 283, 1437–1444. [CrossRef](https://doi.org/10.1074/jbc.M707183200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18024575)

173. Sturner, E.; Behl, C. The role of the multifunctional bag3 protein in cellular protein quality control and in disease. *Front. Mol. Neurosci.* 2017, 10, 177. [CrossRef](https://doi.org/10.3389/fnmol.2017.00177) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28978988)

174. Lin, M.T.; Beal, M.F. Mitochondrial dysfunction and oxidative stress in neurodegenerative diseases. *Nature* 2006, 443, 787–795. [CrossRef](https://doi.org/10.1038/nature05292) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17051212)

175. Rubinsztein, D.C. The roles of intracellular protein-degradation pathways in neurodegeneration. *Nature* 2006, 443, 780–786. [CrossRef](https://doi.org/10.1038/nature05291) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17051211)

176. Lopez-Otin, C.; Blasco, M.A.; Partridge, L.; Serrano, M.; Kroemer, G. The hallmarks of aging. *Cell* 2013, 153, 1194–1217. [CrossRef](https://doi.org/10.1016/j.cell.2013.05.039) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23746838)

177. Ott, C.; König, J.; Hohn, A.; Jung, T.; Grune, T. Macroautophagy is impaired in old murine brain tissue as well as in senescent human fibroblasts. *Redox Biol.* 2016, 10, 266–273. [CrossRef](https://doi.org/10.1016/j.redox.2016.07.007) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27474788)

178. Yang, F.; Chu, X.; Yin, M.; Liu, X.; Yuan, H.; Niu, Y.; Fu, L. Mtor and autophagy in normal brain aging and caloric restriction ameliorating age-related cognition deficits. *Behav. Brain Res.* 2014, 264, 82–90. [CrossRef](https://doi.org/10.1016/j.bbr.2014.02.018) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24582980)

179. Diot, A.; Hinks-Roberts, A.; Lodge, T.; Liao, C.; Dombi, E.; Morten, K.; Brady, S.; Fratter, C.; Carver, J.; Muir, R.; et al. A novel quantitative assay of mitophagy: Combining high content fluorescence microscopy and mitochondrial DNA load to quantify mitophagy and identify novel pharmacological tools against pathogenic heteroplasmic mtdna. *Pharmacol. Res.* 2015, 100, 24–35. [CrossRef](https://doi.org/10.1016/j.phrs.2015.07.001) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26188877)

180. Diot, A.; Morten, K.; Poulton, J. Mitophagy plays a central role in mitochondrial ageing. *Mamm. Genome* 2016, 27, 381–395. [CrossRef](https://doi.org/10.1007/s00335-016-9654-4) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27353778)

181. Lopez-Lluch, G.; Irusta, P.M.; Navas, P.; de Cabo, R. Mitochondrial biogenesis and healthy aging. *Exp. Gerontol.* 2008, 43, 813–819. [CrossRef](https://doi.org/10.1016/j.exger.2008.05.012) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18571898)

182. Onyango, I.G.; Lu, J.; Rodova, M.; Lezi, E.; Crafter, A.B.; Swerdlow, R.H. Regulation of neuron mitochondrial biogenesis and relevance to brain health. *Biochim. Biophys. Acta* 2010, 1802, 228–234. [CrossRef](https://doi.org/10.1016/j.bbadis.2009.09.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19766718)

183. Wenz, T. Mitochondria and pgc-1alpha in aging and age-associated diseases. *J. Aging Res.* 2011, 2011, 810619. [CrossRef](https://doi.org/10.4061/2011/810619) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21991438)

184. Picca, A.; Fracasso, F.; Pesce, V.; Cantatore, P.; Joseph, A.M.; Leeuwenburgh, C.; Gadaleta, M.N.; Lezza, A.M. Age- and calorie restriction-related changes in rat brain mitochondrial DNA and tfam binding. *Age* 2013, 35, 1607–1620. [CrossRef](https://doi.org/10.1007/s11357-012-9448-8) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22829138)

185. Grimm, A.; Eckert, A. Brain aging and neurodegeneration: From a mitochondrial point of view. *J. Neurochem.* 2017, 143, 418–431. [CrossRef](https://doi.org/10.1111/jnc.14084) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28481187)

186. Pyo, J.O.; Yoo, S.M.; Ahn, H.H.; Nah, J.; Hong, S.H.; Kam, T.I.; Jung, S.; Jung, Y.K. Overexpression of atg5 in mice activates autophagy and extends lifespan. *Nat. Commun.* 2013, 4, 2300. [CrossRef](https://doi.org/10.1038/ncomms3300) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24005320)

187. Matecic, M.; Smith, D.L.; Pan, X.; Maqani, N.; Bekiranov, S.; Boeke, J.D.; Smith, J.S. A microarray-based genetic screen for yeast chronological aging factors. *PLoS Genet.* 2010, 6, e1000921. [CrossRef](https://doi.org/10.1371/journal.pgen.1000921) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20548913)

188. Palikaras, K.; Lionaki, E.; Tavernarakis, N. Coordination of mitophagy and mitochondrial biogenesis during ageing in c. Elegans. *Nature* 2015, 521, 525–528. [CrossRef](https://doi.org/10.1038/nature14426) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26083757)

189. Lee, J.H.; Budanov, A.V.; Park, E.J.; Birse, R.; Kim, T.E.; Perkins, G.A.; Ocorr, K.; Ellisman, M.H.; Bodmer, R.; Bier, E.; et al. Sestrin as a feedback inhibitor of tor that prevents age-related pathologies. *Science* 2010, 327, 1223–1228. [CrossRef](https://doi.org/10.1126/science.1183098) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20203045)

190. Bjedov, I.; Toivonen, J.M.; Kerr, F.; Slack, C.; Jacobson, J.; Foley, A.; Partridge, L. Mechanisms of life span extension by rapamycin in the fruit fly drosophila melanogaster. *Cell Metabol.* 2010, 11, 35–46. [CrossRef](https://doi.org/10.1016/j.cmet.2009.12.009) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20102420)

191. Hartleben, B.; Godel, M.; Meyer-Schwesinger, C.; Liu, S.; Ulrich, T.; Kobler, S.; Wiech, T.; Grahammer, F.; Arnold, S.J.; Lindenmeyer, M.T.; et al. Autophagy influences glomerular disease susceptibility and maintains podocyte homeostasis in aging mice. *J. Clin. Investig.* 2010, 120, 1084–1096. [CrossRef](https://doi.org/10.1172/JCI40588) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20360970)

192. Rana, A.; Rera, M.; Walker, D.W. Parkin overexpression during aging reduces proteotoxicity, alters mitochondrial dynamics, and extends lifespan. *Proc. Natl. Acad. Sci. USA* 2013, 110, 8638–8643. [CrossRef](https://doi.org/10.1073/pnas.1303296110) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23716700)

193. Wood, J.G.; Rogina, B.; Lavu, S.; Howitz, K.; Helfand, S.L.; Tatar, M.; Sinclair, D. Sirtuin activators mimic caloric restriction and delay ageing in metazoans. *Nature* 2004, 430, 686–689. [CrossRef](https://doi.org/10.1038/nature02789) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15295556)

194. Morselli, E.; Marino, G.; Bennetzen, M.V.; Eisenberg, T.; Megalou, E.; Schroeder, S.; Cabrera, S.; Benit, P.; Rustin, P.; Criollo, A.; et al. Spermidine and resveratrol induce autophagy by distinct pathways converging on the acetylproteome. *J. Cell Biol.* 2011, 192, 615–629. [CrossRef](https://doi.org/10.1083/jcb.201006011) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21307097)

195. Choi, M.S.; Kim, Y.; Jung, J.Y.; Yang, S.H.; Lee, T.R.; Shin, D.W. Resveratrol induces autophagy through death-associated protein kinase 1 (dapk1) in human dermal fibroblasts under normal culture conditions. *Exp. Dermatol.* 2013, 22, 491–494. [CrossRef](https://doi.org/10.1111/exd.12096) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23384265)

196. Park, Y.Y.; Sohn, B.H.; Johnson, R.L.; Kang, M.H.; Kim, S.B.; Shim, J.J.; Mangala, L.S.; Kim, J.H.; Yoo, J.E.; Rodriguez-Aguayo, C.; et al. Yes-associated protein 1 and transcriptional coactivator with pdz-binding motif activate the mammalian target of rapamycin complex 1 pathway by regulating amino acid transporters in hepatocellular carcinoma. *Hepatology* 2016, 63, 159–172. [CrossRef](https://doi.org/10.1002/hep.28183) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26335973)

197. Eisenberg, T.; Knauer, H.; Schauer, A.; Buttner, S.; Ruckenstuhl, C.; Carmona-Gutierrez, D.; Ring, J.; Schroeder, S.; Magnes, C.; Antonacci, L.; et al. Induction of autophagy by spermidine promotes longevity. *Nat. Cell Biol.* 2009, 11, 1305–1314. [CrossRef](https://doi.org/10.1038/ncb1971) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19838195)

198. Pietrocola, F.; Lachkar, S.; Enot, D.P.; Niso-Santano, M.; Bravo-San Pedro, J.M.; Sica, V.; Izzo, V.; Maiuri, M.C.; Madeo, F.; Marino, G.; et al. Spermidine induces autophagy by inhibiting the acetyltransferase ep300. *Cell Death Differ.* 2015, 22, 509–516. [CrossRef](https://doi.org/10.1038/cdd.2014.175) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25410525)

199. Berger, Z.; Ravikumar, B.; Menzies, F.M.; Oroz, L.G.; Underwood, B.R.; Pangelos, M.N.; Schmitt, I.; Wullner, U.; Evert, B.O.; O’Kane, C.J.; et al. Rapamycin alleviates toxicity of different aggregate-prone proteins. *Hum. Mol. Genet.* 2006, 15, 433–442. [CrossRef](https://doi.org/10.1093/hmg/ddi454) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16396988)

200. Spencer, B.; Potkar, R.; Trejo, M.; Rockenstein, E.; Patrick, C.; Gindi, R.; Adame, A.; Wyss-Coray, T.; Masliah, E. Beclin 1 gene transfer activates autophagy and ameliorates the neurodegenerative pathology in alpha-synuclein models of parkinson’s and lewy body diseases. *J. Neurosci.* 2009, 29, 13578–13588. [CrossRef](https://doi.org/10.1523/JNEUROSCI.3984-09.2009) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846718)

201. Moors, T.E.; Hoozemans, J.J.; Ingrassia, A.; Beccari, T.; Parnetti, L.; Chartier-Harlin, M.C.; van de Berg, W.D. Therapeutic potential of autophagy-enhancing agents in parkinson’s disease. *Mol. Neurodegener.* 2017, 12, 11. [CrossRef](https://doi.org/10.1186/s13024-017-0164-7) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28232203)

202. Martin, D.D.; Ladha, S.; Ehrnhoefer, D.E.; Hayden, M.R. Autophagy in huntington disease and huntingtin in autophagy. *Trends Neurosci.* 2015, 38, 26–35. [CrossRef](https://doi.org/10.1016/j.tins.2014.11.003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25460103)

203. Rubinsztein, D.C.; Bento, C.F.; Deretic, V. Therapeutic targeting of autophagy in neurodegenerative and infectious diseases. *J. Exp. Med.* 2015, 212, 979–990. [CrossRef](https://doi.org/10.1084/jem.20150442) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25995268)

204. Majumder, S.; Richardson, A.; Strong, R.; Oddo, S. Inducing autophagy by rapamycin before, but not after, the formation of plaques and tangles ameliorates cognitive deficits. *PLoS ONE* 2011, 6, e25416. [CrossRef](https://doi.org/10.1371/journal.pone.0025416) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21966366)

205. Liang, J.H.; Jia, J.P. Dysfunctional autophagy in Alzheimer’s disease: Pathogenic roles and therapeutic implications. *Neurosci. Bull.* 2014, 30, 308–316. [CrossRef](https://doi.org/10.1007/s12264-014-1420-7) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24718552)

206. Sarkar, S.; Ravikumar, B.; Floto, R.A.; Rubinsztein, D.C. Rapamycin and mtor-independent autophagy inducers ameliorate toxicity of polyglutamine-expanded huntingtin and related proteinopathies. *Cell Death Differ.* 2009, *16*, 46–56. [CrossRef](https://doi.org/10.1038/cdd.2008.170) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19158882)

207. Wang, I.F.; Guo, B.S.; Liu, Y.C.; Wu, C.C.; Yang, C.H.; Tsai, K.J.; Shen, C.K. Autophagy activators rescue and alleviate pathogenesis of a mouse model with proteinopathies of the tar DNA-binding protein 43. *Proc. Natl. Acad. Sci. USA* 2012, *109*, 15024–15029. [CrossRef](https://doi.org/10.1073/pnas.1208857109) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23012413)

208. Li, L.; Zhang, S.; Zhang, X.; Li, T.; Tang, Y.; Liu, H.; Yang, W.; Le, W. Autophagy enhancer carbamazepine alleviates memory deficits and cerebral amyloid-beta pathology in a mouse model of Alzheimer’s disease. *Curr. Alzheimer Res.* 2013, *10*, 433–441. [CrossRef](https://doi.org/10.2174/1567205011310040004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23551501)

209. Jiang, T.; Yu, J.T.; Zhu, X.C.; Tan, M.S.; Wang, H.F.; Cao, L.; Zhang, Q.Q.; Shi, J.Q.; Gao, L.; Qin, H.; et al. Temsirolimus promotes autophagic clearance of amyloid-beta and provides protective effects in cellular and animal models of Alzheimer’s disease. *Pharmacol. Res.* 2014, *81*, 54–63. [CrossRef](https://doi.org/10.1016/j.phrs.2014.01.005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24486870)

210. Chu, C.; Zhang, X.; Ma, W.; Li, L.; Wang, W.; Shang, L.; Fu, P. Induction of autophagy by a novel small molecule improves abeta pathology and ameliorates cognitive deficits. *PLoS ONE* 2013, *8*, e65367.

211. Tian, Y.; Bustos, V.; Flajolet, M.; Greengard, P. A small-molecule enhancer of autophagy decreases levels of abeta and app-ctf via atg5-dependent autophagy pathway. *FASEB J.* 2011, *25*, 1934–1942. [CrossRef](https://doi.org/10.1096/fj.10-177577) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21454737)

212. Jin, F.; Wu, Q.; Lu, Y.F.; Gong, Q.H.; Shi, J.S. Neuroprotective effect of resveratrol on 6-ohda-induced parkinson’s disease in rats. *Eur. J. Pharmacol.* 2008, *600*, 78–82. [CrossRef](https://doi.org/10.1016/j.ejphar.2008.07.026) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18675303)

213. Vidoni, C.; Secomandi, E.; Castiglioni, A.; Melone, M.A.B.; Isidoro, C. Resveratrol protects neuronal-like cells expressing mutant huntingtin from dopamine toxicity by rescuing atg4-mediated autophagosome formation. *Neurochem. Int.* 2017. [CrossRef](https://doi.org/10.1016/j.neuint.2017.03.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28325533)

214. Hebron, M.L.; Lonskaya, I.; Moussa, C.E. Nilotinib reverses loss of dopamine neurons and improves motor behavior via autophagic degradation of alpha-synuclein in parkinson’s disease models. *Hum. Mol. Genet.* 2013, *22*, 3315–3328. [CrossRef](https://doi.org/10.1093/hmg/ddt173) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23599247)

215. Yu, H.C.; Lin, C.S.; Tai, W.T.; Liu, C.Y.; Shiau, C.W.; Chen, K.F. Nilotinib induces autophagy in hepatocellular carcinoma through ampk activation. *J. Biol. Chem.* 2013, *288*, 18249–18259. [CrossRef](https://doi.org/10.1074/jbc.M113.466987) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23599530)

216. Lonskaya, I.; Hebron, M.L.; Desforges, N.M.; Schachter, J.B.; Moussa, C.E. Nilotinib-induced autophagic changes increase endogenous parkin level and ubiquitination, leading to amyloid clearance. *J. Mol. Med.* 2014, *92*, 373–386. [CrossRef](https://doi.org/10.1007/s00109-013-1080-2) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24317788)

217. Song, R. Mechanism of metformin: A tale of two sites. *Diabetes Care* 2016, *39*, 187–189. [CrossRef](https://doi.org/10.2337/dc15-2618) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26787825)

218. Ma, T.C.; Buescher, J.L.; Oatis, B.; Funk, J.A.; Nash, A.J.; Carrier, R.L.; Hoyt, K.R. Metformin therapy in a transgenic mouse model of huntington’s disease. *Neurosci. Lett.* 2007, *411*, 98–103. [CrossRef](https://doi.org/10.1016/j.neulet.2006.10.047) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17113322)

219. Haghani, M.; Shabani, M.; Tondar, M. The therapeutic potential of berberine against the altered intrinsic properties of the cal neurons induced by abeta neurotoxicity. *Eur. J. Pharmacol.* 2015, *758*, 82–88. [CrossRef](https://doi.org/10.1016/j.ejphar.2015.04.038) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25912888)

220. Kim, M.; Cho, K.H.; Shin, M.S.; Lee, J.M.; Cho, H.S.; Kim, C.J.; Shin, D.H.; Yang, H.J. Berberine prevents nigrostriatal dopaminergic neuronal loss and suppresses hippocampal apoptosis in mice with parkinson’s disease. *Int. J. Mol. Med.* 2014, *33*, 870–878. [CrossRef](https://doi.org/10.3892/ijmm.2014.1576) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24370845)

221. Jiang, W.; Wei, W.; Gaertig, M.A.; Li, S.; Li, X.J. Therapeutic effect of berberine on huntington’s disease transgenic mouse model. *PLoS ONE* 2015, *10*, e0134142. [CrossRef](https://doi.org/10.1371/journal.pone.0134142) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26218516)

222. Yu, R.; Zhang, Z.Q.; Wang, B.; Jiang, H.X.; Cheng, L.; Shen, L.M. Berberine-induced apoptotic and autophagic death of hepg2 cells requires ampk activation. *Cancer Cell Int.* 2014, *14*, 49. [CrossRef](https://doi.org/10.1186/1475-2867-14-49) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24885630)

223. Rodriguez-Navarro, J.A.; Rodriguez, L.; Casarejos, M.J.; Solano, R.M.; Gomez, A.; Perucho, J.; Cuervo, A.M.; Garcia de Yebenes, J.; Mena, M.A. Trehalose ameliorates dopaminergic and tau pathology in parkin deleted/tau overexpressing mice through autophagy activation. *Neurobiol. Dis.* 2010, *39*, 423–438. [CrossRef](https://doi.org/10.1016/j.nbd.2010.04.021) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20434386)

224. Sarkar, S.; Davies, J.E.; Huang, Z.; Tunacliffe, A.; Rubinsztein, D.C. Trehalose, a novel mtor-independent autophagy enhancer, accelerates the clearance of mutant huntingtin and alpha-synuclein. *J. Biol. Chem.* 2007, *282*, 5641–5652. [CrossRef](https://doi.org/10.1074/jbc.M608907200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17158953)

225. Castillo, K.; Nassif, M.; Valenzuela, V.; Rojas, F.; Matus, S.; Mercado, G.; Court, F.A.; van Zundert, B.; Hetz, C. Trehalose delays the progression of amyotrophic lateral sclerosis by enhancing autophagy in motoneurons. *Autophagy* 2013, *9*, 1308–1320. [CrossRef](https://doi.org/10.4161/auto.25888) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23884337)

226. Feng, H.L.; Leng, Y.; Ma, C.H.; Zhang, J.; Ren, M.; Chuang, D.M. Combined lithium and valproate treatment delays disease onset, reduces neurological deficits and prolongs survival in an amyotrophic lateral sclerosis mouse model. *Neuroscience* **2008**, *155*, 567–572. [CrossRef](https://doi.org/10.1016/j.neuroscience.2008.04.039) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18502201)

227. Zhang, X.; Heng, X.; Li, T.; Li, L.; Yang, D.; Du, Y.; Doody, R.S.; Le, W. Long-term treatment with lithium alleviates memory deficits and reduces amyloid-beta production in an aged Alzheimer's disease transgenic mouse model. *J. Alzheimers Dis.* **2011**, *24*, 739–749. [PubMed](https://pubmed.ncbi.nlm.nih.gov/21317566)

228. Fornai, F.; Longone, P.; Cafaro, L.; Kastsiuchenka, O.; Ferrucci, M.; Manca, M.L.; Lazzeri, G.; Spalloni, A.; Bellio, N.; Lenzi, P.; et al. Lithium delays progression of amyotrophic lateral sclerosis. *Proc. Natl. Acad. Sci. USA* **2008**, *105*, 2052–2057. [CrossRef](https://doi.org/10.1073/pnas.0711170105) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18230729)

229. Pizzasegola, C.; Caron, I.; Daleno, C.; Ronchi, A.; Minoia, C.; Carri, M.T.; Bendotti, C. Treatment with lithium carbonate does not improve disease progression in two different strains of sod1 mutant mice. *Amyotroph. Lateral Scler.* **2009**, *10*, 221–228. [CrossRef](https://doi.org/10.3109/17482960802581778) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19173525)

230. Sade, Y.; Toker, L.; Kara, N.Z.; Einat, H.; Rapoport, S.; Moechars, D.; Berry, G.T.; Bersudsky, Y.; Agam, G. Ip3 accumulation and/or inositol depletion: Two downstream lithium’s effects that may mediate its behavioral and cellular changes. *Transl. Psychiatry* **2016**, *6*, e968. [CrossRef](https://doi.org/10.1038/tp.2016.209) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27905436)

231. Sarkar, S.; Floto, R.A.; Berger, Z.; Imarisio, S.; Cordenier, A.; Pasco, M.; Cook, L.J.; Rubinsztein, D.C. Lithium induces autophagy by inhibiting inositol monophosphatase. *J. Cell Biol.* **2005**, *170*, 1101–1111. [CrossRef](https://doi.org/10.1083/jcb.200504022) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16172283)

232. Cardenas, C.; Miller, R.A.; Smith, I.; Bui, T.; Molgo, J.; Muller, M.; Vais, H.; Cheung, K.H.; Yang, J.; Parker, I.; et al. Essential regulation of cell bioenergetics by constitutive insp3 receptor Ca²⁺ transfer to mitochondria. *Cell* **2010**, *142*, 270–283. [CrossRef](https://doi.org/10.1016/j.cell.2010.06.014) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20673999)

233. Williams, A.; Sarkar, S.; Cuddon, P.; Ttofi, E.K.; Saiki, S.; Siddiqi, F.H.; Jahreiss, L.; Fleming, A.; Pask, D.; Goldsmith, P.; et al. Novel targets for huntington’s disease in an mtor-independent autophagy pathway. *Nat. Chem. Biol.* **2008**, *4*, 295–305. [CrossRef](https://doi.org/10.1038/nchembio.80) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18368047)

234. Sun, B.; Zhou, Y.; Halabisky, B.; Lo, I.; Cho, S.H.; Mueller-Steiner, S.; Devidze, N.; Wang, X.; Grubb, A.; Gan, L. Cystatin c-cathepsin b axis regulates amyloid beta levels and associated neuronal deficits in an animal model of Alzheimer’s disease. *Neuron* **2008**, *60*, 247–257. [CrossRef](https://doi.org/10.1016/j.neuron.2008.08.028) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18957225)
