
Accepted Manuscript

Revised Crystal Structure of Human Adenovirus Reveals the Limits on Protein IX Quasi-Equivalence and on Analyzing Large Macromolecular Complexes

S. Kundhavai Natchiar, Sangita Venkataraman, Tina-Marie Mullen, Glen R. Nemerow, Vijay S. Reddy

PII: S0022-2836(18)30639-9  
DOI: doi:10.1016/j.jmb.2018.08.011  
Reference: YJMBI 65849  
To appear in: Journal of Molecular Biology  
Received date: 14 June 2018  
Revised date: 19 July 2018  
Accepted date: 4 August 2018  

Please cite this article as: S. Kundhavai Natchiar, Sangita Venkataraman, Tina-Marie Mullen, Glen R. Nemerow, Vijay S. Reddy, Revised Crystal Structure of Human Adenovirus Reveals the Limits on Protein IX Quasi-Equivalence and on Analyzing Large Macromolecular Complexes. Yjmbi (2018), doi:10.1016/j.jmb.2018.08.011

This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.

Revised crystal structure of human adenovirus reveals the limits on protein  
IX quasi-equivalence and on analyzing large macromolecular complexes  

S. Kundhavai Natchiar${}^{1\mathrm{~S}}$, Sangita Venkataraman${}^{1\mathrm{~Q}}$, Tina-Marie Mullen${}^{2}$, Glen R. Nemerow${}^{2*}$,  
Vijay S. Reddy${}^{1*}$.  

${}^{1}$Department of Integrative Structural and Computational Biology,  
${}^{2}$Department of Immunology and Microbial Sciences  
The Scripps Research Institute, La Jolla, CA 92037, USA  

*Corresponding authors  
gnemerow@scripps.edu  
reddyv@scripps.edu  

Current addresses:  
${}^{\mathrm{~S}}$Centre for Integrative Biology (CBI), Department of Integrated Structural Biology, IGBMC,  
CNRS, Inserm, Université de Strasbourg, 1 rue Laurent Fries, 67404 Illkirch, France.  
${}^{\mathrm{~Q}}$Department of Biochemistry, Acharya Nagarjuna University, Guntur 122413, Andhra Pradesh,  
India.  

Key words: Adenovirus; Crystal structure; Minor proteins; Quasi-equivalence; Protein IX

Abstract

We report the revised crystal structure of a pseudo-typed human adenovirus at 3.8Å resolution that is consistent with the atomic models of minor proteins determined by cryo-electron microscopy (cryo-EM). The diffraction data from multiple crystals were rescaled and merged to increase the data completeness. The densities for the minor proteins were initially identified in the phase-refined omit maps that were further improved by the phases from docked poly-alanine models to build atomic structures. While the trimeric fiber molecules are disordered due to flexibility and imposition of 5-fold symmetry, the remaining major capsid proteins, hexon and penton base (PB) are clearly ordered, with the exception of hypervariable region 1 of hexons, the RGD containing loop and the N-termini of the PB. The exterior minor protein IX together with the interior minor proteins: IIIa and VIII stabilize the adenovirus virion. A segment of pro-peptide of VI (pVIn) is found in the interior cavities of peripentonal hexons and the rest of VI is disordered. While the triskelions substructures formed by the N-termini of IX conform to excellent quasi 3-fold symmetry, the tetrameric coiled-coils formed by the C-termini and organized in parallel and anti-parallel arrangement do not exhibit any quasi-symmetry. This observation also conveys the pitfalls of using the quasi-equivalence as validation criteria for the structural analysis of extended (non-modular) capsid proteins such as IX. Together, these results remedy certain discrepancies in the previous X-ray model in agreement with the cryo-EM models.

Introduction

Human adenoviruses (HAdV) are nonenveloped dsDNA viruses that cause respiratory, ocular and enteric diseases [1, 2]. While these ailments are self-limiting in healthy populations,

they cause significant morbidity in immuno-compromised individuals and organ transplant recipients [3]. Significantly, recombinant and replication-deficient HAdVs are frequently used as gene delivery and vaccine vectors in the clinic [4-8]. Accurate structural details of adenoviruses (AdV) could foster engineering of better vectors for therapeutic applications as well as identify the determinants that mediate binding to host factors (e.g., coagulation factor X).

Adenoviruses are large and complex viruses that have been studied at near atomic resolutions by cryo-electron microscopy (cryo-EM) [9-11] and X-ray crystallography [12, 13]. The crystal structure of a modified HAdV (Ad5F35), Ad5 virus pseudo-typed with the short fiber of Ad35 virus, pushed the limits of the structure determination by the X-ray methods due to its size and complexity. In addition, interpretation of the electron density maps were influenced by the unconventional organization of the minor protein IX in HAdVs, which is different compared to the organization in non-human (animal) adenoviruses (non-HAdV) [14-16]. It is noteworthy that previous mis-assignments of several cement proteins was primarily due to a combination factors associated with the lack of adherence to quasi-equivalence by protein IX and the difficulty of generating accurate data from multiple weakly diffracting crystals [13]. Our experiences suggest that caution must be exercised in using quasi-symmetry as a constraint, particularly related to extended molecules like IX, in interpreting the electron density maps of the icosahedral particles.

However, based on the high-resolution cryo-EM structures of HAdVs from different species (C and D), it is now clear that the triskelion and coiled-coil, four helix bundle (4-HLXB) structures are formed by N-termini and C-termini of IX, respectively. This characteristic organization of IX in HAdVs represents the main distinguishing structural feature of HAdVs.

relative to non-HAdVs, while the arrangement of the other proteins appears to be very similar

[16, 17]. We report the revised interpretation of the crystal structure of HAdV obtained by

rescaling and merging the diffraction data that are consistent with the atomic models for minor

proteins determined by cryo-EM methods [9, 10].

Materials and Methods.

Diffraction data acquisition and processing. The details of the crystallization of Ad5F35,

diffraction data acquisition and processing were described in Reddy and Nemerow (2014) [13].

However, the data were rescaled to increase the data completeness by lowering sigma cutoff to

0.1 in SCALEPACK [18] and merged with SCALA [19]. Briefly, 1642 images were collected

from 178 crystals that were processed and scaled using HKL2000 [18] and outputted as

unmerged data after 3 cycles of post-refinement to obtain the consensus unit cell. Final scaling

and merging was done using SCALA and the structure factor amplitudes were calculated using

the CCP4 program TRUNCATE [20]. The data statistics are shown in Table 1. The structure

factor amplitudes were sharpened by applying a B-factor of -20Å² and used for subsequent

structural analysis.

Structure determination, model building and model refinement. The molecular replacement

solution (Euler angles: 65.443, 109.080, 351.527; Translation: 0., 0., 0.) was obtained using the

poly-alanine model of adenovirus (PDB-ID: 3IYN) excluding the cement proteins and

employing the program Phaser [21]. The atomic models of major capsid proteins (PDB-ID:

3IYN) were oriented in the unit cell according to the Phaser solution and subjected to 200 cycles

of rigid body refinement using the program CNS [22] and the data between 20 and 5Å resolution.
The R-factor for the working set dropped from 0.46 to 0.44 with a root mean square deviation

(RMSD) of 4.2Å for 11,464 C-alpha atoms between the starting and ending models.

Significantly, the diameter of the rigid-body refined model increased by ~8Å relative to the

starting model (PDB-ID: 3IYN).

The rigid-body refined coordinates of the major capsid proteins were used to generate initial

phases and the phase-refinement was carried out using the RAVE (4 cycles) [23], followed by

DMMULTI (20 cycles) [24] by applying 60-fold non-crystallographic symmetry (NCS). Of note,

we deliberately avoided employing the phase extension methods to avoid incorporating the

noise/errors in the context of the P1 space group as observed previously [25]. Furthermore, we

found that the use of Sigmaa weighted coefficients, calculated using the SIGMAA program in

CCP4 [20], yielded slightly better quality (unbiased) maps after density modifications than those

calculated with the conventional 2Fo-Fc coefficients. The necessary atomic masks were

calculated using the programs MAMA [26] and mama2ccp4 [20] with the rigid-body refined

model embedded in a box of closely spaced (4Å) pseudo atoms that are within 15Å distance of

the atomic model and lie between the radius 300 and 425Å of the adenovirus particle. This was

done to fill the space in between the hexon and penton base subunits and within hexon cavities.

The resultant phase-refined (omit) maps clearly showed the density for minor (cement) proteins

consistent with the structures of the models derived from the cryo-EM structures (Fig. S1).

Moreover, we observed electron density, albeit weak density, connecting the triskelion forming

NT region with the corresponding CT helix in of one of the IX(Q) molecules in the original omit

maps. Of note, various factors that contributed to the overall improvements in the electron

density maps are 1) slight improvements in the data completeness by 5%, with an overall

5
completeness of 53.2% (Table 1), 2) sharpening of the structure factor amplitudes with a B-
factor of -20Å², 3) avoidance of phase-extension procedures (in P1 space group), 4) use of
tandem density modification procedures of SIGMAA weighted maps with RAVE [23] followed
by DMMULTI [24] and 5) use of phase refined omit maps for initial model building instead of
difference (Fo-Fc) maps used previously. The poly-alanine models of the minor proteins (PDB-
ID: 3IYN) were docked and adjusted into the electron density maps. The docked minor protein
(poly-alanine) models along with the major capsid proteins were rigid body refined using CNS
followed by another round of phase refinement. The side chains of all the subunits adjusted
according to the density. At this point, we used REFMAC 5 for model refinement by imposing
the strict NCS constraints and followed by another two rounds of phase refinement, model
building and model refinement. The final model R-factor was converged to 0.422 (R-free 0.427)
for the data between 20 and 3.8Å resolution. The high model R-factors are consistent with high
phase refinement R-factor 0.41 and low correlation co-efficient of 0.42. The high R-factors and
low correlation coefficients perhaps indicate weak diffraction data and may also be due to
sharpening of structure factor amplitudes. However, the omit maps calculated using the hexon
and penton base model revealed the densities corresponding to the minor proteins (Fig. S1).
Table 2 describes the extents of different protein models observed in the X-ray structure and
Table 3 shows the revised minor proteins in this report and their counterparts in the previous
reports of the Ad5F35 X-ray model [13].

Results:
The overall X-ray structure and organization of Ad5F35 is similar to the cryo-EM structures of native HAdV-C5 [9, 11] and HAdV-D26 [10]. The specific characteristics of individual proteins are described below.

Hexon. There are four structurally independent hexon trimers that occupy positions 1-4 in the adenovirus capsid (Fig. 1a). While the copies of hexon-1, also known as peripentonal hexons (PPH) surround the pentameric penton base subunits, the hexons-2, 3, and 4 with its 3-fold related counterparts form group of nine hexon (GON) substructures (Fig. 1a). The structures of hexons are highly conserved with RMSDs range between 1.15-1.29 Å for different comparisons of hexon pairs. Furthermore, the average RMSD between different hexons and the crystal structure of independent Ad5 hexon trimer (PDB-ID: 1P30) is ~1.4 Å [27]. While the HVR1 (hyper variable region 1), composed of residues 136-169, is completely disordered, the rest of the HVRs could be readily identified in all the hexon subunits. Table S1 shows the amino acid (aa) residues that form the seven different HVRs in the hexons.

Penton base (PB). Similar to hexon structures, the tertiary and pentameric (quaternary) organization of PB are well conserved in the X-ray and cryo-EM structures (RMSD: 1.9 Å). The residues 54-569 of PB are identified in the Ad5F35 structure with the exception of disordered residues (297-374) in the RGD containing loop. As in the cryo-EM structures, even though the majority of the fiber structure is disordered, there is weak difference (Fo-Fc) density corresponding to the N-terminal tails of the fiber can be seen interacting with the PB. However, no models were built into this density.
Structures of minor (cement) proteins:

Protein IX. There are 240 copies of IX present that stabilize the AdV particle (capsid), by interacting primarily with the hexon subunits. The N-terminal (NT) regions (aa 1-56) of three IX molecules form the (trimeric) triskelion structures by associating with their counterparts from the neighboring (IX) molecules at local (quasi) and strict 3-fold axes (Fig. 2). Whereas the C-terminal (CT) regions of four IX molecules, comprising residues 90-140 form 11-12 turn α-helices that are involved in forming coiled-coil structures. The linker region (aa 57-89) in between includes a stretch of alanine and serine residues (59-72) that connects the above two characteristic substructures of IX and is ordered in only one (Q) of the four distinct IX molecules.

The characteristic CT coiled-coil (4-HLXB) structure is composed of 4 helices from four IX molecules, each of which belongs to a different triskelion structure. Interestingly, three of the four triskelions belong to the reference facet, while the fourth one is part of the neighboring facet (Fig. 2). Of note, while the NT triskelion forming regions adopt similar structures and strictly conform to 3-fold symmetry, the CT helices comprising the 4-HLXB coiled-coils do not exhibit any particular quasi-symmetry as in other human adenovirus structures (Fig 2c) [16, 17]. It is noteworthy that the above protein IX organization in HAdVs is quite different from the arrangement of IX molecules observed in non-human (animal) adenovirus structures. In the latter, the “trimeric” coiled-coils composed of three CT helices are positioned on top of the triskelion structures and both of these substructures conform to 3-fold symmetry [14-16]. In human adenoviruses (HAdV), 240 copies of IX form a continuous hexagonal network on the capsid exterior that stabilizes the outer AdV virion. Although the intricate network of IX molecules stabilize HAdV capsids, it is not necessary for the formation of AdV capsids as IX-deleted
HAdVs are known to assemble into native like particles, however they exhibit lower thermo-stability and infectivity [28, 29].

Protein VIII. Protein VIII is one of the minor proteins involved in cementing the hexons on the capsid interior. This protein of 227 amino acids (aa) is found at two structurally distinct locations (labeled U and V) and therefore a total of 120 (2x60) copies of VIII are present in each AdV particle that closely interact with the bases of hexon on the capsid interior. The protein VIII displays a characteristic obtuse-triangular shaped structure comprising a long helix (aa 38-62), followed by a Ω shaped loop (aa 63-80) and a long two-stranded β-sheet (aa 81-92; 164-175) (Fig. 3). The structure of VIII appears to be well conserved among adenoviruses [10]. Protein VIII undergoes proteolytic cleavage by adenoviral protease (AVP) at two different locations Gly111 and Ala157. While the NT (aa 1-111) and CT (aa 158-227) fragments of VIII closely associate with each other to form the triangular shaped structure (Fig. 3a), the middle fragment of 46 residues (aa 112-157) appears to have been dislodged from its original location near the parent molecules after the cleavage and is thus missing in both X-ray and cryo-EM structures [9, 10]. However, it is quite possible that the missing middle fragments in existence as part of the AdV virion, but its location could not be structurally resolved. This possibility is supported by the results from mass-spectrometry analysis [30].

Importantly, VIII is involved in distinctive “molecular clamp” interactions that glue four hexons together as observed previously [9, 10]. Briefly, the aa residues 29-37 and 106-110 of VIII form short β-strands with the hexon residues 937-944 from two different hexons that reside within the GON structures. In addition, the NT residues (1-28) and CT residues (200-227) of VIII are

9
involved in cementing interactions that join 2 more hexons at the 3-fold junctions (Fig. 3b).

These occur at the PPH-GON and GON-GON interfaces mediated by the VIII molecules at the U and V locations respectively. Of note, the protein VIII (U) together with a molecule of protein IIIa is involved in cementing the hexons underneath the vertex region (see below), while the protein VIII (V) is alone involved in gluing the hexons at the second location.

Protein IIIa. Protein IIIa is the largest (65 kDa, 585 aa) among the minor proteins that are present in the AdV particle. This protein is located underneath the vertex region on the capsid interior and present in 60 copies in each virus particle [30]. It is noteworthy that the location of IIIa was previously suggested to be that of protein VI [31-33] and incorrectly assigned to VI [13]. However, based on the recent high-resolution cryo-EM structures [9, 10] and our revised X-ray structure, it is now clear that protein IIIa is located beneath the vertex region and closely interacts with PPH and connects them with GON substructures on the capsid interior. The location of IIIa in this report is consistent with the cryo-EM assignments.

The ordered region of IIIa (aa 1-300) is composed of mostly α-helices and organized as two domains: N-terminal domain (NTD, aa 1-134) and middle domain (MDLD, aa 135-300) (Fig 3c). The C-terminal domain (CTD; aa 400-585), which also contains a few (predicted) helices, is disordered. The so-called appendage domain (aa 310-390), observed in HAdV-D26 structure that occurs after the MDLD domain [10], is disordered in the Ad5F35 structure as in Ad5 cryo-EM structures [9, 11]. The NTDs from adjacent IIIa molecules, which closely interact with the bases of the pair of juxtaposed PPHs, also associate with each other and thereby gluing neighboring PPHs together. A total of 5 NTDs of IIIa are involved in stabilizing the five PPHs underneath the
vertex region. Additionally, the MDLD of IIIa overlays on top of protein VIII (U), thereby fortifying the interactions that connect the PPHs with the GON facets (Fig3d). Furthermore, the residues 107-120 of NTD are seen interacting with NT (aa 45-51) of penton base (PB). Taken together, both IIIa and VIII play critical roles in cementing the interactions between the PPHs and joining them with the GON facets, beneath the vertex region, on the interior of the AdV capsid.

Protein VI. Protein VI plays multiple key roles in the adenovirus lifecycle: 1) involved in nuclear import of hexons [34], 2) the C-terminal fragment of 11 amino acids (240-250) acts as a cofactor for the adenoviral protease (AVP) [35], 3) essential for endosomal membrane lysis and release of virions into cytoplasm [36-38] and 4) plays a putative role as a cement protein in mediating the interactions between the hexons on the capsid interior. In spite of its multi-functional role, it has been challenging to locate VI in the adenovirus structure. Notwithstanding the suggested locations based on early biochemical experiments, the first glimpse of details on the location of VI in the AdV capsid came from the mass-spectrometry based Hydrogen-Deuterium exchange experiments that revealed the cleaved N-terminal pro-peptide of VI (pVIN) (aa 1-33) interacts with the hexon bases [39] and followed by its identification in the crystal structure HAdV [13]. Subsequently, the location pVIN was confirmed by cryo-EM structures of HAdV26 [10] and HAdV5 [11], even though there exists ambiguity with regards to the directionality of the polypeptide chain and its relevance to proteolytic processing by the AVP.

In the revised crystal structure of Ad5F35 (current report), we observed the density corresponding to pVIN only in the peripentonal hexons, whereas well ordered densities were seen

11
associated with multiple hexons in the cryo-EM structures [10, 11]. This may be due to merging weak (noisy) diffraction data gathered from multiple (~180) crystals. Because of this, we were able to build only the poly-alanine model of pVIn (Fig4a).

Conclusions:

The revised crystal structure of Ad5F35 reveals the structures, locations of major and minor protein and associated interactions that are consistent with the high-resolution cryo-EM structures [9-11]. With the exception of IX, the structures of distinct copies of major and minor capsid proteins in the AdV capsid are conserved. The NT regions of IX adopt very similar (superimposable) structures that are involved in forming 3-fold symmetric triskelion structures (Fig2a, c). Whereas the C-terminal helices along with the preceding linker regions adopt different (non-superimposable) conformations and are responsible for forming the remarkable 4-helical coiled-coil (4-HLXB) structure derived from four IX molecules in the context of assembled virion (Fig. 2a, c). The arrangement of helices in 4-HLXB substructure do not conform to any particular quasi-symmetry and appears to be enforced by the hexagonal shaped hexon trimers in the assembled virion [16]. With the triskelion and 4-HLXB substructures acting as the nodes, the continuous hexagonal-shaped IX network mediates interactions between the hexons within the GON-facets and furthermore strapping the GON substructures together. The minor proteins IIIa and VIII act as “molecular clamps” that reinforce the interactions mainly between the hexons on the capsid interior. Since, only the cleaved N-terminal pro-peptide of VI is seen in the cavity of PPHs, the role of VI as molecular cement remains unclear as does its accessibility to cleavage by the adenoviral protease. Finally, the overall organization of minor

(cement) protein scaffold in human adenovirus emulates the outline of the pseudo $T=25$ quasi-
equivalent icosahedral (hexagonal) lattice (Fig. 4b).

**Author contributions**

T.M.M and G.R.N. produced and purified the Ad5F35 vector. S.K.N and S.V carried out
crystallization. S.K.N, G.R.N and V.S.R. collected the diffraction data. S.K.N and V.S.R
processed the data and determined the structure. G.R.N and V.S.R. interpreted the structure and
wrote the manuscript.

**Data deposition**

The coordinates and structure factors of the revised Ad5F35 structure have been deposited in the
Protein Data Bank (PDB) under the accession code 6CGV that replaces the previous deposition
4CWU.

**ACKNOWLEDGEMENTS**

We acknowledge the support from NIH grants, AI070771, AI103692 to V.S.R and HL054352 to
G.R.N. We thank Nhung Huynh for her assistance with data collection. We also thank Dr. Bob
Fischetti and coworkers at GM/CA beamline at the Advance Photon Source (APS), Chicago, for
technical support with the synchrotron data collection. GM/CA has been funded with federal
funds from the NIH (Y1-CO-1020, Y1-GM-11040) and the APS is supported by US Department
of Energy under the contract DE-AC02-06CH11357. Authors would like to thank Dr. Tom
Goddard for his expert advice on using special features of the Chimera program.

Figure legends.

Figure 1. Structure and organization of capsid proteins in human adenovirus type 5 (HAdV 5), referred here as Ad5F35. a) Left panel depicts an exterior view of HAdV showing the organization of major capsid proteins: hexon (gray, pink, light green and khaki), penton base (PB) (magenta) and minor protein IX (blue, cornflowerblue, cyan and purple). The locations of icosahedral 5-fold, 3-fold and 2-fold symmetry axes are indicated by yellow colored pentagon, triangle and oval respectively. Shown on the right is the zoomed-in view of the gray rectangular region on the left. The numbers 1-4 identifies the structurally unique hexons. The group of nine hexons (GON) substructure is identified by black colored trace. b) A cut open view showing the organization of the capsid proteins on the capsid interior (left). On the right is the zoomed-in view of the rectangular region on the left panel. The hexons and PB are shown in the same colors as in panel (a). The minor proteins IIIa, VIII (U), VIII (V) and pVIn are shown in green, orange, yellow and red, respectively. The locations of icosahedral 5-fold, 3-fold and 2-fold symmetry axes are indicated by red colored pentagon, triangle and oval respectively. c) Tertiary fold of the hexon subunit. The inset shows the representative electron density corresponding to residues 810-814. d) Tertiary fold of the PB subunit. The inset shows the representative electron density for the residues 162-166.

Figure 2. Structure and organization of protein IX molecules in HAdV5 (Ad5F35). a) The hexagonal network composed of 240 protein IX molecules is shown relative to icosahedral cage (yellow). On the right is the zoomed-in view of an icosahedral facet showing the triskelion and four helix bundle (4-HLXB) coiled-coil structures. The asterisk identifies the IX molecule (Q), whose linker region is fully ordered. The disordered linker regions identified by dashed lines. b)

Structure of fully ordered IX molecule (Q) and the corresponding electron density. Inset shows the zoomed-in view of the indicated region by the arrows. c) Superposition of structurally unique IX molecules, labeled P, Q, R and S, shows the structural similarity in the triskelion forming N-terminal (NT) region, while the C-terminal (CT) helices exhibit different conformations dictated by the interacting hexon subunits (not shown). The disordered linker regions are indicated by the dashed-lines.

Figure 3. Structure and associations of minor proteins VIII and IIIa. a) Ribbon diagram of protein VIII. The chains comprising residues 1-111 and 158-227 are color coded blue and red, respectively. The location of the adenoviral protease cleavage sites that releases the middle fragment (112-157) is identified by an arrow. b) Molecular clamping association of protein VIII (U), shown in red, with the peripentonal hexons (PPHs, H1 and H1') and with the hexons 2 and 4 that belong to group of nine (GON) hexons. The location of PB is identified. c) Rainbow colored ribbon diagram showing the ordered region (1-303) of IIIa. The N-terminal domain (NTD) and middle domain (MDLD) are labeled. d) Diagram showing the association of two (neighboring) IIIa molecules, colored in dark and light green, with the hexon subunits and protein VIII, colored red. Please note that the NTDs of IIIa interact closely with each other, while also interacting with the base of PPHs. The CTDs overlay on top of the VIII in reinforcing the cementing interactions that join the PPHs with GON hexons.

Figure 4. a) PPH and N-terminal pro-peptide of VI (pVIn) interactions. Ribbon diagram showing the location of pVIn (in red) in the cavity of PPH, colored gray. Selected PPH residues that are in close proximity to the pVIn are labeled. The directionality and the sequence of pVIn

could not be resolved based on the electron density, hence it was modeled as poly-alanine chain.

b) Organization of minor (cement) protein scaffold in human adenovirus. Shown at the top is the overall arrangement of cement proteins in HAdV 5 that mostly conforms to the trail of pseudo T=25 quasi-equivalent icosahedral lattice (yellow), depicted as hexagons and pentagons. The major capsid proteins hexons, penton base and fiber are not shown for the clarity. At the bottom is the zoomed-in view of the boxed region in the top panel. Color code: IX-P (blue), IX-Q (light blue), IX-R (cyan), IX-S (purple), IIIa (green), VIII-U (orange), VIII-V (yellow) and pVIn (red).

References:

[1] Wold WSM, Horwitz MS. Adenoviruses. Fields virology 5th ed. Philadelphia, PA: Lippincott Williams & Wilkins; 2007.
[2] Gordon YJ, Aoki K, Kinchington PR. Adenovirus keratoconjunctivitis. St. Louis, PA: Mosby; 1996.
[3] Kojaoghlanian T, Flomenberg P, Horwitz MS. The impact of adenovirus infection on the immunocompromised host. Rev Med Virol. 2003;13:155-71.
[4] Schenk-Braat EA, van Mierlo MM, Wagemaker G, Bangma CH, Kaptein LC. An inventory of shedding data from clinical gene therapy trials. J Gene Med. 2007;9:910-21.
[5] Kawabata K, Sakurai F, Koizumi N, Hayakawa T, Mizuguchi H. Adenovirus vector-mediated gene transfer into stem cells. Mol Pharm. 2006;3:95-103.
[6] Barefoot B, Thornburg NJ, Barouch DH, Yu JS, Sample C, Johnston RE, et al. Comparison of multiple vaccine vectors in a single heterologous prime-boost trial. Vaccine. 2008;26:6108-18.
[7] Bett AJ, Dubey SA, Mehrotra DV, Guan L, Long R, Anderson K, et al. Comparison of T cell immune responses induced by vectored HIV vaccines in non-human primates and humans. Vaccine. 2010;28:7881-9.
[8] Zhou Y, Sullivan NJ. Immunology and evolvement of the adenovirus prime, MVA boost Ebola virus vaccine. Curr Opin Immunol. 2015;35:131-6.
[9] Liu H, Jin L, Koh SB, Atanasov I, Schein S, Wu L, et al. Atomic structure of human adenovirus by cryo-EM reveals interactions among protein networks. Science. 2010;329:1038-43.
[10] Yu X, Veesler D, Campbell MG, Barry ME, Asturias FJ, Barry MA, et al. Cryo-EM structure of human adenovirus D26 reveals the conservation of structural organization among human adenoviruses. Sci Adv. 2017;3:e1602670.
[11] Dai X, Wu L, Sun R, Zhou ZH. Atomic Structures of Minor Proteins VI and VII in the Human Adenovirus. Journal of virology. 2017.
[12] Reddy VS, Natchiar SK, Stewart PL, Nemerow GR. Crystal structure of human adenovirus at 3.5 Å resolution. Science. 2010;329:1071-5.

[13] Reddy VS, Nemerow GR. Structures and organization of adenovirus cement proteins provide insights into the role of capsid maturation in virus entry and infection. Proc Natl Acad Sci USA. 2014; 111: 11715-20.

[14] Schoehn G, El Bakkouri M, Fabry CM, Billet O, Estrozi LF, Le L, et al. Three-dimensional structure of canine adenovirus serotype 2 capsid. Journal of virology. 2008; 82: 3192-203.

[15] Cheng L, Huang X, Li X, Xiong W, Sun W, Yang C, et al. Cryo-EM structures of two bovine adenovirus type 3 intermediates. Virology. 2014; 450-451: 174-81.

[16] Reddy VS. The Role of Hexon Protein as a Molecular Mold in Patterning the Protein IX Organization in Human Adenoviruses. J Mol Biol. 2017; 429: 2747-51.

[17] Matteson NL, Barry MA, Reddy VS. Structure-based assessment of protein-protein interactions and accessibility of protein IX in adenoviruses with implications for antigen display. Virology. 2018; 516: 102-7.

[18] Otwinowski Z, Minor W. Processing of X-ray diffraction data collected in oscillation mode. Methods Enzymol. 1997; 276: 307-26.

[19] Evans PR. Scala. Joint CCP4 and ESF-EACBM Newsletter 1997; 33: 22-4.

[20] CCP4. The CCP4 suite: programs for protein crystallography. Acta Crystallogr D Biol Crystallogr. 1994; 50: 760-3.

[21] McCoy AJ, Grosse-Kunstleve RW, Adams PD, Winn MD, Storoni LC, Read RJ. Phaser crystallographic software. J Appl Crystallogr. 2007; 40: 658-74.

[22] Brunger AT, Adams PD, Clore GM, DeLano WL, Gros P, Grosse-Kunstleve RW, et al. Crystallography & NMR system: A new software suite for macromolecular structure determination. Acta Crystallogr D Biol Crystallogr. 1998; 54: 905-21.

[23] Jones TA. a.yaap.asap, @#*? A set of Averaging Programs. In: Dodson EJ, Gover, S. & Wolf, W., editor. Molecular Replacement. Warrington: SERC Daresbury Laboratory; 1992.

[24] Cowtan K. DM: An automated procedure for phase improvement by density modification. Joint CCP4 and ESF-EACBM Newsletter on Protein Crystallography. 1994; 31: 34-8.

[25] Reddy VS. Application of the phase extension method in virus crystallography. Crystallogr Rev. 2016; 22: 128-40.

[26] Kleywegt GJ, Jones TA. Software for handling macromolecular envelopes. Acta Crystallogr D Biol Crystallogr. 1999; 55: 941-4.

[27] Rux JJ, Burnett RM. Type-specific epitope locations revealed by X-ray crystallographic study of adenovirus type 5 hexon. Mol Ther. 2000; 1: 18-30.

[28] Colby WW, Shenk T. Adenovirus type 5 virions can be assembled in vivo in the absence of detectable polypeptide IX. Journal of virology. 1981; 39: 977-80.

[29] Campos SK, Parrott MB, Barry MA. Avidin-based targeting and purification of a protein IX-modified, metabolically biotinylated adenoviral vector. Mol Ther. 2004; 9: 942-54.

[30] Benevento M, Di Palma S, Snijder J, Moyer CL, Reddy VS, Nemerow GR, et al. Adenovirus Composition, Proteolysis and Disassembly Studied by in-depth Qualitative and Quantitative Proteomics. J Biol Chem. 2014.

[31] Everitt E, Lutter L, Philipson L. Structural proteins of adenoviruses. XII. Location and neighbor relationship among proteins of adenovirion type 2 as revealed by enzymatic iodination, immunoprecipitation and chemical cross-linking. Virology. 1975; 67: 197-208.

[32] Stewart PL, Fuller SD, Burnett RM. Difference imaging of adenovirus: bridging the resolution gap between X-ray crystallography and electron microscopy. EMBO J. 1993; 12: 2589-99.

[33] Fabry CM, Rosa-Calatrava M, Conway JF, Zubieta C, Cusack S, Ruigrok RW, et al. A quasi-atomic model of human adenovirus type 5 capsid. EMBO J. 2005; 24:1645-54. Epub 2005 Apr 21.

[34] Wodrich H, Guan T, Cingolani G, Von Seggern D, Nemerow G, Gerace L. Switch from capsid protein import to adenovirus assembly by cleavage of nuclear transport signals. Embo J. 2003; 22:6245-55.

[35] Mangel WF, McGrath WJ, Toledo DL, Anderson CW. Viral DNA and a viral peptide can act as cofactors of adenovirus virion protease activity. Nature. 1993; 361:274-5.

[36] Greber UF. Virus assembly and disassembly: the adenovirus cysteine protease as a trigger factor. Rev Med Virol. 1998; 8:213-22.

[37] Wiethoff CM, Wodrich H, Gerace L, Nemerow GR. Adenovirus protein VI mediates membrane disruption following capsid disassembly. Journal of virology. 2005; 79:1992-2000.

[38] Luisoni S, Suomalainen M, Boucke K, Tanner LB, Wenk MR, Guan XL, et al. Co-option of Membrane Wounding Enables Virus Penetration into Cells. Cell Host Microbe. 2015; 18:75-85.

[39] Snijder J, Benevento M, Moyer CL, Reddy V, Nemerow GR, Heck AJ. The Cleaved N-Terminus of pVI Binds Peripentonal Hexons in Mature Adenovirus. J Mol Biol. 2014.

Table 1 Crystallographic data and refinement statistics

| Data collection | Native |
|-----------------|--------|
| Space group    | P1     |
| Cell dimensions | a=854.25Å, b=855.32Å, c=865.91Å, α=60.38° β=60.43° γ=61.98° |
| Number of crystals | 178 |
| Number of frames | 1642 |
| Resolution (Å) | 129.0-3.8 (3.9-3.8) (129-17) |
| Observed reflections | 15,015,304 |
| Independent reflections | 9,262,042 |
| Completeness (%) | 53.2 (16.7) (85.4) |
| Average redundancy | 1.6 (1.1) (1.8) |
| <I>/σ(I) | 2.8 (1.0) (21.1) |
| R sym | 0.363 (0.432) (0.236) |
| Wilson B-factor | 20.0 Å² |

Refinement

| Resolution (Å) | 20-3.8 Å |
|-----------------|----------|
| F/Sigma cutoff | 2.0      |
| Number of reflections: Work/Test | 8,789,814/274,644 |
| R factor, Work | 0.422    |
| Test/Free | 0.427    |
| r.m.s. deviation of protein Bond length | 0.008 |
| Bond angle | 1.203   |

Resolution range or statistics of highest resolution bin.

Resolution range or statistics of lowest resolution bin.

---

Table 2. Ordered regions of the various proteins in the Adenovirus crystal structure

| Protein         | Ordered regions                                      | Disordered regions                                    |
|-----------------|-----------------------------------------------------|------------------------------------------------------|
| Hexon-1         | 8-952(A), 11-949(B), 4-952(C)                       | 1-7(A), 1-10, 950-952(B); 1-3(C)                      |
| Hexon-2         | 8-952(D), 8-945(E), 7-948(F)                        | 1-7(D), 1-7, 946-952(E), 1-6, 949-952(F)              |
| Hexon-3         | 8-944(G), 8-948(H), 7-951(I)                        | 1-7, 945-952(G), 1-7, 949-952(H), 1-6, 952(I)          |
| Hexon-4         | 10-952(J), 11-942(K), 7-952(L)                      | 1-9(J), 1-10, 943-952(K), 1-6(L)                      |
| Penton base(PB) | 45-296, 375-569(N)                                  | 1-44, 297-374, 570-571(N)                             |
| IIIa            | 1-303(M)                                            | 304-585(M)                                           |
| VI              | 1-24(W)                                             | 25-250(W)                                            |
| VIII-U          | 3-109, 159-222(U); 3-110, 159-224(V)                | 1-2, 110-158, 223-228(U); 1-2, 111-158, 225-228(V)    |
| IX              | 2-53, 95-137(P); 2-134(Q); 2-54, 89-133 (R); 6-58, 90-132(S) | 1, 54-94, 138-140(P), 1, 135-140(Q), 1, 55-88, 134-140(R), 1-5, 59-89, 133-140(S) |

Shown in parenthesis are the chain-IDs of the corresponding subunits as listed in the PDB file.

Of note, residues (136-169) are disordered in all twelve hexon subunits with chain-IDs A through L.

Table 3. Revised minor (cement) proteins and their previous assignments

| Previous assignments [13] | Revised assignments (Current report) | Details of changes between the models |
|--------------------------|-------------------------------------|---------------------------------------|
| IX-triskelion            | IX-triskelion                      | Chain polarity is reversed           |
| IIIa                     | IX-4-HLXB (coiled-coils)          | Reassigned to 4-HLXB                 |
| VI-V complex             | IIIa                              | Reassigned to IIIa                   |
| pVIn                     | pVIn                              | Chain polarity or aa sequence assignments could not be determined. |
| VIII-X                   | VIII-U                            | Improved registration of aa sequence |
| VIII-Y                   | VIII-V                            | Improved registration of aa sequence |

Highlights:

This report describes the structures and scaffolding organization of cement proteins based on the crystal structure of human adenovirus virion that highlights the limits on quasi-equivalence displayed by protein IX in forming a continuous hexagonal lattice that stabilizes adenovirus capsid. The report also provides a word of caution in using quasi-equivalence as a constraint during complex virus structure determinations.

(a)
![Image](image1.png)

(b)
![Image](image2.png)

(c)
![Image](image3.png)

(d)
![Image](image4.png)

Figure 1

(a)

(b)
Tyr49
Thr55
Leu57
CT
linker region

(c)
2
NT
57
136
P
95
Q
134
R
132
S
90
132

Figure 2

(a)
- 38
- 3
- 62
- 81
- 109
- 174
- 222

(c)
- NTD
- 134
- MDLD
- 303

(b)
- H1'
- PB
- H1
- VIII (V)
- H4
- H2

(d)
- H1'
- PB
- NTD
- H1
- CTD
- H4''
- H4
- H2
- H1''
- CTD

(a)

N773

N42

Q30 A34

(b)


Figure 4
