
JBC ARTICLE

**Structural and functional characterization of a modified legionaminic acid involved in glycosylation of a bacterial lipopolysaccharide**

Received for publication, July 19, 2018, and in revised form, October 7, 2018 Published, Papers in Press, October 12, 2018, DOI 10.1074/jbc.RA118.004966

Nathan D. McDonald ${}^{1,2}$, Kristen E. DeMeester ${}^{S2}$, Amanda L. Lewis ${}^{1}$, Catherine Leimkuhler Grimes ${}^{\sharp S}$, and E. Fidelma Boyd ${}^{3}$

From the Departments of ${}^{\dagger}$Biological Sciences and ${}^{S}$Chemistry and Biochemistry, University of Delaware, Newark, Delaware 19716 and the ${}^{\ddagger}$Department of Molecular Microbiology, Washington University School of Medicine, St. Louis, Missouri 63110

Edited by Chris Whitfield

---

Nonulosonic acids (NulOs) are a diverse family of $\alpha$-keto acid carbohydrates present across all branches of life. Bacteria biosynthesize NulOs among which are several related prokaryotic-specific isomers and one of which, $N$-acetylneuraminic acid (sialic acid), is common among all vertebrates. Bacteria display various NulO carbohydrates on lipopolysaccharide (LPS), and the identities of these molecules tune host–pathogen recognition mechanisms. The opportunistic bacterial pathogen *Vibrio vulnificus* possesses the genes for NulO biosynthesis; however, the structures and functions of the *V. vulnificus* NulO glycan are unknown. Using genetic and chemical approaches, we show here that the major NulO produced by a clinical *V. vulnificus* strain CMCP6 is 5-$N$-acetyl-7-$N$-acetyl-$D$-alanyl-legionaminic acid (Leg5Ac7AcAla). The CMCP6 strain could catabolize modified legionaminic acid, whereas *V. vulnificus* strain YJ016 produced but did not catabolize a NulO without the $N$-acetyl-$D$-alanyl modification. *In silico* analysis suggested that Leg5Ac7AcAla biosynthesis follows a noncanonical pathway but appears to be present in several bacterial species. Leg5Ac7AcAla contributed to bacterial outer-membrane integrity, as mutant strains unable to produce or incorporate Leg5Ac7AcAla into the LPS have increased membrane permeability, sensitivity to bile salts and antimicrobial peptides, and defects in biofilm formation. Using the crustacean model, *Artemia franciscana*, we demonstrate that Leg5Ac7AcAla-deficient bacteria have decreased virulence potential compared with WT. Our data indicate that different *V. vulnificus* strains produce multiple NulOs and that the modified legionaminic acid Leg5Ac7AcAla plays a critical role in the physiology, survivability, and pathogenicity of *V. vulnificus* CMCP6.

Nonulosonic acids (NulOs), ${}^{4}$ which include the sialic acids, are a family of nine carbon $\alpha$-keto sugars that serve as critical recognition elements between cells. The most common sugar, $N$-acetylneuraminic acid (Neu5Ac) (otherwise known as sialic acid) (1), is found in most lineages of metazoans and performs a wide range of functions in eukaryotes such as cell to cell communication (Fig. 1). Although these carbohydrates are utilized by eukaryotes, bacteria can both biosynthesize and catabolize sialic acid (1–5). The ability to catabolize sialic acid is widespread among both commensal and pathogenic species (3, 6). Specifically, the catabolism of NulOs is an important phenotype for host–pathogen interactions (3, 6–9). Certain bacteria also utilize sialic acid to decorate their cell-surface structures such as the lipopolysaccharide (LPS) and/or capsule polysaccharide (CPS) (10–17). Sialic acids on the cell surface contribute to the evasion of the host immune response as a form of molecular mimicry (18–24).

Although sialic acid is found in eukaryotes and some prokaryotes, it is not the predominant NulO produced in bacteria. Nine carbon $\alpha$-keto sugars bearing a strong resemblance to sialic acid such as derivatives from 5,7-diamino-3,5,7,9-tetradeoxy-non-2-ulosonic acids (Fig. 1; Fig. S1) have been characterized as prokaryote-specific NulOs. Traditionally, these prokaryote-specific NulOs are further categorized into one of four classes of carbohydrates based on their stereochemistry: pseudaminic, legionaminic (Leg), or two epimers of Leg, 4-epi-legionaminic acid and 8-epi-legionaminic acid (25–27). However, recent studies have described three new NulOs, specifically acinetaminic (Ace) and 8-epiacinetaminic acid found in the CPS of *Acinetobacter baumannii* and fusaminic acid (Fus) from the LPS of *Fusobacterium nucleatum* (Fig. S1) (28–30). The prokaryotic NulOs have been identified in various surface structures, including the LPS and CPS as well as $N$- and $O$-linked glycans on flagella and pili (31–37). Although significant work has investigated the role of sialic acid in molecular mimicry and host evasion, the chemical composition and

---

This work was supported in part by National Science Foundation Career Award DEB-0844409 (to E. F. B.). The authors declare that they have no conflicts of interest with the contents of this article. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.

This article contains Figs. S1–S7.

${}^{1}$ Funded by a University of Delaware graduate fellowship award.

${}^{2}$ Supported by National Institutes of Health Chemistry-Biology Interface (CBI) Training Grant 5T32GM008550.

${}^{3}$ To whom correspondence should be addressed: Dept. of Biological Sciences, 341 Wolf Hall, University of Delaware, Newark, DE 19716. Tel.: 302-831-1088; E-mail: fboyd@udel.edu.

---

${}^{4}$ The abbreviations used are: NulO, nonulosonic acid; Leg5Ac7AcAla, 5-$N$-acetyl-7-$N$-acetyl-$D$-alanyl-legionaminic acid; Leg, legionaminic acid; Ace, acinetaminic acid; Fus, fusaminic acid; PmB, polymyxin B; Leg5Ac7Ala, 5-$N$-acetyl-7-$D$-alanyl-legionaminic acid; DMB: 1,2-diamino-4,5-methylene dioxybenzene; PG, peptidoglycan; LPS, lipopolysaccharide; CPS, capsule polysaccharide; HR LCMS, high-resolution LC-MS; ESI, electrospray ionization.

# Modified legionaminic acid in bacterial lipopolysaccharides

**Nonulosonic Acid Recognition**

![Diagram](https://i.imgur.com/1234567.png)

Figure 1. Nonulosonic acids are a structurally diverse family of carbohydrates present on the surface of cells across all branches of life. These carbohydrates play a critical role in various cell communications and host–pathogen interactions. This work describes a prokaryote-specific nonulosonic acid (Leg5Ac7AcAla) in the lipopolysaccharide of the marine pathogen *V. vulnificus*, which is required for its survivability.

role of bacteria-specific NulOs in host–pathogen interactions remain under-explored. In addition to the seven isomers of the prokaryote-specific NulOs described, these carbohydrates can display various functionalized $N$- and $O$-substitutions such as acetylation. These diverse modifications may further contribute to multiple pathogen–host interaction mechanisms *in vivo* and in the environment (38). The wide array of structural variants and the newest discoveries Ace and Fus motivated our efforts to discover unique functional groups and novel isomers within the model organism *Vibrio vulnificus*.

Previous work from our group confirmed that strains of *V. vulnificus*, a marine bacterium and human opportunistic pathogen, produced NulOs; however, the genomic region responsible for the NulO biosynthesis was highly variable within the species (Fig. 2A) (39). One study fully elucidated the inner core of the LPS from the *V. vulnificus* strain ATCC 27562, which was found to have pseudaminic acid as a carbohydrate component (40). Biochemical analysis of *V. vulnificus* strains with divergent NulO biosynthetic loci revealed that highly variable levels of NulO were produced depending on the genotype; however, no detailed NulO structural information was elucidated (39). Two *V. vulnificus* strains in particular, CMCP6 and YJ016, were studied in more detail. These two strains are nearly identical; however, the NulO biosynthetic region is unrelated (Fig. 2A). *V. vulnificus* strains with the CMCP6-like NulO genes produce ~100-fold more NulO than strains with the YJ016-like genes (41).

Efforts from our laboratory to connect the genetic NulO fingerprint with *V. vulnificus* pathogenicity revealed that NulO-deficient mutants have significantly increased sensitivity to the bacterial antimicrobial peptide polymyxin B (PmB), defects in swimming and biofilm formation, and are attenuated in a mouse bloodstream model of infection (41). The extent of bacterial bloodstream survival and dissemination was proportional to the amount of NulO expressed by the WT strain (NulO expression: CMCP6 ≫ YJ016) (41). In addition, *V. vulnificus* mutants in the NulO biosynthetic pathway were defective in virulence (42).

Given the correlation between NulO display and bacterial pathogenesis, we chemically characterized the carbohydrate biosynthesized in strain CMCP6, which produces higher levels of NulO (39, 41). We found that a modified legionaminic acid is produced in strain CMCP6, with an $N$-acetyl-$D$-alanyl group at C-7 (Leg5Ac7AcAla, Fig. 1). Through gene deletion analysis, we demonstrate that the modified Leg, 5-$N$-acetyl-7-$D$-alanyl-legionaminic acid (Leg5Ac7Ala), lacking an acetyl group, is catabolized via the canonical sialic acid pathway. Further analysis of deletion mutants demonstrate that biosynthesis of the acetylated $D$-alanine carbohydrate is essential for outer membrane integrity, stress survival, and virulence in the brine shrimp model *Artemia franciscana*.

## Results

### V. vulnificus CMCP6 biosynthesizes a modified legionaminic acid

To begin to determine the structure of *V. vulnificus* NulOs embedded in the sequence and genomic organization of the biosynthesis pathways, we first investigated whether the NulO biosynthesis genomic loci were homologous to previously characterized pathways. The predicted biosynthesis operon of strain CMCP6 was similar in gene arrangement and amino acid composition with the biosynthetic region from *Escherichia coli* O161 (Fig. 2B). Specifically, the Nab1, Nab2, and Nab3 proteins of CMCP6 shared 54, 81, and 73% amino acid identity with Lea7, Lea4, and Lea5 of *E. coli* O161. The LPS from *E. coli* O161 contains a 5-$N$-acetyl-7-$D$-alanyl-legionaminic acid (Leg5Ac7Ala) (43). The Leg5Ac7Ala is modified with a $D$-alanine at C-7 compared with the typical acetamido group in legionaminic acid (Fig. 2C). Interestingly, the operon of CMCP6 contained a putative $N$-acetyltransferase (*i.e.* nab5), which was absent from the *E. coli* operon (Fig. 2B). We hypothesized that this $N$-acetyltransferase would acetylate the free amine of the $D$-alanine, resulting in a NulO with a 7-$N$-acetyl-$D$-alanyl modification at C-7 (Fig. 2C). To determine whether CMCP6 does in fact produce this modified Leg, the NulOs were released from CMCP6 via mild acid hydrolysis in 2$_N$ acetic acid, derivatized with the fluorescent molecule 1,2-diamino-4,5-methylene dioxybenzene (DMB), and separated via HPLC as described previously (Fig. 3A) (41, 44). The major NulO peak eluted at ~17 min (Fig. 3B). The derivatized NulO was collected and analyzed by electrospray ionization MS (ESI-MS). The [M + H]$^+$ of 522.17 $m/z$ as well as the [M + Na]$^+$ of 544.21 $m/z$ from the NulO peak were consistent with the predicted modification that corresponds to a derivatized NulO modified with acetylated alanine (Leg5Ac7AcAla) (Fig. 3C).

To characterize in more detail the native NulO structure, the LPS of WT CMCP6 was purified via the hot water/phenol

Modified legionaminic acid in bacterial lipopolysaccharides

A.
![Diagram A](#)

B.
![Diagram B](#)

C.
![Diagram C](#)

Figure 2. Nonulosonic acid biosynthesis loci are nonhomologous between *V. vulnificus* strains. A, predicted NulO biosynthetic genomic regions were identified in *V. vulnificus* isolates via protein BLAST analysis. The regions were downloaded from the genome database and compared with EasyFig. B, BLASTp analysis of the proteins encoded in strain CMCP6 identified the region and shares homology with the previously described biosynthetic loci of *E. coli* O161. C, canonical Leg along with Leg5Ac7Ala of *E. coli* O161 and the predicted Leg5Ac7AcAla of CMCP6. Arrows represent open reading frames, and grayscale between operons represents amino acid homology. Genes encoding proteins predicted to carry out similar functions are colored similarly.

method (45), and the NulO was released from purified LPS of CMCP6 in 6% acetic acid for 4 h at 100 °C. The hydrolysate was separated by size-exclusion chromatography. The fractions containing NulOs, as indicated by staining with *p*-anisaldehyde via thin-layer LC (TLC), were collected and analyzed via high-resolution LC–MS (HR LCMS) to confirm the presence and identity of the native carbohydrate (Fig. 4). The expected ionization pattern for the native NulO (Leg5Ac7AcAla) was confirmed via HR LCMS to include the dehydrated mass [M – 18]<sup>+</sup> of 388 *m/z*, [M + H]<sup>+</sup> of 406 *m/z*, and [M + Na]<sup>+</sup> of 428 *m/z* (Fig. 4).

Detailed structural <sup>1</sup>H NMR and <sup>13</sup>C NMR analyses were conducted on the native NulO (Leg5Ac7AcAla) fractions. 460 mg of LPS was purified from 45 liters of culture, and the LPS was hydrolyzed in acetic acid to release the NulOs, which were further purified for NMR characterization in D<sub>2</sub>O (Fig. S2, A–D). The NMR analysis revealed the presence of the corresponding CH<sub>3</sub> of the alanine functionality upfield (~1.5 ppm) from hydrogens attached to α-carbon downfield (~3.9 ppm) as indicated via 2D <sup>1</sup>H NMR analysis (Fig. S2, C and D). These signature chemical shifts confirmed that the NulO produced by *V. vulnificus* CMCP6 contained an alanyl functionality. In tan-

dem with the MS data and genomic analysis, the NulO structure was determined to be 5-*N*-acetyl-7-*N*-acetyl-*D*-alanyl-legionaminic acid (Leg5Ac7AcAla) (Fig. 2C).

**Catabolism of 5-*N*-acetyl-7-*D*-alanyl-legionaminic acid in *V. vulnificus* CMCP6**

Our previous studies demonstrated that there are variable amounts of NulO produced depending on the genotype of *V. vulnificus* strains. In particular, strain CMCP6 produces 100-fold more NulO than YJ016 under the same culture conditions (Fig. 5, A and B) (41). When characterizing the biosynthetic loci of these strains, the Δ*nab1* deletion resulted in the opposite phenotype in which YJ016 produced more NulO than CMCP6. The Nab1 enzyme is a cytidine monophosphate (CMP)–NulO synthetase and catalyzes the transfer of CMP from CTP to activate the NulO for use by a downstream glycosyltransferase, ultimately linking the sugar to LPS (46). No detectable NulO was observed in the HPLC analysis of DMB-derivatized total cellular fractions of the CMCP6 Δ*nab1* mutant (Fig. 5C), suggesting that the Leg5Ac7AcAla is catabolized by this strain. Conversely, in the YJ016 Δ*nab1* we observe accumulation of a NulO eluting at ~14 min, which suggests that this carbohy-

Modified legionaminic acid in bacterial lipopolysaccharides

A.

B.


![Chemical Structure](attachment:chemical_structure.png)

C.


![HPLC Chromatogram](attachment:hplc_chromatogram.png)

Figure 3. Confirmation of predicted structure in CMCP6 by HPLC and mass spectrometry. A, derivatization reaction of acid released NulOs with the fluorescent molecule DMB. B, nonulosonic acids were derivatized with DMB and separated by reverse-phase HPLC, and absorbance was measured at 373 nm. C, separated DMB-NulOs were isolated and subjected to ESI-MS analysis. The major ion corresponds to the expected size [M + H]<sup>+</sup> of 522 m/z and [M + Na]<sup>+</sup> of 544 m/z consistent with the predicted structure.

19116 J. Biol. Chem. (2018) 293(49) 19113–19126

Grimes_Kristen_NMF11 #191-227 RT: 0.53-0.61 AV: 6 NL: 3.27E7
T: FTMS + p ESI Full ms [200.0000-1200.0000]

406.18295  
z=1  
C₁₆H₂₈O₉N₃ = 406.18201  
2.33419 ppm

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

Chemical Formula: C₁₆H₂₇N₃O₉  
Exact Mass: 405.17473

Figure 4. High-resolution MS analysis of purified nonulosonic acid from *V. vulnificus* CMCP6. The modified nonulosonic acid was released from purified LPS via mild acid hydrolysis in 6% acetic acid, separated by size-exclusion chromatography, and then analyzed by HR MS of separated modified-NulO from CMCP6 LPS.

---

**A.**

![Graph A](https://i.imgur.com/graph_a.png)

**B.**

![Graph B](https://i.imgur.com/graph_b.png)

**C.**

![Graph C](https://i.imgur.com/graph_c.png)

**D.**

![Graph D](https://i.imgur.com/graph_d.png)

Figure 5. Analysis of *nab1* deletion mutants from *V. vulnificus* CMCP and YJ016. Total cellular fractions were derivatized with DMB, and the derivatized NulOs were separated by HPLC monitoring absorbance at 373 from either WT CMCP6 (A), WT YJ016 (B), CMCP6 Δ*nab1* (C), or YJ016 Δ*nab1* (D).

---

drate is not catabolized (Fig. 5D). Mass spectrometry analysis of the derivatized NulO revealed the accumulation of an unfunctionalized NulO such as legionaminic or pseudaminic acid (Fig. S3, A and B).

Because Leg5Ac7AcAla does not appear to accumulate in the CMCP6 Δ*nab1* strain, the carbohydrate might be catabolized further within the cell. Previous work from our group has demonstrated the genes required for sialic acid transport and catabolism are lineage-specific within *V. vulnificus* and that CMCP6 can utilize sialic acid as a sole carbon and energy source (Fig. S4A) (47). Therefore, a double deletion mutant of Δ*nana*/Δ*nab1* in which the first sialic acid catabolic enzyme, *N*-acetylneuraminate lyase (NanA), is deleted along with Δ*nab1* would be predicted to cause the accumulation of Leg5Ac7AcAla within the cell (Fig. 2C and Fig. S4B). The single Δ*nana* deletion mutant produced Leg5Ac7AcAla in similar amounts to WT CMCP6, and this mutant also accumulates sialic acid from the medium inside the cell (Fig. S4C). However, the DMB

Modified legionaminic acid in bacterial lipopolysaccharides

A. CMCP6 ΔnanA/Δnab1

| mAU |  |
| --- | --- |
| 2500 |  |
| 2000 |  |
| 1500 | NulO (de-acetylated) |
| 1000 |  |
| 500 |  |
| 0 |  |

Time (min.)

B.

| Relative abundance |  |
| --- | --- |
| 100 |  |
| 480.26 |  |
|  | AcHN |
|  | HO |
|  | NH |
|  | COOH |
|  | OH |
|  | NH₂ |
|  |  |
| 462.27 |  |
|  | 463.30 |
|  | 464.28 |
|  | 466.48 |
|  | 472.61 |
|  | 476.11 |
|  | 477.43 |
|  | 481.32 |
|  | 482.24 |
|  | 488.22 |
|  | 489.59 |
|  | 496.26 |
|  | 499.70 |
|  | 502.25 |
|  | 503.26 |
|  | 504.27 |
|  | 508.35 |
| 0 |  |

m/z

C.

|  | Bacterial Cell |
| --- | --- |
| Leg5Ac7Ala | CMP-Leg5Ac7Ala |
|  | NanA |
|  | Leg5Ac7Ala accumulates |

Proposed biosynthesis pathway for Leg5Ac7AcAla

Based on the data presented and the previously reported biosynthetic pathways (43, 46), we propose an eight-step pathway for the production of the Leg5Ac7AcAla in V. vulnificus CMCP6 (Fig. 7, A and B). In this model, the biosynthesis of Leg5Ac7AcAla proceeds with several key differences from what has been described for Leg (46). The addition of the D-alanine is predicted to be catalyzed by Nab4 (VV1_0809), which is annotated as an N-acyltransferase. Nab4 contains a partial N-acyltransferase superfamily domain, which includes a broad class of enzymes catalyzing various acylation reactions, including amino acid modifications. We predict that Nab4 is the alanyltransferase due to the shared homology with Lea3 from E. coli O161, which biosynthesizes the D-alanyl-legionaminic acid (Fig. 2C).

In addition to the D-alanine modification, the second unique step in this biosynthetic pathway is the acetylation of this alanine residue by the enzyme encoded by nab5 (VV1_0805). Nab5 is a predicted N-acetyltransferase with a complete acetyltransf_3 domain, which specifically catalyzes N-acetylation reactions. Our analysis of the ΔnanA/Δnab1 deletion mutant demonstrated that Leg5Ac7Ala accumulates in the cell and is lacking the second acetyl group when compared with WT CMCP6 (Fig. 6). This result would suggest that the acetylation reaction occurs after the addition of the nucleotide CMP by Nab1, which was always considered the last step in canonical NulO biosynthesis pathways (Fig. 7B).

Membrane integrity of nonulosonic acid mutants

NulO mutants have increased sensitivity to antimicrobial peptides, specifically PmB (Fig. S5) (41, 48). To determine whether PmB induces outer membrane permeability in the NulO mutant, a lysozyme permeability assay was utilized. Under normal growth conditions, Gram-negative bacteria exhibit resistance to lysozyme, an enzyme that cleaves the carbohydrate units of bacterial peptidoglycan (PG), because the LPS limits access to the PG layer (49). If the outer membrane becomes permeabilized, the lysozyme can gain access to the PG and cause cell lysis. Both the WT and Δnab2 deletion mutant, producing no Leg5Ac7AcAla, were subjected to sub-lethal concentrations of both PmB and lysozyme in combination, and the optical density was measured for cell lysis. Under these conditions, the Δnab2 mutant cells lysed significantly more than WT (Fig. 8A). This result indicates that the PmB is acting synergistically by permeabilizing the outer membrane of the Δnab2 mutant, allowing lysozyme to cause cell lysis, whereas the WT remains unaffected. Previously, we demonstrated that complementation with a functional nab2 gene in Δnab2 restores Leg5Ac7AcAla biosynthesis, although not back to WT levels (41). We evaluated the Δnab2 complement strain for sensitivity to PmB-induced membrane permeability, and we found the defect was rescued compared with the Δnab2 mutant, albeit not to WT levels, likely due to insufficient Leg5Ac7AcAla production (Fig. 8A). To further assess the membrane integrity under physiologically relevant conditions, we performed survival assays in 15% sodium cholate, a bile salt with a detergent-like mechanism of action. CFUs were determined 0, 30, and 60 min

Figure 6. Catabolism of Leg5Ac7Ala in V. vulnificus CMCP6. Total cellular fractions were derivatized with DMB and separated by HPLC, and absorbance was measured for CMCP6 ΔnanA/Δnab1 (A). B, derivatized NulOs were collected and analyzed by ESI-MS revealing a mass of [M + H] of 479 corresponding to the loss of an acetyl group compared with WT. C, schematic of the accumulation of Leg5Ac7Ala in the ΔnanA/Δnab1 double mutant of V. vulnificus CMCP6.

derivatization of the total cellular fractions of the CMCP6 ΔnanA/Δnab1 followed by HPLC revealed the accumulation of a different NulO molecule, as the HPLC retention time and signal intensity were different from what was shown for Leg5Ac7AcAla (Figs. 2C and 6A). Mass spectrometry analysis of this molecule accumulating within the ΔnanA/Δnab1 mutant revealed a mass of [M + H]⁺ 480.2 m/z, 42 less than WT Leg5Ac7AcAla corresponding to the loss of an acetyl group from the sugar (Fig. 6B). This result shows that 5-N-acetyl-7-D-alanyl-legionaminic acid (Leg5Ac7Ala) can be catabolized via NanA when Nab1 is inactivated (Fig. 6C).

Modified legionaminic acid in bacterial lipopolysaccharides

A.
![Diagram A](https://i.imgur.com/1234567.png)

B.
![Diagram B](https://i.imgur.com/8901234.png)

Figure 7. Proposed biosynthesis pathway for Leg5Ac7AcAla within V. vulnificus CMCP6. A, CMCP6 Leg5Ac7AcAla biosynthesis operon. B, function of enzymes within the pathway were assigned via homology analysis and biochemical characterizations of deletion mutant strains. The biosynthesis of Leg5Ac7AcAla proceeds via an eight-step process, including a putative D-alanyltransferase (Nab4) and an N-acetyltransferase (Nab5) that functions following CMP addition.

following exposure. The Δnab2 mutant exhibited an approximate log decrease in CFUs immediately following exposure to bile salts, and this defect continued throughout the experiment (Fig. 8B). Together, these results indicate that the biosynthesis of Leg5Ac7AcAla and that the LPS modification by Leg5Ac7AcAla are important for the outer membrane integrity of V. vulnificus particularly in the presence of antimicrobial peptides. The Δnab2 complement strain was evaluated for bile sensitivity and shown to rescue the phenotype; however, the strain was more resistant to sodium cholate compared with WT CMCP6 (Fig. 8B). This result suggests that the decreased levels of Leg5Ac7AcAla produced in the complement may be more advantageous for cholate resistance in V. vulnificus CMCP6.

Previously, we demonstrated that the Δnab2 mutant of CMCP6 was attenuated in a mouse bloodstream model of infection (41). Because of the effects to membrane integrity and sensitivity in the presence of PmB, the Δnab2 mutant could be defective in human serum, which contains the antimicrobial complement cascade but lacks any immune cells. However, survival assays in total human serum were conducted, and CFUs were measured over the course of 2 h post-exposure, and we found that the Δnab2 mutant had no survival defect (Fig. S6A). A translucent variant strain CMCP6/T lacking a capsule, previously shown to be essential for serum resistance (50), was cleared after serum exposure for 30 min (Fig. S6A). All strains were able to survive in heat-inactivated serum (Fig. S6B).

**Leg5Ac7Ala contributes to biofilm formation**

Previous work demonstrated that the Δnab2 mutant of V. vulnificus CMCP6 was defective in biofilm formation, and NulO biosynthesis has been implicated in biofilm formation in several other bacteria (36, 41, 51–53). It was speculated that the presence of NulO may contribute to the V. vulnificus biofilm formation mechanism. A preliminary investigation into the role of NulOs in biofilm was conducted utilizing crystal violet staining. After 24 h, the Δnab1 mutant exhibited a decrease in biofilm formation compared with WT CMCP6, which was restored albeit not back to WT levels following nab1 complementation (Fig. 8C and Fig. S7). However, this defect was abolished in the ΔnanA/Δnab1 double deletion mutant, which accumulates Leg5Ac7Ala within the cell (Fig. 8C). This result suggests that the accumulation of Leg5Ac7Ala overrides the original defect in biofilm formation, despite the inability to activate the carbohydrate with a CMP moiety (Fig. 8C, Fig. 2C). A more detailed analysis is required to fully understand the composition and regulation of biofilm formation in these NulO mutants of V. vulnificus.

**Leg5Ac7AcAla mutants are attenuated in virulence toward A. franciscana**

Because of the conservation of nonulosonic acid biosynthesis pathways within V. vulnificus, we speculated that these sugars would be important for survival and/or pathogenesis in the marine environment. A. franciscana is a model crustacean and has previously been used to assess V. vulnificus virulence factors (54–56). Here, the ability of WT CMCP6 and the Δnab1 and Δnab2 deletion mutants to kill A. franciscana nauplii was examined. Artemia cysts were hatched in artificial seawater, and subsequently, the nauplii were exposed to \(1 \times 10^6\) CFUs of either WT, nab mutants, or autoclaved CMCP6 control and were monitored as alive or dead 24 and 48 h following exposure. At the end of 2 days, there was a 21.7% survival rate of nauplii that were infected with WT CMCP6. Comparatively, there

Modified legionaminic acid in bacterial lipopolysaccharides

A.
![Graph A](graph_a.png)

B.
![Graph B](graph_b.png)

C.
![Graph C](graph_c.png)

D.
![Graph D](graph_d.png)

Figure 8. Leg5Ac7AcAla biosynthesis supports outer membrane integrity in *V. vulnificus* CMCP6 and enables biofilm formation and virulence in brine shrimp. A, polymyxin B induced membrane permeability. WT CMCP6, the Δ*nab2* deletion mutant, or the *nab2* complement were subjected to both PmB (10 μg/ml) and lysozyme (10 μg/ml) in PBS or PBS alone. Cell lysis was monitored as a change in optical density (*A*~595~). B, bile salt survival assay. Survival assays of WT CMCP6, the Δ*nab2* mutant, or the *nab2* complement were performed in 15% sodium cholate, a bile salt. CFU were determined for each strain prior to exposure and at 0, 30, and 60 min following exposure. C, biofilm formation of *V. vulnificus* strains. The strains were grown statically for 24 h in Marine broth at room temperature. Biofilms were stained with 0.1% crystal violet and photographed, and the *A* was measured following resuspension in DMSO. Individual points represent average biological replicates from six technical replicates. Lines indicate mean and S.D. D, *A. franciscana* model of infection. *Artemia* nauplii were hatched in artificial seawater and exposed to 1 × 10^6^ CFU of either WT CMCP6, Δ*nab1*, Δ*nab2*, the *nab1* complement, the *nab2* complement, or heat-killed (HK) control CMCP6. Survival of the nauplii was determined at 24 and 48 h following exposure. Individual points represent independent experiments consisting of three technical replicates. Bar graph indicates mean, and error bars represent S.D. Welch’s ANOVA with Games-Howell post-hoc analyses. *p* < 0.05.

was 59 and 57.7% survival for the Δ*nab1* and Δ*nab2* deletion mutants, respectively (Fig. 8D). The WT CMCP6 was significantly more virulent toward the nauplii than the Leg5Ac7 AcAla null strains. However, these strains are more virulent than the heat-killed bacteria control in which 85% of the nauplii survived (Fig. 8D). Complementation of the Δ*nab1* and Δ*nab2* mutants with functional copies of *nab1* or *nab2* did not completely restore virulence potential as the nauplii exhibited a survival rate of 51.7% following exposure to these strains (Fig. 8D). The inability to complement back to WT is likely due to the insufficient Leg5Ac7AcAla production across the 48-h incubation time.

Discussion

Nonulosonic acid chemistry and structural diversity are expanding fields, with new isomers and modifications still being discovered and characterized. These unique carbohydrates, key building blocks of LPS, are implicated in various pathologies of virulent bacteria as well as their overall survivability. Here, we characterized the NulOs of the opportunistic human pathogen *V. vulnificus* and found structural diversity within the species. A modified legionaminic acid in the LPS of strain CMCP6 was determined, Leg5Ac7AcAla (Figs. 2C and 4 and Fig. S2). Our previous work demonstrated that the *nab* alleles of strain CMCP6 were present within a large number of

Modified legionaminic acid in bacterial lipopolysaccharides

V. vulnificus  
CMCP6  

V. metoecus  

V. parahaemolyticus  

V. kanaloae  

Aliivibrio wodanis  

Aeromonas popoffi  

Figure 9. Homologous Leg5Ac7AcAla biosynthetic genomic regions outside of *V. vulnificus*. Key NulO biosynthetic proteins were used as query to identify the homologous pathways in bacteria outside of *V. vulnificus*. The genomic regions were downloaded and used for comparative genomics analysis. The gray bars between coding regions represent amino acid homology. The ORFs predicted to catalyze similar reactions are colored the same. Figure was created in EasyFig.

clinical isolates of *V. vulnificus* suggesting that this NulO is prominent within this species (39). In addition, Leg5Ac7AcAla is not limited to *V. vulnificus*, because bioinformatics analysis identified homologous biosynthetic pathways in many other bacterial species (Fig. 9). Included in this list are several fish pathogens (such as Aliivibrio wodanis and Aeromonas popoffii) as well as the significant human pathogen Vibrio parahaemolyticus. Indeed, Leg5Ac7AcAla was identified in the LPS of an O-untypeable strain of *V. parahaemolyticus* (57).

Deletion of key biosynthetic enzyme genes suggested that acetylation of the alanyl group occurs after the addition of the CMP sugar carrier. The activation with CMP is typically considered the final step of the NulO biosynthesis prior to incorporation into the downstream structures. We propose that *V. vulnificus* utilizes a post-CMP–NulO modification strategy in which the free amine of the D-alanine is acetylated. Interestingly, previous work has proposed other modifications of NulOs that occur on the CMP-NulO in Campylobacter jejuni (58).

The genomic regions encoding the NulO biosynthetic pathway in *V. vulnificus* strains CMCP6 and YJ016 are highly variable in gene content and arrangement with very little shared homology. The lack of *nab4* and *nab5* homologs in YJ016 indicates that the N-acetyl-D-alanyl NulO modification does not occur in this strain. The biochemical analysis of the NulO of YJ016 via HPLC and ESI-MS indicates that this carbohydrate is of the di-N-acetylated variety characteristic of bacterial NulOs without any further modifications. Furthermore, this carbohydrate is possibly a stereoisomer other than Leg, due to the inability of this NulO to be catabolized by the sialic acid catabolism pathway and NanA.

The Leg5Ac7Ala NulO can be catabolized via the sialic acid degradation pathway in *V. vulnificus* CMCP6 (Fig. 6). This finding is of particular interest as the ability to utilize sialic acid as a nutrient has been implicated in host colonization in various pathogens (6, 7, 9). Previously, we demonstrated that sialic acid catabolism enhances colonization by Vibrio cholerae (6, 7). We showed that sialic acid catabolic pathways are largely associated with mucosa-associated commensals and pathogens (3, 6, 7). Both Leg5Ac7AcAla and sialic acid have the same stereochemistry in the carbohydrate core. Therefore, the sialic acid catabolic enzymes may exhibit substrate promiscuity and act on Leg allowing catabolism of these prokaryotic NulOs as we have demonstrated here with NanA of *V. vulnificus*. Further research is required to determine whether prokaryote-specific NulOs such as Leg can be scavenged from diverse microbial populations and utilized as an exogenous energy source contributing to niche expansion.

The NulO biosynthetic mutants have increased sensitivity to the antimicrobial peptide PmB, which is also able to permeabilize the membrane of the mutants compared with WT. We predicted that the defect in membrane integrity in the presence of PmB could explain our previous work, which showed that these mutants are readily cleared by the immune system *in vivo* (41). However, we found here that these same mutants are not sensitive to human serum *in vitro*, suggesting there is something present *in vivo*, beyond the antimicrobial activity of the complement cascade, that is responsible for the clearing of the NulO-deficient strains. This result is further supported by a recent study examining *V. vulnificus* YJ016 growth in serum that found the NulO biosynthetic genes were not involved in serum survival (59).

Modified legionaminic acid in bacterial lipopolysaccharides

Table 1  
Strains and plasmids used in this study  

| Strain/plasmid | Genotype/description | Refs. |
|----------------|-----------------------|-------|
| **V. vulnificus strains** |  |  |
| CMCP6 | Clinical isolate | 63 |
| YJ016 | Clinical isolate | 64 |
| JBL0803 | CMCP6Δnab1 (VV1_0803) | This study |
| JBL0808 | CMCP6Δnab2 (VV1_0808) | 41 |
| NMD0730/0803 | CMCP6ΔnanA/Δnab1 (VV2_0730/VV1_0803) | This study |
| CMCP6/T | Translucent colony morphology variant of CMCP6. Serum-sensitive | This study |
| JBL20730 | CMCP6ΔnanA (VV2_0730) | This study |
| SAM0316 | YJ016Δnab1 | This study |
| **E. coli strains** |  |  |
| DH5αλpir | Pirφ80dlacZΔM15δ(lacZYA-argF)U169recA1hsdR17deoRthi-1supE44gyrA96relA1 |  |
| B2155DAP | Donor for bacterial conjugation; thr-1004 pro thi strA hsdS lacZΔM15 (FlacTRQJ36proAB) dapAerm' pirRP4 (Km' from SM10) |  |
| **Plasmids** |  |  |
| pBBR1MCS | Expression vector; Cmr | 65 |
| pDS132 | Suicide plasmid; Cmr sacB | 66 |
| pDS2_0730 | V. vulnificus nanA knockout vector (VV2_0730 SOE product) |  |
| pDS0803 | V. vulnificus CMCP6 nab1 knockout vector (VV1_0803 SOE product) |  |
| pDS0316 | V. vulnificus YJ016 nab1 knockout vector (VV0316 SOE product) |  |
| pBBR0803 | V. vulnificus nab1 complement expression (VV0803C PCR product) | This study |
| pBBR0808 | V. vulnificus nab2 complement expression (VV0808C PCR product) | 41 |

The conserved ability to produce NulOs within *V. vulnificus* implies that these carbohydrates are essential for survival in the aquatic reservoir. Naturally, *V. vulnificus* can be commensal with phytoplankton, fish, and various crustaceans where in some cases the bacteria can be pathogenic. We show that NulO biosynthesis is required for virulence in the model crustacean organism *A. franciscana*. One explanation for the decreased virulence of the Leg5Ac7AcAla null strains is that there may be antimicrobial peptides produced by the *A. franciscana* brine shrimp which the NulO mutants are sensitive to resulting in their attenuation. Additionally, the mutated carbohydrates of LPS along with the fact that these strains are not fully motile may affect binding to the chitinous surface of the shrimp resulting in decreased virulence.

The NulOs described here and their carbohydrate modifications are highly prokaryote-specific. Two genes, *nab4* and *nab5*, which putatively catalyze the addition of D-alanine and acetylation of D-alanine, respectively, are unique in that D-alanine is predominantly limited to bacteria. The distinctive functions of NulO-modifying enzymes such as Nab4 and Nab5 as well as the significant role of NulOs in numerous pathogens makes the NulO biosynthetic proteins promising targets for antimicrobial therapies. Because of their prominence on surface structures, the NulO modifications such as the 7-N-acetyl-D-alanyl group described here can be targeted for vaccine development.

We have demonstrated that genetic diversity at the nonulosonic acid biosynthesis loci within *V. vulnificus* results in variable NulO structures ultimately altering the LPS. We chemically characterized a functionalized NulO, Leg5Ac7AcAla, in the pathogenic strain *V. vulnificus* CMCP6 and showed that modified legionaminic acid can be catabolized by the cell. Physiological characterizations of NulO mutants demonstrate how these carbohydrates contribute to biofilm formation, membrane integrity, and virulence toward crustaceans. These findings contribute to the ongoing efforts toward deconvoluting the production, structural elements, and processing of NulOs displayed by many species of bacteria. This information is critical

19122 J. Biol. Chem. (2018) 293(49) 19113–19126

Modified legionaminic acid in bacterial lipopolysaccharides

Table 2  
Primers used in this study  

| Primers           | Sequence 5′–3′                                                                                   | Product size |
|-------------------|--------------------------------------------------------------------------------------------------|--------------|
| SOEVV1_0803A      | TCTAGAAAAGCGTGCAACAAGTGGATATTCTCA                                                               | 241          |
| SOEVV1_0803B      | AACGGCACCTCTGCCCCCCGCT                                                                          |              |
| SOEVV1_0803C      | AGCGGGGGCAGAGGTGCCGTTAAGCTCGGTTGATATTGACCATGAG                                                   | 227          |
| SOEVV1_0803D      | GAGCTCGCCCCACTCCACTTGCTGA                                                                       |              |
| SOEVV1_0803FF     | GATTACTTCGGTGGCGAAAG                                                                            | 1063         |
| SOEVV1_0803FR     | GAATCAGGTAACGTGGTGCT                                                                           |              |
| SOEVV0316A        | TCTAGAACGGCGCTTTCCAATCACATCGG                                                                  | 251          |
| SOEVV0316B        | CACTATCGCGCCCCGATGCAC                                                                          |              |
| SOEVV0316C        | GTGCATCGGGCGCGATAGTGCCAAAAAGAGCGAGCGGTGGATA                                                     | 224          |
| SOEVV0316D        | GAGCTCAGAAAGGCGTTCACGCATCGC                                                                    |              |
| SOEVV0316FF       | GCCCAAATGGGATAAAAACC                                                                           | 1068         |
| SOEVV0316FR       | AGCTCGCAACAGCATAGCTT                                                                           |              |
| SOEVV2_0730A      | TCTAGAAATCCGCATCGCGCATTTC                                                                      | 247          |
| SOEVV2_0730B      | AATTACGCTGCGCCACTGTA                                                                           |              |
| SOEVV2_0730C      | TACAGTGGCGCAGCGTAATTATCAACGCCGAATGGAATATC                                                       | 240          |
| SOEVV2_0730D      | GAGCTCCTGATCGCCGCTCCTCATAC                                                                     |              |
| SOEVV2_0730FF     | TGCAGGGCACCTCTTTTTAT                                                                           | 996          |
| SOEVV2_0730FR     | GCTATTG CCTGAGCTGCTTT                                                                           |              |
| VV1_0803FC        | AAGCTTAAGGTGAATAATCGAATGAAA                                                                    | 744          |
| VV1_803RC         | TCTAGACGGCAACAAGAATAGTTTTA                                                                     |              |
| VV1_0808FC        | AAGCTTCGCAAGCACC AAAAGTTATT                                                                    | 1079         |
| VV1_0808FR        | TCTAGAGTGGCGGTGACTACACAGATT                                                                    |              |

2% plates supplemented with 10% sucrose to select for the second round of homologous recombination. The double cross-  
colonies were confirmed to be deletion mutants via PCR. Complement strains were generated similarly using primers listed in  
**Table 2** and cloned into the expression vector pBBR1MCS. Complement strains were induced with 0.05 mm isopropyl  
1-thio-β-D-galactopyranoside.

### DMB derivatization of *V. vulnificus* NulOs

*V. vulnificus* cultures were grown overnight in LB 2%. A  
1:100 dilution was made from overnight cultures into LB 2%  
NaCl, and cultures were grown for 4 h. Cells were pelleted and  
washed in PBS. Bacterial pellets were resuspended in 4 ml of 2 N  
acetic acid and incubated at 80 °C for 3 h as described previously (41, 44). Following the release of the NulOs via mild acid  
hydrolysis, cell debris was removed via centrifugation, and the  
soluble supernatant was passed through a 10-kDa molecular  
mass cutoff filter. The low molecular mass fraction (flow-  
through) containing NulOs was then derivatized with DMB  
(Dojindo Molecular Technologies), as described previously (39,  
44). In brief, the reaction consisted of 1.4 mM acetic acid, 0.7 M  
2-mercaptoethanol, 18 mM sodium hydrosulfite, and 7 mM  
DMB incubated in the dark at 50 °C for 2 h.

### TLC of separated NulOs

Thin layer chromatography (TLC) in which glass plates  
coated with silica gel (250 μm, Silica Gel HL, Sorbent Technologies) were used and visualized with shortwave 254 nm UV  
light or developed upon heating with *p*-anisaldehyde.

### HPLC and MS analysis of *V. vulnificus* NulOs

Analytical and semi-preparative HPLC was performed on an  
Agilent Series 1100 using a Phenomenex® Luna 5-μm C18 column (250 × 10.00 nm). Low-resolution mass spectra were  
obtained on a Shimadzu LCMS 2020 or a Waters SQD2 instrument using ESI. High-resolution mass spectra (HR-MS), ESI  
mode, were obtained on a ThermoFisher Scientific Q-Exactive  

Orbitrap at the Mass Spectroscopy Facility at the Department  
of Chemistry, University of Delaware.

### Nuclear magnetic resonance of purified NulOs

All NMR spectra were recorded on Bruker AV 400 MHz and  
AV III 600 MHz spectrometers. Proton chemical shifts were  
recorded in parts per million (ppm) on the δ scale, downfield  
from tetramethylsilane and referenced from an internal standard of residual protium in the NMR solvents (HDO: δ 4.87).  
Data for ¹³C NMR were reported in ppm downfield from  
tetramethylsilane. NMR data were reported as follows: chemical shift, multiplicity (s = singlet, d = doublet, t = triplet, q =  
quartet, m = multiplet, and br = broad), coupling constant in  
Hz, integration, and assignment based on two-dimensional  
COSY, HSQC, and HMBC experiments. NMR processing was  
done using MestReNova 2005–2017 Research SL Software.  
Chemical shifts of Leg5Ac7AcAla are as follows: ¹H NMR (600  
MHz, deuterium oxide) δ 5.09 (s, 1H), 5.00 (d, J = 6.2 Hz, 1H),  
4.94–4.84 (m, 3H), 4.52 (d, J = 10.8 Hz, 1H), 4.40–4.34 (m,  
2H), 4.34–4.26 (m, 3H), 4.19 (ddt, J = 19.1, 12.5, 6.7 Hz, 5H),  
4.11 (d, J = 6.2 Hz, 2H), 4.09–3.93 (m, 9H), 3.86 (ddq, J = 15.8,  
10.0, 5.9 Hz, 5H), 3.82–3.66 (m, 7H), 3.65–3.55 (m, 3H), 3.50  
(dd, J = 11.8, 6.5 Hz, 1H), 3.23 (dd, J = 13.8, 5.8 Hz, 1H), 3.14 (t,  
J = 7.8 Hz, 1H), 3.06 (t, J = 7.8 Hz, 1H), 2.11–2.00 (m, 14H), 1.98  
(d, J = 25.1 Hz, 8H), 1.94 (d, J = 5.4 Hz, 7H), 1.32 (t, J = 6.5 Hz,  
2H), 1.12 (s, 1H), 1.11 (d, J = 5.8 Hz, 4H); ¹³C NMR (101 MHz,  
D₂O) δ 174.92, 174.80, 173.95, 173.79, 173.64, 173.17, 98.94,  
98.43, 94.10, 78.06, 75.62, 72.85, 72.62, 71.93, 70.69, 70.45,  
70.30, 68.07, 67.02, 66.16, 65.76, 64.54, 63.68, 62.33, 53.52,  
53.42, 52.27, 50.10, 49.80, 48.62, 48.07, 45.80, 44.55, 36.41,  
36.23, 23.65, 21.99, 21.89, 21.80, 21.65, 21.31, 19.14, 16.86,  
16.48, 15.18.

### Purification of *V. vulnificus* lipopolysaccharide and isolation  
of NulO

*V. vulnificus* cultures were grown overnight in LB 2%. Cells  
were pelleted and washed in PBS followed by subsequent  
washes in ethanol and acetone to dry the bacterial pellets.

# Modified legionaminic acid in bacterial lipopolysaccharides

Lipopolysaccharide was isolated from dried bacterial pellets using the hot water phenol method as described previously (45). Pellets were resuspended in equal volumes of water and 90% phenol at 70 °C with stirring for 30 min. All phases were dia- lyzed to remove residual phenol. Proteins and nucleic acid were precipitated in 50% trichloroacetic acid (TCA) for 1 h at 4 °C. The precipitate was removed by centrifugation, and the LPS- containing fraction was lyophilized. Nonulosonic acids were removed by mild acid hydrolysis in 6% acetic acid for 3 h at 100 °C until a precipitate formed that was removed via centrif- ugation. The acid-hydrolyzed fraction was further separated via size-exclusion chromatography on Sephadex G-25. NulO con- taining fractions were combined and used for NMR analysis.

## Bacterial survival assays in sodium cholate and human serum

To assess the fitness of NulO mutants, survival assays were conducted as described previously (61). *V. vulnificus* strains were cultured for 2 h to mid-exponential phase in LB 2%. Cells were pelleted and washed with PBS at which time CFUs were determined prior to exposure to the stress. For bile survival assays, bacteria were pelleted and suspended in an equivalent volume of 15% sodium cholate in LB and incubated at 37 °C for 1 h. CFUs were enumerated at 0, 30, and 60 min post-exposure. Experiments were completed in triplicate with at least two bio- logical replicates. For serum survival assays, strains were grown for 2 h to ~4.25 × 10⁸ CFU/ml and were washed and res- pended in PBS. A total of 12 μl were added to 788 μl of either active human serum (Innovative Research Novi, MI) or heat- inactivated serum, which was incubated at 56 °C for 30 min, and CFUs were enumerated every 30 min for 2 h. Experiments were completed in triplicate with at least two biological replicates.

### Polymyxin B sensitivity

Exponential cells were washed and resuspended in PBS at an A of ~0.5 and added to a 96-well plate. Polymyxin B was added to a final concentration of 80 μg/ml. Cell lysis was monitored as a drop in optical density. The A₅₉₅ was measured every 5 min with 10 s of shaking prior to each measurement using a Sunrise microplate reader and Magellan plate reader software. Experi- ments performed in triplicate and two biological replicates.

### Biofilm assay

*V. vulnificus* strains were grown overnight in 2% LB. A 1:40 inoculum of the overnight cultures was used to inoculate Marine broth in 96-well polystyrene plates (Corning Inc., CLS2593). Cultures were incubated statically at room temper- ature for 24 h allowing the biofilms to form. Following incuba- tion, cultures were removed, and loosely adherent cells were washed twice with 200 μl of PBS. Biofilms were stained with 0.1% crystal violet for 30 min. The stain was removed, and bio- films were again washed twice with 200 μl of PBS. The stained biofilms were photographed and subsequently resuspended in dimethyl sulfoxide (DMSO) to solubilize the biofilm. The opti- cal density (A₅₉₅) was measured for 1:10 diluted biofilms. Bio- film determinations were performed with six replicate wells and three biological replicates.

### Statistical analysis

Biological and technical replicates are indicated under “Experimental procedures” and figure legends. The data were analyzed by Welch’s ANOVA followed by a Games-Howell post hoc test. Data are presented as mean values ± S.D. A *p* < 0.05 was considered significant.

---

### Author contributions—N. D. M. and E. F. B. conceptualization; N. D. M. and K. E. D. formal analysis; N. D. M. validation; N. D. M. investigation; N. D. M. and K. E. D. visualization; N. D. M. and K. E. D. methodology; N. D. M. and E. F. B. writing-original draft; N. D. M., E. F. B., K. E. D., C. L. G., and A. L. L. writing-review and editing; E. F. B. and C. L. G. resources; E. F. B. and C. L. G. supervi- sion; E. F. B. and C. L. G. funding acquisition; E. F. B. project administration.

---

### Acknowledgments—We thank Dr. Yiben Wang for technical help and Dr. Yuriy Knirel for advice on NulO isolation. We thank Dr. Papa Nii Asare-Okai (Director of the MS Facility) and Dr. Steve Bai (Department of Chemistry and Biochemistry, University of Delaware) for assis- tance in MS and NMR. For instrumentation support, the Delaware COBRE and INBRE Programs supported this project with National Institutes of Health Grants 5 P30 GM110758-02, P20GM104316- 01A1, and P20 GM103446 from NIGMS.

---

### References

1. Angata, T., and Varki, A. (2002) Chemical diversity in the sialic acids and related α-keto acids: an evolutionary perspective. Chem. Rev. 102, 439–469 [CrossRef Medline](https://doi.org/10.1021/cr000049t)
2. Severi, E., Hood, D. W., and Thomas, G. H. (2007) Sialic acid utilization by bacterial pathogens. Microbiology 153, 2817–2822 [CrossRef Medline](https://doi.org/10.1099/mic.0.2007/009287-0)
3. Almagro-Moreno, S., and Boyd, E. F. (2009) Insights into the evolution of sialic acid catabolism among bacteria. BMC Evol. Biol. 9, 118 [CrossRef Medline](https://doi.org/10.1186/1471-2148-9-118)

19124 J. Biol. Chem. (2018) 293(49) 19113–19126

4. Lewis, A. L., Desa, N., Hansen, E. E., Knirel, Y. A., Gordon, J. I., Gagneux, P., Nizet, V., and Varki, A. (2009) Innovations in host and microbial sialic acid biosynthesis revealed by phylogenomic prediction of nonulosonic acid structure. *Proc. Natl. Acad. Sci. U.S.A.* 106, 13552–13557 [CrossRef](https://doi.org/10.1073/pnas.0908385106) [Medline](https://pubmed.ncbi.nlm.nih.gov/19706509)

5. Vimr, E. R., Kalivoda, K. A., Deszo, E. L., and Steenbergen, S. M. (2004) Diversity of microbial sialic acid metabolism. *Microbiol. Mol. Biol. Rev.* 68, 132–153 [CrossRef](https://doi.org/10.1128/MMBR.68.1.132-153.2004) [Medline](https://pubmed.ncbi.nlm.nih.gov/15006920)

6. McDonald, N. D., Lubin, J. B., Chowdhury, N., and Boyd, E. F. (2016) Host-derived sialic acids are an important nutrient source required for optimal bacterial fitness *in vivo*. *MBio* 7, e02237-15 [Medline](https://pubmed.ncbi.nlm.nih.gov/27903656)

7. Almagro-Moreno, S., and Boyd, E. F. (2009) Sialic acid catabolism confers a competitive advantage to pathogenic *Vibrio cholerae* in the mouse intestine. *Infect. Immun.* 77, 3807–3816 [CrossRef](https://doi.org/10.1128/IAI.00173-09) [Medline](https://pubmed.ncbi.nlm.nih.gov/19403701)

8. King, S. J. (2010) Pneumococcal modification of host sugars: a major contributor to colonization of the human airway? *Mol. Oral Microbiol.* 25, 15–24 [CrossRef](https://doi.org/10.1111/j.2041-1014.2009.00540.x) [Medline](https://pubmed.ncbi.nlm.nih.gov/19840118)

9. Chang, D. E., Smalley, D. J., Tucker, D. L., Leatham, M. P., Norris, W. E., Stevenson, S. J., Anderson, A. B., Grissom, J. E., Laux, D. C., Cohen, P. S., and Conway, T. (2004) Carbon nutrition of *Escherichia coli* in the mouse intestine. *Proc. Natl. Acad. Sci. U.S.A.* 101, 7427–7432 [CrossRef](https://doi.org/10.1073/pnas.0402456101) [Medline](https://pubmed.ncbi.nlm.nih.gov/15128938)

10. Vimr, E. R., Aaronson, W., and Silver, R. P. (1989) Genetic analysis of chromosomal mutations in the polysialic acid gene cluster of *Escherichia coli* K1. *J. Bacteriol.* 171, 1106–1117 [CrossRef](https://doi.org/10.1128/JB.171.3.1106-1117.1989) [Medline](https://pubmed.ncbi.nlm.nih.gov/2538883)

11. Vimr, E., Steenbergen, S., and Cieslewicz, M. (1995) Biosynthesis of the polysialic acid capsule in *Escherichia coli* K1. *J. Ind. Microbiol.* 15, 352–360 [CrossRef](https://doi.org/10.1007/BF01569856) [Medline](https://pubmed.ncbi.nlm.nih.gov/7580385)

12. Parsons, N. J., Patel, P. V., Tan, E. L., Andrade, J. R., Nairn, C. A., Goldner, M., Cole, J. A., and Smith, H. (1988) Cytidine 5′-monophospho-N-acetyl-neuraminic acid and a low molecular weight factor from human blood cells induce lipopolysaccharide alteration in gonococci when conferring resistance to killing by human serum. *Microb. Pathog.* 5, 303–309 [CrossRef](https://doi.org/10.1016/0882-4010(88)90036-8) [Medline](https://pubmed.ncbi.nlm.nih.gov/3203628)

13. Silver, R. P., Aaronson, W., and Vann, W. F. (1988) The K1 capsular polysaccharide of *Escherichia coli*. *Rev. Infect. Dis.* 10, Suppl. 2, 282–286 [Medline](https://pubmed.ncbi.nlm.nih.gov/3047486)

14. Wessels, M. R., Pozsgay, V., Kasper, D. L., and Jennings, H. J. (1987) Structure and immunochemistry of an oligosaccharide repeating unit of the capsular polysaccharide of type III group B *Streptococcus*. A revised structure for the type III group B streptococcal polysaccharide antigen. *J. Biol. Chem.* 262, 8262–8267 DOI not found. [Medline](https://pubmed.ncbi.nlm.nih.gov/3597250)

15. Lewis, A. L., Robinson, L. S., Agarwal, K., and Lewis, W. G. (2016) Discovery and characterization of *de novo* sialic acid biosynthesis in the phylum Fusobacterium. *Glycobiology* 26, 1107–1119 [CrossRef](https://doi.org/10.1093/glycob/cww035) [Medline](https://pubmed.ncbi.nlm.nih.gov/27189788)

16. Inzana, T. J., Glindemann, G., Cox, A. D., Wakarchuk, W., and Howard, M. D. (2002) Incorporation of N-acetylneuraminic acid into *Haemophilus somnus* lipoooligosaccharide (LOS): enhancement of resistance to serum and reduction of LOS antibody binding. *Infect. Immun.* 70, 4870–4879 [CrossRef](https://doi.org/10.1128/IAI.70.8.4870-4879.2002) [Medline](https://pubmed.ncbi.nlm.nih.gov/12117954)

17. Kondadi, P. K., Rossi, M., Twelkmeyer, B., Schur, M. J., Li, J., Schott, T., Paulin, L., Auvinen, P., Hänninen, M. L., Schweda, E. K., and Wakarchuk, W. (2012) Identification and characterization of a lipopolysaccharide α,2,3-sialyltransferase from the human pathogen *Helicobacter bizzozeronii*. *J. Bacteriol.* 194, 2540–2550 [CrossRef](https://doi.org/10.1128/JB.06556-11) [Medline](https://pubmed.ncbi.nlm.nih.gov/22307255)

18. Carlin, A. F., Uchiyama, S., Chang, Y. C., Lewis, A. L., Nizet, V., and Varki, A. (2009) Molecular mimicry of host sialylated glycans allows a bacterial pathogen to engage neutrophil Siglec-9 and dampen the innate immune response. *Blood* 113, 3333–3336 [CrossRef](https://doi.org/10.1182/blood-2008-09-179888) [Medline](https://pubmed.ncbi.nlm.nih.gov/19147826)

19. Carlin, A. F., Lewis, A. L., Varki, A., and Nizet, V. (2007) Group B streptococcal capsular sialic acids interact with siglecs (immunoglobulin-like lectins) on human leukocytes. *J. Bacteriol.* 189, 1231–1237 [CrossRef](https://doi.org/10.1128/JB.01488-06) [Medline](https://pubmed.ncbi.nlm.nih.gov/17189346)

20. Ram, S., Sharma, A. K., Simpson, S. D., Gulati, S., McQuillen, D. P., Pangburn, M. K., and Rice, P. A. (1998) A novel sialic acid binding site on factor H mediates serum resistance of sialylated *Neisseria gonorrhoeae*. *J. Exp. Med.* 187, 743–752 [CrossRef](https://doi.org/10.1084/jem.187.5.743) [Medline](https://pubmed.ncbi.nlm.nih.gov/9480986)

21. Schneider, M. C., Exley, R. M., Chan, H., Feavers, I., Kang, Y. H., Sim, R. B., and Tang, C. M. (2006) Functional significance of factor H binding to *Neisseria meningitidis*. *J. Immunol.* 176, 7566–7575 [CrossRef](https://doi.org/10.4049/jimmunol.176.12.7566) [Medline](https://pubmed.ncbi.nlm.nih.gov/16751394)

22. Bax, M., Kuijf, M. L., Heikema, A. P., van Rijs, W., Bruijns, S. C., García-Vallejo, J. J., Crocker, P. R., Jacobs, B. C., van Vliet, S. J., and van Kooyk, Y. (2011) *Campylobacter jejuni* lipoooligosaccharides modulate dendritic cell-mediated T cell polarization in a sialic acid linkage-dependent manner. *Infect. Immun.* 79, 2681–2689 [CrossRef](https://doi.org/10.1128/IAI.01244-10) [Medline](https://pubmed.ncbi.nlm.nih.gov/21330513)

23. Chang, Y. C., Olson, J., Beasley, F. C., Tung, C., Zhang, J., Crocker, P. R., Varki, A., and Nizet, V. (2014) Group B *Streptococcus* engages an inhibitory Siglec through sialic acid mimicry to blunt innate immune and inflammatory responses *in vivo*. *PLoS Pathog.* 10, e1003846 [CrossRef](https://doi.org/10.1371/journal.ppat.1003846) [Medline](https://pubmed.ncbi.nlm.nih.gov/24586149)

24. Inzana, T. J., Balyan, R., and Howard, M. D. (2012) Decoration of *Histophilus somni* lipoooligosaccharide with N-acetyl-5-neuraminic acid enhances bacterial binding of complement factor H and resistance to killing by serum and polymorphonuclear leukocytes. *Vet. Microbiol.* 161, 113–121 [CrossRef](https://doi.org/10.1016/j.vetmic.2012.06.010) [Medline](https://pubmed.ncbi.nlm.nih.gov/22771586)

25. Tsvetkov, Y. E., Shashkov, A. S., Knirel, Y. A., and Zähringer, U. (2001) Synthesis and identification in bacterial lipopolysaccharides of 5,7-diacetamido-3,5,7,9-tetradeoxy-D-glycero-D-galacto- and -D-glycero-D-talo-non-2-ulosonic acids. *Carbohydr. Res.* 331, 233–237 [CrossRef](https://doi.org/10.1016/S0008-6215(01)00169-2) [Medline](https://pubmed.ncbi.nlm.nih.gov/11451268)

26. Knirel, Y. A., Vinogradov, E. V., L’Vov, V. L., Kocharova, N. A., Shashkov, A. S., Dmitriev, B. A., and Kochetkov, N. K. (1984) Sialic acids of a new type from the lipopolysaccharides of *Pseudomonas aeruginosa* and *Shigella boydii*. *Carbohydr Res.* 133, C5–C8 [CrossRef](https://doi.org/10.1016/S0008-6215(00)85140-2) [Medline](https://pubmed.ncbi.nlm.nih.gov/6746883)

27. Knirel, Y. A., Rietschel, E. T., Marre, R., and Zähringer, U. (1994) The structure of the O-specific chain of *Legionella pneumophila* serogroup 1 lipopolysaccharide. *Eur. J. Biochem.* 221, 239–245 [CrossRef](https://doi.org/10.1111/j.1432-1033.1994.tb18858.x) [Medline](https://pubmed.ncbi.nlm.nih.gov/8188970)

28. Kenyon, J. J., Marzaioli, A. M., De Castro, C., and Hall, R. M. (2015) 5,7-Di-N-acetyl-acetaminic acid: a novel non-2-ulosonic acid found in the capsule of an *Acinetobacter baumannii* isolate. *Glycobiology* 25, 644–654 [CrossRef](https://doi.org/10.1093/glycob/cwu168) [Medline](https://pubmed.ncbi.nlm.nih.gov/25533389)

29. Kenyon, J. J., Notaro, A., Hsu, L. Y., De Castro, C., and Hall, R. M. (2017) 5,7-Di-N-acetyl-8-epiacetaminic acid: a new non-2-ulosonic acid found in the K73 capsule produced by an *Acinetobacter baumannii* isolate from Singapore. *Sci. Rep.* 7, 11357 [CrossRef](https://doi.org/10.1038/s41598-017-09950-8) [Medline](https://pubmed.ncbi.nlm.nih.gov/28555027)

30. Vinogradov, E., St Michael, F., and Cox, A. D. (2017) The structure of the LPS O-chain of *Fusobacterium nucleatum* strain 25586 containing two novel monosaccharides, 2-acetamido-2,6-dideoxy-L-altrose and a 5-acetimidoylamino-3,5,9-trideoxy-glucono-non-2-ulosonic acid. *Carbohydr. Res.* 440, 10–15 [Medline](https://pubmed.ncbi.nlm.nih.gov/28189024)

31. Thibault, P., Logan, S. M., Kelly, J. F., Brisson, J. R., Ewing, C. P., Trust, T. J., and Guerry, P. (2001) Identification of the carbohydrate moieties and glycosylation motifs in *Campylobacter jejuni* flagellin. *J. Biol. Chem.* 276, 34862–34870 [CrossRef](https://doi.org/10.1074/jbc.M103458200) [Medline](https://pubmed.ncbi.nlm.nih.gov/11395534)

32. Schirm, M., Soo, E. C., Aubry, A. J., Austin, J., Thibault, P., and Logan, S. M. (2003) Structural, genetic and functional characterization of the flagellin glycosylation process in *Helicobacter pylori*. *Mol. Microbiol.* 48, 1579–1592 [CrossRef](https://doi.org/10.1046/j.1365-2958.2003.03495.x) [Medline](https://pubmed.ncbi.nlm.nih.gov/12753426)

33. Knirel, Y. A., Shashkov, A. S., Tsvetkov, Y. E., Jansson, P. E., and Zähringer, U. (2003) 5,7-Diamino-3,5,7,9-tetradeoxynon-2-ulosonic acids in bacterial glycopolymers: chemistry and biochemistry. *Adv. Carbohydr. Chem. Biochem.* 58, 371–417 [CrossRef](https://doi.org/10.1016/S0065-2318(03)58008-8) [Medline](https://pubmed.ncbi.nlm.nih.gov/14594209)

34. MacLean, L. L., Vinogradov, E., Pagotto, F., and Perry, M. B. (2011) Characterization of the lipopolysaccharide O-antigen of *Cronobacter turicensis* HPB3287 as a polysaccharide containing a 5,7-diacetamido-3,5,7,9-tetradeoxy-D-glycero-D-galacto-non-2-ulosonic acid (legionaminic acid) residue. *Carbohydr. Res.* 346, 2589–2594 [CrossRef](https://doi.org/10.1016/j.carres.2011.05.013) [Medline](https://pubmed.ncbi.nlm.nih.gov/21645688)

35. Twine, S. M., Paul, C. J., Vinogradov, E., McNally, D. J., Brisson, J. R., Mullen, J. A., McMullin, D. R., Jarrell, H. C., Austin, J. W., Kelly, J. F., and Logan, S. M. (2008) Flagellar glycosylation in *Clostridium botulinum*. *FEBS J.* 275, 4428–4444 [CrossRef](https://doi.org/10.1111/j.1742-4658.2008.06535.x) [Medline](https://pubmed.ncbi.nlm.nih.gov/18662012)

36. Friedrich, V., Janesch, B., Windwarder, M., Maresch, D., Braun, M. L., Megson, Z. A., Vinogradov, E., Goneau, M. F., Sharma, A., Altmann, F., Messner, P., Schoenhofen, I. C., and Schäffer, C. (2017) *Tannerella forsythia* strains display different cell-surface nonulosonic acids: biosynthetic

pathway characterization and first insight into biological implications. Glycobiology 27, 342–357 [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

37. Kurniyati, K., Kelly, J. F., Vinogradov, E., Robotham, A., Tu, Y., Wang, J., Liu, J., Logan, S. M., and Li, C. (2017) A novel glycan modifies the flagellar filament proteins of the oral bacterium Treponema denticola. Mol. Microbiol. 103, 67–85 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

38. Matthew, Z., and Kiefel, M. (2013) The occurrence and biological significance of the α-keto-sugars pseudaminic acid and legionaminic acid within pathogenic bacteria. RSC Adv. 4, 3413–3421

39. Lewis, A. L., Lubin, J. B., Argade, S., Naidu, N., Choudhury, B., and Boyd, E. F. (2011) Genomic and metabolic profiling of nonulosonic acids in Vibrionaceae reveal biochemical phenotypes of allelic divergence in Vibrio vulnificus. Appl. Environ. Microbiol. 77, 5782–5793 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

40. Vinogradov, E., Wilde, C., Anderson, E. M., Nakhamchik, A., Lam, J. S., and Rowe-Magnus, D. A. (2009) Structure of the lipopolysaccharide core of Vibrio vulnificus type strain 27562. Carbohydr. Res. 344, 484–490 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

41. Lubin, J. B., Lewis, W. G., Gilbert, N. M., Weimer, C. M., Almagro-Moreno, S., Boyd, E. F., and Lewis, A. L. (2015) Host-like carbohydrates promote bloodstream survival of Vibrio vulnificus in vivo. Infect. Immun. 83, 3126–3136 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

42. Yamamoto, M., Kashimoto, T., Tong, P., Xiao, J., Sugiyama, M., Inoue, M., Matsunaga, R., Hosohara, K., Nakata, K., Yokota, K., Oguma, K., and Yamamoto, K. (2015) Signature-tagged mutagenesis of Vibrio vulnificus. J. Vet. Med. Sci. 77, 823–828 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

43. Li, X., Perepelov, A. V., Wang, Q., Senchenkova, S. N., Liu, B., Shevelev, S. D., Guo, X., Shashkov, A. S., Chen, W., Wang, L., and Knirel, Y. A. (2010) Structural and genetic characterization of the O-antigen of Escherichia coli O161 containing a derivative of a higher acidic diamino sugar, legionaminic acid. Carbohydr. Res. 345, 1581–1587 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

44. Lewis, A. L., Nizet, V., and Varki, A. (2004) Discovery and characterization of sialic acid O-acetylation in group B Streptococcus. Proc. Natl. Acad. Sci. U.S.A. 101, 11123–11128 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

45. Westphal, O., and Jann, K. (1965) Extraction with Phenol-Water and Further Applications of the Procedure. Methods in Carbohydrate Chemistry (BeMiller, J. N., ed) pp. 83–91, Academic Press, New York

46. Schoenhofen, I. C., Vinogradov, E., Whitfield, D. M., Brisson, J. R., and Logan, S. M. (2009) The CMP-legionaminic acid pathway in campylobacter: biosynthesis involving novel GDP-linked precursors. Glycobiology 19, 715–725 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

47. Lubin, J. B., Kingston, J. J., Chowdhury, N., and Boyd, E. F. (2012) Sialic acid catabolism and transport gene clusters are lineage specific in Vibrio vulnificus. Appl. Environ. Microbiol. 78, 3407–3415 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

48. Naito, M., Frirdich, E., Fields, J. A., Pryjma, M., Li, J., Cameron, A., Gilbert, M., Thompson, S. A., and Gaynor, E. C. (2010) Effects of sequential Campylobacter jejuni 81–176 lipo oligosaccharide core truncations on biofilm formation, stress survival, and pathogenesis. J. Bacteriol. 192, 2182–2192 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

49. Davis, K. M., and Weiser, J. N. (2011) Modifications to the peptidoglycan backbone help bacteria to establish infection. Infect. Immun. 79, 562–570 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

50. Williams, T. C., Ayrapetyan, M., Ryan, H., and Oliver, J. D. (2014) Serum survival of Vibrio vulnificus: role of genotype, capsule, complement, clinical origin, and in situ incubation. Pathogens 3, 822–832 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

51. Anderson, G. G., Goller, C. C., Justice, S., Hultgren, S. J., and Seed, P. C. (2010) Polysaccharide capsule and sialic acid-mediated regulation promote biofilm-like intracellular bacterial communities during cystitis. Infect. Immun. 78, 963–975 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

52. Jurcisek, J., Greiner, L., Watanabe, H., Zaleski, A., Apicella, M. A., and Bakaletz, L. O. (2005) Role of sialic acid and complex carbohydrate biosynthesis in biofilm formation by nontypeable Haemophilus influenzae in the chinchilla middle ear. Infect. Immun. 73, 3210–3218 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

53. Greiner, L. L., Watanabe, H., Phillips, N. J., Shao, J., Morgan, A., Zaleski, A., Gibson, B. W., and Apicella, M. A. (2004) Nontypeable Haemophilus influenzae strain 2019 produces a biofilm containing N-acetylneuraminic acid that may mimic sialylated O-linked glycans. Infect. Immun. 72, 4249–4260 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

54. Lee, M. N., Kim, S. K., Li, X. H., and Lee, J. H. (2014) Bacterial virulence analysis using brine shrimp as an infection model in relation to the importance of quorum sensing and proteases. J. Gen. Appl. Microbiol. 60, 169–174 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

55. Ha, C., Kim, S. K., Lee, M. N., and Lee, J. H. (2014) Quorum sensing-dependent metalloprotease VvpE is important in the virulence of Vibrio vulnificus to invertebrates. Microb. Pathog. 71–72, 8–14

56. Kim, B. S., Jang, S. Y., Bang, Y. J., Hwang, J., Koo, Y., Jang, K. K., Lim, D., Kim, M. H., and Choi, S. H. (2018) QStatin, a selective inhibitor of quorum sensing in Vibrio species. MBio 9, e02262–17 [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

57. Hashii, N., Isshiki, Y., Iguchi, T., and Kondo, S. (2003) Structural characterization of the carbohydrate backbone of the lipopolysaccharide of Vibrio parahaemolyticus O-untypeable strain KX-V212 isolated from a patient. Carbohydr. Res. 338, 2711–2719 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

58. McNally, D. J., Aubry, A. J., Hui, J. P., Khieu, N. H., Whitfield, D., Ewing, C. P., Guerry, P., Brisson, J. R., Logan, S. M., and Soo, E. C. (2007) Targeted metabolomics analysis of Campylobacter coli VC167 reveals legionaminic acid derivatives as novel flagellar glycans. J. Biol. Chem. 282, 14463–14475 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

59. Carda-Diéguez, M., Silva-Hernández, F. X., Hubbard, T. P., Chao, M. C., Waldor, M. K., and Amaro, C. (2018) Comprehensive identification of Vibrio vulnificus genes required for growth in human serum. Virulence 9, 981–993 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

60. Horton, R. M., Hunt, H. D., Ho, S. N., Pullen, J. K., and Pease, L. R. (1989) Engineering hybrid genes without the use of restriction enzymes: gene splicing by overlap extension. Gene 77, 61–68 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

61. Haines-Menges, B., Whitaker, W. B., and Boyd, E. F. (2014) Alternative σ factor RpoE is important for Vibrio parahaemolyticus cell envelope stress response and intestinal colonization. Infect. Immun. 82, 3667–3677 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

62. Velkov, T., Soon, R. L., Chong, P. L., Huang, J. X., Cooper, M. A., Azad, M. A., Baker, M. A., Thompson, P. E., Roberts, K., Nation, R. L., Clements, A., Strugnell, R. A., and Li, J. (2013) Molecular basis for the increased polymyxin susceptibility of Klebsiella pneumoniae strains with under-acylated lipid A. Innate Immun. 19, 265–277 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

63. Kim, Y. R., Lee, S. E., Kim, C. M., Kim, S. Y., Shin, E. K., Shin, D. H., Chung, S. S., Choy, H. E., Progulske-Fox, A., Hillman, J. D., Handfield, M., and Rhee, J. H. (2003) Characterization and pathogenic significance of Vibrio vulnificus antigens preferentially expressed in septicemic patients. Infect. Immun. 71, 5461–5471 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

64. Chen, C. Y., Wu, K. M., Chang, Y. C., Chang, C. H., Tsai, H. C., Liao, T. L., Liu, Y. M., Chen, H. J., Shen, A. B., Li, J. C., Su, T. L., Shao, C. P., Lee, C. T., Hor, L. I., and Tsai, S. F. (2003) Comparative genome analysis of Vibrio vulnificus, a marine pathogen. Genome Res. 13, 2577–2587 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

65. Kovach, M. E., Phillips, R. W., Elzer, P. H., Roop, R. M., 2nd., and Peterson, K. M. (1994) pBBR1MCS: a broad-host-range cloning vector. BioTechniques 16, 800–802 [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)

66. Philippe, N., Alcaraz, J. P., Coursange, E., Geiselmann, J., and Schneider, D. (2004) Improvement of pCVD442, a suicide plasmid for gene allele exchange in bacteria. Plasmid 51, 246–255 [CrossRef](https://doi.org/) [Medline](https://www.ncbi.nlm.nih.gov/pubmed/)
