
Review

# Modulation of Innate Immune Responses by the Influenza A NS1 and PA-X Proteins

**Aitor Nogales** ${}^{1,2,*}$, **Luis Martinez-Sobrido** ${}^{1,\text{ID}}$, **David J. Topham** ${}^{1,3,\text{ID}}$ and **Marta L. DeDiego** ${}^{1,3,4,*\text{ID}}$

1. Department of Microbiology and Immunology, University of Rochester, Rochester, New York, NY 14642, USA; Luis_martinez@urmc.rochester.edu (L.M.-S.); David_Topham@urmc.rochester.edu (D.J.T.)
2. Centro de Investigación en Sanidad Animal (CISA)-INIA, Valdeolmos, 28130 Madrid, Spain
3. David H. Smith Center for Vaccine Biology and Immunology, University of Rochester, Rochester, New York, NY 14642, USA
4. Department of Molecular and Cell Biology, Centro Nacional de Biotecnología (CNB-CSIC), Campus Universidad Autónoma de Madrid, 28049 Madrid, Spain
* Correspondence: Aitor_nogales@urmc.rochester.edu (A.N.); Marta.Lopez@cnb.csic.es (M.L.D.)

Received: 22 November 2018; Accepted: 8 December 2018; Published: 12 December 2018

---

**Abstract:** Influenza A viruses (IAV) can infect a broad range of animal hosts, including humans. In humans, IAV causes seasonal annual epidemics and occasional pandemics, representing a serious public health and economic problem, which is most effectively prevented through vaccination. The defense mechanisms that the host innate immune system provides restrict IAV replication and infection. Consequently, to successfully replicate in interferon (IFN)-competent systems, IAV has to counteract host antiviral activities, mainly the production of IFN and the activities of IFN-induced host proteins that inhibit virus replication. The IAV multifunctional proteins PA-X and NS1 are virulence factors that modulate the innate immune response and virus pathogenicity. Notably, these two viral proteins have synergistic effects in the inhibition of host protein synthesis in infected cells, although using different mechanisms of action. Moreover, the control of innate immune responses by the IAV NS1 and PA-X proteins is subject to a balance that can determine virus pathogenesis and fitness, and recent evidence shows co-evolution of these proteins in seasonal viruses, indicating that they should be monitored for enhanced virulence. Importantly, inhibition of host gene expression by the influenza NS1 and/or PA-X proteins could be explored to develop improved live-attenuated influenza vaccines (LAIV) by modulating the ability of the virus to counteract antiviral host responses. Likewise, both viral proteins represent a reasonable target for the development of new antivirals for the control of IAV infections. In this review, we summarize the role of IAV NS1 and PA-X in controlling the antiviral response during viral infection.

**Keywords:** influenza a virus; NS1; PA; PA-X; interferon; host antiviral response; innate immunity; cytokines; virus–host interactions; pathogenesis; virus evolution; vaccine

---

## 1. Introduction

### 1.1. Influenza A Virus (IAV)'s Relevance to Human Health

IAV are members of the Orthomyxoviridae family of enveloped viruses, which contain an eight-segmented, single-stranded (ss), negative-sense RNA genome [1]. IAV can infect multiple hosts, including birds, pigs, dogs, horses, bats, and humans [2–5]. These viruses undergo reassortment in wild hosts, leading to the emergence of novel strains with epidemic or pandemic potential in humans. In fact, IAV is one of the main causes of respiratory infections in humans, and are responsible

---

Viruses 2018, 10, 708; doi:10.3390/v10120708  
www.mdpi.com/journal/viruses

for seasonal epidemics each year and occasional pandemics of great consequence, such as the 1918 "Spanish flu" [2,6,7]. IAV is classified in subtypes according to the hemagglutinin (HA; H1–H18) and neuraminidase (NA; N1–N11) sequences [8], with 18 HA subtypes and 11 NA subtypes currently circulating [4,9–11]. In humans, the most frequent seasonal IAV circulating nowadays are the H3N2 and the H1N1 subtypes of IAV [12,13]. However, human cases of the highly pathogenic H5N1 avian IAV continue to occur throughout parts of East and Southeast Asia, the Middle East, Africa, and Europe [9,14]. In addition, outbreaks of H6, H7, H9, and H10 IAV in poultry and zoonotic infections of humans with these viral strains have been reported [9,14]. Fortunately, these antigenically novel IAV have yet to sustain human-to-human transmission, and have, therefore, failed to generate a potentially devastating human pandemic [9,14]. The first IAV pandemic in the 21st century started in 2009 with the emergence of a quadruple-reassortant swine-origin H1N1 IAV (pH1N1) [15,16], which, in less than one year, infected more than 600,000 individuals around the world, although this number is likely underestimated. Importantly, this virus is efficiently transmitted among humans and continues circulating seasonally [12,17,18]. Although comprehensive vaccination programs are in place, the global disease burden from seasonal influenza results in 1 billion infections, 3–5 million cases of severe disease, and between 250,000 and 500,000 deaths annually, according to the World Health Organization (WHO) [7,19], leading also to a significant worldwide economic impact.

### 1.2. Interferon and Inflammatory Cytokine Responses during IAV Infection

The host innate immune system restricts IAV replication [20]. When IAV infects a cell, pathogen-associated molecular patterns (PAMPs) are recognized by pattern recognition receptors (PRRs). This recognition activates signaling pathways, leading to the production of type I (IFNα and IFNβ) and III (IFNλ) interferons (IFNs) and cytokines involved in inflammatory processes [20]. IAV double-stranded (ds) or single-stranded (ss) RNAs are recognized by the membrane-associated PRR Toll-like receptors (TLRs) 3 or 7 and 8, respectively [20,21], and by the cytoplasmic PRR retinoic acid-inducible gene I (RIG-I) that detect double-stranded (ds)RNA and 5′-triphosphates of the (-) single-stranded (ss)RNA viral genome [20,22] generated during virus replication. The 5′ and 3′ ends of the influenza’s viral RNAs contain partially complementary sequences that are able to form short 5′ppp-dsRNA structures [23]. The consequence of TLRs and RIG-I detection of IAV PAMPs is the activation of transcription factors, such as IFN regulatory factors (IRF) 3 and 7, the nuclear factor kappa β (NF-κB), and the activator protein 1 (AP-1) transcription factors, that are responsible for the transcription of type I and type III IFN [20,24,25] and pro-inflammatory cytokines [26]. Secreted type I and III IFNs act in the same cell or in surrounding cells to induce the expression of IFN-stimulated genes (ISGs), many of which display antiviral activity [20,25,27]. Type I and III IFNs signal through different receptors; however, the pathways converge in the phosphorylation of the signal transducer and activator of transcription (STAT) 1 and STAT2 transcription factors [28]. Phosphorylated forms of STAT1 and STAT2 then associate with IRF9 to form a heterotrimeric ISG factor 3 (ISGF3) complex [28]. ISGF3 then translocates to the nucleus, where it binds to IFN-stimulated response elements (ISREs) sequences present in the promoter of ISGs to upregulate their transcription [28,29]. The Janus protein tyrosine kinase 1 (JAK1) and tyrosine kinase 2 (TYK2) are critical for phosphorylation and activation of STAT1 and STAT2 [30]. In addition, STAT1 is phosphorylated by IKKε during IFN signaling, and this step is critical for the IFN-inducible antiviral response [31,32].

The inflammasome is a molecular complex formed by the proteins Nucleotide oligomerization domain (NOD)-like receptor family member LRR- and Pyrin domain containing-3 (NLRP3), Apoptosis-associated Speck-like containing a caspase-recruitment domain (ASC), and pro-caspase 1 [33]. NLRP3 is expressed by myeloid cells, such as monocytes, macrophages, neutrophils, and dendritic cells [34], and also by human bronchial epithelial cells [35]. The inflammasome activation involves the sensing of the PAMPs by NLRP3. Moreover, protein flux through the viral matrix protein 2 (M2) ion channel in the trans-Golgi network triggers NLRP3 activation [36]. In addition, it has been shown that IAV PB1-F2 [37] and dsRNA [38] activate the inflammasome, which stimulates the activation of

caspase-1. Active caspase-1 cleaves inactive pro-interleukin (IL)-1β and pro-IL-18 into their mature forms IL-1β and IL-18, respectively, which are released from the cell to stimulate inflammatory processes [39].

### 1.3. ISGs against IAV Infections

Many ISGs, such as myxovirus resistance (Mx), IFN-induced transmembrane (IFITM), protein kinase R (PKR), and 2′-5′ oligoadenylate synthetase (OAS)-ribonuclease L (RNaseL), limit IAV replication [20,40]. In mice, Mx1 is expressed in the nucleus and displays antiviral activity [41] by blocking viral mRNA elongation [42,43], whereas Mx2 is expressed in the cytoplasm and does not show antiviral activity. In humans, the MxA (also known as Mx1) protein is cytosolic, and has potent antiviral activity against IAV [44]. Inhibition of IAV by MxA involves binding to the viral nucleoprotein (NP) and the inhibition of viral transcription [44,45]. By contrast, the MxB (also known as Mx2) protein is present at the cytoplasmic face of nuclear pore complexes and it does not inhibit IAV infection [46].

Members of the interferon-induced transmembrane (IFITM) proteins family are able to prevent infection before a virus can traverse the lipid bilayer of the cell. At least three human IFITM proteins (IFITM1, IFITM2, and IFITM3) have been described to have antiviral activity by blocking virus–host cell membrane fusion following viral attachment and endocytosis [47,48]. Cumulative evidence indicates that IFITM1 is localized predominantly at the plasma membrane, while IFITM2 and 3 localize to endosomal and lysosomal compartments, respectively [49].

Activation of OAS by dsRNA produces poly(A) chains with 2′-5′ phosphodiester bonds [50], which bind to and activate constitutively expressed RNaseL, leading to the cleavage of viral and cellular ssRNA, resulting in the inhibition of virus replication [51]. In addition, it has been shown that some of the viral ssRNA degradation products bind to and activate RIG-I, leading to an enhanced activation of IFN transcription [22,52]. Activity of the OAS/RNase L pathway was also demonstrated in vitro against many viruses, in particular RNA viruses [53]. However, some RNA viruses, such as IAV, are hardly affected, because they express antagonist proteins (see Section 2.2).

Protein kinase R (PKR) is activated by dsRNA or by the cellular protein activator of the interferon-induced PKR (PACT), resulting in autophosphorylation and phosphorylation of cellular proteins, including the α subunit of the eukaryotic initiation factor 2 (eIF2α) [54]. This phosphorylation leads to inhibition of protein synthesis, including viral proteins, in infected cells. In addition, PKR activates the transcription factor NF-κB by phosphorylating IkB [55]. Furthermore, PKR stabilizes type I IFN mRNA, thereby ensuring robust IFN protein production [56]. It has been shown that PKR-deficient mice are more susceptible to various viral infections, including IAV, further demonstrating the remarkable antiviral activity of this ISG product [57].

## 2. The IAV Non-Structural 1 (NS1) Protein

### 2.1. NS1 Protein Introduction

The IAV genome segment 8 encodes the non-structural (NS) mRNA as a continuous primary transcript [58] from which NS1 is synthesized. In addition, alternative processing of this transcript using a weak 5′ splice site results in a less abundant spliced product encoding the nuclear export protein (NEP) [58]. This second transcript accounts for 10–15% of the NS-derived mRNA (Figure 1A) [59]. NS1 is most often a 230 amino acid protein [60]. However, mutations that either suppress the stop codon at position 231 or create a premature stop codon result in length variations [60]. For instance, from the late 1940s until the middle of the 1980s, NS1 of human IAV harbored the C-terminus, seven-amino-acids extension 231-RRNKMAD-237 [60,61]. Conversely, the NS1 protein of the 2009 pH1N1, like that of most swine H1N1 IAV, has only 219 amino acids [60–62]. The IAV NS1 protein contains at least four distinct domains: (i) The N-terminal domain (the first 73 amino acids) contains an RNA-binding domain (RBD) [60] (Figure 1B), which forms three α-helices important for the dimerization of the protein and for the binding to dsRNA [63–65]. It was shown that residues at positions 38 and 41 of the NS1 RBD

affect the binding to dsRNA without affecting NS1 dimerization [65,66]. A nuclear localization signal (NLS) (amino acids 35–41) overlaps with the RBD sequence [67,68]. In addition, most IAV strains encode a second NLS at the C-terminal end (amino acids 216–221) of the protein [67]. (ii) The RBD is followed by a 10–15 amino acids linker (L) domain (Figure 1B), which has high flexibility and connects the RBD and effector domain (ED). (iii) The 88–202 ED (Figure 1B), which contains a β-sheet structure and a long central helix. The ED of most IAV strains contains a nuclear export signal (NES, amino acids 137–146) that favors NS1 protein localization at both the nucleus and the cytoplasm [69]. Notably, more than 50 host proteins have been reported to interact with the NS1 ED (Table 1 and data not shown, reviewed in [70]), although some of these interactions do not occur with all of the NS1 proteins encoded by different IAV strains, and it has been suggested that host-adaptation processes can be responsible for this variability [60,62,71–74]. Those host proteins include, among others, the 30 kDa subunit of cleavage and polyadenylation specificity factor (CPSF30), the poly(A)-binding protein II (PABPII), the eukaryotic initiation factor 4G (eIF4GI), the poly(A)-binding protein I (PABPI), and multiple host factors involved in antiviral response, such as RIG-I, E3 ligase tripartite motif-containing protein 25 (TRIM25), PKR, or the regulatory subunit p85-β of phosphatidylinositol 3-kinase (PI3K) (reviewed in [70], Table 1). Finally, (iv) the last domain contains a C-terminal tail (CTT) of 11–33 amino acids, which is intrinsically disordered, and includes a PDZ-binding motif, which affects virus pathogenesis and is not present in all IAV NS1 proteins [75,76] (Figure 1B).

A)

![Diagram A](attachment://diagram_a.png)

B)

![Diagram B](attachment://diagram_b.png)

Figure 1. A schematic representation of an influenza A virus (IAV) non-structural (NS) segment, viral transcripts, and NS1 domains. (A) An IAV NS RNA segment is indicated by a blue box and non-coding regions (NCR) are indicated with white boxes. IAV NS1 and nuclear export protein (NEP) transcripts are indicated with yellow and red boxes, respectively. IAV NS1 and NEP open reading frames (ORFs) shared the first 30 nucleotides in the N-terminus. The numbers on the top of the bars represent the ORF length and nucleotide splice positions. (B) The NS1 protein is divided into four distinct regions: The N-terminal RNA-binding domain (RBD; amino acids 1–73, yellow), the linker sequence (L; amino acids 74–88, red), the effector domain (ED; amino acids 89–202, blue), and the C-terminal tail (CTT; amino acids 203 to the end, gray). Note that both the L and the CTT can vary in length among different IAV strains, and, although a 237 amino-acids-length NS1 has been represented, the NS1 can be 219, 230, and 237 amino acids in length. Nuclear localization and export signals (NLS and NES, respectively) are indicated with black boxes at the bottom.

### 2.2. Mechanisms of IAV NS1 Protein Inhibition of Innate Immune Responses

The IAV NS1 protein allows the virus to replicate efficiently by suppressing the host innate immune response by a variety of mechanisms (reviewed in [60,70,77]) (Figure 2), including the direct or indirect interaction with host factors (selected examples are shown in Table 1). Accordingly,

IAVs lacking or expressing truncated forms of NS1 [78–81], expressing reduced levels of NS1 [82], or encoding amino acid mutations affecting NS1 functions [72, 82–84] are severely impaired in cells competent in the production of type I IFN, whereas they show similar levels of replication in type I IFN-deficient cells (e.g., Vero cells) [78]. Thus, IAV encoding NS1 proteins with an altered anti-IFN function lack virulence in IFN-competent animals, but are pathogenic in mice defective in key proteins of the type I IFN pathway.

The different NS1 protein domains are involved in regulating the host innate immune response, and in this review we will summarize some of the most characteristic functions. It has been shown that the NS1 RBD mutations R38A and K41A inhibit the binding of NS1 protein to dsRNA [65], a function critical for counteraction to innate immune responses, leading to virus attenuation [73, 85]. By using primary differentiated respiratory epithelial cell cultures infected with a recombinant virus encoding a mutation in the RBD (R38A), it was revealed that the RBD is a critical regulator of both cytokine production and cytokine sensitivity during IAV infection [86]. Furthermore, using the PR8 backbone and the R38A mutant virus, it was shown that a functional RBD is required for IFN suppression and virulence in mice [73]. Moreover, IAV NS1, via its RBD, inhibits OAS activation because it outcompetes OAS for interaction with dsRNA [87] (Figure 2D and Table 1).

The molecular mechanisms affecting NS1’s ability to counteract innate immune responses include inhibiting RIG-I activation by sequestration of this RNA helicase and its activating ligand [88–90], or by interaction with TRIM25 or Riplet, which results in the suppressed ubiquitination and activation of RIG-I [91–93] (Figure 2A and Table 1). Notably, IAV encoding NS1 proteins carrying amino acid changes E96A/E97A in a putative protein-protein interaction motif in the ED domain are unable to bind TRIM25, causing virus attenuation and higher IFN induction [91]. However, these amino acid changes are not involved in the inhibition of RIG-I activation mediated by Riplet [92]. Additionally, NS1 indirectly regulates RIG-I signaling by increasing the expression of the ubiquitin-editing protein A20 after infection (Table 1). A20 suppresses IRF3-mediated induction of type I IFN and ISGs [94, 95].

### Table 1. Interactions of the IAV NS1 protein with cellular proteins that are involved in innate immune responses.

| Host Factor<sup>a</sup> | NS1 Domain<sup>b</sup>       | Protein Function<sup>c</sup>                                                                                   | Reference     |
|------------------------|-----------------------------|---------------------------------------------------------------------------------------------------------------|---------------|
| TRIM25                 | RBD and ED                  | Ubiquitin E3 ligase and ISG15 E3 ligase. Modification of RIG-I by poly-ubiquitin chains synthesized by TRIM25 is essential for RIG-I activation | [91, 92]      |
| Riplet                 | RBD and ED                  | E2-dependent E3 ubiquitin-protein ligase. Modification of RIG-I by poly-ubiquitin chains synthesized by Riplet is essential for RIG-I activation | [92]          |
| RIG-I                  | RBD                         | Cytoplasmic PRR of viral nucleic acids, activating innate immune responses                                          | [96]          |
| hPAF1C                 | CTT                         | Transcription of Hox and Wnt genes and histone modifications                                                     | [97]          |
| IKKβ                   | CTT                         | Serine kinase involved in NF-κB activation and IFN and proinflammatory responses                                  | [98]          |
| IRF3                   | Indirect effect             | Transcriptional regulator factor of type I IFN-dependent immune responses                                         | [99]          |
| AP-1                   | Indirect effect             | Transcription factor involved in IFN and pro-inflammatory cytokines induction                                    | [100]         |
| A20                    | Indirect effect             | Ubiquitin-editing enzyme that contains both ubiquitin ligase and deubiquitinase activities. Suppresses IRF3-mediated IFN induction | [94, 95]      |
| PKR                    | RBD and ED                  | IFN-induced dsRNA-dependent serine/threonine-protein kinase leading to protein translation inhibition               | [101–103]     |
| RNase L                | Indirect effect             | Endoribonuclease leading to host gene expression inhibition by mRNA degradation                                   | [87]          |

Table 1. Cont.

| Host Factor<sup>a</sup> | NS1 Domain<sup>b</sup> | Protein Function<sup>c</sup> | Reference |
|---------------------|-----------------------|---------------------------------------------|-----------|
| PACT               | NS1                   | Activation of EIF2AK2/PKR                  | [101,104] |
| OAS                | RBD and indirect effect | Polymerizes higher oligomers of 2'-5'-oligoadenylates that bind to RNaseL leading to its activation | [51,87]   |
| NLRP3              | ED and indirect effect | Sensor component of the NLRP3 inflammasome, leading to inflammatory responses | [105-107] |
| CPSF30             | RBD and ED            | Processing of mRNAs, necessary for host gene protein expression | [62,72,74,108-110] |
| PABP II            | CTT                   | Formation of mRNA precursors adding a poly(A) tail, necessary for gene expression | [111,112] |
| eIF4GI             | RBD and ED            | Recognition of the mRNA cap, ATP-dependent unwinding of 5'-terminal secondary structure and recruitment of mRNA to the ribosome. | [113]     |
| p85β               | ED                    | PI3K subunit                               | [114,115] |
| NXF1               | RBD and ED            | Nuclear export of mRNA                     | [116]     |
| RAE1               | RBD and ED            | mRNA nucleocytoplasmic transport           | [116]     |
| P15                | ED                    | Nuclear export of mRNA                     | [116]     |
| E1B-AP5            | RBD and ED            | Transcriptional regulator                  | [116]     |

<sup>a</sup> TRIM25: E3 ligase tripartite motif-containing protein 25; RIG-I: pattern recognition receptor (PRR) retinoic acid-inducible gene I; hPAF1C: human PAF1 complex; IKK-β: inhibitor of kappa β kinase; IRF-3: Interferon regulatory factor; PKR: protein kinase R; PACT: protein activator of the interferon-induced PKR; OAS: 2'-5'-oligo A synthetase; NLRP3: NOD-like receptor family member LRR-and Pyrin domain containing-3; CPSF30: 30kDa subunit of cleavage and polyadenylation specificity factor; PABP II: poly(A)-binding protein II; eIF4G: eukaryotic initiation factor 4G; NXF1: nuclear RNA export Factor 1; RAE1: Ribonucleic Acid Export 1; E1B-AP5: adenovirus early region 1B-associated protein 5. <sup>b</sup> RBD: Receptor binding domain; ED: effector domain; CTT: C-terminus tail; <sup>c</sup> Protein function description. IFN; interferon.

dependent on Y89/M93 [114], L141/E142 [115], and P164/P167 [122] of the effector domain, all of which are located adjacent to each other within a cleft between the two NS1 monomers.

Figure 2. Direct and indirect effects of the IAV NS1 protein on innate immune responses. (A) IAV NS1 decreases RIG-I activation, and IFN responses, through the sequestration of dsRNA [88–90], or by interaction with TRIM25 or Riplet, which results in the suppressed ubiquitination and activation of RIG-I, which signals through the mitochondrial antiviral signaling protein (MAVS) to induce IFN responses [91,92]. (B) NS1 inhibits the IRF3 [99], NF-κβ [117], and AP-1 [100] transcription factors, impairing IFN production, and, therefore, the induction of IFN-stimulated gene (ISG) products. In addition, NS1 directly inhibits the antiviral activities of the ISGs PKR and OAS-RNaseL. (C) The IAV NS1 protein binds dsRNA and PKR, leading to decreased PKR activity to phosphorylate eIF2α, and host translation inhibition [101–103]. (D) The IAV NS1 protein, via the dsRNA-binding activity of its RBD, inhibits OAS activation, blocking RNA degradation [87]. (E) The IAV NS1 protein also inhibits NLRP3 inflammasome activation [105–107], impairing the cleavage of pro-interleukin (IL)-1β and pro-IL-18 into their mature forms IL-1β and IL-18, respectively, which are released from the cell to stimulate inflammatory processes. (F) NS1 proteins from some human and avian IAV strains bind to CPSF30, blocking the cleavage of immature mRNAs (pre-mRNAs) and the recruitment of the poly(A) polymerase to add the poly(A) tail [62,72,74,108–110,123]. The IAV NS1 protein also binds to the poly(A)-binding protein II (PABPII), inhibiting its ability to stimulate the synthesis of long poly(A) tails [111]. These last two processes lead to host shutoff of protein synthesis [111]. (G) Additionally, the NS1 of influenza A/WSN/33 H1N1 (WSN) binds NXF1, p15, RAE1, and E1B-AP5, which interact with both mRNAs and nucleoporins to direct mRNAs through the nuclear pore complex, blocking their function, and likely facilitating host cellular shutoff [116].

### 2.3. Effect of the IAV NS1 Protein on Inhibition of Gene Expression

NS1 proteins from some human and avian IAV strains bind to CPSF30, blocking the processing of cellular mRNAs [62,72,74,108–110,124] (Figure 2F and Table 1). The region of IAV NS1 that binds CPSF30 is largely hydrophobic, and is primarily defined by the amino acid residues K110, I117, I119,

Q121, V180, G183, G184, and W187 [108], with W187 being a hydrophobic amino acid that is essential for dimerization of the NS1 ED [108]. The NS1 binding to CPSF30 inhibits the recognition by the CPSF complex of polyadenylation signals at the 3′ end of mRNAs during transcription, blocking the cleavage of immature mRNAs (pre-mRNAs) and the recruitment of the poly(A) polymerase to add the poly(A) tail [123,124]. The poly(A) tail of eukaryotic mRNAs is required for RNA stability, nucleus export, and translation [123]. As a consequence, unprocessed cellular pre-mRNAs accumulate in the nucleus (Figure 2F), leading to an inhibition of general gene expression, including IFN, ISGs encoding antiviral activities, and pro-inflammatory genes [70].

**A)**

![Image A](image_a.png)

**B)**

![Image B](image_b.png)

Figure 3. The tridimensional structure of the IAV NS1 ED coupled to the F2/F3 domain of CPSF30. A/Udorn/72 H3N2 strain NS1 ED bound to the F2/F3 fragment of the human CPSF30 was previously crystallized [108] (protein data bank (PDB) entry 2RHK). Colors were included using the MacPyMOL Molecular Graphics system (pymol.org). Each monomer of the NS1 ED is represented in green colors. Monomers of the F2/F3 fragment of the human CPSF30 are represented in blue colors. The artificially introduced NS1 amino acid residues 108, 125, and 189 restoring NS1–CPSF0 binding (A) [125], and the residues 90, 123, 125, and 131 found in naturally circulating pH1N1 viruses (B) [126], are indicated in reddish colors (orange, purple, and pink). Figure adapted from [126].

The IAV NS1 protein also binds to the poly(A)-binding protein II (PABPII) (Figure 2F and Table 1), inhibiting the ability of PABPII to stimulate the synthesis of long poly(A) tails [111]. Because of this interaction, cellular pre-mRNAs that contain short poly(A) tails (around 12 nucleotides) accumulate in the nucleus of IAV-infected cells [111] (Figure 2F). The binding of the IAV NS1 protein to CPSF30 and PABPII block pre-mRNA processing and the nuclear export of mRNAs, therefore leading to a general inhibition of host gene expression [70,110,111].

The IAV NS1 protein also enhances the translational rate of viral, but not cellular, mRNAs [113]. Using A/Victoria/3/75 H3N2, it was shown that NS1 binds the eukaryotic initiation factor 4GI (eIF4GI) (Table 1), the large subunit of the cap-binding complex eIF4F, suggesting that NS1 recruits eIF4GI specifically to the 5′ untranslated region (5′ UTR) of the viral mRNA, allowing for the preferential translation of the IAV mRNAs, which likely leads, indirectly, to host cellular shutoff [113].

The NS1 protein of the classical swine H1N1 IAV encoded 230 amino acids until the mid-1960s, when a stop codon emerged at position 220 and caused an 11 amino acid truncation at the C-terminus [127]. This 219 amino acid NS1 has subsequently been predominant in the classical swine H1N1, and emerged in the 2009 pH1N1 IAV [112]. The C-terminal-truncated NS1 of the 2009 pH1N1 virus was inefficient at blocking host gene expression, while an extension of the truncated NS1 to its full-length 230 amino acid form increased the ability of the protein to inhibit host gene expression [112]. The mechanism that mediates the increased inhibition of host gene expression by the full-length NS1 involved the restoration of NS1’s binding to PABPII [112]. In vitro and in vivo characterization of two 2009 pH1N1 viruses encoding the C-terminal 11 amino acid truncated 220 amino acids and

the full-length 230 amino acid NS1 showed that the C-terminal 11 amino acid truncation did not significantly affect virus replication. However, this truncation increased virus pathogenicity in mice by causing more severe hemorrhage and alveolitis in the lungs [112].

The H3N2 IAV circulating in humans from ~1950 to ~1987 encoded an NS1 protein with a 7 amino acids C-terminal extension (237 amino acids in total). No significant differences in the ability to counteract IFN responses were observed between proteins encoding the NS1 230 and 237 amino acid proteins [61]. However, the virus encoding the 237 amino-acid-length NS1 protein outcompeted the mutant virus during mixed infections, suggesting that the 7 amino acid extension in NS1 slightly affected the growth or fitness of the virus [61].

Recently, it was found that the NS1 protein can bind cellular dsDNA, preventing the loading of the transcriptional machinery to the DNA, decreasing the expression of antiviral genes [128]. The residues R38 and K41 in the NS1 RBD are important to this binding [128].

Additionally, the NS1 of the IAV WSN binds many components of the mRNA's export machinery: nuclear RNA export Factor 1 (NXF1), p15, Ribonucleic Acid Export 1 (RAE1), and adenovirus early region 1B-associated protein 5 (E1B-AP5), which interact with both mRNAs and nucleoporins to direct mRNAs through the nuclear pore complex, blocking their function [116] (Figure 2G and Table 1). These data indicate different cellular mechanisms mediating the inhibition or regulation of host gene expression by the IAV NS1 protein.

## 2.4. Effect of IAV NS1 on Virus Replication, Induction of Cytokines, and Pathogenesis

### 2.4.1. Effect of Partial and Complete NS1 Deletions on Virus Replication, Induction of Cytokines, and Pathogenesis

Viruses encoding truncated NS1 proteins are severely impaired in replication in cultured cells, and are highly attenuated in vivo, because of defects of the truncated NS1 proteins to counteract IFN responses. Examples include a PR8 H1N1 virus encoding the first 99 or 126 amino acids of NS1 [81, 129, 130], H5N1 viruses encoding the first 73, 99, and 126 amino acids of NS1 [131], an A/Texas/36/91 H1N1 virus encoding the first 126 amino acids of NS1 [132], and equine Influenza virus (EIV) [133, 134] and canine Influenza virus (CIV) [80], which are H3N8 viruses encoding the first 73, 99, and 126 amino acids of NS1.

Infection with a PR8 H1N1 virus encoding an NS1 protein expressing only the first 125 amino acids (PR8 NS1–125) induced significantly higher amounts of IFN-β, IL-6, C-C motif Chemokine ligand 3 (CCL3), macrophage inflammatory protein 1-alpha (MIP-1α), and tumor necrosis factor (TNF) than the parental PR8 virus. Nevertheless, the PR8 NS1–125 virus was as efficient as the parental PR8 virus in inhibiting IL-1β and IL-18 release in infected macrophages [118]. Mutant viruses lacking or possessing nonfunctional RBD and dimerization domains induced significantly higher amounts of biologically active IL-1β and IL-18 than the parental or PR8 NS1–125 viruses [118]. In infected macrophages, these defective viruses induced rapid apoptosis, correlating with the enhanced activity of caspase-1. These results indicated that the N-terminal domain of the PR8 NS1 protein might control caspase-1 activation, leading to a repression in the maturation of pro-IL1β-, pro-IL18-, and caspase-1-dependent apoptosis in infected human macrophages [118].

### 2.4.2. Effect of NS1 Mutations Affecting NS1-Mediated Host Shutoff on Innate Immune Responses and Virus Pathogenesis

IAV encoding amino acid changes critical for NS1-mediated host shutoff showed altered pathogenesis. For example, a single amino acid change (I106M) in the NS1 protein from an avian-derived H7N9 IAV (A/Shanghai/1/2013 and A/Shanghai/2/2013 strains) restores the NS1’s ability to bind CPSF30 and the ability to block host gene expression, including IFN response genes [135]. Furthermore, a recombinant H7N9 virus expressing NS1 I106M replicated to higher titers in vivo and is subtly more virulent than the parental virus [135] (Table 2). Notably, PR8 NS1 cannot bind CPSF30 due to mutations at positions 103 and 106 [73]. Interestingly, the restoration of CPSF30

binding by introducing amino acid changes at these amino acid residues (L103F and I106M) resulted in enhanced virulence [73]. Likewise, using the highly pathogenic A/Hong Kong/483/97 H5N1 virus, it has been shown that the amino acid changes L103F and I106M increase the binding of NS1 protein to CPSF30, the ability to inhibit host gene expression, and virus pathogenicity in mice [74,136] (Table 2). The increased virulence of the H5N1 virus is associated with a faster systemic spread of the virus, particularly to the brain, where increased viral replication and severe pathology occur [136]. This augmented spread was also associated with increased cytokine and chemokine levels in extrapulmonary tissues [136].

In general, the NS1 proteins of the H3N2 and H2N2 IAV bind CPSF30, inhibiting general host gene expression [137]. However, we have recently shown that an amino acid change I64T in the NS1 protein of a virus from a patient infected with a seasonal H3N2 IAV leads to an impairment in its ability to bind CPSF30, and, therefore, a lack of inhibition of host gene expression [83] (Table 2). Furthermore, this mutation also led to reduced virus replication in cultured cells and to virus attenuation in vivo [83]. Similarly, viruses encoding the D189N and V194I amino acid changes in seasonal H3N2 IAV lead to NS1 proteins showing decreased binding to CPSF30, and reduced NS1-mediated inhibition of host gene expression [84] (Table 2). Similar to the virus encoding the mutation I64T, IAV containing the D189N and V194I amino acid changes were attenuated in vivo, with the virus containing the V194I amino acid change more attenuated than the virus with D189N, most likely because the V194I substitution confers a temperature-sensitive phenotype [84].

The NS1 protein encoded by pH1N1 viruses at the beginning of the 2009 pandemic did not inhibit general gene expression because of a lack of interaction with CPSF30 [125]. Interestingly, by introducing three amino acid changes (R108K, E125D, and G189D), the pH1N1 NS1 protein regained the ability to interact with CPSF30 (Figure 3A), and was more efficient at antagonizing host innate immune responses in primary human epithelial cells than its WT counterpart [125] (Table 2). However, the amino acid changes R108K, E125D, and G189D had no significant effect on virus replication in human or swine tissue culture cells [125]. Surprisingly, in a mouse model of infection, the pH1N1 mutant virus containing the NS1 R108K, E125D, and G189D amino acid changes appeared to cause less morbidity and was cleared faster than the original 2009 pH1N1 virus [125]. The mutant virus also grew with reduced titers in the upper respiratory tracts of ferrets [125]. We have recently described that currently circulating pH1N1 viruses encode an NS1 with six amino acid changes (E55K, L90I, I123V, E125D, K131E, and N205S) compared to the viruses circulating at the origin of the 2009 pandemic [126]. Remarkably, these six amino acid changes increased the ability of the pH1N1 NS1 protein (pH1N1/NSs-6mut) to inhibit host gene expression, mainly by restoring the binding of NS1 to CPSF30 [126] (Table 2). Some of these residues (i.e., 90 and 131) are far from the NS1–CPSF30 interface (Figure 3B). However, they still could be affecting the NS1 binding to CPSF30 by affecting the conformational structure of residues near or in the protein–protein interface. Moreover, they could be affecting the formation of NS1 dimers. As a consequence, a recombinant pH1N1 virus containing these six residue changes in the NS1 protein (pH1N1/NSs-6mut) inhibited innate IFN and proinflammatory responses more efficiently than the parental pH1N1 virus. However, virus titers were similar in cell cultures and in mouse lungs, and the virus containing the six amino acid changes (pH1N1/NSs-6mut) was partially attenuated as compared to the pH1N1 WT [126]. Notably, the pH1N1/NSs-6mut virus induced decreased levels of pro-inflammatory cytokines, likely due to the increased inhibition of host gene expression mediated by interaction of the evolved NS1 protein with CPSF30 [126]. These diminished levels of inflammation induced by the pH1N1/NSs-6mut virus are likely responsible for the attenuated disease phenotype, most likely representing a mechanism of host–virus adaptation affecting viral pathogenesis [126]. Moreover, these data suggest a selective advantage for viruses encoding an NS1 protein that is able to inhibit general gene expression in the human host, as viruses encoding NS1 proteins that are able to inhibit general host gene expression are selected in the population.

Moreover, it has been shown that a K186E amino acid substitution in the H3N8 CIV (A/canine/NY/dog23/2009 strain) and EIV (A/equine/Ohio/1/2003 strain) IAV restores their ability to bind CPSF 30 and inhibit host gene expression [62,72] (Table 2).

**Table 2. The IAV NS1 amino acid residues that are involved in the interaction with CPSF 30.**

| NS1 Amino Acid Residue | IAV | Reference |
|------------------------|-----|-----------|
| 184                    | A/Udorn/72H3N2 | [108] |
| 103 and 106            | A/Puerto Rico/8/34H1N1<br>A/Hong Kong/483/97H5N1 | [74,108,138] |
| 108, 125, 189, 55, 90, 123, 125,<br>131, and 205 | A/California/04/09H1N1 | [125,126] |
| 106                    | A/Shanghai/1/2013H7N9<br>A/Shanghai/2/2013H7N9 | [135] |
| 186                    | A/canine/NY/dog23/2009H3N8<br>A/equine/Ohio/1/2003H3N8 | [62,72] |
| 64, 189, and 194       | A/Victoria/361/2011H3N2<br>A/Perth/16H3N2 | [83,84] |

### 2.4.3. Effect of NS1 Mutations Affecting Inflammatory Processes

A G45R amino acid change in the NS1 protein of PR8 H1N1 contributes to virulence by increasing the expression of pro-inflammatory cytokines in mice [139]. PR8 viruses encoding the NS1 gene from a high pathogenic avian A/Hong Kong/156/97H5N1 virus were resistant to IFN and TNF antiviral activities and showed increased pathogenicity in pigs [140] and in mice [141]. Interestingly, a glutamic acid (E) at position 92 in the A/Hong Kong/156/97H5N1 NS1 protein was critical for this phenotype, as an E92D mutant virus grew with lower titers and was attenuated in pigs [140] and in mice [141]. Mice infected with a virus encoding the NS1 protein of A/Hong Kong/156/97H5N1 exhibited elevated pulmonary concentrations of the inflammatory cytokines IL-1α, IL-1β, IL-6, and IFN-γ and the chemokine CXCL10, and decreased concentrations of the anti-inflammatory cytokine IL-10 [141]. This cytokine imbalance is likely responsible for the unusual severity of the IAV H5N1 and correlates with the clinical findings observed in two humans who died of IAV H5N1 infection [141].

It is becoming clearer that the morbidity and pathogenesis caused by IAV are consequences of the inflammatory response [142]. Therefore, the inhibition of inflammasome activation by the IAV NS1 protein likely contributes to decreasing virus virulence [105–107]. However, more studies are needed to analyze the relevance of NS1-mediated inflammasome inhibition in virus pathogenesis, since most of the studies were performed in cell culture and not in animal systems. Therefore, the effect of NS1 mutations on IFN and pro-inflammatory cytokine induction likely has positive or negative effects of virus replication and pathogenesis, depending on the balance between IFN and pro-inflammatory responses. In addition, we and others have shown that other viral proteins could have a role in the modulation of the host innate immune system, and the interplay between NS1 and other viral factors (e.g., PA-X), which is discussed later in this review, will require more investigation.

### 2.5. Modulation of IAV NS1 Expression for the Development of More Efficient Live-Attenuated Influenza Vaccines (LAIV)

IAV infections represent a severe public health and economic concern that is most effectively prevented through vaccination [143]. However, despite the application of effective and broad vaccination programs, IAV infections are responsible for global epidemics every year, and the efficiency of current influenza vaccines is suboptimal [144,145]. The decreased efficacy of seasonal influenza vaccines is in part due to an antigenic drift in the two main proteins (HA and NA) responsible for inducing neutralizing antibodies [12,145–148]. Therefore, there is an urgent need to develop

more effective vaccines against influenza viruses. Currently, three types of influenza virus vaccines have been approved for their use in humans by the U.S. Food and Drug Administration (FDA): inactivated influenza vaccines (IIV), live-attenuated influenza vaccines (LAIV), and recombinant viral HA [143, 145, 149, 150]. IIV is the most common vaccine for the control of influenza infections, and it has been shown to elicit protective humoral immunity [144, 145]. However, it is known that LAIV elicit more rapid and broader efficient innate and adaptive immune responses than IIV, and can provide more efficient cross-reactive T-cell-mediated protection against heterologous influenza viruses [143]. Commercially available LAIV are based on the cold-adapted, temperature-sensitive, attenuated master donor virus (MDV) A/Ann Arbor/6/60 H2N2 that replicates in the upper respiratory tract but does not damage the lower respiratory tract due to the elevated temperatures that restrict viral replication [151]. However, new approaches to attenuate the virus(es) in the vaccine are also needed to overcome the limitations that are associated with the current LAIV [143, 145, 148].

IAV lacking the NS1 gene or encoding truncations in the NS1 gene are attenuated in vivo. Therefore, they represent promising LAIV candidates for the prevention of influenza infections [81]. For example, a PR8 virus expressing only the first 126 amino acids of NS1 was attenuated in mice and fully protects against a lethal viral challenge, even in aged mice [129]. Similarly, viruses expressing only the first 73 or 126 amino acids of the NS1 protein of A/NewYork/1682/2009 pH1N1 protected mice and ferrets from WT viral challenge [152]. Vaccination with a virus expressing only the first N-terminal 126 amino acids of A/Texas/36/1991 H1N1 was also effective in protecting macaques from homologous WT viral challenge [132]. This protection was in the absence of significant or long-lasting adverse effects and through the induction of a robust adaptive immune response [132]. Likewise, A/canine/NY/dog23/2009 H3N8 viruses encoding truncated (NS1–73, NS1–99, and NS1–126) or lacking (∆NS1) NS1 were reduced in their ability to replicate ex vivo and in vivo [80]. Moreover, immunization of mice by a single intranasal dose of these NS1-deficient or truncated mutant viruses protected against WT H3N8 CIV challenge [80]. Similarly, an equine H3N8 virus (A/equine/Kentucky/5/2002) encoding a truncated NS1 (NS1–126) was attenuated in vivo and provided protection against WT EIV challenge in horses [133, 134]. Remarkably, a clinical trial in phase I/II in humans showed that a replication-deficient trivalent influenza vaccine containing the IAV strains A/Brisbane/59/07 H1N1-like, A/Brisbane/10/07 H3N2-like, and B/Florida/04/06-like, all lacking the NS1 protein, is safe and induces significant levels of antibodies that protect against disease caused by the influenza virus [153].

We have explored another strategy for generating LAIV based on NS1 modifications [82]. We have been able to generate recombinant influenza PR8 viruses encoding misrepresented mammalian codons (codon deoptimization, CD) comprising the entire NS gene (NS1 and NEP) or the mRNA corresponding to the individual viral NS1 or NEP, without modifying the respective splicing and packaging signals [82]. The growth of these synthetic CD PR8 viruses was impaired in vivo, while they retained immunogenicity, and conferred protection upon a single intranasal administration dose, against both homologous (PR8, H1N1) and heterologous (X31, a reassortant virus carrying HA and NA proteins from an H3N2 virus) viral challenges [82]. These results open the possibility of using this NS CD approach, alone or in combination with other attenuation strategies, for their implementation as LAIV.

Recently, a systematic approach for vaccine development that eliminates IFN-modulating functions while maintaining virus replication fitness has been described [154]. Using this approach, a highly IFN-sensitive virus, encoding three amino acid changes in PB2, three amino acid changes in M1, and two amino acid changes in NS1, was highly attenuated in IFN-competent hosts. However, this virus was able to induce transient IFN responses, and robust cellular and humoral responses, providing protection against homologous and heterologous challenges [154].

Viruses 2018, 10, 708

3. The IAV PA-X Protein

IAV segment 3 encodes the PA and the PA-X proteins [155]. PA is translated directly from the PA mRNA, and is required for virus replication and transcription, together with the other components of the viral RNA-dependent RNA polymerase (RdRp), PB2 and PB1 [156]. PA-X is translated as a +1 frameshift open reading frame (ORF) from the PA viral segment [155] (Figure 4). During translation, the ribosome shifts at a specific sequence in the PA mRNA, a U-rich region followed by a rare codon [157]. These rare codons promote ribosomal frame-shifting because they are typically decoded more slowly [157]. Because of the +1 frameshift, during the translation of PA, the codons CGU CAG (amino acids RQ) are read as GUC (V), not reading the C nucleotide during PA-X translation and producing a shift in the open reading frame (Figure 4) [155]. PA-X shares the same first N-terminal 191 amino acids with the PA protein, including the endonuclease domain (Figure 4). However, the ribosomal frameshift produces a PA-X containing a unique short C-terminal sequence [155]. Most of the human IAV strains contain a 61 amino acids C-terminal extension (252 amino acid PA-X), with the important exception of the 2009 pH1N1 viruses that encode a 41 amino acids extension (232 amino acid PA-X) [158] (Figure 4).

![Diagram](attachment:diagram.png)

Figure 4. A schematic representation of the IAV PA viral segment and the PA and PA-X open reading frames (ORFs). Blue and red boxes indicate the ORF for PA and PA-X, respectively. The +1 frameshift motif (UCC UUU CGU C) at position 191 is indicated. Bold and italics in the frameshift motif (C nucleotide) indicate that the nucleotide C is not read during PA-X translation. PA-X proteins containing 232 or 252 amino acids if the C-terminal region has a 41 or 61 amino acid extension, respectively, are indicated.

3.1. The IAV PA-X Protein's Effect on Host Gene Expression

PA-X induces the shutoff of host protein expression in infected cells, contributing to the blocking of cellular antiviral responses [124,155,159–163]. However, PA-X-induced suppression of host protein synthesis not only affects antiviral genes, and the effect of PA-X expression on virus fitness and virulence seems to be strain-specific [158,162–166]. The PA-X-mediated shutoff activity involves the N-terminal endonucleolytic domain, resulting in the degradation of host mRNAs. Accordingly, mutations in the endonuclease active site of PA-X inactivate the protein’s ability to induce host cellular shutoff [167]. PA-X selectively targets cellular mRNAs while not affecting viral mRNAs, thereby ensuring successful viral replication and counteracting the antiviral response in the host [160]. PA-X selectively degrades host RNA polymerase II (Pol II)-transcribed mRNAs and non-coding RNAs in the nucleus of infected cells, while sparing the products of polymerases I and III [124,161]. Complete degradation of host mRNAs following PA-X-mediated endonucleolytic cleavage is dependent on the host 5′->3′-exonuclease Xrn1 [161]. The host protein shutoff activity of IAV PA-X is stronger than that of PA or the N-terminal PA domain, indicating that the C-unique terminal region of PA-X increases inhibition of host protein expression [159,168]. Two independent studies demonstrated that a PA-X expressing the first 15 amino acids in the C-terminal region (positions 192–206) displays the same

host cellular shutoff activity as the full-length protein, and that six basic amino acids (195R, 198K, 199R, 202K, 203K, and 206K) play a key role in PA-X's ability to inhibit host gene expression [167,169]. In addition, other studies reported that amino acids 233–252 in the C-terminal region (found in most IAV strains, but not in the 2009 pH1N1 IAV strains) of PA-X enhance the suppression of host gene expression of human (pH1N1) or avian (H5N1 and H9N2) IAV strains [166,170].

Cellular host protein shutoff activity is strongly associated with nuclear accumulation of the PA-X protein, which is mainly mediated by four conserved basic residues (198K, 199R, 202K, and 203K) in the C-terminal region of PA-X [161]. Supporting these results, a recent study found that the first nine amino acids in the C-terminal region of PA-X (amino acids 192–200) are sufficient for nuclear localization, and that three basic amino acids (195K, 198K, and 199R) are key for host shutoff ability and PA-X's nuclear accumulation [167]. Accordingly, an additional study revealed that PA-X mutants with amino acid changes at positions 4, 9, 24, 27, 39, 45, 87, 94, 106, 107, 108, 119, 120, 123, 124, 125, 148, 154, 160, 163, 168, and 171 predominantly localized in the cytoplasm and displayed decreased shutoff activity, suggesting that these mutations decreased the shutoff activity of PA-X by affecting PA-X's translocation to the nucleus [171]. Other studies indicate that PA-X degrades mature mRNAs localized both in the nucleus and in the cytoplasm. However, degradation of mRNAs by PA-X in the nucleus was more efficient than in the cytoplasm, thereby showing that PA-X's ability to degrade host cellular mRNAs occurs at different functional sites [167].

### 3.2. Effect of IAV PA-X on Virus Pathogenesis and Suppression of Innate Immune Responses

The strain-specific loss of PA-X expression can increase or decrease viral replication and virulence. For example, loss of PA-X expression in the 2009 pH1N1 A/Beijing/16/2009 strain and in the highly pathogenic avian A/Anhui/1/2005 and A/chicken/Jiangsu/K0402/2010 H5N1 strains increased viral replication, virulence, and host inflammatory response in mice [172,173] and in birds (A/chicken/Jiangsu/K0402/2010 strain) [163]. Similarly, PA-X expression was found to decrease the pathogenicity of the pandemic A/Brevig Mission/1/1918 H1N1 virus in mice [155]. Recently, we have found that the currently circulating pH1N1 viruses have evolved to encode four amino acid changes in the PA-X protein (V100I, N204S, R221Q, and L229S) [71]. These amino acid differences between early (2009) and more recent (2015) pH1N1 isolates are responsible for decreased PA-X-mediated shutoff, including innate immune response genes [71]. Higher pro-inflammatory responses that correlate with increased virulence were observed after infection with recombinant viruses expressing the PA-X protein from currently circulating viruses that have a decreased host shutoff activity [71]. In contrast, recent work has described that loss of PA-X in the A/California/04/2009 pH1N1 or A/chicken/Hebei/LC/2008 H9N2 viruses leads to a reduction in viral replication and pathogenicity as well as an increased host innate immune response in mice [159,165,166]. These studies suggest that the effect of PA-X on virus replication and pathogenesis could be host- and strain-specific, perhaps reflecting adaptation to a given host.

The biological significance of PA-X's length was studied by generating human A/Beijing/16/2009 and A/California/04/2009 pH1N1, and the avian A/tree sparrow/Jiangsu/1/2008 H5N1 and A/chicken/Hebei/LC/2008 H9N2 recombinant viruses, encoding 61 (192–252) or 41 (192–232) amino acids at the C-terminal region [166,170]. The results from two independent studies suggested that the last 233–252 amino acid extension slightly increased viral replication and virulence in mice, strengthened the viral-induced inflammatory response and apoptosis, and elevated the host shutoff ability by PA-X as compared to those encoding the 41 amino acid extension [166,170].

### 3.3. Modulation of PA-X to Generate LAIVs

It has been described that inhibition of PA-X expression leads to a slight virus attenuation of certain IAV strains [165,166]. Therefore, this strategy could be used in some IAV strains as an additional measure to generate LAIV. Interestingly, mice infected with a A/California/04/2009 pH1N1 expressing lower levels of the PA-X protein induced higher titers of IFN expression and grew with

lower titers in the lungs than the parental pH1N1 virus [159]. However, slightly higher titers of anti-HA and neutralizing antibodies were likely produced after infection with the PA-X mutant pH1N1 virus [159] (Table 3), a critical aspect for the induction of protective immune responses, which should be further analyzed.

**Table 3. Properties of live-attenuated influenza vaccines (LAIV) and currently circulating wild-type (WT) pH1N1 viruses.**

| Properties | pH1N1 LAIV (ref. [174]) | pH1N1 WT (ref. [71]) |
| --- | --- | --- |
|  | PA<sub>WT</sub> NS1<sub>WT</sub> | PA<sub>MUT</sub> NS1<sub>WT</sub> | PA<sub>WT</sub> NS1<sub>MUT</sub> | PA<sub>MUT</sub> NS1<sub>MUT</sub> | PA<sub>WT</sub> NS1<sub>WT</sub> | PA<sub>MUT</sub> NS1<sub>WT</sub> | PA<sub>WT</sub> NS1<sub>MUT</sub> | PA<sub>MUT</sub> NS1<sub>MUT</sub> |
| Inhibition of host gene expression<sup>a</sup> | + | – | + | – | + | – | + | – |
|  | – | – | + | + | – | – | + | + |
| Pathogenicity in mice/Viral replication in mouse lungs | +++/+++ | –/– | –/– | +++/+++ | ++/+ | +++/+++ | +/+ | +++/+++ |
| Induction of innate immune response in mice<sup>b</sup>/Induction of humoral responses | +++/+++ | –/– | –/– | +++/+++ | +/ND | ++/ND | +/ND | +++/ND |

<sup>a</sup> PA (top) and NS1 (bottom) proteins with the ability to inhibit (+) or not inhibit (–) host gene expression. ND: Not determined.  
<sup>b</sup> Measured by the levels of IFN-β, CCL2, or monocyte chemotactic protein 1 (MCP-1) and TNF mRNAs in lungs from infected mice [71, 174].

## 4. Interplay of the IAV NS1 and PA-X Proteins

### 4.1. Interaction of the IAV NS1 and PA-X Proteins on Virus Fitness and Virulence

The IAV NS1 and PA-X proteins have the synergistic ability to inhibit host gene expression, although using different mechanisms. However, whereas many studies have described the individual effect of IAV NS1 and PA-X on host gene expression, the interplay between these two proteins has been barely studied. We have described that recombinant LAIV pH1N1 viruses encoding NS1 and PA-X proteins that simultaneously inhibit or do not inhibit host gene expression are impaired in viral growth properties in cultured cells, correlating with virus attenuation in vivo, in comparison to recombinant pH1N1 viruses in which only one of the viral proteins (NS1 or PA-X) inhibited host gene expression [174] (Table 3). Remarkably, by using a pH1N1 LAIV as a backbone, we have observed greater differences in virus pathogenicity than those detected using a WT virus, suggesting that the LAIV virus could be a better option to study virulence factors and virus-host interactions. Importantly, contrary to other studies [166], we did not observe a higher innate or humoral response with the PA-X-deficient virus in mice, which is most likely due to the reduced viral replication of the attenuated virus [174]. These results suggested, for the first time, that inhibition of host protein expression by IAV is subject to a strict balance that can determine the successful progression of viral infection [160, 174]. This knowledge raises questions about the interpretation of experiments that only include mutations in one but not both viral proteins and further demonstrates how virulence in IAV is a multigenic factor where more than one viral gene should be considered.

Multiple mutations have appeared, and they are most likely associated with host adaptation [2, 15, 125, 126]. As indicated above, the NS1 protein of the currently circulating pH1N1 IAV has gained the ability to inhibit host gene expression by acquiring the amino acid substitutions E55K, L90I, I123V, E125D, K131E, and N205S [126]. Based on our previous observation with the LAIV pH1N1 [174], we assessed the ability of PA-X from recent pH1N1 IAV strains with mutated NS1 proteins to inhibit host gene expression [71]. Interestingly, we observed multiple amino acid changes in the PA-X protein (V100I, N204S, R221Q, and L229S). While amino acid changes led to increased NS1-mediated inhibition of host gene expression, changes in recent pH1N1 strains resulted in decreased PA-X-mediated inhibition of host gene expression [71]. Importantly, a recombinant pH1N1 virus containing PA, PA-X, and NS1 genes from currently circulating viruses replicated to higher titers in culture cells and in mice, was more virulent in vivo, and induced a higher activation of pro-inflammatory responses in the lungs [71] (Table 3).

Importantly, these results suggest that a balance in the ability of the NS1 and PA-X proteins to regulate host cellular shutoff is beneficial for IAV [71], confirming our previous findings using the pH1N1 LAIV [174] (Table 3). Although a strong inhibition of host gene expression, including genes with antiviral activity, during viral infection is likely beneficial for IAV, an excessive inhibition of host protein synthesis could affect the expression of multiple host factors required for efficient virus replication, which could be deleterious for the virus [160]. On the other hand, a virus without the ability to block inhibition of host gene expression, including host genes involved in antiviral defense, could be severely attenuated [160]. Thus, a balance in inhibition of host gene expression mediated by IAV NS1 and PA-X proteins is important for viral fitness not only in cultured cells but also in vivo [71,174]. More research into how NS1 and PA-X activities are balanced in different IAV strains and the effects of this balance on viral replication and pathogenesis are really needed. Moreover, other viral factors, such as PB1-F2 [37] or PB2 [175], could also play a key role in modulating the antiviral response induced during IAV infection. These results also demonstrate the need to monitor IAV for co-evolution in the viral genome that could result in more pathogenic viruses or potential pandemics threats.

### 4.2. Inhibition of Host Gene Expression by IAV NS1 and PA-X Proteins for the Development of More Efficient LAIV

We have hypothesized that modulating the ability of NS1 and/or PA-X to counteract the innate immune responses could be a suitable strategy to generate improved LAIV [174]. Therefore, we have explored how the interplay between PA-X and NS1 affects the development of attenuated IAV strains that might be used as vaccines [143]. We have tested a set of A/California/04/09 pH1N1 LAIV encoding PA-X and NS1 proteins with different abilities to inhibit host gene expression [174]. Antibodies specific to total viral proteins, HA, and NA were detected at similar levels in sera from mice infected with viruses where only one of the proteins (PA-X or NS1) has the ability to inhibit gene expression. However, and consistent with the levels of viral replication, antibody levels were lower in serum samples from mice infected with viruses where both proteins have or do not have the ability to inhibit host gene expression [174]. Those data suggest the feasibility of genetically controlling the ability of IAV PA-X and NS1 proteins to inhibit host immune responses, and the feasibility of implementing this approach, alone or in combination with other methodologies, for the development of more effective LAIV to combat disease caused by this important respiratory pathogen [174].

Since the MDV of the IAV LAIV remains constant between seasons, and only the viral HA and NA genes are updated [176], it has been suggested that pre-existing immunity to the internal proteins of the MDV, such as NS1 and PA-X, could limit the response to the LAIV, which must replicate in order to be immunogenic and provide protection against subsequent viral infections. In fact, this could be one of the reasons for the low efficacy of LAIV [13,177]. Understanding how seasonal IAV vaccines are influenced by a pre-existing immunity will be important for developing next-generation IAV vaccines. Moreover, futures strategies to develop new vaccines against IAV, which are highly desired, might require modifying the MDV A/Ann Arbor/6/60 H2N2 currently used for the preparation of seasonal and pandemic LAIV.

## 5. Conclusions

IAV and other viruses hijack the cellular machinery for the progression of viral infection. In order to replicate in the host, IAV encode two viral proteins, NS1 and PA-X, that have developed distinct mechanisms to counteract innate immune responses and the antiviral state produced in the infected and neighboring cells [60,70,160]. Interestingly, one synergistic mechanism used by IAV NS1 and PA-X proteins to counteract the immune response consists in their ability to block host protein synthesis, although using different mechanisms [70,74,155,160,161]. Blocking cellular protein expression contributes to dampening the antiviral response as well as to diverting the cellular machinery towards viral replication [160]. Moreover, controlling the inflammatory response could be important for host survival during viral replication and spread. However, an exacerbated blocking of host protein synthesis can be deleterious and may affect successful viral replication [160]. Likewise,

the lack of control of host protein expression would result in the induction of antiviral responses to control viral infection [160]. Therefore, an optimal control of host gene expression and innate immune responses are required for efficient viral replication in IFN-competent systems [160]. It has been reported that IAV PA-X and NS1 proteins are important contributors to maintain this balance, although other viral and host factors could also be playing an important role in this phenomenon. Modulating the ability of IAV to control the antiviral response produced during infection could be a feasible strategy to develop better LAIV [174]. Moreover, surveillance of those virulence factors will be important to predict new pandemics threats. Finally, monitoring the evolution of IAV NS1 and PA-X and their respective ability to inhibit host gene expression will provide important information on the mechanism(s) of viral host adaptation [71].

Author Contributions: M.L.D. drafted the manuscript. M.L.D. and A.N. wrote the manuscript and designed the tables/illustrations. L.M.-S. critically revised the manuscript and provided ideas, concepts, and edits. D.J.T. edited the manuscript.

Funding: Influenza virus research in the Luis Martínez-Sobrido and David J. Topham laboratories is partially funded by the National Institute of Allergy and Infectious Diseases (NIAID), the National Institutes of Health (NIH), and the Department of Health and Human Services under the Centers of Excellence for Influenza Research and Surveillance (CEIRS) contract No. HHSN272201400005C (New York Influenza Center of Excellence, NYICE); by the Department of Defense (DoD) PR17046 (Luis Martinez-Sobrido); and with funds from a University of Rochester Research Award to Aitor Nogales and Marta Lopez de Diego. We also give thanks for the funds from Comunidad de Madrid, Spain, reference 2017-T1/BMD-5155 (Marta Lopez de Diego).

Conflicts of Interest: The authors declare no conflict of interest.

### Abbreviations

| ACIP | Advisory Committee on Immunization Practices |
| ASC | Apoptosis-associated Speck-like containing a caspase-recruitment domain |
| ATF | Activating transcription factor |
| CD | Codon deoptimized |
| CIV | Canine influenza virus |
| CPSF30 | Cleavage and polyadenylation specificity factor 30 |
| CTT | C-terminal tail |
| dsRNA | double-stranded RNA |
| E1B-AP5 | Adenovirus early region 1B-associated protein 5 |
| ED | Effector domain |
| EIV | Equine influenza virus |
| eIF2α | α subunit of the eukaryotic initiation factor 2 |
| eIF4G | Eukaryotic initiation factor 4G |
| FDA | Food and drug administration |
| HA | Hemagglutinin |
| IAV | Influenza A virus |
| IFN | Interferon |
| IV | Influenza inactivated vaccine |
| IKK | inhibitor of kappa β kinase |
| IL | Interleukin |
| IRF | Interferon regulatory factor |
| ISGs | Interferon (IFN)-stimulated genes |
| LAIV | Live-attenuated influenza vaccine |
| M2 | Matrix protein 2 |
| MDV | Master donor virus |
| NA | Neuraminidase |
| NCR | Non-coding region |
| NEP | Nuclear export protein |
| NES | Nuclear export signal |

NF-κB          | Nuclear factor kappa β  
NLRP3          | NOD-like receptor family member LRR- and Pyrin domain containing-3  
NLS            | Nuclear localization signal  
NS1            | Non-structural protein 1  
NSs            | NS split  
NXF1           | Nuclear RNA export Factor 1  
OAS            | Oligoadenylate synthase  
ORF            | Open reading frame  
PA             | Polymerase acid  
PABP           | Poly(A)-binding protein  
PACT           | Protein activator of the IFN-induced PKR  
PAMPs          | Pathogen-associated molecular patterns  
PB1            | Polymerase basic 1  
PB2            | Polymerase basic 2  
pH1N1          | pandemic H1N1 virus  
PI3K           | p85-β of phosphatidylinositol 3-kinase  
PKR            | Protein kinase R  
Pol II         | Polymerase II  
PR8            | A/Puerto Rico/8/34  
PRR            | Pattern recognition receptor  
RAE1           | Ribonucleic Acid Export 1  
RBD            | RNA-binding domain  
RIG-I          | Retinoic acid inducible gene I  
RNase L        | Ribonuclease L  
RdRp           | RNA-dependent RNA polymerase  
ssRNA          | single-stranded RNA  
TNF            | Tumor necrosis factor  
TLRs           | Toll-like receptors  
TRIM25         | E3 ligase tripartite motif-containing protein 25  
UTR            | Untranslated region  
WT             | Wild-type  
WHO            | World Health Organization  
WSN            | A/WSN/33  

## References

1. Palese, P.; Shaw, M.L. Orthomyxoviridae: The viruses and their replication. In *Fields Virology*; Knipe, D., Howley, P., Eds.; Lippincott Williams & Wilkins: Philadelphia, PA, USA, 2006; pp. 1648–1689.
2. Parrish, C.R.; Kawaoka, Y. The origins of new pandemic viruses: The acquisition of new host ranges by canine parvovirus and influenza A viruses. *Annu. Rev. Microbiol.* **2005**, *59*, 553–586. [CrossRef] [PubMed]
3. Steel, J.; Lowen, A.C.; Mubareka, S.; Palese, P. Transmission of influenza virus in a mammalian host is increased by PB2 amino acids 627K or 627E/701N. *PLoS Pathog.* **2009**, *5*, e1000252. [CrossRef] [PubMed]
4. Bailey, E.S.; Choi, J.Y.; Fieldhouse, J.K.; Borkenhagen, L.K.; Zemke, J.; Zhang, D.; Gray, G.C. The continual threat of influenza virus infections at the human-animal interface: What is new from a one health perspective? *Evol. Med. Public Health* **2018**, *2018*, 192–198. [CrossRef] [PubMed]
5. Gonzalez, G.; Marshall, J.F.; Morrell, J.; Robb, D.; McCauley, J.W.; Perez, D.R.; Parrish, C.R.; Murcia, P.R. Infection and pathogenesis of canine, equine, and human influenza viruses in canine tracheas. *J. Virol.* **2014**, *88*, 9208–9219. [CrossRef] [PubMed]
6. Thompson, W.W.; Shay, D.K.; Weintraub, E.; Brammer, L.; Cox, N.; Anderson, L.J.; Fukuda, K. Mortality associated with influenza and respiratory syncytial virus in the United States. *JAMA* **2003**, *289*, 179–186. [CrossRef] [PubMed]
7. Taubenberger, J.K.; Morens, D.M. Influenza: The once and future pandemic. *Public Health Rep.* **2010**, *125* (Suppl. 3), 16–26. [CrossRef] [PubMed]

8. Memorandum, W. A revision of the system of nomenclature for influenza viruses: A WHO memorandum. *Bull. World Health Organ.* 1980, 58, 585–591.

9. Medina, R.A.; Garcia-Sastre, A. Influenza A viruses: New research developments. *Nat. Rev. Microbiol.* 2011, 9, 590–603. [CrossRef] [PubMed]

10. Tong, S.; Zhu, X.; Li, Y.; Shi, M.; Zhang, J.; Bourgeois, M.; Yang, H.; Chen, X.; Recuenco, S.; Gomez, J.; et al. New world bats harbor diverse influenza A viruses. *PLoS Pathog.* 2013, 9, e1003657. [CrossRef]

11. Kuchipudi, S.V.; Nissly, R.H. Novel flu viruses in bats and cattle: “Pushing the Envelope” of Influenza Infection. *Vet. Sci.* 2018, 5, 71. [CrossRef]

12. Clark, A.M.; DeDiego, M.L.; Anderson, C.S.; Wang, J.; Yang, H.; Nogales, A.; Martinez-Sobrido, L.; Zand, M.S.; Sangster, M.Y.; Topham, D.J. Antigenicity of the 2015–2016 seasonal H1N1 human influenza virus HA and NA proteins. *PLoS ONE* 2017, 12, e0188267. [CrossRef] [PubMed]

13. Grohskopf, L.A.; Sokolow, L.Z.; Broder, K.R.; Walter, E.B.; Bresee, J.S.; Fry, A.M.; Jernigan, D.B. Prevention and Control of Seasonal Influenza with Vaccines: Recommendations of the Advisory Committee on Immunization Practices—United States, 2017–18 Influenza Season. *Am. J. Transplant.* 2017, 17, 2970–2982. [CrossRef] [PubMed]

14. Bui, C.M.; Chughtai, A.A.; Adam, D.C.; MacIntyre, C.R. An overview of the epidemiology and emergence of influenza A infection in humans over time. *Arch. Public Health* 2017, 75, 15. [CrossRef] [PubMed]

15. Garten, R.J.; Davis, C.T.; Russell, C.A.; Shu, B.; Lindstrom, S.; Balish, A.; Sessions, W.M.; Xu, X.; Skepner, E.; Deyde, V.; et al. Antigenic and genetic characteristics of swine-origin 2009 A(H1N1) influenza viruses circulating in humans. *Science* 2009, 325, 197–201. [CrossRef] [PubMed]

16. Trifonov, V.; Khiabanian, H.; Rabadan, R. Geographic dependence, surveillance, and origins of the 2009 influenza A (H1N1) virus. *N. Engl. J. Med.* 2009, 361, 115–119. [CrossRef] [PubMed]

17. Su, Y.C.; Bahl, J.; Joseph, U.; Butt, K.M.; Peck, H.A.; Koay, E.S.; Oon, L.L.; Barr, I.G.; Vijaykrishna, D.; Smith, G.J. Phylodynamics of H1N1/2009 influenza reveals the transition from host adaptation to immune-driven selection. *Nat. Commun.* 2015, 6, 7952. [CrossRef] [PubMed]

18. Anderson, C.S.; Ortega, S.; Chaves, F.A.; Clark, A.M.; Yang, H.; Topham, D.J.; DeDiego, M.L. Natural and directed antigenic drift of the H1 influenza virus hemagglutinin stalk domain. *Sci. Rep.* 2017, 7, 14614. [CrossRef]

19. Girard, M.P.; Cherian, T.; Pervikov, Y.; Kiely, M.P. A review of vaccine research and development: Human acute respiratory infections. *Vaccine* 2005, 23, 5708–5724. [CrossRef]

20. Iwasaki, A.; Pillai, P.S. Innate immunity to influenza virus infection. *Nat. Rev. Immunol.* 2014, 14, 315–328. [CrossRef]

21. Goff, P.H.; Hayashi, T.; He, W.; Yao, S.; Cottam, H.B.; Tan, G.S.; Crain, B.; Krammer, F.; Messer, K.; Pu, M.; et al. Synthetic Toll-Like Receptor 4 (TLR4) and TLR7 Ligands Work Additively via MyD88 To Induce Protective Antiviral Immunity in Mice. *J. Virol.* 2017, 91, JVI-01050. [CrossRef]

22. Weber-Gerlach, M.; Weber, F. Standing on three legs: Antiviral activities of RIG-I against influenza viruses. *Curr. Opin. Immunol.* 2016, 42, 71–75. [CrossRef] [PubMed]

23. Pflug, A.; Guilligay, D.; Reich, S.; Cusack, S. Structure of influenza A polymerase bound to the viral RNA promoter. *Nature* 2014, 516, 355–360. [CrossRef] [PubMed]

24. Wolff, T.; Ludwig, S. Influenza viruses control the vertebrate type I interferon system: Factors, mechanisms, and consequences. *J. Interferon Cytokine Res.* 2009, 29, 549–557. [CrossRef] [PubMed]

25. Hermant, P.; Michiels, T. Interferon-lambda in the context of viral infections: Production, response and therapeutic implications. *J. Innate Immun.* 2014, 6, 563–574. [CrossRef] [PubMed]

26. Mogensen, T.H.; Paludan, S.R. Molecular pathways in virus-induced cytokine production. *Microbiol. Mol. Biol. Rev.* 2001, 65, 131–150. [CrossRef] [PubMed]

27. Crotta, S.; Davidson, S.; Mahlakoiv, T.; Desmet, C.J.; Buckwalter, M.R.; Albert, M.L.; Staeheli, P.; Wack, A. Type I and type III interferons drive redundant amplification loops to induce a transcriptional signature in influenza-infected airway epithelia. *PLoS Pathog.* 2013, 9, e1003773. [CrossRef] [PubMed]

28. Levy, D.E.; Marie, I.J.; Durbin, J.E. Induction and function of type I and III interferon in response to viral infection. *Curr. Opin. Virol.* 2011, 1, 476–486. [CrossRef]

29. Hoffmann, H.H.; Schneider, W.M.; Rice, C.M. Interferons and viruses: An evolutionary arms race of molecular interactions. *Trends Immunol.* 2015, 36, 124–138. [CrossRef]

30. Schindler, C.; Levy, D.E.; Decker, T. JAK-STAT signaling: From interferons to cytokines. *J. Biol. Chem.* 2007, 282, 20059–20063. [CrossRef]

31. Tenovever, B.R.; Ng, S.L.; Chua, M.A.; McWhirter, S.M.; Garcia-Sastre, A.; Maniatis, T. Multiple functions of the IKK-related kinase IKKepsilon in interferon-mediated antiviral immunity. *Science* **2007**, *315*, 1274–1278. [CrossRef]

32. Ng, S.L.; Friedman, B.A.; Schmid, S.; Gertz, J.; Myers, R.M.; tenOever, B.R.; Maniatis, T. I kappa B kinase epsilon (IKK epsilon) regulates the balance between type I and type II interferon responses. *Proc. Natl. Acad. Sci. USA* **2011**, *108*, 21170–21175. [CrossRef] [PubMed]

33. Groslambert, M.; Py, B.F. Spotlight on the NLRP3 inflammasome pathway. *J. Inflamm. Res.* **2018**, *11*, 359–374. [CrossRef] [PubMed]

34. Guarda, G.; Zenger, M.; Yazdi, A.S.; Schroder, K.; Ferrero, I.; Menu, P.; Tardivel, A.; Mattmann, C.; Tschopp, J. Differential expression of NLRP3 among hematopoietic cells. *J. Immunol.* **2011**, *186*, 2529–2534. [CrossRef] [PubMed]

35. Pothlichet, J.; Meunier, I.; Davis, B.K.; Ting, J.P.; Skamene, E.; von Messling, V.; Vidal, S.M. Type I IFN triggers RIG-I/TLR3/NLRP3-dependent inflammasome activation in influenza A virus infected cells. *PLoS Pathog.* **2013**, *9*, e1003256. [CrossRef] [PubMed]

36. Ichinohe, T.; Pang, I.K.; Iwasaki, A. Influenza virus activates inflammasomes via its intracellular M2 ion channel. *Nat. Immunol.* **2010**, *11*, 404–410. [CrossRef] [PubMed]

37. McAuley, J.L.; Tate, M.D.; MacKenzie-Kludas, C.J.; Pinar, A.; Zeng, W.; Stutz, A.; Latz, E.; Brown, L.E.; Mansell, A. Activation of the NLRP3 inflammasome by IAV virulence protein PB1-F2 contributes to severe pathophysiology and disease. *PLoS Pathog.* **2013**, *9*, e1003392. [CrossRef] [PubMed]

38. Allen, I.C.; Scull, M.A.; Moore, C.B.; Holl, E.K.; McElvania-TeKippe, E.; Taxman, D.J.; Guthrie, E.H.; Pickles, R.J.; Ting, J.P. The NLRP3 inflammasome mediates in vivo innate immunity to influenza A virus through recognition of viral RNA. *Immunity* **2009**, *30*, 556–565. [CrossRef] [PubMed]

39. Schroder, K.; Tschopp, J. The inflammasomes. *Cell* **2010**, *140*, 821–832. [CrossRef]

40. Villalon-Letelier, F.; Brooks, A.G.; Saunders, P.M.; Londrigan, S.L.; Reading, P.C. Host cell restriction factors that limit influenza a infection. *Viruses* **2017**, *9*, 376. [CrossRef]

41. Staeheli, P.; Haller, O.; Boll, W.; Lindenmann, J.; Weissmann, C. Mx protein: Constitutive expression in 3T3 cells transformed with cloned Mx cDNA confers selective resistance to influenza virus. *Cell* **1986**, *44*, 147–158. [CrossRef]

42. Krug, R.M.; Shaw, M.; Broni, B.; Shapiro, G.; Haller, O. Inhibition of influenza viral mRNA synthesis in cells expressing the interferon-induced Mx gene product. *J. Virol.* **1985**, *56*, 201–206. [PubMed]

43. Haller, O.; Arnheiter, H.; Pavlovic, J.; Staeheli, P. The Discovery of the Antiviral Resistance Gene Mx: A Story of Great Ideas, Great Failures, and Some Success. *Annu. Rev. Virol.* **2018**, *5*, 33–51. [CrossRef] [PubMed]

44. Turan, K.; Mibayashi, M.; Sugiyama, K.; Saito, S.; Numajiri, A.; Nagata, K. Nuclear MxA proteins form a complex with influenza virus NP and inhibit the transcription of the engineered influenza virus genome. *Nucleic Acids Res.* **2004**, *32*, 643–652. [CrossRef] [PubMed]

45. Pavlovic, J.; Arzet, H.A.; Hefti, H.P.; Frese, M.; Rost, D.; Ernst, B.; Kolb, E.; Staeheli, P.; Haller, O. Enhanced virus resistance of transgenic mice expressing the human MxA protein. *J. Virol.* **1995**, *69*, 4506–4510. [PubMed]

46. Goujon, C.; Moncorge, O.; Bauby, H.; Doyle, T.; Ward, C.C.; Schaller, T.; Hue, S.; Barclay, W.S.; Schulz, R.; Malim, M.H. Human MX2 is an interferon-induced post-entry inhibitor of HIV-1 infection. *Nature* **2013**, *502*, 559–562. [CrossRef] [PubMed]

47. Brass, A.L.; Huang, I.C.; Benita, Y.; John, S.P.; Krishnan, M.N.; Feeley, E.M.; Ryan, B.J.; Weyer, J.L.; van der Weyden, L.; Fikrig, E.; et al. The IFITM proteins mediate cellular resistance to influenza A H1N1 virus, West Nile virus, and dengue virus. *Cell* **2009**, *139*, 1243–1254. [CrossRef] [PubMed]

48. Bailey, C.C.; Zhong, G.; Huang, I.C.; Farzan, M. IFITM-Family Proteins: The Cell’s First Line of Antiviral Defense. *Annu. Rev. Virol.* **2014**, *1*, 261–283. [CrossRef] [PubMed]

49. Diamond, M.S.; Farzan, M. The broad-spectrum antiviral functions of IFIT and IFITM proteins. *Nat. Rev. Immunol.* **2013**, *13*, 46–57. [CrossRef] [PubMed]

50. Clemens, M.J.; Williams, B.R. Inhibition of cell-free protein synthesis by pppA2′p5′A2′p5′A: A novel oligonucleotide synthesized by interferon-treated L cell extracts. *Cell* **1978**, *13*, 565–572. [CrossRef]

51. Silverman, R.H.; Weiss, S.R. Viral phosphodiesterases that antagonize double-stranded RNA signaling to RNase L by degrading 2-5A. *J. Interferon Cytokine Res.* **2014**, *34*, 455–463. [CrossRef] [PubMed]

52. Malathi, K.; Dong, B.; Gale, M., Jr.; Silverman, R.H. Small self-RNA generated by RNase L amplifies antiviral innate immunity. *Nature* 2007, 448, 816–819. [CrossRef](https://doi.org/10.1038/nature06008) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17637670)

53. Drappier, M.; Michiels, T. Inhibition of the OAS/RNase L pathway by viruses. *Curr. Opin. Virol.* 2015, 15, 19–26. [CrossRef](https://doi.org/10.1016/j.coviro.2015.06.001) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26163036)

54. Patel, R.C.; Sen, G.C. PACT, a protein activator of the interferon-induced protein kinase, PKR. *EMBO J.* 1998, 17, 4379–4390. [CrossRef](https://doi.org/10.1093/emboj/17.15.4379) [PubMed](https://pubmed.ncbi.nlm.nih.gov/9687505)

55. Kumar, A.; Haque, J.; Lacoste, J.; Hiscott, J.; Williams, B.R. Double-stranded RNA-dependent protein kinase activates transcription factor NF-kappa B by phosphorylating I kappa B. *Proc. Natl. Acad. Sci. USA* 1994, 91, 6288–6292. [CrossRef](https://doi.org/10.1073/pnas.91.14.6288) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8022810)

56. Schulz, O.; Pichlmair, A.; Rehwinkel, J.; Rogers, N.C.; Scheuner, D.; Kato, H.; Takeuchi, O.; Akira, S.; Kaufman, R.J.; Reis e Sousa, C. Protein kinase R contributes to immunity against specific viruses by regulating interferon mRNA integrity. *Cell Host Microbe* 2010, 7, 354–361. [CrossRef](https://doi.org/10.1016/j.chom.2010.03.009) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20412769)

57. Balachandran, S.; Roberts, P.C.; Brown, L.E.; Truong, H.; Pattnaik, A.K.; Archer, D.R.; Barber, G.N. Essential role for the dsRNA-dependent protein kinase PKR in innate immunity to viral infection. *Immunity* 2000, 13, 129–141. [CrossRef](https://doi.org/10.1016/S1074-7613(00)00016-8)

58. Lamb, R.A.; Lai, C.J. Sequence of interrupted and uninterrupted mRNAs and cloned DNA coding for the two overlapping nonstructural proteins of influenza virus. *Cell* 1980, 21, 475–485. [CrossRef](https://doi.org/10.1016/0092-8674(80)90468-0)

59. Robb, N.C.; Jackson, D.; Vreede, F.T.; Fodor, E. Splicing of influenza A virus NS1 mRNA is independent of the viral NS1 protein. *J. Gen. Virol.* 2010, 91, 2331–2340. [CrossRef](https://doi.org/10.1099/vir.0.021053-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20400666)

60. Marc, D. Influenza virus non-structural protein NS1: Interferon antagonism and beyond. *J. Gen. Virol.* 2014, 95, 2594–2611. [CrossRef](https://doi.org/10.1099/vir.0.066897-0)

61. Lohrmann, F.; Dijkman, R.; Stertz, S.; Thiel, V.; Haller, O.; Staeheli, P.; Kochs, G. Emergence of a C-terminal seven-amino-acid elongation of NS1 in around 1950 conferred a minor growth advantage to former seasonal influenza A viruses. *J. Virol.* 2013, 87, 11300–11303. [CrossRef](https://doi.org/10.1128/JVI.00989-13)

62. Chauche, C.; Nogales, A.; Zhu, H.; Goldfarb, D.; Ahmad Shanizza, A.I.; Gu, Q.; Parrish, C.R.; Martinez-Sobrido, L.; Marshall, J.F.; Murcia, P.R. Mammalian adaptation of an avian influenza A virus involves stepwise changes in NS1. *J. Virol.* 2018, 92, JVI-01875. [CrossRef](https://doi.org/10.1128/JVI.01875-17) [PubMed](https://pubmed.ncbi.nlm.nih.gov/29950394)

63. Chien, C.Y.; Tejero, R.; Huang, Y.; Zimmerman, D.E.; Rios, C.B.; Krug, R.M.; Montelione, G.T. A novel RNA-binding motif in influenza A virus non-structural protein 1. *Nat. Struct. Biol.* 1997, 4, 891–895. [CrossRef](https://doi.org/10.1038/nsb1197-891) [PubMed](https://pubmed.ncbi.nlm.nih.gov/9341178)

64. Liu, J.; Lynch, P.A.; Chien, C.Y.; Montelione, G.T.; Krug, R.M.; Berman, H.M. Crystal structure of the unique RNA-binding domain of the influenza virus NS1 protein. *Nat. Struct. Biol.* 1997, 4, 896–899. [CrossRef](https://doi.org/10.1038/nsb1197-896)

65. Wang, W.; Riedel, K.; Lynch, P.; Chien, C.Y.; Montelione, G.T.; Krug, R.M. RNA binding by the novel helical domain of the influenza virus NS1 protein requires its dimer structure and a small number of specific basic amino acids. *RNA* 1999, 5, 195–205. [CrossRef](https://doi.org/10.1017/S135583829998178X) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10072470)

66. Donelan, N.R.; Basler, C.F.; Garcia-Sastre, A. A recombinant influenza A virus expressing an RNA-binding-defective NS1 protein induces high levels of beta interferon and is attenuated in mice. *J. Virol.* 2003, 77, 13257–13266. [CrossRef](https://doi.org/10.1128/JVI.77.23.13257-13266.2003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/14612550)

67. Greenspan, D.; Palese, P.; Krystal, M. Two nuclear location signals in the influenza virus NS1 nonstructural protein. *J. Virol.* 1988, 62, 3020–3026. [PubMed](https://pubmed.ncbi.nlm.nih.gov/2843324)

68. Melen, K.; Kinnunen, L.; Fagerlund, R.; Ikonen, N.; Twu, K.Y.; Krug, R.M.; Julkunen, I. Nuclear and nucleolar targeting of influenza A virus NS1 protein: Striking differences between different virus subtypes. *J. Virol.* 2007, 81, 5995–6006. [CrossRef](https://doi.org/10.1128/JVI.02681-06)

69. Li, Y.; Yamakita, Y.; Krug, R.M. Regulation of a nuclear export signal by an adjacent inhibitory sequence: The effector domain of the influenza virus NS1 protein. *Proc. Natl. Acad. Sci. USA* 1998, 95, 4864–4869. [CrossRef](https://doi.org/10.1073/pnas.95.9.4864)

70. Hale, B.G.; Randall, R.E.; Ortin, J.; Jackson, D. The multifunctional NS1 protein of influenza A viruses. *J. Gen. Virol.* 2008, 89, 2359–2376. [CrossRef](https://doi.org/10.1099/vir.0.2008/000686-0)

71. Nogales, A.; Martinez-Sobrido, L.; Chiem, K.; Topham, D.J.; DeDiego, M.L. Functional Evolution of the 2009 Pandemic H1N1 Influenza NS1 and PA in Humans. *J. Virol.* 2018, 92, e01206-18. [CrossRef](https://doi.org/10.1128/JVI.01206-18)

72. Nogales, A.; Chauche, C.; DeDiego, M.L.; Topham, D.J.; Parrish, C.R.; Murcia, P.R.; Martinez-Sobrido, L. The K186E Amino Acid Substitution in the Canine Influenza Virus H3N8 NS1 Protein Restores Its Ability to Inhibit Host Gene Expression. *J. Virol.* 2017, 91, JVI-00877. [CrossRef](https://doi.org/10.1128/JVI.00877-17) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28717155)

73. Steidle, S.; Martinez-Sobrido, L.; Mordstein, M.; Lienenklaus, S.; Garcia-Sastre, A.; Staheli, P.; Kochs, G. Glycine 184 in nonstructural protein NS1 determines the virulence of influenza A virus strain PR8 without affecting the host interferon response. *J. Virol.* **2010**, *84*, 12761–12770. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

74. Kochs, G.; Garcia-Sastre, A.; Martinez-Sobrido, L. Multiple anti-interferon actions of the influenza A virus NS1 protein. *J. Virol.* **2007**, *81*, 7011–7021. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

75. Hale, B.G. Conformational plasticity of the influenza A virus NS1 protein. *J. Gen. Virol.* **2014**, *95*, 2099–2105. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

76. Jackson, D.; Hossain, M.J.; Hickman, D.; Perez, D.R.; Lamb, R.A. A new influenza virus virulence determinant: The NS1 protein four C-terminal residues modulate pathogenicity. *Proc. Natl. Acad. Sci. USA* **2008**, *105*, 4381–4386. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

77. Klemm, C.; Boergeling, Y.; Ludwig, S.; Ehrhardt, C. Immunomodulatory nonstructural proteins of influenza a Viruses. *Trends Microbiol.* **2018**, *26*, 624–636. [CrossRef](https://doi.org/CrossRef)

78. Garcia-Sastre, A.; Egorov, A.; Matassov, D.; Brandt, S.; Levy, D.E.; Durbin, J.E.; Palese, P.; Muster, T. Influenza A virus lacking the NS1 gene replicates in interferon-deficient systems. *Virology* **1998**, *252*, 324–330. [CrossRef](https://doi.org/CrossRef)

79. Kochs, G.; Koerner, I.; Thiel, L.; Kothlow, S.; Kaspers, B.; Ruggli, N.; Summerfield, A.; Pavlovic, J.; Stech, J.; Staeheli, P. Properties of H7N7 influenza A virus strain SC35M lacking interferon antagonist NS1 in mice and chickens. *J. Gen. Virol.* **2007**, *88*, 1403–1409. [CrossRef](https://doi.org/CrossRef)

80. Nogales, A.; Huang, K.; Chauche, C.; DeDiego, M.L.; Murcia, P.R.; Parrish, C.R.; Martinez-Sobrido, L. Canine influenza viruses with modified NS1 proteins for the development of live-attenuated vaccines. *Virology* **2017**, *500*, 1–10. [CrossRef](https://doi.org/CrossRef)

81. Talon, J.; Salvatore, M.; O’Neill, R.E.; Nakaya, Y.; Zheng, H.; Muster, T.; Garcia-Sastre, A.; Palese, P. Influenza A and B viruses expressing altered NS1 proteins: A vaccine approach. *Proc. Natl. Acad. Sci. USA* **2000**, *97*, 4309–4314. [CrossRef](https://doi.org/CrossRef)

82. Nogales, A.; Baker, S.F.; Ortiz-Riano, E.; Dewhurst, S.; Topham, D.J.; Martinez-Sobrido, L. Influenza A virus attenuation by codon deoptimization of the NS gene for vaccine development. *J. Virol.* **2014**, *88*, 10525–10540. [CrossRef](https://doi.org/CrossRef)

83. DeDiego, M.L.; Nogales, A.; Lambert-Emo, K.; Martinez-Sobrido, L.; Topham, D.J. NS1 Protein Mutation I64T Affects Interferon Responses and Virulence of Circulating H3N2 Human Influenza A Viruses. *J. Virol.* **2016**, *90*, 9693–9711. [CrossRef](https://doi.org/CrossRef)

84. Nogales, A.; Martinez-Sobrido, L.; Topham, D.J.; DeDiego, M.L. NS1 Protein Amino Acid Changes D189N and V194I Affect Interferon Responses, Thermosensitivity, and Virulence of Circulating H3N2 Human Influenza A Viruses. *J. Virol.* **2017**, *91*, e01930-16. [CrossRef](https://doi.org/CrossRef)

85. Trapp, S.; Soubieux, D.; Lidove, A.; Esnault, E.; Lion, A.; Guillory, V.; Wacquier, A.; Kut, E.; Quere, P.; Larcher, T.; et al. Major contribution of the RNA-binding domain of NS1 in the pathogenicity and replication potential of an avian H7N1 influenza virus in chickens. *Virol. J.* **2018**, *15*, 55. [CrossRef](https://doi.org/CrossRef)

86. Newby, C.M.; Sabin, L.; Pekosz, A. The RNA binding domain of influenza A virus NS1 protein affects secretion of tumor necrosis factor alpha, interleukin-6, and interferon in primary murine tracheal epithelial cells. *J. Virol.* **2007**, *81*, 9469–9480. [CrossRef](https://doi.org/CrossRef)

87. Min, J.Y.; Krug, R.M. The primary function of RNA binding by the influenza A virus NS1 protein in infected cells: Inhibiting the 2′-5′ oligo (A) synthetase/RNase L pathway. *Proc. Natl. Acad. Sci. USA* **2006**, *103*, 7100–7105. [CrossRef](https://doi.org/CrossRef)

88. Mibayashi, M.; Martinez-Sobrido, L.; Loo, Y.M.; Cardenas, W.B.; Gale, M., Jr.; Garcia-Sastre, A. Inhibition of retinoic acid-inducible gene I-mediated induction of beta interferon by the NS1 protein of influenza A virus. *J. Virol.* **2007**, *81*, 514–524. [CrossRef](https://doi.org/CrossRef)

89. Guo, Z.; Chen, L.M.; Zeng, H.; Gomez, J.A.; Plowden, J.; Fujita, T.; Katz, J.M.; Donis, R.O.; Sambhara, S. NS1 protein of influenza A virus inhibits the function of intracytoplasmic pathogen sensor, RIG-I. *Am. J. Respir. Cell Mol. Biol.* **2007**, *36*, 263–269. [CrossRef](https://doi.org/CrossRef)

90. Opitz, B.; Rejaibi, A.; Dauber, B.; Eckhard, J.; Vinzing, M.; Schmeck, B.; Hippenstiel, S.; Suttrop, N.; Wolff, T. IFNbata induction by influenza A virus is mediated by RIG-I which is regulated by the viral NS1 protein. *Cell. Microbiol.* **2007**, *9*, 930–938. [CrossRef](https://doi.org/CrossRef)

91. Gack, M.U.; Albrecht, R.A.; Urano, T.; Inn, K.S.; Huang, I.C.; Carnero, E.; Farzan, M.; Inoue, S.; Jung, J.U.; Garcia-Sastre, A. Influenza A virus NS1 targets the ubiquitin ligase TRIM25 to evade recognition by the host viral RNA sensor RIG-I. *Cell Host Microbe* **2009**, *5*, 439–449. [CrossRef](https://doi.org/CrossRef)

92. Rajsbaum, R.; Albrecht, R.A.; Wang, M.K.; Maharaj, N.P.; Versteeg, G.A.; Nistal-Villan, E.; Garcia-Sastre, A.; Gack, M.U. Species-specific inhibition of RIG-I ubiquitination and IFN induction by the influenza A virus NS1 protein. *PLoS Pathog.* **2012**, *8*, e1003059. [CrossRef]

93. Koliopoulos, M.G.; Lethier, M.; van der Veen, A.G.; Haubrich, K.; Hennig, J.; Kowalinski, E.; Stevens, R.V.; Martin, S.R.; Reis e Sousa, C.; Cusack, S.; et al. Molecular mechanism of influenza A NS1-mediated TRIM25 recognition and inhibition. *Nat. Commun.* **2018**, *9*, 1820. [CrossRef]

94. Feng, W.; Sun, X.; Shi, N.; Zhang, M.; Guan, Z.; Duan, M. Influenza a virus NS1 protein induced A20 contributes to viral replication by suppressing interferon-induced antiviral response. *Biochem. Biophys. Res. Commun.* **2017**, *482*, 1107–1113. [CrossRef]

95. Maelfait, J.; Roose, K.; Vereecke, L.; Mc Guire, C.; Sze, M.; Schuijs, M.J.; Willart, M.; Ibanez, L.I.; Hammad, H.; Lambrecht, B.N.; et al. A20 Deficiency in Lung Epithelial Cells Protects against Influenza A Virus Infection. *PLoS Pathog.* **2016**, *12*, e1005410. [CrossRef]

96. Pichlmair, A.; Schulz, O.; Tan, C.P.; Naslund, T.I.; Liljestrom, P.; Weber, F.; Reis e Sousa, C. RIG-I-mediated antiviral responses to single-stranded RNA bearing 5′-phosphates. *Science* **2006**, *314*, 997–1001. [CrossRef]

97. Marazzi, I.; Ho, J.S.; Kim, J.; Manicassamy, B.; Dewell, S.; Albrecht, R.A.; Seibert, C.W.; Schaefer, U.; Jeffrey, K.L.; Prinjha, R.K.; et al. Suppression of the antiviral response by an influenza histone mimic. *Nature* **2012**, *483*, 428–433. [CrossRef]

98. Gao, S.; Song, L.; Li, J.; Zhang, Z.; Peng, H.; Jiang, W.; Wang, Q.; Kang, T.; Chen, S.; Huang, W. Influenza A virus-encoded NS1 virulence factor protein inhibits innate immune response by targeting IKK. *Cell. Microbiol.* **2012**, *14*, 1849–1866. [CrossRef]

99. Talon, J.; Horvath, C.M.; Polley, R.; Basler, C.F.; Muster, T.; Palese, P.; Garcia-Sastre, A. Activation of interferon regulatory factor 3 is inhibited by the influenza A virus NS1 protein. *J. Virol.* **2000**, *74*, 7989–7996. [CrossRef]

100. Ludwig, S.; Wang, X.; Ehrhardt, C.; Zheng, H.; Donelan, N.; Planz, O.; Pleschka, S.; Garcia-Sastre, A.; Heins, G.; Wolff, T. The influenza A virus NS1 protein inhibits activation of Jun N-terminal kinase and AP-1 transcription factors. *J. Virol.* **2002**, *76*, 11166–11171. [CrossRef]

101. Li, S.; Min, J.Y.; Krug, R.M.; Sen, G.C. Binding of the influenza A virus NS1 protein to PKR mediates the inhibition of its activation by either PACT or double-stranded RNA. *Virology* **2006**, *349*, 13–21. [CrossRef]

102. Min, J.Y.; Li, S.; Sen, G.C.; Krug, R.M. A site on the influenza A virus NS1 protein mediates both inhibition of PKR activation and temporal regulation of viral RNA synthesis. *Virology* **2007**, *363*, 236–243. [CrossRef]

103. Schierhorn, K.L.; Jolmes, F.; Bespalowa, J.; Saenger, S.; Peteranderl, C.; Dzieciolowski, J.; Mielke, M.; Budt, M.; Pleschka, S.; Herrmann, A.; et al. Influenza A Virus Virulence Depends on Two Amino Acids in the N-Terminal Domain of Its NS1 Protein To Facilitate Inhibition of the RNA-Dependent Protein Kinase PKR. *J. Virol.* **2017**, *91*, e00198-17. [CrossRef]

104. Tawaratsumida, K.; Phan, V.; Hrincius, E.R.; High, A.A.; Webby, R.; Redecke, V.; Hacker, H. Quantitative proteomic analysis of the influenza A virus nonstructural proteins NS1 and NS2 during natural cell infection identifies PACT as an NS1 target protein and antiviral host factor. *J. Virol.* **2014**, *88*, 9038–9048. [CrossRef]

105. Cheong, W.C.; Kang, H.R.; Yoon, H.; Kang, S.J.; Ting, J.P.; Song, M.J. Influenza A Virus NS1 Protein Inhibits the NLRP3 Inflammasome. *PLoS ONE* **2015**, *10*, e0126456. [CrossRef]

106. Park, H.S.; Liu, G.; Thulasi Raman, S.N.; Landreth, S.L.; Liu, Q.; Zhou, Y. NS1 Protein of 2009 Pandemic Influenza A Virus Inhibits Porcine NLRP3 Inflammasome-Mediated Interleukin-1 Beta Production by Suppressing ASC Ubiquitination. *J. Virol.* **2018**, *92*, JVI-00022. [CrossRef]

107. Moriyama, M.; Chen, I.Y.; Kawaguchi, A.; Koshiba, T.; Nagata, K.; Takeyama, H.; Hasegawa, H.; Ichinohe, T. The RNA- and TRIM25-Binding Domains of Influenza Virus NS1 Protein Are Essential for Suppression of NLRP3 Inflammasome-Mediated Interleukin-1beta Secretion. *J. Virol.* **2016**, *90*, 4105–4114. [CrossRef]

108. Das, K.; Ma, L.C.; Xiao, R.; Radvansky, B.; Aramini, J.; Zhao, L.; Marklund, J.; Kuo, R.L.; Twu, K.Y.; Arnold, E.; et al. Structural basis for suppression of a host antiviral response by influenza A virus. *Proc. Natl. Acad. Sci. USA* **2008**, *105*, 13093–13098. [CrossRef]

109. Noah, D.L.; Twu, K.Y.; Krug, R.M. Cellular antiviral responses against influenza A virus are countered at the posttranscriptional level by the viral NS1A protein via its binding to a cellular protein required for the 3′ end processing of cellular pre-mRNAs. *Virology* **2003**, *307*, 386–395. [CrossRef]

110. Nemeroff, M.E.; Barabino, S.M.; Li, Y.; Keller, W.; Krug, R.M. Influenza virus NS1 protein interacts with the cellular 30 kDa subunit of CPSF and inhibits 3′end formation of cellular pre-mRNAs. *Mol. Cell* **1998**, *1*, 991–1000. [CrossRef]

111. Chen, Z.; Li, Y.; Krug, R.M. Influenza A virus NS1 protein targets poly(A)-binding protein II of the cellular 3′-end processing machinery. *EMBO J.* **1999**, *18*, 2273–2283. [CrossRef]

112. Tu, J.; Guo, J.; Zhang, A.; Zhang, W.; Zhao, Z.; Zhou, H.; Liu, C.; Chen, H.; Jin, M. Effects of the C-terminal truncation in NS1 protein of the 2009 pandemic H1N1 influenza virus on host gene expression. *PLoS ONE* **2011**, *6*, e26175. [CrossRef]

113. Aragon, T.; de la Luna, S.; Novoa, I.; Carrasco, L.; Ortin, J.; Nieto, A. Eukaryotic translation initiation factor 4GI is a cellular target for NS1 protein, a translational activator of influenza virus. *Mol. Cell. Biol.* **2000**, *20*, 6259–6268. [CrossRef]

114. Hale, B.G.; Jackson, D.; Chen, Y.H.; Lamb, R.A.; Randall, R.E. Influenza A virus NS1 protein binds p85beta and activates phosphatidylinositol-3-kinase signaling. *Proc. Natl. Acad. Sci. USA* **2006**, *103*, 14194–14199. [CrossRef]

115. Li, Y.; Anderson, D.H.; Liu, Q.; Zhou, Y. Mechanism of influenza A virus NS1 protein interaction with the p85beta, but not the p85alpha, subunit of phosphatidylinositol 3-kinase (PI3K) and up-regulation of PI3K activity. *J. Biol. Chem.* **2008**, *283*, 23397–23409. [CrossRef]

116. Satterly, N.; Tsai, P.L.; van Deursen, J.; Nussenzveig, D.R.; Wang, Y.; Faria, P.A.; Levay, A.; Levy, D.E.; Fontoura, B.M. Influenza virus targets the mRNA export machinery and the nuclear pore complex. *Proc. Natl. Acad. Sci. USA* **2007**, *104*, 1853–1858. [CrossRef]

117. Wang, X.; Li, M.; Zheng, H.; Muster, T.; Palese, P.; Beg, A.A.; Garcia-Sastre, A. Influenza A virus NS1 protein prevents activation of NF-kappaB and induction of alpha/beta interferon. *J. Virol.* **2000**, *74*, 11566–11573. [CrossRef]

118. Stasakova, J.; Ferko, B.; Kittel, C.; Sereinig, S.; Romanova, J.; Katinger, H.; Egorov, A. Influenza A mutant viruses with altered NS1 protein function provoke caspase-1 activation in primary human macrophages, resulting in fast apoptosis and release of high levels of interleukins 1beta and 18. *J. Gen. Virol.* **2005**, *86*, 185–195. [CrossRef]

119. Ehrhardt, C.; Wolff, T.; Pleschka, S.; Planz, O.; Beermann, W.; Bode, J.G.; Schmolke, M.; Ludwig, S. Influenza A virus NS1 protein activates the PI3K/Akt pathway to mediate antiapoptotic signaling responses. *J. Virol.* **2007**, *81*, 3058–3067. [CrossRef]

120. Hrincius, E.R.; Dierkes, R.; Anhlan, D.; Wixler, V.; Ludwig, S.; Ehrhardt, C. Phosphatidylinositol-3-kinase (PI3K) is activated by influenza virus vRNA via the pathogen pattern receptor Rig-I to promote efficient type I interferon production. *Cell. Microbiol.* **2011**, *13*, 1907–1919. [CrossRef]

121. Lu, X.; Masic, A.; Liu, Q.; Zhou, Y. Regulation of influenza A virus induced CXCL-10 gene expression requires PI3K/Akt pathway and IRF3 transcription factor. *Mol. Immunol.* **2011**, *48*, 1417–1423. [CrossRef]

122. Shin, Y.K.; Li, Y.; Liu, Q.; Anderson, D.H.; Babiu, L.A.; Zhou, Y. SH3 binding motif 1 in influenza A virus NS1 protein is essential for PI3K/Akt signaling pathway activation. *J. Virol.* **2007**, *81*, 12730–12739. [CrossRef]

123. Chan, S.; Choi, E.A.; Shi, Y. Pre-mRNA 3′-end processing complex assembly and function. *Wiley Interdiscip. Rev. RNA* **2011**, *2*, 321–335. [CrossRef]

124. Levene, R.E.; Gaglia, M.M. Host Shutoff in Influenza A Virus: Many Means to an End. *Viruses* **2018**, *10*, 475. [CrossRef]

125. Hale, B.G.; Steel, J.; Medina, R.A.; Manicassamy, B.; Ye, J.; Hickman, D.; Hai, R.; Schmolke, M.; Lowen, A.C.; Perez, D.R.; et al. Inefficient control of host gene expression by the 2009 pandemic H1N1 influenza A virus NS1 protein. *J. Virol.* **2010**, *84*, 6909–6922. [CrossRef]

126. Clark, A.M.; Nogales, A.; Martinez-Sobrido, L.; Topham, D.J.; DeDiego, M.L. Functional Evolution of Influenza Virus NS1 Protein in Currently Circulating Human 2009 Pandemic H1N1 Viruses. *J. Virol.* **2017**, *91*, JVI-00721. [CrossRef]

127. Hale, B.G.; Steel, J.; Manicassamy, B.; Medina, R.A.; Ye, J.; Hickman, D.; Lowen, A.C.; Perez, D.R.; Garcia-Sastre, A. Mutations in the NS1 C-terminal tail do not enhance replication or virulence of the 2009 pandemic H1N1 influenza A virus. *J. Gen. Virol.* **2010**, *91*, 1737–1742. [CrossRef]

128. Anastasina, M.; Le May, N.; Bugai, A.; Fu, Y.; Soderholm, S.; Gaelings, L.; Ohman, T.; Tynell, J.; Kyttanen, S.; Barboric, M.; et al. Influenza virus NS1 protein binds cellular DNA to block transcription of antiviral genes. *Biochim. Biophys. Acta* **2016**, *1859*, 1440–1448. [CrossRef]

129. Pica, N.; Langlois, R.A.; Krammer, F.; Margine, I.; Palese, P. NS1-truncated live attenuated virus vaccine provides robust protection to aged mice from viral challenge. *J. Virol.* **2012**, *86*, 10293–10301. [CrossRef]

130. Kochs, G.; Martinez-Sobrido, L.; Lienenklaus, S.; Weiss, S.; Garcia-Sastre, A.; Staeheli, P. Strong interferon-inducing capacity of a highly virulent variant of influenza A virus strain PR8 with deletions in the NS1 gene. *J. Gen. Virol.* 2009, *90*, 2990–2994. [CrossRef]

131. Steel, J.; Lowen, A.C.; Pena, L.; Angel, M.; Solorzano, A.; Albrecht, R.; Perez, D.R.; Garcia-Sastre, A.; Palese, P. Live attenuated influenza viruses containing NS1 truncations as vaccine candidates against H5N1 highly pathogenic avian influenza. *J. Virol.* 2009, *83*, 1742–1753. [CrossRef]

132. Baskin, C.R.; Bielefeldt-Ohmann, H.; Garcia-Sastre, A.; Tumpey, T.M.; Van Hoeven, N.; Carter, V.S.; Thomas, M.J.; Proll, S.; Solorzano, A.; Billharz, R.; et al. Functional genomic and serological analysis of the protective immune response resulting from vaccination of macaques with an NS1-truncated influenza virus. *J. Virol.* 2007, *81*, 11817–11827. [CrossRef]

133. Chambers, T.M.; Quinlivan, M.; Sturgill, T.; Cullinane, A.; Horohov, D.W.; Zamarin, D.; Arkins, S.; Garcia-Sastre, A.; Palese, P. Influenza A viruses with truncated NS1 as modified live virus vaccines: Pilot studies of safety and efficacy in horses. *Equine Vet. J.* 2009, *41*, 87–92. [CrossRef]

134. Quinlivan, M.; Zamarin, D.; Garcia-Sastre, A.; Cullinane, A.; Chambers, T.; Palese, P. Attenuation of equine influenza viruses through truncations of the NS1 protein. *J. Virol.* 2005, *79*, 8431–8439. [CrossRef]

135. Ayllon, J.; Domingues, P.; Rajsbaum, R.; Miorin, L.; Schmolke, M.; Hale, B.G.; Garcia-Sastre, A. A single amino acid substitution in the novel H7N9 influenza A virus NS1 protein increases CPSF30 binding and virulence. *J. Virol.* 2014, *88*, 12146–12151. [CrossRef]

136. Spesock, A.; Malur, M.; Hossain, M.J.; Chen, L.M.; Njaa, B.L.; Davis, C.T.; Lipatov, A.S.; York, I.A.; Krug, R.M.; Donis, R.O. The virulence of 1997 H5N1 influenza viruses in the mouse model is increased by correcting a defect in their NS1 proteins. *J. Virol.* 2011, *85*, 7048–7058. [CrossRef]

137. Kuo, R.L.; Zhao, C.; Malur, M.; Krug, R.M. Influenza A virus strains that circulate in humans differ in the ability of their NS1 proteins to block the activation of IRF3 and interferon-beta transcription. *Virology* 2010, *408*, 146–158. [CrossRef]

138. Twu, K.Y.; Kuo, R.L.; Marklund, J.; Krug, R.M. The H5N1 influenza virus NS genes selected after 1998 enhance virus replication in mammalian cells. *J. Virol.* 2007, *81*, 8112–8121. [CrossRef]

139. Kaewborisuth, C.; Kaplan, B.; Zanin, M.; Finkelstein, D.; Webby, R.J.; Lekcharoensuk, P. G45R on nonstructural protein 1 of influenza A virus contributes to virulence by increasing the expression of proinflammatory cytokines in mice. *Arch. Virol.* 2017, *162*, 45–55. [CrossRef]

140. Seo, S.H.; Hoffmann, E.; Webster, R.G. Lethal H5N1 influenza viruses escape host anti-viral cytokine responses. *Nat. Med.* 2002, *8*, 950–954. [CrossRef]

141. Lipatov, A.S.; Andreansky, S.; Webby, R.J.; Hulse, D.J.; Rehg, J.E.; Krauss, S.; Perez, D.R.; Doherty, P.C.; Webster, R.G.; Sangster, M.Y. Pathogenesis of Hong Kong H5N1 influenza virus NS gene reassortants in mice: The role of cytokines and B- and T-cell responses. *J. Gen. Virol.* 2005, *86*, 1121–1130. [CrossRef]

142. Ramos, I.; Fernandez-Sesma, A. Modulating the Innate Immune Response to Influenza A Virus: Potential Therapeutic Use of Anti-Inflammatory Drugs. *Front. Immunol.* 2015, *6*, 361. [CrossRef]

143. Nogales, A.; Martinez-Sobrido, L. Reverse genetics approaches for the development of influenza vaccines. *Int. J. Mol. Sci.* 2016, *18*, 20. [CrossRef]

144. Belongia, E.A.; Kieke, B.A.; Donahue, J.G.; Greenlee, R.T.; Balish, A.; Foust, A.; Lindstrom, S.; Shay, D.K. Effectiveness of inactivated influenza vaccines varied substantially with antigenic match from the 2004–2005 season to the 2006–2007 season. *J. Infect Dis.* 2009, *199*, 159–167. [CrossRef]

145. Osterholm, M.T.; Kelley, N.S.; Sommer, A.; Belongia, E.A. Efficacy and effectiveness of influenza vaccines: A systematic review and meta-analysis. *Lancet Infect. Dis.* 2012, *12*, 36–44. [CrossRef]

146. Kim, H.; Webster, R.G.; Webby, R.J. Influenza Virus: Dealing with a Drifting and Shifting Pathogen. *Viral Immunol.* 2018, *31*, 174–183. [CrossRef]

147. DeDiego, M.L.; Anderson, C.S.; Yang, H.; Holden-Wiltse, J.; Fitzgerald, T.; Treanor, J.J.; Topham, D.J. Directed selection of influenza virus produces antigenic variants that match circulating human virus isolates and escape from vaccine-mediated immune protection. *Immunology* 2016, *148*, 160–173. [CrossRef]

148. Wong, S.S.; Webby, R.J. Traditional and new influenza vaccines. *Clin. Microbiol. Rev.* 2013, *26*, 476–492. [CrossRef]

149. Belshe, R.B.; Edwards, K.M.; Vesikari, T.; Black, S.V.; Walker, R.E.; Hultquist, M.; Kemble, G.; Connor, E.M.; Group, C.-T.C.E.S. Live attenuated versus inactivated influenza vaccine in infants and young children. *N. Engl. J. Med.* 2007, *356*, 685–696. [CrossRef]

150. Cox, M.M.; Patriarca, P.A.; Treanor, J. FluBlok, a recombinant hemagglutinin influenza vaccine. *Influenza Other Respir Viruses* **2008**, *2*, 211–219. [CrossRef]

151. Chan, W.; Zhou, H.; Kemble, G.; Jin, H. The cold adapted and temperature sensitive influenza A/Ann Arbor/6/60 virus, the master donor virus for live attenuated influenza vaccines, has multiple defects in replication at the restrictive temperature. *Virology* **2008**, *380*, 304–311. [CrossRef]

152. Zhou, B.; Li, Y.; Belser, J.A.; Pearce, M.B.; Schmolke, M.; Subba, A.X.; Shi, Z.; Zaki, S.R.; Blau, D.M.; Garcia-Sastre, A.; et al. NS-based live attenuated H1N1 pandemic vaccines protect mice and ferrets. *Vaccine* **2010**, *28*, 8015–8025. [CrossRef] [PubMed]

153. Mossler, C.; Groiss, F.; Wolzt, M.; Wolschek, M.; Seipelt, J.; Muster, T. Phase I/II trial of a replication-deficient trivalent influenza virus vaccine lacking NS1. *Vaccine* **2013**, *31*, 6194–6200. [CrossRef]

154. Du, Y.; Xin, L.; Shi, Y.; Zhang, T.H.; Wu, N.C.; Dai, L.; Gong, D.; Brar, G.; Shu, S.; Luo, J.; et al. Genome-wide identification of interferon-sensitive mutations enables influenza vaccine design. *Science* **2018**, *359*, 290–296. [CrossRef] [PubMed]

155. Jagger, B.W.; Wise, H.M.; Kash, J.C.; Walters, K.A.; Wills, N.M.; Xiao, Y.L.; Dunfee, R.L.; Schwartzman, L.M.; Ozinsky, A.; Bell, G.L.; et al. An Overlapping Protein-Coding Region in Influenza A Virus Segment 3 Modulates the Host Response. *Science* **2012**, *337*, 199–204. [CrossRef] [PubMed]

156. Resa-Infante, P.; Jorba, N.; Coloma, R.; Ortin, J. The influenza virus RNA synthesis machine: Advances in its structure and function. *RNA Biol.* **2011**, *8*, 207–215. [CrossRef]

157. Firth, A.E.; Jagger, B.W.; Wise, H.M.; Nelson, C.C.; Parsawar, K.; Wills, N.M.; Napthine, S.; Taubenberger, J.K.; Digard, P.; Atkins, J.F. Ribosomal frameshifting used in influenza A virus expression occurs within the sequence UCC_UUU_CGU and is in the +1 direction. *Open Biol.* **2012**, *2*, 120109. [CrossRef]

158. Shi, M.; Jagger, B.W.; Wise, H.M.; Digard, P.; Holmes, E.C.; Taubenberger, J.K. Evolutionary conservation of the PA-X open reading frame in segment 3 of influenza A virus. *J. Virol.* **2012**, *86*, 12411–12413. [CrossRef]

159. Hayashi, T.; MacDonald, L.A.; Takimoto, T. Influenza A Virus Protein PA-X Contributes to Viral Growth and Suppression of the Host Antiviral and Immune Responses. *J. Virol.* **2015**, *89*, 6442–6452. [CrossRef]

160. Khaperskyy, D.A.; McCormick, C. Timing Is Everything: Coordinated Control of Host Shutoff by Influenza A Virus NS1 and PA-X Proteins. *J. Virol.* **2015**, *89*, 6528–6531. [CrossRef]

161. Khaperskyy, D.A.; Schmaling, S.; Larkins-Ford, J.; McCormick, C.; Gaglia, M.M. Selective Degradation of Host RNA Polymerase II Transcripts by Influenza A Virus PA-X Host Shutoff Protein. *PLoS Pathog.* **2016**, *12*, e1005427. [CrossRef]

162. Hu, J.; Ma, C.; Liu, X. PA-X: A key regulator of influenza A virus pathogenicity and host immune responses. *Med. Microbiol. Immunol.* **2018**, *207*, 1–15. [CrossRef]

163. Hu, J.; Mo, Y.Q.; Wang, X.Q.; Gu, M.; Hu, Z.L.; Zhong, L.; Wu, Q.W.; Hao, X.L.; Hu, S.L.; Liu, W.B.; et al. PA-X Decreases the Pathogenicity of Highly Pathogenic H5N1 Influenza A Virus in Avian Species by Inhibiting Virus Replication and Host Response. *J. Virol.* **2015**, *89*, 4126–4142. [CrossRef] [PubMed]

164. Bavagnoli, L.; Cucuzza, S.; Campanini, G.; Rovida, F.; Paolucci, S.; Baldanti, F.; Maga, G. The novel influenza A virus protein PA-X and its naturally deleted variant show different enzymatic properties in comparison to the viral endonuclease PA. *Nucleic Acids Res.* **2015**, *43*, 9405–9417. [CrossRef]

165. Gao, H.J.; Xu, G.L.; Sun, Y.P.; Qi, L.; Wang, J.L.; Kong, W.L.; Sun, H.L.; Pu, J.; Chang, K.C.; Liu, J.H. PA-X is a virulence factor in avian H9N2 influenza virus. *J. Gen. Virol.* **2015**, *96*, 2587–2594. [CrossRef] [PubMed]

166. Lee, J.; Yu, H.; Li, Y.; Ma, J.; Lang, Y.; Duff, M.; Henningson, J.; Liu, Q.; Li, Y.; Nagy, A.; et al. Impacts of different expressions of PA-X protein on 2009 pandemic H1N1 virus replication, pathogenicity and host immune responses. *Virology* **2017**, *504*, 25–35. [CrossRef]

167. Hayashi, T.; Chaimayo, C.; McGuinness, J.; Takimoto, T. Critical Role of the PA-X C-Terminal Domain of Influenza A Virus in Its Subcellular Localization and Shutoff Activity. *J. Virol.* **2016**, *90*, 7131–7141. [CrossRef]

168. Desmet, E.A.; Bussey, K.A.; Stone, R.; Takimoto, T. Identification of the N-terminal domain of the influenza virus PA responsible for the suppression of host protein synthesis. *J. Virol.* **2013**, *87*, 3108–3118. [CrossRef]

169. Oishi, K.; Yamayoshi, S.; Kawaoka, Y. Mapping of a Region of the PA-X Protein of Influenza A Virus That Is Important for Its Shutoff Activity. *J. Virol.* **2015**, *89*, 8661–8665. [CrossRef] [PubMed]

170. Gao, H.; Sun, H.; Hu, J.; Qi, L.; Wang, J.; Xiong, X.; Wang, Y.; He, Q.; Lin, Y.; Kong, W.; et al. Twenty amino acids at the C-terminus of PA-X are associated with increased influenza A virus replication and pathogenicity. *J. Gen. Virol.* **2015**, *96*, 2036–2049. [CrossRef]

171. Oishi, K.; Yamayoshi, S.; Kawaoka, Y. Identification of novel amino acid residues of influenza virus PA-X that are important for PA-X shutoff activity by using yeast. *Virology* **2018**, *516*, 71–75. [CrossRef](https://doi.org/10.1016/j.virol.2018.03.004)

172. Gao, H.; Sun, Y.; Hu, J.; Qi, L.; Wang, J.; Xiong, X.; Wang, Y.; He, Q.; Lin, Y.; Kong, W.; et al. The contribution of PA-X to the virulence of pandemic 2009 H1N1 and highly pathogenic H5N1 avian influenza viruses. *Sci. Rep.* **2015**, *5*, 8262. [CrossRef](https://doi.org/10.1038/srep08262) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25693888)

173. Hu, J.; Mo, Y.; Gao, Z.; Wang, X.; Gu, M.; Liang, Y.; Cheng, X.; Hu, S.; Liu, W.; Liu, H.; et al. PA-X-associated early alleviation of the acute lung injury contributes to the attenuation of a highly pathogenic H5N1 avian influenza virus in mice. *Med. Microbiol. Immunol.* **2016**, *205*, 381–395. [CrossRef](https://doi.org/10.1007/s00430-016-0441-3) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26984566)

174. Nogales, A.; Rodriguez, L.; DeDiego, M.L.; Topham, D.J.; Martinez-Sobrido, L. Interplay of PA-X and NS1 Proteins in Replication and Pathogenesis of a Temperature-Sensitive 2009 Pandemic H1N1 Influenza A Virus. *J. Virol.* **2017**, *91*, JVI-00720. [CrossRef](https://doi.org/10.1128/JVI.00720-17) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28450586)

175. Aydillo, T.; Ayllon, J.; Pavlisin, A.; Martinez-Romero, C.; Tripathi, S.; Mena, I.; Moreira-Soto, A.; Vicente-Santos, A.; Corrales-Aguilar, E.; Schwemmle, M.; et al. Specific Mutations in the PB2 Protein of Influenza A Virus Compensate for the Lack of Efficient Interferon Antagonism of the NS1 Protein of Bat Influenza A-Like Viruses. *J. Virol.* **2018**, *92*, JVI-02021. [CrossRef](https://doi.org/10.1128/JVI.02021-17) [PubMed](https://pubmed.ncbi.nlm.nih.gov/29472586)

176. Toback, S.L.; Levin, M.J.; Block, S.L.; Belshe, R.B.; Ambrose, C.S.; Falloon, J. Quadrivalent Ann Arbor strain live-attenuated influenza vaccine. *Expert Rev. Vaccines* **2012**, *11*, 1293–1303. [CrossRef](https://doi.org/10.1586/erv.12.100)

177. Caspard, H.; Mallory, R.M.; Yu, J.; Ambrose, C.S. Live-attenuated influenza vaccine effectiveness in children from 2009 to 2015–2016: A systematic review and meta-analysis. *Open Forum Infect. Dis.* **2017**, *4*, ofx111. [CrossRef](https://doi.org/10.1093/ofid/ofx111) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28553519)
