
Review

# Virus–Host Interactions Involved in Lassa Virus Entry and Genome Replication

**María Eugenia Loureiro**, *Alejandra D'Antuono* and Nora López

Centro de Virología Animal (CEVAN), CONICET-SENASA, Av Sir Alexander Fleming 1653, Martínez, Provincia de Buenos Aires B1640CSI, Argentina; adantuono@gmail.com (A.D.); noramlopar@gmail.com (N.L.)

* Correspondence: eugenialoureiro@yahoo.com.ar; Tel.: +54-11-4105-4100

Received: 21 December 2018; Accepted: 26 January 2019; Published: 29 January 2019

---

**Abstract:** Lassa virus (LASV) is the causative agent of Lassa fever, a human hemorrhagic disease associated with high mortality and morbidity rates, particularly prevalent in West Africa. Over the past few years, a significant amount of novel information has been provided on cellular factors that are determinant elements playing a role in arenavirus multiplication. In this review, we focus on host proteins that intersect with the initial steps of the LASV replication cycle: virus entry and genome replication. A better understanding of relevant virus–host interactions essential for sustaining these critical steps may help to identify possible targets for the rational design of novel therapeutic approaches against LASV and other arenaviruses that cause severe human disease.

**Keywords:** Lassa fever; arenavirus; LASV; virus–host interactions; entry; replication

---

## 1. Introduction

The Arenaviridae family includes viruses carried by mammalian hosts, classified in the Mammarenavirus genus, and members that infect reptilian hosts, which belong to the Reptarenavirus and Hartmanivirus genera [1]. Mammarenaviruses comprise 35 currently recognized species that are classified into two main groups, Old World (OW) and New World (NW) viruses. Within the NW group, viruses are divided into Clade A, Clade A-recombinant (Clade D), Clade B, and Clade C, according to their phylogenetic relationships. Clade B includes the apathogenic Tacaribe virus (TCRV), along with the known South American pathogens that produce severe hemorrhagic disease in humans: Junín virus (JUNV), the causative agent of Argentine hemorrhagic fever; and Machupo, Chapare, Guanarito, and Sabia viruses. OW mammarenaviruses include the prototypic lymphocytic choriomeningitis virus (LCMV), of worldwide distribution, and other viruses endemic to the African continent such as Mopeia (MOPV), Lujo (LUJV), and Lassa virus (LASV). LASV is the causative agent of Lassa fever (LF), a human hemorrhagic disease transmitted through contact with infected rodents (*Mastomys* spp.) that is particularly prevalent in Nigeria, Liberia, Sierra Leone, and Guinea. After infection, an average incubation time of 10 days is usually followed by general flu-like symptoms, including fever, malaise, and headache. Hemorrhagic and/or neurologic involvement can be associated with severe cases of LF [2]. Up to 500,000 infections and >5000 deaths occur every year, with mortality rates which can rise up to 50% in hospitalized patients, 90% in women in the last month of pregnancy, and nearly 100% mortality in fetuses [3]. Neurological sequelae including deafness are common features in LF survivors [4,5].

Arenaviruses are enveloped viruses with a negative-sense RNA genome, consisting of two single-stranded segments named S (ca. 3.4 kb) and L (ca. 7.2 kb), each encoding two proteins with an ambisense strategy for expression. The S segment encodes the nucleoprotein (NP) and the precursor of the envelope glycoprotein complex (GPC), while the L segment encodes the viral RNA-dependent RNA polymerase (L) and a matrix protein (Z) that is involved in virus assembly and budding [6].

The open reading frames, in opposite orientations, are separated by a noncoding intergenic region predicted to fold into strong stem-loop structures [7].

GPC is expressed as a single precursor polypeptide that is cleaved twice by cellular proteases to generate a stable signal peptide (SSP), a receptor-binding subunit (GP1), and a trans-membrane fusion subunit (GP2). Both the peripheral GP1 and the SSP remain noncovalently associated with GP2, and assemble into the trimeric glycoprotein (GP) complex that mediates receptor recognition and fusion of the viral and host cell membranes [8–10].

NP is the most abundant viral protein both in virions and infected cells, and plays critical roles during arenavirus life cycle. NP associates tightly with the viral genomic and antigenomic RNAs forming ribonucleoprotein (RNP) complexes called nucleocapsids. Nucleocapsids bind the L polymerase, constituting the biologically active units for transcription of subgenomic viral mRNAs and for viral genome replication [11–13]. In addition, NP interacts with the Z matrix protein and contributes to the packaging of RNPs into viral particles during virion morphogenesis [14–16]. Crystallographic studies revealed that LASV NP is organized in two distinct domains [17]. The N-terminal domain contains a basic crevice, initially proposed to be an m7GTP cap binding site and later reported to function in binding RNA [17,18]. The C-terminal domain of NP harbors a functional 3′-5′ exoribonuclease activity of the DExD/H-box protein family that has been shown to oppose the host type I interferon (IFN-I)-mediated immune response during viral infection. In this regard, NP is capable of degrading small viral double-stranded RNA fragments that could function as pathogen-associated molecular patterns, to prevent their recognition by cellular pattern recognition receptors (PRRs) [17,19–21]. In addition, the role of NP in the negative regulation of IFN-I production has been linked to its ability to prevent the nuclear translocation and transcriptional activity of the nuclear factor kappa B (NF-kB), and its direct association with the retinoic acid-inducible gene I (RIG-I) and I-kappa-B kinase epsilon (IKKε), thereby inhibiting the activation and nuclear translocation of the interferon regulatory factor 3 (IRF-3) [22–24].

Following arenavirus entry, nucleocapsids are delivered into the cytoplasm of the host cell where transcription and replication of viral RNA segments occur. The arenavirus Z protein directs the assembly and budding of infectious particles from the plasma membrane, co-opting proteins from the endosomal sorting complexes required for transport (ESCRT) that facilitate virus egress [25].

Over the past few years, a significant amount of novel information has accrued regarding cellular proteins that play a role in the arenavirus life cycle, including in pathogenesis, immune evasion and virus entry and egress [25–28]. Here, we summarize current knowledge on host factors that are involved in LASV entry and discuss factors crucial for LASV RNA replication. Deepening the knowledge about relevant virus-host interactions essential for sustaining these early critical steps may help identify possible targets for the rational design of novel therapeutic approaches against LASV and other arenaviruses that cause severe human disease.

## 2. Virus–Host Interactions Involved in LASV Entry

### 2.1. α-Dystroglycan (α-DG) Is the Principal Receptor for LASV Entry

Arenaviruses primarily attach to cells by binding of their surface GP to specific receptor/entry factors at the plasma membrane of host cells. α-DG was the first entry receptor discovered for LASV, as well as for other OW and for Clade C NW arenaviruses [29,30], and its interaction with GP has been widely characterized [31]. DG is expressed as a precursor and proteolytically cleaved to generate the mature α and β subunits that together serve as a molecular bridge between the extracellular matrix and the cytoplasm [32–35]. α-DG is found in the extracellular compartment, where it binds components such as laminin, and requires O-glycosylation to perform its biological functions [36,37], whereas the transmembrane β subunit (β-DG) docks to the cytoskeleton by associating to the cytoplasmic adaptor proteins dystrophin and utrophin [35,38,39]. DG is expressed in most cell types, but its expression patterns and glycosylation levels differ depending on the tissue [34,35].

Further reports on arenavirus biology provide evidence that the α-DG receptor requires a specific type of glycosylation for efficient virus attachment; in particular, O-mannosylation, rarely found in mammals [40, 41]. LASV tightly binds to the “matriglycan” platform displayed on α-DG, a polymer composed of 3-xylose-α1, 3-glucuronic acid-β1 (Xylα1-3GlcAβ1-3) disaccharide repeats, which is linked to α-DG through phosphorylated O-mannose. Like-acetylglucosaminyltransferase (LARGE) is required for the attachment of ligand-binding moieties to phosphorylated O-mannose on α-DG [42]. Moreover, the recent determination of the crystal structure of the mature LASV and LCMV GPs in prefusion conformation demonstrates that the GP ectodomain engages matriglycan via multiple contacts, with a close similarity to the molecular mechanisms driving α-DG recognition of host extracellular matrix (ECM) proteins [43, 44]. Therefore, it is reasonable to conceive that LASV would mimic the behavior of different host ECMs that normally interact with glycosylated membrane receptors to gain access to the cell. Indeed, post-translational modification of α-DG by the glycosyltransferase LARGE is required for both efficient LASV infection and laminin binding [45–47]. Mutagenesis and functional studies further identified two threonine (Thr) residues (Thr317 and Thr319) within a highly conserved amino acid motif in α-DG that play a key role in LARGE-mediated α-DG modification, and are required for recognition by LASV GP and laminin [48]. These findings further support the idea that arenaviruses displaying high affinity for α-DG may be able to compete with host ligands and displace them from the receptor to infiltrate the cell [45, 48]. Thereafter, upon receptor recognition, the encounter of cellular α-DG with LASV GP induces tyrosine phosphorylation of β-DG’s cytosolic domain as well as it triggers β-DG’s dissociation from the cytoskeletal adaptor utrophin [49]. Then, it is envisioned that this later step of detachment of virus-bound DG from the actin-based cytoskeleton may ease subsequent endocytosis of the virus–receptor complex.

### 2.2. LASV Can Use Phosphatidylserine Receptors to Enter the Cell

The existence of alternative viral receptors was initially suggested by the observations that certain cell-types deficient in functional α-DG, i.e., hepatocytes, could be highly susceptible to LASV infection and that mice lacking the LARGE gene sustain LASV replication at a level comparable to that in wild-type mice [35, 50]. cDNA library screening studies from Kawaoka’s group singled out the Tyro3/Axl/Mer (TAM) receptor tyrosine kinases Axl and Tyro3/Dtk as potential LASV receptor candidates [51]. TAM family members are tyrosine-kinases which expose tandem immunoglobulin-related domains to the extracellular matrix. These domains interact with host protein S (ProS) and growth arrest-specific gene 6 (Gas6), which are serum proteins that bind the negatively charged phospholipid phosphatidylserine (PtdSer). PtdSer is translocated from the inner leaflet to the external leaflet of the plasma membrane in apoptotic cells, where it acts as a signal for professional phagocytes (macrophages and dendritic cells) as well as non-professional phagocytes (e.g., epithelial cells) [52, 53]. TAM receptors have been shown to play a role in virus entry of several RNA viruses such as Ebola (EBOV), dengue (DENV) and Zika, through a mechanism termed “apoptotic mimicry” [54–56]. This mechanism involves recognition of PtdSer exposed on the viral surface, incorporated from the cellular lipid bilayer during the budding process, as a signal for virus uptake [57–59]. Of note, there is evidence that cells infected with the NW arenavirus Pichinde display PtdSer on their plasma membranes; therefore, it is reasonable to conceive that both NW and OW arenaviruses could benefit from PtdSer receptors for viral entry [60]. In the case of LASV, initial studies using a HIV-based lentiviral vector pseudotyped with LASV GP empirically confirmed the capability of Axl to facilitate viral entry to cells lacking optimal carbohydrate modification of α-DG or to DG knockout cells [51]. Experiments from the Choe’s group applying alternative lentiviral pseudotyped platforms showed no enhancement of LASV (or LCMV) entry upon overexpression of Axl in human embryonic kidney (HEK-293T) cells [61] and hypothesized that LASV internalization via α-DG may be preferred over PtdSer receptors. Later, experiments from the Kunz’s group using a recombinant LCMV system expressing LASV GP (rLCMV-LASVGP) ultimately confirmed that the endogenous expression of Axl

does not actually enhance viral entry in the presence of fully functional α-DG receptor but it strongly augments viral infection in the absence of α-DG [62].

In line with these findings, T-cell immunoglobulin mucin I (TIM-1) has also been recently identified as a PtdSer receptor for LASV entry [63]. TIM receptors are cell surface glycoproteins that display an extracellular immunoglobulin variable-like domain (IgV), bearing a structural pocket with high affinity for PtdSer [64]. Unlike TAM receptors, TIM directly binds PtdSer, without the need for the Gas6 or ProS adaptors. It was demonstrated that TIM-I also mediates entry of vesicular stomatitis virus pseudovirions bearing LASV GP, either in the absence of α-DG or under conditions where it is inadequately glycosylated [63]. This behavior resembles that of Axl, suggesting a similarity between the entry route promoted by TAM and TIM receptors. In this sense, although functional α-DG would be the first LASV receptor of choice, the use of PtdSer receptors could function as a non-canonical GP-independent mechanism exploited by the virus to expand the spectrum of cellular tropism.

### 2.3. DC-SIGN and LSECtin Lectin Receptors Can Mediate LASV Cell Entry

Two additional receptor candidates were identified in the cDNA screening studies [51]: dendritic cell-specific intercellular adhesion molecule -3 grabbing nonintegrin (DC-SIGN) and liver and lymph node sinusoidal endothelial calcium-dependent lectin (LSECtin), both of which belong to the calcium-dependent (C-type) family of lectins. Of note, there is previous evidence indicating that DC-SIGN can facilitate infection of the NW JUNV and other enveloped viruses, including EBOV and Rift Valley fever virus [65–67]. Likewise, LSECtin has been implicated in entry of EBOV, SARs coronavirus and Japanese encephalitis virus [68–70]. As observed for TAM receptors, both DC-SIGN and LSECtin enhanced the susceptibility of cells to infection by LASV GP-pseudotyped lentivirus and participate in LASV entry independently of α-DG [51]. These lectin receptors were more effective at enhancing virus infection than TAM receptors, but importantly, none of these alternative receptors showed higher efficiency than properly modified α-DG, the principal portal of entry for LASV [51] (Figure 1). It was also shown that DC-SIGN or LSECtin binding to LASV GP is carbohydrate-specific, a fact that is characteristic of this C-type lectins family [51]. Strikingly, experiments using monocyte-derived immature human dendritic cells (MDDCs), which lack expression of Axl and Tyro3, demonstrated that upregulated expression of DC-SIGN correlates with enhanced virus attachment and productive infection, and that highly mannosylated glycans exposed on LASV GP1 surface interact with DC-SIGN during attachment [71]. Thus, DC-SIGN and LSECtin may facilitate LASV entry into dendritic cells, which represent the preferred early targets for arenavirus infection [72,73].

### 2.4. LASV Entry Involves Macropinocytosis and Intracellular LAMP1 Receptor for Virus Fusion

Upon receptor binding, OW arenaviruses including LASV, enter the host cell by a clathrin-independent endocytic process followed by transport to late endosomal compartments, where pH-dependent fusion of viral and cell membrane takes place [74,75]. Strikingly, sodium hydrogen exchangers (NHEs) have been identified through a genome-wide small interfering RNA screen, as host factors involved in the multiplication of LCMV in human cells [76]. Based on pharmacological and genetic analysis, Iwasaki et al. further validated NHE as entry factors for LCMV and LASV, implicating macropinocytosis in arenavirus entry [77]. Moreover, using the pseudotyped rLCMV-LASVGP and a panel of specific inhibitors for cellular factors involved in the regulation of macropinocytosis, Oppliger et al. showed that DG-mediated LASV entry depends on regulatory factors, including NHE, associated with this pathway [78].

α-Dystroglycan O-mannosylation

---

**optimal**

![Diagram of optimal α-dystroglycan O-mannosylation](#)

**low to absent**

![Diagram of low to absent α-dystroglycan O-mannosylation](#)

---

Receptor-mediated endocytosis

---

Figure 1. Model of cell receptor/s recognition by Lassa virus (LASV). Left panel. The α-dystroglycan (α-DG) receptor needs to be O-mannosylated for efficient virus attachment. In the presence of a fully functional α-DG receptor, LASV enters host cells after binding to the matriglycan platform displayed on α-DG. Right panel. In the absence of α-DG or in conditions where it is inadequately glycosylated, phosphatidylserine (PtdSer)-binding receptors (TAM; TIM) and C-type lectin receptors (DC-SIGN; LSECtin) can mediate α-DG-independent entry. TAM kinases bind Gas6 or ProS serum proteins, which bind to PtdSer molecules exposed on the viral envelope membrane. TIM directly binds PtdSer, without a need for the Gas6 or ProS adaptors. C-type lectins interact with glycans on the LASV glycoprotein (GP). PM: plasma membrane; TAM: Tyro3/Axl/Mer; TIM: T-cell immunoglobulin mucin; DC-SIGN: dendritic cell-specific intercellular adhesion molecule-3 nonintegrin; LSECtin: liver and lymph node sinusoidal endothelial calcium-dependent lectin. Cartoon diagram not to scale.

An unbiased haploid genetic screening in α-DG-deficient cells pinpointed the lysosome-associated membrane protein 1 (LAMP1) as a late endosomal co-receptor specifically required for efficient LASV entry [79]. LAMP1 is mostly found in lysosomes, but it also locates in other endosomal structures. It is hypothesized that the acidic pH of the late endosome destabilizes the high affinity interaction between LASV GP and α-DG, resulting in a “receptor switch” to LAMP1. In this sense, LAMP1 would work as a secondary intracellular receptor that helps induce the GP conformational changes needed for virus fusion. This interpretation is supported by a series of biochemical studies that demonstrated that LAMP1 directly interacts with LASV GP in a pre-fusion configuration, and which described that the strength of this interaction is modulated by pH conditions, where a drop in pH can destabilize LASV GP affinity for α-DG, thereby inducing potent binding to LAMP1 [79]. Furthermore, structural and functional studies showed that LASV GP1 conformation is stable at low pH conditions, at which it displays a triad of histidine residues that are involved in LAMP1 binding [80,81]. In this regard, LAMP1 facilitates LASV exit from earlier endosomal compartments, avoiding prolonged exposure to

a harsh proteolytic environment, increasing the overall efficiency of LASV entry and infection [82]. Besides LAMP1, the haploid genetics screening also pointed out α-2,3-sialyltransferase ST3GAL4, as well as additional factors involved in N-glycosylation and sialylation, as being important for LASV entry. Mutations in ST3GAL4, yielding a specific deficiency in sialylation of LAMP1, totally abrogate its ability to interact with LASV GP [79]. This highlights the strict requirement of a specific glycosylated version of LAMP1 for the biochemical interaction with GP to take place. In sum, it is envisioned that LASV would initially attach to the cell surface via α-DG, a step which would lead to delivery of virions to endosomes. Later, as virus-containing vesicles acidify, LASV would dissociate from the α-DG receptor, gaining affinity for LAMP1, and therefore completing the internalization process in a LASV-unique manner that is distinguishable from a standard endocytic process [83,84].

Altogether, these findings describing virus–host interactions involved in LASV entry disclose the diverse spectrum of mechanisms implemented by the virus to efficiently fulfill its internalization and fusion. On the one hand, the ability to utilize entry strategies alternative to α-DG receptor (such as PtdSer or lectin receptors), provides the virion with versatility to enlarge its cell tropism and promotes access to selected cell-targets such as dendritic cells, which are high privileged sites for early LASV productive infection. On the other side, LASV engagement of the late endosomal receptor LAMP1 likely guarantees the optimal spatial conditions required for virus fusion in close proximity to the endosome membrane. Given that not only α-DG- but also Axl-mediated entry involve LAMP1 co-factor [62], it is conceivable that multiple pathways converge at similar late endosomal compartments to efficiently accomplish LASV entry.

## 3. Role of Virus–Host Interactions Involving the LASV Replication Complex

### 3.1. Role of DEAD-Box RNA Helicase 3 (DDX3) in Viral Replication

A series of large-scale proteomic studies applying mass-spectrometry have been undertaken to comprehensively identify novel human protein candidates that could interact with the arenavirus proteins [85–89]. Special attention has been paid to NP-binding partners due to the multifunctional role of NP in the viral cycle, which involves crucial interactions with L and Z viral proteins [12,13,15–17,90], and its ability to hijack host factors to inhibit the antiviral innate immune response [19,22,24,91]. It is believed that identifying essential NP–host cell protein interactions can pave the way in the rational design of novel strategies to tackle arenavirus infections. One of the LASV NP interactors recently identified in human cells is DDX3, a protein belonging to the DEAD (Asp-Glu-Ala-Asp) box RNA helicase family, which harbors ATPase and RNA helicase activities [89]. Of note, DDX3 has also emerged in proteomic studies of virus-infected cells, as a novel interacting partner of the OW LCMV and the NW JUNV NPs [85]. CRISPR/Cas9-mediated deletion of DDX3 gene has been shown to lead to a significant reduction in virus yields of LASV, LCMV or JUNV in cell culture. Subsequently, lentiviral-mediated reconstitution of DDX3 expression resulted in a notable recovery in the infection rate of the three viruses, indicating a relevant role of DDX3 in virus growth as a proviral cellular factor [89].

DDX3 is known to be involved in multiple steps of RNA metabolism, including RNA transcription and the initiation of translation in host cells [92–94]. As other DEAD-box RNA helicases, such as DDX1 and DDX5, DDX3 appears to facilitate replication of different RNA viruses, as the alphavirus Venezuelan equine encephalitis virus and the hepatitis C virus (HCV), among others [95–100]. DDX3 is also required for translation of mRNAs containing a long or structured 5′ untranslated region (UTR), such as human immunodeficiency virus type-1 (HIV-1) genomic RNA (gRNA). Indeed, it was reported that DDX3 interacts with the 5′ region of the target mRNA, binds the eukaryotic translation initiation factor 4G (eIF4G) and poly A-binding protein cytoplasmic 1 (PABP), and interacts with HIV-1 Tat protein to facilitate translation of HIV-1 mRNAs [96,101]. In reference to arenaviruses, it was demonstrated that translation of a synthetic arenavirus mRNA analog was unaffected in DDX3-deficient cells, indicating no critical engagement of DDX3 in viral mRNA translation

initiation [89]. In contrast, minireplicon assay-based experiments demonstrated that the pro-arenaviral activity of DDX3 strongly depends on DDX3's ability to promote viral RNA synthesis, involving both previously described DDX3 ATPase and helicase RNA-unwinding activities in this function [89,102].

Strikingly, alternative roles have been ascribed to DDX3 in the context of different viral infections [103,104]. On the one hand, DDX3 is considered an antiviral factor given that it is involved in the innate immune response against some viruses such as HIV-1, DENV and HCV [105–107]. DDX3 has been shown to collaborate in the production of IFN-I, through interaction with components of the RIG-I-mediated IFN-I induction pathway [108–110]. However, in contrast to this IFN-I promoting capacity of DDX3, mechanistic analysis has provided evidence that, in the case of LCMV, DDX3 suppresses the IFN-I response at late times of infection, still it remains to be confirmed whether this IFN-I-suppressive role of DDX3 is sustained in the context of an infection with the pathogenic LASV [89]. Secondly, DDX3 is known to be an essential component for stress granule (SG) assembly, and to interact with other SG proteins, such as eIF4E [111]. Different proteomic approaches based on mass-spectrometry have singled out new arenavirus NP-binding candidates related to the SG biology; including but not limited to the Ras GTPase-activating protein-binding protein 1 (G3BP1), eIF2α, apoptosis-inducing factor mitochondrion-associated 1 (AIFM1) and PABP, yet none of them have been confirmed as LASV interactors by alternative biochemical methods [85,89]. Of note, colocalization experiments have revealed the association of the NW arenavirus TCRV replication-transcription complexes (RTCs), where NP accumulates, with G3BP1 and a non-canonical collection of ribosomal proteins, including the ribosomal proteins RPS6 and RPL10a, as well as translation initiation factors eIF4G and eIF4A [112]. In this regard, the finding that JUNV infection inhibits SG formation [113] might be related to the NP-mediated sequestration of DDX3 and other SG-related proteins, resulting in the lack of availability of essential factors needed for SG nucleation. Similarly, in the case of influenza virus infections, it has been hypothesized that the interaction of DDX3 with the viral NS1 protein prevents DDX3 binding to eIF4E and PABP1 as well as DDX3–NP interaction, thus suppressing SG formation, NP recruitment into SGs, and DDX3 antiviral activity [114]. Therefore, it is possible that in a similar way, LASV NP may counteract DDX3 antiviral function and in turn use DDX3 to enhance its own replication.

### 3.2. Other RNA Helicases Potentially Involved in LASV Replication

In addition to DDX3, a number of cellular proteins functionally related to RNA biosynthesis and ribonucleoprotein complex assembly, including the DEAD-box helicase 5 (DDX5, also referred to as RNA helicase p68) and RNA helicase A (namely DHX9), members of the DExD/Hbox protein family, have been identified as potential overlapping targets of the LCMV L protein and the NP of LASV, LCMV, and/or JUNV (Figure 2) [85,87,89]. Moreover, they have already been confirmed as interactors of the RNA-dependent RNA polymerase (RdRp) of other RNA viruses. For example, DDX5 has been shown to interact with the C-terminal region of HCV NS5B, and has been suggested to be part of the HCV replicase complex [115]. Similarly, DDX5 associates with the influenza A virus PB1 and PB2 proteins [116] and it is needed for an efficient activity of the viral polymerase [117]. Evidence has been provided that DHX9 interacts with the viral genomic RNA and non-structural protein 3 (nsP3) within active replication complexes in Chikungunya virus (CHIKV)-infected cells, displaying an inhibitory effect on viral RNA synthesis and an enhancing effect on viral genome translation, which may imply a regulatory role in CHIKV life cycle [118]. Porcine reproductive and respiratory syndrome virus (family *Arteriviridae*) nucleocapsid protein interacts with DHX9 polymerase to overcome premature termination of viral RNA synthesis [119]. Overall, DExD/H-box helicases emerge as host factors selectively hijacked by polymerases and nucleocapsid or nonstructural proteins from several viruses to facilitate their multiplication. Further work must be carried out to validate the binding of DDX5 and DHX9 helicases to LASV and/or other arenavirus proteins and provide a mechanistic model that could explain the relevance of these interactions.

Pathogens 2019, 8, 17

NP                                                                 L

DDX6<sup>85,88</sup>
HNRNPK<sup>85,88</sup>
HNRNPCL1<sup>85</sup>
HNRNPUL2<sup>85</sup>

DDX3X<sup>85,87,89</sup>
DDX5<sup>85,87-89</sup>
DDX1<sup>85,87,88</sup>
DHX9<sup>85,87</sup>
HNRNPC<sup>85,87-89</sup>
HNRNPF<sup>85,87,89</sup>
HNRNPU<sup>85,87,89</sup>
HNRNPA2B1<sup>85,87,88</sup>
HNRNPUL1<sup>85,87,88</sup>

Figure 2. Cellular DExD/H-box helicases and heterogeneous nuclear ribonucleoproteins (hnRNPs) identified among binding partners of the nucleoproteins (NPs) of LASV, lymphocytic choriomeningitis virus (LCMV), and/or Junín virus (JUNV) in different proteomics approaches [85,88,89]. Targets common to the LCMV L polymerase [107] are depicted. LASV NP binding partners are highlighted in green. References are indicated for each target.

Given the observation that many host factors converge as common binding-partners of RNA viruses, it is intriguing whether this is a consequence of the conserved conformational structure shared among proteins from different families of viruses. In particular, segmented negative strand viruses (sNSV) polymerases share key conserved motifs, specifically those corresponding to the fingers, palm and thumb subdomains within the RdRp domain located in the central part of the polypeptide chain. The central ring-like RdRp domain is linked to appendages that would be dedicated to 5′ mRNA capping activities [120–123]. Based on the amino acid sequence motif conservation displayed by sNSV polymerases and the structural similarity between the polymerases of the bunyavirus La Crosse encephalitis virus and influenza virus revealed by crystallographic studies, an overall structural configuration has been proposed for arenavirus L protein. It consists of a canonical RdRp core with N- and C-extensions that form a cavity connected to the exterior by four tunnels (NTP entry, template entry, template exit, and product exit) [124,125]. Structural information has led to a model for sNSV vRNA synthesis in which the L protein would operate in either transcription mode or replication mode, not only during initiation, but also along the whole RNA synthesis process [125,126]. A key question that needs to be addressed is how the polymerase switches from one mode to the other. For arenaviruses, there is genetic and biochemical evidence that L–L interaction is essential for polymerase activity, suggesting that the L polymerase may function in an oligomeric conformation [127]. Thus, as proposed for influenza virus [128,129], a conformational transition of the polymerase from a monomeric state during transcription to an oligomeric state during replication might be hypothesized. In this sense, it is probable that the association with cellular partners may additionally modulate the switch from the transcriptase to the replicase mode of the arenavirus polymerase. To date, DDX3 is the sole NP and L protein interactor for LASV or any other arenavirus that has been demonstrated to contribute in viral RNA synthesis, although the precise underlying mechanism remains to be fully understood. However, it is tempting to speculate that the binding of DDX3 (and eventually other cellular RNA helicases) with NP and L in a replicase complex, may contribute to the unwinding of viral RNA secondary structures such as RNA hairpins within the non-coding intergenic regions, which must be read-through during replication to accomplish full length genome and antigenome RNA

synthesis. Likewise, DDX3 and/or other host helicases may facilitate encapsidation of the nascent chain by NP during replication.

### 3.3. Host Heterogeneous Nuclear Ribonucleoproteins as Candidate Factors Required for the LASV Life Cycle

Other host factors that have recently emerged as potential candidate partners of LASV as well as LCMV and JUNV NP and/or L polymerase are the components of the heterogeneous nuclear ribonucleoprotein (hnRNP) family, including hnRNPA2/B1, one of the most abundant hnRNPs belonging to the A/B type [85,88,89]. HnRNPs are RNA-binding proteins involved in processing pre-mRNAs as well as in mRNA translation, trafficking and stability. Notably, previous reports have already demonstrated the interaction between JUNV NP and hnRNP A1 and that depletion of hnRNP A1 and A2 caused a strong inhibition of virus yield, suggesting a key role of hnRNPs A/B in JUNV multiplication [130]. Additionally, hnRNP K, an LCMV and JUNV NP binding partner detected in the proteomic screening by King et al. [85], has also been reported as a necessary host factor required for JUNV multiplication [131]. Likewise, studies from different groups have similarly ascribed relevant functions to hnRNPs in other viral infections. For example, hnRNP A2/B1 has been proposed to work as a positive regulator in viral RNA synthesis of influenza A virus, hnRNP A2 has been shown to regulate the trafficking of HIV-1 genomic RNA, and hnRNP K has been shown to support vesicular stomatitis virus replication by regulating cell survival and cellular gene expression [132–134]. Altogether, although further research needs to be completed to unveil the importance of hnRNPs in the Arenaviridae family, it is intriguing whether any of these proteins would specifically associate with the LASV RNP complex, playing a critical function either in viral genome transcription and/or replication.

### 3.4. Z Protein Interactors

Arenavirus Z matrix protein has been proposed to drive a mechanism to ensure accurate packaging of all necessary virion components. Apart from its pivotal role in virus assembly and budding [135,136], Z also inhibits viral RNA synthesis by directly binding the L polymerase to trigger its catalytic inactivation, and this can still occur in the absence of host cellular factors [137,138]. Of note, it has been hypothesized that the Z-L complex not only guarantees downregulation of viral gene expression, but also serves as a platform for the functional polymerase to remain locked on the template and be properly packaged into the mature virion [138]. Interestingly, a recent proteomics approach based on JUNV Z has pinpointed a number of targets that were incorporated to Z virus-like particles (VLPs) and purified JUNV particles and which include common interactors of the L and NP proteins, such as DDX3, DDX5, DHX9, hnRNPA2/B1, and PABP [86]. Actually, this is not a totally unexpected observation. For instance, it has been demonstrated for HIV-1 that DHX9 protein stimulates transcription of HIV-1 RNA as well as it associates with the Gag protein to ensure it is adequately recruited into virus particles during the assembly process [139]. In line with this, it is predictable that the association of Z protein with key cellular factors required for viral RNA synthesis may concomitantly facilitate virus packaging and/or make these factors available to complement the activity of viral RNP upon cell entry.

### 4. Concluding Remarks

LASV is currently considered a top priority emerging pathogen causing severe hemorrhagic fever outbreaks [140]. At this moment, treatment is limited to the nonspecific antiviral ribavirin, shown to be partially effective in LASV infections [141], and the use of favipiravir is still under investigation [142,143]. Moreover, there is no FDA-approved vaccine against LASV or any other arenavirus to date. Therefore, there is an urgent need to develop novel approaches to combat and prevent the infection with these viruses.

In the last years, several reports have provided robust evidence that LASV can utilize alternative entry factors apart from the well-characterized α-DG [144]. Novel lectin and PtdSer receptors have

emerged as non-canonical LASV ports of entry, proposed to enhance the viral cell tropism and/or redirect infection to selected cell types.

Studies on virus-host interactions have also improved our understanding of the mechanisms driving LASV replication. A recent series of proteomic approaches oriented to the analysis of the interactome of arenavirus NP, L, and Z proteins in human cells singled out several host factors, such as DExD/H-box helicases and heterogeneous nuclear RNPs, which might be potentially involved in arenavirus RNA synthesis. Particularly, it was demonstrated that the DDX3 ATP-dependent RNA helicase is a LASV target and may be dually exploited to both suppress the host immunity and promote viral replication and/or transcription, since DDX3 ATPase and helicase activities are involved in promoting optimal levels of viral RNA synthesis. Notably, it has recently been reported that a chemical compound directed to an RNA binding site of DDX3 protein displayed broad-spectrum antiviral activity against HIV drug-resistant strains, HCV, DENV, and West Nile virus infection [145]. In this regard, it is intriguing whether the use of DDX3-blocking compounds could be tentatively applied as a novel weapon to battle LASV infections.

Funding: M.E.L., A.D.A. and N.L. are members of the Argentinean Council of Investigation (CONICET). This research received no external funding.

Acknowledgments: The authors would like to apologize to all those researchers whose excellent studies were not included in this review due to space limitations.

Conflicts of Interest: The authors declare no conflict of interest.

## References

1. Maes, P.; Alkhovsky, S.V.; Bao, Y.; Beer, M.; Birkhead, M.; Briese, T.; Buchmeier, M.J.; Calisher, C.H.; Charrel, R.N.; Choi, I.R.; et al. Taxonomy of the family Arenaviridae and the order Bunyavirales: Update 2018. *Arch. Virol.* **2018**, *163*, 2295–2310. [CrossRef]
2. Basler, C.F. Molecular pathogenesis of viral hemorrhagic fever. *Semin. Immunopathol.* **2017**, *39*, 551–561. [CrossRef] [PubMed]
3. Ogbu, O.; Ajuluchukwu, E.; Uneke, C.J. Lassa fever in West African sub-region: An overview. *J. Vector Borne Dis.* **2007**, *44*, 1–11. [PubMed]
4. Cashman, K.A.; Wilkinson, E.R.; Zeng, X.; Cardile, A.P.; Facemire, P.R.; Bell, T.M.; Bearss, J.J.; Shaia, C.I.; Schmaljohn, C.S. Immune-Mediated Systemic Vasculitis as the Proposed Cause of Sudden-Onset Sensorineural Hearing Loss following Lassa Virus Exposure in Cynomolgus Macaques. *mBio* **2018**, *9*, e01896-18. [CrossRef] [PubMed]
5. Mateer, E.J.; Huang, C.; Shehu, N.Y.; Paessler, S. Lassa fever-induced sensorineural hearing loss: A neglected public health and social burden. *PLoS Negl. Trop. Dis.* **2018**, *12*, e0006187. [CrossRef]
6. Buchmeier, M.J.; De la Torre, J.C.; Peters, C.J. *Arenaviridae: The Viruses and Their Replication*; Wolters Kluwer Health/Lippincott Williams & Wilkins: Philadelphia, PA, USA, 2007.
7. Kiening, M.; Weber, F.; Frishman, D. Conserved RNA structures in the intergenic regions of ambisense viruses. *Sci. Rep.* **2017**, *7*, 16625. [CrossRef]
8. Eschli, B.; Quirin, K.; Wepf, A.; Weber, J.; Zinkernagel, R.; Hengartner, H. Identification of an N-terminal trimeric coiled-coil core within arenavirus glycoprotein 2 permits assignment to class I viral fusion proteins. *J. Virol.* **2006**, *80*, 5897–5907. [CrossRef]
9. Burri, D.J.; da Palma, J.R.; Kunz, S.; Pasquato, A. Envelope glycoprotein of arenaviruses. *Viruses* **2012**, *4*, 2162–2181. [CrossRef]
10. Hastie, K.M.; Saphire, E.O. Lassa virus glycoprotein: Stopping a moving target. *Curr. Opin. Virol.* **2018**, *31*, 52–58. [CrossRef]
11. Hass, M.; Golnitz, U.; Muller, S.; Becker-Ziaja, B.; Gunther, S. Replicon system for Lassa virus. *J. Virol.* **2004**, *78*, 13793–13803. [CrossRef]
12. Lee, K.J.; Novella, I.S.; Teng, M.N.; Oldstone, M.B.; de La Torre, J.C. NP and L proteins of lymphocytic choriomeningitis virus (LCMV) are sufficient for efficient transcription and replication of LCMV genomic RNA analogs. *J. Virol.* **2000**, *74*, 3470–3477. [CrossRef]

13. Lopez, N.; Jacamo, R.; Franze-Fernandez, M.T. Transcription and RNA replication of tacaribe virus genome and antigenome analogs require N and L proteins: Z protein is an inhibitor of these processes. *J. Virol.* 2001, 75, 12241–12251. [CrossRef] [PubMed]

14. Casabona, J.C.; Levingston Macleod, J.M.; Loureiro, M.E.; Gomez, G.A.; Lopez, N. The RING domain and the L79 residue of Z protein are involved in both the rescue of nucleocapsids and the incorporation of glycoproteins into infectious chimeric arenavirus-like particles. *J. Virol.* 2009, 83, 7029–7039. [CrossRef]

15. Ortiz-Riano, E.; Cheng, B.Y.; de la Torre, J.C.; Martinez-Sobrido, L. The C-terminal region of lymphocytic choriomeningitis virus nucleoprotein contains distinct and segregable functional domains involved in NP-Z interaction and counteraction of the type I interferon response. *J. Virol.* 2011, 85, 13038–13048. [CrossRef] [PubMed]

16. Shtanko, O.; Imai, M.; Goto, H.; Lukashevich, I.S.; Neumann, G.; Watanabe, T.; Kawaoka, Y. A role for the C terminus of Mopeia virus nucleoprotein in its incorporation into Z protein-induced virus-like particles. *J. Virol.* 2010, 84, 5415–5422. [CrossRef] [PubMed]

17. Qi, X.; Lan, S.; Wang, W.; Schelde, L.M.; Dong, H.; Wallat, G.D.; Ly, H.; Liang, Y.; Dong, C. Cap binding and immune evasion revealed by Lassa nucleoprotein structure. *Nature* 2010, 468, 779–783. [CrossRef] [PubMed]

18. Hastie, K.M.; Liu, T.; Li, S.; King, L.B.; Ngo, N.; Zandonatti, M.A.; Woods, V.L.; de la Torre, J.C., Jr.; Saphire, E.O. Crystal structure of the Lassa virus nucleoprotein-RNA complex reveals a gating mechanism for RNA binding. *Proc. Natl. Acad. Sci. USA* 2011, 108, 19365–19370. [CrossRef] [PubMed]

19. Carnec, X.; Baize, S.; Reynard, S.; Diancourt, L.; Caro, V.; Tordo, N.; Bouloy, M. Lassa virus nucleoprotein mutants generated by reverse genetics induce a robust type I interferon response in human dendritic cells and macrophages. *J. Virol.* 2011, 85, 12093–12097. [CrossRef]

20. Reynard, S.; Russier, M.; Fizet, A.; Carnec, X.; Baize, S. Exonuclease domain of the Lassa virus nucleoprotein is critical to avoid RIG-I signaling and to inhibit the innate immune response. *J. Virol.* 2014, 88, 13923–13927. [CrossRef]

21. Huang, Q.; Shao, J.; Lan, S.; Zhou, Y.; Xing, J.; Dong, C.; Liang, Y.; Ly, H. In vitro and in vivo characterizations of the Pichinde viral NP exoribonuclease function. *J. Virol.* 2015, 89, 6595–6607. [CrossRef]

22. Pythoud, C.; Rodrigo, W.W.; Pasqual, G.; Rothenberger, S.; Martinez-Sobrido, L.; de la Torre, J.C.; Kunz, S. Arenavirus nucleoproteins target interferon regulatory factor-activating kinase IKKepsilon. *J. Virol.* 2012, 86, 7728–7738. [CrossRef] [PubMed]

23. Rodrigo, W.W.; Ortiz-Riano, E.; Pythoud, C.; Kunz, S.; de la Torre, J.C.; Martinez-Sobrido, L. Arenavirus nucleoproteins prevent activation of nuclear factor kappa B. *J. Virol.* 2012, 86, 8185–8197. [CrossRef] [PubMed]

24. Zhou, S.; Cerny, A.M.; Zacharia, A.; Fitzgerald, K.A.; Kurt-Jones, E.A.; Finberg, R.W. Induction and inhibition of type I interferon responses by distinct components of lymphocytic choriomeningitis virus. *J. Virol.* 2010, 84, 9452–9462. [CrossRef] [PubMed]

25. Urata, S.; Yasuda, J. Molecular mechanism of arenavirus assembly and budding. *Viruses* 2012, 4, 2049–2079. [CrossRef]

26. Shao, J.; Liang, Y.; Ly, H. Human hemorrhagic Fever causing arenaviruses: Molecular mechanisms contributing to virus virulence and disease pathogenesis. *Pathogens* 2015, 4, 283–306. [CrossRef] [PubMed]

27. Hayes, M.; Salvato, M. Arenavirus evasion of host anti-viral responses. *Viruses* 2012, 4, 2182–2196. [CrossRef]

28. Fedeli, C.; Moreno, H.; Kunz, S. Novel Insights into Cell Entry of Emerging Human Pathogenic Arenaviruses. *J. Mol. Biol.* 2018, 430, 1839–1852. [CrossRef] [PubMed]

29. Cao, W.; Henry, M.D.; Borrow, P.; Yamada, H.; Elder, J.H.; Ravkov, E.V.; Nichol, S.T.; Compans, R.W.; Campbell, K.P.; Oldstone, M.B. Identification of alpha-dystroglycan as a receptor for lymphocytic choriomeningitis virus and Lassa fever virus. *Science* 1998, 282, 2079–2081. [CrossRef] [PubMed]

30. Spiropoulou, C.F.; Kunz, S.; Rollin, P.E.; Campbell, K.P.; Oldstone, M.B. New World arenavirus clade C, but not clade A and B viruses, utilizes alpha-dystroglycan as its major receptor. *J. Virol.* 2002, 76, 5140–51463. [CrossRef]

31. Acciani, M.; Alston, J.T.; Zhao, G.; Reynolds, H.; Ali, A.M.; Xu, B.; Brindley, M. Mutational analysis of Lassa virus glycoprotein highlights regions required for alpha-dystroglycan utilization. *J. Virol.* 2017, 91, e00574-17. [CrossRef]

32. Holt, K.H.; Crosbie, R.H.; Venzke, D.P.; Campbell, K.P. Biosynthesis of dystroglycan: Processing of a precursor propeptide. *FEBS Lett.* 2000, 468, 79–83. [CrossRef]

33. Ibragimov-Beskrovnaya, O.; Ervasti, J.M.; Leveille, C.J.; Slaughter, C.A.; Sernett, S.W.; Campbell, K.P. Primary structure of dystrophin-associated glycoproteins linking dystrophin to the extracellular matrix. *Nature* **1992**, *355*, 696–702. [CrossRef]

34. Ibragimov-Beskrovnaya, O.; Milatovich, A.; Ozcelik, T.; Yang, B.; Koepnick, K.; Francke, U.; Campbell, K.P. Human dystroglycan: Skeletal muscle cDNA, genomic structure, origin of tissue specific isoforms and chromosomal localization. *Hum. Mol. Genet.* **1993**, *2*, 1651–1657. [CrossRef]

35. Barresi, R.; Campbell, K.P. Dystroglycan: From biosynthesis to pathogenesis of human disease. *J. Cell Sci.* **2006**, *119*, 199–207. [CrossRef] [PubMed]

36. Ervasti, J.M.; Campbell, K.P. A role for the dystrophin-glycoprotein complex as a transmembrane linker between laminin and actin. *J. Cell Biol.* **1993**, *122*, 809–823. [CrossRef] [PubMed]

37. Michele, D.E.; Campbell, K.P. Dystrophin-glycoprotein complex: Post-translational processing and dystroglycan function. *J. Biol. Chem.* **2003**, *278*, 15457–15460. [CrossRef] [PubMed]

38. Ervasti, J.M.; Campbell, K.P. Membrane organization of the dystrophin-glycoprotein complex. *Cell* **1991**, *66*, 1121–1131. [CrossRef]

39. Chung, W.; Campanelli, J.T. WW and EF hand domains of dystrophin-family proteins mediate dystroglycan binding. *Mol. Cell Biol. Res. Commun.* **MCBRC 1999**, *2*, 162–171. [CrossRef]

40. Imperiali, M.; Thoma, C.; Pavoni, E.; Brancaccio, A.; Callewaert, N.; Oxenius, A. O Mannosylation of alpha-dystroglycan is essential for lymphocytic choriomeningitis virus receptor function. *J. Virol.* **2005**, *79*, 14297–14308. [CrossRef]

41. Kunz, S.; Rojek, J.M.; Kanagawa, M.; Spiropoulou, C.F.; Barresi, R.; Campbell, K.P.; Oldstone, M.B. Posttranslational modification of alpha-dystroglycan, the cellular receptor for arenaviruses, by the glycosyltransferase LARGE is critical for virus binding. *J. Virol.* **2005**, *79*, 14282–14296. [CrossRef]

42. Yoshida-Moriguchi, T.; Campbell, K.P. Matriglycan: A novel polysaccharide that links dystroglycan to the basement membrane. *Glycobiology* **2015**, *25*, 702–713. [CrossRef] [PubMed]

43. Hastie, K.M.; Igonet, S.; Sullivan, B.M.; Legrand, P.; Zandonatti, M.A.; Robinson, J.E.; Garry, R.F.; Rey, F.A.; Oldstone, M.B.; Saphire, E.O. Crystal structure of the prefusion surface glycoprotein of the prototypic arenavirus LCMV. *Nat. Struct. Mol. Biol.* **2016**, *23*, 513–521. [CrossRef]

44. Hastie, K.M.; Zandonatti, M.A.; Kleinfelter, L.M.; Heinrich, M.L.; Rowland, M.M.; Chandran, K.; Branco, L.M.; Robinson, J.E.; Garry, R.F.; Saphire, E.O. Structural basis for antibody-mediated neutralization of Lassa virus. *Science* **2017**, *356*, 923–928. [CrossRef] [PubMed]

45. Kunz, S.; Rojek, J.M.; Perez, M.; Spiropoulou, C.F.; Oldstone, M.B. Characterization of the interaction of lassa fever virus with its cellular receptor alpha-dystroglycan. *J. Virol.* **2005**, *79*, 5979–5987. [CrossRef] [PubMed]

46. Rojek, J.M.; Spiropoulou, C.F.; Campbell, K.P.; Kunz, S. Old World and clade C New World arenaviruses mimic the molecular mechanism of receptor recognition used by alpha-dystroglycan's host-derived ligands. *J. Virol.* **2007**, *81*, 5685–5695. [CrossRef] [PubMed]

47. Kanagawa, M.; Saito, F.; Kunz, S.; Yoshida-Moriguchi, T.; Barresi, R.; Kobayashi, Y.M.; Muschler, J.; Dumanski, J.P.; Michele, D.E.; Oldstone, M.B.; et al. Molecular recognition by LARGE is essential for expression of functional dystroglycan. *Cell* **2004**, *117*, 953–964. [CrossRef]

48. Hara, Y.; Kanagawa, M.; Kunz, S.; Yoshida-Moriguchi, T.; Satz, J.S.; Kobayashi, Y.M.; Zhu, Z.; Burden, S.J.; Oldstone, M.B.; Campbell, K.P. Like-acetylglucosaminyltransferase (LARGE)-dependent modification of dystroglycan at Thr-317/319 is required for laminin binding and arenavirus infection. *Proc. Natl. Acad. Sci. USA* **2011**, *108*, 17426–17431. [CrossRef]

49. Moraz, M.L.; Pythoud, C.; Turk, R.; Rothenberger, S.; Pasquato, A.; Campbell, K.P.; Kunz, S. Cell entry of Lassa virus induces tyrosine phosphorylation of dystroglycan. *Cell. Microbiol.* **2013**, *15*, 689–700. [CrossRef] [PubMed]

50. Imperiali, M.; Sporri, R.; Hewitt, J.; Oxenius, A. Post-translational modification of {alpha}-dystroglycan is not critical for lymphocytic choriomeningitis virus receptor function in vivo. *J. Gen. Virol.* **2008**, *89*, 2713–2722. [CrossRef]

51. Shimojima, M.; Stroher, U.; Ebihara, H.; Feldmann, H.; Kawaoka, Y. Identification of cell surface molecules involved in dystroglycan-independent Lassa virus cell entry. *J. Virol.* **2012**, *86*, 2067–2078. [CrossRef]

52. Lemke, G.; Rothlin, C.V. Immunobiology of the TAM receptors. *Nat. Rev. Immunol.* **2008**, *8*, 327–336. [CrossRef] [PubMed]

53. Lemke, G. Biology of the TAM receptors. *Cold Spring Harb. Perspect. Biol.* 2013, 5, a009076. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

54. Shimojima, M.; Takada, A.; Ebihara, H.; Neumann, G.; Fujioka, K.; Irimura, T.; Jones, S.; Feldmann, H.; Kawaoka, Y. Tyro3 family-mediated cell entry of Ebola and Marburg viruses. *J. Virol.* 2006, 80, 10109–10116. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

55. Meertens, L.; Carnec, X.; Lecoin, M.P.; Ramdasi, R.; Guivel-Benhamine, F.; Lew, E.; Lemke, G.; Schwartz, O.; Amara, A. The TIM and TAM families of phosphatidylserine receptors mediate dengue virus entry. *Cell Host Microbe* 2012, 12, 544–557. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

56. Meertens, L.; Labeau, A.; Dejarnac, O.; Cipriani, S.; Sinigaglia, L.; Bonnet-Madin, L.; Le Charpentier, T.; Hafirassou, M.L.; Zamborlini, A.; Cao-Lormeau, V.M.; et al. Axl Mediates ZIKA Virus Entry in Human Glial Cells and Modulates Innate Immune Responses. *Cell Rep.* 2017, 18, 324–333. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

57. Morizono, K.; Xie, Y.; Olafsen, T.; Lee, B.; Dasgupta, A.; Wu, A.M.; Chen, I.S. The soluble serum protein Gas6 bridges virion envelope phosphatidylserine to the TAM receptor tyrosine kinase Axl to mediate viral entry. *Cell Host Microbe* 2011, 9, 286–298. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

58. Mercer, J.; Helenius, A. Vaccinia virus uses macropinocytosis and apoptotic mimicry to enter host cells. *Science* 2008, 320, 531–535. [CrossRef](https://doi.org/CrossRef)

59. Amara, A.; Mercer, J. Viral apoptotic mimicry. *Nat. Rev. Microbiol.* 2015, 13, 461–469. [CrossRef](https://doi.org/CrossRef)

60. Soares, M.M.; King, S.W.; Thorpe, P.E. Targeting inside-out phosphatidylserine as a therapeutic strategy for viral diseases. *Nat. Med.* 2008, 14, 1357–1362. [CrossRef](https://doi.org/CrossRef)

61. Jemielity, S.; Wang, J.J.; Chan, Y.K.; Ahmed, A.A.; Li, W.; Monahan, S.; Bu, X.; Farzan, M.; Freeman, G.J.; Umetsu, D.T.; et al. TIM-family proteins promote infection of multiple enveloped viruses through virion-associated phosphatidylserine. *PLoS Pathog.* 2013, 9, e1003232. [CrossRef](https://doi.org/CrossRef)

62. Fedeli, C.; Torriani, G.; Galan-Navarro, C.; Moraz, M.L.; Moreno, H.; Gerold, G.; Kunz, S. Axl Can Serve as Entry Factor for Lassa Virus Depending on the Functional Glycosylation of Dystroglycan. *J. Virol.* 2018, 92, e01613-17. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

63. Brouillette, R.B.; Phillips, E.K.; Patel, R.; Mahauad-Fernandez, W.; Moller-Tank, S.; Rogers, K.J.; Dillard, J.A.; Cooney, A.L.; Martinez-Sobrido, L.; Okeoma, C.; et al. TIM-1 Mediates Dystroglycan-Independent Entry of Lassa Virus. *J. Virol.* 2018, 92, e00093-18. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

64. Freeman, G.J.; Casasnovas, J.M.; Umetsu, D.T.; DeKruyff, R.H. TIM genes: A family of cell surface phosphatidylserine receptors that regulate innate and adaptive immunity. *Immunol. Rev.* 2010, 235, 172–189. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

65. Martinez, M.G.; Bialecki, M.A.; Belouzard, S.; Cordo, S.M.; Candurra, N.A.; Whittaker, G.R. Utilization of human DC-SIGN and L-SIGN for entry and infection of host cells by the New World arenavirus, Junin virus. *Biochem. Biophys. Res. Commun.* 2013, 441, 612–617. [CrossRef](https://doi.org/CrossRef)

66. Leger, P.; Tetard, M.; Youness, B.; Cordes, N.; Rouxel, R.N.; Flamand, M.; Lozach, P.Y. Differential Use of the C-Type Lectins L-SIGN and DC-SIGN for Phlebovirus Endocytosis. *Traffic* 2016, 17, 639–656. [CrossRef](https://doi.org/CrossRef)

67. Alvarez, C.P.; Lasala, F.; Carrillo, J.; Muniz, O.; Corbi, A.L.; Delgado, R. C-type lectins DC-SIGN and L-SIGN mediate cellular entry by Ebola virus in cis and in trans. *J. Virol.* 2002, 76, 6841–6844. [CrossRef](https://doi.org/CrossRef)

68. Powlesland, A.S.; Fisch, T.; Taylor, M.E.; Smith, D.F.; Tissot, B.; Dell, A.; Pohlmann, S.; Drickamer, K. A novel mechanism for LSECtin binding to Ebola virus surface glycoprotein through truncated glycans. *J. Biol. Chem.* 2008, 283, 593–602. [CrossRef](https://doi.org/CrossRef)

69. Gramberg, T.; Hofmann, H.; Moller, P.; Lalor, P.F.; Marzi, A.; Geier, M.; Krumbiegel, M.; Winkler, T.; Kirchhoff, F.; Adams, D.H.; et al. LSECtin interacts with filovirus glycoproteins and the spike protein of SARS coronavirus. *Virology* 2005, 340, 224–236. [CrossRef](https://doi.org/CrossRef)

70. Shimojima, M.; Takenouchi, A.; Shimoda, H.; Kimura, N.; Maeda, K. Distinct usage of three C-type lectins by Japanese encephalitis virus: DC-SIGN, DC-SIGNR, and LSECtin. *Arch. Virol.* 2014, 159, 2023–2031. [CrossRef](https://doi.org/CrossRef)

71. Goncalves, A.R.; Moraz, M.L.; Pasquato, A.; Helenius, A.; Lozach, P.Y.; Kunz, S. Role of DC-SIGN in Lassa virus entry into human dendritic cells. *J. Virol.* 2013, 87, 11504–11515. [CrossRef](https://doi.org/CrossRef)

72. Baize, S.; Kaplon, J.; Faure, C.; Pannetier, D.; Georges-Courbot, M.C.; Deubel, V. Lassa virus infection of human dendritic cells and macrophages is productive but fails to activate cells. *J. Immunol.* 2004, 172, 2861–2869. [CrossRef](https://doi.org/CrossRef)

Pathogens 2019, 8, 17

73. Macal, M.; Lewis, G.M.; Kunz, S.; Flavell, R.; Harker, J.A.; Zuniga, E.I. Plasmacytoid dendritic cells are productively infected and activated through TLR-7 early after arenavirus infection. *Cell Host Microbe* **2012**, *11*, 617–630. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

74. Quirin, K.; Eschli, B.; Scheu, I.; Poort, L.; Kartenbeck, J.; Helenius, A. Lymphocytic choriomeningitis virus uses a novel endocytic pathway for infectious entry via late endosomes. *Virology* **2008**, *378*, 21–33. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

75. Rojek, J.M.; Sanchez, A.B.; Nguyen, N.T.; de la Torre, J.C.; Kunz, S. Different mechanisms of cell entry by human-pathogenic Old World and New World arenaviruses. *J. Virol.* **2008**, *82*, 7677–7687. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

76. Panda, D.; Das, A.; Dinh, P.X.; Subramaniam, S.; Nayak, D.; Barrows, N.J.; Pearson, J.L.; Thompson, J.; Kelly, D.L.; Ladunga, I.; et al. RNAi screening reveals requirement for host cell secretory pathway in infection by diverse families of negative-strand RNA viruses. *Proc. Natl. Acad. Sci. USA* **2011**, *108*, 19036–19041. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

77. Iwasaki, M.; Ngo, N.; de la Torre, J.C. Sodium hydrogen exchangers contribute to arenavirus cell entry. *J. Virol.* **2014**, *88*, 643–654. [CrossRef](https://doi.org/CrossRef)

78. Oppliger, J.; Torriani, G.; Herrador, A.; Kunz, S. Lassa Virus Cell Entry via Dystroglycan Involves an Unusual Pathway of Macropinocytosis. *J. Virol.* **2016**, *90*, 6412–6429. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

79. Jae, L.T.; Raaben, M.; Herbert, A.S.; Kuehne, A.I.; Wirchnianski, A.S.; Soh, T.K.; Stubbs, S.H.; Janssen, H.; Damme, M.; Saftig, P.; et al. Virus entry. Lassa virus entry requires a trigger-induced receptor switch. *Science* **2014**, *344*, 1506–1510. [CrossRef](https://doi.org/CrossRef)

80. Cohen-Dvashi, H.; Cohen, N.; Israeli, H.; Diskin, R. Molecular mechanism for LAMP1 recognition by Lassa Virus. *J. Virol.* **2015**, *89*, 7584–7592. [CrossRef](https://doi.org/CrossRef)

81. Li, S.; Sun, Z.; Pryce, R.; Parsy, M.L.; Fehling, S.K.; Schlie, K.; Siebert, C.A.; Garten, W.; Bowden, T.A.; Strecker, T.; et al. Acidic pH-Induced conformations and LAMP1 binding of the Lassa Virus glycoprotein spike. *PLoS Pathog.* **2016**, *12*, e1005418. [CrossRef](https://doi.org/CrossRef)

82. Hulseberg, C.E.; Fénéant, L.; Szymańska, K.M.; White, J.M. LAMP1 increases the efficiency of Lassa Virus infection by promoting fusion in less acidic endosomal compartments. *MBio* **2018**, *2*, e01818-17. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

83. Cohen-Dvashi, H.; Israeli, H.; Shani, O.; Katz, A.; Diskin, R. Role of LAMP1 Binding and pH Sensing by the Spike Complex of Lassa Virus. *J. Virol.* **2016**, *90*, 10329–10338. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

84. Israeli, H.; Cohen-Dvashi, H.; Shulman, A.; Shimon, A.; Diskin, R. Mapping of the Lassa virus LAMP1 binding site reveals unique determinants not shared by other old world arenaviruses. *PLoS Pathog.* **2017**, *13*, e1006337. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

85. King, B.R.; Hershkowitz, D.; Eisenhauer, P.L.; Weir, M.E.; Ziegler, C.M.; Russo, J.; Bruce, E.A.; Ballif, B.A.; Botten, J. A Map of the Arenavirus Nucleoprotein-Host Protein Interactome Reveals that Junin Virus Selectively Impairs the Antiviral Activity of Double-Stranded RNA-Activated Protein Kinase (PKR). *J. Virol.* **2017**, *91*, e00763-17. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

86. Ziegler, C.M.; Eisenhauer, P.; Kelly, J.A.; Dang, L.N.; Beganovic, V.; Bruce, E.A.; King, B.R.; Shirley, D.J.; Weir, M.E.; Ballif, B.A.; et al. A proteomic survey of Junin virus interactions with human proteins reveals host factors required for arenavirus replication. *J. Virol.* **2018**, *92*, e01565-17. [CrossRef](https://doi.org/CrossRef)

87. Khamina, K.; Lercher, A.; Caldera, M.; Schliehe, C.; Vilagos, B.; Sahin, M.; Kosack, L.; Bhattacharya, A.; Majek, P.; Stukalov, A.; et al. Characterization of host proteins interacting with the lymphocytic choriomeningitis virus L protein. *PLoS Pathog.* **2017**, *13*, e1006758. [CrossRef](https://doi.org/CrossRef)

88. Iwasaki, M.; Minder, P.; Cai, Y.; Kuhn, J.H.; Yates, J.R., 3rd; Torbett, B.E.; de la Torre, J.C. Interactome analysis of the lymphocytic choriomeningitis virus nucleoprotein in infected cells reveals ATPase Na+/K+ transporting subunit Alpha 1 and prohibitin as host-cell factors involved in the life cycle of mammarenaviruses. *PLoS Pathog.* **2018**, *14*, e1006892. [CrossRef](https://doi.org/CrossRef)

89. Loureiro, M.E.; Zorzetto-Fernandes, A.L.; Radoshitzky, S.; Chi, X.; Dallari, S.; Marooki, N.; Leger, P.; Foscaldi, S.; Harjono, V.; Sharma, S.; et al. DDX3 suppresses type I interferons and favors viral replication during Arenavirus infection. *PLoS Pathog.* **2018**, *14*, e1007125. [CrossRef](https://doi.org/CrossRef)

90. Loureiro, M.E.; D’Antuono, A.; Levingston Macleod, J.M.; Lopez, N. Uncovering viral protein-protein interactions and their role in arenavirus life cycle. *Viruses* **2012**, *4*, 1651–1667. [CrossRef](https://doi.org/CrossRef)

91. Martinez-Sobrido, L.; Zuniga, E.I.; Rosario, D.; Garcia-Sastre, A.; de la Torre, J.C. Inhibition of the type I interferon response by the nucleoprotein of the prototypic arenavirus lymphocytic choriomeningitis virus. *J. Virol.* 2006, *80*, 9192–9199. [CrossRef]

92. Chang, P.C.; Chi, C.W.; Chau, G.Y.; Li, F.Y.; Tsai, Y.H.; Wu, J.C.; Wu Lee, Y.H. DDX3, a DEAD box RNA helicase, is deregulated in hepatitis virus-associated hepatocellular carcinoma and is involved in cell growth control. *Oncogene* 2006, *25*, 1991–2003. [CrossRef] [PubMed]

93. Lai, M.C.; Chang, W.C.; Shieh, S.Y.; Tarn, W.Y. DDX3 regulates cell growth through translational control of cyclin E1. *Mol. Cell. Biol.* 2010, *30*, 5444–5453. [CrossRef]

94. Cruciat, C.M.; Dolde, C.; de Groot, R.E.; Ohkawara, B.; Reinhard, C.; Korswagen, H.C.; Niehrs, C. RNA helicase DDX3 is a regulatory subunit of casein kinase 1 in Wnt-beta-catenin signaling. *Science* 2013, *339*, 1436–1441. [CrossRef] [PubMed]

95. Ariumi, Y.; Kuroki, M.; Abe, K.; Dansako, H.; Ikeda, M.; Wakita, T.; Kato, N. DDX3 DEAD-box RNA helicase is required for hepatitis C virus RNA replication. *J. Virol.* 2007, *81*, 13922–13926. [CrossRef]

96. Lai, M.C.; Wang, S.W.; Cheng, L.; Tarn, W.Y.; Tsai, S.J.; Sun, H.S. Human DDX3 interacts with the HIV-1 Tat protein to facilitate viral mRNA translation. *PLoS ONE* 2013, *8*, e68665. [CrossRef]

97. Yasuda-Inoue, M.; Kuroki, M.; Ariumi, Y. Distinct DDX DEAD-box RNA helicases cooperate to modulate the HIV-1 Rev function. *Biochem. Biophys. Res. Commun.* 2013, *434*, 803–808. [CrossRef]

98. Chahar, H.S.; Chen, S.; Manjunath, N. P-body components LSM1, GW182, DDX3, DDX6 and XRN1 are recruited to WNV replication sites and positively regulate viral replication. *Virology* 2013, *436*, 1–7. [CrossRef]

99. Amaya, M.; Brooks-Faulconer, T.; Lark, T.; Keck, F.; Bailey, C.; Raman, V.; Narayanan, A. Venezuelan equine encephalitis virus non-structural protein 3 (nsP3) interacts with RNA helicases DDX1 and DDX3 in infected cells. *Antivir. Res.* 2016, *131*, 49–60. [CrossRef] [PubMed]

100. Shih, J.W.; Tsai, T.Y.; Chao, C.H.; Wu Lee, Y.H. Candidate tumor suppressor DDX3 RNA helicase specifically represses cap-dependent translation by acting as an eIF4E inhibitory protein. *Oncogene* 2008, *27*, 700–714. [CrossRef]

101. Soto-Rifo, R.; Rubilar, P.S.; Limousin, T.; de Breyne, S.; Decimo, D.; Ohlmann, T. DEAD-box protein DDX3 associates with eIF4F to promote translation of selected mRNAs. *EMBO J.* 2012, *31*, 3745–3756. [CrossRef] [PubMed]

102. Garbelli, A.; Beermann, S.; Di Cicco, G.; Dietrich, U.; Maga, G. A motif unique to the human DEAD-box protein DDX3 is important for nucleic acid binding, ATP hydrolysis, RNA/DNA unwinding and HIV-1 replication. *PLoS ONE* 2011, *6*, e19810. [CrossRef]

103. Ariumi, Y. Multiple functions of DDX3 RNA helicase in gene regulation, tumorigenesis, and viral infection. *Front. Genet.* 2014, *5*, 423. [CrossRef] [PubMed]

104. Valiente-Echeverria, F.; Hermoso, M.A.; Soto-Rifo, R. RNA helicase DDX3: At the crossroad of viral replication and antiviral immunity. *Rev. Med. Virol.* 2015, *25*, 286–299. [CrossRef] [PubMed]

105. Gringhuis, S.I.; Hertoghs, N.; Kaptein, T.M.; Zijlstra-Willems, E.M.; Sarrami-Forooshani, R.; Sprokholt, J.K.; van Teijlingen, N.H.; Kootstra, N.A.; Booiman, T.; van Dort, K.A.; et al. HIV-1 blocks the signaling adaptor MAVS to evade antiviral host defense after sensing of abortive HIV-1 RNA by the host helicase DDX3. *Nat. Immunol.* 2017, *18*, 225–235. [CrossRef]

106. Li, G.; Feng, T.; Pan, W.; Shi, X.; Dai, J. DEAD-box RNA helicase DDX3X inhibits DENV replication via regulating type one interferon pathway. *Biochem. Biophys. Res. Commun.* 2015, *456*, 327–332. [CrossRef] [PubMed]

107. Oshiumi, H.; Ikeda, M.; Matsumoto, M.; Watanabe, A.; Takeuchi, O.; Akira, S.; Kato, N.; Shimotohno, K.; Seya, T. Hepatitis C virus core protein abrogates the DDX3 function that enhances IPS-1-mediated IFN-beta induction. *PLoS ONE* 2010, *5*, e14258. [CrossRef] [PubMed]

108. Schroder, M.; Baran, M.; Bowie, A.G. Viral targeting of DEAD box protein 3 reveals its role in TBK1/IKKepsilon-mediated IRF activation. *EMBO J.* 2008, *27*, 2147–2157. [CrossRef]

109. Soulat, D.; Burckstummer, T.; Westermayer, S.; Goncalves, A.; Bauch, A.; Stefanovic, A.; Hantschel, O.; Bennett, K.L.; Decker, T.; Superti-Furga, G. The DEAD-box helicase DDX3X is a critical component of the TANK-binding kinase 1-dependent innate immune response. *EMBO J.* 2008, *27*, 2135–2146. [CrossRef] [PubMed]

110. Gu, L.; Fullam, A.; Brennan, R.; Schroder, M. Human DEAD box helicase 3 couples IkappaB kinase epsilon to interferon regulatory factor 3 activation. *Mol. Cell. Biol.* 2013, *33*, 2004–2015. [CrossRef]

111. Shih, J.W.; Wang, W.T.; Tsai, T.Y.; Kuo, C.Y.; Li, H.K.; Wu Lee, Y.H. Critical roles of RNA helicase DDX3 and its interactions with eIF4E/PABP1 in stress granule assembly and stress response. *Biochem. J.* 2012, *441*, 119–129. [CrossRef]

112. Baird, N.L.; York, J.; Nunberg, J.H. Arenavirus infection induces discrete cytosolic structures for RNA replication. *J. Virol.* 2012, *86*, 11301–11310. [CrossRef]

113. Linero, F.N.; Thomas, M.G.; Boccaccio, G.L.; Scolaro, L.A. Junin virus infection impairs stress-granule formation in Vero cells treated with arsenite via inhibition of eIF2alpha phosphorylation. *J. Gen. Virol.* 2011, *92*, 2889–2899. [CrossRef] [PubMed]

114. Thulasi Raman, S.N.; Liu, G.; Pyo, H.M.; Cui, Y.C.; Xu, F.; Ayalew, L.E.; Tikoo, S.K.; Zhou, Y. DDX3 Interacts with Influenza A Virus NS1 and NP Proteins and Exerts Antiviral Function through Regulation of Stress Granule Formation. *J. Virol.* 2016, *90*, 3661–3675. [CrossRef] [PubMed]

115. Goh, P.Y.; Tan, Y.J.; Lim, S.P.; Tan, Y.H.; Lim, S.G.; Fuller-Pace, F.; Hong, W. Cellular RNA helicase p68 relocalization and interaction with the hepatitis C virus (HCV) NS5B protein and the potential role of p68 in HCV RNA replication. *J. Virol.* 2004, *78*, 5288–5298. [CrossRef] [PubMed]

116. Jorba, N.; Juarez, S.; Torreira, E.; Gastaminza, P.; Zamarreno, N.; Albar, J.P.; Ortin, J. Analysis of the interaction of influenza virus polymerase complex with human cell factors. *Proteomics* 2008, *8*, 2077–2088. [CrossRef]

117. Bortz, E.; Westera, L.; Maamary, J.; Steel, J.; Albrecht, R.A.; Manicassamy, B.; Chase, G.; Martinez-Sobrido, L.; Schwemmle, M.; Garcia-Sastre, A. Host- and strain-specific regulation of influenza virus polymerase activity by interacting cellular proteins. *mBio* 2011, *16*, e00151-11. [CrossRef] [PubMed]

118. Matkovic, R.; Bernard, E.; Fontanel, S.; Eldin, P.; Chazal, N.; Hassan Hersi, D.; Merits, A.; Peloponese, J.M., Jr.; Briant, L. The host DHX9 DExH Box helicase is recruited to Chikungunya virus replication complexes for optimal genomic RNA translation. *J. Virol.* 2018, JVI.01764. [CrossRef]

119. Liu, L.; Tian, J.; Nan, H.; Tian, M.; Li, Y.; Xu, X.; Huang, B.; Zhou, E.; Hiscox, J.A.; Chen, H. Porcine Reproductive and Respiratory Syndrome Virus Nucleocapsid Protein Interacts with Nsp9 and Cellular DHX9 To Regulate Viral RNA Synthesis. *J. Virol.* 2016, *90*, 5384–5398. [CrossRef]

120. Wilda, M.; Lopez, N.; Casabona, J.C.; Franze-Fernandez, M.T. Mapping of the tacaribe arenavirus Z-protein binding sites on the L protein identified both amino acids within the putative polymerase domain and a region at the N terminus of L that are critically involved in binding. *J. Virol.* 2008, *82*, 11454–11460. [CrossRef]

121. Kranzusch, P.J.; Schenk, A.D.; Rahmeh, A.A.; Radoshitzky, S.R.; Bavari, S.; Walz, T.; Whelan, S.P. Assembly of a functional Machupo virus polymerase complex. *Proc. Natl. Acad. Sci. USA* 2010, *107*, 20069–20074. [CrossRef]

122. Morin, B.; Coutard, B.; Lelke, M.; Ferron, F.; Kerber, R.; Jamal, S.; Frangeul, A.; Baronti, C.; Charrel, R.; de Lamballerie, X.; et al. The N-terminal domain of the arenavirus L protein is an RNA endonuclease essential in mRNA transcription. *PLoS Pathog.* 2010, *6*, e1001038. [CrossRef] [PubMed]

123. Brunotte, L.; Lelke, M.; Hass, M.; Kleinsteuber, K.; Becker-Ziaja, B.; Gunther, S. Domain structure of Lassa virus L protein. *J. Virol.* 2011, *85*, 324–333. [CrossRef] [PubMed]

124. Reguera, J.; Gerlach, P.; Cusack, S. Towards a structural understanding of RNA synthesis by negative strand RNA viral polymerases. *Curr. Opin. Struct. Biol.* 2016, *36*, 75–84. [CrossRef] [PubMed]

125. Ferron, F.; Weber, F.; de la Torre, J.C.; Reguera, J. Transcription and replication mechanisms of Bunyaviridae and Arenaviridae L proteins. *Virus Res.* 2017, *234*, 118–134. [CrossRef]

126. Gerlach, P.; Malet, H.; Cusack, S.; Reguera, J. Structural Insights into Bunyavirus Replication and Its Regulation by the vRNA Promoter. *Cell* 2015, *161*, 1267–1279. [CrossRef]

127. Sanchez, A.B.; de la Torre, J.C. Genetic and biochemical evidence for an oligomeric structure of the functional L polymerase of the prototypic arenavirus lymphocytic choriomeningitis virus. *J. Virol.* 2005, *79*, 7262–7268. [CrossRef]

128. Chang, S.; Sun, D.; Liang, H.; Wang, J.; Li, J.; Guo, L.; Wang, X.; Guan, C.; Boruah, B.M.; Yuan, L.; et al. Cryo-EM structure of influenza virus RNA polymerase complex at 4.3 Å resolution. *Mol. Cell* 2015, *57*, 925–935. [CrossRef]

129. Ortin, J.; Martin-Benito, J. The RNA synthesis machinery of negative-stranded RNA viruses. *Virology* 2015, *479–480*, 532–544. [CrossRef]

130. Maeto, C.A.; Knott, M.E.; Linero, F.N.; Ellenberg, P.C.; Scolaro, L.A.; Castilla, V. Differential effect of acute and persistent Junin virus infections on the nucleo-cytoplasmic trafficking and expression of heterogeneous nuclear ribonucleoproteins type A and B. *J. Gen. Virol.* 2011, *92*, 2181–2190. [CrossRef]

131. Brunetti, J.E.; Scolaro, L.A.; Castilla, V. The heterogeneous nuclear ribonucleoprotein K (hnRNP K) is a host factor required for dengue virus and Junin virus multiplication. *Virus Res.* 2015, *203*, 84–91. [CrossRef]

132. Chang, C.K.; Chen, C.J.; Wu, C.C.; Chen, S.W.; Shih, S.R.; Kuo, R.L. Cellular hnRNP A2/B1 interacts with the NP of influenza A virus and impacts viral replication. *PLoS ONE* 2017, *12*, e0188214. [CrossRef] [PubMed]

133. Levesque, K.; Halvorsen, M.; Abrahamyan, L.; Chatel-Chaix, L.; Poupon, V.; Gordon, H.; DesGroseillers, L.; Gatignol, A.; Mouland, A.J. Trafficking of HIV-1 RNA is mediated by heterogeneous nuclear ribonucleoprotein A2 expression and impacts on viral assembly. *Traffic* 2006, *7*, 1177–1193. [CrossRef] [PubMed]

134. Dinh, P.X.; Das, A.; Franco, R.; Pattnaik, A.K. Heterogeneous nuclear ribonucleoprotein K supports vesicular stomatitis virus replication by regulating cell survival and cellular gene expression. *J. Virol.* 2013, *87*, 10059–10069. [CrossRef] [PubMed]

135. Strecker, T.; Eichler, R.; Meulen, J.; Weissenhorn, W.; Dieter Klenk, H.; Garten, W.; Lenz, O. Lassa virus Z protein is a matrix protein and sufficient for the release of virus-like particles [corrected]. *J. Virol.* 2003, *77*, 10700–10705. [CrossRef] [PubMed]

136. Fehling, S.K.; Lennartz, F.; Strecker, T. Multifunctional nature of the arenavirus RING finger protein Z. *Viruses* 2012, *4*, 2973–3011. [CrossRef] [PubMed]

137. Loureiro, M.E.; Wilda, M.; Livingston Macleod, J.M.; D’Antuono, A.; Foscaldi, S.; Marino Buslje, C.; Lopez, N. Molecular determinants of arenavirus Z protein homo-oligomerization and L polymerase binding. *J. Virol.* 2011, *85*, 12304–12314. [CrossRef] [PubMed]

138. Kranzusch, P.J.; Whelan, S.P. Arenavirus Z protein controls viral RNA synthesis by locking a polymerase-promoter complex. *Proc. Natl. Acad. Sci. USA* 2011, *108*, 19743–19748. [CrossRef]

139. Roy, B.B.; Hu, J.; Guo, X.; Russell, R.S.; Guo, F.; Kleiman, L.; Liang, C. Association of RNA helicase a with human immunodeficiency virus type 1 particles. *J. Biol. Chem.* 2006, *281*, 12625–12635. [CrossRef]

140. WHO. List of Blueprint Priority Diseases. Available online: <https://www.who.int/blueprint/priority-diseases/en/> (accessed on 28 January 2019).

141. McCormick, J.B.; King, I.J.; Webb, P.A.; Scribner, C.L.; Craven, R.B.; Johnson, K.M.; Elliott, L.H.; Belmont-Williams, R. Lassa fever. Effective therapy with ribavirin. *N. Engl. J. Med.* 1986, *314*, 20–26. [CrossRef]

142. Safronetz, D.; Rosenke, K.; Westover, J.B.; Martellaro, C.; Okumura, A.; Furuta, Y.; Geisbert, J.; Saturday, G.; Komeno, T.; Geisbert, T.W.; et al. The broad-spectrum antiviral favipiravir protects guinea pigs from lethal Lassa virus infection post-disease onset. *Sci. Rep.* 2015, *12*, 14775. [CrossRef]

143. Raabe, V.N.; Kann, G.; Ribner, B.S.; Morales, A.; Varkey, J.B.; Mehta, A.K.; Lyon, G.M.; Vanairsdale, S.; Faber, K.; Becker, S.; et al. Favipiravir and Ribavirin Treatment of Epidemiologically Linked Cases of Lassa Fever. *Clin. Infect. Dis.* 2017, *1*, 855–859. [CrossRef] [PubMed]

144. Torriani, G.; Galan-Navarro, C.; Kunz, S. Lassa Virus Cell Entry Reveals New Aspects of Virus-Host Cell Interaction. *J. Virol.* 2017, *91*, 1–8. [CrossRef] [PubMed]

145. Brai, A.; Fazi, R.; Tintori, C.; Zamperini, C.; Bugli, F.; Sanguinetti, M.; Stigliano, E.; Este, J.; Badia, R.; Franco, S.; et al. Human DDX3 protein is a valuable target to develop broad spectrum antiviral agents. *Proc. Natl. Acad. Sci. USA* 2016, *113*, 5388–5393. [CrossRef] [PubMed]
