
FEMS Microbiology Reviews 54 (1988) 85–110
Published by Elsevier

FER 00086

# Microbial metabolism of mandelate: a microcosm of diversity

Charles A. Fewson

Department of Biochemistry, University of Glasgow, Glasgow G12 8QQ, U.K.

Received 5 May 1987  
Accepted 5 August 1987

Key words: Aromatic; Bacterium; Enzyme; Evolution; Fungus; Genetics

## 1. SUMMARY

This review highlights the diversity of prokaryotic and eukaryotic microorganisms that can metabolise mandelate and it describes how a wide range of compounds related to mandelate is formed in many environments. The chief aspects that are summarised include the various pathways whereby mandelate and its structural analogues are converted into catechol or protocatechuate, the properties of the enzymes that are involved in the pathways, and the regulation and genetics of the pathways. The review incorporates the idea that the study of peripheral metabolic pathways is particularly useful for illuminating evolutionary speculations and it concludes with a list of questions that need to be answered.

## 2. INTRODUCTION

Supniewski [1] discovered that the ‘Bacille pyocyanique’ converted mandelate into benzoate. However, it was the wide-ranging work of Stanier and his colleagues [2–8] which elucidated the mechanism of mandelate degradation in some pseudomonads and introduced the mandelate pathway as a worthy topic of biochemical, genetic and physiological investigation and so of evolutionary speculation. Subsequent work has established this pathway as a paradigm of microbial metabolism. Mandelate, which is by no stretch of the imagination a common substance, is degraded by at least as many different pathways as is glucose – perhaps by more. Some of these pathways may have had independent origins, presumably by the recruitment of suitable enzymes from other areas of metabolism. Even in those organisms in which the enzymes are very similar and probably had common origins, their regulation is often quite different. This is not surprising in view of the fact that the ability to metabolise mandelate, although not so widespread as to be unremarkable, is found in a varied selection of Gram-negative and Gram-positive bacteria as well as in filamentous fungi and yeast. Many of the enzymes are of interest for their own sakes: some have unusual and intriguing mechanisms, a few are membrane-bound, several involve the incorporation of molecular O₂, and some seem to exhibit crypticity.

## 3. MANDELATE AND RELATED COMPOUNDS IN NATURE

Mandelic acid (2-hydroxy-2-phenylacetic acid, α-hydroxybenzeneacetic acid, α-hydroxyphenyl

Correspondence to: C.A. Fewson, Department of Biochemistry, University of Glasgow, Glasgow G12 8QQ, U.K.

0168-6445/88/$09.10 © 1988 Federation of European Microbiological Societies

acetic acid, phenylglycollic acid) exists in two enantiomeric forms (Fig. 1). The pKₐ is 3.37 so under most physiological conditions the predominant form will be the carboxylate mandelate ion.

There are few reports of the occurrence of free mandelate in Nature. It has been found in tissues of plants such as wheat leaves [9] and grapes [10]. Many plants accumulate glycosides of mandelonitrile, e.g. prunasin and sambunigrin are β-glucosides of D- and L-mandelonitrile respectively, and amygdalin, which is formed in almonds and in peach and apricot pits, is the β-gentiobioside of D-mandelonitrile [11]. However, these compounds are formed in plants from L-phenylalanine via mandelonitrile rather than from mandelate itself [12]. Further, their breakdown by plant glucosidases gives mandelonitrile which in turn is hydrolysed by mandelonitrile lyase to give HCN and benzaldehyde [11,13,14] (Fig. 2a). The toxicity of these compounds results in the notorious infertility of soil under peach trees [13,15]. A variety of arthropods (Lepidoptera, Coleoptera, Chilopoda, Diplopoda) produce repellent mixtures of HCN and benzaldehyde from D-mandelonitrile or its glycosides such as prunasin which they are able to store in substantial amounts [16,17]. If any of these mandelonitriles find their way into soil or other environments then presumably they may give rise to free mandelate by hydrolytic reactions (Fig. 2b, c). Mandelate and phenylglyoxylate (benzoylformate) are continually added to natural environments from animal urine. Normal human urine concentrations of mandelate are 1–15 μM and in phenylketonuria as much as 600 μM has been reported [21]. Mandelate is formed by the breakdown of naturally-occurring compounds [22] and is also found after industrial or domestic exposure to styrene [e.g. 23–25] (Fig. 3), doping

![Figure 1](https://i.imgur.com/1234567.png)
*Fig. 1. Enantiomers of mandelic acid.*

![Figure 2](https://i.imgur.com/2345678.png)
*Fig. 2. Possible pathways for the enzymic breakdown of mandelonitrile by (a) mandelonitrile lyase (E.C. 4.1.2.10), (b) nitrilase (nitrile amidohydrolase, E.C. 3.5.5.1) or (c) successive hydrations by nitrile hydratase and an amidase [see 14,18–20].*

with the stimulant pemoline [21,26], administration of ethylbenzene or acetophenone [27] (Fig. 3), or ingestion of drugs such as N-(phenylethyl)-3,3-diphenylpropylamine or 2-(3-hydroxyphenylethylamino)pyridine [21]. Mandelic acid is also a component of various pharmaceutical formulations, e.g. mandelyltropeine, a mydriatic agent [28], 3,3,5-trimethylcyclohexanyl mandelate, a vasoactive substance [29], and methenamine mandelate, a urinary antiseptic [e.g. 30,31], and their administration presumably leads to the excretion of fairly large amounts of mandelate. A dilute solution of mandelic acid is used in some hospitals as a bladder irrigation fluid to prevent urinary tract infections associated with urethral catheterization [32]. Mandelate is also incorporated in some selective media used for the isolation of bacteria such as Salmonella spp. [33,34] and mandelate esters have been used as repellants for *Tribolium confusum* [35]. It is therefore possible that substantial amounts of mandelate are continually entering many environments even though there seem to be no reports of its concentration in soils, natural waters, etc.; this could mean that it has not been systematically looked for or that it is rapidly metabolised.

The facile isolation of microorganisms able to metabolise mandelate has been contrasted with the lack of evidence for significant amounts of

Fig. 3. Mammalian metabolism of styrene, ethylbenzene and acetophenone to form urinary mandelate and phenylglyoxylate [25,27].

mandelate in nature. This apparent paradox, coupled with the recognition that many of the mandelate enzymes tolerate ring-substitution and are active with hydroxylated and methoxylated substrates, has led to an assumption that hydroxy- and methoxy-substituted mandelates derived from lignin and related natural products are the principal substrates of the organisms in nature [36,37]. Unfortunately, there appears to be no good evidence for this. The phenylpropanoid monomeric units of lignin might be expected to yield C₆–C₃ or C₆–C₁ (as a result of β-oxidation) products after depolymerisation, not C₆–C₂ compounds such as mandelate. This turns out to be the case

because the commonest low-$M_{r}$ aromatic compounds in soil seem to be $C_{6}-C_{3}$ (e.g. $p$-coumarate and ferulate) and $C_{6}-C_{1}$ (e.g. protocatechuate and vanillate, vanillin and 4-hydroxybenzaldehyde) [38,39] and even these are present in the soil water at concentrations of only about 10 $\mu$M [40], although of course pool size gives no indication of flux. Nevertheless, whilst ring-substituted mandelates may not be formed from lignin, there is a great deal of evidence that they are commonly formed from other precursors. D(-)-4-Hydroxy-3-methoxymandelate (vanillylmandelate, vanil-

mandelate) and 3,4-dihydroxymandelate, together with related compounds such as 4-hydroxy-3-methoxyphenylglycol and 3,4-dihydroxyphenylglycol are metabolites of adrenaline (epinephrine) and noradrenaline (norepinephrine) and substantial amounts are continually being excreted into the environment in the urine and faeces of normal animals, and even greater amounts are excreted in certain pathological conditions (see Fig. 4 for structures of these compounds) [41–44]. Octopamine, an invertebrate neurotransmitter also found in mammalian brain, is metabolised to 4-hydroxy-

![noradrenaline](attachment://noradrenaline.png)
noradrenaline

![adrenaline](attachment://adrenaline.png)
adrenaline

![octopamine](attachment://octopamine.png)
octopamine

![D(-)-4-hydroxy-3-methoxy-mandelic acid](attachment://D(-)-4-hydroxy-3-methoxy-mandelic_acid.png)
D(-)-4-hydroxy-3-methoxy-mandelic acid

![D(-)-3,4-dihydroxymandelic acid](attachment://D(-)-3,4-dihydroxymandelic_acid.png)
D(-)-3,4-dihydroxymandelic acid

![D(-)-4-hydroxymandelic acid](attachment://D(-)-4-hydroxymandelic_acid.png)
D(-)-4-hydroxymandelic acid

![4-hydroxy-3-methoxy-phenylglycol](attachment://4-hydroxy-3-methoxy-phenylglycol.png)
4-hydroxy-3-methoxy-phenylglycol

![3,4-dihydroxyphenyl glycol](attachment://3,4-dihydroxyphenyl_glycol.png)
3,4-dihydroxyphenyl glycol

![chloramphenicol](attachment://chloramphenicol.png)
chloramphenicol

![4-nitromandelic acid](attachment://4-nitromandelic_acid.png)
4-nitromandelic acid

Fig. 4. Adrenaline, octopamine, chloramphenicol and structural analogues of mandelic acid.

D-phenylalanine
L-phenylalanine
L-tyrosine

![Chemical Structure](https://i.imgur.com/1234567.png)

Fig. 5. Tentative pathways for the formation of mandelate and 4-hydroxymandelate by the fungus *Aspergillus niger* (→), the phytoplankton *Isochrysis galbana* and *Navicula incerta* (→) and the red alga *Odonthalia floccosa* (→→) [based on 48–50].
mandelate which is excreted [45]. L-Phenylalanine, phenylacetate and L-tyrosine are converted into mandelate and 4-hydroxymandelate by a wide range of organisms including *Polyporus hispidus* [46], *Penicillium chrysogenum* [47], *Aspergillus niger* [48], phytoplankton [49] and red algae [50], although some of the enzymes thought to be involved in the postulated metabolic pathways (Fig. 5) have not been identified and in particular the mechanism whereby the α-carbon of phenylacetate (or 4-hydroxyphenylacetate) is hydroxylated to give mandelate (or 4-hydroxymandelate) is not known. Crowden [51] has suggested on the basis of isotope incorporation experiments that the fungus *Polyporus tumulosus* forms 4-hydroxymandelate, 2,5-dihydroxymandelate and 3,4-dihydroxymandelate from shikimate. Similarly, 3-carboxymandelate and 3-carboxy-4-hydroxymandelate are formed in various plants [52]. Lastly, Lingens and colleagues [53] have proposed that a pathway for the microbial degradation of chloramphenicol gives rise to 4-nitromandelate.

### 4. OCCURRENCE OF MICROORGANISMS THAT CAN UTILISE MANDELATE

Strzelczyk et al. [54] found that up to about a quarter of all the benthic microorganisms isolated from the sandy sediment and ‘dy’-type sediment of a eutrophic lake could grow on mandelate and the mandelate-utilisers included members of the

---

**Table 1**

Microorganisms that are able to utilise mandelate

| Species | Enantiomer used<sup>a</sup> | Reference |
| --- | --- | --- |
| **(i) Bacteria** |  |  |
| *Acinetobacter calcoaceticus* | 3/106: D or L; 1/106: L only; 5/106: phenylglyoxylate | [60] |
|  | 8/35: D or L; 1/35: D only | [61] |
| *Arthrobacter* spp. | 1/130: unspecified enantiomer; and | [62] |
|  | 3/150 phenylglyoxylate |  |
| *Arthrobacter-Corynebacterium*<sup>b</sup> |  | [54] |
| *Azotobacter beijerinckii*<sup>c</sup> | D and L | [58] |
| *Bacillus sphaericus*<sup>c</sup> | L only | [58] |
| *Bacillus* spp.<sup>b</sup> |  | [54] |
| *Nocardia* spp.<sup>b</sup> |  | [54] |
| *Pseudomonas aeruginosa* | All 29 strains tested used L but not D (1 strain did not grow on phenylglyoxylate) | [57] |
| *Pseudomonas caryophylli* | Some strains use L | [62a] |
| *Pseudomonas convexa*<sup>c</sup> (putida? [58]) | D and L | [63] |
| *Pseudomonas fluorescens* | 3/94: L only; 21/94: phenylglyoxylate | [57] |
| *Pseudomonas multivorans* (cepacia) | 4/17: L only; 15/17: phenylglyoxylate | [57] |
| *Pseudomonas putida* | 5/41: D only; 3/41: D or L; 16/41: phenylglyoxylate | [57] |
| **(ii) Fungi** |  |  |
| *Aspergillus flavus*<sup>c</sup> | D or L | [64] |
| *Aspergillus niger*<sup>d</sup> | D or L | [65] |
| *Byssochlamys fulva*<sup>c</sup> | D or L | [66] |
| *Neurospora crassa*<sup>c,d</sup> | D or L | [67] |
| *Rhodotorula graminis*<sup>c</sup> | D or L | [68] |
| Unidentified yeast | L | [58] |

---

<sup>a</sup> Incidence of positive strains out of the total number of strains tested.  
<sup>b</sup> Enantiomer not specified.  
<sup>c</sup> Only one strain recorded.  
<sup>d</sup> These strains oxidised mandelate after growth in its presence together with other carbon sources but there is no evidence as to whether they can grow on mandelate as sole source of carbon and energy.
Arthrobacter-Corynebacterium group, Nocardia spp., Bacillus spp. and Gram-negative rods. Laboratory-grown activated sludge and treatment plant activated sludge contained large numbers of mandelate-utilising organisms [55, 56]. A wide range of both prokaryotic and eukaryotic microorganisms can metabolise one or both enantiomers of mandelate (Table 1). Many of these observations were made with organisms that were already held in culture collections. However, standard enrichment procedures using mandelate as the sole source of carbon allow the isolation of mandelate-utilising organisms [e.g. 37, 57, 58]. Alternate transfer between media containing the two separate enantiomers gives organisms able to use both D- and L-mandelate and these usually have two stereospecific mandelate dehydrogenases although in a few cases (so far always *Pseudomonas putida* biotype A) there is a mandelate racemase [37]. Organisms with only a single mandelate dehydrogenase and no racemase can be isolated by replica-plating from agar containing one enantiomer onto medium containing the other enantiomer and looking for colonies that fail to grow on the replica plates [57].

There is no evidence about the extent to which mandelate may be co-metabolised or partially metabolised by individual species of microorganisms or about the possibility of it being degraded by mixed populations. Further, all the studies have concentrated on aerobic conditions and there appears to be no report of anaerobic mandelate metabolism although many other aromatic compounds can be degraded anaerobically [e.g. 59].

### 5. CHEMOTAXIS TOWARDS MANDELATE

Chemotaxis towards potential nutrients is presumably of selective advantage to motile microorganisms. D,L-Mandelate was found to be an effective chemoattractant for *Pseudomonas putida* in capillary assays of chemotaxis [69]. However, experiments with blocked mutants indicated that the bacteria were actually responding to phenylglyoxylate, the first product of mandelate metabolism in this organism. Harwood et al. suggested that the synthesis of the putative chemoreceptor for phenylglyoxylate could not be subject to exactly the same regulatory control as the mandelate enzymes themselves because the two processes had slightly different inducer specificities. They also concluded that the fact that cells respond to endogenously generated phenylglyoxylate suggests that the chemotactic response may be mediated by recognition at a receptor site within the cell; alternatively, the phenylglyoxylate may be excreted and then recognised by an external receptor [69].

### 6. UPTAKE OF MANDELATE

Mandelate is a mild antiseptic agent [e.g. 32], presumably by virtue of its ability to destroy the integrity of bacterial membranes, and it therefore supports microbial growth only at relatively low concentrations (approx. 5–20 mM is generally used in growth media). At first sight it would appear that mandelate should enter bacteria by free diffusion. Harwood and Gibson pointed out that ‘Weak acids and bases with low polar characteristics can cross biological membranes with relative ease, distributing themselves so that the population of uncharged molecules is equal on both sides, and the total concentration of charged and uncharged molecular species on each side of the membrane is a function of the pH gradient’ [70]. In experiments whose results were consistent with this notion, *Rhodopseudomonas palustris* did not appear to concentrate free benzoate against a concentration gradient; even so, the conversion of internal benzoate into benzoylCoA was so rapid that benzoate uptake was extremely effective (apparent \(K_m < 1\) μM) [70]. Nevertheless there is evidence that *Pseudomonas putida* possesses an inducible specific active transport system that can accumulate benzoate against a 150-fold concentration gradient, appears to be activated by the membrane potential rather than ATP hydrolysis, and has a high affinity for its substrate (\(K_m\) approx. 20 μM) [71, 72].

Hegeman found no evidence for the existence of a mandelate permease in *P. putida* and even in non-induced bacteria there was rapid equilibration of mandelate across the membrane [73]. In *Acin*

etobacter calcoaceticus, which appeared to have no permeability barrier against benzoate, there did appear to be a barrier against the entry of mandelate unless the bacteria had been grown under conditions where the mandelate-metabolising enzymes were induced [74]. Higgins and Mandelstam obtained several lines of evidence for the existence of an inducible, specific uptake system for mandelate in *P. putida*; uptake was sensitive to 2,4-dinitrophenol, occurred against a concentration gradient, but accumulated mandelate only if the extracellular concentration was low [75,76]. Interpretation of results of experiments on mandelate uptake may be confused by the fact that bacterial mandelate dehydrogenases are membrane bound (see Section 8.2) and so mandelate may need only to reach the periplasm in order to be dehydrogenated. Further, bacterial transport systems seem invariably to be stereospecific so this factor would have to be taken into account in testing for the existence and operation of uptake systems for D- and/or L-mandelate.

### 7. PATHWAYS FOR THE METABOLISM OF MANDELATE

#### 7.1. Pathways in prokaryotes and eukaryotes

The initial attack on mandelate can be by racemization, ring hydroxylation or stereospecific dehydrogenation (Fig. 6a, b′, b″, e).

**Pseudomonas putida** A.3.12 (PRS 1; ATCC 12633, NCIB 9494; previously referred to as *Pseudomonas fluorescens*; strain 90 of [57]) can grow on both D- and L-mandelate. D-Mandelate is converted into benzaldehyde by the successive actions of mandelate racemase, L-mandelate dehydrogenase and phenylglyoxylate decarboxylase (Fig. 6a, b′, c) and there are then two isofunctional benzaldehyde dehydrogenases, one specific for NAD and one for NADP (Fig. 6d), which form benzoate [2–8,73,77,78]. Other strains of *P. putida* carry out the same reactions [37]. In addition some strains (e.g. ATCC 17426; strain 49 of [57]) can grow on D-mandelate but not L-mandelate and in this case appear to possess a D-mandelate dehydrogenase (Fig. 6b″) but lack a mandelate racemase [37,58]. Finally, one strain has a mandelate racemase and a D-mandelate dehydrogenase and so again can grow on both enantiomers of mandelate [37]. Benzoate is converted into catechol by the action of benzoate oxygenase and 3,5-cyclohexadiene-1,2-diol-1-carboxylate dehydrogenase (Fig. 6i, j) [79–81]. Catechol is subject to ortho (intradiol) cleavage by catechol 1,2-dioxygenase and enters the β-ketoadipate pathway to produce succinate and acetyl-CoA which join the amphibolic pathways and allow growth [82–85].

**Pseudomonas aeruginosa** can grow on only L-mandelate, which is oxidised by an L-mandelate dehydrogenase and the rest of the pathway is the same as that in *P. putida* (Fig. 6) [86]. There is some evidence for the existence of two NADP-linked benzaldehyde dehydrogenases, although the results based on differential induction and mutational analysis were not entirely conclusive [86].

Some strains of *Acinetobacter calcoaceticus* (e.g. EBF 65/174) can grow on both enantiomers of mandelate and have two stereospecific mandelate dehydrogenases; some (e.g. NCIB 8250) have only an L-mandelate dehydrogenase and so can grow on L-mandelate but not D-mandelate; and some (e.g. EBF 65/65) have a D-mandelate dehydrogenase but no L-mandelate dehydrogenase [60,61, 87]. Those strains that have only a single dehydrogenase can give rise to mutants that have a second dehydrogenase, specific for the other enantiomer, and in every case the novel enzymes (which may arise by the expression of cryptic genes) are coordinately regulated along with the pre-existing enzymes [87–89]. Phenylglyoxylate is then metabolised exactly as in *P. putida* (Fig. 6) except that there is only a single benzaldehyde dehydrogenase [90–94].

‘**Pseudomonas convexa**’ (probably a strain of *P. putida* [58]) can grow on both enantiomers of mandelate but the catabolic pathway involved shows some variations on the general theme found in most other bacteria. In this case there is a racemase but this is followed by ring hydroxylation (Fig. 6e) rather than dehydrogenation; the product, 4-hydroxymandelate, is oxidatively decarboxylated directly to 4-hydroxybenzaldehyde and 4-hydroxyphenylglyoxylate is not an intermediate. The substrate for ring cleavage is proto-

Fig. 6. Metabolism of mandelate in Acinetobacter calcoaceticus (→). Pseudomonas putida (→), *P*. convexa (---), Aspergillus niger, Neurospora crassa and Rhodotorula graminis (→). Enzymes: a, mandelate racemase; b′, L(+)-mandelate dehydrogenase; b″, D(-)-mandelate dehydrogenase; c, phenylglyoxylate decarboxylase; d, benzaldehyde dehydrogenase (there are several isofunctional enzymes); e, L-mandelate 4-hydroxylase; f, L-4-hydroxymandelate oxidase; g, benzoate 4-hydroxylase; h, 4-hydroxybenzoate 3-hydroxylase; i, benzoate 1,2-oxygenase; j, 3,5-cyclohexadiene-1,2-diol-1-carboxylate dehydrogenase; i + j = 'benzoate oxidase'.
catechuate, rather than catechol as in most bacteria (Fig. 6) [63,95–97].

The filamentous fungi *Aspergillus niger* [65] and *Neurospora crassa* [67] and the yeast *Rhodotorula graminis* [68,98] can metabolise both enantiomers of mandelate. There are two stereospecific dehydrogenases to form phenylglyoxylate and this is then converted into benzoate. *A. niger*, like *P. putida*, has an NAD-linked as well as an NADP-linked benzaldehyde dehydrogenase [65]. The subsequent pathway differs from that found in *P. putida* or *Acinetobacter calcoaceticus* in that benzoate undergoes two successive ring hydroxylations to form protocatechuate as the substrate for ring cleavage (Fig. 6). A similar pathway seems to operate in *Aspergillus flavus* [64] and *Byssochlams fulva* [66,99].

The metabolism of the products of ring cleavage has been extensively dealt with elsewhere [e.g. 83,84,100,101] and is beyond the scope of the present review. The properties of the β-ketoadipate pathway enzymes, their regulation and possible evolutionary relationships, have been the subject of a fascinating series of papers by Stanier, Ornston and their colleagues [e.g. 100,102,103] and are of considerable significance to the more general question of evolution of metabolic pathways.

### 7.2. Degradation of a range of compounds via the mandelate pathway

A wide variety of organic substrates can be converted into amphibolic intermediates by being channelled into the mandelate pathway (Fig. 7) and this may be part of the reason why the mandelate pathway is so common in microorganisms.

### 7.3. Degradation of ring-substituted mandelates

Many of the mandelate enzymes can tolerate extensive ring-substitution, especially at the 3-, 4- and 5-positions. Several organisms can therefore metabolise, or partially metabolise, an extensive array of mandelate analogues (Fig. 8). *Pseudomonas putida* [115], *Acinetobacter calcoaceticus* [90,91] and *Aspergillus niger* [65,116] can convert one or both enantiomers of 4-hydroxymandelate into protocatechuate. *A. calcoaceticus* also forms

protocatechuate from L-3,4-dihydroxymandelate and L-4-hydroxy-3-methoxymandelate [90,91]. *P. putida* can oxidise 4-hydroxy-3-methoxymandelate [117–119] but the enantiomer that is metabolised depends on whether the strain possesses an L-mandelate dehydrogenase or a D-mandelate dehydrogenase because mandelate racemase is unusually specific amongst the mandelate enzymes and cannot use 4-hydroxy-3-methoxymandelate (or 3,4-dihydroxymandelate) as substrate [58,120]. This lack of substrate ambiguity invalidated claims to be able to use an enzyme preparation from *P. fluorescens* strain A.3.12 to measure normal urinary 4-hydroxy-3-methoxymandelate [121], which is the D-enantiomer [120], although use of an appropriate strain (*P. putida* MB-15, ATCC 17426) does give an acceptable procedure [119]. In practice urinary 4-hydroxy-3-methoxymandelate concentrations are usually measured by chemical procedures such as mass fragmentography [122], high-performance liquid chromatography [e.g. 123] or gas chromatography [e.g. 124].

The much more restricted substrate specificity of the ring-cleavage enzymes and of the enzymes of the β-ketoadipate pathway means that compounds can only be completely oxidised if they can be converted into an appropriate substrate for one of the oxygenative ring-cleavage enzymes, such as catechol or protocatechuate [e.g. 85,101]. The result is that many substrates of the mandelate enzymes, such as 3-hydroxymandelate [90,91] (Fig. 8) or halogenated mandelates [77,91], can be only partially metabolised; this sort of effect may nevertheless be very important in natural environments where the mixed populations of organisms may totally oxidise compounds which are beyond the metabolic scope of any individual organism.

Sze and Dagley [124a] recently isolated a strain of *Acinetobacter* that can grow on D,L-4-hydroxy-3-methoxymandelate or D,L-4-hydroxymandelate but cannot grow on either L- or D-mandelate. The organism was obtained by elective culture from a cattle yard that was presumably a rich source of urinary 4-hydroxy-3-methoxymandelate. Results of experiments with bacterial extracts indicated the presence of fairly unspecific L- and D-mandelate dehydrogenases, phenylglyoxylate decarboxylase and benzaldehyde dehydrogenase. The prod-
Fig. 7. Degradation of several precursors via the mandelate pathway. Examples of the various conversions are: a and b, *Pseudomonas putida* [104] and *Flavobacterium* sp. [105]; c, *Penicillium chrysogenum* [47], mechanism uncertain, see Section 3; d, *Pseudomonas putida* [77], *Lactobacillus* sp. [106], *Bacillus* sp. [107]; e, *Pseudomonas putida* [77,108]; f and g, *Pseudomonas putida* [77]; h, *Pseudomonas putida* [77]; i, *Acinetobacter calcoaceticus* [90–93,109,110], *Pseudomonas putida* [3,111]; j, see Fig. 2 and Section 3; k, *Pseudomonas putida* containing TOL plasmid [e.g. 112]; l, occurs in mammalian tissues (see Fig. 3) but not so far found in microorganisms [see e.g. 113,114].

ucts of 4-hydroxy-3-methoxymandelate and 4-hydroxymandelate metabolism were shown to be 4-hydroxy-3-methoxybenzoate and 4-hydroxybenzoate, respectively, and these were converted into 3-O-methylgallate and protocatechuate by two NADPH-dependent hydroxylases. There appeared to be no benzoate oxygenase activity, which would explain the failure of the organism to grow on mandelate. Surprisingly, 3-O-methylgallate and protocatechuate were shown to be subject to meta (extradiol) ring cleavage. This appears to be the first report of mandelate metabolism in which ortho fission of the benzene nucleus is not involved.

mandelic acid
2-hydroxy-
mandelic acid
3-hydroxy-
mandelic acid
4-hydroxy-
mandelic acid
3,4-dihydroxy-
mandelic acid
4-hydroxy-3-methoxy-
mandelic acid

catechol
ring cleavage and
further metabolism to
succinate + acetylCoA

protocatechuic
acid

Fig. 8. Metabolism of ring-substituted mandelates by *Acinetobacter calcoaceticus*. Based on [91].

8. PROPERTIES OF THE ENZYMES OF THE
MANDELATE PATHWAY

8.1. Mandelate racemase (Fig. 6a; E.C.5.1.2.2)
This is one of the best studied of all the
mandelate enzymes [58,120,125–132] and has been
thoroughly reviewed [37]. The enzyme, which is
found only in strains of *Pseudomonas putida* bio-
type A [37], is tetrameric, contains four identical
sub-units (*M*<sub>r</sub> 69 500), and has an absolute re-
quirement for a divalent metal ion (e.g. Mg<sup>2+</sup> or
Mn<sup>2+</sup>) for activity. Mandelate racemase has some

claims to catalyse one of the simplest of all en-
zymic reactions: a non-redox racemization in
which no organic cofactor is involved. Various
lines of evidence suggest that the reaction mecha-
nism involves a carbanion intermediate (Fig. 9a):
(a) there is a primary deuterium isotope effect of
about 5 on *V*<sub>max</sub>/*K*<sub>m</sub>, (b) mandelate analogues
with electron-withdrawing substituents at the *para*
position have greater *V*<sub>max</sub> values than mandelate
itself, and (c) there is a deuterium or tritium
exchange between the solvent and the hydrogen
on the *α*-carbon [37]. Initial velocity measure-

the terminal acceptor. Both enzymes from *A.* calcoaceticus have been purified to homogeneity and have been partially characterised [136, 137]. This organism also possesses a pair of NAD(P)-independent, membrane-bound lactate dehydrogenases and these have also been purified and characterised [136, 138]. Comparison of the properties of the four dehydrogenases (Table 2) has led to the following conclusions: (a) The two enzymes specific for the D-enantiomers of lactate or of mandelate strongly resemble each other, as do the two enzymes specific for the L-enantiomers (ease of extraction by detergents, subunit $M_r$, pH optima, pI values, susceptibility to thiol reagents, and presence of FMN or FAD as cofactors) and so each of these pairs of enzymes may have had a common evolutionary origin. If so, the ability of D-mandelate dehydrogenase to dehydrogenate D-lactate (Table 2) may be an evolutionary relic of the recruitment of a lactate dehydrogenase (perhaps following gene duplication) to oxidise mandelate; or it may simply be a consequence of the similarity in chemical structures. (b) The two pairs of enzymes, although at first sight similar to each other in an overall way, are sufficiently different (especially flavin specificity and subunit $M_r$) that a common origin for all four enzymes is either unlikely or was much more distant. Only sequencing studies will provide the data to test these ideas.

The properties of mandelate dehydrogenases from eukaryotic microorganisms are much less well understood, although it would be extremely interesting to be able to compare them with the equivalent prokaryotic enzymes. Information derived from preliminary investigations with *Aspergillus niger* is confusing. In strain UBC 814 there was reported to be an L-mandelate dehydrogenase that was ‘soluble’, reduced 2,6-dichloroindophenol and was stimulated by FAD or FMN. This strain also contained a D-mandelate oxidase that was ‘particulate’ (although it is not clear what organelle was involved), unstable, showed no requirement for any of the cofactors tested, and could not reduce 2,6-dichloroindophenol [65, 139]. In a different strain of *A. niger*, isolated in Bangalore, there were D- and L-mandelate dehydrogenases, each associated with both the ‘soluble’ and ‘par-

Table 2  
Comparison of the lactate and mandelate dehydrogenases from *Acinetobacter calcoaceticus* $^{a}$  
All the enzymes are located in the inner membrane, are NAD(P)-independent and stereospecific.

| Solubilisation by non-ionic detergents (% activity released by Triton X-100 or Lubrol) | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|-----------------------------|--------------------------|--------------------------|--------------------------|--------------------------|
| 43–72                       | 55–76                    | 87–91                    | 83–98                    |

| Solubilisation by ionic detergents (% activity released by cholate or deoxycholate) | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|-----------------------------|--------------------------|--------------------------|--------------------------|--------------------------|
| 4–42                        | 13–32                    | 78–91                    | 63–87                    |

| Subunit $M_{r}$ | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|------------------|--------------------------|--------------------------|--------------------------|--------------------------|
| 44000            | 40000                    | 60000                    | 630000                   |

| pI value | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|----------|--------------------------|--------------------------|--------------------------|--------------------------|
| 4.2      | < 4.0                    | 5.5                      | 5.8                      |

| pH optimum | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|------------|--------------------------|--------------------------|--------------------------|--------------------------|
| 7.5        | 7.5                      | 8.0                      | 7.7                      |

| $K_{m}$ value ($\mu$M) for substrate | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|--------------------------------------|--------------------------|--------------------------|--------------------------|--------------------------|
| 186                                 | 83                       | 385 (in addition D-lactate is a substrate but $K_{m} = 4.8 \, \text{mM}$) | 308 |

| Assay conditions | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|-------------------|--------------------------|--------------------------|--------------------------|--------------------------|
| 2,6-Dichloroindophenol reduction measured in the presence of $N$-methylphenazonium methosulphate or $N$-methylphenazonium ethosulphate. All require bovine serum albumin (or Triton X-100 in the case of D-mandelate dehydrogenase and D-lactate dehydrogenase) in the reaction mixture for full activity. |

| Inhibitors | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|------------|--------------------------|--------------------------|--------------------------|--------------------------|
| None is affected by a wide range of chelating agents. All are moderately susceptible to several thiol-blocking reagents. All are inhibited by oxalate. |

| Inhibition by $p$-chloromercuribenzoate | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|----------------------------------------|--------------------------|--------------------------|--------------------------|--------------------------|
| Moderate                              | Moderate                 | Very severe             | Severe                   |

| Cofactor (non-covalently bound) | L-Mandelate dehydrogenase | L-Lactate dehydrogenase | D-Mandelate dehydrogenase | D-Lactate dehydrogenase |
|-------------------------------|--------------------------|--------------------------|--------------------------|--------------------------|
| FMN                           | FMN                      | FAD                      | FAD                      |

$^{a}$ Based on [137] with information from [135–138].
[143,146] although they do not cross-react immunologically [148]. The enzymes from both *Aspergillus niger* [65,149] and *Acinetobacter calcoaceticus* [150] appeared to be allosterically regulated by ATP and ADP respectively but there is no evidence that this has any physiological significance. The pH optimum for the phenylglyoxylate decarboxylase from all organisms tested is low (approx. 6) and as with the yeast pyruvate decarboxylase, dissociation of sub-units and loss of TPP occur at higher pH values [143,146,147].

### 8.4. Benzaldehyde dehydrogenases (E.C.1.2.1.7, 1.2.1.28)

There are pairs of isofunctional benzaldehyde dehydrogenase associated with the mandelate pathway in *Pseudomonas putida*, *Aspergillus niger* and perhaps *P. aeruginosa* (see Section 7.1) but in no case is it clear whether the separate enzymes have different functions. In addition, many organisms can grow on benzaldehyde or benzyl alcohol (Fig. 7) and in these cases additional benzaldehyde dehydrogenases are induced. Thus in both *Acinetobacter calcoaceticus* [36,90,92] and *P. putida* [151,152] there are extra benzaldehyde dehydrogenases which are under quite separate control from the mandelate enzymes and are induced only in the presence of benzaldehyde or benzyl alcohol. In view of the plethora of aromatic aldehyde dehydrogenases that are known to exist in microorganisms [153], it is surprising that so little is known about the benzaldehyde dehydrogenases associated with the mandelate pathway. The NADP-specific enzyme from *P. putida* has a native \(M_r\) of approx. 200,000, is activated by K⁺ ions, and is strongly inhibited by thiol reagents [154]. The NAD-specific benzaldehyde dehydrogenase I from *A. calcoaceticus* is a tetramer of identical sub-units (\(M_r\) 4 × 55,000) and is also activated by K⁺ ions [92,155].

### 8.5. L-Mandelate 4-hydroxylase (Fig. 6e; L-Mandelate 4-monooxygenase)

This soluble, inducible enzyme from *Pseudomonas convexa* requires tetrahydropteridine, NADPH, Fe²⁺ and O₂ for activity and is stereospecific for the L-enantiomer of its substrate. The native \(M_r\) is about 91,000 and the pH optimum is

at 5.4 [63,95,97]. Mandelate 4-hydroxylase has not yet been found in any other organisms but it would be worth looking for in fungi where hydroxylation generally seems to occur at an earlier stage in the mandelate pathway than it does in bacteria [e.g. 48,65–68].

### 8.6. L-4-Hydroxymandelate oxidase (Fig. 6f)

An inducible, membrane-bound L-4-hydroxymandelate oxidase was solubilised by butan-1-ol treatment of membranes from *Pseudomonas convexa* and partially purified and characterised [63,96]. The enzyme catalyses the one-step oxidative decarboxylation of its substrate to form 4-hydroxybenzaldehyde without involving the intermediate formation of 4-hydroxyphenylglyoxylate. FAD and Mn²⁺ ions are required for activity. The enzyme is very specific for its substrate and various analogues (e.g. L-mandelate, 3,4-dihydroxymandelate) are not substrates and some are inhibitors. Curiously, mushroom tyrosinase catalyses a similar oxidative decarboxylation of 3,4-dihydroxymandelate with the intermediate formation of a quinone methide intermediate [156] and so L-4-hydroxymandelate oxidase may involve a parallel mode of action (Fig. 10).

### 8.7. Benzoate 4-hydroxylase (Fig. 6g, Fig. 11a; benzoate 4-monooxygenase, E.C. 1.14.13.12)

The benzoate 4-hydroxylases from both *Aspergillus niger* [157] and a soil pseudomonad [158] are soluble, inducible enzymes which catalyse the conversion of benzoate into 4-hydroxybenzoate with equimolar consumption of NADPH and O₂. Tetrahydropteridine is used as cofactor [157,158].

![Chemical Reaction Diagram](https://i.imgur.com/chemical_reaction.png)

**Fig. 10. Possible mechanism of action of L-4-hydroxymandelate oxidase. Based on the proposed mechanism for a similar reaction catalysed by mushroom tyrosinase [156].**
The enzyme from *Rhodotorula graminis* is quite different for although it uses NADPH as electron donor, it is particulate, pteridine-independent and may involve a flavin as cofactor [159].

### 8.8. 4-Hydroxybenzoate 3-hydroxylase (Fig. 6h, Fig. 11b; 4-hydroxybenzoate 3-monooxygenase, E.C. 1.14.13.2)

Crystalline 4-hydroxybenzoate hydroxylase has been obtained from several *Pseudomonas* species (including *P. putida* strain A.3.12) and it has been extensively studied as an example of a flavin-dependent mixed function oxidase [reviewed in 160]. There is a stoichiometric requirement for NADPH and O₂. FAD is used as cofactor with enzyme-bound FADH₂ serving as the direct electron donor to O₂ in the oxygenation reaction [160].

---

### 8.9. Benzoate 1,2-oxygenase (Fig. 6i, Fig. 11c; benzoate hydroxylase, E.C. 1.13.99.2)

The benzoate oxygenase complex of *Pseudomonas arvilla* has two components: an NADH-cytochrome c reductase containing one FAD and one (2Fe-2S*) cluster per \( M_r \) of about 37500 and benzoate oxygenase which contains four (2Fe-2S*) clusters per \( M_r \) of about 275000 but no haem or flavin [161–163].

---

### 8.10. 3,5-Cyclohexadiene-1,2-diol-1-carboxylate dehydrogenase (Fig. 6j, Fig. 11d; dihydrodihydroxybenzoate dehydrogenase)

The NAD-dependent decarboxylation of the diol substrate to catechol is catalysed by a dehydrogenase that has been purified from *Alcaligenes eutrophus*. The enzyme is probably a tetramer of identical sub-units (\( M_r \) 4 × 24000) and the proposed mechanism involves dehydrogenation to an unstable β-keto acid which decarboxylates to form catechol [81].

---

## 9. REGULATION OF THE MANDELATE PATHWAY

### 9.1. Co-ordinate expression of the mandelate enzymes

Work on the mandelate pathway led to two concepts that were influential in the early stages of development of ideas about enzyme induction in bacteria [2–5, 164, 165]. The first was the principle of ‘simultaneous adaptation’: growth on a given substrate produces cells that can oxidise not only that compound but also its catabolites. The second was ‘sequential induction’: the idea that each enzyme is induced by its substrate and so there will be sequential induction in a catabolic pathway as successive inducers are formed by the action of each preceding enzyme. The notion of simultaneous adaptation still has some value as one line of attack in the problem of elucidating novel catabolic pathways. In general, however, the two concepts have been submerged by the realisation that (a) clusters of enzymes (operons), rather than single enzymes, are often subject to coordinate regulation, (b) enzyme products rather than substrates may be inducers, (c) gratuitous

---

**Fig. 11. Various oxidative hydroxylation reactions of the mandelate pathway:**  
a, benzoate 4-hydroxylase; b, 4-hydroxybenzoate 3-hydroxylase; c, benzoate 1,2-oxygenase; d, 3,5-cyclohexadiene-1,2-diol-1-carboxylate dehydrogenase.
induction is common, and (d) problems with permeation may give false negative results [3,4, 164–166].

The mandelate enzymes appear to be inducible in all the organisms, both prokaryotic and eukaryotic, that have been examined [e.g. 77,90, 91,95,98,99,116]. However, detailed measurements of the specificity and co-ordination of control have been made only with *Pseudomonas putida*, *P*. *aeruginosa* and *Acinetobacter calcoaceticus* using approaches such as (a) estimation of the degree of correlation between differential rates of synthesis of the enzymes under various conditions of induction and repression [93], (b) characterisation of blocked mutants [73,93,167], (c) induction by phenoxyacetate or thiophenoxyacetate [(phenylthio) acetate] (Fig. 12) which are gratuitous inducers in both *P.* *putida* and *A.* *calcoaceticus* [73,93], (d) co-ordinate anti-induction by 2-phenylpropionate (Fig. 12) in *A.* *calcoaceticus* [168], (e) isolation and characterisation of constitutive mutants [78,168], and (f) properties of superinducible mutants of *P.* *putida* isolated by enrichment in the presence of 2,3,4,5,6-pentafluoromandelate (Fig. 12), an inhibitor of mandelate dehydrogenase [169]. Various ring-substituted analogues can also serve as inducers [77,90,91,93]. The patterns of induction in the three species of bacteria examined are rather different from each other with respect to both specificity and degree of coordination (Fig. 13). More detailed work at the molecular level will be necessary before any worthwhile generalisations can be made but the mandelate enzymes clearly provide another example of the fact that very similar enzymes may be subject to quite different regulation in different microorganisms, suggesting that evolutionary constraints on catalytic functions and on regulation are almost certainly not the same and may well be varied, accidental and obscure [170].

The enzymes converting benzoate into catechol or into 4-hydroxybenzoate and then protocatechuate appear to be controlled independently of the enzymes of the upper part of the pathway [e.g. 77,90,98].

### 9.2. Repression of gene expression and its consequences for growth on mandelate

Mandelate enzymes are usually subject to some sort of catabolite repression, e.g. by succinate in *Pseudomonas putida* [165,171] and *Acinetobacter calcoaceticus* [93,172] or by glucose in *Byssochlamyms fulva* [99] and *Rhodotorula graminis* [98]. Growth on mixtures of substrates can therefore be diauxic [98,172]. Analysis of the sequential use of pairs of substrates by *P.* *putida* and of the extent of enzyme induction and repression that occurred in appropriate mutants under various growth conditions led to the suggestion that the mandelate pathway is subject to ‘multi-sensitive end-product repression’ in which products of successive coordinately induced groups of enzymes feedback repress the preceding groups of enzymes (Fig. 14) [151,171,173]. Feedback repression also occurs in the mandelate pathway of *A.* *calcoaceticus* [172]. It is not yet clear whether these feedback mechanisms are specific, or whether they are manifestations of fairly non-specific anti-induction of the

![Structures](https://i.imgur.com/1234567.png)

Fig. 12. Structures of some compounds that affect the induction of the mandelate enzymes. See text for details. In addition, an extremely extensive array of compounds has been tested for their effects as substrates, inhibitors, inducers and anti-inducers of the mandelate enzymes [77,93,168].
D-MANDELATE
↓
mandelate
racemase
↓
L-MANDELATE
↓
L-mandelate
dehydrogenase
↓
PHENYLGLYOXYLATE
↓
phenylglyoxylate
decarboxylase
↓
BENZALDEHYDE
↓
benzaldehyde
dehydrogenase(s)
↓
BENZOATE

D-mandelate,
L-mandelate
or
phenylglyoxylate
↓
phenylglyoxylate
↓
phenylglyoxylate
↓
phenylglyoxylate
or
β-ketoacidipate

Acinetobacter
calcoaceticus
Pseudomonas
putida
Pseudomonas
aeruginosa

Fig. 13. Specificity and co-ordination of induction of the mandelate enzymes in three species of bacteria. References are in the text.

L-MANDELATE
↓
L-mandelate
dehydrogenase
↓
PHENYLGLYOXYLATE
↓
phenylglyoxylate
decarboxylase
↓
BENZALDEHYDE
↓
benzaldehyde
dehydrogenase(s)
↓
BENZOATE
↓
'benzoate
oxidase'
↓
CATECHOL-
↓
catechol
1,2-oxygenase
↓
CIS,CIS -MUCONATE
↓
SUCCINATE
+
ACETYL-CoA

same type as that caused by 2-phenylpropionate [168]. Inducer-exclusion might also be involved [165,171] but this would depend on whether or not there are indeed specific transport mechanisms for mandelate and related compounds (see Section 6). The repressive effects are often quasi-competitive in nature and can be masked if one compound is in great excess [165,168,171].

In A. calcoaceticus mandelate metabolism dominates benzyl alcohol metabolism even though benzyl alcohol alone supports a faster growth rate than does mandelate [36,109,172]. Batch cultures of A. calcoaceticus growing on mandelate or phenylglyoxylate show an unusual non-exponential pattern. There are transient accumulations of benzaldehyde or benzyl alcohol in the growth medium caused by the limitation of mandelate oxidation by low activities of benzaldehyde dehydrogenase and the diversion of reducing power to the formation of benzyl alcohol [172]. P. putida usually grows exponentially on mandelate as sole

Fig. 14. Multi-sensitive end-product repression in the mandelate pathway of Pseudomonas putida. Based on [151,171].
carbon source [77,172] but this organism also accumulates benzyl alcohol under certain conditions [174] even though it has two benzaldehyde dehydrogenases.

### 9.3. Feedback inhibition does not appear to be important

There appear to have been no reports of specific feedback inhibition effects on any of the mandelate enzymes [see e.g. 172]. It may be that possession of peripheral catabolic pathways for utilizing less common substrates that are only occasionally present in the environment is so great a selective advantage that there has been no further selection of enzymes that are susceptible to feedback inhibition. Recruitment of enzymes to serve in peripheral pathways would possibly be prejudiced if the enzymes were already subject to feedback regulation of a kind that accidentally hindered rather than helped their new functions. Nevertheless one might have thought that there would be need to regulate flux through these pathways to ensure the balanced provision of amphibolic intermediates and to avoid the accumulation of intermediates which might have deleterious effects.

### 9.4. Do the mandelate enzymes exist as a multi-enzyme complex?

In vivo cross-linking studies with dimethylsuberimidate and \({}^{13}\) C NMR experiments have been used to test if there is any physical association amongst the mandelate enzymes in \(P\). putida [174]. Five of the mandelate enzyme activities could be detected in a cross-linked complex [174]. The in vivo cross-linking by dimethylimidates has been confirmed in \(P\). putida and \(A\). calcoaceticus [175] but it seems to be an open question as to whether there is a specific ‘mandelate complex’, whether the cross-linking is non-specific, or whether some sort of transient enzyme interactions might occur [see e.g. 176].

### 10. GENETICS OF THE MANDELATE PATHWAY

In *Pseudomonas putida* the genes encoding the first five enzymes of the mandelate pathway appear to be linked and may even be contiguous [169,177–180]. These genes show supra-operonic clustering with genes of functionally related pathways for the degradation of 4-hydroxybenzoate, quinate, shikimate, phenylacetate, nicotinate, histidine, tyrosine and phenylalanine [180,181]. Whilst there is also some clustering of genes for related pathways in *P. aeruginosa*, in this organism two of the mandelate genes are separated by a group of genes encoding enzymes for the degradation of benzoate and catechol [182,183]. In *Acinetobacter calcoaceticus* the mandelate genes appear to be clustered near the auxotrophic marker *phe-1* but are not all contiguous with each other [89,184]. A gene responsible for the expression of the cryptic L-mandelate dehydrogenase appears to be close to a gene required for the activity of D-mandelate dehydrogenase [89].

The enzymes of peripheral catabolic pathways are often plasmid-encoded [185] and it will be interesting to see whether this turns out to be the case for the mandelate enzymes in any organism.

### 11. FUTURE WORK

Specific questions that need to be answered include:

(a) Does the conversion of 4-hydroxymandelate into 2,5-dihydroxymandelate observed in *Polyporus tumulosus* [51] involve an intramolecular rearrangement of the type invoked for the conversion of 4-hydroxyphenylacetate into homogentisate [186,187]?

(b) What is the mechanism of 4-hydroxymandelate oxidase which allows it to do a task that usually requires two successive enzymes (see Section 8.6)?

(c) Why do some organisms have multiple benzaldehyde dehydrogenases associated with the mandelate pathway (see Section 8.4)?

(d) Is the fact that some mandelate dehydrogenases use NAD(P) rather than flavin as cofactor simply a consequence of their independent evolutionary origins, or does it have functional significance in allowing the step to be reversed (see Section 8.2)?

More general questions include:
(a) What is the precise arrangement of the structural and regulatory genes required for expression of the mandelate enzymes in different organisms?

(b) Is there a multi-enzyme mandelate complex? Does channelling of the mandelate intermediates occur? Are there transient enzyme interactions in the pathway?

(c) What are the molecular mechanisms regulating gene expression?

(d) Are there specific transport mechanisms for these aromatic compounds? Is there chemotaxis towards the substrates? If so, how is it elicited?

(e) How common are cryptic genes for mandelate dehydrogenase [87–89] and other enzymes that are potentially important in these pathways? [See 188.] How do such enzymes become silent in the first place, what are the mechanisms that lead to their expression, and what is their ecological and evolutionary significance?

(f) What are the ecological roles of mandelate-utilising organisms? How common are these organisms? To what extent and how often are their substrates formed? What exactly are the substrates?

Finally, it will be necessary to ask:

(a) Can gene cloning and sequencing, coupled with further work on the properties of the enzyme proteins, provide sufficient clues to unravel the evolutionary relationships amongst the mandelate enzymes of the prokaryotes, of the eukaryotes, and between the enzymes of both these groups of organisms? To what extent has there been gene duplication (e.g., in the case of the benzaldehyde dehydrogenases)? Had some analogous enzymes different origins, perhaps by recruitment from quite different metabolic pathways (e.g., the mandelate dehydrogenases)?

(b) Is it correct to conclude that peripheral enzymic pathways for the degradation of relatively uncommon substrates that are only spasmodically present in the environment may serve as convenient fast-running ‘clocks’ for following enzyme evolution, in contrast to the enzymes of the central metabolic pathways (e.g., glycolysis [189]) which are always essential, must be carefully regulated and whose evolution is therefore subject to much tighter constraints?

---

### 12. CONCLUSIONS

The mandelate pathway provides an attractive miniature world for studying virtually every aspect of microbial biochemistry, physiology, genetics, ecology and evolution. The range of organisms and pathways involved is sufficiently great that worthwhile questions can be asked and answered, but not so excessive that too much distracting detail is accumulated.

---

### ACKNOWLEDGEMENTS

It has been a pleasure to have access to the excellent University Library here in Glasgow and in particular I am grateful for the constructive assistance of Margaret Coutts, Alison Faichney and their colleagues.

My research has been supported by grants from the Medical Research Council and the Science and Engineering Research Council.

---

### REFERENCES

[1] Supniewski, J. (1924) Transformation des composés aromatiques sous l’influence du Bacille pyocyanique. C. R. Soc. Pol. Biol. 90, 1111–1112.

[2] Stanier, R.Y. (1947) Simultaneous adaptation: a new technique for the study of metabolic pathways. J. Bacteriol. 54, 339–348.

[3] Stanier, R.Y. (1948) The oxidation of aromatic compounds by fluorescent pseudomonads. J. Bacteriol. 55, 477–494.

[4] Stanier, R.Y. (1950) Problems of bacterial oxidative metabolism. Bact. Rev. 14, 179–191.

[5] Stanier, R.Y. (1951) Enzymatic adaptation in bacteria. Annu. Rev. Microbiol. 5, 35–56.

[6] Gunsalus, I.C., Gunsalus, C.F. and Stanier, R.Y. (1953) The enzymatic conversion of mandelic acid to benzoic acid. I. Gross fractionation of the system into soluble and particulate components. J. Bacteriol. 66, 538–542.

[7] Stanier, R.Y., Gunsalus, I.C. and Gunsalus, C.F. (1953) The enzymatic conversion of mandelic acid to benzoic acid. II. Properties of the particulate fractions. J. Bacteriol. 66, 543–547.

[8] Gunsalus, C.F., Stanier, R.Y. and Gunsalus, I.C. (1953) The enzymatic conversion of mandelic acid to benzoic acid. III. Fractionation and properties of the soluble enzymes. J. Bacteriol. 66, 548–553.

[9] Carles, J. and Bourguet, M. (1957) Les acides organiques du blé. C. R. Soc. Biol. Toulouse 151, 383–387.

[10] Colagrande, O. (1959) Formazione ed evoluzione degli acidi organici durante la maturazione dell'uva. Ann. Microbiol. Enzimol. 9, 62–72.

[11] Conn, E.E. (1981) Cyanogenic glycosides. In: The Biochemistry of Plants, Vol. 7 (Stumpf, P.K. and Conn, E.E., Eds.), pp. 479–500. Academic Press, New York.

[12] Poulton, J.E. and Shin, S.-I. (1983) Prunasin biosynthesis by cell-free extracts from black cherry (*Prunus serotina* Ehrh.) fruits and leaves. Z. Naturforsch. 38C, 369–374.

[13] Börner, H. (1960) Liberation of organic substances from higher plants and their role in the soil sickness problem. Bot. Rev. 26, 393–424.

[14] Xu, L.-L., Singh, B.K. and Conn, E.E. (1986) Purification and characterisation of mandelonitrile lyase from *Prunus lyoni*. Arch. Biochem. Biophys. 250, 322–328.

[15] Israel, D.W., Giddens, J.E. and Powell, W.W. (1973) Toxicity of peach tree roots. Plant Soil 39, 103–112.

[16] Duffey, S.S. and Towers, G.H.N. (1978) On the biochemical basis of HCN production in the millipede *Harpaphe haydeniana* (Xystodesnidae : Polydesmida). Can. J. Zool. 56, 7–16.

[17] Nahrstedt, A. and Davis, R.H. (1986) (R)Mandelonitrile and prunasin, the sources of hydrogen cyanide in all stages of *Paropsis atomaria* (Coleoptera: Chrysomelidae). Z. Naturforsch. 41C, 928–934.

[18] Bandyopadhyay, A.K., Nagasawa, T., Asano, Y., Fujishiro, K., Tani, Y. and Yamada, H. (1986) Purification and characterisation of benzonitrilases from *Arthrobacter* sp. strain J-1. Appl. Environ. Microbiol. 51, 302–306.

[19] Jallageas, J.-C., Arnaud, A. and Galzy, P. (1980) Bioconversions of nitriles and their applications. Adv. Biochem. Eng. 14, 1–32.

[20] Nagasawa, T., Nanba, H., Ryuno, K., Takeuchi, K. and Yamada, H. (1987) Nitrile hydratase of *Pseudomonas chloraphis* B23. Purification and characterisation. Europ. J. Biochem. 162, 691–698.

[21] Luthe, H., Ludwig-Köhn, H. and Langenbeck, U. (1983) Quantitative gas chromatographic mass spectrometric determination of mandelic acid in blood plasma. Biomed. Mass Spectrum 10, 183–186.

[22] Singh, P.D.A., Jackson, J.R. and James, S.P. (1985) Metabolism of mandelonitrile in the rat. Biochem. Pharmacol. 34, 2207–2209.

[23] Chakrabarti, S.K. (1979) New fluorometric analysis for mandelic and phenylglyoxylic acids in urine as an index to styrene exposure. Clin. Chem. 25, 592–595.

[24] Wilson, H.K., Robertson, S.M., Waldron, H.A. and Gompertz, D. (1983) Effect of alcohol on the kinetics of mandelic acid excretion in volunteers exposed to styrene vapour. Br. J. Ind. Med. 40, 75–80.

[25] Korn, M., Wodarz, R., Schoknecht, W., Weichardt, H. and Bayer, E. (1984) Styrene metabolism in man: gas chromatographic separation of mandelic acid enantiomers in the urine of exposed persons. Arch. Toxicol. 55, 59–63.

[26] Cummins, L.M. and Perry, J.E. (1969) Spectrophotometric determination of pemoline and mandelic acid in biological fluids. J. Pharm. Sci. 58, 762–763.

[27] Sullivan, H.R., Miller, W.M. and McMahon, R.E. (1976) Reaction pathways of in vivo stereoselective conversion of ethylbenzene to (−)-mandelic acid. Xenobiotica 6, 49–54.

[28] Stecher, P.G. (Ed.) (1968) The Merck Index, 8th edition. Merck and Co., Rahway, N.J.

[29] Middleton, B., Middleton, A., Micink, A., White, D.A. and Bell, G.D. (1983) The inhibition of hepatic S-3-hydroxy-3-methylglutaryl-CoA reductase by 3,3,5-trimethylcyclohexanol and its mandelic acid ester, cyclandelate. Biochem. Pharmacol. 32, 649–651.

[30] Greenwood, D. and Slack, R.C.B. (1981) The antibacterial activity of hexamine (methenamine), hexamine hippurate and hexamine mandelate. Infection 9, 223–227.

[31] Kevorkian, C.G., Merritt, J.L. and Ilstrup, D.M. (1984) Methenamine mandelate with acidification: an effective urinary antiseptic in patients with neurogenic bladder. Mayo Clin. Proc. 59, 523–529.

[32] Van Patten, P.L. (1979) Mandelic acid and urinary tract infections. Antonie van Leeuwenhoek J. Microbiol. 45, 622–623.

[33] Watson, D.C. and Walker, A.P. (1978) A modification of Brilliant Green agar for improved isolation of *Salmonella*. J. Appl. Bacteriol. 45, 195–204.

[34] Jones, P.W., Collins, P. and Hayle, A.J. (1984) The effect of sodium sulphacetamide and sodium mandelate in brilliant green agar on the growth of salmonellas. J. Appl. Bacteriol. 57, 423–428.

[35] McGovern, T.P., Zettler, J.L. and McDonald, L.L. (1984) Repellents for adult *Tribolium confusum*. Mandelic acid esters of branched-chain alcohols. J. Ga. Entomol. Soc. 19, 356–366.

[36] Beggs, J.D. and Fewson, C.A. (1977) Regulation of synthesis of benzyl alcohol dehydrogenase in *Acinetobacter calcoaceticus* NCIB 8250. J. Gen. Microbiol. 103, 127–140.

[37] Kenyon, G.L. and Hegeman, G.D. (1979) Mandelate racemase. Adv. Enzymol. 50, 325–360.

[38] Whitehead, D.C. (1964) Identification of p-hydroxybenzoic, vanillic, p-coumaric and ferulic acids in soils. Nature 202, 417–418.

[39] Vance, G.F., Boyd, S.A. and Mokma, D.L. (1985) Extraction of phenolic compounds from a spodosol profile: an evaluation of three extractants. Soil Sci. 140, 412–420.

[40] Stevenson, F.J. (1967) In: Soil Biology (McLaren, A.D. and Peterson, G.H., Eds.) pp. 119–146. Edward Arnold, London.

[41] Smith, D.M., Paul, R.M., McGeer, E.G. and McGeeer, P.L. (1959) A general chromatographic survey of aromatic compounds obtained from urine. Canad. J. Biochem. Physiol. 37, 1493–1515.

[42] Goodall, M. and Alton, H. (1969) Metabolism in the

human of 3,4-dihydroxymandelic acid, one of the metabolites of noradrenaline and adrenaline. Biochem. Pharmacol. 18, 295–302.

[43] Lun, A., Hacker, R.R., Brown, R.G. and Hurnik, J.F. (1976) Determination of vanilmandelic acid in pig urine and chicken feces by gas-liquid chromatography. Anal. Biochem. 73, 267–273.

[44] Tuchman, M., Morris, C.L., Ramnaraine, M.L., Bowers, L.D. and Krivit, W. (1985) Value of random urinary homovanillic acid and vanillylmandelic acid levels in the diagnosis and management of patients with neuroblastoma: comparison with 24-hour urine collections. Pediatrics 75, 324–328.

[45] Axelrod, J. and Saavedra, J.M. (1977) Octopamine. Nature 265, 501–504.

[46] Perrin, P.W. and Towers, G.H.N. (1973) Metabolism of aromatic acids by *Polyporus hispidus*. Phytochem. 12, 583–587.

[47] Hockenhall, D.J.D., Walker, A.D., Wilkin, G.D. and Winder, F.G. (1952) Oxidation of phenylacetic acid by *Penicillium chrysogenum*. Biochem. J. 50, 605–609.

[48] Kishore, G., Sugumaran, M. and Vaidyanathan, C.S. (1974) *p*-Hydroxymandelic acid – a key intermediate in the metabolism of DL(±)-phenylalanine by *Aspergillus niger*. Biochem. Biophys. Res. Commun. 56, 851–859.

[49] Landymore, A.F., Antia, N.J. and Towers, G.H.N. (1978) The catabolism of L-phenylalanine, L-tyrosine, and some related aromatic compounds by two marine species of phytoplankton. Phycologia 17, 319–328.

[50] Manley, S.L. and Chapman, D.J. (1979) Metabolism of L-tyrosine to 4-hydroxybenzaldehyde and 3-bromo-4-hydroxybenzaldehyde by chloroplast-containing fractions of *Odonthalia floccosa* (Esp.) Falk. Plant Physiol. 64, 1032–1038.

[51] Crowden, R.K. (1967) Biosynthesis of the polyphenolic acid metabolites of *Polyporus tumulosus* Cooke. Canad. J. Microbiol. 13, 181–197.

[52] Larsen, P.O. and Wieczorkowska, E. (1975) Intermediates in the metabolism of *m*-carboxy-substituted aromatic amino acids in plants. Phenylpyruvic acids, mandelic acids, and phenylglyoxylic acids. Biochim. Biophys. Acta 381, 409–415.

[53] Lingens, F., Eberhardt, H. and Oltmanns, O. (1966) Mikrobieller Abbau des Chloramphenicols. Biochim. Biophys. Acta 130, 345–354.

[54] Strzelczyk, E., Donderski, W. and Lewosz, W. (1972) Utilization of aromatic compounds by benthic microorganisms of an eutrophic lake. Acta Microbiol. Pol. 4, 191–200.

[55] Prakasam, T.B.S. and Dondero, N.C. (1970) Aerobic heterotrophic bacterial populations of sewage and activated sludge. V. Analysis of population structure and activity. Appl. Microbiol. 19, 671–680.

[56] Prakasam, T.B.S. and Dondero, N.C. (1970) Aerobic heterotrophic bacterial populations of sewage and activated sludge. IV. Adaptation of activated sludge to utilization of aromatic compounds. Appl. Microbiol. 19, 663–670.

[57] Stanier, R.Y., Palleroni, N.J. and Doudoroff, M. (1966)

The aerobic pseudomonads: a taxonomic study. J. Gen. Microbiol. 43, 159–271.

[58] Hegeman, G.D., Rosenberg, E.Y. and Kenyon, G.L. (1970) Mandelic acid racemase from *Pseudomonas putida*. Purification and properties of the enzyme. Biochemistry 9, 4029–4036.

[59] Sleat, R. and Robinson, J.P. (1984) The bacteriology of anaerobic degradation of aromatic compounds. J. Appl. Bacteriol. 57, 381–394.

[60] Baumann, P., Doudoroff, M. and Stanier, R.Y. (1968) A study of the Moraxella group. II. Oxidase-negative species (genus *Acinetobacter*). J. Bacteriol. 95, 1520–1541.

[61] Fewson, C.A., Unpublished observations.

[62] Stevenson, I.L. (1967) Utilisation of aromatic hydrocarbons by *Arthrobacter* spp. Can. J. Microbiol. 13, 205–211.

[62a] Palleroni, N.J. (1984) Family I. Pseudomonadaceae, in Bergey’s Manual of Systematic Bacteriology, Vol. 1 (Krieg, N.R., Ed.), pp. 141–199. Williams & Wilkins, Baltimore, MD.

[63] Bhat, S.G. and Vaidyanathan, C.S. (1976) Involvement of 4-hydroxymandelic acid in the degradation of mandelic acid by *Pseudomonas convexa*. J. Bacteriol. 127, 1108–1118.

[64] Iyayi, C.B. and Dart, R.K. (1980) Metabolism of mandelic acid by *Aspergillus flavus*. Microbiol. Lett. 15, 127–133.

[65] Jamaluddin, M., Subba Rao, P.V. and Vaidyanathan, C.S. (1970) Involvement of the protocatechuate pathway in the metabolism of mandelic acid by *Aspergillus niger*. J. Bacteriol. 101, 786–793.

[66] Dart, R.K. and Iyayi, C.B. (1981) Degradation of mandelic acid by *Byssochlamys fulva*. Microbiol. Lett. 16, 27–33.

[67] Ramakrishna Rao, D.N. and Vaidyanathan, C.S. (1977) Metabolism of mandelic acid by *Neurospora crassa*. Can. J. Microbiol. 23, 1496–1499.

[68] Durham, D.R., McNamee, C.G. and Stewart, D.B. (1984) Dissimilation of aromatic compounds in *Rhodotorula graminis*: biochemical characterisation of pleiotropically negative mutants. J. Bacteriol. 160, 771–777.

[69] Harwood, C.S., Rivelli, M. and Ornston, L.N. (1984) Aromatic acids are chemoattractants for *Pseudomonas putida*. J. Bacteriol. 160, 622–628.

[70] Harwood, C.S. and Gibson, J. (1986) Uptake of benzoate by *Rhodopseudomonas palustris* grown anaerobically in light. J. Bacteriol. 165, 504–509.

[71] Thayer, J.R. and Wheelis, M.L. (1976) Characterisation of a benzoate permease mutant of *Pseudomonas putida*. Arch. Microbiol. 110, 37–42.

[72] Thayer, J.R. and Wheelis, M.L. (1982) Active transport of benzoate in *Pseudomonas putida*. J. Gen. Microbiol. 128, 1749–1753.

[73] Hegeman, G.D. (1966) Synthesis of the enzymes of the mandelate pathway by *Pseudomonas putida*. II. Isolation and properties of blocked mutants. J. Bacteriol. 91, 1155–1160.

[74] Cook, A.M. and Fewson, C.A. (1972) Evidence for

specific transport mechanisms for aromatic compounds in bacterium NCIB 8250. Biochim. Biophys. Acta 290, 384–388.

[75] Higgins, S.J. and Mandelstam, J. (1972) Regulation of pathways degrading aromatic substrates in *Pseudomonas putida*. Enzymic response to binary mixtures of substrates. Biochem. J. 126, 901–916.

[76] Higgins, S.J. and Mandelstam, J. (1972) Evidence for induced synthesis of an active transport factor for mandelate in *Pseudomonas putida*. Biochem. J. 126, 917–922.

[77] Hegeman, G.D. (1966) Synthesis of the enzymes of the mandelate pathway by *Pseudomonas putida*. I. Synthesis of enzymes by the wild type. J. Bacteriol. 91, 1140–1154.

[78] Hegeman, G.D. (1966) Synthesis of the enzymes of the mandelate pathway by *Pseudomonas putida*. III. Isolation and properties of constitutive strains. J. Bacteriol. 91, 1161–1167.

[79] Reiner, A.M. and Hegeman, G.D. (1971) Metabolism of benzoic acid by bacteria. Accumulation of (−)-3,5-cyclohexadiene-1,2-diol-1-carboxylic acid by a mutant strain of *Alcaligenes eutrophus*. Biochemistry 10, 2530–2536.

[80] Reiner, A.M. (1971) Metabolism of benzoic acid by bacteria: 3,5-cyclohexadiene-1,2-diol-1-carboxylic acid is an intermediate in the formation of catechol. J. Bacteriol. 108, 89–94.

[81] Reiner, A.M. (1972) Metabolism of aromatic compounds in bacteria. Purification and properties of the catechol-forming enzyme, 3,5-cyclohexadiene-1,2-diol-1-carboxylic acid (NAD⁺) oxidoreductase (decarboxylating). J. Biol. Chem. 247, 4960–4965.

[82] Stanier, R.Y., Sleeper, B.P., Tsuchida, M. and MacDonald, D.L. (1950) The bacterial oxidation of aromatic compounds. III. The enzymatic oxidation of catechol and protocatechuic acid to β-ketoadipic acid. J. Bacteriol. 59, 137–151.

[83] Ornston, L.N. and Stanier, R.Y. (1964) Mechanism of β-ketoadipate formation by bacteria. Nature 204, 1279–1283.

[84] Stanier, R.Y. and Ornston, L.N. (1973) The β-ketoadipate pathway. Adv. Microbiol. Physiol. 9, 89–151.

[85] Fewson, C.A. (1981) Biodegradation of aromatics with industrial relevance. In: Microbial Degradation of Xenobiotics and Recalcitrant Compounds (Leisinger, T., Cook, A.M., Hütter, R. and Nuesch, J., Eds.), pp. 141–179. Academic Press, London.

[86] Rosenberg, S.L. (1971) Regulation of the mandelate pathway in *Pseudomonas aeruginosa*. J. Bacteriol. 108, 1257–1269.

[87] Hills, C.A. and Fewson, C.A. (1983) Mutant strains of *Acinetobacter calcoaceticus* possessing additional mandelate dehydrogenases. Identification and preliminary characterisation of the enzymes. Biochem. J. 209, 379–386.

[88] Hills, C.A. and Fewson, C.A. (1983) Regulation of expression of novel mandelate dehydrogenases in mutants of *Acinetobacter calcoaceticus*. J. Gen. Microbiol. 129, 2009–2015.

[89] Vakeria, D., Vivian, A. and Fewson, C.A. (1984) Isolation, characterisation and mapping of mandelate pathway mutants of *Acinetobacter calcoaceticus*. J. Gen. Microbiol. 130, 2893–2903.

[90] Kennedy, S.I.T. and Fewson, C.A. (1968) Enzymes of the mandelate pathway in bacterium NCIB 8250. Biochem. J. 107, 497–506.

[91] Kennedy, S.I.T. and Fewson, C.A. (1968) Metabolism of mandelate and related compounds by bacterium NCIB 8250. J. Gen. Microbiol. 53, 259–273.

[92] Livingstone, A., Fewson, C.A., Kennedy, S.I.T. and Zatman, L.J. (1972) Two benzaldehyde dehydrogenases in bacterium N.C.I.B. 8250. Distinguishing properties and regulation. Biochem. J. 130, 927–935.

[93] Livingstone, A. and Fewson, C.A. (1972) Regulation of the enzymes converting L-mandelate into benzoate in bacterium N.C.I.B. 8250. Biochem. J. 130, 937–946.

[94] Canovas, J.L. and Stanier, R.Y. (1967) Regulation of the enzymes of the β-ketoadipate pathway in *Moraxella calcoacetica*. I. General aspects. Europ. J. Biochem. 1, 289–300.

[95] Bhat, S.G., Ramanayaganan, M. and Vaidyanathan, C.S. (1973) Mandelic acid-4-hydroxylase, a new inducible enzyme from *Pseudomonas convexa*. Biochem. Biophys. Res. Commun. 52, 834–842.

[96] Bhat, S.G. and Vaidyanathan, C.S. (1976) Purification and properties of L-4-hydroxymandelate oxidase from *Pseudomonas convexa*. Europ. J. Biochem. 68, 323–331.

[97] Bhat, S.G. and Vaidyanathan, C.S. (1976) Purification and properties of L-mandelate-4-hydroxylase from *Pseudomonas convexa*. Arch. Biochem. Biophys. 176, 314–323.

[98] Durham, D.R. (1984) Initial reactions involved in the dissimilation of mandelate by *Rhodotorula graminis*. J. Bacteriol. 160, 778–780.

[99] Iyayi, C.B. and Dart, R.K. (1986) Control of mandelic acid degradation in *Byssochlamys fulva*. Microbios 45, 161–167.

[100] Ornston, L.N. and Parke, D. (1977) The evolution of induction mechanisms in bacteria: insights derived from the study of the β-ketoadipate pathway. Curr. Topics Cell. Regul. 12, 210–262.

[101] Dagley, S. (1978) Pathways for the utilisation of organic growth substrates. In: The Bacteria, Vol. VI (Bacterial Diversity) (Ornston, L.N. and Sokatch, J.R., Eds), pp. 305–388. Academic Press, New York, NY.

[102] Stanier, R.Y. (1968) Biochemical and immunological studies on the evolution of a metabolic pathway in bacteria. In: Chemotaxonomy and Serotaxonomy (Hawkes, J.G., Ed.), pp. 201–225. Academic Press, London.

[103] Ornston, L.N. and Yeh, W.-K. (1981) Toward molecular natural history. Microbiology 1981, 140–143.

[104] Van den Tweel, W.J.J., Smits, J.P. and De Bont, J.A.M. (1986) Microbial metabolism of D- and L-phenylglycine by *Pseudomonas putida*. Arch. Microbiol. 144, 169–174.

[105] Van den Tweel, W.J.J. and De Bont, J.A.M. (1987) Metabolism of both stereoisomers of phenylglycine by

different routes in *Flavobacterium F2X*. J. Gen. Microbiol. 133, 745–754.

[106] Mayer, P. (1926) Zur Biochemie des Asymmetrieproblems. Biochem. Z. 174, 420–424.

[107] Neuberg, C. and Simon, E. (1928) Weitere Untersuchungen über die stereochemische Spezifität der Ketaldehydmutase. Biochem. Z. 200, 468–472.

[108] Laverack, P.D. and Clarke, P.H. (1979) Selection of mandelamide constitutive mutants in continuous culture of *Pseudomonas putida*. Biotech. Lett. 1, 353–358.

[109] Beggs, J.D., Cook, A.M. and Fewson, C.A. (1976) Regulation of growth of *Acinetobacter calcoaceticus* NCIB 8250 on benzyl alcohol in batch culture. J. Gen. Microbiol. 96, 365–374.

[110] Beggs, J.D. and Fewson, C.A. (1977) Regulation of synthesis of benzyl alcohol dehydrogenase in *Acinetobacter calcoaceticus* NCIB 8250. J. Gen. Microbiol. 103, 127–140.

[111] Collins, J. and Hegeman, G.D. (1984) Benzyl alcohol metabolism by *Pseudomonas putida*: a paradox resolved. Arch. Microbiol. 138, 153–160.

[112] Worsley, M.J. and Williams, P.A. (1975) Metabolism of toluene and xylenes by *Pseudomonas putida* (*arvilla*) mt-2: evidence for a new function of the TOL plasmid. J. Bacteriol. 124, 7–13.

[113] Sielicki, M., Focht, D.D. and Martin, J.P. (1978) Microbial transformations of styrene and \[^{14}C\]styrene in soil and enrichment cultures. Appl. Environ. Microbiol. 35, 124–128.

[114] Baggi, G., Boga, M.M., Catelani, D., Galli, E. and Treccani, V. (1983) Styrene catabolism by a strain of *Pseudomonas fluorescens*. System. Appl. Microbiol. 4, 141–147.

[115] Gunter, S. (1953) The enzymatic oxidation of p-hydroxymandelic acid to p-hydroxybenzoic acid. J. Bacteriol. 66, 341–346.

[116] Kishore, G., Sugumaran, M. and Vaidyanathan, C.S. (1976) Metabolism of DL-(±)-phenylalanine by *Aspergillus niger*. J. Bacteriol. 128, 182–191.

[117] Brisse, B. and Dirschel, W. (1969) Über den enzymatischen Abbau von DL-4-Hydroxy-3-methoxy-phenylglyoxysäure (Vanilloylameisensäuer) zu Vanillinsäure. Hoppe-Seyler’s Z. Physiol. Chem. 350, 787–790.

[118] Dirschel, W. and Brisse, B. (1969) Über den enzymatischen Abbau von DL-3-Methoxy-4-hydroxy-mandel säure zu Vanillinsäure. Act. Endocrinol. 61, 104–110.

[119] Hegeman, G.D. and Ellman, G.L. (1973) Measurement of urinary d(−)-vanillylmandelic acid. U.S. Patent 3, 749, 643.

[120] Weil-Malherbe, H. (1966) Some properties of mandelate racemase from *Pseudomonas fluorescens*. Biochem. J. 101, 169–175.

[121] Rosano, C.L. (1964) Enzymatic method for determination of vanillyl mandelic acid. Clin. Chem. 19, 673–677.

[122] Muskiet, F.A.J., Nagel, G.T. and Wolthers, B.G. (1980) Simultaneous determination of unconjugated homovanillic acid, vanilmandelic acid, and 3-methoxy-4-hydroxyphenylethylene glycol, in serum by mass frag-

mentography and deuterated internal standards. Anal. Biochem. 109, 130–136.

[123] Kodama, K. and Nakata, T. (1984) Direct determination of urinary vanillylmandelic acid and homovanillic acid by high-performance liquid chromatography on an anion-exchange column. J. Chromatogr. 311, 369–374.

[124] Tuchman, M., Crippin, P.J. and Krivit, W. (1983) Capillary gas-chromatographic determination of urinary homovanillic acid and vanillylmandelic acid. Clin. Chem. 29, 828–831.

[124a] Sze, I.S.-Y. and Dagley, S. (1987) Degradation of substituted mandelic acids by *meta* fission reactions. J. Bacteriol. 169, 3833–3835.

[125] Kenyon, G.L. and Hegeman, G.D. (1970) Mandelic acid racemase from *Pseudomonas putida*. Evidence favouring a carbanion intermediate in the mechanism of action. Biochemistry 9, 4036–4043.

[126] Fee, J.A., Hegeman, G.D. and Kenyon, G.L. (1974) Mandelate racemase from *Pseudomonas putida*. Subunit composition and absolute divalent metal ion requirement. Biochemistry 13, 2528–2532.

[127] Fee, J.A., Hegeman, G.D. and Kenyon, G.L. (1974) Mandelate racemase from *Pseudomonas putida*. Affinity labelling of the enzyme by D,L-α-phenylglycidate in the presence of magnesium ion. Biochemistry 13, 2533–2538.

[128] Maggio, E.T., Kenyon, G.L., Mildvan, A.S. and Hegeman, G.D. (1975) Mandelate racemase from *Pseudomonas putida*. Magnetic resonance and kinetic studies of the mechanism of catalysis. Biochemistry 14, 1131–1139.

[129] Sharp, T.R., Hegeman, G.D. and Kenyon, G.L. (1977) Mandelate racemase from *Pseudomonas putida*. Absence of detectable intermolecular proton transfer accompanying racemization. Biochemistry 16, 1123–1128.

[130] Kenyon, G.L. and Hegeman, G.D. (1977) Mandelate racemase. Meth. Enzymol. 46, 541–548.

[131] Sharp, T.R., Hegeman, G.D. and Kenyon, G.L. (1979) A direct kinetic assay for mandelate racemase using circular dichroic measurements. Anal. Biochem. 94, 329–334.

[132] Whitman, C.P., Hegeman, G.D., Cleland, W.W. and Kenyon, G.L. (1985) Symmetry and asymmetry in mandelate racemase catalysis. Biochemistry 24, 3936–3942.

[133] Whitman, C.P., Craig, J.C. and Kenyon, G.L. (1985) Synthesis, chiroptical properties and absolute configuration of α-phenylglycidic acid. Tetrahedron 41, 1183–1192.

[134] Nakada, D. and Nozu, K. (1958) Protoplast formation and mandelic acid dehydrogenase localisation in *Pseudomonas fluorescens*. Jap. J. Microbiol. 2, 137–141.

[135] Allison, N., O’Donnell, M.J., Hoey, M.E. and Fewson, C.A. (1985) Membrane-bound lactate dehydrogenases and mandelate dehydrogenases of *Acinetobacter calcoaceticus*. Location and regulation of expression. Biochem. J. 227, 753–757.

[136] Allison, N., O’Donnell, M.J. and Fewson, C.A. (1985)

[137] Hoey, M.E., Allison, N., Scott, A.J. and Fewson, C.A. (1987) Purification and properties of L-mandelate dehydrogenase and comparison with other membrane-bound dehydrogenases from *Acinetobacter calcoaceticus*. Biochem. J. In Press.

[138] Allison, N. and Fewson, C.A. (1986) Purification and properties of L-lactate dehydrogenase from *Acinetobacter calcoaceticus*. FEMS Microbiol. Lett. 36, 183–186.

[139] Ramanarayanan, M. and Vaidyanathan, C.S. (1973) Mandelate oxidase of *Aspergillus niger*. Part I Properties of particulate D(−)-mandelate oxidase. Ind. J. Biochem. Biophys. 10, 254–256.

[140] McNamee, C., Wattam, R., Flickinger, J., Stewart, D. and Durham, D.R. (1984) Dissimilation of aromatic compounds by *Rhodotorula graminis*. In: Genetic Control of Environmental Pollutants (Omenn, G.S. and Hollaender, A., Eds.), pp. 370–372. Plenum, New York, NY.

[141] Baker, D.P. (1987) Unpublished observations.

[142] Binder-Votrba, G. (1926) Über die Vergärung der Phenylglyoxylsäure. Biochem. Z. 174, 440–442.

[143] Hegeman, G.D. (1970) Benzoylformate decarboxylase (*Pseudomonas putida*). Meth. Enzymol. 17, 674–678.

[144] Sampson, D. and Hunt, A.L. (1969) Studies of benzoylformate decarboxylase in *Pseudomonas aeruginosa*. Proc. Austral. Biochem. Soc. 2, 48.

[145] Sampson, D. and Hunt, A.L. (1970) Properties of benzoylformate decarboxylase – a non-oxidative α-keto acid decarboxylase from *Pseudomonas aeruginosa*. Proc. Austral. Biochem. Soc. 3, 91.

[146] Barrowman, M.M. and Fewson, C.A. (1985) Phenylglyoxylate decarboxylase and phenylpyruvate decarboxylase from *Acinetobacter calcoaceticus*. Curr. Microbiol. 12, 235–240.

[147] Ullrich, J. (1982) Structure–function relationships in pyruvate decarboxylase of yeast and wheat germ. Ann. N.Y. Acad. Sci. 378, 287–305.

[148] Barrowman, M.M., Harnett, W., Scott, A.J., Fewson, C.A. and Kusel, J.R. (1986) Immunological comparison of microbial TPP-dependent non-oxidative α-keto acid decarboxylases. FEMS Microbiol. Lett. 34, 57–60.

[149] Jamaluddin, M. (1971) Ph.D. Thesis, Indian Institute of Science, Bangalore.

[150] Jamaluddin, M. and Fewson, C.A. (1973) Negative cooperativity in the benzoylformate decarboxylase of *Acinetobacter calcoaceticus* NCIB 8250. Trans. Biochem. Soc. 1, 1258–1260.

[151] Stevenson, I.L. and Mandelstam, J. (1965) Induction and multi-sensitive end-product repression in the two converging pathways degrading aromatic substances in *Pseudomonas fluorescens*. Biochem. J. 96, 354–362.

[152] Collins, J. and Hegeman, G. (1984) Benzyl alcohol metabolism by *Pseudomonas putida*: a paradox resolved. Arch. Microbiol. 138, 153–160.

[153] MacKintosh, R.W. and Fewson, C.A. (1987) Microbial aromatic alcohol and aldehyde dehydrogenases. In: Enzymology and Molecular Biology of Carbonyl Metabolism. (Aldehyde Dehydrogenase, Aldo-Keto Reductase, and Alcohol Dehydrogenase (Weiner, H. and Flynn, T.G., Eds.), pp. 259–273. Alan Liss Inc., New York, NY.

[154] Stachow, C.S., Stevenson, I.L. and Day, D. (1967) Purification and properties of nicotinamide adenine dinucleotide phosphate-specific benzaldehyde dehydrogenase from *Pseudomonas*. J. Biol. Chem. 242, 5294–5300.

[155] Chalmers, R.M. (1987) Unpublished observations.

[156] Sugumaran, M. (1986) Tyrosinase catalyses an unusual oxidative decarboxylation of 3,4-dihydroxymandelate. Biochemistry 25, 4489–4492.

[157] Reddy, C.C. and Vaidyanathan, C.S. (1975) Purification, properties and induction of a specific benzoate-4-hydroxylase from *Aspergillus niger* (UBC 814). Biochim. Biophys. Acta 384, 46–57.

[158] Reddy, C.C. and Vaidyanathan, C.S. (1976) Purification and properties of benzoate-4-hydroxylase from a soil pseudomonad. Arch. Biochem. Biophys. 177, 488–498.

[159] McNamee, C.G. and Durham, D.R. (1985) Properties of a membrane-associated benzoate-4-hydroxylase from *Rhodotorula graminis*. Biochem. Biophys. Res. Commun. 129, 485–492.

[160] Massey, V. and Hemmerich, P. (1975) Flavin and pteridine monooxygenases. In: The Enzymes, Vol. 12 (Boyer, P.D., Ed.), pp. 191–252. Academic Press, London.

[161] Yamaguchi, M. and Fujisawa, H. (1978) Characterisation of NADH-cytochrome c reductase, a component of benzoate 1,2-dioxygenase system from *Pseudomonas arvilla* C-1. J. Biol. Chem. 253, 8848–8853.

[162] Yamaguchi, M. and Fujisawa, H. (1980) Purification and characterisation of an oxygenase component in benzoate 1,2-dioxygenase system from *Pseudomonas arvilla* C-1. J. Biol. Chem. 255, 5058–5063.

[163] Yamaguchi, M. and Fujisawa, H. (1981) Reconstitution of iron-sulphur cluster of NADH-cytochrome c reductase, a component of benzoate 1,2-dioxygenase system from *Pseudomonas arvilla* C-1. J. Biol. Chem. 256, 6783–6787.

[164] Stanier, R.Y., Hegeman, G.D. and Ornston, L.N. (1964) The mechanism of so-called ‘sequential induction’ in *Pseudomonas fluorescens*. Colloq. Int. Cent. Nat. Rech. Sci. (Marseille), pp. 227–236.

[165] Ornston, L.N. (1971) Regulation of catabolic pathways in *Pseudomonas*. Bacteriol. Rev. 35, 87–116.

[166] Anderson, J.J. and Dagley, S. (1980) Catabolism of aromatic acids in *Trichosporon cutaneum*. J. Bacteriol. 141, 534–543.

[167] Rosenberg, S.L. (1971) Regulation of the mandelate enzymes in *Pseudomonas aeruginosa*. J. Bacteriol. 108, 1257–1269.

[168] Fewson, C.A., Livingstone, A. and Moyes, H.M. (1978) Mutants of *Acinetobacter calcoaceticus* NCIB 8250 con-

stitutive for the mandelate enzymes. J. Gen. Microbiol. 106, 233–239.

[169] Hegeman, G.D. and Root, R.T. (1976) The effect of a non-metabolizable analog on mandelate catabolism in *Pseudomonas putida*. Arch. Microbiol. 110, 19–25.

[170] Baumberg, S. (1981) The evolution of metabolic regulation. In: 32nd Symp. Soc. Gen. Microbiol., pp. 229–272.

[171] Mandelstam, J. and Jacoby, G.A. (1965) Induction and multi-sensitive end-product repression in the enzymic pathway degrading mandelate in *Pseudomonas fluorescens*. Biochem. J. 94, 569–577.

[172] Cook, A.M., Beggs, J.D. and Fewson, C.A. (1975) Regulation of growth of *Acinetobacter calcoaceticus* NCIB 8250 on L-mandelate in batch culture. J. Gen. Microbiol. 91, 325–337.

[173] Mandelstam, J. (1964) Metabolic repression of induced enzymes in *Escherichia coli* and *Pseudomonas fluorescens*. Coll. Int. Cent. Nat. Rech. Sci. (Marseille), pp. 221–225.

[174] Halpin, R.A., Hegeman, G.D. and Kenyon, G.L. (1981) Carbon-13 nuclear magnetic resonance studies of mandelate metabolism in whole bacterial cells and in isolated, in vivo cross-linked enzyme complexes. Biochemistry 20, 1525–1533.

[175] Hoey, M.E. (1987) Ph.D. thesis, University of Glasgow.

[176] Srivastava, D.K. and Bernhard, S.A. (1986) Metabolite transfer via enzyme–enzyme complexes. Science 234, 1081–1086.

[177] Chakrabarty, A.M., Gunsalus, C.F. and Gunsalus, I.C. (1968) Transduction and the clustering of genes in fluorescent pseudomonads. Proc. Natl. Acad. Sci. U.S.A. 60, 168–175.

[178] Chakrabarty, A.M. and Gunsalus, I.C. (1969) Defective phage and chromosome mobilization in *Pseudomonas*

[179] Chakrabarty, A.M. and Gunsalus, I.C. (1969) Autonomous replication of a defective transducing phage in *Pseudomonas putida*. Virology 38, 92–104.

[180] Wheelis, M.L. and Stanier, R.Y. (1970) The genetic control of dissimilatory pathways in *Pseudomonas putida*. Genetics 66, 245–266.

[181] Leidigh, B.J. and Wheelis, M.L. (1973) The clustering on the *Pseudomonas putida* chromosome of genes specifying dissimilatory functions. J. Molec. Evol. 2, 235–242.

[182] Kemp, M.B. and Hegeman, G.D. (1968) Genetic control of the β-ketoadipate pathway in *Pseudomonas aeruginosa*. J. Bacteriol. 96, 1488–1499.

[183] Rosenberg, S.L. and Hegeman, G.D. (1969) Clustering of functionally related genes in *Pseudomonas aeruginosa*. J. Bacteriol. 99, 353–355.

[184] Vakeria, D., Fewson, C.A. and Vivian, A. (1985) Gene transfer in *Acinetobacter calcoaceticus* NCIB 8250. FEMS Microbiol. Lett. 26, 141–145.

[185] Williams, P.A. (1981) Catabolic plasmids. Trends Biochem. Sci. 6, 23–26.

[186] Sugumaran, M. and Vaidyanathan, C.S. (1978) Metabolism of aromatic compounds. J. Ind. Inst. Sci. 60, 57–123.

[187] Harelard, W.A., Crawford, R.L., Chapman, P.J. and Dagley, S. (1975) Metabolic function and properties of 4-hydroxyphenylacetic acid 1-hydroxylase from *Pseudomonas acidovorans*. J. Bacteriol. 121, 272–285.

[188] Hall, B.G., Yokoyama, S. and Calhoun, D.H. (1983) Role of cryptic genes in microbial evolution. Mol. Biol. Evol. 1, 109–124.

[189] Fothergill-Gilmore, L.A. (1986) The evolution of the glycolytic pathway. Trends Biochem. Sci. 11, 47–51.
