
Journal of the Neurological Sciences 399 (2019) 217–226

Contents lists available at ScienceDirect

Journal of the Neurological Sciences

journal homepage: www.elsevier.com/locate/jns

Review Article

Genetics of amyotrophic lateral sclerosis: A review

Stéphane Mathis<sup>a,b,*</sup>, Cyril Goizet<sup>c,d</sup>, Antoine Soulages<sup>a</sup>, Jean-Michel Vallat<sup>e</sup>, Gwendal Le Masson<sup>f,g</sup>

<sup>a</sup> Department of Neurology, Nerve-Muscle Unit, CHU Bordeaux, (Pellegrin Hospital), University of Bordeaux, F-33000 Bordeaux, France  
<sup>b</sup> ALS Center, Nerve-Muscle Unit, CHU Bordeaux, (Pellegrin Hospital), University of Bordeaux, F-33000 Bordeaux, France  
<sup>c</sup> Department of Medical Genetics, ‘Centre de Référence Maladies Rares Neurogénétique’, CHU Bordeaux (Pellegrin Hospital), University of Bordeaux, 33000 Bordeaux, France  
<sup>d</sup> Laboratoire MRGM, INSERM U1211, F-33000 Bordeaux, France  
<sup>e</sup> Department of Neurology, ‘Centre de référence neuropathies rares’, 2 avenue Martin Luther King, F-87000 Limoges, France  
<sup>f</sup> Neurocentre Magendie, ‘Physiopathologie de la Plasticité Neuronale’, University of Bordeaux, U862, F-33000 Bordeaux, France  
<sup>g</sup> INSERM, Neurocentre Magendie, ‘Physiopathologie de la Plasticité Neuronale’, U862, F-33000 Bordeaux, France  

ARTICLE INFO

Keywords:  
Amyotrophic lateral sclerosis  
Genetics  
C9ORF72  
SOD1  
TDP-43  
FUS  
Classification  

ABSTRACT

Amyotrophic lateral sclerosis (ALS) is a neurodegenerative disorder of the motor pathways, invariably leading to death within a few years of onset. Most cases of ALS are sporadic, but familial forms of the disease (FALS) constitute 10% of the cases. Since the first identification of a causative gene in the 1990s and with recent advances in genetics, more than twenty genes have now been linked to FALS. This increased number of genes led to a tremendous amount of research, clearly contributed to a better understanding of the pathophysiology of this disorder, and paved the way for the development of new therapeutics and new hope for this fatal disease.

1. Introduction

Amyotrophic lateral sclerosis (ALS) has become a well-identified disease since the end of the 19th century, but its pathogenesis is still poorly understood despite the considerable efforts made in recent years [1]. Although the global incidence of ALS has tended to increase slightly over the years, especially in Western societies (probably in part due to the progressive aging of the population), ALS remains rare; its incidence is estimated to be 2–3 per 100,000 individuals in Europe and 0.7–0.8 per 100,000 individuals in Asia [2]. This neurodegenerative disorder is usually fatal within 2–5 years [3], with an estimated worldwide mortality of 30,000 patients a year [1]. The mean age of onset is 65, even if younger patients may be affected [4]. Primarily affecting the motor pathways, ALS is characterized by a combination of both upper and lower motor neuron degeneration [5]. Nevertheless, non-motor signs may also be associated with this disorder (cognitive dysfunction, frontotemporal dementia, extrapyramidal features, etc.), so ALS is currently widely regarded as a multisystem degeneration [6]. Moreover, several clinical variants of the disease are still known under the term ‘ALS’, broadening the phenotypical spectrum of the disease [5].

Our knowledge about ALS has progressively increased over the years, though no unifying hypotheses for its pathogenesis have emerged thus far. Most cases of ALS are sporadic (SALS), but a few have clearly been classified as ‘familial ALS’ (FALS) due to the identification of some causative genes over the three last decades. Many new pathophysiological concepts have emerged from the study of these genetic forms, and new therapeutic perspectives have appeared.

2. Genetics and amyotrophic lateral sclerosis: an old concept

In the middle of the 19th century, some authors, such as Jean Cruveilhier (1791–1874) [7] and Jacob Augustus Lockhart Clarke (1817–1880) [8], elaborated on the concept of ‘progressive muscular atrophy’. In relation to this clinical syndrome, some cases have been identified as having degeneration of the anterior horn of the spinal cord, corresponding to ‘amyotrophic lateral sclerosis’, a neurodegenerative disorder that was finally fully clinicopathologically described by Jean-Martin Charcot (1825–1893) [9], currently still known as ‘Charcot's disease’. ALS was long considered a nonfamilial and nonhereditary disorder in its classical form (Charcot's disease), even though some occasional observations of familial occurrences of ALS were reported in the medical literature in the subsequent decades. In the 1950s, an unusually high rate of ALS was reported in Guam (50 cases among

* Corresponding author at: Department of Neurology, Nerve-Muscle Unit, CHU Bordeaux, (Pellegrin Hospital), University of Bordeaux, F-33000 Bordeaux, France.  
E-mail address: stephane.mathis@chu-bordeaux.fr (S. Mathis).

https://doi.org/10.1016/j.jns.2019.02.030

Received 7 August 2018; Received in revised form 1 February 2019; Accepted 20 February 2019  
Available online 21 February 2019  
0022-510X/ © 2019 Elsevier B.V. All rights reserved.

S. Mathis, et al.

25,500 Guamanians), and among them, five were familial cases of ALS; based on this familial aggregation in the Chamorro population of the Mariana Islands, environmental factors and genetic influence were first assumed [10]. A few years later, Kurland and Mulder clearly identified the concept of ‘familial ALS with dominant inheritance’ [11,12]. In the following years, many cases of non-Chamorro FALS with transmission appearing to be autosomal dominant (AD) were confirmed [13–16]. At that time, some authors highlighted the great intra-familial variation in the evolution of FALS, as well as their various phenotypes [17]. As a consequence, ALS cases were initially classified into three different groups: 1) common sporadic form, 2) dominantly inherited form in adults, and 3) juvenile familial form [18].

Following the first descriptions of ALS, its pathological lesions were confined to the upper and lower motor neurons (loss of motor neurons and glial proliferation) [19]. However, in autopsies of some ALS patients, especially FALS patients, degeneration of the posterior columns (in 50% of FALS cases, against only 10% of SALS cases) was reported [20], as well as degeneration of Clarke's column and the spinocerebellar tracts (rarely observed in SALS at that time) [21]. Moreover, some non-motor signs were observed in ALS since the end of the 19th century, such as cognitive impairment and frontotemporal dementia [22], even though the clinical entity ‘ALS with dementia (frontotemporal degeneration)’ had not been clearly identified before the 1980s [23,24]. In the 1970s, ALS was still pathologically considered a “prominent band in a wide spectrum of subacute or chronic multiple system atrophies, with a predilection for certain parts of the motor system” [19]. Some clinical differences were also observed between FALS and SALS: earlier onset in FALS than in SALS (even juvenile), rare bulbar onset in FALS, and better survival in some FALS families [25]. Finally, in the 1970s, based on the clinicopathological phenotype, other authors proposed the following classification of FALS: 1) FALS with rapid, progressive loss of motor function, with predominantly lower motor neuron manifestations and a course lasting < 5 years (with anterior horn cells and pyramidal tracts); 2) the same as the first type but with additional degeneration of the posterior columns, Clarke's column, and the spinocerebellar tracts; and 3) FALS with long-term survival (> 10 years) [26].

### 3. Recent knowledge of the genetics of ALS

Although most cases of ALS are sporadic, FALS may represent 5–10% of the total number of the ALS cases. However, this prevalence may be underestimated (due to the variable definitions used), with not always a clear delineation between clinically defined FALS and SALS. If large-scale genetic screening becomes more accessible and increasingly more available in routine clinical practice, the genetic diversity and complexity of ALS still forces clinicians to refer to specific genetic testing and to precisely determine which genes are likely to be mutated. Some additional information, such as a family history of ALS and/or dementia, an earlier age at onset or some unusual clinical findings (sensory signs at diagnosis; predominantly lower-limb onset; bimodal curve of evolution), may be strongly suggestive of FALS [27]. In 2011, new criteria were proposed to define FALS, with three levels of certainty (‘definite’, ‘probable’ or ‘possible’) [28]. However, a recent international online survey including 21 different worldwide countries confirmed that there is still a lack of consensus among clinicians regarding the definition of FALS. This survey showed that genetic testing is not always an evidence-based process and is further complicated by significant differences in practice across geographic regions [29]. Finally, it is crucial but not always simple to distinguish between disease-causing genes and genes associated with ALS, as more evidence is needed for causality. For all these reasons, even if genetic counseling in ALS remains complex (with a significant number of cases in ALS being caused by more than one risk variant), any advances and progress in the genetics of ALS will also increase our understanding of the pathogenesis of the disease.

In 1993, *SOD1* (superoxide dismutase 1) was the first gene linked to FALS [30]. With the advances made in sequencing and molecular biological technologies (such as ‘next-generation sequencing’, NGS), many other genes have been identified [31]. The other important genetic abnormality in ALS is the hexanucleotide repeat expansion within the *C9ORF72* gene, first observed in 2011 in patients with frontotemporal degeneration (FTD), then in ALS [32,33]. Currently, more than thirty genes and loci have been linked to FALS ([http://alsod.iop.kcl.ac.uk/](http://alsod.iop.kcl.ac.uk/); Table 1), as well as to some sporadic cases of ALS (SALS). *C9ORF72*, *SOD1*, *TARDBP* and *FUS* are the most commonly mutated genes in ALS, but the frequency of these mutations strongly depends on ancestral origins [34]: most European FALS patients present with mutations of the *C9ORF72* (33.7%), *SOD1* (14.8%), *TARDBP* (4.2%) and *FUS* (2.8%) genes, and other known or unknown genes in 44.5% of European FALS cases; Asian FALS patients present with a different repartition of these genes, with *SOD1* (30%), *FUS* (6.4%), *C9ORF72* (2.3%), and *TARDBP* (1.5%), other known or unknown genes being observed in 59.8% of patients [34]. These findings suggest population-specific genetic risk factors such as the D90A *SOD1* mutation observed in Finno-Scandinavians [35]. Most of these genes confer dominant inheritance (not always with full penetrance), whereas some of them are recessive (*SOD1*, *FUS*, and *Optineurin*) and others can confer only autosomal recessive (*Spastacin*, *Alsin*, and *SIGMAR1*) or are even X-linked (*Ubiquilin-2*) inheritance [36]. Another challenge in FALS is the reduced penetrance of ALS-causing mutations and the genetic pleiotropy, with evidence of both oligogenic (in at least some cases) and polygenic inheritance in apparently sporadic ALS [37]. As a consequence, some isolated FALS cases may appear as SALS, and mutations causing FALS are frequently observed in apparent SALS [38]. In such cases, genetic abnormalities are found in 7.4% of Europeans (mainly *C9ORF72*) and 2.9% of Asians (mainly *SOD1*) [34].

#### 4. Genes most frequently implicated in familial amyotrophic lateral sclerosis

All the genes implicated in FALS are mentioned in Table 1 and Table 2. The localization of the targets (proteins) of the main causative genes of FALS appears in Fig. 1.

##### 4.1. Copper-zinc (Cu/Zn) superoxide dismutase (or *SOD1*)

The antioxidant enzyme *SOD1* is a 32-kilodalton homodimer found in many parts of human cells (nucleus, cytosol, mitochondrial intermembrane space, etc.) that contains one copper and one zinc binding site [39]. The functions of *SOD1* are to protect cells from reactive oxygen species (ROS) and to lower the steady-state concentration of superoxide by converting it into oxygen and hydrogen peroxide (via cyclic oxidation and reduction reactions through their active site metal) [40]. In 1993, the *SOD1* gene was the first mutated gene shown to cause FALS [30]. Currently, > 180 mutations have been described in this gene. Although *SOD1* mutations represent the first genetic cause of ALS in Asians, *SOD1* is only the second most frequent gene involved in Europeans [34]. Moreover, it remains controversial whether all of the variants discovered can be considered pathogenic and thus disease-causative. Patients with *SOD1*-related ALS present some specificities compared to those with ‘classical ALS’: longer duration of disease, earlier age of onset and some clinical characteristics (motor symptoms usually beginning in the lower limbs, with rare occurrences of cognitive disturbances) [41,42].

The discovery of the implication of *SOD1* in ALS has considerably accelerated the understanding of this disease, especially through the development of *SOD1* transgenic animal models of ALS (mice and rats). Mutated *SOD1* triggers neurotoxicity, but the detailed molecular mechanisms of this toxicity are still not fully known. Oxidative stress and iron metabolism dysregulation are observed in both ALS patients carrying *SOD1* mutations and *SOD1* animal models [43]. In cases of mutation, *SOD1* contains an exposed N-terminal short region, the derlin-1-

Table 1  
Classification of inherited forms of amyotrophic lateral sclerosis.

| Mode of inheritance | Current denomination | OMIM | Gene | Chromosome | Proposed denomination | Other neuropsychiatric/ neuromuscular phenotypes |
|--------------------|---------------------|------|------|------------|-----------------------|-----------------------------------------------|
| Autosomal Dominant (AD) | ALS1 | 105400 | SOD1 (superoxide dismutase 1) | 21q22.11 | AD-ALS-SOD1 | - |
|                      | ALS3 | 606640 | Unknown | 18q21 | AD-ALS-Unknown | - |
|                      | ALS4 | 602433 | SETX (senataxin) | 9q34.13 | AD-ALS-SETX | - AR-CA-SETX (SCAR1) |
|                      | ALS6 | 608030 | FUS/TLS (Fused in sarcoma/translated in liposarcoma) | 16q11.2 | AD-ALS-FUS | - AD-ET-FUS (essential tremor type 4) |
|                      | ALS7 | 608031 | Unknown | 20p13 | AD-ALS-Unknown | - AD-SMA-Unknown (Finkel type SMA) |
|                      | ALS8 | 608627 | VAPB (vesicle-associated membrane protein B angiogenin) | 20q13.32 | AD-ALS-VAPB | - AD-SMA-VAPB |
|                      | ALS9 | 611895 | ANG (angiogenin) | 14q11.2 | AD-ALS-ANG | - |
|                      | ALS10 | 612069 | TARDBP (TAR DNA-binding protein 43) | 1p36.22 | AD-ALS-TARDBP | - AD-FTD-TARDBP |
|                      | ALS11 | 612577 | FIG.4 (phosphoinositide 5-phosphatase) | 6q21 | AD-ALS-FIG.4 | - AR-CMT-FIG.4 (CMT4J) |
|                      | ALS12 | 613435 | OPTN (optineurin) | 10p13 | AD-ALS-OPTN | - |
|                      | ALS13 | 183090 | ATXN2 (ataxin-2)<sup>a</sup> | 12q24.12 | - | - AD-CA-ATXN2 (SCA2) - Parkinson's disease<sup>a</sup> |
|                      | ALS14 | 613954 | VCP (valosin-containing protein) | 9p13.3 | AD-ALS-VCP | - AD-CMTax-VCP (CMT2Y) - AD-SPG-VCP - AD-IBM/P/FTD-VCP |
|                      | ALS17 | 614696 | CHMP2B (charged multivesicular body protein 2B) | 3p11.2 | AD-ALS-CHMP2B | - AD-Dementia-CHMP2B |
|                      | ALS18 | 614808 | PFN1 (Profilin-1) | 17p13.2 | AD-ALS-PFN1 | - |
|                      | ALS19 | 615515 | ERBB4 (chorion protein gene ErB.4) | 2q34 | AD-ALS-ERBB4 | - |
|                      | ALS20 | 615426 | HNRNPA1 (heterogeneous nuclear ribonucleoprotein A1) | 12q13 | AD-ALS-HNRNPA1 | - AD-IBM/P/FTD-HNRNPA1 |
|                      | ALS21 | 606070 | MATR3 (matrin-3) | 5q31.2 | AD-ALS-MATR3 | - |
|                      | ALS22 | 616208 | TUBA4A (tubulin alpha-4A) | 2q35 | AD-ALS-TUBA4A | - AD-ALS/FTD-TUBA4A |
|                      | ALS23 | 617839 | ANXA11 (annexin A11) | 10q22.3 | AD-ALS-ANXAII | - |
|                      | -     | -      | DAO (D-amino acid oxidase) | 12q24 | AD-ALS-DAO | - AD-Schizophrenia-DAO |
|                      | -     | -      | KIAA0693/CREST (calcium-responsive transactivator)/SS18L1 (synovial sarcoma translocation gene on chromosome 18-like 1) | 20q13 | - | - |
|                      | FTD-ALS1 | 105550 | C9ORF72 (chromosome 9 open reading frame 72) | 9p21.2 | AD-ALS-C9ORF72 | - AD-ALS/FTD-C9ORF72 - AD-FTD-C9ORF72 |
|                      | FTD-ALS2 | 615911 | CHCHD10 (coiled-coil-helix-coiled-coil-helix domain containing protein 10) | 22q11.23 | AD-ALS-CHCHD10 | - AD-ALS/FTD-CHCHD10 - AD-FTD-CHCHD10 - AD-myopathy-CHCHD10 - AD-SMA-CHCHD10 (Jokely type SMA) |
|                      | FTD-ALS3 | 616437 | SQSTM1/p62 (sequestosome-1) | 5q35.3 | AD-ALS-SQSTM1 | - AD-ALS/FTD-SQSTM1 - AD-FTD-SQSTM1 - AD-NADGP-SQSTM1 - AD-myopathy-SQSTM1 |
|                      | FTD-ALS4 | 616439 | TBK1 (TANK-binding kinase 1) | 12q14.2 | AD-ALS-TBK1 | - AD-ALS/FTD-TBK1 - AD-FTD-TBK1 |
| Autosomal Recessive (AR) | ALS1 | 105400 | SOD1 (superoxide dismutase 1) | 21q22.1 | AR-ALS-SOD1 | - |
|                      | ALS2 | 205100 | KIAA1563 (alsin) | 2q33.1 | AR-ALS-KIAA1563 | - AR-PLS-KIAA1563 - AR-SPG-KIAA1563 |
|                      | ALS5 | 602099 | KIAA1840/SPG11 (spastacin) | 15q15-21.1 | AR-ALS-KIAA1840 | - AR-CMT-KIAA1563 (CMT2X) - AR-SPG-KIAA1840 (SPG11) |
|                      | ALS6 | 608030 | FUS/TLS (Fused in sarcoma/translated in liposarcoma) | 16q11.2 | AR-ALS-FUS | - |
|                      | ALS12 | 613435 | OPTN (Optineurin) | 10p13 | AR-ALS-OPTN | - |
|                      | ALS16 | 614373 | SIGMAR1 (Sigma nonopioid intracellular receptor 1) | 9p13.3 | AR-ALS-SIGMAR1 | - AR-SMA-SIGMAR1 |
|                      | LAHCDA | 611890 | GLE1 (GLE1, RNA export mediator) | 9q34.11 | AR-ALS-GLE1 | - AR-LCCS-GLE1 |
| X-linked (XL)       | ALS15 | 300857 | UBQLN2 (Ubiquilin-2) | Xp11.21 | XL-ALS-UBQLN2 | - XL-ALS/FTD-UBQLN2 |

AD = autosomal dominant; ALS = amyotrophis lateral sclerosis; AR = autosomal recessive; ax = axonal; CA = hereditary cerebellar ataxia; CMT = Charcot-Marie-Tooth disease; de = demyelinating; dHMN = distal hereditary motor neuropathy; ET = essential tremor; FTD = frontotemporal dementia; HSAN = hereditary sensory and autonomic neuropathy; IBM/P/FTD = inclusion body myositis/Paget disease of bones/frontotemporal dementia; int = intermediate; LAHCDA = lethal anterior horn cell disease with athrogryposis; mit = mitochondrial transmission; LCCS: lethal congenital contracture syndrome; NADGP: neurodegeneration with ataxia, dystonia, gaze palsy, childhood onset; PLS = primary lateral sclerosis; SCAR1 = autosomal recessive spinocerebellar ataxia 1 SMA = spinal muscular atrophy; SPG = hereditary spastic paraplegia; XL = X-linked.

<sup>a</sup> Susceptibility (risk factor).

binding region (DBR), which provokes endoplasmic reticulum stress; it has been suggested that pathogenic SOD1 mutations induce a common conformational change in SOD1 that may lead to motor neuron toxicity [44]. Oxidative stress due to mutations of the SOD1 gene has been attributed to a gain in SOD1 toxicity rather than a loss of SOD1 enzymatic function [43]. Excitotoxicity (glutamate-mediated neurotoxicity) is considered a possible pathogenic mechanism of ALS and is the reason why the glutamate release inhibitor riluzole, which is still the only approved drug (with edaravone) for ALS [5], shows modest therapeutic efficiency in this neurodegenerative disorder [45]. Even if controversial in SOD1 ALS patients [46], excitotoxicity may be caused by a selective loss of glutamate transporter-1 (GLT1) through various mechanisms (cleavage by caspase-3, alternative RNA editing, etc.) [47], excessive glutamate efflux or toxicity of glial cells, the consequences of which are

S. Mathis, et al.

Journal of the Neurological Sciences 399 (2019) 217–226

Table 2  
Other genes potentially linked with amyotrophic lateral sclerosis.

| OMIM | Gene | Chromosome |
|------|------|------------|
| 609894 | UNC13A (homolog of *Caenorhabditis elegans* A) | 19p13.11 |
| 105400 | DCTN1 (dynactin-1) | 2p13.1 |
| 105400 | PRPH (peripherin) | 12q13.12 |
| 105400 | NEFH (neurofilament, heavy polypeptide) | 22q12.2 |
| 601574 | TAF15 (TATA box-binding protein-associated factor 15) | 17q12 |
| 604277 | SPAST (spastin) | 2p22.3 |
| 612722 | ELP3 (elongator acetyltransferase complex subunit 3) | 8p21.1 |
| 150340 | LMNB1 (lamin B1) | 5q23.2 |
| 604588 | NEK1 (never in mitosis gene A-related kinase 1) | 4q33 |
| 600227 | CCNF (cyclin F) | 16p13.3 |
| 611604 | ERLIN1 (endoplasmic reticulum lipid raft-associated protein 1) | 10q24.31 |
| 617921 | KIF5A (kinesin family member 5A) | 12q13.3 |
| 612237 | TAF15 (TATA box-binding protein-associated factor, 68 kD) | 17q12 |
| – | TIA1 (T cell-restricted intracellular antigen-1) | 2p13.3 |
| – | EWSR1 (Ewing sarcoma breakpoint region 1) | 22q12.2 |
| – | GNE (UDP-N-acetylglucosamine-2 epimerase/N-acetylmannosamine-2 kinase) | 9p13.3 |

BCL2: B-cell CLL lymphoma; UDP: uridine diphosphate.

related ALS patients have many more HREMs (> 70) [32,33]. Massive G4C2 HREMs have also been observed in 5–10% of SALS patients in some populations [32,34,59]. The classical association of ALS/FTD may be termed ‘C9FTD/ALS’ when caused by C9ORF72 expansion mutations. In such cases, the phenotype may be FTD, ALS, or a combination of both; moreover, various other neurodegenerative disorders may be observed within the same pedigree (Parkinsonism, primary dementia or primary progressive aphasia) [60]. Interestingly, presymptomatic carriers of the C9ORF72 expansion mutation exhibit differences compared to healthy controls of similar ages, with earlier brain atrophy (especially focal atrophy of the left supra-marginal gyrus) and cognitive alterations [61].

Some hypotheses have been proposed to explain the role of C9ORF72 in the development of C9ORF72-related disorder. The first one was the inhibition of endosomal trafficking and the perturbation of endocytosis (leading to autophagy) caused by a reduction in the C9ORF72 protein level; however, knockout of C9ORF72 in mice does not cause motor neuron degeneration [62]. Another hypothesis was that massive G4C2 HREMs may be neurotoxic by forming length-dependent RNA foci, thereby sequestering RNA binding proteins and disabling the RNA processing machinery [63]. The third hypothesis was based on the repeat-associated non-AUG translation of G4C2 (or G2C4) RNA in five different dipeptide repeat proteins (DPR) called ‘polyGA’, ‘polyGP’, ‘polyGR’, ‘polyPA’ and ‘polyPR’ [64]. Though aggregation of DPR has been observed in the motor neurons of ALS patients, the role of DPR in the development of the disease remains uncertain; recently, polyGA was shown to be the most toxic of the DPR through activation of programmed cell death and trans-activation response (TAR) DNA-binding protein 43 (TDP-43) cleavage in a dose-dependent manner, ultimately modifying the pathogenic status of the other DPR [64]. C9ORF72 may also disrupt the nucleocytoplasmic shuttling at the level of the isoform of the C9ORF72 protein (C9–S), localized at the nuclear membrane in healthy controls: there is a correlation between the loss of C9–S and the mislocalization of TDP-43 [65]. C9ORF72 expansion mutations also lead to vulnerability to Ca2+ permeable α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptor-mediated excitotoxicity, additionally leading to motor neuron degeneration [66]. Finally, haploinsufficiency for C9ORF72 activity triggers neurodegeneration in C9ORF72 ALS patients, this occurring by at least two mechanisms: accumulation of glutamate receptors (leading to excitotoxicity) and impaired clearance of neurotoxic DRP derived from the repeat expansion: in ALS, neurodegeneration is due to this cooperativity between gain- and loss-of-function mechanisms [67].

### 4.3. Trans-activation response (TAR) DNA-binding protein 43 (TDP-43 or TARDBP)

Common histopathological hallmarks exist in many neurodegenerative disorders, such as protein misfolding, accumulation of intra- or extra-cellular protein aggregates and abnormal protein degradation; they are often characterized by pathological inclusions in the brain or spinal cord [68]. This is particularly true in ALS and DFT where cytoplasmic mislocalization and aggregation of TDP-43 is characteristic. The aggregated forms of TDP-43 are characterized by abnormal phosphorylation, truncation, and cytoplasmic mislocalization. Neurodegeneration is probably due to a gain of toxic, abnormal TDP-43 or to a loss of function in physiological TDP-43. It has been proved that phosphorylated TDP-43 (pTDP-43) disseminates in a sequential pattern along the motor pathways, as suggested by the hypothesis of cortico-fungal ('prion-like') propagation of misfolded proteins in ALS [69]; four pathological stages were observed, from the initial (stage 1: agranular motor cortex, motor nuclei of cranial nerves XII–X, VII, V in the brainstem, and spinal cord α-motor neurons) to the final regions (stage 4: anteromedial portions of the temporal lobe, including the hippocampus) [70]. The hypothesis that pTDP-43 spreads across the brain and spinal cord along axonal pathways has been recently confirmed by

### 4.2. Chromosome 9 open reading frame 72 (C9ORF72)

The C9ORF72 gene is located at locus 9p21 of chromosome 9. The first linkage to a locus on chromosome 9p13.2–21.3 was reported in 2006 in Scandinavian and Dutch kindred with AD-ALS and fronto-temporal degeneration [56,57]. In 2011, a massive GGGGCC (G4C2) hexanucleotide repeat expansion mutation (HREM) within intron 1 of C9ORF72 was finally identified as a pathogenic mutation in ALS [32,33]. This HREM has been implicated in many cases of familial or sporadic ALS/FTD, more frequently in Caucasians than in Asians [58]. Healthy subjects may have up to 30 G4C2 HREMs, whereas C9ORF72-

S. Mathis, et al.

Journal of the Neurological Sciences 399 (2019) 217–226

Fig. 1. Localization and role of the targets (proteins) of the main causative genes of familial amyotrophic lateral sclerosis. The blue dashed lines illustrate the impact of C9ORF72 on different sub-cellular structures; the red dashed lines illustrate the impact of SOD1 on different sub-cellular structures; The green dashed line illustrates the impact of FUS on different sub-cellular structures plain arrows illustrates direct proteins interactions (ANG: angiogenin; ANXA11: annexin A11; ATP: adenosine triphosphate; CHMP2B: charged multivesicular body protein-2B; CREST: calcium-responsive transactivator; DAO: D-amino acid oxidase; DCTN1: dynactin-1; DPR: dipeptide repeat protein; ERBB4: chorion protein gene ErB.4; FIG. 4: phosphoinositide 5-phosphatase; FUS: fused in sarcoma; GLE1: GLE1, RNA export mediator; GLT1: glutamate transporter-1; HNRNPA1: heterogeneous nuclear ribonucleoprotein A1; MATR3: matrin-3; mGluR: metabotropic glutamate receptor; OPTN: optineurin; p62: ubiquitin-binding protein p62 (or sequestosome-1); PFN1: profiling-1; PolyPR: poly PR dipeptide; pNa+: persistent sodium channels; RNA: ribonucleic acid; SIGMAR1: sigma nonopid intracellular receptor-1; SOD1: superoxide dismutase-1; TBK1: Tank-binding kinase-1; TDP-43: trans-activation response DNA-binding protein 43; TUBA4A: tubulin alpha-4A; UBQLN2: ubiquilin-2; VAPB: vesicle-associated membrane protein B; VCP: valosin-containing protein). (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)

computational simulation [71]. Moreover, even if ALS and FTD are overlapping syndromes, distinct TDP-43 inclusion morphologies in the anterior cingulate cortex (but not the motor cortex) are observed in ALS and FTD: in FTD, TDP-43 inclusions are significantly rounded (rarely circumferential), in contrast to ALS-FTD, where TDP-43 inclusions are mainly circumferential [72].

Patients with TARDBP-related ALS, first described in 2008 [73–75], present with the classical ALS phenotype with variable onsets and durations of symptoms [76]. TDP-43, encoded by the TARDBP gene, is a ubiquitously expressed nuclear ribonucleoprotein implicated in exon splicing, gene transcription, regulation of mRNA stability, mRNA biosynthesis, and formation of nuclear bodies [77]. Its C-terminal region (a low-complexity domain with a prion-like Gln/Asn-rich sequence) is capable of binding directly to several heterogeneous nuclear ribonucleoproteins mediating protein-protein interactions and splicing inhibitory activity. When its C-terminal region is lacking, TDP-43 becomes unable to mediate splicing inhibition [78]. We also know that TDP-43 deposition leads to targeted RNA instability, causing apoptosis by disrupting energy production and protein synthesis pathways [79]. However, the real biological function of TDP-43 is still unknown [78].

4.4. Fused in sarcoma/translocated in liposarcoma (FUS/TLS or FUS)

FUS/TLS is a nuclear RNA-binding protein that contains a C-terminal domain with a distinct RNA recognition motif surrounded by Arg-Gly-Gly repeats, first identified as an oncoprotein: in malignant human mixoid liposarcoma, the (12;16) rearrangement fuses FUS with the dominant inhibitor of transcription called CHOP (C/EBP Homologous Protein), this FUS-CHOP fusion protein activating the transcription of oncogenes and promoting tumorigenesis [80]. In 2009, the first cases of FUS-related ALS were described with highly variable clinical presentations, from classical ALS to more atypical phenotypes [81,82]. As we saw, TDP-43 neuronal inclusions are a pathological hallmark of both ALS and FTD; recently, it was shown that 10% of FTD patients present inclusions immunoreactive for FUS in neuronal and glial cells, without TDP-43 inclusions [83]. The formation of cytoplasmic FUS inclusions (resembling stress granules: cellular structures containing RNAs and proteins from suspended translation apparatus) promotes cell survival under stressed conditions by redistributing translation resources. But compromised stress granule response in the presence of FUS mutants is considered a contributing factor to motor neuron dysfunction [84,85].

S. Mathis, et al.

In neurons and glial cells, FUS is almost exclusively localized to the nucleus [86]. Moreover, most FALS mutations are clustered in the C-terminal nuclear localization sequence of FUS (chromosome 16), causing the mislocalization of FUS from the nucleus to the cytoplasm and the accumulation of protein inclusions [87]. This suggests two potential disease-causing mechanisms: loss of FUS normal function in the nucleus and gain of toxic function in the cytoplasm [88]. Moreover, FUS proteinopathies confirm the convergence of ALS and FTD pathophysiology.

As the other members of the FET (FUS-EWS-TAF15) family that are Ewing's sarcoma protein (EWS) and the TATA-binding protein associated factor (TAF15), FUS/TLS is a multifunctional DNA/RNA ribonuclear binding protein that plays a possible role in neuronal plasticity and maintenance of dendritic integrity [89,90]. If FUS is involved in various RNA metabolism pathways, its physiologic functions still await thorough characterization. First, FUS is implicated in RNA biogenesis and function: some mutations targeting the 3'-UTR untranslated region of the FUS messenger RNA are able to disrupt the microRNA-FUS regulatory circuitry [91]. Secondly, FUS helps to regulate transcription by binding active chromatin (chromatin binding is required for FUS transcription activation): the ALS mutations dramatically decreased FUS chromatin binding, and consequently may cause a loss of function of the nuclear FUS [88]. FUS also plays a critical role in the regulation of the basal level of the manganese SOD gene (SOD2) transcription, SOD2 being a highly-regulated antioxidant enzyme exclusively localized in mitochondria [92]. Thirdly, FUS plays a pivotal role in the immediate response to DNA damage by interacting with poly-ADP-ribose chains and involving a direct interaction with histone deacetylase 1 [93,94]. Finally, FUS is a crucible for embryogenesis by orchestrating splicing of developmental regulators during gastrulation [95].

### 4.5. Some other causative genes in familial amyotrophic lateral sclerosis

#### 4.5.1. Vesicle-associated membrane protein associated protein B (VAPB)
Mutations of the VAPB gene (mainly the P56S mutation) give motor neuron disorders first observed in some Brazilian families in the 1960's [96]. Initially classified as a proximal spinal muscular atrophy ('autosomal dominant late adult spinal muscular atrophy, type Finkel'), other symptoms, such as myotonia, were then reported [97]. VAPB-related motor neuron disorders are characterized by a longer course of evolution than the classical form of ALS [98].

#### 4.5.2. Valosin-containing protein (VCP)
Mutations of the VCP gene usually cause a multisystem disease combining inclusion-body myopathy, Paget's disease and FTD. However, it is also a rare cause of FALS, found in < 2% of the whole FALS [99].

#### 4.5.3. Angiogenin (ANG)
As VEGF (vascular endothelial growth factor), ANG is a crucible in the angiogenic process of motor neuron death. It usually presents as classical phenotype of ALS, sometimes with parkinsonism or FTD [100].

#### 4.5.4. Senataxin (SETX)
Mutations of the SETX gene may cause ataxia with oculomotor apraxia type 2 (AOA2), distal hereditary motor neuropathy (dHMN), as well as ALS4, a rare autosomal-dominant juvenile atypical form of ALS: ALS 4 is characterized by very low progression (with normal lifespan) and the absence of respiratory and bulbar symptoms (even in advanced stages of disease) [101].

#### 4.5.5. Alsチン (ALS2)
Mutations of the ALS2 gene cause a group of overlapping autosomal recessive neurodegenerative disorders characterized by a long duration of evolution and the absence of bulbar and respiratory symptoms: infantile-onset ascending hereditary spastic paralysis, juvenile primary lateral sclerosis (JPLS), and juvenile amyotrophic lateral sclerosis (ALS2). Alsチン mutations were never reported in adult patients [102].

#### 4.5.6. Tumor necrosis factor (TNF) receptor-associated factor NF-kB activator (TANK)-binding kinase 1 (TBK1)
The multifunctional kinase TBK1 is a multifunctional kinase participating in a variety of cell signal pathways through phosphorylation of various substrates: two of these substrates are optineurin (OPTN) and p62 (two autophagic adaptors regulating degradation by selective autophagy) that have been identified participating in the pathology of ALS/FTD spectrum. Actually, TBK1 loss of function and missense mutations are not frequently found in ALS/FTD patients: both of them are associated with an increased risk for ALS/FTD spectrum [103].

### 5. Are there genetic susceptibility factors in amyotrophic lateral sclerosis?

As many causative genes of ALS have been identified, other genes seem to be involved in ALS susceptibility. The first example was given in 2010 of ATXN2, encoding the ataxin-2 protein. As TDP-43, ataxin-2 is mislocalized in the neurons of ALS patients. High-length polyglutamine (polyQ) expansions (> 34 Qs) in ATXN2 cause spinocerebellar ataxia type 2 (SCA2) [104] and rarely familial parkinsonism [105], whereas intermediate-length polyQ expansions (27–33) represent a genetic risk factor for sporadic ALS [106]. Recently, some authors demonstrated that ataxin-1 polyQ expansions (> 32) also represent a risk factor for ALS; in addition, intermediate expansions of both ataxin-1 and ataxin-2 might be genetic modifiers of ALS-FTD [107]. Ataxin-2 is a stress-regulated protein localized in the rough endoplasmic reticulum of specific neurons [108]. The sequestration and depletion of ataxin-2 contribute to expanded ataxin-1 (a protein involved in transcriptional regulation and cell signaling) and induce neurodegeneration [109]. Activation of caspase-3 in response to cellular stress depends on ataxin-2 intermediate-length polyQ expansion; a possible explanation of this risk factor of ALS is that intermediate-length ataxin-2 polyQ expansions may enhance stress-induced TDP-43C-terminal cleavage and phosphorylation in human cells by lowering the threshold at which cells activate caspases upon cellular stress [110]. Other genes may also represent susceptibility genes in ALS, such as DCTN1, PRPH or NEFH (Table 2), though confirmation of these genes is still lacking [111]. A combination of genetic variants in several (oligogenic) or many (polygenic) genes may explain individual genetic vulnerability [112].

### 6. Challenges for the classification of genetic forms of amyotrophic lateral sclerosis

Because FALS may sometimes be apparent SALS, the term 'inherited ALS' is probably more appropriate than 'FALS'. > 30 forms of inherited ALS have been identified so far (Table 1), and additional genes remain to be discovered in the future. Since the first gene was identified in 1993 (SOD1), FALS has been renamed as 'ALS' associated with a number (ALS1, ALS2, etc.), and the last number used currently is '23' (OMIM), while some genes have no clear denomination yet (DAO, etc.). Initially, only autosomal-dominant forms were found, but now we know that autosomal-recessive and X-linked forms of FALS exist. Moreover, for the same gene, two different modes of inheritance may be observed, such as for SOD1 or FUS. Finally, one gene may generate various clinical phenotypes; for example, SETX may yield typical FALS or a form of inherited cerebellar ataxia (Table 1). Though the number of genes is not as elevated in inherited ALS as in other neurogenetic disorders, such as Charcot-Marie-Tooth (CMT), some difficulties still exist in the actual classification of inherited forms of ALS.

All recent discoveries in the field of genetic bases of rare diseases have generated increasing difficulty in the classification of genetically heterogeneous conditions, especially in neurogenetic disorders [113].
S. Mathis, *et al.*

By highlighting such complexity, some of us recently proposed another way to classify neurogenetic disorders [113, 114]. Our proposal was first applied to the complex classification of CMT, where many nomenclatures were aggregated, based on the first official classification set in 1968 [115]. Finally, the old names and classifications, which were usually based on numbers and letters, now appear to be unreliable due to the many confusing denominations [116]. Based on our observations, we have proposed to update the classification of inherited neuropathies by using a simple-to-use three-module approach: a) mode of inheritance (autosomal dominant, AD; autosomal recessive, AR; X-linked, XL; mitochondrial transmission, mit; sporadic, S), b) phenotype (CMT, etc.) and c) name of the gene (using the term ‘unknown’ for unknown genes) [115]. One of the advantages of this proposal is the simple implementation of any new genetic data [113]. Since these proposals, most of the many comments received from colleagues worldwide have been encouraging, as found in a recent online survey [117, 118].

Finally, we think that such a classification system may be used for many other neurogenetic and neuromuscular disorders [113, 114, 119], including FALS. For example, *FUS*-related disorders would be then termed ‘AD-ALS-FUS’ or ‘AR-ALS-FUS’ (function of the mode and transmission of FALS), or even ‘AD-ET-FUS’ in case or ‘hereditary essential tremor type 4’ (*Table 1*). Such proposals for a new classification are probably premature and currently not useful for FALS. However, such a way of classify neurogenetic disorders (including FALS) may be of value in the future with the probable increased number of new causative genes. Even if our proposals for CMT have been well received by the scientific community [117, 118], the validation of a new classification method for genetic disorders would require further discussions by expert working groups and discussions within the scientific and medical communities [117].

### 7. Gene therapy: a therapeutic way to explore in amyotrophic lateral sclerosis?

The emergence of multidisciplinary, specialized ALS clinics has increased both the quality of life and the survival of ALS patients, respiratory management through early non-invasive ventilation being probably one of the most important determinants of survival; however, there is still no curative treatment for ALS patients [5]. Gene therapy approaches, including antisense oligonucleotides (ASOs) and viral-directed gene delivery, have shown enormous potential to enable therapeutic modulation of gene expression in diverse monogenic neurological conditions, including ALS [120].

Gene therapy generally refers to the transfer of a therapeutic gene into a target tissue and the maintenance of gene function for a sufficient length of time [121]. However, the anatomical and functional complexity of the human brain, especially the existence of the blood-brain barrier, limits the biodistribution of the vector into the CNS: even assuming successful delivery of the vector via direct intraparenchymal, intrathecal, intracerebroventricular, or systemic administration, targeting a sufficient number of cells to achieve an adequate level of gene augmentation/suppression within a safe and therapeutic window appears challenging [122]. Based on the recent successes observed in spinal muscular atrophy, RNA-targeted therapies have been proposed as potential strategies for treating ALS, but still with some limitations: no RNA therapeutics have been found to be effective for ALS patients, but such a therapeutic approach could be considered in the future for the treatment of all forms of ALS [121].

ASOs bind to the mRNA by Watson-Crick base pairing, so one method of lowering SOD1 is to target the SOD1 mRNA using ASOs (that activate RNase H1 that destroys the targeted RNA) [121]. After first demonstrations of the utility of ASOs in animal models, a first human clinical trial began in 2013 (IONIS Pharmaceuticals (“ISIS Pharmaceuticals” until December 2015): it consisted in the intrathecal delivery of ASO against SOD1 (ISIS333611) for patients with *SOD1* FALS,

showing that it should be a feasible treatment for neurological disorders (ISIS333611 has no dose-limiting toxic effects or other safety and tolerability concerns; the CSF and plasma drug concentrations were consistent with those predicted in preclinical studies) [123]. Based on these results, a second generation *SOD1*-targeting ASOs, BIIB067 (IONIS-SOD1RX) is currently being tested in an ongoing phase I randomized, placebo-controlled escalating dose trial (NCT02623699), as well as the long-term extension study of the effects of BIIB067 (NCT03070119) in *SOD1* FALS patients [124]. Another interesting option is to target genes to introduce correction mutations in induced pluripotent stem cells from fibroblasts of *SOD1* and *FUS* FALS patients by using the CRISPR/Cas9 system [125].

Intraventricular administration ASOs fail to reduce *C9ORF72* RNA levels, but may reduce toxic RNA foci and dipeptide aggregates, restore normal gene expression markers and protect against glutamate toxicity, as observed in a recent study on mice expressing *C9ORF72* RNAs with up to 450 GGGGCC repeats or with one or both *C9ORF72* alleles inactivated [126]. The administration of ASO-targeting *ATXN2* to the CNS of *TDP43* transgenic mice markedly extends their lifespan and improves their motor performance [127]. This approach (targeting a modifier gene that does not directly cause ALS and reducing ataxin-2 levels), represents an original therapy for TDP-43 proteinopathy: this type of approach will likely be essential for treating sporadic ALS because of the cellular role of TDP-43, which is present in nearly all ALS patients [121]. Finally, with other treatments, such advances may be another step through a combinatorial therapeutic approach targeting different pathogenic mechanisms in independent cell type [128].

#### 8. Conclusions

ALS is an old but mysterious disease that is sporadic in most cases. FALS has been observed for many decades, with currently more than thirty causative genes having been discovered. Even if inherited forms of ALS represent only 10% of ALS cases, genetic advances contribute to a better understanding of the pathophysiology of the disease, leading to new potential treatments in the future for this neurodegenerative disorder, which is still fatal. Based on the recent successes observed in spinal muscular atrophy, RNA-targeted therapies have been proposed as potential strategies for treating ALS but are still without evident success thus far, as no RNA therapeutics have been found to be effective for ALS. Such therapeutic approaches for the treatment of all forms of ALS will undoubtedly be further developed.

#### Competing interests

The authors declare that they have no competing interests.

#### Funding

For the remaining authors (SM, CG, AS, JMV and GLM), none were declared.

#### Acknowledgements

We thank Mrs Jackie Stewart for the design of the Figure 1.

#### References

[1] D. Petrov, C. Mansfield, A. Moussy, O. Hermine, *ALS clinical trials review: 20 years of failure. Are we any closer to registering a new treatment?* Front. Aging Neurosci. 9 (2017) 68.

[2] O. Hardiman, A. Al-Chalabi, A. Chio, E.M. Corr, G. Logroscino, W. Robberecht, et al., *Amyotrophic lateral sclerosis,* Nat. Rev. Dis. Primers. 3 (2017) 17085.

[3] L.H. van den Berg, *Therapy of amyotrophic lateral sclerosis remains a challenge,* Lancet Neurol. 13 (2014) 1062–1063.

[4] M.A. van Es, O. Hardiman, A. Chio, A. Al-Chalabi, R.J. Pasterkamp, J.H. Veldink, et al., *Amyotrophic lateral sclerosis,* Lancet. 390 (2017) 2084–2098.

[5] S. Mathis, P. Couratier, A. Julian, J.M. Vallat, P. Corcia, G. Le Masson,
[6] A. Hubers, A.C. Ludolph, A. Rosenbohm, E.H. Pinkhardt, J.H. Weishaupt, J. Dorst, Amyotrophe Lateralsklerose. Eine Multisystemdegeneration, Nervenarzt 87 (2016) 179–188.

[7] J. Crueuilhier, Sur la paralysie musculaire progressive atrophique, Arch. Gen. Med. 1 (1853) 561–603.

[8] C.B. Radcliffe, J.A. Lockhart Clarke, An important case of paralysis and muscular atrophy with disease of the nervous centres, Brit. Foreign. Med. Chir. Rev. 30 (1862) 215–225.

[9] J.M. Charcot, De la sclérose latérale amyotrophique, Progr. Med 2 (1874) 41–42 325–327; 453–455.

[10] A. Arnold, D.C. Edgren, V.S. Palladino, Amyotrophic lateral sclerosis; fifty cases observed on Guam, J. Nerv. Ment. Dis. 117 (1953) 135–139.

[11] L.T. Kurland, D.W. Mulder, Epidemiologic investigations of amyotrophic lateral sclerosis. 2. Familial aggregations indicative of dominant inheritance. I, Neurology 5 (1955) 182–196.

[12] L.T. Kurland, D.W. Mulder, Epidemiologic investigations of amyotrophic lateral sclerosis. 2. Familial aggregations indicative of dominant inheritance. II, Neurology 5 (1955) 249–268.

[13] W.F. Haberlandt, Amyotrophische Lateralsklerose. Klinisch-pathologische und genetisch-demographische Studie, Gustav Fisher Verlag, Stuttgart, 1964.

[14] M. Bonduelle, P. Bouyges, G. Lorneau, J. Keller, Etude clinique et évolutive de 125 cas de sclérose latérale amyotrophique, Presse Med. 78 (1970) 827–832.

[15] H. Husquinet, G. Franck, Hereditary amyotrophic lateral sclerosis transmitted for five generations, Clin. Genet. 18 (1980) 109–115.

[16] W.K. Engel, L.T. Kurland, I. Klatzo, An inherited disease similar to amyotrophic lateral sclerosis with a pattern of posterior column involvement. An intermediate form? Brain 82 (1959) 203–220.

[17] S. Gimenez-Roldan, A. Esteban, Prognosis in hereditary amyotrophic lateral sclerosis, Arch. Neurol. 34 (1977) 706–708.

[18] G. Wohlfart, I. Gamstorp, Chronic progressive disorders of the lower motor neuron, in: R.D. Adams, L.M. Eaton, G.M. Shy (Eds.), Neuromuscular Disorders (the Motor Unit and its Disorders), The William & Wilkins Company, Baltimore, 1960, pp. 659–696.

[19] B. Brownell, D.R. Oppenheimer, J.T. Hughes, The central nervous system in motor neurone disease, J. Neurol. Neurosurg. Psychiatry 33 (1970) 338–357.

[20] J. Tanaka, H. Nakamura, Y. Tabuchi, K. Takahashi, Familial amyotrophic lateral sclerosis: features of multisystem degeneration, Acta Neuropathol. 64 (1984) 22–29.

[21] C.W. Metcalf, A. Hirano, Amyotrophic lateral sclerosis. Clinicopathological studies of a family, Arch. Neurol. 24 (1971) 518–523.

[22] O. Dornblüth, Anatomische Untersuchung eines Falles von amyotrophischer Laterosklerose, Neurol. Centralblatt. 8 (1889) 377–386.

[23] Y. Mitsuyama, S. Takamiya, Presenile dementia with motor neuron disease in Japan. A new entity? Arch. Neurol. 36 (1979) 592–593.

[24] J. Wikstrom, A. Paetau, J. Palo, R. Sulkava, M. Haltia, Classic amyotrophic lateral sclerosis with dementia, Arch. Neurol. 39 (1982) 681–683.

[25] B. Moulard, W. Camu, A. Malafosse, M. Billiard, M. Baldy-Moulinier, Etude clinique des formes familiales de sclérose latérale amyotrophique. Revue de la littérature, Rev. Neurol. (Paris) 153 (1997) 314–324.

[26] W.A. Horton, R. Eldridge, J.A. Brody, Familial motor neuron disease. Evidence for at least three different types, Neurology 26 (1976) 460–465.

[27] W. Camu, J. Khors, B. Moulard, F. Salachas, V. Briolotti, G.A. Rouleau, et al., Genetics of familial ALS and consequences for diagnosis. French ALS Research Group, J. Neurol. Sci. 165 (Suppl. 1) (1999) 21–26.

[28] S. Byrne, P. Bede, M. Elamin, K. Kenna, C. Lynch, R. McLaughlin, et al., Proposed criteria for familial amyotrophic lateral sclerosis, Amyotroph. Lateral Scler. 12 (2011) 157–159.

[29] A. Vajda, R.L. McLaughlin, M. Heverin, O. Thorpe, S. Abrahams, A. Al-Chalabi, et al., Genetic testing in ALS: a survey of current practices, Neurology. 88 (2017) 991–999.

[30] D.R. Rosen, Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis, Nature. 364 (1993) 362.

[31] A.E. Renton, A. Chio, B.J. Traynor, State of play in amyotrophic lateral sclerosis genetics, Nat. Neurosci. 17 (2014) 17–23.

[32] M. DeJesus-Hernandez, I.R. Mackenzie, B.F. Boeve, A.L. Boxer, M. Baker, N.J. Rutherford, et al., Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS, Neuron. 72 (2011) 245–256.

[33] A.E. Renton, E. Majounie, A. Waite, J. Simon-Sanchez, S. Rollinson, J.R. Gibbs, et al., A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD, Neuron. 72 (2011) 257–268.

[34] Z.Y. Zou, Z.R. Zhou, C.H. Che, C.Y. Liu, R.L. He, H.P. Huang, Genetic epidemiology of amyotrophic lateral sclerosis: a systematic review and meta-analysis, J. Neurol. Neurosurg. Psychiatry 88 (2017) 540–549.

[35] M.A. van Es, C. Dahlberg, A. Birve, J.H. Veldink, L.H. van den Berg, P.M. Andersen, Large-scale SOD1 mutation screening provides evidence for genetic heterogeneity in amyotrophic lateral sclerosis, J. Neurol. Neurosurg. Psychiatry 81 (2010) 562–566.

[36] M. Zufiria, F.J. Gil-Bea, R. Fernandez-Torron, J.J. Poza, J.L. Munoz-Blanco, R. Rojas-Garcia, et al., ALS: a bucket of genes, environment, metabolism and unknown ingredients, Prog. Neurobiol. 142 (2016) 104–129.

[37] M. van Blitterswijk, M.A. van Es, E.A. Hennekam, D. Doijes, W. van Rheenen, J. Medic, et al., Evidence for an oligogenic basis of amyotrophic lateral sclerosis, Hum. Mol. Genet. 21 (2012) 3776–3784.

[38] K.P. Kenna, R.L. McLaughlin, S. Byrne, M. Elamin, M. Heverin, E.M. Kenny, et al., Delineating the genetic heterogeneity of ALS using targeted high-throughput sequencing, J. Med. Genet. 50 (2013) 776–783.

[39] J.S. Valentine, P.A. Doucette, S. Zittin Potter, Copper-zinc superoxide dismutase and amyotrophic lateral sclerosis, Annu. Rev. Biochem. 74 (2005) 563–593.

[40] J. Azadmanesh, G.E.O. Borgstahl, A review of the catalytic mechanism of human manganese superoxide dismutase, Antioxidants. (Basel) 7 (2018) pii E25.

[41] P.M. Andersen, P. Nilsson, M.L. Keranen, L. Forsgren, J. Hagglund, M. Karlsborg, et al., Phenotypic heterogeneity in motor neuron disease patients with CuZn-superoxide dismutase mutations in Scandinavia, Brain. 120 (1997) 1723–1737.

[42] P. Corcia, P. Valdmanis, S. Millecamps, C. Lionnet, H. Blasco, K. Mouzat, et al., Phenotype and genotype analysis in amyotrophic lateral sclerosis with TARDBP gene mutations, Neurology. 78 (2012) 1519–1526.

[43] Y. Hayashi, K. Homma, H. Ichijo, SOD1 in neurotoxicity and its controversial roles in SOD1 mutation-negative ALS, Adv. Biol. Regul. 60 (2016) 95–104.

[44] T. Fujisawa, K. Homma, N. Yamaguchi, H. Kadowaki, N. Tsuburaya, I. Naguro, et al., A novel monoclonal antibody reveals a conformational alteration shared by amyotrophic lateral sclerosis-linked SOD1 mutants, Ann. Neurol. 72 (2012) 739–749.

[45] G. Bensimon, L. Lacomblez, V. Meininger, A controlled trial of riluzole in amyotrophic lateral sclerosis. ALS/Riluzole Study Group, N. Engl. J. Med. 330 (1994) 585–591.

[46] K. Li, T.J. Hala, S. Seetharam, D.J. Poulsen, M.C. Wright, A.C. Lepore, GLT1 overexpression in SOD1(G93A) mouse cervical spinal cord does not preserve diaphragm function or extend disease, Neurobiol. Dis. 78 (2015) 12–23.

[47] L. Morel, M. Regan, H. Higashimori, S.K. Ng, C. Esau, S. Videnisky, et al., Neuronal exosomal miRNA-dependent translational regulation of astroglial glutamate transporter GLT1, J. Biol. Chem. 288 (2013) 7105–7116.

[48] E. Miquel, A. Cassina, L. Martinez-Palma, C. Bolatto, E. Trias, M. Gandelman, et al., Modulation of astrocytic mitochondrial function by dichloroacetate improves survival and motor performance in inherited amyotrophic lateral sclerosis, PLoS One 7 (2012) e34776.

[49] M. Szelechowski, N. Ameodo, E. Obre, C. Léger, L. Allard, M. Bonneu, et al., Metabolic reprogramming in amyotrophic lateral sclerosis, Sci. Rep. 8 (2018) 3953.

[50] G. Le Masson, S. Przedborski, L.F. Abbott, A computational model of motor neuron degeneration, Neuron. 83 (2014) 975–988.

[51] N. Lamanauskas, A. Nistri, Riluzole blocks persistent Na+ and Ca2+ currents and modulates release of glutamate via presynaptic NMDA receptors on neonatal rat hypoglossal motoneurons in vitro, Eur. J. Neurosci. 27 (2008) 2501–2514.

[52] G.A. Morfini, D.A. Bosco, H. Brown, R. Gatto, A. Kaminska, Y. Song, et al., Inhibition of fast axonal transport by pathogenic SOD1 involves activation of p38 MAP kinase, PLoS One 8 (2013) e65235.

[53] J.I. Ayers, S.E. Fromholt, V.M. O'Neal, J.H. Diamond, D.R. Borchelt, Prion-like propagation of mutant SOD1 misfolding and motor neuron disease spread along neuroanatomical pathways, Acta Neuropathol. 131 (2016) 103–114.

[54] S.B. Prusiner, Novel proteinaceous infectious particles cause scrapie, Science. 216 (1982) 136–144.

[55] M. Polymenidou, D.W. Cleveland, The seeds of neurodegeneration: prion-like spreading in ALS, Cell. 147 (2011) 498–508.

[56] C. Vance, A. Al-Chalabi, D. Ruddy, B.N. Smith, X. Hu, J. Sreedharan, et al., Familial amyotrophic lateral sclerosis with frontotemporal dementia is linked to a locus on chromosome 9p13.2–21.3, Brain 129 (2006) 868–876.

[57] M. Morita, A. Al-Chalabi, P.M. Andersen, B. Hosler, P. Sapp, E. Englund, et al., A locus on chromosome 9p confers susceptibility to ALS and frontotemporal dementia, Neurology. 66 (2006) 839–844.

[58] E. Majounie, A.E. Renton, K. Mok, E.G. Dopper, A. Waite, S. Rollinson, et al., Frequency of the C9orf72 hexanucleotide repeat expansion in patients with amyotrophic lateral sclerosis and frontotemporal dementia: a cross-sectional study, Lancet Neurol. 11 (2012) 323–330.

[59] B.N. Smith, S. Newhouse, A. Shatunov, C. Vance, S. Topp, L. Johnson, et al., The C9ORF72 expansion mutation is a common cause of ALS+/−FTD in Europe and has a single founder, Eur. J. Hum. Genet. 21 (2013) 102–108.

[60] B.F. Boeve, K.B. Boylan, N.R. Graff-Radford, M. DeJesus-Hernandez, D.S. Knopman, O. Pedraza, et al., Characterization of frontotemporal dementia and/or amyotrophic lateral sclerosis associated with the GGGGCC repeat expansion in C9ORF72, Brain. 135 (2012) 765–783.

[61] A. Bertrand, J. Wen, D. Rinaldi, M. Houot, S. Sayah, A. Camuzat, et al., Early cognitive, structural, and microstructural changes in presymptomatic C9orf72 carriers younger than 40 years, JAMA. Neurol. 75 (2017) 236–245.

[62] M. Koppers, A.M. Blokhuis, H.J. Westeneng, M.L. Terpstra, C.A. Zundel, R. Vieira de Sa, et al., C9orf72 ablation in mice does not cause motor neuron degeneration or motor deficits, Ann. Neurol. 78 (2015) 426–438.

[63] Y.B. Lee, H.J. Chen, J.N. Peres, J. Gomez-Deza, J. Attig, M. Stalekar, et al., Hexanucleotide repeats in ALS/FTD form length-dependent RNA foci, sequester RNA binding proteins, and are neurotoxic, Cell Rep. 5 (2013) 1178–1186.

[64] Y.B. Lee, P. Baskaran, J. Gomez-Deza, H.J. Chen, A.L. Nishimura, B.N. Smith, et al., C9orf72 poly GA RAN-translated protein plays a key role in amyotrophic lateral sclerosis via aggregation and toxicity, Hum. Mol. Genet. 26 (2017) 4765–4777.

[65] S. Prpar Mihevc, S. Darovic, A. Kovanda, A. Bajc Cesnik, V. Zupunski, B. Rogelj, Nuclear trafficking in amyotrophic lateral sclerosis and frontotemporal lobar degeneration, Brain. 140 (2017) 13–26.

[66] B.T. Selvaraj, M.R. Livesey, C. Zhao, J.M. Gregory, O.T. James, E.M. Cleary, et al., C9ORF72 repeat expansion causes vulnerability of motor neurons to Ca(2+)-permeable AMPA receptor-mediated excitotoxicity, Nat. Commun. 9 (2018) 347.

[67] Y. Shi, S. Lin, K.A. Staats, Y. Li, W.H. Chang, S.T. Hung, et al., Haploinsufficiency leads to neurodegeneration in C9ORF72 ALS/FTD human induced motor neurons, Nat. Med. 24 (2018) 313–325.

[68] M.S. Forman, J.Q. Trojanowski, V.M. Lee, Neurodegenerative diseases: a decade of discoveries paves the way for therapeutic breakthroughs, Nat. Med. 10 (2004) 1055–1063.

[69] H. Braak, J. Brettschneider, A.C. Ludolph, V.M. Lee, J.Q. Trojanowski, K. Del Tredici, Amyotrophic lateral sclerosis—a model of corticofugal axonal spread, Nat. Rev. Neurol. 9 (2013) 708–714.

[70] J. Brettschneider, K. Del Tredici, J.B. Toledo, J.L. Robinson, D.J. Irwin, M. Grossman, et al., Stages of pTDP-43 pathology in amyotrophic lateral sclerosis, Ann. Neurol. 74 (2013) 20–38.

[71] R. Schmidt, M.A. de Reus, L.H. Scholtens, L.H. van den Berg, M.P. van den Heuvel, Simulating disease propagation across white matter connectome reveals anatomical substrate for neuropathology staging in amyotrophic lateral sclerosis, Neuroimage. 124 (2016) 762–769.

[72] R.H. Tan, Y. Yang, W.S. Kim, C. Dobson-Stone, J.B. Kwok, M.C. Kiernan, et al., Distinct TDP-43 inclusion morphologies in frontotemporal lobar degeneration with and without amyotrophic lateral sclerosis, Acta. Neuropathol. Commun 5 (2017) 76.

[73] M.A. Gitcho, R.H. Baloh, S. Chakraverty, K. Mayo, J.B. Norton, D. Levitch, et al., TDP-43 A315T mutation in familial motor neuron disease, Ann. Neurol. 63 (2008) 535–538.

[74] J. Sreedharan, I.P. Blair, V.B. Tripathi, X. Hu, C. Vance, B. Rogelj, et al., TDP-43 mutations in familial and sporadic amyotrophic lateral sclerosis, Science. 319 (2008) 1668–1672.

[75] E. Kabashi, P.N. Valdmanis, P. Dion, D. Spiegelman, B.J. McConkey, C. Vande Velde, et al., TARDBP mutations in individuals with sporadic and familial amyotrophic lateral sclerosis, Nat. Genet. 40 (2008) 572–574.

[76] I.R. Mackenzie, R. Rademakers, M. Neumann, TDP-43 and FUS in amyotrophic lateral sclerosis and frontotemporal dementia, Lancet Neurol. 9 (2010) 995–1007.

[77] F. Kametani, T. Obi, T. Shishido, H. Akatsu, S. Murayama, Y. Saito, et al., Mass spectrometric analysis of accumulated TDP-43 in amyotrophic lateral sclerosis brains, Sci. Rep. 6 (2016) 23281.

[78] C.C. Chou, Y. Zhang, M.E. Umoh, S.W. Vaughan, I. Lorenzini, F. Liu, et al., TDP-43 pathology disrupts nuclear pore complexes and nucleocytoplasmic transport in ALS/FTD, Nat. Neurosci. 21 (2018) 228–239.

[79] E.M. Tank, C. Figueroa-Romero, L.M. Hinder, K. Bedi, H.C. Archbold, X. Li, et al., Abnormal RNA stability in amyotrophic lateral sclerosis, Nat. Commun. 9 (2018) 2845.

[80] A. Crozat, P. Aman, N. Mandahl, D. Ron, Fusion of CHOP to a novel RNA-binding protein in human myxoid liposarcoma, Nature. 363 (1993) 640–644.

[81] C. Vance, B. Rogelj, T. Hortobagyi, K.J. De Vos, A.L. Nishimura, J. Sreedharan, et al., Mutations in FUS, an RNA processing protein, cause familial amyotrophic lateral sclerosis type 6, Science. 323 (2009) 1208–1211.

[82] T.J. Kwiatkowski Jr., D.A. Bosco, A.L. Leclerc, E. Tamrazian, C.R. Vanderburg, C. Russ, et al., Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis, Science. 323 (2009) 1205–1208.

[83] M. Neumann, R. Rademakers, S. Roeber, M. Baker, H.A. Kretzschmar, I.R. Mackenzie, A new subtype of frontotemporal lobar degeneration with FUS pathology, Brain. 132 (2009) 2922–2931.

[84] D.A. Bosco, N. Lemay, H.K. Ko, H. Zhou, C. Burke, T.J. Kwiatkowski Jr.et al., Mutant FUS proteins that cause amyotrophic lateral sclerosis incorporate into stress granules, Hum. Mol. Genet. 19 (2010) 4160–4175.

[85] Y.R. Li, O.D. King, J. Shorter, A.D. Gitler, Stress granules as crucibles of ALS pathogenesis, J. Cell Biol. 201 (2013) 361–372.

[86] P. Aman, I. Panagopoulos, C. Lassen, T. Fioretos, M. Mencinger, H. Toresson, et al., Expression patterns of the human sarcoma-associated genes FUS and EWS and the genomic structure of FUS, Genomics. 37 (1996) 1–8.

[87] J. Gal, J. Zhang, D.M. Kwinter, J. Zhai, H. Jia, J. Jia, et al., Nuclear localization sequence of FUS and induction of stress granules by ALS mutants, Neurobiol. Aging 32 (2011) e27–e40.

[88] L. Yang, J. Gal, J. Chen, H. Zhu, Self-assembled FUS binds active chromatin and regulates gene transcription, Proc. Natl. Acad. Sci. U. S. A. 111 (2014) 17809–17814.

[89] R. Fujii, S. Okabe, T. Urushido, K. Inoue, A. Yoshimura, T. Tachibana, et al., The RNA binding protein TLS is translocated to dendritic spines by mGluR5 activation and regulates spine morphology, Curr. Biol. 15 (2005) 587–593.

[90] R. Fujii, T. Takumi, TLS facilitates transport of mRNA encoding an actin-stabilizing protein to dendritic spines, J. Cell Sci. 118 (2005) 5755–5765.

[91] S. Dini Modigliani, M. Morlando, L. Errichelli, M. Sabatelli, I. Bozzoni, An ALS-associated mutation in the FUS 3'-UTR disrupts a microRNA-FUS regulatory circuitry, Nat. Commun. 5 (2014) 4335.

[92] S.K. Dhar, J. Zhang, J. Gal, Y. Xu, L. Miao, B.C. Lynn, et al., Fused in sarcoma is a novel regulator of manganese superoxide dismutase gene transcription, Antioxid. Redox Signal. 20 (2014) 1550–1566.

[93] W.Y. Wang, L. Pan, S.C. Su, E.J. Quinn, M. Sasaki, J.C. Jimenez, et al., Interaction

[94] A.S. Mastrocola, S.H. Kim, A.T. Trinh, L.A. Rodenkirch, R.S. Tibbetts, The RNA-binding protein fused in sarcoma (FUS) functions downstream of poly(ADP-ribose) polymerase (PARP) in response to DNA damage, J. Biol. Chem. 288 (2013) 24731–24741.

[95] D.S. Dichmann, R.M. Harland, Fus/TLS orchestrates splicing of developmental regulators during gastrulation, Genes Dev. 26 (2012) 1351–1363.

[96] N. Finkel, A forma pseudomiopática tardia da atrofia muscular progressiva hereditária familiar, Arq. Neuropsiquiatr. 20 (1962) 307–322.

[97] A. Richieri-Costa, A. Rogatko, R. Levisky, N. Finkel, O. Frota-Pessoa, Autosomal dominant late adult spinal muscular atrophy, type Finkel, Am. J. Med. Genet. 9 (1981) 119–128.

[98] A.L. Nishimura, M. Mitne-Neto, H.C. Silva, A. Richieri-Costa, S. Middleton, D. Cascio, et al., A mutation in the vesicle-trafficking protein VAPB causes late-onset spinal muscular atrophy and amyotrophic lateral sclerosis, Am. J. Hum. Genet. 75 (2004) 822–831.

[99] J.O. Johnson, J. Mandrioli, M. Benatar, Y. Abramzon, V.M. Van Deerlin, J.Q. Trojanowski, et al., Exome sequencing reveals VCP mutations as a cause of familial ALS, Neuron. 68 (2010) 857–864.

[100] M.A. van Es, H.J. Schelhaas, P.W. van Vught, N. Ticozzi, P.M. Andersen, E.J. Groen, et al., Angiogenin variants in Parkinson disease and amyotrophic lateral sclerosis, Ann. Neurol. 70 (2011) 964–973.

[101] Y.Z. Chen, C.L. Bennett, H.M. Huynh, I.P. Blair, I. Puls, J. Irobi, et al., DNA/RNA helicase gene mutations in a form of juvenile amyotrophic lateral sclerosis (ALS4), Am. J. Hum. Genet. 74 (2004) 1128–1135.

[102] M. Helal, N. Mazaheri, B. Shalbafan, R.A. Malamiri, N. Dilaver, R. Buchert, et al., Clinical presentation and natural history of infantile-onset ascending spastic paralysis from three families with an ALS2 founder variant, Neurol. Sci. 39 (2018) 1917–1925.

[103] R. Cui, M. Tuo, P. Li, C. Zhou, Association between TBK1 mutations and risk of amyotrophic lateral sclerosis/frontotemporal dementia spectrum: a meta-analysis, Neurol. Sci. 39 (2018) 811–820.

[104] G. Imbert, F. Saudou, G. Yvert, D. Devys, Y. Trottier, J.M. Garnier, et al., Cloning of the gene for spinocerebellar ataxia 2 reveals a locus with high sensitivity to expanded CAG/glutamine repeats, Nat. Genet. 14 (1996) 285–291.

[105] K. Gwinn-Hardy, J.Y. Chen, H.C. Liu, T.Y. Liu, M. Boss, W. Seltzer, et al., Spinocerebellar ataxia type 2 with parkinsonism in ethnic Chinese, Neurology. 55 (2000) 800–805.

[106] A.C. Elden, H.J. Kim, M.P. Hart, A.S. Chen-Plotkin, B.S. Johnson, X. Fang, et al., Ataxin-2 intermediate-length polyglutamine expansions are associated with increased risk for ALS, Nature. 466 (2010) 1069–1075.

[107] F.L. Conforti, R. Spataro, W. Sproviero, R. Mazzei, F. Condino, et al., Ataxin-1 and ataxin-2 intermediate-length PolyQ expansions in amyotrophic lateral sclerosis, Neurology. 79 (2012) 2315–2320.

[108] D.P. Huynh, M.R. Del Bigio, D.H. Ho, S.M. Pulst, Expression of ataxin-2 in brains from normal individuals and patients with Alzheimer's disease and spinocerebellar ataxia 2, Ann. Neurol. 45 (1999) 232–241.

[109] I. Al-Ramahi, A.M. Perez, J. Lim, M. Zhang, R. Sorensen, M. de Haro, et al., dAtaxin-2 mediates expanded Ataxin-1-induced neurodegeneration in a Drosophila model of SCA1, PLoS Genet. 3 (2007) e234.

[110] M.P. Hart, A.D. Gitler, ALS-associated ataxin 2 polyQ expansions enhance stress-induced caspase 3 activation and increase TDP-43 pathological modifications, J. Neurosci. 32 (2012) 9133–9142.

[111] D. Recabarren-Leiva, M. Alarcon, New insights into the gene expression associated to amyotrophic lateral sclerosis, Life Sci. 193 (2018) 110–123.

[112] S. Morgan, A. Shatunov, W. Sproviero, A.R. Jones, M. Shoai, D. Hughes, et al., A comprehensive analysis of rare genetic variation in amyotrophic lateral sclerosis in the UK, Brain. 140 (2017) 1611–1618.

[113] J.M. Vallat, C. Goizet, L. Magy, S. Mathis, Too many numbers and complexity: time to update the classifications of neurogenetic disorders? J. Med. Genet. 43 (2016) 647–650.

[114] J.M. Vallat, C. Goizet, M. Tazir, P. Couratier, L. Magy, S. Mathis, Classifications of neurogenetic diseases: an increasingly complex problem, Rev. Neurol. (Paris) 172 (2016) 339–349.

[115] S. Mathis, C. Goizet, M. Tazir, C. Magdelaine, A.S. Lia, L. Magy, et al., Charcot-Marie-tooth diseases: an update and some new proposals for the classification, J. Med. Genet. 52 (2015) 681–690.

[116] S. Mathis, C. Goizet, M. Tazir, L. Magy, J.M. Vallat, Reasons Charcot-Marie-tooth disease due to mutations in the MME gene should not be named AR-CMT2T, Ann. Neurol. 80 (2016) 477.

[117] L. Magy, S. Mathis, G. Le Masson, C. Goizet, M. Tazir, J.M. Vallat, Updating the classification of inherited neuropathies. Results of an international survey, Neurology 90 (2018) e870–e876.

[118] C.J. Klein, Inherited neuropathy precision classification: What's in a name? Neurology. 90 (2018) e870–e876.

[119] S. Mathis, M. Tazir, L. Magy, F. Duval, G. Le Masson, M. Duchesne, et al., History and current difficulties in classifying inherited myopathies and muscular dystrophies, J. Neurol. Sci. 384 (2018) 50–54.

[120] B.E. Deverman, B.M. Ravina, K.S. Bankiewicz, S.M. Paul, D.W.Y. Sah, Gene therapy for neurological disorders: progress and prospects, Nat. Rev. Drug Discov. 17 (2018) 767.

[121] S. Mathis, G. Le Masson, RNA-targeted therapies and amyotrophic lateral sclerosis,

S. Mathis, *et al.*

[122] X.M. Anguela, K.A. High, Entering the modern era of gene therapy, Annu. Rev. Med. 70 (2018) 273–288.

[123] T.M. Miller, A. Pestronk, W. David, J. Rothstein, E. Simpson, S.H. Appel, et al., An antisense oligonucleotide against SOD1 delivered intrathecally for patients with SOD1 familial amyotrophic lateral sclerosis: a phase 1, randomised, first-in-man study, Lancet Neurol. 12 (2013) 435–442.

[124] C.V. Ly, T.M. Miller, Emerging antisense oligonucleotide and viral therapies for amyotrophic lateral sclerosis, Curr. Opin. Neurol. 31 (2018) 648–654.

[125] L. Wang, F. Yi, L. Fu, J. Yang, S. Wang, Z. Wang, et al., CRISPR/Cas9-mediated targeted gene correction in amyotrophic lateral sclerosis patient iPSCs, Protein.

[126] J. Jiang, Q. Zhu, T.F. Gendron, S. Saberi, M. McAlonis-Downes, A. Seelman, et al., Gain of toxicity from ALS/FTD-linked repeat expansions in C9ORF72 is alleviated by antisense oligonucleotides targeting GGGGCC-containing RNAs, Neuron 90 (2016) 535–550.

[127] L.A. Becker, B. Huang, G. Bieri, R. Ma, D.A. Knowles, P. Jafar-Nejad, et al., Therapeutic reduction of ataxin-2 extends lifespan and reduces pathology in TDP-43 mice, Nature. 544 (2017) 367–371.

[128] A.E. Frakes, L. Braun, L. Ferraiuolo, D.C. Guttridge, B.K. Kaspar, Additive amelioration of ALS by co-targeting independent pathogenic mechanisms, Ann. Clin. Transl. Neurol. 4 (2017) 76–86.
