
Virologica Sinica
https://doi.org/10.1007/s12250-019-00095-3

---

**REVIEW**

---

**Multiple Roles of HIV-1 Capsid during the Virus Replication Cycle**

Mariia Novikova${}^{1}$ · Yulan Zhang${}^{2}$ · Eric O. Freed${}^{1}$ ${}^{\mathrm{id}}$ · Ke Peng${}^{2}$ ${}^{\mathrm{ID}}$

Received: 17 November 2018 / Accepted: 16 January 2019  
© The Author(s) 2019

---

**Abstract**

Human immunodeficiency virus-1 capsid (HIV-1 CA) is involved in different stages of the viral replication cycle. During virion assembly, CA drives the formation of the hexameric lattice in immature viral particles, while in mature virions CA monomers assemble in cone-shaped cores surrounding the viral RNA genome and associated proteins. In addition to its functions in late stages of the viral replication cycle, CA plays key roles in a number of processes during early phases of HIV-1 infection including trafficking, uncoating, recognition by host cellular proteins and nuclear import of the viral pre-integration complex. As a result of efficient cooperation of CA with other viral and cellular proteins, integration of the viral genetic material into the host genome, which is an essential step for productive viral infection, successfully occurs. In this review, we will summarize available data on CA functions in HIV-1 replication, describing in detail its roles in late and early phases of the viral replication cycle.

---

**Keywords** Human immunodeficiency virus-1 (HIV-1) · Capsid (CA) · Assembly · Post entry · Uncoating and nuclear import · Inhibitor

---

**Introduction**

Assembly and release of HIV-1 virions occur predominantly at the plasma membrane of infected cells (Sundquist and Krausslich 2012; Freed 2015). The major structural viral protein, Gag, is essential and sufficient for the formation of virus-like particles (VLPs). Gag is synthesized in the cytosol as a polyprotein composed of four domains—matrix (MA), capsid (CA), nucleocapsid (NC) and p6—and two short peptides, SP1 and SP2 (Fig. 1A). An N-terminal myristyl group, and a cluster of basic residues in MA, that binds the phospholipid PI(4,5)P₂, appear to be the main determinants of Gag targeting and binding to the inner leaflet of the plasma membrane (Ono et al. 2004; Tang et al. 2004; Saad et al. 2006). CA–CA interactions play a central role in driving the formation of a hexameric Gag lattice that contains gaps in the released virion to form a spherical shape (Wright et al. 2007; Carlson et al. 2008; Briggs et al. 2009). NC is responsible for packaging of the viral genomic RNA, and p6 is required for budding of newly assembled virions. For conversion into a fully infectious particle, incorporation of the Gag-Pol polyprotein comprising three viral enzymes—reverse transcriptase (RT), integrase (IN) and protease (PR)—is needed. Concomitant with or shortly after budding of the immature virion, PR cleaves the Gag precursor molecules into the distinct proteins thus initiating the maturation process (Fig. 1B). These PR-driven proteolysis events result in significant structural rearrangements of the virion interior, including the assembly of the fullerene-like, cone-shaped core, which is essential for productive viral infection. The core is composed of the mature capsid, a protein shell formed by ~250 CA hexamers and 12 CA pentamers that close off both ends of the structure, and the viral ribonucleoprotein complex (Ganser et al. 1999; Pornillos et al. 2011) (Fig. 1C). In the cytoplasm of a newly infected cell the reverse transcription of the viral genomic RNA results in the formation of a double stranded viral DNA that is

---

Mariai Novikova and Yulan Zhang have contributed equally to this work.

Eric O. Freed  
efreed@nih.gov  

Ke Peng  
pengke@wh.iov.cn  

${}^{1}$ Virus-Cell Interaction Section, HIV Dynamics and Replication Program, Center for Cancer Research, National Cancer Institute, Frederick, MD 21702, USA  

${}^{2}$ State Key Laboratory of Virology, Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan 430071, China

---

Published online: 26 April 2019

Fig. 1 Structure of the HIV-1 virion and its components. A HIV-1 Gag domain organization. The HIV-1 Gag consists of four domains—matrix (MA), capsid (CA), nucleocapsid (NC) and p6—and two short peptides, SP1 and SP2, that are cleaved into distinct proteins by the viral protease during maturation. B Immature and mature HIV-1 virions. In the immature virion (left) Gag molecules are radially organized in the hexameric lattice (Gag domains are shown in the same colors as in A). Two molecules of viral genomic RNA (magenta) per virion are packaged. Trimers of Env protein are embedded in the viral membrane. Gag-Pol molecules that produce viral enzymes—protease (PR) (purple), reverse transcriptase (RT)

(orange) and integrase (IN) (brown)—are present in 1:20 ratio to Gag. In the mature virion (right), MA retains bound to the viral membrane, released CA forms a characteristic cone-shaped core, NC is bound to the viral genomic RNA (reprinted from Freed 2015). Electron microscopy images of the immature (left) and mature (right) virions are shown (Novikova and Freed, unpublished data). Scale bar, 100 nm. C Structure of the mature capsid (Pornillos et al. 2011), reprinted with permission from Nature Publishing Group. Capsid is formed by a hexameric CA lattice (CA-NTDs in orange, CA-CTDs in blue) with 12 embedded CA pentamers (yellow) allowing for the closing off of both ends of the conical structure.

Role of CA in HIV-1 Assembly and Maturation

Structure of the CA Monomer

As mentioned above, CA is critical for assembly of both immature and mature viral particles. HIV-1 CA is an α-helical protein composed by two domains—an N-terminal domain (CA-NTD) and a C-terminal domain (CA-CTD)—connected by a short linker (Fig. 2A). CA-NTD consists of seven α-helices and a characteristic extended cyclophilin A (CypA)-binding loop (Gamble et al. 1996; Gitti et al. 1996). In the immature Gag lattice, the N-terminal end of CA-NTD is unstructured and linked to the membrane-bound MA domain, while in the context of the mature CA monomer the proteolytically released N-terminus folds into a β-hairpin (von Schwedler et al. 1998). CA-CTD contains a short 3<sub>10</sub> helix, the major homology region (MHR)—a highly conserved element in all orthoretroviruses required

In the first part of this review we summarize the current data on the structural role of CA in assembly of HIV-1 virions. In addition, CA-binding inhibitors of late stages of the viral replication cycle will be reviewed. In the second part, we will focus on CA-mediated processes occurring after fusion of the HIV-1 virion with the target cell and before viral DNA integration in the host genome. Host restriction factors that counteract CA functions will be also briefly discussed.

M. Novikova et al.: Role of HIV-1 Capsid

A
B

CypA-binding loop
CA-NTD
CA-CTD

β-hairpin

C

six-helix bundle

Fig. 2 Arrangement of CA in immature and mature HIV-1 virions.

A Structure of the CA monomer. CA consists of two α-helical domains—CA-NTD and CA-CTD—that are connected by a short linker. The CypA-binding loop and β-hairpin are indicated. Helices of CA-NTD: helix 1 (ruby), helix 2 (cyan), helix 3 (orange), helix 4 (blue), helix 5 (magenta), helix 6 (light green) and helix 7 (yellow). Helices of CA-CTD: 3<sub>10</sub> helix (brown), helix 8 (wheat), helix 9 (green), helix 10 (purple) and helix 11 (red) [PDB ID: 5MCX (Mattei et al. 2016a)]. B Structural arrangement of the CA layer in intact immature (left) and mature (right) HIV-1 virions. CA-NTDs and CA-CTDs of two CA monomers from neighbouring hexamers are colored in blue and red, respectively. All other CA-NTDs and CA-CTDs are in cyan and in orange, respectively. Some important interfaces involved in the formation of the two structures are shown. Hexagons indicate a sixfold interface in the individual hexamer in both structures. In the immature lattice (left) homo-dimeric (ovals) and homo-trimeric (triangles) interfaces important for connecting neighboring hexamers are formed by helices 1 and 2 of CA-NTDs, respectively [PDB ID: 4USN (Schur et al. 2015)]. In the mature virion (right), inter-hexamer interactions are formed by helices 10 and 11 residing at threefold interfaces (triangles) and by residues from helix 9 at twofold interfaces (ovals) [PDB ID: 5MCX (Mattei et al. 2016a)]. C CA-CTD and the N-terminal seven residues of SP1 (side view—left; top view—right) in the six-helix bundle. Residues of SP1 (light green), CA-CTD (orange), MHR loop (red), loop connecting helices 9 and 10 (cyan), and a β-turn (yellow) are key elements for the formation of the CA-SP1 bundle in the immature Gag lattice [PDB ID: 5I4T (Wagner et al. 2016)]. The arrow indicates the cleavage site between CA and SP1.

for viral replication—and four α-helices (Gamble et al. 1997). Like the N-terminus of the CA-NTD, the C-terminal residues of the CA-CTD undergo significant structural rearrangements during maturation. In immature virions, the C-terminus of the CA domain is part of the CA-SP1 junction that forms a six-helix bundle (Fig. 2B, left; Fig. 2C), a critical structural element in the Gag hexamer (Wright et al. 2007). Upon proteolysis at the CA-SP1 junction, this region becomes disordered (Gamble et al. 1997). Although the structure of the mature CA monomer is highly conserved across retroviruses, the arrangement of CA subunits in the immature and mature lattices, and the shape of mature cores, vary significantly among retroviruses (Zhang et al. 2015; Mattei et al. 2016b).

The Role of CA in Assembly of the Immature HIV-1 Gag Lattice

Numerous structural and mutational studies have characterized the role of HIV-1 CA in the immature and mature Gag lattices and revealed substantial differences in CA–CA contacts required for assembly of the two structures (Fig. 2B, left and right, respectively) (Lingappa et al. 2014; Mattei et al. 2016b). In the immature lattice, each CA-NTD forms an extensive network of interactions with CA-NTDs from the same and neighbouring hexamers. Within hexamers, residues in helix 4 of each CA-NTD interact with residues between helices 5 and 6 in neighbouring CA-NTDs. Inter-hexameric contacts are mediated by helices 1 and 2, which form homo-dimeric and homo-trimeric interfaces, respectively (Schur

et al. 2015) (Fig. 2B, left). Recently, the short loop between helices 6 and 7, located at the threefold inter-hexamer interface, was shown to be important for assembly of the immature Gag lattice (Novikova et al. 2018). Interestingly, deletion of the entire CA-NTD only modestly decreases production of VLPs, although the resulting particles are more heterogeneous in size relative to the WT (Borsetti et al. 1998). This finding indicates a major role for the CA-CTD in assembly of immature virions. The CA-CTD, similarly to the CA-NTD, is involved in the formation of both intra- and inter-hexameric contacts. Helix 9 forms a homo-dimeric interface linking neighboring hexamers, while residues from the MHR and other regions are important for generating the Gag hexamer. In contrast to the mature capsid, there are no extensive intra-protemer CA-NTD-CA-CTD contacts (Schur et al. 2015). One of the distinctive features of the immature HIV-1 Gag lattice is the formation of goblet-like structures within individual hexamers in which the cup is formed by the CA-CTD and the stem represents a six-helix bundle formed by the CA-SP1 junction region, as determined by cryo-ET and X-ray crystallography (Schur et al. 2016; Wagner et al. 2016) (Fig. 2C). The six-helix bundle is formed by eight C-terminal residues of the CA-CTD and the N-terminal seven residues of SP1. Several MHR residues, the loop connecting helices 9 and 10 and a β-turn, formed by four residues downstream of helix 11, are required to maintain this structure and stabilize the hexamer (Wagner et al. 2016). Two recent studies (Dick et al. 2018; Mallery et al. 2018) demonstrated that the negatively charged small molecule inositol hexaphosphate (IP6) facilitates assembly of the six-helix bundle in the immature Gag lattice, and upon Gag proteolysis promotes the formation of the mature capsid. Two CA residues, Lys158 and Lys227, from the MHR and the CA-SP1 junction, respectively, are arranged in two positively charged rings that constitute the IP6 binding site in the Gag lattice (Dick et al. 2018; Mallery et al. 2018).

Apart from the structural role of the CA-SP1 six-helix bundle in Gag lattice formation, this element also modulates the maturation process by sequestering the CA-SP1 PR cleavage site. Two preceding cleavages between MA-CA and SP1-NC are essential to destabilize the structure of the six-helix bundle, thus exposing the CA-SP1 cleavage site (Fig. 2C) to PR to complete Gag processing (Schur et al. 2016; Wagner et al. 2016). The CA-SP1 bundle is inherently flexible and conformationally dynamic, allowing for a balance between the requirement for stability during assembly and flexibility to expose the CA-SP1 cleavage site during maturation (Schur et al. 2016; Wagner et al. 2016) and see below. A recent cryo-ET study using a panel of Gag cleavage site mutants provided data suggesting that destabilization of the CA-SP1 bundle is a key determinant in the process of structural maturation (Mattei et al. 2018).


The Role of CA in the Formation of the Mature Capsid

Upon completion of PR-mediated Gag processing, the newly liberated CA proteins assemble into the cone-shaped capsid within the virion. The mechanism of mature lattice formation is still incompletely understood. Most studies have supported a disassembly and *de novo* reassembly model in which the released CA monomers reassemble into the conical capsid (Briggs et al. 2006; Keller et al. 2013; Woodward et al. 2015). Some studies, however, support a displacive transition model, in which the immature CA lattice transforms into the mature capsid without free CA monomers being released into the virion interior (Meng et al. 2012; Frank et al. 2015). The formation of the core through a combination of both mechanisms has also been suggested (Ning et al. 2016).

The structures of CA hexamers and pentamers, and their arrangement in the mature lattice, have been determined in cryo-EM and X-ray crystallography studies using assembled mature CA tubes and WT or cross-linked CA hexameric lattices (Li et al. 2000; Pornillos et al. 2009, 2011; Zhao et al. 2013; Gres et al. 2015). Recently, the CA arrangement in the mature capsid within HIV-1 virions has been resolved by cryo-ET (Mattei et al. 2016a). The structure of hexamers in the intact core appears to be similar to one of the previously described structures (Gres et al. 2015). Intra-hexamer interactions are formed by CA-NTD–CA-NTD and CA-NTD–CA-CTD contacts between adjacent CA monomers. There is also a substantial cluster of intra-subunit CA-NTD–CA-CTD contacts in the mature lattice. At the sixfold symmetry axis interface of the mature hexamer, the residue Arg18 from helix 1 constitutes a selective channel for incoming nucleotides needed for reverse transcription (Jacques et al. 2016). The same Arg ring has been recently shown to be involved in coordination of IP6 molecules that promote assembly of CA hexamers and regulate capsid stability (Dick et al. 2018; Mallery et al. 2018). Hexamers are linked together by a number of contacts: several residues from the N-terminus of the CA-CTD, together with helix 9, form a twofold interface, whereas helices 10 and 11 are key structural elements at the threefold interface (Fig. 2B, right). The structure of the mature lattice is stabilized by abundantly present water molecules that modulate interactions between CA monomers (Gres et al. 2015). Small structural movements between two domains of the CA monomer, as well as between CA-CTDs at the two- and three-fold axis inter-hexameric interfaces in the mature capsid, provide twists and tilts necessary for variable curvature of the conical structure (Mattei et al. 2016a). In contrast to hexamers, cryo-ET analysis of CA pentamers in the mature
cores in intact virions (Mattei *et al.* 2016a) revealed significant differences in the arrangement of CA protomers compared to the previously described structure of cross-linked pentamers (Pornillos *et al.* 2011). Pentamers were found at sites of high curvature (Mattei *et al.* 2016a) suggesting that either angle of curvature determines where the 12 pentamers are placed, or pentamer positions determine the angle of curvature of the mature lattice. These cryo-ET results also revealed that pentamers expose different amino acid residues on the outer surface of the capsid relative to hexamers (Mattei *et al.* 2016a); these findings have potential implications for the interaction of capsids with host factors post-entry (see below).

### CA-Targeted Inhibitors of HIV-1 Assembly and Maturation

The diverse role of CA in HIV-1 assembly and maturation suggests that CA could be a good target for therapeutic intervention. Currently, none of the antiretroviral drugs in clinical use inhibit the late stages of the HIV-1 replication cycle by targeting Gag-mediated steps in assembly or maturation. However, a number of CA-targeted inhibitors that block virion assembly and/or maturation have been identified and characterized [reviewed in Tedbury and Freed (2015), Spearman (2016) and Carnes *et al.* (2018a)]. Some of CA-binding compounds are briefly described below. One of the most promising groups includes bevirimat (BVM), a betulinic acid-derived compound, and its derivatives, which block the proteolytic release of CA from CA-SP1 (Li *et al.* 2003; Zhou *et al.* 2004; Urano *et al.* 2016). BVM, the first-in-class maturation inhibitor, was suggested to bind inside the six-helix bundle (Schur *et al.* 2016; Wagner *et al.* 2016; Purdy *et al.* 2018), thereby stabilizing its structure and preventing CA-SP1 cleavage. Virions produced in the presence of BVM exhibit aberrant morphology characterized by spherical, acentric cores and an electron-dense layer under the viral membrane (Li *et al.* 2003) that is a stabilized remnant of the immature Gag lattice (Keller *et al.* 2011). Although BVM did not proceed beyond phase II clinical trials due to the presence of Gag polymorphisms that reduced compound efficacy in a significant number of patients, development of more potent BVM analogs and other compounds with similar mechanism of action has been proceeding (Nowicka-Sans *et al.* 2016; Urano *et al.* 2016). CA-SP1 processing can also be blocked by another small-molecule inhibitor, PF-46396 (Blair *et al.* 2009; Waki *et al.* 2012). Although this compound is structurally distinct from BVM, it likely acts via a similar mechanism, involving stabilization of the CA-SP1 six-helix bundle. The propagation of viruses in the presence PF-46396 led to the selection of resistant mutants, including several that also confer resistance to BVM (Waki *et al.* 2012). These results suggest that PF-46396 and BVM share a similar binding site (Blair *et al.* 2009; Waki *et al.* 2012). Selection experiments performed with BVM (Adamson *et al.* 2006) and PF-46396 (Waki *et al.* 2012) led to the emergence of resistant viruses that were dependent on the maturation inhibitor for their replication. These maturation inhibitor-dependent mutants off-set the stabilizing effect of maturation inhibitor binding by destabilizing the six-helix bundle. These destabilizing mutants were in turn off-set by acquisition of the SP1-T8I mutation, which, like maturation inhibitor binding, stabilized the six-helix bundle (Waki *et al.* 2012; Fontana *et al.* 2016). These studies revealed a subtle balance in the stability of the CA-SP1 bundle; this region of Gag needs to be sufficiently stable to enable assembly of the immature Gag lattice, but also sufficiently conformationally dynamic to allow exposure of the CA-SP1 cleavage site and PR-mediated cleavage at that site.

A set of CA inhibitors that bind to either the CA-NTD or the CA-CTD have also been described. CAP-1 is a small molecule that binds at the base of the CA-NTD and inhibits capsid assembly, most likely by disrupting intersubunit CA-NTD–CA-CTD interactions within the mature hexamer (Tang *et al.* 2003; Kelly *et al.* 2007). A relatively high concentration of the compound, ~100 μmol/L, is needed to reduce infectivity of produced viral particles by 95% in cell-based assays (Tang *et al.* 2003). The CAP-1-targeting pocket of the CA-NTD overlaps with the binding site of two other groups of compounds—benzodiazepines (BDs) and benzimidazoles (BMs). Both families inhibit the formation of mature virions, albeit by different mechanisms. While the BD compounds significantly inhibit virion release, and the produced virus particles exhibit morphological defects, the BM inhibitors prevent the formation of mature cores but only modestly affect viral production (Fader *et al.* 2011; Lemke *et al.* 2012). One of the developed BM-based compounds, compound 1 (C1), has also been shown to interfere with assembly of the mature capsid; this inhibitor binds to a unique site on the top of the CA-NTD, near the base of the CypA-binding loop (Goudreau *et al.* 2013; Lemke *et al.* 2013; Wang *et al.* 2017). A defect in formation of mature infectious virions was also observed upon treatment of HIV-infected cells with the small molecule PF74, which binding site is located at the intersubunit CA-NTD–CA-CTD interface in the mature hexamer (Blair *et al.* 2010; Bhattacharya *et al.* 2014; Price *et al.* 2014). This compound exhibits a dual antiviral activity, inhibiting the assembly of the mature core and early stages of viral infection. Recently, a compound GS-CA1, that acts similar to PF74 by inhibiting early and late stages of viral replication and occupies the same binding site as PF74, showed very promising data and is being tested in clinical development studies (Perrier *et al.* 2017; Tse *et al.* 2017). Another CA-targeted compound, a 12-mer α-helical
peptide CAI (capsid assembly inhibitor), binds to the hydrophobic cavity formed by CA helices 8, 9 and 11 located in the CA-CTD, thus altering the CA-CTD dimer interface, and inhibits immature- and mature-like particle assembly *in vitro* (Sticht *et al.* 2005; Ternois *et al.* 2005). Although CAI was effective in *in vitro* experiments it failed to demonstrate inhibitory activity in cell-based assays due to a low membrane permeability. To enhance its cell permeability, CAI was modified by hydrocarbon stapling (Bhat-tacharya *et al.* 2008; Zhang *et al.* 2008). One of such CAI derivatives, NYAD-1, was able to penetrate into cells and inhibit viral production and immature- and mature-like particle assembly in cell-based systems (Zhang *et al.* 2008). Another group of CA inhibitors includes modified 2-arylquinazoline compounds that were shown to target the same CA-CTD pocket as CAI *in vitro*, and inhibit viral replication at low micromolar concentrations (Machara *et al.* 2016). The most likely mechanism of action of these compounds is inhibition of viral particle assembly and formation of the mature core, although additional studies need to be performed to exclude an off-target effect. Given that many of the reported CA-binding inhibitors display weak antiviral activity or exhibit insensitivity to naturally occurring Gag polymorphs, studies on identification and characterization of compounds interfering with structural functions of CA is of great importance to develop and implement novel anti-HIV-1 therapeutic drugs.

demonstrated that a capsid-derived structure is associated with the viral RTC/PIC complexes and facilitates a number of early infection events in the cytoplasm and, potentially, also in the nucleus.

Discrepancies among different studies may derive from the ill-defined physical features of these complexes. A major complication that challenges the investigation of early post-entry events is that only a minority of the incoming viral particles undergoes productive infection and the large amount of abortive infection events may cause noise/background signals that compromise the final readout. Development of modern imaging strategies that can track replication complexes undergoing productive infection has provided a unique opportunity to observe these rare events *in situ*. Together with complementary methods, modern imaging investigation has led to a series of novel findings supporting the functional presence of CA, or even a capsid-derived structure, during early HIV-1 infection events (Peng *et al.* 2014; Chin *et al.* 2015; Stultz *et al.* 2017; Francis and Melikyan 2018). In the following sections we will summarize evidence that supports “complete uncoating” and evidence that supports the functional presence of CA during early infection events. We will then try to reconcile these observations and propose a working model that will evolve with future in-depth investigation.

### Evidence Supporting “Complete Uncoating” and Proposed Roles for Uncoating

In the early reports of characterizing components of RTC/ PICs, the viral replication complexes were isolated from the cytoplasm of infected cells. Cells were lysed followed by fractionation over sucrose gradients by ultracentrifugation (Miller *et al.* 1997; Fassati and Goff 2001). The fraction that contained detectable HIV-1 DNA was considered to contain RTCs/PICs and was analyzed for the presence of viral proteins. In these efforts, MA, RT and IN were readily detectable but little or no CA was detected (Miller *et al.* 1997; Fassati and Goff 2001). In contrast, characterization of RTC/PIC components of murine leukemia virus (MLV) showed clear CA association (Fassati and Goff 1999). Accordingly, it was proposed that the HIV-1 CA protein was most likely lost during reverse transcription. It is currently believed that the capsid/capsid-derived structure undergoes dynamic structural remodeling during these early events resulting in metastable complexes (Campbell and Hope 2015; Yamashita and Engelman 2017). It is possible that the remodeled structure is not stable enough to withstand the *in vitro* purification steps employed in these pioneering studies. Recently, some studies reported that CA becomes undetectable within 60 min after virus entry on the majority of the intracellular

### The Roles of CA during Post-Entry and Nuclear Import Events

#### “Uncoating”

After being released into the cytoplasm, the HIV-1 core is transformed into a viral ribonucleoprotein complex to accommodate reverse transcription of the single-stranded viral RNA genome into double-stranded DNA, and to facilitate integration of the viral genome into the host chromosome (Hu and Hughes 2012). Accordingly, these complexes are referred to as the viral reverse transcription complex (RTC) and pre-integration complex (PIC), respectively. The RTC/PICs are believed to undergo a so-called “uncoating” event, losing some of the CA protein and/or the capsid lattice during reverse transcription and nuclear entry (Ambrose and Aiken 2014) (Fig. 3). But the mechanism of when, where and how the “uncoating” event occurs has long been an important yet highly debated topic. One model proposes that the capsid structure needs to be fully disassembled for the virus to complete reverse transcription and nuclear entry (Campbell and Hope 2015). However, accumulating evidence from different groups has

M. Novikova et al.: Role of HIV-1 Capsid

![Diagram](attachment:diagram.png)

Fig. 3 A proposed model for post-entry events of HIV-1 infection mediated by CA-interacting host factors. Microtubule-associated proteins MAP1, kinesin-1 adaptor protein FEZ1, and dynein adaptor protein BICD2 bind to CA protein and facilitate the inward trafficking of HIV-1 reverse transcription complex (RTC) and pre-integration complex (PIC) towards the nuclear membrane. Upon arrival at the nuclear pore complex (NPC), Nup358 mediates docking of RTC/PIC through interacting with CA. Nup153 then mediates PIC translocation through the NPC by interacting with CA, and potentially facilitates nuclear PIC (n-PIC) release from the nuclear basket in concert with CPSF6. Afterwards, CPSF6, together with LEDGF, associate with the n-PIC to facilitate HIV-1 integration. TNPO3 might indirectly facilitate these processes through mediating proper CPSF6 nuclear localization. Several restriction factors also interfere with these post-entry infection events through interacting with CA. TRIM5 binds to the CA of RTC/PIC causing premature uncoating and inhibits reverse transcription. MxB restricts RTC/PIC nuclear entry by interacting with CA. Nuclear factor NONO binds to the CA on n-PIC and promotes cGAS sensing of HIV viral DNA genome in the nucleus. CA in the incoming viral capsid, RTC, and PIC is indicated in green. Viral nucleic acid (RNA or DNA) is shown inside the CA-containing complexes. Host dependency factors are shown in pink and host restriction factors in blue.

between “uncoating” and reverse transcription still awaits further characterization and the potential reason why CA was not detected on the RTCs/PICs in these studies will be discussed in the following section.

Evidence Supporting the Functional Presence of CA on RTC/ PIC

Lentiviruses are unique among retroviruses in their ability to infect non-dividing cells. Being able to integrate their newly synthesized DNA genomes in the host cell chromatin without the dissolution of the nuclear envelope (NE) that occurs during mitosis implies that the lentiviral PIC must be able to cross the intact NE. Initial studies suggested that MA and Vpr were responsible for HIV-1 nuclear import (Bukrinsky et al. 1993; von Schwedler et al. 1994; Gallay et al. 1995a, b); however, these results were contested (Freed and Martin 1994; Freed et al. 1995, 1997). In recent years, a number of studies have shown the

important functionality of CA and/or capsid structure during the early infection events of the RTC/PIC pathway. HIV-1 CA was first indicated to be important for early infection events by studies reporting that CA is the determinant for infection of non-dividing cells, as replacing HIV-1 CA with MLV CA in Gag chimeras impaired viral nuclear entry (Yamashita and Emerman 2004; Yamashita *et al.* 2007). Consistent with this discovery, it has been demonstrated that host factors, including Transportin 3 (TNPO3), Nucleoporin 153 (Nup153) and RAN binding protein 2 (RanBP2), facilitate HIV-1 nuclear entry in a CA-dependent manner (Matreyek and Engelman 2011; Zhou *et al.* 2011). Imaging-based investigations provided direct evidence of the functional presence of CA on the RTC/PIC in infected cells. In 2002, a study combining fluorescent imaging with electron microscopy to visualize RTCs/PICs in infected cells observed that around 67% of the complexes contained CA proteins (McDonald *et al.* 2002). However, this pioneering effort could not ascertain whether the association of CA with the RTC/PIC is functionally relevant. In 2014, a robust EdU labeling strategy was used to identify RTC/PICs that have undergone reverse transcription. This study found that, in contrast to the previous study, nearly all the cytoplasmic RTCs/PICs contained detectable CA signal arguing for the functional presence and relevance of CA in these complexes (Peng *et al.* 2014). It is not clear why these two studies reported different levels of CA association. It should be noted that a polyclonal CA antibody was used to detect CA in the Peng *et al.* study, while a monoclonal antibody was used in the McDonald *et al.* study. The polyclonal CA antibody provides the advantage that it can recognize multiple CA epitopes while the single epitope recognized by the CA monoclonal antibody may be shielded by conformational changes and/or associated host factors especially in the later stage of the RTC/PIC pathway. It is possible that different antibodies used in these two studies could partially explain the inconsistency. This may also explain why other previous studies that used CA monoclonal antibodies did not detect CA on viral complexes after 1 h of infection (McDonald *et al.* 2002; Xu *et al.* 2013). It should also be noted that CA association with cytoplasmic RTCs/PICs appears to be cell type dependent, as less than 50% of cytoplasmic RTCs were CA positive in infected monocyte-derived macrophage (MDM), a natural HIV-1 target cell (Peng *et al.* 2014). Whether this is due to the presence of host restriction factors in MDM that cause premature HIV-1 uncoating needs further investigation. Nevertheless, the functional presence of CA on cytoplasmic RTCs has been confirmed by a number of studies using a variety of labeling strategies (Chin *et al.* 2015; Francis *et al.* 2016; Stultz *et al.* 2017; Francis and Melikyan 2018).

The inconsistencies regarding the presence and functional relevance of CA in the RTC/PIC pathway are likely explained by the use of different methods, materials, and systems in the above-mentioned studies. However, accumulating evidence supports the functional presence of CA throughout the HIV-1 early infection events including cytoplasmic trafficking, NE docking and nuclear import. In the following sections, the roles of CA and interacting host factors during these different stages of HIV-1 early infection will be discussed in further detail.

### CA-Mediated Cytoplasmic Trafficking

The cytoplasm is a dense environment with a high concentration of cellular proteins and organelles. Cargos with a molecular mass larger than 500 kDa cannot diffuse freely and the mega-dalton viral replication complexes are clearly too large for passive diffusion. For this reason, viruses utilize the cytoskeletal network to achieve directed movement (Walsh and Naghavi 2019). HIV-1 has been reported to traffic towards the nucleus along microtubules in a CA-dependent manner, and a number of host factors have been implicated in this interaction (Fig. 3).

#### MAP1A and MAP1S

As the microtubule network has been suggested to provide a path for the movement of HIV-1 replication complexes towards the nucleus, the association between RTCs/PICs and microtubules is likely to be important for HIV-1 trafficking. A yeast two-hybrid screen identified the microtubule-associated proteins MAP1A and MAP1S as interaction partners for HIV-1 CA. Depletion of these two MAP1 proteins reduced HIV-1 capsid association with microtubules, impaired HIV-1 trafficking towards the nucleus and resulted in reduced infectivity. Taken together, these results suggest that MAP1 proteins tether incoming viral capsids to the microtubule network through binding to CA and promote HIV-1 trafficking towards the nucleus (Fernandez *et al.* 2015).

#### FEZ1 and Kinesin-1

Adaptor and motor proteins mediate cargo translocation along microtubules. Identification of adaptor and motor proteins that mediate HIV-1 trafficking would be important for understanding the mechanism of HIV-1 trafficking along microtubules. Malikov *et al.* identified Fasciculation And Elongation Protein Zeta 1 (FEZ1) as a kinesin-1 adaptor protein that binds CA during HIV-1 infection (Malikov *et al.* 2015). FEZ1 depletion resulted in viral particles exhibiting bi-directional movement without net trafficking to the nucleus. The interaction between FEZ1
and kinesin-1 was shown to be important for the ability of FEZ1 to promote HIV-1 infection, suggesting that FEZ1 mediates kinesin-1-dependent HIV-1 inward trafficking along microtubules. Interestingly, kinesins are motor proteins involved in plus-end-directed movement which, intuitively speaking, would not facilitate HIV-1 inward trafficking. In addition, the authors found that both dynein and kinesin-1 motors are required for HIV-1 trafficking towards the nucleus. As described below, the mechanism by which HIV-1 employs opposing motors to achieve microtubule-dependent inward trafficking began to be clarified by recent discoveries of dynein adaptor's participation in this process.

### BICD2

Bicaudal D2 (BICD2) is a dynein adaptor protein that was recently found in two studies to facilitate HIV-1 trafficking in the cytoplasm (Dharan *et al.* 2017; Carnes *et al.* 2018b). Depletion of BICD2 did not affect reverse transcription but lead to significantly reduced nuclear entry (Dharan *et al.* 2017; Carnes *et al.* 2018b). Using live-cell imaging, Dharan *et al.* further revealed that BICD2 depletion reduced the speed and directed transport of cytoplasmic HIV-1 capsids, resulting in a nuclear entry defect (Dharan *et al.* 2017). BICD2 was found to interact with intracellular HIV-1 capsids and *in vitro* CA-NC complexes. The CC3 domain of BICD2 was shown to be critical for this interaction (Dharan *et al.* 2017; Carnes *et al.* 2018b). Depletion of BICD2 resulted in accumulation of viral replication complexes in the cytoplasm, which triggered stronger interferon-I (IFN-I) responses in infected differentiated THP-1 macrophages (Dharan *et al.* 2017). Together, these two studies established the role of dynein adaptor BICD2 in mediating HIV-1 cytoplasmic trafficking towards the nucleus. Thus, both kinesin and dynein adaptor proteins contribute to HIV-1 inward movement. Whether and how the two adaptor proteins (FEZ1 and BICD2) function in concert to mediate HIV-1 inward trafficking is an interesting question for future studies.

It should also be noted that there are studies suggesting that microtubules are dispensable for RTC/PIC cytoplasmic trafficking (Vinay Pathak, personal communication). Indeed, microtubule-independent cytoplasmic trafficking was observed in live-cell imaging (McDonald *et al.* 2002) and disruption of the microtubule network by nocodazole treatment inhibited HIV-1 infection by only approximately twofold (Bukrinskaya *et al.* 1998). Further investigation is needed to establish the role of microtubules during RTC/PIC trafficking.

---

### CA-Mediated NE Docking

Entry through the nuclear pore complex (NPC) is a major pathway for HIV-1 nuclear import, particularly during infection of non-dividing cells such as terminally differentiated macrophages. After inward cytoplasmic trafficking, HIV-1 RTC/PICs dock at the cytoplasmic side of the NPC to initiate nuclear import. Current studies suggest that Nucleoporin 358 (Nup358)/RanBP2 (hereafter referred to as Nup358) is an important nucleoporin mediating RTC/PIC docking at the NPC (Di Nunzio *et al.* 2012; Burdick *et al.* 2017) (Fig. 3). Nup358 is a component of the cytoplasmic filament of the NPC (Walther *et al.* 2002) and promotes nuclear import in a cargo- and transport receptor-specific manner (Walde *et al.* 2012). The role of Nup358 in HIV-1 replication was first identified in genome-wide screens for host factors required for HIV-1 infection (so-called HIV-1 dependency factors) (Brass *et al.* 2008; König *et al.* 2008). Depletion of Nup358 impaired HIV-1 nuclear entry as revealed by a decrease in the accumulation of 2-LTR circles (Di Nunzio *et al.* 2012), often used as a measure of productive nuclear import. Nup358 interacts with *in vitro*-assembled HIV-1 CA-NC complexes (Di Nunzio *et al.* 2012), which serve as a surrogate for mature capsids (Ganser *et al.* 1999), and directly associates with intracellular viral replication complexes (Dharan *et al.* 2016; Burdick *et al.* 2017). The N74D and P90A mutations in CA impair this association and these viral mutants do not rely on Nup358 for nuclear entry (Dharan *et al.* 2016). Quantitative microscopy further revealed that depletion of Nup358 reduced the number of HIV-1 replication complexes stably associated with the NE (Burdick *et al.* 2017). Taken together, these studies strongly suggested that Nup358 determines HIV-1 NE docking in a CA-dependent manner, potentially through direct CA–Nup358 interaction. Interestingly, Nup358 was also reported to be translocated into the cytoplasm in a CA-and kinesin family member 5B (KIF5B)-dependent manner during HIV-1 infection and this cytoplasmic translocation is suggested to be important for HIV-1 nuclear entry (Dharan *et al.* 2016). It has been proposed that cytoplasmic translocation of Nup358 may disrupt the NPC and/or promote HIV-1 uncoating and thus indirectly facilitate viral nuclear entry (Dharan *et al.* 2016).

---

### CA-Mediated Nuclear Import

In addition to regulating docking at the NE, CA was also reported to mediate PIC transport through the nuclear pore by interacting with several host factors such as Nup153, TNPO3 and the cleavage and polyadenylation specificity factor 6 (CPSF6) (Fig. 3).
Nup153

Nup153 is a component of the basket of the nuclear pore complex and plays an essential role in NPC assembly (Vollmer *et al.* 2015). Nup153 was first identified as a host dependency factor for HIV-1 infection through genome-wide screening (Brass *et al.* 2008; König *et al.* 2008). Depletion of Nup153 did not affect reverse transcription but impaired HIV-1 nuclear import as revealed by reduced HIV-1 2-LTR circle accumulation and nuclear PICs (Matreyek and Engelman 2011; Di Nunzio *et al.* 2012, 2013). CA was found to determine the Nup153 dependency and the CA mutants N74D and P90A were shown to be largely insensitive to Nup153 depletion (Matreyek and Engelman 2011). Biochemical analysis further revealed that Nup153 directly interacts with *in vitro* assembled HIV-1 CA–NC complexes and CA monomers (Di Nunzio *et al.* 2013; Matreyek *et al.* 2013; Buffone *et al.* 2018). It was therefore proposed that Nup153 facilitates HIV-1 nuclear entry by directly binding to CA molecules on RTCs/PICs. Notably, Nup153 binds CA hexamers with much higher affinity than CA monomers (Price *et al.* 2014) suggesting that at least some hexameric CA remains intact on RTCs/PICs during transport through the nuclear pore. Depletion of Nup153 was also reported to reduce integration and alter integration site selection (Matreyek and Engelman 2011; Koh *et al.* 2013). Whether and how these additional functionalities of Nup153 are related to its role in mediating HIV-1 nuclear import will no doubt be the subject of future studies.

TNPO3

TNPO3 is a β-karyopherin that transports serine/arginine-rich splicing factors into the nucleus. Like Nup153, TNPO3 was also initially identified as a HIV-1 dependency factor through genome-wide RNA interference screens (Brass *et al.* 2008). Depletion of TNPO3 did not affect viral reverse transcription but reduced the number of proviruses implying a role during nuclear entry (De Iaco *et al.* 2013; Fricke *et al.* 2013). Similarly to Nup153, CA determines TNPO3 dependency and *in vitro* biochemical analysis showed that TNPO3 can bind CA-NC complexes (Krishnan *et al.* 2010; De Iaco and Luban 2011; Valle-Casuso *et al.* 2012). Although it was initially proposed that TNPO3 mediates HIV-1 nuclear entry through binding to CA on the RTC/PIC, accumulating evidence supports a role for TNPO3 in facilitating HIV-1 nuclear entry indirectly through regulating the localization of CPSF6 (De Iaco *et al.* 2013; Fricke *et al.* 2013; Maertens *et al.* 2014).

CPSF6

CPSF6 is a component of the cleavage factor 1 (CFIm) complex that functions in mRNA polyadenylation. CPSF6 was first identified to be relevant for HIV-1 infection through a cDNA library screen in which a truncated form of CPSF6 was found to inhibit HIV-1 replication at the step of nuclear entry (Lee *et al.* 2010). It was further revealed that the inhibitory effect of truncated CPSF6 was dependent on a direct CA-CPSF6 interaction. Forced evolution experiments led to the selection of the CA-N74D mutant, which lost CPSF6 binding and escaped the antiviral activity of truncated CPSF6 (Lee *et al.* 2010, 2012). These pioneering discoveries triggered a series of in-depth investigations which suggested that in contrast to truncated CPSF6, which displays cytosolic localization, the intact, full-length CPSF6, which is predominantly nuclear, may function as an HIV dependency factor by facilitating viral nuclear import (Chin *et al.* 2015). CPSF6 is a serine/arginine-rich protein that is transported into the nucleus by TNPO3. Indeed, depletion of TNPO3 results in significantly higher levels of cytoplasmic CPSF6, which interacts with RTCs/PICs and may consequently impair viral trafficking and/or nuclear entry. This mechanism also explains the discovery that depletion of TNPO3 lead to reduced HIV-1 nuclear entry. Notably, it was shown by two research groups that CPSF6 binds hexameric CA with much higher affinity than monomeric CA (Bhattacharya *et al.* 2014; Price *et al.* 2014). This would again suggest that a certain level of hexameric CA remains associated with PICs during and after passing through the nuclear pore. Recent studies reported that the CA-CPSF6 interaction regulates PIC intranuclear localization and directs HIV-1 integration to actively transcribed euchromatin (Sowd *et al.* 2016; Achuthan *et al.* 2018). It is possible that the CA-CPSF6 interaction mediates nuclear events beyond nuclear entry and integration; how these events are coordinated by CPSF6 together with other host factors, such as Nup153 and TNPO3, will be interesting questions to follow in the future.

These well-characterized interactions between CA and host proteins unequivocally established the role and functional presence of CA during passage through the nuclear pore. But how this large RTC/PIC passes through the nuclear pore remains enigmatic. The nuclear pore has a central opening of around 40 nm (Bui *et al.* 2013) which is believed to determine the maximal cargo size (Pante and Kann 2002). While the dimension of the RTC/PIC is currently unknown, it is possible that the diameter may be larger than 40 nm given that the broad end of the HIV-1 capsid was determined to be $56 \pm 5$ nm (Briggs *et al.* 2006). Intuitively, it appears difficult to understand how the mega-structure of the RTC/PIC can pass through the

nuclear pore. Recent studies reveal that the NPC may undergo dynamic structural re-organization to accommodate translocation of large cargo, especially during viral nuclear entry (Knockenhauer and Schwartz 2016). At the same time, the RTC/PIC likely undergoes structural remodeling and potentially partial uncoating which may result in a complex that fits the opening of the NPC. Furthermore, the fact that CA can bind multiple nucleoporins indicates that the CA protein itself may function as a transportin to facilitate nuclear entry of the “large” RTC/PIC through the nuclear pore. The exact molecular mechanism of RTC/PIC passing through the nuclear pore awaits in-depth investigation and will contribute to the general understanding of transport mechanism of large cargoes through the nuclear pore.

### The Presence of CA on the Nuclear PIC

While the functional relevance of CA during early HIV-1 infection events in the cytoplasm and at the NE has started to become clear, the presence and potential role of CA on the nuclear PIC (n-PIC) is still largely uncharacterized. In 2011, Zhou *et al.* reported detection of nuclear CA in HIV-1 infected cells and further determined the timing of CA nuclear accumulation, implying a role for CA in post-nuclear entry events (Zhou *et al.* 2011). The presence of nuclear CA was corroborated in a study from Peng *et al.*, in which distinctive CA signals were detected on nearly all n-PICs in infected MDMs (Peng *et al.* 2014). In that study, viral DNA staining was employed to confirm that the detected n-PICs represented productive replication complexes suggesting that the associated CA may be functionally relevant. The association of CA with nuclear replication complexes was then confirmed by a number of studies from different groups in different infection contexts (Chin *et al.* 2015; Hulme *et al.* 2015; Chen *et al.* 2016; Burdick *et al.* 2017; Stultz *et al.* 2017; Francis and Melikyan 2018). Despite the growing consensus that at least some CA remains associated with the PIC after nuclear entry, the role of CA on the n-PIC is not well understood. A study reporting that the CA-CPSF6 interaction contributes to directed HIV-1 integration (Sowd *et al.* 2016) provides compelling evidence of CA functionality after nuclear entry. A very recent study reported that the host factor NONO binds to HIV CA protein on n-PIC and facilitates cGAS-mediated sensing of HIV DNA in the nucleus (Lahaye *et al.* 2018). It should be noted that the functional significance of this mechanism is more pronounced for HIV-2 CA than for HIV-1 CA due to stronger binding affinity with NONO (Lahaye *et al.* 2018). This study not only confirmed the presence of CA on n-PIC but also suggests that the nuclear CA could mediate HIV innate sensing in the nucleus.

---

### CA-Targeted Restriction Factors

As an integral component of the RTC/PIC, CA not only mediates interactions with host dependency factors to facilitate early infection events but is also the target of several host restriction factors that block the RTC/PIC pathway via different mechanisms (Fig. 3).

#### TRIM5

Tripartite motif-containing protein 5 alpha (TRIM5α) was first identified in an effort to search for species-specific restriction factors that block HIV-1 infection of cells from Old World monkeys (Stremlau *et al.* 2004). Nonhuman primate TRIM5α proteins, such as rhesus (rh)TRIM5α, inhibit HIV-1 infection by directly binding to the capsid (Wagner *et al.* 2018). This partially explains why in some cases HIV-1 cannot productively infect cells from nonhuman primates. Binding of rhTRIM5α to capsid causes premature uncoating and inhibits reverse transcription (Black and Aiken 2010; Kutluay *et al.* 2013). Further analysis showed that the RING domain of the TRIM5 protein is important for restriction, suggesting that ubiquitin ligase activity is involved (Kim *et al.* 2011; Lienlaf *et al.* 2011).

#### MxB

Myxovirus resistance protein 2 (MxB/Mx2) (hereafter denoted MxB) is a recently identified HIV-1 restriction factor that is induced by IFNα (Goujon *et al.* 2013; Kane *et al.* 2013; Liu *et al.* 2013). MxB does not inhibit reverse transcription but blocks nuclear entry, as revealed by a reduction in the accumulation of 2-LTR circles. Several studies have established that MxB targets CA to restrict HIV-1 nuclear entry (Busnadiego *et al.* 2014; Buffone *et al.* 2015; Schulte *et al.* 2015). Domain mapping further determined that the N-terminal domain (NTD) of MxB determines restriction against HIV-1 (Goujon *et al.* 2014). Strikingly, adding the MxB NTD to non-restrictive factors such as MX1 or canine MxB rendered these chimeric proteins restrictive to HIV-1 infection (Busnadiego *et al.* 2014; Goujon *et al.* 2014). Interestingly, CA binding is necessary but not sufficient for MxB restriction (Fribourg *et al.* 2014; Fricke *et al.* 2014). A recent study investigated the functional crosstalk between NPC, MxB, CypA and CA and reported that restriction by MxB is largely dependent on CypA and the composition of the NPC (Kane *et al.* 2018). Accordingly, the mechanism of MxB restriction is proposed to be context dependent in different cell types with varying levels of nucleoporins and CypA (Kane *et al.* 2018).

Conclusion

HIV-1 CA mediates a number of processes required for productive HIV-1 infection. Ongoing studies continue to reveal CA regions important for structural integrity, either of immature or mature HIV-1 virions, as well as novel CA interfaces needed for interaction with host cellular cofactors or restriction factors. Given that CA-targeted inhibitors have not been implemented in clinical use so far, novel data on CA functions not only expand our understanding of HIV-1 biology but also provide useful information that could result in the development of novel antiviral therapeutics.

Acknowledgements We thank members of the Freed and Peng laboratories for critical review of this report. Research in the Freed laboratory is supported by the Intramural Research Program of the Center for Cancer Research, National Cancer Institute, NIH and the Intramural AIDS Targeted Antiviral Program. Research in the Peng laboratory is supported by National Natural Science Foundation of China (No. 31770188), Strategic Priority Research Program of the Chinese Academy of Sciences (XDB29010000), the National Science and Technology Major Project (No. 2018ZX10101004), the Special major program of Wuhan Institute of Virology (No. WIV-135-TP1), the Hundred Talents Program of Chinese Academy of Sciences, and the State Key Laboratory of Virology open projects (No. 2017IOV003).

Compliance with ethics standards

Conflict of interest The authors declare that they have no conflict of interest.

Animal and Human Rights Statement This article does not contain any studies with human or animal subjects performed by any of the authors.

Open Access This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<http://creativecommons.org/licenses/by/4.0/>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made.

References

Ambrose Z, Aiken C (2014) HIV-1 uncoating: connection to nuclear entry and regulation by host proteins. Virology 454–455:371–379

Bhattacharya S, Zhang H, Debnath AK, Cowburn D (2008) Solution structure of a hydrocarbon stapled peptide inhibitor in complex with monomeric C-terminal domain of HIV-1 capsid. J Biol Chem 283:16274–16278

Bhattacharya A, Alam SL, Fricke T, Zadrozny K, Sedzicki J, Taylor AB, Demeler B, Pornillos O, Ganser-Pornillos BK, Diaz-Griffero F, Ivanov DN, Yeager M (2014) Structural basis of HIV-1 capsid recognition by PF74 and CPSF6. Proc Natl Acad Sci USA 111:18625–18630

Black LR, Aiken C (2010) TRIM5 alpha disrupts the structure of assembled HIV-1 capsid complexes in vitro. J Virol 84:6564–6569

Blair WS, Cao J, Fok-Seang J, Griffin P, Isaacson J, Jackson RL, Murray E, Patick AK, Peng Q, Perros M, Pickford C, Wu H, Butler SL (2009) New small-molecule inhibitor class targeting human immunodeficiency virus type 1 virion maturation. Antimicrob Agents Chemother 53:5080–5087

Blair WS, Pickford C, Irving SL, Brown DG, Anderson M, Bazin R, Cao J, Ciaramella G, Isaacson J, Jackson L, Hunt R, Kjerrstrom A, Nieman JA, Patick AK, Perros M, Scott AD, Whitby K, Wu H, Butler SL (2010) HIV capsid is a tractable target for small molecule therapeutic intervention. PLoS Pathog 6:e1001220

Borsetti A, Ohagen A, Gottlinger HG (1998) The C-terminal half of the human immunodeficiency virus type 1 Gag precursor is sufficient for efficient particle assembly. J Virol 72:9313–9317

Brass AL, Dykxhoorn DM, Benita Y, Yan N, Engelman A, Xavier RJ, Lieberman J, Elledge SJ (2008) Identification of host proteins required for HIV infection through a functional genomic screen. Science 319:921–926

Briggs JA, Grunewald K, Glass B, Forster F, Krausslich HG, Fuller SD (2006) The mechanism of HIV-1 core assembly: insights from three-dimensional reconstructions of authentic virions. Structure 14:15–20

Briggs JA, Riches JD, Glass B, Bartonova V, Zanetti G, Krausslich HG (2009) Structure and assembly of immature HIV. Proc Natl Acad Sci USA 106:11090–11095

Buffone C, Schulte B, Opp S, Diaz-Griffero F (2015) Contribution of MxB oligomerization to HIV-1 capsid binding and restriction. J Virol 89:3285–3294

Buffone C, Martinez-Lopez A, Fricke T, Opp S, Severgnini M, Cifola I, Petiti L, Frabetti S, Skorupka K, Zadrozny KK, Ganser-Pornillos BK, Pornillos O, Di Nunzio F, Diaz-Griffero F (2018) Nup153 unlocks the nuclear pore complex for HIV-1 nuclear translocation in nondividing cells. J Virol 92:e00648-00618

Bui KH, von Appen A, DiGuilio AL, Ori A, Sparks L, Mackmull MT, Bock T, Hagen W, Andres-Pons A, Glavy JS, Beck M (2013) Integrated structural analysis of the human nuclear pore complex scaffold. Cell 155:1233–1243

Bukrinskaya A, Brichacek B, Mann A, Stevenson M (1998) Establishment of a functional human immunodeficiency virus type 1 (HIV-1) reverse transcription complex involves the cytoskeleton. J Exp Med 188:2113–2125

Bukrinsky MI, Haggerty S, Dempsey MP, Sharova N, Adzhubel A, Spitz L, Lewis P, Goldfarb D, Emerman M, Stevenson M (1993) A nuclear localization signal within HIV-1 matrix protein that governs infection of non-dividing cells. Nature 365:666–669

Burdick RC, Delviks-Frankenberry KA, Chen J, Janaka SK, Sastri J, Hu WS, Pathak VK (2017) Dynamics and regulation of nuclear import and nuclear movements of HIV-1 complexes. PLoS Pathog 13:e1006570

Busnadiego I, Kane M, Rihn SJ, Preugschas HF, Hughes J, Blanco-Melo D, Strouvelle VP, Zang TM, Willett BJ, Boutell C,

Bieniasz PD, Wilson SJ (2014) Host and viral determinants of Mx2 antiretroviral activity. J Virol 88:7738–7752

Campbell EM, Hope TJ (2015) HIV-1 capsid: the multifaceted key player in HIV-1 infection. Nat Rev Microbiol 13:471–483

Carlson LA, Briggs JA, Glass B, Riches JD, Simon MN, Johnson MC, Muller B, Grunewald K, Krausslich HG (2008) Three-dimensional analysis of budding sites and released virus suggests a revised model for HIV-1 morphogenesis. Cell Host Microbe 4:592–599

Carnes SK, Sheehan JH, Aiken C (2018a) Inhibitors of the HIV-1 capsid, a target of opportunity. Curr Opin HIV AIDS 13:359–365

Carnes SK, Zhou J, Aiken C (2018b) HIV-1 engages a dynein-dynactin-BICD2 complex for infection and transport to the nucleus. J Virol. https://doi.org/10.1128/JVI.00358-18:e00358-00318

Chen NY, Zhou L, Gane PJ, Opp S, Ball NJ, Nicastro G, Zufferey M, Buffone C, Luban J, Selwood D, Diaz-Griffero F, Taylor I, Fassati A (2016) HIV-1 capsid is involved in post-nuclear entry steps. Retrovirology 13:28

Chin CR, Perreira JM, Savidis G, Portmann JM, Aker AM, Feeley EM, Smith MC, Brass AL (2015) Direct visualization of HIV-1 replication intermediates shows that capsid and CPSF 6 modulate HIV-1 intra-nuclear invasion and integration. Cell Rep 13:1717–1731

De Iaco A, Luban J (2011) Inhibition of HIV-1 infection by TNPO3 depletion is determined by capsid and detectable after viral cDNA enters the nucleus. Retrovirology 8:98

De Iaco A, Santoni F, Vannier A, Guipponi M, Antonarakis S, Luban J (2013) TNPO3 protects HIV-1 replication from CPSF6-mediated capsid stabilization in the host cell cytoplasm. Retrovirology 10:20

Dharan A, Talley S, Tripathi A, Mamede JI, Majetschak M, Hope TJ, Campbell EM (2016) KIF5B and Nup358 cooperatively mediate the nuclear import of HIV-1 during infection. PLoS Pathog 12:e1005700

Dharan A, Opp S, Abdel-Rahim O, Keceli SK, Imam S, Diaz-Griffero F, Campbell EM (2017) Bicaudal D2 facilitates the cytoplasmic trafficking and nuclear import of HIV-1 genomes during infection. Proc Natl Acad Sci USA 114:E10707–E10716

Di Nunzio F, Danckaert A, Fricke T, Perez P, Fernandez J, Perret E, Roux P, Shorte S, Charneau P, Diaz-Griffero F, Arhel NJ (2012) Human nucleoporins promote HIV-1 docking at the nuclear pore, nuclear import and integration. PLoS ONE 7:e46037

Di Nunzio F, Fricke T, Miccio A, Valle-Casuso JC, Perez P, Souque P, Rizzi E, Severgnini M, Mavilio F, Chameau P, Diaz-Griffero F (2013) Nup153 and Nup98 bind the HIV-1 core and contribute to the early steps of HIV-1 replication. Virology 440:8–18

Dick RA, Zadrozny KK, Xu C, Schur FKM, Lyddon TD, Ricana CL, Wagner JM, Perilla JR, Ganser-Pornillos BK, Johnson MC, Pornillos O, Vogt VM (2018) Inositol phosphates are assembly co-factors for HIV-1. Nature 560:509–512

Fader LD, Bethell R, Bonneau P, Bos M, Bousquet Y, Cordingley MG, Coulombe R, Deroy P, Faucher AM, Gagnon A, Goudreau N, Grand-Maitre C, Guse I, Hucke O, Kawai SH, Lacoste JE, Landry S, Lemke CT, Malenfant E, Mason S, Morin S, O’Meara J, Simoneau B, Titolo S, Yoakim C (2011) Discovery of a 1,5-dihydrobenzo[b][1,4]diazepine-2,4-dione series of inhibitors of HIV-1 capsid assembly. Bioorg Med Chem Lett 21:398–404

Fassati A (2012) Multiple roles of the capsid protein in the early steps of HIV-1 infection. Virus Res 170:15–24

Fassati A, Goff SP (1999) Characterization of intracellular reverse transcription complexes of Moloney murine leukemia virus. J Virol 73:8919–8925

Fassati A, Goff SP (2001) Characterization of intracellular reverse transcription complexes of human immunodeficiency virus type 1. J Virol 75:3626–3635

Fernandez J, Portilho DM, Danckaert A, Munier S, Becker A, Roux P, Zambo A, Shorte S, Jacob Y, Vidalain PO, Charneau P, Clavel F, Arhel NJ (2015) Microtubule-associated proteins 1 (MAP1) promote human immunodeficiency virus type I (HIV-1) intracytoplasmic routing to the nucleus. J Biol Chem 290:4631–4646

Fontana J, Keller PW, Urano E, Ablan SD, Steven AC, Freed EO (2016) Identification of an HIV-1 mutation in spacer peptide 1 that stabilizes the immature CA-SP1 lattice. J Virol 90:972–978

Francis AC, Melikyan GB (2018) Single HIV-1 imaging reveals progression of infection through CA-dependent steps of docking at the nuclear pore, uncoating, and nuclear transport. Cell Host Microbe 23:536–548e536

Francis AC, Marin M, Shi J, Aiken C, Melikyan GB (2016) Time-resolved imaging of single HIV-1 uncoating in vitro and in living cells. PLoS Pathog 12:e1005709

Frank GA, Narayan K, Bess JW Jr, Del Prete GQ, Wu X, Moran A, Hartnell LM, Earl LA, Lifson JD, Subramaniam S (2015) Maturation of the HIV-1 core by a non-diffusional phase transition. Nat Commun 6:5854

Freed EO (2015) HIV-1 assembly, release and maturation. Nat Rev Microbiol 13:484–496

Freed EO, Martin MA (1994) HIV-1 infection of non-dividing cells. Nature 369:107–108

Freed EO, Englund G, Martin MA (1995) Role of the basic domain of human immunodeficiency virus type 1 matrix in macrophage infection. J Virol 69:3949–3954

Freed EO, Englund G, Maldarelli F, Martin MA (1997) Phosphorylation of residue 131 of HIV-1 matrix is not required for macrophage infection. Cell 88:171–173 (discussion 173–174)

Fribourg JL, Nguyen HC, Matreyek KA, Alvarez FJD, Summers BJ, Dewdney TG, Aiken C, Zhang P, Engelman A, Xiong Y (2014) Structural insight into HIV-1 restriction by MxB. Cell Host Microbe 16:627–638

Fricke T, Valle-Casuso JC, White TE, Brandariz-Nunez A, Bosche WJ, Reszka N, Gorelick R, Diaz-Griffero F (2013) The ability of TNPO3-depleted cells to inhibit HIV-1 infection requires CPSF6. Retrovirology 10:46

Fricke T, White TE, Schulte B, de Souza Aranha Vieira DA, Dharan A, Campbell EM, Brandariz-Nunez A, Diaz-Griffero F (2014) MxB binds to the HIV-1 core and prevents the uncoating process of HIV-1. Retrovirology 11:68

Gallay P, Swingler S, Aiken C, Trono D (1995a) HIV-1 infection of nondividing cells: C-terminal tyrosine phosphorylation of the viral matrix protein is a key regulator. Cell 80:379–388

Gallay P, Swingler S, Song J, Bushman F, Trono D (1995b) HIV nuclear import is governed by the phosphotyrosine-mediated binding of matrix to the core domain of integrase. Cell 83:569–576

Gamble TR, Vajdos FF, Yoo S, Worthylake DK, Houseweart M, Sundquist WI, Hill CP (1996) Crystal structure of human cyclophilin A bound to the amino-terminal domain of HIV-1 capsid. Cell 87:1285–1294

Gamble TR, Yoo S, Vajdos FF, von Schwedler UK, Worthylake DK, Wang H, McCutcheon JP, Sundquist WI, Hill CP (1997) Structure of the carboxyl-terminal dimerization domain of the HIV-1 capsid protein. Science 278:849–853

Ganser BK, Li S, Klishko VY, Finch JT, Sundquist WI (1999) Assembly and analysis of conical models for the HIV-1 core. Science 283:80–83

Gitti RK, Lee BM, Walker J, Summers MF, Yoo S, Sundquist WI (1996) Structure of the amino-terminal core domain of the HIV-1 capsid protein. Science 273:231–235

Goudreau N, Lemke CT, Faucher AM, Grand-Maitre C, Goulet S, Lacoste JE, Rancourt J, Malenfant E, Mercier JF, Titolo S, Mason SW (2013) Novel inhibitor binding site discovery on HIV-1 capsid N-terminal domain by NMR and X-ray crystallography. ACS Chem Biol 8:1074–1082

Goujon C, Moncorge O, Bauby H, Doyle T, Ward CC, Schaller T, Hue S, Barclay WS, Schulz R, Malim MH (2013) Human MX2 is an interferon-induced post-entry inhibitor of HIV-1 infection. Nature 502:559–562

Goujon C, Moncorge O, Bauby H, Doyle T, Barclay WS, Malim MH (2014) Transfer of the amino-terminal nuclear envelope targeting domain of human MX2 converts MX1 into an HIV-1 resistance factor. J Virol 88:9017–9026

Gres AT, Kirby KA, KewalRamani VN, Tanner JJ, Pornillos O, Sarafianos SG (2015) X-ray crystal structures of native HIV-1 capsid protein reveal conformational variability. Science 349:99–103

Hu WS, Hughes SH (2012) HIV-1 reverse transcription. Cold Spring Harb Perspect Med 2:a006882

Hulme AE, Perez O, Hope TJ (2011) Complementary assays reveal a relationship between HIV-1 uncoating and reverse transcription. Proc Natl Acad Sci USA 108:9975–9980

Hulme AE, Kelley Z, Foley D, Hope TJ (2015) Complementary assays reveal a low level of CA associated with viral complexes in the nuclei of HIV-1-infected cells. J Virol 89:5350–5361

Jacques DA, McEwan WA, Hilditch L, Price AJ, Towers GJ, James LC (2016) HIV-1 uses dynamic capsid pores to import nucleotides and fuel encapsidated DNA synthesis. Nature 536:349–353

Kane M, Yadav SS, Bitzegeio J, Kutluay SB, Zang T, Wilson SJ, Schoggins JW, Rice CM, Yamashita M, Hatziiioannou T, Bieniasz PD (2013) MX2 is an interferon-induced inhibitor of HIV-1 infection. Nature 502:563–566

Kane M, Rebensburg SV, Takata MA, Zang TM, Yamashita M, Kvaratskhelia M, Bieniasz PD (2018) Nuclear pore heterogeneity influences HIV-1 infection and the antiviral activity of MX2. Elife 7:e35738

Keller PW, Adamson CS, Heymann JB, Freed EO, Steven AC (2011) HIV-1 maturation inhibitor bevirimat stabilizes the immature Gag lattice. J Virol 85:1420–1428

Keller PW, Huang RK, England MR, Waki K, Cheng N, Heymann JB, Craven RC, Freed EO, Steven AC (2013) A two-pronged structural analysis of retroviral maturation indicates that core formation proceeds by a disassembly-reassembly pathway rather than a displacive transition. J Virol 87:13655–13664

Kelly BN, Kyere S, Kinde I, Tang C, Howard BR, Robinson H, Sundquist WI, Summers MF, Hill CP (2007) Structure of the antiviral assembly inhibitor CAP-1 complex with the HIV-1 CA protein. J Mol Biol 373:355–366

Kim J, Tipper C, Sodroski J (2011) Role of TRIM5alpha RING domain E3 ubiquitin ligase activity in capsid disassembly, reverse transcription blockade, and restriction of simian immunodeficiency virus. J Virol 85:8116–8132

Knockenhauer KE, Schwartz TU (2016) The nuclear pore complex as a flexible and dynamic gate. Cell 164:1162–1171

Koh Y, Wu X, Ferris AL, Matreyek KA, Smith SJ, Lee K, KewalRamani VN, Hughes SH, Engelman A (2013) Differential effects of human immunodeficiency virus type 1 capsid and cellular factors nucleoporin 153 and LEDGF/p75 on the efficiency and specificity of viral DNA integration. J Virol 87:648–658

Konig R, Zhou Y, Elleder D, Diamond TL, Bonamy GM, Irelan JT, Chiang CY, Tu BP, De Jesus PD, Lilley CE, Seidel S, Opaluch AM, Caldwell JS, Weitzman MD, Kuhen KL, Bandyopadhyay S, Ideker T, Orth AP, Miraglia LJ, Bushman FD, Young JA,

Chanda SK (2008) Global analysis of host-pathogen interactions that regulate early-stage HIV-1 replication. Cell 135:49–60

Krishnan L, Matreyek KA, Oztop I, Lee K, Tipper CH, Li X, Dar MJ, Kewalramani VN, Engelman A (2010) The requirement for cellular transportin 3 (TNPO3 or TRN-SR2) during infection maps to human immunodeficiency virus type 1 capsid and not integrase. J Virol 84:397–406

Kutluay SB, Perez-Caballero D, Bieniasz PD (2013) Fates of retroviral core components during unrestricted and TRIM5-restricted infection. PLoS Pathog 9:e1003214

Lahaye X, Gentili M, Silvin A, Conrad C, Picard L, Jouve M, Zueva E, Maurin M, Nadalin F, Knott GJ, Zhao B, Du F, Rio M, Amiel J, Fox AH, Li P, Etienne L, Bond CS, Colleaux L, Manel N (2018) NONO detects the nuclear HIV capsid to promote cGAS-mediated innate immune activation. Cell 175(488–501):e422

Lee K, Ambrose Z, Martin TD, Oztop I, Mulky A, Julias JG, Vandegraaff N, Baumann JG, Wang R, Yuen W, Takemura T, Shelton K, Taniuchi I, Li Y, Sodroski J, Littman DR, Coffin JM, Hughes SH, Unutmaz D, Engelman A, KewalRamani VN (2010) Flexible use of nuclear import pathways by HIV-1. Cell Host Microbe 7:221–233

Lee K, Mulky A, Yuen W, Martin TD, Meyerson NR, Choi L, Yu H, Sawyer SL, Kewalramani VN (2012) HIV-1 capsid-targeting domain of cleavage and polyadenylation specificity factor 6. J Virol 86:3851–3860

Lemke CT, Titolo S, von Schwedler U, Goudreau N, Mercier JF, Wardrop E, Faucher AM, Coulombe R, Banik SS, Fader L, Gagnon A, Kawai SH, Rancourt J, Tremblay M, Yoakim C, Simoneau B, Archambault J, Sundquist WI, Mason SW (2012) Distinct effects of two HIV-1 capsid assembly inhibitor families that bind the same site within the N-terminal domain of the viral CA protein. J Virol 86:6643–6655

Lemke CT, Titolo S, Goudreau N, Faucher AM, Mason SW, Bonneau P (2013) A novel inhibitor-binding site on the HIV-1 capsid N-terminal domain leads to improved crystallization via compound-mediated dimerization. Acta Crystallogr D Biol Crystallogr 69:1115–1123

Li S, Hill CP, Sundquist WI, Finch JT (2000) Image reconstructions of helical assemblies of the HIV-1 CA protein. Nature 407:409–413

Li F, Goila-Gaur R, Salzwedel K, Kilgore NR, Reddick M, Matallana C, Castillo A, Zoumplis D, Martin DE, Orenstein JM, Allaway GP, Freed EO, Wild CT (2003) PA-457: a potent HIV inhibitor that disrupts core condensation by targeting a late step in Gag processing. Proc Natl Acad Sci USA 100:13555–13560

Lienlaf M, Hayashi F, Di Nunzio F, Tochio N, Kigawa T, Yokoyama S, Diaz-Griffero F (2011) Contribution of E3-ubiquitin ligase activity to HIV-1 restriction by TRIM5alpha(rh): structure of the RING domain of TRIM5alpha. J Virol 85:8725–8737

Lingappa JR, Reed JC, Tanaka M, Chutiraka K, Robinson BA (2014) How HIV-1 Gag assembles in cells: putting together pieces of the puzzle. Virus Res 193:89–107

Liu Z, Pan Q, Ding S, Qian J, Xu F, Zhou J, Cen S, Guo F, Liang C (2013) The interferon-inducible MxB protein inhibits HIV-1 infection. Cell Host Microbe 14:398–410

Machara A, Lux V, Kozisek M, Grantz Saskova K, Stepanek O, Kotora M, Parkan K, Pavova M, Glass B, Sehr P, Lewis J, Muller B, Krausslich HG, Konvalinka J (2016) Specific inhibitors of HIV capsid assembly binding to the C-terminal domain of the capsid protein: evaluation of 2-arylquinazolines as potential antiviral compounds. J Med Chem 59:545–558

Maertens GN, Cook NJ, Wang W, Hare S, Gupta SS, Oztop I, Lee K, Pye VE, Cosnefroy O, Snijders AP, KewalRamani VN, Fassati A, Engelman A, Cherepanov P (2014) Structural basis for nuclear import of splicing factors by human Transportin 3. Proc Natl Acad Sci USA 111:2728–2733

Malikov V, da Silva ES, Jovasevic V, Bennett G, de Souza Aranha Vieira DA, Schulte B, Diaz-Griffero F, Walsh D, Naghavi MH (2015) HIV-1 capsids bind and exploit the kinesin-1 adaptor FEZ1 for inward movement to the nucleus. Nat Commun 6: 6660

Mallery DL, Marquez CL, McEwan WA, Dickson CF, Jacques DA, Anandapadamanaban M, Bichel K, Towers GJ, Saiardi A, Bocking T, James LC (2018) IP6 is an HIV pocket factor that prevents capsid collapse and promotes DNA synthesis. Elife 7:e35335

Matreyek KA, Engelman A (2011) The requirement for nucleoporin NUP153 during human immunodeficiency virus type 1 infection is determined by the viral capsid. J Virol 85: 7818–7827

Matreyek KA, Yucel SS, Li X, Engelman A (2013) Nucleoporin NUP153 phenylalanine-glycine motifs engage a common binding pocket within the HIV-1 capsid protein to mediate lentiviral infectivity. PLoS Pathog 9: e1003693

Mattei S, Glass B, Hagen WJ, Krausslich HG, Briggs JA (2016a) The structure and flexibility of conical HIV-1 capsids determined within intact virions. Science 354: 1434–1437

Mattei S, Schur FK, Briggs JA (2016b) Retrovirus maturation—an extraordinary structural transformation. Curr Opin Virol 18: 27–35

Mattei S, Tan A, Glass B, Muller B, Krausslich HG, Briggs JAG (2018) High-resolution structures of HIV-1 Gag cleavage mutants determine structural switch for virus maturation. Proc Natl Acad Sci USA 115: E9401–E9410

McDonald D, Vodicka MA, Lucero G, Svitkina TM, Borisy GG, Emerman M, Hope TJ (2002) Visualization of the intracellular behavior of HIV in living cells. J Cell Biol 159: 441–452

Meng X, Zhao G, Yufenyuy E, Ke D, Ning J, Delucia M, Ahn J, Gronenborn AM, Aiken C, Zhang P (2012) Protease cleavage leads to formation of mature trimer interface in HIV-1 capsid. PLoS Pathog 8: e1002886

Miller MD, Farnet CM, Bushman FD (1997) Human immunodeficiency virus type 1 preintegration complexes: studies of organization and composition. J Virol 71: 5382–5390

Ning J, Erdemci-Tandogan G, Yufenyuy EL, Wagner J, Himes BA, Zhao G, Aiken C, Zandi R, Zhang P (2016) In vitro protease cleavage and computer simulations reveal the HIV-1 capsid maturation pathway. Nat Commun 7: 13689

Novikova M, Adams LJ, Fontana J, Gres AT, Balasubramaniam M, Winkler DC, Kudchodkar SB, Soheilian F, Sarafianos SG, Steven AC, Freed EO (2018) Identification of a structural element in HIV-1 Gag required for virus particle assembly and maturation. MBio 9: e01567-01518

Nowicka-Sans B, Protack T, Lin Z, Li Z, Zhang S, Sun Y, Samanta H, Terry B, Liu Z, Chen Y, Sin N, Sit SY, Swidorski JJ, Chen J, Venables BL, Healy M, Meanwell NA, Cockett M, Hanumegoda U, Regueiro-Ren A, Krystal M, Dicker IB (2016) Identification and characterization of BMS-955176, a second-generation HIV-1 maturation inhibitor with improved potency, antiviral spectrum, and Gag polymorphic coverage. Antimicrob Agents Chemother 60: 3956–3969

Ono A, Ablan SD, Lockett SJ, Nagashima K, Freed EO (2004) Phosphatidylinositol (4, 5)-bisphosphate regulates HIV-1 Gag targeting to the plasma membrane. Proc Natl Acad Sci USA 101: 14889–14894

Pante N, Kann M (2002) Nuclear pore complex is able to transport macromolecules with diameters of about 39 nm. Mol Biol Cell 13: 425–434

Peng K, Muranyi W, Glass B, Laketa V, Yant SR, Tsai L, Cihlar T, Muller B, Krausslich HG (2014) Quantitative microscopy of functional HIV post-entry complexes reveals association of replication with the viral capsid. Elife 3: e04114

Perez-Caballero D, Hatziioannou T, Zhang F, Cowan S, Bieniasz PD (2005) Restriction of human immunodeficiency virus type 1 by

TRIM-CypA occurs with rapid kinetics and independently of cytoplasmic bodies, ubiquitin, and proteasome activity. J Virol 79: 15567–15572

Perrier M, Bertine M, Le Hingrat Q, Joly V, Visseaux B, Collin G, Landman R, Yazdanpanah Y, Descamps D, Charpentier C (2017) Prevalence of gag mutations associated with in vitro resistance to capsid inhibitor GS-CA1 in HIV-1 antiretroviral-naive patients. J Antimicrob Chemother 72: 2954–2955

Pornillos O, Ganser-Pornillos BK, Kelly BN, Hua Y, Whitby FG, Stout CD, Sundquist WI, Hill CP, Yeager M (2009) X-ray structures of the hexameric building block of the HIV capsid. Cell 137: 1282–1292

Pornillos O, Ganser-Pornillos BK, Yeager M (2011) Atomic-level modelling of the HIV capsid. Nature 469: 424–427

Price AJ, Jacques DA, McEwan WA, Fletcher AJ, Essig S, Chin JW, Halambage UD, Aiken C, James LC (2014) Host cofactors and pharmacologic ligands share an essential interface in HIV-1 capsid that is lost upon disassembly. PLoS Pathog 10: e1004459

Purdy MD, Shi D, Christowicz J, Hattné J, Gonen T, Yeager M (2018) MicroED structures of HIV-1 Gag CTD-SP1 reveal binding interactions with the maturation inhibitor bevirimat. Proc Natl Acad Sci USA 115: 13258–13263

Saad JS, Miller J, Tai J, Kim A, Ghanam RH, Summers MF (2006) Structural basis for targeting HIV-1 Gag proteins to the plasma membrane for virus assembly. Proc Natl Acad Sci USA 103: 11364–11369

Schulte B, Buffone C, Opp S, Di Nunzio F, De Souza Aranha Vieira DA, Brandariz-Nunez A, Diaz-Griffero F (2015) Restriction of HIV-1 requires the N-terminal region of MxB as a capsid-binding motif but not as a nuclear localization signal. J Virol 89: 8599–8610

Schur FK, Hagen WJ, Rumlova M, Ruml T, Muller B, Krausslich HG, Briggs JA (2015) Structure of the immature HIV-1 capsid in intact virus particles at 8.8 Å resolution. Nature 517: 505–508

Schur FK, Obr M, Hagen WJ, Wan W, Jakobi AJ, Kirkpatrick JM, Sachse C, Krausslich HG, Briggs JA (2016) An atomic model of HIV-1 capsid-SP1 reveals structures regulating assembly and maturation. Science 353: 506–508

Sowd GA, Serrao E, Wang H, Wang W, Fadel HJ, Poeschla EM, Engelman AN (2016) A critical role for alternative polyadenylation factor CPSF6 in targeting HIV-1 integration to transcriptionally active chromatin. Proc Natl Acad Sci USA 113: E1054–E1063

Spearman P (2016) HIV-1 Gag as an antiviral target: development of assembly and maturation inhibitors. Curr Top Med Chem 16: 1154–1166

Sticht J, Humbert M, Findlow S, Bodem J, Muller B, Dietrich U, Werner J, Krausslich HG (2005) A peptide inhibitor of HIV-1 assembly in vitro. Nat Struct Mol Biol 12: 671–677

Stremlau M, Owens CM, Perron MJ, Kiessling M, Autissier P, Sodroski J (2004) The cytoplasmic body component TRIM5alpha restricts HIV-1 infection in Old World monkeys. Nature 427: 848–853

Stultz RD, Cenker JJ, McDonald D (2017) Imaging HIV-1 genomic DNA from entry through productive infection. J Virol 91: e00034-00017

Sundquist WI, Krausslich HG (2012) HIV-1 assembly, budding, and maturation. Cold Spring Harb Perspect Med 2: a006924

Tang C, Loeliger E, Kinde I, Kyere S, Mayo K, Barklis E, Sun Y, Huang M, Summers MF (2003) Antiviral inhibition of the HIV-1 capsid protein. J Mol Biol 327: 1013–1020

Tang C, Loeliger E, Lunsford P, Kinde I, Beckett D, Summers MF (2004) Entropic switch regulates myristate exposure in the HIV-1 matrix protein. Proc Natl Acad Sci USA 101: 517–522

Tedbury PR, Freed EO (2015) HIV-1 gag: an emerging target for antiretroviral therapy. Curr Top Microbiol Immunol 389:171–201

Ternois F, Sticht J, Duquerroy S, Krausslich HG, Rey FA (2005) The HIV-1 capsid protein C-terminal domain in complex with a virus assembly inhibitor. Nat Struct Mol Biol 12:678–682

Tse WC, Link JO, Mulato A, Niedziela-Majka A, Rowe W, Somoza JR, Villasenor AG, Yant SR, Zhang JR, Zheng J (2017) Discovery of novel potent HIV capsid inhibitors with long-acting potential. In: Conference on retroviruses and opportunistic infections Abstract 38—new HIV drugs, formulations, combinations, and resistance

Urano E, Ablan SD, Mandt R, Pauly GT, Sigano DM, Schneider JP, Martin DE, Nitz TJ, Wild CT, Freed EO (2016) Alkyl amine bevirimat derivatives are potent and broadly active HIV-1 maturation inhibitors. Antimicrob Agents Chemother 60:190–197

Valle-Casuso JC, Di Nunzio F, Yang Y, Reszka N, Lienlaf M, Arhel N, Perez P, Brass AL, Diaz-Griffero F (2012) TNPO3 is required for HIV-1 replication after nuclear import but prior to integration and binds the HIV-1 core. J Virol 86:5931–5936

Vollmer B, Lorenz M, Moreno-Andres D, Bodenhofer M, De Magistris P, Astrinidis SA, Schooley A, Flotenmeyer M, Leptihn S, Antonin W (2015) Nup153 recruits the Nup107-160 complex to the inner nuclear membrane for interphasic nuclear pore complex assembly. Dev Cell 33:717–728

von Schwedler U, Kornbluth RS, Trono D (1994) The nuclear localization signal of the matrix protein of human immunodeficiency virus type 1 allows the establishment of infection in macrophages and quiescent T lymphocytes. Proc Natl Acad Sci USA 91:6992–6996

von Schwedler UK, Stemmler TL, Klishko VY, Li S, Albertine KH, Davis DR, Sundquist WI (1998) Proteolytic refolding of the HIV-1 capsid protein amino-terminus facilitates viral core assembly. EMBO J 17:1555–1568

Wagner JM, Zadrozny KK, Christowicz J, Purdy MD, Yeager M, Ganser-Pornillos BK, Pornillos O (2016) Crystal structure of an HIV assembly and maturation switch. Elife 5:e17063

Wagner JM, Christensen DE, Bhattacharya A, Dawidziak DM, Roganowicz MD, Wan Y, Pumroy RA, Demeler B, Ivanov DN, Ganser-Pornillos BK, Sundquist WI, Pornillos O (2018) General model for retroviral capsid pattern recognition by TRIM5 proteins. J Virol 92:e01563-01517

Waki K, Durell SR, Soheilian F, Nagashima K, Butler SL, Freed EO (2012) Structural and functional insights into the HIV-1 maturation inhibitor binding pocket. PLoS Pathog 8:e1002997

Walde S, Thakar K, Hutten S, Spillner C, Nath A, Rothbauer U, Wiemann S, Kehlenbach RH (2012) The nucleoporin Nup358/RanBP2 promotes nuclear import in a cargo- and transport receptor-specific manner. Traffic 13:218–233

Walsh D, Naghavi MH (2019) Exploitation of cytoskeletal networks during early viral infection. Trends Microbiol 27:39–50

Walther TC, Pickersgill HS, Cordes VC, Goldberg MW, Allen TD, Mattaj IW, Fornerod M (2002) The cytoplasmic filaments of the nuclear pore complex are dispensable for selective nuclear protein import. J Cell Biol 158:63–77

Wang W, Zhou J, Halambage UD, Jurado KA, Jamin AV, Wang Y, Engelman AN, Aiken C (2017) Inhibition of HIV-1 maturation via small-molecule targeting of the amino-terminal domain in the viral capsid protein. J Virol 91:e02155-02116

Woodward CL, Cheng SN, Jensen GJ (2015) Electron cryotomography studies of maturing HIV-1 particles reveal the assembly pathway of the viral core. J Virol 89:1267–1277

Wright ER, Schoeler JB, Ding HJ, Kieffer C, Fillmore C, Sundquist WI, Jensen GJ (2007) Electron cryotomography of immature HIV-1 virions reveals the structure of the CA and SP1 Gag shells. EMBO J 26:2218–2226

Xu H, Franks T, Gibson G, Huber K, Rahm N, Strambio De Castillia C, Luban J, Aiken C, Watkins S, Sluis-Cremer N, Ambrose Z (2013) Evidence for biphasic uncoating during HIV-1 infection from a novel imaging assay. Retrovirology 10:70

Yamashita M, Emerman M (2004) Capsid is a dominant determinant of retrovirus infectivity in nondividing cells. J Virol 78:5670–5678

Yamashita M, Engelman AN (2017) Capsid-dependent host factors in HIV-1 infection. Trends Microbiol 25:741–755

Yamashita M, Perez O, Hope TJ, Emerman M (2007) Evidence for direct involvement of the capsid protein in HIV infection of nondividing cells. PLoS Pathog 3:1502–1510

Yap MW, Dodding MP, Stoye JP (2006) Trim-cyclophilin A fusion proteins can restrict human immunodeficiency virus type 1 infection at two distinct phases in the viral life cycle. J Virol 80:4061–4067

Zhang H, Zhao Q, Bhattacharya S, Waheed AA, Tong X, Hong A, Heck S, Curreli F, Goger M, Cowburn D, Freed EO, Debnath AK (2008) A cell-penetrating helical peptide as a potential HIV-1 inhibitor. J Mol Biol 378:565–580

Zhang W, Cao S, Martin JL, Mueller JD, Mansky LM (2015) Morphology and ultrastructure of retrovirus particles. AIMS Biophys 2:343–369

Zhao G, Perilla JR, Yufenyuy EL, Meng X, Chen B, Ning J, Ahn J, Gronenborn AM, Schulten K, Aiken C, Zhang P (2013) Mature HIV-1 capsid structure by cryo-electron microscopy and all-atom molecular dynamics. Nature 497:643–646

Zhou J, Yuan X, Dismuke D, Forshey BM, Lundquist C, Lee KH, Aiken C, Chen CH (2004) Small-molecule inhibition of human immunodeficiency virus type 1 replication by specific targeting of the final step of virion maturation. J Virol 78:922–929

Zhou L, Sokolskaja E, Jolly C, James W, Cowley SA, Fassati A (2011) Transportin 3 promotes a nuclear maturation step required for efficient HIV-1 integration. PLoS Pathog 7:e1002194
