
Cell

Cryo-EM Structure of Chikungunya Virus in Complex with the Mxra8 Receptor

### Graphical Abstract
![Graphical Abstract Image](image_url)

### Authors
Katherine Basore, Arthur S. Kim, Christopher A. Nelson, ..., Jonathan Smith, Michael S. Diamond, Daved H. Fremont

### Correspondence
diamond@wusm.wustl.edu (M.S.D.), fremont@wustl.edu (D.H.F.)

### In Brief
Basore et al. describe an X-ray crystal structure of Mxra8, an entry receptor for arthritogenic alphaviruses, and cryo-EM structures of Mxra8 bound to CHIKV. Mxra8 has an unusual Ig-like domain architecture with a head-to-head arrangement, with domain 1 emanating from a loop of domain 2. Mxra8 binds into a complex quaternary cleft formed between two E2-E1 heterodimers within a trimeric spike, while also contacting E1 from a neighboring spike. Mxra8 occupancy is lower when E3 protein is retained on the virion.

### Highlights
- A 2.2 Å resolution X-ray structure of Mxra8 shows an unusual Ig-like fold topology
- Mxra8 has a complex quaternary binding site that spans CHIKV E2 and E1 proteins
- Mutational and epitope analyses support cryo-EM structures of Mxra8 with CHIKV
- The presence of CHIKV E3 protein on the virion affects the mode of Mxra8 binding

---

Basore et al., 2019, Cell 177, 1725–1737  
June 13, 2019 © 2019 Elsevier Inc.  
https://doi.org/10.1016/j.cell.2019.04.006

Article

# Cryo-EM Structure of Chikungunya Virus in Complex with the Mxra8 Receptor

Katherine Basore,$^{1}$ Arthur S. Kim,$^{1,2}$ Christopher A. Nelson,$^{1,2}$ Rong Zhang,$^{2}$ Brittany K. Smith,$^{1}$ Carla Uranga,$^{6}$ Lo Vang,$^{6}$ Ming Cheng,$^{3}$ Michael L. Gross,$^{3}$ Jonathan Smith,$^{6}$ Michael S. Diamond,$^{1,2,4,*}$ and David H. Fremont$^{1,4,5,7,*}$

$^{1}$Department of Pathology & Immunology, Washington University School of Medicine, Saint Louis, MO 63110, USA  
$^{2}$Department of Medicine, Washington University School of Medicine, Saint Louis, MO 63110, USA  
$^{3}$Department of Chemistry, Washington University, Saint Louis, MO 63110, USA  
$^{4}$Department of Molecular Microbiology, Washington University School of Medicine, Saint Louis, MO 63110, USA  
$^{5}$Department of Biochemistry & Molecular Biophysics, Washington University School of Medicine, Saint Louis, MO 63110, USA  
$^{6}$PaxVax, San Diego, CA 92121, USA  
$^{7}$Lead Contact  

*Correspondence: diamond@wusm.wustl.edu (M.S.D.), fremont@wustl.edu (D.H.F.)  
https://doi.org/10.1016/j.cell.2019.04.006

## SUMMARY

Mxra8 is a receptor for multiple arthritogenic alphaviruses that cause debilitating acute and chronic musculoskeletal disease in humans. Herein, we present a 2.2 Å resolution X-ray crystal structure of Mxra8 and 4 to 5 Å resolution cryo-electron microscopy reconstructions of Mxra8 bound to chikungunya (CHIKV) virus-like particles and infectious virus. The Mxra8 ectodomain contains two strand-swap-ped Ig-like domains oriented in a unique disulfide-linked head-to-head arrangement. Mxra8 binds by wedging into a cleft created by two adjacent CHIKV E2-E1 heterodimers in one trimeric spike and engaging a neighboring spike. Two binding modes are observed with the fully mature VLP, with one Mxra8 binding with unique contacts. Only the high-affinity binding mode was observed in the complex with infectious CHIKV, as viral maturation and E3 occupancy appear to influence receptor binding-site usage. Our studies provide insight into how Mxra8 binds CHIKV and creates a path for developing alphavirus entry inhibitors.

## INTRODUCTION

Alphaviruses are positive-sense, single-stranded, enveloped RNA viruses and are among the most important arthropod-borne viruses causing disease in humans (Powers et al., 2001). This genus includes chikungunya (CHIKV), Mayaro (MAYV), O’nyong’nyong (ONNV), and Ross River (RRV) viruses, which are emerging beyond their historical boundaries and now cause debilitating acute and chronic polyarthritis, affecting millions of people in Africa, Asia, Europe, and the Americas. Despite their epidemic potential, there are no specific therapies or licensed vaccines for any alphavirus infection.

Alphavirus genomes encode four non-structural and five structural proteins. The non-structural proteins are required for virus replication, protein modification, and immune evasion. The structural proteins (capsid [C]-envelope [E]3-E2-6K-E1) are synthesized from a subgenomic promoter and cleaved co- and post-translationally. The E1 envelope glycoprotein participates in cell fusion (Lescar et al., 2001), whereas the E2 envelope glycoprotein binds to entry factors (Smith et al., 1995; Zhang et al., 2005) and initiates clathrin-dependent endocytosis (DeTulleo and Kirchhausen, 1998; Lee et al., 2013; Ooi et al., 2013). The E3 protein is essential for the proper folding of p62 (precursor to E2) and the formation of the p62-E1 heterodimer (Carleton et al., 1997; Mulvey and Brown, 1995) but is cleaved by furin-like proteases during the maturation process in the trans-Golgi network (Heidner et al., 1996). Mature alphaviruses are ~700-Å-diameter icosahedral particles that assemble at the plasma membrane and contain a lipid bilayer with 240 embedded E2-E1 heterodimers assembled into 80 trimeric spikes with T = 4 icosahedral symmetry (Cheng et al., 1995; Kos-tyuchenko et al., 2011; Paredes et al., 1993) and a nucleocapsid containing a single copy of genomic RNA.

Crystallographic studies of the precursor p62-E1, the mature E2-E1 glycoprotein complex, and the E1 protein (Lescar et al., 2001; Li et al., 2010; Roussel et al., 2006; Voss et al., 2010) have elucidated the glycoprotein structures. Several alphavirus virions structures also were described by cryo-electron microscopy (cryo-EM) (Chen et al., 2018; Cheng et al., 1995; Hasan et al., 2018; Kostyuchenko et al., 2011; Lee et al., 1998; Li et al., 2010; Pletnev et al., 2001; Zhang et al., 2011, 2002). The E1 ectodomain consists of three β-barrel domains termed Domain I (DI), DII, and DIII. The fusion peptide is located at the distal end of DII. E1 monomers lie at the base of the surface spikes and form trimers surrounding the icosahedral axes. E2 localizes to a long, thin, leaf-like structure on the top of the spike. The mature E2 protein contains three domains with immunoglobulin (Ig)-like folds: the N-terminal domain A, located at the center; domain B at the lateral tip; and the C-terminal domain C, located close to E1 and the viral membrane.

We recently used a genome-wide CRISPR/Cas9-based screen to identify mouse Mxra8, a cell adhesion molecule containing two Ig-like domains, as a receptor for multiple arthritogenic alphaviruses, including CHIKV, RRV, MAYV, and ONNV.
(Zhang et al., 2018). Importantly, the human ortholog MXRA8 also bound to CHIKV and other alphaviruses, and its cell surface expression was required for efficient infection of primary human target cells, including fibroblasts, skeletal muscle cells, and chondrocytes. Mxra8 bound directly to CHIKV particles and enhanced attachment and internalization into cells, and Mxra8-Fc fusion protein or anti-Mxra8 monoclonal antibodies (mAbs) blocked CHIKV infection of several cell types. Administration of Mxr8a-Fc protein or anti-Mxra8 blocking mAbs to mice reduced CHIKV or ONNV infection and associated joint swelling. Despite defining several biological characteristics of CHIKV interaction with this receptor, structural insight as to how Mxra8 engages the alphavirus spike proteins on the virion is lacking.

To date, there has been limited structural information of the binding of receptors to icosahedral enveloped viruses. Here, we describe the X-ray crystal structure of the Mxra8 ectodomain and cryo-EM reconstructions of CHIKV virus-like particles (VLPs, produced as capsid-E3-E2-6K-E1 but lacking viral RNA) and fully infectious CHIKV in complex with Mxra8. Mxra8 has an unusual architecture, as its two Ig-like domains are oriented in a disulfide-linked head-to-head arrangement, with domain 1 inserted into a loop of domain 2. Mxra8 binds into a cleft formed between two CHIKV E2-E1 heterodimers within a trimeric spike and also extends to engage an adjacent trimeric spike. Two binding modes for Mxra8 were observed with mature CHIKV VLPs, which is consistent with a high- and low-affinity binding-site model supported by surface plasmon resonance measurements. The low-affinity binding sites, however, were sterically obscured by the retention of E3 on infectious CHIKV. Overall, our structural analysis defines how CHIKV engages its receptor Mxra8 to facilitate attachment and infection of cells. This information may inform the basis of therapies and improved vaccine designs that mitigate disease of multiple emerging alphaviruses.

SEC-MALS analysis revealed that cleaved mammalian-cell-generated Mxra8 is monomeric, with a molecular weight of 38 kDa (Figure S1D), and N-terminal sequencing using Edman degradation showed that the mature N terminus of the protein begins at position 23 (Figure S2). These preparations of Mxra8 were used for the crystallographic and cryo-EM studies described below.

Previous studies have obtained high-resolution structural information about how neutralizing antibodies engage CHIKV using VLPs, which contain the structural proteins but lack infectious viral RNA and can be imaged under lower biosafety conditions (Long et al., 2015; Sun et al., 2013). To begin to define the structural basis for interaction between Mxra8 and arthritogenic alphaviruses (Zhang et al., 2018), we produced CHIKV VLPs (strain 37997) after transient transfection of HEK-293 cells with a plasmid encoding C-E3-E2-6K-E1 (Akahata et al., 2010); equivalent preparation of VLPs were used in phase 1 and 2 human clinical trials for vaccine protection against CHIKV disease (Chang et al., 2014; NCT02562482 and NCT03483961). Soluble VLPs were collected, buffer exchanged, and monitored for purity by SDS-PAGE and antigenicity by western blotting with anti-capsid and anti-envelope-protein antibodies (Figures S1E and S1F). Dynamic light scattering experiments revealed a particle diameter of ∼680 Å (Figure S1G), which is close to the expected external diameter of 700 Å for a CHIKV virion (Cheng et al., 1995).

### X-Ray Crystallographic Structure of Mxra8

Diffraction quality crystals of bacterially produced murine Mxra8 (ectodomain residues 23 to 296) were obtained in space group I222 with cell dimensions a = 72.59 Å, b = 142.85 Å, and c = 195.59 Å. There are two copies of Mxra8 in the asymmetric unit. The structure was solved by molecular replacement using junctional adhesion molecule-like protein (JAML; PDB: 3MJ6) as a model. There was visible density for all but the first nine and last two residues of chain A and all but the first seven and last two residues of chain B. The final statistics for the Mxra8 model, refined to a resolution of 2.2 Å, are provided in Table S1. Mxra8 consists of two Ig-V-like domains arranged in a head-to-head orientation, with the CDR-like loops pointing towards one another. The two FG-loops (CDR3-like) are connected by a disulfide bond (Figures 1A–1C). Unexpectedly, the first two β-strands of each domain are swapped, such that the full domain 1 is seen as an insertion into the BC-loop (CDR1-like) of domain 2. Domains 1 and 2 are similar in sequence (Figure S2) and nearly identical in fold, with an average root-mean-square deviation (RMSD) of 0.90 Å over 68 Cα positions for chain A and 1.13 Å over 67 positions for chain B in the crystal structure. In comparison, the two domain 1 structures in the asymmetric unit align with an RMSD of 0.4 Å over 110 residues, whereas the two domain 2 structures align with an RMSD of 1.0 Å over 104 residues. The hinge connecting the two domains is flexible, and comparison of the two independent Mxra8 structures indicates that domain 1 can move by at least ∼43° relative to D2 (Figure S3) (Hayward and Berendzen, 1998).

### Cryo-EM Reconstruction of Mxra8 Bound to CHIKV VLPs

Electron micrographs of CHIKV VLPs with or without bound mammalian-cell-derived Mxra8 were recorded using a 300

### RESULTS

#### Mxra8 Protein and CHIKV VLPs

The soluble ectodomain of murine Mxra8 (amino acids 23 through 296) was produced as *E. coli* inclusion bodies that were oxidatively refolded (Figure S1A). Size exclusion chromatography coupled with multi-angle light scattering (SEC-MALS) analysis reveals that refolded Mxra8 is monomeric with a molecular weight of 31 kDa (Figure S1B). Each Mxra8 monomer contains three disulfide linkages, as indicated by a monoisotopic molecular weight of 31,092.45 determined by electrospray mass spectrometry. This value corresponds to the predicted weight of the native expression construct, minus the first methionine removed by aminopeptidase and six hydrogens lost during formation of the three disulfide bonds. We also produced a mammalian-expressed soluble form of Mxra8 by engineering a Mxra8-Fc fusion protein (Zhang et al., 2018) with a human rhinovirus 3C (HRV) protease site between the C terminus of Mxra8 and the Fc domain. After production of Mxra8-Fc fusion protein in human embryonic kidney (HEK)-293F cells and purification by protein A affinity chromatography, we cleaved the Fc domain with HRV and isolated Mxra8 in the flowthrough of a second round of protein A agarose affinity chromatography (Figure S1C).
Figure 1. Mxra8 Crystal Structure and Topology Diagram

(A) Ribbon model of the mouse Mxra8 protein structure. The β-strands are labeled following standard convention. The two Ig-like domains are colored from the N to C terminus in a rainbow spectrum of blue to red, with the disulfides shown as yellow ball and stick bonds. The positions of the cysteines forming the interdomain disulfide bond are indicated. The N and C termini are labeled in lowercase.

(B) Secondary structure diagram of Mxra8, numbered by sequence positions in the X-ray structure. Colored arrows indicate the β-strands as in (A). Yellow lines connect the cysteine positions forming disulfide bonds. The AB strand swaps anchor the connections between the D1 and D2 domains.

(C) Cartoon schematic of Mxra8, with labeled D1 and D2 domains, N and C termini, B-C linkers, and cysteine residues forming the interdomain disulfide bond. Mxra8 is bowed. A rotation of ~155° around the hinge region is required for alignment of the two domains.

See Figures S1, S2, and S3 and Table S1.

kV Titan Krios cryo-electron microscope with Gatan K2 detector at the Washington University Center for Cellular Imaging (WUCCI) (Figure 2A; Table S2). The images were corrected for beam-induced motion using MotionCor2 (Zheng et al., 2017). Particles were auto-picked from the micrographs and subjected to two-dimensional classification to remove ice contamination and debris (Figure 2B). The remaining particles underwent ab initio three-dimensional reconstruction and refinement in cisTEM (Grant et al., 2018). The reconstructions of CHIKV VLP alone and with Mxra8 were determined at overall resolutions of 4.16 Å and 4.06 Å (Figures 2C and 2D), respectively, based on Fourier Shell Correlation (FSC) analysis (Figures 2E and 2F).

The three-dimensional reconstructions showed no large conformational changes in the structure of the CHIKV VLP when Mxra8 is bound (Figures 3A and 3B). Despite the different chemical environments of the structural glycoproteins within the asymmetric unit of the particle, Mxra8 binds E2-E1 heterodimers in a 1:1 ratio, with 240 Mxra8 molecules bound per virion. The local resolution of the reconstruction of CHIKV VLP-Mxra8 complex indicates that the membrane-proximal regions of the E2-E1 heterodimers were the best resolved, with an estimated resolution of 3.9 to 5.4 Å for E1 ectodomains. In comparison, the radially distant trimeric spikes and Mxra8 were resolved less well, with an estimated resolution of 4.2 to 8.0 Å for Mxra8 chains (Figures 3C and 3F). Mxra8 binds to the CHIKV VLP with a complex quaternary epitope, wedging into a cleft created by two E2-E1 heterodimers in one trimeric spike, effectively wrapping around the distal end of one E2-E1 heterodimer (herein referred to as “wrapped”) and contacting an adjacent heterodimer within the same trimeric spike (the “intraspoke” heterodimer). Mxra8 also engages another heterodimer on a neighboring spike (the “interspike” heterodimer) (Figures 3D and 3E).

### Building and Refinement of Cryo-EM Structural Models

We iteratively built an atomic model of the CHIKV VLP structural proteins and Mxra8 using a combination of published CHIKV crystal structures and de novo modeling. As a starting point,

Figure 2. Two-Dimensional cisTEM Analysis of CHIKV Particles with or without Mxra8 Bound

(A) Raw electron micrographs of CHIKV VLPs.

(B) Two-dimensional classification scheme for binning of CHIKV VLPs.

(C and D) Two-dimensional equatorial slices of CHIKV VLP alone (C) or CHIKV VLP in complex with Mxra8 (D). Dimensions: the outer radius of the nucleocapsid shell (~180 Å), lipid bilayer (~240 Å), E1 protein glycoprotein shell (~280 Å), E2 protein spike (~340 Å), and bound Mxra8 (~350 Å) from the viral center.

(E and F) Fourier shell correlation (FSC) plot versus resolution for CHIKV VLP alone (E) or CHIKV VLP with bound Mxra8 (F).

See Figure S1 and Table S2.

Mxra8 engages the A and B domains of E2 (residues 18, 26–29, 71–72, 74–76, 119–121, 123, 178–182, 189, 191, 193, 212–214, and 221–223), the fusion loop in E1 (83, 85, and 87–91), and domain II of E1 (223 and 226–227). On the intra-spike heterodimer, Mxra8 interacts with domain A and the β-linker of E2 (residues 5–6, 62, 64, 144, 150, 157–160, 263–265, and 267). Mxra8 at site 1 also makes unique contacts to domains I and II of E1 (residues 37, 130, 132, 142–143, 145–147, 152, 154, 156–157, 263–265, and 267); these interactions were not observed at sites 2, 3, and 4. In contrast, at all binding positions except site 1, Mxra8 makes additional contacts with domain II of E1 (residues 71–74, 112, 115, 206–207, and 209–212) of the inter-spike heterodimer (Figures 4C, S4, S5, and S6; Table S4). The average overall binding interface at each of the four sites in an asymmetric unit is ~2,100 Å². The largest contributing interface is between Mxra8 and the wrapped heterodimer (~1,200 Å²), followed by the intraspikes heterodimer (~600 Å²) and the interspike heterodimer (~300 Å²).

Mxra8 residues at the binding interface also were identified in our model. Both D1 and the hinge region contribute to the binding of the wrapped heterodimer (residues 56–57, 63–66, 71, 73, 82–86, 89, 91–99, 104, 137, 139, 141, 144, 146, 148, 247, and 250–251). To the intraspikes heterodimer, E2 is contacted by amino acids in D1 and the hinge region of Mxra8 at all binding sites (residues 60, 62–63, 112, 114–115, 190, 193–196, 198, 200, and 227–229), and E1 is contacted by only D2 residues of Mxra8 at binding site 1 (46, 230–232, 236–237, 239–240, 260, and 262). To the interspike heterodimer, only D2 amino acids of Mxra8 at binding sites 2, 3, and 4 are at the binding interface (residues 45, 208, 210–213, 217, 219, 235–237, 241, 262–264, 266, and 290–291) (Figures 4C and S2; Table S4).

To evaluate our docking and modeling of Mxra8, we first assessed the general fit of the atomic coordinates and searched

1728 Cell 177, 1725–1737, June 13, 2019

Figure 3. Cryo-EM Reconstruction of CHIKV Particles with or without Mxra8 Binding

(A–C) Paired, colored surface representations (top) and equatorial cross-sections (bottom) of CHIKV VLP (A), CHIKV VLP + Mxra8 (B), and local resolution of CHIKV VLP + Mxra8 (C). The white triangle indicates one icosahedral asymmetric unit. The 5-fold (i5), 3-fold (i3), and 2-fold (i2) icosahedral axes of symmetry are indicated with a pentagon, triangles, and an oval, respectively. Trimeric spikes are labeled “i3” if coincident with the i3 axes and “q3” if on a quasi-3-fold axis. Black arrows: directions of icosahedral symmetry axes (i2, i3, q3, and i5). (A and B) Radial distance color scheme: red, electron dense core and RNA; yellow, capsid; green, membrane lipid; dark blue, E1; cyan, E2 spike; and magenta, Mxra8. Scale bars: 100 Å.

(D) Zoomed-in view of the q3 spike, highlighting the three E2-E1 heterodimers that interact with Mxra8 at site 3: the wrapped (light gray), the intraspikes (medium gray), and the interspike (dark gray) heterodimers. The i5 and i2 axes of symmetry are labeled with a pentagon and an oval, respectively. Mxra8 is colored by domain (D2, dark magenta; D1, light magenta).

(E and F) Side views of Mxra8 at site 3. (E) Mxra8 and the wrapped, intraspikes, and interspike heterodimers are colored to match (D). (F) Mxra8 and the wrapped, intraspikes, and interspike heterodimers are colored by local resolution, as reported in (C): red, 4 Å; yellow, 6 Å; and green, 8 Å.

See Tables S2 and S3.

Figure 4. Atomic Model of Mxra8 Interaction with CHIKV

(A) The refined model of Mxra8 and CHIKV structural proteins (E1, E2, transmembrane [TM] helices, and Capsid) in the electron density map of the CHIKV VLP with Mxra8 reconstruction at threshold σ = 1, viewed from the side (left) and top (right) of the asymmetric unit.

(B) Cartoon of the asymmetric unit viewed from the side (left) and top (right) with all domains labeled. “i3” and “q3” refer to the icosahedral and quasi-3-fold axes, respectively.

(C) Individual E2-E1 subunits at the binding interface, specifically site 1 Mxra8 and its wrapped heterodimer (left), site 1 Mxra8 with its intraspike heterodimer (middle), and site 3 Mxra8 with its interspike heterodimer (right). Mxra8 and structural proteins are colored by domain. Mxra8: dark magenta, D2; light magenta, D1. E1: DI, light gray; DII, medium gray; DIII, dark gray; fusion loop, orange; TM region, black. E2: A domain, light cyan; β-linker, medium cyan; domain B, dark cyan; domain C, medium blue; TM region, dark blue. Capsid, forest green. Domain labels are boxed if the domain is at the binding interface. Disulfide bonds are shown as yellow balls and sticks.

(D) Ribbon model of the refined site 1 Mxra8 in its electron density map, with the N to C terminus in a rainbow spectrum of blue to red and the disulfide bonds shown as yellow balls and sticks. The N and C termini are labeled in lowercase and the β-strands are labeled in uppercase. The density map is viewed at contour level σ = 1.7.

(E) Zoomed-in view of the protrusion density and atomic structure of the N-linked glycan at residue 118, viewed at contour level σ = 1.4. The σ value is the standard deviation of density values above the mean in the map.

See Figures S2, S3, and S6 and Table S4.
this allowed us to model 10 additional residues at these sites past the G strand of D2 that is beyond the register of the crystal structure.

In support of our model, biolayer interferometry (BLI) competition binding experiments revealed that Mxra8 cannot bind to CHIKV VLPs in the presence of a subset of previously described anti-Mxra8 mAbs that inhibit infection ([Zhang et al., 2018](https://doi.org/10.1038/s41594-018-0076-0) (Figure 5A). Epitope mapping via hydrogen-deuterium-exchange mass spectrometry analysis shows that competing mAbs bind to Mxra8 at its interface with CHIKV, whereas non-competing mAbs bind at other sites (Figures 5B, S7A, and S7B). We observed competitive inhibition with mAbs 4E7.D10 and 8F7.E1, which map to an epitope in D1 of Mxra8 in residues adjacent to the E2 A domain of the wrapped heterodimer, and mAb 1G11.E6, whose epitope in D2 residues is adjacent to the E1 fusion loop of the wrapped heterodimer. In contrast, mAbs 1H1.F5 and 3G2.F5, which do not compete in the BLI assay, map to predominantly solvent-exposed peptides in D1 near the interspike heterodimer. If the orientations of the Mxra8 Ig-like domains were inverted, the epitope of mAbs 1H1.F5 and 3G2.F5 would be inaccessible due to wrapped heterodimer contacts with E2 domain B and the E1 fusion loop. The alternate domain-flipped model also is not consistent with the observed ability of mAb 1G11.E6 to compete for virus binding (Figures S7C and S7D).

As another test of the model, charge-reversal point mutations were introduced in Mxra8 at wrapped heterodimer interface residues with high percentages of buried surface area, as well as other control residues not predicted to be part of the binding interface (Figure S2). Wild-type (WT) and mutant Mxra8 cDNAs were transfected into 293T cells, which lack expression of endogenous human MXRA8 ([Zhang et al., 2018](https://doi.org/10.1038/s41594-018-0076-0)), and tested for their surface expression and ability to support CHIKV infection. Several mutations (e.g., A49R, V88R, A93R, G117R, and F119R) in Mxra8 affected surface expression and were not analyzed further. For mutations with WT levels of Mxra8 surface expression, three substitutions (D89R, G94R, and R97E) resulted in marked reductions in CHIKV infection, and two other residues (V98R and Y99R) showed smaller yet reproducible decreases (Figure 5C). All five of these residues are at the interface of domain A of E2 of the wrapped site (Figure 5D). Thus, the binding competition and mapping results support the identified residues at the Mxra8 interface, as well as the positioning of D1 and D2 in our electron density map.

For the interface on CHIKV (Figure 5E), many of the identified E2 contact residues are supported by coincident antibody mapping and mutagenesis data (Figure S5; Table S4). Several neutralizing human mAbs against CHIKV that disrupt Mxra8 binding ([Zhang et al., 2018](https://doi.org/10.1038/s41594-018-0076-0)) map to epitopes in the A domain of E2 ([Long et al., 2015](https://doi.org/10.1016/j.chom.2015.05.010); [Smith et al., 2015](https://doi.org/10.1016/j.chom.2015.05.011)) that are shared with the Mxra8 binding site (Figure 5F). Previous alanine scanning mutagenesis mapping data identified four solvent-exposed residues in domain A of E2 as important for optimal Mxra8 binding ([Zhang et al., 2018](https://doi.org/10.1038/s41594-018-0076-0)); E2 residues D71, T116, and I121 are at the wrapped interface, whereas W64 contacts Mxra8 at the intra-spike interface in our model (Figure 5G). In the original study, mutations in E1 were not evaluated for binding to Mxra8 because it was expected that E2 and not E1 contributed to

receptor engagement ([Strauss et al., 1994](https://doi.org/10.1016/0092-8674(94)90030-6); [Tucker and Griffin, 1991](https://doi.org/10.1016/0092-8674(91)90536-8)).

### Modes of Mxra8 Binding to CHIKV VLPs and Infectious Virus

We also generated a 4.99 Å cryo-EM reconstruction of CHIKV infectious particles with Mxra8 and built an atomic model (Figure 6A; Tables S2 and S3). In contrast to the CHIKV VLP, strong electron density was seen for the E3 protein in all four chemical environments of the infectious CHIKV (Figures 6B and 6C), which, similarly, was observed in other infectious and non-infectious alphavirus particle preparations ([Wu et al., 2008](https://doi.org/10.1073/pnas.0800589105); [Yap et al., 2017](https://doi.org/10.1073/pnas.1700510114); [Zhang et al., 2011](https://doi.org/10.1073/pnas.1100894108)). In the CHIKV infectious particles, weaker electron density was seen for Mxra8 compared to the CHIKV VLP reconstruction. Only one of the four potential binding sites in the asymmetric unit showed interpretable Mxra8 density (Figure 6B). This suggested there might be a lower occupancy of Mxra8 binding on the infectious particle compared to that on the fully mature VLP, potentially due to steric hindrance by the residual E3 protein in three of four chemical environments. To test this hypothesis, we docked in the difference map of Mxra8 from the mature Mxra8-VLP complex onto our Mxra8-bound CHIKV infectious virus reconstruction and assessed for clashes. E3 appears to sterically obstruct Mxra8 binding at the i3 (site 4) and two of three of the q3 binding sites (sites 2 and 3), allowing for 60 binding sites per virion at site 1 (Figure S8). We note that electron density for Mxra8 on the CHIKV VLP is strongest at site 1, which corresponds to the Mxra8 binding site on the infectious virion (Figure 6C). This finding is consistent with our binding analysis of Mxra8 to CHIKV VLPs by surface plasmon resonance, where the raw traces did not fit well to kinetic and equilibrium 1:1 binding models (Figure 6D). One explanation for the higher occupancy of Mxra8 at one site in an asymmetric unit is a site-specific higher binding affinity, due to the slightly different chemical environments of each of the four E2-E1 heterodimers within the asymmetric unit. We generated a model assuming one high-affinity site and three lower but equal affinity sites (see STAR Methods); this model produced a substantially better fit to our surface plasmon resonance (SPR) binding data (Figure 6E). Mxra8 binds the high-affinity site with a kinetically derived K<sub>D</sub> of ~84 nM and the lower affinity sites with a K<sub>D</sub> of ~270 nM. Similar SPR measurements and two-site binding analysis were performed for human monomeric MXRA8 (as determined by SEC-MALS), which is 78% identical to the murine protein (Figures S2 and S9).

### DISCUSSION

Cell culture infection experiments with mouse and human cells and in vivo pathogenesis studies in mice defined Mxra8 as a cell surface receptor required for optimal infectivity and induction of musculoskeletal disease by multiple arthritogenic alphaviruses ([Zhang et al., 2018](https://doi.org/10.1038/s41594-018-0076-0)). Here, our X-ray crystallography and single-particle cryo-EM analyses of Mxra8, VLPs, and infectious virus provide structural insight into how CHIKV engages Mxra8 to facilitate interactions with target cells. Our study adds to the limited structural knowledge of how receptors bind to enveloped icosahedral virions. Only one other structure of an enveloped,
Figure 5. Binding Competition, Epitope Mapping, and Mutagenesis Supporting the Mxra8-CHIKV Cryo-EM Model

(A) Competition BLI assay traces. CHIKV VLP was captured on the biosensor with an anti-CHIKV mAb and then dipped into wells containing Mxra8 ectodomain alone or with an anti-Mxra8 mAb or isotype control mAb. One experiment of three is shown.

(B) Surface and ribbon diagrams of Mxra8 at site 3 and the E2-E1 domains at its binding interface. Structural proteins are colored by domain. E1: DI, light gray; DII, medium gray; fusion loop, orange. E2: A domain, light cyan; β-linker, medium cyan; domain B, dark cyan. ' denotes domains within the intraspike heterodimer and " for the interspike heterodimer; the wrapped heterodimer is labeled without symbols. Surface representation of Mxra8 models are colored with the HDX-mapped epitopes of anti-Mxra8 mAbs, where shades of violet correspond to mAb 1G11.E6, green for mAbs 1H1.F5 and 3G2.F5, and yellow for mAbs 4E7.D10 and 8F7.E1.

(C) Trans-complementation of 293T cells (which lack endogenous MXRA8) with charge-reversal mutated Mxra8. (Top) CHIKV infection was determined by flow cytometric analysis of intracellular E2 protein. (Bottom) Relative surface expression of Mxra8 by flow cytometry using anti-Mxra8 mAbs. Data are pooled from three experiments performed in triplicate and normalized to the WT controls, and the mean ± SD values are shown (one-way ANOVA with Dunnett's post-test: *p < 0.05; ****p < 0.0001).

(D) Ribbon diagram of Mxra8 model at site 1 with side chains of mutated residues shown as balls and sticks. Mutations that result in statistically significant decreases in CHIKV infection are colored. Inset: zoomed-in view of the surface representation of Mxra8 at the interface. Side chains of residues of E2 (cyan ribbon) at this interface are displayed as balls and sticks.
icosahedral virus complexed with its receptor exists. In a 25 Å resolution cryo-EM structure, Dengue virus (DENV) was complexed with the carbohydrate recognition domain of DC-SIGN; the only contacts were with protruding N-linked glycans in domain II of the E protein ([Pokidycheva et al., 2006](https://doi.org/10.1074/jbc.M511893200)). In contrast, we observed a complex network of quaternary protein-protein interactions with Mxra8 engaging two E2-E1 heterodimers within one trimeric spike, as well as another heterodimer on a neighboring spike. The specific binding determinants we observed are supported by our previous mutagenesis analysis of E2, structure-guided mutations that we introduced into Mxra8, and epitope maps of mAbs against CHIKV and Mxra8 that directly block virus-receptor interactions. Our structures indicate that Mxra8 can bind at four distinct sites in the icosahedral asymmetric unit of the CHIKV VLP but only one site in the infectious virus, which retains E3.

The quaternary interactions formed between Mxra8 and multiple envelope proteins would effectively cross-link CHIKV spikes in a manner analogous to a previously defined broadly neutralizing mAb (CHK-265) that binds domain B on one trimer and domain A on an adjacent spike ([Fox et al., 2015](https://doi.org/10.1074/jbc.M114.623488)). The cross-linking of the viral structural proteins by Mxra8, while facilitating attachment and entry, might create a conundrum for viral fusion in the endosome, which requires domain B on E2 to undergo a substantive conformational shift to expose fully the underlying hydrophobic fusion loop in domain II of E1 ([Li et al., 2010](https://doi.org/10.1074/jbc.M110.141880); [Voss et al., 2010](https://doi.org/10.1074/jbc.M110.141880)). After clathrin-mediated endocytosis, fusion of CHIKV occurs within the acidic environment of early endosomes ([Hoornweg et al., 2016](https://doi.org/10.1074/jbc.M116.733736)). Although further studies are required, we speculate that some of the Mxra8 binding interactions with CHIKV E1 and E2 may be sensitive to acidic pH, such that, upon transiting to the early endosome, the cross-linked trimers can separate and allow the structural transitions required for fusion to occur. In preliminary experiments, we observed decreased affinity of binding of Mxra8 to recombinant pE2-E1 under conditions of mildly acidic pH (K.B., unpublished results). It is plausible that the strength of Mxra8 interactions with viral proteins may regulate the stage of endocytosis and pH of fusion of some arthritogenic alphaviruses.

The alphavirus structural polyprotein is processed in the endoplasmic reticulum to yield E3-E2 (p62) and E1, which form heterodimers and oligomerize as trimers to generate the immature spike ([Uchime et al., 2013](https://doi.org/10.1074/jbc.M113.490086)). In the Golgi network, furin-like proteases cleave E3 from E2 (yielding E2-E1) to render the spikes optimally fusogenic. However, the cleavage and dissociation event can be variable in a cell-type- and virus-type-specific manner, as E3 remains covalently or non-covalently associated with the mature virus in some alphaviruses, including Sindbis virus, Semliki Forest virus, and VEEV ([Heidner et al., 1996](https://doi.org/10.1074/jbc.271.49.31359); [Zhang et al., 2011](https://doi.org/10.1074/jbc.M111.265008); [Ziemiecki](https://doi.org/10.1074/jbc.M111.265008)).

---

(E) Surface representation of all CHIKV structural protein domains at the interface of Mxra8 at site 3 (shown as ribbon diagram), colored by E2-E1 heterodimer: wrapped, light gray; intraspikes, medium gray; interspike, dark gray. CHIKV protein residues are colored magenta if they lose over 30% solvent surface area upon Mxra8 binding as calculated by PDBePISA ([http://www.ebi.ac.uk/pdbe/pisa/](http://www.ebi.ac.uk/pdbe/pisa/)).

(F and G) Zoomed-in view of D1 of Mxra8. (F) CHIKV E2 residues identified as epitopes for neutralizing human anti-CHIKV mAbs ([Long et al., 2015](https://doi.org/10.1074/jbc.M115.653883); [Smith et al., 2015](https://doi.org/10.1074/jbc.M115.653883)) that also block binding to Mxra8 ([Zhang et al., 2018](https://doi.org/10.1074/jbc.M118.765883)) are labeled and colored blue. (G) Four CHIKV E2 residues at the CHIKV-Mxra8 interface, as defined by previous alanine scanning mutagenesis data ([Zhang et al., 2018](https://doi.org/10.1074/jbc.M118.765883)) are labeled and colored green.

See Figures S2, S4, S5, and S7 and Table S4.
Figure 6. Modes of Mxra8 Engagement by CHIKV

(A) Two-dimensional equatorial slice (top) and Fourier shell correlation (FSC) plot (bottom) of CHIKV infectious particles in complex with Mxra8.

(B and C) Views of asymmetric units of CHIKV infectious particles (B) or CHIKV VLP (C) with bound Mxra8 electron density at high contour (left) and low contour (middle and right), colored by structural proteins if within 6 Å of docked model coordinates. Mxra8 coordinates were removed from the model and E3 was docked in for (B). The σ value is the standard deviation of density values above the mean. Color scheme: gray, E1; cyan, E2; yellow, E3; capsid, green; Mxra8 and other unexplained density, magenta.

(D and E) SPR kinetic sensograms and steady-state analysis of murine Mxra8 binding to CHIKV VLPs fit to a 1:1 binding model (D) or two-site model (one high-affinity site and three lower but equal affinity sites) (E). Raw experimental traces are in black, fit traces are in red. Inset: Scatchard plot (4 experiments; mean, SEM, and χ² values).

See Figures S8 and S9.
Our cryo-EM map of Mxra8 bound to CHIKV is corroborated by mutational analysis and coincidence mapping of antibodies that blocked Mxra8 or CHIKV binding. We previously identified solvent-accessible residues W64, D71, T116, and I121 in the A domain of E2 as essential for optimal Mxra8 engagement by using an E2 alanine-scanning mutagenesis library (Fox et al., 2015; Smith et al., 2015; Zhang et al., 2018). Our cryo-EM analysis identified three of these amino acids as direct interface residues (W64, D71, and I121), with one additional residue (T116) positioned in close proximity to Mxra8. When charge-reversal point mutations were introduced in Mxra8 residues at the wrapped heterodimer interface (D89, G94, R97, V98, and Y99), reductions in CHIKV infection were observed. In further support of the structurally determined Mxra8 binding site on CHIKV, neutralizing human mAbs recognizing epitopes in the A domain inhibited Mxra8 binding, whereas others localizing to distinct sites had less effect (Zhang et al., 2018). The epitopes of the anti-CHIKV mAbs (Smith et al., 2015) that completely blocked Mxra8 binding directly overlap the structural binding sites of Mxra8 and CHIKV E2. Our epitope mapping of a panel of anti-Mxra8 mAbs and experiments assessing their ability to block virus binding provides further support for the cryo-EM atomic model.

Mxra8 is a receptor for multiple arthritogenic, but not encephalitic, alphaviruses (Zhang et al., 2018). An alignment of amino acid sequences corresponding to CHIKV residues comprising the Mxra8 binding site reveals that determinants are generally conserved among arthritogenic alphaviruses (44% conserved) but are more divergent in encephalitic alphaviruses (14% conserved) (Figures S4 and S5), which likely explains the negligible interaction or binding requirement for infectivity of Mxra8 with VEEV or related encephalitic alphaviruses (Zhang et al., 2018). Cryo-EM structures with other alphaviruses, including ONNV, MAYV, and RRV, should provide further insight into the critical E2, E1, and Mxra8 contacts that facilitate attachment, entry, and infection. Such analysis could identify targets on either the viral structural proteins or Mxra8 to facilitate the development of agents capable of disrupting these interactions. This approach could form the basis of a therapy that mitigates infection by multiple arthritogenic alphaviruses.

### STAR★METHODS

Detailed methods are provided in the online version of this paper and include the following:

- **KEY RESOURCES TABLE**
- **CONTACT FOR REAGENT AND RESOURCE SHARING**
- **EXPERIMENTAL MODEL AND SUBJECT DETAILS**
  - Viruses
- **METHOD DETAILS**
  - VLP Production and Purification
  - Mxra8 Protein Generation and Purification
  - Multi-angle Light Scattering Analysis
  - Mxra8 Crystallography and Structural Analysis
  - SPR-Based Mxra8 Binding Assay
  - BLI-Based Competition Binding Assay

---

○ Cryo-EM Sample Preparation, Data Collection, and Single Particle Reconstruction  
○ Model Building  
○ Mxra8 Expression Experiments in 293T Cells  
○ Hydrogen-Deuterium Exchange Mass Spectrometry  
○ HDX Data Analysis and Epitope Assignment  

**DATA AND SOFTWARE AVAILABILITY**

**SUPPLEMENTAL INFORMATION**

Supplemental Information can be found online at [https://doi.org/10.1016/j.cell.2019.04.006](https://doi.org/10.1016/j.cell.2019.04.006).

**ACKNOWLEDGMENTS**

This study was supported by NIH grants R01AI114816, R01AI123348, R01AI095436, and T32AI007172 and contract HHSN272201700060C (CSGID). The mass spectrometry was supported by NIH P41GM103422. We thank J. Fitzpatrick, M. Rau, and M. Tobias of the Washington University Center for Cellular Imaging for assistance with cryo-EM data collection and processing; Rui Zhang of Washington University for helpful suggestions; J. Nix of ALS beamline 4.2.2 for assistance with X-ray diffraction collection and processing; and J. Alexander of PaxVax for critical comments on the manuscript.

**AUTHOR CONTRIBUTIONS**

K.B. undertook the cryo-EM reconstructions and asymmetric unit atomic modeling with support from C.A.N. and D.H.F. A.S.K. engineered and purified the Mxra8 proteins and developed crystallization conditions. C.A.N., A.S.K., and D.H.F. solved the Mxra8 structure. B.S. aided in Mxra8 model building. K.B. performed and analyzed the SPR and BLI data. R.Z. performed the CHIKV infection experiments and provided key virus reagents. C.U. and L.V. produced and purified the CHIKV VLPs. J.S. provided key resources for CHIKV VLP generation. M.C. and M.L.G. performed and analyzed the HDX epitope mapping. K.B. and A.S.K. performed data analysis. M.S.D., K.B., and D.H.F. wrote the initial manuscript draft, with the other authors providing editorial comments.

**DECLARATION OF INTERESTS**

M.S.D. is a consultant for Inbios and Atreca and is on the Scientific Advisory Board of Moderna. D.H.F. is a founder of Courier Therapeutics. L.V. and J.S. are employees of and have equity in PaxVax, which has a vaccine program for emerging viruses, including CHIKV.

Received: November 9, 2018  
Revised: January 22, 2019  
Accepted: April 1, 2019  
Published: May 9, 2019

**REFERENCES**

Adams, P.D., Afonine, P.V., Bunkóczi, G., Chen, V.B., Davis, I.W., Echols, N., Headd, J.J., Hung, L.W., Kapral, G.J., Grosse-Kunstleve, R.W., et al. (2010). PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. D Biol. Crystallogr. 66, 213–221.

Afonine, P.V., Grosse-Kunstleve, R.W., Echols, N., Headd, J.J., Moriarty, N.W., Mustyakimov, M., Terwilliger, T.C., Urzhumtsev, A., Zwart, P.H., and Adams, P.D. (2012). Towards automated crystallographic structure refinement with phenix.refine. Acta Crystallogr. D Biol. Crystallogr. 68, 352–367.

Akahata, W., Yang, Z.Y., Andersen, H., Sun, S., Holdaway, H.A., Kong, W.P., Lewis, M.G., Higgs, S., Rossmann, M.G., Rao, S., and Nabel, G.J. (2010). A virus-like particle vaccine for epidemic Chikungunya virus protects nonhuman primates against infection. Nat. Med. 16, 334–338.
Bern, M., Cai, Y., and Goldberg, D. (2007). Lookup peaks: a hybrid of de novo sequencing and database search for protein identification by tandem mass spectrometry. Anal. Chem. 79, 1393–1400.

Bramucci, E., Paiardini, A., Bossa, F., and Pascarella, S. (2012). PyMod: sequence similarity searches, multiple sequence-structure alignments, and homology modeling within PyMOL. BMC Bioinformatics 13 (Suppl 4), S2.

Brien, J.D., Lazear, H.M., and Diamond, M.S. (2013). Propagation, quantification, detection, and storage of West Nile virus. Curr Protoc Microbiol 31, 15D.3.1–15D.3.18.

Busby, S.A., Chalmers, M.J., and Griffin, P.R. (2007). Improving digestion efficiency under H/D exchange conditions with activated pepsinogen coupled columns. Int. J. Mass Spectrom. 259, 130–139.

Carleton, M., Lee, H., Mulvey, M., and Brown, D.T. (1997). Role of glycoprotein PE2 in formation and maturation of the Sindbis virus spike. J. Virol. 71, 1558–1566.

Chang, L.J., Dowd, K.A., Mendoza, F.H., Saunders, J.G., Sitar, S., Plummer, S.H., Yamshchikov, G., Sarwar, U.N., Hu, Z., Enama, M.E., et al.; VRC 311 Study Team (2014). Safety and tolerability of chikungunya virus-like particle vaccine in healthy adults: a phase 1 dose-escalation trial. Lancet 384, 2046–2052.

Chen, L., Wang, M., Zhu, D., Sun, Z., Ma, J., Wang, J., Kong, L., Wang, S., Liu, Z., Wei, L., et al. (2018). Implication for alphavirus host-cell entry and assembly indicated by a 3.5Å resolution cryo-EM structure. Nat. Commun. 9, 5326.

Cheng, R.H., Kuhn, R.J., Olson, N.H., Rossmann, M.G., Choi, H.K., Smith, T.J., and Baker, T.S. (1995). Nucleocapsid and glycoprotein organization in an enveloped virus. Cell 80, 621–630.

DeTulleo, L., and Kirchhausen, T. (1998). The clathrin endocytic pathway in viral infection. EMBO J. 17, 4585–4593.

Emsley, P., Lohkamp, B., Scott, W.G., and Cowtan, K. (2010). Features and development of Coot. Acta Crystallogr. D Biol. Crystallogr. 66, 486–501.

Fox, J.M., Long, F., Edeling, M.A., Lin, H., van Duijl-Richter, M.K.S., Fong, R.H., Kahle, K.M., Smit, J.M., Jin, J., Simmons, G., et al. (2015). Broadly Neutralizing Alphavirus Antibodies Bind an Epitope on E2 and Inhibit Entry and Egress. Cell 163, 1095–1107.

Grant, T., Rohou, A., and Grigorieff, N. (2018). cisTEM, user-friendly software for single-particle image processing. eLife 7, e35383.

Hasan, S.S., Sun, C., Kim, A.S., Watanabe, Y., Chen, C.L., Klose, T., Buda, G., Crispin, M., Diamond, M.S., Klimenta, W.B., and Rossmann, M.G. (2018). Cryo-EM Structures of Eastern Equine Encephalitis Virus Reveal Mechanisms of Virus Disassembly and Antibody Neutralization. Cell Rep. 25, 3136–3147.e5.

Hayward, S., and Berendsen, H.J. (1998). Systematic analysis of domain motions in proteins from conformational change: new results on citrate synthase and T4 lysozyme. Proteins 30, 144–154.

Heidner, H.W., Knott, T.A., and Johnston, R.E. (1996). Differential processing of sindbis virus glycoprotein PE2 in cultured vertebrate and arthropod cells. J. Virol. 70, 2069–2073.

Holm, L., and Laakso, L.M. (2016). Dali server update. Nucleic Acids Res. 44, W351–W355.

Hoornweg, T.E., van Duijl-Richter, M.K.S., Ayala Nuñez, N.V., Albulescu, I.C., van Hemert, M.J., and Smit, J.M. (2016). Dynamics of Chikungunya Virus Cell Entry Unraveled by Single-Virus Tracking in Living Cells. J. Virol. 90, 4745–4756.

Hu, X., Wang, H., Ke, H., and Kuhlman, B. (2007). High-resolution design of a protein loop. Proc Natl Acad Sci U S A 104, 17668–17673.

Kabsch, W. (2010). XDS. Acta Crystallogr D Biol. Crystallogr 66, 125–132.

Kostyuchenko, V.A., Jakana, J., Liu, X., Haddow, A.D., Aung, M., Weaver, S.C., Chiu, W., and Lok, S.M. (2011). The structure of barmah forest virus as revealed by cryo-electron microscopy at a 6-angstrom resolution has detailed transmembrane protein architecture and interactions. J. Virol. 85, 9327–9333.

Lee, S., Kuhn, R.J., and Rossmann, M.G. (1998). Probing the potential glycoprotein binding site of sindbis virus capsid protein with dioxane and model building. Proteins 33, 311–317.

Lee, R.C., Hapuarachchi, H.C., Chen, K.C., Hussain, K.M., Chen, H., Low, S.L., Ng, L.C., Lin, R., Ng, M.M., and Chu, J.J. (2013). Mosquito cellular factors and functions in mediating the infectious entry of chikungunya virus. PLoS Negl. Trop. Dis. 7, e2050.

Lescar, J., Roussel, A., Wien, M.W., Navaza, J., Fuller, S.D., Wengler, G., Wengler, G., and Rey, F.A. (2001). The Fusion glycoprotein shell of Semliki Forest virus: an icosahedral assembly primed for fusogenic activation at endosomal pH. Cell 105, 137–148.

Li, L., Jose, J., Xiang, Y., Kuhn, R.J., and Rossmann, M.G. (2010). Structural changes of envelope proteins during alphavirus fusion. Nature 468, 705–708.

Long, F., Fong, R.H., Austin, S.K., Chen, Z., Klose, T., Fokine, A., Liu, Y., Porta, J., Sapparapu, G., Akahata, W., et al. (2015). Cryo-EM structures elucidate neutralizing mechanisms of anti-chikungunya human monoclonal antibodies with therapeutic activity. Proc. Natl. Acad. Sci. USA 112, 13898–13903.

Mukherjee, S., Sirohi, D., Dowd, K.A., Chen, Z., Diamond, M.S., Kuhn, R.J., and Pierson, T.C. (2016). Enhancing dengue virus maturation using a stable furin over-expressing cell line. Virology 497, 33–40.

Mulvey, M., and Brown, D.T. (1995). Involvement of the molecular chaperone BiP in maturation of Sindbis virus envelope glycoproteins. J. Virol. 69, 1621–1627.

Nelson, C.A., Wilen, C.B., Dai, Y.N., Orchard, R.C., Kim, A.S., Stegeman, R.A., Hsieh, L.L., Smith, T.J., Virgin, H.W., and Fremont, D.H. (2018). Structural basis for murine norovirus engagement of bile acids and the CD300lf receptor. Proc. Natl. Acad. Sci. USA 115, E9201–E9210.

Ooi, Y.S., Stiles, K.M., Liu, C.Y., Taylor, G.M., and Kielian, M. (2013). Genome-wide RNAi screen identifies novel host proteins required for alphavirus entry. PLoS Pathog. 9, e1003835.

Pal, P., Dowd, K.A., Brien, J.D., Edeling, M.A., Gorlatov, S., Johnson, S., Lee, I., Akahata, W., Nabel, G.J., Richter, M.K.S., et al. (2013). Development of a highly protective combination monoclonal antibody therapy against Chikungunya virus. PLoS Pathog. 9, e1003312.

Papadopoulos, J.S., and Agarwala, R. (2007). COBALT: constraint-based alignment tool for multiple protein sequences. Bioinformatics 23, 1073–1079.

Paredes, A.M., Brown, D.T., Rothnagel, R., Chiu, W., Schoepp, R.J., Johnston, R.E., and Prasad, B.V. (1993). Three-dimensional structure of a membrane-containing virus. Proc. Natl. Acad. Sci. USA 90, 9095–9099.

Pascal, B.D., Willis, S., Lauer, J.L., Landgraf, R.R., West, G.M., Marciano, D., Novick, S., Goswami, D., Chalmers, M.J., and Griffin, P.R. (2012). HDX workbench: software for the analysis of H/D exchange MS data. J. Am. Soc. Mass Spectrom. 23, 1512–1521.

Pletnev, S.V., Zhang, W., Mukhopadhyay, S., Fisher, B.R., Hernandez, R., Brown, D.T., Baker, T.S., Rossmann, M.G., and Kuhn, R.J. (2001). Locations of carbohydrate sites on alphavirus glycoproteins show that E1 forms an icosahedral scaffold. Cell 105, 127–136.

Pokidysheva, E., Zhang, Y., Battisti, A.J., Bator-Kelly, C.M., Chipman, P.R., Xiao, C., Gregorio, G.G., Hendrickson, W.A., Kuhn, R.J., and Rossmann, M.G. (2006). Cryo-EM reconstruction of dengue virus in complex with the carbohydrate recognition domain of DC-SIGN. Cell 124, 485–493.

Powers, A.M., Brault, A.C., Shirako, Y., Strauss, E.G., Kang, W., Strauss, J.H., and Weaver, S.C. (2001). Evolutionary relationships and systematics of the alphaviruses. J. Virol. 75, 10118–10131.

Robert, X., and Gouet, P. (2014). Deciphering key features in protein structures with the new ENDscript server. Nucleic Acids Res. 42, W320–W324.

Rohou, A., and Grigorieff, N. (2015). CTFFIND4: Fast and accurate defocus estimation from electron micrographs. J Struct Biol 192, 216–221.

Roussel, A., Lescar, J., Vaney, M.C., Wengler, G., Wengler, G., and Rey, F.A. (2006). Structure and interactions at the viral surface of the envelope protein E1 of Semliki Forest virus. Structure 14, 75–86.

Seiradake, E., Lortat-Jacob, H., Billet, O., Kremer, E.J., and Cusack, S. (2006). Structural and mutational analysis of human Ad37 and canine adenovirus 2 fiber heads in complex with the D1 domain of coxsackie and adenovirus receptor. J. Biol. Chem. 281, 33704–33716.

Smith, T.J., Cheng, R.H., Olson, N.H., Peterson, P., Chase, E., Kuhn, R.J., and Baker, T.S. (1995). Putative receptor binding sites on alphaviruses as visualized by cryoelectron microscopy. Proc. Natl. Acad. Sci. USA 92, 10648–10652.

Smith, S.A., Silva, L.A., Fox, J.M., Flyak, A.I., Kose, N., Sapparapu, G., Khomandiak, S., Ashbrook, A.W., Kahle, K.M., Fong, R.H., et al. (2015). Isolation and Characterization of Broad and Ultrapotent Human Monoclonal Antibodies with Therapeutic Activity against Chikungunya Virus. Cell Host Microbe 18, 86–95.

Strauss, J.H., Wang, K.S., Schmaljohn, A.L., Kuhn, R.J., and Strauss, E.G. (1994). Host-cell receptors for Sindbis virus. Arch. Virol. Suppl. 9, 473–484.

Sun, S., Xiang, Y., Wataru, A., Holdaway, H.A., Pal, P., Zheng, X., Diamond, M.S., Nabel, G.J., and Rossmann, M.G. (2013). Structural analyses at pseudo atomic resolution of Chikungunya virus and antibodies show mechanisms of neutralization. eLife 2, e00435.

Tucker, P.C., and Griffin, D.E. (1991). Mechanism of altered Sindbis virus neu- rovirulence associated with a single-amino-acid change in the E2 Glycoprotein. J. Virol. 65, 1551–1557.

Uchime, O., Fields, W., and Kiilian, M. (2013). The role of E3 in pH protection during alphavirus assembly and exit. J. Virol. 87, 10255–10262.

Voss, J.E., Vaney, M.C., Duquerroy, S., Vonrhein, C., Girard-Blanc, C., Crublet, E., Thompson, A., Bricogne, G., and Rey, F.A. (2010). Glycoprotein organization of Chikungunya virus particles revealed by X-ray crystallography. Nature 468, 709–712.

Wu, S.R., Haag, L., Sjöberg, M., Garoff, H., and Hammar, L. (2008). The dynamic envelope of a fusion class II virus. E3 domain of glycoprotein E2 precursor in Semliki Forest virus provides a unique contact with the fusion protein E1. J. Biol. Chem. 283, 26452–26460.

Yan, Y., Grant, G.A., and Gross, M.L. (2015). Hydrogen-Deuterium Exchange Mass Spectrometry Reveals Unique Conformational and Chemical Transformations Occurring upon [4Fe-4S] Cluster Binding in the Type 2 L-Serine Dehydrogenase from Legionella pneumophila. Biochemistry 54, 5322–5328.

Yap, M.L., Klose, T., Urakami, A., Hasan, S.S., Akahata, W., and Rossmann, M.G. (2017). Structural studies of Chikungunya virus maturation. Proc. Natl. Acad. Sci. USA 114, 13703–13707.

Zhang, W., Mukhopadhyay, S., Pletnev, S.V., Baker, T.S., Kuhn, R.J., and Rossmann, M.G. (2002). Placement of the structural proteins in Sindbis virus. J. Virol. 76, 11645–11658.

Zhang, W., Heil, M., Kuhn, R.J., and Baker, T.S. (2005). Heparin binding sites on Ross River virus revealed by electron cryo-microscopy. Virology 332, 511–518.

Zhang, R., Hryc, C.F., Cong, Y., Liu, X., Jakana, J., Gorchakov, R., Baker, M.L., Weaver, S.C., and Chiu, W. (2011). 4.4 Å cryo-EM structure of an enveloped alphavirus Venezuelan equine encephalitis virus. EMBO J. 30, 3854–3863.

Zhang, R., Kim, A.S., Fox, J.M., Nair, S., Basore, K., Klimstra, W.B., Rimkunas, R., Fong, R.H., Lin, H., Poddar, S., et al. (2018). Mxra8 is a receptor for multiple arthritogenic alphaviruses. Nature 557, 570–574.

Zheng, S.Q., Palovcak, E., Armache, J.P., Verba, K.A., Cheng, Y., and Agard, D.A. (2017). MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods 14, 331–332.

Ziemiecki, A., and Garoff, H. (1978). Subunit composition of the membrane glycoprotein complex of Semliki Forest virus. J. Mol. Biol. 122, 259–269.

Zivanov, J., Nakane, T., Forsberg, B.O., Kimanius, D., Hagen, W.J., Lindahl, E., and Scheres, S.H. (2018). New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, e42166.

STAR★METHODS

KEY RESOURCES TABLE

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
|---|---|---|
| Antibodies |  |  |
| Rabbit polyclonal anti-CHIKV 181/25 | IBT Bioservices | Cat#04-0008 |
| HRP-conjugated Goat anti-mouse IgG | Thermo Fisher | Cat#65-6120; RRID:AB_2533967 |
| CHK-265mAb | Patel al., 2013 | N/A |
| Hamster anti-mouse Mxra8mAb1G 11.E 6 | Zhang et al., 2018 | N/A |
| Hamster anti-mouse Mxra8mAb1H 1.F 5 | Zhang et al., 2018 | N/A |
| Hamster anti-mouse Mxra8mAb3G 2.F 5 | Zhang et al., 2018 | N/A |
| Hamster anti-mouse Mxra8mAb4E 7.D 10 | Zhang et al., 2018 | N/A |
| Hamster anti-mouse Mxra8mAb7F 1.D 8 | Zhang et al., 2018 | N/A |
| Hamster anti-mouse Mxra8mAb8F 7.E 1 | Zhang et al., 2018 | N/A |
| Hamster anti-mouse Mxra8mAb9G 2.D 6 | Zhang et al., 2018 | N/A |
| Bacterial and Virus Strains |  |  |
| Chikungunya virus (strain 181/25) | BEI resources | Cat#NR-13222 |
| CHIKV VLP (strain 37997) | Akahata et al., 2010 | N/A |
| Chemicals, Peptides, and Recombinant Proteins |  |  |
| EZ-Link-NHS-PEG 4-Biotin | Thermo Fisher | Cat# 21330 |
| Mammalian-expressed mouse Mxra 8 | Zhang et al., 2018 | N/A |
| Mammalian-expressed human Mxra 8 | Zhang et al., 2018 | N/A |
| Bacterial refolded mouse Mxra 8 ectodomain 23-296 | This paper | GenBank:NM_024263 |
| Critical Commercial Assays |  |  |
| QuikChange II XL Site-Directed Mutagenesis Kit | Agilent | Cat# 200522 |
| Pierce HRV3C Protease Solution Kit | Thermo Fisher | Cat# 88946 |
| Deposited Data |  |  |
| Crystal structure of murine Mxra 8 ectodomain | This paper | PDB:6NK3 |
| Electron Cryo-Microscopy of Chikungunya VLP alone | This paper | PDB:6NK5; EMDB:EMD-9393 |
| Electron Cryo-Microscopy of Chikungunya VLP in complex with mouse Mxra 8 receptor | This paper | PDB:6NK6; EMDB:EMD-9394 |
| Electron Cryo-Microscopy of Chikungunya in complex with mouse Mxra 8 receptor | This paper | PDB:6NK7; EMDB:EMD-9395 |
| Mapping data of anti-Mxra8mAbs | This paper | IEDB: 1000791 |
| Experimental Models: Cell Lines |  |  |
| HEK-293 | ATCC | Cat# CRL-1573 |
| Vero | ATCC | Cat# CCL-81 |
| Expi-293 | Thermo Fisher | Cat# A 14527 |
| BL21(DE3) | Thermo Fisher | Cat# C 600003 |
| Recombinant DNA |  |  |
| Codon-optimized mouse Mxra 8 cloned into pCDNA 3.4 vector | Zhang et al., 2018 | N/A |
| Codon-optimized human Mxra 8 cloned into pCDNA 3.4 vector | Zhang et al., 2018 | N/A |
| Codon-optimized mouse Mxra 8 cloned into pET21a vector | This paper | N/A |

(Continued on next page)

Continued

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
|---------------------|--------|------------|
| Software and Algorithms |  |  |
| AstraV version 5.3.4.20 | Wyatt Technology | https://www.wyatt.com/products/software/astra.html |
| BIAevaluation version 3.1 | GE Healthcare | https://www.biacore.com/lifesciences/service/downloads/software_licenses/biaevaluation/ |
| PyMOL version 1.7.4 | Schrödinger, LLC | https://pymol.org/2/ |
| COOT version 0.8.9.1 | Emsley et al., 2010 | https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/ |
| PHENIX version 1.14 | Afonine et al., 2012 | https://www.phenix-online.org |
| PDBePISA version 1.52 | EMBL-EBI | http://www.ebi.ac.uk/pdbe/pisa/ |
| Chimera version 1.13.1 | RBVI at UCSF | https://www.cgl.ucsf.edu/chimera/ |
| cisTEM version 1.0.0 | Grant et al., 2018 | https://cistem.org |
| RELION version 3.0 | Zivanov et al., 2018 | https://www3.mrc-lmb.cam.ac.uk/relion/index.php?title=Main_Page |
| ESPrift version 3.0 | Robert and Gouet, 2014 | http://esprit.ibcp.fr/ESPrift/ESPrift/ |
| DynDom | Hayward and Berendsen, 1998 | http://dyndom.cmp.uea.ac.uk/dyndom/ |
| Prism version 8 | GraphPad | https://www.graphpad.com/scientific-software/prism/ |

| Other |  |  |
|-------|---|---|
| 300 mesh, R 2/2 holey carbon grids | Quantifoil | Cat#4430G |
| Series S Sensor Chip CM 5 | GE Healthcare | Cat#BR100530 |
| Streptavidin (SA) Biosensor | ForteBio | Cat#18-5020 |

---

**CONTACT FOR REAGENT AND RESOURCE SHARING**

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Daved H. Fremont (<fremont@wustl.edu>).

---

**EXPERIMENTAL MODEL AND SUBJECT DETAILS**

**Viruses**

CHIKV strain 181/25 was obtained from the World Reference Center for Emerging Viruses and Arboviruses (S. Weaver and K. Plante, Galveston, TX). The virus was propagated in Vero cells (ATCC) in DMEM supplemented with 10% FBS, concentrated by sucrose gradient ultracentrifugation, and titrated by standard focus-forming assays ([Brien et al., 2013; Pal et al., 2013](https://doi.org/10.1016/j.cell.2013.04.016)).

---

**METHOD DETAILS**

**VLP Production and Purification**

CHIKV VLPs were produced via transient transfection of C-E3-E2-6K-E1 (CHIKV strain 37997) plasmid DNA ([Akahata et al., 2010](https://doi.org/10.1038/nm.2149)) into HEK293 cells (obtained from Vaccine Research Center, NIH), and purified via Q Sepharose XL (GE Healthcare, GE17-5072-01) anion chromatography. The peak of interest was diafiltered into a buffer containing 218 mM sucrose, 10 mM potassium phosphate, and 25 mM citrate, pH 7.2. The material was sterile-filtered using a 0.2 μM filter, 500 μL aliquots were made and stored at -80°C. The final material was analyzed by BCA assay for protein concentration, Coomassie-stained SDS-PAGE gel with densitometry, and western blotting with rabbit polyclonal anti-CHIKV 181/25 (04-0008, IBT Bioservices) and a secondary horseradish peroxidase conjugated goat anti-rabbit antibody (65-6120, ThermoFisher Scientific).

**Mxra8 Protein Generation and Purification**

To generate recombinant bacterially derived mouse Mxra8 protein, residues 23-296 (Accession number NM_024263) were codon optimized, synthesized, and inserted into the pET21a expression vector. After sequence verification, Mxra8 was expressed in BL21(DE3) *E. coli* cells. Cells were grown to an OD~600~ of 0.8, induced with 1 mM IPTG for 5 h at 37°C, resuspended in 50 mM Tris-HCl, 1 mM EDTA, 0.01% NaN₃, and 1 mM DTT (TEND) buffer supplemented with 25% sucrose, and lysed in 50 mM Tris-HCl,

1 mM EDTA, 0.01% NaN₃, 1 mM DTT, 200 mM NaCl, 1% sodium deoxycholate and 1% Triton X-100. Inclusion bodies were isolated and washed extensively in TEND buffer with 100 mM NaCl and 0.5% Triton X-100 and then TEND buffer alone. Inclusion bodies (~200 mg) were denatured in 6M guanidinium hydrochloride, 100 mM Tris-HCl, and 20 mM β-mercaptoethanol. Denatured protein was oxidatively refolded overnight at 4°C in 100 mM Tris-HCl, 400 mM L-arginine, 5 mM reduced glutathione, 0.5 mM oxidized glutathione, 10 mM EDTA and 200 mM phenylmethylsulfonyl fluoride. Protein was concentrated using a 10 kDa molecular weight cut-off stirred cell concentrator (EMD Millipore) and purified by HiLoad 16/600 Superdex 75 size exclusion chromatography (GE Healthcare). Mxra8 mutants were generated using a QuikChange II XL Site-Directed Mutagenesis Kit (Agilent) and verified by DNA sequencing. Mxra8 mutant proteins were expressed and purified as described above.

To generate mammalian Mxra8 protein, residues 23-336 (Accession number NM_024263) were codon optimized, synthesized, and inserted into the pCDNA3.4 vector upstream of the HRV 3C cleavage sequence (LEVLFQGP) and the mouse IgG2b Fc region. After sequence confirmation, Mxra8-HRV-Fc protein was expressed using the Expi293 mammalian expression system (Thermo Fisher). Briefly, 200 μg of plasmid was diluted in Opti-MEM, incubated with HYPE-5 transfection reagent (OZ Biosciences), and added drop-wise to cells (10⁶ cells/ml). Transfected cells were supplemented daily with Expi293 media and 2% (w/v) Cell Boost Supplement (HyClone). Four days post transfection, supernatant was harvested by centrifugation and purified by Protein A Sepharose (Thermo Fisher) chromatography. The mouse IgG2b region was cleaved using the Pierce HRV 3C Protease Solution Kit (Thermo Fisher). Cleaved Fc fragments were depleted using Protein A Sepharose chromatography, and HRV-cleaved Mxra8 was purified by HiLoad 10/600 Superdex 75 size exclusion chromatography (GE Healthcare). Purity was determined by SDS-PAGE analysis.

### Multi-angle Light Scattering Analysis

Recombinant Mxra8 proteins were loaded onto a size exclusion chromatography column in sizing buffer (150 mM NaCl, 20 mM HEPES pH 7.4, 0.01% NaN₃) in series with a Dawn HELEOS-II 18-angle light scattering detector (Wyatt), Optilab rEX refractive index monitor (Wyatt), and photodiode array detector 996 (Waters). The light scattering, refractive index change, and UV light absorbance were measured, and the molecular weight of the eluted protein was calculated using the Astra V macromolecular characterization software package (Wyatt).

### Mxra8 Crystallography and Structural Analysis

Mouse Mxra8 was crystallized by hanging drop vapor diffusion at 15.0 mg/ml in 100 mM Tris-HCl, pH 8.5, and 8% PEG 8000. Crystals were looped and flash cooled in mother liquor supplemented with 25% ethylene glycol. Data were remotely collected at Advanced Light Source MBC Beamline 4.2.2 and merged and processed with XDS ([Kabsch, 2010](https://doi.org/10.1107/S090744491002593X)). Molecular replacement was performed using Phaser in the Phenix software ([Adams et al., 2010](https://doi.org/10.1107/S0907444910011222)) and junctional adhesion molecule-like protein (JAML; PDB: 3MJ6) as the search model. Refinement and model building were carried out in Phenix and COOT ([Emsley et al., 2010](https://doi.org/10.1107/S0907444910009034)). Data collection and refinement statistics are reported in [Table S1](#table-s1). Structures were assessed using Molprobity and figures were generated using Chimera ([http://www.cgl.ucsf.edu/chimera](http://www.cgl.ucsf.edu/chimera)) and PyMOL ([Bramucci et al., 2012](https://doi.org/10.1016/j.jmb.2012.03.020)).

### SPR-Based Mxra8 Binding Assay

SPR binding experiments were performed on a Biacore T200 system (GE Healthcare) to measure the kinetics and affinity of Mxra8 binding to CHIKV VLPs. Experiments were performed at 30 μl/min and 25°C using HBS-EP (0.01 M HEPES pH 7.4, 0.15 M NaCl, 3 mM EDTA, 0.005% v/v Surfactant P20) as running buffer. CHK-265 mAb ([Pal et al., 2013](https://doi.org/10.1016/j.virol.2013.04.010)) was immobilized onto a CM5 sensor chip (GE Healthcare) using standard amine coupling chemistry, and CHIKV VLPs were captured. Recombinant Mxra8 was injected over a range of concentrations (2 μM to 20 nM) for 200 sec, followed by a 600 sec dissociation period. As a negative control, murine norovirus was captured with mAb A6.2, as described previously ([Nelson et al., 2018](https://doi.org/10.1016/j.jmb.2018.01.010)). Real-time data were analyzed using BIAevaluation 3.1 (GE Healthcare). For the equal affinity binding sites model, kinetic profiles and steady-state equilibrium concentration curves were fitted using a global 1:1 binding algorithm with drifting baseline.

For the model of Mxra8 binding to 1 of 4 sites with higher affinity, the following fits were used:

Equilibrium Analysis:
$$
RU = \frac{k_1[A]}{1 + k_1[A]} + \frac{3k_2[A]}{1 + k_2[A]}
$$

where \( k_1 \) corresponds to the high-affinity site and \( k_2 \) denotes the low-affinity sites.

Kinetic Analysis:
$$
\begin{aligned}
& \text{Reaction equations: } A + B1 \stackrel{k_{a1}}{\rightleftharpoons} AB1 \quad A + B2 \stackrel{k_{a2}}{\rightleftharpoons} AB2 \\
& \phantom{\text{Reaction equations: }} \stackrel{k_{d1}}{} \quad \stackrel{k_{d2}}{}
\end{aligned}
$$

where \( A = \) analyte (Mxra8), \( B1 = \) high-affinity sites of ligand (CHIKV VLP), and \( B2 = \) low-affinity sites of ligand (CHIKV VLP)

Differential equations:
$$
\frac{d[B1]}{dt} = -\left(k_{a1}[A][B1] - k_{d1}[AB1]\right)
$$

e3 Cell 177, 1725–1737.e1–e5, June 13, 2019
$\frac{d[B2]}{dt} = -3(k_{a2}[A][B2] - k_{d2}[AB2])$

$\frac{d[AB1]}{dt} = (k_{a1}[A][B1] - k_{d1}[AB1])$

$\frac{d[AB2]}{dt} = 3(k_{a2}[A][B2] - k_{d2}[AB2])$

### BLI-Based Competition Binding Assay

Binding of Mxra8 and anti-Mxra8 mAbs to captured CHIKV VLP was monitored in real-time at $25^\circ$C using an Octet-Red96 device (Pall ForteBio). CHK-265 (100 $\mu$g) was mixed with biotin (EZ-Link-NHS-PEG4-Biotin, Thermo Fisher) at a molar ratio of 20:1 biotin:mAb, incubated at room temperature for 30 min, then unreacted biotin was removed by passage through a desalting column (5 mL Zeba Spin 7K MWCO, Thermo Fisher). Biotinylated-CHK-265 was loaded onto streptavidin biosensors (ForteBio) until saturation, typically 10 $\mu$g/ml for 2 min, in 10 mM HEPES (pH 7.4), 150 mM NaCl, 3 mM EDTA, and 0.005% P20 surfactant with 1% BSA. CHIKV VLP was then added for 5 min. The biosensors then were dipped into wells containing 1 $\mu$M Mxra8 alone or with 1$\mu$M of hamster anti-Mxra8 mAbs (1G11.E6, 1H1.F5, 3G2.F5, 4E7.D10, 8F7.E1, or an isotype control mAb) for 5 min, followed by a 5-min dissociation.

### Cryo-EM Sample Preparation, Data Collection, and Single Particle Reconstruction

CHIKV VLP with and without cleaved Mxra8 and CHIKV infectious virus (181/25 strain) with cleaved Mxra8 in molar excess were flash cooled on holey carbon EM grids in liquid ethane under BSL-2 containment conditions using an FEI Vitrobot (ThermoFisher). Movies of the samples were recorded with the software EPU (Thermo Fisher) using a K2 Summit electron detector (Gatan) mounted on a Bioquantum 968 GIF Energy Filter (Gatan) attached to a Titan Krios microscope operating at 300 keV with an electron dose of 50 e$^-$/Å$^2$ and a magnification of 18,000x. The movies (30 frames, 300 ms exposure per frame) were corrected for beam-induced motion using MotionCor2 ([Zheng et al., 2017](https://doi.org/10.1038/nmeth.4211)). Contrast transfer function parameters of the electron micrographs were estimated using CTFFIND4 ([Rohou and Grigorieff, 2015](https://doi.org/10.1016/j.jsb.2015.08.008)) in cisTEM ([Grant et al., 2018](https://doi.org/10.1016/j.jsb.2017.11.008)). Particles were auto-picked, and underwent reference-free 2D classification, *ab initio* 3D reconstruction, and 3D refinement in cisTEM. Local resolution was estimated using RELION-3 ([Zivanov et al., 2018](https://doi.org/10.1038/s41592-018-0038-6)). Additional information regarding the number of images and particles is listed in [Table S2](#table-s2). Structural visualization of the 3D reconstructions was performed using the programs Chimera ([http://www.cgl.ucsf.edu/chimera](http://www.cgl.ucsf.edu/chimera)) and PyMOL ([Bramucci et al., 2012](https://doi.org/10.1016/j.jmb.2012.03.021)).

### Model Building

The initial models of the CHIKV structural proteins (E1, E2, TM regions, and Capsid) with or without Mxra8 were built into the density of a subunit by docking the components from the crystal structures of CHIKV p62-E1 (PDB: 3N42), the CHIKV capsid (PDB: 5J23), and the modeled TM regions of VEEV (PDB: 3J0C) using the fitmap command in Chimera. All components were docked in as one rigid body except for Mxra8, where D1 (residues 70–188) and D2 (32–59; 201–294) were docked as two separate bodies. Amino acid substitutions and Mxra8 hinge regions (residues 60–69; 189–200) were added or built manually in COOT to reflect the strain of CHIKV used for VLP production (CHIKV-37997) and infectious virus (CHIKV-181/25) for cryo-EM studies. The subunit models then underwent real-space refinement using PHENIX with default parameters plus rigid body refinement and secondary-structure and torsion restraints with the initial components as the reference. Rigid bodies for E1 were divided into domains I and II (residues 1–292), domain III (293–381), stem (382–412), and transmembrane region (413–442). E2 was divided into the N-linker region (residues 5–15), domain A (residues 16–134), domain B (residues 173–231), domain C (residues 269–341), $\beta$-linker (residues 135–172, 232–268), and the stem, transmembrane region, and cytoplasmic tail (residues 342–423). The capsid protein was divided into two rigid bodies (residues 111–176 and residues 177–261), and Mxra8 was divided into D1 (residues 70–188) and D2 (32–59; 201–294). The refined subunits were used to build the asymmetric unit with adjacent subunits to prevent clashes and optimize interactions. These models underwent further real-space refinement. COOT was used to fix regions manually with poor geometry. After optimization, coordinates of the asymmetric units were checked by MolProbity ([Table S3](#table-s3)). Contact residues were identified, and buried surface areas were calculated using PDBePISA ([www.ebi.ac.uk/pdbe/pisa/](http://www.ebi.ac.uk/pdbe/pisa/)).

### Mxra8 Expression Experiments in 293T Cells

The C-terminal FLAG tagged mouse Mxra8 corresponding to the transcript (NM_024263) and its corresponding charge reversal mutants were synthesized commercially (Genewiz) in a pCAGGS mammalian expression vector. Plasmids were transfected transiently in 293T cells using FuGene HD according to the manufacturer’s instructions. One day after transfection cells were inoculated with CHIKV-181/25 (MOI 3) for 12.5 h, and then harvested, fixed with paraformaldehyde (PFA), permeabilized, and processed for CHIKV expression using 1 $\mu$g/ml of humanized CHK-166 ([Pal et al., 2013](https://doi.org/10.1016/j.chom.2013.03.001)) and anti-FLAG M2 antibody (Sigma-Aldrich #1804). After washing,cells were stained with 2 μg/ml of goat anti-human IgG conjugated with Alexa Fluor 647 (Thermo Scientific) and 2 μg/ml of goat anti-mouse IgG conjugated with Alexa Fluor 488 (Thermo Scientific) and subjected to flow cytometry analysis on a MACSQuant Analyzer 10 (Miltenyi Biotec). To assess surface versus intracellular expression of Mxra8, cells were harvested at 36 h post transfection using TrypLE (Thermo Scientific), and incubated with a serum polyclonal hamster anti-mouse Mxra8 antibody (1:400) at 4°C for 25 min. After washing, intact cells were stained with 2 μg/ml of goat anti-Armenian hamster IgG H&L conjugated with Alexa Fluor 647 (Abcam #173004) for 25 min at 4°C. After two additional washes, cells were fixed with 2% PFA for 10 min at room temperature. After permeabilization, cells were stained with 1 μg/ml of anti-FLAG M2 antibody (Sigma-Aldrich #1804), and then with 2 μg/ml of goat anti-mouse IgG conjugated with Alexa Fluor 488 (Thermo Scientific). Cells were subjected to flow cytometry analysis and processed using FlowJo software (Tree Star).

### Hydrogen-Deuterium Exchange Mass Spectrometry

Continuous HDX labeling of Mxra8 with or without the anti-Mxra8 mAbs was performed at 25°C for 0, 10, 30, 60, 120, 900, 3,600 and 14,400 sec as previously described with the following modifications ([Yan et al., 2015](https://doi.org/10.1016/j.cell.2015.03.027)). Briefly, stock solutions of both Mxra8 with or without the mAbs (50 μM) were prepared in 1X PBS pH 7.4 and incubated at 25°C for at least 1 h. Continuous labeling with deuterium was initiated by diluting 1 μl of the stock samples 25-fold in 24 μl of deuterated 1X PBS buffer (Sigma-Aldrich). HDX control samples (non-deuterated) were prepared in the same way with H₂O. Quenching was performed under reducing condition by adding a solution of 500 mM Tris (2-carboxyethyl) phosphine hydrochloride (TCEP HCl) and 4 M guanidine hydrochloride in 1X PBS buffer pH 7.4 (adjusted using sodium hydroxide to pH 2.5) to the reaction vial at a 1:1 volume ratio. The sample was mixed and incubated for 1 min at 25°C before loading on to a custom-built HDX platform for desalting, on-line pepsin digestion, and reversed-phase separation and directly injected into the mass spectrometer for analysis.

The sample was passed over a custom-packed 2 × 20 mm pepsin column at 200 μL/min; immobilized pepsin was prepared by a published procedure ([Busby et al., 2007](https://doi.org/10.1021/ac070060u)). The peptides resulting from digestion were captured by a 2.1 × 20 mm Zorbax Eclipse XDB-C8 trap column (Agilent) and desalted at 200 μL/min of H₂O containing 0.1% trifluoroacetic acid for 3 min. The peptides were separated by a 2.1 × 50 mm C18 column (2.5 μm XSelect CSH C18; Waters) with a 9.5-min gradient of 5% to 100% acetonitrile in 0.1% formic acid at a flow rate of 100 μl/min delivered by a LEAP 3× Ti pump (LEAP technologies, NC). The linear part of the gradient from 0.3 min to 5.5 min raised the acetonitrile content from 15% to 50%, during which time most of the peptides eluted from the C18 column. The entire fluidic system was kept in an ice bath except for the pepsin column to minimize back exchange. Duplicate measurements were carried out for each of the time points.

Acquired spectra were analyzed using HDX workbench software ([Pascal et al., 2012](https://doi.org/10.1021/pr300184w)) against a peptide set generated as described below. The deuterium level was normalized to the maximum deuterium concentration (96%) contained in the reaction vial. A peptide list used to search the HDX data was identified first by a tandem MS experiment in a data-dependent mode on a LTQ-FT mass spectrometer (Thermo Scientific). The six most abundant ions were submitted to collision-induced dissociation fragmentation. Product-ion spectra were submitted to Byonic (Protein Metrics, version 2.11.0; [Bern et al., 2007](https://doi.org/10.1021/pr070225e)) for identification and peptide set build-up, then manually inspected, and the validated peptides were used for the HDX analysis.

### HDX Data Analysis and Epitope Assignment

The epitopes were identified as regions/sequences of amino acids (not single residues) and determined from peptide level HDX-MS data. The data trends in the overlapping peptides were analyzed using a conservative approach in making assignments. For each peptide, the relative difference in deuterium uptake was calculated by dividing the difference in deuterium uptake between Mxra8 alone and Mxra8 with mAb for each of the time points. Epitope peptides were initially identified as those peptides with relative differences in deuterium uptake of greater than 15% at two or more consecutive time points. If two or more overlapping peptides corresponded to a given region, the larger peptide was chosen. Selected peptides were screened to exclude the regions/sequences of amino acids exhibiting less than 15% relative difference in deuterium uptake between the apo- and the holo- states, as evidenced by another constituent peptide with shorter sequence length.

### DATA AND SOFTWARE AVAILABILITY

The accession numbers for the density maps reported in this paper are Electron Microscopy Data Bank: 9393, 9394, and 9395. The accession numbers for the crystallographic structure of Mxra8 and the corresponding refined atomic models of the EM reconstructions reported in this paper are PDB: 6NK3, 6NK4, 6NK5, and 6NK6. The accession number for the mapping data of anti-Mxra8 mAbs reported in this paper is Immune Epitope Database and Analysis Resource (IEDB): 1000791.
Supplemental Figures

**Figure S1. Biochemical Characterization of Soluble Mxra8 Proteins and CHIKV VLPs, Related to Figures 1 and 2**

(A–D). Mxra8 characterization. (A and B) Bacterially derived Mxra8 ectodomain was expressed in BL21 cells, oxidatively refolded, and purified by size exclusion chromatography. (A) Coomassie-stained SDS-PAGE of refolded Mxra8 under non-reducing and reducing conditions. (B) SEC-MALS of refolded Mxra8 showing a monomeric form at ~31 kDa. (C and D). Mxra8-Fc was produced in Expi-293 cells, purified by protein A affinity chromatography, and digested with HRV protease to release the monomeric form of Mxra8 from the Fc region. (C). Coomassie-stained SDS-PAGE under non-reducing and reducing conditions of 5 μg of undigested (left) and HRV-digested (right) Mxra8. One representative experiment of three is shown. (D) SEC-MALS of soluble Mxra8 showing a monomeric form at ~38 kDa.

(E–G) VLP characterization. After transfection of HEK-293 cells with plasmid DNA encoding CHIKV C-E3-E2-6K-E1 (strain 37997), VLPs were harvested from the supernatant and purified by anion exchange chromatography. (E) Coomassie-stained 4% to 12% SDS-PAGE of 2 μg of purified VLPs. E2 and E1 co-migrate at ~52–53 kDa. Capsid migrates at ~35 kDa. One representative experiment of two is shown. (F) Immunoblotting of 0.25 μg of VLPs with a rabbit anti-CHIKV polyclonal antibody. One representative experiment of two is shown. (G) Dynamic light scattering analysis of CHIKV VLPs. A single homogeneous peak of 68 nm (680 Å) ± 5.3 is shown. Data are representative of six experiments from two different VLP preparations.

Mouse_Mxra8
Human_Mxra8
Wrapped
Intraspike
Interspike

A2
B2
Hinge
C1
C1'
C1''

Mouse_Mxra8
Human_Mxra8
Wrapped
Intraspike
Interspike

D1
E1
F1
G1
H1
A1
B1

Mouse_Mxra8
Human_Mxra8
Wrapped
Intraspike
Interspike

Hinge
C2
C2'
C2''
D2
E2
F2

Mouse_Mxra8
Human_Mxra8
Wrapped
Intraspike
Interspike

G2

Figure S2. Sequence Alignment of Mouse and Human Mxra8 Orthologs, Related to Figures 1, 4, and 5

Alignment was performed between mouse (NP_077225) and human (NP_001269514) Mxra8 using PROMALS3D (Papadopoulos and Agarwala, 2007) with mouse Mxra8 numbering. The Figure was prepared using ESPript 3.0 (Robert and Gouet, 2014). Domains 1 (light magenta) and 2 (dark magenta) are indicated above the sequence, along with the labeled secondary structures. Triangles indicate the N and C termini of the crystal structure (black) and the cryo-EM model at sites 2, 3, and 4 (red). Red boxes signify conserved residues; white boxes and black letters, non-conserved residues. Numbers under the alignment indicate contacts between Mxra8 and individual E2-E1 heterodimers, given as percentage buried surface area calculated by PDBePISA (http://www.ebi.ac.uk/pdbe/pisa/): 1 represents 10% buried surface area, 2 represents 20% buried surface area, and so on. “Wrapped” denotes contacts to the wrapped E2-E1 heterodimer making the primary contacts to Mxra8. “Intraspike” refers to the intraspike heterodimer, which is adjacent to the wrapped heterodimer but within the same trimeric spike. “Interspike” refers to the interspike heterodimer, which is in the neighboring spike. Red numbers indicate contacts to CHIKV E3.

Figure S3. Domain Rotation in Mxra8, Related to Figures 1 and 4

(Upper left) Ribbon drawings of the Mxra8 structural models obtained from cryo-EM refinement shown superposed on the basis of the membrane proximal domain 2 (D2). These structures correspond to the four different binding sites (sites 1–4) on the CHIKV VLP. Analysis of the domain movements using DynDom ([Hayward and Berendsen, 1998](https://doi.org/10.1006/jmbi.1998.1757)) suggest that all four domains are essentially identical in conformation. (Upper right) In contrast, comparison of the two Mxra8 models from the asymmetric unit of the X-ray crystal structure (chains A and B) show a difference in domain orientation, yielding a rotation of ~43° around the axis shown in black. Furthermore, the conformations of both chains A and B vary from that seen in the cryo-EM structure of Mxra8 bound to VLP (lower panels). To facilitate comparison, the position of domain 2 was held constant throughout all four panels. The rotation axes pass through the hinge region near the interdomain disulfide. The domain movements include both bending and twisting motions, as indicated by differences in the relative orientations of the rotation axes. Depiction of helical twists have been removed for clarity.

E1
B0 C0 D0 a b c c'
1 CHIKV YEH VTVI P NTVGVP YKTL VNRP GYSP MVLEMELOSVTLEP TLSLDYITCEYKT VIPSPY VKCCG TAEC KDKSLPDYS CKVFT GVYPFM
MAYV YEH TAVIP NQVGFP YKAH VAREGYSPLT LQMVVETSELP TLNLEYITCDYKT KVPSPY VKCCG TAEC RTQDKPEYKCAVFT GVYPFM
RRV YEH TATIP NVVGFP YKAH IERNGFSP MTLQLEVVTSELP TLNLEYITCEYKT VVPSPY IKCCG TSEC SSKEQPDYQCKVYT GVYPFM
ONNV YEH ATVIP NTVGVP YKTL VS RPGYSP MVLEMELOSVTLEP ALSLDYITCEYKT ITPSPY VKCCG TAEC KAKNLPDYNCKVFT GVYPFM
SFV YEH STVM P NVVGFP YKAH IERPGYSP LTLQMVVETSELP TLNLEYITCEYKT VVPSPY VKCCG ASEC STKEKPDYQCKVYT GVYPFM
2 SINV YEH ATTVP NVPQIP YKAL VERAGYAP LNLEITVMSSEVLB STNOEYITCKFTT VVPSPK VKCCG SLEC QPAAAHADYTCKVFG GVYPFM
WEEV FEH ATTVP NVPGIP YKAL VERAGYAP LNLEIKVVSELTP STNKEYITCKFHT VIPSPQ VKCCG SLEC KASSKADYTCRVFG GVYPFM
EEEV YEH TAVM PNKVGIP YKAL VERPGYAP VHLQIQLVNTRIIIP STNLEYITCKYKT KVPSPV VKCCG ATQC SSKPHPDYQCQVFT GVYPFM
VEEV YEH ATTMP SQAGIS YNTIVNRAGYAP LPISITPTKIKLIP TVNLEYITCHYKT GMDSPA IKCCG SQEC TPTYRPDEQCKVFT GVYPFM

Wrapped
Intraspike
Interspike
3
5631

c" d e E0 F0 G0 H0 
90 100 110 120 130 140 150 160 170
1 CHIKV WGGAY YCF CDA EN TQL SEA HV EKSES CKTEF ASAYRAHTASASAKLRV LYQGNNITVA.AYA NGDHAVTVKDAKFVVGPMSAWT PFDN
MAYV WGGAY YCF CDS EN TM SEAY VERADV CKHDY AAAYRAHTASLRAKIKVTYGTVNQTVE.AYANGDHAVTIAGTKFIFGPVSTAWT PFDT
RRV WGGAY YCF CDS EN TQL SEA YV DRSDVC KHDHASAYKAHTASLKATIRISYGTTNQTTE.AFVNGEHAVNVGGSKFIFGPISSTAWS PFDN
ONNV WGGAY YCF CDA EN TQL SEA HV EKSES CKTEF ASAYRAHTASVASAKLRVFYQGNNITVS.AYANGDHAVTVKDAFVIGPLSSAWS PFDN
SFV WGGAY YCF CDS EN TQL SEA YV DRSDVC RHDHASAYKAHTASLKAKRVMYGNVNQTD.VYVNGDHAVTIGGTQFIFGPLSSAWT PFDN
2 SINV WGGAQ OCF CDS EN SQ MSEA YVELSADCATDHAQAIKVHTAAMKVGLRIVYGNNTTSFLD.VYVNGVTPGTSKDLKVIA GPISASFT PFDH
WEEV WGGAQ OCF CDS EN TQL SEA YVEFAPDC TIDHAVALRVHTAALKVGLRIVYGNTTAYLD.TFVNGVTPGSSRDLKVIA GPISA AFS PFDH
EEEV WGGAY YCF CDT EN TQM SEAYVERSEECSIDHAKAYKVHT GTVQAMVINITYGSVSWSRADVYVNGETPAKIGDAKLIII GPLSSAWS PFDN
VEEV WGGAY YCF CDT EN TQV SKAY VMKSDD CLADHAEAYKAHTASVQAFLINITVGEHSIVTT.VYVNGETPVNFNGVKITAGPL SSTAWT PFD R

Wrapped
Intraspike
Interspike
112
1 3 25 7 62 3 1 21

f g h h' i j α1 α2 k
180 190 200 210 220 230 240 250 260
1 CHIKV KIV VYKGDV YNM DYPPFGAGRPGQEGDIQSRTPESDKDYAN TOQLVRPAAGTVHVBYSOAPS GFKYW LKERGAS LQHT APFGC QIAT
MAYV KIV VYKGEV YNQDFPPYGAGQPGRFGDIOSRTLDSKDLYANTGLKLARPAAGNIHVBYTOQPSGFKTWQKDRDSP LNAKAPFGCTIQ T
RRV KIV VYKKDD VYNODFPPYGSGQPGRFGDIQSRTVESKDLYANTALKLSRPSPGVHVBYTQTPSGFKYW LKEKGSSLNTK APFGCKIKT
ONNV KIV VYKGEV YNM DYPPFGAGRPGQEGDIQSRTPDSKD VYANTQILILQRPAAGAIHVBYSOAPS GFKYW LKEKGASLQHT APFGCOIAT
SFV KIV VYKDEV FNQDFPPYGSGQPGRFGDIQSRTVESNDLYANTALKLARPSPGMVHVBYTQTPSGFKYW LKEKGTA LNTK APFGCOIKT
2 SINV KVV IRGLV VNYDFPEYCAMKP GAEGDIQATSLTSKDLIASTDIRLLKPSAKNVHVBYTQAA SGFEMWKNNNSGRBLQET APFGCKIAV
WEEV KVV IRKGL VNYDFPEYCAMKP GAEGDIQASILDATDIVARTLLRSVKNIVHVBYTQAVSGYEMWKNNNSGRBLQET APFGCKIEV
EEEV KVV VYGHE VNYDFPEGTGKAGSEGD IQSRTSTSN DLYANTNLKLQRQAGIVHTPFTQAPSGFERWRDKGAFLNDV APFGCSI AL
VEEV KIV QYAGE IYNYDFPEGAGQP GAEGDIOSRTVSSSDLYANTNLVLORPKAGAIHVBYTOAPSGFEQWKDKAPS LKFT APFGCEIYT

Wrapped
Intraspike
Interspike
1 54

32 3286

1 Io η2 A B C D C' E
270 280 290 300 310 320 330 340 350
1 CHIKV NPVARAVNC AVGNIP ISIDIPDAAFT RVVDAPS VTDMSC EVPACKTHSS DFGGV AIIKY TASKKGKC AVHSMTNAV TIREADVEVEGNSO
MAYV NPVARAMNC AVGNIP VSM DIADS AFTRLTDAPIISELLCTVSTC THSS DFGGV AAVLSY KVEKAGRC DVHSHSNVAVLQE..VSIEA EGR
RRV NPV RAMDC AVGSIP VSM IDIPDS AFTRVVDABAVTDLS CQVVVCTHSS DFGGV VATLSY KTDKP GKCAVHSHSNVATLQEBATVDVKEDGK
ONNV NPVARAVNC AVGNIP VSM IDIPDAAFTRVTDAPSITDMSC EBASC THSS DFGGV AAVIKY TASKKGKC AVHSVTNAV TIREBPNVDEGTAQ
SFV NPVARAMNC AVGNIP VSM NLPSDAFTRIVEAPITIDLCTVATC THSS DFGGV VLTLY KTDNK GDCSVHSHSNVATLQEATAKV TAGK
2 SINV NPLRAVDC SYGNIP ISIDIPNAAFIRTSDAPLVSTVK CDVSECTYSA DFGGM ATLQY VSDREQCPVHSHSSTATLQEESTVHVLEKGA
WEEV EPLRASNC AYGHIP ISIDIPDAAEVRSSESP TILEVSC .TADC IYSA DFGGS ILTLY KADREGHC PVHSHSTTAVLKEATTHV TATGS
EEEV EPLRAENC AVG SIP ISIDIPDAAFTRISETPTVSDLECKITECTYAS DFGGI ATVAY KSSKAGNCPIHSPSGVAVIKENDTVLAESGS
VEEV NPIRAENC AVGSTB LAFDIPDALFTRVSETPTLSAAEC TLNECVYSS DFGGI ATV KY SASKSGKC AVHVPSGTATLKEAAVELTEQGS

Wrapped
Intraspike
Interspike
3 4 2

E F G m El stem TM SEGMENT
360 370 380 390 400 410 420 430
1 CHIKV LQIS FST ALASAEFRVQVC STQVHCAAACHPEKDHIV NYPASH TTLGVQDISTAMS WVQKITGGVGLIVAVAALILI VVLCVSFSRH
MAYV SVIH FST ASAAPSEIVSVC SSRATCTAKEPEKDHVV TYPANHNGITLPDL SSTAMTWAQHLAGGVGLLIALAVLIL IVTCTILRR.
RRV VTVH FST ASASP AEKVSVCDAKTCTAAACEPKDHIV VPYGASHNNQVFP DMSTAMTWVQRLASGLGLALIAVVV LVLTCTIMRR.
ONNV LQIA FST ALASAEFKVQIC STLVHC SATCHPEKDHIV NYPSPH TTTLGVQDISTAMS WVQKITGGVGLVVAAI AALILI IVLCVSFSRH
SFV VTLH FST ASASPSEVVS LC SARATCSASC EPEKDHIV PYAASHSNVVF PDMSGTALS WVQKISGGLGFAIGAILVL VVVTCI GLRR.
2 SINV VTVHFST ASPQANEIVSLCGKKTTCNAECKPEADHIV VSTPHKNDQEFQA AISKTSSWLFLFGGASSILLIIG...LMIFACSMMLTS
WEEV ITLHFST SSPQANEIVSLCGKKTTCNAECKPEADHIT GEPHKVDQEFQA AVS KTSWNWLLALFGGASSLIVVG...IVLVCSSMLIN
EEEV FTTFH FST ANIHPAEKLQVCTSAVTKKGDCCKPEKDHIV DYP AQHTDLLTSAISATAWSWLKV LVGCTS AFI VLG...LIATAVV AL.VL
VEEV ATIH FST ANIHP EERLQICTSYVTC KGDCCHPEKDHIV THPQYHAQTFTA AVSKTAWTW LTSLLCGGS AVII IIIG...LVLATIVAMYVL

Wrapped
Intraspike
Interspike

Figure S4. Sequence Alignment of E1 Proteins of Arthritogenic and Encephalitic Alphaviruses, Related to Figure 5

Amino acid sequence alignment of E1 proteins of arthritogenic (CHIKV, ABX40011; MAYV, AAY45742; RRV, ACV67002; ONNV, AOS52786; SFV, AAM64227; and SINV, AUV65223) and encephalitic alphaviruses (WEEV, AAF60166; EEEV, AAF04796; and VEEV, AAB02517). Structure based sequence alignments were performed between alphaviruses that do (group 1, left margin) or do not (group 2, left margin) require Mxra8 for infection using PROMALS3D with CHIKV numbering. The Figure was prepared using ESPript 3.0. Domains I (light gray), II (medium gray), III (dark gray), the fusion loop (orange), and the E1 stem, TM segment, and cytoplasmic tail (black) are indicated above the sequence, along with the secondary structure features and nomenclature (PDB: 3N42; [Voss et al., 2010]). Red boxes, 100% conserved; white boxes and red letters; homologous residues; white boxes and black letters, non-conserved residues. Numbers under the alignment indicate contacts between Mxra8 and individual E2-E1 heterodimers, given as percentage buried surface area calculated by PDBePISA: 1 represents 10% buried surface area, 2 represents 20% buried surface area, and so on. “Wrapped” denotes contacts to the wrapped E2-E1 heterodimer making the primary contacts to Mxra8. “Intraspike” refers to the intraspikes heterodimer, which is adjacent to the wrapped heterodimer but within the same trimeric spike. “Interspike” refers to the interspike heterodimer, which is in the neighboring spike.

E3
| α1 | α2 | α3 |
| --- | --- | --- |
| 1 CHIKV | S.LALP.VLCILANTTFPCSQPBCTPCCYEKEPESTLRMLEDNVMRPGYYQLLKASLTCSP.HRQRST |
| MAYV | A.APTVTAMCILANVSFPCFQBS CSPCCYEGPEPTLRMLEENVNSEGYYELLHAAVCKNSSRSKRST |
| RRV | S.AAL..MMCILANTSFPCCSPPCYPCCCYEKQPEOTLRMLEDNVNRPGYYELLEASMTCRNRSRHRSV |
| ONNV | S.LALP.VMCILANTTFPCSQQPCAPCCYEKKPEETLRMLEDNVMQPGYYQLLDSALACSQ.RRQRNA |
| SFV | S.APLITAMCVLANATFPCFQPCVPCCYENNAEATLRMLEDNVDRPGYYDLLQAALTCRNGTRHRSV |
| 2 SINV | SAAPLVTTAMCLLGNSVFPCNRPP...TCYTREPSRAIDILEENVNHEAYDTLLNAILRCGS.SGRS..K |
| WEEV | S...LVTALCVLSNVTFPCDKPP...VCYSLAPERITLDVLEENVNDNPNYDTLLLENVLKCPS..RRP..K |
| EEEV | S...LATVMCLANITFPCDQPCPMCCEKNPHETLTMLLEQN YDSRTYDQLLDAAVKCNA.RRTRRDL |
| VEEV | S...LVTTMCLLANVTFFCAQPP...ICYDRKPAETLAMLSVNVDNPGYDELLLEAAVKCPG..RKRRST |

Wrapped  
Intraspikes  
Interspike  

E2
| η1 | i1 | i2 | AA | Ba | i3 | i4 | i5 | i6 | CA |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 CHIKV | NFNVYKATRPYPYLACHCPDCGE GHSCHSPIALERIRNEATDGTLKI OIVSLQIGIKTDDSHDWTKLRYMD ...SHTPADAERAGLLVR |
| MAYV | HFNAYKLTRPYVAYCADCGMGHSC HSPAMIBENVQADATDGTLKIQFAS QIGLTKADTHDHTKIRYAE...GH DIAEAARSTLK VH |
| RRV | HFNVYKATRPYLAXCADCGDGYFC YSPVAIEKIRDDEASDGMLKIQVSA OIGLDKAGTHAHTKIRYMA...GHD VQESKRDSL RVY |
| ONNV | NFNVYKVTRBYPYLAHC PDCGE GHSCHSPIALERIRSEATDGTLKIQVSLQIGIKTDDSHDWTKLRYMD ...SHTPVDAARSGLFVR |
| SFV | HFNVYKATRPYIAYCADCGAGHSC HSPVAIEAIRSEATDGMLKIQFSAO IGIDKSNDNHDTKIRYAD...GHA IENAVRSSLKVA |
| 2 SINV | ...DFTLTS PYLGTC SYCHHTEPCFPSPIKEQVWDEADDNTIRIQTSAQFGYDQSGAASSNKYRYSLEQDHTVKEGTMDDIKIS |
| WEEV | ...DFTLTS PYLGFCPNCRHSAPCFSPIKEINVWEDSDGSIRIQVSAQFGYNQAGTADVTKFRYMSSYDHDHDI EEDSMEKIAIS |
| EEEV | HFTRYKLARBPIADCPNCGHSRCD SPIAIEEVRGDAHAGAIRIQTSAME GLKT DG.VDLAYMSFMN...GKTQKS IKIDNLHVR |
| VEEV | LFNEYKLTRPYMARCCIRC AV.GSCHSPIAIEAVKSDGDGYVRLQTSSQYG LDSSGNLKGR TMRYDM...HGTIKEIPLHQVSLY |

Wrapped  
Intraspikes  
Interspike  

| DA | EA | FA | GA | b | c |
| --- | --- | --- | --- | --- | --- |
| 1 CHIKV | TSAPCTITGTMGHFILARC PKGETLTVGFTDSRKISHTC THPFHHEPPVIGRERFHSRPQHG KELPCSTYVQS TAATAEETEVHM |
| MAYV | SSSEC AVTGTMGHFILAKCPPGEVISVSFVDSKNEQRTCRIAYHHEQRLIIGREFTVRPHHGIELPC TTYQLTTAETSEEIDMHM |
| RRV | TSAACSIHGTMGHFIVAHCPPGDYLKVSFEDADSHVKA CKVQYKHDP LPVVGREKFVVRPHFGVELPC TSYQLTTAPTDEEIDMHT |
| ONNV | TSAPCTITGTMGHFILARC PKGETLTVGFVDSRRISHTCMHPFHHEPPLIIGREKFHSRPQHGKELPC STYVHTTAATAEETEVHM |
| SFV | TSGDCFVHGTMGHFILATCPPEGFLQVS IQDTRNAVRA CRIQYHHD PQPVVGREKFTIRPHYGKEIPCGTTYQQT TAETVEEIDMHM |
| 2 SINV | TSGPCRRLSYKGYFLLAKCPPGDSVTVSIASSN.SATSC TMARKIKPKF FVGREKYDLPPVHGKIPCTVYDRLKETTAGYITMHR |
| WEEV | TSGPCRRLGHKGYFLLAQCPPGDSVTVSITSGA.SENS CTVEFKIRRKFVGREEYLFPPVHGKRVKCHVYDHLKERSAGYITMHR |
| EEEV | TSAPCSLVSHHGYYILAOCPPTVTVGFHDGP.NRHTCTVAHKVEFRPVGREERYHPEHVELPC NRYTHKRADQGHYVEMHQ |
| VEEV | TSRPCHIVDGHGYFLLARCPAGDSITMEFKKDS.VRHSCSVPYEVKFNPVVGRELYTHPEHGVEOAQCQVYAHD AQNRGAYVEMHL |

Wrapped  
Intraspikes  
Interspike  

| i7 | η2 AB | BB | CB | C'B | EB | η3 | FB | d |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 CHIKV | PPDTPDRTLMTQOSGNVKITV.NGQTVRYKCNGGS.NEGLTTTDKVINNCK.IDQCHAAVTNHNKWQYNSPLVPRNAELGDRKG |
| MAYV | PDI PDRTILSQOSGNVKITV.NGRTVKYSCSGSK.PSGTTTTDKTINSCT.VDKCQAYVTSHTKWQFNSPFVPR.AEQAE RKG |
| RRV | PDI PDRTLLSQTAGNVKITA.GGRTIRYNCTCGRD.NVGTTS TDKTINTCK.IDQCHAAVTSHDKWQFTSPFVPR.ADQTARRG |
| ONNV | PDTDYTLMTQAGNVKITV.BGQTVRYKCKCDGS.NEGLITADKVINNCK.VDQCHTAVTNHHKWQYNSPLTPRNSEQGDRKG |
| SFV | PDTDRTLLSQOSGNVKITV.GGKKVKYNCTCGTG.NVGTTNSDMTINTCL.IEOCHVSVTDHKKWQFNSPFVPR.ADEPARKG |
| 2 SINV | GPHAYTSYLEESSGKVYAKPPSGKNITYECKCGDY.KTGTVVTRTEITGC TAIKQC VAYKSDQTKWFNSPDLIR.HADHTAQG |
| WEEV | SPHAYKSYLKEASEGEVYIKPDSGKNVTYECKCGDY.STGIVSTPTKMNGCTKAKOQCIAYKSDQTKWFNSPDLIR.HTDHSVQG |
| EEEV | GLVADHSSL SIHSAKVKITVPSGAQVKYYCKC..PDVREGITSSDHTTCTDVKQORAYLIDNKKWYNFGRLLR.GEGDTFKG |
| VEEV | GSEVDSSSLVSLSGSSVTVTPPDGTSALVECECGGTKISETINKTKQFSQCTKKEQORAYRLQNDKWYNSDKLPK.AAGATLKG |

Wrapped  
Intraspikes  
Interspike  

| e | f | Ac | Bc | Cc | C'c | Ec | Fc | Gc |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 CHIKV | KTIHIPFPLANVT CRVPKARNPTVTYGKNQVTMLLYPDHPTLLSYRNMGQEPNYHEEWVTHKKEVTLTVPTEGLEVTWGNENE PYKY |
| MAYV | KVHIPFLINTTCRVP LAPEALVRSKREATLSLHPIHPTLLSYRTLGREPVFD EQWITTQTEVTIPVPVEGV EYRWGNHKPQRL |
| RRV | KVHVPFPLTNVT CRVP LARAPDVVTYGKEVTLRLIHPDHPTLLSYSRSLGAEPHPYEEWVDKFSERIIPVTEEGIEYQWGN NPPVRL |
| ONNV | KHIIPFLVNTTCRVPKARNPTVTYGKNRV TLLLYPDHPTLLSYRAMGRIPDYHEEWITNKEEISITVPAEGL ETVWGNNDPYKY |
| SFV | KVHIPFLDNI TC RVPMA REPTVIHGKREVTLHLH PDHPTLLSYSRTLGEDPQYHEEWV TAAVERTIPVPVDGM EYWGNNDP VRL |
| 2 SINV | KLHLPFKLIPSTCMVPVAHAPNVVHGFKHISLQD TDHLTLLTRRLGANPEPTTEWIIGKTVRNFTVDRDGL EYIWGNHEPVRV |
| WEEV | KLHIPFRLIPTVC PVPLAHTPTVTKWFKGITLHLTATRPTLLTRKLGLRADATAEWITGTTSRNFSWGREGL EYTWGNHEPVRV |
| EEEV | KLHVPFVPVKACIATLAPELLVEHKHRTLILHLHPDHP TLLTRSLGSDANPIRQWIERPTTVNETVTGEGL EYTWGNHPPKRV |
| VEEV | KLHVPFLLADGKCTVPLAPEPMITFGFRSVSLKHPKNPTYLITRQLADEPHYTHELISEPAVRNFTVTEKGWEFWGNHPPKRF |

Wrapped  
Intraspikes  
Interspike  

| E2 stem | TM SEGMENT | Cytoplasmic tail |
| --- | --- | --- |
| 1 CHIKV | WPQ MSTNGTAHGHPHEII LYYYELYPTMVIVSVASFVLLSMVGTA VGMCVCA RRC CI TPYELTPGATVPFLLSLILCCV RTTKA |
| MAYV | WSQ LT TDGRAHGWPHEIII EYYYGLHBTTTIVVVVAAVSVVLLSVAASVYMCVVARNKCLTPYALTPGAVVPVTIGVLCCAPKAHA |
| RRV | WAQ LTTEGKPHGWPEIIQ YYYGLYPAATTIAAVSGASLMALLTLAATCCMLATARRKCLTPYELTPGATIPFLLGVILCCA PRATA |
| ONNV | WPQ LSTNGTAHGHPHEII LYYYELYPTTTIAVLAAA SIVITSLVGSLGMCICARRRCI TPYELTPGATIPFLLGVILCCA PRATA |
| SFV | WSQ LTTEGKPHGWPEIVQ YYYGLYPAATVSAVVGM SLLALISIFASCYMLVAARRSKCLTPYALTPGAVVPWTLGIILCCA PRAHA |
| 2 SINV | YAQ ESAPGDPHGWPHEIVQ HYYHRHPVYTI LAVASAAVAMMI GVTVAALCAC KARRBCLTPYALAPNATVPTALAVALCCIRPTNA |
| WEEV | WAQ ESAPSDDHGWPHEII IHYYHRHPVYTVIVLCGVALA ILVG TASSAACITKARRDCLTPYALAPNATVPTALAVALCCIRPTNA |
| EEEV | WAQ ESGEGNPHGWPEVVYY NRYPLTTIGLCTCVAITIMVSCVTSVWLLCRA RNLCITPYKLAPNAQVIPILLALILCCA IKPTKA |
| VEEV | WAQ ETAPGNPHGLPHEVI THYYHRYP MSTI LGLSICAAIATVSVAASTWLFCRSRVACLTPYRLTPNARIPFCLAVALCCA RTARA |

Wrapped  
Intraspikes  
Interspike  

Figure S5. Sequence Alignment of p62 (E3-E2) Proteins of Arthritogenic and Encephalitic Alphaviruses, Related to Figure 5

Amino acid sequence alignment of p62 (E3 and E2) proteins of arthritogenic (CHIKV, ABX40011; MAYV, AAY45742; RRV, ACV67002; ONNV, AOS52786; SFV, AAM64227; and SINV, AUV65223) and encephalitic alphaviruses (WEEV, AAF60166; EEEV, AAF04796; and VEEV, AAB02517). Structure based sequence alignments were performed between alphaviruses that do (group 1, left margin) or do not (group 2, left margin) require Mxra8 for infection using PROMALS with CHIKV numbering. The Figure was prepared using ESPript 3.0. E3 (yellow), and E2 domains A (light cyan), B (medium cyan), the β-linker (medium cyan),

(legend continued on next page)

and the stem, TM segment, and cytoplasmic tail (dark blue) are indicated above the sequence, along with the secondary structure features and nomenclature (PDB 3N42; *Voss et al.*, 2010). Red boxes, 100% conserved; white boxes and red letters: homologous residues; white boxes and black letters, non-conserved residues. Numbers under the alignment indicate contacts between Mxra8 and individual E2-E1 heterodimers, given as percentage buried surface area calculated by PDBePISA: 1 represents 10% buried surface area, 2 represents 20% buried surface area, and so on. “Wrapped” denotes contacts to the wrapped E2-E1 heterodimer making the primary contacts to Mxra8. “Intraspike” refers to the intraspikes heterodimer, which is adjacent to the wrapped heterodimer but within the same trimeric spike. “Interspike” refers to the interspike heterodimer, which is in the neighboring spike.

Figure S6. Unique E1 Contacts by Mxra8 at Site 1, Related to Figure 4

(A) Top view of the asymmetric unit of CHIKV VLP with labeled Mxra8 binding sites. Site 1 is the Mxra8 (magenta) site that makes additional contacts to domains I and II of E1 directly beneath it (gray). E2, cyan; capsid, green. (B and C) Side views (top panels) and zoomed-in views (bottom panels) of site 1 (B) and site 2 (C), highlighting the unique residues of E1 (orange) that contact site 1 of Mxra8 binding. Contacts were identified from the atomic model using PDBePISA ([http://www.ebi.ac.uk/pdbe/pisa/](http://www.ebi.ac.uk/pdbe/pisa/)).

A

4E7.D10

B

| Start | End | Sequence                     | 1G11.E6 | 1H1.F5 | 3G2.F5 | 4E7.D10 | 7F1.D8 | 8F7.E1 | 9G2.D6 | Isotype |
|-------|-----|------------------------------|---------|--------|---------|---------|--------|--------|---------|----------|
| 39    | 43  | SVVSL                        |         |        |         |         |        |        |         |          |
| 41    | 48  | VSLAAGTQ                     |         |        |         |         |        |        |         |          |
| 44    | 51  | AAGTQAVL                     |         |        |         |         |        |        |         |          |
| 44    | 50  | AAGTOAV                       |         |        |         |         |        |        |         |          |
| 91    | 119 | YSAGEQRVYEPDRPRLLSPSAFHDGNF  |         |        |         |         |        |        |         |          |
| 91    | 107 | YSAGEQRVYEPDRPRPL             |         |        |         |         |        |        |         |          |
| 96    | 107 | QRVYEPDRPRL                  |         |        |         |         |        |        |         |          |
| 97    | 107 | RVYEPDRPRL                   |         |        |         |         |        |        |         |          |
| 108   | 119 | LLSPSAFHDGNF                 |         |        |         |         |        |        |         |          |
| 108   | 114 | LLPSAF                        |         |        |         |         |        |        |         |          |
| 150   | 173 | AVRLEVTEDPLLSRAYWDGEKEL      |         |        |         |         |        |        |         |          |
| 154   | 173 | EVTEDPLLSRAYWDGEKEL          |         |        |         |         |        |        |         |          |
| 154   | 164 | EVTEDPLLSRA                   |         |        |         |         |        |        |         |          |
| 162   | 173 | SRAYWDGEKEL                  |         |        |         |         |        |        |         |          |
| 165   | 173 | YWDGEKEL                      |         |        |         |         |        |        |         |          |
| 183   | 202 | MTCINRAHVWTDHRHLEEAQQ        |         |        |         |         |        |        |         |          |
| 183   | 199 | MTCINRAHVWTDHRHLEE           |         |        |         |         |        |        |         |          |
| 184   | 202 | TCINRAHVWTDHRHLEEAQQ         |         |        |         |         |        |        |         |          |
| 184   | 199 | TCINRAHVWTDHRHLEE            |         |        |         |         |        |        |         |          |
| 185   | 202 | CINRAHVWTDHRHLEEAQQ          |         |        |         |         |        |        |         |          |
| 185   | 199 | CINRAHVWTDHRHLEE             |         |        |         |         |        |        |         |          |
| 186   | 199 | INRAHVWTDHRHLEE              |         |        |         |         |        |        |         |          |
| 225   | 248 | YASGERRAYGPPFLRDRVSVTNA      |         |        |         |         |        |        |         |          |
| 225   | 240 | YASGERRAYGPPFLRD             |         |        |         |         |        |        |         |          |
| 241   | 248 | RVSVTNA                       |         |        |         |         |        |        |         |          |
| 249   | 256 | FARGDFS                       |         |        |         |         |        |        |         |          |
| 249   | 255 | FARGDFS                       |         |        |         |         |        |        |         |          |
| 249   | 254 | FARGDF                        |         |        |         |         |        |        |         |          |

C

D

Figure S7. Epitope Mapping of Hamster Anti-Mxra8 mAbs by Hydrogen-Deuterium Exchange and Mass Spectrometry, Related to Figure 5  
(A) Representative HDX kinetic plots of peptides from mAbs of each of the 3 classified groups, both in the presence (black lines) and absence (red lines) of Mxra8. Regions are considered to contain the binding epitopes if they show reduced rates or extents of hydrogen-deuterium exchange. All experiments were performed in duplicate, and data are representative of two experiments.

(legend continued on next page)
(B) Heatmap depicting the average difference of deuterium incorporation between the mAb alone and with Mxra8 present across all seven time points (ΔD%). Coloring of the peptides correspond to the mAb classification. Yellow: mAbs 4E7.D10, 8F7.E1; violet: mAbs 1G11.E6, 7F1.D8; green: mAbs 1H1.F5, 3G2.F5, 9G2.D6. Consensus peptides for epitope designation and depiction: 41–51, light violet; 91–107, yellow; 162–173, light green; 185–202, green; 241–256, violet.

(C and D) Surface and ribbon diagrams of Mxra8 at site 3 and the E2-E1 domains at its binding interface. CHIKV structural proteins are colored by domain. E1: DI, light gray; DIi, medium gray; fusion loop, orange. E2: A domain, light cyan; β-linker, medium cyan; domain B, dark cyan. ’ denotes domains within the intraspike heterodimer, and ” for the interspike heterodimer; the wrapped heterodimer is labelled without symbols. Surface representation of Mxra8 models are colored with the HDX-mapped epitopes of anti-Mxra8 mAbs, where shades of violet correspond to mAb 1G11.E6, green for mAbs 1H1.F5 and 3G2.F5, and yellow for mAbs 4E7.D10 and 8F7.E1. (C) Mxra8 model shown is of the properly docked model, with D2 proximal to the viral membrane. (D) Alternative Mxra8 model where the Mxra8 docking was flipped, positioning D1 proximal to the viral membrane and D2 extending away from the virion. This model underwent an identical building and refinement process as described in the **STAR Methods** and resulted in a poorer fit into the cryo-EM density map as evident by visual inspection and a lower correlation coefficient.
Figure S8. Presence of E3 Can Sterically Hinder Mxra8 Binding, Related to Figure 6

(A) View of asymmetric unit of CHIKV infectious particles with difference map of Mxra8 docked on. E1, E2, E3, and Mxra8 sites are labeled, with the high-affinity site 1 labeled with a square.

(B–D) Zoomed-in views highlight the amount of steric clashing of Mxra8 at site with E3 from the adjacent spike. We calculated the surface area of occluded Mxra8 density (zoned within 2 Å of E3 coordinates) as follows: site 1 (B), 19.6 Å²; site 2 (C), 218.4 Å²; site 3 (D), 421.4 Å²; and site 4 (E), 400.3 Å².

Color scheme for (A–E): gray, E1; cyan, E2; transparent yellow, E3; capsid, green; Mxra8, magenta. Prime labels refer to Mxra8 sites on adjacent spikes.

(F) Cartoon schematic summary of results: the red triangle and the dashed lines depict two distinct representations of one icosahedral asymmetric unit. The 5-fold (i5), 3-fold (i3), and 2-fold (i2) icosahedral axes of symmetry are indicated with a black pentagon, triangles, and oval, respectively. The white triangles and ovals indicate the quasi-3-fold and 2-fold axes of symmetry, respectively. Trimeric spikes are labeled “i3” if coincident with the i3 axes, and “q3” if on a quasi-3-fold axes. Color scheme: gray, E1; cyan, E2; transparent yellow, E3; site 1 Mxra8, magenta; sites 2, 3, and 4 Mxra8, purple. X denotes sites of large steric clashes.

Figure S9. Human Mxra8 Binding to CHIKV VLPs, Related to Figure 6

(A) SEC-MALS of cleaved human Mxra8 showing a monomeric form at ~34 kDa.

(B and C) Kinetic sensograms and steady-state analysis of human Mxra8 binding to CHIKV VLPs fit to a 1:1 binding model (B) or a two-site binding model (C) (one high-affinity site and three low but equal affinity sites). Raw experimental traces are in black, fit traces are in red. Inset: Scatchard plot (2 experiments; mean, standard error of the mean [SEM], and χ² values).
