
Received: 13 February 2019 | Revised: 5 July 2019 | Accepted: 7 July 2019  
DOI: 10.1002/humu.23865  

RESEARCH ARTICLE  

**Novel SCA19/22-associated KCND3 mutations disrupt human Kv4.3 protein biosynthesis and channel gating**

Cheng-Tsung Hsiao${}^{1,2,3,4*}$ | Ssu-Ju Fu${}^{3,5*}$ | Yo-Tsen Liu${}^{2,4}$ | Yi-Hsiang Lu${}^{5}$  
Ciao-Yu Zhong${}^{5}$ | Chih-Yung Tang${}^{3}$ | Bing-Wen Soong${}^{2,6,7,8†}$ | Chung-Jiuang Jeng${}^{5,6†}$  

${}^{1}$Department of Internal Medicine, Taipei Veterans General Hospital Taoyuan Branch, Taoyuan, Taiwan  
${}^{2}$Department of Neurology, Taipei Veterans General Hospital, Taipei, Taiwan  
${}^{3}$Graduate Institute of Physiology, College of Medicine, National Taiwan University, Taipei, Taiwan  
${}^{4}$Department of Neurology, School of Medicine, National Yang-Ming University, Taipei, Taiwan  
${}^{5}$Institute of Anatomy and Cell Biology, School of Medicine, National Yang-Ming University, Taipei, Taiwan  
${}^{6}$Brain Research Center, National Yang-Ming University, Taipei, Taiwan  
${}^{7}$Department of Neurology, Shuang Ho Hospital, Taipei Medical University, New Taipei City, Taiwan  
${}^{8}$Taipei Neuroscience Institute, Taipei Medical University, Taipei, Taiwan  

Correspondence  
Chung-Jiuang Jeng, Institute of Anatomy and Cell Biology, School of Medicine, National Yang-Ming University, #155, Section 2, Linung Street, Peitou District, 112 Taipei, Taiwan. Email: cjjen@ym.edu.tw  

Bing-Wen Soong, Department of Neurology, Shuang Ho Hospital, and Taipei Neuroscience Institute, Taipei Medical University, #291, Zhongzheng Rd., Zhonghe District, 235 New Taipei City, Taiwan. Email: bwsoong@gmail.com  

Funding information  
Ministry of Science and Technology, Taiwan, Grant/Award Numbers: 105-2320-B-010-016-MY3 (C.J.J.), 103-2314-B-010-049-MY3, 104-2745-B-010-004, 106-2321-B-010-010, 107-2314-B-010-017 (B.W.S.); National Yang-Ming University School of Medicine, Grant/Award Number: 107F-M01 (C.J.J.); Ministry of Education, Taiwan, Grant/Award Numbers: The Higher Education Sprout Project to the Brain Research Center, National Yang-Ming University (C.J.J.)  

---

**Abstract**  
Mutations in the human voltage-gated K⁺ channel subunit Kv4.3-encoding KCND3 gene have been associated with the autosomal dominant neurodegenerative disorder spinocerebellar ataxia types 19 and 22 (SCA19/22). The precise pathophysiology underlying the dominant inheritance pattern of SCA19/22 remains elusive. Using cerebellar ataxia-specific targeted next-generation sequencing technology, we identified two novel KCND3 mutations, c.950G>A (p.C317Y) and c.1123C>T (p.P375S) from a cohort with inherited cerebellar ataxias in Taiwan. The patients manifested notable phenotypic heterogeneity that includes cognitive impairment. We employed in vitro heterologous expression systems to inspect the biophysical and biochemical properties of human Kv4.3 harboring the two novel mutations, as well as two previously reported but uncharacterized disease-related mutations, c.1013T>A (p.V338E) and c.1130C>T (p.T377M). Electrophysiological analyses revealed that all of these SCA19/22-associated Kv4.3 mutant channels manifested loss-of-function phenotypes. Protein chemistry and immunofluorescence analyses further demonstrated that these mutants displayed enhanced protein degradation and defective membrane trafficking. By coexpressing Kv4.3 wild-type with the disease-related mutants, we provided direct evidence showing that the mutants instigated anomalous protein biosynthesis and channel gating of Kv4.3. We propose that the dominant inheritance pattern of SCA19/22 may be explained by the dominant-negative effects of the mutants on protein biosynthesis and voltage-dependent gating of Kv4.3 wild-type channel.

---

*Cheng-Tsung Hsiao and Ssu-Ju Fu are joint first authors.  
Bing-Wen Soong and Chung-Jiuang Jeng are joint senior authors.*

2088 © 2019 Wiley Periodicals, Inc. wileyonlinelibrary.com/journal/humu Human Mutation.2019; 40:2088-2107.

HSIAO ET AL.

KEYWORDS
channelopathy, ion channel, molecular genetics, protein misfolding, spinocerebellar ataxia

1 | INTRODUCTION

A wide variety of voltage-gated K⁺ (Kᵥ) channels contribute to setting membrane excitability in distinct subcellular compartments of neurons. In the somatodendritic compartment, the subthreshold A-type transient K⁺ current (Iₛₐ) activates and inactivates rapidly and plays a critical role in somatodendritic signal integration (Hoffman, Magee, Colbert, & Johnston, 1997; Nadal, Amarillo, Vega-Saenz de Miera, & Rudy, 2001). The molecular organization of a subset of Iₛₐ comprises the pore-forming Kᵥ4.3 channel subunit and the auxiliary subunit K⁺ channel-interacting protein (KChIP; An et al., 2000; Dilks, Ling, Cockett, Sokol, & Numann, 1999; Jerng, Pfaffinger, & Covarrubias, 2004; Serodio & Rudy, 1998). Kᵥ4.3 is expressed in a vast region of the brain, including the cerebral cortex, hippocampus, pons, and cerebellum (Dilks et al., 1999; Isbrandt et al., 2000; Ohya et al., 2001; Serodio, Vega-Saenz de Miera, & Rudy, 1996). In the cerebellum, Kᵥ4.3 is present in deep nuclei, granule cells, Purkinje cells, and various interneurons (Kollo, Holderith, & Nusser, 2006). Intriguingly, Kᵥ4.3 expression patterns in both granule and Purkinje cells appear to change during cerebellar development, implying an additional role of the K⁺ channel in postnatal migration of cerebellar neurons (Hsu, Huang, & Tsaur, 2003).

Spinocerebellar ataxia (SCA) refers to a heterogeneous group of autosomal dominant neurodegenerative disorders characterized by progressive cerebellar ataxia along with variable pyramidal, extrapyramidal, cerebral, or spinal cord symptoms (Durr, 2010; Schols, Bauer, Schmidt, Schulte, & Riess, 2004). Based on the chronological order of chromosomal localization of disease-related genes, SCAs have been numerically classified into more than 40 subtypes (Soong & Morrison, 2018). Among them, SCA19/22 (MIM# 607346) is associated with mutations in the KCND3 gene that encodes the Kᵥ4.3 subunit (Duarrí et al., 2012; Lee et al., 2012). To date, 15 SCA19/22-associated KCND3 mutations have been identified in patients with cerebellar ataxia (Choi et al., 2017; Coutelier et al., 2017, 2018; Duarrí et al., 2012, 2013; Huin et al., 2017; Kurihara et al., 2018; Lee et al., 2012; Paucar et al., 2018; Smets et al., 2015).

As a result of limited data on the biophysical and/or biochemical abnormalities of disease-causing Kᵥ4.3 mutant channels (Duarrí et al., 2012, 2015; Lee et al., 2012), the precise molecular pathophysiology underlying SCA19/22 remains elusive. Specifically, despite the fact that virtually all the characterized Kᵥ4.3 mutants display loss-of-function phenotypes, it is inconclusive whether these mutants may exert a significant dominant-negative effect on the functional expression of their wild-type (WT) counterpart. Moreover, although some of the mutants are associated with defective protein stability, it is still an open question whether they may instigate enhanced protein degradation of Kᵥ4.3 WT. In this report, we addressed these critical questions by characterizing the clinical and molecular features of four SCA19/22-associated KCND3 mutations from a Han Chinese cohort with cerebellar ataxia in Taiwan. Our biophysical and biochemical analyses underscore the pathogenic role

of impaired protein biosynthesis and altered channel gating in the dominant-negative effect of SCA19/22-causing Kᵥ4.3 mutants.

2 | MATERIALS AND METHODS

2.1 | Subjects

From 585 unrelated pedigrees with inherited cerebellar ataxia, we selected 93 molecularly unassigned individuals, in whom molecular diagnoses of SCA1, 2, 3, 6, 7, 8, 10, 12, 17, 31, 35, and 36 and dentatorubral-pallidoluysian atrophy had been excluded (Hsiao et al., 2017). All the recruited subjects underwent standard clinical and neurological evaluations, including Scale for the Assessment and Rating of Ataxia (SARA), to assess neurological functions (Lee et al., 2011; Schmitz-Hubsch et al., 2006). Magnetic resonance image (MRI) and magnetic resonance spectroscopy (MRS) were performed to assess structural and metabolite changes, respectively, of the cerebellum (Chen et al., 2014). The protocol for this study was approved by the institutional review board of Taipei Veterans General Hospital and all participants provided written informed consents.

2.2 | Genetic analyses

Genomic DNA was extracted from white blood cells in the peripheral blood with standard protocols. A next-generation sequencing (NGS) approach utilizing a cerebellar ataxia-specific targeted sequencing panel was carried out to sequence 101 cerebellar ataxia-associated genes as reported previously (Hsiao et al., 2017). This targeted sequencing panel covered the whole sequences of all eight exons and the exon/intron boundaries of KCND3 (reference sequence: NM_004980.4). Sanger sequencing was further used to verify the sequence changes identified in the first stage and to analyze the genotype-phenotype cosegregation in the pedigrees harboring KCND3 variants. In apparently sporadic cases, paternity tests were implemented by analyzing 14 microsatellite markers, including (a) the trinucleotide repeat length in HTT, ATXN1, ATXN2, ATXN3, CACNA1A, ATXN7, ATXN8, PPP2R2B, TBP, and ATN1, (b) pentanucleotide repeat length in ATXN10, and (c) three sets of polymorphic short tandem repeats, PMP22-4A, PMP22-9A, and PMP22-9B in PMP22 (Liao et al., 2015). All KCND3 sequence variants characterized in this study have been submitted to ClinVar database (https://www.ncbi.nlm.nih.gov/clinvar/).

2.3 | Analyses of the pathogenicity of sequence variants

Several bioinformatics tools, including SIFT (Kumar, Henikoff, & Ng, 2009), SNPs&GO (Capriotti et al., 2013), PolyPhen-2 (Adzhubei et al., 2010), and MutationTaster (Schwarz, Cooper, Schuelke, & Seelow, 2014) were used to assess the pathogenicity of KCND3 variants.

also searched for these sequence variants on genetic databases, including Genome Aggregation Database (gnomAD; http://gnomad.broadinstitute.org) and Taiwan Biobank (https://taiwanview.twbiobank.org.tw/index). Amino acid sequences of Kᵥ4.3 homologs and orthologs from multiple species were aligned to determine whether the variations occur at phylogenetically conserved positions.

### 2.4 | Protein homology modeling

Protein homology model of human Kᵥ4.3 was generated with UCSF Chimera (version 1.12, http://www.cgl.ucsf.edu/chimera/) that is interfaced with Modeller (version 9.2, https://salilab.org/modeller/; Eswar et al., 2006; Pettersen et al., 2004). The Kᵥ4.3 model is based on the crystal structure of the chimeric Kᵥ1.2-Kᵥ2.1 channel (PDB ID: 2R9R; Long, Tao, Campbell, & MacKinnon, 2007), with 63% homology (35% identity) in overall sequence and 94% homology (57% identity) in the region from transmembrane S4–S6 segments. Figures were prepared with UCSF Chimera.

### 2.5 | Expression plasmids

Amino-terminal Myc-, Flag-, and GFP-tagged Kᵥ4.3 (Myc-Kᵥ4.3, Flag-Kᵥ4.3, and GFP-Kᵥ4.3) were generated by subcloning human Kᵥ4.3 complementary DNA (cDNA; NM_004980.4) into modified pcDNA3 (Invitrogen) and pEGFP (Clontech) vectors. Disease-associated Kᵥ4.3 mutant constructs were generated by using the QuikChange Site-Directed Mutagenesis Kit (Stratagene). Amino-terminal HA-tagged KChIP2 (HA-KChIP2) was created by subcloning human KChIP2 cDNA (AF199598) into pcDNA3-HA vector (Invitrogen). All the constructs were verified by DNA sequencing.

### 2.6 | Electrophysiology

Oocytes from adult female Xenopus laevis (African Xenopus facility) were used for functional studies. All animal procedures were in conformity with the animal protocol approved by the Institutional Animal Care and Use Committee (IACUC) of National Taiwan University. Xenopus frogs were anesthetized by immersion in ice water containing Tricaine (1.5 g/L) to dissect ovarian follicles, which were cut into pieces and incubated in ND96 ([in mM]: 96 NaCl, 2 KCl, 1.8 MgCl₂, 1.8 CaCl₂, and 5 HEPES, pH 7.2). Follicular membrane of Xenopus oocytes was removed by incubating in Ca²⁺-free ND96 solution containing collagenase (2 mg/mL) on an orbital shaker (200 rpm) for 60–90 min at room temperature. After several washes with collagenase-free, Ca²⁺-free ND96, Stage V–VI oocytes were selected for cRNA injection. For in vitro transcription, appropriate restriction enzymes were applied to linearize cDNAs, from which capped cRNAs were transcribed using the mMessage mMachine T7 kit (Ambion). Unless specified otherwise, cRNAs for WT Kᵥ4.3, mutant Kᵥ4.3, and KChIP2 were mixed in a molar ratio of 1:1:2 and injected in a total volume of 41.4 nl. Injected oocytes were stored in ND96 at 16°C for 2–3 days before being used for functional analyses.

Two-electrode voltage-clamp with OC-725C oocyte clamp (Warner Instruments) was used to record K⁺ currents through Kᵥ4.3 channels. The recording bath contained Ringer solution ([in mM]: 3 KCl, 115 NaCl, 1.8 CaCl₂, 10 HEPES, and 0.4 niflumic acid, pH 7.4 with methanesulfonic acid). With an agarose bridge, the bath solution was connected to a ground chamber (containing 3 M KCl), into which two ground electrodes were inserted. Borosilicate electrodes (0.1–1 MΩ) filled with 3 M KCl were used for voltage recording and current injection. Data were acquired and digitized via Digidata 1440A using pCLAMP 10.2 (Molecular Devices). Oocytes were held at –90 mV and leak currents arising from passive membrane properties were subtracted by using the –P/4 method provided in the pCLAMP system. Data were filtered at 1 kHz and digitized at 10 kHz, and all data were recorded at the room temperature (20–22°C). We adopted a previously data normalization procedure (Jeng, Chen, Chen, & Tang, 2006) to avoid potential biases imposed by channel expression variations among different batches of oocytes.

To study voltage-dependent activation property of Kᵥ4.3, oocytes were subject to 500-ms test pulses, ranging from –60 mV to +60 mV in 10-mV increment, to generate peak K⁺ current-voltage (I–V) curve. From peak I–V curve, relative K⁺ channel conductance (G/Gₘₐₓ) at a given test potential was calculated as G/Gₘₐₓ = ΔIₖ/ΔIₖ,+6₀, where ΔIₖ is current increment determined from peak K⁺ current difference between adjacent test pulses, and ΔIₖ,+6₀ is the ΔIₖ at the test potential +60 mV. Steady-state voltage-dependent activation of Kᵥ4.3 was then analyzed by fitting G/Gₘₐₓ-voltage (G–V) curve with a Boltzmann equation: G/Gₘₐₓ = 1/{1 + exp [(V₀.₅ₐ – V)/kₐ]}, where V₀.₅ₐ is the half-activation voltage, and kₐ is the activation slope factor. To study steady-state fast inactivation, oocytes were subject to 1-s prepulse stepping from –120 to +10 mV with 10-mV increment, followed by a +60-mV test pulse for 500 ms. Normalized peak K⁺ currents at +60 mV (I/Iₘₐₓ) were then plotted against corresponding prepulse potentials. Steady-state voltage-dependent inactivation of Kᵥ4.3 was then analyzed by fitting I/Iₘₐₓ-voltage curve with a Boltzmann function: I/Iₘₐₓ = 1/{1 + exp [(V₀.₅ᵢ – V)/kᵢ]}, where V₀.₅ᵢ is the half-inactivation voltage, and kᵢ is the inactivation slope factor. Window current analyses were determined from the apex and the area of the triangular overlap area between activation and inactivation curves.

### 2.7 | Cell culture and transfection

Human embryonic kidney (HEK) 293T cells were maintained in Dulbecco's modified Eagle medium (DMEM) supplemented with 2 mM L-glutamine, 10% fetal bovine serum (HyClone), and 100 units/ml penicillin/streptomycin with 95% air and 5% CO₂ in a humidified incubator at 37°C, and passaged every 3–4 days. Cells were plated onto 6-, 12-, or 24-well cell culture plates 24 hr before transfection. Transient transfection was performed by using standard calcium phosphate method. Unless stated otherwise, 0.5–1 μg of Kᵥ4.3 cDNA was added into each well. For coexpression experiments, cDNAs for WT Kᵥ4.3, mutant Kᵥ4.3, and KChIP2 were mixed in a molar ratio of 1:1:2. Transfected cells were maintained at 37°C for 48 hr before being processed for further experiments. Where necessary, at 24 hr posttransfection, cells were treated with the proteasome inhibitor

HSIAO ET AL.

(a) Pedigree A


KCND3 c.950G>A (p.C317Y)
A.III-1
ACACTGAAGAGCTGTGCCTCCGAAC TG
Thr Leu Lys Ser Cys Ala Ser Glu Leu


(b) Pedigree B


KCND3 c.1123C>T (p.P375S)
B.I-2
GGAGACATGGTGCC TAA GACGATTGCA
Gly Asp Met Val Pro Lys Thr Ile Ala
Ser


(c)

(d)

FIGURE 1 Clinical characterization of novel disease-associated KCND3 mutations. (a) and (b) Pedigrees of the individuals carrying novel KCND3 mutations are shown on the left, and electropherograms of the individuals who underwent genetic analyses are shown on the right. Arrows denote probands, filled symbols represent symptomatic members, open symbols indicate unaffected individuals, circles stand for females, squares correspond to males, diagonal lines refer to the deceased, WT/WT points to wild-type, and MT/WT specifies heterozygous mutation. Star symbols above electropherograms designate the locations of mutations. In pedigree A, the heterozygous KCND3 mutation c.950G>A (p.C317Y) was detected in patient A.III-1, while his healthy family members (A.II-1, A.II-2, and A.III-2) have two WT alleles. In pedigree B, the heterozygous KCND3 mutation c.1123C>T (p.P375S) was identified in the index case and his symptomatic mother (B.II-2 and B.I-2), but not in the healthy family member (B.II-1). (c) and (d) Sagittal and axial sections of T1-weighted MRI for probands A.III-1 and B.II-2. These neuroimages show generalized atrophy in the cerebrum and cerebellum (including the cerebellar hemispheres and vermis). The brainstem, however, is relatively spared.

MG132 (10 μg/ml; Sigma-Aldrich) or the lysosomal inhibitor chloroquine (100 μg/ml; Sigma-Aldrich) for 24 hr.

### 2.8 Immunoblotting

Transfected cells were washed two times with ice-cold phosphate-buffered saline (PBS; [in mM]: 136 NaCl, 2.5 KCl, 1.5 KH₂PO₄, and 6.5 Na₂HPO₄, pH 7.4), centrifuged, and resuspended in lysis buffer ([in mM]: 150 NaCl, 5 EDTA, 50 Tris-HCl, 1 PMSF, 1% Triton X-100, pH 7.6, supplemented protease inhibitor mixture). Laemmli sample buffer was added to lysates, and samples were sonicated on ice (three times for 5 s each) and heated at 70°C for 5 min. Samples were then separated by 7.5–10% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE), electrophoretically transferred to nitrocellulose membranes, and detected using mouse anti-Myc (1:5,000; clone 9E10), rabbit anti-Flag (1:5,000; Sigma-Aldrich), rabbit anti-GFP (1:2,500, Abcam), rat anti-HA (1:5,000; Roche), rabbit anti-GAPDH (1:8,000; GeneTex), or rabbit anti-α-tubulin (1:5,000; GeneTex). Blots were exposed to horseradish-peroxidase-conjugated goat antimouse IgG (1:5,000; Jackson ImmunoResearch),

goat antirabbit IgG (1:5,000; Jackson ImmunoResearch), or goat anti-rat IgG (1:5,000; Santa Cruz Biotechnology), and revealed by an enhanced chemiluminescence detection system (Thermo Fisher Scientific). Data shown are representative of at least three independent experiments. Densitometric scans of immunoblots were quantified with ImageJ (National Institute of Health).

### 2.9 Cycloheximide chase assays

Twenty-four hours after transfection, cells were subject to treatment with the protein synthesis inhibitor cycloheximide (100 μg/ml; Sigma-Aldrich) for 0–6 hr. For quantitative analyses of protein degradation time course, Kv4.3 protein densities in response to different cycloheximide treatment durations were standardized as the ratio of Kv4.3 signal to the cognate tubulin signal, followed by normalization to the corresponding control at 0 hr. Logarithmic scale of normalized attenuation of Kv4.3 protein density was plotted against linear scale of cycloheximide treatment duration. Protein half-life was then derived from single linear-regression fit of the semilogarithmic plot. Kv4.3 protein half-life values were determined from individual experimental trials, followed by data pooling for statistical analyses.

TABLE 1 Clinical features and bioinformatics analyses of the patients with novel KCND3 mutations

| Cases | A.III-1 | B.II-2 | B.I-2 |
|-------|---------|--------|-------|
| Mutations | c.950G>A(p.C317Y) | c.1123C>T(p.P375S) | c.1123C>T(p.P375S) |
| Clinical characteristics |  |  |  |
| Gender | Male | Male | Female |
| Age at exam (years) | 24 | 39 | 69 |
| Initial presentation | Developmental delay | Cognitive impairment | Gait disturbance |
| Ataxia | Ataxia since age 15 | Ataxia since age 10–15 | Ataxia since age 36, bedridden at age 69, passed away at age 74 |
| Current SARA:12/40 |  | Current SARA:17/40 |  |
| Movement disorders | Myoclonus, dystonia | Dystonia, bradykinesia | Not noted |
| Cognitive impairment | Yes, MMSE:12/30 | Yes, MMSE:14/30 | N.A. |
| Neuroimages | Atrophy of the cerebra, cerebellar hemispheres, and vermis | Atrophy of the cerebra, cerebellar hemispheres, and vermis | N.A. |
| Sequencing database |  |  |  |
| In-house database | 0/1,000 | 0/1,000 | 0/1,000 |
| Taiwan Biobank | 0/997 | 0/997 | 0/997 |
| gnomAD | 0/138,622 | 0/138,622 | 0/138,622 |
| Bioinformatics tools |  |  |  |
| SIFT<sup>a</sup> | Score: 0.05 | Score: 0 | Score: 0 |
|  | (Damaging) | (Damaging) | (Damaging) |
| SNPs&GO<sup>b</sup> | Reliability index: 9 | Reliability index: 9 | Reliability index: 9 |
|  | (Disease-related) | (Disease-related) | (Disease-related) |
| PolyPhen-2<sup>c</sup> | Score: 1.000 | Score: 1.000 | Score: 1.000 |
|  | (Probably damaging) | (Probably damaging) | (Probably damaging) |
| MutationTaster<sup>d</sup> | Prob: 0.999999999963983 | Prob: 0.999999999974573 | Prob: 0.999999999974573 |
|  | (Disease causing) | (Disease causing) | (Disease causing) |

Abbreviations: MMSE, Mini-Mental State Examination; N.A., not available; SARA, Scale for the Assessment and Rating of Ataxia.

<sup>a</sup>SIFT (https://sift.bii.a-star.edu.sg/) is a tool for predicting the effect of an amino acid substitution on protein function based on sequence homology. The results of prediction could be defaulted as damaging if the score is ≤0.05 or as tolerated if the score is >0.05. Both KCND3 variants were predicted to be damaging (SIFT score ≤ 0.05).

<sup>b</sup>SNPs&GO (https://snps-and-go.biocomp.unibo.it/snps-and-go/) is a website server for the prediction of the effect of human disease-related single point protein mutations. The reliability index, ranging from 0 (unreliable) to 10 (reliable), indicates the confidence in the prediction. Both KCND3 variants were predicted to be disease-related with a high predicting confidence.

<sup>c</sup>PolyPhen-2 (polymorphism phenotyping v2; http://genetics.bwh.harvard.edu/pph2/) is a tool for predicting the effect of an amino acid substitution on protein structure and function. Scores range between 0 and 1. The higher the score is, and the higher the possibility for a damaging effect of a sequence variant was predicted. The polyphen-2 scores of both KCND3 variants were calculated as 1.000, and thus were predicted to be probably damaging.

<sup>d</sup>MutationTaster (http://www.mutationtaster.org/) is a tool to evaluate the disease-causing potential of a sequence variant. The probability score (prob) indicates the probability of the prediction, with 1 indicating the highest confidence of the prediction. Both KCND3 variants were predicted to be disease causing with very high probability scores close to 1.

### 2.10 | Biotinylation of cell surface proteins

Transfected cells were gently washed with D-PBS (Sigma-Aldrich) supplemented with 0.5 mM CaCl₂, 2 mM MgCl₂, followed by incubation in 1 mg/ml sulfo-NHS-LC-biotin (Thermo Fisher Scientific) in D-PBS at 4°C for 1 hour on a smooth rocker. To terminate biotinylation, biotin reagents were removed, and cells were rinsed three times with 100 mM glycine in PBS, followed by once with TBS buffer ([in mM]: 20 Tris-HCl and 150 NaCl, pH 7.4). Cells were solubilized in the lysis buffer. After centrifugation, insolubilized materials were removed, and solubilized cell lysates were incubated overnight at 4°C with streptavidin-agarose beads (Thermo Fisher Scientific). Beads were washed two times in the lysis buffer, once in a high-salt buffer ([in mM]: 500 NaCl, 5 EDTA, 50 Tris-HCl, 0.1% Triton X-100, pH 7.6), followed by once in a low-salt buffer ([in mM]: 2 EDTA, 10 Tris-HCl, 0.1% Triton X-100, pH 7.6). To elute biotin-streptavidin complexes, beads were heated at 70°C for 5 min in Laemmli sample buffer.

### 2.11 | Coimmunoprecipitation

Transfected cells were solubilized in ice-cold lysis buffer, and insolubilized materials were removed by centrifugation. Solubilized lysates were pre-cleared with protein G sepharose beads (GE Healthcare Biosciences) for 1 hour at 4°C, and incubated with anti-Flag antibody for 4 hours at 4°C, followed by incubation with protein G sepharose beads for 16 hours at 4°C. Beads were gently spun down, washed two times in buffer A ([in mM] 100 NaCl, 4 KCl, 2.5 EDTA, 20 NaHCO₃, 20 Tris-HCl, pH 7.5) supplemented with 0.1% Triton X-100, and then washed two times in buffer A. The immune complexes were eluted from the beads by heating at 70°C for 5 min in the Laemmli sample buffer.

### 2.12 | Immunofluorescence

Forty-eight hours after transfection, cells were seeded on coverslips, rinsed in PBS, and then fixed with methanol (at 4°C) or 4%

paraformaldehyde (at room temperature) in PBS for 20 min. Cells were permeabilized and blocked with a blocking buffer (5% normal goat serum in 20mM phosphate buffer, 0.1% Triton X-100, 0.45 M NaCl, pH 7.4) for 60 min at 4°C. Primary antibodies were appropriately diluted and then applied in the blocking buffer overnight at 4°C. Immunoreactivities were visualized with goat-antimouse antibodies conjugated to Alexa Fluor 568 (1:200; Invitrogen) and goat-anti-rat antibodies conjugated to Alexa Fluor 633 (1:200; Invitrogen). A Leica TCS SP5 laser-scanning confocal microscope was utilized to acquire fluorescence images.

### 2.13 | Statistical analyses

All statistical analyses were performed with Origin 7.0 (Microcal Software). Numerical values were presented as mean ± SEM. The significance of the difference between two means was tested using Student's *t* test. Where necessary, means from multiple groups were additionally compared using one-way analysis of variance (ANOVA). The data that support the findings of this study are available from the corresponding authors, upon reasonable request.

---

### 3 | RESULTS

#### 3.1 | Clinical investigation of patients with novel disease-associated KCND3 mutations

In 93 patients with molecularly unassigned, inherited cerebellar ataxia, we searched for potential SCA-associated genetic variants utilizing a disease-specific targeted sequencing panel as previously reported (Hsiao et al., 2017). From two distinct pedigrees, we identified two novel disease-associated variants, c.950G>A (p.C317Y) and c.1123C>T (p.P375S), located in exon 2 and exon 3 of KCND3, respectively.

---

(a)

| S4 | C317 | V338 | P375 | T377 |
| --- | --- | --- | --- | --- |
| Human Kv4.3 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGSSASKFTSIPASFWYTIVTMTTLGYGDMVPKTIAKGIFGS |
| Human Kv4.2 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGSSASKFTSIPAAFWYTIVTMTTLGYGDMVPKTIAKGIFGS |
| Human Kv4.1 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGTNKTNFTSIPAAFWYTIVTMTTLGYGDMVPSSTIAKGIFGS |
| Bovine Kv4.3 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGSSASKFTSIPASFWYTIVTMTTLGYGDMVPKTIAKGIFGS |
| Giant panda Kv4.3 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGSSASKFTSIPASFWYTIVTMTTLGYGDMVPKTIAKGIFGS |
| Mouse Kv4.3 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGSSASKFTSIPASFWYTIVTMTTLGYGDMVPKTIAKGIFGS |
| Rat Kv4.3 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGSSASKFTSIPASFWYTIVTMTTLGYGDMVPKTIAKGIFGS |
| Chicken Kv4.3 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGSSASKFTSIPASFWYTIVTMTTLGYGDMVPKTIAKGIFGS |
| Zebrafish Kv4.3 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGSSSSKFTSIPASFWYTIVTMTTLGYGDMVPKTIAKGIFGS |
| African clawed frog Kv4.3 | RIFKFSRHSQLRILGYTLKSCASELGFLLLFSLTMAIIIFATVMFYAEKGSTSSKFTSIPASFWYTIVTMTTLGYGDMVPKTIAKGIFEFS |

(b)
![](image_b.png)

(c)
![](image_c.png)

(d)
![](image_d.png)

(e)
![](image_e.png)

---

**FIGURE 2** Molecular characterization of SCA19/22-associated Kv4.3 mutations. (a) Protein sequence alignment of the S4–S6 region of human Kv4 isoforms (4.1, 4.2, and 4.3) and Kv4.3 orthologs from various animal species. Red-colored residues denote the locations of the novel variants (p.C317Y and p.P375S) characterized in this study, and green residues represent two previously reported variants (p.V338E and p.T377M). (b–e) Homology models of human Kv4.3 channel based on the crystal structure of a chimeric voltage-dependent K⁺ channel (PDB ID: 2R9R). A Kv4.3 subunit containing six transmembrane segments (S1–6) is demonstrated on the left. The transmembrane segments are colored red (S1), orange (S2), yellow (S3), green (S4), and purple (S6). Asterisks denote the locations of disease-associated mutations. Dotted squares represent the focused regions that are enlarged and shown on the right to compare the side chain structure between WT and mutant residues. Carbon, sulfate, oxygen, and nitrogen atoms are colored gray, yellow, red, and blue, respectively.

Genotype-phenotype cosegregation was confirmed by using Sanger sequencing. In pedigree A, only the proband (A.III-1) harbored the c.950G>A variant (Figure 1a). As suggested by analyses of the parentage relationship between this index case and his parents using 14 microsatellite markers, c.950G>A appears to be a de novo variant. The c.1123C>T variant in pedigree B, on the other hand, was found in both the proband (B.II-2) and his affected mother (B.I-2), but not in his unaffected sister (B.II-1; Figure 1b).

Potential pathogenicity of these novel KCND3 variants was confirmed by various bioinformatics tools. In silico analyses with either SIFT (Kumar et al., 2009) or PolyPhen-2 programs (Adzhubei et al., 2010) predicted that both variants are damaging. SNPs&GO analysis (Capriotti et al., 2013) indicated that these variants are significantly disease-related, which was further supported by a high MutationTaster prediction value (>0.9999; Schwarz et al., 2014). Moreover, the two KCND3 variants were neither observed in 1,000 ethnically matched control chromosomes from our in-house database, nor present in Taiwan Biobank and gnomAD database (Table 1), indicating the rarity of these variants in the general population. Given that all three affected patients carried heterozygous mutations and that, in pedigree B, the c.1123C>T mutation was present in proband B.II-2 but not his unaffected sibling (Figure 1b), these novel KCND3 mutations are very likely associated with a dominant inheritance pattern.

Clinical features of the three patients harboring the novel KCND3 mutations are summarized in Table 1. Patient A.III-1 (c.950G>A; p.C317Y) is a 24-year-old male who exhibited delayed motor developmental milestones. He displayed myoclonus over the limbs at age 14, followed by slowly progressive gait unsteadiness, frequent falling accidents, and cognitive impairment at age 15. Neurological examination revealed scanty speech with mild dysarthria, saccadic eye pursuits, significant dysmetria in both upper and lower limbs, foot dystonia while walking, myoclonus predominantly involving upper limbs, and truncal titubation. The SARA score was 12/40 (Video S1 and Supp. Video Legend), and the mini-mental state examination (MMSE) score was 12/30. As shown in Figure 1c, neuroimages demonstrated marked atrophy of the cerebrum, cerebellar hemispheres, and vermis. In addition, MRS analysis demonstrated reduction of the N-acetyl aspartate/creatine (NAA/Cr) ratio in bilateral cerebellar hemispheres and cerebellar vermis, which suggested the presence of borderline metabolic abnormalities in the cerebellum.

Patient B.II-2 (c.1123C>T; p.P375S), a 39-year-old male, exhibited slowly progressive cerebellar ataxia since adolescence. Stiffness of hands and feet and frequent eye-blinking became obvious at age 33, and a walking cane was required at age 38. Despite the presence of mild cognitive impairment, he managed to remain totally independent. Neurological examination revealed mild intellectual disability, mild dysarthria, slow eye movements, bilateral blepharospasm, dysmetria, and dystonia in both hands and feet. His SARA and MMSE scores were 17/40 and 14/30, respectively (Supp. Video S1 and Supp. Video Legend). Consistent with the aforementioned neurological deficits, his brain MRI displayed significant cerebellar atrophy, albeit less prominent than that seen in patient A.III-1 (Figure 1d). MRS further revealed an impaired biochemical profile in the cerebellar vermis.

Patient B.I-2, the mother of patient B.II-2, experienced slowly progressive gait difficulty at age 36, and later became bedridden for a long period of time. She passed away at age 74.

### 3.2 | Molecular localization of disease-associated Kv4.3 mutations

Similar to other Kv channels, the Kv4.3 subunit comprises six transmembrane segments (S1–S6) and an ion-conducting pore loop in the S5-S6 linker region. Figure 2a depicts protein sequence alignment of the S4-S6 region of Kv4 channels from various species. The novel p.C317Y and p.P375S mutations are located in the S4-S5 and S5-S6 linker regions, respectively. In addition, we highlighted the locations of two previously reported but uncharacterized SCA19/22-associated Kv4.3 mutations, c.1013T>A (p.V338E) and c.1130C>T (p.T377M; Lee et al., 2012; Table 2). As clearly indicated in the sequence alignment analysis, the residues of interest, p.C317, p.V338, p.P375, and p.T377, are highly conserved in the three human Kv4 isoforms, as well as in the listed Kv4.3 orthologs from a wide variety of animal species, implying potential evolutionary importance of these residues in Kv4 channels.

Protein homology models of human Kv4.3 in Figure 2b-e illustrate the impact of the mutations on local protein micro-environments. p.C317 and p.V338 are located in the amino-and carboxyl-terminal regions, respectively, of the S5 transmembrane segment. Substitution of cysteine by the phenol-containing tyrosine at residue 317 results in dramatic increase in the mean volume by about 92.5% (from 106 to 204 Å³; Chothia, 1984; Figure 2b). In addition to about 9.2% increase in the mean volume (from 142 to 155 Å³), mutation at residue 338 replaces the nonpolar

**TABLE 2 Clinical overview of the SCA19/22-associated Kv4.3 mutations p.V338E and p.T377M**

| Sequence variant | c.1013T>A | c.1130C>T |
|-------------------|------------|------------|
| Amino acid change | p.V338E    | p.T377M    |
| Variant effect     | Missense   | Missense   |
| Inheritance       | Autosomal dominant | Autosomal dominant |
| Onset of ataxia   | Adult-onset | Adolescence or adult-onset |
| Cognitive impairment | Suspected | Yes<sup>a</sup> |
| Involuntary movement | Grimacing face | Not reported |
| Recurrent mutation | No         | Yes        |
| Ethnicity         | Japanese   | Japanese; Swedish |
| Reference         | Lee et al. (2012) | Lee et al. (2012); Paucar et al. (2018) |

Note: The information is a simplified summary of previously reported literature.

<sup>a</sup>Whereas the first reported Japanese pedigree harboring the p.T377M mutation displayed no cognitive abnormality (Lee et al., 2012), a recent study in the Swedish p.T377M pedigree demonstrated significant cognitive impairments (Paucar et al., 2018).

HSIAO ET AL.

(a)

+60 mV  
-60 mV  

WT  
C317Y  
V338E  
P375S  
T377M  

1 uA  
50 ms  

(b)  

+KChIP2  

WT  
C317Y  
V338E  
P375S  
T377M  

5 uA  
100 ms  

(c)  

+KChIP2  

WT  
WT:C317Y = 1:1  
WT:V338E = 1:1  
WT:P375S = 1:1  

5 uA  
100 ms  

(d)  

+KChIP2  

WT  
WT:T377M = 1:1  
WT:T377M = 1:3  
WT:T377M = 1:5  

5 uA  
100 ms  

FIGURE 3 Loss-of-function phenotypes of disease-associated Kᵥ4.3 mutants. Functional expression of Myc-Kᵥ4.3 channel subunits in Xenopus oocytes. (a), (b) Expression of individual Kᵥ4.3 constructs in the absence (a) or the presence (b) of the auxiliary subunit KChIP2. (Left panels) Representative current traces in response to a voltage protocol comprising test potentials ranging from –60 mV to +60 mV in 10-mV steps. (Center panels) Peak K⁺ current amplitudes were plotted against corresponding test pulse potentials. Coexpression with KChIP2 led to at least four-fold increase in the peak current amplitude at +60 mV. (Right panels) Normalized peak current amplitudes at +60 mV. Without KChIP2 (a): p.C317Y, 0.01 ± 0.004; p.V338E, 0.02 ± 0.004; p.P375S, 0.01 ± 0.003; and p.T377M, 0.07 ± 0.01. With KChIP2 (b): p.C317Y, 0.05 ± 0.006; p.V338E, 0.01 ± 0.006; p.P375S, 0.01 ± 0.002; and p.T377M, 0.3 ± 0.04. Data from the same batch of oocytes on a given day were normalized with respect to the mean amplitude of the WT, followed by pooling multiple normalized data from different days as well as different batches of oocytes. (c), (d) Dominant-negative effects of the disease-associated mutants on WT channels. In the presence of KChIP2, the mutants were coexpressed with the WT in either 1:1 (c) or increasing (d) molar ratios. (Left panels) Representative current traces. (Right panels) Normalized Kᵥ4.3 peak current amplitudes at +60 mV in the presence of mutants: (+)p.C317Y, 0.7 ± 0.03; (+)p.V338E, 0.6 ± 0.04; (+)p.P375S, 0.6 ± 0.04; (+)p.T377M, (1:1) 0.9 ± 0.06, (1:3) 0.8 ± 0.08, and (1:5) 0.7 ± 0.04. Data were normalized with respect to the mean amplitude of the WT current in the absence of mutant subunits. Numbers of observations are labeled in parentheses. Asterisks denote significant difference from the WT control (*p < .05)

(a)

+40 mV  
-80 mV  

2.5 µA  
100 ms  

+60 mV  
-90 mV  
-120 mV  

2.5 µA  
250 ms  

(b)  

G/Gmax  

Voltage (mV)  

(c)  

G/Gmax  

WT  
WT+C317Y  
WT+V338E  
WT+P375S  
WT+T377M  

Voltage (mV)  

I/Imax  

Voltage (mV)  

I/Imax  

WT  
T377M  

Voltage (mV)  

(d)  

I/Imax  

WT  
WT+C317Y  

Voltage (mV)  

G/Gmax  

WT  
WT+C317Y  

↓ 83%  

Voltage (mV)  

I/Imax  

WT  
WT+V338E  

Voltage (mV)  

G/Gmax  

WT  
WT+V338E  

↓ 22%  

Voltage (mV)  

I/Imax  

WT  
WT+P375S  

Voltage (mV)  

G/Gmax  

WT  
WT+P375S  

↓ 72%  

Voltage (mV)  

I/Imax  

WT  
WT+T377M  

Voltage (mV)  

G/Gmax  

WT  
WT+T377M  

↓ 57%  

Voltage (mV)  

FIGURE 4 Continued.

HSIAO ET AL.

TABLE 3 Voltage-dependent gating properties of Kv4.3 WT and mutant channels

| (+KChIP2) | Activation | Inactivation |
| --- | --- | --- |
|  | $V_{0.5a}$ (mV) | $k_a$ | $n$ | $V_{0.5i}$ (mV) | $k_i$ | $n$ |
| Homotetramer |  |  |  |  |  |  |
| WT | -28.9 ± 0.5 | 5.6 ± 0.4 | 17 | -49.8 ± 0.3 | 5.6 ± 0.2 | 15 |
| p.T377M | -25.4 ± 0.7* | 5.8 ± 0.6 | 18 | -49.7 ± 0.4 | 5.9 ± 0.4* | 12 |
| Heterotetramer |  |  |  |  |  |  |
| WT+p.C317Y | -16.4 ± 1.2* | 7.5 ± 1.1* | 16 | -60.5 ± 0.5* | 8.0 ± 0.4* | 13 |
| WT+p.V338E | -25.5 ± 0.7* | 6.4 ± 0.6 | 18 | -50.6 ± 0.5 | 5.9 ± 0.4* | 15 |
| WT+p.P375S | -23.9 ± 0.5* | 5.4 ± 0.4 | 12 | -52.7 ± 0.5* | 6.1 ± 0.5* | 13 |
| WT+p.T377M | -19.9 ± 0.9* | 7.5 ± 0.8* | 16 | -51.7 ± 0.5 | 6.1 ± 0.4* | 15 |

Note: See the Methods section and Figure 4 for more details on fitting Kv4.3 current-voltage relationship with Boltzmann equation. $V_{0.5a}$: half-activation voltage. $k_a$: activation slope factor. $V_{0.5i}$: half-inactivation voltage. $k_i$: inactivation slope factor. $n$: number of observations. Asterisks denote significant difference from the WT homotetramer (*p < .05).

valine with the charged, acidic glutamate in the cytoplasmic portion of the S5 segment (Figure 2c). On the other hand, both p.P375 and p.T377 are located in the carboxyl-terminal region of the pore loop, in the vicinity of the signature sequence GYG D that contributes to the formation of the ion selectivity filter in Kv channels (Chapman, Krovetz, & VanDongen, 2001). At residue 375, substitution of the pyrrolidine-containing, nonpolar proline with the polar serine leads to about 23.2% reduction in the mean volume (from 129 to 99 Å³; Figure 2d). At residue 377, in contrast, replacement of the polar threonine with the nonpolar methionine causes a substantial 40.2% enlargement in the mean volume (from 122 to 171 Å³; Figure 2e).

course (Bahring et al., 2001; Takimoto et al., 2002), the current level of WT and p.T377M was substantially enhanced, even though the peak amplitude of p.T377M remained notably smaller (~34% of the WT; Figure 3b). Interestingly, coexpression with KChIP2 resulted in barely detectable p.C317Y currents (~5% of the WT), but no significant K⁺ currents for both p.V338E and p.P375S. The preceding data clearly indicate that the four SCA-related Kv4.3 mutants are associated with loss-of-function phenotypes.

To directly address the physiological basis underlying the dominant inheritance of these mutations, next we coexpressed WT and mutant subunits (Kv4.3 heterotetramer) in the presence of KChIP2. Figure 3c demonstrates that equimolar coexpression with the nonfunctional p.V338E and p.P375S led to about 45% and 38% reduction, respectively, of WT current amplitude. Similarly, p.C317Y suppressed the functional expression of WT by about 32%. Furthermore, upon coexpression with WT, the functional p.T377M instigated considerable diminution of the current level of the Kv4.3 heterotetramer in a concentration-dependent manner, up to about 31% decrease in the molar ratio of 1:5 (Figure 3d).

In addition, we assessed the effect of the mutations on the voltage-dependent gating process of human Kv4.3 channel, which can be readily analyzed by studying steady-state voltage-dependent activation and inactivation properties (Figure 4a). Figure 4b and Table 3 highlight that, compared with WT, p.T377M homotetramer was associated with depolarizing shift (~3.5 mV) of the half-activation voltage ($V_{0.5a}$), as well as subtle but statistically significant increase in the inactivation slope factor ($k_i$). As a result of the mutant's low-current level, analysis of p.C317Y gating property was not available. Importantly, upon coexpression with their WT counterpart, the four mutants notably altered the

FIGURE 4 Alteration of Kv4.3 voltage-dependent gating by disease-associated mutants. (a) Representative WT Kv4.3 current traces in response to the test pulse protocols while analyzing voltage-dependent activation (top) and inactivation (bottom) properties. Kv4.3 and KChIP2 subunits were coexpressed in equimolar ratio in Xenopus oocytes. (b) Steady-state activation (top) and inactivation (bottom) curves of Kv4.3 WT (black) and p.T377M (blue) homotetramers. (c) Steady-state activation (top) and inactivation (bottom) curves of WT/mutant heterotetramers. WT and the indicated mutant subunits were coexpressed in the 1:1 molar ratio. Lines represent Boltzmann equation fits to the data points. See the Methods section and Table 3 for more details on the analyses of voltage gating properties. (d) Window current analyses. (Left panels) Steady-state activation and inactivation curves of WT homotetramer and WT/mutant heterotetramers (WT/p.C317Y, WT/p.V338E, WT/p.P375S, and WT/p.T377M heterotetramer, respectively) were assembled from those shown in (c). (Right panels) The window current of each indicated construct is defined by the triangular area underneath the overlap of activation and inactivation curves. See Table 4 for more details on the parameters of window current analyses.

TABLE 4 Window current parameters of Kv4.3 channels

| (+KChIP2) | Peak voltage (mV) | Normalized area |
|-----------|------------------|-----------------|
| WT        | -39.5            | 1.00            |
| WT+p.C317Y | -37.8           | 0.17            |
| WT+p.V338E | -39.2           | 0.78            |
| WT+p.P375S | -38.1           | 0.28            |
| WT+p.T377M | -37.6           | 0.43            |

Note: As illustrated in Figure 4d, the window current is defined by the triangular area underneath the overlap of steady-state activation and inactivation curves. The peak voltage refers to the membrane potential at which the apex of window current triangle is located. The area of window current triangle was determined by integration. For each WT/mutant heterotetramer, the area of window current was normalized with respect to that of the WT homotetramer.

steady-state voltage-dependent activation and inactivation of the heterotetrameric Kv4.3 channels (Figure 4c, Table 3). For example, p.C317Y instigated about 12.5-mV and 10.7-mV shift of the half-activation and half-inactivation voltages (V0.5a and V0.5i), respectively. The activation and inactivation slope factors (ka and ki) were also substantially enhanced by about 34% and 43%, respectively, suggesting the presence of reduced voltage sensitivity.

The superimposed region of voltage-dependent activation and inactivation curves is known as the window current, which represents the range of membrane potential in which a significant fraction of inactivating ion channels remains persistently open. Given that Kv4.3 belongs to the fast and extensively inactivating A-type K+ current, the size of Kv4.3 window current is determined by the relative position of steady-state activation and inactivation curves. In other words, a small change in the voltage dependence parameters (V0.5a, V0.5i, ka, and ki) of Kv4.3 may manifest a sizable alteration in the channel's window current (Anderson et al., 2010). As depicted in Figure 4d and Table 3, the Kv4.3 WT/p.C317Y heterotetramer was associated with depolarizing (rightward) shift of the activation curve (V0.5a), as well as hyperpolarizing (leftward) shift of the inactivation curve (V0.5i). Furthermore, p.C317Y induced significant increase in the slope factors (ka, ki) that further diminished the extent of the overlap between activation and inactivation curves. Altogether, coexpression with p.C317Y resulted in about 83% reduction in the area of Kv4.3 window current (Table 4). Similarly, coexpression with p.V338E, p.P375S, or p.T377M led to notable decreases of Kv4.3 window currents, ranging from 22% to 72% (Figure 4d and Table 4). These data are consistent with the idea that the four SCA19/22-causing Kv4.3 mutants may exert considerable dominant-negative effects on both voltage-dependent gating and effective K+ current level of human Kv4.3 channel.

### 3.4 Dominant-negative effects of disease-associated mutations on human Kv4.3 protein biosynthesis

The reduced current levels of the Kv4.3 mutants shown in Figure 3a,b can be attributed to mutation-induced alteration of channel gating/permeation function. Alternatively, the observation may also suggest the presence of disrupted protein biosynthesis that leads to dramatically decreased Kv4.3 protein levels at the cell surface. To address the latter hypothesis, we went on to study protein expression of Myc-Kv4.3 subunit perse in HEK293T cells. Figure 5a demonstrates that, compared with the WT subunit, total protein levels of the Kv4.3 mutants were significantly lower by about 21–46%. By applying the cycloheximide chase experiment, which determines the time course of protein turnover, we further showed that the mutant proteins were associated with about 28–39% reduction in protein half-life (Figure 5b,c), indicating the presence of defective protein stability. The protein expression of Kv4.3 WT and mutants was effectively increased in the presence of the proteasome inhibitor MG132 (Figure 5d) or the lysosome inhibitor chloroquine (Figure 5e), suggesting that both the proteasomal and the lysosomal degradation pathways may contribute to defective protein stability of the disease-associated Kv4.3 mutants.

As mentioned above, the auxiliary subunit KChIP2 plays a critical role in promoting Kv4.3 protein expression and cell surface localization (An et al., 2000; Jerng & Pfaffinger, 2014; Takimoto et al., 2002). In the presence of KChIP2, however, the total protein level of the Kv4.3 mutants remained significantly lower than that of their WT counterpart (Figure 6a). Likewise, upon coexpression with the auxiliary subunit, the protein half-life of the WT (~10.2 hr) was still substantially longer than that of the mutants (ranging from ~4.6 to 6.3 hr; Figure 6b,c). In addition to total protein level, we also examined cell surface Kv4.3 protein level by applying surface biotinylation experiment. Figure 6d illustrates that the disease-related mutants are associated with about 43–84% reduction in surface protein level.

Paradoxically, whereas p.C317Y and p.V338E manifested the most prominent decrease (>50%) in total protein level (Figure 6a), p.P375S and p.T377M displayed the most dramatic decline (>70%) in surface protein level (Figure 6d). This apparent discrepancy implies that p.P375S and p.T377M may additionally be associated with a disruption in their trafficking from the endoplasmic reticulum (ER) to the plasma membrane. Confocal microscopic image analyses revealed that, in HEK293T cells, KChIP2 was predominantly anchored at the plasma membrane (Jerng & Pfaffinger, 2014; Takimoto et al., 2002), and that KChIP2 indeed efficiently facilitated membrane localization of WT Kv4.3 as well as mutant subunits (Figure 7). Nevertheless, even in the presence of KChIP2, a detectable fraction of Kv4.3 protein manifested cytoplasmic aggregates that colocalized with the ER-resident protein calnexin (Figure 7), suggesting the existence of ER-retention. To quantitatively assess the ER-to-membrane trafficking property of Kv4.3 protein, we focused on the biotinylation data to calculate the ratio of surface to total protein levels. The rightmost panel in Figure 6d demonstrates that, compared with their WT counterpart, p.P375S and p.T377M, but not p.C317Y and p.V338E, exhibited significantly defective membrane trafficking efficiency, thereby yielding the lowest surface protein level.

The next question we addressed was whether the dominant inheritance pattern of the KCND3 mutations may be ascribed to mutant-induced defective Kᵥ4.3 biosynthesis. To this end, we coexpressed Myc-Kᵥ4.3 mutants with Flag-Kᵥ4.3 WT subunit. The interaction between Flag-Kᵥ4.3 WT and Myc-Kᵥ4.3 mutants was confirmed by coimmunoprecipitation experiments (Figure 8a).

Figure 8b,c further illustrate that, regardless of the presence of KChIP2, coexpression with the disease-related mutants led to about 20–57% reduction in total Flag-WT protein expression. Coexpression with the mutants also resulted in about 35–68% decrease in Flag-WT protein half-life (Figure 8d,e) consistent with the presence of significant mutant-induced enhanced

FIGURE 5 Continued.

degradation of WT. Accordingly, upon coexpressing with the mutants, cell surface Flag-WT protein level was substantially diminished (Figure 8f). Moreover, p.P375S and p.T377M, but not p.C317Y and p.V338E, significantly impaired membrane trafficking efficiency of WT subunit (Figure 8f). Overall, our observations suggest that the four SCA-causing mutants may effectively perturb Kv4.3 WT protein stability and membrane trafficking in a dominant-negative manner.

The clinical relevance of KCND3 in cerebellar ataxia has been increasingly recognized by unmasking KCND3 sequence variants through genetic investigations. The two novel KCND3 mutations presented in this study, along with another previously reported SCA19/22-associated KCND3 mutation, c.679_681delTTC (p.F227del; Lee et al., 2012), were identified from a Taiwanese cohort of 585 unrelated families with inherited cerebellar ataxias, suggesting that KCND3 mutations may account for at least 0.5% (3/585) of dominantly inherited cerebellar ataxias in Taiwan. In comparison, a previous study involving multinational screening for dominantly inherited cerebellar ataxia-associated genes among 412 index cases (excluding polyglutamine expansion diseases) reported four patients with KCND3 variants (Coutelier et al., 2017), while another investigation of 319 undiagnosed ataxia cases identified one patient carrying a KCND3 variant (Coutelier et al., 2018).

Emerging evidence points to the existence of phenotypic heterogeneity in SCA19/22. For example, a recently identified ataxic patient with a KCND3 variant was characterized by episodic gait disorder (Coutelier et al., 2018), and another KCND3 variant-carrying patient was identified from a cohort of 39 cases with episodic ataxia (Choi et al., 2017). Furthermore, paroxysmal disorder, such as epilepsy, was also reported in SCA19/22 patients (Huin et al., 2017; Smets et al., 2015). In fact, the phenotypic spectrum of SCA19/22 has been widened to include a wide variety of different movement disorders, such as myoclonus, dystonia, and Parkinsonism (Duarrilet al., 2012; Huin et al., 2017; Kurihara et al., 2018). Interestingly, myoclonus and dystonia were also observed in our patients carrying the novel p.C317Y and p.P375S mutations (Table 1). These observations are consistent with the notion that defective human Kv4.3 channel function may disturb the physiology of the cerebellar circuit and thereby drive the development of movement disorders like dystonia in SCA19/22.

Another intriguing issue on phenotypic heterogeneity in SCA19/22 concerns the extent of intellectual disability (Huin et al., 2017;

4 | DISCUSSION

4.1 | Phenotypic heterogeneity in SCA19/22

Using comprehensive genetic screening, we identified two novel missense KCND3 variants, c.950G>A (p.C317Y) and c.1123C>T (p.P375S; Figure 1). We provided several lines of critical evidence highlighting the pathogenic significance of the novel KCND3 variants. First, genotype-phenotype cosegregation analyses confirmed their match for dominant inheritance pattern in both pedigrees. Second, the sequence changes were absent in 1,000 ethnically matched control chromosomes as well as in several genetic polymorphism databases. Third, in silico analyses unanimously supported their deleterious potentials (Table 1). Fourth, both variants result in alteration of evolutionarily conserved amino acid residues in the Kv4.3 channel (Figure 2). Last and perhaps most importantly, the mutant proteins encoded by these KCND3 variants manifested significant defects in protein stability, membrane trafficking, and channel gating (Figures 3–8 and Tables 3 and 4). Based on the guideline for the interpretation of sequence variants recommended by the American College of Medical Genetics and Genomics (ACMG) and the Association of Molecular Pathology (Richards et al., 2015), we conclude that these KCND3 variants are pathogenic mutations (Table 5).

FIGURE 5 Enhanced protein degradation of disease-associated Kv4.3 mutants. Biochemical characterization of Myc-Kv4.3 subunits in HEK293T cells. (a) (Left) Representative immunoblots showing total protein expression of various Kv4.3 constructs. Cell lysates were subject to immunoblotting analyses with the indicated antibodies (α-Myc and α-Tubulin). Molecular weight markers (in kDa) are labeled to the left. Tubulin expression was chosen as the loading control. (Right) Normalized total protein levels (n = 7–8): p.C317Y, 0.6 ± 0.1; p.V338E, 0.5 ± 0.1; p.P375S, 0.6 ± 0.1; and p.T377M, 0.8 ± 0.1. Total Kv4.3 protein density was standardized as the ratio to the cognate total tubulin signal, followed by normalization with respect to the WT control. Asterisks denote significant difference from the WT control (*p < 0.05). (b), (c) Quantification of Kv4.3 protein degradation kinetics for WT (black), p.C317Y (red), p.V338E (green), p.P375S (blue), and p.T377M (cyan) homotetramers. (b) Representative immunoblots showing Kv4.3 protein turn-over time course in response to the indicated treatment durations (hr) of 100 μg/ml cycloheximide (CHX). (c) (Left) For each Kv4.3 construct, protein densities were standardized with respect to the cognate tubulin signal, followed by normalization to the corresponding non-CHX treatment (0 hr) control. Data points represent the average of 6–12 independent experiments. (Center) The Kv4.3 degradation time courses at the left were replotted on a semi-logarithmic scale, followed by linear-regression fit to determine protein half-life. (Right) For each Kv4.3 construct, protein half-life values derived from individual experimental trials were pooled together for statistical analyses. Mean protein half-lives (hr): WT, 5.5 ± 0.5; p.C317Y, 4.0 ± 0.4; p.V338E, 3.4 ± 0.6; p.P375S, 3.6 ± 0.4; and p.T377M, 3.8 ± 0.4. Asterisks denote significant difference from the WT control (*p < .05). (d), (e) (Left) Representative immunoblots demonstrating the effect of treatment with 10 μg/ml MG132 (d) or 100 μg/ml chloroquine (CQ) (e) on Kv4.3 protein level. (Right) Quantitative analyses for each Kv4.3 construct (n = 8). (+)MG132: WT, 2.2 ± 0.4; p.C317Y, 2.0 ± 0.3; p.338E, 2.3 ± 0.4; p.P375S, 2.9 ± 0.2; p.T377M, 2.7 ± 0.2. (+)chloroquine: WT, 2.5 ± 0.3; p.C317Y, 2.5 ± 0.4; p.338E, 1.6 ± 0.3; p.P375S, 2.0 ± 0.4; and p.T377M, 1.9 ± 0.3. Asterisks denote significant difference from the cognate no-treatment control (*p < .05)

(a)
+ HA-KChIP2

(Myc-)
WT C317Y V338E P375S T377M

α-Myc
100

α-HA
35

α-Tubulin
55

Normalized signal
1.0
0.8
0.6
0.4
0.2
0.0
(Myc-) WT C317Y V338E P375S T377M
+ HA-KChIP2

(b)
Myc-WT + HA-KChIP2
CHX 0 2 4 6 hr
α-Myc
100
70
α-HA
35
α-Tubulin
55

Myc-C317Y + HA-KChIP2
CHX 0 2 4 6 hr
α-Myc
100
70
α-HA
35
α-Tubulin
55

Myc-V338E + HA-KChIP2
CHX 0 2 4 6 hr
α-Myc
100
70
α-HA
35
α-Tubulin
55

Myc-P375S + HA-KChIP2
CHX 0 2 4 6 hr
α-Myc
100
70
α-HA
35
α-Tubulin
55

Myc-T377M + HA-KChIP2
CHX 0 2 4 6 hr
α-Myc
100
70
α-HA
35
α-Tubulin
55

(c)

Normalized signal
1.0
0.8
0.6
0.4
0.2
0.0
CHX treatment (hr)

Normalized signal
1.0
0.8
0.6
0.4
0.2
0.0
CHX treatment (hr)

Protein half-life (hr)
12
10
8
6
4
2
0
(Myc-) WT C317Y V338E P375S T377M
+ HA-KChIP2

(d)
+ HA-KChIP2

(Myc-)
WT C317Y V338E P375S T377M

α-Myc
100

α-GAPDH
35

α-Myc
100

α-HA
35

α-GAPDH
35

Normalized signal
1.0
0.8
0.6
0.4
0.2
0.0
(Myc-) WT C317Y V338E P375S T377M
+ HA-KChIP2
Surface signal

Normalized ratio
1.5
1.0
0.5
0.0
(Myc-) WT C317Y V338E P375S T377M
+ HA-KChIP2
Surface/total signal

FIGURE 6 Continued.
Kurihara et al., 2018; Paucar et al., 2018; Smets et al., 2015). Earlier, a Japanese SCA19/22 patient with the p.T377M mutation was not reported to exhibit significant cognitive impairment (Lee et al., 2012). However, a recent study of Swedish p.T377M patients reported the presence of widespread brain hypometabolism and intellectual abnormalities (Paucar et al., 2018). In the current study, both patients A.III-1 (p.C317Y) and B.II-2 (p.P375S) displayed notable intellectual disability since their adolescence. Taken together, it is likely that mutations in Kv4.3 channels may manifest a heterogeneous group of neurological symptoms, including cerebellar ataxia, epilepsy, Parkinsonism, myoclonus, dystonia, and cognitive impairment.

### 4.2 | Dominant-negative mechanisms of SCA19/22

Besides p.C317Y and p.P375S, we also examined two previously reported but uncharacterized mutations, p.V338E and p.T377M. As illustrated in the Kv4.3 protein homology models in Figure 2, p.C317 and p.V338 are located in the S5 transmembrane segment that plays a critical role in the voltage-dependent gating of K⁺ channels (Shieh, Klemic, & Kirsch, 1997), whereas p.P375 and p.T377 are located in the carboxyl-terminal region of the pore loop that contributes to the formation of the K⁺ ion selectivity filter (Chapman et al., 2001). Using both biophysical and biochemical analyses, we provided direct evidence showing that the four SCA19/22-associated Kv4.3 mutant proteins manifest loss-of-function phenotypes (Figure 3). Moreover, all the Kv4.3 mutants are associated with enhanced protein degradation, with p.P375S and p.T377M additionally showing defective membrane trafficking (Figures 5–7). Interestingly, p.M373I, another SCA19/22-associated mutation in the pore loop, has also been shown to display defective membrane trafficking (Duarri et al., 2015), raising the possibility that, during protein biosynthesis, the carboxyl-terminal region of the pore loop may be essential for proper export of Kv4.3 channel from the ER to the plasma membrane. Future understanding of the molecular mechanisms governing protein quality control and membrane transport of Kv4.3 channel may thus provide important insight into the molecular pathophysiology of SCA19/22.

The key aim of this study is to elucidate the physiological basis underlying the dominant inheritance pattern of SCA19/22. By coexpressing with Kv4.3 WT, we demonstrated that the disease-causing mutants significantly suppress peak current amplitude of Kv4.3 channels (Figure 3). Notably, the mutants also profoundly alter the voltage-dependent gating of Kv4.3 WT. As indicated by window current analyses (Figure 4d and Table 4), these mutant-induced changes in steady-state voltage dependence property may further reduce effective Kv4.3 current level by about 22–83%. In agreement with our observations, another SCA19/22-associated mutant, p.T352P, has also been shown to repress WT current level in a dominant-negative manner (Duarri et al., 2015). Importantly, a considerable portion of the mutant-suppressed current amplitude can be ascribed to enhanced protein degradation and defective membrane trafficking of WT subunit (Figure 8), further highlighting the pathogenic significance of anomalous protein biosynthesis in cerebellar ataxia. Taken together results, these data are consistent with the notion that the dominant inheritance pattern of SCA19/22 can largely be explained by the dominant-negative effect of disease-related mutants on protein biosynthesis and voltage-dependent gating of Kv4.3 WT channel. Nevertheless, we cannot exclude the possibility that haploinsufficiency may also contribute to dominant inheritance pattern of other types of SCA19/22-associated KCND3 mutations.

Given the essential role of Kv4.3 as IA in the somatodendritic compartment (Hoffman et al., 1997; Nadal et al., 2001), dominant-negative effects instigated by SCA19/22-associated mutants may inflict sizeable disruption of postsynaptic signal integration in neurons. Emerging evidence supports the idea that alteration in voltage-dependent gating of postsynaptic Kv4 channels may profoundly modify somatodendritic membrane excitability, which in turn may fundamentally change the firing pattern of neuronal action potentials. In cerebellar granule cells, for example, synaptic inputs from mossy fibers may induce hyperpolarizing shifts in both half-activation and half-inactivation voltages of Kv4 channels, thereby invoking long-term increase in granule cell excitability (Rizwan, Zhan, Zamponi, & Turner, 2016). Furthermore, in cerebellar granule and stellate cells, T-type Ca²⁺ (Caᵥ3) channels form a signaling complex with Kv4 and the Ca²⁺-sensing KChIP, whereby Caᵥ3-mediated Ca²⁺ influx significantly enhances IA by causing depolarizing shift of Kv4 half-inactivation voltage (Anderson et al., 2010; Heath et al., 2014; Molineux, Fernandez, Mehaffey, & Turner, 2005). The physiological significance of this Caᵥ3-Kv4 signaling complex is further underlined

---

**FIGURE 6** KChIP2 coexpression fails to rescue defective biosynthesis of Kv4.3 mutants. Coexpression of Myc-Kv4.3 and HA-KChIP2 subunits in HEK293T cells. (a) Comparison of Kv4.3 protein expression. Normalized total protein levels (*n* = 8–10): p.C317Y, 0.5 ± 0.1; p.V338E, 0.5 ± 0.1; p.P375S, 0.6 ± 0.1; and p.T377M, 0.7 ± 0.1. (b), (c) Quantification of Kv4.3 protein degradation kinetics for WT (*black*), p.C317Y (*red*), p.V338E (*green*), p.P375S (*blue*), and p.T377M (*cyan*) homotetramers in the presence of KChIP2 (*n* = 10–15). Mean protein half-lives (hr): WT, 10.2 ± 1.0; p.C317Y, 6.3 ± 1.0; p.V338E, 4.6 ± 0.2; p.P375S, 5.1 ± 0.7; and p.T377M, 6.2 ± 0.6. (d) Surface biotinylation assay of Kv4.3. (*Left*) Representative immunoblots. Cell lysates from biotinylated intact cells were subject to either direct immunoblotting analyses (*Total*) or streptavidin pull-down before immunoblotting (*Surface*). (*Center*) Quantification of Kv4.3 surface expression. Normalized surface protein levels (*n* = 5–6): p.C317Y, 0.6 ± 0.07; p.V338E, 0.5 ± 0.05; p.P375S, 0.3 ± 0.05; and p.T377M, 0.2 ± 0.04. Surface Kv4.3 protein density was standardized as the ratio to the cognate total GAPDH signal, followed by normalization with respect to the WT control. (*Right*) Comparison of membrane trafficking efficiency (*Surface/Total*), which was calculated as surface protein density divided by the corresponding standardized total protein density. The ratio was normalized with respect to the WT control. Normalized membrane trafficking efficiencies: p.C317Y, 1.1 ± 0.2; p.V338E, 1.2 ± 0.1; p.P375S, 0.4 ± 0.1; and p.T377M, 0.2 ± 0.1. Asterisks denote significant difference from the WT control (*p* < .05)
FIGURE 7 Subcellular localization of Kᵥ4.3 proteins. Representative immunofluorescence images of HEK293T cells over-expressing GFP-Kᵥ4.3 in the absence (HA vector; upper panels) or the presence (HA-KChIP2; lower panels) of KChIP2. Fluorescence signals of GFP (green), the ER-resident protein calnexin (red), KChIP2 (magenta), and the nuclear counterstain DAPI (blue) were acquired using confocal microscopy. Arrows and arrowheads represent ER- and plasma membrane localization, respectively. Scale bar: 10 μm

(a)
| GFP-WT | Calnexin | HA-Vector | Merge |
| --- | --- | --- | --- |
|  |  |  |  |
| GFP-WT | Calnexin | HA-KChIP2 | Merge |

(b)
| GFP-C317Y | Calnexin | HA-Vector | Merge |
| --- | --- | --- | --- |
|  |  |  |  |
| GFP-C317Y | Calnexin | HA-KChIP2 | Merge |

(c)
| GFP-V338E | Calnexin | HA-Vector | Merge |
| --- | --- | --- | --- |
|  |  |  |  |
| GFP-V338E | Calnexin | HA-KChIP2 | Merge |

(d)
| GFP-P375S | Calnexin | HA-Vector | Merge |
| --- | --- | --- | --- |
|  |  |  |  |
| GFP-P375S | Calnexin | HA-KChIP2 | Merge |

(e)
| GFP-T377M | Calnexin | HA-Vector | Merge |
| --- | --- | --- | --- |
|  |  |  |  |
| GFP-T377M | Calnexin | HA-KChIP2 | Merge |

(a)
Myc-WT + HA-KChIP2
Input IP : α-Flag

(Flag-) Vector WT
100
70
α-Myc
100
70
α-Flag
35
α-GAPDH

Flag-WT + HA-KChIP2
Input IP : α-Flag

(Myc-) WT C317Y V338E P375S T377M
100
70
α-Myc
100
70
α-Flag
35
α-GAPDH

(b)
Flag-WT

(Myc-) WT C317Y V338E P375S T377M
100
70
α-Flag
100
70
α-Myc
55
α-Tubulin

(c)
Flag-WT
+ HA-KChIP2

(Myc-) WT C317Y V338E P375S T377M
100
70
α-Flag
100
70
α-Myc
35
α-HA
55
α-Tubulin

(d)
Flag-WT
+ Myc-WT, HA-KChIP2
CHX 0 2 4 6 hr
100
70
α-Flag
100
70
α-Myc
35
α-HA
55
α-Tubulin

Flag-WT
+ Myc-C317Y, HA-KChIP2
CHX 0 2 4 6 hr
100
70
α-Flag
100
70
α-Myc
35
α-HA
55
α-Tubulin

Flag-WT
+ Myc-V338E, HA-KChIP2
CHX 0 2 4 6 hr
100
70
α-Flag
100
70
α-Myc
35
α-HA
55
α-Tubulin

Flag-WT
+ Myc-P375S, HA-KChIP2
CHX 0 2 4 6 hr
100
70
α-Flag
100
70
α-Myc
35
α-HA
55
α-Tubulin

Flag-WT
+ Myc-T377M, HA-KChIP2
CHX 0 2 4 6 hr
100
70
α-Flag
100
70
α-Myc
35
α-HA
55
α-Tubulin

(e)

Norm. Flag-WT signal
1.0-
0.8-
0.6-
0.4-
0.2-
0.0-
CHX treatment (hr)

Norm. Flag-WT signal
1.0-
0.8-
0.6-
0.4-
0.2-
0.0-
CHX treatment (hr)

Protein half-life (hr)
8
*
*
*
*
(Myc-) WT C317Y V338E P375S T377M
Flag-WT
+ HA-KChIP2

(f)
Flag-WT + HA-KChIP2
Total Surface

(Myc-) WT C317Y V338E P375S T377M WT C317Y V338E P375S T377M
100
70
α-Flag
100
70
α-Myc
40
α-Actin

Normalized signal
1.0-
*
0.5-
*
*
0.0-
(Myc-) WT C317Y V338E P375S T377M
Flag-WT + HA-KChIP2
Surface signal

Normalized ratio
1.5-
1.0-
*
*
*
*
0.5-
0.0-
(Myc-) WT C317Y V338E P375S T377M
Flag-WT + HA-KChIP2
Surface/total signal

FIGURE 8 Continued.

HSIAO ET AL.

TABLE 5 ACMG Classification of KCND3 mutations in this study

| Sequence variant and amino acid change | c.950G>A p.C317Y | c.1013T>A p.V338E | c.1123C>T p.P375S | c.1130C>T p.T377M |
|---------------------------------------|------------------|-------------------|-------------------|-------------------|
| ACMG criteria                          |                  |                   |                   |                   |
| De novo mutation (PS2)                 | Yes              | No                | No                | No                |
| Functional studies (PS3)               | Yes              | Yes               | Yes               | Yes               |
| Mutational hotspot (PM1)               | Yes              | Yes               | Yes               | Yes               |
| Absent in population databases (PM2)   | Yes              | Yes               | Yes               | Yes               |
| Cosegregation (PP1)                    | Yes              | Yes               | Yes               | Yes               |
| In silico analysis evidence (PP3)      | Yes              | Yes               | Yes               | Yes               |
| ACMG classification                    | Pathogenic       | Pathogenic        | Pathogenic        | Pathogenic        |

Abbreviations: ACMG, American College of Medical Genetics and Genomics; PS2, pathogenic strong 2; PS3, pathogenic strong 3; PM1, pathogenic moderate 1; PM2, pathogenic moderate 2; PP1, pathogenic supporting 1; PP3, pathogenic supporting 3.

by the findings that Caᵥ3 current density critically determines regional differences in membrane excitability and spike output of granule cells (Heath et al., 2014), and that extracellular Ca²⁺ may regulate inhibitory outputs from stellate cells to cerebellar Purkinje cells (Anderson et al., 2013). Consequently, it is likely that dominant-negative effects of SCA19/22-associated mutants on voltage-dependent gating of Kᵥ4.3 WT may similarly result in aberrant network excitability in the cerebellar circuits, which may in part manifest some of the ataxic features observed in the patients.

Despite the implementation of several clinical trials based on mechanistic studies of the pathogenesis of SCA, currently there is no specific medication approved for treating SCA patients (Ashizawa, Oz, & Paulson, 2018). Therapeutic approaches targeting voltage-gated K⁺ channels have been implicated in the treatment for cancer, arrhythmia, multiple sclerosis, and epilepsy (Wulff, Castle, & Pardo, 2009). Interestingly, 4-aminopyridine (4-AP), a specific K⁺ channel blocker, was shown to restore the pacemaker activity of Purkinje cells, and has been applied for treating patients with a specific form of cerebellar ataxia (Alvina & Khodakhah, 2010). It remains an open question, however, whether 4-AP may be suitable for SCA19/22 patients. Furthermore, based on our findings that the pathomechanism of SCA19/22 may entail defective Kᵥ4.3 protein biosynthesis, it would be worthwhile in the future to identify novel chemical agents effective in facilitating protein level and/or functional expression of human Kᵥ4.3 channel, followed by thorough evaluation of their therapeutic potential for SCA19/22 patients.

In conclusion, we utilized the NGS technology to identify two novel KCND3 mutations from two distinct pedigrees with molecularly unassigned SCA. Our clinical observations broaden the genotypic and phenotypic spectrum of SCA19/22 to include cerebellar ataxia, developmental delay, cognitive impairment, myoclonus, and dystonia, thereby emphasizing the significance of KCND3 mutations in a wide range of different neurological disorders. Importantly, our multidisciplinary approaches to understanding the pathogenesis of SCA19/22 may shed light on future development of novel therapeutic strategy.

ACKNOWLEDGMENTS

We are grateful to Drs. Yi-Chu-Liao, Pei-Chien Tsai, and Yi-Chung Lee for targeted sequencing analyses, Ms. Jing-Jia Huang and Yun-Chia Chien for technical assistance, and the participation of all the patients.

AUTHOR CONTRIBUTIONS

B.W.S. and C.J.J. conceived and designed the study. C.T.H., Y.T.L., and B.W.S. collected patient information and executed neurogenetics

FIGURE 8 Disease-associated mutants disrupt protein biosynthesis of Kᵥ4.3 WT. Flag-Kᵥ4.3 WT (Flag-WT) was equimolar coexpressed with Myc-Kᵥ4.3 WT (Myc-WT) or various mutants in HEK293T cells. (a) Coimmunoprecipitation experiments in the presence of KChIP2. (Left) Coexpression of Myc-WT with the Flag vector or Flag-WT. Myc-WT was immunoprecipitated by the anti-Flag antibody only when Flag-WT was present. (Right) Coexpression of Flag-WT with Myc-WT or individual mutants. All of the Myc-Kᵥ4.3 constructs were readily coimmunoprecipitated with Flag-WT. (b), (c) Quantification of total Flag-WT protein level in the presence of the indicated Myc-Kᵥ4.3 constructs (n = 5–10). (b) Normalized Flag-WT protein levels in the absence of KChIP2: (+)p.C317Y, 0.8 ± 0.04; (+)p.V338E, 0.5 ± 0.06; (+)p.P375S, 0.7 ± 0.04; and (+)p.T377M, 0.8 ± 0.04. (c) Normalized Flag-WT protein levels in the presence of KChIP2: (+)p.C317Y, 0.7 ± 0.11; (+)p.V338E, 0.4 ± 0.04; (+)p.P375S, 0.6 ± 0.09; and (+)p.T377M, 0.7 ± 0.10. Standardized total Flag-WT protein density was normalized with respect to the Myc-WT coexpression control. (d), (e) Quantification of Flag-WT protein degradation kinetics in the presence of KChIP2 and Myc-WT (black), p.C317Y (red), p.V338E (green), p.P375S (blue), and p.T377M (cyan) (n = 5–12). Mean Flag-WT protein half-lives (hr): (+)WT, 7.6 ± 0.8; (+)p.C317Y, 3.9 ± 0.5; (+)p.V338E, 2.4 ± 0.3; (+)p.P375S, 3.3 ± 0.4; and (+)p.T377M, 4.9 ± 0.4. (f) Quantification of Flag-WT membrane trafficking efficiency in the presence of KChIP2 and the indicated Myc-Kᵥ4.3 constructs (n = 8–11). Normalized Flag-WT surface protein levels: (+)p.C317Y, 0.7 ± 0.1; (+)p.V338E, 0.3 ± 0.1; (+)p.P375S, 0.5 ± 0.1; and (+)p.T377M, 0.4 ± 0.1. Normalized Flag-WT membrane trafficking efficiencies: (+)p.C317Y, 0.9 ± 0.06; (+)p.V338E, 1.2 ± 0.14; (+)p.P375S, 0.5 ± 0.08; and (+)p.T377M, 0.3 ± 0.04. Asterisks denote significant difference from the Myc-WT control (*p < .05)

analyses. S. J. F. and C. Y. Z. completed electrophysiological experiments. S. J. F. and Y. H. L. performed subcloning and biochemical studies. S. J. F. and Y. H. L. carried out immunofluorescence work. C. T. H., S. J. F., and C. J. J. prepared the figures. C. T. H., C. Y. T., and C. J. J. drafted and revised the manuscript with inputs from the co-authors. All authors approved the final version of the manuscript.

**CONFLICT OF INTERESTS**

The authors report no competing interests.

**ORCID**

Chih-Yung Tang [http://orcid.org/0000-0003-1065-2865](http://orcid.org/0000-0003-1065-2865)

Chung-Jiuan Jeng [http://orcid.org/0000-0001-6271-5704](http://orcid.org/0000-0001-6271-5704)

**REFERENCES**

Adzhubei, I. A., Schmidt, S., Peshkin, L., Ramensky, V. E., Gerasimova, A., Bork, P., ... Sunyaev, S. R. (2010). A method and server for predicting damaging missense mutations. *Nature Methods*, 7(4), 248–249. https://doi.org/10.1038/nmeth0410-248

Alvina, K., & Khodakhah, K. (2010). The therapeutic mode of action of 4-aminopyridine in cerebellar ataxia. *Journal of Neuroscience*, 30(21), 7258–7268. https://doi.org/10.1523/JNEUROSCI.3582-09.2010

An, W. F., Bowlby, M. R., Betty, M., Cao, J., Ling, H. P., Mendoza, G., & Rhodes, K. J. (2000). Modulation of A-type potassium channels by a family of calcium sensors. *Nature*, 403(6769), 553–556. https://doi.org/10.1038/35000592

Anderson, D., Engbers, J. D., Heath, N. C., Bartoletti, T. M., Mehaffey, W. H., Zamponi, G. W., & Turner, R. W. (2013). The Cav3-Kv4 complex acts as a calcium sensor to maintain inhibitory charge transfer during extracellular calcium fluctuations. *Journal of Neuroscience*, 33(18), 7811–7824. https://doi.org/10.1523/JNEUROSCI.5384-12.2013

Anderson, D., Mehaffey, W. H., Iftinca, M., Rehak, R., Engbers, J. D., Hameed, S., & Turner, R. W. (2010). Regulation of neuronal activity by Cav3-Kv4 channel signaling complexes. *Nature Neuroscience*, 13(3), 333–337. https://doi.org/10.1038/nn.2493

Ashizawa, T., Oz, G., & Paulson, H. L. (2018). Spinocerebellar ataxias: Prospects and challenges for therapy development. *Nature Reviews Neurology*, 14(10), 590–605. https://doi.org/10.1038/s41582-018-0051-6

Bahring, R., Dannenberg, J., Peters, H. C., Leicher, T., Pongs, O., & Isbrandt, D. (2001). Conserved Kv4 N-terminal domain critical for effects of Kv channel-interacting protein 2.2 on channel expression and gating. *Journal of Biological Chemistry*, 276(26), 23888–23894. https://doi.org/10.1074/jbc.M101320200

Capriotti, E., Calabrese, R., Fariselli, P., Martelli, P. L., Altman, R. B., & Casadio, R. (2013). WS-SNPs&GO: A web server for predicting the deleterious effect of human protein variants using functional annotation. *BMC Genomics*, 14(Suppl 3), S6. https://doi.org/10.1186/1471-2164-14-s3-s6

Chapman, M. L., Krovetz, H. S., & VanDongen, A. M. (2001). GYGD pore motifs in neighbouring potassium channel subunits interact to determine ion selectivity. *Journal of Physiology*, 530(Pt 1), 21–33.

Chen, H. C., Lirng, J. F., Soong, B. W., Guo, W. Y., Wu, H. M., Chen, C. C., & Chang, C. Y. (2014). The merit of proton magnetic resonance spectroscopy in the longitudinal assessment of spinocerebellar ataxias and multiple system atrophy-cerebellar type. *Cerebellum & Ataxias*, 1, 17. https://doi.org/10.1186/s40673-014-0017-4

Choi, K. D., Kim, J. S., Kim, H. J., Jung, I., Jeong, S. H., Lee, S. H., & Choi, J. H. (2017). Genetic variants associated with episodic ataxia in Korea.

Scientific Reports, 7(1), 13855. https://doi.org/10.1038/s41598-017-14254-7

Chothia, C. (1984). Principles that determine the structure of proteins. *Annual Review of Biochemistry*, 53, 537–572. https://doi.org/10.1146/annurev.bi.53.070184.002541

Coutelier, M., Coarelli, G., Monin, M. L., Konop, J., Davoine, C. S., Tesson, C., & Durr, A. (2017). A panel study on patients with dominant cerebellar ataxia highlights the frequency of channelopathies. *Brain*, 140(6), 1579–1594. https://doi.org/10.1093/brain/awx081

Coutelier, M., Hammer, M. B., Stevanin, G., Monin, M. L., Davoine, C. S., Mochel, F., & Durr, A. (2018). Efficacy of exome-targeted capture sequencing to detect mutations in known cerebellar ataxia genes. *JAMA Neurol.*, 75(5), 591–599. https://doi.org/10.1001/jamaneurol.2017.5121

Dilks, D., Ling, H. P., Cockett, M., Sokol, P., & Numann, R. (1999). Cloning and expression of the human kv4.3 potassium channel. *Journal of Neurophysiology*, 81(4), 1974–1977. https://doi.org/10.1152/jn.1999.81.4.1974

Duarrri, A., Jezierska, J., Fokkens, M., Meijer, M., Schelhaas, H. J., den Dunnen, W. F., & Verbeek, D. S. (2012). Mutations in potassium channel kcnd3 cause spinocerebellar ataxia type 19. *Annals of Neurology*, 72(6), 870–880. https://doi.org/10.1002/ana.23700

Duarrri, A., Lin, M. C., Fokkens, M. R., Meijer, M., Smeets, C. J., Nibbeling, E. A., & Verbeek, D. S. (2015). Spinocerebellar ataxia type 19/22 mutations alter heterocomplex Kv4.3 channel function and gating in a dominant manner. *Cellular and Molecular Life Science*, 72(17), 3387–3399. https://doi.org/10.1007/s00018-015-1894-2

Duarrri, A., Nibbeling, E., Fokkens, M. R., Meijer, M., Boddeke, E., Lagrange, E., & Verbeek, D. S. (2013). The L450F [Corrected] mutation in KCND3 brings spinocerebellar ataxia and Brugada syndrome closer together. *Neurogenetics*, 14(3-4), 257–258. https://doi.org/10.1007/s10048-013-0370-0

Durr, A. (2010). Autosomal dominant cerebellar ataxias: Polyglutamine expansions and beyond. *Lancet Neurology*, 9(9), 885–894. https://doi.org/10.1016/s1474-4422(10)70183-6

Eswar, N., Webb, B., Marti-Renom, M. A., Madhusudhan, M. S., Eramian, D., Shen, M. Y., & Sali, A. (2006). Comparative protein structure modeling using Modeller. *Current Protocols in Bioinformatics*, 15, 5.6.1–5.6.30. https://doi.org/10.1002/0471250953 .bi0506s15

Heath, N. C., Rizwan, A. P., Engbers, J. D., Anderson, D., Zamponi, G. W., & Turner, R. W. (2014). The expression pattern of a Cav3-Kv4 complex differentially regulates spike output in cerebellar granule cells. *Journal of Neuroscience*, 34(26), 8800–8812. https://doi.org/10.1523/JNEUROSCI.0981-14.2014

Hoffman, D. A., Magee, J. C., Colbert, C. M., & Johnston, D. (1997). K+ channel regulation of signal propagation in dendrites of hippocampal pyramidal neurons. *Nature*, 387(6636), 869–875. https://doi.org/10.1038/43119

Hsiao, C. T., Liu, Y. T., Liao, Y. C., Hsu, T. Y., Lee, Y. C., & Soong, B. W. (2017). Mutational analysis of ITPR1 in a Taiwanese cohort with cerebellar ataxias. *PLoS One*, 12(11), e0187503. https://doi.org/10.1371/journal.pone.0187503

Hsu, Y. H., Huang, H. Y., & Tsaur, M. L. (2003). Contrasting expression of Kv4.3, an A-type K+ channel, in migrating Purkinje cells and other post-migratory cerebellar neurons. *European Journal of Neuroscience*, 18(3), 601–612.

Huin, V., Strubi-Vuillaume, I., Dujardin, K., Brion, M., Delliaux, M., Dellacherie, D., & Devos, D. (2017). Expanding the phenotype of SCA19/22: Parkinsonism, cognitive impairment and epilepsy. *Parkinsonism & Related Disorders*, 45, 85–89. https://doi.org/10.1016/j.parkreldis.2017.09.014

Isbrandt, D., Leicher, T., Waldschutz, R., Zhu, X., Luhmann, U., Michel, U., & Pongs, O. (2000). Gene structures and expression profiles of three

human KCND (Kv4) potassium channels mediating A-type currents I
(TO) and I(SA). Genomics, 64(2), 144–154. https://doi.org/10.1006/
geno.2000.6117

Jeng, C. J., Chen, Y. T., Chen, Y. W., & Tang, C. Y. (2006). Dominant-negative
effects of human P/Q-type Ca2+ channel mutations associated with
episodic ataxia type 2. American Journal of Physiology: Cell Physiology,
290(4), C1209–C1220. https://doi.org/10.1152/ajpcell.00247.2005

Jerng, H. H., & Pfaffinger, P. J. (2014). Modulatory mechanisms and
multiple functions of somatodendritic A-type K (+) channel auxiliary
subunits. Frontiers in Cellular Neuroscience, 8, 82. https://doi.org/10.
3389/fncel.2014.00082

Jerng, H. H., Pfaffinger, P. J., & Covarrubias, M. (2004). Molecular
physiology and modulation of somatodendritic A-type potassium
channels. Molecular and Cellular Neuroscience, 27(4), 343–369. https://
doi.org/10.1016/j.mcn.2004.06.011

Kollo, M., Holderith, N. B., & Nusser, Z. (2006). Novel subcellular
distribution pattern of A-type K+ channels on neuronal surface.
Journal of Neuroscience, 26(10), 2684–2691. https://doi.org/10.1523/
jneurosci.5257-05.2006

Kumar, P., Henikoff, S., & Ng, P. C. (2009). Predicting the effects of
coding non-synonymous variants on protein function using the SIFT
algorithm. Nature Protocols, 4(7), 1073–1081. https://doi.org/10.1038/
nprot.2009.86

Kurihara, M., Ishiura, H., Sasaki, T., Otsuka, J., Hayashi, T., Terao, Y., & 
Tsui, S. (2018). Novel de novo KCND3 mutation in a Japanese patient
with intellectual disability, cerebellar ataxia, myoclonus, and dystonia.
Cerebellum, 17(2), 237–242. https://doi.org/10.1007/s12311-017-
0883-4

Lee, Y. C., Durr, A., Majczenko, K., Huang, Y. H., Liu, Y. C., Lien, C. C., &
Soong, B. W. (2012). Mutations in KCND3 cause spinocerebellar
ataxia type 22. Annals of Neurology, 72(6), 859–869. https://doi.org/
10.1002/ana.23701

Lee, Y. C., Liao, Y. C., Wang, P. S., Lee, I. H., Lin, K. P., & Soong, B. W.
(2011). Comparison of cerebellar ataxias: A three-year prospective
longitudinal assessment. Movement Disorders, 26(11), 2081–2087.
https://doi.org/10.1002/mds.23809

Liao, Y. C., Liu, Y. T., Tsai, P. C., Chang, C. C., Huang, Y. H., Soong, B. W., &
Lee, Y. C. (2015). Two novel de novo GARS mutations cause early-
onset axonal charcot-marie-tooth disease. PLoS One, 10(8), e0133423.
https://doi.org/10.1371/journal.pone.0133423

Long, S. B., Tao, X., Campbell, E. B., & MacKinnon, R. (2007). Atomic
structure of a voltage-dependent K+ channel in a lipid membrane-like
environment. Nature, 450(7168), 376–382. https://doi.org/10.1038/
nature06265

Molineux, M. L., Fernandez, F. R., Mehaffey, W. H., & Turner, R. W. (2005).
A-type and T-type currents interact to produce a novel spike latency-
voltage relationship in cerebellar stellate cells. Journal of Neuroscience,
25(47), 10863–10873. https://doi.org/10.1523/jneurosci.3436-05.2005

Nadal, M. S., Amarillo, Y., Vega-Saenz de Miera, E., & Rudy, B. (2001).
Evidence for the presence of a novel Kv4-mediated A-type K(+)
channel-modifying factor. Journal of Physiology, 537(Pt 3), 801–809.

Ohya, S., Tanaka, M., Oku, T., Furuyama, T., Mori, N., Giles, W. R., &
Imaizumi, Y. (2001). Regional expression of the splice variants of
Kv4.3 in rat brain and effects of C-terminus deletion on expressed
K+ currents. Life Sciences, 68(15), 1703–1716.

Paucar, M., Bergendal, A., Gustavsson, P., Nordenskjold, M., Laffita-Mesa,
J., Savitcheva, I., & Svenningsson, P. (2018). Novel features and
abnormal pattern of cerebral glucose metabolism in spinocerebellar
ataxia 19. Cerebellum, 17(4), 465–476. https://doi.org/10.1007/
s12311-018-0927-4

Pettersen, E. F., Goddard, T. D., Huang, C. C., Couch, G. S., Greenblatt, D.
M., Meng, E. C., & Ferrin, T. E. (2004). UCSF chimera--a visualization
system for exploratory research and analysis. Journal of Computational
Chemistry, 25(13), 1605–1612. https://doi.org/10.1002/jcc.20084

Richards, S., Aziz, N., Bale, S., Bick, D., Das, S., Gastier-Foster, J., & Rehm,
H. L. (2015). Standards and guidelines for the interpretation of
sequence variants: A joint consensus recommendation of the
American College of Medical Genetics and Genomics and the
Association for Molecular Pathology. Genetics in Medicine, 17(5),
405–424. https://doi.org/10.1038/gim.2015.30

Rizwan, A. P., Zhan, X., Zamponi, G. W., & Turner, R. W. (2016). Long-term
potentiation at the mossy fiber-granule cell relay invokes postsynaptic
second-messenger regulation of Kv4 channels. Journal of Neuroscience,
36(44), 11196–11207. https://doi.org/10.1523/jneurosci.2051-16.
2016

Schmitz-Hubsch, T., du Montcel, S. T., Baliko, L., Berciano, J., Boesch, S.,
Depondt, C., & Fancellu, R. (2006). Scale for the assessment and rating
of ataxia: Development of a new clinical scale. Neurology, 66(11),
1717–1720. https://doi.org/10.1212/01.wnl.0000219042.60538.92

Schols, L., Bauer, P., Schmidt, T., Schulte, T., & Riess, O. (2004). Autosomal
dominant cerebellar ataxias: Clinical features, genetics, and patho-
genesis. Lancet Neurology, 3(5), 291–304. https://doi.org/10.1016/
s1474-4422(04)00737-9

Schwarz, J. M., Cooper, D. N., Schuelke, M., & Seelow, D. (2014).
MutationTaster2: Mutation prediction for the deep-sequencing age.
Nature Methods, 11(4), 361–362. https://doi.org/10.1038/nmeth.2890

Serodio, P., & Rudy, B. (1998). Differential expression of Kv4 K+ channel
subunits mediating subthreshold transient K+ (A-type) currents in rat
brain. Journal of Neurophysiology, 79(2), 1081–1091. https://doi.org/
10.1152/jn.1998.79.2.1081

Serodio, P., Vega-Saenz de Miera, E., & Rudy, B. (1996). Cloning of a novel
component of A-type K+ channels operating at subthreshold poten-
tials with unique expression in heart and brain. Journal of Neurophy-
siology, 75(5), 2174–2179. https://doi.org/10.1152/jn.1996.75.5.2174

Shieh, C. C., Klemic, K. G., & Kirsch, G. E. (1997). Role of transmembrane
segment S5 on gating of voltage-dependent K+ channels. Journal of
General Physiology, 109(6), 767–778.

Smets, K., Duarri, A., Deconinck, T., Ceulemans, B., van de Warrenburg, B.
P., Zuchner, S., & Baets, J. (2015). First de novo KCND3
mutation causes severe Kv4.3 channel dysfunction leading to early
onset cerebellar ataxia, intellectual disability, oral apraxia and
epilepsy. BMC Medical Genetics, 16, 51. https://doi.org/10.1186/
s12881-015-0200-3

Soong, B. W., & Morrison, P. J. (2018). Spinocerebellar ataxias. Handbook
of Clinical Neurology, 155, 143–174. https://doi.org/10.1016/b978-0-
444-64189-2.00010-x

Takimoto, K., Yang, E. K., & Conforti, L. (2002). Palmitoylation of
KChIP splicing variants is required for efficient cell surface
expression of Kv4.3 channels. Journal of Biological Chemistry,
277(30), 26904–26911. https://doi.org/10.1074/jbc.M203651200

Wulff, H., Castle, N. A., & Pardo, L. A. (2009). Voltage-gated potassium
channels as therapeutic targets. Nature Reviews Drug Discovery, 8(12),
982–1001. https://doi.org/10.1038/nrd2983

---

**SUPPORTING INFORMATION**

Additional supporting information may be found online in the
Supporting Information section.

---

How to cite this article: Hsiao C-T, Fu S-J, Liu Y-T, et al. Novel
SCA19/22-associated KCND3 mutations disrupt human Kv4.3
protein biosynthesis and channel gating. Human Mutation.
2019;40:2088–2107. https://doi.org/10.1002/humu.23865
