
Chen et al. Journal of Biomedical Science (2019) 26:80
https://doi.org/10.1186/s12929-019-0569-y

![MOST Logo](https://via.placeholder.com/150)

The cost of publication in *Journal of Biomedical Science* is borne by the Ministry of Science and Technology, Taiwan.

---

**REVIEW**                                                                                       **Open Access**

# Ubiquitin-mediated regulation of autophagy

Ruey-Hwa Chen${}^{1,2*}$, Yu-Hsuan Chen${}^{1,2}$ and Tzu-Yu Huang${}^{1,2}$

---

## Abstract

Autophagy is a major degradation pathway that utilizes lysosome hydrolases to degrade cellular constituents and is often induced under cellular stress conditions to restore cell homeostasis. Another prime degradation pathway in the cells is ubiquitin-proteasome system (UPS), in which proteins tagged by certain types of polyubiquitin chains are selectively recognized and removed by proteasome. Although the two degradation pathways are operated independently with different sets of players, recent studies have revealed reciprocal cross talks between UPS and autophagy at multiple layers. In this review, we summarize the roles of protein ubiquitination and deubiquitination in controlling the initiation, execution, and termination of bulk autophagy as well as the role of ubiquitination in signaling certain types of selective autophagy. We also highlight how dysregulation of ubiquitin-mediated autophagy pathways is associated with a number of human diseases and the potential of targeting these pathways for disease intervention.

**Keywords:** Protein ubiquitination, Autophagy, Selective autophagy, Ubiquitin E3 ligase, Deubiquitinating enzyme

---

## Introduction

Ubiquitin-proteasome system (UPS) and autophagy are two major cellular degradation machineries in eukaryotes, both of which are crucial in eliminating misfolded/unfolded proteins to maintain cell and tissue homeostasis and to prevent aging-related changes and a plethora of human diseases. In general, short-lived and soluble misfolded/unfolded proteins are targeted by UPS, whereas long-lived and insoluble protein aggregates are eliminated by autophagy [1, 2]. The cargos of autophagy are not limited to proteins and include dysfunctional or superfluous organelles. Although the two systems are operated independently, recent studies have revealed multiple layers of interconnections between UPS and autophagy. For instance, inhibition of UPS leads to a compensatory stimulation of autophagy via several mechanisms, whereas autophagy inhibition activates or impairs proteasomal flux depending on the cellular and environmental conditions [3, 4]. In addition, components of either system can serve as the proteolytic targets of the other system [4]. In this review, we chose to focus on the role of protein ubiquitination in regulating autophagy. Other aspects of the crosstalk between UPS and autophagy have been reviewed elsewhere [4, 5].

### Overview of protein ubiquitination

Ubiquitination is a posttranslational modification involving the conjugation of the 76 amino acid ubiquitin to the lysine residue of other proteins. This modification is mediated by the sequential action of E1 ubiquitin-activating enzyme, E2 ubiquitin conjugating enzyme, and E3 ubiquitin ligase [6]. The removal of ubiquitin from the substrate is catalyzed by a class of deubiquitinating enzymes (DUBs) [7]. Ubiquitin contains seven lysine residues and one N-terminal methionine residue, each of which can be attached to another ubiquitin moiety. As a consequence, proteins can be modified by ubiquitin monomer or polymer with different length and linkage types, making ubiquitination as one of the most elaborate and versatile posttranslational modifications [8–10]. In the homotypic polyubiquitination, all building blocks of the chain are connected through the same lysine or methionine residue and a total of eight different chain types can be formed. To add the complexity, heterotypic chain, which contains more than one linkage types, can also be formed and can be further categorized into

---

* Correspondence: rhchen@gate.sinica.edu.tw
${}^{1}$ Academia Sinica, Institute of Biological Chemistry, Taipei 115, Taiwan
${}^{2}$ Institute of Biochemical Sciences, College of Life Science, National Taiwan University, Taipei 100, Taiwan

© The Author(s). 2019 Open Access This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ([http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/)), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver ([http://creativecommons.org/publicdomain/zero/1.0/](http://creativecommons.org/publicdomain/zero/1.0/)) applies to the data made available in this article, unless otherwise stated.

mixed and branched chains. Importantly, these structurally distinct ubiquitin modifications are recognized by effector proteins with linkage-specific ubiquitin-binding domains to result in diverse functional outcomes, such as degradation, signal transduction, and alteration in subcellular localization. For instance, K48- and K11-linked chains are pivotal signals for proteasomal degradation, whereas K6, K27, K33, K63, and linear chains are usually of a nondegradative fate [8–10]. Recent studies have further revealed the role of branched ubiquitin chain in changing the nondegradative to degradative fate or in the enhancement of degradative signal [10–13].

**Overview of autophagy**

Autophagy is a lysosome-based degradation program activated by various cellular stresses including nutrient/energy starvation, hypoxia, ER stress, hypoxia, and organelle damage. During autophagic process, double-membrane vesicles, termed autophagosomes, are formed in the cytoplasm to sequester cellular components. This is followed by fusion of autophagosome with lysosome and degradation/recycling of sequestered cellular components to generate macromolecular building blocks [2, 14, 15]. The prime functions of autophagy include the removal of harmful substances (such as damaged organelles, protein aggregates, and intracellular pathogens), adaptation to metabolic stresses, and renovation during differentiation and development. Dysfunction of autophagic process has been associated with numerous diseases, including infectious diseases, cancer, neurodegeneration, cardiovascular disorders, and aging [16–18].

In the past decade, the molecular mechanisms of autophagy have been intensively studied. Autophagy initiation is governed by the ULK1 serine/threonine kinase which forms a complex with FIP200, ATG13, and ATG101 [19, 20]. Upon various cellular stresses, ULK1 is activated, resulting in the phosphorylation of multiple downstream factors to trigger the autophagy cascade. One effector of ULK1 is the class III PI3K complex, which contains the lipid kinase VPS34 and regulator proteins Beclin-1, VPS15, and ATG14 [21]. ULK1 promotes the activation and the recruitment of class III PI3K complex to the autophagosome formation site (phagophore), where it generates PI3P to function in autophagosome nucleation [22]. ATG9, the only transmembrane protein in the core autophagic machinery, is thought to supply membrane to autophagosome [23]. Additionally, ATG9 binds ATG2 and WIPI proteins (ATG18 in yeast, the PI3P effectors), to participate in the early stage of autophagosome biogenesis from ER [24]. Further expansion and completion of the autophagosome depends on the two ubiquitin-like conjugation systems [25]. The ATG12 conjugation system is responsible for the conjugation of ubiquitin-like protein ATG12 to ATG5, which in turn forms a complex with

ATG16L1. The ATG12-ATG5-ATG16L1 complex functions as the E3 ligase for the second conjugation system, in which the ubiquitin-like LC3 subfamily proteins (ATG8 in yeast) are conjugated to the membrane-residing phosphatidylethanolamine (PE). PE modification of the LC3 family proteins are essential for the elongation and closure of autophagosome membrane. To achieve autophagic degradation, autophagosome needs to fuse with lysosome or late endosome. The fusion requires UVRAG-containing class III PI3K complex (also known as the PI3K complex II), tethering factors such as HOPS complex, SNARE proteins such as STX17 on autophagosome membrane and VAMP8 and SNAP29 on endosome/lysosome, RAB proteins such as RAB7, and the LC3 family proteins [26]. After fusion, the inner membrane of autophagosome and materials wrapped in the inner membrane are degraded and the resulting small molecules are recycled to the cytosol.

**Regulation of autophagy induction by ubiquitination and deubiquitination**

Induction of autophagy needs to be tightly controlled for cells to cope with various stressed conditions. Reversible ubiquitination of the core autophagy induction factors, i.e., the subunits of ULK1 and PI3K complexes, has been revealed as a common mechanism for turning on and off the autophagy process under different cellular contexts. In addition, ubiquitination participates in positive feedback regulations for timely induction of autophagy.

**The role of E3 ligases**

The ubiquitin ligase TRAF6, which mediates the formation of K63-linked ubiquitin chain, plays important roles in autophagy induction. TRAF6 promotes K63 ubiquitination of ULK1, thereby enhancing ULK1 stability and function [27]. Notably, recruitment of ULK1 to TRAF6 requires the cofactor AMBRA1, a subunit of class III PI3K. Since ULK1 phosphorylates and activates AMBRA1, the TRAF6-mediated ULK1 ubiquitination participates in a positive feedback mechanism to potentiate autophagy initiation. TRAF6 also catalyzes the K63 ubiquitination of Beclin-1 [28]. This ubiquitination occurs at the BH3 domain of Beclin-1 and thus blocks Beclin-1 interaction with Bcl-2 to promote autophagy in response to the activation of Toll-like receptor (TLR) 4. Under starvation, Beclin-1 K63 ubiquitination is mediated by Cul4 E3 ligase with AMBRA1 as a substrate adaptor, thereby promoting autophagy [29]. However, the role of AMBRA1 in regulating autophagy initiation is complex. Another study reported that AMBRA1 is transiently dissociated from Cul4 at an early stage of autophagy induction. The released AMBRA1 inhibits Cul5 ubiquitin ligase, thereby stabilizing mTOR inhibitor DEPTOR [30]. Since ULK1 activity is required for the dissociation of AMBRA1 from Cul4,

this regulation of AMBRA1 binding partner establishes a feedback mechanism for a rapid autophagy induction. In contrast to TRAF6 and AMBRA1, the ubiquitin ligases NEDD4 and RNF216 promote Beclin-1 proteasomal degradation to inhibit autophagy by assembling K11- and K48-linked ubiquitin chains on Beclin-1, respectively [31, 32]. Other autophagy inducing factors also undergo degradable ubiquitination. For instance, AMBRA1 K48 ubiquitination is promoted by RNF2 E3 ligase, leading to its proteasomal degradation [33]. VPS34 and ATG14 ubiquitination and degradation are mediated by Cull1 E3 ligase containing FBXL20 and Cul3 E3 ligase containing ZBTB16 in response to DNA damage and G protein coupled signaling, respectively [34, 35]. Together, subunits of the ULK1 and VPS34 complexes are targeted by multiple ubiquitin ligases (Fig. 1a). While K63 ubiquitination promotes autophagy induction in response to stressed conditions or accelerates autophagy initiation through feedback mechanisms, ubiquitination by K48- and K11-linked chain types impairs autophagy induction through degradation of the core autophagic proteins.

The roles of DUBs

Among the autophagy inducing factors, Beclin-1 is a popular target for ubiquitination. Similarly, Beclin-1 appears as a hub for DUB-mediated regulation (Fig. 2). A20, which specifically targets the K63-linked ubiquitin chain, antagonizes the function of TRAF6 on modifying Beclin-1, thereby attenuating autophagy induction in response to TLR signaling [28]. Belcin-1 K63 ubiquitination is also negatively controlled by USP14, resulting in autophagy inhibition. Importantly, USP14 is itself activated by Akt-mediated phosphorylation and this mechanism contributes to the inhibition of autophagy activity by Akt [36]. Several DUBs influence on Beclin-1 degradable ubiquitination. For instance, USP10 and USP13 reduce Beclin-1 ubiquitination to prevent its degradation. The function of USP10 and USP13 is reversed by a chemical compound, spautin-1, which inhibits autophagy by promoting Beclin-1 degradation. Interestingly, Beclin-1 positively controls the stability of USP10 and USP13, suggesting the existence of a feedback mechanism to maintain Beclin-1 level [37]. Beclin-1 stabilization

![Figure 1](attachment:Figure_1.png)

Fig. 1 Functional roles of ubiquitin ligases in regulating autophagy. Summary of the proteins acting in the initiation (a), autophagosome biogenesis (b), and autophagosome maturation (c) steps of the autophagic process that are subjected to ubiquitination by various E3 ligases. The ubiquitin chain types and the effect of ubiquitination on autophagy (promotion or inhibition) are indicated.

Chen et al. Journal of Biomedical Science (2019) 26:80

Page 4 of 12

![Diagram](attachment:diagram.png)

Fig. 2 Functional roles of DUBs in regulating autophagy initiation. Summary of DUBs that regulate autophagy initiation by targeting ULK1 or Beclin-1. The ubiquitin chain types and the effect of ubiquitination on autophagy (promotion or inhibition) are indicated.

is also promoted by USP19 and ataxin 3, which specifically removes K11- and K48-ubiquitin chain from Belcin-1, respectively [38, 39]. Finally, Beclin-1 is indirectly regulated by DUB USP33, which deubiquitates Beclin-1 partner RALB [40]. This deubiquitination event is important for the binding of RALB with the exocyst component EXO84 and Beclin-1, which in turn drives the assembly of active ULK1 and Beclin-1-VPS34 complex for autophagy initiation [41]. The ability of Belcin-1 to be targeted by multiple DUBs highlights the importance of reversible ubiquitination in regulating autophagy initiation under different cellular contexts, even though the upstream signals regulating these deubiquitination events and the specific ubiquitin ligases that counteract these DUBs remain mostly uncharacterized.

Besides Beclin-1, ULK1 is regulated by DUBs (Fig. 2). ULK1 K63 ubiquitination is antagonized by USP1. This function of USP1, however, regulates ULK1 cellular compartmentalization by promoting ULK1 localization to the Triton X-100 soluble fraction. Depletion of USP1 or inhibition of USP1 activity by small molecular inhibitor leads to the formation of ULK1 insoluble aggregates which also contain p62 and the aggregation marker HDAC6, thereby inhibiting canonical autophagic flux but promoting lysosome-mediated degradation of p62 [42]. The degradable ULK1 ubiquitination is reversed by USP20. Under basal conditions, USP20 maintains ULK1 level to facilitate autophagy initiation. Under prolonged starvation, the binding of USP20 to ULK1 is diminished, leading to autophagy inhibition [43]. Thus, ULK1 deubiquitination could control the dynamics of autophagy process and the decision between canonical and nonconventional autophagy.

Regulation of autophagosome biogenesis by ubiquitination

A key event for autophagosome biogenesis is the recruitment of PI3P-binding proteins to phagophore, such as WIPI2. This is followed by phagophore recruitment of ATG12-ATG5-ATG16L1 complex for the lipidation of LC3 family proteins and subsequent autophagosome expansion [44, 45]. Recent studies have revealed that both WIPI2 and ATG16L1 are subjected to ubiquitin-mediated regulation (Fig. 1b). Ubiquitination of ATG16L1 is mediated by gigaxonin [46], a substrate adaptor of Cul3 ubiquitin ligase mutated in a neurodegenerative disease called giant axonal neuropathy [47]. Interestingly, gigaxonin-mediated ubiquitination promotes ATG16L1 degradation through both proteasomal and autophagic routes and ATG16L1 aggregates are accumulated in gigaxonin knockout neurons. As to WIPI2, the ubiquitin ligase HUWE1 is responsible for its ubiquitination and proteasomal degradation. Importantly, targeting WIPI2 to HUWE1 requires mTORC1-dependent phosphorylation on S395 of WIPI2, uncovering a link of mTORC1 to WIPI2 degradation for autophagy inhibition [48]. During mitosis, WIPI2 ubiquitination and degradation are also potentiated. This is mediated by the Cul4 family of ubiquitin ligase, whose activity is elevated in mitosis due to increased Cul4 neddylation. Importantly, the reduction of autophagy activity in mitosis through WIPI2 ubiquitination is important for the proper progression of mitotic phase, as restoration of WIPI2 during mitosis induces mitotic slippage and cell senescence [49]. Thus, WIPI2 ubiquitination is regulated by nutrient availability and cell cycle to influence on autophagy activity.

The LC3 family protein GABARAP is itself a ubiquitin-like protein. Interestingly, GABARAP can also be modified by K48-linked ubiquitin chain through the activity of centrosome-residing ubiquitin ligase Mib1 [50] (Fig. 1 b). The centriolar satellite protein PCM1, however, binds GABARAP to protect it from Mib1-mediated ubiquitination and degradation. This stabilization of GABARAP allows it trafficking along with PCM1 from centrosome

reservoir to phagophore during starvation, thereby facilitating the formation of GABARAP-positive autophagosome. Thus, the centriolar satellite controls GABARAP ubiquitination and trafficking to regulate autophagosome biogenesis.

The cysteine protease ATG4 is responsible for processing the LC3 to facilitate its lipidation and for deconjugating LC3-II at the final step of autophagy [51, 52]. The membrane-associated ubiquitin ligase RNF5 targets a specific membrane pool of ATG4B for ubiquitination and degradation (Fig. 1b), thereby limiting LC3 processing to restrict autophagy activity in basal condition. Upon starvation or alteration in cell redox states, the binding of RNF5 to ATG4B is attenuated, which contributes to autophagy induction [53].

### The emerging role of ubiquitination in autophagosome maturation

The fusion of autophagosome with lysosome is required for autophagic flux. EPG5, a RAB7A effector, is localized to late-endosome/lysosome and promotes their fusion with autophagosome by binding to LC3 [54]. USP8, a DUB localized to the endocytic compartment, binds EPG5 and removes K63-linked ubiquitin chain from EPG5 [55]. This deubiquitination event enhances EPG5 binding to LC3, thus potentiating autophagic flux to maintain the identity of embryonic stem cell. The E3 ligase responsible for EPG5 K63 ubiquitination remains undetermined. UVRAG, a subunit of class III PI3K complex specifically required for autophagosome maturation, is modified by K29/K33 non-canonical ubiquitin chain by E3 ligase SMURF1 [56] (Fig. 1c). This ubiquitination decreases the binding of UVRAG with its inhibitor Rubicon, thereby enhancing autophagic flux. Hence, these ubiquitination events on EPG5 and UVRAG mainly affect their interaction with other proteins, rather than promoting degradation.

### Ubiquitin controls autophagy termination

Autophagy is a self-limiting process. It is turned on for cell to cope with various stressed conditions. However, once the stressed situation is resolved, autophagy machinery needs to be turned off to avoid excessive degradation. Ubiquitin-mediated degradation appears to be an ideal mechanism for terminating a cellular process, such as autophagy. Indeed, several ubiquitin-based mechanisms are involved in time-dependent or feedback regulation for autophagy termination (Fig. 3).

- The WD40 protein AMBRA1 acts as a component for both class III PI3K complex and Cul4 ubiquitin ligase complex. AMBRA1 undergoes Cul4-dependent self-ubiquitination and degradation. However, at the early stage of autophagy induction, AMBRA1 is transiently dissociated from Cul4, rendering its stabilization. AMBRA1 re-associates with Cul4 at later time points to result in its downregulation. This mechanism contributes in part to autophagy termination as expression of a Cul4-binding deficient AMBRA1 mutant leads to a prolonged autophagy response [30].
- Besides Cul4 ubiquitin ligase, the Cul3 ubiquitin ligase containing KLHL20 as the substrate adaptor is found to play a major role in autophagy termination. Upon autophagy induction, the Cul3-KLHL20 complex specifically targets the autophosphorylated ULK1 for ubiquitination and degradation. Additionally, KLHL20 is recruited to phagophore where it binds and ubiquitinates VPS34 and Beclin-1. Furthermore, other subunits of the ULK1 and VPS34 complexes, such as ATG13 and ATG14, are also degraded after KLHL20-mediated degradation of their partners, even though they are not direct substrates of KLHL20. Thus, KLHL20 participates in feedback regulations for promoting the degradation of multiple autophagy inducing factors after the induction of autophagy. Depletion of KLHL20 or expression of an autophosphorylation-defective ULK1 mutant leads to a prolonged autophagy response and an increased cell death under starvation [57].
- The HECT family ubiquitin ligase NEDD4L and DUB USP20 also participate in autophagy termination. During prolonged starvation, NEDD4L catalyzes the K27 and K29 ubiquitination on ULK1 [58], whereas the interaction between USP20 and ULK1 is attenuated [43]. Both mechanisms lead to downregulation of ULK1 protein level. Thus, multiple E3 ligases and DUB act in concert to limit ULK1 protein abundance, thereby contributing to autophagy termination. Importantly, the ULK1 mRNA is consistently present and its translation is induced when mTOR is reactivated by the release of building blocks from the autolysosome. This mechanism allows the recovery of ULK1 protein level for the next run of autophagy induction [58].

### Ubiquitin signaling in selective autophagy

#### Overview of selective autophagy

Autophagy was originally considered as a nonselective bulk degradation process, but numerous studies have later reported the selective degradation of various cellular organelles or substances via autophagy mechanism, including mitochondria, ER, peroxisome, lipid droplet, ribosome, midbody, nucleus, protein aggregate, and specific pathogens [59]. In theory, selective autophagy should result in a more specific removal of damaged or harmful cellular components and thus could be more important in disease prevention than bulk autophagy. To achieve selectivity, the cargos are often linked to LC3 family proteins directly or indirectly via ubiquitin-dependent or independent mechanisms. This review focuses only on the ubiquitin-dependent selective autophagy. Different from the bulk autophagy where

Chen et al. Journal of Biomedical Science (2019) 26:80

Page 6 of 12

![Diagram](attachment:diagram.png)

Fig. 3 Mechanisms for ubiquitin-mediated autophagy termination. The ULK1 and VPS34 complexes are stable in the induction phase of autophagy. After the execution phase of autophagy, several components of the two complexes are degraded via direct or indirect action of indicated E3 ligases, thereby contributing to autophagy termination.

protein ubiquitination often plays a modulating role, protein ubiquitination in many types of selective autophagy serves as a mark for cargo recognition and a signal for process initiation. Ubiquitinated proteins generated on the surface of cargos are responsible for the recruitment of specific autophagy adaptor proteins (also known as autophagy receptors), such as p62, OPTN, NBR1, NDP52, and TAX1BP1 [60, 61]. Since these autophagy adaptors possess both ubiquitin-binding domain and LC3-interacting region (LIR), they function as bridges to recruit LC3 to the cargos. Certain autophagy adaptor, such as NDP52, also recruits upstream autophagy initiating complex to the cargos [62, 63]. In this way, autophagy machinery generates autophagosome to specifically engulf the cargos. Below, we discuss the role of ubiquitination in the initiation and regulation of several types of selective autophagy (Fig. 4).

**Mitophagy**

The best studied ubiquitin-dependent selective autophagy mechanism is mitophagy, in which the protein kinase PINK1 and E3 ligase Parkin play a key role in building the ubiquitin chains on the outer surface of damaged mitochondria. Upon mitochondrial damage, PINK1 is stabilized on mitochondrial membrane to recruit Parkin [64–66] and phosphorylates the S65 residue on both ubiquitin and the UBL domain of Parkin, which act in concert to activate Parkin on mitochondria [67–69]. Parkin in turn catalyzes the ubiquitination of numerous mitochondrial outer membrane proteins [70, 71]. Recent studies indicate that these ubiquitinated proteins not only facilitate the recruitment of autophagy adaptors but also serve as PINK1 substrates to establish a feedforward mechanism for reinforcing the PINK1-Parkin pathway [68, 72]. Quantitative proteomic study identified numerous mitochondrial proteins whose ubiquitination is dependent on Parkin [73]. Furthermore, multiple ubiquitin chain types, such as K6, K11, K48 and K63 are generated following mitochondrial depolarization [68]. It is generally believed that the identity of the substrates is less important than the density of ubiquitin chains on mitochondria to determine the onset of mitophagy [74]. Consequently, autophagy adaptors are recruited to the damaged mitochondria. CRISPR-mediated knockout analysis on HeLa cells revealed that OPTN, NDP52 and TAX1BP1 are redundantly required for

Chen et al. Journal of Biomedical Science (2019) 26:80

Page 7 of 12

---

### Mitophagy

- **Parkin**
- **PINK1**
- **Ub**
- **TAX1BP1**
- **LC3**
- **NDP52**
- **OPTN**
- **USP30**

### Xenophagy

- **LC3**
- **NBR1**
- **p62**
- **Smurf1**
- **M. tuberculosis**
- **Salmonella**
- **HOIP1**
- **LRSAM1**
- **ARIH**
- **NDP52**
- **LC3**

### Pexophagy

- **PEX2**
- **PMP70 / PEX5**
- **mono Ub**
- **LC3**
- **NBR1**

### Aggrephagy

- **Cul1/FBXO27**
- **p62**
- **LC3**
- **NBR1**
- **Oligomerized p62**

### Lysophagy

- **LC3**

---

**Fig. 4** Ubiquitin-dependent selective autophagy. Summary of the molecular mechanisms of major types of selective autophagy using protein ubiquitination as a mark of the cargo. The E3 ligases and DUB involved in generating or removing the ubiquitin chain and the autophagy adaptors used to link ubiquitinated cargos to LC3 are indicated.

---

mitophagy, with OPTN playing the most prominent role [75]. OPTN further recruits TBK1 to promote mitophagy through a feedback mechanism [76, 77]. Nevertheless, other study indicated the crucial role of p62 in Parkin-dependent autophagy in mouse macrophages and embryonic fibroblasts [78, 79]. It is unclear whether this discrepancy is owing to the difference in the relative abundance of these adaptors in different cell types.

Besides Parkin, mitophagy can be regulated by other factors that influence on the ubiquitination of mitochondrial membrane proteins. USP30, a transmembrane DUB localized on the mitochondrial outer membrane, antagonizes the function of Parkin by removal of ubiquitin chains from mitochondria [80]. Interestingly, USP30 undergoes a Parkin-dependent monoubiquitination and proteasomal degradation, thus establishing a feedforward mechanism for Parkin to promote mitophagy. Additionally, E3 ligases other than Parkin that target mitochondrial fusion and fission machineries [81, 82] can also regulate mitophagy, as damaged mitochondria need to go through a fission process to be enclosed into the autophagosome [83].

### Pexophagy

Peroxisomes are ubiquitous organelles involving in modulation of metabolic responses and redox regulation [84]. In mammals, damaged peroxisomes are removed through ubiquitin-dependent selective autophagy pathway [85]. Consistently, an increase in ubiquitinated proteins on the surface of peroxisomes induces pexophagy. Peroxisome membrane proteins PEX5 and PMP70 are targeted for monoubiquitination under stressed conditions through the peroxisome E3 ligase PEX2 [86]. As to the autophagy adaptors, p62 and NBR1 act in a cooperated fashion to link ubiquitinated peroxisome to autophagic machinery [85, 87].

### Lysophagy

Although bulk autophagy and selective autophagy require the fusion with lysosome for autophagic flux,

damaged lysosome is itself removed by an autophagic process called lysophagy. Lysophagy utilizes a ubiquitin-dependent selective autophagy mechanism, as ubiquitinated proteins, p62, and LC3 are all found on the surface of damaged lysosomes [88, 89]. The damaged lysosome membranes are also decorated with galectin-3 [89], which is presumably due to the exposure of the luminal proteins to the cytosol side following membrane rupture. Recent study indicates that FBXO27, a membrane localized substrate adaptor of Cull1 ubiquitin ligase, catalyzes the ubiquitination of N-glycoproteins exposed to the damaged lysosome, thereby facilitating the recruitment of autophagy adaptor p62 [90].

### Xenophagy

In addition to cellular organelles, ubiquitin-dependent selective autophagy is also exploited to eliminate intracellular pathogens such as *Salmonella*, *Listeria*, and *Mycobacterium*, a process called xenophagy [91]. In the host cells, these pathogens are quickly marked by ubiquitin chains on their surface. Multiple host E3 ligases are reported to ubiquitinate pathogens. For instance, Smurf1 and Parkin are involved in the ubiquitination of *M. tuberculosis* [92, 93]. LRSAM1, ARIH, and HOIPI complex are responsible for *Salmonella* ubiquitination [23, 94, 95]. Of note, the ubiquitin chain types generated by these E3 ligases are different. While LRSAM1 generates K6 and K27 chains, ARIH and HOIP1 form K48 chain and M1 chain, respectively. These different ubiquitin chains are clustered to form distinct foci on bacteria surface [96]. The M1 chain specifically recruits OPTN, whereas the recruitment of p62 and NDP52 to bacteria is independent of M1 chain, demonstrating their non-redundant functions [97]. In addition to inducing xenophagy, the M1 chain on bacteria activates NF-kB pathway to promote proinflammatory cytokine secretion, thereby inhibiting bacteria proliferation [96, 97].

### Aggrephagy

Aggrephagy is induced in response to various proteotoxic conditions, such as inhibition of proteasome or chaperons and interference with productive translation, in which aggregates of ubiquitinated proteins are observed [98]. Formation of such aggregates requires p62 [99]. Recent studies indicate that p62 drives the aggregate formation via a process called liquid-liquid phase separation [61, 100]. In addition to the ubiquitin binding domain (UBA), p62 contains a oligomerization domain (PB1). Oligomerization of p62 allows a high-avidity binding of ubiquitinated proteins via UBA domain and finally condenses the ubiquitinated proteins into larger structures. Subsequently, P62 tethers LC3 to the condensates through its LIR to facilitate a selective sequestration of ubiquitin condensates to the autophagosome. Other

autophagy adaptor, such as NBR1, can also contribute to the condensation by interacting with p62 [101].

Since ubiquitinated proteins can also be targeted to undergo proteasomal degradation, one intriguing question is how to distinguish the autophagy fate from proteasome fate. Although p62-mediated condensation may be a determining factor to direct ubiquitinated proteins to the autophagy pathway, it is worth noting that p62 can also function as a direct adaptor to recruit ubiquitinated proteins to the proteasome in cytosol or nucleus [102, 103]. Another possibility for determining the fate of ubiquitinated protein is the quality of ubiquitin chains. It is thought that K48-ubiquitinated proteins are degraded by proteasome, whereas K63 chain modified proteins are substrates of aggrephagy. However, M1, K63, and K48 chains can all trigger phase separation in vitro via binding to p62, albeit with a lower efficiency than the K48 chain [61, 100]. Perhaps the nature of aggrephagy substrates do not have much difference from those of the proteasome substrates and, rather, the high concentration of ubiquitin chains determines the aggrephagy fate by favoring a p62-mediated phase separation [101].

#### Ubiquitin-mediated autophagy regulation in diseases

##### Neurodegenerative diseases

There is increasing evidence for the association of autophagy dysfunction with various neurodegenerative diseases, such as Alzheimer’s disease, tauopathies, Parkinson’s disease (PD), polyglutamine disorders, and amyotrophic lateral sclerosis [104]. The most well-known neurodegenerative disease associated with defects in ubiquitin-mediated autophagy is PD, which is the second most common late-onset neurodegenerative disease resulted from the loss of dopaminergic neurons in the substantia nigra pars compacta. Mutations in genes encoding either PINK1 or Parkin are associated with autosomal recessive forms of PD [105]. Mice deficient in either Parkin or PINK1 exhibit mitochondrial impairments, but most of them cannot recapitulate the prime features of human PD, that is, loss of dopaminergic neurons [106, 107]. A recent study generated by Parkin homozygous knockout in the background of mice with the expression of a proof-reading defective mtDNA polymerase (called mutator mice). The combination of Parkin knockout and mtDNA mutation leads to the loss of dopaminergic neurons selectively in the substantia nigra and motor defect [108]. This genetic evidence, in conjunction with the mitochondrial dysfunction found in brain and other organs of PD patients [106], point out the importance of mitophagy in PD etiology.

Another type of neurodegenerative disorder is caused by mutant proteins with the expansion of continuous stretches of glutamine (called polyQ repeats), including Huntington’s disease, spinocerebellar ataxia (SCA), and

spinal and bulbar muscular atrophy [104]. A recent study uncovered a link of ubiquitin-mediated autophagy regulation to various polyQ diseases. Ataxin 3 is a polyQ-containing DUB and its polyQ expansion is associated with SCA type 3, in which neurodegeneration occurs in the striatum and cerebellum [109]. Interestingly, the normal function of ataxin 3 is to remove the polyubiquitin chain from Beclin-1, leading to its stabilization [38]. With this function, ataxin 3 is required for starvation-induced autophagy. Importantly, several proteins with expanded polyQ repeats, including ataxin 3 itself, can compete with ataxin 3 for binding Beclin-1, in a polyQ length-dependent fashion. Furthermore, although ataxin 3 with expanded polyQ repeats elicits higher binding affinity to Beclin-1, it is defective in removing ubiquitin chain from Beclin-1. Thus, these findings identify a link of ataxin 3 to autophagy regulation and, more importantly, suggest that impairment of Beclin-1-mediated autophagy accounts for one mechanism of polyQ repeat-associated neurodegenerative diseases.

**Infectious diseases and inflammation**

As described above, ubiquitin serves as a tag to facilitate the autophagic degradation of intracellular pathogens (xenophagy) and a number of ubiquitin E3 ligases are involved in the addition of such tag. Since autophagy core machinery is also required for the xenophagy process, regulators that affect ubiquitin-dependent turnover of autophagic core factors could also control xenophagy. For instance, RNF216, which targets Beclin-1 for ubiquitination and degradation, promotes *Listeria monocytogenes* proliferation and distribution in cell and mouse models [32]. Nevertheless, it should be noted that the bulk autophagy could elicit housekeeping function to restrict inflammation, thereby favoring pathogen survival [91]. The balance between selective autophagy and anti-inflammation could determine the outcome of infection and immunological functions. One example for ubiquitination-mediated balance of anti-infection arm and anti-inflammation arm lies in USP19-depedent Beclin-1 deubiquitination [39]. On one hand, this deubiquitination stabilizes Beclin-1 to favor autophagy-dependent pathogen clearance. On the other hand, the stabilized Beclin-1 binds to the CARD domain of MAVS to prevent MAVS-RIG-I association, thereby inhibiting type I interferon production and anti-viral immunity.

Liver disease, metabolic syndromes and cancer

Autophagy is important in controlling hepatocyte lipid metabolism to maintain normal liver functions [110]. Autophagy deficiency by ATG7 knockout aggravates liver steatosis induced by high fat diet and promotes the development of liver adenoma [111]. Conversely, liver steatosis impairs autophagy through ATG7 downregulation [112]. One important function of autophagy to regulate lipid metabolism is the turnover of lipid droplets via a selective autophagy process called lipophagy [111]. Similar to other selective autophagy processes, lipophagy requires certain core autophagic factors. A recent study reveals an inhibitory role of HUWE1-mediated WIPI2 degradation in lipid droplet turnover in the liver, leading to the accumulation of liver neural lipids [48]. Besides liver disease, ubiquitin-mediated autophagy regulation is implicated in other metabolic syndromes. For instance, failure of autophagy termination by KLHL20 deficiency potentiates muscle atrophy in diabetes mouse model [57].

Autophagy plays complex roles in cancer, which may depend on the different stages of cancer development. In the tumor initiating stage, autophagy suppresses carcinogenesis. However, once tumor is formed, tumor cells exploit the autophagic process for them to survive in the harsh environments [17]. The impact of ubiquitin-mediated autophagy regulation on tumor formation and progression is poorly studied. A recent study reported that the Smurf1-induced UVRAG ubiquitination promotes not only autophagosome maturation but hepatocellular carcinoma (HCC) growth [56]. Furthermore, phosphorylation of UVRAG at S522, which disrupts Smurf1 binding, correlates with poor survival of HCC patients. These findings support a tumor suppressive role of autophagy in HCC.

**Conclusion and perspectives**

In this review, we discussed the impact of protein ubiquitination in autophagy regulation. Protein ubiquitination serves as an ‘eat me’ signal for many types of selective autophagy by recruiting autophagic adaptors and subsequently the core autophagic proteins. In contrast to the “signaling” role of ubiquitination in selective autophagy, protein ubiquitination mainly plays modulating role in almost every step of bulk autophagy. The initiation and nucleation steps of autophagosome formation are most prevalently regulated by ubiquitination, meaning that ubiquitination controls the onset of autophagic process in response to various stressed conditions. Nevertheless, later steps of autophagosome formation and autophagosome maturation are also subjected to ubiquitin-mediated regulation. Furthermore, ubiquitin-mediated protein turnover has been used as a prime mechanism for autophagy termination under prolonged stress conditions, thereby preventing the detrimental effect of excessive autophagic degradation. The pleiotropic role of protein ubiquitination in autophagy regulation highlights the tight crosstalk between the two major cellular degradation machineries. Dysregulation of ubiquitin-mediated autophagy process has been implicated in many disease states, such as neurodegeneration, infectious diseases, liver diseases and

metabolic syndromes. With the important role of autophagy in maintaining normal physiology and homeostasis, it is expected to uncover further linkages between dysregulation of ubiquitin-mediated autophagy pathways and various human diseases, especially for age-related diseases. In this regard, targeting of these pathways by modulating the activity of E3 ligase or DUB could be exploited as a strategy for disease intervention and has been an area receiving considerable attention. For example, the small molecular inhibitor of USP10 and USP13, called spautin-1, is capable of antagonizing the ubiquitination and degradation of Beclin-1 and p53, two tumor suppressor proteins, and therefore is a promising anti-cancer agent [37]. In the future, an improved understanding of how ubiquitin-mediated autophagy regulation contributes to the pathology of human diseases and the development of less toxic and more specific agents will benefit more patients.

### Abbreviations
ATG: Autophagy-related; DUB: Deubiquitinating enzymes; LIR: LC3-interacting region; PD: Parkinson's disease; PE: Phosphatidylethanolamine; SCA: Spinocerebellar ataxia; TLR: Toll-like receptor; UBA: Ubiquitin binding domain; UPS: Ubiquitin-proteasome system

### Acknowledgements
Not applicable

### Authors' contributions
All authors collected and reviewed literatures and wrote the manuscript. Y-H C and T-Y H designed and illustrated figures. All authors read and approved the final manuscript.

### Funding
This work was supported by MOST Frontier Grant MOST106–2321-B-001-009 and an intramural fund from Institute of Biological Chemistry, Academia Sinica.

### Availability of data and materials
Not applicable

### Ethics approval and consent to participate
Not applicable

### Consent for publication
Not applicable

### Competing interests
The authors declare that they have no competing interests.

Received: 31 May 2019 Accepted: 1 October 2019  
Published online: 21 October 2019

### References
1. Groll M, Huber R. Substrate access and processing by the 20S proteasome core particle. Int J Biochem Cell Biol. 2003;35:606–16.
2. Klionsky DJ. Autophagy: from phenomenology to molecular understanding in less than a decade. Nat Rev Mol Cell Biol. 2007;8:931–7.
3. Ji CH, Kwon YT. Crosstalk and interplay between the ubiquitin-proteasome system and autophagy. Mol Cell. 2017;40:441–9.
4. Kocaturk NM, Gozuacik D. Crosstalk between mammalian autophagy and the ubiquitin-proteasome system. Front Cell Dev Biol. 2018;6:128.
5. Dikic I. Proteasomal and Autophagic degradation systems. Annu Rev Biochem. 2017;86:193–224.
6. Hershko A, Ciechanover A. The ubiquitin system. Annu Rev Biochem. 1998;67:425–79.

7. Clague MJ, Barsukov I, Coulson JM, Liu H, Rigden DJ, Urbe S. Deubiquitylases from genes to organism. Physiol Rev. 2013;93:1289–315.
8. Behrends C, Harper JW. Constructing and decoding unconventional ubiquitin chains. Nat Struct Mol Biol. 2011;18:520–8.
9. Kulathu Y, Komander D. Atypical ubiquitylation - the unexplored world of polyubiquitin beyond Lys48 and Lys63 linkages. Nat Rev Mol Cell Biol. 2012;13:508–23.
10. Yau R, Rape M. The increasing complexity of the ubiquitin code. Nat Cell Biol. 2016;18:579–86.
11. Ohtake F, Saeki Y, Ishido S, Kanno J, Tanaka K. The K48-K63 branched ubiquitin chain regulates NF-kappaB signaling. Mol Cell. 2016;64:251–66.
12. Ohtake F, Tsuchiya H, Saeki Y, Tanaka K. K63 ubiquitylation triggers proteasomal degradation by seeding branched ubiquitin chains. Proc Natl Acad Sci U S A. 2018;115:E1401–8.
13. Yau RG, Doerner K, Castellanos ER, Haakonsen DL, Werner A, Wang N, et al. Assembly and function of heterotypic ubiquitin chains in cell-cycle and protein quality control. Cell. 2017;171:918–33e20.
14. Mizushima N. A brief history of autophagy from cell biology to physiology and disease. Nat Cell Biol. 2018;20:521–7.
15. Ohsumi Y. Historical landmarks of autophagy research. Cell Res. 2014;24:9–23.
16. Dikic I, Elazar Z. Mechanism and medical implications of mammalian autophagy. Nat Rev Mol Cell Biol. 2018;19:349–64.
17. Jiang P, Mizushima N. Autophagy and human diseases. Cell Res. 2014;24:69–79.
18. Murrow L, Debnath J. Autophagy as a stress-response and quality-control mechanism: implications for cell injury and human disease. Annu Rev Pathol. 2013;8:105–37.
19. Wirth M, Joachim J, Tooze SA. Autophagosome formation--the role of ULK1 and Beclin1-PI3KC3 complexes in setting the stage. Semin Cancer Biol. 2013;23:301–9.
20. Wong PM, Puente C, Ganley IG, Jiang X. The ULK1 complex: sensing nutrient signals for autophagy activation. Autophagy. 2013;9:124–37.
21. Kihara A, Noda T, Ishihara N, Ohsumi Y. Two distinct Vps34 phosphatidylinositol 3-kinase complexes function in autophagy and carboxypeptidase Y sorting in Saccharomyces cerevisiae. J Cell Biol. 2001;152:519–30.
22. Russell RC, Yuan HX, Guan KL. Autophagy regulation by nutrient signaling. Cell Res. 2014;24:42–57.
23. Noda T. Autophagy in the context of the cellular membrane-trafficking system: the enigma of Atg9 vesicles. Biochem Soc Trans. 2017;45:1323–31.
24. Gomez-Sanchez R, Rose J, Guimaraes R, Mari M, Papinski D, Rieter E, et al. Atg9 establishes Atg2-dependent contact sites between the endoplasmic reticulum and phagophores. J Cell Biol. 2018;217:2743–63.
25. Nakatogawa H. Two ubiquitin-like conjugation systems that mediate membrane formation during autophagy. Essays Biochem. 2013;55:39–50.
26. Zhao YG, Zhang H. Autophagosome maturation: an epic journey from the ER to lysosomes. J Cell Biol. 2019;218:757–70.
27. Nazio F, Strappazzon F, Antonioli M, Bielli P, Cianfanelli V, Bordi M, et al. mTOR inhibits autophagy by controlling ULK1 ubiquitylation, self-association and function through AMBRA1 and TRAF6. Nat Cell Biol. 2013;15:406–16.
28. Shi CS, Kehrl JH. TRAF6 and A20 regulate lysine 63-linked ubiquitination of Beclin-1 to control TLR4-induced autophagy. Sci Signal. 2010;3:ra42.
29. Xia P, Wang S, Du Y, Zhao Z, Shi L, Sun L, et al. WASH inhibits autophagy through suppression of Beclin 1 ubiquitination. EMBO J. 2013;32:2685–96.
30. Antonioli M, Albiero F, Nazio F, Vescovo T, Perdomo AB, Corazzari M, et al. AMBRA1 interplay with cullin E3 ubiquitin ligases regulates autophagy dynamics. Dev Cell. 2014;31:734–46.
31. Platta HW, Abrahamsen H, Thoresen SB, Stenmark H. Nedd4-dependent lysine-11-linked polyubiquitination of the tumour suppressor Beclin 1. Biochem J. 2012;441:399–406.
32. Xu C, Feng K, Zhao X, Huang S, Cheng Y, Qian L, et al. Regulation of autophagy by E3 ubiquitin ligase RNF216 through BECN1 ubiquitination. Autophagy. 2014;10:2239–50.
33. Xia P, Wang S, Huang G, Du Y, Zhu P, Li M, et al. RNF2 is recruited by WASH to ubiquitinate AMBRA1 leading to downregulation of autophagy. Cell Res. 2014;24:943–58.
34. Xiao J, Zhang T, Xu D, Wang H, Cai Y, Jin T, et al. FBXL20-mediated Vps34 ubiquitination as a p53 controlled checkpoint in regulating autophagy and receptor degradation. Genes Dev. 2015;29:184–96.

Chen et al. Journal of Biomedical Science (2019) 26:80

35. Zhang T, Dong K, Liang W, Xu D, Xia H, Geng J, et al. G-protein-coupled receptors regulate autophagy by ZBTB16-mediated ubiquitination and proteasomal degradation of Atg14L. Elife. 2015;4:e06734.
36. Xu D, Shan B, Sun H, Xiao J, Zhu K, Xie X, et al. USP14 regulates autophagy by suppressing K63 ubiquitination of Beclin 1. Genes Dev. 2016;30:1718–30.
37. Liu J, Xia H, Kim M, Xu L, Li Y, Zhang L, et al. Beclin 1 controls the levels of p53 by regulating the deubiquitination activity of USP10 and USP13. Cell. 2011;147:223–34.
38. Ashkenazi A, Bento CF, Ricketts T, Vicinanza M, Siddiqi F, Pavel M, et al. Polyglutamine tracts regulate beclin 1-dependent autophagy. Nature. 2017; 545:108–11.
39. Jin S, Tian S, Chen Y, Zhang C, Xie W, Xia X, et al. USP19 modulates autophagy and antiviral immune responses by deubiquitinating Beclin-1. EMBO J. 2016;35:866–80.
40. Simicek M, Lievens S, Laga M, Guzenko D, Aushev VN, Kalev P, et al. The deubiquitylase USP33 discriminates between RALB functions in autophagy and innate immune response. Nat Cell Biol. 2013;15:1220–30.
41. Bodemann BO, Orvedahl A, Cheng T, Ram RR, Ou YH, Formstecher E, et al. RalB and the exocyst mediate the cellular starvation response by direct activation of autophagosome assembly. Cell. 2011;144:253–67.
42. Raimondi M, Cesselli D, Di Loreto C, La Marra F, Schneider C, Demarchi F. USP1 (ubiquitin specific peptidase 1) targets ULK1 and regulates its cellular compartmentalization and autophagy. Autophagy. 2019;15:613–30.
43. Kim JH, Seo D, Kim SJ, Choi DW, Park JS, Ha J, et al. The deubiquitinating enzyme USP20 stabilizes ULK1 and promotes autophagy initiation. EMBO Rep. 2018;19.
44. Dooley HC, Razi M, Polson HE, Girardin SE, Wilson MI, Tooze SA. WIPI2 links LC3 conjugation with PI3P, autophagosome formation, and pathogen clearance by recruiting Atg12-5-16L1. Mol Cell. 2014;55:238–52.
45. Polson HE, de Lartigue J, Rigden DJ, Reedijk M, Urbe S, Clague MJ, et al. Mammalian Atg18 (WIPI2) localizes to omegasome-anchored phagophores and positively regulates LC3 lipidation. Autophagy. 2010;6:506–22.
46. Scrivo A, Codogno P, Bomont P. Gigaxonin E3 ligase governs ATG16L1 turnover to control autophagosome production. Nat Commun. 2019;10:780.
47. Kang JJ, Liu IY, Wang MB, Srivatsan ES. A review of gigaxonin mutations in giant axonal neuropathy (GAN) and cancer. Hum Genet. 2016;135:675–84.
48. Wan W, You Z, Zhou L, Xu Y, Peng C, Zhou T, et al. mTORC1-regulated and HUWE1-mediated WIPI2 degradation controls autophagy flux. Mol Cell. 2018;72:303–15e6.
49. Lu G, Yi J, Gubas A, Wang YT, Wu Y, Ren Y, et al. Suppression of autophagy during mitosis via CUL4-RING ubiquitin ligases-mediated WIPI2 polyubiquitination and proteasomal degradation. Autophagy. 2019;15:1917–34.
50. Joachim J, Razi M, Judith D, Wirth M, Calamita E, Encheva V, et al. Centriolar satellites control GABARAP Ubiquitination and GABARAP-mediated autophagy. Curr Biol. 2017;27:2123–36e7.
51. Tanida I, Sou YS, Ezaki J, Minematsu-Ikeguchi N, Ueno T, Kominami E. HsAtg4B/HsApg4B/autophagin-1 cleaves the carboxyl termini of three human Atg8 homologues and delipidates microtubule-associated protein light chain 3- and GABAA receptor-associated protein-phospholipid conjugates. J Biol Chem. 2004;279:36268–76.
52. Tanida I, Ueno T, Kominami E. Human light chain 3/MAP1LC3B is cleaved at its carboxyl-terminal Met121 to expose Gly120 for lipidation and targeting to autophagosomal membranes. J Biol Chem. 2004;279:47704–10.
53. Kuang E, Okumura CY, Sheffy-Levin S, Varsano T, Shu VC, Qi J, et al. Regulation of ATG4B stability by RNF5 limits basal levels of autophagy and influences susceptibility to bacterial infection. PLoS Genet. 2012;8:e1003007.
54. Wang Z, Miao G, Xue X, Guo X, Yuan C, Wang Z, et al. The Vici syndrome protein EPG5 is a Rab7 effector that determines the fusion specificity of Autophagosomes with late endosomes/lysosomes. Mol Cell. 2016;63:781–95.
55. Gu H, Shi X, Liu C, Wang C, Sui N, Zhao Y, et al. USP8 maintains embryonic stem cell stemness via deubiquitination of EPG5. Nat Commun. 2019;10:1465.
56. Feng X, Jia Y, Zhang Y, Ma F, Zhu Y, Hong X, et al. Ubiquitination of UVRAG by SMURF1 promotes autophagosome maturation and inhibits hepatocellular carcinoma growth. Autophagy. 2019;15:1130–49.
57. Liu CC, Lin YC, Chen YH, Chen CM, Pang LY, Chen HA, et al. Cul3-KLHL20 ubiquitin ligase governs the turnover of ULK1 and VPS34 complexes to control autophagy termination. Mol Cell. 2016;61:84–97.
58. Nazio F, Carinci M, Valacca C, Bielli P, Strappazzon F, Antonioli M, et al. Fine-tuning of ULK1 mRNA and protein levels is required for autophagy oscillation. J Cell Biol. 2016;215:841–56.

59. Khaminets A, Behl C, Dikic I. Ubiquitin-dependent and independent signals in selective autophagy. Trends Cell Biol. 2016;26:6–16.
60. Stolz A, Ernst A, Dikic I. Cargo recognition and trafficking in selective autophagy. Nat Cell Biol. 2014;16:495–501.
61. Zaffagnini G, Martens S. Mechanisms of selective autophagy. J Mol Biol. 2016;428:1714–24.
62. Ravenhill BJ, Boyle KB, von Muhlinen N, Ellison CJ, Masson GR, Otten EG, et al. The cargo receptor NDP52 initiates selective autophagy by recruiting the ULK complex to cytosol-invading Bacteria. Mol Cell. 2019;74(2):320–9.
63. Vargas JNS, Wang C, Bunker E, Hao L, Maric D, Schiavo G, et al. Spatiotemporal control of ULK1 activation by NDP52 and TBK1 during selective autophagy. Mol Cell. 2019;74(2):347–62.
64. Geisler S, Holmstrom KM, Treis A, Skujat D, Weber SS, Fiesel FC, et al. The PINK1/Parkin-mediated mitophagy is compromised by PD-associated mutations. Autophagy. 2010;6:871–8.
65. Matsuda N, Sato S, Shiba K, Okatsu K, Saisho K, Gautier CA, et al. PINK1 stabilized by mitochondrial depolarization recruits Parkin to damaged mitochondria and activates latent Parkin for mitophagy. J Cell Biol. 2010;189:211–21.
66. Vives-Bauza C, Zhou C, Huang Y, Cui M, de Vries RL, Kim J, et al. PINK1-dependent recruitment of Parkin to mitochondria in mitophagy. Proc Natl Acad Sci USA. 2010;107:378–83.
67. Kondapalli C, Kazlauskaite A, Zhang N, Woodroof HI, Campbell DG, Gourlay R, et al. PINK1 is activated by mitochondrial membrane potential depolarization and stimulates Parkin E3 ligase activity by phosphorylating serine 65. Open Biol. 2012;2:120080.
68. Ordureau A, Sarraf SA, Duda DM, Heo JM, Jedrychowski MP, Sviderskiy VO, et al. Quantitative proteomics reveal a feedforward mechanism for mitochondrial PARKIN translocation and ubiquitin chain synthesis. Mol Cell. 2014;56:360–75.
69. Wauer T, Simicek M, Schubert A, Komander D. Mechanism of phospho-ubiquitin-induced PARKIN activation. Nature. 2015;524:370–4.
70. Chan NC, Salazar AM, Pham AH, Sweredoski MJ, Kolawa NJ, Graham RL, et al. Broad activation of the ubiquitin-proteasome system by Parkin is critical for mitophagy. Hum Mol Genet. 2011;20:1726–37.
71. Geisler S, Holmstrom KM, Skujat D, Fiesel FC, Rothfuss OC, Kahle PJ, et al. PINK1/Parkin-mediated mitophagy is dependent on VDAC1 and p62/SQSTM1. Nat Cell Biol. 2010;12:119–31.
72. Okatsu K, Koyano F, Kimura M, Kosako H, Saeki Y, Tanaka K, et al. Phosphorylated ubiquitin chain is the genuine Parkin receptor. J Cell Biol. 2015;209:111–28.
73. Sarraf SA, Raman M, Guarani-Pereira V, Sowa ME, Huttlin EL, Gygi SP, et al. Landscape of the PARKIN-dependent ubiquitylome in response to mitochondrial depolarization. Nature. 2013;496:372–6.
74. Harper JW, Ordureau A, Heo JM. Building and decoding ubiquitin chains for mitophagy. Nat Rev Mol Cell Biol. 2018;19:93–108.
75. Lazarou M, Sliter DA, Kane LA, Sarraf SA, Wang C, Burman JL, et al. The ubiquitin kinase PINK1 recruits autophagy receptors to induce mitophagy. Nature. 2015;524:309–14.
76. Heo JM, Ordureau A, Paulo JA, Rinehart J, Harper JW. The PINK1-PARKIN mitochondrial Ubiquitylation pathway drives a program of OPTN/NDP52 recruitment and TBK1 activation to promote Mitophagy. Mol Cell. 2015;60:7–20.
77. Richter B, Sliter DA, Herhaus L, Stolz A, Wang C, Beli P, et al. Phosphorylation of OPTN by TBK1 enhances its binding to Ub chains and promotes selective autophagy of damaged mitochondria. Proc Natl Acad Sci USA. 2016;113: 4039–44.
78. Rojansky R, Cha MY, Chan DC. Elimination of paternal mitochondria in mouse embryos occurs through autophagic degradation dependent on PARKIN and MUL1. Elife. 2016;5.
79. Zhong Z, Umemura A, Sanchez-Lopez E, Liang S, Shalapour S, Wong J, et al. NF-kappaB restricts Inflammasome activation via elimination of damaged mitochondria. Cell. 2016;164:896–910.
80. Bingol B, Tea JS, Phu L, Reichelt M, Bakalariski CE, Song Q, et al. The mitochondrial deubiquitinase USP30 opposes parkin-mediated mitophagy. Nature. 2014;510:370–5.
81. Nakamura N, Kimura Y, Tokuda M, Honda S, Hirose S. MARCH-V is a novel mitofusin 2- and Drp1-binding protein able to change mitochondrial morphology. EMBO Rep. 2006;7:1019–22.
82. Yonashiro R, Ishido S, Kyo S, Fukuda T, Goto E, Matsuki Y, et al. A novel mitochondrial ubiquitin ligase plays a critical role in mitochondrial dynamics. EMBO J. 2006;25:3618–26.

Chen et al. Journal of Biomedical Science (2019) 26:80

---

83. Campello S, Strappazzon F, Cecconi F. Mitochondrial dismissal in mammals, from protein degradation to mitophagy. Biochim Biophys Acta. 1837; 2014: 451–60.

84. Wanders RJ, Waterham HR, Ferdinandusse S. Metabolic interplay between peroxisomes and other subcellular organelles including mitochondria and the endoplasmic reticulum. Front Cell Dev Biol. 2015; 3: 83.

85. Kim PK, Hailey DW, Mullen RT, Lippincott-Schwartz J. Ubiquitin signals autophagic degradation of cytosolic proteins and peroxisomes. Proc Natl Acad Sci U S A. 2008; 105: 20567–74.

86. Sargent G, van Zutphen T, Shatseva T, Zhang L, Di Giovanni V, Bandsma R, et al. PEX2 is the E3 ubiquitin ligase required for pexophagy during starvation. J Cell Biol. 2016; 214: 677–90.

87. Deosaran E, Larsen KB, Hua R, Sargent G, Wang Y, Kim S, et al. NBR1 acts as an autophagy receptor for peroxisomes. J Cell Sci. 2013; 126: 939–52.

88. Hung YH, Chen LM, Yang JY, Yang WY. Spatiotemporally controlled induction of autophagy-mediated lysosome turnover. Nat Commun. 2013; 4: 2111.

89. Maejima I, Takahashi A, Omori H, Kimura T, Takabatake Y, Saitoh T, et al. Autophagy sequesters damaged lysosomes to control lysosomal biogenesis and kidney injury. EMBO J. 2013; 32: 2336–47.

90. Yoshida Y, Yasuda S, Fujita T, Hamasaki M, Murakami A, Kawawaki J, et al. Ubiquitination of exposed glycoproteins by SCF (FBXO27) directs damaged lysosomes for autophagy. Proc Natl Acad Sci U S A. 2017; 114: 8574–9.

91. Sharma V, Verma S, Seranova E, Sarkar S, Kumar D. Selective autophagy and Xenophagy in infection and disease. Front Cell Dev Biol. 2018; 6: 147.

92. Franco LH, Nair VR, Scharn CR, Xavier RJ, Torrealba JR, Shiloh MU, et al. The ubiquitin ligase Smurf1 functions in selective autophagy of Mycobacterium tuberculosis and anti-tuberculous host defense. Cell Host Microbe. 2017; 21: 59–72.

93. Manzanillo PS, Ayres JS, Watson RO, Collins AC, Souza G, Rae CS, et al. The ubiquitin ligase parkin mediates resistance to intracellular pathogens. Nature. 2013; 501: 512–6.

94. Huett A, Heath RJ, Begun J, Sassi SO, Baxt LA, Vyas JM, et al. The LRR and RING domain protein LRSAM1 is an E3 ligase crucial for ubiquitin-dependent autophagy of intracellular Salmonella Typhimurium. Cell Host Microbe. 2012; 12: 778–90.

95. Polajnar M, Dietz MS, Heilemann M, Behrends C. Expanding the host cell ubiquitylation machinery targeting cytosolic Salmonella. EMBO Rep. 2017; 18: 1572–85.

96. van Wijk SJL, Fricke F, Herhaus L, Gupta J, Hotte K, Pampaloni F, et al. Linear ubiquitination of cytosolic Salmonella Typhimurium activates NF-kappaB and restricts bacterial proliferation. Nat Microbiol. 2017; 2: 17066.

97. Noad J, von der Malsburg A, Pathe C, Michel MA, Komander D, Randow F. LUBAC-synthesized linear ubiquitin chains restrict cytosol-invading bacteria by activating autophagy and NF-kappaB. Nat Microbiol. 2017; 2: 17063.

98. Bjorkoy G, Lamark T, Brech A, Outzen H, Perander M, Overvatn A, et al. p62/SQSTM1 forms protein aggregates degraded by autophagy and has a protective effect on huntingtin-induced cell death. J Cell Biol. 2005; 171: 603–14.

99. Demishtein A, Fraiberg M, Berko D, Tirosh B, Elazar Z, Navon A. SQSTM1/p62-mediated autophagy compensates for loss of proteasome polyubiquitin recruiting capacity. Autophagy. 2017; 13: 1697–708.

100. Sun D, Wu R, Zheng J, Li P, Yu L. Polyubiquitin chain-induced p62 phase separation drives autophagic cargo segregation. Cell Res. 2018; 28: 405–15.

101. Zaffagnini G, Savova A, Danieli A, Romanov J, Tremel S, Ebner M, et al. p62 filaments capture and present ubiquitinated cargos for autophagy. EMBO J. 2018; 37.

102. Cohen-Kaplan V, Livneh I, Avni N, Fabre B, Ziv T, Kwon YT, et al. p62- and ubiquitin-dependent stress-induced autophagy of the mammalian 26S proteasome. Proc Natl Acad Sci U S A. 2016; 113: E7490–9.

103. Seibenhener ML, Babu JR, Geetha T, Wong HC, Krishna NR, Wooten MW. Sequestosome 1/p62 is a polyubiquitin chain binding protein involved in ubiquitin proteasome degradation. Mol Cell Biol. 2004; 24: 8055–68.

104. Menzies FM, Fleming A, Caricasole A, Bento CF, Andrews SP, Ashkenazi A, et al. Autophagy and Neurodegeneration: pathogenic mechanisms and therapeutic opportunities. Neuron. 2017; 93: 1015–34.

105. Ryan BJ, Hoek S, Fon EA, Wade-Martins R. Mitochondrial dysfunction and mitophagy in Parkinson's: from familial to sporadic disease. Trends Biochem Sci. 2015; 40: 200–10.

106. Pickrell AM, Youle RJ. The roles of PINK1, parkin, and mitochondrial fidelity in Parkinson's disease. Neuron. 2015; 85: 257–73.

107. Whitworth AJ, Pallanck LJ. PINK1/Parkin mitophagy and neurodegeneration—what do we really know in vivo? Curr Opin Genet Dev. 2017; 44: 47–53.

---

108. Pickrell AM, Huang CH, Kennedy SR, Ordureau A, Sideris DP, Hoekstra JG, et al. Endogenous Parkin preserves dopaminergic Substantia Nigral neurons following mitochondrial DNA mutagenic stress. Neuron. 2015; 87: 371–81.

109. Riess O, Rub U, Pastore A, Bauer P, Schols L. SCA3: neurological features, pathogenesis and animal models. Cerebellum. 2008; 7: 125–37.

110. Czaja MJ, Ding WX, Donohue TM Jr, Friedman SL, Kim JS, Komatsu M, et al. Functions of autophagy in normal and diseased liver. Autophagy. 2013; 9: 1131–58.

111. Singh R, Kaushik S, Wang Y, Xiang Y, Novak I, Komatsu M, et al. Autophagy regulates lipid metabolism. Nature. 2009; 458: 1131–5.

112. Yang L, Li P, Fu S, Calay ES, Hotamisligil GS. Defective hepatic autophagy in obesity promotes ER stress and causes insulin resistance. Cell Metab. 2010; 11: 467–78.

---

### Publisher’s Note
Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

---

**Ready to submit your research? Choose BMC and benefit from:**

- fast, convenient online submission
- thorough peer review by experienced researchers in your field
- rapid publication on acceptance
- support for research data, including large and complex data types
- gold Open Access which fosters wider collaboration and increased citations
- maximum visibility for your research: over 100M website views per year

At BMC, research is always in progress.

[Learn more](https://biomedcentral.com/submissions)
