
Cell

Structural Basis of Poxvirus Transcription: Vaccinia RNA Polymerase Complexes

Graphical Abstract

![Image Description](image_url)

Authors
Clemens Grimm, Hauke S. Hillen, Kristina Bedenk, ..., Aladar A. Szalay, Patrick Cramer, Utz Fischer

Correspondence
pcramer@mpibpc.mpg.de (P.C.), utz.fischer@biozentrum.uni-wuerzburg.de (U.F.)

In Brief
Structural analyses of poxvirus RNA polymerase complexes reveal how the components collaborate to effect transcription with host cell cytoplasm.

Highlights
- Isolation and characterization of two native Vaccinia virus transcription complexes
- 2.8 Å cryo-EM structures of core and complete Vaccinia RNA polymerase (vRNAP)
- The multisubunit core vRNAP resembles host Pol II with Vaccinia-specific features
- Complete vRNAP integrates activities required for Vaccinia early transcription

Grimm et al., 2019, Cell 179, 1537–1550  
December 12, 2019 © 2019 Elsevier Inc.  
https://doi.org/10.1016/j.cell.2019.11.024

Article

# Structural Basis of Poxvirus Transcription: Vaccinia RNA Polymerase Complexes

Clemens Grimm,$^{1,9}$ Hauke S. Hillen,$^{2,9}$ Kristina Bedenk,$^{1}$ Julia Bartuli,$^{1}$ Simon Neyer,$^{2}$ Qian Zhang,$^{5}$ Alexander Hüttenhofer,$^{6}$ Matthias Erlacher,$^{6}$ Christian Dienemann,$^{2}$ Andreas Schlosser,$^{7}$ Henning Urlaub,$^{3,4}$ Bettina Böttcher,$^{1,7}$ Aladar A. Szalay,$^{8,5}$ Patrick Cramer,$^{2,*}$ and Utz Fischer$^{1,5,8,10,*}$

$^{1}$Department of Biochemistry and Cancer Therapy Research Center (CTRC), Theodor Boveri-Institute, University of Würzburg, Am Hubland, 97074 Würzburg, Germany  
$^{2}$Department of Molecular Biology, Max Planck Institute for Biophysical Chemistry, Am Fassberg 11, 37077 Göttingen, Germany  
$^{3}$Bioanalytical Mass Spectrometry Group, Max Planck Institute for Biophysical Chemistry, Am Fassberg 11, 37077 Göttingen, Germany  
$^{4}$Bioanalytics, Institute for Clinical Chemistry, University Medical Center Göttingen, Robert Koch Straße 40, 37075 Göttingen, Germany  
$^{5}$Genelux Corporation, 3030 Bunker Hill Street, San Diego, CA 92109, USA  
$^{6}$Division of Genomics and RNomics, Biocenter, Medical University of Innsbruck, 6020 Innsbruck, Austria  
$^{7}$Rudolf-Virchow-Center, University of Würzburg, Josef-Schneider-Straße 4, 97080 Würzburg, Germany  
$^{8}$Helmholtz Institute for RNA-based Infection Research (HIRI) and Helmholtz Centre for Infection Research (HZI), 97080 Würzburg, Germany  

$^{9}$These authors contributed equally  
$^{10}$Lead Contact  

*Correspondence: pcramer@mpibpc.mpg.de (P.C.), utz.fischer@biozentrum.uni-wuerzburg.de (U.F.)  
https://doi.org/10.1016/j.cell.2019.11.024  

## SUMMARY

expression events were extensively studied for *Vaccinia* virus, a non-pathogenic prototype of the *Poxviridae* family. These studies uncovered a virus-encoded multisubunit RNA polymerase and an array of associated factors that ensure the expression of the viral genome ([Broyles, 2003; Kates and McAuslan, 1967; Munyon et al., 1967]).

Upon infection, *Vaccinia* virus enters the cell via micropinocytosis and becomes uncoated ([Chi and Liu, 2012; Moss, 2012]). Whereas the viral genome is silent in these initial events, all subsequent steps of the replication cycle are dependent on viral transcription and translation processes. Poxviruses coordinate the different processes of DNA replication and virion formation through timing of expression of individual genes grouped into early, intermediate, and late classes ([Baldick and Moss, 1993]). Accordingly, early genes encode factors involved in events that shortly follow infection, such as viral DNA replication and intermediate gene expression, whereas later processes of the infection cycle, such as virion assembly, require the expression of intermediate and late class gene products.

*Vaccinia* RNA polymerase (vRNAP) consists of eight subunits encoded by early viral genes and termed according to their apparent molecular masses Rpo147, Rpo132, Rpo35, Rpo30, Rpo22, Rpo19, Rpo18, and Rpo7 ([Liu et al., 2010]). These subunits show varying degrees of homology to subunits of RNA polymerase II (Pol II), suggesting an evolutionary relationship with the host transcription apparatus ([Zimmermann et al., 2018]). At the level of amino acid residues, the two largest subunits (i.e., Rpo147 and Rpo132) have been reported to be approximately 20% identical to RPB1 and RPB2 of Pol II, respectively ([Ahn et al., 1990; Ahn and Moss, 1992; Amegadzie et al., 1991; Broyles and Moss, 1986; Knutson and Broyles, 2008]). To date, there is no structural information on vRNAPs and their complexes.

vRNAP has the catalytic potential to synthesize RNA in a DNA-dependent manner. However, *in vivo* it requires additional factors in order to become specifically directed to viral early, intermediate, and late class genes. Early transcription has been

## INTRODUCTION

The eukaryotic nucleus contains the machineries for DNA replication and gene transcription. Many viruses rely on factors of the host cell for their replication and transcription and therefore require at least a transient nuclear phase to ensure viral propagation. A remarkable exception among eukaryotic DNA viruses are the members of the *Poxviridae* family, whose replication and transcription are confined to the cytoplasm ([Moss, 2013]). These processes require virus-encoded factors for the production of mature mRNAs from the viral genome. Such cytosolic gene

Figure 1. Purification and Characterization of Vaccinia Virus RNA Polymerase Complexes

(A) Purification of Rpo132 and its associated proteins from GLV-1h439-infected cells using anti-FLAG affinity chromatography. Mock purification was performed from cells infected with untagged GLV-1h68. Specific proteins from the GLV-1h493 elution were resolved on SDS gels and identified by mass spectrometry.

(B) Anti-FLAG eluate from cell extracts infected with GLV-1h439 was separated on a 10%–30% sucrose gradient and proteins visualized by silver staining on SDS-PAGE.

(C) RNA extension assay with a nucleic acid scaffold mimicking an elongation complex transcription bubble.

(D) Transcription assay with a linearized pSB24 template containing a Vaccinia virus early promoter and early gene termination signal.

See also Figure S1.

studied most extensively and shown to require the heterodimeric Vaccinia early transcription factor (VETF), which interacts with early promoters upstream and downstream of the initiation site (Barnes et al., 2015). Together with Rap94, VETF mediates the recruitment of vRNAP to its promoters and its transition into active elongation (Liu et al., 2010). Rap94 has also been proposed to connect vRNAP with VETF and NPH-I to facilitate termination (Zimmermann et al., 2018). Other virus-encoded proteins are used to add a 5′-terminal m7G-cap and a 3′-terminal poly(A)-tail to viral RNAs. They include the heterodimeric Vaccinia termination factor/capping enzyme (VTF/CE), consisting of subunits D1 and D12, and the termination factor NPH-I, which acts together with a poly(A) polymerase to form polyadenylated 3′ ends. Whether these factors are part of defined functional vRNAP complexes is unknown.

Here, we describe the isolation of two distinct vRNAP complexes from human cells infected by Vaccinia virus: the ~500 kDa vRNAP core enzyme and the ~900 kDa complete enzyme with six additional viral proteins plus tRNA-GlnTTG (chr17.trna16-GlnTTG, termed tRNA<sup>Gln</sup>) from the host. We determine the structures of these two complexes by cryoelectron microscopy (cryo-EM). Whereas the core complex represents the active core RNA polymerase, the complete enzyme apparently represents the packaged machinery containing the factors for early gene transcription. The structures reveal similarities and differences between the viral cytoplasmic transcription apparatus and the nuclear RNA polymerase machinery. Our results form the basis for unraveling the molecular mechanisms of poxvirus gene transcription and RNA processing and enabled structure determination of functional vRNAP complexes reported in the accompanying paper (Hillen et al., 2019).

### RESULTS

#### Purification of Vaccinia vRNAP Complexes

We developed a purification strategy for the isolation of vRNAP complexes based on the recombinant Vaccinia virus strain GLV-1h439. This virus is derived from the Vaccinia Lister strain GLV-1h68 and expresses a C-terminally hemagglutinin (HA)/FLAG tagged vRNAP subunit Rpo132 (Figure S1A). GLV-1h439 multiplied at similar rates to the untagged parental GLV-1h68 strain upon infection of HeLa cells, suggesting that the tag on Rpo132 does not interfere with the transcriptional activity and replication of the virus (Figure S1B).

For affinity purification of vRNAP, HeLaS3 cells were infected with GLV-1h439. Extract from infected cells was then subjected to purification on an anti-FLAG column and tagged Rpo132, along with its interacting partners, was eluted with FLAG peptide (Figure S1C). The eluate was separated by gel electrophoresis (Figure 1A) and analyzed by mass spectrometry (Table S2). All known subunits of the vRNAP core enzyme (Table S1), as well as the transcription factor Rap94, the capping enzyme VTF/CE (D1/D12), the termination factor NPH-I, and the early transcription factor subunits VETF-I and VETF-s (A7/D11), were enriched in the GLV-1h439 elution. None of these factors were enriched in a control purification performed with extracts from cells infected

A

Rpo147
Clamp core
Clamp head
Clamp core
Dock
Pore 1
Funnel
Cleft
Foot
Cleft
Jaw
Cleft
Clamp core

Active site

Rpo132
External 1
Protrusion
Lobe
Fork
External 2
Hybrid Binding
Wall
Hybrid Binding
Anchor Clamp Tail

Rpo35
Dimerization
Domain 2
Dimerization
Rpo22
Jaw
Assembly
Rpo19
Tail
Assembly
Rpo18
Tip
CTD
Rpo7
Zink-Tail Bundle

NTD
Insertion
NTD TFIIS-like
Tail

Rap94
NTD
Domain 2
L2
B-Cyclin
L3
CTD
B-Ribbon|B-linker
B-reader

B

Rap94
CTD
Rpo132
Rap94
D2
Rpo18
Rpo30
Bridge helix
Rpo19
Rpo22
Rpo147
Rap94
CTD
Rpo132
Rpo7
Rap94
D2
Rpo18
Rpo35
Active site
Rpo19
Rap94
L2
Rpo147

Front view
Top view

Active site metal A
Zn²⁺

C

Jaw
Funnel
Active site Mg²⁺
Cleft
Pore 1
Bridge helix
Active site
Foot
Clamp head
Clamp core
Linker
Dock
Rpo147
External 2
Lobe
Protrusion
Fork
Anchor
Wall
Clamp
Tail
Rpo132
Domain 2
α4d
α3c
α3a
α3b
NH₂
COOH
Dimerization
Rpo7
COOH
NH₂
Side view
CTD
Tip
Jaw
Assembly
Rpo22
α1a
α3
Assembly
Rpo35

(legend on next page)

Cell 179, 1537–1550, December 12, 2019 1539
with the untagged virus (Figure 1A). This purification also identified the viral core protein E11L and the host tRNA<sup>Gln</sup> (Table S3) as new factors associated with the *Vaccinia* virus transcription apparatus.

### Vaccinia RNAP Complexes Are Functional

When the eluate was analyzed by sucrose gradient centrifugation and mass spectrometry, two major complexes became apparent. The lighter complex contained all subunits of the vRNAP core enzyme including sub-stoichiometric amounts of Rap94 (Figure 1B). Biochemical characterization showed that this complex represents the catalytically active RNA polymerase core enzyme, because it is capable of elongating an RNA primer in vitro (Figure 1C). However, no transcriptional activity was detected on an artificial gene under the control of a fully double-stranded viral promoter (Figures 1D and S1D), confirming that the core enzyme requires additional factors for initiation.

The second, heavier complex contained all subunits of the core enzyme and additionally VTF/CE, NPH-I, VETF-I, VETF-s, E11L, and tRNA<sup>Gln</sup> (Figure 1B). This complex was capable of early promoter-dependent transcription initiation, elongation, and termination at a viral termination signal in vitro (Figures 1C and 1D). Taken together, the first complex represents the catalytically active core vRNAP enzyme, whereas the second complex represents a complete enzyme that comprises core vRNAP and viral transcription and RNA processing factors and is capable of carrying out all steps of the early *Vaccinia* transcription cycle.

### Structure of Vaccinia Core vRNAP

We analyzed the core vRNAP by single-particle cryo-EM and obtained a reconstruction at 2.8 Å resolution (Figures S2A–S2G; Table S4). The high resolution allowed for placement and adjustment of homology models or de novo modeling of all eight subunits. The reconstruction showed additional densities, which were found to stem from Rap94 according to chemical cross-linking (Figure S2H; Table S6). Focused classification and refinement yielded improved maps that allowed us to model two domains of Rap94 on opposite sides of the polymerase. The resulting structure of the vRNAP core enzyme has good stereochemical quality (Table S4) and contains all eight core vRNAP subunits, four structural zinc ions, the catalytic magnesium ion A, and two domains of Rap94.

The structure shows that core vRNAP resembles multisubunit RNA polymerases in eukaryotic cells and, in particular, Pol II (Figure 2; Video S1). Based on structural and sequence homology, we annotated domains in all subunits in accordance to their counterparts in S.cerevisiae Pol II, which serves as a paradigm for eukaryotic multisubunit RNA polymerases (Figures 2, S3,

S4, and S5) (Cramer et al., 2001). The two large subunits, Rpo147 and Rpo132, form two sides of a central cleft that holds the active center, giving vRNAP the typical bilobal appearance of multisubunit RNA polymerases found in all three domains of life (Figure 2B) (Armache et al., 2005). Subunits Rpo35 and Rpo7 form a subassembly on the back of the polymerase body that contacts both large subunits (Figure 2C).

The entry path for the DNA duplex to the cleft is lined by two “jaws” formed by Rpo147 and subunit Rpo22 (Figure 2C). Rpo22 assembles with subunits Rpo19 and Rpo18 on the periphery of the polymerase (Figure 2C). Rpo18 protrudes slightly from the polymerase body, forming a stalk. At its base, Rpo18 is anchored to the polymerase body and Rpo19, which in turn bridges to Rpo22. Rpo30 is only partially visible in the structure and binds with its N-terminal domain on the outside of the enzyme near the “funnel” domain of Rpo147 (Figure 2B). The *Vaccinia*-specific transcription factor Rap94 is likewise only partially visible in the core vRNAP structure, with two of its domains (domain 2 [D2] and C-terminal domain) binding to the periphery of the polymerase on opposite sides of the cleft (Figure 2B).

#### vRNAP Contains a Conserved Core

Seven of the eight core vRNAP subunits show structural homology to subunits found in Pol II, albeit their degree of similarity differs (Figure 3A; Table S1). Therefore, we carried out a structure-based comparison between vRNAP and S. cerevisiae Pol II that provides insight into the functional roles for the individual subunits of vRNAP (Figures 3, S3, S4, and S5) (Armache et al., 2003). The two large subunits Rpo147 and Rpo132, which form the body of the polymerase, are highly similar to their Pol II counterparts Rpb1 and Rpb2, respectively (Figures 3B, S3, and S4). In particular, the active center and nucleic acid-binding regions are structurally conserved. The active site is formed by an invariant DxDxD motif in Rpo147 that binds the catalytic metal ion A (Figures 2 and S3; Video S1) and is flanked by the bridge helix of Rpo147 that traverses the cleft (Figure 2B). However, both Rpo147 and Rpo132 lack several regions and are smaller compared to the yeast counterparts (Figures 3B, S3, and S4).

In all known multisubunit RNA polymerases, the two large subunits are anchored to a dimeric platform at the back of the enzyme, formed by Rpb3 and Rpb11 in the case of Pol II (Cramer et al., 2000, 2001; Engel et al., 2013; Fernández-Tornero et al., 2013; Hirata et al., 2008; Hoffmann et al., 2015; Zhang et al., 1999). The vRNAP subunit Rpo35 combines features of both Rpb3 and Rpb11 in one polypeptide (Figures 2C, 3A, and S5). It contains a Rpb3-like N-terminal part and a C-terminal part that are similar to Rpb11. However, it lacks the zinc-binding motif

---

**Figure 2. Structure of Core Vaccinia RNAP**

(A) Schematic depiction of vRNAP subunits. Functional domains are annotated based on structure-based sequence alignment with S.cerevisiae RNA Pol II (Armache et al., 2005; Cramer et al., 2001). Regions not visible in the core vRNAP structure are shown transparently.

(B) Structure of the core *Vaccinia* RNA polymerase enzyme. The protein is shown in cartoon depiction, with helices depicted as cylinders. Subunits are colored as in (A). The active site metal A and bound structural zinc ions are shown as spheres. See also Figure S2.

(C) Cartoon depiction of *Vaccinia* RNAP subunits with structural details shown. Rpo147 and Rpo132 domains are colored as indicated in (A). The location of the subunits in the enzyme is indicated schematically.

See also Figures S2, S3, S4, and S5 and Video S1.

1540 Cell 179, 1537–1550, December 12, 2019
Figure 3. Comparison of Vaccinia RNA Polymerase to S.cerevisiae Pol II

(A) Comparison of subunit composition between core vRNAP and S.cerevisiae Pol II (PDB: 1WCM) (Armache et al., 2005). The enzymes are depicted in schematic surface representation. Homologous subunits are indicated in the table and colored accordingly.

(B) Detailed comparison of core vRNAP (left) and S.cerevisiae Pol II (right) (PDB: 1WCM) (Armache et al., 2005). The largely conserved core is depicted as schematic surface in gray, and the differing regions are depicted as cartoon. Regions specific to vRNAP are shown in green and regions specific to Pol II in red. Regions located at the back of the enzyme are labeled transparently.
See also Figures S3, S4, and S5.

Vaccinia-Specific Polymerase Periphery

The structure-based comparison also demonstrates that the enzyme surface deviates substantially from that of other multisubunit RNA polymerases (Figure 3B). In particular, vRNAP does not contain counterparts to the Pol II surface subunits Rpb4, Rpb8, Rpb9, and Rpb12 (Figure 3A). Moreover, differences in related subunits of vRNAP and Pol II also map to the surface of the enzymes (Figure 3B). For example, the clamp core domain in the largest subunit is smaller in vRNAP but larger and involved in transcription factor interactions in Pol II (Bernecky et al., 2017; Martinez-Rucobo et al., 2011; Plaschka et al., 2016). Likewise, the jaw and foot domains in the largest subunit Rpo147 are also smaller. Rpo147 also does not possess the long and repetitive C-terminal domain (CTD) found in its Pol II counterpart Rpb1. Instead, it contains a short C-terminal tail (C-tail) (res. 1,259–1,286) (Figure S3), which is mobile in the core vRNAP structure and hence not visible. The second largest subunit, Rpo132, lacks several small regions and contains a few insertions compared to its Pol II counterpart Rpb2. It has an extended C-tail that emerges from the clamp and wraps around the polymerase, traversing across subunit Rpo19 and toward the foot domain of Rpo147 (Figures 2C and S4).

The jaws of vRNAP, formed by Rpo147 and Rpo22, also show unique features. Whereas the C-terminal assembly domain of Rpo22 is highly conserved, its jaw domain adopts a unique fold (Figures 2C and 3B) and lacks the TPSA motif found in its Pol II counterpart Rpb5 that interacts with downstream DNA (Figure S5B) (Bernecky et al., 2016). The opposite side of the jaw, formed by Rpo147, is smaller and adopts a different orientation than in Pol II. Near this domain, the unique viral subunit Rpo30 binds at the rim of the cleft (Figures 2B and 3B). Rpo30 does not have a counterpart in Pol II, but its N-terminal domain (NTD) is located in a similar position on the polymerase as the dissociable Pol II elongation factor TFIIS (Figure 3A), which Rpo30 has been suggested to functionally

and the regions responsible for interactions with Rpb12 and Rpb10 in Pol II (Figure S5A), consistent with the absence of a Rpb12-like subunit in vRNAP. The corresponding location of Rpb12 on vRNAP is instead occupied by a helical insertion in Rpo35. Rpo7 interacts with Rpo35 and closely resembles the Pol II subunit Rpb10, in both structure and location in the enzyme complex (Figures 2C and S5A). The C-terminal tail of Rpo7, however, extends farther, forming additional interactions with Rpo35 and Rpo132. The Rpo35/Rpo7 subassembly therefore represents the viral equivalent to the Rpb3/10/11/12 subassembly in Pol II and the α₂ homodimer in bacterial RNA polymerases (Zhang et al., 1999).

Rpo22 structurally resembles Rpb5 and is located at a similar position (Figures 2B and 3A), as predicted previously (Knutson and Broyles, 2008). Rpo19 is a structural and functional homolog of the Pol II subunit Rpb6. As for the latter, the N-terminal tail of Rpo19 is mobile and hence invisible in our structure (Figure 2A). The regions flanking the conserved assembly domain of Rpo19 (α1a and α3) are unique to the viral enzyme. Also, helix α1a forms a contact to Rpo22 that is not observed between the corresponding Pol II subunits Rpb5 and Rpb6 (Figures 2C and S5B). The foot domain of Rpo147 lacks some regions found in its Pol II counterpart, and this space is partially occupied by the Rpo19 α1a helical insertion (Figure S3). In summary, this detailed comparison of vRNAP to Pol II shows that the enzyme core is largely conserved between vRNAP and other multisubunit polymerases.

resemble based on sequence analysis (Ahn et al., 1990; Hagler and Shuman, 1993).

A prominent unique feature of vRNAP is its one-subunit stalk, formed by Rpo18, which is homologous to the Pol II subunit Rpb7. Eukaryotic nuclear RNA polymerases I, II, and III and archaeal RNA polymerase all contain a heterodimeric stalk (Armache et al., 2005; Engel et al., 2013; Fernández-Tornero et al., 2013; Hirata et al., 2008; Hoffmann et al., 2015). In Pol II, the stalk is comprised of subunits Rpb4 and Rpb7 (Armache et al., 2003) and is involved in multiple protein interactions with transcription factors during different stages of the transcription cycle (Bernecky et al., 2017; Plaschka et al., 2016; Vos et al., 2018). The overall fold of Rpo18 is virtually identical to Rpb7, except for a smaller C-terminal region (Figures 2C and S5B). Rpo18 uses its tip domain to bind the polymerase core with conserved structural elements (Figure S5B). The Rpo18 tip domain may restrict movement of the clamp, as proposed for Rpb7 (Armache et al., 2003). In comparison to the Rpb4-Rpb7 stalk, the C-terminal domain of Rpo18 appears tilted toward the polymerase as it protrudes from the enzyme surface (Figure 3A). In summary, these comparisons suggest that the surface of vRNAP has evolved specialized features, likely to facilitate interactions with virus-specific transcription factors.

**Transcription Factor Rap94 Spans the vRNAP Cleft**

The core vRNAP structure contains the poxvirus-specific transcription factor Rap94 bound to the enzyme periphery (Figure 2B). Rap94 may be involved in the recognition of early viral promoters (Ahn et al., 1994) and transcription termination (Christen et al., 2008). However, no structural information is available for Rap94, and sequence-based homology searches do not detect substantial homology to any known proteins. The two Rap94 domains resolved in the core vRNAP structure occupy distant locations on the polymerase surface on opposite sides of the cleft. One of these Rap94 domains, which we refer to as D2, comprises residues 107–292 and binds to the top of the vRNAP clamp, interacting with both Rpo147 and Rpo132 (Figure 2B; Video S1). It is located close to Rpo18 and may stabilize the stalk in the observed orientation. It consists of a β sheet flanked by helical regions on either side and shows no structural similarity to factors known to interact with the clamp of Pol II. The CTD of Rap94 comprises residues 637–795 and is located at the lobe of Rpo132 (Figure 2B). The CTD contacts the protrusion domain with a β sheet (res. 661–686). The fold of the Rap94 CTD does not resemble known Pol II transcription factors. The two Rap94 domains are connected via extended linkers that wrap around the polymerase like a belt (Figures 2A and 2B). These linkers traverse the binding sites of Pol II subunits that are absent in vRNAP, including the C-ribbon domain of Rpb9 and the zinc-binding motif of Rpb12. The central region of Rap94 (res. 317–587) is not visible in the core vRNAP structure.

**Rap94 Forms a Bridge between vRNAP Core and Additional Factors**

The complete vRNAP structure shows well-defined density for all parts of Rap94, which interacts with bound factors. In addition to the two domains observed in the core vRNAP structure, the NTD (res. 1–94) and the central region (res. 325–580) of Rap94 are well defined. The Rap94 domains are distributed over the entire complex and are connected by extended linker regions (Figures 4A and 5A). Linker 1 (L1; res. 94–107) connects the NTD to D2. Linker 2 (L2; res. 292–325) emerges from D2 next to the Rpo18 stalk and extends toward Rpo19, passing the C-tail of Rpo147 (Figure 5B). It then continues along the polymerase dock domain to the back of vRNAP. On the other side of the cleft, linker 3 (L3; res. 581–637) extends near the wall and protrusion domains of Rpo132, where it traverses the binding site of Rpb12 in Pol II. L3 then extends through a groove formed by the wall and external domains of Rpo132 and to the funnel helices of Rpo147 and the Rap94 CTD (Figure 5C).

The N-terminal region of Rap94 interacts with the C-terminal region of NPH-I. Together, they fold into a domain-like module that contacts VTF/CE and that we termed the “CE connector” (CEC). The CEC forms a wedge between the RNA triphosphatase/guanyltransferase and the methyltransferase domains of VTF/CE, keeping the two domains apart by 10 Å compared to the VTF/CE crystal structure (Figure 5D). A further contact between Rap94 and NPH-I is buttressed by the dimeric E11 core protein (Figures 5E). D2 of Rap94 adapts tRNA<sup>Gln</sup> to the core vRNAP (Figure 5F). In contrast to the situation in the core vRNAP structure, the C-tail of Rpo147 is ordered in the complete vRNAP and adopts an extended structure that tethers VTF/CE (Figure 5B). Thus, Rap94 is highly modular and serves as a scaffold to assemble the complete vRNAP complex.

**Structure of Complete Vaccinia vRNAP**

We next determined the structure of the complete vRNAP that contains additional transcription and RNA processing factors. We collected a cryo-EM dataset from the pooled fractions 15–17 of the gradient shown in Figure 1B, which yielded a reconstruction at 2.8 Å resolution (Figure S6). The core vRNAP model could be unambiguously docked into the reconstruction with mi-

**The Rap94 Central Region Resembles Pol II Initiation Factor TFIIB**

The central region of Rap94 in the complete vRNAP (res. 325–580) is reminiscent of a large portion of the Pol II initiation factor TFIIB (Figure 5G), and we therefore term it “B-homology region.” It comprises a B-ribbon element (res. 325–371), a B-reader hairpin (res. 372–385), a B-linker (res. 386–396), and a B-cyclin domain (res. 397–580). In particular, the zinc ribbon fold and

Figure 4. Structure of the Complete vRNAP Complex

(A) Schematic depiction of the additional Vaccinia transcription factors VTF/CE, VETF-I, E11, and NPH-I contained in the complete vRNAP complex with domains indicated. Rpo30 and Rap94 are also present in the core vRNAP complex.

(B) Overview of the complete vRNAP model, color coding as in (A). vRNAP is shown in gray. The orientation of the view in the left panel is related to the view in in the left panel of Figure 2B by an approximately 30° rotation counter-clockwise around the viewing axis followed by an approximately 30° rotation counter-clockwise vertical rotation. The protein is shown in cartoon depiction, with helices depicted as cylinders.

See also Figure S6.

the zinc binding site in the B-ribbon are well conserved between Rap94 and TFIIB. However, the N-terminal part of the B-ribbon is formed by two unique helices in Rap94 that participate in zinc coordination via H328 instead of a cysteine. The B-linker and B-reader appear smaller compared to their TFIIB counterparts but occupy comparable locations between the dock and clamp domains of the polymerase (Sainsbury et al., 2013). The B-cyclin domain of Rap94 corresponds to the N-terminal cyclin domain of TFIIB with respect to its fold and location. Thus, the B-homology region in Rap94 occupies a similar location as TFIIB in Pol II transcription initiation complexes (Plaschka et al., 2016; Sainsbury et al., 2013), suggesting that Rap94 may function like TFIIB during transcription initiation.

### Subunit Rpo30 Distantly Resembles the Pol II Elongation Factor TFIIS

The structures show that the core vRNAP subunit Rpo30 shares similarities with eukaryotic TFIIS, as suggested based on sequence analysis (Ahn et al., 1990; Hagler and Shuman, 1993). The Rpo30 N-terminal domain (res. 23–130) binds to the rim of the polymerase funnel (Figure 6A) at the location occupied by TFIIS domain II on Pol II (Kettenberger et al., 2003, 2004). Despite their similar location, these domains differ in sequence and structure. In particular, the Rpo30 NTD contains an insertion (res. 50–100) that wraps around the base of the jaw domain and meanders into the cleft toward the trigger loop, a mobile element of the active center (Figure 6A, inset). The NTD of Rpo30 is connected to a linker region that extends to the Rpo147 funnel helices, forming a short single-turn helical segment (Figure 6A).

The CTD of Rpo30 (res. 131–259) shows sequence similarity to domain III of TFIIS, a zinc ribbon that inserts into the polymerase pore to reach the active site of the enzyme (Figure S7A) (Kettenberger et al., 2003). This domain is mobile in both of our structures, but it can likely insert into the polymerase pore and reach the vRNAP active site, as observed for domain III of TFIIS (Figure 6A) (Kettenberger et al., 2003, 2004). This domain can trigger nucleolytic RNA cleavage at the Pol II active site, and Vaccinia vRNAP has been shown to harbor nucleolytic activity; this has been suggested to be conferred by Rpo30 (Hagler and Shuman, 1993). Thus, Rpo30 contains an NTD that binds to the polymerase in a manner reminiscent of domain II of TFIIS and a mobile CTD that likely uses a TFIIS-like mechanism to trigger RNA cleavage at the vRNAP active site.

### Rpo30 Places Its Phosphorylated C-Tail in the Active Center

Rpo30 additionally contains a C-tail (res. 200–259) that is not resolved in the core vRNAP structure but is clearly visible in

Figure 5. Rap94 and Its Role in the Complete vRNAP Complex

(A) Location of Rap94 in the complete vRNAP structure. The whole model is shown as transparent gray solvent accessible surface with Rap94 shown as solid cartoon. The active site metal A is shown as sphere.

(B) Details of the Rpo147 C-tail and the Rap94 linker 2 (L2). These two elements are shown in worm mode and the rest of the model as solvent accessible surface. The Rpo147 C-tail was visible as a diffuse corridor in the cryo-EM density and was manually modeled as Cα trace for this figure. The quality of the density for this element did not allow assignment of side chains; therefore, this stretch is omitted in the deposited model.

(C) The extended Rap94 linker 3 (L3, shown as worm) connects the B-cyclin domain to the CTD and binds into a cleft on the cRNAP core. The model except for Rap94-L3 and the Rpo147 C-tail is shown as solvent accessible surface.

(D) Close-up view of the CEC and its interactions with VTF/CE and the NPH-I helicase module. Proteins are shown as cartoon with coloring as in Figure 4.

(E) Details of the E11-Rap94 interactions.

(F) Details of the Rap94 domain 2 interactions.

(G) Comparison of the Rap94 B-homology region (top) to the corresponding elements of yeast TFIIB (PDB: 4BBR) (Sainsbury et al., 2013) (bottom).

Figure 6. Structure and Interactions of Subunit Rpo30

(A) Comparison of Vaccinia Rpo30 and S. cerevisiae TFIIS. The proteins are depicted schematically with domains indicated. The position of Rpo30 on the core vRNAP complex is shown on the left, with the rest of the enzyme shown as transparent surface representation with coloring as in Figure 2A. The position of TFIIS in the Pol II reactivation intermediate complex (PDB: 3PO3) (Cheung and Cramer, 2011) is shown on the right, with the rest of the enzyme shown as transparent surface representation.

(B) Cross section through the solvent-accessible surface of the complete vRNAP complex model in the area of the active center cleft. The phosphorylated C-tail of Rpo30 is shown in orange as sticks and the phosphate-moiety shown as purple spheres. The Rap94 B-reader is shown as green worm.

See also Figure S7A.

confirming previous predictions (Henikoff, 1993). SNF2 family proteins are ATP-driven motors with two lobes that are connected by one (INO80, Figure S7C, middle panel) or two (SNF2, Figure S7C, right panel) extended “brace” helices and two protrusions that facilitate DNA interactions. The lobes of NPH-I are connected by a single brace helix, and the C-lobe contains the “protrusion II” found in members of the SNF2 family (Figure S7C, left panel). An additional common feature is the surface at the inside of the brace formed by the two helicase domains, which is lined by stretches of conserved amino acid motifs denoted as motif I-VI (Figure 7B, left panel). The motif II (Walker B) sequence qualifies NPH-I as a DExH helicase and is strictly conserved over all members of the poxviridae family (Deng and Shuman, 1998). NPH-I additionally contains a unique C-terminal region (res. 561–639) that contacts the NTD of Rap94 as part of the CEC through multiple interactions, including an inter-protein β sheet. NPH-I may therefore have evolved from a common ancestor of the SNF2 family and has adapted to its virus-specific function by the acquisition of its CTD.

Host tRNA<sup>Gln</sup> Is an Integral Component of the Complete vRNAP

A peculiar feature of the complete vRNAP complex is the presence of the host tRNA<sup>Gln</sup>. RNA sequencing identified the isoacceptor tRNAs GlnTTG and GlnCTG as the predominant species (Table S3). We therefore modeled the tRNA as tRNA<sup>Gln</sup>. The binding site of this tRNA molecule is located on the periphery and the acceptor arm points away from the center of the complex (Figure 4B). Only weak density could be detected for the acceptor arm of the tRNA, as it is not supported by any protein contacts and hence partially mobile. tRNA<sup>Gln</sup> contacts D2 of Rap94, which forms a broad interface with the anticodon- and

the complete vRNAP structure (Figure S7A). This tail inserts into the pore of the polymerase, running past the active site and into the region that is predicted to interact with the DNA-RNA hybrid at the floor of the active center cleft (Figure 6B). The interactions that hold the C-tail in place are centered around three phosphorylated SP sequence motifs for which we find clear density peaks that allowed us to obtain an atomic model for this Rpo30 region. Although the function of the Rpo30 C-tail remains unknown, structural superposition with a Pol II elongation complex (Gnatt et al., 2001) shows that it would interfere with binding of the DNA-RNA hybrid and thus impair formation of a transcribing complex. In the accompanying paper (Hillen et al., 2019), we show that the DNA-RNA hybrid indeed binds at the expected position and would clash with the Rpo30 C-tail. This indicates that the Rpo30 C-tail must be displaced for transcription.

**Termination Factor NPH-I Resembles Chromatin Remodelers**

The complete vRNAP structure also contains the Vaccinia termination factor NPH-I, consisting of NTDs and CTDs (N-lobe and C-lobe, respectively). NPH-I is located with its N-lobe near the RNA exit pore of vRNAP (Figures 4B and 7A). A structural homology search shows a striking similarity to the chromatin remodeler INO80 of the SNF2 family (Eustermann et al., 2018) (Figure 7B),

Figure 7. Interactions of NPH-I and VETF in the Complete vRNAP Complex

(A) Location of VETF, NPH-I, E11, and tRNA<sup>Gln</sup> in complete vRNAP. The whole model is shown as transparent gray solvent accessible surface with the factors shown as solid cartoon models. Color coding as in Figure 4.

(B) Details of the NPH-I fold and location of its helicase motifs (left). Comparison to INO80 (right) (PDB: 6FHS) (Eustermann et al., 2018). Corresponding regions are colored identically.

(C) Details if the NPH-I interactions with the tRNA anticodon loop.

(D) Details of the VETF-I fold and its tRNA interactions. Disulfide bridges are shown as sticks. See also Figure S7B.

Consistent with this, VETF has been described as a stable heterodimer of VETF-I and VETF-s (Broyles and Moss, 1988). It is likely that during promoter recognition, there are major rearrangements in the complete vRNAP that lead to a positioning of mobile VETF regions onto the promoter DNA.

DISCUSSION

Here, we present a purification procedure for endogenous Vaccinia vRNAP complexes from infected cells and report the first structures of core and complete vRNAP complexes. A comparison to cellular enzymes, in particular eukaryotic Pol II, confirms the common evolutionary origin of multisubunit RNA polymerases and suggests functions of various vRNAP subunits during transcription. Whereas the two large subunits and the active center cleft are generally conserved, peripheral domains, subunits, and factors display virus-specific features. In particular, the viral factor Rap94 associates with vRNAP and contains a central region that resembles the Pol II initiation factor TFIIB and is thus likely involved in transcription initiation. Furthermore, the subunit Rpo30 distantly resembles the Pol II elongation factor TFIIS and likely confers RNA cleavage activity to vRNAP. Such nucleolytic activity appears conserved among multisubunit RNA polymerases and allows for rescue of the transcription machinery in case of backtracking or misincorporation (Fish and Kane, 2002). Whereas the protein that facilitates transcript cleavage is stably associated with polymerase I (Pol I) and polymerase III (Pol III) (Engel et al., 2013; Fernández-Tornero et al., 2013; Hoffmann et al., 2015; Neyer et al., 2016), Pol II requires the auxiliary factor TFIIS (Kettenberger et al., 2003). A similar function is fulfilled by the transcript cleavage factors GreA and GreB in bacterial transcription (Borukhov et al., 1993; Opalka et al., 2003; Polyakov et al., 1998; Stebbins et al., 1995). Rpo30 also contains a C-tail that is specific to poxviridae and not found in other large DNA viruses (Mirzakhanyan and Gereshon, 2017). Phosphorylation of this tail region occurs in

D-arm (Figure 5F). This interaction displays no prominent contacts to particular bases in this area and hence does not confer binding specificity. However, the anticodon loop of tRNA<sup>Gln</sup> is oriented in a manner in which it can be specifically read out by the NPH-I N-lobe (Figure 7C) and VETF-I<sup>365–436</sup> (Figure 7D), which may confer specificity for tRNA<sup>Gln</sup>. Due to the many observed interactions of tRNA<sup>Gln</sup>, it is likely important for the stability of the complete vRNAP complex.

The Initiation Factor VETF Is Anchored to Complete vRNAP

The Vaccinia initiation factor VETF is known to bind promoter DNA upstream and downstream of the transcription start site during initiation of early transcription (Broyles et al., 1991). In the complete vRNAP structure, we observe a central domain of the large VETF subunit (VETF-I<sup>365–436</sup>). This domain has a novel fold that is stabilized by three disulfide bonds and provides the connection between tRNA<sup>Gln</sup>, the TPase module of VTF/CE, and the Rpo18 stalk of the vRNAP core enzyme (Figures 7A and 7D). Although only this domain of the 710-amino-acid VETF-I peptide chain is visible in our density, it is likely that the entire heterodimeric protein is anchored to the complex by this means, as we detect VETF-I and VETF-s in stoichiometric amounts in the sucrose gradient peak fraction (Figure 1B).

packaged virions (Ngo et al., 2016), and it can occupy the vRNAP active site, raising the possibility that this is a regulatory modification. A comparable observation has been made in the apo form of Pol I, in which a peptide region of the largest subunit occupies the active center cleft (Engel et al., 2013; Fernández-Tornero et al., 2013).

A striking feature of vRNAP is the C-tail located on the largest subunit Rpo147. Whereas this tail is flexible in the core vRNAP complex, it binds to the capping enzyme in the complete vRNAP structure. Although structurally not related, the vRNAP C-tail may thus resemble the Pol II CTD with respect to its function in capping enzyme recruitment, but the Pol II CTD more generally acts as an integration hub for transcription-coupled processes (Harlen and Churchman, 2017; Jasnovidova and Stefl, 2013). The CTD recruits various factors during different phases of transcription in a phosphorylation-dependent manner (Buratowski, 2009; Hsin and Manley, 2012) and is also involved in recruitment of the capping enzyme (Cho et al., 1997; Fabrega et al., 2003; McCracken et al., 1997; Noe Gonzalez et al., 2018). In the accompanying paper, we show that the Rpo147 C-tail acts as a tether and alters structure upon rearrangements in the complete vRNAP complex that accompany the formation of an active co-transcriptional capping complex (Hillen et al., 2019).

The additional factors observed in the complete vRNAP structure are unique to the viral machinery. Rap94 acts as an integral building block of the complete vRNAP, because it bridges the interaction between the polymerase and the associated factors. Consistent with this, a loss of this factor leads to generation of virions that lack vRNAP (Zhang et al., 1994). Rap94 binds NPH-I and locks VTF/CE away from the vRNAP core. The structural similarity and location of the Rap94 central region to TFIIB hint at a functional role during transcription initiation. Consistent with this, Rap94 D2 occupies a position that resembles the location of the initiation factor TFIIE in the Pol II preinitiation complex (PIC) (Plaschka et al., 2016), and the Rap94 CTD is found at a location that is congruent with that of TFIIF in Pol II initiation complexes (He et al., 2016; Plaschka et al., 2016). Based on its biochemical composition and activity, it is likely that the complete vRNAP complex represents a unit that is packaged into viral progenies and used for early viral transcription upon virus entry into a host cell.

Our structures also rationalize known functional data. Antibodies directed against an epitope within the CEC of Rap94 inhibit the formation of the PIC *in vitro* (Mohamed et al., 2002), underlining the importance of Rap94 for transcription initiation. Likewise, mutations and deletions within the NPH-I portion of the CEC inhibit termination without affecting its ATPase activity (Mohamed and Niles, 2000; Piacente et al., 2003). For early transcription termination, a sequence motif in the transcribed mRNA triggers the ATPase activity of the single-stranded DNA helicase NPH-I (Broyles, 2003). It has previously been demonstrated that both Rap94 and VTF/CE are involved in the recognition of the termination motif, which may pause the elongating polymerase (Christen et al., 2008; Luo et al., 1995; Tate and Gollnick, 2015). NPH-I may then cause transcript extrusion from the active site by a 5′ to 3′ translocase activity on the non-template strand (Hindman and Gollnick, 2016; Tate and Gollnick, 2011). Provided the observed location of the CEC near the putative RNA exit tunnel is relevant for a termination intermediate, CEC may be

involved in the recognition of the termination signal. Finally, our finding that NPH-I structurally resembles chromatin remodeling ATPases supports the forward translocation model of Vaccinia transcription termination.

We also identified the homodimeric viral core protein E11 as a stoichiometric component of the complete vRNAP. The structure suggests that E11 is a major contributor to the stability of the complete vRNAP. E11 is a late viral product, and two temperature-sensitive mutants have been previously identified to map to its gene (Kato et al., 2008; Wang and Shuman, 1996). One of these, G66R, does not affect the virus morphogenesis but rather leads to the formation of non-infectious viral particles under non-permissive conditions (Wang and Shuman, 1996). According to the crystal structure of E11, this G66R mutant maps to a tight beta-hairpin and is likely to be a structural mutant. Of note, temperature-sensitive mutations in VETF-s and Rap94 have been reported to lead to a defect in protein packaging into mature virions (Kane and Shuman, 1992; Li et al., 1994). These findings are consistent with the idea that the complete vRNAP is the unit that is incorporated into viral progenies and initiates early transcription immediately after virus internalization during the infection cycle.

The incorporation of an uncharged host tRNA<sup>Gln</sup> molecule into a transcription complex is so far unprecedented. The tRNA<sup>Gln</sup> forms an integral part of the complete vRNAP particle, and a presumed loss of tRNA<sup>Gln</sup> is therefore likely to destabilize the complete vRNAP complex. These observations suggest that it might be part of a regulatory mechanism to synchronize the Vaccinia replication cycle to the metabolic status of the host cell. It is interesting in this regard to note that viral replication critically depends on the amino acid glutamine as the primary energy source (Fontaine et al., 2014). It is hence a possibility that the complete vRNAP forms in the late phase of viral infection when glutamine becomes limiting and uncharged tRNA<sup>Gln</sup> accumulates.

Vaccinia virus transcription serves as a paradigm for the molecular biology of nucleo-cytoplasmic large DNA viruses, which include poxviruses and the African swine fever virus. Unlike most other viruses, which rely on the host transcription machinery, they utilize a virus-encoded multisubunit RNA polymerase, which contains a conserved core in different virus taxa (Koonin and Yutin, 2001; Mirzakhanyan and Gershon, 2017). The vRNAP structures presented here provide the first structural insight into the transcription machinery of poxviridae. This provides a framework for future studies aimed at a mechanistic characterization of the viral transcription cycle. In particular, snapshots of vRNAP initiation, elongation, and termination will shed light on the transitions that occur during these processes and decipher the mechanisms by which the virus-specific factors mediate transcription. As a first step in this direction, we provide structures of transcribing and co-transcriptional capping complexes of Vaccinia vRNAP in the accompanying paper (Hillen et al., 2019).

---

### STAR★METHODS

Detailed methods are provided in the online version of this paper and include the following:

- KEY RESOURCES TABLE
- LEAD CONTACT AND MATERIALS AVAILABILITY

- EXPERIMENTAL MODEL AND SUBJECT DETAILS
- METHOD DETAILS
  - Generation of recombinant Vaccinia virus GLV-1h439
  - Viral replication analysis
  - vRNAP purification
  - Initiation assay
  - Mass spectrometry analysis
  - Cross-linking mass spectrometry (XLMS)
  - Rnaseq analysis
  - Structure determination of core vRNAP
  - Structure determination of complete vRNAP
  - X-Ray structure determination of E11
- QUANTIFICATION AND STATISTICAL ANALYSIS
- DATA AND CODE AVAILABILITY

SUPPLEMENTAL INFORMATION

Supplemental Information can be found online at https://doi.org/10.1016/j.cell.2019.11.024.

ACKNOWLEDGMENTS

We would like to thank members of the Cramer and Fischer laboratories for helpful advice and discussion. We thank Christoph Wigge and Jyotishman Veepaschit for support in cryo-EM sample preparation. Cryo-EM of the complete vRNAP was carried out in the cryo-EM facility of the Julius-Maximilians University Würzburg (DFG - INST 93/903-1 FUGG). We further thank Chung-Tine Lee (deceased) for XLMS. This work was supported by the Deutsche Forschungsgemeinschaft within SFB860 and SPP1935 to P.C. and H.U. and under Germany’s Excellence Strategy (EXC 2067/1-390729940) to P.C., the European Research Council Advanced Investigator Grant TRANSREGULON (grant agreement no. 693023) to P.C., and the Volkswagen Foundation to P.C. U.F. and A.S. were supported by the Deutsche Forschungsgemeinschaft (Fi 573 7-2 and 18-1) and the Genelux Corporation. A.H. and M.E. obtained funding from the Austrian Science Fund (SFB F4411 and P-30486-BBL). The synchrotron data were collected at beamline P13 operated by EMBL Hamburg at the PETRA III storage ring (DESY, Hamburg, Germany). We would like to thank Thomas Schneider and the team of beamline P13 for assistance in using the beamline.

AUTHOR CONTRIBUTIONS

C.G. processed the complete vRNAP cryo-EM data, built and refined the model, analyzed the model, and prepared figures. H.S.H. processed the core vRNAP cryo-EM data, built and refined the model, analyzed the model, and prepared figures. K.B. established the purification procedure of the vRNAP complexes. J.B. performed functional vRNAP assays, purified the vRNAP complexes, and prepared figures. S.N. prepared cryo-EM samples of core vRNAP and acquired cryo-EM data. A.A.S. and Q.Z. generated the recombinant Vaccinia virus strain and provided virus for vRNAP purification. C.D. assisted with cryo-EM data collection and processing. A.S. and H.U. performed MS analysis and XLMS, respectively. A.H. and M.E. performed RNA-seq analysis of complete vRNAP. B.B. performed initial cryo-EM analysis of the complete vRNAP complex and assisted with cryo-EM data collection of the complete vRNAP complex. P.C., A.A.S., and U.F. designed and supervised the project and acquired funding. C.G., H.S.H., P.C., and U.F. wrote the manuscript.

DECLARATION OF INTERESTS

Q.Z. is a full-time employee and A.S. and U.F. were former salaried employees of the Genelux Corporation. Q.Z., A.S., and U.F. are company shareholders of the Genelux Corporation.

Received: April 26, 2019  
Revised: August 20, 2019  
Accepted: November 14, 2019  
Published: December 12, 2019  

REFERENCES

Adams, P.D., Afonine, P.V., Bunkóczi, G., Chen, V.B., Davis, I.W., Echols, N., Headd, J.J., Hung, L.-W., Kapral, G.J., Grosse-Kunstleve, R.W., et al. (2010). PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. D Biol. Crystallogr. 66, 213–221.

Ahn, B.Y., Gershon, P.D., Jones, E.V., and Moss, B. (1990). Identification of rpo30, a vaccinia virus RNA polymerase gene with structural similarity to a eukaryotic transcription elongation factor. Mol. Cell. Biol. 10, 5433–5441.

Ahn, B.Y., Gershon, P.D., and Moss, B. (1994). RNA polymerase-associated protein Rap94 confers promoter specificity for initiating transcription of vaccinia virus early stage genes. J. Biol. Chem. 269, 7552–7557.

Ahn, B.Y., and Moss, B. (1992). RNA polymerase-associated transcription specificity factor encoded by vaccinia virus. Proc. Natl. Acad. Sci. USA 89, 3536–3540.

Amegadzie, B.Y., Holmes, M.H., Cole, N.B., Jones, E.V., Earl, P.L., and Moss, B. (1991). Identification, sequence, and expression of the gene encoding the second-largest subunit of the vaccinia virus DNA-dependent RNA polymerase. Virolology 180, 88–98.

Armache, K.-J., Kettenberger, H., and Cramer, P. (2003). Architecture of initiation-competent 12-subunit RNA polymerase II. Proc. Natl. Acad. Sci. USA 100, 6964–6968.

Armache, K.-J., Mitterweger, S., Meinhart, A., and Cramer, P. (2005). Structures of complete RNA polymerase II and its subcomplex, Rpb4/7. J. Biol. Chem. 280, 7131–7134.

Baldick, C.J., Jr., and Moss, B. (1993). Characterization and temporal regulation of mRNAs encoded by vaccinia virus intermediate-stage genes. J. Virol. 67, 3515–3527.

Barnes, C.O., Calero, M., Malik, I., Graham, B.W., Spåhr, H., Lin, G., Cohen, A.E., Brown, I.S., Zhang, Q., Pullara, F., et al. (2015). Crystal Structure of a Transcribing RNA Polymerase II Complex Reveals a Complete Transcription Bubble. Mol. Cell 59, 258–269.

Bernecky, C., Herzog, F., Baumeister, W., Plitzko, J.M., and Cramer, P. (2016). Structure of transcribing mammalian RNA polymerase II. Nature 529, 551–554.

Bernecky, C., Plitzko, J.M., and Cramer, P. (2017). Structure of a transcribing RNA polymerase II-DSIF complex reveals a multidentate DNA-RNA clamp. Nat. Struct. Mol. Biol. 24, 809–815.

Biasini, M., Bienert, S., Waterhouse, A., Arnold, K., Studer, G., Schmidt, T., Kiefer, F., Gallo Cassarino, T., Bertoni, M., Bordoli, L., and Schwede, T. (2014). SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. Nucleic Acids Res. 42, W252–W258.

Bond, C.S., and Schüttelkopf, A.W. (2009). ALINE: a WYSIWYG protein-sequence alignment editor for publication-quality alignments. Acta Crystallogr. D Biol. Crystallogr. 65, 510–512.

Borukhov, S., Sagitov, V., and Goldfarb, A. (1993). Transcript cleavage factors from E. coli. Cell 72, 459–466.

Broyles, S.S. (2003). Vaccinia virus transcription. J. Gen. Virol. 84, 2293–2303.

Broyles, S.S., and Moss, B. (1986). Homology between RNA polymerases of poxviruses, prokaryotes, and eukaryotes: nucleotide sequence and transcriptional analysis of vaccinia virus genes encoding 147-kDa and 22-kDa subunits. Proc. Natl. Acad. Sci. USA 83, 3141–3145.

Broyles, S.S., and Moss, B. (1988). DNA-dependent ATPase activity associated with vaccinia virus early transcription factor. J. Biol. Chem. 263, 10761–10765.

Broyles, S.S., Li, J., and Moss, B. (1991). Promoter DNA contacts made by the vaccinia virus early transcription factor. J. Biol. Chem. 266, 15539–15544.

Buratowski, S. (2009). Progression through the RNA polymerase II CTD cycle. Mol. Cell 36, 541–546.

Chen, V.B., Arendall, W.B., 3rd, Headd, J.J., Keedy, D.A., Immormino, R.M., Kapral, G.J., Murray, L.W., Richardson, J.S., and Richardson, D.C. (2010). MolProbity: all-atom structure validation for macromolecular crystallography. Acta Crystallogr. D Biol. Crystallogr. 66, 12–21.

Cheung, A.C.M., and Cramer, P. (2011). Structural basis of RNA polymerase II backtracking, arrest and reactivation. Nature 471, 249–253.

Chi, P.-I., and Liu, H.-J. (2012). Molecular Signaling and Cellular Pathways for Virus Entry. International Scholarly Research Notices 2013, 1–8.

Cho, E.J., Takagi, T., Moore, C.R., and Buratowski, S. (1997). mRNA capping enzyme is recruited to the transcription complex by phosphorylation of the RNA polymerase II carboxy-terminal domain. Genes Dev. 11, 3319–3326.

Christen, L.A., Piacente, S., Mohamed, M.R., and Niles, E.G. (2008). Vaccinia virus early gene transcription termination factors VTF and Rap94 interact with the U9 termination motif in the nascent RNA in a transcription ternary complex. Virolology 376, 225–235.

Cotter, C.A., Earl, P.L., Wyatt, L.S., and Moss, B. (2017). Preparation of Cell Cultures and Vaccinia Virus Stocks. Curr. Protoc. Protein Sci. 89, 5.12.1–5.12.18.

Cramer, P., Bushnell, D.A., Fu, J., Gnatt, A.L., Maier-Davis, B., Thompson, N.E., Burgess, R.R., Edwards, A.M., David, P.R., and Kornberg, R.D. (2000). Architecture of RNA polymerase II and implications for the transcription mechanism. Science 288, 640–649.

Cramer, P., Bushnell, D.A., and Kornberg, R.D. (2001). Structural basis of transcription: RNA polymerase II at 2.8 angstrom resolution. Science 292, 1863–1876.

Deng, L., and Shuman, S. (1998). Vaccinia NPH-I, a DExH-box ATPase, is the energy coupling factor for mRNA transcription termination. Genes Dev. 12, 538–546.

Emsley, P., and Cowtan, K. (2004). Coot: model-building tools for molecular graphics. Acta Crystallogr. D Biol. Crystallogr. 60, 2126–2132.

Emsley, P., Lohkamp, B., Scott, W.G., and Cowtan, K. (2010). Features and development of Coot. Acta Crystallogr. D Biol. Crystallogr. 66, 486–501.

Engel, C., Sainsbury, S., Cheung, A.C., Kostrewa, D., and Cramer, P. (2013). RNA polymerase I structure and transcription regulation. Nature 502, 650–655.

Eustermann, S., Schall, K., Kostrewa, D., Lakomek, K., Strauss, M., Moldt, M., and Hopfner, K.-P. (2018). Structural basis for ATP-dependent chromatin remodelling by the INO80 complex. Nature 556, 386–390.

Fabrega, C., Shen, V., Shuman, S., and Lima, C.D. (2003). Structure of an mRNA capping enzyme bound to the phosphorylated carboxy-terminal domain of RNA polymerase II. Mol. Cell 11, 1549–1561.

Falkner, F.G., and Moss, B. (1990). Transient dominant selection of recombinant vaccinia viruses. J. Virol. 64, 3108–3111.

Fernández-Tornero, C., Moreno-Morcillo, M., Rashid, U.J., Taylor, N.M.I., Ruiz, F.M., Gruene, T., Legrand, P., Steuerwald, U., and Müller, C.W. (2013). Crystal structure of the 14-subunit RNA polymerase I. Nature 502, 644–649.

Fish, R.N., and Kane, C.M. (2002). Promoting elongation with transcript cleavage stimulatory factors. Biochim. Biophys. Acta 1577, 287–307.

Fontaine, K.A., Camarda, R., and Lagunoff, M. (2014). Vaccinia virus requires glutamine but not glucose for efficient replication. J. Virol. 88, 4366–4374.

Gnatt, A.L., Cramer, P., Fu, J., Bushnell, D.A., and Kornberg, R.D. (2001). Structural basis of transcription: an RNA polymerase II elongation complex at 3.3 Å resolution. Science 292, 1876–1882.

Hagler, J., and Shuman, S. (1993). Nascent RNA cleavage by purified ternary complexes of vaccinia RNA polymerase. J. Biol. Chem. 268, 2166–2173.

Harlen, K.M., and Churchman, L.S. (2017). The code and beyond: transcription regulation by the RNA polymerase II carboxy-terminal domain. Nat. Rev. Mol. Cell Biol. 18, 263–273.

He, Y., Yan, C., Fang, J., Inouye, C., Tjian, R., Ivanov, I., and Nogales, E. (2016). Near-atomic resolution visualization of human transcription promoter opening. Nature 533, 359–365.

Henikoff, S. (1993). Transcriptional activator components and poxvirus DNA-dependent ATPases comprise a single family. Trends Biochem. Sci. 18, 291–292.

Hillen, H.S., Bartuli, J., Grimm, C., Dienemann, C., Bedenk, K., Urlaub, H., Szalay, A., Fischer, U., and Cramer, P. (2019). Structural basis of Poxvirus transcription: transcribing and capping complexes. Cell 179. Published online December 12, 2019. https://doi.org/10.1016/j.cell.2019.11.023.

Hindman, R., and Gollnick, P. (2016). Nucleoside Triphosphate Phosphohydrolase I (NPH I) Functions as a 5′ to 3′ Translocase in Transcription Termination of Vaccinia Early Genes. J. Biol. Chem. 291, 14826–14838.

Hirata, A., Klein, B.J., and Murakami, K.S. (2008). The X-ray crystal structure of RNA polymerase from Archaea. Nature 451, 851–854.

Hoffmann, N.A., Jakobi, A.J., Moreno-Morcillo, M., Glatt, S., Kosinski, J., Hagen, W.J.H., Sachse, C., and Müller, C.W. (2015). Molecular structures of unbound and transcribing RNA polymerase III. Nature 528, 231–236.

Hsin, J.-P., and Manley, J.L. (2012). The RNA polymerase II CTD coordinates transcription and RNA processing. Genes Dev. 26, 2119–2137.

Jasnovidova, O., and Stefl, R. (2013). The CTD code of RNA polymerase II: a structural view. Wiley Interdiscip. Rev. RNA 4, 1–16.

Jones, D.T. (1999). Protein secondary structure prediction based on position-specific scoring matrices. J. Mol. Biol. 292, 195–202.

Kane, E.M., and Shuman, S. (1992). Temperature-sensitive mutations in the vaccinia virus H4 gene encoding a component of the virion RNA polymerase. J. Virol. 66, 5752–5762.

Kates, J.R., and McAuslan, B.R. (1967). Poxvirus DNA-dependent RNA polymerase. Proc. Natl. Acad. Sci. USA 58, 134–141.

Kato, S.E.M., Moussatche, N., D’Costa, S.M., Bainbridge, T.W., Prins, C., Strahl, A.L., Shatzer, A.N., Brinker, A.J., Kay, N.E., and Condit, R.C. (2008). Marker rescue mapping of the combined Condit/Dales collection of temperature-sensitive vaccinia virus mutants. Virology 375, 213–222.

Kettenberger, H., Armache, K.-J., and Cramer, P. (2003). Architecture of the RNA polymerase II-TFIIS complex and implications for mRNA cleavage. Cell 114, 347–357.

Kettenberger, H., Armache, K.-J., and Cramer, P. (2004). Complete RNA polymerase II elongation complex structure and its interactions with NTP and TFIIS. Mol. Cell 16, 955–965.

Knutson, B.A., and Broyles, S.S. (2008). Expansion of poxvirus RNA polymerase subunits sharing homology with corresponding subunits of RNA polymerase II. Virus Genes 36, 307–311.

Koonin, E.V., and Yutin, N. (2001). Nucleo-cytoplasmic Large DNA Viruses (NCLDV) of Eukaryotes (American Cancer Society).

Kyrieleis, O.J., Chang, J., de la Peña, M., Shuman, S., and Cusack, S. (2014). Crystal structure of vaccinia virus mRNA capping enzyme provides insights into the mechanism and evolution of the capping apparatus. Structure 22, 452–465.

Li, J., Pennington, M.J., and Broyles, S.S. (1994). Temperature-sensitive mutations in the gene encoding the small subunit of the vaccinia virus early transcription factor impair promoter binding, transcription activation, and packaging of multiple virion components. J. Virol. 68, 2605–2614.

Liu, Y., Schmidt, B., and Maskell, D.L. (2010). MSAProbs: multiple sequence alignment based on pair hidden Markov models and partition function posterior probabilities. Bioinformatics 26, 1958–1964.

Liu, X., Li, M., Xia, X., Li, X., and Chen, Z. (2017). Mechanism of chromatin remodelling revealed by the Snf2-nucleosome structure. Nature 544, 440–445.

Luo, Y., Hagler, J., and Shuman, S. (1991). Discrete functional stages of vaccinia virus early transcription during a single round of RNA synthesis in vitro. J. Biol. Chem. 266, 13303–13310.

Luo, Y., Mao, X., Deng, L., Cong, P., and Shuman, S. (1995). The D1 and D12 subunits are both essential for the transcription termination factor activity of vaccinia virus capping enzyme. J. Virol. 69, 3852–3856.
Martinez-Rucobo, F.W., Sainsbury, S., Cheung, A.C.M., and Cramer, P. (2011). Architecture of the RNA polymerase-Spt4/5 complex and basis of universal transcription processivity. EMBO J. 30, 1302–1310.

McCracken, S., Fong, N., Rosonina, E., Yankulov, K., Brothers, G., Siderovski, D., Hessel, A., Foster, S., Shuman, S., and Bentley, D.L. (1997). 5′-Capping enzymes are targeted to pre-mRNA by binding to the phosphorylated carboxy-terminal domain of RNA polymerase II. Genes Dev. 11, 3306–3318.

Mirzakhanyan, Y., and Gershon, P.D. (2017). Multisubunit DNA-Dependent RNA Polymerases from Vaccinia Virus and Other Nucleocytoplasmic Large-DNA Viruses: Impressions from the Age of Structure. Microbiol. Mol. Biol. Rev. 81, e00010–e00017.

Mohamed, M.R., and Niles, E.G. (2000). Interaction between nucleoside triphosphate phosphohydrolase I and the H4L subunit of the viral RNA polymerase is required for vaccinia virus early gene transcript release. J. Biol. Chem. 275, 25798–25804.

Mohamed, M.R., Christen, L.A., and Niles, E.G. (2002). Antibodies directed against an epitope in the N-terminal region of the H4L subunit of the vaccinia virus RNA polymerase inhibit both transcription initiation and transcription termination, in vitro. Virology 299, 142–153.

Moss, B. (2012). Poxvirus cell entry: how many proteins does it take? Viruses 4, 688–707.

Moss, B. (2013). Poxvirus DNA replication. Cold Spring Harb. Perspect. Biol. 5, a010199.

Munyon, W., Paoletti, E., and Grace, J.T., Jr. (1967). RNA polymerase activity in purified infectious vaccinia virus. Proc. Natl. Acad. Sci. USA 58, 2280–2287.

Neyer, S., Kunz, M., Geiss, C., Hantsche, M., Hodirnau, V.-V., Seybert, A., Engel, C., Scheffer, M.P., Cramer, P., and Frangakis, A.S. (2016). Structure of RNA polymerase I transcribing ribosomal DNA genes. Nature 540, 607–610.

Ngo, T., Mirzakhanyan, Y., Moussatche, N., and Gershon, P.D. (2016). Protein Primary Structure of the Vaccinia Virion at Increased Resolution. J. Virol. 90, 9905–9919.

Noe Gonzalez, M., Sato, S., Tomomori-Sato, C., Conaway, J.W., and Conaway, R.C. (2018). CTD-dependent and -independent mechanisms govern co-transcriptional capping of Pol II transcripts. Nat. Commun. 9, 3392.

Opalka, N., Chlenov, M., Chacon, P., Rice, W.J., Wriggers, W., and Darst, S.A. (2003). Structure and function of the transcription elongation factor GreB bound to bacterial RNA polymerase. Cell 114, 335–345.

Pettersen, E.F., Goddard, T.D., Huang, C.C., Couch, G.S., Greenblatt, D.M., Meng, E.C., and Ferrin, T.E. (2004). UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612.

Piacente, S.C., Christen, L.A., Mohamed, M.R., and Niles, E.G. (2003). Effect of selected mutations in the C-terminal region of the vaccinia virus nucleoside triphosphate phosphohydrolase I on binding to the H4L subunit of the viral RNA polymerase and early gene transcription termination in vitro. Virology 310, 109–117.

Plaschka, C., Hantsche, M., Dienemann, C., Burzinski, C., Plitzko, J., and Cramer, P. (2016). Transcription initiation complex structures elucidate DNA opening. Nature 533, 353–358.

Polyakov, A., Richter, C., Malhotra, A., Koulich, D., Borukhov, S., and Darst, S.A. (1998). Visualization of the binding site for the transcript cleavage factor GreB on Escherichia coli RNA polymerase. J. Mol. Biol. 281, 465–473.

Rohou, A., and Grigorieff, N. (2015). CTFFIND4: Fast and accurate defocus estimation from electron micrographs. J. Struct. Biol. 192, 216–221.

Rould, M.A., Perona, J.J., Söll, D., and Steitz, T.A. (1989). Structure of E. coli glutaminy-tRNA synthetase complexed with tRNA(Gln) and ATP at 2.8 Å resolution. Science 246, 1135–1142.

Sainsbury, S., Niesser, J., and Cramer, P. (2013). Structure and function of the initially transcribing RNA polymerase II-TFIIB complex. Nature 493, 437–440.

Scheres, S.H.W. (2012). RELION: implementation of a Bayesian approach to cryo-EM structure determination. J. Struct. Biol. 180, 519–530.

Schrodinger, L.L.C. (2015). The PyMOL Molecular Graphics System, Version 1.8.

Stebbins, C.E., Borukhov, S., Orlova, M., Polyakov, A., Goldfarb, A., and Darst, S.A. (1995). Crystal structure of the GreA transcript cleavage factor from Escherichia coli. Nature 373, 636–640.

Stothard, P. (2000). The sequence manipulation suite: JavaScript programs for analyzing and formatting protein and DNA sequences. Biotechniques 28, 1102–1104, 1104.

Tate, J., and Gollnick, P. (2011). Role of forward translocation in nucleoside triphosphate phosphohydrolase I (NPH I)-mediated transcription termination of vaccinia virus early genes. J. Biol. Chem. 286, 44764–44775.

Tate, J., and Gollnick, P. (2015). The role of vaccinia termination factor and cis-acting elements in vaccinia virus early gene transcription termination. Virology 485, 179–188.

Tegunov, D., and Cramer, P. (2019). Real-time cryo-electron microscopy data preprocessing with Warp. Nat. Methods 16, 1146–1152.

Vos, S.M., Pöllmann, D., Caizzi, L., Hofmann, K.B., Rombaut, P., Zimniak, T., Herzog, F., and Cramer, P. (2016). Architecture and RNA binding of the human negative elongation factor. eLife 5, e14981.

Vos, S.M., Farnung, L., Boehning, M., Wigge, C., Linden, A., Urlaub, H., and Cramer, P. (2018). Structure of activated transcription complex Pol II-DSIF-PAF-SPT6. Nature 560, 607–612.

Wang, S.P., and Shuman, S. (1996). A temperature-sensitive mutation of the vaccinia virus E11 gene encoding a 15-kDa virion component. Virology 216, 252–257.

Yang, B., Wu, Y.-J., Zhu, M., Fan, S.-B., Lin, J., Zhang, K., Li, S., Chi, H., Li, Y.-X., Chen, H.-F., et al. (2012). Identification of cross-linked peptides from complex samples. Nat. Methods 9, 904–906.

Zhang, K. (2016). Gctf: Real-time CTF determination and correction. J. Struct. Biol. 193, 1–12.

Zhang, Y., Ahn, B.Y., and Moss, B. (1994). Targeting of a multicomponent transcription apparatus into assembling vaccinia virus particles requires RAP94, an RNA polymerase-associated protein. J. Virol. 68, 1360–1370.

Zhang, G., Campbell, E.A., Minakhin, L., Richter, C., Severinov, K., and Darst, S.A. (1999). Crystal structure of Thermus aquaticus core RNA polymerase at 3.3 Å resolution. Cell 98, 811–824.

Zheng, S.Q., Palovcak, E., Armache, J.-P., Verba, K.A., Cheng, Y., and Agard, D.A. (2017). MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods 14, 331–332.

Zimmermann, L., Stephens, A., Nam, S.-Z., Rau, D., Kübler, J., Lozajic, M., Gabler, F., Söding, J., Lupas, A.N., and Alva, V. (2018). A Completely Reimplemented MPI Bioinformatics Toolkit with a New HHpred Server at its Core. J. Mol. Biol. 430, 2237–2243.

Zivanov, J., Nakane, T., Forsberg, B.O., Kimanius, D., Hagen, W.J., Lindahl, E., and Scheres, S.H. (2018). New tools for automated high-resolution cryo-EM structure determination in RELION-3. eLife 7, 163.
STAR★METHODS

KEY RESOURCES TABLE

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
|---------------------|--------|------------|
| Bacterial and Virus Strains |  |  |
| GLV-1h68 | Genelux Corporation | N/A |
| GLV-1h439 | This paper | N/A |
| BL21(DE3)pLysS | Promega Corporation | L1195 |
| Deposited Data |  |  |
| Core vRNAP structure | This paper | PDB:6RIC |
| Core vRNAP cryo-EM map | This paper | EMD-4888 |
| Complete vRNAP structure | This paper | PDB:6RFL |
| Complete vRNAP cryo-EM map | This paper | EMD-4868 |
| E11 crystal structure | This paper | PDB:6RFG |
| INO80 Chromatin Remodeler Structure | Eustermann et al., 2018 | PDB:6FHS |
| S.cerevisiae Pol II crystal structure | Armache et al., 2005 | PDB:1WCM |
| B.taurus Pol II elongation complex structure | Bernecky et al., 2016 | EMD-3218 |
| Vaccinia capping enzyme crystal structure | Kyrieleis et al., 2014 | PDB:4CKB |
| SNF2 Structure | Liu et al., 2017 | PDB:5X0X |
| Bacterial tRNA<sup>Gln</sup> Structure | Rould et al., 1989 | PDB:1GSG |
| S.cerevisiae Pol II-TFIIS crystal structure | Sainsbury et al., 2013 | PDB:4BBR |
| S.cerevisiae Pol II backtracked reactivation structure | Cheung and Cramer, 2011 | PDB:3P03 |
| Experimental Models: Cell Lines |  |  |
| HeLaS3 | Sigma-Aldrich | 87110901 |
| African green monkey kidney fibroblasts (CV-1) | ATCC | CCL-70 |
| Oligonucleotides |  |  |
| A24R-5:CCTAGTAGTAACGAAACATCCATCGATAC | This paper | N/A |
| A24Rtag-3:CCTTGTAATCAGCGTAATCTGGAACATCGT | This paper | N/A |
| ATGGGTACATGGTCACCAGAAAAGACGGCTT GAG |  |  |
| A25Ltag-5:GTCCAGATTACGCTGATTACAAGGATGA | This paper | N/A |
| CGACGATAAGGCAGCAT AATATTCTAGTTTGGTAGTA |  |  |
| GATACATATCAATATCATCA |  |  |
| A25L-3:TCGATCGCCTTCAGAAAGACTG | This paper | N/A |
| A24R-5:CCTAGTAGTAACGAAACATCCATCGATAC | This paper | N/A |
| A25L-3:TCGATCGCCTTCAGAAAGACTG | This paper | N/A |
| In-vitro transcription template strand: | IDT DNA | N/A |
| GACTTATGATCGGATAAGAGTCCAGCCAATGACA |  |  |
| GATGCCCTCATAGCC |  |  |
| In-vitro transcription non-template strand: | IDT DNA | N/A |
| GGCTATGAGGCATCCCATGCGTTGAGGACTCTTA |  |  |
| TCCGATCAT AAGTC |  |  |
| In-vitro transcription RNA primer: | This paper | N/A |
| GAGUUGUAAUAACAAGGGAAAUGUCAUUGG |  |  |
| Recombinant DNA |  |  |
| Plasmid pSB24 | Luo et al., 1991 | pSB24 |
| Software and Algorithms |  |  |
| MotionCor2 | Zheng et al., 2017 | https://msg.ucsf.edu/em/software/motioncor2.html |
| Gctf | Zhang, 2016 | https://www.mrc-lmb.cam.ac.uk/kzhang/ |
(Continued on next page)

Continued

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
|---------------------|--------|------------|
| Gautomatch          | Kai Zhang | https://www.mrc-lmb.cam.ac.uk/kzhang/ |
| Relion 3.0          | Scheres, 2012 | https://github.com/3dem/relion |
| Warp                | Tegunov and Cramer, 2019 | http://www.warpem.com |
| Phenix              | Adams et al., 2010 | http://www.phenix-online.org |
| PyMol 1.8           | Schrodinger, 2015 | http://www.pymol.org |
| Coot                | Emsley and Cowtan, 2004 | https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/ |
| UCSF Chimera        | Pettersen et al., 2004 | https://www.cgl.ucsf.edu/chimera/ |
| Aline               | Bond and Schüttelkopf, 2009 | https://bondxray.org/software/aline.html |

---

### LEAD CONTACT AND MATERIALS AVAILABILITY

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact Utz Fischer (utz.fischer@biozentrum.uni-wuerzburg.de). All unique/stable reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement.

---

### EXPERIMENTAL MODEL AND SUBJECT DETAILS

Male African green monkey kidney fibroblasts (CV-1) were purchased from the American Type Culture Collection (ATCC No. CCL-70). Human HeLa S3 cells (female) were cultured in a 37°C incubator equilibrated with 5% CO2 and 95% humidified atmosphere. Both cell lines were cultured in DMEM (GIBCO) supplemented with 10% FCS and 1% Penicillin/Streptomycin.

---

### METHOD DETAILS

#### Generation of recombinant Vaccinia virus GLV-1h439

GLV-1h439 was derived from GLV-1h68 with a HA-tag and FLAG-tag inserted at the end of A24R gene (encoding vRNAP subunit Rpo132). For insertion of the HA/FLAG-doubletag, an A24R transfer vector was constructed. DNA fragments (termed A and B), flanking about 500 bps of each side of the insertion site of the A24R gene were first amplified via PCR with primers A24R-5/A23R-tag3 (product A) and A25Ltag-5/A25L-3 (product B) as listed in the Key Resources Table. A second round of PCR linked A and B fragments into product C with primers A24R-5 and A25L-3. The PCR product C was cloned into the pCR-Blunt II-TOPO vector using Zero Blunt TOPO PCR cloning Kit (Invitrogen). The resulting construct pCRII-A24Rtag4 was sequence confirmed. A p7.5E-gpt cDNA fragment (*E.coli* xanthine-guanine phosphoribosyltransferase gene under the control of vaccinia 7.5 early promoter), released by XbaI and PstI restriction digest from the TK transfer vector, was then subcloned into pCRII-A24Rtag4. The gpt selection-expression cassette was located outside the *Vaccinia* virus DNA that directs homologous recombination into the virus genome, allowing for transient dominant selection of vaccinia recombinants (Falkner and Moss, 1990). The final construct A24Rtag-gpt2 was sequence confirmed, and used to make recombinant virus GLV-1h439, with GLV-1h68 as the parental virus.

#### Viral replication analysis

Replication of recombinant GLV-1h439 and GLV-1h68 was performed using a standard plaque assay (Cotter et al., 2017). HeLa S3 cells were grown in 24-well plates and infected with virus at a multiplicity of infection (MOI) of 1. After incubation for 1 h at 37°C, medium was replaced by fresh growth medium and samples were collected 2, 24, 48 and 72 h post viral infection (hpi). After three freeze-thaw cycles, lysates were titrated by plaque assay on CV-1 cells. The assay was performed in triplicate and all samples were measured in duplicates.

#### vRNAP purification

For purification of vRNAP from infected cells, HeLa S3 cells were grown in 15-cm plates up to 80%–90% of confluence. The cells were infected with purified GLV-1h439 with a MOI of 1.2. After 24 h the cells were pelleted and resuspended in lysis buffer (50 mM HEPES, pH 7.5, 150 mM NaCl, 1.5 mM MgCl2, 0.5% [v/v] NP-40, 1 mM DTT, and complete EDTA-free protease inhibitor cocktail [Sigma-Aldrich]). For vRNAP purification, the extract was incubated for 3 h at 4°C with 200 μL anti-FLAG Agarose (Sigma). Beads were washed four times with buffer containing 50 mM HEPES, pH 7.5, 150 mM NaCl, 1.5 mM MgCl2, 0.1% [v/v] NP-40 and 1 mM DTT and equilibrated with elution buffer (50 mM HEPES, pH 7.5, 150 mM NaCl, 1.5 mM MgCl2 and 1 mM DTT). The bead-bound proteins were eluted with 3 x FLAG peptide, resolved in 12% Bis-Tris gels and visualized by silver staining. For purification of native vRNAP, the eluate of the anti-FLAG column was concentrated to 1 mg/mL and layered on top of a 10% - 30% sucrose

e2 Cell 179, 1537–1550.e1–e6, December 12, 2019

gradient and centrifuged for 16 h and 35,000 rpm at 4°C in a Beckman 60Ti swing-out rotor. Gradient fractions were fractionated manually, separated by SDS-PAGE and proteins visualized by silver staining.

### Initiation assay

Plasmid pSB24, containing a G-less cassette downstream of a synthetic vaccinia virus early promoter was generously provided by Dr. Steven Broyles (Purdue University). Construction of the pSB24 vector with Vaccinia virus early termination signal was described in ([Luo et al., 1991](https://doi.org/10.1073/pnas.88.19.8578)). Briefly, by standard genetic manipulation, the sequence from BamHI site to HindIII site of the pSB24 was replaced with the duplex oligonucleotides. The insert sequences include three tandem copies of Vaccinia early termination signal. A typical *in vitro* transcription had a volume of a 50 μl and contained 40 mM Tris-HCl, pH 7.9, 1 mM DTT, 2 mM spermidine, 6 mM MgCl2, 1 mM ATP, 1 mM CTP, 1 mM GTP, 0.1 mM UTP, 20 μCi α[32P]-UTP [6000 Ci/mmol], 80 μM SAM, 400 ng of Ndel-linearized pSB24 template as well as purified core or complete vRNAP ([Luo et al., 1991](https://doi.org/10.1073/pnas.88.19.8578)). The reaction was incubated at 30°C for the indicated time points before RNA was extracted and precipitated with isopropanol. Transcripts were analyzed by denaturing gel electrophoresis and visualized by autoradiography.

### Mass spectrometry analysis

For protein identification by in-gel digestion each gel lane was cut into 15 slices. The gel bands were destained with 30% acetonitrile in 0.1 M NH₄HCO₃ (pH 8.0), shrunk with 100% acetonitrile, and dried in a vacuum concentrator (Concentrator 5301, Eppendorf, Germany). Digests were performed with 0.1 μg trypsin per gel band overnight at 37°C in 0.1 M NH₄HCO₃ (pH 8.0). After removing the supernatant, peptides were extracted from the gel slices with 5% formic acid, and extracted peptides were pooled with the supernatant. Nano LC-MS/MS analyses were performed on an Orbitrap Fusion (Thermo Scientific) equipped with a PicoView Ion Source (New Objective) and coupled to an EASY-nLC 1000 (Thermo Scientific). Peptides were loaded on capillary columns (PicoFrit, 30 cm x 150 μm ID, New Objective) self-packed with ReproSil-Pur 120 C18-AQ, 1.9 μm (Dr. Maisch) and separated with a 30-min linear gradient from 3% to 30% acetonitrile and 0.1% formic acid and a flow rate of 500 nL/min. Both MS and MS/MS scans were acquired in the Orbitrap analyzer with a resolution of 60,000 for MS scans and 15,000 for MS/MS scans. HCD fragmentation with 35% normalized collision energy was applied. A Top Speed data-dependent MS/MS method with a fixed cycle time of 3 s was used. Dynamic exclusion was applied with a repeat count of 1 and an exclusion duration of 30 s; singly charged precursors were excluded from selection. Minimum signal threshold for precursor selection was set to 50,000. Predictive AGC was used with AGC a target value of 2e5 for MS scans and 5e4 for MS/MS scans. EASY-IC was used for internal calibration. Data analysis was performed against UniProt Vaccinia Virus database with PEAKS 8.5 software (Bioinformatics Solution Inc.) with the following parameters: parent mass tolerance: 8 ppm, fragment mass tolerance: 0.02 Da, enzyme: trypsin, variable modifications: oxidation (M), pyro-glutamate (N-term. Q), phosphorylation (STY), carbamidomethylation (C). Results were filtered to 1% PSM-FDR by target-decoy approach.

### Cross-linking mass spectrometry (XLMS)

Protein cross-linking of purified complexes and subsequent mass spectrometry was performed as described previously ([Vos et al., 2018](https://doi.org/10.1074/mcp.RA117.000504)). Briefly, samples were crosslinked with BS3 (Thermo Fisher Scientific) and incubated for 30 min at 30°C. The reaction was quenched by adding 100 mM Tris-HCl pH 7.5 and 20 mM ammonium bicarbonate (final concentrations) and incubation for 15 min at 30°C. Proteins were precipitated with 300 mM sodium acetate pH 5.2 and four volumes of acetone overnight at –20°C. The protein was pelleted by centrifugation, briefly dried, and resuspended in 4 M urea and 50 mM ammonium bicarbonate. Crosslinked proteins were reduced with DTT and alkylated ([Vos et al., 2016](https://doi.org/10.1074/mcp.M115.055983)). After dilution to 1 M urea with 50 mM ammonium bicarbonate (pH 8.0), the crosslinked protein complex was digested with trypsin in a 1:50 enzyme-to-protein ratio at 37°C overnight. Peptides were acidified with trifluoroacetic acid (TFA) to a final concentration of 0.5% (v/v), desalted on MicroSpin columns (Harvard Apparatus) following manufacturer’s instructions and vacuum-dried. Dried peptides were dissolved in 50 μL 30% acetonitrile/0.1% TFA and peptide size exclusion (pSEC, Superdex Peptide 3.2/300 column on an ÄKTAmicro system, GE Healthcare) was performed to enrich for crosslinked peptides at a flow rate of 50 μL min–1. Fractions of 50 μL were collected. Fractions containing the crosslinked peptides (1–1.7 ml) were vacuum-dried and dissolved in 2% acetonitrile/0.05% TFA (v/v) for analysis by LC–MS/MS.

Crosslinked peptides were analyzed as technical duplicates on an Orbitrap Fusion or Orbitrap Fusion Lumos Tribrid Mass Spectrometer (Thermo Fisher Scientific), coupled to a Dionex UltiMate 3000 UHPLC system (Thermo Fisher Scientific) equipped with an in-house-packed C18 column (ReproSil-Pur 120 C18-AQ, 1.9 μm pore size, 75 μm inner diameter, 30 cm length, Dr. Maisch GmbH). Samples were separated applying the following 58 min gradient: mobile phase A consisted of 0.1% formic acid (v/v), mobile phase B of 80% acetonitrile/0.08% formic acid (v/v). The gradient started with 5% B, increasing to 8% B on Fusion and 15% on Fusion Lumos, within 3 min, followed by 8%–42% B and 15%–46% B within 43 min accordingly, then keeping B constant at 90% for 6 min. After each gradient the column was again equilibrated to 5% B for 6 min. The flow rate was set to 300 nL min–1. MS1 spectra were acquired with a resolution of 120,000 in the Orbitrap covering a mass range of 380–1580 m/z. Injection time was set to 60 ms and automatic gain control target to 5 × 105. Dynamic exclusion covered 10 s. Only precursors with a charge state of 3–8 were included. MS2 spectra were recorded with a resolution of 30,000 in the Orbitrap, injection time was set to 128 ms, automatic gain control target to 5 × 104 and the isolation window to 1.6 m/z. Fragmentation was enforced by higher-energy collisional dissociation at 30%.

Raw files were converted to mgf format using ProteomeDiscoverer 1.4 (Thermo Scientific, signal-to-noise ratio 1.5, 1,000–10,000 Da precursor mass). For identification of crosslinked peptides, files were analyzed by pLink (v. 1.23), pFind group

(Yang et al., 2012) using BS3 as crosslinker and trypsin as digestion enzyme with maximal two missed cleavage sites. Carbamidomethylation of cysteines was set as a fixed modification, oxidation of methionines as a variable modification. Searches were conducted in combinatorial mode with a precursor mass tolerance of 5 Da and a fragment ion mass tolerance of 20 ppm The used database contained all proteins within the complex. The false discovery rate was set to 0.01. Results were filtered by applying a precursor mass accuracy of ± 10 ppm Spectra of both technical duplicates were combined and evaluated manually.

## RNaseq analysis

Libraries were generated from the isolated RNA fraction following the Ion Torrent Ion Total RNA-seq kit v2 (Thermo Fisher; Art. No. 4475936) protocol with the following modifications. Before the libraries were generated 40 ng of the gel-purified RNA was digested with 10U of RNase T1 (Thermos Fisher; Art. No. EN0541) for 1 min at room temperature. After PCI extraction and ethanol precipitation, the RNA was pre-treated with 5 U Antarctic phosphatase (New England Biolabs; Art. No. M0289) for 30 min at 37°C. After heat inactivation at 65°C, the RNA was phosphorylated by 20 U T4 polynucleotide kinase (New England Biolabs; Art. No. M0201) for 60 min at 37°C. Adaptor ligation was carried out for 16 hours at 16°C followed by an incubation of 10 min at 50°C. Reverse transcription (RT) was performed employing SuperScript III with incubations at 42°C, 50°C and 55°C for 45, 15 and 10 min, respectively. The RT reactions were purified and the cDNA was amplified by Platinum PCR SuperMix High Fidelity. Resulting libraries were sequenced using a Ion Proton (Ion Torrent TM) with high-Q.

## Structure determination of core vRNAP

Following sucrose gradient purification, fraction 11 (Figure 1B) was diluted 1:50 and concentrated in a Vivaspin concentrator to a concentration of roughly 50 μg/mL to remove the sucrose. For cryo-EM analysis the sample was centrifuged for 2h at 21,000 g and diluted 1:1 in a buffer containing 20mM HEPES, pH 7.5, 200mM (NH₄)₂SO₄, 1mM MgCl₂ and 5mM 2-mercaptoethanol. 4μl of sample were applied to glow discharged UltrAu 2/2 (Quantifoil) grids at 4°C and 95% humidity in a Vitrobot (FEI Company), blotted for 8.5 s at blot force 14 and plunge-frozen in liquid ethane. Cryo-EM data was collected on a Titan Krios G2 electron microscope (FEI Company) operated at 300 kV with a K2 direct electron detection device operated in counting mode (Gatan) and an energy filter (Gatan) set to a slit width of 15 eV. Movie stacks of 39 frames were acquired with a total dose of 55 e⁻/Å² in counting mode at a nominal magnification of 165,000x, corresponding to a calibrated pixel size of 0.81 Å/pixel. Dose weighting and motion correction was performed using MotionCor2 (Zheng et al., 2017). Per-micrograph contrast-transfer function (CTF) estimation was done using Gctf (Zhang, 2016), as implemented in Relion (Scheres, 2012). A subset of 4,065 particles was manually picked from the micrographs and used for reference-free 2D classification in Relion and the resulting class averages were used to generate reference projections. These were then used as templates for automated particle picking using Gautomatch ([https://www.mrc-lmb.cam.ac.uk/kzhang/](https://www.mrc-lmb.cam.ac.uk/kzhang/)).

A total of 479,618 particles were extracted with a box size of 300 pixels in Relion and subjected to reference-free 2D classification followed by initial global 3D refinement using the B.taurus Pol II elongation complex structure as reference (EMD 3218) (Bernecky et al., 2016), which yielded a reconstruction at 3.1 Å overall resolution (Figure S2). Further 3D classification revealed two distinct states of vRNAP corresponding to ‘open’ and ‘closed’ state clefts, similar to the motion observed previously for Pol II (Cramer et al., 2000; 2001). As the two reconstructions did not show any further differences and the closed state class contained more particles, this class was used for further refinement. Per-particle CTF and motion correction was performed on this particle subset using Warp (Tegunov and Cramer, 2019) and CTF and beam tilt refinement was additionally performed using Relion. The resulting final reconstruction from 3D refinement in Relion achieved an overall resolution of 2.8 Å after post-processing with a sharpening B-factor of –79 Å². This cryo-EM density was of excellent quality, with clear sidechain densities for the majority of the complex and occasional density for bound ions. However, we refrained from modeling ions or waters, with the exception of the catalytic metal ion A as its location and identity can be inferred from previous crystallographic studies as well as the structural zinc ions which are each complexed by four cysteine or histidine residues. In addition to the well-resolved core, the cryo-EM map showed fragmented densities on either side of the vRNAP cleft which were not of sufficient quality for model building. To improve these regions, soft masks encompassing them were cut out from the global reconstruction that was previously low-pass filtered to 10 Å. Focused 3D classification using these masks and the particle subset used in the global refinement was then used to identify particle subpopulations with strong occupancy in the desired region. These particle subpopulations were then subjected to focused 3D refinement, which was initially run without a reference mask until the refinement converged to local searches, from where on the respective mask was provided for alignment of particles within the masked region. Post-processing of these maps was performed in Relion using the same soft masks also used in focused classification and refinement. This approach yielded improved densities for the previously poorly resolved regions.

The initial model of core vRNAP was constructed by docking homology models of Rpo147 and Rpo132 generated by Swissmodel (Biasini et al., 2014) into the cryoEM density, followed by manual rebuilding of all residues in Coot (Emsley et al., 2010). Subunits Rpo35, Rpo22, Rpo19, Rpo18 and Rpo7 were built *de novo* in Coot. The density for the most distal strands of Rpo18 was weak and improved only moderately upon focused classification and refinement, thus indicating potential mobility. Subunit Rpo30 was built *de novo* in the improved map obtained by focused refinement for its binding region. Cross-linking coupled to mass-spectrometry indicated that the initially fragmented densities remaining on either side of the cleft represent Rap94 (Figure S2H), and these regions could be built *de novo* after focused classification and refinement in the respective maps. The Rap94 linker regions L2 and L4

could be partially built de novo in the global reconstruction. After fitting of all models, very weak density remained at the back of vRNAP, which corresponds to the B-homology domain of Rap94. Extensive focused classification and refinements efforts on this region yielded improved maps around the B-ribbon and B-cyclin domains, but these were not of sufficient quality for reliable model building and we thus omitted these parts from the core vRNAP model. In total, the structure contains models for Rpo147 (UniProt B9U1I2; res. 2-207; 217-1268), Rpo132 (UniProt: B9U1Q1; res. 8-122; 126-418; 422-448; 458-789; 797-825; 841-1162), Rpo35 (UniProt: B9U1R2; res. 3-305), Rpo22 (UniProt: B9U1I0; res. 1-184); Rpo19 (UniProt: B9U1M4; res. 61-164), Rpo18 (UniProt: B9U1K4; res. 2-108; 136-159), Rpo7 (UniProt: B9U1G3; res. 2-62), Rpo30 (UniProt: B9U1D1; res. 23-62; 67-151) and Rap94 (UniProt: B9U1I7; res. 106-134; 160-316; 588-619; 627-650; 655-795). The structure was refined using phenix.real_space_refine (Adams et al., 2010) against a composite map generated from the global refinement map and the focused refinement map using phenix.combine_focused_maps by weighting the individual parts according to their cross-correlation with the model. To validate this approach, the model was similarly refined against the locally sharpened density obtained during the Relion local resolution estimation, which yielded comparable final results. The final structure displays excellent stereochemistry, as verified by Molprobity (Chen et al., 2010) (Table S4).

Figures were created with PyMol (Schrodinger, 2015) and UCSF Chimera (Pettersen et al., 2004). Angular distribution plots were created using a tool distributed with Warp (Tegunov and Cramer, 2019). Sequence identity scores in Table S1 were calculated using Ident and Sim (https://www.bioinformatics.org/sms2/ident_sim.html) (Stothard, 2000) with the structure-based alignments as input.

### Structure determination of complete vRNAP

Sample were prepared as for the core vRNAP. For cryo-EM data collection, R 1.2/1.3 holey carbon grids (Quantifoil) were glow discharged for 90 s (Plasma Cleaner model PDC-002. Harrick Plasma Ithaca, NY/USA) at medium power and 3.5 μL of C2 sample was applied inside a Vitrobot Mark IV (FEI) at 4°C and 100% relative humidity. The grids were blotted for 3 s and with blot force 5 and plunged into liquid ethane. The Cryo-EM datasets were collected with a Thermo Fisher Titan Krios G3 and a Falcon III camera (Thermo-Fischer). Data was acquired with EPU at 300 keV and a primary magnification of 75,000 (calibrated pixel size 1.0635 Å) in movie-mode with 25 fractions per movie and integrating the electron-signal. The total exposure was 50 e/Å² over an exposure time of 4.5 s with 2 exposures per hole.

Dose-weighted, motion-corrected sums of the micrograph movies were calculated with Motioncorr2 (Zheng et al., 2017). The contrast-transfer function of each micrograph was fitted with CTFFind4 (Rohou and Grigorieff, 2015). An initial set of 1,500 particles was selected manually and subjected to a 2D-Classification in Relion3-beta (Zivanov et al., 2018). 12 reasonable class averages were selected as templates for subsequent automated particle picking within Relion and 256,452 particles were picked from 2,224 micrographs. The dataset was then cleaned up by four cycles of 2D classification and particle sorting followed by manual selection of classes based on the appearance of their class averages resulting in a final dataset of 190,000 good particles. A subset of 20,000 particles was used to generate an initial model. An initial 3D classification with Relion yielded two major classes which differed obviously in the density for VTF/CE, and were subjected to 3D refinement. The class of the large particle yielded a 3.3 Å reconstruction. A second round of automated particle picking was performed with projections from the reconstruction of the large particle as picking templates and yielded a dataset of 858,702 particles. This dataset was then cleaned up by four cycles of 2D classification and particle sorting followed by manual particle selection resulting in a final dataset of 618,338 good particles. A 3D classification of this dataset yielded only highly similar classes and the reconstruction using the full, unclassified dataset yielded the highest resolution of 2.98 Å. With further per-particle CTF refinement including a per-dataset beam-tilt refinement and per-particle motion-correction ('polishing') within Relion3 we obtained a reconstruction with 2.75 Å resolution.

For model building and refinement, the complete vRNAP density was unambiguously docked with the previously built core vRNAP model, the crystallographic models of VTF/CE (PDB: 4CKB) (Kyrieleis et al., 2014), the E11 homodimer and bacterial tRNA<sup>Gln</sup> extracted from PDB entry 1GSG (Rould et al., 1989). The residual density for VETF-I<sup>365-436</sup>, NPH-I, and Rap94 was assigned and traced manually within Coot (Emsley et al., 2010) with the guidance of secondary structure predictions from PsiPred (Jones, 1999) and XLMS data. The final model was refined with Phenix.real_space_refine including an ADP refinement step. During refinement secondary structure, mild Ramachandran and reference model restraints from the VTF/CE and E11 crystallographic models were imposed. After a further cycle of manual inspection and automated refinement, water molecules were placed with Coot and a final round of refinement with Phenix.real_space_refine was applied.

### X-Ray structure determination of E11

Bacterially overexpressed, hexa-histidine tagged E11 protein was bound to Ni-NTA-Agarose, eluted with 200 mM Imidazole and dialysed against TBS. The tag was cleaved with tobacco etch virus protease and a final gel filtration chromatography was performed. Crystals were obtained with the hanging drop vapor diffusion method with reservoir solution containing 20% PEG 4000. For crystallographic phase determination the crystals were derivatized with sodium ethylmercurithiosalicylate and a SAD experiment was performed at beamline MX1/P13 of the PETRA III storage ring of the Deutsches Elektronen-Synchrotron (DESY). Phasing and initial model building were performed with Phenix.autosol. The model was then refined against a native dataset collected at the same beamline with Phenix.refine and completed manually within Coot. After three more cycles of manual corrections and automated refinement including water placement and TLS refinement, the R-factors converged.

QUANTIFICATION AND STATISTICAL ANALYSIS

No statistical methods were used to predetermine sample size. The experiments were not randomized, and the investigators were not blinded to allocation during experiments and outcome assessment.

DATA AND CODE AVAILABILITY

Coordinate files for the core and complete vRNAP complex structures were deposited with the Protein Data Bank and are available under accession codes 6RIC and 6RFL, respectively. The cryo-EM density maps for the core and complete vRNAP complexes were deposited with the Electron Microscopy Data Base (EMDB) and are available under accession codes EMD-4888 and EMD-4868, respectively. Coordinate and structure files for the E11 crystal structure were deposited with the Protein Data Bank and are available under accession code 6RFG.

Supplemental Figures

**A**

GLV-1h68


    F14.5L      J2R       A56R
    PE/L        TFR       lacZ
    RUC-GFP     PE/L P7.5
                gusA      P11


GLV-1h439


    F14.5L      J2R       A24R       A56R
    PE/L        TFR       HA-FLAG-Tag   P11
    RUC-GFP     PE/L P7.5  gusA
                lacZ


**B**


    10⁷
    |
    |
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ────
    |           ─
Cell

A  
![](image1.png)  

B  
![](image2.png)  

C  
3608 micrographs  
Autopicking (Gautomatch) → 479,015 good particles  
Manual clean-up  
2D classification  
Global 3D refinement  
3.1 Å  
Global 3D classification  
closed cleft  
open cleft  
11%  
51,999 particles  
11.4%  
53,696 particles  
8.5%  
40,866 particles  
31.8%  
152,591 particles  
14.8%  
70,948 particles  
22.7%  
108,915 particles  
3D refinement post-processing  
Warp CTF estimation  
CTF refinement (Relion)  
2.8 Å global reconstruction  

D  
Rap94 CTD mask  
Rpo30 mask  
Rap94 D2 mask  
Focused classification and refinement  
Rap94 D2 mask  
Rap94 CTD mask  
Rpo30 mask  
3.3 Å  
3.0 Å  
3.2 Å  

E  
Fourier Shell Correlation  
Global Rap94 CTD mask  
Rap94 D2 mask  
Rpo30 mask  
FSC = 0.5  
FSC = 0.143  
3.3 Å  
3.2 Å  
3.0 Å  
2.8 Å  
Resolution [1/Å]  

F  
No. of particles  
4918  
3278  
1639  
1  

G  
Top view  
Back view  
4.8 Å  
4.3 Å  
3.8 Å  
3.3 Å  
2.8 Å  

H  
Back view  
1  
K667  
K109  
2  
K206  
K177  
K204  
K70  
K148  
3  
K316  
K326  

(legend on next page)

Figure S2. Structure Determination of Core vRNAP, Related to Figure 2

(A) Exemplary cryo-EM micrograph of the core vRNAP dataset.

(B) The 32 best aligning class averages from unsupervised 2D classification.

(C) Cryo-EM processing workflow for structure determination.

(D) Focused classification and refinement workflow for improved local maps.

(E) Fourier Shell Correlation (FSC)-plots for cryo-EM reconstructions used.

(F) Angular distribution plot for the global reconstruction of core vRNAP.

(G) Local resolution estimation for the global reconstruction of core vRNAP as implemented in Relion.

(H) Bis(sulfosuccinimidyl)suberate (BS3) crosslinks identified by mass spectrometry used for positioning of Rap94 domains. (Left) Overview of the core vRNAP structure with regions where strong crosslinks occurred indicated. (Indent 1-3) Proteins are shown in cartoon representation with coloring as in Figure 2. Crosslinked lysine residues are shown as sticks. Selected strong crosslinks are shown as lines.

A

Vaccinia Rpo147
1 77 190 282 311 370 441 574 726 787 921 996 1096 1207 1246 1286
Clamp Clamp Clamp Dock Pore 1 Funnel Cleft Foot Cleft Jaw Cleft Clamp
core head core Active site Active site core

S.cerevisiae Rpb1
1 95 235 346 375 436 508 663 809 871 1059 1141 1275 1395 1436 1535 1733
Clamp Clamp Clamp Dock Pore 1 Funnel Cleft Foot Cleft Jaw Cleft Linker CTD
core head core

B

Vaccinia Rpo147 S.cerevisiae Rpb1
1 10 20 30 40 50 60 70 80 90 100 110 120 130 140 150 160 170 180 190 200 210 220 230 240 250 260 270 280 290 300 310 320 330 340 350 360 370 380 390 400 410 420 430 440 450 460 470 480 490 500 510 520 530 540 550 560 570 580 590 600 610 620 630 640 650 660 670 680 690 700 710 720 730 740 750 760 770 780 790 800 810 820 830 840 850 860 870 880 890 900 910 920 930 940 950 960 970 980 990 1000 1010 1020 1030 1040 1050 1060 1070 1080 1090 1100 1110 1120 1130 1140 1150 1160 1170 1180 1190 1200 1210 1220 1230 1240 1250 1260 1270 1280 1290 1300 1310 1320 1330 1340 1350 1360 1370 1380 1390 1400 1410 1420 1430 1440 1450 1460 1470 1480 1490 1500 1510 1520 1530 1540 1550 1560 1570 1580 1590 1600 1610 1620 1630 1640 1650 1660 1670 1680 1690 1700 1710 1720 1730 1740 1750 1760 1770 1780 1790 1800 1810 1820 1830 1840 1850 1860 1870 1880 1890 1900 1910 1920 1930 1940 1950 1960 1970 1980 1990 2000 2010 2020 2030 2040 2050 2060 2070 2080 2090 2100 2110 2120 2130 2140 2150 2160 2170 2180 2190 2200 2210 2220 2230 2240 2250 2260 2270 2280 2290 2300 2310 2320 2330 2340 2350 2360 2370 2380 2390 2400 2410 2420 2430 2440 2450 2460 2470 2480 2490 2500 2510 2520 2530 2540 2550 2560 2570 2580 2590 2600 2610 2620 2630 2640 2650 2660 2670 2680 2690 2700 2710 2720 2730 2740 2750 2760 2770 2780 2790 2800 2810 2820 2830 2840 2850 2860 2870 2880 2890 2900 2910 2920 2930 2940 2950 2960 2970 2980 2990 3000 3010 3020 3030 3040 3050 3060 3070 3080 3090 3100 3110 3120 3130 3140 3150 3160 3170 3180 3190 3200 3210 3220 3230 3240 3250 3260 3270 3280 3290 3300 3310 3320 3330 3340 3350 3360 3370 3380 3390 3400 3410 3420 3430 3440 3450 3460 3470 3480 3490 3500 3510 3520 3530 3540 3550 3560 3570 3580 3590 3600 3610 3620 3630 3640 3650 3660 3670 3680 3690 3700 3710 3720 3730 3740 3750 3760 3770 3780 3790 3800 3810 3820 3830 3840 3850 3860 3870 3880 3890 3900 3910 3920 3930 3940 3950 3960 3970 3980 3990 4000 4010 4020 4030 4040 4050 4060 4070 4080 4090 4100 4110 4120 4130 4140 4150 4160 4170 4180 4190 4200 4210 4220 4230 4240 4250 4260 4270 4280 4290 4300 4310 4320 4330 4340 4350 4360 4370 4380 4390 4400 4410 4420 4430 4440 4450 4460 4470 4480 4490 4500 4510 4520 4530 4540 4550 4560 4570 4580 4590 4600 4610 4620 4630 4640 4650 4660 4670 4680 4690 4700 4710 4720 4730 4740 4750 4760 4770 4780 4790 4800 4810 4820 4830 4840 4850 4860 4870 4880 4890 4900 4910 4920 4930 4940 4950 4960 4970 4980 4990 5000 5010 5020 5030 5040 5050 5060 5070 5080 5090 5100 5110 5120 5130 5140 5150 5160 5170 5180 5190 5200 5210 5220 5230 5240 5250 5260 5270 5280 5290 5300 5310 5320 5330 5340 5350 5360 5370 5380 5390 5400 5410 5420 5430 5440 5450 5460 5470 5480 5490 5500 5510 5520 5530 5540 5550 5560 5570 5580 5590 5600 5610 5620 5630 5640 5650 5660 5670 5680 5690 5700 5710 5720 5730 5740 5750 5760 5770 5780 5790 5800 5810 5820 5830 5840 5850 5860 5870 5880 5890 5900 5910 5920 5930 5940 5950 5960 5970 5980 5990 6000 6010 6020 6030 6040 6050 6060 6070 6080 6090 6100 6110 6120 6130 6140 6150 6160 6170 6180 6190 6200 6210 6220 6230 6240 6250 6260 6270 6280 6290 6300 6310 6320 6330 6340 6350 6360 6370 6380 6390 6400 6410 6420 6430 6440 6450 6460 6470 6480 6490 6500 6510 6520 6530 6540 6550 6560 6570 6580 6590 6600 6610 6620 6630 6640 6650 6660 6670 6680 6690 6700 6710 6720 6730 6740 6750 6760 6770 6780 6790 6800 6810 6820 6830 6840 6850 6860 6870 6880 6890 6900 6910 6920 6930 6940 6950 6960 6970 6980 6990 7000 7010 7020 7030 7040 7050 7060 7070 7080 7090 7100 7110 7120 7130 7140 7150 7160 7170 7180 7190 7200 7210 7220 7230 7240 7250 7260 7270 7280 7290 7300 7310 7320 7330 7340 7350 7360 7370 7380 7390 7400 7410 7420 7430 7440 7450 7460 7470 7480 7490 7500 7510 7520 7530 7540 7550 7560 7570 7580 7590 7600 7610 7620 7630 7640 7650 7660 7670 7680 7690 7700 7710 7720 7730 7740 7750 7760 7770 7780 7790 7800 7810 7820 7830 7840 7850 7860 7870 7880 7890 7900 7910 7920 7930 7940 7950 7960 7970 7980 7990 8000 8010 8020 8030 8040 8050 8060 8070 8080 8090 8100 8110 8120 8130 8140 8150 8160 8170 8180 8190 8200 8210 8220 8230 8240 8250 8260 8270 8280 8290 8300 8310 8320 8330 8340 8350 8360 8370 8380 8390 8400 8410 8420 8430 8440 8450 8460 8470 8480 8490 8500 8510 8520 8530 8540 8550 8560 8570 8580 8590 8600 8610 8620 8630 8640 8650 8660 8670 8680 8690 8700 8710 8720 8730 8740 8750 8760 8770 8780 8790 8800 8810 8820 8830 8840 8850 8860 8870 8880 8890 8900 8910 8920 8930 8940 8950 8960 8970 8980 8990 9000 9010 9020 9030 9040 9050 9060 9070 9080 9090 9100 9110 9120 9130 9140 9150 9160 9170 9180 9190 9200 9210 9220 9230 9240 9250 9260 9270 9280 9290 9300 9310 9320 9330 9340 9350 9360 9370 9380 9390 9400 9410 9420 9430 9440 9450 9460 9470 9480 9490 9500 9510 9520 9530 9540 9550 9560 9570 9580 9590 9600 9610 9620 9630 9640 9650 9660 9670 9680 9690 9700 9710 9720 9730 9740 9750 9760 9770 9780 9790 9800 9810 9820 9830 9840 9850 9860 9870 9880 9890 9900 9910 9920 9930 9940 9950 9960 9970 9980 9990 10000 10010 10020 10030 10040 10050 10060 10070 10080 10090 10100 10110 10120 10130 10140 10150 10160 10170 10180 10190 10200 10210 10220 10230 10240 10250 10260 10270 10280 10290 10300 10310 10320 10330 10340 10350 10360 10370 10380 10390 10400 10410 10420 10430 10440 10450 10460 10470 10480 10490 10500 10510 10520 10530 10540 10550 10560 10570 10580 10590 10600 10610 10620 10630 10640 10650 10660 10670 10680 10690 10700 10710 10720 10730 10740 10750 10760 10770 10780 10790 10800 10810 10820 10830 10840 10850 10860 10870 10880 10890 10900 10910 10920 10930 10940 10950 10960 10970 10980 10990 11000 11010 11020 11030 11040 11050 11060 11070 11080 11090 11100 11110 11120 11130 11140 11150 
Figure S3. Structure-Based Sequence Alignment of Rpo147 and S. cerevisiae Rpb1, Related to Figure 3

(A) Schematic depiction of Vaccinia Rpo147 and the homologous S. cerevisiae Pol II subunit Rpb1 with domains indicated. Insertions and deletions are indicated by connecting lines, with differing regions shown with dashed lines. Regions with differing fold are indicated by crossed connecting lines.

(B) Structure-based sequence alignment with secondary structure elements depicted and colored according to domains as in Figures 2A and 2C. Sheet regions are shown as arrows, helical region as cylinders. Invariant residues are colored in dark blue and conserved residues in light blue. Regions differing in fold are colored in green (vRNAP-specific) and red (Pol II-specific). The alignment was generated with MSAProbs (Liu et al., 2010) within the MPI Bioinformatics Toolkit (Zimmermann et al., 2018), visualized using Aline (Bond and Schüttelkopf, 2009) and manually edited by comparison to the S.cerevisiae Pol II structure (PDB 1WCM) (Armache et al., 2005). In Rpo147, helices α8 and α9 in the polymerase clamp core domain are shortened. Helices α27, α28, α32 and α34, which are located in the foot domain of Rpb1, are absent. The jaw domain is substantially reduced, lacking Rpb1 regions 1158-1188 and 1245-1253.

A

Vaccinia Rpo132
S.cerevisiae Rpb2

External 1
Protrusion
Lobe
Fork
Hybrid Binding
Wall
Hybrid Binding
Anchor Clamp Tail

External 2 External 1

B

Vaccinia Rpo132
S.cerevisiae Rpb2

MSDLANSEKYyDEDPYGFE
10
DESAPI
30
SWAVISAFRE
50
KGLVSQOLD
70
SFNQFVDYTLQDIIICEDSTLLLEQL
90
AQHTTESDNISRKYEI
110
SFGKIYTKM
130
VNESDGVTHALYPQEARNLITYSSGLFVDKKRTYEADVPGRELKYLEIAE
150
ESEDDS
170
EGKDSHLIKIPLLIGYGNKNPLDTA
190
...
KYLVPNVIGGVFINKQSVEKVGINLVKEKITWWPKFRVVKP
210
...
ESDLYK
230
LKECPFDMGGYFINGSSEKVLIAQERSAGNI.VQVFKKAAPSP
250
NSFTFSFSVSVPNPTRRYHKISLDISQ
270
...
LEALNISTKTFITVNIVLLSQYLSRVSLEFRRSLSYD.MPPEV
290
ISHVAEIRSALEK...GSBFISTLQVKLYGR
310
EGSS
330
IKATLPYIKQDIPIVIFRAEGILPDGEILEHICYDVNDWQM
350
VYLVNAI
370
IDSAKRITESITDFNDITYINDLV
390
EAEHIKKKSQ..LTINEFKYEMLHNFLPHMNYTP..DQLKGFYMISLLR
410
LEMALKPCVEDG
430
...
FVIQDRETALDFIGRRGTALGIKKEKRIQYAKDILQKEFLPHITQLEGFE
450
SRKAFFFLGYMIN
470
KFLYC
490
IFHTSRYPDRDSMVCHRILTYGKYFETLAHDELENYIGN
510
RNDIMNNHKNRGTYAVNIVLTTTPGLNHAFSSLS
530
RLLC
550
ALDRKDQQDDRDFGKKRLDLAQFLKTLFKKLTKDIFFRYMQRTVEEAHD
570
FNMKLAINAKTITSGLKYAAT
590
GKFK
610
...
KSDGSYRTHPHYSWMQNISIPRSVGFPDQVKIS
630
KMFSVRKYHPSQYLIFCSSDVPERGPQVGLVSQLS
650
GNWGEQKKAMSSRAGVSQVLNRYTYSTLSHLRRNTNPIGRDG..
670
KLAKPRQLHNTHWGLVCPAETPEGQACGLVKNL
690
VLSSITN
710
ILTSEYLDLEKK
730
ICEYIRSYKDDIS....
750
YFETGFPFITENALVASLNPNMICDFVTDFRRKRKMGFFGNL
770
LMSCISV
790
...
GTDPMPITFLSEWGMEPLEDYPHPQS
810
PDATRVFVNGVWHGVHR..NPARLMETLATLLRRKG.DINP
830
EVGITLVRDHMN
850
EIRNIGAGRVRPF
870
FVDNGELMM...
890
DVCEPELSRLDD...
910
MTFSDIQKEFPH
930
EVSMIRDIREKELKIF
950
FTDAGRVRPFL
970
IVEDDESLGHKELKVRKGHIAKLMATEYQOIEGGFEDVEEYTWSSLNE..
990
VIMEVDIB
1010
QFTVESNVCESVQKFRMM...
1030
SKDERQYDLCDFPAEFRDGYVASL
1050
LVGINHNSQPRAIIPFPDHNQSDBRN
1070
IVEYIDAEEESILIAMQPEDLEPA
1090
EANEENDLDVDPAKRIRVSHHATTFTHEIHPSMILGVAASII
1110
LQSA
1130
LGCAQA
1150
KQAISCLSSDIRNKID.NGIHMYPERPIVISKALETSDKIAANCF
1170
GQHVTLAMSYGINKQEDGIITIKKQFIQR
1190
YQSA
1210
MGKQAMGVFLTNYNYVRMDTMANI
1230
LYYQKPLGTTTRAMEYLK
1250
FRELPAQONAIVAAACYSGYNQEDSMIMNQSSIDR
1270
GGLDIV
1290
VTAKKHQVEIPL
1310
LENFN....
1330
NKEEDRSNAYSKEESNGLVRLLNAFLESGDAMARNISSRTL
1350
EDDFARD
1370
GLFRSLFFRSYMDQEEK
1390
KYGMSTITETFEKPQRTN
1410
NTLRMKHGTYDKLDDDG
1430
LIAPGVRVSGEDVIIIGKT
1450
PISPDEEEELGQR
1470
N
1490
...
QISFDVSEKYTDMYKSRVERVQVELT.DKVKV
1510
RLTMKERRP
1530
ILGDKFTTTRTS
1550
QKGTVAYVADETELPYDENGI
1570
TP
1590
TAYHSKRDASTPLRS
1610
TENGIVDQVL
1630
LVTTNQDGLKFVKVRRTTKI
1650
PQIGDKFASRHG
1670
QKGTIGITYRREDMPFTAEGIVP
1690
DVINSTS
1710
FSRKTISM
1730
LEVILTAAYSAKPY
1750
NNKGERNPVCFPSSNETSIDTYMQFAK
1770
QCYEHSNPKLSDDEELSDKIFC
1790
DLINPHA
1810
PSRMTVAH
1830
IELCLS
1850
SKVAALSGN...
1870
EGDASPFTDITVEGISKLLR
1890
EHGY.............
1910
QSRG
1930
EKILYDPE
1950
TDKPYASKVFFFGPIYYLR
1970
RLHLTQDKATVR
1990
CRGKTKKL
2010
IROAN
2030
EGRKRGGIKFGE
2050
MERDCLI
2070
AHGAANTIT
2090
FEMYNGHTG
2110
KKLMAQIFFFGPTYYQ
2130
RLRHMVDDKIHA
2150
ARGPMQV
2170
LTRQPV
2190
EGERSR
2210
DGGLRFGE
2230
MERDCMI
2250
AHGAASFLK
2270
EVLK
2290
DSEEDYQDMYVC
2310
ENC
2330
GDI
2350
AAQIK
2370
GINTCLRC
2390
SKLNL
2410
SPL
2430
LTK
2450
OTTTHVS
2470
KVFLTQMNA
2490
RGVKV
2510
KLDFER
2530
RPPSFYK
2550
ERLMEASDAF.RMHICO
2570
GLMTVIAK
2590
LNHNQFEC
2610
KGCDNKIDIYQ
2630
HIPYAAK
2650
KLLLFQELMAMNITPRL
2670
YTDRSRDF...
2690
PLDKVDLKPSFLV
1160
1164
...
1224
(legend on next page)

Figure S4. Structure-Based Sequence Alignment of Rpo132 and S. cerevisiae Rpb2, Related to Figure 3.

(A) Schematic depiction of Vaccinia Rpo132 and the homologous S. cerevisiae Pol II subunit Rpb2 with domains indicated. Insertions and deletions are indicated by connecting lines, with differing regions shown with dashed lines. Regions with differing fold are indicated by crossed connecting lines.

(B) Structure-based sequence alignment with secondary structure elements depicted and colored according to domains as in [Figures 2A and 2C](#). Sheet regions are shown as arrows, helical region as cylinders. Invariant residues are colored in dark blue and conserved residues in light blue. Regions differing in fold are colored in green (vRNAP-specific) and red (Pol II-specific). The alignment was generated with MSAProbs ([Liu et al., 2010](#)) within the MPI Bioinformatics Toolkit ([Zimmermann et al., 2018](#)), visualized using Aline ([Bond and Schüttelkopf, 2009](#)) and manually edited by comparison to the S. cerevisiae Pol II structure (PDB 1WCM) ([Armache et al., 2005](#)). Helices α7 and α8 in the lobe domain are extended in the Rpo132. In the protrusion domain, the region between α11 and α12 differs between the yeast and viral proteins. The most prominent differences are located in the external domains, in particular in the regions between β16 and α16 and α17, and between α19 and β24. The region following β28 (res. 784-797), which contacts upstream DNA in the yeast Pol II ([Barnes et al., 2015](#)), is reduced and adopts a different conformation in the viral enzyme.

A

Vaccinia Rpo35
S.cerevisiae Rpb3
S-cerevisiae Rpb11

Vaccinia Rpo35
S.cerevisiae Rpb3
S-cerevisiae Rpb11

Vaccinia Rpo35
S.cerevisiae Rpb3
S-cerevisiae Rpb11

Vaccinia Rpo35
S.cerevisiae Rpb3
S-cerevisiae Rpb11

Vaccinia Rpo7
S.cerevisiae Rpb10

B

Vaccinia
Rpo22
S.cerevisiae
Rpb5

Vaccinia
Rpo19
S.cerevisiae
Rpb6

Vaccinia
Rpo18
S.cerevisiae
Rpb7

(legend on next page)

Figure S5. Structure-Based Sequence Alignment of Rpo35, Rpo22, Rpo19, Rpo18, and Rpo7 with Corresponding *S. cerevisiae* Pol II Subunits, Related to Figure 3

Structure-based sequence alignments with secondary structure elements depicted and colored according to domains as in [Figure 3](#). Sheet regions are shown as arrows, helical region as cylinders. Invariant residues are colored in dark blue and conserved residues in light blue. Regions differing in fold are colored in green (vRNAP-specific) and red (Pol II-specific). The alignment was generated with MSAProbs ([Liu et al., 2010](#)) within the MPI Bioinformatics Toolkit ([Zimmermann et al., 2018](#)), visualized using Aline ([Bond and Schüttelkopf, 2009](#)) and manually edited by comparison to the *S. cerevisiae* Pol II structure (PDB 1WCM) ([Armache et al., 2005](#)).

(A) Schematic depiction of *Vaccinia* Rpo35 and Rpo7 and the homologous *S. cerevisiae* Pol II subunits Rpb3, Rpb11 and Rpb10 with domains indicated and structure-based sequence alignment between the proteins. Insertions and deletions are indicated by connecting lines, with differing regions shown with dashed lines. Regions with differing fold are indicated by crossed connecting lines. The region resembling the non-conserved domain of Rpb3 responsible for interactions with Rpb10 and Rpb12 is reduced in Rpo35, with the Zn-binding motif lacking altogether.

(B) Schematic depiction of *Vaccinia* Rpo22, Rpo19 and Rpo18 and the homologous *S. cerevisiae* Pol II subunits Rpb5, Rpb6 and Rpb7 with domains indicated and structure-based sequence alignments. Depiction as in (A). Like Rpb7, Rpo18 binds to the polymerase core via its K1 helical turn and its tip loop in the amino terminal tip domain. These elements form a wedge between the N-terminal region of Rpo147, the switch 5 region, the Rpo132 anchor, and helix α1 of Rpo19, all of which are conserved between *Vaccinia* and Pol II. The Rpo18 tip domain may therefore restrict movement of the clamp, as proposed for Rpb7 in Pol II ([Armache et al., 2003](#)). The C-terminal domain of Rpo19 forms a β-barrel-like structure but appears tilted toward the polymerase body compared to Rpb4/7.

Cell

B

A

C
2224
micrographs
(Relion)
Autopicking
858,702
particles
2D and 3D
classification
3D refinement
post-processing
618,338
good particles
2.8 Å
reconstruction
single, homogenous class

D
front view
side view
back view
2.6 Å
3.2 Å
3.8 Å
4.4 Å
5.0 Å

E

F
12476
8317
4159
1
Fourier Shell Correlation
1.0
FSC=0.500
0.8
0.6
FSC=0.143
0.4
0.2
0.0
Resolution [1/Å]
(legend on next page)

Figure S6. Structure Determination of Complete vRNAP, Related to Figure 4

(A) Exemplary cryo-EM micrograph of the complete vRNAP complex dataset.

(B) Selected Class averages from unsupervised 2D classification in Relion.

(C) Cryo-EM processing workflow for structure determination.

(D) Local resolution estimates mapped to the cryo EM density isosurface representation.

(E) Angular particle orientation map.

(F) Fourier Shell Correlation (FSC)-plot.

A

Vaccinia Rpo30
S.cerevisiae TFIIS
QDHAPGNAEDKTTVGESVNGVQQPASSQSDAMKQDKYVSTKPRNSKNDGVDTAIYHHKLRDQLV...KALYDV...LAKESEHPPQSIL
90          100           110            120             130              140               150                160                 170

MENVYISSYSSNEQTSMAVAA TDIRELLSQYVDDANEDLI EWAMEKSSKYYIKNILG
10           20            30             40              50               57

a1
a2

T
a1

a3
a4
a5

Vaccinia Rpo30
S.cerevisiae TFIIS
NTKSNIETKFESKNIGIEYSKDSRNKLSY...RNRPSIATNLEYKTLCDMIKGSTEKEFLRYLFGIKCKKGVEYNIDKIK.
HTAKAISEMNKVNNCDTNEAAYKARYRIYSNVISKNPDLKHKIANGDITPEFLATCDAKDLAAPPKQKIEEIAKQNLYNAQGATI
60          70           80            90             100              110               120                130                 140

a2
a3
a4
a5
a6
a7

Vaccinia Rpo30
S.cerevisiae TFIIS
DVSYNDYFNVLDEKYNTPCPNCKSRNTTPMMIQTRAADEPPLVRHACKQHKPPKFRARNLNVTTSIHENKEITEILPDNNPSP
ERSVTDRFT...CGKKKEKKVSYYQLQTRSAADEPLTTFCTCEACGNRWKFS...
150         160        170         180        190         200        210         220        261

Zn-binding
Acidic loop
Zn-
binding

C-tail (resolved in complete vRNAP)

Vaccinia Rpo30
S.cerevisiae TFIIS
PESPEPASPIDDGLIRATFDRNDEPPEDDE
230         240        250         259
309


B


C

C-Terminus
N-Lobe
Brace-Helices
DNA
Brace-Helix
Protrusion II
C-Lobe
NPH-I

N-Lobe
DNA
C-Lobe
Protrusion II
INO80

N-Lobe
DNA
C-Lobe
Protrusion II
SNF2

Figure S7. Sequence Alignment of Rpo30 and S. cerevisiae TFIIS and Structural Details of NPH-I and E11, Related to Figures 4, 5, and 6
(A) Structure-based sequence alignment of Rpo30 and S. cerevisiae TFIIS with secondary structure elements depicted and colored according to domains as in Figure 6. Sheet regions are shown as arrows, helical region as cylinders. Invariant residues are colored in dark blue and conserved residues in light blue. Regions differing in fold are colored in green (vRNAP-specific) and red (Pol II-specific). The alignment was generated with MSAProbs (Liu et al., 2010) within the MPI Bioinformatics Toolkit (Zimmermann et al., 2018), visualized using Aline (Bond and Schüttelkopf, 2009) and manually edited by comparison to the S. cerevisiae
(legend continued on next page)

Pol II structure (PDB 1WCM) ([Armache et al., 2005](https://doi.org/10.1038/nature04009)). The Zink-binding regions are highlighted in pink and the conserved acidic residues of TFIIIS that enter the Pol II active site (DEP motif) are highlighted in green.

(B) Fold and topology of the E11 crystal structure. Topology (left). Fold and secondary structure elements in cartoon style (right). The two protomers of the homodimer are in orange and yellow, respectively.

(C) Comparison of the ATPase domains of NPH-I to those of the chromatin remodelers INO80 (PDB 6FHS) ([Eustermann et al., 2018](https://doi.org/10.1073/pnas.1804845115)) and SNF2 (from PDB ID 5X0X) ([Liu et al., 2017](https://doi.org/10.1073/pnas.1704558114)). The characteristic structural elements are color-coded and labeled.
