
Cell Host & Microbe

Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody

Graphical Abstract

Authors
Fanli Yang, Sheng Lin, Fei Ye, ..., Yuquan Wei, George F. Gao, Guangwen Lu

Correspondence
gaof@im.ac.cn (G.F.G.), lugw@scu.edu.cn (G.L.)

In Brief
Yang et al. report the crystal structures of rabies virus glycoprotein in the free form at ~pH-8.0 and in the antibody-bound form at ~pH-6.5. This structural analysis delineates the conformational changes induced by low pH and the recognition of neutralizing antibodies, specifically detailing the interaction with RABV-specific antibody 523-11.

Highlights
- RABV-G ecto-domain protein structure is revealed using a fusion-loop substitution strategy
- Crystal structures of RABV-G at both basic and acidic pHs are solved
- Structural comparison reveals basis of pH-dependent structural transitions in RABV-G
- Complex structure shows 523-11 antibody binds a bipartite epitope for neutralization

Yang et al., 2020, Cell Host & Microbe 27, 1–13
March 11, 2020 © 2019 Elsevier Inc.
https://doi.org/10.1016/j.chom.2019.12.012

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

**Cell Host & Microbe**

**Article**

# Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody

Fanli Yang,$^{1,8}$ Sheng Lin,$^{1,8}$ Fei Ye,$^{1,8}$ Jing Yang,$^{1,8}$ Jianxun Qi,$^{2,8}$ Zhujun Chen,$^{1}$ Xi Lin,$^{1}$ Jichao Wang,$^{1}$ Dan Yue,$^{1}$ Yanwei Cheng,$^{1}$ Zimin Chen,$^{1}$ Hua Chen,$^{1}$ Yu You,$^{1}$ Zhonglin Zhang,$^{1}$ Yu Yang,$^{1}$ Ming Yang,$^{1}$ Honglu Sun,$^{1}$ Yuhua Li,$^{3}$ Yu Cao,$^{1,4}$ Shengyong Yang,$^{1}$ Yuquan Wei,$^{1}$ George F. Gao,$^{2,5,6,7,*}$ and Guangwen Lu$^{1,9,**}$

$^{1}$West China Hospital Emergency Department (WCHED), State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Collaborative Innovation Center of Biotherapy, Chengdu, Sichuan 610041, China  
$^{2}$CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China  
$^{3}$Division of Arboviral Vaccines, National Institutes of Food and Drug Control (NIFDC), Beijing 100050, China  
$^{4}$Disaster Medicine Center, West China Hospital, Sichuan University, Chengdu, Sichuan 610041, China  
$^{5}$Savaid Medical School, University of Chinese Academy of Sciences, Beijing 100049, China  
$^{6}$National Institute for Viral Disease Control and Prevention, Chinese Center for Disease Control and Prevention (China CDC), Beijing 102206, China  
$^{7}$Research Network of Immunity and Health (RNIH), Beijing Institutes of Life Science, Chinese Academy of Sciences, Beijing 100101, China  
$^{8}$These authors contributed equally  
$^{9}$Lead Contact  

*Correspondence: gaof@im.ac.cn (G.F.G.), lugw@scu.edu.cn (G.L.)  
https://doi.org/10.1016/j.chom.2019.12.012*

## SUMMARY

Rabies virus (RABV), the etiological agent for the lethal disease of rabies, is a deadly zoonotic pathogen. The RABV glycoprotein (RABV-G) is a key factor mediating virus entry and the major target of neutralizing antibodies. Here, we report the crystal structures of RABV-G solved in the free form at ~pH-8.0 and in the complex form with a neutralizing antibody 523-11 at ~pH-6.5, respectively. RABV-G has three domains, and the basic-to-acidic pH change results in large domain re-orientations and concomitant domain-linker re-constructions, switching it from a bent hairpin conformation into an extended conformation. During such low-pH-induced structural transitions, residues located in the domain-linker are found to play important roles in glycoprotein-mediated membrane fusion. Finally, the antibody interacts with RABV-G mainly through its heavy chain and binds to a bipartite conformational epitope in the viral protein for neutralization. These structures provide valuable information for vaccine and drug design.

## INTRODUCTION

Within the *Lyssavirus* genus of the *Rhabdoviridae* family (*Amara-singhe et al.*, 2018; *Lyles et al.*, 2013), rabies virus (RABV) is a prototypic member of the genus and by far the most common causative agent of human rabies (*Fooks et al.*, 2017). The virus is readily transmitted through the bite of an infected animal, most commonly a dog, and can infect and replicate in the central nervous system, subsequently leading to severe neurological diseases (*Hemachudha et al.*, 2013; *Lafon*, 2011). Clinical features of RABV infection typically include multiple neuronal dysfunctions, manifestations of which almost inevitably lead to death (*Hemachudha et al.*, 2013; *Ugolini and Hemachudha*, 2018). Although efficacious rabies vaccines are available, the disease remains a major public health concern, causing an estimated 60,000 deaths worldwide each year (*Fooks et al.*, 2014; *Hampson et al.*, 2015).

RABV contains a single-stranded negative-sense RNA genome that encodes five structural proteins: nucleoprotein (N), phosphoprotein (P), matrix protein (M), glycoprotein (G), and RNA-dependent RNA polymerase (L) (*Lyles et al.*, 2013). Among these viral proteins, the glycoprotein of RABV (RABV-G) plays a pivotal role in the pathogenesis of the virus by mediating both viral recognition of and attachment to cellular receptors (e.g., heparan sulfate [*Sasaki et al.*, 2018], nicotinic acetylcholine receptor (nAchR) [*Lentz et al.*, 1982], neural cell adhesion molecule [*Thoulouze et al.*, 1998], neurotrophin receptor p75NTR [*Tuffereau et al.*, 1998], and metabotropic glutamate receptor subtype 2 [*Wang et al.*, 2018a]) and the fusion of the viral envelope with the cell membrane after virion endocytosis and acidification of the endosome lumen, thereby initiating the entry of the virus particles into host cells (*Albertini et al.*, 2012a). In addition, as the only protein present on the surface of the virus, RABV-G is also the major target for neutralizing antibodies (*Flamand et al.*, 1993; *Prehaud et al.*, 1988; *Seif et al.*, 1985).

To establish infection, RABV will first engage host receptors and then enter cells via the endocytic pathway (*Albertini et al.*, 2012a). It is believed that the acidic environment in the endosomal compartments could trigger a series of conformational changes in RABV-G to mediate membrane fusion (*Albertini et al.*, 2012a). Along this fusion process induced by low pH,

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

RABV-G could transit among several different conformational states ([Roche and Gaudin, 2002](https://doi.org/10.1016/S0042-6822(02)00357-8)). These states are proposed to exist in a pH-dependent thermodynamic equilibrium, such that it is shifted from the pre-fusion state toward the post-fusion state as the pH decreases ([Gaudin et al., 1993](https://doi.org/10.1016/0042-6822(93)90047-8); [Gaudin et al., 1991](https://doi.org/10.1016/0042-6822(91)90210-8); [Roche and Gaudin, 2002](https://doi.org/10.1016/S0042-6822(02)00357-8)). This results in a reversible structural transition, which distinguishes RABV-G from other viral glycoproteins that undergo completely irreversible transitions induced by a reduction in pH ([Gaudin, 2000](https://doi.org/10.1016/S0042-6822(00)00165-6); [Harrison, 2015](https://doi.org/10.1016/j.cell.2015.02.033)). Such reversibility is proposed to enable RABV-G recovery to its native pre-fusion conformation after transport through the acidic compartments of the Golgi apparatus ([Gaudin et al., 1995](https://doi.org/10.1016/0042-6822(95)00050-8)). Similar reversible transitions have also been reported for members of the Vesiculovirus genus in the Rhabdoviridae family ([Albertini et al., 2012b](https://doi.org/10.1016/j.jmb.2012.05.010); [Baquero et al., 2015](https://doi.org/10.1016/j.jmb.2015.04.010); [Roche et al., 2006](https://doi.org/10.1016/j.jmb.2006.03.035), [2007](https://doi.org/10.1016/j.jmb.2007.03.010)). However, the glycoproteins of RABV and vesiculovirus share very low sequence identities, i.e., RABV-G (CVS-11 strain) shows ~18% sequence identity with vesicular stomatitis virus glycoprotein (VSV-G) and ~20% identity with the Chandipura virus glycoprotein (CHAV-G).

The atomic structures of VSV- and CHAV-G have been reported ([Baquero et al., 2017](https://doi.org/10.1016/j.jmb.2017.03.010), [2015](https://doi.org/10.1016/j.jmb.2015.04.010); [Roche et al., 2006](https://doi.org/10.1016/j.jmb.2006.03.035), [2007](https://doi.org/10.1016/j.jmb.2007.03.010)). These structures reveal a protein fold composed of three domains that are arranged in a bent conformation in the pre-fusion state and in an elongated conformation in the post-fusion state. The structural transition therefore requires substantial rearrangements of the individual domains. It is notable that both VSV/CHAV- and RABV-G are proposed to assemble into homo-trimers on viral surface ([Baquero et al., 2015](https://doi.org/10.1016/j.jmb.2015.04.010); [Gaudin et al., 1992](https://doi.org/10.1016/0042-6822(92)90040-8); [Whitt et al., 1991](https://doi.org/10.1016/0042-6822(91)90209-U)). The solved pre- and post-fusion structures of VSV-G indeed revealed a trimeric assembly architecture in the crystals ([Roche et al., 2006](https://doi.org/10.1016/j.jmb.2006.03.035), [2007](https://doi.org/10.1016/j.jmb.2007.03.010)). In light of the observed trimeric organizations, it is believed that the pre-to-post transition is topologically impossible without transient protein de-trimerization ([Albertini et al., 2012b](https://doi.org/10.1016/j.jmb.2012.05.010); [Baquero et al., 2013](https://doi.org/10.1016/j.jmb.2013.03.010); [Kim et al., 2017](https://doi.org/10.1016/j.jmb.2017.03.010)). Indeed, in a previously reported structure of CHAV-G, the protein molecules in the asymmetric unit (captured in the early- and late-intermediate states, respectively) are organized in the crystal in a dimeric but not trimeric packing mode ([Baquero et al., 2017](https://doi.org/10.1016/j.jmb.2017.03.010)). Furthermore, monomeric glycoprotein species have also been identified on the surface of VSV, though as minor populations in comparison to the glycoprotein trimers ([Albertini et al., 2012b](https://doi.org/10.1016/j.jmb.2012.05.010); [Baquero et al., 2017](https://doi.org/10.1016/j.jmb.2017.03.010)). While these studies have delineated the basis for the pH-dependent structural transitions of vesiculovirus glycoprotein, the structures of RABV-G and any other lyssavirus glycoprotein remain unsolved, even though the rabies disease has been described since ~2000 B.C. ([Adamson, 1977](https://doi.org/10.1016/0042-6822(77)90040-8)).

RABV-G adopts a type I transmembrane topology with a large ecto-domain. This glycoprotein ecto-domain (G-ecto) is the authentic receptor-binding entity and the true target for neutralizing antibodies. In this study, we developed a fusion-loop substitution strategy, via which homogeneous RABV-G ecto-domain protein could be obtained. We solved crystal structures of the viral protein at ~pH-8.0 in the free form and at ~pH-6.5 in an antibody-bound form. At basic pH, the RABV-G ecto-domain structure adopts a bent hairpin conformation, exhibiting typical pre-fusion features, whereas at acidic pH the RABV-G ecto-domain is of an elongated structure, showing post-fusion characteristics. The dislocated C terminus in the two structures, however, highlights the possibility that the solved structures provide snapshots of the protein in early- and late-intermediate states during low-pH-induced structural transitions. Finally, we also reveal the mechanism of RABV recognition and neutralization by an antibody via the complex structure. The atomic information should be able to facilitate future vaccine and drug design.

## RESULTS

### Preparation of a Well-Behaved RABV-G Ecto-Domain Protein

Excluding the signal peptide sequence, the ecto-domain of RABV-G (CVS-11 strain) spans residues 1–439 of the mature protein sequence ([Figure 1A](#fig1)). Expression of the wild-type (WT) protein only yielded irregular aggregates. Thus, we designed a series of ecto-domain constructs to screen for constructs suitable for crystallization. Among these, simultaneous substitution of the two putative fusion loops (residues 73–79 and 117–125) with a Gly-Gly-Ser-Gly-Gly linker ([Figure 1A](#fig1)) gave a well-behaved protein preparation. On ion-exchange and gel-filtration columns, this fusion-loop substituted form of RABV-G (hereafter referred to as RABV-G-ΔFuLp) gave a symmetric elution peak, indicating its high homogeneity in solution ([Figures 1B and 1C](#fig1)).

We then tested recognition of RABV-G-ΔFuLp by RABV-specific monoclonal antibodies (mAbs). A series of reported mAbs, including SOJB, SOJA and SO57 ([Prosnik et al., 2003](https://doi.org/10.1016/S0042-6822(03)00370-8)), EP5G3 and GD2D12 ([Ando et al., 2005](https://doi.org/10.1016/j.virol.2005.04.020)), Fab094 ([Liu et al., 2011](https://doi.org/10.1016/j.virol.2011.04.008)), and 523-11 ([Dietzschold et al., 1990](https://doi.org/10.1016/0042-6822(90)90040-8)) (an antibody recognizing a conformational epitope, see results section below “neutralization basis by antibody 523-11”) were prepared to high-purity ([Figure 1D](#fig1)) and used in an ELISA-binding assay. All tested RABV antibodies, but not an isotype-control antibody (5E6 [Qiu et al., 2011](https://doi.org/10.1016/j.jmb.2011.03.010) that targets the ebola virus glycoprotein), were observed to bind to RABV-G-ΔFuLp, albeit with different binding efficacy ([Figure 1D](#fig1)). We also tested the binding of RABV-G-ΔFuLp to the RABV receptor using the 32-mer peptide derived from nAchR via surface plasmon resonance (SPR). Several reports have shown that RABV infection could be inhibited, via specific G/peptide interactions, by the torpedo peptide (T32) with the highest potency, by the bovine peptide (C32) with less efficiency, and by the human peptide (H32) with the least efficacy ([Sajjanar et al., 2017](https://doi.org/10.1016/j.jmb.2017.03.010), [2016](https://doi.org/10.1016/j.jmb.2016.03.010)). Falling in good accordance with these studies, RABV-G-ΔFuLp interacted most effectively with T32 (with an affinity of ~1.7 μM in KD) and less potently with C32 (~3.1 μM in KD) and only showed a marginal binding to H32 (>70 μM in KD) ([Figure 1E](#fig1)). Antibody 523-11 is reported to target antigenic site III in RABV-G, and SOJB and SO57 to antigenic site I ([Figure S1A](#figS1)) ([Dietzschold et al., 1990](https://doi.org/10.1016/0042-6822(90)90040-8); [Marissen et al., 2005](https://doi.org/10.1016/j.virol.2005.04.020)). The footprint of the nAchR-peptide in RABV-G is proposed to locate between residues 175–203 ([Figure S1A](#figS1)) ([Lentz, 1990](https://doi.org/10.1016/0042-6822(90)90040-8)), which overlaps with antigenic site II ([Benmansour et al., 1991](https://doi.org/10.1016/0042-6822(91)90209-U)). The results of our ELISA- and SPR-binding assays therefore have shown that our protein could be recognized by RABV-G ligands (antibodies and the receptor-derived peptides) targeting at least three different sites in the glycoprotein. Thus, we believe RABV-G-ΔFuLp has retained the proper folding, despite substitution of the fusion loops.

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

![](image.png)

**Figure 1. Biochemical and Biophysical Characterization of RABV-G-ΔFuLp**

(A) A schematic view of the protein-engineering strategy used to yield RABV-G-ΔFuLp. The signal peptide (SP), ecto-domain, transmembrane domain (TM), cytoplasmic domain (cyto), and the two putative fusion loops (FuLp1 and FuLp2) of RABV-G are marked with the boundary-residue numbers. The substitution of the FuLps with the Gly-Gly-Ser-Gly-Gly linkers are highlighted.

(B and C) Solution behavior of RABV-G-ΔFuLp on a Source 15Q (B) and a Superdex 200 Increase 10/300 GL (C) column. The inset figure shows the SDS-PAGE analyses of the pooled samples.

(D) Interactions between RABV-G-ΔFuLp and the indicated antibodies. Left panel: SDS-PAGE analyses of the indicated antibodies under reducing and non-reducing conditions. Right panel: ELISA-binding profiles. Mean values were determined from three independent experiments and are presented with SD values indicated by error bars.

(E) Interactions between RABV-G-ΔFuLp and the nAchR derived peptides via SPR. The recorded binding profiles are shown. See also Figures S1 and S2.

On a calibrated Superdex-200 increase column, RABV-G-ΔFuLp was eluted at ~14.5 mL, indicating that the protein is a monomer (Figure 1C). However, previous results suggested that RABV-G would assemble into homo-trimers on the viral surface (Gaudin et al., 1992; Whitt et al., 1991). We therefore further characterized the solution-oligomerization behaviors of RABV-G-ΔFuLp via the sucrose gradient centrifugation and the analytical ultra-centrifugation assays. As a control, we also prepared a similar fusion-loop substituted form of the VSV G-ecto protein (VSV-G-ΔFuLp) and characterized its solution-assembly behavior in parallel with RABV-G-ΔFuLp. While VSV-G-ΔFuLp showed an apparent low-pH-induced trimerization, as reported previously (Baquero et al., 2015), RABV-G-ΔFuLp remained monomeric over the pH-range of 8.0–5.0 (Figures S2A and S2B). We further performed SPR experiments to investigate if homo-binding of RABV-G-ΔFuLp was too weak to be detected by the centrifugation assays. While no detectable binding was recorded at pH-8.0, we observed weak homo-interactions for the protein at pH-5.5 (Figure S2C). The affinity was determined to be ~40.3 μM in KD. This represents a very low self-assembly capability such that stable protein trimers can hardly be observed in solution.

Because expression of the WT ecto-domain protein of RABV-G only yielded irregular aggregates that were not applicable for solution-behavior analysis, we alternatively performed a cell-based cross-linking assay to determine whether the substitution of the fusion loops affected trimeric assembly of the viral protein. In the experiment, full-length RABV-G or its fusion-loop substituted form was first expressed on the cell surface and then subjected to different cross-linkers before western blotting analysis. As expected, WT RABV-G was found to cross-link and form trimers. In addition, the substituted glycoprotein variant showed a similar trimerization pattern along the cross-linker gradients when compared with that of the WT glycoprotein (Figure S2D), demonstrating that fusion-loop substitutions did not affect RABV-G trimerization at least on the cell surface.

**Structure of RABV-G at ~pH-8.0**

Using RABV-G-ΔFuLp, we first crystallized the protein at ~pH-8.0 and solved the structure at 3.1-Å resolution. The final model, which was refined to Rwork = 0.244 and Rfree = 0.282 (Table 1), contained two RABV-G-ΔFuLp molecules of essentially the same structure in the asymmetric unit (Figure S3A). Because the solved structure showed substantial homology to the reported VSV-G and CHAV-G structures (Baquero et al., 2017; Roche et al., 2007), we therefore accordingly divided our structure into three domains: a fusion domain (FD), a pleckstrin homology domain (PHD), and a central domain (CD) (Figures 2A–2D).

FD is composed of three anti-parallel β-sheets (dhi, gekj, and gef) and one α-helix (B), which further assemble into two subdomains. One subdomain consists of the dhi sheet and several surrounding surface loops that are sterically connected to PHD, whereas the other subdomain is composed of the gef and gekj sheets, forming a slight concave surface where helix B perches. The two fusion loops, which are replaced individually with a five-residue linker and are not visible in the structure, locate at the distal end of the second subdomain (Figure 2B). PHD is a complete β-barrel structure. It is formed by two anti-parallel β-sheets (opq and cimn), and links on one side to FD via the cd and kl interloops, and on the other side to CD via the Ac and qC linkers (Figure 2C). It is notable that the qC linker is rather flexible. The electron density for residues Gln256-Glu269 in the linker is not traceable. This region has been shown to refold into a helical

Cell Host & Microbe 27, 1–13, March 11, 2020 3

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

Table 1. Data Collection and Structure Refinement Statistics

| RABV-G-ΔFuLp (pH 8.0) | tr-RABV-G-ΔFuLp/523-11 (pH 6.5) |
|------------------------|----------------------------------|
| **Data Collection**    |                                  |
| Space Group            | C2                             | P212121                     |
| Cell Dimensions        |                                  |                              |
| $a, b, c (\AA)$       | 192.44, 138.45, 61.44           | 82.89, 93.94, 213.49         |
| $\alpha, \beta, \gamma (^{\circ})$ | 90.00, 100.02, 90.00      | 90.00, 90.00, 90.00         |
| Resolution ($\AA$)     | 50.00–3.10 (3.21–3.10)          | 50.00–2.90 (3.00–2.90)       |
| $R_{\text{merge}}$     | 0.130 (1.444)                   | 0.139 (2.057)                |
| CC1/2                  | 0.985 (0.659)                   | 0.982 (0.514)                |
| $I / \Sigma I$         | 14.45 (1.17)                    | 14.76 (0.96)                 |
| Completeness (%)       | 99.7 (99.7)                     | 100.0 (100.0)                |
| Redundancy             | 6.8 (6.7)                       | 10.9 (11.3)                  |

| **Refinement**         |                                  |
| Resolution ($\AA$)     | 47.39–3.10                      | 46.97–2.90                   |
| No. Reflections        | 28656                           | 37497                        |
| $R_{\text{work}} / R_{\text{free}}$ | 0.2442/0.2822              | 0.2185/0.2662                |
| No. Atoms              |                                  |                              |
| Protein                | 5695                            | 9342                         |
| Ligand/ion             | 0                               | 0                            |
| Water                  | 0                               | 0                            |
| B-Factors              |                                  |                              |
| Protein                | 123.7                           | 96.7                         |
| Ligand/ion             | 0                               | 0                            |
| Water                  | 0                               | 0                            |
| RMS Deviations         |                                  |                              |
| Bond Lengths ($\AA$)   | 0.005                           | 0.004                        |
| Bond Angles ($^{\circ}$) | 0.947                          | 0.794                        |
| Ramachandran Plot (%)  |                                  |                              |
| Favored Region         | 92.81                           | 95.47                        |
| Allowed Region         | 6.77                            | 4.45                         |
| Outlier Region         | 0.42                            | 0.09                         |

In each case, a single crystal was used to collect the data. Values in parentheses are for the highest-resolution shell.

structure under acidic pH conditions, indicating its role in the low-pH-induced structural transitions of RABV-G (see results section below “basis of low-pH-induced structural transition”). CD consists of nine β-strands (a, b, r, s, t, u, v, w, and x) that assemble into a compact core-barrel and six helices (A, C, D, E, F, and G) that surround the core-barrel on the lateral side (Figure 2D). While forming a compact domain, helices F and G in CD and their long inter-helix loop extend out and are further stabilized by the neighboring molecule within the asymmetric unit (Figure S3A). In total, seven disulfide bonds are observed in the structure, including the Cys 35/Cys 207, Cys 24/Cys 283, Cys 61/Cys 94, Cys 159/Cys 169, Cys 189/ Cys 228, Cys 223/Cys 252, and Cys 344/Cys 351 pairs (Figures 2B–2D). Topologically, the three domains are arranged such that FD is inserted into PHD and that PHD is further inserted into CD (Figure S4). Sterically, CD folds back onto the PHD/FD junction region, leading to the formation of an overall bent hairpin conformation with a height of approximately 80 Å (Figure 2A). This value is in accord with the molecular dimensions of the protein at high pH, as previously calculated by electron microscopy (EM) (Gaudin et al., 1993). Furthermore, this domain arrangement is also similar to that observed in the pre-fusion structure of VSV-G and in an early-intermediate structure of CHAV-G (Figure 2A).

Structure of RABV-G at ~pH-6.5

We then screened RABV-G-ΔFuLp and the protein constructs with various C-terminal truncations for crystallizations below pH-7, either in the free form or in combination with different antibodies using the single-chain fragment variable (scFv). A truncated RABV-G-ΔFuLp (designated as tr-RABV-G-ΔFuLp hereafter) that lacked C-terminal residues Phe 406-Lys 439 was successfully crystallized with antibody 523-11 at ~pH-6.5. Despite the truncation, tr-RABV-G-ΔFuLp still forms a stable

4 Cell Host & Microbe 27, 1–13, March 11, 2020

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

---

**Figure 2. Structure of RABV-G at ~pH-8.0**

(A) An overview of the structure. Domains CD, PHD, and FD are colored green, cyan, and magenta, respectively, and domain-linkers are in gray. The secondary structural elements are denoted in alphabetical order along the protein sequence (lowercase letters for β-strands and uppercase letters for α-helices). The regions that are not visible in the structure are indicated with dashed lines. The height of the molecule is labeled. The inset figure highlights the trace of the structure by marking residue boundaries for each color. The lower panel shows the domain-arrangement patterns for the pre-fusion structure of VSV-G (PDB: 5I2S) and an early-intermediate structure of CHAV-G (PDB: 5MDM).

(B–D) Comparison of the individual RABV-G domains with those of VSV-G and CHAV-G. Upper panel: the three RABV-G domains generated by rotation of the molecule in (A) around a vertical axis for ~90°. Bottom panel: pairwise structure superposition of the indicated domains and the calculated rmsd values. (B) Comparison of the FD domains. (C) Comparison of the PHD domains. (D) Comparison of the CD domains.

The structural elements referred to in the text are labeled. See also Figures S3, S4, S6, and S7.

---

complex with the antibody in solution, as with the un-truncated protein (Figure S5). The final structure, solved at 2.9-Å and refined to Rwork = 0.219 and Rfree = 0.266 (Table 1), contains two tr-RABV-G-ΔFuLp/523-11 complexes in the asymmetric unit (Figure S3B). In this low-pH structure of RABV-G, the three domains of the protein are arranged in a linear fashion (Figure 3A), resulting into an overall elongated conformation that resembles those reported for the VSV-G (rmsd: ~4.6 Å/332 Cas) and CHAV-G (~3.7 Å/329 Cas) post-fusion structures as well as a CHAV-G (~5.0 Å/235 Cas) structure in a late-intermediate state (Figures 3B–3D). The molecular size of the protein is measured to be ~121 Å in height (Figure 3A), which is in agreement with the EM calculations of the protein at low pH (Gaudin et al., 1993).

**Basis of Low-pH-Induced Structural Transition**

It is notable that the solved glycoprotein structures derived at ~pH-8.0 and ~pH-6.5 exhibit significant difference in overall fold. To characterize this low-pH-induced structural change, we further compared the two structures based on the individual domains. When superimposed on PHD, CD was flipped by ~180° and FD had rotated by ~59° (Figure 4A). The CD-flip was the result of a large loop reorientation in the Ac linker (Figure 4B), whereas the FD-rotation was caused by conforma-

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

**Figure 3. Structure of RABV-G at ~pH-6.5**

(A) An overview of the structure. Domains CD, PHD, and FD are colored green, cyan, and magenta, respectively, and domain-linkers are in gray. The height of the molecule, the secondary structural elements, and the disulfide linkages are labeled. The inset figure highlights the trace of the structure by marking residue boundaries for each color.

(B–D) Superimposition of our structure (cyan) onto the structures (gray) of (B) VSV-G (PDB: 5I2M) and (C) CHAV-G (PDB: 4D6W) in the post-fusion state and (D) CHAV-G (PDB: 5MDM) in a late-intermediate state.

The calculated rmsd values are labeled. See also Figures S3, S4, and S6.

We subsequently developed a cell-based syncytia formation assay to ascertain the functional importance of the aforementioned interactions. We mutated these four amino acids either individually or in combination and analyzed the resultant glycoprotein mutants for their capacities to induce syncytia formation in parallel with WT glycoprotein. WT RABV-G induced the formation of large syncytia, whereas no syncytia were observed to form with the D211A and H261A mutants. Syncytia still formed with the D266A and E269A mutants. Nevertheless, simultaneous substitution of the two amino acids with alanine completely abolished syncytia formation (Figures 5C and 5D). The competency of RABV-G E269A mutant in membrane fusion is in agreement with the Irkut lyssavirus naturally harboring such mutation in its native glycoprotein (Figure S4). Taken together, these results revealed that interactions mediated by these amino acids are indispensable, which demonstrates the functional importance of the qC-linker refolding in RABV-G mediated membrane fusion. Similar loop-to-helix transitions have also been observed in the equivalent domain linkers of VSV-G and CHAV-G (Baquero et al., 2017, 2015; Roche et al., 2006, 2007).

Antibody 17D2 is directed against peptide Asp255-Val272 of RABV-G (Gaudin, 1997), corresponding to the qC linker, which experiences a refolding process (from loop to helix) upon a reduction in pH. Accordingly, this antibody was shown to recognize the denatured glycoprotein but not the glycoprotein at low pH (Gaudin et al., 1996). As for an N-terminal epitope extending residues Glu10-Ser15, it is only partially exposed at pH-8.0 but becomes fully solvent exposed at pH-6.5 (Figure S1B), echoing previous studies showing that this epitope is only accessible in the low-pH conformation (Gaudin et al., 1996).

**A Mutagenesis Study Identifies Key Residues Required for RABV-G Function**

It is interesting that the qC linker of RABV-G, as observed in our structures, would experience a loop-to-helix transition as the pH decreases (Figures 5A and 5B). In the low pH structure, part of the qC linker refolds to assemble as an extension of helix C. Such transitions bring Asp211 and His261 into close proximity with Glu269 and Asp266, enabling the formation of strong polar and van der Waals (vdw) interactions to stabilize the elongated conformation of RABV-G. These four residues, however, are unlikely to interact with each other because of the high flexibility of this qC interloop (His261, Asp266, and Glu269 are lacking traceable electron densities) at high-pH (Figures 5A and 5B). These four residues are highly conserved among lyssavirus glycoproteins. Only the Irkut lyssavirus contains a single mutation such that Glu269 is substituted by an alanine (Figure S4).

**Neutralization Basis by Antibody 523-11**

For the glycoproteins from the CVS-11, Flury, and SAD-B19 strains, the glycoprotein mediated syncytia formation was blocked by antibody 523-11 (Figure 6A). Furthermore, 523-11 also effectively inhibited the infection of cells by the live virus of the Flury strain (Figure 6B). In the complex structure, this antibody exclusively binds to the CD domain of RABV-G, recognizing two discontinuous epitope regions in the viral protein (Figure 6C). The first epitope region includes helix E and the small tu β-hairpin (Figure 6C). Multiple residues in the region are observed to H-bond with the antibody. These include Asn336, Ile339-Lys342, and Lys346 in RABV-G interacting with Asp31, Asn33, Ser57, Tyr101, and Asp102 in the antibody heavy chain (Figure 6D) and the glycoprotein residue Arg350 with Tyr50 in the antibody light chain (Figure 6E). In addition, glycoprotein

6 Cell Host & Microbe 27, 1–13, March 11, 2020

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

**Figure 4. Basis of Low-pH-Induced Structural Transition in RABV-G**

(A) Superposition of the basic- (green) and acidic-pH (magenta) structures of RABV-G based on PHD. The domain orientation differences observed in CD and FD are marked. The shaded regions highlight domain-linkers that are either re-oriented or reconstructed during the transition, which are further illustrated for clarity in (B), (C), and (D), respectively.

(B) Re-orientation of the Ac linker.

(C) Re-construction of the qC linker.

(D) Conformational changes in the cd and kl linkers.

(E–G) Superposition of the individual domain structures with the calculated rmsd values marked in parentheses. (E) Superposition of the CD domains. (F) Superposition of the PHD domains. (G) Superposition of the FD domains.

Those structural elements referred to in the text are labeled. See also Figures S1 and S4.

in antigen/antibody interactions featured with lower affinities at low pH than at neutral pH (Chaparro-Riggers et al., 2012; Devanaboyina et al., 2013; Igawa et al., 2010).

We also screened a total of 2,215 RABV-G sequences in NCBI. As expected, a majority of the epitope residues are conserved (Figure 6H). The most variable site is observed at position 346. While the glycoprotein of CVS-11 strain in our structure is featured with Lys346, most of the RABV strains, however, possess an Arginine residue at this position. We found that both Flury and SAD-B19 strains contain an Arg346 in the glycoprotein. In addition, the SAD-B19 glycoprotein contains two other mutations (Ile339 to Leu339 and His370 to Asn370) in the epitope region (Figure 6H), making it a representative that is most divergently related to CVS-11 glycoprotein. In light of our observations that 523-11 can also block the syncytia formation mediated by the Flury and SAD-B19 glycoproteins, we believe this antibody has a wide breadth against different rabies virus strains.

**DISCUSSION**

Eighteen genera are currently recognized within the *Rhabdoviridae* family (Amarasinghe et al., 2018). Prior to this study, preparation of homogeneous glycoproteins via a heterologous recombinant-expression system had not been successfully carried out. The vesiculovirus glycoproteins used for crystallization in previous studies were derived from protease-digestion of the viral particles (Baquero et al., 2015; Roche et al., 2006). In this study, we developed a fusion-loop substitution strategy to obtain a well-behaved and properly folded RABV G-ecto protein. Using the same strategy, we also obtained a vesiculovirus G-ecto protein (VSV-G-ΔFuLp), which could form a trimer at low pH, as previously observed for the WT ecto-domain protein (Baquero et al., 2015). Thus, this method may be suitable for preparation of viral glycoproteins from other rhabdovirus genera.

Along membrane fusion induced by low pH, rhabdovirus glycoprotein would transit among different conformational

Cell Host & Microbe 27, 1–13, March 11, 2020 7

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

![](https://i.imgur.com/yourimageA.png)
![](https://i.imgur.com/yourimageB.png)
![](https://i.imgur.com/yourimageC.png)
![](https://i.imgur.com/yourimageD.png)

**Figure 5. An Important Role of qC-Linker Re-construction in Glycoprotein-Mediated Membrane Fusion**

(A and B) Comparison of the basic- and acidic-pH structures showing the qC-linker changes and the resultant amino acid interaction differences. Residues His261, Asp266, and Glu269 (colored in cyan) are not traceable in the structure at high pH, and their steric positions are deduced based on the orientation of the qC-interloop. (A) View of the indicated residues in the basic-pH structure of RABV-G. (B) View of the indicated residues in the acidic-pH structure of RABV-G. These residues are further mutated in full-length RABV-G and the resultant G-mutants are tested for their capacities to induce syncytia formation in BHK-21 cells.

(C) Representative cell images highlighting the variant capability of the indicated glycoproteins to induce syncytia formation. For clarity, cells are co-transfected for simultaneous expression of EGFP and RABV-G. GFP is a negative control, for which only EGFP is expressed. The arrows mark the formed syncytia. Scale bar: 200 μm.

(D) Representative cell images demonstrating the proper cell-surface location of the indicated glycoproteins. Scale bar: 20 μm.

See also **Figure S4**.

states (**Baquero et al., 2013; Gaudin et al., 1993, 1991; Roche and Gaudin, 2002**). For the glycoprotein of vesiculoviruses, previous studies have described structural features of the protein in four conformational states: the pre-fusion, an early intermediate, a late intermediate, and the post-fusion states, respectively (**Baquero et al., 2017, 2015; Roche et al., 2006, 2007**). When the four structures are aligned sequentially, we find that the status of the protein C-terminal element (located to or dislocated from the glycoprotein FD domain) coincides well with the transition process (**Figure S6**). In the pre-fusion structure, the C-element folds mainly as a loop. It extends over a surface groove on domains CD and FD and ties its terminus up to FD. This would lock the glycoprotein in the bent conformation. In the early-intermediate structure, the glycoprotein remains in the bent conformation, but its C-element is dislocated from FD, which represents an unlock process to allow domain rearrangement at next stage. In the late-intermediate structure, the three domains of the glycoprotein re-orient into an extended conformation and the C-element remains dislocated from FD. Finally, the terminal element refolds into a lateral helix and makes part of the six-helix bundle in the post-fusion trimer, re-locking its terminus to the FD domain. Noted that the unlocking process logically represents a prerequisite for domain rearrangements, we believe the “dislocated” status of the

C-element might be an indication of the protein in the intermediate state. As a matter of fact, in a model of membrane fusion mediated by VSV-G, the protein C-terminal loop is proposed to extend into the close vicinity of FD in both the pre-fusion and post-fusion states but remain stretching away from FD during the transitions (**Baquero et al., 2013**). Despite that the two asymmetric-unit copies in our structures exhibit slight conformational differences for the C-terminal loop (**Figures S3A and S3B**), they nonetheless both stretch the C-element away from FD and therefore feature a “dislocated” status (**Figure S6**). Although its steric position is stabilized by the neighboring molecule via extensive crystal contacts, the observed “dislocated” status well parallels our structures to the intermediate structures of CHAV-G. We thus propose that our structures likely have snapshot the protein in the early- and late-intermediate states during the low-pH-induced structural transitions. Nevertheless, we still cannot exclude the possibility, though unlikely, that the region is just flexible and stabilized in the observed conformation by the crystal contact.

On the viral surface, the rhabdovirus glycoprotein exists mainly as trimeric spikes (**Gaudin et al., 1992; Whitt et al., 1991**). In solution, however, the G-ecto protein does not necessarily exist as trimers. In fact, the ecto-domain protein of VSV/CHAV-G has been shown to behave mainly as monomers at

8 Cell Host & Microbe 27, 1–13, March 11, 2020

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

Figure 6. Basis of RABV-G Neutralization by Antibody 523-11

(A) Blockage of the glycoprotein-mediated cell-cell fusion by 523-11. Syncytia formed are marked with arrows. Scale bar: 200 μm.

(B) Inhibition of RABV infection by 523-11. BHK-21 cells are left uninfected (Blank) or infected with the Flury strain of RABV in the absence (RABV) or presence of the indicated antibodies (RABV + Iso-control and RABV + 523-11) and stained with a FITC-labeled RABV N-protein antibody (green) and Evans blue (light red). Representative cell images are shown. Scale bar: 200 μm.

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

pH above 7.5 ([Baquero et al., 2015](https://doi.org/10.1016/j.chom.2019.12.012)). To obtain homogeneous RABV-G protein for structural studies, we substituted the two fusion loops with the Gly-Gly-Ser-Gly-Gly linkers. The subsequent RABV-G-ΔFuLp protein expectedly was monomeric at basic pH. Unlike the VSV/CHAV protein that assembles into stable trimers following low pH induction, RABV-G-ΔFuLp is of weak self-assembly capacity at acidic pH and no RABV-G-ΔFuLp trimers can be detected in solution. Such incompetency in trimerization, however, is unlikely the result of fusion-loop substitutions. In the post-fusion trimer structure of VSV-G, their fusion loops are out of the vdw distance. Consistently, VSV G-ecto protein lacking the fusion loops (VSV-G-ΔFuLp) is also able to assemble into stable trimers at low pH. In addition, our cross-linking assay shows that RABV-G-ΔFuLp, after reconstruction to include the transmembrane and cytoplasmic domains, could form trimers at the cell surface in a similar way to WT RABV-G. The limited overall cross-linking efficiency in cells might have arisen from some technical limitations: e.g., the low-pH cross-links with glutaraldehyde are reversible, and BS³ and EGS are not very efficient at a pH below 7. In light of these observations, we believe the weak homo-binding capability is likely an intrinsic characteristic of RABV-G ecto-domain. Echoing the notion, previous studies have shown that RABV-G trimer is not as stable as VSV-G and is easily disrupted upon extraction from the viral surface using detergents ([Gaudin et al., 1992; Whitt et al., 1991](https://doi.org/10.1016/j.chom.2019.12.012)). It is also notable that the pre-to-post structural transitions for rhabdovirus glycoprotein are proposed to require transient protein de-trimerization ([Albertini et al., 2012b; Baquero et al., 2013; Kim et al., 2017](https://doi.org/10.1016/j.chom.2019.12.012)). In this context, it seems more biologically compatible with the reported transition-reversibility when RABV-G ecto-domain itself does not possess capacity for strong self-assembly.

It is also noteworthy that the dislocated C-element remains a helical fold (as helix F or F1 and F2 helix halves) in our structures, forming contrast to the C-element in the CHAV-G structures that folds largely as a loop ([Figure S6](https://doi.org/10.1016/j.chom.2019.12.012)). In light of the less flexibility of a helix than a loop, such pre-formed helix in RABV-G might not be as easily re-constructed for six-helix bundle formation as a loop observed in the vesiculovirus glycoprotein. This might correlate with the low-affinity homo-binding for RABV-G-ΔFuLp at low pH. A full explanation on this, however, requires the trimeric post-fusion structure of the protein, which should be studied in the future. Despite great efforts, we have not been able to crystallize RABV-G-ΔFuLp in the trimeric post-fusion state thus far. Without experimentally determined trimer structures, we alternatively constructed trimeric models for RABV-G based on the VSV-G trimer architectures. In such trimer models, the 523-11 epitope is fully exposed ([Figure S1C](https://doi.org/10.1016/j.chom.2019.12.012)), echoing the results from our live-virus infection assay. Using the Flury strain, we have shown that 523-11 could neutralize the viral infection, indicating it can block cell entry of the viral particles on which the glycoprotein mainly exists as trimers.

Despite low sequence identities, RABV-G and VSV-/CHAV-G share substantial structural homologies. They all consist of three domains. In addition, lowering the pH similarly induces domain rearrangements, leading the glycoprotein from a bent hairpin conformation to an extended conformation. These observations demonstrate that lyssaviruses and vesiculoviruses have evolutionarily conserved membrane-fusion mechanisms, in which relatively rigid domains in combination with flexible and reconstructible domain-linkers constitute the basis of glycoprotein conformational changes in response to environmental pH. It is also noteworthy that four disulfide-cysteines (out of fourteen) in RABV-G are located at the domain boundaries. Because all the disulfide-related cysteine residues are invariantly preserved in RABV and other lyssavirus glycoproteins ([Figure S4](https://doi.org/10.1016/j.chom.2019.12.012)), we therefore predict conserved disulfide organizations and resembled rearrangement of domains bordered at the corresponding disulfide linkages in these viruses.

Finally, the pre- and post-exposure prophylaxis remain the most commonly used measures to combat human rabies ([Ugolini and Hemachudha, 2018](https://doi.org/10.1016/j.chom.2019.12.012)). A properly folded glycoprotein is a potentially ideal immunogen for vaccination. Thus, it is worth trying to test the immunization effect of RABV-G-ΔFuLp in the future. In addition, targeting the low-pH-induced conformational changes of virus envelope glycoprotein is already demonstrated as a feasible way of identifying inhibitory agents interfering with the viral entry process. For instance, the hydrophobic pockets at the interface of domain-I/II in both dengue virus E and bovine viral diarrhea virus E2 are proved to be druggable sites ([Bollini et al., 2018; Zhou et al., 2008](https://doi.org/10.1016/j.chom.2019.12.012)). In the basic-pH structure of RABV-G, we also identified several surface pockets around the domain junctions ([Figure S7](https://doi.org/10.1016/j.chom.2019.12.012)). In light of the domain rearrangements for RABV-G during the structural transitions, we believe these pockets might be targeted for design of small-molecule inhibitors aiming to block the observed structural changes and thereby the glycoprotein-mediated membrane fusion.

### STAR★METHODS

Detailed methods are provided in the online version of this paper and include the following:

- **KEY RESOURCES TABLE**
- **LEAD CONTACT AND MATERIALS AVAILABILITY**
- **EXPERIMENTAL MODEL AND SUBJECT DETAILS**
  - Cell Lines and Viruses

(C) Structure of the RABV-G/523-11 complex. Coloring scheme: RABV-G, light blue; antibody heavy chain, green (CDR loops in orange); and light chain, cyan (CDR loops in light pink). The bipartite epitope (epitope regions 1 and 2) is marked by encircling using dashed lines. The virulence-related Arg333 residue is marked.

(D-F) Amino acid interaction details at the paratope/epitope binding interface. Dashed lines indicate H-bonds. (D) Interactions with the antibody heavy chain in epitope region 1. (E) Interactions with antibody light chain in region 1. (F) Interactions with antibody heavy chain in region 2.

(G) A western blot assay characterizing the interactions of the indicated antibodies with denatured RABV-G-ΔFuLp.

(H) Comparison of the epitope residues in RABV-G of different strains. The sequence logo was generated using WebLogo. The interface amino acids are marked with its residue numbers above the sequence. The sequence alignment highlights those residues that are different in glycoproteins of the CVS-11, Flury, and SAD-B19 strains.

See also [Figure S5](https://doi.org/10.1016/j.chom.2019.12.012).

10 Cell Host & Microbe 27, 1–13, March 11, 2020

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

### METHOD DETAILS

- Protein Expression and Purification
- Single Chain Fragment Variable (scFv) Preparation
- Recombinant Full-Length Antibody Preparation
- Indirect ELISA
- Analytical Gel Filtration Chromatography
- Crystallization
- Data Collection and Structure Determination
- Cell-Cell Fusion Assay for Syncytia Formation and the Fusion Inhibition by Antibody 523-11
- Immunofluorescent Assay Verifying Surface-Location of WT and Mutant RABV-G
- Virus Neutralization by Antibody 523-11
- Protein Oligomerization Assay
- Cell Based Cross-Linking Assay
- Surface Plasmon Resonance Assay
- Verification of the Conformational Epitope Recognized by 523-11
- Sequence Logo Visualization Using WebLogo
- Identification of Potential Druggable-Pockets on the Basic-pH Structure of RABV-G
- Trimer Model Building for RABV-G

### QUANTIFICATION AND STATISTICAL ANALYSIS

### DATA AND CODE AVAILABILITY

### REFERENCES

Adams, P.D., Afonine, P.V., Bunkóczi, G., Chen, V.B., Davis, I.W., Echols, N., Head, J.J., Hung, L.W., Kapral, G.J., Grosse-Kunstleve, R.W., et al. (2010). PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. D Biol. Crystallogr. 66, 213–221.

Adamson, P.B. (1977). The spread of rabies into Europe and the probable origin of this disease in antiquity. J. R. Asiatic Soc. Gr. Br. Irel. 2, 140–144.

Albertini, A.A., Baquero, E., Ferlin, A., and Gaudin, Y. (2012a). Molecular and cellular aspects of rhabdovirus entry. Viruses 4, 117–139.

Albertini, A.A., Mérigoux, C., Libersou, S., Madiona, K., Bressanelli, S., Roche, S., Lepault, J., Melki, R., Vachette, P., and Gaudin, Y. (2012b). Characterization of monomeric intermediates during VSV glycoprotein structural transition. PLoS Pathog. 8, e1002556.

Amarasinghe, G.K., Aréchiga Ceballos, N.G., Banyard, A.C., Basler, C.F., Bavari, S., Bennett, A.J., Blasdell, K.R., Briese, T., Bukreyev, A., Cai, Y., et al. (2018). Taxonomy of the order Mononegavirales: update 2018. Arch. Virol. 163, 2283–2294.

Ando, T., Yamashiro, T., Takita-Sonoda, Y., Mannen, K., and Nishizono, A. (2005). Construction of human Fab library and isolation of monoclonal Fabs with rabies virus-neutralizing ability. Microbiol. Immunol. 49, 311–322.

Baquero, E., Albertini, A.A., Raux, H., Abou-Hamdan, A., Boeri-Erba, E., Ouldali, M., Buonocore, L., Rose, J.K., Lepault, J., Bressanelli, S., and Gaudin, Y. (2017). Structural intermediates in the fusion-associated transition of vesiculovirus glycoprotein. EMBO J. 36, 679–692.

Baquero, E., Albertini, A.A., Raux, H., Buonocore, L., Rose, J.K., Bressanelli, S., and Gaudin, Y. (2015). Structure of the low pH conformation of Chandipura virus G reveals important features in the evolution of the vesiculovirus glycoprotein. PLoS Pathog. 11, e1004756.

Baquero, E., Albertini, A.A., Vachette, P., Lepault, J., Bressanelli, S., and Gaudin, Y. (2013). Intermediate conformations during viral fusion glycoprotein structural transition. Curr. Opin. Virol. 3, 143–150.

Benmansour, A., Leblois, H., Coulon, P., Tuffereau, C., Gaudin, Y., Flamand, A., and Lafay, F. (1991). Antigenicity of rabies virus glycoprotein. J. Virol. 65, 4198–4203.

Bollini, M., Leal, E.S., Adler, N.S., Aucar, M.G., Fernández, G.A., Pascual, M.J., Merwaiss, F., Alvarez, D.E., and Cavasotto, C.N. (2018). Discovery of novel bovine viral diarrhea inhibitors using structure-based virtual screening on the envelope protein E2. Front. Chem. 6, 79.

Chaparro-Riggers, J., Liang, H., DeVay, R.M., Bai, L., Sutton, J.E., Chen, W., Geng, T., Lindquist, K., Casas, M.G., Boustany, L.M., et al. (2012). Increasing serum half-life and extending cholesterol lowering in vivo by engineering antibody with pH-sensitive binding to PCSK9. J. Biol. Chem. 287, 11090–11097.

Collaborative Computational Project, Number 4 (1994). The CCP4 suite: programs for protein crystallography. Acta Crystallogr. D Biol. Crystallogr. 50, 760–763.

Crooks, G.E., Hon, G., Chandonia, J.M., and Brenner, S.E. (2004). WebLogo: a sequence logo generator. Genome Res. 14, 1188–1190.

Devanaboyina, S.C., Lynch, S.M., Ober, R.J., Ram, S., Kim, D., Puig-Canto, A., Breen, S., Kasturirangan, S., Fowler, S., Peng, L., et al. (2013). The effect of pH dependence of antibody-antigen interactions on subcellular trafficking dynamics. MAbs 5, 851–859.

Dietzschold, B., Gore, M., Casali, P., Ueki, Y., Rupprecht, C.E., Notkins, A.L., and Koprowski, H. (1990). Biological characterization of human monoclonal antibodies to rabies virus. J. Virol. 64, 3087–3090.

Emsley, P., and Cowtan, K. (2004). Coot: model-building tools for molecular graphics. Acta Crystallogr. D Biol. Crystallogr. 60, 2126–2132.

Flamand, A., Raux, H., Gaudin, Y., and Ruigrok, R.W. (1993). Mechanisms of rabies virus neutralization. Virology 194, 302–313.

Fooks, A.R., Banyard, A.C., Horton, D.L., Johnson, N., McElhinney, L.M., and Jackson, A.C. (2014). Current status of rabies and prospects for elimination. Lancet 384, 1389–1399.

### SUPPLEMENTAL INFORMATION

Supplemental Information can be found online at https://doi.org/10.1016/j.chom.2019.12.012.

### ACKNOWLEDGMENTS

This work was supported by the National Key R&D Program of China (grant no. 2016YFC1200305), the National Natural Science Foundation of China (NSFC, grant nos. 31570157 and 81522026), the Sichuan Outstanding Youth Science & Technology Funding (grant no. 2016JQ0001), and the Outstanding Youth Foundation of Sichuan University (grant no. 2016SCU04B01). We thank staff of BL19U1 and BL17U1 beamlines at National Center for Protein Sciences Shanghai and Shanghai Synchrotron Radiation Facility (Shanghai, People’s Republic of China) for assistance during data collection and Prof. Ping Yin at Huazhong Agricultural University for help with the analytical ultra-centrifugation experiments. We also thank Liwen Bianji, Edanz Editing China, for editing the English text. G.F.G. is a leading principal investigator of the NSFC Innovative Research Group (grant no. 81621091).

### AUTHOR CONTRIBUTIONS

G.L. and G.F.G. conceived the study. G.L. supervised the whole project. F. Yang., S.L., F. Ye., and J.Y. conducted most of the experiments. F. Ye, F. Yang, and S.L. collected the datasets and G.L. and J.Q. solved the structures. Zhujun Chen, X.L., J.W., D.Y., Y. Cheng, Zimin Chen, H.C., Y. You, Z.Z., Y. Yang, M.Y., and H.S. assisted with the cell maintenance and protein preparations. G.L. and G.F.G. wrote the manuscript, and Y.L., Y. Cao, S.Y., and Y.W. participated in the manuscript editing and discussion.

### DECLARATION OF INTERESTS

G.L., F. Yang., S.L., F. Ye, and J.Y. have a patent related to this work.

Received: September 25, 2019  
Revised: December 6, 2019  
Accepted: December 30, 2019  
Published: January 30, 2020

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

Fooks, A.R., Cliquet, F., Finke, S., Freuling, C., Hemachudha, T., Mani, R.S., Müller, T., Nadin-Davis, S., Picard-Meyer, E., Wilde, H., and Banyard, A.C. (2017). Rabies. Nat. Rev. Dis. Prim. 3, 17091.

Gaudin, Y. (1997). Folding of rabies virus glycoprotein: epitope acquisition and interaction with endoplasmic reticulum chaperones. J. Virol. 71, 3742–3750.

Gaudin, Y. (2000). Reversibility in fusion protein conformational changes. The intriguing case of rhabdovirus-induced membrane fusion. Subcell. Biochem. 34, 379–408.

Gaudin, Y., Raux, H., Flamand, A., and Ruigrok, R.W. (1996). Identification of amino acids controlling the low-pH-induced conformational change of rabies virus glycoprotein. J. Virol. 70, 7371–7378.

Gaudin, Y., Ruigrok, R.W., Knossow, M., and Flamand, A. (1993). Low-pH conformational changes of rabies virus glycoprotein and their role in membrane fusion. J. Virol. 67, 1365–1372.

Gaudin, Y., Ruigrok, R.W., Tuffereau, C., Knossow, M., and Flamand, A. (1992). Rabies virus glycoprotein is a trimer. Virology 187, 627–632.

Gaudin, Y., Tuffereau, C., Durrer, P., Flamand, A., and Ruigrok, R.W. (1995). Biological function of the low-pH, fusion-inactive conformation of rabies virus glycoprotein (G): G is transported in a fusion-inactive state-like conformation. J. Virol. 69, 5528–5534.

Gaudin, Y., Tuffereau, C., Segretain, D., Knossow, M., and Flamand, A. (1991). Reversible conformational changes and fusion activity of rabies virus glycoprotein. J. Virol. 65, 4853–4859.

Hampson, K., Coudeville, L., Lembo, T., Sambo, M., Kieffer, A., Attlan, M., Barrat, J., Blanton, J.D., Briggs, D.J., Cleaveland, S., et al. (2015). Estimating the global burden of endemic canine rabies. PLoS Negl. Trop. Dis. 9, e0003709.

Harrison, S.C. (2015). Viral membrane fusion. Virology 479–480, 498–507.

Hemachudha, T., Ugolini, G., Wacharapluesadee, S., Sungkarat, W., Shuangshoti, S., and Laothamatas, J. (2013). Human rabies: neuropathogenesis, diagnosis, and management. Lancet Neurol. 12, 498–513.

Igawa, T., Ishii, S., Tachibana, T., Maeda, A., Higuchi, Y., Shimaoka, S., Moriyama, C., Watanabe, T., Takubo, R., Doi, Y., et al. (2010). Antibody recycling by engineered pH-dependent antigen binding improves the duration of antigen neutralization. Nat. Biotechnol. 28, 1203–1207.

Kim, I.S., Jenni, S., Stanifer, M.L., Roth, E., Whelan, S.P., van Oijen, A.M., and Harrison, S.C. (2017). Mechanism of membrane fusion induced by vesicular stomatitis virus G protein. Proc. Natl. Acad. Sci. USA 114, E28–E36.

Lafon, M. (2011). Evasive strategies in rabies virus infection. Adv. Virus Res. 79, 33–53.

Laskowski, R.A., MacArthur, M.W., Moss, D.S., and Thornton, J.M. (1993). PROCHECK: a program to check the stereochemical quality of protein structures. J. Appl. Crystallogr. 26, 283–291.

Lentz, T.L. (1990). Rabies virus binding to an acetylcholine receptor alpha-subunit peptide. J. Mol. Recognit. 3, 82–88.

Lentz, T.L., Burrage, T.G., Smith, A.L., Crick, J., and Tignor, G.H. (1982). Is the acetylcholine receptor a rabies virus receptor? Science 215, 182–184.

Liu, X., Lin, H., Tang, Q., Li, C., Yang, S., Wang, Z., Wang, C., He, Q., Cao, B., Feng, Z., et al. (2011). Characterization of a human antibody fragment Fab and its calcium phosphate nanoparticles that inhibit rabies virus infection with vaccine. PLoS One 6, e19848.

Lu, G., Hu, Y., Wang, Q., Qi, J., Gao, F., Li, Y., Zhang, Y., Zhang, W., Yuan, Y., Bao, J., et al. (2013). Molecular basis of binding between novel human coronavirus MERS-CoV and its receptor CD26. Nature 500, 227–231.

Lyles, D.S., Kuzmin, I.V., and Rupprecht, C.E. (2013). Rhabdoviridae. In Fields Virology, Sixth Edition, D.M. Knipe and P.M. Howley, eds. (Lippincott Williams & Wilkins), pp. 885–922.

Marissen, W.E., Kramer, R.A., Rice, A., Weldon, W.C., Niezgoda, M., Faber, M., Slootstra, J.W., Meloen, R.H., Clijsters-van der Horst, M., Visser, T.J., et al. (2005). Novel rabies virus-neutralizing epitope recognized by human monoclonal antibody: fine mapping and escape mutant analysis. J. Virol 79, 4672–4678.

Murshudov, G.N., Vagin, A.A., and Dodson, E.J. (1997). Refinement of macromolecular structures by the maximum-likelihood method. Acta Crystallogr. D Biol. Crystallogr. 53, 240–255.

Otwinowski, Z., and Minor, W. (1997). Processing of X-ray diffraction data collected in oscillation mode. Methods Enzymol. 276, 307–326.

Prehaud, C., Coulon, P., LaFay, F., Thiers, C., and Flamand, A. (1988). Antigenic site II of the rabies virus glycoprotein: structure and role in viral virulence. J. Virol. 62, 1–7.

Prosniak, M., Faber, M., Hanlon, C.A., Rupprecht, C.E., Hooper, D.C., and Dietzschold, B. (2003). Development of a cocktail of recombinant-expressed human rabies virus-neutralizing monoclonal antibodies for postexposure prophylaxis of rabies. J. Infect. Dis. 188, 53–56.

Qiu, X., Alimonti, J.B., Melito, P.L., Fernando, L., Ströher, U., and Jones, S.M. (2011). Characterization of zaire Ebolavirus glycoprotein-specific monoclonal antibodies. Clin. Immunol. 141, 218–227.

Read, R.J. (2001). Pushing the boundaries of molecular replacement with maximum likelihood. Acta Crystallogr. D Biol. Crystallogr. 57, 1373–1382.

Roche, S., Bressanelli, S., Rey, F.A., and Gaudin, Y. (2006). Crystal structure of the low-pH form of the vesicular stomatitis virus glycoprotein G. Science 313, 187–191.

Roche, S., and Gaudin, Y. (2002). Characterization of the equilibrium between the native and fusion-inactive conformation of rabies virus glycoprotein indicates that the fusion complex is made of several trimers. Virology 297, 128–135.

Roche, S., Rey, F.A., Gaudin, Y., and Bressanelli, S. (2007). Structure of the prefusion form of the vesicular stomatitis virus glycoprotein G. Science 315, 843–848.

Sajjanar, B., Dhusia, K., Saxena, S., Joshi, V., Bisht, D., Thakuria, D., Manjunathareddy, G.B., Ramteke, P.W., and Kumar, S. (2017). Nicotinic acetylcholine receptor alpha 1(nAChRalpha1) subunit peptides as potential antiviral agents against rabies virus. Int. J. Biol. Macromol. 104, 180–188.

Sajjanar, B., Saxena, S., Bisht, D., Singh, A.K., Manjunatha Reddy, G.B., Singh, R., Singh, R.P., and Kumar, S. (2016). Effect of nicotinic acetylcholine receptor alpha 1 (nAChRalpha1) peptides on rabies virus infection in neuronal cells. Neuropeptides 57, 59–64.

Sasaki, M., Anindita, P.D., Ito, N., Sugiyama, M., Carr, M., Fukuhara, H., Ose, T., Maenaka, K., Takada, A., Hall, W.W., et al. (2018). The role of heparan sulfate proteoglycans as an attachment factor for rabies virus entry and infection. J. Infect. Dis. 217, 1740–1749.

Seif, I., Coulon, P., Rollin, P.E., and Flamand, A. (1985). Rabies virulence: effect on pathogenicity and sequence characterization of rabies virus mutations affecting antigenic site III of the glycoprotein. J. Virol. 53, 926–934.

Tamura, K., Peterson, D., Peterson, N., Stecher, G., Nei, M., and Kumar, S. (2011). MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods. Mol. Biol. Evol. 28, 2731–2739.

Thoulouze, M.I., Lafage, M., Schachner, M., Hartmann, U., Cremer, H., and Lafon, M. (1998). The neural cell adhesion molecule is a receptor for rabies virus. J. Virol. 72, 7181–7190.

Tuffereau, C., Bénéjean, J., Blondel, D., Kieffer, B., and Flamand, A. (1998). Low-affinity nerve-growth factor receptor (P75NTR) can serve as a receptor for rabies virus. EMBO J. 17, 7250–7259.

Ugolini, G., and Hemachudha, T. (2018). Rabies: changing prophylaxis and new insights in pathophysiology. Curr. Opin. Infect. Dis. 31, 93–101.

Wang, J., Wang, Z., Liu, R., Shuai, L., Wang, X., Luo, J., Wang, C., Chen, W., Wang, X., Ge, J., et al. (2018a). Metabotropic glutamate receptor subtype 2 is a cellular receptor for rabies virus. PLoS Pathog. 14, e1007189.

Wang, Q.-S., Zhang, K.-H., Cui, Y., Wang, Z.-J., Pan, Q.-Y., Liu, K., Sun, B., Zhou, H., Li, M.-J., Xu, Q., et al. (2018b). Upgrade of macromolecular crystallography beamline BL17U1 at SSRF. Nucl. Sci. Tech. 29, 68.

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

Whitt, M.A., Buonocore, L., Prehaud, C., and Rose, J.K. (1991). Membrane fusion activity, oligomerization, and assembly of the rabies virus glycoprotein. Virology 185, 681–688.

Zhang, S., Lu, G., Qi, J., Li, Y., Zhang, Z., Zhang, B., Fan, Z., Yan, J., and Gao, G.F. (2013). Competition of cell adhesion and immune recognition: insights into the interaction between CRTAM and nectin-like 2. Structure 21, 1430–1439.

Zhang, W.-Z., Tang, J.-C., Wang, S.-S., Wang, Z.-J., Qin, W.-M., and He, J.-H. (2019). The protein complex crystallography beamline (BL₁₉U₁) at the Shanghai Synchrotron Radiation Facility. Nucl. Sci. Tech. 30, 170.

Zhou, Z., Khaliq, M., Suk, J.E., Patkar, C., Li, L., Kuhn, R.J., and Post, C.B. (2008). Antiviral compounds discovered by virtual screening of small-molecule libraries against dengue virus E protein. ACS Chem. Biol. 3, 765–775.

STAR★METHODS

KEY RESOURCES TABLE

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
| --- | --- | --- |
| Antibodies |  |  |
| Mouse monoclonal anti-Flag | Zen BioScience | Cat# 390002 |
| Goat polyclonal anti-mouse IgG | Zen BioScience | Cat# 511103 |
| Goat polyclonal anti-human IgG | Merck Millipore | Cat#AP113P; RRID:AB_11214132 |
| Mouse monoclonal anti-RABV-N | Merck Millipore | Cat#5500; RRID:AB_917372 |
| SOJB | This paper | N/A |
| SO57 | This paper | N/A |
| EP5G3 | This paper | N/A |
| SOJA | This paper | N/A |
| GD2D12 | This paper | N/A |
| 523-11 | This paper | N/A |
| Fab094 | This paper | N/A |
| 5E6 | (Qiu et al., 2011) | N/A |
| 523-11-scFv | This paper | N/A |
| Bacterial and Virus Strains |  |  |
| E.coli BL21(DE3) | Transgen | Cat#CD601 |
| E.coli DH5α | Transgen | Cat#CD201 |
| E.coli DH10Bac | Invitrogen | Cat#10361012 |
| Rabies virus Flury strain | LUFANG BIOLOGY | N/A |
| Chemicals, Peptides, and Recombinant Proteins |  |  |
| DMEM medium | Gibco | Cat#C11995500BT |
| Fetal bovine serum (FBS) | Gibco | Cat#10437-028 |
| SIM SF medium | Sino Biological Inc. | Cat#MSF1 |
| SIM HF medium | Sino Biological Inc. | Cat#MHF1 |
| Penicillin | Sangon | Cat#A600135 |
| Streptomycin | Sangon | Cat#A610494 |
| BICINE | Hampton Research | Cat#HR2-509 |
| Ammonium phosphate dibasic | Sigma-Aldrich | Cat#09839 |
| Polyethylene glycol 300 | Hampton Research | Cat#HR2-517 |
| Polyethylene glycol 3350 | Hampton Research | Cat#HR2-527 |
| Glycine | Sangon | Cat#A610235 |
| Paraformaldehyde | Biosharp | Cat#BL539A |
| Sucrose | BioFroxx | Cat#1245GR500 |
| Glutaraldehyde | Sangon | Cat#A600875 |
| Sulfo-EGS | Sangon | Cat#C120220 |
| BS³ | Thermo Fisher Scientific | Cat#21580 |
| Tween-20 | Sangon | Cat#A100777 |
| T32:SGEWVMKDYRGWKHWVYYTCCPDTPYLDDITYH | GL Biochem Corporation | N/A |
| C32:SGEWVIKESRGWKHWVFYACCPSTPYLDITYH | GL Biochem Corporation | N/A |
| H32:SGEWVIKESRGWKHSVTYSCCPDTPYLDITYH | GL Biochem Corporation | N/A |
| Deposited Data |  |  |
| RABV-G-ΔFuLp(pH 8.0) | This paper | PDB:6LGX |
| tr-RABV-G-ΔFuLp/523-11-scFv(pH 6.5) | This paper | PDB:6LGW |

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
|---------------------|--------|------------|
| **Experimental Models: Cell Lines** |  |  |
| Spodoptera frugiperda (Sf9) cell | Invitrogen | Cat# 11496015 |
| Trichoplusia ni (Hi5) cell | Invitrogen | Cat# B85502 |
| HEK293T cell | ATCC | Cat# ATCC CRL-3216 |
| Baby hamster kidney (BHK-21) cell | ATCC | Cat# ATCC CRL-10 |
| **Recombinant DNA** |  |  |
| pFastBac1-RABV-G-ΔFuLp | This paper | N/A |
| pFastBac1-tr-RABV-G-ΔFuLp | This paper | N/A |
| pFastBac1-VSV-G-ΔFuLp | This paper | N/A |
| pCAGGS-Full-length-RABV-G-CVS-11 | This paper | N/A |
| pCAGGS-Full-length-RABV-G-Flury | This paper | N/A |
| pCAGGS-Full-length-RABV-G-SADB19 | This paper | N/A |
| pCAGGS-Full-length-RABV-G-D211A | This paper | N/A |
| pCAGGS-Full-length-RABV-G-H261A | This paper | N/A |
| pCAGGS-Full-length-RABV-G-D266A | This paper | N/A |
| pCAGGS-Full-length-RABV-G-E269A | This paper | N/A |
| pCAGGS-Full-length-RABV-G-D266A/E269A | This paper | N/A |
| pCAGGS-Full-length-RABV-G-ΔFuLp | This paper | N/A |
| pCAGGS-SOJB-H | This paper | N/A |
| pCAGGS-SOJB-L | This paper | N/A |
| pCAGGS-SO57-H | This paper | N/A |
| pCAGGS-SO57-L | This paper | N/A |
| pCAGGS-EP5G3-H | This paper | N/A |
| pCAGGS-EP5G3-L | This paper | N/A |
| pCAGGS-SOJA-H | This paper | N/A |
| pCAGGS-SOJA-L | This paper | N/A |
| pCAGGS-GD2D12-H | This paper | N/A |
| pCAGGS-GD2D12-L | This paper | N/A |
| pCAGGS-523-11-H | This paper | N/A |
| pCAGGS-523-11-L | This paper | N/A |
| pCAGGS-Fab094-H | This paper | N/A |
| pCAGGS-Fab094-L | This paper | N/A |
| pET-21a-523-11-scFv | This paper | N/A |
| pEGFP-N1 | Invitrogen | N/A |
| **Software and Algorithms** |  |  |
| GraphPad Prism 6 | GraphPad | https://www.graphpad.com/ |
| HKL2000 | (Otwinowski and Minor, 1997) | http://www.hkl-xray.com/ |
| CCP4 | (Collaborative Computational Project, 1994) | http://www.ccp4.ac.uk/ |
| PHENIX | (Adams et al., 2010) | http://www.phenix-online.org/ |
| COOT | (Emsley and Cowtan, 2004) | https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/ |
| PyMOL | Schrödinger | https://pymol.org/ |
| BI Ae valuation 4.1 | GE Healthcare | N/A |
| MEGA 5 | (Tamura et al., 2011) | https://www.megasoftware.net/ |
| WebLogo service | (Crooks et al., 2004) | http://weblogo.berkeley.edu/logo.cgi |
| MOE 2018.01 | Chemical Computing Group | https://www.chemcomp.com/ |

(Continued on next page)

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

---

**Continued**

| REAGENT or RESOURCE | SOURCE          | IDENTIFIER       |
|---------------------|-----------------|------------------|
| Other              |                 |                  |
| HisTrap excel      | GE Healthcare   | Cat# 17-3712-06  |
| Source 15Q         | GE Healthcare   | Cat# 17-0947-05  |
| Superdex 200 Increase 10/300 GL | GE Healthcare   | Cat# 28-9909-44  |
| HiLoad 16/600 Superdex 75 pg | GE Healthcare   | Cat# 28-9893-33  |
| HiTrap rProtein A FF | GE Healthcare   | Cat# 17-5079-01  |
| 96-well microtiter plates | Corning        | Cat# 3590       |
| Amicon Stirred Cell concentrator | Merck Millipore | Cat# UFSC 40001 |
| BIACore X-100     | GE Healthcare   | N/A             |
| FE28 pH meter     | Mettler-Toledo Instruments | Cat# B 547736899 |
| LE438 electrode   | Mettler-Toledo Instruments | N/A             |

---

**LEAD CONTACT AND MATERIALS AVAILABILITY**

Plasmids and mAbs generated in this study will be made available on request but may require a payment and/or a completed Material Transfer agreement if there is potential for commercial application. Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Guangwen Lu ([lugw@scu.edu.cn](mailto:lugw@scu.edu.cn)).

---

**EXPERIMENTAL MODEL AND SUBJECT DETAILS**

**Cell Lines and Viruses**

Spodoptera frugiperda (Sf9) cells and Trichoplusia ni (Hi5) cells were individually maintained in the SFM SF medium and the SFM HF medium using a non-humidified shaker at 27°C.

HEK293T and baby hamster kidney (BHK-21) cells were maintained in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal bovine serum (FBS), 100 units of Penicillin and 0.1 mg/ml of Streptomycin with 5% CO₂ at 37°C. The rabies virus Flury strain was propagated and titered in BHK-21 cells.

All *E. coli* strains were cultured in Lysogeny broth (LB) medium (1% w/v Tryptone, 0.5% w/v yeast extract and 1% w/v NaCl) using a non-humidified shaker at 37°C.

---

**METHOD DETAILS**

**Protein Expression and Purification**

The RABV-G-ΔFuLp, tr-RABV-G-ΔFuLp, and VSV-G-ΔFuLp proteins used for further experiments (crystallization, ELISA, sucrose density gradient centrifugation, analytical gel filtration chromatography, analytical ultra-centrifugation) were expressed using the Bac-to-Bac baculovirus expression system (Invitrogen). The coding sequences (codon optimized for insect cells) for full-length glycoprotein of rabies virus CVS-11 strain (GenBank: ADJ29911.1) and vesicular stomatitis virus (VSV) Indiana strain (GenBank: ACK77584.1) were synthesized by the GENEWIZ Corporation. RABV-G-ΔFuLp (residues 1-439, replacing residues 73-79 and 117-125 with the Gly-Gly-Ser-Gly-Gly linker, respectively), tr-RABV-G-ΔFuLp (residues 1-405, replacing residues 73-79 and 117-125 with Gly-Gly-Ser-Gly-Gly) and VSV-G-ΔFuLp (residues 1-422, replacing residues 70-77 and 115-119 with Gly-Gly-Ser-Gly-Gly) were individually sub-cloned into the pFastBac1 vector. For each construct, a previously described gp67 signal peptide sequence ([Lu et al., 2013](https://doi.org/10.1016/j.jmb.2013.01.010)) was added to the protein N terminus for protein secretion, and a 6×His tag was added to the C terminus to facilitate further purification processes. Transfection and virus amplification were conducted with Sf9 cells, and the recombinant proteins were produced in Hi5 cells. The cell culture supernatants were collected 96 hours after infection and passed through a 5-ml HisTrap excel column (GE Healthcare) for primary purification. The recovered proteins were further purified on a Source 15Q column (GE Healthcare) and then on a Superdex 200 Increase 10/300 GL column (GE Healthcare). Finally, the proteins were exchanged into a buffer consisting of 20 mM Tris-HCl (pH 8.0) and 150 mM NaCl for further use.

**Single Chain Fragment Variable (scFv) Preparation**

The antibody scFvs were prepared for crystallization screening in complex with the RABV-G proteins. In total of seven scFvs, including SOJB, SO57, EP5G3, SOJA, GD2D12, 523-11, and Fab094, were expressed as inclusion bodies in *E. coli* and refolded in vitro. For each scFv, the variable regions of the light chain (*V*ₗ) and heavy chain (*V*ₕ) were joined with a (Gly-Gly-Gly-Ser)₄ linker. Briefly, the encoding fragment for each scFv was sub-cloned into pET-21a (Novagen) via the NdeI and BamHI restriction sites. The resultant recombinant plasmids were then transformed into *E. coli* BL21 (DE3) for expression. Expression was conducted with the

e3 Cell Host & Microbe 27, 1–13.e1–e7, March 11, 2020

Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

addition of 500 μM isopropyl-β-D-thiogalactoside (IPTG) followed by induction at 37°C for about 6 hours. Inclusion bodies were then extracted and refolded as previously described ([Zhang et al., 2013](https://doi.org/10.1016/j.chom.2019.12.012)). In brief, aliquots of inclusion bodies were diluted dropwise into the refolding buffer (100 mM Tris-HCl (pH 8.0), 400 mM L-Arg HCl, 2 mM EDTA, 5 mM reduced glutathione, 0.5 mM oxidized glutathione) at the final protein concentration of 0.3 mg/ml and incubated overnight to complete the refolding process. Subsequently, the refolded protein was concentrated using an Amicon Stirred Cell concentrator (Merck Millipore) with a 10-kDa cutoff membrane and then adjusted to a buffer consisting of 20 mM Tris-HCl (pH 8.0) and 150 mM NaCl. The scFv proteins were then further purified by gel filtration using a HiLoad 16/600 Superdex 75 pg column (GE Healthcare).

### Recombinant Full-Length Antibody Preparation

The recombinant full-length antibodies were prepared in the study for ELISA, conformational epitope verification, cell-cell fusion inhibition, and virus neutralization assays. All the antibodies were expressed in HEK293T cells in the IgG1 subtype. The constant-region sequences containing residues 144 to 473 of a human immunoglobulin IgG1 heavy chain (GenBank: CAA75030.1) and residues 131 to 236 of a human immunoglobulin kappa light chain (GenBank: CAA75031.1) were cloned into the pCAGGS vector to yield the backbone plasmids pCAGGS-HC and pCAGGS-LC, respectively. The \(V_{\mathrm{H}}\) and \(V_{\mathrm{L}}\) sequences of the antibodies described above were then first engineered to include an IL-2 signal peptide coding sequence and then sub-cloned into pCAGGS-HC and pCAGGS-LC, generating the final plasmids encoding the heavy chain and light chain of the individual IgG1 antibodies. For antibody expression, the plasmid pairs (the heavy chain plasmid pairing with the corresponding light chain plasmid) were co-transfected into HEK293T cells. At 48 hours post-transfection, the supernatants containing the recombinant antibodies were harvested and subjected to HiTrap rProtein A FF column (GE Healthcare) for purification. Finally, the antibodies were eluted with 0.1 M glycine (pH 3.0) and immediately buffer-exchanged into PBS (136 mM NaCl, 2.6 mM KCl, 8 mM \(\mathrm{Na}_{2} \mathrm{HPO}_{4}\), 2 mM \(\mathrm{KH}_{2} \mathrm{PO}_{4}\), pH 7.4). The purity of the antibodies was analyzed by reducing and non-reducing SDS-PAGE and visualized by staining with Coomassie blue. An IgG1-type antibody (5E6) targeting Ebola virus glycoprotein was prepared from insect cells and is used as an isotype-control antibody.

### Indirect ELISA

For the binding between RABV-G-ΔFuLp and monoclonal antibodies, 96-well microtiter plates (Corning) were coated with purified RABV-G-ΔFuLp at 200 ng/well in 0.05 M carbonate-bicarbonate coating buffer (pH 9.6) overnight at 4°C. The wells were then blocked for 1 hour at room temperature with PBS containing 5% skimmed milk. After blocking, 5-fold serially-diluted full-length antibodies (SOJB, S057, EP5G3, SOJA, GD2D12, 523-11, Fab094, and 5E6 as an isotype-control) were added to the wells and incubated for 1.5 hours, followed by the addition of goat anti-human IgG-HRP (Merck Millipore) and incubation for another 1 hour. In each step, the plates were fully washed with PBST. The reaction was conducted in dark at room temperature for about 3 minutes and was stopped with 2 M HCl. The emission OD450 was monitored using a microplate reader (Thermo). Three independent experiments were conducted and the data was graphed using Prism 6 software (GraphPad).

### Analytical Gel Filtration Chromatography

Purified RABV-G-ΔFuLp and tr-RABV-G-ΔFuLp were individually mixed with 523-11 scFv at the indicated pHs (for pH 8.0 in a buffer consisting of 20 mM Tris-HCl (pH 8.0), 150 mM NaCl, and for pH 6.5 in a buffer consisting of 20 mM MES-NaOH (pH 6.5), 150 mM NaCl), followed by incubation at 4°C for 2 hours. The samples, including the individual RABV-G proteins (RABV-G-ΔFuLp and tr-RABV-G-ΔFuLp), 523-11-scFv and their mixtures (RABV-G-ΔFuLp/523-11-scFv and tr-RABV-G-ΔFuLp/523-11-scFv), were then analyzed with a calibrated Superdex 200 Increase 10/300 GL (GE Healthcare) column. The chromatographs were recorded and overlaid onto each other. The pooled proteins were analyzed on SDS-PAGE gel and visualized by staining with Coomassie blue. The gel filtration fractions containing the 523-11-scFv/RABV-G-ΔFuLp and 523-11-scFv/tr-RABV-G-ΔFuLp complexes were concentrated to 8 mg/ml in 20 mM Tris-HCl (pH 8.0), 150 mM NaCl for further crystallization trials.

### Crystallization

All crystals were obtained by the vapour-diffusion sitting-drop method. In brief, 1 μl protein was mixed with 1 μl reservoir solution. The subsequent mixture was then equilibrated against 90 μl reservoir solution at 18°C. The initial crystallization screenings were carried out using the commercial crystallization kits (Hampton Research and Molecular Dimensions). The conditions that yield crystals were then optimized. Diffractable crystals of the free RABV-G-ΔFuLp protein were obtained in an optimized condition consisting of 0.1M BICINE (pH 8.0) and 14% Polyethylene glycol 300 with a protein concentration of 7.5 mg/ml. For the tr-RABV-G-ΔFuLp/523-11-scFv complex, the crystals were grown in the No.2-2 condition of Morpheus MD 1-46 kit (containing 30 mM Diethylene glycol, 30 mM Triethylene glycol, 30 mM Tetraethylene glycol, 30 mM Pentaethylene glycol, 55.5 mM MES monohydrate and 44.5 mM Imidazole (pH 6.5), 20% v/v Ethylene glycol, and 10% w/v PEG 8000) with a protein concentration of 8 mg/ml.

### Data Collection and Structure Determination

For data collection, all crystals were flash-cooled in liquid nitrogen after a brief soaking in reservoir solution supplemented with 20% (v/v) glycerol. Diffraction data were collected at Shanghai Synchrotron Radiation Facility (SSRF) beamlines BL17U1 and BL19U1 ([Wang et al., 2018b; Zhang et al., 2019](https://doi.org/10.1016/j.chom.2019.12.012)). All data were processed with HKL2000 ([Otwinski and Minor, 1997](https://doi.org/10.1016/j.chom.2019.12.012)) for indexing, integration and scaling. The initial phases for the tr-RABV-G-ΔFuLp/523-11-scFv complex were determined by molecular replacement with the PHASER program ([Read, 2001](https://doi.org/10.1016/j.chom.2019.12.012)) in CCP4 suite using the V-region coordinates from the PDB entry of 4M43 as the search model.
Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

Unambiguous placement of the 523-11-scFv heavy and light chains provided the necessary phases to extend the map to cover tr-RABV-G-ΔFuLp using iterative rounds of model building and density modification in PHENIX ([Adams et al., 2010](https://doi.org/10.1016/j.jmb.2010.04.036)), COOT ([Emsley and Cowtan, 2004](https://doi.org/10.1107/S0907444904019158)) and REFMAC5 ([Murshudov et al., 1997](https://doi.org/10.1107/S0907444997001806)). The free RABV-G-ΔFuLp structure was solved by molecular replacement using PHASER ([Read, 2001](https://doi.org/10.1107/S0907444901004875)) with the individual domains of the solved tr-RABV-G-ΔFuLp structure as the search models. Finally, the atomic models were completed with COOT ([Emsley and Cowtan, 2004](https://doi.org/10.1107/S0907444904019158)) and further refined with PHENIX ([Adams et al., 2010](https://doi.org/10.1016/j.jmb.2010.04.036)). The stereochemical quality of the final model was assessed through the program PROCHECK ([Laskowski et al., 1993](https://doi.org/10.1006/jmbi.1993.1509)). The final data processing and structure refinement statistics are summarized in [Table 1](#table1). All structural figures were generated using PyMOL ([http://www.pymol.org](http://www.pymol.org)).

### Cell–Cell Fusion Assay for Syncytia Formation and the Fusion Inhibition by Antibody 523-11

Sequences encoding full-length RABV-G of the CVS-11 strain (GenBank: ADJ29911.1), SAD-B19 strain (UniProKB: P16288.1) and Flury strain (GenBank: ADD84790.1), and the RABV-G mutants based on the glycoprotein sequence of the CVS-11 strain (including D211A, H261A, D266A, E269A, D266A/E269A) were first engineered to include a C-terminal Flag tag (DYKDDDDK) and then sub-cloned into pCAGGS vector to yield the expression plasmids (WT and mutant pCAGGS_RABV-G). For protein expression, BHK-21 cells were pre-seeded on 12-well plates for overnight culturing. When 80% confluency was obtained, the cells were co-transfected with pEGFP-N1 and the WT or the mutant pCAGGS_RABV-G plasmids. Cells transfected with pEGFP-N1 alone were used as the negative control. After 24 hours, cells were treated with the fusion buffer (20 mM MES in DMEM) for 10 minutes at 37°C at pH 5.5 (measured by a FE28 pH meter equipped with LE438 electrode (Mettler-Toledo Instruments)). The Cells were then washed with DMEM containing 20 mM HEPES (pH 7.0) once, followed by incubation in DMEM culture medium at 37°C for 1 hour. Finally, the cells were visualized for syncytia formation by fluorescence microscope (Olympus).

For the fusion inhibition experiment, the cells were first pre-incubated at 37°C for 1 hour with the indicated antibodies (5E6 as an isotype control and 523-11) at 100 nM, and then low-pH treated at pH 5.5 at 37°C for 10 minutes. Following washing, the cells were treated and cultivated as described above before visualization for syncytia formations by fluorescence microscope.

### Immunofluorescent Assay Verifying Surface-Location of WT and Mutant RABV-G

BHK-21 cells were seeded on glass coverslips in 24-well plates. The cells (on glass coverslip) were transfected with the WT or the mutant pCAGGS_RABV-G plasmids. The blank pCAGGS plasmid was transfected serving as the negative control. Cells were fixed for 30 min with 4% PFA (paraformaldehyde) 24 hours post-transfection and then blocked for 1 hour at room temperature with PBS containing 5% skimmed milk. Cells were incubated with the full-length antibody SOJB at room temperature for 2 hours, followed by washing and incubation with Alexa Fluor 488 conjugated goat anti-human IgG (BBI) for 1.5 hour. Following washing, cells were incubated with DAPI solution to stain nucleus. Images were visualized by LSM 880 (Zeiss).

### Virus Neutralization by Antibody 523-11

BHK-21 cells were first seeded in 96-well plates for overnight culturing. 50 μl rabies virus stock solution (2×10³ TCID₅₀/ml, Flury strain) was pre-incubated with 50 μl DMEM, or 50 μl 5E6 antibody solution (an isotype-control antibody at 200 nM) or 50 μl 523-11 antibody solution (at 200 nM) at 37°C for 1 hour. Subsequently, the virus mixture was added to the BHK-21 monolayers in 96-well plates in quadruplet and incubated for 1 hour. The supernatant was then removed and replaced with fresh DMEM medium. The cells were cultivated for another 48 hours at 37°C, fixed with 4% paraformaldehyde, stained by FITC-labeled RABV N-protein monoclonal antibody (Merck Millipore) supplemented with 0.025% (w/v) evans blue, and finally visualized by fluorescence microscope (Olympus).

### Protein Oligomerization Assay

The oligomeric states of RABV-G-ΔFuLp and VSV-G-ΔFuLp were analyzed by sucrose density gradient centrifugation and analytical ultra-centrifugation.

The sucrose gradient centrifugation assay was performed as previously described with small modifications ([Baquero et al., 2015](https://doi.org/10.1016/j.virusres.2015.02.010)). In brief, 150 μg of the indicated proteins were first incubated in 200 μl buffer containing either 50 mM MES-NaOH (adjusted to pH 5.0 and pH 6.5, respectively) or 50 mM HEPES-NaOH (adjusted to pH 8.0) plus 100 mM NaCl at 4°C for 1 hour. The protein solution was then overlaid on a 5-to-20% w/v continuous sucrose gradient prepared in 100 mM NaCl, 50 mM MES-NaOH or 100 mM NaCl, 50 mM HEPES-NaOH at the indicated pHs. Following centrifugation at 35,000 rpm for 16 hours, 12 equal fractions (1 ml/fraction) were collected manually from the bottom of the gradient. The collected samples were then analyzed by SDS-PAGE and visualized by staining with Coomassie blue.

The analytical ultracentrifugation assay was performed in a Beckman Coulter XL-I analytical ultracentrifuge using two-channel centerpieces. RABV-G-ΔFuLp or VSV-G-ΔFuLp were buffer-exchanged to 50 mM MES-NaOH (pH 5.0), 100 mM NaCl or 50 mM HEPES-NaOH (pH 8.0), 100 mM NaCl at the concentration of A₂₈₀ = 0.8 absorbance units and incubated at 4°C for 1 hour. Data were collected via absorbance detection at 18°C with a rotor speed of 45,000 r.p.m. The SV-AUC data were globally analyzed using the SEDFIT program and were fitted to a continuous c(s) distribution model to determine the molecular mass of each protein.

e5 Cell Host & Microbe 27, 1–13.e1–e7, March 11, 2020
Please cite this article in press as: Yang et al., Structural Analysis of Rabies Virus Glycoprotein Reveals pH-Dependent Conformational Changes and Interactions with a Neutralizing Antibody, Cell Host & Microbe (2019), https://doi.org/10.1016/j.chom.2019.12.012

# Cell Based Cross-Linking Assay

The oligomeric states of the cell-surface-located WT RABV-G (Full-length RABV-G) and the fusion-loop substituted form of RABV-G (Full-length RABV-G-ΔFuLp) under different pHs were characterized by chemical cross-linking using the amine-group specific and water-soluble cross-linkers, including Glutaraldehyde (Sangon), Sulfo-EGS (Sangon), and BS³ (Thermo). In brief, sequences encoding full-length RABV-G or full-length RABV-G-ΔFuLp (whose fusion loop residues 73–79 and 117–125 were individually replaced with the Gly-Gly-Ser-Gly-Gly linker) were first engineered to include a C-terminal Flag tag (DYKDDDDK) and then sub-cloned into pCAGGS vector. The subsequent expression plasmids were then individually transfected into HEK293T cells for protein expression. Forty-eight hours post-transfection, the cells were harvested and washed thoroughly with 5 mM HEPES (pH 7.0), 150 mM NaCl. About 1.0×10⁵ cells for each sample were incubated in the buffers containing 50 mM MES-NaOH (adjusted to pH 5.5, 6.0, and 6.5, respectively) or 50 mM HEPES-NaOH (adjusted to pH 7.5) plus 150 mM NaCl at room temperature for 30 minutes. The aforementioned cross-linkers at the indicated final concentrations (Glutaraldehyde: 0%, 0.000625%, 0.00125%, 0.0025%, 0.005%; Sulfo-EGS: 0, 0.625, 1.25, 2.5, 5 mM; BS³: 0, 0.078, 0.156, 0.313, 0.625 mM) were then added to the cells. After a 30-min incubation at room temperature, the reactions were quenched by the addition of 50 mM Tris-HCl (pH 7.5).

For the western blot analysis, the above-prepared samples were separated by 8% SDS-PAGE and transferred to PVDF membranes. After blockage using 5% skimmed milk in TBS (20 mM Tris-HCl (pH 7.5), 150 mM NaCl), the membranes were incubated with the anti-Flag antibody (Zen BioScience) for 1.5 hours and then with the HRP-conjugated goat anti-mouse IgG (Zen BioScience) antibody for 1 hour. For each step, the membranes were fully washed with TBST. The final signal was detected using the SuperLumia ECL Kit (Abbkine) following the manufacturer’s instructions.

# Surface Plasmon Resonance Assay

All the SPR assays were carried out using the BIACore X-100 system with the CM5 chip (GE Healthcare) at room temperature.

To characterize the interactions between RABV-G-ΔFuLp and the nAchR-derived peptides, three peptides (T32: SGEWVMK DYRGWKHWVYYTCCPDTPYLDDITYH, C32: SGEWVIKESRGWKHWVFYACC PSTPYLDITYH, H32: SGEWVIKESRGWKHSVTYSCC PDTPYLDDITYH) were chemically synthesized by GL Biochem Corporation. For all the measurements, the peptide and the RABV-G-ΔFuLp protein were prepared in a PBST buffer consisting of 1×PBS and 0.005% (v/v) Tween-20. To clearly visualize the binding, about 9000 response units of RABV-G-ΔFuLp were immobilized on the chip. Gradient concentrations of the indicated peptides were then flowed over RABV-G-ΔFuLp on the chip surface. After each cycle, the sensor surface was regenerated via a 30-second treatment using 10 mM NaOH.

To determine the affinities between RABV-G-ΔFuLp and antibody 523-11 at pH 8.0 and pH 6.5, 523-11 scFv was immobilized on the chip with about 800 response units. Gradient concentrations of RABV-G-ΔFuLp in HEPES buffer (20 mM HEPES (pH 8.0), 150 mM NaCl, 0.005% (v/v) Tween-20) or MES buffer (20 mM MES (pH 6.5), 150 mM NaCl, 0.005% (v/v) Tween-20) were then flowed over 523-11 scFv on the chip surface. To characterize the self-binding homo-interactions for RABV-G-ΔFuLp, the protein was immobilized on the chip with about 800 response units. Gradient concentrations of RABV-G-ΔFuLp in HEPES buffer (20 mM HEPES (pH 8.0), 500 mM NaCl, 0.005% (v/v) Tween-20) or MES buffer (20 mM MES (pH 5.5), 500 mM NaCl, 0.005% (v/v) Tween-20) were then flowed over the chip surface. After each cycle, the sensor surface was regenerated via a 15-second treatment using 5 mM NaOH.

In each case, the obtained kinetic data were analyzed with the BLAevaluation software 4.1 for dissociation constant calculations, using the 1:1 Langmuir binding model for the slow-on/slow-off data and the steady-state affinity model for the fast-on/fast-off data, respectively.

# Verification of the Conformational Epitope Recognized by 523-11

Briefly, 400 ng RABV-G-ΔFuLp protein were firstly subjected to SDS-PAGE and then transferred to PVDF membranes. Following blockage with TBS (20 mM Tris-HCl (pH 7.5), 150mM NaCl) supplemented with 5% skimmed milk at room temperature for 1.5 hours, the membranes were incubated with the indicated antibodies (SOJB, 523-11, and 5E6 as an isotype control) at room temperature for 1.5 hours and then with the HRP-conjugated goat anti-human IgG (Merck Millipore) antibody for 1 hour. For each step, the membranes were fully washed with TBST. The final signal was detected using the SuperLumia ECL Kit (Abbkine) following the manufacturer’s instructions.

# Sequence Logo Visualization Using WebLogo

A total of 2215 RABV-G sequences were downloaded from NCBI. Amino acid sequence extending from 333 to 388 (based on the mature protein sequence) was selected for alignment and then visualized for sequence-conservation using the WebLogo service ([Crooks et al., 2004](https://www.weblogo.net)).

# Identification of Potential Druggable-Pockets on the Basic-pH Structure of RABV-G

The site finder function of the program Molecular Operating Environment (MOE) 2018.01 ([https://www.chemcomp.com/](https://www.chemcomp.com/)) was used to identify surface pockets around the domain junctions.

# Trimer Model Building for RABV-G

For trimer model building, we initially tried to construct the models by superimposing three copies of RABV-G derived from the basic-pH structure individually to the three VSV-G molecules in the pre-fusion trimer (PDB: 5I2S) and aligning three copies of RABV-G from

the acidic-pH structure onto the VSV-G molecules in the post-fusion trimer (PDB: 5I2M), respectively. We noted that when using the three-domain RABV-G molecule as a whole, it would lead to steric conflicts among the three central helices (helix C) in the CD domain. We also noted that previous structural studies on vesiculovirus glycoproteins have highlighted an orientation difference of CD relative to PHD between the intermediate and the post-fusion structures ([Baquero et al., 2017](https://doi.org/10.1016/j.chom.2019.12.012)). We therefore first separated the RABV-G CD domain from the PHD-FD domains and then constructed the trimeric models by superimposition of the CD and PHD-FD domains individually onto the equivalent domains of VSV-G. We also omitted the dislocated C-element from the CD domain when building the models, which would otherwise also cause steric conflicts with the neighboring molecule. This leads to the pre- and post-fusion trimeric architectures for RABV-G without conflicts among the three protomers.

**QUANTIFICATION AND STATISTICAL ANALYSIS**

Indirect ELISA data reported in [Figure 1](#fig1)D were conducted with triplicate measurements and standard deviations (SDs) were calculated using GraphPad Prism 6.

**DATA AND CODE AVAILABILITY**

All data generated or analyzed during this study are included in this published article (and its [Supplemental Information](#suppinfo)). Atomic coordinates and structure factors of the reported crystal structures have been deposited in the Protein Data Bank ([https://www.rcsb.org](https://www.rcsb.org); PDB: 6LGX, 6LGW).
