
Pathomechanism Heterogeneity in the Amyotrophic Lateral Sclerosis and Frontotemporal Dementia Disease Spectrum: Providing Focus Through the Lens of Autophagy

Rebecca L. Casterton${}^{a}$, Rachel J. Hunt${}^{a}$ and Manolis Fanto${}^{a,b}$

${}^{a}$ - Department of Basic and Clinical Neuroscience, King's College London, 125 Coldharbour Lane, SE5 9NU London, United Kingdom  
${}^{b}$ - Institut du Cerveau et de la Moelle épineère (ICM), 47, bd de l'hôpital, F-75013 Paris, France  

Correspondence to Manolis Fanto: Department of Basic and Clinical Neuroscience, King's College London, 125 Coldharbour Lane, SE5 9NU London, United Kingdom. [manolis.fanto@kcl.ac.uk](mailto:manolis.fanto@kcl.ac.uk)  
[https://doi.org/10.1016/j.jmb.2020.02.018](https://doi.org/10.1016/j.jmb.2020.02.018)  
Edited by Viktor Korolchuk  

### Abstract

Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) constitute aggressive neurodegenerative pathologies that lead to the progressive degeneration of upper and lower motor neurons and of neocortical areas, respectively. In the past decade, the identification of several genes that cause these disorders indicated that the two diseases overlap in a multifaceted spectrum of conditions. The autophagy-lysosome system has been identified as a main intersection for the onset and progression of neurodegeneration in ALS/FTD. Genetic evidence has revealed that several genes with a mechanistic role at different stages of the autophagy process are mutated in patients with ALS/FTD. Moreover, the three main proteins aggregating in ALS/FTD, including in sporadic cases, are also targeted by autophagy and affect this process. Here, we examine the varied dysfunctions and degrees of involvement of the autophagy-lysosome system that have been discovered in ALS/FTD. We argue that these findings shed light on the pathological mechanisms in the ALS/FTD spectrum and conclude that they have important consequences both for treatment options and for the basic biomolecular understanding of how this process intersects with RNA metabolism, the other major cellular process reported to be dysfunctional in part of the ALS/FTD spectrum.

© 2020 Elsevier Ltd. All rights reserved.

---

### Introduction

Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease characterised by the degeneration of upper motor neurons in the motor cortex and lower motor neurons in the spinal cord ([Fig. 1](#fig1)). This results in progressive muscle weakness, paralysis, and eventually death, which predominantly occurs owing to respiratory failure [1]. In approximately 95% of cases, ALS arises sporadically without any previous family history, with the remaining 5% of all cases resulting from an inherited familial mutation [2]. ALS is typically an adult-onset disease with a relatively aggressive progression, with a median survival of approximately 3 years after diagnosis. In general, familial and sporadic disease forms are phenotypically indistinguishable [3].

Frontotemporal dementia (FTD) is a grouping of neurodegenerative diseases caused by selective neurodegeneration of the frontal and temporal cortices ([Fig. 1](#fig1)). Clinically, FTD has several variants that differ in accordance with their phenotypic combination of disordered behaviour, language and personality symptoms [4]. At the post-mortem neuropathological level, all clinical variants display frontal and temporal lobar degeneration (FTLD), of which there are several variants categorised by the species of protein aggregate found in affected neurons [5]. Similar to ALS, FTD exists in both sporadic and familial forms, although FTD has a slightly higher familial occurrence than ALS, with reports of 30–50% of patients having an inherited genetic mutation [6].

It is now well established that ALS and FTD overlap and relate to one another on a disease spectrum [6]. The two diseases can either occur separately or simultaneously in affected individuals, and the latter situation is described as ALS/FTD ([Fig. 1](#fig1)). The evidence for this has been established from combined clinical and neuropathological

© 2020 Elsevier Ltd. All rights reserved.
Journal of Molecular Biology (2020) 432, 2692–2713

Autophagy in Amyotrophic Lateral Sclerosis and Frontotemporal Dementia 2693

Genetic Mutations

SOD1  
FUS  
TARDBP  
PFN1  
FIG4  
ALS2  

ALS

C9ORF72  
VCP  
TBK1  
p62/SQSTM1  
OPTN  
UBQLN2  
ATP13A2  
DCTN  
VAPB  

ALS/FTD

MAPT  
GRN  
CHMP2B  

FTD

Spinal motor neurons  
Motor cortex Betz cells  
ALS-FUS  
ALS-TDP  
ALS-SOD1  

Frontal & temporal cortical neurons  
FTLD-tau  
FTLD-TDP  
FTLD-U  
FTLD-FUS  

Associated Clinical Phenotypes

Progressive muscle weakness  
UMN + LMN signs  
Muscle atrophy  
Muscle fasciculations  

Emotional blunting  
Impaired executive skills  
Personality change  
Confrontational behaviour  
Speech and/or language deficits  

Fig. 1. The ALS/FTD spectrum of neurodegenerative pathologies. ALS and FTD are linked in the form of a disease spectrum: individuals may be affected only by one disease, or alternatively an individual diagnosed with ALS may go on to also develop FTD (and vice versa). This combined condition is described as ALS/FTD. At the genetic level, some genes are known to strongly associate differentially with just one end of the spectrum, whilst the majority of identified genetic loci (listed here in the centre of the spectrum) can result in both diseases either separately or as combined ALS/FTD. At the neuropathological level, the two diseases affect distinct neuronal populations but share a commonality in the protein aggregates present by which they are neuropathologically classified. At the level of clinical phenotype, there is a large degree of heterogeneity. ALS/FTD is inherently more clinically heterogeneous, as in addition to the phenotype variation within ALS and FTD as separate entities, the balance of how strongly each disease contributes to the overall combined ALS/FTD clinical phenotype must also be considered. This is highly variable between individual cases. Pure ALS clinical phenotypes are generally less heterogeneous by comparison, and variation in presentation is fairly well accounted for, for example, due to bulbar or spinal onset. FTD, by contrast, as a grouping of diseases in itself is far more complex. Therefore, it is important to note that the clinical phenotypes listed here can occur in different overlapping combinations and vary in severity according to the particular type of FTD. FTD, frontotemporal dementia; ALS, amyotrophic lateral sclerosis.

observations and prolific genetic screening over the past two decades. At the clinical level, cognitive changes indicative of frontotemporal dysfunction are found in around 50% of patients with ALS [7], of which 15% meet clinical diagnostic criteria for FTD [8] and in diagnosed patient with FTD, a similar percentage have also been found to subsequently develop ALS [9]. ALS and FTD also share an overlap at the neuropathological level, the classification system for which has undergone frequent revisions, reflecting an increasing complexity and apparent disease heterogeneity as new genetic loci and proteins involved in pathology are identified [5,10,11]. The pathological hallmark of disease within the ALS/FTD spectrum is the presence of ubiquitinated neuronal cytoplasmic inclusions of the RNA-binding protein, TAR DNA-binding protein-43 kDa (TDP-43). This coincides with an observed loss of TDP-43 from the nucleus, where it usually localises. The cytoplasmic TDP-43 inclusions are frequently found to be phosphorylated and to contain truncated C-terminal fragments of the protein [12]. TDP-43 pathology applies to both sporadic and familial cases of ALS, FTD and ALS/FTD and is found in both the presence and absence of a causal mutation in TAR DNA-binding protein (TARDBP), which encodes the TDP-43 protein [13]. Cytoplasmic TDP-43 aggregates are found in affected neurons of 97% of ALS cases, and 50% of FTD cases, familial and sporadic combined [6]. There are some exceptions to this, for example, ALS arising from superoxide dismutase 1 (SOD1) mutations, wherein cases are distinguished by ubiquitinated cytoplasmic inclusions that are negative for TDP-43 immunoreactivity [14]. SOD1 is a free radical scavenger, converting superoxide anions to hydrogen peroxide to permit their elimination from the cell. Mutations in SOD1 are among the most frequently identified mutations in ALS and are found throughout the length of the protein [15]. Mutations in the RNA-binding protein fused in sarcoma (FUS) are also associated with ALS [16,17] and FTD [18] and provide an additional exception to the hallmark TDP-43 aggregate

neuropathology. Cytoplasmic mislocalisation of FUS, and FUS-positive inclusions have been found in the brain tissue of patients with ALS/FTD both with and without FUS mutations and are mostly negative for TDP-43 inclusions [16–18]. For FTD, where TDP-43 inclusions are not observed in affected neurons at post-mortem, tau, FUS or ubiquitin make up the predominant pathological protein inclusion [11] (Fig. 1). As in other neurodegenerative diseases, several nonexclusive mechanisms have been proposed to explain how proteinaceous aggregates lead to the neuronal cell death in ALS/FTD, including toxic oligomeric formations [19], phase separations and sequestration of other factors [20] and alterations of nucleocytoplasmic transport [21,22]. It has also been recently suggested that dysfunction in the autophagy-lysosome system and nuclear misfolded protein aggregates can lead to a specific form of cell death, karyoptosis [23,24], resulting in the type of nuclear degeneration often reported in ALS/FTD models [21,22].

At the genetic level, more than 25 genes have currently been associated with ALS, FTD and ALS/FTD and although some of these known genetic loci can cause all three diseases, some differentially associate with pure ALS, pure FTD or ALS/FTD. For instance, mutations in SOD1 are more commonly linked to pure ALS, whilst mutations in Microtubule-associated protein tau (MAPT) or Granulin (GRN) are associated with pure FTD [6]. For combined ALS/FTD, many causal mutations have been identified. The most common is a hexanucleotide GGGGCC repeat expansion mutation in Chromosome 9 Open Reading Frame 72 (C9ORF72) [10,25], the identification of which was crucial to conceptualisation of the ALS/FTD disease spectrum.

ALS/FTD is a complex disease with a highly heterogeneous aetiology for which there is currently no satisfactory treatment available. A challenge to therapy, as well as our fundamental understanding of what causes selective neurodegeneration in ALS/FTD, is that almost every neuronal cellular function is

![Diagram](attachment:diagram.png)

Fig. 2. Proteins involved in ALS/FTD and the autophagy cycle. Proteins identified to be relevant for ALS/FTD through genetic mutations or aggregation affect the autophagy-lysosome system at a number of stages. Expression of ATG proteins are affected by TDP43. VAPB and the C9ORF72 proteins affect autophagosome initiation. The two autophagy receptors p62 and OPTN and the kinase that phosphorylates them, TBK1, affect the formation and cargo recognition of autophagosomes. Ubiquilin2 and possibly CHMP2B and ALS2 affect the closure and maturation of the autophagosome. Progranulin and VCP affect formation of functional lysosomes. Dynactin1, Kinesin1 and possibly Profilin1 and AnnexinA11 affect lysosomal transport. VCP, CHMP2B, ATP13A2, CACNA1A and EPG5 control autophagosome lysosome fusion. FIG4 and SPG11 affect auto-lysosome clearance and lysosome recycling. The key aggregating proteins, TDP-43, SOD-1, FUS, the DPRs formed from C9ORF72, as well as the RNA stress granules are all subject to autophagic degradation and this may fail in the disease. FTD, frontotemporal dementia; FUS, fused in sarcoma; DPRs, dipeptide repeat proteins; OPTN, optineurin; VAPB, vesicle-associated membrane protein-associated protein B; ALS, amyotrophic lateral sclerosis; FIG4, factor-induced gene 4.
affected or has been implicated at some stage in pathology [26]. However, a bird's eye view of implicated genes and their functions reveals two key pathological molecular processes: RNA processing and autophagy. Both TDP-43 and FUS are RNA binding proteins and RNA toxicity has been proposed to be a key pathomechanism in ALS/FTD caused by mutations in C9ORF72 (see the following section). A detailed description of RNA processing dysfunctions in ALS/FTD is beyond the scope of this article and has been covered in a number of authoritative reviews [6,27].

Autophagy is a fundamental cellular degradative pathway conserved throughout evolution with important roles in homeostasis, removal of defective proteins and organelles, and adaptation to changing metabolic demands [28]. The process of autophagy can be divided into multiple steps (Fig. 2). Briefly, upon autophagy initiation, autophagosomes are formed from cup-shaped structures called phagophores; after cargo recognition by proteins associated to the phagophore, the membranes expand and close to form the autophagosome. Autophagosomes fuse with lysosomes, leading to the formation of autolysosomes and degradation of autophagic cargo. Finally, both nutrients and membranes are recycled. As ALS/FTD is characterized by accumulation of misfolded proteins, functional autophagy is essential to protect neurons from toxic protein aggregates and a reduction of autophagy often promotes their fatal accumulation. Therefore, the conceptualisation of ALS/FTD as an autophagy disease is an attractive hypothesis that deserves significant consideration and autophagy-related therapeutic strategies might potentially be of significance for ALS/FTD. Here, we focus on the evidence that links dysfunctions in the autophagy-lysosome system to the pathology of ALS/FTD and also discuss the relevance of these specific dysfunctions to the basic understanding of pathomechanisms and autophagy-restoring therapy development.

most common cause of ALS/FTD, causing up to ~33% of familial ALS cases and ~5% of sporadic ALS cases [10,25,29]. There are three nonmutually exclusive hypotheses of how the hexanucleotide expansion confers neurotoxicity: (1) haploinsufficiency of the C9ORF72 protein itself; (2) RNA foci formed from the transcribed G4C2 repeat expansion; and (3) five different dipeptide repeat proteins (DPRs) formed via the abnormal phenomenon of repeat-associated non-ATG (RAN) translation of hexanucleotide expansion RNA [30].

C9ORF72 ALS/FTD patient fibroblasts show an increase in p62 positive autophagosomes [31]. In addition, C9ORF72 patient-derived iPSC neurons demonstrate p62 accumulation, are less able to tolerate cell stress from pharmacological inhibition of autophagy [32], and also display abnormally enlarged autophagosomes [31]. The C9ORF72-generated DPRs have been shown to be degraded by autophagy and the ubiquitin-proteasome system (UPS) [33,34], and at least some of them colocalise in aggregates with p62 in post-mortem brains [35,36].

Current evidence indicates C9ORF72 functions as a guanine nucleotide exchange factor (GEF), activating Rab GTPases and enabling their regulation of membrane trafficking [37]. However, it has also been suggested that C9ORF72 potentially functions as a GTPase activating protein (GAP) [38] which generally cause signal cessation by induction of GTP hydrolysis [39]. C9ORF72 has indeed been shown to interact with several Rab proteins [40], in a manner which implies it may have a role in autophagy initiation (Fig. 2). For instance, C9ORF72 can interact with Rab1a and the ULK1 autophagy initiation complex. The interaction with Rab1a appears to subsequently regulate transportation of ULK1 to newly forming phagophores [41]. In accordance with a role in autophagosome formation, C9ORF72 has also been found to localise with LC3-positive puncta under basal conditions [40]. In C9ORF72 patient-derived induced motor neurons (iMNs) a pH-sensitive GFP-RFP-LC3 autophagosome-lysosome fusion reporter indicated that C9ORF72 iMNs have fewer unfused autophagosomes, potentially owing to autophagosome formation impairment [42].

Further to C9ORF72 interacting with Rab proteins to facilitate ULK1-mediated autophagosome formation, it has been established that C9ORF72 forms a protein complex with SMCR8 and WD repeat-containing protein 41 (WDR41), which then interacts with ULK1 to induce autophagy [42,43]. This complex has also been demonstrated to interact weakly with p62 and OPTN [44]. Knockdown of either C9ORF72 or SMCR8 in vitro results in fewer autophagosomes at basal conditions, no increase in autophagosomes in response to rapamycin treatment, and normal autophagosome levels upon

ALS/FTD Genes with a Mechanistic Link to the Autophagy-Lysosome System

The genetic discoveries of the last decade have identified numerous genes mutated in patients with ALS/FTD that have a mechanistic link to the autophagy process, including lysosomal clearance (Fig. 2). Here, we discuss them in detail, stressing the role they have in autophagy and the dysfunctions of the autophagy-lysosome system caused by their mutations in ALS/FTD.

C9ORF72

A GGGGCC (G4C2) hexanucleotide repeat expansion mutation in intron 1 of the C9ORF72 gene is the
starvation: collectively suggesting impaired autophagy induction [43]. Consistent with this, a reduction in mTOR activation is reported in C9ORF72 knockout mice [45].

Examination of the role of the C9ORF72/SMCR8 complex in Smcr8 knockout mice and fibroblasts showed that when Smcr8 is absent, mTORC1 and AKT activity is upregulated. However, intriguingly the same is not reported in C9ORF72-deficient fibroblasts, despite Smcr8 levels being significantly reduced. This raises the question as to the relative importance of C9ORF72 within the complex [38].

Besides an involvement in autophagy initiation and autophagosome formation, C9ORF72 has a potential role in lysosome function and trafficking. The evidence for this stems from several observations. First, in serum-starved HEK cells C9ORF72 has been shown to shift from a diffuse cytoplasmic distribution to a lysosomal localisation, preferentially over autophagosome localisation [46]. However, regarding the interplay between C9ORF72 and lysosomes, there may be some discrepancies between different cell types, which ought to be taken into consideration in future investigation, as a recent study in C9ORF72 ALS/FTD patient-derived iMNs found only 20% of C9ORF72 localised to lysosomes. Furthermore, this level of localisation was comparable with control iMNs [47]. In keeping with its known functional interaction with Rab proteins, C9ORF72 may be involved in the regulation of trafficking events at the endolysosomal level which mediate autophagy. C9ORF72 was found to be associated with lysosomes in N2A cells and furthermore was observed to synchronously migrate with lysotracker-positive vesicles [40]. It is also possible that lysosome function regulation is more specific to C9ORF72 itself than any of its known complex partners. For instance, whilst C9ORF72 knockout HeLa cells exhibit enlarged lysosomes and late endosomes with a dense perinuclear localisation, SMCR8 knockout cells do not. Furthermore, the possibility that C9ORF72/SMCR8 regulates lysosomal function as a means of coordinating the mTORC1 response to changes in amino acid availability is potentially ruled out, as mTOR recruitment to lysosomes in response to amino acid starvation was maintained in both C9ORF72 and SMCR8 knockout cells [46].

Finally, there is conflicting evidence as to whether C9ORF72 loss alters lysosome number: an increase has been reported in C9ORF72 knockout mouse embryonic fibroblasts (MEFs) [45], bone-derived macrophages and macroglia but not in primary cortical neurons in a C9ORF72 null mouse model [48] or C9ORF72 patient motor cortex and spinal cord [48]. The opposite effect has also been reported in C9ORF72 patient-derived iMNs and in C9ORF72 null mouse motor neurons, which display fewer lysosomes than controls. This has been taken to

suggest that a loss of C9ORF72 may also have an impact on lysosome biogenesis, which has also recently been hypothesised to have a link with glutamate excitotoxicity in C9ORF72 ALS/FTD [49].

It is inconclusive whether the C9ORF72 hexanucleotide expansion mutation pathologically increases or impairs autophagy. A recent study which examined the effect of C9ORF72 as a known risk factor for corticobasal degeneration (CBD) dissected the effect of repeat expansion size in C9ORF72 CBD patient-derived iPSC neural progenitors. Typically, a large repeat expansion size above pathological threshold is known to be associated with pathological RNA foci and DPRs. However, this study found that intermediate repeat expansion size was not associated with these two features, and instead reported an increase in C9ORF72 expression, accompanied by gene expression changes in autophagy and vesicle trafficking pathways [50]. Repeat expansion size is known to be heterogeneous (in the range of ~30 to over 1000) in C9ORF72 ALS/FTD [10,51], and the findings of this study raise the interesting idea that there may be varying pathogenesis and pathological phenotypes within C9ORF72 ALS/FTD relating to autophagy, depending upon expansion size. This is a level of complexity that will require further consideration in models and patient research in future.

In summary, there is substantial evidence that C9ORF72 has several, potentially complex, roles in autophagy. However, it remains to be established whether C9ORF72 haploinsufficiency causes dysfunction in these roles, and whether this is sufficient to contribute to disease pathogenesis. Indeed, the current consensus in the field tends towards the two other hypothesised causes of C9ORF72 ALS/FTD pathology: hexanucleotide expansion RNA foci and expanded DPRs, as having a relatively greater contribution to disease pathogenesis. There is support for this in the fact that to date no mouse models of C9ORF72 haploinsufficiency display a clear ALS/FTD phenotype linked to neurodegeneration [30]. However, C9ORF72 reduction and autophagy impairment clearly enhance the hexanucleotide expansion pathology as recently demonstrated [52]. Nevertheless, current evidence highlights C9ORF72 as a peculiar case in the relation between autophagy and ALS/FTD: the protein itself has an implied role in autophagy, which appears to be affected in disease, but only an ancillary role in C9ORF72-based ALS/FTD disease pathogenesis.

Progranulin

Progranulin (PGRN) is a secreted growth factor encoded by the granulin (GRN) gene. Heterozygous mutations in GRN, which result in PGRN deficiency

are predominantly associated with FTD on the ALS/FTD disease spectrum and reportedly account for 20% of familial FTD cases [53, 54].

There is considerable genetic evidence that autophagy and lysosome function are implicated in PGRN pathology (Fig. 2). A recent genome-wide RNAi screen performed in Neuro2A (N2A) cells reported an enrichment of autophagy-lysosome pathway gene hits which were capable of increasing PGRN levels [55]. In an RNA-seq analysis of the cortex of heterozygous and homozygous knockout PGRN mice, approximately 10% of upregulated genes in homozygous PGRN knockout mice were associated with lysosome function [56].

The depletion of PGRN appears to be consistently associated with impairment of autophagy flux in experimental models. For instance, accumulation of the lysosomal membrane protein LAMP1 is observed in PGRN null MEFs [57], and is also recapitulated in FTD patient brains [58]. In another PGRN null mouse model, an age-dependent cortical accumulation of LAMP1 and of the autophagy cargo receptor p62 is demonstrated, in addition to elevated expression of the autophagy-regulating transcription factor TFEB and of the lysosomal enzyme Ctsd [59]. Autophagic flux assays performed in neurons of PGRN homozygous null mice showed potential dysfunction at the level of autophagosome clearance, and an accumulation of undigested LC3-positive autophagosomes was also observed *in vivo* [56]. Interestingly, a link has been suggested between PGRN deficiency in FTD and impairment of TDP-43 clearance. PGRN null mice demonstrate hallmark cytoplasmic TDP-43 accumulation [60] and *in vitro*, PGRN null mouse cortical neurons transfected with a cytoplasmic localising TDP-43 construct show an accelerated accumulation of TDP-43 compared with wildtype mouse neurons [56]. There has also been some indication that GRN, derived from PGRN, may have a functional role at lysosomes. *In vitro*, PGRN was found to colocalise with LAMP1 in HeLa cells but GRNs were found to be the most concentrated in lysosomal cell lysate fractions, and indeed were seen to be localised to lysosome membranes [61].

Thus, PGRN appears to be implicated in lysosome function in autophagy. However, there are outstanding questions regarding its relevance to different cell types, and whether pathology is cell autonomous. In this respect, autophagy defects in PGRN FTD may be particularly relevant to pathogenesis in microglia, as PGRN knock out microglia have been shown to display an altered turnover of lysosomal cathepsins. Interestingly, this has been hypothesised as a potential compensatory mechanism for impairment of lysosomes and effective autophagic protein clearance in neurons [57].

---

Optineurin

Optineurin (OPTN) is a phosphoprotein with well-characterised roles in vesicular trafficking and NF-kB anti-viral and antibacterial signalling. Similar to p62 (see the following section), OPTN is a known autophagy receptor (Fig. 2) primarily for xenophagy and mitophagy [62], possessing both an LC3-interacting region (LIR) and a ubiquitin-associated domain (UBA). Mutations in OPTN are responsible for up to 1% of familial and ~3% of sporadic ALS [63]. In HeLa cells, both wild-type and mutant OPTN colocalised with mutant huntingtin (htt) and SOD1 inclusions in a ubiquitin-independent manner. Aggregation of mutant htt and SOD1 was increased by knockdown of OPTN but decreased by the autophagy activator rapamycin. The aggregation caused by OPTN shRNA was rescued by overexpression of wild-type and of UBA-mutant OPTN. However, the aggregation was not rescued by OPTN with a mutation in the LIR, or treatment of the cells with rapamycin, indicating overall that OPTN can function to clear proteinaceous aggregates via a ubiquitin-independent autophagy pathway [64]. OPTN also colocalised with *in vivo* inclusion bodies of mutant htt in a transgenic mouse model, and a TDP-43 truncation mutant in N2A cells, but these interactions were largely abrogated by mutation of the OPTN UBA domain. Overexpression of wild-type OPTN reduced the presence of htt inclusion bodies, but this reduction was abolished by treatment with the autophagy inhibitors bafilomycin A₁ and 3MA, or deletion/mutation of the OPTN LIR, demonstrating that OPTN can also degrade neurotoxic aggregates by canonical ubiquitin-dependent autophagy. Overexpression of an OPTN mutant in the UBA domain precipitated an increase in htt inclusions, which was attributed to the dominant-negative sequestering of endogenous wild-type OPTN into nonfunctional hybrid oligomers [65].

As well as facilitating autophagy via the recruitment of cargo to the phagophore, OPTN has been shown to induce autophagosome formation [66] and promote autophagosome-lysosome fusion [67, 68]. In MEFs, OPTN recruited the E3 ligase-like complex ATG12-5-16L1 to the phagophore where the complex conjugates phosphatidylethanolamine to LC3-I, creating LC3-II. LC3-II is required for the closure of the phagophore into the autophagosome. Expression of the ALS-associated mutant OPTN<sup>E478G</sup> in OPTN<sup>-/-</sup> MEFs resulted in deficient autophagosome formation [69]. In N2A cells, autophagosome maturation was shown to be reciprocally dependent on binding of OPTN to the small GTPase Rab1a — knockdown of either component reduced autophagosome formation — suggesting that Rab1a may play a role in the recruitment of ATG12-5-16L1 by OPTN [70]. Later in the autophagy pathway, the

molecular motor myosin VI is required for autophagosome-lysosome fusion [67]. Myosin VI delivers the endocytic membranes needed for autolysosome formation to the autophagosome via interaction with the endosomal adaptor Tom1, and autophagy receptors — including OPTN — on the autophagosome [67]. The ALS-associated OPTN<sup>E478G</sup> and OPTN<sup>Q398X</sup> mutations in the myosin VI binding domain, which overlaps the UBA domain, reduced OPTN-myosin VI binding and autophagosome-lysosome fusion. Consistently, the physical association of OPTN with myosin VI was reduced in the spinal cords of ALS cases compared with controls [68].

Overall, the multifaceted role of OPTN in the autophagy process suggests that patients with ALS/FTD with an OPTN mutation may develop their pathology through an impairment of the autophagy pathway.

### TANK-binding kinase 1

The multidomain protein TANK-binding kinase 1 (TBK1) is a component of many biological pathways, including interferon induction, AKT and Ras signaling, and cell division. Mutations causing ALS and FTD have been identified within all regions of TBK1, including the serine/threonine kinase domain, the ubiquitin-like domain (ULD) and both C-terminal coiled-coil domains (CCD1/2) [71–74]. The reported incidence of TBK1 mutations varies four-fold between sporadic and familial ALS (~0.9% versus ~4%) [71,72]. A role for TBK1 in autophagy was uncovered simultaneously with that of OPTN: phosphorylation of OPTN by TBK1 was found to increase the binding affinity of OPTN for LC3, and the degradation of the *Salmonella* bacterium by autophagy [62]. Subsequently, another autophagy receptor, p62, was also identified as a target of TBK1. Phosphorylation of p62 Ser-403 by TBK1 enhanced the binding of p62 to K48/63 ubiquitinated substrates and the subsequent clearance of the cargo by autophagy [75]. ALS/FTD-associated mutations in the kinase domain, ULD and CCD2 of TBK1 have all been shown to abolish the interaction with, and thus phosphorylation of, OPTN [72,76]. The G217R mutation in the kinase domain also attenuated the homodimerisation required for activation of TBK1 [76].

Alongside modulation of autophagy receptors, there is mounting evidence that TBK1 can also play multiple roles in autophagy initiation. In HEK293T and HeLa cells, phosphorylation of syntaxin 17 by TBK1 was necessary for relocalisation of syntaxin 17 from the Golgi to the preautophagosome complex Atg13-FIP200. Knockout of either syntaxin 17 or TBK1 reduced the association of Atg13, FIP200 and ULK1 required for autophagosome biogenesis [77]. The C9ORF72 binding partner SMCR8 is also a substrate of TBK1. In neuronal GT1-7 cells, phosphorylation of SMCR8 by TBK1 is required for the autophagy-promoting GEF activity of the C9ORF72/SMCR8 complex [78]. Phosphorylation of the autophagy receptor NDP52 by TBK1 also mediated the interaction of NDP52 with FIP200/ULK1 on mitochondria and peroxisomes, leading to the activation of selective autophagy [79]. In contrast, knockout of TBK1 from MEFs did not affect the early stages of autophagy but did reduce the maturation of autophagosomes to autolysosomes, possibly by inhibiting the delivery of the lysosomal protease Cathepsin D to the autophagosome [75]. TBK1 may also contribute to the maturation of the autophagosome by regulating microtubule dynamics — and thus autophagosome transport — as the mitotic apparatus proteins CEP170 and NuMA are TBK1 substrates, but this has yet to be demonstrated in neurons [80].

The association of TBK1 with multiple stages of the autophagy pathway is reminiscent of its important substrates, OPTN and p62, and the TBK1-OPTN/p62 axis (Fig. 2) represents the most straightforward indication of the causative role of autophagy defects in ALS/FTD.

### p62/SQSTM1

SQSTM1, or p62, is a ubiquitously expressed protein [81] involved in a multitude of cellular stress response, housekeeping and growth pathways. However, it is best known as the prototype autophagy receptor and plays a key role in protein turnover, recognising ubiquitinated substrates (Fig. 2) and targeting them to the autophagic [82] and proteasomal systems [83]. An N-terminal PB1 domain mediates the self-polymerisation of p62 which is required for localisation of p62 to the early autophagosome [84]. At its C-terminal, p62 possesses LIR and UBA domains, which facilitate interactions with the phagophore [85] and ubiquitinated cargo [86], respectively. p62 itself is degraded by autophagy due to its association with LC3-II, and this is particularly evident during the upregulation of autophagy induced by starvation [82,87].

Mutations in p62 have been shown to be the cause of ~1% of ALS cases and ~2% of FTD cases [88–90], and occur throughout the length of the protein [88]. The ALS mutation L341V in the LIR domain has been shown to reduce the affinity of p62 for LC3-II and to impair the degradation of autophagic substrates [91]. Recently, a C-terminally cleaved form of p62, designated p62<sup>TRM</sup>, has been identified in skin fibroblasts. p62<sup>TRM</sup>, which is created by caspase-8 in response to toll-like receptor 3 activation, cannot be metabolised by autophagy during starvation. When the availability of nutrients is restored, p62<sup>TRM</sup> is able to activate mTOR and thus inhibit autophagy. The FTD-associated mutations, p62<sup>D329G</sup> and p62<sup>D329H</sup>, predicted to alter the

caspase-8 cleavage site, did indeed abolish the creation of p62<sup>TRM</sup> and the upregulation of mTOR activity. Deletion of the UBA domain of p62<sup>D329G</sup> was required to re-establish the activation of mTOR, suggesting that the UBA domain is required for the inhibition of autophagy, and that mutations in the p62 UBA domain might cause aberrant upregulation of autophagy [92].

ALS/FTD mutations also occur in the promoter region of p62, suggesting that dysregulation of p62 expression, and a consequent hypothetical alteration to autophagy, may also be pathological [93]. Knockout of p62 from mice expressing the H46R SOD1 mutation increased the build-up of insoluble SOD1 in the spinal cord, accelerated motor neuron degeneration and death of the animals, but interestingly did not impair autophagic flux [94]. Conversely, overexpression of p62 in the SOD1<sup>H46R</sup> mouse also shortened lifespan but did not exacerbate motor neuron loss. Overexpression of p62 caused the accumulation of insoluble aggregates of polyubiquitinated proteins, but the overall assessment of autophagy activation was not definitive [95]. These potentially conflicting results mean that further investigation is needed before conclusions regarding the role of p62 in a possible two-hits' model in ALS/FTD can be drawn.

### Dynactin1

Dynactin1 (DCTN1) is the largest subunit of DCTN, which forms part of a motor complex with dynein, a molecular motor protein. Together, they facilitate retrograde transport of substrates required for autophagy. Several DCTN1 mutations have been associated with ALS/FTD spectrum disease, but it appears that they are a rare cause of ALS, estimated to cause between 0.5% and 3% sporadic ALS cases [96, 97].

DCTN1 conditional knockout mice show an age-dependent motor phenotype, and 20% spinal motor neuron loss by 18 months [98]. There is evidence implicating autophagy in DCTN1 ALS pathology. Spinal motor neurons of Dctn1 conditional knockout mice demonstrate accumulation of LC3-II, LAMP1 and LAMP2 but no reported changes in Atg3, Atg5, Atg7 and LC3-I [98]. Mice overexpressing mutant Dctn1 demonstrate cytoplasmic accumulation of autophagosome and lysosome proteins in spinal motor neurons: a feature associated with impaired autophagosome and lysosome transportation [99, 100]. Knockdown of DCTN1 in HeLa cells redistributes lysosomes to the cell periphery along with an accumulation of autophagosomes. Under basal conditions this does not impair autophagosome-lysosome fusion. However, upon challenging cells with starvation, the rate of autophagosome-lysosome fusion becomes reduced [101].

Overall, evidence so far suggests that autophagy dysfunction associated with DCTN1 mutations is most likely due to impaired transport of autophagosomes and lysosomes (Fig. 2). However, this mechanism has yet to be confirmed in patient with ALS tissue or iPSC neuron models.

#### Valosin-containing protein

Valosin-containing protein (VCP, also known as p97) is a member of the AAA-ATPase family with a significant role in protein quality control and clearance, especially in the endoplasmic reticulum (ER) [102]. More recently, a potential involvement of VCP in protein degradation via the autophagy pathway, particularly in relation to lysosomes (Fig. 2) has been suggested through study of VCP mutations and dysfunction in ALS.

To date, over 45 mutations have been identified in VCP causing inclusion body myopathy with FTD and Paget's disease of bone (IBMPFD) [103] and/or ALS, accounting for 1–2% of familial ALS cases [104]. Accumulation of lysosomes and nondegradative autophagosomes are a feature in muscle [105] and myoblasts *in vitro* from patients with IBMPFD/ALS with VCP mutations [106]. In relation to accumulation of defective lysosomes, it has recently been identified that VCP has a crucial, direct involvement in lysophagy. This process was shown to be impaired in MEFs expressing the IBMPFD/ALS VCP<sup>R155H</sup> mutation and in VCP<sup>R155H</sup> patient muscle fibres [107]. In a VCP knockout mouse model, p62, LC3 and damaged late endosomes were found to accumulate in muscle, resulting in alterations to localisation, activity and signalling of TFEB [108].

VCP has also been implicated in the fusion of functional lysosomes with autophagosomes. For instance, a study in the tubular network of lysosomes found in *Drosophila* muscle sarcoplasm observed that RNAi knockdown or inhibition of VCP resulted in a shift from Atg8 positive vesicle colocalisation with the lysosome network, to a collapsed lysosome network with adjacent Atg8 positive vesicles [109].

Finally, impaired RNA stress granule degradation by autophagy is another area of IBMPFD/ALS pathology in which VCP may have a role. In yeast, inactivation of the VCP homologue Cdc48 resulted in an accumulation of RNA granules [110], which was also demonstrated in HeLa cells, where VCP was knocked down or pharmacologically inhibited [110, 111]. HeLa cells expressing either VCP<sup>R155H</sup> or VCP<sup>A232</sup> also demonstrated this RNA granule phenotype, where additionally, RNA granules contained TDP-43 [110]. Overall, it appears that VCP mutations causing IBMPFD/ALS may be particularly relevant to lysosomal function in autophagy. However, this has so far predominantly been demonstrated in muscle owing to the link with IBMPFD.

remains to be seen if these observations are consistently recapitulated in motor neurons.

### PFN1

Profilin1 (PFN1) is an actin binding protein that is vital for growth and formation of actin filaments from actin monomers, and has been found to be mutated in cases of both sporadic and familial ALS, accounting for an estimated 1–2% of cases [112–115]. PFN1 mutant mice show a progressive, age-dependent loss of motor function and motor neuron degeneration, and display accumulation of p62 in spinal cord motor neurons, as well as PFN1 aggregates positive for p62 [116]. In an SH-SY5Y cell overexpression model of several different mutant variants of PFN1, p62 aggregates and overall cytoplasmic accumulation of p62 were also reported [117]. In this model, LC3 was also found to be sequestered in PFN1 aggregates [117]. Taken together, this suggests that there are likely to be changes to autophagy function arising from PFN1 ALS-associated mutations, and it has been suggested that this may be linked to the observed coaggregation relationship of PFN1 with TDP-43 [117].

However, PFN1 mutations have thus far predominantly been investigated in their capacity to exert pathology through microtubule and axonal defects, which most likely impacts the transport of autophagy pathway components (Fig. 2). Very little is currently understood regarding a mechanism via which PFN1 may have a direct impact through regulation of autophagy.

### Vesicle-associated membrane protein-associated protein B

Mutations in the ER-localised vesicle-associated membrane protein-associated protein B (VAPB) are a rare cause of ALS, occurring at an estimated frequency of up to 0.6% of ALS patients [118]. VAPB binds mitochondrial protein tyrosine phosphatase-interacting protein-51 (PTPIP51) and the interaction tethers mitochondria to the ER. It has been shown that autophagosomes form at ER-mitochondrial contact sites [119]. Overexpression of either VAPB or PTPIP51 in HEK293T and HeLa cells led to a decrease in autophagosome formation, whilst knock-down of either VAPB or PTPIP51 increased autophagosome biogenesis, suggesting that the VAPB-PTPIP51 tether can regulate the initiation of autophagy. Inhibition of the mitochondrial Ca²⁺ uniporter or the IP3 receptor was shown to override both VAPB and PTPIP51 overexpression, restoring autophagosome formation and demonstrating that the VAPB-PTPIP51 tether regulates autophagy by controlling ER-mitochondrial Ca²⁺ transfer [120]. Consistent with these findings, upregulated autophagy markers were detected in mice with a knocked-in ALS-associated mutation, VAPB<sup>P56S</sup>. Mutant VAPB<sup>P56S</sup> mislocalized from the ER to autophagosomes [121]. VAPB has also been implicated in autophagosome formation via its interaction with the membrane trafficking protein Rab3GAP1 [122,123] and its stabilization of ULK1 at autophagosome formation sites in the ER [124]. VAPB also functions to tether the ER to the Golgi complex, mediating the transfer of phosphatidylinositol-4-phosphate (PtdIns4P) from the Golgi to the ER. *Drosophila* null for the VAPB homologue, *vap*, were shown to have higher levels of PtdIns4P in the Golgi, leading to increased formation of endosomes, consequent lysosomal expansion and dysfunction, and autophagy impairment [125].

Overall, there is considerable evidence that VAPB mutations in ALS bring about neurodegeneration through autophagy dysfunctions at the early stages of autophagosome formation (Fig. 2).

### UBQLN2

Mutations in Ubiquilin 2 (UBQLN2) account for 0.3–0.4% of ALS/FTD cases, and are enriched in the PXX repeat region of the protein [126–128]. These mutations have been associated with alterations in the phase separation of UBQLN2 and have yet to be conclusively linked to autophagy. However, mutations in other regions of UBQLN2 may disrupt the identified autophagic functions of the protein [127,128].

UBQLN2 has both a ubiquitin-like domain (ULD) and a UBA domain, and functions as a linker in the UPS, delivering cargo to the proteasome via cargo-UBA and proteasome-ULD interactions, respectively. However, evidence suggests that the function of UBQLN2 in autophagy is not that of an autophagy cargo receptor, rather that it has a role in autophagosome maturation and clearance [129,130]. Recently, loss of the *Drosophila* UBQLN2 orthologue, *Ubiquilin*, was also shown to stall autophagic flux, causing a build-up of autophagosomes and nonacidic autolysosomes. Ubiquilin physically interacted with the lysosomal vacuolar H⁺-ATPase (v-ATPase) V100 subunit, regulating its inclusion into the v-ATPase and the acidification of the lysosome [131], and providing further evidence that the pathological effect of UBQLN2 occurs downstream of cargo reception (Fig. 2).

### CHMP2B

Charged multivesicular protein 2B (CHMP2B) is a subcomponent of the endosomal sorting complex required for transport (ESCRT-III). ESCRT-III is responsible for membrane remodelling and budding in endocytic trafficking and vesicle sorting of proteins

in multivesicular bodies (MVBs), for degradation through fusion with lysosomes.

CHMP2B has been implicated in both FTD and ALS, but mutations are extremely infrequent and account for <1% of ALS-FTD spectrum disease. In patients with FTD, two abnormal transcripts were initially identified. Both arise from the same substitution mutation in the exon 6 acceptor splice site, which results in retention of intron 5, causing the premature truncation of the protein C-terminus during translation [132]. Although CHMP2B mutations were initially thought to only associate with FTD, missense mutations have been subsequently identified in cases of pure ALS [133].

The mechanistic role of CHMP2B in MVB formation, as part of ESCRT-III, has several potential applications in autophagy, and thus the potential to impact autophagy dysfunction driving ALS-FTD. First, frontal cortical neurons of CHMP2B FTD patient brain and patient-derived fibroblasts, display cytoplasmic accumulation of late endosomes and enlarged, abnormally shaped MVBs [134]. This was also observed for the CHMP2B mutations associated with ALS [133].

Cells overexpressing mutant CHMP2B demonstrate functional sorting of proteins into MVBs, even in abnormally structured MVBs but show reduced endosome-lysosome fusion. This suggests that the dysfunction implicated in pathology from mutant CHMP2B is downstream of protein sorting into MVBs and is taking place at the level of endosome-lysosome fusion [134]. In agreement with this, expression of CHMP2Bintron5 in vitro is reported to result in an accumulation of Lysotracker negative autophagosomes [135]. Transgenic mice expressing truncated CHMP2B display the same neuropathological phenotype as patients. These mice develop age-dependent p62 neuronal cytoplasmic inclusions in the brain and spinal cord [136,137], as well as build-up of autofluorescence in close juxtaposition with late endosomes and MVBs and an accumulation of the lysosomal protein LAMP1, characteristic of lysosomal storage disorder [138].

It is less clear whether CHMP2B mutations also impact autophagy at the level of the forming autophagosome. For example, one study in Drosophila and HEK cells suggested that CHMP2B and its genetic modifier syntaxin-13 (STX-13) may act synergistically to facilitate autophagosome closure, and that this may be disrupted in disease where ESCRT-III is dysfunctional, for example, due to a CHMP2B mutation [139]. However, a fluorescence assay recently developed to distinguish open phagophores from closed autophagosomes identified CHMP2A but not CHMP2B to be crucial to phagophore closure [140].

In sum, evidence mainly focussing on the C-terminus truncation CHMP2B mutations predominantly indicates that autophagy function is impacted

at the late endosome and lysosome fusion stages of the pathway (Fig. 2). This is primarily linked to disruption of CHMP2B function in relation to ESCRT-III.

ALS2

The ALS2 gene encodes Alsip, which self-oligomerises and functions as a GEF for the small GTPase Rab5 [141]. ALS2 mutations have been identified in several rare, juvenile motor neuron diseases, including an extremely rare autosomal recessive form of ALS: ALS type 2 [142,143].

Autophagy function, and particularly early stage events involving vesicle and endosome trafficking, may be implicated in ALS pathogenesis arising from ALS2 mutations (Fig. 2). In corticospinal motor neurons of ALS2 knockout mice, an accumulation of p62 and LC3-II, indicative of autophagy dysfunction, was observed [144]. In addition, loss of functional Alsip has been shown to exacerbate toxicity of SOD1 in mutant mice which were also lacking p62. These mice also demonstrated accumulation of large vesicles in spinal cord neurons, attributed to impaired autophagosome formation [94].

Current understanding of Alsip function suggests that any possible role in pathology relating to autophagy may be linked to autophagosome maturation and vesicular trafficking. Furthermore, a more complete mechanistic understanding of how ALS2 modifies SOD1 pathology, and indeed whether a modifying effect can occur in the context of other ALS genes is an additional avenue for future investigation.

Relationship Between Autophagy and the Key Proteins Aggregating in ALS/FTD

Not all genes known to be mutated in ALS/FTD have identified roles in the canonical autophagy-lysosome pathways, but disease pathogenesis has nonetheless been shown to be intimately linked to autophagy in these cases, and frequently so, in sporadic cases not yet attributed to a clear cause.

TDP-43

Cytoplasmic aggregations of the ubiquitinated RNA-binding protein TDP-43 are considered a hallmark pathology in ALS/FTD, although mutations in TARDBP itself only underly 1–4% of ALS and FTD cases [29]. TDP-43 is degraded by multiple catabolic pathways as impairment of either autophagy or the UPS [56,145] leads to the cytoplasmic mislocalisation and accumulation of TDP-43. Motor neuron-
specific knock-out of murine UPS and autophagy pathway components revealed that it was inhibition of the UPS, rather than autophagy, that recapitulated the TDP-43 pathology seen in ALS [146]. However, detailed *in vitro* studies have indicated that turnover of soluble monomeric wild-type TDP-43 is predominantly mediated by the UPS but that autophagy is required for clearance of larger oligomers and aggregates [147, 148]. Consistently, the autophagy cargo receptor p62/SQSTM1 has been found to colocalise with TDP-43 inclusions in the cerebellum, spinal cord and subcortical regions of patients with ALS/FTD [149, 150]. Across a wide range of ALS/ FTD models, using an even wider range of activating compounds, it has been demonstrated that upregulation of autophagy leads to enhanced clearance of TDP-43 aggregates, and amelioration of neurodegenerative symptoms. Stimulating autophagy via the mTOR inhibitor rapamycin has been shown to reduce TDP-43 aggregates in human and murine cell lines [151], the forebrain of an FTD mouse model [152] and a *Drosophila* model of TDP-43 overexpression [153]. An alternative mTOR inhibitor, tamoxifen, also reduced the number of TDP-43 inclusions in the FTLD-U mouse brain, as did the mTOR-independent autophagy activators spermidine and carbamazepine [152]. The compounds fluphenazine and methotrimprazine, which induce autophagy by unknown mechanisms, significantly reduced the presence of mutant TDP-43 aggregates in rat primary neurons, and promoted the survival of human iPSC-derived motor neurons and astrocytes carrying an ALS-associated mutation in TDP-43 [154].

Other studies have identified TDP-43 as a regulator of autophagy. Mice with pan-neuronal overexpression of the 25 kDa C-terminal of TDP-43 had reduced levels of a panel of autophagy markers including Atg3, Atg5, LC3, p62 and Beclin 1, but interestingly there was no downregulation of Atg7 [155]. The 25 kDa C-terminal of TDP-43 lacks the RNA recognition motifs (RRMs) that can bind and stabilise mRNA transcripts, and knockdown of full-length TDP-43 in N2A cells did reduce the level of Atg7 mRNA and protein. This loss was rescued by overexpression of wild-type TDP-43, but not a construct lacking the RRM1, suggesting that TDP-43 acts to stabilise Atg7 mRNA via this domain [156]. Lowered levels of Atg7 were also detected in the hippocampus and spinal cord of mice with conditional knockout of TDP-43, and *Drosophila* null or mutant for the TDP-43 homologue TBPH [157]. Autophagic flux was reduced in both the N2A cell and murine models.

TDP-43 has also been shown to stabilise mRNA encoding the mTOR adaptor protein raptor, via interaction with both its RRM domains. TDP-43 knockdown in both neuronal and non-neuronal cell lines led to a loss of raptor, compromising mTOR activity, and leading to the dephosphorylation and consequent nuclear translocation of the transcription factor TFEB. TFEB regulates the expression of genes controlling lysosome biogenesis and autophagy [158], and these were significantly upregulated by TDP-43 knockdown. However, despite this upregulation, autophagy was found to be impaired, and the defect was traced to a decrease in expression of *DCTN1*, another ALS/FTD gene (see section above), and consequent inhibition of retrograde transport and autophagosome-lysosome fusion [159]. The downregulation of *DCTN1* expression upon TDP-43 knockdown suggests that TDP-43 may also act to stabilise *DCTN1* mRNA. In summary, TDP-43 toxicity in ALS appears to be exacerbated by a feedback loop that downregulates autophagy on loss or mutation of TDP-43, thus amplifying TDP-43 aggregation and consequently further reducing autophagy (Fig. 2).

### Superoxide dismutase 1

Mutations in SOD1 are among the most frequently identified mutations in ALS, accounting for ~15% of familial ALS cases and ~1% of sporadic ALS [29]. In SOD1-mediated ALS, turnover of wild-type and mutant SOD1 is also mediated by both the UPS and autophagy, with inhibition of autophagy promoting the accumulation of both soluble and aggregated SOD1, and neuronal death [160]. The autophagy regulators Beclin 1, p62 and Atg12-Atg5 were found to be raised in the spinal ventral horn and primary motor cortex of post-mortem ALS samples [161]. Cases associated with SOD1 mutation displayed a significantly higher upregulation of all markers in the ventral horn compared with C9ORF72 ALS, or cases of unknown cause, suggesting that autophagy plays a particularly critical role in SOD1 ALS [161].

Autophagy markers are generally found to be augmented in SOD1 mutant models, with evidence suggesting that SOD1 can affect multiple steps in the autophagy pathway. Decreased phosphorylation of the mTOR target site of the autophagy-initiating kinase ULK1 was detected in mice overexpressing the SOD1<sup>G85R</sup> mutant, suggesting inhibition of mTOR signalling and a consequent upregulation of autophagy initiation [162]. Higher levels of LC3-II have been found in both *in vitro* and *in vivo* studies of the SOD1<sup>G93A</sup> mouse, indicating enhanced formation of autophagosomes and cargo recruitment [163, 164]. However, these alterations may also be consistent with clearance impairment and with a stalling between positive autophagy induction signalling and ineffective degradation, also found in other cases of neurodegeneration [165].

Indeed, further downstream in the autophagy process, there is evidence that SOD1 mutations may disrupt autophagosome maturation by impairing axonal retrograde transport. Slower retrogradetransport was found in the SOD1<sup>G93A</sup> mouse, beginning shortly before the onset of motor symptoms. The most acute inhibition of transport occurred at the early symptomatic stage, before the frequency of faster-moving vesicles returned to ~60% of control in the symptomatic and late symptomatic stages. The phenomenon was specific to motor neurons; retrograde transport was not decreased in SOD1-G93A sensory neurons compared with wild-type [166]. Mechanistically, SOD1 mutants have been reported to interact with the retrograde motor protein dynein, both sequestering dynein into cytoplasmic inclusions [167], and out-competing the dynein binding partner snapin, preventing snapin from linking dynein to endosomal cargo [168]. Both phenomena were found to occur whilst the mice were pre-symptomatic, and autophagy was shown to be impaired in the latter study.

Unlike TDP-43 ALS/FTD, the artificial upregulation of autophagy for therapeutic intentions has had mixed results in SOD1 models. Pharmacological activation of autophagy by treatment of the SOD1-G93A mouse with lithium [169], trehalose [170], resveratrol [171] and progesterone [172] has been shown to prolong lifespan and attenuate disease symptoms. Consistently, motor neuron-like NSC34 cells with genetic upregulation of autophagy via p70 S6 kinase inhibition [173] and Beclin 1 overexpression [174] displayed enhanced degradation of SOD1<sup>G93A</sup>. Trehalose treatment of rat primary microglia expressing SOD1<sup>G93A</sup> and SOD1<sup>A4V</sup> upregulated autophagy, reduced microglial activation and the build-up of mutant SOD1, and was protective to cocultured neurons [175]. Conversely, upregulation of autophagy by rapamycin treatment enhanced motor neuron loss and accelerated death in the SOD1<sup>G93A</sup> mouse [176]. Consistent with the latter result, inhibition of autophagy by n-butyldienephthalide reduced motor neuron degeneration and increased lifespan in the SOD1<sup>G93A</sup> model [177,178].

These mixed results suggest that the effects of autophagy in SOD1 ALS are complex, and dependent on additional cofactors or underlying conditions. For example, the suppression of neurodegeneration that was achieved by lithium treatment of the SOD1<sup>G93A</sup> model was obtained in male B6SJL mice [169]. Strikingly, this neuroprotection was not recapitulated in female SOD1<sup>G93A</sup> mice of the C57BL/6J or 129S2/Sv strains, indicating that autophagy in SOD1<sup>G93A</sup> is sensitive to sex and/or genetic background [179]. Further, the lifespans of mice expressing either human SOD1<sup>G93A</sup> or human SOD1<sup>G127X</sup> were decreased by the heterozygous deletion of the autophagosome regulator, Beclin 1. The mice displayed impaired autophagy and an increased amount of insoluble mutant SOD1 [161]. In contrast, Beclin 1 haploinsufficiency in mice transgenic for murine SOD1<sup>G86R</sup> (a model of

human SOD1<sup>G85R</sup>) promoted a longer lifespan, even though it also led to the accumulation of high-molecular-weight aggregates of SOD1 [180]. It is not clear whether the lifespan differences in these models were due to the type of mutation in SOD1, the origin of the transgene, or both. Interestingly, treatment of the SOD1<sup>G93A</sup> mouse with rilmenidine boosted both autophagy and mitophagy [181]. Whilst the authors of this study detected an increase in insoluble SOD1 aggregates, they attributed the enhanced motor neuron degeneration and hind limb paralysis to the dramatic loss of mitochondria from spinal motor neurons, hypothetically owing to the hyperactivated mitophagy. The hypothesis that the functional status of mitochondria may be the master determinant of whether autophagy activation — with minor or major concomitant upregulation of mitophagy — is beneficial in SOD1 ALS is supported by observations from other studies. The lithium and resveratrol treatments that activated autophagy and successfully extended the lifespan of SOD1<sup>G93A</sup> mice also increased the number of motor neuron mitochondria and mitochondrial biogenesis, respectively [169,171]. Where stimulation of autophagy in the SOD1<sup>G93A</sup> mouse by rapamycin was detrimental, the number of vacuolised mitochondria was increased [176], and consistently, where inhibition of autophagy by n-butyldienephthalide was beneficial, mitochondrial morphological defects were decreased [177].

Whilst mitochondrial function could underpin the success or failure of autophagy activation, SOD1 ALS models also exhibit a temporal sensitivity to autophagy induction that may also explain the variation in reported results. SOD1<sup>G93A</sup> mice with knockout of Atg7 in motor neurons showed accelerated decline in the initial stages of the disease, with an earlier onset of tremor and increased denervation of lower limb muscles, but by disease end stage had a higher percentage of innervated lower limb muscles and a substantially longer (>20%) lifespan [182]. Consistently, a study of the SOD1<sup>G93A</sup> mouse lumbar spinal cord that used a food restriction protocol to promote autophagy, found that food restriction was beneficial to mutant SOD1 clearance at disease onset, but increased the amount of mutant SOD1 at the pre-end stage of the disease. However, it was not fully clear if autophagy was successfully induced by the food restriction at all timepoints [183]. Coupled with the findings presented previously, these results suggest that dysregulation of autophagy in SOD1 ALS may comprise a two-pronged assault on motor neuron survival. Inhibition of retrograde transport at the presymptomatic and early disease stages could impair autophagy exactly when it would be most beneficial, whilst detrimental upregulation of autophagy, perhaps by inhibition of mTOR signalling, may occur at later disease stages.
Fused in sarcoma

Mutations in the RNA-binding protein FUS have a prevalence of 3–6% in familial ALS and <1% in sporadic ALS [29]. In FUS-associated ALS/FTD, the autophagy receptor p62 has been found to colocalise with FUS aggregates [184]. Overexpression of both wild-type FUS and ALS-linked mutant FUSR514G and FUSR521C transgenes inhibited autophagy induction and flux in the murine lumbar spinal cord, and at the transcriptomic level, overexpression of wild-type FUS led to an upregulation of lysosome-related genes [185]. Stimulating autophagy with small-molecule inhibitors of the PI3K/Akt/mTOR pathway reduced aggregation of the FUSP525L mutant in human iPSC-derived neurons [186,187]. Consistently, rapamycin treatment of mouse cortical neurons also decreased the aggregation of overexpressed wild-type and FUSR521C [188].

Evidence implicating physiological levels of wild-type FUS in the regulation of autophagy is conflicting. Postnatal knock out of FUS from the mouse central nervous system using a tamoxifen-inducible Cre did not cause degeneration of motor neurons or the neuromuscular junction. Knockout of FUS also did not affect the progression of the ALS-phenotype in mice expressing the FUSP525L mutant [189]. Taken together, these results suggest that FUS toxicity in ALS/FTD is mediated by a gain-of-function mechanism, rather than loss of function. However, there is also mounting evidence that FUS participates in vesicle transport from the ER, which is dependent on Rab1 and required for autophagosome formation [190], suggesting that mutation of FUS in ALS/FTD could result in a loss-of-function autophagy impairment. Expression of mutant TDP-43, SOD1 and FUS has been shown to impair vesicle exit from the ER, with overexpression of Rab1 rescuing the transport deficit in each case [191]. In N2A cells, coexpression of Rab1 restored the phagophore formation and autophagy flux that was impeded by expression of mutant FUS [192]. Current understanding of FUS-mediated ALS/FTD and of any autophagy dysfunction is therefore not as detailed as in TDP-43 and SOD1 ALS/FTD and will require further investigation before the full picture emerges.

and altered autophagic flux has been found in several cases.

Intermediate-length polyglutamine tract expansions in the RNA-binding protein Ataxin-2 (ATXN2), a cause of spinocerebellar ataxia 2, have been associated with ALS [193], and autophagy was impaired in CAG repeat knock-in HEK cells and patient fibroblasts [194].

Polyglutamine repeat expansions in the voltage-gated calcium channel subunit CACNA1A cause Spinocerebellar ataxia 6 and have also been associated with ALS [195]. In Drosophila and mouse cerebellar neurons, CACNA1A mutants cause defects in the fusion of lysosomes with endosomes and autophagosomes [196].

The Vici-syndrome causing gene, EPG5 [197] encodes a factor that regulates tethering and fusion between lysosome and autophagosomes [198] and has been identified as a modifier of the toxicity of C9ORF72-generated DPRs [199]. Intriguingly, the knock-out mice for EPG5 display an ALS-like phenotype [200].

Mutations in the kinesin-1 heavy chain isoform KIF5A, already known to cause a form of spastic paraplegia, have been discovered in patients with ALS [201,202]. The kinesin-1 complex mediates anterograde transport in neurons, including the transport of lysosomes from the cytoplasm to the distal tip of the axon. Fusion of these transported lysosomes with distal autophagosomes is necessary for the retrograde transport and maturation of the autolysosome, and ultimately, the degradation of targeted proteins. Knockdown of KIF5A in rat primary hippocampal neurons resulted in a dramatic loss of lysosomes from the axon, and knockdown of the kinesin-1-lysosome coupling factor BORC caused the accumulation of autophagosomes in the distal axon, suggesting that KIF5A mutations in ALS may impair autophagy by preventing the turnover of synaptic autophagosomes [203].

Mutations in another spastic paraplegia gene, spatacsin (SPG11), have also been linked with juvenile ALS [204], and it has been established that SPG11 is required at the end of the autophagy pathway for the reformation of autophagy-competent lysosomes from autolysosomes. Loss of SPG11 caused a depletion of free lysosomes and accumulation of autolysosomes [205].

The phosphatase Fig4, derived from the factor-induced gene 4 (FIG4), is primarily associated with the demyelinating Charcot-Marie Tooth disease 4J but has also been found to associate with sporadic and familial ALS [206,207]. Neuropathological evidence from mouse and in vitro models suggests that FIG4 mutations give rise to autophagic clearance dysfunction. For instance, Fig4 null mice demonstrate brain and spinal cord accumulation of p62 and LC3-II and lysosomal markers LAMP-1 and LAMP-2 [208]. However, thus far, TDP-43 has not
been associated with FIG4 pathology. No pathological form of FIG4 has been found incorporated into TDP-43 aggregates present in patients with FIG4 ALS [209], and Fig4 knockout mice display typical p62 ubiquitin-positive inclusions which are not associated with TDP-43 and do not show nuclear loss of TDP-43 [208].

The ATPase Type 13A2 (ATP13A2 - also known as PARK9) is a gene associated with a rare inherited autosomal recessive form of juvenile-onset Parkinson's disease with dementia, juvenile-onset hereditary spastic paraplegia and juvenile-onset neuronal ceroid lipofuscinosis [210]. However, mutations in ATP13A2 were recently identified in early-onset patients with ALS/FTD [211]. ATP13A2 is a member of the P-type ATPase family localised at lysosomes and late endosomes, and implicated in the autophagy pathway [210,212]. ATP13A2 knockout mice demonstrate an age-dependent autophagy impairment phenotype, with p62, LC3 and MVB accumulation [213]. In patient fibroblasts, and in cells where ATP13A2 is knocked down, the accumulation of lysosomes has been observed [214], as well as mislocalisation of mutant ATP13A2 from lysosomes to the ER [210,215]. It has been suggested that ATP13A2 may impact autophagy at the autophagosome-lysosome fusion stage, via a lysosome-localising interaction with HDAC6.

### Conclusions and Perspectives

The complexity of ALS/FTD is gradually being untangled by the progress of genetic discovery. Functional studies, including those on autophagy, help to categorize the different forms of pathology and present future challenges both for medical treatment and basic biomolecular understanding. Autophagy enhancement is often considered as a potential treatment in neurodegenerative disease. However, any potential autophagy-enhancing therapies ought to consider the significant differences in pathology emerging within the ALS/FTD spectrum. In consideration of this, we can probably separate the ALS/FTD spectrum into three or four groups. MAPT-based pure FTD and its links to autophagy are not covered in this review. Of those covered here, one grouping is constituted by the SOD1-based pure ALS cases, in which autophagy plays a primary role for the control of the SOD1-aggregation toxicity. In light of the contradictory results for the effects of autophagy enhancement in SOD1-models, it appears that it would be particularly challenging to devise an autophagy-based therapy for this ALS group. A second grouping to consider is FUS-based ALS/FTD, which is possibly connected to TDP-43 but also sufficiently independent to fall in a separate group. However, further elucidation of the

role that the autophagy-lysosome system plays in the pathomechanism of this grouping is required before any rational design of autophagy-based therapies can be undertaken. A final separate group that can be clustered together are the ALS/FTD cases which, irrespective of the initial cause, are characterised by marked TDP-43 pathology. We reason that, based on the current understanding in the field, this set of patients may represent the most promising group for autophagy-enhancing treatments.

In terms of a complete understanding of the pathomechanism of TDP-43 and FUS-based ALS/FTD, however, the future challenge will be the full definition of the interplay between autophagy and RNA processing, the other main molecular pathway disrupted in these subclasses of ALS/FTD. Are these two parallel pathological pathways or is there a hierarchy between the two processes? Is autophagy affecting the disease only through RNA stress granule clearance or is it also relevant for a separate pathomechanism?

Some interconnection between the two pathways has already been revealed in the field. Autophagy has been implicated in the clearance of RNA stress granules associated with ALS/FTD [111,188,216–218]. More recently, the calcium-dependent phospholipid binding protein Annexin A11 was found mutated in ALS/FTD cohorts [219]. Annexin A11 was shown to link RNA stress granules to the cytoplasmic surface of the lysosome, facilitating relocation of the granules to the synapse as the organelle is transported down the axon [220] and this mechanism was disrupted by ALS-causing mutations. Whether the localisation of annexin A11 at the lysosomes implicates a disruption in the way the autophagy-lysosome system handles RNA granules (Fig. 2) is an attractive question that deserves further investigation.

In conclusion, although these studies provide significant information on the interplay between autophagy and RNA processing, it remains to be established whether the only role of autophagy in TDP-43 and FUS-based ALS/FTD is that of handling RNA stress granules or whether (notwithstanding the evident connections with RNA processing) its role remains separate, and more traditionally linked to protein quality control and aggregation toxicity.

---

### Acknowledgements

This work was supported by a grant from the Motor Neurone Disease Association (855-791) to M.F. R.L.C. is supported by a studentship form the UK Medical Research Council.
Author Contributions

R.L.C. wrote the introduction, other parts of the manuscript and assembled Fig. 1. R.J.H. wrote parts of the manuscript. M.F. revised the manuscript and assembled Fig. 2.

Declarations of Interest

None.

Received 6 September 2019;  
Received in revised form 15 February 2020;  
Accepted 17 February 2020  
Available online 29 February 2020  

Keywords:  
Autophagy;  
TDP-43;  
ALS;  
FTD  

References

[1] M.C. Kiernan, S. Vucic, B.C. Cheah, M.R. Turner, A. Eisen, O. Hardiman, et al., Amyotrophic lateral sclerosis, Lancet 377 (2011) 942–955.

[2] M.R. Turner, A. Al-Chalabi, A. Chio, O. Hardiman, M.C. Kiernan, J.D. Rohrer, et al., Genetic screening in sporadic ALS and FTD, J. Neurol. Psychiatry (2017) 1042–1044. England: (c) Article author(s) (or their employer(s) unless otherwise stated in the text of the article) 2017. All rights reserved. No commercial use is permitted unless otherwise expressly granted.

[3] B. Swinnen, W. Robberecht, The phenotypic variability of amyotrophic lateral sclerosis, Nat. Rev. Neurol. 10 (2014) 661–670.

[4] C.V. Greaves, J.D. Rohrer, An update on genetic fronto-temporal dementia, J. Neurol. 266 (2019) 2075–2086.

[5] I.R. Mackenzie, M. Neumann, Reappraisal of TDP-43 pathology in FTLD-U subtypes, Acta Neuropathol. 134 (2017) 79–96.

[6] S.C. Ling, M. Polymenidou, D.W. Cleveland, Converging mechanisms in ALS and FTD: disrupted RNA and protein homeostasis, Neuron 79 (2013) 416–438.

[7] T. van Langenhove, J. van der Zee, C. van Broeckhoven, The molecular basis of the frontotemporal lobar degeneration–amyotrophic lateral sclerosis spectrum. https://doi.org/10.3109/078538902012665471.2012.

[8] G.M. Ringholz, S.H. Appel, M. Bradshaw, N.A. Cooke, D.M. Mosnik, P.E. Schulz, Prevalence and patterns of cognitive impairment in sporadic ALS, Neurology 65 (2005) 586–590.

[9] T. van Langenhove, O. Piguet, J.R. Burrell, C. Leyton, D. Foxe, M. Abela, et al., Predicting development of amyotrophic lateral sclerosis in frontotemporal dementia, J. Alzheimers Dis. 58 (2017) 163–170.

[10] M. DeJesus-Hernandez, I.R. Mackenzie, B.F. Boeve, A.L. Boxer, M. Baker, N.J. Rutherford, et al., Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS, Neuron 72 (2011) 245–256.

[11] M. Neumann, I.R.A. Mackenzie, Review: Neuropathology of non-tau frontotemporal lobar degeneration, Neuropathol. Appl. Neurobiol. 45 (2019) 19–40.

[12] M. Neumann, D.M. Sampathu, L.K. Kwong, A.C. Truax, M.C. Micsenyi, T.T. Chou, et al., Ubiquitinated TDP-43 in frontotemporal lobar degeneration and amyotrophic lateral sclerosis, Science 314 (2006) 130–133.

[13] J. Sreedharan, I.P. Blair, V.B. Tripathi, X. Hu, C. Vance, B. Rogelj, et al., TDP-43 mutations in familial and sporadic amyotrophic lateral sclerosis, Science 319 (2008) 1668–1672.

[14] I.R. Mackenzie, E.H. Bigio, P.G. Ince, F. Geser, M. Neumann, N.J. Cairns, et al., Pathological TDP-43 distinguishes sporadic amyotrophic lateral sclerosis from amyotrophic lateral sclerosis with SOD1 mutations, Ann. Neurol. 61 (2007) 427–434.

[15] D.R. Rosen, T. Siddique, D. Patterson, D.A. Figlewicz, P. Sapp, A. Hentati, et al., Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis, Nature 362 (1993) 59–62.

[16] T.J. Kwiatkowski Jr., D.A. Bosco, A.L. Leclerc, E. Tamrazian, C.R. Vanderburg, C. Russ, et al., Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis, Science 323 (2009) 1205–1208.

[17] C. Vance, B. Rogelj, T. Hortobagyi, K.J. De Vos, A.L. Nishimura, J. Sreedharan, et al., Mutations in FUS, an RNA processing protein, cause familial amyotrophic lateral sclerosis type 6, Science 323 (2009) 1208–1211.

[18] M. Neumann, R. Rademakers, S. Roeber, M. Baker, H.A. Kretzschmar, I.R.A. Mackenzie, A new subtype of frontotemporal lobar degeneration with FUS pathology, Brain 132 (2009) 2922–2931.

[19] E.A. Proctor, L. Fee, Y. Tao, R.L. Redler, J.M. Fay, Y. Zhang, et al., Nonnative SOD1 trimer is toxic to motor neurons in a model of amyotrophic lateral sclerosis, Proc. Natl. Acad. Sci. U. S. A. 113 (2016) 614–619.

[20] F. Gasset-Rosa, S. Lu, H. Yu, C. Chen, Z. Melamed, L. Guo, et al., Cytoplasmic TDP-43 de-mixing independent of stress granules drives inhibition of nuclear import, loss of nuclear TDP-43, and cell death, Neuron 102 (2019) 339–357 e7.

[21] C.-C. Chou, Y. Zhang, M.E. Umoh, S.W. Vaughan, I. Lorenzini, F. Liu, et al., TDP-43 pathology disrupts nuclear pore complexes and nucleocytoplasmic transport in ALS/FTD, Nat. Neurosci. 21 (2018) 228–239.

[22] B.D. Freibaum, Y. Lu, R. Lopez-Gonzalez, N.C. Kim, S. Almeida, K.H. Lee, et al., GGGGCC repeat expansion in C9orf72 compromises nucleocytoplasmic transport, Nature 525 (2015) 129–133.

[23] O. Baron, M. Fanto, Karyoptosis: a novel type of cell death caused by chronic autophagy inhibition, Autophagy (2018) 1–5.

[24] F. Napoletano, O. Baron, P. Vandenameele, B. Mollereau, M. Fanto, Intersections between regulated cell death and autophagy, Trends Cell Biol. 29 (2019) 323–338.

[25] A.E. Renton, E. Majounie, A. Waite, J. Simon-Sanchez, S. Rollinson, J.R. Gibbs, et al., A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD, Neuron 72 (2011) 257–268.

[26] J.H. Weishaupt, T. Hyman, I. Dikic, Common molecular pathways in amyotrophic lateral sclerosis and frontotemporal dementia, Trends Mol. Med. 22 (2016) 769–783.

[27] J.K. Nussbacher, R. Tabet, G.W. Yeo, C. Lagier-Tourenne, Disruption of RNA metabolism in neurological diseases and emerging therapeutic interventions, Neuron 102 (2019) 294–320.

[28] N. Mizushima, M. Komatsu, Autophagy: renovation of cells and tissues, Cell 147 (2011) 728–741.

[29] Z.Y. Zou, Z.R. Zhou, C.H. Che, C.Y. Liu, R.L. He, H.P. Huang, Genetic epidemiology of amyotrophic lateral sclerosis: a systematic review and meta-analysis, J. Neurol. Neurosurg. Psychiatry 88 (2017) 540–549.

[30] R. Balendra, A.M. Isaacs, C9orf72 -mediated ALS and FTD: multiple pathways to disease, Nat. Rev. Neurol. 14 (2018) 544–558.

[31] Y. Aoki, R. Manzano, Y. Lee, R. Dafinca, M. Aoki, A.G.L. Douglas, et al., C9orf72 and RAB7L1 regulate vesicle trafficking in amyotrophic lateral sclerosis and frontotemporal dementia, Brain 140 (2017) 887–897.

[32] S. Almeida, E. Gascon, H. Tran, H.J. Chou, T.F. Gendron, S. DeGroot, et al., Modeling key pathological features of frontotemporal dementia with C9ORF72 repeat expansion in iPSC-derived human neurons, Acta Neuropathol. 126 (2013) 385–399.

[33] R. Cristofani, V. Crippa, G. Vezzoli, P. Rusmini, M. Galbiati, M.E. Cicardi, et al., The small heat shock protein B8 (HSPB8) efficiently removes aggregating species of dipeptides produced in C9ORF72-related neurodegenerative diseases, Cell Stress Chaperones 23 (2018) 1–12.

[34] M. Yamakawa, D. Ito, T. Honda, K. Kubo, M. Noda, K. Nakajima, et al., Characterization of the dipeptide repeat protein in the molecular pathogenesis of c9FTD/ALS, Hum. Mol. Genet. 24 (2015) 1630–1645.

[35] K. Mori, T. Arzberger, F.A. Grasser, I. Gijselinck, S. May, K. Rentzsch, et al., Bidirectional transcripts of the expanded C9orf72 hexanucleotide repeat are translated into aggregating dipeptide repeat proteins, Acta Neuropathol. 126 (2013) 881–893.

[36] K. Mori, S.M. Weng, T. Arzberger, S. May, K. Rentzsch, E. Kremmer, et al., The C9orf72 GGGGCC repeat is translated into aggregating dipeptide-repeat proteins in FTLD/ALS, Science 339 (2013) 1335–1338.

[37] T.P. Levine, R.D. Daniels, A.T. Gatta, L.H. Wong, M.J. Hayes, The product of C9orf72, a gene strongly implicated in neurodegeneration, is structurally related to DENN Rab-GEFs, Bioinformatics 29 (2013) 499–503.

[38] Y. Lan, P.M. Sullivan, F. Hu, SMCR8 negatively regulates AKT and MTORC1 signaling to modulate lysosome biogenesis and tissue homeostasis, Autophagy 15 (2019) 871–885.

[39] J.L. Bos, H. Rehmann, A. Wittinghofer, GEFs and GAPs: critical elements in the control of small G proteins, Cell 129 (2007) 865–877.

[40] M.A. Farg, V. Sundaramoorthy, J.M. Sultana, S. Yang, R.A. Atkinson, V. Levina, et al., C9ORF72, implicated in amyotrophic lateral sclerosis and frontotemporal dementia, regulates endosomal trafficking, Hum. Mol. Genet. 23 (2014) 3579–3595.

[41] C.P. Webster, E.F. Smith, C.S. Bauer, A. Moller, G.M. Hautbergue, L. Ferraiuolo, et al., The C9orf72 protein interacts with Rab1a and the ULK1 complex to regulate initiation of autophagy, EMBO J. (2016) 1656–1676.

[42] P.M. Sullivan, X. Zhou, A.M. Robins, D.H. Paushter, D. Kim, M.B. Smolka, et al., The ALS/FTLD associated protein C9orf72 associates with SMCR8 and WDR41 to regulate the autophagy-lysosome pathway, Acta Neuropathol. Commun. 4 (2016).

[43] M. Yang, C. Liang, K. Swaminathan, S. Herrlinger, F. Lai, R. Shiekhattar, et al., A C9ORF72/SMCR8-containing complex regulates ULK1 and plays a dual role in autophagy, Sci. Adv. 2 (9) (2016) e1601167.

[44] C. Sellier, M. Campanari, C. Julie Corbier, A. Gaucherot, I. Kolb-Cheynel, M. Oulad-Abdelghani, et al., Loss of C9ORF72 impairs autophagy and synergizes with polyQ Ataxin-2 to induce motor neuron dysfunction and cell death, EMBO J. 35 (2016) 1276–1297.

[45] J. Ugolino, Y.J. Ji, K. Conchina, J. Chu, R.S. Nirujogi, A. Pandey, et al., Loss of C9orf72 enhances autophagic activity via deregulated mTOR and TFEB signaling, PLoS Genet. 12 (2016).

[46] J. Amick, A. Roczniai-Ferguson, S.M. Ferguson, C9orf72 binds SMCR8, localizes to lysosomes, and regulates mTORC1 signaling, Mol. Biol. Cell 27 (2016) 3040–3051.

[47] Y. Shi, S. Lin, K.A. Staats, Y. Li, W.H. Chang, S.T. Hung, et al., Haploinsufficiency leads to neurodegeneration in C9ORF72 ALS/FTD human induced motor neurons, Nat. Med. 24 (2018) 313–325.

[48] J.G. O'Rourke, L. Bogdanik, A. Yáñez, D. Lall, A.J. Wolf, A. Muhammad, et al., C9orf72 is required for proper macrophage and microglial function in mice, Science 351 (2016) 1324–1329.

[49] Y. Shi, S.T. Hung, G. Rocha, S. Lin, G.R. Linares, K.A. Staats, et al., Identification and therapeutic rescue of autophagosome and glutamate receptor defects in C9ORF72 and sporadic ALS neurons, JCI Insight 5 (2019).

[50] C.P. Cali, M. Patino, Y.K. Tai, W.Y. Ho, C.A. McLean, C.M. Morris, et al., C9orf72 intermediate repeats are associated with corticobasal degeneration, increased C9orf72 expression and disruption of autophagy, Acta Neuropathol. 138 (5) (2019) 795–811.

[51] A. Iacoangeli, A. Al Khleifat, A.R. Jones, W. Sproviero, A. Shatunov, S. Opie-Martin, et al., C9orf72 intermediate expansions of 24-30 repeats are associated with ALS, Acta Neuropathol. Commun. 7 (2019) 115.

[52] M. Boivin, V. Pfister, A. Gaucherot, F. Ruffenach, L. Negroni, C. Sellier, et al., Reduced autophagy upon C9ORF72 loss synergizes with dipeptide repeat protein toxicity in G4C2 repeat expansion disorders, EMBO J. 39 (4) (2020) e100574.

[53] M. Baker, I.R. Mackenzie, S.M. Pickering-Brown, J. Gass, R. Rademakers, C. Lindholm, et al., Mutations in progranulin cause tau-negative frontotemporal dementia linked to chromosome 17, Nature 442 (2006) 916–919.

[54] M. Cruts, I. Gijselinck, J. van der Zee, S. Engelborghs, H. Wils, D. Pirici, et al., Null mutations in progranulin cause ubiquitin-positive frontotemporal dementia linked to chromosome 17q21, Nature 442 (2006) 920–924.

[55] L.P. Elia, A.R. Mason, A. Alijagic, S. Finkbeiner, Genetic regulation of neuronal progranulin reveals a critical role for the autophagy-lysosome pathway, J. Neurosci. 39 (2019) 3332–3344.

[56] M.C. Chang, K. Srinivasan, B.A. Friedman, E. Suto, Z. Modrusan, W.P. Lee, et al., Progranulin deficiency causes impairment of autophagy and TDP-43 accumulation, J. Exp. Med. 214 (2017) 2611–2628.

[57] J.K. Gotzl, A.V. Colombo, K. Fellerer, A. Reifschneider, G. Werner, S. Tahirovic, et al., Early lysosomal maturation deficits in microglia triggers enhanced lysosomal activity in other brain cells of progranulin knockout mice, Mol. Neurodegener. 13 (2018) 48.

[58] J.K. Gotzl, K. Mori, M. Damme, K. Fellerer, S. Tahirovic, G. Kleinberger, et al., Common pathobiochemical hallmarks of progranulin-associated frontotemporal lobar degeneration and neuronal ceroid lipofuscinosis, Acta Neuropathol. 127 (2014) 845–860.

[59] Y. Tanaka, G. Suzuki, T. Matsuwaki, M. Hosokawa, G. Serrano, T.G. Beach, et al., Progranulin regulates lysosomal function and biogenesis through acidification of lysosomes, Hum. Mol. Genet. 26 (2017) 969–988.

[60] Y. Tanaka, J.K. Chambers, T. Matsuwaki, K. Yamanouchi, M. Nishihara, Possible involvement of lysosomal dysfunction in pathological changes of the brain in aged progranulin-deficient mice, Acta Neuropathol. Commun. 2 (2014) 78.

[61] C.J. Holler, G. Taylor, Q. Deng, T. Kukar, Intracellular proteolysis of progranulin generates stable, lysosomal granulins that are haploinsufficient in patients with frontotemporal dementia caused by GRN mutations, eNeuro 4 (2017).

[62] P. Wild, H. Farhan, D.G. McEwan, S. Wagner, V.V. Rogov, N.R. Brady, et al., Phosphorylation of the autophagy receptor optineurin restricts Salmonella growth, Science 333 (2011) 228–233.

[63] H. Maruyama, H. Morino, H. Ito, Y. Izumi, H. Kato, Y. Watanabe, et al., Mutations of optineurin in amyotrophic lateral sclerosis, Nature 465 (2010) 223–226.

[64] J. Korac, V. Schaeffer, I. Kovacevic, A.M. Clement, B. Jungblut, C. Behl, et al., Ubiquitin-independent function of optineurin in autophagic clearance of protein aggregates, J. Cell Sci. 126 (2013) 580–592.

[65] W.C. Shen, H.Y. Li, G.C. Chen, Y. Chern, P.H. Tu, Mutations in the ubiquitin-binding domain of OPTN/optineurin interfere with autophagy-mediated degradation of misfolded proteins by a dominant-negative mechanism, Autophagy 11 (2015) 685–700.

[66] X. Shen, H. Ying, Y. Qiu, J.S. Park, R. Shyam, Z.L. Chi, et al., Processing of optineurin in neuronal cells, J. Biol. Chem. 286 (2011) 3618–3629.

[67] D.A. Tumbarello, B.J. Waxse, S.D. Arden, N.A. Bright, J. Kendrick-Jones, F. Buss, Autophagy receptors link myosin VI to autophagosomes to mediate Tom1-dependent autophagosome maturation and fusion with the lysosome, Nat. Cell Biol. 14 (2012) 1024–1035.

[68] V. Sundaramoorthy, A.K. Walker, V. Tan, J.A. Fifita, E.P. McCann, K.L. Williams, et al., Defects in optineurin- and myosin VI-mediated cellular trafficking in amyotrophic lateral sclerosis, Hum. Mol. Genet. 24 (2015) 3830–3846.

[69] M. Bansal, S.C. Moharir, S.P. Sailasree, K. Sirohi, C. Sudhakar, D.P. Sarathi, et al., Optineurin promotes autophagosome formation by recruiting the autophagy-related Atg12-5-16L1 complex to phagophores containing the Wipi2 protein, J. Biol. Chem. 293 (2018) 132–147.

[70] G.J. Song, H. Jeon, M. Seo, M. Jo, K. Suk, Interaction between optineurin and Rab1a regulates autophagosome formation in neuroblastoma cells, J. Neurosci. Res. 96 (2018) 407–415.

[71] E.T. Cirulli, B.N. Lasseigne, S. Petrovski, P.C. Sapp, P.A. Dion, C.S. Leblond, et al., Exome sequencing in amyotrophic lateral sclerosis identifies risk genes and pathways, Science 347 (2015) 1436–1441.

[72] A. Freischmidt, T. Wieland, B. Richter, W. Ruf, V. Schaeffer, K. Muller, et al., Haploinsufficiency of TBK1 causes familial ALS and fronto-temporal dementia, Nat. Neurosci. 18 (2015) 631–636.

[73] I. Gijselinck, S. Van Mossevelde, J. van der Zee, A. Sieben, S. Philtjens, B. Heeman, et al., Loss of TBK1 is a frequent cause of frontotemporal dementia in a Belgian cohort, Neurology 85 (2015) 2116–2125.

[74] G. Tohnai, R. Nakamura, J. Sone, M. Nakatochi, D. Yokoi, M. Katsuno, et al., Frequency and characteristics of the TBK1 gene variants in Japanese patients with sporadic amyotrophic lateral sclerosis, Neurobiol. Aging 64 (2018) 158, e15–.e19.

[75] M. Pilli, J. Arko-Mensah, M. Ponpuak, E. Roberts, S. Master, M.A. Mandell, et al., TBK-1 promotes autophagy-mediated antimicrobial defense by controlling autophagosome maturation, Immunity 37 (2012) 223–234.

[76] M. de Majo, S.D. Topp, B.N. Smith, A.L. Nishimura, H.J. Chen, A.S. Gkazi, et al., ALS-associated missense and nonsense TBK1 mutations can both cause loss of kinase function, Neurobiol. Aging 71 (2018) 266, e1–.e10.

[77] S. Kumar, Y. Gu, Y.P. Abudu, J.A. Bruun, A. Jain, F. Farzam, et al., Phosphorylation of syntaxin 17 by TBK1 controls autophagy initiation, Dev. Cell 49 (2019) 130–144.e6.

[78] C. Sellier, M.-L. Campanari, C. Julie Corbier, A. Gaucherot, I. Kolb-Cheynel, M. Oulad-Abdelghani, et al., Loss of C9ORF72 impairs autophagy and synergizes with polyQ Ataxin-2 to induce motor neuron dysfunction and cell death, EMBO J. 35 (2016) 1276–1297.

[79] J.N.S. Vargas, C. Wang, E. Bunker, L. Hao, D. Maric, G. Schiavo, et al., Spatiotemporal control of ULK1 activation by NDP52 and TBK1 during selective autophagy, Mol. Cell 74 (2019) 347–362.e6.

[80] S. Pillai, J. Nguyen, J. Johnson, E. Haura, D. Coppola, S. Chellappan, Tank binding kinase 1 is a centrosome-associated kinase necessary for microtubule dynamics and mitosis, Nat. Commun. 6 (2015) 10072.

[81] I. Joung, J.L. Strominger, J. Shin, Molecular cloning of a phosphotyrosine-independent ligand of the p56lck SH2 domain, Proc. Natl. Acad. Sci. U. S. A. 93 (1996) 5991–5995.

[82] G. Bjørkøy, T. Lamark, A. Brech, H. Outzen, M. Perander, A. Øvervatn, et al., p62/SQSTM1 forms protein aggregates degraded by autophagy and has a protective effect on huntingtin-induced cell death, J. Cell Biol. 171 (2005) 603–614.

[83] M.L. Seibenhener, J.R. Babu, T. Geetha, H.C. Wong, N.R. Krishna, M.W. Wooten, Sequestosome 1/p62 is a polyubiquitin chain binding protein involved in ubiquitin proteasome degradation, Mol. Cell. Biol. 24 (2004) 8055–8068.

[84] E. Itakura, N. Mizushima, p62 targeting to the autophagosome formation site requires self-oligomerization but not LC3 binding, J. Cell Biol. 192 (2011) 17–27.

[85] S. Pankiv, T.H. Clausen, T. Lamark, A. Brech, J.A. Bruun, H. Outzen, et al., p62/SQSTM1 binds directly to Atg8/LC3 to facilitate degradation of ubiquitinated protein aggregates by autophagy*[S], J. Biol. Chem. 282 (2007) 24131–24145.

[86] R.K. Vadlamudi, I. Joung, J.L. Strominger, J. Shin, p62, a phosphotyrosine-independent ligand of the SH2 domain of p56(lck), belongs to a new class of ubiquitin-binding proteins, J. Biol. Chem. 271 (1996) 20235–20237.

[87] M.H. Sahani, E. Itakura, N. Mizushima, Expression of the autophagy substrate SQSTM1/p62 is restored during prolonged starvation depending on transcriptional upregulation and autophagy-derived amino acids, Autophagy 10 (2014) 431–441.

[88] F. Fecto, J. Yan, S.P. Vemula, E. Liu, Y. Yang, W. Chen, et al., SQSTM1 mutations in familial and sporadic amyotrophic lateral sclerosis, Arch. Neurol. 68 (2011) 1440–1446.

[89] R. Elisa, I. Rainiero, A. Chiò, E. Rogaeva, D. Galimberti, P. Fenoglio, et al., SQSTM1 mutations in frontotemporal lobar degeneration and amyotrophic lateral sclerosis, Neurology 79 (2012) 1556–1562.

[90] I. Le Ber, A. Camuzat, R. Guerreiro, K. Bouya-Ahmed, J. Bras, G. Nicolas, et al., SQSTM1 mutations in French patients with frontotemporal dementia or frontotemporal dementia with amyotrophic lateral sclerosis, JAMA Neurol. 70 (2013) 1403–1410.

[91] A. Goode, K. Butler, J. Long, J. Cavey, D. Scott, B. Shaw, et al., Defective recognition of LC3B by mutant SQSTM1/p62 implicates impairment of autophagy as a pathogenic mechanism in ALS-FTLD, Autophagy 12 (2016) 1094–1104.

[92] J. Sanchez-Garrido, V. Sancho-Shimizu, A.R. Shenoy, Regulated proteolysis of p62/SQSTM1 enables differential control of autophagy and nutrient sensing, Sci. Signal. 11 (2018).

[93] E. Rubino, I. Rainiero, A. Chio, E. Rogaeva, D. Galimberti, P. Fenoglio, et al., SQSTM1 mutations in frontotemporal lobar degeneration and amyotrophic lateral sclerosis, Neurology 79 (2012) 1556–1562.

[94] S. Hadano, S. Mitsui, L. Pan, A. Otomo, M. Kubo, K. Sato, et al., Functional links between SQSTM1 and ALS2 in the pathogenesis of ALS: cumulative impact on the protection against mutant SOD1-mediated motor dysfunction in mice, Hum. Mol. Genet. 25 (2016) 3321–3340.

[95] S. Mitsui, A. Otomo, M. Nozaki, S. Ono, K. Sato, R. Shirakawa, et al., Systemic overexpression of SQSTM1/p62 accelerates disease onset in a SOD1(H46R)-expressing ALS mouse model, Mol. Brain 11 (2018) 30.

[96] C. Munch, R. Sedlmeier, T. Meyer, V. Homberg, A.D. Sperfeld, A. Kurt, et al., Point mutations of the p150 subunit of dynactin (DCTN1) gene in ALS, Neurology 63 (2004) 724–726.

[97] I. Puls, C. Jonnakuty, B.H. LaMonte, E.L. Holzbaur, M. Tokito, E. Mann, et al., Mutant dynactin in motor neuron disease, Nat. Genet. 33 (2003) 455–456.

[98] J. Yu, C. Lai, H. Shim, C. Xie, L. Sun, C.X. Long, et al., Genetic ablation of dynactin p150Glued in postnatal neurons causes preferential degeneration of spinal motor neurons in aged mice, Mol. Neurodegener. 13 (2018).

[99] F.M. Laird, M.H. Farah, S. Ackerley, A. Hoke, N. Maragakis, J.D. Rothstein, et al., Motor neuron disease occurring in a mutant dynactin mouse model is characterized by defects in vesicular trafficking, J. Neurosci. 28 (2008) 1997–2005.

[100] J.R. Levy, C.J. Sumner, J.P. Caviston, M.K. Tokito, S. Ranganathan, L.A. Ligon, et al., A motor neuron disease-associated mutation in p150Glued perturbs dyneactin function and induces protein aggregation, J. Cell Biol. 172 (2006) 733–745.

[101] K.I. Ishikawa, S. Saiki, N. Furuya, Y. Imamichi, Y. Tsuboi, N. Hattori, p150(glued) deficiency impairs effective fusion between autophagosomes and lysosomes due to their redistribution to the cell periphery, Neurosci. Lett. 690 (2019) 181–187.

[102] M.M. Olszewski, C. Williams, K.C. Dong, A. Martin, The Cdc48 unfoldase prepares well-folded protein substrates for degradation by the 26S proteasome, Commun. Biol. 2 (2019).

[103] G.D. Watts, D. Thomasova, S.K. Ramdeen, E.C. Fulchiero, S.G. Mehta, D.A. Drachman, et al., Novel VCP mutations in inclusion body myopathy associated with Paget disease of bone and frontotemporal dementia, Clin. Genet. 72 (2007) 420–426.

[104] J.O. Johnson, J. Mandrioli, M. Benatar, Y. Abramzon, V.M. Van Deerlin, J.Q. Trojanowski, et al., Exome sequencing reveals VCP mutations as a cause of familial ALS, Neuron 68 (2010) 857–864.

[105] J.S. Ju, R.A. Fuentealba, S.E. Miller, E. Jackson, D. Piwnica-Worms, R.H. Baloh, et al., Valosin-containing protein (VCP) is required for autophagy and is disrupted in VCP disease, J. Cell Biol. 187 (2009) 875–888.

[106] E. Tresse, F.A. Salomons, J. Vesa, L.C. Bott, V. Kimonis, T.P. Yao, et al., VCP/p97 is essential for maturation of ubiquitin-containing autophagosomes and this function is impaired by mutations that cause IBMPFD, Autophagy 6 (2010) 217–227.

[107] C. Papadopoulos, P. Kirchner, M. Bug, D. Grum, L. Koever, N. Schulze, et al., VCP/p97 cooperates with YOD1, UBXD1 and PLAA to drive clearance of ruptured lysosomes by autophagy, EMBO J. 36 (2017) 135–150.

[108] K. Arhzaouy, C. Papadopoulos, N. Schulze, S.K. Pittman, H. Meyer, C.C. Weihl, VCP maintains lysosomal homeostasis and TFEB activity in differentiated skeletal muscle, Autophagy 15 (2019) 1082–1099.

[109] A.E. Johnson, H. Shu, A.G. Hauswirth, A. Tong, G.W. Davis, VCP-dependent muscle degeneration is linked to defects in a dynamic tubular lysosomal network in vivo, eLife 4 (2015) e07366.

[110] J.R. Buchan, R.M. Kolaitis, J.P. Taylor, R. Parker, Eukaryotic stress granules are cleared by granulophagy and Cdc48/VCP function, Cell 153 (2013) 1461–1474.

[111] S.J. Seguin, F.F. Morelli, J. Vinet, D. Amore, S. De Biasi, A. Poletti, et al., Inhibition of autophagy, lysosome and VCP function impairs stress granule assembly, Cell Death Differ. 21 (2014) 1838–1851.

[112] Y. Chen, Z.Z. Zheng, R. Huang, K. Chen, W. Song, B. Zhao, et al., PFN1 mutations are rare in Han Chinese populations with amyotrophic lateral sclerosis, Neurobiol. Aging 34 (2013) 1922, e1–5.

[113] C. Ingre, J.E. Landers, N. Rizik, A.E. Volk, C. Akimoto, A. Birve, et al., A novel phosphorylation site mutation in profilin 1 revealed in a large screen of US, Nordic, and German amyotrophic lateral sclerosis/frontotemporal dementia cohorts, Neurobiol. Aging 34 (2013) 1708, e1–6.

[114] B.N. Smith, C. Vance, E.L. Scotter, C. Troakes, C.H. Wong, S. Topp, et al., Novel mutations support a role for profilin 1 in the pathogenesis of ALS, Neurobiol. Aging 36 (2015) 1602, e17–27.

[115] C.H. Wu, A. Giampetruzzi, H. Tran, C. Fallini, F.B. Gao, J.E. Landers, A Drosophila model of ALS reveals a partial loss of function of causative human PFN1 mutants, Hum. Mol. Genet. 26 (2017) 2146–2155.

[116] C. Yang, E.W. Danielson, T. Qiao, J. Metterville, R.H. Brown Jr., J.E. Landers, et al., Mutant PFN1 causes ALS phenotypes and progressive motor neuron degeneration in mice by a gain of toxicity, Proc. Natl. Acad. Sci. U. S. A. 113 (2016) E6209–E6218.

[117] Y. Tanaka, T. Nonaka, G. Suzuki, F. Kametani, M. Hasegawa, Gain-of-function profilin 1 mutations linked to familial amyotrophic lateral sclerosis cause seed-dependent intracellular TDP-43 aggregation, Hum. Mol. Genet. 25 (2016) 1420–1433.

[118] A.L. Nishimura, M. Mitne-Neto, H.C. Silva, A. Richieri-Costa, S. Middleton, D. Cascio, et al., A mutation in the vesicle-trafficking protein VAPB causes late-onset spinal muscular atrophy and amyotrophic lateral sclerosis, Am. J. Hum. Genet. 75 (2004) 822–831.

[119] M. Hamasaki, N. Furuta, A. Matsuda, A. Nezu, A. Yamamoto, N. Fujita, et al., Autophagosomes form at ER-mitochondria contact sites, Nature 495 (2013) 389–393.

[120] P. Gomez-Suaga, S. Paillusson, R. Stoica, W. Noble, D.P. Hanger, C.C.J. Miller, The ER-mitochondria tethering complex VAPB-PTPIP51 regulates autophagy, Curr. Biol. 27 (2017) 371–385.

[121] F. Larroquette, L. Seto, P.L. Gaub, B. Kamal, D. Wallis, R. Larivière, et al., Vapb/Amyotrophic lateral sclerosis 8 knock-in mice display slowly progressive motor behavior defects accompanying ER stress and autophagic response, Hum. Mol. Genet. 24 (2015) 6515–6529.

[122] D. Hantan, Y. Yamamoto, T. Sakisaka, VAP-B binds to Rab3GAP1 at the ER: its implication in nuclear envelope formation through the ER-Golgi intermediate compartment, Kobe J. Med. Sci. 60 (2014) E48–E56.

[123] N. Spang, A. Feldmann, H. Huesmann, F. Bekbulat, V. Schmitt, C. Hiebel, et al., RAB3GAP1 and RAB3GAP2 modulate basal and rapamycin-induced autophagy, Autophagy 10 (2014) 2297–2309.

[124] Y.G. Zhao, N. Liu, G. Miao, Y. Chen, H. Zhao, H. Zhang, The ER contact proteins VAPA/B interact with multiple autophagy proteins to modulate autophagosome biogenesis, Curr. Biol. 28 (2018) 1234–1245, e4.

[125] D. Mao, G. Lin, B. Tepe, Z. Zuo, K.L. Tan, M. Senturk, et al., VAMP associated proteins are required for autophagic and lysosomal degradation by promoting a PtdIns4P-mediated endosomal pathway, Autophagy 15 (2019) 1214–1233.

[126] H.-X. Deng, W. Chen, S.-T. Hong, K.M. Boycott, G.H. Gorrie, N. Siddique, et al., Mutations in UBQLN2 cause dominant X-linked juvenile and adult-onset ALS and ALS/dementia, Nature 477 (2011) 211–215.

[127] H. Daoud, H. Suhail, A. Szuto, W. Camu, F. Salachas, V. Meininge, et al., UBQLN2 mutations are rare in French and French-Canadian amyotrophic lateral sclerosis, Neurobiol. Aging 33 (2012) 2230, e1–.e5.

[128] L. Dillen, T. Van Langenhove, S. Engelborghs, M. Vandenbulcke, S. Sarafof, I. Tournev, et al., Explorative genetic study of UBQLN2 and PFN1 in an extended Flanders-Belgian cohort of frontotemporal lobar degeneration patients, Neurobiol. Aging 34 (2013) 1711.e1–1711.e5.

[129] C. Rothenberg, D. Srinivasan, L. Mah, S. Kaushik, C.M. Peterhoff, J. Ugolino, et al., Ubiquilin functions in autophagy and is degraded by chaperone-mediated autophagy, Hum. Mol. Genet. 19 (2010) 3219–3232.

[130] E.-N. N'Diaye, K.K. Kajihara, I. Hsieh, H. Morisaki, J. Debnath, E.J. Brown, PLIC proteins or ubiquilins regulate autophagy-dependent cell survival during nutrient starvation, EMBO Rep. 10 (2009) 173–179.

[131] M. Sentürk, G. Lin, Z. Zuo, D. Mao, E. Watson, A.G. Mikos, et al., Ubiquilins regulate autophagic flux through mTOR

[132] G. Skibinski, N.J. Parkinson, J.M. Brown, L. Chakrabarti, S.L. Lloyd, H. Hummerich, et al., Mutations in the endosomal ESCRTIII-complex subunit CHMP2B in frontotemporal dementia, Nat. Genet. 37 (2005) 806–808.

[133] L.E. Cox, L. Ferraiuolo, E.F. Goodall, P.R. Heath, A. Higginbottom, H. Mortiboys, et al., Mutations in CHMP2B in lower motor neuron predominant amyotrophic lateral sclerosis (ALS), PLoS One 5 (2010) e9872.

[134] H. Urwin, A. Authier, J.E. Nielsen, D. Metcalf, C. Powell, K. Froud, et al., Disruption of endocytic trafficking in frontotemporal dementia with CHMP2B mutations, Hum. Mol. Genet. 19 (2010) 2228–2238.

[135] J.A. Lee, A. Beigneux, S.T. Ahmad, S.G. Young, F.B. Gao, ESCRT-III dysfunction causes autophagosome accumulation and neurodegeneration, Curr. Biol. 17 (2007) 1561–1567.

[136] S. Ghazi-Noori, K.E. Froud, S. Mizielinska, C. Powell, M. Smidak, M. Fernandez de Marco, et al., Progressive neuronal inclusion formation and axonal degeneration in CHMP2B mutant transgenic mice, Brain 135 (2012) 819–832.

[137] A. Vernay, L. Therreau, B. Blot, V. Risson, S. Dirrig-Grosch, R. Waegaert, et al., A transgenic mouse expressing CHMP2Bintron5 mutant in neurons develops histological and behavioural features of amyotrophic lateral sclerosis and frontotemporal dementia, Hum. Mol. Genet. 25 (2016) 3341–3360.

[138] E.L. Clayton, S. Mizielinska, J.R. Edgar, T.T. Nielsen, S. Marshall, F.E. Norona, et al., Frontotemporal dementia caused by CHMP2B mutation is characterised by neuronal lysosomal storage pathology, Acta Neuropathol. 130 (2015) 511–523.

[139] Y. Lu, Z. Zhang, D. Sun, S.T. Sweeney, F.B. Gao, Syntaxin 13, a genetic modifier of mutant CHMP2B in frontotemporal dementia, is required for autophagosome maturation, Mol. Cell 52 (2013).

[140] Y. Takahashi, H. He, Z. Tang, T. Hattori, Y. Liu, M.M. Young, et al., An autophagy assay reveals the ESCRT-III component CHMP2A as a regulator of phagophore closure, Nat. Commun. 9 (2018).

[141] A. Otomo, S. Hadano, T. Okada, H. Mizumura, R. Kunita, H. Nishijima, et al., ALS2, a novel guanine nucleotide exchange factor for the small GTPase Rab5, is implicated in endosomal dynamics, Hum. Mol. Genet. 12 (2003) 1671–1687.

[142] S. Hadano, C.K. Hand, H. Osuga, Y. Yanagisawa, A. Otomo, R.S. Devon, et al., A gene encoding a putative GTPase regulator is mutated in familial amyotrophic lateral sclerosis 2, Nat. Genet. 29 (2001) 166–173.

[143] Y. Yang, A. Hentati, H.X. Deng, O. Dabbagh, T. Sasaki, M. Hirano, et al., The gene encoding alsin, a protein with three guanine-nucleotide exchange factor domains, is mutated in a form of recessive amyotrophic lateral sclerosis, Nat. Genet. 29 (2001) 160–165.

[144] M. Gautam, J.H. Jara, G. Sekerkova, M.V. Yasvoina, M. Martina, P.H. Ozdinler, Absence of alsin function leads to corticospinal motor neuron vulnerability via novel disease mechanisms, Hum. Mol. Genet. 25 (2016) 1074–1087.

[145] J. van Eersel, Y.D. Ke, A. Gladbach, M. Bi, J. Götz, J.J. Kril, et al., Cytoplasmic accumulation and aggregation of TDP-

43 upon proteasome inhibition in cultured neurons, PLoS One 6 (2011).

[146] Y. Tashiro, M. Urushitani, H. Inoue, M. Koike, Y. Uchiyama, M. Komatsu, et al., Motor neuron-specific disruption of proteasomes, but not autophagy, replicates amyotrophic lateral sclerosis, J. Biol. Chem. 287 (2012) 42984–42994.

[147] E.L. Scotter, C. Vance, A.L. Nishimura, Y.B. Lee, H.J. Chen, H. Urwin, et al., Differential roles of the ubiquitin proteasome system and autophagy in the clearance of soluble and aggregated TDP-43 species, J. Cell Sci. 127 (2014) 1263–1278.

[148] R. Cascella, G. Fani, C. Capitini, P. Rusmini, A. Poletti, C. Cecchi, et al., Quantitative assessment of the degradation of aggregated TDP-43 mediated by the ubiquitin proteasome system and macroautophagy, FASEB J. 31 (2017) 5609–5624.

[149] Y. Mizuno, M. Amari, M. Takatama, H. Aizawa, B. Mihara, K. Okamoto, Immunoreactivities of p62, an ubiquitin-binding protein, in the spinal anterior horn cells of patients with amyotrophic lateral sclerosis, J. Neurol. Sci. 249 (2006) 13–18.

[150] M. Hiji Masanori, T. Takahashi, H. Fukuba, H. Yamashita, T. Kohriyama, M. Matsumoto, White matter lesions in the brain with frontotemporal lobar degeneration with motor neuron disease: TDP-43-immunopositive inclusions colocalize with p62, but not ubiquitin, Acta Neuropathol. 116 (2008) 183–191.

[151] A. Caccamo, S. Majumder, J.J. Deng, Y. Bai, F.B. Thornton, S. Oddo, Rapamycin rescues TDP-43 mislocalization and the associated low molecular mass neurofilament instability, J. Biol. Chem. 284 (2009) 27416–27424.

[152] I.F. Wang, B.S. Guo, Y.C. Liu, C.C. Wu, C.H. Yang, K.J. Tsai, et al., Autophagy activators rescue and alleviate pathogenesis of a mouse model with proteinopathies of the TAR DNA-binding protein 43, Proc. Natl. Acad. Sci. U. S. A. 109 (2012) 15024–15029.

[153] C.W. Cheng, M.J. Lin, C.K.J. Shen, Rapamycin alleviates pathogenesis of a new Drosophila model of ALS-TDP, J. Neurogenet. 29 (2015) 59–68.

[154] S.J. Barmada, A. Serio, A. Arjun, B. Bilican, A. Daub, D.M. Ando, et al., Autophagy induction enhances TDP43 turnover and survival in neuronal ALS models, Nat. Chem. Biol. 10 (2014) 677–685.

[155] A. Caccamo, D.M. Shaw, F. Guarino, A. Messina, A.W. Walker, S. Oddo, Reduced protein turnover mediates functional deficits in transgenic mice expressing the 25 kDa C-terminal fragment of TDP-43, Hum. Mol. Genet. 24 (2015) 4625–4635.

[156] J.K. Bose, C.C. Huang, C.K.J. Shen, Regulation of autophagy by neuropathological protein TDP-43, J. Biol. Chem. 286 (2011) 44441–44448.

[157] A. Donde, M. Sun, Y.H. Jeong, X. Wen, J. Ling, S. Lin, et al., Upregulation of ATG7 attenuates motor neuron dysfunction associated with depletion of TARDBP/TDP-43, Autophagy 16 (4) (2019) 672–682.

[158] C. Settembre, C. Di Malta, V.A. Polito, M. Garcia Arencibia, F. Vetrini, S. Erdin, et al., TFEB links autophagy to lysosomal biogenesis, Science 332 (2011) 1429–1433.

[159] Q. Xia, H. Wang, Z. Hao, C. Fu, Q. Hu, F. Gao, et al., TDP-43 loss of function increases TFEB activity and blocks autophagosome-lysosome fusion, EMBO J. 35 (2016) 121–142.

[160] T. Kabuta, Y. Suzuki, K. Wada, Degradation of amyotrophic lateral sclerosis-linked mutant Cu,Zn-superoxide dismutase proteins by macroautophagy and the proteasome, J. Biol. Chem. 281 (2006) 30524–30533.

[161] E. Tokuda, T. Brännström, P.M. Andersen, S.L. Marklund, Low autophagy capacity implicated in motor system vulnerability to mutant superoxide dismutase, Acta Neuropathol. Commun. 4 (2016) 6.

[162] U. Bandyopadhyay, M. Nagy, W.A. Fenton, A.L. Horwich, Absence of lipofuscin in motor neurons of SOD1-linked ALS mice, Proc. Natl. Acad. Sci. U. S. A. 111 (2014) 11055–11060.

[163] N. Morimoto, M. Nagai, Y. Ohta, K. Miyazaki, T. Kurata, M. Morimoto, et al., Increased autophagy in transgenic mice with a G93A mutant SOD1 gene, Brain Res. 1167 (2007) 112–117.

[164] L. Li, X. Zhang, W. Le, Altered macroautophagy in the spinal cord of SOD1 mutant mice, Autophagy 4 (2008) 290–293.

[165] O. Baron, A. Boudi, C. Dias, M. Schilling, A. Nolle, G. Vizcay-Barrena, et al., Stall in canonical autophagy-lysosome pathways prompts nucleophagy-based nuclear breakdown in neurodegeneration, Curr. Biol. 27 (2017) 3626–36242 e6.

[166] L.G. Bilsland, E. Sahai, G. Kelly, M. Golding, L. Greensmith, G. Schiavo, Deficits in axonal transport precede ALS symptoms in vivo, Proc. Natl. Acad. Sci. U. S. A. 107 (2010) 20523–20528.

[167] F. Zhang, A.L. Strom, K. Fukada, S. Lee, L.J. Hayward, H. Zhu, Interaction between familial amyotrophic lateral sclerosis (ALS)-linked SOD1 mutants and the dynein complex, J. Biol. Chem. 282 (2007) 16691–16699.

[168] Y. Xie, B. Zhou, M.Y. Lin, S. Wang, K.D. Foust, Z.H. Sheng, Endolysosomal deficits augment mitochondria pathology in spinal motor neurons of asymptomatic fALS mice, Neuron 87 (2015) 355–370.

[169] F. Fornai, P. Longone, L. Cafaro, O. Kastsiuchenka, M. Ferrucci, M.L. Manca, et al., Lithium delays progression of amyotrophic lateral sclerosis, Proc. Natl. Acad. Sci. U. S. A. 105 (2008) 2052–2057.

[170] X. Zhang, S. Chen, L. Song, Y. Tang, Y. Shen, L. Jia, et al., MTOR-independent, autophagic enhancer trehalose prolongs motor neuron survival and ameliorates the autophagic flux defect in a mouse model of amyotrophic lateral sclerosis, Autophagy 10 (2014) 588–602.

[171] R. Mancuso, J. del Valle, L. Modol, A. Martinez, A.B. Granado-Serrano, O. Ramirez-Núñez, et al., Resveratrol improves motoneuron function and extends survival in SOD1G93A ALS mice, Neurotherapeutics 11 (2014) 419–432.

[172] J. Kim, T.Y. Kim, K.S. Cho, H.N. Kim, J.Y. Koh, Autophagy activation and neuroprotection by progesterone in the G93A-SOD1 transgenic mouse model of amyotrophic lateral sclerosis, Neurobiol. Dis. 59 (2013) 80–85.

[173] J. Sun, Y. Mu, Y. Jiang, R. Song, J. Yi, J. Zhou, et al., Inhibition of p70 S6 kinase activity by A77 1726 induces autophagy and enhances the degradation of superoxide dismutase 1 (SOD1) protein aggregates, Cell Death Dis. 9 (2018).

[174] Y.M. Wei, B. Han, Beclin1 decreases the RIPA-insoluble fraction of amyotrophic lateral sclerosis-linked SOD1 mutant via autophagy, Neurosci. Lett. 690 (2019) 106–111.

[175] F. Massenzio, E. Peña-Altamira, S. Petrella, M. Virgili, G. Zuccheri, A. Miti, et al., Microglial overexpression of

fALS-linked mutant SOD1 induces SOD1 processing impairment, activation and neurotoxicity and is counteracted by the autophagy inducer trehalose, Biochim. Biophys. Acta Mol. Basis Dis. 1864 (2018) 3771–3785.

[176] X. Zhang, L. Li, S. Chen, D. Yang, Y. Wang, X. Zhang, et al., Rapamycin treatment augments motor neuron degeneration in SOD1 G93A mouse model of amyotrophic lateral sclerosis, Autophagy 7 (2011) 412–425.

[177] K.W. Hsueh, T.W. Chiou, S.F. Chiang, T. Yamashita, K. Abe, C.V. Borlongan, et al., Autophagic down-regulation in motor neurons remarkably prolongs the survival of ALS mice, Neuropharmacology 108 (2016) 152–160.

[178] Q.M. Zhou, J.J. Zhang, S. Li, S. Chen, W.D. Le, n-butylenephthalide treatment prolongs life span and attenuates motor neuron loss in SOD1G93A mouse model of amyotrophic lateral sclerosis, CNS Neurosci. Ther. 23 (2017) 375–385.

[179] C. Pizzasegola, I. Caron, C. Daleno, A. Ronchi, C. Minoia, M.T. Carri, et al., Treatment with lithium carbonate does not improve disease progression in two different strains of SOD1 mutant mice, Amyotroph Lateral Scler. 10 (2009) 221–228.

[180] M. Nassif, V. Valenzuela, D. Rojas-Rivera, R. Vidal, S. Matus, K. Castillo, et al., Pathogenic role of BECN1/ Beclin 1 in the development of amyotrophic lateral sclerosis, Autophagy 10 (2014) 1256–1271.

[181] N.D. Perera, R.K. Sheean, C.L. Lau, Y.S. Shin, P.M. Beart, M.K. Horne, et al., Rilmenidine promotes MTOR-independent autophagy in the mutant SOD1 mouse model of amyotrophic lateral sclerosis without slowing disease progression, Autophagy 14 (2018) 534–551.

[182] N.D. Rudnick, C.J. Griffey, P. Guarnieri, V. Gerbino, X. Wang, J.A. Piersaint, et al., Distinct roles for motor neuron autophagy early and late in the SOD1G93A mouse model of ALS, Proc. Natl. Acad. Sci. U. S. A. 114 (2017) E8294–E8303.

[183] K. Zhang, P. Shi, T. An, Q. Wang, J. Wang, Z. Li, et al., Food restriction-induced autophagy modulates degradation of mutant SOD1 in an amyotrophic lateral sclerosis mouse model, Brain Res. 1519 (2013) 112–119.

[184] H.X. Deng, H. Zhai, E.H. Bigio, J. Yan, F. Fecto, K. Ajroud, et al., FUS-immunoreactive inclusions are a common feature in sporadic and non-SOD1 familial amyotrophic lateral sclerosis, Ann. Neurol. 67 (2010) 739–748.

[185] S.C. Ling, S.G. Dastidar, S. Tokunaga, W.Y. Ho, K. Lim, H. Ilieva, et al., Overriding FUS autoregulation in mice triggers gain-of-toxic dysfunctions in RNA metabolism and autophagy-lysosome axis, eLife 8 (2019).

[186] L. Marrone, H.C.A. Drexler, J. Wang, P. Tripathi, T. Distler, P. Heisterkamp, et al., FUS pathology in ALS is linked to alterations in multiple ALS-associated proteins and rescued by drugs stimulating autophagy, Acta Neuropathol. 138 (2019) 67–84.

[187] L. Marrone, I. Poser, I. Casci, J. Japtok, P. Reinhardt, A. Janosch, et al., Isogenic FUS-eGFP iPSC reporter lines enable quantification of FUS stress granule pathology that is rescued by drugs inducing autophagy, Stem Cell Reports 10 (2018) 375–389.

[188] H.H. Ryu, M.H. Jun, K.J. Min, D.J. Jang, Y.S. Lee, H.K. Kim, et al., Autophagy regulates amyotrophic lateral sclerosis-linked fused in sarcoma-positive stress granules in neurons, Neurobiol. Aging 35 (2014) 2822–2831.


[189] A. Sharma, A.K. Lyashchenko, L. Lu, S.E. Nasrabadyy, M. Elmaleh, M. Mendelsohn, et al., ALS-associated mutant FUS induces selective motor neuron degeneration through toxic gain of function, Nat. Commun. 7 (2016) 10465.

[190] F.C. Zoppino, R.D. Militello, I. Slavin, C. Alvarez, M.I. Colombo, Autophagosome formation depends on the small GTPase Rab1 and functional ER exit sites, Traffic (Copenhagen, Denmark) 11 (2010) 1246–1261.

[191] K.Y. Soo, M. Halloran, V. Sundaramoorthy, S. Parakh, R.P. Toth, K.A. Southam, et al., Rab1-dependent ER-Golgi transport dysfunction is a common pathogenic mechanism in SOD1, TDP-43 and FUS-associated ALS, Acta Neuropathol. 130 (2015) 679–697.

[192] K.Y. Soo, J. Sultana, A.E. King, R.A.K. Atkinson, S.T. Warrach, V. Sundaramoorthy, et al., ALS-associated mutant FUS inhibits macroautophagy which is restored by overexpression of Rab1, Cell Death Discov. 1 (2015) 15030.

[193] A.C. Elden, H.-J. Kim, M.P. Hart, A.S. Chen-Plotkin, B.S. Johnson, X. Fang, et al., Ataxin-2 intermediate-length polyglutamine expansions are associated with increased risk for ALS, Nature 466 (2010) 1069–1075.

[194] S. Paul, W. Dansithong, K.P. Figueroa, D.R. Scoles, S.M. Pulst, Staufen1 links RNA stress granules and autophagy in a model of neurodegeneration, Nat. Commun. 9 (2018) 3648.

[195] D. Brenner, K. Muller, R. Gastl, M. Gorges, M. Otto, E.H. Pinkhardt, et al., Analysis of CACNA1A CAG repeat lengths in patients with familial ALS, Neurobiol. Aging 74 (2019) 235, e5–e8.

[196] X. Tian, U. Gala, Y. Zhang, W. Shang, S. Nagarkar Jaiswal, A. di Ronza, et al., A voltage-gated calcium channel regulates lysosomal fusion with endosomes and autophagosomes and is required for neuronal homeostasis, PLoS Biol. 13 (2015) e1002103-e.

[197] S. Byrne, L. Jansen, J.M. U-King-Im, A. Siddiqui, H.G. Lidov, I. Bodi, et al., EPG5-related Vici syndrome: a paradigm of neurodevelopmental disorders with defective autophagy, Brain 139 (2016) 765–781.

[198] Z. Wang, G. Miao, X. Xue, X. Guo, C. Yuan, Z. Wang, et al., The Vici syndrome protein EPG5 is a Rab7 effector that determines the fusion specificity of autophagosomes with late endosomes/lysosomes, Mol. Cell 63 (2016) 781–795.

[199] N.J. Kramer, M.S. Haney, D.W. Morgens, A. Jovicic, J. Couthouis, A. Li, et al., CRISPR-Cas9 screens in human cells and primary neurons identify modifiers of C9ORF72 dipeptide-repeat-protein toxicity, Nat. Genet. 50 (2018) 603–612.

[200] H. Zhao, Y.G. Zhao, X. Wang, L. Xu, L. Miao, D. Feng, et al., Mice deficient in Epg5 exhibit selective neuronal vulnerability to degeneration, J. Cell Biol. 200 (2013) 731–741.

[201] D. Brenner, R. Yilmaz, K. Muller, T. Grehl, S. Petri, T. Meyer, et al., Hot-spot KIF5A mutations cause familial ALS, Brain 141 (2018) 688–697.

[202] A. Nicolas, K.P. Kenna, A.E. Renton, N. Ticozzi, F. Faghri, R. Chia, et al., Genome-wide analyses identify KIF5A as a novel ALS gene, Neuron 97 (2018) 1268–1283.e6.

[203] G.G. Farias, C.M. Guardia, R. De Pace, D.J. Britt, J.S. Bonifacino, BORC/kinesin-1 ensemble drives polarized transport of lysosomes into the axon, Proc. Natl. Acad. Sci. U. S. A. 114 (2017) E2955–E2964.

[204] H. Daoud, S. Zhou, A. Noreau, M. Sabbagh, V. Belzil, A. Dionne-Laporte, et al., Exome sequencing reveals

SPG11 mutations causing juvenile ALS, Neurobiol. Aging 33 (2012) 839, e5–9.

[205] J. Chang, S. Lee, C. Blackstone, Spastic paraplegia proteins spastizin and spatacsin mediate autophagic lysosome reformation, J. Clin. Investig. 124 (2014) 5249–5262.

[206] C.Y. Chow, J.E. Landers, S.K. Bergren, P.C. Sapp, A.E. Grant, J.M. Jones, et al., Deleterious variants of FIG4, a phosphoinositide phosphatase, in patients with ALS, Am. J. Hum. Genet. 84 (2009) 85–88.

[207] A. Osmanovic, I. Ragnau, A. Kosfeld, S. Abdulla, C. Janssen, B. Auber, et al., FIG4 variants in central European patients with amyotrophic lateral sclerosis: a whole-exome and targeted sequencing study, Eur. J. Hum. Genet. 25 (2017) 324–331.

[208] C.J. Ferguson, G.M. Lenk, M.H. Meisler, Defective autophagy in neurons and astrocytes from mice deficient in PI(3,5)P2, Hum. Mol. Genet. 18 (2009) 4868–4878.

[209] T. Kon, F. Mori, K. Tanji, Y. Miki, Y. Toyoshima, M. Yoshida, et al., ALS-associated protein FIG4 is localized in Pick and Lewy bodies, and also neuronal nuclear inclusions, in polyglutamine and intranuclear inclusion body diseases, Neuropathology 34 (2014) 19–26.

[210] A. Ramirez, A. Heimbach, J. Grundemann, B. Stiller, D. Hampshire, L.P. Cid, et al., Hereditary parkinsonism with dementia is caused by mutations in ATP13A2, encoding a lysosomal type 5 P-type ATPase, Nat. Genet. 38 (2006) 1184–1191.

[211] R. Spataro, M. Kousi, S.M.K. Farhan, J.R. Willer, J.P. Ross, P.A. Dion, et al., Mutations in ATP13A2 (PARK9) are associated with an amyotrophic lateral sclerosis-like phenotype, implicating this locus in further phenotypic expansion, Hum. Genomics 13 (1) (2019) 19.

[212] D. Ramonet, A. Podhajska, K. Stafa, S. Sonnay, A. Trancikova, E. Tsika, et al., PARK9-associated

ATP13A2 localizes to intracellular acidic vesicles and regulates cation homeostasis and neuronal integrity, Hum. Mol. Genet. 21 (2012) 1725–1743.

[213] R. Wang, J. Tan, T. Chen, H. Han, R. Tian, Y. Tan, et al., ATP13A2 facilitates HDAC6 recruitment to lysosome to promote autophagosome–lysosome fusion, J. Cell Biol. 218 (2019) 267–284.

[214] M. Usenovic, E. Tresse, J.R. Mazzulli, J.P. Taylor, D. Krainc, Deficiency of ATP13A2 leads to lysosomal dysfunction, alpha-synuclein accumulation, and neurotoxicity, J. Neurosci. 32 (2012) 4240–4246.

[215] J. Tan, T. Zhang, L. Jiang, J. Chi, D. Hu, Q. Pan, et al., Regulation of intracellular manganese homeostasis by Kufor-Rakeb syndrome-associated ATP13A2 protein, J. Biol. Chem. 286 (2011) 29654–29662.

[216] M. Chitiprolu, C. Jagow, V. Tremblay, E. Bondy-Chorney, G. Paris, A. Savard, et al., A complex of C9ORF72 and p62 uses arginine methylation to eliminate stress granules by autophagy, Nat. Commun. 9 (2018) 2794.

[217] S. Markmiller, S. Soltanieh, K.L. Server, R. Mak, W. Jin, M.Y. Fang, et al., Context-dependent and disease-specific diversity in protein interactions within stress granules, Cell 172 (2018) 590–604 e13.

[218] J.R. Buchan, R.M. Kolaitis, J.P. Taylor, R. Parker, Eukaryotic stress granules are cleared by autophagy and Cdc48/ VCP function, Cell 153 (2013) 1461–1474.

[219] B.N. Smith, S.D. Topp, C. Fallini, H. Shibata, H.J. Chen, C. Troakes, et al., Mutations in the vesicular trafficking protein annexin A11 are associated with amyotrophic lateral sclerosis, Sci. Transl. Med. 9 (2017).

[220] Y.C. Liao, M.S. Fernandopulle, G. Wang, H. Choi, L. Hao, C.M. Drerup, et al., RNA granules Hitchhike on lysosomes for long-distance transport, using annexin A11 as a molecular tether, Cell 179 (2019) 147–164 e20.
