
Since January 2020 Elsevier has created a COVID-19 resource centre with free information in English and Mandarin on the novel coronavirus COVID-19. The COVID-19 resource centre is hosted on Elsevier Connect, the company's public news and information website.

Elsevier hereby grants permission to make all its COVID-19-related research that is available on the COVID-19 resource centre - including this research content - immediately available in PubMed Central and other publicly funded repositories, such as the WHO COVID database with rights for unrestricted research re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for free by Elsevier for as long as the COVID-19 resource centre remains active.

Clinical Immunology 215 (2020) 108448

Contents lists available at ScienceDirect Clinical Immunology journal homepage: www.elsevier.com/locate/yclim

Review Article

COVID-19: Immunology and treatment options

Susanna Felsenstein ${ }^{a}$, Jenny A. Herbert ${ }^{b}$, Paul S. McNamara ${ }^{b}$, Christian M. Hedrich ${ }^{b,c,*}$

${ }^{a}$ Department of Infectious Diseases and Immunology, Alder Hey Children's NHS Foundation Trust Hospital, Liverpool, UK  
${ }^{b}$ Department of Women's & Children's Health, Institute of Translational Medicine, University of Liverpool, Liverpool, UK  
${ }^{c}$ Department of Paediatric Rheumatology, Alder Hey Children's NHS Foundation Trust Hospital, Liverpool, UK  

ABSTRACT

The novel coronavirus SARS-CoV2 causes COVID-19, a pandemic threatening millions. As protective immunity does not exist in humans and the virus is capable of escaping innate immune responses, it can proliferate, unhindered, in primarily infected tissues. Subsequent cell death results in the release of virus particles and intracellular components to the extracellular space, which result in immune cell recruitment, the generation of immune complexes and associated damage. Infection of monocytes/macrophages and/or recruitment of uninfected immune cells can result in massive inflammatory responses later in the disease. Uncontrolled production of pro-inflammatory mediators contributes to ARDS and cytokine storm syndrome. Antiviral agents and immune modulating treatments are currently being trialled. Understanding immune evasion strategies of SARS-CoV2 and the resulting delayed massive immune response will result in the identification of biomarkers that predict outcomes as well as phenotype and disease stage specific treatments that will likely include both antiviral and immune modulating agents.

1. Introduction

Until the SARS outbreak (2002), during which coronaviruses (CoV) showcased their potential for epidemic spread and significant pathogenicity in humans, they were mainly known as causes of mild respiratory and gastrointestinal disease [1]. Over the last two decades, three novel Betacoronaviruses, Severe Acute Respiratory Syndrome (SARS)-CoV, Middle East Respiratory Syndrome (MERS)-CoV and SARS-CoV2, have crossed the species barrier and caused significant outbreaks characterized by high case-fatality rates in humans [2–4] . The latest addition to human pathogenic coronaviruses (hCoVs) is SARS-CoV2, the cause of COVID-19. At the time of submission of this review SARS-CoV2 has infected over 2.6 million people worldwide and claimed 185.000 lives, threatening many more ([https://gisanddata.maps.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6](https://gisanddata.maps.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6)). In the following, epidemiological and clinical features of COVID-19, pathophysiological mechanisms, and already available and future therapeutic options will be discussed based on limited evidence available, and extrapolation from related viral disease.

2. Epidemiology & clinical presentation

The first hCoVs were described in 1966, E229-CoV and OC43-CoV [5,6]. They are part of a group of currently four known seasonal hCoVs (shCoV) that also includes HKU1-CoV and NL63-CoV, which were only discovered in 2005 [7,8]. All shCoVs are globally endemic and frequently cause common colds, accounting for 2-18% of all respiratory tract infections [9–13]. By their fourth birthday, 75% of children show antibodies directed against at least one of the shCoVs [14,15]. Anti-shCoVs antibodies provide some cross-immunity and antibody-mediated protection against infection by other species within the group [16]. While their overall pathogenic potential is comparatively low, in the immunocompromised, infants, the elderly and those with pre-existing pulmonary disorders, shCoVs can cause severe respiratory or sepsis-like presentations [17–21]. OC43 displays some neurotropism and can cause demyelination and CNS infections in vulnerable patient groups [22,23]. While estimates of their contribution to annual respiratory illness vary, shCoVs remain asymptomatic in approximately 50% of cases [24–26].

This is in stark contrast to the clinical presentation encountered in infections with so-called “novel coronaviruses” SARS-CoV, MERS-CoV and SARS-CoV2, which are associated with morbidity and case-fatality ratios that far exceed the ones in shCoVs.

The SARS pandemic of 2002/3 originated in Foshan, Guangdong province, China and spread to South East Asia, Europe and North America [27]. Containment was declared by the end of 2003, with no re-emergence reported since. Overall, 8096 probable cases caused 774 deaths, resulting in a mortality rate of 9.6% ([https://www.who.int/csr/sars/en/](https://www.who.int/csr/sars/en/)). Mortality strongly correlated with age, approaching 7% for those younger, and 55% for those older than 60 years [28]. Health care workers in contact with SARS patients demonstrated a very low seroconversion rate of 2% in asymptomatic individuals. Less than 5% of all affected were children, and post-containment seroprevalence among

* Corresponding author at: Institute in the Park, Alder Hey Children's NHS Foundation Trust Hospital, East Prescot Road, Liverpool L14 5AB, UK.
E-mail address: christian.hedrich@liverpool.ac.uk (G.M. Hedrich).

[https://doi.org/10.1016/j.clim.2020.108448](https://doi.org/10.1016/j.clim.2020.108448)

Received 24 April 2020  
Available online 27 April 2020  
1521-6616/ © 2020 Elsevier Inc. All rights reserved.

S. Felsenstein, et al.

children considered high-risk for significant exposure was extremely low. This suggests that subclinical SARS among children had not occurred [29–31]. Approximately 20% of SARS patients required intensive care support for acute respiratory distress syndrome (ARDS), half of whom died within the following 28 days [32].

The severe clinical phenotype of SARS was replicated during the emergence of MERS in 2012, which continues to circulate, albeit to a lesser extent [36]. To date, 2494 cases of MERS have occurred worldwide, presenting as severe pneumonia, and resulting in respiratory and multiorgan failure, with a case-fatality-ratio of 35%-45% [37]. Individuals with comorbidities, males, and the immunocompromised are considered at particularly high risk.

In both previous novel coronavirus outbreaks, the severity of the clinical manifestation has puzzled clinicians. Common features included massive inflammatory cell infiltration of the lungs resulting in acute lung injury (ALI) and ARDS, highly elevated inflammatory markers in the serum, evidence of monocyte/macrophage activation, activated coagulation and pro-inflammatory cytokine and chemokine profiles [33–38]. This soon led to the implication of the host response as an important factor in this fulminant disease process [38]. Animal models of SARS suggest that lung inflammation intensifies after viral clearance, peaking as late as 14 days after infection [39], and similar observations were made in human SARS patients. This suggests that clinical deterioration later in the disease course was likely not due to uncontrolled viral replication, but rather uncontrolled immune responses and associated damage [40,41].

Similar descriptions of clinical presentations in COVID-19 are now emerging. Presenting features of cough and fever subacutely progress to respiratory distress and acute respiratory distress syndrome (ARDS) in 8-19% of patients, with the elderly and those with underlying comorbidities especially cardiovascular disease, diabetes mellitus, chronic pulmonary disorders or renal disease especially at risk https://www.epicentro.iss.it/coronavirus/bollettino/Report-COVID-2019_24_marzo_eng.pdf [42–45]. It is estimated that about 14% of COVID-19 patients develop respiratory symptoms requiring supplemental oxygen, and approximately 5% develop a need for mechanical ventilation [44–46] The CDC reports an overall case-fatality rate of 2.3%, though higher at 14.8% in patients over 80 years of age and 49% among the critically ill requiring mechanical ventilation [46].

The pulmonary pathology in COVID-19 is characterized by diffuse alveolar damage, and focal reactive hyperplasia of pneumocytes with patchy inflammatory cellular infiltration and evidence of intravascular thrombosis. Monocytes, macrophages, and lymphocytes infiltrate the pulmonary interstitium [47,48]. The severe pulmonary inflammatory infiltrate of pulmonary tissue impedes alveolar gas exchange. In addition, one fifth of hospitalized patients develop significant cardiovascular morbidity, characterized by troponin rise, tachyarrhythmias and thromboembolic events, which is strongly associated with mortality risk [49–51]. Common features of COVID-19 patients requiring hospitalization and intensive care level support therefore are severe pneumonia with hypoxic respiratory failure of subacute onset evolving into ARDS, with a clinical picture characterized by fevers, lymphopenia, highly elevated C-reactive protein, proinflammatory cytokines, serum ferritin, and D-Dimers. Histopathological evidence of a prominent pulmonary infiltrate dominated by monocytes and macrophages, vasculitis and hypercoagulability is seen [52,53].

Based on current knowledge, clinical pictures, disease pathology and progression of SARS-CoV2 infections are similar causing significant morbidity and mortality that may be associated with hyperinflammatory responses in a subset of patients.

## Viral structure, host range and cell entry mechanisms

Coronaviruses are highly prevalent animal pathogens with a wide host range. Overall, thousands of species of coronaviruses are known [54,55]. Currently, seven CoVs are recognized as human pathogens [1].

The family of Coronaviridae is divided into two subfamilies: Coronavirinae and Torovirinae. Coronavirinae include the genera Alpha-, Betacoronaviruses, infecting only mammals, and Gamma-, and Deltacoronaviruses which infect both mammals and birds. Human CoVs E229 and NL63 are human pathogenic alpha-, while OC43 and HKU1 and all novel CoVs (including SARS-CoV2) are betacoronaviruses. The potential of Toroviruses to cause disease in humans is unknown https://talk.ictvonline.org/ictv-reports/ictv_9th_report/positive-sense-rna-viruses-2011/w/posrna_viruses/222/coronaviridae .

Coronaviruses (CoVs) are large enveloped viruses with a single-stranded, nonsegmented, positive sense RNA genome that spans approximately 30 kilobases, making it the largest known genome of any RNA virus [56]. Being RNA viruses, CoVs readily evolve by mutation and homologous and non-homologous recombination, which expands their host range and facilitates crossing of species barriers. Extensive animal reservoirs, especially among bats, genetic recombination among CoVs, and their plasticity in terms of receptor use renders CoVs highly effective at host switching, sometimes across wide taxonomic distances [57,58].

All hCoVs are thought to be zoonoses. Novel coronaviruses SARS-CoV, MERS-CoV and SARS-CoV2 are comparatively poorly adapted to humans, which affects their pathogenic potential [55,59]. Their genomic proximity to animal CoVs may allow for ongoing interspecies recombination events, as observed in MERS [60]. MERS-CoV, SARS-CoV and SARS-CoV2 have a natural reservoir in bats. Infection of humans likely occurred through intermediate hosts, including dromedary camels (MERS), the masked palm civet (SARS) and the pangolin (SARS-CoV2) [61]. As wild palm civets do not carry SARS-CoV, it must be assumed that the proximity of animals in markets facilitates recombination events and the emergence of novel viruses that may be pathogenic in humans [62,63].

Coronaviruses are spherical in shape. Their most prominent feature are club-like projections on the virus surface which are referred to as “spikes”. The virus membrane contains four structural components, the spike (S), envelope (E), membrane (M) and nucleocapsid (N) protein [56] (Fig. 1). For SARS-CoV and SARS-CoV2, the S protein is the primary determinant for host tropism and pathogenicity. It is the main target for neutralizing antibodies and therefore of great interest in terms of immunological response and vaccine design [64]. The spike structure is formed by homotrimers of S-glycoproteins, each of which

![Structure of SARS-CoV2](https://i.imgur.com/1234567.png)

**Fig. 1. Structure of SARS-CoV2.** The spike protein (S) facilitates binding to the trans-membrane ACE2 host receptor; the envelope (E) protein together with the membrane (M) protein form the viral envelope and determine its shape; the hemagglutinin esterase (HE) protein may resemble another cell entry mechanism of novel CoVs; the nucleocapsid (N) protein in bound to the RNA genome of the virus to form the nucleocapsid.

S. Felsenstein, et al.

consists of two subunits, whereby S1 forms the part involved in receptor recognition, and S2 is highly conserved, anchors the protein in the viral membrane and facilitates viral fusion [65–67]. S1 contains a hypervariable loop which differs greatly between betacoronaviruses on both size and sequence. Viral entry requires the proteolysis of the S protein in two locations, a process that utilizes host proteases, and results in irreversible conformational changes of the S protein [64, 67]. Some anti-SARS-CoV antibodies in humans mimic receptor engagement, thus modeling conformational S protein changes upon antigen-antibody interaction [67]. The amino acid sequence of receptor binding sites of SARS-CoV2 is 74% homologous to that of SARS-CoV suggesting similar or even identical cell entry mechanisms for both viruses [68].

NL63, SARS-CoV and SARS-CoV2 all use the transmembrane angiotensin converting enzyme (ACE)2 as host receptor, whereas MERS CoV utilizes dipeptidylpeptidase-4 (DPP4) [65]. Both receptors are transmembrane ectoenzymes that are highly conserved among mammals, thus facilitating interspecies transfer. However, their enzymatic activity in itself is not necessary for successful binding and fusion [69–71].

Binding affinity of the S protein of SARS-CoV2 and ACE2 is high. High sequence and conformational conservation of the S protein observed across SARS-CoV2 and SARS-CoV allows for some level of cross neutralization of the two viruses *in vitro* [64, 68].

Hemagglutinin residues enhance binding by allowing interactions with sialic acid residues on host cell surfaces. Betacoronaviruses feature yet another structural protein, hemagglutinin-esterase (HE) which binds sialic acid on cell surfaces [72] (Fig. 1). This may enhance the virus' ability to bind and invade host cell surfaces and may constitute a virulence factor in novel hCoVs.

### 4. Immune pathology of COVID-19

While an estimated 80% of SARS-CoV2 infections are asymptomatic or result in mild disease, the remaining 20% of patients are severely or critically unwell [73, 74]. Currently, limited information is available on host factors affecting individual outcomes in COVID-19.

#### 4.1. Mechanisms of infection and immune evasion

While data on SARS-CoV2 are still sparse, aforementioned parallels with SARS-CoV and MERS-CoV may (for now) allow extrapolation of knowledge to understand how SARS-CoV2 escapes the host's immune response. Notably, SARS-CoV2 shares almost 80% RNA sequence homology with SARS-CoV, and 50% with MERS-CoV [75], with SARS-CoV2 exhibiting additional genomic regions when compared to SARS-CoV. In particular, the viral spike protein, which binds to the host cell receptor, is 20-30 amino acids longer than SARS-CoV, and other closely related coronaviruses [75]. Thus, it is possible, even likely, that SARS-CoV2 uses similar immune evasion strategies to other coronaviruses, but additional as yet undiscovered mechanisms may also be utilized by SARS-CoV2 [76].

As mentioned above, SARS-CoV and SARS-CoV2 both use ACE2 as their host cell receptor to establish infection (Fig. 2A) [77]. ACE2 is expressed in almost all organs in the body. ACE2 has been shown to be highly expressed on surfactant producing type 2 alveolar cells, and on ciliated and goblet cells in the airways; these cells likely provide a portal of entry for the virus in humans [78–80]. High ACE2 expression is also observed on the intestinal epithelium [81]. Furthermore, ACE2 is expressed on cardiac cells and vascular endothelia, which may explain cardiovascular complications in some patients [53]. For SARS-CoV, infection of immune cells including monocytes/macrophages and T cells has been observed. It is not clear to date whether and to what extent SARS-CoV-2 can also infect these cell types. ACE2 is also, but at lower levels and not ubiquitously, expressed on monocytes and macrophages, so this may also provide an entry mechanism into immune cells for SARS-CoV-2. However, other receptors and/or phagocytosis of virus containing immune complexes may also be involved (Fig. 1B) [76, 82, 83].

The host response and clearance of viral infections heavily relies on type I interferon (T1IFN) expression [84]. Expression of T1IFN and down-stream signals modulate cell responses and reprogram cells into an “anti-viral state”, subsequently promoting infection control and pathogen clearance [85]. As a first step, immune cells sense viral infection through identification of virus derived pattern associated molecular patterns (PAMPs), such as viral RNA. These bind to and activate pattern recognition receptors (PRRs) in/on immune cells and result in immune cell activation (Fig. 2). RNAs viruses, such as SARS-CoV, SARS-CoV2 and MERS-CoV are detected by endosomal RNA PRRs, including Toll-like receptors (TLR-)3 and 7 and/or cytoplasmic RNA sensors, namely retinoic acid-inducible gene I (RIG-I) and melanoma differentiation-associated protein 5 (MDA5) (Fig. 2). Usually, TLR3/7 activation results in nuclear translocation of the transcription factors NFκB and IRF3, while RIG-1/MDA5 activation result in activation of IRF3. In turn, this triggers increased expression of T1IFN (through IRF3) and other innate pro-inflammatory cytokines (IL-1, IL-6, TNF-α through NFκB) [76, 86]. In this context, T1IFN and other innate pro-inflammatory cytokines promote their own expression through auto-amplification: T1IFN activate the IFN-α receptor complex (IFNAR) which results in the phosphorylation/activation of STAT family transcription factors 1 and 2 (Fig. 2), while IL-1, IL-6, and TNF receptor activation feeds into pro-inflammatory cytokine expression though the transcription factor NFκB (Fig. 2) [85–87]. Activation and priming of innate and adaptive immune responses should result in pathogen clearance and recovery.

However, in a proportion of infected individuals, SARS-CoV, MERS-CoV and likely SARS-CoV2 evade immune system recognition through suppression of these mechanisms, a phenomenon associated with more severe disease and poorer prognosis [38, 88, 89](Fig. 2, red symbols). SARS-CoV has been shown to alter ubiquitination and degradation of RNA sensors (RIG-I and MDA5). It inhibits activation of mitochondrial antiviral-signaling protein (MAVS), which are essential for the activation and nuclear translocation of IRF3 in response to cytoplasmic RNA sensor activation. Furthermore, SARS-CoV, and likely SARS-CoV2, inhibit the TNF receptor-associated factors (TRAF) 3 and 6, which are central for the induction of IRF-3/7 in response to TLR3/7 and/or RIG-I and MDA-5 ligation as well as NFκB signalling pathways (which are usually activated in response to TLR3/7 ligation or cytokine receptor signaling) [88]. Lastly, novel coronaviruses can counteract T1IFN signaling through inhibition of STAT family transcription factor phosphorylation [86]. Taken together, suppression of innate immune mechanisms in infected epithelial cells and, to some extent, infected monocytes/macrophages allow novel coronaviruses to proliferate without triggering the innate anti-viral response machinery of these cells.

However, at a later stage, infected cells undergo cell death and release virus particles together with intracellular components that trigger innate inflammatory mechanisms through their recognition by PRRs in/on innate immune cells. As a result of this innate immune activation and resultant expression of pro-inflammatory cytokines (including IL-1β, IL-6, TNF-α, etc.), adaptive immune cells become involved in the host’s defense against viral infections. T lymphocytes play a central role in this anti-viral response, including CD4⁺ T cell derived cytokines, CD8⁺ T cell mediated cytotoxicity, and B cell activation resulting in antibody production. Novel coronaviruses may also (partially) escape these mechanisms through the induction of T cell apoptosis [90]. However, lymphocytes may also become depleted due to the expression of pro-inflammatory cytokines by (not infected) innate immune cells that become recruited to the lungs and trigger hyper-inflammation, seen during the development of a “cytokine storm” [91].

S. Felsenstein, et al.

Clinical Immunology 215 (2020) 108448

Fig. 2. Immune evasion strategies of SARS-CoV2. A) SARS-CoV2 infects airway epithelial cells through interactions with the trans-membrane enzyme ACE2 (a). While RNA viruses usually activate TLR3 and/or 7 in endosomes (b) and cytosolic RNA sensors RIG-I and MDA-5 (c), SARS-COV2 effectively suppresses the activation of TNF receptor-associated factors (TRAF) 3 and 6, thereby limiting activation of the transcription factors NFκB and IRF3 and 7, thereby suppressing early pro-inflammatory responses through type I interferons (IFN) and pro-inflammatory effector cytokines IL-1, IL-6 and TNF-α (red symbols). Furthermore, novel CoVs inhibit the activation of STAT transcription factors (d) in response to type I IFN receptor activation, which further limits antiviral response mechanisms. Altogether, this prohibits virus containment through activation of anti-viral programs and the recruitment of immune cells. B) Tissue monocytes/macrophages express ACE2 to a significantly lower extent, making infection through this route less likely (a). However, immune complexes consisting of ineffective antibodies against e.g. seasonal CoVs and virus particles may be taken up by macrophages through Fcγ receptors resulting in their infection (b). In a process referred to as antibody directed enhancement (ADE), virions inhibit type I IFN signaling in infected macrophages while allowing pro-inflammatory IL-1, IL-6 and TNF-α expression, which may contribute to hyperinflammation and cytokine storm syndrome (c,d). Inhibited type 1 IFN signaling suppresses anti-viral programs, while increased IL-1, IL-6 and TNF-α expression auto-amplifies itself through positive feedback loops (f).

4.2. Hyperinflammation and cytokine storm

While symptoms of COVID-19 disease may be (sometimes only slightly) milder in comparison to infections with SARS-CoV or MERS-CoV, several key pathogen-associated and clinical features of disease are similar and we can extrapolate knowledge from what is already known about the pathophysiology of SARS and MERS.

In COVID-19, as in SARS or MERS, several key findings were associated with poor outcomes in cohort studies, and suggest hyper-inflammation may be linked to more severe disease. Three early studies from Wuhan linked cytopenia and/or significantly elevated inflammatory parameters with severe disease and unfavorable outcomes. One study, involving 99 patients reported neutrophilia (38%), lymphopenia (35%), and increased systemic inflammatory proteins (IL-6 in 52%, and CRP in 84%) as common symptoms in COVID-19 disease [45]. Another study involving 41 individuals, linked severe disease culminating in ICU admission and mortality, with neutrophilia and lymphopenia [4]. The third study reported significant leukopenia (11.8%), lymphopenia (77.6%), thrombopenia (41.2%), anemia (48.2%), hypofibrinogenemia (22.4%), and hypo-albuminemia (78.8%) in a cohort of 85 patients who died from COVID-19 [83,92]. These observations are in line with findings in severe or lethal cases of SARS and MERS, in which increased numbers of neutrophils and monocytes/macrophages are present in the airways [83,93]. Other groups reported severe clinical phenotypes and ICU dependency of patients to be associated with increased plasma levels of innate chemokines, specifically C-X-C motif chemokine 10 (CXCL10)/Interferon gamma-induced protein 10 (IP-10), chemokine (C-C motif) ligand 2 (CCL2)/monocyte chemoattractant protein 1 (MCP-1), Macrophage Inflammatory Protein (MIP-)1A/CCL3, and the pro-inflammatory cytokine TNF-α [2]. This, indeed, is similar to the situation reported in SARS and MERS in which uncontrolled inflammation centrally contributes to poor outcomes [94–96].

Though seemingly contradictory to mechanisms of immune evasion discussed above, enhanced innate immune activation, including increased T1IFN, IL-1β, IL-6, and TNF-α expression centrally contributes to morbidity and mortality in COVID-19, MERS and SARS. One possible explanation is the induction of endothelial and vascular cell damage and cell death as a result of viral replication. Virus-induced inflammation cell death, including necrosis or pyroptosis result in pro-inflammatory cytokine expression, (uninfected) immune cell recruitment and activation [97]. Mice infected with SARS-CoV exhibit

S. Felsenstein, et al.

Clinical Immunology 215 (2020) 108448

Fig. 3. Inflammatory response through monocytes/macrophages. Uninfected monocytes/macrophages from the blood stream invade the lungs where they detect virus particles and/or cytoplasmic and nuclear components. Within immune complexes, these particles are taken up into the cell (a) where they are presented to TLRs, activating NFκB and/or IRF dependent pro-inflammatory pathways (b,c). As a result, uninfected monocytes/macrophages produce significant amounts of pro-inflammatory cytokines (d,e) which recruit additional innate and adaptive immune cells and cause additional tissue damage.

excessive T1IFN secretion from myeloid cells in infected tissues. Indeed, immune evasion through the suppression of anti-viral responses and T1IFN expression in respiratory epithelia results in high viral loads [38]. From this, it is hypothesized that (not infected) monocytes/macrophages and neutrophils recruited to the site of infection exhibit strong and poorly controlled inflammatory responses, resulting in tissue damage and systemic inflammation, both of which contribute to morbidity and mortality [53](Fig. 3).

Another factor thought to contribute to organ damage and poor outcomes is the early production of neutralizing antibodies against coronaviruses. Antibody-dependent enhancement (ADE) is a phenomenon shown to contribute to damage accrual during viral infections. It has been shown to promote cellular uptake of virus particles bound in immune complexes, through their binding to Fcγ receptors (FcγR). This may contribute to aforementioned persistent viral replication in immune cells (including newly infected antigen-presenting cells), but also immune complex mediated inflammatory responses (Figs. 2,3,4), that contribute to tissue and organ damage, including acute respiratory distress syndrome (ARDS) [98–100]. Indeed, a subset of COVID-19 patients reportedly develop vasculitic lesions, blood vessel occlusion and infarctions. Histopathologic reports from tissue sections suggests features associated with immune complex mediated vasculitis, including infiltration of monocytes and lymphocytes within and around

blood vessels, wall thickening, and focal hemorrhage [53,101–103].

As is true for a number of systemic autoimmune/inflammatory conditions, uncontrolled activation of immune responses is (likely) not limited to the innate mechanisms. As a result of pro-inflammatory cytokine expression and the presence of nuclear antigens (from cell and tissue damage), adaptive immune cells may become activated and trigger a “second wave” of inflammation (potentially in those patients who deteriorate after 7-10 days of infection). Indeed, adaptive immune cells, namely T lymphocytes, which are observed in lung tissue sections of COVID-19 patients with ARDS and/or cytokine storm, may drive inflammation at later disease stages. Similar mechanisms have been reported in influenza and other viral infections [104,105]. Overall, severely ill COVID-19 patients experiencing cytokine storm exhibit lymphopenia and sometimes atrophy of the lymphatic tissues, namely lymph nodes and spleen [51,106,107]. This is in line with reports in primary and secondary forms of Hemophagocytic lymphohistiocytosis (HLH) and associated cytokine storm, which result in inflammatory cell death and hypo-cellularity of lymphatic organs [108–110].

### 4.3. Host factors affecting individual risk and outcomes

Poor outcomes are associated with age; indeed, children appear to contract SARS-CoV2 and usually do not develop severe symptoms or

**Complement activation**

**Immune complex deposition**

**Mast cell degranulation**

**Immune cell recruitment**

**Pro-inflammatory cytokine expression**

**Neutrophil and NK cell degranulation**

**Tissue inflammation and damage**

**Tissue inflammation and damage**

**Vasodilation and oedema**

Fig. 4. Inflammatory mechanisms in immune complex vasculitis.

S. Felsenstein, et al.

complications. This is surprising as children are prone to viral infections including severe manifestations. More than 75% of children get exposed to seasonal coronaviruses before their $4^{\text {th }}$ birthday and seroconverts. However, antibody titres wane over time, most obvious in those over 60 years [110]. This may reduce immune response to SARS-CoV2 in the elderly as (limited) cross-reactivity between anti-seasonal coronavirus and anti-SARS antibodies exists, but also contribute to increased inflammation and complications. Immunological recall effects exist as anti-seasonal coronavirus titres increase in sera of convalescent SARS patients [111] which may influence immune pathology. As mentioned above, antibody-bound virions can enter susceptible cells, such as macrophages through Fcγ receptor ligation in a process termed antibody-dependent enhancement (ADE) [112]. In other viral infections (e.g. Dengue fever), ADE allows immune cell infection and reduces type I IFN dependent antiviral responses while promoting pro-inflammatory IL-6 and TNF-α expression [113,114]. Furthermore, massive recall antibody production in individuals with a history of exposure to seasonal coronaviruses but waning titres, such as the elderly, can result in immune complex deposition and promote inflammation and damage, including immune complex vasculitis [110].

Another age-dependent disease mechanism may be associated with live vaccinations (e.g. measles or BCG). Vaccines protect beyond their target antigen through induction of innate immune mechanisms termed non-specific heterologous effects. Individuals who received BCG vaccination produce increased levels of pro-inflammatory IL-1β and TNF-α in response to *S. aureus* or *Candida spp.*, and BCG vaccinated infants exhibit reduced infection-related mortality [115]. However, heterologous immune responses to unrelated antigens may also contrite to inflammation-related complications. Frequently, adults exhibit memory T cells that are specific to antigens they were never exposed to, and cross-reactive memory T cells can narrow the T cell response by favoring “high affinity” clones. Indeed, limited memory T cell repertoires are a feature of immune senescence and associated with disease progression and T cell mediated damage in other viral infections, such as virus hepatitis and infective mononucleosis [116].

As mentioned above, ACE2 acts as transmembrane cellular receptor for SARS-CoV2 allowing cell infection [117]. Variable ACE2 expression patterns affect disease susceptibility between tissues (e.g. respiratory epithelia vs immune cells), but potentially also between individuals (men vs women, children vs adults) thereby determining disease progression and outcomes. Recently, it has been suggested that ACE2 expression is highest in children and young women, that its expression decreases with age, and is lowest in individuals with chronic disease, including diabetes and hypertension, inversely correlating with risk for severe disease and unfavorable outcomes [118]. While ACE2 facilitates viral entry into cells, it also plays a role in controlling infection and inflammation. ACE2 is part of the ACE2/angiotensin-[1–7]/MAS system as it counteracts the pro-inflammatory effects of the angiotensin-2. It catalyzes angiotensin-2 processing into angiotensin-1-7, which counteracts vasoconstriction, modulates leukocyte migration, cytokine expression, and fibrogenic pathways [119]. Thus, ACE2 contributes to limiting tissue inflammation while favoring repair mechanisms. Furthermore, “high” ACE2 expression may be of benefit as SARS-CoV2 virus particles may compete with angiotensin-2 for cell surface binding sites and cellular uptake. Thus, relatively increased ACE2 expression may explain why children and young adults, especially young women, are relatively protected from COVID19 and associated complications.

Taken together, novel coronaviruses, such as SARS-CoV2, may effectively suppress early T1IFN responses, which contributes to uncontrolled virus replication resulting in delayed and potentially increased cytokine responses at later stages. Early and sufficient control of virus replication and pathogen clearance may be altered in individuals at risk, such as the elderly, patients with diabetes or metabolic syndrome, etc. [74,75]. Healthy children and young people, on the other hand, may effectively control viral load at early stages of infection

and less frequently develop severe disease and life-threatening complications. Lastly, early antibody production may result in integration of viable virus into immune cells and increased viral replication, resulting in immune complex mediated pathology, which may contribute to pathology in young patients with no obvious risk factors [100].

## 5. Treatment

The rapid spread of SARS-CoV2 infection globally, has led to the immediate need for a vaccine or therapeutic intervention to prevent or treat COVID-19 disease. Due to the speed at which the virus has spread globally there are few studies on potential therapeutics interventions or vaccine candidates. Further, due to the minimal severity of the SARS (774 deaths globally) and MERS (866 deaths globally) epidemics, few studies to generate a vaccine or therapeutic for other closely related coronaviruses have been undertaken, which could have efficacy for COVID-19 disease. Clinical trials testing treatments for COVID-19 are being undertaken, results from large randomized studies though remain outstanding at this stage. As a result, the following sections are not to be mistaken as evidence based treatment recommendations, but reflect (mostly) anecdotal experience with experimental treatment, extrapolation of data from related conditions, and expert opinion (Fig. 4).

### 5.1. Anti-viral treatment

#### 5.1.1. (Hydroxy-)Chloroquine

Medical use of Chloroquine dates back decades. Its phosphate and sulphate derivatives are administered as antimalarials, and hydroxychloroquine is widely used as immunomodulatory agent in systemic lupus erythematosus. In addition, chloroquine has antiviral activity against Influenza, Chikungunya virus, seasonal CoVs, and SARS [120–123]. As for these viruses, cell entry and replication of SARS-CoV2 depends on pH-dependent internalization by endocytosis and lysosomal fusion (Fig. 2). Itself being a weak base, hydroxychloroquine follows the cellular pH gradient and accumulates in the acid environment of endolysosomes and other acidic cell organelles, thereby alkalinizing endosomes. In addition, hydroxychloroquine interferes with the terminal glycosylation of ACE2, interfering with virus binding [123].

Antiviral activity of chloroquine derivatives against SARS-CoV2 was identified *in vitro* early on [124]. Based on this, the drug was rapidly introduced into clinical use, and preliminary reports suggested improved viral clearance and clinical outcomes in COVID-19 patients receiving a 10-days course of Hydroxychloroquine [125]. A small French pilot study, randomizing 36 patients with COVID-19 suggested accelerated viral clearance in patients treated with a combination of hydroxychloroquine and azithromycin [126]. However, others have challenged results and found no benefit in either disease outcome or viral clearance [127]. Disappointingly, the largest (also retrospective) study to date assessing Hydroxychloroquine on its own or in combination with azithromycin found no benefit, but indeed an increased mortality risk among patients receiving hydroxychloroquine [128]. A study exploring chloroquine diphosphate in two dosing regimens was forced to terminate early for concerns over increased mortality in the high dose arm. The authors conclude that treatment with high dose chloroquine for 10 days is not sufficiently safe and should no longer be used in severe SARS-CoV2 patients [129].

Immunomodulatory effects of hydroxychloroquine are well established, and may enhance its therapeutic effect in COVID-19 complicated by macrophage activation and cytokine storm [130]. Alkalization of endosomes reduces proteolysis, chemotaxis, phagocytosis, receptor recycling, and interferes with processing of epitopes displayed by antigen-presenting cells [131]. This overall contributes to decreased production of IL-1, IL-6 and prostaglandins, and alters intracellular calcium and TLR dependent signaling. Furthermore, preventing the acidification of lysosomes, hydroxychloroquine impairs cellular autophagy, a critical step for innate and adaptive immunity activation [132]. Finally,

S. Felsenstein, et al.

Clinical Immunology 215 (2020) 108448

Fig. 5. Potential therapeutic targets in COVID-19. While no approved and evidence-based treatments are available for COVID-19, a number of treatments promise potential. Virus particles may be caught and inactivated using antibodies from convalescent patients. Recombinant soluble ACE2 protein may bind SARS-CoV2 and/or mediate anti-inflammatory effects to prevent pulmonary damage and hyper-inflammation. (Hydroxy-)chloroquine, potentially in combination with azithromycin), can change the pH of endosomes and reduce virus entry and replication. Furthermore, both medications have immune-modulating effects that may control pro-inflammatory cytokine expression. Anti-viral treatment with protease inhibitors (lopinavir, ritonavir, etc.) and/or nucleoside analogues (remdesivir, etc.) can limit virus replication. As SARS-CoV2 suppresses antiviral cytokine production, virus clearance may also be supported by the substitution of type I interferons, which activate their cytokine receptor (IFNAR) and induce anti-viral cellular programs.

Hyperinflammation and resulting tissue damage may be prevented through immune modulation. Blocking IL-1 signaling (e.g., through recombinant IL-1 receptor antagonist anakinra) or IL-6 signaling (e.g., through IL-6 receptor antibody tocilizumab) may limit further immune activation, tissue damage, and cytokine storms. Additional, less specific effects may be mediated through corticosteroids, immunoglobulins, hydroxychloroquine, and/or azithromycin.

hydroxychloroquine has antithrombotic effects, which may be beneficial in COVID-19, where inflammatory stimuli and endothelial injury activate coagulation and promote micro-thrombus formation [133,134].

While generally deemed safe when administered at correct dosing and under close monitoring, the therapeutic range of chloroquine and its derivatives is narrow. Side effects include conduction defects, cardiomyopathy, retinopathy, and hypoglycemia [135,136].

### 5.1.2. Azithromycin

As mentioned above, synergistic effects of azithromycin and hydroxychloroquine against SARS-CoV2 have been observed *in vitro*, which appeared to translate into clinical practice [126,137,138]. Interestingly, azithromycin is also a weak base, and also accumulates in endosomes, with an alkalinizing effect at least equivalent to Hydroxychloroquine. In addition to its antimicrobial properties, azithromycin is sometimes used for its immunomodulatory properties, especially in patients with chronic pulmonary disorders. Azithromycin polarizes macrophages towards an anti-inflammatory M2 phenotype, and inhibits pro-inflammatory STAT1 and NFκB signaling pathways [139,140]. In the context of anti-inflammatory effects, it is of particular interest that azithromycin is used in patients requiring intensive care for non-COVID-19 related ARDS and is associated with a significant reduction in mortality and shorter time to extubation [141–143].

Adverse cardiac effects and proarrhythmogenic properties of hydroxychloroquine, especially in combination with macrolide antibiotics, such as Azithromycin, deserves particular mention [144]. Hydroxychloroquine, azithromycin, and, to a lesser extent, lopinavir have been associated with prolongation of the QTc interval and increase the risk for tachyarrhythmias and sudden cardiac death. Careful consideration of patient risk profile, pre-treatment ECG assessment, and monitoring of pharmacokinetics, fluid and electrolyte status, and polypharmacy are essential for the management of critically ill COVID-19 patients [145].

### 5.1.3. Remdesivir and other nucleoside analogues

Nucleoside analogues are explored as treatment options for COVID-19. Candidates include favipiravir, galdesivir, ribavirin, and remdesivir, with the latter having received the most attention. Remdesivir, a prodrug to adenosine [146], was originally developed for the treatment of hemorrhagic fever viruses, namely Ebola (EBOV) and Marburg viruses, but underperformed in EBOV treatment compared to antibody

### 5.1.4. Protease inhibitors Lopinavir/ritonavir (LPV/r)

The combination of lopinavir and ritonavir (LPV/r), better known by tradenames Aluvia® and Kaletra®, is a frequently used antiretroviral treatment for HIV. Combining two protease inhibitors limits otherwise extensive CYP3A4 activation and drug metabolism, thereby resulting in much improved bioavailability of LPV [154]. Proteases are critical for

viral replication, as they cleave both structural and functional proteins from precursor viral polypeptides (Fig. 5), thus enabling maturation into an infectious virion particle. LPV/r is mainly metabolized in the liver, and thus pre-existing hepatic impairment is considered a relative contraindication [154].

In SARS, LPV/r in combination with Ribavirin was associated with a significant reduction in unfavorable outcomes (ARDS and death) as compared to ribavirin alone (2.4% versus 28.8%) [155]. Similar observations were made in a retrospective cohort study involving >1000 SARS patients, where LPV/r was associated with significantly reduced mortality and need for intubation [73,156]; and it is currently being investigated by the WHO for use in MERS patients in an ongoing randomized clinical trial. Early use of LPV/r was recommended, based on aforementioned pathophysiological considerations rather than clinical data.

In view of the *in vitro* activity against both SARS and MERS and the limited clinical data available for LPV/r in treatment of critically ill SARS-CoV2 patients, a randomized open-label trial was undertaken in Wuhan, China [157]. It recruited almost 200 COVID-19 patients, randomized to either standard treatment or added LPV/r for 14 days. Whilst confirming safety of LPV/r use in COVID-19, no significant differences were seen between groups in relation to survival or time to recovery; thus leaving the authors question whether a combination of LPV/r with a nucleoside analogue, such as ribavirin, would have resulted in improved outcomes. Trials exploring the therapeutic potential for LPV/r are currently ongoing (https://www.remapcap.org/coronavirus; https://www.who.int/emergencies/diseases/novel-coronavirus-2019/global-research-on-novel-coronavirus-2019-ncov/solidarity-clinical-trial-for-covid-19-treatments).

### 5.1.5. Recombinant soluble ACE2

As ACE2 has been identified as a key molecule for cell invasion (see above), its therapeutic blockade to control disease and aid viral clearance has been suggested [158]. However, unselective ACE blockade with currently available agents may be problematic as it could alter angiotensin-1 through -7, which have anti-inflammatory and anti-fibrotic properties [159]. Indeed, depletion of ACE2 by SARS-CoV2 may potentially contribute to increased disease activity in critically ill COVID-19 patients. In animal studies, ACE2 protects from ARDS [160–162], while angiotensin II contributes to pulmonary pathology, including edema and fibrosis [163]. Thus, accumulation of angiotensin II in the absence of ACE2 may aggravate disease and organ damage. Consequently, ACE2 induction has recently been suggested for COVID-19 treatment [85,159,164,165]. However, effects of ACE2 may vary between tissues and environments. Intestinal epithelia produce much higher levels of ACE2 than bronchial epithelia which is notable as not all patients develop gastrointestinal symptoms and when they do, symptoms tend to be mild, and some patients remain SARS-CoV2 positive in stool samples long after respiratory specimen became negative [78,166–168]. Based on these observations, one could suggest that high-level ACE2 expression such as that seen in the intestine and in contrast to the respiratory tract, or in children and young people as compared to individuals at risk (the elderly, especially when obese or chronically ill), may protect from inflammation and tissue damage. However, additional factors, such as the immunological micro-environment or regionally variable microbiomes may significantly affect virus uptake, replication and/or clearance. Thus, the exact role(s) of ACE2 in the context of COVID-19 remains to be unveiled and may be complex.

The administration of recombinant human ACE2 to neutralize viruses prior to their attachment to the host cells is also being explored as a therapeutic option in the future. In the attempt of exploiting the anti-inflammatory effect of the ACE2/Ang-[1–7]/Mas axis in non-COVID-19 related ARDS, first pilot trials in humans have been published [169], and whilst data supporting its efficacy as an ARDS treatment option remains outstanding, the treatment appeared safe and was well tolerated.

### 5.1.6. Type 1 interferons

As mentioned above, SARS-CoV2 effectively inhibits the expression of type 1 interferons [38]. Resulting tissue damage and expression of pro-inflammatory cytokines and chemokines from infected monocytes/macrophages promote excessive immune cell infiltration and cytokine responses [114]. More recently, also abortive infection in T lymphocytes with SARS-CoV2 has been suggested [170], but detailed characterization remains outstanding. Altogether, unaltered virus replication in the presence of tissue damage and inflammatory cytokine expression can explain ARDS and cytokine storms in COVID-19. Overcoming immune evasion and enhancing antiviral activity may be a logical treatment strategy.

In SARS and MERS patients, recombinant interferons have been used with varying success. While antiviral activity of recombinant IFN-α2a, IFN-α2b, IFN-β1a and IFN-β1b was shown *in vitro* for MERS, SARS and SARS CoV2, neither mortality nor viral clearance were affected by recombinant interferons in MERS [171,172]. However, the time of administration may be critical, as suggested by a mouse model of IFN I treatment for MERS [173], therefore human patients may have received treatment too late to be fully effective.

### 5.1.7. Plasma from convalescent patients

Convalescent plasma, i.e. plasma from individuals following COVID-19 resolution and rich in immunoglobulins directed against SARS-CoV2, is being entertained as possible treatment option [174,175]. Anecdotal use in SARS, MERS, Ebola and Influenza patients supports its use as a neutralizing and/or immunomodulatory agent [176,177]. However, a larger randomized controlled assessment of hyperimmune intravenous immunoglobulin use for severe influenza [178,179] and Ebola [180] showed this intervention to not be superior to placebo. Similarly, rigorously evaluated data for its use in coronaviral infections is lacking—not only for its use in SARS-CoV2 [181], and a feasibility study exploring its use in MERS found that in many survivors, antibody titres were not high enough, thus further limiting the donor pool [182]. Variable dosing, issues surrounding donor recruitment in times of rapidly increasing patient numbers, and drawbacks regarding safety of widespread use of human blood products all limit the availability and utility as widely available treatment option.

Finally, in viruses that are subject to ADE (such as SEARS-CoV2, see above) by non-neutralizing antibodies, the option of plasma therapy also holds significant risks. This complication has recently been exemplified by anti-Zika virus antibodies enhancing Dengue virus infection [183]. Thus, the administration of hyperimmune/convalescent plasma may carry the risk of significant illness upon future exposure to related or yet-to-emerge coronaviruses.

### 5.2. Calming the cytokine storm through immune modulation

As mentioned above, current management of COVID-19 is mainly supportive and approved treatments based on scientific evidence are not available. Main causes of death include ARDS and cytokine storm syndrome (also referred to as macrophage activation syndrome, MAS or secondary Hemophagocytic histiocytosis, HLH) [74,92,106,107]. Indeed, ARDS occurs in 50% of patients with cytokine storm syndrome [184]). Considering impressively rapid development of systemic and pulmonary inflammation in a subset of patients with COVID-19, early identification and control of derailed immune responses is of utmost importance. Based on data from Chinese cohorts, markers associated with cytokine storm in other conditions may be predictive of poor outcomes in COVID-19, which include leukopenia, lymphopenia, thrombopenia, hypoalbuminemia, significantly elevated CRP and IL-6, hyperfibringenemia, and prolonged thrombin time [74,185,186]. However, this needs to be tested prospectively, and other more sensitive and specific biomarker may be identified.

S. Felsenstein, et al.

First data on cytokine storm syndrome and its catastrophic effects on tissues and organs was generated in patients with familial HLH, in which mutations in associated genes (including *PRF1*, *UNC13D*, *STX11*, *STXP2*, *LYST*, *XIAP*, and others) result in systemic inflammation and, if not controlled, death [109]. Standard treatment in these conditions include high-dose corticosteroids (dexamethasone), the calcineurin inhibitor cyclosporine A, chemotherapy with etoposide, and ultimately stem cell transplantation [187]. While the underlying molecular causes of familial HLH are different to COVID-19 associated cytokine storm syndrome, clinical (fevers, organomegaly in some patients) and laboratory features (cytopenias, massively elevated inflammatory parameters including CRP, ESR and ferritin, hypalbuminemia, hyperfibrinogenemia, etc.) and consequences (tissue and organ damage, death) overlap. Furthermore, based on observations in the H1N1 influenza pandemic in 2009, a significant proportion of individuals developing disease-associated secondary cytokine storm syndrome may have mutations in one or more genes associated with familial HLH (in H1N1 36% of fatalities were associated with mutations in genes associated with the perforin pathway [188]). Thus, clinical management of COVID-19 associated cytokine storm syndrome may, to some extent, be informed by what we know from familial HLH. However, treatment of COVID-19 associated cytokine storm should be more targeted and not include cytotoxic drugs and/or stem cell transplantation, as it is secondary to an infection, which will hopefully be cleared.

Corticosteroids are used in primary and secondary forms of HLH, and can control inflammation in ARDS [91,189]. First preliminary data from SARS and COVID-19 suggest that high-dose steroids did not have beneficial effects on lung injury [190,191]. Instead, high-dose corticosteroids are associated with complications in other forms of ARDS, including avascular osteonecrosis [192]. Short courses of low- or medium-dose corticosteroids, however, have been suggested to be of benefit in a Chinese cohort of critically ill COVID-19 patients [193]. Taken together, the limited data on the efficacy and safety of corticosteroids in ARDS are anecdotal and not conclusive; controlled trials do not exist. As their use is associated with widely variable effects on pathogen clearance, and evidence for their efficacy is lacking, high-dose corticosteroids cannot be generally recommended for the treatment of COVID-19 [194], and the use of low dose regimens must be trialed in formal and controlled studies.

Intravenous immunoglobulins (IVIG) are used in systemic autoimmune/inflammatory conditions to control systemic inflammation through several mechanisms, including the capture of activated complement factors, blockade of Fcγ receptors, inhibition of B and T lymphocyte differentiation and activation, neutralisation of cytokines and antibodies, etc. [195]. As mentioned above, immune complexes containing viable virus may mediate infection, activate Fcγ receptors, and/or be deposited in tissues and organs, lastly resulting in pro-inflammatory responses [196]. Of note, ARDS and cytokine storm in SARS coincided with serum conversion in a majority of patients supporting these arguments. Furthermore, patients who ultimately died, seroconverted significantly earlier when compared to individuals who recovered from infection [40,197]. Based on these observations, IVIG may be of benefit to some patients by inhibiting Fcγ receptors and limiting antibody-dependent enhancement (discussed above). Furthermore, aforementioned “classical” anti-inflammatory effects may limit systemic inflammation, and anti-pathogen properties may be supportive in cases with bacterial superinfection or in patients who previously cleared SARS-CoV2 and developed specific antibodies [186,198].

The blockade of cytokines associated with hyper-inflammation during COVID-19 is a more targeted approach when compared to the use of systemic corticosteroids, and is a promising therapeutic avenue. Indeed, first anecdotal reports suggest efficacy at least in some patients. The IL-6 receptor antagonist tocilizumab has been used successfully in patients with secondary cytokine storm syndrome [199], including COVID-19 [200,201]. Several studies have started or are about to be launched, investigating efficacy and safety of tocilizumab in patients

with secondary cytokine storm syndrome in COVID-19 (including ChiCTR2000029765 in China) [202].

The recombinant IL-1 receptor antagonist anakinra was originally developed to control cytokine storm and associated tissue damage in sepsis patients [203]. Subsequently, anakinra has successfully been used in patients with cytokine storm syndrome secondary to autoimmune/inflammatory [204,205] infectious or malignant disease [206]. Anakinra may have significant potential at controlling hyper-inflammation in severe COVID-19 disease, considering the absence of severe side-effects in aforementioned sepsis trials [203], and reduced frequency of neutropenia and hepatotoxicity when compared to tocilizumab. Currently, anakinra is being trialled in a randomised placebo-controlled study in children and adults with COVID-19 associated cytokine storm syndrome in China (NCT02780583) [91].

**Inhibition of Janus kinases (JAK)** with small molecules is a relatively new concept used in systemic autoimmune/inflammatory conditions. JAKs are involved in cytokine receptor signaling, including (but not limited to) the IL-6 receptor, as well as type 1 and type 2 IFN receptors [91,207]. They mediate the phosphorylation of STAT family transcription factors which are, in turn, involved in pro-inflammatory cytokine expression. Thus, JAK inhibitors efficiently limit cytokine expression, and may aid in controlling cytokine storms [91]. However, JAKs are also centrally involved in controlling the expression of T1IFN, which plays a key role in limiting virus replication and initiating pathogen clearance [208]. At least in the initial stages of COVID-19 disease, when virus replication and infection may be limited to the epithelium, SARS-CoV2 likely limits T1IFN expression (see above). Therefore, additional inhibition of JAK through small molecules may be counterproductive as they further limit pathogen containment and clearance, and may cause unforeseeable complications. Thus, JAK inhibition may not be the most suitable “target-directed” treatment option in COVID-19 associated cytokine storm syndrome and/or ARDS. To our knowledge, at least two clinical trials are ongoing to test efficacy and safety of JAK inhibitors in severe COVID-19 (ChiCTR2000030170, ChiCTR2000029580).

6. **COVID-19 in patients receiving immune modulating treatment**

The previously discussed mechanisms of infection, immune evasion, and dysregulation of innate and adaptive immune responses cause significant concern for and among patients on systemic immune modulating treatments, including patients with malignant or systemic autoimmune/inflammatory diseases. Based on previous coronavirus outbreaks (SARS and MERS) and first small observational studies in COVID-19 cohorts, risk factors for poor outcomes include old age, presence of comorbidities (diabetes, metabolic syndrome, etc.), obesity, male sex, coronary heart disease, chronic obstructive pulmonary disease, and kidney disease [209]. Of note, immune modulation or suppression was not identified as a risk factor for poor prognosis in China or Italy [186,210]. While this could generally be considered “good news”, immune suppression and associated altered immune function may predispose patients to infection and potentially prolong virus spreading. Furthermore, as COVID-19 is associated with lymphopenia, patients receiving immune modulating treatment may be prone to secondary infections, such as bacterial pneumonia.

As discussed above, some immune modulating drugs may protect from viral infections. Antimalarial drugs (chloroquine, hydroxychloroquine) may inhibit tissue infection and viral replication [53,103]. Furthermore, immune modulating medications (anti-malarial drugs, classical as well as biologic DMARDs, and others) may prevent or control cytokine storm syndromes.

Uncontrolled discontinuation of immune modulating treatment may result in disease flares in autoimmune/inflammatory conditions, organ rejection in transplant patients, or reoccurrence of malignancies, which (on top of obvious effects) may also all increase the risk for viral infection. Thus, national and international societies, including the ACR

and EULAR, recommend continuation of treatment in the absence of symptoms and alterations to existing treatment regimens only in agreement with and under close monitoring by the responsible clinical service [211, 212]. International collaboration is needed and under way to safely assess individual risk in these vulnerable patient groups. Until reliable data is available, close clinical monitoring and social distancing should be prioritized.

## 7. Conclusions

As immunity does not exist and a significant proportion of humans develop severe disease, the novel coronavirus SARS-CoV2 is a threat to millions globally. SARS-CoV2 has the capacity to escape innate immune responses, which allows the pathogen to produce large copy numbers in primarily infected tissues, usually airway epithelia. Through the infection of innate immune cells and/or the recruitment of uninfected cells from the circulation to the primary site of infection, massive immune reactions induce hyperinflammation that can result in a cytokine storm and life-threatening complications. We are only beginning to understand host factors, such as differential expression of cell surface proteins that may determine infection risk, disease presentation and outcomes. Unveiling tissue and stage specific factors contributing to pathology will result in new, effective and disease stage specific therapeutic approaches that control virus replication while limiting inflammatory damage until vaccinations become available.

# References

[1] Z. Song, Y. Xu, L. Bao, L. Zhang, P. Yu, Y. Qu, et al., From SARS to MERS, thrusting coronaviruses into the spotlight, Viruses 11 (1) (2019).

[2] C. Drosten, S. Gunther, W. Preiser, S. van der Werf, H.R. Brodt, S. Becker, et al., Identification of a novel coronavirus in patients with severe acute respiratory syndrome, N. Engl. J. Med. 348 (20) (2003) 1967–1976.

[3] A.M. Zaki, S. van Boheemen, T.M. Bestebroer, A.D. Osterhaus, R.A. Fouchier, Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia, N. Engl. J. Med. 367 (19) (2012) 1814–1820.

[4] F. Wu, S. Zhao, B. Yu, Y.M. Chen, W. Wang, Z.G. Song, et al., A new coronavirus associated with human respiratory disease in China, Nature. 579 (7798) (2020) 265–269.

[5] D. Hamre, J.J. Procknow, A new virus isolated from the human respiratory tract, Proc. Soc. Exp. Biol. Med. 121 (1) (1966) 190–193.

[6] K. McIntosh, J.H. Dees, W.B. Becker, A.Z. Kapikian, R.M. Chanock, Recovery in tracheal organ cultures of novel viruses from patients with respiratory disease, Proc. Natl. Acad. Sci. U. S. A. 57 (4) (1967) 933–940.

[7] L. van der Hoek, K. Pyrc, M.F. Jebbink, W. Vermeulen-Oost, R.J. Berkhout, K.C. Wolthers, et al., Identification of a new human coronavirus, Nat. Med. 10 (4) (2004) 368–373.

[8] P.C. Woo, S.K. Lau, Y. Huang, H.W. Tsoi, K.H. Chan, K.Y. Yuen, Phylogenetic and recombination analysis of coronavirus HKU1, a novel coronavirus from patients with pneumonia, Arch. Virol. 150 (11) (2005) 2299–2311.

[9] E.R. Gaunt, A. Hardie, E.C. Claas, P. Simmonds, K.E. Templeton, Epidemiology and clinical presentations of the four human coronaviruses 229E, HKU1, NL63, and OC43 detected over 3 years using a novel multiplex real-time PCR method, J. Clin. Microbiol. 48 (8) (2010) 2940–2947.

[10] A. Annan, F. Ebach, V.M. Corman, R. Krumkamp, Y. Adu-Sarkodie, A.M. Eis-Hubinger, et al., Similar virus spectra and seasonality in paediatric patients with acute respiratory disease, Ghana and Germany. Clin Microbiol Infect. 22 (4) (2016) 340–346.

[11] J.A. Berkley, P. Munywoki, M. Ngama, S. Kazungu, J. Abwao, A. Bett, et al., Viral etiology of severe pneumonia among Kenyan infants and children, JAMA. 303 (20) (2010) 2051–2057.

[12] H.E. Larson, S.E. Reed, D.A. Tyrrell, Isolation of rhinoviruses and coronaviruses from 38 colds in adults, J. Med. Virol. 5 (3) (1980) 221–229.

[13] S.R. Dominguez, C.C. Robinson, K.V. Holmes, Detection of four human coronaviruses in respiratory infections in children: a one-year study in Colorado, J. Med. Virol. 81 (9) (2009) 1597–1604.

[14] R. Dijkman, M.F. Jebbink, N.B. El Idrissi, K. Pyrc, M.A. Muller, T.W. Kuipers, et al., Human coronavirus NL63 and 229E seroconversion in children, J. Clin. Microbiol. 46 (7) (2008) 2368–2373.

[15] X. Shao, X. Guo, F. Esper, C. Weibel, J.S. Kahn, Seroprevalence of group I human coronaviruses in children, J. Clin. Virol. 40 (3) (2007) 207–213.

[16] R. Dijkman, M.F. Jebbink, E. Gaunt, J.W. Rossen, K.E. Templeton, T.W. Kuipers, et al., The dominance of human coronavirus OC43 and NL63 infections in infants, J. Clin. Virol. 53 (2) (2012) 135–139.

[17] T.K. Cabeca, A.M. Passos, C. Granato, N. Bellei, Human coronavirus occurrence in different populations of Sao Paulo: A comprehensive nine-year study using a pancoronavirus RT-PCR assay, Braz. J. Microbiol. 44 (1) (2013) 335–339.

[18] N. Friedman, H. Alter, M. Hindiyeh, E. Mendelson, Y. Shemer Avni, M. Mandelboim, Human coronavirus infections in Israel: epidemiology, clinical symptoms and summer seasonality of HCoV-HKU1, Viruses 10 (10) (2018).

[19] T.K. Cabeca, C. Granato, N. Bellei, Epidemiological and clinical features of human coronavirus infections among different subsets of patients, Influenza Other Respir. Viruses 7 (6) (2013) 1040–1047.

[20] T.K. Cabeca, E. Carraro, A. Watanabe, C. Granato, N. Bellei, Infections with human coronaviruses NL63 and OC43 among hospitalised and outpatient individuals in Sao Paulo, Brazil. Mem Inst Oswaldo Cruz. 107 (5) (2012) 693–694.

[21] X.Y. Zheng, Y.J. Xu, W.J. Guan, L.F. Lin, Regional, age and respiratory-secretion-specific prevalence of respiratory viruses associated with asthma exacerbation: a literature review, Arch. Virol. 163 (4) (2018) 845–853.

[22] A. Boucher, M. Desforges, P. Duquette, P.J. Talbot, Long-term human coronavirus-myelin cross-reactive T-cell clones derived from multiple sclerosis patients, Clin. Immunol. 123 (3) (2007) 258–267.

[23] S. Morfopoulou, J.R. Brown, E.G. Davies, G. Anderson, A. Virasami, W. Qasim, et al., Human Coronavirus OC43 Associated with Fatal Encephalitis, N. Engl. J. Med. 375 (5) (2016) 497–498.

[24] T. Shi, K. McLean, H. Campbell, H. Nair, Aetiological role of common respiratory viruses in acute lower respiratory infections in children under five years: A systematic review and meta-analysis, J. Glob. Health 5 (1) (2015) 010408.

[25] F. Milano, A.P. Campbell, K.A. Guthrie, J. Kuypers, J.A. Englund, L. Corey, et al., Human rhinovirus and coronavirus detection among allogeneic hematopoietic stem cell transplantation recipients, Blood. 115 (10) (2010) 2088–2094.

[26] R.J. Singleton, L.R. Bulkow, K. Miernyk, C. DeByle, L. Pruitt, K.B. Hummel, et al., Viral respiratory infections in hospitalized and community control children in Alaska, J. Med. Virol. 82 (7) (2010) 1282–1290.

[27] R.M. Anderson, C. Fraser, A.C. Ghani, C.A. Donnelly, S. Riley, N.M. Ferguson, et al., Epidemiology, transmission dynamics and control of SARS: the 2002-2003 epidemic, Philos. Trans. R. Soc. Lond. Ser. B Biol. Sci. 359 (1447) (2004) 1091–1105.

[28] C.A. Donnelly, M.C. Fisher, C. Fraser, A.C. Ghani, S. Riley, N.M. Ferguson, et al., Epidemiological and genetic analysis of severe acute respiratory syndrome, Lancet Infect. Dis. 4 (11) (2004) 672–683.

[29] M. Ip, P.K. Chan, N. Lee, A. Wu, T.K. Ng, L. Chan, et al., Seroprevalence of antibody to severe acute respiratory syndrome (SARS)-associated coronavirus among health care workers in SARS and non-SARS medical wards, Clin. Infect. Dis. 38 (12) (2004) e116–e118.

[30] G.M. Leung, W.W. Lim, L.M. Ho, T.H. Lam, A.C. Ghani, C.A. Donnelly, et al., Seroprevalence of IgG antibodies to SARS-coronavirus in asymptomatic or subclinical population groups, Epidemiol. Infect. 134 (2) (2006) 211–221.

[31] P.P. Lee, W.H. Wong, G.M. Leung, S.S. Chiu, K.H. Chan, J.S. Peiris, et al., Risk-stratified seroprevalence of SARS coronavirus in children residing in a district with point-source outbreak compared to a low-risk area, Hong Kong Med J. 14 (Suppl. 4) (2008) 17–20.

[32] L.Y. Yam, R.C. Chen, N.S. Zhong, SARS: ventilatory and intensive care, Respirology. 8 (Suppl) (2003) S31–S35.

[33] L.F. Ng, M.L. Hibberd, E.E. Ooi, K.F. Tang, S.Y. Neo, J. Tan, et al., A human in vitro model system for investigating genome-wide host responses to SARS coronavirus infection, BMC Infect. Dis. 4 (2004) 34.

[34] G.Y. Oudit, Z. Kassiri, C. Jiang, P.P. Liu, S.M. Poutanen, J.M. Penninger, et al., SARS-coronavirus modulation of myocardial ACE2 expression and inflammation in patients with SARS, Eur. J. Clin. Investig. 39 (7) (2009) 618–625.

[35] R. Leth-Larsen, F. Zhong, V.T. Chow, U. Holmskov, J. Lu, The SARS coronavirus spike glycoprotein is selectively recognized by lung surfactant protein D and activates macrophages, Immunobiology. 212 (3) (2007) 201–211.

[36] H.K. Law, C.Y. Cheung, H.Y. Ng, S.F. Sia, Y.O. Chan, W. Luk, et al., Chemokine up-regulation in SARS-coronavirus-infected, monocyte-derived human dendritic cells, Blood. 106 (7) (2005) 2366–2374.

[37] R. Matsuyama, H. Nishiura, S. Kutsuna, K. Hayakawa, N. Ohmagari, Clinical determinants of the severity of Middle East respiratory syndrome (MERS): a systematic review and meta-analysis, BMC Public Health 16 (1) (2016) 1203.

[38] R. Channappanavar, S. Perlman, Pathogenic human coronavirus infections: causes and consequences of cytokine storm and immunopathology, Semin. Immunopathol. 39 (5) (2017) 529–539.

[39] C. Clay, N. Donart, N. Fomukong, J.B. Knight, W. Lei, L. Price, et al., Primary severe acute respiratory syndrome coronavirus infection limits replication but not lung inflammation upon homologous rechallenge, J. Virol. 86 (8) (2012) 4234–4244.

[40] J.S. Peiris, C.M. Chu, V.C. Cheng, K.S. Chan, I.F. Hung, L.L. Poon, et al., Clinical progression and viral load in a community outbreak of coronavirus-associated SARS pneumonia: a prospective study, Lancet. 361 (9371) (2003) 1767–1772.

[41] W.K. Wang, S.Y. Chen, I.J. Liu, C.L. Kao, H.L. Chen, B.L. Chiang, et al., Temporal relationship of viral load, ribavirin, interleukin (IL)-6, IL-8, and clinical progression in patients with severe acute respiratory syndrome, Clin. Infect. Dis. 39 (7) (2004) 1071–1075.

[42] X. Yang, Y. Yu, J. Xu, H. Shu, J. Xia, H. Liu, et al., Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study, Lancet Respir. Med. (2020), https://doi.org/10.1016/S2213-2600(20)30079-5 pii: S2213-2600(20)30079-5. [Epub ahead of print].

[43] Wang D, Hu B, Hu C, Zhu F, Liu X, Zhang J, et al. Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China. JAMA. 2020.

[44] W.J. Guan, Z.Y. Ni, Y. Hu, W.H. Liang, C.Q. Ou, J.X. He, et al., Clinical Characteristics of Coronavirus Disease 2019 in China, N. Engl. J. Med. (2020), https://doi.org/10.1056/NEJMoa2002032 [Epub ahead of print].

[45] N. Chen, M. Zhou, X. Dong, J. Qu, F. Gong, Y. Han, et al., Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study, Lancet. 395 (10223) (2020) 507–513.

[46] Z. Wu, J.M. McGoggan, Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China: Summary of a Report of 72314 Cases From the Chinese Center for Disease Control and Prevention, JAMA.

S. Felsenstein, et al.

(2020), https://doi.org/10.1001/jama.2020.2648 [Epub ahead of print].

[47] S. Tian, Y. Xiong, H. Liu, L. Niu, J. Guo, M. Liao, et al., Pathological study of the 2019 novel coronavirus disease (COVID-19) through postmortem core biopsies, Mod. Pathol. (2020), https://doi.org/10.1038/s41379-020-0536-x [Epub ahead of print].

[48] L.M. Barton, E.J. Duval, E. Stroberg, S. Ghosh, S. Mukhopadhyay, COVID-19 Autopsies, Oklahoma, USA. Am J Clin Pathol, 2020.

[49] S. Shi, M. Qin, B. Shen, Y. Cai, T. Liu, F. Yang, et al., Association of Cardiac Injury With Mortality in Hospitalized Patients With COVID-19 in Wuhan, China, JAMA Cardiol, 2020.

[50] T. Guo, Y. Fan, M. Chen, X. Wu, L. Zhang, T. He, et al., Cardiovascular Implications of Fatal Outcomes of Patients With Coronavirus Disease 2019 (COVID-19), JAMA Cardiol. (2020), https://doi.org/10.1001/jamacardio.2020.1017 [Epub ahead of print].

[51] Q. Ruan, K. Yang, W. Wang, L. Jiang, J. Song, Clinical predictors of mortality due to COVID-19 based on an analysis of data of 150 patients from Wuhan, China, Intensive Care Med, 2020.

[52] Zhang J, Liu P, Wang M, Wang J, Chen J, Yuan W, et al. The clinical data from 19 critically ill patients with coronavirus disease 2019: a single-centered, retrospective, observational study. Z Gesundh Wiss. 2020:1-4.

[53] W. Zhang, Y. Zhao, F. Zhang, Q. Wang, T. Li, Z. Liu, et al., The use of anti-inflammatory drugs in the treatment of people with severe coronavirus disease 2019 (COVID-19): The Perspectives of clinical immunologists from China, Clin. Immunol. 214 (2020) 108393.

[54] J.F. Drexler, F. Gloza-Rausch, J. Glende, V.M. Corman, D. Muth, M. Goettsche, et al., Genomic characterization of severe acute respiratory syndrome-related coronavirus in European bats and classification of coronaviruses based on partial RNA-dependent RNA polymerase gene sequences, J. Virol. 84 (21) (2010) 11336–11349.

[55] J.F. Drexler, V.M. Corman, C. Drosten, Ecology, evolution and classification of bat coronaviruses in the aftermath of SARS, Antivir. Res. 101 (2014) 45–56.

[56] A.R. Fehr, S. Perlman, Coronaviruses: an overview of their replication and pathogenesis, Methods Mol. Biol. 1282 (2015) 1–23.

[57] S.J. Anthony, C.K. Johnson, D.J. Greig, S. Kramer, X. Che, H. Wells, et al., Global patterns in coronavirus diversity, Virus Evol. 3 (1) (2017) vex012.

[58] C. Kreuder Johnson, P.L. Hitchens, T. Smiley Evans, T. Goldstein, K. Thomas, A. Clements, et al., Spillover and pandemic properties of zoonotic viruses with high host plasticity, Sci. Rep. 5 (2015) 14830.

[59] V.M. Corman, H.J. Baldwin, A.F. Tateno, R.M. Zerbinati, A. Annan, M. Owusu, et al., Evidence for an Ancestral Association of Human Coronavirus 229E with Bats, J. Virol. 89 (23) (2015) 11858–11870.

[60] J.S. Sabir, T.T. Lam, M.M. Ahmed, L. Li, Y. Shen, S.E. Abo-Aba, et al., Co-circulation of three camel coronavirus species and recombination of MERS-CoVs in Saudi Arabia, Science. 351 (6268) (2016) 81–84.

[61] X.Y. Ge, J.L. Li, X.L. Yang, A.A. Chmura, G. Zhu, J.H. Epstein, et al., Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor, Nature. 503 (7477) (2013) 535–538.

[62] S. Su, G. Wong, W. Shi, J. Liu, A.C.K. Lai, J. Zhou, et al., Epidemiology, Genetic Recombination, and Pathogenesis of Coronaviruses, Trends Microbiol. 24 (6) (2016) 490–502.

[63] J. Cui, F. Li, Z.L. Shi, Origin and evolution of pathogenic coronaviruses, Nat. Rev. Microbiol. 17 (3) (2019) 181–192.

[64] R.J. Hulswit, C.A. de Haan, B.J. Bosch, Coronavirus Spike Protein and Tropism Changes, Adv. Virus Res. 96 (2016) 29–57.

[65] M.A. Tortorici, D. Veesler, Structural insights into coronavirus entry, Adv. Virus Res. 105 (2019) 93–116.

[66] M.A. Tortorici, A.C. Walls, Y. Lang, C. Wang, Z. Li, D. Koerhuis, et al., Structural basis for human coronavirus attachment to sialic acid receptors, Nat. Struct. Mol. Biol. 26 (6) (2019) 481–489.

[67] A.C. Walls, X. Xiong, Y.J. Park, M.A. Tortorici, J. Snijder, J. Quispe, et al., Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion, Cell. 176 (5) (2019) 1026–1039 e15.

[68] X. Ou, Y. Liu, X. Lei, P. Li, D. Mi, L. Ren, et al., Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV, Nat. Commun. 11 (1) (2020) 1620.

[69] B.J. Bosch, S.L. Smits, B.L. Haagmans, Membrane ectopeptidases targeted by human coronaviruses, Curr Opin Virol. 6 (2014) 55–60.

[70] G. Lu, Q. Wang, G.F. Gao, Bat-to-human: spike features determining ‘host jump’ of coronaviruses SARS-CoV, MERS-CoV, and beyond, Trends Microbiol. 23 (8) (2015) 468–478.

[71] H. Kleine-Weber, S. Pohlmann, M. Hoffmann, Spike proteins of novel MERS-coronavirus isolates from North- and West-African dromedary camels mediate robust viral entry into human target cells, Virology. 535 (2019) 261–265.

[72] A. Klausegger, B. Strobl, G. Regl, A. Kaser, W. Luytjes, R. Vlasak, Identification of a coronavirus hemagglutinin-esterase with a substrate specificity different from those of influenza C virus and bovine coronavirus, J. Virol. 73 (5) (1999) 3737–3743.

[73] J.F. Chan, S. Yuan, K.H. Kok, To KK, H. Chu, J. Yang, et al., A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster, Lancet. 395 (10223) (2020) 514–523.

[74] C. Huang, Y. Wang, X. Li, L. Ren, J. Zhao, Y. Hu, et al., Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China, Lancet. 395 (10223) (2020) 497–506.

[75] R. Lu, X. Zhao, J. Li, P. Niu, B. Yang, H. Wu, et al., Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding, Lancet. 395 (10224) (2020) 565–574.

[76] E. Promptchara, C. Ketloy, T. Palaga, Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic, Asian Pac. J. Allergy Immunol. 38 (1) (2020) 1–9.


[77] P. Zhou, X.L. Yang, X.G. Wang, B. Hu, L. Zhang, W. Zhang, et al., A pneumonia outbreak associated with a new coronavirus of probable bat origin, Nature. 579 (7798) (2020) 270–273.

[78] I. Hamming, W. Timens, M.L. Bulthuis, A.T. Lely, G. Navis, H. van Goor, Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis, J. Pathol. 203 (2) (2004) 631–637.

[79] A.C. Sims, R.S. Baric, B. Yount, S.E. Burkett, P.L. Collins, R.J. Pickles, Severe acute respiratory syndrome coronavirus infection of human ciliated airway epithelia: role of ciliated cells in viral spread in the conducting airways of the lungs, J. Virol. 79 (24) (2005) 15511–15524.

[80] W.H. Sungnak, C. Bécavin, M. Berg, HCA Lung Biological Network. SARS-CoV-2 Entry Genes Are Most Highly Expressed in Nasal Goblet and Ciliated Cells within Human Airways, arXiv:200306122 (2020).

[81] H. Xu, L. Zhong, J. Deng, J. Peng, H. Dan, X. Zeng, et al., High expression of ACE2 receptor of 2019-nCoV on the epithelial cells of oral mucosa, Int J Oral Sci. 12 (1) (2020) 8.

[82] N. Zhu, D. Zhang, W. Wang, X. Li, B. Yang, J. Song, et al., A Novel Coronavirus from Patients with Pneumonia in China, 2019, N. Engl. J. Med. 382 (8) (2020) 727–733.

[83] S. Perlman, A.A. Dandekar, Immunopathogenesis of coronavirus infections: implications for SARS, Nat. Rev. Immunol. 5 (12) (2005) 917–927.

[84] A. Ben Addi, A. Lefort, X. Hua, F. Libert, D. Communi, C. Ledent, et al., Modulation of murine dendritic cell function by adenine nucleotides and adenosine: involvement of the A(2B) receptor, Eur. J. Immunol. 38 (6) (2008) 1610–1620.

[85] H.M. Lazear, J.W. Schoggins, M.S. Diamond, Shared and Distinct Functions of Type I and Type III Interferons, Immunity. 50 (4) (2019) 907–923.

[86] E. de Wit, N. van Doremalen, D. Falzarano, V.J. Munster, SARS and MERS: recent insights into emerging coronaviruses, Nat. Rev. Microbiol. 14 (8) (2016) 523–534.

[87] A. Alunno, I. Padjen, A. Fanouriakis, D.T. Boumpas, Pathogenic and therapeutic relevance of JAK/STAT signaling in systemic lupus erythematosus: integration of distinct inflammatory pathways and the prospect of their inhibition with an oral agent, Cells 8 (8) (2019).

[88] E. Kindler, V. Thiel, F. Weber, Interaction of SARS and MERS Coronaviruses with the Antiviral Interferon Response, Adv. Virus Res. 96 (2016) 219–243.

[89] X. Lu, J. Pan, J. Tao, D. Guo, SARS-CoV nucleocapsid protein antagonizes IFN-beta response by targeting initial step of IFN-beta induction pathway, and its C-terminal region is critical for the antagonism, Virus Genes 42 (1) (2011) 37–45.

[90] Y. Yi, P.N.P. Lagniton, S. Ye, E. Li, R.H. Xu, COVID-19: what has been learned and to be learned about the novel coronavirus disease, Int. J. Biol. Sci. 16 (10) (2020) 1753–1766.

[91] R.Q. Cron, W.W. Chatham, The Rheumatologist’s Role in Covid-19, J. Rheumatol. (2020), https://doi.org/10.3899/jrheum.200334 pii: jrheum.200334 [Epub ahead of print].

[92] Y. Du, L. Tu, P. Zhu, M. Mu, R. Wang, P. Yang, et al., Clinical Features of 85 Fatal Cases of COVID-19 from Wuhan: A Retrospective Observational Study, Am. J. Respir. Crit. Care Med. (2020), https://doi.org/10.1164/rccm.202003-0543OC [Epub ahead of print].

[93] A. Zumla, D.S. Hui, S. Perlman, Middle East respiratory syndrome, Lancet. 386 (9997) (2015) 995–1007.

[94] W.H. Mahallawi, Khabour OF, Q. Zhang, H.M. Makhdoum, B.A. Suliman, MERS-CoV infection in humans is associated with a pro-inflammatory Th1 and Th17 cytokine profile, Cytokine. 104 (2018) 8–13.

[95] J.M. Nicholls, L.L. Poon, K.C. Lee, W.F. Ng, S.T. Lai, C.Y. Leung, et al., Lung pathology of fatal severe acute respiratory syndrome, Lancet. 361 (9371) (2003) 1773–1778.

[96] C.K. Wong, C.W. Lam, A.K. Wu, W.K. Ip, N.L. Lee, I.H. Chan, et al., Plasma inflammatory cytokines and chemokines in severe acute respiratory syndrome, Clin. Exp. Immunol. 136 (1) (2004) 95–103.

[97] G.K. Atkin-Smith, M. Duan, W. Chen, I.K.H. Poon, The induction and consequences of Influenza A virus-induced cell death, Cell Death Dis. 9 (10) (2018) 1002.

[98] Y. Fu, Y. Cheng, Y. Wu, Understanding SARS-CoV-2-Mediated Inflammatory Responses: From Mechanisms to Potential Therapeutic Tools, Virol. Sin. (2020), https://doi.org/10.1007/s12250-020-00207-4 [Epub ahead of print].

[99] A. Takada, Y. Kawaoka, Antibody-dependent enhancement of viral infection: molecular mechanisms and in vivo implications, Rev. Med. Virol. 13 (6) (2003) 387–398.

[100] Y. Jin, H. Yang, W. Ji, W. Wu, S. Chen, W. Zhang, et al., Virology, Epidemiology, Pathogenesis, and Control of COVID-19, Viruses 12 (4) (2020).

[101] Z. Xu, L. Shi, Y. Wang, J. Zhang, L. Huang, C. Zhang, et al., Pathological findings of COVID-19 associated with acute respiratory distress syndrome, Lancet Respir. Med. 8 (4) (2020) 420–422.

[102] X.H. Yao, T.Y. Li, Z.C. He, Y.F. Ping, H.W. Liu, S.C. Yu, et al., A pathological report of three COVID-19 cases by minimally invasive autopsies, Zhonghua Bing Li Xue Za Zhi 49 (0) (2020) E009.

[103] A. Schnabel, C.M. Hedrich, Childhood Vasculitis, Front. Pediatr. 6 (2018) 421.

[104] X.J. Guo, P.G. Thomas, New fronts emerge in the influenza cytokine storm, Semin. Immunopathol. 39 (5) (2017) 541–550.

[105] A. Shimabukuro-Vornhagen, P. Godel, M. Subklewe, H.J. Stemmler, H.A. Schlosser, M. Schlaak, et al., Cytokine release syndrome, J Immunother Cancer. 6 (1) (2018) 56.

[106] C. Qin, L. Zhou, Z. Hu, S. Zhang, S. Yang, Y. Tao, et al., Dysregulation of immune response in patients with COVID-19 in Wuhan, China. Clin Infect Dis. (2020), https://doi.org/10.1093/cid/ciaa248 pii: ciaa248 [Epub ahead of print].

[107] G. Chen, D. Wu, W. Guo, Y. Cao, D. Huang, H. Wang, et al., Clinical and immunological features of severe and moderate coronavirus disease 2019, J. Clin. Invest. (2020), https://doi.org/10.1172/JCI137244 pii: 137244 [Epub ahead of print].

[108] B.A. Croker, J.A. O’Donnell, M. Gerlic, Pyroptotic death storms and cytopenia, Curr. Opin. Immunol. 26 (2014) 128–137.

[109] G.N. Usmani, B.A. Woda, P.E. Newburger, Advances in understanding the

S. Felsenstein, et al.

pathogenesis of HLH, Br. J. Haematol. 161 (5) (2013) 609–622.
[110] X. Gao, H. Zhou, C. Wu, Y. Xiao, L. Ren, G. Paranhos-Baccala, et al., Antibody against nucleocapsid protein predicts susceptibility to human coronavirus infection, J. Inf. Secur. 71 (5) (2015) 599–602.
[111] X.Y. Che, L.W. Qiu, Z.Y. Liao, Y.D. Wang, K. Wen, Y.X. Pan, et al., Antigenic cross-reactivity between severe acute respiratory syndrome-associated coronavirus and human coronaviruses 229E and OC43, J. Infect. Dis. 191 (12) (2005) 2033–2037.
[112] A. Roberts, E.W. Lamirande, L. Vogel, J.P. Jackson, C.D. Paddock, J. Guarner, et al., Animal models and vaccines for SARS-CoV infection, Virus Res. 133 (1) (2008) 20–32.
[113] J. Flipse, M.A. Diosa-Toro, T.E. Hoornweg, D.P. van de Pol, S. Urcuqui-Inchima, J.M. Smit, Antibody-Dependent Enhancement of Dengue Virus Infection in Primary Human Macrophages, Balancing Higher Fusion against Antiviral Responses. Sci Rep. 6 (2016) 29201.
[114] C.Y. Cheung, L.L. Poon, I.H. Ng, W. Luk, S.F. Sia, M.H. Wu, et al., Cytokine responses in severe acute respiratory syndrome coronavirus-infected macrophages in vitro: possible relevance to pathogenesis, J. Virol. 79 (12) (2005) 7819–7826.
[115] L.C.J. de Bree, V. Koeken, L.A.B. Joosten, P. Aaby, C.S. Benn, R. van Crevel, et al., Non-specific effects of vaccines: Current evidence and potential implications, Semin. Immunol. 39 (2018) 35–43.
[116] H.S. Goodridge, S.S. Ahmed, N. Curtis, T.R. Kollmann, O. Levy, M.G. Netea, et al., Harnessing the beneficial heterologous effects of vaccination, Nat. Rev. Immunol. 16 (6) (2016) 392–400.
[117] C.M. Hedrich, COVID-19 - Considerations for the paediatric rheumatologist, Clin. Immunol. 214 (2020) 108420.
[118] J.J.Q.X. Chen, K. Liu, Z. Yu, W. Tao, W. Gong, J.D.J. Han, Individual Variation of the SARS-CoV2 Receptor ACE2 Gene Expression and Regulation. Individual Variation of the SARS-CoV2 Receptor ACE2 Gene Expression and Regulation, https://www.preprints.org/manuscript/202003.0191/v1, (2020).
[119] Simoes E. Silva AC, K.D. Silveira, A.J. Ferreira, M.M. Teixeira, ACE2, angiotensin-(1-7) and Mas receptor axis in inflammation and fibrosis, Br. J. Pharmacol. 169 (3) (2013) 477–492.
[120] A. Savarino, J.R. Boelaert, A. Cassone, G. Majori, R. Cauda, Effects of chloroquine on viral infections: an old drug against today’s diseases? Lancet Infect. Dis. 3 (11) (2003) 722–727.
[121] Y. Yan, Z. Zou, Y. Sun, X. Li, K.F. Xu, Y. Wei, et al., Anti-malaria drug chloroquine is highly effective in treating avian influenza A H5N1 virus infection in an animal model, Cell Res. 23 (2) (2013) 300–302.
[122] E. Keyaerts, L. Vijgen, P. Maes, J. Neyts, M. Van Ranst, In vitro inhibition of severe acute respiratory syndrome coronavirus by chloroquine, Biochem. Biophys. Res. Commun. 323 (1) (2004) 264–268.
[123] M.J. Vincent, E. Bergeron, S. Benjamnet, B.R. Erickson, P.E. Rollin, T.G. Ksiazek, et al., Chloroquine is a potent inhibitor of SARS coronavirus infection and spread, Virol. J. 2 (2005) 69.
[124] M. Wang, R. Cao, L. Zhang, X. Yang, J. Liu, M. Xu, et al., Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro, Cell Res. 30 (3) (2020) 269–271.
[125] J. Gao, Z. Tian, X. Yang, Breakthrough: Chloroquine phosphate has shown apparent efficacy in treatment of COVID-19 associated pneumonia in clinical studies, Biosci Trends. 14 (1) (2020) 72–73.
[126] P. Gautret, J.C. Lagier, P. Parola, V.T. Hoang, L. Meddeb, M. Mailhe, et al., Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open-label non-randomized clinical trial, Int. J. Antimicrob. Agents 105949 (2020).
[127] J.M. Molina, C. Delaugerre, J. Le Goff, B. Mela-Lima, D. Ponscarme, L. Goldwirt, et al., No evidence of rapid antiviral clearance or clinical benefit with the combination of hydroxychloroquine and azithromycin in patients with severe COVID-19 infection, Med. Mal. Infect. (2020), https://doi.org/10.1016/j.medmal.2020.03.006 pii: S0399-077X(20)30085-8. [Epub ahead of print].
[128] J.N. Magagnoli, F. Pereira, T. Cummings, J.W. Hardin, S.S. Sutton, J. Ambati, Outcomes of Hydroxychloroquine usage in United States veterans hospitalized with COVID-19, medRciv Server (2020), https://doi.org/10.1101/2020.04.16.20065920.
[129] Silva Borba MGFAV, V. Sousa Sampaio, M. Almeida Araújo Alexandre, G. Cardoso Melo, M. Brito, M.P. Gomes Mourão, J.D. Brito-Sousa, D. Baía-da-Silva, Chloroquine diphosphate in two different dosages as adjunctive therapy of hospitalized patients with severe respiratory syndrome in the context of coronavirus (SARS-CoV-2) infection: Preliminary safety results of a randomized, double-blinded, phase IIb clinical trial (CloroCovid-19 Study), medRxiv Preprint (2020), https://www.medrxiv.org/content/10.1101/2020.04.07.20056424v1.full.pdf.
[130] D. Zhou, S.M. Dai, Q. Tong, COVID-19: a recommendation to examine the effect of hydroxychloroquine in preventing infection and progression, J. Antimicrob. Chemother. (2020), https://doi.org/10.1093/jac/dkaa114 pii: dkaa114. [Epub ahead of print].
[131] H.K. Ziegler, E.R. Unanue, Decrease in macrophage antigen catabolism caused by ammonia and chloroquine is associated with inhibition of antigen presentation to T cells, Proc. Natl. Acad. Sci. U. S. A. 79 (1) (1982) 175–178.
[132] I. Ben-Zvi, S. Kivity, P. Langevitz, Y. Shoenfeld, Hydroxychloroquine: from malaria to autoimmunity, Clin. Rev. Allergy Immunol. 42 (2) (2012) 145–153.
[133] R. Nosal, V. Jancinova, M. Petrikova, Chloroquine inhibits stimulated platelets at the arachidonic acid pathway, Thromb. Res. 77 (6) (1995) 531–542.
[134] N. Tang, H. Bai, X. Chen, J. Gong, D. Li, Z. Sun, Anticoagulant treatment is associated with decreased mortality in severe coronavirus disease 2019 patients with coagulopathy, J. Thromb. Haemost. (2020), https://doi.org/10.1111/jth.14817 [Epub ahead of print].
[135] C. Chatre, F. Roubille, H. Vernhet, C. Jorgensen, Y.M. Pers, Cardiac Complications Attributed to Chloroquine and Hydroxychloroquine: A Systematic Review of the Literature, Drug Saf. 41 (10) (2018) 919–931.
[136] N. Costedoat-Chalumeau, B. Dunogue, G. Leroux, N. Morel, M. Jallouli, V. Le Guern, et al., A Critical Review of the Effects of Hydroxychloroquine and

Chloroquine on the Eye, Clin. Rev. Allergy Immunol. 49 (3) (2015) 317–326.
[137] P. Gautret, J.C. Lagier, P. Parola, V.T. Hoang, L. Meddeb, J. Sevestre, et al., Clinical and microbiological effect of a combination of hydroxychloroquine and azithromycin in 80 COVID-19 patients with at least a six-day follow up: A pilot observational study, Travel Med. Infect. Dis. 101663 (2020).
[138] J.L.B. Andreania, I. Duflota, P. Jardota, C. Rollanda, M. Boxberger, J.Y. Bou Khalila, J.P. Baudouin, N. Wurtza, vitro testing of Hydroxychloroquine and Azithromycin on SARS-CoV-2 shows synergistic effect. Mediterranean Infection, https://www.mediterranee-infection.com/wp-content/uploads/2020/03/La-Scola-et-al-V1.pdf, (2020).
[139] D. Haydar, T.J. Cory, S.E. Birket, B.S. Murphy, K.R. Pennypacker, A.P. Sinai, et al., Azithromycin Polarizes Macrophages to an M2 Phenotype via Inhibition of the STAT1 and NF-kappaB Signaling Pathways, J. Immunol. 203 (4) (2019) 1021–1030.
[140] J.C. Gensel, T.J. Kopper, B. Zhang, M.B. Orr, W.M. Bailey, Predictive screening of M1 and M2 macrophages reveals the immunomodulatory effectiveness of post spinal cord injury azithromycin treatment, Sci. Rep. 7 (2017) 40144.
[141] A.J. Walkey, R.S. Wiener, Macrolide antibiotics and survival in patients with acute lung injury, Chest. 141 (5) (2012) 1153–1159.
[142] K. Kawamura, K. Ichikado, M. Suga, M. Yoshioka, Efficacy of azithromycin for treatment of acute exacerbation of chronic fibrosing interstitial pneumonia: a prospective, open-label study with historical controls, Respiration. 87 (6) (2014) 478–484.
[143] K. Kawamura, K. Ichikado, M. Takaki, Y. Eguchi, K. Anan, M. Suga, Adjunctive therapy with azithromycin for moderate and severe acute respiratory distress syndrome: a retrospective, propensity score-matching analysis of prospectively collected data at a single center, Int. J. Antimicrob. Agents 51 (6) (2018) 918–924.
[144] M. Garcia-Cremades, B.P. Solans, E. Hughes, J.P. Ernest, E. Wallender, F. Aweeka, et al., Optimizing hydroxychloroquine dosing for patients with COVID-19: An integrative modeling approach for effective drug repurposing, Clin. Pharmacol. Ther. (2020), https://doi.org/10.1002/cpt.1856 [Epub ahead of print].
[145] J.L. Sapp, W. Alqarawi, C.J. MacIntyre, R. Tadros, C. Steinberg, J.D. Roberts, et al., Guidance On Minimizing Risk of Drug-Induced Ventricular Arrhythmia During Treatment of COVID-19: A Statement from the Canadian Heart Rhythm Society, Can J Cardiol. (2020), https://doi.org/10.1016/j.cjca.2020.04.003 pii: S0828-282X(20)30325-1. [Epub ahead of print].
[146] D. Siegel, H.C. Hui, E. Doerffler, M.O. Clarke, K. Chun, L. Zhang, et al., Discovery and Synthesis of a Phosphoramidate Prodrug of a Pyrrolo[2,1-f][triazin-4-amino] Adenine G-Nucleoside (GS-5734) for the Treatment of Ebola and Emerging Viruses, J. Med. Chem. 60 (5) (2017) 1648–1661.
[147] M.L. Agostini, E.I. Andres, A.C. Sims, R.L. Graham, T.P. Sheahan, X. Lu, et al., Coronavirus susceptibility to the antiviral remdesivir (GS-5734) Is mediated by the viral polymerase and the proofreading exoribonuclease, mBio 9 (2) (2018).
[148] T.P. Sheahan, A.C. Sims, S.R. Leist, A. Schafer, J. Won, A.J. Brown, et al., Comparative therapeutic efficacy of remdesivir and combination lopinavir, ritonavir, and interferon beta against MERS-CoV, Nat. Commun. 11 (1) (2020) 222.
[149] C.J. Gordon, E.P. Tchesnokov, J.Y. Feng, D.P. Porter, M. Gotte, The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus, J. Biol. Chem. 295 (15) (2020) 4773–4779.
[150] E.P. Tchesnokov, J.Y. Feng, D.P. Porter, M. Gotte, Mechanism of inhibition of ebola virus RNA-dependent rna polymerase by remdesivir, Viruses 11 (4) (2019).
[151] E. Hillaker, J.J. Belfer, A. Bondici, H. Murad, L.E. Dumkow, Delayed Initiation of Remdesivir in a COVID-19 Positive Patient, Pharmacotherapy. (2020), https://doi.org/10.1002/phar.2403 [Epub ahead of print].
[152] M.L. Holshue, C. DeBolt, S. Lindquist, K.H. Lofy, J. Wiesman, H. Bruce, et al., First Case of 2019 Novel Coronavirus in the United States, N. Engl. J. Med. 382 (10) (2020) 929–936.
[153] J. Grein, N. Ohmagari, D. Shin, G. Diaz, E. Asperges, A. Castagna, et al., Compassionate Use of Remdesivir for Patients with Severe Covid-19, N. Engl. J. Med. (2020), https://doi.org/10.1056/NEJMoa2007016 [Epub ahead of print].
[154] A. Chandwani, J. Shuter, Lopinavir/ritonavir in the treatment of HIV-1 infection: a review, Ther. Clin. Risk Manag. 4 (5) (2008) 1023–1033.
[155] C.M. Chu, V.C. Cheng, I.F. Hung, M.M. Wong, K.H. Chan, K.S. Chan, et al., Role of lopinavir/ritonavir in the treatment of SARS: initial virological and clinical findings, Thorax. 59 (3) (2004) 252–256.
[156] K.S. Chan, S.T. Lai, C.M. Chu, E. Tsui, C.Y. Tam, M.M. Wong, et al., Treatment of severe acute respiratory syndrome with lopinavir/ritonavir: a multicentre retrospective matched cohort study, Hong Kong Med J. 9 (6) (2003) 399–406.
[157] B. Cao, Y. Wang, D. Wen, W. Liu, J. Wang, G. Fan, et al., A Trial of Lopinavir-Ritonavir in Adults Hospitalized with Severe Covid-19, N. Engl. J. Med. (2020), https://doi.org/10.1056/NEJMoa2001283 [Epub ahead of print].
[158] Z.A. Abassi, K. Skorecki, S.N. Heyman, S. Kinaneh, Z. Armaly, Covid-19 infection and mortality - A physiologist’s perspective enlightening clinical features and plausible interventional strategies, Am. J. Phys. Lung Cell. Mol. Phys. (2020), https://doi.org/10.1152/ajplung.00097.2020 [Epub ahead of print].
[159] H. Jakovac, COVID-19 - is the ACE2 just a foe? Am J Physiol Lung Cell Mol Physiol, (2020).
[160] Y. Li, Y. Cao, Z. Zeng, M. Liang, Y. Xue, C. Xi, et al., Angiotensin-converting enzyme 2/angiotensin-(1-7)/Mas axis prevents lipopolysaccharide-induced apoptosis of pulmonary microvascular endothelial cells by inhibiting JNK/NF-kappaB pathways, Sci. Rep. 5 (2015) 8209.
[161] R. Ye, Z. Liu, ACE2 exhibits protective effects against LPS-induced acute lung injury in mice by inhibiting the LPS-TLR4 pathway, Exp. Mol. Pathol. 113 (2020) 104350.
[162] Y. Imai, K. Kuba, S. Rao, Y. Huan, F. Guo, B. Guan, et al., Angiotensin-converting enzyme 2 protects from severe acute lung failure, Nature. 436 (7047) (2005) 112–116.
[163] R.P. Marshall, P. Gohlke, R.C. Chambers, D.C. Howell, S.E. Bottoms, T. Unger, et al., Angiotensin II and the fibroproliferative response to acute lung injury, Am.

S. Felsenstein, et al.

J. Phys. Lung Cell. Mol. Phys. 286 (1) (2004) L156–L164.
[164] Y. Wu, Compensation of ACE2 Function for Possible Clinical Management of 2019-nCoV-Induced Acute Lung Injury, Virol. Sin. 286 (1) (2020) L156–L164 Epub 2003 May 16.
[165] D. Batlle, J. Wysocki, K. Satchell, Soluble angiotensin-converting enzyme 2: a potential approach for coronavirus infection therapy? Clin. Sci. (Lond.) 134 (5) (2020) 543–545.
[166] D. Harmer, M. Gilbert, R. Borman, K.L. Clark, Quantitative mRNA expression profiling of ACE 2, a novel homologue of angiotensin converting enzyme, FEBS Lett. 532 (1–2) (2002) 107–110.
[167] W. Wang, Y. Xu, R. Gao, R. Lu, K. Han, G. Wu, et al., Detection of SARS-CoV-2 in Different Types of Clinical Specimens, JAMA. (2020), https://doi.org/10.1001/jama.2020.3786 [Epub ahead of print].
[168] F. Xiao, M. Tang, X. Zheng, Y. Liu, X. Li, H. Shan, Evidence for gastrointestinal infection of SARS-CoV-2, Gastroenterology. (2020), https://doi.org/10.1053/j.gastro.2020.02.055 pii: S0016-5085(20)30282-1. [Epub ahead of print].
[169] A. Khan, C. Benthin, B. Zeno, T.E. Albertson, J. Boyd, J.D. Christie, et al., A pilot clinical trial of recombinant human angiotensin-converting enzyme 2 in acute respiratory distress syndrome, Crit. Care 21 (1) (2017) 234.
[170] X. Wang, W. Xu, G. Hu, S. Xia, Z. Sun, Z. Liu, et al., SARS-CoV-2 infects T lymphocytes through its spike protein-mediated membrane fusion, Cell. Mol. Immunol. (2020), https://doi.org/10.1038/s41423-020-0424-9 [Epub ahead of print].
[171] Y.M. Arabi, S. Shalhoub, Y. Mandourah, F. Al-Hameed, A. Al-Omari, E. Al Qasim, et al., Ribavirin and Interferon Therapy for Critically Ill Patients With Middle East Respiratory Syndrome: A Multicenter Observational Study, Clin. Infect. Dis. 70 (9) (2020) 1837–1844.
[172] M.E. Morra, L. Van Thanh, M.G. Kamel, A.A. Ghazy, A.M.A. Altibi, L.M. Dat, et al., Clinical outcomes of current medical approaches for Middle East respiratory syndrome: A systematic review and meta-analysis, Rev. Med. Virol. 28 (3) (2018) e1977.
[173] R. Channappanavar, A.R. Fehr, J. Zheng, C. Wohlford-Lenane, J.E. Abrahante, M. Mack, et al., IFN-I response timing relative to virus replication determines MERS coronavirus infection outcomes, J. Clin. Invest. 130 (2019) 3625–3639.
[174] E.M. Bloch, S. Shoham, A. Casadevall, B.S. Sachais, B. Shaz, J.L. Winters, et al., Deployment of convalescent plasma for the prevention and treatment of COVID-19, J. Clin. Invest. (2020), https://doi.org/10.1172/JCI138745 pii: 138745. [Epub ahead of print].
[175] T.C. Luke, A. Casadevall, S.J. Watowich, S.L. Hoffman, J.H. Beigel, T.H. Burgess, Hark back: passive immunotherapy for influenza and other serious infections, Crit. Care Med. 38 (4 Suppl) (2010) e66–e73.
[176] J. Mair-Jenkins, M. Saavedra-Campos, J.K. Baillie, P. Cleary, F.M. Khaw, W.S. Lim, et al., The effectiveness of convalescent plasma and hyperimmune immunoglobulin for the treatment of severe acute respiratory infections of viral etiology: a systematic review and exploratory meta-analysis, J. Infect. Dis. 211 (1) (2015) 80–90.
[177] A. Casadevall, L.A. Pirofski, The convalescent sera option for containing COVID-19, J. Clin. Invest. 130 (4) (2020) 1545–1548.
[178] I.F. Hung, To KK, C.K. Lee, K.L. Lee, K. Chan, W.W. Yan, et al., Convalescent plasma treatment reduced mortality in patients with severe pandemic influenza A (H1N1) 2009 virus infection, Clin. Infect. Dis. 52 (4) (2011) 447–456.
[179] R.T. Davey Jr., E. Fernandez-Cruz, N. Markowitz, S. Pett, A.G. Babiker, D. Wentworth, et al., Anti-influenza hyperimmune intravenous immunoglobulin for adults with influenza A or B infection (FLU-IVIG): a double-blind, randomised, placebo-controlled trial, Lancet Respir. Med. 7 (11) (2019) 951–963.
[180] J. van Griensven, T. Edwards, X. de Lamballerie, M.G. Semple, P. Gallian, S. Baize, et al., Evaluation of Convalescent Plasma for Ebola Virus Disease in Guinea, N. Engl. J. Med. 374 (1) (2016) 33–42.
[181] J.H. Beigel, H.H. Nam, P.L. Adams, A. Krafft, W.L. Ince, S.S. El-Kamary, et al., Advances in respiratory virus therapeutics - A meeting report from the 6th isirv Antiviral Group conference, Antivir. Res. 167 (2019) 45–67.
[182] Y.M. Arabi, A.H. Hajeer, T. Luke, K. Raviprakash, H. Balkhy, S. Johani, et al., Feasibility of Using Convalescent Plasma Immunotherapy for MERS-CoV Infection, Saudi Arabia. Emerg Infect Dis. 22 (9) (2016) 1554–1561.
[183] A.B. Kawiecki, R.C. Christofferson, Zika Virus-Induced Antibody Response Enhances Dengue Virus Serotype 2 Replication In Vitro, J. Infect. Dis. 214 (9) (2016) 1357–1360.
[184] A. Seguin, L. Galicier, D. Boutboul, V. Lemiale, E. Azoulay, Pulmonary Involvement in Patients With Hemophagocytic Lymphohistiocytosis, Chest. 149 (5) (2016) 1294–1301.
[185] Y. Gao, T. Li, M. Han, X. Li, D. Wu, Y. Xu, et al., Diagnostic Utility of Clinical Laboratory Data Determinations for Patients with the Severe COVID-19, J. Med. Virol. (2020), https://doi.org/10.1002/jmv.25770 [Epub ahead of print].
[186] F. Ferro, E. Elefante, C. Baldini, E. Bartoloni, I. Puxeddu, R. Talarico, et al., COVID-19: the new challenge for rheumatologists, Clin. Exp. Rheumatol. 38 (2) (2020) 175–180.


[187] J.I. Henter, A. Horne, M. Arico, R.M. Egeler, A.H. Filipovich, S. Imashuku, et al., HLH-2004: Diagnostic and therapeutic guidelines for hemophagocytic lymphohistiocytosis, Pediatr. Blood Cancer 48 (2) (2007) 124–131.
[188] G.S. Schulert, M. Zhang, N. Fall, A. Husami, D. Kissell, A. Hanosh, et al., Whole-Exome Sequencing Reveals Mutations in Genes Linked to Hemophagocytic Lymphohistiocytosis and Macrophage Activation Syndrome in Fatal Cases of H1N1 Influenza, J. Infect. Dis. 213 (7) (2016) 1180–1188.
[189] M.A. Matthay, R.L. Zemans, G.A. Zimmerman, Y.M. Arabi, J.R. Beitler, A. Mercat, et al., Acute respiratory distress syndrome, Nat Rev Dis Primers. 5 (1) (2019) 18.
[190] S.C. Chang, Clinical findings, treatment and prognosis in patients with severe acute respiratory syndrome (SARS), J Chin Med Assoc. 68 (3) (2005) 106–107.
[191] G. Rowlands, B. Tabassum, P. Campbell, S. Harvey, A. Vaittinen, L. Stobbart, et al., The evidence-based development of an intervention to improve clinical health literacy practice, Int. J. Environ. Res. Public Health 17 (5) (2020).
[192] J.F. Griffith, G.E. Antonio, S.M. Kumta, D.S. Hui, J.K. Wong, G.M. Joynt, et al., Osteonecrosis of hip and knee in patients with severe acute respiratory syndrome treated with steroids, Radiology. 235 (1) (2005) 168–175.
[193] L. Shang, J. Zhao, Y. Hu, R. Du, B. Cao, On the use of corticosteroids for 2019-nCoV pneumonia, Lancet. 395 (10225) (2020) 683–684.
[194] C.D. Russell, J.E. Millar, J.K. Baillie, Clinical evidence does not support corticosteroid treatment for 2019-nCoV lung injury, Lancet. 395 (10223) (2020) 473–475.
[195] J.H.O. Hoffmann, A.H. Enk, High-Dose Intravenous Immunoglobulin in Skin Autoimmune Disease, Front. Immunol. 10 (2019) 1090.
[196] L. Liu, Q. Wei, Q. Lin, J. Fang, H. Wang, H. Kwok, et al., Anti-spike IgG causes severe acute lung injury by skewing macrophage responses during acute SARS-CoV infection, JCI Insight 4 (4) (2019).
[197] L. Zhang, F. Zhang, W. Yu, T. He, J. Yu, C.E. Yi, et al., Antibody responses against SARS coronavirus are correlated with disease outcome of infected individuals, J. Med. Virol. 78 (1) (2006) 1–8.
[198] M.G. Prabagar, H.J. Choi, J.Y. Park, S. Loh, Y.S. Kang, Intravenous immunoglobulin-mediated immunosuppression and the development of an IVIG substitute, Clin. Exp. Med. 14 (4) (2014) 361–373.
[199] S.L. Maude, N. Frey, P.A. Shaw, R. Aplenc, D.M. Barrett, N.J. Bunin, et al., Chimeric antigen receptor T cells for sustained remissions in leukemia, N. Engl. J. Med. 371 (16) (2014) 1507–1517.
[200] X. Zhang, K. Song, F. Tong, M. Fei, H. Guo, Z. Lu, et al., First case of COVID-19 in a patient with multiple myeloma successfully treated with tocilizumab, Blood Adv. 4 (7) (2020) 1307–1310.
[201] J.M. Michot, L. Albiges, N. Chaput, V. Saada, F. Pommeret, F. Griscelli, et al., Tocilizumab, an anti-IL6 receptor antibody, to treat Covid-19-related respiratory failure: a case report, Ann. Oncol. (2020), https://doi.org/10.1016/j.annonc.2020.03.300 pii: S0923-7534(20)36387-0. [Epub ahead of print].
[202] P. Mehta, D.F. McAuley, M. Brown, E. Sanchez, R.S. Tattersall, J.J. Manson, et al., COVID-19: consider cytokine storm syndromes and immunosuppression, Lancet. 395 (10229) (2020) 1033–1034.
[203] B. Shakoor, J.A. Carcillo, W.W. Chatham, R.L. Amdur, H. Zhao, C.A. Dinarello, et al., Interleukin-1 Receptor Blockade Is Associated With Reduced Mortality in Sepsis Patients With Features of Macrophage Activation Syndrome: Reanalysis of a Prior Phase III Trial, Crit. Care Med. 44 (2) (2016) 275–281.
[204] C.M. Hedrich, N. Bruck, B. Fiebig, M. Gahr, Anakinra: a safe and effective first-line treatment in systemic onset juvenile idiopathic arthritis (SoJIA), Rheumatol. Int. 32 (11) (2012) 3525–3530.
[205] P.A. Nigrovic, M. Mannion, F.H. Prince, A. Zeft, C.E. Rabinovich, M.A. van Rossum, et al., Anakinra as first-line disease-modifying therapy in systemic juvenile idiopathic arthritis: report of forty-six patients from an international multi-center series, Arthritis Rheum. 63 (2) (2011) 545–555.
[206] H.E. Sonmez, S. Demir, Y. Bilginer, S. Ozen, Anakinra treatment in macrophage activation syndrome: a single center experience and systemic review of literature, Clin. Rheumatol. 37 (12) (2018) 3329–3335.
[207] S.U. Seo, M.N. Kweon, Virome-host interactions in intestinal health and disease, Curr Opin Virol. 37 (2019) 63–71.
[208] F. Zhou, T. Yu, R. Du, G. Fan, Y. Liu, Z. Liu, et al., Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study, Lancet. 395 (10229) (2020) 1054–1062.
[209] C.C. Lai, Y.H. Liu, C.Y. Wang, Y.H. Wang, S.C. Hsueh, M.Y. Yen, et al., Asymptomatic carrier state, acute respiratory disease, and pneumonia due to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2): Facts and myths, J Microbiol Immunol Infect. (2020), https://doi.org/10.1016/j.jmii.2020.02.012 pii: S1684-1182(20)30040-2. [Epub ahead of print].
[210] L. D'Antiga, Coronaviruses and immunosuppressed patients, The facts during the third epidemic, Liver Transpl, 2020.
[211] (ACR). Coronavirus Disease (COVID-19). https://wwwrheumatologyorg/announcements. 2020.
[212] (EULAR). EULAR Guidance for patients COVID-19 https://wwwweularorg/eular_guidance_for_patients_covid19_outbreakcfm. 2020.
