
Review

# The Human Coronavirus Disease COVID-19: Its Origin, Characteristics, and Insights into Potential Drugs and Its Mechanisms

**Lo'ai Alanagreh** *, Foad Alzoughool and Manar Atoum*

Department of Medical Laboratory Sciences, Faculty of Applied Medical Sciences, The Hashemite University, Zarqa 13133, Jordan; foad@hu.edu.jo (F.A.); manar@hu.edu.jo (M.A.)

* Correspondence: loai-alanagreh@hu.edu.jo; Tel.: +962-(05)-3903333 (ext. 5596)

Received: 31 March 2020; Accepted: 27 April 2020; Published: 29 April 2020

---

**Abstract:** The emerging coronavirus disease (COVID-19) swept across the world, affecting more than 200 countries and territories. Genomic analysis suggests that the COVID-19 virus originated in bats and transmitted to humans through unknown intermediate hosts in the Wuhan seafood market, China, in December of 2019. This virus belongs to the *Betacoronavirus* group, the same group of the 2003 severe acute respiratory syndrome coronavirus (SARS-CoV), and for the similarity, it was named SARS-CoV-2. Given the lack of registered clinical therapies or vaccines, many physicians and scientists are investigating previously used clinical drugs for COVID-19 treatment. In this review, we aim to provide an overview of the CoVs origin, pathogenicity, and genomic structure, with a focus on SARS-CoV-2. Besides, we summarize the recently investigated drugs that constitute an option for COVID-19 treatment.

**Keywords:** COVID-19; SARS-CoV-2; antiviral therapies

---

## 1. Introduction

The coronavirus disease 2019 (COVID-19) was first identified in patients with severe respiratory disease in Wuhan, China. The causative agent was a novel coronavirus scientifically named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) [1,2]. Since its discovery, more than 2,850,000 cases have been infected, including nearly 200,000 who have died. The worrisome features of COVID-19 are its apparent ability to spread readily and its propensity to cause severe disease in older adults and patients with existing health conditions [3,4]. Moreover, coronaviruses are well-known to mutate and recombine [5]. The genomic sequence of SARS-CoV-2 has changed since it was first reported. Some scientists believe that these changes have enhanced the virulence of the virus and there are two circulating strains, deadly strain “L”, and less virulent one “S” [6]. Currently, there are no specific antiviral treatments or vaccines available for COVID-19. Treatments are mainly focusing on symptomatic and respiratory support according to protocols issued by the health authority in each country, where many countries are following WHO protocol [3,4]. Nearly all cases with severe symptoms accept oxygen. In critical cases and life-threatening situations, passive immunization through the convalescent plasma and immunoglobulin G transfusion can be used as a rescue treatment [7]. Given the current situation, finding a treatment has become a global public health priority. Many companies are working on finding a medication or a viable vaccine for SARS-CoV-2. However, this is not a short process; scientists estimate the period to have one, about 12–18 months.

The COVID-19 is the third novel coronavirus to cause a large-scale epidemic in the twenty-first century after the Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) in 2003 [8–10] and the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) in 2012 [11–13]. Our knowledge and

---

Pathogens 2020, 9, 331; doi:10.3390/pathogens9050331 www.mdpi.com/journal/pathogens

experience in fighting the previous two epidemics can be used for creating treatment strategies against this pandemic. Recently, different medications and a combination of drugs have shown promising results. For example, the combination of chloroquine and azithromycin [14]. Furthermore, the antiviral drug remdesivir and the malaria drug chloroquine have gained much attention globally as potential medications for COVID-19 treatment. On the other hand, different companies are working on finding a treatment in a creative method, antiviral RNA, for example [15].

Given the lack of specific treatment for COVID-19, many groups are working urgently to find an alternative strategy to control the replication and spreading of the virus. We have explored different scientific databases searching for the most promising options for the treatment of COVID-19 and provide an overview of our findings. We are going to review the characteristics of coronaviruses, its origin, genome structure, and replication to explain how each treatment strategy could act on preventing or slowing the viral infection.

## 2. Coronaviruses

Coronaviruses (CoVs) are a large group of viruses common among many animals, including humans. They can cause respiratory illnesses in humans and gastrointestinal illnesses in animals. Under the electron microscope, virions of CoVs have large peplomers that make it look like a crown, hence the name corona, meaning “crown” or “halo” [16–18]. Before 2003, human CoVs were not considered a deadly virus. The circulating strains were causing mild symptoms in immunocompetent people. Typically, coronavirus symptoms include runny nose, cough, sore throat, headache, and fever that can last for several days. However, in immunocompromised patients, there is a chance that the virus could cause a lower respiratory illness like pneumonia and bronchitis [16,17]. In 2003, the world was shocked by the first pandemic of the 21st century; the Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) emerged in Guangdong, China, resulting in 774 deaths and more than 8000 patients [16,19,20]. Nine years later, a strain of CoV evolved in Saudi Arabia to cause the Middle East Respiratory Syndrome Coronavirus (MERS-CoV), approximately 2500 cases have been confirmed, including 861 deaths with a fearful case-fatality rate of 34.4% [20].

### 2.1. Coronaviruses Diversity

CoVs members belong to the subfamily *Coronovirinae* within the family *Coronaviridae* and the order *Nidovirales*. Based on their protein sequences and phylogenetic relationships, members of the *Coronovirinae* subfamily can be classified into four groups: Alphacoronaviruses, Betacoronaviruses, Gammacoronaviruses, and Deltacoronaviruses. Gammacoronaviruses and Deltacoronaviruses infect birds and might infect mammals, but never reported to cause any illnesses in humans [20,21]. On the other hand, Alphacoronaviruses and Betacoronaviruses are capable of causing respiratory illnesses in humans and gastrointestinal illnesses in animals. Before December 2019, six common coronaviruses (members of Alphacoronaviruses and Betacoronaviruses) were known to infect humans. HCoV-229E, and HCoV-NL63 and they belong to the Alphacoronaviruses. HCoV-OC43, and HCoV-HKU1 belong to lineage A Betacoronaviruses, and the two deadly viruses SARS-CoV and MERS-CoV belong to lineages B and C of Betacoronaviruses, respectively [20]. The genomic analysis found that SARS-CoV-2 belongs to the Betacoronavirus group, lineage B [4,5].

CoVs are zoonotic pathogens originating in animals and can be transmitted to humans through direct contact. All CoVs that caused epidemics (including COVID-19) are believed to be originated in bats. Bats are hosts of many coronaviruses [17,22]. However, in most cases, these viruses were transmitted to humans through an intermediate animal host. SARS-CoV started through direct contact with market civets cats [23]. MERS-CoV transmitted directly to humans from dromedary camels [11–13]. The COVID-19 is suspected to be emerged in the seafood market in Wuhan, China, [1,20]. Most of the early reported cases have been in that market, which was closed later by the Chinese authority. Evolutionary analysis of COVID-19 virus revealed that it is most similar to the bat SARS-like coronaviruses, and for the similarity, it was named SARS-CoV-2. In summary, most of the scientific

report believes that SARS-CoV-2 was originated in bats and transmitted to humans through intermediate animal host in the seafood market [5]. Nevertheless, researchers are yet to find a definitive answer to which animal serves as an intermediate host.

## 2.2. Coronavirus Genome Structure and Replication

The CoVs genome is a single-stranded positive-sense RNA (+ssRNA) molecule. The genome size ranges between 27–32 kbp, one of the largest known RNA viruses (Figure 1) [20,24]. The genomic structure of CoVs contains at least six open reading frames (ORFs). The first ORFs (ORF1a/b), located at the 5′ end, about two-thirds of the whole genome length, and encodes a polyprotein1a,b (pp1a, pp1b) [25]. Other ORFs are located on 3′ end encodes at least four structural proteins: envelope glycoprotein spike (S), responsible for recognizing host cell receptors [26]. Membrane (M) proteins, responsible for shaping the virions [27]. The envelope (E) proteins, responsible for virions assembly and release [28]. The nucleocapsid (N) proteins are involved in packaging the RNA genome and in the virions and play roles in pathogenicity as an interferon (IFN) inhibitor [29,30]. In addition to the four main structural proteins, there are structural and accessory proteins that are species-specific, such as HE protein, 3a/b protein, and 4a/b protein [24]. Once the viral genome enters the cytoplasm of the target cell, and given it is a positive-sense RNA genome, it translates into two polyproteins 1a, b (pp1a, pp1b). These polyproteins are processed into 16 non-structural proteins (NSPs) to form a replication-transcription complex (RTC) that is involved in genome transcription and replication. Consequently, a nested set of subgenomic RNAs (sgRNAs) is synthesized by RTC in the form of discontinuous transcription [24].

![Diagram](attachment://image.png)

Figure 1. The genomic organization of SARS-CoV-2. The genome encodes two large genes ORF1a (yellow), ORF1b (blue), which encode 16 non-structural proteins (NSP1–NSP16). These NSPs are processed to form a replication-transcription complex (RTC) that is involved in genome transcription and replication. For example, NSP3 and NSP5 encode for Papain-like protease (PLP) and 3CL-protease, respectively. Both proteins function in polypeptides cleaving and block the host innate immune response. NSP12 encodes for RNA-dependent RNA polymerase (RdRp). NSP15 encodes for RNA helicase. The structural genes encode the structural proteins, spike (S), envelope (E), membrane (M), and nucleocapsid (N), highlighted in green. The accessory proteins (shades of grey) are unique to SARS-CoV-2 in terms of number, genomic organization, sequence, and function (figure created with biorender.com).

SARS-CoV-2 primarily infects ciliated bronchial epithelial cells and type II pneumocytes, where it binds to the surface receptor, angiotensin-converting enzyme 2 (ACE2), through S glycoprotein found on its surface (Figure 2) [2,31–33]. When S glycoprotein binds to the ACE2, the cleavage of trimer S protein is triggered by the cell surface-associated transmembrane protease serine 2 (TMPRSS2) and cathepsin. S glycoprotein includes two subunits, S1 and S2. S1 determines the host range and cellular tropism and facilitates viral attachment to the target cells. S2 is a unit that mediates the fusion of viral and cellular membranes, ensuring viral entry through endocytosis. [31]. The affinity between the virus’s surface proteins and its receptors is a critical step for viral entry. Understanding the mechanism of SARS-CoV-2 could provide more insights into the viral transmission and reveal therapeutic targets. A recent study showed that the affinity between S glycoprotein of SARS-CoV-2 and ACE2 binding efficiency is 10–20 fold higher than that of SARS-CoV, which could explain the highly infectious ability of SARS-CoV-2 [4,34].

![Figure 2](attachment:figure_2.png)

Figure 2. The life cycle of SARS-CoV-2 in the host cells. The S glycoproteins of the virion bind to the cellular receptor angiotensin-converting enzyme 2 (ACE2) and enters target cells through an endosomal pathway. Following the entry of the virus into the host cell, the viral RNA is unveiled in the cytoplasm. ORF1a and ORF1ab are translated to produce pp1a and pp1ab polyproteins, which are cleaved by the proteases of the RTC. During replication, RTC drives the production full length (-) RNA copies of the genome and used as templates for full-length (+) RNA genomes. During transcription, a nested set of sub-genomic RNAs (sgRNAs), is produced in a manner of discontinuous transcription (fragmented transcription). Even though these sgRNAs may have several open reading frames (ORFs), only the closest ORF (to the 5′ end) will be translated. Following the production of SARS-CoV-2 structural proteins, nucleocapsids are assembled in the cytoplasm and followed by budding into the lumen of the endoplasmic reticulum (ER)-Golgi intermediate compartment. Virions are then released from the infected cell through exocytosis (figure created with biorender.com).

## 3. Potential COVID-19 Treatment

Antiviral drugs usually interfere with the viral replication machinery in the target cells. Given that viruses have similar actions of infection, different studies suggest the use of successful antiviral drugs

for the treatment of COVID-19. Oseltamivir, peramivir, and zanamivir are neuraminidase inhibitors that have been used for influenza treatment. A recent report suggests the potential activity of these drugs against SAS-CoV-2 by interfering with the virions budding and releasing from the cell [35]. However, this virus uses different mechanisms other than neuraminidase to facilitate its release out of the infected cell. Another recent study suggests that ribavirin if combined with interferon-β, might be a promising drug for inhibition of SARS-CoV-2 replication [7]. Although this claim was based on previous in vitro findings [36], no other studies have shown the same results. Moreover, two reports did not recommend the use of this drug. On one hand, because it has no significant activity against SAS-CoV, and on the other hand, ribavirin is toxic with hemolysis activity [37, 38]. Even though, there are many drugs with general antiviral activity; that can interfere with viral entry or blocking the virus receptors. Some examples are listed in Table 1. These drugs are either under investigation, showed antiviral activity against SARS-CoV-2, or constituted a treatment option of COVID-19.

Table 1. Common and potent antiviral drugs.

| Author | Drugs | Therapy Strategy Categories | Mechanisms of Therapy | Status |
|--------|--------|-----------------------------|------------------------|--------|
| [14,39,40] | Chloroquine phosphate/ hydroxychloroquine | Anti-malaria anti-viral anti-inflammatory | Increasing endosomal pH, interfering with the glycosylation of cellular receptors of SARS-CoV-2, immunomodulator | FDA approved to be used in an emergency situation, implemented in many treatment protocols |
| [41] | Remdesivir | Antiviral drug (Nucleoside analogue) | Interfering with the viral replication | Investigational antiviral, clinical trials are in progress |
| [42] | Baricitinib | Rheumatoid arthritis (RA) drug, AP 2-associated protein kinase 1 (AAK1) inhibitor | Interfering with viral entry by inhibiting one of the endocytosis regulators | FDA approved |
| [43] | lopinavir/ritonavir | HIV protease inhibitor | Could act by inhibiting SARS-CoV-2 protease for proteins cleavage, interfering with virus replication | FDA approved |
| [44] | Darunavir | HIV protease inhibitor | Could act by inhibiting SARS-CoV-2 protease for proteins cleavage, interfering with virus replication | FDA approved |
| [45] | Camostat Mesylate | Transmembrane protease, serine 2 (TMPRSS2) inhibitor | Interfering with viral entry | Japan approved |
| [46] | Favipiravir | Nucleoside analog | Binds to the viral RdRp and reduce its reproduction | Investigational |
| [47] | Cepharanthine, Selamectin, and mefloquine hydrochloride | Anti-viral Anti-inflammatory activities | Significantly reduced cytopathic effects of SARS-CoV-2, and decrease the viral load | Investigational |
| [48] | Ivermectin | Anti-parasite | Inhibits SARS-CoV-2 replication in vitro | FDA approved |

3.1. Chloroquine and Hydroxychloroquine

Although many drugs have been reported to show promising results against SARS-CoV-2, none of these drugs have gained much attention like the antimalarial drugs Chloroquine (CQ) and Hydroxychloroquine (HCQ) [14, 49, 50]. HCQ is a CQ analog that has the same mechanism of action with a safer profile making it the prioritized drug to treat malaria, viruses, and autoimmune conditions [51]. There are several potential molecular mechanisms of HCQ action against SARS-CoV-2 that have been postulated. Generally, HCQ is a weak base drug that accumulates in the cell acidic compartments such as lysosomes and endosomes. This accumulation increases the pH and inhibits the maturation of these endosomal compartments. Therefore, HCQ might be interfering with the endocytosis of

SARS-CoV-2 and subsequently affecting the virus entry and exit from the host cell [7,50]. Moreover, HCQ might reduce glycosylation of ACE2, interfering with SARS-CoV-2 to bind effectively to the host cell [51]. On the other hand, a recent publication hypothesizes that HCQ reduces the production of pro-inflammatory cytokines (such as interleukin-6, IFN-alpha, and TNF), thereby inhibiting various immune pathways that might lead to acute respiratory distress syndrome (ARDS) [52]. Currently, the use of hydroxychloroquine is approved by many of COVID-19 treatment protocols, especially in combination with other antibiotics and antiviral drugs, where others seriously considering it [1,4,40,50].

### 3.2. Remdesivir

Remdesivir is an adenosine nucleoside analog drug showing antiviral activity against Ebola and Marburg viruses, and other RNA viruses [53]. It acts through interfering with the viral RNA polymerase and evades proofreading by viral exoribonuclease, leading to a decrease in viral RNA production [54,55]. A recent report found that a combination of remdesivir and chloroquine inhibited SARS-CoV-2 in vitro [56]. Very recently, the drug gained more attention after a recent clinical trial found that remdesivir is highly effective in stopping the replication mechanism of SARS-CoV-2.

### 3.3. Losartan and Telmisartan

Losartan and telmisartan are drugs that belong to the angiotensin II receptor antagonist group. They block the substances that cause blood vessels narrowing, which lowers blood pressure and improves blood flow by relaxing the blood vessels. It has been suggested that Losartan can be used to reduce the aggressiveness and mortality of SARS-CoV-2 [57–59]. The fact that these drugs could block the ACE2 receptor should mean the ability to interfere with the entry of SARS-CoV-2 to the target cell. Using this drug group to treat COVID-19 seems vital at this point, particularly in combination with other antiviral drugs.

### 3.4. Baricitinib

Baricitinib is a drug used basically for the treatment of rheumatoid arthritis. It acts as an inhibitor for Janus kinase (JAK, U.S. Food and Drug Administration, 2018). Inhibition of the JAK pathway might interrupt the virus entry and the virions intracellular assembly by interfering with the endocytosis process [42].

### 3.5. Lopinavir/Ritonavir

It is a combination of both lopinavir and ritonavir drugs that have been used for HIV treatment and prevention. These drugs act by inhibiting the HIV protease enzyme through forming an inhibitor-enzyme complex [60]. A recent study showed that combination of these drugs reduced SARS-CoV-2 viral load and improved the clinical symptoms during the treatment of COVID-19 patients [43].

### 3.6. Darunavir

Darunavir is a second generation of HIV-1 protease inhibitors. It is a non-peptidic inhibitor of protease that enters itself in the active site of protease through several hydrogen bonds [61]. A recent report of an in vitro experiment indicated that using darunavir at a concentration of 300 μM significantly inhibits SARS-CoV-2 replication. Its inhibition efficiency increased by 280-fold in comparison to the control group [44].

### 3.7. Camostat Mesylate

Camostat Mesylate is a type II transmembrane serine protease (TMSPSS2) inhibitor [62]. Since that SARS-CoV-2 entry mechanism involving the use of cellular protease TMPRSS2, a protease inhibitor would block entry and thus could be a treatment option [45].

Pathogens 2020, 9, 331

## 3.8. Cepharanthine, Selamectin, and Mefloquine Hydrochloride

These are medications used as anti-malarial, antiviral, and anti-inflammatory activities. Those drugs showed significantly decreased viral RNA yield in treated cells, making them potential drugs for treating COVID-19 infections [47].

## 3.9. SARS-CoV-Specific Human Monoclonal Antibody (CR3022)

Passive immunization (PI) is a method to obtain instant, short-term fortification against infectious agents by introducing pathogen-specific antibodies to patients. These specific antibodies can bind to the pathogen antigens and block its interaction with a cellular receptor. It is extremely applicable in the case of viral antigens that facilitate the attachment to the target receptors [63]. After exposure to a viral infection, the body of the patient creates antibodies to fight off the virus. These antibodies in the blood of a recovered patient can be collected as convalescent plasma (CP) and transferred to the blood of a newly infected patient. Consequently, it can neutralize the pathogen, boost the immunity of that patient [64]. CR3022 is a human monoclonal antibody previously isolated from a convalescent SARS patient, used to target a highly conserved epitope that enables cross-reactive binding between SARS-CoV-2 and SARS-CoV [65]. CR3022 might have the potential to be candidate therapeutics for the prevention and treatment of COVID-19 patients, especially in life-threatening situations [66].

## 3.10. CRISPR/Cas13d System

Clustered regularly interspaced short palindromic repeats (CRISPR) is a family of repeated DNA sequences that can be found in the genomes of bacteria and archaea. These fragments are molecular remaining of bacteriophages infections and act as a defense mechanism to the prokaryotic cell. They can detect and destroy specific foreign DNA fragments, most probably from similar bacteriophages subsequent infections. CRISPR with chaperoning Cas proteins is a powerful technique that has been derived from the bacterial defense mechanism and implemented in gene editing of our interest [67]. Currently, there is an ambitious study suggesting that CRISPR/Cas13d system can be used to accurately digest the SARS-CoV-2 RNA genome, hence limiting its ability to reproduce [15]. Theoretically, this approach is excellent not only against COVID-19 but for the treatment and prevention of different RNA viruses' infections. However, we have no idea if it is practical or not.

## 4. Conclusions

The COVID-19 pandemic poses a significant threat to the global public health systems. The increasing number of cases and deaths have put the international community on alert that the worst scenarios are possible. Despite our knowledge of the SARS-CoV-2 infectious cycle, there is no clear strategy for COVID-19 patients' treatment. Based on recent experimental findings and recommendations, physicians are investigating different potential drugs that showed antiviral activity against SARS-CoV-2. Only time to tell which one of these drugs is going to work. In the meantime, scientists around the globe are working aggressively to find clinical therapies or vaccines against the virus.

### Author Contributions:

The authors contributed equally to this work. All authors have read and agreed to the published version of the manuscript.

### Funding:

This research received no external funding.

### Conflicts of Interest:

The authors declare no conflict of interest.

References

1. Huang, C.; Wang, Y.; Li, X.; Ren, L.; Zhao, J.; Hu, Y.; Zhang, L.; Fan, G.; Xu, J.; Gu, X.; et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. *Lancet* **2020**, *395*, 497–506. [CrossRef]

2. Zhou, P.; Yang, X.-L.; Wang, X.G.; Hu, B.; Zhang, L.; Zhang, W.; Si, R.H.; Zhu, Y.; Li, B.; Huang, C.-L.; et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. *Nature* **2020**, *579*, 270–273.

3. Guan, W.-J.; Ni, Z.-Y.; Hu, Y.; Liang, W.-H.; Ou, C.-Q.; He, J.-X.; Liu, L.; Shan, H.; Lei, C.-L.; Hui, D.S.; et al. Clinical characteristics of coronavirus disease 2019 in China. *New Engl. J. Med.* **2020**. [CrossRef] [PubMed]

4. Guo, Y.-R.; Cao, Q.-D.; Hong, Z.-S.; Tan, Y.-Y.; Chen, S.-D.; Jin, H.; Tan, K.-S.; Wang, D.-Y.; Yan, Y. The origin, transmission and clinical therapies on coronavirus disease 2019 (COVID-19) outbreak—An update on the status. *Mil. Med Res.* **2020**, *7*, 1–10. [CrossRef] [PubMed]

5. Lu, R.; Zhao, X.; Li, J.; Niu, P.; Yang, B.; Wu, H.; Wang, W.; Song, H.; Huang, B.; Zhu, N.; et al. Genomic characterisation and epidemiology of 2019 novel coronavirus: Implications for virus origins and receptor binding. *Lancet* **2020**, *395*, 565–574. [CrossRef]

6. Tang, X.; Wu, C.; Li, X.; Song, Y.; Yao, X.; Wu, X.; Duan, Y.; Zhang, H.; Wang, Y.; Qian, Z.; et al. On the origin and continuing evolution of SARS-CoV-2. *Natl. Sci. Rev.* **2020**. [CrossRef]

7. Zhang, L.; Liu, Y. Potential interventions for novel coronavirus in China: A systematic review. *J. Med. Virol.* **2020**, *92*, 479–490. [CrossRef]

8. Lee, N.; Hui, D.S.; Wu, A.; Chan, P.K.S.; Cameron, P.; Joynt, G.; Ahuja, A.T.; Yung, M.Y.; Leung, C.; To, K.; et al. A major outbreak of severe acute respiratory syndrome in Hong Kong. *New Engl. J. Med.* **2003**, *348*, 1986–1994. [CrossRef]

9. Peiris, J.; Lai, S.; Poon, L.L.; Guan, Y.; Yam, L.; Lim, W.; Nicholls, J.M.; Yee, W.; Yan, W.; Cheung, M.; et al. Coronavirus as a possible cause of severe acute respiratory syndrome. *Lancet* **2003**, *361*, 1319–1325. [CrossRef]

10. De Wit, E.; Van Doremalen, N.; Falzarano, D.; Munster, V. SARS and MERS: Recent insights into emerging coronaviruses. *Nat. Rev. Genet.* **2016**, *14*, 523–534. [CrossRef]

11. Alagaili, A.N.; Briese, T.; Mishra, N.; Kapoor, V.; Sameroff, S.C.; De Wit, E.; Munster, V.; Hensley, L.E.; Zalmout, I.S.; Kapoor, A.; et al. Middle east respiratory syndrome coronavirus infection in dromedary camels in Saudi Arabia. *mBio* **2014**, *5*, e00884-14. [CrossRef] [PubMed]

12. Memish, Z.A.; Mishra, N.; Olival, K.J.; Fagbo, S.; Kapoor, V.; Epstein, J.H.; AlHakeem, R.; Durosinloun, A.; Al Asmari, M.; Islam, A.; et al. Middle East respiratory syndrome coronavirus in bats, Saudi Arabia. *Emerg. Infect.* **Dis. 2013**, *19*, 1819–1823. [CrossRef] [PubMed]

13. Zaki, A.M.; Van Boheemen, S.; Bestebroer, T.; Osterhaus, A.D.; Fouchier, R.A.M. Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. *New Engl. J. Med.* **2012**, *367*, 1814–1820. [CrossRef] [PubMed]

14. Gautret, P.; Lagier, J.-C.; Parola, P.; Hoang, V.T.; Meddeb, L.; Mailhe, M.; Doudier, B.; Courjon, J.; Giordanengo, V.; Vieira, V.E.; et al. Hydroxychloroquine and azithromycin as a treatment of COVID-19: Results of an open-label non-randomized clinical trial. *Int. J. Antimicrob. Agents* **2020**, *105949*. [CrossRef] [PubMed]

15. Nguyen, T.M.; Zhang, Y.; Pandolfi, P.P. Virus against virus: A potential treatment for 2019-nCov (SARS-CoV-2) and other RNA viruses. *Cell Res.* **2020**, *30*, 189–190. [CrossRef] [PubMed]

16. Wang, L.-F.; Shi, Z.; Zhang, S.; Field, H.; Daszak, P.; Eaton, B.T. Review of bats and SARS. *Emerg. Infect.* **Dis. 2006**, *12*, 1834–1840. [CrossRef]

17. Ge, X.-Y.; Li, J.; Yang, X.; Chmura, A.; Zhu, G.; Epstein, J.H.; Mazet, J.K.; Hu, B.; Zhang, W.; Peng, C.; et al. Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. *Nature* **2013**, *503*, 535–538. [CrossRef]

18. Chen, Y.; Guo, D. Molecular mechanisms of coronavirus RNA capping and methylation. *Virol. Sin.* **2016**, *31*, 3–11. [CrossRef]

19. Drexler, J.F.; Corman, V.M.; Drosten, C. Ecology, evolution and classification of bat coronaviruses in the aftermath of SARS. *Antivir. Res.* **2014**, *101*, 45–56. [CrossRef]

20. Cui, J.; Li, F.; Daszak, P. Origin and evolution of pathogenic coronaviruses. *Nat. Rev. Genet.* **2018**, *17*, 181–192. [CrossRef]

21. Woo, P.C.; Lau, S.K.; Lam, C.S.F.; Lau, C.C.Y.; Tsang, A.K.L.; Lau, J.H.N.; Bai, R.; Teng, J.L.L.; Tsang, C.C.C.; Wang, M.; et al. Discovery of seven novel mammalian and avian coronaviruses in the genus deltacoronavirus supports bat coronaviruses as the gene source of alphacoronavirus and betacoronavirus and avian coronaviruses as the gene source of gammacoronavirus and deltacoronavirus. *J. Virol.* **2012**, *86*, 3995–4008. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

22. Ithete, N.; Stoffberg, S.; Corman, V.M.; Cottontail, V.M.; Richards, L.R.; Schoeman, C.; Drosten, C.; Drexler, J.F.; Preiser, W. Close relative of human middle east respiratory syndrome coronavirus in Bat, South Africa. *Emerg. Infect. Dis.* **2013**, *19*, 1697–1699. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

23. Guan, Y.; Zheng, B.J.; He, Y.Q.; Liu, X.L.; Zhuang, Z.X.; Cheung, C.L.; Luo, S.W.; Li, P.H.; Zhang, L.J.; Butt, K.M.; et al. Isolation and characterization of viruses related to the SARS coronavirus from animals in Southern China. *Science* **2003**, *302*, 276–278. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

24. Chen, Y.; Liu, Q.; Guo, D. Emerging coronaviruses: Genome structure, replication, and pathogenesis. *J. Med. Virol.* **2020**, *92*, 418–423. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

25. Masters, P.S. *The Molecular Biology of Coronaviruses*; Elsevier: Amsterdam, The Netherlands, 2006; Volume 66, pp. 193–292.

26. Demogines, A.; Farzan, M.; Sawyer, S. Evidence for ACE2-Utilizing Coronaviruses (CoVs) related to severe acute respiratory syndrome CoV in Bats. *J. Virol.* **2012**, *86*, 6350–6353. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

27. Neuman, B.; Kiss, G.; Kunding, A.; Bhella, D.; Baksh, M.F.; Connelly, S.; Droese, B.; Klaus, J.P.; Makino, S.; Sawicki, S.G.; et al. A structural analysis of M protein in coronavirus assembly and morphology. *J. Struct. Biol.* **2010**, *174*, 11–22. [CrossRef](https://doi.org/CrossRef)

28. DeDiego, M.L.; Álvarez, E.; Almazán, F.; Rejas, M.T.; Lamirande, E.; Roberts, A.; Shieh, W.-J.; Zaki, S.R.; Subbarao, K.; Enjuanes, L. A severe acute respiratory syndrome coronavirus that lacks the E Gene is attenuated in vitro and in vivo. *J. Virol.* **2006**, *81*, 1701–1713. [CrossRef](https://doi.org/CrossRef)

29. Fehr, A.R.; Perlman, S. Coronaviruses: An overview of their replication and pathogenesis. In *Therapeutic Antibodies*; Springer: Berlin/Heidelberg, Germany, 2015; Volume 1282, pp. 1–23.

30. Cui, L.; Wang, H.; Ji, Y.; Yang, J.; Xu, S.; Huang, X.; Wang, Z.; Qin, L.; Tien, P.; Zhou, X.; et al. The nucleocapsid protein of coronaviruses acts as a viral suppressor of RNA silencing in mammalian cells. *J. Virol.* **2015**, *89*, 9029–9043. [CrossRef](https://doi.org/CrossRef)

31. Hoffmann, M.; Kleine-Weber, H.; Schroeder, S.; Krüger, N.; Herrler, T.; Erichsen, S.; Schiergens, T.S.; Herrler, G.; Wu, N.-H.; Nitsche, A.; et al. SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. *Cell* **2020**, *181*, 271–280.e8. [CrossRef](https://doi.org/CrossRef)

32. Li, W.; Moore, M.J.; Vasilieva, N.; Sui, J.; Wong, S.K.; Berne, M.A.; Somasundaran, M.; Sullivan, J.L.; Luzuriaga, K.; Greenough, T.C.; et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. *Nature* **2003**, *426*, 450–454. [CrossRef](https://doi.org/CrossRef)

33. Qian, Z.; Travanty, E.A.; Oko, L.; Edeen, K.; Berglund, A.; Wang, J.; Ito, Y.; Holmes, K.V.; Mason, R.J. Innate immune response of human alveolar Type II cells infected with severe acute respiratory syndrome–coronavirus. *Am. J. Respir. Cell Mol. Biol.* **2013**, *48*, 742–748. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

34. Letko, M.; Marzi, A.; Munster, V. Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses. *Nat. Microbiol.* **2020**, *5*, 562–569. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

35. Lu, H. Drug treatment options for the 2019-new coronavirus (2019-nCoV). *Biosci. Trends* **2020**, *14*, 69–71. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

36. Morgenstern, B.; Michaelis, M.; Baer, P.; Doerr, H.W.; Cinatl, J., Jr. Ribavirin and interferon-β synergistically inhibit SARS-associated coronavirus replication in animal and human cell lines. *Biochem. Biophys. Res. Commun.* **2005**, *326*, 905–908. [CrossRef](https://doi.org/CrossRef)

37. Booth, C.; Matukas, L.M.; Tomlinson, G.; Rachlis, A.R.; Rose, D.B.; Dwosh, H.A.; Walmsley, S.; Mazzulli, T.; Avendano, M.; Derkach, P.; et al. Clinical features and short-term outcomes of 144 patients with SARS in the Greater Toronto Area. *JAMA* **2003**, *289*, 2801. [CrossRef](https://doi.org/CrossRef)

38. Tan, E.L.; Ooi, E.E.; Lin, C.-Y.; Tan, H.C.; Ling, A.E.; Lim, B.; Stanton, L.W. Inhibition of SARS coronavirus infection in vitro with clinically approved antiviral drugs. *Emerg. Infect. Dis.* **2004**, *10*, 581–586. [CrossRef](https://doi.org/CrossRef)

39. Gao, J.; Tian, Z.; Yang, X. Breakthrough: Chloroquine phosphate has shown apparent efficacy in treatment of COVID-19 associated pneumonia in clinical studies. *Biosci. Trends* **2020**, *14*, 72–73. [CrossRef](https://doi.org/CrossRef)

40. Yao, X.; Ye, F.; Zhang, M.; Cui, C.; Huang, B.; Niu, P.; Liu, X.; Zhao, L.; Dong, E.; Song, C.; et al. In vitro antiviral activity and projection of optimized dosing design of hydroxychloroquine for the treatment of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). *Clin. Infect. Dis.* **2020**. [CrossRef]

41. Martinez, M.A. Compounds with therapeutic potential against novel respiratory 2019 coronavirus. *Antimicrob. Agents Chemother.* **2020**. [CrossRef]

42. Richardson, P.; Griffin, I.; Tucker, C.; Smith, D.; Oechsle, O.; Phelan, A.; Stebbing, J. Baricitinib as potential treatment for 2019-nCoV acute respiratory disease. *Lancet* **2020**, *395*, e30–e31. [CrossRef]

43. Lim, J.; Jeon, S.; Shin, H.-Y.; Kim, M.J.; Seong, Y.M.; Lee, W.J.; Choe, K.-W.; Kang, Y.M.; Lee, B.; Park, S.-J. Case of the index patient who caused tertiary transmission of coronavirus disease 2019 in Korea: The application of lopinavir/ritonavir for the treatment of COVID-19 pneumonia monitored by quantitative RT-PCR. *J. Korean Med Sci.* **2020**, *35*, e79. [CrossRef] [PubMed]

44. Dong, L.; Hu, S.; Gao, J. Discovering drugs to treat coronavirus disease 2019 (COVID-19). *Drug Discov. Ther.* **2020**, *14*, 58–60. [CrossRef] [PubMed]

45. Hoffmann, M.; Kleine-Weber, H.; Krüger, N.; Müller, M.; Drosten, C.; Pöhlmann, S.; A Muller, M. The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells. *bioRxiv* **2020**. [CrossRef]

46. Cai, Q.; Yang, M.; Liu, D.; Chen, J.; Shu, D.; Xia, J.; Liao, X.; Gu, Y.; Cai, Q.; Yang, Y.; et al. Temporary Removal: Experimental Treatment with Favipiravir for COVID-19: An open-label control study. *Eng.* **2020**. [CrossRef]

47. Fan, H.-H.; Wang, L.-Q.; Liu, W.-L.; An, X.-P.; Liu, Z.-D.; He, X.-Q.; Song, L.-H.; Tong, Y.-G. Repurposing of clinically approved drugs for treatment of coronavirus disease 2019 in a 2019-novel coronavirus (2019-nCoV) related coronavirus model. *Chin. Med. J.* **2020**. [CrossRef]

48. Caly, L.; Druce, J.D.; Catton, M.G.; Jans, D.A.; Wagstaff, K.M. The FDA-approved Drug Ivermectin inhibits the replication of SARS-CoV-2 in vitro. *Antivir. Res.* **2020**, *104787*. [CrossRef]

49. Liu, J.; Cao, R.; Xu, M.; Wang, X.; Zhang, H.; Hu, H.; Li, Y.; Hu, Z.; Zhong, W.; Wang, M. Hydroxychloroquine, a less toxic derivative of chloroquine, is effective in inhibiting SARS-CoV-2 infection in vitro. *Cell Discov.* **2020**, *6*, 1–4. [CrossRef]

50. Colson, P.; Rolain, J.-M.; Lagier, J.-C.; Brouqui, P.; Raoult, D. Chloroquine and hydroxychloroquine as available weapons to fight COVID-19. *Int. J. Antimicrob. Agents* **2020**, *55*, 105932. [CrossRef]

51. Devaux, C.A.; Rolain, J.-M.; Colson, P.; Raoult, D. New insights on the antiviral effects of chloroquine against coronavirus: What to expect for COVID-19? *Int. J. Antimicrob. Agents* **2020**, *105938*. [CrossRef]

52. Schrezenmeier, E.; Dorner, T. Mechanisms of action of hydroxychloroquine and chloroquine: Implications for rheumatology. *Nat. Rev. Rheumatol.* **2020**, *16*, 155–166. [CrossRef]

53. Warren, T.K.; Jordan, R.; Lo, M.K.; Ray, A.S.; Mackman, R.L.; Soloveva, V.; Siegel, D.; Perron, M.; Bannister, R.; Hui, H.C.; et al. Therapeutic efficacy of the small molecule GS-5734 against Ebola virus in rhesus monkeys. *Nature* **2016**, *531*, 381–385. [CrossRef] [PubMed]

54. Agostini, M.L.; Andres, E.L.; Sims, A.C.; Graham, R.L.; Sheahan, T.P.; Lu, X.; Smith, E.C.; Case, J.B.; Feng, J.Y.; Jordan, R.; et al. Coronavirus Susceptibility to the Antiviral Remdesivir (GS-5734) is mediated by the viral polymerase and the proofreading Exoribonuclease. *mBio* **2018**, *9*, e00221-18. [CrossRef] [PubMed]

55. Tchesnokov, E.P.; Feng, J.Y.; Porter, D.P.; Gotte, M. Mechanism of inhibition of Ebola Virus RNA-Dependent RNA polymerase by remdesivir. *Viruses* **2019**, *11*, 326. [CrossRef] [PubMed]

56. Wang, M.; Cao, R.; Zhang, L.; Yang, X.; Liu, J.; Xu, M.; Shi, Z.; Hu, Z.; Zhong, W.; Xiao, G. Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro. *Cell Res.* **2020**, *30*, 269–271. [CrossRef]

57. Sparks, M.A.; South, A.; Welling, P.; Luther, J.M.; Cohen, J.; Byrd, J.B.; Burrell, L.M.; Batlle, D.; Tomlinson, L.; Bhalla, V.; et al. Sound science before quick judgement regarding RAS Blockade in COVID-19. *Clin. J. Am. Soc. Nephrol.* **2020**. [CrossRef]

58. Gurwitz, D. Angiotensin receptor blockers as tentative SARS-CoV-2 therapeutics. *Drug Dev. Res.* **2020**. [CrossRef]

59. U.S. Food and Drug Administration. Development and Approval Process (Drugs). Available online: <https://www.fda.gov/drugs/development-approval-process-drugs> (accessed on 1 March 2020).

60. Sham, H.L.; Kempf, D.J.; Molla, A.; Marsh, K.C.; Kumar, G.N.; Chen, C.-M.; Kati, W.; Stewart, K.; Lal, R.; Hsu, A.; et al. ABT-378, a Highly potent inhibitor of the human immunodeficiency virus protease. *Antimicrob. Agents Chemother.* **1998**, *42*, 3218–3224. [CrossRef]

61. Leonis, G.; Czyżnikowska, Ż.; Megariotis, G.; Reis, H.; Papadopoulos, M.G. Computational studies of darunavir into HIV-1 protease and DMPC bilayer: Necessary conditions for effective binding and the role of the flaps. *J. Chem. Inf. Model.* **2012**, *52*, 1542–1558. [CrossRef]

62. Kawase, M.; Shirato, K.; Van Der Hoek, L.; Taguchi, F.; Matsuyama, S. Simultaneous treatment of human bronchial epithelial cells with serine and cysteine protease inhibitors prevents severe acute respiratory syndrome coronavirus entry. *J. Virol.* **2012**, *86*, 6537–6545. [CrossRef]

63. Rajam, G.; Sampson, J.; Carlone, G.M.; Ades, E.W. an augmented passive immune therapy to treat fulminant bacterial infections. In *Recent Patents Anti-Infective Drug Discovery*; Bentham Science Publishers: Sharjah, UAE, 2010; pp. 157–167. [CrossRef]

64. Marano, G.; Vaglio, S.; Pupella, S.; Facco, G.; Catalano, L.; Liumbruno, G.M.; Grazzini, G. Convalescent plasma: New evidence for an old therapeutic tool? *Blood Transfus.* **2015**, *14*, 152–157.

65. Yuan, M.; Wu, N.C.; Zhu, X.; Lee, C.-C.D.; So, R.T.Y.; Lv, H.; Mok, C.K.P.; Wilson, I.A. A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV. *Science* **2020**, eabb7269. [CrossRef] [PubMed]

66. Tian, X.; Li, C.; Huang, A.; Xia, S.; Lu, S.; Daszak, P.; Lu, L.; Jiang, S.; Yang, Z.; Wu, Y.; et al. Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. *Emerg. Microbes Infect.* **2020**, *9*, 382–385. [CrossRef] [PubMed]

67. Fu, Q.; Sun, J.; Yan, Y. The functional aspects of bacterial CRISPR-cas systems and interactions between phages and its bacterial hosts—a review. *Acta Microbiol. Sin.* **2015**, *55*, 251–257.
