
International Journal of  
Molecular Sciences

Review

# Autophagy in Neurodegenerative Diseases: A Hunter for Aggregates

Hyungsun Park ${}^{1,2}$, Ju-Hee Kang ${}^{2,3}$ and Seongju Lee ${}^{1,2,*}$

1. Department of Anatomy, College of Medicine, Inha University, Incheon 22212, Korea; hyungsun@inha.edu  
2. Hypoxia-related Disease Research Center, College of Medicine, Inha University, Incheon 22212, Korea; johykang@inha.ac.kr  
3. Department of Pharmacology, College of Medicine, Inha University, Incheon 22212, Korea  

* Correspondence: lees@inha.ac.kr; Tel.: +82-32-860-9891  

Received: 7 April 2020; Accepted: 7 May 2020; Published: 10 May 2020

---

**Abstract:** Cells have developed elaborate quality-control mechanisms for proteins and organelles to maintain cellular homeostasis. Such quality-control mechanisms are maintained by conformational folding via molecular chaperones and by degradation through the ubiquitin-proteasome or autophagy-lysosome system. Accumulating evidence suggests that impaired autophagy contributes to the accumulation of intracellular inclusion bodies consisting of misfolded proteins, which is a hallmark of most neurodegenerative diseases. In addition, genetic mutations in core autophagy-related genes have been reported to be linked to neurodegenerative diseases, such as Alzheimer's disease, Parkinson's disease, and Huntington's disease. Conversely, the pathogenic proteins, such as amyloid β and α-synuclein, are detrimental to the autophagy pathway. Here, we review the recent advances in understanding the relationship between autophagic defects and the pathogenesis of neurodegenerative diseases and suggest autophagy induction as a promising strategy for the treatment of these conditions.

**Keywords:** autophagy; neurodegenerative disease; protein aggregates

---

## 1. Autophagy Process

Autophagy is an evolutionary conserved intracellular degradation process. Mammals present three types of autophagic processes according to the adopted mechanism of cargo transport into lysosomes: chaperone-mediated autophagy, microautophagy, and macroautophagy, hereafter simply referred to as autophagy. Unnecessary or misfolded proteins and damaged subcellular organelles are engulfed by autophagy-specific double membrane-bound vesicles, called autophagosomes, and delivered to the lysosomes for breakdown. In almost all cells, autophagy occurs at a low basal level under inhibition by the mammalian target of rapamycin complex 1 (mTORC1), a key regulator of autophagy, to sustain cellular homeostasis. Upon various types of cellular stress, such as nutrient deprivation, growth factor withdrawal, or hypoxia, autophagy is released from mTORC1 inhibition and is highly upregulated to meet high energy demands.

### 1.1. Initiation of Autophagy

The autophagy process (summarized in Figure 1) consists of a series of molecular events, each of which is managed by autophagy-specific complexes whose activity is tightly regulated by upstream signaling molecules. Two main signaling molecules of autophagy, mTORC1 and AMP-activated protein kinase (AMPK), conversely regulate autophagy activity by phosphorylating Unc-51 like autophagy activating kinase 1 (ULK1) at different sites. mTORC1 inhibits autophagy initiation by phosphorylating ULK1 at Ser757, whereas AMPK promotes autophagy initiation by phosphorylating ULK1 at Ser317, Ser555, and Ser777 [1–3]. The ULK1 complex, known as an autophagy preinitiation

Int. J. Mol. Sci. 2020, 21, 3369; doi:10.3390/ijms21093369
www.mdpi.com/journal/ijms

complex, is composed of the ULK1 protein kinase, FIP200/RB1CC1, and the regulatory subunits ATG101 and ATG13, which are known to activate ULK1 through conformational changes [4,5]. The activated ULK1 complex phosphorylates a downstream autophagy initiation complex, the class III phosphoinositide 3-kinase (PI3K) VPS34 complex, which is composed of Beclin-1, VPS34, VPS15, and ATG14L [6–9]. The VPS34 complex generates phosphatidylinositol 3-phosphate (PI3P) in specialized phospholipid membranes, such as the endoplasmic reticulum (ER), ER-mitochondria contact sites, and ER-plasma membrane contact sites [10–12]. PI3P-binding proteins, such as zinc finger FYVE domain-containing protein 1 (ZFYVE1)/DFCP1 or WD repeat domain phosphoinositide-interacting proteins (WIPIs), are recruited to the PI3P-enriched membrane structures, called omegasomes, and further recruit autophagy-related proteins to generate the phagophore structure [13,14].

![Diagram](image.png)

Figure 1. Overview of the autophagy pathway. Upon cellular stresses, mTORC1 and AMPK phosphorylate the ULK1 complex and the VPS34 complex sequentially (P in yellow circle, activation; P in red circle, inhibition). The VPS34 complex generates PI3P in lipid-enriched membranes, which recruit PI3P-binding proteins. This lipid-protein complex facilitates the assembly of autophagy-related proteins, resulting in the formation of omegasomes. During the growth of the omegasomes to the autophagosomes, the ubiquitin-like conjugation systems of ATG proteins convert LC3 to LC3-II by conjugating PE, and the autophagic receptors and adaptor proteins transport cargoes to the autophagosomes. The matured autophagosomes are transported near the lysosomes by a dynein-dynactin complex, where they form the autolysosomes through the SNARE complex. After the fusion, lysosomal proteases degrade the cargoes. Meanwhile, PI(4,5)P₂ builds lysosomal tubule formation sites, generating the proto-lysosomes required for maintaining the free-lysosome pool in the cytosol. The detailed mechanisms are described in Section 1.

### 1.2. Autophagosome Formation

Two ubiquitin-like conjugation systems toward ATG8s (LC3A/B/C, GABARAP, GABARAPL1, and GABARAPL2 in mammals) are required for elongation of the phagophore. In the first system, ATG12 is covalently linked to ATG5 through the ATG7- and ATG10-mediated ubiquitin-like activating pathway [15]. ATG16L1 non-covalently binds to ATG5 through the interaction of coiled-coil domains and forms the ATG12-ATG5-ATG16L1 complex, which acts like the E3 ubiquitin ligase on the phagophore [16,17]. In the second system, the protease ATG4 cleaves the C-terminal region of LC3 to produce LC3-I into

the cytosol [18,19]. Next, LC3-I undergoes a cascade reaction of ATG7, ATG3, and ATG12–ATG5–ATG16L1, resulting in the formation of LC3-II conjugated with phosphatidylethanolamine (PE) [20]. LC3-II is tightly associated with the double-membrane phagophore during the elongation period [21,22].

Many recent studies have revealed the detailed regulatory mechanisms of phagophore expansion. ATG9A is the only transmembrane protein of the autophagy core proteins. When autophagy is activated, ATG9A is transferred from the trans-Golgi network (TGN) or the endocytic compartments to the omegasomes in a ULK1- or retromer complex-dependent manner [23–25]. The translocation of ATG9A-containing single-membrane vesicles to the outer membrane of autophagosomes is a crucial step for autophagosome formation, suggesting that the ATG9A-containing vesicles act as lipid bilayer donors in this process [26–29]. Another ATG protein, ATG2, is associated with the phagophore through the PI3P binding ability of ATG18 [30–32]. The ATG2-ATG18 complex tethers the PI3P lipid-enriched membrane to the phagophore, which is required to control the size of the early autophagic structures [33–35]. Furthermore, the yeast Atg2 protein was reported to possess a lipid transfer activity, which is essential for autophagy [36]. Although the precise mechanisms of autophagosome closure are still not clear, accumulating evidence suggests that ATG2, VPS21, and the endosomal sorting complexes required for transport (ESCRT) complex play a role in autophagosome closure [35,37–39].

### 1.3. Autophagosome Maturation

To fuse with lysosomes, autophagosomes move toward them through the intracellular trafficking systems. The dynein-dynactin motor complex and Rab proteins on the microtubules are necessary for autophagosome movement to lysosome-concentrated regions [40,41]. Particularly in neurons, most autophagosomes are generated in the axon tip and undergo a dynein-mediated retrograde transport to the soma [42–44]. Meanwhile, lysosomes also migrate toward autophagosomes in a GTPase- or retromer-dependent manner [45–47]. Finally, autophagosomes fuse with lysosomes to form autolysosomes through the soluble N-ethylmaleimide-sensitive-factor attachment protein receptor (SNARE) complex [41,48,49]. At the end of the autophagy process, clathrin-mediated lysosomal tubular structures are extended from the autolysosomes, referred to as autophagic–lysosomal reformation (ALR) [50]. PI(4,5)P₂ generated by the phosphatidylinositol-4-phosphate 5-kinase (PIP5K) is an essential factor for the initiation of autolysosome tubulation [51]. The large GTPase dynamin 2 is also involved in the scission of the lysosomal tubules during ALR [52,53].

### 1.4. Selective Autophagy

Initially, autophagy was thought of as a nonselective, bulk degradation process [54]. However, most autophagy is now regarded as a selective degradation process, as many kinds of autophagic receptors have been identified [55–57]. Autophagic receptors recognize and recruit specific cargoes to the autophagosome, for example, p62 for protein aggregates and nuclear FMR1 interacting protein 1 (NUFIP1) for ribosomes. Depending on the target molecules, selective autophagy is divided into aggrephagy for protein aggregates, mitophagy for mitochondria, ER-phagy for ER, pexophagy for peroxisomes, xenophagy for pathogens, lipophagy for liposomes, lysophagy for lysosomes, proteaphagy for proteasomes, ribophagy for ribosomes, and granulophagy for granules [58]. Autophagic receptors mostly recognize target molecules via their conjugated poly-ubiquitin chains through a specific domain, such as the ubiquitin-associated (UBA) domain on p62 or NBR1 and the zinc finger (ZF) domain on OPTN [59–62]. After this recognition occurs, the autophagic receptors transfer the cargoes to autophagosomes by directly binding with ATG8s through the LC3-interacting region (LIR) or the GABARAP interaction motif (GIM) [63,64].

Growing evidence has connected the genetic variants of the autophagic receptors to various neurodegenerative diseases. Since the accumulation of protein aggregates is seen in most neurodegenerative diseases, it is plausible that the malfunction of the autophagic receptors, rather than other autophagy-related proteins, might be related to the pathogenesis of neurodegenerative diseases. The relationships between

various autophagic receptors and neurodegenerative diseases are described in each section and summarized in Table 1.

Table 1. Association between various autophagic receptors/adaptors and neurodegenerative diseases.

| Neurodegenerative Disease | Autophagic Receptor/Adaptor | Target Aggregates | Reference |
|---------------------------|----------------------------|--------------------|-----------|
| Alzheimer's disease       | p62                        | Aβ                 | [65]      |
|                           | p62                        | tau                | [66–69]   |
|                           | NDP52                      | tau                | [70]      |
|                           | OPTN                       | tau                | [69]      |
|                           | PICALM                     | Aβ, APP-CTF        | [71, 72]  |
| Parkinson's disease       | p62                        | α-synuclein        | [73, 74]  |
| Huntington's disease      | p62                        | mHtt               | [75, 76]  |
|                           | OPTN                       | mHtt               | [77, 78]  |
|                           | ALFY                       | mHtt               | [76, 79]  |
| Amyotrophic lateral sclerosis | p62                        | TDP-43             | [80–82]   |
|                           | p62                        | SOD1               | [83, 84]  |
|                           | p62                        | UBQLN2             | [85, 86]  |
|                           | OPTN                       | TDP-43             | [78, 87]  |
|                           | OPTN                       | SOD1               | [77, 87, 88] |
|                           | UBQLN2                     | TDP-43             | [89]      |

2. Autophagy in the Nervous System and Neurodegenerative Diseases

2.1. Roles of Autophagy in the Nervous System

Autophagy is a lysosomal degradation pathway, which contributes not only to provide nutrients but also to remove harmful materials, such as misfolded proteins and invading microorganisms, to support cellular homeostasis and survival. Thus, defects in autophagy have been associated with various human diseases, including cancer, metabolic disorders, and neurodegenerative diseases. Hereafter, we will focus on the relationship between autophagic defects and neurodegenerative diseases.

Neurons are believed to be particularly vulnerable to malfunction of autophagy. Neurons are polarized cells with large amounts of cytoplasm and spatial heterogeneous endosomal populations; thus, the accumulation of cellular waste would represent a heavy burden for these cells [90, 91]. The cellular waste also does not get diluted, since neurons do not divide. Indeed, knockout of Atg genes in mice led to embryonic or neonatal lethality, suggesting the indispensability of autophagy and generation of brain-specific knockout model systems to investigate the role of autophagy in the brain [92, 93]. Knockout of the core autophagy protein, ATG5 or ATG7, in mouse neurons caused accumulation of polyubiquitinated inclusion bodies and behavioral defects [94, 95]. Similarly, Purkinje cell-specific knockout of ATG5 or ATG7 induced accumulation of autophagosome-like double-membrane structures in axonal swelling, followed by deficits in motor function [96, 97]. Depletion of another core autophagy protein, FIP200, in mouse neurons resulted in a reduction of autophagosome formation and cerebellar degeneration with progressive neuronal loss [98]. Silencing of p62, an autophagic receptor that specifically recognizes polyubiquitinated proteins, increased the formation of neurofibrillary tangles and behavioral abnormalities in mice and zebrafish [66, 99]. Conversely, overexpression of p62 in the mouse brain attenuated amyloid β (Aβ) pathology and ameliorated cognitive ability by upregulating autophagic clearance [65]. Recently, an unbiased genome-wide screening in the mouse striatum revealed that some ATG proteins, such as ATG5, ATG7, ATG101, and ATG16L1, are crucial for neuronal survival [100]. Those results indicate that autophagy is required to maintain neuronal homeostasis and remove protein aggregates, which are fundamental steps to prevent neuronal disorders.

Int. J. Mol. Sci. 2020, 21, 3369

## 2.2. Relationship of Autophagy Pathway with Pathogenesis of Neurodegenerative Diseases

Most neurodegenerative diseases exhibit pathological abnormal protein aggregates, developing neurofibrillary tangles; for example, Aβ and C-terminal fragments (CTF) of the amyloid precursor protein (APP) in Alzheimer's disease (AD), mutant α-synuclein in Parkinson's disease (PD), polyglutamine (polyQ)-expanded huntingtin (mHtt) in Huntington's disease (HD), and mutant superoxide dismutase 1 (SOD1) and TAR DNA-binding protein 43 (TDP-43) in amyotrophic lateral sclerosis (ALS) [57,101,102]. These protein aggregates observed in neurodegenerative diseases are mainly targeted towards the autophagy–lysosome degradation pathway. Similarly, genetic mutations in autophagic receptors, such as p62, OPTN, NBR1, and ALFY/WDFY3, have often been associated with neurodegenerative diseases [56,57]. Aging, the most common risk factor for neurodegeneration, also significantly decreases autophagic activity [103]. Therefore, it is thought that dysfunctional autophagy can lead to the development of neurodegenerative diseases. Recently, various studies have suggested that the modulation of autophagy could be a promising strategy for treating these conditions. The clearance of aggregate-prone proteins, such as mHtt, insoluble tau, and Aβ42, was increased by the activation of autophagy and dependent on the aggrephagy receptor, p62 [65,75,104–106]. Conversely, inhibition of autophagy using 3-MA or bafilomycin A1 (Baf. A1) increased mHtt aggregates in cell culture systems and rat brains [106,107]. Here, we review the recent advances in understanding the pathogenesis of neurodegenerative diseases in relation to autophagy (summarized in Figure 2) and discuss autophagy modulation as a therapeutic strategy for these diseases.

![Figure 2](#)

**Figure 2.** Interactions between the autophagy process and neurodegenerative disease-associated proteins. Increasing evidence for interactions between autophagy and neurodegenerative diseases suggests that not only the decrease of autophagic activity is one of causes of the diseases, but mutations in disease-associated genes also inhibit autophagy in various stages. The possible interruptions in each autophagic process from initiation to reformation of free lysosomes by the different neurodegenerative disease-associated mutant proteins are shown. The detailed relationships between the mutant proteins and autophagy pathway are described in Section 2.

Int. J. Mol. Sci. 2020, 21, 3369

2.2.1. Alzheimer's Disease (AD)

AD is the most common neurodegenerative disease, causing dementia. Accumulation of Aβ plaques and tau neurofibrillary tangles in the brain of AD patients is considered a pathological characteristic of the disease, as well as a crucial part of its pathogenesis. Aβ, a central component of amyloid plaques, is a small peptide derived from the sequential processing of APP. APP is mainly cleaved into the TGN and the endosomes by sequential activation of α-, β- and γ-secretase [108,109]. The clearance of Aβ and APP–CTF is mainly achieved through autophagy [110–112]. The increase in the activity of p62 or transcription factor EB (TFEB) was shown to attenuate Aβ plaque formation, resulting in the improvement of AD pathology in mice [65,113]. Contrastingly, increased Aβ oligomers hindered autophagic activity through the impairment of trafficking and lysosome biogenesis in animal models [114,115]. The ultrastructural analysis of the brain of AD patients revealed that cathepsin-containing autolysosomes accumulated due to defects of lysosomal proteolysis [116–118]. Furthermore, the levels of autophagy-related proteins are frequently altered in AD patients' samples [119–121]. The genes linking autophagy to AD are discussed in detail below.

Some mutations causing familial AD are reported to affect autophagy. Presenilin 1 (PSEN1/PS-1) is a component of the γ-secretase complex that generates Aβ peptide [122]. To date, a number of genetic mutations of PSEN1 have been found to affect APP processing and to be linked with AD pathogenesis [123]. Several loss-of-function analyses have revealed that PSEN1 is required for lysosomal homeostasis and transcription of autophagy-related genes [124–126]. PSEN1 mutations causing familial AD lead to aberrant v-ATPase trafficking to lysosomes, thus resulting in lysosomal alkalization and accumulation of dysfunctional autolysosomes [126–128]. PSEN1-deficient neural stem cells also exhibited reduced TFEB expression, leading to decreased transcription levels of autophagy-related genes [125].

Various single nucleotide polymorphisms (SNPs) and abnormal cleaved forms of phosphatidylinositol-binding clathrin assembly protein (PICALM) have been reported in AD [129,130]. PICALM is a clathrin adaptor protein involved in the clathrin-mediated endocytosis of SNARE proteins and APP. PICALM and APP colocalize during endocytosis, and genetic downregulation of PICALM selectively decreases endocytosis of APP and Aβ plaque generation in the brain of mice [71]. Recent studies have shown that PICALM is involved in multiple steps of the autophagic pathway, regardless of its role in APP endocytosis. Indeed, PICALM affects the formation and maturation of autophagosomes by regulating endocytosis of SNAREs, such as VAMP2, VAMP3, and VAMP8 [131]. SNAREs are known to mediate the fusion of autophagosomes with lysosomes [49]. Additionally, PICALM functions as an autophagic receptor and makes a complex with adaptor protein 2 (AP-2). The AP2-PICALM complex bridges the APP-CTF and LC3B for autophagic degradation [72].

The decreased autophagy activity in AD patients may also be attributed to Beclin-1 and VPS35, a core retromer component. Reduced levels of Beclin-1 and VPS35 have been reported in AD patients [132,133]. Genetic downregulation of Beclin-1 in mice resulted in reduced neuronal autophagy and Aβ accumulation, leading to neurodegeneration [133]. Mechanistically, Beclin-1 promotes APP trafficking from the cell surface to the autophagosomes by direct interaction with APP via an evolutionarily conserved domain (ECD) [134]. Interestingly, Beclin-1 regulates phagocytosis in neurons by regulating VPS35 protein levels [132]. Knockdown of VPS35 in culture cells has also been found to cause Aβ accumulation [135].

Degradation of aberrant phosphorylated tau also depends on autophagy [110,136]. Accumulation of proteins in the autophagy–lysosomal pathway, such as p62, LC3, and LAMP1, as well as autophagic and lysosomal defects were observed in the post-mortem brains of patients that had AD [67]. The protein aggregates were colocalized with hyperphosphorylated tau (S422) [67,68]. Hyperphosphorylated tau showed direct bindings with aggrephagy receptors, such as p62, NDP52, and OPTN, and was subjected to autophagic degradation [66,69,70]. PICALM also mediates autophagic degradation of tau [131]. Besides, impaired transport by the dynein–dynactin complex, which contributes to autophagosome trafficking, increased tau-positive filamentous inclusions [137]. Contrastingly, autophagy activation

accelerated the degradation of phosphorylated tau, resulting in the inhibition of tau aggregation in vitro and in vivo [138, 139].

### 2.2.2. Parkinson's Disease (PD)

PD is a progressive neurodegenerative movement disorder, characterized by pathological α-synuclein inclusions called Lewy bodies in the dopaminergic neurons of the substantia nigra and multiplication of the α-synuclein gene [140]. Knockout of ATG7 was shown to induce an age-dependent increase in the formation of p62-containing α-synuclein inclusion bodies in dopaminergic neurons and motor function deficits in aged mice [73]. Several studies indicate that α-synuclein bearing pathogenic mutations is degraded by the autophagy-lysosome system [141–144]. Conversely, α-synuclein inclusions impair the autophagic pathway in several steps. For instance, it was demonstrated that α-synuclein inclusions reduce omegasome formation by inducing the mislocalization of ATG9A [145, 146]. Besides, α-synuclein aggregates damage the retrograde transport of autophagosomes, but not the autophagosome-lysosome fusion [146, 147]. Finally, α-synuclein disrupts the activity of the lysosomal aspartyl protease cathepsin D (CTSD) and the autophagic degradation process [148–150].

Transcriptional alteration of autophagy-related genes is frequently observed in PD. During the symptomatic stage of PD, when most dopaminergic neurons in the midbrain are already lost, the TFEB-mediated transcription of Beclin-1, CTSD, and LAMP1 is decreased compared to the pre-symptomatic stage [151]. Moreover, the expression of human A30P α-synuclein increased the nuclear translocation of the transcriptional repressor zinc finger protein with KRAB and SCAN domains 3 (ZKSCAN3), which regulates the transcription of LC3 and p62 [152]. As expected, the pathological features of PD, including neurodegeneration in the midbrain, motor dysfunction, and α-synuclein accumulation, were improved by TFEB overexpression [153, 154].

Mutations in glucocerebrosidase (GBA), a lysosomal enzyme degrading glucosylceramide, represent the most common genetic risk factor for PD. The PD-associated GBA mutations (N370S and L444P) reduce its protein levels and enzymatic activity and impair its trafficking from the ER to the lysosomes. This induces ER stress and the accumulation of target lipids in lysosomes, which eventually results in autophagy-lysosomal dysfunction [155–157]. Patients with sporadic PD show a selective decrease of GBA activity accompanied by increased α-synuclein inclusions at an early stage [158, 159]. Direct inhibition or N370S mutation of GBA promotes the accumulation of α-synuclein oligomers [156, 158, 160]. Moreover, it was demonstrated that glucocerebroside, the GBA target lipid, can promote α-synuclein fibril formation; the lysosomal membrane-bound α-synuclein fibrils can then inhibit the activity and trafficking of GBA through direct binding, leading to further exacerbation of PD [161, 162].

The next most common genetic risk factor in PD is represented by mutations in the leucine-rich repeat kinase 2 (LRRK2/PARK8) gene; more than 40 pathogenic LRRK2 mutations have been reported in patients with PD [163]. However, it is still controversial whether the role of LRRK2 in autophagy is associated with PD pathology. Some studies have shown that LRRK2 loss impairs the autophagy-lysosome pathway, resulting in cell death [164, 165]. Interestingly, many pathogenic mutations in LRRK2 are gain-of-function mutations, for example, G2019S and R1441C [166]. The LRRK2 gain-of-function mutations increase its kinase activity, but impair autophagic degradation, similar to LRRK2 deficiency [167–170]. This paradox would be explained by some studies showing that the LRRK2-G2019S mutant inhibits the endocytic vesicular trafficking by decreasing small GTPase activity and that the LRRK2-R1441C mutant inhibits lysosomal functions due to its defective binding to the lysosomal v-ATPase [171, 172].

The loss-of-function mutants of ATPase cation transporting 13A2 (ATP13A2), which are characterized in an early-onset form of PD, were reported to be retained in the ER and not translocated to the lysosomes [173]. ATP13A2 is a lysosomal type 5 P-type ATPase, and therefore essential for the maintenance of the lysosomal pH. In PD patients, ATP13A2 protein levels were found to be decreased in dopaminergic neurons, and the existing low amounts of ATP13A2 proteins were located in the Lewy

bodies [174]. Furthermore, the PD-associated mutations of ATP13A2 were reported to cause impairment of lysosomal acidification [174]. Recent studies have shown the molecular mechanisms underlying the ATP13A2-mediated autophagy–lysosome pathway. Depletion of ATP13A2 induces the retention in the cytosol of TFEB, a critical transcription factor for autophagy-related genes, by regulating mTORC1 activity [175]. Moreover, the downregulation of Synaptotagmin 11 (SYT11) caused by ATP13A2 deficiency induces lysosomal dysfunction, leading to disruption of autophagic degradation [175]. Another study reported that ATP13A2 enhances autophagosome–lysosome fusion by facilitating HDAC6-dependent cortactin deacetylation that leads to the assembling of an F-actin network [176]. In addition, a mutation in ATP13A2 was shown to cause α-synuclein accumulation and silencing of α-synuclein was able to attenuate the neurotoxicity induced by ATP13A2 depletion [177], suggesting that loss of ATP13A2 may contribute to PD pathology via α-synuclein accumulation.

The autophagy defects observed in PD may be partly attributed to the mutation of VPS35, a core retromer complex component, which has been reported to regulate trafficking of lysosomal protease [46,178]. mRNA levels of VPS35 were found to be decreased in the PD substantia nigra [179]. A familial PD-associated VPS35 mutation (D620N) was shown to impair the recruitment of the WASP and Scar homolog (WASH) complex to the endosomes, resulting in the mislocalization of ATG9A and defective autophagy [180].

### 2.2.3. Huntington's Disease (HD)

HD is an autosomal-dominant progressive neurodegenerative disease, exhibiting motor dysfunction, behavioral disturbances, and cognitive dysfunction. The abnormal expansion of a polyQ repeat in exon 1 produces mHtt proteins that form ubiquitin-positive aggregates with β-sheet rich structures, leading to cytotoxicity in the striatum and cortex [181,182]. The overexpression of mHtt induces progressive motor deficits accompanied by the accumulation of autophagosomes [183,184]. Similarly, the accumulation of autophagic vacuoles was observed in HD patients [185,186]. A number of studies have shown that autophagy clears aggregate-prone proteins with polyQ expansion from cell culture systems to in vivo [106,141,187–189]. In those studies, autophagy blockage by treatment with an autophagy inhibitor, such as 3-MA or Baf.A1, or genetic modulation increased mHtt aggregation. Contrastingly, an autophagy activator, such as rapamycin or trehalose, reduced the number of inclusion bodies. The turnover rate of mHtt varied depending on its structure, which influenced its interaction with the aggrephagy receptors p62 and OPTN [75,77,107]. The heterozygous deficiency of the aggrephagy adaptor ALFY, which facilitates the recruitment of the p62-binding intracellular inclusions to the autophagosomes, also promoted the aberrant accumulation of mHtt and HD progression in a mouse model [76,79]. A recent genome-wide screening in the striatum has suggested that many autophagy-related genes, such as Atg4b, Tfeb, and Atlastin 3, appear to prevent mHtt toxicity [100].

The normal huntingtin acts as a scaffolding protein for various autophagy proteins to facilitate cargo recognition and prevent Beclin-1 ubiquitination [190–192]. However, mHtt fails to recognize cytosolic cargoes in the autophagosomes [193]. mHtt was also shown to reduce the Beclin-1 activity by recruiting it to the inclusion bodies or inactivating a striatal specific GTPase, RASD family member 2 (RASD2/Rhes), which promotes the dissociation of inhibitory interactions between Beclin-1 and Bcl-2 [192,194]. Moreover, huntingtin knockdown or mHtt overexpression in neurons can lead to defective retrograde transport of the autophagosomes [42].

### 2.2.4. Amyotrophic Lateral Sclerosis (ALS)

ALS is a rare progressive neurodegenerative disease, characterized by the death of motor neurons controlling the voluntary muscles as well as by cytoplasmic ubiquitin-positive inclusion formation. Most ALS cases are sporadic, but about 5% to 10% are caused by inherited genetic mutations in SOD1, TDP-43, fused in sarcoma (FUS), or ubiquilin 2 (UBQLN2) [195–198]. Many studies have suggested that autophagy might contribute to ALS pathogenesis. First, enlarged autophagosomes containing
p62-positive aggregates were observed in ALS mouse models and ALS patients [80,199]. Secondly, motor neuron-specific ATG7 knockout mice bearing a SOD1 pathogenic mutation have been found to have accelerated neuromuscular junction disruption as well as tremor, which are characteristics of ALS [199]. Thirdly, silencing of TDP-43 or one of its ALS-associated mutations increased the transcription of Bcl-2 and abnormal ATG4B proteins, resulting in autophagic defects [200–202]. Finally, autophagy activation reduced TDP-43 aggregates and improves the survival of human motor neurons bearing TDP-43 mutations [203].

UBQLN2, another genetic risk factor for familial ALS, has been associated with autophagy as well. UBQLN2 forms a complex with LC3 and ubiquitinates cargo proteins in the autophagosomes like the autophagy adaptor [204,205]. Moreover, UBQLN2 facilitates the autophagosome–lysosome fusion [204]. Loss of UBQLN2 was shown to increase ubiquitinated TDP-43 levels and impair autophagic degradation by promoting the fragmentation of lysosomal v-ATPase [89,206]. The overexpression of an ALS-associated UBQLN2 mutant, such as P497H, caused the formation of UBQLN2 inclusions bearing p62 and the accumulation of ubiquitinated-protein aggregates, resulting in defective autophagy and ALS-like phenotypes [85,86,207].

Recently, the correlation between ALS and aggrephagy receptor proteins, such as p62 and OPTN, has become more apparent. As mentioned above, TDP-43 inclusions were found to colocalize with p62, and to be degraded by p62 overexpression [81,82]. Conversely, p62 deficiency exacerbated ALS symptoms caused by the increase of insoluble protein aggregates [83,99]. Importantly, about half of the ALS-related mutations are located in the PB1, LIR, and UBA regions of p62 [57]. These regions are responsible for interacting with cargo proteins or LC3, suggesting that the ALS pathology originating from p62 mutations may be linked with the inefficient delivery of protein aggregates to the autophagosomes [84,99,208]. Another aggrephagy receptor, OPTN, promotes the clearance of protein aggregates by recognizing the inclusions through its coiled-coil domain [77]. OPTN silencing or its ALS-associated mutations increased the number of TDP-43- or SOD1-containing inclusions, resulting in necroptosis-dependent axonal degeneration [77,87,88]. In addition to its role as an aggrephagy receptor, OPTN contributes to autophagosome trafficking through MYO6 and TOM1 [209]. The interaction between OPTN and MYO6 is often disrupted in ALS patients or in case of mutations [78,210].

ALS can be also developed due to mutations in the ESCRT-III complex, which is important for autophagosome maturation. ALS-associated point mutations in CHMP2B, a subunit of the ESCRT-III complex, have been found to cause accumulation of autophagosomes and mislocalization of lysosomes, leading to the formation of intracellular inclusions in various model systems [211–213].

### 2.2.5. Hereditary Spastic Paraplegia (HSP)

HSP is a diverse group of inherited neurodegenerative disorders characterized by axonal degeneration of the corticospinal motor neurons, leading to progressive weakness and spasticity of the legs. Until now, over 80 spastic gait (SPG) genetic loci have been reported, and more than 60 genes have been identified [214]. While various SPG genes exist, the molecular etiology underlying HSPs converges to a small number of cellular functions, such as membrane trafficking, mitochondrial function, organelle shaping and biogenesis, axonal transport, and lipid metabolism [215,216]. The most common form of autosomal-recessive HSP is induced by mutations in *SPG11* encoding spatacsin. Loss of spatacsin in mouse neurons has been found to induce aberrant lysosomal homeostasis, including the accumulation of autolysosomes and impairment of ALR [217,218]. Similar to spatacsin, depletion of spastizin, encoded by *SPG15*, diminished ALR initiation, resulting in reduced numbers of free lysosomes, and consequently in accumulation of cellular garbage [218]. Spastizin also regulates autophagosome maturation by interacting with the Beclin-1-UVRAG-Rubicon complex or by promoting Rab-dependent endosome–autophagosome fusion [219,220].

Several SPG genes encode adaptor protein (AP) complex subunits. The AP complex is located in the endosome or the TGN, and it is important for intracellular cargo sorting between distinct organelles.
The AP-4 subunits β-1, μ-1, ε-1, and σ-1 are encoded by SPG47, SPG50, SPG51, and SPG52, respectively, and associated with childhood-onset HSP [221]. As the AP-4 complex mediates the sorting of the target proteins from the TGN to another membrane structure, AP-4 deficiency results in the missorting of ATG9A-containing vesicles from the TGN to autophagosome formation sites in the neurons [221, 222]. Another AP complex subunit linked with HSP is AP-5 ζ-1, which is encoded by SPG48. A previous study reported that AP-5 interacts with spatacsin and spastizin [223]. Similar to the phenotypes obtained through silencing of spatacsin or spastizin, old AP-5 ζ-1 knockout mice exhibited degeneration of the corticospinal tract and incorporeal Golgi apparatus [224]. Furthermore, defects of autophagic clearance and ALR impairment can be observed in the AP-5 ζ-1 knockout mice.

## 3. Autophagy Upregulation as a Therapeutic Strategy for Neurodegenerative Diseases

As mentioned above, various genetic mutations that reduce autophagic activity are linked to the pathology of inherited neurodegenerative diseases, and appear to aggravate them. Recent studies have shown that reducing the accumulation of intracellular aggregate-prone proteins by autophagy upregulation is beneficial to delay disease progression in AD, PD, HD, and ALS, suggesting that autophagy induction can be used as a therapeutic strategy for most neurodegenerative diseases [101, 102, 104, 106, 225–227] (summarized in Figure 3). However, extensive autophagy is detrimental to maintain cellular homeostasis. Therefore, caution should be taken when targeting autophagy for the treatment of neurodegenerative diseases.

![Figure 3](attachment:Figure_3.png)

**Figure 3. Mechanisms of therapeutics of autophagy-inducing agents.** Treatment of various autophagy-inducing agents enhances the clearance of aggregate-prone proteins in an mTOR-dependent or -independent manner. The mTOR-dependent autophagy inducers (red), rapamycin and curcumin, directly inhibit mTORC1 activity, resulting in activation of the ULK1 complex. Conversely, mTOR-independent agents (blue) upregulate autophagy activity through various intracellular signaling cascade or lysosome biogenesis. The simplified actions of the agents are drawn, and detailed mechanisms are described in Section 3. AC = adenylyl cyclase, PLC-ε = phospholipase C-epsilon, DAG = diacylglycerol.

Int. J. Mol. Sci. 2020, 21, 3369

3.1. mTOR-Dependent Autophagy Inducing Agents

Two types of small molecules can induce autophagy, in an mTOR-dependent or -independent manner. The first identified mTOR-dependent autophagy inducer is rapamycin. Rapamycin inhibits the kinase activity of mTORC1 by allosteric binding. Though not approved for clinical use, rapamycin treatment was shown to reduce neuronal death and improve most neurodegenerative disease symptoms in experimental settings by inducing autophagy [104,106,226,227]. Due to the limited absorption of rapamycin, its derivatives (defined rapalogs) have been developed, such as temsirolimus (CCI-779), everolimus (RAD001), and ridaforolimus (AP23575). Among them, everolimus has been recently approved by the Food and Drug Administration (FDA) for tuberous sclerosis treatment. In addition to rapamycin, curcumin, a natural compound that inhibits the PI3K/Akt/mTOR pathway, has been described as having a therapeutic effect on neurodegenerative diseases [228,229]. Curcumin has been shown to enhance the expression levels of autophagy-related proteins, such as Beclin-1, ATG5, or ATG16L1, and motor proteins essential for retrograde axonal transport, resulting in an increase of the autophagic flux and the clearance of intracellular aggregates [113,228,230,231].

3.2. mTOR-Independent Autophagy Inducing Agents

Along-term use of mTOR-dependent autophagy inducers could result in side effects, because the mTOR signaling has diverse autophagy-independent functions, such as ribosome biogenesis [102]. Therefore, mTOR-independent autophagy inducers have been developed. The representative mTOR-independent autophagy inducer is trehalose, which enhances autophagy by activating AMPK or TFEB [232,233]. Several studies have reported that this disaccharide promotes the clearance of protein aggregates and reduces neuronal death, thereby alleviating the progression of neurodegeneration diseases [138,141,148,234,235]. Some mood stabilizers, such as verapamil, loperamide, clonidine, and calpastatin, induce the degradation of aggregate-prone proteins by the autophagy-lysosome system via a reduction in inositol phosphate 3 (IP3) levels, which inhibits the autophagosome formation step. In the phosphoinositol cycle, lithium inhibits inositol monophosphatase (IMPase), whereas carbamazepine and valproic acid inhibit inositol synthesis [236,237].

Lysosomal alkalization by aberrant lysosomal ATPase causes autophagosome accumulation, resulting in the inhibition of the degradative pathway. Therefore, the re-acidification of lysosomes may be the proper strategy for autophagy enhancement. Restoring lysosomal pH using poly (DL-lactide-co-glycolide) (PLGA) acidic nanoparticles was shown to rescue lysosomal deficits and autophagic degradation [124,206].

Another therapeutic strategy involves small peptides. The 267–284 amino acid residues of Beclin-1 are known to be essential for autophagy induction [238]. The Tat-Beclin-1 peptide is a cell-permeable peptide of Beclin-1 (aa 267–284) conjugated with the HIV-1 Tat protein. The Tat-Beclin-1 enhances autophagy initiation by interacting with the autophagy inhibitor, Golgi-associated plant pathogenesis-related protein 1 (GAPR-1/GLIPR2), resulting in Beclin-1 distribution into the cytosol. This peptide also increases autophagosome formation in neurons. Besides, Tat-Beclin-1 has been shown to reduce mHtt aggregates in vitro.

4. Conclusions

Although inherited neurodegenerative diseases are caused by mutations in various genes, the accumulation of protein aggregates is a common characteristic they share. Thus, neurodegenerative diseases are considered proteinopathies. Numerous studies indicate that several genes associated with these conditions are involved in the autophagy–lysosome pathway and the intracellular protein aggregates can disrupt several steps of autophagy. Thus, we speculate that autophagy upregulation can ameliorate neurodegenerative diseases. In various experimental systems, autophagy activation reduces the accumulation of inclusion bodies and further alleviates neurodegeneration phenotypes. However, the intervention on neurodegenerative diseases using autophagy inducers is only in its infancy.

The pharmacological autophagy modulation strategies currently in use are mostly based on the overall induction of the whole autophagy process. Besides, excessive autophagy activation results in toxic effects. Selective approaches should be implemented to target specific autophagy steps. To this end, it is necessary to thoroughly understand the roles of autophagy in various neurodegenerative diseases.

**Author Contributions:** This review article was conceptualized by H.P., J.-H.K. and S.L.; H.P. and S.L. wrote the manuscript; J.-H.K. professionally edited the manuscript. All authors have read and agreed to the published version of the manuscript.

**Funding:** This work was supported by the Basic Science Research Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Science and ICT (MRC 2014R1A5A2009392) and the Ministry of Education (2016R1A6A3A04006478 and 2019S1A5C2A03082727).

**Conflicts of Interest:** The authors declare that they have no conflict of interest.

**References**

1. Di Nardo, A.; Wertz, M.H.; Kwiatkowski, E.; Tsai, P.T.; Leech, J.D.; Greene-Colozzi, E.; Goto, J.; Dilsiz, P.; Talos, D.M.; Clish, C.B.; et al. Neuronal Tsc1/2 complex controls autophagy through AMPK-dependent regulation of ULK1. *Hum. Mol. Genet.* **2014**, *23*, 3865–3874. [CrossRef] [PubMed]
2. Jang, M.; Park, R.; Kim, H.; Namkoong, S.; Jo, D.; Huh, Y.H.; Jang, I.S.; Lee, J.I.; Park, J. AMPK contributes to autophagosome maturation and lysosomal fusion. *Sci. Rep.* **2018**, *8*, 12637. [CrossRef] [PubMed]
3. Kim, J.; Kundu, M.; Viollet, B.; Guan, K.L. AMPK and mTOR regulate autophagy through direct phosphorylation of Ulk1. *Nat. Cell Biol.* **2011**, *13*, 132–141. [CrossRef] [PubMed]
4. Lin, M.G.; Hurley, J.H. Structure and function of the ULK1 complex in autophagy. *Curr. Opin. Cell Biol.* **2016**, *39*, 61–68. [CrossRef] [PubMed]
5. Turco, E.; Fracchiolla, D.; Martens, S. Recruitment and Activation of the ULK1/Atg1 Kinase Complex in Selective Autophagy. *J. Mol. Biol.* **2020**, *432*, 123–134. [CrossRef] [PubMed]
6. Park, J.M.; Seo, M.; Jung, C.H.; Grunwald, D.; Stone, M.; Otto, N.M.; Toso, E.; Ahn, Y.; Kyba, M.; Griffin, T.J.; et al. ULK1 phosphorylates Ser30 of BECN1 in association with ATG14 to stimulate autophagy induction. *Autophagy* **2018**, *14*, 584–597. [CrossRef] [PubMed]
7. Nascimbeni, A.C.; Codogno, P.; Morel, E. Phosphatidylinositol-3-phosphate in the regulation of autophagy membrane dynamics. *FEBS J.* **2017**, *284*, 1267–1278. [CrossRef]
8. Kim, B.W.; Jin, Y.; Kim, J.; Kim, J.H.; Jung, J.; Kang, S.; Kim, I.Y.; Kim, J.; Cheong, H.; Song, H.K. The C-terminal region of ATG101 bridges ULK1 and PtdIns3K complex in autophagy initiation. *Autophagy* **2018**, *14*, 2104–2116. [CrossRef]
9. Russell, R.C.; Tian, Y.; Yuan, H.; Park, H.W.; Chang, Y.Y.; Kim, J.; Kim, H.; Neufeld, T.P.; Dillin, A.; Guan, K.L. ULK1 induces autophagy by phosphorylating Beclin-1 and activating VPS34 lipid kinase. *Nat. Cell Biol.* **2013**, *15*, 741–750. [CrossRef]
10. Nascimbeni, A.C.; Giordano, F.; Dupont, N.; Grasso, D.; Vaccaro, M.I.; Codogno, P.; Morel, E. ER-plasma membrane contact sites contribute to autophagosome biogenesis by regulation of local PI3P synthesis. *EMBO J.* **2017**, *36*, 2018–2033. [CrossRef]
11. Molino, D.; Zemirli, N.; Codogno, P.; Morel, E. The Journey of the Autophagosome through Mammalian Cell Organelles and Membranes. *J. Mol. Biol.* **2017**, *429*, 497–514. [CrossRef] [PubMed]
12. Hamasaki, M.; Furuta, N.; Matsuda, A.; Nezu, A.; Yamamoto, A.; Fujita, N.; Oomori, H.; Noda, T.; Haraguchi, T.; Hiraoka, Y.; et al. Autophagosomes form at ER-mitochondria contact sites. *Nature* **2013**, *495*, 389–393. [CrossRef] [PubMed]
13. Puri, C.; Vicinanza, M.; Ashkenazi, A.; Gratian, M.J.; Zhang, Q.; Bento, C.F.; Renna, M.; Menzies, F.M.; Rubinsztein, D.C. The RAB11A-Positive Compartment Is a Primary Platform for Autophagosome Assembly Mediated by WIPI2 Recognition of PI3P-RAB11A. *Dev. Cell* **2018**, *45*, 114–131.e118. [CrossRef] [PubMed]
14. Dooley, H.C.; Razi, M.; Polson, H.E.; Girardin, S.E.; Wilson, M.I.; Tooze, S.A. WIPI2 links LC3 conjugation with PI3P, autophagosome formation, and pathogen clearance by recruiting Atg12-5-16L1. *Mol. Cell* **2014**, *55*, 238–252. [CrossRef]
15. Mizushima, N.; Sugita, H.; Yoshimori, T.; Ohsumi, Y. A new protein conjugation system in human. The counterpart of the yeast Apg12p conjugation system essential for autophagy. *J. Biol. Chem.* **1998**, *273*, 33889–33892. [CrossRef]

16. Lystad, A.H.; Carlsson, S.R.; de la Ballina, L.R.; Kauffman, K.J.; Nag, S.; Yoshimori, T.; Melia, T.J.; Simonsen, A. Distinct functions of ATG16L1 isoforms in membrane binding and LC3B lipidation in autophagy-related processes. *Nat. Cell Biol.* 2019, *21*, 372–383. [CrossRef]

17. Dudley, L.J.; Cabodevilla, A.G.; Makar, A.N.; Sztacho, M.; Michelberger, T.; Marsh, J.A.; Houston, D.R.; Martens, S.; Jiang, X.; Gammoh, N. Intrinsic lipid binding activity of ATG16L1 supports efficient membrane anchoring and autophagy. *EMBO J.* 2019, *38*, e1000554. [CrossRef]

18. Satoo, K.; Noda, N.N.; Kumeta, H.; Fujioka, Y.; Mizushima, N.; Ohsumi, Y.; Inagaki, F. The structure of Atg4B-LC3 complex reveals the mechanism of LC3 processing and delipidation during autophagy. *EMBO J.* 2009, *28*, 1341–1350. [CrossRef]

19. Maruyama, T.; Noda, N.N. Autophagy-regulating protease Atg4: Structure, function, regulation and inhibition. *J. Antibiot.* 2018, *71*, 72–78. [CrossRef]

20. Sakoh-Nakatogawa, M.; Matoba, K.; Asai, E.; Kirisako, H.; Ishii, J.; Noda, N.N.; Inagaki, F.; Nakatogawa, H.; Ohsumi, Y. Atg12-Atg5 conjugate enhances E2 activity of Atg3 by rearranging its catalytic site. *Nat. Struct. Mol. Biol.* 2013, *20*, 433–439. [CrossRef]

21. Martens, S.; Nakamura, S.; Yoshimori, T. Phospholipids in Autophagosome Formation and Fusion. *J. Mol. Biol.* 2016, *S0022-2836*, 30455–30457. [CrossRef] [PubMed]

22. Randall-Demllo, S.; Chieppa, M.; Eri, R. Intestinal epithelium and autophagy: Partners in gut homeostasis. *Front. Immunol.* 2013, *4*, 301. [CrossRef] [PubMed]

23. Popovic, D.; Dikic, I. TBC1D5 and the AP2 complex regulate ATG9 trafficking and initiation of autophagy. *EMBO Rep.* 2014, *15*, 392–401. [CrossRef] [PubMed]

24. Zhou, C.; Ma, K.; Gao, R.; Mu, C.; Chen, L.; Liu, Q.; Luo, Q.; Feng, D.; Zhu, Y.; Chen, Q. Regulation of mATG9 trafficking by Src- and ULK1-mediated phosphorylation in basal and starvation-induced autophagy. *Cell Res.* 2017, *27*, 184–201. [CrossRef]

25. Young, A.R.; Chan, E.Y.; Hu, X.W.; Kochl, R.; Crawshaw, S.G.; High, S.; Hailey, D.W.; Lippincott-Schwartz, J.; Tooze, S.A. Starvation and ULK1-dependent cycling of mammalian Atg9 between the TGN and endosomes. *J. Cell. Sci.* 2006, *119*, 3888–3900. [CrossRef]

26. Bento, C.F.; Puri, C.; Moreau, K.; Rubinsztein, D.C. The role of membrane-trafficking small GTPases in the regulation of autophagy. *J. Cell. Sci.* 2013, *126*, 1059–1069. [CrossRef]

27. Mari, M.; Griffith, J.; Rieter, E.; Krishnappa, L.; Klionsky, D.J.; Reggiori, F. An Atg9-containing compartment that functions in the early steps of autophagosome biogenesis. *J. Cell. Biol.* 2010, *190*, 1005–1022. [CrossRef]

28. Imai, K.; Hao, F.; Fujita, N.; Tsuji, Y.; Oe, Y.; Araki, Y.; Hamasaki, M.; Noda, T.; Yoshimori, T. Atg9A trafficking through the recycling endosomes is required for autophagosome formation. *J. Cell. Sci.* 2016, *129*, 3781–3791. [CrossRef]

29. Feng, Y.; Klionsky, D.J. Autophagic membrane delivery through ATG9. *Cell Res.* 2017, *27*, 161–162. [CrossRef]

30. Rieter, E.; Vinke, F.; Bakula, D.; Cebollero, E.; Ungermann, C.; Proikas-Cezanne, T.; Reggiori, F. Atg18 function in autophagy is regulated by specific sites within its beta-propeller. *J. Cell. Sci.* 2013, *126*, 593–604. [CrossRef]

31. Zheng, J.X.; Li, Y.; Ding, Y.H.; Liu, J.J.; Zhang, M.J.; Dong, M.Q.; Wang, H.W.; Yu, L. Architecture of the ATG2B-WDR45 complex and an aromatic Y/HF motif crucial for complex formation. *Autophagy* 2017, *13*, 1870–1883. [CrossRef] [PubMed]

32. Obara, K.; Sekito, T.; Niimi, K.; Ohsumi, Y. The Atg18-Atg2 complex is recruited to autophagic membranes via phosphatidylinositol 3-phosphate and exerts an essential function. *J. Biol. Chem.* 2008, *283*, 23972–23980. [CrossRef] [PubMed]

33. Chowdhury, S.; Otomo, C.; Leitner, A.; Ohashi, K.; Aebersold, R.; Lander, G.C.; Otomo, T. Insights into autophagosome biogenesis from structural and biochemical analyses of the ATG2A-WIPI4 complex. *Proc. Natl. Acad. Sci. USA* 2018, *115*, E9792–E9801. [CrossRef]

34. Bakula, D.; Muller, A.J.; Zuleger, T.; Takacs, Z.; Franz-Wachtel, M.; Thost, A.K.; Brigger, D.; Tschan, M.P.; Frickey, T.; Robenek, H.; et al. WIPI3 and WIPI4 beta-propellers are scaffolds for LKB1-AMPK-TSC signalling circuits in the control of autophagy. *Nat. Commun.* 2017, *8*, 15637. [CrossRef] [PubMed]

35. Velikkakath, A.K.; Nishimura, T.; Oita, E.; Ishihara, N.; Mizushima, N. Mammalian Atg2 proteins are essential for autophagosome formation and important for regulation of size and distribution of lipid droplets. *Mol. Biol. Cell* 2012, *23*, 896–909. [CrossRef]

36. Osawa, T.; Kotani, T.; Kawaoka, T.; Hirata, E.; Suzuki, K.; Nakatogawa, H.; Ohsumi, Y.; Noda, N.N. Atg2 mediates direct lipid transfer between membranes for autophagosome formation. *Nat. Struct. Mol. Biol.* 2019, 26, 281–288. [CrossRef]

37. Takahashi, Y.; He, H.; Tang, Z.; Hattori, T.; Liu, Y.; Young, M.M.; Serfass, J.M.; Chen, L.; Gebru, M.; Chen, C.; et al. An autophagy assay reveals the ESCRT-III component CHMP2A as a regulator of phagophore closure. *Nat. Commun.* 2018, 9, 2855. [CrossRef]

38. Lee, J.A.; Beigneux, A.; Ahmad, S.T.; Young, S.G.; Gao, F.B. ESCRT-III dysfunction causes autophagosome accumulation and neurodegeneration. *Curr. Biol.* 2007, 17, 1561–1567. [CrossRef]

39. Zhou, F.; Zou, S.; Chen, Y.; Lipatova, Z.; Sun, D.; Zhu, X.; Li, R.; Wu, Z.; You, W.; Cong, X.; et al. A Rab5 GTPase module is important for autophagosome closure. *PLoS Genet.* 2017, 13, e1007020. [CrossRef]

40. Kimura, S.; Noda, T.; Yoshimori, T. Dynein-dependent movement of autophagosomes mediates efficient encounters with Lysosomes. *Cell Struct. Funct.* 2008, 33, 109–122. [CrossRef]

41. Nakamura, S.; Yoshimori, T. New insights into autophagosome-lysosome fusion. *J. Cell. Sci.* 2017, 130, 1209–1216. [CrossRef] [PubMed]

42. Wong, Y.C.; Holzbaur, E.L. The regulation of autophagosome dynamics by huntingtin and HAP1 is disrupted by expression of mutant huntingtin, leading to defective cargo degradation. *J. Neurosci.* 2014, 34, 1293–1305. [CrossRef] [PubMed]

43. Cheng, X.T.; Zhou, B.; Lin, M.Y.; Cai, Q.; Sheng, Z.H. Axonal autophagosomes recruit dynein for retrograde transport through fusion with late endosomes. *J. Cell. Biol.* 2015, 209, 377–386. [CrossRef] [PubMed]

44. Maday, S.; Holzbaur, E.L. Autophagosome biogenesis in primary neurons follows an ordered and spatially regulated pathway. *Dev. Cell* 2014, 30, 71–85. [CrossRef]

45. Nian, F.S.; Li, L.L.; Cheng, C.Y.; Wu, P.C.; Lin, Y.T.; Tang, C.Y.; Ren, B.S.; Tai, C.Y.; Fann, M.J.; Kao, L.S.; et al. Rab18 Collaborates with Rab7 to Modulate Lysosomal and Autophagy Activities in the Nervous System: An Overlapping Mechanism for Warburg Micro Syndrome and Charcot-Marie-Tooth Neuropathy Type 2B. *Mol. Neurobiol.* 2019, 56, 6095–6105. [CrossRef]

46. Maruzs, T.; Lorincz, P.; Szatmari, Z.; Szeplaki, S.; Sandor, Z.; Lakatos, Z.; Puska, G.; Juhasz, G.; Sass, M. Retromer Ensures the Degradation of Autophagic Cargo by Maintaining Lysosome Function in Drosophila. *Traffic* 2015, 16, 1088–1107. [CrossRef]

47. Farfel-Becker, T.; Roney, J.C.; Cheng, X.T.; Li, S.; Cuddy, S.R.; Sheng, Z.H. Neuronal Soma-Derived Degradative Lysosomes Are Continuously Delivered to Distal Axons to Maintain Local Degradation Capacity. *Cell Rep.* 2019, 28, 51–64.e54. [CrossRef]

48. Matsui, T.; Jiang, P.; Nakano, S.; Sakamaki, Y.; Yamamoto, H.; Mizushima, N. Autophagosomal YKT6 is required for fusion with lysosomes independently of syntaxin 17. *J. Cell. Biol.* 2018, 217, 2633–2645. [CrossRef]

49. Itakura, E.; Kishi-Itakura, C.; Mizushima, N. The hairpin-type tail-anchored SNARE syntaxin 17 targets to autophagosomes for fusion with endosomes/lysosomes. *Cell* 2012, 151, 1256–1269. [CrossRef]

50. Yu, L.; McPhee, C.K.; Zheng, L.; Mardones, G.A.; Rong, Y.; Peng, J.; Mi, N.; Zhao, Y.; Liu, Z.; Wan, F.; et al. Termination of autophagy and reformation of lysosomes regulated by mTOR. *Nature* 2010, 465, 942–946. [CrossRef]

51. Rong, Y.; Liu, M.; Ma, L.; Du, W.; Zhang, H.; Tian, Y.; Cao, Z.; Li, Y.; Ren, H.; Zhang, C.; et al. Clathrin and phosphatidylinositol-4,5-bisphosphate regulate autophagic lysosome reformation. *Nat. Cell Biol.* 2012, 14, 924–934. [CrossRef] [PubMed]

52. Chen, Y.; Yu, L. Recent progress in autophagic lysosome reformation. *Traffic* 2017, 18, 358–361. [CrossRef] [PubMed]

53. Schulze, R.J.; Weller, S.G.; Schroeder, B.; Krueger, E.W.; Chi, S.; Casey, C.A.; McNiven, M.A. Lipid droplet breakdown requires dynamin 2 for vesiculation of autolysosomal tubules in hepatocytes. *J. Cell. Biol.* 2013, 203, 315–326. [CrossRef] [PubMed]

54. Mizushima, N.; Komatsu, M. Autophagy: Renovation of cells and tissues. *Cell* 2011, 147, 728–741. [CrossRef]

55. Kraft, C.; Peter, M.; Hofmann, K. Selective autophagy: Ubiquitin-mediated recognition and beyond. *Nat. Cell Biol.* 2010, 12, 836–841. [CrossRef]

56. Scrivo, A.; Bourdenx, M.; Pampliega, O.; Cuervo, A.M. Selective autophagy as a potential therapeutic target for neurodegenerative disorders. *Lancet. Neurol.* 2018, 17, 802–815. [CrossRef]

57. Deng, Z.; Purtell, K.; Lachance, V.; Wold, M.S.; Chen, S.; Yue, Z. Autophagy Receptors and Neurodegenerative Diseases. *Trends. Cell Biol.* 2017, 27, 491–504. [CrossRef]

58. Conway, O.; Akpinar, H.A.; Rogov, V.V.; Kirkin, V. Selective Autophagy Receptors in Neuronal Health and Disease. *J. Mol. Biol.* 2019, S0022-S2836, 2483–2509. [CrossRef]

59. Zaffagnini, G.; Savova, A.; Danieli, A.; Romanov, J.; Tremel, S.; Ebner, M.; Peterbauer, T.; Sztacho, M.; Trapannone, R.; Tarafder, A.K.; et al. p62 filaments capture and present ubiquitinated cargos for autophagy. *EMBO J.* 2018, 37, e98308. [CrossRef]

60. Isogai, S.; Morimoto, D.; Arita, K.; Unzai, S.; Tenno, T.; Hasegawa, J.; Sou, Y.S.; Komatsu, M.; Tanaka, K.; Shirakawa, M.; et al. Crystal structure of the ubiquitin-associated (UBA) domain of p62 and its interaction with ubiquitin. *J. Biol. Chem.* 2011, 286, 31864–31874. [CrossRef]

61. Walinda, E.; Morimoto, D.; Sugase, K.; Konuma, T.; Tochio, H.; Shirakawa, M. Solution structure of the ubiquitin-associated (UBA) domain of human autophagy receptor NBR1 and its interaction with ubiquitin and polyubiquitin. *J. Biol. Chem.* 2014, 289, 13890–13902. [CrossRef] [PubMed]

62. Suzuki, N.; Rohaim, A.; Kato, R.; Dikic, I.; Wakatsuki, S.; Kawasaki, M. A novel mode of ubiquitin recognition by the ubiquitin-binding zinc finger domain of WRNIP1. *FEBS J.* 2016, 283, 2004–2017. [CrossRef] [PubMed]

63. Birgisdottir, A.B.; Lamark, T.; Johansen, T. The LIR motif—crucial for selective autophagy. *J. Cell. Sci.* 2013, 126, 3237–3247. [CrossRef] [PubMed]

64. Rogov, V.V.; Stolz, A.; Ravichandran, A.C.; Rios-Szwed, D.O.; Suzuki, H.; Kniss, A.; Lohr, F.; Wakatsuki, S.; Dotsch, V.; Dikic, I.; et al. Structural and functional analysis of the GABARAP interaction motif (GIM). *EMBO Rep.* 2017, 18, 1382–1396. [CrossRef]

65. Caccamo, A.; Ferreira, E.; Branca, C.; Oddo, S. p62 improves AD-like pathology by increasing autophagy. *Mol. Psychiatry* 2017, 22, 865–873. [CrossRef]

66. Ramesh Babu, J.; Lamar Seibenhener, M.; Peng, J.; Strom, A.L.; Kemppainen, R.; Cox, N.; Zhu, H.; Wooten, M.C.; Diaz-Meco, M.T.; Moscat, J.; et al. Genetic inactivation of p62 leads to accumulation of hyperphosphorylated tau and neurodegeneration. *J. Neurochem.* 2008, 106, 107–120. [CrossRef]

67. Piras, A.; Collin, L.; Gruninger, F.; Graff, C.; Ronnback, A. Autophagic and lysosomal defects in human tauopathies: Analysis of post-mortem brain from patients with familial Alzheimer disease, corticobasal degeneration and progressive supranuclear palsy. *Acta Neuropathol. Commun.* 2016, 4. [CrossRef]

68. Collin, L.; Bohrmann, B.; Gopfert, U.; Oroszlan-Szovik, K.; Ozmen, L.; Gruninger, F. Neuronal uptake of tau/pS422 antibody and reduced progression of tau pathology in a mouse model of Alzheimer’s disease. *Brain* 2014, 137, 2834–2846. [CrossRef]

69. Xu, Y.; Zhang, S.; Zheng, H. The cargo receptor SQSTM1 ameliorates neurofibrillary tangle pathology and spreading through selective targeting of pathological MAPT (microtubule associated protein tau). *Autophagy* 2019, 15, 583–598. [CrossRef]

70. Jo, C.; Gundemir, S.; Pritchard, S.; Jin, Y.N.; Rahman, I.; Johnson, G.V. Nrf2 reduces levels of phosphorylated tau protein by inducing autophagy adaptor protein NDP52. *Nat. Commun.* 2014, 5, 3496. [CrossRef]

71. Xiao, Q.; Gil, S.C.; Yan, P.; Wang, Y.; Han, S.; Gonzales, E.; Perez, R.; Cirrito, J.R.; Lee, J.M. Role of phosphatidylinositol clathrin assembly lymphoid-myeloid leukemia (PICALM) in intracellular amyloid precursor protein (APP) processing and amyloid plaque pathogenesis. *J. Biol. Chem.* 2012, 287, 21279–21289. [CrossRef] [PubMed]

72. Tian, Y.; Chang, J.C.; Fan, E.Y.; Flajolet, M.; Greengard, P. Adaptor complex AP2/PICALM, through interaction with LC3, targets Alzheimer’s APP-CTF for terminal degradation via autophagy. *Proc. Natl. Acad. Sci. USA* 2013, 110, 17071–17076. [CrossRef] [PubMed]

73. Sato, S.; Uchihara, T.; Fukuda, T.; Noda, S.; Kondo, H.; Saiki, S.; Komatsu, M.; Uchiyama, Y.; Tanaka, K.; Hattori, N. Loss of autophagy in dopaminergic neurons causes Lewy pathology and motor dysfunction in aged mice. *Sci. Rep.* 2018, 8, 2813. [CrossRef] [PubMed]

74. Tanji, K.; Odagiri, S.; Miki, Y.; Maruyama, A.; Nikaido, Y.; Mimura, J.; Mori, F.; Warabi, E.; Yanagawa, T.; Ueno, S.; et al. p62 Deficiency Enhances alpha-Synuclein Pathology in Mice. *Brain Pathol.* 2015, 25, 552–564. [CrossRef]

75. Fu, Y.; Wu, P.; Pan, Y.; Sun, X.; Yang, H.; Difiglia, M.; Lu, B. A toxic mutant huntingtin species is resistant to selective autophagy. *Nat. Chem. Biol.* 2017, 13, 1152–1154. [CrossRef]

76. Filimonenko, M.; Isakson, P.; Finley, K.D.; Anderson, M.; Jeong, H.; Melia, T.J.; Bartlett, B.J.; Myers, K.M.; Birkeland, H.C.; Lamark, T.; et al. The selective macroautophagic degradation of aggregated proteins requires the PI3P-binding protein Alfy. *Mol. Cell* **2010**, *38*, 265–279. [CrossRef]

77. Korac, J.; Schaeffer, V.; Kovacevic, I.; Clement, A.M.; Jungblut, B.; Behl, C.; Terzic, J.; Dikic, I. Ubiquitin-independent function of optineurin in autophagic clearance of protein aggregates. *J. Cell. Sci.* **2013**, *126*, 580–592. [CrossRef]

78. Shen, W.C.; Li, H.Y.; Chen, G.C.; Chern, Y.; Tu, P.H. Mutations in the ubiquitin-binding domain of OPTN/optineurin interfere with autophagy-mediated degradation of misfolded proteins by a dominant-negative mechanism. *Autophagy* **2015**, *11*, 685–700. [CrossRef]

79. Fox, L.M.; Kim, K.; Johnson, C.W.; Chen, S.; Croce, K.R.; Victor, M.B.; Eenjes, E.; Bosco, J.R.; Randolph, L.K.; Dragatsis, I.; et al. Huntington’s Disease Pathogenesis Is Modified In Vivo by Alfy/Wdfy3 and Selective Macroautophagy. *Neuron* **2019**, *105*, 813–821.e816. [CrossRef]

80. Cykowski, M.D.; Powell, S.Z.; Appel, J.W.; Arumanayagam, A.S.; Rivera, A.L.; Appel, S.H. Phosphorylated TDP-43 (pTDP-43) aggregates in the axial skeletal muscle of patients with sporadic and familial amyotrophic lateral sclerosis. *Acta Neuropathol. Commun.* **2018**, *6*, 28. [CrossRef]

81. Brady, O.A.; Meng, P.; Zheng, Y.; Mao, Y.; Hu, F. Regulation of TDP-43 aggregation by phosphorylation and p62/SQSTM1. *J. Neurochem.* **2011**, *116*, 248–259. [CrossRef] [PubMed]

82. Hiji, M.; Takahashi, T.; Fukuba, H.; Yamashita, H.; Kohriyama, T.; Matsumoto, M. White matter lesions in the brain with frontotemporal lobar degeneration with motor neuron disease: TDP-43-immunopositive inclusions co-localize with p62, but not ubiquitin. *Acta Neuropathol.* **2008**, *116*, 183–191. [CrossRef] [PubMed]

83. Hadano, S.; Mitsui, S.; Pan, L.; Otomo, A.; Kubo, M.; Sato, K.; Ono, S.; Onodera, W.; Abe, K.; Chen, X.; et al. Functional links between SQSTM1 and ALS2 in the pathogenesis of ALS: Cumulative impact on the protection against mutant SOD1-mediated motor dysfunction in mice. *Hum. Mol. Genet.* **2016**, *25*, 3321–3340. [CrossRef] [PubMed]

84. Gal, J.; Strom, A.L.; Kwinter, D.M.; Kilty, R.; Zhang, J.; Shi, P.; Fu, W.; Wooten, M.W.; Zhu, H. Sequestosome 1/p62 links familial ALS mutant SOD1 to LC3 via an ubiquitin-independent mechanism. *J. Neurochem.* **2009**, *111*, 1062–1073. [CrossRef]

85. Wu, Q.; Liu, M.; Huang, C.; Liu, X.; Huang, B.; Li, N.; Zhou, H.; Xia, X.G. Pathogenic UbqIn2 gains toxic properties to induce neuron death. *Acta Neuropathol.* **2015**, *129*, 417–428. [CrossRef]

86. Chen, T.; Huang, B.; Shi, X.; Gao, L.; Huang, C. Mutant UBQLN2(P497H) in motor neurons leads to ALS-like phenotypes and defective autophagy in rats. *Acta Neuropathol. Commun.* **2018**, *6*, 122. [CrossRef]

87. Maruyama, H.; Morino, H.; Ito, H.; Izumi, Y.; Kato, H.; Watanabe, Y.; Kinoshita, Y.; Kamada, M.; Nodera, H.; Suzuki, H.; et al. Mutations of optineurin in amyotrophic lateral sclerosis. *Nature* **2010**, *465*, 223–226. [CrossRef]

88. Ito, Y.; Ofengeim, D.; Najafov, A.; Das, S.; Saberi, S.; Li, Y.; Hitomi, J.; Zhu, H.; Chen, H.; Mayo, L.; et al. RIPK1 mediates axonal degeneration by promoting inflammation and necroptosis in ALS. *Science* **2016**, *353*, 603–608. [CrossRef]

89. Jantrapirom, S.; Lo Piccolo, L.; Yoshida, H.; Yamaguchi, M. Depletion of Ubiquilin induces an augmentation in soluble ubiquitinated Drosophila TDP-43 to drive neurotoxicity in the fly. *Biochim. Biophys. Acta Mol. Basis. Dis.* **2018**, *1864*, 3038–3049. [CrossRef]

90. Yap, C.C.; Winckler, B. Harnessing the power of the endosome to regulate neural development. *Neuron* **2012**, *74*, 440–451. [CrossRef]

91. Nixon, R.A. The role of autophagy in neurodegenerative disease. *Nat. Med.* **2013**, *19*, 983–997. [CrossRef] [PubMed]

92. Kuma, A.; Komatsu, M.; Mizushima, N. Autophagy-monitoring and autophagy-deficient mice. *Autophagy* **2017**, *13*, 1619–1628. [CrossRef] [PubMed]

93. Levine, B.; Kroemer, G. Autophagy in the pathogenesis of disease. *Cell* **2008**, *132*, 27–42. [CrossRef] [PubMed]

94. Hara, T.; Nakamura, K.; Matsui, M.; Yamamoto, A.; Nakahara, Y.; Suzuki-Migishima, R.; Yokoyama, M.; Mishima, K.; Saito, I.; Okano, H.; et al. Suppression of basal autophagy in neural cells causes neurodegenerative disease in mice. *Nature* **2006**, *441*, 885–889. [CrossRef]

95. Komatsu, M.; Waguri, S.; Chiba, T.; Murata, S.; Iwata, J.; Tanida, I.; Ueno, T.; Koike, M.; Uchiyama, Y.; Kominami, E.; et al. Loss of autophagy in the central nervous system causes neurodegeneration in mice. *Nature* **2006**, *441*, 880–884. [CrossRef]

96. Komatsu, M.; Wang, Q.J.; Holstein, G.R.; Friedrich, V.L., Jr.; Iwata, J.-I.; Kominami, E.; Chait, B.T.; Tanaka, K.; Yue, Z. Essential role for autophagy protein Atg7 in the maintenance of axonal homeostasis and the prevention of axonal degeneration. *Proc. Natl. Acad. Sci. USA* **2007**, *104*, 14489–14494. [CrossRef]

97. Nishiyama, J.; Miura, E.; Mizushima, N.; Watanabe, M.; Yuzaki, M. Aberrant membranes and double-membrane structures accumulate in the axons of Atg5-null Purkinje cells before neuronal death. *Autophagy* **2007**, *3*, 591–596. [CrossRef]

98. Liang, C.C.; Wang, C.; Peng, X.; Gan, B.; Guan, J.L. Neural-specific deletion of FIP200 leads to cerebellar degeneration caused by increased neuronal death and axon degeneration. *J. Biol. Chem.* **2010**, *285*, 3499–3509. [CrossRef]

99. Lattante, S.; de Calbiac, H.; Le Ber, I.; Brice, A.; Ciura, S.; Kabashi, E. Sqstm1 knock-down causes a locomotor phenotype ameliorated by rapamycin in a zebrafish model of ALS/FTLD. *Hum. Mol. Genet.* **2015**, *24*, 1682–1690. [CrossRef]

100. Wertz, M.H.; Mitchem, M.R.; Pineda, S.S.; Hachigian, L.J.; Lee, H.; Lau, V.; Powers, A.; Kulicke, R.; Madan, G.K.; Colic, M.; et al. Genome-wide In Vivo CNS Screening Identifies Genes that Modify CNS Neuronal Survival and mHTT Toxicity. *Neuron* **2020**, *S0896-S6273*, 76–89. [CrossRef]

101. Menzies, F.M.; Fleming, A.; Caricasole, A.; Bento, C.F.; Andrews, S.P.; Ashkenazi, A.; Fullgrabe, J.; Jackson, A.; Jimenez Sanchez, M.; Karabiyik, C.; et al. Autophagy and Neurodegeneration: Pathogenic Mechanisms and Therapeutic Opportunities. *Neuron* **2017**, *93*, 1015–1034. [CrossRef] [PubMed]

102. Frake, R.A.; Ricketts, T.; Menzies, F.M.; Rubinsztein, D.C. Autophagy and neurodegeneration. *J. Clin. Investig.* **2015**, *125*, 65–74. [CrossRef] [PubMed]

103. Stavoe, A.K.; Gopal, P.P.; Gubas, A.; Tooze, S.A.; Holzbaur, E.L. Expression of WIPI2B counteracts age-related decline in autophagosome biogenesis in neurons. *Elife* **2019**, *8*, e44219. [CrossRef] [PubMed]

104. Spilman, P.; Podlutskaya, N.; Hart, M.J.; Debnath, J.; Gorostiza, O.; Bredesen, D.; Richardson, A.; Strong, R.; Galvan, V. Inhibition of mTOR by rapamycin abolishes cognitive deficits and reduces amyloid-beta levels in a mouse model of Alzheimer’s disease. *PLoS ONE* **2010**, *5*, e9979. [CrossRef]

105. Berger, Z.; Ravikumar, B.; Menzies, F.M.; Oroz, L.G.; Underwood, B.R.; Panganos, M.N.; Schmitt, I.; Wullner, U.; Evert, B.O.; O’Kane, C.J.; et al. Rapamycin alleviates toxicity of different aggregate-prone proteins. *Hum. Mol. Genet.* **2006**, *15*, 433–442. [CrossRef]

106. Ravikumar, B.; Duden, R.; Rubinsztein, D.C. Aggregate-prone proteins with polyglutamine and polyalanine expansions are degraded by autophagy. *Hum. Mol. Genet.* **2002**, *11*, 1107–1117. [CrossRef]

107. Tsvetkov, A.S.; Arrasate, M.; Barmada, S.; Ando, D.M.; Sharma, P.; Shaby, B.A.; Finkbeiner, S. Proteostasis of polyglutamine varies among neurons and predicts neurodegeneration. *Nat. Chem. Biol.* **2013**, *9*, 586–592. [CrossRef]

108. Choy, R.W.; Cheng, Z.; Schekman, R. Amyloid precursor protein (APP) traffics from the cell surface via endosomes for amyloid beta (Abeta) production in the trans-Golgi network. *Proc. Natl. Acad. Sci. USA* **2012**, *109*, E2077–E2082. [CrossRef]

109. Zhang, T.; Chen, D.; Lee, T.H. Phosphorylation Signaling in APP Processing in Alzheimer’s Disease. *Int. J. Mol. Sci.* **2020**, *21*, 209. [CrossRef]

110. Lee, S.; Mankhong, S.; Kang, J.H. Extracellular Vesicle as a Source of Alzheimer’s Biomarkers: Opportunities and Challenges. *Int. J. Mol. Sci.* **2019**, *20*, 1728. [CrossRef]

111. Uddin, M.S.; Stachowiak, A.; Mamun, A.A.; Tzvetkov, N.T.; Takeda, S.; Atanasov, A.G.; Bergantin, L.B.; Abdel-Daim, M.M.; Stankiewicz, A.M. Autophagy and Alzheimer’s Disease: From Molecular Mechanisms to Therapeutic Implications. *Front. Aging Neurosci.* **2018**, *10*, 4. [CrossRef] [PubMed]

112. Ji, X.R.; Cheng, K.C.; Chen, Y.R.; Lin, T.Y.; Cheung, C.H.A.; Wu, C.L.; Chiang, H.C. Dysfunction of different cellular degradation pathways contributes to specific beta-amyloid42-induced pathologies. *FASEB J.* **2018**, *32*, 1375–1387. [CrossRef] [PubMed]

113. Song, J.X.; Malampati, S.; Zeng, Y.; Durairajan, S.S.K.; Yang, C.B.; Tong, B.C.; Iyaswamy, A.; Shang, W.B.; Sreenivasmurthy, S.G.; Zhu, Z.; et al. A small molecule transcription factor EB activator ameliorates beta-amyloid precursor protein and Tau pathology in Alzheimer’s disease models. *Aging Cell* **2020**, *19*, e13069. [CrossRef] [PubMed]

114. Ling, D.; Song, H.J.; Garza, D.; Neufeld, T.P.; Salvaterra, P.M. Abeta42-induced neurodegeneration via an age-dependent autophagic-lysosomal injury in Drosophila. *PLoS ONE* **2009**, *4*, e4201. [CrossRef]

115. Tammineni, P.; Ye, X.; Feng, T.; Aikal, D.; Cai, Q. Impaired retrograde transport of axonal autophagosomes contributes to autophagic stress in Alzheimer’s disease neurons. *eLife* 2017, 6, e21776. [CrossRef]

116. Nixon, R.A.; Wegiel, J.; Kumar, A.; Haung, W.; Peterhoff, C.; Cataldo, A.; Cuervo, A.M. Extensive involvement of autophagy in Alzheimer disease: An immuno electron microscopy study. *J. Neuropathol. Exp. Neurol.* 2005, 64, 113–122. [CrossRef]

117. Nixon, R.A.; Yang, D.S. Autophagy failure in Alzheimer’s disease—locating the primary defect. *Neurobiol. Dis.* 2011, 43, 38–45. [CrossRef]

118. Boland, B.; Kumar, A.; Lee, S.; Platt, F.M.; Wegiel, J.; Yu, W.H.; Nixon, R.A. Autophagy induction and autophagosome clearance in neurons: Relationship to autophagic pathology in Alzheimer’s disease. *J. Neurosci.* 2008, 28, 6926–6937. [CrossRef]

119. Manczak, M.; Kandimalla, R.; Yin, X.; Reddy, P.H. Hippocampal mutant APP and amyloid beta-induced cognitive decline, dendritic spine loss, defective autophagy, mitophagy and mitochondrial abnormalities in a mouse model of Alzheimer’s disease. *Hum. Mol. Genet.* 2018, 27, 1332–1342. [CrossRef]

120. Castellazzi, M.; Patergnani, S.; Donadio, M.; Giorgi, C.; Bonora, M.; Bosi, C.; Brombo, G.; Pugliatti, M.; Seripa, D.; Zuliani, G.; et al. Autophagy and mitophagy biomarkers are reduced in sera of patients with Alzheimer’s disease and mild cognitive impairment. *Sci. Rep.* 2019, 9, 20009. [CrossRef]

121. Cho, S.J.; Lim, H.J.; Jo, C.; Park, M.H.; Han, C.; Koh, Y.H. Plasma ATG5 is increased in Alzheimer’s disease. *Sci. Rep.* 2019, 9, 4741. [CrossRef] [PubMed]

122. De Strooper, B.; Saftig, P.; Craessaerts, K.; Vanderstichele, H.; Guhde, G.; Annaert, W.; Von Figura, K.; Van Leuven, F. Deficiency of presenilin-1 inhibits the normal cleavage of amyloid precursor protein. *Nature* 1998, 391, 387–390. [CrossRef] [PubMed]

123. Cacace, R.; Sleegers, K.; Van Broeckhoven, C. Molecular genetics of early-onset Alzheimer’s disease revisited. *Alzheimers Dement.* 2016, 12, 733–748. [CrossRef] [PubMed]

124. Lee, J.H.; McBrayer, M.K.; Wolfe, D.M.; Haslett, L.J.; Kumar, A.; Sato, Y.; Lie, P.P.; Mohan, P.; Coffey, E.E.; Kompella, U.; et al. Presenilin 1 Maintains Lysosomal Ca(2+) Homeostasis via TRPML1 by Regulating vATPase-Mediated Lysosome Acidification. *Cell Rep.* 2015, 12, 1430–1444. [CrossRef]

125. Chong, C.M.; Ke, M.; Tan, Y.; Huang, Z.; Zhang, K.; Ai, N.; Ge, W.; Qin, D.; Lu, J.H.; Su, H. Presenilin 1 deficiency suppresses autophagy in human neural stem cells through reducing gamma-secretase-independent ERK/CREB signaling. *Cell Death Dis.* 2018, 9, 879. [CrossRef]

126. Lee, J.H.; Yu, W.H.; Kumar, A.; Lee, S.; Mohan, P.S.; Peterhoff, C.M.; Wolfe, D.M.; Martinez-Vicente, M.; Massey, A.C.; Sovak, G.; et al. Lysosomal proteolysis and autophagy require presenilin 1 and are disrupted by Alzheimer-related PS1 mutations. *Cell* 2010, 141, 1146–1158. [CrossRef]

127. Wolfe, D.M.; Lee, J.H.; Kumar, A.; Lee, S.; Orenstein, S.J.; Nixon, R.A. Autophagy failure in Alzheimer’s disease and the role of defective lysosomal acidification. *Eur. J. Neurosci.* 2013, 37, 1949–1961. [CrossRef]

128. Coffey, E.E.; Beckel, J.M.; Laties, A.M.; Mitchell, C.H. Lysosomal alkalization and dysfunction in human fibroblasts with the Alzheimer’s disease-linked presenilin 1 A246E mutation can be reversed with cAMP. *Neuroscience* 2014, 263, 111–124. [CrossRef]

129. Xu, W.; Tan, L.; Yu, J.T. The Role of PICALM in Alzheimer’s Disease. *Mol. Neurobiol.* 2015, 52, 399–413. [CrossRef]

130. Ando, K.; Brion, J.P.; Stygelbout, V.; Suain, V.; Authélet, M.; Dedecker, R.; Chanut, A.; Lacor, P.; Lavaur, J.; Sazdovitch, V.; et al. Clathrin adaptor CALM/PICALM is associated with neurofibrillary tangles and is cleaved in Alzheimer’s brains. *Acta Neuropathol.* 2013, 125, 861–878. [CrossRef]

131. Moreau, K.; Fleming, A.; Imarisio, S.; Lopez Ramirez, A.; Mercer, J.L.; Jimenez-Sanchez, M.; Bento, C.F.; Puri, C.; Zavodszky, E.; Siddiqi, F.; et al. PICALM modulates autophagy activity and tau accumulation. *Nat. Commun.* 2014, 5, 4998. [CrossRef] [PubMed]

132. Lucin, K.M.; O’Brien, C.E.; Bieri, G.; Czirr, E.; Mosher, K.I.; Abbey, R.J.; Mastroeni, D.F.; Rogers, J.; Spencer, B.; Masliah, E.; et al. Microglial beclin 1 regulates retromer trafficking and phagocytosis and is impaired in Alzheimer’s disease. *Neuron* 2013, 79, 873–886. [CrossRef] [PubMed]

133. Pickford, F.; Masliah, E.; Britschgi, M.; Lucin, K.; Narasimhan, R.; Jaeger, P.A.; Small, S.; Spencer, B.; Rockenstein, E.; Levine, B.; et al. The autophagy-related protein beclin 1 shows reduced expression in early Alzheimer disease and regulates amyloid beta accumulation in mice. *J. Clin. Investig.* 2008, 118, 2190–2199. [CrossRef] [PubMed]

134. Swaminathan, G.; Zhu, W.; Plowey, E.D. BECN1/Beclin 1 sorts cell-surface APP/amyloid beta precursor protein for lysosomal degradation. *Autophagy* **2016**, *12*, 2404–2419. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

135. Small, S.A.; Kent, K.; Pierce, A.; Leung, C.; Kang, M.S.; Okada, H.; Honig, L.; Vonsattel, J.P.; Kim, T.W. Model-guided microarray implicates the retromer complex in Alzheimer’s disease. *Ann. Neurol.* **2005**, *58*, 909–919. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

136. Dolan, P.J.; Johnson, G.V. A caspase cleaved form of tau is preferentially degraded through the autophagy pathway. *J. Biol. Chem.* **2010**, *285*, 21978–21987. [CrossRef](https://doi.org/CrossRef)

137. Butzlaff, M.; Hannan, S.B.; Karsten, P.; Lenz, S.; Ng, J.; Vossfeldt, H.; Prussing, K.; Pflanz, R.; Schulz, J.B.; Rasse, T.; et al. Impaired retrograde transport by the Dynein/Dynactin complex contributes to Tau-induced toxicity. *Hum. Mol. Genet.* **2015**, *24*, 3623–3637. [CrossRef](https://doi.org/CrossRef)

138. Krüger, U.; Wang, Y.; Kumar, S.; Mandelkow, E.M. Autophagic degradation of tau in primary neurons and its enhancement by trehalose. *Neurobiol. Aging* **2012**, *33*, 2291–2305. [CrossRef](https://doi.org/CrossRef)

139. Wang, Y.; Martinez-Vicente, M.; Kruger, U.; Kaushik, S.; Wong, E.; Mandelkow, E.M.; Cuervo, A.M.; Mandelkow, E. Tau fragmentation, aggregation and clearance: The dual role of lysosomal processing. *Hum. Mol. Genet.* **2009**, *18*, 4153–4170. [CrossRef](https://doi.org/CrossRef)

140. Singleton, A.B.; Farrer, M.; Johnson, J.; Singleton, A.; Hague, S.; Kachergus, J.; Hulihan, M.; Peuralinna, T.; Dutra, A.; Nussbaum, R.; et al. alpha-Synuclein locus triplication causes Parkinson’s disease. *Science* **2003**, *302*, 841. [CrossRef](https://doi.org/CrossRef)

141. Sarkar, S.; Davies, J.E.; Huang, Z.; Tunacliffe, A.; Rubinsztein, D.C. Trehalose, a novel mTOR-independent autophagy enhancer, accelerates the clearance of mutant huntingtin and alpha-synuclein. *J. Biol. Chem.* **2007**, *282*, 5641–5652. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

142. Yan, J.Q.; Yuan, Y.H.; Chu, S.F.; Li, G.H.; Chen, N.H. E46K Mutant alpha-Synuclein Is Degraded by Both Proteasome and Macroautophagy Pathway. *Molecules* **2018**, *23*, 2839. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

143. Vogiatzi, T.; Xilouri, M.; Vekrellis, K.; Stefanis, L. Wild type alpha-synuclein is degraded by chaperone-mediated autophagy and macroautophagy in neuronal cells. *J. Biol. Chem.* **2008**, *283*, 23542–23556. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

144. Webb, J.L.; Ravikumar, B.; Atkins, J.; Skepper, J.N.; Rubinsztein, D.C. Alpha-Synuclein is degraded by both autophagy and the proteasome. *J. Biol. Chem.* **2003**, *278*, 25009–25013. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

145. Winslow, A.R.; Chen, C.W.; Corrochano, S.; Acevedo-Arozena, A.; Gordon, D.E.; Peden, A.A.; Lichtenberg, M.; Menzies, F.M.; Ravikumar, B.; Imarisio, S.; et al. α-Synuclein impairs macroautophagy: Implications for Parkinson’s disease. *J. Cell. Biol.* **2010**, *190*, 1023–1037. [CrossRef](https://doi.org/CrossRef)

146. Tanik, S.A.; Schultheiss, C.E.; Volpicelli-Daley, L.A.; Brunden, K.R.; Lee, V.M. Lewy body-like alpha-synuclein aggregates resist degradation and impair macroautophagy. *J. Biol. Chem.* **2013**, *288*, 15194–15210. [CrossRef](https://doi.org/CrossRef)

147. Volpicelli-Daley, L.A.; Gamble, K.L.; Schultheiss, C.E.; Riddle, D.M.; West, A.B.; Lee, V.M. Formation of alpha-synuclein Lewy neurite-like aggregates in axons impedes the transport of distinct endosomes. *Mol. Biol. Cell* **2014**, *25*, 4010–4023. [CrossRef](https://doi.org/CrossRef)

148. Hoffmann, A.C.; Minakaki, G.; Menges, S.; Salvi, R.; Savitskiy, S.; Kazman, A.; Vicente Miranda, H.; Mielenz, D.; Klucken, J.; Winkler, J.; et al. Extracellular aggregated alpha synuclein primarily triggers lysosomal dysfunction in neural cells prevented by trehalose. *Sci. Rep.* **2019**, *9*, 544. [CrossRef](https://doi.org/CrossRef)

149. Dinter, E.; Saridaki, T.; Nippold, M.; Plum, S.; Diederichs, L.; Komnig, D.; Fensky, L.; May, C.; Marcus, K.; Voigt, A.; et al. Rab7 induces clearance of alpha-synuclein aggregates. *J. Neurochem.* **2016**, *138*, 758–774. [CrossRef](https://doi.org/CrossRef)

150. Moors, T.E.; Paciotti, S.; Ingrassia, A.; Quadri, M.; Breedveld, G.; Tasegian, A.; Chiasserini, D.; Eusebi, P.; Duran-Pacheco, G.; Kremer, T.; et al. Characterization of Brain Lysosomal Activities in GBA-Related and Sporadic Parkinson’s Disease and Dementia with Lewy Bodies. *Mol. Neurobiol.* **2019**, *56*, 1344–1355. [CrossRef](https://doi.org/CrossRef)

151. Decressac, M.; Mattsson, B.; Weikop, P.; Lundblad, M.; Jakobsson, J.; Bjorklund, A. TFEB-mediated autophagy rescues midbrain dopamine neurons from alpha-synuclein toxicity. *Proc. Natl. Acad. Sci. USA* **2013**, *110*, E1817–E1826. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

152. Lei, Z.; Cao, G.; Wei, G. A30P mutant alpha-synuclein impairs autophagic flux by inactivating JNK signaling to enhance ZKSCAN3 activity in midbrain dopaminergic neurons. *Cell Death Dis.* **2019**, *10*, 133. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

153. Arotcarena, M.L.; Bourdenx, M.; Dutheil, N.; Thiolat, M.L.; Doudnikoff, E.; Dovero, S.; Ballabio, A.; Fernagut, P.O.; Meissner, W.G.; Bezard, E.; et al. Transcription factor EB overexpression prevents neurodegeneration in experimental synucleinopathies. *JCI Insight* **2019**, *4*, 129719. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

154. Torra, A.; Parent, A.; Cuadros, T.; Rodriguez-Galvan, B.; Ruiz-Bronchal, E.; Ballabio, A.; Bortolozzi, A.; Vila, M.; Bove, J. Overexpression of TFEB Drives a Pleiotropic Neurotrophic Effect and Prevents Parkinson’s Disease-Related Neurodegeneration. *Mol. Ther.* **2018**, *26*, 1552–1567. [CrossRef](https://doi.org/CrossRef)

155. Garcia-Sanz, P.; Orgaz, L.; Bueno-Gil, G.; Espadas, I.; Rodriguez-Traver, E.; Kulisevsky, J.; Gutierrez, A.; Davila, J.C.; Gonzalez-Polo, R.A.; Fuentes, J.M.; et al. N370S-GBA1 mutation causes lysosomal cholesterol accumulation in Parkinson’s disease. *Mov. Disord.* **2017**, *32*, 1409–1422. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

156. Sidransky, E.; Lopez, G. The link between the GBA gene and parkinsonism. *Lancet. Neurol.* **2012**, *11*, 986–998. [CrossRef](https://doi.org/CrossRef)

157. Velayati, A.; Yu, W.H.; Sidransky, E. The role of glucocerebrosidase mutations in Parkinson disease and Lewy body disorders. *Curr. Neurol. Neurosci. Rep.* **2010**, *10*, 190–198. [CrossRef](https://doi.org/CrossRef)

158. Parnetti, L.; Chiasserini, D.; Persichetti, E.; Eusebi, P.; Varghese, S.; Qureshi, M.M.; Dardis, A.; Deganuto, M.; De Carlo, C.; Castrioto, A.; et al. Cerebrospinal fluid lysosomal enzymes and alpha-synuclein in Parkinson’s disease. *Mov. Disord.* **2014**, *29*, 1019–1027. [CrossRef](https://doi.org/CrossRef)

159. Murphy, K.E.; Gysbers, A.M.; Abbott, S.K.; Tayebi, N.; Kim, W.S.; Sidransky, E.; Cooper, A.; Garner, B.; Halliday, G.M. Reduced glucocerebrosidase is associated with increased alpha-synuclein in sporadic Parkinson’s disease. *Brain* **2014**, *137*, 834–848. [CrossRef](https://doi.org/CrossRef)

160. Manning-Bog, A.B.; Schule, B.; Langston, J.W. Alpha-synuclein-glucocerebrosidase interactions in pharmacological Gaucher models: A biological link between Gaucher disease and parkinsonism. *Neurotoxicology* **2009**, *30*, 1127–1132. [CrossRef](https://doi.org/CrossRef)

161. Mazzulli, J.R.; Xu, Y.H.; Sun, Y.; Knight, A.L.; McLean, P.J.; Caldwell, G.A.; Sidransky, E.; Grabowski, G.A.; Krainc, D. Gaucher disease glucocerebrosidase and alpha-synuclein form a bidirectional pathogenic loop in synucleinopathies. *Cell* **2011**, *146*, 37–52. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

162. Yap, T.L.; Velayati, A.; Sidransky, E.; Lee, J.C. Membrane-bound α-synuclein interacts with glucocerebrosidase and inhibits enzyme activity. *Mol. Genet. Metab.* **2013**, *108*, 56–64. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

163. Esteves, A.R.; Cardoso, S.M. LRRK2 at the Crossroad Between Autophagy and Microtubule Trafficking: Insights into Parkinson’s Disease. *Neuroscientist* **2017**, *23*, 16–26. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

164. Tong, Y.; Yamaguchi, H.; Giaime, E.; Boyle, S.; Kopan, R.; Kelleher, R.J., 3rd; Shen, J. Loss of leucine-rich repeat kinase 2 causes impairment of protein degradation pathways, accumulation of alpha-synuclein, and apoptotic cell death in aged mice. *Proc. Natl. Acad. Sci. USA* **2010**, *107*, 9879–9884. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

165. Dodson, M.W.; Leung, L.K.; Lone, M.; Lizzio, M.A.; Guo, M. Novel ethyl methanesulfonate (EMS)-induced null alleles of the Drosophila homolog of LRRK2 reveal a crucial role in endolysosomal functions and autophagy in vivo. *Dis. Model. Mech.* **2014**, *7*, 1351–1363. [CrossRef](https://doi.org/CrossRef)

166. Kett, L.R.; Dauer, W.T. Leucine-rich repeat kinase 2 for beginners: Six key questions. *Cold Spring Harb. Perspect. Med.* **2012**, *2*, a009407. [CrossRef](https://doi.org/CrossRef)

167. Alegre-Abrarategui, J.; Christian, H.; Lufino, M.M.; Mutihac, R.; Venda, L.L.; Ansorge, O.; Wade-Martins, R. LRRK2 regulates autophagic activity and localizes to specific membrane microdomains in a novel human genomic reporter cellular model. *Hum. Mol. Genet.* **2009**, *18*, 4022–4034. [CrossRef](https://doi.org/CrossRef)

168. Plowey, E.D.; Cherra, S.J., 3rd; Liu, Y.J.; Chu, C.T. Role of autophagy in G2019S-LRRK2-associated neurite shortening in differentiated SH-SY5Y cells. *J. Neurochem.* **2008**, *105*, 1048–1056. [CrossRef](https://doi.org/CrossRef)

169. Ramonet, D.; Daher, J.P.; Lin, B.M.; Stafa, K.; Kim, J.; Banerjee, R.; Westerlund, M.; Pletnikova, O.; Glauser, L.; Yang, L.; et al. Dopaminergic neuronal loss, reduced neurite complexity and autophagic abnormalities in transgenic mice expressing G2019S mutant LRRK2. *PLoS ONE* **2011**, *6*, e18568. [CrossRef](https://doi.org/CrossRef)

170. Sanchez-Danes, A.; Richaud-Patin, Y.; Carballo-Carbajal, I.; Jimenez-Delgado, S.; Caig, C.; Mora, S.; Di Guglielmo, C.; Ezquerra, M.; Patel, B.; Giralt, A.; et al. Disease-specific phenotypes in dopamine neurons from human iPS-based models of genetic and sporadic Parkinson’s disease. *EMBO Mol. Med.* **2012**, *4*, 380–395. [CrossRef](https://doi.org/CrossRef)

171. Gomez-Suaga, P.; Rivero-Rios, P.; Fdez, E.; Blanca Ramirez, M.; Ferrer, I.; Aiastui, A.; Lopez De Munain, A.; Hilfiker, S. LRRK2 delays degradative receptor trafficking by impeding late endosomal budding through decreasing Rab7 activity. *Hum. Mol. Genet.* **2014**, *23*, 6779–6796. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

172. Wallings, R.; Connor-Robson, N.; Wade-Martins, R. LRRK2 interacts with the vacuolar-type H+-ATPase pump a1 subunit to regulate lysosomal function. *Hum. Mol. Genet.* **2019**, *28*, 2696–2710. [CrossRef](https://doi.org/CrossRef) [PubMed](https://pubmed.ncbi.nlm.nih.gov/PubMed)

173. Ramirez, A.; Heimbach, A.; Grundemann, J.; Stiller, B.; Hampshire, D.; Cid, L.P.; Goebel, I.; Mubaidin, A.F.; Wriekat, A.L.; Roeper, J.; et al. Hereditary parkinsonism with dementia is caused by mutations in ATP13A2, encoding a lysosomal type 5 P-type ATPase. *Nat. Genet.* **2006**, *38*, 1184–1191. [CrossRef](https://doi.org/10.1038/ng1884) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16906163)

174. Dehay, B.; Ramirez, A.; Martinez-Vicente, M.; Perier, C.; Canron, M.H.; Doudnikoff, E.; Vital, A.; Vila, M.; Klein, C.; Bezard, E. Loss of P-type ATPase ATP13A2/PARK9 function induces general lysosomal deficiency and leads to Parkinson disease neurodegeneration. *Proc. Natl. Acad. Sci. USA* **2012**, *109*, 9611–9616. [CrossRef](https://doi.org/10.1073/pnas.1206387109)

175. Bento, C.F.; Ashkenazi, A.; Jimenez-Sanchez, M.; Rubinsztein, D.C. The Parkinson’s disease-associated genes ATP13A2 and SYT11 regulate autophagy via a common pathway. *Nat. Commun.* **2016**, *7*, 11803. [CrossRef](https://doi.org/10.1038/ncomms11803)

176. Wang, R.; Tan, J.; Chen, T.; Han, H.; Tian, R.; Tan, Y.; Wu, Y.; Cui, J.; Chen, F.; Li, J.; et al. ATP13A2 facilitates HDAC6 recruitment to lysosome to promote autophagosome-lysosome fusion. *J. Cell. Biol.* **2019**, *218*, 267–284. [CrossRef](https://doi.org/10.1083/jcb.201809057)

177. Usenovic, M.; Tresse, E.; Mazzulli, J.R.; Taylor, J.P.; Krainc, D. Deficiency of ATP13A2 leads to lysosomal dysfunction, alpha-synuclein accumulation, and neurotoxicity. *J. Neurosci.* **2012**, *32*, 4240–4246. [CrossRef](https://doi.org/10.1523/JNEUROSCI.5918-11.2012)

178. Miura, E.; Hasegawa, T.; Konno, M.; Suzuki, M.; Sugeno, N.; Fujikake, N.; Geisler, S.; Tabuchi, M.; Oshima, R.; Kikuchi, A.; et al. VPS35 dysfunction impairs lysosomal degradation of alpha-synuclein and exacerbates neurotoxicity in a Drosophila model of Parkinson’s disease. *Neurobiol. Dis.* **2014**, *71*, 1–13. [CrossRef](https://doi.org/10.1016/j.nbd.2014.06.004)

179. MacLeod, D.A.; Rhinn, H.; Kuwahara, T.; Zolin, A.; Di Paolo, G.; McCabe, B.D.; Marder, K.S.; Honig, L.S.; Clark, L.N.; Small, S.A.; et al. RAB7L1 interacts with LRRK2 to modify intraneuronal protein sorting and Parkinson’s disease risk. *Neuron* **2013**, *77*, 425–439. [CrossRef](https://doi.org/10.1016/j.neuron.2012.11.032)

180. Zavodszky, E.; Seaman, M.N.; Moreau, K.; Jimenez-Sanchez, M.; Breusegem, S.Y.; Harbour, M.E.; Rubinsztein, D.C. Mutation in VPS35 associated with Parkinson’s disease impairs WASH complex association and inhibits autophagy. *Nat. Commun.* **2014**, *5*, 3828. [CrossRef](https://doi.org/10.1038/ncomms4828)

181. Takeuchi, T.; Nagai, Y. Protein Misfolding and Aggregation as a Therapeutic Target for Polyglutamine Diseases. *Brain Sci.* **2017**, *7*, 128. [CrossRef](https://doi.org/10.3390/brainsci7090128) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28873005)

182. Maat-Schieman, M.L.; Dorsman, J.C.; Smoor, M.A.; Siesling, S.; Van Duinen, S.G.; Verschuuren, J.J.; den Dunnen, J.T.; Van Ommen, G.J.; Roos, R.A. Distribution of inclusions in neuronal nuclei and dystrophic neurites in Huntington disease brain. *J. Neuropathol. Exp. Neurol.* **1999**, *58*, 129–137. [CrossRef](https://doi.org/10.1097/00005072-199902000-00007) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10028027)

183. Heng, M.Y.; Duong, D.K.; Albin, R.L.; Tallaksen-Greene, S.J.; Hunter, J.M.; Lesort, M.J.; Osmand, A.; Paulson, H.L.; Detloff, P.J. Early autophagic response in a novel knock-in model of Huntington disease. *Hum. Mol. Genet.* **2010**, *19*, 3702–3720. [CrossRef](https://doi.org/10.1093/hmg/ddq282) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20530516)

184. Erie, C.; Sacino, M.; Houle, L.; Lu, M.L.; Wei, J. Altered lysosomal positioning affects lysosomal functions in a cellular model of Huntington’s disease. *Eur. J. Neurosci.* **2015**, *42*, 1941–1951. [CrossRef](https://doi.org/10.1111/ejn.12942)

185. Nagata, E.; Sawa, A.; Ross, C.A.; Snyder, S.H. Autophagosome-like vacuole formation in Huntington’s disease lymphoblasts. *Neuroreport* **2004**, *15*, 1325–1328. [CrossRef](https://doi.org/10.1097/00001756-200405250-00026)

186. Rudnicki, D.D.; Pletnikova, O.; Vonsattel, J.P.; Ross, C.A.; Margolis, R.L. A comparison of huntington disease and huntington disease-like 2 neuropathology. *J. Neuropathol. Exp. Neurol.* **2008**, *67*, 366–374. [CrossRef](https://doi.org/10.1097/NEN.0b013e31816a8d57)

187. Wu, J.C.; Qi, L.; Wang, Y.; Kegel, K.B.; Yoder, J.; Difiglia, M.; Qin, Z.H.; Lin, F. The regulation of N-terminal Huntingtin (Htt552) accumulation by Beclin1. *Acta Pharmacol. Sin.* **2012**, *33*, 743–751. [CrossRef](https://doi.org/10.1038/aps.2012.26)

188. Proenca, C.C.; Stoehr, N.; Bernhard, M.; Seger, S.; Genoud, C.; Roscic, A.; Paganetti, P.; Liu, S.; Murphy, L.O.; Kuhn, R.; et al. Atg4b-dependent autophagic flux alleviates Huntington’s disease progression. *PLoS ONE* **2013**, *8*, e68357. [CrossRef](https://doi.org/10.1371/journal.pone.0068357)

189. Jia, K.; Hart, A.C.; Levine, B. Autophagy genes protect against disease caused by polyglutamine expansion proteins in Caenorhabditis elegans. *Autophagy* **2007**, *3*, 21–25. [CrossRef](https://doi.org/10.4161/auto.3585)

190. Ochaba, J.; Lukacsovich, T.; Csikos, G.; Zheng, S.; Margulis, J.; Salazar, L.; Mao, K.; Lau, A.L.; Yeung, S.Y.; Humbert, S.; et al. Potential function for the Huntingtin protein as a scaffold for selective autophagy. *Proc. Natl. Acad. Sci. USA* **2014**, *111*, 16889–16894. [CrossRef](https://doi.org/10.1073/pnas.1413593111)

191. Rui, Y.N.; Xu, Z.; Patel, B.; Chen, Z.; Chen, D.; Tito, A.; David, G.; Sun, Y.; Stimming, E.F.; Bellen, H.J.; et al. Huntingtin functions as a scaffold for selective macroautophagy. *Nat. Cell Biol.* **2015**, *17*, 262–275. [CrossRef](https://doi.org/10.1038/ncb3114) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25621959)

192. Ashkenazi, A.; Bento, C.F.; Ricketts, T.; Vicinanza, M.; Siddiqi, F.; Pavel, M.; Squitieri, F.; Hardenberg, M.C.; Imarisio, S.; Menzies, F.M.; et al. Polyglutamine tracts regulate beclin 1-dependent autophagy. *Nature* **2017**, *545*, 108–111. [CrossRef](https://doi.org/10.1038/nature22071) [PubMed](https://pubmed.ncbi.nlm.nih.gov/28332516)

193. Martinez-Vicente, M.; Talloczy, Z.; Wong, E.; Tang, G.; Koga, H.; Kaushik, S.; de Vries, R.; Arias, E.; Harris, S.; Sulzer, D.; et al. Cargo recognition failure is responsible for inefficient autophagy in Huntington's disease. *Nat. Neurosci.* **2010**, *13*, 567–576. [CrossRef] [PubMed]

194. Mealer, R.G.; Murray, A.J.; Shahani, N.; Subramaniam, S.; Snyder, S.H. Rhes, a striatal-selective protein implicated in Huntington disease, binds beclin-1 and activates autophagy. *J. Biol. Chem.* **2014**, *289*, 3547–3554. [CrossRef]

195. Chen, S.; Sayana, P.; Zhang, X.; Le, W. Genetics of amyotrophic lateral sclerosis: An update. *Mol. Neurodegener.* **2013**, *8*. [CrossRef]

196. Li, H.Y.; Yeh, P.A.; Chiu, H.C.; Tang, C.Y.; Tu, B.P. Hyperphosphorylation as a defense mechanism to reduce TDP-43 aggregation. *PLoS ONE* **2011**, *6*, e23075. [CrossRef]

197. Williams, K.L.; Warraich, S.T.; Yang, S.; Solski, J.A.; Fernando, R.; Rouleau, G.A.; Nicholson, G.A.; Blair, I.P. UBQLN2/ubiquilin 2 mutation and pathology in familial amyotrophic lateral sclerosis. *Neurobiol. Aging* **2012**, *33*, 2527-e3. [CrossRef]

198. Deng, H.X.; Chen, W.; Hong, S.T.; Boycott, K.M.; Gorrie, G.H.; Siddique, N.; Yang, Y.; Fecto, F.; Shi, Y.; Zhai, H.; et al. Mutations in UBQLN2 cause dominant X-linked juvenile and adult-onset ALS and ALS/dementia. *Nature* **2011**, *477*, 211–215. [CrossRef]

199. Rudnick, N.D.; Griffey, C.J.; Guarnieri, P.; Gerbino, V.; Wang, X.; Piersaint, J.A.; Tapia, J.C.; Rich, M.M.; Maniatis, T. Distinct roles for motor neuron autophagy early and late in the SOD1(G93A) mouse model of ALS. *Proc. Natl. Acad. Sci. USA* **2017**, *114*, E8294–E8303. [CrossRef]

200. Ling, J.P.; Pletnikova, O.; Troncoso, J.C.; Wong, P.C. TDP-43 repression of nonconserved cryptic exons is compromised in ALS-FTD. *Science* **2015**, *349*, 650–655. [CrossRef]

201. Torres, P.; Ramirez-Nunez, O.; Romero-Guevara, R.; Bares, G.; Granado-Serrano, A.B.; Ayala, V.; Boada, J.; Fontdevila, L.; Povedano, M.; Sanchis, D.; et al. Cryptic exon splicing function of TARDBP interacts with autophagy in nervous tissue. *Autophagy* **2018**, *14*, 1398–1403. [CrossRef] [PubMed]

202. Huang, S.L.; Wu, L.S.; Lee, M.; Chang, C.W.; Cheng, W.C.; Fang, Y.S.; Chen, Y.R.; Cheng, P.L.; Shen, C.J. A robust TDP-43 knock-in mouse model of ALS. *Acta Neuropathol. Commun.* **2020**, *8*, 3. [CrossRef] [PubMed]

203. Barmada, S.J.; Serio, A.; Arjun, A.; Bilican, B.; Daub, A.; Ando, D.M.; Tsvetkov, A.; Pleiss, M.; Li, X.; Peisach, D.; et al. Autophagy induction enhances TDP43 turnover and survival in neuronal ALS models. *Nat. Chem. Biol.* **2014**, *10*, 677–685. [CrossRef] [PubMed]

204. N'Diaye, E.N.; Kajihara, K.K.; Hsieh, I.; Morisaki, H.; Debnath, J.; Brown, E.J. PLIC proteins or ubiquilins regulate autophagy-dependent cell survival during nutrient starvation. *EMBO Rep.* **2009**, *10*, 173–179. [CrossRef]

205. Rothenberg, C.; Srinivasan, D.; Mah, L.; Kaushik, S.; Peterhoff, C.M.; Ugolino, J.; Fang, S.; Cuervo, A.M.; Nixon, R.A.; Monteiro, M.J. Ubiquilin functions in autophagy and is degraded by chaperone-mediated autophagy. *Hum. Mol. Genet.* **2010**, *19*, 3219–3232. [CrossRef]

206. Senturk, M.; Lin, G.; Zuo, Z.; Mao, D.; Watson, E.; Mikos, A.G.; Bellen, H.J. Ubiquilins regulate autophagic flux through mTOR signalling and lysosomal acidification. *Nat. Cell Biol.* **2019**, *21*, 384–396. [CrossRef]

207. Osaka, M.; Ito, D.; Suzuki, N. Disturbance of proteasomal and autophagic protein degradation pathways by amyotrophic lateral sclerosis-linked mutations in ubiquilin 2. *Biochem. Biophys. Res. Commun.* **2016**, *472*, 324–331. [CrossRef]

208. Goode, A.; Butler, K.; Long, J.; Cavey, J.; Scott, D.; Shaw, B.; Sollenberger, J.; Gell, C.; Johansen, T.; Oldham, N.J.; et al. Defective recognition of LC3B by mutant SQSTM1/p62 implicates impairment of autophagy as a pathogenic mechanism in ALS-FTLD. *Autophagy* **2016**, *12*, 1094–1104. [CrossRef]

209. Tumbarello, D.A.; Waxse, B.J.; Arden, S.D.; Bright, N.A.; Kendrick-Jones, J.; Buss, F. Autophagy receptors link myosin VI to autophagosomes to mediate Tom1-dependent autophagosome maturation and fusion with the lysosome. *Nat. Cell Biol.* **2012**, *14*, 1024–1035. [CrossRef]

210. Sundaramoorthy, V.; Walker, A.K.; Tan, V.; Fifita, J.A.; McCann, E.P.; Williams, K.L.; Blair, I.P.; Guillemain, G.J.; Farg, M.A.; Atkin, J.D. Defects in optineurin—and myosin VI-mediated cellular trafficking in amyotrophic lateral sclerosis. *Hum. Mol. Genet.* **2015**, *24*, 3830–3846. [CrossRef]

211. Parkinson, N.; Ince, P.G.; Smith, M.O.; Highley, R.; Skibinski, G.; Andersen, P.M.; Morrison, K.E.; Pall, H.S.; Hardiman, O.; Collinge, J.; et al. ALS phenotypes with mutations in CHMP2B (charged multivesicular body protein 2B). *Neurology* **2006**, *67*, 1074–1077. [CrossRef] [PubMed]

212. Han, J.H.; Ryu, H.H.; Jun, M.H.; Jang, D.J.; Lee, J.A. The functional analysis of the CHMP2B missense mutation associated with neurodegenerative diseases in the endo-lysosomal pathway. *Biochem. Biophys. Res. Commun.* **2012**, *421*, 544–549. [CrossRef](https://doi.org/10.1016/j.bbrc.2012.03.137) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22487656)

213. Cox, L.E.; Ferraiuolo, L.; Goodall, E.F.; Heath, P.R.; Higginbottom, A.; Mortiboys, H.; Hollinger, H.C.; Hartley, J.A.; Brockington, A.; Burness, C.E.; et al. Mutations in CHMP2B in lower motor neuron predominant amyotrophic lateral sclerosis (ALS). *PLoS ONE* **2010**, *5*, e9872. [CrossRef](https://doi.org/10.1371/journal.pone.0009872) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20300587)

214. Blackstone, C. Hereditary spastic paraplegia. *Handb. Clin. Neurol.* **2018**, *148*, 633–652. [CrossRef](https://doi.org/10.1016/B978-0-444-63918-7.00037-8)

215. Fink, J.K. Hereditary spastic paraplegia: Clinico-pathologic features and emerging molecular mechanisms. *Acta Neuropathol.* **2013**, *126*, 307–328. [CrossRef](https://doi.org/10.1007/s00401-013-1140-5)

216. Lee, S.; Park, H.; Zhu, P.P.; Jung, S.Y.; Blackstone, C.; Chang, J. Hereditary spastic paraplegia SPG8 mutations impair CAV1-dependent, integrin-mediated cell adhesion. *Sci. Signal.* **2020**, *13*, eaau7500. [CrossRef](https://doi.org/10.1126/scisignal.aau7500)

217. Varga, R.E.; Khundadze, M.; Damme, M.; Nietzsche, S.; Hoffmann, B.; Stauber, T.; Koch, N.; Hennings, J.C.; Franzka, P.; Huebner, A.K.; et al. In Vivo Evidence for Lysosome Depletion and Impaired Autophagic Clearance in Hereditary Spastic Paraplegia Type SPG11. *PLoS Genet.* **2015**, *11*, e1005454. [CrossRef](https://doi.org/10.1371/journal.pgen.1005454)

218. Chang, J.; Lee, S.; Blackstone, C. Spastic paraplegia proteins spastizin and spatacsin mediate autophagic lysosome reformation. *J. Clin. Investig.* **2014**, *124*, 5249–5262. [CrossRef](https://doi.org/10.1172/JCI75830)

219. Vantaggiato, C.; Panzeri, E.; Castelli, M.; Citterio, A.; Arnoldi, A.; Santorelli, F.M.; Liguori, R.; Scarlato, M.; Musumeci, O.; Toscano, A.; et al. ZFYVE26/SPASTIZIN and SPG11/SPATACSIN mutations in hereditary spastic paraplegia types AR-SPG15 and AR-SPG11 have different effects on autophagy and endocytosis. *Autophagy* **2019**, *15*, 34–57. [CrossRef](https://doi.org/10.1080/15548627.2018.1523050)

220. Vantaggiato, C.; Crimella, C.; Airolidi, G.; Polishchuk, R.; Bonato, S.; Brighina, E.; Scarlato, M.; Musumeci, O.; Toscano, A.; Martinuzzi, A.; et al. Defective autophagy in spastizin mutated patients with hereditary spastic paraparesis type 15. *Brain* **2013**, *136*, 3119–3139. [CrossRef](https://doi.org/10.1093/brain/awt208)

221. Behne, R.; Teinert, J.; Wimmer, M.; D'Amore, A.; Davies, A.K.; Scarrott, J.M.; Eberhardt, K.; Brechmann, B.; Chen, I.P.; Buttermore, E.D.; et al. Adaptor protein complex 4 deficiency: A paradigm of childhood-onset hereditary spastic paraplegia caused by defective protein trafficking. *Hum. Mol. Genet.* **2020**, *29*, 320–334. [CrossRef](https://doi.org/10.1093/hmg/ddz305) [PubMed](https://pubmed.ncbi.nlm.nih.gov/31468584)

222. Davies, A.K.; Itzhak, D.N.; Edgar, J.R.; Archuleta, T.L.; Hirst, J.; Jackson, L.P.; Robinson, M.S.; Borner, G.H.H. AP-4 vesicles contribute to spatial control of autophagy via RUSC-dependent peripheral delivery of ATG9A. *Nat. Commun.* **2018**, *9*, 3958. [CrossRef](https://doi.org/10.1038/s41467-018-06368-8) [PubMed](https://pubmed.ncbi.nlm.nih.gov/30122466)

223. Hirst, J.; Borner, G.H.; Edgar, J.; Hein, M.Y.; Mann, M.; Buchholz, F.; Antrobus, R.; Robinson, M.S. Interaction between AP-5 and the hereditary spastic paraplegia proteins SPG11 and SPG15. *Mol. Biol. Cell* **2013**, *24*, 2558–2569. [CrossRef](https://doi.org/10.1091/mbc.E12-11-0818) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23599537)

224. Khundadze, M.; Ribaudo, F.; Hussain, A.; Rosentreter, J.; Nietzsche, S.; Thelen, M.; Winter, D.; Hoffmann, B.; Afzal, M.A.; Hermann, T.; et al. A mouse model for SPG48 reveals a block of autophagic flux upon disruption of adaptor protein complex five. *Neurobiol. Dis.* **2019**, *127*, 419–431. [CrossRef](https://doi.org/10.1016/j.nbd.2019.03.010)

225. Ciechanover, A.; Kwon, Y.T. Degradation of misfolded proteins in neurodegenerative diseases: Therapeutic targets and strategies. *Exp. Mol. Med.* **2015**, *47*, e147. [CrossRef](https://doi.org/10.1038/emm.2015.14)

226. Bove, J.; Martinez-Vicente, M.; Vila, M. Fighting neurodegeneration with rapamycin: Mechanistic insights. *Nat. Rev. Neurosci.* **2011**, *12*, 437–452. [CrossRef](https://doi.org/10.1038/nrn3035)

227. Malagelada, C.; Jin, Z.H.; Jackson-Lewis, V.; Przedborski, S.; Greene, L.A. Rapamycin protects against neuron death in in vitro and in vivo models of Parkinson’s disease. *J. Neurosci.* **2010**, *30*, 1166–1175. [CrossRef](https://doi.org/10.1523/JNEUROSCI.4518-09.2010)

228. Forouzanfar, F.; Read, M.I.; Barreto, G.E.; Sahebkar, A. Neuroprotective effects of curcumin through autophagy modulation. *IUBMB Life* **2019**, *72*, 652–664. [CrossRef](https://doi.org/10.1002/iub.1996)

229. Wang, C.; Zhang, X.; Teng, Z.; Zhang, T.; Li, Y. Downregulation of PI3K/Akt/mTOR signaling pathway in curcumin-induced autophagy in APP/PS1 double transgenic mice. *Eur. J. Pharmacol.* **2014**, *740*, 312–320. [CrossRef](https://doi.org/10.1016/j.ejphar.2014.05.040)

230. Liang, J.; Zhou, F.; Xiong, X.; Zhang, X.; Li, S.; Li, X.; Gao, M.; Li, Y. Enhancing the retrograde axonal transport by curcumin promotes autophagic flux in N2a/APP695swe cells. *Aging* **2019**, *11*, 7036–7050. [CrossRef](https://doi.org/10.18632/aging.102108)

231. Jiang, T.F.; Zhang, Y.J.; Zhou, H.Y.; Wang, H.M.; Tian, L.P.; Liu, J.; Ding, J.Q.; Chen, S.D. Curcumin ameliorates the neurodegenerative pathology in A53T alpha-synuclein cell model of Parkinson’s disease through the downregulation of mTOR/p70S6K signaling and the recovery of macroautophagy. *J. Neuroimmune Pharmacol.* **2013**, *8*, 356–369. [CrossRef](https://doi.org/10.1007/s11481-013-9455-9) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23529796)

232. Rusmini, P.; Cortese, K.; Crippa, V.; Cristofani, R.; Cicardi, M.E.; Ferrari, V.; Vezzoli, G.; Tedesco, B.; Meroni, M.; Messi, E.; et al. Trehalose induces autophagy via lysosomal-mediated TFEB activation in models of motoneuron degeneration. *Autophagy* **2019**, *15*, 631–651. [CrossRef](https://doi.org/10.1080/15548627.2019.1580586) [PubMed](https://pubmed.ncbi.nlm.nih.gov/30851888)

233. DeBosch, B.J.; Heitmeier, M.R.; Mayer, A.L.; Higgins, C.B.; Crowley, J.R.; Kraft, T.E.; Chi, M.; Newberry, E.P.; Chen, Z.; Finck, B.N.; et al. Trehalose inhibits solute carrier 2A (SLC2A) proteins to induce autophagy and prevent hepatic steatosis. *Sci. Signal.* **2016**, *9*, ra21. [CrossRef](https://doi.org/10.1126/scisignal.aad6767) [PubMed](https://pubmed.ncbi.nlm.nih.gov/27053657)

234. Castillo, K.; Nassif, M.; Valenzuela, V.; Rojas, F.; Matus, S.; Mercado, G.; Court, F.A.; van Zundert, B.; Hetz, C. Trehalose delays the progression of amyotrophic lateral sclerosis by enhancing autophagy in motoneurons. *Autophagy* **2013**, *9*, 1308–1320. [CrossRef](https://doi.org/10.4161/auto.25586)

235. Li, Y.; Guo, Y.; Wang, X.; Yu, X.; Duan, W.; Hong, K.; Wang, J.; Han, H.; Li, C. Trehalose decreases mutant SOD1 expression and alleviates motor deficiency in early but not end-stage amyotrophic lateral sclerosis in a SOD1-G93A mouse model. *Neuroscience* **2015**, *298*, 12–25. [CrossRef](https://doi.org/10.1016/j.neuroscience.2015.02.045)

236. Williams, A.; Sarkar, S.; Cuddon, P.; Ttofi, E.K.; Saiki, S.; Siddiqi, F.H.; Jahreiss, L.; Fleming, A.; Pask, D.; Goldsmith, P.; et al. Novel targets for Huntington’s disease in an mTOR-independent autophagy pathway. *Nat. Chem. Biol.* **2008**, *4*, 295–305. [CrossRef](https://doi.org/10.1038/nchembio.79)

237. Sarkar, S.; Floto, R.A.; Berger, Z.; Imarisio, S.; Cordenier, A.; Pasco, M.; Cook, L.J.; Rubinsztein, D.C. Lithium induces autophagy by inhibiting inositol monophosphatase. *J. Cell. Biol.* **2005**, *170*, 1101–1111. [CrossRef](https://doi.org/10.1083/jcb.200504022)

238. Shoji-Kawata, S.; Sumpter, R.; Leveno, M.; Campbell, G.R.; Zou, Z.; Kinch, L.; Wilkins, A.D.; Sun, Q.; Pallauf, K.; MacDuff, D.; et al. Identification of a candidate therapeutic autophagy-inducing peptide. *Nature* **2013**, *494*, 201–206. [CrossRef](https://doi.org/10.1038/nature11884)
