
Review

# On the Host Side of the Hepatitis E Virus Life Cycle

Noémie Oechslin<sup>1</sup>, Darius Moradpour<sup>2</sup> and Jérôme Gouttenoire<sup>3,*</sup>

Division of Gastroenterology and Hepatology, Lausanne University Hospital and University of Lausanne, CH-1011 Lausanne, Switzerland; Noemie.Oechslin@unil.ch (N.O.); Darius.Moradpour@chuv.ch (D.M.)

* Correspondence: Jerome.Gouttenoire@chuv.ch

Received: 28 April 2020; Accepted: 21 May 2020; Published: 22 May 2020

---

**Abstract:** Hepatitis E virus (HEV) infection is one of the most common causes of acute hepatitis in the world. HEV is an enterically transmitted positive-strand RNA virus found as a non-enveloped particle in bile as well as stool and as a quasi-enveloped particle in blood. Current understanding of the molecular mechanisms and host factors involved in productive HEV infection is incomplete, but recently developed model systems have facilitated rapid progress in this area. Here, we provide an overview of the HEV life cycle with a focus on the host factors required for viral entry, RNA replication, assembly and release. Further developments of HEV model systems and novel technologies should yield a broader picture in the future.

**Keywords:** HEV; host factor; particle production; viral replication; virus entry

---

## 1. Introduction

Hepatitis E virus (HEV) has been identified as a cause of the waterborne hepatitis outbreaks in the early 1980s [1,2]. The viral genome was cloned and sequenced in 1990, allowing the development of serological tests to study its epidemiology [3,4]. The virus has been classified in the *Hepeviridae* family, and most human pathogenic strains belong to species Orthohepevirus A [5]. Members of this species can be classified into 8 genotypes (gt): gt 1 and 2 are restricted to humans and are transmitted via the fecal-oral route, mainly through contaminated drinking water. Gt 3 and 4 cause zoonotic infections and the transmission occurs mainly via the consumption of un(der)cooked pork, wild boar or deer meat. Gt 5 and 6 are found in wild boar, and gt 7 as well as 8 infect dromedary and Bactrian camels, respectively. Gt 7 has been identified in an immunosuppressed patient after consumption of camel milk and meat [6] (reviewed in [7]) but no transmission to humans has thus far been reported for gt 5, 6 and 8. More recently, rabbit HEV (closely related to gt 3 within species Orthohepevirus A) and rat HEV (belonging to species Orthohepevirus C) have also been found to infect humans [8–11].

HEV is a small, non-enveloped, icosahedral virus with a diameter of 27–34 nm [2]. It contains a 7.2 kb single-stranded, positive-sense RNA genome which possesses a m7G cap at its 5′ and a poly-A tail at its 3′ end (Figure 1A). The HEV genome harbors 3 open reading frames (ORF). ORF1 encodes the viral replicase, ORF2 the capsid and ORF3 a small protein involved in virion secretion via its potential ion channel activity [12]. The first contact between HEV and host cells occurs through interaction with as yet poorly characterized entry factor(s). After endocytosis, the viral genome is released into the cytoplasm and the host translational machinery produces the ORF1 replicase, which drives viral RNA replication (Figure 1B). During this step, two RNA species are produced from a negative-strand RNA intermediate: a full-length genomic RNA and a subgenomic RNA of 2.2 kb [13,14]. Translation of the subgenomic RNA yields the ORF2 and ORF3 proteins. Later steps of the HEV life cycle include viral assembly and release of newly produced virions. Very similar to hepatitis A virus (HAV), another hepatotropic positive-strand RNA virus, HEV is found as a ‘quasi-enveloped’ virion (eHEV) wrapped in exosomal membranes in blood and as a naked particle in bile and feces (reviewed in [15]) (Figure 1B).

---

Cells 2020, 9, 1294; doi:10.3390/cells9051294
www.mdpi.com/journal/cells

Figure 1. Genome organization and life cycle of hepatitis E virus (HEV). (A) The 7.2 kb positive-strand RNA genome has a 5′ 7-methylguanylate cap (m⁷G cap) and a 3′ polyadenylated tail (poly-A). It harbors 3 open reading frames (ORFs). ORF1 encodes a replicase of about 190 kDa comprising different functional domains, including a methyltransferase (Met), an RNA helicase (Hel) and an RNA-dependent RNA polymerase (RdRp), as well as less well-characterized domains, such as the Y domain, a putative papain-like cysteine protease (PCP), a hypervariable region (HVR) and the Macro domain. ORF2 and ORF3 encode the viral capsid and a small protein involved in virus secretion respectively, which are translated from a 2.2 kb subgenomic RNA generated during viral replication. (B) The HEV life cycle can be dissected into the following steps: (1) viral entry by as yet unidentified receptor(s), (2) endocytosis and release of the viral positive-strand RNA genome (+) into the cytosol, (3) translation of the ORF1 protein to allow replication of the full-length and generation of the subgenomic RNA through a negative-strand RNA intermediate (-), (4) translation of the subgenomic RNA to produce the ORF2 and ORF3 proteins and (5) genome packaging, virion assembly and release of the virus into the bloodstream and the bile from the basolateral and apical sides, respectively. ER, endoplasmic reticulum; MVB, multivesicular body.

As obligate intracellular pathogens, viruses have developed strategies to hijack and manipulate host cell pathways in order to ensure productive infection. Moreover, RNA viruses, especially those with relatively limited genome size and coding capacity, such as HEV, are particularly dependent on the host cell machinery. Because the tools to study HEV have been limited until recently, only little is known about the host factors involved in the various steps of the viral life cycle. Studies performed in heterologous settings, such as the yeast two hybrid system, identified cellular factors interacting with HEV proteins [16, 17]. However, most of these candidates remain to be validated and further studied using infectious cell culture systems, in vivo models and liver biopsies from patients with hepatitis E. In this review, we shall focus on host factors whose involvement in the viral life cycle has been validated in HEV infection settings.

## 2. HEV Entry

As HEV is present in a non-enveloped ("naked") and a quasi-enveloped form (eHEV), the entry pathway of the virus may differ for these two forms. Our knowledge of HEV entry remains scarce but studies using virus-like particles (VLP) as a model system have highlighted possible host factors involved in the initial attachment to the cell and virus internalization, including the 78-kDa glucose-regulated protein (GRP78), ATP synthase subunit β (ATPB5) and asialoglycoprotein receptor (ASGPR) [18–20]. Notably, non-enveloped HEV was shown to interact with heparan sulfate proteoglycans (HSPG), likely syndecans [21, 22], which are expressed on the surface of many cell types. Hence, treatment of susceptible hepatoma cell lines with heparinase I considerably reduced VLP binding as well as HEV infection [21] (Figure 2). Of note, HSPG are known to mediate cell attachment of several enveloped and nonenveloped viruses, including, among others, herpes simplex virus, hepatitis C virus (HCV), norovirus and human immunodeficiency virus [23–27].

![Figure 2](attachment:Figure_2.png)

**Figure 2. Non-enveloped versus quasi-enveloped hepatitis E virus entry.** Non-enveloped hepatitis E virus (HEV) is believed to first bind to heparan sulfate proteoglycans and integrin α3 at the cell surface (left panel). The virus is then internalized via clathrin- and dynamin 2-dependent endocytosis, followed by release of the viral genome into the cytoplasm by an as yet unknown mechanism, possibly involving a conformational change of the capsid. The cofactor(s) and receptor(s) allowing entry of quasi-enveloped HEV (eHEV) are unknown (right panel). Internalization requires clathrin- and dynamin 2-dependent endocytosis as well as trafficking through Rab5- (early) as well as Rab7-positive (late) endosomes and eventually lysosomes to allow release of the viral genome into the cytoplasm, likely by a process similar to that of naked HEV.

A recent microarray analysis comparing gene expression in permissive versus non-permissive cells identified integrin α3 as an entry factor for HEV [28]. Integrins belong to a family of transmembrane

proteins localized at the cell surface which can bind to extracellular matrix (ECM) as well as cell surface and intracellular ligands. These interactions trigger a myriad of intracellular signals modulating cell behavior through effects on actin microfilaments, thereby connecting the extracellular space and the cytoskeleton (reviewed in [29]). Integrins were shown to function as receptors for Kaposi's sarcoma-associated herpesvirus [30] and, interestingly, also for HAV [31]. In the case of HEV, overexpression of integrin α3 in non-permissive cells allowed non-enveloped HEV infection only and, conversely, knockout of integrin α3 gene in permissive cells prevented entry of non-enveloped HEV but not of eHEV [28]. Together with the physical interaction between integrin α3 and HEV, the data strongly suggests that this ECM receptor is an entry factor for the non-enveloped viral particle (Figure 2). Integrins are expressed in a broad array of tissues, including in the intestine [26,32], where HEV is known to replicate and which may represent the initial site of infection [33,34]. However, in some organs, such as the liver, integrin α3 expression is rather low [35–37], raising the possibility that integrin α3 may act as a cofactor for HEV entry rather than as the key receptor. Further studies are needed to clarify the entry pathway used by non-enveloped HEV.

Internalization of non-enveloped HEV is believed to occur through clathrin- and dynamin 2-dependent endocytosis [22,38,39] (Figure 2). In this context, membrane cholesterol was also shown to be important for HEV entry, as treatment of cells with cholesterol sequestering agents significantly reduced VLP uptake [39]. Following endocytosis, the HEV genome needs to be uncoated to be released into the cytoplasm. It is likely that the capsid protein itself plays a crucial role in this poorly understood process, possibly by undergoing conformational changes induced by interaction with a host protein, as it is seen in other non-enveloped viruses, such as human papilloma virus, murine polyomavirus and poliovirus [40–42].

Since the viral capsid is not exposed at the surface of eHEV (see Section 4), the quasi-enveloped particle may use different, as yet unknown entry factor(s). However, as for the naked virus, eHEV is internalized through clathrin- and dynamin 2-dependent endocytosis [38,39]. Unlike non-enveloped HEV, eHEV entry is dependent on the small GTPases, Rab5 and Rab7, both of which are involved in endosomal trafficking. In fact, depletion of one or the other has been shown to considerably reduce virus infection [22] (Figure 2). In addition, eHEV entry depends on endosomal acidification [22], indicating that transition of the endosome to a more acidic cell compartment is necessary (Figure 2). Lysosomal lipid degradation appears to be a required step in eHEV entry. Indeed, depletion of the late endosomal and lysosomal Niemann-Pick disease type C1 protein, involved in cholesterol extraction, significantly reduced eHEV infection [22]. Moreover, treatment with an inhibitor of lysosomal acid lipase, responsible for lipid degradation, resulted in a dose-dependent reduction of eHEV cell entry [22].

These observations suggest that the quasi-envelope of eHEV is removed in endolysosomes. The viral capsid may subsequently interact with an as yet unidentified host factor and undergo the conformational changes required for genome release into the cytoplasm. A similar mechanism was recently proposed for HAV [31]. Since both quasi-enveloped and non-enveloped HEV are internalized in vesicles belonging to the endosomal pathway, it is plausible that they use a common host factor to allow uncoating and release of the genome into the cytoplasm (detailed in [43]).

The ORF3 protein is present within eHEV and interacts with the capsid [44]. However, its role in eHEV entry, uncoating and genome release remains to be explored.

Taken together, some HEV entry factors and pathways have been identified, but key elements are still missing. The capacity of HEV to infect a broad array of cell types [45–51], possibly related to the diverse extrahepatic manifestations of hepatitis E [52–55], strongly suggests that the entry receptor(s) is (are) not specific to hepatocytes but ubiquitously expressed.

## 3. Viral RNA Replication

Upon release of the viral positive-strand RNA genome into the cytoplasm, the host translational machinery, including ribosomal subunits and elongation factors, seeds on the 5′ untranslated region to start translation of the ORF1 protein. Among the different components, the eukaryotic translation

initiation factor 4F (eIF4F) complex has been identified as being important for HEV replication. This complex is known to be involved in the cap-dependent translation and replication of several viruses (reviewed in [56]). In this context, an RNA interference-based loss-of-function study showed that components of the eIF4F complex are required for efficient HEV replication, while known negative regulatory factors of this pathway limit viral RNA synthesis [57]. Along these lines, silvestrol, a natural compound inhibiting part of the eIF4F complex, efficiently inhibits HEV replication in vitro, and in a mouse model in vivo, ORF2 protein production and virus spread [58,59]. Interestingly, silvestrol was reported to also inhibit corona-, picorna-, alpha-, flavi- and filo-viruses and, therefore, displays broad antiviral activity [60–63].

ORF1 encodes the functional domains required for viral RNA synthesis, including an RNA-dependent RNA polymerase (RdRp) located at the C-terminal end of the polyprotein. In addition to the RdRp, the replicase encoded by ORF1 comprises an RNA helicase, a methyltransferase as well as less well-characterized domains, including the Macro and Y domains, a hypervariable region and a putative papain-like cysteine protease (PCP) (Figure 1A). Of note, the PCP has been associated with deubiquitination and the Macro domain with deribosylation activities likely involved in the posttranslational modifications of host proteins [64,65]. Positive-strand RNA viruses commonly process their polyproteins into individual functional proteins by viral, and in some instances, cellular proteases. In the case of HEV, however, it is unsettled whether, and if so, by which mechanism, the ORF1 protein is processed. While some studies reported processing of the ORF1 polyprotein, including by the PCP or the cellular proteases thrombin and factor Xa [66], others have found that the major form of ORF1 protein is unprocessed (reviewed in [53,67]). Further studies in complete cell culture systems and using more sensitive techniques for the detection of ORF1 protein are required [68].

As for all positive-strand RNA viruses, replication includes synthesis of a complementary negative-strand RNA which serves as a template for the production of positive-strand genomic and, in the case of HEV, an additional subgenomic RNA. The key enzyme responsible for these steps is the RdRp. The mechanisms regulating the production of these RNA species from the negative-strand RNA are still unknown but imply cis-acting elements within the genome [69]. Regulation of transcription may also involve host factor(s), such as heterogeneous nuclear ribonucleoproteins (hnRNP) [70,71]. hnRNPs play a role in nuclear RNA metabolism and have been reported to re-localize to the cytoplasm of HEV-infected cells [71], similarly to what had been observed for infection with other RNA viruses [72–76], suggesting a potential role of hnRNPs in genomic and subgenomic HEV RNA transcription.

Replication of positive-strand RNA viruses takes place in membrane-associated replication complexes composed of viral proteins, the replicating viral RNA, rearranged cellular membranes and other host factors [77]. The subcellular site of HEV RNA replication has not been identified to date but recent advances such as the development of tagged functional HEV genomes may facilitate progress in this area [68]. Insertion of a hemagglutinin epitope tag in the ORF1 polyprotein allowed us to visualize the HEV replicase together with viral RNA in cytoplasmic dot-like structures, likely indicating the site of active replication. ORF1 protein was found to colocalize best with exosomal markers as well as with the ORF2 and ORF3 proteins, suggesting that HEV RNA replication takes place in close proximity to virion assembly sites [68]. Virus-induced membrane rearrangements may serve several purposes, including the physical support and organization of the replication complex, the compartmentalization and local concentration of viral and host factors required for RNA replication, tethering of the viral RNA during unwinding and coordination of its translation, replication and packaging, provision of lipid constituents important for replication, and physical protection from host antiviral defenses (reviewed in [77,78]). Of note, HCV as well as picorna- and coronaviruses require the guanine nucleotide-exchange factor Golgi brefeldin A resistance factor 1 (GBF1) for the induction of membrane alterations making up their viral replication complexes [79–83]. Interestingly, similar observations have been made for HEV [84]. However, no colocalization of GBF1 with HEV ORF1 and ORF2 proteins or relocalization of GBF1 upon HEV infection have been observed. One may thus hypothesize that GBF1 is not recruited to HEV replication sites and is not involved in replication

complex formation but rather plays an indirect role in HEV replication [84], as also proposed for HCV [79,80] and mouse hepatitis virus [85].

# 4. Virion Assembly and Infectious Particle Release

Virion assembly consists of the packaging of genomic viral RNA in the capsid. While the subcellular site of HEV assembly has not been identified yet, it is likely tightly connected to replication complexes (see above). The mechanisms driving HEV assembly are poorly understood but early observations showed that RNA and the ORF2 protein can spontaneously assemble into virus-like particles in insect cells [86] (reviewed in [87]). These findings argue in favor of a self-assembly process involving a limited number of host factors.

The ORF2 protein exists in several forms, of which a non-glycosylated form is involved in virion formation [88–91]. It is currently debated whether this form, harboring a truncated and non-functional signal peptide, results from translation at an alternative start site [90] or from proteolytic cleavage, as suggested by mass spectrometry analyses [89]. The non-glycosylated ORF2 protein may represent the major intracellular form detected in the cytosol as well as the cell nucleus [89,91,92]. While the role of nuclear ORF2 protein is unknown, its nucleocytoplasmic shuttling likely involves the host nuclear import/export machineries. In contrast to the non-glycosylated form packaging the viral genome, the glycosylated form of ORF2 protein is rapidly released extracellularly through the secretory pathway [88–91,93,94]. This implies recognition of its signal peptide by the translocon, translocation into the ER lumen, cleavage by signal peptidase, followed by sialylation as well as N- and O-glycosylation by yet to be defined glycosyltransferases in the ER and Golgi [89], and secretion. Glycosylated ORF2 protein is present in at least two forms, of which the smaller may result from cleavage by furin-like proteases at an RRR motif [91]. Overall, these studies suggest that ORF2 protein likely has more than one function in genome packaging and point toward the implication of different host factors.

HEV assembly involves the non-glycosylated ORF2 protein and the viral RNA but does not require the ORF3 protein, as genomes harboring a mutated ORF3 start codon yield infectious particles; however, these are not secreted from the cell [95,96]. Infectious virus is believed to be released as quasi-enveloped particles from both the basolateral and apical sides of hepatocytes, facing the liver sinusoids and the bile canaliculi, respectively [33,97,98]. The pseudo-envelope of virions secreted into the bile is likely delipidated by bile acids and/or pancreatic enzymes (Figure 3). Interestingly, HEV is preferentially secreted from the apical side, explaining the high viral load detected in feces. Based on these observations, different host factors may be involved in virus secretion from the apical and basolateral sides of hepatocytes. Moreover, HEV ORF3 protein is essential for the secretion of infectious particles, possibly by connecting the capsid with the host factors required for egress. Phosphorylation of ORF3 protein at a serine residue (Ser 70 and Ser 71 in gt 3 and gt 1, respectively) may trigger the interaction with assembled non-glycosylated ORF2 protein [44]. Based on sequence information, phosphorylation may involve the p34cdc2 kinase and mitogen-activated protein kinase [99]. Furthermore, ORF3 protein recruits Tsg101, a member of the endosomal sorting complex required for transport-I (ESCRT-I), via a highly conserved PSAP motif [100–102]. The presence of this motif in the C-terminal region of the ORF3 protein is required for virus release [101]. It is known that proline-rich motifs P(S/T)AP and PPXY (X being any amino acid), which are also called "late domains", are essential for budding of enveloped viruses such as human immunodeficiency virus (HIV)-1 or Ebola [103,104]. The ESCRT machinery is composed of 4 complexes, ESCRT-0 to ESCRT-III, which are involved in membrane remodeling, leading to budding reactions or membrane involution. These complexes can further recruit accessory proteins, such as vacuolar protein sorting 4 (Vps4) and apoptosis-linked gene-2 interacting protein X (Alix), both interacting with the ESCRT-III complex, to close newly formed vesicles (reviewed in [105]). Further confirming the requirement for the ESCRT machinery, hepatocyte growth factor-regulated tyrosine kinase substrate (Hrs), a member of ESCRT-0, and Vps4 were shown to be required for HEV particle release [102,106]. In addition, the envelope wrapping the virion has

been shown to be derived from exosomes, small vesicles generated from multivesicular bodies (MVB) by the ESCRT pathway [102, 106]. These MVB, which fuse with the plasma membrane to release their content into the extracellular milieu, require the Rab27a protein, which has been shown to colocalize with ORF3 protein [102, 106] (Figure 3).

![Diagram](attachment:diagram.png)

Figure 3. Assembly and release of infectious hepatitis E virus. Packaging of the viral genome into the capsid is believed to occur by spontaneous self-assembly of the non-glycosylated ORF2 protein. The non-glycosylated ORF2 protein may undergo post-translational modification by yet unknown enzyme(s). Formation of the quasi-enveloped particle involves phosphorylation and palmitoylation of the ORF3 protein and the ESCRT machinery, of which the components Tsg101, Hrs, Vps4 and Alix were shown to be required (see text for abbreviations). The virion is wrapped in an exosomal membrane harboring the tetraspanins CD9, CD63 and CD81, as well as the trans-Golgi network protein 2 (TGOLN2), Alix and Tsg101. Release of quasi-enveloped HEV (eHEV) involves Rab27a-dependent trafficking of multivesicular bodies (MVB) and fusion with the plasma membrane. Secreted particles remain associated with the lipid membrane in the bloodstream while they are delipidated in the bile. Asterisks indicate host factors that were not found on the quasi-envelope of eHEV.

Current evidence indicates that none of the viral proteins are present on the surface of eHEV but identified ORF3 protein beneath the quasi-envelope [107–109]. Palmitoylation at N-terminal cysteine residues mediates membrane association as well as stability of the ORF3 protein and is required for virus secretion [110]. A membrane topology where ORF3 protein is located entirely on the cytosolic side, corresponding to the exosome lumen, has therefore been proposed [110] and is further supported by the interaction of its PSAP motif with Tsg101. Of note, palmitoylation is a reversible protein modification taking place in the cytosol, which increases the hydrophobicity of a protein and contributes to its membrane association (reviewed in [111]). This implies the requirement for one or more as yet unidentified S-palmitoyl-transferase(s) of the host cell [110].

Given the origin of the quasi-envelope, host proteins, and more specifically, exosomal proteins, can be present on eHEV. Indeed, particles released from infected cells and positive for ORF2 and ORF3 proteins were shown to harbor classical exosomal markers, including the tetraspanins CD81, CD63 and CD9, as well as the ESCRT components Alix and Tsg101 [106, 108, 112]. Interestingly, trans-Golgi network protein 2 (TGOLN2) has also been detected on eHEV [107] (Figure 3). This observation and the fact that TGOLN2 is located in the cytoplasm confirms that quasi-enveloped particles are produced within the cell and not at the plasma membrane.

The quasi-enveloped nature of virions circulating in blood may provide several advantages to HEV, including protection from neutralizing antibodies. However, the presence of eHEV in blood may also favor virus dissemination to organs other than the liver, as exosomes containing viral genetic material were shown to lead to productive infection by other viruses and to modulate cellular responses (reviewed in [113]). Hence, it is plausible that the production of quasi-enveloped virions has additional functions beyond the spread of HEV within the liver.

## 5. Conclusions and Perspectives

Although HEV research is a rapidly growing field, our current knowledge of the virus life cycle is still limited by important gaps. We lack key information on virus entry, including the cellular receptor(s), and on the uncoating of the viral RNA. Moreover, future efforts should concentrate on the molecular mechanisms and subcellular compartments involved in RNA replication and assembly. While virus release remains one of the best studied steps of the viral life cycle, many aspects need to be clarified, in particular the contribution of host factors and the composition and role of quasi-enveloped particles. Considering the ability of HEV to replicate in different tissues as well as to infect a wide range of animals, one may hypothesize that HEV host dependency is not very selective, facilitating the crossing of species barriers.

Studies on the HEV life cycle currently rely on the use of in vitro model systems, which have certain limitations, especially with respect to host factors present in differentiated hepatocytes. As an example, a stimulatory effect of cyclophilin inhibitors on HEV replication was reported initially in hepatoma cells [114]. However, in stem cell-derived hepatocyte-like cells, this observation was confirmed only with a cell culture-adapted infectious clone, but not with natural HEV isolates [115]. Future improvements of in vitro models should include the use of natural HEV isolates together with primary and stem cell-derived hepatocyte-like cells, polarized cell culture models, as well as ex vivo and in vivo infection model systems to confirm in vitro findings. Ultimately, key findings will have to be validated in liver specimens from patients with hepatitis E.

Obtaining a broader and unbiased view of the host factors involved in the HEV life cycle will likely depend on novel technologies, such as clustered regularly interspaced short palindromic repeats (CRISPR)/Cas9-based genome-wide screening and proteomic proximity labeling approaches. CRISPR/Cas9-based screens have advanced the understanding of, among others, virus entry [116, 117] and replication [118, 119], and have also facilitated the identification of new antiviral targets [120]. Proximity labeling has been successfully used, for example, to characterize the microenvironment of coronavirus replication complexes [121]. In the future, the combination of improved HEV model systems and of novel technologies should improve our knowledge of the host factors required for productive HEV infection.

**Author Contributions:** N.O. and J.G. conceived the article; N.O. and J.G. wrote the original draft and prepared figures; D.M. edited and reviewed the manuscript. All authors have read and agreed to the published version of the manuscript.

**Funding:** This work was funded by the Swiss National Science Foundation (31003A_179424 to D.M. and CRSK-3_190706 to J.G.), the Novartis Foundation (18C140 to D.M.) and the Gilead Sciences International Research Scholars Program in Liver Disease (Award 2015 to J.G.).

**Conflicts of Interest:** The authors declare no conflict of interest.

References

1. Khuroo, M.S. Study of an epidemic of non-A, non-B hepatitis. Possibility of another human hepatitis virus distinct from post-transfusion non-A, non-B type. *Am. J. Med.* 1980, 68, 818–824. [CrossRef]

2. Balayan, M.S.; Andjaparidze, A.G.; Savinskaya, S.S.; Ketiladze, E.S.; Braginsky, D.M.; Savinov, A.P.; Poleschuk, V.F. Evidence for a virus in non-A, non-B hepatitis transmitted via the fecal-oral route. *Intervirology* 1983, 20, 23–31.

3. Reyes, G.R.; Purdy, M.A.; Kim, J.P.; Luk, K.C.; Young, L.M.; Fry, K.E.; Bradley, D.W. Isolation of a cDNA from the virus responsible for enterically transmitted non-A, non-B hepatitis. *Science* 1990, 247, 1335–1339. [CrossRef] [PubMed]

4. Tam, A.W.; Smith, M.M.; Guerra, M.E.; Huang, C.C.; Bradley, D.W.; Fry, K.E.; Reyes, G.R. Hepatitis E virus (HEV): Molecular cloning and sequencing of the full-length viral genome. *Virology* 1991, 185, 120–131. [CrossRef]

5. Smith, D.B.; Simmonds, P.; Jameel, S.; Emerson, S.U.; Harrison, T.J.; Meng, X.J.; Okamoto, H.; Van der Poel, W.H.; Purdy, M.A. Consensus proposals for classification of the family Hepeviridae. *J. Gen. Virol.* 2015, 96, 1191–1192. [CrossRef] [PubMed]

6. Lee, G.H.; Tan, B.H.; Teo, E.C.; Lim, S.G.; Dan, Y.Y.; Wee, A.; Aw, P.P.; Zhu, Y.; Hibberd, M.L.; Tan, C.K.; et al. Chronic infection with camelid hepatitis E virus in a liver transplant recipient who regularly consumes camel meat and milk. *Gastroenterology* 2016, 150, 355.e353–357.e353. [CrossRef]

7. Nimgaonkar, I.; Ding, Q.; Schwartz, R.E.; Ploss, A. Hepatitis E virus: Advances and challenges. *Nat. Rev. Gastroenterol. Hepatol.* 2018, 15, 96–110. [CrossRef]

8. Abravanel, F.; Lhomme, S.; El Costa, H.; Schwartz, B.; Peron, J.M.; Kamar, N.; Izopet, J. Rabbit hepatitis E virus infections in humans, France. *Emerg. Infect. Dis.* 2017, 23, 1191–1193. [CrossRef]

9. Sahli, R.; Fraga, M.; Semela, D.; Moradpour, D.; Gouttenoire, J. Rabbit HEV in immunosuppressed patients with hepatitis E acquired in Switzerland. *J. Hepatol.* 2019, 70, 1023–1025. [CrossRef]

10. Sridhar, S.; Yip, C.C.Y.; Wu, S.; Cai, J.; Zhang, A.J.; Leung, K.H.; Chung, T.W.H.; Chan, J.F.W.; Chan, W.M.; Teng, J.L.L.; et al. Rat hepatitis E virus as cause of persistent hepatitis after liver transplant. *Emerg. Infect. Dis.* 2018, 24, 2241–2250. [CrossRef]

11. Sridhar, S.; Yip, C.C.; Wu, S.; Chew, N.F.; Leung, K.H.; Chan, J.F.; Zhao, P.S.; Chan, W.M.; Poon, R.W.; Tsoi, H.W.; et al. Transmission of rat hepatitis E virus infection to humans in Hong Kong: A clinical and epidemiological analysis. *Hepatology* 2020. [CrossRef] [PubMed]

12. Ding, Q.; Heller, B.; Capuccino, J.M.; Song, B.; Nimgaonkar, I.; Hrebikova, G.; Contreras, J.E.; Ploss, A. Hepatitis E virus ORF3 is a functional ion channel required for release of infectious particles. *Proc. Natl. Acad. Sci. USA* 2017, 114, 1147–1152. [CrossRef] [PubMed]

13. Graff, J.; Torian, U.; Nguyen, H.; Emerson, S.U. A bicistronic subgenomic mRNA encodes both the ORF2 and ORF3 proteins of hepatitis E virus. *J. Virol.* 2006, 80, 5919–5926. [CrossRef] [PubMed]

14. Ichiyama, K.; Yamada, K.; Tanaka, T.; Nagashima, S.; Jirintai; Takahashi, M.; Okamoto, H. Determination of the 5′-terminal sequence of subgenomic RNA of hepatitis E virus strains in cultured cells. *Arch. Virol.* 2009, 154, 1945–1951. [CrossRef]

15. Feng, Z.; Hensley, L.; McKnight, K.L.; Hu, F.; Madden, V.; Ping, L.; Jeong, S.H.; Walker, C.; Lanford, R.E.; Lemon, S.M. A pathogenic picornavirus acquires an envelope by hijacking cellular membranes. *Nature* 2013, 496, 367–371. [CrossRef]

16. Geng, Y.; Yang, J.; Huang, W.; Harrison, T.J.; Zhou, Y.; Wen, Z.; Wang, Y. Virus host protein interaction network analysis reveals that the HEV ORF3 protein may interrupt the blood coagulation process. *PLoS ONE* 2013, 8, e56320. [CrossRef]

17. Subramani, C.; Nair, V.P.; Anang, S.; Mandal, S.D.; Pareek, M.; Kaushik, N.; Srivastava, A.; Saha, S.; Shalimar; Nayak, B.; et al. Host-virus protein interaction network reveals the involvement of multiple host processes in the life cycle of hepatitis E virus. *mSystems* 2018, 3, e00135-17. [CrossRef]

18. Yu, H.; Li, S.; Yang, C.; Wei, M.; Song, C.; Zheng, Z.; Gu, Y.; Du, H.; Zhang, J.; Xia, N. Homology model and potential virus-capsid binding site of a putative HEV receptor Grp78. *J. Mol. Model.* 2011, 17, 987–995. [CrossRef]

19. Ahmed, Z.; Holla, P.; Ahmad, I.; Jameel, S. The ATP synthase subunit β (ATP5B) is an entry factor for the hepatitis E virus. *bioRxiv* 2016. [CrossRef]

20. Zhang, L.; Tian, Y.; Wen, Z.; Zhang, F.; Qi, Y.; Huang, W.; Zhang, H.; Wang, Y. Asialoglycoprotein receptor facilitates infection of PLC/PRF/5 cells by HEV through interaction with ORF2. *J. Med. Virol.* 2016, 88, 2186–2195. [CrossRef]

21. Kalia, M.; Chandra, V.; Rahman, S.A.; Sehgal, D.; Jameel, S. Heparan sulfate proteoglycans are required for cellular binding of the hepatitis E virus ORF2 capsid protein and for viral infection. *J. Virol.* 2009, 83, 12714–12724. [CrossRef] [PubMed]

22. Yin, X.; Ambardekar, C.; Lu, Y.; Feng, Z. Distinct entry mechanisms for nonenveloped and quasi-enveloped hepatitis E viruses. *J. Virol.* 2016, 90, 4232–4242. [CrossRef] [PubMed]

23. WuDunn, D.; Spear, P.G. Initial interaction of herpes simplex virus with cells is binding to heparan sulfate. *J. Virol.* 1989, 63, 52–58. [CrossRef] [PubMed]

24. Shieh, M.T.; WuDunn, D.; Montgomery, R.I.; Esko, J.D.; Spear, P.G. Cell surface receptors for herpes simplex virus are heparan sulfate proteoglycans. *J. Cell Biol.* 1992, 116, 1273–1281. [CrossRef] [PubMed]

25. Mondor, I.; Ugolini, S.; Sattentau, Q.J. Human immunodeficiency virus type 1 attachment to HeLa CD4 cells is CD4 independent and gp120 dependent and requires cell surface heparans. *J. Virol.* 1998, 72, 3623–3634. [CrossRef] [PubMed]

26. Barth, H.; Schafer, C.; Adah, M.I.; Zhang, F.; Linhardt, R.J.; Toyoda, H.; Kinoshita-Toyoda, A.; Toida, T.; Van Kuppevelt, T.H.; Depla, E.; et al. Cellular binding of hepatitis C virus envelope glycoprotein E2 requires cell surface heparan sulfate. *J. Biol. Chem.* 2003, 278, 41003–41012. [CrossRef] [PubMed]

27. Tamura, M.; Natori, K.; Kobayashi, M.; Miyamura, T.; Takeda, N. Genogroup II noroviruses efficiently bind to heparan sulfate proteoglycan associated with the cellular membrane. *J. Virol.* 2004, 78, 3817–3826. [CrossRef]

28. Shiota, T.; Li, T.C.; Nishimura, Y.; Yoshizaki, S.; Sugiyama, R.; Shimojima, M.; Saijo, M.; Shimizu, H.; Suzuki, R.; Wakita, T.; et al. Integrin alpha3 is involved in non-enveloped hepatitis E virus infection. *Virology* 2019, 536, 119–124. [CrossRef]

29. Takada, Y.; Ye, X.; Simon, S. The integrins. *Genome Biol.* 2007, 8, 215. [CrossRef]

30. Akula, S.M.; Pramod, N.P.; Wang, F.Z.; Chandran, B. Integrin alpha3beta1 (CD 49c/29) is a cellular receptor for Kaposi’s sarcoma-associated herpesvirus (KSHV/HHV-8) entry into the target cells. *Cell* 2002, 108, 407–419. [CrossRef]

31. Rivera-Serrano, E.E.; González-López, O.; Das, A.; Lemon, S.M. Cellular entry and uncoating of naked and quasi-enveloped human hepatoviruses. *Elife* 2019, 8, e43983. [CrossRef] [PubMed]

32. Choy, M.Y.; Richman, P.I.; Horton, M.A.; MacDonald, T.T. Expression of the VLA family of integrins in human intestine. *J. Pathol.* 1990, 160, 35–40. [CrossRef] [PubMed]

33. Marion, O.; Lhomme, S.; Nayrac, M.; Dubois, M.; Pucelle, M.; Requena, M.; Miguerees, M.; Abravanel, F.; Peron, J.M.; Carrere, N.; et al. Hepatitis E virus replication in human intestinal cells. *Gut* 2020, 69, 901–910. [CrossRef] [PubMed]

34. Oechslin, N.; Moradpour, D.; Gouttenoire, J. Hepatitis E virus finds its path through the gut. *Gut* 2020, 69, 796–798. [CrossRef]

35. de Melker, A.A.; Sterk, L.M.; Delwel, G.O.; Fles, D.L.; Daams, H.; Weening, J.J.; Sonnenberg, A. The A and B variants of the alpha 3 integrin subunit: Tissue distribution and functional characterization. *Lab. Invest.* 1997, 76, 547–563.

36. Volpes, R.; van den Oord, J.J.; Desmet, V.J. Distribution of the VLA family of integrins in normal and pathological human liver tissue. *Gastroenterology* 1991, 101, 200–206. [CrossRef]

37. Volpes, R.; van den Oord, J.J.; Desmet, V.J. Integrins as differential cell lineage markers of primary liver tumors. *Am. J. Pathol.* 1993, 142, 1483–1492.

38. Kapur, N.; Thakral, D.; Durgapal, H.; Panda, S.K. Hepatitis E virus enters liver cells through receptor-dependent clathrin-mediated endocytosis. *J. Viral Hepat.* 2012, 19, 436–448. [CrossRef]

39. Holla, P.; Ahmad, I.; Ahmed, Z.; Jameel, S. Hepatitis E virus enters liver cells through a dynamin-2, clathrin and membrane cholesterol-dependent pathway. *Traffic* 2015, 16, 398–416. [CrossRef]

40. Bubeck, D.; Filman, D.J.; Cheng, N.; Steven, A.C.; Hogle, J.M.; Belnap, D.M. The structure of the poliovirus 135S cell entry intermediate at 10-angstrom resolution reveals the location of an externalized polypeptide that binds to membranes. *J. Virol.* 2005, 79, 7745–7755. [CrossRef]

41. Cavaldesi, M.; Caruso, M.; Sthandier, O.; Amati, P.; Garcia, M.I. Conformational changes of murine polyomavirus capsid proteins induced by sialic acid binding. *J. Biol. Chem.* 2004, 279, 41573–41579. [CrossRef] [PubMed]

42. Sapp, M.; Bienkowska-Haba, M. Viral entry mechanisms: human papillomavirus and a long journey from extracellular matrix to the nucleus. FEBS J. 2009, 276, 7206–7216. [CrossRef] [PubMed]
43. Yin, X.; Feng, Z. Hepatitis E virus entry. Viruses 2019, 11, 883. [CrossRef]
44. Tyagi, S.; Korkaya, H.; Zafarullah, M.; Jameel, S.; Lal, S.K. The phosphorylated form of the ORF3 protein of hepatitis E virus interacts with its non-glycosylated form of the major capsid protein, ORF2. J. Biol. Chem. 2002, 277, 22759–22767. [CrossRef] [PubMed]
45. Drave, S.A.; Debing, Y.; Walter, S.; Todt, D.; Engelmann, M.; Friesland, M.; Wedemeyer, H.; Neyts, J.; Behrendt, P.; Steinmann, E. Extra-hepatic replication and infection of hepatitis E virus in neuronal-derived cells. J. Viral Hepat. 2016, 23, 512–521. [CrossRef]
46. Fu, R.M.; Decker, C.C.; Dao Thi, V.L. Cell culture models for hepatitis E virus. Viruses 2019, 11, 608. [CrossRef]
47. Gouilly, J.; Chen, Q.; Siewiera, J.; Cartron, G.; Levy, C.; Dubois, M.; Al-Daccak, R.; Izopet, J.; Jabrane-Ferrat, N.; El Costa, H. Genotype specific pathogenicity of hepatitis E virus at the human maternal-fetal interface. Nat. Commun. 2018, 9, 4748. [CrossRef]
48. Knegendorf, L.; Drave, S.A.; Dao Thi, V.L.; Debing, Y.; Brown, R.J.P.; Vondran, F.W.R.; Resner, K.; Friesland, M.; Khera, T.; Engelmann, M.; et al. Hepatitis E virus replication and interferon responses in human placental cells. Hepatol. Commun. 2018, 2, 173–187. [CrossRef]
49. Meister, T.L.; Bruening, J.; Todt, D.; Steinmann, E. Cell culture systems for the study of hepatitis E virus. Antiviral Res. 2019, 163, 34–49. [CrossRef]
50. Shukla, P.; Nguyen, H.T.; Torian, U.; Engle, R.E.; Faulk, K.; Dalton, H.R.; Bendall, R.P.; Keane, F.E.; Purcell, R.H.; Emerson, S.U. Cross-species infections of cultured cells by hepatitis E virus and discovery of an infectious virus-host recombinant. Proc/Natl. Acad. Sci. USA 2011, 108, 2438–2443. [CrossRef]
51. Zhou, X.; Huang, F.; Xu, L.; Lin, Z.; de Vrij, F.M.S.; Ayo-Martin, A.C.; van der Kroeg, M.; Zhao, M.; Yin, Y.; Wang, W.; et al. Hepatitis E virus infects neurons and brains. J. Infect. Dis. 2017, 215, 1197–1206. [CrossRef] [PubMed]
52. Kamar, N.; Bendall, R.; Legrand-Abbravanel, F.; Xia, N.S.; Ijaz, S.; Izopet, J.; Dalton, H.R. Hepatitis E. Lancet 2012, 379, 2477–2488. [CrossRef]
53. Debing, Y.; Moradpour, D.; Neyts, J.; Gouttenoire, J. Update on hepatitis E virology: Implications for clinical practice. J. Hepatol. 2016, 65, 200–212. [CrossRef] [PubMed]
54. Geng, Y.; Zhao, C.; Huang, W.; Harrison, T.J.; Zhang, H.; Geng, K.; Wang, Y. Detection and assessment of infectivity of hepatitis E virus in urine. J. Hepatol. 2016, 64, 37–43. [CrossRef] [PubMed]
55. Kamar, N.; Abravanel, F.; Lhomme, S.; Rostaing, L.; Izopet, J. Hepatitis E virus: Chronic infection, extra-hepatic manifestations, and treatment. Clin. Res. Hepatol. Gastroenterol. 2015, 39, 20–27. [CrossRef] [PubMed]
56. Montero, H.; Perez-Gil, G.; Sampieri, C.L. Eukaryotic initiation factor 4A (eIF4A) during viral infections. Virus Genes 2019, 55, 267–273. [CrossRef]
57. Zhou, X.; Xu, L.; Wang, Y.; Wang, W.; Sprengers, D.; Metselaar, H.J.; Peppelenbosch, M.P.; Pan, Q. Requirement of the eukaryotic translation initiation factor 4F complex in hepatitis E virus replication. Antiviral Res. 2015, 124, 11–19. [CrossRef]
58. Glitscher, M.; Himmelsbach, K.; Woytinek, K.; Johne, R.; Reuter, A.; Spiric, J.; Schwaben, L.; Grunweller, A.; Hildt, E. Inhibition of hepatitis E virus spread by the natural compound silvestrol. Viruses 2018, 10, 301. [CrossRef]
59. Todt, D.; Moeller, N.; Praditya, D.; Kinast, V.; Friesland, M.; Engelmann, M.; Verhoye, L.; Sayed, I.M.; Behrendt, P.; Dao Thi, V.L.; et al. The natural compound silvestrol inhibits hepatitis E virus (HEV) replication in vitro and in vivo. Antiviral Res. 2018, 157, 151–158. [CrossRef]
60. Biedenkopf, N.; Lange-Grunweller, K.; Schulte, F.W.; Weisser, A.; Muller, C.; Becker, D.; Becker, S.; Hartmann, R.K.; Grunweller, A. The natural compound silvestrol is a potent inhibitor of Ebola virus replication. Antiviral Res. 2017, 137, 76–81. [CrossRef]
61. Elgner, F.; Sabino, C.; Basic, M.; Ploen, D.; Grunweller, A.; Hildt, E. Inhibition of Zika virus replication by silvestrol. Viruses 2018, 10, 149. [CrossRef] [PubMed]
62. Henss, L.; Scholz, T.; Grunweller, A.; Schnierle, B.S. Silvestrol inhibits Chikungunya virus replication. Viruses 2018, 10, 592. [CrossRef] [PubMed]
63. Muller, C.; Schulte, F.W.; Lange-Grunweller, K.; Obermann, W.; Madhugiri, R.; Pleschka, S.; Ziebuhr, J.; Hartmann, R.K.; Grunweller, A. Broad-spectrum antiviral activity of the eIF4A inhibitor silvestrol against corona- and picornaviruses. Antiviral Res. 2018, 150, 123–129. [CrossRef] [PubMed]

64. Karpe, Y.A.; Lole, K.S. Deubiquitination activity associated with hepatitis E virus putative papain-like cysteine protease. *J. Gen. Virol.* 2011, 92, 2088–2092. [CrossRef]

65. Li, C.; Debing, Y.; Jankevicius, G.; Neyts, J.; Ahel, I.; Coutard, B.; Canard, B. Viral macro domains reverse protein ADP-ribosylation. *J. Virol.* 2016, 90, 8478–8486. [CrossRef]

66. Kanade, G.D.; Pingale, K.D.; Karpe, Y.A. Activities of thrombin and factor Xa are essential for replication of hepatitis E virus and are possibly implicated in ORF1 polyprotein processing. *J. Virol.* 2018, 92, e01853-17. [CrossRef]

67. LeDesma, R.; Nimgaonkar, I.; Ploss, A. Hepatitis E virus replication. *Viruses* 2019, 11, 719. [CrossRef]

68. Szkolnicka, D.; Pollan, A.; Da Silva, N.; Oechslin, N.; Gouttenoire, J.; Moradpour, D. Recombinant hepatitis E viruses harboring tags in the ORF1 protein. *J. Virol.* 2019, 93, e00459-19. [CrossRef]

69. Ding, Q.; Nimgaonkar, I.; Archer, N.F.; Bram, Y.; Heller, B.; Schwartz, R.E.; Ploss, A. Identification of the intragenomic promoter controlling hepatitis E virus subgenomic RNA transcription. *mBio* 2018, 9, e00769-18. [CrossRef]

70. Kanade, G.D.; Pingale, K.D.; Karpe, Y.A. Protein interactions network of hepatitis E virus RNA and polymerase with host proteins. *Front. Microbiol.* 2019, 10, 2501. [CrossRef]

71. Pingale, K.D.; Kanade, G.D.; Karpe, Y.A. Heterogeneous nuclear ribonucleoproteins participate in hepatitis E virus (HEV) replication. *J. Mol. Biol.* 2020, 432, 2369–2387. [CrossRef] [PubMed]

72. Burnham, A.J.; Gong, L.; Hardy, R.W. Heterogeneous nuclear ribonuclear protein K interacts with Sindbis virus nonstructural proteins and viral subgenomic mRNA. *Virology* 2007, 367, 212–221. [CrossRef] [PubMed]

73. Pettit Kneller, E.L.; Connor, J.H.; Lyles, D.S. hnRNPs relocalize to the cytoplasm following infection with vesicular stomatitis virus. *J. Virol.* 2009, 83, 770–780. [CrossRef] [PubMed]

74. Bourai, M.; Lucas-Hourani, M.; Gad, H.H.; Drosten, C.; Jacob, Y.; Tafforeau, L.; Cassonnet, P.; Jones, L.M.; Judith, D.; Couderc, T.; et al. Mapping of Chikungunya virus interactions with host proteins identified nsP2 as a highly connected viral component. *J. Virol.* 2012, 86, 3121–3134. [CrossRef] [PubMed]

75. Brunetti, J.E.; Scolaro, L.A.; Castilla, V. The heterogeneous nuclear ribonucleoprotein K (hnRNP K) is a host factor required for dengue virus and Junin virus multiplication. *Virus Res.* 2015, 203, 84–91. [CrossRef] [PubMed]

76. Poenisch, M.; Metz, P.; Blankenburg, H.; Ruggieri, A.; Lee, J.Y.; Rupp, D.; Rebhan, I.; Diederich, K.; Kaderali, L.; Domingues, F.S.; et al. Identification of hnRNPK as regulator of hepatitis C virus particle production. *PLoS Pathog.* 2015, 11, e1004573. [CrossRef]

77. Paul, D.; Madan, V.; Bartenschlager, R. Hepatitis C virus RNA replication and assembly: Living on the fat of the land. *Cell Host Microbe* 2014, 16, 569–579. [CrossRef]

78. Moradpour, D.; Penin, F.; Rice, C.M. Replication of hepatitis C virus. *Nature Rev. Microbiol.* 2007, 5, 453–463. [CrossRef]

79. Farhat, R.; Seron, K.; Ferlin, J.; Feneant, L.; Goueslain, L.; Jackson, C.L.; Dubuisson, J.; Rouillé, Y. Identification of class II ADP-ribosylation factors as cellular factors required for hepatitis C virus replication. *Cell. Microbiol.* 2016, 18, 1121–1133. [CrossRef]

80. Goueslain, L.; Alsaleh, K.; Horellou, P.; Roingeard, P.; Descamps, V.; Duverlie, G.; Ciczora, Y.; Wychowski, C.; Dubuisson, J.; Rouille, Y. Identification of GBF1 as a cellular factor required for hepatitis C virus RNA replication. *J. Virol.* 2010, 84, 773–787. [CrossRef]

81. Belov, G.A.; Feng, Q.; Nikovics, K.; Jackson, C.L.; Ehrenfeld, E. A critical role of a cellular membrane traffic protein in poliovirus RNA replication. *PLoS Pathog.* 2008, 4, e1000216. [CrossRef] [PubMed]

82. Lanke, K.H.; van der Schaar, H.M.; Belov, G.A.; Feng, Q.; Duijsings, D.; Jackson, C.L.; Ehrenfeld, E.; van Kuppeveld, F.J. GBF1, a guanine nucleotide exchange factor for Arf, is crucial for coxsackievirus B3 RNA replication. *J. Virol.* 2009, 83, 11940–11949. [CrossRef] [PubMed]

83. Wang, J.; Du, J.; Jin, Q. Class I ADP-ribosylation factors are involved in enterovirus 71 replication. *PLoS ONE* 2014, 9, e99768. [CrossRef] [PubMed]

84. Farhat, R.; Ankavay, M.; Lebsir, N.; Gouttenoire, J.; Jackson, C.L.; Wychowski, C.; Moradpour, D.; Dubuisson, J.; Rouille, Y.; Cocquerel, L. Identification of GBF1 as a cellular factor required for hepatitis E virus RNA replication. *Cell. Microbiol.* 2018, 20, e12804. [CrossRef]

85. Verheije, M.H.; Raaben, M.; Mari, M.; Te Lintelo, E.G.; Reggiori, F.; van Kuppeveld, F.J.; Rottier, P.J.; de Haan, C.A. Mouse hepatitis coronavirus RNA replication depends on GBF1-mediated ARF1 activation. *PLoS Pathog.* 2008, 4, e1000088. [CrossRef]

86. Xing, L.; Li, T.C.; Mayazaki, N.; Simon, M.N.; Wall, J.S.; Moore, M.; Wang, C.Y.; Takeda, N.; Wakita, T.; Miyamura, T.; et al. Structure of hepatitis E virion-sized particle reveals an RNA-dependent viral assembly pathway. *J. Biol. Chem.* 2010, *285*, 33175–33183. [CrossRef]

87. Mori, Y.; Matsuura, Y. Structure of hepatitis E viral particle. *Virus Res.* 2011, *161*, 59–64. [CrossRef]

88. Graff, J.; Zhou, Y.H.; Torian, U.; Nguyen, H.; St Claire, M.; Yu, C.; Purcell, R.H.; Emerson, S.U. Mutations within potential glycosylation sites in the capsid protein of hepatitis E virus prevent the formation of infectious virus particles. *J. Virol.* 2008, *82*, 1185–1194. [CrossRef]

89. Montpellier, C.; Wychowski, C.; Sayed, I.M.; Meunier, J.C.; Saliou, J.M.; Ankavay, M.; Bull, A.; Pillez, A.; Abravanel, F.; Helle, F.; et al. Hepatitis E virus lifecycle and identification of 3 forms of the ORF2 capsid protein. *Gastroenterology* 2018, *154*, e218–223.e218. [CrossRef]

90. Yin, X.; Ying, D.; Lhomme, S.; Tang, Z.; Walker, C.M.; Xia, N.; Zheng, Z.; Feng, Z. Origin, antigenicity, and function of a secreted form of ORF2 in hepatitis E virus infection. *Proc. Natl. Acad. Sci. USA* 2018, *115*, 4773–4778. [CrossRef]

91. Ankavay, M.; Montpellier, C.; Sayed, I.M.; Saliou, J.M.; Wychowski, C.; Saas, L.; Duvet, S.; Aliouat-Denis, C.M.; Farhat, R.; de Masson d’Autume, V.; et al. New insights into the ORF2 capsid protein, a key player of the hepatitis E virus lifecycle. *Sci. Rep.* 2019, *9*, 6243. [CrossRef] [PubMed]

92. Lenggenhager, D.; Gouttenoire, J.; Malehmir, M.; Bawohl, M.; Honcharova-Biletska, H.; Kreutzer, S.; Semela, D.; Neuweiler, J.; Hurlimann, S.; Aepli, P.; et al. Visualization of hepatitis E virus RNA and proteins in the human liver. *J. Hepatol.* 2017, *67*, 471–479. [CrossRef] [PubMed]

93. Jameel, S.; Zafrullah, M.; Ozdener, M.H.; Panda, S.K. Expression in animal cells and characterization of the hepatitis E virus structural proteins. *J. Virol.* 1996, *70*, 207–216. [CrossRef] [PubMed]

94. Zafrullah, M.; Ozdener, M.H.; Kumar, R.; Panda, S.K.; Jameel, S. Mutational analysis of glycosylation, membrane translocation, and cell surface expression of the hepatitis E virus ORF2 protein. *J. Virol.* 1999, *73*, 4074–4082. [CrossRef]

95. Emerson, S.U.; Nguyen, H.; Torian, U.; Purcell, R.H. ORF3 protein of hepatitis E virus is not required for replication, virion assembly, or infection of hepatoma cells in vitro. *J. Virol.* 2006, *80*, 10457–10464. [CrossRef]

96. Yamada, K.; Takahashi, M.; Hoshino, Y.; Takahashi, H.; Ichiyama, K.; Nagashima, S.; Tanaka, T.; Okamoto, H. ORF3 protein of hepatitis E virus is essential for virion release from infected cells. *J. Gen. Virol.* 2009, *90*, 1880–1891. [CrossRef]

97. Capelli, N.; Marion, O.; Dubois, M.; Allart, S.; Bertrand-Michel, J.; Lhomme, S.; Abravanel, F.; Izopet, J.; Chapuy-Regaud, S. Vectorial release of hepatitis E virus in polarized human hepatocytes. *J. Virol.* 2019, *93*. [CrossRef]

98. Dao Thi, V.L.; Wu, X.; Belote, R.L.; Andrea, U.; Takacs, C.N.; Fernandez, J.P.; Vale-Silva, L.A.; Prallet, S.; Decker, C.C.; Fu, R.M.; et al. Stem cell-derived polarized hepatocytes. *Nat. Commun.* 2020, *11*, 1677. [CrossRef]

99. Zafrullah, M.; Ozdener, M.H.; Panda, S.K.; Jameel, S. The ORF3 protein of hepatitis E virus is a phosphoprotein that associates with the cytoskeleton. *J. Virol.* 1997, *71*, 9045–9053. [CrossRef]

100. Surjit, M.; Oberoi, R.; Kumar, R.; Lal, S.K. Enhanced alpha1 microglobulin secretion from hepatitis E virus ORF3-expressing human hepatoma cells is mediated by the tumor susceptibility gene 101. *J. Biol. Chem.* 2006, *281*, 8135–8142. [CrossRef]

101. Nagashima, S.; Takahashi, M.; Jirintai; Tanaka, T.; Yamada, K.; Nishizawa, T.; Okamoto, H. A PSAP motif in the ORF3 protein of hepatitis E virus is necessary for virion release from infected cells. *J. Gen. Virol.* 2011, *92*, 269–278. [CrossRef] [PubMed]

102. Nagashima, S.; Takahashi, M.; Jirintai, S.; Tanaka, T.; Nishizawa, T.; Yasuda, J.; Okamoto, H. Tumour susceptibility gene 101 and the vacuolar protein sorting pathway are required for the release of hepatitis E virions. *J. Gen. Virol.* 2011, *92*, 2838–2848. [CrossRef] [PubMed]

103. Garrus, J.E.; von Schwedler, U.K.; Pornillos, O.W.; Morham, S.G.; Zavitz, K.H.; Wang, H.E.; Wettstein, D.A.; Stray, K.M.; Cote, M.; Rich, R.L.; et al. Tsg101 and the vacuolar protein sorting pathway are essential for HIV-1 budding. *Cell* 2001, *107*, 55–65. [CrossRef]

104. Martin-Serrano, J.; Zang, T.; Bieniasz, P.D. HIV-1 and Ebola virus encode small peptide motifs that recruit Tsg101 to sites of particle assembly to facilitate egress. *Nat. Med.* 2001, *7*, 1313–1319. [CrossRef] [PubMed]

105. Vietri, M.; Radulovic, M.; Stenmark, H. The many functions of ESCRTs. *Nat. Rev. Mol. Cell Biol.* 2020, *21*, 25–42. [CrossRef] [PubMed]

106. Nagashima, S.; Jirintai, S.; Takahashi, M.; Kobayashi, T.; Tanggis; Nishizawa, T.; Kouki, T.; Yashiro, T.; Okamoto, H. Hepatitis E virus egress depends on the exosomal pathway, with secretory exosomes derived from multivesicular bodies. *J. Gen. Virol.* **2014**, *95*, 2166–2175. [CrossRef]

107. Nagashima, S.; Takahashi, M.; Jirintai, S.; Tanggis; Kobayashi, T.; Nishizawa, T.; Okamoto, H. The membrane on the surface of hepatitis E virus particles is derived from the intracellular membrane and contains trans-Golgi network protein 2. *Arch. Virol.* **2014**, *159*, 979–991. [CrossRef]

108. Nagashima, S.; Takahashi, M.; Kobayashi, T.; Nishizawa, T.; Nishiyama, T.; Primadharsini, P.P.; Okamoto, H. Characterization of the quasi-enveloped hepatitis E virus particles released by the cellular exosomal pathway. *J. Virol.* **2017**, *91*, e00822-17. [CrossRef]

109. Qi, Y.; Zhang, F.; Zhang, L.; Harrison, T.J.; Huang, W.; Zhao, C.; Kong, W.; Jiang, C.; Wang, Y. Hepatitis E virus produced from cell culture has a lipid envelope. *PLoS ONE* **2015**, *10*, e0132503. [CrossRef]

110. Gouttenoire, J.; Pollan, A.; Abrami, L.; Oechslin, N.; Mauron, J.; Matter, M.; Oppliger, J.; Szkolnicka, D.; Dao Thi, V.L.; van der Goot, F.G.; et al. Palmitoylation mediates membrane association of hepatitis E virus ORF3 protein and is required for infectious particle secretion. *PLoS Pathog.* **2018**, *14*, e1007471. [CrossRef]

111. Blaskovic, S.; Blanc, M.; van der Goot, F.G. What does S-palmitoylation do to membrane proteins? *FEBS J.* **2013**, *280*, 2766–2774. [CrossRef] [PubMed]

112. Chapuy-Regaud, S.; Dubois, M.; Plisson-Chastang, C.; Bonnefois, T.; Lhomme, S.; Bertrand-Michel, J.; You, B.; Simoneau, S.; Gleizes, P.E.; Flan, B.; et al. Characterization of the lipid envelope of exosome encapsulated HEV particles protected from the immune response. *Biochimie* **2017**, *141*, 70–79. [CrossRef] [PubMed]

113. Chahar, H.S.; Bao, X.; Casola, A. Exosomes and their role in the life cycle and pathogenesis of RNA viruses. *Viruses* **2015**, *7*, 3204–3225. [CrossRef] [PubMed]

114. Wang, Y.; Zhou, X.; Debing, Y.; Chen, K.; Van Der Laan, L.J.; Neyts, J.; Janssen, H.L.; Metselaar, H.J.; Peppelenbosch, M.P.; Pan, Q. Calcineurin inhibitors stimulate and mycophenolic acid inhibits replication of hepatitis E virus. *Gastroenterology* **2014**, *146*, 1775–1783. [CrossRef]

115. Wu, X.; Dao Thi, V.L.; Liu, P.; Takacs, C.N.; Xiang, K.; Andrus, L.; Gouttenoire, J.; Moradpour, D.; Rice, C.M. Pan-genotype hepatitis E virus replication in stem cell-derived hepatocellular systems. *Gastroenterology* **2018**, *154*, 663.e7–674.e7. [CrossRef]

116. Meertens, L.; Hafirassou, M.L.; Couderc, T.; Bonnet-Madin, L.; Kril, V.; Kummerer, B.M.; Labeau, A.; Brugier, A.; Simon-Loriere, E.; Burlaud-Gaillard, J.; et al. FHL1 is a major host factor for Chikungunya virus infection. *Nature* **2019**, *574*, 259–263. [CrossRef]

117. Zhang, R.; Kim, A.S.; Fox, J.M.; Nair, S.; Basore, K.; Klimstra, W.B.; Rimkus, R.; Fong, R.H.; Lin, H.; Poddar, S.; et al. Mxra8 is a receptor for multiple arthritogenic alphaviruses. *Nature* **2018**, *557*, 570–574. [CrossRef]

118. Han, J.; Perez, J.T.; Chen, C.; Li, Y.; Benitez, A.; Kandasamy, M.; Lee, Y.; Andrade, J.; tenOever, B.; Manicassamy, B. Genome-wide CRISPR/Cas9 screen identifies host factors essential for influenza virus replication. *Cell Rep.* **2018**, *23*, 596–607. [CrossRef]

119. Richardson, R.B.; Ohlson, M.B.; Eitson, J.L.; Kumar, A.; McDougal, M.B.; Boys, I.N.; Mar, K.B.; De La Cruz-Rivera, P.C.; Douglas, C.; Konopka, G.; et al. A CRISPR screen identifies IFI6 as an ER-resident interferon effector that blocks flavivirus replication. *Nat. Microbiol.* **2018**, *3*, 1214–1223. [CrossRef]

120. Flint, M.; Chatterjee, P.; Lin, D.L.; McMullan, L.K.; Shrivastava-Ranjan, P.; Bergeron, E.; Lo, M.K.; Welch, S.R.; Nichol, S.T.; Tai, A.W.; et al. A genome-wide CRISPR screen identifies N-acetylglucosamine-1-phosphate transferase as a potential antiviral target for Ebola virus. *Nat. Commun.* **2019**, *10*, 285. [CrossRef]

121. V’Kovski, P.; Gerber, M.; Kelly, J.; Pfaender, S.; Ebert, N.; Braga Lagache, S.; Simillion, C.; Portmann, J.; Stalder, H.; Gaschen, V.; et al. Determination of host proteins composing the microenvironment of coronavirus replicase complexes by proximity-labeling. *Elife* **2019**, *8*, e42037. [CrossRef] [PubMed]
