
Microbiological Research 239 (2020) 126527

Contents lists available at ScienceDirect Microbiological Research journal homepage: www.elsevier.com/locate/micres

Colanic acid biosynthesis in *Escherichia coli* is dependent on lipopolysaccharide structure and glucose availability

Chenhui Wang<sup>a,d</sup>, Hailing Zhang<sup>b</sup>, Jianli Wang<sup>a,d</sup>, Shanshan Chen<sup>a,d</sup>, Zhen Wang<sup>a,d</sup>, Lei Zhao<sup>a,d</sup>, Xiaoyuan Wang<sup>a,c,d,*</sup>

<sup>a</sup> State Key Laboratory of Food Science and Technology, Jiangnan University, Wuxi, 214122, China  
<sup>b</sup> Department of Biological Engineering, College of Life Science, Yantai University, Yantai, 264005, China  
<sup>c</sup> International Joint Laboratory on Food Safety, Jiangnan University, Wuxi, 214122, China  
<sup>d</sup> Key Laboratory of Industrial Biotechnology, Ministry of Education, School of Biotechnology, Jiangnan University, Wuxi, 214122, China  

---

### A R T I C L E I N F O

**Keywords:**  
*Escherichia coli*  
Colanic acid  
Lipopolysaccharide  
Transcriptome  
Glucose availability  

---

### A B S T R A C T

Lipopolysaccharide and colanic acid are important forms of exopolysaccharides located on the cell surface of *Escherichia coli*, but their interrelation with the cell stress response is not well understood. In this study, nine mutant strains with different structures of lipopolysaccharide were constructed from *E. coli* MG1655 by deletion of a single gene or multiple genes. All mutant strains did not produce colanic acid when grown in LB medium, but six of them could produce colanic acid when grown either in M9 medium in which glucose is the sole carbon source or in LB medium supplemented with glucose. The results indicate that colanic acid production in *E. coli* is dependent on both lipopolysaccharide structure and glucose availability. However, transcriptional analysis showed that 20 genes related to the colanic acid biosynthesis and the key gene *rcsA* in the Rcs system were all transcriptionally up-regulated in all of the nine mutant strains no matter they were grown in M9 or LB medium. This suggests that the availability of some nucleotide-sugar precursors shared by the biosynthesis of lipopolysaccharide and colanic acid might play a major role in colanic acid production in *E. coli*. Lipopolysaccharide pathway might have a huge priority to colanic acid pathway to use the common precursors; therefore, the colanic acid is not produced in MG1655 and the nine mutants when grown in LB medium. In the six mutant strains that can produce colanic acid in the glucose rich media, the common precursors might be abundant because they were not needed for synthesizing the mutant lipopolysaccharide.

---

### 1. Introduction

Polysaccharides are important constituents of the surface of bacterial cells, playing an essential role in interaction with the environment and essential for cell survival. In *Escherichia coli* cells, lipopolysaccharide (LPS) is one of the crucial polysaccharides which is part of the outer leaflet of the outer membrane, while colanic acid (CA) is one of the extracellular bacterial polysaccharides secreted to the surrounding environment (*Raetz and Whitfield, 2002*; *Schmid et al., 2015*). Both LPS and CA contribute to biofilm formation and architecture (*Danese et al., 2000*). Typically, LPS is composed of a hydrophobic anchor Lipid A, a nonrepeating core oligosaccharide, and O-antigen sugars (*Raetz and Whitfield, 2002*; *Wang and Quinn, 2010*). Due to the mutation in *rfb* gene cluster, which is related to the biosynthesis of O-antigen, *E. coli* K-12 strains such as MG1655 synthesizes LPS without the O-antigen (*Liu and P.R., 1994*). CA is made from a repeat unit of glucose (Glc), galactose (Gal), fucose (Fuc), and glucuronic acid (GlcA) with a specific ratio of 1:2:2:1, modifying with nonstoichiometric acetyl and pyruvate substituents (*Grant et al., 1969*; *Shugar and Sanderson, 1972*). There are different kinds of sugar components in the core oligosaccharide of LPS and the CA, and Glc and Gal exist in both LPS and CA (*Fig. 1*). In LPS, Glc is the important sugar component in the outer core region. In CA, each repeat unit contains one Glc and two Gals (*Raetz and Whitfield, 2002*; *Sutherland, 1969*; *Whitfield, 2006*).

As shown in *Fig. 1*A, there are three operons involved in encoding enzymes for biosynthesis of core oligosaccharide of LPS: *waaD-waaL* (containing four genes *waaD, waaF, waaC* and *waaL*), *waaU-waaQ* (containing ten genes *waaU, waaZ, waaY, waaR, waaO, waaB, waaS, waaP, waaG* and *waaQ*), and *waaA* (*Yethon et al., 1998*). All of the enzymes are performed sequentially. Heptosyltransferases WaaC adds the first heptose (Hep) in inner-core. WaaF and WaaQ add the second

---

* Corresponding author at: State Key Laboratory of Food Science and Technology, Jiangnan University 1800 Lihu Avenue, Wuxi, 214000 China.  
E-mail address: xwang@jiangnan.edu.cn (X. Wang).

https://doi.org/10.1016/j.micres.2020.126527

Received 27 February 2020; Received in revised form 6 June 2020; Accepted 6 June 2020

Available online 18 June 2020

0944-5013/ © 2020 Elsevier GmbH. All rights reserved.

C. Wang, et al.

Microbiological Research 239 (2020) 126527

A

| O-antigen | WaaL | Hep | WaaU | Glc | WaaR | Glc | WaaO | Glc | WaaG | Hep | WaaF | Hep | WaaC | Kdo2-lipidA |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |  |  | WaaQ |  |  |  |  |  |  |
|  |  |  |  |  |  |  | WaaB | Gal |  | WaaY | P | WaaP | P |  |

outer core inner core

Locus tag b3619 b3620 b3621 b3622 b3623 b3624 b3625 b3626 b3627 b3628 b3629 b3630 b3631 b3632 b3633

waaD waaF waaC waaL waaU waaZ waaY waaR waaO waaB waaS waaP waaG waaQ waaA

B

Gal  
|  
GlcA  
|  
Gal  
Fuc — Glc — Fuc — Glc — Fuc — Glc — Fuc — Glc — Fuc — Fuc  

Gal  
|  
GlcA  
|  
Gal  

Gal  
|  
GlcA  
|  
Gal  

Gal  
|  
GlcA  
|  
Gal  

Locus tag b2062 b2061 b2060 b2059 b2058 b2057 b2056 b2055 b2054 b2053 b2052 b2051 b2050 b2049 b2048 b2047 b2046 b2045 b2044 b2043

wza wzb wzc wcaA wcaB wcaC wcaD wcaE wcaF gmd wcaG wcaH wcal cpsB cpsG wcaJ wzxC wcaK wcaL wcaM

C

MG1655 ΔF ΔG Δ(L-G) Δ(L-Q) ΔF Δ(C-Q) Δ(F-Q) Δ(D-Q)

Hep — Glc — Glc — Glc — Hep — Hep — Kdo2-lipidA  
| | | | | | |  
Gal | P | P  

ΔF  
Hep — Kdo2-lipidA  
|  
P  

ΔP  
Glc — Hep — Hep — Kdo2-lipidA  
| |  
P | P  

ΔG  
Hep — Hep — Hep — Kdo2-lipidA  
| | |  
P | P |  

Δ(L-G)  
Hep — Hep — Hep — Kdo2-lipidA  

Δ(L-Q)  
Hep — Hep — Kdo2-lipidA  

Δ(L-Q) ΔF  
Hep — Kdo2-lipidA  

Δ(C-Q)  
Kdo2-lipidA  

Δ(F-Q)  
Kdo2-lipidA  

Δ(D-Q)  
Kdo2-lipidA  

Fig. 1. (A). Structures of core oligosaccharide in LPS and the genes required for synthesizing this core oligosaccharide in *E. coli* MG1655. (B). Structures of CA and the genes required for synthesizing CA in *E. coli* MG1655. (C). LPS purified from different *E. coli* strains were analyzed by using silver-stained SDS-PAGE; the proposed LPS structures in different mutant strains were shown below.

and third Hep residues to the first Hep respectively. And the first and second Hep residues are decorated with phosphate (P) groups by WaaP and WaaY respectively ([Parker et al., 1992](https://doi.org/10.1016/0022-2836(92)90640-8); [Yethon et al., 1998](https://doi.org/10.1074/jbc.273.48.31898)). Glucosyltransferase WaaG adds the first Glc to the second Hep residue ([Yethon et al., 2000](https://doi.org/10.1074/jbc.M004668200)). A Gal group is added to the first Glc by WaaB. Then WaaO and WaaR add two more Glc residues to the first Glc. In the end, WaaU adds the fourth Hep to the third Glc residue ([Qian et al., 2014](https://doi.org/10.1016/j.molimm.2014.05.002)).

The biosynthesis of CA is performed by enzymes encoded by a gene cluster of 20 genes (wza, wzb, wzc, wcaA, wcaB, wcaC, wcaD, wcaE, wcaF, gmd, wcaG, wcaH, wcal, cpsB, wcaJ, wzxC, wcaK, wcaL, and wcaM) (Fig. 1B). The genes encode enzymes for fucose sugar precursors are included in the gene cluster ([Schmid et al., 2015](https://doi.org/10.1016/j.jmb.2015.04.010)). CpsG and CpsB are involved in the biosynthesis of GDP-mannose, which is converted into GDP-fucose via dehydratase Gmd and reductase WcaG ([Andrianopoulos et al., 1998](https://doi.org/10.1074/jbc.273.48.31898)). The genes responsible for the synthesis of other sugar precursors are dispersed in the genome. WcaJ transfers the first Glc unit to the undecaprenol pyrophosphate (Und-PP), which is the first step for the stepwise assembly of the repeat unit ([Johnson and Wilson, 1977](https://doi.org/10.1073/pnas.74.10.4428); [Patel et al., 2012](https://doi.org/10.1016/j.bbamcr.2012.03.005)). Then other sugar groups are transferred by WcaA, WcaC, WcaE, Wcal and WcaL. The order of the transfer steps is not clarified yet and might be mainly based on the sequence similarities ([Stevenson et al., 1996](https://doi.org/10.1074/jbc.271.41.25459)). WcaD, Wza, Wzb and Wzc are involved in the polymerization and export of CA from the inner membrane to the cell surface ([Whitfield, 2006](https://doi.org/10.1016/j.femsre.2006.08.012)).

Many direct and indirect genetic and environmental factors are reported to activate the synthesis of CA, such as membrane integrity, cell envelope stress, osmotic shock, metabolic stress, growth pH, or temperature ([Dolata et al., 2019](https://doi.org/10.1016/j.micres.2019.126088); [El-Kazzaz et al., 2004](https://doi.org/10.1016/j.femsre.2004.06.001); [Navasa et al., 2009](https://doi.org/10.1016/j.femsre.2009.04.004); [Wu et al., 2019](https://doi.org/10.1016/j.micres.2019.126088)). The effect of single gene mutation in the LPS

C. Wang, et al.

Microbiological Research 239 (2020) 126527

Table 1  
Bacterial strains and plasmids used in this study.

| Strains or plasmids | Description                                                                                   | Sources       |
|---------------------|-----------------------------------------------------------------------------------------------|---------------|
| Strains             |                                                                                               |               |
| JM109               | Wild-type *E. coli*                                                                           | NEB           |
| MG1655              | Wild-type *E. coli* K12; F-λ-rph-1                                                             | CGSC 6300     |
| Δ(L-G)              | Derived from MG1655 by deleting the gene cluster containing waaL, waaU, waaZ, waaY, waaR, waaO, waaB, waaP, waaG | This study    |
| Δ(L-Q)              | Derived from MG1655 by deleting the gene cluster containing waaL, waaU, waaZ, waaY, waaR, waaO, waaB, waaP, waaG, waaQ | This study    |
| Δ(L-Q)ΔF            | Derived from Δ(L-Q) by deleting waaF gene                                                      | This study    |
| Δ(C-Q)              | Derived from MG1655 by deleting the gene cluster containing waaC, waaL, waaZ, waaY, waaR, waaO, waaB, waaP, waaG, waaQ | This study    |
| Δ(F-Q)              | Derived from MG1655 by deleting the gene cluster containing waaF, waaC, waaL, waaZ, waaY, waaR, waaO, waaB, waaP, waaG, waaQ | This study    |
| Δ(D-Q)              | Derived from MG1655 by deleting the gene cluster containing waaD, waaF, waaC, waaL, waaZ, waaY, waaR, waaO, waaB, waaP, waaG, waaQ | This study    |
| ΔF                  | Derived from MG1655 by deleting waaF gene                                                      | This study    |
| ΔP                  | Derived from MG1655 by deleting waaP gene                                                      | This study    |
| ΔG                  | Derived from MG1655 by deleting waaG gene                                                      | This study    |
| Δ(L-Q)ΔrcsA         | Derived from Δ(L-Q) by deleting rcsA gene                                                      | This study    |
| Δ(L-Q)ΔrcsF         | Derived from Δ(L-Q) by deleting rcsF gene                                                      | This study    |
| Plasmids            |                                                                                               |               |
| pCas                | repA101(Ts) kan Pcas-cas9 ParaB-Red lacIq Ptrc-sgRNA-pMB1                                        |               |
| pTargetF            | pMB1 aadA sgRNA                                                                               | (Jiang et al., 2015) |
| pTargetF-(L-G)      | pMB1 aadA sgRNA-(L-G)                                                                         | This study    |
| pTargetF-(L-Q)      | pMB1 aadA sgRNA-(L-Q)                                                                         | This study    |
| pTargetF-(C-Q)      | pMB1 aadA sgRNA-(C-Q)                                                                         | This study    |
| pTargetF-(F-Q)      | pMB1 aadA sgRNA-(F-Q)                                                                         | This study    |
| pTargetF-(D-Q)      | pMB1 aadA sgRNA-(D-Q)                                                                         | This study    |
| pTargetF-F          | pMB1 aadA sgRNA-F                                                                             | This study    |
| pTargetF-P          | pMB1 aadA sgRNA-P                                                                             | This study    |
| pTargetF-G          | pMB1 aadA sgRNA-G                                                                             | This study    |
| pTargetF-rcsA       | pMB1 aadA sgRNA-rcsA                                                                          | This study    |
| pTargetF-rcsF       | pMB1 aadA sgRNA-rcsF                                                                          | This study    |

biosynthesis genes cluster on the synthesis of CA has been studied. Deletion of waaF in *E. coli* W3110 maximally could stimulate the production of CA through the Rcs phosphorelay system (Ren et al., 2016). Amino acid deficiency improves the synthesis of CA in Cronobacter sakazakii (Chen et al., 2018). These studies demonstrated that the secretion of CA is influenced by the defection of LPS and the nutritional condition. However, the mechanism of the effect of LPS structure on CA production is still not clear.

In this study, several mutant strains with different LPS structures were constructed from *E. coli* MG1655 and their ability to synthesize CA was tested. In M9 medium, ΔF, ΔP, ΔG, Δ(L-Q)ΔF, Δ(L-Q) and Δ(L-G) produced CA, but not Δ(D-Q), Δ(F-Q) and Δ(C-Q). In LB medium, almost no CA was produced in any mutant cells. Interestingly, the genes related to CA biosynthesis were all significantly up-regulated in these mutant strains, no matter they were grown in LB or M9 medium.

## 2. Materials methods

### 2.1. Bacterial strains and growth conditions

The strains used in this study are listed in Table 1. These strains were usually grown in media of LB (5 g/L yeast extract, 10 g/L tryptone, and 10 g/L NaCl), M9 (17.1 g/L Na₂HPO₄·12H₂O, 3 g/L KH₂PO₄, 4 g/L glucose, 1 g/l NH₄Cl, 0.5 g/L NaCl, 0.24 g/L MgSO₄, and 0.011 g/L CaCl₂), LBG (5 g/L yeast extract, 10 g/L tryptone, 10 g/L NaCl and 4 g/L glucose), or M9M (17.1 g/L Na₂HPO₄·12H₂O, 3 g/L KH₂PO₄, 4.09 g/L glycerol, 1 g/l NH₄Cl, 0.5 g/L NaCl, 0.24 g/L MgSO₄, and 0.011 g/L CaCl₂). 50 mg/L spectinomycin, 50 mg/L kanamycin, 10 mM arabinose, and 0.5 mM isopropyl-β-D-thiogalactopyranoside (IPTG) were added when needed (Zhao et al., 2018).

### 2.2. Construction of *E. coli* mutant strains

All of the primers used in this work are listed in Table 2. Most primer sequences were designed based on the genome of *E. coli* MG1655. Genomic DNA of *E. coli* MG1655 was extracted and used as the template to amplify the genes and their upstream and downstream fragments by PCR. Plasmids and genomic DNA were obtained by using the TIANGEN prep kit (Tiangen, Beijing, China) under the instruction. 2xUltra-Pfu PCR MasterMix Polymerase (Boerdi, Nanjing, China) was used for PCR. Restriction endonucleases and T4DNA ligase were from Thermo Scientific (Waltham, USA). The genes deletion in the chromosome of *E. coli* strains were operated according to the former method (Jiang et al., 2015). Δ(L-G) was derived from MG1655 by the deletion of waaL-waaG gene cluster. The upstream fragment of the gene cluster was PCR amplified using the primer pair U-(L-G)-F and U-(L-G)-R. The downstream fragment was PCR amplified using the primers D-(L-G)-F and D-(L-G)-R. Overlapping PCR was used to link these fragments by primers U-(L-G)-F and D-(L-G)-R. The pTargetF-(L-G) was constructed from pTargetF by inverse PCR using the primers L-G -sgRAN-F and sgRNA-R. After self-ligation, the plasmid was confirmed by PCR amplification using the primers L-G -sgRAN-F-test and sgRNA-R. Following 200 ng pTargetF-(L-G) and 500 ng of the deletion fragment were added to 80 μl MG1655/pCas competent cells and then electroporated. Mutant cells were recovered at 30°C, 200 rpm immediately, and grown on LB agar medium containing 50 mg/L kanamycin and 50 mg/L spectinomycin. The correct mutant strains were selected by PCR using the primers U-L-G-F and D-L-G-R. At last, pTargetF-(L-G) was cured firstly by adding IPTG. Then pCas was cured after grown at 42°C more than 24 h.

Other mutant strains were similarly constructed by using the above method. Δ(L-Q), Δ(C-Q), Δ(F-Q) and Δ(D-Q) were constructed from MG1655 by deleting waaL-waaQ, waaC-waaQ, waaF-waaQ, and waaD-waaQ, respectively. Δ(L-Q)ΔF was constructed from Δ(L-Q) by deleting the waaF gene. ΔF, ΔP and ΔG were constructed from MG1655 by deleting waaF, waaP and waaG respectively. Δ(L-Q) ΔrcsA and Δ(L-Q) ΔrcsF were constructed from Δ(L-Q) by deleting the rcsA and rcsF, respectively.

### 2.3. Identification of exopolysaccharides of *E. coli* strains grown on agar media

The tannin mordant staining and light microscopy of

Table 2  
Primers used in this study.

| Names               | Sequence (5'-3')                                                                 |
|---------------------|---------------------------------------------------------------------------------|
| U-(L-G)-F           | CCCAAGAAAATTCACCTTCCCTCTATTG                                                    |
| U-(L-G)-R           | GCTGACAGGAATGCGACAATTATGATCGCTCACTTCTTTAAACTTGTTTATT                             |
| D-(L-G)-F           | AATAAACAAAGTTTAAGAAGTGAGGGAATCAATTGTGCATTCCTGTGCAGC                              |
| D-(L-G)-R           | GTAGATGAGATIGCACGGAGGTTG                                                       |
| (L-G)-sgRAN-F       | ACCACCTGTAATGATATCCGGTTTTTAGAGCTAGAAATAGC                                       |
| sgRNA-R            | AGTATTATACCTAGGACTGAGCTAG                                                      |
| U-(L-Q)-F          | CCCAAGAAAATTCACCTTCCCTCTATTG                                                    |
| U-(L-Q)-R          | CTACTCCTGTGCATCAGTAGCGCTCAAGCAGCTCACITTAAACTTGTTTAITC                            |
| D-(L-Q)-F          | GAATAAACAAGTTTAAGAAGTGAGCTGTTGAGCGTACTGATGACAGGAGTAG                             |
| D-(L-Q)-R          | GCGGCTGAATAAGGTAGAGAAGGGC                                                       |
| (L-Q)-sgRAN-F      | ATACAGGTGCAACTAAACGGTTTTTAGAGCTAGAAATAGC                                         |
| U-F-F              | GAATATGTTCGTCAAATCCTGCCC                                                        |
| U-F-R              | TTAAGCATCAAAACCCGATCCGGGCTTTATGCGTCGGGATTCAGCCAGG                                |
| D-F-F              | CCTGGCTGAATCGGAGAGCATAAAGAGCCGGATGCGGGTTTTGATCGTTAA                              |
| D-F-R              | CCAGTAAAACCAATCAATTGTCGCC                                                      |
| F-sgRAN-F          | ATGGTGTGCGGAACCTGGGGGTTTTAGAGCTAGAAATAGC                                         |
| U-(C-Q)-F          | CAGAACGTCCGATGATTTGGTTTTTGCCCC                                                  |
| U-(C-Q)-R          | CAATCACACAAAAAGAGTCGCTAGTGGCCGTCAGGCTTCCCTTTGTAACAATAGC                          |
| D-(C-Q)-F          | GCTATTGTTACAAGAGGAAGGCCTGACGGCCACTAGCGACTTTTTGTGTGATTG                           |
| D-(C-Q)-R          | GCGGCTGAATAAGGTAGAGAAGGGC                                                       |
| (C-Q)-sgRAN-F      | TCAGGGGCTGGTAAAAAGGGTTTTAGAGCTAGAAATAGC                                          |
| U-(D-Q)-F          | CCGTTGTTGGTTTTCCGTGGAATCG                                                       |
| U-(D-Q)-R          | CAATCACACAAAAAGAGTCGCTAGTGGGTAACCTCGAATTATGGAGTCAGAGAC                           |
| D-(D-Q)-F          | GTCTCTGACTCCATAATTCGAAGGTTACCCACTAGCGACTTTTTGTGTGATTG                            |
| D-(D-Q)-R          | GCGGCTGAATAAGGTAGAGAAGGGC                                                       |
| (D-Q)-sgRAN-F      | TCAGGGGCTGGTAAAAAGGGTTTTAGAGCTAGAAATAGC                                          |
| U-P-F              | CATCCTAGTTGTTTAATCTCTGGCTAAC                                                   |
| U-P-R              | GATGGTTGAATTTAAAGAGCCGGGTATGTACTGGAACTAACTTTATTATGAC                            |
| D-P-F              | GTCATAATAAAAGTTAGTTCCAGTACATACGGGGCTTTTTAATCAACCATC                              |
| D-P-R              | GTCAGAATTAATGGCAGCCG                                                            |
| P-sgRAN-F          | AAAGAGCGAAACACAAGGCAAGTTTTAGAGCTAGAAATAGC                                        |
| U-G-F              | GTGGCAAATGTAACAGTGGGGTG                                                         |
| U-G-R              | GCTGACAGGAATGCGACAATTATGATCGGATGGTTGAATTTAAAGAGCC                               |
| D-G-F              | GGCTCTTTTAATTCACCCATCCGATCATAATTGTGCATTCCTGTCAGC                                 |
| D-G-R              | GTAGATGAGATIGCACGGAGGTTG                                                        |
| G-sgRAN-F          | TAGCTTGGCAGAAAAAGCCGTTTTAGAGCTAGAAATAGC                                          |
| U-rcsA-F           | GACACTGAATCTGGCATTAGG                                                           |
| U-rcsA-R           | GGAAACCACCAGTCAGAATGTGCGATAACCCTCACTCAATGCGTAACG                                  |
| D-rcsA-F           | CGTTACGCAATTGAGTGGGGGTATGCGACATTCTGACTGGTGGTTTCC                                 |
| D-rcsA-R           | GCGGTTTTCATCATCTTTCCCTCG                                                        |
| rcsA-sgRAN-F       | GCTGACTGTTAGAAGCATCGGGTTTTAGAGCTAGAAATAGC                                        |
| U-rcsF-F           | CGCAAATCGCTCTGGCAGTTATC                                                         |
| U-rcsF-R           | GCTCGAACTGGAAACTGCTCCACGCATAGTATTTCCCTCAATG                                      |
| D-rcsF-F           | GATTGAGGAAATACTATGCGTGGAGCAGTTTCCAGTTCGAGC                                       |
| D-rcsF-R           | CGGCAAAGGGGAGATACGGTTTG                                                         |
| rcsF-sgRAN-F       | CGTGCGGGTTGGAATGCTCGGGTTTTAGAGCTAGAAATAGC                                        |
| 16sRNA-RT-F        | TCGGGAACCGTGAGACAGG                                                             |
| 16sRNA-RT-R        | CCGCTGGCAACAAAGGATAAG                                                           |
| Wza-RT-F           | GATGGGTGAAGTGAAAGAAACA                                                          |
| Wza-RT-R           | GCGGAGGAATGACAAGAGAT                                                           |
| WcaD-RT-F          | TGATTGCCTTAGGGATTG                                                              |
| WcaD-RT-R          | TATCTTGGCGATGGTTGTG                                                             |
| cpsB-RT-F          | CTGAACAAAACTTACCGAGAACA                                                         |
| cpsB-RT-R          | GCATTAACGGGTCGCTCT                                                              |

exopolysaccharides produced by *E. coli* strains were performed using a published method (Ren et al., 2016). *E. coli* cells were grown on M9, LB and LBG agar media at 30°C for 48 h. Cells from pure colonies were transferred to glass slides and fixed by smearing and heat drying, then treated with the prepared fuchsinsolution for 3 min. For 100 mL fuchsinsolution, 0.3 g basic fuchsins, 10 mL 95% ethanol, and 90 mL 5% phenol were added. Then the samples were washed with flowing water and dried by heating. Next, the samples were treated with the mordant solution for 3 min and then washed with flowing water. The mordant solution needs to be prepared presently which contains 0.3 g/L FeCl₃, 1.5 g/L tannins and 2 g/L saturated solution of potassium alumminum sulfate at the volume ratio of 2:2:5. Finally, 1% methylene blue was used for 30 s and visualized by a 100 μm oil microscope. The exopolysaccharide was stained blue and the cells were red.

2.4. Quantitative detection of CA production by various *E. coli* strains

To obtain the CA samples, *E. coli* strains were grown in M9, LB or LBG liquid medium to the mid-log phase and then heated at 100°C for 15 min to deactivate the enzymes and release CA. Next, the samples were cooled to room temperature and centrifuged at 12,000 × g for 10 min. Then the supernatant was added with 3 volumes of ice-cold anhydrous ethanol and sat quietly on ice for 4 h. After the solution became clear and the precipitate was formed at the bottom of the centrifuge tubes, the sample was centrifuged at 10,000 g for 15 min and dried to remove the ethanol. The dried samples were used for determining CA production (Blumenkrantz NAsboe-Hansen, 1973). Each CA sample was dissolved fully in 1 mL deionized water, mixed with 5 mL sodium tetraborate-sulfuric acid solution (0.475 mg/L), and then heated at 100°C for 5 min. After cooling, 100 mL hydroxyl diphenyl

(1.5 g/L) solution was added to the sample and mixed well. Finally, the absorbance was measured at 526 nm. The liquid medium without cells was used as the control and different concentrations of glucuronic acid were used to make a standard curve. The numbers of cells in different samples were determined by serial diluting and plating the culture. The ratio of the colony numbers grown on the plate of the mutant to that of wild type MG1655 was calculated, and used to normalize the final concentration of CA of each sample (Chen et al., 2018).

### 2.5. Whole genome transcriptional analysis of *E. coli* strains

For transcriptome analysis, *E. coli* cells were cultivated in M9 and LB liquid media at 30°C until they reached the mid-log phase (OD<sub>600</sub> around 1.0). Then, the cells were centrifuged at 13,800 × g for 10 min at 4°C and quick-frozen by the liquid nitrogen. The whole RNA samples of the cells were extracted by using GeneTool Total RNA Isolation Kit, and the contaminated DNA was degraded completely. The concentration and purity of the RNA samples were evaluated to ensure the quality for subsequent experimental results. The libraries of RNA were prepared and sequenced using Illumina HiSeq 2000 (GENEWIZ, Suzhou, China). The whole genome transcriptional analysis was performed according to a published method (Marisch et al., 2013). The differentially expressed genes were determined based on their expression levels in the various strains (Mortazavi et al., 2008). The significant variations were determined using the FIESTA Viewer v.1.0 software; the threshold of false discovery rate (FDR) was set to equal to or less than 0.001, the fold change was set to equal to or less than -2 and equal to or more than 2 (Kim and van de Wiel, 2008). Based on the differentially expressed genes, the KEGG (Kyoto Encyclopedia of Genes and Genomes) pathway and three types of GO (Gene ontology) terms analyses were performed (Ball et al., 2005; Kanehisa et al., 2008). Transcriptome data have been deposited in the NCBI Sequence Read Archive database and linked with accession numbers PRJNA603230 (*E. coli* strains grown in M9 medium) and PRJNA603231 (*E. coli* strains grown in LB medium).

### 2.6. LPS extraction and analysis

LPS were extracted using the hot phenol/water method (Westphal and Jann, 1965). 200 mL overnight culture of *E. coli* cells were centrifuged at 13,800 × g for 10 min, resuspended in 10 mL ddH<sub>2</sub>O, and then mixed with 10 mL preheated 90% phenol at 68°C. The mixture was shaken in a water bath at 68°C for 1 h, cooled to room temperature, centrifuged at 13,800 × g for 20 min, and placed at 4°C for 4 h. The supernatant fluid was transferred into dialysis bags, and dialyzed in dd H<sub>2</sub>O for 24 h. The fluid was vacuum freeze-dried to obtain the crude LPS sample. The crude LPS sample was dissolved in 9 mL ddH<sub>2</sub>O and 1 mL reaction buffer (100 mM pH7.5 Tris-HCl, 25 mM MgCl<sub>2</sub>, 1 mM CaCl<sub>2</sub>). Then DnaseI, RNaseA and proteinase K were added to the mixture to remove the impurities. The residual protein was removed using 5 mL water-saturated phenol. The mixture was centrifuged at 13,800 × g for 20 min, and the supernatant was collected, dialyzed and lyophilized to obtain the pure LPS.

The LPS structure was analyzed by SDS-PAGE followed by silver staining (Tsai and Frasch, 1982). 8 μL LPS suspension (2 mg/mL) was transferred into a 1.5 mL centrifuge tube, and 2 μL of 5 × loading buffer was added. The sample was boiled in a water bath for 10 min and then subjected to SDS-PAGE (5% concentrated gel and 12% separated gel) electrophoretic separation. The electrophoresis gel was stained with silver nitrate to visualize LPS.

### 2.7. RNA extraction and transcriptional analysis using real-time PCR

Total RNA was extracted from *E. coli* cells grown at the middle logarithmic growth phase using the RNeasy Total RNA kit (Qiagen, P.R. China), and transcribed into cDNA using HiScript II 1st Strand cDNA Synthesis Kit (Vazyme Biotech, P.R. China) according to the manufacturer's protocol. The mRNA expression was detected by the SYBR green PCR Master Mix (Vazyme Biotech, P.R. China) and performed using the Applied Biosystems step one RT-PCR system (Life Technologies, California, United States). $2^{-\Delta\Delta CT}$ method was used to analyze the relative fold changes in gene expression from Real-time PCR (Livak and Schmittgen, 2001). The 16S rRNA expression was used as an endogenous control. The transcription of the target gene was quantified as the difference in qPCR threshold cycles ($\Delta C_T$). As one PCR cycle represents a twofold difference in template abundance, fold change values are calculated as $2^{-\Delta\Delta CT}$. Three biological replicates were performed. All qPCR primers were listed in Table 2.

### 2.8. Determination of intracellular D-glucose-6-phosphate

D-glucose-6-phosphate (D-Glc-6-P) was extracted from cells according to Antonio’s method (Zhu et al., 2009). The cells (about $2.0 \times 10^8$) were washed twice with 1 mL PBS solution, followed by centrifugation at 13,800 × g for 5 min. Then the supernatant was removed and 1 mL pre-chilled methanol/chloroform mixture (2:1 by volume) was added to the precipitation. After mixing, the mixture was stored at -20°C for 2 h, and then 500 μL 50% methanol solution (4 mM tricine zwitterion buffer, pH 5.4) was added. After centrifugation at 13,800 × g, 4°C for 10 min, the upper aqueous phase was transferred to a clean 1.5 mL centrifuge tube. The fluid was vacuum freeze-dried and then the D-Glc-6-P extract was obtained.

The enzymatic approach was used for quantification of D-Glc-6-P (Gibon et al., 2002; Batchelor and Zhou, 2004). The extract was dissolved in 50 μL ddH<sub>2</sub>O. 10 μL sample was added in a 96-well plate and treated with 90 μL mixed solution containing 50 mM triethanolamine (TEA, pH 7.6), 1.0 mM MgCl<sub>2</sub>, 100 μM NADP<sup>+</sup>, 10 μM resazurin, 0.1 U/mL glucose-6-phosphate dehydrogenase and 0.2 U/mL lipoamide dehydrogenase. After placing at room temperature for 30 min, 96-well plate assays were analyzed using a microplate reader (Bio-Tek Instruments, Winooski, VT, USA) with excitation at 530 nm (± 12.5 nm) and emission at 590 nm (± 17.5 nm). ddH<sub>2</sub>O was used as a control, and a standard curve was made using different concentrations of D-Glc-6-P (0, 0.2, 0.4, 0.6, 0.8, 1.0, 2.0, 3.0, 4.0, 6.0, 8.0 and 10.0 μM).

## 3. Results

### 3.1. Construction of *E. coli* MG1655 LPS core mutants and phenotypic growth analysis

W3110 and MG1655 are two closely related *E. coli* K12 sub-strains. There are only 21 genomic sequence differences between the two strains, including 13 sites with insertion sequences (Hayashi et al., 2006). However, many differentially expressed genes and proteins were observed between the cells of W3110 and MG1655, including genes regulated by RpoS and genes involved in central metabolism—the glycolysis pathway, the tricarboxylic acid cycle and the glyoxylate bypass (Vijayendran et al., 2007). The significant difference in central metabolism could lead to the difference in the generation of precursor metabolites and energy. The significant difference in regulatory expression between W3110 and MG1655 could affect the biosynthesis of LPS and CA. In our previous study, 10 mutant strains were constructed from *E. coli* W3110 by deleting each gene which encoding the enzyme for formation of the core oligosaccharide of LPS (Ren et al., 2016). When grown in M9 medium at 30°C, ΔwaaF, ΔwaaG and ΔwaaP produced CA but the other 7 mutants not. To test if this finding is strain-specific, waaF, waaG and waaP were individually deleted in *E. coli* MG1655 in this study, resulting in the mutants ΔF, ΔG and ΔP. Since the sugar groups on the oligosaccharide of LPS are added in a strict order, and the enzymes catalyze these additions are encoded by genes in a cluster (Fig. 1A), mutant strains with the deletion of multiple genes in the cluster were also constructed (Table 1). For example, Δ(D-Q) was

Fig. 2. Growth curves of different *E. coli* strains grown in the M9, LB and LBG liquid media.

Table 3  
Growth rates ($\mu$) and doubling times ($t_d$) of different *E. coli* strains grown in M9, LB and LBG media during the logarithmic growth phase.

| Strain | $\mu$/h$^{-1}$ | $t_d$/h |
|--------|-----------------|---------|
|        | M9              | M9      |
|        | LB              | LB      |
|        | LBG             | LBG     |
| MG1655 | 0.18 ± 0.03     | 3.877 ± 0.138 |
| $\Delta F$ | 0.11 ± 0.01    | 6.396 ± 0.044 |
| $\Delta P$ | 0.12 ± 0.03    | 5.666 ± 0.198 |
| $\Delta G$ | 0.16 ± 0.01    | 4.416 ± 0.037 |
| $\Delta(L-G)$ | 0.10 ± 0.03   | 6.742 ± 0.239 |
| $\Delta(L-Q)$ | 0.16 ± 0.02    | 4.352 ± 0.095 |
| $\Delta(L-Q)\Delta F$ | 0.11 ± 0.02   | 6.226 ± 0.162 |
| $\Delta(C-Q)$ | 0.17 ± 0.04    | 4.102 ± 0.210 |
| $\Delta(F-Q)$ | 0.15 ± 0.03    | 4.697 ± 0.181 |
| $\Delta(D-Q)$ | 0.12 ± 0.04    | 5.938 ± 0.248 |

| Strain | $\mu$/h$^{-1}$ | $t_d$/h |
|--------|-----------------|---------|
|        | M9              | M9      |
|        | LB              | LB      |
|        | LBG             | LBG     |
| MG1655 | 0.83 ± 0.01     | 0.838 ± 0.007 |
| $\Delta F$ | 0.80 ± 0.01    | 0.864 ± 0.009 |
| $\Delta P$ | 0.83 ± 0.01    | 0.834 ± 0.009 |
| $\Delta G$ | 0.84 ± 0.01    | 0.830 ± 0.005 |
| $\Delta(L-G)$ | 0.82 ± 0.01   | 0.842 ± 0.002 |
| $\Delta(L-Q)$ | 0.83 ± 0.01    | 0.836 ± 0.007 |
| $\Delta(L-Q)\Delta F$ | 0.83 ± 0.01   | 0.836 ± 0.004 |
| $\Delta(C-Q)$ | 0.75 ± 0.05    | 0.925 ± 0.053 |
| $\Delta(F-Q)$ | 0.69 ± 0.01    | 0.998 ± 0.010 |
| $\Delta(D-Q)$ | 0.76 ± 0.01    | 0.918 ± 0.018 |

| Strain | $\mu$/h$^{-1}$ | $t_d$/h |
|--------|-----------------|---------|
|        | M9              | M9      |
|        | LB              | LB      |
|        | LBG             | LBG     |
| MG1655 | 0.98 ± 0.01     | 0.710 ± 0.003 |
| $\Delta F$ | 0.95 ± 0.01    | 0.729 ± 0.004 |
| $\Delta P$ | 0.96 ± 0.01    | 0.720 ± 0.002 |
| $\Delta G$ | 0.97 ± 0.01    | 0.714 ± 0.006 |
| $\Delta(L-G)$ | 0.96 ± 0.01   | 0.719 ± 0.001 |
| $\Delta(L-Q)$ | 0.96 ± 0.01    | 0.720 ± 0.005 |
| $\Delta(L-Q)\Delta F$ | 0.96 ± 0.01   | 0.727 ± 0.012 |
| $\Delta(C-Q)$ | 0.95 ± 0.01    | 0.729 ± 0.009 |
| $\Delta(F-Q)$ | 0.96 ± 0.01    | 0.720 ± 0.006 |
| $\Delta(D-Q)$ | 0.94 ± 0.01    | 0.737 ± 0.003 |

C. Wang, et al.

Microbiological Research 239 (2020) 126527

A M9

| MG1655 | $\Delta F$ | $\Delta P$ | $\Delta G$ |
| --- | --- | --- | --- |
| $\Delta(L-G)$ | $\Delta(L-Q)$ | $\Delta(L-Q)\Delta F$ | $\Delta(C-Q)$ |
| $\Delta(F-Q)$ | $\Delta(D-Q)$ | $\Delta(L-Q)\Delta rcsA$ | $\Delta(L-Q)\Delta rcsF$ |

B LB

| MG1655 | $\Delta F$ | $\Delta P$ | $\Delta G$ |
| --- | --- | --- | --- |
| $\Delta(L-G)$ | $\Delta(L-Q)$ | $\Delta(L-Q)\Delta F$ | $\Delta(C-Q)$ |
| $\Delta(F-Q)$ | $\Delta(D-Q)$ | $\Delta(L-Q)\Delta rcsA$ | $\Delta(L-Q)\Delta rcsF$ |

C LBG

| MG1655 | $\Delta F$ | $\Delta P$ | $\Delta G$ |
| --- | --- | --- | --- |
| $\Delta(L-G)$ | $\Delta(L-Q)$ | $\Delta(L-Q)\Delta F$ | $\Delta(C-Q)$ |
| $\Delta(F-Q)$ | $\Delta(D-Q)$ | $\Delta(L-Q)\Delta rcsA$ | $\Delta(L-Q)\Delta rcsF$ |

Fig. 3. Light microscopic observation of mordant-stained cell colonies of *E. coli* strains grown on M9 (A), LB (B) and LBG (C) solid media for 48 h at 30 °C. Magnification, ×1,000.

C. Wang, et al.

Microbiological Research 239 (2020) 126527

Fig. 4. Comparison of CA production levels of *E. coli* strains grown in M9, LB and LBG liquid media to the mid-log phase. Error bars indicate standard deviations for three parallel samples.

### 3.3. CA production in *E. coli* MG1655 is controlled by the Rcs phosphorelay system

Gram-negative bacteria have developed several envelope stress response factors, such as rpoE, Cpx and Rcs system, to monitor and combat environmental insults ([Raivio, 2005](https://doi.org/10.1128/MMBR.69.2.292-317.2005); [Wall et al., 2018](https://doi.org/10.1016/j.mib.2018.01.001)). The Rcs phosphorelay system is involved in the secretion of CA ([Guo and Sun, 2017](https://doi.org/10.1016/j.mib.2017.01.001)). It includes the response regulator RcsB, the auxiliary regulator RcsA, the transmembrane sensor kinase RcsC, the transmembrane protein RcsD, and the membrane-bound lipoprotein RcsF. Either RcsB or the RcsA-RcsB dimer activates transcription of the *wca* operon involved in CA biosynthesis. RcsA stabilizes RcsA-RcsB complex binding ability and enhances the regulation activity of RcsB ([Wall et al., 2018](https://doi.org/10.1016/j.mib.2018.01.001)). RcsF senses the envelope stress and conveys the signals to RcsC and RcsD in the cytoplasmic membrane ([Konavalova et al., 2016](https://doi.org/10.1016/j.mib.2016.01.001)).

To investigate whether CA production in LPS mutants of *E. coli* MG1655 is regulated by the Rcs system, *rcsA* and *rcsF* were individually deleted in the LPS mutant Δ(L-Q) which could produce a large amount of CA, resulting in the mutants Δ(L-Q) Δ*rcsA* and Δ(L-Q) Δ*rcsF*. Δ(L-Q) Δ*rcsA* and Δ(L-Q) Δ*rcsF* were grown on M9, LB or LBG plates, and the colonies were stained by tannin mordant and observed under light microscopy; no CA production was found around any cells of Δ(L-Q) Δ*rcsA* and Δ(L-Q) Δ*rcsF* (Fig. 3). When grown in M9, LB or LBG liquid medium, the patterns of CA production in Δ(L-Q) Δ*rcsA* and Δ(L-Q) Δ*rcsF* were similar to that in MG1655 (Fig. 4). A small amount of CA was produced when grown in M9 or LBG, and a very small amount of CA was produced when grown in LB. The results suggest that CA production in LPS mutants of *E. coli* MG1655 is closely regulated by the Rcs system.

### 3.4. Transcriptional analysis of genes relevant to CA biosynthesis in LPS mutants of *E. coli* MG1655

To understand the CA production difference in different LPS mutants of *E. coli*, Δ*F*, Δ*P*, Δ(D-Q), Δ(F-Q), Δ(C-Q), Δ(L-Q)Δ*F* and Δ(L-Q) cells were grown to the mid-log phase in either M9 or LB medium, and the whole cell RNAs were isolated from each strain and the transcriptional levels of the genes involved in CA biosynthesis were analyzed (Fig. 6), using MG1655 as a control.

When grown in M9 medium, all the 20 genes related to CA in Δ*F*, Δ*P*, Δ(D-Q), Δ(F-Q), Δ(C-Q), Δ(L-Q) Δ*F* and Δ(L-Q) cells were transcriptionally up-regulated, but the degree of up-regulation in one strain was quite different from the others (Fig. 6A). The 20 genes were most significantly up-regulated in Δ(L-Q), but only slightly up-regulated in Δ*F*. This is puzzling since Δ*F* produced high amount of CA. To address this, the transcriptional levels of three genes *wza*, *wcaD* and *cpsB* in strains Δ*F*, Δ(L-Q) and Δ(C-Q) were analyzed by using RT-PCR. The

Fig. 5. (A). Light microscopic observation of mordant-stained cell colonies of *E. coli* strains grown on M9M solid medium for 48 h at 30 °C. Magnification, × 1,000. (B). Comparison of CA production levels of *E. coli* strains grown in M9M liquid medium to the mid-log phase. Error bars indicate standard deviations for three parallel samples.

C. Wang, et al.

Microbiological Research 239 (2020) 126527

A
![Image Description](image1)

B
![Image Description](image2)

C
![Image Description](image3)

Fig. 6. Transcriptomic analysis showed that most genes involved in CA biosynthesis were up-regulated in mutant strains grown in M9 (A) and LB (B) liquid media at the mid-log phase, using the MG1655 as the control.

results showed that the transcriptional levels of these three key genes were all up-regulated, which is consistent the transcriptomic analysis results (Fig. 6C). However, the transcriptional levels of the three genes in $\Delta F$ were always higher than that in $\Delta(C-Q)$ in the RT-PCR analysis. The relative transcriptional levels of the same genes were dependent on the analytic methods possibly because the cell growth statuses were different when the samples were taken. The growth rates of all the mutant strains in M9 medium were slow, and the growth statuses of different strains in M9 were different, while the samples taken for transcriptome and RT-PCR were collected at one time point in the growth process. The $wzabc$ operon is involved in the secretion of CA, and the transcriptional level of $wza$, $wzb$ and $wzc$ were up-regulated in $\Delta(L-Q)$ by 5.0, 7.2 and 5.4 folds, respectively. When grown in LB medium, all the 20 genes related to CA in $\Delta F$, $\Delta P$, $\Delta(D-Q)$, $\Delta(F-Q)$, $\Delta(C-Q)$, $\Delta(L-Q)$ $\Delta F$ and $\Delta(L-Q)$ cells were also transcriptionally up-regulated, and the degree of up-regulation in all strains was similar (Fig. 6B). Overall, the degree of transcriptional up-regulation of the 20 genes in each strain grown in LB was larger than that grown in M9.

The transcriptional levels of the five genes involved in the Rcs system were also analyzed (Fig. 6), using MG1655 as a control. Among the five genes, only $rcsA$ was up-regulated. When grown in M9, $rcsA$ was significantly up-regulated in $\Delta(L-Q)$ with a $\log_2R$ value of 3.54, but only slightly up-regulated in other mutant strains. When grown in LB, $rcsA$ was significantly up-regulated in all mutant strains with $\log_2R$ values more than 5. The results are consistent with the transcriptional levels of the 20 genes involved in CA biosynthesis in E. coli LPS mutant strains (Fig. 6), suggesting that RcsA played a major role in CA production in E. coli.

Since all the genes related to CA production in LPS mutant strains were transcriptionally up-regulated no matter the strains were grown in M9 or LB medium (Fig. 6), but only some of these strains grown in M9 produced CA (Fig. 3A), there must be reasons why CA were produced in these strains other than the expression of the genes related to CA production.

### 4. Discussion

In our previous study, 10 E. coli mutant strains with defects in LPS core biosynthesis were constructed from W3110 by deleting the genes $waaF$, $waaG$, $waaP$, $waaB$, $waaU$, $waaR$, $waaO$, $waaC$, $waaQ$, and $waaY$. When these strains were grown in M9, $\Delta waaF$, $\Delta waaG$ and $\Delta waaP$ produced large amounts of CA, but not others. It is believed that the special LPS structures in these mutant strains might activate the expression of the 20 genes related to CA biosynthesis through the Rcs phosphorelay system, because CA could not be synthesized any more when the key genes included in the Rcs system were deleted in the $\Delta waaF$ background (Ren et al., 2016). To further investigate the mechanism of the role of LPS structure on CA production in E. coli, more mutant strains with defects in LPS core biosynthesis were constructed from MG1655 in this study. MG1655 was chosen instead of W3110 to see if the influence of LPS structure on CA production is the same in different E. coli strains. The results showed that $waaF$, $waaG$, and $waaP$

mutant strains cannot produce CA possibly because the genes related to CA biosynthesis are not expressed.

In *E. coli*, the long biosynthetic pathways of LPS and CA involve many reactions. Since both LPS and CA contain oligosaccharide, they share some nucleotide-sugar precursors during their biosynthetic process, such as UDP-α-D-Glc and UDP-α-D-Gal (Fig. 7). UDP-α-D-Glcs are needed for the LPS outer-core biosynthesis in several reactions catalyzed by glucosyltransferases WaaG, WaaO, and WaaR. UDP-α-D-Gal is required for the LPS outer-core biosynthesis in the reaction catalyzed by glucosyltransferase WaaB. UDP-α-D-Glc, UDP-α-D-Gal, UDP-α-D-GlcA are derived from UDP-α-D-Glc, and GDP-β-L-Fuc derived from D-Glc-6-P (Fig. 7). Therefore, CA biosynthesis in *E. coli* needs large amounts of intracellular D-Glc-6-P. CA molecule contains much more Glc and Gal than LPS molecule (Fig. 1). When grown in LB medium, either the wild type *E. coli* or LPS mutant strains could not produce CA, possibly because there were not much D-Glc-6-P available for CA synthesis in the cell.

The intracellular level of D-Glc-6-P, the key precursor for both CA and LPS, in MG1655 and Δ(L-Q) were analyzed (Fig. 7C). At the middle logarithmic growth phase, the levels of D-Glc-6-P in Δ(L-Q) cells were always higher than that in MG1655 cells, no matter they were grown in LB, M9, or LBG, possibly because Δ(L-Q) cells do not need to synthesize the polysaccharide portion of LPS and therefore saved large amounts of D-Glc-6-P. At the late logarithmic growth phase, the levels of D-Glc-6-P in Δ(L-Q) cells were much higher than that in MG1655 cells when they were grown in M9 and LBG media, but similar levels of D-Glc-6-P were observed in Δ(L-Q) and MG1655 cells grown in LB medium. When the cells were grown in M9 or LBG which contains a high concentration of glucose, the common precursors such as D-Glc-6-P might be highly accumulated. In wild type *E. coli*, the common precursors could be used for LPS biosynthesis. While in some *E. coli* LPS mutant strains, such as Δ(L-Q), the common precursors cannot be used for LPS biosynthesis due to the deletion of relevant genes but be used for CA biosynthesis (Fig. 7). A simple sugar, such as glucose, used as the sole carbon source in a minimal medium usually yields relatively higher production of exopolysaccharide, because the more complex is the carbohydrate substrate in the medium, the more difficult is it for the bacterium to utilize (Lin et al., 2016).

The LPS mutant strains Δ(D-Q), Δ(F-Q), and Δ(C-Q) could not produce CA when grown in M9 medium possibly because their LPS structure was damaged too much that the accumulated common precursors might be used to combat the cell envelope damage. Although the above hypotheses need further experimental verification, our results at least indicate that the response to LPS structural change may represent a broad strategy for *E. coli* to survive better. Such strategy may require the regulation among various polysaccharides on the bacterial cell surface, which might be closely related to biofilm formation and pathogenicity (Mack et al., 2009; Lee et al., 2016).

Compared with the previously publication (Ren et al., 2016), this study provides further understanding of the induction of CA production in some LPS defective strains. Firstly, the induction of CA production in the same LPS defective mutant strains was observed in either W3110 or MG1655. Secondly, more CA was produced in the multiple genes deletion mutant Δ(L-Q) than in the single gene deletion mutant ΔF. Thirdly, the genes related to CA biosynthesis were significantly upregulated in all LPS mutant strains, no matter they produced CA or not. This suggests that CA production in some LPS defective strains might be caused by the altered intracellular metabolism.

**Author contributions**

Chenhui Wang and Xiaoyuan Wang conceived and designed the experiments. Chenhui Wang, Hailing Zhang, Zhen Wang, and Si Chen conducted the experiments. Chenhui Wang and Xiaoyuan Wang analyzed data. Xiaoyuan Wang, Jianli Wang, and Lei Zhao contributed reagents, materials, and analysis tools. Chenhui Wang and Xiaoyuan

Wang wrote the paper. All authors read and approved the manuscript.

## Funding information

This work was supported by the National Key R&D Program of China (2017YFC1600102) and the National First-class Discipline Program of Light Industry Technology and Engineering (LITE2018-10).

## Compliance with ethics requirements

This article does not contain any studies with human participants or animals.

## Declaration of Competing Interest

The authors declare no conflict of interest.

## References

Andrianopoulos, K., Wang, L., Reeves, P.R., 1998. Identification of the fucose synthetase gene in the colanic acid gene cluster of *Escherichia coli* K-12. J. Bacteriol. 180 (4), 998–1001.

Ball, C.A., Awad, I.A., Demeter, J., Gollub, J., Hebert, J.M., Hernandez-Boussard, T., Jin, H., Matese, J.C., Nitzberg, M., Wymore, F., Zachariah, Z.K., Brown, P.O., Sherlock, G., 2005. The Stanford Microarray Database accommodates additional microarray platforms and data formats. Nucleic Acids Res. 33 (Database issue), D580–2.

Batchelor, R.H., Zhou, M., 2004. Use of cellular glucose-6-phosphate dehydrogenase for cell quantitation: applications in cytotoxicity and apoptosis assays. Anal. Biochem. 329 (1), 35–42.

Blumenkrantz NASboe-Hansen, G., 1973. New method for quantitative determination of uronic acids. Anal. Biochem. 54 (2), 484–489.

Chen, S., Zhou, Q., Tan, X., Li, Y., Ren, G., Wang, X., 2018. The global response of *Cronobacter sakazakii* cells to amino acid deficiency. Front. Microbiol. 9, 1875.

Danese, P.N., Pratt, L.A., Kolter, R., 2000. Exopolysaccharide production is required for development of *Escherichia coli* K-12 biofilm architecture. J. Bacteriol. 182 (12), 3593–3596.

Dolata, K.M., Montero, I.G., Miller, W., Sievers, S., Sura, T., Wolff, C., Schluter, R., Riedel, K., Robinson, C., 2019. Far-reaching cellular consequences of tat deletion in *Escherichia coli* revealed by comprehensive proteome analyses. Microbiol. Res. 218, 97–107.

El-Kazzaz, W., Morita, T., Tagami, H., Inada, T., Aiba, H., 2004. Metabolic block at early stages of the glycolytic pathway activates the Rcs phosphorelay system via increased synthesis of dTDP-glucose in *Escherichia coli*. Mol. Microbiol. 51 (4), 1117–1128.

Gibon, Y., Vigeolas, H., Tiessen, A., Geigenberger, P., Stitt, M., 2002. Sensitive and high throughput metabolite assays for inorganic pyrophosphate, ADPGlc, nucleotide phosphates, and glycolytic intermediates based on a novel enzymic cycling system. Plant J. 30 (2), 221–235.

Grant, W.D., Sutherland, I.W., Wilkinson, J.F., 1969. Exopolysaccharide colanic acid and its occurrence in the Enterobacteriaceae. J. Bacteriol. 100 (3), 1187–1193.

Guo, X.P., Sun, Y.C., 2017. New insights into the non-orthodox two component rcs phosphorelay system. Front. Microbiol. 8, 2014.

Hagiwara, D., Sugiura, M., Oshima, T., Mori, H., Aiba, H., Yamashino, T., Mizuno, T., 2003. Genome-wide analyses revealing a signaling network of the RcsC-YojN-RcsB phosphorelay system in *Escherichia coli*. J. Bacteriol. 185 (19), 5735–5746.

Hayashi, K., Morooka, N., Yamamoto, Y., Fujita, K., Isono, K., Choi, S., Ohtsubo, E., Baba, T., Wanner, B.L., Mori, H., Horiuchi, T., 2006. Highly accurate genome sequences of *Escherichia coli* K-12 strains MG1655 and W3110. Mol. Syst. Biol. 2 2006.0007.

Jiang, Y., Chen, B., Duan, C., Sun, B., Yang, J., Yang, S., 2015. Multigene editing in the *Escherichia coli* genome via the CRISPR-Cas9 system. Appl. Environ. Microbiol. 81 (7), 2506–2514.

Johnson, J.G., Wilson, D.B., 1977. Role of a sugar-lipid intermediate in colanic acid synthesis by *Escherichia coli*. J. Bacteriol. 129 (1), 225–236.

Kanehisa, M., Araki, M., Goto, S., Hattori, M., Hirakawa, M., Itoh, M., Katayama, T., Kawashima, S., Okuda, S., Tokimatsu, T., Yamanishi, Y., 2008. KEGG for linking genomes to life and the environment. Nucleic Acids Res. 36 (Database issue), D480–4.

Kim, K.I., van de Wiel, M.A., 2008. Effects of dependence in high-dimensional multiple testing problems. BMC Bioinf. 9, 114.

Konovalova, A., Mitchell, A.M., Silhavy, T.J., 2016. A lipoprotein/beta-barrel complex monitors lipopolysaccharide integrity transducing information across the outer membrane. Elife 5.

Lee, K.J., Lee, M.A., Hwang, W., Park, H., Lee, K.H., 2016. Decylated lipopolysaccharides

inhibit biofilm formation by Gram-negative bacteria. Biofouling 32 (7), 711–723.

Lin, B.X., Qiao, Y., Shi, B., Tao, Y., 2016. Polysialic acid biosynthesis and production in *Escherichia coli*: current state and perspectives. Appl. Microbiol. Biotechnol. 100 (1), 1–8.

Liu, D.R., P.R, 1994. *Escherichia coli* K12 regains its O antigen. Microbiology 140, 49–57.

Livak, K.J., Schmittgen, T.D., 2001. Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods 25 (4), 402–408.

Mack, D., Davies, A.P., Harris, L.G., Knobloch, J.K., Rohde, H., 2009. *Staphylococcus epidermidis* biofilms: functional molecules, relation to virulence, and vaccine potential. Top. Curr. Chem. 288, 157–182.

Marisch, K., Bayer, K., Scharl, T., Mairhofer, J., Krempl, P.M., Hummel, K., Razzazi-Fazeli, E., Striedner, G., 2013. A comparative analysis of industrial *Escherichia coli* K-12 and B strains in high-glucose batch cultivations on process-, transcriptome- and proteome level. PLoS One 8 (8), e70516.

Mortazavi, A., Williams, B.A., McCue, K., Schaeffer, L., Wold, B., 2008. Mapping and quantifying mammalian transcriptomes by RNA-Seq. Nat. Methods 5 (7), 621–628.

Navasa, N., Rodriguez-Aparicio, L., Martinez-Blanco, H., Arcos, M., Ferrero, M.A., 2009. Temperature has reciprocal effects on colanic acid and polysialic acid biosynthesis in *E. coli* K92. Appl. Microbiol. Biotechnol. 82 (4), 721–729.

Parker, C.T., Kloser, A.W., Schnaitman, C.A., Stein, M.A., Gottesman, S., Gibson, B.W., 1992. Role of the *rfag* and *rfap* genes in determining the lipopolysaccharide core structure and cell surface properties of *Escherichia coli* K-12. J. Bacteriol. 174 (8), 2525–2538.

Patel, K.B., Toh, E., Fernandez, X.B., Hanuszkiewicz, A., Hardy, G.G., Brun, Y.V., Bernards, M.A., Valvano, M.A., 2012. Functional characterization of UDP-glucose:undecaprenyl-phosphate glucose-1-phosphate transferases of *Escherichia coli* and *Caulobacter crescentus*. J. Bacteriol. 194 (10), 2646–2657.

Qian, J., Garrett, T.A., Raetz, C.R., 2014. In vitro assembly of the outer core of the lipopolysaccharide from *Escherichia coli* K-12 and *Salmonella typhimurium*. Biochemistry 53 (8), 1250–1262.

Raetz, C.R., Whitfield, C., 2002. Lipopolysaccharide endotoxins. Annu. Rev. Biochem. 71, 635–700.

Raivio, T.L., 2005. Envelope stress responses and Gram-negative bacterial pathogenesis. Mol. Microbiol. 56 (5), 1119–1128.

Ren, G., Wang, Z., Li, Y., Hu, X., Wang, X., 2016. Effects of lipopolysaccharide core sugar deficiency on colanic acid biosynthesis in *Escherichia coli*. J. Bacteriol. 198 (11), 1576–1584.

Schmid, J., Sieber, V., Rehm, B., 2015. Bacterial exopolysaccharides: biosynthesis pathways and engineering strategies. Front. Microbiol. 6, 496.

Shugar, S., Sanderson, K.E., 1972. Characterization of a mucoid *Escherichia coli* K12 strain, and chemical analysis of the exopolysaccharide. Can. J. Microbiol. 18 (7), 969–973.

Stevenson, G., Andrianopoulos, K., Hobbs, M., Reeves, P.R., 1996. Organization of the *Escherichia coli* K-12 gene cluster responsible for production of the extracellular polysaccharide colanic acid. J. Bacteriol. 178 (16), 4885–4893.

Sutherland, I.W., 1969. Structural studies on colanic acid, the common exopolysaccharide found in the enterobacteriaceae, by partial acid hydrolysis. Oligosaccharides from colanic acid. Biochem. J. 115 (5), 935–945.

Vijayendran, C., Polen, T., Wendisch, V.F., Friehs, K., Niehaus, K., Flaschel, E., 2007. The plasticity of global proteome and genome expression analyzed in closely related W3110 and MG1655 strains of a well-studied model organism, *Escherichia coli*-K12. J. Biotechnol. 128 (4), 747–761.

Wall, E., Majdalani, N., Gottesman, S., 2018. The complex rcs regulatory cascade. Annu. Rev. Microbiol. 72, 111–139.

Wang, X., Quinn, P.J., 2010. Lipopolysaccharide: biosynthetic pathway and structure modification. Prog. Lipid Res. 49 (2), 97–107.

Westphal, O., Jann, K., 1965. Bacterial lipopolysaccharides. Extraction with phenol-water and further applications of the procedure. Methods Carbohydr. Chem. (5), 83–91.

Whitfield, C., 2006. Biosynthesis and assembly of capsular polysaccharides in *Escherichia coli*. Annu. Rev. Biochem. 75, 39–68.

Wu, H., Chen, S., Ji, M., Chen, Q., Shi, J., Sun, J., 2019. Activation of colanic acid biosynthesis linked to heterologous expression of the polyhydroxybutyrate pathway in *Escherichia coli*. Int. J. Biol. Macromol. 128, 752–760.

Yethon, J.A., Heinrichs, D.E., Monteiro, M.A., Perry, M.B., Whitfield, C., 1998. Involvement of *waaY*, *waaQ*, and *waaP* in the modification of *Escherichia coli* lipopolysaccharide and their role in the formation of a stable outer membrane. J. Biol. Chem. 273 (41), 26310–26316.

Yethon, J.A., Vinogradov, E., Perry, M.B., Whitfield, C., 2000. Mutation of the lipopolysaccharide core glycosyltransferase encoded by *waaG* destabilizes the outer membrane of *Escherichia coli* by interfering with core phosphorylation. J. Bacteriol. 182 (19), 5620–5623.

Zhao, H., Fang, Y., Wang, X., Zhao, L., Wang, J., Li, Y., 2018. Increasing L-threonine production in *Escherichia coli* by engineering the glyoxylate shunt and the L-threonine biosynthesis pathway. Appl. Microbiol. Biotechnol. 102 (13), 5505–5518.

Zhu, A.P., Romero, R., Petty, H.R., 2009. An enzymatic fluorimetric assay for glucose-6-phosphate: application in an in vitro Warburg-like effect. Anal. Biochem. 388 (1), 97–101.
