
Applied Microbiology and Biotechnology
https://doi.org/10.1007/s00253-020-10794-7

MINI-REVIEW

**Motility of Vibrio spp.: regulation and controlling strategies**

Fazlurrahman Khan${}^{1}$ · Nazia Tabassum${}^{2}$ · Raksha Anand${}^{3}$ · Young-Mog Kim${}^{1,4}$

Received: 23 May 2020 / Revised: 15 July 2020 / Accepted: 19 July 2020

© Springer-Verlag GmbH Germany, part of Springer Nature 2020

### Abstract

Flagellar motility in bacteria is a highly regulated and complex cellular process that requires high energy investment for movement and host colonization. Motility plays an important role in the lifestyle of *Vibrio* spp. in the aquatic environment and during host colonization. Flagellar motility in vibrios is associated with several cellular processes, such as movement, colonization, adhesion, biofilm formation, and virulence. The transcription of all flagella-related genes occurs hierarchically and is regulated positively or negatively by several transcription factors and regulatory proteins. The flagellar regulatory hierarchy is well studied in *Vibrio cholerae* and *Vibrio parahaemolyticus*. Here, we compared the regulatory cascade and molecules involved in the flagellar motility of *V. cholerae* and *V. parahaemolyticus* in detail. The evolutionary relatedness of the master regulator of the polar and lateral flagella in different *Vibrio* species is also discussed. Although they can form symbiotic associations of some *Vibrio* species with humans and aquatic organisms can be harmed by several species of *Vibrio* as a result of surface contact, characterized by flagellar movement. Thus, targeting flagellar motility in pathogenic *Vibrio* species is considered a promising approach to control *Vibrio* infections. This approach, along with the strategies for controlling flagellar motility in different species of *Vibrio* using naturally derived and chemically synthesized compounds, is discussed in this review.

### Key points

- *Vibrio* species are ubiquitous and distributed across the aquatic environments.
- The flagellar motility is responsible for the chemotactic movement and initial colonization to the host.
- The transition from the motile into the biofilm stage is one of the crucial events in the infection.
- Several signaling pathways are involved in the motility and formation of biofilm.
- Attenuation of motility by naturally derived or chemically synthesized compounds could be a potential treatment for preventing *Vibrio* biofilm-associated infections.

### Keywords

Biofilm · Molecules · Motility · Regulation · Transcription factors · *Vibrio* species

---

### Introduction

*Vibrio* spp. are Gram-negative motile bacteria, present in both pathogenic and non-pathogenic forms in nature, and have been reported ubiquitously throughout aquatic environments (Osunla and Okoh 2017; Yildiz and Visick 2009). Some pathogenic *Vibrio* spp. are virulent in invertebrates and humans (Austin 2010; Bhunia 2008), whereas some non-pathogenic species exhibit symbiotic relationships with eukaryotic hosts (Ruby et al. 2005). The ability of *Vibrio* spp. to adjust and survive in eukaryotic hosts, and withstand different aquatic environmental conditions, has been attributed to the formation of biofilms (Lutz et al. 2013; Yildiz and Visick 2009). Biofilm is an assemblage of bacteria encased within self-produced extracellular polymeric substances (EPS), which allows for nutrient uptake and hinders the entry of antimicrobial agents

Published online: 20 August 2020
(Donlan and Costerton 2002; Hathroubi et al. 2017). Biofilm formation occurs either inside a colonized host or by attachment to a physical surface in the case of aquatic plankton (Dang and Lovell 2015; Kim et al. 2005). Previous studies showed that the flagellar motility plays an important role in the formation of biofilm in the life cycle of vibrios (Echazarreta and Klose 2019). Although flagellum is very important for the motility and near-surface motility, the main appendages use for the initial attachment are type IV pili such as mannose-sensitive hemagglutinin (MSHA) (Chiavelli et al. 2001), chitin-regulated pili (ChiRP) (Meibom et al. 2004), and toxin co-regulated pili (TCP) (Reguera and Kolter 2005) or adhesins named as flagellum-regulated hemagglutinin (FrhA) (Syed et al. 2009). Reports showed that along with the flagellum, the structural appendage such as pili and exopolysaccharides (produced by themselves) in vibrios, act as structural components for the synthesis of the biofilm (Yildiz and Visick 2009).

Different flagellar morphologies have been observed in different species of *Vibrio*, such as a single sheathed polar flagellum, dual flagella (i.e., sheathed polar and non-sheathed lateral flagella), peritrichous flagella, and a tuft of polar sheathed flagella (Zhu et al. 2013). The polar flagellum is employed for swimming movement in the liquid environment, whereas the lateral flagellum is used for swarming, which occurs once the cells encounter a solid surface or viscous environment (Stewart and McCarter 2003). The flagellar genes responsible for the synthesis of flagellar components are highly regulated and their transcription has been found to occur in a stepwise fashion, i.e., initial synthesis of export component and subsequent synthesis of flagellin proteins (Stewart and McCarter 2003; Syed et al. 2009; Zhu et al. 2013). Such hierarchical transcription of flagellar genes has been studied mostly in *V. cholerae* and *V. parahaemolyticus* (Fig. 1). The hierarchical transcription of both polar (*V. cholerae* and *V. parahaemolyticus*) and lateral (*V. parahaemolyticus*) flagellar genes has been found to be transcribed by RNA polymerase containing alternate sigma factors, such as $\sigma^{54}$ and $\sigma^{28}$ (Prouty et al. 2001; Stewart and McCarter 2003; Syed et al. 2009). The expression of different classes of flagellar genes is a complex regulatory process and is also known to be regulated by $\sigma^{54}$- and $\sigma^{28}$-dependent transcriptional regulators, including several other modulating signaling molecules (Millikan and Ruby 2003; Stewart and McCarter 2003; Tischler and Camilli 2004).

Both pathogenic and non-pathogenic vibrios benefit from flagellar motility for initial colonization on the surface of their host to engage in either a pathogenic (Correa et al. 2000; Gu et al. 2019) or symbiotic (Millikan and Ruby 2003) relationship, respectively. It has been shown that flagellar motility is one of the most highly regulated and high energy-utilizing cellular processes linked to host colonization, chemotaxis, adhesion, biofilm formation, and virulence in vibrios (Butler and

Camilli 2004; Echazarreta and Klose 2019; Gardel and Mekalanos 1996; McCarter 2004; Syed et al. 2009; Yildiz and Visick 2009). Hence, understanding the regulatory mechanisms involved in flagellar motility in vibrios is an important area of research into minimizing human infection, i.e., vibriosis, and neutralizing the negative environmental impact of Vibrios. The signaling cascade involved in flagellar motility has been well studied in *V. cholerae* and *V. parahaemolyticus*. Therefore, the first part of this review paper is focused on a comparative description of the signaling mechanisms involved in the flagellar motility of *V. cholerae* and *V. parahaemolyticus*.

Upon motility-encoded gene expression and activation, other genes and regulatory systems associated with biofilm formation, such as (1) exopolysaccharide production, (2) virulence properties, (3) quorum sensing (QS), and (4) c-di-GMP signaling, are in turn triggered, priming bacteria for host colonization and biofilm establishment (Moisi et al. 2009; Waters et al. 2008). Since surface attachment plays a determining role in the initial stage of bacterial biofilm formation, as well as the virulence properties, attenuating this stage has been proposed as a potential alternative antibiofilm strategy (Beloin et al. 2014; Khan et al. 2019b). Biofilm formation by pathogenic bacteria can withstand most commonly used antibiotics, and consequently, recent strategies to inhibit biofilm formation have shifted towards the use of compounds that are isolated from natural sources (e.g., plants and microorganisms), or synthesized by chemical methods, and have been previously exploited for their antimicrobial and antibacterial activities (Charway et al. 2019; Eom et al. 2014; Lu et al. 2019; Luo et al. 2017; Savoia 2012; Worthington et al. 2012). The use of these compounds as alternatives to conventional antibiotics provides more availability and safety for clinical applications. More importantly, these alternative drug treatments can reduce the risk of future drug resistance by inhibiting multiple targets that are closely associated with biofilm formation stages, such as (1) QS, (2) component of biofilm matrix, and (3) virulence properties (Koo et al. 2017; Roy et al. 2018). Therefore, in an attempt to exploit their potential for inhibiting bacterial motility which facilitates biofilm formation in *Vibrio* spp., current anti-motility strategies employing a wide range of natural and chemically synthesized compounds have been extensively studied. Thus, the second part of this review describes the possible roles of naturally isolated or chemically synthesized compounds in attenuating motility in different species of *Vibrio*.

# Regulation of motility in *V. cholerae* and *V. parahaemolyticus*

Flagellar motility is an energetically expensive phenotypic process (Ni et al. 2020) and plays several important roles in
Appl Microbiol Biotechnol

Class I                      Class II                     Class III                    Class IV
Polar flagella of
V. cholerae
a
flrA
FlrA + σ⁵⁴
flrBC
fliEFGHIJ
flhAFG
fliA (σ²⁸)
FlrC + σ⁵⁴
FliA (σ²⁸)
fliKLMNOPQ
flgBCDEFGHIJ
motY
flgKL
flaA
flhB
flgOP
flgT
cheR-2
VC1384
flaB
flaC
flaD
flaE
flgMN
motAB
motX
cheV-3
cheV-4
MCPs
(flaGfliDflaIfliS)

Polar flagella of
V. parahaemolyticus
b
flaK
FlaK + σ⁵⁴
flaLM
fliEFGHIJKLMNOPQR
flhB
flhAFG
cheYZABorf1orf2 cheWorf3
fliA (σ²⁸)
FlaM + σ⁵⁴
FliA (σ²⁸)
flgAMN
flgBCDEFGHIJ
flgKL
motY
motAB
motX
flaAGHIJK
flaB
flaDE
cheVR
flgMN

Lateral Flagella of
V. parahaemolyticus
c
lafK
motY
fliEFGHIJ
fliMNPQRflhAB
LafK+σ⁵⁴
flgAMN
flgBCDEFGHIJ
fliDSTKLA
motAB
motY
lafK
fliEFGHIJ
FliA (σ²⁸)
flgKL
lafA
flgMN
fliDSTKLA
motAB

Fig. 1 Comparative flagellar gene transcription hierarchies in V. cholerae and V. parahaemolyticus. a Four levels (classes I, II, III, and IV) of polar flagellar hierarchy in V. cholerae (Correa et al. 2005; Echazarreta and Klose 2019; Syed et al. 2009; Zhu et al. 2013), b Four levels (classes I, II, III, and IV) of polar flagellar hierarchy in V. parahaemolyticus (Kim and McCarter 2000; McCarter 2001), and c Three levels (classes I, II, and III) of lateral flagellar hierarchy in V. parahaemolyticus (Stewart and McCarter 2003)

the lifestyle of Vibrio species, such as during colonization, biofilm formation, and production of virulence factors (Echazarreta and Klose 2019). Several mutational analyses of the flagellar genes have shown that pathogenic vibrios produce a host response upon adhesion and invasion of their respective host cells. Furthermore, based on whole-genome transcriptome analysis, it has been found that motility and production of virulence factors are inversely regulated in Vibrio cholerae (Syed et al. 2009). Similar to flagella-mediated motility, chemotaxis (movement in response to chemicals) is well studied and has been found to be correlated with motility behavior (Boin et al. 2004). Reports have shown that, in different Vibrio species, motility is regulated by several cellular processes, such as transcription regulation, the type III secretion system (T3SS), signaling molecules, and the QS signaling system (Gu et al. 2019). Flagellar motility in bacterial species is one of the main regulatory cellular processes and is well studied (Laganenka et al. 2020). However, in most Vibrio species, detailed signaling mechanisms have not been well studied, except in V. cholerae and V. parahaemolyticus. The present review paper describes a comparative signaling mechanism involved in the flagellar motility of V. cholerae and V. parahaemolyticus.

### Regulatory cascades and proteins involved in the motility of V. cholerae

The lifestyle of V. cholerae, both pathogenic to non-pathogenic forms, is tightly regulated by the presence of free-floating motile and non-motile (i.e., aggregates/biofilm) cell states (Silva and Benitez 2016; Tamayo et al. 2010). These two states govern the pathogenesis of this species and its survival amid adverse environmental conditions. A previous study showed that free-floating planktonic cells exhibit virulence, whereas biofilm acts as a survival state, protecting cells from the environmental conditions (Yildiz and Schoolnik 1999). V. cholerae cells possess a single polar flagellum that facilitates its motility through the involvement of a Na⁺-motive force (Fig. 2) (Kojima et al. 1999). The genome of V. cholerae has several polar flagellar genes and transcription of these genes in V. cholerae is highly complex and is regulated by several transcription factors. Previous studies have

Fig. 2 Signaling mechanisms involved in the regulation of motility properties in V. cholerae

categorized the flagellar genes of V. cholerae into a four-level (classes I, II, III, and IV) transcriptional hierarchy based on their level of expression and their transcription regulators (Klose and Mekalanos 1998; Prouty et al. 2001; Syed et al. 2009) (Fig. 1a). The class I gene, which is present at the top of the flagellar hierarchy, synthesizes FlrA and activates the class II gene in the presence of σ⁵⁴ (RpoN). The class II gene synthesizes the MS-ring switch, FlrBC (two-component system), and export apparatus. The active FlrC (phosphorylated form) activates class III genes, which synthesize the basal body-hook, motor component (MotY), and flagellin (FlaA) (Correa et al. 2000; Syed et al. 2009). The transcription of class III genes also occurs in the presence of σ⁵⁴. Class IV genes transcribe alternative flagellin, antisigma factor (i.e., FlgM), and some of the motor components in the presence of σ²⁸, which is encoded by the *fliA* gene in the genome of V. cholerae (Correa et al. 2004; Syed et al. 2009).

Cyclic di-GMP (c-di-GMP) is a second messenger that exhibits diverse cellular roles in different bacterial species (Commichau et al. 2015; Romling et al. 2013). Tasked with motility regulation, c-di-GMP plays a crucial role in the transition of free-floating planktonic cells into the biofilm state (Beyhan et al. 2007; Ryjenkov et al. 2006). Production and degradation of c-di-GMP in Vibrio species are regulated by diguanylate cyclase (DGC) and phosphodiesterase (PDE), as identified during mutational studies (Conner et al. 2017; Lim et al. 2006). The V. cholerae genome encodes a total of 53 proteins with DGC and PDE activity (Galperin et al. 2001; Lim et al. 2006). However, mutational studies have shown that among these proteins, four diguanylate cyclases, CdgD, CdgH, CdgI, and CdgK, can synthesize and enhance the concentration of c-di-GMP (Lim et al. 2006). Similarly, V. cholerae also has two phosphodiesterases, CdgJ and RocS, which are involved in the degradation of c-di-GMP. These PDEs are involved in decreasing the intracellular pool of c-di-GMP (Lim et al. 2006; Liu et al. 2010). A high concentration of c-di-GMP in V. cholerae enhances biofilm formation, which suppresses flagellar motility, and a low concentration of c-di-GMP does the opposite (Srivastava et al. 2013). Research has shown that c-di-GMP inhibits motility in different bacterial species in multiple ways, such as (1) inhibition of flagellar gene expression, (2) enhanced production of extracellular polysaccharides, which impedes flagellar rotation, and (3) initiation of a backstop brake mechanism (Ferreira et al. 2008; Paul et al. 2010; Srivastava et al. 2013; Zorraquino et al. 2013). According to Srivastava et al. (2013), c-di-GMP inhibits the flagellar motility of V. cholerae in two different ways: firstly, by binding at the allosteric site to the master regulator FlrA and making it not available to bind the promoter of the flagellar gene operon *flrBC*, thus inhibiting motility (Fig. 2). An alternative motility-inhibiting mechanism of c-di-GMP is characterized by enhanced production of extracellular Vibrio Polysaccharides (VPS); hence, the accumulated extracellular VPS impedes flagellar rotation, thereby inhibiting motility. The production of the biofilm matrix and motility in V. cholerae are also known to be regulated by the transcriptional regulator VpsT (Krasteva et al. 2010). VpsT is a two-component response regulator that is required for the expression of the *vps* gene after binding to c-di-GMP (Fig. 2) (Casper-Lindley and Yildiz 2004). For the functional activity of VpsT, the oligomerization of chains A and B is important; this process occurs when VpsT binds to c-di-GMP (Krasteva et al. 2010). Furthermore, VpsT also suppresses the expression of flagellar genes, suggesting that it can inversely regulate the motility and formation of biofilms in the presence of c-di-GMP (Krasteva et al. 2010). Similarly, another two-component response regulator, VpsR, activates VPS biosynthesis by binding to the promoter of the *vps* gene (Yildiz et al. 2001). VpsR is active when it is phosphorylated on the aspartate residue, although the kinase responsible for this phosphorylation is not known (Fig. 2) (Lauriano et al. 2004). Notably, VpsR also activates the expression of *vpsT* by binding to its promoter (Beyhan et al. 2007; Zamorano-Sánchez et al. 2015).

In V. cholerae, the histone-like nucleoid structuring protein (H-NS) acts as a transcriptional factor and plays an important

role in regulating biofilm formation and motility (Ghosh et al. 2006; Wang et al. 2012a; Wang et al. 2012b). In *V. cholerae*, H-NS inhibits the formation of biofilms by binding to the promoter of the *vps* gene responsible for the synthesis of the exopolysaccharide matrix (Wang et al. 2012b; Zamorano-Sánchez et al. 2015). Apart from silencing biofilm-forming gene expression in *V. cholerae*, H-NS is also known to attenuate flagellar motility by inhibiting the expression of RNA polymerase subunit σ<sup>54</sup> (*rpoN*) and *flrA* (Wang et al. 2012a). VpsT acts as an anti-repressor of H-NS. When the concentration of c-di-GMP increases, it leads to the dislodging of H-NS from the promoter of *vps*, thereby activating the expression of the latter (Ayala et al. 2015). Similarly, when the concentration of c-di-GMP is low, H-NS displaces VpsT from the promoter of the *vps* gene, thereby suppressing its expression (Ayala et al. 2015). A VpsR-binding motif has been identified downstream of the *vpsT* promoter, close to the binding site of H-NS (Zamorano-Sánchez et al. 2015). Thus, binding of VpsR to the *vpsT* promoter is also known to hinder the binding of H-NS to the *vpsT* promoter.

The activity of master regulator FlrA is also dependent on the concentration of c-di-GMP. When the c-di-GMP concentration increases, the expression of all flagellar genes is suppressed as a result of the binding of c-di-GMP to FlrA (Srivastava et al. 2013). The concentration of c-di-GMP at high cell density (HCD) and the stationary phase is diminished by the expression of the QS regulator HapR (a LuxR homolog) (Fig. 2) and alternative sigma factor (RpoS) (Wang et al. 2011; Waters et al. 2008). Hence, when the concentration of c-di-GMP decreases, its inhibitory effect on FlrA activity is reduced, thereby enhancing motility. Reports have shown that RpoS initiates the transcription of *rpoN* and *flrA*, which is enhanced during the stationary phase (Wang et al. 2012a). Thus, RpoS also plays an important role in regulating flagellar motility in *V. cholerae*. Wang et al. (2012a) proposed two possible mechanisms for this. In the first mechanism, RpoS and RNA polymerase initiate the transcription of *rpoN* and *flrA* after binding to their promoters. The binding of RpoS to the promoters of *rpoN* and *flrA* also neutralizes the inhibitory effect of repressor H-NS, since these promoters become resistant to the binding of H-NS. In the second mechanism, RpoS also neutralizes the inhibitory effect of the H-NS repressor indirectly, by inducing the expression of the integration host factor (IHF). In the stationary phase, IHF is expressed and displaces the binding of H-NS to the promoters of *rpoN* and *flrA*.

VpsT also regulates RpoS activity in response to the concentration of c-di-GMP (Wang et al. 2014). At low concentrations of c-di-GMP, RpoS initiates the transcription of *rpoN* and *flrA*, whereas when the concentration of c-di-GMP increases, the transcription of *rpoS* is repressed by the inhibitory effect of VpsT. Conversely, at low cell density (LCD), HapR is repressed, and thereby, the c-di-GMP pool is increased, resulting in the suppression of *rpoS* by active VpsT (Wang et al. 2014). In HCD, HapR activates the expression of RpoS without the involvement of c-di-GMP (Wang et al. 2011). Thus, active RpoS also lowers the c-di-GMP pool in a manner similar to HapR.

Similar to HapR, AphA is a master regulator of the QS signaling system that regulates motility and biofilm formation in *V. cholerae* in the LCD state (Lu et al. 2018; Rutherford et al. 2011; Yang et al. 2010). The regulatory protein AphA was also found to regulate the motility of other *Vibrio* species, such as *V. parahaemolyticus*, *V. harveyi*, and *V. alginolyticus* (Rutherford et al. 2011; Wang et al. 2013b). Thus, in conclusion, it is clear that in *V. cholerae*, c-di-GMP plays a significant role in regulating the flagellar motility and formation of biofilm via regulating VpsT, VpsR, RpoS, and H-NS regulatory proteins.

Furthermore, it can also be inferred that the enhanced formation of biofilm at LCD and enhanced motility at HCD in *V. cholerae* represent their phenotypic adaptations to different environmental conditions. Indeed, at LCD, more biofilm is formed, which helps these bacteria shield against the impact of environmental stress (Reidl and Klose 2002; Zhu and Mekalanos 2003); whereas at HCD, enhanced flagellar activity helps them detach from the host cells (i.e., dispersion of cells from mature biofilm) and enter the wider environment (Reidl and Klose 2002; Schild et al. 2007).

Cyclic di-GMP also plays a role in activating the ATPase activity of MshE, which is responsible for the secretion of mannose-sensitive hemagglutinin (MSHA) type IV pili on the cell surface (Roelofs et al. 2015). MshE acts as a receptor for binding with c-di-GMP, and at a high concentration of c-di-GMP, the production of MshA pilus is increased. Hence, MshE plays an important role in *V. cholerae* transition from a motile to sessile lifestyle by modulating near-surface motility and initial surface attachment (Floyd et al. 2020; Jones et al. 2015).

Similar examples of regulatory proteins that regulate motility in *V. cholerae* with the involvement of c-di-GMP, such as TfoY, PhoB, and LonA, are described in Table 1. The genome of *V. cholerae* also contains five PilZ domain-containing proteins that serve as effector proteins for the binding of c-di-GMP (Pratt et al. 2007). These PilZ domain-containing proteins regulate some of the important processes in the lifestyle of *V. cholerae*, such as motility, biofilm formation, and virulence. Mutational studies have shown that PilZ domain-containing proteins such as PlzB and PlzC act as positive regulators of both motility and biofilm formation (Pratt et al. 2007). Whereas there was no effect on the motility as a result of *plzD* mutation, the overexpression of PlzD reduced motility and was independent of binding with c-di-GMP (Pratt et al. 2007). Thus, in conclusion, there is also evidence that c-di-GMP plays an important role in the lifestyle of *V. cholerae* in regulating the surface attachment and biofilm formation by

controlling the production of flagella, MSHA pili, and biofilm matrix. Apart from this, some additional transcriptional regulators that govern motility in *V. cholerae*, either directly or indirectly, are summarized in Table 1.

# Regulatory cascades and proteins involved in the motility of *V. parahaemolyticus*

In *V. parahaemolyticus*, some signaling cascades and regulatory proteins control the transition of cells from free-floating plankton to a biofilm state, and vice-versa (Ferreira et al. 2008). This bacterium has two different types of flagella, with one being polar, derived from the Na<sup>+</sup>-motive force, and the other lateral, derived from the H<sup>+</sup>-motive force (Fig. 3). Depending upon the environment, two types of motility occur, i.e., swimming motility in liquid media using the polar flagellum, and swarming motility on the surface of solid media using the lateral flagella (McCarter 2004). A dual flagellar system has also been reported in *V. alginolyticus*, *V. harveyi*, *V. campbellii*, *V. fluvialis*, and *V. proteus* (Shinoda et al. 1985). In *V. parahaemolyticus*, approximately 60 genes that encode the polar flagellar system are present on chromosome I (Kim and McCarter 2000), whereas 38 genes that are responsible for the synthesis of the lateral flagellar system are present on chromosome II (Stewart and McCarter 2003). The hierarchy of polar flagellar gene expression in *V. parahaemolyticus* has also been categorized into four-tiers (classes I, II, III, and IV) in similar ways as reported for *V. cholerae* (Fig. 1b). Whereas the hierarchy for the lateral flagellar gene expression in *V. parahaemolyticus* has three tiers (classes I, II, and III) (Fig. 1c). The polar and lateral flagellar genes in *V. parahaemolyticus* are also transcribed in the presence of both σ<sup>54</sup> and σ<sup>28</sup>. The transcription of class II genes of the polar and lateral flagellar systems is driven by class I gene products such as FlaK (for the polar flagellar system) (McCarter 2001) and LafK (for the lateral flagellar system) (Gode-Potratz et al. 2011) in the presence of σ<sup>54</sup>. The class III genes of the polar flagellar system are transcribed by FlaM and σ<sup>54</sup>, whereas the class III genes of the lateral flagellar system are activated by σ<sup>28</sup>, which is encoded by *fliA* (Stewart and McCarter 2003). The class IV gene of the polar flagellar system is also activated by the σ<sup>28</sup>-dependent regulator (Stewart and McCarter 2003). The lateral flagella play an important role in *V. parahaemolyticus*, enabling it to move on solid surfaces or whenever it encounters a viscous environment. Apart from the regulation of the lateral flagella by LafK, some other regulators, which are independent of σ<sup>54</sup> and σ<sup>28</sup>, govern swarming motility behavior in *V. parahaemolyticus*, and these are also summarized in Table 1.

The *scrABC* operon in *V. parahaemolyticus* encodes for three regulators, namely, ScrA (aminotransferase homolog), ScrB (solute binding protein), and ScrC (has a dual role involving diguanylate cyclase and phosphodiesterase) (Fig. 3)

(Boles and McCarter 2002; Ferreira et al. 2008; Trimble and McCarter 2011). ScrA is identified as a pyridoxal phosphate-dependent aminotransferase and is known to be present in the cytoplasm (Boles and McCarter 2002), although it is also hypothesized to be localized in the periplasm (Trimble and McCarter 2011). ScrB is present in the periplasm and functionally characterized to modulate the activity of ScrC in the presence or absence of ScrA activity. ScrC is a cytoplasmic membrane-spanning protein containing both GGDEF (with diguanylate cyclase activity) and EAL (with phosphodiesterase activity) conserved protein domains (Fig. 3) (Ferreira et al. 2008). The concentration of c-di-GMP in the cells is controlled by both the diguanylate cyclase and phosphodiesterase activity of ScrC, however, these activities are modulated by ScrA and ScrB. In the presence of ScrB, the diguanylate cyclase of ScrC is inhibited, whereas to inhibit its phosphodiesterase activity, both ScrB and ScrA are required (Fig. 3). Although the substrate for the aminotransferase activity of ScrA is not known, it synthesizes QS-like signaling molecules (S-signal) via its synthase activity. The synthesized S-signal is transported outside the cells and is known to communicate with the surrounding cells by binding with the ScrB domain. The bound S-signal and ScrB interact with the periplasmic domain of ScrC, thereby activating the phosphodiesterase activity of ScrC, which results in the degradation of c-di-GMP (Trimble and McCarter 2011). Thus, it is concluded that at HCD (in the presence of the S-signal), the phosphodiesterase of ScrC becomes active, resulting in a decrease in the concentration of c-di-GMP and, consequently, enhanced swarming motility and inhibition of biofilm formation. At LCD (in the absence of the S-signal), the cyclase domain of ScrC becomes active, which enhances the level of c-di-GMP, thereby inducing biofilm formation and suppressing swarming motility. Reports have shown that the expression of swarming motility genes (lateral flagellar genes) and capsule synthesis (*cps*) genes are inversely regulated by the level of c-di-GMP (Fig. 3) (Ferreira et al. 2012; Trimble and McCarter 2011). Recently, Kimbrough et al. (2020) reported that *V. parahaemolyticus* possesses four VpsT-type c-di-GMP-binding Scr transcription factors, namely CpsQ, CpsS, ScrO, and ScrP. These regulators play an important role in controlling biofilm formation in *V. parahaemolyticus*. Among these, CpsQ, CpsS, and ScrO are active only after binding with c-di-GMP, whereas the activity of ScrP is independent of c-di-GMP. CpsQ and ScrO are the most important transcription factors and act as primary regulators of c-di-GMP-dependent expression of genes involved in biofilm formation. CpsS, whose activity is dependent on c-di-GMP, acts as a negative regulator, as confirmed by mutational analysis. The *cpsS* mutant expresses high levels of capsule genes and its colonies become crinkly (Güvener and McCarter 2003). In *V. parahaemolyticus*, CpsS also acts as a negative regulator of *cpsR*, which encodes CpsR an orthologue of VpsR from

| Vibrio species | Proteins | Description | Function | References |
| --- | --- | --- | --- | --- |
| V. cholerae | FlrA | Transcriptional regulator | It acts as a flagellar gene activator in a σ⁵⁴ dependent manner | (Prouty et al. 2001) |
|  | FlrC | Transcriptional regulator | It works as σ⁵⁴ dependent transcriptional activator for the synthesis of the flagellar genes | (Correa and Klose 2005) |
|  | HapR | Transcriptional regulator | It is a LuxR homolog and controls the flux of c-di-GMP. In high cell density, HapR inhibits the production of c-di-GMP in a QS dependent manner | (Waters et al. 2008) |
|  | PhoB | Transcriptional regulator | It regulates the motility property as a result of activation of acgAB operon which synthesizes c-di-GMP metabolic enzymes | (Pratt et al. 2009) |
|  | VxrAB | Two-component system | It is involved in the signaling responsible for the formation of biofilm. The overexpression of VxrAB in vector leads to a decrease in motility | (Techler et al. 2017) |
|  | TfoY | Transcription factor | It is responsible for the controlling motility property from the dense to a dispersive state. The motility regulation by TfoY occurs by the involvement of c-di-GMP at both transcriptional and post-transcriptional level | (Pursley et al. 2018) |
|  | Alpha P | Transcriptional regulator | Known as a master QS regulator. Although the mechanism not clear, it acts as an activator for the flagellar motility in different Vibrio species | (Lu et al. 2018) |
|  | CosR | Transcriptional regulator | In the presence of compatible solute, CosR activate the formation of biofilm property whereas it reduces the motility property | (Shikuma et al. 2013) |
|  | H-NS | Histone-like nucleoid structuring protein acting as a transcriptional regulator | It binds to the FlrA promoter, thereby motility inhibition takes place | (Ayala et al. 2017) |
| V. parahaemolyticus | Chis | Sensor Histidine Kinase | It plays an important role in the pathogenesis by regulating the motility property as well as the production of toxins | (Chourashi et al. 2016) |
|  | LonA | Protein degrading protease | It influences the swimming motility, formation of biofilm and level of c-di-GMP and type VI secretion system | (Rogers et al. 2016) |
|  | FlgO and FlgP | Membrane proteins | Required for the stability of the flagella thereby these protein helps in the attachment and colonization | (Martinez et al. 2009) |
|  | Flak | Transcription factor | It is a σ⁵⁴-dependent transcriptional activator, involved in the expression of class II gene of polar flagellum | (McCarter 2001) |
|  | Lafx | Transcription factor | It is a σ⁵⁴-dependent master regulator, which activates the class II gene of lateral flagellum | (McCarter and Wright 1993) |
|  | VPA1701 | Gnt family transcription factor | It regulates lateral flagellar gene expression by binding to the promoter region of the flgBCDEFGHIJKL cluster | (Gu et al. 2019) |
|  | SwrT and SwrZ | Transcriptional regulator | SwrT is a TetR-like transcription regulator, Whereas, GntR-like transcriptional regulator, which inhibits the swarming motility by repressing the expression of laf gene expression | (Jaques and McCarter 2006) |
|  | Opar | Transcriptional factor | It is LuxR homolog and regulates the switching of free-floating planktonic cells into the non-motile sessile cells in a QS-mediated manner | (Kim and McCarter 2007) |
|  | ScrG | Protein containing GGDEF and EAL domains acting as a transcriptional regulator | It acts as phosphodiesterase, thereby low level of c-di-GMP availability which results in the promotion of swarming motility | (Ferreira et al. 2008) |
|  | ScrC | Membrane-bound bifunctional protein | It has diguanylate cyclase and phosphodiesterase domain. When cell density is high, then its phosphodiesterase is active, which |  |

| Vibrio species | Proteins | Description | Function | References |
| --- | --- | --- | --- | --- |
|  | ToxR | Transcriptional regulator | It has multiple roles such as regulation of biofilm formation, production of toxins in different species of *Vibrio*, however, recent it is found as an activator of swarming and swimming motility | (Chen et al. 2018) |
|  | ZomB | Transmembrane protein | degrade c-di-GMP, thereby enhance motility and inhibit biofilm formation. In reverse when cell density low, then its cyclase domain become active and increase the amount of c-di-GMP, thereby inhibit the motility and promotes motility |  |
|  | OxyR | Transcriptional regulator | It acts as an activator for the motility property | (Chung et al. 2016) |
|  | CalR | LysR-type transcriptional regulator | Mutational studies confirmed that it is required for the reversal of flagellar motor | (Brenzinger et al. 2018) |
|  | HU | DNA-binding protein | Act as a repressor for the swarming motility as well as type III secretion system I Along with the motility regulating property it regulates several important processes such as metabolism, replication, structuring nucleoid, growth virulence, etc. | (Zhang et al. 2017) |
|  | VcrD1 | Component of type III secretion system | Plays a significant role in the flagellar morphogenesis for the expression of some flagellar genes and assist in the secretion of flagellar component | (Phan et al. 2015) |
| *V. algivolyticus* | SflA | Membrane found and cytoplasmic spanning regulatory protein | It controls the spatial arrangement of the flagellum. It inhibits the formation of the peritrichous sheathed flagellum | (Sakuma et al. 2019) |
|  | RstB-RstA System | Two-component system | Apart from the regulating motility, this system plays an important role in other properties such as adhesion, biofilm formation, production of virulence factors and hemolysis | (Huang et al. 2018) |
|  | AcfA | Accessory colonization factors | Play an important role in the pathogenesis of *V. alginolyticus* | (Cai et al. 2018) |
|  | HubP | Polar landmark protein | Mutational analysis confirms that it acts as an activator for the flagellar motility and inhibitor for the biofilm formation | (Takekawa et al. 2016) |
| *V. vulnificus* | IscR | Transcription factors | It regulates the flagellar number. The mutation study showed that deletion of *hubP* gene increases the number of flagella Apart from regulating Fe-S cluster genes, it plays an important role in for the pathogenesis as a result of regulating the motility, adhesion, resistance against the oxidative stress and hemolytic activity | (Lim and Choi 2014) |
|  | AphB | Transcriptional regulator | It plays a significant role in the adhesion, motility, and virulence properties | (Jeong and Choi 2008) |
| *V. fischeri* | SyPK | Oligosaccharide translocase | Usually, it modulates the QS system especially the kinase activity of LuxQ. It increased the expression of *qrrI* thereby it enhanced the motility property | (Miyashiro et al. 2014) |

Fig. 3 Signaling mechanisms involved in the regulation of motility properties in *V. parahaemolyticus* (information obtained from (Boles and McCarter 2002; Ferreira et al. 2008; Kim and McCarter 2007))

![Diagram](attachment:diagram.png)

*V. cholerae* (Yildiz and Visick 2009). CpsR was found to positively regulate the *cpsQ* gene in a c-di-GMP-dependent manner; therefore, CpsQ can directly activate the expression of the *cpsA* biosynthetic operon (Ferreira et al. 2012). ScrP represses *wzyO*, a biofilm gene encoding O-antigen ligase, and was found to be independent of c-di-GMP. Furthermore, in *V. parahaemolyticus*, H-NS acts as a transcriptional activator of the *cpsA* biosynthetic operon, thereby increasing the production of EPS (Zhang et al. 2018). Moreover, H-NS also activates the *cpsQ* gene; however, it has a negative regulatory effect on *scrABC*, which results in the generation of high concentrations of c-di-GMP. To summarize, the expression of the *cpsA* gene in *V. parahaemolyticus* is regulated by a CpsS-CpsR-CpsQ regulatory cascade along with the *scrABC* signaling pathway (Zhang et al. 2018).

Jaques and McCarter (2006) reported three transcriptional regulatory proteins that are involved in controlling swarming motility in *V. parahaemolyticus*, namely, SwrT, SwrZ, and OpaR. OpaR is a transcriptional regulator and homolog of the LuxR regulator of *V. harveyi* (McCarter 1998) and has two ways of repressing swarming motility. SwrT is a TetR-like transcriptional regulator that has been identified as a swarming regulator, as the mutant *swrT* is defective in swarming motility and *laf* gene expression. Furthermore, it has also been found that SwrT represses *swrZ*, which encodes the GntR-like transcriptional regulator SwrZ, which itself inhibits swarming motility by repressing *laf* gene expression.

The QS-mediated regulation of motility in *V. parahaemolyticus* is also known to occur with the LuxR homolog, OpaR (Jaques and McCarter 2006; McCarter 1998). OpaR in *V. parahaemolyticus* inhibits *laf* gene expression in

the lateral flagella (Jaques and McCarter 2006), thereby impeding swarming in a LCD state (Fig. 3). This suggests that *V. parahaemolyticus* exhibits a maximum level of motility in a LCD state, whereas, at HCD, biofilm genes such as *cps* (capsular polysaccharide) are activated (Enos-Berlage and McCarter 2000; McCarter 1998). By activating the *cpsA* gene, cells start producing extracellular polysaccharides, enabling them to adhere to each other as well as surfaces. This suggests that the QS signaling system in *V. parahaemolyticus* works in an opposing manner as compared with *V. cholerae*.

The T3SS transcriptional regulator also plays an important role in the regulation of flagellar motility in *V. parahaemolyticus*. ExsA, a positive transcriptional regulator of the T3SS, was found to inhibit the expression of the *lafk* gene, thereby reducing the swarming motility of *V. parahaemolyticus* (Gode-Potratz et al. 2010). Similarly, VcrD1 is a component of the T3SS and plays a role in both the expression of flagellar genes and secretion of the flagellar component in *V. parahaemolyticus* (Noh et al. 2015). Gode-Potratz et al. (2010) reported a T3SS negative regulator CalR, which is a LysR-type transcription factor that also inhibits swarming motility by repressing expression of the *lafk* gene. A GntR family transcription factor, VPA1701, was found to increase swarming motility in two ways: (1) by enhancing the expression of lateral flagellar genes and (2) by inhibiting the formation of CalR (Gu et al. 2019). Some other examples of regulatory proteins involved either directly or indirectly in regulating swimming or swarming motility in *V. parahaemolyticus* are presented in Table 1.

Motility in other species of *Vibrio* is also known to be regulated by some of the aforementioned regulatory proteins,
although no detailed regulatory cascade has been identified. Some of the transcriptional regulatory and membrane-bound proteins are known to regulate motility in *V. alginolyticus*, *V. vulnificus*, and *V. fischeri*. The motility regulatory function of these proteins in the aforementioned *Vibrio* species is explained in Table 1.

# Evolutionary relatedness of σ<sup>54</sup>-dependent motility transcription regulators in Vibrio spp.

It has been found that the expression of all flagellar genes, from early to middle to late transcription, is regulated by specific transcriptional factors, and that the initiation of early and middle gene transcription is mediated by σ<sup>54</sup>-dependent regulators such as FlrA, FlrC, FlaK, and LafK (Correa et al. 2000; McCarter 2001; Millikan and Ruby 2003; Stewart and McCarter 2003). RpoN is a sigma-54 family sigma factor that has been identified in several *Vibrio* species, such as *V. cholerae*, *V. parahaemolyticus*, *V. alginolyticus*, *V. harveyi*, *V. anguillarum*, and *V. fischeri*, as summarized by Boyd et al. (2015). The functionality of RpoN can vary among *Vibrio* species, however, some of the common properties in these bacteria, such as motility, biofilm formation, QS, the type III secretion system (T3SS), and the type VI secretion system (T6SS), are regulated by RpoN (Boyd et al. 2015). σ<sup>54</sup> is required for motility in most *Vibrio* species and so, to elucidate the evolutionary relatedness of the σ<sup>54</sup>-dependent master regulator of the motility gene in different *Vibrio* species, we constructed a phylogenetic tree based on the protein sequences of FlrA, FlrC, FlaK, and LafK, using MEGA5. The multiple sequence alignment of the protein sequences of all transcription factors was carried out using CLUSTALW. Phylogenetic analysis of the σ<sup>54</sup>-dependent master regulator can help us understand whether all or a few *Vibrio* species share a common or different regulatory scheme. In the phylogenetic tree, FlrA and Flak are mixed and clustered together on a single branch, whereas FlrC and LafK are separated, each on their own single branch (Fig. 4). The presence of FlrA and FlaK from different species of *Vibrio* in a single cluster indicates that the transcription factors involved in activating the class II gene are conserved and belong to a common ancestor. The transcription factor FlrC, which activates the class III gene in multiple *Vibrio* species, is clustered together on a single branch, divergent from the branch containing FlrA and FlaK. The lateral flagella regulator LafK is present in four different species, *V. parahaemolyticus*, *V. alginolyticus*, *V. harveyi*, and *V. campbellii*, with all being clustered together on a single branch (Fig. 4). Although lateral flagella have also been reported in *V. fluvialis* and *V. proteus* (Shinoda et al. 1985), no LafK has been identified in these species. LafK also activates the class II genes of the lateral

flagella like FlrA and FlaK in similar ways to the activation of the class II genes of the polar flagella, yet LafK is present on a separate divergent branch. It is therefore evident that the polar and lateral flagellar systems are structurally different and do not share σ<sup>54</sup>-dependent transcriptional regulators. FlrA, FlrC, and FlaK have been identified in more than 12 different species of *Vibrio* (Fig. 4). Hence, the grouping of identical σ<sup>54</sup>-dependent master regulators on the phylogenetic tree indicates that all of these species share a common evolutionary origin in regard to their motility properties. The involvement of RpoN in the transcription of motility genes in most *Vibrio* species serves as evidence that the master regulator involved in regulating both the polar and lateral flagellar systems shares a common origin, structure, and functionality among vibrios. The presence of FlrA, FlaK, and FlrC in numerous *Vibrio* species suggests that there is a high chance of a common polar flagellar regulatory system in these species. However, the occurrence of LafK in very few *Vibrio* species suggests that these bacteria might have evolved an additional flagellar system (i.e., the lateral flagellar system) for interacting with a viscous environment or solid surface, where swarming motility can be favored.

## Relatedness of the motility regulators in V. cholerae and V. parahaemolyticus

To further check the uniqueness of some of the regulatory proteins involved in regulating the flagellar motility in both *V. cholerae* and *V. parahaemolyticus* a bidirectional best hit was carried out using Geneious® 11.0.2 software (Kearse et al. 2012). The percentage identity obtained between some of the regulatory proteins of both *V. cholerae* and *V. parahaemolyticus* is represented in Fig. 5. The results showed that the master regulator FlrA (AAC46365) of the polar flagellar system of *V. cholerae* exhibits 77.46% identity with FlrA (AWG83153) and FlaK (BAC60516) of *V. parahaemolyticus*. Similarly, the transcription factor FlrC (KNH55311) of *V. cholerae* showed 84.28% identity with FlrC (AYF20536) of *V. parahaemolyticus*. Interestingly, FlrC (KNH55311) which is a transcriptional activator for class III gene of *V. cholerae* also showed 77.46% identity with lateral flagellar transcription regulator LafK (AGB12779) which is an activator of class II genes of *V. parahaemolyticus*. The CpsR of *V. parahaemolyticus* is the orthologue of VpsR of *V. cholerae* and it showed 78.15% identity with each other, whereas the VpsR also showed 50% identity with the two-component response regulator VpsT which is involved in regulating the biofilm formation and flagellar motility in *V. cholerae*. It is known that the QS regulator HapR (AVK79074) in *V. cholerae* indirectly activates the flagellar motility by lowering the c-di-GMP pool,
whereas QS regulator OpaR (AAC46404) in V. parahaemolyticus inhibits the lateral flagellar motility. The percentage identity among HapR and OpaR proteins was found to 71.29%, suggesting these are orthologue. The motility attenuating as well as biofilm inhibitor H-NS (AKB05620) in V. cholerae, showed 66.67% identity with the SwrT (AGB11763) which is known to positively activate the lateral flagellar gene expression in V. parahaemolyticus. Furthermore, the percentage identity obtained among some of the additional motility regulator in V. cholerae as well as in V. parahaemolyticus are also shown in Fig. 5.

Fig. 4 A phylogenetic tree of σ⁵⁴ (RpoN)-dependent transcription factors involved in the regulation of flagellar motility in different species of Vibrio. With the help of MEGA 5, the phylogenetic tree of σ⁵⁴-dependent transcription factors such as FlrA, FlrC, FlaK, and LafK present in different Vibrio species was constructed using the Neighbor-Joining method and a bootstrap value of 1000 (Tamura et al. 2011)

Attenuation of motility by naturally derived or chemically synthesized compounds

Consumption of pathogenic Vibrio species-contaminated water and different types of raw or processed seafood can result in foodborne diseases, although not all species are pathogenic (Yildiz and Visick 2009). Post-invasion bacteria start to adhere to and form a biofilm in the host cell (Tamayo et al. 2010; Teschler et al. 2015). Once biofilms are established, they start secreting numerous toxins that adversely affect the cells, further exacerbating symptoms (Silva and Benitez 2016).

Vibrio alginolyticus 40B (EEZ84013)/FlrA  
[Vibrio alginolyticus 12G01 (EAS77890)/FlaK  
[Vibrio alginolyticus ATCC 17749 (AGV17193)/FlaK  
Vibrio diabolicus (CDT92339)/FlrA  
Vibrio parahaemolyticus (AWG83153)/FlrA  
Vibrio parahaemolyticus BB220P (AGB10537)/FlaK  
Vibrio parahaemolyticus RIMD 2210633 (BAC60516)/FlaK  
Vibrio sp. AND4 (EDP59153)/FlaK  
Vibrio campbellii (ARR05611)/FlrA  
Vibrio harveyi (SQA34296)/FlaK  
Vibrio harveyi 1DA3 (EEZ87752)/FlrA  
Vibrio campbellii ATCC BAA-1116 (AGU95603)/FlaK  
Vibrio rotiferianus LMG 21460 (PIB16242)/FlaK  
Vibrio sp. EJY3 (AEX22798)/FlaK  
Vibrio vulnificus YJ016 (BAC95249)/FlaK  
Vibrio tasmaniensis LGP32 (CAV17811)/FlaK  
Vibrio splendidus ATCC 33789 (EGU42504)/FlaK  
Vibrio crassostreae (CDS95995)/FlrA  
Vibrio sp. J2-31 (CDT96578)/FlrA  
Vibrio sinaloensis DSM 21326 (EGA71360)/FlaK  
Vibrio anguillarum (CDQ50855)/FlrA  
Vibrio mimicus VM573 (EEW11203)/FlrA  
Vibrio cholerae (AAC46365)/FlrA  
Vibrio cholerae B33 (EAZ77079)/FlrA  
Vibrio cholerae O1str.95412 (EMP97098)/FlrA  
Vibrio cholerae O1str.2010EL-1786 (AET27226)/FlrA  
Vibrio cholerae NCTC 8457 (EAZ73082)/FlrA  
Vibrio shilonii AK1 (EDL55011)/FlaK  
Vibrio tapetis subsp. tapetis (SON50204)/FlrA  
Vibrio nigripulchritudo MADA 3020 (CCN48763)/FlrA  
Vibrio fischeri SR5 (EHN70300)/FlrA  
Vibrio fischeri ES114 (AAW86351)/FlrA  
97 Vibrio campbellii ATCC BAA-1116 (AGU97971)  
100 Vibrio harveyi (SQA35102)  
63 Vibrio alginolyticus (SQA41749)  
Vibrio fischeri ES114 (AAW86349)  
Vibrio anguillarum (CDQ50853)  
100 92 Vibrio parahaemolyticus (AYF20536)  
99 Vibrio sp. ES.045 (TVZ29181)  
69 Vibrio campbellii (ARR05613)  
31 Vibrio xiamenensis (SDH52366)  
25 Vibrio crassostreae (TWD72828)  
21 Vibrio gazogenes DSM 21264 (SHE94242)  
10 Vibrio hangzhouensis (SEG34955)  
9 Vibrio cincinnatiensis DSM 19608 (SJZ65768)  
33 100 Vibrio mimicus (SUP12052)  
Vibrio cholerae 623-39 (KNH55311)  
Vibrio cholerae M66-2 (ACP06360)  
98 Vibrio cholerae NCTC 8457 (EAZ73075)  
65 Vibrio cholerae O1str.95412 (EMP97096)  
Vibrio cholerae O395 (ABQ19966)

Recent and past studies have shown that several naturally derived products in their pure form exhibit anti-motility activity towards different *Vibrio* species (Table 2). Although, in some cases, the pure active molecules could not be identified, the extracts obtained from different biological sources have proven anti-motility properties (Srinivasan et al. 2017). In addition, these molecules are effective at biofilm inhibition and attenuation of virulence properties of *Vibrio* species (Table 2). The sources of these molecules include plants, marine organisms, and bacteria. Examples of the anti-motility compounds derived from the plants include citral extracted from lemongrass oil, which at an active concentration of 12.5 μg/ml inhibits swimming (47%) and swarming (50%) of *V. parahaemolyticus* (Sun et al. 2019). Citral was also reported to inhibit biofilm formation and production of autoinducer-2, a gene related to biofilm formation, flagella synthesis, type III secretion, and antibiotic resistance (Sun et al. 2019). Thymoquinone extracted from *Nigella sativa* inhibits both swimming and swarming, along with biofilm formation and attachment to host cells (Guo et al. 2019). Molecular studies have shown that it also suppresses the expression of genes related to motility, membrane proteins, and biofilms in *V. parahaemolyticus* (Guo et al. 2019). Similarly, luteolin isolated from coconut shells and linalool from *Coriandrum sativum* inhibit swarming and swimming, apart from inhibition and eradication of biofilms in *V. harveyi* (Sowndarya et al. 2019). Other examples of plant-derived active molecules, such as curcumin and resveratrol, effectively inhibited the flagellar motility of several species of *Vibrio* (Na et al. 2011; Packiavathy et al. 2013).

There are several examples of microbial compounds with inhibition properties against different *Vibrio* species (Table 2). Polymixin B, extracted from *Bacillus polymyxa*, inhibits swimming and biofilm formation of *V. cholerae* at an active concentration of 25 μg/mL (Giacomucci et al. 2019). 2,6-Di-tert-butyl-4-methyl phenol (DTBMP) extracted from *Chroococcus turgidus* has been reported to inhibit both swimming and swarming in *V. harveyi*, *V. parahaemolyticus*, and *V. vulnificus* (Santhakumari et al. 2018). As well as its biofilm-inhibiting property, it also inhibits the production of exopolysaccharide, the hydrophobicity index, and the production of bioluminescence in *V. harveyi*.

Polyhydroxy butyrate (PHB), an intracellular biopolymer derived from *Brevibacterium casei* MSI04, is reported to exert anti-swimming, anti-swarming, and anti-twitching activity (Seghal Kiran et al. 2016). The degradation intermediate of PHB inhibits the formation of biofilms and the production of hemolysin and QS signaling molecules in *Vibrio* sp. PUGSK8. Similarly, indole-3-carboxaldehyde extracted from *Marinomonas* sp. hinders swarming and biofilm formation in *V. parahaemolyticus*; however, it enhances its swimming motility (Rajalaxmi et al. 2016). Actinonin extracted from *Streptomyces* spp. and *Actinomyces* spp. suppresses swarming

## Naturally derived products

As they are eco-friendly, non-cytotoxic, and cost-effective, natural bioactive compounds are currently used as antimicrobial agents as well as food supplements (Sun et al. 2019).

| Compound name | Chemical structure | Sources | Active concentrations | Types of motility inhibition | Other inhibitory activity | Vibrio species | References |
|---------------|--------------------|---------|----------------------|-----------------------------|--------------------------|--------------|------------|
| Citral        | ![](https://i.imgur.com/CH3.png) | Lemongrass oil | Swimming at 12.5 μg/mL and swarming at 12.5 μg/mL | Swimming and swarming | It also inhibits the formation of biofilm and the production of autoinducer-2. Gene related to the biofilm formation, flagella synthesis, T3SS, and antibiotic resistance were also reported to be repressed by citral | V. parahaemolyticus | (Sun et al. 2019) |
| Polymyxin B   | ![](https://i.imgur.com/PolymyxinB.png) | Bacillus polymyxa | 25 μg/mL | Swimming | It inhibits the formation of biofilm at the initial stage | V. cholerae | (Giacomucci et al. 2019) |
| Thymoquinone | ![](https://i.imgur.com/Thymoquinone.png) | Nigella sativa | 3.2 mg/L | Swimming and swarming | It inhibits the formation of biofilm and also inhibits the attachment to the host cells. It also suppressed the expression of gene related to the motility, membrane protein, and biofilm-associated genes | V. parahaemolyticus | (Guo et al. 2019) |
| 2,6-Di-tert-butyl-4-methylphenol (DTBMP) | ![](https://i.imgur.com/DTBMP.png) | Chromococcus turgidus | 250 μg/mL | Swimming and swarming | Apart from its biofilm inhibiting property, it also inhibits the production of exopolysaccharide and hydrophobicity index. DT BMP also inhibits the production of bioluminescence in V. harveyi | V. harveyi, V. parahaemolyticus and V. vulnificus | (Santhakumar et al. 2018) |
| Extract in ethyl acetate | NA | Piper betle | 400 μg/mL | Swimming | The extract also inhibited the formation of biofilm, production of bioluminescence and exopolysaccharide | V. harveyi | (Srinivasan et al. 2017) |
| AgNPs         | NA | Prepared from the extracts of red algae (Gelidiella acerosa) | 100 μg/mL | Swimming and swarming | It inhibits the formation of biofilm, reduces the production of exopolysaccharide, hydrophobicity, and also reduces the colonization in the animal model. Additionally, it also exhibits the anti-QS activity | V. parahaemolyticus and V. vulnificus | (Satish et al. 2017) |
| ChemBridge-5343641 | ![](https://i.imgur.com/ChemBridge5343641.png) | Synthetic | 8 μg/mL | Swimming and swarming | It also plays a role in the inhibition of biofilm formation and dispersion of mature biofilm. Besides, it also acts as anti-QS as well as anti-bioluminescence | V. harveyi | (Rajamanikandan et al. 2017) |
| 2-(2,5-dimethylphenyl) amino] quinolin-4-yl} amino) ethanol (Q24DA) | ![](https://i.imgur.com/Q24DA.png) | Synthetic | 10 μg/mL | Swarming | It is considered as an inhibitor of Na+-driven flagellar motor. It paralyzed the polar flagellum by interacting with motor protein PomB. It also inhibits the formation of biofilm | V. cholerae | (Rasmussen et al. 2011; Wang et al. 2013a) |
| 3-Amino-1,8-naphthalimide | ![](https://i.imgur.com/3Amino18Naphthalimide.png) | Synthetic | 25 μM | Swimming and swarming | It shows a positive effect on the intestinal colonization of V. cholerae | V. cholerae and V. parahaemolyticus | (Wang et al. 2017) |

| Polyhydroxybutyrate (PHB) |  | Intracellular biopolymer from Brevibacterium casei MSI 04 | 100 μg/mL | Swimming, swarming and twitching | The degradation intermediate of PHB inhibits the formation of biofilm, and production of hemolysin and QS-signalling molecule | Vibrio sp. PUG SK 8 | (Seghal Kiran et al. 2016) |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Polyunsaturated fatty acids |  | Chemically synthesized | 300 μM | Swimming | The exogenous application of PUFA results in the alteration of membrane phospholipid in Vibrio species. It also inhibits the formation of biofilm | V. cholera, V. parahaemolyticus and V. vulnificus | (Moravecetal. 2017) |
| Zinc (Zn), selenium (Se), and manganese (Mn) | NA | Minerals | Zn (0.6 mM), Se (3 mM) and Mn (4.5 mM) | Swimming | These minerals also inhibit the production of cholera toxin and adhesion to the intestinal epithelial cells. These metal also downregulates the expression of gene related to virulence factor such as tcpA, fliA, ctxAB, and toxR | V. cholerae | (Bhattaramet al. 2017) |
| Luteolin |  | Luteolin from-coconut shells and linalool from Coriandrum sativum | Luteolin (200 μg/mL) and linalool (2.5%) | Swarming and swimming | It inhibits the formation of biofilm by inhibiting the production of EPS. It also dispersed mature biofilm. | V. harveyi | (Sowndarya et al. 2019) |
| Curcumin |  | Curcuma longa | 100 μg/mL | Swarming and swimming | It inhibits the formation of biofilm and attenuates the production of several virulence factors | V. parahaemolyticus, V. vulnificus and V. harveyi | (Packiavathy et al. 2013) |
| Indole-3-carboxaldehyde |  | Marinomonas sp. | 200 μg/mL | Swarming | Inhibits the formation of biofilm, however, it enhanced the swimming motility | V. parahaemolyticus | (Rajalaxmi et al. 2016) |
| Resveratrol |  | Natural polyphenol | 30 μM | Swarming | It inhibits the adhesion to the host cell and it downregulated the expression of RtxA1 toxin | V. vulnificus | (Kim et al. 2010) |
| Curcumin |  | Curcumin longa | 30 μM | Swarming | Inhibits the adhesion to the host cell as well as reduction of RtxA1 toxin | V. vulnificus | (Na et al. 2011) |
| Q Statin |  | Chemically synthesized | NA | NA | Although, there is no phenotypic assay for the motility, however, gene expression studies showed that it inhibits the gene related to the virulence, biofilm and motility or chemotaxis | V. vulnificus | (Kim et al. 2018) |
| Phenamil and amiloride |  | Synthetic | Phenamil (40 μM) and amiloride (2.25 mM) | Swarming | These are the inhibitors of the Na⁺-channel of the flagellum | V. parahaemolyticus | (Jaque set al. 1999) |
| Phenamil methanesulfonate |  | Synthetic | 50 μM | Swimming | It is characterized as an inhibitor of the Na⁺-driven flagellar motor | Vibrio harveyi | (Yang and Defoidt 2015) |
| Actinonin |  | Streptomyces spp. and Actinomycetes spp. | 5 μM | Swarming | Act as an antimicrobial agent against foodborne pathogens. It also attenuates the growth of S. Typhimurium and V. vulnificus | V. vulnificus | (Jung et al. 2017) |
| X-gal (5-bromo-4-chloro-3-indolyl-β-d-galactopyranoside |  | Chemically synthesized | 96 μg/mL | Swarming | NA | V. vulnificus MO6-24/O, V. alginolyticus, V. cholerae Non-O1, V. mimicus, or V. parahaemolyticus | (Kim et al. 2006) |
| Pumilacidin-like lipopeptide (CLP1 and CLP2) |  | Bacillus sp. 176 | 100 μg/mL | Swarming | It increases cell aggregation of V. alginolyticus and inhibits the formation of biofilm of other pathogenic bacteria | V. alginolyticus | (Xiu et al. 2017) |

and acts as an antimicrobial agent against foodborne pathogens. It also attenuates the growth of *S. Typhimurium* and *V. vulnificus* in a food model (Jung et al. 2017).

Apart from the isolation of active molecules from different organisms, recent research has also focused on ‘green’ synthesis of different types of nanomaterials using active biological materials from organisms such as plants, fungi, and bacteria (Alavi and Karimi 2018). It has been claimed that these nanomaterials synthesized from diverse biological extracts possess antimicrobial properties against multiple pathogenic bacteria. For example, silver (Ag) nanoparticles extracted from Red Alga (*Gelidiella acerosa*), at an active concentration of 100 μg/mL, demonstrated both anti-swarming and anti-swimming effects on *V. parahaemolyticus* and *V. vulnificus* (Satish et al. 2017). In addition to biofilm inhibition, they reduce the production of exopolysaccharide, displaying hydrophobicity, and colonization in the animal model. Additionally, they also exhibit anti-QS activity (Satish et al. 2017).

### Chemically synthesized compounds

Although naturally derived compounds are considered eco-friendly, cost-effective, and biocompatible, the ever-rising number of antimicrobial-resistant bacterial strains has become a major global concern in the treatment of related infections (Alavi and Karimi 2018). The advancement of modern chemical synthesis strategies has led to rapid synthesis of effective antimicrobial compounds (Wright et al. 2014). In addition, screening of existing compounds by employing high-throughput screening assays (HTS) can accelerate the identification of several motility inhibitors (Rasmussen et al. 2011).

Several chemically synthesized compounds or those identified in silico by screening have been reported to have inhibitory effects on *Vibrio* species. Quinonazoline-2,4-diamino analog (Q24DA) was identified through screening to be able to inhibit the motility of *V. cholerae* by paralyzing the polar flagellum (Rasmussen et al. 2011; Wang et al. 2013a). The mechanism of motility inhibition by Q24DA was found to involve an interaction with Na⁺-conducting channels, which are responsible for the rotation of the flagella motor. Wang et al. (2013a) also identified another antimotility compound, 3-amino 1,8-naphthalimide (3-A18NI), from an HTS (Wang et al. 2017). Compound 3-A18NI inhibits the swarming and swimming motility of *V. parahaemolyticus* and *V. cholerae* and/ the inhibition of cholera toxin production. Similarly, Jaques et al. (1999) identified phenamil and amiloride inhibitors, which also inhibit Na⁺-driven flagellar motility similar to Q24DA. Phenamil inhibitors also inhibit flagellar motility in other *Vibrio* species such as *V. algivolyticus* (Kawagishi et al. 1995) and *V. cholerae* (Rasmussen et al. 2011).

Similarly, HTS QStatin [1-(5-bromothiophene-2-sulfonyl)-1H-pyrazole], identified as a QS inhibitor, alters the master transcriptional regulator SmcR (a LuxR homolog) (Kim et al. 2018). Although no phenotypic assay has been performed for motility, gene expression studies have shown that QStatin inhibits genes related to virulence, biofilms, and motility or chemotaxis in *V. vulnificus*, interfering with the expression of genes regulated by SmcR. Similarly, a study on gene expression in *V. vulnificus* in the presence of sub-MIC tropodithietic acid showed that the latter downregulates motility-related genes; however, no phenotypic proof is available (Dittmann et al. 2019).

Polyunsaturated fatty acids (PUFAs) inhibit the swimming and exogenous application of PUFAs, resulting in the alteration of membrane phospholipids in *Vibrio* species. They also inhibit biofilm formation in *V. cholera*, *V. parahaemolyticus*, and *V. vulnificus* (Moravec et al. 2017).

Not only chemically synthesized or naturally isolated organic compounds, but several dietary minerals such as zinc (Zn), selenium (Se), and manganese (Mn) have also been reported as anti-swimming/anti-motility agents. These metals also inhibit the production of cholera toxin and adhesion to intestinal epithelial cells. Additionally, they downregulate the expression of genes related to virulence factors (Bhattaram et al. 2017). Another chemical X-gal (5-bromo-4-chloro-3-indolyl-β-d-galactopyranoside), used as a chromogenic substrate in molecular biology, effectively inhibited the swarming motility of several species of *Vibrio* (Kim et al. 2006). Other naturally derived or chemically synthesized compounds, their inhibition effect/rate on different motility types, active concentrations, sources for extraction, and target species are shown in Table 2.

### Conclusion and future perspectives

*Vibrio* species are highly diverse in their natural habitat in regard to their lifestyle, occurrence, and functionality. Being ubiquitous in the aquatic environment, some *Vibrio* species exist in free form, and some exhibit a relationship (either pathogenic or symbiotic) with eukaryotic hosts. These bacteria have unique survival properties in the aquatic habitat versus in a host, and reports have found that they can alternate between habitats (Yildiz and Visick 2009). The most important factor responsible for the survival of *Vibrio* spp. in their habitat (either aquatic or eukaryotic host) and protection against environmental stress, the host immune system, and predators, is the formation of the biofilm. The initial colonization of the host, adhesion, formation of biofilm, and production of virulence factors in *Vibrio* species are linked with flagellar motility. Furthermore, the genes associated with these cellular processes are linked together and are known to be tightly
regulated by the involvement of several regulatory and signaling molecules.

The present review has described a comparative signaling mechanism involved in the motility of *V. cholerae* and *V. parahaemolyticus*, which will be helpful in identifying and elucidating the signaling mechanisms and their regulation in other *Vibrio* species. The transcription of polar and lateral flagellar genes in *Vibrio* species is known to be regulated hierarchically by the transcriptional factor. Along with the master regulator of flagellar motility, several other signaling molecules, including c-di-GMP, are known to regulate motility in *Vibrio* species directly or indirectly. Reports have shown that the regulatory networks involved in controlling the polar and lateral flagellar systems are similar in *V. cholerae* and *V. parahaemolyticus*, which are closely related; however, they do not share structural and regulatory components (Tsang and Hoover 2014). Furthermore, the phylogenetic relatedness study of σ⁵⁴-dependent transcription regulators showed that the master regulator of the polar flagellar system is common in most *Vibrio* species; however, it is divergent from the master regulator of lateral flagellar systems. Furthermore, the relatedness analysis of the motility regulators of *V. cholerae* and *V. parahaemolyticus* using bidirectional best hits shows that some of the proteins are orthologue and exhibit high percentage identity.

Based on comparative studies, it is concluded that both *V. cholerae* and *V. parahaemolyticus* have well-developed regulatory mechanisms for the transition from a planktonic to a sessile lifestyle, although the regulatory mechanisms are different. Still, some of the unresolved questions associated with motility and biofilms need to be addressed, such as (1) how *V. parahaemolyticus* controls regulation and what factor triggers the functionality of the two different types of flagella (lateral and polar) during motility activity and the biofilm-forming state; (2) in the case of *V. parahaemolyticus*, what is the chemical nature of the S-signal produced from ScrA in the HCD state, which binds to the ScrB receptor and leads to the activation of the bifunctional protein ScrC; (3) how much c-di-GMP triggers flagellar movement and biofilm formation in both species and how the cell responds to various concentrations of c-di-GMP; (4) some *Vibrio* species utilize flagella as a surface attachment mechanism during biofilm formation, and so, it would be beneficial to know what factor arises, whether from the *Vibrio* cell or the attachment surface, to determine the adherence of flagella to the surface and assist in the formation of biofilm. Therefore, the study of the regulatory cascade and signaling molecules involved in flagellar motility and biofilm formation is an important research area for understanding the lifestyles of both pathogenic and symbiotic *Vibrio* spp.

Although motility is beneficial to symbionts of some non-pathogenic *Vibrio* species, as it facilitates the symbiotic relationship, several pathogenic *Vibrio* species have a negative impact on humans and aquatic organisms. Hence, to ameliorate the pathogenic and other damaging effects of vibrios,

several strategies such as inhibiting biofilm formation, attenuating the motility properties, and suppressing virulence factor production have been developed using naturally isolated or chemically synthesized compounds. The second part of this review described the inhibition of motility, including inhibition of biofilm formation and virulence factors by several compounds that were isolated from natural sources or synthesized by chemical means. Most of these compounds inhibit biofilm formation and suppress virulence properties in *Vibrio* species. In most cases, the inhibition properties of these compounds have been reported at the phenotypic level; however, the exact mechanisms of biofilm inhibition and attenuation of motility properties in these compounds have not been reported to date. In some cases, the expression of genes related to motility, biofilms, production of virulence factors, and toxins in these compounds has been reported. Information on the signaling cascade related to flagellar motility in *Vibrio* species would be useful to elucidate the mechanism of motility inhibition in these compounds using in silico approaches. In silico approaches have become a basis for the initial screening of several therapeutic drugs against microbial infections and other fatal diseases in humans (Amera et al. 2020; Gurung et al. 2020). The detailed procedures, tools, and software used for the computer-aided ligand-based screening of drugs are well reported in the literature (Malik et al. 2017; Morency et al. 2018). The chemical structure of these naturally derived or chemically synthesized compounds can be used to study their binding interactions with motility-related transcriptional factors and signaling proteins. Information obtained from the computer-based screening can be used to elucidate the mechanism of motility inhibition, which can be further validated experimentally. Since a single compound can exhibit multiple inhibitory properties, such as inhibition of motility, biofilm, and virulence, these alternative compounds can be considered as potential therapeutic and preventive agents against pathogenic vibrios. Furthermore, to enhance the efficacy of these compounds, combination therapy is considered another promising approach where these drugs can be either conjugated or combined with conventional antibiotics or other known virulence, biofilm, and motility inhibitors (Rybak and McGrath 1996; Tyers and Wright 2019). Moreover, to extend their specificity, minimize their cytotoxicity, and control their release, nanoformulation could be considered as another approach in which these compounds are loaded or encapsulated on to biocompatible nanoparticles (Forier et al. 2014).

**Author contribution** All authors were involved in the writing and editing of the manuscript.

**Funding information** This work was supported by Basic Science Research Program through the National Research Foundation of Korea funded by the Ministry of Education (NRF-2019R1A2C1087156).
Compliance with ethical standards

Conflict of interest The authors declare that they have no conflict of interest.

Ethical approval This review paper does not contain any studies with human participants or animals.

References

Alavi M, Karimi N (2018) Antiplanktonic, antibiofilm, antiswarming motility and antiquorum sensing activities of green synthesized Ag-TiO2, TiO2-Ag, Ag-Cu and Cu-Ag nanocomposites against multi-drug-resistant bacteria. Artif Cells Nanomed Biotechnol 46(sup3):S399–S413. https://doi.org/10.1080/21691401.2018.1496923

Amera GM, Khan RJ, Pathak A, Jha RK, Muthukumaran J, Singh AK (2020) Computer aided ligand based screening for identification of promising molecules against enzymes involved in peptidoglycan biosynthetic pathway from *Acinetobacter baumannii*. Microb Pathog 147:104205. https://doi.org/10.1016/j.micpath.2020.104205

Austin B (2010) Vibrios as causal agents of zoonoses. Vet Microbiol 140(3–4):310–317. https://doi.org/10.1016/j.vetmic.2009.03.015

Ayala JC, Wang H, Silva AJ, Benitez JA (2015) Repression by H-NS of genes required for the biosynthesis of the *Vibrio cholerae* biofilm matrix is modulated by the second messenger cyclic diguanylic acid. Mol Microbiol 97(4):630–645. https://doi.org/10.1111/mmi.13058

Ayala JC, Silva AJ, Benitez JA (2017) H-NS: an overarching regulator of the *Vibrio cholerae* life cycle. Res Microbiol 168(1):16–25. https://doi.org/10.1016/j.resmic.2016.07.007

Beloin C, Renard S, Ghigo JM, Lebeaux D (2014) Novel approaches to combat bacterial biofilms. Curr Opin Pharmacol 18:61–68. https://doi.org/10.1016/j.coph.2014.09.005

Beyhan S, Bilecen K, Salama SR, Casper-Lindley C, Yildiz FH (2007) Regulation of rugosity and biofilm formation in *Vibrio cholerae*: comparison of VpsT and VpsR regulons and epistasis analysis of vpsT, vpsR, and hapR. J Bacteriol 189(2):388–402. https://doi.org/10.1128/jb.00981-06

Bhattaram V, Upadhyay A, Yin HB, Mooyottu S, Venkitanarayanan K (2017) Effect of dietary minerals on virulence attributes of *Vibrio cholerae*. Front Microbiol 8:911. https://doi.org/10.3389/fmicb.2017.00911

Bhunia AK (2008) *Vibrio cholerae*, *V. parahaemolyticus*, *V. vulnificus* foodborne microbial pathogens: mechanisms and pathogenesis. Springer New York, New York, pp 241–252

Boin MA, Austin MJ, Hase CC (2004) Chemotaxis in *Vibrio cholerae*. FEMS Microbiol Lett 239(1):1–8. https://doi.org/10.1016/j.femsle.2004.08.039

Boles BR, McCarter LL (2002) *Vibrio parahaemolyticus scrABC*, a novel operon affecting swarming and capsular polysaccharide regulation. J Bacteriol 184(21):5946–5954. https://doi.org/10.1128/jb.184.21.5946-5954.2002

Boyd EF, Carpenter MR, Chowdhury N, Cohen AL, Haines-Menges BL, Kalburgi SS, Kingston JJ, Lubin JB, Ongagna-Yhombi SY, Whitaker WB (2015) Post-genomic analysis of members of the family vibrionaceae. Microbiol Spectr 3(5). https://doi.org/10.1128/microbiolspec.VE-0009-2014

Brenzinger S, Pecina A, Mrusek D, Mann P, Volse K, Wimmi S, Ruppert U, Becker A, Ringgaard S, Bange G, Thomann KM (2018) ZomB is essential for flagellar motor reversals in *Shewanella putrefaciens* and *Vibrio parahaemolyticus*. Mol Microbiol 109(5):694–709. https://doi.org/10.1111/mmi.14070

Butler SM, Camilli A (2004) Both chemotaxis and net motility greatly influence the infectivity of *Vibrio cholerae*. Proc Natl Acad Sci U S A 101(14):5018–5023. https://doi.org/10.1073/pnas.0308052101

Cai S, Cheng H, Pang H, Jian J, Wu Z (2018) AcfA is an essential regulator for pathogenesis of fish pathogen *Vibrio alginolyticus*. Vet Microbiol 213:35–41. https://doi.org/10.1016/j.vetmic.2017.11.016

Casper-Lindley C, Yildiz FH (2004) VpsT is a transcriptional regulator required for expression of vps biosynthesis genes and the development of rugose colonial morphology in *Vibrio cholerae* O1 El Tor. J Bacteriol 186(5):1574–1578. https://doi.org/10.1128/jb.186.5.1574-1578.2004

Charway GNA, Park S, Yu D, Je JY, Kim DH, Jung WK, Kim YM (2019) *In vitro* antibacterial and synergistic effect of chitosan-phytochemical conjugates against antibiotic resistant fish pathogenic bacteria. Indian J Microbiol 59(1):116–120. https://doi.org/10.1007/s12088-018-0750-0

Chen L, Qiu Y, Tang H, Hu LF, Yang WH, Zhu XJ, Huang XX, Wang T, Zhang YQ (2018) ToxR is required for biofilm formation and motility of *Vibrio parahaemolyticus*. Biomed Environ Sci 31(11):848–850. https://doi.org/10.3967/bes2018.112

Chiavelli DA, Marsh JW, Taylor RK (2001) The mannose-sensitive hemagglutinin of *Vibrio cholerae* promotes adherence to zooplankton. Appl Environ Microbiol 67(7):3220–3225. https://doi.org/10.1128/aem.67.7.3220-3225.2001

Chourashi R, Mondal M, Sinha R, Debnath A, Das S, Koley H, Chatterjee NS (2016) Role of a sensor histidine kinase ChiS of *Vibrio cholerae* in pathogenesis. Int J Med Microbiol 306(8):657–665. https://doi.org/10.1016/j.ijmm.2016.09.003

Chung CH, Fen SY, Yu SC, Wong HC (2016) Influence of oxyR on growth, biofilm formation, and mobility of *Vibrio parahaemolyticus*. Appl Environ Microbiol 82(3):788–796. https://doi.org/10.1128/AEM.02818-15

Commichau FM, Dickmanns A, Gundlach J, Ficner R, Stulke J (2015) A jack of all trades: the multiple roles of the unique essential second messenger cyclic di-AMP. Mol Microbiol 97(2):189–204. https://doi.org/10.1111/mmi.13026

Conner JG, Zamorano-Sánchez D, Park JH, Sondermann H, Yildiz FH (2017) The ins and outs of cyclic di-GMP signaling in *Vibrio cholerae*. Curr Opin Microbiol 36:20–29. https://doi.org/10.1016/j.mib.2017.01.002

Correa NE, Klose KE (2005) Characterization of enhancer binding by the *Vibrio cholerae* flagellar regulatory protein FlrC. J Bacteriol 187(9):3158–3170. https://doi.org/10.1128/JB.187.9.3158-3170.2005

Correa NE, Lauriano CM, McGee R, Klose KE (2000) Phosphorylation of the flagellar regulatory protein FlrC is necessary for *Vibrio cholerae* motility and enhanced colonization. Mol Microbiol 35(4):743–755. https://doi.org/10.1046/j.1365-2958.2000.01745.x

Correa NE, Barker JR, Klose KE (2004) The *Vibrio cholerae* FlgM homologue is an anti-sigma28 factor that is secreted through the sheathed polar flagellum. J Bacteriol 186(14):4613–4619. https://doi.org/10.1128/jb.186.14.4613-4619.2004

Correa NE, Peng F, Klose KE (2005) Roles of the regulatory proteins FlhF and FlhG in the *Vibrio cholerae* flagellar transcription hierarchy. J Bacteriol 187(18):6324–6332. https://doi.org/10.1128/JB.187.18.6324-6332.2005

Dang H, Lovell CR (2015) Microbial surface colonization and biofilm development in marine environments. Microbiol Mol Biol Rev 80(1):91–138. https://doi.org/10.1128/MMBR.00037-15

Dittmann KK, Porsby CH, Goncalves P, Mateiu RV, Sonnenschein EC, Bentzon-Tilia M, Egan S, Gram L (2019) Tropodithietic acid induces oxidative stress response, cell envelope biogenesis and iron uptake in *Vibrio vulnificus*. Environ Microbiol Rep 11(4):581–588. https://doi.org/10.1111/1758-2229.12771

Donlan RM, Costerton JW (2002) Biofilms: survival mechanisms of clinically relevant microorganisms. Clin Microbiol Rev 15(2):167–193. https://doi.org/10.1128/cmr.15.2.167-193.2002

Echazarreta MA, Klose KE (2019) Vibrio flagellar synthesis. Front Cell Infect Microbiol 9:131. https://doi.org/10.3389/fcimb.2019.00131

Enos-Berlage JL, McCarter LL (2000) Relation of capsular polysaccharide production and colonial cell organization to colony morphology in Vibrio parahaemolyticus. J Bacteriol 182(19):5513–5520. https://doi.org/10.1128/jb.182.19.5513-5520.2000

Eom SH, Lee DS, Jung YJ, Park JH, Choi JI, Yim MJ, Jeon JM, Kim HW, Son KT, Je JY, Lee MS, Kim YM (2014) The mechanism of antibacterial activity of phlorofucofuroeckol-A against methicillin-resistant Staphylococcus aureus. Appl Microbiol Biotechnol 98(23):9795–9804. https://doi.org/10.1007/s00253-014-6041-8

Ferreira RB, Antunes LC, Greenberg EP, McCarter LL (2008) Vibrio parahaemolyticus ScrC modulates cyclic dimeric GMP regulation of gene expression relevant to growth on surfaces. J Bacteriol 190(3):851–860. https://doi.org/10.1128/JB.01462-07

Ferreira RBR, Chodur DM, Antunes LCM, Trimble MJ, McCarter LL (2012) Output targets and transcriptional regulation by a cyclic dimeric GMP-responsive circuit in the Vibrio parahaemolyticus Scr network. J Bacteriol 194(5):914–924. https://doi.org/10.1128/jb.05807-11

Fleitas Martínez O, Cardoso MH, Ribeiro SM, Franco OL (2019) Recent advances in anti-virulence therapeutic strategies with a focus on dismantling bacterial membrane microdomains, toxin neutralization, quorum-sensing interference and biofilm inhibition. Front Cell Infect Microbiol 9:74. https://doi.org/10.3389/fcimb.2019.00074

Floyd KA, Lee CK, Xian W, Nametalla M, Valentine A, Crair B, Zhu S, Hughes HQ, Chlebek JL, Wu DC, Hwan Park J, Farhat AM, Lomba CJ, Ellison CK, Brun YV, Campos-Gomez J, Dalia AB, Liu J, Biais N, Wong GCL, Yildiz FH (2020) C-di-GMP modulates type IV MSHA pilus retraction and surface attachment in Vibrio cholerae. Nat Commun 11(1):1549. https://doi.org/10.1038/s41467-020-15331-8

Forier K, Raemdonck K, De Smedt SC, Demeester J, Coenye T, Braeckmans K (2014) Lipid and polymer nanoparticles for drug delivery to bacterial biofilms. J Control Release 190:607–623. https://doi.org/10.1016/j.jconrel.2014.03.055

Galperin MY, Nikolskaya AN, Koonin EV (2001) Novel domains of the prokaryotic two-component signal transduction systems. FEMS Microbiol Lett 203(1):11–21. https://doi.org/10.1111/j.1574-6968.2001.tb10814.x

Gardel CL, Mekalanos JJ (1996) Alterations in Vibrio cholerae motility phenotypes correlate with changes in virulence factor expression. Infect Immun 64(6):2246–2255

Ghosh A, Paul K, Chowdhury R (2006) Role of the histone-like nucleoid structuring protein in colonization, motility, and bile-dependent repression of virulence gene expression in Vibrio cholerae. Infect Immun 74(5):3060–3064. https://doi.org/10.1128/iai.74.5.3060-3064.2006

Giacomucci S, Cros CD, Perron X, Mathieu-Denoncourt A, Duperthuy M (2019) Flagella-dependent inhibition of biofilm formation by sub-inhibitory concentration of polymyxin B in Vibrio cholerae. PLoS One 14(8):e0221431. https://doi.org/10.1371/journal.pone.0221431

Gode-Potratz CJ, Chodur DM, McCarter LL (2010) Calcium and iron regulate swarming and type III secretion in Vibrio parahaemolyticus. J Bacteriol 192(22):6025–6038. https://doi.org/10.1128/JB.00654-10

Gode-Potratz CJ, Kustusch RJ, Breheny PJ, Weiss DS, McCarter LL (2011) Surface sensing in Vibrio parahaemolyticus triggers a programme of gene expression that promotes colonization and virulence. Mol Microbiol 79(1):240–263. https://doi.org/10.1111/j.1365-2958.2010.07445.x


Gu D, Meng H, Li Y, Ge H, Jiao X (2019) A GntR family transcription factor (VPA1701) for swarming motility and colonization of Vibrio parahaemolyticus. Pathogens 8(4):235. https://doi.org/10.3390/pathogens8040235

Guo D, Yang Z, Zheng X, Kang S, Yang Z, Xu Y, Shi C, Tian H, Xia X (2019) Thymoquinone inhibits biofilm formation and attachment-invasion in host cells of Vibrio parahaemolyticus. Foodborne Pathog Dis 16(10):671–678. https://doi.org/10.1089/fpd.2018.2591

Gurung AB, Ali MA, Lee J, Al-Hemaid F, Farah MA, Al-Anazi KM (2020) Molecular docking elucidates the plausible mechanisms underlying the anticancer properties of acetyldigitoxigenin from Adenium obesum. Saudi J Biol Sci. https://doi.org/10.1016/j.sjbs.2020.04.020

Güvener ZT, McCarter LL (2003) Multiple regulators control capsular polysaccharide production in Vibrio parahaemolyticus. J Bacteriol 185(18):5431–5441. https://doi.org/10.1128/jb.185.18.5431-5441.2003

Hathroubi S, Mekni MA, Domenico P, Nguyen D, Jacques M (2017) Biofilms: microbial shelters against antibiotics. Microb Drug Resist 23(2):147–156. https://doi.org/10.1089/mdr.2016.0087

Huang L, Xu W, Su Y, Zhao L, Yan Q (2018) Regulatory role of the RstB-RstA system in adhesion, biofilm production, motility, and hemolysis. Microbiologyopen 7(5):e00599. https://doi.org/10.1002/mbo3.599

Jaques S, McCarter LL (2006) Three new regulators of swarming in Vibrio parahaemolyticus. J Bacteriol 188(7):2625–2635. https://doi.org/10.1128/JB.188.7.2625-2635.2006

Jaques S, Kim YK, McCarter LL (1999) Mutations conferring resistance to phenamil and amiloride, inhibitors of sodium-driven motility of Vibrio parahaemolyticus. Proc Natl Acad Sci U S A 96(10):5740–5745. https://doi.org/10.1073/pnas.96.10.5740

Jeong HG, Choi SH (2008) Evidence that AphB, essential for the virulence of Vibrio vulnificus, is a global regulator. J Bacteriol 190(10):3768–3773. https://doi.org/10.1128/JB.00058-08

Jones CJ, Utada A, Davis KR, Thongsomboon W, Zamorano Sanchez D, Banakar V, Cegelski L, Wong GC, Yildiz FH (2015) C-di-GMP regulates motile to sessile transition by modulating MshA Pili biogenesis and near-surface motility behavior in Vibrio cholerae. PLoS Pathog 11(10):e1005068. https://doi.org/10.1371/journal.ppat.1005068

Jung D, Yum SJ, Jeong HG (2017) Characterization and evaluation of antimicrobial activity of actinonin against foodborne pathogens. Food Sci Biotechnol 26(6):1649–1657. https://doi.org/10.1007/s10068-017-0190-3

Kawagishi I, Maekawa Y, Atsumi T, Homma M, Imae Y (1995) Isolation of the polar and lateral flagellum-defective mutants in Vibrio alginolyticus and identification of their flagellar driving energy sources. J Bacteriol 177(17):5158–5160. https://doi.org/10.1128/jb.177.17.5158-5160.1995

Kearse M, Moir R, Wilson A, Stones-Havas S, Cheung M, Sturrock S, Buxton S, Cooper A, Markowitz S, Duran C, Thierer T, Ashton B, Meintjes P, Drummond A (2012) Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics 28(12):1647–1649. https://doi.org/10.1093/bioinformatics/bts199

Khan F, Khan MM, Kim YM (2018) Recent progress and future perspectives of antibiofilm drugs immobilized on nanomaterials. Curr Pharm Biotechnol 19(8):631–643. https://doi.org/10.2174/1389201019666180828090052

Khan F, Olokotuyi SF, Kim YM (2019a) Diversity of bacteria and bacterial products as antibiofilm and anti-quorum sensing drugs against pathogenic bacteria. Curr Drug Targets 20(11):1156–1179. https://doi.org/10.2174/1389450120666190423161249

Khan F, Pham DTN, Olokotuyi SF, Kim YM (2019b) Regulation and controlling the motility properties of Pseudomonas aeruginosa.

Appl Microbiol Biotechnol

---

Appl Microbiol Biotechnol 104(1):33–49. https://doi.org/10.1007/s00253-019-10201-w

Khan F, Pham DTN, Olokotuyi SF, Manivasagan P, Oh J, Kim YM (2020) Chitosan and their derivatives: antibiofilm drugs against pathogenic bacteria. Colloids Surf B Biointerfaces 185:110627. https://doi.org/10.1016/j.colsurfb.2019.110627

Kim YK, McCarter LL (2000) Analysis of the polar flagellar gene system of Vibrio parahaemolyticus. J Bacteriol 182(13):3693–3704. https://doi.org/10.1128/jb.182.13.3693-3704.2000

Kim YK, McCarter LL (2007) ScrG, a GGDEF-EAL protein, participates in regulating swarming and sticking in Vibrio parahaemolyticus. J Bacteriol 189(11):4094–4107. https://doi.org/10.1128/JB.01510-06

Kim MY, Park RY, Bai YH, Chung YY, Kim CM, Kim SY, Rhee JH, Shin SH (2006) X-Gal inhibits the swarming of Vibrio species. J Microbiol Methods 66(3):552–555. https://doi.org/10.1016/j.mimet.2006.01.004

Kim JR, Cha MH, Oh DR, Oh WK, Rhee JH, Kim YR (2010) Resveratrol modulates RTX toxin-induced cytotoxicity through interference in adhesion and toxin production. Eur J Pharmacol 642(1–3):163–168. https://doi.org/10.1016/j.ejphar.2010.05.037

Kim BS, Jang SY, Bang YJ, Hwang J, Koo Y, Jang KK, Lim D, Kim MH, Choi SH (2018) QStatin, a selective inhibitor of quorum sensing in Vibrio species. mBio 9(1):e02262-17. https://doi.org/10.1128/mBio.02262-17

Kimbrough JH, Cribbs JT, McCarter LL (2020) Homologous c-di-GMP-binding Scr transcription factors orchestrate biofilm development in Vibrio parahaemolyticus. J Bacteriol 202(6):e00723-e00719. https://doi.org/10.1128/jb.00723-19

Kirn TJ, Jude BA, Taylor RK (2005) A colonization factor links Vibrio cholerae environmental survival and human infection. Nature 438(7069):863–866. https://doi.org/10.1038/nature04249

Klose KE, Mekalanos JJ (1998) Distinct roles of an alternative sigma factor during both free-swimming and colonizing phases of the Vibrio cholerae pathogenic cycle. Mol Microbiol 28(3):501–520. https://doi.org/10.1046/j.1365-2958.1998.00809.x

Kojima S, Yamamoto K, Kawagishi I, Homma M (1999) The polar flagellar motor of Vibrio cholerae is driven by an Na+ motive force. J Bacteriol 181(6):1927–1930

Koo H, Allan RN, Howlin RP, Stoodley P, Hall-Stoodley L (2017) Targeting microbial biofilms: current and prospective therapeutic strategies. Nat Rev Microbiol 15(12):740–755. https://doi.org/10.1038/nrmicro.2017.99

Krasteva PV, Fong JCN, Shikuma NJ, Beyhan S, Navarro MVAS, Yildiz FH, Sondermann H (2010) Vibrio cholerae VpsT regulates matrix production and motility by directly sensing cyclic di-GMP. Science 327(5967):866–868. https://doi.org/10.1126/science.1181185

Laganenka L, López ME, Colin R, Sourjik V (2020) Flagellum-mediated mechanosensing and RflP control motility state of pathogenic Escherichia coli. mBio 11(2): e02269-19. https://doi.org/10.1128/mBio.02269-19

Lauriano CM, Ghosh C, Correa NE, Klose KE (2004) The sodium-driven flagellar motor controls exopolysaccharide expression in Vibrio cholerae. J Bacteriol 186(15):4864–4874. https://doi.org/10.1128/JB.186.15.4864-4874.2004

Lim JG, Choi SH (2014) IscR is a global regulator essential for pathogenesis of Vibrio vulnificus and induced by host cells. Infect Immun 82(2):569–578. https://doi.org/10.1128/IAI.01141-13

Lim B, Beyhan S, Meir J, Yildiz FH (2006) Cyclic-diGMP signal transduction systems in Vibrio cholerae: modulation of rugosity and biofilm formation. Mol Microbiol 60(2):331–348. https://doi.org/10.1111/j.1365-2958.2006.05106.x

Liu X, Beyhan S, Lim B, Linington RG, Yildiz FH (2010) Identification and characterization of a phosphodiesterase that inversely regulates motility and biofilm formation in Vibrio cholerae. J Bacteriol 192(18):4541–4552. https://doi.org/10.1128/jb.00209-10

Lu R, Osei-Adjei G, Huang X, Zhang Y (2018) Role and regulation of the orphan AphA protein of quorum sensing in pathogenic Vibrios. Future Microbiol 13:383–391. https://doi.org/10.2217/fmb-2017-0165

Lu L, Hu W, Tian Z, Yuan D, Yi G, Zhou Y, Cheng Q, Zhu J, Li M (2019) Developing natural products as potential anti-biofilm agents. Chin Med 14:11. https://doi.org/10.1186/s13020-019-0232-2

Luo J, Dong B, Wang K, Cai S, Liu T, Cheng X, Lei D, Chen Y, Li Y, Kong J, Chen Y (2017) Baicalin inhibits biofilm formation, attenuates the quorum sensing-controlled virulence and enhances Pseudomonas aeruginosa clearance in a mouse peritoneal implant infection model. PLoS One 12(4):e0176883. https://doi.org/10.1371/journal.pone.0176883

Lutz C, Erken M, Noorian P, Sun S, McDougald D (2013) Environmental reservoirs and mechanisms of persistence of Vibrio cholerae. Front Microbiol 4:375. https://doi.org/10.3389/fmicb.2013.00375

Malik V, Dhanjal JK, Kumari A, Radhakrishnan N, Singh K, Sundar D (2017) Function and structure-based screening of compounds, peptides and proteins to identify drug candidates. Methods 131:10–21. https://doi.org/10.1016/j.ymeth.2017.08.010

Martinez RM, Dharmasena MN, Kirn TJ, Taylor RK (2009) Characterization of two outer membrane proteins, FlgO and FlgP, that influence Vibrio cholerae motility. J Bacteriol 191(18):5669–5679. https://doi.org/10.1128/JB.00632-09

McCarter LL (1998) OpaR, a homolog of Vibrio harveyi LuxR, controls opacity of Vibrio parahaemolyticus. J Bacteriol 180(12):3166–3173

McCarter LL (2001) Polar flagellar motility of the Vibrionaceae. Microbiol Mol Biol Rev 65(3):445–462. https://doi.org/10.1128/mmbr.65.3.445-462.2001

McCarter LL (2004) Dual flagellar systems enable motility under different circumstances. J Mol Microbiol Biotechnol 7(1–2):18–29. https://doi.org/10.1159/000077866

McCarter LL, Wright ME (1993) Identification of genes encoding components of the swarmer cell flagellar motor and propeller and a sigma factor controlling differentiation of Vibrio parahaemolyticus. J Bacteriol 175(11):3361–3371. https://doi.org/10.1128/jb.175.11.3361-3371.1993

Meibom KL, Li XB, Nielsen AT, Wu CY, Roseman S, Schoolnik GK (2004) The Vibrio cholerae chitin utilization program. Proc Natl Acad Sci U S A 101(8):2524–2529. https://doi.org/10.1073/pnas.0308707101

Millikan DS, Ruby EG (2003) FlrA, a σ54-dependent transcriptional activator in Vibrio fischeri, is required for motility and symbiotic light-organ colonization. J Bacteriol 185(12):3547–3557. https://doi.org/10.1128/jb.185.12.3547-3557.2003

Miyashiro T, Oehlert D, Ray VA, Visick KL, Ruby EG (2014) The putative oligosaccharide translocase SypK connects biofilm formation with quorum signaling in Vibrio fischeri. Microbiologyopen 3(6):836–848. https://doi.org/10.1002/mbo3.199

Moisi M, Jenul C, Butler SM, New A, Tutz S, Reidl J, Klose KE, Camilli A, Schild S (2009) A novel regulatory protein involved in motility of Vibrio cholerae. J Bacteriol 191(22):7027–7038. https://doi.org/10.1128/JB.00948-09

Moravec AR, Siv AW, Hobby CR, Lindsay EN, Norbash LV, Shults DJ, Symes SJK, Giles DK (2017) Exogenous polyunsaturated fatty acids impact membrane remodeling and affect virulence phenotypes among pathogenic Vibrio species. Appl Environ Microbiol 83(22):e01415–e01417. https://doi.org/10.1128/AEM.01415-17

Morency LP, Gaudreault F, Najmanovich R (2018) Applications of the NRGsuite and the molecular docking software FlexAID in computational drug discovery and design. Methods Mol Biol 1762:367–388. https://doi.org/10.1007/978-1-4939-7756-7_18

Mulat M, Pandita A, Khan F (2019) Medicinal plant compounds for combating the multi-drug resistant pathogenic bacteria: a review. Curr Pharm Biotechnol 20(3):183–196. https://doi.org/10.2174/1872210513666190308133429

Na HS, Cha MH, Oh DR, Cho CW, Rhee JH, Kim YR (2011) Protective mechanism of curcumin against *Vibrio vulnificus* infection. FEMS Immunol Med Microbiol 63(3):355–362. https://doi.org/10.1111/j.1574-695X.2011.00855.x

Ni B, Colin R, Link H, Endres RG, Sourjik V (2020) Growth-rate dependent resource investment in bacterial motile behavior quantitatively follows potential benefit of chemotaxis. Proc Natl Acad Sci U S A 117(1):595–601. https://doi.org/10.1073/pnas.1910849117

Noh HJ, Nagami S, Kim MJ, Kim J, Lee NK, Lee KH, Park SJ (2015) Role of VcrD1 protein in expression and secretion of flagellar components in *Vibrio parahaemolyticus*. Arch Microbiol 197(3):397–410. https://doi.org/10.1007/s00203-014-1069-9

Osunla CA, Okoh AI (2017) *Vibrio* pathogens: a public health concern in rural water resources in sub-Saharan Africa. Int J Environ Res Public Health 14(10):1188. https://doi.org/10.3390/ijerph14101188

Packiavathy IA, Sasikumar P, Pandian SK, Veera Ravi A (2013) Prevention of quorum-sensing-mediated biofilm development and virulence factors production in *Vibrio* spp. by curcumin. Appl Microbiol Biotechnol 97(23):10177–10187. https://doi.org/10.1007/s00253-013-4704-5

Paul K, Nieto V, Carlquist WC, Blair DF, Harshey RM (2010) The c-di-GMP binding protein YcgR controls flagellar motor direction and speed to affect chemotaxis by a “backstop brake” mechanism. Mol Cell 38(1):128–139. https://doi.org/10.1016/j.molcel.2010.03.001

Phan NQ, Uebanso T, Shimohata T, Nakahashi M, Mawatari K, Takahashi A (2015) DNA-binding protein HU coordinates pathogenicity in *Vibrio parahaemolyticus*. J Bacteriol 197(18):2958–2964. https://doi.org/10.1128/JB.00306-15

Pratt JT, Tamayo R, Tischler AD, Camilli A (2007) PilZ domain proteins bind cyclic diguanylate and regulate diverse processes in *Vibrio cholerae*. J Biol Chem 282(17):12860–12870. https://doi.org/10.1074/jbc.M611593200

Pratt JT, McDonough E, Camilli A (2009) PhoB regulates motility, biofilms, and cyclic di-GMP in *Vibrio cholerae*. J Bacteriol 191(21):6632–6642. https://doi.org/10.1128/JB.00708-09

Prouty MG, Correa NE, Klose KE (2001) The novel sigma54- and sigma28-dependent flagellar gene transcription hierarchy of *Vibrio cholerae*. Mol Microbiol 39(6):1595–1609. https://doi.org/10.1046/j.1365-2958.2001.02348.x

Pursley BR, Maiden MM, Hsieh ML, Fernandez NL, Severin GB, Waters CM (2018) Cyclic di-GMP regulates TfoY in *Vibrio cholerae* to control motility by both transcriptional and posttranscriptional mechanisms. J Bacteriol 200(7):e00578–e00517. https://doi.org/10.1128/JB.00578-17

Rajalaxmi M, Amsa Devi V, Karutha Pandian S (2016) In vitro evaluation of indole-3-carboxaldehyde on *Vibrio parahaemolyticus* biofilms. Biologia 71(3):247–255

Rajamanikandan S, Jeyakanthan J, Srinivasan P (2017) Discovery of potent inhibitors targeting *Vibrio harveyi* LuxR through shape and e-pharmacophore based virtual screening and its biological evaluation. Microb Pathog 103:40–56. https://doi.org/10.1016/j.micpath.2016.12.003

Rasmussen L, White EL, Pathak A, Ayala JC, Wang H, Wu JH, Benitez JA, Silva AJ (2011) A high-throughput screening assay for inhibitors of bacterial motility identifies a novel inhibitor of the Na+-driven flagellar motor and virulence gene expression in *Vibrio cholerae*. Antimicrob Agents Chemother 55(9):4134–4143. https://doi.org/10.1128/AAC.00482-11

Reguera G, Kolter R (2005) Virulence and the environment: a novel role for *Vibrio cholerae* toxin-coregulated pili in biofilm formation on chitin. J Bacteriol 187(10):3551–3555. https://doi.org/10.1128/jb.187.10.3551-3555.2005

Reidl J, Klose KE (2002) *Vibrio cholerae* and cholera: out of the water and into the host. FEMS Microbiol Rev 26(2):125–139. https://doi.org/10.1111/j.1574-6976.2002.tb00605.x

Roelofs KG, Jones CJ, Helman SR, Shang X, Orr MW, Goodson JR, Galperin MY, Yildiz FH, Lee VT (2015) Systematic identification of cyclic-di-GMP binding proteins in *Vibrio cholerae* reveals a novel class of cyclic-di-GMP-binding ATPases associated with type II secretion systems. PLoS Pathog 11(10):e1005232. https://doi.org/10.1371/journal.ppat.1005232

Rogers A, Townsley L, Gallego-Hernandez AL, Beyhan S, Kwuan L, Yildiz FH (2016) The LonA protease regulates biofilm formation, motility, virulence, and the type VI secretion system in *Vibrio cholerae*. J Bacteriol 198(6):973–985. https://doi.org/10.1128/JB.00741-15

Romling U, Galperin MY, Gomelsky M (2013) Cyclic di-GMP: the first 25 years of a universal bacterial second messenger. Microbiol Mol Biol Rev 77(1):1–52. https://doi.org/10.1128/MMBR.00043-12

Roy R, Tiwari M, Donelli G, Tiwari V (2018) Strategies for combating bacterial biofilms: a focus on anti-biofilm agents and their mechanisms of action. Virulence 9(1):522–554. https://doi.org/10.1080/21505594.2017.1313372

Ruby EG, Urbanowski M, Campbell J, Dunn A, Faini M, Gunsalus R, Lostroh P, Lupp C, McCann J, Millikan D, Schaefer A, Stabb E, Stevens A, Visick K, Whistler C, Greenberg EP (2005) Complete genome sequence of *Vibrio fischeri*: a symbiotic bacterium with pathogenic congeners. Proc Natl Acad Sci U S A 102(8):3004–3009. https://doi.org/10.1073/pnas.0409900102

Rutherford ST, van Kessel JC, Shao Y, Bassler BL (2011) AphA and LuxR/HapR reciprocally control quorum sensing in vibrios. Genes Dev 25(4):397–408. https://doi.org/10.1101/gad.2015011

Rybak MJ, McGrath BJ (1996) Combination antimicrobial therapy for bacterial infections. Guidelines for the clinician. Drugs 52(3):390–405. https://doi.org/10.2165/00003495-199652030-00005

Ryjenkov DA, Simm R, Romling U, Gomelsky M (2006) The PilZ domain is a receptor for the second messenger c-di-GMP: the PilZ domain protein YcgR controls motility in enterobacteria. J Biol Chem 281(41):30310–30314. https://doi.org/10.1074/jbc.C600179200

Sakuma M, Nishikawa S, Inaba S, Nishigaki T, Kojima S, Homma M, Imada K (2019) Structure of the periplasmic domain of SflA involved in spatial regulation of the flagellar biogenesis of *Vibrio* reveals a TPR/SLR-like fold. J Biochem 166(2):197–204. https://doi.org/10.1093/jb/mvz027

Santhakumari S, Jayakumar R, Logalakshmi R, Prabhu NM, Abdul Nazar AK, Karutha Pandian S, Veera Ravi A (2018) In vitro and in vivo effect of 2,6-Di-tert-butyl-4-methylphenol as an antibiofilm agent against quorum sensing mediated biofilm formation of *Vibrio* spp. Int J Food Microbiol 281:60–71. https://doi.org/10.1016/j.ijfoodmicro.2018.05.024

Satish L, Santhakumari S, Gowrishankar S, Pandian SK, Ravi AV, Ramesh M (2017) Rapid biosynthesized AgNPs from *Gelidiella acerosa* aqueous extract mitigates quorum sensing mediated biofilm formation of *Vibrio* species-an in vitro and in vivo approach. Environ Sci Pollut Res Int 24(35):27254–27268. https://doi.org/10.1007/s11356-017-0296-4

Savoia D (2012) Plant-derived antimicrobial compounds: alternatives to antibiotics. Future Microbiol 7(8):979–990. https://doi.org/10.2217/fmb.12.68

Schild S, Tamayo R, Nelson EJ, Qadri F, Calderwood SB, Camilli A (2007) Genes induced late in infection increase fitness of *Vibrio cholerae* after release into the environment. Cell Host Microbe 2(4):264–277. https://doi.org/10.1016/j.chom.2007.09.004

Seghal Kiran G, Priyadharsini S, Dobson ADW, Gnanamani E, Selvin J (2016) Degradation intermediates of polyhydroxy butyrate inhibits phenotypic expression of virulence factors and biofilm formation in luminescent *Vibrio* sp. PUGSK8. NPJ Biofilms Microbi 2(16002). https://doi.org/10.1038/npjbiofilms.2016.2

Shikuma NJ, Davis KR, Fong JN, Yildiz FH (2013) The transcriptional regulator, CosR, controls compatible solute biosynthesis and

transport, motility and biofilm formation in *Vibrio cholerae*. Environ Microbiol 15(5):1387–1399. https://doi.org/10.1111/j.1462-2920.2012.02805.x

Shinoda S, Nakahara N, Uchida E, Hiraga M (1985) Lateral flagellar antigen of *Vibrio alginolyticus* and *Vibrio harveyi*: existence of serovars common to the two species. Microbiol Immunol 29(3): 173–182. https://doi.org/10.1111/j.1348-0421.1985.tb00817.x

Silva AJ, Benitez JA (2016) *Vibrio cholerae* biofilms and cholera pathogenesis. PLoS Negl Trop Dis 10(2):e0004330. https://doi.org/10.1371/journal.pntd.0004330

Sowndarya J, Farisa Banu S, Madhura G, Yuvalakshmi P, Rubini D, Bandeira Junior G, Baldisserotto B, Vadivel V, Nithyanand P (2019) Agro food by-products and essential oil constituents curtail virulence and biofilm of *Vibrio harveyi*. Microb Pathog 135:103633. https://doi.org/10.1016/j.micpath.2019.103633

Srinivasan R, Santhakumari S, Ravi AV (2017) *In vitro* antibiofilm efficacy of *Piper betle* against quorum sensing mediated biofilm formation of luminescent *Vibrio harveyi*. Microb Pathog 110:232–239. https://doi.org/10.1016/j.micpath.2017.07.001

Srivastava D, Hsieh ML, Khataokar A, Neiditch MB, Waters CM (2013) Cyclic di-GMP inhibits *Vibrio cholerae* motility by repressing induction of transcription and inducing extracellular polysaccharide production. Mol Microbiol 90(6):1262–1276. https://doi.org/10.1111/mmi.12432

Stewart BJ, McCarter LL (2003) Lateral flagellar gene system of *Vibrio parahaemolyticus*. J Bacteriol 185(15):4508–4518. https://doi.org/10.1128/jb.185.15.4508-4518.2003

Sun Y, Guo D, Hua Z, Sun H, Zheng Z, Xia X, Shi C (2019) Attenuation of multiple *Vibrio parahaemolyticus* virulence factors by citral. Front Microbiol 10:894. https://doi.org/10.3389/fmicb.2019.00894

Syed KA, Beyhan S, Correa N, Queen J, Liu J, Peng F, Satchell KJ, Yildiz F, Klose KE (2009) The *Vibrio cholerae* flagellar regulatory hierarchy controls expression of virulence factors. J Bacteriol 191(21): 6555–6570. https://doi.org/10.1128/JB.00949-09

Takekawa N, Kwon S, Nishioka N, Kojima S, Homma M (2016) HubP, a polar landmark protein, regulates flagellar number by assisting in the proper polar localization of FlhG in *Vibrio alginolyticus*. J Bacteriol 198(22):3091–3098. https://doi.org/10.1128/JB.00462-16

Tamayo R, Patimalla B, Camilli A (2010) Growth in a biofilm induces a hyperinfectious phenotype in *Vibrio cholerae*. Infect Immun 78(8): 3560–3569. https://doi.org/10.1128/IAI.00048-10

Tamplin ML, Gauzens AL, Huq A, Sack DA, Colwell RR (1990) Attachment of *Vibrio cholerae* serogroup O1 to zooplankton and phytoplankton of Bangladesh waters. Appl Environ Microbiol 56(6):1977–1980

Tamura K, Peterson D, Peterson N, Stecher G, Nei M, Kumar S (2011) MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods. Mol Biol Evol 28(10):2731–2739. https://doi.org/10.1093/molbev/msr121

Teschler JK, Zamorano-Sanchez D, Utada AS, Warner CJ, Wong GC, Lington RG, Yildiz FH (2015) Living in the matrix: assembly and control of *Vibrio cholerae* biofilms. Nat Rev Microbiol 13(5):255–268. https://doi.org/10.1038/nrmicro3433

Teschler JK, Cheng AT, Yildiz FH (2017) The two-component signal transduction system VxrAB positively regulates *Vibrio cholerae* biofilm formation. J Bacteriol 199(18):e00139–e00117. https://doi.org/10.1128/JB.00139-17

Tischler AD, Camilli A (2004) Cyclic diguanylate (c-di-GMP) regulates *Vibrio cholerae* biofilm formation. Mol Microbiol 53(3):857–869. https://doi.org/10.1111/j.1365-2958.2004.04155.x

Trimble MJ, McCarter LL (2011) Bis-(3′-5′)-cyclic dimeric GMP-linked quorum sensing controls swarming in *Vibrio parahaemolyticus*. Proc Natl Acad Sci U S A 108(44):18079–18084. https://doi.org/10.1073/pnas.1113790108

Tsang J, Hoover TR (2014) Themes and variations: regulation of RpoN-dependent flagellar genes across diverse bacterial species. Scientifica (Cairo) 2014:681754–681754. https://doi.org/10.1155/2014/681754

Tyers M, Wright GD (2019) Drug combinations: a strategy to extend the life of antibiotics in the 21st century. Nat Rev Microbiol 17(3):141–155. https://doi.org/10.1038/s41579-018-0141-x

Wang H, Wu J-H, Ayala JC, Benitez JA, Silva AJ (2011) Interplay among cyclic diguanylate, HapR, and the general stress response regulator (RpoS) in the regulation of *Vibrio cholerae* hemagglutinin/protease. J Bacteriol 193(23):6529–6538. https://doi.org/10.1128/JB.05166-11

Wang H, Ayala JC, Benitez JA, Silva AJ (2012a) Interaction of the histone-like nucleoid structuring protein and the general stress response regulator RpoS at *Vibrio cholerae* promoters that regulate motility and hemagglutinin/protease expression. J Bacteriol 194(5):1205–1215. https://doi.org/10.1128/jb.05900-11

Wang H, Ayala JC, Silva AJ, Benitez JA (2012b) The histone-like nucleoid structuring protein (H-NS) is a repressor of *Vibrio cholerae* exopolysaccharide biosynthesis (vps) genes. Appl Environ Microbiol 78(7):2482–2488. https://doi.org/10.1128/aem.07629-11

Wang H, Zhang L, Silva AJ, Benitez JA (2013a) A quinazoline-2,4-diamino analog suppresses *Vibrio cholerae* flagellar motility by interacting with motor protein PomB and induces envelope stress. Antimicrob Agents Chemother 57(8):3950–3959. https://doi.org/10.1128/AAC.00473-13

Wang L, Ling Y, Jiang H, Qiu Y, Qiu J, Chen H, Yang R, Zhou D (2013b) AphA is required for biofilm formation, motility, and virulence in pandemic *Vibrio parahaemolyticus*. Int J Food Microbiol 160(3): 245–251. https://doi.org/10.1016/j.ijfoodmicro.2012.11.004

Wang H, Ayala JC, Benitez JA, Silva AJ (2014) The LuxR-type regulator VpsT negatively controls the transcription of rpoS, encoding the general stress response regulator, in *Vibrio cholerae* biofilms. J Bacteriol 196(5):1020–1030. https://doi.org/10.1128/jb.00993-13

Wang H, Silva AJ, Benitez JA (2017) 3-Amino 1,8-naphthalimide, a structural analog of the anti-cholera drug virstatin inhibits chemically-biased swimming and swarming motility in vibrios. Microbes Infect 19(6):370–375. https://doi.org/10.1016/j.micinf.2017.03.003

Waters CM, Lu W, Rabinowitz JD, Bassler BL (2008) Quorum sensing controls biofilm formation in *Vibrio cholerae* through modulation of cyclic di-GMP levels and repression of vpsT. J Bacteriol 190(7): 2527–2536. https://doi.org/10.1128/JB.01756-07

Worthington RJ, Richards JJ, Melander C (2012) Small molecule control of bacterial biofilms. Org Biomol Chem 10(37):7457–7474. https://doi.org/10.1039/c2ob25835h

Wright PM, Seiple IB, Myers AG (2014) The evolving role of chemical synthesis in antibacterial drug discovery. Angew Chem Int Ed Engl 53(34):8840–8869. https://doi.org/10.1002/anie.201310843

Xiu P, Liu R, Zhang D, Sun C (2017) Pumilacidin-like lipopeptides derived from marine bacterium *Bacillus* sp. strain 176 suppress the motility of *Vibrio alginolyticus*. Appl Environ Microbiol 83(12): e00450–e00417. https://doi.org/10.1128/AEM.00450-17

Yang Q, Defoirdt T (2015) Quorum sensing positively regulates flagellar motility in pathogenic *Vibrio harveyi*. Environ Microbiol 17(4): 960–968. https://doi.org/10.1111/1462-2920.12420

Yang M, Frey EM, Liu Z, Bishar R, Zhu J (2010) The virulence transcriptional activator AphA enhances biofilm formation by *Vibrio cholerae* by activating expression of the biofilm regulator VpsT. Infect Immun 78(2):697–703. https://doi.org/10.1128/iai.00429-09

Yildiz FH, Schoolnik GK (1999) *Vibrio cholerae* O1 El Tor: identification of a gene cluster required for the rugose colony type, exopolysaccharide production, chlorine resistance, and biofilm formation. Proc Natl Acad Sci U S A 96(7):4028–4033. https://doi.org/10.1073/pnas.96.7.4028

Yildiz FH, Visick KL (2009) *Vibrio* biofilms: so much the same yet so different. Trends Microbiol 17(3):109–118. [https://doi.org/10.1016/j.tim.2008.12.004](https://doi.org/10.1016/j.tim.2008.12.004)

Yildiz FH, Dolganov NA, Schoolnik GK (2001) VpsR, a member of the response regulators of the two-component regulatory systems, is required for expression of vps biosynthesis genes and EPS(ETr)-associated phenotypes in *Vibrio cholerae* O1 El Tor. J Bacteriol 183(5):1716–1726. [https://doi.org/10.1128/jb.183.5.1716-1726.2001](https://doi.org/10.1128/jb.183.5.1716-1726.2001)

Zamorano-Sánchez D, Fong JCN, Kilic S, Erill I, Yildiz FH (2015) Identification and characterization of VpsR and VpsT binding sites in *Vibrio cholerae*. J Bacteriol 197(7):1221–1235. [https://doi.org/10.1128/JB.02439-14](https://doi.org/10.1128/JB.02439-14)

Zhang L, Osei-Adjei G, Zhang Y, Gao H, Yang W, Zhou D, Huang X, Yang H, Zhang Y (2017) CalR is required for the expression of T6SS2 and the adhesion of *Vibrio parahaemolyticus* to HeLa cells. Arch Microbiol 199(6):931–938. [https://doi.org/10.1007/s00203-017-1361-6](https://doi.org/10.1007/s00203-017-1361-6)

Zhang L, Weng Y, Wu Y, Wang X, Yin Z, Yang H, Yang W, Zhang Y (2018) H-NS is an activator of exopolysaccharide biosynthesis genes transcription in *Vibrio parahaemolyticus*. Microb Pathog 116:164–167. [https://doi.org/10.1016/j.micpath.2018.01.025](https://doi.org/10.1016/j.micpath.2018.01.025)

Zhu J, Mekalanos JJ (2003) Quorum sensing-dependent biofilms enhance colonization in *Vibrio cholerae*. Dev Cell 5(4):647–656. [https://doi.org/10.1016/s1534-5807(03)00295-8](https://doi.org/10.1016/s1534-5807(03)00295-8)

Zhu S, Kojima S, Homma M (2013) Structure, gene regulation and environmental response of flagella in *Vibrio*. Front Microbiol 4:410. [https://doi.org/10.3389/fmicb.2013.00410](https://doi.org/10.3389/fmicb.2013.00410)

Zorraquino V, García B, Latasa C, Echeverz M, Toledo-Arana A, Valle J, Lasa I, Solano C (2013) Coordinated cyclic-Di-GMP repression of *Salmonella* motility through YcgR and cellulose. J Bacteriol 195(3):417–428. [https://doi.org/10.1128/jb.01789-12](https://doi.org/10.1128/jb.01789-12)

Publisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.
