
Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

Review Article  

# Unpuzzling COVID-19: tissue-related signaling pathways associated with SARS-CoV-2 infection and transmission  

Daniella S. Battagello${}^{1,*}$, Guilherme Dragunas${}^{2,*}$, Marianne O. Klein${}^{1,*}$, Ana L.P. Ayub${}^{3}$, Fernando J. Velloso${}^{4}$ and Ricardo G. Correa${}^{5}$  

${}^{1}$ Department of Anatomy, Institute of Biomedical Sciences, University of São Paulo (USP), São Paulo, Brazil; ${}^{2}$Department of Pharmacology, Institute of Biomedical Sciences, University of São Paulo (USP), São Paulo, Brazil; ${}^{3}$Department of Pharmacology, Federal University of São Paulo (UNIFESP), São Paulo, Brazil; ${}^{4}$Department of Pharmacology, Physiology and Neurosciences, Rutgers-NJMS, Newark, NJ, U.S.A.; ${}^{5}$NCI-Designated Cancer Center, Sanford Burnham Prebys (SBP) Medical Discovery Institute, La Jolla, CA, U.S.A.  

Correspondence: Ricardo Garcia Correa (r.correa@sbpdiscovery.org)  

OPEN ACCESS  

The highly infective coronavirus disease 19 (COVID-19) is caused by a novel strain of coronaviruses – the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) – discovered in December 2019 in the city of Wuhan (Hubei Province, China). Remarkably, COVID-19 has rapidly spread across all continents and turned into a public health emergency, which was ultimately declared as a pandemic by the World Health Organization (WHO) in early 2020. SARS-CoV-2 presents similar aspects to other members of the coronavirus family, mainly regarding its genome, protein structure and intracellular mechanisms, that may translate into mild (or even asymptomatic) to severe infectious conditions. Although the mechanistic features underlying the COVID-19 progression have not been fully clarified, current evidence have suggested that SARS-CoV-2 may primarily behave as other β-coronavirus members. To better understand the development and transmission of COVID-19, unveiling the signaling pathways that may be impacted by SARS-CoV-2 infection, at the molecular and cellular levels, is of crucial importance. In this review, we present the main aspects related to the origin, classification, etiology and clinical impact of SARS-CoV-2. Specifically, here we describe the potential mechanisms of cellular interaction and signaling pathways, elicited by functional receptors, in major targeted tissues/organs from the respiratory, gastrointestinal (GI), cardiovascular, renal, and nervous systems. Furthermore, the potential involvement of these signaling pathways in evoking the onset and progression of COVID-19 symptoms in these organ systems are presently discussed. A brief description of future perspectives related to potential COVID-19 treatments is also highlighted.

## The new coronavirus disease 19: SARS-CoV-2 Origin, classification, transmission and clinical features

Coronaviruses (CoVs), a family of viruses identified in humans in late 1960s [1,2], are considered relevant pathogens that can infect a broad range of hosts, such as bats, rodents, civets, livestock and arabian camels [3]. In humans, CoV infection may result into mild to severe cases conditions that impact the respiratory, gastrointestinal (GI) and/or central nervous system (CNS) systems [3–5]. Taking into consideration their genomic and phylogenetic features, CoVs are single-stranded positive-sense RNA viruses, largely enveloped in a lipid bilayer, that belong to the *Coronaviridae* family (*Coronavirinae* subfamily, *Nidovirales* order). This virus family consists of four genera (*α-*, *β-*, *γ-* and *δ*-coronavirus), among of which only *α*-coronaviruses and *β*-coronaviruses are capable of infecting mammals [3,5–12].

*These authors contributed equally to this work.*

Received: 03 July 2020  
Revised: 10 August 2020  
Accepted: 12 August 2020  
Version of Record published: 21 August 2020  

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).
For the last two centuries, no other severe acute respiratory syndrome (SARS) caused by the other six described human coronaviruses [13], such as SARS-CoV [6,14] and Middle-East respiratory syndrome coronavirus (MERS-CoV) [7,15], has affected world population in such an unprecedented manner as the most recent β-coronavirus SARS-CoV-2 [9,12,16,17].

First named as 2019 novel coronavirus (2019-nCoV), SARS-CoV-2 was discovered in December 2019 in Wuhan (capital of Hubei Province, China), acting as an unknown pneumonia-causing agent. It has been credited that SARS-CoV-2 originated from zoonotic transfer of bat coronaviruses, possibly through animals in this location [9,12,18–20]. Since then, the coronavirus disease 19 (COVID-19) has rapidly spread across all continents, becoming a public health emergency (pandemic) as announced by World Health Organization (WHO) in early 2020 [21].

Possible routes of SARS-CoV-2 viral transmission include direct and contact transmissions, such as human to human interaction by droplet inhalation and/or contact with oral/nasal membranes, as well as nosocomial contamination [11,22]. The main COVID-19 symptoms are fever and cough, but other conditions such as anosmia, cardiovascular and GI disorders have been increasingly reported, thus suggesting the presence of multiple targets of infection outside the respiratory tract [9,11,23]. Moreover, COVID-19 has been reported to be particularly more severe in patients with comorbidities unrelated to the respiratory tract, such as hypertension, diabetes and cardiovascular disease [24–27].

Current literature have presented evidence for the potential ability of SARS-CoV-2 to primarily behave as other coronavirus members, such as SARS-CoV and MERS-CoV, to further induce distinct human conditions, but the mechanisms underlying the development of COVID-19 have been poorly elucidated. Therefore, unveiling the signaling pathways elicited (or repressed) upon entry of SARS-CoV-2 into host cells can provide a better knowledge about COVID-19 and also direct to potential pharmacological targets that may counterbalance some of the crucial pathological marks due to this new coronavirus. In this review, we explore the current knowledge of SARS-CoV-2 infection and transmission, focusing on the main aspects of cellular signaling pathways that are impacted by SARS-CoV-2 in targeted organ systems.

## Mechanisms of cellular interaction by angiotensin-converting enzyme 2

Similar to other β-coronaviruses, SARS-CoV-2 is mainly composed by four compartments with distinct roles in the viral replication: membrane spike glycoprotein (S), membrane (M), envelope (E) and nucleocapsid (N) [5]. Additionally, SARS-CoV-2 present biological features that resemble other β-coronaviruses class members, especially SARS-CoV, such as genome, protein structure, infection mechanisms [mainly involving the interaction with angiotensin-converting enzyme 2 (ACE2)] and tissue tropism [8,16,20,28,29].

The angiotensin-converting enzyme (ACE) homolog metallopeptidase ACE2 is widely expressed in the human body, including renal, lymphoid and cardiovascular tissues as well as gastrointestinal (duodenum, jejunum, ileum, cecum and colon), respiratory and central nervous systems [30]. Hamming et al. (2004) have determined the distribution of ACE2 protein by immunohistochemistry, which corroborated previous mRNA expression data. Moreover, relevant immunolabeling identified ACE2 protein in alveolar epithelium cells and capillary endothelium of the lungs, small intestine epithelia, blood vessels and capillaries of the skin, brain endothelium and renal glomerular epithelium [31]. According to studies that elucidated (i) the synthesis of ACE2, (ii) the detection of ACE2 in organs targeted by SARS-CoV-2 and (iii) the mechanisms associating ACE2 with the invasion/replication of coronaviruses, it has demonstrated that ACE2 serves as a functional receptor of SARS-CoV and, particularly, SARS-CoV-2 (Figure 1) [20,24,28,31,32].

The cellular entry of SARS-CoV-2 is mediated by a high affinity binding of the spike (S) protein to ACE2 and the processing of transmembrane serine protease 2 (TMPRSS2) in the host cell surface to allow spike (S) priming [28]. Taking into consideration the high similarity between SARS-CoV and SARS-CoV-2, in regard to sequence conservation and structure of glycoproteins (Walls et al. 2020; Hoffmann et al. 2020,) and that SARS-CoV leads to the down-regulation of ACE2 [33], a recent study has suggested that this particular interaction may also occur in COVID-19 [34], thus promoting membrane fusion and SARS-CoV-2 entry into host cells. Hence, the co-expression of TMPRSSs and ACE2 is a key factor that determines the entry of SARS-CoV-2 into host cells [16,28,35]. Still, compared with SARS-CoV, it has been showed that the spike (S) protein from the SARS-CoV-2 binds to ACE2 with ~ten-fold higher affinity, which can facilitate virus invasion in the cells and enable its spread to a variety of tissues [36].

Interestingly, single cell transcriptomics has also shown that ACE2 expression is highly correlated with the expression of alanyl aminopeptidase (ANPEP) and dipeptidyl peptidase-4 (DPP4), known receptors for other human

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).
Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

![Image Description](image_url)

Figure 1. Putative tissues/organs infected by SARS-CoV-2 and related COVID-19 symptoms

(1) Viral entry depends on the binding of spike (S) viral envelope protein to ACE2 in the host cell surface. TMPRSS2, together with other proteases, processes S protein and allows viral endocytosis [28]. (2) The digestive system, heart, kidneys, respiratory system and peripheral neurons are tissues/cells where SARS-CoV-2 might infect, generating different symptoms observable in COVID-19 patients [31,32,90,142,196,250–252]. (3) Upon tissue infection, neurons that innervate those tissues could potentially be invaded by SARS-CoV-2 and infect the CNS by trans-synaptic route exchange (via peripheral nerves), thus promoting an interneuronal transfer of SARS-CoV-2, similar to other coronaviruses [183,184]. Nevertheless, other potential routes for CNS infection have also been hypothesized [166,176].

CoVs [37–39]. This result suggests that these peptidases may act as co-receptors or auxiliary SARS-CoV-2 receptors [40]. Furthermore, the identification of the transmembrane glycoprotein CD147 [41] as well as the presence of furin-like cleavage sites in the spike (S) protein (absent for other SARS-CoVs) [42] might be associated to viral-host mechanisms of invasion and pathogenicity of COVID-19.

ACE2 is a key enzyme in the Renin-Angiotensin system (RAS) which plays a physiological role in regulating renal-cardiovascular systems and the innate immunity [43,44]. In the RAS pathway, renin produced in the kidneys cleaves Angiotensinogen from the liver, producing Angiotensin (Ang)-I. The latter is cleaved by ACE into Ang-II (the substrate of ACE2) which binds to the Angiotensin II type 1 receptor (AT1R) and Angiotensin II type 2 receptor (AT2R) [44,45]. Due to its relationship with ACE2, the RAS system appears to have a central role in SARS-CoV-2 infection (Figure 2).

ACE2 is up-regulated via interferon (IFN)-mediated gene expression [46], whereas SARS-CoV-2 induces ACE2 down-regulation in multiple tissues [35]. Even though it might seem beneficial, ACE2 inhibition leads to a more extensive conversion of Ang-I into Ang-II via ACE, which then binds to ATR1 receptors to further promote vascular permeability by JAK/STAT signaling pathway [47–49]. ACE2 is capable of converting Ang-I into angiotensin-(1-7), that binds Mas receptors and leads to anti-fibrotic and anti-inflammatory effects in endothelial cells [49]. Moreover, IL-6, IL-1β and IFN-γ have also been reported to inhibit ACE2 expression, thus changing the balance of Ang-2/Ang-(1-7) in favor of inflammation and vascular permeability [33,50].

## Cytokine storm and activation of signaling pathways after SARS-CoV-2 infection

A number of cytokines, including IL-6, IL-1β, tumor necrosis factor α (TNF-α) and IFN-γ, have been frequently reported to be elevated in COVID-19 [9,51]. A putative systemic outcome due to this effect is known as cytokine release

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

![Image Description](image.png)

Figure 2. Canonical ACE2 pathway links multiple organ damage in COVID-19

SARS-CoV-2 infection down-regulates ACE2 expression and leads to the production of pro-inflammatory mediators, such as IL-6 [35]. Angiotensin-I (Ang-I) is converted into Ang-II by the ACE in the extracellular space. ACE2 is able to further cleave Ang-II to Ang(1-7), which binds MasR receptors on the cell surface and promotes anti-inflammatory, vasodilation and anti-fibrotic effects [35]. Since ACE2 is down-regulated during viral infection, this event will lead to the accumulation of Ang-II and binding to AT1R receptors on cellular membrane. AT1R signals through JAK-STAT and induces fibrosis, pro-inflammatory gene expression and vasoconstriction [48,251]. Multiple organs express ACE2 and are target for SARS-CoV-2. As they lose ACE2-mediated protection, Ang-II signaling contributes to the pathological findings observed in COVID-19 patients, such as disseminated coagulopathy and acute tissue damage [91].

---

syndrome (CRS), also called ‘cytokine storm’. CRS is believed to be a major cause of tissue damage in the pathophysiology of COVID-19 [52]. CRS is characterized by an overactive immune response that results in an excessive systemic increase in pro-inflammatory cytokines in response to external stimuli, autoimmune disease(s) and/or tumorigenesis [53]. CRS is a two-step process where the primary response is characterized by the activation of innate immunity following viral infection in epithelial cells. Epithelial, innate immune and endothelial cells release several cytokines to block the viral replication, while effector cells are recruited to remove infected cells. A secondary cytokine cascade is induced downstream by the sustained release of primary cytokines or by immune cell signaling [54]. IL-6, the most important CRS causative cytokine [55], was found to be increased in the serum of COVID-19 patients presenting acute respiratory distress syndrome (ARDS) [56].

SARS-CoV-2 and the elicited cytokine storm activate distinct signaling pathways in infected cells/tissues. These cascades are important for the disease development and, therefore, may serve as potential therapeutic targets (Figure 3). Certain cytokines whose levels are elevated in COVID-19 patients (i.e. IL-6, IL-1β and IFN-γ), are important activators of the Janus kinase (JAK)/signal transducer of activators of transcription (STAT) JAK/STAT pathway and also able to induce NF-kB signaling [57]. In particular, it has been shown that IL-6 may induce the expression of Ang-II, which, in turn, also promotes the expression of IL-6 itself via JAK/STAT, creating a degenerative feedback loop [58]. JAK/STAT participates in the quick transmission of extracellular signals from cytokines, IFNs, colony-stimulating factors and hormones, promoting changes in gene expression via STAT-related transcription factors [59]. Four JAKs (JAK1/2/3 and TYK2) and seven STATs (STAT1/2/3/4/5a/5b/6) have been noted to impact this pathway, resulting in differential biological outcomes. Upon receptor binding, JAK proteins are cross-phosphorylated and then recruit STAT proteins, which will be further phosphorylated. Upon phosphorylation, STATs dimerize and translocate to the nucleus, where they bind to specific DNA sequences and regulate gene expression [59].

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

![Diagram](attachment:diagram.png)

Figure 3. Signaling pathways involved in COVID-19 pathophysiology

Toll-like receptors (TLRs) 3 and TLR 7/8 recognize SARS-CoV-2 RNA and initiate the inflammatory cascade via type I and type II IFN gene expression and NF-κB nuclear translocation [98,107]. Via NF-κB, the expression of multiple pro-inflammatory genes is stimulated, including pro-IL-1β, pro-IL-18, TNF and IL-6 [62–64]. The virus is also recognized by cytoplasmic NLRP3, which forms, together with ASC and caspase-1 (Casp-1), the inflammasome complex that will cleave and release mature forms of IL-1β and IL-18 [122]. The cytokines IL-1β, IL-18 and TNF bind to specific receptors and promote further NF-κB nuclear translocation and phosphorylation of p38 MAPK, which will lead to great expression of pro-inflammatory cytokines and chemokines [69,135]. IL-6, an important player in COVID-19, binds IL-6R and gp130 receptors to activate JAK/STAT-3 pathway and then contribute to the CRS observed in COVID-19 patients [88].

The nuclear factor-κB (NF-κB) family of transcription factors is responsible for a major overlay of inflammatory signaling, thus enhancing the gene expression of multiple molecules that are also elicited by SARS-CoV-2 infection [60,61]. Canonically, the pathway is triggered upon binding of ligands or antigens to cytokines receptors, toll-like receptors (TLRs) or T-cell receptors [62]. Sequential phosphorylation events lead to the phosphorylation of IKKα and IKKβ, which are crucial kinases involved in the phosphorylation of inhibitory IkB protein which is bound to p50 and p65 NF-κB subunits. Phosphorylation of IkB tags this protein to ubiquination and proteasomal degradation, thus releasing p50/p65 dimers to translocate into the nucleus and then bind to specific enhancer regions that mediate the expression of κB-responsive genes [62–66].

Another intracellular signaling pathway, involving mitogen-activated protein kinases (MAPKs), plays a series of roles in cell differentiation, proliferation and death in response to distinct environmental stimuli. In mammals, MAPKs are involved in three major families: (i) the extracellular signal-regulated kinases (ERKs), (ii) the Jun amino-terminal kinases (JNKs) and (iii) p38 MAPKs, also known as stress-activated protein kinases (SAPKs) [67]. Among them, the p38s have already been described to be involved in SARS-CoV infection and, therefore, suggested as putative targets for COVID-19 treatment [68,69].

The p38 MAPK pathway is largely induced by pro-inflammatory factors and environmental stresses, which prominently impact a subset of physiological events, such as immune response, as well as inflammatory processes [70]. Four variants of the p38 family (α, β, γ and δ) have been identified so far [71–74]. These protein variants present distinct patterns of expression in cells/tissues, where p38α and p38β are ubiquitous while p38γ and p38δ are more tissue-specific [75]. The p38 MAPK pathway consists of three main core protein kinases that act sequentially, thus providing the specificity and diversity inherent to this signaling cascade [70,76]. This pathway may be activated by a variety of extracellular stimulus including cellular stress, G-coupled protein receptors (GPCRs), growth factors, inflammatory cytokines, TGFβ and IL-1 [67]. In the canonical pathway, the main activators of MAP3K are (i) the apoptosis signal-regulating kinase (ASK), (ii) the dual-leucine-zipper-bearing kinase (DLK), (iii) the MAPK/ERK kinase

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).

kinase (MEKK), (iv) the mixed-lineage kinase 3 (MLK3) and (v) the TGF (transforming growth factor) β-activated kinase 1 (TAK1) [70,77]. MAP3Ks are phosphorylated and then directly activate MAP2Ks. MKK3 and MKK6 are the most common MAP2Ks responsible by activating p38 MAPKs, although MKK3 is not able to activate p38β [76–79]. Functionally, p38 can phosphorylate other protein kinases, such as MAP kinase activated protein kinase 2 (MK2), as well as transcription factors (ATF1/2/6) and p53. In addition, p38 can regulate the transcription of genes encoding a number of cytokines and cell surface receptors [80–82]. It has also been described that p38 may function into the post-transcriptional regulation of inflammatory cytokines as well as in TNF-α and IL-1β translation [75,83,84]. In this sense, activation of p38 pathway is fundamental to increase the yields of pro-inflammatory cytokines, such as IL-6, TNF-α and IL-1β, which seem to play major roles in the cytokine storm produced by SARS-CoV-2 infection [61,75]. Importantly, a cross-talk activation of NF-κB signaling is mediated by p38 MAPKs, which might potentiate the production of pro-inflammatory cytokines [85].

In regard to antiviral defense, IL-6 secretion can disturb viral clearance and prolong infection via induction of Th2 polarization and inhibition of Th1 CD4⁺ lymphocytes differentiation, thus impairing IFN-mediated antiviral immunity [86]. IL-6 is also capable of inducing SOCS-1 expression via STAT3, which impairs STAT1 phosphorylation and, ultimately, decreases IFN-γ levels [86,87]. IL-6 binds to IL-6R and gp130 in both T helper 1 (Th1) CD4⁺ lymphocytes and NK cells in the lungs. Upon receptor binding, JAKs are recruited and phosphorylated to further induce STAT-mediated transcription, leading to a prominent release of pro-inflammatory mediators [87]. IL-6 also acts non-canonically by modulating a myriad of signaling cascades, ranging from MAPK to Notch pathways [88]. Taken together, we strongly believe that understanding the multitude of signaling routes affected by viral infections, in a cell/tissue-specific manner, can be crucial for the comprehension of the pathobiology as well as the therapeutics for COVID-19.

# Major organ systems affected by SARS-CoV-2

Considering that ACE2 has been identified in the lungs, intestines, cardiovascular tissues, brain and kidneys [30], these organs have been coincidentally targeted by SARS-CoV-2 to further activate intracellular signaling pathways leading to CRS. Accordingly, symptoms related to all these systems have been reported in COVID-19 patients [9].

## Respiratory system

The main clinical complication due to COVID-19, which also leads to a high fatality rate and affects ~42% of the patients, refers to the ARDS [11]. ARDS is an aggressive lung condition (devoid of cardiovascular causes) which is characterized by a ratio of partial pressure arterial oxygen versus the fraction of inspired oxygen (PaO₂/FiO₂ ratio) lower than 300 mm Hg [89]. COVID-19 patients affected by ARDS develop a severe form of this condition (i.e. PaO₂/FiO₂ ratio < 100 mmHg) in 18.9% of the cases [90]. The main characteristics of ARDS include (i) severe pulmonary edema due to increased vascular permeability and plasma leakage, and (ii) severe hypoxemia as a result of poor gas exchange in the alveoli [89]. Pathophysiological features include (i) the infiltration and aberrantly enrichment of active immune cells (especially neutrophils and mononuclear cells) and platelets, leading to a hyperinflammatory state (generating vascular permeability and diffuse alveolar damage) and (ii) hypercoagulation, resulting in disseminated microvascular coagulation [11,46,89,91]. In COVID-19-associated ARDS, the cytokine storm consisting of high serum levels of IL-1β, TNF and IL-6 is possibly responsible by severe plasma leakage, vascular permeability and disseminated coagulation and thrombosis [9,57].

The expression of the metallopeptidase ACE2, the main entry receptor of SARS-CoV-2 in airway epithelial cells, is quite noticeable. In fact, expression of ACE2 can be detected in large and small bronchial epithelial cells, goblet/club cells, alveolar type II cells and pulmonary endothelial cells [31,90,92,93]. These observations apparently put the lungs as a main target system for SARS-CoV-2 infection.

According to human tests and animal models, the protective effects of ACE2 in lung diseases have been characterized in chronic obstructive pulmonary disease (COPD) [94] and asthma, [95,96]. In this context, the imbalance between ACE/ACE2 appears to favor inflammation and airway remodeling [49]. Therefore, a direct inhibition of ACE2 must be sought with caution due to the potential side effects that may arise from increased Ang-2 levels.

The immune resident and epithelial cells of the lung are the logical first cells to encounter SARS-CoV-2 and, as such, dictate the proliferation rate of the virus and the initiation of the inflammatory cascade. These cell types express molecules capable of recognizing pathogen-associated molecular patterns (PAMPs), such as TLR 3 and TLR 7/8, that can recognize viral nucleic acids and signal downstream to induce the expression of type I IFN genes [97,98]. TLR 3 and TLR7/8 overlay their activation through the adaptor proteins TRIF and MyD88, which lead the recruitment and nuclear import of IFN regulatory factors (IRFs) 3/7 [99,100]. IRFs 3/7 act as transcription factors to induce expression

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

![Figure](attachment:figure.png)

**Figure 4. Possible mechanisms that explain viral persistence and disease severity**

(1) Upon viral infection, cells increase the secretion of IL-6, which will paracrinally induce the expression of SOCS-1 via STAT-3 transcription factors [88]. (2) SOCS-1 hampers IFN antiviral signaling, via STAT-1 DE phosphorylation. The inhibition of expression of IFN genes leads to poor antiviral defense, and, as a consequence, reduced T-bet transcription, leading to defective Th1 polarization of CD4⁺ T cells [86,87,110,253]. (3) Upon viral entry, p38 is phosphorylated and, thereafter, perform downstream phosphorylation of target transcription factors, such as CREB, ATF-1 and AP-1. This effect leads to increased inflammatory gene expression and pro-survival gene expression, which may prolong the viral permanence in the infected cell [136]. (4) Upon recognition of viral particles and LPS from bacteria, TLR4-mediated signaling may lead to increased deleterious inflammation via NF-kB recruitment and p38 phosphorylation [105,106]. TLR4 polymorphism might explain the differential susceptibility to ARDS in COVID-19 patients [106].

---

of IFNs (subtypes α, β and γ), which are major inducers of antiviral response, thus paracrinally acting by receptors that recruit JAKs and STAT1/2 [99]. The recruitment of TRIF and MyD88 by TLR 3 and TLR 7/8 also activates TRAF6 and TAK1, which results in downstream activation of the IKK complex, further phosphorylation and degradation of IkB and, finally, enabling NF-kB nuclear translocation [101].

The great importance of NF-κB toward pro-inflammatory gene expression, particularly in the lungs, is highlighted in a number of studies exploring models of coronavirus infection [102,103]. In mice infected with SARS-CoV, the pharmacological inhibition of NF-kB can drive higher survival rates as well as reduced expression of TNF-α, CCL2 and CXCL2 in the lungs [104]. The NF-κB activation in Calu-3 human bronchial epithelial cells, in response to SARS infection, can mediate an antiviral gene response after 48 h post infection, as indicated by the up-regulation of IFN genes and pro-inflammatory cytokines and chemokines, including IL-6, IL-8 and TNF-α [103].

The relevance of TLRs to stimulate inflammation in the lungs has been well established [98,100]. During acute lung injury, the presence of TLR4 in macrophages acts as a key factor mediating the severity of the inflammation as well as tissue damage [105]. TLR4 polymorphisms have been also correlated to poor ARDS prognosis [106]. Many viruses, such as influenza and rhinoviruses, rely on strategies to silently surpass TLR-based warning signal in the airway epithelium, via inhibition of proteins such as RIG-1 and MDA5, components of TLR4 signaling and IRF-3, to drive IFN production [97]. Interestingly, TLR3 or TL4 knockout (KO) mice are more susceptible to SARS-CoV infection, while TRIF KO mice present greatly exacerbated inflammatory influx [107]. The lack of TRIF hampers IFN expression and leads to clinical features that largely resemble severe ARDS patients [107].

Even though TLRs are crucial for correct viral defense, the constitutive TLR signaling, particularly due to TLR4 [108], might contribute to an excessive inflammation in COVID-19-associated ARDS (Figure 4). Studies involving Ebola virus-like particles have shown that TLR4 also impairs protective antiviral cytokine production in fibroblasts

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).
via suppressor of cytokine signaling 3 (SOCS3), and could be involved in the disease initiation [109]. TLR4 polymorphism and dysregulation of associated pathway, could be potentially correlated with COVID-19-mediated suppression of IFN antiviral response in severe patients [110] and then contribute to excessive lung inflammation at later stages.

Lung epithelial cells, fibroblasts and alveolar macrophages may respond to SARS-CoV infection via NF-κB-mediated transcription of IL-6, IL-8 and TNF [111–113]. Cytokines that are commonly elevated in COVID-19 (such as TNF and IL-1β) can activate NF-κB-mediated gene expression in immune, bronchial epithelial and other lung cells [114]. Therefore, a selective inhibition of NF-κB signaling may serve as an alternate approach to halt excessive inflammation in the respiratory tract. The use of non-steroidal anti-inflammatory drugs or synthetic corticoids in COVID-19 treatment is still under debate, since NF-kB inhibition must be taken with caution, due to its vital role to elicit antiviral related gene expression [115]. Moreover, IL-6 may have pleiotropic effects since it can act differently in several aspects of lung-related diseases. Upon signaling via STAT3, IL-6 has been shown to initiate and potentiate the severity of inflammatory influx in the lungs in murine models of acute lung injury [116–118].

In a double-hit acute lung injury model, IL-6⁻/⁻ mice have displayed reduced inflammatory influx in the bronchoalveolar lavage but a higher protein concentration, suggesting that vascular leakage is potentially enhanced by IL-6 depletion. This effect could be attributed to the interaction of IL-6 with other molecules *in vivo*, since *in vitro* studies have indicated that IL-6 alone may increase permeability of endothelial cell cultures [119]. IL-6 levels are typically elevated in COPD and asthma, so it is thought that IL-6 may play a role in the evolution of these pathologies [120]. Upon further data validation, IL-6 and STAT3 could serve as promising therapeutic options for preventing the progressive severity of COVID-19-associated ARDS.

NOD-, LRR- and pyrin domain-containing protein 3 (NLRP3) is an intracellular protein that mainly serves as danger-associated molecular patterns (DAMPs) and PAMPs sensor, sensing the environment for molecules that could indicate peril and mechanistically assembling a complex with other proteins to form inflammasomes [121]. The NLRP3 inflammasome formation is comprised of two steps: 1) priming via NF-kB or IRFs to induce production of NLRP3 itself, as well as pro-IL-1β and pro-IL-18 and, 2) upon sensing danger-related molecules (for instance, ATP, reactive oxygen species or Ca²⁺), NLRP3 binds to ASC and caspase-1, thus cleaving IL-1β and IL-18 to their mature forms and contributing to the initiation of the inflammatory response [121].

The activation of NLRP3 inflammasome and the resulting production of pro-inflammatory molecules are key factors involved in the suppression of viral infections. NLRP3 can be assembled in response to PAMPs from viruses [122]. IL-1β can induce the production of IFNs and antiviral factors by activating IRF1/STAT1 signaling in fibroblasts and endothelial cells [123], thus representing an important helper for virus elimination from the lungs. *In vitro* studies have shown that the SARS-CoV open read frame (ORF) 8b is capable of activating NLPR3 inflammasomes. However, ORF8b may also promote cellular endoplasmic reticulum (ER) stress and induction of autophagy in epithelial cells as well as pyroptosis-mediated cell death in macrophages [124]. Additionally, SARS ORF3a, which is critical for SARS-CoV virulence, may also activate NLRP3 complex via ubiquitination of ASC, mediated by TRAF3 [125]. Of note, this virus contains a SARS-unique domain (SUD) which is essential for the events of cytokine storm and lung injury mediated by SARS [126]. SUD has been also detected in MERS-CoV and SARS-CoV-2. In SARS-CoV, it has been demonstrated that SUD can also trigger NLRP3 production and lung inflammation by inducing CXCL10 [127]. Altogether, a controlled approach to modulate NLRP3 levels and activity could be used as a potential target against ARDS in COVID-19 patients.

The deleterious role of MAPKs, especially due to the phosphorylation and activation of p38 and JNK, has been demonstrated in different models of lung injury [128–130], as well as in human idiopathic pulmonary fibrosis [131] and lung inflammation [132]. Indeed, ERK, JNK and p38 are involved in several aspects of COPD progression, including mucus overproduction, immune infiltration, fibrosis and airway inflammation and remodeling [133,134]. Thus, pharmacological targeting of MAPKs in ARDS, particularly p38, could also be a promising approach, since this kinase plays major roles in the up-regulation of various inflammatory-related molecules such as TNF, ICAM-1, COX2 and TLR4 [135].

*In vitro* studies, using VERO E6 cells, have also demonstrated a pivotal role of MAPKs in the regulation of SARS-CoV infection. In this context, p38 phosphorylation can potentially lead to downstream cascades and gene expression that modulate cell death but also the persistence of viral infection [136]. Down-regulation of STAT3 levels and recruitment of anti-apoptotic transcription factors (i.e. CREB and ATF-1) could induce cells to lower antiviral response and then allow viral replication [137]. Nevertheless, the pro-inflammatory effects of p38 in cells that compose the lung tissue cannot be ruled out.
Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

# GI system

As observed in SARS and MERS diseases, the GI tract may also be infected by SARS-CoV-2, thus leading to digestive symptoms such as diarrhea, loss of appetite, vomiting, nausea and abdominal pain [138–141]. The number of patients showing any of these symptoms vary, ranging up to 35% patients afflicted by diarrhea in different populations in China [140,142]. Nevertheless, GI manifestations seem to be less prevalent in COVID-19 than SARS and MERS [140].

Regardless of any GI symptoms [143], SARS-CoV-2 has been found in stool specimens and anal/rectal swab of infected patients [142,144–146]. Hence, the possibility of fecal–oral transmission has been largely discussed, with no definite conclusions so far [142,147]. Interestingly, stool samples have been tested positive for viral RNA even after its extinction in the respiratory tract [142], thus indicating that patients should be possibly tested for rectal swabs before discharge in order to retain any potential transmission of the disease [147]. This characteristic also raises attention to the poor sanitary conditions in which some populations may be living, which could facilitate the spread of the virus through contact with feces and/or virus dispersion in the air, albeit this topic is still very controversial [147–149].

Another point of discussion refers to the entrance of virus into the GI system. Some studies have suggested that SARS-CoV-2 may reach the small bowel by mucociliary clearance from the airways, which would allow the patient to ‘swallow’ the viruses [150,151]. On the other hand, the GI tract could act as the first entrance of SARS-CoV-2 in the human body, especially that many patients may present GI manifestations even before developing respiratory symptoms [140,151,152]. Moreover, it has been assumed that the consumption of animal food from a public market at Wuhan, was responsible by the appearance of the first case of COVID-19 in China [24]. In this case, the GI tract could be the first point of entrance for SARS-CoV-2 [34], although this hypothesis has not been proved yet.

The systemic inflammatory response promoted by SARS-CoV-2 invasion (i.e. cytokine storm) may directly or indirectly damage the digestive system and its resident microbiota, which could synergistically act with the viremia to promote respective symptoms [153]. Considering that the intestine is also the largest immune-related organ in the body, an inflammatory response elicited by coronaviruses is definitely relevant for this organ. Besides promoting cellular damage, SARS-CoV-2 may promote changes in the composition and function of the digestive flora which, at certain level, also impacts the respiratory tract by unbalancing the immune system in the airways mucosa. This relationship, recognized as the gut–lung axis, may help explain the progression and outcome of the digestive and respiratory symptoms in some COVID-19 patients [153,154].

The GI tract highly expresses the ACE2 receptor, which is fundamental for the internalization of the virus through the host cellular membrane. ACE2 is expressed in GI compartments where viral nucleocapsid proteins have also been located, which include the glandular cells of gastric, duodenal and rectal epithelia [142]. Furthermore, furin, a serine protease that acts similar to TMPRSS2 by unbounding S1 and S2 domains of the viral spike (S), is abundant in the small intestine enabling coronavirus infection [151]. In the luminal surface of intestinal epithelial cells, ACE2 is associated with the neutral amino acid transporter B⁰AT1, which is necessary for the amino acid traffic in the cell [155]. Since SARS-CoV-2 engages ACE2 as an entry viral receptor, it might also interact with B⁰AT1 and then affect the amino acid transportation in the gut [156].

Tryptophan is an essential amino acid required for the production of niacinamide, also known as vitamin B3 or niacin [102]. According to experiments *in vivo*, it has been shown that ACE2 deficiency may lead to a substantial decrease in tryptophan levels due to the inhibition of B⁰AT1 activity, which also results in an exacerbated inactivation of mammalian target of rapamycin (mTOR) pathway in the gut [156,157]. The mTOR is considered the main target of the PI3K-Akt pathway, acting through the formation of complexes mTORC1 and mTORC2. Usually, mTOR is involved in monitoring nutrient and energy availability in the cell, promoting cell growth, proliferation and survival as well as protein synthesis and transcription [158]. In the small intestine, mTOR pathway induces the antimicrobial peptide expression, which is essential to regulate a series of functions of the gut microbiota. An altered microbiota makes the intestine more prone to inflammation, eventually inducing cellular lesions, diarrhea and colitis [156]. The reduced expression of ACE2 and B⁰AT1 in the intestine due to virus invasion, leading to low tryptophan availability, represents a possible mechanism of how SARS-CoV-2 could lead to digestive symptoms in COVID-19 patients [151].

It has also been suggested that the co-infection of the GI tract and the CNS might be responsible for some of the digestive manifestations. Pathological events including changes in the microbiota content, gut inflammation, and alterations in the gut immune response (upon infection) could all trigger changes in the gut–brain axis and, therefore, lead to symptoms such as nausea, dizziness and anorexia by affecting brain areas that regulate these functions [159]. Furthermore, the peripheral lymphatic system present in the GI tract may communicate to the lymphatic system in the brain, thus enabling potential CNS infections [160,161]. Nevertheless, the mechanisms involved in most of these SARS-CoV2-mediated events are still speculative, so more detailed research is required to help better understanding the temporal and prognostic profiles of COVID-19.

# Nervous system

Patients tested positive for COVID-19 present clinical respiratory symptoms related to viral infection, which include fever, cough and myalgia [9]. Consistent with other coronaviruses (SARS-CoV and MERS-CoV), SARS-CoV-2 has also been associated to particular neurological manifestations [27,162]. Symptoms affecting both peripheral (PNS) and CNS, nervous systems have been reported in ~36% of COVID-19 patients, such as headache, acute cerebral diseases, impaired consciousness, seizure, smell/taste impairment, muscle injury and neuralgia [27,163,164].

Although mechanistic analyses are still in progress, current literature strongly suggests the potential ability of coronaviruses to reach brain-related tissues and cause neurological damage [165]. However, an interesting aspect that remains controversial is the presence of SARS-CoV-2 in the cerebrospinal fluid (CSF) of patients developing neurological symptoms, such as encephalitis/meningitis. Since some studies have reported the absence of virus in the CSF while others have fully detected it, some concerns have been raised about any direct neuroinvasive potential of the novel coronavirus, as also described for SARS-CoV and MERS-CoV [165–169]. Some authors have suggested that the lack of SARS-CoV-2 in the CSF of COVID-19 patients could be mainly related to (i) the low sensitivity of the techniques available, (ii) the reduced viral levels or (iii) clearance of SARS-CoV-2 in the CSF compartment [170]. Based on these current findings, further studies are still necessary to better elucidate this detection of SARS-CoV-2 in the CSF, especially in large cohorts of positive COVID-19 individuals.

According to the mechanisms of how other respiratory viruses as well as coronaviruses may affect the CNS, theories have been proposed to better explain how SARS-CoV-2 may reach the CNS and thus lead to brain damage. Baig et al. (2020) have described that SARS-CoV-2 may reach the CNS via bloodstream passing into the cerebral circulation, reaching the cerebral capillary endothelium which expresses ACE2 [31]. The interaction between spike (S) protein with the vascular ACE2 may lead to the release of virus particles by damaging the endothelial cells of the blood-brain barrier [171]. This effect promotes the viral entry and the consequent activation of ACE2 receptors also expressed in neurons, thus leading to local inflammation and demyelination [27,166,171]. Additionally, virus-driven demyelination in the CNS has been described in several viral infections, including coronaviruses, in humans and animal models [172–175]. Moreover, it has been also described that the hematogenous route provides the ability of respiratory viruses to contaminate leukocytes and then promote the viral dissemination to the brain, whereas some viruses may use peripheral nerves to get access to the CNS [171,176].

Other possibility that might explain the CNS contamination due to SARS-CoV-2 regards the neuronal retrograde route, which is associated with the virus penetration upon nasal infection (using cribiform plate and olfactory bulb as entry routes) [166,171,176]. Nasal infection may primarily lead to damage of the olfactory epithelium, which expresses both ACE2 and TRPMSS2 [166,171,177–179]. The consequent damage on the olfactory endothelium is part of the clinical symptoms presented by COVID-19 patients, particularly related to the PNS, such as anosmia or hyposmia [27,166].

The third route that could promote the entry of SARS-CoV-2 virus into the brain regarding the glymphatic system, a physiological route located in the CNS that shows perivascular tunnels consisting of astroglial cells, connected to the cervical and olfactory lymphatic vessels, that enable the waste elimination and promote the wide distribution of several compounds in the brain [10,160,176]. Moreover, brain damage may occur upon disturbance of the drainage system due to viral infection, thus leading to the entrance of viruses into the CSF [176,180]. Interestingly, some positive COVID-19 patients have been reported to present paranasal sinusitis with the presence of lymph endothelial cells infected by SARS-CoV-2 [181,182]. This particular cell infection appears to be associated with the potential ability to infect by coronaviruses.

Among the theories suggested for viral neuroinvasion (neuronal retrograde/hematogenous/glymphatic routes), the one regarding synapse-connected route is also related to the trans-synaptic exchange of coronavirus particles from peripheral nerves (innervating infected target-tissues), by which the interneuronal transfer of SARS-CoV-2 may occur, that in turn could reach and cause dysfunction of relevant homeostatic brainstem centers, as observed in COVID-19 [183,184].

Considering the presence of neurological manifestations (as described in previous studies), the indicative of brain injury in postmortem analysis [185] and the controversial aspects of SARS-CoV-2 detection in CSF, a recent study has presented neurochemical evidence of brain injury in severe positive COVID-19 patients [186]. By using two classical blood-based biomarkers for CNS injury, the glial fibrillary acidic protein (GFAP) and neurofilament light chain (NfL), it has been shown that moderate/severe COVID-19 patients present significantly higher plasma levels of both biomarkers [186]. Despite the limitations (including further data validation), these data suggest the existence of astrocytic activation/damage at the beginning of the disease (initial response/higher levels of GFAP), as well as
neuronal damage at later stages of a severe condition (illness progression/higher levels of NfL) in COVID-19 patients [186].

Nevertheless, despite the current advances in elucidating the mechanisms of SARS-CoV-2 invasion, the clinical symptoms regarding nervous system and their similarity to the other members of the β-coronaviruses, little is known about the nervous system signaling pathways affected in COVID-19. Based on the similarity with SARS-CoV, SARS-CoV-2 can also promote ACE2 down-regulation due to the damage caused upon virus entry into the host cells [35]. Some recent work has shown that decreased Angiotensin 1-7 levels (caused by loss activity of ACE2 in the conversion of Ang-II into Ang 1-7) necessary to stimulate Mas receptor to further regulate p38 MAPK signaling pathway, can promote increased tissue inflammation [69]. Since RAS has a relevant role in the regulation of brain homeostasis functions, such as modulation of water balance and blood pressure [187], the possible increase in Ang-II levels may be related to the development of severe cases presenting neurological manifestations (i.e. encephalitis and meningitis), as well as in cardiac and lung injuries in some severe COVID-19 patients [27,69,171]. Moreover, some additional evidence has indicated that the up-regulation of p38 MAPK may promote the virus cycle due to its relationship with endocytic mechanisms [69,187]. In addition, since some COVID-19 patients also present apoptosis of endothelial cells, these biological effects might be related to the elevated activity of MAPK signaling [69,180].

With regard to acute cerebral diseases, such as ischemic stroke and cerebral hemorrhage, which are detected in positive COVID-19 patients [27], ACE2 may also play an important role, since has been associated with damage along the brain endothelium (vascular injury) [171]. Moreover, COVID-19 patients with neurological manifestations may develop immunosuppression and elevated levels of D-dimer (a marker for venous thromboembolism) which are associated with a poor prognosis [27,188]. Interesting, patients with severe COVID-19 may also present a prominent increase in cytokine release (hypercytokinemia) [9,189] that, in the context of the nervous system, can potentially compromise its homeostasis [61]. This resulting cytokine environment has been previously reported in other systems affected by viral diseases. Additional studies will be necessary to validate this hypothesis and to further elucidate the putative mechanisms of disease progression in the context of the nervous system.

In addition to the neurological manifestations described for the CNS, the PNS also appears to be affected by COVID-19, since patients have presented smell and/or taste impairments, as well as complications due to the large amount of cytokines released systemically in the face of infection. Koralnik and Tyler [190] described that COVID-19 positive patients developed Guillain–Barre syndrome (GBS) after the onset of viral infection, as an immune mediated complication of SARS-CoV-2. In addition, electrophysiological studies performed in these GSB/COVID-19 patients have indicated a pattern compatible with demyelination process and axonal neuropathy, even without any positive detection of SARS-CoV-2 by RT-PCR in the CSF [191]. Despite the low cohort patients analyzed in this study, the authors highlight the necessity of better epidemiological data and further associations between immune diseases and COVID-19, which can promote a better understanding regarding possible pathogenic mechanisms and therapies [190].

## Cardiovascular system

ACE2 is highly expressed in the cardiovascular (CV) system tissues, possibly playing a major role in the regulation of the ACE2-Ang (1-7) signaling in proliferation, inflammation, vascular fibrosis and remodeling [44,192]. In healthy subjects, the levels of ACE2 in the plasma are very low, in contrast with the high levels found in the plasma of CV disease patients [193,194]. Thus, the CV system can be also affected by SARS-CoV-2 infection and, as such, it may potentially be a key for illness severity. In fact, patients with CV conditions have presented a case fatality rate of 10.5%, which is higher than the the overall COVID-19 cases (i.e. fatality rate of 2.3%) [195].

According to a clinical study that included 138 patients from Wuhan area, myocardial injury was identified in ~7.2% of patients hospitalized and in 22% of patients requiring intensive care [152]. Troponin, a major regulatory protein complex involved in muscle contraction, is typically released during myocardial damage, so the detection of troponin levels in the serum have served as a sensitive and specific test for the diagnosis of CV diseases. According to a report from the National Health Commission of China, ~12% of patients hospitalized due to COVID-19, without history of CV diseases, have presented elevated troponin levels and a high incidence of cardiac arrest during hospitalization, indicating that not only CV diseases could be a risk factor for COVID-19 but the presence of SARS-CoV-2 could also promote myocardial injury [196].

The impact of the CV diseases in severe COVID-19 patients has been clearly demonstrated by a recent study that compared non-surviving and surviving COVID-19 patients [180], in which 52% of the deceased patients presented heart failure, whereas only 12% of the survivors presented the same symptoms. Furthermore, 59% of the non-survival cases (versus only 1% of the survivors) were affected by cardiac injury. Another study has shown that out of 68 patients

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).who died from COVID-19, 13 had previous CV diseases while none of the 82 patients who survived presented a history of CV condition [197].

Despite the evidence, it is still unclear why cardiovascular diseases are so prevalent among the fatalities from COVID-19. One potential explanation relates to the ACE-mediated infection of cardiomyocytes, pericytes and fibroblasts, thus leading into myocardial injury (Hendren et al. 2020). Another hypothesis considers the impact of cytokine storm, triggered by an imbalanced response of T-helper cells and elevated levels of intracellular calcium, which can also promote extensive damage to myocardial cells [180,196].

According to the evaluation of 150 COVID-19 cases identified in Wuhan, the troponin levels in the serum of non-surviving patients were higher than in patients who recovered from the disease. Moreover, the levels of myoglobin, C-reactive protein, serum ferritin and IL-6 were also elevated in these cases, suggesting the presence of a high inflammatory process, such as CRS [197]. Furthermore, plaque rupture, ischemia or vasospasm have also been considered as potential causes of CV disease induced by a COVID-19 inflammatory response. Stress cardiomyopathy can also be triggered by a pro-inflammatory state, which is favored during SARS-Cov-2 infection [198]. In such cases, it is still challenging to infer whether the cytokine storm derived from the inflammatory processes may induce myocarditis-related cardiac events (as a response from SARS-CoV-2 infection) or if these pathological conditions are being exacerbated by a pre-existing CV disease.

Patients with CV disease appear to show elevated levels of certain cytokines, such as TNF-$\alpha$ and IL-6. An *in vitro* study, utilizing human coronary endothelial cells, has shown that the activation of Ang-II and ACE2 can decrease the induction of ICAM-1 by TNF-$\alpha$. Despite the fact that further research *in vivo* still needs to be assessed, this mechanism could be useful as a target toward the treatment of COVID-19 and CV disease patients, since it would potentially alleviate an inflammatory response [199].

Most of the regulatory events involved in CV systems, including the rate and force of myocardial contraction, cardiac hypertrophy and arterial resistance, are impacted by G-protein coupled receptors [200]. More specifically, Ang-II, endothelin-1 and adrenergic receptors are capable of coordinating the regulation of vascular tone, heart rate and contractility in cardiac myocytes, vascular smooth muscle cells (VSMCs) and endothelial cells. At the same time, these receptors are also responsible for some pathological changes, such as excessive cardiac hypertrophy, atherosclerosis and hypertension [201]. In cardiovascular cells, ERK1/2 pathway has been linked to the release of vasoactive molecules from the endothelium [202] as well as to the contraction of VSMC in resistance vessels [203]. In the case of endothelial cells, ERK1/2 phosphorylates an isoform of the effector molecule PLA2, which triggers a cascade that generates a range of prostaglandins, including prostacyclin (PGI2). PGI2 is a vasodilator molecule that inhibits VSMC proliferation and platelet reactivity [201]. Hence, the disruption of this pathway could lead to a subset of CV disorders, including thrombosis, hypertension and atherosclerosis.

Another significant pathway which plays a role in CV diseases involves the p38 MAPKs. In cardiac tissue, p38 is expressed is several isoforms possessing different functions [204]. For instance, the activation of p38$\alpha$ induces apoptosis in cardiac cells while the p38$\beta$ activation induces cardiomyocyte hypertrophy [205]. Also some CV alterations, such as ischemia and reperfusion, may distinctively impact this pathway, by activating p38$\alpha$ and inhibiting p38$\beta$, according to *in vitro* studies [206]. The JAK/STAT pathway has also an important role in cardiac myocytes. Evidence shows that the levels of phosphorylated JAK and STAT3 are associated with myocarditis and some types of cardiopathy [207].

There has been a growing concern that the use of ACE inhibitors and/or angiotensin receptor blockers by patients with CV-related pathologies could increase the expression of ACE2 and then elevate the patient susceptibility to SARS-CoV-2 infection and propagation [208]. However, no clear clinical or scientific evidence of this effect has been established. In fact, a number of arguments have been raised in regard to the use and cessation of these drugs during COVID-19 treatment in these patients. Thus, given the lack of evidence of their benefit (or harm) for therapeutic use in COVID-19, physicians should properly access patients' concerns and evaluate their medical history in other to keep or suspend the prescription for COVID-19 patients with CV conditions [198].

## Renal system

The kidneys have also been indicated as major targets of SARS-CoV-2 infection. Although an early study has not identified any cases of acute kidney injury (AKI) in a cohort of 116 COVID-19 patients from Wuhan area [209], clinical reports have largely supported the association of SARS-CoV-2 infection with kidney conditions. According to a large study including 701 COVID-19 patients, the most frequent finding related to kidney dysfunction was mild to moderate proteinuria (43.9%), possibly due to the disruption of glomerular filtration, while 26.7% of patients exhibited hematuria [210]. Interestingly, a retrospective study has shown that AKI was predominantly found in critically ill

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).patients [211]. Another report focusing on 113 non-surviving COVID-19 patients pointed out that AKI was highly associated with increased mortality [212]. Another study that reviewed records from 13 academic and community hospitals in metropolitan New York, found that AKI was reported in 36.6% of the admitted COVID-19 patients, particularly in patients with respiratory failure who required mechanical ventilation (89% of the cases) [213]. Advanced age and vascular dysfunction (CV disease and/or hypertension) were considered risk factors for AKI in the analyzed population [213]. Autopsy findings have demonstrated prominent acute proximal tubular injury, peritubular erythrocyte aggregation and glomerular fibrin thrombi in the kidneys of Chinese COVID-19 patients [214]. Interestingly, 66% of these patients lacked clinical evidence of AKI, highlighting the possibility of some subclinical kidney injury [214,215].

Several pathophysiological mechanisms have been proposed for the renal injuries observed in COVID-19, including organ-crosstalk and systemic-wide effects. The lung–kidney cross-talk linking alveolar and tubular damage has been previously described in ARDS, which may also be happening in the context of SARS-CoV-2 infection [216]. According to a retrospective study including 357 patients, AKI has been shown to be secondary to pneumonia in 68% of ARDS patients [216]. The damage in the lung–kidney axis may be bidirectional, as shown by the association of IL-6 cytokine released in the serum due to injured renal tubular epithelium with higher alveolar-capillary permeability and pulmonary hemorrhage [217]. Similarly, a heart–kidney cross-talk may also be considered as a contributor of AKI in COVID-19 patients, since cardiomyopathy and acute viral myocarditis can equally contribute to renal hypoperfusion, thus leading to a reduction in the glomerular filtration rate [218].

Systemic effects such as CRS have also been proposed for the etiology of AKI [218]. As previously mentioned, sustained elevation of pro-inflammatory cytokines, like IL-6, IL-1β and TNF-α, in the circulation can induce extensive endothelial dysfunction and disseminated intravascular coagulation, ultimately leading to multiple organ dysfunction syndrome (MODS) [219]. This condition can be directly responsible for renal damage. In fact, TNF-α has been demonstrated to bind directly to TNF receptor-1 in renal tubular cells, triggering apoptosis [220]. Moreover, IL-6 has been extensively reported to be associated with the onset and severity of AKI in patients and animal models, including ischemic AKI, nephrotoxin-induced AKI and sepsis-induced AKI, promoting renal injury via binding to sIL6R and downstream signaling though STAT3 in tubular epithelial cells [221]. IL-6 also induces microcirculation dysfunction and renal vascular permeability, while signaling for further cytokine secretion (IL-6, IL-8 and MCP-1) by renal endothelial cells via the Protein Kinase C pathway [222]. It is worth considering that in a study with 1099 COVID-19 patients, septic shock was present in 6.4% of severe cases [223], raising the possibility that, for a subset of patients, intrarenal inflammation may be partially responsible for the association of AKI to more severe cases of COVID-19 [224].

While systemic effects leading to elevated cytokine release may play a role in the kidney damage, direct SARS-CoV-2 infection has also been shown to be an important underlying cause of renal injury. Direct evidence of SARS-CoV-2 infection in the renal system has been provided by autopsy reports identifying virus particles and vacuoles characteristic of SARS-CoV-2 in the proximal tubular epithelium and podocytes, using electron microscopy [214,225]. These reports support a direct pathophysiological mechanism for the kidney damage due to COVID-19, following SARS-CoV-2 entry via ACE2 receptor [9]. A large transcriptomic study, using single-cell RNA sequencing in 15 normal kidney samples, has indicated a co-localization of ACE2 and TMPRSS genes in 19 individual cell types. The highest co-localization was observed in podocytes and proximal straight tubule cells, corroborating previous clinical and histological findings [226]. Moreover, direct infection of SARS-CoV-2 via ACE2 has been demonstrated in kidney-derived monkey cells (VERO) and kidney organoids derived from human embryonic stem cells [227]. Notably, these organoids exhibited distinct tubular-like structures [detected by Lotus Tetraglobus lectin (LTL), a marker of proximal tubular epithelial cells] as well as podocyte cells highly expressing ACE2. Importantly, direct ACE2 binding to SARS-CoV-2 has been demonstrated by blocking viral infection with human recombinant soluble ACE2 (hrsACE2), providing evidence that ACE2 blockers may act as a viable treatment during the early stages of the disease [227].

In the context of therapeutics, some research groups have postulated that extracorporeal therapies for cytokine removal from the blood may be a viable option to manage CRS and prevent kidney failure [218]. In fact, diverse number of methods (e.g. direct hemoperfusion using a neutro-macroporous, sorbent renal replacement therapy, coupled-plasma filtration adsorption) has focused on the extracorporeal blood filtration to remove the excess of pro-inflammatory cytokines, especially to avoid AKI in the critically ill patients [55,218].
Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904

Blocking viral interaction                                                                                       Blocking viral replication
ANG-I                                                                                                              Structural protein translation
ACE                                                                                                                Novel virion exocytosis
Angiotensin-II                                                                                                   
receptor blockers                                                                                                 
Tocilizumab                                                                                                      
SARS-CoV-2                                                                                                       
ACE inhibitors                                                                                                    
TMPRSS2                                                                                                          
ACE2                                                                                                             
RAS inhibitors                                                                                                    
Blocking specific                                                                                                 
serine proteases                                                                                                  
AT1R                                                                                                              
IL-6R                                                                                                             
JAK                                                                                                               
P                                                                                                                 
STAT 3                                                                                                            
P                                                                                                                 
IL-6                                                                                                              
Baricitinib                                                                                                       
Ribosome                                                                                                          
Host                                                                                                              
translation                                                                                                       
Nucleus                                                                                                           
Remdesivir /                                                                                                     
Favipiravir                                                                                                       
Viral RNA                                                                                                         
RdRp                                                                                                              
RNA replication                                                                                                   
Proteolysis                                                                                                       

Figure 5. Potential drug candidates for COVID-19 treatment

(1) Commercially available angiotensin-II receptor antagonists, such as losartan and derivatives, could act by mitigating the deleterious effects of AT1R activation in COVID-19 [229]. Activation of such signaling cascade results in RAS up-regulation and leads to pro-inflammatory and pro-fibrotic effects in infected tissues [43–45]. (2) Blockage of RAS can prevent tissue damage in COVID-19 patients [229]. (3) The use of ACE inhibitors may also contribute to decrease of the response of RAS system [229]. (4) Inhibitors of serine protease such as TMPRSS2 may prevent the cleavage of S1 and S2 domains in the viral spike (S) protein, decreasing the ability of SARS-CoV-2 to infect cells [28]. Interleukin-6 (IL-6) is one of the major cytokines involved in COVID-19 progression, which leads to CRS and tissue damage due to severe inflammation [51]. The use of commercially available molecules, such as the monoclonal antibody against the IL-6 receptor, (5) Tocilizumab [242] and (6) Baricitinib [246,247], an inhibitor of the JAK/STAT pathway, can halt inflammation and mitigate deleterious effects related to IL-6. (7) Upon viral entry and denudation, viral RNA is released and translated into immature precursor proteins by the host ribosome machinery. These precursor proteins are processed by viral proteases and then form the mature replication complex RNA-dependent RNA polymerase (RdRp), which enables viral RNA replication. Part of the expanded RNA is translated into structural proteins, such as envelope, spike and nucleocapsid that will form and release novel virions [232,233]. The specific RdRp mediated RNA replication can be inhibited by selective drugs, such as Remdesivir [235] and Favipiravir [239], rendering the virus unable to propagate and, consequently, halting the infection.

---

### Future perspectives regarding COVID-19 treatment

No effective treatment for COVID-19 has been discovered so far. However, some significant efforts have been pursued to find successful strategies that may control the spread of the pandemic. In this context, several clinical trials are currently in progress [228]. In regard to potential treatments for SARS-CoV-2 infection, interfering at some specific steps related the disease progression may be a common place for therapeutic intervention (Figure 5).

At first, an appropriate strategy could involve the blockage of the virus to interact with and invade host cells. Regulating molecules along the ACE2 pathway seems a promising approach, where RAS inhibitors have been proposed to prevent CRS as well as the release of pro-inflammatory cytokines [229]. Currently, most of the proposed drugs for COVID-19 treatment are (i) ACE inhibitors, (ii) blockers of Angiotensin II production, and (iii) blockers/antagonists of AT1R (Angiotensin II target) such as Losartan and derivatives [229]. Interestingly, a non-canonical axis of the renin-angiotensin pathway has been recently described, suggesting physiological roles for the metabolic products of angiotensin I and II (angiotensin 1-7 and angiotensin 1-9, respectively), after ACE2 cleavage [45]. This signaling axis also comprises ACE2, AT2R, the proto-oncogene Mas receptor and the Mas-related G protein-coupled receptor member D (MRGPRD). These molecules counteract the effects of the classical renin-angiotensin system, thus acting as putative therapeutic targets to regulate the increased activation of RAS in COVID-19 [229].

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

Another possibility relates to the development of decoy ACE2 receptors for virus attachment, thus abrogating putative tissue invasion. A recombinant form of human ACE2 has already been developed and underwent clinical trial, with promising results against ARDS [230]. Attempts to introduce point mutations in human ACE2 by Crispr-Cas9 system have also been pursued to weaken virus-ACE2 interaction, but related results still require peer-review [231]. Blocking the activity of specific serine proteases (for instance, TMPRSS and furin), which may promote the cleavage of S1 and S2 domains in the viral spike (S) protein, could serve as an alternate route to prevent virus invasion and also decrease NF-κB activation [28].

Remdesivir, a nucleotide analog that acts as RNA polymerase inhibitor, is capable of diminishing viral RNA synthesis [232,233], particularly against SARS-CoV-2, and *in vivo* efficacy in animal models against the MERS-CoV coronavirus. As a result, this drug has undergone clinical trials showing promising results [234–237]. According to one double-blind, randomized, placebo-controlled trial, which encompassed a total of 1063 mild COVID-19 patients, Remdesivir provided very positive effects by shortening the time to recovery [238]. Favipiravir is another example of potential drug that can also inhibit viral replication in COVID-19 [239,240].

Other therapeutic approaches have also proposed the management of CRS [241]. The humanized anti-IL-6 monoclonal antibody Tocilizumab has been empirically used in patients with severe COVID-19. Tocilizumab binds both mIL-6R (membrane bound receptor for IL-6) and sIL-6R (soluble receptor for IL-6), inhibiting both JAK-STAT and MAPK/NF-κB-IL-6 signaling pathways, which potentially prevent CRS-induced organ damage [242]. However, Tocilizumab has also been shown to induce liver injury in, at least, one COVID-19 patient [243].

JAK inhibition has also been considered as a potential therapeutic approach for COVID-19, since a variety of molecules (known to be elevated in COVID-19 patients) signal JAK/STAT pathways. Thus, JAK inhibitors have been readily available and clinically validated [244,245]. Baricitinib is an inhibitor of JAK pathway which is currently used for rheumatoid arthritis. Likewise, this drug is under clinical trial in COVID-19 since it may potentially reduce the cytokine storm in infected tissues, particularly the lungs [59,246,247]. Nevertheless, concerns have arisen due to potential side effects of such upstream inhibition, considering that JAK1 and JAK2 are activated by several molecules and result in multiple biological outcomes, which might prevent the immune system to fight against virus invasion [245,248].

Hence, a relative range of pharmacological approaches have been suggested [249]. Meanwhile, the development of safe and effective drugs and treatments, including vaccination, has been time-consuming while the pandemic has continuously progressed.

## Conclusions

COVID-19 pandemic has brought health and economic impact to the modern society as never before. Globally, people have suffered the consequences of the disease itself or from the prolonged social isolation established to contain a chaotic SARS-CoV-2 spread. Therefore, understanding how this novel coronavirus interacts with cells in the body as well as its effect in distinct body systems is mandatory to help humanity pursue new treatments. Increasing amount of data have been generated in a daily basis, bringing advances in understanding the mechanisms of viral invasion/replication, potential for virulence, tropism and relevance in several tissues affected by SARS-CoV-2. So far, it has been possible to identify a virus-mediated activation of some major signaling pathways, such as NF-κB, JAK-STAT and p38, which potentially elicit the cytokine storm event that appears to be the major cause of tissue injuries. Accordingly, the use of drugs capable of inhibiting viral replication or decreasing the CRS, such as Remdesivir and Tocilizumab respectively, seems to be promising approach. Nevertheless, although our comprehension of viral nature, infection and symptoms have rapidly evolved, the scientific community still has a lot to unravel from COVID-19 biology. Importantly, the rapidness to produce and analyze new data and treatments should not incur in precipitated or wrong conclusions, which could not only compromise the continuous advance of COVID-19 studies but also put the credibility of science *per se* in check.

## Competing Interests

The authors declare that there are no competing interest associated with the manuscript.

## Funding

This work was supported by the Fundação de Amparo à Pesquisa do Estado de São Paulo (FAPESP, São Paulo Research Foundation) [grant numbers 2017/17998-5 (to M.O.K.), 2017/16549-2 (to G.D.), 2019/05641-0 (to A.L.P.A.)]; the Brazilian Coordenação de Aperfeiçoamento de Pessoal de Nível Superior CAPES, Office for the Advancement of Higher Education [grant numbers PNPD/CAPES (to D.S.B.) and 1744709 (to A.L.P.A.)]; and the NIH (U.S.A.) [grant number NS107772 (to F.J.V.)].

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904

# Abbreviations

ACE, angiotensin-converting enzyme; ACE2, angiotensin-converting enzyme 2; AKI, acute kidney injury; Ang, Angiotensin; ARDS, acute respiratory distress syndrome; AT1R, Angiotensin II type 1 receptor; AT2R, Angiotensin II type 2 receptor; CNS, central nervous system; COPD, chronic obstructive pulmonary disease; COVID-19, coronavirus disease 19; CoV, coronavirus; CRS, cytokine release syndrome; ERK, extracellular signal-regulated kinase; GFAP, glial fibrillary acidic protein; GI, gastrointestinal; IFN, interferon; IRF, IFN regulatory factor; JAK/STAT, Janus kinase/signal transducer of activators of transcription; JNK, Jun amino-terminal kinase; MAPK, mitogen-activated protein kinase; MEKK, MAPK/ERK kinase kinase; MERS-CoV, Middle-East respiratory syndrome coronavirus; MLK3, mixed-lineage kinase 3; mTOR, mammalian target of rapamycin; NfL, neurofilament light chain; NF-κB, nuclear factor-κB; NLRP3, pyrin domain-containing protein 3; PAMP, pathogen-associated molecular pattern; PNS, peripheral nervous system; RAS, Renin-Angiotensin system; SAPK, stress-activated protein kinase; SARS, severe acute respiratory syndrome; SARS-CoV, severe acute respiratory syndrome coronavirus; SARS-CoV-2, severe acute respiratory syndrome coronavirus 2; SOCS3, suppressor of cytokine signaling 3; SUD, SARS-unique domain; TAK1, TGF (transforming growth factor) β-activated kinase 1; TLR, toll-like receptor; TMPRSS2, transmembrane serine protease 2; TNF-α, tumor necrosis factor α; VSMC, vascular smooth muscle cell.

# References

1. Tyrrell, D.A.J. and Bynoe, M.L. (1965) Cultivation of a novel type of common-cold virus in organ cultures. *Br. Med. J.* **1**, 1467, [https://doi.org/10.1136/bmj.1.5448.1467](https://doi.org/10.1136/bmj.1.5448.1467)
2. Almeida, J.D., Berry, D.M., Cunningham, C.H., Hamre, D., Hofstad, M.S., Mallucci, L. et al. (1968) Coronaviruses. *Nature* **220**, 2
3. Cui, J., Li, F. and Shi, Z.-L. (2019) Origin and evolution of pathogenic coronaviruses. *Nat. Rev. Microbiol.* **17**, 181–192, [https://doi.org/10.1038/s41579-018-0118-9](https://doi.org/10.1038/s41579-018-0118-9)
4. Kahn, J.S. and McIntosh, K. (2005) History and recent advances in coronavirus discovery. *Pediatr. Infect. Dis. J.* **24**, S223–S227, [https://doi.org/10.1097/01.inf.0000188166.17324.60](https://doi.org/10.1097/01.inf.0000188166.17324.60)
5. Chen, Y., Liu, Q. and Guo, D. (2020) Emerging coronaviruses: genome structure, replication, and pathogenesis. *J. Med. Virol.* **92**, 418–423, [https://doi.org/10.1002/jmv.25681](https://doi.org/10.1002/jmv.25681)
6. Ksiazek, T.G., Erdman, D., Goldsmith, C.S., Zaki, S.R., Peret, T., Emery, S. et al. (2003) A novel coronavirus associated with severe acute respiratory syndrome. *N. Engl. J. Med.* **348**, 1953–1966, [https://doi.org/10.1056/NEJMoa030781](https://doi.org/10.1056/NEJMoa030781)
7. Zaki, A.M., Van Boheemen, S., Bestebroer, T.M., Osterhaus, A.D.M.E. and Fouchier, R.A.M. (2012) Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. *N. Engl. J. Med.* **367**, 1814–1820, [https://doi.org/10.1056/NEJMo1211721](https://doi.org/10.1056/NEJMo1211721)
8. Corman, V.M., Muth, D., Niemeyer, D. and Drosten, C. (2018) Hosts and sources of endemic human coronaviruses. *Advances in Virus Research*, pp. 163–188, Elsevier
9. Huang, C., Wang, Y., Li, X., Ren, L., Zhao, J., Hu, Y. et al. (2020) Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. *Lancet* **395**, 497–506, [https://doi.org/10.1016/S0140-6736(20)30183-5](https://doi.org/10.1016/S0140-6736(20)30183-5)
10. Iroegbu, J.D., Ifenatuoha, C.W. and Ijomone, O.M. (2020) Potential neurological impact of coronaviruses: implications for the novel SARS-CoV-2. *Neurol. Sci.* **41**, 1329–1337, [https://doi.org/10.1007/s10072-020-04469-4](https://doi.org/10.1007/s10072-020-04469-4)
11. Siordia, J.A. (2020) Epidemiology and clinical features of COVID-19: a review of current literature. *J. Clin. Virol.* **127**, [https://doi.org/10.1016/j.jcv.2020.104357](https://doi.org/10.1016/j.jcv.2020.104357)
12. Zhu, N., Zhang, D., Wang, W., Li, X., Yang, B., Song, J. et al. (2020) A novel coronavirus from patients with pneumonia in China, 2019. *N. Engl. J. Med.* **382**, 727–733, [https://doi.org/10.1056/NEJMoa2001017](https://doi.org/10.1056/NEJMoa2001017)
13. Corman, V.M., Lienau, J. and Witzenrath, M. (2019) Coronaviruses as the cause of respiratory infections. *Internist (Berlin)* **60**, 1136–1145, [https://doi.org/10.1007/s00108-019-00671-5](https://doi.org/10.1007/s00108-019-00671-5)
14. Drosten, C., Günther, S., Preiser, W., Van Der Werf, S., Brodt, H.-R., Becker, S. et al. (2003) Identification of a novel coronavirus in patients with severe acute respiratory syndrome. *N. Engl. J. Med.* **348**, 1967–1976, [https://doi.org/10.1056/NEJMoa030747](https://doi.org/10.1056/NEJMoa030747)
15. de Groot, R.J., Baker, S.C., Baric, R.S., Brown, C.S., Drosten, C., Enjuanes, L. et al. (2013) Commentary: Middle East respiratory syndrome coronavirus (MERS-CoV): announcement of the Coronavirus Study Group. *J. Virol.* **87**, 7790–7792, [https://doi.org/10.1128/JVI.01244-13](https://doi.org/10.1128/JVI.01244-13)
16. Walls, A.C., Park, Y.-J., Tortorici, M.A., Wall, A., McGuire, A.T. and Veesler, D. (2020) Structure, function, and antigenicity of the SARS-CoV-2 spike glycoprotein. *Cell* **181**, 281–292.e6, [https://doi.org/10.1016/j.cell.2020.02.058](https://doi.org/10.1016/j.cell.2020.02.058)
17. Wu, F., Zhao, S., Yu, B., Chen, Y.-M., Wang, W., Song, Z.-G. et al. (2020) A new coronavirus associated with human respiratory disease in China. *Nature* **579**, 265–269, [https://doi.org/10.1038/s41586-020-0208-3](https://doi.org/10.1038/s41586-020-0208-3)
18. Guo, Y.R., Cao, Q.D., Hong, Z.S., Tan, Y.Y., Chen, S.D., Jin, H.J. et al. (2020) The origin, transmission and clinical therapies on coronavirus disease 2019 (COVID-19) outbreak - an update on the status. *Mil. Med. Res.* **7**, 11
19. Wang, C., Horby, P.W., Hayden, F.G. and Gao, G.F. (2020) A novel coronavirus outbreak of global health concern. *Lancet* **395**, 470–473, [https://doi.org/10.1016/S0140-6736(20)30185-9](https://doi.org/10.1016/S0140-6736(20)30185-9)
20. Zhou, P., Yang, X.-L., Wang, X.-G., Hu, B., Zhang, L., Zhang, W. et al. (2020) A pneumonia outbreak associated with a new coronavirus of probable bat origin. *Nature* **579**, 270–273, [https://doi.org/10.1038/s41586-020-2012-7](https://doi.org/10.1038/s41586-020-2012-7)
21. Cucinotta, D. and Vanelli, M. (2020) WHO declares COVID-19 a pandemic. *Acta Bio Medica Atenei Parm.* **91**, 157–160
22. Peng, Q.Y., Wang, X.T., Zhang, L.N. and Chinese Critical Care Ultrasound Study G (2020) Findings of lung ultrasonography of novel corona virus pneumonia during the 2019-2020 epidemic. *Intensive Care Med.* **46**, 849–850, [https://doi.org/10.1007/s00134-020-05996-6](https://doi.org/10.1007/s00134-020-05996-6)

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

23 Vetter, P., Vu, D.L., L’Huillier, A.G., Schibler, M., Kaiser, L. and Jacquerioz, F. (2020) Clinical features of covid-19. *BMJ* **369**, 1–2  
24 Chen, N., Zhou, M., Dong, X., Qu, J., Gong, F., Han, Y. et al. (2020) Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. *Lancet* **395**, 507–513, [https://doi.org/10.1016/S0140-6736(20)30211-7](https://doi.org/10.1016/S0140-6736(20)30211-7)  
25 Guan, W., Liang, W., Zhao, Y., Liang, H., Chen, Z., Li, Y. et al. (2020) Comorbidity and its impact on 1590 patients with COVID-19 in China: a nationwide analysis. *Eur. Respir. J.* **55**, 2000547, [https://doi.org/10.1183/13993003.00547-2020](https://doi.org/10.1183/13993003.00547-2020)  
26 Yang, J., Zheng, Y., Gou, X., Pu, K., Chen, Z., Guo, Q. et al. (2020) Prevalence of comorbidities and its effects in patients infected with SARS-CoV-2: a systematic review and meta-analysis. *Int. J. Infect. Dis.* **94**, 91–95, [https://doi.org/10.1016/j.ijid.2020.03.017](https://doi.org/10.1016/j.ijid.2020.03.017)  
27 Mao, L., Jin, H., Wang, M., Hu, Y., Chen, S., He, Q. et al. (2020) Neurologic manifestations of hospitalized patients with coronavirus disease 2019 in Wuhan, China. *JAMA Neurol.* **77**, 1–9, [https://doi.org/10.1001/jamaneurol.2020.1127](https://doi.org/10.1001/jamaneurol.2020.1127)  
28 Hoffmann, M., Kleine-Weber, H., Schroeder, S., Krüger, N., Herrler, T., Erichsen, S. et al. (2020) SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. *Cell* **181**, 271.e8–280.e8, [https://doi.org/10.1016/j.cell.2020.02.052](https://doi.org/10.1016/j.cell.2020.02.052)  
29 Lu, R., Zhao, X., Li, J., Niu, P., Yang, B., Wu, H. et al. (2020) Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. *Lancet* **395**, 565–574, [https://doi.org/10.1016/S0140-6736(20)30251-8](https://doi.org/10.1016/S0140-6736(20)30251-8)  
30 Harmer, D., Gilbert, M., Borman, R. and Clark, K.L. (2002) Quantitative mRNA expression profiling of ACE 2, a novel homologue of angiotensin converting enzyme. *FEBS Lett.* **532**, 107–110, [https://doi.org/10.1016/S0014-5793(02)03640-2](https://doi.org/10.1016/S0014-5793(02)03640-2)  
31 Hamming, I., Timens, W., Bulthuis, M.L.C., Lely, A.T., Navis, G.J. and van Goor, H. (2004) Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis. *J. Pathol.* **203**, 631–637  
32 Ding, Y., He, L., Zhang, Q., Huang, Z., Che, X., Hou, J. et al. (2004) Organ distribution of severe acute respiratory syndrome (SARS) associated coronavirus (SARS-CoV) in SARS patients: implications for pathogenesis virus transmission pathways. *J. Pathol.* **203**, 622–630, [https://doi.org/10.1002/path.1560](https://doi.org/10.1002/path.1560)  
33 Kuba, K., Imai, Y., Rao, S., Gao, H., Guo, F., Guan, B. et al. (2005) A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus-induced lung injury. *Nat. Med.* **11**, 875–879, [https://doi.org/10.1038/nm1267](https://doi.org/10.1038/nm1267)  
34 Zhang, H., Penninger, J.M., Li, Y., Zhong, N. and Slutsky, A.S. (2020) Angiotensin-converting enzyme 2 (ACE2) as a SARS-CoV-2 receptor: molecular mechanisms and potential therapeutic target. *Intensive Care Med.* **46**, 586–590, [https://doi.org/10.1007/s00134-020-05985-9](https://doi.org/10.1007/s00134-020-05985-9)  
35 Gheblawi, M., Wang, K., Viveiros, A., Nguyen, Q., Zhong, J.-C., Turner, A.J. et al. (2020) Angiotensin-converting enzyme 2: SARS-CoV-2 receptor and regulator of the renin-angiotensin system: celebrating the 20th anniversary of the discovery of ACE2. *Circ. Res.* **126**, 1456–1474, [https://doi.org/10.1161/CIRCRESAHA.120.317015](https://doi.org/10.1161/CIRCRESAHA.120.317015)  
36 Wrapp, D., Wang, N., Corbett, K.S., Goldsmith, J.A., Hsieh, C.-L., Abiona, O. et al. (2020) Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. *Science (80-)* **367**, 1260–1263, [https://doi.org/10.1126/science.abb2507](https://doi.org/10.1126/science.abb2507)  
37 Raj, V.S., Mou, H., Smits, S.L., Dekkers, D.H.W., Müller, M.A., Dijkman, R. et al. (2013) Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC. *Nature* **495**, 251–254, [https://doi.org/10.1038/nature12005](https://doi.org/10.1038/nature12005)  
38 Li, F. (2015) Receptor recognition mechanisms of coronaviruses: a decade of structural studies. *J. Virol.* **89**, 1954–1964, [https://jvi.asm.org/content/89/4/1954](https://jvi.asm.org/content/89/4/1954), [https://doi.org/10.1128/JVI.02615-14](https://doi.org/10.1128/JVI.02615-14)  
39 Peck, K.M., Scobey, T., Swanstrom, J., Jensen, K.L., Burch, C.L., Baric, R.S. et al. (2017) Permissivity of dipeptidyl peptidase 4 orthologs to Middle East respiratory syndrome coronavirus is governed by glycosylation and other complex determinants. *J. Virol.* **91**, e00534–17, [https://doi.org/10.1128/JVI.00534-17](https://doi.org/10.1128/JVI.00534-17)  
40 Qi, F., Qian, S., Zhang, S. and Zhang, Z. (2020) Single cell RNA sequencing of 13 human tissues identify cell types and receptors of human coronaviruses. *Biochem. Biophys. Res. Commun.* **526**, 135–140, [https://doi.org/10.1016/j.bbrc.2020.03.044](https://doi.org/10.1016/j.bbrc.2020.03.044)  
41 Wang, K., Chen, W., Zhou, Y.-S., Lian, J.-Q., Zhang, Z., Du, P. et al. (2020) SARS-CoV-2 invades host cells via a novel route: CD147-spike protein. *bioRxiv*, http://biorxiv.org/content/early/2020/03/14/2020.03.14.988345.abstract  
42 Coutard, B., Valle, C., de Lamballerie, X., Canard, B., Seidah, N.G. and Decroly, E. (2020) The spike glycoprotein of the new coronavirus 2019-nCoV contains a furin-like cleavage site absent in CoV of the same clade. *Antiviral Res.* **176**, 104742, [https://doi.org/10.1016/j.antiviral.2020.104742](https://doi.org/10.1016/j.antiviral.2020.104742)  
43 Cao, Z., Wu, Y., Falcon, E. and Sabatier, J.-M. (2020) SARS-CoV-2 & Covid-19: key-roles of the ‘renin-angiotensin’ system/vitamin D impacting drug and vaccine developments. *Infect. Disord. Drug Targets* **20**, 348–349, [https://doi.org/10.2174/1871526520999200505174704](https://doi.org/10.2174/1871526520999200505174704)  
44 Zhang, Z., Chen, L., Zhong, J., Gao, P. and Oudit, G.Y. (2014) ACE2/Ang-(1-7) signaling and vascular remodeling. *Sci. China Life Sci.* **57**, 802–808, [https://doi.org/10.1007/s11427-014-4693-3](https://doi.org/10.1007/s11427-014-4693-3)  
45 Paz Ocaranza, M., Riquelme, J.A., Garcia, L., Jalil, J.E., Chiong, M., Santos, R.A.S. et al. (2020) Counter-regulatory renin-angiotensin system in cardiovascular disease. *Nat. Rev. Cardiol.* **17**, 116–129, [https://doi.org/10.1038/s41569-019-0244-8](https://doi.org/10.1038/s41569-019-0244-8)  
46 Ziegler, C.G.K., Allon, S.J., Nyquist, S.K., Mbano, I.M., Miao, V.N., Tzouanas, C.N. et al. (2020) SARS-CoV-2 receptor ACE2 is an interferon-stimulated gene in human airway epithelial cells and is detected in specific cell subsets across tissues. *Cell* **181**, 1016–1035, [https://doi.org/10.1016/j.cell.2020.04.035](https://doi.org/10.1016/j.cell.2020.04.035)  
47 Marrero, M.B., Schieffer, B., Paxton, W.G., Heerdtt, L., Berkt, B.C., Delafontaine, P. et al. (2003) Direct stimulation of Jak/STAT pathway by the angiotensin II AT1 receptor. *J. Med. Chem.* **46**, 2261–2270  
48 Perrotta, F., Materia, M.G., Cazzola, M. and Bianco, A. (2020) Severe respiratory SARS-CoV2 infection: does ACE2 receptor matter? *Respir. Med.* **168**, 105996, [https://doi.org/10.1016/j.rmed.2020.105996](https://doi.org/10.1016/j.rmed.2020.105996)  
49 Cheng, H., Wang, Y. and Wang, G. (2020) Organ-protective effect of angiotensin-converting enzyme 2 and its effect on the prognosis of COVID-19. *J. Med. Virol.* **92**, 726–730, [https://doi.org/10.1002/jmv.25785](https://doi.org/10.1002/jmv.25785)  
50 De Lang, A., Osterhaus, A.D.M.E. and Haagmans, B.L. (2006) Interferon- γ and interleukin-4 downregulate expression of the SARS coronavirus receptor ACE2 in Vero E6 cells. *Virology* **353**, 474–481  

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).

51  McGonagle, D., Sharif, K., O’Regan, A. and Bridgewood, C. (2020) The role of cytokines including interleukin-6 in COVID-19 induced pneumonia and macrophage activation syndrome-like disease. *Autoimmun. Rev.* **19**, 102537, https://linkinghub.elsevier.com/retrieve/pii/S1568997220300926, [https://doi.org/10.1016/j.autrev.2020.102537](https://doi.org/10.1016/j.autrev.2020.102537)

52  Ye, Q., Wang, B. and Mao, J. (2020) The pathogenesis and treatment of the ‘Cytokine Storm’ in COVID-19. *J. Infect.* **80**, 607–613, https://linkinghub.elsevier.com/retrieve/pii/S0163445320301651, [https://doi.org/10.1016/j.jinf.2020.03.037](https://doi.org/10.1016/j.jinf.2020.03.037)

53  Behrens, E.M. and Koretzky, G.A. (2017) Review: cytokine storm syndrome: looking toward the precision medicine era. *Arthritis Rheumatol* **69**, 1135–1143, [https://doi.org/10.1002/art.40071](https://doi.org/10.1002/art.40071)

54  Guo, X.J. and Thomas, P.G. (2017) New fronts emerge in the influenza cytokine storm. *Semin. Immunopathol.* **39**, 541–550, [https://doi.org/10.1007/s00281-017-0636-y](https://doi.org/10.1007/s00281-017-0636-y)

55  Ronco, C. and Reis, T. (2020) Kidney involvement in COVID-19 and rationale for extracorporeal therapies. *Nat. Rev. Nephrol.* **16**, 308–310, [https://doi.org/10.1038/s41581-020-0284-7](https://doi.org/10.1038/s41581-020-0284-7)

56  Wu, C., Chen, X., Cai, Y., Xia, J., Zhou, X., Xu, S. et al. (2020) Risk factors associated with acute respiratory distress syndrome and death in patients with coronavirus disease 2019 pneumonia in Wuhan, China. *JAMA Intern. Med.* **180**, 1–11, [https://doi.org/10.1001/jamainternmed.2020.0994](https://doi.org/10.1001/jamainternmed.2020.0994)

57  Jose, R.J. and Manuel, A. (2020) COVID-19 cytokine storm: the interplay between inflammation and coagulation. *Lancet Respir. Med.* **2019**, 2019–2020

58  Murakami, M., Kamimura, D. and Hirano, T. (2019) Pleiotropy and specificity: insights from the interleukin 6 family of cytokines. *Immunity* **50**, 812–831, [https://doi.org/10.1016/j.immuni.2019.03.027](https://doi.org/10.1016/j.immuni.2019.03.027)

59  O’Shea, J.J., Schwartz, D.M., Villarino, A.V., Gadina, M., McInnes, I.B. and Laurence, A. (2015) The JAK-STAT pathway: impact on human disease and therapeutic intervention. *Annu. Rev. Med.* **66**, 311–328, [https://doi.org/10.1146/annurev-med-051113-024537](https://doi.org/10.1146/annurev-med-051113-024537)

60  Lawrence, T. (2009) The nuclear factor NF- k B pathway in inflammation. *Cold Spring Harb. Perspect. Biol.* **110**, 637–641

61  Catanzaro, M., Fagiani, F., Racchi, M., Corsini, E., Govoni, S. and Lanni, C. (2020) Immune response in COVID-19: addressing a pharmacological challenge by targeting pathways triggered by SARS-CoV-2. *Signal Transduct. Target Ther.* **5**, 84, [https://doi.org/10.1038/s41392-020-0191-1](https://doi.org/10.1038/s41392-020-0191-1)

62  Li, Q. and Verma, I.M. (2002) NF-κB regulation in the immune system. *Nat. Rev. Immunol.* **2**, 725–734, [https://doi.org/10.1038/nri910](https://doi.org/10.1038/nri910)

63  Tergaonkar, V., Correa, R.G., Ikawa, M. and Verma, I.M. (2005) Distinct roles of IkB proteins in regulating constitutive NF-κB activity. *Nat. Cell Biol.* **7**, 921–923, [https://doi.org/10.1038/ncb1296](https://doi.org/10.1038/ncb1296)

64  Tergaonkar, V. (2006) NFκB pathway: a good signaling paradigm and therapeutic target. *Int. J. Biochem. Cell Biol.* **38**, 1647–1653, https://linkinghub.elsevier.com/retrieve/pii/S1357272506001245, [https://doi.org/10.1016/j.biocel.2006.03.023](https://doi.org/10.1016/j.biocel.2006.03.023)

65  Correa, R.G., Matsui, T., Tergaonkar, V., Rodriguez-Esteban, C., Izpisua-Belmonte, J.C. and Verma, I.M. (2005) Zebrafish IkappaB kinase 1 negatively regulates NF-kappaB activity. *Curr. Biol.* **15**, 1291–1295, [https://doi.org/10.1016/j.cub.2005.06.023](https://doi.org/10.1016/j.cub.2005.06.023)

66  Wong, E.T. and Tergaonkar, V. (2009) Roles of NF-κB in health and disease: mechanisms and therapeutic potential. *Clin. Sci.* **116**, 451–465, https://portlandpress.com/clinsci/article/116/6/451/68639/Roles-of-NFκB-in-health-and-disease-mechanisms-and, [https://doi.org/10.1042/CS20080502](https://doi.org/10.1042/CS20080502)

67  Morrison, D.K. (2012) MAP kinase pathways. *Cold Spring Harb. Perspect. Biol.* **4**, a011254, [https://doi.org/10.1101/cshperspect.a011254](https://doi.org/10.1101/cshperspect.a011254)

68  Mizutani, T., Fukushi, S., Saijo, M., Kurane, I. and Morikawa, S. (2004) Phosphorylation of p38 MAPK and its downstream targets in SARS coronavirus-infected cells. *Biochem. Biophys. Res. Commun.* **319**, 1228–1234, [https://doi.org/10.1016/j.bbrc.2004.05.107](https://doi.org/10.1016/j.bbrc.2004.05.107)

69  Grimes, J.M. and Grimes, K.V. (2020) p38 MAPK inhibition: a promising therapeutic approach for COVID-19. *J. Mol. Cell Cardiol.* **144**, 63–65, [https://doi.org/10.1016/j.yjmcc.2020.05.007](https://doi.org/10.1016/j.yjmcc.2020.05.007)

70  Cuadrado, A. and Nebreda, A.R. (2010) Mechanisms and functions of p38 MAPK signalling. *Biochem. J.* **429**, 403–417, [https://doi.org/10.1042/BJ20100323](https://doi.org/10.1042/BJ20100323)

71  Han, J., Lee, J.D., Tobias, P.S. and Ulevitch, R.J. (1993) Endotoxin induces rapid protein tyrosine phosphorylation in 70Z/3 cells expressing CD14. *J. Biol. Chem.* **268**, 25009–25014

72  Jiang, Y., Chen, C., Li, Z., Guo, W., Gegner, J.A., Lin, S. et al. (1996) Characterization of the structure and function of a new mitogen-activated protein kinase (p38β). *J. Biol. Chem.* **271**, 17920–17926, [https://doi.org/10.1074/jbc.271.30.17920](https://doi.org/10.1074/jbc.271.30.17920)

73  Li, Z., Jiang, Y., Ulevitch, R.J. and Han, J. (1996) The primary structure of p38γ: a new member of p38 group of MAP kinases. *Biochem. Biophys. Res. Commun.* **228**, 334–340, https://linkinghub.elsevier.com/retrieve/pii/S0006291X96916629, [https://doi.org/10.1006/bbrc.1996.1662](https://doi.org/10.1006/bbrc.1996.1662)

74  Jiang, Y., Gram, H., Zhao, M., New, L., Gu, J., Feng, L. et al. (1997) Characterization of the structure and function of the fourth member of p38 group mitogen-activated protein kinases, p38δ;. *J. Biol. Chem.* **272**, 30122–30128, [https://doi.org/10.1074/jbc.272.48.30122](https://doi.org/10.1074/jbc.272.48.30122)

75  Zarubin, T. and Han, J. (2005) Activation and signaling of the p38 MAP kinase pathway. *Cell Res.* **15**, 11–18, [https://doi.org/10.1038/sj.cr.7290257](https://doi.org/10.1038/sj.cr.7290257)

76  Cuevas, B.D., Abell, A.N. and Johnson, G.L. (2007) Role of mitogen-activated protein kinase kinases in signal integration. *Oncogene* **26**, 3159–3171, http://www.nature.com/articles/1210409, [https://doi.org/10.1038/sj.onc.1210409](https://doi.org/10.1038/sj.onc.1210409)

77  Alonso, G., Ambrosino, C., Jones, M. and Nebreda, A.R. (2000) Differential activation of p38 mitogen-activated protein kinase isoforms depending on signal strength. *J. Biol. Chem.* **275**, 40641–40648, [https://doi.org/10.1074/jbc.M007835200](https://doi.org/10.1074/jbc.M007835200)

78  Branco, D. (2003) Mechanism of p38 MAP kinase activation in vivo. *Genes Dev.* **17**, 1969–1978, [https://doi.org/10.1101/gad.1107303](https://doi.org/10.1101/gad.1107303)

79  Parker, C.G., Hunt, J., Diener, K., McGinley, M., Soriano, B., Keesler, G.A. et al. (1998) Identification of Stathmin as a novel substrate for p38 Delta. *Biochem. Biophys. Res. Commun.* **249**, 791–796, https://linkinghub.elsevier.com/retrieve/pii/S0006291X98992506, [https://doi.org/10.1006/bbrc.1998.9250](https://doi.org/10.1006/bbrc.1998.9250)

80  Rouse, J., Cohen, P., Trigon, S., Morange, M., Alonso-Llamazares, A., Zamanillo, D. et al. (1994) A novel kinase cascade triggered by stress and heat shock that stimulates MAPKAP kinase-2 and phosphorylation of the small heat shock proteins. *Cell* **78**, 1027–1037, https://linkinghub.elsevier.com/retrieve/pii/0092867494902771, [https://doi.org/10.1016/0092-8674(94)90277-1](https://doi.org/10.1016/0092-8674(94)90277-1)

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

81 Tan, Y., Rouse, J., Zhang, A., Cariati, S., Cohen, P. and Comb, M.J. (1996) FGF and stress regulate CREB and ATF-1 via a pathway involving p38 MAP kinase and MAPKAP kinase-2. *EMBO J.* **15**, 4629–4642, [https://doi.org/10.1002/j.1460-2075.1996.tb00840.x](https://doi.org/10.1002/j.1460-2075.1996.tb00840.x)  
82 Xu, W., Wang, X., Tocker, A.M., Huang, P., Reith, M.E.A., Liu-Chen, L.Y. et al. (2017) Functional characterization of a novel series of biased signaling dopamine D3 receptor agonists. *ACS Chem. Neurosci.* **8**, 486–500, [https://doi.org/10.1021/acschemneuro.6b00221](https://doi.org/10.1021/acschemneuro.6b00221)  
83 Kotlyarov, A., Neininge, A., Schubert, C., Eckert, R., Birchmeier, C., Volk, H.-D. et al. (1999) MAPKAP kinase 2 is essential for LPS-induced TNF-$\alpha$ biosynthesis. *Nat. Cell Biol.* **1**, 94–97, http://www.nature.com/articles/ncb0699_94, [https://doi.org/10.1038/10061](https://doi.org/10.1038/10061)  
84 Kontoyiannis, D., Pasparakis, M., Pizarro, T.T., Cominelli, F. and Kollias, G. (1999) Impaired on/off regulation of TNF biosynthesis in mice lacking TNF AU-rich elements. *Immunity* **10**, 387–398, https://linkinghub.elsevier.com/retrieve/pii/S1074761300800382, [https://doi.org/10.1016/S1074-7613(00)80038-2](https://doi.org/10.1016/S1074-7613(00)80038-2)  
85 Saccani, S., Pantano, S. and Natoli, G. (2002) p38-dependent marking of inflammatory genes for increased NF-$\kappa$B recruitment. *Nat. Immunol.* **3**, 69–75, [https://doi.org/10.1038/ni748](https://doi.org/10.1038/ni748)  
86 Dienz, O. and Rincon, M. (2009) The effects of IL-6 on CD4 T cell responses. *Clin. Immunol.* **130**, 27–33, [https://doi.org/10.1016/j.clim.2008.08.018](https://doi.org/10.1016/j.clim.2008.08.018)  
87 Velazquez-Salinas, L., Verdugo-Rodriguez, A. and Rodriguez, L.L. (2019) The role of interleukin 6 during viral infections. *Front Microbiol.* **10**, 6–11  
88 Zhang, C., Wu, Z., Li, J.W., Zhao, H. and Wang, G.Q. (2020) The cytokine release syndrome (CRS) of severe COVID-19 and Interleukin-6 receptor (IL-6R) antagonist Tocilizumab may be the key to reduce the mortality. *Int. J. Antimicrob. Agents* **55**, [https://doi.org/10.1016/j.jantimicag.2020.105954](https://doi.org/10.1016/j.jantimicag.2020.105954)  
89 Matthay, M.A., Ware, L.B. and Zimmerman, G.A. (2012) The acute respiratory distress syndrome. *J. Clin. Invest.* **122**, 2731–2740, [https://doi.org/10.1172/JCI60331](https://doi.org/10.1172/JCI60331)  
90 Zhang, H., Rostami, M.R., Leopold, P.L., Mezey, J.G., O’Beirne, S.L., Strulovici-Barel, Y. et al. (2020) Expression of the SARS-CoV-2 ACE2 receptor in the human airway epithelium. *Am. J. Respir. Crit. Care Med.* **202**, 219–229, [https://doi.org/10.1164/rccm.202003-05410C](https://doi.org/10.1164/rccm.202003-05410C)  
91 Tian, S., Xiong, Y., Liu, H., Niu, L., Guo, J., Liao, M. et al. (2020) Pathological study of the 2019 novel coronavirus disease (COVID-19) through postmortem core biopsies. *Mod. Pathol.* **33**, 10–12, [https://doi.org/10.1038/s41379-020-0536-x](https://doi.org/10.1038/s41379-020-0536-x)  
92 Sungnak, W., Huang, N., Bécavin, C., Berg, M., Queen, R., Litvinukova, M. et al. (2020) SARS-CoV-2 entry factors are highly expressed in nasal epithelial cells together with innate immune genes. *Nat. Med.* **26**, 681–687, [https://doi.org/10.1038/s41591-020-0868-6](https://doi.org/10.1038/s41591-020-0868-6)  
93 Lukassen, S., Chua, R.L., Trefzer, T., Kahn, N.C., Schneider, M.A., Muley, T. et al. (2020) SARS -CoV-2 receptor ACE 2 and TMPRSS 2 are primarily expressed in bronchial transient secretory cells. *EMBO J.* **39**, 1–15, [https://doi.org/10.15252/embj.20105114](https://doi.org/10.15252/embj.20105114)  
94 Leung, J.M., Yang, C.X., Tam, A., Shaipanich, T., Hackett, T.-L., Singhera, G.K. et al. (2020) ACE-2 expression in the small airway epithelia of smokers and COPD patients: implications for COVID-19. *Eur. Respir. J.* **55**, 2000688, [https://doi.org/10.1183/13993003.00688-2020](https://doi.org/10.1183/13993003.00688-2020)  
95 Dhawale, V.S., Amara, V.R., Karpe, P.A., Malek, V., Patel, D. and Tikoo, K. (2016) Activation of angiotensin-converting enzyme 2 (ACE2) attenuates allergic airway inflammation in rat asthma model. *Toxicol. Appl. Pharmacol.* **306**, 17–26, [https://doi.org/10.1016/j.taap.2016.06.026](https://doi.org/10.1016/j.taap.2016.06.026)  
96 Bradding, P., Richardson, M., Hinks, T.S.C., Howarth, P.H., Choy, D.F., Arron, J.R. et al. (2020) Ace2, Tmprss2 and Furin gene expression in the airways of people with asthma - implications for Covid-19. *J. Allergy Clin. Immunol.* **146**, 10–12, [https://doi.org/10.1016/j.jaci.2020.05.013](https://doi.org/10.1016/j.jaci.2020.05.013)  
97 Vareille, M., Kieninger, E., Edwards, M.R. and Regamey, N. (2011) The airway epithelium: soldier in the fight against respiratory viruses. *Clin. Microbiol. Rev.* **24**, 210–229, [https://doi.org/10.1128/CMR.00014-10](https://doi.org/10.1128/CMR.00014-10)  
98 Sabroe, I., Parker, L.C., Dower, S.K. and Whyte, M.K.B. (2008) The role of TLR activation in inflammation. *J. Pathol.* **214**, 126–135, [https://doi.org/10.1002/path.2264](https://doi.org/10.1002/path.2264)  
99 Frieman, M. and Baric, R. (2008) Mechanisms of severe acute respiratory syndrome pathogenesis and innate immunomodulation. *Microbiol. Mol. Biol. Rev.* **72**, 672–685, [https://doi.org/10.1128/MMBR.00015-08](https://doi.org/10.1128/MMBR.00015-08)  
100 Lafferty, E.I., Qureshi, S.T. and Schnare, M. (2010) The role of toll-like receptors in acute and chronic lung inflammation. *J. Inflamm.* **7**, 57, http://www.journal-inflammation.com/content/7/1/57, [https://doi.org/10.1186/1476-9255-7-57](https://doi.org/10.1186/1476-9255-7-57)  
101 Kawai, T. and Akira, S. (2007) Signaling to NF-$\kappa$B by Toll-like receptors. *Trends Mol. Med.* **13**, 460–469, [https://doi.org/10.1016/j.molmed.2007.09.002](https://doi.org/10.1016/j.molmed.2007.09.002)  
102 Darby, W.J., McNutt, K.W. and Todhunter, E.N. (2009) Niacin. *Nutr. Rev.* **33**, 289–297, [https://doi.org/10.1111/j.1753-4887.1975.tb05075.x](https://doi.org/10.1111/j.1753-4887.1975.tb05075.x)  
103 Yoshikawa, T., Hill, T.E., Yoshikawa, N., Popov, V.L., Galindo, C.L., Garner, H.R. et al. (2010) Dynamic innate immune responses of human bronchial epithelial cells to severe acute respiratory syndrome-associated coronavirus infection. *PLoS ONE* **5**, e8729, [https://doi.org/10.1371/journal.pone.0008729](https://doi.org/10.1371/journal.pone.0008729)  
104 DeDiego, M.L., Nieto-Torres, J.L., Regla-Nava, J.A., Jimenez-Guardeno, J.M., Fernandez-Delgado, R., Fett, C. et al. (2014) Inhibition of NF-kappaB-mediated inflammation in severe acute respiratory syndrome coronavirus-infected mice increases survival. *J. Virol.* **88**, 913–924, [https://doi.org/10.1128/JVI.02576-13](https://doi.org/10.1128/JVI.02576-13)  
105 Imai, Y., Kuba, K., Neely, G.G., Yaghoubian-Malhami, R., Perkmann, T., van Loo, G. et al. (2008) Identification of oxidative stress and toll-like receptor 4 signaling as a key pathway of acute lung injury. *Cell* **133**, 235–249, [https://doi.org/10.1016/j.cell.2008.02.043](https://doi.org/10.1016/j.cell.2008.02.043)  
106 Ding, Y., Feng, Q., Chen, J. and Song, J. (2019) TLR4/NF-$\kappa$B signaling pathway gene single nucleotide polymorphisms alter gene expression levels and affect ARDS occurrence and prognosis outcomes. *Medicine (Baltimore)* **98**, e16029, [https://doi.org/10.1097/MD.0000000000016029](https://doi.org/10.1097/MD.0000000000016029)  
107 Totura, A.L., Whitmore, A., Agnihotram, S., Schäfer, A., Katze, M.G., Heise, M.T. et al. (2015) Toll-like receptor 3 signaling via TRIF contributes to a protective innate immune response to severe acute respiratory syndrome coronavirus infection. *mBio* **6**, 1–14, [https://doi.org/10.1128/mBio.00638-15](https://doi.org/10.1128/mBio.00638-15)  
108 Olejnik, J., Hume, A.J. and Mühlberger, E. (2018) Toll-like receptor 4 in acute viral infection: too much of a good thing. *PLoS Pathog.* **14**, 1–7, [https://doi.org/10.1371/journal.ppat.1007390](https://doi.org/10.1371/journal.ppat.1007390)  
109 Okumura, A., Rasmussen, A.L., Halfmann, P., Feldmann, F., Yoshimura, A., Feldmann, H. et al. (2015) Suppressor of cytokine signaling 3 is an inducible host factor that regulates virus egress during Ebola virus infection. *J. Virol.* **89**, 10399–10406, [https://doi.org/10.1128/JVI.01736-15](https://doi.org/10.1128/JVI.01736-15)  

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904

110 O’Brien, T.R., Thomas, D.L., Jackson, S.S., Prokunina-Olsson, L., Donnelly, R.P. and Hartmann, R. (2020) Weak induction of interferon expression by SARS-CoV-2 supports clinical. *Clin Infect Dis.*, [https://doi.org/10.1093/cid/ciaa453](https://doi.org/10.1093/cid/ciaa453)

111 Chang, Y.-J., Liu, C.Y.-Y., Chiang, B.-L., Chao, Y.-C. and Chen, C.-C. (2004) Induction of IL-8 release in lung cells via activator protein-1 by recombinant baculovirus displaying severe acute respiratory syndrome-coronavirus spike proteins: identification of two functional regions. *J. Immunol.* **173**, 7602–7614, [https://doi.org/10.4049/jimmunol.173.12.7602](https://doi.org/10.4049/jimmunol.173.12.7602)

112 Wang, W., Ye, L., Ye, L., Li, B., Gao, B., Zeng, Y. et al. (2007) Up-regulation of IL-6 and TNF-α induced by SARS-coronavirus spike protein in murine macrophages via NF-κB pathway. *Virus Res.* **128**, 1–8, [https://doi.org/10.1016/j.virusres.2007.02.007](https://doi.org/10.1016/j.virusres.2007.02.007)

113 Zhang, X., Wu, K., Wang, D., Yue, X., Song, D., Zhu, Y. et al. (2007) Nucleocapsid protein of SARS-CoV activates interleukin-6 expression through cellular transcription factor NF-κB. *Virol.* **365**, 324–335, [https://doi.org/10.1016/j.virol.2007.04.009](https://doi.org/10.1016/j.virol.2007.04.009)

114 Toews, G.B. (2001) Cytokines and the lung. *Eur. Respir. J. Suppl.* **18**, 3–17, [https://doi.org/10.1183/09031936.01.00266001](https://doi.org/10.1183/09031936.01.00266001)

115 Russell, B., Moss, C., Rigg, A. and Van Hemelrijck, M. (2020) COVID-19 and treatment with NSAIDs and corticosteroids: Should we be limiting their use in the clinical setting? *Ecancermedicalscience* **14**, 1–3, [https://doi.org/10.3332/ecancer.2020.1023](https://doi.org/10.3332/ecancer.2020.1023)

116 Severgnini, M., Takahashi, S., Rozo, L.M., Homer, R.J., Kuhn, C., Jhung, J.W. et al. (2004) Activation of the STAT pathway in acute lung injury. *Am. J. Physiol. Lung Cell Mol. Physiol.* **286**, 1282–1292, [https://doi.org/10.1152/ajplung.00349.2003](https://doi.org/10.1152/ajplung.00349.2003)

117 Zhao, J., Yu, H., Liu, Y., Gibson, S.A., Yan, Z., Xu, X. et al. (2016) Protective effect of suppressing STAT3 activity in LPS-induced acute lung injury. *Am. J. Physiol. Lung Cell Mol. Physiol.* **311**, L868–L880, [https://doi.org/10.1152/ajplung.00281.2016](https://doi.org/10.1152/ajplung.00281.2016)

118 Pedroza, M., Schneider, D.J., Karmouty-Quintana, H., Coote, J., Shaw, S., Corrigan, R. et al. (2011) Interleukin-6 contributes to inflammation and remodeling in a model of adenosine mediated lung injury. *PLoS ONE* **6**, e22667, [https://doi.org/10.1371/journal.pone.0022667](https://doi.org/10.1371/journal.pone.0022667)

119 Goldman, J.L., Sammani, S., Kempf, C., Saadat, L., Letsiou, E., Wang, T. et al. (2014) Pleiotropic effects of interleukin-6 in a “two-hit” murine model of acute respiratory distress syndrome. *Pulm. Circ.* **4**, 280–288, [https://doi.org/10.1086/675991](https://doi.org/10.1086/675991)

120 Rincon, M. and Irvin, C.G. (2012) Role of IL-6 in asthma and other inflammatory pulmonary diseases. *Int. J. Biol. Sci.* **8**, 1281–1290, [https://doi.org/10.7150/ijbs.4874](https://doi.org/10.7150/ijbs.4874)

121 Swanson, K.V., Deng, M. and Ting, J.P.Y. (2019) The NLRP3 inflammasome: molecular activation and regulation to therapeutics. *Nat. Rev. Immunol.* **19**, 477–489, [https://doi.org/10.1038/s41577-019-0165-0](https://doi.org/10.1038/s41577-019-0165-0)

122 Zhao, C. and Zhao, W. (2020) NLRP3 inflammasome — a key player in antiviral responses. *Front Immunol.* **11**, 1–8

123 Orzalli, M.H., Smith, A., Jurado, K.A., Iwasaki, A., Garlick, J.A., Kagan, J.C. et al. (2018) An antiviral branch of the IL-1 signaling pathway restricts immune-evasive virus replication article an antiviral branch of the IL-1 signaling pathway restricts immune-evasive virus replication. *Mol. Cell* **71**, 825.e5–840.e5, [https://doi.org/10.1016/j.molcel.2018.07.009](https://doi.org/10.1016/j.molcel.2018.07.009)

124 Shi, C. and Nabar, N.R. (2019) SARS-Coronavirus open reading frame-8b triggers intracellular stress pathways and activates NLRP3 inflammasomes. *Cell Death Discov.* **5**, 101, [https://doi.org/10.1038/s41420-019-0181-7](https://doi.org/10.1038/s41420-019-0181-7)

125 Siu, K., Yuen, K., Castano-Rodriguez, C., Ye, Z., Yeung, M., Fung, S. et al. (2019) Severe acute respiratory syndrome Coronavirus ORF3a protein activates the NLRP3 inflammasome by promoting TRAF3-dependent ubiquitination of ASC. *FASEB J.* **33**, 8865–8877, [https://doi.org/10.1096/fj.201802418R](https://doi.org/10.1096/fj.201802418R)

126 Kusov, Y., Tan, J., Alvarez, E., Enjuanes, L. and Hilgenfeld, R. (2020) A G-quadruplex-binding macrodomain within the “SARS-unique domain” is essential for the activity of the SARS-coronavirus replication-transcription complex. *Virology* **484**, 313–322, [https://doi.org/10.1016/j.virol.2015.06.01](https://doi.org/10.1016/j.virol.2015.06.01)

127 Chang, Y.S., Ko, B.H., Ju, J.C., Chang, H.H., Huang, S.H. and Lin, C.W. (2020) SARS unique domain (SUD) of severe acute respiratory syndrome coronavirus induces NLRP3 inflammasome-dependent CXCL10-mediated pulmonary inflammation. *Int. J. Mol. Sci.* **21**, 1–19, [https://doi.org/10.3390/ijms21093179](https://doi.org/10.3390/ijms21093179)

128 Sun, D.Z., Song, C.Q., Xu, Y.M. and Dong, X.S. (2020) Role of the MAPK pathway in human lung epithelial-like A549 cells apoptosis induced by paraquat. *Genet. Mol. Biol.* **43**, e20190137, [https://doi.org/10.1590/1678-4685-gmb-2019-0137](https://doi.org/10.1590/1678-4685-gmb-2019-0137)

129 Arcaroli, J., Yum, H.K., Kupfner, J., Park, J.S., Yang, K.Y. and Abraham, E. (2001) Role of p38 MAP kinase in the development of acute lung injury. *Clin. Immunol.* **101**, 211–219, [https://doi.org/10.1006/clim.2001.5108](https://doi.org/10.1006/clim.2001.5108)

130 Li, S.W., Wang, C.Y., Jou, Y.J., Yang, T.C., Huang, S.H., Wan, L. et al. (2016) SARS coronavirus papain-like protease induces Egr-1-dependent up-regulation of TGF-β1 via ROS/p38 MAPK/STAT3 pathway. *Sci. Rep.* **6**, 1–13

131 Yoshida, K., Kuwano, K., Hagimoto, N., Watanabe, K., Matsuba, T., Fujita, M. et al. (2002) MAP kinase activation and apoptosis in lung tissues from patients with idiopathic pulmonary fibrosis. *J. Pathol.* **198**, 388–396, [https://doi.org/10.1002/path.1208](https://doi.org/10.1002/path.1208)

132 Qian, F., Deng, J., Wang, G., Ye, R.D. and Christman, J.W. (2016) Pivotal role of mitogen-activated protein kinase-activated protein kinase 2 in inflammatory pulmonary diseases. *Curr. Protein Pept. Sci.* **17**, 332–342, [https://doi.org/10.2174/1389203716666150629121324](https://doi.org/10.2174/1389203716666150629121324)

133 Renda, T., Baraldo, S., Pelaia, G., Bazzan, E., Turato, G., Papi, A. et al. (2008) Increased activation of p38 MAPK in COPD. *Eur. Respir. J.* **31**, 62–69, [https://doi.org/10.1183/09031936.00036707](https://doi.org/10.1183/09031936.00036707)

134 Mercer, B.A. and D’Armiento, J.M. (2006) Emerging role of MAP kinase pathways as therapeutic targets in COPD. *Int. J. Chron. Obstruct. Pulmon. Dis.* **1**, 137–150

135 Feng, Y., Fang, Z., Liu, B. and Zheng, X. (2019) P38MAPK plays a pivotal role in the development of acute respiratory distress syndrome. *Clinics* **74**, 1–5, [https://doi.org/10.6061/clinics/2019/e509](https://doi.org/10.6061/clinics/2019/e509)

136 Mizutani, T. (2010) Signaling pathways of SARS-CoV in vitro and in vivo. *Mol. Biol.* **1102**, 305–322, [https://doi.org/10.1007/978-3-642-03683-5_19](https://doi.org/10.1007/978-3-642-03683-5_19)

137 Mizutani, T. (2007) Signal transduction in SARS-CoV-infected cells. *Ann. N.Y. Acad. Sci.* **1102**, 86–95, [https://doi.org/10.1196/annals.1408.006](https://doi.org/10.1196/annals.1408.006)

138 Cheng, V.C.C., Hung, I.F.N., Tang, B.S.F., Chu, C.M., Wong, M.M.L., Chan, K.H. et al. (2004) Viral replication in the nasopharynx is associated with diarrhea in patients with severe acute respiratory syndrome. *Clin. Infect. Dis.* **38**, 467–475, [https://doi.org/10.1086/382681](https://doi.org/10.1086/382681)

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

139 Arabi, Y.M., Al-Omari, A., Mandourah, Y., Al-Hameed, F., Sindi, A.A., Alraddadi, B. et al. (2017) Critically Ill patients with the middle east respiratory syndrome: A multicenter retrospective cohort study. *Crit. Care Med.* **45**, 1683–1695, [https://doi.org/10.1097/CCM.0000000000002621](https://doi.org/10.1097/CCM.0000000000002621)

140 Wong, S.H., Lui, R.N.S. and Sung, J.J.Y. (2020) Covid-19 and the digestive system. *J. Gastroenterol. Hepatol.* **35**, 744–748, [https://doi.org/10.1111/jgh.15047](https://doi.org/10.1111/jgh.15047)

141 Luo, S., Zhang, X. and Xu, H. (2020) Don’t overlook digestive symptoms in patients with 2019 novel coronavirus disease (COVID-19). *Clin. Gastroenterol. Hepatol.* **18**, 1636–1637, [https://doi.org/10.1016/j.cgh.2020.03.043](https://doi.org/10.1016/j.cgh.2020.03.043)

142 Xiao, F., Tang, M., Zheng, X., Liu, Y., Li, X. and Shan, H. (2020) Evidence for gastrointestinal infection of SARS-CoV-2. *Gastroenterology* **158**, 1831.e3–1833.e3, [https://doi.org/10.1053/j.gastro.2020.02.055](https://doi.org/10.1053/j.gastro.2020.02.055)

143 Young, B.E., Ong, S.W.X., Kalimuddin, S., Low, J.G., Tan, S.Y., Loh, J. et al. (2020) Epidemiologic features and clinical course of patients infected with SARS-CoV-2 in Singapore. *JAMA* **323**, 1488, https://jamanetwork.com/journals/jama/fullarticle/2762688, [https://doi.org/10.1001/jama.2020.3204](https://doi.org/10.1001/jama.2020.3204)

144 Xu, Y., Li, X., Zhu, B., Liang, H., Fang, C., Gong, Y. et al. (2020) Characteristics of pediatric SARS-CoV-2 infection and potential evidence for persistent fecal viral shedding. *Nat. Med.* **26**, 502–505, [https://doi.org/10.1038/s41591-020-0817-4](https://doi.org/10.1038/s41591-020-0817-4)

145 Tang, A., Tong, Z., Wang, H., Dai, Y., Li, K., Liu, J. et al. (2020) Detection of novel coronavirus by RT-PCR in stool specimen from asymptomatic child, China. *Emerg. Infect. Dis.* **26**, 1337–1339, http://wwwnc.cdc.gov/eid/article/26/6/20-0301_article.htm, [https://doi.org/10.3201/eid2606.200301](https://doi.org/10.3201/eid2606.200301)

146 Zhang, W., Du, R.-H., Li, B., Zheng, X.-S., Yang, X.-L., Hu, B. et al. (2020) Molecular and serological investigation of 2019-nCoV infected patients: implication of multiple shedding routes. *Emerg. Microbes Infect.* **9**, 386–389, [https://doi.org/10.1080/22221751.2020.1729071](https://doi.org/10.1080/22221751.2020.1729071)

147 Yeo, C., Kaushal, S. and Yeo, D. (2020) Enteric involvement of coronaviruses: is faecal-oral transmission of SARS-CoV-2 possible? *Lancet Gastroenterol. Hepatol.* **5**, 335–337, [https://doi.org/10.1016/S2468-1253(20)30048-0](https://doi.org/10.1016/S2468-1253(20)30048-0)

148 van Doremalen, N., Bushmaker, T., Morris, D.H., Holbrook, M.G., Gamble, A., Williamson, B.N. et al. (2020) Aerosol and Surface Stability of SARS-CoV-2 as Compared with SARS-CoV-1. *N. Engl. J. Med.* **382**, 1564–1567, [https://doi.org/10.1056/NEJMMc2004973](https://doi.org/10.1056/NEJMMc2004973)

149 Setti, L., Passarini, F., De Gennaro, G., Barbieri, P., Perrone, M.G., Borelli, M. et al. (2020) Airborne transmission route of COVID-19: why 2 meters/6 feet of inter-personal distance could not be enough. *Int. J. Environ. Res. Public Health* **17**, 2932, [https://doi.org/10.3390/ijerph17082932](https://doi.org/10.3390/ijerph17082932)

150 Perlman, S. and Netland, J. (2009) Coronaviruses post-SARS: update on replication and pathogenesis. *Nat. Rev. Microbiol.* **7**, 439–450, [https://doi.org/10.1038/nrmicro2147](https://doi.org/10.1038/nrmicro2147)

151 Mönkemüller, K., Fry, L. and Rickes, S. (2020) Covid-19, Coronavirus, SARS-CoV-2 and the small bowel. *Rev. Esp. Enferm. Dig.* **112**, 383–388

152 Wang, D., Hu, B., Hu, C., Zhu, F., Liu, X., Zhang, J. et al. (2020) Clinical characteristics of 138 hospitalized patients with 2019 novel coronavirus-infected pneumonia in Wuhan, China. *JAMA* **323**, 1061, https://jamanetwork.com/journals/jama/fullarticle/2761044, [https://doi.org/10.1001/jama.2020.1585](https://doi.org/10.1001/jama.2020.1585)

153 Pan, L., Mu, M., Yang, P., Sun, Y., Wang, R., Yan, J. et al. (2020) Clinical characteristics of COVID-19 patients with digestive symptoms in Hubei, China. *Am. J. Gastroenterol.* **115**, 766–773, [https://doi.org/10.14309/ajg.000000000000620](https://doi.org/10.14309/ajg.000000000000620)

154 Budden, K.F., Gellatly, S.L., Wood, D.L.A., Cooper, M.A., Morrison, M., Hugenholtz, P. et al. (2017) Emerging pathogenic links between microbiota and the gut-lung axis. *Nat. Rev. Microbiol.* **15**, 55–63, [https://doi.org/10.1038/nrmicro.2016.142](https://doi.org/10.1038/nrmicro.2016.142)

155 Camargo, S.M.R., Singer, D., Makrides, V., Huggel, K., Pos, K.M., Wagner, C.A. et al. (2009) Tissue-specific amino acid transporter partners ACE2 and collectrin differentially interact with Hartnup Mutations. *Gastroenterology* **136**, 872.e3–882.e3, https://linkinghub.elsevier.com/retrieve/pii/S0016508508018930, [https://doi.org/10.1053/j.gastro.2008.10.055](https://doi.org/10.1053/j.gastro.2008.10.055)

156 Hashimoto, T., Perlot, T., Rehman, A., Trichereau, J., Ishiguro, H., Paolino, M. et al. (2012) ACE2 links amino acid malnutrition to microbial ecology and intestinal inflammation. *Nature* **487**, 477–481, [https://doi.org/10.1038/nature11228](https://doi.org/10.1038/nature11228)

157 Bröer, A., Juelich, T., Vanslambrouck, J.M., Tietze, N., Solomon, P.S., Holst, J. et al. (2011) Impaired nutrient signaling and body weight control in Na+-neutral amino acid cotransporter (Slc6a19)-deficient mouse. *J. Biol. Chem.* **286**, 26638–26651, [https://doi.org/10.1074/jbc.M111.241323](https://doi.org/10.1074/jbc.M111.241323)

158 Ma, X.M. and Blenis, J. (2009) Molecular mechanisms of mTOR-mediated translational control. *Nat. Rev. Mol. Cell Biol.* **10**, 307–318, [https://doi.org/10.1038/nrm2672](https://doi.org/10.1038/nrm2672)

159 Bostanckloğlu, M. (2020) Temporal correlation between neurological and gastrointestinal symptoms of SARS-CoV-2. *Inflamm. Bowel Dis.* **26**, 1–3, [https://doi.org/10.1093/ibd/izaa131](https://doi.org/10.1093/ibd/izaa131)

160 Louveau, A., Smirnov, I., Keyes, T.J., Eccles, J.D., Rouhani, S.J., Peske, J.D. et al. (2015) Structural and functional features of central nervous system lymphatic vessels. *Nature* **523**, 337–341, [https://doi.org/10.1038/nature14432](https://doi.org/10.1038/nature14432)

161 Jessen, N.A., Munk, A.S.F., Lundgaard, I. and Nedergaard, M. (2015) The glymphatic system: a beginner’s guide. *Neurochem. Res.* **40**, 2583–2599, [https://doi.org/10.1007/s11064-015-1581-6](https://doi.org/10.1007/s11064-015-1581-6)

162 Wu, Y., Xu, X., Chen, Z., Duan, J., Hashimoto, K., Yang, L. et al. (2020) Nervous system involvement after infection with COVID-19 and other coronaviruses. *Brain Behav. Immun.* **87**, 18–22, [https://doi.org/10.1016/j.bbi.2020.03.031](https://doi.org/10.1016/j.bbi.2020.03.031)

163 Liu, K., Pan, M., Xiao, Z. and Xu, X. (2020) Neurological manifestations of the coronavirus (SARS-CoV-2) pandemic 2019-2020. *J. Neurol. Neurosurg. Psychiatry* **91**, 669–670, [https://doi.org/10.1136/jnnp-2020-323177](https://doi.org/10.1136/jnnp-2020-323177)

164 Xydakis, M.S., Dehgani-Mobaraki, P., Holbrook, E.H., Geisthoff, U.W., Bauer, C., Hautefort, C. et al. (2020) Smell and taste dysfunction in patients with COVID-19. *Lancet Infect. Dis.* **S1473-3099**, 30293–0, [https://doi.org/10.1016/S1473-3099(20)30293-0](https://doi.org/10.1016/S1473-3099(20)30293-0)

165 Moriguchi, T., Harii, N., Goto, J., Harada, D., Sugawara, H., Takamino, J. et al. (2020) A first case of meningitis/encephalitis associated with SARS-Coronavirus-2. *Int. J. Infect. Dis.* **94**, 55–58, [https://doi.org/10.1016/j.ijid.2020.03.062](https://doi.org/10.1016/j.ijid.2020.03.062)

166 Baig, A.M., Khaleeq, A., Ali, U. and Syeda, H. (2020) Evidence of the COVID-19 virus targeting the CNS: tissue distribution, host-virus interaction, and proposed neurotropic mechanisms. *ACS Chem. Neurosci.* **11**, 995–998, [https://doi.org/10.1021/acschemneuro.0c00122](https://doi.org/10.1021/acschemneuro.0c00122)

167 Helms, J., Kremer, S., Merdji, H., Clere-Jehl, R., Schenck, M., Kummerlen, C. et al. (2020) Neurologic features in severe SARS-CoV-2 infection. *N. Engl. J. Med.* **382**, 2268–2270, [https://doi.org/10.1056/NEJMca2008597](https://doi.org/10.1056/NEJMca2008597)

168 Paniz-Mondolfi, A., Bryce, C., Grimes, Z., Gordon, R.E., Reidy, J., Lednicky, J. et al. (2020) Central nervous system involvement by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2). *J. Med. Virol.* **92**, 699–702, [https://doi.org/10.1002/jmv.25915](https://doi.org/10.1002/jmv.25915)

169 Zhou, L., Zhang, M., Wang, J. and Gao, J. (2020) Sars-Cov-2: underestimated damage to nervous system. *Travel Med. Infect. Dis.* 101642, https://linkinghub.elsevier.com/retrieve/pii/S1477893920301101, [https://doi.org/10.1016/j.tmaid.2020.101642](https://doi.org/10.1016/j.tmaid.2020.101642)

170 Panciani, P.P., Saraceno, G., Zanin, L., Renisi, G., Signorini, L., Battaglia, L. et al. (2020) SARS-CoV-2: “Three-steps” infection model and CSF diagnostic implication. *Brain Behav. Immun.* **87**, 128–129, [https://doi.org/10.1016/j.bbi.2020.05.002](https://doi.org/10.1016/j.bbi.2020.05.002)

171 Desforges, M., Le Coupanec, A., Dubeau, P., Bourgouin, A., Lajoie, L., Dubé, M. et al. (2020) Human coronaviruses and other respiratory viruses: underestimated opportunistic pathogens of the central nervous system? *Viruses* **12**, 14, [https://doi.org/10.3390/v12010014](https://doi.org/10.3390/v12010014)

172 Fazakerley, J.K. and Walker, R. (2003) Virus demyelination. *J. Neurovirol.* **9**, 148–164, [https://doi.org/10.1080/13550280390194046](https://doi.org/10.1080/13550280390194046)

173 Murray, R.S., Cai, G.-Y., Hoel, K., Zhang, J.-Y., Soike, K.F. and Cabirac, G.F. (1992) Coronavirus infects and causes demyelination in primate central nervous system. *Virology* **188**, 274–284, [https://doi.org/10.1016/0042-6822(92)90757-G](https://doi.org/10.1016/0042-6822(92)90757-G)

174 Lane, T.E. and Buchmeier, M.J. (1997) Murine coronavirus infection: a paradigm for virus-induced demyelinating disease. *Trends Microbiol.* **5**, 9–14, [https://doi.org/10.1016/S0966-842X(97)81768-4](https://doi.org/10.1016/S0966-842X(97)81768-4)

175 Stohlman, S.A. and Hinton, D.R. (2001) Viral induced demyelination. *Brain Pathol.* **11**, 92–106, [https://doi.org/10.1111/j.1750-3639.2001.tb00384.x](https://doi.org/10.1111/j.1750-3639.2001.tb00384.x)

176 Li, Z., Liu, T., Yang, N., Han, D., Mi, X., Li, Y. et al. (2020) Neurological manifestations of patients with COVID-19: potential routes of SARS-CoV-2 neuroinvasion from the periphery to the brain. *Front. Med.* 1–9, [https://doi.org/10.1007/s11684-020-0786-5](https://doi.org/10.1007/s11684-020-0786-5)

177 Netland, J., Meyerholz, D.K., Moore, S., Cassell, M. and Perlman, S. (2008) Severe acute respiratory syndrome coronavirus infection causes neuronal death in the absence of encephalitis in mice transgenic for human ACE2. *J. Virol.* **82**, 7264–7275, [https://doi.org/10.1128/JVI.00737-08](https://doi.org/10.1128/JVI.00737-08)

178 Bohmwald, K., Galvez, N., Ríos, M. and Kalergis, A.M. (2018) Neurologic alterations due to respiratory virus infections. *Front. Cell Neurosci.* **12**, 386, [https://doi.org/10.3389/fncel.2018.00386](https://doi.org/10.3389/fncel.2018.00386)

179 Montalvan, V., Lee, J., Bueso, T., De Toledo, J. and Rivas, K. (2020) Neurological manifestations of COVID-19 and other coronavirus infections: a systematic review. *Clin. Neurol. Neurosurg.* **194**, 105921, [https://doi.org/10.1016/j.clineuro.2020.105921](https://doi.org/10.1016/j.clineuro.2020.105921)

180 Zhou, F., Yu, T., Du, R., Fan, G., Liu, Y., Liu, Z. et al. (2020) Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study. *Lancet* **395**, 1054–1062, https://linkinghub.elsevier.com/retrieve/pii/S0140673620305663, [https://doi.org/10.1016/S0140-6736(20)30566-3](https://doi.org/10.1016/S0140-6736(20)30566-3)

181 Varga, Z., Flammer, A.J., Steiger, P., Haberecker, M., Andermatt, R., Zinkernagel, A.S. et al. (2020) Endothelial cell infection and endotheliitis in COVID-19. *Lancet* **395**, 1417–1418, [https://doi.org/10.1016/S0140-6736(20)30937-5](https://doi.org/10.1016/S0140-6736(20)30937-5)

182 Bostanciklioğlu, M. (2020) SARS-CoV2 entry and spread in the lymphatic drainage system of the brain. *Brain Behav. Immun.* **87**, 122–123, [https://doi.org/10.1016/j.bbi.2020.04.080](https://doi.org/10.1016/j.bbi.2020.04.080)

183 Li, Y.C., Bai, W.Z. and Hashikawa, T. (2020) The neuroinvasive potential of SARS-CoV2 may play a role in the respiratory failure of COVID-19 patients. *J. Med. Virol.* **92**, 552–555, [https://doi.org/10.1002/jmv.25728](https://doi.org/10.1002/jmv.25728)

184 Li, Y., Bai, W., Hirano, N., Hayashida, T., Taniguchi, T., Sugita, Y. et al. (2013) Neurotropic virus tracing suggests a membranous-coating-mediated mechanism for transsynaptic communication. *J. Comp. Neurol.* **521**, 203–212, [https://doi.org/10.1002/cne.23171](https://doi.org/10.1002/cne.23171)

185 Reichard, R.R., Kashani, K.B., Boire, N.A., Constantopoulos, E., Guo, Y. and Lucchinetti, C.F. (2020) Neuropathology of COVID-19: a spectrum of vascular and acute disseminated encephalomyelitis (ADEM)-like pathology. *Acta Neuropathol.* **140**, 1–6, [https://doi.org/10.1007/s00401-020-02166-2](https://doi.org/10.1007/s00401-020-02166-2)

186 Kanberg, N., Ashton, N.J., Andersson, L.-M., Yilmaz, A., Lindh, M., Nilsson, S. et al. (2020) Neurochemical evidence of astrocytic and neuronal injury commonly found in COVID-19. *Neurology*, https://n.neurology.org/content/neurology/early/2020/06/16/WNL.00000000000010111.full.pdf, [https://doi.org/10.1212/WNL.00000000000010111](https://doi.org/10.1212/WNL.00000000000010111)

187 Xiao, L., Haack, K.K.V. and Zucker, I.H. (2013) Angiotensin II regulates ACE and ACE2 in neurons through p38 mitogen-activated protein kinase and extracellular signal-regulated kinase 1/2 signaling. *Am. J. Physiol. Cell Physiol.* **304**, C1073–C1079, [https://doi.org/10.1152/ajpcell.00364.2012](https://doi.org/10.1152/ajpcell.00364.2012)

188 Tang, N., Li, D., Wang, X. and Sun, Z. (2020) Abnormal coagulation parameters are associated with poor prognosis in patients with novel coronavirus pneumonia. *J. Thromb. Haemost.* **18**, 844–847, [https://doi.org/10.1111/jth.14768](https://doi.org/10.1111/jth.14768)

189 Mehta, P., McAuley, D.F., Brown, M., Sanchez, E., Tattersall, R.S., Manson, J.J. et al. (2020) COVID-19: consider cytokine storm syndromes and immunosuppression. *Lancet* **395**, 1033, [https://doi.org/10.1016/S0140-6736(20)30628-0](https://doi.org/10.1016/S0140-6736(20)30628-0)

190 Koralnik, I.J. and Tyler, K.L. (2020) COVID-19: a global threat to the nervous system. *Ann. Neurol.* **88**, 1–11, [https://doi.org/10.1002/ana.25807](https://doi.org/10.1002/ana.25807)

191 Toscano, G., Palmerini, F., Ravaglia, S., Ruiz, L., Invernizzi, P., Cuzzoni, M.G. et al. (2020) Guillain-Barré Syndrome associated with SARS-CoV-2. *N. Engl. J. Med.* **382**, NEJM2009191, [https://doi.org/10.1056/NEJM2009191](https://doi.org/10.1056/NEJM2009191)

192 Madjid, M., Safavi-Naeini, P., Solomon, S.D. and Vardeny, O. (2020) Potential effects of coronaviruses on the cardiovascular system. *JAMA Cardiol* **10**, 1–10, https://jamanetwork.com/journals/jamacardiology/fullarticle/2763846

193 Epelman, S., Shrestha, K., Troughton, R.W., Francis, G.S., Sen, S., Klein, A.L. et al. (2009) Soluble angiotensin-converting enzyme 2 in human heart failure: relation with myocardial function and clinical outcomes. *J. Cardiac Fail.* **15**, 565–571, [https://doi.org/10.1016/j.cardfail.2009.01.014](https://doi.org/10.1016/j.cardfail.2009.01.014)

194 Wild, P.S., Dimmeler, S. and Eschenhagen, T. (2020) An epidemiological study exploring a possible impact of treatment with ACE inhibitors or angiotensin receptor blockers on ACE2 plasma concentrations. *J. Mol. Cell Cardiol.* **141**, 108–109, [https://doi.org/10.1016/j.yjmcc.2020.03.018](https://doi.org/10.1016/j.yjmcc.2020.03.018)

195 Wu, Z. and McGoggan, J.M. (2020) Characteristics of and important lessons from the coronavirus disease 2019 (COVID-19) outbreak in China. *JAMA* **323**, 1239, https://jamanetwork.com/journals/jama/fullarticle/2762130, [https://doi.org/10.1001/jama.2020.2648](https://doi.org/10.1001/jama.2020.2648)

196 Zheng, Y.-Y., Ma, Y.-T., Zhang, J.-Y. and Xie, X. (2020) COVID-19 and the cardiovascular system. *Nat. Rev. Cardiol.* **17**, 259–260, [https://doi.org/10.1038/s41569-020-0360-5](https://doi.org/10.1038/s41569-020-0360-5)

197 Ruan, Q., Yang, K., Wang, W., Jiang, L. and Song, J. (2020) Clinical predictors of mortality due to COVID-19 based on an analysis of data of 150 patients from Wuhan, China. *Intensive Care Med.* **46**, 846–848, [https://doi.org/10.1007/s00134-020-05991-x](https://doi.org/10.1007/s00134-020-05991-x)

Clinical Science (2020) 134 2137–2160  
https://doi.org/10.1042/CS20200904  

198 Cheng, R. and Leedy, D. (2020) COVID-19 and acute myocardial injury: the heart of the matter or an innocent bystander? *Heart* **106**, 1122–1124, [https://doi.org/10.1136/heartjnl-2020-317025](https://doi.org/10.1136/heartjnl-2020-317025)  
199 Zhu, L., Carretero, O.A., Xu, J., Harding, P., Ramadurai, N., Gu, X. et al. (2015) Activation of angiotensin II type 2 receptor suppresses TNF-α-induced ICAM-1 via NF-κB: possible role of ACE2. *Am. J. Physiol. Heart Circ. Physiol.* **309**, H827–H834, [https://doi.org/10.1152/ajpheart.00814.2014](https://doi.org/10.1152/ajpheart.00814.2014)  
200 Dzimiri, N. (2002) Receptor crosstalk. Implications for cardiovascular function, disease and therapy. *Eur. J. Biochem.* **269**, 4713–4730, [https://doi.org/10.1046/j.1432-1033.2002.03181.x](https://doi.org/10.1046/j.1432-1033.2002.03181.x)  
201 Wheeler-Jones, C.P.D. (2005) Cell signalling in the cardiovascular system: an overview. *Heart* **91**, 1366–1374, [https://doi.org/10.1136/hrt.2005.072280](https://doi.org/10.1136/hrt.2005.072280)  
202 Houliston, R.A., Pearson, J.D. and Wheeler-Jones, C.P.D. (2001) Agonist-specific cross talk between ERKs and p38 mapk regulates PGI 2 synthesis in endothelium. *Am. J. Physiol. Physiol.* **281**, C1266–C1276, [https://doi.org/10.1152/ajpcell.2001.281.4.C1266](https://doi.org/10.1152/ajpcell.2001.281.4.C1266)  
203 Adam, L.P., Franklin, M.T., Raff, G.J. and Hathaway, D.R. (1995) Activation of mitogen-activated protein kinase in porcine carotid arteries. *Circ. Res.* **76**, 183–190, [https://doi.org/10.1161/01.RES.76.2.183](https://doi.org/10.1161/01.RES.76.2.183)  
204 Nebreda, A.R. and Porras, A. (2000) p38 MAP kinases: beyond the stress response. *Trends Biochem. Sci.* **25**, 257–260, https://linkinghub.elsevier.com/retrieve/pii/S0968000400015954, [https://doi.org/10.1016/S0968-0004(00)01595-4](https://doi.org/10.1016/S0968-0004(00)01595-4)  
205 Wang, Y., Huang, S., Sah, V.P., Ross, J., Brown, J.H., Han, J. et al. (1998) Cardiac muscle cell hypertrophy and apoptosis induced by distinct members of the p38 mitogen-activated protein kinase family. *J. Biol. Chem.* **273**, 2161–2168, [https://doi.org/10.1074/jbc.273.4.2161](https://doi.org/10.1074/jbc.273.4.2161)  
206 Saurin, A.T., Martin, J.L., Heads, R.J., Foley, C., Mockridge, J.W., Wright, M.J. et al. (2000) The role of differential activation of p38-mitogen-activated protein kinase in preconditioned ventricular myocytes. *FASEB J.* **14**, 2237–2246, [https://doi.org/10.1096/fj.99-0671com](https://doi.org/10.1096/fj.99-0671com)  
207 Podewski, E.K., Hilfiker-Kleiner, D., Hilfiker, A., Morawietz, H., Lichtenberg, A., Wollert, K.C. et al. (2003) Alterations in Janus Kinase (JAK)-Signal Transducers and Activators of Transcription (STAT) signaling in patients with end-stage dilated cardiomyopathy. *Circulation* **107**, 798–802, [https://doi.org/10.1161/01.CIR.0000057545.82749.FF](https://doi.org/10.1161/01.CIR.0000057545.82749.FF)  
208 Patel, A.B. and Verma, A. (2020) COVID-19 and angiotensin-converting enzyme inhibitors and angiotensin receptor blockers: what is the evidence? *JAMA.* **11**, 875–879, https://jamanetwork.com/journals/jama/fullarticle/2763803  
209 Wang, L., Li, X., Chen, H., Yan, S., Li, D., Li, Y. et al. (2020) Coronavirus Disease 19 infection does not result in acute kidney injury: an analysis of 116 hospitalized patients from Wuhan, China. *Am. J. Nephrol.* **51**, 343–348, [https://doi.org/10.1159/000507471](https://doi.org/10.1159/000507471)  
210 Cheng, Y., Luo, R., Wang, K., Zhang, M., Wang, Z., Dong, L. et al. (2020) Kidney disease is associated with in-hospital death of patients with COVID-19. *Kidney Int.* **97**, 829–838, [https://doi.org/10.1016/j.kint.2020.03.005](https://doi.org/10.1016/j.kint.2020.03.005)  
211 Yang, X., Yu, Y., Xu, J., Shu, H., Xia, J., Liu, H. et al. (2020) Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study. *Lancet Respir. Med.* **8**, 475–481, [https://doi.org/10.1016/S2213-2600(20)30079-5](https://doi.org/10.1016/S2213-2600(20)30079-5)  
212 Chen, T., Wu, D., Chen, H., Yan, W., Yang, D., Chen, G. et al. (2020) Clinical characteristics of 113 deceased patients with coronavirus disease 2019: retrospective study. *BMJ* **368**, m1091, [https://doi.org/10.1136/bmj.m1091](https://doi.org/10.1136/bmj.m1091)  
213 Hirsch, J.S., Ng, J.H., Ross, D.W., Sharma, P., Shah, H.H., Barnett, R.L. et al. (2020) Acute kidney injury in patients hospitalized with COVID-19. *Kidney Int.* **98**, 209–218, [https://doi.org/10.1016/j.kint.2020.05.006](https://doi.org/10.1016/j.kint.2020.05.006)  
214 Su, H., Yang, M., Wan, C., Yi, L.X., Tang, F., Zhu, H.Y. et al. (2020) Renal histopathological analysis of 26 postmortem findings of patients with COVID-19 in China. *Kidney Int.* **98**, 219–227, [https://doi.org/10.1016/j.kint.2020.04.003](https://doi.org/10.1016/j.kint.2020.04.003)  
215 Fanelli, V., Fiorentino, M., Cantaluppi, V., Gesualdo, L., Stallone, G., Ronco, C. et al. (2020) Acute kidney injury in SARS-CoV-2 infected patients. *Crit. Care* **24**, 155, [https://doi.org/10.1186/s13054-020-02872-z](https://doi.org/10.1186/s13054-020-02872-z)  
216 Panitchote, A., Mehkri, O., Hastings, A., Hanane, T., Demirjian, S., Torbic, H. et al. (2019) Factors associated with acute kidney injury in acute respiratory distress syndrome. *Ann. Intensive Care* **9**, 74, [https://doi.org/10.1186/s13613-019-0552-5](https://doi.org/10.1186/s13613-019-0552-5)  
217 Husain-Syed, F., Slutsky, A.S. and Ronco, C. (2016) Lung-kidney cross-talk in the critically ill patient. *Am. J. Respir. Crit. Care Med.* **194**, 402–414, [https://doi.org/10.1164/rccm.201602-0420CP](https://doi.org/10.1164/rccm.201602-0420CP)  
218 Chen, G., Zhou, Y., Ma, J., Xia, P., Qin, Y. and Li, X. (2020) Is there a role for blood purification therapies targeting cytokine storm syndrome in critically severe COVID-19 patients? *Renal Fail.* **42**, 483–488, [https://doi.org/10.1080/0886022X.2020.1764369](https://doi.org/10.1080/0886022X.2020.1764369)  
219 Teijaro, J.R., Walsh, K.B., Cahalan, S., Fremgen, D.M., Roberts, E., Scott, F. et al. (2011) Endothelial cells are central orchestrators of cytokine amplification during influenza virus infection. *Cell* **146**, 980–991, [https://doi.org/10.1016/j.cell.2011.08.015](https://doi.org/10.1016/j.cell.2011.08.015)  
220 Cunningham, P.N., Dyanov, H.M., Park, P., Wang, J., Newell, K.A. and Quigg, R.J. (2002) Acute renal failure in endotoxemia is caused by TNF acting directly on TNF receptor-1 in kidney. *J. Immunol.* **168**, 5817–5823, [https://doi.org/10.4049/jimmunol.168.11.5817](https://doi.org/10.4049/jimmunol.168.11.5817)  
221 Nechemia-Arbely, Y., Barkan, D., Pizov, G., Shriki, A., Rose-John, S., Galun, E. et al. (2008) IL-6/IL-6R axis plays a critical role in acute kidney injury. *J. Am. Soc. Nephrol.* **19**, 1106–1115, [https://doi.org/10.1681/ASN.2007070744](https://doi.org/10.1681/ASN.2007070744)  
222 Desai, T.R., Leeper, N.J., Hynes, K.L. and Gewertz, B.L. (2002) Interleukin-6 causes endothelial barrier dysfunction via the protein kinase C pathway. *J. Surg. Res.* **104**, 118–123, [https://doi.org/10.1006/jsre.2002.6415](https://doi.org/10.1006/jsre.2002.6415)  
223 Guan, W.J., Ni, Z.Y., Hu, Y., Liang, W.H., Ou, C.Q., He, J.X. et al. (2020) Clinical characteristics of coronavirus disease 2019 in China. *N. Engl. J. Med.* **382**, 1708–1720, [https://doi.org/10.1056/NEJMoa2002032](https://doi.org/10.1056/NEJMoa2002032)  
224 Martinez-Rojas, M.A., Vega-Vega, O. and Bobadilla, N.A. (2020) Is the kidney a target of SARS-CoV-2? *Am. J. Physiol. Renal Physiol.* **318**, F1454–F1462, [https://doi.org/10.1152/ajprenal.00160.2020](https://doi.org/10.1152/ajprenal.00160.2020)  
225 Farkash, E.A., Wilson, A.M. and Jentzen, J.M. (2020) Ultrastructural evidence for direct renal infection with SARS-CoV-2. *J. Am. Soc. Nephrol.* **31**, 1683–1687, [https://doi.org/10.1681/ASN.2020040432](https://doi.org/10.1681/ASN.2020040432)  
226 Pan, X., Xu, D., Zhang, H., Zhou, W., Wang, L. and Cui, X. (2020) Identification of a potential mechanism of acute kidney injury during the COVID-19 outbreak: a study based on single-cell transcriptome analysis. *Intensive Care Med.* **46**, 1114–1116, [https://doi.org/10.1007/s00134-020-06026-1](https://doi.org/10.1007/s00134-020-06026-1)  

© 2020 The Author(s). This is an open access article published by Portland Press Limited on behalf of the Biochemical Society and distributed under the Creative Commons Attribution License 4.0 (CC BY-NC-ND).

227 Monteil, V., Kwon, H., Prado, P., Hagelkruys, A., Wimmer, R.A., Stahl, M. et al. (2020) Inhibition of SARS-CoV-2 infections in engineered human tissues using clinical-grade soluble human ACE2. *Cell* **181**, 905.e7–913.e7, [https://doi.org/10.1016/j.cell.2020.04.004](https://doi.org/10.1016/j.cell.2020.04.004)

228 clinicaltrials.gov *Clinical Trials (COVID-19)*, https://clinicaltrials.gov/ct2/results?cond=COVID-19

229 Annweiler, C., Cao, Z., Wu, Y., Faucon, E., Mouhat, S., Kovacic, H. et al. (2020) Counter-regulatory ‘Renin-Angiotensin’ system-based candidate drugs to treat COVID-19 diseases in SARS-CoV-2-infected patients. *Infect. Disord. Drug Targets* **20**, 1, [https://doi.org/10.2174/1871526520666200518073329](https://doi.org/10.2174/1871526520666200518073329)

230 Khan, A., Benthin, C., Zeno, B., Albertson, T.E., Boyd, J., Christie, J.D. et al. (2017) A pilot clinical trial of recombinant human angiotensin-converting enzyme 2 in acute respiratory distress syndrome. *Crit. Care* **21**, 234, [https://doi.org/10.1186/s13054-017-1823-x](https://doi.org/10.1186/s13054-017-1823-x)

231 Tanaka, P.P., Santos, J.C., Oliveira, E.H. and Miglioli, N.A. (2020) A Crispr-Cas9 system designed to introduce point mutations into the human ACE2 gene to weaken the interaction of the ACE2 receptor with the SARS-CoV-2 S protein. *Preprints* 2, https://www.preprints.org

232 Liu, C., Zhou, Q., Li, Y., Garner, L.V., Watkins, S.P., Carter, L.J. et al. (2020) Research and development on therapeutic agents and vaccines for COVID-19 and related human coronavirus diseases. *ACS Cent. Sci.* **6**, 315–331, [https://doi.org/10.1021/acscentsci.0c00272](https://doi.org/10.1021/acscentsci.0c00272)

233 Maier, H.J., Bickerton, E. and Britton, P. (2015) Coronaviruses: methods and protocols. 1–282

234 Sun, D. (2020) Remdesivir for treatment of COVID-19: combination of pulmonary and IV administration may offer additional benefit. *AAPS J.* **22**, 77, [https://doi.org/10.1208/s12248-020-00459-8](https://doi.org/10.1208/s12248-020-00459-8)

235 Warren, T.K., Jordan, R., Lo, M.K., Ray, A.S., Mackman, R.L., Soloveva, V. et al. (2016) Therapeutic efficacy of the small molecule GS-5734 against Ebola virus in rhesus monkeys. *Nature* **531**, 381–385, [https://doi.org/10.1038/nature17180](https://doi.org/10.1038/nature17180)

236 Grein, J., Ohmagari, N., Shin, D., Diaz, G., Asperges, E., Castagna, A. et al. (2020) Compassionate use of remdesivir for patients with severe Covid-19. *N. Engl. J. Med.* **382**, 2327–2336, [https://doi.org/10.1056/NEJMoa2007016](https://doi.org/10.1056/NEJMoa2007016)

237 Williamson, B.N., Feldmann, F., Schwarz, B., Meade-White, K., Porter, D.P., Schulz, J. et al. (2020) Clinical benefit of remdesivir in rhesus macaques infected with SARS-CoV-2. *Nature*, [https://doi.org/10.1038/s41586-020-2423-5](https://doi.org/10.1038/s41586-020-2423-5)

238 Beigel, J.H., Tomashek, K.M., Dodd, L.E., Mehta, A.K., Zingman, B.S., Kalil, A.C. et al. (2020) Remdesivir for the treatment of Covid-19 — preliminary report. *N. Engl. J. Med.* NEJMoa200776, [https://doi.org/10.1056/NEJMoa2007764](https://doi.org/10.1056/NEJMoa2007764)

239 Furuta, Y., Gowen, B.B., Takahashi, K., Shiraki, K., Smee, D.F. and Barnard, D.L. (2013) Favipiravir (T-705), a novel viral RNA polymerase inhibitor. *Antiviral Res.* **100**, 446–454, https://linkinghub.elsevier.com/retrieve/pii/S0166354213002635, [https://doi.org/10.1016/j.antiviral.2013.09.015](https://doi.org/10.1016/j.antiviral.2013.09.015)

240 Chen, C., Huang, J., Cheng, Z., Wu, J., Chen, S., Zhang, Y. et al. (2020) Favipiravir versus Arbidol for COVID-19: a randomized clinical trial. *medRxiv*

241 Shanmugaraj, B., Siriwattananon, K., Wangkanont, K. and Phoolcharoen, W. (2020) Perspectives on monoclonal antibody therapy as potential therapeutic intervention for Coronavirus disease-19 (COVID-19). *Asian Pac. J. Allergy Immunol.* **38**, 10–18, http://apjai-journal.org/wp-content/uploads/2020/03/2.pdf, [https://doi.org/10.12932/AP-200220-0773](https://doi.org/10.12932/AP-200220-0773)

242 Saha, A., Sharma, A.R., Bhattacharya, M., Sharma, G., Lee, S.S. and Chakraborty, C. (2020) Tocilizumab: a therapeutic option for the treatment of cytokine storm syndrome in COVID-19. *Arch. Med. Res.*, [https://doi.org/10.1016/j.arcmed.2020.05.009](https://doi.org/10.1016/j.arcmed.2020.05.009)

243 Muhović, D., Bojović, J., Bulatović, A., Vukčević, B., Ratković, M., Lazović, R. et al. (2020) First case of drug-induced liver injury associated with the use of tocilizumab in a patient with COVID-19. *Liver Int.*, [https://doi.org/10.1111/liv.14516](https://doi.org/10.1111/liv.14516)

244 Seif, F., Aazami, H., Khoshmirsafa, M., Kamali, M., Mohsenzadegan, M., Pornour, M. et al. (2020) JAK inhibition as a new treatment strategy for patients with COVID-19. *Int. Arch. Allergy Immunol.* **111**, 467–475, [https://doi.org/10.1159/000508247](https://doi.org/10.1159/000508247)

245 Spinelli, F.R., Conti, F. and Gadina, M. (2020) HiJAKing SARS-CoV-2? The potential role of JAK inhibitors in the management of COVID-19. *Sci. Immunol.* **5**, 3–6, [https://doi.org/10.1126/sciimmunol.abc5367](https://doi.org/10.1126/sciimmunol.abc5367)

246 Al-Salama, Z.T. and Scott, L.J. (2018) Baricitinib: a review in rheumatoid arthritis. *Drugs* **78**, 761–772, [https://doi.org/10.1007/s40265-018-0908-4](https://doi.org/10.1007/s40265-018-0908-4)

247 Zhang, J., Xie, B. and Hashimoto, K. (2020) Current status of potential therapeutic candidates for the COVID-19 crisis. *Brain Behav. Immun.* **87**, 59–73, https://linkinghub.elsevier.com/retrieve/pii/S0889159120305894, [https://doi.org/10.1016/j.bbi.2020.04.046](https://doi.org/10.1016/j.bbi.2020.04.046)

248 Ströher, U., DiCaro, A., Li, Y., Strong, J.E., Aoki, F., Plummer, F. et al. (2004) Severe acute respiratory syndrome-related coronavirus is inhibited by interferon-α. *J. Infect. Dis.* **189**, 1164–1167, [https://doi.org/10.1086/382597](https://doi.org/10.1086/382597)

249 Barlow, A., Landolf, K.M., Barlow, B., Yeung, S.Y.A., Heavner, J.J., Claassen, C.W. et al. (2020) Review of emerging pharmacotherapy for the treatment of coronavirus disease 2019. *Pharmacother. J. Hum. Pharmacol. Drug Ther.* **40**, 416–437, [https://doi.org/10.1002/phar.2398](https://doi.org/10.1002/phar.2398)

250 Albini, A., Di Guardo, G., Noonan, D.M. and Lombardo, M. (2020) The SARS-CoV-2 receptor, ACE-2, is expressed on many different cell types: implications for ACE-inhibitor- and angiotensin II receptor blocker-based cardiovascular therapies. *Intern. Emerg. Med.* **15**, 759–766, [https://doi.org/10.1007/s11739-020-02364-6](https://doi.org/10.1007/s11739-020-02364-6)

251 Groß, S., Jahn, C., Cushman, S., Bär, C. and Thum, T. (2020) SARS-CoV-2 receptor ACE2-dependent implications on the cardiovascular system: from basic science to clinical implications. *J. Mol. Cell Cardiol.* **144**, 47–53, [https://doi.org/10.1016/j.yjmcc.2020.04.031](https://doi.org/10.1016/j.yjmcc.2020.04.031)

252 Gu, J., Gong, E., Zhang, B., Zheng, J., Gao, Z., Zhong, Y. et al. (2005) Multiple organ infection and the pathogenesis of SARS. *J. Exp. Med.* **202**, 415–424, [https://doi.org/10.1084/jem.20050828](https://doi.org/10.1084/jem.20050828)

253 Park, A. and Iwasaki, A. (2020) Type I and Type III Interferons – induction, signaling, evasion, and application to combat COVID-19. *Cell Host Microbe* **27**, 870–878, [https://doi.org/10.1016/j.chom.2020.05.008](https://doi.org/10.1016/j.chom.2020.05.008)
