
Review

# Components and Architecture of the Rhabdovirus Ribonucleoprotein Complex

**Christian Riedel** ${}^{1,*}$, **Alexandru A. Henrich** ${}^{2}$, and **Karl-Klaus Conzelmann** ${}^{2}$

${}^{1}$ Institute of Virology, Department of Pathobiology, University of Veterinary Medicine Vienna, 1210 Vienna, Austria  
${}^{2}$ Max von Pettenkofer-Institute Virology, Faculty of Medicine, and Gene Center, LMU Munich, 81377 Munich, Germany; henrich@genzentrum.lmu.de (A.A.H.); conzelmann@genzentrum.lmu.de (K.-K.C.)  

* Correspondence: christiane.riedel@vetmeduni.ac.at

Received: 31 July 2020; Accepted: 27 August 2020; Published: 29 August 2020

---

**Abstract:** Rhabdoviruses, as single-stranded, negative-sense RNA viruses within the order Mononegavirales, are characterised by bullet-shaped or bacteroid particles that contain a helical ribonucleoprotein complex (RNP). Here, we review the components of the RNP and its higher-order structural assembly.

**Keywords:** rhabdoviruses; ribonucleoprotein structure; rabies virus; vesicular stomatitis virus; lagos bat virus; lyssavirus; vesiculovirus

---

## 1. Introduction

Rhabdoviruses (family Rhabdoviridae, order Mononegavirales) are negative-sense, single-stranded RNA viruses with a genome of about 10.8–16.1 kb. The prototypic mammalian rhabdoviruses include the rabies virus (RABV) and vesicular stomatitis virus (VSV), which cause severe human and animal disease. The cell biology of these viruses is highly divergent in many important aspects. VSV is transmitted in nature by insect vectors, causes a rapid general host cell shutdown in mammalian cells and replicates to very high titres. The neurotropic rabies virus is directly transmitted between mammals, grows slowly and relies on stealth strategies to escape host responses. Despite having divergent biology, the viruses share a highly similar genetic and structural organisation.

The small genome encodes the minimal set of five proteins that are all present in the virus particle. They consist of the nucleoprotein (N), the matrix protein (M), the surface glycoprotein (G), the phosphoprotein or polymerase cofactor (P) and the RNA-dependent RNA polymerase (L). In contrast to other members of the mononegavirales, such as paramyxoviruses or filoviruses, rhabdoviruses' enveloped particles are characterised by a bullet-like or bacteroid appearance (Figure 1). The G-protein is the only surface glycoprotein, and therefore the essential factor for virus attachment and entry, including fusion. It belongs to the class III of fusion proteins, together with herpesvirus gB or baculovirus GP64 protein (as reviewed in [1]). Inside the viral envelope sits a helical ribonucleoprotein complex (RNP), which ends with a cone that has tighter helical turns (Figure 1). The RNP's main building blocks are the N- and M-proteins. Both are structurally highly conserved between the genera Vesiculovirus (exemplified by VSV) and Lyssavirus (including RABV and Lagos bat virus (LBV)) (RMSD (root mean square deviation) $C_{\alpha}$ 2.5 Å for N [2], 3.1 Å for M [3]), but the sequence conservation at the amino acid level is only 15% (N-protein). The structural homology implies the need for a specific protein layout to fulfil tasks linked to RNA binding and RNP generation. In contrast to N- and M-proteins, which are incorporated into the RNP at approximately equimolar amounts (~1200:1200 based on a CryoEM reconstruction [4] and 1258:1826 based on biochemical analysis [5]), the P-protein

---

Viruses 2020, 12, 959; doi:10.3390/v12090959  
www.mdpi.com/journal/viruses

is estimated at 466 copies per virion [5], while the estimated number of G-protein is 1205 copies per virion. With a molecular weight of 242 kD, and as the most important protein for genome replication and transcription, the L-protein is the least numerous protein present in the virion, with an estimated 50 copies per virion [5].

![Image Description](image.png)

Figure 1. Cryo-electron tomograms of rabies virus (RABV) and vesicular stomatitis virus (VSV) particles at the same scale. The locations of the M- and N-proteins are indicated by blue circles and purple rectangles, respectively. The location of the G-protein is indicated by red triangles, and the size of the L-protein, as derived from the electron density map of the VSV L-protein generated by [6], is indicated in green. The tomograms were acquired on an FEI Glacios (ThermoFisher, OR, USA) with a Falcon2 (ThermoFisher, OR, USA) direct electron detector, reconstructed in etomo [7] and visualised by employing 3dmod [7].

## 2. N Protein Structure and Function

The atomic structures of the N-proteins of VSV [8] and RABV [9] were resolved using X-ray crystallography (Figure 2). Both proteins were crystallised as rings in a complex with RNA, containing either 10 (VSV) or 11 (RABV) N-protein monomers. The monomer itself can be separated into an N- and C-terminal domain, which are separated by a cleft that is responsible for the binding of the RNA genome. Each N-protein is associated with nine nucleotides, and the N-RNA interaction is mainly coordinated by basic amino acid side chains that interact with the RNA's phosphate groups. The corresponding five residues for RABV are Arg149, Arg168, Arg225, Arg323 and Arg434, while the corresponding six residues for VSV are Arg143, Arg146, Lys155, Lys286, Arg317 and Arg408. Only two of these basic residues - RABV Arg149/VSV Arg143 and RABV Arg434/VSV Arg408 - are structurally conserved between RABV and VSV, resulting in a different arrangement of the nucleotides in the RNA binding cavity [2]. Higher-order N-protein assemblies are stabilised by an N- and a C-terminally protruding extension, which interact with the neighbouring N-protein monomers to form a circular assembly. A tabular presentation of the interacting residues is given in [2].

N-protein is not only essential for RNP formation but is also an essential factor during replication and transcription, as the viral polymerase only processes RNA that is bound to N-protein. Currently, it is not known how the polymerase complex of L and P gains access to the RNA in complex with N-protein and how this process is coordinated.

Figure 2. Comparison of the RABV and VSV N-protein structures (pdb 2gtt (RABV, [9]) and 2gic (VSV, [8])). (A) Comparison between the arrangement of the RABV N-protein in the crystal and docked in the cryo-electron density map from the top (i, view on the N-terminal domain) or the RNA binding groove (ii). Panels i and ii are not at the same scale. (B) Comparison of the residues interacting with the phosphate backbone (light green) of the viral genome between RABV and VSV. Positively charged residues that do not interact with the phosphate backbone but protrude into the RNA binding groove are depicted in dark green. (C) Comparison of the arrangement of N-proteins when docked into the CryoEM reconstructions of the RABV and VSV ribonucleoprotein complex (RNP). The N-monomers used for alignment of the different N-protein arrangements are connected using black arrows. EM: electron microscopy.

Viruses 2020, 12, 959

3. M-Protein Structure and Function

As observed for the N-protein, the sequence conservation of the 20–25 kDa M-protein between the Lagos bat virus (LBV, genus Lyssavirus) and VSV is very low (<10%), whilst the structural conservation is substantial (RMSD of 139 equivalent Cα is 3.1 Å) [3] (Figure 3). The majority of the M-protein consists of a compact, globular structure dominated by a four-stranded β-sheet on one side and two α helices and a small, two-stranded β-sheet on the other [3,10,11]. The N-terminus of the protein (aa) 1–47 in LBV, aa 1–57 in VSV) is mostly unresolved in the crystal structure, except residues 30–37 (LBV) and 41–52 (VSV), which are buried in a hydrophobic pocket of the globular domain of the neighbouring protein [3]. This interaction leads to the formation of non-covalently linked, linear polymers [3].

![Figure 3](https://i.imgur.com/yourimageurl.png)

Figure 3. Comparison of the structure of the Lagos bat virus (LBV) (pdb 2w2s) and VSV (pdb 2w2r) M-proteins [3]. Residues involved in the interaction between the M-proteins are depicted in cyan on the N-terminal fragment and in light blue on the C-terminus.

The M-protein is a multifunctional, central player in the formation of rhabdovirus virions. It is responsible for the condensation of the nucleocapsid [12], which produces the final shape found in the virion [12,13]. The M-protein is indispensable for viruses to form and bud [14], and it interacts with the lipid membrane, G-protein and N-protein. The interaction with the membrane is mediated by electrostatic interactions [11,15,16]. The forces required for the budding process are hypothesised to be generated by M- and G-proteins in a so-called “push and pull” model [17,18]. A special role in budding has been assigned to the “late domain” motif (PPPY aa 24–27 VSV, PPEY aa 35–38 LBV) at the M-protein N-terminus. This motif has been shown to interact with Nedd4 and its ubiquitination is important for efficient budding [19–24]. Furthermore, an interaction of the first 10 N-terminal amino acids with dynamin is essential for virus assembly [25] and the N-terminus of the M-protein is also responsible for the interaction with the N-protein [26] and lipid membranes [27,28]. Apart from its roles in virus assembly, the VSV M-protein suppresses host mRNA transcription and translation by interfering with the export of mRNAs from the nucleus [10,29–31] and the phosphorylation of transcription factors [32–35]. Additionally, the VSV M-protein has been shown to affect immunoproteasome formation via interaction with LMP2 [36].

4. P-Protein Structure and Function

The P-protein (34 kD) is a central (co-)factor in the replication cycle of rhabdoviruses. It can be divided into four domains. These are the N-terminal domain ($P_{\text{NTD}}$, aa 1–90, RABV), which is disordered but adopts defined conformations upon interaction with N or L [37–42]; the central domain ($P_{\text{CED}}$, aa 91–130, RABV); the C-terminal intrinsically disordered domain ($P_{\text{CID}}$, aa 131–195, RABV); and the C-terminal domain ($P_{\text{CD}}$, aa 195–296, RABV). The atomic structures of $P_{\text{CED}}$ (pdb: 2fqm [43], VSV; pdb: 3l32 [44], RABV), which mediates the dimerisation of the P-protein, and $P_{\text{CD}}$ (pdb: 2k47 [45], VSV; pdb: 1vyi [46], RABV), which is important for interacting with the N-RNA complex and parts of the VSV $P_{\text{NTD}}$ with N [37], have been resolved. P-protein forms a parallel dimer with an elongated shape, in which $P_{\text{CED}}$ acts as a central core, from which four flexible structural elements emerge [47,48]. Within the cell and the virus, P is present in different N-terminally truncated forms [49–51], which are generated by alternative initiation sites, and different phosphorylation states [52,53]. Essential functions of the P-protein in the viral life cycle are numerous. The P-protein prevents the self-assembly and RNA-binding of the N-protein (monomeric N-protein not bound to RNA: $N^0$), with one N-protein bound to one or two P-proteins via residues in $P_{\text{NTD}}$ [54–59], thereby ensuring that the N-protein specifically binds to viral RNA but not viral mRNAs or cellular RNAs. The P-protein is also the essential but catalytically inactive cofactor of L-protein. $P_{\text{NTD}}$ is the domain involved in the interaction with the L-protein [40,41,60,61]. This interaction is not essential for the enzymatic function of the L-protein, but rather for the generation of longer RNA molecules [62]. Binding to the N-RNA complex is mediated by $P_{\text{CD}}$ [63,64]. The RABV P-protein is also an antagonist of the interferon system. The P-protein of RABV interferes with the phosphorylation of IRF3 [65], retains activated STAT1 and STAT2 in the cytoplasm [50,66] and interacts with TRIM19 [67–69]. Interactions of the RABV P-protein with additional cellular proteins have been described [39,70–73] but are beyond the scope of this review.

5. L-Protein Structure and Function

The L-protein is the enzymatically active component of the transcriptase and replicase complex (reviewed in [74]), with a molecular weight of 242 kD. The enzymatic activities of the L-protein include an RNA-dependent RNA-polymerase (RdRp, aa 35–865, positions are given for VSV) [6], a GDP polyribonucleotidyl transferase (aa 866–1334) [75], a methyltransferase (aa 1598–1892) [76–79] and it also acts as a polyadenylase [80]. It can be divided into three catalytic and two structural domains. The RdRp and the capping domain form the core of the structure. In the absence of P, the remaining three domains—connector domain, methyltransferase domain and C-terminal domain—have no fixed position regarding the core of the structure [6,40,41,81]. The P-protein segments that bind to the L-protein that can be structurally resolved are aa 49–56 (interaction with the C-terminal domain), 82–89 (binding between the C-terminal domain and RdRp) and 94–105 (binding between the connector and C-terminal domains) for VSV [40]. For RABV, a 37 aa segment of P-protein, ending with Glu87, could be identified as being bound to the connector, the RdRp and the C-terminal domain of L-protein [41].

6. Assembly of the RNP

During replication, newly generated virus genomes are coated with N-protein derived from $N^0$-P complexes, with $N^0$ referring to the monomeric N-protein. It has been shown that $P_{\text{NTD}}$ interacts with $N^0$ via hydrophobic residues and that the interaction interface is not located in the RNA binding groove [37]. The nucleocapsid is formed during replication. Subsequently, unbound M-, P- and L-proteins associate with the newly formed nucleocapsid to generate an RNP that is not yet condensed, as observed for the RNP inside virions. RNPs are transferred to the plasma membrane, where they are recruited to membrane microdomains containing M- and G-proteins at the budding sites [82–85]. Here, the RNP becomes condensed into a tight helix and is stabilised due to the activity of the M-protein. In vitro, it has been demonstrated that the exposure of an N-RNA complex to low pH and low ionic strength is sufficient for the formation of an RNP shape, as observed in the virion [86]. The 5′ end

of the genome is located in the cone of the RNP [4]. The “pushing” action of the M-protein, which is the major budding factor, and the “pulling” action of the G-protein result in the budding of virus particles from the membrane [14,17,87]. VSV RNPs containing the genomic, negative-sense strand are preferentially packaged due to the trailer sequence at the 5′ end [88,89], while RABV packages both the + and – RNAs indiscriminately. The presence of 98% of negative-sense genomes in RABV particles is a result of the highly biased replication that favours the generation of negative-sense, genomic RNAs. To efficiently bud from the cell, rhabdoviruses rely on proteins of the ESCRT complex, which interact with the late-domain present in the N-terminus of the M-protein [22,90].

## 7. Morphology of the RNP in Virions

In the virion, the RNP is condensed into a left-handed helix [4,91] consisting of an inner helical ring of the N-RNA complex and an outer helical ring consisting of M-proteins. The N-terminal domain of the N-protein interacts with the M-protein, whilst the C-terminal domain is located towards the centre of the virion. The RNA-binding cavity of the N-protein, containing the viral genome, is oriented towards the RNP tip. Interactions between the N- and C-terminally protruding extensions of the N-protein connect neighbouring N-proteins along the helical turn. Due to the changes in the angle towards each other and the increased distance between the N-proteins in the CryoEM maps when compared to the arrangement in the crystal, additional stabilising interactions, as observed in the atomic structures of the N-protein of RABV and VSV between the N-protein C-terminal domains, should not be relevant for neighbouring N-proteins on the same helical turn in the RNP trunk [4,91]. However, for VSV, it has been proposed that these interactions are important contributors for the formation of the RNP tip, and it has been shown that mutagenesis of the involved residues (Ile237, Arg309, Tyr324 and Glu419) results in the formation of N-rings with an increased diameter [4]. Even though the structural similarity between RABV and VSV M- and N-proteins is striking, the architecture of the RNP of these two members of the family *Rhabdoviridae* differs substantially (Figure 4). Connections between the helical turns are established between N- and M-proteins in the RNP reconstruction of RABV, and between N- and M-proteins in the RNP reconstruction of VSV. Therefore, the M-protein forms a 3D lattice on the outside of the N-protein-RNA helix in VSV, whereas in RABV, only interactions between neighbouring M-proteins on the same helical turn can be observed. For the VSV M-protein, the CryoEM model suggests that the M-protein N-terminus, which is not resolved in the crystal structure, is responsible for all interactions with neighbouring M- and N-proteins. The resolution of the RABV RNP reconstruction does not allow for unambiguously assigning a certain position of the crystal structure in the CryoEM density map, but still implies that the N-terminus of the M-protein (or parts thereof) mediates the interaction with the N-protein on the neighbouring helical turn. An additional architectural difference is the N-protein helix orientation relative to the central axis of the virion. This angle is 63° in VSV and 44° in RABV. Furthermore, the helical turns in the RABV RNP are 20 Å further apart than in VSV. For VSV, the number of N-proteins per helical turn in the trunk of the RNP was determined to be 37.5. N-proteins are assembled in such a manner that the N-proteins of one helical turn are exactly positioned between N-proteins of the following turn, causing the helix to repeat itself every second turn. This repeating pattern could not be demonstrated in the reconstruction of the RABV RNP, where the arrangement of neighbouring helical turns suggests a repeat every 4–6 turns.

Figure 4. Comparison of the electron density maps of the RNP of RABV (EMD-4995) and VSV (EMD-1663). The viewing direction is indicated above each panel, where: (A) a view from the side, (B) a view from the membrane and (C) a view towards the RNA binding groove. The distance between the helical turns, as well as the deviation of the helical turns from the central axis of the virion, are indicated. The location of the genome is indicated by a black dot or a black dotted line. The N-protein structures (pdb 2gtt (RABV), 2gic (VSV)) are docked in the corresponding densities and shown in purple. The VSV M-protein structures (2w2r, blue) are docked in the orientation, as proposed by [4], whilst the location, but not the orientation, of the RABV M-protein C-terminal domain is depicted as a blue density. The proposed location of the M-protein N-terminus is indicated by red stars. Dashed lines in the middle panel indicate the step to the corresponding M-protein on the next helical turn.

8. Conclusions

Rhabdoviruses possess a characteristic RNP of considerable size, given the small RNA genome. Its architecture, consisting of building blocks that are highly conserved on the structural level but quite divergent at the amino acid level is comparable between VSV and RABV. However, despite the highly homologous building block structure, significant differences can also be observed, especially regarding the helical arrangement, helical repeat and interactions between helical turns. Therefore, additional research efforts are warranted to generate higher-resolution RNP structures to be able to clearly assign and functionally define the M-protein N-terminus, RNA-N-protein interactions inside the virion and the location and amount of L- and P-proteins as minor components of the RNP.

Author Contributions: A.A.H., C.R. and K.-K.C. wrote the manuscript and C.R. generated the figures. All authors have read and agreed to the published version of the manuscript.

Funding: A.A.H. and K.-K.C. were funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) – Project-ID 369799452 – TRR237 A12, Project-ID 118803580 - SFB 870 Z1, and CO 260/5-1. The data was recorded at the EM Facility of the Vienna BioCenter Core Facilities GmbH (VBCF), which is a member of the Vienna BioCenter (VBC), Austria. The authors thank Thomas Heuser and Harald Kotisch for their support during data acquisition. Open Access Funding has been provided by the University of Veterinary Medicine Vienna. Panels in Figures 2–4 were generated with UCSF Chimera.

Conflicts of Interest: The authors declare no conflict of interest.

## References

1. Baquero, E.; Albertini, A.A.; Gaudin, Y. Recent mechanistic and structural insights on class III viral fusion glycoproteins. *Curr. Opin. Struct. Biol.* **2015**, *33*, 52–60. [CrossRef] [PubMed]
2. Luo, M.; Green, T.J.; Zhang, X.; Tsao, J.; Qiu, S. Conserved characteristics of the rhabdovirus nucleoprotein. *Virus Res.* **2007**, *129*, 246–251. [CrossRef]
3. Graham, S.C.; Assenberg, R.; Delmas, O.; Verma, A.; Gholami, A.; Talbi, C.; Owens, R.J.; Stuart, D.I.; Grimes, J.M.; Bourhy, H. Rhabdovirus Matrix Protein Structures Reveal a Novel Mode of Self-Association. *PLoS Pathog.* **2008**, *4*, e1000251. [CrossRef] [PubMed]
4. Ge, P.; Tsao, J.; Schein, S.; Green, T.J.; Luo, M.; Zhou, Z.H. Cryo-EM Model of the Bullet-Shaped Vesicular Stomatitis Virus. *Science* **2010**, *327*, 689–693. [CrossRef]
5. Thomas, D.; Newcomb, W.W.; Brown, J.C.; Wall, J.S.; Hainfeld, J.F.; Trus, B.L.; Steven, A.C. Mass and molecular composition of vesicular stomatitis virus: A scanning transmission electron microscopy analysis. *J. Virol.* **1985**, *54*, 598–607. [CrossRef] [PubMed]
6. Liang, B.; Li, Z.; Jenni, S.; Rahmeh, A.A.; Morin, B.M.; Grant, T.; Grigorieff, N.; Harrison, S.C.; Whelan, S.P.J. Structure of the L Protein of Vesicular Stomatitis Virus from Electron Cryomicroscopy. *Cell* **2015**, *162*, 314–327. [CrossRef]
7. Kremer, J.R.; Mastronarde, D.N.; McIntosh, J.R. Computer visualization of three-dimensional image data using IMOD. *J. Struct. Biol.* **1996**, *116*, 71–76. [CrossRef]
8. Green, T.J.; Zhang, X.; Wertz, G.W.; Luo, M. Structure of the vesicular stomatitis virus nucleoprotein-RNA complex. *Science* **2006**, *313*, 357–360. [CrossRef]
9. Albertini, A.A.V.; Wernimont, A.K.; Muziol, T.; Ravelli, R.B.G.; Clapier, C.R.; Schoehn, G.; Weissenhorn, W.; Ruigrok, R.W.H. Crystal structure of the rabies virus nucleoprotein-RNA complex. *Science* **2006**, *313*, 360–363. [CrossRef]
10. Quan, B.; Seo, H.S.; Blobel, G.; Ren, Y. Vesiculoviral matrix (M) protein occupies nucleic acid binding site at nucleoporin pair (Rae1•Nup98). *Proc. Natl. Acad. Sci. USA* **2014**, *111*, 9127–9132. [CrossRef]
11. Gaudier, M.; Gaudin, Y.; Knossow, M. Crystal structure of vesicular stomatitis virus matrix protein. *EMBO J.* **2002**, *21*, 2886–2892. [CrossRef]
12. Newcomb, W.W.; Brown, J.C. Role of the vesicular stomatitis virus matrix protein in maintaining the viral nucleocapsid in the condensed form found in native virions. *J. Virol.* **1981**, *39*, 295–299. [CrossRef] [PubMed]
13. Newcomb, W.W.; Tobin, G.J.; McGowan, J.J.; Brown, J.C. In vitro reassembly of vesicular stomatitis virus skeletons. *J. Virol.* **1982**, *41*, 1055–1062. [CrossRef] [PubMed]

14. Mebatsion, T.; Weiland, F.; Conzelmann, K.-K. Matrix Protein of Rabies Virus Is Responsible for the Assembly and Budding of Bullet-Shaped Particles and Interacts with the Transmembrane Spike Glycoprotein, *G. J. Virol.* 1999, **73**, 242–250. [CrossRef]

15. Zakowski, J.J.; Wagner, R.R.; Petri, W.A. Role of Matrix Protein in Assembling the Membrane of Vesicular Stomatitis Virus: Reconstitution of Matrix Protein with Negatively Charged Phospholipid Vesicles. *Biochemistry* 1981, **20**, 3902–3907. [CrossRef] [PubMed]

16. Chong, L.D.; Rose, J.K. Membrane association of functional vesicular stomatitis virus matrix protein in vivo. *J. Virol.* 1993, **67**, 407–414. [CrossRef]

17. Mebatsion, T.; König, M.; Conzelmann, K.-K. Budding of Rabies Virus Particles in the Absence of the Spike Glycoprotein. *Cell* 1996, **84**, 941–951. [CrossRef]

18. Robison, C.S.; Whitt, M.A. The Membrane-Proximal Stem Region of Vesicular Stomatitis Virus G Protein Confers Efficient Virus Assembly. *J. Virol.* 2000, **74**, 2239–2246. [CrossRef]

19. Harty, R.N.; Paragas, J.; Sudol, M.; Palese, P. A Proline-Rich Motif within the Matrix Protein of Vesicular Stomatitis Virus and Rabies Virus Interacts with WW Domains of Cellular Proteins: Implications for Viral Budding. *J. Virol.* 1999, **73**, 2921–2929. [CrossRef]

20. Wirblich, C.; Tan, G.S.; Papaneri, A.; Godlewski, P.J.; Orenstein, J.M.; Harty, R.N.; Schnell, M.J. PPEY Motif within the Rabies Virus (RV) Matrix Protein Is Essential for Efficient Virion Release and RV Pathogenicity. *J. Virol.* 2008, **82**, 9730–9738. [CrossRef]

21. Jayakar, H.R.; Murti, K.G.; Whitt, M.A. Mutations in the PPPY Motif of Vesicular Stomatitis Virus Matrix Protein Reduce Virus Budding by Inhibiting a Late Step in Virion Release. *J. Virol.* 2000, **74**, 9818–9827. [CrossRef] [PubMed]

22. Taylor, G.M.; Hanson, P.I.; Kielian, M. Ubiquitin Depletion and Dominant-Negative VPS4 Inhibit Rhabdovirus Budding without Affecting Alphavirus Budding. *J. Virol.* 2007, **81**, 13631–13639. [CrossRef] [PubMed]

23. Harty, R.N.; Brown, M.E.; McGettigan, J.P.; Wang, G.; Jayakar, H.R.; Huibregtse, J.M.; Whitt, M.A.; Schnell, M.J. Rhabdoviruses and the Cellular Ubiquitin-Proteasome System: A Budding Interaction. *J. Virol.* 2001, **75**, 10623–10629. [CrossRef] [PubMed]

24. Soh, T.K.; Whelan, S.P.J. Tracking the Fate of Genetically Distinct Vesicular Stomatitis Virus Matrix Proteins Highlights the Role for Late Domains in Assembly. *J. Virol.* 2015, **89**, 11750–11760. [CrossRef]

25. Raux, H.; Obiang, L.; Richard, N.; Harper, F.; Blondel, D.; Gaudin, Y. The Matrix Protein of Vesicular Stomatitis Virus Binds Dynamin for Efficient Viral Assembly. *J. Virol.* 2010, **84**, 12609–12618. [CrossRef]

26. Chong, L.D.; Rose, J.K. Interactions of normal and mutant vesicular stomatitis virus matrix proteins with the plasma membrane and nucleocapsids. *J. Virol.* 1994, **68**, 441–447. [CrossRef]

27. Lenard, J.; Vanderoef, R. Localization of the membrane-associated region of vesicular stomatitis virus M protein at the N terminus, using the hydrophobic, photoreactive probe 125I-TID. *J. Virol.* 1990, **64**, 3486–3491. [CrossRef]

28. Ye, Z.; Sun, W.; Suryanarayana, K.; Justice, P.; Robinson, D.; Wagner, R.R. Membrane-binding domains and cytopathogenesis of the matrix protein of vesicular stomatitis virus. *J. Virol.* 1994, **68**, 7386–7396. [CrossRef]

29. Von Kobbe, C.; Van Deursen, J.M.A.; Rodrigues, J.P.; Sitterlin, D.; Bachi, A.; Wu, X.; Wilm, M.; Carmo-Fonseca, M.; Izaurralde, E. Vesicular stomatitis virus matrix protein inhibits host cell gene expression by targeting the nucleoporin Nup98. *Mol. Cell* 2000, **6**, 1243–1252. [CrossRef]

30. Faria, P.A.; Chakraborty, P.; Levay, A.; Barber, G.N.; Ezelle, H.J.; Enninga, J.; Arana, C.; Van Deursen, J.; Fontoura, B.M.A. VSV disrupts the Rae1/mrnp41 mRNA nuclear export pathway. *Mol. Cell* 2005, **17**, 93–102. [CrossRef]

31. Rajani, K.R.; Pettit Kneller, E.L.; McKenzie, M.O.; Horita, D.A.; Chou, J.W.; Lyles, D.S. Complexes of Vesicular Stomatitis Virus Matrix Protein with Host Rae1 and Nup98 Involved in Inhibition of Host Transcription. *PLoS Pathog.* 2012, **8**, e1002929. [CrossRef] [PubMed]

32. Welnowska, E.; Castelló, A.; Moral, P.; Carrasco, L. Translation of mRNAs from Vesicular Stomatitis Virus and Vaccinia Virus Is Differentially Blocked in Cells with Depletion of eIF4GI and/or eIF4GII. *J. Mol. Biol.* 2009, **394**, 506–521. [CrossRef] [PubMed]

33. Connor, J.H.; Lyles, D.S. Vesicular Stomatitis Virus Infection Alters the eIF4F Translation Initiation Complex and Causes Dephosphorylation of the eIF4E Binding Protein 4E-BP1. *J. Virol.* 2002, **76**, 10177–10187. [CrossRef] [PubMed]

34. Connor, J.H.; Lyles, D.S. Inhibition of host and viral translation during vesicular stomatitis virus infection: eIF2 is responsible for the inhibition of viral but not host translation. *J. Biol. Chem.* 2005, 280, 13512–13519. [CrossRef](https://doi.org/10.1074/jbc.M506080200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16102906)

35. Neidermyer, W.J.; Whelan, S.P.J. Global analysis of polysome-associated mRNA in vesicular stomatitis virus infected cells. *PLoS Pathog.* 2019, 15, e1007875. [CrossRef](https://doi.org/10.1371/journal.ppat.1007875)

36. Beilstein, F.; Obiang, L.; Raux, H.; Gaudin, Y. Characterization of the Interaction between the Matrix Protein of Vesicular Stomatitis Virus and the Immunoproteasome Subunit LMP2. *J. Virol.* 2015, 89, 11019–11029. [CrossRef](https://doi.org/10.1128/JVI.00528-15)

37. Leyrat, C.; Yabukarski, F.; Tarbouriech, N.; Ribeiro, E.A.; Jensen, M.R.; Blackledge, M.; Ruigrok, R.W.H.; Jamin, M.; Jamin, M. Structure of the vesicular stomatitis virus N0-P complex. *PLoS Pathog.* 2011, 7, e1002248. [CrossRef](https://doi.org/10.1371/journal.ppat.1002248)

38. Leyrat, C.; Jensen, M.R.; Ribeiro, E.A.; Gérard, F.C.A.; Ruigrok, R.W.H.; Blackledge, M.; Jamin, M. The N0-binding region of the vesicular stomatitis virus phosphoprotein is globally disordered but contains transient α-helices. *Protein Sci.* 2011, 20, 542–556. [CrossRef](https://doi.org/10.1002/pro.586)

39. Jespersen, N.E.; Leyrat, C.; Gérard, F.C.; Bourhis, J.M.; Blondel, D.; Jamin, M.; Barbar, E. The LC8-RavP ensemble Structure Evinces A Role for LC8 in Regulating Lyssavirus Polymerase Functionality. *J. Mol. Biol.* 2019, 431, 4959–4977. [CrossRef](https://doi.org/10.1016/j.jmb.2019.06.014)

40. Jenni, S.; Bloyet, L.-M.; Diaz-Avalos, R.; Liang, B.; Whelan, S.P.J.; Grigorieff, N.; Harrison, S.C. Structure of the Vesicular Stomatitis Virus L Protein in Complex with Its Phosphoprotein Cofactor. *Cell Rep.* 2020, 30, 53–60. [CrossRef](https://doi.org/10.1016/j.celrep.2019.12.061)

41. Horwitz, J.A.; Jenni, S.; Harrison, S.C.; Whelan, S.P.J. Structure of a rabies virus polymerase complex from electron cryo-microscopy. *Proc. Natl. Acad. Sci. USA* 2020, 117, 2099–2107. [CrossRef](https://doi.org/10.1073/pnas.1917012117) [PubMed](https://pubmed.ncbi.nlm.nih.gov/31959806)

42. Gould, J.R.; Qiu, S.; Shang, Q.; Ogino, T.; Prevelige, P.E.; Petit, C.M.; Green, T.J. The Connector Domain of Vesicular Stomatitis Virus Large Protein Interacts with the Viral Phosphoprotein. *J. Virol.* 2020, 94. [CrossRef](https://doi.org/10.1128/JVI.01968-19) [PubMed](https://pubmed.ncbi.nlm.nih.gov/32820228)

43. Ding, H.; Green, T.J.; Lu, S.; Luo, M. Crystal Structure of the Oligomerization Domain of the Phosphoprotein of Vesicular Stomatitis Virus. *J. Virol.* 2006, 80, 2808–2814. [CrossRef](https://doi.org/10.1128/JVI.80.6.2808-2814.2006) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16537626)

44. Ivanov, I.; Crépin, T.; Jamin, M.; Ruigrok, R.W.H. Structure of the Dimerization Domain of the Rabies Virus Phosphoprotein. *J. Virol.* 2010, 84, 3707–3710. [CrossRef](https://doi.org/10.1128/JVI.02351-09) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20100846)

45. Ribeiro, E.A.; Favier, A.; Gerard, F.C.A.; Leyrat, C.; Brutscher, B.; Blondel, D.; Ruigrok, R.W.H.; Blackledge, M.; Jamin, M. Solution Structure of the C-Terminal Nucleoprotein-RNA Binding Domain of the Vesicular Stomatitis Virus Phosphoprotein. *J. Mol. Biol.* 2008, 382, 525–538. [CrossRef](https://doi.org/10.1016/j.jmb.2008.07.023) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18675308)

46. Mavrakis, M.; McCarthy, A.A.; Roche, S.; Blondel, D.; Ruigrok, R.W.H. Structure and function of the C-terminal domain of the polymerase cofactor of rabies virus. *J. Mol. Biol.* 2004, 343, 819–831. [CrossRef](https://doi.org/10.1016/j.jmb.2004.08.052) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15381370)

47. Gerard, F.C.A.; Ribeiro, E.D.A.; Albertini, A.A.V.; Gutsche, I.; Zaccai, G.; Ruigrok, R.W.H.; Jamin, M. Unphosphorylated Rhabdoviridae phosphoproteins form elongated dimers in solution. *Biochemistry* 2007, 46, 10328–10338. [CrossRef](https://doi.org/10.1021/bi701081q)

48. Leyrat, C.; Ribeiro, E.A.; Gérard, F.C.A.; Ivanov, I.; Ruigrok, R.W.H.; Jamin, M. Structure, interactions with host cell and functions of rhabdovirus phosphoprotein. *Future Virol.* 2011, 6, 465–481. [CrossRef](https://doi.org/10.2217/fvl.11.36)

49. Chenik, M.; Chebli, K.; Blondel, D. Translation initiation at alternate in-frame AUG codons in the rabies virus phosphoprotein mRNA is mediated by a ribosomal leaky scanning mechanism. *J. Virol.* 1995, 69, 707–712. [CrossRef](https://doi.org/10.1128/JVI.69.2.707-712.1995)

50. Kretzschmar, E.; Peluso, R.; Schnell, M.J.; Whitt, M.A.; Rose, J.K. Normal replication of vesicular stomatitis virus without C proteins. *Virology* 1996, 216, 309–316. [CrossRef](https://doi.org/10.1006/viro.1996.0099)

51. Spiropoulou, C.F.; Nichol, S.T. A small highly basic protein is encoded in overlapping frame within the P gene of vesicular stomatitis virus. *J. Virol.* 1993, 67, 3103–3110. [CrossRef](https://doi.org/10.1128/JVI.67.5.3103-3110.1993) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8481625)

52. Barik, S.; Banerjee, A.K. Sequential phosphorylation of the phosphoprotein of vesicular stomatitis virus by cellular and viral protein kinases is essential for transcription activation. *J. Virol.* 1992, 66, 1109–1118. [CrossRef](https://doi.org/10.1128/JVI.66.2.1109-1118.1992) [PubMed](https://pubmed.ncbi.nlm.nih.gov/1732083)

53. Gupta, A.K.; Blondel, D.; Choudhary, S.; Banerjee, A.K. The Phosphoprotein of Rabies Virus Is Phosphorylated by a Unique Cellular Protein Kinase and Specific Isomers of Protein Kinase, C. *J. Virol.* 2000, 74, 91–98. [CrossRef](https://doi.org/10.1128/JVI.74.1.91-98.2000) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10590110)

54. Mavrakis, M.; Iseni, F.; Mazza, C.; Schoehn, G.; Ebel, C.; Gentzel, M.; Franz, T.; Ruigrok, R.W.H. Isolation and characterisation of the rabies virus N°-P complex produced in insect cells. *Virology* **2003**, *305*, 406–414. [CrossRef]

55. Howard, M.; Wertz, G. Vesicular stomatitis virus RNA replication: A role for the NS protein. *J. Gen. Virol.* **1989**, *70*, 2683–2694. [CrossRef]

56. Peluso, R.W.; Moyer, S.A. Viral proteins required for the in Vitro replication of vesicular stomatitis virus defective interfering particle genome RNA. *Virology* **1988**, *162*, 369–376. [CrossRef]

57. Chen, M.; Ogino, T.; Banerjee, A.K. Interaction of Vesicular Stomatitis Virus P and N Proteins: Identification of Two Overlapping Domains at the N Terminus of P That Are Involved in N0-P Complex Formation and Encapsulation of Viral Genome RNA. *J. Virol.* **2007**, *81*, 13478–13485. [CrossRef]

58. Jacob, Y.; Real, E.; Tordo, N. Functional Interaction Map of Lyssavirus Phosphoprotein: Identification of the Minimal Transcription Domains. *J. Virol.* **2001**, *75*, 9613–9622. [CrossRef]

59. Yabukarski, F.; Leyrat, C.; Martinez, N.; Communie, G.; Ivanov, I.; Ribeiro, E.A.; Buisson, M.; Gerard, F.C.; Bourhis, J.M.; Jensen, M.R.; et al. Ensemble Structure of the Highly Flexible Complex Formed between Vesicular Stomatitis Virus Unassembled Nucleoprotein and its Phosphoprotein Chaperone. *J. Mol. Biol.* **2016**, *428*, 2671–2694. [CrossRef]

60. Chenik, M.; Schnell, M.; Conzelmann, K.K.; Blondel, D. Mapping the Interacting Domains between the Rabies Virus Polymerase and Phosphoprotein. *J. Virol.* **1998**, *72*, 1925–1930. [CrossRef]

61. Castel, G.; Chtéoui, M.; Caignard, G.; Préhaud, C.; Méhouas, S.; Réal, E.; Jallet, C.; Jacob, Y.; Ruigrok, R.W.H.; Tordo, N. Peptides That Mimic the Amino-Terminal End of the Rabies Virus Phosphoprotein Have Antiviral Activity. *J. Virol.* **2009**, *83*, 10808–10820. [CrossRef] [PubMed]

62. De, B.P.; Banerjee, A.K. Requirements and functions of vesicular stomatitis virus L and NS proteins in the transcription process in vitro. *Biochem. Biophys. Res. Commun.* **1985**, *126*, 40–49. [CrossRef]

63. Chenik, M.; Chebli, K.; Gaudin, Y.; Blondel, D. In vivo interaction of rabies virus phosphoprotein (P) and nucleoprotein (N): Existence of two N-binding sites on P protein. *J. Gen. Virol.* **1994**, *75*, 2889–2896. [CrossRef] [PubMed]

64. Fu, Z.F.; Zheng, Y.; Wunner, W.H.; Koprowski, H.; Dietzschold, B. Both the N- and the C-terminal domains of the nominal phosphoprotein of rabies virus are involved in binding to the nucleoprotein. *Virology* **1994**, *200*, 590–597. [CrossRef]

65. Brzózka, K.; Finke, S.; Conzelmann, K.-K. Identification of the Rabies Virus Alpha/Beta Interferon Antagonist: Phosphoprotein P Interferes with Phosphorylation of Interferon Regulatory Factor 3. *J. Virol.* **2005**, *79*, 7673–7681. [CrossRef]

66. Brzózka, K.; Finke, S.; Conzelmann, K.-K. Inhibition of Interferon Signaling by Rabies Virus Phosphoprotein P: Activation-Dependent Binding of STAT1 and STAT2. *J. Virol.* **2006**, *80*, 2675–2683. [CrossRef]

67. Chelbi-Alix, M.K.; Vidy, A.; El Bougrini, J.; Blondel, D. Rabies viral mechanisms to escape the IFN system: The viral protein P interferes with IRF-3, Stat1, and PML nuclear bodies. *J. Interferon Cytokine Res.* **2006**, *26*, 271–280. [CrossRef]

68. Blondel, D.; Regad, T.; Poisson, N.; Pavie, B.; Harper, F.; Pandolfi, P.P.; De Thé, H.; Chelbi-Alix, M.K. Rabies virus P and small P products interact directly with PML and reorganize PML nuclear bodies. *Oncogene* **2002**, *21*, 7957–7970. [CrossRef]

69. Blondel, D.; Kheddache, S.; Lahaye, X.; Dianoux, L.; Chelbi-Alix, M.K. Resistance to Rabies Virus Infection Conferred by the PMLIV Isoform. *J. Virol.* **2010**, *84*, 10719–10726. [CrossRef]

70. Li, Y.; Dong, W.; Shi, Y.; Deng, F.; Chen, X.; Wan, C.; Zhou, M.; Zhao, L.; Fu, Z.F.; Peng, G. Rabies virus phosphoprotein interacts with ribosomal protein L9 and affects rabies virus replication. *Virology* **2016**, *488*, 216–224. [CrossRef]

71. Kammouni, W.; Wood, H.; Saleh, A.; Appolinario, C.M.; Fernyhough, P.; Jackson, A.C. Rabies virus phosphoprotein interacts with mitochondrial Complex I and induces mitochondrial dysfunction and oxidative stress. *J. Neurovirol.* **2015**, *21*, 370–382. [CrossRef]

72. Fouquet, B.; Nikolic, J.; Larrous, F.; Bourhy, H.; Wirblich, C.; Lagaudrière-Gesbert, C.; Blondel, D. Focal Adhesion Kinase Is Involved in Rabies Virus Infection through Its Interaction with Viral Phosphoprotein, P. *J. Virol.* **2015**, *89*, 1640–1651. [CrossRef] [PubMed]

73. Raux, H.; Flamand, A.; Blondel, D. Interaction of the Rabies Virus P Protein with the LC8 Dynein Light Chain. *J. Virol.* **2000**, *74*, 10212–10216. [CrossRef] [PubMed]

74. Ogino, T.; Green, T.J. Transcriptional control and mRNA capping by the GDP polyribonucleotidyltransferase domain of the rabies virus large protein. *Viruses* **2019**, *11*, 504. [CrossRef](https://doi.org/10.3390/v11060504) [PubMed](https://pubmed.ncbi.nlm.nih.gov/31181778)

75. Ogino, T.; Banerjee, A.K. Unconventional Mechanism of mRNA Capping by the RNA-Dependent RNA Polymerase of Vesicular Stomatitis Virus. *Mol. Cell* **2007**, *25*, 85–97. [CrossRef](https://doi.org/10.1016/j.molcel.2006.11.026) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17218278)

76. Rahmeh, A.A.; Li, J.; Kranzusch, P.J.; Whelan, S.P.J. Ribose 2′-O Methylation of the Vesicular Stomatitis Virus mRNA Cap Precedes and Facilitates Subsequent Guanine-N-7 Methylation by the Large Polymerase Protein. *J. Virol.* **2009**, *83*, 11043–11050. [CrossRef](https://doi.org/10.1128/JVI.00688-09) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19542433)

77. Grdzelishvili, V.Z.; Smallwood, S.; Tower, D.; Hall, R.L.; Hunt, D.M.; Moyer, S.A. A Single Amino Acid Change in the L-Polymerase Protein of Vesicular Stomatitis Virus Completely Abolishes Viral mRNA Cap Methylation. *J. Virol.* **2005**, *79*, 7327–7337. [CrossRef](https://doi.org/10.1128/JVI.79.12.7327-7337.2005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15917309)

78. Li, J.; Fontaine-Rodriguez, E.C.; Whelan, S.P.J. Amino Acid Residues within Conserved Domain VI of the Vesicular Stomatitis Virus Large Polymerase Protein Essential for mRNA Cap Methyltransferase Activity. *J. Virol.* **2005**, *79*, 13373–13384. [CrossRef](https://doi.org/10.1128/JVI.79.21.13373-13384.2005)

79. Galloway, S.E.; Wertz, G.W. S-Adenosyl Homocysteine-Induced Hyperpolyadenylation of Vesicular Stomatitis Virus mRNA Requires the Methyltransferase Activity of L Protein. *J. Virol.* **2008**, *82*, 12280–12290. [CrossRef](https://doi.org/10.1128/JVI.01288-08)

80. Hunt, D.M.; Hutchinson, K.L. Amino acid changes in the l polymerase protein of vesicular stomatitis virus which confer aberrant polyadenylation and temperature-sensitive phenotypes. *Virology* **1993**, *193*, 786–793. [CrossRef](https://doi.org/10.1006/viro.1993.1166)

81. Rahmeh, A.A.; Schenk, A.D.; Danek, E.I.; Kranzusch, P.J.; Liang, B.; Walz, T.; Whelan, S.P.J. Molecular architecture of the vesicular stomatitis virus RNA polymerase. *Proc. Natl. Acad. Sci. USA* **2010**, *107*, 20075–20080. [CrossRef](https://doi.org/10.1073/pnas.1007664107) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876108)

82. Brown, E.L.; Lyles, D.S. Organization of the Vesicular Stomatitis Virus Glycoprotein into Membrane Microdomains Occurs Independently of Intracellular Viral Components. *J. Virol.* **2003**, *77*, 3985–3992. [CrossRef](https://doi.org/10.1128/JVI.77.7.3985-3992.2003)

83. Brown, E.L.; Lyles, D.S. A novel method for analysis of membrane microdomains: Vesicular stomatitis virus glycoprotein microdomains change in size during infection, and those outside of budding sites resemble sites of virus budding. *Virology* **2003**, *310*, 343–358. [CrossRef](https://doi.org/10.1016/S0042-6822(03)00042-8)

84. Lyles, D.S. Assembly and Budding of Negative-Strand RNA Viruses. In *Advances in Virus Research*; Academic Press: Cambridge, MA, USA, 2013.

85. Swinteck, B.D.; Lyles, D.S. Plasma Membrane Microdomains Containing Vesicular Stomatitis Virus M Protein Are Separate from Microdomains Containing G Protein and Nucleocapsids. *J. Virol.* **2008**, *82*, 5536–5547. [CrossRef](https://doi.org/10.1128/JVI.00148-08) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385420)

86. Desfosses, A.; Ribeiro, E.A.; Schoehn, G.; Blondel, D.; Guilligay, D.; Jamin, M.; Ruigrok, R.W.H.; Gutsche, I. Self-organization of the vesicular stomatitis virus nucleocapsid into a bullet shape. *Nat. Commun.* **2013**, *4*, 1429. [CrossRef](https://doi.org/10.1038/ncomms2432) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23422686)

87. Cadd, T.L.; Skoging, U.; Liljeström, P. Budding of enveloped viruses from the plasma membrane. *BioEssays* **1997**, *19*, 993–1000. [CrossRef](https://doi.org/10.1002/bies.950191203)

88. Whelan, S.P.J.; Wertz, G.W. The 5′ Terminal Trailer Region of Vesicular Stomatitis Virus Contains a Position-Dependent cis-Acting Signal for Assembly of RNA into Infectious Particles. *J. Virol.* **1999**, *73*, 307–315. [CrossRef](https://doi.org/10.1128/JVI.73.1.307-315.1999)

89. Finke, S.; Conzelmann, K.K. Ambisense gene expression from recombinant rabies virus: Random packaging of positive- and negative-strand ribonucleoprotein complexes into rabies virions. *J. Virol.* **1997**, *71*, 7281–7288. [CrossRef](https://doi.org/10.1128/JVI.71.9.7281-7288.1997)

90. Han, Z.; Lu, J.; Liu, Y.; Davis, B.; Lee, M.S.; Olson, M.A.; Ruthel, G.; Freedman, B.D.; Schnell, M.J.; Wrobel, J.E.; et al. Small-Molecule Probes Targeting the Viral PPxY-Host Nedd4 Interface Block Egress of a Broad Range of RNA Viruses. *J. Virol.* **2014**, *88*, 7294–7306. [CrossRef](https://doi.org/10.1128/JVI.00166-14)

91. Riedel, C.; Vasishtan, D.; Pražák, V.; Ghanem, A.; Conzelmann, K.-K.; Rümenapf, T. Cryo EM structure of the rabies virus ribonucleoprotein complex. *Sci. Rep.* **2019**, *9*, 9639. [CrossRef](https://doi.org/10.1038/s41598-019-46108-z)
