
Innate immune responses to highly pathogenic coronaviruses and other significant respiratory viral infections

Namal P. Liyanage${}^{1*}$, Hanna Salah${}^{1}$, Brianna Sisson${}^{1}$, Rajni Kant Shukla${}^{1}$, Yasasvi Wijewantha${}^{1}$, Don Hayes${}^{2}$, Thorsten Demberg${}^{3}$

${}^{1}$Department of Microbial Infection and Immunity, College of Medicine, The Ohio State University, United States, ${}^{2}$Division of Pulmonary Medicine, Department of Pediatrics, Cincinnati Children's Hospital Medical Center, United States, ${}^{3}$Marker Therapeutics, United States

**Submitted to Journal:**  
Frontiers in Immunology

**Specialty Section:**  
Viral Immunology

**ISSN:**  
1664-3224

**Article type:**  
Review Article

**Received on:**  
04 May 2020

**Accepted on:**  
22 Jul 2020

**Provisional PDF published on:**  
22 Jul 2020

**Frontiers website link:**  
[www.frontiersin.org](http://www.frontiersin.org)

**Citation:**  
Liyanage NP, Salah H, Sisson B, Kant_shukla R, Wijewantha Y, Hayes D and Demberg T (2020) Innate immune responses to highly pathogenic coronaviruses and other significant respiratory viral infections. Front. Immunol. 11:1979. doi: 10.3389/fimmu.2020.01979

**Copyright statement:**  
© 2020 Liyanage, Salah, Sisson, Kant_shukla, Wijewantha, Hayes and Demberg. This is an open-access article distributed under the terms of the [Creative Commons Attribution License (CC BY)](https://creativecommons.org/licenses/by/4.0/). The use, distribution and reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.

This Provisional PDF corresponds to the article as it appeared upon acceptance, after peer-review. Fully formatted PDF and full text (HTML) versions will be made available soon.

Frontiers in Immunology | www.frontiersin.org

Innate immune responses to highly pathogenic coronaviruses and other significant respiratory viral infections

Hanna Salah$^{1\dagger}$, Brianna Sisson$^{1\dagger}$, Rajni Kant Shukla$^{1}$, Yasasvi Wijewantha$^{1}$, Nicholas T Funderburg$^{2}$, Zihai Li$^{3}$, Don Hayes, Jr.$^{4,5}$, Thorsten Demberg$^{6}$, Namal P.M. Liyanage*$^{1,7}$

$^{1}$ Department of Microbial Infection and Immunity, College of Medicine, Ohio State University, Columbus, OH, USA, $^{2}$ School of Health and Rehabilitation Sciences, Division of Medical Laboratory Science, Ohio State University Columbus, OH, USA, $^{3}$ Pelotonia Institute for Immuno-Oncology, The James Comprehensive Cancer Center, The Ohio State University, Columbus, OH $^{4}$ Division of Pulmonary Medicine, Cincinnati Children’s Hospital Medical Center $^{5}$, Department of Pediatrics, University of Cincinnati College of Medicine, Cincinnati, OH, USA, $^{6}$ Marker Therapeutics Inc., 3200 Southwest Freeway, Suite 2240, Houston, TX, USA.

$^{7}$ Department of Veterinary Biosciences, College of Veterinary Medicine, Ohio State University, Columbus, OH,

*Corresponding author

Email: namal.liyanage@osumc.edu

$\dagger$ These authors contributed equally to this work

Running title: Innate immune responses in viral respiratory infections

Abstract.

The new pandemic virus SARS-CoV-2 emerged in China and spread around the world in less than three months, infecting millions of people, and causing countries to shutdown public life and businesses. Nearly all nations were unprepared for this pandemic with healthcare systems stretched to their limits due to the lack of an effective vaccine and treatment. Infection with SARS-CoV-2 can lead to Coronavirus disease 2019 (COVID-19). COVID-19 is respiratory disease that can result in a cytokine storm with stark differences in morbidity and mortality between younger and older patient populations. Details regarding mechanisms of viral entry via the respiratory system and immune system correlates of protection or pathogenesis have not been fully elucidated. Here, we provide an overview of the innate immune responses in the lung to the coronaviruses MERS-CoV, SARS-CoV and SARS-CoV-2. This review provides insight into key innate immune mechanisms that will aid in the development of therapeutics and preventive vaccines for SARS-CoV-2 infection.

Introduction.

Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) reportedly emerged at a live animal market in the Chinese city of Wuhan is currently causing a pandemic and negatively affecting global health[1-3]. There are more than 11 million confirmed SARS-CoV-2 infections with an mortality rate that widely varies by country and region[4]. Even in industrialized countries, SARS-CoV-2 led healthcare systems approach the brink of collapse by overwhelming their capacity and straining resources. Governments and local leaders ordered the shutdown of cities, regions, countries leading to massive disruptions in the local and global economy. Unlike previous Coronavirus (CoV) infections, the rapid global spread, high transmission rate, longer incubation

time, and disease severity of SARS-CoV-2 requires a better understanding of the epidemiology

and immunopathogenesis of this viral outbreak in order to learn from this experience and to

manage future pandemics.

SARS-CoV-2 is a highly pathogenic CoV[5] (case-fatality rate of 3.6–3.8%) [4, 6] that is related

to Severe Acute Respiratory Syndrome CoV (SARS-CoV) (case-fatality rate of 14%–15%) and

the Middle East Respiratory Syndrome CoV (MERS-CoV) (case-fatality rate of 34.4%), see also

Table 1 [7, 8]. SARS-CoV, SARS-CoV-2 and MERS-CoV target the lower respiratory system,

causing respiratory illnesses, including severe pneumonia, acute lung injury (ALI) and acute

respiratory distress syndrome (ARDS) [9, 10]. SARS-CoV-2 infection results in higher

hospitalization rates in the elderly (>65) and persons with pre-existing conditions including

hypertension, diabetes and obesity compared to rates among younger populations without pre-

existing conditions (Table 1) [11, 12]. In addition to an age disparity, males with COVID-19 appear

to have higher risk for worse outcomes and death [12, 13]. Epidemiological research of the SARS

and MERS infections also showed that males had a higher mortality rate than females [13–15].

While SARS-CoV-2 is a novel coronavirus, several important insights have already been made

about its basic mode of transmission. Virus particles are inhaled in respiratory droplets expelled

from the airways of infected individuals. Angiotensin-converting enzyme 2 (ACE2), expressed on

the ciliated bronchial cells, endothelial cells, and type I and II alveolar cells, is the host receptor

for cell entry into the respiratory tract by both SARS-CoV-2 and SARS-CoV (Table 1) [16–19].

The spike protein (S) of CoV is responsible for the entry of the virus into the target cell (Figure 1)

[16, 20]. ACE2 is a type I transmembrane metallo-carboxypeptidase that plays a vital role in the

Renin-Angiotensin System (RAS) [21, 22], which in turn is critical in maintaining blood pressure

homeostasis as well as fluid and salt balance in mammals. ACE2 is found in vascular endothelial

cells, in the renal tubular epithelium, and in Leydig cells of the testes[23]. Studies have shown that

ACE 2 is expressed in gastrointestinal (GI) tissues, making it a potential site for harboring SARS-

CoV[24]. This may be one of the reasons for GI pathology reported in some patients with COVID-

19 and viral shedding in stool. In contrast, MERS-CoV uses dipeptidyl-peptidase 4 (DPP4) as an

entry receptor, which is expressed on endothelial cells and some epithelial tissues (Table 1) [25,

26].

Accumulating data suggest that the innate anti-viral response and adaptive immunity may

contribute to a cytokine storm leading to systemic hyperinflammation and exacerbation of the

disease in patients with a) co-morbidities b) older than 65 years of age c) of the male sex. The

exact role of the innate immune system in disease pathogenesis and prevention between the sexes

and the impact of age is not fully elucidated. In addition, the potential dysregulation of the innate

immune response by SARS-CoV and SARS-CoV-2 is not completely understood which warrants

further research.

Respiratory epithelial cell and resident innate immune cell responses to respiratory viral

infections.

The cells of the airway epithelium are the first line of defense, providing a mechanical barrier

(mucociliary escalator) that expels particles and pathogens via cilia, mucus, and induced

coughing[27, 28]. This barrier includes cells of the pulmonary epithelium and tissue-resident

macrophages and dendritic cells (DCs). The macrophages and DCs express pattern recognition

receptors (PRRs) that can detect molecules from pathogens (Pathogen-Associated Molecular

Patterns - PAMPs) or molecules released from damaged cells (Damage or Danger-Associated

Molecular Patterns - DAMPs) [28-30]. In the lung, there are two populations of macrophages,

93 alveolar and interstitial macrophages[31]. In addition to these macrophages, DCs play a vital role
94 in facilitating the host defenses against respiratory diseases [32-34]. DCs can be divided into
95 plasmacytoid (pDC) and myeloid types (mDC) [35-37]. Macrophages and the two DC subtypes
96 trigger antiviral responses by generating a substantial amount of type I interferon and these cells
97 play important roles in immune surveillance in the airways and the distal lung[38-43]. During
98 steady state, the DC density in lung associated tissue declines from the trachea to the alveoli[44]
99 while the representation of cells in macrophage compartment seems more complex [45]. If a virus
100 infects airway epithelial cells, the viral RNA would be sensed via intrinsic innate receptors,
101 including RIG-1, MDA 5, NLRP 3 inflammasome, and the RNA sensing TLRs 7 and 8. In the case
102 of influenza A infection, triggering the PRRs causes a strong induction of type I interferon (IFN)
103 in epithelial cells[46, 47]. In other viral infections, such as Respiratory Syncytial Virus (RSV),
104 alveolar macrophages are the predominant source of type I IFNs[31]. Furthermore, respiratory
105 epithelial cells and lung macrophages are capable of secreting a broad range of chemokines like
106 IL-8, Macrophage inflammatory protein -1 (MIP-1), RANTES and cytokines including TNF-α,
107 IL-6, IL-1β that influence the types of immune cells being recruited to the area in response to acute
108 viral infections [48, 49].
109 Macrophages, depending on their polarization status of either M1 or M2, and in a similar way as
110 airway epithelial cells, can further elicit a Th1 or Th2 response[28, 31, 49]. In the case of influenza
111 virus infection, the magnitude of epithelial cell response can be proportional to the amount of virus
112 which result in paracrine induction of IFN λ[46]. Not only can airway epithelial cells produce a
113 large array of cytokines/chemokines in response to an acute viral infection, but, depending on the
114 magnitude of PRR engagement and the combination of PAMPs and DAMPs triggered, these
115 epithelial cells can modulate the type of chemokines/cytokines produced and influence the influx

116 of innate and adaptive immune cells[28, 30]. The response to different viral infections is generally
117 similar, however, the response can be tailored in timing, magnitude and the induced gene
118 signatures in response to each virus[50]. Unlike RSV and MERS-CoV, which both productively
119 infect alveolar macrophages[51, 52], seasonal influenza and SARS-CoV usually lead to non-
120 productive infections in these cells [53]. In addition, SARS-CoV infection of primary monocytes
121 yielded little virus, likely due to the suppressive effects of IFN-α[54]. Thus, the initial cell type(s)
122 involved in propagating a viral infection intensifies the complexity of the immune response.
123 Another key factor that determines the magnitude of the immune response is the induction and rate
124 of cell death. Although related, MERS-CoV induces widespread cell death when compared to
125 SARS-CoV in human bronchial epithelial cell cultures[55]. However, the SARS-CoV open
126 reading frame 3a (ORF3a) protein can induce necrotic cell death in a variety of cell lines[56].
127 The same ORF3a protein can activate the NLRP3 inflammasome, leading to activation of NF-κB
128 and elevated secretion of active IL-1β in cell culture[57].
129 Cytokine profiles of macrophages activated by SARS-CoV and MERS-CoV are different[52].
130 Nonetheless, both MERS-CoV and SARS-CoV, in human epithelial cell and fibroblast culture,
131 show a delay (24-30 hrs post-infection) in the induction of proinflammatory cytokines[58], with
132 slightly different cytokine/chemokine profiles. This delay in cytokine induction was confirmed in
133 another study using the same epithelial cell lines[59] as well as in human alveolar type II cells[60].
134 In both cell lines and primary alveolar type II cells, SARS-CoV induced IFN-β, IFN-λ, CXCL10,
135 CXCL11, IL-6, IP-10, and TNF-α[59, 60]. MERS-CoV did not induce IFN-β but induced higher
136 level of IL-8 transcript in cell culture. However, no difference in IL-8 production was observed
137 between SARS-CoV and MERS-CoV at 48 hrs post-infection[58]. This was confirmed in-vivo

138 using a non-human primate model comparing the immune responses to SARS-CoV infection
139 between young adult cynomolgus macaques (3-5 yrs) and older macaques (10-19 yrs) [61].
140 Interestingly, the high induction of IL-8 was observed on a transcript level in the older animals,
141 while the younger once showed higher levels of IFN-β transcript[61]. In all animals, the expression
142 of IFN-β was inversely correlated with the pathology score, supporting the role of IFN-β in
143 controlling disease severity[61] and introducing a potential area of research to define age disparity
144 observed in patients infected with SARS-CoV-2. Both older age and male sex are important factors
145 associated with high mortality of SARS-CoV and SARS-CoV-2 infection[62, 63] Many viruses
146 have evolved to disrupt and subvert the immune responses. A common virus that is well known to
147 affect the lower airway and counteract the immune response is RSV[49, 64]. The RSV genome
148 encodes nonstructural proteins NS1 and NS2 that can block type 1 IFN production and signaling
149 in cell cultures[64]. Similar to RSV, the Measle virus V protein blocks IFN-α and β signaling by
150 inhibiting Stat1 and Stat2 signaling in cell lines[65], whereas MERS-CoV M protein also
151 suppresses type 1 IFN by inactivating IRF3[66], leading to the low expression of IFN-β.
152 In contrast to reports in epithelial cell lines or primary alveolar type II cell culture and observations
153 in non-human primates, SARS-CoV nucleocapsid (N) protein and membrane (M) protein, as well
154 as nsp1, can suppress IFN response via various mechanisms in cell lines[67-69]. To bridge the
155 dichotomy of inhibition of IFN signaling in cell lines, and the **IFN expression in vivo, cells**
156 recruited by the infection need to be considered as a potential source. As previously discussed,
157 infected epithelial cells via paracrine signaling to neighboring cells and resident macrophages,
158 secrete chemokines and cytokines to attract other immune cells.
159 In general, monocytes/macrophages are recruited by CCL3 (MIP-1a), CCL2 (MCP-1) and
160 neutrophils are recruited by IL-8 (CXCL8), CXCL2 and CXCL5 chemokines, all of which can be

161 secreted by airway epithelial cells[49, 70, 71]. Both monocytes and neutrophils are also recruited
162 by complement fragment (anaphylatoxin) C5a (Figure 1). Both Influenza and SARS virus can
163 induce acute lung injury (ALI) which is accompanied by high levels of C5a, leading to the influx
164 and activation of innate immune cells[72] (Figure 1).

165 Serum samples and lung tissue of SARS patients showed high-level expression of CXCL10 (IP-
166 10), which is also found to be induced by SARS-CoV in the epithelial cell line Calu-3[73].
167 Significant neutrophil, macrophage and CD8T-cell infiltration can be found in the lung of SARS
168 patients by immunohistochemistry [74-76]. In addition to post-mortem lung histology analysis in
169 patients with SARS-CoV, experiments using Rhesus macaques infected with SARS-CoV found
170 monocyte and macrophage recruitment. The accumulation of pathogenic inflammatory monocyte-
171 macrophages (IMMs) was also observed in a SARS-CoV mouse model. The accumulation of
172 IMMs resulted in heightened lung inflammatory cytokine/chemokine levels, extensive vascular
173 leakage, and impaired virus-specific T cell responses[77].

174 A strong infiltration of CD68 and Mac387 positive monocytes/macrophages were found in the
175 human and animals lung samples [78, 79]. Macrophages further stimulate fibroblasts to deposit
176 extracellular matrix leading to pulmonary fibrosis[80], which was also observed in patients who
177 recovered from SARS[81, 82]. Autopsy samples acquired from patients with SARS-CoV-2
178 patients contained viral nucleocapsid protein (NP) positive CD68+ macrophages in the capillaries
179 of the spleen and in lymph nodes, indicating that SARS-CoV-2 might migrate into the spleen and
180 lymph nodes through macrophages. This study also found that CD169+ macrophages appear to
181 mediate SARS-CoV-2 into these tissue sites, contributing to virus dissemination [83]. Similar to
182 SARS-CoV-2, SARS viral particles and genomic sequences were detected in monocytes,
183 macrophages as well as within different organs of SARS patients[84]. SARS-CoV was shown to

184 infect both immature and mature human monocyte-derived DCs by electron microscopy and
185 immunofluorescence. The detection of negative strands of SARS-CoV RNA in DCs suggests viral
186 replication, but no increase in viral RNA was observed[85]. As mentioned above, there was no
187 perceivable increase to SARS-CoV replication in primary monocytes [54]. Another study looked
188 at SARS-CoV and MERS-CoV replication in human macrophages, human monocyte-derived
189 macrophages, and dendritic cells (MDDCs) and found that both viruses replicated poorly. MERS-
190 CoV induced IFN-λ1, CXCL 10, and MxA mRNAs in both macrophages and MDDCs, however,
191 SARS-CoV was unable to induce such responses[86]. Interestingly, depletion of inflammatory
192 monocyte-macrophages in the mouse model partially protected from lethal SARS infection[87].
193 These data suggest that monocytes, macrophages and dendritic cells have essential roles in CoV
194 infection. The severity of disease and the response to these viruses seems to be dependent on the
195 induced cytokine/chemokine profiles and the amplification of the immune response by the
196 recruited cells. Growing evidence of dysregulation of an innate anti-viral response originates from
197 studies using clinical samples[88] and murine models [77, 89, 90].

198 In addition to dysregulated cellular responses, the complement system may play an important role
199 in SARS-CoV-2 infection (Figure 1). Evidence comes from SARS infected patients who had lower
200 levels of mannan binding lectin (MBL) in serum compared to healthy controls[91]. The SARS
201 patients with a higher frequency of MBL gene polymorphisms were associated with lower serum
202 levels or deficiency of MBL[91]. It is still unknown if this is also true for COVID-19 patients,
203 which requires further investigation. In cell culture experiments SARS-CoV was able to bind and
204 activate the complement cascade and block viral infection[91]. Preliminary findings in a limited
205 number COVID-19 patients found significant deposits of the membrane attack complex (MAC),
206 C4d and MBL-associated serine protease (MASP) 2 in the microvasculature indicating sustained,

207 systemic activation[92]. The SARS-CoV-2 spike protein was co-localized with C4d and MAC[92].

208 In a non-peer reviewed publication by Gao et al., MERS-CoV, SARS-CoV and SARS-CoV-2 N

209 protein are able to bind to MASP-2 leading to enhanced complement activation[93] (Figure 1). In

210 a later phase of the infection, the complement system might be also triggered via antibodies bound

211 to the virus (Classic activation pathway, Figure 1). This excessive complement activation might

212 play a role in multi organ damage in severe COVID-19 cases[94]. In a MERS-CoV mouse model

213 the blockade of the C5a-C5aR axis alleviated not only lung damage but also spleen damage [95].

214 Mice treated with a monoclonal antibody to C5a showed reduced lung infiltration of CD68+ cells

215 and significantly lower cytokine levels of IL-1β, TNF-α, INF-γ and IL-12 [95]. Complement

216 blockade might be an important way to curb part of the immune dysregulation associated with

217 COVID-19. Overall, we need to look closer at the role of the complement system, the recruited

218 innate immune cells and their combined role in pathogenesis, viral clearance and the eventual

219 resolution of the infection.

220

221 The role of Neutrophils.

222 The most abundant leukocytes, neutrophils, play a critical role in clearing viral infections.

223 Neutrophils, attracted by chemokines/cytokines released by tissue-resident macrophages and DCs,

224 swarm to the site of infection. They recognize and release bioactive compounds, including

225 cytokines, chemokines and ROS, as well as NOS in the very early phase of the infection[96, 97].

226 Neutrophils modulate other innate and adaptive immune responses via cytokine/chemokine release

227 and cell death and, therefore, can ameliorate or exacerbate disease progression. Neutrophils

228 infiltrate tissues infected by CoV, including SARS-CoV, Rat coronavirus (rCoV), and Mouse

229 Hepatitis Virus (MHV). A high neutrophil count in the blood of SARS patients at the time of

230 hospital admission is associated with a poor prognosis[98, 99]. Liu et al. suggested that patients
231 with SARS-CoV-2 pneumonia can be stratified by neutrophil to lymphocyte ratio (NLR) and age
232 [93]. Patients older than 50 years of age and having an NLR ≥ 3.13 had more severe illness, so
233 rapid access to the intensive care unit is required [100, 101]. Experiments in mice showed that
234 SARS-CoV disease severity in older mice correlated with increased pulmonary inflammation and
235 influx of neutrophils[102, 103]. Infection of rats with rCoV could lead to neutrophil infiltrating
236 into the respiratory tract early after inoculation, followed by the recruitment of macrophages and
237 lymphocytes[104]. Infection of mice with a neurotropic murine CoV (MHV-JHM) showed
238 infiltration of neutrophils into the brain as early as the first day after inoculation, which then
239 promoted the recruitment of other types of inflammatory cells into the brain, likely through the
240 loss of the blood-brain barrier integrity[105]. Gene expression analysis in experimentally infected
241 rhesus macaques with MERS-CoV revealed the recruitment of neutrophils into infected lung
242 tissue[106, 107].
243 Angiotensin-converting enzyme inhibitors (ACE-Is) could serve as a potential risk for fatal
244 COVID-19 through the up-regulation of ACE2[108] and may provide a direct linkage to
245 neutrophils and disease progression. Investigators found that ACE2 modulates IL-17-mediated
246 neutrophil influx by impacting STAT3 activity[109]. Animal models used to study the
247 pathogenesis of SARS-CoV-2 have revealed important roles of neutrophils in infection and
248 confirmed findings observed in patients. A new aspect in SARS-CoV-2 infection is the potential
249 role of neutrophil extracellular traps (NETs). The process of NET formation is a specific type of
250 cell death that can be triggered under inflammatory conditions[110, 111], such as GM-CSF+C5a,
251 IL-8, IFN-α+C5a or other TLR response mediators; all conditions present in severe SARS-CoV-
252 2 infection[110, 111]. The NET formation has been observed in COVID-19 patients and may

253 contribute to thrombotic complications in COVID-19 patients [112, 113]. Microvascular injury
254 and thrombosis have been reported in COVID-19 patients, increasing the likelihood that neutrophil
255 NET formation might play a role[92, 114, 115]. NET formation was reported to be involved in
256 clot formation and thrombosis and can further increase inflammation [110, 111]. Therefore
257 neutrophils can attract a second wave of immune cells to the site of infection by
258 cytokine/chemokine secretion as well as via NETosis[116, 117], which included monocytes and
259 natural killer cells. On the other hand aggregated NETs were reported to limit inflammation by
260 degrading cytokines and chemokines and disrupting neutrophil recruitment and activation[118].
261 Despite the presence of neutrophils in SARS-CoV-2-infected tissues, their role in the clearance
262 and/or immunopathology of the viral infection remains unclear. Future studies on the responses of
263 neutrophils to SARS-CoV-2-infection or infected cells in vitro may elucidate the role of
264 neutrophils in the pathogenesis of respiratory CoV infections.
265 
266 The potential role of Natural Killer cells.
267 Natural Killer (NK) cells are a heterogenic immune cell subset that acts promptly to combat viral
268 infections. They produce significant amounts of IFN-γ, kill virus-infected cells, provide direct
269 support to other innate immune cells, and aid in the adaptive immune response to counter viruses.
270 NK cells express activating receptors that detect viral antigens, enabling the destruction of infected
271 cells[119-122]. Lectin-like receptor CD94 and killer immunoglobulin-like receptors, such as
272 CD158b, regulate the function of NK cells. A study of 221 patients with SARS explored the
273 relationship of the number of NK cells and the expression level of their immunoglobulin-like
274 receptor CD158b in the peripheral blood to the severity of SARS[123]. The overall count of NK
275 cells and CD158+ NK cells and the percentage of CD158+ NK cells in patients with SARS were

276 significantly lower than counts in healthy subjects[123]. A separate study that analyzed
277 lymphocytes and lymphocyte subsets in a cohort of 38 patients with SARS observed reduced NK
278 cell counts in 21 patients (55%) [124]. Clinical reports reveal that children appear to have a milder
279 form of SARS-CoV-2, with peripheral blood lymphocyte levels remaining in the normal range,
280 suggesting less immune dysfunction following the disease [125]. This could be attributed to healthy
281 children expressing lymphocytes, especially NK cells, in a greater quantity compared to healthy
282 adults [126]. Interestingly, previous studies found rapid and significant restoration of lymphocyte
283 subsets including, NK cells, in peripheral blood in patients recovering from the initial stages of
284 SARS infection [127]. Although the primary mechanism for the decrease in NK cells and other
285 subsets during disease onset remains unknown, their contribution to SARS-CoV-2 needs further
286 study especially from a treatment perspective.
287 The contribution by the innate lymphocytes.
288 Innate lymphoid cells (ILCs) are a family of innate immune cells that include ILC1, ILC2 and
289 ILC3. Although ILC2 facilitates lung repair after injury, the role of ILCs during respiratory viral
290 infection is not clearly defined [128]. Evidence for the potential involvement of ILC2 cells in the
291 lung during viral infection was reported in a murine model [129]. This study found a rapid
292 accumulation of ILC2 cells in the lung after an influenza virus infection, however their initial
293 contribution to exacerbation of the disease was limited [129]. A recent study identified an
294 interaction between ACE2-expressing SARS-CoV-2 target cells and ILCs in the colon [130]. Thus,
295 elucidating the role of ILC subsets will be important in understanding the pathogenesis of SARS,
296 SARS-CoV-2 and MERS infections.
297

298 Roles of interferons in coronavirus infection.

299 There is distinct evidence indicating an important role of IFNs in SARS and other CoV

300 infections[75, 131]. The sera of patients with SARS revealed the presence of high levels of IL-1,

301 IL-6, INFγ, CCL2, CXCL10, and IL-8 and products of interferon stimulated genes[132, 133]. High

302 expression levels of ISGs such as CD58, IFNAR1, and IFNGR1 and IFN-stimulated chemokines

303 CXCL10 and CCL2 were observed in another cohort of SARS patients and were correlated with

304 the severity of pathogenesis[134]. Significant upregulation of CXCL10 gene expression was

305 observed in the severe phase of patients who died from SARS. This data is corroborated by studies

306 in patients with MERS that found upregulation of CXCL10 in the serum of patients who developed

307 pneumonia[132]. CXCL10 and INFα were also correlated with severity of disease[133].

308 The importance of IFN signaling in response to CoV infection has been well demonstrated in

309 several knockout mouse models. Type I, II, and III IFN signaling deficient mice have increased

310 susceptibility to mouse-adapted SARS-CoV strains[135, 136]. Studies using mice lacking the

311 IFNAR1 and IFNLR1 or STAT1 identified higher SARS-CoV replication in the lungs and delayed

312 virus clearance[137, 138]. Another study with MERS-CoV in mice expressing the human DPP4

313 receptor showed a role for the IFNAR1 in viral clearance and lung inflammation[139]. These

314 mouse models suggest an important role of IFN response for CoV clearance. This quick expanding

315 medical literature is very suggestive of an important role of IFN responses for CoV control and

316 clearance.

317

318 Innate immune evasion strategy of human coronaviruses.

319 Many viruses have evolved to disrupt and subvert the immune response. RSV counteracts the

320 immune response[49, 64]; as discussed earlier, the RSV genome encodes nonstructural proteins

321 (NS1 and NS2) that are able to block type 1 IFN production and signaling in cell cultures[64].
322 Similar to RSV, the Measle virus V protein blocks IFN-α and β signaling by inhibiting Stat1 and
323 Stat2 signaling in cell culture lines[65]. CoVs have developed several ways to escape from innate
324 immune pressure. MERS-CoV M protein suppresses type 1 IFN by blocking the IRF3
325 activation[66], explaining the low expression of IFN-β. In various cell lines, SARS-CoV
326 nucleocapsid (N) protein, membrane (M) protein, as well as nsp1, were reported to suppress IFN
327 response [67, 69, 140]. The nucleocapsid protein (N) of SARS-CoV interferes with the function
328 of IRF3. **Although it does not form a complex with RIG-I or MDA5, RNA binding activity at the**
329 **initial recognition stage of viral RNA potentially contributes to immune evasion[140, 141].**

330 Aside from the HCoV, structural proteins, accessory, and nonstructural proteins (nsp) are involved
331 in innate immunity modulation. In both SARS-CoV and MERS-CoV, host mRNA endonucleolytic
332 cleavage is promoted by nsp1 protein, which modifies capped non-viral RNAs[142, 143]. Nsp1 in
333 SARS-CoV prevents host mRNA translation by interacting with the 40S subunit of the ribosome;
334 in turn, transcription and translation of viral RNA is given preference over the host mRNA[142].
335 Another study found that additional SARS-CoV nsp1 residues interfered with IFN-dependent
336 signaling[144]. IFN production is affected by nsp3 proteins in SARS-CoV and MERS-CoV. These
337 proteins have both papain-like protease (PLpro) and a PLP2 domain, and the PLpro domains in
338 both SARS-CoV and MERS-CoV downregulate mRNA levels of CCL5, INFβ, CXCL10, and
339 other pro-inflammatory cytokines[145]. The suppression of IFN responses by SARS-CoV PLpro
340 is due to the inhibition of phosphorylation of IFN-regulatory factor 3 (IRF3) and its subsequent
341 translocation to the nucleus where it enhances IFN gene transcription[146]. MERS-CoV PLpro
342 also suppresses RIG-I and MDA5 and antagonizes IFN induction[145, 147]. In HCoV-229E and
343 SARS-CoV suppression of IFN responses, the key molecule is a ADP-ribose-1-monophosphatase

344 macrodomain encoded within nsp 3[148]. Accessory proteins are not key in viral replication;
345 however, in human CoV, this group of proteins are involved in diverse cellular signaling, including
346 cell proliferation, apoptosis, and IFN signaling[149]. By downregulating phosphorylation and
347 nuclear translocation of IRF 3, Open Reading Frame ORF3b and-6 interfere with IFNB synthesis
348 and prevent IFNB-induced activation of IFN-stimulated response element (ISRE) in the promoter
349 of ISG in SARS-CoV[141]. In cells transfected with ORF4a, -4b, and -5 of MERS-CoV, IFNB
350 promoter-driven luciferase activity is significantly reduced, and it may follow a similar pattern of
351 suppression of IRF 3 nuclear translocation[10]. Therefore, the suppression of signaling events in
352 infected immune and airway epithelial cells, as well as the magnitude of suppression due to
353 elevated expression levels of these accessory proteins, needs to be further elucidated to understand
354 delayed or hyperimmune responses and cytokine storm that occurs in CoV infection.
355 Summary.
356 In addition to revealing our unpreparedness of handling a worldwide pandemic by a viral infection,
357 COVID-19 exposed our lack of understanding of the pathogenesis of diseases as well as the host
358 immunity. The interaction if the host innate immune system and other factors including age, sex,
359 and pre-existing conditions need further investigation regarding disease severity and morbidity of
360 SARS/MERS and COVID-19. Disease severity and its related progression are further associated
361 with dysregulation of multiple components of both innate and adaptive immune responses leading
362 to a cytokine storm and severe pathology. For the development of the therapeutic intervention, it
363 is vital to elucidate the interplay among the different layers of the innate immune response and
364 their relation to the clinical factors associated with increased morbidity and mortality in CoV
365 infection. Investments in basic science research are needed to help elucidate the roles of different

immune cells, and their contribution to disease severity; it will pave the way to prevent or abrogate

CoV outbreaks and potentially new viruses.

Table 1: Comparison of Immune pathogenesis between highly pathogenic coronaviruses and other significant respiratory viral infections.

Figure 1. Potential Immune Pathogenesis of SARS-Cov-2. A. Replication cycle of SARS-CoV-2: Spike protein on the SARS-CoV-2 binds to angiotensin converting enzyme 2 (ACE2), a cell-surface protein. The virion releases its RNA. Some RNA is translated into proteins by the host cell's machinery. Proteins and RNA are assembled into a new virion in the Golgi and released. B. The innate and adaptive immune responses to Coronavirus (CoV) infection. I. Initiation of immune response via PAMPs/DAMPs. The host innate immune system detects CoV infections by using pattern recognition receptors (PRRs) to recognize pathogen-associated molecular patterns (PAMPs) and Damage (Danger)-Associated Molecular Patterns (DAMPs). II. Activation of T-cells and B-cells via cytokines and activation of the complement system. CoV infection leads to macrophages activation and release of inflammatory cytokines. This in turn activates T and B cells and promotes differentiation. Multiple different T cell subsets (i.e., Th1 and Th17) are involved, releasing cytokines for immune response amplification. III. Activation of Neutrophils (NET formation) Neutrophils, attracted by chemokines/cytokines swarm to the site of infection. Subsequently activated neutrophils undergo degranulation and NET formation releasing intracellular DAMPs, DNA, histones, neutrophil elastase that activate the PRRs of surrounding immune and non-immune cells to induce cytokine secretion. Neutrophils and neutrophil extracellular traps (NETs) drive necroinflammation in COVID-19. The extracellular DNA released

by NETs activates platelets and aggregated NETs provide a scaffold for binding of erythrocytes

and activated platelets that promote thrombus formation. IV. Dendritic Cell mediated activation

of T-cells. DCs present viral antigens to T-cells inducing activation. V. Cytokine and C5a led to

influx of immune cells. Secrete chemokines, cytokines and complement C5a attract immune cells.

C. Effects of CoV-mediated complement activation. SARS-CoV-2 has been shown to activate

the lectin (MBL) complement pathway. Antibodies (early stage IgM or at a later stage IgG) to the

virus can activate the classic complement pathway. Both pathways converge at C3. C3 can be

converted into C3a and C3b. C3b mediates pathogen opsonization and activates the conversion of

C5 into C5a and C5b. C5b mediates the formation of the membrane attack complex, which leads

to cell lysis. C3a and C5a promote immune cell recruitment to the site of infection.

|           | SARS-CoV          | MERS-CoV         | SARS-CoV-2        | Influenza virus (IV)                  | parainfluenza virus (PIV)               | Respiratory syncytial virus (RSV)       | Rhinovirus (RV)                                                                 |
|-----------|-------------------|------------------|-------------------|---------------------------------------|------------------------------------------|------------------------------------------|--------------------------------------------------------------------------------|
| Receptor/s| ACE 2 [150]       | DPP 4 [151]      | ACE 2 [152] DPP 4 [153] | α-2, 3 linkage and α-2,6 linkage [154] | α 2, 3-linked sialic acids [155]         | CX3CR1 [156]                              | ICAM-1, LDLR, and CDHR 3 [157]                                                   |
| Target cells | Multiple cell types in the lower respiratory tract were found to be infected, including type I alveolar epithelium, macrophages, and putative CD34+Oct-4+ stem/progenitor cells in human lungs [84, 158, 159] | Un-ciliated bronchial epithelial cells and type II pneumocytes [162] | Infect mostly human type I and type II pneumocytes and alveolar macrophages [164] | Club cells, ciliated cells, type I and type II alveolar cells [165] | Ciliated epithelial cells of the upper and lower respiratory tract [166] | The ciliated cells of the human airway epithelium are the main target, it also infects basal cells and immune cells, such as Macrophages, B cells CD4+ and CD8+ T cells. [167] | Upper and lower airways epithelial cell [168] |

|  | ciliated bronchial epithelial cells and type II pneumocytes [60,150] |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Mortality | 11% [169] | 34.4% [170] | 3-4% [171] | <0.1% [171] | Unusual in developed countries. preschool population in developing countries has considerable risk of HPIV-induced death. LRI causes 25 to 30% of the deaths in this age group and HPIV causes at least 10% of the LRI [172,173] | Children < 5 years - death uncommon, estimated at 100-500/year.Among US adults, an estimated 14,000 deaths/year. [174] | ---- |
| Effected age | While younger individuals below 18 years of age experience | While younger individuals below 18 years | Patients aged ≥ 60 years showed heavier | The influenza virus with highest sRIR was | Persons of any age [180] | The highest burden of RSV was observed in young infants aged 3-5 months, whereas the burden was also high | RV was more frequently detected in younger children and infants than in older children [183] |

| age experience mild- moderate clinical illness, elderly individuals exhibit worse outcomes after infection with MERS-CoV [176, 177] | mild-moderate clinical illness, elderly individuals exhibit worse outcomes after infection with SARS-CoV [175] | clinical manifestations, greater severity and longer disease courses compared with those aged < 60 years. [178] | A(H1N1) for young children, B for older children, A(H1N1) pdm 2009 for adults, and (A(H3N2) for the elderly. [179] | in those aged 12-20 months. [181] And certain adult populations. These include the elderly, persons with cardiopulmonary diseases, and immunocompromised hosts. [182] |
| --- | --- | --- | --- | --- |
| R0-the reproduction number* [184] | In the range of 2-4 [184] | Saudi Arabia: 0.45-0.98 (Only one study reported 1.9-3.9) South Korea: 2.5-8.1 and Less than 1 in later period or with control intervention [185] | Between 2 and 2.5 [171] | Between 1.28 and 1.8 [186] | 0.92-1.33 for RSV-A and 1.04-1.76 for RSV-B [187] | 1.2-1.83 [188] |

| Incubation period | mean, 5 days; range, 2 to 10 days [189] | 5 to 7 days; range, 2 to 14 days [191] | mean, 5 days; range, 2 to 14 days [190] | 2 days; range, 1 to 4 days [192] | 2 to 7 days [193] | 4 to 6 days [194] | mean, 1·9 days [192] |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Serial interval time (the time between successive cases) | Mean, in Singapore 8.4 days [195] | 12.6 to 14.6 days [185] | 5-6 days [196] | 3 days [171] | ———— | 3.2 days [187] | ———— |
| Comorbidities | Diabetes, other comorbidities (chronic obstructive pulmonary disease, cancer, cardiac disease), and age of 60 years or older [197] acute renal impairment and | Diabetes mellitus, hypertension, ischemic heart disease, congestive heart failure, end-stage renal disease and chronic kidney disease [199] | > 60 years and those with comorbid conditions, such as diabetes, hypertension and cardiovascular disease (CVD), [200-202] | Asthma; diabetes; heart, lung, and neurologic diseases; and pregnancy [203] | immunocompromised and elderly adults [165] | Older adults [204] adults hospitalized with cardiopulmonary infections [205] | Asthma, chronic medical conditions malignancies, or immunosuppression, [206-208] |

|  | proteinuria[198] |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Immune responses: |  |  |  |  |  |  |
| Macrophages | non-productive infections [86] | productive macrophages could contribute to viral spread, excessive inflammation and activation-induced lymphocytic cell death during SARS-CoV-2 infection. [209] | CD169+ non-productive infections More than 90% of resident AMs were lost in the first week after influenza, while the remaining cells had a necrotic phenotype. result in significant morbidity through several pathways, including facilitation of secondary bacterial pneumonia [210] | productive infections [211] | productive infections [51] one of the foremost and only sources of IFN-I, contributing to the establishment of an antiviral state in neighboring cells [42] | productive infections Rhinovirus replication in human macrophages causing activation and nuclear translocation of NF-kB, leading to TNF-a production [212] |
| Monocytes | SARS-CoV- | MDMs were permissive | Decreased | Influenza infection | Inefficient infection of Immature | Inefficient infection of | Airway epithelial cells display significant RV 16 replication |

|  | infected human monocytes produce chemokines that attract the migration of neutrophils, macrophages, and activated T lymphocytes [74, 76] | for MERS-CoV [213] | [101] | markedly inhibit the monocyte chemotactic response and depress the phagocytosis [214] | MDDCs and sub-optimal maturation [215] | Immature MDDCs and sub-optimal maturation [215] | in monocytic cells via an ICAM1-dependent mechanism [216] |
| --- | --- | --- | --- | --- | --- | --- | --- |
| DC | SARS-CoV-infection was abortive in MDDCs. [217] | Immature MDDCs were permissive for MERS-CoV infection, while mature MDDCs were not. [213] | Activated dendritic cells increased by both myeloid DCs (mDCs) and plasmacytoid DCs but only mDCs supported viral replication. [219] | IV was internalized Human Parainfluenza Virus Type 2 Vector induce DC maturation without viral replication/transcription [220] | Infected DCs can promote airway obstruction, enhance disease, and promote more severe allergic responses. -A low cDC1:cDC2 ratio correlates with enhanced disease severity. [221] | increase in type ImDCs decrease in anti-viral types mDCs following RV infection in asthmatics [222] |

| Neutrophils                                                                 | Significant correlation between MERS-CoV viral load and expression levels of neutrophil chemoattractant chemokines IL-8 (CXCL 8) [223] [224] | Activated neutrophils [218] | Increased neutrophil influx [225] | Increased neutrophil [226] | Neutrophil chemotaxis and phagocytosis are increased [227, 228] | Not defined |
|------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------|---------------------------|-----------------------------------|---------------------------|--------------------------------------------------------------------|--------------|
| T cells                                                                    | Lymphopenia [229] Efficiently Infects and kill Primary T Lymphocytes [230]                      | Lymphocytopenia [101] SARS-CoV-2 infects T lymphocytes [231]          | High levels of circulating virus-specific CD4+T cells to two viral internal proteins (nucleoprotein and matrix) in the first phase of infection are associated with subsequent development of severe | T cells are readily infected by the PIV. The capacity of the virus to regulate T-lymphocyte function may play an important role in the failure of the virus to induce lifelong immunity. [233] | Infection with RSV causes a dysregulated antiviral immune response with impaired T cell function as well as exaggerated inflammation via multiple mechanisms [234] | Rhinovirus has the unique ability to bypass antigen presentation and directly infect and activate human cells [235] |

|  |  |  |  | IAV infection. [232] |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| B cells | Lack of Peripheral Memory B Cell Responses in Recovered Patients with SARS [236] | The long-term persistence of antibodies in most patients might be explained by the MERS-CoV infection inducing long-lived memory B cells, which in turn form antibody-secreting plasma cells that are stored in the bone marrow until re-exposure to the same virus or | B cells response against SARS-CoV-2 are detected in the blood around 1 week after the onset of COVID-19 symptoms [238] | Activated B cells differentiates into a plasma blasts, their populations begin to expand rapidly in the lymph node medulla and secrete predominantly class-switched antibody, peaking between 7 to 14 days post-influenza infection [239, 240] |  | There is an increase in circulating B cells, including mature (CD19+CD5+) and precursor (CD19+CD10+) cells, in infants with RSV LRTI, and CD20+B cells and IgM+, IgG+, and IgA+ plasma cells are prominent in postmortem lung tissue from infants with fatal RSV bronchiolitis [241-243] | RVs enter and form viral replication centers in B lymphocytes and induce proliferation of B cells [244] |

|  |  | similar epitopes. [237] |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Antibodies | Neutralizing antibody responses, likely to the S protein, begin to develop by week 2, and most patients develop neutralizing antibodies by week 3 [245, 246] | The response to MERS-CoV generally occurs through antibody-mediated immunity [139] the neutralizing antibody titers at 34 months of infection in 86% of human serum samples were the same as those after 13 months of infection. [247] | Currently, polyclonal antibodies from recovered SARS-CoV-2-infected patients have been used to treat SARS-CoV-2 infection, but no SARS-CoV-2-specific neutralizing mAbs have been reported. [248] | Abs elicited against the HA globular domain during infection or vaccination usually are strain-specific, and they will hardly neutralize subsequent influenza virus strains (homosubtypic protection). [249] | Antibodies to the two surface glycoproteins, F and HN are neutralizing and serum and nasal antibody to either protein protects against PIV infection and ameliorates disease. [250, 251] | Maternally derived RSV neutralizing antibodies protect infants against RSV hospitalization, and when the infant has recurrent wheeze. However, high maternally derived RSV neutralizing antibody levels were associated with an increased risk of recurrent wheeze. [252] | After an RV infection, seroneutralizing antibody titer increase for about a year high preexisting neutralizing antibody titers have been associated with resistance re-infection. [253] |
| Cytokines | IFN-γ, IL-10, IL-1β, IL-6, and IL-12 | MCP-1, MIP-1α and IL-8 chemokines | IL-1, IL-6, L-2, IL-7, IL-10, G-CSF, IP-10, MCP-1, MIP- | IL-6 and chemokines CCL-2/MCP-1, | PIV serotypes differ in their kinetics of replication and cytokine secretion in | IL-1, IL-6, IL-10, and CCL 5 are increased, while IL-10 and | Different RV strains can induce different patterns cytokines and chemokine [263] |

| increases IL-4 decreases IL-2 levels increased, while others argued that it decreased. [229, 254-256] | and the cytokine IL-12 are expressed higher in MERS-CoV infection compared to SARS-CoV infection [217, 257, 258] MERS-CoV induced higher levels of IFN-y, IP-10, IL-12, and RANTES than SARS-CoV. [217] | 1a, and TNFα increased [259, 260] | CCL-4/MIP-1β, CXCL-8/IL-8, CXCL-9/MIG, and CXCL-10/IP-10 are associated with pathogenicity of both avian (H5N1 and H7N9) and human (pdmH1N1 and H3N2) viruses. | human trachea-bronchial airway epithelium. PIV 1 replicated to high titer yet did not induce cytokine secretion until late in infection, while PIV 2 replicated less efficiently but induced an early cytokine peak. PIV 3 replicated to high titer but induced a slower rise in cytokine secretion. The T cell chemoattractants CCL-2/MCP-1, CXCL-8/IL-8, CXCL-9/MIG, and CXCL-10/IP-10 are also related with mortality. [261] | IFN-y are decreased. [259] |  |
| Vaccine candidates | no FDA approved vaccine [264] | no vaccine [265] | no vaccine is currently available [266] | Inactivated Influenza Vaccines (IIVs), | no licensed vaccine [165] | No vaccine but Palivizumab is a monoclonal antibody | no clinically effective rhinovirus vaccine [268] |

|  |  |  |  | Recombinant Influenza Vaccine (RIV 4) and Live Attenuated Influenza Vaccine (LAIV 4) [267] |  | recommended to be administered to high-risk infants and young children. It is given in monthly intramuscular injections during the RSV season, [194] |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Treatment | There is no clear, unified and effective treatment plan for COVID-19 [264] | No specific antiviral treatment [265] | Supportive treatment. No specific antiviral drugs [269] | Antiviral drugs may be a treatment option [270] | No antiviral agents symptomatic Treatment [250] | Supportive care [271] | There are no approved antiviral medications [272] |

399

400 Table 1.

401

402 Abbreviations:

403 Angiotensin-converting enzyme 2 (ACE 2), Severe acute respiratory syndrome coronavirus (SARS-CoV), Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), Middle East respiratory syndrome coronavirus (MERS-CoV), Dipeptidylpeptidase 4 (DPP 4),

404 CX3C chemokine receptor 1 (CX3CR 1), CDHR3:cadherin-related family member 3, ICAM-1: intercellular adhesion molecule 1,

405 LDLR:low-density lipoprotein receptor, conventional Dendritic cells subtypes (cDC1 and cDC2), AMs: alveolar macrophages

406 Lower respiratory infection (LRI), Summary relative illness ratios (sRIRs), Nuclear Factor-Kappa-B Inhibitor Alpha (NFKBIA),

407 Interferon gamma (IFN-y), interleukin-4 (IL-4), Granulocyte colony stimulating factor (G-CSF), Macrophage inflammatory protein-1

408 alpha(MIP-la), Monocyte chemoattractant protein-1 (MCP-1), monocyte-derived macrophages (MDMs) and dendritic cells (MDDCs),

409 type I myeloid (m) DCs (mDC)

A. SARS-CoV-2
   - ACE-2 Receptor
   - Virus Release
   - Assembly
   - Virus Replication
   - Virus RNA
   - Virus Protein

B.
   - Lungs
   - Alveolus
   - SARS-CoV-2
   - ACE-2 Receptor
   - Spike Shedding

C. Endothelial Cell

D.
   - IgM
   - B-Cell
   - Antibodies
   - Classical Pathway
   - C1q
   - Lectin Pathway
   - C3a
   - C3b
   - Chemotaxis
   - Neutrophil
   - Monocyte
   - C5a
   - C5b
   - Membrane Attack Complex
   - C6
   - C7
   - C8
   - C9
   - Lung Damage

E.
   - Type I Pneumocyte
   - Type II Pneumocyte
   - SARS-CoV-2
   - CD8+ T-cell
   - T-cell
   - Plasma Cell
   - NETs
   - Neutrophil
   - Dendritic Cell
   - DAMPs
   - PAMPs
   - Chemokines/Cytokines
   - Cytokine Activation
   - T-cell
   - Macrophage
   - Activated Macrophage
   - Cytokine Storm
     - (IL-1β, IFN-γ, CXCL10, CXCL11, IL-6, IP-10, TNF-α)
   - Granulocyte
   - Capillary
   - C5a
   - NK-Cell
   - B-Cell
   - T-Cell
   - Monocyte

References.

1. Wang, C., et al., A novel coronavirus outbreak of global health concern. Lancet, 2020. 395(10223): p. 470-473.
2. Zhu, N., et al., A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med, 2020. 382(8): p. 727-733.
3. Ryu, S., B.C. Chun, and V.T.F.T. Korean Society of Epidemiology -nCo, An interim review of the epidemiological characteristics of 2019 novel coronavirus. Epidemiol Health, 2020. 42: p. e2020006.
4. Khafaie, M.A. and F. Rahim, Cross-Country Comparison of Case Fatality Rates of COVID-19/SARS-COV-2. Osong Public Health Res Perspect, 2020. 11(2): p. 74-80.
5. Sun, P., et al., Understanding of COVID-19 based on current evidence. J Med Virol, 2020.
6. Ahn, D.G., et al., Current Status of Epidemiology, Diagnosis, Therapeutics, and Vaccines for Novel Coronavirus Disease 2019 (COVID-19). J Microbiol Biotechnol, 2020. 30(3): p. 313-324.
7. Heugel, J., et al., Coronavirus-associated pneumonia in previously healthy children. Pediatr Infect Dis J, 2007. 26(8): p. 753-5.
8. Kuypers, J., et al., Clinical disease in children associated with newly described coronavirus subtypes. Pediatrics, 2007. 119(1): p. e70-6.
9. Drosten, C., et al., Identification of a novel coronavirus in patients with severe acute respiratory syndrome. N Engl J Med, 2003. 348(20): p. 1967-76.
10. Zaki, A.M., et al., Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. N Engl J Med, 2012. 367(19): p. 1814-20.
11. Garg, S., et al., Hospitalization Rates and Characteristics of Patients Hospitalized with Laboratory-Confirmed Coronavirus Disease 2019 - COVID-NET, 14 States, March 1-30, 2020. MMWR Morb Mortal Wkly Rep, 2020. 69(15): p. 458-464.
12. Richardson, S., et al., Presenting Characteristics, Comorbidities, and Outcomes Among 5700 Patients Hospitalized With COVID-19 in the New York City Area. JAMA, 2020.
13. Jin, J.M., et al., Gender Differences in Patients With COVID-19: Focus on Severity and Mortality. Front Public Health, 2020. 8: p. 152.
14. Karlberg, J., D.S. Chong, and W.Y. Lai, Do men have a higher case fatality rate of severe acute respiratory syndrome than women do? Am J Epidemiol, 2004. 159(3): p. 229-31.
15. Alghamdi, I.G., et al., The pattern of Middle East respiratory syndrome coronavirus in Saudi Arabia: a descriptive epidemiological analysis of data from the Saudi Ministry of Health. Int J Gen Med, 2014. 7: p. 417-23.
16. Hoffmann, M., et al., SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell, 2020.
17. Tseng, C.T., et al., Severe acute respiratory syndrome coronavirus infection of mice transgenic for the human Angiotensin-converting enzyme 2 virus receptor. J Virol, 2007. 81(3): p. 1162-73.
18. Hofmann, H., et al., Human coronavirus NL63 employs the severe acute respiratory syndrome coronavirus receptor for cellular entry. Proc Natl Acad Sci USA, 2005. 102(22): p. 7988-93.
19. Hamming, I., et al., Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis. J Pathol, 2004. 203(2): p. 631-7.
20. Li, F., Structure, Function, and Evolution of Coronavirus Spike Proteins. Annu Rev Virol, 2016. 3(1): p. 237-261.
21. Riordan, J.F., Angiotensin-I-converting enzyme and its relatives. Genome Biol, 2003. 4(8): p. 225.

485 22. Wan, Y., et al., Receptor Recognition by the Novel Coronavirus from Wuhan: an Analysis Based on Decade-Long Structural Studies of SARS Coronavirus. J Virol, 2020.94(7).
486 23. Kuba, K., et al., Trilogy of ACE2: a peptidase in the renin-angiotensin system, a SARS receptor, and a partner for amino acid transporters. Pharmacol Ther, 2010.128(1): p. 119-28.
487 24. Ksiazek, T.G., et al., A novel coronavirus associated with severe acute respiratory syndrome. N Engl J Med, 2003.348(20): p. 1953-66.
488 25. Raj, V.S., et al., Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC. Nature, 2013.495(7440): p. 251-4.
489 26. Lambeir, A.M., et al., Dipeptidyl-peptidase IV from bench to bedside: an update on structural properties, functions, and clinical aspects of the enzyme DPP IV. Crit Rev Clin Lab Sci, 2003.40(3): p. 209-94.
490 27. Whitsett, J.A., Airway Epithelial Differentiation and Mucociliary Clearance. Ann Am Thorac Soc, 2018.15(Suppl 3): p. S143-S148.
491 28. Whitsett, J.A. and T. Alenghat, Respiratory epithelial cells orchestrate pulmonary innate immunity. Nat Immunol, 2015.16(1): p. 27-35.
492 29. Grassin-Delyle, S., et al., The Role of Toll-Like Receptors in the Production of Cytokines by Human Lung Macrophages. J Innate Immun, 2020.12(1): p. 63-73.
493 30. Leiva-Juarez, M.M., J.K. Kolls, and S.E. Evans, Lung epithelial cells: therapeutically inducible effectors of antimicrobial defense. Mucosal Immunol, 2018.11(1): p. 21-34.
494 31. Byrne, A.J., et al., Pulmonary macrophages: key players in the innate defence of the airways. Thorax, 2015.70(12): p. 1189-96.
495 32. Kopf, M., C. Schneider, and S.P. Nobs, The development and function of lung-resident macrophages and dendritic cells. Nat Immunol, 2015.16(1): p. 36-44.
496 33. Jung, H.E., T.H. Kim, and H.K. Lee, Contribution of Dendritic Cells in Protective Immunity against Respiratory Syncytial Virus Infection. Viruses, 2020.12(1).
497 34. Duan, M., M.L. Hibbs, and W. Chen, The contributions of lung macrophage and monocyte heterogeneity to influenza pathogenesis. Immunol Cell Biol, 2017.95(3): p. 225-235.
498 35. Reizis, B., Plasmacytoid Dendritic Cells: Development, Regulation, and Function. Immunity, 2019.50(1): p. 37-50.
499 36. Collin, M. and V. Bigley, Human dendritic cell subsets: an update. Immunology, 2018.154(1): p. 3-20.
500 37. Swiecki, M. and M. Colonna, The multifaceted biology of plasmacytoid dendritic cells. Nat Rev Immunol, 2015.15(8): p. 471-85.
501 38. Lau, Y.L., J.S. Peiris, and H.K. Law, Role of dendritic cells in SARS coronavirus infection. Hong Kong Med J, 2012.18 Suppl 3: p. 28-30.
502 39. Fitzgerald-Bocarsly, P., J. Dai, and S. Singh, Plasmacytoid dendritic cells and type I IFN: 50 years of convergent history. Cytokine Growth Factor Rev, 2008.19(1): p. 3-19.
503 40. Merad, M., et al., The dendritic cell lineage: ontogeny and function of dendritic cells and their subsets in the steady state and the inflamed setting. Annu Rev Immunol, 2013.31: p. 563-604.
504 41. Sang, Y., L.C. Miller, and F. Blecha, Macrophage Polarization in Virus-Host Interactions. J Clin Cell Immunol, 2015.6(2).
505 42. Goritzka, M., et al., Alveolar macrophage-derived type I interferons orchestrate innate immunity to RSV through recruitment of antiviral monocytes. J Exp Med, 2015.212(5): p. 699-714.
506 43. Franken, L., M. Schiwon, and C. Kurts, Macrophages: sentinels and regulators of the immune system. Cell Microbiol, 2016.18(4): p. 475-87.
507 44. Condon, T.V., et al., Lung dendritic cells at the innate-adaptive immune interface. J Leukoc Biol, 2011.90(5): p. 883-95.

535 45. Schyns, J., F. Bureau, and T. Marichal, Lung Interstitial Macrophages: Past, Present, and Future. J Immunol Res, 2018. 2018: p. 5160794.
536 46. Ramos, I., et al., Innate Immune Response to Influenza Virus at Single-Cell Resolution in Human Epithelial Cells Revealed Paracrine Induction of Interferon Lambda 1. J Virol, 2019. 93(20).
537 47. Allen, I.C., et al., The NLRP3 inflammasome mediates in vivo innate immunity to influenza A virus through recognition of viral RNA. Immunity, 2009. 30(4): p. 556-65.
538 48. Parker, D. and A. Prince, Innate immunity in the respiratory epithelium. Am J Respir Cell Mol Biol, 2011. 45(2): p. 189-201.
539 49. Vareille, M., et al., The airway epithelium: soldier in the fight against respiratory viruses. Clin Microbiol Rev, 2011. 24(1): p. 210-29.
540 50. VanLeuven, J.T., et al., Lung epithelial cells have virus-specific and shared gene expression responses to infection by diverse respiratory viruses. PLoS One, 2017. 12(6): p. e0178408.
541 51. Panuska, J.R., et al., Productive infection of isolated human alveolar macrophages by respiratory syncytial virus. J Clin Invest, 1990. 86(1): p. 113-9.
542 52. Thistle, J., D. Martinon, and P.M. Weers, Helix 1 tryptophan variants in Galleria mellonella apolipophorin III. Chem Phys Lipids, 2015. 193: p. 18-23.
543 53. Dong, S., et al., Synergy of IL-27 and TNF-alpha in regulating CXCL 10 expression in lung fibroblasts. Am J Respir Cell Mol Biol, 2013. 48(4): p. 518-30.
544 54. Yilla, M., et al., SARS-coronavirus replication in human peripheral monocytes/macrophages. Virus Res, 2005. 107(1): p. 93-101.
545 55. Zhou, J., et al., Middle East respiratory syndrome coronavirus infection: virus-host cell interactions and implications on pathogenesis. Virol J, 2015. 12: p. 218.
546 56. Bonora, M., et al., Molecular mechanisms of cell death: central implication of ATP synthase in mitochondrial permeability transition. Oncogene, 2015. 34(12): p. 1608.
547 57. Siu, K.L., et al., Severe acute respiratory syndrome coronavirus ORF3a protein activates the NLRP3 inflammasome by promoting TRAF3-dependent ubiquitination of ASC. FASEB J, 2019. 33(8): p. 8865-8877.
548 58. Lau, S.K.P., et al., Delayed induction of proinflammatory cytokines and suppression of innate antiviral response by the novel Middle East respiratory syndrome coronavirus: implications for pathogenesis and treatment. J Gen Virol, 2013. 94(Pt 12): p. 2679-2690.
549 59. Yoshikawa, T., et al., Dynamic innate immune responses of human bronchial epithelial cells to severe acute respiratory syndrome-associated coronavirus infection. PLoS One, 2010. 5(1): p. e8729.
550 60. Qian, Z., et al., Innate immune response of human alveolar type II cells infected with severe acute respiratory syndrome-coronavirus. Am J Respir Cell Mol Biol, 2013. 48(6): p. 742-8.
551 61. Smits, S.L., et al., Exacerbated innate host response to SARS-CoV in aged non-human primates. PLoS Pathog, 2010. 6(2): p. e1000756.
552 62. Korean Society of Infectious, D., C. Korea Centers for Disease, and Prevention, Analysis on 54 Mortality Cases of Coronavirus Disease 2019 in the Republic of Korea from January 19 to March 10, 2020. J Korean Med Sci, 2020. 35(12): p. e132.
553 63. Leung, G.M., et al., Prevalence of SARS-CoV antibody in all Hong Kong patient contacts. Hong Kong Med J, 2009. 15 Suppl 9: p. 27-9.
554 64. Pierangeli, A., C. Scagnolari, and G. Antonelli, Respiratory syncytial virus. Minerva Pediatr, 2018. 70(6): p. 553-565.
555 65. Takeuchi, K., et al., Measles virus V protein blocks interferon (IFN)-alpha/beta but not IFN-gamma signaling by inhibiting STAT1 and STAT2 phosphorylation. FEBS Lett, 2003. 545(2-3): p. 177-82.

585 66. Lui, P.Y., et al., Middle East respiratory syndrome coronavirus M protein suppresses type  
I interferon expression through the inhibition of TBK1-dependent phosphorylation of IRF3.  
Emerg Microbes Infect, 2016.5: p. e39.  

587 Narayanan, K., et al., Severe acute respiratory syndrome coronavirus nsp1 suppresses  
host gene expression, including that of type I interferon, in infected cells. J Virol, 2008.  
82(9): p. 4471-9.  

591 68. Siu, K.L., et al., Severe acute respiratory syndrome coronavirus M protein inhibits type I  
interferon production by impeding the formation of TRAF3.TANK.TBK1/IKKepsilon  
complex. J Biol Chem, 2009.284(24): p. 16202-9.  

594 69. Hu, Y., et al., The Severe Acute Respiratory Syndrome Coronavirus Nucleocapsid Inhibits  
Type I Interferon Production by Interfering with TRIM25-Mediated RIG-I Ubiquitination. J  
Virol, 2017.91(8).  

597 70. Kolaczkowska, E. and P. Kubes, Neutrophil recruitment and function in health and  
inflammation. Nat Rev Immunol, 2013.13(3): p. 159-75.  

599 71. Miura, T.A. and K.V. Holmes, Host-pathogen interactions during coronavirus infection of  
primary alveolar epithelial cells. J Leukoc Biol, 2009.86(5): p. 1145-51.  

601 72. Wang, R., et al., The role of C5a in acute lung injury induced by highly pathogenic viral  
infections. Emerg Microbes Infect, 2015.4(5): p. e28.  

603 73. Jiang, Y., et al., Characterization of cytokine/chemokine profiles of severe acute  
respiratory syndrome. Am J Respir Crit Care Med, 2005.171(8): p. 850-7.  

605 74. Hu, W., et al., SARS-CoV regulates immune function-related gene expression in human  
monocytic cells. Viral Immunol, 2012.25(4): p. 277-88.  

607 75. Nicholls, J.M., et al., Lung pathology of fatal severe acute respiratory syndrome. Lancet,  
2003.361(9371): p. 1773-8.  

609 76. Yen, Y.T., et al., Modeling the early events of severe acute respiratory syndrome  
coronavirus infection in vitro. J Virol, 2006.80(6): p. 2684-93.  

611 77. Channappanavar, R., et al., Dysregulated Type I Interferon and Inflammatory Monocyte-  
Macrophage Responses Cause Lethal Pneumonia in SARS-CoV-Infected Mice. Cell Host  
Microbe, 2016.19(2): p. 181-93.  

614 78. Tse, G.M., et al., Pulmonary pathological features in coronavirus associated severe acute  
respiratory syndrome (SARS). J Clin Pathol, 2004.57(3): p. 260-5.  

616 79. Liu, L., et al., Spatiotemporal interplay of severe acute respiratory syndrome coronavirus  
and respiratory mucosal cells drives viral dissemination in rhesus macaques. Mucosal  
Immunol, 2016.9(4): p. 1089-101.  

619 80. Zhou, X. and B.B. Moore, Location or origin? What is critical for macrophage propagation  
of lung fibrosis? Eur Respir J, 2018.51(3).  

621 81. Antonio, G.E., et al., Thin-section CT in patients with severe acute respiratory syndrome  
following hospital discharge: preliminary experience. Radiology, 2003.228(3): p. 810-5.  

623 82. Lau, Y.L. and J.S. Peiris, Pathogenesis of severe acute respiratory syndrome. Curr Opin  
Immunol, 2005.17(4): p. 404-10.  

625 83. yongwen chen, Z.F., Bo Diao, Rongshuai Wang, Gang Wang, Chenhui Wang, Yingjun  
Tan, Liang Liu, Changsong Wang, Ying Liu, Yueping Liu, Zilin Yuan, Liang Ren, Yuzhang  
Wu, The Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)  
Directly Decimates Human Spleens and Lymph Nodes. Medrxiv, 2020.  

629 84. Gu, J., et al., Multiple organ infection and the pathogenesis of SARS. J Exp Med, 2005.  
202(3): p. 415-24.  

631 85. Law, H.K., et al., Chemokine up-regulation in SARS-coronavirus-infected, monocyte-  
derived human dendritic cells. Blood, 2005.106(7): p. 2366-74.  

633 86. Tynell, J., et al., Middle East respiratory syndrome coronavirus shows poor replication but  
significant induction of antiviral responses in human monocyte-derived macrophages and  
dendritic cells. J Gen Virol, 2016.97(2): p. 344-355.

636 87. Channappanavar, R., et al., Sex-Based Differences in Susceptibility to Severe Acute
Respiratory Syndrome Coronavirus Infection. J Immunol, 2017. 198(10): p. 4046-4053.
637 88. Cameron, M.J., et al., Interferon-mediated immunopathological events are associated with
atypical innate and adaptive immune responses in patients with severe acute respiratory
syndrome. J Virol, 2007. 81(16): p. 8692-706.
638 89. Bao, L., et al., The pathogenicity of SARS-CoV-2 in hACE2 transgenic mice. Nature, 2020.
639 90. Jiang, R.D., et al., Pathogenesis of SARS-CoV-2 in Transgenic Mice Expressing Human
Angiotensin-Converting Enzyme 2. Cell, 2020.
640 91. Ip, W.K., et al., Mannose-binding lectin in severe acute respiratory syndrome coronavirus
infection. J Infect Dis, 2005. 191(10): p. 1697-704.
641 92. Magro, C., et al., Complement associated microvascular injury and thrombosis in the
pathogenesis of severe COVID-19 infection: A report of five cases. Transl Res, 2020. 220:
p. 1-13.
642 93. Gao, T., et al., Highly pathogenic coronavirus N protein aggravates lung injury by MASP-
2-mediated complement over-activation. medRxiv, 2020: p. 2020.03.29.20041962.
643 94. Noris, M., A. Benigni, and G. Remuzzi, The case of Complement activation in COVID-19
multiorgan impact. Kidney Int, 2020.
644 95. Jiang, Y., et al., Blockade of the C5a-C5aR axis alleviates lung damage in hDPP4-
transgenic mice infected with MERS-CoV. Emerg Microbes Infect, 2018. 7(1): p. 77.
645 96. Galani, I.E. and E. Andreakos, Neutrophils in viral infections: Current concepts and
caveats. J Leukoc Biol, 2015. 98(4): p. 557-64.
646 97. Camp, J.V. and C.B. Jonsson, A Role for Neutrophils in Viral Respiratory Disease. Front
Immunol, 2017. 8: p. 550.
647 98. Manocha, S., K.R. Walley, and J.A. Russell, Severe acute respiratory distress syndrome
(SARS): a critical care perspective. Crit Care Med, 2003. 31(11): p. 2684-92.
648 99. Lee, N., et al., A major outbreak of severe acute respiratory syndrome in Hong Kong. N
Engl J Med, 2003. 348(20): p. 1986-94.
649 100. Jingyuan Liu, Y.L., Pan Xiang, Lin Pu, Haofeng Xiong, Chuansheng Li, Ming Zhang,
Jianbo Tan, Yanli Xu, Rui Song, Meihua Song, Lin Wang, Wei Zhang, Bing Han, Li Yang,
Xiaojing Wang, Guiqin Zhou, Ting Zhang, Ben Li, Yanbin Wang, Zhihai Chen, Xianbo
Wang, Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel
Coronavirus in the Early Stage. medRxiv, 2020.
650 101. Qin, C., et al., Dysregulation of immune response in patients with COVID-19 in Wuhan,
China. Clin Infect Dis, 2020.
651 102. Nagata, N., et al., Mouse-passaged severe acute respiratory syndrome-associated
coronavirus leads to lethal pulmonary edema and diffuse alveolar damage in adult but not
young mice. Am J Pathol, 2008. 172(6): p. 1625-37.
652 103. Raj, G.D., C.E. Savage, and R.C. Jones, Effect of heterophil depletion by 5-fluorouracil on
infectious bronchitis virus infection in chickens. Avian Pathol, 1997. 26(2): p. 427-32.
653 104. Haick, A.K., et al., Neutrophils are needed for an effective immune response against
pulmonary rat coronavirus infection, but also contribute to pathology. J Gen Virol, 2014.
95(Pt 3): p. 578-590.
654 105. Zhou, J., et al., Neutrophils promote mononuclear cell infiltration during viral-induced
encephalitis. J Immunol, 2003. 170(6): p. 3331-6.
655 106. de Wit, E., et al., Middle East respiratory syndrome coronavirus (MERS-CoV) causes
transient lower respiratory tract infection in rhesus macaques. Proc Natl Acad Sci USA,
2013. 110(41): p. 16598-603.
656 107. Guery, B., et al., Clinical features and viral diagnosis of two cases of infection with Middle
East Respiratory Syndrome coronavirus: a report of nosocomial transmission. Lancet,
2013. 381(9885): p. 2265-72.

686 108. Kuster, G.M., et al., SARS-CoV2: should inhibitors of the renin-angiotensin system be withdrawn in patients with COVID-19? Eur Heart J, 2020.
687 109. Sodhi, C.P., et al., A Dynamic Variation of Pulmonary ACE2 Is Required to Modulate Neutrophil Inflammation in Response to Pseudomonas aeruginosa Lung Infection in Mice. J Immunol, 2019. 203(11): p. 3000-3012.
688 110. Papayannopoulos, V., Neutrophil extracellular traps in immunity and disease. Nat Rev Immunol, 2018. 18(2): p. 134-147.
689 111. Zawrotniak, M. and M. Rapala-Kozik, Neutrophil extracellular traps (NETs) - formation and implications. Acta Biochim Pol, 2013. 60(3): p. 277-84.
690 112. Barnes, B.J., et al., Targeting potential drivers of COVID-19: Neutrophil extracellular traps. J Exp Med, 2020. 217(6).
691 113. Mozzini, C. and D. Girelli, The role of Neutrophil Extracellular Traps in Covid-19: Only an hypothesis or a potential new field of research? Thromb Res, 2020. 191: p. 26-27.
692 114. Wichmann, D., et al., Autopsy Findings and Venous Thromboembolism in Patients With COVID-19. Ann Intern Med, 2020.
693 115. Levi, M., et al., Coagulation abnormalities and thrombosis in patients with COVID-19. Lancet Haematol, 2020. 7(6): p. e438-e440.
694 116. Soehnlein, O., L. Lindbom, and C. Weber, Mechanisms underlying neutrophil-mediated monocyte recruitment. Blood, 2009. 114(21): p. 4613-23.
695 117. Prame Kumar, K., A.J. Nicholls, and C.H.Y. Wong, Partners in crime: neutrophils and monocytes/macrophages in inflammation and disease. Cell Tissue Res, 2018. 371(3): p. 551-565.
696 118. Schauer, C., et al., Aggregated neutrophil extracellular traps limit inflammation by degrading cytokines and chemokines. Nat Med, 2014. 20(5): p. 511-7.
697 119. Norris, S., et al., Selective reduction of natural killer cells and T cells expressing inhibitory receptors for MHC class I in the livers of patients with hepatic malignancy. Cancer Immunol Immunother, 2003. 52(1): p. 53-8.
698 120. Dukers, D.F., et al., Expression of killer cell inhibitory receptors is restricted to true NK cell lymphomas and a subset of intestinal enteropathy-type T cell lymphomas with a cytotoxic phenotype. J Clin Pathol, 2001. 54(3): p. 224-8.
699 121. Ahmad, R., et al., Modulation of expression of the MHC class I-binding natural killer cell receptors, and NK activity in relation to viral load in HIV-infected/AIDS patients. J Med Virol, 2001. 65(3): p. 431-40.
700 122. Husain, Z., et al., HLA-Cw7 zygosity affects the size of a subset of CD158b+ natural killer cells. J Clin Immunol, 2002. 22(1): p. 28-36.
701 123. The involvement of natural killer cells in the pathogenesis of severe acute respiratory syndrome. Am J Clin Pathol, 2004. 121(4): p. 507-11.
702 124. Cui, W., et al., Expression of lymphocytes and lymphocyte subsets in patients with severe acute respiratory syndrome. Clin Infect Dis, 2003. 37(6): p. 857-9.
703 125. Cao, Q., et al., SARS-CoV-2 infection in children: Transmission dynamics and clinical characteristics. J Formos Med Assoc, 2020. 119(3): p. 670-673.
704 126. Cristiani, L., et al., Will children reveal their secret? The coronavirus dilemma. Eur Respir J, 2020. 55(4).
705 127. Li, T., et al., Significant changes of peripheral T lymphocyte subsets in patients with severe acute respiratory syndrome. J Infect Dis, 2004. 189(4): p. 648-51.
706 128. Monticelli, L.A., et al., Innate lymphoid cells promote lung-tissue homeostasis after infection with influenza virus. Nat Immunol, 2011. 12(11): p. 1045-54.
707 129. Li, B.W.S., et al., T cells and ILC2s are major effector cells in influenza-induced exacerbation of allergic airway inflammation in mice. Eur J Immunol, 2019. 49(1): p. 144-156.

130. Qi, F., et al., Single cell RNA sequencing of 13 human tissues identify cell types and receptors of human coronaviruses. Biochem Biophys Res Commun, 2020.
131. Lo, A.W., N.L. Tang, and K.F. To, How the SARS coronavirus causes disease: host or organism? J Pathol, 2006. 208(2): p. 142-51.
132. Min, C.K., et al., Comparative and kinetic analysis of viral shedding and immunological responses in MERS patients representing a broad spectrum of disease severity. Sci Rep, 2016. 6: p. 25359.
133. Kim, E.S., et al., Clinical Progression and Cytokine Profiles of Middle East Respiratory Syndrome Coronavirus Infection. J Korean Med Sci, 2016. 31(11): p. 1717-1725.
134. Moran, J.V., et al., Mobile group II introns of yeast mitochondrial DNA are novel site-specific retroelements. Mol Cell Biol, 1995. 15(5): p. 2828-38.
135. Hogan, R.J., et al., Resolution of primary severe acute respiratory syndrome-associated coronavirus infection requires Stat1. J Virol, 2004. 78(20): p. 11416-21.
136. Frieman, M.B., et al., SARS-CoV pathogenesis is regulated by a STAT1 dependent but a type I, II and III interferon receptor independent mechanism. PLoS Pathog, 2010. 6(4): p. e1000849.
137. Mordstein, M., et al., Lambda interferon renders epithelial cells of the respiratory and gastrointestinal tracts resistant to viral infections. J Virol, 2010. 84(11): p. 5670-7.
138. Mahlakoiv, T., et al., Combined action of type I and type III interferon restricts initial replication of severe acute respiratory syndrome coronavirus in the lung but fails to inhibit systemic virus spread. J Gen Virol, 2012. 93(Pt 12): p. 2601-2605.
139. Zhao, J., et al., Rapid generation of a mouse model for Middle East respiratory syndrome. Proc Natl Acad Sci USA, 2014. 111(13): p. 4970-5.
140. Lu, X., et al., SARS-CoV nucleocapsid protein antagonizes IFN-beta response by targeting initial step of IFN-beta induction pathway, and its C-terminal region is critical for the antagonism. Virus Genes, 2011. 42(1): p. 37-45.
141. Kopecky-Bromberg, S.A., et al., Severe acute respiratory syndrome coronavirus open reading frame (ORF) 3b, ORF 6, and nucleocapsid proteins function as interferon antagonists. J Virol, 2007. 81(2): p. 548-57.
142. Huang, C., et al., SARS coronavirus nsp1 protein induces template-dependent endonucleolytic cleavage of mRNAs: viral mRNAs are resistant to nsp1-induced RNA cleavage. PLoS Pathog, 2011. 7(12): p. e1002433.
143. Lokugamage, K.G., et al., Middle East Respiratory Syndrome Coronavirus nsp1 Inhibits Host Gene Expression by Selectively Targeting mRNAs Transcribed in the Nucleus while Sparing mRNAs of Cytoplasmic Origin. J Virol, 2015. 89(21): p. 10970-81.
144. Jauregui, A.R., et al., Identification of residues of SARS-CoV nsp1 that differentially affect inhibition of gene expression and antiviral signaling. PLoS One, 2013. 8(4): p. e62416.
145. Mielech, A.M., et al., MERS-CoV papain-like protease has deISGylating and deubiquitinating activities. Virology, 2014. 450-451: p. 64-70.
146. Devaraj, S.G., et al., Regulation of IRF-3-dependent innate immunity by the papain-like protease domain of the severe acute respiratory syndrome coronavirus. J Biol Chem, 2007. 282(44): p. 32208-21.
147. Yang, X., et al., Proteolytic processing, deubiquitinase and interferon antagonist activities of Middle East respiratory syndrome coronavirus papain-like protease. J Gen Virol, 2014. 95(Pt 3): p. 614-626.
148. Kuri, T., et al., The ADP-ribose-1'-monophosphatase domains of severe acute respiratory syndrome coronavirus and human coronavirus 229E mediate resistance to antiviral interferon responses. J Gen Virol, 2011. 92(Pt 8): p. 1899-1905.
149. Liu, D.X., et al., Accessory proteins of SARS-CoV and other coronaviruses. Antiviral Res, 2014. 109: p. 97-109.


786 150. Li, W., et al., Angiotensin-converting enzyme 2 is a functional receptor for the SARS
coronavirus. Nature, 2003.426(6965): p.450-4.
787 151. Wang, N., et al., Structure of MERS-CoV spike receptor-binding domain complexed with
human receptor DPP 4. Cell Res, 2013.23(8): p.986-93.
788 152. Yan, R., et al., Structural basis for the recognition of SARS-CoV-2 by full-length human
ACE 2. Science, 2020.367(6485): p.1444-1448.
789 153. Li, Y., et al., The MERS-CoV Receptor DPP 4 as a Candidate Binding Target of the SARS-
CoV-2 Spike. iScience, 2020.23(6): p.101160.
790 154. Shinya, K. and Y. Kawaoka, [Influenza virus receptors in the human airway]. Uirusu, 2006.
791 56(1): p.85-9.
792 155. Tong, J., et al., The Sialic Acid Binding Activity of Human Parainfluenza Virus 3 and
793 Mumps Virus Glycoproteins Enhances the Adherence of Group B Streptococci to HEp-2
794 Cells. Front Cell Infect Microbiol, 2018.8: p.280.
795 156. Anderson, C.S., et al., CX3CR1 as a respiratory syncytial virus receptor in pediatric human
796 lung. Pediatr Res, 2020.87(5): p.862-867.
797 157. Basnet, S., A.C. Palmenberg, and J.E. Gern, Rhinoviruses and Their Receptors. Chest,
798 2019.155(5): p.1018-1025.
799 158. Chen, Y., et al., A novel subset of putative stem/progenitor CD34+Oct-4+ cells is the major
800 target for SARS coronavirus in human lung. J Exp Med, 2007.204(11): p.2529-36.
801 159. Nicholls, J.M., et al., Time course and cellular localization of SARS-CoV nucleoprotein
802 and RNA in lungs from fatal cases of SARS. PLoS Med, 2006.3(2): p.e27.
803 160. Abeloos, J., et al., Anaesthetic and medico-legal problems in patients intoxicated by
804 alcohol. Med Sci Law, 1985.25(2): p.131-5.
805 161. Scobey, T., et al., Reverse genetics with a full-length infectious cDNA of the Middle East
806 respiratory syndrome coronavirus. Proc Natl Acad Sci USA, 2013.110(40): p.16157-62.
807 162. Chu, H., et al., Comparative replication and immune activation profiles of SARS-CoV-2
808 and SARS-CoV in human lungs: an ex vivo study with implications for the pathogenesis
809 of COVID-19. Clin Infect Dis, 2020.
810 163. Sungnak, W., et al., SARS-CoV-2 entry factors are highly expressed in nasal epithelial
811 cells together with innate immune genes. Nat Med, 2020.26(5): p.681-687.
812 164. Fiege, J.K. and R.A. Langlois, Investigating influenza A virus infection: tools to track
813 infection and limit tropism. J Virol, 2015.89(12): p.6167-70.
814 165. Branche, A.R. and A.R. Falsey, Parainfluenza Virus Infection. Semin Respir Crit Care
815 Med, 2016.37(4): p.538-54.
816 166. Persson, B.D., et al., Respiratory syncytial virus can infect basal cells and alter human
817 airway epithelial differentiation. PLoS One, 2014.9(7): p.e102368.
818 167. Raiden, S., et al., Respiratory Syncytial Virus (RSV) Infects CD4+ T Cells: Frequency of
819 Circulating CD4+ RSV+ T Cells as a Marker of Disease Severity in Young Children. J
820 Infect Dis, 2017.215(7): p.1049-1058.
821 168. Ljubin-Sternak, S., et al., The Emerging Role of Rhinoviruses in Lower Respiratory Tract
822 Infections in Children - Clinical and Molecular Epidemiological Study From Croatia, 2017-
823 2019. Front Microbiol, 2019.10: p.2737.
824 169. WHO, Summary table of SARS cases by country, 1 November 2002 - 7 August 2003.
825 2003: https://www.who.int/csr/sars/country/2003_08_15/en/.
826 170. WHO, Middle East respiratory syndrome coronavirus (MERS-CoV). 2019:
827 https://www.who.int/emergencies/mers-cov/en/.
828 171. WHO, Q&A: Influenza and COVID-19 - similarities and differences. 2020:
829 https://www.who.int/emergencies/diseases/novel-coronavirus-2019/question-and-
830 answers-hub/q-a-detail/q-a-similarities-and-differences-covid-19-and-
831 influenza?gclid=EAlaIQobChMIy-
832 WTm6GY6glVi8qyCh3iXwFrEAAVASAAEglDBvD_BwE.

837 172. Henrickson, K.J., Parainfluenza Viruses. Clinical Microbiology Reviews, 2003. 16(2): p. 242-264.
838 
839 173. Berman, S., Epidemiology of acute respiratory infections in children of developing countries. Rev Infect Dis, 1991. 13 Suppl 6: p. S454-62.
840 
841 174. CDC, Respiratory Syncytial Virus-Associated Mortality (RSV-Associated Mortality) 2019 Case Definition. 2019, CDC: https://wwwn.cdc.gov/nndss/conditions/respiratory-syncytial-virus-associated-mortality/case-definition/2019/.
842 
843 
844 175. Peiris, J.S., et al., Coronavirus as a possible cause of severe acute respiratory syndrome. Lancet, 2003. 361(9366): p. 1319-25.
845 
846 176. Assiri, A., et al., Epidemiological, demographic, and clinical characteristics of 47 cases of Middle East respiratory syndrome coronavirus disease from Saudi Arabia: a descriptive study. Lancet Infect Dis, 2013. 13(9): p. 752-61.
847 
848 
849 177. Saad, M., et al., Clinical aspects and outcomes of 70 patients with Middle East respiratory syndrome coronavirus infection: a single-center experience in Saudi Arabia. Int J Infect Dis, 2014. 29: p. 301-6.
850 
851 
852 178. Liu, Y., et al., Association between age and clinical characteristics and outcomes of COVID-19. Eur Respir J, 2020. 55(5).
853 
854 179. Caini, S., et al., Distribution of influenza virus types by age using case-based global surveillance data from twenty-nine countries, 1999-2014. BMC Infect Dis, 2018. 18(1): p. 269.
855 
856 
857 180. Lau, S.K., et al., Human parainfluenza virus 4 outbreak and the role of diagnostic tests. J Clin Microbiol, 2005. 43(9): p. 4515-21.
858 
859 181. Ueno, F., et al., Age-specific incidence rates and risk factors for respiratory syncytial virus-associated lower respiratory tract illness in cohort children under 5 years old in the Philippines. Influenza Other Respir Viruses, 2019. 13(4): p. 339-353.
860 
861 
862 182. Falsey, A.R. and E.E. Walsh, Respiratory syncytial virus infection in adults. Clin Microbiol Rev, 2000. 13(3): p. 371-84.
863 
864 183. Linder, J.E., et al., Human rhinovirus C: Age, season, and lower respiratory illness over the past 3 decades. J Allergy Clin Immunol, 2013. 131(1): p. 69-77 e1-6.
865 
866 184. WHO, Consensus document on the epidemiology of severe acute respiratory syndrome (SARS). 2003, WHO: https://www.who.int/csr/sars/WHOconsensus.pdf?ua=1.
867 
868 185. Park, J.E., et al., MERS transmission and risk factors: a systematic review. BMC Public Health, 2018. 18(1): p. 574.
869 
870 186. Biggerstaff, M., et al., Estimates of the reproduction number for seasonal, pandemic, and zoonotic influenza: a systematic review of the literature. BMC Infect Dis, 2014. 14: p. 480.
871 
872 187. Otomaru, H., et al., Transmission of Respiratory Syncytial Virus Among Children Under 5 Years in Households of Rural Communities, the Philippines. Open Forum Infect Dis, 2019. 6(3): p. ofz045.
873 
874 
875 188. Scully, E.J., et al., Lethal Respiratory Disease Associated with Human Rhinovirus C in Wild Chimpanzees, Uganda, 2013. Emerg Infect Dis, 2018. 24(2): p. 267-274.
876 
877 189. Varia, M., et al., Investigation of a nosocomial outbreak of severe acute respiratory syndrome (SARS) in Toronto, Canada. CMAJ, 2003. 169(4): p. 285-92.
878 
879 190. Virlogeux, V., et al., Comparison of incubation period distribution of human infections with MERS-CoV in South Korea and Saudi Arabia. Sci Rep, 2016. 6: p. 35839.
880 
881 191. Linton, N.M., et al., Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data. J Clin Med, 2020. 9(2).
882 
883 
884 192. Lessler, J., et al., Incubation periods of acute respiratory viral infections: a systematic review. Lancet Infect Dis, 2009. 9(5): p. 291-300.
885 
886 193. CDC, Human Parainfluenza Viruses (HPIVs). 2019, CDC: https://www.cdc.gov/parainfluenza/hcp/clinical.html.
887 


The OCR has been converted into Markdown format as requested.
888 194. CDC, Respiratory Syncytial Virus Infection (RSV). 2018, CDC: https://www.cdc.gov/rsv/clinical/index.html.
889 195. Lipsitch, M., et al., Transmission dynamics and control of severe acute respiratory syndrome. Science, 2003. 300(5627): p. 1966-70.
890 196. WHO, 2019. Q&As on COVID-19 and related health topics, WHO: https://www.who.int/emergencies/diseases/novel-coronavirus-2019/question-and-answers-hub.
891 197. Booth, C.M., et al., Clinical features and short-term outcomes of 144 patients with SARS in the greater Toronto area. JAMA, 2003. 289(21): p. 2801-9.
892 198. Chu, K.H., et al., Acute renal impairment in coronavirus-associated severe acute respiratory syndrome. Kidney Int, 2005. 67(2): p. 698-705.
893 199. Alqahtani, F.Y., et al., Prevalence of comorbidities in cases of Middle East respiratory syndrome coronavirus: a retrospective study. Epidemiol Infect, 2018: p. 1-5.
894 200. Ganguly, R. and R.H. Waldman, Respiratory tract cell-mediated immunity. Bull Eur Physiopathol Respir, 1977. 13(1): p. 95-102.
895 201. Zhou, F., et al., Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study. Lancet, 2020. 395(10229): p. 1054-1062.
896 202. Yang, J., et al., Prevalence of comorbidities and its effects in patients infected with SARS-CoV-2: a systematic review and meta-analysis. Int J Infect Dis, 2020. 94: p. 91-95.
897 203. Jain, S., et al., Hospitalized patients with 2009 H1N1 influenza in the United States, April-June 2009. N Engl J Med, 2009. 361(20): p. 1935-44.
898 204. Ackerson, B., et al., Severe Morbidity and Mortality Associated With Respiratory Syncytial Virus Versus Influenza Infection in Hospitalized Older Adults. Clin Infect Dis, 2019. 69(2): p. 197-203.
899 205. Falsey, A.R., et al., Respiratory syncytial virus infection in elderly and high-risk adults. N Engl J Med, 2005. 352(17): p. 1749-59.
900 206. Iroh Tam, P.Y., L. Zhang, and Z. Cohen, Clinical characteristics and outcomes of human rhinovirus positivity in hospitalized children. Ann Thorac Med, 2018. 13(4): p. 230-236.
901 207. Costa, C., et al., Detection of human rhinoviruses in the lower respiratory tract of lung transplant recipients. Arch Virol, 2011. 156(8): p. 1439-43.
902 208. Parody, R., et al., Upper and lower respiratory tract infections by human enterovirus and rhinovirus in adult patients with hematological malignancies. Am J Hematol, 2007. 82(9): p. 807-11.
903 209. Park, M.D., Macrophages: a Trojan horse in COVID-19? Nat Rev Immunol, 2020. 20(6): p. 351.
904 210. Ghoneim, H.E., P.G. Thomas, and J.A. McCullers, Depletion of alveolar macrophages during influenza infection facilitates bacterial superinfections. J Immunol, 2013. 191(3): p. 1250-9.
905 211. Briggs, C.M., et al., Activation of human macrophages by bacterial components relieves the restriction on replication of an interferon-inducing parainfluenza virus 5 (PIV5) P/V mutant. Microbes Infect, 2011. 13(4): p. 359-68.
906 212. Laza-Stanca, V., et al., Rhinovirus replication in human macrophages induces NF-kappaB-dependent tumor necrosis factor alpha production. J Virol, 2006. 80(16): p. 8248-58.
907 213. Cong, Y., et al., MERS-CoV pathogenesis and antiviral efficacy of licensed drugs in human monocyte-derived antigen-presenting cells. PLoS One, 2018. 13(3): p. e0194868.
908 214. Gardner, I.D. and J.W. Lawton, Depressed human monocyte function after influenza infection in vitro. J Reticuloendothel Soc, 1982. 32(6): p. 443-8.

937 215. Le Nouen, C., et al., Infection and maturation of monocyte-derived human dendritic cells by human respiratory syncytial virus, human metapneumovirus, and human parainfluenza virus type 3. Virology, 2009. 385(1): p. 169-82.

940 216. Zhou, X., et al., Human Airway Epithelial Cells Direct Significant Rhinovirus Replication in Monocytic Cells by Enhancing ICAM1 Expression. Am J Respir Cell Mol Biol, 2017. 57(2): p. 216-225.

943 217. Chu, H., et al., Productive replication of Middle East respiratory syndrome coronavirus in monocyte-derived dendritic cells modulates innate immune response. Virology, 2014. 454-455: p. 197-205.

946 218. Zhou, Z., et al., Heightened Innate Immune Responses in the Respiratory Tract of COVID-19 Patients. Cell Host Microbe, 2020. 27(6): p. 883-890 e2.

948 219. Smed-Sorensen, A., et al., Influenza A virus infection of human primary dendritic cells impairs their ability to cross-present antigen to CD8 T cells. PLoS Pathog, 2012. 8(3): p. e1002572.

951 220. Hara, K., et al., Human parainfluenza virus type 2 vector induces dendritic cell maturation without viral RNA replication/transcription. Hum Gene Ther, 2013. 24(7): p. 683-91.

953 221. Andrade, C.A., et al., Innate Immune Components that Regulate the Pathogenesis and Resolution of hRSV and hMPV Infections. Viruses, 2020. 12(6).

955 222. Kirchberger, S., et al., Human rhinoviruses inhibit the accessory function of dendritic cells by inducing sialoadhesin and B7-H1 expression. J Immunol, 2005. 175(2): p. 1145-52.

957 223. Gralinski, L.E., et al., Complement Activation Contributes to Severe Acute Respiratory Syndrome Coronavirus Pathogenesis. mBio, 2018. 9(5).

959 224. Alosaimi, B., et al., MERS-CoV infection is associated with downregulation of genes encoding Th1 and Th2 cytokines/chemokines and elevated inflammatory innate immune response in the lower respiratory tract. Cytokine, 2020. 126: p. 154895.

962 225. Iwasaki, A. and P.S. Pillai, Innate immunity to influenza virus infection. Nat Rev Immunol, 2014. 14(5): p. 315-28.

964 226. Pickles, R.J. and J.P. DeVincenzo, Respiratory syncytial virus (RSV) and its propensity for causing bronchiolitis. J Pathol, 2015. 235(2): p. 266-76.

966 227. Noah, T.L., et al., Chemokines and inflammation in the nasal passages of infants with respiratory syncytial virus bronchiolitis. Clin Immunol, 2002. 104(1): p. 86-95.

968 228. Sheeran, P., et al., Elevated cytokine concentrations in the nasopharyngeal and tracheal secretions of children with respiratory syncytial virus disease. Pediatr Infect Dis J, 1999. 18(2): p. 115-22.

971 229. Wong, R.S., et al., Haematological manifestations in patients with severe acute respiratory syndrome: retrospective analysis. BMJ, 2003. 326(7403): p. 1358-62.

973 230. Chu, H., et al., Middle East Respiratory Syndrome Coronavirus Efficiently Infects Human Primary T Lymphocytes and Activates the Extrinsic and Intrinsic Apoptosis Pathways. J Infect Dis, 2016. 213(6): p. 904-14.

976 231. Wang, X., et al., SARS-CoV-2 infects T lymphocytes through its spike protein-mediated membrane fusion. Cell Mol Immunol, 2020.

978 232. Zhao, Y., et al., High levels of virus-specific CD4+ T cells predict severe pandemic influenza A virus infection. Am J Respir Crit Care Med, 2012. 186(12): p. 1292-7.

980 233. Sieg, S., et al., Infection and immunoregulation of T lymphocytes by parainfluenza virus type 3. Proc Natl Acad Sci USA, 1994. 91(14): p. 6293-7.

982 234. Openshaw, P.J. and C. Chiu, Protective and dysregulated T cell immunity in RSV infection. Curr Opin Virol, 2013. 3(4): p. 468-74.

984 235. Ilarrazza, R., et al., Rhinovirus has the unique ability to directly activate human T cells in vitro. J Allergy Clin Immunol, 2013. 131(2): p. 395-404.

986 236. Tang, F., et al., Lack of peripheral memory B cell responses in recovered patients with severe acute respiratory syndrome: a six-year follow-up study. J Immunol, 2011. 186(12): p. 7264-8.

987 237. Mubarak, A., W. Alturaiki, and M.G. Hemida, Middle East Respiratory Syndrome Coronavirus (MERS-CoV): Infection, Immunological Response, and Vaccine Development. J Immunol Res, 2019. 2019: p. 6491738.

988 238. Thevarajan, I., et al., Breadth of concomitant immune responses prior to patient recovery: a case report of non-severe COVID-19. Nat Med, 2020. 26(4): p. 453-455.

989 239. Kavaler, J., et al., A B cell population that dominates the primary response to influenza virus hemagglutinin does not participate in the memory response. Eur J Immunol, 1991. 21(11): p. 2687-95.

990 240. Rothaeusler, K. and N. Baumgarth, B-cell fate decisions following influenza virus infection. Eur J Immunol, 2010. 40(2): p. 366-77.

991 241. Roman, M., et al., Respiratory syncytial virus infection in infants is associated with predominant Th-2-like response. Am J Respir Crit Care Med, 1997. 156(1): p. 190-5.

992 242. Reed, J.L., et al., Innate immune signals modulate antiviral and polyreactive antibody responses during severe respiratory syncytial virus infection. J Infect Dis, 2009. 199(8): p. 1128-38.

993 243. Raes, M., et al., Peripheral blood T and B lymphocyte subpopulations in infants with acute respiratory syncytial virus bronchiolitis. Pediatr Allergy Immunol, 1997. 8(2): p. 97-102.

994 244. Aab, A., et al., Human rhinoviruses enter and induce proliferation of B lymphocytes. Allergy, 2017. 72(2): p. 232-243.

995 245. Nie, Y., et al., Neutralizing antibodies in patients with severe acute respiratory syndrome-associated coronavirus infection. J Infect Dis, 2004. 190(6): p. 1119-26.

996 246. Temperton, N.J., et al., Longitudinally profiling neutralizing antibody response to SARS coronavirus with pseudotypes. Emerg Infect Dis, 2005. 11(3): p. 411-6.

997 247. Payne, D.C., et al., Persistence of Antibodies against Middle East Respiratory Syndrome Coronavirus. Emerg Infect Dis, 2016. 22(10): p. 1824-6.

998 248. Jiang, S., C. Hillyer, and L. Du, Neutralizing Antibodies against SARS-CoV-2 and Other Human Coronaviruses. Trends Immunol, 2020. 41(5): p. 355-359.

999 249. Padilla-Quirarte, H.O., et al., Protective Antibodies Against Influenza Proteins. Front Immunol, 2019. 10: p. 1677.

1000 250. Henrickson, K.J., Parainfluenza viruses. Clin Microbiol Rev, 2003. 16(2): p. 242-64.

1001 251. Smith, C.B., et al., Protective effect of antibody to parainfluenza type 1 virus. N Engl J Med, 1966. 275(21): p. 1145-52.

1002 252. Stensballe, L.G., et al., Respiratory syncytial virus neutralizing antibodies in cord blood, respiratory syncytial virus hospitalization, and recurrent wheeze. J Allergy Clin Immunol, 2009. 123(2): p. 398-403.

1003 253. Barclay, W.S., et al., The time course of the humoral immune response to rhinovirus infection. Epidemiol Infect, 1989. 103(3): p. 659-69.

1004 254. Beijing Group of National Research Project for, S., Dynamic changes in blood cytokine levels as clinical indicators in severe acute respiratory syndrome. Chin Med J (Engl), 2003. 116(9): p. 1283-7.

1005 255. Wong, C.K., et al., Plasma inflammatory cytokines and chemokines in severe acute respiratory syndrome. Clin Exp Immunol, 2004. 136(1): p. 95-103.

1006 256. Li, Z., et al., The relationship between serum interleukins and T-lymphocyte subsets in patients with severe acute respiratory syndrome. Chin Med J (Engl), 2003. 116(7): p. 981-4.

1007 257. Zhou, J., et al., Active replication of Middle East respiratory syndrome coronavirus and aberrant induction of inflammatory cytokines and chemokines in human macrophages: implications for pathogenesis. J Infect Dis, 2014. 209(9): p. 1331-42.

1037 258. Cheung, C.Y., et al., Cytokine responses in severe acute respiratory syndrome  
coronavirus-infected macrophages in vitro: possible relevance to pathogenesis. J Virol,  
2005.79(12): p.7819-26.

1038 259. Bohmwald, K., et al., Contribution of Cytokines to Tissue Damage During Human  
Respiratory Syncytial Virus Infection. Front Immunol, 2019.10: p.452.

1040 260. Conti, P., et al., Induction of pro-inflammatory cytokines (IL-1 and IL-6) and lung  
inflammation by Coronavirus-19 (COVI-19 or SARS-CoV-2): anti-inflammatory strategies.  
J Biol Regul Homeost Agents, 2020.34(2).

1041 261. Betakova, T., et al., Cytokines Induced During Influenza Virus Infection. Curr Pharm Des,  
2017.23(18): p.2616-2622.

1042 262. Schaap-Nutt, A., et al., Human parainfluenza virus serotypes differ in their kinetics of  
replication and cytokine secretion in human tracheobronchial airway epithelium. Virology,  
2012.433(2): p.320-8.

1043 263. Rajan, D., et al., Human rhinovirus induced cytokine/chemokine responses in human  
airway epithelial and immune cells. PLoS One, 2014.9(12): p.e114322.

1044 264. Yang, Y., et al., The deadly coronaviruses: The 2003 SARS pandemic and the 2020 novel  
coronavirus epidemic in China. J Autoimmun, 2020.109: p.102434.

1045 265. CDC, Middle East Respiratory Syndrome (MERS). 2019, CDC:  
https://www.cdc.gov/coronavirus/mers/about/prevention.html

1046 266. Cascella, M., et al., Features, Evaluation and Treatment Coronavirus (COVID-19), in  
StatPearls. 2020: Treasure Island (FL).

1047 267. CDC, Influenza vaccines — United States, 2019–20 influenza season*. 2019, CDC:  
https://www.cdc.gov/flu/professionals/vaccines.htm

1048 268. Papi, A. and M. Contoli, Rhinovirus vaccination: the case against. Eur Respir J, 2011.  
37(1): p.5-7.

1049 269. Jiang, F., et al., Review of the Clinical Characteristics of Coronavirus Disease 2019  
(COVID-19). J Gen Intern Med, 2020.35(5): p.1545-1549.

1050 270. CDC, Influenza (Flu). 2019, CDC: https://www.cdc.gov/flu/treatment/index.html.

1051 271. Eiland, L.S., Respiratory syncytial virus: diagnosis, treatment and prevention. J Pediatr  
Pharmacol Ther, 2009.14(2): p.75-85.

1052 272. Greenberg, S.B., Update on Human Rhinovirus and Coronavirus Infections. Semin Respir  
Crit Care Med, 2016.37(4): p.555-71.

Figure 01.JPEG

A.
SARS-CoV-2
- Spike Glycoprotein (S)
- Virus Release
- ACE-2 Receptor
- Assembly
- Virus Replication
- Virus RNA
- Virus Protein

B.
Lungs
- Alveolus
- SARS-CoV-2
- ACE-2 Receptor
- Spike Shedding
- Type I Pneumocyte
- Cytokines
- Antibodies
- CD8+ T-cell
- Plasma Cell
- NETs
- Neutrophil
- NK-Cell
- B-Cell
- T-Cell
- Monocyte
- Granulocyte
- Activated Macrophage
- Cytokine Storm
  - (IL-1β, IFN-I, CXCL10, CXCL11, IL-6, IP-10, TNF-a)

C.
Endothelial Cell

I.
Type II Pneumocyte
- Chemokines/Cytokines
- DAMPs
- PAMPs

II.
Macrophage
- IL-6

III.
Th17
- IL-1β

IV.
Dendritic Cell
- Cytokine Activation
- T-cell

V.
Capillary
- C5a

Chemotaxis
Neutrophil
Monocyte
- IgM
- B-Cell
- Antibodies
- C1q
- Classical Pathway
- Lectin Pathway
- C3a
- C3b
- C5a
- C5b
- Membrane Attack Complex
- C6
- C7
- C8
- C9
- Lung Damage
