
Structural Analysis of Different LINC Complexes Reveals Distinct Binding Modes

Victor E. Cruz, F. Esra Demircioglu and Thomas U. Schwartz*

Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA

Correspondence to Thomas U. Schwartz: tus@mit.edu (T.U. Schwartz)

https://doi.org/10.1016/j.jmb.2020.09.019

Edited by Patrick Griffin

---

### Abstract

Linker of nucleoskeleton and cytoskeleton (LINC) complexes are molecular tethers that span the nuclear envelope (NE) and physically connect the nucleus to the cytoskeleton. They transmit mechanical force across the NE in processes such as nuclear anchorage, nuclear migration, and homologous chromosome pairing during meiosis. LINC complexes are composed of KASH proteins traversing the outer nuclear membrane, and SUN proteins crossing the inner nuclear membrane. Humans have several SUN- and KASH-containing proteins, yet what governs their proper engagement is poorly understood. To investigate this question, we solved high resolution crystal structures of human SUN2 in complex with the KASH-peptides of Nesprin3, Nesprin4, and KASH5. In comparison to the published structures of SUN2-KASH1/2 we observe alternative binding modes for these KASH peptides. While the core interactions between SUN and the C-terminal residues of the KASH peptide are similar in all five complexes, the extended KASH-peptide adopts at least two different conformations. The much-improved resolution allows for a more detailed analysis of other elements critical for KASH interaction, including the KASH-lid and the cation loop, and a possible self-locked state for unbound SUN. In summary, we observe distinct differences between the examined SUN-KASH complexes. These differences may have an important role in regulating the SUN-KASH network.

© 2020 The Author(s). Published by Elsevier Ltd. This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).

---

### Introduction

The nucleus in eukaryotic cells is physically separated from the cytoplasm by a double membrane bilayer known as the nuclear envelope (NE). The outer nuclear membrane (ONM) faces the cytoplasm and is contiguous with the endoplasmic reticulum (ER), while the inner nuclear membrane (INM) borders the nucleoplasm. The two membranes are fused at circular openings where nuclear pore complexes (NPCs) reside. The lumen between the INM and ONM is known as the perinuclear space (PNS), has a thickness of 30–50 nm and is contiguous with the lumen of the ER. Communication between the nucleus and the cytoplasm across the NE is accomplished by two separate molecular entities. On one hand, molecular exchange primarily occurs through the NPC. On the other hand, mechanical signaling is mediated through molecular tethers, the Linker of Nucleoskeleton to Cytoskeleton (LINC) complexes.

The core of LINC complexes resides in the PNS, formed by two proteins. SUN (Sad1 and UNC84) proteins traverse the INM, while KASH (Klarsicht, ANC-1 and Syne Homology) proteins are tail anchored at the ONM and constitute the cytoskeletal anchor of LINC complexes (Figure 1 (a) and (b)). SUN proteins are conserved in all eukaryotes. They possess an N-terminal domain that projects into the nucleus followed by one or three transmembrane helix that span the INM.

LINC Complex Binding Modes

(a)  
Nucleoplasm  
Perinuclear space (PNS)  

(b)  

(c)  

(d)  

Figure 1. Overview of SUN2 and KASH1-6 proteins. (a) Primary structure of full length SUN2 and KASH1-6. Functional domains are highlighted and cysteines involved in disulfide bond formation are labeled. Asterisk in KASH3 and KASH4 denote a conserved cysteine not yet shown to form a disulfide bond with SUN. KASH5 and KASH6 lack the cysteine altogether. (b) Cartoon model of SUN and KASH complexes traversing the inner and outer nuclear membranes respectively. (c) Cartoon of a SUN2 protomer with key features highlighted. The purple sphere represents a potassium ion typically found in the cation loop. (d) Cartoon of the canonical SUN-KASH 3:3 heterohexamer (KASH peptides in yellow).

The nuclear domain of SUN proteins interacts with lamins and heterochromatin, anchoring SUN at the INM. The perinuclear domain of SUN proteins consists of an extended coiled-coil that precedes the conserved C-terminal SUN domain which binds to KASH proteins. To date, five SUN proteins have been identified in humans. SUN1 and 2 are present in all tissues and possess partially redundant functions. SUN3, 4, and 5 are expressed in testis and have a shorter coiled-coil domain than their SUN1/2 counterparts.

The tail anchored KASH proteins project their ~20–30 C-terminal residues, the KASH peptide, into the PNS. Their “PPPX” motif at the very C terminus of the KASH peptide is most distinct, highly conserved, and required for binding to SUN proteins. Currently, 6 KASH domain-containing proteins have been identified in humans. These are the nesprins (nuclear envelope spectrin repeats), including Nesprin1-4, as well as KASH5 (CCDC155), and KASH6 (LRMP). Many of the known interactions of KASH proteins in the cytoplasm involve cytoskeletal elements. Different Nesprins bind different such elements, suggesting distinct roles for specific SUN-KASH complexes. This is perhaps best illustrated by human diseases, that can result from mutations in individual KASH proteins. For example, missense or KASH-less forms of Nesprin-1 and Nesprin-2 are associated with neurological and muscular defects such as Emery Dreifuss muscular dystrophy (EDMD) and spinocerebral ataxia, while mutants of Nesprin-4 are known to cause deafness. Clearly, a detailed molecular understanding of LINC complexes is necessary to understand their distinct biological functions.

The structures of the human heterohexameric SUN2<sub>522-717</sub> – KASH1<sub>8772-8797</sub> and SUN2<sub>522-717</sub> – KASH2<sub>6861-6885</sub> complexes have been determined and provide a general illustration of the mode of interaction at the core of the LINC complex (Figure 1(c) and (d)). The C-terminal ~20 kDa SUN domain (aa 541–717) of SUN2 folds into a compact β sandwich, which trimerizes into a trefoil, supported by a preceding three-stranded coiled-coil element (aa 525–540). KASH1 or 2 bind at the interface between adjacent SUN domains (Figure 1(d)). A loop in the apo-structure folds into a beta hairpin and clamps down the KASH peptide to stabilize the interaction. This element is called the ‘KASH-lid’ (aa 567–587). Since each SUN2 trimer binds 3 KASH peptides at once, the architecture of the complex inherently increasing the interaction strength, distributing expected mechanical force across three discrete sites within the complex. SUN2 also contains a loop structure that coordinates a potassium ion ('cation loop') (aa 593–601), which is essential for KASH binding.

Finally, a disulfide bond can be formed between SUN2 C563 and KASH1 C8774 or KASH2 C6862 via highly conserved cysteine residues. This likely enhances LINC complex stability even further.²³

To advance our understanding of SUN-KASH pairing, we sought to address this question using structural and biochemical tools. We solved high-resolution crystal structures of SUN2₅₂₂₋₇₁₇ in complex with KASH3₉₄₈₋₉₇₅, KASH4₃₇₇₋₄₀₄, and KASH5₅₄₂₋₅₆₂, which complement our previously published SUN2-KASH1 and SUN2-KASH2 complexes. The new structures reveal distinct binding interactions of SUN2 with different KASH peptides. In addition, we solved the structure of unbound, apo-SUN2₅₀₀₋₇₁₇ at high resolution, suggesting a novel mechanism of inactivating Sun. Taken together, this data feeds into the notion that humans have evolved an elaborate LINC-complex network, with the possibility for regulation on multiple levels.

we will refer to these peptides as KASH1-6, for simplicity. KASH peptides were N-terminally fused to 6 histidines, followed by a GB1 solubility tag,²⁴ and a human rhinovirus 3C cleavage site. SUN2 was expressed with a cleavable, trimerizing-GCN4 (triGCN4) tag,²⁵,²⁶ but without an affinity tag (Figure 2(a)). SUN2-KASH complexes were first nickel affinity purified, thereby eliminating apo-SUN2. After proteolytically cleaving GB1 from KASH and triGCN4 from SUN2, SUN2-KASH complexes were separated by size exclusion chromatography, with free GB1 eluting as a separate peak. This way, we were able verify stable interaction of SUN2 with all 6 KASH peptides (Figure 2(b) and (c)).

### Crystallographic analysis of SUN2 KASH complexes

We next set out to structurally characterize SUN2 in complex with KASH3, KASH4, KASH5, and KASH6 by X-ray crystallography. Using a purification strategy discussed previously (Figure 2(c)),²⁵ our initial attempts yielded poorly diffracting crystals of SUN2-KASH3, diffracting in the 4 Å range. After analyzing the crystal packing interactions of these poorly diffracting crystals, we designed SUN2 mutants in the hopes to potentially enhance crystal packing, and hence produce better diffracting crystals.²⁷ Using this approach, we indeed obtained much better diffracting crystals of SUN2 in complex with KASH3 and KASH5 after

---

### Results

#### SUN2 binds to all 6 human KASH peptides

To begin our study, we first examined direct binding of human SUN2 to all 6 currently known KASH proteins. We co-expressed SUN2₅₂₂₋₇₁₇ with the predicted perinuclear KASH peptides of Nesprin-1,-2,-3,-4, CCDC155 (KASH-5), or LRMP (KASH-6) in *Escherichia coli*. Throughout the text

---

### Figure 2. Purification of SUN2-KASH complexes

(a) Primary sequence and cartoon representation of overexpression constructs. For SUN2₅₂₂₋₇₁₇ a cleavable trimerizing GCN4 tag was fused to the N terminus; KASH contains a cleavable N-terminal 6x-Histidine-GB1 affinity/solubility tag. Arrowheads correspond to 3C cut sites; molecular weight of the tagged and digested constructs are denoted.

(b) SDS-PAGE analysis of purification steps of SUN2-KASH preparations. S, soluble protein fraction from bacterial lysate. E, Ni-NTA affinity elution fraction. P, pure protein complex after final gel filtration.

(c) Purification scheme as employed for all SUN2-KASH complex preparations in this study. 6x-His-GB1-KASH was Ni-affinity purified along with bound SUN2 coeluting. Ni-eluate was used for subsequent chromatographic purification steps. Within the chromatograms, regions highlighted in gray were pooled for subsequent purification or crystallization experiments.

introducing 4 surface point mutations in SUN2: Q534D, T683G, M684R, and A685G. For SUN2-KASH4 crystals, wildtype SUN2 was used. Combined with the new tagging and purification scheme described here we finally obtained well diffracting crystals for apo-SUN2<sub>500-717</sub>, SUN2<sub>522-717</sub>-KASH3<sub>948-975</sub>, SUN2<sub>522-717</sub>-KASH4<sub>377-404</sub>, and SUN2<sub>522-717</sub>-KASH5<sub>542-562</sub>. (Table 1). Structures were solved by molecular replacement. The search model used was SUN2 in complex with KASH1 (PDB code 4DXR), after removing both the KASH-lid and the KASH peptide. The general features of SUN-KASH engagement are maintained in all complexes (Figure 3(a)). SUN2 forms a trimer, in the apo- and all KASH-bound forms. In all crystal lattices the trimer is formed through crystallographic symmetry, with the asymmetric unit containing just one protomer. How the 4 surface point mutations improved the diffraction quality of SUN2-KASH3/5 crystals is not obvious, and they had no influence on SUN2 folding or stability. All KASH peptides bind at the interface between adjacent SUN2 monomers despite sequence variations between KASH peptides. However, we also observe a distinct difference between SUN2 bound to KASH1 or 2 as opposed to KASH3, 4, or 5. While KASH1 and 2 kink toward the periphery of the SUN domain,

KASH3, 4, and 5 extend towards the neighboring KASH-lid from the adjacent SUN2 monomer (Figure 3(a) and (b)).

When KASH1, 2, 3, 4, and 5 are aligned, residues between positions 0 (the very C-terminal residue of KASH) to –10 only slightly diverge (Figure 4(a) and (b)). The main differences are start at position –11, which is either a proline or a leucine. When comparing all five KASH peptides side-by-side (Figure 4, S1), we notice that the peptides with proline in –11 form one class, while the other class has a leucine in the same position. The KASH peptides do not share the same binding surface with SUN2 once they exit from under the KASH-lid. Both KASH1 and 2 take a sharp 90° turn at Pro –11 that leads the KASH peptide away from the 3-fold symmetry axis and instead over a neighboring SUN monomer and towards Cys 563 where a disulfide bridge is formed. KASH3, 4, and 5 lack Pro –11, and they adopt the alternate binding conformation described here.

The KASH-lid (aa 567–587) adopts a similar conformation in all five complexes, but it rotates and shifts slightly to accommodate side chains of different sizes present in different KASH peptides. Notably, KASH sequences between position –4

Table 1 Data collection and refinement statistics.

| hsSUN2<sup>a</sup><sub>522-717</sub>-KASH3<sub>948-975</sub> | hsSUN2<sub>522-717</sub>-KASH4<sub>377-404</sub> | hsSUN2<sup>a</sup><sub>522-717</sub>-KASH5<sub>542-562</sub> | hsSUN2<sub>500-717</sub> |
| --- | --- | --- | --- |
| PDB Code | 6WME | 6WMD | 6WMF | 6WMG |
| Data Collection |  |  |  |  |
| Space group | P6<sub>3</sub>22 | P6<sub>3</sub>22 | H32 | H32 |
| Cell dimensions |  |  |  |  |
| a, b, c(Å) | 78.7, 78.7, 173.3 | 80.8, 80.8, 173.9 | 78.4, 78.4, 249.4 | 79.2, 79.2, 199.5 |
| Resolution (Å) | 53.57–1.53 (1.59–1.53) | 69.94–1.50 (1.53–1.50) | 83.15–2.60 (2.69–2.60) | 56.51–1.90 (1.95–1.90) |
| R<sub>p.i.m.</sub>(%) | 1.4 (68.0) | 2.1 (51.4) | 2.5 (46.6) | 4.2 (59.9) |
| I/σ(I) | 68.3 (1.0) | 55.6 (2.0) | 37.3 (0.9) | 24.0 (1.4) |
| CC<sub>1/2</sub>(%) | 99.9 (60.8) | 99.9 (61.8) | 99.9 (50.9) | 99.6 (65.2) |
| Completeness (%) | 98.6 (87.1) | 99.9 (98.6) | 99.2 (94.1) | 99.0 (97.0) |
| Redundancy | 32.8 (25.4) | 23.3 (19.9) | 15.9 (6.3) | 3.7 (3.4) |
| Refinement |  |  |  |  |
| Resolution (Å) | 53.57–1.53 (1.59–1.53) | 69.94–1.50 (1.53–1.50) | 83.15–2.60 (2.73–2.60) | 56.51–1.90 (1.95–1.90) |
| No. reflections | 47,678 | 54,661 | 9383 | 19,072 |
| R<sub>work</sub>/R<sub>free</sub> | 20.7/23.8 | 16.3/18.5 | 22.1/28.7 | 17.1/22.1 |
| No. atoms | 1980 | 2106 | 1694 | 1672 |
| Protein | 1736 | 1739 | 1681 | 1541 |
| Solvent, ions | 244 | 367 | 13 | 131 |
| Wilson B factor (Å²) | 27.7 | 20.1 | 79.3 | 37.5 |
| B factors (Å²) | 40.4 | 30.3 | 96.4 | 52.3 |
| Protein | 38.9 | 28.3 | 96.6 | 52.2 |
| Solvent, ions | 50.7 | 40.1 | 77.9 | 53.9 |
| Root mean squared deviations |  |  |  |  |
| Bond lengths (Å) | 0.006 | 0.008 | 0.008 | 0.017 |
| Bond angles (°) | 0.948 | 0.980 | 1.035 | 1.454 |

a This SUN2 construct contains 4 surface point mutations: Q534D, T683G, M684R, A685G).

6032                                                                 LINC Complex Binding Modes

(a)

SUN2-KASH1                                                                 SUN2-KASH4

KASH1 "N-term"                                                                 KASH4 "N-term"
Disulfide                                                                 KASH lid
KASH1 C-term "PPPL"                                                                 KASH4 C-term "LPPV"

60°                                                                 60°

KASH1 "N-term"                                                                 KASH4 "N-term"
Disulfide                                                                 KASH1 C-term "PPPL"                                                                 KASH4 C-term "LPPV"

(b)

KASH lid                                                                 KASH lid
SUN2                                                                 SUN2
KASH3/4/5 "N-term"                                                                 KASH3/4/5 "N-term"
KASH C-term "PPPX"                                                                 β-Sandwich                                                                 KASH1/2 "N-term"
SUN2                                                                 KASH1                                                                 KASH2
β-Sandwich                                                                 KASH3
SUN2                                                                 KASH4
KASH1/2 "N-term"                                                                 KASH5

Figure 3. Comparison of SUN-KASH binding modes. (a) Space-filling cartoon representation of the two main binding modes, illustrated using the best resolved complex structures for each class (left, SUN2-KASH1, PDB code 4DXR; right, SUN2-KASH4, 6WMD). SUN2 trimer subunits in shades of blue. KASH1 peptide in white, KASH4 peptide in orange. Note the kinked vs. straight conformation of the KASH-peptide. (b) Superposition of the five SUN2-bound KASH peptides, including important SUN2 binding elements. The SUN2 conformation is very similar in all five complex structures.

LINC Complex Binding Modes

(a)

| KASH1 | KASH2 | KASH3 | KASH4 | KASH5 |
|-------|-------|-------|-------|-------|
| ![KASH1](https://via.placeholder.com/100) | ![KASH2](https://via.placeholder.com/100) | ![KASH3](https://via.placeholder.com/100) | ![KASH4](https://via.placeholder.com/100) | ![KASH5](https://via.placeholder.com/100) |

(b)

| H. sapiens KASH1 8769 | s e e d y S C A L S N N F A R S F H P M L R Y T N G P P P L 8797 |
|------------------------|-------------------------------------------------------------|
| H. sapiens KASH2 6857 | - e e d y S C T Q A N N F A R S F Y P M L R Y T N G P P P T 6885 |
| H. sapiens KASH3 948  | - e e d r s c t l a n n F A R S F T L M L R Y N - G P P P T 975 |
| H. sapiens KASH4 377  | - - s g g p c c s h a r i P R T P Y L V L S Y V N G L P P V 404 |
| H. sapiens KASH5 542  | - - - - - - - - g p s p p p T W P H L Q L C Y L - Q P P P V 562 |

Figure 4. Details of SUN2-bound KASH peptides. (a) KASH1-5 are illustrated in stick representation and oriented identically as they are positioned in the SUN2 binding interface. The two views provide detail over the entire KASH sequence. Important positions are numbered, starting at position ‘0’, the very C-terminal residue. Notice position –11, which is where the 90° kink in KASH1 and KASH2 occurs. (b) Structure-based sequence alignment of the 5 KASH peptides for which complex structures with SUN2 exist. Sequence conservation indicated in a blue color gradient. Lower case letters represent residues in the crystallized construct that were invisible, i.e., presumably disordered, in the respective crystal structures.

and –6 of different lengths are accommodated (Figures 3(b) and 4(a) and (b)) and are partially solvent exposed. KASH3/5 possess two residues here whereas KASH1/2/4 have three. To accommodate the lacking residues at positions –5 and –4, KASH3 and KASH5, respectively, form a shorter loop that still maintains the correct register for the remainder of the KASH peptide. Residues preceding this loop are well aligned between different KASH peptides, and so are residues after this loop (Figure 4(b) and (c)). The tyrosine at positions –7 clearly functions as an anchor that is critical in determining the register of the KASH peptide for residues –8 to –11 and facilitates the hydrogen-bonding pattern between

the KASH peptide and the KASH-lid. The solvent exposed loop between positions –4 to –6 together with the anchoring by Tyr –7 grants flexibility in the sequence and length of residues that can be accommodated between the Tyr –7 and the conserved “PPPX” motif.

### The cation loop of SUN2

We can resolve the electron density of the cation loop (Figure 1(c)) in all the structures reported here, but it is especially well ordered in the high-resolution structures of the SUN2-KASH3 and SUN2-KASH4 complexes (Figure 5). The cation loop of SUN2 in complex with KASH3 (Figure 5

6034                                                                 LINC Complex Binding Modes

(a) SUN2-KASH3 (b) SUN2-KASH4

Asp 595
Gln 593
Val 590
Tyr 707
Asn 600

Asp 595
Gln 593
Val 590
Tyr 707
Asn 600

Figure 5. Details of the SUN2 cation loop. (a) Detailed view of the canonical potassium coordination in the cation loop, illustrated using the SUN2-KASH3 coordinates. $2F_{o}-F_{c}$ electron density map contoured at $1.5\sigma$. (b) The SUN2-KASH4 complex constitutes an exception, as it shows two water molecules positioned in the area that is typically occupied by a potassium ion. $2F_{o}-F_{c}$ electron density map contoured at $1.5\sigma$.

(a)) is identical to what is observed in the apo form of SUN2, as well as SUN2 bound to KASH1/2/5. In these structures a potassium ion is coordinated at the center of the cation loop by the carbonyl groups of Val 590, Gln 593, Asp 595, Asn 600, Tyr 707, and by a well-ordered water molecule. Surprisingly, the cation loop of SUN2 in complex with KASH4 lacks a cation (Figure 5(b)). Instead of a potassium ion this structure contains two water molecules that coordinate the cation loop interactions. The first water molecule hydrogen bonds with the main chain carbonyl of Val 590, Gln 593, with the main chain amide of Asp 595, and with its neighboring water molecule. The second water coordinates the carbonyl of Asn 600 and of Tyr 707. The main chain carbonyl of Asp 595 now hydrogen bonds with the amine of the Asn 600 side chain and its backbone carbonyl points outside of the cation loop. In the presence of potassium, the side chain of Asp 595 pairs with the amine of the Asn 600 side chain, however, in the absence of potassium the side chain of Asp 595 points away from the cation loop and is no longer as well ordered in our density maps.

binding pocket that coordinates the conserved "PPPX" motif of KASH (Figure 6). In the context of KASH1 and KASH2, Cys 601 and Cys 705 of SUN2 form a disulfide. Here we observe that when SUN2 is bound to KASH3, KASH4, or KASH5, these cysteines are reduced. Because of the high quality of our electron density maps the redox state of Cys 601 and Cys 705 is unambiguous.

The unbound SUN domain may form a self-blocked state

The main motivation behind solving another apo-SUN2 structure was that we extended the construct N-terminally back to residue 500, in the hopes of resolving a longer stretch of the non-canonical trimeric coiled-coil element of SUN2 (Figure 7). However, these additional residues are not visible in the structure, suggesting that they are unstructured in the context of this construct. Our new structure superimposes well with the published apo-SUN structures: 0.402 Å RMSD with 4DXT0.568 Å RMSD with 3UNP. All published and trimeric SUN crystal structures pack similarly, i.e. with the KASH-lid containing, ONM facing side engaging in a head-to-head contact (Figure 7 (a) and (b)). So far, we concluded that this interaction is non-physiological, primarily on two grounds. First, this conformation would sterically hinder the proper engagement of KASH peptides with the ONM, with the trans-membrane segment of the

Internal disulfide of SUN2

SUN proteins contain a pair of universally conserved cysteines (Cys 601 and Cys 705 in SUN2) which form a disulfide bond that presumably stabilizes the cation loop of SUN2. Additionally, this disulfide forms part of the KASH

LINC Complex Binding Modes

(a) KASH2 "PPPT"
- Pro 6882
- Thr 6885
- Pro 6883
- Pro 6884
- Asn 600
- Cys 601
- SUN2
- Thr 704
- Cys 705
- Ile 706

(b) KASH4 "LPPV"
- Leu 401
- Val 404
- Pro 402
- Pro 403
- Asn 600
- Cys 601
- SUN2
- Thr 704
- Cys 705
- Ile 706

Figure 6. Details of the “PPPX” binding vicinity. (a) Detailed view of the SUN2 area in close contact with the C-terminal 3 residues of the KASH peptide. In the KASH1/2 bound structures the cysteine 601/705 pair is best modeled as oxidized, as shown. 2F<sub>o</sub>-F<sub>c</sub> electron density map contoured at 1.5σ. (b) The same detailed view as in (a) but showing the KASH4 interaction. Here, the cysteine 601/705 pair is best modeled as reduced. 2F<sub>o</sub>-F<sub>c</sub> electron density map contoured at 1.5σ.

KASH peptide immediately adjacent to the C-terminal SUN-binding portion. Second, in the KASH-bound form the interaction surface between adjacent trimers is rather small and generally does not pack well. However, in the apo-form this head-to-head crystal interface has a few remarkable features (Figure 7(a) and (c)). In comparison to the apo-form (Figure 7(a)), the KASH-bound head-to-head interface (Figure 7(b)) is driven apart substantially, while losing many of the packing contacts. The buried interface in the apo-SUN2 crystal measures 2380 Å², well in the range of stable interfaces. For KASH-bound interfaces, the buried area measures below 400 Å². Consistent with this, analysis using the Proteins, Interfaces, Structures and Assemblies (PISA) server reports a dissociation energy of 8.2 kcal/mol for the apo-SUN2 homohexamer, suggesting the interface to be stable in solution. For SUN2-KASH complexes, no such assembly involving six SUN2 protomers is suggested. In addition, the KASH-lid is dramatically remodeled in the apo-SUN2 form. Overlooked in

6036                                                                 LINC Complex Binding Modes

(a)                                                                 (b)

KASH lid

(c)                                                                 (d)

Figure 7. Head-to-head packed apo-SUN2 homotrimers may represent a self-locked state. (a) Packing interface in the crystal between two adjacent apo-SUN2 homotrimers. The KASH-lid (orange) is prominently involved in the 2380 Å² interface. (b) The same packing interface as in (a) but showing the SUN2-KASH4 interaction instead. Notice how the two heterohexamers are much further apart and how the packing interface is significantly smaller. (c) and (d) Are close-ups of the head-to-head interfaces shown in (a) and (b), respectively. Highly conserved residues involved in the postulated self-locked, apo-SUN2 homohexamer are labeled. The labeled interaction network is repeated 6 times in the apo-SUN2 hexameric interface. In the KASH-bound interface, these residues have no comparable role.

previous publications, the apo-conformation specifically buries the highly conserved hydrophobic residues Ile 579 and Leu 581 in the head-to-head interface (Figure 7(a) and (c)). According to PISA analysis, burying these hydrophobics is the main enthalpic contribution driving the interaction. Finally, the well conserved Trp 582 packs tightly against the

Glu 552 – Arg 589 salt bridge in the apo-form. The conservation of Ile 579, Leu 581 and Trp 582 cannot be explained by the SUN-KASH interaction, in which these residues are not directly involved. On the other hand, a functional, self-locked apo-state would readily explain the conservation of these residues, including the Glu 552 – Arg 589 salt bridge.

LINC Complex Binding Modes

Mixed occupancy LINC complexes

After confirming that the core interactions between SUN2 and KASH1-5 are similar, we decided to test if a single SUN2 trimer could bind to two different KASH peptides simultaneously and if there is a preference for either binding mode (Figure 8). The same purification scheme shown in Figure 2 was used in this assay, but an additional GFP fused KASH peptide was co-expressed. Since only one KASH type was affinity-tagged, pulling down the other KASH type indicates a mixed LINC complex. This way, we tested if SUN2 could bind to KASH1 and 2 simultaneously. 6xHis-GB1-KASH2 co-purified with both, SUN2 and GFP-KASH1. Similarly, 6xHis-GB1-KASH4 co-purified with SUN2 and GFP-KASH3. These results show that one SUN2-trimer can bind different KASH peptides at once that share the same binding mode. We were also able to purify SUN2 bound to both 6xHis-GB1-KASH2 and GFP-KASH3, simultaneously, showing that SUN2 can bind two KASH peptides that adopt different binding modes. These experiments support the idea that the individual KASH binding sites are independent of one another and that mixed occupancy has to be considered as a means of regulating LINC complexes.

Discussion

Previous work has shown that SUN and KASH interactions occur with promiscuity, which raises the question of how these SUN KASH bridges can be specifically assembled and regulated. Here we show that SUN2 engages with a KASH peptide in at least two distinct binding modes. We can speculate whether a proline or leucine in the –11 position of KASH may be the determining factor to have the peptide engage in a kinked or linear manner, respectively. The argument for such a determination lies within the difference between the binding of KASH1/2 vs. KASH3. While KASH1/2 are kinked, KASH3 is straight. This is remarkable, because these three peptides are very similar in sequence (Figure 4(b)) and it is suggestive that the proline to leucine change triggers the conformational change. To be certain, though, this would need to be tested experimentally. While we now have the structural evidence that SUN2, and most likely the close homolog SUN1 as well, are inherently promiscuous in binding the 6 different KASH peptides, it is not yet clear whether the same will be true for the testis-specific SUN proteins, SUN3, –4, and –5. Those sequences are far enough diverged from SUN2 to preclude a definitive analysis in the absence of structural data.

An outstanding question in the field remains the presumed higher-order assembly of LINC-complexes into 2D clusters [7]. Such clusters are thought to be necessary in order for LINC complexes to bear significant mechanical load, i.e. during nuclear migration [23,28]. One easy way to envision cluster formation is that KASH-proteins themselves homo-oligomerize through their cytoplasmic elements. That way, KASH-proteins,

(a)  
![Image](image_a.png)

(b)  
SUN2-KASH1/2  
![Image](image_b.png)

Figure 8. A Sun trimer can bind different KASH peptides simultaneously. (a) SUN2<sub>522-717</sub> was recombinantly co-expressed in bacteria with two KASH peptides, one His-tagged, the other GFP-tagged. The complexes were co-purified using Ni-NTA affinity, establishing that a SUN2 trimer can simultaneously bind at least two different KASH peptides. The SDS-PAGE shows three KASH combinations and a negative control experiment. S, soluble bacterial protein extract; U, unbound fraction; E, elution fraction from Ni-affinity experiment. (b) Representative gel filtration experiment using SUN2<sub>522-717</sub>/6xHis-GB1-KASH2/GFP-KASH1 complex purified via Ni-affinity. Chromatogram shows that GFP-KASH1 co-elutes with the main protein peak, confirming integration into the complex. SDS-PAGE analysis (inset) of gel filtration fractions (range indicated in red) confirms that all three input proteins co-elute, indicating stable complex formation. The assay cannot distinguish between the two possible stoichiometries, i.e. a SUN2 trimer bound to either two 6xHis6-GB1-KASH2 and one GFP-KASH1 molecule, or the inverse. With SUN2<sub>522-717</sub> at 28 kDa, 6xHis-GB1-KASH2 at 12 kDa, and GFP-KASH1 at 31 kDa, the molecular weight difference of (28x3 + 12x2 + 31 = ) 139 kDa vs (28x3 + 12 + 31x2 = ) 158 kDa is not resolved in this experimental setup. Apparent molecular weights of three globular standard proteins and the void volume are indicated.

together with SUN-trimers, can form higher-order arrays. \({ }^{11}\) In this model, one KASH-protein oligomer bridges several SUN-trimers. This is particularly obvious, if the KASH-protein oligomer is not a trimer itself. With our data that formally shows that a single SUN-trimer can bind different KASH-peptides simultaneously, we also have to entertain that higher-order clusters may consist of mixed SUN-KASH complexes. This is an aspect of LINC complex biology that so far has been largely ignored. Another element that has not garnered a lot of attention is the consequence of Nesprins having multiple spectrin repeats, which are protein interaction domains. One can envision highly complex ways in which Nesprins contribute to 2D clusters that still await structural and functional analysis.

We observe subtle differences in the cation loop and the redox state of the close cysteines 601 and 705, depending on which of the five KASH peptides is bound (Figures 5 and 6). It remains to be seen whether these differences are biologically meaningful. For example, does the presence of potassium in the cation loop prevent KASH4 interaction, since it is the only structure with water coordination in the cation site? Similarly, is it relevant that the C601-C705 disulfide is reduced in the KASH-3,-4, and -5 bound structures? We do not find any obvious reason, why we see these differences and whether or not they are influenced by the KASH peptides, or whether they may simply be the result of different crystallization conditions. We believe it will be interesting to see more SUN-KASH pairs structurally characterized, at or near atomic resolution, to give a more detailed picture.

Finally, there is an ongoing debate about the regulation of the assembly state of SUN. In order for SUN to bind KASH in a productive and meaningful manner, the cell has to ensure that binding only occurs after SUN has reached the inner nuclear membrane, i.e. traveled from its insertion site within the endoplasmic reticulum, past the circular opening around NPCs, to its destination (Figure 9). If SUN binds KASH before passage to the INM, it will be sterically blocked to pass the NPC, and therefore presumably be non-functional. One way to envision that KASH engagement only happens after SUN reaches its destination is that SUN is kept in a non-binding state before. Studies by the Feng lab suggest that SUN1/2 can adopt a monomeric, KASH-binding incompetent state. \({ }^{29}\) In these structural studies the perinuclear coiled-coil element engages with the SUN domain to occlude the KASH binding site. Thus, one can envision that SUN is kept in this inactive form until it reaches the INM. Once there, a yet-to-be-identified signal could switch SUN from inactive to active. We hypothesize that the head-to-head self-locked apo-SUN structure we observe may better reflect a KASH-binding inhibited state. First, the monomeric SUN form described in \({ }^{29}\) is dependent on a specific coiled-coil truncation being present. Whether this is representative of the behavior of full-length SUN is not proven, a significant limitation to the study. In contrast, for our self-locked structure, there is no principal concern why it could not be physiologically relevant. In fact, since the self-locking would work by tilting the SUN domains away from the ONM, it would be an elegant and simple solution to prevent KASH binding

![Diagram](attachment:diagram.png)

**Figure 9.** Working model for the role of a self-locked apo-SUN form in LINC-Complex maturation. SUN proteins and Nesprins are separately inserted into the endoplasmic reticulum (ER) and the outer nuclear membrane (ONM) (step 1). SUN proteins have to travel past the nuclear pore complex (NPC) to the inner nuclear membrane (INM) before they can functionally engage with ONM-localized Nesprins. Premature binding to Nesprins in the ER would lead to aberrant LINC complexes (crossed out red). A self-locked, hexameric apo-SUN (step 2) would prevent Nesprin engagement in the ER and could be used to safely chaperone SUN to the INM (step 3) where it will properly engage with Nesprins (step 4).
(LINC Complex Binding Modes)

(Figure 9). Our apo-structure now provides a framework for specifically studying a self-locked state in vivo.

In summary, we report near-atomic resolution structures of several SUN2-KASH complexes that shed light on the details of the mode of interaction. Also, we formally show that a Sun trimer can engage with different KASH proteins simultaneously. Much is still to be learned about the nuclear and cytoplasmic anchorage of LINC complexes, as well as functional assembly states, important elements on the path toward a full molecular understanding of mechanotransduction across the NE.

### Materials and Methods

#### Plasmid construction, protein expression and purification

DNA sequences containing human SUN2<sub>522-727</sub> were cloned into a modified bicistronic ampicillin resistant pETDuet-1 vector (EMD Millipore), superfolder GFP-KASH fusions (sfGFP) were cloned and expressed into a modified ampicillin resistant vector, pET-30b(+) (EMD Biosciences). Each KASH peptide (KASH1-6) was C-terminally fused to 6xHistidine-sfGFP using inverse PCR. For crystallography, 6xHistidine tagged SUN2<sub>522-717</sub> was cloned into the first multiple cloning site (MCS) and MBP-fused KASH4<sub>379-404</sub> was cloned into the second MCS. For SUN2<sub>522-717</sub> – KASH3<sub>947-975</sub> and SUN2<sub>522-717</sub> – KASH5<sub>542-562</sub> the KASH peptides were C-terminally fused to 6xHistidine-GB1 tags and SUN2<sub>522-717</sub> remained untagged. All SUN2 and KASH constructs were N-terminally fused with a human rhinovirus 3C protease cleavage site.

Transformed LOBSTR(DE3)-RIL bacterial expression cells<sup>30</sup> were grown at 37 °C to an A<sub>600</sub> of 0.6, then shifted to 18 °C and induced with 0.2 M isopropyl β-D-1-thiogalactopyranoside for 16 h. Cells were harvested by centrifugation at 6000g, resuspended in lysis buffer (50 mM potassium phosphate, pH 8.0, 400 mM NaCl, 40 mM imidazole) and lysed using an LM20 Microfluidizer Processor (Microfluidics). The lysate was cleared by centrifugation at 10,000g for 25 min. The soluble fraction was incubated with Nickel Sepharose 6 Fast Flow beads (GE Healthcare) for 30 min at 4 °C in batch. After the beads were washed with lysis buffer, the protein was eluted (10 mM Tris/ HCl pH 8.0, 150 mM NaCl, 250 mM imidazole). The protein was further purified by size exclusion chromatography using an S75 or S200 16/60 column (GE Healthcare) equilibrated in running buffer (10 mM Tris/HCl, pH 8.0, 150 mM NaCl, and 0.2 mM EDTA).

Protein affinity and solubility-tags were removed with 3C protease and the protein complexes were separated from fusion tags by a second size exclusion step using an S75 16/60 column, under the same conditions.

#### Crystallization

Purified Apo-SUN2<sub>500-717</sub> was concentrated to 7 mg/ml and crystallized in 14.5% (w/v) polyethylene glycol (PEG) 3350, 100 mM potassium formate pH 8.0, and 200 mM sodium bromide. Rhombohedral crystals appeared after 5 days at 18 °C. SUN2<sub>522-717</sub> – KASH3<sub>947-975</sub> complex was concentrated to 5 mg/ml, and crystallized in 16–18% PEG3350, 100 mM ammonium citrate pH 7.0, 100 mM BisTris/HCl pH 6.5, and 10 mM nickel (II) chloride. Large, plate-shaped crystals grew within 2 days at 18 °C. SUN2<sub>522-717</sub> – KASH4<sub>379-404</sub> crystallized at 7 mg/ml in 17% PEG3350, 200 mM magnesium chloride, and 100 mM BisTris/HCl pH 6.5. Crystals appeared after 1 day and finished growing within 3 days, at 18 °C. SUN2<sub>522-717</sub> – KASH5<sub>542-562</sub> was crystallized at 10 mg/ml, in 14% PEG3000, and 100 mM BisTris/HCl pH 6.5. Crystals appeared and finished growing within 1–12 h at 4 °C. Crystals were cryoprotected in their reservoir solution supplemented with 15% glycerol and flash-frozen in liquid nitrogen. Diffraction data were collected at beamline 24ID-C at Argonne National Laboratories.

#### X-ray data collection and structure determination

All data processing was done using programs provided by SBgrid.<sup>31</sup> Data reduction was carried out with HKL2000,<sup>32</sup> molecular replacement with PHASER from the phenix suite,<sup>33</sup> using apo-SUN2 structure (PDB code 4DXT) that lacked the KASH-lid as a search model. The structures were manually built using Coot<sup>34</sup> and refined with phenix.refine. Data collection and refinement statistics are summarized in Table 1. Structure figures were created using PyMOL.<sup>35</sup>

#### Accession Numbers

Coordinates and structure factors have been deposited in the Protein Data Bank under accession codes PDB **6WME** (SUN2<sub>522-717</sub> – KASH3<sub>947-975</sub>), **6WMD** (SUN2<sub>522-717</sub> – KASH4<sub>379-404</sub>), **6WMF** (SUN2<sub>522-717</sub> – KASH5<sub>542-562</sub>), and **6WMG** (apo-SUN2<sub>500-717</sub>).

#### CRedit authorship contribution statement

Victor E. Cruz: Methodology, Investigation, Visualization, Writing - original draft, Writing - review & editing. F. Esra Demircioglu: Methodology, Investigation. Thomas U. Schwartz: Conceptualization, Supervision,
6040

LINC Complex Binding Modes

Methodology, Investigation, Visualization, Writing - original draft, Writing - review & editing.

---

### Acknowledgments

This work was supported by NIH grant R01-AR065484 to T.U.S., the NIH Pre-Doctoral Training Grant T32GM007287 (V.E.C.) and is based on research conducted at the Northeastern Collaborative Access Team beamlines, which are supported by NIH grants P30GM124165 and S10OD021527, and DOE contract DE-AC02-06CH11357. We thank the staff at the Advanced Photon Source beamline 24-ID-C, K. Rajashankar in particular, for assistance in data collection and space group analysis.

### Declaration of Competing Interest

The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.

### Appendix A. Supplementary material

Supplementary data to this article can be found online at [https://doi.org/10.1016/j.jmb.2020.09.019](https://doi.org/10.1016/j.jmb.2020.09.019).

Received 26 April 2020; Accepted 23 September 2020; Available online 12 October 2020

#### Keywords:
SUN proteins; KASH proteins; nucleo-cytoplasmic communication; mechanotransduction; nuclear envelope

---

### References

1. Knockenhauer, K.E., Schwartz, T.U., (2016). The nuclear pore complex as a flexible and dynamic gate. *Cell*, 164, 1162–1171. [https://doi.org/10.1016/j.cell.2016.01.034](https://doi.org/10.1016/j.cell.2016.01.034).
2. Wente, S.R., Rout, M.P., (2010). The nuclear pore complex and nuclear transport. *CSH Perspect. Biol.*, 2, [https://doi.org/10.1101/cshperspect.a000562](https://doi.org/10.1101/cshperspect.a000562) a000562.
3. Görlich, D., Kutay, U., (1999). Transport between the cell nucleus and the cytoplasm. *Annu. Rev. Cell. Dev. Biol.*, 15, 607–660. [https://doi.org/10.1146/annurev.cellbio.15.1.607](https://doi.org/10.1146/annurev.cellbio.15.1.607).
4. Luxton, G.G., Starr, D.A., (2014). KASHing up with the nucleus: novel functional roles of KASH proteins at the cytoplasmic surface of the nucleus. *Curr. Opin. Cell Biol.*, 28, 69–75. [https://doi.org/10.1016/j.ceb.2014.03.002](https://doi.org/10.1016/j.ceb.2014.03.002).
5. Starr, D.A., Fridolfsson, H.N., (2010). Interactions between nuclei and the cytoskeleton are mediated by SUN-KASH nuclear-envelope bridges. *Annu. Rev. Cell Dev. Biol.*, 26,
6. Lee, Y.L., Burke, B., (2018). LINC complexes and nuclear positioning. *Semin. Cell Dev. Biol.*, 82, 67–76. [https://doi.org/10.1016/j.semcdb.2017.11.008](https://doi.org/10.1016/j.semcdb.2017.11.008).
7. Hao, H., Starr, D.A., (2019). SUN/KASH interactions facilitate force transmission across the nuclear envelope. *Nucleus*, 10, 73–80. [https://doi.org/10.1080/19491034.2019.1595313](https://doi.org/10.1080/19491034.2019.1595313).
8. Baum, D.A., Baum, B., (2014). An inside-out origin for the eukaryotic cell. *BMC Biol.*, 12, 76. [https://doi.org/10.1186/s12915-014-0076-2](https://doi.org/10.1186/s12915-014-0076-2).
9. Majumder, S., Willey, P.T., DeNies, M.S., Liu, A.P., Luxton, G.W.G., (2019). A synthetic biology platform for the reconstitution and mechanistic dissection of LINC complex assembly. *J. Cell Sci.*, 132, jcs234153. [https://doi.org/10.1242/jcs.234153](https://doi.org/10.1242/jcs.234153).
10. de Leeuw, R., Gruenbaum, Y., Medalia, O., (2018). Nuclear Lamins: thin filaments with major functions. *Trends Cell Biol.*, 28, 34–45. [https://doi.org/10.1016/j.tcb.2017.08.004](https://doi.org/10.1016/j.tcb.2017.08.004).
11. Sosa, B.A., Kutay, U., Schwartz, T.U., (2013). Structural insights into LINC complexes. *Curr. Opin. Struct. Biol.*, 23, 285–291. [https://doi.org/10.1016/j.sbi.2013.03.005](https://doi.org/10.1016/j.sbi.2013.03.005).
12. Rothballer, A., Kutay, U., (2013). The diverse functional LINC's of the nuclear envelope to the cytoskeleton and chromatin. *Chromosoma*, 122, 415–429. [https://doi.org/10.1007/s00412-013-0417-x](https://doi.org/10.1007/s00412-013-0417-x).
13. Crisp, M., Liu, Q., Roux, K., Rattner, J.B., Shanahan, C., Burke, B., Stahl, P.D., Hodzic, D., (2006). Coupling of the nucleus and cytoplasm: role of the LINC complex. *J. Cell Biol.*, 172, 41–53. [https://doi.org/10.1083/jcb.200509124](https://doi.org/10.1083/jcb.200509124).
14. Padmakumar, V.C., Libotte, T., Lu, W., Zaim, H., Abraham, S., Noegel, A.A., Gotzmann, J., Foisner, R., et al., (2005). The inner nuclear membrane protein Sun1 mediates the anchorage of Nesprin-2 to the nuclear envelope. *J. Cell Sci.*, 118, 3419–3430. [https://doi.org/10.1242/jcs.02471](https://doi.org/10.1242/jcs.02471).
15. Mellad, J.A., Warren, D.T., Shanahan, C.M., (2011). Nesprins LINC the nucleus and cytoskeleton. *Curr. Opin. Cell Biol.*, 23, 47–54. [https://doi.org/10.1016/j.ceb.2010.11.006](https://doi.org/10.1016/j.ceb.2010.11.006).
16. Morimoto, A., Shibuya, H., Zhu, X., Kim, J., Ishiguro, K.-I., Han, M., Watanabe, Y., (2012). A conserved KASH domain protein associates with telomeres, SUN1, and dynactin during mammalian meiosis. *J. Cell Biol.*, 198, 165–172. [https://doi.org/10.1083/jcb.201204085](https://doi.org/10.1083/jcb.201204085).
17. Lindeman, R.E., Pelegri, F., (2012). Localized products of futile cycle/lrmp promote centrosome-nucleus attachment in the zebrafish zygote. *Curr. Biol.*, 22, 843–851. [https://doi.org/10.1016/j.cub.2012.03.058](https://doi.org/10.1016/j.cub.2012.03.058).
18. Burke, B., Stewart, C.L., (2014). Functional architecture of the cell’s nucleus in development, aging, and disease. *Curr. Top. Dev. Biol.*, 109, 1–52. [https://doi.org/10.1016/b978-0-12-397920-9.00006-8](https://doi.org/10.1016/b978-0-12-397920-9.00006-8).
19. Zhou, C., Rao, L., Shanahan, C.M., Zhang, Q., (2018). Nesprin-1/2: roles in nuclear envelope organisation, myogenesis and muscle disease. *Biochem. Soc. Trans.*, 46, 311–320. [https://doi.org/10.1042/bst20170149](https://doi.org/10.1042/bst20170149).
20. Horn, H.F., Brownstein, Z., Lenz, D.R., Shivatzki, S., Dror, A.A., Dagan-Rosenfeld, O., Friedman, L.M., Roux, K.J., et al., (2013). The LINC complex is essential for hearing. *J. Clin. Invest.*, 123, 740–750. [https://doi.org/10.1172/jci66911](https://doi.org/10.1172/jci66911).
21. Wang, W., Shi, Z., Jiao, S., Chen, C., Wang, H., Liu, G., Wang, Q., Zhao, Y., et al., (2012). Structural insights into

SUN-KASH complexes across the nuclear envelope. Cell Res., 22, 1–13. https://doi.org/10.1038/cr.2012.126.

22. Sosa, B.A., Rothballer, A., Kutay, U., Schwartz, T.U., (2012). LINC complexes form by binding of three KASH peptides to domain interfaces of trimeric SUN proteins. Cell, 149, 1035–1047. https://doi.org/10.1016/j.cell.2012.03.046.

23. Cain, N.E., Jahed, Z., Schoenhofen, A., Valdez, V.A., Elkin, B., Hao, H., Harris, N.J., et al., (2018). Conserved SUN-KASH interfaces mediate LINC complex-dependent nuclear movement and positioning. Curr. Biol., 28, 3086–3097.e4. https://doi.org/10.1016/j.cub.2018.08.001.

24. Bao, W.-J., Gao, Y.-G., Chang, Y.-G., Zhang, T.-Y., Lin, X.-J., Yan, X.-Z., Hu, H.-Y., (2006). Highly efficient expression and purification system of small-size protein domains in Escherichia coli for biochemical characterization. Protein Express Purif., 47, 599–606. https://doi.org/10.1016/j.pep.2005.11.021.

25. Demircioglu, F.E., Schwartz, T.U., (2016). Purification and structural analysis of SUN and KASH domain proteins. Methods Enzymol., 569, 63–78. https://doi.org/10.1016/bs.mie.2015.08.011.

26. Ciani, B., Bjelic, S., Honnappa, S., Jawhari, H., Jaussi, R., Payapilly, A., Jowitt, T., et al., (2010). Molecular basis of coiled-coil oligomerization-state specificity. Proc. Natl. Acad. Sci. USA, 107, 19850–19855. https://doi.org/10.1073/pnas.1008502107.

27. Partridge, J.R., Schwartz, T.U., (2009). Crystallographic and biochemical analysis of the Ran-binding zinc finger domain. J. Mol. Biol., 391, 375–389. https://doi.org/10.1016/j.jmb.2009.06.011.

28. Luxton, G.W.G., Gomes, E.R., Folker, E.S., Vintinner, E., Gundersen, G.G., (2010). Linear arrays of nuclear envelope

proteins harness retrograde actin flow for nuclear movement. Science, 329, 956–959. https://doi.org/10.1126/science.1189072.

29. Nie, S., Ke, H., Gao, F., Ren, J., Wang, M., Huo, L., Gong, W., Feng, W., (2016). Coiled-coil domains of SUN proteins as intrinsic dynamic regulators. Structure, 24, 80–91. https://doi.org/10.1016/j.str.2015.10.024.

30. Andersen, K.R., Leksa, N.C., Schwartz, T.U., (2013). Optimized *E. coli* expression strain LOBSTR eliminates common contaminants from His-tag purification. Proteins Struct. Funct. Bioinform., 81, 1857–1861. https://doi.org/10.1002/prot.24364.

31. Morin, A., Morin, A., Eisenbraun, B., Eisenbraun, B., Key, J., Key, J., Sanschagrin, P.C., Sanschagrin, P.C., et al., (2013). Collaboration gets the most out of software. Elife, 2, e01456. https://doi.org/10.7554/elife.01456.

32. Otwinowski, Z., Minor, W., (1997). Processing of X-ray diffraction data collected in oscillation mode. Methods Enzymol., 276, 307–326.

33. Liebschner, D., Afonine, P.V., Baker, M.L., Bunkóczi, G., Chen, V.B., Croll, T.I., Hintze, B., Hung, L.W., et al., (2019). Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix. Acta Crystallogr. D, 75, 861–877. https://doi.org/10.1107/s2059798319011471.

34. Emsley, P., Lohkamp, B., Scott, W.G., Cowtan, K., (2010). Features and development of Coot. Acta Crystallogr. sect. D Biol. Crystallogr., 66, 486–501. https://doi.org/10.1107/s0907444910007493.

35. DeLano, W., (2002). PyMOL, http://www.pymol.org.
