
Desmosome architecture derived from molecular dynamics simulations and cryo-electron tomography

Mateusz Sikora${}^{a,b,1}$, Utz H. Ermel${}^{c,d,1}$, Anna Seybold${}^{c,d,1}$, Michael Kunz${}^{c,d}$, Giulia Calloni${}^{c,e}$, Julian Reitz${}^{c,d}$, R. Martin Vabulas${}^{f}$, Gerhard Hummer${}^{a,c,2}$, and Achilleas S. Frangakis${}^{c,d,2}$

${}^{a}$Theoretical Biophysics Department, Max Planck Institute for Biophysics, 60438 Frankfurt, Germany; ${}^{b}$Faculty of Physics, University of Vienna, 1090 Vienna, Austria; ${}^{c}$Buchmann Institute for Molecular Life Sciences, 60438 Frankfurt, Germany; ${}^{d}$Institute for Biophysics, Goethe University Frankfurt, 60438 Frankfurt, Germany; ${}^{e}$Institute for Biophysical Chemistry, Goethe University Frankfurt, 60438 Frankfurt, Germany; and ${}^{f}$Institute of Biochemistry, Charité–Universitätsmedizin Berlin, 10117 Berlin, Germany

Edited by Barry Honig, Howard Hughes Medical Institute, Columbia University, New York, NY, and approved September 1, 2020 (received for review March 11, 2020)

Desmosomes are cell–cell junctions that link tissue cells experiencing intense mechanical stress. Although the structure of the desmosomal cadherins is known, the desmosome architecture—which is essential for mediating numerous functions—remains elusive. Here, we recorded cryo-electron tomograms (cryo-ET) in which individual cadherins can be discerned; they appear variable in shape, spacing, and tilt with respect to the membrane. The resulting subtomogram average reaches a resolution of ∼26 Å, limited by the inherent flexibility of desmosomes. To address this challenge typical of dynamic biological assemblies, we combine sub-tomogram averaging with atomistic molecular dynamics (MD) simulations. We generate models of possible cadherin arrangements and perform an in silico screening according to biophysical and structural properties extracted from MD simulation trajectories. We find a truss-like arrangement of cadherins that resembles the characteristic footprint seen in the electron micrograph. The resulting model of the desmosomal architecture explains their unique biophysical properties and strength.

cell–cell adhesion | molecular dynamics simulations | cryo-electron tomography | desmosome

M ulticellular organisms depend on cell–cell adhesion that is strong enough to maintain tissue integrity yet pliant enough to allow development and regeneration. Desmosomes are among the strongest cell–cell junctions and are also responsible for signal transduction and force transmission. Their mechanical robustness stems from the arrangement of desmosomal cadherins, namely desmocollins (Dscs) and desmogleins (Dsgs), which are members of the cadherin superfamily. The human genome encodes three Dscs (Dsc1 to Dsc3) and four Dsgs (Dsg1 to Dsg4) isoforms, which each have five extracellular (EC) domains (EC1 to EC5), a single-pass transmembrane region, and a C-terminal intracellular domain that links to the cytoskeleton via the adaptor proteins desmoplakin, plakoglobin, and plakophilin (1). Calcium ions modulate cell–cell adhesion by binding between EC domains of single cadherins and reducing cadherin flexibility (2). Desmosomal cadherins can adopt a Ca${}^{2+}$-independent hyperadhesive state. This even stronger adhesion can revert to a weaker Ca${}^{2+}$-dependent adhesion upon wounding, reflecting desmosome plasticity and sensitivity toward environmental cues (3).

Structures of desmosomal and classical cadherins have been solved by X-ray crystallography (4–6). They show that opposing cadherins interact via the N-terminal tips of their EC1 domains to form a *trans* interaction and via their EC1 and EC2 domains to form a *cis* interaction. Similar interactions have been also shown by electron microscopy and tomography, in which the cadherins displayed a rather oblique arrangement (7) and appeared to constitute a more periodic packing under close-to-native conditions (8). Despite the high sequence similarity between individual cadherins, different junctions vary in their intermembrane spacing and phenotype (9). Attempts to reconcile all cadherin architectures with theoretical models have thus far been limited in scale and stayed at a descriptive level. As a result, the precise molecular architecture of cadherins has remained elusive (10–12).

Here, we combine cryo-electron tomography (cryo-ET) with large-scale molecular dynamics (MD) simulations to produce a molecular model of the desmosome that reconciles current and previous observations, accounts for the biophysical properties, and maps the intermolecular interactions. Using such a combination of methods, we address the high plasticity of the cadherins. As many other cellular structures, cadherin assemblies exhibit high flexibility that cannot be addressed by conventional structural techniques that rely on averaging many identical structures.

### Results

We harvested desmosomes from mouse liver using a mild extraction technique, followed by plunge freezing and cryo-electron tomography (*SI Appendix*, Fig. S1). Cadherin expression in mouse liver is restricted primarily to Dsc2, Dsg2, and Dsg1 (13). To verify constituents of our preparation, we analyzed the protein composition using mass spectrometry on the same desmosomal fraction as used for imaging, which confirmed that Dsc2 and Dsg2 are the predominant desmosomal cadherin isoforms in liver desmosomes (*SI Appendix*, Fig. S2 and Table S1). Similar to the results of Whittock (14), we found that Dsg1 is

#### Significance

The desmosome is a major cell–cell junction connecting cells in tissues under high mechanical load. Currently, while structures of the constituent cadherins are known, the desmosome architecture has remained elusive. The primary reason is the high plasticity of the cadherins. As many other cellular structures, their high flexibility cannot be easily addressed by conventional structural techniques that rely on averaging many identical structures. For this, we combine high-end cryo-electron tomography with large-scale molecular dynamics simulations to produce a molecular model of the desmosome that integrates new with decades-old observations, accounts for the remarkable biophysical properties, and maps the intermolecular interactions.

Author contributions: G.H. and A.S.F. designed research; M.S., U.H.E., A.S., M.K., G.C., J.R., R.M.V., and A.S.F. performed research; and M.S. and A.S.F. wrote the paper.

The authors declare no competing interest.

This article is a PNAS Direct Submission.

This open access article is distributed under Creative Commons Attribution-NonCommercial-NoDerivatives License 4.0 (CC BY-NC-ND).

${}^{1}$M.S., U.H.E., and A.S. contributed equally to this work.

${}^{2}$To whom correspondence may be addressed. Email: gerhard.hummer@biophys.mpg.de or achilleas.frangakis@biophysik.org.

This article contains supporting information online at https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2004563117/-/DCSupplemental.
also present, but in lower abundance and with lower confidence than Dsc and Dsg2, based on intensity-based absolute quantification (iBAQ) ([SI Appendix](https://www.pnas.org/content/suppl/2020/10/19/2004563117.DCSupplemental), Fig. S2). Together with recent findings (5), this suggests that Dsc2 and Dsg2 interact predominantly to form a heterodimer.

Electron microscopy has shown that desmosomes have a lamellar structure with distinct spacing between the opposing cell membranes (~28 nm in plastic-embedded samples and ~32 nm in vitreous samples), a dense midline, and two electron-dense intracellular plaques (3, 7, 8). In our images, the desmosomes appear as 250- to 400-nm-thick discs ($n = 20$). The thickness of the desmosome and attached cytoskeleton precludes imaging from the top. The spacing between the two opposing membranes is ~32 nm ($\pm 3$ nm, $n = 20$), and the dense midline is clearly visible, similarly to what was seen before (7, 8) ([SI Appendix](https://www.pnas.org/content/suppl/2020/10/19/2004563117.DCSupplemental), Fig. S1 and [Movie S1](https://www.pnas.org/content/suppl/2020/10/19/2004563117.DCMovieS1)). Lateral spacing between individual cadherins is ~7 nm but varies, while the desmosomal plaque shows an electron-dense zone with a thickness of 23.5 nm ($\pm 1.8$ nm, $n = 20$) attached to a network of intermediate filaments. In the unprocessed tomograms, individual cadherins can be seen to interact by forming shapes resembling the letters “W,” “S,” and “Y,” similar to the shapes seen by He et al. (7) (Fig. 1A). When the sub-tomograms are aligned on a single cadherin, the variance of the location of the EC4-EC5 domains of the opposing cadherin attached to the opposing membrane was 6 nm, indicating extreme flexibility (Fig. 1B).

From 12 cryo-electron tomograms, we selected 20 desmosomes for further analysis, which yielded ~9,000 sub-tomograms, each of those containing several cadherin heterodimers. The relative populations of cadherin arrangements in the subtomograms (W shapes, 35%; S shapes, 37%; Y shapes, 27%) were very similar to that previously seen in plastic-embedded reconstructions (7). We first manually segregated the cadherin arrangements into four sub-tomogram groups based on the shapes they represented. Each of these groups was processed individually, which resulted in four cryo-ET density maps (Fig. 1C displays W, S, and Y shapes, as the group with the arbitrary shapes did not converge to a specific density) that were all severely affected by the missing wedge, leading to anisotropic resolution. Although no long-range periodicity was visible in individual tomograms, a local periodic arrangement appeared when the sub-tomograms were subjected to alignment and averaging, with a spacing at ~7 nm, similar to what was previously seen in electron tomograms of vitreous sections and elsewhere (Fig. 1C) (8, 15). In the W and S shapes, ~18 cadherins can be seen forming a square sieve-shaped dense midline. The Y shapes instead do not display a high density at the level of the dense midline as can be appreciated in the surface representation (Fig. 1C). The resolution is gradually decreasing when moving away from the center, due to the flexibility of the cadherins, resulting in a fading away of the densities (Fig. 1C). We next merged all sub-tomograms to produce a generalized average at ~26-Å resolution (Fourier shell correlation 0.5 criterion) ([SI Appendix](https://www.pnas.org/content/suppl/2020/10/19/2004563117.DCSupplemental), Fig. S3). We find that the W, S, and Y cadherin arrangements are related by a rotation around a vector roughly orthogonal to the dense midline plane. Since the shapes represent different views, the resulting cryo-ET density map is not affected by the missing wedge and depending on the view, depicts all cadherin arrangements seen in the tomogram (Fig. 2).

Fitting of the modeled Dsc2–Dsg2 heterodimer based on the available crystal structures (including glycans) ([SI Appendix](https://www.pnas.org/content/suppl/2020/10/19/2004563117.DCSupplemental), Fig. S4) shows the cadherins arranged in rows that alternate in direction, with each row rotated by 180° from adjacent rows ([SI Appendix](https://www.pnas.org/content/suppl/2020/10/19/2004563117.DCSupplemental), Fig. S4). Because the fitting of the Dsc2–Dsg2 heterodimer deviated from the cadherin densities of the cryo-ET density map, we used MD of the Dsc2–Dsg2 heterodimers to fit them into the cryo-ET density map (Materials and Methods, Fig. 2 A–C, and [Movies S2](https://www.pnas.org/content/suppl/2020/10/19/2004563117.DCMovieS2) and [S3](https://www.pnas.org/content/suppl/2020/10/19/2004563117.DCMovieS3)). The overall organization of the rigid-body fitting was retained. The distinct W, S, and Y shapes that were visible in the sub-tomogram averages can be also discerned and attributed to particular views of cadherin dimers (Fig. 2D). The EC1 and EC2 domains contribute to the square sieve pattern (Fig. 2A), which is propagated to form a pseudo two-dimensional (2D) crystal-like assembly (Fig. 2E). Each individual cadherin is forming a canonical EC1–EC1 trans interaction with an opposing molecule. It is also in contact with

![Figure 1](attachment:figure_1.png)

Fig. 1. Cryo-electron tomography of isolated mouse liver desmosomes. (A) A slice through the 3D reconstruction of an isolated desmosome superimposed with manually colored cadherin EC domains, the intermediate filaments (IF), and plasma membranes (PMs), as well as the dense midline (DM). Cadherin arrangements, as seen in the tomograms, are colored according to their shape: W = yellow, S = red, Y = blue, and alternative = gray. (B) Measurement of the variability of one cadherin monomer, when the transinteracting monomer is computationally fixed. (C) Individual sub-tomogram averages of the cadherin arrangements. A slice through each sub-tomogram depicting individual shapes, the PMs, and dense midline (DM). Corresponding isosurfaces from the side view, depicting W-, S-, and Y-shaped cadherin arrangements colored as in A. Top view shows the square sieve pattern that appears in all averages.
Fig. 2. Structure of the desmosome and MD flexible fit. (A) View from the cytoplasm (top view) on the cryo-ET density map of the desmosome (membranes are computationally removed) overlaid onto the X-ray structures of Dsg2 and Dsc2 modeled as a heterodimer. Coloring of the individual cadherins is randomized in order to facilitate the recognition of individual cadherins. (B and C) Side views of A show the overall excellent fit of the cadherins into the density maps. (D) Depending on which cadherin pair is selected, different configurations (W, S, or Y) can be reproduced. (E) Proximity map of an individual cadherin (purple blue). Each cadherin forms contacts with three other cadherins from an opposing membrane and/or its own cell (individual EC contacts are indicated). In this case, the “purple-blue” cadherin is forming a cis interaction with the “yellow” cadherin pointing to the right, a trans interaction with the “orange” cadherin on the bottom right, and another trans interaction with the “dark grey” cadherin on top. Cadherins represented by filled squares point toward the reader, cadherins represented with outlined hollow squares point away from the reader. Red-filled and blue-outlined squares indicate the positions of the neighboring cadherins (Fig. 3A).

another cadherin via an EC1–EC2 cis interface and an EC2–EC1 trans contact (Fig. 2E).

The large flexibility of the cadherins limits the resolution in sub-tomogram averaging. Consequently, the fitting of the X-ray structures into the cryo-ET density is ambiguous. To overcome this challenge, we used large-scale atomistic MD simulations as an in silico structural method to identify a chemically valid arrangement that represents the most biologically plausible desmosomal architecture. To do this, we used a low-pass filtered cryo-ET density (36-Å cutoff) that displays only the most prominent structural features, such as the cadherin spacing and the distance between the plasma membranes. Based on the cryo-ET density, we constructed atomistic-scale models of 12 possible Dsc2–Dsg2 heterodimer arrangements and supplemented them

Fig. 3. MD simulations provide a distinct model for the desmosome architecture. (A) Schematic representation of a particular cadherin arrangement. (Left) Dsc2 (red) emanating from the top plasma membrane (PM; blue plane) and Dsg2 (blue) emanating from the bottom PM (blue plane) are shown as “L-shaped” cylindrical arrows. The arrowheads in the dense midline (DM) plane represent the domains EC1 and EC2, and the arrow tails represent domains EC3 to EC5. Contacts between red and blue arrowheads represent a trans interaction, and contacts between arrowheads of the same color represent a cis interaction. (Right) Top views of the DM of three possible arrangements: parallel (P), in which the arc planes of all cadherins from one cell, which are oriented in the same direction, are rotated 90° along the cell–cell axis from the opposing cell; antiparallel (A), in which the arc planes of cadherins in every other row are rotated 180° along the cell–cell axis; and swirl (S), in which the arc planes of cadherins are rotated 90° with respect to each other, producing cyclical patterns. In the S arrangement, four cadherins occupy four neighboring grid vertices and form a left-handed S, mirrored by molecules from the opposing cell. (B) Snapshots at 100 ns from MD simulations of the P and A models. Molecules depicted in gray represent periodic images of the simulation box.

A

| P | S | P | P | S | A | P | cadherin arrangement |
|---|---|---|---|---|---|---|-----------------------|
| yes | yes | yes | no | yes | yes | yes | homogenous population |
| 2-3 | 2-3 | 1-2 | 1-2 | 1-2 | 1-2 | 1-2 | cis between domains |
| 90 | 90 | 113 | 67 | 90 | 90 | 90 | lattice angle [deg] |

| 29 | 20 | 22 | 25 | 21 | 17 | 18 | $F_{a}$ [pN] |
|---|---|---|---|---|---|---|------|
| 130 | 322 | 43 | 49 | 30 | 41 | 70 | 29 |
| 0.50 | 0.33 | 0.25 | 0.02 | 0.05 | -0.13 | 0.22 | -0.04 |
| 0.007 | 0.006 | 0.049 | 0.038 | 0.038 | 0.003 | 0.027 | 0.051 |
| 25.7 | 27.7 | 15.9 | 19.4 | 16.3 | 22.9 | 25.0 | 15.8 |
| 8 | 7 | 6 | 5 | 4 | 3 | 2 | 1 |

| normalized score |
|------------------|

C

| Antiparallel |
|--------------|
| Parallel      |

B

D

E

membrane spacing

| 20 nm | 24 nm | 30 nm | 34 nm | 38 nm |

Fig. 4. (A) Scoring of the 8 most representative models (of 12 tested) ([SI Appendix, Figs. S5 and S7](https://www.pnas.org/content/early/2020/10/19/2004563117#sec-app1) show all models) in green to red coloring. P, A, and S models are compared based on the population (homogenous population refers to the same cadherin type emanating from one membrane, whereas a heterogeneous population would correspond to a mixed population emanating from one side). The *cis* interaction positions are indicated, as is the lattice angle sketched in Fig. 3A for the sieve pattern. The last row shows the rank of each model after cumulative scoring. (B) Average elastic energy of two models of the desmosome as a function of the intermembrane spacing (thick lines). Thin lines are the result of the bootstrapping procedure ([SI Appendix](https://www.pnas.org/content/early/2020/10/19/2004563117#sec-app1)). Note that the A model reaches its energy minimum at ~32 nm and the P model at ~24 nm. (C) Total force acting on the membranes surrounding the desmosome models as function of the intermembrane spacing. (D) Stiffness Yel of P and A models at equilibrium intermembrane spacing. The dimensionless Yel quantifies the resistance to stretching and shearing and was derived from elastic network model calculations ([SI Appendix](https://www.pnas.org/content/early/2020/10/19/2004563117#sec-app1)). (E) Snapshots taken at 100 ns in a squeezing simulation with the P and A models at distinct intermembrane spacing.

with water and ions in the concentration used in the experiment, yielding between 1 and 2 million atoms in each model. Importantly, in our calculations we did consider heterogeneous and homogeneous populations of heterodimers emanating from each cell. In the homogeneous case, one cell presents Dsg2, and the other presents Dsc2; in the heterogeneous case, they are mixed ([SI Appendix, Fig. S5](https://www.pnas.org/content/early/2020/10/19/2004563117#sec-app1)). Further, we partially constrained the movement of each cadherin in two ways: 1) by keeping the EC5 domain of every cadherin in the membrane-proximal position and 2) by constraining the EC1–EC2 or EC2–EC3 domains to maintain model-specific *cis* interactions [as previously suggested by Harrison et al. (5)]. These constraints led to three major arrangements: 1) parallel, in which all cadherins from one membrane exhibit the same orientation; 2) antiparallel, where cadherins form rows that are rotated 180° to each other; and 3) swirl, in which every cadherin is rotated 90° to its neighbor (Fig. 3A and [SI Appendix](https://www.pnas.org/content/early/2020/10/19/2004563117#sec-app1)). Finally, we allowed deviations from the square sieve pattern toward a more rhomboid shape with different lattice angles (Fig. 3A; [SI Appendix, Fig. S5](https://www.pnas.org/content/early/2020/10/19/2004563117#sec-app1) shows a detailed description of each model, and Movie S4 shows an explanation of the model buildup).

For each model, we performed explicit solvent atomistic simulations lasting at least 100 ns and measured three physical properties in the obtained trajectories: 1) the forces exerted by the molecules on the membranes ($F_m$); 2) the forces measured at the *cis* interfaces ($F_c$); and 3) the overall stiffness of the desmosome ($Y$), as measured by the Young’s modulus (*Materials and Methods*). In addition, we quantified the robustness of the various interfaces by measuring the drift of the EC domains after 20 ns of unrestrained simulation ($d_c$). Finally, we assessed the biological plausibility of each model by turning to several disease-related mutations found on the surfaces of both Dsc2 and Dsg2 (5), as is it likely that they perturb interprotein interactions and should therefore be located in proximity to neighboring cadherin

Fig. 5. Desmosomal molecular architecture. (A) A minimal set of cadherins able to produce an antiparallel arrangement, with transparent molecules denoting the neighboring molecules. The plasma membranes are indicated as lipid bilayer colored in red and gray. In the dense midline plane, each node of such a lattice is composed of two types of interspersed EC1–EC2 interfaces. (B) The first type is a $cis$-$trans$-$cis$ interaction, as previously suggested by the X-ray structure of classical cadherins. (C) The second type is a $trans$-$trans$-$trans$ interaction. Both interactions are necessary to sustain the antiparallel arrangement of the cadherins.

molecules ($V_m$) (*Materials and Methods* and *SI Appendix*, Fig. S6 have values calculated for all models). We extended the simulations of the most plausible models up to 1.4 $\mu$s of simulation time, with an accumulated length of all simulations exceeding 12 $\mu$s. An example of model creation and sample trajectory is shown in Movie S3. Representative snapshots are also shown in Fig. 3B and *SI Appendix*, Fig. S5.

We next explored the biophysical properties of each model and used them to score simulated cadherin arrangements in the context of physiological properties of the desmosome. Favorable scores were assigned to the models with low $F_m$, $F_c$, and $d_c$ values and with relatively high stiffness ($Y$). Models were then ranked based on their performance in all criteria (Fig. 4A and *SI Appendix*, Fig. S7). From the calculations, we derive the following findings. 1) The interchange of Dsg2 and Dsc2 molecules protruding from the same membrane as well as moderate change in the lattice angle does not influence the biophysical properties of the overall model, suggesting that the desmosome architecture crucially depends on the overall arrangement of cadherins but not on the details of a single cadherin molecule. 2) An EC2–EC3 $cis$ interaction exerts extensive strain on the EC5–membrane connection ($F_m$) and thus, does not permit the membrane distance to be maintained at ~32 nm. In addition, the EC2–EC3 interaction is too unstable to be biologically possible. 3) Lattice angles that are too narrow or too wide cause more strain on EC2 and EC3 domains, which in turn, propagates along the EC4 and EC5 domains and ultimately, translates to increased $F_m$, rendering such an arrangement not possible.

Architecturally, the swirl models do not provide a good fit because the spacing of the square sieve pattern in the midline is distorted. The distance between EC2 domains in cadherins from adjacent rows is either 4.5 or 9.5 nm, which structurally does not match the cryo-ET density maps. Models with a parallel arrangement of cadherins have consistently low stiffness. Antiparallel models show good properties on all physiologically relevant measurements. Nevertheless, to further compare the relaxed intermembrane distance in the antiparallel and parallel models, we conducted simulated expansion and compression experiments by changing the membrane distance from 20 to 38 nm (*Materials and Methods* has details). Membranes were simulated as elastic walls, using a force constant [100 kJ/(mol·nm²)] that mimicked the elastic response of a real plasma membrane (Fig. 4B) (16, 17). Positive forces indicate squeezing of the model, while negative forces indicate stretching. Consistent with its higher level of stiffness, the model with an antiparallel arrangement of cadherins shows positive forces for

intermembrane distances below 32 nm. In contrast, a parallel arrangement produces much more pliable structures that resist compression only when the intermembrane distance drops below 24 nm, as is reflected in the potential energy curves (Fig. 4B) and net forces (Fig. 4C). Consolidating these findings, we applied an elastic network approximation to measure the stiffness of both models along the directions orthogonal and parallel to the plasma membranes at their equilibrium spacing (Fig. 4D). We find the antiparallel model to be five times stiffer in the direction orthogonal to the membranes and more than two times stiffer toward the other two orthogonal directions. The higher resistance of the antiparallel model stems from the mutual support of cadherins, which under pressure, touch their nearest neighbors symmetrically in all directions, forming a truss-like structure akin to that found in bridges (Fig. 4E). In contrast, the cadherin molecules in the parallel models all lean in the same direction and find support in a neighbor only at relatively large compression values. Altogether, we find that the antiparallel model best captures the structural and biophysical properties of the desmosome, resulting in a distinct structural arrangement (Fig. 5A, *SI Appendix*, Fig. S8, and Movie S5). Importantly, mixing the cadherins with Dsg2 and Dsc2 present on both sides of the desmosome, does not significantly influence the calculated biophysical properties. The cadherin molecules emanating from one membrane form a row, with each new row rotated 180° from the previous row. The cadherins molecules emanating from the opposing membrane are also arranged in rows rotated 180° to each other, and in addition, they are rotated 90° to the cadherins molecules from the opposing membrane. The cadherins are predicted to build two interfaces at the central midline. The first interface is exactly the same interface seen for C-cadherins and desmosomal cadherins by X-ray crystallography (4, 5) (Fig. 5B). The cadherins interact via a $trans$ interaction between the EC1 domains of Dsc2 and Dsg2. The $trans$ interaction is flanked by two $cis$ interactions involving the EC2 domains (6) (Fig. 5B). In the second interface, the cadherins form again a $trans$ interaction between the EC1 domains, which is flanked by the EC2 domains of neighboring cadherins, creating two additional $trans$ interactions. These $trans$ interactions use the same interface as a classical $cis$ interaction, however, with the cadherin pointing toward the opposing membrane and not the same membrane as a $cis$ interaction would require. This creates a heterotetrameric interaction between the EC1 and EC2 domains of both, which has not previously been observed. Each of these interfaces propagates in an alternating manner in the 2D array.

Discussion

Here, we combine cryo-ET and MD simulations to create an atomistic model of the desmosome architecture. The model resembles the sub-tomogram average to show that desmosomal cadherins emanate from the membranes of opposing cells and interact at the dense midline, creating a sieve-like pattern. While the model and the cryo-ET density match well at the midline, the EC3–EC5 regions of cadherins display large flexibility in the MD simulations, potentially because the intracellular anchoring proteins were not included in the MD model.

Given the limited resolution that can be achieved due to the flexibility of the cadherins, our in silico search screens through a wide cross-section of plausible desmosome architectures. Our model resembles previously observed densities, such as those seen in vitreous cryosectioning (8) ([SI Appendix, Fig. S9](https://www.pnas.org/cgi/doi/10.1073/pnas.2004563117)) and in plastic-embedded samples (7), and explains how the different views previously reported can be reconciled in one single model.

In addition, our model locates disease-associated residues in the cadherins at positions in the assembly where mutations could have an effect on the structural integrity of the desmosome ([SI Appendix, Fig. S10](https://www.pnas.org/cgi/doi/10.1073/pnas.2004563117)). Lastly, glycosylation plays an important role in the cadherin-based adhesion assembly (18, 19). The glycans, however, are not visible in the cryo-electron tomograms, possibly due to their high mobility. Adding the typical biantennary glycans to known glycosylation sites produces an extremely dense packing ([SI Appendix, Fig. S4B](https://www.pnas.org/cgi/doi/10.1073/pnas.2004563117)), which contributes further to desmosome stiffness and protects the surface of cadherins that are not involved in *cis* or *trans* interactions from creating spurious contacts.

Our model integrates cryo-ET data with rich biophysical and genetic data with molecular simulations. The methodology we demonstrate here is general and should prove useful also for other large and flexible cellular assemblies, for which only partial structural information exists and which cannot be subjected to averaging procedures.

Materials and Methods

Isolation of Desmosomes. To isolate desmosomes from mouse liver, a method based on the protocol of Tsukita and Tsukita (20) was used ([SI Appendix, Fig. S1](https://www.pnas.org/cgi/doi/10.1073/pnas.2004563117)). A 4-to 6-mo-old wild-type BL6 mouse was used for each isolation experiment. The liver was excised and immediately soaked in ice-cold physiological saline solution. All subsequent steps were performed at 4 °C. The liver was minced with a razor blade and incubated in hypotonic solution (1 mM NaHCO₃, protease inhibitor [complete Tablets EDTA-free EASYpack; Roche] according to the manufacturer's recommendation) for 30 min with regular shaking. Swollen samples were homogenized in 3 vol of hypotonic solution using a loose-fitting homogenizer (Wheaton). The homogenate was diluted in 40 mL hypotonic solution, filtered twice through two layers of gauze (Hartmann), and centrifuged at 1,500 × *g* for 10 min. The pellet was resuspended in 2.62 mL hypotonic solution and mixed with 19.38 mL 55% (wt/vol) sucrose in a petri dish by adding it dropwise and incubating it for another 10 min while magnetic stirring. The solution was transferred into an ultracentrifuge tube (Beckman Coulter) for Ti70 rotors, carefully layered with 6 mL 42.9% (wt/vol) sucrose, and centrifuged at 100,000 × *g* for 1 h. The bile canaliculi and desmosome fraction were recovered at the 42.9:48.5% interface, diluted in 2 vol of hypotonic solution, and centrifuged at 4,000 × *g* for 30 min. The pellet was resuspended in Nonidet P-40 solution [100 mM KCl, 1 mM MgCl₂, 1% (vol/vol) Nonidet P-40, 10 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid, pH 7.5] to selectively solubilize the membrane components not stabilized by strong hydrophilic interactions and then sonicated using a Branson Sonifier 450 with a 3-mm Double Stepped Microtip (2 × 15 pulses at setting 5.5) to spread the sample prior to plunge freezing. The isolated desmosomal fraction was then prepared either for cryo-electron tomography or for mass spectrometry.

Permutations of the Isolation Technique. The addition of several chemicals to the extraction solutions (hypotonic and Nonidet P-40 solution) was tested. First, Ca²⁺ (5 mM) was added, and the ultrastructure was analyzed (15). Further, cross-linkers such as glutaraldehyde (0.5% in 100 mM cacodylate buffer) and bis(sulfosuccinimidyl) suberate (2.5 mM) were added to the extraction solutions in order to fix cadherin molecules prior to homogenization. To investigate whether cadherin arrangement is dependent on tissue type, desmosomes were also extracted from mouse heart using the same extraction protocol. In all permutations used, the cadherin organization was very similar to that of the liver desmosomes.

Mass Spectrometry. To investigate the protein composition associated with the observed cadherin organization, we used mass spectrometry on the same final desmosomal fraction. Sample preparation for mass spectrometry was performed using the filter-aided sample preparation method (21). Briefly, 50 μg of desmosomal fraction was reduced with 0.1 M dithiothreitol for 5 min at 95 °C, diluted four times with urea buffer [UB; 8 M urea, 50 mM Tris (hydroxymethyl)aminomethane/HCl, pH 8.5], and loaded onto spin filters with 30-kDa cutoff (Microcon; Merck). After 45-min alkylation at 23 °C with 50 mM iodoacetamide in UB, proteins were digested overnight at 37 °C with 1 μg trypsin (sequencing grade; Promega) in 50 mM ammonium bicarbonate buffer. Peptides were eluted from the filters with ammonium bicarbonate buffer and acidified with 0.1% trifluoroacetic acid final concentration. Eluted peptides were desalted and fractionated (three fractions) on combined C18/strong cation exchange StageTips (Thermo Fisher Scientific). Peptides were dried in a SpeedVac and resolved in 10 μL of 1% acetonitrile and 0.1% formic acid.

Liquid chromatography tandem mass spectrometry (MS) was performed on a Q Exactive Plus equipped with an ultraHPLCy unit (Easy-nLC1000) and a Nanospray Flex Ion-Source (all three from Thermo Fisher Scientific). Peptides were separated on an in-house packed column (75-μm inner diameter, 12.5-cm length, 2.4-μm Reprosil-Pur C18 resin) using a gradient from mobile phase A (4% acetonitrile, 0.1% formic acid) to 30% mobile phase B (80% acetonitrile, 0.1% formic acid) for 60 min followed by a second step to 60% B for 30 min, with a flow rate of 300 nL/min. MS data were recorded in data-dependent mode selecting the 10 most abundant precursor ions for higher-energy collisional dissociation with a normalized collision energy of 27. The full MS scan range was set from 350 to 2,000 mass-to-charge ratio (*m/z*) with a resolution of 70,000. Ions with a charge of two or greater were selected for tandem MS scan with a resolution of 17,500 and an isolation window of 2*m/z*. The maximum ion injection time for the survey scan and the tandem MS scans was 80 s, and the ion target values were set to 3e6 and 1e5, respectively. Dynamic exclusion of selected ions was set to 60 s. Data were acquired using Xcalibur software (Thermo Fisher Scientific).

MS raw files were analyzed with Max Quant (version 1.5.3.30) (22) using default parameters. Enzyme specificity was set to trypsin, and a minimal peptide length of seven amino acids was required. Carbamidomethylcysteine was set as a fixed modification, while N-terminal acetylation and methionine oxidation were set as variable modifications. The spectra were searched against the UniProtKBmouse FASTA database (50,189 entries; downloaded in January 2016) for protein identification with a false discovery rate of 1%. Identified peptides that were shared between two or more proteins were combined and reported in protein group. Hits in any one of three categories (false positives, only identified by site, and known contaminants) were excluded from further analysis. For label-free quantification, the minimum ratio count was set to one. Absolute protein abundances were estimated by iBAQ calculation (23).

Preparation for Cryo-ET. Lacey carbon grids (Plano) were glow discharged for 20 s at low air pressure using a homemade device. The sonicated sample was mixed 20:1 with fiducial markers (Protein A conjugated to 5 nm colloidal gold; Cell Biology Department, University Medical Centre Utrecht), and 3 μL of the mixture was added to each grid. The grids were immediately plunge frozen in liquid ethane by a Vitrobot Mark IV (FEI) with –1 blotting force, 1-s blotting time, and 100% humidity at 4 °C after blotting with prewetted Plano filter paper. Cryogrids were mounted into autoloader grids with FEI C-clip rings in an EM FC6 cryomicrotome (Leica) that was cooled with liquid nitrogen under gaseous flow to –160 °C.

Cryo-ET. Tilt series were recorded using automated SerialEM data acquisition ([bio3d.colorado.edu/SerialEM/](http://bio3d.colorado.edu/SerialEM/)) on a 300-kV FEI Titan Krios in energy-filtered transmission electron microscopy mode with a K2 direct detector (Gatan) and a Gatan GIF Quantum SE postcolumn energy filter at 64,000× magnification and a pixel size of 1.1 Å at sample level. The camera was operated in superresolution mode (8 k × 8 k images) and dose fractionation mode with four frames and a total exposure time of 2 s. The tilt range was between –60° and 60° with 3° increment, a commutative electron dose accumulated at 80 e⁻/Å², and the defocus set at –4 μm (range from –3 to –10).

Image Processing. Tomograms were aligned with fiducial markers using methods as described in Díez et al. (24), reconstructed by supersampling simultaneous algebraic reconstruction technique at a pixel size of 4.4 Å (25),
and three-dimensional (3D) Contrast transfer function corrected (26, 27). 3D reconstructions were visualized with the EM package in Amira (FEI & Zuse Institute) (28) and analyzed using the TOM package (29).

The flexibility of the cadherin EC1 domains was measured using Amira and statistical analysis (mean, SD) in Microsoft Excel, with n displaying measurements in different tomograms.

For a measurement of the flexibility cadherin arrangement, sub-tomogram averaging was performed. Cadherin heterodimers spanning the intermembrane space were selected using three points for each assembly: two at the EC4–EC5 domains and one in the center.

### Sub-Tomogram Averaging and Classification.
For a more precise view of the cadherin arrangement, sub-tomogram averaging was performed. Cadherin heterodimers spanning the intermembrane space were selected using three points for each assembly: two at the EC4–EC5 domains and one in the center. Selected heterodimers were first visually grouped based on shape into W shapes, S shapes, and Y shapes. Initial orientations were assigned such that the vector from a position at the EC4–EC5 domain to the center is aligned (Fig. 1). The manually classified groups were initially aligned and averaged separately using a cylindrical mask with the graphics processing unit-accelerated sub-tomogram averaging routine of Artiatomi ([https://github.com/uermel/Artiatomi](https://github.com/uermel/Artiatomi)), allowing free rotation on a cone around the z axis (*SI Appendix*, Fig. S3B).

To validate the grouping and identify possible oligomer shapes, each of the aligned groups was imported to RELION-2.0 (30) using custom MATLAB scripts ([https://github.com/uermel/Artiatomi](https://github.com/uermel/Artiatomi)). For each subset, standard RELION-2.0 3D classification without translational or rotational search was performed using the averages from the initial alignment as references. The 3D classification converged back to the original manually classified shapes (*SI Appendix*, Fig. S3C) and did not reveal substantially different oligomer shapes, but it allowed removal of noisy or erroneously selected particles, leaving a total of 3,656 particles.

Under the assumption that the individual shapes may represent different views of the same cadherin architecture distorted by the missing wedge we pooled them together to generate a start reference. Based on this start reference, new initial orientations and positions were calculated for all particles. Using these orientations, particles were aligned and averaged, again allowing free rotation on a cone around the z axis and yielding the final map (*SI Appendix*, Fig. S3D).

Additionally, the flexibility of the cadherin EC1 domains was measured using Amira and statistical analysis (mean, SD) in Microsoft Excel, with n displaying measurements in different tomograms

### Fitting to the Cryo-ET Maps.
The antiparallel model after 600-ns simulation time was copied and shifted in x and y (Cartesian coordinates) forming a 4 × 4 lattice of dimers (4 million atoms, including solvent and ions; see below for the details of the simulations). Map was aligned with the model at the midline, and dummy atoms were added and fixed in the centers of the most prominent features of the columnar densities. Soft harmonic forces were then added [100 kJ/(mol·nm²)], gently pulling the EC3 and EC4 domains into these densities over the course of 20 ns and using the same simulation setup as in the production runs.

### MD Simulations Parameters.
All atom MD simulations, unless stated otherwise, were performed in the GROMACS 2018.1 software suite (31) using the Amber99SB*-ildn (32–34) force field with improved ion definitions (35) and the transferable intermolecular potential with 4 points water model with the dispersion correction (TIP4P-D) (36). The integration time step was 2 fs. Bonds were restrained using the linear constraint solver for molecular simulations (LINCS) algorithm (37). Nonbonded interactions were cut off at 1 nm. Electrostatic interactions were calculated using the particle mesh Ewald algorithm (38) and the potential shift Verlet modifier. All systems were energy minimized with the steepest descent algorithm in vacuo and solvated with water and ions (for ion concentrations, see below). During equilibration, the temperature was increased gradually from 100 to 300 K over 32 ps, first in a canonical and subsequently an isothermal-isobaric ensemble. Production runs were performed at 300 K (39) and ambient pressure (1 bar) (40). Unless stated otherwise, coupling constants for the thermostat and barostat were 0.1 and 4 ps, respectively, and semiisotropic pressure coupling was used.

### Preparation of a Dsc/Dsg Heterodimer.
The full-length structure of Dsg2 was obtained from the Protein Data Bank (PDB; ID code 5ERD) (5). The full-length structure of Dsc2 is not available to date. Instead we combined the structure of a chimeric protein (PDB ID code 5J5J) as well as a truncated mutant (PDB ID code 5ERP). In all structures, missing atoms were added, and steric clashes were removed using SwissPDBViewer (41). The *trans* interface of the heterodimer is not known but is assumed to follow the same tryptophan exchange mechanism between EC1 domains as in other classical cadherins (5). We modeled the Dsc2–Dsg2 heterodimer based on a Dsg2 homodimer template. The EC1 domain of Dsc2 was aligned with the corresponding domain in one of the chains of the Dsg2 dimer using PyMOL (Version 2.0; Schrödinger, LLC). Next, the template chain was removed, and the strand-swap dimer was created by inserting the tryptophan residues into the corresponding hydrophobic pockets using the PyMOL sculpting tool. The EC1 heterodimer was subjected to a 200-ns simulation (with isotropic pressure coupling). A representative snapshot of the EC1 heterodimer structure was taken as a template for the construction of a full heterodimer. To this end, the full-length structure of Dsg2 was aligned with the template, followed by the removal of the EC1 domain of Dsg2. To model Dsc2, the 5J5J structure was aligned with the template, followed by the removal of the EC1 domain of 5J5J. Next, the structures of the EC2 to EC5 domains of Dsc2 (5ERP) were aligned onto the remaining part of 5J5J, which was subsequently removed. Finally, the aligned EC2 to EC5 domains of Dsc2 and Dsg2 were merged with corresponding EC1 domains using the PyMOL sculpting tool. Care was taken to retain crystal-like calcium binding sites and calcium ion positions at all of the interdomain hinges. To prevent artificially charged C termini in the cadherins, amide capping was applied to the last amino acid. The full-length heterodimer was again energy minimized and gently stretched using Gromacs pull code along its long axis to facilitate the generation of starting configurations without steric clashes. In this process, extra care was taken not to unfold any secondary structures (average per-domain Cα rmsd to crystal structures was 1.63 Å). The final structure (together with all calcium ions bound in the hinge regions) was used as a starting structure for further simulation.

### Generation of the Lattices.
Desmosomal cadherins form 2D lattices spanning the intermembrane space. To generate the initial setup for lattice simulations, the longest axis of the heterodimer model was oriented along the z axis, and copies of the dimer were propagated on a square lattice, with lattice-spanning vectors **a** and **b** and with an angle γ between them. The simulation box consisted of four to nine copies of the initial dimer, connected via periodic boundary conditions to form a quasi-infinite lattice.

### cis Interface.
For both Dsc2 and Dsg2, alignment of the EC1 and EC2 domains onto their classical cadherin counterparts was performed using the Dali server (42). All Cα atoms within 3 Å in the C-cadherin *cis* interface were assumed to be closer than 3 Å in the *cis* interface of desmosomal cadherins. In the lattice simulations, a soft harmonic restraining potential between centers of mass of predicted *cis* interfaces of neighboring cadherins was imposed. The spring constant was set to 40 kJ/(mol·nm²), and the spring length was taken as the native distance between corresponding atoms in C-cadherin (i.e., 6.7 Å). To represent an infinite lattice, *cis* interface restraints were also imposed for cadherins in contact with the periodic images.

### Membrane.
C-terminal Cα atoms of each Dsc2 and Dsg2 molecule were tethered using a harmonic potential to a corresponding number of inert gas (argon) atoms restrained in z but free to move in the xy plane. To obtain physiologically relevant spacing, these anchor atoms were placed at z = 0 and z = 31.5 nm for the lower and upper row of cadherins, respectively. The membrane-proximal residues, not resolved in the known structures (6 amino acids for Dsg2 and 12 amino acids for Dsc2), were modeled using a Gaussian chain approximation (43), yielding spring constants for the harmonic restraining potential of 4.65 and 10.24 kJ/(mol·nm²), respectively.

### Simulation Details.
Periodic boundary conditions were used in all dimensions, with triclinic simulation boxes described by vectors **a**, **b**, and **c** = 39 nm, where angles α and β were kept at 90° and the γ angle changed as described in Results. Lattices described above after solvation were supplemented with ions at the following concentrations: 100 mM KCl, 5 mM CaCl₂, and 1 mM MgCl₂. Bound calcium ions were assumed not to contribute to the bulk concentration. The number of added ions was calculated using the screening layer tally by container average potential method (SLTCAP) (44), which was extended to divalent ions and ion mixtures. To this end, the neutrality condition was formulated as in the original publication: all mono- and divalent ions were assumed to contribute to the average electrostatic potential. Since the solution of the neutrality condition is no longer analytic, a numerical solution was found using the SymPy nsolve function (45). Based on that, the number of ions of each species was calculated and added to the system.Finally, the first 20 ns of each production run were treated as equilibration and removed from the analysis.

### Calculation of the Observables.
$F_a$, $F_c$: Force values acting on harmonic restraints were reported every 100 ps and averaged over all molecules. Values were reported as mean and SD. For different systems during the equilibration procedure, small differences, $\delta Z$, occurred between the actual spacing between anchoring beads and the reference value of 31.5 nm. For comparison between systems, $F_a$ values were corrected according to

$$
F_a = F_a^0 + k_c \delta Z,
$$

where $k_c = 7.45 \mathrm{kJ/(mol\ nm^2)}$.

$d_c$: In the simulations without restraints on *cis* interfaces, the distance between centers of mass of amino acids from neighboring cadherins that contributed to a given interface was tracked over 20 ns, and means and SDs were reported.

$V_m$: For each disease-related surface residue on each cadherin, a sphere of 1-nm radius ($R_m$) around their alpha carbons was monitored for interactions. The presence of alpha carbons of amino acids belonging to neighboring cadherins and the total volume of such intruding residues, $V_i$, were calculated according to Chothia (46). The volume of such intruding residues was calculated as a ratio,

$$
V_m = V_i / (4/3 \pi R_m^3).
$$

Values were averaged over time for all mutation sites and all cadherins. Mean and SD were reported.

$Y$: The Young modulus was calculated as follows. When no large deformations occur, fluctuations of the longitudinal span $\zeta$ of each cadherin can be approximated as harmonic, with probability density given by

$$
P(\zeta) = \frac{1}{\sigma_\zeta \sqrt{(2\pi)}} e^{-\frac{1}{2} (\zeta - \zeta_0)^2 / (\sigma_\zeta)}.
$$

Here, $\zeta_0$ is the mean value of $\zeta$, $\sigma_\zeta = \sqrt{(k_B T / k_\zeta)}$ is its SD, $k_B$ is the Boltzmann constant, $T$ is temperature in Kelvin, and $k_\zeta$ is the elastic spring constant of each cadherin; therefore, $k_\zeta = k_B T / \sigma_\zeta^2$. Since all cadherins are coupled to each other via multiple interactions within the midline, it is more reasonable to treat the whole simulated section of the desmosome as a single harmonic system. To this end, $\zeta$ was taken as an average spacing between the C-terminal alpha carbons of the cadherins in the upper and lower rows. To obtain an estimate of an error, $k_\zeta$ was calculated as an average of blocks of 25 time points over the analyzed trajectory, and $\sigma_k$ was calculated as SD. To obtain a material property of the desmosome that can be quantified experimentally, a Young modulus was calculated as

$$
Y = k_\zeta \zeta_0 / (|a| \cdot |b|),
$$

and its error was estimated as

$$
\sigma_Y = \sigma_k \zeta_0 / (|a| \cdot |b|) + k_\zeta \sigma_\zeta / (|a| \cdot |b|).
$$

#### Stiffness from Elastic Network Model.
To compare the stiffness of different desmosome architectures, we constructed elastic network models for tetramers of Dsc2–Dsg2 heterodimers (with four cadherins protruding from each membrane) in the C$_\alpha$ representation (47) with a cutoff distance of 15 Å and uniform spring constant $C$. The vector $R = (R_1, \dots, R_N)^T$ of atom displacements in response to external forces $F = (F_1, \dots, F_N)^T$ can be calculated as

$$
R = H^{-1} F,
$$

where $H^{-1}$ denotes the pseudoinverse of the Hessian matrix $H$. The dimensionless stiffness can then be calculated as

$$
Y_{el} = \sqrt{\frac{\sum_{i=1}^N F_i^2}{C^2 \sum_{i=1}^N R_i^2}} = \sqrt{\frac{F^T F}{C^2 F^T (H^{-1})^2 F}}.
$$

We applied uniform forces, $|F_i| = 1$, to the C$_\alpha$ atoms of the EC5 domains. To probe desmosome stretching normal to the membranes and shearing parallel to the membranes in two principal directions, the forces $F_i$ were directed along the $z$, $y$, and $x$ axes, respectively, with opposite force directions for cadherins in the upper and lower rows.

#### Compression Simulations.
To emulate membrane-excluded volume, flat-bottomed potentials were introduced such that the centers of mass of the EC5 domains would be harmonically repelled from membranes with the restoring force:

$$
F(z) =
\begin{cases}
-k_m (z - z_0) & \text{if } z < z_0 \\
0 & \text{otherwise}
\end{cases}
$$

with $k_{mem} = 100 \mathrm{kJ/mol}$ per nanometer$^2$ and $z_0$ defined as membrane position plus half a radius of gyration of an immunoglobulin domain. The membrane position was mirrored for the molecules in the upper part of the simulation box.

Snapshots (100 ns) from production simulations of parallel and antiparallel models were used to initialize steered MD simulations in which the intermembrane distance varied between 20 and 38 nm. New systems were initialized from snapshots taken every $\Delta d = 1$ nm at membrane distances of $d_i = 20$ nm to $d_i = 34$ nm and additionally, at 38 nm. Each of these was equilibrated for 20 ns and evolved to 100 ns of simulation time. During the simulations, the forces acting on anchors ($F_a$) and on membranes ($F_m$) were recorded every 100 ps. The total force acting on every Dsc2/Dsg2 heterodimer was then calculated for each system $d_i$ as

$$
F(d_i) = 0.5 \times \left( F_a^{top}(d_i) + F_m^{top}(d_i) - F_a^{bottom}(d_i) - F_m^{bottom}(d_i) \right),
$$

where $top$ and $bottom$ denote upper and lower membrane, respectively, and averages are taken over time and over single cadherins.

The potential of mean force corresponds to the reversible work done by membranes to squeeze the desmosome, which was calculated as

$$
W(d_i + \Delta d) = -\sum_{d=0}^{d=d_i+\Delta d} F(d) \Delta d.
$$

Since $W$ is defined within an additive constant, the resulting curves were shifted to have a minimum at $W = 0$ kJ/mol [i.e., where $F(d) = 0$]. To account for possible sampling errors, a bootstrapping procedure was employed, with work recalculated 200 times from a randomly selected subset of 6% measurements.

---

### Code Availability.
The reconstruction, sub-tomogram averaging, and classification code referenced here are available in Github (Artiatomi: [https://github.com/uermel/Artiatomi](https://github.com/uermel/Artiatomi)).

### Data Availability.
The 26-Å density map of the cryo-ET density was deposited in the Electron Microscopy Data Bank (accession no. **EMD-11678**), and the fitted model is in the PDB (ID code **7A7D**). Other raw data are available from the corresponding authors upon reasonable request.

### ACKNOWLEDGMENTS.
We thank Max Linke and Lukas Stelzl for initial discussions. M.S. and G.H. thank the Max Planck Society for support and the Max Planck Computing and Data Facility for computational resources. M.S. was supported by Schroedinger Fellowship 43232-B28 from the Fonds zur Förderung der wissenschaftlichen Forschung (FWF). This work was supported by the LOEWE Schwerpunkt Dynamem (G.H. and A.S.F.). The F30 direct electron detector was funded by Deutsche Forschungsgemeinschaft (DFG) INST Grant 161/813-1. G.H. was supported by DFG Grant SFB902. A.S.F. was supported by the following DFG Grants: (1) FR 1653/6-1 (to A.S.), (2) FR 1653/13-1 (to U.H.E.), and (3) FR 1653/14-1 (to M.K.).

---

### References
1. R. L. Dusek, L. M. Godsell, K. J. Green, Discriminating roles of desmosomal cadherins: Beyond desmosomal adhesion. *J. Dermatol. Sci.* **45**, 7–21 (2007).
2. M. Sotomayor, K. Schulten, The allosteric role of the Ca2+ switch in adhesion and elasticity of C-cadherin. *Biophys. J.* **94**, 4621–4633 (2008).
3. D. R. Garrod, M. Y. Berika, W. F. Bardsley, D. Holmes, L. Taberner, Hyper-adhesion in desmosomes: Its regulation in wound healing and possible relationship to cadherin crystal structure. *J. Cell Sci.* **118**, 5743–5754 (2005).
4. T. J. Boggon *et al.*, C-cadherin ectodomain structure and implications for cell adhesion mechanisms. *Science* **296**, 1308–1313 (2002).
5. O. J. Harrison *et al.*, Structural basis of adhesive binding by desmocollins and desmogleins. *Proc. Natl. Acad. Sci. U.S.A.* **113**, 7160–7165 (2016).
6. L. Shapiro *et al.*, Structural basis of cell-cell adhesion by cadherins. *Nature* **374**, 327–337 (1995).
7. W. He, P. Cowin, D. L. Stokes, Untangling desmosomal knots with electron tomography. *Science* **302**, 109–113 (2003).
8. A. Al-Amoudi, D. C. Díez, M. J. Betts, A. S. Frangakis, The molecular architecture of cadherins in native epidermal desmosomes. *Nature* **450**, 832–837 (2007).
9. D. Garrod, M. Chidgey, Desmosome structure, composition and function. *Biochim. Biophys. Acta* **1778**, 572–587 (2008).
10. J. Chen, J. Newhall, Z.-R. Xie, D. Leckband, Y. Wu, A computational model for kinetic studies of cadherin binding and clustering. *Biophys. J.* **111**, 1507–1518 (2016).
11. Y. Narui, M. Sotomayor, Tuning inner-ear tip-link affinity through alternatively spliced variants of protocadherin-15. *Biochemistry* **57**, 1702–1710 (2018).
12. Y. Wu *et al.*, Cooperativity between trans and cis interactions in cadherin-mediated junction formation. *Proc. Natl. Acad. Sci. U.S.A.* **107**, 17592–17597 (2010).
13. B. Holthöfer, R. Windoffer, S. Troyanovsky, R. E. Leube, Structure and function of desmosomes. *Int. Rev. Cytol.* **264**, 65–163 (2007).
14. N. V. Whittock, Genomic sequence analysis of the mouse desmoglein cluster reveals evidence for six distinct genes: Characterization of mouse DSG4, DSG5, and DSG6. *J. Invest. Dermatol.* **120**, 970–980 (2003).

--- 

www.pnas.org/cgi/doi/10.1073/pnas.2004563117   |   Sikora et al.
15. H. Tariq *et al.*, Cadherin flexibility provides a key difference between desmosomes and adherens junctions. *Proc. Natl. Acad. Sci. U.S.A.* **112**, 5395–5400 (2015).

16. B. Daily, E. L. Elson, G. I. Zahalak, Cell poking. Determination of the elastic area compressibility modulus of the erythrocyte membrane. *Biophys. J.* **45**, 671–682 (1984).

17. S. Steltenkamp *et al.*, Mechanical properties of pore-spanning lipid bilayers probed by atomic force microscopy. *Biophys. J.* **91**, 217–226 (2006).

18. S. Carvalho, C. A. Reis, S. S. Pinho, Cadherins glycans in cancer: Sweet players in a bitter process. *Trends Cancer* **2**, 519–531 (2016).

19. M. D. Langer, H. Guo, N. Shashikanth, J. M. Pierce, D. E. Leckband, N-glycosylation alters cadherin-mediated intercellular binding kinetics. *J. Cell Sci.* **125**, 2478–2485 (2012).

20. S. Tsukita, S. Tsukita, Isolation of cell-to-cell adherens junctions from rat liver. *J. Cell Biol.* **108**, 31–41 (1989).

21. J. R. Wiśniewski, A. Zougman, N. Nagaraj, M. Mann, Universal sample preparation method for proteome analysis. *Nat. Methods* **6**, 359–362 (2009).

22. J. Cox, M. Mann, MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. *Nat. Biotechnol.* **26**, 1367–1372 (2008).

23. B. Schwanhäusser *et al.*, Global quantification of mammalian gene expression control. *Nature* **473**, 337–342 (2011).

24. D. C. Diez, A. Seybert, A. S. Frangakis, Tilt-series and electron microscope alignment for the correction of the non-perpendicularity of beam and tilt-axis. *J. Struct. Biol.* **154**, 195–205 (2006).

25. M. Kunz, A. S. Frangakis, Super-sampling SART with ordered subsets. *J. Struct. Biol.* **188**, 107–115 (2014).

26. M. Kunz, Z. Yu, A. S. Frangakis, M-free: Mask-independent scoring of the reference bias. *J. Struct. Biol.* **192**, 307–311 (2015).

27. M. Kunz, A. S. Frangakis, Three-dimensional CTF correction improves the resolution of electron tomograms. *J. Struct. Biol.* **197**, 114–122 (2017).

28. S. Pruggnaller, M. Mayr, A. S. Frangakis, A visualization and segmentation toolbox for electron microscopy. *J. Struct. Biol.* **164**, 161–165 (2008).

29. A. Korinek, F. Beck, W. Baumeister, S. Nickell, J. M. Plitzko, Computer controlled cryo-electron microscopy-TOM² a software package for high-throughput applications. *J. Struct. Biol.* **175**, 394–405 (2011).

30. S. H. W. Scheres, RELION: Implementation of a Bayesian approach to cryo-EM structure determination. *J. Struct. Biol.* **180**, 519–530 (2012).

31. M. J. Abraham *et al.*, GROMACS: High performance molecular simulations through multi-level parallelism from laptops to supercomputers. *SoftwareX* **1–2**, 19–25 (2015).

32. R. B. Best, G. Hummer, Optimized molecular dynamics force fields applied to the helix-coil transition of polypeptides. *J. Phys. Chem. B* **113**, 9004–9015 (2009).

33. V. Hornak *et al.*, Comparison of multiple Amber force fields and development of improved protein backbone parameters. *Proteins* **65**, 712–725 (2006).

34. K. Lindorff-Larsen *et al.*, Improved side-chain torsion potentials for the Amber ff99SB protein force field. *Proteins* **78**, 1950–1958 (2010).

35. S. Mamatkulov, N. Schwierz, Force fields for monovalent and divalent metal cations in TIP3P water based on thermodynamic and kinetic properties. *J. Chem. Phys.* **148**, 074504 (2018).

36. S. Piana, A. G. Donchev, P. Robustelli, D. E. Shaw, Water dispersion interactions strongly influence simulated structural properties of disordered protein states. *J. Phys. Chem. B* **119**, 5113–5123 (2015).

37. B. Hess, P-LINCS: A parallel linear constraint solver for molecular simulation. *J. Chem. Theory Comput.* **4**, 116–122 (2008).

38. T. Darden, D. York, L. Pedersen, Particle mesh Ewald: An N·log(N) method for Ewald sums in large systems. *J. Chem. Phys.* **98**, 10089–10092 (1993).

39. G. Bussi, D. Donadio, M. Parrinello, Canonical sampling through velocity rescaling. *J. Chem. Phys.* **126**, 014101 (2007).

40. M. Parrinello, A. Rahman, Polymorphic transitions in single crystals: A new molecular dynamics method. *J. Appl. Phys.* **52**, 7182–7190 (1981).

41. M. U. Johansson, V. Zoete, O. Michielin, N. Guex, Defining and searching for structural motifs using DeepView/Swiss-PdbViewer. *BMC Bioinformatics* **13**, 173 (2012).

42. L. Holm, L. M. Laakso, Dali server update. *Nucleic Acids Res.* **44**, W351–W355 (2016).

43. C. Hyeon, G. Morrison, D. Thirumalai, Force-dependent hopping rates of RNA hairpins can be estimated from accurate measurement of the folding landscapes. *Proc. Natl. Acad. Sci. U.S.A.* **105**, 9604–9609 (2008).

44. J. D. Schmit, N. L. Kariyawasam, V. Needham, P. E. Smith, SLTCAP: A simple method for calculating the number of ions needed for MD simulation. *J. Chem. Theory Comput.* **14**, 1823–1827 (2018).

45. A. Meurer *et al.*, SymPy: Symbolic computing in Python. *PeerJ Comput. Sci.* **3**, e103 (2017).

46. C. Chothia, Principles that determine the structure of proteins. *Annu. Rev. Biochem.* **53**, 537–572 (1984).

47. M. M. Tirion, Large amplitude elastic motions in proteins from a single-parameter, atomic analysis. *Phys. Rev. Lett.* **77**, 1905–1908 (1996).
