
Journal Pre-proof

Mechanisms of SARS-CoV-2 transmission and pathogenesis

Andrew G. Harrison, Tao Lin, Penghua Wang

PII: S1471-4906(20)30233-7  
DOI: https://doi.org/10.1016/j.it.2020.10.004  
Reference: TREIMM 1738  

To appear in: Trends in Immunology

Please cite this article as: A.G. Harrison, T. Lin and P. Wang, Mechanisms of SARS-CoV-2 transmission and pathogenesis, Trends in Immunology (2020), https://doi.org/10.1016/j.it.2020.10.004

This is a PDF file of an article that has undergone enhancements after acceptance, such as the addition of a cover page and metadata, and formatting for readability, but it is not yet the definitive version of record. This version will undergo additional copyediting, typesetting and review before it is published in its final form, but we are providing this version to give early visibility of the article. Please note that, during the production process, errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.

© 2020 Published by Elsevier.

Journal Pre-proof

Mechanisms of SARS-CoV-2 transmission and pathogenesis

Andrew G. Harrison${}^{1*}$, Tao Lin${}^{1*}$, Penghua Wang${}^{1\#}$

${}^{1}$ Department of Immunology, School of Medicine, University of Connecticut Health Center, Farmington, CT 06030, USA

* A.G.H. and T.L. contributed equally

# Address correspondence to: Penghua Wang, Ph.D., Department of Immunology, School of Medicine, University of Connecticut Health Center, Farmington, CT 06030, USA. Email: pewang@uchc.edu, Tel: 860-679-6393

Abstract

The emergence of SARS-coronavirus 2 (SARS-CoV-2) marks the third highly pathogenic coronavirus to spill over into the human population. SARS-CoV-2 is highly transmissible with a broad tissue tropism that is likely perpetuating the pandemic. However, important questions remain regarding its transmissibility and pathogenesis. In this review, we summarize current SARS-CoV-2 research, with an emphasis on transmission, tissue tropism, viral pathogenesis, and immune antagonism. We further present advances in animal models that are important for understanding the pathogenesis of SARS-CoV-2, vaccine development, and therapeutic testing. When necessary, comparisons are made from studies with SARS to provide further perspectives on COVID-19, as well as draw inferences for future investigations.

Key words: severe acute respiratory syndrome, coronavirus, COVID-19, SARS, SARS-CoV-2

The Emergence of a Third, Novel Coronavirus  
Current State of the COVID-19 Pandemic  

Coronaviruses (CoVs) have caused three large-scale outbreaks in the last two decades: severe acute respiratory syndrome (SARS), Middle Eastern respiratory syndrome (MERS) and now Coronavirus disease 2019 (COVID-19). The origin of the COVID-19 pandemic was traced back to a cluster of pneumonia cases connected to a wet seafood market in Wuhan City, Hubei Province, China [1]. Following the likely spillover of a zoonotic (see Glossary) virus, further work confirmed the etiological agent to be a novel Betacoronavirus related to SARS-CoV [1, 2].

The first case developed symptoms on December 1, 2019 after which rapid human-to-human transmission and intercontinental spread later ensued, being declared a pandemic by the World Health Organization (WHO) in March, 2020 [3]. Since then, ~35-million people have been infected by SARS-CoV-2 with over 1 million deaths in 235 countries, areas or territories [4]. Although SARS-CoV-2 appears to be less lethal than SARS-CoV or MERS-CoV, its transmissibility is much higher. To find solutions that contain this raging pandemic, global research efforts have been quickly mobilized, each day resulting in new advances in basic and clinical research, therapy, diagnosis, vaccine and drug development, as well as epidemiology.

Here, we conduct a comprehensive review of the current state of COVID-19 research with a principal focus on the mechanisms of transmission and pathogenesis of SARS-CoV-2 stemming from clinical and animal studies.

## SARS-CoV-2 Characteristics  
### SARS-CoV-2 Genome & Structure

CoVs of the family Coronaviridae are enveloped, positive sense single-stranded RNA viruses [5]. All of the highly pathogenic CoVs, including SARS-CoV-2, belong to the Betacoronavirus genus, group 2 [5]. The SARS-CoV-2 genome sequence shares ~80% sequence identity with SARS-CoV and ~50% with MERS-CoV [1, 6]. Its genome consists of 14 open reading frames (ORFs), two-thirds of which encode 16 nonstructural proteins (nsp 1-16) that make up the replicase complex [6, 7]. The remaining one-third encodes nine accessory proteins (ORF) and four structural proteins: spike (S), envelope (E), membrane (M) and nucleocapsid (N), among which Spike mediates SARS-CoV entry into host cells [8]. However, the S gene of SARS-CoV-2 is highly variable from SARS-CoV, sharing less than 75% nucleotide identity [1, 6, 9]. Spike has a receptor-binding domain (RBD) that mediates direct contact with a cellular receptor, angiotensin-converting enzyme 2 (ACE2), and an S1/S2 polybasic cleavage site that is proteolytically cleaved by cellular cathepsin L and the transmembrane protease serine 2 (TMPRSS2) (Figure 1) [1, 9, 10]. TMPRSS2 facilitates viral entry at the plasma membrane surface, whereas cathepsin L activates SARS-CoV-2 Spike in endosomes and can compensate for entry into cells that lack TMPRSS2 (Figure 1) [10]. Once the genome is released into the host cytosol, ORF1a and ORF1b are translated into viral replicase proteins, which are cleaved into individual nsps (via host and viral proteases: PLpro); these form the RNA-dependent RNA polymerase (nsp12 derived from ORF1b) [8]. Here, the replicase components rearrange the endoplasmic reticulum (ER) into double-membrane vesicles (DMVs) that facilitate viral replication of genomic and subgenomic RNAs (sgRNA), the latter which are translated into accessory and viral structural proteins that facilitate virus particle formation (Figure 1) [11] [12].

Tissue Tropism of SARS-CoV-2
The establishment of viral tropism depends on susceptibility and permissiveness of a specific host cell. During the SARS epidemic, patients often presented with respiratory-like illnesses that progressed to severe pneumonia, observations mirroring the disease course of COVID-19, suggesting that the lung was the primary tropism of SARS-CoV-2 [13]. Both CoVs were then found to bind the same entry receptor, ACE2 [1, 14, 15]. Of note, the key mutations in the RBD of SARS-CoV-2 Spike make additional close contacts with ACE2, correlating with higher binding affinity and perhaps increased infectivity [1] [14, 16]. The presence of a unique furin cleavage site at the S1/S2 junction of SARS-CoV-2 Spike is also suspected to enhance human transmission events, although this remains to be further investigated [17] [18]. The currently predominant SARS-CoV-2 isolate worldwide carries a D614G mutation that is absent from its presumptive common ancestor, and is more infectious, likely underlying in part, an increased human-to-human transmission efficiency [19-21]. Although associated with an increased viral load in the upper respiratory tract of COVID-19 patients, the D614G variant does not correlate with disease severity, suggesting that pathogenesis of severe COVID-19 is linked to mechanisms that are more than just SARS-CoV-2 infectivity [21].

Once SARS-CoVs enter into the host via the respiratory tract, airway, and alveolar epithelial cells, vascular endothelial cells and alveolar macrophages are among their first targets of viral entry [22-24]. These cells are probably ‘ground-zero’ for early infection and subsequent replication due to their modest expression of ACE2 [25]. Although ACE2 mRNA is detected in human and many mammalian (bat, ferret, cat and dog etc.) lung biopsies, their expression is rather low, compared to extrapulmonary tissues [26]. Thus, the permissiveness of these cells to SARS-CoVs may depend on additional, unappreciated cell-intrinsic factors that aid in efficient
4
infection. First, viral entry may heavily depend on the expression of TMPRSS2, as nearly undetectable amounts of ACE2 still support SARS-CoV entry, so long as TMPRSS2 is present [27]. Second, the mRNA expression of cellular genes such as ESCRT (endosomal sorting complex required for transport) machinery gene members (including CHMP3, CHMP5, CHMP1A, and VPS37B) related to pro-SARS-CoV-2 lifecycle is higher in a small population of human type II alveolar cells with abundant ACE2, relative to ACE2-deficient cells [28]. This suggests that SARS-CoV-2 hijacks a small population of type II alveolar cells with high expression of ACE2 and other pro-viral genes for its productive replication. Third, the lung—as the main tropism of SARS-CoVs—may be contingent on the regulation of ACE2 at the transcriptional and protein levels [24, 29, 30] [25, 31]. For example, in human airway epithelial cells, ACE2 gene expression is upregulated by type I and II interferons [25, 31] during viral infection. Lastly, compared to other SARS-CoVs, SARS-CoV-2 Spike contains a unique insertion of RRAR at the S1/S2 cleavage site [17] [18]. This site can be pre-cleaved by furin, thus reducing the dependence of SARS-CoV-2 on target cell proteases (TMPRSS2/cathepsin L) for entry [17] [18] and potentially extending its cellular tropism, given that proteolytically active furin is abundantly expressed in human bronchial epithelial cells [32, 33].

One of the distinctions between SARS-CoV and SARS-CoV-2 is the latter's ability to efficiently infect the upper respiratory tract (URT), such as nasopharyngeal (NP) and/or oropharyngeal (OP) tissues, possibly due to its higher affinity for ACE2, which is expressed in human nasal and oral tissues [34] [35] [23, 25, 36]. The readily detectable titers of SARS-CoV-2 in the URT mucus of COVID-19 patients during prodromal periods might contribute to explaining the more rapid and effective transmissibility of SARS-CoV-2 relative to SARS-CoV [37].

Human CoVs often cause enteric infections, with variable degrees of pathogenicity [38].

Indeed, ACE2 and TMPRSS2 are abundantly expressed within the human and many other mammalian intestinal tracts, specifically the brush border of intestinal enterocytes [23, 25, 26, 39]. Accordingly, gastrointestinal illness has been frequently reported in COVID-19 patients [40, 41], consistent with the recovery of SARS-CoV from SARS patients' stool samples [42], suggesting a potential fecal-oral route of transmission for these two CoVs. Of note, ~20% of COVID-19 patients examined have had detectable SARS-CoV-2 RNA in feces, even after respiratory symptoms subsided, suggesting that SARS-CoV-2 titers might be prolonged in the intestinal tract [41]. Although further testing is warranted, these data suggest the possibility that fecal-oral transmission of SARS-CoV-2 might occur. Evidently, robust epidemiological studies are needed to conclusively demonstrate if COVID-19 patients recovering from respiratory illness are able to spread SARS-CoV-2.

Transmission Dynamics of SARS-CoV-2

Human CoVs are transmitted primarily through respiratory droplets, but aerosol, direct contact with contaminated surfaces, and fecal-oral transmission were also reported during the SARS epidemic [43-45]. Early reports of patients with cough, lung ground glass opacities, and symptom progression to severe pneumonia, suggested communicability of SARS-CoV-2 via the respiratory route (Figure 2) [1-3]. Direct transmission by respiratory droplets is reinforced by productive SARS-CoV-2 replication in both the URT and LRT, and the increasing number of reports indicating human-to-human spread among close contacts exhibiting active coughing (Figure 2) [35, 46-48]. So far, the basic reproduction number (R₀) is ~2.2, based on early case

tracking in the beginning of the pandemic, with a doubling time of 5 days [47] [49]. Furthermore, there is now evidence for non-symptomatic/pre-symptomatic spread of SARS-CoV-2, which is in contrast to the transmission dynamics of SARS-CoV [50]. This finding underscores the ability of SARS-CoV-2 to colonize and replicate in the throat during early infection [37, 51, 52]. Based on these apparent disparities in virus transmission, one study modeled the transmission dynamics of SARS-CoV-2 in pre-symptomatic individuals, and indicated that the pre-symptomatic R₀ has approached the threshold for sustaining an outbreak on its own (R₀ > 1); by contrast, the corresponding estimates for SARS-CoV were approximately zero [49]. Similarly, asymptomatic spread of SARS-CoV-2 has been documented throughout the course of the pandemic [48] [51, 53-56]. Understanding the relative importance of cryptic transmission to the current COVID-19 pandemic is essential for public health authorities to make the most comprehensive and effective disease control measures that include mask-wearing, contact tracing, and physical isolation.

For SARS-CoV-2, various modes of transmission have been proposed, including aerosol, surface contamination, fecal-oral route, representing confounding factors in the current COVID-19 pandemic; thus, their relative importance is still being investigated (Figure 2) [57]. Aerosol transmission (spread > 1m) was implicated in the Amoy Gardens outbreak during the SARS epidemic, but the inconsistency of these findings in other settings suggested that SARS-CoV was likely an opportunistic airborne infection [43, 58]. Similarly, no infectious SARS-CoV-2 virions have been isolated, though viral RNA was detectable in the air of COVID-19 hospital wards [59]. Generation of experimental aerosols carrying SARS-CoV-2 (comparable to those that might be generated by humans) have offered the plausibility of airborne transmission, but the aerodynamic characteristics of SARS-CoV-2 during a natural course of infection is still an area of intense inquiry [60]. Nonetheless, deposition of virus-laden aerosols might contaminate

objects (e.g. fomites) and contribute to human transmission events [59, 61]. Finally, fecal-oral transmission has also been considered as a potential route of human spread, but this route remains an enigma despite evidence of RNA-laden aerosols being found nearby toilet bowls, along with detectable SARS-CoV-2 RNA in rectal swabs during the precursor epidemic of COVID-19 in China [41, 59, 62].

### SARS-CoV-2 Pathogenesis

#### Clinical Presentation of COVID-19

In general, common cold CoVs tend to cause mild URT symptoms and occasional gastrointestinal involvement (Figure 3). On the contrary, infection with the highly pathogenic CoVs, including SARS-CoV-2, causes severe 'flu'-like symptoms that can progress to acute respiratory distress (ARDS), pneumonia, renal failure, and death [46, 48, 63, 64]. The most common symptoms are fever, cough, and dyspnea, accounting for 83%, 82% and 31% of COVID-19 patients (n=99), respectively in one epidemiological study [65]. The incubation period in COVID-19 is rapid, 5-6 days, versus 2-11 days in SARS-CoV infections [38, 47, 48].

As the pandemic is progressing, it has become increasingly clear that COVID-19 encompasses not only rapid respiratory/gastrointestinal illnesses, but can have long-term ramifications such as myocardial inflammation [66]. Furthermore, severe COVID-19 is not restricted to the aged population as initially reported; children and young adults are also at risk [67]. From a diagnostic perspective, COVID-19 presents with certain ‘hallmark’ laboratory and radiological indices, which can be helpful in assessing disease progression (Table 1). Together, COVID-19 initially

presents with ‘flu’-like symptoms and can later progress to life-threatening systemic inflammation and multi-organ dysfunction.

# Age-associated COVID-19 Severity

It is widely accepted that the aging process predisposes individuals to certain infectious diseases [68]. In the case of COVID-19, older age is associated with greater COVID-19 morbidity, admittance to the ICU, progressing to ARDS, higher fever, and greater mortality rates [69] [70, 71]. Moreover, lymphocytopenia, neutrophilia, elevated inflammation-related indices, and coagulation-related indicators have been consistently reported in older (≥65 years old) relative to young and middle-aged COVID-19 patients [Table 1; ([72, 73]) [46, 65, 71, 74, 75]]. At the cellular level, a lower capacity of CD4⁺ and CD8⁺ T-cells to produce IFN-γ and IL-2, as well as an impairment in T-cell activation from dendritic cells (DCs) in acute COVID-19 patients (≥55 years old), might potentially compromise an optimal adaptive immune response [76]. Based on examples from mice, a productive CD4⁺ T-cell response relies heavily on lung resident DCs (rDCs) and abates SARS-CoV infection [77, 78]. However, whether a reduction in the DC population in the lungs of older, more severe patients causes sub-optimal T-cell activation during SARS-CoV-2 infection remains to be robustly investigated.

Higher proportions of proinflammatory macrophages and neutrophils have also been observed in the bronchoalveolar lavage fluid (BALF) of COVID-19 patients with severe symptoms compared with those exhibiting mild symptoms (Key Figure, Figure 4) [79].

Accordingly, proinflammatory cytokines (e.g., IL-6, IL-8) are elevated in the BALF of severe

COVID-19 patients, along with higher expression of inflammatory chemokines (e.g. CCL2) in macrophages relative to non-severe COVID-19 patients [79-82]. Indeed, similar inflammatory milieux have been associated with severe lung pathology in SARS patients, along with the notable ‘cytokine storm’ that can present in critically ill COVID-19 patients [83, 84] [71, 85-87]. These proinflammatory mediators can, in turn, perpetuate lung disease by elevating C-reactive protein (CRP) from the liver (Table 1) through the signal transducer and activator of transcription 3 (STAT3)-IL-6 signaling [88]. Therefore, a rise in CRP concentrations can correlate with elevated serum IL-6 production observed in COVID-19 patients [79, 80] [88].

From another angle, formation of neutrophil extracellular traps (NETs) inside micro-vessels is highly pronounced in patients with severe relative to mild COVID-19, implicating NETs as possible potentiators of COVID-19 pathogenesis [89]. The recruitment of these activated neutrophils and monocytes may be driven by pulmonary endothelial cell dysfunction through vascular leakage, tissue edema, endotheliitis, and possibly, disseminated intravascular coagulation (DIC) pathways; indeed, a recent study demonstrated direct SARS-CoV-2 infection of vascular endothelial cells with concomitant accumulation of inflammatory mononuclear cells (e.g. neutrophils) in multiple organs (lung, heart, kidney, small bowel and liver) in patients with severe COVID-19 (Key Figure, Figure 4) [90]. In fact, many COVID-19 patients have met the DIC case definition based on elevated serum D-dimer amounts and prolonged prothrombin times [91, 92]. Together, it is reasonable to assume that direct viral insult and immune cell recruitment escalate endothelial contractility and the loosening of gap junctions, thus promoting vascular leakage and the systemic impairment of the circulatory system in this pathology.

Journal Pre-proof

SARS-CoV-2 Innate Immune Evasion Strategies: Examples from other Betacoronavirus Infections

The recognition of virus infection begins with the detection of viral nucleic acid by host cell pattern recognition receptors (PRRs) that signal downstream via recruited adaptor proteins, ubiquitin ligases, and kinases culminating in the transcription factors and ultimate expression of immune genes including IFNs, cytokines, and chemokines etc. (Figure 5). The interferon (IFN) pathway is often a primary target of evasion due to its rapidity and potency in eliminating viral infection. CoVs have evolved multiple mechanisms to target the signaling components of several PRR-IFN pathways to survive in host cells (Figure 5). CoVs are highly sensitive to IFN and therefore act at several levels in these pathways to antagonize mammalian immune recognition, interfering with downstream signaling, or inhibiting specific interferon stimulated gene (ISG) products [93]. Specifically, CoVs can avoid immune sensing via i) the formation of DMVs that sequester viral nucleic acid from being recognized by PRRs and ii) direct ablation of the functionality of immune signaling molecules by viral proteins [11, 94]. The structural and functional conservation of these proteins across the Betacoronavirus genus and in nsps between SARS-CoV and SARS-CoV-2, suggests that some of these suppressive mechanisms might be employed by SARS-CoV-2 (see below) [1]. Indeed, patients with severe COVID-19 have reported an imbalanced immune response with high concentrations of inflammatory cytokines/chemokines, but little circulating IFN-β or IFN-λ, resulting in persistent viremia [95]. Of note, among several respiratory viruses tested, SARS-CoV-2 has demonstrated to most potently suppress type I and type III IFN expression in both human bronchial epithelial cells and ferrets [81]. Thus, evasion of IFN signaling by SARS-CoV-2 and impaired IFN production in

human peripheral blood immune cells might contribute to the productive viral replication, transmission, and severe pathogenesis during COVID-19, although further testing is warranted to fully dissect these putative evasion pathways [95].

With regard to functional conservation of viral proteins, SARS-CoV and MERS-CoV nsps and accessory proteins circumvent viral RNA-sensing pathways at multiple stages (e.g. RIG-I, MDA-5) through proteasomal degradation and/or prevention of protein activation (Figure 5) [94]. Functional conservation between SARS-CoV and MERS-CoV PL<sup>pro</sup> (encoded by nsp3) proteins has been reported, where these proteins target the initial PRR signaling cascade at multiple levels of the pathway including—but not limited to—RIG-I, MAVS, TBK1, IRF3 and NF-κB (Figure 5) [96-98]. The SARS-CoV PL<sup>pro</sup> also targets the DNA-sensing pathway at STING (Figure 5); antagonizing this pathway might be important as mitochondrial stress during dengue virus infection triggers IFN-β production that is dependent on STING activation [99, 100]. Recent evidence suggests the SARS-CoV-2 PL<sup>pro</sup> might also inhibit IFN-I expression in human kidney epithelial cells, yet the mechanisms remain to be defined [101]. Moreover, nsp1 of highly pathogenic HCoVs, including SARS-CoV and MERS-CoV displays a pleiotropic effect, targeting several components of IFN-I signaling (Figure 5) [102, 103]. This potent suppressive function of nsp1 also appears to be maintained in SARS-CoV-2, primarily through shutdown of translational machinery and prevention of immune gene expression [101, 104, 105].

Furthermore, because there are only five accessory genes in the MERS-CoV genome compared to eight and seven in the SARS-CoV and SARS-CoV-2 genomes, respectively, similar immunosuppressive mechanisms may exist but appear to be mediated via different proteins [106, 107]. For example, SARS-CoVs ORF6 can inhibit IRF3 activation and STAT1 nuclear translocation, whereas this same effect is obtained by ORF4a/b and ORF5 of MERS-CoV
(Figure 5) [118, 119]. Coincidentally, the apparent loss of these proteins may provide evidence for why MERS-CoV is more sensitive to IFN treatment than SARS-CoVs in primary and continuous cells of the human airways [108]. The SARS-CoV-2 proteins appear to have stronger inhibitory effects than their counterparts of highly pathogenic SARS- and MERS-CoV [105]. In light of these findings, SARS-CoV-2 has replicated more efficiently than SARS-CoV in ex vivo human lung explants, possibly through the greater suppression of IFN-I/III cytokines [109]; further work will be needed to discern if the suppressive nature of SARS-CoV-2 can impact virus transmission during early phases of COVID-19, when IFNs are typically important for virus control. The 'common-cold' CoVs (e.g. HCoV-229E) and murine hepatitis virus (MHV) also compensate for the loss of many supplementary immunosuppressive proteins through capping viral mRNAs via nsp16 2'-O-methyltransferase (2'-O-MTase), and mutants lacking this activity exhibit diminished replication and dissemination in mice [110]. Further investigation is thus warranted to determine if these evasion genes might account for the increased virulence observed in individuals infected with SARS-CoV-2 (see Outstanding Questions).

### Animal Models of SARS-CoV-2

#### Mouse Models

Given that SARS-CoV-2 uses the same ACE2 entry receptor as SARS-CoV, rapidly deploying mouse models for pathogenesis studies were well underway within weeks of the pandemic's inception. However, various impediments remain for SARS-CoV-2 in productively infecting mice in these models, as it is unable to bind mouse ACE2 (mACE2) [111]. To overcome these prerequisites, several mouse models have been developed that recapitulate certain components of
human COVID-19. One of these strategies is to genetically modify mice to express human ACE 2 (hACE 2) (humanized mice) under the epithelial cell-specific cytokeratin-18 (Krt<sup>18</sup>) promoter [112], a universal chicken beta-actin promoter [113], or the endogenous mACE 2 promoter [111]. All these mice are susceptible to SARS-CoV-2 infection, but phenotypic disease varies because of differential hACE 2 tissue expression [112][113][111]. For instance, Krt18-hACE 2 and beta-actin-hACE 2-transgenic mice rapidly succumb to SARS-CoV-2 infection with lung infiltration of inflammatory immune cells inducing severe pulmonary disease, accompanied by evident thrombosis and anosmia, which partially recapitulate human COVID-19 [114] [115]. As the onset of severe histopathological changes occurs days after peak virus infection, these models recapture the delayed morbidity seen in COVID-19 patients as a result of inflammatory cell infiltration [115]. Therefore, employing humanized mouse models of severe SARS-CoV-2 infection might be useful for testing the efficacy of antiviral drugs, vaccines, and immune therapeutics that ablate hyperinflammation [114]. However, the broad expression of hACE 2 in these models significantly expands SARS-CoV-2 tissue tropisms and might alter its pathogenic mechanisms [114] [115]. For example, both SARS-CoV and SARS-CoV-2 infection lead to encephalitis in these mouse models, which is not common in COVID-19 patients [113, 115, 116]. Considering the fact that the majority of human SARS-CoV-2 infections are asymptomatic or mild, mice originally bearing mACE 2 that is replaced by hACE 2 may be more appropriate for assessing pathogenesis and tissue tropism[111]. This model develops mild lung pathology, with SARS-CoV-2 infection being restricted to the lung and intestine [111]. In addition to the transgenic modification, mice can also be sensitized to SARS-CoV-2 infection via transient transduction of adenovirus (Ad5)-or adeno-associated virus (AAV)-expressing hACE 2 in respiratory tissues, akin to the approach previously used for MERS-CoV infection [117-119].

These mice develop viral pneumonia, weight loss, severe pulmonary pathology, and a high viral load in the lung, consistent with human COVID-19 [119]. This approach might be quickly adapted to many genetically modified mouse strains that might provide mechanisms of SARS-CoV-2 pathogenesis and protective immune responses. This model is limited, however, by the transient ectopic expression of hACE2 from the Ad5/AAV vector that can induce mild bronchial inflammation and expand cell tropism of SARS-CoV-2 and thus, presumably alter disease pathogenesis [120].

Rather than genetic modification in host animals, viruses can also be genetically modified and be used in model animals [121, 122]. For instance, in one study, serial passaging of SARS-CoV-2 in mice led to enrichment of a N501Y viral mutant that elicited interstitial pneumonia and inflammatory responses in both aged and young wild-type BALB/c mice [123]. Another mouse-adapted SARS-CoV-2 strain (MA10) carrying three mutations in the RBD of Spike protein caused severe lung pathology and ARDS in mice, characteristic of severe COVID-19 [124]. Despite the three mutations in the RBD of the mouse-adapted Spike, vaccination with full-length SARS-CoV-2 Spike elicited robust neutralizing antibody titers and complete protection against a secondary challenge with MA10 [124]; these findings suggest that this strain may be applicable to pathogenesis studies, as well as antiviral drug and vaccine testing in rodents.

## Non-human primate Models

The role of non-human primates (NHP) in evaluating coronavirus pathogenesis cannot be understated. Depending on the NHP model utilized, clinical signs/symptoms may be mild or absent entirely [125-127]. In rhesus macaques, several studies have noted reduced appetite,

Journal Pre-proof

transient fevers (1 day post infection: dpi) and mild weight loss without overt signs of respiratory distress or mortality [125-127]. By contrast, cynomolgus macaques did not display any observational signs of disease in another study [126]. Although certain NHPs appear to only mimic mild disease (if any), rhesus macaques have exhibited high viral loads in nasal swabs, throat samples, and BALF early post inoculation, and viral RNA was still measurable by qPCR in the trachea and lung on day 21 p.i., highlighting the apparent tropism of SARS-CoV-2 for the URT and lingering viral nucleic acid in respiratory tissues after resolution of disease [51, 125]. SARS-CoV-2 has also been detected in nasal swabs at 10 dpi in NHPs, consistent with the prolonged URT shedding of virus in COVID-19 patients at ~9 dpi [51, 125, 126, 128]. The tropism of SARS-CoV-2 for the LRT in NHPs has also been recapitulated by the development of multifocal lesions and interstitial pneumonia, supporting the hypothesis that lung injury is driven by increased infiltration of neutrophils and macrophages into the lung following viral infection [125-127]. However, additional hallmarks of severe disease are absent in NHPs, particularly the characteristic systemic ‘cytokine storm’ present in COVID-19 patients; indeed, only transient elevations of serum inflammatory cytokines have been observed in NHPs, and have been reported to decline rapidly by 2 dpi [125]. Overall, these NHP models have displayed mild disease accompanied by viral dissemination in the URT and LRT, leading to localized lung inflammation, but devoid of the sustained systemic inflammatory response that has been noted in COVID-19 patients. Thus, NHPs models might be useful for studying mild COVID-19 characteristics, but presumably provide little information on the pathogenic mechanism(s) of severe COVID-19. To partially overcome this issue, aged rhesus macaques (15 years old) have been tested following SARS-CoV-2 infection and have demonstrated shedding of the virus for longer periods of time (14 days), as well as increased radiological and histopathological changes
such as thickened alveolar septum and diffuse severe interstitial pneumonia when compared to young macaques (3-5 years old) [129]. Therefore, these studies highlight the importance of also considering the age factor, as an additional variable, when selecting animal models that might closely, or accurately, recapitulate human disease.

Evaluating efficacious vaccine candidates in NHPs will also be important for understanding correlates of protection against SARS-CoV-2. Accordingly, reports of antibody-dependent enhancement, as well as of non-neutralizing humoral responses to the conserved regions of SARS-CoV-2, raise concerns on our future ability to effectively administer an immunogen without inducing immunopathology [130, 131]. Furthermore, upon viral challenge, lymphocytes have expanded in rhesus macaque models around 5 dpi with complementary B-cell responses against SARS-CoV-2 Spike appearing 14–18 dpi in blood samples [125]; expansion of these adaptive immune compartments was analogous to those observed in COVID-19 patients [37, 125, 132–134]. Subsequent re-challenged rhesus macaque have presented a rapid anamnestic immune response characterized by significantly higher neutralizing antibody (NAb) titers than the primary infection macaques [127]. Thus, protective efficacy seems to depend primarily on NAb titers, at least in NHPs, and so far, T-cell numbers have not substantially increased following re-challenge in the serum of these animals, and in a secondary study, CD4⁺ and CD8⁺ cytokine (e.g., IFN-γ) responses did not correlate with immune protection from DNA vaccines with different components of the SARS-CoV-2 Spike protein[135] [127].

Although these animals have failed to manifest overt signs of infection and respiratory compromise, NHPs still represent the ‘gold standard’ for evaluating the protective efficacy of human-bound SARS-CoV-2 vaccines based on parallels to humans in terms of viral tropism, immunopathology, and correlates of protection[127]. Further research is urgently needed to

17
explore the durability of immune responses to SARS-CoV-2, considering reports of waning immunity to other CoVs and the detection of pre-existing cross-reactive 'common-cold' CoV T-cells with SARS-CoV-2 in naïve humans (see Outstanding Questions) [136, 137].

# Concluding Remarks

The emergence of SARS-CoV-2 as the most recent example of zoonotic virus spillovers into humans underscores the fundamental need for well-funded surveillance organizations. The unrivaled spread of SARS-CoV-2 urgently demands that the global science community acts in harmony to disseminate accurate and stipulatory knowledge, with an immediate potential to influence policy and public health strategies/interventions at the national and local levels. Studies stemming from previous CoVs have jumpstarted our basic understandings of SARS-CoV-2 biology and facilitated the rapid deployment of vaccine candidates into clinical trials. Although clinical symptoms of COVID-19 indeed resemble some aspects of SARS and MERS, respectively, distinct and significant disparities exist in the transmissibility and immune responses to SARS-CoV-2 in order to modify therapeutics that protect against the full spectrum of disease (see Outstanding Questions), additional research will need to define the molecular mechanisms contributing to severe immunopathology in some patients, whereas others are completely protected from clinical symptoms. Prior work involving other CoVs, together with current and future studies of SARS-CoV-2 should provide society with the foundational knowledge to prepare for expected seasonal resurgences of SARS-CoV-2, as well as any potential spillover of additional CoVs into human populations, but hopefully, current

investigations will make important advances and contribute to increased knowledge and preparedness in this regard.

Acknowledgements

This work was supported by a National Institutes of Health grant R01AI132526 to P.W.

The authors declare no competing financial/non-financial interest.

References

1. Zhou, P. et al. (2020) A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579, 270–273.
2. Peiris, J. et al. (2003) Coronavirus as a possible cause of severe acute respiratory syndrome. The Lancet 361 (9366), 1319-1325.
3. Huang, C. et al. (2020) Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395 (10223), 497-506.
4. Organization, W.H. (2020) Coronavirus disease (COVID-19) pandemic. https://www.who.int/emergencies/diseases/novel-coronavirus-2019, (accessed 2020).
5. Gorbelenya, A.E. et al. (2020) The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. Nature Microbiology 5, 536-544.
6. Lu, R. et al. (2020) Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. The Lancet 395, 565-574.
7. Zhang, Y.Z. and Holmes, E.C. (2020) A Genomic Perspective on the Origin and Emergence of SARS-CoV-2. Cell 181 (2), 223–227.
8. Perlman, S. and Netland, J. (2009) Coronaviruses post-SARS: update on replication and pathogenesis. Nature Reviews Microbiology 7 (6), 439-450.
9. Wu, F. et al. (2020) A new coronavirus associated with human respiratory disease in China. Nature 579, 265-269.
10. Hoffmann, M. et al. (2020) SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell 181 (2), 271-280.
11. Snijder, E.J. et al. (2006) Ultrastructure and Origin of Membrane Vesicles Associated with the Severe Acute Respiratory Syndrome Coronavirus Replication Complex. Journal of Virology 80 (12), 5927–5940.
12. Wu, H.-Y. and Brian, D.A. (2010) Subgenomic messenger RNA amplification in coronaviruses. Proc Natl Acad Sci USA 107 (27), 12257-12262.
13. Peiris, J. et al. (2003) Clinical progression and viral load in a community outbreak of coronavirus-associated SARS pneumonia: a prospective study. The Lancet 361 (9371), 1767–1772.
14. Wrapp, D. et al. (2020) Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 367 (6483), 1260-1263.
15. Li, W. et al. (2003) Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454.
16. Yan, R. et al. (2020) Structural basis for the recognition of SARS-CoV-2 by full-length human ACE2. Science 367, 1444–1448.
17. Shang, J. et al. (2020) Cell entry mechanisms of SARS-CoV-2. Proc Natl Acad Sci USA 117 (21), 11727-11734.
18. Hoffmann, M. et al. (2020) A Multibasic Cleavage Site in the Spike Protein of SARS-CoV-2 Is Essential for Infection of Human Lung Cells. Mol Cell 78 (4), 779-784 e5.
19. Yurkovetskiy, L. et al. (2020) Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant. Cell. https://doi.org/10.1016/j.cell.2020.09.032 In Press
20. Li, Q. et al. (2020) The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity. Cell 182 (5), 1284–1294.

21. Korber, B. et al. (2020) Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus. Cell 182 (4), 812–827.
22. Jia, H.P. et al. (2005) ACE2 Receptor Expression and Severe Acute Respiratory Syndrome Coronavirus Infection Depend on Differentiation of Human Airway Epithelia. Journal of Virology 79 (23), 14614–14621.
23. Hamming, I. et al. (2004) Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis. The Journal of Pathology 203 (2), 631–637.
24. Kuba, K. et al. (2005) A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus-induced lung injury. Nature Medicine 11 (8), 875–879.
25. Ziegler, C.G.K. et al. (2020) SARS-CoV-2 receptor ACE2 is an interferon-stimulated gene in human airway epithelial cells and is detected in specific cell subsets across tissues. Cell 181 (5), 1016-1035.
26. Sun, K. et al. (2020) Atlas of ACE2 gene expression in mammals reveals novel insights in transmission of SARS-CoV-2. Preprint in BioRxiv. doi:https://doi.org/10.1101/2020.03.30.015644
27. Shulla, A. et al. (2011) A Transmembrane Serine Protease Is Linked to the Severe Acute Respiratory Syndrome Coronavirus Receptor and Activates Virus Entry. Journal of Virology 85 (2), 873–882.
28. Zhao, Y. et al. (2020) Single-cell RNA expression profiling of ACE2, the receptor of SARS-CoV-2. American Journal of Respiratory and Critical Care Medicine 202 (5), 756-759.
29. Imai, Y. et al. (2005) Angiotensin-converting enzyme 2 protects from severe acute lung failure. Nature 436, 112-116.
30. Wang, S. et al. (2008) Endocytosis of the receptor-binding domain of SARS-CoV spike protein together with virus receptor ACE2. Virus Research 136 (1), 8–15.
31. Smith, J.C. et al. (2020) Cigarette smoke exposure and inflammatory signaling increase the expression of the SARS-CoV-2 receptor ACE2 in the respiratory tract. Developmental Cell 53 (5), 514-529.
32. Follis, K.E. et al. (2006) Furin cleavage of the SARS coronavirus spike glycoprotein enhances cell-cell fusion but does not affect virion entry. Virology 350 (2), 358-369.
33. Lukassen, S. et al. (2020) SARS-CoV-2 receptor ACE2 and TMPRSS2 are primarily expressed in bronchial transient secretory cells. The EMBO Journal, 39:e105114.
34. Hui, K.P.Y. et al. (2020) Tropism, replication competence, and innate immune responses of the coronavirus SARS-CoV-2 in human respiratory tract and conjunctiva: an analysis in ex-vivo and in-vitro cultures. Lancet Respiratory Medicine 8 (7), 687-695.
35. Team, T.C.-I. (2020) Clinical and virologic characteristics of the first 12 patients with coronavirus disease 2019 (COVID-19) in the United States. Nature Medicine (26), 861–868.
36. Xu, H. et al. (2020) High expression of ACE2 receptor of 2019-nCoV on the epithelial cells of oral mucosa. International Journal of Oral Science 12 (8).
37. Wölfel, R. et al. (2020) Virological assessment of hospitalized patients with COVID-2019. Nature 581, 465-469.
38. Su, S. et al. (2016) Epidemiology, Genetic Recombination, and Pathogenesis of Coronaviruses. Trends in Microbiology 24 (6), 490-502.
39. Zhang, H. et al. (2020) The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes Gut 69 (6), 1010–1018.

40. Cholankeril, G. et al. (2020) High Prevalence of Concurrent Gastrointestinal Manifestations in Patients with SARS-CoV-2: Early Experience from California. Gastroenterology 159 (2), 775-777.

41. Xiao, F. et al. (2020) Evidence for Gastrointestinal Infection of SARS-CoV-2. Gastroenterology 158, 1831-1833.

42. Leung, W.K. et al. (2003) Enteric involvement of severe acute respiratory syndrome-associated coronavirus infection. Gastroenterology 125 (4), 1011–1017.

43. Yu, I.T.S. et al. (2004) Evidence of Airborne Transmission of the Severe Acute Respiratory Syndrome Virus. The New England Journal of Medicine 350, 1731-1739.

44. Otter, J.A. et al. (2016) Transmission of SARS and MERS coronaviruses and influenza virus in healthcare settings: the possible role of dry surface contamination. The Journal of Hospital Infection 92 (3), 235–250.

45. Li, Y. et al. (2004) Role of air distribution in SARS transmission during the largest nosocomial outbreak in Hong Kong. International Journal of Indoor Environment and Health 15 (2), 83-95.

46. Wang, D. et al. (2020) Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China. JAMA 323 (11), 1061-1069.

47. Li, Q. et al. (2020) Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia. The New England Journal of Medicine 382 (13), 1199-1207.

48. Chan, J.F.W. et al. (2020) A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet 395, 514-523.

49. Ferrett, L. et al. (2020) Quantifying SARS-CoV-2 transmission suggests epidemic control with digital contact tracing. Science 368 (6491), eabb6936.

50. Arons, M.M. et al. (2020) Presymptomatic SARS-CoV-2 Infections and Transmission in a Skilled Nursing Facility. The New England Journal of Medicine 382, 2081-2090.

51. Zou, L. et al. (2020) SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients. The New England Journal of Medicine 382, 1177-1179.

52. Pan, Y. et al. (2020) Viral load of SARS-CoV-2 in clinical samples. The Lancet Infectious Diseases 20 (4), 411-412.

53. Mizumoto, K. et al. (2020) Estimating the asymptomatic proportion of coronavirus disease 2019 (COVID-19) cases on board the Diamond Princess cruise ship, Yokohama, Japan, 2020. Eurosurveillance 25 (10).

54. Heng Meng et al. (2020) CT imaging and clinical course of asymptomatic cases with COVID-19 pneumonia at admission in Wuhan, China. The Journal of Infection 81 (1), 33-39.

55. Camilla Rothe et al. (2020) Transmission of 2019-nCoV Infection from an Asymptomatic Contact in Germany. The New England Journal of Medicine 382 (10), 970-971.

56. Guiqing He et al. (2020) The clinical feature of silent infections of novel coronavirus infection (COVID-19) in Wenzhou. Journal of Medical Virology 92 (10), 1761-1763.

57. Mukhra, R. et al. (2020) Possible modes of transmission of Novel coronavirus SARS-CoV-2: a review. Acta Biomed 91 (3), e2020036.

58. Tomlinson, B. and Cockram, C. (2003) SARS: experience at Prince of Wales Hospital, Hong Kong. The Lancet 361, 1486-1487.

59. Yuan Liu et al. (2020) Aerodynamic analysis of SARS-CoV-2 in two Wuhan hospitals. Nature 582, 557–560.

60. Doremalen, N.v. et al. (2020) Aerosol and Surface Stability of SARS-CoV-2 as Compared with SARS-CoV-1. The New England Journal of Medicine 382, 1564-1567.

61. Ong, S.W.X. et al. (2020) Air, Surface Environmental, and Personal Protective Equipment Contamination by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) From a Symptomatic Patient. JAMA 323 (16), 1610-1612.

62. Xu, Y. et al. (2020) Characteristics of pediatric SARS-CoV-2 infection and potential evidence for persistent fecal viral shedding. Nature Medicine 26, 502–505.

63. Ksiazek, T.G. et al. (2003) A Novel Coronavirus Associated with Severe Acute Respiratory Syndrome. The New England Journal of Medicine 348 (20), 1953-1966.

64. Guery, B. et al. (2013) Clinical features and viral diagnosis of two cases of infection with Middle East Respiratory Syndrome coronavirus: a report of nosocomial transmission. The Lancet 381 (9885), 2265-2272.

65. Chen, N. et al. (2020) Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. The Lancet 395 (10223), 507–513.

66. Puntmann, V.O. et al. (2020) Outcomes of Cardiovascular Magnetic Resonance Imaging in Patients Recently Recovered From Coronavirus Disease 2019 (COVID-19). JAMA Cardiology.

67. Chao, J.Y. et al. (2020) Clinical Characteristics and Outcomes of Hospitalized and Critically Ill Children and Adolescents with Coronavirus Disease 2019 (COVID-19) at a Tertiary Care Medical Center in New York City. Journal of Pediatrics 233, 14-19.

68. Nikolich-Zugich, J. (2018) The Twilight of Immunity: Emerging Concepts in Aging of the Immune System. Nature Immunology 19 (1), 10-19.

69. Wu, J.T. et al. (2020) Estimating clinical severity of COVID-19 from the transmission dynamics in Wuhan, China. Nature Medicine 26, 506–510.

70. Chow, R.D. and Chen, S. (2020) The aging transcriptome and cellular landscape of the human lung in relation to SARS-CoV-2. Preprint in BioRxiv. doi:https://doi.org/10.1101/2020.04.07.030684

71. Yang, X. et al. (2020) Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study. Lancet Respiratory Medicine 8 (5), 475-481.

72. Imai, S.T.K. et al. (2020) Clinical characteristics of COVID-19 in 104 people with SARS-CoV-2 infection on the Diamond Princess cruise ship: a retrospective analysis. Lancet Infectious Diseases 20 (9), 1043-1050.

73. Zhang, B. et al. (2020) Clinical characteristics of 82 cases of death from COVID-19. PLoS ONE 15 (7).

74. Wu, C. et al. (2020) Risk Factors Associated With Acute Respiratory Distress Syndrome and Death in Patients With Coronavirus Disease 2019 Pneumonia in Wuhan, China. JAMA Internal Medicine 180, 7.

75. Liu, K. et al. (2020) Clinical features of COVID-19 in elderly patients: A comparison with young and middle-aged patients. The Journal of Infection 80 (6), 14-18.

76. Zhou, R. et al. (2020) Acute SARS-CoV-2 Infection Impairs Dendritic Cell and T Cell Responses. Immunity.

77. Chen, J. et al. (2010) Cellular Immune Responses to Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) Infection in Senescent BALB/c Mice: CD4+ T Cells Are Important in Control of SARS-CoV Infection. Journal of Virology 84 (3), 1289–1301.

78. Zhao, J. et al. (2009) Evasion by Stealth: Inefficient Immune Activation Underlies Poor T Cell Response and Severe Disease in SARS-CoV-Infected Mice. PLoS Pathogens 5 (10), e1000636.

79. Liao, M. et al. (2020) Single-cell landscape of bronchoalveolar immune cells in patients with COVID-19. Nature Medicine 26 (842–844).

80. Qin, C. et al. (2020) Dysregulation of immune response in patients with COVID-19 in Wuhan, China. Clinical Infectious Diseases 71 (15), 762–768.

81. Blanco-Melo, D. et al. (2020) Imbalanced host response to SARS-CoV-2 drives development of COVID-19. Cell 181 (5).

82. Chen, H. et al. (2020) SARS-CoV-2 activates lung epithelia cell proinflammatory signaling and leads to immune dysregulation in COVID-19 patients by single-cell sequencing. Preprint in MedRxiv. doi: https://doi.org/10.1101/2020.05.08.20096024

83. Ding, Y. et al. (2003) The clinical pathology of severe acute respiratory syndrome (SARS): a report from China. The Journal of Pathology 200 (3), 282–289.

84. Nicholls, J.M. et al. (2003) Lung pathology of fatal severe acute respiratory syndrome. The Lancet 361 (9371), 1773–1778.

85. Ruan, Q. et al. Clinical predictors of mortality due to COVID-19 based on an analysis of data of 150 patients from Wuhan, China. Intensive Care Medicine 46, 845–848.

86. Moore, J.B. and June, C.H. (2020) Cytokine release syndrome in severe COVID-19. Science 368 (6490), 473–474.

87. Channappanavar, R. and Perlman, S. (2017) Pathogenic human coronavirus infections: causes and consequences of cytokine storm and immunopathology. Seminars in Immunopathology 39 (5), 529–539.

88. Marnell, L. et al. (2005) C-reactive protein: Ligands, receptors and role in inflammation. Clinical Immunology 117 (2), 104–111.

89. Zuo, Y. et al. (2020) Neutrophil extracellular traps in COVID-19. JCI Insight 5 (11).

90. Varga, Z. et al. (2020) Endothelial cell infection and endotheliitis in COVID-19. Lancet 395, 1417–1418.

91. Zhou, F. et al. (2020) Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study. The Lancet 395 (10229), 1054–1062.

92. Tang, N. et al. (2020) Abnormal coagulation parameters are associated with poor prognosis in patients with novel coronavirus pneumonia. Journal of Thrombosis Haemostasis 18 (4), 844–847.

93. Totura, A.L. and Baric, R. (2012) SARS coronavirus pathogenesis: host innate immune responses and viral antagonism of interferon. Current Opinion in Virology 2 (3), 264–275.

94. E.Kindler et al. (2016) Interaction of SARS and MERS Coronaviruses with the Antiviral Interferon Response. Advances in Viral Research 96, 219–243.

95. Hadjadj, J. et al. (2020) Impaired type I interferon activity and inflammatory responses in severe COVID-19 patients. Science Immunology.

96. Devaraj, S.G. et al. (2007) Regulation of IRF-3 dependent innate immunity by the Papain-like protease domain of the SARS coronavirus. Journal of Biological Chemistry 282 (44), 32208–32221.

97. Frieman, M. et al. (2009) Severe Acute Respiratory Syndrome Coronavirus Papain-Like Protease Ubiquitin-Like Domain and Catalytic Domain Regulate Antagonism of IRF3 and NF-kB Signaling. Journal of Virology 83, 6689–6705.

98. Bailey-Elkin, B.A. et al. (2014) Crystal Structure of the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Papain-like Protease Bound to Ubiquitin Facilitates Targeted Disruption of Deubiquitinating Activity to Demonstrate Its Role in Innate Immune Suppression. Journal of Biological Chemistry 289 (50), 34667–34682.

99. Aarreberg, L.D. et al. (2019) Interleukin-1β Induces mtDNA Release to Activate Innate Immune Signaling via cGAS-STING. Molecular Cell 74 (4), 801–815.

100. Sun, L. et al. (2012) Coronavirus Papain-like Proteases Negatively Regulate Antiviral Innate Immune Response through Disruption of STING-Mediated Signaling. PLoS One 7 (2), e30802.

101. Lei, X. et al. (2020) Activation and evasion of type I interferon responses by SARS-CoV-2. Nature Communications 11, 3810.

102. Lokugamage, K.G. et al. (2012) Severe acute respiratory syndrome coronavirus protein nsp1 is a novel eukaryotic translation inhibitor that represses multiple steps of translation initiation. Journal of Virology 86 (24), 13598-13608.

103. Lokugamage, K.G. et al. (2015) Middle East Respiratory Syndrome Coronavirus nsp1 Inhibits Host Gene Expression by Selectively Targeting mRNAs Transcribed in the Nucleus while Sparing mRNAs of Cytoplasmic Origin. Journal of Virology 89 (21), 10970-109981.

104. Thoms, M. et al. (2020) Structural basis for translational shutdown and immune evasion by the Nsp1 protein of SARS-CoV-2. Science 369 (6508), 1249-1255.

105. Xia, H. et al. (2020) Evasion of type-I interferon by SARS-CoV-2. Cell Reports. 33 (1). https://doi.org/10.1016/j.celrep.2020.108234

106. Yang, Y. et al. (2013) The structural and accessory proteins M, ORF4a, ORF4b, and ORF 5 of Middle East respiratory syndrome coronavirus (MERS-CoV) are potent interferon antagonists. Protein & Cell 4 (12), 951-961.

107. Kopecky-Bromberg, S.A. et al. (2007) Severe Acute Respiratory Syndrome Coronavirus Open Reading Frame (ORF) 3b, ORF 6, and Nucleocapsid Proteins Function as Interferon Antagonists. Journal of Virology 81 (2), 548-557.

108. Zielecki, F. et al. (2013) Human Cell Tropism and Innate Immune System Interactions of Human Respiratory Coronavirus EMC Compared to Those of Severe Acute Respiratory Syndrome Coronavirus. Journal of Virology 87 (9), 5300-5304.

109. Chu, H. et al. (2020) Comparative replication and immune activation profiles of SARS-CoV-2 and SARS-CoV in human lungs: an ex vivo study with implications for the pathogenesis of COVID-19. Clinical Infectious Diseases 71 (6), 1400-1409.

110. Züst, R. et al. (2011) Ribose 2′-O-methylation provides a molecular signature for the distinction of self and non-self mRNA dependent on the RNA sensor Mda5. Nature Immunology 12, 137-143.

111. Bao, L. et al. (2020) The pathogenicity of SARS-CoV-2 in hACE2 transgenic mice. Nature.

112. McCray, P.B. et al. (2007) Lethal Infection of K18-hACE2 Mice Infected with Severe Acute Respiratory Syndrome Coronavirus. Journal of Virology 81 (2), 813–821.

113. Tseng, C.-T.K. et al. (2007) Severe Acute Respiratory Syndrome Coronavirus Infection of Mice Transgenic for the Human Angiotensin-Converting Enzyme 2 Virus Receptor. Journal of Virology 81 (3), 1162–1173.

114. Zheng, J. et al. (2020) K18-hACE2 Mice for Studies of COVID-19 Treatments and Pathogenesis Including Anosmia. Preprint in BioRxiv. 2020 Aug 10. doi:10.1101/2020.08.07.242073

115. Winkler, E.S. et al. (2020) SARS-CoV-2 infection of human ACE2-transgenic mice causes severe lung inflammation and impaired function. Nature Immunology. https://doi.org/10.1038/s41590-020-0778-2

116. Netland, J. et al. (2008) Severe Acute Respiratory Syndrome Coronavirus Infection Causes Neuronal Death in the Absence of Encephalitis in Mice Transgenic for Human ACE2. Journal of Virology 82 (15), 7264-7275.
117. Hassan, A.O. et al. (2020) A SARS-CoV-2 Infection Model in Mice Demonstrates Protection by Neutralizing Antibodies. Cell 182, 1-10.
118. Zhao, J. et al. (2014) Rapid generation of a mouse model for Middle East respiratory syndrome. Proc Natl Acad Sci USA 111 (13), 4970-4975.
119. Jing Sun et al. (2020) Generation of a Broadly Useful Model for COVID-19 Pathogenesis Vaccination, and Treatment. Cell. 182, (3) 734-743.e5
120. Wold, W.S. and Toth, K. (2013) Adenovirus vectors for gene therapy, vaccination and cancer gene therapy. Curr Gene Ther 13 (6), 421-33.
121. Roberts, A. et al. (2007) A Mouse-Adapted SARS-Coronavirus Causes Disease and Mortality in BALB/c Mice. PLoS Pathogens 3 (1), e5.
122. Dinnon, K.H. et al. (2020) A mouse-adapted SARS-CoV-2 model for the evaluation of COVID-19 medical countermeasures. Nature. https://doi.org/10.1038/s41586-020-2708-8
123. Gordon, D.E. et al. (2020) A SARS-CoV-2 protein interaction map reveals targets for drug repurposing. Nature 583, 459–468.
124. Leist, S.R. et al. (2020) A Mouse-adapted SARS-CoV-2 induces Acute Lung Injury (ALI) and mortality in Standard Laboratory Mice. Cell. doi: 10.1016/j.cell.2020.09.050
125. Munster, V.J. et al. (2020) Respiratory disease in rhesus macaques inoculated with SARS-CoV-2. Nature 585, 268–272.
126. Rockx, B. et al. (2020) Comparative pathogenesis of COVID-19, MERS, and SARS in a nonhuman primate model. Science 368 (6494), 1012-1015.
127. Chandrashekar, A. et al. (2020) SARS-CoV-2 infection protects against rechallenge in rhesus macaques. Science 369 (6505), 812-817.
128. Lawler, J.V. et al. (2006) Cynomolgus Macaque as an Animal Model for Severe Acute Respiratory Syndrome. PLoS Medicine 3 (5), e149.
129. Yu, P. et al. (2020) Age-related rhesus macaque models of COVID-19. Animal Models and Experimental Medicine 3 (1), 93-97.
130. Tseng, C.-T. et al. (2012) Immunization with SARS Coronavirus Vaccines Leads to Pulmonary Immunopathology on Challenge with the SARS Virus. PLoS One 7 (4), e35421.
131. Liu, L. et al. (2019) Anti-spike IgG causes severe acute lung injury by skewing macrophage responses during acute SARS-CoV infection. JCI Insight 4 (4), e123158.
132. Zhao, J. et al. (2020) Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019 Clinical Infectious Diseases (ciaa344).
133. Long, Q.X. et al. (2020) Antibody responses to SARS-CoV-2 in patients with COVID-19. Nature Medicine 26, 845–848.
134. Mathew, D. et al. (2020) Deep immune profiling of COVID-19 patients reveals distinct immunotypes with therapeutic implications. Science 369 (6508), eabc8511.
135. Yu, J. et al. (2020) DNA vaccine protection against SARS-CoV-2 in rhesus macaques. Science. 369(6505):806-811
136. Edridge, A.W.D. et al. (2020) Seasonal coronavirus protective immunity is short-lasting. Nature Medicine. https://doi.org/10.1038/s41591-020-1083-1
137. Grifoni, A. et al. (2020) Targets of T Cell Responses to SARS-CoV-2 Coronavirus in Humans with COVID-19 Disease and Unexposed Individuals. Cell 181 (7), 1489–1501.

Journal Pre-proof

138. Zhang, J.-j. et al. (2020) Clinical characteristics of 140 patients infected with SARS-CoV-2 in Wuhan, China. Allergy 75(7), 1730-1741.

139. Liao, J. et al. (2020) Epidemiological and Clinical Characteristics of COVID-19 in Adolescents and Young Adults. The Innovation 1(1).

140. Rifno, N. et al. (2020) Neurologic manifestations in 1760 COVID-19 patients admitted to Papa Giovanni XXIII Hospital, Bergamo, Italy. Journal of Neurology.
https://doi.org/10.1007/s00415-020-10251-5

Table 1: Common laboratory indices and radiological findings of COVID-19 patients

| Laboratory Findings | Reference | [65] | [138] | [73] | [139] |
|---------------------|-----------|------|-------|------|-------|
| Patient Population   | Middle-aged, hospitalized | n=99 | Middle-aged, Hospitalized/ICU | n=1099 | Elderly, mild-severe | n=71 | Middle-aged, hospitalized adults | n=82 | Elderly, deceased | n=140 | Young, mild disease | n=46 |
| Ground-glass opacities | 14 | 56.4<sup>b</sup> | 100 | 99.3 | -- | 63 |
| Pneumonia (unilateral or bilateral)<sup>c</sup> | 100 | 91.1 | 73 | -- | 93.9 | -- |
| Lymphocytopenia | 35 | 33.2 | [1.1-3.2 x10<sup>9</sup>/mL] | 37 | [<1300/mm<sup>3</sup>] | [<1200/μL] | 75.4 | [1.1-3.2 x10<sup>9</sup>/mL] | 89.2 | [<1.0 x10<sup>9</sup>/L] | 63 | [<1.5 x10<sup>9</sup>/L] |
| Leukopenia | 9 | 33.7 | [3.5-9.5 x10<sup>9</sup>/mL] | 21 | [<4000/mm<sup>3</sup>] | [<4000/μL] | 19.6 | [3.5-9.5 x10<sup>9</sup>/mL] | 21.7 | [<4 x10<sup>9</sup>/L] |
| Thrombocytopenia | 12 | 36.2 | [125-350 x10<sup>9</sup>/mL] | 10 | [<150,000/mm<sup>3</sup>] | [<15x10<sup>4</sup>/μL] | -- | 24.3 | [<100 x10<sup>9</sup>/L] | 21.7 | [<150 x10<sup>9</sup>/L] |
| Neutrophilia | 38 | -- | [1.8-6.3 x10<sup>9</sup>/mL] | -- | -- | -- | 74.3 | >6.3 | x10<sup>9</sup>/L | -- |
| ↑ CRP [mg/L]<sup>e</sup> | 86 | 60.7 | [0-5] | ≥10 | 59 | >3 | 91.9 | [0-3] | 100 | ≥10 | 19.6 | ≥10 |
| ↑ Alanine aminotransferase [U/L] | 28 | 21.3 | [9-50] | >40 | 18 | -- | 30.6 | >40 | 15.2 | >40 |
| ↑ D-dimer [μg/L] | 36 | 46.4 | [0-1.5] | ≥500 | -- | 43.2 | [0-243] | 97.1 | >550 | 15.2 | ≥500 |
| ↑ Prothrombin | 5 | -- | -- | -- | 100 | -- |

27

time [10.5-13.5] [12.3-
[s] 14.3]

a Values for each laboratory manifestation represent the percentage of patients with that clinical finding above or below the normal range (listed below in brackets); dashed lines indicate measurements not taken during referenced study.

b Percentage of patients with lung ground-glass opacity on a chest CT scan (technique specifically denoted in original study).

c Pneumonia was not always definitively mentioned in some studies, albeit lung manifestations were commonly recorded.

d Upward arrow denotes an elevation of measured indices above reference value for those percentage of patients

e CRP (C-reactive protein)

f In the last 24 hours leading up to death, all 13 patients which were included for this metric had a prothrombin time of >12.1s.

### Glossary

Aerosol: suspension of fine solid or liquid droplets in the air (or a gas medium), such as dusts, mists, or fumes.

Anamnestic immune response: memory immune response to a previously encountered antigen.

Angiotensin-converting enzyme 2 (ACE2): cell surface enzyme of endothelial, epithelial, and other cells, with a well-defined function in maintaining normal blood pressure.

Anosmia: partial or complete loss of the sense of smell.

Antibody-dependent enhancement: phenomenon by which antibodies against a virus are suboptimal to the virus and enhance its entry into host cells.

Convalescence period: the time of gradual recovery after an illness or injury.

Correlates of protection: quantifiable parameters such as antibodies, indicating that a host is protected against microbial infection.

Cytokine storm: severe immune reaction in which the body releases too many cytokines into the blood too quickly.

D-dimer: fibrin degradation product in the blood after a clot is degraded by fibrinolysis.

Disseminated intravascular coagulation (DIC): condition in which blood clots form throughout the body and block small blood vessels, leading to multiorgan failure.

Fecal-oral transmission: route of disease transmission by which an infectious agent in fecal materials is passed to the mouth of another.

Fomite: inanimate object (clothes, utensil, and furniture etc.) that, when contaminated with an infectious agent, can transfer the infectious agent to a new host.

Furin: proprotein convertase that cleaves a precursor protein into a biologically active state.

Incubation period: timeframe elapsed between when a host is first exposed to an infectious agent and when signs or symptoms begin to appear.

Lung ground glass opacity: nonspecific radiological description of an area of increased opacity in the lung through which vessels and bronchial structures are still visible.

Neutralizing antibody (NAb): an antibody that binds a pathogen with high affinity and prevents the latter from exerting its biological effect.

Neutrophil extracellular traps (NETs): networks of extracellular fibers, primarily composed of DNA from neutrophils due to chromatin decondensation, which can 'trap' extracellular pathogens.

Pattern recognition receptor: germline-encoded host sensor that recognizes a signature pattern in microbial molecules.

Prodromal period: the time immediately following the incubation period of a microbial infection in which a host begins to experience symptoms or changes in behavior/functioning.

Prothrombin time: measurement of the extrinsic pathway of coagulation.

R₀ (reproductive number): the expected number of new disease cases generated by one case. An R₀ > 1 indicates the outbreak will expand; R₀ < 1 the outbreak will die out.

Respiratory droplet: small aqueous droplet produced by exhalation, consisting of saliva or mucus and other matter derived from respiratory tract surfaces.

Zoonotic disease: infectious disease caused by a pathogen that has crossed a species barrier from animals to humans.

## Figure Legends

**Figure 1. The SARS-CoV-2 lifecycle.** SARS-related coronavirus (SARS-CoV and SARS-CoV-2) lifecycle commences by binding of the envelope spike protein to its cognate receptor, angiotensin-converting enzyme 2 (ACE2). Efficient host cell entry then depends on 1) cleavage of the S1/S2 site by the surface transmembrane protease serine 2 (TMPRSS2) and/or 2) endolysosomal cathepsin L, which mediates virus-cell membrane fusion at the cell surface and endosomal compartments, respectively. Through either entry mechanism, the RNA genome is released into the cytosol where it is translated into the replicase proteins (open reading frame 1a/b: ORF1a/b). The polyproteins (pp1a and pp1ab) are cleaved by a virus-encoded protease into individual replicase complex nonstructural proteins (nsp's) (including the RNA-dependent RNA polymerase: RdRp). Replication begins in virus-induced double-membrane vesicles (DMVs) derived from the ER, which ultimately integrate to form elaborate webs of convoluted membranes. Here, the incoming positive strand genome then serves as a template for full-length negative strand RNA and subgenomic RNA. Subgenomic RNA translation results in both structural proteins and accessory proteins (simplified here as N, S, M, E) that are inserted into the endoplasmic reticulum-Golgi intermediate compartment (ERGIC) for virion assembly. Finally, subsequent positive-sense RNA genomes are incorporated into newly synthesized virions which are secreted from the plasma membrane.[6, 8, 11, 12] (Figure generated with BioRender)

**Figure 2. Proposed SARS-CoV-2 transmission routes.** The ongoing COVID-19 pandemic has resulted in numerous accounts of different transmission routes between humans. Droplet transmission (>5μM) is the most pronounced and heavily implicated mode of transmission reported during the pandemic. Direct contact spread from one infected individual to a second, naive person has also been considered a driver of human-to-human transmission, especially in households with close interactions between family members. The contagiousness of SARS-CoV-2 after disposition on fomites (e.g., door handle) is still under investigation, but is likely a compounding factor for transmission events, albeit less frequently than droplet or contact-driven transmission. Both airborne and fecal-oral human-to-human transmission events were reported in
the precursor SARS-CoV epidemic but have yet to be observed in the current crises. Solid arrows show confirmed viral transfer from one infected person to another with a declining gradient in arrow width denoting the relative contributions of each transmission route. Dashed lines show the plausibility of that transmission type but have yet to be confirmed. SARS-CoV-2 symbol in “infected patient” indicate where RNA/infectious virus has been detected[43, 44, 47-49, 57, 59] [60]. (Figure generated with BioRender)

---

Figure 3. Clinical symptoms of COVID-19. COVID-19 manifestations in humans have been described to incorporate multiple body systems with varying degrees of onset and severity. Both the upper respiratory tract and lower respiratory tract manifestations are often the most noticeable if a patient is not asymptomatic, in addition to systemic symptoms that are the most frequently reported regardless of disease severity. Red highlighted signs/symptoms tend to be overrepresented in severe patients—but common symptoms are also present in more advanced COVID-19. SARS-CoV-2 virus symbol denotes where a live virus and/or viral RNA has been isolated. ARDS: Acute respiratory distress syndrome [37, 46, 48, 66, 140]. (Figure generated with BioRender)

---

Key Figure, Figure 4. A brief overview of lung pathology in COVID-19 patients. Following inhalation of SARS-CoV-2 into the respiratory tract, the virus traverses deep into the lower lung where it infects a range of cells including alveolar airway epithelial cells, vascular endothelial cells and alveolar macrophages. Upon entry, SARS-CoV-2 can likely be detected by cytosolic innate immune sensors, as well as endosomal toll-like receptors (TLRs) that signal downstream to produce type-I/III interferons (IFNs) and proinflammatory mediators. The high concentration of inflammatory cytokines/chemokines amplifies the destructive tissue damage via endothelial dysfunction and vasodilation allowing the recruitment of immune cells, in this case, macrophages and neutrophils. Vascular leakage and compromised barrier function promote endotheliitis and lung edema limiting gas exchange that then facilitates a hypoxic environment leading to respiratory/organ failure. The inflammatory milieu induces endothelial cells to upregulate leukocyte adhesion molecules, thereby promoting the accumulation of immune cells that may also contribute to the rapid progression of respiratory failure. Hyperinflammation in the lung further induces transcriptional changes in macrophages and neutrophils that perpetuate tissue damage that ultimately leads to irreversible lung damage. Recent evidence suggests systemic inflammation may induce long-term sequelae in heart tissues [66, 79, 80, 82, 84, 87, 90, 95]. (Figure generated with BioRender)

---

Figure 5. Evasion of the PRR-IFN-I pathways by CoVs. A simplified schematic of the canonical interferon (IFN) response after sensing RNA viruses. Viral nucleic acid is first recognized by pattern-recognition receptors (PRRs) (e.g. retinoic acid-inducible gene I, RIG-I) that perpetuate signal transduction through an adaptor complex on the mitochondrial (Mitochondrial antiviral-signaling protein, MAVS) or endoplasmic reticulum (Stimulator of interferon genes, STING) membrane surface. Here, the PRR-adaptor interactions recruit kinases that converge into a large complex, leading to phosphorylation of interferon regulatory factor 3/7

---
(IRF3/7) and nuclear factor NF-κB (NF-κB) transcription factors that enter the nucleus and transcribe IFN genes. Type-I, type-III IFNs then signal in an autocrine or paracrine manner through the Janus kinase 1 (JAK1)/signal transducer and activator of transcription 1 and 2 (STAT1/2) pathway, culminating in antiviral interferon-stimulated gene (ISG) transcription. Listed here are SARS-CoV (abbreviated CoV), SARS-CoV-2 (abbreviated CoV-2) and MERS-CoV (abbreviated M-CoV) IFN-I antagonists, which make these viruses resistant to interferon responses. IFN-III is also implicated in exhibiting potent antiviral effects in lung/intestinal tissues, but the underlying evasion strategies of this pathway for these viruses are currently unknown. SARS-CoV proteins are highlighted in blue, while functions of SARS-CoV-2 and MERS-CoV proteins are highlighted in red and green, respectively. Question mark symbol (?) denotes SARS-CoV-2 protein bound a member of that signaling pathway in [123], but further work is necessary to confirm its immunological mechanism. SARS-CoV-2 proteins with * denotes functional conservation with SARS-CoV [93-95, 100-107] [96, 98]. (Figure generated with BioRender).

Outstanding Questions

- Which animal(s) serves as the natural reservoir of SARS-CoV-2?
- Does active replication of SARS-CoV-2 in the upper respiratory tract contribute to enhanced transmissibility in humans?
- Is intestinal SARS-CoV-2 infection a source of virus transmission?
- Which SARS-CoV-2 proteins antagonize innate and adaptive immune responses? Do the SARS-CoV-2 proteins with more potent antagonistic immune functions increase virulence in humans compared to other HCoVs?
- Why do some recovered patients fail to develop neutralizing antibodies?
- What are the host and/or viral factors driving inflammatory imbalances in severe COVID-19 cases?
- What are the underlying mechanisms contributing to an inadequate IFN response to SARS-CoV-2?
- What are the correlates of immune protection for SARS-CoV-2 and will they provide sterilizing immunity?
- Will candidate vaccines against SARS-CoV-2 also be effective in elderly subpopulations (with or without comorbidities)?

Highlights Box

- The emergence of SARS-CoV-2 from China and the rapidity of a worldwide pandemic has promoted global collaboration, built on the body of work established from previous SARS-CoV and MERS-CoV outbreaks. These past experiences have aided the swiftness by which the research community has responded with an astonishing body of work.
- SARS-CoV-2 is a novel virus in the Betacoronavirus genus and exhibits similarities to SARS-CoV in genome structure, tissue tropism and viral pathogenesis. Yet, SARS-CoV-2 appears to be more transmissible and the diversity of immune responses are poorly understood.
- Highly pathogenic coronaviruses display potent interferon (IFN) antagonism, which is evident in cases of severe COVID-19 with reduced interferon signaling, and an overaggressive immune response compounded by heightened cytokines/chemokines.
- Animal models for SARS-CoV-2 recapitulate important aspects of human COVID-19 that are essential for evaluating current and prospective antiviral therapeutics and vaccine candidates.
