
# The RAL signaling network: Cancer and beyond

**Lisa H. Apken and Andrea Oeckinghaus**  
Institute of Molecular Tumor Biology, Faculty of Medicine, University of Münster, Münster, Germany  
*Corresponding author: e-mail address: oeckinga@ukmuenster.de*

## Contents

1. **The RAS family of small GTPases**  
   2
2. **RAL GTPases: An introduction**  
   5
3. **Regulation of RAL activity and function**  
   7
   - **3.1 RALGEFs and their regulation**  
     10
   - **3.2 RALGAPs and their regulation**  
     14
   - **3.3 GEF-independent activation of RAL GTPases**  
     19
   - **3.4 RAL regulation through posttranslational modifications**  
     20
4. **The RAL signaling network: Effector proteins and regulated processes**  
   26
   - **4.1 The effector protein RALBP1: Regulation of actin dynamics and endocytosis**  
     26
   - **4.2 Effector proteins SEC5 and EXO84: Regulation of exocytosis and exocyst-independent functions**  
     31
   - **4.3 Crosstalk of RAL and mTORC signaling**  
     37
   - **4.4 Regulation of phospholipase D1 activity**  
     38
   - **4.5 Control of transcription factor activities**  
     39
5. **RAL signaling in cancer**  
   41
   - **5.1 Pancreatic cancer**  
     44
   - **5.2 Lung cancer**  
     46
   - **5.3 Bladder cancer**  
     48
   - **5.4 Breast cancer**  
     49
   - **5.5 Colorectal cancer**  
     49
   - **5.6 Colitis-associated cancer**  
     50
   - **5.7 Gastric cancer**  
     51
   - **5.8 Hepatocellular carcinoma**  
     51
   - **5.9 Hematopoietic malignancies**  
     52
   - **5.10 Malignant peripheral nerve sheath tumors**  
     54
   - **5.11 Medulloblastoma**  
     54
   - **5.12 Melanoma**  
     54
   - **5.13 Oral squamous cell carcinoma**  
     55
   - **5.14 Ovarian cancer**  
     55
   - **5.15 Prostate cancer**  
     56
   - **5.16 Squamous cell carcinoma**  
     56

**International Review of Cell and Molecular Biology**  
ISSN 1937-6448  
https://doi.org/10.1016/bs.ircmb.2020.10.005  
© 2020 Elsevier Inc. All rights reserved.

ARTICLE IN PRESS

Lisa H. Apken and Andrea Oeckinghaus

6. RALopathies 57  
7. Inhibition of RAL activity 58  
8. Conclusion 62  
Acknowledgments 63  
References 63  

### Abstract

The RAL proteins RALA and RALB belong to the superfamily of small RAS-like GTPases (guanosine triphosphatases). RAL GTPases function as molecular switches in cells by cycling through GDP- and GTP-bound states, a process which is regulated by several guanine exchange factors (GEFs) and two heterodimeric GTPase activating proteins (GAPs). Since their discovery in the 1980s, RALA and RALB have been established to exert isoform-specific functions in central cellular processes such as exocytosis, endocytosis, actin organization and gene expression. Consequently, it is not surprising that an increasing number of physiological functions are discovered to be controlled by RAL, including neuronal plasticity, immune response, and glucose and lipid homeostasis. The critical importance of RAL GTPases for oncogenic RAS-driven cellular transformation and tumorigenesis still attracts most research interest. Here, RAL proteins are key drivers of cell migration, metastasis, anchorage-independent proliferation, and survival. This chapter provides an overview of normal and pathological functions of RAL GTPases and summarizes the current knowledge on the involvement of RAL in human disease as well as current therapeutic targeting strategies. In particular, molecular mechanisms that specifically control RAL activity and RAL effector usage in different scenarios are outlined, putting a spotlight on the complexity of the RAL GTPase signaling network and the emerging theme of RAS-independent regulation and relevance of RAL.

---

### 1. The RAS family of small GTPases

RAL (*Ras*-like) GTPases belong to the RAS superfamily of small GTPases whose founding members are the RAS GTPases HRAS, KRAS and NRAS (*Harvey*, *Kirsten* and *Neuroblastoma* *Rat* *sarcoma*), first identified in the early 1980s (*Chang et al.*, 1982; *DeFeo et al.*, 1981; *Ellis et al.*, 1982). It has since become evident that RAS GTPases harbor activating mutations in about 25 % of all human cancers, making them the most frequently mutated oncogenes (*Hobbs et al.*, 2016). To date, about 170 small GTPases of the RAS superfamily have been annotated in humans and subdivided into five main subfamilies based on similarities in sequence, structure and function: RHO, RAB, ARF, RAN and RAS GTPases (*Colicelli*, 2004; *Goitre et al.*, 2014; *Rojas et al.*, 2012; *Wennerberg et al.*, 2005). The RHO (*Ras* homolog) family comprises about 20 proteins that control cytoskeletal dynamics, cell migration, and cell cycle progression (*Aspenström*, 2018;

Hodge and Ridley, 2016; Jaffe and Hall, 2005). The RAB (Ras-related protein in brain) proteins represent the biggest family with over 65 members regulating intracellular vesicle transport and membrane trafficking (Pylypenko et al., 2018; Zerial and McBride, 2001; Zhen and Stenmark, 2015). Similar to RAB, the ARF (ADP-ribosylation factor) family affects vesicle formation and membrane trafficking (Donaldson and Jackson, 2011; Jackson and Bouvet, 2014; Sztul et al., 2019). The most abundant small GTPase is RAN (Ras-related nuclear protein), which orchestrates nuclear transport and mitotic spindle formation (Matchett et al., 2014; Melchior, 2001; Weis, 2003). The 39 members of the RAS subfamily, including the RAL GTPases RALA and RALB, are involved in signal transduction in a variety of different signaling pathways.

Small GTPases are hydrolytic enzymes that convert guanosine triphosphate (GTP) into guanosine diphosphate (GDP). In their GTP-bound form, small GTPases are in their “ON” state and interact with so-called effector proteins to convey a downstream signal. GTPase activating proteins (GAPs) facilitate the hydrolysis of GTP, leaving the small GTPase GDP-bound. This “OFF”-state is reversed by guanine nucleotide exchange factors (GEFs), which promote the substitution of GDP with GTP. This dual-mode conformation (GTP- or GDP-bound) of small GTPases represents the basis for their function as molecular switches, integrating extracellular stimuli to control fundamental signaling networks (Cherfils and Zeghouf, 2013).

The unifying molecular feature of all GTPases is the 20kDa GTP-binding domain, referred to as G-domain. The catalytic G-domain consists of five α-helices and six β-sheets and contains five conserved sequence motifs (G1–G5) (Fig. 1A) (Bourne et al., 1991). These sequence motifs either make direct contact with the nucleotide or an essential Mg²⁺ ion or are required to coordinate other amino acids that do so (Fig. 1B). The sequence motif G1 (GxxxxGKS/T) contains the phosphate-binding loop (P-loop), which interacts with the β- and γ-phosphates of GTP with lysine (K) being essential for nucleotide binding (Vetter, 2014). The G2 motif harbors a conserved threonine (T) that makes contact with the γ-phosphate of GTP. The aspartic acid (D) of the G3 motif (DxxG/Q/H/T) coordinates, mediated by water, the Mg²⁺ ion, which is indispensable for tight binding of the nucleotide and GTP hydrolysis. Specificity for guanine is conferred by aspartic acid (D) of the G4 motif (N/TKxD) and by alanine (A) of the G5 motif (A/C/S/TSAK/L), which forms a main chain interaction with the O6 oxygen of guanine (Bourne et al., 1991; Paduch et al., 2001; Vetter, 2014; Vetter and Wittinghofer, 2001; Wittinghofer and Vetter, 2011).

ARTICLE IN PRESS

Lisa H. Apken and Andrea Oeckinghaus

A

| G1 | P-loop |
| --- | --- |
| HRAS | MTEYKLVVVGAGGVGKSALTIQLIQNHFVDEYDPTIEDSYRKQVVIDGE 49 |
| KRAS | MTEYKLVVVGAGGVGKSALTIQLIQNHFVDEYDPTIEDSYRKQVVIDGE 49 |
| NRAS | MTEYKLVVVGAGGVGKSALTIQLIQNHFVDEYDPTIEDSYRKQVVIDGE 49 |
| RALA | MAANKPKGNSLALHKVIMVGSGGVGKSALTQFMYDEFVEDYEPTKADSYRKKVVLDGE 60 |
| RALB | MAANKSKGQSSLALHKVIMVGSGGVGKSALTQFMYDEFVEDYEPTKADSYRKKVVLDGE 60 |


:: :::::::************::: .**:::*:** *********::***


| G3 | switch II |
| --- | --- |
| HRAS | TCLLDILDQEEYSAMRDQYMRTGEGFLCVFAINNTKSFEDIHQYREQIKRVKDS-DD 108 |
| KRAS | TCLLDILDQEEYSAMRDQYMRTGEGFLCVFAINNTHYREQIKRVKDS-ED 108 |
| NRAS | TCLLDILDQEEYSAMRDQYMRTGEGFLCVFAINNSKSFADINLYREQIKRVKDS-DD 108 |
| RALA | EVQIDILDQEDYAAIRDNYFRSGEGFLCVFSITEMESFAATADFREQILRVKED-EN 119 |
| RALB | EVQIDILDQEDYAAIRDNYFRSGEGFLLVFSITEHESFTATAEFREQILRVKAEEDK 120 |


:************::*:***::***:****** **::*. : :**


| G4 | G5 |
| --- | --- |
| HRAS | VPMVLVGNKCDLAAR-TVESRQAQDLARSYGIPYIETSAKTRQGVEDAFYTLVREIRQHK 167 |
| KRAS | VPMVLVGNKCDLPSR-TVDTKQAQDLARSYGIPFIETSAKTRQRVEDAFYTLVREIRQYR 167 |
| NRAS | VPMVLVGNKCDLPTR-TVDTKQAHELAKSYGIPFIETSAKTRQGVEDAFYTLVREIRQYR 167 |
| RALA | VPFLLVGNKSDLEDKRQVSVEEAKNRAEQWNVNYVETSAKTRANVDKVFFDLMREIRARK 179 |
| RALB | IPLLVVGNKSDLEERRQVPVEEARKSAAEWGVQYVETSAKTRANVDKVFFDLMREIRTKK 180 |


:*:::*****.* * : .*:. *...:. ::******* *:..*: *:*****


| HVR | Caax |
| --- | --- |
| HRAS | LRKLNPPDESG------PGCMSCK-CVLS 189 |
| KRAS | LKKISKEE-KT------PGCVKIKKCIIM 189 |
| NRAS | MKKLNSSDDGT------QGCMGLP-CVVM 189 |
| RALA | MEDSKEKNKGKKKRSLAKRIRER-CCIL 206 |
| RALB | MSENKDNGKSKSSKN-KKSFKER-CCLL 206 |


: . . :


B

![Diagram of the "loaded spring" switch mechanism](#)

Fig. 1 Molecular features of small GTPases of the RAS family. (A) Alignment of amino acid sequences of human RAS and RAL GTPases. Sequence motifs, switch regions and the hypervariable region (HVR) are highlighted. (B) Schematic of the “loaded spring” switch mechanism. Main contacts between RAL GTPases and nucleotide are indicated. Numbering confers to human RALA. Switch I and switch II are pulled in by interaction of the main chain NH groups of T46 and G71 with the γ-phosphate (left). Release of the γ-phosphate upon GTP hydrolysis returns the switch regions to a relaxed state (right). The switch cycle is regulated by guanine nucleotide exchange factors (GEFs) and GTPase activating proteins (GAPs).

The interplay of all these elements is required to allow nucleophilic attack of a coordinated water molecule on the γ-phosphate of GTP. The domains undergoing major conformational changes upon GTP hydrolysis are the switch regions, switch I and switch II, which overlap with the G2 and G3 motifs, respectively. The switch regions keep the outbound γ-phosphate in place for the nucleophilic attack through hydrogen bonds between the main chains of threonine (T) (switch I/G2) and glycine (G) (switch II/G3) and γ-phosphate oxygens. After release of the γ-phosphate, the switch regions flip back into their relaxed state, thus unlatching the “loaded spring” (*Vetter and Wittinghofer, 2001*). The switch regions are also the sites of effector binding and their conformational change upon GTP/GDP cycling allows for sensing of the nucleotide loading state and specific interaction of effectors with the GTP-bound forms of GTPases.

A large number of small GTPases are posttranslationally modified by lipids, tethering them to different cellular membranes. Most GTPases of the RAS family contain a Caax motif following the G-domain, which is prenylated with a farnesyl or geranylgeranyl moiety (see Section 3.4.1). Distinct subcellular localization can confer specificity to signaling outcome by bringing GTPases in contact with specific effectors and can also modulate activity through controlled trafficking of GTPase regulatory proteins such as GAPs or GEFs.

---

### 2. RAL GTPases: An introduction

The two RAL GTPases RALA and RALB were discovered in cDNA library screenings for new RAS family members in 1986 and 1989, respectively (*Chardin and Tavitian, 1986, 1989*). RAL GTPases are expressed ubiquitously and conserved in evolution with single orthologs present in zebrafish (*Danio rerio*, RalA) and invertebrates such as *Drosophila melanogaster* (RalA) and *Caenorhabditis elegans* (RAL-1) (*Fig. 2*) (*Frische et al., 2007; Ghiglione et al., 2008*). Despite the presence of RAS, no RAL genes were identified in *Saccharomyces cerevisiae* or *S. pombe*.

The two RAL isoforms are highly homologous (82 % identity) and of the RAS subfamily GTPases, they share the highest similarity with the classical RAS proteins (48 % identity to total protein sequence and 55 % toward G-domain). RAL GTPases are slightly longer than RAS proteins with an N-terminal extension of 11 amino acids and an additional 5 or 6 amino in the C-terminal tail after the G-domain (*Fig. 1A*). Three-dimensional structures of RALA in its free form, bound to GDP, or complexed with its effectors SEC5 and EXO84 or the RALGEF RGL2, and RALB bound

RAL-1 Caenorhabditis elegans  MASKKASGTLPPQQQVVHKVIMVGTGGVGKSALTLQFMYDEFVEEYEPTKADSYRKKVVL 60
RalA Drosophila melanogaster ---MS---KKPTAGPALHKVIMVGSGGVGKSALTLQFMYDEFVEDYEPTKADSYRKKVVL 54
RalA Danio rerio MAAAK---PKGQNSLALHKVIMVGSGGVGKSALTLQFMYDEFVEDYEPTKADSYRKKVVL 57
RALA Homo sapiens MAANK---PKGQNSLALHKVIMVGSGGVGKSALTLQFMYDEFVEDYEPTKADSYRKKVVL 57
RALA Mus musculus MAANK---PKGQNSLALHKVIMVGSGGVGKSALTLQFMYDEFVEDYEPTKADSYRKKVVL 57
RALB Homo sapiens MAANK---SKGQSSLALHKVIMVGSGGVGKSALTLQFMYDEFVEDYEPTKADSYRKKVVL 57
RALB Mus musculus MAANK---GKSQGSLVLHKVIMVGSGGVGKSALTLQFMYDEFVEDYEPTKADSYRKKVVL 57
.:********:*****************:*************

G3 switch II

RAL-1 Caenorhabditis elegans DGEECSIDILD TAGQEDYSAIRDNY YRSGEGFICVFSILDMESFEATNEFREQILRVKNS 120
RalA Drosophila melanogaster DGEEVQIDILD TAGQEDYAAIRDNY FRSGEGFLCVFSITDDESFQATQEFREEQILRVKND 114
RalA Danio rerio DGEEVQIDILD TAGQEDYAAIRDNY FRSGEGFLCVFSITELESFAAATADFREQILRVKED 117
RALA Homo sapiens DGEEVQIDILD TAGQEDYAAIRDNY FRSGEGFLCVFSITEMESFAATADFREQILRVKED 117
RALA Mus musculus DGEEVQIDILD TAGQEDYAAIRDNY FRSGEGFLCVFSITEMESFAATADFREQILRVKED 117
RALB Homo sapiens DGEEVQIDILD TAGQEDYAAIRDNY FRSGEGFLLVFSITEHESTATAEFREQILRVKAE 117
RALB Mus musculus DGEEVQIDILD TAGQEDYAAIRDNY FRSGEGFLLVFSITEHESTATAEFREQILRVKSE 117
**** .**************:*******: ***** : *** ** :********** .

G4 G5

RAL-1 Caenorhabditis elegans DSSVPIVLVGNKGDMRDQRVVSAELCRQRAEQWGCHYVET SAKKRENVDKVFYDLMREMK 180
RalA Drosophila melanogaster -ESIPFLLVGNKCDLNDKRKVPLSECQLRAQQWAVPYVET SAKTRENVDKVFFDLMREIR 173
RalA Danio rerio -ENVPFLLVGNKSDLEDRRQVSADEAKARADQWGVCYVET SAKTRANVDKVFFDLMREIR 176
RALA Homo sapiens -ENVPFLLVGNKSDLEDKRQVSVEEAKNRAEQWNVNYVET SAKTRANVDKVFFDLMREIR 176
RALA Mus musculus -ENVPFLLVGNKSDLEDKRQVSVEEAKNRADQWNVNYVET SAKTRANVDKVFFDLMREIR 176
RALB Homo sapiens EDKIPLLVVGNKSDLLEERRQVPVEEARSKAEEWGVQYVET SAKTRANVDKVFFDLMREIR 177
RALB Mus musculus EDKIPLLVVGNKSDLLEERRQVPVEARGKAEEWGVQYVET SAKTRANVDKVFFDLMREIR 177
..:*::::**** *:.:::* * . . :*::* ******** * ************:

HVR Caax

RAL-1 Caenorhabditis elegans RR KGGSSQAQTGIDASASSGRKKRSGIKKHCTIL 213
RalA Drosophila melanogaster SR KTEDSKATSGR---AKDRCKK--RRLKCTLL 201
RalA Danio rerio AR KMEDSKEKNGK---KKRKSЛАКRIRERCCIL 206
RALA Homo sapiens AR KMEDSKEKNGK---KKRKSЛАКRIRERCCIL 206
RALA Mus musculus AR KMEDSKEKNGK---KKRKSЛАКRIRERCCIL 206
RALB Homo sapiens TK KMSNENKDNGK---KSSKN-KKSKFERCCLL 206
RALB Mus musculus AK KMSNENKDNGR---KSSKS-KKSKFERCCLL 206
:* .. .. . : :* :*

Fig. 2 Alignment of RAL amino acid sequences from different organisms. Sequence motifs, switch regions and the hypervariable region (HVR) are highlighted.

to the GTP analog GMPPNP or the RAL-binding domain of the effector RALBP1 have been solved ([Bum-Erdene et al., 2020](https://doi.org/10.1093/nar/gkaa1008); [Fenwick et al., 2009](https://doi.org/10.1074/jbc.M808878200); [Fukai et al., 2003](https://doi.org/10.1074/jbc.M307885200); [Jin et al., 2005](https://doi.org/10.1074/jbc.M505287200); [Mott and Owen, 2010](https://doi.org/10.1074/jbc.M110.149886); [Nicely et al., 2004](https://doi.org/10.1074/jbc.M405888200); [Popovic et al., 2016](https://doi.org/10.1074/jbc.M116.739887)). These studies confirm a very similar tertiary fold between RALA and RALB and reveal that different effectors differentially engage the two switch regions for interaction (e.g., SEC5 binds predominantly to switch I, while EXO84 binds both switch regions). Amino acids D71 and I75 were identified as residues critical for conferring specificity for RALGEFs ([Popovic et al., 2016](https://doi.org/10.1074/jbc.M116.739887)). The sequences of both switch regions are identical between RALA and RALB. In line with this homology, RALA and RALB interact with the same set of effector proteins with similar affinities ([Fenwick et al., 2009](https://doi.org/10.1074/jbc.M808878200)). Nevertheless, distinct functions have been attributed to the isoforms *in vivo*. This specificity is at least in part conferred

by differences in the C-terminal tail, also called hypervariable region (HVR) (Fig. 1A), which can be targeted by posttranslational modifications controlling localization and activity (see also Section 3.4).

Both RAL GTPases have been genetically ablated in mice (Peschard et al., 2012). While *RalB* knockout mice are viable without an obvious phenotype, *RalA* deficient animals suffer from exencephaly and are embryonically lethal, identifying a role of RALA in neural tube closure. The exencephaly phenotype is aggravated by deletion of both RAL GTPases and *RalA/B* double-deficient embryos do not live past gastrulation. This suggests differences but also partially overlapping functions of RAL GTPases *in vivo*. The important role of RAL in development is reinforced by knockout models in invertebrates. Disruption of *ral-1* in *C. elegans* disturbed exocytosis, tubulogenesis and cell polarity revealing an essential role for fertility (Armenti et al., 2014) and in cooperation with a *rap-1* mutant for viability (Frische et al., 2007). Furthermore, RalA signaling is necessary for *Drosophila* development where overexpression experiments suggest a conserved function in dorsal closure during embryogenesis (Holly et al., 2015; Sawamoto et al., 1999).

In mammalian cells, RALA and RALB have been shown to be involved in diverse cellular processes including exocytosis, endocytosis, cytokinesis, actin cytoskeleton organization, autophagy, second messenger production, gene transcription, anchorage-independent proliferation, and survival (Bodemann and White, 2008; Feig, 2003; Gentry et al., 2014; Shirakawa and Horiuchi, 2015; Yan and Theodorescu, 2018). The role attracting most attention is their function downstream of oncogenic RAS mutants and importance for cellular transformation. Here, RAL GTPases have been implicated in proliferation, survival, invasion, and metastasis of different human tumors, including pancreatic, lung, colon, prostate, bladder, and skin cancer (Yan and Theodorescu, 2018).

---

### 3. Regulation of RAL activity and function

RAL GTPases possess low hydrolytic activity. They rely on RALGEFs to promote their transition from a GDP– to a GTP-bound state and on RALGAPs to accelerate GTP hydrolysis and thus terminate effector interactions (Fig. 3). Eight different RALGEFs and two RALGAP protein complexes have been identified to date.

RAS-dependent activation of RAL is often conveyed through the RALGDS family of RALGEFs, which are *bona fide* RAS effectors (Ferro and Trabalzini, 2010). However, RAL activity can also be elicited by signal

ARTICLE IN PRESS

Lisa H. Apken and Andrea Oeckinghaus

![Diagram](attachment:diagram.png)

Fig. 3 RALGEF- and RALGAP-dependent regulation of RAL activity. An extracellular stimulus such as epidermal growth factor (EGF) or insulin activates RAS GTPases. GTP-bound RAS interacts with RALGEFs harboring a Ras-association (RA) domain. RALGEF activity can also be further influenced by PI3K (Phosphoinositid-3-kinase) in a complementary activation mechanism. RALGEFs with a pleckstrin homology (PH) domain are activated by largely unknown signals. RALGEFs promote the substitution of GDP to GTP, thus increasing the level of RAL-GTP and promoting effector engagement. The catalytic activity of RAL is accelerated by RALGAPs, which consist of α- and β-subunits. RALGAP complexes thus return RAL GTPases to their GDP-bound "OFF" state. The activity of RALGAPs is modulated by interaction with κB-RAS proteins and by AKT2-mediated phosphorylation and subsequent association with 14-3-3 proteins.

transduction mechanisms that do not involve RAS. These mechanisms include both GEF-dependent and -independent processes. The RALGPS family of GEFs is generally activated independent of RAS, but also RALGDS can be engaged in a RAS-independent fashion (Bhattacharya et al., 2002; Ceriani et al., 2007; De Bruyn et al., 2000; Ferro and Trabalzini, 2010; Rebhun et al., 2000; Rondaij et al., 2008). Besides, activation of RAL GTPases can be achieved by inactivation of RALGAP complexes, e.g., through

Ral GTPase signaling

AKT2-dependent phosphorylation (Chen et al., 2011c, 2014; Gridley et al., 2005, 2006; Leto et al., 2013) (Fig. 3). An additional level of regulation is imposed through posttranslational modifications of RAL GTPases, including lipidation, phosphorylation, and ubiquitination and the (resulting) control of subcellular localization (see Section 3.4). Remarkably, much remains to be learned about which factors mediate RAL activation in response to the various RAL-inducing stimuli.

A

RALGDS


412 REM 386 CDC25 798 RA 914 aa
249 648 885


RGL1


65 REM 232 CDC25 648 RA 768 aa
196 501 735


RGL2


88 REM 243 CDC25 648 RA 777 aa
212 513 735


RGL3


65 REM 247 CDC25 613 RA 710 aa
200 505 700


RGL4


219 CDC25 473 aa
471


RALGPS1


50 CDC25 PXXP PH 557 aa
298 431 543


RALGPS2


49 CDC25 PXXP PH 583 aa
287 457 569


RCC2


103 168 221 273 348 403 448 522 aa
165 219 271 347 401 447 501


B

REM
↓
CDC25 RA


PI3K


PDK1
REM CDC25 RA


PKC


RAS
GTP


TIPE2


Fig. 4 Domain structure and regulation of RALGEFs. (A) Domain architecture of human RALGEF proteins. All RALGEFs, except RCC2, contain a CDC25 homology domain, responsible for their catalytic activity. The RALGDS family GEFs, RALGDS, RGL1, RGL2, and RGL3 harbor a Ras exchanger motif (REM), which stabilizes the CDC25 domain and is thus also critical for catalytic GEF activity. These four GEFs also comprise a RAS-association (RA) domain. The RALGPS family GEFs contain a Pleckstrin homology (PH) domain instead of REM and RA domains and a PxxP motif. RCC2 contains seven RCC1 motifs which constitute an RCC1-like domain. (B) Autoinhibitory mechanism of RALGDS activation. Activity of the CDC25 domain is inhibited by interaction with the N-terminal REM domain. Binding of RAS-GTP to the RA domain recruits PI3K which in turn activates PDK1. The interaction of PDK1 with the REM relieves the inhibitory effect on the CDC25 domain. This process can be counteracted through RALGDS phosphorylation by protein kinase C (PKC), which prevents release of the inhibitory REM/CDC25 interaction. In addition, TIPE2 can impede activation of RALGDS by competitively binding to the RA domain, thus prohibiting an interaction with GTP-bound RAS.

ARTICLE IN PRESS

Lisa H. Apken and Andrea Oeckinghaus

## 3.1 RALGEFs and their regulation

The first RALGEF, **RAL** guanine nucleotide dissociation stimulator (RALGDS), was identified in 1993 (Albright et al., 1993). Besides RALGDS, seven additional multidomain GEF proteins (RGL1, RGL2, RGL3, RGL4, RALGPS1, RALGPS2, and RCC2) have since been discovered to act on RALA and RALB in mammalian cells (Fig. 4A). *C. elegans* and *Drosophila* express a single RALGDS-like RALGEF. All identified RALGEFs, except RCC2, exert their catalytic function via a CDC25 homology domain, which is similar to the catalytic domain of RAS- and RAPGEFs (van Dam et al., 2009). Since the majority of studies addressing RALGEF function have so far focused on RALGDS, our understanding of distinct GEF usage and GEF-conveyed specificity to RAL-dependent downstream events remains fragmentary (for review, see Ferro and Trabalzini, 2010). In the following we summarize current knowledge on the regulation and functions of the individual RALGEFs.

### 3.1.1 RALGDS

In addition to the CDC25 domain, RALGDS contains an N-terminal **RAS** exchanger motif (REM), which stabilizes the CDC25 domain and regulates its activity (Quilliam et al., 2002). Via a C-terminal **Ras** association (RA) domain RALGDS associates with GTP-bound RAS GTPases. This interaction recruits RALGDS to the plasma membrane, delivering it to RAL GTPases in response to growth factor receptor stimulation by epidermal growth factor (EGF) or insulin (Fig. 3) (Kishida et al., 1997; Matsubara et al., 1999; Wolthuis et al., 1998b). Whether RALGDS also interacts with other GTP-loaded GTPases, such as RAP1A and RRAS *in vivo*, as has been suggested in *Drosophila* and by *in vitro* assays, remains a matter of debate (Kikuchi et al., 1994; Kishida et al., 1997; M’Rabet et al., 1999; Mirey et al., 2003; Peterson et al., 1996; Takaya et al., 2007; Urano et al., 1996; Wolthuis et al., 1996, 1998a; Zwartkruis et al., 1998). In addition to membrane recruitment via the RA domain, a complementary activation mechanism for RALGDS has been described, in which PI3K (Phosphoinositid-3-kinase), activated through RAS-GTP, promotes the binding of its target PDK1 (Phosphoinositide-dependent kinase-1) to the N-terminus of RALGDS. The association of PDK1 relieves an inhibitory interaction of the N-terminal region of RALGDS with its catalytic domain without requirement of PDK1 kinase function (Fig. 4B) (Rusanescu et al., 2001; Tian et al., 2002; Yoshizaki et al., 2007). This finding highlights the potential for crosstalk between PI3K and RALGDS/RAL signaling (Urano et al.,

Ral GTPase signaling

1996; White et al., 1996; Wolthuis et al., 1996). RALGDS GEF activity can be attenuated by PKC (protein kinase C)-dependent phosphorylation of the N-terminal region, since this phosphorylation hinders release of the interaction between N-terminus and the catalytic domain (Fig. 4B) (Rusanescu et al., 2001).

RALGDS can also be activated by RAS-independent mechanisms. Stimulation of cells with the chemotactic formyl-Met-Leu-Phe (fMLP) peptide causes translocation of RALGDS:β-arrestin complexes to the plasma membrane and subsequent RALGDS dissociation. This allows RALGDS to activate RAL GTPases, which then mediate cytoskeletal reorganization to support cell migration (Bhattacharya et al., 2002). This mechanism is also used after stimulation of the angiotensin II type I receptor (AT1R) to promote RALA-dependent activation of phospholipase-δ1 (PLC-δ1) (Godin et al., 2010). Furthermore, calmodulin binding to RALGDS has been demonstrated to be required for thrombin-induced exocytosis of Weibel-Palade bodies (WPBs), endothelial cell-specific storage organelles that contain hormones, chemokines or enzymes (Rondaij et al., 2008). However, the underlying molecular mechanisms remain elusive.

Besides RAS-independent activation, RALGDS has also been described to control RAL-independent processes. RALGDS bridges PDK1 to its target AKT, thus promoting AKT phosphorylation and AKT-dependent cell proliferation in response to EGF or insulin stimulation (Hao et al., 2008). In addition, the anti-inflammatory protein TIPE2 (TNFAIP8L2, tumor necrosis factor-α-induced protein-8L2) has been shown to interact with the RA of RALGDS, preventing RAS association (Gus-Brautbar et al., 2012). Depletion of TIPE2 results in activation of both RAL and AKT, supporting the finding that both of these pathways are controlled by RALGDS, and in enhanced migration and resistance to cell death. Importantly, this finding suggests that RALGDS might represent a hub to integrate inflammatory and proliferative signaling cues. While not experimentally addressed, it is tempting to speculate that RALGDS-driven AKT activation might contribute to AKT-dependent phosphorylation of RALGAPα subunits, an event that inhibits GAP activity (see Section 3.2.2) and that would thus further boost RAL activation.

RALGDS so far remains the only RALGEF whose *in vivo* role has been studied by generation of a conventional knockout mouse model. *Ralgds* knockout mice are viable with no obvious phenotype, demonstrating that RALGDS is not crucial for mouse development (González-García et al., 2005). RALGDS deficiency however leads to strong reduction of tumor incidence and malignant progression when mice are topically treated with

DMBA/TPA (7,12-dimethylbenzanthracene/12-O-tetradecanoylphorbol-13-acetate) revealing a critical involvement of RALGDS in skin carcinogenesis (González-García et al., 2005).

### 3.1.2 RGL1, RGL2/Rlf and RGL3

Three paralogs of RALGDS, RALGDS-like 1, 2, and 3 (RGL1, RGL2/Rlf, RGL3) have been identified to comprise the same domain organization (REM/CDC25/RA domains) and high sequence identity with RALGDS. Like RALGDS, RGLs represent RAS effectors as they interact with GTP-bound classical RAS GTPases via their RA domains (Kikuchi et al., 1994; Peterson et al., 1996; Shao and Andres, 2000; Wolthuis et al., 1996). RGL1 and RGL2 have been characterized as the critical GEFs triggering RALB activity to drive cell invasion of HEK cells upon expression of mutated HRAS. Furthermore, they have been shown to be required for autophagy in HeLa cells (Singh et al., 2019; Zago et al., 2018). Expression of membrane-targeted and thus active RGL2 can drive cellular growth and silencing of RGL2 inhibits colony formation and invasion of pancreatic cancer cells, suggesting an important role in this tumor entity (Vigil et al., 2010). RGL3 has been identified as an effector of both RAS and RIT GTPases (Shao and Andres, 2000). As constitutive activation of RIT can also trigger cellular transformation, RGL3-dependent RAL activation may be relevant in this context.

### 3.1.3 RGR/RGL4

The RALGEF RGR (RALGDS related), also called RGL4, has been identified as part of a fusion oncogene, comprising the rabbit ortholog of RAD23 and an N-terminally deleted RGR, in DMBA-induced rabbit squamous cell carcinoma. This fusion protein, termed RCS, contains only the catalytic CDC25 homology domain of RGR (D'Adamo et al., 1997). A follow-up study presented evidence that overexpression of RGR triggers cellular transformation, induces activation of MAPK (Mitogen-activated protein kinase) and JNK (c-Jun N-terminal kinase) signaling and enhances not only RAL- but also RAS-GTP levels (Hernandez-Muñoz et al., 2000). This raise of RAS-GTP levels differs from what has been seen for other RALGEFs, which present high specificity for RAL GTPases, and is not observed in *in vitro* assays (D'Adamo et al., 1997). Nevertheless, the RGR part is likely critical for conferring the oncogenic function of RCS. But only a dominant-negative form of RAS, but not RHO or RAL, inhibited RGR-induced transformation, suggesting that RAS activation is the key process and questioning the relevance of RAL GTPases downstream of RCS.
A tumorigenic role of RGR was further supported by the generation of different transgenic mouse lines. Expression of RGR in the brain, eye and muscle driven by an MSV promoter results in frequent cataracts of the eye, associated Harderian gland adenomas and less frequently fibrosarcomas in hind limbs or tail (Jiménez et al., 2004). Transgenic expression of $RGR$ in T cells leads to the development of thymic lymphomas and interestingly, mutations in human $RGR$ have been linked to T-cell malignancies (Jiménez et al., 2004; Leonardi et al., 2002). However, the relevant contribution of enhanced RAL signaling to tumorigenesis and the function of RGR in normal RAL activation remain unclear.

### 3.1.4 RALGPS1 and RALGPS2

A second group of RALGEFs comprises the RALGEF with PH domain and SH3-binding motif 1 and 2 proteins (RALGPS1, RALGPS2) (Ceriani et al., 2007; De Bruyn et al., 2000; Rebhun et al., 2000). RALGPS1 and RALGPS2 lack the RA and REM domains and are thus activated in a RAS-independent fashion. Instead, RALGPS1 and RALGPS2 contain a PxxP motif and a pleckstrin homology (PH) domain, which can bind to phospholipids, C-terminally to the catalytic CDC25 homology domain. The PH domain delivers RALGPS proteins to the plasma membrane, which is necessary for GEF activity (De Bruyn et al., 2000; Rebhun et al., 2000). The PH domain of RALGPS2 hereby preferentially binds the phospholipid phosphatidylinositol-(4,5)-bisphosphate (PI4,5P₂) and to a lesser extent phosphatidylinositol-(3,4)-bisphosphate (PI3,4P₂), phosphatidylinositol (3,5)-bisphosphate (PI3,5P₂) and phosphatidylinositol (3,4,5)-triphosphate (PI3,4,5P₃) (Ceriani et al., 2007). PI3K phosphorylates PI4,5P₂ to yield PI3,4,5P₃. Inhibitor studies addressing a potential role of PI3K in RALGPS-induced RAL activation have so far delivered conflicting results (Ceriani et al., 2007; Rebhun et al., 2000), but suggest that PI3K may be involved in regulation of RALGPS localization (Ceriani et al., 2007). Whether other kinases that influence the cellular phospholipid composition might play a role in tuning RALGPS activity remains to be investigated.

Regarding cellular functions, RALGPS2 has been shown to be essential for growth of bladder cancer cells. Here, RALGPS2 interacts with the transmembrane MHC class III protein LST1 (leukocyte specific transcript 1) and RALA to drive the generation of tunneling nanotubes, thin actin-rich membrane protrusions, in a SEC5-dependent manner (D'Aloia et al., 2018). The PxxP motif mediates interactions with the SH3 (Src homology 3) domains of GRB2 (growth factor receptor bound protein 2), NCK (non-catalytic region of tyrosine kinase adaptor protein 1), PLCγ (phospholipase C gamma),and SRC (SRC Proto-Oncogene/ Non-Receptor Tyrosine Kinase) *in vitro* ([Ceriani et al., 2007](https://doi.org/10.1038/sj.onc.1209654); [Rebhun et al., 2000](https://doi.org/10.1074/jbc.M001894200)). While the activity of the RASGEF SOS1 (Son of Sevenless homolog 1), which contains domains similar to the PH and PxxP domains of RALGPs, is regulated by interaction with GRB2, it remains unclear whether this is also the case for RALGPs ([Li et al., 1993](https://doi.org/10.1073/pnas.90.11.5075); [Olivier et al., 1993](https://doi.org/10.1073/pnas.90.11.5075); [Rozakis-Adcock et al., 1993](https://doi.org/10.1073/pnas.90.11.5075)). Resulting cellular effects of the other described interactions remain elusive as well.

### 3.1.5 RCC2

The most recently discovered RALGEF is RCC2 (regulator of chromatin condensation 2), a protein with known involvement in mitosis ([Papini et al., 2015](https://doi.org/10.1038/ncomms7886)). RCC2 is related to RCC1, a GEF for the small GTPase RAN. RCC2 thus differs from the other RALGEFs in respect to domain architecture as it does not contain a CDC25 domain, but a large RCC1-like domain (RLD) spanning almost the complete protein, and an N-terminal nuclear localization sequence. RCC2 activates only RALA, but not RALB, and depletion of RCC2 or RALA perturbs spindle assembly and alters levels of the CPC (chromosomal passenger complex) at centromers in HeLa cells.

A growing number of studies establish an oncogenic role of RCC2. Increased levels of RCC2 are present in lung adenocarcinoma, colorectal, breast and ovarian cancer and associated with poor patient outcome ([Bruun et al., 2015](https://doi.org/10.1038/onc.2015.10); [Chen et al., 2019b](https://doi.org/10.1016/j.jprot.2019.01.001); [Gong et al., 2019](https://doi.org/10.1016/j.jprot.2019.01.001); [Pang et al., 2017](https://doi.org/10.1016/j.jprot.2017.01.001); [Wang et al., 2020](https://doi.org/10.1016/j.jprot.2020.01.001)). In this context, RCC2 was frequently linked to regulation of cell proliferation, migration, invasion, and epithelial–mesenchymal transition (EMT) induction. However, while RALA knockdown could reverse the RCC2-induced effects on ovarian cell proliferation, a potential involvement of RAL GTPases in the observed phenotypes has not been addressed elsewhere ([Gong et al., 2019](https://doi.org/10.1016/j.jprot.2019.01.001)). Rather RCC2 has been proposed to control JNK activity in lung adenocarcinoma and drive EMT of breast cancer cells through activation of the WNT pathway ([Gong et al., 2019](https://doi.org/10.1016/j.jprot.2019.01.001); [Pang et al., 2017](https://doi.org/10.1016/j.jprot.2017.01.001)). Furthermore, induction of apoptosis by RCC2 knockdown in HeLa, MDA-MB-231 breast cancer and NCI-H23 lung cancer cells could be rescued by constitutively active RAC1 ([Wu et al., 2018](https://doi.org/10.1016/j.jprot.2018.01.001)). In line, RCC2 has been reported to directly interact with RAC1 and regulate its activity, suggesting that RAC1 might be a direct target of RCC2 in addition to RALA ([Song et al., 2018](https://doi.org/10.1016/j.jprot.2018.01.001)).

### 3.2 RALGAPs and their regulation

In 1991, RAL-specific GAP activity was reported in the cytosolic fraction of brain and testis lysates ([Emkey et al., 1991](https://doi.org/10.1074/jbc.266.10.6348)). Purification via gel filtration indicated a protein of relative large size or a protein complex, a fact that was
RALGAP1
RALGAPA1 —— T735 —— 1149 —— GAP —— 2036 aa
RALGAPβ —— 1392 —— 1494 aa

RALGAP2
RALGAPA2 —— S486 S696 T715 —— 1149 —— GAP —— 1873 aa
RALGAPβ —— 1392 —— 1494 aa

TSC
TSC2 —— S939 S981 S1210 —— 1531 —— GAP —— 1807 aa
TSC1 —— 1164 aa

RAP1GAP
GAP —— 663 aa
GAP —— 663 aa

Fig. 5 Domain architecture of human RAPGAP domain-containing GAP complexes. The two RALGAP complexes consist of a catalytic α-subunit (α1 or α2) and a common regulatory β-subunit. The catalytic RAPGAP domain is indicated in red (GAP) and the cryptic RAPGAP domain located in the β-subunit is marked in white. The heterodimeric TSC complex comprises the catalytic TSC2 subunit and TSC1, which fulfills a regulatory function. Identified AKT-dependent phosphorylation sites in RALGAPA1, RALGAPA2, and TSC2 are indicated. The RAP1GAP complex is a homodimer.

confirmed upon molecular identification of the two RALGAP complexes by different groups more than 10 years later (Chen et al., 2011b; Gridley et al., 2005, 2006; Shirakawa et al., 2009). RALGAP1 and RALGAP2 are heterodimers that consist of a common regulatory RALGAPβ (p170, RGC1, KIAA1219) subunit and one of the two related α-subunits RALGAPA1 (p240, GARNL1, TULIP1, GRIBE, KIAA0884) or RALGAPA2 (p220, AS250, RGC2, C20orf74, KIAA1272) (Fig. 5). RALGAPs have been shown to specifically accelerate GTP hydrolysis by RALA and RALB, but not other small GTPases such as HRAS, RAP1, RHEB, RHOA, RAB27A, or RAN (Chen et al., 2011c; Shirakawa et al., 2009).

### 3.2.1 Complex architecture and catalytic GAP mechanism
The RALGAPA subunits harbor the actual catalytic activity of the complexes, conveyed by a GAP domain located in their C-termini. This domain belongs to the family of RAPGAP domains, sharing sequence similarity with

the GAP domains of two other RAS family GAPs: RAP1GAP, which acts on the small GTPase RAP1 (Rubinfeld et al., 1991), and TSC2 (tuberous sclerosis complex 2/tuberin), the cognate GAP for RHEB (Ras homolog enriched in brain) (Gridley et al., 2005, 2006; Inoki et al., 2003; Rubinfeld et al., 1992; Shirakawa et al., 2009). These three GAPs rely on an “asparagine thumb” catalytic mechanism in contrast to other RASGAPs using an “arginine finger” (Daumke et al., 2004; Li et al., 2004; Shirakawa et al., 2009). In classical RASGAPs, the GAP stabilizes the position of a glutamine located in the switch II region of the respective GTPase (e.g., Q61 in KRAS), which is catalytically essential as it coordinates the attacking water molecule. Second, the GAP positions an arginine (“arginine finger”) into the phosphate-binding site, neutralizing negative charge at the γ-phosphate and stabilizing the transition state (Scheffzek et al., 1997). Instead of a conserved arginine, the RAPGAP domains have an asparagine as catalytic residue (N1903 in RALGAPα1 and N1742 in RALGAPα2). It coordinates and stabilizes the water molecule for GTP hydrolysis, thus mimicking the function of the GTPase glutamine in the RASGAP-mediated hydrolysis mechanism. In addition, RAL, RAP and RHEB GTPases are proposed to utilize a conserved tyrosine (Y43 in RAL) for positioning of the GTP and enabling insertion of the catalytic “asparagine thumb” (Daumke et al., 2004; Hansmann et al., 2020; Scrima et al., 2008).

While RAP1GAP functions as a homodimer, the RALGAP and TSC complexes represent functional heterodimers, where the non-catalytic subunits TSC1 and RALGAPβ are indispensable for catalytic activity. But besides fulfilling seemingly similar functions, TSC1 and RALGAPβ share no sequence homology or common domain mapping (Chen et al., 2011b; Shirakawa et al., 2009). While no protein domains are annotated in TSC1, a cryptic GAP domain is found in RALGAPβ (van Dam et al., 2011b). However, the existent sequence alterations in the catalytic loop region make it unlikely that this domain has catalytic function (Shirakawa et al., 2009). Its presence likely reflects that the RALGAP subunits presumably originated from a single gene, with the gene product forming a homodimeric complex, and that the heterodimeric complex emerged after gene duplication (van Dam et al., 2011b). The molecular function of the regulatory RALGAPβ subunit is not completely understood. But the fact that RALGAPβ is required for GAP activity in *in vitro* assays using recombinant proteins strongly suggests that RALGAPβ is involved in mediating or stabilizing a complex architecture that allows for efficient GAP function or alternatively in increasing binding affinity toward RAL GTPases

(R Shirakawa et al., 2009). The definite answer to this question likely awaits structural insight into RALGAP assembly. While structures of protein domains of TSC1 and TSC2 have been solved (Gai et al., 2016; Hansmann et al., 2020; Qin et al., 2016; Sun et al., 2013; Zech et al., 2016), no structures of RALGAP subunits or subassemblies have been published to date. The only available information concerning complex architecture comes from a domain mapping study demonstrating that the N- and C-terminal parts (aa 1–589 and 1755–2036) of RALGAPα1 and different C-terminal regions of RALGAPβ (aa 474–end) participate in complex formation (Chen et al., 2019a). Despite the fact that evidence points to multiple intra-complex contacts between the subunits, a RALGAPβ-derived peptide deduced from the identified minimal binding region (aa 1121–1158) was shown to disturb RALGAP complex assembly and thus promote muscle insulin sensitivity via enhanced RAL activity (Chen et al., 2019a). The TSC subunits interact via the N-terminal region of TSC2 (aa 1–400) and the C-terminus of TSC1 (aa 721–997) (Hodges et al., 2001; Santiago Lima et al., 2014; Van Slegtenhorst et al., 1998; Zech et al., 2016) and the RAP1GAP subunits associate via N-terminal dimerization domains, which additionally requires binding of a stabilizing C-terminal helix (Daumke et al., 2004). Thus, it remains an open question how similar the architecture, functionality and regulation of RALGAP and TSC complexes really are. Interestingly, *C. elegans* does not harbor TSC1 or TSC2 genes (van Dam et al., 2011a) and the RALGAP ortholog HGAP (heterodimeric GTPase activating protein) consisting of HGAP1 and HGAP2 takes over the role of a universal GAP to control RAL and RHEB signaling (Reiner and Lundquist, 2018). The mammalian RALGAP however, acts specifically on RAL GTPases with no direct cross-regulation of RHEB (Emkey et al., 1991; Shirakawa et al., 2009).

The two mammalian RALGAPα subunits share 55 % sequence identity with a highly conserved GAP domain and stretches of higher conservation in the N-terminal and central regions of the proteins. In vitro, RALGAPα1 and α2 both show activity toward RALA and RALB (Chen et al., 2011b; Shirakawa et al., 2009). However, the RALGAPα subunits differ in their expression pattern. While RALGAPα1 appears to be the major subunit in brain and skeletal muscle, RALGAPα2 shows high expression in liver, lung, colon, and adipose tissue. Other organs such as thymus, kidney, pancreas, spleen, and testis express comparable levels of both α-subunits (Chen et al., 2014; Gridley et al., 2005, 2006; Shirakawa et al., 2009). Thus, tissue specificity may be conferred by this differential expression pattern.

Both, RALGAPα1 and RALGAPα2 have been shown to be targeted by AKT-mediated phosphorylation, which hinders efficient promotion of RAL-driven GTP hydrolysis (Chen et al., 2011b, 2014; Gridley et al., 2006). Beyond that, it remains unclear whether there are regulatory or functional differences between the two α-subunits.

### 3.2.2 Regulation of RALGAP activity

Little is known about regulatory mechanisms governing RALGAP activity. Like RAP1GAP and TSC, RALGAP is subject to phosphorylation. RALGAPα2 has been identified as a substrate of AKT2, but not AKT1, upon insulin stimulation in 3T3-L1 adipocytes with S486, S696 and T715 serving as phosphorylation sites (Fig. 5) (Chen et al., 2011b; Gridley et al., 2005, 2006). Phosphorylation of RALGAPα2 results in T715-dependent association of 14-3-3 proteins, reduced RALGAP activity and thus elevated levels of RAL-GTP in cells (Chen et al., 2011b; Leto et al., 2013). RALGAPα1 was reported to be phosphorylated at T735 by AKT2 in response to insulin stimulation and upon muscle contraction, which also led to association with 14-3-3 proteins, increased RALA activity and stimulation of translocation of the glucose transporter GLUT4 in muscle cells (Chen et al., 2014). Thus, relieving the negative regulation through RALGAP complexes plays a role in signal-induced RAL activation, demonstrating that upstream signals employ both GEFs and GAPs to convey GTPase activation and respective downstream signaling (Fig. 3). Both, phosphorylation and 14-3-3 binding of RALGAPα subunits does not alter their catalytic activity *in vitro*, suggesting that these processes might rather affect subcellular localization and/or recruitment of RALGAP to RAL-containing membrane compartments. A similar scenario has been described for TSC2, which is phosphorylated at S939, S981 and S1210 by AKT in response to growth signals, triggering 14-3-3 binding, translocation to the cytosol and consequent loss of inhibition of RHEB (Cai et al., 2006; DeYoung et al., 2008; Inoki et al., 2002, 2003; Li et al., 2002). However, little is known about the exact subcellular localization of RALGAP complexes. In cell fractionation experiments with 3T3-L1 adipocytes, RALGAP2 was detected in the cytosol and a low density microsome fraction consisting primarily of small vesicles from the endosomal system, cytoskeletal components, and elements of the Golgi apparatus and endoplasmic reticulum (Gridley et al., 2005, 2006). But a potential regulation of RALGAP via control of (trans)localization has not yet been addressed.

Besides phosphorylation, binding of the κB-RAS (NF-κ-B inhibitor-interacting RAS-like protein) proteins has been shown to affect RALGAP

activity (Fig. 3). κB-RAS1 and κB-RAS2 are small GTPases of the RAS subfamily, that harbor unique features. They lack the C-terminal Caax motif necessary for membrane attachment and seem to be constitutively GTP-bound, likely due to amino acid substitutions at positions 12, 13 and 61 (Fenwick et al., 2000). κB-RAS proteins have originally been identified as interactors of the NF-κB inhibitor protein IkBβ and have since then been described as negative regulators of NF-κB activity with the underlying mechanisms remaining a matter of debate (Chen et al., 2003, 2004; Fenwick et al., 2000; Oeckinghaus et al., 2014; Postler and Ghosh, 2015; Tago et al., 2010). κB-RAS proteins are also constitutively associated with RALGAP complexes, with binding occurring via the N-terminal 150 amino acids of RALGAPα1 or RALGAPα2. κB-RAS association with RALGAP is dependent on GTP binding and required to uphold RALGAP activity. Genetic depletion of κB-RAS GTPases results in increased RALA- and RALB-GTP levels and consequent enhancement of anchorage-independent proliferation and invasion of transformed cells (Beel et al., 2020; Oeckinghaus et al., 2014). Whether κB-RAS binding has direct effects on catalytic activity or whether it might be involved in appropriate localization of RALGAP complexes remains to be determined. Potential parallels to TSC are hereby conceivable as the RAG (Ras-related GTP-binding protein) GTPases are involved in TSC recruitment to the lysosome upon amino acid removal where it acts on the small GTPase RHEB to shut off mTORC1 (mammalian target of rapamycin complex 1) signaling (Demetriades et al., 2014). As the GAP domain comprises only about 8 % of the complete protein sequence of RALGAP complexes, it is likely that additional regulatory mechanisms converging on RALGAP will be identified in the future.

## 3.3 GEF-independent activation of RAL GTPases

GEF-independent activation of RAL GTPases occurs in response to increased levels of intracellular Ca²⁺ and RAL GTPases have been implied to take part in Ca²⁺-dependent intracellular signaling. RAL GTPases bind calmodulin in a Ca²⁺-dependent manner via a C-terminal calmodulin binding domain (Sidhu et al., 2005a; Wang et al., 1997). Calmodulin binding removes RAL GTPases from membranes, e.g., in synapses, independent of their nucleotide-binding state (Park et al., 1999; Wang and Roufogalis, 1999). In addition, Ca²⁺/calmodulin binding has been suggested to directly stimulate GTP binding and hydrolysis activity through unknown mechanisms (Park, 2001; Wang and Roufogalis, 1999). RAL activity can be elicited by the

ionophore ionomycin and various platelet agonists, including α-thrombin in a Ca²⁺-dependent manner (Clough et al., 2002; Wolthuis et al., 1998a). Also LPA, IGF-1 and EGF stimulation-elicted RAL activation is dampened by a phospholipase C inhibitor, suggesting that these stimuli might engage both RAS-dependent as well as RAS-independent pathways to control RAL GTPases (Hofer et al., 1998; Suzuki et al., 2000; Uegaki et al., 2019). Furthermore, direct activation of RAL through nitric oxide has been reported (Mittar et al., 2004).

### 3.4 RAL regulation through posttranslational modifications

RAL GTPases can be posttranslationally modified by lipidation, phosphorylation and ubiquitination (Fig. 6). These modifications can control subcellular localization, activity and effector usage and thus overlay an additional regulatory network upon the classical GDP/GTP cycle.

#### 3.4.1 Lipidation of RAL GTPases

RAL GTPases localize not only to the plasma membrane but are also found at intracellular vesicles, such as dense granules (Mark et al., 1996) or endosomes (Chen et al., 2006), and mitochondria (Kashatus et al., 2011; Pollock et al., 2019). Like many RAS family members, RALA and RALB terminate in a Caax-like motif: CCIL and CCLL, respectively. This tetrapeptide motif is targeted for posttranslational addition of a geranylgeranyl moiety (Kinsella et al., 1991), which leads to membrane anchoring of RALA and RALB (Fig. 6A). The last L hereby conveys specificity for the cytosolic geranylgeranyltransferase type I (GGTase I) adding the 20-carbon isoprenoid geranylgeranyl to the first C of the tetrapeptide (Casey et al., 1991; Falsetti et al., 2007; Moores et al., 1991; Reid et al., 2004; Yokoyama et al., 1991). Subsequently, the remaining three amino acids (CIL/CLL) are endoproteolytically cleaved by the protease RCE1 (Ras converting enzyme 1), which is associated with the endoplasmic reticulum (Boyartchuk et al., 1997; Kim et al., 1999; Otto et al., 1999). Here, the geranylgeranyl-carrying cysteine is methylated by ICMT (isoprenyl cysteine carboxyl methyltransferase) (Leung et al., 2007). Interestingly, while both RAL GTPases are modified by this series of posttranslational modifications, they differ in their responses to loss of different steps of the modification process (Gentry et al., 2015). RCE1 is necessary for association of both RALA and RALB with the plasma membrane, but not endosomes, and RCE1 loss leads to enhanced RAL activity. Whether altered interactions with RALGEFs and/or RALGAPs underlie the latter phenomenon remains to be investigated. In contrast, ICMT

Fig. 6 RAL regulation through posttranslational modifications. (A) Regulation of RAL membrane anchoring. RALA and RALB are geranylgeranylated by geranylgeranyltransferase type I (GGTase-I) on the first cysteine of their Caax motives. At the endoplasmic reticulum RAL is then processed by RAS converting enzyme 1 (RCE1), which cleaves the three C-terminal amino acids, and isoprenyl cysteine carboxyl methyltransferase (ICMT), which methylates the now free, isoprenylated cysteine residue. This process facilitates membrane anchoring of both RALA and RALB. RAL GTPases have also been described to be modified by palmitoylation. Palmitoylation of CCxL motif-containing proteins is carried out by Golgi-associated protein acetyltransferase (PAT) and can be reversed by acylprotein thioesterase (APT). Furthermore, RalB is fatty-acylated at K200 promoting membrane localization. This is controlled by the acylase sirtuin-2 (SIRT2). (B) Phosphorylation of RAL GTPases. RalA is phosphorylated at S194 by Aurora A.

deficiency disturbs plasma membrane localization of only RALB, but endosomal localization of RALA, and increases protein stability of RALB but not RALA through unknown mechanisms (Gentry et al., 2015). As the HVRs of both RALA and RALB contain similar positive charge based on lysine-rich stretches and no other discernible membrane targeting information, the differential behavior of RAL GTPases may be based on variations in serine phosphorylation events in the HVR (see also Section 3.4.2). Since ICMT and RCE1 deficiency will have wide spread cellular effects, it is however not fully clear whether these findings are indeed due to direct effects on RAL modifications or mediated indirectly, e.g., through altered interactions with other (CaaX-terminating) binding partners. Nevertheless, this highlights that the effects of pharmacological inhibition of ICMT or RCE1 on RAL-controlled signaling will be complex and likely hard to predict.

Dual lipid modification has been suggested as a common process for proteins terminating in a CCax motif, where a palmitoyl moiety is added to the second cysteine after geranylgeranylation of the first by Golgi-associated protein acetyltransferase (PAT) (Fig. 6A). Palmitoylation can be reversed by acylprotein thioesterase (APT). Indeed, palmitoylation of exogenous and endogenous RALA and RALB has been reported (Gentry et al., 2015; Nishimura and Linder, 2013). Mutational analysis replacing C₂ with serine suggested that palmitoylation is required for the localization of RALB (but not RALA) to the plasma membrane (Gentry et al., 2015). However, potential regulatory mechanisms and cellular effects of RAL palmitoylation remain elusive.

Protein lysine fatty acylation, the attachment of long-chain fatty acyl groups such as myristoyl or palmitoyl to lysine residues, has also recently been reported for K200 in RALB, but not RALA, under basal cellular conditions (Fig. 6A) (Spiegelman et al., 2019). This acylation is regulated by SIRT2 (Sirtuin-2), an NAD-dependent protein lysine deacetylase.

Fig. 6—Cont'd kinase A and by protein kinase A (PKA) at an unknown residue. Phosphorylation of S183 and S194 is reversed by protein phosphatase 2A (PP2A). Protein kinase C (PKC) phosphorylates RalB at S198. RalB is also regulated by ubiquitination. Several ubiquitinated residues have been identified (K47, K57, K120, K129, K146, K160). K47 has been shown to be functionally relevant and is deubiquitinated by the ubiquitin-specific peptidase 33 (USP33). (C) Phosphorylation-dependent localization of RAL GTPases. RALA phosphorylation by Aurora kinase A can translocate the GTPase to mitochondria or endosomes and promotes interaction with the RAL effector RALBP1 (RAL-binding protein 1). Phosphorylation of RALB by PKC induces RALB translocation to endosomes. This enhances RALB interaction with RALBP1 over interaction with SEC5.

Ral GTPase signaling

Fatty acylation has also been described for KRAS4a where it hinders endomembrane localization, interaction with RAF and thus cellular transformation (Jiang et al., 2017). In the case of RALB however, fatty acylation promotes plasma membrane localization, GTP binding and invasion of the A549 cell line, while proliferation and anchorage-independent growth are not affected (Spiegelman et al., 2019).

### 3.4.2 Phosphorylation of RAL GTPases

In addition to lipid modification, RAL GTPases are also regulated by post-translational phosphorylation of serine residues located in the HVR affecting subcellular localization and activity (Fig. 6B). In 2005, Wu and colleagues first identified phosphorylation of RALA at S194 during a search for novel Aurora Kinase A substrates (Wu et al., 2005). Mutation of S194 to alanine rendered constitutively active RALA<sup>V23G</sup> unable to enhance cell migration and invasion of MDCK (Madin-Darby canine kidney) cells, suggesting that phosphorylation promotes RALA activity. In the following, several groups confirmed phosphorylation of RALA at S194 (Kashatus et al., 2011; Lim et al., 2010; Sablina et al., 2007) and identified PKA (Protein kinase A) as an additional kinase able to target RALA with the exact acceptor sites remaining to be identified (Wang et al., 2010).

Phosphorylation of RALA at S194 results in its translocation from the plasma membrane to internal organelles, including recycling and early endosomes, and in enhanced interaction of RALA with the effector RALBP1, but not SEC5. This leads to increased RALBP1 GAP activity toward the RHO GTPase CDC42 (cell division cycle 42) (Lim et al., 2010). Aurora kinase A, which normally functions as a mitotic kinase controlling cytokinesis, is frequently overexpressed in human cancers and involved in promoting tumor development through aberrant signaling (Keen and Taylor, 2004). Interestingly, activated Aurora kinase A can promote cellular transformation and tumor growth of pancreatic cell lines cooperatively with oncogenic RAS and this depends on S194 phosphorylation of RALA. This suggests that Aurora kinase A and RAS oncogenic signaling converge at the level of RALA (Lim et al., 2010). As Aurora kinase A regulates a variety of cancer-associated proteins it remains to be determined to what extent anti-tumor effects of Aurora kinase A inhibitors are mediated through inhibition of RAL phosphorylation (Katsha et al., 2015).

The relevance of S194 phosphorylation of RALA was further supported by the finding that the tumor suppressor PP2Aβ (serine-threonine phosphatase 2 Aβ), but not cancer-derived mutant forms, dephosphorylate this
site (as well as S183) thereby dampening RAL activity. Importantly, RAL GTPases are required for the cellular transformation observed upon ablation of PP2Aβ (Sablina et al., 2007). Whether this phenomenon is relevant in cells lacking oncogenic RAS signaling and which RAL downstream effectors are involved remains to be clarified.

An independent study from the Counter lab added further complexity by providing evidence that Aurora kinase A-dependent S194 phosphorylation also partially translocates RALA to mitochondria, where it promotes recruitment and CDK-1 mediated phosphorylation of the large GTPase DRP1, a critical regulator of mitochondrial fission, in a RALBP1-dependent fashion (Kashatus et al., 2011). Consequently, knockdown of either RALA or RALBP1 disturbs proper segregation of mitochondria during cytokinesis, resulting in impaired energy production and a decreased number of metabolically active cells. Regulation of mitochondrial fission might thus be one mechanism through which Aurora kinase A and RALA promote cancer. This further suggests that RALA phosphorylation represents a regulated mechanism to tune RAL activity toward certain outcomes.

RALB can be phosphorylated by PKC (Protein kinase C) at S198 *in vitro* and *in vivo* upon stimulation of cells with PMA (phorbol 12-myristate 13-acetate) (Wang et al., 2010). Similar to what is observed for RALA, S198 phosphorylation translocates RALB from the plasma membrane to the endosomal compartment and decreases interaction with the effector SEC5, while promoting RALBP1 binding (Martin et al., 2012; Wang et al., 2010). S198A mutation renders RALB unable to support anchorage-independent growth, actin cytoskeletal organization, motility, as well as tumor and metastasis formation of UM-UC-3 human bladder cancer cells (Wang et al., 2010). Furthermore, a role for RALB phosphorylation in regulating fusion of exocytic vesicles at the plasma membrane and consequent effects on the trafficking of α5-integrin have been reported (Martin et al., 2012). Here a model has been put forward, in which after RALB-dependent docking of the exocyst to fusion sites, phosphorylation triggers RALB dissociation thus allowing for SNARE-mediated vesicle fusion. However, a phosphatase that would dynamically regulate such a cycle to allow for efficient exocytosis and balanced function of RALB in exocytosis (via SEC5) and endocytosis (via RALBP1) remains to be identified.

In summary, the above-mentioned studies highlight that both GTP binding and phosphorylation are required for full and situation-adapted activity of RAL GTPases, implying that kinase inhibition might be a largely unexplored strategy to pharmacologically dampen or shift RAL signaling.
RAL GTPase signaling

### 3.4.3 Ubiquitination of RAL GTPases

RAL GTPases can also be modified by non-degradative, regulatory ubiquitination (Fig. 6B) (Neyraud et al., 2012; Simicek et al., 2013). RALA, but not RALB, ubiquitination was shown to increase under anchorage-independent conditions, but did not depend on RALA activation. Artificial fusion of ubiquitin to the N-terminus of RALA leads to increased plasma membrane localization and suggests an involvement of RALA ubiquitination in the regulation of lipid raft endocytosis. In contrast, fusion of ubiquitin to the N-terminus of RALB induced accumulation on not further defined intracellular punctate structures (Neyraud et al., 2012). However, it remains questionable whether forced N-terminal fusion of ubiquitin can mimic the physiological consequences of endogenous ubiquitination.

Simicek and colleagues identified the deubiquitinase USP33 (Ubiquitin-specific peptidase 33) as binding partner of RAL GTPases. Despite interacting with both RAL GTPases, USP33 only controls ubiquitination of RALB. Several lysines in RALB (K47, K57, K120, K129, K146, and K160) were identified to be modified by ubiquitination upon depletion of USP33 by mass spectrometry, with K47 and K160 being the most prominent (Simicek et al., 2013). Of these sites so far only K47, located in switch I, has been shown to be functionally relevant. Interestingly, instead of affecting subcellular localization, ubiquitination of K47 is involved in regulating substrate specificity of RALB by sterically hindering interaction with EXO84 but promoting SEC5 interaction. Virus infection, which triggers RALB-SEC5-TBK1 (TBK1, Tank-binding kinase 1) signaling, and nutrient starvation, which promotes RALB-EXO84-Beclin-1 complex formation, have been identified as physiological processes that make use of this regulatory tool to promote inflammatory signaling and autophagy, respectively (Simicek et al., 2013). Responsible E3 ligases and the molecular mechanisms that control RAL activity through regulation of ubiquitin ligase or hydrolase activities have not yet been discovered. USP33 has a higher affinity for RAL-GDP than RAL-GTP explaining how regulation through GTP/GDP cycling and ubiquitination might be linked. Potential effects of K160 ubiquitination have also not been addressed; however, K160 corresponds to K147 in RAS, whose monoubiquitination has been shown to hamper regulation of RAS by GAPs (Sasaki et al., 2011), suggesting that modification of RALB at K160 could be involved in direct regulation of RAL activity.

ARTICLE IN PRESS

Lisa H. Apken and Andrea Oeckinghaus

4. The RAL signaling network: Effector proteins and regulated processes

Research of the last two decades has made it clear that different ways of activation and a large roster of effector proteins exist to allow RAL GTPases to control a variety of cellular processes (Fig. 7). In the following we will summarize the main effector proteins and cellular activities that have been shown to be controlled by RAL GTPases.

### 4.1 The effector protein RALBP1: Regulation of actin dynamics and endocytosis

The most extensively studied RAL effector is RALBP1 (Ral-binding protein 1/RLIP76). RALBP1 is a multidomain protein containing a RHO/RAC GAP domain, a RAL binding domain, through which it associates with both switch regions of RALA and RALB, and two ATP binding sites (Awasthi et al., 2008; Fenwick et al., 2010). RALBP1 regulates actin dynamics by exhibiting GAP activity toward the small RHO GTPases CDC42 and RAC1 (Ras-related C3 botulinum toxin substrate 1), which modulate the actin cytoskeleton to stimulate filopodia and lamellipodia formation, respectively (Fig. 8) (Cantor et al., 1995; Jullien-Flores et al., 1995). Interestingly in pancreatic cancer cells, the role of RALBP1 in formation of invadopodia has been attributed to the ATP binding motifs in RALBP1 and not its RHOGAP domain (Neel et al., 2012). The ATP binding sites are responsible for RALBP1-mediated transport of glutathione electrophile-conjugates (see below), but it has not been addressed whether this function is indeed relevant for filopodia formation in pancreatic cancer cells. Furthermore, RALA has also been reported to directly bind filamin-A (filamin-1) to promote formation of filopodia, suggesting that multiple ways exist through which RALBP1 controls actin cytoskeleton dynamics (Ohta et al., 1999).

RALBP1 also interacts with ARNO (ARF nucleotide-binding site opener/Cytohesin-2), a GEF for ARF6 (ADP-ribosylation factor 6) to regulate cell spreading and migration (Lee et al., 2014; Wurtzel et al., 2015). ARF6 orchestrates several steps in endocytic membrane trafficking, but is also involved in actin remodeling at the cell periphery. The latter function is mediated by further recruitment of ARNO to ARF6-GTP after ARF6 activation. This further amplifies ARF6 activity, but also results in ARNO binding the RAC1-GEF dock180/ELMO complex to indirectly

Ral GTPase signaling

Fig. 7 Summary of RAL-interacting proteins. RALGEFs (RALGDS, RGL1, RGL2, RGL3, RGL4, RALGPS1, RALGPS2, and RCC2) promote GDP to GTP exchange of RAL GTPases. The RALGAP complexes, RALGAP1 and RalGAP2, return RALA and RALB to their GDP-bound "OFF" state. An enzymatic cascade of isoprenyl cysteine carboxyl methyltransferase (ICMT), Ras converting enzyme 1 (RCE1) and geranylgeranyltransferase type I (GGTasel) realizes geranylgeranylation of the Caax-like motif of RAL GTPases, resulting in membrane localization. Aurora kinase A, protein kinase A (PKA) and protein kinase C (PKC) phosphorylate residues in the C-terminal hypervariable region of RAL proteins, a process which can be reversed by serine-threonine phosphatase 2A (PP2A). The ubiquitin-specific peptidase 33 (USP33) removes ubiquitin modifications from RALB and acylation is reversed by the deacetylase sirtuin-2 (SIRT2). RAL GTPases can interact with a variety of effector proteins, including Filamin-A, RALBP1, SEC5, EXO84, PLD1, and PLC-δ1 and have been suggested to control the activity of different transcription factors such as FOXO4, TCF, RREB-1, NF-κB, STAT3, and ZONAB. Besides transcription, RAL GTPases regulate second messenger production, cell survival, immune responses, exo- and endocytosis, autophagy, and actin organization. Protein structure of GDP-bound human RALA (PDB: 6P0O).

ARTICLE IN PRESS

Lisa H. Apken and Andrea Oeckinghaus

Fig. 8 Cellular processes controlled by the RAL effector RALBP1. GTP-bound RAL interacts with the RAL-binding domain of RALBP1. Via the RHOGAP domain, RALBP1 regulates the catalytic activity of the RHO GTPases CDC42 (cell division cycle 42) and RAC1 (Ras-related C3 botulinum toxin substrate 1), which control actin cytoskeleton-mediated lamellipodia and filopodia formation, respectively. The latter can also be regulated by direct interaction of RAL GTPases with Filamin-A. RALBP1 regulates endocytosis through interaction with different regulators of receptor-mediated endocytosis and is involved in internalization of, e.g., Frizzled-7, and AMPA (α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid) receptors. Interaction of AP-2 (adaptor protein complex 2) with the N-terminus of RALBP1 and C-terminal interaction of RALBP1 with REPS1 and REPS2 (RalBP1-associated Eps domain-containing protein 1/2) orchestrates clathrin-dependent endocytosis via RAB11-FIP2 and EPS15 or Epsin-1. RALBP1 can also negatively regulate endocytosis during mitosis by cyclin-B1-dependent recruitment of CDK1 (cyclin-dependent kinase 1), followed by CDK1-dependent phosphorylation and thus inhibition of Epsin-1. Cyclin-B1-mediated engagement of CDK1 additionally regulates mitochondrial fission through phosphorylation of the GTPase DRP1 (Dynamin-related protein 1) during mitosis. RALBP1 promotes the cytosolic translocation of p27 from the nucleus, thereby relieving inhibitory function on CDK1-driven proliferative processes. The translocation of p27 is reversed by PLD1 (phospholipase D1). Via its ATP binding motifs, RALBP1 is involved in export of glutathione electrophile-conjugated (GS-E) toxicants including chemotherapeutics.

activate RAC1 (D'Souza-Schorey and Chavrier, 2006; Van Acker et al., 2019). RALBP1 also mediates ARF6 activation downstream of integrin signaling and oncogenic RAS to deliver raft microdomains to the plasma membrane to support anchorage-independent proliferation signaling (Pawar et al., 2016).

RALBP1 binds to a variety of proteins that are known regulators of endocytosis. While the N-terminus of RALBP1 can associate with the adaptor complex AP-2 (adaptor protein complex 2), which mediates clathrin-dependent endocytosis at the plasma membrane, the Eps homology (EH) domain-containing proteins REPS1 and REPS2/POB1 (RalBP1-associated Eps domain-containing protein 1/2) bind to RALBP1 in the C-terminal region, suggesting a scaffolding function of RALBP1 (Fig. 8) (Ikeda et al., 1998; Jullien-Flores et al., 2000; Nakashima et al., 1999; Yamaguchi et al., 1997). REPS1 interacts with RAB11-FIP2 (Rab11 family-interacting protein 2) and REPS2 with Epsin-1 and EPS15 (Epidermal growth factor receptor substrate 15), all regulators of receptor-mediated endocytosis (Cullis et al., 2002; Morinaka et al., 1999). Whether the interaction of RAL-GTP with RALBP1 has an activating or inhibiting effect on endocytosis however remains a matter of debate. Expression of constitutively active RALB<sup>G23V</sup> dampened endocytosis of transferrin and EGF receptors. Use of the D49N mutant that hampers interaction with RALBP1 but leaves interaction with SEC5 and EXO84 unaltered suggested that this effect requires RALBP1, suggesting a model in which RAL activity counteracts endocytosis via RALBP1 (Jullien-Flores et al., 2000). Furthermore during mitosis, RALBP1 has been reported to prevent endocytosis through cyclin-B1-dependent recruitment of CDK1 (Cyclin-dependent kinase 1) and subsequent phosphorylation of Epsin-1 by CDK1 (Rossé et al., 2003). In contrast, Han and colleagues report that, while RALA maintains a basal state of AMPA (α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid) receptors under resting conditions, RALA serves a positive regulatory role in receptor endocytosis in neurons upon stimulation (Han et al., 2009). NMDA (N-methyl-D-aspartate) receptor stimulation leads to RALA activation and recruitment of RALBP1 to the plasma membrane. RALBP1 is then dephosphorylated in a NMDA receptor activation-dependent manner, resulting in the interaction of RALBP1 with PSD-95 (Postsynaptic density-95) and promotion of AMPA receptor endocytosis. These findings suggest that the endocytic role of RAL/RALBP1 might be switchable, e.g., by additional posttranslational modification. RAL GTPases are also required for efficient internalization of the WNT receptor Frizzled-7 and consequent promotion of efficient WNT signaling. WNT signaling is important for intestinal stem cell function and

regeneration and thus a critical determinant of intestinal health. Consequently, dampened signaling leads to crypt death while hyper activation drives tumorigenesis (Clevers, 2006). Ablation of RALA or RALB in murine intestinal stem cells therefore results in impaired tissue regeneration upon damage and genetic deletion of both RAL GTPases to rapid crypt death due to impaired WNT signaling (Johansson et al., 2019). Internalization of WNT signalosomes was also hampered by knockdown of RALBP1, suggesting RALBP1 as the relevant RAL effector mediating Frizzled-7 endocytosis. This role of RAL GTPases is conserved in *Drosophila* (Johansson et al., 2019).

RALBP1 has also been linked to mitochondria. Recruitment of cyclin-B1/CDK1 to mitochondria via RALBP1 is required for mitochondrial fission during mitosis via promotion of CDK1-dependent phosphorylation of DRP1 (Dynamin-related protein 1/DNM1L) (Fig. 8) (Kashatus et al., 2011). Interestingly, RALA and RALB have also been reported to translocate to mitochondria upon mitochondrial depolarization, a process that coordinates the timing of when mitochondria can enter the fission/fusion cycle (Pollock et al., 2019); however, potential functional consequences remain to be determined.

Furthermore, RAL GTPases can modulate localization of the cyclin-dependent kinase inhibitor p27<sup>Kip1</sup> via RALBP1 (Tazat et al., 2013). RALBP1 activation leads to cytoplasmic localization of p27 away from the nuclear cyclin/CDK complexes (Fig. 8). This phenomenon has been demonstrated to be relevant for RALBP1 activity to impair TGFβ-mediated growth arrest in epithelial cells. Interestingly, activation of PLD1 (phospholipase D1) counteracts RALBP1 function and drives nuclear accumulation of p27. However, PLD1 activation seems to occur independently of RAL in this context (see also Section 4.4).

RALBP1 is overexpressed in different cancers and deletion of RALBP1 has been demonstrated to impair survival and tumorigenic growth and to reduce spontaneous and induced neoplasia in TP53 knockout mice (Awasthi et al., 2008, 2018; Singhal et al., 2006, 2007b). Depletion of RALBP1 by antibodies or siRNA has anti-tumor activity in murine models of melanoma and lung cancer and enhances sensitivity to chemotherapy. In this context, several studies have highlighted that dependent on its ATP binding domains, RALBP1 can transport glutathione electrophile-conjugated endogenous as well as exogenous toxicants such as different chemotherapeutics and might thus contribute to multidrug resistance (Awasthi et al., 2000; Drake et al., 2007; Janigro et al., 2007; Singhal et al., 2007a, 2007b). Notably, clear evidence for RAL control of this

transporter function has not yet been provided. Furthermore, RALBP1 is necessary for migration and metastasis of human cancer cell lines, effects that likely depend on control of RAC1 and CDC42 activities (Wu et al., 2010). Thus, RALBP1 is clearly important in tumorigenesis, but the exact contribution of distinct RALBP1-controlled cellular processes (e.g., ATP-dependent transport and control of actin dynamics) in different scenarios awaits further clarification. While pro-cancer effects could be at least in part also explainable by enhanced mitogenic signaling due to impaired receptor endocytosis, evidence to support such a theory has not yet been provided.

### 4.2 Effector proteins SEC5 and EXO84: Regulation of exocytosis and exocyst-independent functions

The exocyst complex is a multisubunit complex comprising eight proteins (SEC3, 5, 6, 8, 10, and 15, and EXO70 and 84). Exocyst assembly is a dynamic process bringing together subcomplexes that constitute a targeting patch at the plasma membrane and vesicle-associated subcomplexes (He and Guo, 2009). Several small GTPases are involved in regulating exocyst localization and assembly, including RHO, RAB, ARF and RAL GTPases. Their activity orchestrates when, where and how exocytosis occurs making the exocyst an integration platform for different incoming signals (Heider and Munson, 2012). Exocyst function is for example required for correct targeting and tethering of secretory vesicles to membrane domains such as the basolateral surface of polarized epithelial cells, neuronal synapses, growth cones of neuronal cells or the leading edge of migrating cells (Guo et al., 2000; Hsu et al., 2004; Wu and Guo, 2015). Also, insulin-dependent plasma membrane translocation and fusion of vesicles containing the glucose transporter GLUT4 (Glucose transporter type 4) requires the exocyst (Chen et al., 2006; Inoue et al., 2003).

The exocyst subunits SEC5 (EXOC2) and EXO84 (EXOC8) have been identified as RAL effectors. RAL GTPases can be present on the plasma membrane and/or secretory vesicles and regulate the subcellular localization and assembly of the exocyst complex via interaction with both SEC5 and EXO84 (Moskalenko et al., 2003; Spiczka and Yeaman, 2008). Engagement of RHOA via RAL/SEC5 interaction with the RHOGEF GEF-H1 can contribute to this function (Pathak et al., 2012). SEC5 and EXO84 bind RAL GTPases in a competitive manner and cell fractionation experiments revealed that SEC5 and EXO84 reside in separate subcomplexes with different subcellular localizations (Jin et al., 2005; Moskalenko et al., 2003). This likely explains their involvement in different cellular functions. Knockdown of

SEC5 and EXO84 reduces RALGEF-mediated transformation and oncogenic RAS-driven tumor growth, suggesting that RAL GTPases promote tumorigenesis at least in part through SEC5 and EXO84 (Hazelett and Yeaman, 2012; Issaq et al., 2010).

Through orchestrating exocyst assembly RAL GTPases control diverse exocytic pathways in different cell types, releasing granule contents or delivering membrane proteins (Fig. 9). RALA for example regulates the docking

![Diagram](attachment:diagram.png)

Fig. 9 Cellular processes regulated by the RAL effectors SEC5 and EXO84. SEC5 and EXO84 are components of the octameric exocyst complex, a main regulator of exocytosis. Interaction of GTP-bound RAL with SEC5 and EXO84 regulates localization and assembly of the exocyst. Engagement of RHOA via RAL/SEC5 interaction with the RHOGEF GEF-H1 might contribute to this function. TBK1 (Tank-binding kinase-1) and PKC (protein kinase C) mediated phosphorylation of EXO84 and SEC5, respectively, results in disengagement of RALA and vesicle fusion. RAL coordinates membrane protrusions through recruitment of the WRC (Wave regulatory complex) and ARP2/3 (Actin-related protein 2/3) complex. The RALA/exocyst axis is furthermore involved in the formation of tunneling nanotubes. This is probably mediated by SEC5 and association with M-SEC and can occur upon recruitment of RAL to the transmembrane protein LST1 (leukocyte specific transcript). Upon nutrient starvation, association of RALB with EXO84 promotes the assembly of a complex consisting of ULK1 (Unc-51-like kinase 1), PI3KC3 (Phosphatidylinositol 3-kinase catalytic subunit type 3/VPS34) and beclin-1, which initiates autophagy. Furthermore, interaction of RALB with SEC5 has been shown to control cell survival and immune responses via TBK1.

of insulin-containing vesicles at plasma membrane release sites in pancreatic β-cells, as well as targeting of GLUT4 storage vesicles in adipocytes ([Chen et al., 2007](https://doi.org/10.1038/nm1659); [Lopez et al., 2008](https://doi.org/10.1038/nm1790); [Saito et al., 2008](https://doi.org/10.1038/nm1791); [Tsuboi et al., 2005](https://doi.org/10.1038/nm1298)). During GLUT4 vesicle trafficking the RHO GTPase TC10 anchors an exocyst subcomplex on the plasma membrane though interaction with EXO70 ([Inoue et al., 2003](https://doi.org/10.1038/nm918)), while RALA is localized to GLUT4 containing vesicles, where it interacts with both SEC5 and EXO84 ([Chen et al., 2007](https://doi.org/10.1038/nm1659)). Activation of RALA upon insulin stimulation has been shown to critically depend on abrogation of RALGAP function through AKT2-dependent phosphorylation of RALGAPα1 at T735 in skeletal muscle and adipose tissue ([Chen et al., 2019a](https://doi.org/10.1038/s41590-019-0370-0); [Skorobogato et al., 2018](https://doi.org/10.1038/s41590-018-0170-0)). Adipose-specific knock-out of RALGAPβ thus increases GLUT4 plasma membrane localization and renders diet-induced obese mice less susceptible to metabolic disease due to enhanced RALA-dependent glucose uptake into brown fat tissue ([Skorobogato et al., 2018](https://doi.org/10.1038/s41590-018-0170-0)). In line, RALGAPα1 ablation in skeletal muscle leads to improved glycemic control in mice ([Chen et al., 2019a](https://doi.org/10.1038/s41590-019-0370-0)). RALGAP/RAL activity is also relevant for exocyst-dependent plasma membrane translocation of fatty acid translocase CD36 and thus lipid control. Consequently, knock-in of a T735A mutation in RALGAPα1, that renders RALGAP constitutively active and able to dampen RAL activity, hampered both glucose and fatty acid uptake in mice and led to the development of metabolic syndrome ([Chen et al., 2019a](https://doi.org/10.1038/s41590-019-0370-0)). Disengagement from the exocyst occurs through RALA-triggered phosphorylation of exocyst subunits. RALA activation induces TBK1-dependent phosphorylation of EXO84, which reduces its affinity for both RALA and the exocyst leading to dissociation from the complex ([Uhm et al., 2017](https://doi.org/10.1038/cr.2017.10)). In addition, RALA triggers PKC-dependent phosphorylation of SEC5 in its RAL-binding domain ([Chen et al., 2011c](https://doi.org/10.1038/cr.2011.10)). These events allow for efficient fusion of vesicles with the plasma membrane in response to insulin and dynamic control of tethering and fusion events. Thus, the RALGAP/RAL signaling module represents an important regulator of glucose and lipid homeostasis via control of exocyst function.

In epithelial cells, the RALA/exocyst axis is crucial for the appropriate transport of basolateral membrane proteins such as E-cadherin or EGF receptor ([Grindstaff et al., 1998](https://doi.org/10.1038/35002600); [Moskalenko et al., 2002](https://doi.org/10.1038/ni788); [Shipitsin and Feig, 2004](https://doi.org/10.1038/ncb1104-1095)). RAL-regulated exocytosis is also relevant during immune responses. In platelets, the RAL-SEC5 axis controls the exocytosis of dense granules, which is essential for platelet activation in regions of thrombus formation. The RAL-dependent tethering of the granules also reduces the threshold of Ca²⁺ concentration required to trigger the RAB27/
MUNC13–4 fusion machinery, suggesting concerted actions of these two inputs (Kawato et al., 2008; Wang et al., 2004). Additionally, RAL-regulated exocytosis is required for the secretion of cytotoxic proteins by neutral killer and cytotoxic T cells as well as of antimicrobials by neutrophils (Chen et al., 2011a; Sánchez-Ruiz et al., 2011). In endothelial cells, RALA controls exocytosis of Weibel–Palade bodies and thus secretion of the adhesive Willebrand factor during control of platelet aggregation and coagulation (De Leeuw et al., 1999, 2001; Rondaij et al., 2008).

In neurons, RAL GTPases exert functions at both the pre- and postsynaptic membranes (Polzin et al., 2002; Teodoro et al., 2013) and establish polarity during neuronal development and neuroblast migration (Das et al., 2014). Presynaptic RAL GTPases are required for providing a readily releasable pool of synaptic vesicles, but not for direct release. Through trafficking of NMDA and AMPA receptors, RAL GTPases are involved in organization of the postsynapse (Polzin et al., 2002; Teodoro et al., 2013). In neuronal development, RALA promotes the interaction of EXO84 with the adaptor protein PAR6 (Partitioning defective 6 homolog alpha), a component of the PAR complex, regulating cell polarity (Das et al., 2014). This function is conserved in *C. elegans* (Armenti et al., 2014). In *Drosophila* neuronal development, correct Ral/exocyst function is critical to enable appropriate flight behavior, since it controls synaptic exocytosis in dopaminergic neurons determining neuronal wiring of the flight circuit (Richhariya et al., 2017, 2018). In addition, RalA signaling in differentiated muscles is required for flight of adult animals (Richhariya and Hasan, 2020).

RAL GTPase-dependent regulation of the exocyst is also important in Schwann cell biology (Galino et al., 2019; Ommer et al., 2019). Schwann cells (SCs) produce myelin to enhance velocity of peripheral nerves. Differentiation to the myelin-producing state is called radial sorting and involves proliferation of SCs as well as the expansion of cellular extensions into bundles of unsorted axons (Jessen et al., 2015). While deletion of either RALA or RALB is without effect, knockout of both RAL proteins leads to persistent radial sorting defects, resulting deficits in SC formation and maintenance, and complex peripheral nerve defects. In culture, RAL-deficient SCs show reduced capabilities to produce extensions and form lamellipodia (Ommer et al., 2019). In addition, RAL GTPases are required for SC repair after nerve injury, affecting process formation and migration of SCs as well as myelination (Galino et al., 2019). In support, Silva-Rodrigues and colleagues reported that the RalA-exocyst pathway is critical for peripheral axon ensheathment in *Drosophila* larvae based on its control of growth and
development of wrapping glial cells, highlighting the evolutionary conservation of this function of RAL GTPases (Silva-Rodrigues et al., 2020).

Via engagement of the exocyst the RAL GTPases also support cell migration of normal, cancer and immune cells through delivery of membrane as well as α5-integrin, which allows for adhesion to extracellular matrix, to leading edges (de Gorter et al., 2008; Oxford et al., 2005; Rossé et al., 2006; Schmoranz er al., 2003; Takaya et al., 2004). Recently, using an optogenetics approach, RALB was activated with both spatial and temporal control to decipher the processes required for mediating invasiveness downstream of RALB (Zago, 2018). This study finds that exocyst-dependent recruitment of the Wave regulatory complex (WRC) to the cell front is critical for the formation of protrusions and invasion of transformed cells (Fig. 9). The WRC, a five subunit heteromeric complex, is involved in actin cytoskeleton dynamics through interaction with the ARP2/3 complex (Actin-related protein 2/3 complex) and represents a known RAC1 GTPase effector (Biondini et al., 2016). Interestingly, the described role of the WRC in RALB-driven invasion has been suggested to be independent of RAC1 activity, but directly coordinated by the exocyst.

Additionally, RAL GTPases play a role in cytokinesis through exocyst control. Here, RALA and RALB have been shown to act in a concerted fashion where RALA tethers the exocyst to the cleavage furrow in early cytokinesis, while RALB drives subsequent recruitment of the exocyst to the midbody ring to promote abscission and complete cytokinesis. Interestingly, these two processes are likely GEF-specifically initiated by RAL activation through RALGDS and RALGPS1 or RGL1 and RALGPS2, respectively (Cascone et al., 2008).

RAL/exocyst signaling has also been shown to have an important developmental role in C. elegans vulva patterning, which is governed by an EGF gradient inducing RAS effector switching between RAS/RALGEF/RAL-1 and RAS-RAF signaling (Zand et al., 2011). RAL-1 controls 2° cell fate via interaction with EXO84 launching a MAPK effector cascade (MAP4K/MAP3K/p38) (Shin et al., 2018). Interestingly, the RAL-independent scaffolding function of RALGDS bridging PDK1 to AKT upon PI3K induction (Hao et al., 2008; Tian et al., 2002) is conserved in this context and contributes to 1° cell fate (Shin et al., 2019). Cell fate is also regulated by Ral signaling in Drosophila eye development where Ral activity inhibits ligand-independent Notch activation with the underlying molecular mechanism remaining unclear (Cho and Fischer, 2011).

Finally, the RALA/exocyst axis has been found to be involved in the promotion of tunneling nanotubes (TNTs), which are long, thin membrane extensions providing a means of intercellular communication (Fig. 9) (Norden et al., 2020). TNTs have been shown to connect pericytes, and immune cells, but also other cell types, and are used to transfer organelles, soluble markers or electric signals (Abounit and Zurzolo, 2012; Alarcon-Martinez et al., 2020). M-SEC (TNFaip2, tumor necrosis factor-α-induced protein 2) has been identified as a critical mediator of TNT formation (Hase et al., 2009). RALA was found to directly bind M-SEC and localize to M-SEC positive TNTs. This interaction is essential for promotion of TNTs and requires binding of RALA to exocyst subunits. In support, RALA is also recruited to the plasma membrane by the transmembrane MHC class III protein LST1 (leukocyte specific transcript), promoting its interaction with SEC5 and subsequent TNT formation (Schiller et al., 2013). The exocyst might hereby support nanotube formation through both membrane supply and actin remodeling.

In addition, RAL GTPases control exocyst (subunit)-dependent functions that are not connected to exocytosis (Fig. 9). RALB for example recruits and activates TBK1 via SEC5 (Chien et al., 2006). This interaction at least in part confers the anti-apoptotic effect of constitutively active RALB in HeLa cells. In line with this, RALB and TBK1 have been shown to be required for anti-apoptotic NF-κB activity in mutated KRAS-driven lung cancer (Barbie et al., 2009). Interestingly, the RALB/SEC5/TBK1 axis also represents an integral part of the immune response of normal airway epithelial cells to extracellular dsRNA or Sendai virus infection, supporting both IRF3 (Interferon regulatory factor 3) and NF-κB activity. These findings allow the hypothesis that oncogenic transformation hijacked a normal cellular defense mechanism, provided by RALB/SEC5/TBK1 signaling, to counteract and overcome apoptotic triggers. Bodemann and colleagues furthermore reported that RALB association with EXO84 promotes nutrient starvation-induced autophagocytosis through triggering assembly of ULK1/Beclin-1/PI3KC3 autophagy initiation complexes (ULK1 (Unc-51-like kinase 1), PI3KC3 (Phosphatidylinositol 3-kinase catalytic subunit type 3/VPS34)) independent of exocyst function (Bodemann et al., 2011). Attachment of ubiquitin to K47 of RALB sterically hinders EXO84 interaction and thus triggers a switch from interaction with EXO84/Beclin-1 and autophagy control to binding of SEC5/TBK1 in innate immune response regulation (Simicek et al., 2013). In line, dsRNA promotes RALB ubiquitination while nutrient starvation initiates deubiquitination

of RALB by the deubiquitinase USP33. These findings suggest that RALB can control metabolic states by balancing cell growth versus cell maintenance. In *Drosophila*, RalA regulates autophagy during developmentally regulated cell death in salivary glands, where it is at least in part due to upregulated Notch signaling, but interestingly not starvation-induced autophagy in the fat body (Tracy et al., 2016). This difference might be due to organism- or context-dependent regulation of autophagy.

### 4.3 Crosstalk of RAL and mTORC signaling

mTORC1 is a kinase complex consisting of the serine/threonine kinase mTOR, Raptor (regulatory protein associated with mTOR), and mLST8 (mammalian lethal with Sec13 protein 8). mTORC1 integrates nutrient and growth factor availability inputs to appropriately adapt cell growth. The RAG (Ras-related GTP-binding protein) and RHEB GTPases govern mTORC1 activity through control of lysosomal localization and direct activation, respectively (Saxton and Sabatini, 2017). Amino acid availability activates RAGs, enabling them to bind Raptor and recruit mTORC1 to the lysosomal surface, where Rheb is also located. RHEB is activated by the presence of growth factors, which induce TSC2 phosphorylation and thus inhibition of TSC complex function (Saxton and Sabatini, 2017). Consequently, concerted activation of RAGs and RHEB is required to trigger mTORC1 activation. Several studies have now also implicated RAL GTPases in the mTORC1 signaling network. Knockdown of RALGDS or RALA, but not RALB, impairs the amino acid- and glucose-, but not insulin-stimulated phosphorylation of S6 kinase and 4E-BP1 (Eukaryotic translation initiation factor 4E-binding protein 1) by mTORC1 in HeLa cells (Maehama et al., 2008). RALA knockdown also reduces RHEB-induced S6 kinase phosphorylation and a constitutively active RAL can trigger low level mTORC1 activation in RHEB knockdown cells. It was thus hypothesized that RAL GTPases function downstream of RHEB or can substitute for RHEB when nutrient sensing pathways are activated. One possible explanation might come from the fact that mTOR binding to Raptor is promoted by phosphatidic acid (PA), a product of phospholipase D, whose activation upon nutrient stimulation depends on RAL GTPases (Xu et al., 2011) (see Section 4.4).

In line with the general notion of intertwined RAL and mTORC1 signaling, genetic ablation of RALGAPβ has been reported to result in upregulation of mTORC1 activity and suppression of autophagy in murine

embryonic fibroblasts (Martin et al., 2014). This effect was shown to be mediated by RALB in a SEC5-dependent manner, suggesting that phospholipase D might rather not be involved. Instead, it was hypothesized that RALB might control trafficking of mTORC1 to the plasma membrane to control activity. Such a scenario would be reminiscent of the role of RAG GTPases, which act as recruiting platform for mTORC1 on the lysosomal membrane (Sancak et al., 2010). Interestingly, the same study also reported that loss of the TSC complex results in enhanced RAL-GTP levels despite the fact that TSC does not show RALGAP activity *in vitro*, suggesting vice versa crosstalk between RALGAP/RAL and TSC/RHEB pathways. Also, in *C. elegans*, where orthologs of RHEB, RAL, and RALGAP, but not TSC are present, deletion of RALGAPβ phenocopied CeTORC1 overexpression, resulting in reduced life span (Martin et al., 2014). While these experiments underpin the evolutionary conservation of TSC and RALGAP, more studies are required to clearly delineate the extent, impact and underlying mechanisms of these crosstalk phenomena.

### 4.4 Regulation of phospholipase D1 activity

RAL GTPases have been shown to associate with PLD1 (phospholipase D1), but in a GTP-binding independent manner. Instead, interaction occurs with the N-terminal 11 amino acids of RAL that are not present in RAS GTPases and is enhanced by Ca²⁺ (Kim et al., 1998; Luo et al., 1998; Sidhu et al., 2005b). PLD1 catalyzes hydrolysis of phosphatidylcholine (PC) to choline and the second messenger phosphatidic acid (PA) in response to G protein-coupled receptor stimulation. As described above, promotion of Raptor binding to mTOR by PA has been suggested as potential mechanism underlying the importance of RALA-driven PLD activity for amino acid- and glucose-induced mTORC1 activation (Xu et al., 2011). PLD1 can also be activated by ARF GTPases and has in this context been suggested to be involved in vesicle transport and membrane trafficking. RALA and ARF1 can synergistically activate PLD1 downstream of v-SRC and v-RAS, indicating a possible impact of RALA on vesicle trafficking via PLD1 (Jiang et al., 1995; Kim et al., 1998; Luo et al., 1998). Other studies have implicated RALA/PLD2 in constitutive endocytosis of metabotropic glutamate receptors and FcγR-mediated phagocytosis (Bhattacharya et al., 2004; Corrotte et al., 2010). Furthermore, RALB/PLD interaction has been described in calcium-triggered exocytosis and neurite branching (Lalli and Hall, 2005; Vitale et al., 2005).
RAL GTPases also interact with another phospholipase, PLC-δ1 (Phosphoinositide phospholipase C-δ-1) (Sidhu et al., 2005b). The PI4,5P₂-specific PLC-δ1 produces the second messengers diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP₃). DAG activates PKC and IP₃ mediates the intracellular release of Ca²⁺ (Rhee and Bae, 1997). Calmodulin binding inhibits the activity of PLC-δ1 which is antagonized by RAL interaction (Grujic and Bhullar, 2009).

### 4.5 Control of transcription factor activities

The best characterized example of RAL control of a transcriptional regulator is the Y-box transcription factor ZONAB (zonula occludens 1-associated nucleic acid binding protein), a known transcriptional repressor. RAL proteins directly interact with ZONAB in a cell density-dependent manner (Fig. 10) (Frankel et al., 2005). When cell density is low, interaction with RAL sequesters ZONAB in the cytosol and thus relieves repression of ZONAB-dependent promoters. As cell contacts increase, RAL dissociation from ZONAB allows for nuclear translocation of the transcriptional repressor. In addition, RAL GTPases mediate oncogenic RAS-induced alleviation of ZONAB-dependent transcriptional repression (Frankel et al., 2005). Respective upregulated genes however remain to be determined.

Over the years, RAL GTPases have been reported to regulate the activity of a variety of additional transcription factors in different cellular systems (Fig. 10). However, in most cases it has remained unclear how the signal is translated downstream of RAL, whether the regulation is indeed direct and what the cellular consequences of these regulations might be. Also, effects of GEF overexpression cannot always be reproduced by expression of constitutively activated RAL GTPases in this context (Ruiter et al., 2001). Thus careful evaluation of potential RAL-independent effects of GEFs will be required to support some of the reported findings.

Overexpression of constitutively active RGL2 induces activation of TCF (ternary complex factor), which stimulates the c-fos promoter, and of the Forkhead family transcription factor FOXO4 (Forkhead box protein O4/AFX), which can upregulate p27<sup>Kip</sup> to induce cell cycle arrest (Kops et al., 1999; Ruiter et al., 2001; Wolthuis et al., 1997). In the latter case, RALA functions cooperatively with PI3K signaling (Kops et al., 1999). In both cases relevance of RAL for the observed effects has been provided by the use of dominant-negative RAL mutants. RAL GTPases are also required for activation of FOXO4 through JNK-mediated phosphorylation
ARTICLE IN PRESS

Lisa H. Apken and Andrea Oeckinghaus

TNF
oxidative stress
cell density
EGF

![Diagram](#)

Fig. 10 RAL control of transcription. In response to oxidative stress or TNF (tumor necrosis factor) signaling RAL promotes the assembly of a complex consisting of JIP1 (c-Jun amino-terminal-interacting protein 1), MLK3 (Mixed lineage kinase 3/MAP3K11), MKK4 (MAP kinase kinase 4/MAP2K4) and JNK (c-Jun N-terminal kinase) with the latter phosphorylating the transcription factor FOXO4 (Forkhead box protein O4). Sensing cell density, RAL GTPase interaction with ZONAB (zonula occludens 1-associated nucleic acid binding protein) relieves its repression of transcription. Other transcription factors that can be controlled by RAL GTPases are TCF (ternary complex factor), RREB-1 (Ras-responsive element-binding protein-1), NF-κB (Nuclear factor κB), and STAT3 (Signal transducer and activator of transcription 3). Upon EGF signalling RAL GTPases invoke STAT3 via activation of the kinase SRC.

in response to oxidative stress or TNF (Tumor necrosis factor) stimulation (Essers et al., 2004). JNK activation hereby occurs through RALA-mediated assembly of an activating scaffold complex containing JIP1 (c-Jun amino-terminal-interacting protein 1), MLK3 (Mixed lineage kinase 3/ MAP3K11), MKK4 (MAP kinase 4/MAP2K4), and JNK, a mechanism which is conserved in C. elegans (van den Berg et al., 2013). In contrast, experiments performed in Drosophila implied that RAL GTPases
suppress JNK activity (Balakireva et al., 2006; Sawamoto et al., 1999). How these contradictory findings can be rationalized is unclear. It is noteworthy though that RALGEFs have been characterized to rather function as RAP than as RAS effectors in *Drosophila* (Mirey et al., 2003), suggesting that RAP GTPases might play a more prominent role in RAL regulation in this organism than RAS.

Expression of constitutively active RAL mutants mobilizes STAT3 (Signal transducer and activator of transcription 3) through the activation of the kinase SRC, a process which is required for EGF-induced STAT3 activation (Goi et al., 2000). Inversely, STAT3 can induce RAL activity through increasing RALGDS expression (Senga et al., 2001). RAL GTPases have also been shown to promote the activation of NF-κB to increase expression of cyclinD (Henry et al., 2000). In line, RALB promotes TBK1 activation in mutant KRAS-expressing lung cancer cells to upregulate an NF-κB gene signature that supports cellular survival (Barbie et al., 2009).

In bladder cancer cells, analysis of altered gene expression after RALA and RALB depletion demonstrated a large overlap between affected genes and identified RREB-1 (Ras-responsive element-binding protein-1) as a putative regulated transcription factor (Oxford et al., 2007). The obtained gene signatures are associated with cancer aggressiveness, supporting clinical relevance of RAL signaling (Smith et al., 2012). However, much remains to be learned about how deregulation of transcriptional programs contributes to the critical role of RAL signaling in tumorigenesis.

### 5. RAL signaling in cancer

Research of the last three decades has revealed a complex entanglement of RAL GTPase signaling in various types of malignancies. Additionally, many studies addressed functions of RAL regulatory proteins or RAL effectors; however, evidence for participation of deregulated RAL activity in the observed phenotypes has not always been provided. A role of RAL GTPases in RAS-driven tumorigenesis had been implicated by the characterization of RALGEFs as RAS effectors, but was initially neglected as constitutively active mutants of RAL did not transform NIH3T3 murine fibroblasts (Urano et al., 1996). Instead, RALGEFs and active RAL mutants enhanced proliferation in combination with activation of RAS or other RAS effector pathways (Urano et al., 1996; White et al., 1996; Wolthuis et al., 1997). The Counter lab then reported that of the three main RAS effector pathways the RALGEF/RAL axis is the most potent to induce
transformation of human cells, suggesting species or cell type differences and reviving research interest into RAL (Hamad et al., 2002). Using pathway-specific activating RAS mutants several studies unraveled that activation of the RAL pathway is sufficient and required for RAS-induced transformation of human cells as well as for invasion and metastasis (Hamad et al., 2002; Rangarajan et al., 2004; Ward et al., 2001). Studies of White and colleagues revealed that RALA is necessary for anchorage-independent proliferation and RALB for survival of tumor cells to collaboratively maintain tumorigenicity (Chien and White, 2003). Since then, RAL GTPases have been identified as important players in many cancer types with the recurrent theme of cell type-specific and frequently divergent functions despite high molecular identity.

The list of malignancies with described RAL involvement includes both mutant RAS-dependent and -independent cancers (Fig. 11). Increased RAL

![Figure 11](attachment://image.png)

Fig. 11 Cancer entities with reported RAL involvement in tumorigenesis. Cancer types for which a clear contribution independent of oncogenic RAS signaling has been described are highlighted by green boxes. Cancer types in which RAL GTPases have been reported to function downstream of oncogenic RAS mutants are marked by blue boxes.

expression or activity has for example been observed in colorectal carcinoma (Gyorffy et al., 2015; Martin et al., 2011), hematopoietic malignancies (Seibold et al., 2019), malignant peripheral nerve sheath tumor (Bodempudi et al., 2009), melanoma (Zipfel et al., 2010), oral squamous cell carcinoma (Gao et al., 2019) and bladder cancer (Saito et al., 2013) cell lines regardless of RAS mutation or activation. Here, much remains to be learned about the differential involvement of specific GEFs, the impact of RALGAP deregulation and the role of epigenetic and posttranslational control mechanisms. A roster of RAL effectors has been identified, many of which control functions that could promote tumorigenesis in different ways. However, in many cases exact effector contributions to downstream tumorigenic effects of RAL signaling remain uncharacterized. Since treatment efforts are likely to benefit significantly from combination therapies including interference with the RAL signaling axis, identification of targetable upstream or downstream mediators of RAL activity remains an important task.

Cancer stem cells (CSCs), also called tumor-initiating cells, are highly aggressive and resistant to drug-induced apoptosis, constituting a malignant cell pool, from which tumors can repopulate. They are considered critical for metastasis and tumor relapse so that there is great scientific interest in identifying pathways that confer CSC characteristics (Patel and Chen, 2012). In many tumor types these cells are commonly identified via the expression of specific surface markers, such as CD24, CD44 or CD133. An elegant study from the Cheresh laboratory has identified integrin αvβ3 as a marker of cells with stem-like properties in breast, lung and pancreatic cancer, which is associated with an increase in tumor progression and drug resistance (Seguin et al., 2014). Interestingly, integrin αvβ3 in its unligated state connects to KRAS via galectin-3, resulting in RALB activation. Active RALB then triggers TBK1 and NF-κB activation to enhance anchorage-independent growth, tumor initiation, self-renewal and drug resistance. Silencing of RALB specifically affects integrin β3-positive tumors, while knockdown of RALA, AKT or ERK have similar growth reducing effects in β3-positive and β3-negative tumor cells. Depletion of TBK1 or the NF-κB subunit c-Rel counteracts integrin β3-mediated self-renewal and drug resistance and combination of the NF-κB pathway inhibitor bortezomib and the EGF receptor inhibitor erlotinib leads to loss of the β3-positive population of CSCs (Seguin et al., 2014). These results strongly suggest a role for RALB in stemness and demonstrate that pharmacological interference with events downstream of RAL can be effective in reversing it. In addition, RALA activation was found to be high in the

CD44-positive fraction of the lung cancer cell line A549 (Male et al., 2012) and in CD133-positive hepatocellular carcinoma, malignant peripheral nerve sheath tumor (MPNST) and medulloblastoma cells (Borrego-Diaz et al., 2012; Ezzeldin et al., 2014; Ginn et al., 2016). However, whether RALA activity is indeed critical to CSC function in these cases remains to be elucidated.

Interestingly, oncogenic hot spot mutations as found in RAS that would render RAL GTPases constitutively GTP-locked have not been identified in tumors. Mutations in RAL regulatory proteins or RAL effectors are also rather infrequent and it remains unclear whether the ones detected are indeed functionally relevant or represent passenger mutations (Bodemann and White, 2008). A potential explanation might be that RAL GTPases require cycling for efficient effector engagement (Feig, 2003). The use of RAL mutants to deduce information therefore needs to be considered with care. In the following we review key findings concerning the role of RAL or RAL regulators in different cancer entities.

### 5.1 Pancreatic cancer

Pancreatic cancer represents the tumor type for which involvement of RAL signaling has been studied most intensively. Most pancreatic tumors arise in the exocrine tissue, with pancreatic ductal adenocarcinoma (PDAC) representing the most common tumor type. Activating KRAS mutations are considered the initiating event and are found in more than 90 % of all PDACs (Bailey et al., 2016).

In oncogenic KRAS signaling in the pancreas, the RALGEF/RAL pathway plays a prominent role. RAL GTPases are activated in pancreatic cancer cell lines and patient tissues more frequently than other RAS effector pathways and are required for cellular transformation (Jones et al., 2008; Lim et al., 2005, 2006). Knockdown studies in human patient cell lines delineated that RALA and RALB are responsible for different functions in PDAC cells. RALA is critical for anchorage-independent proliferation, affecting both colony formation *in vitro* as well as xenograft growth *in vivo* (Lim et al., 2005, 2006). Persistent RALA activity is hereby required for tumor maintenance since inducible knockdown in established primary tumors results in tumor regression. RALB is important for survival and invasion of most PDAC cell lines. Also, RALB depletion reduces metastasis of cells after tail vein injection *in vivo*, which is only minimally affected by knockdown of RALA (Falsetti et al., 2007; Lim et al., 2006; Neel et al., 2012).

In this context, RALB orchestrates formation of invadopodia, actin-rich membrane protrusions, which play a role in metalloproteinase secretion, to support invasion. This function is mediated by RALBP1 and interestingly requires the ATPase but not the RHOGAP function of RALBP1 for unknown reasons (Neel et al., 2012).

RALGEF and RALGAP functions also play a role in PDAC. Overexpression of the RALGEF RGL2 has been detected in PDAC cell lines and pancreatic tumors. Knockdown of RGL2 is sufficient to diminish RALA- and RALB-GTP levels in PDAC cells, resulting in decreased anchorage-independent proliferation and invasion. Thus, RGL2 might be the RALGEF transmitting oncogenic RAS signals to both RAL isoforms in this cell type (Vigil et al., 2010). While the dominant-negative RALA${}^{31\mathrm{~N}}$ mutant, which blocks GEF function, inhibits colony growth of PDAC cell lines, the constitutively active RALA${}^{75\mathrm{~L}}$ mutant surprisingly is not able to rescue the effect of RGL2 depletion. This has led to the hypothesis of potential RAL-independent functions of RGL2, but might also reflect that RALA binding to RGL2 or GTP/GDP cycling is required for functional output. In PDAC cells, RGL2 is colocalized with RALB, but not RALA, at the leading edge of migrating PDAC cells and this localization of RALB is lost upon RGL2 depletion (Vigil et al., 2010). Thus, while RALA is likely activated by a cytosolic pool of RGL2 without altering its subcellular localization, RGL2 seems to regulate RALB function directly and also indirectly by membrane recruitment. This again highlights the importance of subcellular localization of GTPases and their regulatory proteins for signaling outcome.

In addition, low expression of κB-RAS proteins, which are required for RALGAP activity, has been detected to occur frequently in PDAC patients and to inversely correlate with histological grade (Beel, 2020; Oeckinghaus et al., 2014). Ablation of κB-RAS in a KRAS${}^{\mathrm{G12D}}$-driven mouse model of PDAC facilitates development of invasive carcinoma and reduces median survival. κB-RAS depletion hereby upregulates both RALA- and RALB-GTP levels through loss of RALGAP activity and thus enhances anchorage-independent proliferation as well as invasion of murine and human PDAC cell lines. Interestingly, κB-RAS deficiency also promotes acinar-to-ductal metaplasia during tumor initiation and prevents acinar regeneration after chemically induced acute pancreatitis, suggesting a general role for RAL GTPases in acinar plasticity in the pancreas (Beel, 2020). Loss or deregulation of RALGAP function and consequent loss of RAL control thus represent a risk factor for pancreatic cancer.

Interestingly, silencing or dominant-negative blockade of cyclin-dependent kinase 5 (CDK5) was shown to impair anchorage-independent proliferation, invasion and migration of PDAC cells *in vitro* as well as orthotopic tumor formation and metastasis *in vivo* through inhibition of the RAS-RAL signaling axis with the underlying molecular mechanism remaining to be elucidated ([Feldmann et al., 2010](https://doi.org/10.1038/onc.2010.46)).

### 5.2 Lung cancer

Also in lung adenocarcinomas, which harbor KRAS mutations in about 30 % of all cases ([Ding et al., 2008](https://doi.org/10.1016/j.cell.2008.07.029); [Karachaliou et al., 2013](https://doi.org/10.1016/j.lungcan.2013.04.005); [Westcott and To, 2013](https://doi.org/10.1016/j.jtho.2013.01.002)), is the importance of the RAL pathway well established. Mutant KRAS-expressing non-small cell lung cancer (NSCLC) cell lines exhibit elevated RAL GTPase levels and aberrant activation of RALA ([Guin et al., 2013](https://doi.org/10.1038/onc.2013.10)). In NSCLC patients, high expression of RAL GTPases is associated with advanced stage ([Kim et al., 2014](https://doi.org/10.1038/onc.2013.499)) and poor survival of patients with KRAS mutation ([Guin et al., 2013](https://doi.org/10.1038/onc.2013.10)). siRNA-mediated knockdown of RALA or RALB or combined depletion of both proteins reduces anchorage-independent and -dependent proliferation and tumor growth of most mutant KRAS-expressing cell lines *in vitro* with cell type-specific efficiency. In contrast, wildtype KRAS-expressing cells are largely insensitive to RAL silencing ([Guin et al., 2013](https://doi.org/10.1038/onc.2013.10); [Male et al., 2012](https://doi.org/10.1038/onc.2012.10)). Knockdown of either RALA or RALB impairs tumor growth when KRAS<sup>G12C</sup>-expressing H2122 cells are grafted subcutaneously ([Guin et al., 2013](https://doi.org/10.1038/onc.2013.10)) and RALA knockdown in the NSCLC cell line A549 decreases invasion capabilities and increases apoptosis ([Male et al., 2012](https://doi.org/10.1038/onc.2012.10)). In line, the RAL inhibitors BQU57 and RBC8 both reduce anchorage-independent proliferation of lung cancer cell lines H2122 and H358 ([Yan et al., 2014](https://doi.org/10.1038/onc.2014.10)). Sensitivity of KRAS mutant lung cancer cell lines to RALB silencing has been ascribed to RALB-dependent activation of TBK1, for which a synthetic lethal interaction with oncogenic KRAS has been reported ([Barbie et al., 2009](https://doi.org/10.1038/nature08054)). In this context, TBK1 was shown to activate anti-apoptotic NF-κB signaling to ensure cellular survival, suggesting a role for RALB-mediated TBK1 activation in specific upregulation of NF-κB activity downstream of oncogenic RAS in lung cancer.

While in cell lines reduction of either RAL GTPase is sufficient to dampen anchorage-independent proliferation, genetic ablation of RALA and RALB in a KRAS<sup>G12D</sup>-driven lung cancer mouse model suggests redundant functions of the RAL GTPases ([Peschard et al., 2012](https://doi.org/10.1038/onc.2012.10)).

Lung-specific knockout of either RALA or RALB is unremarkable, concurrent ablation of both RAL GTPases in combination with KRAS<sup>G12D</sup> expression however results in decreased cell proliferation and almost completely blocks tumor development *in vivo*. In support of potential underlying compensatory effects, activation levels of RALA increase when RALB is deleted and vice versa (Peschard et al., 2012). This discrepancy between silencing of RAL GTPases in fully transformed human tumor cells and genetic ablation in the murine model might be based on the fact that the former addresses a role in tumor maintenance whereas in the knockout mouse model RAL GTPases are already ablated at tumor initiation, potentially allowing for compensatory mechanisms to affect transformation. As the role of RAL GTPases has not yet been investigated by direct genetic deletion in other cancer mouse models, it remains unclear whether this concept might be more general.

Studies in NSCLC cell lines also put forward the notion that different KRAS mutations selectively engage specific effector pathways (Guin and Theodorescu, 2015). While KRAS<sup>G12C</sup>- and KRAS<sup>G12V</sup>-expressing NSCLC cell lines were suggested to rely on RAL signaling, activation of RAF-MEK-ERK and PI3K-AKT-mTOR was prominent in cells with KRAS<sup>G12D</sup> mutation (Guin et al., 2013; Ihle et al., 2012). KRAS<sup>G12C</sup> was found to regulate anchorage-independent proliferation via activation of RAL signaling and patients harboring a G12C or G12V mutation show shorter progression-free survival (Guin et al., 2013; Ihle et al., 2012). In contrast, analysis of a large group of Korean NSCLC patients suggested no correlation between RAL activation and distinct KRAS mutations (Kim et al., 2014).

The recently identified RALGEF RCC2 shows elevated expression in lung cancer samples (Pang et al., 2017; Wu et al., 2018) and high RCC2 expression correlates with advanced tumor stage, lymph node metastasis and poorer overall survival. In *in vitro* experiments, RCC2 overexpression leads to increased proliferation, cell migration and invasion. Enhanced metastasis of cells overexpressing RCC2 was confirmed *in vivo* and was ascribed to increased JNK activation, resulting in expression of the matrix metalloproteases MMP-2 and MMP-9 and epithelial–mesenchymal transition (EMT) (Pang et al., 2017). RCC2 was also found to negatively regulate the activity of RAC1-directed apoptosis (Song et al., 2018; Wu et al., 2018). Whether and how the RAL pathway is involved in these phenotypes however remains to be clarified. The role of other RALGEFs in NSCLC cell lines has been addressed by systematic silencing (Santos et al., 2016).

While depletion of RGL1 and RALGPS1 is without effect, silencing of RALGDS, RGL2, RGL3, and most prominently RALGPS2 impairs anchorage-dependent and -independent proliferation of several cell lines. RALGPS2 silencing also caused cell cycle arrest in G0/G1 phase, an increase in the number of apoptotic cells and upregulation of the cell cycle inhibitors p27 and p21 (Santos et al., 2016). Depletion of RALA, RALB or both did not mimic all of these effects, bringing potential RAL-independent signaling into play.

## 5.3 Bladder cancer

Mutations in RAS occur in about 10% of human bladder tumors (Kompier et al., 2010; Ouerhani and Elgaaied, 2011). Both RAL isoforms were found to be abnormally expressed and activated in HRAS mutant but also in invasive bladder cancer cell lines that did not express mutant HRAS, but instead presented with reduced expression of RALGAPα2 (see below) (Saito et al., 2013; Smith et al., 2007). In tumor samples, RALA and RALBP1 are overexpressed alongside with elevated levels of Aurora kinase A and expression correlates with higher tumor grades (Smith et al., 2007). In the tested array, only one bladder cancer cell line contained a mutation in RALA (Q97E), confirming that RAL mutations play a minor role in aberrant activation (Smith et al., 2007). Additionally, RALGPS2 has been identified to be important for proliferation and formation of actin-rich tunneling nanotubes of the bladder cancer cell line 5637 by interaction with the transmembrane MHC class III protein leukocyte specific transcript 1 (LST1) (D’Aloia et al., 2018). This function requires RALA and its interaction with SEC5. Smith and colleagues established a core RAL transcriptional signature comprising genes that were deregulated upon depletion of both RALA and RALB from bladder cancer cells. This signature connects the RAL signaling pathway to cancer stage, invasion, metastasis, and survival (Smith et al., 2012). RREB-1 (Ras-responsive element-binding protein 1) was identified as a transcription factor mediating transcriptional changes in this context (Oxford et al., 2007).

Interestingly, RALA and RALB were attributed redundant functions in tumor proliferation but antagonistic roles in migration in the KRAS mutant bladder cancer cell line UM-UC-3 (Oxford et al., 2005). While both RALA and RALB can support cellular growth, RALA inhibits and RALB drives cell motility and metastasis. This function of RALB is promoted by PKC-dependent phosphorylation of S198 of RALB, which is necessary for actin cytoskeletal organization, anchorage-independent proliferation,
migration and experimental metastasis of UM-UC-3 cells (Wang et al., 2010). The metastasis phenotype might be conveyed by RALBP1 as ablation of this RAL effector represses metastasis formation in the lung, while only minimally affecting subcutaneous tumor growth (Wu et al., 2010).

In line with the fact that RAS mutations are relatively infrequent in bladder cancer, reduced expression of RALGAPα2, the predominant RALGAPα isoform in bladder, was identified as an alternative cause of RAL hyperactivation (Saito et al., 2013). Homozygous knockout of RALGAPα2 leads to higher tumor incidence and enhances invasiveness, including metastasis formation in the lung, in a chemically induced mouse model of bladder cancer. Furthermore, decreased expression of RALGAPα2 correlates with advanced tumor stage and poor survival in human patients emphasizing clinical relevance of the RAL pathway in bladder cancer (Saito et al., 2013).

### 5.4 Breast cancer

An important step in breast cancer progression is the transition from estrogen-dependent to estrogen-independent growth (Clarke et al., 2003). RAL activity is required for this conversion and controls cell proliferation in cases where it is associated with overexpression of the EGF receptor (Yu and Feig, 2002). RALA and RALB expression has been shown to be elevated in breast cancer cells compared to neighboring normal cells, with RALB expression increasing further at sites of invasion and in metastases (Zago, 2018). Consistently, silencing of RALB decreases invasion of breast cancer cell lines. In addition, elevated levels of the RALGEF RCC2 are found in breast cancer and correlate with poor patient survival. *In vitro* and *in vivo* experiments link RCC2 to the regulation of cell proliferation and migration of breast cancer cells. Mechanistically, EMT was suggested to be activated by WNT signaling in RCC2 overexpressing cells (Chen et al., 2019a). Thus, a potential contribution of RAL signaling to this phenotype remains unaddressed.

### 5.5 Colorectal cancer

RALA and RALB are aberrantly activated in colorectal cancer cell lines regardless of their KRAS mutation status, suggesting additional RAS-independent activation mechanisms in this tumor type (Gyorffy et al., 2015; Martin et al., 2011). RAL GTPases are overexpressed in human tumor samples compared to paired healthy tissue and a RAL pathway-specific gene
signature, generated from three cell lines with stable RALA silencing, correlates with poor prognosis of colorectal cancer patients (Gyorffy et al., 2015; Martin et al., 2011). However, in cell lines, RALA and RALB seem to act antagonistically: while RALA promotes anchorage-independent proliferation, RALB was found to counteract it (Gyorffy et al., 2015; Martin et al., 2011). The RAL mutants D49E and D49N, which interfere with binding of exocyst subunits and RALBP1, respectively, were used to delineate the responsible RAL effectors (Martin et al., 2011). Both RAL GTPases rely on binding to the effector RALBP1, but interact with different components of the exocyst complex, providing a possible explanation for their differing functions (Martin et al., 2011; Martin and Der, 2012). RALA requires EXO84 for increased proliferation, whereas RALB utilizes SEC5 for inhibition of the same process. RALA function hereby seems to be dominant as concurrent knockdown of both GTPases prevents RALB-driven enhancement of anchorage-independent proliferation. Since knockdown of one RAL isoform increased mRNA levels of the other, compensatory mechanisms might contribute to complexity (Martin et al., 2011).

RALBP1 was independently described as a driver of colorectal cancer cell proliferation as depletion of RALBP1 blocks tumor growth in a SW480 xenograft mouse model (Singhal et al., 2007b). Furthermore, RALBP1 overexpression in tumor samples was identified as a prognostic marker for poor survival and early relapse in patients, suggesting relevance for human disease (Mollberg et al., 2012).

RCC2 has also been suggested as a prognostic marker in colorectal cancer and can drive migration and metastasis of colorectal cancer cells (Bruun et al., 2015; Song et al., 2018). The prognostic correlation of RCC2 expression hereby differs depending on the cancer phenotype: diminished RCC2 expression, due to a mutation in the 5′UTR region, is associated with improved survival of patients with microsatellite instable tumors. In contrast, in microsatellite stable cancer patients, reduced RCC2 expression correlates with poor prognosis (Bruun et al., 2015). Whether deregulated RAL activity is relevant in this context has not yet been investigated.

### 5.6 Colitis-associated cancer

The RAL signaling pathway has also been shown to promote colitis-associated cancer (CAC) (Iida et al., 2020). Expression of the RALGAP subunit RALGAPα2 is downregulated in CAC patient samples and lower

expression levels correlate with an invasive phenotype. Genetic ablation of RALGAPα2 leads to enhanced development of CAC in a chemically induced colitis-associated cancer mouse model and knockdown of RALGAPα2 in a CAC cell line enhances invasion. Mechanistically, the invasive phenotype has been attributed to increased matrix metalloproteinase-9 and matrix metalloproteinase-13 expression and activation of the NLRP3 inflammasome, elevating interleukin-1β levels, which might contribute to cancer-promoting inflammation *in vivo*.

## 5.7 Gastric cancer

The RALGEF RCC2 has been reported to be frequently overexpressed in gastric cancer. RCC2 knockdown dampens cell proliferation, migration and invasion *in vitro* and tumor growth of subcutaneous grafts, suggesting a tumor-promoting function for RCC2 in gastric cancer (Matsuo et al., 2013; Wang et al., 2020). The RAL inhibitor RBC8 impaired migration and invasion, suggesting that RCC2-driven RAL activity is relevant for the observed phenotype. Upregulation of RCC2 in gastric cancer might in part be mediated by downregulation of miRNA-29c, which can target RCC2 mRNA (Matsuo et al., 2013).

## 5.8 Hepatocellular carcinoma

Hepatocellular carcinoma (HCC) is the main type of primary liver cancer and one of the most common human malignancies. While RAS mutations are rather infrequent, overactivation of RAS has been observed in this tumor type and HRAS depletion shown to inhibit hepatocarcinogenesis (Teufel et al., 2007). Relevance of the RAL pathway in HCC has been provided by several studies (Ezzeldin et al., 2014; Kodama et al., 2016; Wang et al., 2009). In a HCC-developing mouse model, liver tumors present with elevated levels of active RAL (Ezzeldin et al., 2014). RALA activity is increased in HCC cell lines and malignant human tissue samples, which also showed overexpression of Aurora Kinase A, RALGDS and RALBPI (Ezzeldin et al., 2014; Wang et al., 2009). In CD133-expressing HCC stem cells, RAS, RALA and its activator Aurora Kinase A were hyperactivated, suggesting a potential role of the RAL pathway in HCC initiation (Ezzeldin et al., 2014). Upregulation of RCC2 was also detected in HCC cell lines and patient samples and correlated with pathological grade (Chen et al., 2020). Additional support for an important role of RAL signaling comes from a whole genome genetic screen in mice, which identified RALGAPα2 as a
candidate tumor suppressor gene in HCC (Kodama et al., 2016). Indeed, silencing of RALGAPα2, RALGAPα1 or RALGAPβ boosts tumor growth of grafted HCC cell lines *in vivo*. Despite the fact that inhibition of RAL signaling with RBC8 has only mild anti-proliferative effects on HCC cell lines *in vitro*, treatment of xenograft-bearing mice with RBC8 leads to a significant reduction in hepatocellular tumor growth. Importantly, simultaneous inhibition of the RAL and the RAF-MEK-ERK pathway by RBC8 and Sorafenib, respectively, exhibits synergistic effects resulting in almost complete abolition of tumor growth (Kodama et al., 2016). This highlights that RAL inhibition should be considered a relevant strategy for combination therapies and further effort is needed to clearly identify better targetable, tumor-relevant RAL downstream signaling events.

Inhibition of Aurora Kinase A can significantly diminish proliferation of HCC mice xenografts identifying Aurora kinase A as a potential therapeutic target in HCC (Benten et al., 2009; Ezzeldin et al., 2014). Additionally, invasion and viability of HCC cell lines declines *in vitro* upon treatment with the geranylgeranyl transferase inhibitor GGTI (Ezzeldin et al., 2014). While both of these inhibitors have been shown to affect RAL-GTP levels, they have pleiotropic cellular effects making clear conclusions concerning the relevance of RAL GTPases based on these results difficult.

### 5.9 Hematopoietic malignancies

In acute myeloid leukemia (AML) mutations in RAS GTPases are common and RAL GTPases mediate oncogenic signaling downstream of mutant NRAS or KRAS (Eckfeldt et al., 2016; Pomeroy and Eckfeldt, 2020). Eckfeldt and colleagues studied an AML transplantation mouse model, using cells that stably express the MLL-AF9 oncogene in combination with suppressible expression of NRAS<sup>G12V</sup>. Suppression of NRAS<sup>G12V</sup>, but not treatment with PI3K or MAPK inhibitors, induces apoptosis of the leukemic cells in this model, suggesting that the PI3K and MAPK effector pathways do not convey the survival signal downstream of mutated NRAS. While not tested *in vivo*, silencing of RALB in leukemic K562 cells leads to diminished TBK1 phosphorylation and a strong increase in apoptotic markers, suggesting that RALB-dependent TBK1 activation impacts cellular survival as had been previously described for lung cancer (Barbie et al., 2009). In support, TBK1 phosphorylation levels were found to be elevated in patient-derived AML blasts (Eckfeldt et al., 2016). A potential role of RALA was not addressed.
Besides, RALB might also play a role in leukemia development occurring independently of RAS mutations. In the above described NRAS<sup>G12V</sup>/MLL-AF9 AML mouse model, leukemia occasionally relapses after the suppression of NRAS<sup>G12V</sup> expression. Interestingly, relapsed cells show increased levels of RALB and enhanced TBK1 phosphorylation despite being independent of mutant NRAS signaling ([Pomeroy et al., 2017](https://doi.org/10.1038/nm.4365)). The CDK5 inhibitor dinaciclib, which has previously been suggested to act through inhibition of RALB ([Feldmann et al., 2010](https://doi.org/10.1038/onc.2010.35)), is able to prevent leukemic relapse. Dinaciclib decreases RALB-GTP level and TBK1 phosphorylation and induces apoptosis of AML cell lines and primary AML patient samples. Furthermore, dinaciclib treatment reduces leukemic burden in patient-derived AML xenografts ([Pomeroy et al., 2017](https://doi.org/10.1038/nm.4365)). Whether dinaciclib action can be fully attributed to reduction of RAL-GTP levels in this context remains to be further investigated.

RAL GTPases were also identified to convey survival of multiple myeloma (MM) cells ([Seibold, 2019](https://doi.org/10.1038/s41590-019-0390-0)). RAL activation occurs independently of RAS mutation and is unaffected by oncogenic RAS silencing. In primary MM patient samples, RALA is aberrantly expressed and depletion of RALA and to a lesser extent of RALB reduces cell viability *in vitro*. This effect might be mediated by the exocyst since EXO84 and SEC5 (but not RALBP1) were identified as RALA interaction partners in mass spectrometry analyses. Thus in contrast to AML, RALA seems to be more critical for cellular viability in MM than RALB. Notably, RALA but not RALB knockdown results in decreased AKT phosphorylation. In line, RALB but not RALA silencing in combination with chemical inhibition of the PI3K pathway displays synergistic effects on cell viability *in vitro*, suggesting that crosstalk between RALA and PI3K pathways might exist ([Seibold, 2019](https://doi.org/10.1038/s41590-019-0390-0)). While less important for cellular viability, RALB has been reported to be important for migration of MM cells in response to the chemokine stromal cell-derived factor-1 (SDF-1) ([de Gorter et al., 2008](https://doi.org/10.1038/leu.2008.13)). Stimulation with SDF-1 results in rapid activation of RALB and promotes migration of OPM-1 and NCI-H929 cell lines as well as primary patient-derived MM cells. Induction or RALB activity by SDF-1 was found to be independent from RAS, and RAL signaling necessary but not sufficient for migration.

In chronic myelogenous leukemia (CML), enhanced RALA activity was reported to occur commonly and to be required for transformation, progression and survival of cells, ([Gu et al., 2016](https://doi.org/10.1182/blood-2015-09-670883); [Zhu et al., 2012](https://doi.org/10.1182/blood-2011-10-388588)). Here, RALA has been suggested as a target of microRNA 181a (miRNA-181a), which is
differentially expressed in various leukemia types. Silencing of RALA mimics effects of miR-181a expression in respect to inhibition of cell cycle progression and promotion of apoptosis of K562 cells, suggesting that effects of miR-181a might at least be partially attributable to deregulation of RALA (Fei et al., 2012).

In addition, mutations in and fusions with the RALGEF RGL4/RGR have been connected to T-cell malignancies, but since RGR also functions as a GEF toward RAS the precise implications of the RAL pathway remain to be elucidated (Jiménez et al., 2004; Leonardi et al., 2002; Osei-Sarfo et al., 2011).

### 5.10 Malignant peripheral nerve sheath tumors

Malignant peripheral nerve sheath tumors (MPNST) are aggressive soft tissue sarcomas of neural origin. About 50 % of cases are linked to loss of the RASGAP neurofibromatosis–1 (NF-1), which results in hyperactivation of RAS (Farid et al., 2014). RALA activity was found to be high in malignant peripheral nerve sheath tumor mouse cell lines, including cell lines in which RAS-GTP levels were normal (Bodempudi et al., 2009). Functional experiments suggest an influence of RALA on invasion and proliferation of cell lines and xenografts. Here, silencing of RALA results in reduced expression of EMT marker proteins, providing a potential explanation for the observed altered invasion capabilities (Bodempudi et al., 2009).

### 5.11 Medulloblastoma

To date, a single study addressed RAL signaling in medulloblastoma, a mainly pediatric brain cancer (Ginn et al., 2016). RALA was found hyperactivated in medulloblastoma cell lines and overexpressed in a panel of human tumor samples. In *in vitro* experiments RALA promotes tumor proliferation and invasiveness, and CD133-positive medulloblastoma stem cells show elevated levels of RALA activation. A potential role of RALB in this tumor entity was not investigated.

### 5.12 Melanoma

About 30 % of melanomas harbor NRAS and 60 % BRAF mutations, which are mostly mutually exclusive (Brose et al., 2002). Investigation of the importance of different RAS effector pathways in immortalized murine melanocytes deficient for p19/ARF revealed that ectopic expression of membrane-targeted RGL2 (Rlf-CAAX) best mimics the morphological
changes induced by constitutively active NRAS<sup>Q61K</sup> and drives both anchorage-independent proliferation and invasion (Mishra et al., 2010). This might be due to RGL2-dependent activation of RALA, which is aberrantly activated in several melanoma cell lines irrespectively of the mutation status of NRAS and BRAF. Indeed, silencing of RALA and to a lesser extent of RALB impairs tumor growth of melanoma cell line xenografts (Zipfel et al., 2010). These findings suggest that RAL signaling can be relevant in melanomas with or without RAS mutations. Additionally, RALBP1 has been found to be overexpressed in malignant melanoma cells. Inhibition or depletion of RALBP1 results in increased apoptosis *in vitro* and strongly reduced tumor proliferation in a xenograft mouse model (Singhal et al., 2006).

### 5.13 Oral squamous cell carcinoma

Oral squamous cell carcinoma (OSCC) is the most common oral malignancy and has generally low RAS mutation rates (Zanaruddin et al., 2013). In OSCC, RALGAPα2 was found downregulated in patient tumor samples relative to healthy tissue and low expression inversely correlated with survival (Gao et al., 2019). Low mRNA and protein levels of RALGAPα1, RALGAPα2, and RALGAPβ correlate with increased activity of RALA and RALB and enhanced invasion of OSCC cell lines. Reintroduction of RALGAPα2 is able to dampen migration and invasion in these cell lines, whereas anchorage-independent and -dependent proliferation are not affected. Combined treatment of cells with DNA methyltransferase, histone methyltransferase, and histone deacetylase inhibitors can restore RALGAPα2 mRNA levels, suggesting altered histone modifications (H4Ac, H3K9me2, and H3K27me3) in the RALGAPα2 promoter region in combination with DNA methylation as mechanisms downregulating RALGAPα2 mRNA expression (Gao et al., 2019). Thus, epigenetic mechanisms might at least in part promote OSCC by reducing RALGAPα2 expression and consequently enhancing RAL-GTP level.

### 5.14 Ovarian cancer

The frequently occurring downregulation of microRNA miR-33-3p in ovarian cancer has been described as reason for upregulation of RCC2 levels in this tumor type (Buranjiang et al., 2019). RCC2 was found to be overexpressed in cisplatin-resistant ovarian cancer cell lines and tissues alongside with RALA and RALBP1, and high RCC2 levels were associated
with poor prognosis (Gong et al., 2019). RCC2 overexpression promotes cancer cell proliferation and migration and inhibits apoptosis, effects that could be counteracted by knockdown of RALA, demonstrating that RCC2 exerts its role through mobilization of RAL signaling.

### 5.15 Prostate cancer

In prostate cancer, RAL signaling seems to be involved in migration and metastasis rather than proliferation of tumor cells. RALA is associated with the development of a highly metastatic prostate cancer stage after androgen depletion, where reactive oxygen species activate RALA, resulting in augmented VEGF-C (vascular endothelial growth factor-C) synthesis (Rinaldo et al., 2007). Consistently, activation of the RAS/RALGEF/RAL pathway in the prostate cancer cell line DU145 enables bone metastasis and ablation of RALA diminishes bone metastasis formation by the prostate cancer cell line PC3 (Yin et al., 2007). Interestingly, activation of RAF/ERK signaling promotes colonization of the brain after injection of tumor cells. These findings indicate a role for RAL signaling in organ-specific metastasis with RALBP1 identified as RAL effector mediating this process (Wu et al., 2010). Similar observations of increased migration and invasion with unaffected cell proliferation were made for the knockdown of RALGAPα2 in prostate cancer cell lines (Uegaki et al., 2019). In line, invasion, but not tumor weight, is enhanced upon RALGAPα2 knockout in a prostate cancer mouse model driven by chemically induced, prostate-specific ablation of the tumor suppressor PTEN (phosphatase and tensin homolog deleted on chromosome 1) (Ratnacaram et al., 2008; Uegaki et al., 2019). Diminished expression of RALGAPα2 in prostate cancer patient samples, with stronger downregulation in malignant than benign tumor samples, and high expression of a RAL-dependent transcriptional signature, which correlates with decreased survival, androgen-independent progression and seminal vesicle invasion, emphasizes the clinical relevance of RALGAP-control on RAL activity in this tumor entity (Smith et al., 2012; Uegaki et al., 2019). Aurora kinase A is also overexpressed in prostate cancer with a metastatic phenotype and might be involved in driving RAL activity (Varambally et al., 2005).

### 5.16 Squamous cell carcinoma

The effect of RAL signaling in squamous cell carcinoma (SCC) represents a matter of debate. In a bioengineered 3D tissue model of RAS-driven SCC
of the skin, RALA but not RALB decrease leads to diminished E-cadherin expression, possibly mediated by EXO84, and progression to a malignant phenotype (Sowalsky et al., 2010). A lower RAL transcriptional signature (derived from studies in bladder cancer cells) in SCC compared to normal mucosa further supports an inhibitory function of the RAL pathway in this tumor type (Smith et al., 2012).

In contrast, tumor formation in a chemically induced (DMBA-TPA) skin carcinogenesis mouse model with high incidence of HRAS mutations is not altered by single knockout of RALA nor of RALB (Peschard et al., 2012). The same study however reported that also for KRAS<sup>G12D</sup>-driven lung adenocarcinoma formation single knockout of RALA or RALB is without effect, while double deficiency almost completely abrogates tumor development. It thus remains unclear what the consequences of complete RAL deficiency would be for SCC formation *in vivo*. Knockout of RALGDS results in reduced tumor appearance, smaller size of tumors and decreased malignant progression in the same SCC carcinogenesis model (González-García et al., 2005). This phenotype is associated with evasion from apoptosis through promotion of the JNK/SAPK pathway rather than diminished cell proliferation. While it is possible that the RALGDS knock-out animals reveal the phenotype of lack of activity of both RAL GTPases, a contribution from RALGDS-controlled but RAL-independent signaling cannot be excluded.

### 6. RALopathies

RASopathies are a clinically defined group of genetic syndromes caused by mutations in the RAS/MAPK pathway (Rauen, 2013). In the style of this nomenclature we will summarize currently known genetic syndromes linked to RAL GTPases under the term RALopathies in this chapter.

Hiatt and colleagues identified a set of patients with developmental disabilities, including intellectual disability, delayed or absent speech and impaired motor skills, that all harbor mutations in *RALA* (Hiatt et al., 2018). All RAL mutations are located in the nucleotide-binding G1, G4 or G5 motifs, with V25 (G1) and K128 (G4) being the most prominently targeted residues, and result in reduced effector binding possibly through impaired nucleotide affinity. According to the authors in 6 out of 11 patients, the *RALA* variant was the only plausible variation that might be pathogenic. A separate study identified an additional patient with a V25M mutation in *RALA*, who presented with similar symptoms (Okamoto et al., 2019).

Prior to its characterization as a GAP protein, the RALGAPα1 subunit had been identified and cloned as a brain-expressed candidate gene for neurological deficits of a patient with microcephaly, abnormal brain development, psychomotor retardation, and other neurological features associated with 14q chromosomal deletions (Schwarzbraun et al., 2004). An independent study also suggests RALGAPα1 haploinsufficiency due to a 14q microdeletion to be the relevant factor in a patient with developmental delay and intractable epilepsy. Furthermore, bi-allelic variants of *RALGAPA1* in four neuropediatric patients with intellectual disabilities and infantile spasms were recently described (Wagner et al., 2020). As in the patients with *RALA* variants, finger abnormalities, muscular hypotonia, epilepsy, and characteristic facial abnormalities were present besides neurological symptoms. RALGAPα1 was absent in fibroblasts derived from the respective patients, suggesting a loss of function effect which results in increased RAL-GTP levels. While the similarities between the phenotypes described in patients with *RALA* variants or RALGAPα1 haploinsufficiency are striking and support the notion that deregulation of RALA signaling may have an impact on neuronal function and brain development, it remains to be clarified whether and how decreased effector binding (as described for *RALA* variant patients) and enhanced RAL-GTP levels (due to RALGAPα1 loss) might result in similar downstream developmental effects. While no RALGAPα1 knockout mouse models have been described to date, deficiency of RALGAPα2 did not result in apparent abnormalities (Saito et al., 2013). However, RALGAPα1 expression is specifically high in brain in comparison to RALGAPα2, so that one might hypothesize that specific and non-redundant functions of RALGAPα1 in neuronal development and function might exist. Noteworthy, knockdown of a zebrafish homolog of RALGAPα1 leads to brain developmental delay (Shimojima et al., 2009). To determine the exact role(s) of the RALGAPα1/RALA signaling module in neurodevelopment in the future, the generation and characterization of respective knockout and knock-in animal models will likely be indispensable.

---

### 7. Inhibition of RAL activity

In recent years different strategies have been pursued to inhibit the activity of RAL GTPases (Fig. 12), which—like all small GTPases—are not simple targets for pharmacological interference. Small GTPases possess high affinity for guanine nucleotides in the picomolar range, while cellular

Fig. 12 Strategies to directly inhibit RAL GTPases. The two geranylgeranyl transferase I inhibitors GGTI-2417 and -2418 prevent association of RAL GTPases with the plasma membrane and thus prohibit activation. With stapled peptides the interaction of RAL with effectors is competitively inhibited. The small molecules RBC8 and its derivative BQU57 lock RAL GTPases in their GDP-bound state and covalent binding of an aryl sulfonyl fluoride, 4-(N-(5-methoxypyridin-3-yl)sulfamoyl)benzene-1-sulfonyl fluoride prevents GEF-mediated nucleotide exchange.

GDP/GTP concentrations are millimolar. This makes strategies to compete with nucleotide-binding unfeasible (Cox et al., 2014). In addition, structural analyses show that the surface of RAL-GTP is rather smooth without prominent pockets that might be able to accommodate small molecule inhibitors. The lack of RAL mutations however represents a theoretical advantage for additional approaches to target the pathway as the wildtype RAL proteins should be susceptible to manipulation of regulators of their activity, e.g., RALGEFs. GEF inhibitor design has been successful in some cases, e.g., for ARF, RHO and RAC1, but it is generally challenging as it requires targeting of protein-protein interactions (Gray et al., 2020). For RAL
GTPases this is further complicated by the limited structural information on RALGEFs. The same holds true for RAL effector signaling, where interference with protein-protein interactions so far represents the main strategy of attack, and very few easily targetable mediators such as kinases have been identified to date. Targeting of Aurora kinase A or TBK1 however are noteworthy options (Oral et al., 2017; Reilly et al., 2013; Yan et al., 2016).

The best-studied approach to inhibit RAL activity prevents RAL plasma membrane localization by use of geranylgeranyl transferase I inhibitors (GGTIs) (Fig. 12) (Berndt et al., 2011). Application of different small molecule GGTIs results in reduced tumor growth of xenografts (Kazi et al., 2009; Lu et al., 2009; Sun et al., 1998, 1999) and tumor regression in transgenic mice (Kazi et al., 2009; Sun et al., 2003). The anti-tumor effects of GGTI-2417 were hereby directly linked to hampering of RAL GTPase membrane association. GGTI-2417-triggered reduction of RALA activity results in decreased anchorage-independent proliferation and reduction of RALB activity in abrogation of anchorage-dependent growth and survival of the pancreatic cancer cell line MiaPaCa-2 (Falsetti et al., 2007). Furthermore, diminished membrane localization of RALB, but not RALA, has been attributed to migration and invasion inhibition effects of GGTI-298 in OSCC cell lines (Hamada et al., 2011). In agreement, knockdown of RALB reduced the increased migration and invasion of glioma cells overexpressing geranylgeranyl transferase β (GGTβ) (Song et al., 2015). GGTIs were also shown to reduce RALA-GTP levels and cellular proliferation of medulloblastoma, HCC and MPNST cells; however, evidence for causality of RAL deregulation for this phenotype has not yet been provided (Bodempudi et al., 2009; Ezzeldin et al., 2014; Ginn et al., 2016).

Since most small RAS-like GTPases harbor the C-terminal Caax motif that is targeted by geranylgeranylation, GGTIs are likely to have pleiotropic effects so that general toxicity might reasonably be expected. However, regression of breast tumor xenografts has been achieved using a subcutaneous pump administering GGTI-2418 in preclinical trials with mice without significant toxicity (Kazi et al., 2009). Importantly, GGTI-2418 (PTX-100) was well tolerated in a phase I clinical trial assessing dose-limiting toxicity, safety and tolerability of the inhibitor (Karasic et al., 2019). Dosage and frequency of GGTI-2418 administration are currently further evaluated in an ongoing phase I clinical trial (NCT03900442) to overcome issues of short half-life. Therefore, GGTIs might still represent a potential approach to pharmacologically interfere with RAL signaling.
Other strategies aim at targeting RAL by direct binding. Based on the structure of the RAL-RALBP1 complex, α-helical-stapled peptides were designed that selectively bind GTP-bound RALA and RALB and thus compete with effector binding ([Thomas et al., 2016](https://doi.org/10.1038/nchembio.2175)). The peptides resemble two α-helices of RALBP1 that engage in interaction with the switch regions of RAL GTPases. In cells, the stapled peptides disturb interaction of endogenous RALB with SEC5 and inhibit RALB- and EXO84-dependent autophagosome formation upon nutrient starvation ([Thomas et al., 2016](https://doi.org/10.1038/nchembio.2175)). The described peptides enter HEK293T cells, distribute throughout the cytosol as well as the nucleus and are active for at least 24 h, suggesting that they might indeed have therapeutic value.

A second structure-based approach is focused on small molecules selectively binding to GDP- and not GTP-bound RAL with the idea that these substances might lock RAL in its inactive state. A virtual screening approach was employed to predict small molecules that would be able to occupy an area in close proximity to the nucleotide-binding pocket that is formed by the switch II and α2 and α3 helices ([Yan et al., 2014](https://doi.org/10.1038/nchembio.1635)). This screen identified the compounds RBC6, RBC8 and RBC10, all of which decrease RAL activity in a dose-dependent fashion and interfere with RAL binding to RALBP1. Further optimization trials led to characterization of the derivative compound BQU57 with superior performance and drug properties. Administration of RBC8 and its derivative BQU57 reduces anchorage-independent proliferation of lung cancer cell lines *in vitro* and dampens growth of respective xenografts *in vivo* ([Yan et al., 2014](https://doi.org/10.1038/nchembio.1635)). These inhibitors thus represent a promising starting point for further medicinal lead optimization and efficacy testing, which has just recently been licensed to Machavert Pharmaceuticals. Besides, RBC8 and BQU57 have already proven valuable experimental tools to scientifically investigate the RAL pathway. Successful inhibition of RAL activation and/or downstream pathways *in vitro* has been reported for prostate cancer ([Uegaki et al., 2019](https://doi.org/10.1038/s41467-019-10118-8)), chronic myelogenous leukemia ([Gu et al., 2016](https://doi.org/10.1038/onc.2016.20)), colorectal cancer ([Gyorffy et al., 2015](https://doi.org/10.1038/onc.2015.20)), and pancreatic cancer cells ([Beel, 2020](https://doi.org/10.1038/s41467-020-16008-0)) as well as in primary acini, the acinar cell line AR42J ([Beel, 2020](https://doi.org/10.1038/s41467-020-16008-0)), and 3T3-L1 adipocytes ([Skorobogato et al., 2018](https://doi.org/10.1038/s41467-018-05108-2)). Effective concentrations ranged from 1–5 μM ([Gu et al., 2016](https://doi.org/10.1038/onc.2016.20)) to 50 μM ([Skorobogato et al., 2018](https://doi.org/10.1038/s41467-018-05108-2); [Uegaki et al., 2019](https://doi.org/10.1038/s41467-019-10118-8)) for RBC8 and 1–100 μM for BQU57. *In vivo* efficacy of RBC8 or BQU57 treatment was shown for lung cancer xenografts ([Yan et al., 2014](https://doi.org/10.1038/nchembio.1635)), hepatocellular carcinoma xenografts ([Kodama et al., 2016](https://doi.org/10.1038/onc.2016.20)) and insulin-stimulated glucose uptake in mice ([Skorobogato et al., 2018](https://doi.org/10.1038/s41467-018-05108-2)).
Hereby, RBC8 was administered by intraperitoneal injection (Skorobogatko et al., 2018; Yan et al., 2014) or by oral gavage (Kodama et al., 2016) in a concentration of 50mg/kg bodyweight. It is noteworthy that, while the original characterization demonstrated that RBC8 and BQU57 act specifically on RALA and RALB but not on RAS or RHOA (Yan et al., 2014), RBC8 was shown to interfere with functions of platelets derived from RALA/B platelet-specific double knockout mice, thus raising concerns about potential uncharacterized off-target activities (Walsh et al., 2019).

An alternative approach furthermore revealed that aryl sulfonyl fluorides can covalently attach to Y82 of RALA and RALB and irreversibly prevent RGL2-conveyed RAL nucleotide exchange (Bum-Erdene et al., 2020). For a potential use *in vivo*, these inhibitors would however require significant optimization to prevent hydrolysis, increase overall stability and improve binding affinity. Nevertheless, structural analysis showed that the compounds are embedded in a pocket exhibiting physico-chemical properties that might be suitable for small molecule binding, a finding which may fuel new efforts of structure-guided drug design.

### 8. Conclusion

The RAL GTPase signaling network controls central physiological and pathological/neoplastic functions. Research of the last 34 years has taught us a lot and clearly established the RALGEF/RAL axis as the third main effector pathway in RAS-driven carcinogenesis. Nevertheless, many pieces of the RAL puzzle are still missing, leaving us with plenty of exciting questions to address.

One key aspect is how specificity is achieved in incoming and outgoing signals of RALA and RALB. Is this primarily due to distinct spatial localizations of the players involved? And if so, what are the regulatory mechanisms determining these subcellular localizations and how are they employed to tune signaling outcome downstream of RAL? In this context, it will be crucial to obtain a more detailed picture of RALA and/or RALB activation and inactivation by distinct GEFs and GAPS in different cellular scenarios. Especially for the more recently identified RALGAP complexes, future research is likely to identify additional regulatory events.

In light of the significant research interest in RAS-driven cancer, there is also surprisingly little information about the molecular mechanisms and cellular processes that RAL GTPases use to support tumor initiation, progression and maintenance in different tumor entities. RAL GTPase signaling
might represent an example of a molecular hub orchestrating diverse tumor-relevant processes that could exactly for that reason become an Achilles’ heel in cancer biology and should thus be considered a highly relevant pharmacological target. However, a more comprehensive mechanistic knowledge is a prerequisite for successful therapeutic approaches to interfere with RAL signaling and to potentially identify better targetable mediators. In addition, a better understanding of the regulation and role of RAL GTPases in RAS-independent carcinogenesis is required. Conditional genetic ablation of RAL GTPases, their regulators and effectors combined with thorough cell biological studies and screening efforts will likely be the driving force in this respect.

It will also be important to further explore the role of RAL in orchestrating exocytic and endocytic processes as these can be linked to biologically important tasks, including neuronal and immune cell functions as well as metabolism homeostasis. Thus, deregulation of RAL activity might very well be relevant for human disease beyond neoplasia, an aspect that needs to be better understood and taken into account when designing therapeutic strategies. Thus, RAL GTPases should be considered an autonomous signaling module with original regulation and widespread cellular impact.

# Acknowledgments

We wish to acknowledge Prof. Michael Meisterernst for his support and thank Stephanie Beel and Prof. Daniel Kümmel for critical reading of the manuscript. This work was supported by grants of the German Research Foundation (DFG; OE531/2-2) and the Innovative Medical Research (IMF) Initiative of the University of Münster (OE121701) to A.O. Amino acid alignments were conducted with Clustal Omega (Sievers et al., 2011) and figures were created with BioRender.com.

# References

Abounit, S., Zurzolo, C., 2012. Wiring through tunneling nanotubes—from electrical signals to organelle transfer. J. Cell Sci. 125 (5), 1089–1098. [https://doi.org/10.1242/JCS.083279](https://doi.org/10.1242/JCS.083279). The Company of Biologists Ltd.

Alarcon-Martinez, L., et al., 2020. Interpericyte tunnelling nanotubes regulate neurovascular coupling. Nature, 1–5. [https://doi.org/10.1038/s41586-020-2589-x](https://doi.org/10.1038/s41586-020-2589-x). Nature Publishing Group.

Albright, C.F., et al., 1993. Characterization of a guanine nucleotide dissociation stimulator for a ras-related GTPase. EMBO J. 12 (1), 339–347. [https://doi.org/10.1002/j.1460-2075.1993.tb05662.x](https://doi.org/10.1002/j.1460-2075.1993.tb05662.x). John Wiley & Sons, Ltd.

Armenti, S.T., Chan, E., Nance, J., 2014. Polarized exocyst-mediated vesicle fusion directs intracellular lumenogenesis within the C. elegans excretory cell. Dev. Biol. 394 (1), 110–121. [https://doi.org/10.1016/j.ydbio.2014.07.019](https://doi.org/10.1016/j.ydbio.2014.07.019). Dev. Biol.

Aspenström, P., 2018. Fast-cycling Rho GTPases. Small GTPases, 1–8. [https://doi.org/10.1080/21541248.2017.1391365](https://doi.org/10.1080/21541248.2017.1391365). Taylor & Francis.

Awasthi, S., et al., 2000. Novel function of human RLIP76: ATP-dependent transport of glutathione conjugates and doxorubicin. Biochemistry 39 (31), 9327–9334. https://doi.org/10.1021/bi992964c. Biochemistry.

Awasthi, S., et al., 2008. RLIP76 and cancer. Clin. Cancer Res. 14 (14), 4372–4377. https://doi.org/10.1158/1078-0432.CCR-08-0145. American Association for Cancer Research.

Awasthi, S., et al., 2018. Rlip depletion prevents spontaneous neoplasia in TP53 null mice. Proc. Natl. Acad. Sci. U. S. A. 115 (15), 3918–3923. https://doi.org/10.1073/pnas.1719586115. National Academy of Sciences.

Bailey, P., et al., 2016. Genomic analyses identify molecular subtypes of pancreatic cancer. Nature 531 (7592), 47–52. https://doi.org/10.1038/nature16965. Nature Publishing Group.

Balakireva, M., et al., 2006. The Ral/exocyst effector complex counters c-Jun N-terminal kinase-dependent apoptosis in Drosophila melanogaster. Mol. Cell Biol. 26 (23), 8953–8963. https://doi.org/10.1128/MCB.00506-06. American Society for Microbiology Journals.

Barbie, D.A., et al., 2009. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Nature 462 (7269), 108–112. https://doi.org/10.1038/nature08460. Nature.

Beel, S., et al., 2020. κB-Ras and Ral GTPases regulate acinar to ductal metaplasia during pancreatic adenocarcinoma development and pancreatitis. Nat. Commun. 11. https://doi.org/10.1038/s41467-020-17226-0, 3409.

Benten, D., et al., 2009. Aurora kinase inhibitor PHA-739358 suppresses growth of hepatocellular carcinoma in vitro and in a xenograft mouse model. Neoplasia (New York, NY) 11 (9), 934–944. https://doi.org/10.1593/neo.09664. Neoplasia Press.

Berndt, N., Hamilton, A.D., Sebti, S.M., 2011. Targeting protein prenylation for cancer therapy. Nat. Rev. Cancer 11 (11), 775–791. https://doi.org/10.1038/nrc3151. Nature Publishing Group.

Bhattacharya, M., et al., 2002. β-Arrestins regulate a Ral-GDS-Ral effector pathway that mediates cytoskeletal reorganization. Nat. Cell Biol. 4 (8), 547–555. https://doi.org/10.1038/ncb821. Nature Publishing Group.

Bhattacharya, M., et al., 2004. Class I metabotropic glutamate receptors at GABAergic synapses in monkeys. J. Neurosci. 19 (15), 6488–6496. https://doi.org/10.1523/JNEUROSCI.3155-04.2004. Society for Neuroscience.

Biondini, M., et al., 2016. Direct interaction between exocyst and Wave complexes promotes cell protrusions and motility. J. Cell Sci. 129 (20), 3756–3769. https://doi.org/10.1242/jcs.187336. The Company of Biologists Ltd.

Bodemann, B.O., White, M.A., 2008. Ral GTPases and cancer: linchpin support of the tumorigenic platform. Nat. Rev. Cancer 8 (2), 133–140. https://doi.org/10.1038/nrc2296. Nature Publishing Group.

Bodemann, B.O., et al., 2011. RalB and the exocyst mediate the cellular starvation response by direct activation of autophagosome assembly. Cell 144 (2), 253–267. https://doi.org/10.1016/j.cell.2010.12.018.

Bodempudi, V., et al., 2009. Ral overactivation in malignant peripheral nerve sheath tumors. Mol. Cell Biol. 29 (14), 3964–3974. https://doi.org/10.1128/MCB.01153-08. American Society for Microbiology (ASM).

Borrego-Diaz, E., et al., 2012. Overactivation of Ras signaling pathway in CD133+ MPNST cells. J. Neurooncol. 108 (3), 423–434. https://doi.org/10.1007/s11060-012-0852-1. Springer.

Bourne, H.R., Sanders, D.A., McCormick, F., 1991. The GTPase superfamily: conserved structure and molecular mechanism. Nature 349 (6305), 117–127. https://doi.org/10.1038/349117a0. Nature Publishing Group.

Boyartchuk, V.L., Ashby, M.N., Rine, J., 1997. Modulation of ras and α-factor function by carboxyl-terminal proteolysis. Science 275 (5307), 1796–1800. https://doi.org/10.1126/science.275.5307.1796. American Association for the Advancement of Science.

Brose, M.S., et al., 2002. Relative reciprocity of NRAS and PTEN/MMAC1 alterations in cutaneous melanoma cell lines. Cancer Res. 60 (7), 1800–1804. American Association for Cancer Research. Available at: https://cancerres.aacrjournals.org/content/62/23/6997. (Accessed 6 July 2020).

Bruun, J., et al., 2015. Regulator of chromosome condensation 2 identifies high-risk patients within both major phenotypes of colorectal cancer. Clin. Cancer Res. 21 (16). https://doi.org/10.1158/1078-0432.CCR-14-3294.

Bum-Erdene, K., et al., 2020. Small-molecule covalent bond formation at tyrosine creates a binding site and inhibits activation of Ral GTPases. Proc. Natl. Acad. Sci. U. S. A. 117 (13), 7131–7139. https://doi.org/10.1073/pnas.1913654117.

Buranjiang, G., et al., 2019. MicroRNA-331-3p inhibits proliferation and metastasis of ovarian cancer by targeting RCC2. Arch. Med. Sci. 15 (6), 1520–1529. https://doi.org/10.5114/aoms.2018.77858.

Cai, S.-L., et al., 2006. Activity of TSC2 is inhibited by AKT-mediated phosphorylation and membrane partitioning. J. Cell Biol. 173 (2), 279. https://doi.org/10.1083/JCB.200507119. The Rockefeller University Press.

Cantor, S.B., Urano, T., Feig, L.A., 1995. Identification and characterization of Ral-binding protein 1, a potential downstream target of Ral GTPases. Mol. Cell Biol. 15 (8), 4578–4584. https://doi.org/10.1128/mcb.15.8.4578. American Society for Microbiology Journals.

Cascone, I., et al., 2008. Distinct roles of RalA and RalB in the progression of cytokinesis are supported by distinct RalGEFs. EMBO J. 27 (18), 2375–2387. https://doi.org/10.1038/emboj.2008.166. European Molecular Biology Organization.

Casey, P.J., Thissen, J.A., Moomaw, J.F., 1991. Enzymatic modification of proteins with a geranylgeranyl isoprenoid. Proc. Natl. Acad. Sci. U. S. A. 88 (19), 8631–8635. https://doi.org/10.1073/pnas.88.19.8631.

Ceriani, M., et al., 2007. Functional analysis of RalGPS2, a murine guanine nucleotide exchange factor for RalA GTPase. Exp. Cell Res. 313 (11), 2293–2307. https://doi.org/10.1016/J.YEXCR.2007.03.016. Academic Press.

Chang, E.H., et al., 1982. Human genome contains four genes homologous to transforming genes of Harvey and Kirsten murine sarcoma viruses. Proc. Natl. Acad. Sci. U. S. A. 79 (16), 4848. https://doi.org/10.1073/PNAS.79.16.4848. National Academy of Sciences.

Chardin, P., Tavitian, A., 1986. The ral gene: a new ras related gene isolated by the use of a synthetic probe. EMBO J. 5 (9), 2203–2208. https://doi.org/10.1002/J.1460-2075.1986.TB04485.X. Wiley-Blackwell.

Chardin, P., Tavitian, A., 1989. Coding sequences of human ralA and ralB cDNAs. Nucleic Acids Res. 17 (11), 4380. Oxford University Press.

Chen, Y., Wu, J., Ghosh, G., 2003. κB-Ras binds to the unique insert within the ankyrin repeat domain of IkBβ and regulates cytoplasmic retention of IkBβ·NF-κB complexes. J. Biol. Chem. 278 (25), 23101–23106. https://doi.org/10.1074/jbc.M301021200. Department of Chemistry and Biochemistry, University of California San Diego, La Jolla, California 92093-0359, USA.

Chen, Y., et al., 2004. Inhibition of NF-κB activity by IkBβ in association with κB-Ras. Mol. Cell Biol. 24 (7), 3048–3056. https://doi.org/10.1128/mcb.24.7.3048-3056.2004. Department of Chemistry and Biochemistry, University of California-San Diego, La Jolla, California 92093, USA.

Chen, X.W., et al., 2006. RalA-exocyst-dependent recycling endosome trafficking is required for the completion of cytokinesis. J. Biol. Chem. 281 (50), 38609–38616. https://doi.org/10.1074/jbc.M512847200.

Chen, X.W., et al., 2007. Activation of RalA is required for insulin-stimulated Glut4 trafficking to the plasma membrane via the exocyst and the motor protein Myo1c. Dev. Cell 13 (3), 391–404. https://doi.org/10.1016/j.devcel.2007.07.007.

Chen, C.X.-J., et al., 2011a. Control of secondary granule release in neutrophils by Ral GTPase. J. Biol. Chem. 286 (13), 11724–11733. https://doi.org/10.1074/jbc.M110.154203. American Society for Biochemistry and Molecular Biology.

Chen, X.W., Leto, D., Xiong, T., et al., 2011b. A Ral GAP complex links PI 3-kinase/Akt signaling to RalA activation in insulin action. Mol. Biol. Cell 22 (1), 141–152. https://doi.org/10.1091/mbc.E10-08-0665. Edited by T. F. J. Martin.

Chen, X.W., Leto, D., Xiao, J., et al., 2011c. Exocyst function regulated by effector phosphorylation. Nat. Cell Biol. 13 (5), 580–588. https://doi.org/10.1038/ncb2226.

Chen, Q., et al., 2014. GARNL1, a major RalGAP α subunit in skeletal muscle, regulates insulin-stimulated RalA activation and GLUT4 trafficking via interaction with 14-3-3 proteins. Cell. Signal. 26 (8), 1636–1648. https://doi.org/10.1016/j.cellsig.2014.04.012. Elsevier Inc.

Chen, Q., et al., 2019a. Targeting RalGAPα1 in skeletal muscle to simultaneously improve postprandial glucose and lipid control. Sci. Adv. 5 (4), eaav4116. https://doi.org/10.1126/sciadv.aav4116. American Association for the Advancement of Science.

Chen, Z., et al., 2019b. RCC2 promotes breast cancer progression through regulation of Wnt signaling and inducing EMT. J. Cancer 10 (27), 6837–6847. https://doi.org/10.7150/jca.36430.

Chen, Q., et al., 2020. RCC2 contributes to tumor invasion and chemoresistance to cisplatin in hepatocellular carcinoma. Hum. Cell 33 (3), 709–720. https://doi.org/10.1007/s13577-020-00353-7. Springer.

Cherfils, J., Zeghouf, M., 2013. Regulation of small GTPases by GEFs, GAPs, and GDIs. Physiol. Rev. 93 (1), 269–309. https://doi.org/10.1152/physrev.00003.2012. American Physiological Society Bethesda, MD.

Chien, Y., White, M.A., 2003. RAL GTPases are linchpin modulators of human tumour-cell proliferation and survival. EMBO Rep. 4 (8), 800–806. https://doi.org/10.1038/sj.emborep.embor.embor899. EMBO Press.

Chien, Y., et al., 2006. RalB GTPase-mediated activation of the IkB family kinase TBK1 couples innate immune signaling to tumor cell survival. Cell 127 (1), 157–170. https://doi.org/10.1016/J.CELL.2006.08.034. Cell Press.

Cho, B., Fischer, J.A., 2011. Ral GTPase promotes asymmetric Notch activation in the Drosophila eye in response to Frizzled/PCP signaling by repressing ligand-independent receptor activation. Development 138 (7), 1349–1359. https://doi.org/10.1242/dev.056002. Company of Biologists.

Clarke, R., et al., 2003. Antiestrogen resistance in breast cancer and the role of estrogen receptor signaling. Oncogene 22 (47), 7316–7339. https://doi.org/10.1038/sj.onc.1206937. Nature Publishing Group.

Clevers, H., 2006. Wnt/β-catenin signaling in development and disease. Cell 127 (3), 469–480. https://doi.org/10.1016/J.CELL.2006.10.018. Cell Press.

Clough, R.R., Sidhu, R.S., Bhullar, R.P., 2002. Calmodulin binds RalA and RalB and is required for the thrombin-induced activation of Ral in human platelets. J. Biol. Chem. 277 (32), 28972–28980. https://doi.org/10.1074/JBC.M201504200. American Society for Biochemistry and Molecular Biology.

Colicelli, J., 2004. Human RAS superfamily proteins and related GTPases. Sci. STKE 2004 (250), RE13. https://doi.org/10.1126/stke.2502004re13. NIH Public Access.

Corrotte, M., et al., 2010. Ral isoforms are implicated in FcγR-mediated phagocytosis: activation of phospholipase D by RalA. J. Immunol. 185 (5), 2942–2950. https://doi.org/10.4049/JIMMUNOL.0903138. American Association of Immunologists.

Cox, A.D., et al., 2014. Drugging the undruggable RAS: mission possible? Nat. Rev. Drug Discov. 13 (11), 828–851. https://doi.org/10.1038/nrd4389.

Cullis, D.N., et al., 2002. Rab11-FIP2, an adaptor protein connecting cellular components involved in internalization and recycling of epidermal growth factor receptors. J. Biol. Chem. 277 (51), 49158–49166. https://doi.org/10.1074/jbc.M206316200. American Society for Biochemistry and Molecular Biology.

D’Adamo, D.R., et al., 1997. rsc: a novel oncogene with structural and functional homology with the gene family of exchange factors for Ral. Oncogene 14 (11), 1295–1305. https://doi.org/10.1038/sj.onc.1200950. Nature Publishing Group.

D’Aloia, A., et al., 2018. RalGPS2 is involved in tunneling nanotubes formation in 5637 bladder cancer cells. Exp. Cell Res. 362 (2), 349–361. https://doi.org/10.1016/j.yexcr.2017.11.036.

D’Souza-Schorey, C., Chavrier, P., 2006. ARF proteins: roles in membrane traffic and beyond. Nat. Rev. Mol. Cell Biol. 7 (5), 347–358. https://doi.org/10.1038/nrm1910. Nature Publishing Group.

Das, A., et al., 2014. RalA promotes a direct exocyst-Par6 interaction to regulate polarity in neuronal development. J. Cell Sci. 127 (3), 686–699. https://doi.org/10.1242/jcs.145037. The Company of Biologists Ltd.

Daumke, O., et al., 2004. The GTPase-activating protein Rap1GAP uses a catalytic asparagine. Nature 429 (6988), 197–201. https://doi.org/10.1038/nature02505. Nature Publishing Group.

De Bruyn, K.M.T., et al., 2000. Ra1GEF2, a pleckstrin homology domain containing guanine nucleotide exchange factor for Ral. J. Biol. Chem. 275 (38), 29761–29766. https://doi.org/10.1074/jbc.M001160200. American Society for Biochemistry and Molecular Biology.

de Gorter, D.J.J., et al., 2008. The small GTPase Ral mediates SDF-1-induced migration of B cells and multiple myeloma cells. Blood 111 (7), 3364–3372. https://doi.org/10.1182/blood-2007-08-106583.

De Leeuw, H.P.J.C., et al., 1999. Small GTP-binding protein RalA associates with Weibel-Palade bodies in endothelial cells. Thromb. Haemost. 82 (3), 1177–1181. https://doi.org/10.1055/s-0037-1614349.

De Leeuw, H.P.J.C., et al., 2001. Small GTP-binding protein Ral modulates regulated exocytosis of von Willebrand factor by endothelial cells. Arterioscler. Thromb. Vasc. Biol. 21 (6), 899–904. https://doi.org/10.1161/01.ATV.21.6.899.

DeFeo, D., et al., 1981. Analysis of two divergent rat genomic clones homologous to the transforming gene of Harvey murine sarcoma virus. Proc. Natl. Acad. Sci. U. S. A. 78 (6 I), 3328–3332. https://doi.org/10.1073/pnas.78.6.3328.

Demetriades, C., Doumpas, N., Teleman, A.A., 2014. Regulation of TORC1 in response to amino acid starvation via lysosomal recruitment of TSC2. Cell 156 (4), 786–799. https://doi.org/10.1016/J.CELL.2014.01.024. Cell Press.

DeYoung, M.P., et al., 2008. Hypoxia regulates TSC1/2–mTOR signaling and tumor suppression through REDD1-mediated 14–3–3 shuttling. Genes Dev. 22 (2), 239. https://doi.org/10.1101/GAD.1617608. Cold Spring Harbor Laboratory Press.

Ding, L., et al., 2008. Somatic mutations affect key pathways in lung adenocarcinoma. Nature 455 (7216), 1069–1075. https://doi.org/10.1038/nature07423. Nature Publishing Group.

Donaldson, J.G., Jackson, C.L., 2011. ARF family G proteins and their regulators: roles in membrane transport, development and disease. Nat. Rev. Mol. Cell Biol. 12 (6), 362–375. https://doi.org/10.1038/nrm3117.

Drake, K.J., et al., 2007. RALBP1/RLIP76 mediates multidrug resistance. Int. J. Oncol. 30 (1), 139–144. Available at: http://www.ncbi.nlm.nih.gov/pubmed/17143522. (Accessed: 29 June 2020).

Eckfeldt, C.E., et al., 2016. RALB provides critical survival signals downstream of Ras in acute myeloid leukemia. Oncotarget 7 (40), 65147. https://doi.org/10.18632/oncotarget.11431. Impact Journals, LLC.

Ellis, R.W., et al., 1982. Mouse cells contain two distinct ras gene mRNA species that can be translated into a p21 onc protein. Mol. Cell Biol. 2 (11), 1339–1345. https://doi.org/10.1128/mcb.2.11.1339. American Society for Microbiology (ASM).

Emkey, R., Freedman, S., Feig, L.A., 1991. Characterization of a GTPase-activating protein for the Ras-related Ral protein. J. Biol. Chem. 266 (15), 9703–9706.

Essers, M.A.G., et al., 2004. FOXO transcription factor activation by oxidative stress mediated by the small GTPase Ral and JNK. EMBO J. 23 (24), 4802–4812. https://doi.org/10.1038/sj.emboj.7600476. John Wiley & Sons, Ltd.

Ezzeldin, M., et al., 2014. RalA signaling pathway as a therapeutic target in hepatocellular carcinoma (HCC). Mol. Oncol. 8 (5), 1043–1053. https://doi.org/10.1016/j.molonc.2014.03.020. John Wiley & Sons, Ltd.

Falsetti, S.C., et al., 2007. Geranylgeranyltransferase I inhibitors target RalB to inhibit anchorage-dependent growth and induce apoptosis and RalA to inhibit anchorage-independent growth. Mol. Cell Biol. 27 (22), 8003–8014. https://doi.org/10.1128/mcb.00057-07.

Farid, M., et al., 2014. Malignant peripheral nerve sheath tumors. Oncologist 19 (2), 193–201. https://doi.org/10.1634/theoncologist.2013-0328. Wiley-Blackwell.

Fei, J., et al., 2012. MiR-181a post-transcriptionally downregulates oncogenic rala and contributes to growth inhibition and apoptosis in chronic myelogenous leukemia (CML). PLoS One 7 (3). https://doi.org/10.1371/journal.pone.0032834. Public Library of Science.

Feig, L.A., 2003. Ral-GTPases: approaching their 15 minutes of fame. Trends Cell Biol. https://doi.org/10.1016/S0962-8924(03)00152-1. Elsevier Current Trends.

Feldmann, G., et al., 2010. Inhibiting the cyclin-dependent kinase CDK5 blocks pancreatic cancer formation and progression via suppression of Ras-Ral signaling. Cancer Res. 70 (11), 4460. https://doi.org/10.1158/0008-5472.CAN-09-1107. NIH Public Access.

Fenwick, C., et al., 2000. A subclass of Ras proteins that regulate the degradation of IkappaB. Science 287 (5454), 869–873. Section of Immunobiology and Department of Molecular Biophysics and Biochemistry, Howard Hughes Medical Institute, Yale University School of Medicine, New Haven, CT 06510, USA.

Fenwick, R.B., et al., 2009. Solution structure and dynamics of the small GTPase RalB in its active conformation: significance for effector protein binding. Biochemistry 48 (10), 2192–2206. https://doi.org/10.1021/bi802129d. American Chemical Society.

Fenwick, R.B., et al., 2010. The RalB-RLIP76 complex reveals a novel mode of ral-effector interaction. Structure 18 (8), 985–995. https://doi.org/10.1016/J.STR.2010.05.013. Cell Press.

Ferro, E., Trabalzini, L., 2010. RalGDS family members couple Ras to Ral signalling and that’s not all. Cell. Signal. 22 (12), 1804–1810. https://doi.org/10.1016/j.cellsig.2010.05.010.

Frankel, P., et al., 2005. RalA interacts with ZONAB in a cell density-dependent manner and regulates its transcriptional activity. EMBO J. 24 (1), 54–62. https://doi.org/10.1038/sj.emboj.7600497. John Wiley & Sons, Ltd.

Frische, E.W., et al., 2007. RAP-1 and the RAL-1/exocyst pathway coordinate hypodermal cell organization in Caenorhabditis elegans. EMBO J. 26 (24), 5083–5092. https://doi.org/10.1038/sj.emboj.7601922. John Wiley & Sons, Ltd.

Fukai, S., et al., 2003. Structural basis of the interaction between RalA and Sec5, a subunit of the sec6/8 complex. EMBO J. 22 (13), 3267–3278. https://doi.org/10.1093/emboj/cdg329. John Wiley & Sons, Ltd.

Gai, Z., et al., 2016. Structure of the TBC1D7-TSC1 complex reveals that TBC1D7 stabilizes dimerization of the TSC1 C-terminal coiled coil region. J. Mol. Cell Biol. 8 (5), 411–425. https://doi.org/10.1093/jmcb/mjw001. Oxford Academic.

Galino, J., et al., 2019. RalGTPases contribute to Schwann cell repair after nerve injury via regulation of process formation. J. Cell Biol. 218 (7), 2370–2387. https://doi.org/10.1083/jcb.201811002. The Rockefeller University Press.

Gao, P., et al., 2019. Ral GTPase activation by downregulation of RalGAP enhances oral squamous cell carcinoma progression. J. Dent. Res. 98 (9), 1011–1019. https://doi.org/10.1177/0022034519860828. SAGE PublicationsSage CA: Los Angeles, CA.

Gentry, L.R., et al., 2014. Ral small GTPase signaling and oncogenesis: more than just 15 minutes of fame. Biochim. Biophys. Acta (BBA) Mol. Cell Res. 1843 (12), 2976–2988. https://doi.org/10.1016/J.BBAMCR.2014.09.004. Elsevier.

Gentry, L.R., et al., 2015. Divergent roles of C AAX motif-signaled posttranslational modifications in the regulation and subcellular localization of Ral GTPases. J. Biol. Chem. 290 (37), 22851–22861. https://doi.org/10.1074/jbc.M115.656710.

Ghiglione, C., et al., 2008. *Drosophila* RalA is essential for the maintenance of Jak/Stat signaling in ovarian follicles. EMBO Rep. 9 (7), 676–682. https://doi.org/10.1038/embor.2008.79. John Wiley & Sons, Ltd.

Ginn, K.F., et al., 2016. RalA is overactivated in medulloblastoma. J. Neurooncol 130 (1), 99–110. https://doi.org/10.1007/s11060-016-2236-4.

Godin, C.M., et al., 2010. The small GTPase Ral couples the angiotensin II type 1 receptor to the activation of phospholipase C-delta 1. Mol. Pharmacol. 77 (3), 388–395. https://doi.org/10.1124/mol.109.061069. American Society for Pharmacology and Experimental Therapeutics.

Goi, T., et al., 2000. An EGF receptor/Ral-GTPase signaling cascade regulates c-Src activity and substrate specificity. EMBO J. 19 (4), 623–630. https://doi.org/10.1093/emboj/19.4.623. John Wiley & Sons, Ltd.

Goitre, L., et al., 2014. The Ras Superfamily of Small GTPases: The Unlocked Secrets. Humana Press, Totowa, NJ, pp. 1–18.

Gong, S., et al., 2019. RCC2, a regulator of the RalA signaling pathway, is identified as a novel therapeutic target in cisplatin-resistant ovarian cancer. FASEB J. 33 (4), 5350–5365. https://doi.org/10.1096/fj.201801529RR.

González-García, A., et al., 2005. RalGDS is required for tumor formation in a model of skin carcinogenesis. Cancer Cell 7 (3), 219–226. https://doi.org/10.1016/J.CCR.2005.01.029. Cell Press.

Gray, J.L., von Delft, F., Brennan, P.E., 2020. Targeting the small GTPase superfamily through their regulatory proteins. Angew. Chem. Int. Ed. 59, 6342–6366. https://doi.org/10.1002/anie.201900585. John Wiley & Sons Ltd.

Gridley, S., et al., 2005. Novel insulin-elicted phosphoproteins in adipocytes. Cell. Signal. 17 (1), 59–66. https://doi.org/10.1016/J.CELLSIG.2004.05.013. Pergamon.

Gridley, S., et al., 2006. Adipocytes contain a novel complex similar to the tuberous sclerosis complex. Cell. Signal. 18 (10), 1626–1632. https://doi.org/10.1016/J.CELLSIG.2006.01.002. Pergamon.

Grindstaff, K.K., et al., 1998. Sec6/8 complex is recruited to cell–cell contacts and specifies transport vesicle delivery to the basal-lateral membrane in epithelial cells. Cell 93 (5), 731–740. https://doi.org/10.1016/S0092-8674(00)81435-X. Cell Press.

Grujic, O., Bhullar, R.P., 2009. Ral GTPase interacts with the N-terminal in addition to the C-terminal region of PLC-delta1. Biochem. Biophys. Res. Commun. 383 (4), 401–405. https://doi.org/10.1016/j.bbrc.2009.04.043.

Gu, C., et al., 2016. RalA, a GTPase targeted by miR-181a, promotes transformation and progression by activating the Ras-related signaling pathway in chronic myelogenous leukemia. Oncotarget 7 (15), 20561–20573. https://doi.org/10.18632/oncotarget.7987.

Guin, S., Theodorescu, D., 2015. The RAS-RAL axis in cancer: evidence for mutation-specific selectivity in non-small cell lung cancer. Acta Pharmacol. Sin. 36 (3), 291–297. https://doi.org/10.1038/aps.2014.129. 2015/01/05. Nature Publishing Group.

Guin, S., et al., 2013. Contributions of KRAS and RAL in non-small-cell lung cancer growth and progression. J. Thorac. Oncol. 8 (12), 1492–1501. https://doi.org/10.1097/JTO.0000000000000007. Elsevier.

Guo, W., et al., 2000. Protein complexes in transport vesicle targeting. Trends Cell Biol. 10 (6), 251–255. https://doi.org/10.1016/S0962-8924(00)01754-2. Elsevier Current Trends.

Gus-Brautbar, Y., et al., 2012. The anti-inflammatory TIPE2 is an inhibitor of the oncogenic Ras. Mol. Cell 45 (5), 610–618. https://doi.org/10.1016/j.molcel.2012.01.006. CellPress.

Gyorffy, B., et al., 2015. Effects of RAL signal transduction in KRAS-and BRAF-mutated cells and prognostic potential of the RAL signature in colorectal cancer. Oncotarget 6 (15), 13334–13346. https://doi.org/10.18632/oncotarget.3871. Impact Journals, LLC.

Hamad, N.M., et al., 2002. Distinct requirements for Ras oncogenesis in human versus mouse cells. Genes Dev. 16 (16), 2045–2057. https://doi.org/10.1101/gad.993902. Cold Spring Harbor Laboratory Press.

Hamada, M., et al., 2011. Involvement of RhoA and RalB in geranylgeranylltransferase I inhibitor-mediated inhibition of proliferation and migration of human oral squamous cell carcinoma cells. Cancer Chemother. Pharmacol. 68 (3), 559–569. https://doi.org/10.1007/s00280-010-1520-9. Springer.

Han, K., et al., 2009. Regulated RalBP1 binding to RalA and PSD-95 controls AMPA receptor endocytosis and LTD. PLoS Biol. 7 (9), e1000187. https://doi.org/10.1371/journal.pbio.1000187. Edited by M. D. Ehlers. Public Library of Science.

Hansmann, P., et al., 2020. Structure of the TSC2 GAP domain: mechanistic insight into catalysis and pathogenic mutations. Structure 28 (8), 933–942.e4. https://doi.org/10.1016/j.str.2020.05.008.

Hao, Y., Wong, R., Feig, L.A., 2008. RalGDS couples growth factor signaling to Akt activation. Mol. Cell Biol. 28 (9), 2851–2859. https://doi.org/10.1128/MCB.01917-07. American Society for Microbiology Journals.

Hase, K., et al., 2009. M-Sec promotes membrane nanotube formation by interacting with Ral and the exocyst complex. Nat. Cell Biol. 11 (12), 1427–1432. https://doi.org/10.1038/ncb1990. Nature Publishing Group.

Hazelett, C.C., Yeaman, C., 2012. Sec5 and Exo84 mediate distinct aspects of RalA-dependent cell polarization. PLoS One 7 (6), e39602. https://doi.org/10.1371/journal.pone.0039602. Edited by C. Gottardi. Public Library of Science.

He, B., Guo, W., 2009. The exocyst complex in polarized exocytosis. Curr. Opin. Cell Biol. 21 (4), 537–542. https://doi.org/10.1016/J.CEB.2009.04.007. Elsevier Current Trends.

Heider, M.R., Munson, M., 2012. Exorcising the exocyst complex. Traffic 13 (7), 898–907. https://doi.org/10.1111/j.1600-0854.2012.01353.x. John Wiley & Sons, Ltd.

Henry, D.O., et al., 2000. Ral GTPases contribute to regulation of cyclin D1 through activation of NF-κB. Mol. Cell Biol. 20 (21), 8084–8092. https://doi.org/10.1128/MCB.20.21.8084-8092.2000. American Society for Microbiology Journals.

Hernandez-Muñoz, I., et al., 2000. The Rgr oncogene (homologous to RalGDS) induces transformation and gene expression by activating Ras, Ral and Rho mediated pathways. Oncogene 19 (23), 2745–2757. https://doi.org/10.1038/sj.onc.1203586. Nature Publishing Group.

Hiatt, S.M., et al., 2018. De novo mutations in the GTP/GDP-binding region of RALA, a RAS-like small GTPase, cause intellectual disability and developmental delay. PLoS Genet. 14 (11), e1007671. https://doi.org/10.1371/journal.pgen.1007671. Edited by G. P. Copenhaver.

Hobbs, G.A., Der, C.J., Rossman, K.L., 2016. RAS isoforms and mutations in cancer at a glance. J. Cell Sci. 129 (7), 1287–1292. https://doi.org/10.1242/jcs.182873. The Company of Biologists Ltd.

Hodge, R.G., Ridley, A.J., 2016. Regulating Rho GTPases and their regulators. Nat. Rev. Mol. Cell Biol. 17 (8), 496–510. https://doi.org/10.1038/nrm.2016.67.

Hodges, A.K., et al., 2001. Pathological mutations in TSC1 and TSC2 disrupt the interaction between hamartin and tuberin. Hum. Mol. Genet. 10 (25), 2899–2905. https://doi.org/10.1093/hmg/10.25.2899. Oxford Academic.

Hofer, F., Berdeaux, R., Martin, G.S., 1998. Ras-independent activation of Ral by a Ca2+-dependent pathway. Curr. Biol. 8 (14), 839–844. https://doi.org/10.1016/s0960-9822(98)70327-6. Cell Press.

Holly, R.M., et al., 2015. A rapid, membrane-dependent pathway directs furrow formation through RalA in the early Drosophila embryo. Development 142 (13), 2316–2328. https://doi.org/10.1242/dev.120998.

Hsu, S.-C., et al., 2004. The exocyst complex in polarized exocytosis. Int. Rev. Cytol. 233, 243–265. https://doi.org/10.1016/S0074-7696(04)33006-8. Academic Press.

Ihle, N.T., et al., 2012. Effect of KRAS oncogene substitutions on protein behavior: implications for signaling and clinical outcome. J. Natl. Cancer Inst. 104 (3), 228–239. https://doi.org/10.1093/jnci/djr523. Oxford University Press.

Iida, T., et al., 2020. Down-regulation of RalGTPase-activating protein promotes colitis-associated cancer via NLRP3 inflammasome activation. Cell. Mol. Gastroenterol. Hepatol. 9 (2), 277–293. https://doi.org/10.1016/j.jcmgh.2019.10.003.

Ikeda, M., et al., 1998. Identification and characterization of a novel protein interacting with Ral-binding protein 1, a putative effector protein of Ral. J. Biol. Chem. 273 (2), 814–821. https://doi.org/10.1074/jbc.273.2.814.

Inoki, K., et al., 2002. TSC2 is phosphorylated and inhibited by Akt and suppresses mTOR signalling. Nat. Cell Biol. 4 (9), 648–657. https://doi.org/10.1038/ncb839.

Inoki, K., et al., 2003. Rheb GTPase is a direct target of TSC2 GAP activity and regulates mTOR signaling. Genes Dev. 17 (15), 1829–1834. https://doi.org/10.1101/gad.1110003. Cold Spring Harbor Laboratory Press.

Inoue, M., et al., 2003. The exocyst complex is required for targeting of Glut4 to the plasma membrane by insulin. Nature 422 (6932), 629–633. https://doi.org/10.1038/nature01533. Nature Publishing Group.

Issaq, S.H., Lim, K.-H., Counter, C.M., 2010. Sec5 and Exo84 foster oncogenic Ras-mediated tumorigenesis. Mol. Cancer Res. 8 (2), 223–231. https://doi.org/10.1158/1541-7786.MCR-09-0189. American Association for Cancer Research.

Jackson, C.L., Bouvet, S., 2014. Arfs at a glance. J. Cell Sci. 127 (Pt 19), 4103–4109. https://doi.org/10.1242/jcs.144899.

Jaffe, A.B., Hall, A., 2005. RHO GTPASES: biochemistry and biology. Annu. Rev. Cell Dev. Biol. 21 (1), 247–269. https://doi.org/10.1146/annurev.cellbio.21.020604.150721. Annual Reviews.

Janigro, D., et al., 2007. RLIP76 in AED drug resistance. Epilepsia 48 (6), 1218–1219. author reply 1219-20 https://doi.org/10.1111/j.1528-1167.2007.01155_4.x.

Jessen, K.R., Mirsky, R., Lloyd, A.C., 2015. Schwann cells: development and role in nerve repair. Cold Spring Harb. Perspect. Biol. 7 (7), a020487. https://doi.org/10.1101/cshperspect.a020487. Cold Spring Harbor Laboratory Press.

Jiang, H., et al., 1995. Involvement of Ral GTPase in v-Src-induced phospholipase D activation. Nature 378 (6555), 409–412. https://doi.org/10.1038/378409a0. Nature Publishing Group.

Jiménez, M., et al., 2004. The Rgr oncogene induces tumorigenesis in transgenic mice. Cancer Res. 64 (17), 6041–6049. https://doi.org/10.1158/0008-5472.CAN-03-3389.

Jin, R., et al., 2005. Exo84 and Sec5 are competitive regulatory Sec6/8 effectors to the RalA GTPase. EMBO J. 24 (12), 2064–2074. https://doi.org/10.1038/sj.emboj.7600699. John Wiley & Sons, Ltd.

Jing, H., et al., 2017. SIRT2 and lysine fatty acylation regulate the transforming activity of K-Ras4a. eLife 6, e32436. https://doi.org/10.7554/eLife.32436.

Johansson, J., et al., 2019. RAL GTPases drive intestinal stem cell function and regeneration through internalization of WNT signalosomes. Cell Stem Cell 24 (4), 592–607.e7. https://doi.org/10.1016/j.stem.2019.02.002. Cell Press.

Jones, S., et al., 2008. Core signaling pathways in human pancreatic cancers revealed by global genomic analyses. Science (New York, NY) 321 (5897), 1801–1806. https://doi.org/10.1126/science.1164368.

Jullien-Flores, V., et al., 1995. Bridging Ral GTPase to Rho pathways. RLIP76, a Ral effector with CDC42/Rac GTPase-activating protein activity. J. Biol. Chem. 270 (38), 22473–22477. https://doi.org/10.1074/jbc.270.38.22473.

Jullien-Flores, V., et al., 2000. RLIP76, an effector of the GTPase Ral, interacts with the AP2 complex: involvement of the Ral pathway in receptor endocytosis. J. Cell Sci. 113 (Pt 16), 2837–2844. Available at: http://www.ncbi.nlm.nih.gov/pubmed/10910768. (Accessed: 26 June 2020).

Karachaliou, N., et al., 2013. KRAS Mutations in Lung Cancer. Clin. Lung Cancer 14 (3), 205–214. https://doi.org/10.1016/J.CLLC.2012.09.007. Elsevier.

Karasic, T.B., et al., 2019. A phase I study of GGTI-2418 (Geranylgeranyl transferase I inhibitor) in patients with advanced solid tumors. Target. Oncol. 14 (5), 613–618. https://doi.org/10.1007/s11523-019-00661-5.

Kashatus, D.F., et al., 2011. RALA and RALBP1 regulate mitochondrial fission at mitosis. Nat. Cell Biol. 13 (9), 1108–1117. https://doi.org/10.1038/ncb2310.

Katsha, A., et al., 2015. Aurora kinase A in gastrointestinal cancers: time to target. Mol. Cancer 14 (1), 106. https://doi.org/10.1186/s12943-015-0375-4. BioMed Central.

Kawato, M., et al., 2008. Regulation of platelet dense granule secretion by the Ral GTPase-exocyst pathway. J. Biol. Chem. 283 (1), 166–174. https://doi.org/10.1074/jbc.M705340200. American Society for Biochemistry and Molecular Biology.

Kazi, A., et al., 2009. Blockade of protein geranylgeranylation inhibits Cdk2-dependent p27Kip1 phosphorylation on Thr187 and accumulates p27Kip1 in the nucleus: implications for breast cancer therapy. Mol. Cell Biol. 29 (8), 2254–2263. https://doi.org/10.1128/MCB.01029-08. American Society for Microbiology (ASM).

Keen, N., Taylor, S., 2004. Aurora-kinase inhibitors as anticancer agents. Nat. Rev. Cancer 4 (12), 927–936. https://doi.org/10.1038/nrc1502. Nature Publishing Group.

Kikuchi, A., et al., 1994. ralGDS family members interact with the effector loop of ras p21. Mol. Cell Biol. 14 (11), 7483–7491. https://doi.org/10.1128/mcb.14.11.7483. American Society for Microbiology (ASM).

Kim, J.H., et al., 1998. Activation of phospholipase D1 by direct interaction with ADP-ribosylation factor 1 and RalA. FEBS Lett. 430 (3), 231–235. https://doi.org/10.1016/S0014-5793(98)00661-9. John Wiley & Sons, Ltd.

Kim, E., et al., 1999. Disruption of the mouse Rce1 gene results in defective Ras processing and mislocalization of Ras within cells. J. Biol. Chem. 274 (13), 8383–8390. https://doi.org/10.1074/jbc.274.13.8383. American Society for Biochemistry and Molecular Biology.

Kim, E.Y., et al., 2014. KRAS oncogene substitutions in Korean NSCLC patients: clinical implication and relationship with pAKT and RalGTPases expression. Lung Cancer 85 (2), 299–305. https://doi.org/10.1016/J.LUNGSCAN.2014.04.012. Elsevier.

Kinsella, B.T., Erdman, R.A., Maltese, W.A., 1991. Carboxyl-terminal isoprenylation of ras-related GTP-binding proteins encoded by rac1, rac2, and ralA. J. Biol. Chem. 266 (15), 9786–9794.

Kishida, S., et al., 1997. Colocalization of Ras and Ral on the membrane is required for Ras-dependent Ral activation through Ral GDP dissociation stimulator. Oncogene 15 (24), 2899–2907. https://doi.org/10.1038/sj.onc.1201473. Nature Publishing Group.

Kodama, T., et al., 2016. Two-step forward genetic screen in mice identifies Ral GTPase-activating proteins as suppressors of hepatocellular carcinoma. Gastroenterology 151 (2), 324–337.e12. https://doi.org/10.1053/j.gastro.2016.04.040. W.B. Saunders.

Kompijer, L.C., et al., 2010. FGFR3, HRAS, KRAS, NRAS and PIK3CA mutations in bladder cancer and their potential as biomarkers for surveillance and therapy. PLoS One 5 (11), e13821. https://doi.org/10.1371/journal.pone.0013821. Public Library of Science.

Kops, G.J.P.L., et al., 1999. Direct control of the Forkhead transcription factor AFX by protein kinase B. Nature 398 (6728), 630–634. https://doi.org/10.1038/19328. Nature Publishing Group.

Lalli, G., Hall, A., 2005. Ral GTPases regulate neurite branching through GAP-43 and the exocyst complex. J. Cell Biol. 171 (5), 857–869. https://doi.org/10.1083/jcb.200507061. The Rockefeller University Press.

Lee, S., Wurtzel, J.G.T., Goldfinger, L.E., 2014. The RLIP76 N-terminus binds ARNO to regulate PI 3-kinase, Arf6 and Rac signaling, cell spreading and migration. Biochem. Biophys. Res. Commun. 454 (4), 560–565. https://doi.org/10.1016/j.bbrc.2014.10.114. NIH Public Access.

Leonardi, P., et al., 2002. Human rgr: transforming activity and alteration in T-cell malignancies. Oncogene 21 (33), 5108–5116. https://doi.org/10.1038/sj.onc.1205694.

Leto, D., et al., 2013. Negative regulation of the RalGAP complex by 14-3-3. J. Biol. Chem. 288 (13), 9272–9283. https://doi.org/10.1074/jbc.M112.426106.

Leung, K.F., et al., 2007. Rab GTPases containing a CAAX motif are processed post-geranylgeranylation by proteolysis and methylation. J. Biol. Chem. 282 (2), 1487–1497. https://doi.org/10.1074/JBC.M605557200. American Society for Biochemistry and Molecular Biology.

Li, N., et al., 1993. Guanine-nucleotide-releasing factor hSos1 binds to Grb2 and links receptor tyrosine kinases to Ras signalling. Nature 363 (6424), 85–88. https://doi.org/10.1038/363085a0.

Li, Y., et al., 2002. Regulation ofTSC2by 14-3-3 binding. J. Biol. Chem. 277 (47), 44593–44596. https://doi.org/10.1074/jbc.C200510200. American Society for Biochemistry and Molecular Biology.

Li, Y., Inoki, K., Guan, K.-L., 2004. Biochemical and functional characterizations of small GTPase Rheb and TSC2 GAP activity. Mol. Cell Biol. 24 (18), 7965–7975. https://doi.org/10.1128/MCB.24.18.7965-7975.2004. American Society for Microbiology.

Lim, K.-H., et al., 2005. Activation of RalA is critical for Ras-induced tumorigenesis of human cells. Cancer Cell 7 (6), 533–545. https://doi.org/10.1016/J.CCR.2005.04.030. Cell Press.

Lim, K.-H., et al., 2006. Divergent roles for RalA and RalB in malignant growth of human pancreatic carcinoma cells. Curr. Biol. 16 (24), 2385–2394. https://doi.org/10.1016/J.CUB.2006.10.023. Cell Press.

Lim, K.-H., et al., 2010. Aurora-A phosphorylates, activates, and relocates the small GTPase RalA. Mol. Cell Biol. 30 (2), 508–523. https://doi.org/10.1128/MCB.00916-08. American Society for Microbiology (ASM).

Lopez, J.A., et al., 2008. The RalA GTPase is a central regulator of insulin exocytosis from pancreatic islet beta cells. J. Biol. Chem. 283 (26), 17939–17945. https://doi.org/10.1074/jbc.M800321200. American Society for Biochemistry and Molecular Biology.

Lu, J., et al., 2009. In vivo antitumor effect of a novel inhibitor of protein geranylgeranyltransferase I. Mol. Cancer Ther. 8 (5), 1218. https://doi.org/10.1158/1535-7163.MCT-08-1122. NIH Public Access.

Luo, J.-Q., et al., 1998. Functional association between Arf and RalA in active phospholipase D complex. Proc. Natl. Acad. Sci. 95 (7), 3632–3637. https://doi.org/10.1073/PNAS.95.7.3632. National Academy of Sciences.

M’Rabet, L., et al., 1999. Differential fMet-Leu-Phe- and platelet-activating factor-induced signaling toward Ral activation in primary human neutrophils. J. Biol. Chem. 274 (31), 21847–21852. https://doi.org/10.1074/jbc.274.31.21847.

Maehama, T., et al., 2008. RalA functions as an indispensable signal mediator for the nutrient-sensing system. J. Biol. Chem. 283 (50), 35053–35059. https://doi.org/10.1074/jbc.M805822200.

Male, H., et al., 2012. Inhibition of RalA signaling pathway in treatment of non-small cell lung cancer. Lung Cancer 77 (2), 252–259. https://doi.org/10.1016/j.lungcan.2012.03.007. Elsevier.

Mark, B.L., Jilkina, O., Bhullar, R.P., 1996. Association of Ral GTP-binding protein with human platelet dense granules. Biochem. Biophys. Res. Commun. 225 (1), 40–46. https://doi.org/10.1006/bbrc.1996.1128.

Martin, T.D., Der, C.J., 2012. Differential involvement of RalA and RalB in colorectal cancer. Small GTPases 3 (2), 126–130. https://doi.org/10.4161/sgtp.19571. Taylor & Francis.

Martin, T.D., et al., 2011. Activation and involvement of Ral GTPases in colorectal cancer. Cancer Res. 71 (1), 206–215. https://doi.org/10.1158/0008-5472.CAN-10-1517. NIH Public Access.

Martin, T.D., et al., 2012. Phosphorylation by protein kinase Cα regulates RalB small GTPase protein activation, subcellular localization, and effector utilization. J. Biol. Chem. 287 (18), 14827. https://doi.org/10.1074/JBC.M112.344986. American Society for Biochemistry and Molecular Biology.

Martin, T.D., et al., 2014. Ral and Rheb GTPase activating proteins integrate mTOR and GTPase signaling in aging, autophagy, and tumor cell invasion. Mol. Cell 53 (2), 209–220. https://doi.org/10.1016/j.molcel.2013.12.004. Cell Press.

Matchett, K.B., et al., 2014. Ran GTPase in Nuclear Envelope Formation and Cancer Metastasis. Cancer Biology and the Nuclear Envelope. 773 Springer, New York, NY, pp. 323–351.

Matsubara, K., et al., 1999. Plasma membrane recruitment of RalGDS is critical for Ras-dependent Ral activation. Oncogene 18 (6), 1303–1312. https://doi.org/10.1038/sj.onc.1202425. Nature Publishing Group.

Matsuo, M., et al., 2013. MiR-29c is downregulated in gastric carcinomas and regulates cell proliferation by targeting RCC2. Mol. Cancer 12, 15. https://doi.org/10.1186/1476-4598-12-15.

Melchior, F., 2001. Ran GTPase cycle: one mechanism—two functions. Curr. Biol. 11 (7), R257–R260. https://doi.org/10.1016/S0960-9822(01)00132-4. Cell Press.

Mirey, G., et al., 2003. A Ral guanine exchange factor-ral pathway is conserved in drosophila melanogaster and sheds new light on the connectivity of the Ral, Ras, and Rap pathways. Mol. Cell Biol. 23 (3), 1112–1124. https://doi.org/10.1128/MCB.23.3.1112-1124.2003.

Mishra, P.J., et al., 2010. Dissection of RAS downstream pathways in melanomagenesis: a role for Ral in transformation. Oncogene 29 (16), 2449–2456. https://doi.org/10.1038/onc.2009.521. Nature Publishing Group.

Mittar, D., Sehajpal, P.K., Lander, H.M., 2004. Nitric oxide activates Rap1 and Ral in a Ras-independent manner. Biochem. Biophys. Res. Commun. 322 (1), 203–209. https://doi.org/10.1016/J.BBRC.2004.07.107. Academic Press.

Mollberg, N.M., et al., 2012. Overexpression of RalBP1 in colorectal cancer is an independent predictor of poor survival and early tumor relapse. Cancer Biol. Ther. 13 (8), 694–700. https://doi.org/10.4161/cbt.20087. Taylor & Francis.

Moores, S.L., et al., 1991. Sequence dependence of protein isoprenylation. J. Biol. Chem. 266 (22), 14603–14610.

Morinaka, K., et al., 1999. Epsin binds to the EH domain of POB1 and regulates receptor-mediated endocytosis. Oncogene 18 (43), 5915–5922. https://doi.org/10.1038/sj.onc.1202974. Nature Publishing Group.

Moskalenko, S., et al., 2002. The exocyst is a Ral effector complex. Nat. Cell Biol. 4 (1), 66–72. https://doi.org/10.1038/ncb728. Nature Publishing Group.

Moskalenko, S., et al., 2003. Ral GTPases regulate exocyst assembly through dual subunit interactions. American Society for Biochemistry and Molecular Biology, J. Biol. Chem. 278 (51), 51743–51748. https://doi.org/10.1074/jbc.M308702200.

Mott, H.R., Owen, D., 2010. RLIP76 (RalBP1): the first piece of the structural puzzle. Small GTPases 1 (3), 157–160. https://doi.org/10.4161/sgtp.1.3.14233. Taylor & Francis.

Nakashima, S., et al., 1999. Small G protein Ral and its downstream molecules regulate endocytosis of EGF and insulin receptors. EMBO J. 18 (13), 3629–3642. https://doi.org/10.1093/emboj/18.13.3629. John Wiley & Sons, Ltd.

Neel, N.F., et al., 2012. The RalB small GTPase mediates formation of invadopodia through a GTPase-activating protein-independent function of the RalBP1/RLIP76 effector. Mol. Cell Biol. 32 (8), 1374–1386. https://doi.org/10.1128/MCB.06291-11. American Society for Microbiology.

Neyraud, V., et al., 2012. RalA and RalB proteins are ubiquitinated GTPases, and ubiquitinated RalA increases lipid raft exposure at the plasma membrane. J. Biol. Chem. 287 (35), 29397–29405. https://doi.org/10.1074/jbc.M112.357764. American Society for Biochemistry and Molecular Biology.

Nicely, N.I., et al., 2004. Crystal structures of Ral-GppNHp and Ral-GDP reveal two binding sites that are also present in Ras and Rap. Structure 12 (11), 2025–2036. https://doi.org/10.1016/J.STR.2004.08.011. Cell Press.

Nishimura, A., Linder, M.E., 2013. Identification of a novel prenyl and palmitoyl modification at the CaaX Motif of Cdc42 that regulates RhoGDI binding. Mol. Cell Biol. 33 (7), 1417–1429. https://doi.org/10.1128/mcb.01398-12.

Norden, P.R., Sun, Z., Davis, G.E., 2020. Control of endothelial tubulogenesis by Rab and Ral GTPases, and apical targeting of caveolin-1-labeled vacuoles. PLoS One 15 (6), e0235116. https://doi.org/10.1371/journal.pone.0235116. Edited by K. K. Wary. Public Library of Science.

Oeckinghaus, A., et al., 2014. κB-Ras proteins regulate both NF-κB-dependent inflammation and Ral-dependent proliferation. Cell Rep. 8 (6), 1793–1807. https://doi.org/10.1016/j.celrep.2014.08.015.

Ohta, Y., et al., 1999. The small GTPase RalA targets filamin to induce filopodia. Proc. Natl. Acad. Sci. 96 (5), 2122–2128. https://doi.org/10.1073/PNAS.96.5.2122. National Academy of Sciences.

Okamoto, N., et al., 2019. RALA mutation in a patient with autism spectrum disorder and Noonan syndrome-like phenotype. Congenit. Anom. 59 (6), 195–196. https://doi.org/10.1111/cga.12327.

Olivier, J.P., et al., 1993. A Drosophila SH2-SH3 adaptor protein implicated in coupling the sevenless tyrosine kinase to an activator of Ras guanine nucleotide exchange, Sos. Cell 73 (1), 179–191. https://doi.org/10.1016/0092-8674(93)90170-U.

Ommer, A., et al., 2019. Ral GTPases in Schwann cells promote radial axonal sorting in the peripheral nervous system. J. Cell Biol. 218 (7), 2350–2369. https://doi.org/10.1083/jcb.201811150. The Rockefeller University Press.

Oral, E.A., et al., 2017. Inhibition of IKKε and TBK1 improves glucose control in a subset of patients with type 2 diabetes. Cell Metab. 26 (1), 157–170.e7. https://doi.org/10.1016/J.CMET.2017.06.006. Cell Press.

Osei-Sarfo, K., et al., 2011. The human Rgr oncogene is overexpressed in T-cell malignancies and induces transformation by acting as a GEF for Ras and Ral. Oncogene 30 (34), 3661–3671. https://doi.org/10.1038/onc.2011.93.

Otto, J.C., et al., 1999. Cloning and characterization of a mammalian prenyl protein-specific protease. J. Biol. Chem. 274 (13), 8379–8382. https://doi.org/10.1074/jbc.274.13.8379. American Society for Biochemistry and Molecular Biology.

Ouerhani, S., Elgaaied, A.B.A., 2011. The mutational spectrum of HRAS, KRAS, NRAS and FGFR3 genes in bladder cancer. Cancer Biomark. 10 (6), 259–266. https://doi.org/10.3233/CBM-2012-0254.

Oxford, G., et al., 2005. RalA and RalB: antagonistic relatives in cancer cell migration. Cancer Res. 65 (16), 7111–7120. https://doi.org/10.1158/0008-5472.CAN-04-1957.

Oxford, G., et al., 2007. Expression profiling of Ral-depleted bladder cancer cells identifies RREB-1 as a novel transcriptional Ral effector. Oncogene 26 (50), 7143–7152. https://doi.org/10.1038/sj.onc.1210521. Nature Publishing Group.

Paduch, M., Jeleń, F., Otlewski, J., 2001. Structure of small G proteins and their regulators. Acta Biochim. Pol. 48 (4), 829–850. Poland.

Pang, B., et al., 2017. Overexpression of RCC2 enhances cell motility and promotes tumor metastasis in lung adenocarcinoma by inducing epithelial-mesenchymal transition. Clin. Cancer Res. 23 (18), 5598–5610. https://doi.org/10.1158/1078-0432.CCR-16-2909.

Papini, D., et al., 2015. TD-60 links RalA GTPase function to the CPC in mitosis. Nat. Commun. 6 (1), 7678. https://doi.org/10.1038/ncomms8678. Nature Publishing Group.

Park, J.B., 2001. Regulation of GTP-binding state in RalA through Ca2+ and calmodulin. Exp. Mol. Med. 33 (1), 54–58. https://doi.org/10.1038/emm.2001.10.

Park, J.-B., Lee, J.-Y., Kim, J.-W., 1999. Dissociation of RalA from synaptic membranes by Ca2+/calmodulin. Biochem. Biophys. Res. Commun. 263 (3), 765–769. https://doi.org/10.1006/BBRC.1999.1463. Academic Press.

Patel, P., Chen, E.I., 2012. Cancer stem cells, tumor dormancy, and metastasis. Front. Endocrinol. 3, 125. https://doi.org/10.3389/fendo.2012.00125.

Pathak, R., et al., 2012. The microtubule-associated Rho activating factor GEF-H1 interacts with exocyst complex to regulate vesicle traffic. Dev. Cell 23 (2), 397–411. https://doi.org/10.1016/J.DEVCEL.2012.06.014. Cell Press.

Pawar, A., et al., 2016. Ral-Arf6 crosstalk regulates Ral dependent exocyst trafficking and anchorage independent growth signalling. Cell. Signal. 28 (9), 1225–1236. https://doi.org/10.1016/J.CELLSIG.2016.05.023. Pergamon.

Peschard, P., et al., 2012. Genetic deletion of RALA and RALB small GTPases reveals redundant functions in development and tumorigenesis. Curr. Biol. 22 (21), 2063–2068. https://doi.org/10.1016/j.cub.2012.09.013.

Peterson, S.N., et al., 1996. Identification of a novel RalGDS-related protein as a candidate effector for Ras and Rap1. J. Biol. Chem. 271 (47), 29903–29908. https://doi.org/10.1074/jbc.271.47.29903. American Society for Biochemistry and Molecular Biology.

Pollock, S.R., et al., 2019. RalA and RalB relocalization to depolarized mitochondria depends on clathrin-mediated endocytosis and facilitates TBK1 activation. PLoS One 14 (4), e0214764. https://doi.org/10.1371/journal.pone.0214764. Edited by I. R. Nabi. Public Library of Science.

Polzin, A., et al., 2002. Ral-GTPase influences the regulation of the readily releasable pool of synaptic vesicles. Mol. Cell Biol. 22 (6), 1714–1722. https://doi.org/10.1128/mcb.22.6.1714-1722.2002. American Society for Microbiology Journals.

Pomeroy, E.J., Eckfeldt, C.E., 2020. Targeting Ras signaling in AML: RALB is a small GTPase with big potential. Small GTPases 11 (1), 39. https://doi.org/10.1080/21541248.2017.1339765. Taylor & Francis.

Pomeroy, E.J., et al., 2017. Ras oncogene-independent activation of RALB signaling is a targetable mechanism of escape from NRAS(V12) oncogene addiction in acute myeloid leukemia. Oncogene 36 (23), 3263–3273. https://doi.org/10.1038/onc.2016.471.

Popovic, M., et al., 2016. The structure of the guanine nucleotide exchange factor Rlf in complex with the small G-protein Ral identifies conformational intermediates of the exchange reaction and the basis for the selectivity. J. Struct. Biol. 193 (2), 106–114. https://doi.org/10.1016/j.jsb.2015.12.006.

Postler, T.S., Ghosh, S., 2015. Bridging the gap: a regulator of NF-κB linking inflammation and cancer. J. Oral Biosci. 57 (3), 143–147. https://doi.org/10.1016/j.job.2015.05.001.

Pylypenko, O., et al., 2018. Rab GTPases and their interacting protein partners: structural insights into Rab functional diversity. Small GTPases 9 (1–2), 22–48. https://doi.org/10.1080/21541248.2017.1336191.

Qin, J., et al., 2016. Structural basis of the interaction between tuberous sclerosis complex 1 (TSC1) and Tre2-Bub2-Cdc16 domain family member 7 (TBC1D7). J. Biol. Chem. 291 (16), 8591–8601. https://doi.org/10.1074/jbc.M115.701870. American Society for Biochemistry and Molecular Biology.

Quilliam, L.A., Rebhun, J.F., Castro, A.F., 2002. A growing family of guanine nucleotide exchange factors is responsible for activation of Ras-family GTPases. Prog. Nucleic Acid Res. Mol. Biol. 71, 391–444. https://doi.org/10.1016/s0079-6603(02)71047-7.

Rangarajan, A., et al., 2004. Species- and cell type-specific requirements for cellular transformation. Cancer Cell 6 (2), 171–183. https://doi.org/10.1016/J.CCR.2004.07.009. Cell Press.

Ratnacaram, C.K., et al., 2008. Temporally controlled ablation of PTEN in adult mouse prostate epithelium generates a model of invasive prostatic adenocarcinoma. Proc. Natl. Acad. Sci. U. S. A. 105 (7), 2521–2526. https://doi.org/10.1073/pnas.0712021105.

Rauen, K.A., 2013. The RASopathies. Annu. Rev. Genomics Hum. Genet. 14, 355. https://doi.org/10.1146/ANNUREV-GENOM-091212-153523. NIH Public Access.

Rebhun, J.F., Chen, H., Quilliam, L.A., 2000. Identification and characterization of a new family of guanine nucleotide exchange factors for the Ras-related GTPase Ral. J. Biol. Chem. 275 (18), 13406–13410. https://doi.org/10.1074/JBC.C000085200. American Society for Biochemistry and Molecular Biology.

Reid, T.S., et al., 2004. Crystallographic analysis of CaaX prenyltransferases complexed with substrates defines rules of protein substrate selectivity. J. Mol. Biol. 343 (2), 417–433. https://doi.org/10.1016/j.jmb.2004.08.056.

Reilly, S.M., et al., 2013. An inhibitor of the protein kinases TBK1 and IKK-ε improves obesity-related metabolic dysfunctions in mice. Nat. Med. 19 (3), 313–321. https://doi.org/10.1038/nm.3082. Nature Publishing Group.

Reiner, D.J., Lundquist, E.A., 2018. Small GTPases. WormBook 2018, 1–65. https://doi.org/10.1895/wormbook.1.67.2.

Rhee, S.G., Bae, Y.S., 1997. Regulation of phosphoinositide-specific phospholipase C isozymes. J. Biol. Chem. 272 (24), 15045–15048. https://doi.org/10.1074/jbc.272.24.15045. American Society for Biochemistry and Molecular Biology.

Richhariya, S., Hasan, G., 2020. Ral function in muscle is required for flight maintenance in Drosophila. Small GTPases 11 (3), 174–179. https://doi.org/10.1080/21541248.2017.1367456.

Richhariya, S., et al., 2017. A pupal transcriptomic screen identifies Ral as a target of store-operated calcium entry in Drosophila neurons. Sci. Rep. 7, 42586. https://doi.org/10.1038/srep42586. Nature Publishing Group.

Richhariya, S., et al., 2018. dSTIM- and Ral/exocyst-mediated synaptic release from pupal dopaminergic neurons sustains Drosophila flight. eNeuro 5 (3). https://doi.org/10.1523/ENEURO.0455-17.2018. Society for Neuroscience.

Rinaldo, F., et al., 2007. RalA regulates vascular endothelial growth factor-C (VEGF-C) synthesis in prostate cancer cells during androgen ablation. Oncogene 26 (12), 1731–1738. https://doi.org/10.1038/sj.onc.1209971. Nature Publishing Group.

Rojas, A.M., et al., 2012. The Ras protein superfamily: evolutionary tree and role of conserved amino acids. J. Cell Biol. 196 (2), 189–201. https://doi.org/10.1083/jcb.201103008. The Rockefeller University Press.

Rondaij, M.G., et al., 2008. Guanine exchange factor RalGDS mediates exocytosis of Weibel-Palade bodies from endothelial cells. Blood 112 (1), 56–63. https://doi.org/10.1182/blood-2007-07-099309. American Society of Hematology.

Rossé, C., et al., 2003. RLIP, an effector of the Ral GTPases, is a platform for Cdk1 to phosphorylate epsin during the switch off of endocytosis in mitosis. J. Biol. Chem. 278 (33), 30597–30604. https://doi.org/10.1074/jbc.M302191200. American Society for Biochemistry and Molecular Biology.

Rossé, C., et al., 2006. RalB mobilizes the exocyst to drive cell migration. Mol. Cell Biol. 26 (2), 727–734. https://doi.org/10.1128/MCB.26.2.727-734.2006. American Society for Microbiology Journals.

Rozakis-Adcock, M., et al., 1993. The SH2 and SH3 domains of mammalian Grb2 couple the EGF receptor to the Ras activator mSos1. Nature 363 (6424), 83–85. https://doi.org/10.1038/363083a0.

Rubinfeld, B., et al., 1991. Molecular cloning of a GTPase activating protein specific for the Krev-1 protein p21rap1. Cell 65 (6), 1033–1042. https://doi.org/10.1016/0092-8674(91)90555-D. Cell Press.

Rubinfeld, B., et al., 1992. Localization of the rap1GAP catalytic domain and sites of phosphorylation by mutational analysis. Mol. Cell Biol. 12 (10), 4634–4642. https://doi.org/10.1128/mcb.12.10.4634.

Ruiter, N.D.D., Burgering, B.M.T., Bos, J.L., 2001. Regulation of the forkhead transcription factor AFX by Ral-dependent phosphorylation of threonines 447 and 451. Mol. Cell Biol. 21 (23), 8225–8235. https://doi.org/10.1128/MCB.21.23.8225-8235.2001. American Society for Microbiology Journals.

Rusanescu, G., et al., 2001. Regulation of Ras signaling specificity by protein kinase C. Mol. Cell Biol. 21 (8), 2650–2658. https://doi.org/10.1128/MCB.21.8.2650-2658.2001. American Society for Microbiology (ASM).

Sablina, A.A., et al., 2007. The tumor suppressor PP2A Aβ regulates the RalA GTPase. NIH Public Access, Cell 129 (5), 969–982. https://doi.org/10.1016/j.cell.2007.03.047.

Saito, T., Shibasaki, T., Seino, S., 2008. Involvement of Exoc31, a protein structurally related to the exocyst subunit Sec6, in insulin secretion. Biomed. Res. 29 (2), 85–91. https://doi.org/10.2220/biomedres.29.85. Biomedical Research Press.

Saito, R., et al., 2013. Downregulation of Ral GTPase-activating protein promotes tumor invasion and metastasis of bladder cancer. Oncogene 32 (7), 894–902. Department of Urology, Graduate School of Medicine, Kyoto University, Kyoto, Japan.

Sancak, Y., et al., 2010. Ragulator-Rag complex targets mTORC1 to the lysosomal surface and is necessary for its activation by amino acids. Cell 141 (2), 290–303. https://doi.org/10.1016/J.CELL.2010.02.024. Elsevier.

Sánchez-Ruiz, J., et al., 2011. Ral GTPases regulate cell-mediated cytotoxicity in NK cells. J. Immunol. 187 (5), 2433–2441. https://doi.org/10.4049/JIMMUNOL.1003089. American Association of Immunologists.

Santiago Lima, A.J., et al., 2014. Identification of regions critical for the integrity of the TSC1-TSC2-TBC1D7 complex. PLoS One 9 (4), e93940. https://doi.org/10.1371/journal.pone.0093940. Edited by S. R. Singh. Public Library of Science.

Santos, A.O., Parrini, M.C., Camonis, J., 2016. RalGPS2 is essential for survival and cell cycle progression of lung cancer cells independently of its established substrates Ral GTPases. PLoS One 11 (5), e0154840. https://doi.org/10.1371/journal.pone.0154840. Edited by D. J. Reiner. Public Library of Science.

Sasaki, A.T., et al., 2011. Ubiquitination of K-Ras enhances activation and facilitates binding to select downstream effectors. Sci. Signal. 4 (163), ra13. https://doi.org/10.1126/scisignal.2001518. NIH Public Access.

Sawamoto, K., et al., 1999. The Drosophila Ral GTPase regulates developmental cell shape changes through the Jun NH2-terminal kinase pathway. J. Cell Biol. 146 (2), 361–372. https://doi.org/10.1083/jcb.146.2.361. The Rockefeller University Press.

Saxton, R.A., Sabatini, D.M., 2017. mTOR signaling in growth, metabolism, and disease. Cell 168 (6), 960–976. https://doi.org/10.1016/J.CELL.2017.02.004. Cell Press.

Scheffzek, K., et al., 1997. The Ras-RasGAP complex: structural basis for GTPase activation and its loss in oncogenic ras mutants. Science 277 (5324), 333–339. https://doi.org/10.1126/science.277.5324.333.

Schiller, C., et al., 2013. LST1 promotes the assembly of a molecular machinery responsible for tunneling nanotube formation. J. Cell Sci. 126 (3), 767–777. https://doi.org/10.1242/JCS.114033. The Company of Biologists Ltd.

Schmoranzer, J., Kreitzer, G., Simon, S.M., 2003. Migrating fibroblasts perform polarized, microtubule-dependent exocytosis towards the leading edge. J. Cell Sci. 116 (22), 4513–4519. https://doi.org/10.1242/jcs.00748. The Company of Biologists Ltd.

Schwarzbraun, T., et al., 2004. Cloning, genomic structure, and expression profiles of TULIP1 (GARNL1), a brain-expressed candidate gene for 14q13-linked neurological phenotypes, and its murine homologue. Genomics 84 (3), 577–586. https://doi.org/10.1016/J.YGENO.2004.04.013. Academic Press.

Scrima, A., et al., 2008. The Rap-RapGAP complex: GTP hydrolysis without catalytic glutamine and arginine residues. EMBO J. 27 (7), 1145–1153. https://doi.org/10.1038/emboj.2008.30. European Molecular Biology Organization.

Seguin, L., et al., 2014. An integrin β3–KRAS–RalB complex drives tumour stemness and resistance to EGFR inhibition. Nat. Cell Biol. 16 (5), 457–468. https://doi.org/10.1038/ncb2953. Nature Publishing Group.

Seibold, M., et al., 2019. RAL GTPases mediate multiple myeloma cell survival and are activated independently of oncogenic RAS. Haematologica 105 (9), 2316–2326. https://doi.org/10.3324/haematol.2019.223024.

Senga, T., et al., 2001. JAK/STAT3-dependent activation of the RalGDS/Ral pathway in M1 mouse myeloid leukemia cells. J. Biol. Chem. 276 (35), 32678–32681. https://doi.org/10.1074/JBC.M105749200. American Society for Biochemistry and Molecular Biology.

Shao, H., Andres, D.A., 2000. A novel RalGEF-like protein, RGL3, as a candidate effector for rit and Ras. J. Biol. Chem. 275 (35), 26914–26924. https://doi.org/10.1074/jbc.M002241200.

Shimojima, K., et al., 2009. TULIP1 (RALGAPA1) haploinsufficiency with brain development delay. Genomics 94 (6), 414–422. https://doi.org/10.1016/J.YGENO.2009.08.015. Academic Press.

Shin, H., et al., 2018. Ral signals through a MAP4 kinase-p38 MAP kinase cascade in C. elegans cell fate patterning. Cell Rep. 24 (10), 2669–2681.e5. https://doi.org/10.1016/j.celrep.2018.08.011.

Shin, H., et al., 2019. Developmental fidelity is imposed by genetically separable RalGEF activities that mediate opposing signals. PLoS Genet. 15 (5), e1008056. https://doi.org/10.1371/journal.pgen.1008056. Public Library of Science.

Shipitsin, M., Feig, L.A., 2004. RalA but Not RalB enhances polarized delivery of membrane proteins to the basolateral surface of epithelial cells. Mol. Cell Biol. 24 (13), 5746–5756. https://doi.org/10.1128/MCB.24.13.5746-5756.2004. American Society for Microbiology Journals.

Shirakawa, R., Horiuchi, H., 2015. Ral GTPases: crucial mediators of exocytosis and tumourigenesis. J. Biochem. 157 (5), 285–299. https://doi.org/10.1093/jb/mvv029. Oxford University Press.

Shirakawa, R., et al., 2009. Tuberous sclerosis tumor suppressor complex-like complexes act as GTPase-activating proteins for Ral GTPases. J. Biol. Chem. 284 (32), 21580–21588. https://doi.org/10.1074/jbc.M109.012112. American Society for Biochemistry and Molecular Biology.

Sidhu, R.S., et al., 2005a. Calmodulin binding to the small GTPase Ral requires isoprenylated Ral. Biochem. Biophys. Res. Commun. 336 (1), 105–109. https://doi.org/10.1016/J.BBRC.2005.08.053. Academic Press.

Sidhu, R.S., Clough, R.R., Bhullar, R.P., 2005b. Regulation of phospholipase C-δ1 through direct interactions with the small GTPase ral and calmodulin. J. Biol. Chem. 280 (23), 21933–21941. https://doi.org/10.1074/jbc.M412966200.

Sievers, F., et al., 2011. Fast, scalable generation of high-quality protein multiple sequence alignments using clustal omega. Mol. Syst. Biol. 7, 539. https://doi.org/10.1038/msb.2011.75.

Silva-Rodrigues, J.F., et al., 2020. Peripheral axonal ensheathment is regulated by RalA GTPase and the exocyst complex. Development 147 (3). https://doi.org/10.1242/dev.174540. Oxford University Press for The Company of Biologists Limited.

Simicek, M., et al., 2013. The deubiquitylase USP33 discriminates between RALB functions in autophagy and innate immune response. Nat. Cell Biol. 15 (10), 1220–1230. https://doi.org/10.1038/ncb2847. Nature Publishing Group.

Singh, M.K., et al., 2019. Localization of RalB signaling at endomembrane compartments and its modulation by autophagy. Sci. Rep. 9 (1), 8910. https://doi.org/10.1038/s41598-019-45443-1. Nature Publishing Group.

Singhal, S.S., Awasthi, Y.C., Awasthi, S., 2006. Regression of melanoma in a murine model by RLIP76 depletion. Cancer Res. 66 (4), 2354–2360. https://doi.org/10.1158/0008-5472.CAN-05-3534. American Association for Cancer Research.

Singhal, S.S., Singhal, J., Nair, M.P., et al., 2007a. Doxorubicin transport by RALBP1 and ABCG2 in lung and breast cancer. Int. J. Oncol. 30 (3), 717–725. Available at: http://www.ncbi.nlm.nih.gov/pubmed/17273774. (Accessed: 29 June 2020).

Singhal, S.S., Singhal, J., Yadav, S., et al., 2007b. Regression of lung and colon cancer xenografts by depleting or inhibiting RLIP76 (Ral-binding protein 1). Cancer Res. 67 (9), 4382–4389. https://doi.org/10.1158/0008-5472.CAN-06-4124.

Skorobogatko, Y., et al., 2018. RalA controls glucose homeostasis by regulating glucose uptake in brown fat. Proc. Natl. Acad. Sci. U. S. A. 115 (30), 7819–7824. https://doi.org/10.1073/pnas.1801050115. National Academy of Sciences.

Smith, S.C., et al., 2007. Expression of Ral GTPases, their effectors, and activators in human bladder cancer. Clin. Cancer Res. 13 (13), 3803–3813. https://doi.org/10.1158/1078-0432.CCR-06-2419. American Association for Cancer Research.

Smith, S.C., et al., 2012. Transcriptional signatures of Ral GTPase are associated with aggressive clinicopathologic characteristics in human cancer. Cancer Res. 72 (14), 3480–3491. https://doi.org/10.1158/0008-5472.CAN-11-3966. NIH Public Access.

Song, X., et al., 2015. Involvement of RalB in the effect of geranylgeranytransferase I on glioma cell migration and invasion. Clin. Transl. Oncol. 17 (6), 477–485. https://doi.org/10.1007/s12094-014-1263-x. Springer.

Song, C., et al., 2018. RCC2 is a novel p53 target in suppressing metastasis. Oncogene 37 (1), 8–17. https://doi.org/10.1038/onc.2017.306.

Sowalsky, A.G., et al., 2010. RalA suppresses early stages of Ras-induced squamous cell carcinoma progression. Oncogene 29 (1), 45–55. https://doi.org/10.1038/onc.2009.307. Nature Publishing Group.

Spiczka, K.S., Yeaman, C., 2008. Ral-regulated interaction between Sec5 and paxillin targets Exocyst to focal complexes during cell migration. J. Cell Sci. 121 (Pt 17), 2880–2891. https://doi.org/10.1242/jcs.031641.

Spiegelman, N.A., et al., 2019. SIRT2 and lysine fatty acylation regulate the activity of RalB and cell migration. ACS Chem. Biol. 14 (9), 2014–2023. https://doi.org/10.1021/acschembio.9b00492. American Chemical Society.

Sun, J., et al., 1998. Both farnesyltransferase and geranylgeranyltransferase I inhibitors are required for inhibition of oncogenic K-Ras prenylation but each alone is sufficient to suppress human tumor growth in node mouse xenografts. Oncogene 16 (11), 1467–1473. https://doi.org/10.1038/sj.onc.1201656. Nature Publishing Group.

Sun, J., et al., 1999. Antitumor efficacy of a novel class of non-thiol-containing pep-tidomimetic inhibitors of farnesyltransferase and geranylgeranyltransferase I: combination therapy with the cytotoxic agents cisplatin, taxol, and gemcitabine. Cancer Res. 59 (19), 4919–4926.

Sun, J., et al., 2003. Geranylgeranyltransferase I inhibitor GGTI-2154 induces breast carcinoma apoptosis and tumor regression in H-Ras transgenic mice. Cancer Res. 63 (24), 8922–8929. American Association for Cancer Research. Available at: https://cancerres.aacrjournals.org/content/63/24/8922.long. (Accessed 6 July 2020).

Sun, W., et al., 2013. Crystal structure of the yeast TSC1 core domain and implications for tuberous sclerosis pathological mutations. Nat. Commun. 4, 1–8. https://doi.org/10.1038/ncomms3135. Nature Publishing Group.

Suzuki, J., et al., 2000. Involvement of Ras and Ral in chemotactic migration of skeletal myoblasts. Mol. Cell Biol. 20 (13), 4658–4665. https://doi.org/10.1128/MCB.20.13.4658-4665.2000. American Society for Microbiology Journals.

Sztul, E., et al., 2019. ARF GTPases and their GEFs and GAPs: concepts and challenges. Mol. Biol. Cell 30 (11), 1249–1271. https://doi.org/10.1091/mbc.E18-12-0820.

Tago, K., et al., 2010. KappaB-Ras is a nuclear-cytoplasmic small GTPase that inhibits NF-kappaB activation through the suppression of transcriptional activation of p65/RelA. J. Biol. Chem. 285 (40), 30622–30633. Laboratory of Signal Transduction, Department of Cell Biology, Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0192, Japan.

Takaya, A., et al., 2004. RalA activation at nascent lamellipodia of epidermal growth factor-stimulated Cos7 cells and migrating madin-darby canine kidney cells. Mol. Biol. Cell 15 (6), 2549–2557. https://doi.org/10.1091/mbc.e03-11-0857. American Society for Cell Biology.

Takaya, A., et al., 2007. R-Ras regulates exocytosis by Rgl2/Rlf-mediated activation of RalA on endosomes. Mol. Biol. Cell 18 (5), 1850. https://doi.org/10.1091/MBC.E06-08-0765. American Society for Cell Biology.

Tazat, K., et al., 2013. Dual effects of Ral-activated pathways on p27 localization and TGF-β signaling. Mol. Biol. Cell 24 (11), 1812–1824. https://doi.org/10.1091/mbc.e13-01-0007. Edited by C.-H. Heldin. The American Society for Cell Biology.

Teodoro, R.O., et al., 2013. Ral mediates activity-dependent growth of postsynaptic membranes via recruitment of the exocyst. EMBO J. 32 (14), 2039–2055. https://doi.org/10.1038/emboj.2013.147. John Wiley & Sons, Ltd.

Teufel, A., et al., 2007. Genetics of hepatocellular carcinoma. World J. Gastroenterol. 13 (16), 2271–2282. https://doi.org/10.3748/wjg.v13.i16.2271. Baishideng Publishing Group Inc.

Thomas, J.C., et al., 2016. Inhibition of Ral GTPases using a stapled peptide approach. J. Biol. Chem. 291 (35), 18310–18325. https://doi.org/10.1074/jbc.M116.720243. American Society for Biochemistry and Molecular Biology.

Tian, X., et al., 2002. PDK1 mediates growth factor-induced Ral-GEF activation by a kinase-independent mechanism. EMBO J. 21 (6), 1327–1338. https://doi.org/10.1093/emboj/21.6.1327. John Wiley & Sons, Ltd.

Tracy, K., Velentzas, P.D., Baehrecke, E.H., 2016. Ral GTPase and the exocyst regulate autophagy in a tissue-specific manner. EMBO Rep. 17 (1), 110–121. https://doi.org/10.15252/embr.201541283. John Wiley & Sons, Ltd.

Tsuboi, T., et al., 2005. Mammalian exocyst complex is required for the docking step of insulin vesicle exocytosis. J. Biol. Chem. 280 (27), 25565–25570. https://doi.org/10.1074/JBC.M501674200. American Society for Biochemistry and Molecular Biology.

Uegaki, M., et al., 2019. Downregulation of RalGTPase-activating protein promotes invasion of prostatic epithelial cells and progression from intraepithelial neoplasia to cancer during prostate carcinogenesis. Carcinogenesis 40 (12), 1535–1544. https://doi.org/10.1093/carcin/bgz082.

Uhm, M., et al., 2017. Phosphorylation of the exocyst protein Exo84 by TBK1 promotes insulin-stimulated GLUT4 trafficking. Sci. Signal. 10 (471). https://doi.org/10.1126/scisignal.aah5085.

Urano, T., Emkey, R., Feig, L.A., 1996. Ral-GTPases mediate a distinct downstream signaling pathway from Ras that facilitates cellular transformation. EMBO J. 15 (4), 810–816. https://doi.org/10.1002/j.1460-2075.1996.tb00416.x.

Van Acker, T., Tavernier, J., Peelman, F., 2019. The small GTPase Arf6: an overview of its mechanisms of action and of its role in host–pathogen interactions and innate immunity. Int. J. Mol. Sci. 20 (9), 2209. https://doi.org/10.3390/ijms20092209. Multidisciplinary Digital Publishing Institute.

van Dam, T.J.P., et al., 2009. Phylogeny of the CDC25 homology domain reveals rapid differentiation of Ras pathways between early animals and fungi. Cell. Signal. 21 (11), 1579–1585. https://doi.org/10.1016/j.cellsig.2009.06.004. Elsevier Inc.

van Dam, T.J.P., et al., 2011a. Evolution of the TOR pathway. J. Mol. Evol. 73 (3–4), 209–220. https://doi.org/10.1007/s00239-011-9469-9. Springer.

van Dam, T.J.P., Bos, J.L., Snel, B., 2011b. Evolution of the Ras-like small GTPases and their regulators. Small GTPases 2 (1), 4. https://doi.org/10.4161/sgtp.2.1.15113. Taylor & Francis.

van den Berg, M.C.W., et al., 2013. The small GTPase RALA controls c-Jun N-terminal kinase-mediated FOXO activation by regulation of a JIP1 scaffold complex. J. Biol. Chem. 288 (30), 21729–21741. https://doi.org/10.1074/JBC.M113.463885. American Society for Biochemistry and Molecular Biology.

Van Slegtenhorst, M., et al., 1998. Interaction between hamartin and tuberin, the TSC1 and TSC2 gene products. Hum. Mol. Genet. 7 (6), 1053–1057. https://doi.org/10.1093/hmg/7.6.1053.

Varambally, S., et al., 2005. Integrative genomic and proteomic analysis of prostate cancer reveals signatures of metastatic progression. Cancer Cell 8 (5), 393–406. https://doi.org/10.1016/J.CCR.2005.10.001. Cell Press.

Vetter, I.R., 2014. The structure of the G domain of the Ras superfamily. In: Ras Superfamily Small G Proteins: Biology and Mechanisms 1. Springer Vienna, Vienna, pp. 25–50, https://doi.org/10.1007/978-3-7091-1806-1_2.

Vetter, I. R. and Wittinghofer, A. (2001) ‘The guanine nucleotide-binding switch in three dimensions’, Science, 294(5545), pp. 1299 LP – 1304. doi: 10.1126/science.1062023.

Vigil, D., et al., 2010. Aberrant overexpression of the Rgl2 Ral small GTPase-specific guanine nucleotide exchange factor promotes pancreatic cancer growth through Ral-dependent and Ral-independent mechanisms. Mol. Biol. 285 (45), 34729–34740. https://doi.org/10.1074/jbc.M110.116756.

Vitale, N., et al., 2005. The small GTPase RalA controls exocytosis of large dense core secretory granules by interacting with ARF6-dependent phospholipase D1. J. Biol. Chem. 280 (33), 29921–29928. https://doi.org/10.1074/JBC.M413748200. American Society for Biochemistry and Molecular Biology.

Wagner, M., et al., 2020. Bi-allelic variants in RALGAPA1 cause profound neurodevelopmental disability, muscular hypotonia, infantile spasms, and feeding abnormalities. Am. J. Hum. Genet. 106 (2), 246–255. https://doi.org/10.1016/J.AJHG.2020.01.002. Cell Press.

Walsh, T.G., Wersäll, A., Poole, A.W., 2019. Characterisation of the Ral GTPase inhibitor RBC8 in human and mouse platelets. Cell. Signal. 59, 34–40. https://doi.org/10.1016/j.cellsig.2019.03.015.

Wang, K.L., Roufogalis, B.D., 1999. Ca2+/calmodulin stimulates GTP binding to the Ras-related protein Ral-A. J. Biol. Chem. 274 (21), 14525–14528. https://doi.org/10.1074/jbc.274.21.14525.

Wang, K.L., Khan, M.T., Roufogalis, B.D., 1997. Identification and characterization of a calmodulin-binding domain in Ral-A, a Ras-related GTP-binding protein purified from human erythrocyte membrane. J. Biol. Chem. 272 (25), 16002–16009. https://doi.org/10.1074/JBC.272.25.16002. American Society for Biochemistry and Molecular Biology.

Wang, L., Li, G., Sugita, S., 2004. RalA-exocyst interaction mediates GTP-dependent exocytosis. J. Biol. Chem. 279 (19), 19875–19881. https://doi.org/10.1074/jbc.M400522200. American Society for Biochemistry and Molecular Biology.

Wang, K., et al., 2009. Immunogenicity of RalA and its tissue-specific expression in hepatocellular carcinoma. Int. J. Immunopathol. Pharmacol. 22 (3), 735–743. https://doi.org/10.1177/039463200902200319. SAGE PublicationsSage UK: London, England.

Wang, H., et al., 2010. Phosphorylation of RalB is important for bladder cancer cell growth and metastasis. Cancer Res. 70 (21), 8760–8769. https://doi.org/10.1158/0008-5472.CAN-10-0952. NIH Public Access.

Wang, P., et al., 2020. RCC2 interacts with small GTPase RalA and regulates cell proliferation and motility in gastric cancer. Onco. Targets. Ther. 13, 3093–3103. https://doi.org/10.2147/OTT.S228914. Dove Press.

Ward, Y., et al., 2001. Signal pathways which promote invasion and metastasis: critical and distinct contributions of extracellular signal-regulated kinase and ral-specific guanine exchange factor pathways. Mol. Cell Biol. 21 (17), 5958–5969. https://doi.org/10.1128/mcb.21.17.5958-5969.2001.

Weis, K., 2003. Regulating access to the genome: nucleocytoplasmic transport throughout the cell cycle. Cell 112 (4), 441–451. https://doi.org/10.1016/S0092-8674(03)00082-5. Cell Press.

Wennberg, K., Rossman, K.L., Der, C.J., 2005. The Ras superfamily at a glance. J. Cell Sci. 118 (5), 843–846. https://doi.org/10.1242/jcs.01660.

Westcott, P.M.K., To, M.D., 2013. The genetics and biology of KRAS in lung cancer. Chin. J. Cancer 32 (2), 63–70. https://doi.org/10.5732/cjc.012.10098. BioMed Central.

White, M.A., et al., 1996. A role for the Ral guanine nucleotide dissociation stimulator in mediating Ras-induced transformation. J. Biol. Chem. 271 (28), 16439–16442. https://doi.org/10.1074/jbc.271.28.16439. American Society for Biochemistry and Molecular Biology.

Wittinghofer, A., Vetter, I.R., 2011. Structure-function relationships of the G domain, a canonical switch motif. Annu. Rev. Biochem. 80 (1), 943–971. https://doi.org/10.1146/annurev-biochem-062708-134043. Annual Reviews.

Wolthuis, R.M.F., et al., 1996. RalGDS-like factor (Rlf) is a novel Ras and Rap1A-associating protein. Oncogene 13 (2), 353–362. Available at: https://europepmc.org/article/med/8710374. (Accessed: 5 May 2020).

Wolthuis, R.M.F., et al., 1997. Stimulation of gene induction and cell growth by the Ras effector Rlf. EMBO J. 16 (22), 6748–6761. https://doi.org/10.1093/emboj/16.22.6748. John Wiley & Sons, Ltd.

Wolthuis, R.M.F., Franke, B., et al., 1998a. Activation of the small GTPase Ral in platelets. Mol. Cell Biol. 18 (5), 2486–2491. https://doi.org/10.1128/mcb.18.5.2486. American Society for Microbiology (ASM).

Wolthuis, R.M.F., Zwartkruis, F., et al., 1998b. Ras-dependent activation of the small GTPase Ral. Curr. Biol. 8 (8), 471–474. https://doi.org/10.1016/S0960-9822(98)70183-6. Cell Press.

Wu, B., Guo, W., 2015. The exocyst at a glance. J. Cell Sci. 128 (16), 2957–2964. https://doi.org/10.1242/jcs.156398. The Company of Biologists Ltd.

Wu, J.-C., et al., 2005. Identification of V23RalA-Ser194 as a critical mediator for aurora-A-induced cellular motility and transformation by small pool expression screening. J. Biol. Chem. 280 (10), 9013–9022. https://doi.org/10.1074/JBC.M411068200. American Society for Biochemistry and Molecular Biology.

Wu, Z., et al., 2010. RalBP1 is necessary for metastasis of human cancer cell lines. Neoplasia (New York, NY) 12 (12), 969. https://doi.org/10.1593/NEO.101080. Neoplasia Press.

Wu, N., et al., 2018. RCC2 over-expression in tumor cells alters apoptosis and drug sensitivity by regulating Rac1 activation. BMC Cancer 18 (1), 67. https://doi.org/10.1186/s12885-017-3908-y.

Wurtzel, J.G.T., et al., 2015. RLIP76 regulates Arf6-dependent cell spreading and migration by linking ARNO with activated R-Ras at recycling endosomes. Biochem. Biophys. Res. Commun. 467 (4), 785–791. https://doi.org/10.1016/j.bbrc.2015.10.064.

Xu, L., et al., 2011. Phospholipase D mediates nutrient input to mammalian target of rapamycin complex 1 (mTORC1). J. Biol. Chem. 286 (29), 25477–25486. https://doi.org/10.1074/JBC.M111.249631. American Society for Biochemistry and Molecular Biology.

Yamaguchi, A., et al., 1997. An Eps homology (EH) domain protein that binds to the Ral-GTPase target, RalBP1. J. Biol. Chem. 272 (50), 31230–31234. https://doi.org/10.1074/jbc.272.50.31230.

Yan, C., Theodorescu, D., 2018. RAL GTPases: biology and potential as therapeutic targets in cancer. Pharmacol. Rev. 70 (1), 1–11. https://doi.org/10.1124/pr.117.014415. American Society for Pharmacology and Experimental Therapeutics.

Yan, C., et al., 2014. Discovery and characterization of small molecules that target the GTPase Ral. Nature 515 (7527), 443–447. https://doi.org/10.1038/nature13713.

Yan, M., et al., 2016. Aurora-A kinase: a potent oncogene and target for cancer therapy. Med. Res. Rev. 36 (6), 1036–1079. https://doi.org/10.1002/med.21399. John Wiley & Sons, Ltd.

Yin, J., et al., 2007. Activation of the RalGEF/Ral pathway promotes prostate cancer metastasis to bone. Mol. Cell Biol. 27 (21), 7538–7550. https://doi.org/10.1128/MCB.00955-07. American Society for Microbiology (ASM).

Yokoyama, K., et al., 1991. A protein geranylgeranyltransferase from bovine brain: implications for protein prenylation specificity. Proc. Natl. Acad. Sci. U. S. A. 88 (12), 5302–5306. https://doi.org/10.1073/pnas.88.12.5302.

Yoshizaki, H., et al., 2007. Akt–PDK1 complex mediates epidermal growth factor-induced membrane protrusion through ral activation. Mol. Biol. Cell 18 (1), 119–128. https://doi.org/10.1091/mbc.e06-05-0467. Edited by M. A. Schwartz. The American Society for Cell Biology.

Yu, Y., Feig, L.A., 2002. Involvement of R-Ras and Ral GTPases in estrogen-independent proliferation of breast cancer cells. Oncogene 21 (49), 7557–7568. https://doi.org/10.1038/sj.onc.1205961. Nature Publishing Group.

Zago, G., et al., 2018. RalB directly triggers invasion downstream Ras by mobilizing the wave complex. eLife 7, e40474. https://doi.org/10.7554/eLife.40474.

Zanaruddin, S.N.S., et al., 2013. Common oncogenic mutations are infrequent in oral squamous cell carcinoma of Asian origin. PLoS One 8 (11), e80229. https://doi.org/10.1371/journal.pone.0080229. Edited by A. Yeudall. Public Library of Science.

Zand, T.P., Reiner, D.J., Der, C.J., 2011. Ras effector switching promotes divergent cell fates in C. elegans vulval patterning. Dev. Cell 20 (1), 84–96. https://doi.org/10.1016/J.DEVCEL.2010.12.004. Cell Press.

Zech, R., et al., 2016. Structure of the tuberous sclerosis complex 2 (TSC2) N terminus provides insight into complex assembly and tuberous sclerosis pathogenesis. J. Biol. Chem. 291 (38), 20008–20020. https://doi.org/10.1074/jbc.M116.732446. American Society for Biochemistry and Molecular Biology.

Zerial, M., McBride, H., 2001. Rab proteins as membrane organizers. Nat. Rev. Mol. Cell Biol. 2 (2), 107–117. https://doi.org/10.1038/35052055.

Zhen, Y., Stenmark, H., 2015. Cellular functions of Rab GTPases at a glance. J. Cell Sci. 128 (17), 3171–3176. https://doi.org/10.1242/jcs.166074.

Zhu, X., et al., 2012. Inhibition of small GTPase RalA regulates growth and arsenic-induced apoptosis in chronic myeloid leukemia (CML) cells. Cell. Signal. 24 (6), 1134–1140. https://doi.org/10.1016/j.cellsig.2012.01.016.

Zipfel, P.A., et al., 2010. Ral activation promotes melanogenesis. Oncogene 29 (34), 4859–4864. https://doi.org/10.1038/onc.2010.224.

Zwartkruis, F.J.T., et al., 1998. Extracellular signal-regulated activation of Rap1 fails to interfere in Ras effector signalling. EMBO J. 17 (20), 5905–5912. https://doi.org/10.1093/emboj/17.20.5905. European Molecular Biology Organization.
