
European Journal of Medicinal Chemistry 223 (2021) 113654

Contents lists available at ScienceDirect

European Journal of Medicinal Chemistry

journal homepage: http://www.elsevier.com/locate/ejmech

Potential pharmacological strategies targeting the Niemann-Pick C1 receptor and Ebola virus glycoprotein interaction

Marcos Morales-Tenorio ${ }^{a}$, Tiziana Ginex ${ }^{a}$, Miguel Ángel Cuesta-Geijo ${ }^{a,b}$, Nuria E. Campillo ${ }^{a}$, César Muñoz-Fontela ${ }^{c,d}$, Covadonga Alonso ${ }^{b}$, Rafael Delgado ${ }^{e}$, Carmen Gil ${ }^{a,*}$

${ }^{a}$ Centro de Investigaciones Biológicas Margarita Salas (CSIC), Ramiro de Maeztu 9, 28040 Madrid, Spain  
${ }^{b}$ Dpt. Biotechnology, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Ctra. de la Coruña km 7.5, 28040, Madrid, Spain  
${ }^{c}$ Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany  
${ }^{d}$ German Center for Infection Research (DZIF), Partner Site Hamburg-Lübeck-Borstel-Riems, Hamburg, Germany  
${ }^{e}$ Instituto de Investigación Hospital 12 de Octubre, 28041, Madrid, Spain  

ARTICLE INFO

Article history:
Received 14 April 2021
Received in revised form
27 May 2021
Accepted 13 June 2021
Available online 19 June 2021

Keywords:
Niemann-Pick C1
NPC1
Ebola virus disease
EBOV
Viruses
Cholesterol

ABSTRACT

Niemann-Pick C1 (NPC1) receptor is an intracellular protein located in late endosomes and lysosomes whose main function is to regulate intracellular cholesterol trafficking. Besides being postulated as necessary for the infection of highly pathogenic viruses in which the integrity of cholesterol transport is required, this protein also allows the entry of the Ebola virus (EBOV) into the host cells acting as an intracellular receptor. EBOV glycoprotein (EBOV-GP) interaction with NPC1 at the endosomal membrane triggers the release of the viral material into the host cell, starting the infective cycle. Disruption of the NPC1/EBOV-GP interaction could represent an attractive strategy for the development of drugs aimed at inhibiting viral entry and thus infection. Some of the today available EBOV inhibitors were proposed to interrupt this interaction, but molecular and structural details about their mode of action are still preliminary thus more efforts are needed to properly address these points. Here, we provide a critical discussion of the potential of NPC1 and its interaction with EBOV-GP as a therapeutic target for viral infections.

© 2021 The Authors. Published by Elsevier Masson SAS. This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).

Contents

1. Introduction ..... 2
2. EBOV infection and available therapies ..... 3
3. NPC1 and its role in EBOV infection ..... 4
4. Small molecules with a supposed NPC1-related anti-EBOV activity ..... 4
5. Anti-NPC1 antibodies ..... 9
6. NPC1 in other viral infections ..... 9
7. Final remarks ..... 9
Declaration of competing interest ..... 10
Acknowledgments ..... 10
Abbreviations ..... 10
References ..... 10

* Corresponding author.
E-mail address: carmen.gil@csic.es (C. Gil).

https://doi.org/10.1016/j.ejmech.2021.113654
0223-5234/© 2021 The Authors. Published by Elsevier Masson SAS. This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).

M. Morales-Tenorio, T. Ginex, M.A. Cuesta-Geijo et al.

European Journal of Medicinal Chemistry 223 (2021) 113654

# 1. Introduction

The endomembrane system is essential in cell compartmentalisation and homeostasis, and lipids are key members in the landscape of intracellular structures [1,2]. Viruses usurp their metabolism and transport during infection by reprogramming lipid synthesis to change their distribution in the cellular membranes to construct viral platforms for replication. Lipids are reordered to promote viral assembly and transmission or by using the lipid transporter proteins [3,4].

Cholesterol and its inter-organelle traffic are essential for cellular membrane homeostasis and adequate cellular performance [5], thus cholesterol levels in the cell are tightly regulated by uptake, biosynthesis and efflux from different compartments [6]. Upon cholesterol uptake and further hydrolysis in the late endosomal organelles, free cholesterol is trafficked to a variety of cellular destinations via the harmonised action of Niemann-Pick type C 1 (NPC1) and NPC2 transporter proteins [7].

The NPC1 receptor owes its name to the Niemann-Pick type C disease (NP-C), first identified in 1914 by Albert Niemann [8] and whose anatomic description was established by Ludwig Pick [9]. NP-C disease is a rare genetic disease characterised by a cellular lipid storage disorder [10,11]. Cholesterol transport is the main process affected, resulting in excessive intracellular accumulation of unesterified cholesterol in lysosomes [12]. Mutations in the *NPC1* gene cause the disease but also the *NPC2* gene is involved in NP-C disease and intracellular cholesterol transport [13].

The *NPC1* gene encodes the NPC1 receptor which is a 165-kDa glycoprotein of 1278 amino acids that is mainly located in late endosomes and lysosomes (LE/Lys) [14]. It contains 13 transmembrane domains, 3 large and 4 small luminal loops, 6 small cytoplasmic loops and a cytoplasmic tail. The NPC1 transmembrane regions host the sterol-sensing domain (SSD) [15]. In addition to the transmembrane regions, three luminal domains can be distinguished: the N-terminal domain (NTD), the middle luminal domain (MLD) and the C-terminal domain (CTD, also called the cysteine-rich domain) (Fig. 1). The soluble intralysosomal protein NPC2 acts cooperatively with NPC1 to provide cholesterol transport. Remarkably, the cholesterol trafficking at this stage is dependent of the functional interaction of LE/Lys lysobisphosphatidic acid (LBPA) with NPC2 [16]. Once the NTD receives cholesterol from NPC2, it is transferred to the SSD pocket in the membrane [17–19]. This SSD domain consists of approximately 180 amino acids that are organised into five consecutive membrane domains, the transmembrane regions 3 to 7.[20] The N-terminal (NTD) domain consists of 240 amino acids, which project into the lumen, while the other two large luminal domains are loops that span between transmembrane helices 2/3 and 8/9. As stated above, the binding site of cholesterol is localised in the NTD [17].

Apart from its regulatory role on cholesterol homeostasis, an illuminating screening by Carette et al. [22] allowed the identification of NPC1 as the main host factor responsible for EBOV and other filovirus entries. In fact, as a member of the Filoviridae family of viruses, the Ebola virus (species Zaire ebolavirus, EBOV) uses the NPC1 receptor to trigger a membrane fusion process, which allows the expulsion of its genome into the cytoplasm and subsequent replication. The cholesterol synthesis inhibitor U18666A has been used as a preliminary pharmacological tool to study EBOV infection *in vitro*, thus opening new avenues toward the identification of effective EBOV entry inhibitors. Simultaneously, Côté et al. [23] discovered an active benzylpiperazine adamantane derivative against EBOV from the screening of a small library of compounds and subsequent optimisation led to more potent derivatives. These compounds led them to identify NPC1 as a key receptor for virus entry.

NPC1 has also been linked to infection by other viruses such as the human immunodeficiency virus type-1 (HIV-1) [24], Chikungunya virus and some hepatoviruses [25,26]. As it will be explained below, hydrophobic amines such U18666A or the antidepressant imipramine have been used as pharmacological tools to study the role of NPC1 in virus infections (Fig. 2). In fact, both compounds are also responsible for the inhibition of intracellular cholesterol transport from lysosomes to the endoplasmic reticulum, thus mimicking the molecular phenotype of NP-C disease [27].

The typical low mutation rate of host proteins makes host-directed antiviral therapy a quite attractive method, but it inevitably leads to a more complicated medicinal chemistry scenario for at least two reasons. First, in the context of an NPC1-related antiviral therapy, impairment of important host processes such as the well-known downregulation of cholesterol transport indeed represents a significant secondary effect that should be avoided. Second, besides the measured antiviral effect on quite often surrogate models, no clear evidence of the precise mechanism of action has been provided. Taken together, these pieces of information significantly challenge the setup of an effective medicinal chemistry protocol for the optimisation of the antiviral activity. On the other hand, host factors such as NPC1 have been demonstrated to be critical entry factors for many related viruses, thus the identification of a successful pharmacological strategy based on their

![Fig. 1. The overall structure of NPC1 receptor (PDB code 3JD8).21 (SSD: Sterol-sensing domain; TM: Transmembrane region; MLD: Middle luminal domain; CTD: C-terminal domain; NTD: N-terminal domain).](image1.png)

![U18666A](chemical_structure_1.png)
![Imipramine](chemical_structure_2.png)

Fig. 2. Pharmacological tools used to study the role of NPC1 in virus infections.

M. Morales-Tenorio, T. Ginex, M.A. Cuesta-Geijo et al.

involvement would lead to the identification of useful broad-spectrum antivirals.

In this review, we will try to offer a systematic description of the available antiviral strategies related to EBOV infection as antibodies and small compounds. Special attention will be paid to the NPC1 receptor and its potential role in EBOV infection due to its interaction with EBOV glycoprotein (EBOV-GP). In this context, small molecules reported to act as NPC1-related antivirals will be critically examined. Additionally, the involvement of the NPC1 receptor in the infectivity promoted by divergent viruses will be also introduced.

2. EBOV infection and available therapies

EBOV and Marburg virus (*Marburg marburgvirus*, MARV) are RNA viruses belonging to the *Filoviridae* family. This family of mononegaviruses is mainly characterised by causing haemorrhagic fevers in primates [28]. The first members of the genus *Ebolavirus* were discovered in 1976 in Zaire (now the Democratic Republic of Congo) and Sudan (*Sudan ebolavirus*, SUDV) [29]. After the discovery of these new viruses, additional virus species of the same genus have been identified. In total, there are six members of the *Ebola-virus* genus, namely *Zaire ebolavirus* (EBOV), *Tai Forest ebolavirus* (Tai Forest virus, TAFV), *Sudan ebolavirus* (SUDV), *Reston ebolavirus* (Reston virus, RESTV), *Bundibugyo ebolavirus* (Bundibugyo virus, BUDV) and *Bombali ebolavirus* (Bombali virus, BOMV) [30].

The entry of the EBOV into cells involves a set of complex interaction processes with multiple cellular factors. EBOV virions interact with several molecules on the cell surface and are internalised mainly via macropinocytosis, although a clathrin-dependent endocytosis is used as an alternative entry route [31,32].

The 19 kb viral RNA genome encodes eight major proteins: nucleoprotein (NP), virion-attached glycoprotein (GP), a soluble GP (sGP), the L-polymerase (L) protein and the viral proteins (VP) VP24, VP30, VP35 and VP40. L protein is an RNA-dependent polymerase that forms a complex with VP30 responsible for viral genome transcription and replication. VP24 and VP35 mediate nucleocapsid transport and inhibit interferon signalling to facilitate avoidance of host immune response. NP encapsulates the viral genome within the nucleocapsid while VP40 is the matrix protein. GP is a trimeric glycoprotein found on the surface of the virion and is essential for binding to cells [33]. Structurally, EBOV-GP is included in the category of class I viral fusion proteins. Mature GP is divided into GP₁ and GP₂ subunits by cleavage of the GP₀ precursor polypeptide by furin proteases during virus assembly. EBOV-GP is a chalice-shaped protein where GP₂ forms the base and GP₁ the cup assembled into a 450-kDa trimer at the surface of nascent virions. Both subunits remain linked through non-covalent interactions and a disulfide bond. GP₁ mediates interactions with the viral receptors and regulates the activity of GP₂, which carries out membrane fusion [34,35]. Surrounding the chalice is an N-glycan-containing cap region and a heavily N- and O-glycosylated mucin-like domain of GP₁. This glycan seems to protect GP from neutralising antibodies [36].

In the process of cell attachment, the previously mentioned N-linked or O-linked glycans on EBOV-GP interact non-specifically with C-type lectins (DC-SIGN, L-SIGN) and β-integrins on the cell surface leading to EBOV entry, although details of how these interactions lead to virion internalisation are not fully understood [37]. The virions are internalised mainly via macropinocytosis and traffic the endosomal compartment into the cell. At this stage, viral trafficking seems to be also promoted by the action of Ca²⁺ ion channels at the endo-lysosomal system, the two-pore channels (TPCs). Their relationship with the entry of EBOV could be related to the endosomal concentration of Ca²⁺, which influences the process

of membrane fusion [38–40]. Notably, to be fusion competent, EBOV-GP must be cleaved by host proteases, primarily cathepsins L and B in acidified late endosomes. This cleavage removes 60% of the amino acids of GP₁, the mucin-like domain and the glycan cap [41]. Cleaved GP (GPC) is then ready for interaction with the NPC1 receptor, triggering the membrane fusion process that releases the viral genome to the cytoplasm.

The prototypic ebolavirus, EBOV, causes severe disease and eventually haemorrhagic fever in humans with a high mortality rate. Since the discovery of the virus in 1976, the scientific community's efforts to find a vaccine or treatment for Ebola virus disease (EVD) have not ceased, with increased attempts after following severe outbreaks in recent years [42,43]. The first EBOV vaccine based on the induction of neutralising antibodies and specific T-cell responses against EBOV glycoprotein (EBOV-GP) has recently been approved by the FDA and distributed in outbreak areas [44]. However, despite its efficacy, outbreaks continue to occur at random intervals, particularly in rural regions of central Africa. Thus, geography and unpredictability are the main challenges for vaccination coverage [45]. Because of these caveats, there is an urgent need for medical countermeasures to treat and cure infected patients. In this regard, several small molecules and monoclonal antibodies (mAbs) are effective *in vitro* and preclinical studies. Only mAbs targeting EBOV-GP have been recently demonstrated to have a significant impact on mortality in a randomised controlled clinical trial in the Democratic Republic of Congo 2018-19 outbreak in North Kivu [46]. A historical breakthrough has been represented by the approval by the FDA of the monoclonal antibody therapy for EVD. In this sense, REGN-EB3, Inmazeb™, a mixture of three monoclonal antibodies (atoltivimab, mafivimab, and odesivimab-ebgn) has become the first FDA-approved treatment for EBOV infection in adult and paediatric patients [47]. In December 2020, a second treatment (Ansuvimab-zykl by Ridgeback Biotherapeutics) received FDA approval for EVD treatment [48].

In the context of clinical trials with antiviral drugs against EVD, trials with the nucleotide analogues favipiravir and remdesivir (Fig. 3) must be mentioned. In the case of favipiravir, a non-randomised trial was conducted in Guinea in 2014-2015. The authors did not reach any conclusions regarding efficacy and tolerance, in part due to the lack of randomisation of the clinical trials and the rapid metabolism of favipiravir [49,50]. For remdesivir, a clinical trial was conducted in the Democratic Republic of Congo in 2018. EVD patients were divided into several groups, to which were administered the triple monoclonal antibody ZMapp, the antiviral remdesivir, the single monoclonal antibody Mab114 or the triple monoclonal antibody REGN-EB3. Due to the high mortality in the remdesivir group, the drug ceased to be administered [46].

M. Morales-Tenorio, T. Ginex, M.A. Cuesta-Geijo et al.

Anyway, efforts are far from being concluded. Previously reported data configure only the beginning of a new era in EVD treatment. In fact, the fatality rate coupled with limited therapeutics, still makes this pathogen a major public health threat [53]. For this reason, a thorough understanding of the virus infection process is necessary. This would help in finding out potential weaknesses and thus would offer a possibility to know what actions can be taken to mitigate disease caused by EBOV infection. At the same time, due to the technical challenges of working with highly pathogenic viruses, surrogate viral models, such as viral pseudotypes or EBOV-like particles are extensively used to study EBOV entry and fusion. Both surrogate models can be handled under BSL-2 conditions instead of the BSL-4 laboratories required to work with the replicative EBOV [54]. The work with pseudotyped viruses is based on the use of recombinant viruses (e.g. vesicular stomatitis virus (VSV) or retroviruses) that encode a protein of the virus under study, in this case, the EBOV-GP, in its envelope. In addition, they usually express reporter genes, such as genes encoding for fluorescent proteins or the luciferase [55]. Even if convenient and rapid, this strategy introduces a significant approximation with the risk to preclude a better and more comprehensive understanding of the mechanism of action of the chemicals under study.

### 3. NPC1 and its role in EBOV infection

As previously mentioned in the introduction, Carette et al. [22] and Côté et al. [23] independently identified the role of NPC1 as a key factor in the entry of the virus. This evidence was supported by the observed susceptibility of fibroblasts from NP-C patients to infection by filoviruses. Cells carrying NPC1 mutations were either minimally or not infected by VSV particles pseudotyped with the EBOV-GP (VSV/EBOV-GP) and by the whole EBOV. Moreover, the infection was restored under the expression of NPC1, thus proving the involvement of this receptor in the entry process. In addition, the authors identified the cholesterol synthesis inhibitor U18666A and the antidepressant imipramine (Fig. 2) as inhibitors of the pseudotyped virus infection. Both compounds have been previously reported to induce a phenotype similar to NPC1 deficiency [56,57]. Côté et al. [23] reached the same conclusion based on a chemical genetic approach using small molecules. The identification of a new series of benzylpiperazines that inhibited EBOV infection led them to delve into their possible mechanism of action, demonstrating a clear linkage between EBOV-GP and NPC1 in the infection mechanism *in vitro*. Thereafter, *in vivo* studies in mouse models of EBOV disease confirmed the involvement of NPC1 in EBOV infection. In fact, while wild-type mice presented high viral loads and susceptibility to mouse-adapted EBOV, *Npc1*⁻/⁻ mice were resistant to infection [58].

The interaction that allows the release of the viral genome into the cytoplasm followed by replication and production of new virions occurs between GPcl and NPC1 (Fig. 4). Several studies have shown that the middle luminal domain of NPC1 (NPC1–C or MLD) is responsible for the binding to GPcl, and the solved crystal structure of NPC1–C in complex with GPcl confirms this mechanism [59]. It is important to remark that this function of viral recognition is independent of the role of NPC1 as a cholesterol transporter [60].

Structurally, NPC1–C is mainly composed of a central core of seven α helices surrounded by seven β strands and two protruding loops (Loop 1 and Loop 2 in Fig. 4) which bind to EBOV-GP after preliminary removal of its glycan cap (yellow surface in Fig. 4) by host proteases [21,61]. Once cleaved, stabilisation of primed EBOV-GP is driven by local conformational rearrangements as the upward movement of the short 3 [10] helix at the β3–α1 segment and the small shifting of β7–β8 loop of the GP₁ subunit. These changes significantly affect the shape of GPcl, by favouring a deeper insertion of the Loop 1 of NPC1–C into the EBOV-GPcl binding surface (Fig. 4). Once EBOV-GP and NPC1 interact, membrane fusion can occur, allowing the endosomal output of the viral genome. The exact series of events governing this process remains unknown but recent studies have demonstrated that the GP₂-mediated membrane fusion process was influenced by some factors including the preliminary removal of the GP glycan cap, acidic pH and endosomal Ca²⁺ concentration [40].

Hydrophobic interactions mainly contribute to stabilising the protein-protein complex at the level of the contact surface between NPC1–C and EBOV-GPcl (Fig. 4).

At the level of Loop 1, polar interactions are formed between D129 and Y134 of NPC1–C and T83 of GPcl and between the backbone nitrogen of V133 (NPC1–C) and the backbone oxygen atom of T144 (GPcl).

At the level of Loop 2, hydrogen-bonding interactions are formed between the backbone nitrogen atom of T144 in GPcl and the backbone oxygen of Q49 of NPC1–C and between the nitrogen atom of S142 (GPcl) and the backbone oxygen of P52 (NPC1–C).

Finally, the global geometry of the protein-protein complex is further stabilised by the intramolecular C121–C147 disulfide bond at the EBOV-GPcl binding surface (Fig. 4A).

Because of its key role in viral infection, the interaction between NPC1 and EBOV-GP could represent an attractive drug target when undertaking the development of drugs to treat EVD. Inhibition of NPC1-GP binding would interfere with the release of the viral material from late endosomes, thus preventing virus replication and subsequent infection.

### 4. Small molecules with a supposed NPC1-related anti-EBOV activity

Antiviral therapies can be focused on viral (the so-called direct-acting antivirals) and host targets. Virus-directed therapeutics are generally selective and optimised for specific virus factors but they have the drawback that easily induce drug-resistance [62]. On the other hand, host-directed antivirals have gained increased attention in the anti-infective field, especially for their known low mutation rate, although some concerns regarding potential toxicity ultimately would be alleviated by the relatively short timeframe of the treatment in case of acute infection [63,64]. Host-directed therapies can yield broad-spectrum therapeutics which would stimulate the body's response to infection through a mechanism not directly related to the virus. Combining both approaches could be very useful, not only to avoid resistances but also because some combinations can have synergistic effects.

In the context of anti-EBOV therapy focused on virus entry, NPC1 as key entry receptor, has been proposed as a host target for the development of anti-EBOV drugs. Here, the antiviral effect could be obtained by a direct or indirect (allosteric) inhibition of the interaction between NPC1 and EBOV-GP. In most of these cases, a concomitant downregulation of cholesterol transport proceeding from binding to NPC1 was also observed.

The series of adamantane dipeptide piperazines including compounds **3.0** and **3.47** [23,65] (Table 1) that allowed the identification of the key role of NPC1 in the filovirus entry process fall within the class of chemicals which act by interfering with the NPC1-GP interaction. Piperazine **3.0** was identified as an inhibitor of EBOV infection by incubating Vero cells with VSV particles pseudotyped with EBOV-GP and with native EBOV, in the presence of **3.0**. To improve the activity of compound **3.0**, a methoxycarbonyl benzyloxy group was added in the ortho position of the aromatic ring attached to piperazine, resulting in compound **3.47**. This led to an increase in activity from an IC₅₀ = 1.6 μM for **3.0** to an

M. Morales-Tenorio, T. Ginex, M.A. Cuesta-Geijo et al.

European Journal of Medicinal Chemistry 223 (2021) 113654

![Image Description](image.png)

Fig. 4. Crystallographic structures (surface representation) of the inactive EBOV-GP (PDB ID: 3CSY) [36] in presence of the glycan cap (in yellow) and primed EBOV-GPcl (PDB ID: 5F1B) [59] bound to the NPC1-C domain (in light blue). (A; B) Residues involved in the interaction of GPcl with Loop 1 (A) and Loop 2 (B) of NPC1-C are also reported as sticks.

IC<sub>50</sub> = 130 nM for **3.47** in the VSV pseudotypes. This class of antiviral compounds was found to inhibit the binding of GPcl to membranes expressing NPC1 in a concentration-dependent manner by co-immunoprecipitation. However, despite their potency, the physicochemical and pharmacokinetic properties of **3.47** were far from being optimal for *in vivo* studies. In fact, although it was tested in EBOV-infected mice, treatment with **3.47** resulted in no increase in survival probably due to limited drug-like properties [66]. Further studies were done in the attempt to improve its ADME properties by making structural modifications to the main core. The addition of a chlorine or fluorine group to the 5 position on the phenyl ring improved the antiviral activity by almost 10-fold, with the 5-chloro slightly preferred. The adamantane ring was substituted by a spiro-cyclopropane difluoro cyclohexane to improve stability in microsomes. After an extensive medicinal chemistry program, two optimized candidates were obtained, compounds **3.23** and **3.24** ([Table 1](#table-1)) showing an EC<sub>50</sub> values of 52 nM and 17 nM in native EBOV (Makona isolate), respectively. As expected, they were active not only in the Makona isolates, but also in the Mayinga and Kikwit isolates that caused the outbreaks in Africa in 2014, 1976 and 1995 respectively [67].

Two other compounds identified as disruptors of the *in vitro* interaction of primed EBOV-GP with NPC1 were **MBX2254** and **MBX2270** ([Table 1](#table-1)). These two small molecules were identified following a cell-based HTS using HIV pseudotyped viruses with EBOV-GP (HIV/EBOV-GP). Mechanism of action studies revealed that both induced cholesterol accumulation in late endosomes and lysosomes and inhibited the interaction of thermolysin-cleaved EBOV-GP with NPC1-C domain in a dose-dependent manner, suggesting that they could interfere with EBOV-GP/NPC1 interaction [68]. Recently, two diarylsulfide derivatives, **SC198** and **SC073**, and carbazole **SC816** ([Table 1](#table-1)) have been described as antiviral inhibitors in the low micromolar range in a lentiviral EBOV-GP-pseudotyped infection assay. Using an ELISA-based assay, authors found that these compounds inhibited the interaction between EBOV-GPcl and NPC1 domain C, similarly to the MBX series [69]. In all these studies, the hypothesis of a disruption of the complex formed between NPC1 and EBOV-GP was formulated from indirect experimental observations. No more structural details about how these compounds interfere with the NPC1-GP interaction were given. It is worth noting that only the C domain of NPC1 was used to study the mechanism of action of the antiviral activity of these series, thus leading to the potential risk of an oversimplification of the biological context. More studies and perhaps newly ad-hoc designed experimental models would be needed to shed light on the structural details responsible for the supposed mechanisms of action on NPC1. Indeed, a proximity ligation assay to detect *in situ* bindings of EBOV-GP to NPC1 in infected cells has recently been published [70]. This experiment will be very useful to decipher the mechanism of action of entry inhibitors.

There is another class of entry inhibitor that can allosterically interfere with NPC1/EBOV-GP interaction, in this case by acting at the level of the fusion loop at the GP<sub>1</sub>-GP<sub>2</sub> interface of EBOV-GP. This is the case of a benzodiazepine derivative **7** ([Table 2](#table-2)) discovered after an HTS cell-based assay with HIV pseudotyped viruses that inhibit also the replicative virus in the same range with an IC<sub>50</sub> = 10 μM. Computational studies suggest that it binds to a hydrophobic pocket presents at the interface between GP<sub>1</sub> and GP<sub>2</sub>. Residues involved are V66, L68, N69, L70, L184 and L186 from the GP<sub>1</sub> subunit, and Y517, M548, L554, and L558 on the GP<sub>2</sub> internal fusion loop. Subsequent mutational analysis sustains that N69, L70, L184, I185, L186, K190 and K191 are crucial for the binding of compound **7.71** This binding to EBOV-GP could negatively affect the stability of the glycoprotein at the GP<sub>1</sub>-GP<sub>2</sub> dimerization surface. This ligand-induced destabilization would diffuse along with the EBOV-GP structure up to the cavity that would accommodate NPC1-C, thus compromising the final protein-protein interaction at the binding site (NPC1-Bs) (see [Fig. 5](#fig-5)). Several other compounds have been reported to produce the same binding mechanism, which was also supported by crystallographic data. Selective estrogen receptor modulators (SERMs), such as toremifene ([Table 2](#table-2)) have been identified as antivirals after an HTS cell-based assay.

M. Morales-Tenorio, T. Ginex, M.A. Cuesta-Geijo et al.

European Journal of Medicinal Chemistry 223 (2021) 113654

Table 1  
Reported anti-EBOV compounds acting on NPC1/EBOV-GP interaction.

| Comp. | Chemical structure | In vitro activity | Additional studies available | Ref. |
|-------|---------------------|--------------------|------------------------------|------|
| 3.0   | ![](chemical_structure_3.0.png) | VSV/EBOV-GP: IC₅₀ = 1.6 μM | Inhibition of NPC1/EBOV-GPcl interaction (co-immunoprecipitation) | [23] |
| 3.47  | ![](chemical_structure_3.47.png) | VSV/EBOV-GP: IC₅₀ = 130 nM<br>EBOV (Makona):<br>EC₅₀ = 160 nM | Inhibition of NPC1/EBOV-GPcl interaction (co-immunoprecipitation)<br>In vivo mouse infection model: no increase in survival | [23,65,66] |
| 3.23  | ![](chemical_structure_3.23.png) | VSV/EBOV-GP: IC₅₀ = 28 nM<br>EBOV (Makona): EC₅₀ = 52 nM | Improved ADME properties | [67] |
| 3.24  | ![](chemical_structure_3.24.png) | VSV/EBOV-GP: IC₅₀ = 40 nM<br>EBOV (Makona): EC₅₀ = 17 nM | Improved ADME properties | [67] |
| MBX2254 | ![](chemical_structure_MBX2254.png) | HIV/EBOV-GP: IC₉₀ = 8.6 μmol/<br>L<br>EBOV:<br>IC₅₀ = 8.07 μmol/L | Inhibition of NPC1/EBOV-GPcl interaction (ELISA) | [68] |
| MBX2270 | ![](chemical_structure_MBX2270.png) | HIV/EBOV-GP:<br>IC₉₀ = 25.7 μmol/L<br>EBOV:<br>IC₅₀ = 10 μmol/L | Inhibition of NPC1/EBOV-GPcl interaction (ELISA) | [68] |
| SC198  | ![](chemical_structure_SC198.png) | HIV/EBOV-GP: IC₅₀ = 4.69 μM | Inhibition of NPC1/EBOV-GPcl interaction (ELISA) | [69] |
| SC073  | ![](chemical_structure_SC073.png) | HIV/EBOV-GP: IC₅₀ = 2.04 μM | Inhibition of NPC1/EBOV-GPcl interaction (ELISA) | [69] |
| SC816  | ![](chemical_structure_SC816.png) | HIV/EBOV-GP: IC₅₀ = 0.37 μM | Inhibition of NPC1/EBOV-GPcl interaction (ELISA) | [69] |

carried out with EBOV expressing the enhanced green fluorescent protein (eGFP). These SERMs were able to inhibit EBOV entry at later steps. In fact, studies about their mechanism of action revealed that the antiviral effect is not related to the estrogen-signalling pathway but mediated somehow by EBOV-GP [72]. This hypothesis was confirmed by the X-ray structure of toremifene (Table 2) in the hydrophobic cavity between GP₁ and GP₂. This interaction has been postulated to decrease the stability of GP, thus preventing viral fusion [73]. Other antiviral compounds reported to occupy the same cavity by crystallographic studies are benzotropine, sertraline, paroxetine and bepridil (Table 2) [74]. In the same way, the antidepressant and antipsychotic drugs imipramine, clomipramine and thioridazine also directly interact with and destabilise EBOV-GP (Table 2) [75] as well as members of a novel adamantane

M. Morales-Tenorio, T. Ginex, M.Á. Cuesta-Geijo et al.

European Journal of Medicinal Chemistry 223 (2021) 113654

Table 2  
Reported anti-EBOV compounds binding to a hydrophobic pocket in the interface between GP₁ and GP₂.

| Comp.       | Chemical structure                                                                 | In vitro activity                                                                 | Additional studies available                                                                                     | Reference |
|-------------|-----------------------------------------------------------------------------------|----------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|-----------|
| Compound 7  | ![](https://via.placeholder.com/100)                                              | HIV/EBOV-GP: IC₉₀ = 12.1 μM<br>EBOV: IC₅₀ = 10 μM                                   | Protein destabilization through binding at the GP₁-GP₂ interface (computational studies)                             | [71]      |
| Toremifene  | ![](https://via.placeholder.com/100)                                              | EBOV: IC₅₀ = 0.162 μM                                                             | Protein destabilization through binding at the GP₁-GP₂ interface (X-ray data)<br>In vivo mouse infection model: increase in survival |           |
| Benztrpine  | ![](https://via.placeholder.com/100)                                              | EBOV: IC₅₀ = 8.07 μM                                                              | Protein destabilization through binding at the GP₁-GP₂ interface (X-ray data)                                      | [74,78]   |
| Sertraline  | ![](https://via.placeholder.com/100)                                              | EBOV: IC₅₀ = 3.13 μM                                                              | Protein destabilization through binding at the GP₁-GP₂ interface (X-ray data)                                      | [74,78]   |
| Paroxetine  | ![](https://via.placeholder.com/100)                                              | EBOV: IC₅₀ = 7.45 μM                                                              | Protein destabilization through binding at the GP₁-GP₂ interface (X-ray data)                                      | [74,78]   |
| Bepridil    | ![](https://via.placeholder.com/100)                                              | EBOV: IC₅₀ = 5.08 μM                                                              | Protein destabilization through binding at the GP₁-GP₂ interface (X-ray data)                                      | [74,78]   |
| Imipramine  | ![](https://via.placeholder.com/100)                                              | EBOV-like particles: IC₅₀ = 13.7 μM                                                | Protein destabilization through binding at the GP₁-GP₂ interface (X-ray data)<br>In vivo mouse infection model: moderate increase in survival | [66,75,79] |
| Clomipramine | ![](https://via.placeholder.com/100)                                             | EBOV: IC₅₀ = 11.4 μM                                                              | Protein destabilization through binding at the GP₁-GP₂ interface (X-ray data)                                      | [75,78]   |
| Thioridazine | ![](https://via.placeholder.com/100)                                             | EBOV: IC₅₀ = 6.24 μM                                                              | Protein destabilization through binding at the GP₁-GP₂ interface (X-ray data)                                      | [75,78]   |

(continued on next page)

M. Morales-Tenorio, T. Ginex, M.A. Cuesta-Geijo et al.

European Journal of Medicinal Chemistry 223 (2021) 113654

Table 2 (continued)

| Comp. | Chemical structure | In vitro activity | Additional studies available | Reference |
|-------|---------------------|--------------------|------------------------------|-----------|
| Compound 38 | ![Chemical Structure](https://i.imgur.com/compound38.png) | VSV/EBOV-GP: EC₅₀ = 14 nM<br>EBOV: EC₅₀ = 0.24 μM | Protein destabilization through binding at the GP₁-GP₂ interface (X-ray data) | [76] |
| Compound 35 | ![Chemical Structure](https://i.imgur.com/compound35.png) | HIV/EBOV-GP: EC₅₀ = 12 nM<br>EBOV: EC₅₀ = 0.31 μM | Protein destabilization through binding at the GP₁-GP₂ interface (experimental data with GP mutants) | [77] |

![Figure 5](https://i.imgur.com/figure5.png)

Fig. 5. (A) Up, (B) side views and domain composition (C) for the homotrimeric EBOV-GPcl (PDB ID: 6QD8). [80] Binding modes for imipramine (D; PDB ID: 6G9B) [75], thioridazine (E; PDB ID: 6G95) [75], toremifene (F; PDB ID: 5JQ7) [73], and clomipramine (G; PDB ID: 6G9I) [75] are also reported. Residues of the Fusion Loop (FL) binding site mainly involved in protein-ligand interactions are highlighted as sticks.

carboxamide chemical series (compound 38, Table 2) [76]. Moreover, a new family of 4-(aminomethyl)benzamide-based entry inhibitors were proposed to act at the toremifene site according to molecular docking and experimental studies with GP mutants (compound 35, Table 2) [77]. Both new classes were initially tested in pseudotype viruses, and their activities were also confirmed against wild-type EBOV.

For some of the compounds reported in Table 2, experimental confirmation of this novel mode of action has been provided [73–75]. According to the crystallographic data, a druggable binding site is formed in correspondence of the fusion loop (FL in Fig. 5), between the GP₁ and GP₂ subunits. This hydrophobic cavity is targeted by imipramine, thioridazine, toremifene and clomipramine, which form hydrophobic contacts with I38, L43, R64, V66, and L184 of GP₁ and L558 of the GP1 subunit and with Y517, M548, L554, L558 of the fusion peptide in the GP₂ subunit. A very similar binding mode can be observed for imipramine and clomipramine (Fig. 5D and G respectively). The two benzene rings fused to a seven-membered azepine group occupy the right site of the cavity with the chlorine atom of clomipramine that is placed near Y517. A slight shift toward the R64 and Y517 can be observed in the crystallographic complex of EBOV-GPcl with toremifene (Fig. 5F), where an additional edge-to-face π-stacking interaction is formed between one benzene ring of the compound and Y517 at the FL-GP₂. Finally, a slightly different binding mode was observed for thioridazine (Fig. 5E) with the tricyclic moiety taking a vertical orientation into the FL binding site and the 1-methyl-piperidine moiety placed around V66, L184, Y517 and M548.

M. Morales-Tenorio, T. Ginex, M.A. Cuesta-Geijo et al.

5. Anti-NPC1 antibodies

As mentioned before, in 2020 a mixture of three mAbs was approved by the FDA, being the first FDA-approved treatment for EVD [47]. This has represented a major breakthrough for the control of this deadly disease. Besides the use of mAbs against the EBOV-GP [81], alternative strategies based on antibodies against cellular host proteins have been also proposed. This was the case of the development of bispecific-antibodies (bsAbs) capable to contemporary target key epitopes on both the EBOV-GP and the NPC1 receptor. Although it is still in its infancy, this strategy could allow to interfere with the endosomal delivery of the virus within the framework of a potentiated passive immunotherapy. Moreover, as NPC1 is the entry receptor of all known filovirus, the application of a multi-target immunotherapy through the combination of Abs targeting the NPC1/EBOV-GP interaction with those that are specific for a conserved and exposed epitope of known EBOV-GPs is expected to improve significantly the effectiveness of this strategy against multiple filoviruses [82].

6. NPC1 in other viral infections

NPC1 receptor is a common host target of several viral diseases that could be useful for the development of antiviral treatments. In fact, different studies have reported the relevance of NPC1 in several viral infections. In most cases, those reports presented the effect of NPC1 inhibitors such as hydrophobic amines, U18666A and imipramine (Fig. 2), used as pharmacological tools because of their ability to mimic the NP-C disease phenotype [27]. These drugs have shown inhibition of viral replication of distant viral families, independently of their type of viral genome. These viruses exploit NPC1 to gain the cytoplasm from the late endosomes and initiate the infection. NPC1 is a common entry host factor for the fusion of many enveloped viruses. However, others such as Vesicular Stomatitis Virus (Rhabdoviridae family) or Influenza A virus (Orthomyxoviridae family) infect in an NPC1-independent manner [22,83]. HIV-1 (Retroviridae family) seems to be dependent on the intracellular cholesterol trafficking pathway mediated by NPC1 as indicated by experiments conducted with cells from patients with NP-C disease and/or under U18666A treatment [84]. Additionally, the NPC1-like 1 receptor, expressed on the membrane of enterocytes and hepatocytes, was identified as a key entry factor for hepatitis C virus (HCV) (RNA enveloped virus of the Flaviviridae family) [25]. In fact, HCV redistributes free cholesterol and other lipids in the cell to its replication site [85], not only via the oxysterol-binding protein (OSBP) mediating transport at Golgi-Endoplasmic reticulum (ER) contact sites but also at ER-late endosome/lysosomes membrane contact sites [86,87]. Compounds U18666A and imipramine strongly impair the replication not only of Chikungunya virus (RNA enveloped virus of the Togaviridae family) but also of other members of the Flaviviridae family, including Zika Virus (ZIKV), West Nile Virus (WNV), Yellow Fever Virus (YFV) and Dengue Virus (DENV) without any secondary cytotoxic effects, similarly to HCV and Hepatitis B Virus (HBV) [25,26,88,89]. Baculovirus BmNVP (double-stranded DNA enveloped virus of the Baculoviridae family), a powerful agent for gene delivery and expression, and relevant in controlling insect populations, infects insect cells via NPC1 domain C, binding directly to the viral glycoprotein GP64, which is responsible for both receptor binding and fusion. As in the above-described viruses, BmNVP infection is impaired by treating cells with U18666A and imipramine [90]. Similarly, African swine fever virus (double-stranded DNA enveloped virus of the Asfarviridae family) entry and fusion were impaired in cells treated with U18666A [91], endosomal exit did not occur and virions were retained in cholesterol-laden endosomes.

Classical Swine Fever virus (single-stranded positive-sense RNA within the Flaviviridae family), was shown to be blocked at the replication stage but no effect was observed at binding or viral entry in U18666A treated cells [92].

In the present context, a possible relationship has been also suggested between severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) (RNA enveloped virus of the Coronoviridae family), the agent responsible for COVID-19 disease, and the NPC1 receptor [93,94]. We have recently experimentally confirmed the interaction of NPC1 with the SARS-CoV-2 nucleoprotein (N). Our findings demonstrate a key role for NPC1 in SARS-CoV-2 infection and lead us to propose this receptor as a potential therapeutic target for COVID-19 [95].

7. Final remarks

The development of broad-spectrum antivirals against emerging viruses is crucial, and therefore the identification of the most suitable host cell factors and pathways commonly used by different viruses is of utmost importance to develop effective antiviral drugs [96]. In the present review, the relevance of intracellular cholesterol traffic in the infectivity of divergent viruses and the possible role of NPC1 in these events has been shown. Thus, just as for certain viruses, its involvement could be related to intracellular cholesterol traffic. For others viruses such as EBOV, it serves as a key intracellular receptor to pass through the cell since the interaction between EBOV-GP and the C domain of NPC1 allows it. This is the reason why NPC1 could be considered an interesting therapeutic target to fight diseases caused by different viruses sharing the same entry mechanism. According to what discussed in the previous sections, molecules capable of perturbing NPC1/EBOV-GP protein-protein interaction might have the potential to combat the infection by blocking membrane fusion and thus virus entry. Antiviral drugs acting at the receptor-virus interaction surface could avoid possible toxic effects associated with NPC1 receptor blockade and eventually could have additional benefits derived from the combination of “hybrid” receptor-virus and virus-directed agents. As discussed in this review, several small molecules have been reported to act at this level and also bispecific-antibodies against both, EBOV-GP and NPC1 are under study.

Recently, a novel approach based on antisense oligonucleotides acting on NPC1 has been also reported with in vitro inhibitory efficacy [97]. This opens up the possibility of new therapeutic approaches in the context of EBOV infection. NPC1/EBOV-GP inhibitors might be useful therapeutics by themselves or in combination therapies, as is the case in infections such as HIV or HCV infections, in which the knowledge of key determinants of the viral cycle has resulted in successful therapeutic approaches [98].

Several points remain to be properly assessed. In fact, little data are available about the hypothesized mechanism of action of the antiviral compounds supposed to act by inhibiting or disrupting the NPC1/EBOV-GP interaction, thus more efforts are needed for a better comprehension of the structural features characterising the molecular target at the level of the NPC1/EBOV-GP interaction surface. This would permit a proper setup for a successful drug discovery protocol. Additionally, albeit the efficacy of compounds acting on the pseudotyped virus with the EBOV-GP being well correlated with efficacy in the replicative EBOV, in vivo proof of concept with the inhibitors of the NPC1/EBOV-GP interaction is needed to confirm their therapeutic potential.

In summary, although further research efforts are needed in the field, we believe that there are proofs showing that the NPC1 receptor could offer a great potential for the development of

M. Morales-Tenorio, T. Ginex, M.A. Cuesta-Geijo et al.

antivirals against EBOV infection. Moreover, given the previously commented evidences about the involvement of the common host target, NPC1 in other virus infection, a potential broad-spectrum antiviral activity might be also hypothesized for this class of inhibitors thus, allowing their effectiveness to be expanded beyond filoviruses.

### Declaration of competing interest

The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.

### Acknowledgments

The project leading to these results has received funding from “la Caixa” Banking Foundation under the project code LCF/PR/HR19/52160012. This research was partially supported through the Spanish Ministry of Science and Innovation (RTI2018-097305-R-I00), the Instituto de Investigación Carlos III, ISCIII, (FIS PI 1801007) and the European Commission Horizon 2020 Framework Programme Project VIRUSCAN FETPROACT-2016: 731868. M.M. holds a pre-doctoral FPU grant (FPU18/03493) from MICINN. Graphical abstract was done using templates from BioRender.

### Abbreviations

| ADME | Absorption, Distribution, Metabolism and Excretion |
| BOMV | Bombali virus |
| bsAbs | bispecific-antibodies |
| BUDV | Bundibugyo virus |
| COVID-19 | coronavirus infection disease |
| CTD | C-terminal domain |
| DENV | Dengue Virus |
| EBOV | Ebola virus |
| EBOV-GP | Ebola virus glycoprotein |
| ER | endoplasmic reticulum |
| EVD | Ebola virus disease |
| FDA | Food and Drug Administration |
| FL | fusion loop |
| GP | glycoprotein |
| GPcl | cleaved glycoprotein |
| HBV | Hepatitis B virus |
| HCV | hepatitis C virus |
| HIV | human immunodeficiency virus |
| HIV-1 | human immunodeficiency virus type-1 |
| HIV/EBOV-GP | HIV particles pseudotyped with the EBOV-GP |
| HTS | high-throughput screen |
| IC₅₀ | 50% inhibitory concentration |
| L | L-polymerase |
| LBPA | lysobisphosphatidic acid |
| LDLs | low-density lipoproteins |
| LE | late endosomes |
| Lys | lysosomes |
| mAbs | monoclonal antibodies |
| MARV | Marburg virus |
| MLD | middle luminal domain |
| NP | nucleoprotein |
| NP-C | Niemann-Pick type C disease |
| NPC1 | Niemann-Pick type C1 |
| NPC1-C | NPC1 receptor domain C |
| NPC2 | Niemann-Pick type C2 |
| NTD | N-terminal domain |
| OSBP | oxysterol-binding protein |
| PDB | protein data bank |

| RESTV | Reston virus |
| SARS-CoV-2 | severe acute respiratory syndrome coronavirus 2 |
| SERMs | selective estrogen receptor modulators |
| sGP | soluble glycoprotein |
| SSD | sterol-sensing domain |
| SUDV | Sudan virus |
| TAFV | Tai Forest virus |
| TM | transmembrane region |
| TPCs | two-pore channels |
| VP | viral proteins |
| VSV | vesicular stomatitis virus |
| VSV/EBOV-GP | VSV particles pseudotyped with the EBOV-GP |
| WNV | West Nile Virus |
| YFV | Yellow Fever Virus |
| ZIKV | Zika Virus |

### References

[1] E. Ikonen, Cellular cholesterol trafficking and compartmentalization, Nat. Rev. Mol. Cell Biol. 9 (2) (2008) 125–138.

[2] J. Luo, L. Jiang, H. Yang, B.L. Song, Routes and mechanisms of post-endosomal cholesterol trafficking: a story that never ends, Traffic 18 (4) (2017) 209–217.

[3] M. Mazzon, J. Mercer, Lipid interactions during virus entry and infection, Cell Microbiol. 16 (10) (2014) 1493–1502.

[4] N. Altan-Bonnet, Lipid tales of viral replication and transmission, Trends Cell Biol. 27 (3) (2017) 201–213.

[5] F.R. Maxfield, G. van Meer, Cholesterol, the central lipid of mammalian cells, Curr. Opin. Cell Biol. 22 (4) (2010) 422–429.

[6] K. Simons, E. Ikonen, How cells handle cholesterol, Science 290 (5497) (2000) 1721–1726.

[7] E. Ikonen, Mechanisms of cellular cholesterol compartmentalization: recent insights, Curr. Opin. Cell Biol. 53 (2018) 77–83.

[8] A. Niemann, Ein unbekanntes krankheitsbild, Jahrb. Kinderheilk. 79 (1914) 1–10.

[9] L. Pick, Niemann–Pick’s disease and other forms of so-called xanthomatosis, Am. J. Med. Sci. 185 (5) (1933) 615–616.

[10] T. Geberhiwot, A. Moro, A. Dardis, U. Ramaswami, S. Sirrs, M.P. Marfa, M.T. Vanier, M. Walterfang, S. Bolton, C. Dawson, B. Héron, M. Stampfer, J. Imrie, C. Hendriksz, P. Gissen, E. Crushell, M.J. Coll, Y. Nadjar, H. Klünemann, E. Mengel, M. Hrebicek, S.A. Jones, D. Ory, B. Bembi, M. Patterson, Consensus clinical management guidelines for Niemann–Pick disease type C, Orphanet J. Rare Dis. 13 (1) (2018) 50.

[11] E. Mengel, M. Pineda, C.J. Hendriksz, M. Walterfang, J.V. Torres, S.A. Kolb, Differences in Niemann–Pick disease type C symptomatology observed in patients of different ages, Mol. Genet. Metab. 120 (3) (2017) 180–189.

[12] J. Sokol, J. Blanchette–Mackie, H.S. Kruth, N.K. Dwyer, L.M. Amende, J.D. Butler, E. Robinson, S. Patel, R.O. Brady, M.E. Comly, M.T. Vanier, P.G. Pentchev, Type C Niemann–Pick disease. Lysosomal accumulation and defective intracellular mobilization of low density lipoprotein cholesterol, J. Biol. Chem. 263 (7) (1988) 3411–3417.

[13] E.D. Carstea, J.A. Morris, K.G. Coleman, S.K. Loftus, D. Zhang, C. Cummings, J. Gu, M.A. Rosenfeld, W.J. Pavan, D.B. Krizman, J. Nagle, M.H. Polymeropoulos, S.L. Sturley, Y.A. Ioannou, M.E. Higgins, M. Comly, A. Cooney, A. Brown, C.R. Kaneski, J. Blanchette–Mackie, N.K. Dwyer, E.B. Neufeld, T. Chang, L. Liscum, J.F. Strauss, K. Ohno, M. Zeigler, R. Carmi, J. Sokol, D. Markie, R.R. O’Neil, O.P. Van Diggelen, M. Elleder, M.C. Patterson, R.O. Brady, M.T. Vanier, P.G. Pentchev, D.A. Tagle, Niemann–Pick C1 disease gene: homology to mediators of cholesterol homeostasis, Science 277 (5323) (1997) 228–231.

[14] M.E. Higgins, J.P. Davies, F.W. Chen, Y.A. Ioannou, Niemann–Pick C1 is a late endosome-resident protein that transiently associates with lysosomes and the trans-Golgi network, Mol. Genet. Metab. 68 (1) (1999) 1–13.

[15] J.P. Davies, Y.A. Ioannou, Topological analysis of Niemann–Pick C1 protein reveals that the membrane orientation of the putative sterol-sensing domain is identical to those of 3-hydroxy-3-methylglutaryl-CoA reductase and sterol regulatory element binding protein cleavage-activating protein, J. Biol. Chem. 275 (32) (2000) 24367–24374.

[16] L.A. McCauliff, A. Langan, R. Li, O. Ilnytska, D. Bose, M. Waghalter, K. Lai, P.C. Kahn, J. Storch, Intracellular cholesterol trafficking is dependent upon NPC2 interaction with lysobisphosphatidic acid, Elife 8 (2019), e50832.

[17] H.J. Kwon, L. Abi–Mosleh, M.L. Wang, J. Deisenhofer, J.L. Goldstein, M.S. Brown, R.E. Infante, Structure of N-terminal domain of NPC1 reveals distinct subdomains for binding and transfer of cholesterol, Cell 137 (7) (2009) 1213–1224.

[18] D.E. Sleet, J.A. Wiseman, M. El–Banna, S.M. Price, L. Verot, M.M. Shen, G.S. Tint, M.T. Vanier, S.U. Walkley, P. Lobel, Genetic evidence for nonredundant functional cooperativity between NPC1 and NPC2 in lipid transport, Proc. Natl. Acad. Sci. U.S.A. 101 (16) (2004) 5886–5891.

[19] X. Li, F. Lu, M.N. Trinh, P. Schmiego, J. Seemann, J. Wang, G.3. Blobel, 3 Å

structure of Niemann–Pick C1 protein reveals insights into the function of the C–Terminal luminal domain in cholesterol transport, Proc. Natl. Acad. Sci. U.S.A. 114 (34) (2017) 9116–9121.

[20] P.E. Kuwabara, M. Labouesse, The sterol–sensing domain: multiple families, a unique role? Trends Genet. 18 (4) (2002) 193–201.

[21] X. Gong, H. Qian, X. Zhou, J. Wu, T. Wan, P. Cao, W. Huang, X. Zhao, X. Wang, P. Wang, Y. Shi, G.F. Gao, Q. Zhou, N. Yan, Structural insights into the Niemann–Pick C1 (NPC1)–mediated cholesterol transfer and Ebola infection, Cell 165 (6) (2016) 1467–1478.

[22] J.E. Carette, M. Raaben, A.C. Wong, A.S. Herbert, G. Obernosterer, N. Mulherkar, A.I. Kuehne, P.J. Kranzusch, A.M. Griffin, G. Ruthel, P.D. Cin, J.M. Dye, S.P. Whelan, K. Chandran, T.R. Brummelkamp, Ebola virus entry requires the cholesterol transporter Niemann–Pick C1, Nature 477 (7364) (2011) 340–343.

[23] M. Côté, J. Misasi, T. Ren, A. Bruchez, K. Lee, C.M. Filone, L. Hensley, Q. Li, D. Ory, K. Chandran, J. Cunningham, Small molecule inhibitors reveal Niemann–Pick C1 is essential for ebolavirus infection, Nature 477 (7364) (2011) 344–348.

[24] Y. Tang, I.C. Leao, E.M. Coleman, R.S. Broughton, J.E.K. Hildreth, Deficiency of Niemann–Pick type C–1 protein impairs release of Human Immunodeficiency virus type 1 and results in Gag accumulation in late endosomal/lysosomal compartments, J. Virol. 83 (16) (2009) 7982–7995.

[25] B. Sainz, N. Barretto, D.N. Martin, N. Hiraga, M. Imamura, S. Hussain, K.A. Marsh, X. Yu, K. Chayama, W.A. Alrefai, S.L. Uprichard, Identification of the Niemann–Pick C1–like 1 cholesterol absorption receptor as a new Hepatitis C virus entry factor, Nat. Med. 18 (2) (2012) 281–285.

[26] S. Wichit, R. Hamel, E. Bernard, L. Talignani, F. Diop, P. Ferraris, F. Liegeois, P. Ekchariyawat, N. Luplertlop, P. Surasombatpattana, F. Thomas, A. Merits, V. Choumet, P. Roques, H. Yssel, L. Briant, D. Missé, Imipramine inhibits Chikungunya virus replication in human skin fibroblasts through interference with intracellular cholesterol trafficking, Sci. Rep. 7 (1) (2017) 3145.

[27] K.W. Underwood, B. Andemariam, G.L. McWilliams, L. Liscum, Quantitative analysis of hydrophobic amine inhibition of intracellular cholesterol transport, J. Lipid Res. 37 (7) (1996) 1556–1568.

[28] J.H. Kuhn, S. Becker, H. Ebihara, T.W. Geisbert, K.M. Johnson, Y. Kawaoka, W.I. Lipkin, A.I. Negredo, S.V. Netesov, S.T. Nichol, G. Palacios, C.J. Peters, A. Tenorio, V.E. Volchkov, P.B. Jahrling, Proposal for a revised taxonomy of the family Filoviridae: classification, names of taxa and viruses, and virus abbreviations, Arch. Virol. 155 (12) (2010) 2083–2103.

[29] K.M. Johnson, J.V. Lange, P.A. Webb, F.A. Murphy, Isolation and partial characterisation of a new virus causing acute haemorrhagic fever in Zaire, Lancet 309 (8011) (1977) 569–571.

[30] T. Goldstein, S.J. Anthony, A. Gbakima, B.H. Bird, J. Bangura, A. Tremeau–Bravard, M.N. Belaganahalli, H.L. Wells, J.K. Dhanota, E. Liang, M. Grodus, R.K. Jangra, V.A. DeJesus, G. Lasso, B.R. Smith, A. Jambai, B.O. Kamara, S. Kamara, W. Bangura, C. Monagin, S. Shapira, C.K. Johnson, K. Saylors, E.M. Rubin, K. Chandran, W.I. Lipkin, J.A.K. Mazet, The discovery of Bombali virus adds further support for bats as hosts of ebolaviruses, Nat. Microbiol. 3 (10) (2018) 1084–1089.

[31] A.L. Rasmussen, Host factors in Ebola infection, Annu. Rev. Genom. Hum. Genet. 17 (1) (2016) 333–351.

[32] P. Aleksandrowicz, A. Marzi, N. Biedenkopf, N. Beimforde, S. Becker, T. Hoenen, H. Feldmann, H.J. Schnittler, Ebola virus enters host cells by macropinocytosis and clathrin–mediated endocytosis, J. Infect. Dis. 204 (Suppl 3) (2011) S957–S967.

[33] D. Cantoni, J.S. Rossman, Ebolaviruses: new roles for old proteins, PLoS Neglected Trop. Dis. 12 (5) (2018), e0006349.

[34] E.H. Miller, K. Chandran, Filovirus entry into cells—new insights, Curr. Opin. Virol. 2 (2) (2012) 206–214.

[35] J.E. Lee, E.O. Saphire, Ebolavirus glycoprotein structure and mechanism of entry, Future Virol. 4 (6) (2009) 621–635.

[36] J.E. Lee, M.L. Fusco, A.J. Hessell, W.B. Oswald, D.R. Burton, E.O. Saphire, Structure of the Ebola virus glycoprotein bound to a human survivor antibody, Nature 454 (7201) (2008) 177–182.

[37] C.P. Alvarez, F. Lasala, J. Carrillo, O. Muñiz, A.L. Corbí, R. Delgado, C–type lectins DC–SIGN and L–SIGN mediate cellular entry by Ebola virus in cis and in trans, J. Virol. 76 (13) (2002) 6841–6844.

[38] Y. Sakurai, A.A. Kolokoltsov, C.C. Chen, M.W. Tidwell, W.E. Bauta, N. Klugbauer, C. Grimm, C. Wahl–Schott, M. Biel, R.A. Davey, Two–pore channels control Ebola virus host cell entry and are drug targets for disease treatment, Science 347 (6225) (2015) 995–998.

[39] C.J. Penny, K. Vassileva, A. Jha, Y. Yuan, X. Chee, E. Yates, M. Mazzon, B.S. Kilpatrick, S. Muallem, M. Marsh, T. Rahman, S. Patel, Mining of Ebola virus entry inhibitors identifies approved drugs as two–pore channel pore blockers, Biochim. Biophys. Acta Mol. Cell Res. 1866 (7) (2019) 1151–1161.

[40] D.K. Das, U. Bulow, W.E. Diehl, N.D. Durham, F. Senjobe, K. Chandran, J. Luban, J.B. Munro, Conformational changes in the Ebola virus membrane fusion machine induced by pH, Ca²⁺, and receptor binding, PLoS Biol. 18 (2) (2020), e3000626.

[41] K. Chandran, N.J. Sullivan, U. Felbor, S.P. Whelan, J.M. Cunningham, Endosomal proteolysis of the Ebola virus glycoprotein is necessary for infection, Science 308 (5728) (2005) 1643–1645.

[42] S. Baize, D. Pannetier, L. Oestereich, T. Rieger, L. Koivogui, N. Magassouba, B. Soropogui, M.S. Sow, S. Keïta, H. De Clerck, A. Tiffany, G. Dominguez, M. Loua, A. Traoré, M. Kolié, E.R. Malano, E. Heleze, A. Bocquin, S. Mély,

H. Raoul, V. Caro, D. Cadar, M. Gabriel, M. Pahlmann, D. Tappe, J. Schmidt–Chanasit, B. Impouma, A.K. Diallo, P. Formenty, M.V. Herp, S. Günther, Emergence of Zaire Ebola virus disease in Guinea, N. Engl. J. Med. 371 (15) (2014) 1418–1425.

[43] O.I. Kalenga, M. Moeti, A. Sparrow, V.K. Nguyen, D. Lucey, T.A. Ghebreyesus, The ongoing Ebola epidemic in the democratic republic of Congo, 2018–2019, N. Engl. J. Med. 381 (4) (2019) 373–383.

[44] E.O. Saphire, A vaccine against Ebola virus, Cell 181 (1) (2020) 6.

[45] J. Cohen, Ebola outbreak continues despite powerful vaccine, Science 364 (6437) (2019) 223.

[46] S. Mulangu, L.E. Dodd, R.T. Davey, O.T. Mbaya, M. Proschan, D. Mukadi, M.L. Manzo, D. Nzolo, A.T. Oloma, A. Ibanda, R. Ali, S. Coulibaly, A.C. Levine, R. Grais, J. Diaz, C. Lane, J. Muyembe–Tamfum, A randomized, controlled trial of Ebola virus disease therapeutics, N. Engl. J. Med. 381 (24) (2019) 2293–2303.

[47] A. Markham, REGN–EB3: first approval, Drugs 81 (1) (2021) 175–178.

[48] https://www.fda.gov/drugs/drug-safety-and-availability/fda-approves-treatment-ebola-virus (accessed 2021–05–25).

[49] D. Sissoko, C. Laouenan, E. Folgesson, A.B. M'Lebing, A.H. Beavogui, S. Baize, A.M. Camara, P. Maes, S. Shepherd, C. Danel, et al., Experimental treatment with favipiravir for Ebola virus disease (the JIKI Trial): a historically controlled, single–arm proof–of–concept trial in Guinea, PLoS Med. 13 (3) (2016), e1001967.

[50] T.H.T. Nguyen, J. Guedj, X. Anglaret, C. Laouénan, V. Madelain, A.M. Taburet, S. Baize, D. Sissoko, B. Pastorino, A. Rodallec, G. Piorkowski, S. Carazo, M.N. Conde, J. Gala, J.A. Bore, C. Carbonnelle, F. Jacquot, H. Raoul, D. Malvy, X. de Lamballerie, Favipiravir pharmacokinetics in Ebola–infected patients of the JIKI trial reveals concentrations lower than targeted, PLoS Neglected Trop. Dis. 11 (2) (2017), e0005389.

[51] L. Oestereich, A. Lüdtke, S. Wurr, T. Rieger, C. Muñoz–Fontela, S. Günther, Successful treatment of advanced Ebola virus infection with T–705 (favipiravir) in a small animal model, Antivir. Res. 105 (1) (2014) 17–21.

[52] T.K. Warren, R. Jordan, M.K. Lo, A.S. Ray, R.L. Mackman, V. Soloveva, D. Siegel, M. Perron, R. Bannister, H.C. Hui, N. Larson, R. Strickley, J. Wells, K.S. Stuthman, S.A. Van Tongeren, N.L. Garza, G. Donnelly, A.C. Shurtleff, C.J. Retterer, D. Gharaibeh, R. Zamani, T. Kenny, B.P. Eaton, E. Grimes, L.S. Welch, L. Gomba, C.L. Wilhelmsen, D.K. Nichols, J.E. Nuss, E.R. Nagle, J.R. Kugelman, G. Palacios, E. Doerffler, S. Neville, E. Carra, M.O. Clarke, L. Zhang, W. Lew, B. Ross, Q. Wang, K. Chun, L. Wolfe, D. Babusis, Y. Park, K.M. Stray, I. Trancheva, J.Y. Feng, O. Barauskas, Y. Xu, P. Wong, M.R. Braun, M. Flint, L.K. McMullan, S. Chen, R. Fears, S. Swaminathan, D.L. Mayers, C.F. Spiropoulou, W.A. Lee, S.T. Nichol, T. Cihlar, S. Bavari, Therapeutic efficacy of the small molecule GS–5734 against Ebola virus in rhesus monkeys, Nature 531 (7594) (2016) 381–385.

[53] H. Feldman, T.W. Geisbert, Ebola haemorrhagic fever, Lancet 377 (9768) (2011) 849–862.

[54] C. Salata, A. Calistri, G. Alvisi, M. Celestino, C. Parolin, G. Palù, Ebola virus entry: from molecular characterization to drug discovery, Viruses 11 (3) (2019) 274.

[55] A.C. Wong, R.G. Sandesara, N. Mulherkar, S.P. Whelan, K. Chandran, A forward genetic strategy reveals destabilizing mutations in the ebolavirus glycoprotein that alter its protease dependence during cell entry, J. Virol. 84 (1) (2010) 163–175.

[56] R.J. Cenedella, Cholesterol synthesis inhibitor U18666A and the role of sterol metabolism and trafficking in numerous pathophysiological processes, Lipids 44 (6) (2009) 477–487.

[57] C. Rodriguez–Lafresse, R. Rousson, J. Bonnet, P.G. Pentchev, P. Louisot, M.T. Vanier, Abnormal cholesterol metabolism in imipramine–treated fibroblast cultures. Similarities with Niemann–Pick type C disease, Biochim. Biophys. Acta Lipids Lipid. Metabol. 1043 (2) (1990) 123–128.

[58] A.S. Herbert, C. Davidson, A.I. Kuehne, R. Bakken, S.Z. Braigen, K.E. Gunn, S.P. Whelan, T.R. Brummelkamp, N.A. Twenhafel, K. Chandran, S.U. Walkley, J.M. Dye, Niemann–Pick C1 is essential for ebolavirus replication and pathogenesis in vivo, mBio 6 (3) (2015) e00565–e00615.

[59] H. Wang, Y. Shi, J. Song, J. Qi, G. Lu, J. Yan, G.F. Gao, Ebola viral glycoprotein bound to its endosomal receptor Niemann–Pick C1, Cell 164 (1–2) (2016) 258–268.

[60] E.H. Miller, G. Obernosterer, M. Raaben, A.S. Herbert, M.S. Deffieu, A. Krishnan, E. Ndungo, R.G. Sandesara, J.E. Carette, A.I. Kuehne, G. Ruthel, S.R. Pfeffer, J.M. Dye, S.P. Whelan, T.R. Brummelkamp, K. Chandran, Ebola virus entry requires the host–programmed recognition of an intracellular receptor, EMBO J. 31 (8) (2012) 1947–1960.

[61] Y. Zhao, J. Ren, K. Harlos, D.I. Stuart, Structure of glycosylated NPC1 luminal domain C reveals insights into NPC2 and Ebola virus interactions, FEBS Lett. 590 (5) (2016) 605–612.

[62] A.N. Matthew, F. Leidner, G.J. Lockbaum, M. Henes, J. Zephyr, S. Hou, D.N. Rao, J. Timm, L.N. Rusere, D.A. Ragland, J.L. Paulsen, K. Prachanronarong, D.I. Soumana, E.A. Nalivaika, N.K. Yilmaz, C.A. Schiffer, Drug design strategies to avoid resistance in direct–acting antivirals and beyond, Chem. Rev. 121 (6) (2021) 3238–3270.

[63] K. Lin, P. Gallay, Curing a viral infection by targeting the host: the example of cyclophilin inhibitors, Antivir. Res. 99 (1) (2013) 68–77.

[64] N. Kumar, S. Sharma, R. Kumar, B.N. Tripathi, S. Barua, H. Ly, B.T. Rouse, Host–Directed antiviral therapy, Clin. Microbiol. Rev. 33 (3) (2020) e00168–19.

M. Morales-Tenorio, T. Ginex, M.A. Cuesta-Geijo et al.

[65] K. Lee, T. Ren, M. Côté, B. Gholamreza, J. Misasi, A. Bruchez, J. Cunningham, Inhibition of Ebola virus infection: identification of Niemann–Pick C1 as the target by optimization of a chemical probe, ACS Med. Chem. Lett. 4 (2) (2013) 239–243.

[66] A.S. Herbert, C. Davidson, A.I. Kuehne, R. Bakken, S.Z. Braigen, K.E. Gunn, S.P. Whelan, T.R. Brummelkamp, N.A. Twenhafel, K. Chandran, S.U. Walkley, J.M. Dye, Niemann–Pick C1 is essential for ebolavirus replication and pathogenesis in vivo 6 (3) (2015) e00565–15.

[67] H. Liu, Y. Tian, K. Lee, P. Krishnan, M.K.M. Wang, S. Whelan, E. Mevers, V. Soloveva, B. Dedic, X. Liu, J.M. Cunningham, Identification of potent Ebola virus entry inhibitors with suitable properties for in vivo studies, J. Med. Chem. 61 (14) (2018) 6293–6307.

[68] A. Basu, D.M. Mills, D. Mitchell, E. Ndungo, J.D. Williams, A.S. Herbert, J.M. Dye, D.T. Moir, K. Chandran, J.L. Patterson, L. Rong, T.L. Bowlin, Novel small molecule entry inhibitors of Ebola virus, J. Infect. Dis. 212 (Suppl 2) (2015) S425–S434.

[69] F. Lasala, A. García-Rubia, C. Requena, I. Galindo, M.A. Cuesta–Geijo, I. García-Dorival, P. Bueno, N. Labiod, J. Luczkowiak, A. Martinez, N.E. Campillo, C. Alonso, R. Delgado, C. Gil, Identification of potential inhibitors of protein–protein interaction useful to fight against Ebola and other highly pathogenic viruses, Antivir. Res. 186 (2021) 105011.

[70] E. Mittler, T. Alkutkar, R.K. Jangra, K. Chandran, Direct intracellular visualization of Ebola virus–receptor interaction by in situ proximity ligation, mBio 12 (1) (2021) e03100–e03120.

[71] A. Basu, B. Li, D.M. Mills, R.G. Panchal, S.C. Cardinale, M.M. Butler, N.P. Peet, H. Majgier-Baranowska, J.D. Williams, I. Patel, D.T. Moir, S. Bavari, R. Ray, M.R. Farzan, L. Rong, T.L. Bowlin, Identification of a small-molecule entry inhibitor for filoviruses, J. Virol. 85 (7) (2011) 3106–3119.

[72] L.M. Johansen, J.M. Brannan, S.E. Delos, C.J. Shoemaker, A. Stossel, C. Lear, B.G. Hoffstrom, L.E. DeWald, K.L. Schornberg, C. Scully, J. Lehár, L.E. Hensley, J.M. White, G.G. Olinger, FDA-approved selective estrogen receptor modulators inhibit Ebola virus infection, Sci. Transl. Med. 5 (190) (2013), 190ra79.

[73] Y. Zhao, J. Ren, K. Harlos, D.M. Jones, A. Zeltina, T.A. Bowden, S. Padilla–Parra, E.E. Fry, D.I. Stuart, Toremifene interacts with and destabilizes the Ebola virus glycoprotein, Nature 535 (7610) (2016) 169–172.

[74] J. Ren, Y. Zhao, E.E. Fry, D.I. Stuart, Target identification and mode of action of four chemically divergent drugs against Ebola virus infection, J. Med. Chem. 61 (3) (2018) 724–733.

[75] Y. Zhao, J. Ren, E.E. Fry, J. Xiao, A.R. Townsend, D.I. Stuart, Structures of Ebola virus glycoprotein complexes with tricyclic antidepressant and antipsychotic drugs, J. Med. Chem. 61 (11) (2018) 4938–4945.

[76] M.B. Plewe, N.V. Sokolova, V.R. Gantla, E.R. Brown, S. Naik, A. Fetsko, D.D. Lorimer, D.M. Dranow, H. Smutney, J. Bullen, R. Sidhu, A. Master, J. Wang, E.A. Kallel, L. Zhang, B. Kalveram, A.N. Freiberg, G. Henkel, K. McCormack, Discovery of adamantane carboxamides as Ebola virus cell entry and glycoprotein inhibitors, ACS Med. Chem. Lett. 11 (2020) 1160–1167.

[77] I.N. Gaisina, N.P. Peet, L. Wong, A.M. Schafer, H. Cheng, M. Anantpadma, R.A. Davey, G.R. Thatcher, L. Rong, Discovery and structural optimization of 4-(aminomethyl)benzamides as potent entry inhibitors of Ebola and Marburg virus infections, J. Med. Chem. 63 (13) (2020) 7211–7225.

[78] L.M. Johansen, L.E. DeWald, C.J. Shoemaker, B.G. Hoffstrom, C.M. Lear-Rooney, A. Stossel, E. Nelson, S.E. Delos, J.A. Simmons, J.M. Grenier, L.T. Pierce, H. Pajouhesh, J. Lehár, L.E. Hensley, P.J. Glass, J.M. White, G.G. Olinger, A screen of approved drugs and molecular probes identifies therapeutics with anti-Ebola virus activity, Sci. Transl. Med. 7 (290) (2015), 290ra89.

[79] J. Kouznetsova, W. Sun, C. Martínez–Romero, G. Tawa, P. Shinn, C.Z. Chen, A. Schimmer, P. Sanderson, J.C. McKew, W. Zheng, A. García–Sastre, Identification of 53 compounds that block Ebola virus-like particle entry via a repurposing screen of approved drugs, Emerg. Microb. Infect. 3 (12) (2014) e84.

[80] S.A. Ehrhardt, M. Zehner, V. Krähling, H. Cohen–Dvashi, C. Kreer, N. Elad, H. Gruell, M.S. Ercanoglu, P. Schommers, L. Gieselmann, R. Eggeling, C. Dahlke,

[81] T. Wolf, N. Pfeifer, M.M. Addo, R. Diskin, S. Becker, F. Klein, Polyclonal and convergent antibody response to Ebola virus vaccine RVSV–ZEBOV, Nat. Med. 25 (10) (2019) 1589–1600.

[82] K.L. O'Donnell, A. Marzi, Immunotherapeutics for Ebola virus disease: hope on the horizon, Biol. Targets & Ther. 15 (2021) 79–86.

[83] A.Z. Wec, E.K. Nyakatura, A.S. Herbert, K.A. Howell, F.W. Holtsberg, R.R. Bakken, E. Mittler, J.R. Christin, S. Shulenin, R.K. Jangra, S. Bharrhan, A.I. Kuehne, Z.A. Bornholdt, A.I. Flyak, E.O. Saphire, J.E. Crowe Jr., M.J. Aman, J.M. Dye, J.R. Lai, K. Chandran, A "Trojan horse" bispecific–antibody strategy for broad protection against ebolaviruses, Science 354 (6310) (2016) 350–354.

[84] S. Amini Bavil Olyaae, Y.J. Choi, J.H. Lee, M. Shi, I.C. Huang, M. Farzan, J.U. Jung, The antiviral effector IFITM3 disrupts intracellular cholesterol homeostasis to block viral entry, Cell Host Microbe 13 (4) (2013) 452–464.

[85] Y. Tang, I.C. Leao, E.M. Coleman, R.S. Broughton, J.E.K. Hildreth, Deficiency of Niemann–Pick type C–1 protein impairs release of Human Immunodeficiency virus type 1 and results in Gag accumulation in late endosomal/lysosomal compartments, J. Virol. 83 (16) (2009) 7982–7995.

[86] I.K. Stoeck, J.Y. Lee, K. Tabata, I. Romero-Brey, D. Paul, P. Schult, V. Lohmann, L. Kaderali, R. Bartenschlager, Hepatitis C virus replication depends on endosomal cholesterol homeostasis, J. Virol. 92 (1) (2017) e01196–17.

[87] H. Wang, J.W. Perry, A.S. Lauring, P. Neddermann, R. De Francesco, A.W. Tai, Oxysterol-binding protein is a phosphatidylinositol 4-kinase effector required for HCV replication membrane integrity and cholesterol trafficking, Gastroenterology 146 (5) (2014) 1373–1385.

[88] I. Khan, D.S. Katikaneni, Q. Han, L. Sanchez–Felipe, K. Hanada, R.L. Ambrose, J.M. Mackenzie, K.V. Konan, Modulation of hepatitis C virus genome replication by glycosphingolipids and four-phosphate adaptor protein 2, J. Virol. 88 (21) (2014) 12276–12295.

[89] J. Lucifora, K. Esser, U. Protzer, Ezetimibe blocks hepatitis B virus infection after virus uptake into hepatocytes, Antivir. Res. 97 (2) (2013) 195–197.

[90] J.F. Osuna–Ramos, J.M. Reyes–Ruiz, R.M. Del Ángel, The role of host cholesterol during flavivirus infection, Front. Cell Infect. Microbiol. 8 (2018) 388.

[91] Z. Li, Y. Fan, J. Wei, X. Mei, Q. He, Y. Zhang, T. Li, M. Long, J. Chen, J. Bao, G. Pan, C. Li, Z. Zhou, Baculovirus utilizes cholesterol transporter Niemann–Pick C1 for host cell entry, Front. Microbiol. 10 (2019) 2825.

[92] M.A. Cuesta–Geijo, L. Barrado–Gil, I. Galindo, R. Muñoz–Moreno, C. Alonso, Redistribution of endosomal membranes to the African swine fever virus replication site, Viruses 9 (6) (2017) 133.

[93] X.D. Liang, Y.N. Zhang, C.C. Liu, J. Chen, X.N. Chen, A.S. Balonch, B. Zhou, U18666A inhibits classical swine fever virus replication through interference with intracellular cholesterol trafficking, Vet. Microbiol. 238 (2019) 108436.

[94] S.L. Sturley, T. Rajakumar, N. Hammond, K. Higaki, Z. Márka, S. Márka, A.B. Munkacsi, Potential COVID–19 therapeutics from a rare disease: weaponizing lipid dysregulation to combat viral infectivity, J. Lipid Res. 61 (7) (2020) 972–982.

[95] R.A. Ballout, D. Sviridov, M.I. Bukrinsky, A.T. Remaley, The lysosome: a potential juncture between SARS–CoV–2 infectivity and Niemann–Pick disease type C, with therapeutic implications, Faseb. J. 34 (6) (2020) 7253–7264.

[96] I. García–Dorival, M.A. Cuesta–Geijo, I. Galindo, J. Urquiza, A. Del Puerto, C. Gil, N. Campillo, A. Martínez, C. Alonso, Identification of Niemann–Pick C1 receptor as a novel SARS-CoV-2 intracellular target, BioRxiv 12 (19) (2020) 423584, https://doi.org/10.1101/2020.12.19.423584.

[97] S.H.E. Kaufmann, A. Dorhoi, R.S. Hotchkiss, R. Bartenschlager, Host-directed therapies for bacterial and viral infections, Nat. Rev. Drug Discov. 17 (1) (2018) 35–56.

[98] A. Sadewasser, E. Dietzel, S. Michel, M. Klüver, M. Helfer, T. Thelemann, R. Klar, M. Eickmann, S. Becker, F. Jaschinski, Anti–Niemann Pick C1 single-stranded oligonucleotides with locked nucleic acids potentially reduce Ebola virus infection in vitro, Mol. Ther. Nucleic Acids 16 (2019) 686–697.

[99] V. Calvaruso, S. Petta, A. Craxì, Is global elimination of HCV realistic? Liver Int. 38 (Suppl 1) (2018) S40–S46.
