
# Assisted assembly of bacteriophage T7 core components for genome translocation across the bacterial envelope

Mar Pérez-Ruiz${}^{a,1}$, Mar Pulido-Cid${}^{a}$, Juan Román Luque-Ortega${}^{b}$, José María Valpuesta${}^{a}$, Ana Cuervo${}^{a,2}$, and José L. Carrascosa${}^{a,2}$

${}^{a}$Centro Nacional de Biotecnología, Madrid 28049, Spain; and ${}^{b}$Centro de Investigaciones Biológicas Margarita Salas, Madrid 28040, Spain

Edited by David J. DeRosier, Brandeis University, Waltham, MA, and approved July 14, 2021 (received for review December 29, 2020)

In most bacteriophages, genome transport across bacterial envelopes is carried out by the tail machinery. In viruses of the *Podoviridae* family, in which the tail is not long enough to traverse the bacterial wall, it has been postulated that viral core proteins assembled inside the viral head are translocated and reassembled into a tube within the periplasm that extends the tail channel. Bacteriophage T7 infects *Escherichia coli*, and despite extensive studies, the precise mechanism by which its genome is translocated remains unknown. Using cryo-electron microscopy, we have resolved the structure of two different assemblies of the T7 DNA translocation complex composed of the core proteins gp15 and gp16. Gp15 alone forms a partially folded hexamer, which is further assembled upon interaction with gp16 into a tubular structure, forming a channel that could allow DNA passage. The structure of the gp15-gp16 complex also shows the location within gp16 of a canonical transglycosylase motif involved in the degradation of the bacterial peptidoglycan layer. This complex docks well in the tail extension structure found in the periplasm of T7-infected bacteria and matches the sixfold symmetry of the phage tail. In such cases, gp15 and gp16 that are initially present in the T7 capsid eightfold-symmetric core would change their oligomeric state upon reassembly in the periplasm. Altogether, these results allow us to propose a model for the assembly of the core translocation complex in the periplasm, which furthers understanding of the molecular mechanism involved in the release of T7 viral DNA into the bacterial cytoplasm.

protein structure | cryo-EM | bacteriophage | DNA translocation | transglycosylase

Bacteriophages (phages) are viruses that infect bacteria and are considered to be the most abundant entities on Earth. Members of the order *Caudovirales* are double-stranded DNA (dsDNA) tailed phages (1, 2) that can infect a wide variety of hosts, and they are present in many different environments (3). These viruses possess a tail protein complex that is assembled at one special vertex of their icosahedral capsid, known as the portal vertex (4). Connector or portal proteins create an entry channel to the viral capsid and serve as a docking point for the tail complex (4, 5). Tailed phages generally show a common infection strategy and thus have strong structural homologies, although the protein machinery responsible for host adsorption has specific characteristics depending on each phage family (3). Some phages of the *Podoviridae* family present a short, noncontractile tail that cannot traverse the complex bacterial wall of gram-negative bacteria (6), whereas others do not present tail machinery at all. These latter viruses have developed an alternative mechanism to cross the bacterial envelope using internal capsid proteins or membranes that are able to assemble tubular structures during infection (7–11).

Bacteriophage T7 is a well characterized member of the *Podoviridae* family that infects *Escherichia coli*. The viral particle is composed of a 55-nm icosahedral capsid and a 23-nm short noncontractile tail (12). The most remarkable structure of the T7 viral particle is the internal core, a cylindrical structure that is ~290-Å long and ~170-Å wide located on top of the connector, which stabilizes the packaged DNA inside the capsid (13). This complex is made up of three proteins: gp14 (20.8 kDa), gp15 (84.2 kDa), and gp16 (144 kDa) (9, 13). These internal core proteins are not essential for morphogenesis of the viral capsid, but they are required to translocate the viral genome during the T7 infection process (11, 14, 15). A lytic transglycosylase motif present in gp16 is essential during infection at temperatures below 20 °C to overcome the highly cross-linked peptidoglycan (16–18).

As is the case for most phages, T7 first binds reversibly to a primary receptor that allows correct tail orientation in relation to the bacterial surface. Then, an irreversible interaction takes place with the rough lipopolysaccharide (LPS) (19), which causes conformational changes in the tail leading to the opening of the channel. Later, a tubular conduit is assembled, probably composed of the core proteins, and crosses the bacterial wall (9, 11), although it is not clear how this is accomplished. One hypothesis was proposed by Hu et al. (11) in a study using cryo-electron tomography, in which they described the presence of transient tubular structures during T7 infection. They proposed that the core complex formed by gp14, gp15, and gp16 could disassemble after adsorption and pass through the open tail channel in a completely or partially unfolded state (11). According to this hypothesis, partially unfolded

### Significance

Bacteriophage T7 infects *Escherichia coli* bacteria, and its genomic DNA traverses the bacterial cell envelope, but the precise mechanism used by the virus remains unknown. Previous studies suggested that proteins found inside the viral capsid (core proteins) disassemble and reassemble in the bacterial periplasm to form a DNA translocation channel. In this article, we have solved the structure of two different assemblies of the core proteins gp15 and gp16. These findings confirm the ability of core proteins to form tubes compatible with the periplasmic space and show the location of the transglycosylase enzyme involved in peptidoglycan degradation. Our results reveal key structural details of the assembly of the core translocation complex involved in the DNA transport through the bacterial wall.

Author contributions: M.P.-R., A.C., and J.L.C. designed research; M.P.-R., M.P.-C., J.R.L.-O., and A.C. performed research; M.P.-R., J.R.L.-O., J.M.V., A.C., and J.L.C. analyzed data; and M.P.-R., J.M.V., A.C., and J.L.C. wrote the paper.

The authors declare no competing interest.

This article is a PNAS Direct Submission.

Published under the [PNAS license](https://www.pnas.org/site/misc/terms.xhtml).

${}^{1}$Present address: Novo Nordisk Foundation Centre for Protein Research, University of Copenhagen, 2200 Copenhagen, Denmark.

${}^{2}$To whom correspondence may be addressed. Email: acuervo@cnb.csic.es or jlcarras@cnb.csic.es.

This article contains supporting information online at [https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2026719118/-/DCSupplemental](https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2026719118/-/DCSupplemental).

Published August 20, 2021.
gp14 would be ejected through the channel of the tail complex and then refolded to form a pore in the outer membrane of *E. coli*, which allows unfolded gp15 and gp16 to cross (11, 18). Once in the periplasm, gp15 and gp16 oligomerize as a tubular structure that spans the periplasm and internal membrane and reaches the cytoplasm. When the channel formed by gp14, gp15, and gp16 is complete, translocation of the T7 genome into the bacteria cytoplasm can take place.

Here, we report two cryo-electron microscopy (cryo-EM) structures of the T7 core assemblies: gp15 alone (505 kDa) and that of the complex formed by gp15 and gp16 (gp15–gp16 complex; 1.365 MDa). The gp15 protein alone forms a tubular structure in vitro, although its carboxyl-terminal half is disordered. The gp15–gp16 complex is also a tubular structure, but this time with a fully folded gp15. Although only a small fragment of gp16 is observed in the structure of the complex, the solved model comprises the transglycosylase domain. In this article, we have shown that these bacteriophage proteins, which form part of the mature virus in the fully structured core complex with eightfold symmetry (13, 15, 20, 21), are able to unfold during the infection process, exit the phage, and reassemble into a hexameric tubular structure whose size is compatible with the translocation of viral DNA across the bacterial envelope.

### Results

#### Phage T7 Gp15 Forms Tubular Oligomers In Vitro.
Recombinant gp15 was expressed, purified (*SI Appendix*, Fig. S1 A, Left), and subsequently characterized by analytical centrifugation (AUC) (*SI Appendix*, Fig. S1B) and dynamic light scattering (DLS) (*SI Appendix*, Fig. S1C). The two techniques revealed that more than 80% of the sample was present as an oligomer of ~540 kDa (a calculation based on sedimentation coefficient [AUC] and diffusion coefficient [DLS]), compatible with the presence of a hexamer, and less than 10% was a monomer regardless of the concentration of sample used (from 0.25 mg/mL to 1 mg/mL) (*SI Appendix*, Fig. S1 B, C, and E). Aliquots of the purified protein were negatively stained and observed by transmission electron microscopy (*SI Appendix*, Fig. S2A). A total of 1,648 particles were selected and processed as described in Methods. The two-dimensional (2D) classification rendered two main classes (*SI Appendix*, Fig. S2B), one with a cylindrical shape and hosting a channel, and the other, presumably the orthogonal view, with a sixfold symmetry that strengthens the notion of gp15 forming a hexamer. The three-dimensional (3D) reconstruction confirmed the tubular shape of the oligomer, ~130-Å long and an average width of ~100 Å (*SI Appendix*, Fig. S2C).

We next resorted to cryo-EM to try to generate a high-resolution structure of the gp15 oligomer. For that, aliquots of gp15 were vitrified as described in Methods. An FEI/Thermo Fisher ScientificTalos Arctica equipped with a Falcon III was used for data acquisition. A total of 2,470 movies were acquired (*SI Appendix*, Fig. S3A), and 444,807 total particles were picked, from which, after classification, 50,980 were selected as described in Methods (*SI Appendix*, Fig. S3B). A 3.64-Å resolution map of the oligomer was obtained in which C6 symmetry was applied (Fig. 1 and *SI Appendix*, Fig. S3 C–E). The resolution seemed to be very similar throughout all the 3D reconstruction, which points to a very rigid structure being formed (*SI Appendix*, Fig. S3E). The shape of the oligomeric gp15 is that of a nozzle of 140-Å length and 88 Å at the widest point (Fig. 1A). A channel runs along the longitudinal axis, with a wider entrance at one end (36 Å) and a narrower one at the other (23 Å) but always wide enough to allow the passage of DNA (Fig. 1B).

The high resolution of the map allowed us an ab initio tracing of the gp15 polypeptide chain, from residues K59 to L374, that encompassed 15 α-helices and 2 β-strands (Fig. 1 C–E). The structure is composed of three helical bundle domains (HBD) that we have named nozzle (residues 59 through 170), HBD1 (residues 171 through 302), and HBD2, which contains the long α-helix 15 (residues 303 through 374) (Fig. 1E). This leaves undetermined the first 58 residues and the last 373 ones. The absence of more than half of the protein density in the reconstructed hexamer could be due either to protein degradation, domain flexibility, or partial folding. However, our analytical ultracentrifugation and DLS assays (*SI Appendix*, Fig. S1 D and E) unambiguously confirmed the presence of an oligomer whose molecular mass is that of a full-length hexameric gp15. The possibility of the oligomer being fragmented during the vitrification process (22) could also be ruled out, as the dimensions of the gp15 oligomer 3D reconstructed using negative staining agree with those of the vitrified complex (compare Fig. 1A with *SI Appendix*, Fig. S2C). However, it is interesting to note a blurry area in the 2D classes of both negatively stained and vitrified samples in the area that we assigned to the carboxyl-terminal region (arrow in *SI Appendix*, Figs. S2B and S3B), suggesting that this region of the polypeptide chain is disordered rather than degraded. In summary, purified gp15 uses its N-terminal half to form a tubular, hexameric structure, whereas the carboxyl-terminal half is disordered and does not contribute to the final structure.

#### T7 Gp16 Stabilizes and Promotes the Folding of the Full Structure of the Gp15 Hexamer.
Recombinant gp16 core protein was also expressed and purified (*SI Appendix*, Fig. S1 A, Center), with a monomeric mass of ~150 kDa, similar to what is predicted from its amino acid sequence. The gp16 oligomeric state was analyzed using AUC and DLS, and the two techniques pointed to the protein being present as a monomer (*SI Appendix*, Fig. S1 B, D, and E). Negative-staining electron microscopy revealed the presence of proteins of small size and aggregates (*SI Appendix*, Fig. S2D), and its heterogeneity prevented the use of cryo-EM for higher-resolution characterization.

Since the internal core of T7 is formed by proteins gp14, gp15, and gp16, we next wanted to know what sort of structure is formed by proteins gp15 and gp16. For this purpose, we coincubated both proteins, which resulted in the formation of a high-molecular-mass complex. AUC experiments revealed, together with the presence of free gp16, another peak with an experimental sedimentation coefficient of 15.1 S, a value whose correction to standard conditions (s20, w = 23.6 S) was compatible with a molecular mass of around 1.4 MDa (*SI Appendix*, Fig. S1 B and E). This molecular mass is in agreement with the presence of a complex made of the hexameric gp15 already established (500 kDa) and another six monomers of gp16 (~865 kDa).

The gp15–gp16 complex was next purified by ultracentrifugation in a glycerol gradient, which confirmed the formation of the high-molecular-mass complex (compare the fraction of the gp15–gp16 complex with regard to those of gp15 or gp16; *SI Appendix*, Fig. S1 A, Right). The complex was first analyzed by negative-staining EM that showed a homogeneous tubular assembly of similar shape to that of gp15, albeit much longer (*SI Appendix*, Fig. S2E). This was confirmed by the 3D reconstruction of the gp15–gp16 complex. A total of 2,553 particles were used for a 2D classification, which again revealed two main classes (*SI Appendix*, Fig. S2F), one corresponding to the end-on view, which clearly showed the sixfold symmetry already observed in the gp15 oligomer, and a side view with a cylindrical shape, again similar, albeit longer than that of gp15. The 3D reconstruction carried out using these classes and imposing sixfold symmetry reinforced the notion of the gp15–gp16 complex having a tubular structure of ~250-Å long and an average width of ~120 Å (*SI Appendix*, Fig. S2G).

We next resorted to cryo-EM to generate a high-resolution structure of the gp15–gp16 complex. For that, aliquots of gp15–gp16 were vitrified, and the best grids were loaded onto an FEI/Thermo Fisher Scientific Titan Krios cryoelectron microscope operated at 300 kV equipped with a Gatan K2 summit detector. A total of 5,506 movies were acquired (*SI Appendix*, Fig. S4A), and 828,762 total particles were picked, from which, after classification, 72,882 were selected as described in Methods (*SI Appendix*,
Fig. 1. Structure of the bacteriophage T7 gp15 core protein. (A) Side, top, and bottom views of the cryo-EM map of gp15 assembly at 3.6-Å resolution showing the dimensions of the complex. (B) Longitudinal cut of the gp15 cryo-EM density map showing the dimensions of the internal channel. (C) Ribbon representation of the gp15 atomic model. Each monomer is depicted in a different color. (D) Docking of the gp15 atomic model into the corresponding cryo-EM density map. (E) Ribbon representation of a gp15 monomer; the structure is colored showing the different domains, and the N- and carboxyl-terminal residues are labeled.

Fig. S4B). A 3.19-Å resolution map of the gp15–gp16 complex with imposed C6 symmetry was finally generated (Fig. 2 and SI Appendix, Fig. S4 C–E). The 3D reconstruction of the complex revealed a tubular structure 210 Å long, 70 Å longer than the gp15 oligomer and 138 Å at its widest point (Fig. 2A). The tube is asymmetric with two different halves: one narrower and identical to the gp15 oligomeric structure (Fig. 2B) and a wider one with two protrusions, one roughly at the center of the tube that we called wing (arrow in Fig. 2A) and the other at its widest point (Fig. 2A and B). The channel that runs along the whole structure is widest in its central part (31 to 37 Å) and has the two narrowest points at the two entrances of the structure (21 and 23 Å; Fig. 2C).

As with the gp15 oligomer, ab initio tracing of the polypeptide chain of the gp15–gp16 complex was straightforward (Fig. 2 D–F), except for the wings, which were less defined (SI Appendix, Figs. S2F and S4B). As described in the previous section, half of the structure coincided with that of the gp15 hexamer, and the tracing of this part gave identical results (Fig. 2 B, E, and F). Surprisingly, tracing of the rest of the density showed that the majority of the newly reconstructed polypeptide chain corresponded to regions of the gp15 peptide that were not visible in the 3D reconstruction of the gp15 oligomer. This time, we were able trace the gp15 polypeptide chain from residues E60 to E701, which included 30 α-helices and four β-strands (Fig. 2 D–F) and that could be divided in six domains: the previously described nozzle domain, HBD1, and HBD2 plus the newly visualized α-15 (now from residues A303 through N396), HBD3 (residues G397 through N524) and base domain (residues G525 through E701)

(Fig. 2D). The search in the DALI server did not reveal any structurally homologous protein. The 3D reconstruction revealed that gp15 entirely builds the channel of the complex, and analysis of the electrostatic surface of the channel walls showed an overall negative charge with three positive rings at the nozzle, α-15, and basal domains (Fig. 2G). The atomic structure of gp15 in complex with gp16 strongly suggests that our first interpretation of the gp15 hexamer structure was correct, and the regions not previously observed in the gp15 hexamer were indeed disordered. This clearly indicates that gp16 stabilizes and assists gp15 in reaching its full, native structure.

T7 Gp16 Shows a Conserved Transglycosylase Site that Is Visualized in the 3D Reconstruction of the Gp15–Gp16 Complex. As the gp15 sequence was almost completely traced (Fig. 2D), the only regions not determined in the volume of the gp15–gp16 complex were the wings, which were assigned to gp16 (Fig. 2 E and F). Although the wings presented lower resolution than the rest of the structure (SI Appendix, Fig. S4E), the density was good enough to allow the tracing of 196 residues (SI Appendix, Fig. S4F). The determined structure, when compared to others in the DALI server (23), had a high similarity to that of the *E. coli* SltY lytic transglycosylase (Fig. 3). This agreed with previous studies (17, 24) that have already shown that the N-terminal sequence of the T7 gp16 protein presents homology with the carboxyl-terminal sequence of the *E. coli* lytic transglycosylase SltY (Fig. 3). This reinforced the tracing of a part of gp16, which encompasses residues Y4 to R224 (Fig. 2H), with the exception of two loops

Fig. 2. Structure of the bacteriophage T7 gp15–gp16 tubular core complex. (A) Side, top, and bottom views of the cryo-EM map of the gp15–gp16 complex at 3.19-Å resolution showing the dimensions of the complex. The wing domain is pointed out with an arrow. (B) Overlapping of the gp15–gp16 complex (transparent) and gp15 cryo-EM maps (solid). (C) Longitudinal cut showing the dimensions of the internal channel. (D) Ribbon representation of gp15; the structure is colored showing the different domains, and the N- and carboxyl-terminal residues are labeled. (E) Ribbon representation of the gp15–gp16 atomic model; gp15 is shown in turquoise with one subunit colored in green and gp16 in purple with one subunit colored in red. (F) Overlapping of the gp15–gp16 atomic model with its cryo-EM density map. (G) Electrostatic potential showing a longitudinal cut of gp15–gp16 indicating the inner channel surface (Right). Blue color represents a 10 kcal/(mol · e) positive potential, while red represents a –10 kcal/(mol· e) negative potential. The electropositive-charged rings in the internal channel are highlighted with arrows. (H) Ribbon representation gp16 monomer; the structure is colored showing the different domains, and the N- and carboxyl-terminal residues are labeled.

(residues F40 to M54 and L66 to R76). The reconstructed part of gp16 can be divided into three parts: the transglycosylase domain (residues Y4 to V145), a long, disordered region, the loop domain (residues A146 to P205), and two small helices that we called the tip domain (residues F206 to R224) (Fig. 2H). These results show that the gp15 and gp16 interact to form a stable complex in which gp15, which by itself is an oligomer with only half of its sequence structured, gets completely folded with the assistance of gp16. Paradoxically, 85% of gp16 remains unstructured or flexible in the gp15–gp16 complex, despite the fact it stabilizes the overall gp15 carboxyl-terminal conformation. As described, the largest visible part of gp16 in the gp15–gp16 complex is the transglycosylase domain, which consists of seven α-helices and, as expected, docking of this domain into the E. coli SltY catalytic domain showed a good overlapping of the two structures (Fig. 3A–C) (17, 25), from L15 in helix α-1 to L143 in helix α-7 (RMSD 0.951 Å). There were some regions whose overlap was slightly worse (Fig. 3B): the gp16 helices α-3 and helix α-5 are inclined around 10° with respect to SltY helices α-31 and α-33; as mentioned before, the loops between gp16 helix α-2 to helix α-3 (residues F40 to M54) and helix α-3 to helix α-4 (residues L66 to R76) were not properly defined and could not be traced, and the loop between gp16 helix α-5 and helix α-6 is shorter in the case of gp16 and lies in a different position.

The analysis of both proteins demonstrated that gp16 and SltY share the same structural arrangement of the active site, including the well-characterized catalytic residue E37, found at the end of helix α-2 (16, 17). E37 in α-2 overlaps with the active-site amino acid SltY-E478 in α-30 (Fig. 3C), confirming the main role of this residue in peptidoglycan degradation during the infection process (16). Concerning conserved residues important for the architecture of the active site in SltY (17, 26), while the GXMQ site is not well observed in gp16, the AYNXG site (in gp16 helix α-5) showed a perfect overlapping (Fig. 3B and C). Surprisingly,

Fig. 3. The T7 gp16 transglycosylase domain. (A) Ribbon representation of the reconstructed part of the gp16 monomer when bound to gp15 (purple), docked into the *E. coli* SltY lytic transglycosylase domain (PDB 1SLY, green). (B) Zoom-in of the dashed square shown in A, with arrows pointing to the gp16 α-helices. (C) Overlapping of gp16 and SltY helices involved in the active site: gp16 helices α-2, α-3, and α-5 (purple) and SltY helices α-30, α-31 and α-33 (green). Conserved gp16 residues (in light pink) E37 (catalytic glutamate), A107, Y108, and N109; and conserved SltY residues (in light green) E478 (catalytic glutamate), A551, Y552, and N553 are shown in a stick representation. (D) Sequence alignment of gp16 (from M1 to L146) and *E. coli* SltY genes (from L456 to L592) obtained with ESPrit 3 software (48); identical residues are highlighted in red, both catalytic glutamates are shown with an asterisk. Secondary structure elements and residue number for both proteins are also indicated.

the residues in SltY helix α-33 involved in interaction with peptidoglycan (17, 26) do not show exactly the same position as their gp16 counterparts. As mentioned before, its equivalent helix (gp16 helix α-5) is turned about 10° with respect to that of StlY (Fig. 3B), suggesting some structural flexibility in this region. The existence of certain flexibility in the transglycosylase domain is also supported by the lower resolution of this area compared to the rest of the structure (*SI Appendix*, Fig. S4E). We suggest that certain flexibility might be needed in this domain to perform its biological function as a peptidoglycan-degrading enzyme.

**Structural Characterization of the Interactions that Stabilize the Gp15–G16 Complex.** We then analyzed in detail the interactions that induce stabilization of gp15 upon interaction with a small part of gp16 (Fig. 4). This interaction is of a staggered nature, with one subunit of gp16 interacting with two gp15 subunits: on one hand, the transglycosylase domain and the loop domain of a gp16 molecule interact with the HBD2 and the α-15 domains of one gp15 subunit (Fig. 4 $A$, $B$, and $C$), and on the other hand, continuation of the loop and the tip domain of the same gp16 molecule interact with the HBD3 and the base domain of the adjacent gp15 subunit (Fig. 4 $D-F$). Gp15 builds the inner core of the complex and presents the highest intermonomeric surface of interaction with 36,715 Å² of contacts between protomers and a total buried surface of 220,290 Å². In contrast, the gp16 subunits hardly interact with each other in the resolved structure, and they depend on the formation of an initial gp15 hexamer to assemble as an oligomer regardless of the concentration tested (*SI Appendix*, Fig. S1 $B$, $D$, and $E$). Gp16 interacts with gp15 through the transglycosylase and tip domains, but the most remarkable interaction is through its 50-residue loop, which follows closely the newly folded domains of gp15: the end of the α-helix 15 (Fig. 4 $B$ and $C$) as well as the HBD3 and the base domains of the adjacent gp15 subunit (Fig. 4 $E$ and $F$). These interactions, based on hydrophobic contacts (Fig. 4 $B$ and $E$), salt bridges, and hydrogen bonds (Fig. 4 $C$ and $F$), are probably key

Fig. 4. Structural analysis of the gp15–gp16 complex interaction. (A) Representation of two subunits of gp15 (in ribbon blue and turquoise) and one subunit of gp16 (in surface purple) showing the interaction of one gp16 subunit with two gp15 molecules. The location of the domains is labeled. (B) Hydrophobic interaction between gp15 and gp16 showing a detailed view of the square depicted in A. Gp16 is showed in hydrophobic surface calculated with Chimera software (47) according to the Kyte and Doolittle scale (54), orange represents more hydrophobic residues, white neutral and tan polar residues. Four hydrophobic clusters are pointed out with an arrow. (C) The same view in B showing one subunit of gp15 (blue) and gp16 (purple) in ribbon representation. In C, the residues involved in hydrogen bond and salt bridge formation between gp15 α15 (D375, D385, K389, Q390) and gp16 loop (Q180, Q184, P187, S189) are indicated. The position of the last residue of gp15 α-15 folded in the gp15 structure solved alone is indicated with an arrow. (D) The same representation as in A in a 180° rotation. (E) Hydrophobic interaction between gp15 and gp16 showing a detailed view of the square depicted in D. Gp15 hydrophobic surface was calculated the same as in B and follows the same color scale. The four hydrophobic clusters are pointed out with an arrow. (F) The same view in E showing gp15 (turquoise) and gp16 (purple) in ribbon representation. The residues involved in hydrogen bond and salt bridge formation between gp15 HBD3 and base domains (N426, K446, R494, K599, N624, Q627) and gp16 loop and tip (T190, D193, Q199, K204, D209, D219) are indicated. For clarity, only one subunit of gp15 protein is shown in panels B, C, and F.

in assisting the complete folding of gp15 and thus in the gp15–gp16 complex formation.

**Discussion**

The mechanism of DNA ejection of bacteriophage T7 has been the subject of intense study and controversy for many years. The T7 core proteins assemble inside the capsid as a toroidal eightfold symmetry structure surrounded by the DNA (13), and it has been postulated that they unfold and refold as a tubular structure

in the periplasmic space (11). In this article, we describe the structure of the gp15–gp16 complex formed by the two main components of the gp14–gp15–gp16 core tail tube extension complex. The atomic model of gp15–gp16 solved by cryo-EM shows that these two T7 core proteins are able to assemble in vitro into a hexameric tubular structure, very different from the eightfold toroidal structure assembled in the capsid, supporting the hypothesis that the T7 core can build an “extensible tail.” The length of the complex (around 210 Å) is compatible
with the dimensions of the bacterial periplasmic space [usually between 130 and 250 Å (27)]. In order to check this hypothesis, we docked our 3D reconstruction of the gp15–gp16 in vitro complex into the cryoelectron tomography T7 reconstruction obtained by Hu and colleagues (11) (Fig. 5A). Docking of our gp15–gp16 complex into the corresponding part of the bacteriophage reconstruction was compatible with the overall dimensions of the previously solved structure (11), suggesting that our 3D model assembled in vitro could be the T7 transitory translocation conduit for DNA ejection. The absence of capsid and portal protein interactions in our in vitro system, as is also the case in the reassembly of the proteins in the periplasmic space, could have favored that the core proteins adopt the tubular core conformation. The structure of the gp15–gp16 complex has a negatively charged channel with an overall diameter of 30 Å, which can accommodate a dsDNA molecule. Nevertheless,

the entrance and exit of the channel are narrowed (21 to 24 Å, Fig. 2C), thus suggesting that the gp15–gp16 complex characterized in this work could be an intermediate state of the translocation complex.

Although only 206 residues of the gp16 protein were solved in our translocation complex, this structure shows the key interactions that are required for the full folding of gp15 and also the presence of a canonical transglycosylase domain (Fig. 3), which shares the same structural arrangement of the active site with sltY *E. coli* transglycosylase enzyme (Fig. 3C). Observation of the full gp15–gp16 complex size by AUC strongly suggests that the missing gp16 region in this work is disordered and thus not observed in our structural studies. It is important to reinforce the notion that due to their biological function, gp15 and g16 proteins, which are fully structured in the mature virus and form part of the eightfolded core complex (13, 15, 20, 21), are able to

A

![Diagram](https://i.imgur.com/1234567.png)

B

Step 1 Monomer  
Step 2 Partially folded tubular oligomer  
Step 3 Fully folded tubular oligomer  
Step 4 Undetermined  

![Diagram](https://i.imgur.com/8901234.png)

Fig. 5. Assembly of the T7 translocation complex in the bacterial periplasmic space. (A) Docking of the 3D reconstruction of the tubular core complex assembled in vitro (gp15 in light blue, gp16 in purple) into the 3D model of bacteriophage T7 during DNA ejection obtained by cryo-electron tomography (cryo-ET) in Hu et al. (11). (B) Schematic of the proposed assembly model of the gp15 and gp16 tubular core complex during infection, showing T7 bacteriophage (gray); *E. coli* bacterial wall composed of the internal membrane (IM), outer membrane (OM), and peptidoglycan (PG), and the assembly of the viral proteins gp14 (red), gp15 (green) and gp16 (purple). The three proteins form part of the tubular core complex, a structured assembly present at the vertex of the mature phage. After interaction with the bacterial membrane, the three proteins undergo a sequential unfolding to traverse the phage tail. Gp14 refolds and forms an oligomer that allows the passing of gp15 through the OM (Step 1). Gp15 hexamerizes part of its structure (Step 2) and requires the presence of gp16 to form the complete hexameric tubular structure, which is long enough to reach the IM (Step 3). Although most of gp16 is still unstructured, a transglycosylase domain is formed that hydrolyzes the PG layer. A yet-unknown factor is needed for the total assembly of gp16, and thus the formation of fully structured and active periplasmic tubular core complex is ready for the translocation of the viral genome (Step 4).

Pérez-Ruiz et al.  
Assisted assembly of bacteriophage T7 core components for genome translocation across the bacterial envelope   PNAS | 7 of 10  
https://doi.org/10.1073/pnas.2026719118unfold to traverse the viral tail channel and reach the periplasm and refold in a different oligomeric state as a hexamer, probably improving its interaction with the tail complex. In our studies, we have captured, in part, this unfolded intermediate state for gp15, which forms hexamers on its own in which only half of the sequence is structured. These results are also supported by previous circular dichroism studies (28), in which they showed that gp15 alone presented only 39.5% of α-helical content instead of the 54% calculated for fully folded gp15. The interaction of gp15 and gp16 in the periplasm induces the folding of the whole gp15 and a small portion of gp16 including the transglycosylase domain, but the rest of the protein remains unstructured. Further interactions with other phage proteins (such as gp14) or bacterial membrane components (18, 28) may be essential to promote the full assembly of gp16 and to trigger the conformational changes needed in the complex to ensure a fully open translocation path.

The intermediate structures characterized in this work allow us to propose a model for the assembly of gp15 and gp16 proteins outside the phage structure during T7 infection (Fig. 5B). These two proteins, together with gp14, form the core complex of the mature virus (Fig. 5B). Interaction of tail fibers with the T7 LPS receptor triggers the conformational changes needed for tail opening and subsequent capsid core disassembly (19). Previous work has suggested that gp14 is the first to exit the viral particle to build a pore in the outer bacterial membrane (Fig. 5B, step 1) (18, 29). Monomers of gp15 are then released from the viral capsid into the *E. coli* periplasmic space and interact with the gp14 pore, probably through the N-terminal, electropositive area of gp15 (Fig. 2G). In a following step, gp15 monomers oligomerize into a partially folded hexameric tube (Fig. 5B, step 2). Subsequently, the arrival of gp16 and its interaction with the gp15 hexamer generates the gp15–gp16 complex and the transglycosylase motifs for bacterial peptidoglycan degradation, thus puncturing the peptidoglycan layer (Fig. 5B, step 3). Gp16 acts as a sort of chaperone by assisting the folding of the full structure of the gp15 hexamer, which extends its tubular structure to reach the inner bacterial membrane. In a final, as yet nonstructurally characterized step (Fig. 5B, step 4), the partially folded gp15–gp16 complex could interact with the inner bacterial membrane, especially thanks to gp16’s ability to associate with lipid bilayers (18, 28). Thus, it may allow formation of the fully structured and active periplasmic tubular core complex ready for the translocation of the viral genome. The lack of structure in the preliminary steps for both gp15 and gp16 could permit their exit from the phage head, traversing the gp14 pore, and only after a proper interaction (gp15 with gp16, gp16 with a yet unknown molecule) reach their final, structured conformation.

### Methods

#### Protein Expression and Purification. The bacteriophage T7 *gp15* gene was inserted into the pRSET-B vector (*amp<sup>r</sup>* and 6x-His tag) from Invitrogen between the XhoI and EcoRI restriction sites. The gp15 protein was overexpressed in *E. coli* C41 (DE3) grown in Luria-Bertani broth supplemented with 100 mg/mL ampicillin at 37 °C, with induction by 1 mM isopropyl-β-D-1-thiogalactopyranoside for 3 h when an optical density (O.D.<sub>600</sub>) of 0.4 to 0.6 was reached. Bacterial cells were resuspended in 100 mM sodium citrate dihydrate, 10 mM MgCl<sub>2</sub>, 1 mM phenylmethyl sulfonyl fluoride (PMSF), 100 mM Tris pH 8.0, and Complete Protease Inhibitor Mixture Tablets (Roche) and lysed using a cell disruptor. Cell debris was discarded after centrifugation for 15 min at 3,000 × *g*. After that, gp15 was purified following two chromatographic steps. Briefly, the protein was purified, always at 10 °C, and loaded onto a HisTrap high performance column equilibrated in buffer A: 100 mM Tris pH 8.0, 100 mM sodium citrate dihydrate, 10 mM MgCl<sub>2</sub>, and 1 mM PMSF and eluted in buffer A with 250 mM imidazole. The second step of purification consisted of a size-exclusion chromatography using a Superdex200 HiLoad column equilibrated in buffer A. Finally, the purified protein was concentrated using Amicon Ultra 50 k (Millipore).

The bacteriophage T7 *gp16* gene was cloned into the pET-16b vector (*amp<sup>r</sup>* and 10x-His tag) from Novagen and inserted between *Ndel* and *Xhol* restriction sites. Overexpression of gp16 in *E. coli* C41 (DE3) was carried out under specific conditions due to the toxic effects of its transglycosylase activity. The culture was grown at 37 °C using autoinduction media (30). Afterward, when the O.D.<sub>600</sub> reached ~0.5, the culture was transferred to 16 °C for 48 h. The protein was harvested and purified following the same steps described for gp15.

The gp15–gp16 tubular core complex was obtained in vitro after incubation of the purified proteins for 30 min at room temperature in a thermo block with shaking (Eppendorf). The proteins were incubated using 180 pmol each at a 1:1 ratio. The complexes were purified in a 10 to 40% (vol/vol) glycerol gradient in buffer A, and the gradient was formed into Gradient Station Bio-comp station with 53s/86°/20 rpm. The gradient was stored ~1 h at 4 °C, and then the complexes were separated by ultracentrifugation at 165,500 × *g* for 17h using a SW55 Ti rotor at 10 °C. The fractions were collected from bottom to top and checked in a 10% sodium dodecyl sulfate-polyacrylamide gel electrophoresis gel.

#### Sample Checking by Negative-Staining Electron Microscopy. After obtaining pure and suitable gp15, gp16, and gp15–gp16 core proteins, a preliminary electron microscopy evaluation was performed using negative staining. The samples were transferred to an 80 kV JEM1010 (Jeol) with a complementary metal oxide semiconductor 4 × 4 sensor (TemCam F416) yielding a pixel size of 2.5 Å/pix at the Electron Microscopy Services of the Centro de Biología Molecular Severo Ochoa (CBMSO-CSIC), and data processing was performed using the same workflow in each sample, using the Scipion software framework (31). The contrast transfer function (CTF) was estimated using ctffind4 (32). Particles were manually picked using XMIPP (33), and 2D and 3D classification was performed using RELION, the latter applying C6 symmetry (34). A total of 1,648 and 2,553 particles were used to 3D reconstruct gp15 and gp15–gp16, respectively, at a low resolution (~20 to 22 Å).

#### Cryo-EM Sample Preparation and Data Acquisition. Grids containing either gp15 or the gp15–gp16 tubular core complex were vitrified using a Thermo Fisher Scientific Vitrobot Mark IV. R2/2 Quantifoil grids were glow-discharged for 1 min and loaded with 3 μL purified sample at around 1 mg/mL for gp15 and 1.5 mg/mL for the gp15–gp16 tubular core complex. Samples of gp15 and the gp15–gp16 complex were then incubated for 3 or 1 min, respectively, at 95% humidity and 22 °C inside the Vitrobot, cleaned, and blotted twice in buffer A to remove the excess glycerol in the buffer prior to freezing. Grids were then blotted for 3.5 s with forces between –5 and –7 and plunge-frozen in liquid ethane.

The gp15 grids were transferred to a Talos Arctica (Thermo Fisher Scientific) electron microscope operated at 200 kV (at the cryo-EM Service, Centro Nacional de Biotecnología (CNB), and 2,470 movies fractioned in 62 frames were recorded in an automated fashion on a Falcon III (Thermo Fisher Scientific) detector using EPU (Thermo Fisher Scientific) with a pixel size of 0.855 Å/pix; fraction exposure time was ~35 s, and the total accumulated dose was ~30 e<sup>−</sup>/Å<sup>2</sup> (~0.48 e<sup>−</sup>/Å<sup>2</sup>/frame) ([SI Appendix](#app1){ref-type="sec"}, [Table S1](#tS1){ref-type="table"}).

For the gp15–gp16 complex, the grids were transferred to a Titan Krios (Thermo Fisher Scientific) electron microscope operated at 300 kV (at the Electron Bio-Imaging Centre \[eBIC\], Diamond Light Source). A total of 5,506 movies fractioned in 32 frames were collected on a K2 Summit detector (Gatan) using EPU (Thermo Fisher Scientific) with a pixel size of 1,047 Å/pix, 8 s of fraction exposure time, and total accumulated dose of ~40 e<sup>−</sup>/Å<sup>2</sup> (~1.25 e<sup>−</sup>/Å<sup>2</sup>/frame) ([SI Appendix](#app1){ref-type="sec"}, [Table S1](#tS1){ref-type="table"}).

#### Image Processing and Map Calculation. Cryo-EM data processing was performed using the Scipion software framework (31). Dose-fractionated image stacks were motion-corrected and dose-weighted using MotionCor2 (35). The CTF was estimated using ctffind4 and xmipp3 software (32, 33) for the gp15 data set and GCTF software (36) for the tubular core complex data set. Particles of gp15 were picked using Gautomatch (developed by Zhang K, MRC Laboratory of Molecular Biology), and the gp15–gp16 tubular core complex was picked using RELION (34). The particles extracted were classified using RELION 2D and 3D (34, 37), and the initial volume was built using Ransac (33). C6 symmetry was applied for gp15 and the gp15–gp16 complex. The volumes were obtained using RELION 3D autorefine (38) with 50,980 and 72,882 particles for gp15 and the gp15–gp16 complex, respectively. In the case of the gp15–gp16 data, multiple rounds of CTF refinement and Bayesian polishing (39) were performed before the final 3D autorefinement and postprocessing with RELION. In all the cases, structure resolutions were estimated from RELION Fourier shell correlation curves applying the 0.143 cutoff criteria (34, 40, 41), and local resolutions were computed with MonoRes (42). Final volumes for gp15 and the gp15–gp16 complex were respectively postprocessed for local sharpening with LocaldeBlur (43), using MonoRes (42) volume as input.Model Building and Coordinate Refinement. The long version of gp15 tubular core protein present in the gp15–gp16 complex was traced ab initio with Coot (44), using the PSIPRED server (45) to predict secondary structure as a guide. This structure was then used as a template to trace the short version of the partially folded gp15. Gp16 was first traced ab initio with Coot using the PSIPRED (45) secondary structure prediction information; this initial model was used as input in the DALI server (23), allowing us to identify the homology of the traced region of gp16 with SltY (1sly) and Mltc (4cfo) transglycosylases that were then used as a guide to build the final gp16 model. In both cases, atomic models were refined using PHENIX real space refinement (46). Refinement and validation scores are shown in *SI Appendix*, Table S1.

Structure Analysis. Protein interaction surfaces and residues involved were analyzed with the PDBePISA server. Electrostatic potential surface and hydrophobic surfaces were calculated with Chimera (47). The searches for homologous structures was carried out using the DALI server (23). Protein sequence alignment was performed with ESPrit 3 software (48), and the secondary structure chart of the traced proteins was obtained with ENDscript 2.0 software (48).

Sedimentation Velocity Assays. Sedimentation velocity assays were performed at the Molecular Interaction Facility at the Centro de Investigaciones Biológicas Margarita Salas (CIB-CSIC). A total of 320 μL aliquots of gp15, gp16, and the gp15–gp16 complex were loaded onto analytical ultracentrifugation cells at concentrations of 1, 0.5, and 0.2 mg/mL in 100 mM sodium citrate dihydrate, 10 mM MgCl₂ 100 mM Tris, pH 8.0 buffer. The experiments were carried out at 10 °C and 43,000 rpm in an XL-I analytical ultracentrifuge (Beckman-Coulter Inc.) equipped with both ultraviolet–visible spectroscopy absorbance and Raleigh interference detection systems, using an An-50Ti rotor and 12-mm Epon-charcoal standard double-sector centerpieces. Sedimentation profiles were recorded at 280 nm, and differential sedimentation coefficient distributions were calculated by least-squares boundary modeling of sedimentation velocity data using the continuous distribution c(s) Lamm equation model as implemented by SEDFIT (49). These s values were corrected to standard conditions (water, 20 °C, and infinite dilution) (50) using the program SEDNTERP (51) to get the corresponding standard s values (*s*₂₀,*w*).

DLS. DLS experiments were also carried out at the Molecular Interaction Facility at the CIB-CSIC. The experiments were performed using a Protein Solutions DynaPro MS/X instrument (Protein Solutions) at 20 °C and a 90° light scattering cuvette. The samples, under the same experimental conditions used in AUC, were centrifuged for 10 min at 12,000 × g and 21 °C just before measurements. Data were collected and analyzed with Dynamics V6 Software.

Estimation of Molar Mass of the Core Proteins from Hydrodynamic Measurements. The apparent molar mass of single sedimenting solute species (*M*) was calculated using measured values of the sedimentation coefficients (*s*) and the diffusion coefficient *D* according to the Svedberg equation (52). The estimated molar mass obtained using this relation is independent of the shape of the sedimenting/diffusing species, as frictional coefficients for sedimentation and diffusion cancel in the derivation of the equation.

Data Availability. The electron microscopy maps were deposited in the Electron Microscopy Data Bank ([https://www.ebi.ac.uk/emdb/](https://www.ebi.ac.uk/emdb/)) under accession codes EMD-10911 and EMD-10912. The atomic coordinates structure factors were deposited in the Protein Data Bank ([https://www.rcsb.org](https://www.rcsb.org)) under accession codes 6YSZ and 6YT5.

Note Added in Proof. During the review of this manuscript, an article by Swanson et al. (53) appeared, which solved the structure of the T7 ejectosome, very similar to the gp15-gp16 tubular core complex described here.

ACKNOWLEDGMENTS. This work was supported by the Ministry of Science, Innovation and Universities of Spain, Grant BFU 2014-54181 (to J.L.C.) and Contracts SEV-2013-0347 (to A.C.) and BES-2015-073615 to (M.P.-R.). We thank Dr. Luis Alberto Campos for his advice and discussion of the results. This work benefited from access to the Image Processing Center (I2PC), an Instruct-ERIC centre. Financial support was provided by Instruct-ERIC (PID 7366). We wish to thank the personnel of the following facilities for help during electron microscopy checking, cryo-EM, and molecular interactions data acquisition. Rhe Electron Microscopy Services of the CBMSO-CSIC is acknowledged. The access to the cryo-EM CNB-CSIC facility in the context of the CRIOMECORR Project (ESFRI-2019-01-CSIC-16) is acknowledged, as is the Molecular Interactions Facility of the CIB-CSIC in Madrid (Spain). We are grateful to the eBIC at Diamond Light Source in Didcot (United Kingdom), for access to its cryo-EM facility thanks to the proposal EM15997. We are grateful to Erney Ramírez-Aportela for his help in the application of sharpening methods to improve the density of the gp15–gp16 complex map. The professional editing service NB Revisions was used for technical preparation of the text prior to submission. The CNB is a Severo Ochoa Center of Excellence (Ministerio de Asuntos Económicos y Transformación Digital Award SEV 2017-0712).

1. A. Bertin, M. de Frutos, L. Letellier, Bacteriophage-host interactions leading to genome internalization. *Curr. Opin. Microbiol.* **14**, 492–496 (2011).
2. A. A. Aksyuk, M. G. Rossmann, Bacteriophage assembly. *Viruses* **3**, 172–203 (2011).
3. F. L. Nobrega *et al.*, Targeting mechanisms of tailed bacteriophages. *Nat. Rev. Microbiol.* **16**, 760–773 (2018).
4. A. Cuervo, J. L. Carrascosa, Viral connectors for DNA encapsulation. *Curr. Opin. Biotechnol.* **23**, 529–536 (2012).
5. S. R. Casjens, The DNA-packaging nanomotor of tailed bacteriophages. *Nat. Rev. Microbiol.* **9**, 647–657 (2011).
6. S. R. Casjens, I. J. Molineux, Short noncontractile tail machines: Adsorption and DNA delivery by podoviruses. *Adv. Exp. Med. Biol.* **726**, 143–179 (2012).
7. L. Sun *et al.*, Icosahedral bacteriophage ΦX174 forms a tail for DNA transport during infection. *Nature* **505**, 432–435 (2014).
8. B. Peralta *et al.*, Mechanism of membranous tunnelling nanotube formation in viral genome delivery. *PLoS Biol.* **11**, e1001667 (2013).
9. A. Bhardwaj, A. S. Olia, G. Cingolani, Architecture of viral genome-delivery molecular machines. *Curr. Opin. Struct. Biol.* **25**, 1–8 (2014).
10. C. Wang, J. Tu, J. Liu, I. J. Molineux, Structural dynamics of bacteriophage P22 infection initiation revealed by cryo-electron tomography. *Nat. Microbiol.* **4**, 1049–1056 (2019).
11. B. Hu, W. Margolin, I. J. Molineux, J. Liu, The bacteriophage t7 virion undergoes extensive structural remodeling during infection. *Science* **339**, 576–579 (2013).
12. A. Cuervo *et al.*, Structures of T7 bacteriophage portal and tail suggest a viral DNA retention and ejection mechanism. *Nat. Commun.* **10**, 3746 (2019).
13. X. Agirrezabala *et al.*, Maturation of phage T7 involves structural modification of both shell and inner core components. *EMBO J.* **24**, 3820–3829 (2005).
14. G. S. Roeder, P. D. Sadowski, Bacteriophage T7 morphogenesis: Phage-related particles in cells infected with wild-type and mutant T7 phage. *Virology* **76**, 263–285 (1977).
15. M. E. Cerritelli, J. F. Conway, N. Cheng, B. L. Trus, A. C. Steven, Molecular mechanisms in bacteriophage T7 procapsid assembly, maturation, and DNA containment. *Adv. Protein Chem.* **64**, 301–323 (2003).
16. M. Moak, I. J. Molineux, Peptidoglycan hydrolytic activities associated with bacteriophage virions. *Mol. Microbiol.* **51**, 1169–1183 (2004).
17. M. Moak, I. J. Molineux, Role of the Gp16 lytic transglycosylase motif in bacteriophage T7 virions at the initiation of infection. *Mol. Microbiol.* **37**, 345–355 (2000).
18. S. Leptihn, J. Gottschalk, A. Kuhn, T7 ejectosome assembly: A story unfolds. *Bacteriophage* **6**, e1128513 (2016).
19. V. A. González-García *et al.*, Conformational changes leading to T7 DNA delivery upon interaction with the bacterial receptor. *J. Biol. Chem.* **290**, 10038–10044 (2015).
20. F. Guo *et al.*, Visualization of uncorrelated, tandem symmetry mismatches in the internal genome packaging apparatus of bacteriophage T7. *Proc. Natl. Acad. Sci. U.S.A.* **110**, 6811–6816 (2013).
21. W. Chen *et al.*, Structural changes of a bacteriophage upon DNA packaging and maturation. *Protein Cell* **11**, 374–379 (2020).
22. A. J. Noble *et al.*, Reducing effects of particle adsorption to the air-water interface in cryo-EM. *Nat. Methods* **15**, 793–795 (2018).
23. L. Holm, DALI and the persistence of protein shape. *Protein Sci.* **29**, 128–140 (2020).
24. H. Engel, B. Kazemier, W. Keck, Murein-metabolizing enzymes from Escherichia coli: Sequence analysis and controlled overexpression of the slt gene, which encodes the soluble lytic transglycosylase. *J. Bacteriol.* **173**, 6773–6782 (1991).
25. A. M. Thunnissen *et al.*, Doughnut-shaped structure of a bacterial muramidase revealed by X-ray crystallography. *Nature* **367**, 750–753 (1994).
26. E. J. van Asselt, A. M. Thunnissen, B. W. Dijkstra, High resolution crystal structures of the Escherichia coli lytic transglycosylase Slt70 and its complex with a peptidoglycan fragment. *J. Mol. Biol.* **291**, 877–898 (1999).
27. G. Seltmann, O. Holst, “Periplasmic space and rigid layer” in *The Bacterial Cell Wall*, G. Seltmann, O. Holst, Eds. (Springer, Berlin, Heidelberg, 2002), pp. 103–132.
28. D. Lupo *et al.*, The T7 ejection nanomachine components gp15-gp16 form a spiral ring complex that binds DNA and a lipid membrane. *Virology* **486**, 263–271 (2015).
29. C. Y. Chang, P. Kemp, I. J. Molineux, Gp15 and gp16 cooperate in translocating bacteriophage T7 DNA into the infected cell. *Virology* **398**, 176–186 (2010).
30. F. W. Studier, Protein production by auto-induction in high density shaking cultures. *Protein Expr. Purif.* **41**, 207–234 (2005).
31. J. M. de la Rosa-Trevín *et al.*, Scipion: A software framework toward integration, reproducibility and validation in 3D electron microscopy. *J. Struct. Biol.* **195**, 93–99 (2016).
32. A. Rohou, N. Grigorieff, CTFFIND4: Fast and accurate defocus estimation from electron micrographs. *J. Struct. Biol.* **192**, 216–221 (2015).

Pérez-Ruiz et al.  
Assisted assembly of bacteriophage T7 core components for genome translocation across the bacterial envelope   PNAS | 9 of 10  
https://doi.org/10.1073/pnas.2026719118
33. J. M. de la Rosa-Trevín et al., Xmipp 3.0: An improved software suite for image processing in electron microscopy. *J. Struct. Biol.* **184**, 321–328 (2013).

34. S. H. Scheres, RELION: Implementation of a Bayesian approach to cryo-EM structure determination. *J. Struct. Biol.* **180**, 519–530 (2012).

35. S. Q. Zheng et al., MotionCor2: Anisotropic correction of beam-induced motion for improved cryo-electron microscopy. *Nat. Methods* **14**, 331–332 (2017).

36. K. Zhang, Gctf: Real-time CTF determination and correction. *J. Struct. Biol.* **193**, 1–12 (2016).

37. S. H. Scheres, Processing of structurally heterogeneous cryo-EM data in RELION. *Methods Enzymol.* **579**, 125–157 (2016).

38. J. Zivanov et al., New tools for automated high-resolution cryo-EM structure determination in RELION-3. *eLife* **7**, e42166 (2018).

39. J. Zivanov, T. Nakane, S. H. W. Scheres, A Bayesian approach to beam-induced motion correction in cryo-EM single-particle analysis. *IUCrJ* **6**, 5–17 (2019).

40. S. H. Scheres, S. Chen, Prevention of overfitting in cryo-EM structure determination. *Nat. Methods* **9**, 853–854 (2012).

41. P. B. Rosenthal, R. Henderson, Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy. *J. Mol. Biol.* **333**, 721–745 (2003).

42. J. L. Vilas et al., MonoRes: Automatic and accurate estimation of local resolution for electron microscopy maps. *Structure* **26**, 337–344.e4 (2018).

43. E. Ramirez-Aportela et al., Automatic local resolution-based sharpening of cryo-EM maps. *Bioinformatics* **36**, 765–772 (2020).

44. P. Emsley, K. Cowtan, Coot: Model-building tools for molecular graphics. *Acta Crystallogr. D Biol. Crystallogr.* **60**, 2126–2132 (2004).

45. D. W. Buchan, F. Minneci, T. C. Nugent, K. Bryson, D. T. Jones, Scalable web services for the PSIPRED Protein Analysis Workbench. *Nucleic Acids Res.* **41**, W349-57 (2013).

46. P. V. Afonine et al., Real-space refinement in PHENIX for cryo-EM and crystallography. *Acta Crystallogr. D Struct. Biol.* **74**, 531–544 (2018).

47. E. F. Pettersen et al., UCSF Chimera--a visualization system for exploratory research and analysis. *J. Comput. Chem.* **25**, 1605–1612, 10.1002/jcc.20084 (2004).

48. X. Robert, P. Gouet, Deciphering key features in protein structures with the new ENDscript server. *Nucleic Acids Res.* **42**, W320-4 (2014).

49. P. Schuck, Size-distribution analysis of macromolecules by sedimentation velocity ultracentrifugation and lamm equation modeling. *Biophys. J.* **78**, 1606–1619 (2000).

50. K. E. Holde, W. C. Johnson, P. S. Ho, *Principles of Physical Biochemistry* (Pearson Prentice Hall, 1985).

51. T. Laue, B. Shah, T. Ridgeway, S. Pelletier, “Computer-aided interpretation of analytical sedimentation data for proteins” in *Analytical Ultracentrifugation in Biochemistry and Polymer Science*, S. Harding, A. Rowe, J. Horton, Eds. (Royal Society of Chemistry, Cambridge, 1992), pp. 90–125.

52. T. Svedberg, K. O. Pedersen, *The Ultracentrifuge* (Clarendon Press, Oxford, 1940).

53. N. A. Swanson et al., Cryo-EM structure of the periplasmic tunnel of T7 DNA-ejectosome at 2.7 Å resolution. *Mol. Cell*, 10.1016/j.molcel.2021.06.001 (2021).

54. J. Kyte, R. F. Doolittle, A simple method for displaying the hydrophobic character of a protein. *J. Mol. Biol.* **157**, 105–132 (1982).
