
SCIENCE ADVANCES | RESEARCH ARTICLE

**VIROLOGY**

# Measles and Nipah virus assembly: Specific lipid binding drives matrix polymerization

Michael J. Norris¹, Monica L. Husby², William B. Kiosses¹, Jieyun Yin¹, Roopashi Saxena², Linda J. Rennick³, Anja Heiner⁴, Stephanie S. Harkins¹, Rudramani Pokhrel⁵, Sharon L. Schendel¹, Kathryn M. Hastie¹, Sara Landeras-Bueno¹†, Zhe Li Salie¹‡, Benhur Lee⁶, Prem P. Chapagain⁵,⁷, Andrea Maisner⁴, W. Paul Duprex³, Robert V. Stahelin²*, Erica Ollmann Saphire¹*

Measles virus, Nipah virus, and multiple other paramyxoviruses cause disease outbreaks in humans and animals worldwide. The paramyxovirus matrix (M) protein mediates virion assembly and budding from host cell membranes. M is thus a key target for antivirals, but few high-resolution structures of paramyxovirus M are available, and we lack the clear understanding of how viral M proteins interact with membrane lipids to mediate viral assembly and egress that is needed to guide antiviral design. Here, we reveal that M proteins associate with phosphatidylserine and phosphatidylinositol 4,5-bisphosphate [PI(4,5)P₂] at the plasma membrane. Using x-ray crystallography, electron microscopy, and molecular dynamics, we demonstrate that PI(4,5)P₂ binding induces conformational and electrostatic changes in the M protein surface that trigger membrane deformation, matrix layer polymerization, and virion assembly.

## INTRODUCTION

Paramyxoviruses are enveloped, negative-sense RNA viruses that are among the most infectious and pathogenic viruses known (1, 2). Most paramyxoviruses spread through respiratory droplets. Measles virus (MeV) still infects >7 million people and causes >100,000 deaths annually (3, 4). Multiple other paramyxoviruses cause devastating annual disease in humans, including Nipah virus (NiV) with up to 90% lethality (5), and parainfluenza virus III (PIV-III), a leading cause of childhood hospitalization (6). Avian and ruminant paramyxoviruses threaten global food supplies and cause substantial annual economic losses (7, 8). Novel paramyxoviruses, to which humans are immunologically naive, may yet emerge with substantial pandemic potential (9). Despite the number and breadth of these threats, no specific therapies are yet available against any paramyxovirus.

Paramyxoviruses bud from the plasma membranes (PMs) of infected cells. Viral matrix (M) proteins coordinate budding of new virions by marshaling other viral structural components, including surface glycoproteins and viral replication complexes at assembly sites along the host PM. Expression of M alone can drive budding of virus-like particles (VLPs) of most paramyxoviruses (10), while loss or mutation of M severely impairs viral replication (11, 12).

During infection, M proteins form a paracrystalline lattice at discrete assembly sites underlying the PM of infected cells (13, 14). These M lattices bridge viral glycoproteins and the internal ribonucleocapsid complex containing the RNA genome to form the virion particle (10, 13–15). However, the nature of M protein binding to membranes, why virus assembly happens largely at the PM, what triggers viral matrix polymerization, and whether interaction of M with lipid membranes alone is sufficient to form outward protrusions in the PM remain unclear.

Here, we show that phosphatidylserine (PS) and phosphatidylinositol 4,5-bisphosphate [PI(4,5)P₂] mediate MeV- and NiV-M protein interactions with host membranes. We describe the first high-resolution crystal structures for MeV- and NiV-M proteins, demonstrate that dimerization is critical for virion formation, and illuminate structural rearrangements in M that occur upon binding of PI(4,5)P₂ with a crystal structure of NiV-M protein bound to a soluble form of PI(4,5)P₂. We further show that these lipid-induced structural rearrangements alter the M dimer shape and electrostatics to promote M protein lattice polymerization and membrane curvature that together form the virion. The M-lipid complex structure also reveals new three-dimensional (3D) templates for design of agents to inhibit paramyxovirus assembly.

## RESULTS

### Electrostatic interactions govern NiV-M PM localization but not MeV-M

Paramyxovirus M proteins are known to localize at the PM inner leaflet during viral infection (16), but the mechanism by which this localization occurs remains poorly characterized. Electrostatic interactions between the positively charged surface of M proteins and the negatively charged PM inner leaflet are hypothesized to facilitate membrane localization (14, 17). However, earlier studies indicated that shielding electrostatic interactions with high salt does not inhibit membrane binding of some paramyxovirus M proteins, suggesting that membrane localization may instead involve interaction with specific lipids (18–20). To better understand the mechanism and specificity of M localization at the PM, we tested the impact of PM charge neutralization on M protein localization by treating cells with sphingosine, a positively charged lipid molecule

---

¹ Center for Infectious Disease and Vaccine Research, La Jolla Institute for Immunology, La Jolla, CA 92037, USA.  
² Department of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, IN 47907, USA.  
³ Center for Vaccine Research, University of Pittsburgh School of Medicine, Pittsburgh, PA 15261, USA.  
⁴ Institute of Virology, Philipps University Marburg, Marburg, Germany.  
⁵ Department of Physics, Florida International University, Miami, FL 33199, USA.  
⁶ Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA.  
⁷ Biomolecular Sciences Institute, Florida International University, Miami, FL 33199, USA.  

*Corresponding author. Email: erica@lji.org (E.O.S.); rstaheli@purdue.edu (R.V.S.)  

† These authors contributed equally to this work.  

‡ Present address: Loxo Oncology, Eli Lilly, 10290 Campus Point Dr, San Diego, CA 92121, USA.

that incorporates into the PM inner leaflet. The presence of sphingosine neutralizes the overall negative surface charge of the inner leaflet and, in turn, substantially reduces nonspecific electrostatic interactions at the PM but does not inhibit specific charge-based interactions (21). Here, we found that sphingosine does not affect PM localization of MeV-M (Fig. 1A). This result is similar to that for eVP40 (22), a control protein that interacts with specific lipid head groups (fig. S1A), suggesting that MeV-M PM localization involves interactions with specific lipids rather than nonspecific, negative charge-dependent membrane association. Meanwhile, in the presence of sphingosine, NiV-M PM localization is reduced (Fig. 1B). This reduction is similar to that seen for the control protein KRφ-RFP that localizes to the PM via nonspecific electrostatic interactions (fig. S1B) (23). Together, these results indicate that

![Figure](https://i.imgur.com/yourimageurl.png)

Fig. 1. MeV-M and NiV-M preferentially interact with PS and PI(4,5)P₂. (A and B) Left: COS-7 cells expressing enhanced green fluorescent protein (EGFP)-fused proteins and treated with sphingosine or ethanol vehicle. (A and B) Right: Quantifying PM localization as percent cells with EGFP localized at the PM (N≥ 45 cells per replicate). Liposome sedimentation assays and densitometric analysis of MeV- or NiV-M binding to large unilamellar vesicles (LUVs) containing (C) phospholipids or (D) phosphatidylinositols alone or (E) combined (fig. S1, C to E) (N≥ 5). (F and G) Left: CLSM images of COS-7 cells expressing EGFP-MeV-M (F) or EGFP-NiV-M (G) alone (left) or with MycVPtase-WT (middle) or MycVPtase-Δ1 (right). (H) VLP budding from cells in the presence or absence of MycVPtase-WT or MycVPtase-Δ1. (I and J) Left: ISA-2011B or dimethyl sulfoxide (DMSO) vehicle treatment of COS-7 cells expressing EGFP-MeV-M (I) or EGFP-NiV-M (J). (F, G, I, and J) Right: PM localization of MeV- and NiV-M [see (A) and (B)]. (K) NiV-M VLP budding from DMSO- or ISA-2011B–treated cells. Normalized budding indices for (H) and (K) were calculated from integrated immunoblot intensities (fig. S2, D and E, respectively). Scale bars, 10 µm. Values represent means ± SEM of three independent experiments. One-way analysis of variance (ANOVA) with Tukey’s post hoc test was used for group comparisons. Student’s t test compared treatment and vehicle group for each protein. ns, P > 0.05, **P ≤ 0.01, ***P ≤ 0.001, and ****P ≤ 0.0001. Abbreviations as per main text.

Norris et al., Sci. Adv. 8, eabn1440 (2022) 20 July 2022

MeV-M traffics to the PM largely through interactions with specific lipids, whereas electrostatic associations facilitate NiV-M localization to the PM.

### Measles and Nipah M proteins selectively bind PS and PI(4,5)P₂ in membranes

Although paramyxovirus M proteins intrinsically bind cellular membranes during assembly, it is unclear whether particular lipid head groups anchor M to the inner membrane leaflet (19, 20, 24, 25). To understand how MeV- and NiV-M anchor to membranes, we used liposome sedimentation assays to identify specific lipids critical for membrane binding. No significant binding was observed of MeV- or NiV-M to large unilamellar vesicles (LUVs) comprising phosphatidylcholine (PC) and phosphatidylethanolamine (PE), or negatively charged phosphatidic acid (Fig. 1C and fig. S1C). MeV- and NiV-M did, however, associate with LUVs carrying anionic PS (Fig. 1C and fig. S1C) and certain phosphoinositides (PIs).

PIs are a minor lipid species in cell membranes yet play vital roles in membrane trafficking and cell signaling. There are seven PI species present in eukaryotic membranes (26). The presence of any PI species in LUVs increased MeV- and NiV-M binding over PC:PE control LUVs (Fig. 1D and fig. S1D). However, MeV-M exhibited significant binding to LUVs containing the bisphosphates PI(3,5)P₂ (phosphatidylinositol 3,5-bisphosphate) or PI(4,5)P₂ and NiV-M interacted with LUVs containing PI(4,5)P₂, or the tri-PI phosphatidylinositol 3,4,5-trisphosphate [PI(3,4,5)P₃] (Fig. 1D and fig. S1D).

### M proteins cooperatively bind PI(4,5)P₂ and PS

Peripheral proteins can interact with multiple lipids in the PM to increase protein-membrane affinity (27). We find that inclusion of PS in LUVs containing PI(4,5)P₂ enhances MeV- and NiV-M binding by ~2.5- and 5-fold, respectively, compared to LUVs with PI(4,5)P₂ alone (Fig. 1E and fig. S1E). PS and PI(4,5)P₂ are the predominant anionic lipids in the PM (28), suggesting that synergistic binding with the viral matrix could facilitate PM anchoring and subsequent M-driven viral budding.

### PI(4,5)P₂ is essential for measles and Nipah M membrane interaction in live cells

We then investigated whether MeV- or NiV-M PI(4,5)P₂ association detected in liposome sedimentation assays was functionally significant in a cellular system in which PI(4,5)P₂ primarily localizes to the PM inner leaflet (29). We coexpressed enhanced green fluorescent protein (EGFP)-MeV-M or EGFP-NiV-M with a PI 5-phosphatase (MycVPTase) that cleaves the phosphate from the D5 position of PI(4,5)P₂ to reduce PI(4,5)P₂ levels in the PM and determined M protein localization. Inactive MycVPTase (MycVPTase-Δ1) and proteins that interact with PI(4,5)P₂ (PLCδ-PH and VP40) or PS (LactC2) were used as controls (fig. S2, A to C). We found that MycVPTase-dependent PI(4,5)P₂ depletion significantly reduces both EGFP-MeV-M and EGFP-NiV-M PM localization (Fig. 1, F and G). Furthermore, VLP production of NiV-M was also substantially reduced in the presence of the wild-type (WT) MycVPTase (Fig. 1H and fig. S2D). Similarly, the PI-5-kinase-α (PIP5ka) inhibitor ISA-2011B, which inhibits production of PI(4,5)P₂ from its PI(4)P precursor, also reduced EGFP-MeV-M and EGFP-NiV-M PM localization as well as NiV-M VLP production (Fig. 1, I to K and fig. S2E).

Like PI(4,5)P₂, PI(3,4,5)P₃, which is uniquely bound by NiV-M, is also found in the PM inner leaflet (29). The phosphatidylinositol 3-kinase inhibitor wortmannin reduces PI(3,4,5)P₃ levels in the PM (30). Our results suggest that NiV-M PM localization is independent of PI(3,4,5)P₃ as evidenced by the lack of effect by wortmannin (fig. S2, F to H).

Conversely, PI(3,5)P₂, preferred by MeV-M, is primarily found in multivesicular bodies (MVBs) (29). The phosphatidylinositol 3-phosphate 5-kinase inhibitor apilimod reduces cellular levels of PI(3,5)P₂ (31). Here, apilimod-dependent reduction of PI(3,5)P₂ in intracellular membranes decreased EGFP-MeV-M PM localization, which highlights a potential role for PI(3,5)P₂ in MeV-M trafficking to the PM (fig. S2, I to L). Together, these data suggest that PI(4,5)P₂ is the primary driver of membrane interaction for both MeV- and NiV-M. In addition, PI(3,5)P₂ may enhance interaction of MeV-M with MVBs.

### PI(4,5)P₂ drives conformational changes in paramyxovirus M dimers

Because PI(4,5)P₂ was important for PM localization and anchoring of both MeV- and NiV-M, we next explored the nature of this lipid interaction at a molecular level. We determined the crystal structures of both M proteins alone, as well as the crystal structure of NiV-M in complex with the short-chain, water-soluble form of PI(4,5)P₂ [C8-PI(4,5)P₂]. Consistent with other paramyxovirus M proteins (14, 17), both Apo forms of MeV- and NiV-M form head-to-tail dimers, stabilized by hydrophobic and electrostatic interactions, with a buried surface of ~2300 Å² (Fig. 2, A and B, and table S1). The protomers have two structurally analogous domains joined by a flexible linker region. A twisted β sandwich surrounded by several short α helices forms the core of each domain. Both MeV- and NiV-M crystal structures exhibit a predominant positive surface charge and a large, uniform surface-exposed basic patch in each C-terminal domain (CTD) (Fig. 2, A and B, bottom). Size exclusion chromatography (SEC) coupled to multiangle light scattering analysis and single-particle negative-stain electron microscopy (nsEM) further confirm the dimeric nature of MeV- and NiV-M in solution (fig. S3, A to F).

The M-lipid crystal structure (Fig. 2C and table S1) reveals that C8-PI(4,5)P₂ binding induces extensive conformational rearrangements (fig. S4A) that expand the dimer interface, flatten the membrane-interacting surface from the concave shape of the Apo structure, and alter the surface electrostatics. Specifically, lipid binding expands the dimer interface by 33 residues compared to Apo-NiV-M, likely increasing matrix lattice stability. Expansion of the interprotomer interaction in the lipid-bound dimer occurs as α helix 1 in the Apo-NiV-M dimer interface unwinds. This unwinding increases mobility of the C-terminal downstream 20 residues, which rearrange from their initial position behind their own protomer to thread forward and reach across to a hydrophobic pocket in the N-terminal domain (NTD) of the other protomer (Fig. 2D). Although the N-terminal 30 residues are disordered in the Apo-NiV-M structure, they can be modeled in the lipid-bound NiV-M structure and occupy the region behind the protomer that was previously occupied by the C terminus in the Apo-NiV-M structure (movie S1). The conformational change also opens one basic patch and creates a second basic patch compared to the Apo form (compare Fig. 2, B and C, bottom). The first patch opens upon displacement of α helix 2 from CTD β sheet 1 to accommodate the negatively charged C8-PI(4,5)P₂ molecule (Fig. 2E). In addition, the C8-PI(4,5)P₂ acyl chains both pack against hydrophobic side chains in the CTD basic patch

SCIENCE ADVANCES | RESEARCH ARTICLE

**A** Apo-measles-M  
2x  
Strep 1  
EK  
NTD Linker CTD  
163 176 335  

**B** Apo-Nipah-M  
2x  
Strep 1  
EK  
NTD Linker CTD  
180 191 352  

**C** PI(4,5)P₂-bound  
Nipah-M  

NTD CTD  
CTD NTD  

NTD CTD  
CTD NTD  

Basic  
patch 2  

Basic  
patch  

Basic  
patch 1  

-5.00 5.00  
-5.00 5.00  
-5.00 5.00  

**D**  
α Helix 1  

α Helix 1  

Apo-NiV-M  
PI(4,5)P₂-bound NiV-M  

**E**  
Beta 1  
~6 Å  
α Helix 2  

α Helix 2  
~12 Å  

Apo-NiV-M  
PI(4,5)P₂-bound NiV-M  

Fig. 2. Binding of PI(4,5)P₂ drives conformational changes in NiV M protein. (A and B) Top: Schematic of MeV- and NiV-M expression constructs. Middle: Crystal structure of M protein dimers. Bottom: MeV- and NiV-M electrostatic surface potential on a space-filling model with positively and negatively charged regions shown in blue and red, respectively. The CTD basic patch is outlined. Electrostatic potential maps were generated using PDB2PQR and APBS software (81, 82). EK, enterokinase. (C) Top: Crystal structure of NiV-M dimer in complex with C8-PI(4,5)P₂. NiV-M protomers are colored purple and gray, and C8-PI(4,5)P₂ molecules are colored cyan, orange, and red (for carbon, phosphorus, and oxygen atoms, respectively). Bottom: Electrostatic surface potential of C8-PI(4,5)P₂-bound NiV-M. The CTD basic patch and newly formed basic patch in the N-terminal domain (NTD) are outlined. (D) Apo-NiV-M (left) and C8-PI(4,5)P₂-bound NiV-M (right) showing the conformational change in the C-terminal 20 residues (red). (E) Close-up of PI(4,5)P₂ binding pocket in Apo-NiV-M (left) and C8-PI(4,5)P₂-bound NiV-M (right). The separation between α helix 2 and β sheet 1 increases by 6 Å in the conformational change required to accommodate lipid binding.

(fig. S4B). The second basic patch forms as the basic C terminus drapes across the neighboring protomer NTD (compare Fig. 2, B and C, bottom). This second basic patch increases available surface area for interactions with negatively charged lipid head groups and can generate binding sites for membrane lipids like PS.

Structural rearrangements induced by C8-PI(4,5)P₂ binding also increase the angle between adjacent protomers (Fig. 3A). This transition from the bowl-shaped Apo to the flatter, plate-shaped surface of the bound conformation changes the geometry of the basic patches (Fig. 3B). In this new conformation, the basic patch extends to the side of the molecule such that the membrane must bend upward to maintain contact between the negatively charged lipid head groups and the basic patch (Fig. 3C). The conformational change also alters the M protein surface electrostatics, revealing negatively charged patches in the center of the donut-shaped membrane binding surface (compare Fig. 2, B and C, bottom). We hypothesize that the geometry of the basic patches together with electrostatic repulsion generated from the negatively charged patches imposes negative local curvature, similar to that of a convex object, that facilitates initial membrane deformation at budding sites (Fig. 3C).

---

**Budding requires M protein dimerization**

Previous results with related paramyxovirus M proteins suggested that dimerization is critical for virion formation (14, 17). To determine the role of M dimerization in MeV and NiV assembly, we introduced destabilizing mutations at the dimer interface of MeV- and NiV-M (Fig. 4, A and B). We confirmed that the selected mutations disrupt the dimer interface using a bimolecular fluorescence complementation (BiFC) assay (32). Overall expression levels of each mutant were similar to WT M (fig. S5, A and B). However, both confocal laser scanning microscopy (CLSM) (fig. S5, A and B) and flow cytometry (Fig. 4C and fig. S5, C and D) revealed that mutant MeV- and NiV-M BiFC signals are diminished. SEC on representative MeV- and NiV-M mutants confirmed that dimer formation is abolished relative to WT and suggests that any signal detected in BiFC analysis of these mutants is likely due to aggregation (fig. S5, E and F). To examine the role of M dimerization, we quantified the VLP production from mutant and WT M proteins. WT MeV- and NiV-M were detected both in total cell lysates and culture medium of mammalian cells, confirming that MeV- and NiV-M expression alone can drive VLP formation (fig. S5, G and H). Meanwhile, VLP

of VLP production in mammalian cells transfected with WT and mutant NiV-M, no NiV-M Arg mutant promoted VLP production, suggesting that each Arg in the newly formed PI(4,5)P₂ pocket is important for NiV-M–mediated virus assembly and budding (Fig. 5B and fig. S6B). However, to delineate whether the observed reduction in budding is due to disruption of PI(4,5)P₂ interaction, or if adding a negative charge to the basic patch more broadly disrupts budding, we made additional “control” mutations to nearby residues in the basic patch that are not predicted to be important for PI(4,5)P₂ coordination and tested them in budding assays (Fig. 5A, left inset, green residues). Two of the additional mutants, N199E (adjacent to R198) and V243E (near R245), completely abrogated VLP formation and a third additional mutant P332E (adjacent to R333) had a 70% reduction in VLP formation (Fig. 5B and fig. S6B). However, a fourth mutation, Q291E (near R287), showed a ~60% increase in budding relative to WT levels (Fig. 5B and fig. S6B). This result indicates that M-driven assembly and budding is dependent not only on the coordination of PI(4,5)P₂ but also on the maintenance of the overall positive charge on the outer periphery of the basic patch.

We further investigated whether the selected charge-switching Arg-to-Glu mutations (R198E, R287E, and R333E) in the PI(4,5)P₂ binding pocket affects lipid binding as predicted. First, we measured the ability of EGFP-NiV-M bearing R198E, R287E, or R333E mutations to localize with the PM. As expected, we found that all three mutant proteins had significantly reduced PM localization relative to WT (Fig. 5C and fig. S6C). Second, we expressed and purified a representative mutant (R287E) and performed in vitro liposome binding assays. The R287E mutant exhibited a 30% increase in binding to PC:PE liposomes relative to WT (Fig. 5D and figs. S1D and S6D). However, there was no significant difference in sedimentation of NiV-M R287E for PC:PE liposomes with and without PI(4,5)P₂ (Fig. 5D and fig. S6D), suggesting that the R287E mutant lost the ability to bind to PI(4,5)P₂. Together, these results indicate that positively charged residues present in the CTD basic patch are required for binding of PI(4,5)P₂ and subsequent M-driven assembly and budding.


### PI(4,5)P₂-induced conformational changes in M are required for budding

As the M-lipid crystal structure also revealed an extensive conformational change in which the C-terminal 20 residues rearrange to bind a hydrophobic pocket in the NTD of the adjacent protomer, we sought to understand whether this change is required for M assembly and budding. To disrupt hydrogen bonding to the C-terminal main chain during conformational rearrangement, we mutated Y64, R191, D304, and the highly conserved G95 (Fig. 5A, right inset, yellow residues). In the Apo-NiV-M structure, these residues are solvent-exposed and not readily involved in protein folding or M dimer formation. These mutants lost all budding activity (Fig. 5E and fig. S6E), suggesting that interactions made by the NiV-M C-terminal 20 residues following conformational rearrangement upon PI(4,5)P₂ binding are indeed a prerequisite for assembly and budding. Additional control mutations to nearby residues in the hydrophobic NTD pocket that do not participate in binding to the C-terminal main chain (Fig. 5A, right inset, green residues) had no significant impact on budding, further indicating that the PI(4,5)P₂-induced conformational change is required for budding (Fig. 5E and fig. S6E). In a similar panel of mutants predicted to disrupt MeV-M conformational changes, VLP formation decreased by up to 60% (fig. S7, A to C).


### Binding and coordination of PI(4,5)P₂ is required for PM localization and M budding

The negatively charged C8-PI(4,5)P₂ molecule is coordinated by R198, R287, and R333 of NiV-M (Fig. 5A, left inset, yellow residues). R333 is part of α helix 1 in Apo-NiV-M but is displaced by ~7.6 Å in the bound form to coordinate C8-PI(4,5)P₂. To determine the role of PI(4,5)P₂ binding in M budding, we next made charge-switching Arg-to-Glu mutations (R198E, R287E, and R333E) in the newly identified CTD basic pocket that receives PI(4,5)P₂, and also at R245, which is disordered in the structure but could participate in PI(4,5)P₂ coordination as suggested by all-atom molecular dynamics (MD) simulations with C8-PI(4,5)P₂ (fig. S6A). In Western blot analysis

SCIENCE ADVANCES | RESEARCH ARTICLE

A Measles-M B Nipah-M C BiFC flow cytometry

Measles M Nipah M

D VLP budding

Measles M Nipah M

Fig. 4. Mutations to residues involved in the dimer interface of M proteins inhibit VLP production. (A and B) Surface representation of dimeric MeV-M (A) and NiV-M (B) showing the dimer opened to display the dimeric interface with residues targeted for mutagenesis highlighted in magenta. (C) Flow cytometry of bimolecular fluorescence complementation (BiFC) fluorescence of WT or mutant MeV-M (left) or NiV-M (right). Mean fluorescence intensity for mutants is normalized to that of WT ($N \geq 10,000$ cells per experiment). Data are from three independent experiments. (D) Comparison of VLP budding between WT and mutants targeting the dimeric interface of MeV-M (left) or NiV-M (right). Normalized budding index was calculated from immunoblot integrated intensities (fig. S5, G and H). Values represent means $\pm$ SEM of three independent experiments. $*P \leq 0.05$, $**P \leq 0.01$, $***P \leq 0.001$, and $****P \leq 0.0001$ by one-way ANOVA.

PI(4,5)P₂-bound Nipah-M

A

B VLP budding C PM localization D Liposome sedimentation E VLP budding

Fig. 5. Inhibition of VLP production by mutation of lipid binding pocket residues and residues that drive conformational change of M proteins. (A) Crystal structure of NiV-M-C8-PI(4,5)P₂. Left inset: NiV-M CTD PI(4,5)P₂ binding site. Right inset: Conformational change in C-terminal residues (purple) extending across the hydrophobic pocket in the NTD of the adjacent protomer (gray). In both insets, interacting and control residues are colored yellow and green, respectively. (B) Effect of NiV-M PI(4,5)P₂ binding pocket mutations on VLP budding. Normalized budding index was calculated from integrated immunoblot intensities (fig. S6B). (C) Quantifying PM localization in COS-7 cells expressing the indicated EGFP-fused NiV-M protein for 24 hours. The percentage PM localization is the ratio of EGFP fluorescence intensity at the PM to that of the whole cell (fig. S6C). (D) Liposome sedimentation assays and Western blotting of NiV-M R287E binding to PC:PE LUVs with or without PI(4,5)P₂. (E) VLP budding of WT NiV-M and NiV-M having mutated C terminus-coordinating residues. Normalized budding index was calculated from integrated immunoblot intensities (fig. S6E). For all VLP budding, values represent means $\pm$ SEM of three independent experiments. ns, $P > 0.05$, $*P \leq 0.05$, $**P \leq 0.01$, and $****P \leq 0.0001$ by one-way ANOVA. Abbreviations as per main text.

whereas control mutants did not affect budding. This result suggests that similar molecular changes occur in MeV-M upon lipid binding.

**Modeling the PM association of NiV-M and MeV-M**

To simulate membrane association between Apo- (Fig. 2B) and lipid-bound (Fig. 2C) conformations of NiV-M, we performed all-atom MD simulations using realistic membrane bilayers containing 1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine (POPC), 1-palmitoyl-2-oleoyl-sn-glycero-3-phospho-L-serine (POPS), 1-palmitoyl-2-oleoyl-sn-glycero-3-phosphoethanolamine (POPE), palmitoylsphingomyelin, cholesterol, and PI(4,5)P₂ at relevant ratios to model the PM in terms of charge, lipid packing, and hydrophobic core structure. Before the simulation, and after minimization and equilibration, both protein conformations were ~10 Å below the PM lower leaflet and only weak interactions between proteins and membranes occurred (0 μs; Fig. 6, A and B). For the Apo-NiV-M conformation, one M protomer interacted with the membrane for ~0.040 μs before the second protomer began interacting. By ~0.20 μs, many basic CTD residues in Apo-NiV-M interacted with anionic lipid head groups, and, by the end of the 0.5-μs simulation, 35 lipids interacted with the dimer (Fig. 6A and movie S2). In contrast, the PI(4,5)P₂-bound NiV-M conformation interacted rapidly with membrane, and, by ~0.06 μs, most basic CTD residues in both protomers and several residues in the C terminus that drape across the adjacent protomer interacted with the membrane. By the end of the 1-μs simulation, 43 lipids interacted with the NiV-M dimer in the PI(4,5) P₂-bound conformation (Fig. 6B and movie S2).

The PI(4,5)P₂-bound NiV-M had nearly twofold more hydrogen bonds with lipid head groups than Apo-NiV-M during the MD simulation (Fig. 6C). The rapidity of membrane interaction and increased number of membrane interactions further support that the conformational change in NiV-M observed in the PI(4,5)P₂-bound NiV-M crystal structure (Fig. 2C) favors membrane association and protein stabilization at the PM lower leaflet. In the Apo-NiV-M conformation, hydrogen bonding with the PM involves almost exclusively CTD Arg residues (Fig. 6D), whereas hydrogen bonding of the lipid-bound conformation also involves CTD Arg residues, including R198, R245, R287, and R333 (Fig. 5A), as well as additional interactions with R244, K286, and R261. Extensive hydrogen bonding between membrane head groups and both the C-terminal R348 that drapes across the NTD and the NTD R57 present in the second basic patch was also observed (Fig. 6D).

We next calculated the number of lipid-protein contacts as a function of time (Fig. 6, E and F). Almost all lipid types bind to M proteins in both the Apo and bound conformations, but PS forms more contacts to PI(4,5)P₂-bound NiV-M than the Apo conformation (Fig. 6, E and F). Furthermore, PI(4,5)P₂ (colored red in Fig. 6, A and B) makes substantially more interactions with basic residues than do other lipids. These results agree well with our experimental observations that PI(4,5)P₂ and PS are important for PM localization and interaction (Fig. 1). Using a homology model of bound MeV-M generated from bound-NiV-M coordinates to perform the same MD simulations, lipid-bound MeV-M also forms significantly more hydrogen bonds with the PM and substantially more contacts with PI(4,5)P₂ and POPS than the Apo-MeV-M conformation (fig. S8 and movie S3).

**Matrix proteins induce membrane deformation and curvature**

As nascent virions form, the PM lipid bilayer curves outward (33), but whether the matrix alone achieves this membrane curvature or viral surface glycoproteins or cellular proteins are also involved is unclear (10, 16). To examine this membrane curvature, we incubated MeV- or NiV-M with giant unilamellar vesicles (GUVs) lacking or containing PI(4,5)P₂ and analyzed structural alterations in the lipid bilayer by CLSM. Both MeV- and NiV-M produced significant membrane deformation characterized by filamentous, spherical, and flat membrane protrusions in GUVs containing PI(4,5)P₂ (Fig. 7, A and B). As the M proteins are outside the GUVs, all protrusions pointed inward to mimic negative curvature observed during virus budding. GUVs containing PS, but only trace amounts of PI(4,5)P₂, remain spherical in the presence of MeV- or NiV-M, except at the highest M protein concentration tested (Fig. 7, A and B). Thus, at low protein densities, interactions between M proteins and PS alone do not appear to induce membrane deformation, whereas the conformational changes that we observed upon interaction with PI(4,5)P₂ (Fig. 2) can induce membrane curvature. We also examined whether disrupting coordination of PI(4,5)P₂ affects membrane deformation of GUVs by incubating purified NiV-M R287E with GUVs containing 10% PI(4,5)P₂. Even at 10 μM (double the highest concentration tested for WT), GUVs remained spherical in the presence of NiV-M R287E (Fig. 7C). These results suggest that interaction with PI(4,5) P₂ is necessary for membrane deformation.

**Interaction with PI(4,5)P₂ induces matrix protein lattice assembly**

We next sought to determine whether interaction with PS and PI(4,5)P₂ would induce M protein lattice polymerization. nsEM of vesicles composed of POPC, POPS, and PI(4,5)P₂ incubated with MeV- or NiV-M illustrated formation of long, ordered filaments (Fig. 8, A and B). These filaments formed only in the presence of PI(4,5)P₂, suggesting that higher-order oligomeric assembly requires this lipid (fig. S9, A to D). Filaments comprised a helical lattice of M protein dimers with ~12 to 14 M dimers per turn and a ~27° helical twist for MeV-M (Fig. 8A), and 14 to 16 dimers per helical turn and a ~7.7° twist for NiV-M (Fig. 8B). Projecting 2D class averages onto 3D cylinders and docking x-ray structures into the 3D volumes showed that MeV- and NiV-M polymerization involves interactions between adjacent dimers and those in rows immediately above and below (Fig. 8, C and D). The spacing and arrangement of M dimer subunits in these filaments are consistent with M protein assemblies in tomographic reconstructions of the M layers of authentic paramyxoviruses (fig. S9E) (13, 14).

We next used scanning electron microscopy to delineate whether M-driven filament formation is relevant in cells and indicative of viral budding. The cell surfaces of human embryonic kidney (HEK) 293 and COS-7 cells transiently expressing MeV- or NiV-M also exhibit abundant spherical and filamentous protrusions that are not present on the surface of mock-transfected cells (Fig. 8, E to G).

**Pharmacological reduction of PI(4,5)P₂ inhibits measles and Nipah virus infection**

The PIP5kα inhibitor ISA-2011B reduced MeV- and NiV-M localization at the PM (Fig. 1, I and J) as well as NiV-M VLP production (Fig. 1K). To address whether inhibition of PI(4,5)P₂ production (Fig. 9A) also affects MeV and NiV in the context of authentic virus, we conducted ISA-2011B inhibitor studies in live MeV and NiV infections. ISA-2011B dose-dependently reduces replication of recombinant MeV modified to express EGFP [rMVKS-EGFP(3)] relative to control-treated cells, with little effect on cell viability

SCIENCE ADVANCES | RESEARCH ARTICLE

Apo NiV-M

PI(4,5)P₂-bound NiV-M

Fig. 6. MD simulations of NiV-M interaction with the PM. Initial and final snapshots of NiV-M association with the PM in the (A) Apo and (B) PI(4,5)P₂-bound conformation observed in the simulations. Left and right panels display representative initial and final configurations, respectively. Lipids interacting with protein are highlighted as follows: [PI(4,5)P₂, red; PS, green; cholesterol, POPC, POPE, and palmitoylsphingomyelin, gray]. (C) Time evolution of hydrogen bond formation between membrane lipids and Apo- (light purple) or PI(4,5)P₂-bound (dark purple) NiV-M. (D) Surface model of Apo-NiV-M (left) and PI(4,5)P₂-bound NiV-M (right). Residue colors correspond to the number of hydrogen bonds formed with the membrane within the last 200 ns of the simulation. (E and F) Time evolution of hydrogen bonds between various lipid types and the Apo-NiV-M (E) or PI(4,5)P₂-bound conformation (F).

(Fig. 9, B and C). In NiV-infected Vero76 cells, ISA-2011B significantly reduced NiV plaque size and, by extension, cytopathic effect (Fig. 9D), suggesting that inhibitor-mediated reductions in PI(4,5) P₂ levels are a promising strategy to reduce paramyxoviral spread.

DISCUSSION

M proteins play a fundamental role in paramyxovirus assembly and release by binding cellular membranes, self-assembling, and organizing viral components at budding sites (10). Here, we identify a critical lipid head group bound by M proteins during virus assembly, solve structures of M proteins alone and in complex with this lipid, and demonstrate marked conformational changes upon lipid binding that lead to membrane interactions, viral assembly, curvature, and budding. The assembly identified resembles that observed in authentic virions. We further demonstrate using mutagenesis that dimerization, lipid coordination, and conformational rearrangement of the C terminus are all essential for viral budding.

SCIENCE ADVANCES | RESEARCH ARTICLE

**A** DOPC:POPE:DOPS:TF-TMR PI(4,5)P₂  
(50:10:30:0.2)

| 0% PI(4,5)P₂ | 5% PI(4,5)P₂ | 10% PI(4,5)P₂ |
|--------------|--------------|--------------|
| No protein   |              |              |
| MeV-M        |              |              |
| 2.5 μM       |              |              |
| MeV-M        |              |              |
| 5 μM         |              |              |

**B** DOPC:POPE:DOPS:TF-TMR PI(4,5)P₂  
(50:10:30:0.2)

| 0% PI(4,5)P₂ | 5% PI(4,5)P₂ | 10% PI(4,5)P₂ |
|--------------|--------------|--------------|
| No protein   |              |              |
| NiV-M        |              |              |
| 2.5 μM       |              |              |
| NiV-M        |              |              |
| 5 μM         |              |              |

**C** DOPC:POPE:DOPS:TF-TMR PI(4,5)P₂  
(50:10:30:0.2)

| 10% PI(4,5)P₂ |
|---------------|
| NiV-M         |
| R287E         |
| 10 μM         |

---

Fig. 7. Matrix proteins induce membrane deformation in the presence of PI(4,5)P₂. (A and B) Representative CLSM images of GUVs comprising DOPC (dioleoylphosphatidylcholine), POPE, and DOPS and 0 (left), 5 (middle), or 10 (right) mole percent (mol %) PI(4,5)P₂ with TopFluor (TF)–tetramethylrhodamine (TMR) PI(4,5)P₂ (0.2 mol %; red fluorescence) after incubation with no protein (top), 2.5 μM (middle), or 5 μM (bottom) MeV-M (A) or NiV-M (B). No membrane deformation occurred without MeV-M or NiV-M. Filamentous (yellow arrow), spherical (white arrow), or flat (green arrow) protrusions into the GUVs interior appeared after adding MeV-M or NiV-M. (C) Representative CLSM image of GUVs comprising DOPC, POPE, DOPS, PI(4,5)P₂, and TopFluor TMR PI(4,5)P₂ at the indicated mol % after incubation with 10 μM NiV-M R287E. No membrane deformation could be detected for this mutant. Scale bars, 10 μm.

Together, these data provide the framework for a model of viral budding (Fig. 10) in which monomeric M initially assembles into a dimeric conformation (Fig. 10, step 1) present in both MeV- and NiV-M Apo crystal structures (Fig. 2, A and B). The large, conserved, hydrophobic dimeric interface of the Apo form gives rise to a doughnut-shaped molecule with a positively charged concave surface. Mutations in the dimer interface impede dimer formation and M-driven budding (Fig. 4), supporting a role for M dimers as the basic unit for paramyxovirus matrix assembly (14, 17, 34).

Next, M proteins localize and assemble at the PM (Fig. 10, step 2). Nonspecific electrostatic interactions between positively charged residues in the CTD basic patch of paramyxovirus and closely related pneumovirus M proteins were thought to drive interactions with negatively charged membranes (14, 17, 35, 36). Like other paramyxovirus M proteins, we show that the CTD surface of both MeV- and NiV-M has a highly basic patch of residues (Fig. 2, A and B, bottom). Our assays with sphingosine indicated that MeV-M localization appears to be independent of the PM inner leaflet negative charge (Fig. 1A), whereas NiV-M may rely on nonspecific electrostatic interactions to reach the PM (Fig. 1B). Liposome sedimentation assays indicated that MeV-M interacts with PI(3,5)P₂ (Fig. 1D and fig. S1D), which is more abundant in intracellular vesicles within the endocytic trafficking pathway than the PM (37). Thus, MeV-M could be trafficked to the PM via association with PI(3,5)P₂ on intracellular vesicles. The potential involvement of PI(3,5)P₂-mediated trafficking is supported by inhibition of MeV-M PM localization by apilimod, which reduces PI(3,5)P₂ levels (fig. S2I). In this mechanism, MeV-M already localized to the PM before treatment would be unaffected by apilimod, but no new MeV-M could reach the PM.

Upon arriving at the PM, M proteins must bind and anchor to the inner leaflet (Fig. 10, step 3). Although paramyxovirus M proteins have intrinsic membrane affinity (13, 14, 20, 38, 39), the detailed mechanism of M membrane interactions is unclear. Our data indicate that both MeV- and NiV-M associate with membranes containing PS (Fig. 1C and fig. S1C), which was previously shown to be important for PM anchoring of other viral M proteins (40–42). We also show that, similar to filovirus and retrovirus M proteins (VP40 and gag, respectively) (22, 43, 44), MeV- and NiV-M proteins specifically associate with membranes containing PI(4,5)P₂ in vitro and in cells (Fig. 1, D and F to K). The presence of both PS and PI(4,5)P₂ induces synergistic binding by MeV- and NiV-M (Fig. 1E), indicating that binding to one membrane component increases the affinity of M for the other and for the membrane in general.

Our crystal structure of NiV-M bound to C8-PI(4,5)P₂ demonstrates sweeping conformational rearrangements in the M dimer (Figs. 2, C to E, and 3, A to C), including expansion of the dimer interface and a transition from a concave to a flat membrane-binding surface. This transition is accompanied by opening of a pocket in the CTD basic patch that receives C8-PI(4,5)P₂ (Fig. 5A). Mutating residues that coordinate interactions with the PI(4,5)P₂ head group

SCIENCE ADVANCES | RESEARCH ARTICLE

**A** MeV-M + POPC:POPS:PI(4,5)P₂ (65:30:5)
96,000× 2D class average

**B** NiV-M + POPC:POPS:PI(4,5)P₂ (65:30:5)
96,000× 2D class average

**C**
Projection of lattice

**D**
Projection of lattice

**E**

HEK293

Mock

MeV-M

NiV-M

**F**
Mock NiV-M MeV-M

**G**
COS-7 cells

HEK293 cells

Fig. 8. Matrix proteins self-assemble in the presence of PI(4,5)P₂ and form spherical and filamentous extensions from cells. (A and B) Left: Representative electron micrographs (×96,000 magnification) of negatively stained liposomes comprising POPC:POPS:PI(4,5)P₂ (65:30:5 mol %, respectively) with 10 μM MeV-M (A) or NiV-M (B). Scale bars, 50 nm. Right: Representative 2D class averages of helical filaments showing twist and diameter of the helical assembly. (C and D) Isosurface representation of 2D class averages in (A) and (B) projected onto a 3D cylinder and refined without helical parameters. Inset: Close-up of the volume (gray transparent surface) shows packing of fitted lipid-bound homology model of MeV-M or crystal structure of NiV-M bound to PI(4,5)P₂. (E and F) Scanning electron microscopy of COS-7 and HEK293 cells transfected with MeV- or NiV-M. (E) Representative micrographs of HEK293 cells expressing the indicated protein (or mock transfected) showing spherical particles budding from the cell surface. (F) Representative micrographs of COS-7 cells and HEK293 cells expressing the indicated protein (or mock transfected) showing filamentous particles budding from the cell surface. (G) Quantification of filaments per square micrometer protruding from COS-7 (top) or HEK293 (bottom) cells for mock, MeV-, or NiV-M expressing cells ($N \geq 5$ cells, $n \geq 14$ regions 1 μm by 1 μm). ****$P \leq 0.0001$.

abrogates VLP budding, supporting a function of PI(4,5)P₂ as a direct membrane anchor and the finding that membrane association of paramyxovirus M proteins is not dependent entirely on electrostatics (Fig. 5B). Other paramyxovirus M proteins also encode positively charged residues in homologous sites; mutations to these residues in canine distemper virus, a close relative of MeV, inhibit M-induced VLP formation (45), suggesting that this site is structurally similar across paramyxovirus M proteins. Similarly, the importance of the C terminus in paramyxovirus M-driven budding has also been recently highlighted for PIV (46). In addition to coordination of the PI(4,5)P₂ head group, the C8-PI(4,5)P₂ 1′ and 2′ acyl chains are both buried within a hydrophobic pocket of NiV-M. Membrane anchoring of proteins via acyl chain binding is seen for other peripheral membrane proteins (47, 48), including the viral matrix domain of HIV-1 gag (49). However, truncated acyl chains have also been observed to nonspecifically bind to hydrophobic pockets on soluble proteins (50). Further work is needed to clarify the degree to which acyl chains contribute to NiV-M membrane anchoring.

In addition to providing a membrane anchor, we show that PI(4,5)P₂ binding is an allosteric trigger to form a new basic patch in the NTD. This newly formed basic patch could act as a secondary positively charged binding site for other PM lipids like PS, which could synergistically anchor the M protein to the inner leaflet. Consistent with this model, our MD simulations with bilayers

SCIENCE ADVANCES | RESEARCH ARTICLE

**Fig. 9. Pharmacological reduction of PI(4,5)P₂ inhibits live measles and NiV infection in cell culture.** (A) Phosphatidylinositol-4-phosphate 5-kinase type 1 alpha (PIP5K1A) catalyzes formation of PI(4,5)P₂ from PI4P and can be inhibited by ISA-2011B. (B) Left: Representative images of Vero-hCD150 cells 42 hours after infection with rMV<sup>K</sup>EGFP(3) [multiplicity of infection (MOI), 0.1] and indicated ISA-2011B concentration. Right: Percent rMV<sup>K</sup>EGFP(3) infection in ISA-2011B–treated cells normalized to DMSO-treated cells (EGFP signal as a surrogate for infection) (*n* = 2 independent experiments performed in triplicate). (C) Vero-hCD150 cells with indicated ISA-2011B concentration for 48 hours. The *y* axis denotes cell death (normalized to DMSO-treated cells) 48 hours after treatment. Values represent means ± SEM (*n* = 3 independent experiments performed in duplicate). (D) Left: Representative images of fixed and Giemsa-stained Vero76 cells 44 hours after infection with NiV (MOI, 0.001) and treated with ISA-2011B as indicated. Right: Percentage cells exhibiting cytopathic effect after ISA-2011B treatment normalized to DMSO-treated cells. ns, *P* > 0.05, \**P* ≤ 0.05, **\**P* ≤ 0.01 and ****\**P* ≤ 0.0001 by one-way ANOVA.

mimicking the PM (Fig. 6) show that PI(4,5)P₂-induced conformational changes increase the surface area and the number of residues interacting with the PM. The PI(4,5)P₂-bound conformation makes more hydrogen bonds with the PM and has more contacts with additional lipids, such as PS, compared to the Apo conformation (Fig. 6). Several of these additional contacts involve residues in the newly formed NTD basic patch, which is consistent with our finding that cooperative binding to PI(4,5)P₂ and PS enhances M membrane interaction (Fig. 1E). The conformational change in the lipid-bound form could both further stabilize M proteins on the membrane and provide an allosteric mechanism to ensure PM localization, as the secondary binding site is revealed only after binding to PI(4,5)P₂, which is primarily located in the lower leaflet of the PM (51).

Formation of new virions from the PM of infected cells requires outward curvature of the inner PM leaflet (Fig. 10, step 4). We show here that MeV- and NiV-M alone deform lipid membranes to generate spherical or filamentous protrusions with negative curvature and that PI(4,5)P₂ augments this deformation (Fig. 7). Intrinsic membrane deformation activity of HIV-1 (44, 52) and Ebola virus (22, 53) M proteins involves PI(4,5)P₂, as do cellular proteins like inverse Bin-Amphiphysin-Rvs (I-BAR) domain proteins, which drive formation of cellular filopodia via a PI(4,5)P₂-dependent mechanism (54). Our structure of NiV-M bound to C8-PI(4,5)P₂ shows that,

upon PI(4,5)P₂ binding, NiV-M adopts an almost flat membrane-interacting surface with a radius of curvature similar to the I-BAR protein, Pinkbar (planar intestinal- and kidney-specific BAR domain protein) (Fig. 3A). Pinkbar proteins have a nearly flat membrane-binding surface and promote formation of planar membrane sheets (55). Similar to I-BAR proteins, paramyxovirus M proteins could drive spontaneous local curvature of the membrane upon interaction with the inner leaflet. The conformational change in M alters the surface electrostatics and the geometry of the basic patches. We hypothesize that, with the basic patches primarily on the side of the protein, the membrane must bend upward to maintain contact with the positively charged lipid head groups. Together with the electrostatic repulsion generated from the negatively charged patches revealed in the center of the donut-shaped membrane binding surface, the protein drives outward local curvature (Fig. 10, step 4). A similar mechanism of local membrane curvature has been proposed for membrane interactions of the homopentameric B subunit of bacterial Shiga toxin (56).

Next, M proteins must self-assemble to form the matrix lattice (Fig. 10, step 5). In nsEM images, we observe helical assembly of M dimers only in the presence of PI(4,5)P₂, suggesting that conformational rearrangements induced upon interaction with PI(4,5)P₂ favor M polymerization (Fig. 8, A to D). Because the M dimers coat

SCIENCE ADVANCES | RESEARCH ARTICLE

Dimerization                      Plasma membrane localization          Anchoring to PM lipids           Generation of local curvature
1                                 2a                                    3                                4

Extracellular
Cytosol

Measles-M
Nipah-M

Virion formation

Oligomerization
5

6a
High concentration of M

6b
Low concentration of M

Fig. 10. Model for membrane association and assembly of paramyxovirus M proteins. Monomeric M proteins initially assemble into a dimeric confirmation (step 1). M dimers then localize to the PM inner leaflet through trafficking on PI(3,5)P₂-containing vesicles (step 2a) or nonspecific electrostatic interactions (step 2b). Anchoring to the PM inner leaflet is achieved through specific binding to PI(4,5)P₂ (magenta; step 3). Interaction with PI(4,5)P₂ causes a conformational change in M (dashed box) to open a basic patch in the NTD that then interacts with other anionic PM lipids such as PS (purple lipid). The M surface transitions from concave to flat to drive spontaneous local membrane curvature (step 4). Membrane deformation caused by one M protein creates strong, long-range attraction between membrane-bound M proteins promoting large-scale oligomerization and matrix lattice formation that further potentiates membrane deformation (step 5). M clustering and oligomerization induces local asymmetric clustering of bound lipids, reducing energy needed for membrane curvature. We hypothesize that, at sufficiently high surface densities, M proteins form a scaffold that shapes the membrane into a stable tubule or filament structure (step 6a), but at lower surface densities, M proteins shape membranes into spherical structures (step 6b). Schematic created with BioRender.com.

the exterior of the lipid filaments, the positive curvature observed in the filament structures is likely to be nonphysiological. However, the observation that M can form filaments having different curvatures [i.e., negative curvature in GUVs (Fig. 7) and positive curvature on liposomes (Fig. 8)] could suggest that dynamic plasticity of M packing could play a role in virion morphogenesis and resistance to mechanical stress. These data indicate that M lattice formation only occurs in the presence of PI(4,5)P₂, suggesting that the conformational rearrangements observed in our crystal structure favor M polymerization. Furthermore, MeV- and NiV-M filament diameters are similar to those found for helical assemblies of purified M from Sendai virus (43 to 56 nm), a paramyxovirus that infects mice (57), and the closely related pneumoviruses human respiratory syncytial virus (HRSV) and human metapneumovirus (HMPV) (HRSV, 29 nm; HMPV, 37 nm) (35, 58). The subunit spacing of individual M dimers within the helical lattice of our MeV- and NiV-M filaments is also similar to tomographic reconstructions of MeV and Newcastle disease virus (NDV) matrix layers from authentic virus (fig. S9E) (13, 14), indicating that the packing in our in vitro-assembled M filaments recapitulates the packing and assembly of M in the virion, regardless of the direction of membrane curvature, and is likely highly conserved across paramyxoviruses. Notably, alignment of MeV- and NiV-M filaments during 2D classification (Fig. 8, A and B) required application of a circular mask to first align a small subsection of the filament, suggesting that MeV- and NiV-M helical filaments comprise locally ordered patches of M dimers, similar to those observed in tomograms of the Ebola virus matrix layer (59). Membrane-bound proteins can readily diffuse and self-organize, and membrane deformation created by one protein can, in turn, recruit a second neighboring protein (60). Interactions between M and the lipid bilayer may generate strong, long-range attraction between bound M proteins, leading to large-scale protein oligomerization or scaffolding.

Extensive M polymerization leads to formation of viral particles extending from the surface of cells (Fig. 10, step 6). The angle of M dimer-dimer interactions was previously proposed to contribute to membrane curvature (14, 17). M protein clustering and oligomerization could promote asymmetric clustering of their bound lipids. Increased packing of PI(4,5)P₂ would induce local lateral asymmetry in the inner leaflet of the PM, thereby altering its physical properties to reduce energy barriers to induce membrane curvature (61). In addition, M protein oligomerization could mold the underlying lipid bilayer into a larger 3D structure that protrudes from the membrane. For BAR domain proteins, local protein density dictates the 3D protrusion morphology. At sufficiently high surface densities, BAR domain proteins form a scaffold that shapes the membrane into a stable tubule or filament structure (62, 63), but, at lower densities, spherical structures form (64, 65). Our nsEM demonstrates M proteins interacting with PI(4,5)P₂ self-assemble into long filaments with helical organization of M dimers. Virion morphology could be similarly dictated by the local concentration of M proteins and PI(4,5)P₂. Scanning electron microscopy evidence indicates that MeV- and NiV-M produce both spherical and filamentous VLPs at the surface of cells (Fig. 8, E to G). Filamentous particles have been observed for HPIV-2 and NDV paramyxoviruses (66, 67) and

Norris et al., Sci. Adv. **8**, eabn1440 (2022)   20 July 2022

pneumoviral HRSV and HMPV (68, 69). Thus, increased abundance of M-M contacts leads to filament formation, whereas reduced M crowding results in decreased M-M contacts leading to reduced membrane curvature and formation of spherical virions (Fig. 10, steps 6a and 6b). A previous work with HRSV aligns with this possibility in that the surface area of the virion membrane covered by M varies significantly depending on virus particle morphology; filamentous particles had the highest coverage (86%), and spherical viruses had the lowest (24%) (70).

The filamentous form may remain cell-associated and confer a selective advantage in cell-to-cell spread. The elongated shape may facilitate infection of neighboring cells while evading host mucociliary clearance (71, 72). Viral propagation by cell-to-cell fusion has been previously described for MeV (73, 74) and NiV (75).

Our data additionally indicate that decreasing cellular PI(4,5)P₂ by the PIP5kα inhibitor ISA-2011B causes M protein mislocalization (Fig. 1, I and J) and effectively inhibits propagation of both MeV and NiV in cell culture (Fig. 9). To the best of our knowledge, this is the first report to provide proof of principle that targeting the PI(4,5)P₂ biosynthetic pathway could be a viable broad-spectrum treatment for paramyxovirus infections. Targeting lipid metabolism is an effective antiviral strategy against multiple viruses (76, 77). Specifically, compounds that reduce cellular cholesterol can inhibit HPIVs (78). Although we found minimal toxicity for ISA-2011B in cell culture (Fig. 9C), others found that targeting lipid biosynthesis can have adverse events in vivo (77, 79). For this reason, structure-guided development of small-molecule inhibitors that target the conserved PI(4,5)P₂ binding site on paramyxovirus M proteins will be important for development of broad-spectrum antivirals against paramyxovirus infections.

Paramyxoviruses represent a major risk to human and animal health, and it is vital that efforts continue to focus on new targets within the virus replication cycle. Although future investigations are needed to determine how M proteins, lipids, and host proteins work in concert to form virions, illuminating structures and membrane interactions of M can explain how they direct virion formation and will provide 3D templates for the design of agents to inhibit paramyxovirus assembly and egress.

protein bound was determined using densitometry analysis in ImageJ, according to the following equation: Percent protein bound = (density pellet/density supernatant + pellet) × 100. Values are reported as means ± SEM. Unless otherwise indicated, three replicates were performed in duplicate.

**Enzymatic depletion of PI(4,5)P₂ in live cells**

Cells were transfected with indicated plasmids alone or cotransfected with plasmids coding for the constitutive Myc-VPtase-WT (active) or Myc-VPtase-Δ1 (inactive) as previously described (22, 44). As a positive control, we evaluated the subcellular location of the PI(4,5) P₂ sensor PLCδ-PH, which interacts with the PM by binding PI(4,5) P₂. PLCδ-PH (pleckstrin homology domain of phospholipase C delta) localized to the PM when expressed alone (fig. S2A, left column) or with the catalytically inactive MycVPtase-Δ1 mutant (fig. S2A, right column), but upon coexpression with WT MycVPtase, which reduces PM PI(4,5)P₂ levels, PLCδ-PH localized instead to the cytosol (fig. S2A, middle column). To confirm that MycVPtase expression does not interfere with peripheral protein binding to other anionic lipids (e.g., PS) within the PM, cells expressing the PS sensor Lact-C2 alone (fig. S2C, left column) or coexpressed with either MycVPtase-WT (fig. S2C, middle) or MycVPtase-Δ1 (fig. S2C, right) were also imaged. For each condition, Lact-C2 localized to the PM in ~75% of cells, indicating that enzymatic depletion of PI(4,5)P₂ did not affect PS within the PM or peripheral protein binding to PS, in agreement with a previous report (44).

**Pharmacological treatments**

**Sphingosine/charge neutralization assays**

Sphingosine aliquots were dried under a steady stream of N₂ and stored at –20°C until use. On each experimental day, a fresh aliquot was thawed and resuspended in ethanol to a final concentration of 75 mM. At 24 hours after transfection, cells were treated with either sphingosine (final concentration, 37.5 μM) or ethanol (1:2000, v/v) for 1 hour at 37°C. Cells were prepared for live cell imaging.

**Wortmannin/PIP(3) depletion**

Wortmannin (AC328590010) was purchased from Thermo Fisher Scientific (Hampton, NH). Aliquots were prepared in ethanol and stored at –20°C until use. On each experimental day, a fresh aliquot of wortmannin (200 μM) was thawed. At 24 hours after transfection, cells were treated with either wortmannin (final concentration, 100 nM) or ethanol (1:2000, v/v) for 1 hour at 37°C. Cells were then prepared for live cell imaging.

**Apilimod/PI(3)P depletion**

Apilimod (HY-14644) was purchased from MedChem Express (Monmouth Junction, NJ). Aliquots were prepared in ultra-pure grade dimethyl sulfoxide (DMSO) (VWR; 97063-136) and stored at –20°C until use. On each experimental day, a fresh aliquot of apilimod (200 μM) was thawed. At 24 hours after transfection, cells were treated with either apilimod (final concentration, 200 nM) or DMSO (1:2000, v/v) for 1 to 1.5 hours at 37°C. Cells were prepared for live cell imaging. As a positive control for apilimod treatment, the diameter of lysosomal-associated membrane protein 1 (LAMP-1)-positive intracellular vesicles was measured and found to be substantially increased in treated compared to untreated cells (2.2. μm versus 1.7 μm), which is consistent with previous reports (fig. S2L) (31).

**ISA-2011B/PIP5kα inhibition**

ISA-2011B (HY-16937) was purchased from MedChem Express (Monmouth Junction, NJ). Aliquots were prepared in ultra-pure

**MATERIALS AND METHODS**

**Liposome sedimentation assays**

Liposome sedimentation assays were performed as described in detail in (80). Briefly, MeV- and NiV-M (0.01 mg/ml) were incubated with LUVs (400 mM) at a 1:1 volume for 30 min at room temperature (RT). Following incubation, protein-bound LUVs (pellet fraction) were separated from unbound protein (supernatant fraction) through centrifugation. Samples were then subjected to SDS-polyacrylamide gel electrophoresis (SDS-PAGE) and Western blotting. Equal volumes of pellet and supernatant fractions were loaded into a 10% (w/v) SDS-PAGE gel and separated at 150 V for 45 min at RT. Samples were transferred to a nitrocellulose membrane (100 V, 45 min in ice) using ice-cold transfer buffer. Membranes were blocked with 5% (w/v) milk-TBST [20 mM tris, 150 mM NaCl, and 0.1% (w/v) Tween 20 (pH 7.4)] and subsequently probed for their respective antibodies. Horseradish peroxidase-conjugated antibodies were detected using Pierce enhanced chemiluminescence (ECL) reagent (Thermo Fisher Scientific, PI32209) or Clarity ECL substrate (Bio-Rad, 1705060) on an Amersham imager 600 (GE Healthcare). Percent

grade DMSO and stored at –20°C until use. On each experimental day, a fresh aliquot of ISA-2011B (40 mM) was thawed. At 8 hours after transfection, cells were treated with either ISA-2011B (final concentration, 40 μM) or DMSO (1:1000, v/v) for 24 hours at 37°C. Cells were then fixed using 4% (w/v) paraformaldehyde (PFA) in phosphate-buffered saline (PBS) and stored at 4°C until imaging.

### Cellular confocal microscopy

For BiFC experiments, cells were seeded onto No. 1 glass coverslips (Thermo Fisher Scientific) and the next day transfected with the indicated plasmids. At 24 hours after transfection, cells were fixed with 4% (w/v) PFA (in PBS) and permeabilized with 0.3% (v/v) Triton X-100. MeV-M proteins were probed using rabbit anti-EGFP polyclonal antibody (1:1000) (Invitrogen, #A6455). The Venus fluorescent protein used in BiFC experiments is a genetic variant of EGFP; this polyclonal anti-EGFP also recognizes both halves of the Venus fluorescent protein (Invitrogen). NiV-M proteins were probed using rabbit anti-NiV-M antibodies (1:1000) (38). This was followed by detection with secondary anti-rabbit immunoglobulin G conjugated to Alexa Fluor 647 (Invitrogen, #A27040) diluted 1:1000. All antibodies were diluted in PBS + 0.1% (v/v) Tween 20. All 3D images were acquired with a Zeiss CLSM 880 Airyscan using a 63× [1.46 numerical aperture (NA)] objective and the 32-channel GaAsP-photomultiplier tube area detector. All image stacks (on average 30 slices) were acquired with Nyquist resolution parameters using a 0.17-μm step size and an optimal frame size of 1932 × 1932. All 12-bit images were acquired using the full dynamic intensity range (0 to 4096) that was determined with the population of cells having the moderate to brightest signal expression. All Airyscan-acquired images were batch-processed using the Airyscan processing module using ideal 3D default settings.

For cellular lipid interaction assays, cells were seeded onto No 1.5 glass bottom eight-well plates (MatTek) at 70% confluence. Cells were post-stained with Hoechst 33342 (Thermo Fisher Scientific, PI62249) (final concentration, 16 μM) and wheat germ agglutinin (WGA) Alexa Fluor 647 (Thermo Fisher Scientific, W32466) (final concentration, 5 μg/ml) before imaging. Live cell imaging was performed in a live cell imaging solution (Thermo Fisher Scientific, A14291DJ), or cells were fixed with 4% (w/v) PFA (in PBS) and stored at 4°C and protected from light until imaging. Confocal imaging experiments were performed on the Zeiss LSM 880 upright microscope using an LD C-Apochromat 40× 1.1 NA water objective or Plan Apochromat 631.4× NA oil objective. A 405-nm laser was used to excite Hoechst stain, and argon lasers were used to excite EGFP (488 nm), mCherry/RFP/TopFluor (561 nm), and WGA Alexa Fluor 647 (633 nm). Percent cells with PM localization were ratio metrically determined by the number of cells with fluorescence detected at the PM compared to the number of cells without fluorescence at the PM. In each of the three replicates performed, at least 45 cells for each condition were counted. Values are reported as the means ± SEM.

### Flow cytometry

HEK293T cells were seeded in six-well plates (0.8 × 10⁶ cells per well). The next day, 250 μl of transfection mix (2.5 μg of total plasmid DNA and 7.5 μl of TransIT-LT1) was added dropwise to each ~80% confluent monolayer. Wells received a transfection mix containing plasmid DNA encoding WT VN-Matrix + VC-Matrix alone, WT VN-Matrix + VC-Matrix + mCherry, mutant VN-Matrix +

VC-Matrix + mCherry, mCherry alone, or no DNA. Twenty-four hours after transfection, cells were prepared for flow cytometry. Briefly, cells were washed once with ice-cold PBS, detached using 0.05% trypsin, fixed in 1% (w/v) PFA (in PBS), and collected by centrifugation at 200g for 5 min. Pelleted cells were resuspended in fluorescence-activated cell sorting buffer (2% fetal bovine serum in PBS) and stored on ice for flow cytometric analysis. Flow cytometry was performed using a BD LSR II flow cytometer (BD Biosciences) equipped with 488- and 561-nm laser lines. Per sample, 50,000 events were collected. FlowJo software was used to gate cells. Dead cells and cell debris were excluded from analysis using forward scatter versus side scatter, and cell aggregates were excluded by using SSC versus pulse width. Cells with BiFC signal or mCherry expression alone were used to determine the compensation and cutoff for cells without fluorescence. mCherry vector was cotransfected with BiFC constructs, and mCherry expression was used as a positive marker to indicate successful transfection. Only mCherry⁺ cells were used for measurement of BiFC fluorescence. Mean fluorescence intensity was reported as percent of positive and normalized to WT within each DNA transfection plasmid.

### VLP budding assay

Budding of VLPs into cell supernatants was detected by Western blot analyses. WT/mutant MeV-M bearing a FLAG-tag and NiV-M bearing a hemagglutinin-tag were cloned into pCMV and pcDNA3.1, respectively, and transfected into cells using TrasnIT-LT1 transfection reagent (Mirus). VLPs were harvested 24 hours after transfection. Cell culture medium was spun down at 3500 rpm for 20 min to pellet any cells out of the media. The cleared supernatants were then ultracentrifuged at 30,000 rpm with an SW-60 rotor (Beckman) for 2 hours through a 20% (w/v) sucrose cushion, 50 mM tris (pH 7.4), and 100 mM NaCl. Pelleted VLPs were resuspended in 1× NuPAGE LDS sample buffer (Thermo Fisher Scientific). Cell lysates were collected by washing cells twice with PBS followed by lysis in CytoBuster (Millipore). VLPs and cell lysates were electrophoresed on SDS denaturing gels, transferred onto polyvinylidene difluoride Immobilon-P transfer membranes (Millipore), and probed with an anti-Flag or anti-NiV-M primary antibody. The relative intensities of the bands were quantified by densitometry with a ChemiDoc MP imaging system (Bio-Rad) and ImageJ. The budding index was defined as the amount of MeV-M or NiV-M in the VLPs divided by the amount in the cell lysate across each independent experiment and normalized to the average WT MeV-M or NiV-M.

### GUV imaging

Immediately before imaging, GUVs were diluted in glucose suspension buffer [250 mM glucose, 150 mM NaCl, and 20 mM Hepes (pH 7.4)] at a 1:10 ratio. GUVs were placed on a 6-mm-diameter chamber made from a silicon sheet using a core sampling tool (EMS, #69039-60) and imaged before protein addition to ensure proper GUV formation and suspension. The indicated concentrations of NiV-M and MeV-M were added to the GUVs, and imaging was performed at 37°C on a Nikon Eclipse Ti Confocal inverted microscope (Nikon, Japan), using a Plan Apochromat ×60 1.4 NA oil objective. The 561-nm argon laser was used to excite TopFluor tetramethylrhodamine PI(4,5)P₂. In each experiment, a z-stack was taken before protein addition. After protein addition, a 40-min time lapse (1 frame/5 s) was acquired followed by z-stacks.

# Protein expression and purification

One-liter cultures of Sf9 cells at a density of $1.5 \times 10^6$ cells/ml were infected with baculovirus at a multiplicity of infection (MOI) of 1 encoding either MeV-M fused to a twin StrepII tag or NiV-M fused to a twin StrepII tag. Forty-eight hours after infection, cells were pelleted at 4000g. Cell pellets were resuspended at 4 ml/g in buffer [MeV-M: 50 mM tris (pH 8.0), 1 M NaCl; NiV-M: 50 mM tris (pH 8.0), 500 mM NaCl] and lysed using a microfluidizer. The clarified lysate was bound to a 2-ml slurry of Strep-Tactin Superflow plus (Qiagen) overnight at 4°C using the batch method. The slurry was poured into a column, and the flow-through buffer was collected. The resin was washed with 30 volumes of buffer, and the protein was eluted in 10 to 15 volumes of buffer containing 5 mM desthiobiotin. The protein was subjected to SEC using a HiLoad Superdex 200 preparative grade column (Cytiva). Fractions corresponding to MeV-M or NiV-M dimer were concentrated using Amicon Ultra centrifugal filters (10,000 molecular weight cutoff) and used for crystallization. NDSB-201 was added to MeV-M before concentration to reduce aggregation.

## Crystallization

Crystals of MeV-M were grown by the hanging drop method using a 1:1 ratio of protein to well solution with a protein concentration of 10 to 11 mg/ml. Well solution consisted of 19% (v/v) isopropanol, 19% (w/v) polyethylene glycol, molecular weight 4000 (PEG-4000), 5% (v/v) glycerol, and 0.095 M sodium citrate (pH 5.6). Crystals formed after 2 to 3 days. The crystals of NiV-M were grown using the hanging drop method with a 2:1 ratio of protein (2.6 mg/ml) to well solution and a well solution containing 0.1 M tris (pH 7.5) and 20% PEG-400. Crystals grew in 2 to 3 weeks. To obtain crystals of NiV-M in complex with C8-PI(4,5)P₂, purified protein (2.6 mg/ml) was mixed with 1 mM of C8-PI(4,5)P₂ (Avanti, #850185P) incubated for 1 hour at 37°C before setting trays. Crystals were grown using the sitting drop method using a 1:1 ratio of protein:well [0.1 M Hepes (pH 7.5) and 2.0 M ammonium sulfate]. Initial crystals were crushed using seed beads and used in subsequent optimization trials. High-quality crystals were formed using the sitting drop method with a 3:2:1 ratio of protein:well:seed in the same well solution.

## Crystallography data collection

All crystals were washed in their respective well solutions before freezing in liquid nitrogen. Data for MeV-M were collected using the Lilly Research Laboratories Collaborative Access Team (LRL-CAT) beamline at Sector 31 of the Advanced Photon Source. Data for NiV-M were collected at the Stanford Synchrotron Radiation Lightsource beamline 12-2. Data for NiV-M + C8-PI(4,5)P₂ were collected at the Argonne National Laboratory Advanced Photon Source on the GM/CA 23-ID-B beamline.

## Scanning electron microscopy

For scanning electron microscopy experiments, cells were seeded onto collagen-coated coverslips in 12-well plates at 30% confluency. Twenty-four hours after transfection, cells on coverslips were fixed in 2.5% (w/v) glutaraldehyde in 0.1 M sodium cacodylate buffer, postfixed in buffered 1% (v/v) osmium tetroxide, dehydrated with a graded series of ethanol, and dried in a Tousimis 931 critical point dryer. Dried samples were coated with platinum in a Cressington 208HR sputter coater and imaged in a FEI Nova NanoSEM 200.

---

# Negative-stain electron microscopy

MeV-M and NiV-M for single-particle analysis were diluted to a concentration of 0.02 mg/ml before negative staining. For incubation with lipid vesicles, purified MeV-M or NiV-M (10 μM) was incubated with the indicated lipids (400 μM) for 3 days at 37°C. Three microliters of sample was each applied to freshly plasma-cleaned carbon and formvar-coated 400-mesh copper grids (Electron Microscopy Sciences). After 1 min, excess liquid was wicked off and the grids were briefly washed on droplets of double-distilled water, followed by two brief washes on droplets of a 2% (v/v) aqueous uranyl acetate solution and lastly incubated on a 2% (v/v) aqueous uranyl acetate solution for 1 min. Excess stain was removed and the grids were dried thoroughly. Each sample was examined on an FEI Titan Halo 300-kV electron microscope with a Falcon 3EC camera. M proteins were imaged at ×75,000 magnification, and M proteins with liposomes were imaged at ×96,000 magnification.

## MD simulations

All-atom MD simulations were run at 300-K temperature with Langevin temperature coupling and a friction coefficient of 1 ps⁻¹. The NiV-M and MeV-M membrane systems were run for 1 s each, whereas the Apo-NiV-M and Apo-MeV-M systems were run for 0.5 s each. For calculations of hydrogen bonds, a cutoff distance of 3.5 Å and a cutoff angle of 30° were used.

## Virus inhibition assay

### Measles virus

For MeV inhibition studies, $2 \times 10^4$ Vero-hSLAM cells in 96-well plates were infected with rMV^KSEGFP(3) at an MOI of 0.1 in the presence of ISA-2011B or DMSO at the indicated concentrations. At 42 hours after infection, when cytopathic effect was maximal in the no-inhibitor control cells, fluorescent images were acquired.

### Nipah virus

All inhibition and infection studies with NiV were performed under biosafety level 4 conditions at the Institute of Virology, Philipps University Marburg. Briefly, $2 \times 10^4$ Vero76 cells seeded into 96-well plates in Dulbecco’s modified Eagle’s medium + 5% (v/v) fetal calf serum and supplemented with the indicated concentration of ISA-2011B or DMSO were infected with NiV_Malaysia at an MOI of 0.001 at 37°C. At 44 hours after infection, cells were fixed with 4% (w/v) PFA, permeabilized with ethanol, and stained with Giemsa to visualize NiV spread and plaque formation.

## Cell viability assay

To confirm that a decrease in viral infection correlated with the inhibition of viral replication and not an increase in cell death, a viability screen was run in tandem using uninfected Vero-hSLAM cells. Briefly, cells were seeded at a density of 6000 cells per well in black, flat, clear-bottom 96-well plates (Costar) and allowed to attach overnight at 37°C. The next day, medium was replaced with 200 μl of fresh medium containing indicated concentrations of compound. Cells were cultured at 37°C for 2 days. Following incubation, cell viability was determined by the addition of 20 μl of PrestoBlue viability reagent to the culture medium. The mixture was incubated at 37°C for 1 hour, and the fluorescent signal was quantified on a TECAN Spark 10M fluorescent plate reader.

Quantification and statistical analysis
Statistical details of experiments, including numbers of replicates and measures of precision, can be found in the figure legends, figures, Results, and Materials and Methods. All analyses were performed with GraphPad Prism, version 7. Additional details concerning experimental methods and data processing for structural analyses are provided in Supplementary Materials and Methods.

SUPPLEMENTARY MATERIALS
Supplementary material for this article is available at https://science.org/doi/10.1126/sciadv.abn1440

View/request a protocol for this paper from Bio-protocol.

REFERENCES AND NOTES
1. R. A. Lamb, G. D. Parks, in Fields Virology: Sixth Edition, D. M. Knipe, P. M. Howley, Eds. (Lippincott Williams and Wilkins, 2013), pp. 957–995.
2. B. Rima, A. Balkema-Buschmann, W. G. Dundon, P. Duprex, A. Easton, R. Fouchier, G. Kurath, R. Lamb, B. Lee, P. Rota, L. Wang; ICTV Report Consortium, ICTV virus taxonomy profile: Paramyxoviridae. J. Gen. Virol. **100**, 1593–1594 (2019).
3. A. Dabbagh, R. L. Laws, C. Steulet, L. Dumolard, M. N. Mulders, K. Kretsinger, J. P. Alexander, P. A. Rota, J. L. Goodson, Progress toward regional measles elimination—Worldwide, 2000–2017. MMWR Morb. Mortal. Wkly Rep. **67**, 1323–1329 (2018).
4. A. Portnoy, M. Jit, M. Ferrari, M. Hanson, L. Brenzel, S. Verguet, Estimates of case-fatality ratios of measles in low-income and middle-income countries: A systematic review and modelling analysis. Lancet Glob. Health **7**, e472–e481 (2019).
5. M. Amaya, C. C. Broder, Vaccines to emerging viruses: Nipah and Hendra. Annu. Rev. Virol. **7**, 447–473 (2020).
6. G. R. Abedi, M. M. Prill, G. E. Langley, M. E. Wikswo, G. A. Weinberg, A. T. Curns, E. Schneider, Estimates of parainfluenza virus-associated hospitalizations and cost among children aged less than 5 years in the United States, 1998–2010. J. Pediatric Infect. Dis. Soc. **5**, 7–13 (2016).
7. V. R. Brown, S. N. Bevins, A review of virulent Newcastle disease viruses in the United States and the role of wild birds in viral persistence and spread. Vet. Res. **48**, 68 (2017).
8. B. A. Jones, K. M. Rich, J. C. Mariner, J. Anderson, M. Jeggo, S. Thevasagayam, Y. Cai, A. R. Peters, P. Roeder, The economic impact of eradicating peste des petits ruminants: A benefit-cost analysis. PLOS ONE **11**, e0149982 (2016).
9. P. A. Thibault, R. E. Watkinson, A. Moreira-Soto, J. F. Drexler, B. Lee, Zoonotic potential of emerging paramyxoviruses: Knowns and unknowns. Adv. Virus Res. **98**, 1–55 (2017).
10. M. S. Harrison, T. Sakaguchi, A. P. Schmitt, Paramyxovirus assembly and budding: Building particles that transmit infections. Int. J. Biochem. Cell Biol. **42**, 1416–1429 (2010).
11. T. Cathomen, B. Mrkic, D. Spehner, R. Drillien, R. Naef, J. Pavlovic, A. Aguzzi, M. A. Billette, R. Cattaneo, A matrix-less measles virus is infectious and elicits extensive cell fusion: Consequences for propagation in the brain. EMBO J. **17**, 3899–3908 (1998).
12. E. Dietzel, L. Kolesnikova, B. Sawatsky, A. Heiner, M. Weis, G. P. Kobinger, S. Becker, V. von Messling, A. Maisner, Nipah virus matrix protein influences fusogenicity and is essential for particle infectivity and stability. J. Virol. **90**, 2514–2522 (2015).
13. Z. Ke, J. D. Strauss, C. M. Hampton, M. A. Brindley, R. S. Dillard, F. Leon, K. M. Lamb, R. K. Plemper, E. R. Wright, Promotion of virus assembly and organization by the measles virus matrix protein. Nat. Commun. **9**, 1736 (2018).
14. A. J. Battisti, G. Meng, D. C. Winkler, L. W. McGinnes, P. Plevka, A. C. Steven, T. G. Morrison, M. G. Rossmann, Structure and assembly of a paramyxovirus matrix protein. Proc. Natl. Acad. Sci. U.S.A. **109**, 13996–14000 (2012).
15. G. Ray, P. T. Schmitt, A. P. Schmitt, C-terminal DxD-containing sequences within paramyxovirus nucleocapsid proteins determine matrix protein compatibility and can direct foreign proteins into budding particles. J. Virol. **90**, 3650–3660 (2016).
16. F. El Najjar, A. P. Schmitt, R. E. Dutch, Paramyxovirus glycoprotein incorporation, assembly and budding: A three way dance for infectious particle production. Viruses **6**, 3019–3054 (2014).
17. Y. C. Liu, J. Grusovin, T. E. Adams, Electrostatic interactions between hendra virus matrix proteins are required for efficient virus-like-particle assembly. J. Virol. **92**, e00143-18 (2018).
18. K. S. Faaberg, M. E. Peeples, Association of soluble matrix protein of Newcastle disease virus with liposomes is independent of ionic conditions. Virology **166**, 123–132 (1988).
19. R. Stricker, G. Mottet, L. Roux, The Sendai virus matrix protein appears to be recruited in the cytoplasm by the viral nucleocapsid to function in viral assembly and budding. J. Gen. Virol. **75**, 1031–1042 (1994).
20. P. Riedl, M. Moll, H.-D. Klenk, A. Maisner, Measles virus matrix protein is not cotransported with the viral glycoproteins but requires virus infection for efficient surface targeting. Virus Res. **83**, 1–12 (2002).

21. T. Yeung, G. E. Gilbert, J. Shi, J. Silvius, A. Kapus, S. Grinstein, Membrane phosphatidylserine regulates surface charge and protein localization. Science **319**, 210–213 (2008).
22. K. A. Johnson, G. J. F. Taghon, J. L. Scott, R. V. Stahelin, The ebola virus matrix protein, VP40, requires phosphatidylinositol 4,5-bisphosphate (PI(4,5)P2) for extensive oligomerization at the plasma membrane and viral egress. Sci. Rep. **6**, 19125 (2016).
23. T. Yeung, M. Terebiznik, L. Yu, J. Silvius, W. M. Abidi, M. Philips, T. Levine, A. Kapus, S. Grinstein, Receptor activation alters inner surface potential during phagocytosis. Science **313**, 347–351 (2006).
24. R. Subhashri, M. S. Shaila, Characterization of membrane association of Rinderpest virus matrix protein. Biochem. Biophys. Res. Commun. **355**, 1096–1101 (2007).
25. A. V. Shnyrova, J. Ayllon, I. I. Mikhaljov, E. Villar, J. Zimmerberg, V. A. Frolov, Vesicle formation by self-assembly of membrane-bound matrix proteins into a fluidlike budding domain. J. Cell Biol. **179**, 627–633 (2007).
26. G. R. V. Hammond, J. E. Burke, Novel roles of phosphoinositides in signaling, lipid transport, and disease. Curr. Opin. Cell Biol. **63**, 57–67 (2020).
27. W. Cho, R. V. Stahelin, Membrane-protein interactions in cell signaling and membrane trafficking. Annu. Rev. Biophys. Biomol. Struct. **34**, 119–151 (2005).
28. Y. Yang, M. Lee, G. D. Fairn, Phospholipid subcellular localization and dynamics. J. Biol. Chem. **293**, 6230–6240 (2018).
29. J. E. Burke, Structural basis for regulation of phosphoinositide kinases and their involvement in human disease. Mol. Cell **71**, 653–673 (2018).
30. G. Powis, R. Bonjouklian, M. M. Berggren, A. Gallegos, R. Abraham, C. Ashendel, L. Zalkow, W. F. Matter, J. Dodge, G. Grindey, C. J. Vlahos, Wortmannin, a potent and selective inhibitor of phosphatidylinositol-3-kinase. Cancer Res. **54**, 2419–2423 (1994).
31. D. Sbrissa, G. Naisan, O. C. Ikonomov, A. Shisheva, Apilimod, a candidate anticancer therapeutic, arrests not only Ptdlns(3,5)P2 but also Ptdlns5P synthesis by PIKfyve and induces bafilomycin A1-reversible aberrant endomembrane dilation. PLOS ONE **13**, e0204532 (2018).
32. T. K. Kerppola, Bimolecular fluorescence complementation (BiFC) analysis as a probe of protein interactions in living cells. Annu. Rev. Biophys. **37**, 465–487 (2008).
33. S. Welsch, B. Müller, H.-G. Kräusslich, More than one door - Budding of enveloped viruses through cellular membranes. FEBS Lett. **581**, 2089–2097 (2007).
34. F. Bringolf, M. Herren, M. Wyss, B. Vidondo, J. P. Langedijk, A. Zurbriggen, P. Plattet, Dimerization efficiency of canine distemper virus matrix protein regulates membrane-budding activity. J. Virol. **91**, e00521-17 (2017).
35. C. Leyrat, M. Renner, K. Harlos, J. T. Huiskonen, J. M. Grimes, Structure and self-assembly of the calcium binding matrix protein of human metapneumovirus. Structure **22**, 136–148 (2014).
36. V. A. Money, H. K. McPhee, J. A. Mosely, J. M. Sanderson, R. P. Yeo, Surface features of a Mononegavirales matrix protein indicate sites of membrane interaction. Proc. Natl. Acad. Sci. U.S.A. **106**, 4441–4446 (2009).
37. G. van Meer, D. R. Voelker, G. W. Feigenson, Membrane lipids: Where they are and how they behave. Nat. Rev. Mol. Cell Biol. **9**, 112–124 (2008).
38. Y. E. Wang, A. Park, M. Lake, M. Pentecost, B. Torres, T. E. Yun, M. C. Wolf, M. R. Holbrook, A. N. Freiberg, B. Lee, Ubiquitin-regulated nuclear-cytoplasmic trafficking of the Nipah virus matrix protein is important for viral budding. PLOS Pathog. **6**, e1001186 (2010).
39. M. Ringel, A. Heiner, L. Behner, S. Halwe, L. Sauerhering, N. Becker, E. Dietzel, B. Sawatsky, L. Kolesnikova, A. Maisner, Nipah virus induces two inclusion body populations: Identification of novel inclusions at the plasma membrane. PLOS Pathog. **15**, e1007733 (2019).
40. E. Adu-Gyamfi, K. A. Johnson, M. E. Fraser, J. L. Scott, S. P. Soni, K. R. Jones, M. A. Digman, E. Gratton, C. R. Tessier, R. V. Stahelin, Host cell plasma membrane phosphatidylerine regulates the assembly and budding of ebola virus. J. Virol. **89**, 9440–9453 (2015).
41. S. Bobone, M. Hilsch, J. Storm, V. Dunsing, A. Herrmann, S. Chiantia, Phosphatidylerine lateral organization influences the interaction of influenza virus matrix protein 1 with lipid membranes. J. Virol. **91**, e00267-17 (2017).
42. M. Doktorova, F. A. Heberle, R. L. Kingston, G. Khelashvili, M. A. Cuendet, Y. Wen, J. Katsaras, G. W. Feigenson, V. M. Vogt, R. A. Dick, Cholesterol promotes protein binding by affecting membrane electrostatics and solvation properties. Biophys. J. **113**, 2004–2015 (2017).
43. K. J. Wijesinghe, R. V. Stahelin, Investigation of the lipid binding properties of the marburg virus matrix protein VP40. J. Virol. **90**, 3074–3085 (2015).
44. A. Ono, S. D. Ablan, S. J. Lockett, K. Nagashima, E. O. Freed, Phosphatidylinositol (4,5) bisphosphate regulates HIV-1 Gag targeting to the plasma membrane. Proc. Natl. Acad. Sci. U.S.A. **101**, 14889–14894 (2004).
45. N. P. Kadzioch, M. Gast, F. Origgi, P. Plattet, Clustered lysine residues of the canine distemper virus matrix protein regulate membrane association and budding activity. J. Virol. **95**, e01269-20 (2020).
46. H. Ueda, N. Yamakawa, K. Takeuchi, Amino- and carboxyl-terminal ends of the bovine parainfluenza virus type 3 matrix protein are important for virion and virus-like particle release. Virology **561**, 17–27 (2021).

47. E. K. J. Tuominen, C. J. A. Wallace, P. K. J. Kinnunen, Phospholipid-cytochrome c interaction: Evidence for the extended lipid anchorage. *J. Biol. Chem.* **277**, 8822–8826 (2002).

48. P. K. J. Kinnunen, On the molecular-level mechanisms of peripheral protein-membrane interactions induced by lipids forming inverted non-lamellar phases. *Chem. Phys. Lipids* **81**, 151–166 (1996).

49. J. S. Saad, J. Miller, J. Tai, A. Kim, R. H. Ghanam, M. F. Summers, Structural basis for targeting HIV-1 Gag proteins to the plasma membrane for virus assembly. *Proc. Natl. Acad. Sci. U.S.A.* **103**, 11364–11369 (2006).

50. U. Kragh-Hansen, F. Hellec, B. de Foresta, M. le Maire, J. V. Möller, Detergents as probes of hydrophobic binding cavities in serum albumin and other water-soluble proteins. *Biophys. J.* **80**, 2898–2911 (2001).

51. R. Behnia, S. Munro, Organelle identity and the signposts for membrane traffic. *Nature* **438**, 597–604 (2005).

52. F. Mücksch, V. Laketa, B. Müller, C. Schultz, H.-G. Kräusslich, Synchronized HIV assembly by tunable PIP₂ changes reveals PIP₂ requirement for stable Gag anchoring. *eLife* **6**, e25287 (2017).

53. J. B. Gc, B. S. Gerstman, R. V. Stahelin, P. P. Chapagain, The Ebola virus protein VP40 hexamer enhances the clustering of PI(4,5)P₂ lipids in the plasma membrane. *Phys. Chem. Chem. Phys.* **18**, 28409–28417 (2016).

54. J. Saarikangas, H. Zhao, A. Pykäläinen, P. Laurinmäki, P. K. Mattila, P. K. J. Kinnunen, S. J. Butcher, P. Lappalainen, Molecular mechanisms of membrane deformation by I-BAR domain proteins. *Curr. Biol.* **19**, 95–107 (2009).

55. A. Pykäläinen, M. Boczkowska, H. Zhao, J. Saarikangas, G. Rebowski, M. Jansen, J. Hakanen, E. V. Koskela, J. Peränen, H. Vihinen, E. Jokitalo, M. Salminen, E. Ikonen, R. Dominguez, P. Lappalainen, Pinkbar is an epithelial-specific BAR domain protein that generates planar membrane structures. *Nat. Struct. Mol. Biol.* **18**, 902–907 (2011).

56. W. Pezeshkian, A. G. Hansen, L. Johannes, H. Khandelia, J. C. Shillcock, P. B. S. Kumar, J. H. Ipsen, Membrane invagination induced by Shiga toxin B-subunit: From molecular structure to tube formation. *Soft Matter* **12**, 5164–5171 (2016).

57. M. H. Heggeness, P. R. Smith, P. W. Choppin, In vitro assembly of the nonglycosylated membrane protein (M) of Sendai virus. *Proc. Natl. Acad. Sci. U.S.A.* **79**, 6232–6236 (1982).

58. H. K. McPhee, J. L. Carlisle, A. Beeby, V. A. Money, S. M. D. Watson, R. P. Yeo, J. M. Sanderson, Influence of lipids on the interfacial disposition of respiratory syncytical virus matrix protein. *Langmuir* **27**, 304–311 (2011).

59. W. Wan, M. Clarke, M. J. Norris, L. Kolesnikova, A. Koehler, Z. A. Bornholdt, S. Becker, E. O. Saphire, J. A. Briggs, Ebola and Marburg virus matrix layers are locally ordered assemblies of VP40 dimers. *eLife* **9**, e59225 (2020).

60. M. Simunovic, E. Evergren, A. Callan-Jones, P. Bassereau, Curving cells inside and out: Roles of BAR domain proteins in membrane shaping and its cellular implications. *Annu. Rev. Cell Dev. Biol.* **35**, 111–129 (2019).

61. P. A. Janmey, P. K. J. Kinnunen, Biophysical properties of lipids and dynamic membranes. *Trends Cell Biol.* **16**, 538–546 (2006).

62. B. Sorre, A. Callan-Jones, J. Manzi, B. Goud, J. Prost, P. Bassereau, A. Roux, Nature of curvature coupling of amphiphysin with membranes depends on its bound density. *Proc. Natl. Acad. Sci. U.S.A.* **109**, 173–178 (2012).

63. F. Bonazzi, T. R. Weikl, Membrane morphologies induced by Arc-shaped scaffolds are determined by Arc angle and coverage. *Biophys. J.* **116**, 1239–1247 (2019).

64. Z. Shi, T. Baumgart, Membrane tension and peripheral protein density mediate membrane shape transitions. *Nat. Commun.* **6**, 5974 (2015).

65. J. M. Isas, M. R. Ambroso, P. B. Hegde, J. Langen, R. Langen, Tubulation by amphiphysin requires concentration-dependent switching from wedging to scaffolding. *Structure* **23**, 873–881 (2015).

66. Q. Yao, R. W. Compans, Filamentous particle formation by human parainfluenza virus type 2. *J. Gen. Virol.* **81**, 1305–1312 (2000).

67. F. B. Bang, Formation of filamentous forms of Newcastle disease virus in hypertonic concentration of sodium chloride. *Proc. Soc. Exp. Biol. Med.* **71**, 50–52 (1949).

68. L. Liljeroos, M. A. Krzyzaniak, A. Helenius, S. J. Butcher, Architecture of respiratory syncytial virus revealed by electron cryotomography. *Proc. Natl. Acad. Sci. U.S.A.* **110**, 11133–11138 (2013).

69. Z. Ke, R. S. Dillard, T. Chirkova, F. Leon, C. C. Stobart, C. M. Hampton, J. D. Strauss, D. Rajan, C. A. Rostad, J. V. Taylor, H. Yi, R. Shah, M. Jin, T. V. Hartert, R. S. Peebles Jr., B. S. Graham, M. L. Moore, L. J. Anderson, E. R. Wright, The morphology and assembly of respiratory syncytial virus revealed by cryo-electron tomography. *Viruses* **10**, 446 (2018).

70. G. Kiss, J. M. Holl, G. M. Williams, E. Alonas, D. Vanover, A. W. Lifland, M. Gudheti, R. C. Guerrero-Ferreira, V. Nair, H. Yi, B. S. Graham, P. J. Santangelo, E. R. Wright, Structural analysis of respiratory syncytial virus reveals the position of M2-1 between the matrix protein and the ribonucleoprotein complex. *J. Virol.* **88**, 7602–7617 (2014).

71. J. Seladi-Schulman, J. Steel, A. C. Lowen, Spherical influenza viruses have a fitness advantage in embryonated eggs, while filament-producing strains are selected in vivo. *J. Virol.* **87**, 13343–13353 (2013).

72. P. J. Campbell, S. Danzy, C. S. Kyriakis, M. J. Deymier, A. C. Lowen, J. Steel, The M segment of the 2009 pandemic influenza virus confers increased neuraminidase activity, filamentous morphology, and efficient contact transmissibility to A/Puerto Rico/8/1934-based reassortant viruses. *J. Virol.* **88**, 3802–3814 (2014).

73. F. Herschke, S. Plumet, T. Duhen, O. Azocar, J. Druelle, D. Laine, T. F. Wild, C. Rabourdin-Combe, D. Gerlier, H. Valentin, Cell-cell fusion induced by measles virus amplifies the type I interferon response. *J. Virol.* **81**, 12859–12871 (2007).

74. W. P. Duprex, S. McQuaid, L. Hangartner, M. A. Billeter, B. K. Rima, Observation of measles virus cell-to-cell spread in astrocytoma cells by using a green fluorescent protein-expressing recombinant virus. *J. Virol.* **73**, 9568–9575 (1999).

75. B. Lamp, E. Dietzel, L. Kolesnikova, L. Sauerhering, S. Erbar, H. Weingartl, A. Maisner, Nipah virus entry and egress from polarized epithelial cells. *J. Virol.* **87**, 3143–3154 (2013).

76. A. Beziau, D. Brand, E. Piver, The role of phosphatidylinositol phosphate kinases during viral infection. *Viruses* **12**, 1124 (2020).

77. A. Fernández-Oliva, P. Ortega-González, C. Risco, Targeting host lipid flows: Exploring new antiviral and antibiotic strategies. *Cell. Microbiol.* **21**, e12996 (2019).

78. S. Bajimaya, T. Hayashi, T. Frankl, P. Bryk, B. Ward, T. Takimoto, Cholesterol reducing agents inhibit assembly of type I parainfluenza viruses. *Virology* **501**, 127–135 (2017).

79. C. Spickler, J. Lippens, M.-K. Laberge, S. Desmeules, É. Bellavance, M. Garneau, T. Guo, O. Hucke, P. Leyssen, J. Neyts, F. H. Vaillancourt, A. Décor, J. O’Meara, M. Franti, A. Gauthier, Phosphatidylinositol 4-kinase III beta is essential for replication of human rhinovirus and its inhibition causes a lethal phenotype in vivo. *Antimicrob. Agents Chemother.* **57**, 3358–3368 (2013).

80. M. M. Julkowska, J. M. Rankenberg, C. Testerink, Liposome-binding assays to assess specificity and affinity of phospholipid-protein interactions. *Methods Mol. Biol.* **1009**, 261–271 (2013).

81. N. A. Baker, D. Sept, S. Joseph, M. J. Holst, J. A. McCammon, Electrostatics of nanosystems: Application to microtubules and the ribosome. *Proc. Natl. Acad. Sci. U.S.A.* **98**, 10037–10041 (2001).

82. T. J. Dolinsky, P. Czodrowski, H. Li, J. E. Nielsen, J. H. Jensen, G. Klebe, N. A. Baker, PDB2PQR: Expanding and upgrading automated preparation of biomolecular structures for molecular simulations. *Nucleic Acids Res.* **35**, W522–W525 (2007).

83. M. Pentecost, A. A. Vashisht, T. Lester, T. Voros, S. M. Beaty, A. Park, Y. E. Wang, T. E. Yun, A. N. Freiberg, J. A. Wohlschlegel, B. Lee, Evidence for ubiquitin-regulated nuclear and subnuclear trafficking among Paramyxovirinae matrix proteins. *PLOS Pathog.* **11**, e1004739 (2015).

84. Y. J. Shyu, H. Liu, X. Deng, C.-D. Hu, Identification of new fluorescent protein fragments for bimolecular fluorescence complementation analysis under physiological conditions. *Biotechniques* **40**, 61–66 (2006).

85. E. Adu-Gyamfi, M. A. Digman, E. Gratton, R. V. Stahelin, Investigation of Ebola VP40 assembly and oligomerization in live cells using number and brightness analysis. *Biophys. J.* **102**, 2517–2525 (2012).

86. N. Ono, H. Tatsuo, Y. Hidaka, T. Aoki, H. Minagawa, Y. Yanagi, Measles viruses on throat swabs from measles patients use signaling lymphocytic activation molecule (CDw150) but not CD46 as a cellular receptor. *J. Virol.* **75**, 4399–4401 (2001).

87. K. Lemon, R. D. de Vries, A. W. Mesman, S. McQuaid, G. van Amerongen, S. Yüksel, M. Ludlow, L. J. Rennick, T. Kuiken, B. K. Rima, T. B. H. Geijtenbeek, A. D. M. Osterhaus, W. Paul Duprex, R. L. de Swart, Early target cells of measles virus after aerosol infection of non-human primates. *PLOS Pathog.* **7**, e1001263 (2011).

88. M. Moll, S. Diederich, H.-D. Klenk, M. Czub, A. Maisner, Ubiquitous activation of the Nipah virus fusion protein does not require a basic amino acid at the cleavage site. *J. Virol.* **78**, 9705–9712 (2004).

89. A. Darszon, C. A. Vandenberg, M. Schönfeld, M. H. Ellisman, N. C. Spitzer, M. Montal, Reassembly of protein-lipid complexes into large bilayer vesicles: Perspectives for membrane reconstitution. *Proc. Natl. Acad. Sci. U.S.A.* **77**, 239–243 (1980).

90. Y. Yamashita, M. Oka, T. Tanaka, M. Yamazaki, A new method for the preparation of giant liposomes in high salt concentrations and growth of protein microcrystals in them. *Biochim. Biophys. Acta* **1561**, 129–134 (2002).

91. C. Vonrhein, C. Flensburg, P. Keller, A. Sharff, O. Smart, W. Paciorek, T. Womack, G. Bricogne, Data processing and analysis with the auto PROC toolbox. *Acta Crystallogr. D Biol. Crystallogr.* **67**, 293–302 (2011).

92. P. D. Adams, P. V. Afonine, G. Bunkóczi, V. B. Chen, I. W. Davis, N. Echols, J. J. Headd, L.-W. Hung, G. J. Kapral, R. W. Grosse-Kunstleve, A. J. McCoy, N. W. Moriarty, R. Oeffner, R. J. Read, D. C. Richardson, J. S. Richardson, T. C. Terwilliger, P. H. Zwart, PHENIX: A comprehensive Python-based system for macromolecular structure solution. *Acta Crystallogr. D Biol. Crystallogr.* **66**, 213–221 (2010).

93. P. Emsley, B. Lohkamp, W. G. Scott, K. Cowtan, Features and development of Coot. *Acta Crystallogr. D Biol. Crystallogr.* **66**, 486–501 (2010).

94. O. S. Smart, T. O. Womack, C. Flensburg, P. Keller, W. Paciorek, A. Sharff, C. Vonrhein, G. Bricogne, Exploiting structure similarity in refinement: Automated NCS and target-structure restraints in BUSTER. *Acta Crystallogr. D Biol. Crystallogr.* **68**, 368–380 (2012).

95. J. Painter, E. A. Merritt, Optimal description of a protein structure in terms of multiple groups undergoing TLS motion. *Acta Crystallogr. D Biol. Crystallogr.* **62**, 439–450 (2006).

96. J. Painter, E. A. Merritt, TLSMD web server for the generation of multi-group TLS models. *J. Appl. Cryst.* **39**, 109–111 (2006).

97. A. T. Brünger, Free R value: A novel statistical quantity for assessing the accuracy of crystal structures. *Nature* **355**, 472–475 (1992).

98. E. Krissinel, K. Henrick, Inference of macromolecular assemblies from crystalline state. *J. Mol. Biol.* **372**, 774–797 (2007).

99. E. Folta-Stogniew, Oligomeric states of proteins determined by size-exclusion chromatography coupled with light scattering, absorbance, and refractive index detectors. *Methods Mol. Biol.* **328**, 97–112 (2006).

100. A. Punjani, J. L. Rubinstein, D. J. Fleet, M. A. Brubaker, cryoSPARC: Algorithms for rapid unsupervised cryo-EM structure determination. *Nat. Methods* **14**, 290–296 (2017).

101. E. F. Pettersen, T. D. Goddard, C. C. Huang, E. C. Meng, G. S. Couch, T. I. Croll, J. H. Morris, T. E. Ferrin, UCSF ChimeraX: Structure visualization for researchers, educators, and developers. *Protein Sci.* **30**, 70–82 (2021).

102. S. He, S. H. W. Scheres, Helical reconstruction in RELION. *J. Struct. Biol.* **198**, 163–176 (2017).

103. E. L. Wu, X. Cheng, S. Jo, H. Rui, K. C. Song, E. M. Dávila-Contreras, Y. Qi, J. Lee, V. Monje-Galvan, R. M. Venable, J. B. Klauda, W. Im, CHARMM-GUI membrane builder toward realistic biological membrane simulations. *J. Comput. Chem.* **35**, 1997–2004 (2014).

104. N. Eswar, B. Webb, M. A. Marti-Renom, M. S. Madhusudhan, D. Eramian, M.-Y. Shen, U. Pieper, A. Sali, Comparative protein structure modeling using Modeller. *Curr Protoc Bioinformatics*. Chapter 5, Unit 5.6 (2007).

105. A. Zachowski, Phospholipids in animal eukaryotic membranes: Transverse asymmetry and movement. *Biochem. J.* **294**, 1–14 (1993).

106. J. B. Gc, B. S. Gerstman, P. P. Chapagain, Membrane association and localization dynamics of the Ebola virus matrix protein VP40. *Biochim. Biophys. Acta Biomembr.* **1859**, 2012–2020 (2017).

107. J. C. Phillips, R. Braun, W. Wang, J. Gumbart, E. Tajkhorshid, E. Villa, C. Chipot, R. D. Skeel, L. Kalé, K. Schulten, Scalable molecular dynamics with NAMD. *J. Comput. Chem.* **26**, 1781–1802 (2005).

108. J. Huang, S. Rauscher, G. Nawrocki, T. Ran, M. Feig, B. L. de Groot, H. Grubmüller, A. D. MacKerell Jr., CHARMM36m: An improved force field for folded and intrinsically disordered proteins. *Nat. Methods* **14**, 71–73 (2017).

109. U. Essmann, L. Perera, M. L. Berkowitz, T. Darden, H. Lee, L. G. Pedersen, A smooth particle mesh Ewald method. *J. Chem. Phys.* **103**, 8577–8593 (1995).

110. J.-P. Ryckaert, G. Ciccotti, H. J. C. Berendsen, Numerical integration of the cartesian equations of motion of a system with constraints: Molecular dynamics of n-alkanes. *J. Comput. Phys.* **23**, 327–341 (1977).

111. W. Humphrey, A. Dalke, K. Schulten, VMD: Visual molecular dynamics. *J. Mol. Graph.* **14**, 33–38 (1996).

112. J. Schindelin, I. Arganda-Carreras, E. Frise, V. Kaynig, M. Longair, T. Pietzsch, S. Preibisch, C. Rueden, S. Saalfeld, B. Schmid, J.-Y. Tinevez, D. J. White, V. Hartenstein, K. Eliceiri, P. Tomancak, A. Cardona, Fiji: An open-source platform for biological-image analysis. *Nat. Methods* **9**, 676–682 (2012).

113. M. Weis, L. Behner, M. Hoffmann, N. Krüger, G. Herrler, C. Drosten, J. F. Drexler, E. Dietzel, A. Maisner, Characterization of African bat henipavirus GH-M74a glycoproteins. *J. Gen. Virol.* **95**, 539–548 (2014).

Acknowledgments: We gratefully acknowledge K. Fuller and T. Alkutkar for technical assistance. Data for the NiV-M structure were collected on beamline 12-2 of the Stanford Synchrotron Radiation Lightsource. Use of the Stanford Synchrotron Radiation Lightsource, SLAC National Accelerator Laboratory, is supported by the U.S. Department of Energy (DOE), Office of Science, and Office of Basic Energy Sciences under contract no. DE-AC02-76SF00515. The SSRL Structural Molecular Biology Program is supported by the DOE Office of Biological and Environmental Research and by the National Institutes of Health (NIH) and National Institute of General Medical Sciences (P30GM133894). The contents of this publication are solely the responsibility of the authors and do not necessarily represent the official views of NIGMS or NIH. Data for the NiV-M + C8-PI(4,5)P₂ structure were collected on beamlines 23-ID-B of the Advanced Photon Source, a U.S. DOE Office of Science user facility operated for the DOE Office of Science by Argonne National Laboratory under contract no. DE-AC02-06CH11357. Data for the MeV-M structure were obtained through use of the Lilly Research Laboratories Collaborative Access Team (LRL-CAT) beamline at Sector 31 of the Advanced Photon Source provided by Eli Lilly Company, which operates the facility. The La Jolla Institute Zeiss LSM 880 was acquired through the Shared Instrumentation Grant Program (S10): NIH S10OD021831.

Funding: This work was supported by institutional funds of the La Jolla Institute for Immunology (to E.O.S.); NIH/NIAID AI081077 and NIH S100D027043 (to R.V.S.); the Henry L. Hillman Foundation (to W.P.D.); the Deutsche Forschungsgemeinschaft (German Research Foundation, Projektnummer 197785619 - SFB 1021) (to A.M.); and NIH/NIAID AI125536 AI123449 (to B.L.).

Author contributions: Conceptualization: M.J.N., R.V.S., and E.O.S. Methodology: M.J.N., M.L.H., W.B.K., S.S.H., P.P.C., A.M., L.J.R., B.L., and W.P.D. Validation: M.J.N., M.L.H., W.B.K., J.Y., L.J.R., A.H., and S.S.H. Formal analysis: M.J.N. and M.L.H. Investigation: M.J.N., M.L.H., W.B.K., J.Y., R.S., L.J.R., A.H., S.S.H., R.P., S.L.-B., Z.L.S., and P.P.C. Resources: B.L., A.M., W.P.D., R.V.S., and E.O.S. Data curation: M.J.N. Writing—original draft: M.J.N. and S.L.S. Writing—review and editing: M.J.N., S.L.S., K.M.H., L.J.R., B.L., P.P.C., A.M., W.P.D., R.V.S., and E.O.S. Visualization: M.J.N., M.L.H., W.B.K., and P.P.C. Supervision: P.P.C., A.M., W.P.D., R.V.S., and E.O.S. Project administration: M.J.N., R.V.S., and E.O.S. Funding acquisition: B.L., P.P.C., A.M., W.P.D., R.V.S., and E.O.S.

Competing interests: The authors declare that they have no competing interests.

Data and materials availability: All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. Requests for resources and reagents should be directed to and will be fulfilled by the lead contacts, E.O.S. (erica@lji.org) and R.V.S. (rstaheli@purdue.edu). Structure factors and associated model coordinates of MeV-M, NiV-M, and NiV-M + 08:0 PI(4,5)P₂ have been deposited in the Protein Data Bank (www.rcsb.org) under accession numbers 7SKS, 7SKT, and 7SKU, respectively.

Submitted 5 November 2021  
Accepted 6 June 2022  
Published 20 July 2022  

10.1126/sciadv.abn1440
