
Systematic Review

# Genotype-Phenotype Correlations in Human Diseases Caused by Mutations of LINC Complex-Associated Genes: A Systematic Review and Meta-Summary

Emily C. Storey ${}^{1,2}$ and Heidi R. Fuller ${}^{1,2,*}$

${}^{1}$ School of Pharmacy and Bioengineering, Keele University, Staffordshire ST5 5BG, UK  
${}^{2}$ Wolfson Centre for Inherited Neuromuscular Disease, TORCH Building, RJAH Orthopaedic Hospital, Oswestry SY10 7AG, UK  

* Correspondence: h.r.fuller@keele.ac.uk; Tel.: +44-(0) 1782-734546

---

**Abstract:** Mutations in genes encoding proteins associated with the linker of nucleoskeleton and cytoskeleton (LINC) complex within the nuclear envelope cause different diseases with varying phenotypes including skeletal muscle, cardiac, metabolic, or nervous system pathologies. There is some understanding of the structure of LINC complex-associated proteins and how they interact, but it is unclear how mutations in genes encoding them can cause the same disease, and different diseases with different phenotypes. Here, published mutations in LINC complex-associated proteins were systematically reviewed and analyzed to ascertain whether patterns exist between the genetic sequence variants and clinical phenotypes. This revealed *LMNA* is the only LINC complex-associated gene in which mutations commonly cause distinct conditions, and there are no clear genotype-phenotype correlations. Clusters of *LMNA* variants causing striated muscle disease are located in exons 1 and 6, and metabolic disease-associated *LMNA* variants are frequently found in the tail of lamin A/C. Additionally, exon 6 of the emerin gene, *EMD*, may be a mutation “hot-spot”, and diseases related to *SYNE1*, encoding nesprin-1, are most often caused by nonsense type mutations. These results provide insight into the diverse roles of LINC-complex proteins in human disease and provide direction for future gene-targeted therapy development.

**Keywords:** LINC complex; nuclear envelope; laminopathies; lamin A/C; *LMNA*; *SYNE1*; *EMD*; nesprin; emerin

---

## 1. Introduction

Mutations in genes encoding nuclear envelope (NE) proteins have been associated with a range of human disorders that are collectively known as “nuclear envelopathies” [1–3], additionally, the multiple diseases caused by mutations in the nuclear lamins are termed “laminopathies” [4,5]. Mutations in these genes give rise to diseases affecting skeletal and cardiac muscle (such as Emery-Dreifuss muscular dystrophy (EDMD), limb-girdle muscular dystrophy (LGMD), congenital muscular dystrophy (L-CMD) and dilated cardiomyopathy with conduction defects (DCM)), nervous system diseases (including Charcot-Marie tooth disease (CMT), cerebellar ataxia (SCAR8) and amyotrophic lateral sclerosis (ALS)), and metabolic diseases (such as lipodystrophies), amongst others, as shown in (Figure 1).

The NE is a lipid bilayer structure, functioning to separate the nuclear contents from the cytoplasm. It consists of two lipid bilayer membranes, the inner nuclear membrane (INM) and the outer nuclear membrane (ONM), which are separated by a gap known as the perinuclear space (PNS) [6]. The ONM is a continuation of the rough endoplasmic reticulum, whilst the INM interacts with chromatin and the nuclear lamina (NL), a protein meshwork that composes the nuclear skeleton [7]. Type V-intermediate filament proteins, nuclear lamins, are the major components of the NL [8–10]. Lamins A and C are A-type lamins and are the alternatively spliced products of the *LMNA* gene. B-type lamins,

---

Cells 2022, 11, 4065. https://doi.org/10.3390/cells11244065   https://www.mdpi.com/journal/cells

lamina B1 and lamina B2, on the other hand, are encoded by two different genes, LMNB1 and LMNB2 [11]. The NL is connected to the cytoskeleton through a network called the Linker of Nucleoskeleton and Cytoskeleton (LINC) complex [12]. The central components of the LINC complex are SUN1/SUN2, encoded by SUN1/SUN2, and nesprin proteins, derived from SYNE1/SYNE2 [13]. These proteins are situated in the INM and ONM, respectively [13]. The Sad1 and UNC-84 (SUN) and Klarsicht/Anc-1/Syne Homology (KASH) domains of the respective proteins interact in the PNS to form a connection that spans the INM, PNS and ONM [14,15]. SUN proteins are anchored to the NL by their N-termini, attaching the LINC complex to the NE, whilst the cytoplasmic domains of the nesprins connect to the cytoskeleton [16–18]. Additionally, a number of other NE proteins are known to be associated with the LINC complex including INM proteins emerin and TMEM43 (also known as LUMA), and isoforms of four-and-a-half lim domain protein 1 (FHL1) which shuttle between the nucleus and the cytoplasm [19].

![Diagram](attachment:diagram.png)

Figure 1. The Linker of Nucleoskeleton and Cytoskeleton (LINC) complex and associated human diseases. The Linker of Nucleoskeleton and Cytoskeleton (LINC) complex primarily consists of SUN1/SUN2 and nesprin proteins, that are located at the inner nuclear membrane (INM) and outer nuclear membrane (ONM), respectively. The respective proteins interact via their Sad1-UNC-84 and Klarsicht/Anc-1/Syne homology (KASH) domains in the lumen of the NE, creating a physical connection between the nuclear lamina (NL) and the cytoskeleton. The NL comprises of type-V intermediate filament proteins, the nuclear lamins. Lamins A/C are A-type nuclear lamins, whilst lamin B1/B2 are B-type lamins. Various other NE proteins interact with the LINC complex including integral INM protein emerin, TMEM43/LUMA and isoforms of four-and-a-half LIM domain protein 1 (FHL1). Mutations in the genes encoding these proteins have been associated with a range of diseases exhibiting different phenotypes. Abbreviations: Emery-Dreifuss muscular dystrophy (EDMD), cerebellar ataxia (SCAR8), dilated cardiomyopathy (DCM), limb-girdle muscular dystrophy (LGMD), familial partial lipodystrophy (FPLD), congenital muscular dystrophy (L-CMD), Charcot-Marie-Tooth disease (CMT), Hutchinson-Gilford progeria syndrome (HGPS), mandibuloacral dysplasia (MADA), atypical Werner syndrome (WS). Created with BioRender.com.
Though there is some understanding of the structure of LINC complex-associated proteins and how they interact with one another and neighbouring NE proteins [20], it is unclear how mutations in genes encoding them can simultaneously cause the same disease, and different diseases with entirely different phenotypes. Mutations in *LMNA*, *EMD*, *SYNE1/SYNE2* and *TMEM43* for example, cause EDMD [21–25], while mutations in *LMNA* affecting the ubiquitously expressed lamin A/C protein lead to more than 15 distinct tissue-specific diseases (Figure 1) [26]. Two hypotheses attempt to explain how one protein could cause entirely different diseases: the structural hypothesis and the gene expression hypothesis. The structural hypothesis proposes that a mutated lamina may result in the nucleus being unable to resist high mechanical strain within cells, particularly within tissues exposed to high mechanical strain [27]. The gene expression hypothesis suggests that *LMNA* mutations affect the regulation of gene expression [11,28]. Accumulating evidence suggests these two may not be mutually exclusive. Further complicating the situation, identical mutations in *LMNA* have been found to cause different conditions [29,30]. While there has been intensive investigation into laminopathies, it is still largely unknown why mutations in the same protein, and especially, why identical mutations can cause different diseases. This hugely complicates the development of therapies for *LMNA*-related diseases.

Whilst research attempting to provide insight into laminopathies has been thorough, studies surrounding the development of diseases associated with mutations affecting other LINC complex-associated proteins and their relation to human diseases is mostly dispersed across many articles and has not been recently synthesised. The aim of this study was to conduct a systematic review into published mutations of genes encoding NE proteins associated with the LINC complex: *LMNA*, *LMNB1*, *LMNB2*, *EMD*, *TMEM43*, *SUN1/SUN2* and *SYNE1/SYNE2*, and to analyse the list of mutations to determine whether any patterns exist between genetic sequence variants and clinical aspects of the disease.

## 2. Methods

### 2.1. Overview of Methods

This systematic review of mutations in NE proteins associated with the LINC complex was conducted following Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines [31]. This guidance was developed to ensure the transparent and complete reporting of systematic reviews and meta-analyses.

### 2.2. Eligibility Criteria

#### Inclusion Criteria

Articles were selected if they clearly described a human mutation in a gene encoding a NE protein associated with the LINC complex (*LMNA*, *LMNB1*, *LMNB2*, *EMD*, *TMEM43*, *SUN1/SUN2*, *SYNE1/SYNE2*) using the standard nomenclature for describing genetic sequence variants (i.e., describing the nucleotide base pair change and/or amino acid change). Cases where there were multiple mutations in the same gene were included. As identical mutations affecting certain proteins, notably lamin A, are known to cause entirely different diseases, multiple reports of the same mutation were included so that the disease phenotype in different cases could be recorded and compared. Both single and multiple case reports were included.

### 2.3. Exclusion Criteria

Articles were excluded if they did not report a mutation in a gene encoding a NE protein associated with the LINC complex, along with articles that mentioned relevant NE proteins in the title or abstract but failed to describe a mutation in the main body of the article. Cases where more than one mutation was reported in different genes were not included, as it is unclear which is the causative mutation. Asymptomatic carriers were excluded as they do not reflect the patient population that are affected by disease and do not fall within the scope of this study. Non-human or man-made mutations that were created in a laboratory setting were also excluded. It is known that the accumula-tion of pre-lamin A induces premature senescence within cells, and this is attributed to the development of the following premature aging disorders [32]: Hutchinson-Gilford progeria syndrome (HGPS), atypical Werner syndrome/atypical progeria, mandibuloacral dysplasia and restrictive dermopathy. These diseases were excluded as they are caused by the expression of truncated forms of pre-lamin A that remain farnesylated within cells due to specific mutations that are known to result in the deletion of the ZMPSTE24 protease site [33–35]. Additionally, for the same reasons, articles were also excluded if they only mentioned mutations in pre-lamin A. Adult onset autosomal dominant leukodystrophy (ADLD) was excluded as it is known that this disease is caused by a genomic deletion upstream of the lamin B1 gene causing overexpression of LMNB1. The causation of this disease is entirely different to other nuclear envelopathies as ADLD is caused by genomic copy number variants as opposed to point or frameshift mutations. Articles were not included if they were not published in English.

### 2.4. Information Sources

Only bibliographic databases that were deemed relevant to the systematic review topic were searched including Medline (EBSCO) (18th May 2022), CINAHPlus (EBSCO) (18th May 2022) and AMED (EBSCO) (18th May 2022). UniProt, neXtProt and The Human Protein Atlas were accessed as knowledge sources to aid with the analysis of the results yielded in this review.

### 2.5. Search Strategy

No limits or restrictions on date or time-period were applied to the search strategy, however searches were restricted to only include articles that were published in English. No published search filters were used. The database searching yielded a total of 2218 results. There were 875 duplications that were removed using Mendeley Reference Manager and manually, leaving a total of 1343 articles to be screened. The search strategy, including search terms and a full breakdown of search results, can be viewed in detail in Supplementary Materials S1; Tables S1–S3.

### 2.6. Selection Process

Two reviewers blind screened the articles gathered from the searches using the eligibility criteria outlined above. The web-based application Rayyan was used for this process [36]. Any conflicts were recorded and discussed further between the two reviewers until a consensus was reached.

### 2.7. Data Collection Process

One reviewer independently collected data from each report manually. Data was tabulated and recorded using Microsoft Excel. For each NE protein, mutations were recorded using HGVS cDNA nomenclature and protein nomenclature. In some instances, if this information was not available (i.e., cDNA nomenclature was stated, but protein nomenclature was not or vice-versa), this was manually determined where possible. The type of mutation (i.e., missense, nonsense, insertion, deletion, duplication, frameshift) was also noted using HGVS nomenclature. The exon, codon and protein structure where the mutation is located were recorded or determined based on other information. The number of reports associated with each mutation were recorded if this information was stated in the article (this was therefore recorded as “minimum number of reports of mutation” as this was not always available). Diseases reported as being associated with each mutation were documented and if clinical details were available, disease severity was ranked (either mild, moderate or severe). The raw data from the data collection process can be viewed in Supplementary Materials S2; Tables S4–S6.
2.8. Risk of Bias and Certainty Assessment

Articles were rejected in the screening process if the reviewers determined that the information on the mutation was questionable, recorded incorrectly or unclearly. Examples include those where the mutation was not reported using standard nomenclature, or if the position of the mutation on the codon did not align with the cDNA position. Due to the nature of this systematic review, it was possible to manually check whether mutations were correctly recorded by looking at the mutation (the nucleotide or protein change) on the FASTA genetic sequence available on The National Center for Biotechnology Information database.

2.9. Limitations of the Study

Some mutations are reported directly to genetic databases and as a result are not published in journal articles. This may be a limitation to this study, as a number of variants may not have been recorded. There are, however, also limitations to database searching. Therefore, we believe that a systematic review was the most appropriate method to use to conduct this study.

3. Results

3.1. Study Selection

Bibliographic databases Medline ($n = 2048$), CINAHLPlus ($n = 165$) and AMED ($n = 5$) were searched, yielding a total of 2218 records. Before screening, duplicate records ($n = 875$) were removed. The remaining 1342 records were subjected to blind screening by two reviewers, based on the inclusion and exclusion criteria previously outlined. There were 878 records that were excluded during the first round of screening, this left 464 records that were sought for retrieval. In the final round of screening, 464 reports were assessed for eligibility. Records were excluded due to one of the following reasons: the article was in a foreign language ($n = 12$), multiple mutations in different proteins were described as causing the same disease in the report ($n = 6$), the mutation was not specified, was unrelated to the study or unclear ($n = 42$), the disease type was excluded as stated in the exclusion criteria ($n = 2$). As a result, 402 records were included in the review (Figure 2).

3.2. Nuclear Lamins

A high density of LMNA mutations at exons 1 and 6 cause striated muscle disease, whilst mutations causing metabolic disease increase in frequency from exon 7 onwards.

In total, 519 unique mutations in LMNA, the gene that encodes the alternatively spliced products, lamins A and C, were identified. Four mutations were found to be lamin C specific. Lamin C is derived from LMNA using an alternative splice site located in intron 10, therefore the C-terminus of lamin C differs to that of lamin A [37].

While mutations in LMNA have already been associated with more than 15 different phenotypes [26], this study revealed 37 conditions related to LMNA variants (Figure 3A). Moreover, identical mutations were found to give rise to different conditions. An astonishing 201 mutations were associated with more than one phenotype, and 99 mutations were linked to disorders affecting entirely different tissues and/or organ systems. Notorious examples of this include the variants p.Arg644Cys, p.Arg453Trp, p.Arg527Pro and p.Ser573Leu which were associated with 11, 6, 5 and 5 different diseases, respectively, each affecting multiple tissues (Table 1). One of the widest ranging examples among these is the p.Arg644Cys variant which has been linked to EDMD, LGMD, L-CMD, atrioventricular block (AVB), DCM, arrhythmogenic right ventricular cardiomyopathy, left ventricular noncompaction, insulin resistance, FPLD, and CMT [30,38–43].

Identification of studies via databases and registers

**Records identified from:**
- Databases
  - Medline (n=2048)
  - CINAHLPlus (n=165)
  - AMED (n=5)

**Records removed before screening:**
- Duplicate records removed (n=875)
- Records marked as ineligible by automation tools* (n = 0)
- Records removed for other reasons (n = 0)
*Not used

**Records screened:**
(n=1342)

**Reports sought for retrieval:**
(n=464)

**Reports assessed for eligibility:**
(n = 464)

**Records excluded:**
(n=878)

**Reports excluded:**
- Foreign language (n = 12)
- Multiple mutations (n = 6)
- Mutation not specified/unclear/unrelated (n = 42)
- Disease type excluded (n = 2)

**Studies included in review**
(n = 402)
**Reports of excluded studies**
(n = 62)

Figure 2. PRISMA flow diagram outlining the search and selection process of identifying records for the systematic review. The databases that were searched were Medline (n = 2048), CINAHPlus (n = 165) and AMED (n = 5), yielding a total of 2218 records. Duplicate records (n = 875) were removed prior to screening, then 1254 records were subjected to blind screening by two independent reviewers. 878 reports were excluded, and 464 records were retrieved. This left a total of 464 reports that were assessed for eligibility. There were 402 studies that were included in the review, whilst 62 were excluded. Reports were excluded for the following reasons: foreign language (n = 12), multiple mutations (n = 6), mutations were not specified, unclear or unrelated (n = 42) or contained a disease type that was listed within the exclusion criteria (n = 2). Created with BioRender.com.

A

Phenotypes Associated With LMNA Mutations

| Cardiac | Skeletal Muscle | Metabolic | Nervous System | Other |

B

Coding vs Intronic LMNA Mutations

- 91.91% 477 Coding
- 8.09% 42 Intronic

Total=519

C

Coding Region LMNA Mutations

- 71.70% 342 Missense
- 12.37% 59 Frameshift
- 6.29% 30 Nonsense
- 1.26% 6 Insertion
- 5.87% 28 Deletion
- 1.47% 7 Deletion/Insertion
- 1.05% 5 Duplication

Total=477

D

Phenotype Frequency in LMNA Exons

| Cardiac | Skeletal Muscle | Nervous System | Metabolic | Other |

Figure 3. LMNA mutations. (A) Phenotypes associated with LMNA mutations. There were 37 different conditions that were found to be associated with LMNA mutations, some which were caused by identical LMNA variants. These can be broadly classified into diseases affecting cardiac muscle, skeletal muscle, the metabolic system and the nervous system. A few variants caused diseases that fell into other categories. (B) Coding vs. intronic mutations. A total of 519 LMNA mutations were identified.

477 (91.91%) were coding mutations, whilst 42 (8.09%) were splice-site variants. (C) Coding region mutations. Missense mutations were the most frequently occurring type of LMNA mutations (342, 71.70%). Frameshift (59, 12.37%), nonsense (30, 6.29%), deletion (28, 5.87%), deletion/insertion (7, 1.47%), insertion (6, 1.26%) and duplication (5, 1.05%) mutations were also identified. (D) Mutations associated with different disease types and their frequency in LMNA exons. The most commonly occurring disease types arising from LMNA mutations were diseases affecting skeletal (262) and cardiac (260) muscle. These disease types clustered at exons 1 and 6. A total of 55 mutations were associated with metabolic disease, and these mostly appeared to increase in frequency from exon 7 onwards. Only 16 mutations caused nervous system disease and four caused other conditions which fell into different disease categories. Created with GraphPad Prism.

Table 1. Examples of LMNA mutations known to cause distinct phenotypes. The p.Arg644Cys variant was associated with 11 phenotypes affecting skeletal and cardiac muscle, metabolism, and the nervous system. Whilst the p.Arg453Trp, p.Arg527Pro and p.Ser537Leu mutations each caused diseases associated with three different tissue or organ systems. Abbreviations: Emery-Dreifuss muscular dystrophy (EDMD), limb-girdle muscular dystrophy (LGMD), congenital muscular dystrophy (L-CMD), atrioventricular block (AVB), dilated cardiomyopathy (DCM), arrhythmogenic right ventricular cardiomyopathy (ARVC), left ventricular noncompaction (LVNC), insulin resistance (IR), familial partial lipodystrophy (FPLD), Charcot-Marie-Tooth disease (CMT), congenital fibre type disproportion myopathy (CFTD).

| Protein Nomenclature | c.DNA Nomenclature | Associated Diseases                                                                 | Min. Number of Reports of Variant | References           |
|----------------------|--------------------|------------------------------------------------------------------------------------|-----------------------------------|----------------------|
|                      |                    | Skeletal Muscle                                                                    |                                   |                      |
|                      |                    | Cardiac                                                                            |                                   |                      |
|                      |                    | Metabolic                                                                          |                                   |                      |
|                      |                    | Nervous System                                                                     |                                   |                      |
| p.Arg644Cys          | 1930C>T            | EDMD, muscular dystrophy, LGMD, L-CMD                                              | AVB, DCM, ARVC, LVNC              | IR, FPLD             | CMT                     | 23                  | [24, 30, 39–47]     |
| p.Arg453Trp          | 1357C>T            | EDMD, LGMD, L-CMD, CFTD                                                            | DCM                               | FPLD                 |                          | 37                  | [44, 48–69]         |
| p.Arg527Pro          | 1580G>C            | EDMD, LGMD, L-CMD                                                                  | DCM                               | FPLD                 |                          | 21                  | [22, 44, 46, 56,    |
|                                                                                  |                                                                                              |                                                                 |                          | 58, 60, 65, 66,     |
|                                                                                  |                                                                                              |                                                                 |                          | 68–71]              |
| p.Ser573Leu          | 1718C>T            | LGMD                                                                               | DCM, AVB                          | Diabetes, FPLD       |                          | 14                  | [44, 45, 72–74]     |

There were 477 mutations located in the coding region of the LMNA gene (91.92%), and 42 intronic splice-site mutations (8.09%) (Figure 3B). The most frequently occurring type of mutations were missense mutations (n = 342), accounting for 71.70% of the mutations found in the coding region of LMNA. Frameshift (n = 59, 12.37%), nonsense (n = 30, 6.29%), deletion (28, 5.87%), deletion/insertion (n = 7, 1.47%), insertion (n = 6, 1.26%) and duplication (n = 5, 1.05%) mutations were also identified (Figure 3C). Arginine residues were found to be mutated more often than other amino acids, with 24.32% (n = 116) of coding region mutations being the result of arginine mutations. Mutations were distributed across all 12 exons of the LMNA gene. The most mutations were found to be located on exon 1 (n = 105, 22.01%) and exon 6 (n = 75, 15.72%). Very few mutations (n = 8) were found within exons 12 and 10. Only one (0.22%) mutation was found to be situated in exon 12, and seven mutations were found in exon 10 (1.51%).

Given that LMNA mutations cause a broad range of diseases, analysis was next performed to determine whether genotype influenced disease phenotype. Using MeSH

terms, each laminopathy identified in this study was classified into one of the following disease categories; skeletal muscle disease, cardiac disease, metabolic disease, nervous system disease or other. Each individual LMNA mutation was then assigned a disease category, based on the disease(s) that it was found to cause. If a mutation caused more than one disease type, it was counted in more than one category (for example, if a mutation caused DCM and EDMD, it would be categorised as causing skeletal muscle disease and cardiac disease). Disease phenotype was then examined in relation to the position of the causative mutation on the LMNA gene, and consequently the affected protein structures and domains.

The most frequently occurring disease categories associated with LMNA mutations were skeletal muscle diseases (*n* = 262 mutations) and cardiac diseases (*n* = 260 mutations). These two disease types seemed to occur most often in exons 1 and 6 (Figure 3D). It was found that 67 mutations causing skeletal muscle disease were in exon 1, and 47 were found to be within exon 6, whilst 48 cardiac disease-associated mutations were located in exon 1, and exon 6 also harboured 48 variants (Figures 3D and 4). Interestingly, in cases where a single mutation caused multiple disease types, most commonly cardiac and skeletal muscle diseases were found to be caused by the same mutation. It is important to also consider that skeletal muscle diseases including EDMD, L-CMD and LGMD do also often cause cardiac abnormalities, therefore there may be some crossover between skeletal and cardiac disease phenotypes. Of 99 mutations known to cause diseases affecting multiple tissues or organ systems, 63 (64.29%) were found to cause both cardiac and skeletal muscle disease, whilst 11 (11.22%) mutations caused cardiac and skeletal muscle disease along with another type of disease(s). There were 55 mutations associated with diseases of the metabolic system. These mutations were distributed throughout the 12 exons of the LMNA gene (Figures 3D and 4). The frequency of metabolic disease mutations, however, generally appears to increase from exon 7 onwards (besides the mutations found in exon 1). Exon 7 through to 12 encode the tail region of lamin A (Figure 4). Only 16 mutations were found to cause nervous system disease, whilst 4 mutations caused other conditions which fell into other disease categories.

### 3.3. Fewer Mutations Are Reported in B-Type Lamins Compared to Lamin A/C

Despite their relatively similar size, very few mutations were found in B-type lamins compared to lamins A and C. There were 13 mutations associated with LMNB1, all causing neural tube defects (NTDs) including microcephaly (*n* = 5), spina bifida (*n* = 7) and anencephaly (*n* = 1). Only 4 mutations were identified in LMNB2, and these were associated with acquired partial lipodystrophy.

### 3.4. Inner Nuclear Membrane

#### 3.4.1. EMD Is Implicated in Diseases Other Than Emery-Dreifuss Muscular Dystrophy

There were 83 mutations found in *EMD*, the gene encoding emerin. We found that 92.86% (*n* = 78) of *EMD* mutations were attributed to EDMD. This study also found that mutations in emerin have been linked to the development of DCM, cardiac conduction disease (CCD), atrial fibrillation (AF), LGMD and rigid spine syndrome (RSS). It is important to note that DCM, CCD and AF are often manifestations of EDMD, although in these cases they have been recorded as isolated diseases, not symptoms of EDMD. Only eleven *EMD* mutations were splice site variants, the remaining 72 (86.25%) mutations were found in the coding region (Figure 5A). Within the coding region of the gene, many mutations were frameshift (*n* = 32, 44.44%), deletion (*n* = 9, 12.50%) and nonsense (*n* = 15, 20.83%) variants. A notable proportion of missense mutations (*n* = 14, 19.44%) were also identified (Figure 5B). *EMD* consists of 6 exons, and mutations were found throughout the emerin gene. However, 39.75% (*n* = 33) mutations were found in exon 6, suggesting that this region may be a mutation hot-spot (Figure 5C).

Head (1-30)

Central rod domain (31-381)

1A 1B 2

Codon:
Exon: 1 119 171 213 270 312 386

Key:
Skeletal
Cardiac
Metabolic
Nervous System
Other

Tail (382-664)

Ig-like CaaX

386 466 496 536 566 656 664

7 8 9 10 11 12

LMNA splice site

Figure 4. The structure of the LMNA gene and protein product lamin A. LMNA encodes lamin A/C. Lamin C is produced via a splice site located at intron 10 and therefore it's C-terminus differs to lamin A. LMNA consists of 12 exons. Each coding region LMNA mutation was assigned to a disease category based on the disease type it caused. If it caused more than one type of disease, it was considered in multiple categories. These mutations were then marked below the LMNA gene at the appropriate codon, in a colour that corresponds to its related disease type(s). Above the LMNA gene is the structure of lamin A, consisting of a head, central rod and tail domain. Created with BioRender.com.

3.4.2. Mutations in Inner Nuclear Membrane Protein, TMEM43, and Integral LINC Complex Components SUN1/SUN2, Are Also Linked to Emery-Dreifuss Muscular Dystrophy

In this study, mutations in the INM protein TMEM43 were also found to be linked to the development of EDMD. Only two mutations in TMEM43 were identified in the literature, however the evidence linking the mutations to EDMD is weak. No other diseases were found to be associated with mutations in this gene. Although we found no mutations in SUN1/SUN2 that were directly found to cause disease, SUN1/SUN2 variants have been identified as disease severity modifiers in EDMD when mutations in LMNA or EMD are present [75].

A
Coding vs Intronic EMD Mutations

- 86.75% 72 Coding
- 13.25% 11 Intronic

Total=83

B
Coding Region EMD Mutations

- 44.44% 32 Frameshift
- 20.83% 15 Nonsense
- 19.44% 14 Missense
- 12.50% 9 Deletion
- 2.78% 2 Insertion

Total=72

C
Mutation Frequency in EMD Exons

Figure 5. EMD mutations. (A) Coding vs. intronic mutations. A total of 83 EMD mutations were identified. 73 (86.75%) were coding region mutations and eleven (13.25%) were intronic variants. (B) Coding region mutations. The largest percentage of coding region mutations were frameshift (32, 44.44%), deletion (9, 12.50%) and nonsense (15, 20.83%) variants. Missense mutations (14, 19.44%) also accounted for a considerable proportion of the mutations identified. A couple of insertion (2, 2.78%) mutations were also identified. (C) Mutation frequency in EMD exons. Mutations most frequently occurred within exon 6, accounting for 45.21% (33) of total coding region mutations. Exon 2 contained the second highest proportion of mutations (17, 23.29%), followed by exons 3 (7, 9.59%) and 4 (6, 8.22%). Exon 1 and 5 contained the fewest EMD variants (5, 6.85% and 4, 5.48%, respectively). Created with GraphPad Prism.

3.5. Outer Nuclear Membrane

Most often, nesprin-1 nonsense mutations cause nervous system disorders, particularly spinocerebellar ataxia 8 (SCAR8). In total, 141 mutations in SYNE1 were identified. Out of these, four mutations were found to specifically affect the expression of short isoform nesprin-1α. Only three mutations were identified in SYNE2. Most of the SYNE1 mutations were coding region mutations ($n = 127$, 90.07%), with only 14 intronic mutations being identified (9.93%) (Figure 6A). 48.82% ($n = 62$) of mutations in the coding region of the gene were nonsense (Figure 6B). Additionally, missense ($n = 32$, 25.20%), frameshift ($n = 30$, 23.62%) and deletion ($n = 3$, 2.36%) mutations were identified (Figure 6B). No duplication or insertion mutations were discovered. These mutations were found to be distributed mostly throughout the region of the gene that encodes the spectrin repeats (SRs), with seven mutations being found in the N-terminal calponin homology (CH) domain and three in the KASH domain.

A

Coding vs Intrinsic SYNE1 Mutations

- 90.07% 127 Coding
- 9.93% 14 Intrinsic

Total=141

B

Coding Region SYNE1 Mutations

- 25.20% 32 Missense
- 48.82% 62 Nonsense
- 23.62% 30 Frameshift
- 2.36% 3 Deletion

Total=127

C

Phenotypes associated with SYNE1 mutations

- Nervous System
- Skeletal Muscle
- Cardiac

Number of Mutations

Spino-cerebellar ataxia  
Amyotrophic lateral sclerosis  
Emery-Dreifuss muscular dystrophy  
Dilated cardiomyopathy  
Myopathic arthrogryposis  
Muscle weakness  

Figure 6. SYNE1 mutations. (A) Coding vs. intronic mutations. There were 127 (90.07%) mutations identified in the coding region of SYNE1 and 14 (9.93%) were intronic. (B) Coding region mutations. Almost half (48.82%, 62) of coding region mutations were nonsense. Many mutations were also missense (32, 25.20%) and frameshift (30, 23.62%) variants. Three (2.36%) deletion mutations were also identified. (C) Diseases associated with SYNE1 variants. The most common disease resulting from SYNE1 variants was autosomal recessive spinocerebellar ataxia (SCAR8). 122 (85.82%) mutations were related to this disease. Amyotrophic lateral sclerosis was also found to be frequently caused by SYNE1 mutations (19, 13.48%). Seven cases of Emery-Dreifuss muscular dystrophy (EDMD) were related to SYNE1 variants, and a handful of mutations were associated with myopathic arthrogryposis (2), dilated cardiomyopathy (DCM) (5) and general muscle weakness (1). Created with GraphPad Prism.

Mutations in SYNE1 predominantly cause diseases of the nervous system. Of the mutations identified, 122 were associated with SCAR8, whilst 19 mutations were found to cause ALS (Figure 6C). Fewer mutations were related to the development of EDMD ($n = 7$), myopathic arthrogryposis ($n = 1$), generalized muscle weakness ($n = 1$), and DCM ($n = 5$) (Figure 6C). Two of the SYNE2 mutations caused EDMD, whilst a single mutation was found to cause ALS.

4. Discussion

In this study we systematically reviewed and summarised known mutations in genes encoding LINC complex-associated proteins that are related to human disease, providing a useful resource for researchers in this field. We have demonstrated that there appear to be no obvious correlations between the position of lamin A/C mutations on the LMNA gene and different disease phenotypes. However, we found a high frequency of striated muscle disease-causing LMNA variants located within exons 1 and 6, whilst mutations associated with metabolic diseases often occurred in exon 7 onwards. A potential emerin mutation "hot-spot" was also identified within exon 6 of EMD. We have also observed that most often SYNE1 nonsense variants lead to the development of nervous system disease.

LMNA variants were associated with 35 different conditions, and in some cases, identical mutations caused diseases with entirely different phenotypes. A notorious example of
this is the mutation p.Arg644Cys, which causes 11 different disease phenotypes affecting skeletal and cardiac muscle \[38–42\], the metabolic system \[30\], and the nervous system \[43\]. It remains unclear how this could be, and no answers have arisen from studying the position of known mutations on the *LMNA* gene in relation to different disease phenotypes. This study has, however, revealed that there are no instances in which an identical mutation in the other nuclear envelope genes studied here is responsible for causing so many distinct diseases. One explanation for this could be that other genetic disease modifiers are present in addition to *LMNA* mutations, which may affect the phenotypic outcome of the disease-causing variant. Expressivity modifiers which modulate disease severity have already been identified in conjunction with *LMNA* mutations. Whilst our searches found no mutations in the genes encoding integral LINC complex proteins SUN1/SUN2 were found to directly cause human disease, the presence of SUN1/SUN2 variants alongside other Emery-Dreifuss muscular dystrophy EDMD-related genes, including *LMNA*, cause a more severe EDMD phenotype \[75\]. The synergistic effects of *LMNA* and *DES* (encoding desmin) or *EMD* mutations have also been found to alter EDMD disease severity \[76–78\]. Perhaps the presence of different modifier genes in combination with *LMNA* mutations could affect lamin A/C expression in variable tissue types leading to the presentation of different diseases. Modifier effects may also influence pleiotropy, causing individuals that share the same target allele to show a range of phenotypes \[79,80\], though such variation in disease phenotypes resulting from one gene, such as *LMNA*, has never been attributed to modifiers before. *MKS1*, the gene encoding the protein associated with Meckel syndrome type 1, has been identified as a pleiotropic modifier gene in Bardet-Biedl syndrome (BBS). BBS-patients harbouring mutations in both *MKS* and *BBS* (BBS protein) were found to suffer with seizures, which are not typically associated with either *MKS* or *BBS*. Hence, the combination of the two mutations generated a completely novel phenotype \[81\]. Individual modifier genes with pleiotropic effects have also been found in various cystic fibrosis-related phenotypes \[82\].

It is also important to mention that premature aging syndromes including Hutchinson-Gilford progeria syndrome (HGPS), atypical Werner syndrome/atypical progeria, mandibuloacral dysplasia and restrictive dermopathy are also associated with *LMNA* mutations. These diseases are believed to be caused by specific mutations that are known to result in the deletion of the ZMPSTE24 protease site, leading to the accumulation of truncated forms of pre-lamin A that remain farnesylated within cells \[33–35\]. As genotype-phenotype studies have already successfully elucidated that the mutations leading to these diseases affect the ZMPSTE24 site, we did not include this type of diseases within this review. By identifying that *LMNA* mutations related to progeria lead to the deletion of this protease site, it has allowed the advances in our understanding of the development of the disease, which has in turn lead to the identification of potential therapeutic candidates \[83\].

Interestingly, few mutations were identified in B-type lamins, lamin B1 and B2, compared to lamin A/C. This may be because a loss of B-type lamins is incompatible with postnatal life. It has been argued that due to their vital roles in neurodevelopment, B-type lamins are indispensable in mammalian cells. Lamin B1 and B2 deficiency has been found to cause elongated nuclei in neurons and to be essential for neuronal migration in the developing brain \[84–86\]. Moreover, mice harbouring a mutation in lamin B1 did not survive long after birth due to severe developmental abnormalities \[87\], and lamin B2-deficient mice exhibited brain deformities and died at birth \[84\]. Another explanation for a lack of mutations in *LMNB1/LMNB2* may be that the expression of A-type lamins compensate when B-type lamins are not present. Some evidence from studying neurodevelopment in mouse models supports this theory, suggesting that low levels or absence of lamin A/C may exacerbate the effects of lamin B deficiency. For example, the absence of *Lmna* in neurons of lamin B1-deficient mouse embryos is associated with abnormal nuclear morphology and severe neuropathology \[85\]. An alternative hypothesis may be that the loss of one of either lamin B1 or B2 is compensated by the other. A study using *Lmnb1*<sup>B2/B2</sup> and *Lmnb2*<sup>B1/B1</sup> mice (mice that make lamin B2 from the *Lmnb1* locus and vice-versa), however, found that
increased production of one B-type lamin doesn't appear to fully compensate the other, as both mouse models exhibited neuronal abnormalities [88]. Duplications of the *LMNB1* gene have additionally been found to cause autosomal dominant leukodystrophy, although we decided to exclude this due to these mutations being copy number variations [89].

In contrast to the spectrum of diseases caused by *LMNA* mutations, we observed that *EMD* mutations cause multiple diseases, with overlapping clinical phenotypes. Mutations in *EMD* have long been associated with causing EDMD but are also attributed to DCM, AF, CCD and RSS. Whilst these have presented as isolated diseases in certain cases, they are also known symptoms of EDMD [90, 91]. A possible explanation as to why *LMNA* mutations cause conditions with very distinct phenotypes, whilst other NE proteins do not, could be due to the highly complex and varied roles of lamin A/C. Besides having structural functions, such as maintaining the integrity of the NE [92, 93], lamin A/C has also been implicated in DNA replication, chromatin organisation, cell differentiation, and is known to interact with several transcription factors [94–96]. The consequences of mutations on any or all these processes could have a range of effects. Additionally, lamin A/C is known to interact with a number of different proteins that may be expressed in a tissue-specific manner themselves [97, 98]. Mutations in *LMNA* may therefore lead to differential interactions of mutant lamins and their associated proteins [99]. Commonly laminopathies are caused by missense mutations, and arginine residues were found to be frequently substituted. Approximately 30% of genetic diseases are due to mutations at arginine and glycine residues [100]. This is a result of arginine being high mutable because of the deamination of 5′-CpG dinucleotides in arginine codons [101], the relatively high frequency of arginine in human proteins, and that arginine mutates to other amino acids with very different chemical properties [100]. Variants affecting arginine residues may have devastating effects on protein stability, as arginine is frequently involved in the formation of salt bridges, and may disrupt and interfere with protein interactions and post-translational modification sites as arginine is also common in protein-active or binding sites [102].

Clusters of mutations corresponding to different laminopathies were also found at different exons of the *LMNA* gene. Many mutations causing skeletal myopathies or cardiac diseases occurred within exons 1 and 6. The N-terminal head of lamin A/C is encoded within a region of exon 1, whilst exon 6 corresponds to coil 2 within the central rod domain. These structures are both involved in lamin assembly. Although lamin assembly mechanisms are still not fully understood, lamin dimers are believed to longitudinally associate “head-to-tail” [11, 103], whereby the N- and C-terminals of lamin interact. Evidence has also shown that the central rod domains have a high propensity to form “coiled-coil” dimers [104]. Consequently, mutations that affect these structures could impede these processes. Distortions in lamina structure and assembly have been previously observed in cell lines transfected with *LMNA* mutations associated with DCM [105]. Incorrect lamin assembly may then lead to a compromised NL, as previously observed in cells harbouring *LMNA* mutations [106–108]. As per the “structural hypothesis” theory that attempts to explain the tissue-specificity of laminopathies, a fragile NL leads to structural weakness resulting in the nucleus being unable to withstand high mechanical stress within cells [109]. This is particularly pertinent within striated muscle, which is exposed to high mechanical tension.

*LMNA* mutations related to metabolic diseases also appeared to generally increase in frequency from exon 7 onwards, the area which encodes the C-terminal tail region of lamin A. It is understood that the tail domain contains the binding sites for most lamin-binding proteins. Lamin A interacts with and tethers emerin to the NE via a binding site in this region [110–112]. This interaction could be altered by mutations, displacing emerin from the NE and as a result, affecting emerin function. Mutations in the tail region of lamin A have previously been shown to affect emerin binding in vivo, however these variants were related to cardiomyopathy and EDMD [113]. Emerin may be involved in metabolic processes [114, 115], although the evidence is limited. Lamin A is also thought to regulate 12-lipogenase (12-LOX) through an interaction at a binding site located in the lamin A tail domain [116]. Mounting evidence has shown that 12-LOX contributes to
the progression of diabetes, and therapeutic interventions to limit pro-inflammatory 12-LOX metabolites may positively impact on outcomes for diabetes co-morbidities \[117–120\]. LMNA variants that affect this binding region may result in a dysregulation of 12-LOX which may contribute to diabetes pathology. Other notable binding sites in this region include actin and DNA \[95,121,122\], but it is unclear how these interactions may be linked to metabolic disease pathology.

As well as lamin A/C mutation clusters being identified within specific exons of LMNA, emerin mutations appeared to most often occur within exon 6 of EMD suggesting this region of the gene may be a mutation “hot-spot”. It is important to note, however, that exon 6 is the largest exon of the EMD gene consisting of 104 amino acids, while exons 4 and 2 are 44 and 35 amino acids in length, and the other exons range from 16–27 amino acids long. While the higher occurrence of mutations in exon 6 may therefore be explained in part because of its larger size, it may not fully account for the differences since exon 4 is 42% of the size of exon 6, but was found to have 6 mutations described, compared to 33 for exon 6. As the majority of these variants are frameshift, deletion and nonsense mutations, they will most likely result in the loss of, or the production of a non-functional truncated form of emerin \[25,123,124\]. When diagnosing patients with EDMD in resource-restricted settings, it may be useful to consider first sequencing exon 6, given the large number of emerin mutations located in this region. It is important to bear in mind, though, that many EMD mutations were located in other exons of the gene.

Most disease-causing SYNE1 variants result in the production of a truncated form of nesprin-1, because of nonsense mutations. Mutations impairing nesprin-1 function may affect its affinity to interact with both SUN1/SUN2 at the PNS, and with the cytoskeleton in the cytoplasm \[12,125\]. This would result in the dissolution of the LINC complex, causing the connection between the NL and cytoskeleton to become weakened. SYNE1 mutations commonly cause SCAR8, suggesting that nesprin-1 is indispensable in the brain. Nesprin-1/2 have been found to be crucial for proper neurodevelopment, and the LINC complex has been found to be the main mediator coupling the nucleus to the cytoskeleton in migrating postmitotic neurons \[126\]. Furthermore, the LINC complex has been associated with fundamental aspects of embryonic development, particularly radial migration \[126\]. SYNE1 variants may hinder these processes resulting in devastating effects. Interestingly SYNE2 has only been associated with very few cases of human disease, suggesting that maybe nesprin-1 expression might compensate when there is a lack of nesprin-2.

The finding here that almost 50% of SYNE1 coding region mutations are nonsense suggests that stop-codon readthrough (also known as suppression therapy) should be considered as a treatment option for nesprin-1 related disorders. Suppression therapy utilises pharmacological compounds, which are often aminoglycoside antibiotics, that suppress translation termination at in-frame premature termination codons to restore the translation of a full-length, functional peptide \[127–133\]. A number of clinical trials have been conducted where aminoglycoside antibiotics have been administered to patients carrying nonsense mutations with diseases including cystic fibrosis \[127,128,134\], hemophilia \[131\] and Hailey-Hailey disease \[135\] showing varying effects including the partial restoration of full-length, functional protein. Moreover, a nonaminoglycoside known as ataluren (previously PTC124) was the first stop-codon readthrough therapy to gain global approval in 2014 for treatment of nonsense mutation Duchenne muscular dystrophy \[136,137\]. Stop-codon readthrough may also be applicable for other NE proteins whereby a significant proportion of causative mutations are nonsense.

In this study, we focused on examining how the individual proteins associated with the LINC complex contribute to human disease, but the LINC complex as a whole may also be involved in disease pathophysiology. The LINC complex is involved in rearward nuclear movement and contributes to centrosome orientation. This is facilitated by SUN2 and nesprin-2 assembling into transmembrane-associated nuclear (TAN) lines that link actin cables to the nucleus at the INM through anchorage by emerin and A-type lamins \[138\]. Depletion of SUN2, nesprin-2, or lamin A/C in C2C12 cells hindered nuclear movement,and in particular, reduction of nesprin-2 also interfered with cell migration and myoblast fusion into myotubes \[139\]. This suggests that the LINC complex is important for proper myogenic differentiation. The LINC complex is also involved in mechanotransduction, the process by which mechanical stimuli is translated into biochemical signals, allowing cells to adjust their structure and function to the physical environment around them, by allowing force transmission across the NE \[140,141\]. When exploring nuclear lamin expression in response to external stimuli, it was found that the expression of lamin A/C, relative to B-type lamins, have been found to scale with substrate stiffness both in-vivo and in-vitro \[142\]. Furthermore, interference of the LINC complex is known to impair intracellular force transmission from the cytoskeleton to the nucleus \[143\]. When taken together, these findings imply that mutations in any of the LINC complex proteins may also disrupt the LINC complex and its associated functions, which could be related to disease development. From in-vitro studies of cells derived from patients, we also know that mutations in certain LINC complex proteins have implications on others, due to their involvement as components of a protein structure. For example, emerin has been found to be mislocalized in fibroblasts from LGMD patients harbouring LMNA mutations, therefore it is important to consider the effect of mutations in LINC complex components on the LINC complex itself \[109\]. The LINC complex has also been implicated in other disease-related processes, and whilst this was not within the scope of this study, considering the role of these proteins in other biological conditions does allow us to understand more about their function \[144\]. Mutations in SYNE1, for example, have been implicated in human glioblastoma progression and survival \[145\], and SYNE1 has also been identified as a potential driver gene in cervical cancer development \[146\]. Disruption of the LINC complex in glandular epithelial cells was also found to cause the development of aberrant glandular acini, suggesting the LINC complex is important in maintaining tissue architecture, which could be related to cancer pathophysiology \[147\]. In addition, imbalanced nucleoskeletal connections have been found to create cell polarity defects in HGPS as well as normal physiological aging. As mentioned previously, HGPS is caused by an accumulation of truncated pre-lamin A also known as “progerin” \[148\]. Progerin affects the mobility of LINC complex-associated proteins, SUN2, nesprin-2 and emerin, which in turn causes defective nuclear positioning and cell polarity which is essential for migration \[149\].

It is understood that many mutations are reported directly to genetic databases and as a result are not published in journal articles. This may be a limitation to this study and as a result a number of mutations may not be recorded and therefore missing in the final analysis. However, there are also limitations to database searching. We found that it was not possible to easily filter mutations by disease or phenotype, therefore it was more appropriate to conduct a systematic review into the literature where we could find clinical details within journal articles. It is also important to consider that some patients with mutations in genes encoding NE proteins displayed “like-phenotype” symptoms. These may not necessarily be strictly classified as diseases but may be diseases mimicking the phenotypes of other diseases. This is particularly relevant to the many diseases associated with LMNA variants. It is necessary to include these phenotypes within this study as it is interesting and valuable to report that these phenotypes are linked to mutations in these genes, even if these cases don’t allow for a full clinical classification. This could be considered a limitation of the literature, and what is reported within the literature. Another constraint is that some variants, such as the LMNA p.Arg644Cys mutation, are controversial in that they have been reported as both pathogenic and benign \[41,150\]. We tried to mitigate against this by ranking the pathogenicity likelihood of variants, but it is possible that some mutations may not be pathogenic. Considering the potential for differences between information contained within genetic databases compared with the literature, it would be useful in future to undertake a systematic comparison of the two information sources. This would allow for a more robust method of identifying discrepancies between the pathogenicity ranking of mutations, and whether results of phenotype-genotype comparison studies would be altered if some variants are re-classified as benign.
5. Conclusions

Overall, many unanswered questions remain surrounding LMNA mutations, and why faults with this one protein can cause a spectrum of different diseases, especially since this phenomenon is not seen with in other NE proteins. Despite these uncertainties, other observations were made which offer some insight into the potential downstream consequences of lamin A/C mutations. Striated muscle diseases appear to be most frequent in exons 1 and 6 of LMNA, which correspond to areas of the protein involved in lamin-lamin interactions. Subsequently, mutations located in these exons may prevent correct lamin assembly. Variants associated with metabolic diseases generally increase in frequency within the area of LMNA which encodes the tail domain of lamin A/C. This region facilitates interactions between numerous proteins and lamin A/C. These interactions could be disrupted by mutations that affect these binding sites. Additionally, exon 6 of EDM may be an emerin mutation “hot-spot”, which is perhaps useful to know when diagnosing EDMD, while nonsense SYNE1 mutations are frequently the cause of SCAR8, suggesting a compromised LINC complex particularly disrupts processes in the brain. Stop-codon read-through therapy may be a beneficial treatment option for SCAR8 and more research may be needed in this area. Overall, these results offer insight into the varied roles of LINC-complex associated proteins in human disease and provide direction for future gene-targeted therapy development. Moreover, identifying conditions affected by mutations in different NE proteins may allow for sharing of therapies or therapeutic development strategies.

Supplementary Materials: The following supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/cells11244065/s1](https://www.mdpi.com/article/10.3390/cells11244065/s1), Supplementary Materials S1 (Excel document containing Tables S1–S3) and Supplementary Materials S2 (Excel document containing Tables S4–S6).

Author Contributions: H.R.F. conceived of the study and designed the study with E.C.S. All authors contributed to the data analysis and interpretation and wrote and approved the final manuscript. All authors have read and agreed to the published version of the manuscript.

Funding: This work was supported by funding from the Keele ACORN scheme, Keele University, UK (HRF and ECS) and the Orthopaedic Institute Ltd., RJAH Orthopaedic Hospital, Oswestry, UK (RPG182) (HRF). ECS is aligned to the EPSRC/MRC Doctoral Training Centre in Regenerative Medicine (Loughborough, Keele and Nottingham Universities).

Institutional Review Board Statement: Not applicable.

Informed Consent Statement: Not applicable.

Acknowledgments: The authors are grateful to Ian Holt, RJAH, for reviewing the manuscript and providing helpful feedback.

Conflicts of Interest: The authors declare no conflict of interest.

## References

1. Nagano, A.; Arahata, K. Nuclear Envelope Proteins and Associated Diseases. *Curr. Opin. Neurol.* **2000**, *13*, 533–539. [CrossRef] [PubMed]
2. Chi, Y.H.; Chen, Z.J.; Jeang, K.T. The Nuclear Envelopathies and Human Diseases. *J. Biomed. Sci.* **2009**, *16*, 96. [CrossRef] [PubMed]
3. Somech, R.; Shaklai, S.; Amariglio, N.; Rechavi, G.; Simon, A.J. Nuclear Envelopathies—Raising the Nuclear Veil. *Pediatr. Res.* **2005**, *57*, 8–15. [CrossRef] [PubMed]
4. Worman, H.J.; Bonne, G. “Laminopathies”: A Wide Spectrum of Human Diseases. *Exp. Cell Res.* **2007**, *313*, 2121–2133. [CrossRef] [PubMed]
5. Song, S.; Zhang, Y.; Zhong, N. Laminopathies—One Gene, Multiple Diseases. *Beijing Da Xue Xue Bao.* **2005**, *37*, 96–99. [PubMed]
6. Gerace, L.; Burke, B. Functional Organization of the Nuclear Envelope. *Annu. Rev. Cell Biol.* **2003**, *4*, 335–374. [CrossRef]
7. Watson, M.L. The Nuclear Envelope; Its Structure and Relation to Cytoplasmic Membranes. *J. Biophys. Biochem. Cytol.* **1955**, *1*, 257–270. [CrossRef]
8. Linde, N.; Stick, R. Intranuclear Membranes Induced by Lipidated Proteins Are Derived from the Nuclear Envelope. *Nucleus* **2010**, *1*, 343–353. [CrossRef]

9. Güttinger, S.; Laurell, E.; Kutay, U. Orchestrating Nuclear Envelope Disassembly and Reassembly during Mitosis. *Nat. Rev. Mol. Cell Biol.* 2009, *10*, 178–191. [CrossRef]

10. Taddei, A.; Hediger, F.; Neumann, F.R.; Gasser, S.M. The Function of Nuclear Architecture: A Genetic Approach. *Annu. Rev. Genet.* 2004, *38*, 305–345. [CrossRef]

11. Stuurman, N.; Heins, S.; Aebi, U. Nuclear Lamins: Their Structure, Assembly, and Interactions. *J. Struct. Biol.* 1998, *122*, 42–66. [CrossRef] [PubMed]

12. Crisp, M.; Liu, Q.; Roux, K.; Rattner, J.B.; Shanahan, C.; Burke, B.; Stahl, P.D.; Hodzic, D. Coupling of the Nucleus and Cytoplasm: Role of the LINC Complex. *J. Cell Biol.* 2006, *172*, 41–53. [CrossRef] [PubMed]

13. Zhang, Q.; Skepper, J.N.; Yang, F.; Davies, J.D.; Hegyi, L.; Roberts, R.G.; Weissberg, P.L.; Ellis, J.A.; Shanahan, C.M. Nesprins: A Novel Family of Spectrin-Repeat-Containing Proteins That Localize to the Nuclear Membrane in Multiple Tissues. *J. Cell Sci.* 2001, *114*, 4485–4498. [CrossRef] [PubMed]

14. Cain, N.E.; Jahed, Z.; Schoenhofen, A.; Valdez, V.A.; Elkin, B.; Hao, H.; Harris, N.J.; Herrera, L.A.; Woolums, B.M.; Mofrad, M.R.K.; et al. Conserved SUN-KASH Interfaces Mediate LINC Complex-Dependent Nuclear Movement and Positioning. *Curr. Biol.* 2018, *28*, 3086–3097.e4. [CrossRef] [PubMed]

15. Tapley, E.C.; Starr, D.A. Connecting the Nucleus to the Cytoskeleton by SUN-KASH Bridges across the Nuclear Envelope. *Curr. Opin. Cell Biol.* 2013, *25*, 57–62. [CrossRef]

16. Zhou, Z.; Du, X.; Cai, Z.; Song, X.; Zhang, H.; Mizuno, T.; Suzuki, E.; Yee, M.R.; Berezov, A.; Murali, R.; et al. Structure of Sad1-UNC84 Homology (SUN) Domain Defines Features of Molecular Bridge in Nuclear Envelope. *J. Biol. Chem.* 2012, *287*, 5317–5326. [CrossRef]

17. Wilhelmsen, K.; Litjens, S.H.M.; Kuikman, I.; Tshimbalanga, N.; Janssen, H.; Van Bout, I.D.; Raymond, K.; Sonnenberg, A. Nesprin-3, a Novel Outer Nuclear Membrane Protein, Associates with the Cytoskeletal Linker Protein Plectin. *J. Cell Biol.* 2005, *171*, 799. [CrossRef] [PubMed]

18. Zhang, Q.; Ragnauth, C.; Greener, M.J.; Shanahan, C.M.; Roberts, R.G. The Nesprins Are Giant Actin-Binding Proteins, Orthologous to Drosophila Melanogaster Muscle Protein MSP-300. *Genomics* 2002, *80*, 473–481. [CrossRef]

19. Meinke, P.; Nguyen, T.D.; Wehnert, M.S. The LINC Complex and Human Disease. *Biochem. Soc. Trans.* 2011, *39*, 1693–1697. [CrossRef] [PubMed]

20. Storey, E.C.; Holt, I.; Morris, G.E.; Fuller, H.R. Muscle Cell Differentiation and Development Pathway Defects in Emery-Dreifuss Muscular Dystrophy. *Neuromuscul. Disord.* 2020, *30*, 443–456. [CrossRef]

21. Gueneau, L.; Bertrand, A.T.; Jais, J.-P.; Salih, M.A.; Stojkovic, T.; Wehnert, M.; Hoeltzenbein, M.; Spuler, S.; Saitoh, S.; Verschueren, A.; et al. Mutations of the FHL1 Gene Cause Emery-Dreifuss Muscular Dystrophy. *Am. J. Hum. Genet.* 2009, *85*, 338–353. [CrossRef] [PubMed]

22. Bonne, G.; Di Barletta, M.R.; Varnous, S.; Bécane, H.-M.; Hammouda, E.-H.; Merlini, L.; Muntoni, F.; Greenberg, C.R.; Gary, F.; Urtizberea, J.-A.; et al. Mutations in the Gene Encoding Lamin A/C Cause Autosomal Dominant Emery-Dreifuss Muscular Dystrophy. *Nat. Genet.* 1999, *21*, 285–288. [CrossRef] [PubMed]

23. Zhang, Q.; Bethmann, C.; Worth, N.F.; Davies, J.D.; Wasner, C.; Feuer, A.; Ragnauth, C.D.; Yi, Q.; Mellad, J.A.; Warren, D.T.; et al. Nesprin-1 and -2 Are Involved in the Pathogenesis of Emery Dreifuss Muscular Dystrophy and Are Critical for Nuclear Envelope Integrity. *Hum. Mol. Genet.* 2007, *16*, 2816–2833. [CrossRef] [PubMed]

24. Liang, W.-C.; Mitsuihashi, H.; Keduka, E.; Nonaka, I.; Noguchi, S.; Nishino, I.; Hayashi, Y.K. TMEM43 Mutations in Emery-Dreifuss Muscular Dystrophy-Related Myopathy. *Ann. Neurol.* 2011, *69*, 1005–1013. [CrossRef] [PubMed]

25. Bione, S.; Maestrini, E.; Rivella, S.; Mancini, M.; Regis, S.; Romeo, G.; Toniolo, D. Identification of a Novel X-Linked Gene Responsible for Emery-Dreifuss Muscular Dystrophy. *Nat. Genet.* 1994, *8*, 323–327. [CrossRef] [PubMed]

26. Crasto, S.; My, I.; Di Pasquale, E. The Broad Spectrum of LMNA Cardiac Diseases: From Molecular Mechanisms to Clinical Phenotype. *Front. Physiol.* 2020, *11*, 761. [CrossRef] [PubMed]

27. Sullivan, T.; Escalante-Alcalde, D.; Bhatt, H.; Anver, M.; Bhat, N.; Nagashima, K.; Stewart, C.L.; Burke, B. Loss of A-Type Lamin Expression Compromises Nuclear Envelope Integrity Leading to Muscular Dystrophy. *J. Cell Biol.* 1999, *147*, 913–919. [CrossRef]

28. Simon, D.N.; Wilson, K.L. Partners and Post-Translational Modifications of Nuclear Lamins. *Chromosoma* 2013, *122*, 13–31. [CrossRef]

29. Mercuri, E.; Poppe, M.; Quinlivan, R.; Messina, S.; Kinali, M.; Demay, L.; Bourke, J.; Richard, P.; Sewry, C.; Pike, M.; et al. Extreme Variability of Phenotype in Patients with an Identical Missense Mutation in the Lamin A/C Gene: From Congenital Onset with Severe Phenotype to Milder Classic Emery-Dreifuss Variant. *Arch. Neurol.* 2004, *61*, 690–694. [CrossRef]

30. Rankin, J.; Auer-Grumbach, M.; Bagg, W.; Colclough, K.; Nguyen, T.D.; Fenton-May, J.; Hattersley, A.; Hudson, J.; Jardine, P.; Josifova, D.; et al. Extreme Phenotypic Diversity and Nonpenetrance in Families with the LMNA Gene Mutation R644C. *Am. J. Med. Genet. A* 2008, *146A*, 1530–1542. [CrossRef]

31. Liberati, A.; Altman, D.G.; Tetzlaff, J.; Mulrow, C.; Gøtzsche, P.C.; Ioannidis, J.P.A.; Clarke, M.; Devereaux, P.J.; Kleijnen, J.; Moher, D. The PRISMA Statement for Reporting Systematic Reviews and Meta-Analyses of Studies That Evaluate Healthcare Interventions: Explanation and Elaboration. *BMJ* 2009, *339*, b2700. [CrossRef] [PubMed]

32. Bonello-Palot, N.; Simoncini, S.; Robert, S.; Bourgeois, P.; Sabatier, F.; Levy, N.; Dignat-George, F.; Badens, C. Prelamin A Accumulation in Endothelial Cells Induces Premature Senescence and Functional Impairment. *Atherosclerosis* 2014, *237*, 45–52. [CrossRef] [PubMed]

33. Galant, D.; Gaborit, B.; Desgrouas, C.; Abdesselam, I.; Bernard, M.; Levy, N.; Merono, F.; Coirault, C.; Roll, P.; Lagarde, A.; et al. A Heterozygous ZMPSTE24 Mutation Associated with Severe Metabolic Syndrome, Ectopic Fat Accumulation, and Dilated Cardiomyopathy. *Cells* **2016**, *5*, 21. [CrossRef](https://doi.org/10.3390/cells5040021) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

34. Pendás, A.M.; Zhou, Z.; Cadiñanos, J.; Freije, J.M.P.; Wang, J.; Hultenby, K.; Astudillo, A.; Wernerson, A.; Rodríguez, F.; Tryggvason, K.; et al. Defective Prelamin A Processing and Muscular and Adipocyte Alterations in Zmpste24 Metalloproteinase–Deficient Mice. *Nat. Genet.* **2002**, *31*, 94–99. [CrossRef](https://doi.org/10.1038/ng978) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

35. Mutesa, L.; Pierquin, G.; Cwiny-Ay, N.; Buzizi, P.; Bours, V. Hutchinson-Gilford Progeria Syndrome: Clinical and Molecular Analysis in an African Patient. *Rev. Med. Liege* **2007**, *62*, 155–158.

36. Ouzzani, M.; Hammady, H.; Fedorowicz, Z.; Elmagarmid, A. Rayyan—A Web and Mobile App for Systematic Reviews. *Syst. Rev.* **2016**, *5*, 210. [CrossRef](https://doi.org/10.1186/s13643-016-0380-4)

37. Lin, F.; Worman, H.J. Structural Organization of the Human Gene Encoding Nuclear Lamin A and Nuclear Lamin C. *J. Biol. Chem.* **1993**, *268*, 16321–16326. [CrossRef](https://doi.org/10.1016/S0021-9258(19)80001-X)

38. Avila, G.M.; González, A.P.; Abad, A.; Fournier, B.G.; León, S.R.; Corral, J.A.M.; Fernández, C.P. Is the Next Generation Sequencing the Essential Tool for the Early Diagnostic Approach in Congenital Muscular Dystrophy? New Mutation in the Gen LMNA Associated with Serious Phenotype. *Neurol. India* **2021**, *69*, 1835–1837. [CrossRef](https://doi.org/10.4103/0028-3886.329856)

39. Mercuri, E.; Brown, S.C.; Nihoyannopoulos, P.; Poulton, J.; Kinali, M.; Richard, P.; Piercy, R.J.; Messina, S.; Sewry, C.; Burke, M.M.; et al. Extreme Variability of Skeletal and Cardiac Muscle Involvement in Patients with Mutations in Exon 11 of the Lamin A/C Gene. *Muscle Nerve* **2005**, *31*, 602–609. [CrossRef](https://doi.org/10.1002/mus.20296)

40. Perrot, A.; Hussein, S.; Ruppert, V.; Schmidt, H.H.J.; Wehnert, M.S.; Duong, N.T.; Posch, M.G.; Panek, A.; Dietz, R.; Kindermann, I.; et al. Identification of Mutational Hot Spots in LMNA Encoding Lamin A/C in Patients with Familial Dilated Cardiomyopathy. *Basic Res. Cardiol.* **2009**, *104*, 90–99. [CrossRef](https://doi.org/10.1007/s00395-008-0762-5)

41. Quarta, G.; Syrris, P.; Ashworth, M.; Jenkins, S.; Zuborne Alapi, K.; Morgan, J.; Muir, A.; Pantazis, A.; McKenna, W.J.; Elliott, P.M. Mutations in the Lamin A/C Gene Mimic Arrhythmogenic Right Ventricular Cardiomyopathy. *Eur. Heart J.* **2012**, *33*, 1128–1136. [CrossRef](https://doi.org/10.1093/eurheartj/ehr452) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

42. Parent, J.J.; Towbin, J.A.; Jefferies, J.L. Left Ventricular Noncompaction in a Family with Lamin A/C Gene Mutation. *Texas Heart Inst. J.* **2015**, *42*, 73–76. [CrossRef](https://doi.org/10.14503/THIJ-14-3973) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

43. Dohrn, M.F.; Glöckle, N.; Mulahasanovic, L.; Heller, C.; Mohr, J.; Bauer, C.; Riesch, E.; Becker, A.; Battke, F.; Hörtnagel, K.; et al. Frequent Genes in Rare Diseases: Panel-Based next Generation Sequencing to Disclose Causal Mutations in Hereditary Neuropathies. *J. Neurochem.* **2017**, *143*, 507–522. [CrossRef](https://doi.org/10.1111/jnc.14108) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

44. Lin, E.W.; Brady, G.F.; Kwan, R.; Nesvizhskii, A.I.; Omary, M.B. Genotype-Phenotype Analysis of LMNA-Related Diseases Predicts Phenotype-Selective Alterations in Lamin Phosphorylation. *FASEB J.* **2020**, *34*, 9051–9073. [CrossRef](https://doi.org/10.1096/fj.202000459R)

45. Pasotti, M.; Klersy, C.; Pilotto, A.; Marziliano, N.; Rapezzi, C.; Serio, A.; Mannarino, S.; Gambarin, F.; Favalli, V.; Grasso, M.; et al. Long-Term Outcome and Risk Stratification in Dilated Cardiolaminopathies. *J. Am. Coll. Cardiol.* **2008**, *52*, 1250–1260. [CrossRef](https://doi.org/10.1016/j.jacc.2008.06.036)

46. Ditaranto, R.; Boriani, G.; Biffi, M.; Lorenzini, M.; Graziosi, M.; Ziacchi, M.; Pasquale, F.; Vitale, G.; Berardini, A.; Rinaldi, R.; et al. Differences in Cardiac Phenotype and Natural History of Laminopathies with and without Neuromuscular Onset. *Orphanet J. Rare Dis.* **2019**, *14*, 263. [CrossRef](https://doi.org/10.1186/s13023-019-1263-2)

47. Møller, D.V.; Pham, T.T.; Gustafsson, F.; Hedley, P.; Ersbøll, M.K.; Bundgaard, H.; Andersen, C.B.; Torp-Pedersen, C.; Køber, L.; Christiansen, M. The Role of Lamin A/C Mutations in Danish Patients with Idiopathic Dilated Cardiomyopathy. *Eur. J. Heart Fail.* **2009**, *11*, 1031–1035. [CrossRef](https://doi.org/10.1093/eurjhf/hfp108)

48. Refaat, M.M.; Hassanieh, S.; Ballout, J.A.; Zakka, P.; Hotait, M.; Khalil, A.; Bitar, F.; Arabi, M.; Arnaout, S.; Skouri, H.; et al. Non-Familial Cardiomyopathies in Lebanon: Exome Sequencing Results for Five Idiopathic Cases. *BMC Med. Genom.* **2019**, *12*, 33. [CrossRef](https://doi.org/10.1186/s12885-019-5400-2)

49. Sanna, T.; Dello Russo, A.; Toniolo, D.; Vytopil, M.; Pelargonio, G.; De Martino, G.; Ricci, E.; Silvestri, G.; Giglio, V.; Messano, L.; et al. Cardiac Features of Emery-Dreifuss Muscular Dystrophy Caused by Lamin A/C Gene Mutations. *Eur. Heart J.* **2003**, *24*, 2227–2236. [CrossRef](https://doi.org/10.1016/S0195-668X(03)00435-5)

50. Kandert, S.; Wehnert, M.; Müller, C.R.; Buendia, B.; Dabauvalle, M.-C. Impaired Nuclear Functions Lead to Increased Senescence and Inefficient Differentiation in Human Myoblasts with a Dominant p.R545C Mutation in the LMNA Gene. *Eur. J. Cell Biol.* **2009**, *88*, 593–608. [CrossRef](https://doi.org/10.1016/j.ejcb.2009.04.004)

51. Favreau, C.; Dubosclard, E.; Östlund, C.; Vigouroux, C.; Capeau, J.; Wehnert, M.; Higuet, D.; Worman, H.J.; Courvalin, J.C.; Buendia, B. Expression of Lamin A Mutated in the Carboxyl-Terminal Tail Generates an Aberrant Nuclear Phenotype Similar to That Observed in Cells from Patients with Dunnigan-Type Partial Lipodystrophy and Emery-Dreifuss Muscular Dystrophy. *Exp. Cell Res.* **2003**, *282*, 14–23. [CrossRef](https://doi.org/10.1016/S0014-4827(02)00019-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

52. Park, Y.-E.; Hayashi, Y.K.; Goto, K.; Komaki, H.; Hayashi, Y.; Inuzuka, T.; Noguchi, S.; Nonaka, I.; Nishino, I. Nuclear Changes in Skeletal Muscle Extend to Satellite Cells in Autosomal Dominant Emery-Dreifuss Muscular Dystrophy/Limb-Girdle Muscular Dystrophy 1B. *Neuromuscul. Disord.* **2009**, *19*, 29–36. [CrossRef](https://doi.org/10.1016/j.nmd.2008.10.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

53. Díaz-Manera, J.; Alejaldre, A.; González, L.; Olivé, M.; Gómez-Andrés, D.; Muelas, N.; Vílchez, J.J.; Llauger, J.; Carbonell, P.; Márquez-Infante, C.; et al. Muscle Imaging in Muscle Dystrophies Produced by Mutations in the EMD and LMNA Genes. *Neuromuscul. Disord.* **2016**, *26*, 33–40. [CrossRef](https://doi.org/10.1016/j.nmd.2015.11.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

54. Sewry, C.A.; Brown, S.C.; Mercuri, E.; Bonne, G.; Feng, L.; Camici, G.; Morris, G.E.; Muntoni, F. Skeletal Muscle Pathology in Autosomal Dominant Emery-Dreifuss Muscular Dystrophy with Lamin A/C Mutations. Neuropathol. Appl. Neurobiol. 2001, 27, 281–290. [CrossRef] [PubMed]

55. Vytopil, M.; Benedetti, S.; Ricci, E.; Galluzzi, G.; Dello Russo, A.; Merlini, L.; Boriani, G.; Gallina, M.; Morandi, L.; Politano, L.; et al. Mutation Analysis of the Lamin A/C Gene (LMNA) among Patients with Different Cardiomycular Phenotypes. J. Med. Genet. 2003, 40, e132. [CrossRef] [PubMed]

56. Astejada, M.N.; Goto, K.; Nagano, A.; Ura, S.; Noguchi, S.; Nonaka, I.; Nishino, I.; Hayashi, Y.K. Emerinopathy and Laminopathy Clinical, Pathological and Molecular Features of Muscular Dystrophy with Nuclear Envelopathy in Japan. Acta Myol. Myopathies Cardiomyopathies Off. J. Mediterr. Soc. Myol. 2007, 26, 159–164.

57. Colomer, J.; Iturriaga, C.; Bonne, G.; Schwartz, K.; Manilal, S.; Morris, G.E.; Puche, M.; Fernández-Alvarez, E. Autosomal Dominant Emery-Dreifuss Muscular Dystrophy: A New Family with Late Diagnosis. Neuromuscul. Disord. 2002, 12, 19–25. [CrossRef]

58. Bonne, G.; Mercuri, E.; Muchir, A.; Urtizberea, A.; Bécane, H.M.; Recan, D.; Merlini, L.; Wehnert, M.; Boor, R.; Reuner, U.; et al. Clinical and Molecular Genetic Spectrum of Autosomal Dominant Emery-Dreifuss Muscular Dystrophy Due to Mutations of the Lamin A/C Gene. Ann. Neurol. 2000, 48, 170–180. [CrossRef]

59. Kajino, S.; Ishihara, K.; Goto, K.; Ishigaki, K.; Noguchi, S.; Nonaka, I.; Osawa, M.; Nishino, I.; Hayashi, Y.K. Congenital Fiber Type Disproportion Myopathy Caused by LMNA Mutations. J. Neurol. Sci. 2014, 340, 94–98. [CrossRef]

60. Ge, L.; Zhang, C.; Wang, Z.; Chan, S.H.S.; Zhu, W.; Han, C.; Zhang, X.; Zheng, H.; Wu, L.; Jin, B.; et al. Congenital Muscular Dystrophies in China. Clin. Genet. 2019, 96, 207–215. [CrossRef]

61. Ebert, M.; Wijnmaalen, A.P.; de Riva, M.; Trines, S.A.; Androulakis, A.F.A.; Glashan, C.A.; Schalij, M.J.; van Tintelen, J.; Jongbloed, J.D.H.; Zeppenfeld, K. Prevalence and Prognostic Impact of Pathogenic Variants in Patients With Dilated Cardiomyopathy Referred for Ventricular Tachycardia Ablation. JACC. Clin. Electrophysiol. 2020, 6, 1103–1114. [CrossRef] [PubMed]

62. Vytopil, M.; Vohanka, S.; Vlasinova, J.; Toman, J.; Novak, M.; Toniolo, D.; Ricotti, R.; Lukas, Z. The Screening for X-Linked Emery-Dreifuss Muscular Dystrophy amongst Young Patients with Idiopathic Heart Conduction System Disease Treated by a Pacemaker Implant. Eur. J. Neurol. 2004, 11, 531–534. [CrossRef] [PubMed]

63. Liang, W.-C.; Jong, Y.-J.; Wang, C.-H.; Wang, C.-H.; Tian, X.; Chen, W.-Z.; Kan, T.-M.; Minami, N.; Nishino, I.; Wong, L.-J.C. Clinical, Pathological, Imaging, and Genetic Characterization in a Taiwanese Cohort with Limb-Girdle Muscular Dystrophy. Orphanet J. Rare Dis. 2020, 15, 160. [CrossRef] [PubMed]

64. Sanga, S.; Ghosh, A.; Kumar, K.; Polavarapu, K.; Preethish-Kumar, V.; Vengalil, S.; Nashi, S.; Bardhan, M.; Arunachal, G.; Raju, S.; et al. Whole-Exome Analyses of Congenital Muscular Dystrophy and Congenital Myopathy Patients from India Reveal a Wide Spectrum of Known and Novel Mutations. Eur. J. Neurol. 2021, 28, 992–1003. [CrossRef]

65. Muchir, A.; Medioni, J.; Laluc, M.; Massart, C.; Arimura, T.; van der Kooi, A.J.; Desguerre, I.; Mayer, M.; Ferrer, X.; Briault, S.; et al. Nuclear Envelope Alterations in Fibroblasts from Patients with Muscular Dystrophy, Cardiomyopathy, and Partial Lipodystrophy Carrying Lamin A/C Gene Mutations. Muscle Nerve 2004, 30, 444–450. [CrossRef]

66. Fan, Y.; Tan, D.; Song, D.; Zhang, X.; Chang, X.; Wang, Z.; Zhang, C.; Chan, S.H.-S.; Wu, Q.; Wu, L.; et al. Clinical Spectrum and Genetic Variations of LMNA -Related Muscular Dystrophies in a Large Cohort of Chinese Patients. J. Med. Genet. 2020, 58, 326–333. [CrossRef]

67. Magagnotti, C.; Bachi, A.; Zerbini, G.; Fattore, E.; Fermo, I.; Riba, M.; Previtali, S.C.; Ferrari, M.; Andolfi, A.; Benedetti, S. Protein Profiling Reveals Energy Metabolism and Cytoskeletal Protein Alterations in LMNA Mutation Carriers. Biochim. Biophys. Acta 2012, 1822, 970–979. [CrossRef] [PubMed]

68. Choi, S.A.; Cho, A.; Kim, S.Y.; Kim, W.J.; Shim, Y.K.; Lee, J.S.; Jang, S.S.; Lim, B.C.; Kim, H.; Hwang, H.; et al. Importance of Early Diagnosis in LMNA-Related Muscular Dystrophy for Cardiac Surveillance. Muscle Nerve 2019, 60, 668–672. [CrossRef]

69. Makri, S.; Clarke, N.F.; Richard, P.; Maugenre, S.; Demay, L.; Bonne, G.; Guicheney, P. Germinal Mosaicism for LMNA Mimics Autosomal Recessive Congenital Muscular Dystrophy. Neuromuscul. Disord. 2009, 19, 26–28. [CrossRef]

70. Tan, D.; Yang, H.; Yuan, Y.; Bonnemann, C.; Chang, X.; Wang, S.; Wu, Y.; Wu, X.; Xiong, H. Phenotype-Genotype Analysis of Chinese Patients with Early-Onset LMNA-Related Muscular Dystrophy. PLoS ONE 2015, 10, e0129699. [CrossRef]

71. Di Barletta, M.R.; Ricci, E.; Galluzzi, G.; Tonali, P.; Mora, M.; Morandi, L.; Romorini, A.; Voit, T.; Orstavik, K.H.; Merlini, L.; et al. Different Mutations in the LMNA Gene Cause Autosomal Dominant and Autosomal Recessive Emery-Dreifuss Muscular Dystrophy. Am. J. Hum. Genet. 2000, 66, 1407–1412. [CrossRef] [PubMed]

72. Rudnik-Schöneborn, S.; Botzenhart, E.; Eggermann, T.; Senderek, J.; Schoser, B.G.H.; Schröder, R.; Wehnert, M.; Wirth, B.; Zerres, K. Mutations of the LMNA Gene Can Mimic Autosomal Dominant Proximal Spinal Muscular Atrophy. Neurogenetics 2007, 8, 137–142. [CrossRef] [PubMed]

73. Francisco, A.R.G.; Santos Gonçalves, I.; Veiga, F.; Mendes Pedro, M.; Pinto, F.J.; Brito, D. Complex Phenotype Linked to a Mutation in Exon 11 of the Lamin A/C Gene: Hypertrophic Cardiomyopathy, Atrioventricular Block, Severe Dyslipidemia and Diabetes. Rev. Port. Cardiol. 2017, 36, 669.e1–669.e4. [CrossRef] [PubMed]

74. Gigli, M.; Merlo, M.; Graw, S.L.; Barbati, G.; Rowland, T.J.; Slavov, D.B.; Stolfo, D.; Haywood, M.E.; Dal Ferro, M.; Altinier, A.; et al. Genetic Risk of Arrhythmic Phenotypes in Patients With Dilated Cardiomyopathy. J. Am. Coll. Cardiol. 2019, 74, 1480–1490. [CrossRef]

75. Meinke, P.; Mattioli, E.; Haque, F.; Antoku, S.; Columbaro, M.; Straatman, K.R.; Worman, H.J.; Gundersen, G.G.; Lattanzi, G.; Wehnert, M.; et al. Muscular Dystrophy-Associated SUN1 and SUN2 Variants Disrupt Nuclear-Cytoskeletal Connections and Myonuclear Organization. *PLoS Genet.* **2014**, *10*, e1004605. [CrossRef]

76. Ben Yaou, R.; Toutain, A.; Arimura, T.; Demay, L.; Massart, C.; Peccate, C.; Muchir, A.; Llense, S.; Deburgrave, N.; Leturcq, F.; et al. Multitissular Involvement in a Family with LMNA and EMD Mutations: Role of Digenic Mechanism? *Neurology* **2007**, *68*, 1883–1894. [CrossRef]

77. Muntoni, F.; Bonne, G.; Goldfarb, L.; Mercuri, E.; Piercy, R.; Burke, M.; Yaou, R.; Richard, P.; Récan, D.; Shatunov, A.; et al. Disease Severity in Dominant Emery Dreifuss Is Increased by Mutations in Both Emerin and Desmin Proteins. *Brain A J. Neurol.* **2006**, *129*, 1260–1268. [CrossRef]

78. Granger, B.; Gueneau, L.; Drouin-Garraud, V.; Pedergnana, V.; Gagnon, F.; Ben Yaou, R.; du Montcel, S.; Bonne, G. Modifier Locus of the Skeletal Muscle Involvement in Emery-Dreifuss Muscular Dystrophy. *Hum. Genet.* **2011**, *129*, 149–159. [CrossRef]

79. Riordan, J.D.; Nadeau, J.H. From Peas to Disease: Modifier Genes, Network Resilience, and the Genetics of Health. *Am. J. Hum. Genet.* **2017**, *101*, 177. [CrossRef]

80. Cerrone, M.; Remme, C.A.; Tadros, R.; Bezzina, C.R.; Delmar, M. Beyond the One Gene–One Disease Paradigm. *Circulation* **2019**, *140*, 595–610. [CrossRef]

81. Leitch, C.C.; Zaghloul, N.A.; Davis, E.E.; Stoetzel, C.; Diaz-Font, A.; Rix, S.; Al-Fadhel, M.; Lewis, R.A.; Eyaid, W.; Banin, E.; et al. Hypomorphic Mutations in Syndromic Encephalocele Genes Are Associated with Bardet-Biedl Syndrome. *Nat. Genet.* **2008**, *40*, 443–448. [CrossRef] [PubMed]

82. Li, W.; Soave, D.; Miller, M.R.; Keenan, K.; Lin, F.; Gong, J.; Chiang, T.; Stephenson, A.L.; Durie, P.; Rommens, J.; et al. Unraveling the Complex Genetic Model for Cystic Fibrosis: Pleiotropic Effects of Modifier Genes on Early Cystic Fibrosis-Related Morbidities. *Hum. Genet.* **2014**, *133*, 151–161. [CrossRef] [PubMed]

83. Harhouri, K.; Frankel, D.; Bartoli, C.; Roll, P.; De Sandre-Giovannoli, A.; Lévy, N. An Overview of Treatment Strategies for Hutchinson-Gilford Progeria Syndrome. *Nucleus* **2018**, *9*, 246–257. [CrossRef] [PubMed]

84. Coffinier, C.; Chang, S.Y.; Nobumori, C.; Tu, Y.; Farber, E.A.; Toth, J.I.; Fong, L.G.; Young, S.G. Abnormal Development of the Cerebral Cortex and Cerebellum in the Setting of Lamin B2 Deficiency. *Proc. Natl. Acad. Sci. USA* **2010**, *107*, 5076–5081. [CrossRef] [PubMed]

85. Coffinier, C.; Jung, H.J.; Nobumori, C.; Chang, S.; Tu, Y.; Barnes, R.H.; Yoshinaga, Y.; De Jong, P.J.; Vergnes, L.; Reue, K.; et al. Deficiencies in Lamin B1 and Lamin B2 Cause Neurodevelopmental Defects and Distinct Nuclear Shape Abnormalities in Neurons. *Mol. Biol. Cell* **2011**, *22*, 4683–4693. [CrossRef] [PubMed]

86. Coffinier, C.; Fong, L.G.; Young, S.G. LINCing Lamin B2 to Neuronal Migration: Growing Evidence for Cell-Specific Roles of B-Type Lamins. *Nucleus* **2010**, *1*, 407–411. [CrossRef]

87. Vergnes, L.; Péterfy, M.; Bergo, M.O.; Young, S.G.; Reue, K. Lamin B1 Is Required for Mouse Development and Nuclear Integrity. *Proc. Natl. Acad. Sci. USA* **2004**, *101*, 10428–10433. [CrossRef]

88. Lee, J.M.; Tu, Y.; Tatar, A.; Wu, D.; Nobumori, C.; Jung, H.J.; Yoshinaga, Y.; Coffinier, C.; De Jong, P.J.; Fong, L.G.; et al. Reciprocal Knock-in Mice to Investigate the Functional Redundancy of Lamin B1 and Lamin B2. *Mol. Biol. Cell* **2014**, *25*, 1666. [CrossRef]

89. Padiath, Q.S.; Saigoh, K.; Schiffmann, R.; Asahara, H.; Yamada, T.; Koeppen, A.; Hogan, K.; Ptácek, L.J.; Fu, Y.-H. Lamin B1 Duplications Cause Autosomal Dominant Leukodystrophy. *Nat. Genet.* **2006**, *38*, 1114–1123. [CrossRef]

90. Emery, A.E.H. Emery-Dreifuss Syndrome. *Syndr. Mon. J. Med. Genet.* **1989**, *26*, 637–641. [CrossRef]

91. Boriani, G.; Gallina, M.; Merlini, L.; Bonne, G.; Toniolo, D.; Amati, S.; Biffi, M.; Martignani, C.; Frabetti, L.; Bonvicini, M.; et al. Clinical Relevance of Atrial Fibrillation/Flutter, Stroke, Pacemaker Implant, and Heart Failure in Emery-Dreifuss Muscular Dystrophy: A Long-Term Longitudinal Study. *Stroke* **2003**, *34*, 901–908. [CrossRef] [PubMed]

92. Broers, J.L.V.; Peeters, E.A.G.; Kuijpers, H.J.H.; Endert, J.; Bouten, C.V.C.; Oomens, C.W.J.; Baaijens, F.P.T.; Ramaekers, F.C.S. Decreased Mechanical Stiffness in LMNA-/– Cells Is Caused by Defective Nucleo-Cytoskeletal Integrity: Implications for the Development of Laminopathies. *Hum. Mol. Genet.* **2004**, *13*, 2567–2580. [CrossRef] [PubMed]

93. Lammerding, J.; Kamm, R.D.; Lee, R.T.; Schulze, P.C.; Takahashi, T.; Kozlov, S.; Sullivan, T.; Stewart, C.L. Lamin A/C Deficiency Causes Defective Nuclear Mechanics and Mechanotransduction. *J. Clin. Investig.* **2004**, *113*, 370–378. [CrossRef] [PubMed]

94. Hutchison, C.J. Lamins: Building Blocks or Regulators of Gene Expression? *Nat. Rev. Mol. Cell Biol.* **2002**, *3*, 848–858. [CrossRef] [PubMed]

95. Zastrow, M.S.; Vlcek, S.; Wilson, K.L. Proteins That Bind A-Type Lamins: Integrating Isolated Clues. *J. Cell Sci.* **2004**, *117*, 979–987. [CrossRef]

96. Ellis, D.; Jenkins, H.; Whitfield, W.; Hutchinson, C. GST-Lamin Fusion Proteins Act as Dominant Negative Mutants in Xenopus Egg Extract and Reveal the Function of the Lamina in DNA Replication. *J. Cell Sci.* **1997**, *110*, 2507–2518. [CrossRef]

97. Schirmer, E.C.; Gerace, L. The Nuclear Membrane Proteome: Extending the Envelope. *Trends Biochem. Sci.* **2005**, *30*, 551–558. [CrossRef]

98. Dittmer, T.A.; Sahni, N.; Kubben, N.; Hill, D.E.; Vidal, M.; Burgess, R.C.; Roukos, V.; Misteli, T. Systematic Identification of Pathological Lamin A Interactors. *Mol. Biol. Cell* **2014**, *25*, 1493–1510. [CrossRef]

99. Kubben, N.; Voncken, J.W.; Demmers, J.; Calis, C.; Van Almen, G.; Pinto, Y.M.; Misteli, T. Identification of Differential Protein Interactors of Lamin A and Progerin. *Nucleus* **2010**, *1*, 513–525. [CrossRef]

100. Vitkup, D.; Sander, C.; Church, G.M. The Amino-Acid Mutational Spectrum of Human Genetic Disease. Genome Biol. 2003, 4, R72. [CrossRef]

101. Cooper, D.N.; Youssoufian, H. The CpG Dinucleotide and Human Genetic Disease. Hum. Genet. 1988, 78, 151–155. [CrossRef]

102. Betts, M.J.; Russell, R.B. Amino Acid Properties and Consequences of Substitutions. In Bioinformatics for Geneticists; Barnes, M.R., Gray, I.C., Eds.; John Wiley & Sons, Ltd.: New York, NY, USA, 2003; pp. 289–316.

103. Stuurman, N.; Sasse, B.; Fisher, P.A. Intermediate Filament Protein Polymerization: Molecular Analysis of Drosophila Nuclear Lamin Head-to-Tail Binding. J. Struct. Biol. 1996, 117, 1–15. [CrossRef] [PubMed]

104. Ahn, J.; Jo, I.; Kang, S.; Hong, S.; Kim, S.; Jeong, S.; Kim, Y.H.; Park, B.J.; Ha, N.C. Structural Basis for Lamin Assembly at the Molecular Level. Nat. Commun. 2019, 10, 1–12. [CrossRef]

105. Bhattacharjee, P.; Dasgupta, D.; Sengupta, K. DCM Associated LMNA Mutations Cause Distortions in Lamina Structure and Assembly. Biochim. Biophys. Acta—Gen. Subj. 2017, 1861, 2598–2608. [CrossRef] [PubMed]

106. Ostlund, C.; Bonne, G.; Schwartz, K.; Worman, H.J. Properties of Lamin A Mutants Found in Emery-Dreifuss Muscular Dystrophy, Cardiomyopathy and Dunnigan-Type Partial Lipodystrophy. J. Cell Sci. 2001, 114, 4435–4445. [CrossRef] [PubMed]

107. Vigouroux, C.; Auclair, M.; Dubosclard, E.; Poucheclet, M.; Capeau, J.; Courvalin, J.C.; Buendia, B. Nuclear Envelope Disorganization in Fibroblasts from Lipodystrophic Patients with Heterozygous R482Q/W Mutations in the Lamin A/C Gene. J. Cell Sci. 2001, 114, 4459–4468. [CrossRef] [PubMed]

108. Muchir, A.; Van Engelen, B.G.; Lammens, M.; Mislow, J.M.; McNally, E.; Schwartz, K.; Bonne, G. Nuclear Envelope Alterations in Fibroblasts from LGMD1B Patients Carrying Nonsense Y259X Heterozygous or Homozygous Mutation in Lamin A/C Gene. Exp. Cell Res. 2003, 291, 352–362. [CrossRef]

109. Osmanagic-Myers, S.; Foisner, R. The Structural and Gene Expression Hypotheses in Laminopathic Diseases—Not so Different after All. Mol. Biol. Cell 2019, 30, 1786–1790. [CrossRef]

110. Clements, L.; Manilal, S.; Love, D.R.; Morris, G.E. Direct Interaction between Emerin and Lamin A. Biochem. Biophys. Res. Commun. 2000, 267, 709–714. [CrossRef]

111. Lee, K.K.; Haraguchi, T.; Lee, R.S.; Koujin, T.; Hiraoka, Y.; Wilson, K.L. Distinct Functional Domains in Emerin Bind Lamin A and DNA-Bridging Protein BAF. J. Cell Sci. 2001, 114, 4567–4573. [CrossRef]

112. Sakaki, M.; Koike, H.; Takahashi, N.; Sasagawa, N.; Tomioka, S.; Arahata, K.; Ishiura, S. Interaction between Emerin and Nuclear Lamins. J. Biochem. 2001, 129, 321–327. [CrossRef] [PubMed]

113. Holt, I.; Östlund, C.; Stewart, C.L.; thi Man, N.; Worman, H.J.; Morris, G.E. Effect of Pathogenic Mis-Sense Mutations in Lamin A on Its Interaction with Emerin in Vivo. J. Cell Sci. 2003, 116, 3027–3035. [CrossRef] [PubMed]

114. Mziaut, H.; Korza, G.; Elkahloun, A.G.; Ozols, J. Induction of Stearoyl CoA Desaturase Is Associated with High-Level Induction of Emerin RNA. Biochem. Biophys. Res. Commun. 2001, 282, 910–915. [CrossRef] [PubMed]

115. Dharmaraj, T.; Guan, Y.; Liu, J.; Badens, C.; Gaborit, B.; Wilson, K.L. Rare BANF1 Alleles and Relatively Frequent EMD Alleles Including “Healthy Lipid” Emerin p.D149H in the ExAC Cohort. Front. Cell Dev. Biol. 2019, 7, 48. [CrossRef]

116. Tang, K.; Finley, R.L.; Nie, D.; Honn, K.V. Identification of 12-Lipoxygenase Interaction with Cellular Proteins by Yeast Two-Hybrid Screening. Biochemistry 2000, 39, 3185–3191. [CrossRef]

117. Dobrian, A.D.; Morris, M.A.; Taylor-Fishwick, D.A.; Holman, T.R.; Imai, Y.; Mirmira, R.G.; Nadler, J.L. Role of the 12-Lipoxygenase Pathway in Diabetes Pathogenesis and Complications. Pharmacol. Ther. 2019, 195, 100–110. [CrossRef]

118. Dong, C.; Liu, S.; Cui, Y.; Guo, Q. 12-Lipoxygenase as a Key Pharmacological Target in the Pathogenesis of Diabetic Nephropathy. Eur. J. Pharmacol. 2020, 879, 173122. [CrossRef]

119. Grzesik, W.J.; Nadler, J.L.; Machida, Y.; Nadler, J.L.; Imai, Y.; Morris, M.A. Expression Pattern of 12-Lipoxygenase in Human Islets With Type 1 Diabetes and Type 2 Diabetes. J. Clin. Endocrinol. Metab. 2015, 100, E387–E395. [CrossRef]

120. Prasad, K.M.R.; Thimmalapura, P.R.R.; Woode, E.A.A.; Nadler, J.L. Evidence That Increased 12-Lipoxygenase Expression Impairs Pancreatic β Cell Function and Viability. Biochem. Biophys. Res. Commun. 2003, 308, 427–432. [CrossRef]

121. Sasseville, A.M.-J.; Langelier, Y. In Vitro Interaction of the Carboxy-Terminal Domain of Lamin A with Actin. FEBS Lett. 1998, 425, 485–489. [CrossRef]

122. Stierlé, V.; Couprie, J.; Ostlund, C.; Krimm, I.; Zinn-Justin, S.; Hossenlopp, P.; Worman, H.J.; Courvalin, J.-C.; Duband-Goulet, I. The Carboxyl-Terminal Region Common to Lamins A and C Contains a DNA Binding Domain. Biochemistry 2003, 42, 4819–4828. [CrossRef] [PubMed]

123. Koch, A.J.; Holaska, J.M. Emerin in Health and Disease. Semin. Cell Dev. Biol. 2014, 29, 95–106. [CrossRef] [PubMed]

124. Manilal, S.; Recan, D.; Sewry, C.A.; Hoeltzenbein, M.; Llense, S.; Leturcq, F.; Deburgrave, N.; Barbot, J.; Man, N.; Muntoni, F.; et al. Mutations in Emery-Dreifuss Muscular Dystrophy and Their Effects on Emerin Protein Expression. Hum. Mol. Genet. 1998, 7, 855–864. [CrossRef] [PubMed]

125. Zhou, C.; Li, C.; Zhou, B.; Sun, H.; Koullourou, V.; Holt, I.; Puckelwartz, M.J.; Warren, D.T.; Hayward, R.; Lin, Z.; et al. Novel Nesprin-1 Mutations Associated with Dilated Cardiomyopathy Cause Nuclear Envelope Disruption and Defects in Myogenesis. Hum. Mol. Genet. 2017, 26, 2258–2276. [CrossRef]

126. Zhang, X.; Lei, K.; Yuan, X.; Wu, X.; Zhuang, Y.; Xu, T.; Xu, R.; Han, M. SUN1/2 and Syne/Nesprin-1/2 Complexes Connect Centrosome to the Nucleus during Neurogenesis and Neuronal Migration in Mice. Neuron 2009, 64, 173–187. [CrossRef]

127. Clancy, J.P.; Bebök, Z.; Ruiz, F.; King, C.; Jones, J.; Walker, L.; Greer, H.; Hong, J.; Wing, L.; Macaluso, M.; et al. Evidence That Systemic Gentamicin Suppresses Premature Stop Mutations in Patients with Cystic Fibrosis. *Am. J. Respir. Crit. Care Med.* 2001, **163**, 1683–1692. [CrossRef]

128. Wilschanski, M.; Famini, C.; Blau, H.; Rivlin, J.; Augarten, A.; Avital, A.; Kerem, B.; Kerem, E. A Pilot Study of the Effect of Gentamicin on Nasal Potential Difference Measurements in Cystic Fibrosis Patients Carrying Stop Mutations. *Am. J. Respir. Crit. Care Med.* 2000, **161**, 860–865. [CrossRef]

129. Malik, V.; Rodino Klapac, L.R.; Viillet, L.; Mendell, J.R. Aminoglycoside-Induced Mutation Suppression (Stop Codon Readthrough) as a Therapeutic Strategy for Duchenne Muscular Dystrophy. *Ther. Adv. Neurol. Disord.* 2010, **3**, 379–389. [CrossRef]

130. Politano, L.; Nigro, G.; Nigro, V.; Piluso, G.; Papparella, S.; Paciello, O.; Comi, L. Gentamicin Administration in Duchenne Patients with Premature Stop Codon. Preliminary Results. *Off. J. Mediterr. Soc. Myol.* 2003, **22**, 15–21.

131. James, P.D.; Raut, S.; Rivard, G.E.; Poon, M.C.; Warner, M.; McKenna, S.; Leggo, J.; Lillicrap, D. Aminoglycoside Suppression of Nonsense Mutations in Severe Hemophilia. *Blood* 2005, **106**, 3043–3048. [CrossRef]

132. Taguchi, A.; Hamada, K.; Shiozuka, M.; Kobayashi, M.; Murakami, S.; Takayama, K.; Taniguchi, A.; Usui, T.; Matsuda, R.; Hayashi, Y. Structure-Activity Relationship Study of Leucyl-3-Epi-Deoxynegamycin for Potent Premature Termination Codon Readthrough. *ACS Med. Chem. Lett.* 2017, **8**, 1060–1065. [CrossRef] [PubMed]

133. Hamada, K.; Omura, N.; Taguchi, A.; Baradaran-Heravi, A.; Kotake, M.; Arai, M.; Takayama, K.; Taniguchi, A.; Roberge, M.; Hayashi, Y. New Negamycin-Based Potent Readthrough Derivative Effective against TGA-Type Nonsense Mutations. *ACS Med. Chem. Lett.* 2019, **10**, 1450–1456. [CrossRef] [PubMed]

134. Wilschanski, M.; Yahav, Y.; Yaacov, Y.; Blau, H.; Bentur, L.; Rivlin, J.; Aviram, M.; Bdolah-Abram, T.; Bebok, Z.; Shushi, L.; et al. Gentamicin-Induced Correction of CFTR Function in Patients with Cystic Fibrosis and CFTR Stop Mutations. *N. Engl. J. Med.* 2009, **349**, 1433–1441. [CrossRef] [PubMed]

135. Kellermayer, R.; Szigeti, R.; Keeling, K.M.; Bedekovics, T.; Bedwell, D.M. Aminoglycosides as Potential Pharmacogenetic Agents in the Treatment of Hailey-Hailey Disease. *J. Investig. Dermatol.* 2006, **126**, 229–231. [CrossRef]

136. PTC Therapeutics, Inc. *PTC Therapeutics Receives Conditional Approval in the European Union for Translarna™ for the Treatment of Nonsense Mutation Duchenne Muscular Dystrophy*; PTC Therapeutics, Inc.: South Plainfield, NJ, USA, 2014.

137. McDonald, C.M.; Campbell, C.; Torricelli, R.E.; Finkel, R.S.; Flanigan, K.M.; Goemans, N.; Heydemann, P.; Kaminska, A.; Kirschner, J.; Muntoni, F.; et al. Ataluren in Patients with Nonsense Mutation Duchenne Muscular Dystrophy (ACT DMD): A Multicentre, Randomised, Double-Blind, Placebo-Controlled, Phase 3 Trial. *Lancet* 2017, **390**, 1489–1498. [CrossRef]

138. Folker, E.S.; Östlund, C.; Luxton, G.W.G.; Worman, H.J.; Gundersen, G.G. Lamin A Variants That Cause Striated Muscle Disease Are Defective in Anchoring Transmembrane Actin-Associated Nuclear Lines for Nuclear Movement. *Proc. Natl. Acad. Sci. USA* 2011, **108**, 131–136. [CrossRef]

139. Chang, W.; Antoku, S.; Östlund, C.; Worman, H.J.; Gundersen, G.G. Linker of Nucleoskeleton and Cytoskeleton (LINC) Complex-Mediated Actin-Dependent Nuclear Positioning Orients Centrosomes in Migrating Myoblasts. *Nucleus* 2015, **6**, 77–88. [CrossRef]

140. Kirby, T.J.; Lammerding, J. Emerging Views of the Nucleus as a Cellular Mechanosensor. *Nat. Cell Biol.* 2018, **20**, 373–381. [CrossRef]

141. Lombardi, M.L.; Jaalouk, D.E.; Shanahan, C.M.; Burke, B.; Roux, K.J.; Lammerding, J. The Interaction between Nesprins and Sun Proteins at the Nuclear Envelope Is Critical for Force Transmission between the Nucleus and Cytoskeleton. *J. Biol. Chem.* 2011, **286**, 26743–26753. [CrossRef]

142. Swift, J.; Ivanovska, I.L.; Buxboim, A.; Harada, T.; Dingal, P.C.D.P.; Pinter, J.; Pajerowski, J.D.; Spinler, K.R.; Shin, J.W.; Tewari, M.; et al. Nuclear Lamin-A Scales with Tissue Stiffness and Enhances Matrix-Directed Differentiation. *Science* 2013, **341**, 1240104. [CrossRef]

143. Brosig, M.; Ferralli, J.; Gelman, L.; Chiquet, M.; Chiquet-Ehrismann, R. Interfering with the Connection between the Nucleus and the Cytoskeleton Affects Nuclear Rotation, Mechanotransduction and Myogenesis. *Int. J. Biochem. Cell Biol.* 2010, **42**, 1717–1728. [CrossRef] [PubMed]

144. Matsumoto, A.; Hieda, M.; Yokoyama, Y.; Nishioka, Y.; Yoshidome, K.; Tsujimoto, M.; Matsuura, N. Global Loss of a Nuclear Lamina Component, Lamin A/C, and LINC Complex Components SUN1, SUN2, and Nesprin-2 in Breast Cancer. *Cancer Med.* 2015, **4**, 1547–1557. [CrossRef] [PubMed]

145. Masica, D.L.; Karchin, R. Correlation of Somatic Mutation and Expression Identifies Genes Important in Human Glioblastoma Progression and Survival. *Cancer Res.* 2011, **71**, 4561. [CrossRef] [PubMed]

146. Xu, Y.; Luo, H.; Hu, Q.; Zhu, H. Identification of Potential Driver Genes Based on Multi-Genomic Data in Cervical Cancer. *Front. Genet.* 2021, **12**, 598304. [CrossRef] [PubMed]

147. Zhang, Q.; Narayanan, V.; Mui, K.L.; O’Bryan, C.S.; Anderson, R.H.; KC, B.; Cabe, J.I.; Denis, K.B.; Antoku, S.; Roux, K.J.; et al. Mechanical Stabilization of the Glandular Acinus by Linker of Nucleoskeleton and Cytoskeleton Complex. *Curr. Biol.* 2019, **29**, 2839.e4. [CrossRef]

148. De Sandre-Giovannoli, A.; Bernard, R.; Cau, P.; Navarro, C.; Amiel, J.; Boccaccio, I.; Lyonnet, S.; Stewart, C.L.; Munnich, A.; Le Merrer, M.; et al. Lamin A Truncation in Hutchinson-Gilford Progeria. *Science* 2003, **300**, 2055. [CrossRef]

149. Chang, W.; Wang, Y.; Gant Luxton, G.W.; Östlund, C.; Worman, H.J.; Gundersen, G.G. Imbalanced Nucleocytoskeletal Connections Create Common Polarity Defects in Progeria and Physiological Aging. *Proc. Natl. Acad. Sci. USA* **2019**, *116*, 3578–3583. [CrossRef]

150. Cann, F.; Corbett, M.; O’Sullivan, D.; Tennant, S.; Hailey, H.; Grieve, J.H.K.; Broadhurst, P.; Rankin, R.; Dean, J.C.S. Phenotype-Driven Molecular Autopsy for Sudden Cardiac Death. *Clin. Genet.* **2017**, *91*, 22–29. [CrossRef]
