
Received: 21 November 2023 | Revised: 20 February 2024 | Accepted: 4 March 2024

DOI: 10.1002/pro.4964

RESEARCH ARTICLE

HadBD dehydratase from *Mycobacterium tuberculosis* fatty acid synthase type II: A singular structure for a unique function

Pascaline Bories${}^{1}$ | Julie Rima${}^{1}$ | Samuel Tranier${}^{1}$ | Julien Marcoux${}^{1}$ | Yasmina Grimoire${}^{1}$ | Mathilde Tomaszczyk${}^{1}$ | Anne Launay${}^{2}$ | Hedia Marrakchi${}^{1}$ | Odile Burlet-Schiltz${}^{1}$ | Lionel Mourey${}^{1}$ | Karine Fata${}^{2}$ | Manuelle Ducoux-Petit${}^{1}$ | Fabienne Bardou${}^{1}$ | Cécile Bon${}^{1}$ | Annaik Quémard${}^{1}$

${}^{1}$Institut de Pharmacologie et de Biologie Structurale (IPBS), Université de Toulouse, CNRS, Université Toulouse III - Paul Sabatier (UPS), Toulouse, France  
${}^{2}$Service de TP de Biochimie, Université de Toulouse, Université Toulouse III - Paul Sabatier (UPS), Toulouse, France

---

**Correspondence**

Fabienne Bardou, Cécile Bon, and Annaik Quémard, IPBS CNRS-UPS UMR 5089, 205 Route de Narbonne, BP 64182, 31077 Toulouse Cedex 4, France.  
Email: fabienne.bardou@ipbs.fr, cecile.bon@ipbs.fr, and annaik.quemard@ipbs.fr

**Funding information**

Ministère de l'Enseignement Supérieur et de la Recherche; MSDAVENIR; Agence Nationale de la Recherche

**Review Editor:** John Kuriyan

---

**Abstract**

Worldwide, tuberculosis is the second leading infectious killer and multidrug resistance severely hampers disease control. Mycolic acids are a unique category of lipids that are essential for viability, virulence, and persistence of the causative agent, *Mycobacterium tuberculosis* (*Mtb*). Therefore, enzymes involved in mycolic acid biosynthesis represent an important class of drug targets. We previously showed that the (3R)-hydroxyacyl-ACP dehydratase (HAD) protein HadD is dedicated mainly to the production of ketomycolic acids and plays a determinant role in *Mtb* biofilm formation and virulence. Here, we discovered that HAD activity requires the formation of a tight heterotetramer between HadD and HadB, a HAD unit encoded by a distinct chromosomal region. Using biochemical, structural, and cell-based analyses, we showed that HadB is the catalytic subunit, whereas HadD is involved in substrate binding. Based on HadBD${}_{Mtb}$ crystal structure and substrate-bound models, we identified determinants of the ultra-long-chain lipid substrate specificity and revealed details of structure-function relationship. HadBD${}_{Mtb}$ unique function is partly due to a wider opening and a higher flexibility of the substrate-binding crevice in HadD, as well as the drastically truncated central α-helix of HadD hotdog fold, a feature described for the first time in a HAD enzyme. Taken together, our study shows that HadBD${}_{Mtb}$, and not HadD alone, is the

---

Pascaline Bories, Julie Rima, and Samuel Tranier contributed equally to this work.

This is an open access article under the terms of the [Creative Commons Attribution-NonCommercial-NoDerivs License](https://creativecommons.org/licenses/by-nc-nd/4.0/), which permits use and distribution in any medium, provided the original work is properly cited, the use is non-commercial and no modifications or adaptations are made.

© 2024 The Authors. Protein Science published by Wiley Periodicals LLC on behalf of The Protein Society.

Protein Science. 2024;33:e4964.  
https://doi.org/10.1002/pro.4964

biologically relevant functional unit. These results have important implications for designing innovative antivirulence molecules to fight tuberculosis, as they suggest that the target to consider is not an isolated subunit, but the whole HadBD complex.

KEYWORDS

(3R)-hydroxyacyl-ACP dehydratase HadBD, catalytic subunit, crystal structure, fatty acid synthase type II system, hotdog helix, *Mycobacterium tuberculosis*, mycolic acids, structure-function relationships, substrate-binding subunit, ultra-long-chain substrates

1 | INTRODUCTION

Despite the availability of a vaccine and antibiotics that led to improvement in patient care, tuberculosis (TB) remains a major public health concern. Indeed, it is one of the leading causes of death due to a single infectious agent (*Mycobacterium tuberculosis* (*Mtb*)) in the world, responsible for about 10 million new cases and 1.3 million deaths each year (WHO, 2023). The lack of control of the disease is due to many factors, notably a complicated treatment regimen involving the use of multiple antibiotics over the course of several months, and the emergence of multi- and extremely-drug resistant *Mtb* variants. Therefore, design of new therapeutic strategies for TB remains one of the most urgent unmet medical need (WHO, 2023). In this context, targeting proteins required for *Mtb* virulence but not for essential physiological processes was proposed as a strategy that may thwart infection, while reducing likelihood of resistance (Dickey et al., 2017).

*Mtb* synthesizes a variety of atypical lipids that contribute to the complexity and very low permeability of its cell envelope. They include mycolic acids (MAs), high molecular weight (up to 100 carbon atoms) α-branched and β-hydroxylated fatty acids with different types of chemical modifications that are major components of the external “mycomembrane” (Quémard, 2016). Three main classes, α-, methoxy-, and keto-MAs, coexist in *Mtb*. MAs are not only essential for the bacterial viability but they play determinant roles in *Mtb* virulence and persistence by modulating the immune response of the infected host (Daffe et al., 2019). Furthermore, keto-MAs are essential for the formation of *Mtb* biofilms, complex multibacterial structures detected in the lungs of TB patients and shown to contribute to both *Mtb* virulence and drug tolerance in mice (Chakraborty et al., 2021; Sambandan et al., 2013). MA biosynthesis pathway requires a fatty acid synthase type II (FAS-II) with unique capacities to produce “meromycoloyl chains”, the ultra-long (up to C<sub>60</sub>-C<sub>70</sub>) main chains of MAs, that occur only in mycobacteria and related genera (Quémard, 2016). Importantly, several specific anti-TB drugs, such as isoniazid and the recently approved delamanid, have been shown to exert their effects by targeting components of the *Mtb* FAS-II system (Daffe et al., 2019). This highlights the relevance of this system as a target for anti-TB drug discovery and development.

The acyl carrier protein (ACP)-dependent FAS-II multienzyme system catalyzes iterative elongation cycles. The third reaction step of the cycle is performed by (3R)-hydroxyacyl-ACP dehydratases (HADs). Two *Mtb* HADs have been well characterized: HadAB and HadBC (Biswas et al., 2015; Sacco et al., 2007; Slama et al., 2016). They are heterotetrameric enzymes composed of HadA and HadB, or HadB and HadC, respectively, where HadA, HadB, and HadC are all encoded by a single gene cluster (Dong et al., 2015; Sacco et al., 2007; Singh et al., 2022). We showed that HadAB mediates the early FAS-II elongation cycles common to all MA types, whereas HadBC is preferentially dedicated to the methoxy-MA pathway (Slama et al., 2016). Unexpectedly, we recently discovered an additional HAD protein, HadD, whose gene lies outside this gene cluster, and is critical for ketomycolic acid biosynthesis and *Mtb* virulence (Lefebvre et al., 2020). Overexpression of *hadD* gene in *Mtb* profoundly alters the mycolic acid profile (Lefebvre et al., 2020), while its deletion causes a sharp drop in the content of keto-MAs, deeply impacting *Mtb* physiology. It results notably in a marked impairment in biofilm formation (Lefebvre et al., 2020), an important bacterial defense mechanism recently shown to be promoted by cyclic di-GMP which, by binding to the Lsr2 receptor, leads to upregulation of *hadD* expression (Ling et al., 2024). Moreover, mice infected with *Mtb hadD* deletion mutant show a marked decrease of the bacterial loads in the lungs and spleen (Lefebvre et al., 2020), in agreement with the reported role of keto-MAs in the virulence of *Mtb* in macrophages and in mice and their impact on host immune system (Dao et al., 2008; Dubnau et al., 2000; Yuan et al., 1998). Therefore, HadD constitutes a potential target for a therapeutic antivirulence strategy and for preventing drug tolerance. For this

purpose, it is crucial to have a thorough understanding of the molecular structure and function of HadD protein.

In the context of the HadAB and HadBC heterotetramers (Dong et al., 2015; Singh et al., 2022), all three proteins (HadA, HadB, and HadC) belong to the hydratase 2 subfamily, and therefore display distinct structures when compared to the classical FabZ/FabA-type HADs found in bacterial FAS-II systems (Heath & Rock, 1996). However, only HadB contains an intact catalytic dyad and therefore catalyzes the dehydration reaction, whereas HadA and HadC accommodate the growing meromycolyl chain (Sacco et al., 2007). Based on the primary structure analysis, HadD also belongs to the hydratase 2 subfamily and features an intact catalytic dyad, suggesting that it may be catalytically active (Lefebvre et al., 2020). Here, we used biochemical, structural, and cell-based analyses to examine the structure and function of HadD and thus complete our understanding of *Mtb* FAS-II HADs. Unexpectedly, we discovered that HadD interacts with HadB to form HadBD heterotetramer, although they are encoded by genes from distinct gene clusters. We provide here a comprehensive view of the structure-function relationships of the unique HadBD enzyme, which represents an essential breakthrough for future inhibitor development.

---

## 2 | RESULTS

### 2.1 | HadD belongs to a dehydratase core within the FAS-II system

Although HadD is encoded by a distinct chromosomal region, it displays high degree of similarity with HadA, HadB, and HadC proteins. Therefore, we wondered whether HadD is a standalone protein or whether it interacts with other *Mtb* HADs. In our previous work in *Msm*, we observed that HadD<sub>*Msm*</sub> (MSMEG_0948) interacts with HadAB (Lefebvre et al., 2018). The relatively high sequence identity (58%, Figure S1) between HadD<sub>*Msm*</sub> and HadD<sub>*Mtb*</sub> (Rv0504c) proteins suggests that HadD<sub>*Mtb*</sub> might behave the same and also interact with HadAB. However, phenotypic analyses showed that HadD<sub>*Msm*</sub> and HadD<sub>*Mtb*</sub> are not strict orthologs and have distinct functions (Lefebvre et al., 2018; Lefebvre et al., 2020), suggesting that HadD<sub>*Mtb*</sub> may exhibit a distinct binding profile. To provide HadD<sub>*Mtb*</sub>-specific insights, we examined if HadD of *Mtb* complex (including among others *Mtb* and *M. bovis* species) interacts with other HAD proteins. Pull-downs were performed in the surrogate *M. bovis* BCG as previously reported for other mycobacterial protein complexes (Plocinski et al., 2014). This strain is closely related to *Mtb* both at genetic and MA content levels; HadD<sub>*Mtb*</sub> and HadD<sub>*BCG*</sub> (BCG_0547c)

have 100% sequence identity and *M. bovis* BCG synthesizes two MA types found in *Mtb* (*α* and keto-MA; Dubnau et al., 1997). Thus, we transformed *M. bovis* BCG with an expression plasmid carrying the *hadABC* operon from *Mtb* modified in order to produce HadA fused to an enhanced Green Fluorescent Protein (eGFP) tag at its N-terminus. This bait protein was co-purified with interacting proteins using anti-GFP magnetic beads and, after extensive washes, released in mild conditions. The elution fractions, analyzed by quantitative nanoLC-MS/MS after trypsin digestion, displayed as expected a strong enrichment (fold change > 10, *p*-value < 0.001) in HadB. Interestingly, the fractions were also reproducibly highly enriched in HadC and HadD (Figure 1). This suggests the existence of a dehydratase core including HadA, HadB, HadC, and HadD within the FAS-II system of *Mtb* complex species.

### 2.2 | HadD directly interacts with HadB subunit during heterologous coproduction

To see whether HadD could function as a hydratase, we examined its primary sequence more closely. Although

![Figure 1](attachment:Figure1.png)

**FIGURE 1** HadD makes strong specific interactions with HadAB from FAS-II in *M. bovis* BCG. Proteomic analysis of pull-down fractions using eGFP-HadA<sub>*Mtb*</sub> as a bait. Volcano plot showing enriched proteins in eGFP-HadA-based pull-down (PD HadA) versus control pull-down (PD Ctrl; using simple eGFP as a bait). Statistical analysis was performed from four independent biological replicates by two-sided Student's *t*-test in Proline 2.1 (Bouyssie et al., 2020). Significantly enriched proteins [fold change (PD HadA/PD Ctrl) > 2; *p*-value < 0.05] are located in the right upper panel, beyond the gray threshold lines. Highly enriched proteins [fold change (PD HadA/PD Ctrl) > 10; *p*-value < 0.001] are located beyond the magenta threshold dotted lines. The bait, eGFP-HadA, and enriched copurifying HAD proteins (HadB, HadC, and HadD) are colored in red.
HadD<sub>Mtb</sub> holds the putative catalytic dyad Asp (D37) and His (H42), its hydratase 2 catalytic motif “F-x(2)-a-x(2)-D-x(2)-P-x-H-x(5)-A” is degenerated (Figure S1). This is distinct from the catalytically active HadB<sub>Mtb</sub>, which bears a strictly conserved hydratase 2 motif “[YF]-x(1,2)-[LIVG]-[STGC]-G-D-x-N-P-[LIV]-H-x(5)-[AS]” (including the catalytic dyad D36 and H41; Sacco et al., 2007). Furthermore, HadD<sub>Mtb</sub> shares 24% and 28% sequence identity with HadA<sub>Mtb</sub> and HadC<sub>Mtb</sub>, respectively (compared to 33% between HadA<sub>Mtb</sub> and HadC<sub>Mtb</sub>). Thus, HadD<sub>Mtb</sub> is much closer to the latter two than to HadB<sub>Mtb</sub>, with which it shares only 10% identity (Figure S1). These sequence-based insights suggest that HadD<sub>Mtb</sub> may associate with HadB<sub>Mtb</sub> to form a heterocomplex similar to what was observed for HadA and HadC. To answer this question, HadB<sub>Mtb</sub> and N-terminal polyHis-tagged HadD<sub>Mtb</sub> (H-HadD<sub>Mtb</sub>) proteins were first coproduced in Escherichia coli. H-HadD<sub>Mtb</sub> was purified by cobalt-based IMAC, followed by size exclusion chromatography (SEC). SDS-PAGE analysis of the purification fractions indicated that the untagged HadB<sub>Mtb</sub> subunit copurified with H-HadD<sub>Mtb</sub> protein (Figures 2a and S2a). The elution volume during the gel filtration step suggested that the protein complex formed a tetramer in solution (Figure S2b). Small angle scattering experiments coupled with SEC (SEC-SAXS) validated a tetrameric state of the complex with an estimated molecular weight of 71 kDa, and displayed a scattering pattern and an internal distance

(a)
(b)
(c)

![Image Description]

FIGURE 2 HadD<sub>Mtb</sub> and HadB<sub>Mtb</sub> assemble into heterocomplexes in recombinant *E. coli* and in mycobacterial cells. (a) SDS-PAGE of SEC purification fraction from an *E. coli* recombinant strain coproducing H-HadD<sub>Mtb</sub> and HadB<sub>Mtb</sub> proteins. The 15% acrylamide gel was stained using Coomassie Brilliant blue. Two lanes of the same gel are displayed (see the complete gel in Figure S2a). MW, molecular weight markers. (b) Native MS of purification fractions. Left panel, multicharged spectrum; right panel, deconvoluted spectrum. Theoretical masses, excluding N-terminal Met residues, of HadB/H-HadD heterotetramer [(HadBD)<sub>2</sub>; black stars]: 70,390 Da and heterodimer (HadBD, gray circles): 35,195 Da. (c) Split-Trp assays in *M. smegmatis*. Recombinant *Msm ΔhisA* producing pairs of *Mtb* proteins fused either to the N- or C-terminal fragment of *S. cerevisiae* Trp1p protein (Ntrp, Ctrp) were diluted 1-, 10-, and 100-fold and spotted in parallel onto minimal and Trp-supplemented media. Tested protein pairs were Ntrp-HadB<sub>Mtb</sub>/HadD<sub>Mtb</sub>-Ctrp and Ntrp-HadD<sub>Mtb</sub>/HadD<sub>Mtb</sub>-Ctrp. Ntrp-Esat6/Cfp10-Ctrp and Ntrp-HadB/HadC-Ctrp protein pairs were used as positive controls (both HadB/HadC and the early secreted T-cell antigens Esat6/Cfp10 from *Mtb* form tight 1:1 complexes (O’Hare et al., 2008; Sacco et al., 2007). Ntrp-HadB/Cfp10-Ctrp and Ntrp-Esat6/HadD-Ctrp pairs were used as negative controls. Images were taken after a 3-week incubation at 30°C.distribution consistent with a tetramer (Figure S2c). Moreover, native MS (nMS) analysis clearly showed the presence of a heterotetramer $(\text{HadBD}_{Mtb})_2$ at the expected molecular weight (MW) of 70.4 kDa (Figure 2b). We also observed the presence of a tiny amount of heterodimer $\text{HadBD}_{Mtb}$ (MW of 35.2 kDa), which is most likely due to partial gas phase dissociation of the tetramer in the mass spectrometer.

Altogether, these experiments demonstrate that, when $hadD$ and $hadB$ are heterologously coexpressed, $\text{HadD}_{Mtb}$ protein associates with $\text{HadB}_{Mtb}$ to form a heterotetramer in solution, which is reminiscent of the quaternary structures observed for both HadAB and HadBC (Dong et al., 2015; Sacco et al., 2007; Singh et al., 2022).

### 2.3 | HadB<sub>Mtb</sub> and HadD<sub>Mtb</sub> tightly interact in mycobacterial cells

To address whether $\text{HadB}_{Mtb}$ and $\text{HadD}_{Mtb}$ interactions are physiologically relevant and rule out the possibility that $\text{HadD}_{Mtb}$ forms catalytically competent homodimeric or homotetrameric enzyme, we employed protein–protein interaction (PPI) sensing experiments in whole cells based on the Split-Trp method adapted to the nonpathogenic rapidly growing *Msm* species (O'Hare et al., 2008) and validated with *Mtb* HADs (Sacco et al., 2007). The tryptophan (Trp) auxotrophic *Msm ΔhisA* strain was cotransformed with a pair of plasmids, each one encoding a target protein ($\text{HadB}_{Mtb}$ or $\text{HadD}_{Mtb}$) in fusion with either the N-terminal fragment (Ntrp) or the C-terminal fragment (Ctrp) of *Saccharomyces cerevisiae* protein Trp1p involved in Trp biosynthesis. If an in vivo interaction occurs between the target proteins encoded by the pair of plasmids, Trp1p function is restored and the recombinant strain grows without exogenous Trp supply. $\text{HadB}_{Mtb}$-$\text{HadC}_{Mtb}$ dehydratase and Esat6-Cfp10 pathogenicity protein complexes from *Mtb*, whose PPIs were previously validated through this Split-Trp method (O'Hare et al., 2008; Sacco et al., 2007), were taken as positive controls. *Msm ΔhisA* producing the protein pair Ntrp-$\text{HadD}_{Mtb}$/HadD<sub>Mtb</sub>-Ctrp did not grow on the medium depleted in Trp, whereas the strain producing the pair Ntrp-$\text{HadB}_{Mtb}$/HadD<sub>Mtb</sub>-Ctrp displayed a strong growth on this medium (Figure 2c). This growth was more pronounced than with the dehydratase pair Ntrp-$\text{HadB}_{Mtb}$/$\text{HadC}_{Mtb}$-Ctrp and similar to that observed with the control pair Ntrp-Esat6/Cfp10-Ctrp. Thus, these experiments provide validation that the interactions between $\text{HadB}_{Mtb}$ and $\text{HadD}_{Mtb}$ we observed when they are heterologously expressed also occur in mycobacteria. Furthermore, we saw no evidence for $\text{HadD}_{Mtb}$ homo-complex formation in mycobacteria. Taken together, this

suggests that heterotetramer $(\text{HadBD}_{Mtb})_2$ is the physiologically relevant species. As the following part of the manuscript is focused on the enzymes of *Mtb*, $\text{HadB}_{Mtb}$, and $\text{HadD}_{Mtb}$ will be simply named HadB and HadD, respectively.

#### 2.4 | HadBD exhibits distinct structural features when compared to HadAB and HadBC

To provide further insights, we solved the crystal structure of the HadBD complex. Each HadBD heterodimer displays a global double-hotdog-like structure highly similar to that observed for HadAB and HadBC, both protomers arranged head-to-tail exhibiting a canonical single hotdog (SHD) fold that includes a central helix (the “hot-dog helix”) wrapped into a five-stranded antiparallel β-sheet and an overhanging segment (Figure 3a). The heterotetramer $(\text{HadBD})_2$ observed in the crystal structure is formed by relating two HadBD heterodimers by a crystallographic 2-fold symmetry axis (Figure 3b). This quaternary structure is reminiscent of the HadAB and HadBC heterotetrameric structures (Figure S3). Based on PDBePISA server calculations (Krissinel & Henrick, 2007), HadB and HadD protomers interact with each other via a large buried surface area of 1363 Å² to form a heterodimer (Figures 3a,b and S4). In contrast, the heterotetramer is formed via limited interface areas between two HadB protomers (514 Å²) from each heterodimer on one side, and HadD protomers (492 Å²) on the other side (Figure 3b). Although heterodimer formation involves a larger total interface, the heterotetramer is the predominant species in solution, as discussed above (Figures 2b and S2b,c).

Next, we compared tertiary structures of HadB and HadD (this work), and previously reported HadA (Dong et al., 2015) and HadC (Singh et al., 2022) structures (Figure 3c). The extra C-terminal tail (12–21 residues) in the HadA, HadC, and HadD protomers compared to HadB (Figure S1) was largely not visible in the different structures, strongly suggesting that this region was disordered. Overall, all four proteins are structurally similar, with RMSD values (based on Ca atoms) in the 2.3–2.7 Å range for HadB versus all other Had subunits, and between 1.0 and 1.3 Å when comparing HadA, HadC, and HadD. Additionally, HadB subunits from the HadAB, HadBC, and HadBD structures superpose with a RMSD of 0.5 Å, suggesting that complex formation does not lead to conformational changes within HadB.

Although Had subunits share the same overall fold formed of the central five-stranded β-sheet and three
(a)
overhanging
segment

hotdog helix

(b)
hotdog helix

overhanging
segment

(c) Missing residues: 147-158
Missing residue: 142
Missing residues: 145-158
Missing residues: 152-166

HadA<sub>HadAB</sub> (PDB 4rlj)
HadB<sub>HadBD</sub>
HadC<sub>HadBC</sub> (PDB 5zy8)
HadD<sub>HadBD</sub>

(d)

FIGURE 3 Comparison of HAD structures highlights distinct structural features in HadBD. In HadAB (PDB 4RLJ), HadA is colored yellow-orange; HadB dark gray; in HadBC (PDB 5ZY8), HadB is colored light gray, HadC pink; in HadBD (this work, PDB 8PWZ), HadB is colored gray, HadD light green. (a) Superposition of crystal structures of the mycobacterial FAS-II HADs. View of the heterodimers. The three heterodimers display similar global double-hotdog-like structures. (b) View of the heterotetrameric HadBD assembly. (See Figure S3 for the superposition with the heterotetrameric structures of HadAB and HadBC). (c) and (d) Comparison of HadA<sub>Mtb</sub>, HadB<sub>Mtb</sub>, HadC<sub>Mtb</sub>, and HadD<sub>Mtb</sub> tertiary structures. Sequence lengths: HadB, 142 residues; HadA, 158 residues; HadC and HadD, 166 residues. The crystallized form of HadC was truncated from its C-terminal end. C-terminal parts of HadA, HadC, and HadD are missing in the structures (see Figure S1). Secondary structure labeling is the same as in Figure S1. (c) Ribbon representation of the tertiary structures colored by average B-factor values. Main chain atoms are colored from blue to white to red in B-factor ascending order. The same limits for color ramping were used, which correspond to the extreme values (i.e., 15 and 141 Å²) found in the PDB files of HadAB, HadBC, and HadBD (taking all chains into account). (d) Average B-factor values for main chain atoms versus residue. The color code of the different subunits is the same as in (a) and (b). When appropriate, continuous and dashed lines are used for the different chains in the asymmetric unit. The horizontal lines represent the overall average B value (computed over all residues for each chain).

α-helices, they exhibit notable differences, such as additional 3<sub>10</sub> helices (ηA in HadA, ηC in HadA and HadB) and β-strands, as well as different lengths for the β-strands and connecting loops (Figures S1 and 3c). One striking difference is the shorter α3 helix found in HadD, which makes a single turn—compared to four turns in

BORIES ET AL.

(a)

| Specific activity V1 (μmol/min/mg) |
| --- |
| 1.2 |
| 1.0 |
| 0.8 |
| 0.6 |
| 0.4 |
| 0.2 |
| 0.0 |

C12:1-CoA C16:1-CoA

Substrate

(c)

| Specific activity V1 (μmol/min/mg) |
| --- |
| 1.2 |
| 1.0 |
| 0.8 |
| 0.6 |
| 0.4 |
| 0.2 |
| 0.0 |

HadB-HadD WT-WT H41F-WT WT-H42F WT-H42Q H41F-H42F

HadBD form

(b)

| Relative intensity (%) |
| --- |
| 100 |
| 50 |
| 0 |

control

[M+H]+ [M+Na]+ [M-H+2Na]+ [M-2H+3Na]+ [M-3H+4Na]+

948.10 970.09 992.07 1014.06 1036.06

| Relative intensity (%) |
| --- |
| 100 |
| 50 |
| 0 |

+HadBD

[M+Na]+ [M-H+2Na]+ [M-2H+3Na]+ [M-3H+4Na]+

970.09 988.11 992.07 1010.08 1014.05 1032.06 1036.06 1054.06

Mass (m/z)

FIGURE 4 HadBD forms an active enzyme where HadB is required for catalysis. (a) and (c) Enzyme activity assays monitored by spectrophotometry. (a) HadBD substrate specificity. (b) MALDI-TOF MS analysis of HadBD product. Assays were done in the presence of trans-2-C12:1-CoA. Ions peaks of the hydration product, the 3-hydroxydodecanoyl-CoA, were detected in reaction media only in the presence of HadBD (WT) but not in the control assays (without protein). The substrate ion peaks (proton and mono- to tetra-sodium adducts) are labeled in black and the hydration product ion peaks (mono- to tetra-sodium adducts) are labeled in red. (c) Activities of wild-type (WT) HadBD and variants mutated in the catalytic histidine residue of HadB and/or the equivalent residue in HadD. Data are means ± average deviations from triplicates. Significant statistical difference compared with specific activity in the presence of C12:1-CoA (a) or with HadBD WT-WT specific activity (c): **p-value < 0.01, *p-value < 0.05; ns: no statistical difference (p-value > 0.05).

the case of HadA, HadB, and HadC—followed by a series of hydrogen bonded turns and bends (Figure 3c). Given that helix α3 (i.e., the hotdog helix) partially lines the substrate-binding tunnel in HADs, its distinct features in HadD might be functionally relevant. Another important feature is the large variations observed in the average temperature factors (B-factors) for the different structures (Figure 3d). The lower B-factor values for HadA and HadB protomers from HadAB (average values of 29 and 26 Å², respectively) indicated that both HadA and HadB in the HadAB structure are globally more ordered as reflected by the higher resolution limit obtained (1.7 Å; Dong et al., 2015) compared to HadBC (2.9 Å; Singh et al., 2022) and HadBD (2.0 Å), at similar crystallization temperatures (289 K, 298 K and 285 K for HadAB, HadBC, and HadBD, respectively). Taken together, the comparison of available structures revealed that HadB retains the same structure regardless of the binding partners, thus suggesting that any differences in activity or substrate preference are due to the distinct features of HadA, HadC, and HadD, such as the hotdog helix and protein flexibility.

2.5 | HadBD forms an active enzyme where HadB is the catalytic subunit

To verify that HadBD is an enzymatic entity, we used the well-established in vitro assay that employs enoyl derivatives as model substrates (Heath & Rock, 1996; Sacco et al., 2007). Kinetic activity assays were performed in the presence of trans-2-dodecenoyl-CoA (trans-2-C12:1-CoA) or trans-2-hexadecenoyl-CoA (trans-2-C16:1-CoA), which have previously been validated as surrogate substrates for HadAB and HadBC (Sacco et al., 2007). Data showed that HadBD was active on both substrates (Figure 4a). The reaction led to an increase of 18 mass units, detected by MALDI-TOF MS, in agreement with the expected mass for the hydration product (Figure 4b). Interestingly, HadBD activity was 10 times higher in the presence of the longest enoyl-CoA (Figure 4a). This is consistent with the involvement of HadD in late elongation cycles during mycolic acid biosynthesis, implying the transformation of ultra-long-chain natural molecules (Lefebvre et al., 2020).

Structural alignment revealed that the catalytic dyad of HadD (D37 and H42) superimposes perfectly with that of HadB (D36 and H41; Figure 5a, Figure S1), which suggests the possibility that HadD is also enzymatically active. However, out of three additional residues of HadB that have been proposed to be involved in the stabilization of the anionic intermediate (A57, G59, and N38) in HadAB (Biswas et al., 2015; Dong et al., 2015) (Figure 5b), only the alanine is conserved in HadD (A58; Figures 5a,b and S1). Furthermore, the straight hotdog helix of HadB forms a positive macrodipole culminating on G59 (Figure 5c), thus likely stabilizing the negative charge of the substrate transition state (Figure 5b). As mentioned, the hotdog helix in HadD is kinked and significantly shorter, so the corresponding residue in HadD, L60, cannot have this stabilizing effect (Figures 5c and 3c). Finally, the catalytic residues in HadB lie in a wide crevice that is easily accessible to the substrate, whereas the accessibility of the catalytic dyad in HadD is significantly restrained (Figure 5d). These structural features

FIGURE 5 HadB and HadD structure comparison suggests that HadB is the catalytic subunit. (a) HadD (pale green) and HadB (gray) protomers aligned via their putative catalytic residues. Catalytic residues of HadB (D36 and H41) and their equivalent in HadD (D37 and H42) are displayed as sticks and residues possibly involved in the oxyanion hole (A57, G59, and N38 of HadB; A58, L60, and H39 of HadD) are in orange. (b) Proposed oxyanion hole for the stabilization of the anionic intermediate. (c) Hot-dog helices of HadD and HadB with the ending residue (L60 and G59, respectively). G59 in HadB takes part in a positive macrodipole, in contrast to L60 in HadD where the hotdog helix is very short and kinked. (d) Representation of the solvent-accessible surface around HadB (gray) and HadD (light green). HadBD displays a large cavity around residues D36 and H41 of HadB while residues D37 and H42 of HadD, in close contact with HadB, are hardly accessible. There is a rotation of approximately 180° along the vertical axis between both views.
led us to hypothesize that HadB is the only catalytic subunit in the HadBD complex.

To verify this hypothesis, we generated mutant isoforms of HadBD, where the catalytic histidine in HadB (H41) and/or the corresponding residue in HadD (H42) were replaced by a Phe or a Gln residue, two substitutions that are classically used to modify the catalytic histidine residue in dehydratase domains/proteins (Faille et al., 2017; Fiers et al., 2016). We heterologously produced and purified these complexes using the same conditions as those used for the wild-type (WT) HadBD complex. The catalytic activities of the purified HadBD mutant forms were then compared to that of the WT complex in the presence of trans-2-C16:1-CoA. Although an important decrease of the protein thermostability, with a drop of about 12°C in melting temperature ($T_{\mathrm{m}}$), was observed by differential scanning fluorimetry (DSF) upon mutation of H42 in HadD (Figure S5), both WT-H42F and WT-H42Q HadB-HadD variants displayed activities close to that of the WT enzyme (Figure 4c). In contrast, both H41F-WT and H41F-H42F HadB-HadD forms displayed no detectable activity (Figure 4c). This total loss of activity due to HadB H41F mutation is unlikely to be linked to a change in protein structure and stability, as shown by the relatively moderate $T_{\mathrm{m}}$ shift (3.3°C) observed (Figure S5). Altogether our data provide evidence that HadB constitutes the catalytic subunit of the HadBD enzyme while HadD most likely plays the role of the long chain substrate-binding subunit, similarly to what is observed for HadAB and HadBC dehydratases (Biswas et al., 2015; Dong et al., 2015; Sacco et al., 2007; Singh et al., 2022).

### 2.6 | Molecular determinants of the HadBD CoA and ACP binding

We used molecular modeling to map out the mode of substrate binding potentially employed by HadBD. Thus, a model of HadBD in interaction with a CoA derivative was first generated by using the structure of Candida tropicalis 2-enoyl-CoA hydratase 2:(3R)-hydroxydecanoyl-CoA (HDC) complex (PDB 1PN4; Koski et al., 2004) as a template. We observed, after the superposition with the HadB protomer, that HDC molecule would fit reasonably well within HadBD structure (Figure S6a,b). This model suggested that R86 and F87 from HadB and F88 from HadD would be key for interaction with the adenine and phosphopantetheine moieties of CoA (Figure S6b). In addition, R90 of HadD might be involved in the stabilization of the solvent-exposed 3′-phosphate group of the ribose, a stabilization fulfilled through a salt bridge with K103 in C. tropicalis hydratase 2 (Koski et al., 2004).

Furthermore, in the model (Figure S6b), the G59 main-chain amine and the N38-Nε2 side-chain atom of HadB appear well positioned to stabilize the substrate transition state during catalysis (Figure 5b). We also modeled the interaction between HadBD and an ACP derivative based on the homodimeric structure of E. coli FabA:C9-holo-ACP complex (PDB 4KEH; Nguyen et al., 2014). We observed that the acidic surface of the ACP protein could potentially be stabilized by electrostatic interactions with a positively charged patch at the HadBD surface (Figure S6c), delineated by R86, R107, K109, K130, and R134 of HadB, and R90, K93, and H119 of HadD (Figure S6d). It is noteworthy that R113 and R145 from HadA in HadAB, which were assumed to interact with the negatively charged residues from AcpM (Dong et al., 2015), the endogenous ACP involved in Mtb FAS-II, are not conserved in HadD (replaced by I116 and E148) nor in HadC (replaced by D113 and Q145; Figure S1). They might be compensated in HadD by residues R90 and R86, respectively, located in the same region, and in HadC by four positively charged residues (R152, K154, K157, R164) situated in the C-terminal end and playing a role in AcpM stabilization (Figure S1; Singh et al., 2022). Thus, taken together, our models highlight structural features of HadBD compatible with both CoA and ACP engagement.

#### 2.7 | Enhanced flexibility and accessibility of the substrate-binding crevice in HadBD

Given our previous work on HadD, HadBD is expected to synthesize ultra-long fatty acyl chains (Lefebvre et al., 2020). From our modeling, we observed that the C10 acyl chain of HDC can fit in a crevice at the interface between HadB and HadD, close to the catalytic dyad (Figure 6a). This strongly suggests that this crevice would accommodate the 10–12 carbon proximal part of the growing meromycoloyl chain. The opening of this crevice is defined by two loops, namely loop α3-β2 downstream the hotdog helix and loop β3-β4, corresponding to poorly ordered segments, that is, regions that display the highest B-factor values in HadD (Figure 3c,d). Importantly, compared to HadA which handles substrates with shorter chain lengths (Lefebvre et al., 2020; Sacco et al., 2007), HadD features an almost complete opening of the substrate-binding crevice due to the displacement of the loop α3-β2 (Figure 6a). In HadC, which accommodates ultra-long acyl chains like HadD (Lefebvre et al., 2020), loop α3-β2 is even more flexible than in HadD and consequently not visible in the structure (Figure 6a). Furthermore, in both HadBD and HadBC structures, the
FIGURE 6 The wider opening of HadBD and HadBC substrate-binding crevices compared to HadAB. The PDB codes and color codes of the structures are as shown in Figure 3a,b. (a) Crevice likely accommodating the proximal part of the growing meromycolyl chain. Superposition of heterodimeric crystal structures of the mycobacterial FAS-II HADs. For greater clarity, only the HadB subunit of HadBD is shown. The HDC molecule (colored by atom type with carbons in cyan: acyl chain, and orange: coenzyme A) was modeled in the crevice based on the structural superposition with the *Candida tropicalis* 2-enoyl-CoA hydratase:HDC complex (PDB 1PN4). Catalytic residues are colored in yellow. (b) A second dead-end cavity in the putative substrate-binding crevice of HadBD. The surface of this cavity is colored in hot pink. The PEG molecule (colored by atom type with carbons in green) bound in the bottom cavity of the crevice in HadAB fits well in the corresponding cavity in HadBD, while the HDC molecule (colored as in a) lies above in the crevice.

region around helix α2 (I40–A55) of HadB that borders the other side of the crevice opening is also disordered, suggesting that this region is also flexible (Figure 6a, Figure 3c,d), which might also facilitate the binding of ultra-long chain substrates. Additionally, as mentioned, the region of the hotdog helix, upstream of the loop α3–β2 and partially lining the substrate-binding crevice (Figure 6a), is another point of difference between HadA and HadC/HadD. In HadD, this helix is uniquely short, whereas in HadC it is more flexible (Figure 3c,d). Collectively, the increased openness and flexibility of the substrate-binding crevice in HadBC and HadBD are most likely determinants that facilitate the loading of the longest-size growing meromycolyl chains.

We then searched for solvent-protected sites that might allow further elongation of the meromycolyl chain. A closer look at the mycobacterial HAD structures reveals that the substrate-binding crevice in HadAB, HadBC, and HadBD is in fact bidentate, with the second narrow, dead-end cavity that could accommodate chains ranging from 6 to 12 carbon atoms in length, depending on the structure (Figure 6b). Interestingly, this cavity was found to accommodate either a PEG molecule (Figure 6b) or flavonoid inhibitors in different HadAB structures (Dong et al., 2015). However, it is unlikely that a complete C<sub>48</sub>–C<sub>62</sub> meromycolyl chain could fit into this bidentate crevice. Interestingly, we identified two structural features that might be involved in accommodating the distal part of the lipid intermediates during elongation. One is a long, sinuous channel that passes through the entire HAD heterotetramer, making the junction between the two substrate-binding crevices of

both heterodimers (Figure 7a). While this channel is continuous in HadBC, in both HadBD and HadAB, the entrance into the second heterodimer takes place through a charged and narrow neck and would require possible local rearrangements to adjust the ultra-long, apolar chain. The second feature is the occurrence of hydrophobic areas at the surface of the mycobacterial HADs (Figure 7b). In HadBD, we observed an apolar patch located on HadD surface, next to the HDC methyl end, at the exit of the substrate-binding crevice. It is flanked by the C-terminal part of helix α3 and the N-terminal part of strand β2, and includes 13 residues (L72, I74, F75, F78, I80, I82, I84, V87, M145, L146, G147, A150, and H151) that colocalize to create a large region of high hydrophobicity (Figure 7b, inset). This region and other apolar patches are also found in HadAB and HadBC (Figure 7b), although slightly distinct between the three enzymes. Given their hydrophobic character and size, these areas might be involved in the stabilization of a putatively solvent-exposed moiety of the

HadAB                                                                 HadBC                                                                 HadBD
(a)
![Image](https://via.placeholder.com/300x300)
(b)
![Image](https://via.placeholder.com/300x300)

FIGURE 7 Putative accommodating sites for the distal part of the meromycolyl chain in HADs. The different protomers are colored as in Figure 3 (HadA: yellow-orange, HadC: pink, HadD: pale green; HadB in light gray, dark gray, or gray depending on the considered heterotetramer). (a) Heterotetramer-crossing channel. Cavity volumes, as calculated by PyMOL software, are colored in hot pink. A long channel crossing the whole heterotetramer is observed in HadBC. A similar but discontinuous channel is observed in HadAB and HadBD. Pink arrows indicate the channel entrance and exit. (b) Surface apolar patches near the substrate-binding crevice exit. Electrostatic potential of the heterodimeric surface, as calculated by PyMOL, of the models of each heterotetramer in complex with the (3R)-hydroxydecanoyl-CoA (HDC; the acyl moiety is represented as sticks and colored by atom type with carbons in cyan). The second heterodimer is drawn as a light blue cartoon. Apolar surfaces are shown in white, the positive charges are in blue and the negative charges are in red. The red and sky blue dashed lines indicate the putative paths of the distal part of the meromycolyl chain through apolar areas. Inset: zoom on the apolar patch located at the exit of the substrate-binding crevice; the hydrophobic residues are indicated except for G147, not visible in this orientation. The PEG molecule as observed at the surface of HadAB crystal structure (PDB 4RLJ) is colored by atom type with carbons in green.
meromycolyl chain. Supporting this hypothesis, previously solved HadAB structure includes a PEG molecule located on an apolar surface, at about 24 Å from the exit of the substrate-binding crevice. This PEG molecule might mimic a portion of the C₃₈–C₅₂ solvent-exposed meromycolyl chain (Figure 7b). Thus, taken together, the substrate-binding crevice, the PEG binding region and the apolar patches between the two might accommodate the entire length of the meromycolyl chain (Figure 7b).

HadC, or HadD, suggesting that its role as a catalytic subunit is unchanged.

We also mapped features that potentially determine substrate binding specificity in HadBD, both in terms of polar head and chain length. Importantly, the structural model of the HadBD:acyl-ACP complex suggests that the AcpM moiety, which transfers the growing meromycolyl chain between the different enzymes during FAS-II elongation cycles, would bind to a positively charged patch on the HadBD surface, which is in agreement with previous studies of HadAB and HadBC, as well as FAS systems (Biswas et al., 2015; Finzel et al., 2015; Leesong et al., 1996; Lou et al., 2019; Musayev et al., 2005; Singh et al., 2022; Zhang et al., 2016). Regarding chain length specificity, HadAB is involved in the early FAS-II elongation steps sufficient for the biosynthesis of α-meromycolyl chains (C₄₈–C₅₂), a distinct chain length preference from HadBD and HadBC that are involved in late FAS-II cycles leading to the production of the longer-chain oxygenated-meromycolyl chains (C₅₄–C₆₂; Lefebvre et al., 2020). In agreement with these differences, we observed that HadBD structure more closely resembles HadBC than HadAB. The main point of difference is the presence of an open and flexible putative substrate-binding crevice in HadBC and HadBD, which seems to be narrower in HadAB. Specifically, the larger opening of the substrate-binding crevice is due to the positioning of the loop downstream of the hotdog helix and an increased flexibility of the hotdog helix in HadC and HadD. The hotdog helix is also kinked and drastically shorter in HadD, further contributing to the open nature and flexibility of the substrate-binding crevice. This is in agreement with the role of the hotdog helix as a determinant of the substrate accommodation capacity described for some hotdog family members. Indeed, a reduced size of the central helix was also described for enoyl-CoA hydratase domain/proteins binding bulky substrates such as the eukaryotic multifunctional enzymes type 2, required for the peroxisomal β-oxidation of very-long-chain fatty acids and the synthesis of bile acids, and the Mtb ChsH1–ChsH2 heterotetramer involved in cholesterol catabolism (Koski et al., 2004, 2005; Yang et al., 2014). In HadBD, the hotdog helix is even shorter than in the latter enzymes. To the best of our knowledge, HadBD is the first HAD enzyme described with a truncated hotdog helix. Surprisingly, HadC has a hotdog helix only slightly shorter (14 residues) than the standard length (16 residues) found in the enoyl-CoA hydratase from Aeromonas caviae, AcRH, that is specific for very short (4–6 carbon long) substrates (Hisano et al., 2003). Therefore, in HadBC, the absence of truncated hotdog helix is most likely compensated for by the greater flexibility and distinct orientation of the HadC

3 | DISCUSSION

Deciphering the structure-function relationships of HadD from *Mtb* is of paramount importance since this protein plays an important role in the virulence of one of the world's most successful pathogens (Lefebvre et al., 2020). Although HadD is encoded by a chromosomal region distinct from that of the other HAD subunits of the FAS-II system, that is, HadA, HadB, and HadC, we showed here that HadD exhibits direct and tight interactions with HadB. Therefore, our studies rule out the possibility that HadD exists as a standalone protein. Rather, we present evidence that HadD and HadB assemble to form a heterotetramer with a quaternary structure similar to those described for the two other mycobacterial FAS-II dehydratases, HadAB and HadBC (Dong et al., 2015; Sacco et al., 2007). Interestingly, our pull-down experiments combined with proteomic analysis suggest that the FAS-II system from *Mtb* complex holds a dehydratase core that would contain a combination of HAD enzymes. This will be the subject of future investigation.

Our kinetic assays demonstrated that the HadBD heterotetramer possesses a trans-2-enoyl hydratase activity like the other HADs when separated from their FAS-II complex (Sacco et al., 2007). We also showed evidence that HadB is the catalytic subunit, and that HadD plays a role in substrate binding despite having an intact catalytic dyad. This suggests that factors other than the conserved dyad may be important for catalysis and we document some key differences between HadB and HadD that may contribute to this: (1) the degenerate nature of the hydratase 2 motif in HadD; (2) the significantly lower accessibility of the catalytic dyad in HadD; and (3) a severely shorter hotdog helix in HadD. The latter point was found to induce a loss of catalytic activity for enoyl-CoA hydratase subunits, likely correlating with a reduced ability to stabilize the reaction intermediate (Koski et al., 2004; Yang et al., 2014). Furthermore, our structural analysis revealed that HadB assumes the same conformation regardless of whether it is bound to HadA,
hotdog helix (Singh et al., 2022) as well as the greater flexibility of the downstream α3–β2 loop. Another factor that may play a role in acyl chain length specificity is the three-dimensional structure of the substrate. For example, it has been shown that different MA classes can adopt distinct folded conformations, depending on the functional groups held by their meromycolyl chain (Groenewald et al., 2014; Villeneuve et al., 2013). One may hypothesize that the higher flexibility and wider opening of the substrate-binding crevice in HadBD and HadBC (involved in keto- and methoxy-MA synthesis) would promote the accommodation of a completely or partially folded oxygenated chain. In contrast, the narrower and more rigid channel of HadAB (involved in α-MA synthesis) would selectively bind a fully extended chain.

Despite increased openness and flexibility, the size of the crevice is unexpectedly short and able to accommodate only a 10-to 12-carbon-long chain, which is far less than the full meromycolyl chains. It cannot be ruled out that the crystal structure of the apo form of HadBD does not fully represent the structure of the enzyme bound to its natural substrate. However, the presence of short binding pockets has already been observed for other enzymes involved in the biosynthesis of mycolic acids (Bergeret et al., 2012; Kim et al., 2023; Li et al., 2015; Musayev et al., 2005), and may represent a conserved feature that is compensated by other mechanisms. Our structural and modeling analysis suggests two additional features that may be involved in the stabilization of the large remaining portion of the growing meromycolyl chain: (1) a long sinuous channel that crosses the entire heterotetramer; and (2) a large apolar surface patch in the vicinity of the substrate exit, which may play a role in stabilizing the part of the solvent-exposed meromycolyl chain via hydrophobic interactions. However, at this point we cannot rule out the possibility that additional factors, such as distinct proteins of the FAS-II system or the plasma membrane contribute to the stabilization of the solvent-exposed lipid chain. Indeed, an array of data suggest that the MA biosynthesis machinery could be associated with the inner leaflet of the plasma membrane, which would facilitate the export of mycolic acid derivatives toward their final destination, the mycomembrane (Kim et al., 2023; Quémard, 2016).

In conclusion, this work provides evidence that HadD does not operate as a standalone HAD but functions in the context of the HadBD heterotetramer, despite being encoded by a gene distant from the hadA-C cluster. Furthermore, although the presence of the intact catalytic dyad led to expectation that HadD subunit could drive catalysis, our data show that HadBD

enzymatic activity is due to HadB and that HadD plays a role in binding specificity. We highlight the unique structural features of HadBD linked to its very specific function leading to the production of the extremely long-chain keto-MAs. These insights will inform future efforts to design inhibitors of HadBD, which, given the role of HadD in Mtb virulence and biofilm formation (Lefebvre et al., 2020; Ling et al., 2024), may offer innovative alternatives and/or complement to the classical anti-TB antibiotics, expanding efficacy, and thwarting drug resistance and tolerance.

4 | MATERIALS AND METHODS

4.1 | Pull-downs

For pull-down experiments using eGFP-HadA enzyme as a bait, a *M. bovis* BCG Pasteur (ATCC 35734) strain was transformed by the plasmid pKW08::egfp-hadABC<sub>Mtb</sub>. The latter was constructed by cloning the *hadABC* (*Rv*0635-*Rv*0636-*Rv*0637) operon of *MtbH37Rv* (ATCC 27294) in the tetracycline-inducible vector pKW08::egfp (kindly provided by Andrzej Dziembowski; Plocinski et al., 2014; Williams et al., 2010) between *BamH*I and *Hind*III sites, downstream of the enhanced Green Fluorescent Protein (eGFP) tag coding sequence followed by the Tobacco Etch Virus (TEV) cleavage site. The control strain used for these experiments was *M. bovis* BCG/pKW08::egfp. The resulting strains were grown in Middlebrook 7H9 broth (Difco) supplemented with 50 μg/mL hygromycin, 10% (v/v) ADC and 0.2% (v/v) glycerol, and the production of the proteins was increased by adding 20 ng/mL tetracycline (Sigma) for 48 h during the exponential phase. The affinity purification method used was adapted from a previously published protocol (Lefebvre et al., 2018). Bacteria were centrifuged at 4000 g for 10 min at 4°C, and resuspended in lysis buffer: 50 mM potassium phosphate buffer pH 7.6, 150 mM NaCl, 2 mM AEBSF (Euromedex), 1.7 mM dodecyl β-D-maltoside, 5 μg/mL DNase I and 5 μg/mL RNase A. Bacteria were lysed with extraction beads using a Biorupteur Pico (Diagenode) for 30 cycles (30 son, 30 soff). The lysate was clarified by centrifugation at 13,000 g for 30 min at 4°C and filtered through 0.2 μm filters, then incubated in the presence of GFP-Trap®_MA magnetic beads (Chromotek) for 2 h at 4°C under agitation. After four washes with 50 mM Tris buffer pH 7.6, 150 mM NaCl, 0.1% (v,v) Triton X-100, proteins were eluted by cleavage using 10 units AcTEV™ protease (Fisher) in 50 mM Tris buffer pH 8.0, 150 mM NaCl, 0.5 mM EDTA, 1 mM DTT, overnight at 4°C.

4.2 | Proteomic analysis

Sample preparation and nanoLC-MS/MS for proteomic analysis were performed as previously described (Duguet et al., 2017). Briefly, after reduction and alkylation of cysteine residues, eluted proteins were concentrated in one band by 12% SDS-PAGE. The proteins were then digested with trypsin (1 μg trypsin/50 μg proteins) overnight at 37°C. The resulting peptides were extracted from the gel, then dried, and finally resuspended in 2% acetonitrile, and 0.05% trifluoroacetic acid. Peptides were analyzed by nanoLC-MS/MS using an UltiMate 3000 RSLCnano system (Dionex, Amsterdam, The Netherlands; 5%-25% gradient for 75 min then 25%-50% gradient for 30 min of 80% acetonitrile, 0.2% formic acid at 300 nL/min flow rate) coupled to an LTQ-Orbitrap Velos mass spectrometer (ThermoScientific, Bremen, Germany).

Raw MS files were analyzed by MS-angel and Proline (Bouyssie et al., 2020) softwares. Data were searched with the MASCOT search engine against *M. bovis* BCG Pasteur 1173P2 of the UniProtKB protein database (last update 2021-04-16, 3895 entries) and a list of potential contaminant sequences provided in Proline. Bioinformatics data analysis was performed as described previously (Duguet et al., 2017). The dataset contains MS results from the analysis of four biological replicates. The statistical proteomic analysis was performed on the LFQ intensities from the “abundance” columns of Proline table. The MS proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE (Perez-Riverol et al., 2019) partner repository with the dataset identifier PXD043977.

For co-production of HadB<sub>Mtb</sub> and N-terminal His<sub>6</sub>-tagged HadD<sub>Mtb</sub> or the corresponding mutant proteins, *E. coli* BL21(DE3) strain (Novagen) was transformed by the different combinations of pCDFDuet-1::hadB and pET-15b::hadD constructs. The resulting strains were grown in a 1 L culture at 37°C under shaking (180 rpm) in LB supplemented with carbenicillin (50 μg/mL) and streptomycin (50 μg/mL). At an OD<sub>600</sub> of 0.6, gene expression was induced by the addition of 0.5 mM (WT, simple mutant, and HadBH41F-HadDH42Q strains) or 0.1 mM (HadBH41F-HadDH42F) IPTG, and bacteria were grown further under shaking (180 rpm) for 4 h at 30°C (WT strain) 15 h at 20°C (all mutant strains). After centrifugation, the cell pellet was washed in 25 mM Tris buffer, 150 mM NaCl, pH 8.0, and frozen at –80°C. The latter was resuspended in 50 mM Tris buffer, 300 mM NaCl, pH 8.0 plus 5% glycerol (buffer A). After addition of 1 mM AEBSF, 5 μg/mL DNase I, 5 μg/mL RNase A, and 100 μg/mL lysozyme, bacteria were lysed by sonication with a Branson Sonifier Cell Disrupter B15 (four 30-s cycles, microtip 5, 50% duty cycle) or an Emulsiflex C5 Homogenizer (Avestin; 500–1000 psi, 3 times). After centrifugation at 18,500 g for 20 min at 4°C, the supernatant was loaded onto 0.5 mL TALON® superflow metal affinity resin (Clontech) equilibrated with buffer A plus 10 mM imidazole. After extensive washes in the same buffer then in buffer A plus 28 mM imidazole, the HadB/H-HadD protein complex was eluted at 200 mM imidazole. The elution fractions were pooled and dialyzed (MWCO 6–8 kDa) twice for 30 min, in 50 mM Tris/HCl pH 8.0, 150 mM NaCl, and 5% glycerol (buffer B), then concentrated by ultrafiltration on a Vivaspin unit (MWCO 10 kDa). The resulting protein solution was loaded onto an analytical S200 gel filtration column equilibrated with buffer B. The molecular weights corresponding to the elution peaks were estimated based on the column calibration curve provided by the manufacturer. The elution fractions containing HadBD heterotrimer were pooled, concentrated by ultrafiltration on a Vivaspin unit (MWCO 10 kDa), and stored at –80°C after cryofreezing.

4.3 | Protein production and purification

The *Rv0504c* (*hadD*) gene from *Mtb* H37Rv (ATCC 27294) was amplified by PCR and the product was ligated into pET-15b vector (Novagen) between NdeI and XhoI restriction sites, downstream of the poly-His tag coding sequence; the absence of mutation was verified by sequencing. The *Rv0636* (*hadB*) gene from *Mtb* H37Rv was amplified by PCR and the product was ligated into pCDFDuet-1 vector (Novagen) between NcoI and BamH1 restriction sites; the absence of mutation was verified by sequencing. Site-directed mutagenesis of the *Rv0504c* and *Rv0636* genes was carried out by reassembly PCR using pET-15b::Rv0504c and pCDFDuet-1::Rv0636 vectors, respectively, as well as primers containing the desired mutation (Table S1). Mutated DNA was used to transform *E. coli* DH5α competent cells and the introduction of the point mutation was confirmed by sequencing of purified plasmids.

4.4 | Small angle scattering

Small-angle X-ray scattering data were collected at the BM29 beamline (ESRF, Grenoble, France) using a Pilatus 1M detector. The detector-distance of respectively 2.867 m covers a momentum transfer range of 0.04 < Q < 4.94 nm<sup>−1</sup>. SAXS experiments coupled with SEC (SEC-SAXS) were carried out at 20°C on a BioSec 130 column, with HadBD concentration and sample

volume of respectively 8.2 mg/mL and 50 μL. Fresh DTT (2 mM) was added to the sample and centrifugation was performed just before data collection. Each measurement consisted of 10 frames of 2 s for which radiation damage was systematically investigated. Data processing was performed with the ATSAS suite of programs (Franke et al., 2017). Evaluation of the molecular weight was done based on the Porod volume, as implemented in SAXSMoW (Piiadov et al., 2019). Fitting of the theoretical scattering curves computed from both the tetrameric and dimeric crystallographic structures of HadBD with the experimental data in solution was done using CRYSOL (Svergun et al., 1995).

### 4.5 Native mass spectrometry

Prior to native MS analysis, HadBD sample at 4.8 mg/mL was desalted in 200 mM ammonium acetate, pH 7 using Micro Bio-Spin devices (Bio-Rad, Marnes-la-Coquette, France). Samples were analyzed on a Synapt G2-Si mass spectrometer (Waters Scientific, Wilmslow, UK) running in sensitivity mode, positive ion mode and coupled to an automated chip-based nano-electrospray source (Triversa Nanomate, Advion Biosciences, Ithaca, NY). The voltage applied to the chip and the cone voltage were set to 1.6 kV and 100 V, respectively. The instrument was calibrated with a 2-mg/mL cesium iodide solution in 50% isopropanol. Raw data were acquired in the 1000–8000 m/z range with MassLynx 4.1 (Waters, Manchester, UK) and deconvoluted with UniDec 4.4.0 (Marty et al., 2015) using the following parameters: *m/z* range: 1000–8000 Th; Gaussian smoothing: 100; charge range: 5–30; mass range: 10,000–100,000 Da; sample mass every 10 Da; smooth charge states distributions; smooth nearby points: some; suppress artifacts: none; peak detection range: 50 Da, and peak detection threshold: 0.25.

### 4.6 Split-Trp experiments in *M. smegmatis*

The Split-Trp experiments in *Msm* were performed based on previously described methods (Chao et al., 2010; O'Hare et al., 2008; Sacco et al., 2007) and modified as follows. Briefly, expression plasmids encoding each of the target proteins (*HadB*~*Mtb*~ and *HadD*~*Mtb*~) or a control protein (*HadC*~*Mtb*~) in fusion with either the Ntrp or the Ctrp fragment of *Saccharomyces cerevisiae* Trp1p protein were constructed by cloning *Rv0636*, *Rv0504c*, and *Rv0637* genes from *Mtb* H37Rv (ATCC 27294) between AflII and ClaI restriction sites of pJC10 and pJC11 plasmids (Chao et al., 2010; generously provided by Dr. Yossef Av-Gay),

under the control of an inducible acetamidase promoter. A tryptophan auxotrophic *Msm ΔhisA* strain (O'Hare et al., 2008; kindly provided by Prof. Yossef Av-Gay) was cotransformed with pairs of plasmids encoding Ntrp and Ctrp fusion proteins, and grown for 3 days at 37°C on LB supplemented with hygromycin (100 μg/mL) and apramycin (60 μg/mL). Recombinant *Msm ΔhisA* colonies were resuspended in 3 mL of LB plus 0.05% Tween 80 and antibiotics and grown at 37°C with shaking for 48 h; the expression of the genes of interest was induced by adding 0.2% (w/v) acetamide for 6 h before stopping the culture (at an OD~600~ of 1–2). Bacteria from 500-μL aliquots were harvested at 10,000 g, washed twice with 1 mL of water plus 0.05% Tween 80, and resuspended in about 1 mL of the same mixture, adjusting all the samples to the same OD~600~. Five microliters of aliquots (dilutions 1, 1/10 and 1/100) of each sample were spotted onto 7H9 agar plates supplemented by 0.2% Gro, 0.2% acetamide, 60 μg/mL histidine, 100 μg/mL hygromycin, 60 μg/mL apramycin, and depleted in tryptophan. The same dilutions were spotted on plates containing 60 μg/mL tryptophan to check growth extent. Plates were incubated at 30°C for 3 weeks. Experiments were performed in triplicate. Additionally, *Msm ΔhisA* strains producing either Ntrp-Esat6/Cfp10-Ctrp (pJC10 and pJC11-based expression plasmids provided by Dr. Yossef Av-Gay; Chao et al., 2010) or Ntrp-HadB~*Mtb*~/HadC~*Mtb*~-Ctrp were used as positive controls while two strains producing pairwise combinations of proteins from distinct complexes (Ntrp-HadB~*Mtb*~/Cfp10~*Mtb*~-Ctrp and Ntrp-Esat6~*Mtb*~/HadD~*Mtb*~-Ctrp) were used as negative controls.

### 4.7 Crystallization and structure determination

Purified HadBD protein was concentrated to a final concentration of 6 mg/mL in 50 mM Tris/HCl pH 8, 150 mM NaCl, and 5% glycerol. All crystallization experiments were carried out at 12°C using the sitting drop vapor diffusion method. Best crystals of HadBD were obtained within a week with a 1:1 (v/v) ratio of protein to precipitant solution (0.1 M Na cacodylate pH 6.5, 40% MPD, and 5% PEG 8k). X-ray experiments were carried out at 100 K. Crystals were directly frozen in a cryogenic nitrogen stream and stored in liquid nitrogen until the data were collected. Diffraction data were measured from hexagonal crystals diffracting to ca. 1.9 Å on Xaloc Beamline at the ALBA synchrotron (Cerdanyola del Vallès, Spain) using a PILATUS 6M detector. X-ray images were indexed, integrated, and scaled on-line using autoPROC and STARANISO (Vonrhein et al., 2011; Vonrhein et al., 2018) due to a strong anisotropy of the diffraction
pattern. Programs from the CCP4 software suite (Winn et al., 2011) were used for subsequent crystallographic calculations. The structure was solved by the molecular replacement method using PHASER (McCoy et al., 2007) and a starting model derived from the structure of HadAB from *M. tuberculosis* (PDB 4RLJ). Then manual rebuilding using Coot (Emsley et al., 2010) followed by refinement steps using Refmac5 (Murshudov et al., 2011) were performed iteratively. Crystals belong to the hexagonal space group $P6_522$ ($a = b = 96.0 \, \text{\AA}$, $c = 143.7 \, \text{\AA}$) with one heterodimer in the asymmetric unit and a solvent content of 55%. The refined structure at a resolution of 2.0 Å comprises residues 2–141 of HadB and 13–151 of HadD. The final Rwork/Rfree factors are 0.246 and 0.290, respectively (Table S2). Data collection and refinement statistics are given in Table S2. The coordinates have been deposited to the PDB with the PDB ID 8PWZ. The surfaces of heterodimerization and tetramerization were measured according to PDBePISA analysis (Krissinel & Henrick, 2007). Figures of protein structures were drawn using the program PyMOL Molecular Graphics System, Version 2.0 Schrödinger, LLC.

### 4.8 | Structure-based alignments

For structure-based multiple sequence alignments, all structures, either experimental (this work for HadB<sub>Mtb</sub> and HadD<sub>Mtb</sub>; PDB 4RLJ chain A for HadA<sub>Mtb</sub>; PDB 5ZY8 chain C for HadC<sub>Mtb</sub>) or predicted using AlphaFold2 (Jumper et al., 2021) in the case of HadD<sub>Msm</sub>, were superimposed using the Superpose program as implemented in the CCP4 suite (Krissinel & Henrick, 2007). The tertiary structure alignment of HAD protomers was performed using the secondary-structure matching algorithm of the same Superpose program.

### 4.9 | Substrate synthesis and purification

Trans-2-dodecenoyl-CoA and trans-2-hexadecenoyl-CoA were synthesized using the following procedure based on the mixed anhydride method (Goldman & Vagelos, 1961). Briefly, three molar equivalents (with respect to CoA) of the corresponding free carboxylic acid (trans-2-dodecenoic or trans-2-hexadecenoic acid, Larodan) were dissolved in 100 μL anhydrous *t*-butyl methyl ether (*t*-BME, Sigma-Aldrich) per mg of acid. After the addition of 1.2 molar equivalents (with respect to carboxylic acid) of both diisopropyl amine and ethyl chloroformate, the reaction was incubated overnight at room temperature. The formed mixed anhydride was separated from ammonium chloride crystals and concentrated under nitrogen. One molar equivalent (with respect to carboxylic acid) of free CoA trilithium salt (Sigma-Aldrich) was dissolved in a 1:1:1 0.3 M NaHCO₃:ethanol:ethyl acetate mixture at a 5 mg/mL final concentration then added to the mixed anhydride solution. After a 15-min incubation at room temperature, the reaction was quenched by the addition of acetic acid. The trans-2-enoyl-CoA produced was washed three times by one volume of ethyl acetate. It was then purified by HPLC over a 250 × 8 mm ProntoSIL C18 H column (Bischoff) equilibrated with 30% CH₃OH in 20 mM NaH₂PO₄, using an Ultimate 3000 HPLC (ThermoFischer) apparatus with detection at 260 nm. Elution was performed at 1 mL/min flow rate using a multiple-step elution program: a gradient from 30% to 40% CH₃OH in 20 mM NaH₂PO₄ (1%/min) with a plateau, then a raise to 100% CH₃OH in H₂O (12%/min) with a plateau. The nature of purified substrates was confirmed by MALDI-TOF MS analysis (see below).

### 4.10 | Enzyme activity assays

Enzymes belonging to the (*R*)-specific enoyl hydratase/hydroxyacyl dehydratase family preferentially catalyze the hydration reaction when they are isolated from their metabolic machinery (Heath & Rock, 1996; Sacco et al., 2007). Therefore, their in vitro activities are most often studied in the presence of enoyl derivatives. Hydratase activity was monitored at 263 nm using a UVmc2 spectrophotometer (SAFAS). Kinetic assays were performed in a quartz cuvette in a total volume of 400 μL at room temperature, in 100 mM sodium phosphate buffer pH 7.0 in the presence of 3 μM of either trans-2-C12:1-CoA or trans-2-C16:1-CoA. After equilibration of the baseline, reactions were started by adding either 20 or 100 nM of purified (HadBD)<sub>2</sub> isoform, respectively, and measurements were recorded for 1–3 min. The initial reaction rates were determined by linear fitting. Enzyme and substrate concentrations were chosen within a window generating a linear response. At 263 nm wavelength, trans-2-enoyl-CoAs display an absorbance shift of 0.67 for a variation of concentration of 100 μM. The average specific activities and deviations were calculated from triplicates. Statistical differences in specific activities were determined by two-tailed *t*-tests. A *p*-value < 0.05 was considered as statistically significant. The nature of the products formed during the enzymatic assays was confirmed by MALDI-TOF MS analysis (see below).
BORIES ET AL.

4.11 | MALDI-TOF MS

Matrix-assisted laser desorption ionization-time of flight mass spectrometry (MALDI-TOF MS) analyses were performed in the positive ionization and reflectron mode, using the 5800 MALDI-TOF/TOF Analyzer (Applied Biosystems/ABsciex) equipped with an Nd:YAG laser (349nm wavelength). MS and MS/MS spectra were acquired with a total of 2500 shots at a fixed laser intensity of 4000 (instrument-specific units) and 400-Hz pulse rate. Reaction media of enzyme activity assays were diluted 10 folds in water prior to analysis. One μl samples were spotted onto the target plate, mixed with 1 μL of matrix [10mg/mL of 2,5-dihydroxybenzoic acid (Sigma-Aldrich) in water:acetonitrile, 8:2 (v/v)]. Samples were allowed to crystallize at room temperature. MS data were acquired using the instrument's default calibration.

4.12 | Differential scanning fluorimetry

The DSF experiments were carried out as described (Niesen et al., 2007). Samples were loaded into a clear 96-well PCR plate (Bio-Rad) with each well containing a final volume of 20 μL. The concentration of heterodimer in each well was 16.9 μM (0.60 mg/mL) for HadBwt/H-HadDwt, 18 μM (0.64 mg/mL) for HadB(H41F)/H-HadDwt, 16.9 μM (0.60 mg/mL) for HadBwt/H-HadD (H42F), and HadBwt/H-HadD(H42Q), 17.2 μM (1.6 mg/mL) for HadB(H41F)/H-HadD(H42F). All experiments were performed in triplicate in 50mM Tris-HCl, pH 8.0, 150mM NaCl, 5% glycerol, and 10X SYPRO Orange (Invitrogen). The PCR plates were sealed with optical quality sealing tape (Bio-Rad). DSF experiments were carried out using a CFX96 real-time PCR system (Bio-Rad) set to use the 480/500 excitation and 560/580 emission filters. The samples were heated from 20 to 99.5°C at the rate of 6°C/min. A single fluorescence measurement was taken every 0.3°C. Melting temperatures were determined by performing a curve fit to the Boltzmann equation.

Supervision; funding acquisition. Lionel Mourey: Conceptualization; formal analysis; writing – original draft; writing – review and editing; supervision; funding acquisition. Manuelle Ducoux-Petit: Conceptualization; formal analysis; writing – review and editing; supervision. Fabienne Bardou: Conceptualization; investigation; formal analysis; writing – original draft; writing – review and editing; supervision. Cécile Bon: Conceptualization; investigation; formal analysis; writing – original draft; writing – review and editing; supervision. Annaïk Quémard: Conceptualization; investigation; formal analysis; writing – original draft; writing – review and editing; supervision; funding acquisition

ACKNOWLEDGMENTS

We are grateful to R. Boulon, S. Gavalda, N. Eynard, C. Lefebvre, and E. Ye (IPBS, Toulouse) for technical help and to M. Daffé (IPBS, Toulouse) for fruitful discussion. We also thank Prof. A. Dziembowski (International Institute of Molecular and Cell Biology in Warsaw) for the kind gift of pKW08::egfp plasmid, and to Dr. H. O'Hare (University of Leicester), Prof. Y. Av-Gay (University of British Columbia) and their colleagues for the generous gifts of split-Trp tools and technical advice. We also thank the scientific staff of the European Synchrotron Radiation Facility (Grenoble, France), SOLEIL (Gif sur Yvette, France), and ALBA (Barcelona, Spain) for the use of their excellent data collection facilities. We particularly thank the staff of beamlines Xaloc at ALBA and BM29 at the European Synchrotron Radiation Facility, where the crystallographic and SAXS experiments were, respectively, conducted. The biophysical and macromolecular crystallography equipment used in this study are part of the Integrated Screening Platform of Toulouse (PICT, IPBS, IBiSA). We acknowledge funding from the Agence Nationale de la Recherche (FASMY, grant ANR-14-CE16-0012), MSDAVENIR (FIGHT-TB: Unconventional Strategies for Tuberculosis Treatment), and the French Ministry of Research with the Investissement d'Avenir Infrastructures Nationales en Biologie et Santé program (ProFI, Proteomics French Infrastructure project, ANR-10-INBS-08).

AUTHOR CONTRIBUTIONS

Pascaline Bories: Investigation; formal analysis; writing – review and editing. Julie Rima: Investigation; formal analysis. Samuel Tranier: Conceptualization; investigation; formal analysis; writing – original draft; writing – review and editing; supervision. Julien Marcoux: Investigation; formal analysis; writing – review and editing. Yasmina Grimoire: Investigation. Mathilde Tomaszczyk: Investigation. Anne Launay: Resources. Karine Fata: Resources. Hedia Marrakchi: Writing – review and editing. Odile Burlet-Schiltz:

CONFLICT OF INTEREST STATEMENT

The authors declare no conflicts of interest.

ORCID

Annaïk Quémard [![ORCID](https://orcid.org/images/orcid_16x16.png)](https://orcid.org/0000-0002-5545-3424)

REFERENCES

Bergeret F, Gavalda S, Chalut C, Malaga W, Quemard A, Pedelacq JD, et al. Biochemical and structural study of the

atypical acyltransferase domain from the mycobacterial polyketide synthase Pks13. J Biol Chem. 2012;287:33675–90.

Biswas R, Dutta A, Dutta D, Hazra D, Banerjee DR, Basak A, et al. Crystal structure of dehydratase component HadAB complex of mycobacterial FAS-II pathway. Biochem Biophys Res Commun. 2015;458:369–74.

Bouyssie D, Hesse AM, Mouton-Barbosa E, Rompais M, Macron C, Carapito C, et al. Proline: an efficient and user-friendly software suite for large-scale proteomics. Bioinformatics. 2020;36:3148–55.

Chakraborty P, Bajeli S, Kaushal D, Radotra BD, Kumar A. Biofilm formation in the lung contributes to virulence and drug tolerance of *Mycobacterium tuberculosis*. Nat Commun. 2021;12:1606.

Chao JD, Papavinasasundaram KG, Zheng X, Chavez-Steenbock A, Wang X, Lee GQ, et al. Convergence of Ser/Thr and two-component signaling to coordinate expression of the dormancy regulon in *Mycobacterium tuberculosis*. J Biol Chem. 2010;285:29239–46.

Daffe M, Quémard A, Marrakchi H. Mycolic acids: from chemistry to biology. In: Geiger O, editor. Biogenesis of fatty acids, lipids and membranes. Cham: Springer; 2019. p. 181–216.

Dao DN, Sweeney K, Hsu T, Gurcha SS, Nascimento IP, Roshevsky D, et al. Mycolic acid modification by the *mmaA4* gene of *M. tuberculosis* modulates IL-12 production. PLoS Pathog. 2008;4:e1000081.

Dickey SW, Cheung GYC, Otto M. Different drugs for bad bugs: antivirulence strategies in the age of antibiotic resistance. Nat Rev Drug Discov. 2017;16:457–71.

Dong Y, Qiu X, Shaw N, Xu Y, Sun Y, Li X, et al. Molecular basis for the inhibition of beta-hydroxyacyl-ACP dehydratase HadAB complex from *Mycobacterium tuberculosis* by flavonoid inhibitors. Protein Cell. 2015;6:504–17.

Dubnau E, Chan J, Raynaud C, Mohan VP, Laneelle MA, Yu K, et al. Oxygenated mycolic acids are necessary for virulence of *Mycobacterium tuberculosis* in mice. Mol Microbiol. 2000;36:630–7.

Dubnau E, Laneelle MA, Soares S, Benichou A, Vaz T, Prome D, et al. *Mycobacterium bovis* BCG genes involved in the biosynthesis of cyclopropyl keto- and hydroxy-mycolic acids. Mol Microbiol. 1997;23:313–22.

Duguet F, Locard-Paulet M, Marcellin M, Chaoui K, Bernard I, Andreoletti O, et al. Proteomic analysis of regulatory T cells reveals the importance of Themis1 in the control of their suppressive function. Mol Cell Proteomics. 2017;16:1416–32.

Emsley P, Lohkamp B, Scott WG, Cowtan K. Features and development of coot. Acta Crystallogr D Biol Crystallogr. 2010;66:486–501.

Faille A, Gavalda S, Slama N, Lherbet C, Maveyraud L, Guillet V, et al. Insights into substrate modification by dehydratases from type I polyketide synthases. J Mol Biol. 2017;429:1554–69.

Fiers WD, Dodge GJ, Sherman DH, Smith JL, Aldrich CC. Vinylous dehydration by a polyketide dehydratase domain in Curacin biosynthesis. J Am Chem Soc. 2016;138:16024–36.

Finzel K, Nguyen C, Jackson David R, Gupta A, Tsai S-C, Burkart Michael D. Probing the substrate specificity and protein-protein interactions of the *E. coli* fatty acid dehydratase. FabA Chem Biol. 2015;22:1453–60.


Franke D, Petoukhov MV, Konarev PV, Panjkovich A, Tuukkanen A, Mertens HDT, et al. ATSAS 2.8: a comprehensive data analysis suite for small-angle scattering from macromolecular solutions. J Appl Cryst. 2017;50:1212–25.

Goldman P, Vagelos PR. The specificity of triglyceride synthesis from diglycerides in chicken adipose tissue. J Biol Chem. 1961;236:2620–3.

Groenewald W, Baird MS, Verschoor JA, Minnikin DE, Croft AK. Differential spontaneous folding of mycolic acids from *Mycobacterium tuberculosis*. Chem Phys Lipids. 2014;180:15–22.

Heath RJ, Rock CO. Roles of the FabA and FabZ beta-hydroxyacyl-acyl carrier protein dehydratases in *Escherichia coli* fatty acid biosynthesis. J Biol Chem. 1996;271:27795–801.

Hisano T, Tsuge T, Fukui T, Iwata T, Miki K, Doi Y. Crystal structure of the (R)-specific enoyl-CoA hydratase from *Aeromonas caviae* involved in polyhydroxyalkanoate biosynthesis. J Biol Chem. 2003;278:617–24.

Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with AlphaFold. Nature. 2021;596:583–9.

Kim SK, Dickinson MS, Finer-Moore J, Guan Z, Kaake RM, Echeverria I, et al. Structure and dynamics of the essential endogenous mycobacterial polyketide synthase Pks13. Nat Struct Mol Biol. 2023;30:296–308.

Koski KM, Haapalainen AM, Hiltunen JK, Glumoff T. Crystal structure of 2-enoyl-CoA hydratase 2 from human peroxisomal multifunctional enzyme type 2. J Mol Biol. 2005;345:1157–69.

Koski MK, Haapalainen AM, Hiltunen JK, Glumoff T. A two-domain structure of one subunit explains unique features of eukaryotic hydratase 2. J Biol Chem. 2004;279:24666–72.

Krissinel E, Henrick K. Inference of macromolecular assemblies from crystalline state. J Mol Biol. 2007;372:774–97.

Leesong M, Henderson BS, Gillig JR, Schwab JM, Smith JL. Structure of a dehydratase-isomerase from the bacterial pathway for biosynthesis of unsaturated fatty acids: two catalytic activities in one active site. Structure. 1996;4:253–64.

Lefebvre C, Boulon R, Ducoux M, Gavalda S, Laval F, Jamet S, et al. HadD, a novel fatty acid synthase type II protein, is essential for alpha- and epoxy-mycolic acid biosynthesis and mycobacterial fitness. Sci Rep. 2018;8:6034.

Lefebvre C, Frigui W, Slama N, Lauzeral-Vizcaino F, Constant P, Lemassu A, et al. Discovery of a novel dehydratase of the fatty acid synthase type II critical for ketomycolic acid biosynthesis and virulence of *Mycobacterium tuberculosis*. Sci Rep. 2020;10:2112.

Li W, Gu S, Fleming J, Bi L. Crystal structure of FadD32, an enzyme essential for mycolic acid biosynthesis in mycobacteria. Sci Rep. 2015;5:15493.

Ling X, Liu X, Wang K, Guo M, Ou Y, Li D, et al. Lsr2 acts as a cyclic di-GMP receptor that promotes keto-mycolic acid synthesis and biofilm formation in mycobacteria. Nat Commun. 2024;15:695.

Lou JW, Lyer KR, Hasan SMN, Cowen LE, Mazhab-Jafari MT. Electron cryomicroscopy observation of acyl carrier protein translocation in type I fungal fatty acid synthase. Sci Rep. 2019;9:12987.

Marty MT, Baldwin AJ, Marklund EG, Hochberg GKA, Benesch JLP, Robinson CV. Bayesian deconvolution of mass

and ion mobility spectra: from binary interactions to polydis-
perse ensembles. Anal Chem. 2015;87:4370–6.
McCoy AJ, Grosse-Kunstleve RW, Adams PD, Winn MD,
Storoni LC, Read RJ. Phaser crystallographic software. J Appl
Cryst. 2007;40:658–74.
Murshudov GN, Skubak P, Lebedev AA, Pannu NS, Steiner RA,
Nicholls RA, et al. REFMAC 5 for the refinement of macromo-
lecular crystal structures. Acta Crystallogr D Biol Crystallogr.
2011;67:355–67.
Musayev F, Sachdeva S, Neel Scarsdale J, Reynolds KA, Wright HT.
Crystal structure of a substrate complex of Mycobacterium
tuberculosis β-ketoacyl-acyl carrier protein synthase III (FabH)
with lauroyl-coenzyme A. J Mol Biol. 2005;346:1313–21.
Nguyen C, Haushalter RW, Lee DJ, Markwick PRL, Bruegger J,
Caldara-Festin G, et al. Trapping the dynamic acyl carrier pro-
tein in fatty acid biosynthesis. Nature. 2014;505:427–31.
Niesen FH, Berglund H, Vedadi M. The use of differential scanning
fluorimetry to detect ligand interactions that promote protein
stability. Nat Protoc. 2007;2:2212–21.
O'Hare H, Juillerat A, Dianiskova P, Johnsson K. A split-protein
sensor for studying protein-protein interaction in mycobacteria.
J Microbiol Methods. 2008;73:79–84.
Perez-Riverol Y, Csordas A, Bai J, Bernal-Llinares M,
Hewapathirana S, Kundu DJ, et al. The PRIDE database and
related tools and resources in 2019: improving support for
quantification data. Nucleic Acids Res. 2019;47:D442–50.
Piadov V, Ares de Araujo E, Oliveira Neto M, Craievich AF,
Polikarpov I. SAXSMoW 2.0: online calculator of the molecu-
lar weight of proteins in dilute solution from experimental
SAXS data measured on a relative scale. Protein Sci. 2019;28:
454–63.
Plocinski P, Laubitz D, Cysewski D, Stodus K, Kowalska K,
Dziembowski A. Identification of protein partners in mycobac-
teria using a single-step affinity purification method. PLoS One.
2014;9:e91380.
Quémard A. New insights into the mycolate-containing compound
biosynthesis and transport in mycobacteria. Trends Microbiol.
2016;24:725–38.
Sacco E, Covarrubias AS, O'Hare HM, Carroll P, Eynard N,
Jones TA, et al. The missing piece of the type II fatty acid
synthase system from Mycobacterium tuberculosis. Proc Natl
Acad Sci USA. 2007;104:14628–33.
Sambandan D, Dao DN, Weinrick BC, Vilcheze C, Gurcha SS,
Ojha A, et al. Keto-mycolic acid-dependent pellicle formation
confers tolerance to drug-sensitive Mycobacterium tuberculosis.
MBio. 2013;4:e00222-00213.
Singh BK, Biswas R, Bhattacharyya S, Basak A, Das AK. The
C-terminal end of mycobacterial HadBC regulates AcpM inter-
action during the FAS-II pathway: a structural perspective.
FEBS J. 2022;289:4963–80.
Slama N, Jamet S, Frigui W, Pawlik A, Bottai D, Laval F, et al. The
changes in mycolic acid structures caused by hadC mutation

have a dramatic effect on the virulence of Mycobacterium tuber-
culosis. Mol Microbiol. 2016;99:794–807.
Svergun D, Barberato C, Koch MHJ. CRYSOL—a program to evalu-
ate X-ray solution scattering of biological macromolecules from
atomic coordinates. J Appl Cryst. 1995;28:768–73.
Villeneuve M, Kawai M, Horiuchi K, Watanabe M, Aoyagi Y,
Hitotsuyanagi Y, et al. Conformational folding of mycobacterial
methoxy- and ketomycolic acids facilitated by alpha-methyl
trans-cyclopropane groups rather than cis-cyclopropane units.
Microbiology. 2013;159:2405–15.
Vonrhein C, Flensburg C, Keller P, Sharff A, Smart O, Paciorek W,
et al. Data processing and analysis with the autoPROC toolbox.
Acta Crystallogr D Biol Crystallogr. 2011;67:293–302.
Vonrhein C, Tickle IJ, Flensburg C, Keller P, Paciorek W, Sharff A,
et al. Advances in automated data analysis and processing
within autoPROC, combined with improved characterisation,
mitigation and visualisation of the anisotropy of diffraction
limits using STARANISO. Acta Crystallogr A. 2018;74:A360.
WHO. Global tuberculosis report 2023. Geneva: World Health
Organization; 2023.
Williams KJ, Joyce G, Robertson BD. Improved mycobacterial tetra-
cycline inducible vectors. Plasmid. 2010;64:69–73.
Winn MD, Ballard CC, Cowtan KD, Dodson EJ, Emsley P,
Evans PR, et al. Overview of the CCP4 suite and current devel-
opments. Acta Crystallogr D Biol Crystallogr. 2011;67:235–42.
Yang M, Guja KE, Thomas ST, Garcia-Diaz M, Sampson NS. A dis-
tinct MaoC-like enoyl-CoA hydratase architecture mediates
cholesterol catabolism in Mycobacterium tuberculosis. ACS
Chem Biol. 2014;9:2632–45.
Yuan Y, Zhu Y, Crane DD, Barry CE3rd. The effect of oxygenated myco-
lic acid composition on cell wall function and macrophage growth in
Mycobacterium tuberculosis. Mol Microbiol. 1998;29:1449–58.
Zhang L, Xiao J, Xu J, Fu T, Cao Z, Zhu L, et al. Crystal structure of
FabZ-ACP complex reveals a dynamic seesaw-like catalytic
mechanism of dehydratase in fatty acid biosynthesis. Cell Res.
2016;26:1330–44.

SUPPORTING INFORMATION
Additional supporting information can be found online
in the Supporting Information section at the end of this
article.

How to cite this article: Bories P, Rima J,
Tranier S, Marcoux J, Grimoire Y, Tomaszczyk M,
et al. HadBD dehydratase from Mycobacterium
tuberculosis fatty acid synthase type II: A singular
structure for a unique function. Protein Science.
2024;33(4):e4964. https://doi.org/10.1002/pro.4964
