
Review

# Hepatitis E Virus: What More Do We Need to Know?

Endrit Shahini ${}^{1,*}$, Antonella Argentiero ${}^{2}$, Alessandro Andriano ${}^{3}$, Francesco Losito ${}^{1}$, Marcello Maida ${}^{4}$, Antonio Facciorusso ${}^{5}$, Raffaele Cozzolongo ${}^{1,\dagger}$ and Erica Villa ${}^{6,*,\dagger}$

1. Gastroenterology Unit, National Institute of Gastroenterology-IRCCS “Saverio de Bellis”, Castellana Grotte, 70013 Bari, Italy; francesco.losito@irccsdebellis.it (F.L.); raffaele.cozzolongo@irccsdebellis.it (R.C.)
2. Istituto Tumori “Giovanni Paolo II”, Medical Oncology Unit, 70124 Bari, Italy; argentieroantonella@gmail.com
3. Department of Precision and Regenerative Medicine and Ionian Area, University of Bari Aldo Moro Medical School, 70124 Bari, Italy; a.andriano2@studenti.uniba.it
4. Gastroenterology and Endoscopy Unit, S. Elia-Raimondi Hospital, 93100 Caltanissetta, Italy; marcello.maida@hotmail.it
5. Department of Medical and Surgical Sciences, University of Foggia, 71122 Foggia, Italy; antonio.facciorusso@unifg.it
6. Gastroenterology Unit, CHIMOMO Department, University of Modena & Reggio Emilia, Via del Pozzo 71, 41121 Modena, Italy

* Correspondence: endrit.shahini@irccsdebellis.it (E.S.); erica.villa@unimore.it (E.V.)

† These authors contributed equally to this work.

---

**Abstract:** Hepatitis E virus (HEV) infection is typically a self-limiting, acute illness that spreads through the gastrointestinal tract but replicates in the liver. However, chronic infections are possible in immunocompromised individuals. The HEV virion has two shapes: exosome-like membrane-associated quasi-enveloped virions (eHEV) found in circulating blood or in the supernatant of infected cell cultures and non-enveloped virions (“naked”) found in infected hosts' feces and bile to mediate inter-host transmission. Although HEV is mainly spread via enteric routes, it is unclear how it penetrates the gut wall to reach the portal bloodstream. Both virion types are infectious, but they infect cells in different ways. To develop personalized treatment/prevention strategies and reduce HEV impact on public health, it is necessary to decipher the entry mechanism for both virion types using robust cell culture and animal models. The contemporary knowledge of the cell entry mechanism for these two HEV virions as possible therapeutic target candidates is summarized in this narrative review.

**Keywords:** HEV; viral cell entry; inhibitors; antiviral treatment; pregnancy; chronic infection

---

## 1. Introduction

Hepatitis E virus (HEV), an established human pathogen, infects humans and various other animal species [1]. HEV spreads directly by contact with infected animals, contaminated food, or the environment [1].

In 2016, HEV was classified as a “quasi-enveloped” virus (eHEV), with both non-enveloped (“naked”) and enveloped forms, similar to HAV [1]. HEV is shed in feces as a non-enveloped virus, but in cell culture, it has a lipid envelope [1].

HEV is a poorly understood zoonotic virus that causes acute and chronic viral hepatitis [2,3]. The incidence of chronic HEV infection/chronic hepatitis is extremely low, owing primarily to immune-deficient status (i.e., transplant recipients, human immunodeficiency virus patients with low CD4 counts, and patients with hematologic malignancies receiving chemotherapy) [3].

HEV infection is widespread worldwide, with distinct differences in transmission and disease outcomes in resource-rich versus resource-limited areas. It is estimated that 20 million HEV infections occur annually, resulting in 3.4 million clinical cases of hepatitis E and 70,000 hepatitis E-related deaths worldwide [4].

Disease manifestation, infection source, and route of transmission vary by HEV genotype and epidemiology in both endemic and non-endemic global settings [1–4]. In developed countries, hepatitis E occurs primarily irregularly or in clusters, whereas in developing countries, endemic or epidemic hepatitis E is occasionally associated with large outbreaks [5]. Specifically, minority populations with serious social problems, such as poverty, low income, unemployment, a lack of education, and inadequate housing, are more likely to contract HEV infection [6].

The primary route of HEV transmission is fecal-oral via contaminated drinking water or consumption of raw animal meat products [7]. HEV infects many species, including bats, ferrets, rabbits, and chickens. However, swine, deer, and wild boar are the primary reservoirs for transmission to humans [1].

In 1978, HEV was identified as an epidemic of non-A and non-B hepatitis from Kashmir, India [8]. Then, since the discovery of swine HEV in pigs in the United States (1997) [9], novel strains of HEV have been genetically identified from over a dozen animal species, including domestic and rabbit, wild pig, rat, camel, mongoose, and chick [5,6,9–12].

This review will elucidate the mechanism of viral cell entry and current and alternative therapies for HEV infection. Antivirals inhibit the viral life cycle and have distinct advantages in suppressing viral infections, making them of particular interest due to their potential therapeutic or prophylactic applications. This premise is especially critical for immunosuppressed chronic HEV infection subjects who have failed conventional therapy and pregnant women with HEV-induced severe hepatic injury, even leading to lethal cases and fetus death.

## 2. HEV Genotypes

Several HEV genotypes have varying degrees of host specificity [13]. There have been eight HEV genotypes identified so far [14].

HEV genotypes 1 and 2 infect only humans and primates, whereas genotypes 3 and 4 infect both humans and other mammals (including deer, swine, dolphins, cows, nonhuman primates, and bears) [15,16]. Genotype 3 is most commonly found in industrialized countries (Europe and America) [15]. Furthermore, HEV genotype 3 infection is usually asymptomatic, but it can induce severe acute hepatitis in patients with chronic liver disease, as well as chronic hepatitis and cirrhosis in immunocompromised patients [15]. HEV genotypes 5 and 6 have been isolated from wild boars, while genotypes 7 and 8 have been identified in camels from the United Arab Emirates and China, respectively [14].

Interestingly, rodent-associated HEV is gaining particular attention because it can infect humans as a secondary host and cause severe acute and/or chronic hepatitis [16]. Additionally, HEV genotypes have specific differences in prevalence by geographic region.

## 3. Virology

### 3.1. HEV Composition and Structure

HEV is a small, icosahedral, non-enveloped 7.2 kb single-stranded positive-sense RNA virus that belongs to Paslahepivirus balayani according to the current nomenclature (similar to hepatitis A virus of the Picornaviridae) with a 27–34 nm diameter [17–21].

There are currently eight known HEV genotypes [20]. The majority of the strains that infect humans (HEV genotypes from 1 to 4 and genotype 7) belong to two species: orthohepevirus A (HEV genotypes from 1 to 8) and orthohepevirus C [15].

Virions shed in feces are non-enveloped, whereas virions are released in cell culture and circulating in the blood as quasi-enveloped due to a lipid envelope [22].

HEV causes self-limiting acute hepatitis in humans [23]. Capsomers are composed of a single capsid protein homodimer forming the HEV capsid. Each capsid protein contains three domains (with a polysaccharide-binding site that may interact with cellular receptors) that form distinct structural elements: S (constant capsid), P1 (three-fold protrusions), and P2 (two-fold spikes with neutralizing epitopes) [23].
The HEV genome organization is similar to many alphaviruses, with a methyl guanosine cap (m7G) and nonstructural genes at the 5′ end and structural genes at the 3′ end with polyadenylated (PolyA) tail [23]. The 5′ untranslated region of HEV is 28 nucleotides long, and bases 2–26 have the potential to form a complex secondary structure or hairpin [23].

The genome of HEV is polyadenylated at the 3′ end and consists of three partly overlapping open reading frames (ORFs) (Figure 1) [24]:

- The length of ORF1 varies among HEV genotypes; ORF1 codes for nonstructural proteins involved in virus replication (i.e., methyltransferase, Y domain, papain-like cysteine protease, hypervariable region, X or macro domain, helicase, and RNA-dependent RNA polymerase) [25].
- The ORF2 has 112–114 codons that are divided into three structural domains [shell (S), middle (M), and protruding (P)] [20,26]. HEV encodes two isoforms of the ORF2 capsid protein, which are distributed in different subcellular compartments and serve distinct functions throughout the HEV lifecycle [27]. Recombinant ORF2 proteins expressed in bacterial and insect cells have been shown to self-assemble into virus-like particles, the crystal structures of which have been determined [20,21,28,29]. The ORF2 protein was found to bind specifically to the 5′ end of the HEV genome [23]. Several techniques for amplifying the ORF2 gene have been reported, as the ORF2 gene is frequently utilized to analyze and genotype HEV strains [30].
- ORF3 has 123 codons and encodes a small phosphoprotein, a functional iron channel that appears to function as a viroporin, allowing infectious virions to be released from infected cells [31].
- ORF4, a new ORF discovered within the coding region of ORF1, appears to be restricted to genotype 1 HEV and is translated into a protein that increases activity of the RNA-dependent RNA polymerase [1,18,32,33]; p-ORF4 consists of 124 aa, promotes replication of the virus, is indispensable for the life cycle of HEV genotype 1, and is possibly involved in the increased severity of infection in pregnancy [1,18].

### A HEV genome

![HEV Genome Diagram](attachment:HEV_Genome_Diagram.png)

### B HEV virion

![HEV Virion Diagram](attachment:HEV_Virion_Diagram.png)

#### 1) Quasi-enveloped
#### 2) Non-enveloped

Figure 1. Hepatitis E virus (HEV) genome and virion structure. The figure illustrates the genomic organization and virion structure of the Hepatitis E virus (HEV). (A) Genomic Organization: The HEVgenome is a 7.2 kilobase positive-sense single-stranded RNA molecule. It is organized with non-structural genes at the 5′ end and a methyl guanosine cap (m7G), as well as structural genes at the 3′ end with polyadenylated (PolyA) tail. The 5′ untranslated region of HEV is 28 nucleotides long, with bases 2–26 having the potential to form a complex secondary structure. Furthermore, the HEV genome is polyadenylated at the 3′ end and consists of three partially overlapping open reading frames (ORFs). It is depicted as a linear arrangement of genetic elements, including these three ORFs. ORF1 produces nonstructural proteins that are required for viral replication. ORF2 produces the viral capsids, which form the viral particle’s capsid. ORF3 is a small multifunctional protein, wrapped by a membrane, which is necessary for virion release and pathogenesis. (B) Virion organization: The HEV virus has two varieties: (1) quasi-enveloped virions, which have an outer membrane analogous to exosomes and have been isolated from HEV-infected cells’ bloodstream and cell culture supernatant; (2) non-enveloped virions found in the feces and bile of infected hosts.

The HEV virion comes in two types: eHEV, which are covered by an external membrane similar to exosomes and have been isolated from the bloodstream and cell culture supernatant of HEV-infected cells, and non-enveloped virions, which were found in infected hosts’ feces and bile [34–36]. The simultaneous presence of the quasi-enveloped and non-enveloped viruses reflects HEV’s versatility in various host environments. The lipid envelope of quasi-enveloped virions may improve bloodstream survival and cell entry, whereas the stability of non-enveloped virions in feces and bile environments promotes efficient transmission via the fecal–oral route.

In isopycnic gradient centrifugation, eHEV particles band at a much lower buoyant density (~1.10 g/cm³) than traditional, non-enveloped virions (~1.25 g/cm³) [34] and are insensitive to neutralizing anti-capsid antibodies in standard neutralization assays [37].

### 3.2. The Viral Cell Entry, Endosomal Transport and Uncoating

Although HEV is primarily transmitted enterically, it is unclear how it crosses the gut barrier and enters the portal bloodstream, and the first round of infection requires quasi-enveloped HEV [33] or infrequently non-enveloped HEV [38].

HEV gut infection was demonstrated in a pig model, where the HEV RNA was detected by PCR in small intestinal and colonic tissue. HEV was found to replicate efficiently in polarized Caco-2 human colonic adenocarcinoma cells [38].

The inadequacy of successful HEV cell cultivation techniques hindered HEV life cycle research [38]. In studies conducted in heterologous settings, cellular factors interacting with HEV proteins that need to be validated and studied further have been identified [39,40].

The contemporary growth of HEV infection methods has allowed for the thorough investigation of the whole life cycle of HEV for the first time [41,42]. In vitro, HEV has a broad and distinct cell tropism. Marion et al. [43] developed various cell culture systems (exploiting tissue derived from small bowel resections) using as infecting agents HEV genotype 1 and 3 isolates from chronic hepatitis E (CHE) patients’ stool samples, as well as virus derived from the genotype 3 Kernow-C1 p6 clone. They showed that HEV can infect intestinal cells and provided the first evidence for productive HEV infection of authentic human primary intestinal cells.

HEV can cross the intestinal vascular endothelium and travel to the liver via the portal venous system, infecting hepatocytes from the basolateral side [38]. HEV replicates in intestinal cells and releases infectious particles from both the basolateral membrane and (preferentially) apical sides of (polarized) primary enterocytes before entering the bloodstream and liver as quasi-enveloped virions (low density), resulting in viremia of membrane-cloaked infectious virions [33,38]. Furthermore, non-enveloped HEV virions without a “quasi-envelope” can be found in the blood of some infected subjects (less than 20% of the circulating virus), especially late in infection when the liver is damaged [33].

The virus’s entry process may vary since HEV exists in both non-enveloped (“naked”) and quasi-enveloped HEV shapes (Figure 2). The discovery of quasi-enveloped HEV particles completely changed the understanding of the HEV life cycle and pathogenesis,
as evidenced by recent research that revealed that quasi-enveloped HEV particles are transmissible and are most likely uniquely responsible for infection within the host. The capsid would have to cross two layers of membranes to deliver the HEV genome into the cytoplasm in the presence of an eHEV.

![Diagram](attachment:diagram.png)

Figure 2. Hepatitis E virus (HEV) viral life cycle. 1. Attachment and Entry: Non-enveloped HEV attaches to specific receptors on the surface of susceptible host cells, including hepatocytes in the liver. This interaction is mediated by clathrin, heparan sulfate, integrin alpha 3, and dynamin 2 molecules. This engagement triggers receptor-mediated endocytosis, leading to viral host cell internalization. Alternatively, evidence shows that quasi-enveloped HEV enters cells via endocytosis followed by degradation of the membrane within the endolysosome. 2. Uncoating and Releasing of Genomic RNA: Once inside the host cell, HEV's viral capsid undergoes disassembly within the endosome. The precise mechanism of HEV capsid uncoating remains still unclear. 3. Translation and Polyprotein Processing: Within the cytoplasm, the HEV genomic RNA is translated into a large polyprotein. The nonstructural proteins encoded by ORF1 play pivotal roles in viral replication and other processes. 4. Genome Replication and Transcription: The HEV RNA-dependent RNA polymerase (RdRp), a product of ORF1, catalyzes the HEV genome replication and transcription. This step results in the production of novel viral RNA genomes and subgenomic RNA species. 5. Assembly of Viral Particles:
The newly synthesized viral genomic RNA is encapsidated by the structural capsid protein encoded by ORF2. This encapsidation process forms complete viral particles within the cytoplasm. 6. Maturation and Transport: The mature viral particles undergo structural changes and assemble into quasi-enveloped, spherical virions. The process of viral release from the host cell likely involves the endosomal sorting complexes required for transport (ESCRT) machinery. 7. Circulation and Transmission: After the fusion of multivesicular bodies (MVB) with plasma membranes, viral particles are released via the exosomal pathway so that only quasi-enveloped enters the bloodstream. Canalicular bile salts cause the degradation of the coating membrane and the production of naked particles. These particles reach the environment through bile secretion into the intestine, becoming the major source of contamination and transmission.

Figure 2 depicts a comprehensive overview of the HEV viral life cycle.

Although some HEV entry elements and mechanisms were determined, key variables remain unidentified. HEV's infectious capacity of a wide range of cells [41–48], which may be related to the various extrahepatic manifestations of hepatitis E [49–52], strongly suggests that unknown entry factors may be widely expressed. However, several host factors have been implicated in cell adherence and/or entry of naked HEV. Phosphatidylserine is found in the eHEV membrane and can attach to its receptor on target cells, such as TIM-1 [53,54]. Due to inadequate cell attachment, cellular intake of eHEV virions is less effective than that of naked HEV [34]. Studies using virus-like particles (VLP) as a model system have identified potential host factors involved in virus attachment and internalization, such as the 78 kDa glucose-regulated protein (GRP78), ATP synthase subunit (ATPB5), and asialoglycoprotein receptor (ASGPR) [55,56]. Specifically, heparinase I treatment of susceptible hepatoma cell lines significantly reduced VLP binding, as well as HEV infection [57].

Non-enveloped HEV particles are internalized into hepatocytes via clathrin, heparan sulfate, integrin alpha 3, and dynamin 2-dependent endocytosis [34,58–60]. In addition, proof indicates that quasi-enveloped HEV penetrates cells through endocytosis and then undergoes membrane degradation inside the endolysosome [33,38]. Notably, both non-enveloped and quasi-enveloped HEV have been demonstrated to bind to heparan sulfate proteoglycans, specifically syndecans, which are present on the outer layers of a variety of cells [34,57] and are known to mediate cell attachment of different types of viruses [54–58]. This engagement causes endocytosis through receptors, which leads to viral host cell internalization. However, the exact mechanism of HEV capsid uncoating is still unknown.

GRP78 is a molecular chaperone that is located in the endoplasmic reticulum. It is also known as binding immunoglobulin protein (BiP). The existence of GRP78 on the outer layer of cells has been stated and linked to several enveloped and non-enveloped viruses' binding and entrance [61–64].

ASGPR is a type of protein receptor that connects glycoproteins lacking sialic acid modifications on the basolateral surface of hepatic cells. Coimmunoprecipitation, pull-down, and ELISA analysis have revealed an intimate relationship between the ASGR1-ASGR2 ectodomains and HEV ORF2 [56]. Ectopic ASGRP expression raised HEV adhering in HeLa cells, whereas ASGRP exhaustion decreased HEV attaching yet did not affect virion discharge in PLC/PRF/5 cells. HEV binding to PLC/PRF/5 cells was also reduced by anti-ASGPR 229 antibody and purified ASGPR ectodomain.

Although ATP synthase is primarily a protein found in mitochondria, a small portion is expressed on the external layer of cells and has been linked to other viral infections [65]. The function of ATP5B in HEV entrance was confirmed using antibody/siRNA-mediated systems and pathogenic HEV derived from the feces of patients with hepatitis E [66].

Recently, ITGA3 (Integrin Alpha 3) was shown to be a HEV entrance component of PLC/PRF/5 cells [67]. Integrins are a type of transmembrane protein found on the cell surface of various tissues (e.g., the gut) that can bind to the extracellular matrix, cell surface, and the actin microfilaments of the cytoskeleton [68–70], which are known to be sites of HEV replication and infection [43,71]. A recent microarray study compared gene expression in permissive versus non-permissive cells [67]. ITGA3 excessive expression in non-permissive
cells enabled infection solely of non-enveloped HEV. Accordingly, in permissive cells, knocking out the ITGA3 gene hindered non-enveloped HEV yet not eHEV admission [67]. However, ITGA3 expression is low in some organs, such as the liver [72–74], suggesting that ITGA3 functions as a co-factor for HEV entrance instead of as the primary receptor.

The Rab5 and Rab7 GTPases, which play a role in endosomal transport and acidification, are necessary for eHEV cellular compartment entrance, implying that the endosome must transition to a more acidic cell compartment to allow the uncoating process to take place [34]. Endosomal trafficking appears to be required for cell entry by HEV virions, which can be decreased by inhibiting endosomal acidification. Indeed, culture cells treated with the bafilomycin A1/NH4Cl (lysosomotropic agents) dramatically reduce eHEV's ability to infect cells, implying that viral entry needs endosomal acidification. Low pH, on the other hand, is insufficient for eHEV cell access because it does not affect the density or infectivity of eHEV virions [34].

An independent study confirmed that HEV-like fragments travel to Rab5-positive spaces before being degraded in acidic lysosomal compartments [60]. The present investigation discovered that actin, the membrane-derived cholesterol, and the pathway governed by PI3K are required for HEV incorporation and spread of infection, though a low pH environment is not. Notably, lysosomal lipid degradation is required to get into eHEV. The loss of the late endosomal (and lysosomal) Niemann–Pick disease type C1 protein, which plays a role in cholesterol removal, substantially decreased eHEV infection [34]. Furthermore, inhibiting lipid degradation with a lysosomal acid lipase inhibitor decreased eHEV cell entrance in a dose-dependent fashion [34]. Furthermore, treating cells with cholesterol-sequestering agents significantly reduced VLP uptake [60]. Additionally, cells treated with a unique inhibitor of lysosomal acid lipase, an enzyme crucial to the processing of lipids whose activity hydrolyzes cholesteryl esters and triglycerides in lysosomes [75], showed a dose-related decrease in eHEV infectivity. Accordingly, there was no reduction in non-enveloped HEV spread [34].

It was recently discovered that blocking Rab5 and Rab7, as well as lysosomotropic factors, does not affect naked HEV infectivity [34]. These findings imply that naked HEV particles are uncoated before entering the Rab5+ space and that capsid co-localization to Rab5 is most likely due to empty capsids that remain in the endocytic pathway after genome release. As seen in other non-enveloped viruses [76–78], the viral capsid could come into contact with a mysterious host element and encounter the structural alterations needed for uncoating and genome release into the cytoplasm [79].

ORF3, an eHEV protein that interacts with the capsid [80], has been shown to interact with ORF2 and may disrupt receptor binding [80]. Also, ORF3 has ion channel activity, which is required for HEV egress [31]. Its contribution to eHEV entrance, uncoating, and genetic discharge is still unidentified.

More recently, the Epidermal Growth Factor receptor (EGFR) was suggested to be connected with the HEV admission phase in HepG2 cells, a low invasive HCC cell line, since downregulation of EGFR by siRNA affected viral entry. On the other hand, EGFR has already been involved in the entry process of hepatitis B virus (HBV) and hepatitis C virus [81,82]. These findings could open further therapeutic approaches using EGFR inhibitors in patients with CHE.

Although these findings are encouraging, further study will be needed to verify the importance of these elements in HEV cell adherence and permission during HEV spread of infection, ideally in primary human hepatic cells.

### 3.3. Virion Assembly and Infectious Particle Release

To ensure successful transmission, viruses have created techniques to take control of and modify host cell pathways.

HEV replicase proteins are encoded by ORF 1. After the viral host cell internalization via endocytosis, the translational system generates the ORF1 polyprotein, which moves the
HEV RNA replication process. A negative-strand RNA intermediate is converted into a full-length genomic RNA and a subgenomic RNA type during this phase \[80,81\].

The biological mechanism beneath the metabolism of HEV polyprotein remains one of the most contentious concerns in HEV biology due to its implication as a potential therapeutic target. A previous study concluded that the expressed ORF1 gene product is not proteolytically processed \[83\]. Several studies have shown that the ORF1 polyprotein is proteolytically processed in vaccinia virus, baculovirus, and eukaryotic expression systems, but it is unclear whether this proteolysis is caused by viral or host proteases \[84\].

The latest study endorses the theory that pORF1 is missing HEV protease activity and is processed by host proteases \[85\]. Accordingly, ORF1 expression in cell-free and prokaryotic systems does not result in polyprotein processing \[85\]. Research additionally supported the function of host protein proteases that participate in blood the coagulation process, such as cellular serine proteases Xa (in the papain-like cysteine protease domain) and thrombin (in the X domain), in processing the HEV ORF1 polyprotein \[85\]. To further investigate the polyprotein processing of HEV ORF1, a novel BacMam strategy was recently used, in which the entire HEV genotype 3 genome was cloned into a BacMam vector \[85\]. Specifically, Huh7 immortal cell lines made up of epithelial-like and tumorigenic cells were infected with a recombinant baculovirus containing the HEV genome, obtaining several protein fragments of 18, 35, 37, and 56 kDa, corresponding to papain-like cysteine protease, methyltransferase, RNA-dependent RNA polymerase, and ORF2, respectively, indicating that proteolytic processing occurred \[85\].

ORF2 and ORF3 proteins are produced by translating subgenomic RNA. Later stages of the HEV life cycle include viral assembly and the release of newly produced virions.

Virion assembly is the encapsulation of genomic HEV RNA in the capsid. While the exact location of HEV assembly is undetermined, it is probably linked to distinct replication structures. Even though the underlying processes of human HEV assemblage are unresolved, preliminary studies discovered that RNA and the ORF2 protein can assemble themselves into virus-like particles (VLPs) without RNA in insect cell membranes \[21,82\].

HEV generates two isoforms of the ORF2 capsid protein, which operate in various roles throughout the HEV lifecycle \[27\]. The infectious ORF2 protein is a structural component of virions, whereas the genome-free secreted and glycosylated ORF2 proteins most likely serve as a humoral immune decoy \[27\].

ORF2 protein moves through the nucleus early during infection to control specific cellular functions, such as the infected cell’s antiviral responses. There have been identified factors that influence ORF2 nuclear imports and exports \[27\]. The N-terminal region of ORF2 contains an Arginine-Rich Motif (ARM), which facilitates nuclear import. Furthermore, the ARM plays an important role in fine-tuning the partitioning and stoichiometry of the ORF2 protein across the nuclear, cytosolic, and reticular pathways \[27\]. Additionally, the signal peptide and ARM work together to control the fate of the ORF2 protein. Indeed, in addition to mediating ORF2 targeting the endoplasmic reticulum membrane, the signal peptide is likely capable of forming a reverse signal-anchor topology. This topology inversion is driven by ARM’s flanking charged residues following the positive-inside rule, resulting in the ORF2i protein being anchored to the cytosolic side of membranes \[27\].

There are considerable types of the ORF2 protein, one of which plays a role in virion development \[86–89\]. A specific shape, which contains a shortened and inactive signal peptide, is currently under discussion if it is a product of alternative start site translation \[26\] or proteolytic cleavage \[87\]. The non-glycosylated ORF2 protein is thought to be the major internal shape encountered within the cytosol and cell nucleus \[87–89\].

While the nuclear ORF2 protein’s function is unresolved, the nucleocytoplasmic ORF2 transporting is presumably mediated by the nuclear import and export apparatus. The glycosylated form of ORF2 protein is quickly secreted extracellularly (as opposed to the non-glycosylated form that packages the HEV genome) \[26,86,87,90,91\]. This entails the translocon recognizing its signal peptide, translocating into the endoplasmic reticulum lumen, cleavage by signal peptidase, sialylation, and Nand O-glycosylation by still unknown
glycosyl-transferase enzymes in the endoplasmic reticulum and Golgi apparatus [87], and secretion.

The ORF2 protein is glycosylated in no fewer than two distinct states, the smallest of which could be the result of division by furin-like enzymes at the RRR pattern [88], meaning that the ORF2 protein may play multiple roles in genome packaging mediated by various host variables.

Many studies found that ORF3 is required for viral particle egress and biogenesis of lipid membrane-wrapped HEV particles, which are now known as quasi-enveloped particles.

Because genes with a mutated ORF3 start codon produce infectious components, HEV assembly solely necessitates the non-glycosylated ORF2 material and the viral RNA. These, however, are not released by the cell [92, 93]. The infectious form of HEV is thought to be discharged from the basolateral as well as apical sides of the liver cells, located adjacent to the liver sinusoids and bile canaliculi, respectively [43, 94, 95].

The current model of eHEV biogenesis entails viral hijacking of the cellular endosomal sorting complex required for transport (ESCRT) apparatus (Figure 2), resulting in the formation of an exosome-like vesicle comprising the viral capsid and ORF3 protein [96].

The ESCRT machinery is made up of four structures that take part in membrane remodeling, ESCRT-0 through ESCRT-III [97]. These complexes can also recruit accessory proteins, such as Alix and Vps4, which interact with the ESCRT-III complex [98]. Hrs, an ESCRT-0 member, and Vps4 were also discovered to be needed for HEV particle output [37, 99].

Furthermore, the envelope surrounding the virion is derived from exosomes, which are small vesicles produced by the ESCRT machinery from multivesicular bodies (MVB) [37, 99]. To combine with the outer layer of the plasma membrane and emit its components into the extracellular environment, these MVBs necessitate the Rab27a proteins, which were demonstrated to interact alongside the ORF3 protein [37, 99]. A PSAP late domain pattern at the C-terminal of the ORF3 protein primarily connects with the cellular TSG101 protein, an ESCRT-I protein, which then attracts ESCRT-II and -III structures to stimulate HEV capsid developing into MVB [100]. Following MVB fusion with the plasma membrane, single membrane-encased eHEV particles are released into the circulating blood or into the bile duct, where bile acid destroys the quasi-envelope [35, 37].

As a result, the high viral load of non-enveloped HEV virions observed in feces is most likely due to the destroying process of the quasi-envelope shell of eHEV particles by bile acid, which is primarily expelled via the bile canaliculi from hepatocytes across the apical membrane [101]. Hence, HEV is shed in the feces of infected individuals as stable, non-enveloped virions with their genome encapsulated in a naked protein shell [101]. Likewise, because of the detergent action of bile, naked virions are far more resilient in the surrounding environment, ensuring a direct and easier host spreading.

Additionally, the viral protein ORF3 is required for HEV infectious virions to be secreted, possibly by connecting the capsid to host egress factors. Furthermore, ORF3 protein serine phosphorylation (by p34cdc2 and mitogen-activated protein kinases) may result in an interaction with non-glycosylated ORF2 protein [80, 102].

In addition, the ORF3 protein recruits Tsg101, a member of the endosomal organizing complex required for ESCRT-I, via a highly preserved PSAP motif [99, 100, 103]. The presence of this pattern in the C-terminus of the ORF3 protein is crucial for HEV discharge [100]. Similarly, other viruses with envelopes (such as HIV-1 or Ebola) require distinct proline-rich patterns (or “late domains”) for budding [103, 104].

According to existing proof, none of the HEV proteins are present on the eHEV outer membrane, but ORF3 protein has been recognized beneath the quasi-enveloped type [95, 105, 106]. Palmitoylation of the ORF3 protein’s N-terminal cysteine residues mediates membrane association and stability, as well as virus secretion [107]. As a result, a membrane topology with ORF3 protein completely on the cytosolic side that corresponds to the exosome lumen was provided [107], and this is confirmed by the PSAP motif’s relationship to Tsg101. As a result, a membrane topology in which ORF3 protein is entirely on the cytosolic side, corresponding to the exosome lumen, has been proposed [107], which
is supported by the interaction of its PSAP motif with Tsg101. S-palmitoyl-transferase-mediated palmitoylation is a reversible change process to a protein that takes place in the cytosol and raises a protein’s hydrophobic nature while also contributing to membrane association [108].

Host proteins, especially exosomal proteins, can be observed on eHEV due to their origin. Of course, viral fragments provided from ORF2/ORF3 protein-positive infected cells have been identified as having unique exosomal elements (i.e., tetraspanins, CD63, CD81, and CD9, as well as Tsg101 and Alix) [37, 95, 109]. Exosomes containing HEV genetic material have been displayed to spread other viruses productively and to regulate cell reactions [110].

Given that HEV may benefit from the quasi-enveloped nature of bloodstream virions, including immunity from neutralizing antibodies, their production may serve some function other than propagating HEV throughout the liver.

On the other hand, a fascinating experimental study shed light on a potential mechanism of HEV-associated neuroinvasion [3]. The study found that quasi-enveloped and naked HEVs can cross the blood–brain barrier without a TNF-α mediated process and infect brain microvascular endothelial cells in vitro [3].

### 3.4. Therapeutic Options for HEV Infection

The HEV’s multifaceted infectiousness and the lack of a cell culture system have limited the attempts to adequately dissect the interaction with the host and to set up antiviral strategies, particularly regarding the inhibition of viral entry. However, recent innovations in cell cultures showed that HEV has an array of replication phases similar to those found in other RNA viruses. Consequently, although substances inhibiting viral entry are not yet at hand for clinical use, several antivirals already used with other RNA viruses and other still experimental drugs in the preclinical phase could offer a highly desirable therapeutic option, as chronic HEV infection with genotype 3 has been linked with chronic liver disease and cirrhosis development in immunocompromised patients [111].

Early experimental data with sofosbuvir [112] were not confirmed in clinical studies [113]. Sofosbuvir monotherapy in 10 immunocompromised patients obtained only a moderate decrease in viremia with a rebound after treatment completion [113].

Non-pregnant patients with chronic HEV infection were shown to benefit from a 12-week course of ribavirin (RBV) monotherapy [111]. RBV is a guanosine nucleoside analog that acts as a viral polymerase inhibitor, and it has been usually given to solid organ transplant recipients in conjunction with immunosuppressive therapy reduction [114–129]. Table 1 contains details about the described drugs and their mechanisms of action in immunocompromised patients.

Although the mechanism of action is not well defined [130], Marion et al. recently showed that RBV inhibited HEV secretion from the basal side of intestinal cells by more than 80% but only by 20% from the apical side. While this disparity remains unexplained, it highlights the importance of future studies evaluating the efficacy of antiviral drugs in both the liver and the gut [43]. Interestingly, the same authors showed that the HEV in the intestinal reservoir might be partially resistant to ribavirin, thus offering a possible explanation for the endogenous reinfections ensuing ribavirin withdrawal in chronically HEV-infected individuals [43]. Of note, the emergence of viral mutations may contribute to resistance against ribavirin treatment.

Combination treatments with ibrutinib and RBV revealed that ibrutinib had no effect on RBV’s antiviral effect and did not directly promote HEV replication in an elderly patient with lymphoplasmacytic lymphoma [127].

In terms of other potential substances targeting non-entry viral sites, 3-(4-Hydroxyphenyl) propionic acid, a HEV methyltransferase inhibitor, revealed a substantial virological response in an in silico analysis. This substance, however, will require additional validation by in vivo testing in animal models and pharmacokinetic/pharmacodynamic investigations [128].
| Author           | Year | Study Type       | Immunosuppressed Patient Group                                      | N.of pts | HEV Genotype (N.pts) | Drug        | Mechanism of Action                          | Dosage/Daily | Treatment Duration(mo) | Virological Response,% | Adverse Events     |
|------------------|------|------------------|--------------------------------------------------------------------|----------|---------------------|-------------|----------------------------------------------|--------------|------------------------|-----------------------|--------------------|
| Kamar [114]      | 2010 | Pilot study      | Kidney transplantation/simultaneous kidney-pancreas transplantation | 6        | 3f(4),3c(2)         | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 3                      | 100                   | Anemia             |
| Mallet [115]     | 2010 | Case report      | Simultaneous kidney-pancreas transplantation, hematological disease | 2        | 3f(1),3c(1)         | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 3                      | 100                   | Not relevant       |
| Alric [116]      | 2011 | Case report      | Hematological disease                                              | 1        | 3c                  | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 3                      | 100                   | Not relevant       |
| Chaillon [117]   | 2011 | Case report      | Heart transplantation                                              | 1        | 3c                  | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 3                      | 100                   | Well-tolerated anemia |
| de Niet [118]    | 2012 | Case report      | Kidney transplantation                                             | 1        | 3                   | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 3                      | 100                   | Not relevant       |
| Del Bello [119]  | 2012 | Case report      | Liver transplantation                                              | 1        | 3f                  | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 3                      | 100                   | Not relevant       |
| Pischke [120]    | 2012 | Prospective      | Heart transplantation                                              | 4        | n.a.                | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 5                      | 75                    | Not relevant       |
| Hajji [121]      | 2013 | Case report      | HIV                                                                | 1        | 3f                  | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 3                      | 100                   | Not relevant       |
| Junge [122]      | 2013 | Case report      | Liver transplantation                                              | 1        | n.a.                | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 6                      | 100                   | Not relevant       |
| Koning [123]     | 2013 | Retrospective    | Heart transplantation                                              | 4        | 3                   | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 3-12                   | 75                    | Not relevant       |
| Neukam [124]     | 2013 | Case report      | HIV                                                                | 1        | 3                   | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 6                      | 100                   | Not relevant       |
| Pischke [125]    | 2013 | Prospective case series | kidney transplantation, heart transplantation, lung transplantation | 11       | n.a.                | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 5                      | 82                    | Not relevant       |
| Riezebos-Brilman [126] | 2013 | Case report | Lung transplantation                                              | 2        | 3                   | RBV monotherapy | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 4                      | 100                   | Not relevant       |

Table 1. Cont.

| Author         | Year | Study Type                          | Immunosuppressed Patient Group                              | N.of pts | HEV Genotype (N.pts) | Drug      | Mechanism of Action                                   | Dosage/Daily | Treatment Duration(mo) | Virological Response,% | Adverse Events       |
|----------------|------|-------------------------------------|-----------------------------------------------------------|----------|----------------------|-----------|-------------------------------------------------------|--------------|------------------------|-----------------------|----------------------|
| Schlevogt B    | 2019 | Case report + in vitro studies      | Lymphoplasmacytic lymphoma (hepatoma cell line HepG 2)     | 1        | 3c                   | RBV       | Guanosine nucleoside analog(viral polymerase inhibitor) | 600-1000mg   | 2                      | 100                   | Severe exanthema     |
| [127]          |      |                                     |                                                           |          |                      | Ibrutinib | BTK inhibitor                                         | 3.33 microM   | 48h                    | Moderate antiviral effect against HEV replicating isolates | No cytotoxicity |
| Hooda P        | 2022 | In silico (Docking analysis)        | n.a.                                                      | n.a.     | N.A.                 | HPPA      | HEV MTase inhibitor                                    | 800M         | n.a.                   | HEV-RNA copies decreased significantly from ~3.2×106 in untreated cells to ~4.3×102.8 | Because the homology between human and HEV-MTase is so low, HPPA is unlikely to affect the host Mtase |
| [128]          |      |                                     |                                                           |          |                      |           |                                                       |              |                        |                       |                      |
| Bhise N        | 2023 | Drug repurposing strategy: experimental + in silico (Docking analysis) | n.a.                                                      | n.a.     | 1-3                  | ART       | (A) binding to the HEV helicase active site, potentially affecting ATP hydrolysis activity; (B) inhibition of the HEVRdRp | 19.5µM        | (EC 50) and 78 µM       | n.a.                   | (A) ART showed a 24% and 55% inhibition of the helicase; (B) ART showed 26% and 40% inhibition of the RdRp activity | In silico toxicity studies revealed no risk of reproductive or developmental toxicity |
| [129]          |      |                                     |                                                           |          |                      |           |                                                       |              |                        |                       |                      |

HEV, Hepatitis E virus; RBV, ribavirin; HIV, human immunodeficiency virus; BTK, Bruton's tyrosine kinase; HPPA, 3-(4-Hydroxyphenyl) propionic acid; Mtase, methyltransferase; ART, Artesunate; RdRp, RNA-dependent RNA polymerase, n.a.=Not available.
A recent study \[129\] showed that the antimalarial Artesunate was able to inhibit HEV replication by directly blocking both the helicase and the RNA-dependent RNA polymerase. A further element of interest about this drug is represented by the fact that it is safe in pregnant women, who are well-known high-risk individuals. However, it may be moderately mutagenic, tumorigenic, and hepatotoxic if utilized in elevated dosages for a lengthy period, thus requiring further testing in animal models.

Because HBV and HEV endemicity overlap significantly in many Asian countries where double HEV-HBV infections can occur, HEV superinfection in patients with chronic hepatitis B can result in acute exacerbation of underlying viral hepatitis, raising the chance of developing decompensated cirrhosis and mortality while displaying a subclinical course of acute HEV infection during pregnancy \[131\].

To reduce the likelihood of HEV epidemics, preventive measures should be carried out, with investigational HEV vaccines playing a critical role. HEV vaccination has been developed and is being used in China, but its efficacy in CHB patients has yet to be established in the United States \[131\].

Several HEV vaccines are currently in development, with some already available and demonstrating promising results. A 2007 phase-II trial, despite a high vaccine efficacy, has previously failed due to a lack of commercial value of HEV recombinant protein \[132\]. In 2010, a vaccine obtained from the ORF2 capsid protein of HEV genotype 1 was developed and licensed in China under the name Hecolin® \[133\]. All HEV vaccine trials showed cross-genotype activity because HEV genotype 4 is the most common in China \[134\]. After nearly five years of vaccination, this vaccine provided long-term HEV protection in most previously seronegative patients \[135\]. Additionally, it was predicted that anti-HEV immunogenicity could be maintained for up to 30 years following vaccination \[136\]. Notably, many HEV vaccine studies have been conducted in large Asian populations, demonstrating high efficacy and safety even in preventing maternal and neonatal deaths in non-pregnant women \[137\] (NCT03168412, NCT02417597, and NCT02759991). Regardless, a phase Ia/Ib trial in the United States with 25 healthy non-pregnant females confirmed a total immunoglobulin serological response to the Hecolin® vaccine (NCT03827395). Two further vaccine candidates are being tested in clinical trials with VLPs from insect cells and *E. coli* \[138\].

## 4. Conclusions and Future Directions

The considerable and changing epidemiologic scenario and the threat of HEV transmission highlight the significance of future HEV prophylactic healthcare measures.

The evidence indicates that HEV can infect intestinal cells, supporting the idea that the gut is the site of initial infection and viral amplification before being transported to the liver \[71\]. As a result, the gut may act as a reservoir for HEV replication, especially during chronic infection, which, together with the emergence of viral mutations, could explain viral relapses in some treated patients.

Specifically, the cell access strategies utilized by non-enveloped and quasi-enveloped HEV components diverge, and HEV entrance necessitates lysosomal degradation of the viral membrane. Of course, understanding the entry mechanisms of both HEV virion types will depend on discovering their specific cell-surface receptors.

Even though HEV investigation is a quickly expanding domain, the current knowledge of the virus’s life cycle must be enhanced. We are missing critical information on how HEV viruses enter cells, such as cellular receptors and viral RNA uncoating. Future research should also focus on the biological processes and distinct compartments within hepatic cells that handle the replication of RNA and viral assemblage.

While HEV release constitutes one of the most extensively researched phases in the virus’s life cycle, several aspects, particularly the contribution of host-related variables and the nature and function of quasi-enveloped particles, persist unidentified, limiting the discovery of HEV-specific antiviral agents. Given HEV’s skill to reproduce in a variety of
tissues and infect an extensive number of beings, it might hypothesized that HEV host dependency may not be highly selective, enabling the virus to cross biodiversity barriers.

Nowadays, research studies on the HEV biological clock rely on laboratory system models that suffer from limitations, particularly in terms of host factors present in differentiated hepatocytes. For example, cyclophilin inhibitors were found to have a stimulatory effect on HEV replication in hepatoma cells [71]. This was demonstrated in stem cell-derived hepatocyte-like cells using a modified in-lab cell clone infectious model but not with authentic HEV isolates [139].

Future enhancements to in vitro experiments ought to encompass the application of naturally produced HEV isolates together with original and hepatocyte-like cells derived from stem cells, polarized cell cultivation models, and both ex vivo/in vivo infection experiments to verify the laboratory outcomes. Key findings in hepatic samples from hepatitis E individuals must eventually be confirmed.

To acquire a more comprehensive and uninfluenced picture of the host-related variables engaged in the HEV life cycle, cutting-edge methods that incorporate CRISPR/Cas9-based genome-wide examination (that have advanced our understanding of virus entrance [140, 141], reproduction [142, 143], and the designation of unique therapeutic targets [144] and proteomic proximity determining methods (as successfully employed for characterizing the microenvironment of coronavirus replication complexes [145])) are expected to be used.

In the future, incorporating enhanced HEV simulators and technological innovations should produce a greater comprehension of the host-related variables necessary for effective HEV infection.

Ultimately, subsequent studies emphasizing the immune-mediated effects of HEV infection will probably aid in developing personalized therapies, enhancing the clinical outcomes for individuals with acute and chronic HEV hepatitis and conceivably preventing HEV-associated liver cancer.

**Author Contributions:** Conceptualization, investigation, writing—original draft preparation, E.S.; methodology, E.S. and E.V.; writing—review and editing, E.S., A.A. (Antonella Argentiero) and A.A. (Alessandro Andriano); data curation, E.S. and A.A. (Antonella Argentiero); supervision, A.A. (Antonella Argentiero), F.L., M.M., A.F., R.C. and E.V.; project administration, E.S. and E.V.; validation, formal analysis, E.S., A.A. (Antonella Argentiero), A.A. (Alessandro Andriano), F.L., M.M., A.F., R.C. and E.V. All authors have read and agreed to the published version of the manuscript.

**Funding:** This research received no external funding.

**Acknowledgments:** The authors acknowledge Biorender for providing comprehensive medical and biological figures and fruitful datasets for the international scientific community. The authors' publishing license has been deposited under agreement number BR25P4VMZE.

**Conflicts of Interest:** The authors declare no conflicts of interest.

**References**

1. Nimgaonkar, I.; Ding, Q.; Schwartz, R.E.; Ploss, A. Hepatitis E Virus: Advances and Challenges. *Nat. Rev. Gastroenterol. Hepatol.* **2018**, *15*, 96–110. [CrossRef]
2. Gupta, D.N.; Smetana, H.F. The Histopathology of Viral Hepatitis as Seen in the Delhi Epidemic (1955–56). *Indian J. Med. Res.* **1957**, *45*, 101–113.
3. Tian, D.; Li, W.; Heffron, C.L.; Wang, B.; Mahsoub, H.M.; Sooryanarain, H.; Hassebroek, A.M.; Clark-Deener, S.; LeRoith, T.; Meng, X.-J. Hepatitis E Virus Infects Brain Microvascular Endothelial Cells, Crosses the Blood-Brain Barrier, and Invades the Central Nervous System. *Proc. Natl. Acad. Sci. USA* **2022**, *119*, e2201862119. [CrossRef]
4. Rein, D.B.; Stevens, G.A.; Theaker, J.; Wittenborn, J.S.; Wiersma, S.T. The Global Burden of Hepatitis E Virus Genotypes 1 and 2 in 2005. *Hepatology* **2012**, *55*, 988–997. [CrossRef]
5. Geng, Y.; Zhao, C.; Geng, K.; Wang, C.; Wang, X.; Liu, H.; Wang, Y. High Seroprevalence of Hepatitis E Virus in Rabbit Slaughterhouse Workers. *Transbound. Emerg. Dis.* **2019**, *66*, 1085–1089. [CrossRef]
6. Halánová, M.; Veseliny, E.; Kalinová, Z.; Jarčuška, P.; Janičko, M.; Urbančíková, I.; Pella, D.; Dražilová, S.; Babinská, I.; HepaMeta Team. Seroprevalence of Hepatitis E Virus in Roma Settlements: A Comparison with the General Population in Slovakia. *Int. J. Environ. Res. Public Health* **2018**, *15*, 904. [CrossRef]

7. Dalton, H.R.; Bendall, R.; Ijaz, S.; Banks, M. Hepatitis E: An Emerging Infection in Developed Countries. *Lancet Infect. Dis.* **2008**, *8*, 698–709. [CrossRef]

8. Khuroo, M.S.; Khuroo, M.S.; Khuroo, N.S. Hepatitis E: Discovery, global impact, control and cure. *World J. Gastroenterol.* **2016**, *22*, 7030–7045. [CrossRef]

9. Meng, X.J.; Purcell, R.H.; Halbur, P.G.; Lehman, J.R.; Webb, D.M.; Tsareva, T.S.; Haynes, J.S.; Thacker, B.J.; Emerson, S.U. A Novel Virus in Swine Is Closely Related to the Human Hepatitis E Virus. *Proc. Natl. Acad. Sci. USA* **1997**, *94*, 9860–9865. [CrossRef]

10. Meng, X.-J. Expanding Host Range and Cross-Species Infection of Hepatitis E Virus. *PLoS Pathog.* **2016**, *12*, e1005695. [CrossRef]

11. Sridhar, S.; Yip, C.C.-Y.; Wu, S.; Chew, N.F.-S.; Leung, K.-H.; Chan, J.F.-W.; Zhao, P.S.; Chan, W.-M.; Poon, R.W.-S.; Tsoi, H.-W.; et al. Transmission of Rat Hepatitis E Virus Infection to Humans in Hong Kong: A Clinical and Epidemiological Analysis. *Hepatology* **2021**, *73*, 10–22. [CrossRef]

12. Cangin, C.; Focht, B.; Harris, R.; Strunk, J.A. Hepatitis E Seroprevalence in the United States: Results for Immunoglobulins IGG and IGM. *J. Med. Virol.* **2019**, *91*, 124–131. [CrossRef]

13. Sun, P.; Lin, S.; He, S.; Zhou, E.-M.; Zhao, Q. Avian Hepatitis E Virus: With the Trend of Genotypes and Host Expansion. *Front. Microbiol.* **2019**, *10*, 1696. [CrossRef]

14. Ahmed, R.; Nasheri, N. Animal reservoirs for hepatitis E virus within the Paslahepevirus genus. *Vet. Microbiol.* **2023**, *278*, 109618. [CrossRef]

15. Nicot, F.; Dimeglio, C.; Miguereas, M.; Jeanne, N.; Latour, J.; Abravanel, F.; Ranger, N.; Harter, A.; Dubois, M.; Lameiras, S.; et al. Classification of the Zoonotic Hepatitis E Virus Genotype 3 Into Distinct Subgenotypes. *Front. Microbiol.* **2021**, *11*, 634430. [CrossRef] [PubMed]

16. Andonov, A.; Robbins, M.; Borlang, J.; Cao, J.; Hatchette, T.; Stueck, A.; Deschambault, Y.; Murnaghan, K.; Varga, J.; Johnston, L. Rat Hepatitis E Virus Linked to Severe Acute Hepatitis in an Immunocompetent Patient. *J. Infect. Dis.* **2019**, *220*, 951–955. [CrossRef] [PubMed]

17. Balayan, M.S.; Andjaparidze, A.G.; Savinskaya, S.S.; Ketiladze, E.S.; Braginsky, D.M.; Savinov, A.P.; Poleschuk, V.F. Evidence for a Virus in Non-A, Non-B Hepatitis Transmitted via the Fecal-Oral Route. *Intervirology* **1983**, *20*, 23–31. [CrossRef] [PubMed]

18. Yadav, K.K.; Boley, P.A.; Fritts, Z.; Kenney, S.P. Ectopic Expression of Genotype 1 Hepatitis E Virus ORF4 Increases Genotype 3 HEV Viral Replication in Cell Culture. *Viruses* **2021**, *13*, 75. [CrossRef]

19. Krawczynski, K.; Bradley, D.W. Enterically Transmitted Non-A, Non-B Hepatitis: Identification of Virus-Associated Antigen in Experimentally Infected Cynomolgus Macaques. *J. Infect. Dis.* **1989**, *159*, 1042–1049. [CrossRef]

20. Yamashita, T.; Mori, Y.; Miyazaki, N.; Cheng, R.H.; Yoshimura, M.; Unno, H.; Shima, R.; Moriishi, K.; Tsukihara, T.; Li, T.C.; et al. Biological and Immunological Characteristics of Hepatitis E Virus-like Particles Based on the Crystal Structure. *Proc. Natl. Acad. Sci. USA* **2009**, *106*, 12986–12991. [CrossRef]

21. Guu TS, Y.; Liu, Z.; Ye, Q.; Mata, D.A.; Li, K.; Yin, C.; Zhang, J.; Tao, Y.J. Structure of the Hepatitis E Virus-like Particle Suggests Mechanisms for Virus Assembly and Receptor Binding. *Proc. Natl. Acad. Sci. USA* **2009**, *106*, 12992–12997. [CrossRef] [PubMed]

22. Primadharsini, P.P.; Nagashima, S.; Okamoto, H. Genetic Variability and Evolution of Hepatitis E Virus. *Viruses* **2019**, *11*, 456. [CrossRef]

23. Purdy, M.A.; Drexler, J.F.; Meng, X.-J.; Norder, H.; Okamoto, H.; Van der Poel WH, M.; Reuter, G.; de Souza, W.M.; Ulrich, R.G.; Smith, D.B. ICTV Virus Taxonomy Profile: Hepeviridae 2022. *J. Gen. Virol.* **2022**, *103*, 001778. [CrossRef] [PubMed]

24. Koonin, E.V.; Gorbalenya, A.E.; Purdy, M.A.; Rozanov, M.N.; Reyes, G.R.; Bradley, D.W. Computer-Assisted Assignment of Functional Domains in the Nonstructural Polyprotein of Hepatitis E Virus: Delineation of an Additional Group of Positive-Strand RNA Plant and Animal Viruses. *Proc. Natl. Acad. Sci. USA* **1992**, *89*, 8259–8263. [CrossRef] [PubMed]

25. Cao, D.; Meng, X.-J. Molecular Biology and Replication of Hepatitis E Virus. *Emerg. Microbes Infect.* **2012**, *1*, e17. [CrossRef] [PubMed]

26. Yin, X.; Ying, D.; Lhomme, S.; Tang, Z.; Walker, C.M.; Xia, N.; Zheng, Z.; Feng, Z. Origin, Antigenicity, and Function of a Secreted Form of ORF2 in Hepatitis E Virus Infection. *Proc. Natl. Acad. Sci. USA* **2018**, *115*, 4773–4778. [CrossRef] [PubMed]

27. Hervouet, K.; Ferrié, M.; Ankavay, M.; Montpellier, C.; Camuzet, C.; Alexandre, V.; Dembélé, A.; Lecoeur, C.; Foe, A.T.; Bouquet, P.; et al. An Arginine-Rich Motif in the ORF2 capsid protein regulates the hepatitis E virus lifecycle and interactions with the host cell. *PLoS Pathog.* **2022**, *18*, e1010798. [CrossRef] [PubMed]

28. Xing, L.; Li, T.C.; Mayazaki, N.; Simon, M.N.; Wall, J.S.; Moore, M.; Wang, C.Y.; Takeda, N.; Wakita, T.; Miyamura, T.; et al. The structure of hepatitis E virion-sized particles reveals an RNA-dependent viral assembly pathway. *J. Biol. Chem.* **2010**, *285*, 33175–33183. [CrossRef] [PubMed]

29. Meng, X.J. Recent Advances in Hepatitis E Virus. *J. Viral Hepat.* **2010**, *17*, 153–161. [CrossRef]

30. Ali, M.M.; Gul, M.; Imran, M.; Ijaz, M.; Azeem, S.; Ullah, A.; Yaqub, H.M.F. Molecular identification and genotyping of hepatitis E virus from Southern Punjab, Pakistan. *Sci. Rep.* **2024**, *14*, 223. [CrossRef]

31. Ding, Q.; Heller, B.; Capuccino JM, V.; Song, B.; Nimgaonkar, I.; Hrebikova, G.; Contreras, J.E.; Ploss, A. Hepatitis E Virus ORF3 Is a Functional Ion Channel Required for Release of Infectious Particles. *Proc. Natl. Acad. Sci. USA* **2017**, *114*, 1147–1152. [CrossRef] [PubMed]

32. Nair, V.P.; Anang, S.; Subramani, C.; Madhvi, A.; Bakshi, K.; Srivastava, A.; Shalimar; Nayak, B.; Ranjith Kumar, C.T.; Surjit, M. Endoplasmic Reticulum Stress Induced Synthesis of a Novel Viral Factor Mediates Efficient Replication of Genotype-1 Hepatitis E Virus. *PLoS Pathog.* **2016**, *12*, e1005521. [CrossRef] [PubMed]

33. Das, A.; Rivera-Serrano, E.E.; Yin, X.; Walker, C.M.; Feng, Z.; Lemon, S.M. Cell entry and release of quasi-enveloped human hepatitis viruses. *Nat. Rev. Microbiol.* 2023, *21*, 573–589. [CrossRef](https://doi.org/10.1038/s41579-023-00860-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

34. Yin, X.; Ambardekar, C.; Lu, Y.; Feng, Z. Distinct Entry Mechanisms for Nonenveloped and Quasi-Enveloped Hepatitis E Viruses. *J. Virol.* 2016, *90*, 4232–4242. [CrossRef](https://doi.org/10.1128/JVI.00026-16) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

35. Nagashima, S.; Takahashi, M.; Kobayashi, T.; Tanggis; Nishizawa, T.; Nishiyama, T.; Primadharsini, P.P.; Okamoto, H. Characterization of the Quasi-Enveloped Hepatitis E Virus Particles Released by the Cellular Exosomal Pathway. *J. Virol.* 2017, *91*, e00822-17. [CrossRef](https://doi.org/10.1128/JVI.00822-17) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

36. Nagashima, S.; Jirintai, S.; Takahashi, M.; Kobayashi, T.; Tanggis; Nishizawa, T.; Kouki, T.; Yashiro, T.; Okamoto, H. Hepatitis E Virus Egress Depends on the Exosomal Pathway, with Secretory Exosomes Derived from Multivesicular Bodies. *J. Gen. Virol.* 2014, *95 Pt 10*, 2166–2175. [CrossRef](https://doi.org/10.1099/vir.0.067088-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

37. Takahashi, M.; Tanaka, T.; Takahashi, H.; Hoshino, Y.; Nagashima, S.; Jirintai; Mizuo, H.; Yazaki, Y.; Takagi, T.; Azuma, M.; et al. Hepatitis E Virus (HEV) Strains in Serum Samples Can Replicate Efficiently in Cultured Cells despite the Coexistence of HEV Antibodies: Characterization of HEV Virions in Blood Circulation. *J. Clin. Microbiol.* 2010, *48*, 1112–1125. [CrossRef](https://doi.org/10.1128/JCM.01858-09)

38. Okamoto, H. Culture Systems for Hepatitis E Virus. *J. Gastroenterol.* 2013, *48*, 147–158. [CrossRef](https://doi.org/10.1007/s00535-012-0687-0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

39. Geng, Y.; Yang, J.; Huang, W.; Harrison, T.J.; Zhou, Y.; Wen, Z.; Wang, Y. Virus Host Protein Interaction Network Analysis Reveals That the HEV ORF3 Protein May Interrupt the Blood Coagulation Process. *PLoS ONE* 2013, *8*, e56320. [CrossRef](https://doi.org/10.1371/journal.pone.0056320)

40. Subramani, C.; Nair, V.P.; Anang, S.; Mandal, S.D.; Pareek, M.; Kaushik, N.; Srivastava, A.; Saha, S.; Shalimar; Nayak, B.; et al. Host-Virus Protein Interaction Network Reveals the Involvement of Multiple Host Processes in the Life Cycle of Hepatitis E Virus. *mSystems* 2018, *3*, e00135-17. [CrossRef](https://doi.org/10.1128/mSystems.00135-17)

41. Fu, R.M.; Decker, C.C.; Dao Thi, V.L. Cell Culture Models for Hepatitis E Virus. *Viruses* 2019, *11*, 608. [CrossRef](https://doi.org/10.3390/v11060608) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

42. Meister, T.L.; Bruening, J.; Todt, D.; Steinmann, E. Cell Culture Systems for the Study of Hepatitis E Virus. *Antivir. Res.* 2019, *163*, 34–49. [CrossRef](https://doi.org/10.1016/j.antiviral.2018.12.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

43. Marion, O.; Lhomme, S.; Nayrac, M.; Dubois, M.; Pucelle, M.; Requena, M.; Miguereas, M.; Abravanel, F.; Peron, J.M.; Carrere, N.; et al. Hepatitis E Virus Replication in Human Intestinal Cells. *Gut* 2020, *69*, 901–910. [CrossRef](https://doi.org/10.1136/gutjnl-2019-319654) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

44. Drave, S.A.; Debing, Y.; Walter, S.; Todt, D.; Engelmann, M.; Friesland, M.; Wedemeyer, H.; Neyts, J.; Behrendt, P.; Steinmann, E. Extra-Hepatic Replication and Infection of Hepatitis E Virus in Neuronal-Derived Cells. *J. Viral Hepat.* 2016, *23*, 512–521. [CrossRef](https://doi.org/10.1111/jvh.12455) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

45. Gouilly, J.; Chen, Q.; Siewiera, J.; Cartron, G.; Levy, C.; Dubois, M.; Al-Daccak, R.; Izopet, J.; Jabrane-Ferrat, N.; El Costa, H. Genotype Specific Pathogenicity of Hepatitis E Virus at the Human Maternal-Fetal Interface. *Nat. Commun.* 2018, *9*, 4748. [CrossRef](https://doi.org/10.1038/s41467-018-07170-7) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

46. Knegendorf, L.; Drave, S.A.; Dao Thi, V.L.; Debing, Y.; Brown RJ, P.; Vondran FW, R.; Resner, K.; Friesland, M.; Khera, T.; Engelmann, M.; et al. Hepatitis E Virus Replication and Interferon Responses in Human Placental Cells. *Hepatol. Commun.* 2018, *2*, 173–187. [CrossRef](https://doi.org/10.1002/hep4.1155) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

47. Shukla, P.; Nguyen, H.T.; Torian, U.; Engle, R.E.; Faulk, K.; Dalton, H.R.; Bendall, R.P.; Keane, F.E.; Purcell, R.H.; Emerson, S.U. Cross-Species Infections of Cultured Cells by Hepatitis E Virus and Discovery of an Infectious Virus-Host Recombinant. *Proc. Natl. Acad. Sci. USA* 2011, *108*, 2438–2443. [CrossRef](https://doi.org/10.1073/pnas.1016762108) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

48. Zhou, X.; Huang, F.; Xu, L.; Lin, Z.; de Vrij FM, S.; Ayo-Martin, A.C.; van der Kroeg, M.; Zhao, M.; Yin, Y.; Wang, W.; et al. Hepatitis E Virus Infects Neurons and Brains. *J. Infect. Dis.* 2017, *215*, 1197–1206. [CrossRef](https://doi.org/10.1093/infdis/jix010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

49. Kamar, N.; Bendall, R.; Legrand-Abravanel, F.; Xia, N.-S.; Ijaz, S.; Izopet, J.; Dalton, H.R. Hepatitis E. *Lancet* 2012, *379*, 2477–2488. [CrossRef](https://doi.org/10.1016/S0140-6736(12)60290-2)

50. Debing, Y.; Moradpour, D.; Neyts, J.; Gouttenoire, J. Update on Hepatitis E Virology: Implications for Clinical Practice. *J. Hepatol.* 2016, *65*, 200–212. [CrossRef](https://doi.org/10.1016/j.jhep.2016.03.033)

51. Geng, Y.; Zhao, C.; Huang, W.; Harrison, T.J.; Zhang, H.; Geng, K.; Wang, Y. Detection and Assessment of Infectivity of Hepatitis E Virus in Urine. *J. Hepatol.* 2016, *64*, 37–43. [CrossRef](https://doi.org/10.1016/j.jhep.2015.09.011) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

52. Kamar, N.; Abravanel, F.; Lhomme, S.; Rostaing, L.; Izopet, J. Hepatitis E Virus: Chronic Infection, Extra-Hepatic Manifestations, and Treatment. *Clin. Res. Hepatol. Gastroenterol.* 2015, *39*, 20–27. [CrossRef](https://doi.org/10.1016/j.clinre.2014.10.004)

53. Jemieliity, S.; Wang, J.J.; Chan, Y.K.; Ahmed, A.A.; Li, W.; Monahan, S.; Bu, X.; Farzan, M.; Freeman, G.J.; Umetsu, D.T.; et al. TIM-Family Proteins Promote Infection of Multiple Enveloped Viruses through Virion-Associated Phosphatidylserine. *PLoS Pathog.* 2013, *9*, e1003232. [CrossRef](https://doi.org/10.1371/journal.ppat.1003232) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

54. Das, A.; Maury, W.; Lemon, S.M. TIM1 (HAVCR1): An Essential “Receptor” or an “Accessory Attachment Factor” for Hepatitis A Virus? *J. Virol.* 2019, *93*, e01793-18. [CrossRef](https://doi.org/10.1128/JVI.01793-18) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

55. Yu, H.; Li, S.; Yang, C.; Wei, M.; Song, C.; Zheng, Z.; Gu, Y.; Du, H.; Zhang, J.; Xia, N. Homology Model and Potential Virus-Capsid Binding Site of a Putative HEV Receptor Grp78. *J. Mol. Model.* 2011, *17*, 987–995. [CrossRef](https://doi.org/10.1007/s00894-010-0784-7) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

56. Zhang, L.; Tian, Y.; Wen, Z.; Zhang, F.; Qi, Y.; Huang, W.; Zhang, H.; Wang, Y. Asialoglycoprotein Receptor Facilitates Infection of PLC/PRF/5 Cells by HEV through Interaction with ORF2. *J. Med. Virol.* 2016, *88*, 2186–2195. [CrossRef](https://doi.org/10.1002/jmv.24540) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

57. Kalia, M.; Chandra, V.; Rahman, S.A.; Sehgal, D.; Jameel, S. Heparan Sulfate Proteoglycans Are Required for Cellular Binding of the Hepatitis E Virus ORF2 Capsid Protein and for Viral Infection. *J. Virol.* 2009, *83*, 12714–12724. [CrossRef](https://doi.org/10.1128/JVI.00955-09) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

58. Yin, X.; Li, X.; Feng, Z. Role of Envelopment in the HEV Life Cycle. *Viruses* 2016, *8*, 229. [CrossRef](https://doi.org/10.3390/v8060160) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

59. Kapur, N.; Thakral, D.; Durgapal, H.; Panda, S.K. Hepatitis E Virus Enters Liver Cells through Receptor-Dependent Clathrin-Mediated Endocytosis. *J. Viral Hepat.* 2012, **19**, 436–448. [CrossRef]

60. Holla, P.; Ahmad, I.; Ahmed, Z.; Jameel, S. Hepatitis E Virus Enters Liver Cells through a Dynamin-2, Clathrin and Membrane Cholesterol-Dependent Pathway. *Traffic* 2015, **16**, 398–416. [CrossRef]

61. Chu, H.; Chan, C.-M.; Zhang, X.; Wang, Y.; Yuan, S.; Zhou, J.; Au-Yeung, R.K.-H.; Sze, K.-H.; Yang, D.; Shuai, H.; et al. Middle East Respiratory Syndrome Coronavirus and Bat Coronavirus HKU9 Both Can Utilize GRP78 for Attachment onto Host Cells. *J. Biol. Chem.* 2018, **293**, 11709–11726. [CrossRef] [PubMed]

62. Nain, M.; Mukherjee, S.; Karmakar, S.P.; Paton, A.W.; Paton, J.C.; Abdin, M.Z.; Basu, A.; Kalia, M.; Vrati, S. GRP78 Is an Important Host Factor for Japanese Encephalitis Virus Entry and Replication in Mammalian Cells. *J. Virol.* 2017, **91**, e02274-16. [CrossRef] [PubMed]

63. Triantafilou, K.; Fradelizi, D.; Wilson, K.; Triantafilou, M. GRP78, a Coreceptor for Coxsackievirus A9, Interacts with Major Histocompatibility Complex Class I Molecules Which Mediate Virus Internalization. *J. Virol.* 2002, **76**, 633–643. [CrossRef] [PubMed]

64. Honda, T.; Horie, M.; Daito, T.; Ikuta, K.; Tomonaga, K. Molecular Chaperone BiP Interacts with Borna Disease Virus Glycoprotein at the Cell Surface. *J. Virol.* 2009, **83**, 12622–12625. [CrossRef] [PubMed]

65. Fongsaran, C.; Jirakanwisal, K.; Kuadkitkan, A.; Wikan, N.; Wintachai, P.; Thepparit, C.; Ubol, S.; Phaonakrop, N.; Roytrakul, S.; Smith, D.R. Involvement of ATP Synthase β Subunit in Chikungunya Virus Entry into Insect Cells. *Arch. Virol.* 2014, **159**, 3353–3364. [CrossRef]

66. Ueda, K.; Suwanmanee, Y. ATP5B Is an Essential Factor for Hepatitis B Virus Entry. *Int. J. Mol. Sci.* 2022, **23**, 9570. [CrossRef] [PubMed]

67. Shiota, T.; Li, T.-C.; Nishimura, Y.; Yoshizaki, S.; Sugiyama, R.; Shimojima, M.; Saijo, M.; Shimizu, H.; Suzuki, R.; Wakita, T.; et al. Integrin A3 Is Involved in Non-Enveloped Hepatitis E Virus Infection. *Virology* 2019, **536**, 119–124. [CrossRef] [PubMed]

68. Barth, H.; Schafer, C.; Adah, M.I.; Zhang, F.; Linhardt, R.J.; Toyoda, H.; Kinoshita-Toyoda, A.; Toida, T.; Van Kuppevelt, T.H.; Depla, E.; et al. Cellular Binding of Hepatitis C Virus Envelope Glycoprotein E2 Requires Cell Surface Heparan Sulfate. *J. Biol. Chem.* 2003, **278**, 41003–41012. [CrossRef] [PubMed]

69. Choy, M.Y.; Richman, P.I.; Horton, M.A.; MacDonald, T.T. Expression of the VLA Family of Integrins in Human Intestine. *J. Pathol.* 1990, **160**, 35–40. [CrossRef] [PubMed]

70. Takada, Y.; Ye, X.; Simon, S. The Integrins. *Genome Biol.* 2007, **8**, 215. [CrossRef]

71. Oechslin, N.; Moradpour, D.; Gouttenoire, J. Hepatitis E Virus Finds Its Path through the Gut. *Gut* 2020, **69**, 796–798. [CrossRef]

72. de Melker, A.A.; Sterk, L.M.; Delwel, G.O.; Fles, D.L.; Daams, H.; Weening, J.J.; Sonnenberg, A. The A and B Variants of the Alpha 3 Integrin Subunit: Tissue Distribution and Functional Characterization. *Lab. Investig.* 1997, **76**, 547–563.

73. Volpes, R.; van den Oord, J.J.; Desmet, V.J. Distribution of the VLA Family of Integrins in Normal and Pathological Human Liver Tissue. *Gastroenterology* 1991, **101**, 200–206. [CrossRef] [PubMed]

74. Volpes, R.; van den Oord, J.J.; Desmet, V.J. Integrins as Differential Cell Lineage Markers of Primary Liver Tumors. *Am. J. Pathol.* 1993, **142**, 1483–1492.

75. Dubland, J.A.; Francis, G.A. Lysosomal Acid Lipase: At the Crossroads of Normal and Atherogenic Cholesterol Metabolism. *Front. Cell Dev. Biol.* 2015, **3**, 3. [CrossRef] [PubMed]

76. Bubeck, D.; Filman, D.J.; Cheng, N.; Steven, A.C.; Hogle, J.M.; Belnap, D.M. The Structure of the Poliovirus 135S Cell Entry Intermediate at 10-Angstrom Resolution Reveals the Location of an Externalized Polypeptide That Binds to Membranes. *J. Virol.* 2005, **79**, 7745–7755. [CrossRef] [PubMed]

77. Cavaldesi, M.; Caruso, M.; Sthandier, O.; Amati, P.; Garcia, M.I. Conformational Changes of Murine Polyomavirus Capsid Proteins Induced by Sialic Acid Binding. *J. Biol. Chem.* 2004, **279**, 41573–41579. [CrossRef]

78. Sapp, M.; Bienkowska-Haba, M. Viral Entry Mechanisms: Human Papillomavirus and a Long Journey from Extracellular Matrix to the Nucleus. *FEBS J.* 2009, **276**, 7206–7216. [CrossRef]

79. Rivera-Serrano, E.E.; González-López, O.; Das, A.; Lemon, S.M. Cellular Entry and Uncoating of Naked and Quasi-Enveloped Human Hepatoviruses. *Elife* 2019, **8**, e43983. [CrossRef]

80. Tyagi, S.; Korkaya, H.; Zafrullah, M.; Jameel, S.; Lal, S.K. The Phosphorylated Form of the ORF3 Protein of Hepatitis E Virus Interacts with Its Non-Glycosylated Form of the Major Capsid Protein, ORF2. *J. Biol. Chem.* 2002, **277**, 22759–22767. [CrossRef]

81. Menzo, S.; Clementi, M.; Alfani, E.; Bagnarelli, P.; Iacovacci, S.; Manzin, A.; Dandri, M.; Natoli, G.; Levrero, M.; Carloni, G. Trans-Activation of Epidermal Growth Factor Receptor Gene by the Hepatitis B Virus X-Gene Product. *Virology* 1993, **196**, 878–882. [CrossRef]

82. Lupberger, J.; Zeisel, M.B.; Xiao, F.; Thumann, C.; Fofana, I.; Zona, L.; Davis, C.; Mee, C.J.; Turek, M.; Gorke, S.; et al. EGFR and EphA2 Are Host Factors for Hepatitis C Virus Entry and Possible Targets for Antiviral Therapy. *Nat. Med.* 2011, **17**, 589–595. [CrossRef]

83. Suppiah, S.; Zhou, Y.; Frey, T.K. Lack of processing of the expressed ORF1 gene product of hepatitis E virus. *Virol. J.* 2011, **8**, 245. [CrossRef] [PubMed]

84. Cancela, F.; Noceti, O.; Arbiza, J.; Mirazo, S. Structural aspects of hepatitis E virus. *Arch. Virol.* 2022, **167**, 2457–2481. [CrossRef] [PubMed]

85. Sehgal, D.; Thomas, S.; Chakraborty, M.; Jameel, S. Expression and processing of the Hepatitis E virus ORF1 nonstructural polyprotein. *Virol. J.* 2006, **3**, 38. [CrossRef]

86. Graff, J.; Zhou, Y.-H.; Torian, U.; Nguyen, H.; St Claire, M.; Yu, C.; Purcell, R.H.; Emerson, S.U. Mutations within Potential Glycosylation Sites in the Capsid Protein of Hepatitis E Virus Prevent the Formation of Infectious Virus Particles. *J. Virol.* 2008, **82**, 1185–1194. [CrossRef]

87. Montpellier, C.; Wychowski, C.; Sayed, I.M.; Meunier, J.-C.; Saliou, J.-M.; Ankavay, M.; Bull, A.; Pillez, A.; Abravanel, F.; Helle, F.; et al. Hepatitis E Virus Lifecycle and Identification of 3 Forms of the ORF2 Capsid Protein. *Gastroenterology* 2018, **154**, 211–223.e8. [CrossRef] [PubMed]

88. Ankavay, M.; Montpellier, C.; Sayed, I.M.; Saliou, J.-M.; Wychowski, C.; Saas, L.; Duvet, S.; Aliouat-Denis, C.-M.; Farhat, R.; de Masson d’Autume, V.; et al. New Insights into the ORF2 Capsid Protein, a Key Player of the Hepatitis E Virus Lifecycle. *Sci. Rep.* 2019, **9**, 6243. [CrossRef]

89. Lenggenhager, D.; Gouttenoire, J.; Malehmir, M.; Bawohl, M.; Honcharova-Biletska, H.; Kreutzer, S.; Semela, D.; Neuweiler, J.; Hürlimann, S.; Aepli, P.; et al. Visualization of Hepatitis E Virus RNA and Proteins in the Human Liver. *J. Hepatol.* 2017, **67**, 471–479. [CrossRef]

90. Jameel, S.; Zafrullah, M.; Ozdener, M.H.; Panda, S.K. Expression in Animal Cells and Characterization of the Hepatitis E Virus Structural Proteins. *J. Virol.* 1996, **70**, 207–216. [CrossRef]

91. Zafrullah, M.; Ozdener, M.H.; Kumar, R.; Panda, S.K.; Jameel, S. Mutational Analysis of Glycosylation, Membrane Translocation, and Cell Surface Expression of the Hepatitis E Virus ORF2 Protein. *J. Virol.* 1999, **73**, 4074–4082. [CrossRef] [PubMed]

92. Emerson, S.U.; Nguyen, H.; Torian, U.; Purcell, R.H. ORF3 Protein of Hepatitis E Virus Is Not Required for Replication, Virion Assembly, or Infection of Hepatoma Cells In Vitro. *J. Virol.* 2006, **80**, 10457–10464. [CrossRef] [PubMed]

93. Yamada, K.; Takahashi, M.; Hoshino, Y.; Takahashi, H.; Ichiyama, K.; Nagashima, S.; Tanaka, T.; Okamoto, H. ORF3 Protein of Hepatitis E Virus Is Essential for Virion Release from Infected Cells. *J. Gen. Virol.* 2009, **90 Pt 8**, 1880–1891. [CrossRef] [PubMed]

94. Capelli, N.; Marion, O.; Dubois, M.; Allart, S.; Bertrand-Michel, J.; Lhomme, S.; Abravanel, F.; Izopet, J.; Chapuy-Regaud, S. Vectorial Release of Hepatitis E Virus in Polarized Human Hepatocytes. *J. Virol.* 2019, **93**, e01207-18. [CrossRef] [PubMed]

95. Dao Thi, V.L.; Wu, X.; Belote, R.L.; Andrea, U.; Takacs, C.N.; Fernandez, J.P.; Vale-Silva, L.A.; Prallet, S.; Decker, C.C.; Fu, R.M.; et al. Stem Cell-Derived Polarized Hepatocytes. *Nat. Commun.* 2020, **11**, 1677. [CrossRef] [PubMed]

96. Feng, Z.; Lemon, S.M. Peek-a-Boo: Membrane Hijacking and the Pathogenesis of Viral Hepatitis. *Trends Microbiol.* 2014, **22**, 59–64. [CrossRef] [PubMed]

97. Ren, X.; Hurley, J.H. Proline-Rich Regions and Motifs in Trafficking: From ESCRT Interaction to Viral Exploitation. *Traffic* 2011, **12**, 1282–1290. [CrossRef] [PubMed]

98. Vietri, M.; Radulovic, M.; Stenmark, H. The Many Functions of ESCRTs. *Nat. Rev. Mol. Cell Biol.* 2020, **21**, 25–42. [CrossRef] [PubMed]

99. Nagashima, S.; Takahashi, M.; Jirintai, S.; Tanaka, T.; Nishizawa, T.; Yasuda, J.; Okamoto, H. Tumour Susceptibility Gene 101 and the Vacuolar Protein Sorting Pathway Are Required for the Release of Hepatitis E Virions. *J. Gen. Virol.* 2011, **92 Pt 12**, 2838–2848. [CrossRef]

100. Nagashima, S.; Takahashi, M.; Jirintai, N.; Tanaka, T.; Yamada, K.; Nishizawa, T.; Okamoto, H. A PSAP Motif in the ORF3 Protein of Hepatitis E Virus Is Necessary for Virion Release from Infected Cells. *J. Gen. Virol.* 2011, **92 Pt 2**, 269–278. [CrossRef]

101. Feng, Z.; Hirai-Yuki, A.; McKnight, K.L.; Lemon, S.M. Naked Viruses That Aren’t Always Naked: Quasi-Enveloped Agents of Acute Hepatitis. *Annu. Rev. Virol.* 2014, **1**, 539–560. [CrossRef] [PubMed]

102. Zafrullah, M.; Ozdener, M.H.; Panda, S.K.; Jameel, S. The ORF3 Protein of Hepatitis E Virus Is a Phosphoprotein That Associates with the Cytoskeleton. *J. Virol.* 1997, **71**, 9045–9053. [CrossRef] [PubMed]

103. Garrus, J.E.; von Schwedler, U.K.; Pornillos, O.W.; Morham, S.G.; Zavitz, K.H.; Wang, H.E.; Wettstein, D.A.; Stray, K.M.; Côté, M.; Rich, R.L.; et al. Tsg101 and the Vacuolar Protein Sorting Pathway Are Essential for HIV-1 Budding. *Cell* 2001, **107**, 55–65. [CrossRef] [PubMed]

104. Martin-Serrano, J.; Zang, T.; Bieniasz, P.D. HIV-1 and Ebola Virus Encode Small Peptide Motifs That Recruit Tsg101 to Sites of Particle Assembly to Facilitate Egress. *Nat. Med.* 2001, **7**, 1313–1319. [CrossRef] [PubMed]

105. Nagashima, S.; Takahashi, M.; Jirintai, S.; Tanggis; Kobayashi, T.; Nishizawa, T.; Okamoto, H. The Membrane on the Surface of Hepatitis E Virus Particles Is Derived from the Intracellular Membrane and Contains Trans-Golgi Network Protein 2. *Arch. Virol.* 2014, **159**, 979–991. [CrossRef] [PubMed]

106. Qi, Y.; Zhang, F.; Zhang, L.; Harrison, T.J.; Huang, W.; Zhao, C.; Kong, W.; Jiang, C.; Wang, Y. Hepatitis E Virus Produced from Cell Culture Has a Lipid Envelope. *PLoS ONE* 2015, **10**, e0132503. [CrossRef] [PubMed]

107. Gouttenoire, J.; Pollán, A.; Abram, L.; Oechslin, N.; Mauron, J.; Matter, M.; Oppliger, J.; Szkolnicka, D.; Dao Thi, V.L.; van der Goot, F.G.; et al. Palmitoylation Mediates Membrane Association of Hepatitis E Virus ORF3 Protein and Is Required for Infectious Particle Secretion. *PLoS Pathog.* 2018, **14**, e1007471. [CrossRef] [PubMed]

108. Blaskovic, S.; Blanc, M.; van der Goot, F.G. What Does S-Palmitoylation Do to Membrane Proteins? *FEBS J.* 2013, **280**, 2766–2774. [CrossRef] [PubMed]

109. Chapuy-Regaud, S.; Dubois, M.; Plisson-Chastang, C.; Bonnefois, T.; Lhomme, S.; Bertrand-Michel, J.; You, B.; Simoneau, S.; Gleizes, P.-E.; Flan, B.; et al. Characterization of the Lipid Envelope of Exosome Encapsulated HEV Particles Protected from the Immune Response. *Biochimie* 2017, **141**, 70–79. [CrossRef]

110. Chahar, H.S.; Bao, X.; Casola, A. Exosomes and Their Role in the Life Cycle and Pathogenesis of RNA Viruses. *Viruses* **2015**, *7*, 3204–3225. [CrossRef]

111. Kamar, N.; Dalton, H.R.; Abravanel, F.; Izopet, J. Hepatitis E Virus Infection. *Clin. Microbiol. Rev.* **2014**, *27*, 116–138. [CrossRef] [PubMed]

112. Dao Thi, V.L.; Debing, Y.; Wu, X.; Rice, C.M.; Neyts, J.; Moradpour, D.; Gouttenoire, J. Sofosbuvir Inhibits Hepatitis E Virus Replication In Vitro and Results in an Additive Effect When Combined with Ribavirin. *Gastroenterology* **2016**, *150*, 82–85.e4. [CrossRef] [PubMed]

113. Cornberg, M.; Koch, A.; Lohse, A.; Hardtke, S.; Manns, M.P.; Wedemeyer, H. Sofosbuvir Monotherapy Fails to Achieve HEV RNA Elimination in Patients with Chronic Hepatitis E—The HepNet SoftE Pilot Study. *J. Hepatol.* **2020**, *73*, 696–699. [CrossRef] [PubMed]

114. Kamar, N.; Rostaing, L.; Abravanel, F.; Garrouste, C.; Lhomme, S.; Esposito, L.; Basse, G.; Cointault, O.; Ribes, D.; Nogier, M.B.; et al. Ribavirin Therapy Inhibits Viral Replication on Patients with Chronic Hepatitis E Virus Infection. *Gastroenterology* **2010**, *139*, 1612–1618. [CrossRef] [PubMed]

115. Mallet, V.; Nicand, E.; Sultanik, P.; Chakvetadze, C.; Tessé, S.; Thervet, E.; Mouthon, L.; Sogni, P.; Pol, S. Brief Communication: Case Reports of Ribavirin Treatment for Chronic Hepatitis E. *Ann. Intern. Med.* **2010**, *153*, 85–89. [CrossRef]

116. Alric, L.; Bonnet, D.; Beynes-Rauzy, O.; Izopet, J.; Kamar, N. Definitive Clearance of a Chronic Hepatitis E Virus Infection with Ribavirin Treatment. *Am. J. Gastroenterol.* **2011**, *106*, 1562–1563. [CrossRef]

117. Chaillon, A.; Sirinelli, A.; De Muret, A.; Nicand, E.; d’Alteroche, L.; Goudeau, A. Sustained Virologic Response with Ribavirin in Chronic Hepatitis E Virus Infection in Heart Transplantation. *J. Heart Lung Transplant.* **2011**, *30*, 841–843. [CrossRef]

118. de Niet, A.; Zaaijer, H.L.; ten Berge, I.; Weegink, C.J.; Reesink, H.W.; Beuers, U. Chronic Hepatitis E after Solid Organ Transplantation. *Neth. J. Med.* **2012**, *70*, 261–266. [PubMed]

119. Del Bello, A.; Arné-Bes, M.C.; Lavayssière, L.; Kamar, N. Hepatitis E Virus-Induced Severe Myositis. *J. Hepatol.* **2012**, *57*, 1152–1153. [CrossRef]

120. Pischke, S.; Stiefel, P.; Franz, B.; Bremer, B.; Suneetha, P.V.; Heim, A.; Ganzenmueller, T.; Schlue, J.; Horn-Wichmann, R.; Raupach, R.; et al. Chronic Hepatitis E in Heart Transplant Recipients. *Am. J. Transplant.* **2012**, *12*, 3128–3133. [CrossRef]

121. Hajji, H.; Gérolami, R.; Solas, C.; Moreau, J.; Colson, P. Chronic Hepatitis E Resolution in a Human Immunodeficiency Virus (HIV)-Infected Patient Treated with Ribavirin. *Int. J. Antimicrob. Agents* **2013**, *41*, 595–597. [CrossRef]

122. Junge, N.; Pischke, S.; Baumann, U.; Goldschmidt, I.; Manns, M.; Wedemeyer, H.; Pfister, E.-D. Results of Single-Center Screening for Chronic Hepatitis E in Children after Liver Transplantation and Report on Successful Treatment with Ribavirin. *Pediatr. Transplant.* **2013**, *17*, 343–347. [CrossRef]

123. Koning, L.; Pas, S.D.; de Man, R.A.; Balk, A.H.M.M.; de Knegt, R.J.; ten Kate, F.J.; Osterhaus, A.D.M.E.; van der Eijk, A.A. Clinical Implications of Chronic Hepatitis E Virus Infection in Heart Transplant Recipients. *J. Heart Lung Transplant.* **2013**, *32*, 78–85. [CrossRef]

124. Neukam, K.; Barreiro, P.; Macías, J.; Avellón, A.; Cifuentes, C.; Martín-Carbonero, L.; Echevarría, J.M.; Vargas, J.; Soriano, V.; Pineda, J.A. Chronic Hepatitis E in HIV Patients: Rapid Progression to Cirrhosis and Response to Oral Ribavirin. *Clin. Infect. Dis.* **2013**, *57*, 465–468. [CrossRef]

125. Pischke, S.; Hardtke, S.; Bode, U.; Birkner, S.; Chatzikyrkou, C.; Kauffmann, W.; Bara, C.L.; Gottlieb, J.; Wenzel, J.; Manns, M.P.; et al. Ribavirin Treatment of Acute and Chronic Hepatitis E: A Single-Centre Experience. *Liver Int.* **2013**, *33*, 722–726. [CrossRef] [PubMed]

126. Riezebos-Brilman, A.; Puchhammer-Stöckl, E.; van der Weide, H.Y.; Haagsma, E.B.; Jaksch, P.; Bejvl, I.; Niesters, H.G.; Verschuuren, E.A.M. Chronic Hepatitis E Infection in Lung Transplant Recipients. *J. Heart Lung Transplant.* **2013**, *32*, 341–346. [CrossRef] [PubMed]

127. Schlevogt, B.; Kinast, V.; Reusch, J.; Kerkhoff, A.; Praditya, D.; Todt, D.; Schmidt, H.H.; Steinmann, E.; Behrendt, P. Chronic Hepatitis E Virus Infection during Lymphoplasmacytic Lymphoma and Ibrutinib Treatment. *Pathogens* **2019**, *8*, 129. [CrossRef]

128. Hooda, P.; Chaudhary, M.; Parvez, M.K.; Sinha, N.; Sehgal, D. Inhibition of Hepatitis E Virus Replication by Novel Inhibitor Targeting Methyltransferase. *Viruses* **2022**, *14*, 1778. [CrossRef] [PubMed]

129. Bhise, N.; Agarwal, M.; Thakur, N.; Akshay, P.S.; Cherian, S.; Lole, K. Repurposing of Artesunate, an Antimalarial Drug, as a Potential Inhibitor of Hepatitis E Virus. *Arch. Virol.* **2023**, *168*, 147. [CrossRef] [PubMed]

130. Feld, J.J.; Hoofnagle, J.H. Mechanism of Action of Interferon and Ribavirin in Treatment of Hepatitis C. *Nature* **2005**, *436*, 967–972. [CrossRef]

131. Nasir, M.; Wu, G.Y. HEV and HBV Dual Infection: A Review. *J. Clin. Transl. Hepatol.* **2020**, *8*, 313–321. [CrossRef] [PubMed]

132. Shrestha, M.P.; Scott, R.M.; Joshi, D.M.; Mammen, M.P., Jr.; Thapa, G.B.; Thapa, N.; Myint, K.S.; Fourneau, M.; Kuschner, R.A.; Shrestha, S.K.; et al. Safety and efficacy of a recombinant hepatitis E vaccine. *New Engl. J. Med.* **2007**, *356*, 895–903. [CrossRef] [PubMed]

133. Li, S.; Zhang, J.; Xia, N. Lessons from hepatitis E vaccine design. *Curr. Opin. Virol.* **2015**, *11*, 130–136. [CrossRef] [PubMed]

134. Gabrielli, F.; Alberti, F.; Russo, C.; Cursaro, C.; Seferi, H.; Margotti, M.; Andreone, P. Treatment Options for Hepatitis A and E: A Non-Systematic Review. *Viruses* **2023**, *15*, 1080. [CrossRef]

135. Zhang, J.; Zhang, X.F.; Huang, S.J.; Wu, T.; Hu, Y.M.; Wang, Z.Z.; Wang, H.; Jiang, H.M.; Wang, Y.J.; Yan, Q.; et al. Long-term efficacy of a hepatitis E vaccine. *New Engl. J. Med.* **2015**, *372*, 914–922. [CrossRef] [PubMed]

136. Su, Y.Y.; Huang, S.J.; Guo, M.; Zhao, J.; Yu, H.; He, W.G.; Jiang, H.M.; Wang, Y.J.; Zhang, X.F.; Cai, J.P.; et al. Persistence of antibodies acquired by natural hepatitis E virus infection and effects of vaccination. Clin. Microbiol. Infect. Off. Publ. Eur. Soc. Clin. Microbiol. Infect. Dis. 2017, 23, 336.e1–336.e4. [CrossRef] [PubMed]

137. Zhu, F.C.; Zhang, J.; Zhang, X.F.; Zhou, C.; Wang, Z.Z.; Huang, S.J.; Wang, H.; Yang, C.L.; Jiang, H.M.; Cai, J.P.; et al. Efficacy and safety of a recombinant hepatitis E vaccine in healthy adults: A large-scale, randomised, double-blind placebo-controlled, phase 3 trial. Lancet 2010, 376, 895–902. [CrossRef] [PubMed]

138. Li, Y.; Huang, X.; Zhang, Z.; Li, S.; Zhang, J.; Xia, N.; Zhao, Q. Prophylactic Hepatitis E Vaccines: Antigenic Analysis and Serological Evaluation. Viruses 2020, 12, 109. [CrossRef] [PubMed]

139. Wu, X.; Dao Thi, V.L.; Liu, P.; Takacs, C.N.; Xiang, K.; Andrus, L.; Gouttenoire, J.; Moradpour, D.; Rice, C.M. Pan-Genotype Hepatitis E Virus Replication in Stem Cell-Derived Hepatocellular Systems. Gastroenterology 2018, 154, 663–674.e7. [CrossRef] [PubMed]

140. Meertens, L.; Hafirassou, M.L.; Couderc, T.; Bonnet-Madin, L.; Kril, V.; Kümmerer, B.M.; Labeau, A.; Brugier, A.; Simon-Loriere, E.; Burlaud-Gaillard, J.; et al. FHL1 Is a Major Host Factor for Chikungunya Virus Infection. Nature 2019, 574, 259–263. [CrossRef]

141. Zhang, R.; Kim, A.S.; Fox, J.M.; Nair, S.; Basore, K.; Klimstra, W.B.; Rimkunas, R.; Fong, R.H.; Lin, H.; Poddar, S.; et al. Mxra8 Is a Receptor for Multiple Arthritogenic Alphaviruses. Nature 2018, 557, 570–574. [CrossRef] [PubMed]

142. Han, J.; Perez, J.T.; Chen, C.; Li, Y.; Benitez, A.; Kandasamy, M.; Lee, Y.; Andrade, J.; tenOever, B.; Manicassamy, B. Genome-Wide CRISPR/Cas9 Screen Identifies Host Factors Essential for Influenza Virus Replication. Cell Rep. 2018, 23, 596–607. [CrossRef] [PubMed]

143. Richardson, R.B.; Ohlson, M.B.; Eitson, J.L.; Kumar, A.; McDougal, M.B.; Boys, I.N.; Mar, K.B.; De La Cruz-Rivera, P.C.; Douglas, C.; Konopka, G.; et al. A CRISPR Screen Identifies IFI6 as an ER-Resident Interferon Effector That Blocks Flavivirus Replication. Nat. Microbiol. 2018, 3, 1214–1223. [CrossRef] [PubMed]

144. Flint, M.; Chatterjee, P.; Lin, D.L.; McMullan, L.K.; Shrivastava-Ranjan, P.; Bergeron, É.; Lo, M.K.; Welch, S.R.; Nichol, S.T.; Tai, A.W.; et al. A Genome-Wide CRISPR Screen Identifies N-Acetylglucosamine-1-Phosphate Transferase as a Potential Antiviral Target for Ebola Virus. Nat. Commun. 2019, 10, 285. [CrossRef] [PubMed]

145. V’kovski, P.; Gerber, M.; Kelly, J.; Pfaender, S.; Ebert, N.; Braga Lagache, S.; Simillion, C.; Portmann, J.; Stalder, H.; Gaschen, V.; et al. Determination of Host Proteins Composing the Microenvironment of Coronavirus Replicase Complexes by Proximity-Labeling. Elife 2019, 8, e42037. [CrossRef]

Disclaimer/Publisher’s Note: The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.
