
The Reduction of Insulin Gene  
Transcription in HIT-T15 β Cells  
Chronically Exposed to High  
Glucose Concentration Is  
Associated with the Loss of  
RIPE3b1 and STF-1 Transcription  
Factor Expression  

Arun Sharma, L. Karl Olson, R. Paul Robertson, and  
Roland Stein  

Department of Molecular Physiology and Biophysics (A.S., R.S.)  
Vanderbilt University  
Nashville, Tennessee 37232.  

Division of Diabetes, Endocrinology, and Metabolism (L.K.O., P.R.)  
Department of Medicine  
University of Minnesota  
Minneapolis, Minnesota, 55455  

The most important regulator of insulin gene expression in pancreatic β-cells is glucose, which affects gene transcription, mRNA translation, and secretion. Insulin gene transcription is both positively and negatively regulated by glucose. Recently, we have shown that the inhibition of insulin gene transcription caused by passaging HIT T-15 β-cells, in the presence of high glucose, was due, in part, to reduced expression of a key regulator of insulin enhancer-mediated expression, somatostatin transcription factor-1 (STF-1). In this study, we have examined whether the activity of the other essential transcription regulators of insulin gene expression, the RIPE3b1 and insulin control element (ICE) activators, were also influenced in these HIT T-15 cells. The results show that the binding and trans-activation functions of the RIPE3b1 activator are reduced in parallel with the loss in STF-1 and insulin gene expression. In contrast, the regulatory properties of the ICE activator are unaffected. Our studies indicate that insulin gene transcription is inhibited by glucose through a mechanism involving reduced expression of both the RIPE3b1 and STF-1 activators in HIT T-15 cells but is independent of the ICE activator. (Molecular Endocrinology 9: 1127–1134, 1995)

INTRODUCTION

The insulin gene is transcribed exclusively in pancreatic islet β-cells as a result of the interaction of cellular factors with its enhancer, which is located between nucleotides –340 and –91 relative to transcription start site (reviewed in Ref. 1). Detailed mutagenesis studies have demonstrated that the Flat-E (–201 to –196), RIPE3b1 (–115 to –107), and ICE (–100 to –91) elements are essential for efficient expression from this region in β-cells (2–7). The trans-acting factors that bind to and activate transcription from each of these elements appear to be principally expressed in islet cells (5, 6, 8, 9). Thus, the expression of the activators of Flat-E and insulin control element (ICE)-mediated transcription, the somatostatin transcription factor-1 (STF-1) (5, 10–12) and insulin activator factor (INSAF) (13) proteins, respectively, are detected primarily in β-cells but also appear to be present in at least a subset of the islet α-, δ-, and PP cells. The ubiquitous HEB and E2A (i.e. E12, E47, and E2/5) basic helix-loop-helix proteins also comprise a part of the ICE activator (14–17). The factor(s) stimulating RIPE3b1 element-mediated transcription have not been isolated, yet activator binding is only detected in β-extracts (6), suggesting that its expression may also be limited to pancreatic cell types.

The STF-1, INSAF, and RIPE3b1 activators also appear to be important transcriptional determinants of other pancreas-specific genes. This is most clearly exemplified for STF-1, as this homeoprotein can bind to and activate transcription from cis-acting regulatory

0888-8809/95/$3.00/0  
Molecular Endocrinology  
Copyright © 1995 by The Endocrine Society

sequences in both the insulin (β) and somatostatin (δ)  
genes (5, 10–12). In addition, recent studies have demonstrated that pancreas formation is prevented in mice carrying a null mutation in STF-1 (18). Since the expression of STF-1 precedes that of insulin and somatostatin during development (12, 19), this factor must also be essential in the expression of a key developmental determinant(s) of the pancreas. These observations indicate that understanding the detailed mechanisms by which STF-1, INSAF, and the RIPE3b1 activators regulate insulin gene transcription may provide insight into how these factors also control the expression of other genes involved in differentiation and cell-specific gene expression.

Glucose is the most important physiological effector of insulin gene transcription (20–22), and the STF-1, INSAF, and RIPE3b1 activators are the essential regulators of this response (23–26). Insulin gene transcription is both positively and negatively regulated by glucose (23, 24, 26, 27). The RIPE3b1 and ICE activators play a key role in glucose-stimulated expression (23, 26). Transcriptional activation by RIPE3b1 results from increased binding activity, whereas the transactivation potential of the ICE activator and not its cis-element-binding capabilities is effected in glucose-induced β-cells (26). The importance of the STF-1 activator in this response is unclear (23–25). However, STF-1 plays an important role in glucose-induced repression of insulin gene transcription (27, 28). Thus, we demonstrated that STF-1 synthesis was dramatically reduced in the β-cell line, HIT[11.1L], which had lost its ability to transcribe the insulin gene as a result of continuous growth in the presence of high glucose (11.1 mM) (28). Reduced STF-1 levels appear to result from a defect in processing of the STF-1 precursor RNA to mRNA (28).

In the studies described herein we present evidence indicating that insulin gene transcription is suppressed in HIT-T15 cells passaged chronically in high glucose due to reduced expression of both the RIPE3b1 and STF-1 activators. Thus, the functional properties of the RIPE3b1 and ICE activators were compared with STF-1 in early passage HIT-T15 cells (HIT[11.1E]) and late passage HIT T-15 cells grown in the presence of 11.1 (HIT[11.1L]) or 0.8 mM glucose (HIT[0.8L]). Previously, we had shown that insulin and STF-1 were expressed in both HIT[11.1E] and HIT[0.8L] cells, whereas STF-1 synthesis was inhibited in HIT[11.1L] cells concomitant with the decrease in insulin gene transcription (27, 28). RIPE3b1 and STF-1 binding activity was regulated similarly in these cell lines. In contrast, we found that the ICE activator was not only present but functionally active in HIT[11.1L] cells. These results indicate that the activity of the insulin gene transcription factors required for glucose-stimulated expression, RIPE3b1 and the ICE activator, are differentially affected in HIT[11.1L] cells. Furthermore, they suggest that the loss in insulin gene transcription in HIT[11.1L] cells is a consequence of reduced expression of both the RIPE3b1 and STF-1 activators.


RESULTS

RIPE3b1 Binding Activity Is Inhibited in HIT[11.1L] Cells

Previously we demonstrated that the binding activity of the STF-1 transcription factor was reduced significantly in HIT-T15 extracts prepared from cells chronically cultured in the presence of high glucose, HIT[11.1L], when analyzed against either early passage cells grown in high glucose, HIT[11.1E], or late passage cells cultured in low glucose, HIT[0.8L] (27, 28). The loss in STF-1 expression was associated with the decrease in insulin gene transcription in HIT[11.1L] cells and resulted from a decrease in the synthesis of STF-1. To determine whether ICE and RIPE3b1 activator formation was also affected in HIT[11.1L], the gel mobility shift assay was used to measure the levels of these activator complexes in extracts prepared from HIT[11.1E], HIT[11.1L], and HIT[0.8L] cells. The levels of the STF-1 and adenovirus-5 major late transcription factor (MLTF) were also measured in this analysis. Binding reactions were conducted with an equal amount of extract protein and ³²P-labeled oligodeoxynucleotide probe.

MLTF is a ubiquitously distributed factor and serves as an internal control in these experiments, as its expression does not appear to be altered by glucose treatment (26). As shown in Fig. 1C, similar levels of MLTF element binding were detected in HIT[11.1E], HIT[11.1L], and HIT[0.8L] extracts. The level of RIPE3b1, STF-1, and ICE activator binding were also compared in each of these extracts. There are several protein-DNA complexes detected with each of these insulin element probes in β-cell extracts, and the specificity of these complexes were analyzed in a competition experiment in the presence of a specific [i.e. wild type (WT)] or nonspecific (i.e. mutant) oligonucleotide in the binding reaction. The mutant contains either a one (i.e. ICE), two (i.e. RIPE3b1), or three-nucleotide substitution (STF-1) which eliminates activator function in vivo and in vitro (5, 6, 26, 29). The wild type oligonucleotide could effectively compete for formation of the RIPE3b1 (Fig. 1A), STF-1 (Fig. 1B), and ICE (Fig. 1C) activator complex; however, the activator-binding site mutant could not (Fig. 1). RIPE3b1 and STF-1 activator binding were reduced to a similar low level in HIT[11.1L] extracts (Fig. 1, A and B). However, ICE activator binding did not decrease in HIT[11.1E] extracts when compared with either HIT[11.1L] or HIT[0.8L] (Fig. 1C). These results indicate that RIPE3b1 and STF-1 activator binding activity is specifically and selectively reduced under conditions that inhibit insulin gene transcription in vivo.

ICE Activation Is Unaffected in HIT[11.1L] Cells

Since the transactivation potential of the ICE activator appears to be regulated in glucose-stimulated

Inhibition of Insulin Gene Transcription by Glucose

(A)

| Probe | RIPE3b1 |
| --- | --- |
| HIT Extract |  |
| Competitor |  |

(B)

| Probe | FLAT-E |
| --- | --- |
| HIT Extract |  |
| Competitor |  |

(C)

| Probe | ICE | MLTF |
| --- | --- | --- |
| HIT Extract |  |  |
| Competitor |  |  |

(D)

| Relative Binding Activity |
| --- |
| HIT Cell Line | RIPE3b1 | FLAT-E | MLTF | ICE |
| 11.1E | 1.0 | 1.0 | 1.0 | 1.0 |
| 11.1L | 0.08 | 0.06 | 1.0 | 1.3 |
| 0.8L | 0.85 | 1.2 | 1.3 | 0.9 |

Fig. 1. RIPE3b1 Activator-Binding Activity Is Reduced in HIT[11.1L] Cells

Equal concentrations of HIT[11.1E], HIT[11.1L], or HIT[0.8L] protein extract were analyzed for RIPE3b1 (A), STF-1 (B), and ICE and MLTF activator binding (C). Arrows indicate the position of the specific activator complex. The molar ratio of cold wild type (W) and mutant (M) competitor to the probe was 50. D, Quantification of the amounts of the activator bands. Relative amounts of the Flat-E, RIPE3b1, ICE, and MLTF activator-binding complexes were determined by densitometric scanning of the autoradiogram and are expressed relative to the activity in HIT[11.1E] cells. The results were reproduced on several different occasions; data shown are from one representative experiment. The identity of the Flat-E- and ICE-binding complexes was also confirmed using antisera that specifically recognized STF-1 and the E2A-encoded proteins of the ICE activator (data not shown).

HIT-T15 cells (26), and not its DNA-binding activity, we analyzed whether ICE activator function *in vivo* was altered between HIT[11.1E] and HIT[11.1L] cells. ICE-mediated activation was analyzed in cells transfected with the ICE-driven construct, ICE

OVEC-1 LUC, which contains three copies of the ICE activator-binding site inserted directly upstream of the rabbit β-globin TATA box in OVEC-1 (Fig. 2A) (30). Previous studies with ICE OVEC-1 have demonstrated that expression is β-cell type specific (7).

MOL ENDO · 1995
1130

(A)

ICE-OVEC-1
![Diagram](https://i.imgur.com/1234567.png)
TATA
LUC

(B)

Relative Luciferase Activity
![Bar Chart](https://i.imgur.com/8901234.png)
HIT-T 15 Cell Lines

Fig. 2. ICE-Mediated Activation in HIT[11.1L] Cells

A, Schematic representation of the ICE OVEC-1 LUC reporter gene used. Three copies of an oligonucleotide containing rat insulin II gene ICE sequences from –102 to –87 bp were cloned just upstream of the rabbit β-globin promoter in OVEC-1 LUC (30). The orientations of the ICE inserts are indicated by the arrows. B, HIT[11.1E] or HIT[11.1L] cells were transfected with either ICE OVEC-1 LUC or OVEC-1 LUC. Results are expressed as the ratio of the ICE OVEC-1 LUC activity divided by the LUC activity of OVEC-1 LUC ± SEM. The average activity of OVEC-1 LUC in HIT[11.1E] and HIT[11.1L] cells was 799 and 517 RLU, respectively.

The activity of ICE OVEC-1 LUC and OVEC-1 LUC is depicted in Fig. 2B. The ICE-driven construct was significantly more active than OVEC-1 in both HIT[11.1E] and HIT[11.1L] cells. ICE OVEC-1 LUC activity was stimulated 14.7 ± 2- and 10.4 ± 2.8-fold in HIT[11.1E] and HIT[11.1L] cells, respectively. Together with the observation that ICE activator-binding activity does not change appreciably between HIT[11.1E] and HIT[11.1L] cells, these results indicate that the loss in insulin gene transcription in HIT[11.1L] cells is not a consequence of reduced ICE activator function.

RIPE3b1-Binding Activity Is Stimulated in Response to Glucose in HIT[11.1L] Cells

The RIPE3b1 activator mediates glucose-stimulated transcription of the insulin gene (26). RIPE3b1-binding activity is induced during this response. We considered that the mechanism by which glucose inhibits RIPE3b1 binding in HIT[11.1L] cells might be related to that involved in induction. If so, we reasoned that RIPE3b1-binding activity in HIT[11.1L] cells would not be stimulated by glucose. Detailed characterization of

insulin secretion from HIT-T15 cells has shown that they respond to glucose concentrations below 1 mm glucose (31).

RIPE3b1 binding was measured in HIT[11.1E], HIT[11.1L], and HIT[0.8L] nuclear extracts prepared from cells grown for 24 h in 0.2, 0.8, or 11.1 mm glucose (Fig. 3). As was also illustrated in Fig. 1A, the overall RIPE3b1-binding activity in HIT[11.1E] and HIT[0.8L] extracts is much greater than in HIT[11.1L] extracts. However, RIPE3b1 binding in each of these cell lines was stimulated in response to glucose. Furthermore, binding activity was increased by approximately the same fold (6-fold) between 0.8 and 11.1 mm glucose in HIT[11.1E], HIT[11.1L], and HIT[0.8L] cells (Fig. 3). These results indicate that the signals required for glucose-induced RIPE3b1 binding are different from those involved in inhibiting RIPE3b1 binding.

The Reduction in RIPE3b1-Binding Activity Contributes to the Loss of Insulin Expression in HIT[11.1L] Cells

We next asked whether the reduction in RIPE3b1-binding activity was related to the loss in insulin gene transcription in HIT[11.1L] cells. To address this question, the importance of the RIPE3b1, ICE, and Flat-E elements to insulin gene transcription was compared in HIT[11.1L], HIT[11.1E], and HIT[0.8L] cells. Appro-

HIT-T 15
11.1 E
11.1 L
0.8 L
Glucose (mM)
0.2 0.8 11.1
0.2 0.8 11.1
0.2 0.8 11.1
RIPE3b1 →
![Gel Electrophoresis Image](https://i.imgur.com/5678901.png)

Fig. 3. RIPE3b1 Binding Is Stimulated by Glucose in HIT[11.1L] Cells

Equal concentrations of HIT-T15 protein extract prepared from HIT[11.1E], HIT[11.1L], and HIT[0.8L] cells grown in the medium containing either 0.2, 0.8, or 11.1 mm glucose were analyzed for RIPE3b1-binding activity. The arrow indicates the position of the RIPE3b1 activator complex. The relative levels (level at 11.1 mm/level at 0.8 mM) of the RIPE3b1-specific complexes was determined by densitometric scanning. HIT[11.1E], 6.3; HIT[11.1L], 6.2; HIT[0.8L], 6.7.

Inhibition of Insulin Gene Transcription by Glucose

appropriate mutations in these elements within the insulin expression plasmid, -238 WT LUC, which contains rat insulin II gene sequences spanning the region from -238 to +2 base pairs (bp) linked to the luciferase reporter gene, were made, and their effects were analyzed in vivo. Luciferase activity from the insulin constructs was normalized in each cell line to the activity obtained from a Rous sarcoma virus-LUC expression construct transfected in parallel.

The relative activities of -238 WT LUC, -238 Em LUC, -238 b1m LUC, and -238 ICEm LUC in the various HIT-T15 cell lines are shown in Fig. 4. The -238 WT LUC activity was similar between the insulin-expressing cell lines, HIT[11.1E] and HIT[0.8L], although dramatically reduced in HIT[11.1L] cells. -238 WT LUC expression in these cell lines recapitulates the relative transcription level for the insulin gene (27, 32). Mutating the Flat-E, RIPE3b1, or ICE elements in -238 WT LUC inhibited expression from 70–98% in HIT[11.1E] and HIT[0.8L] cells. The reduction in -238 LUC expression by the Flat-E, RIPE3b1, and ICE mutations is consistent with a number of other previous findings (3, 4, 6, 7), which suggest that the activators of these elements interact synergistically to stimulate insulin enhancer-mediated expression.

The effect of the Flat-E and RIPE3b1 mutations on -238 WT LUC activity in HIT[11.1L] cells was much less dramatic than that observed in HIT[11.1E] and HIT[0.8L] cells. Thus, these mutations only reduced -238 WT LUC activity by approximately 50% (Fig. 4B). In contrast, the activity of the ICE mutant -238 expression construct in HIT[11.1L] cells was decreased to the same low level found in the insulin-expressing HIT-T15 cell lines (i.e., ~96%). The insensitivity of the Flat-E and RIPE3b1 mutations on -238 WT LUC expression is corroborated by our observation that HIT[11.1L] cells have reduced levels of the STF-1 and RIPE3b1 activators. We conclude from these results that the lack of insulin gene transcription in HIT[11.1L] cells can be directly attributed to their inability to effectively express the STF-1 and RIPE3b1 activators.

### DISCUSSION

Control of insulin gene transcription is principally mediated by the STF-1, RIPE3b1, and ICE activators. Each of these effectors plays an essential role in glucose-regulated expression in islet β-cells. Glucose can both positively and negatively affect insulin gene transcription. The RIPE3b1 and ICE activators mediate glucose-induced expression, a relatively rapid response (i.e., hours) to this effector. Previously we had proposed that the reduction in insulin gene transcription in HIT-T15 cells passaged chronically in the presence of high glucose (i.e., HIT[11.1L] cells) resulted from a block in STF-1 synthesis (28). In this study it is shown that the lack of RIPE3b1 transcription factor expression also contributes to the reduction in insulin

---

**Fig. 4. RIPE3b1- and Flat-E-Mediated Activation Is Reduced in HIT[11.1L] Cells**

A, The structure of the LUC expression plasmids -238 WT LUC, -238 Flat-E mutant (-238 Em LUC), -238 b1 mutant (-238 b1m LUC), and -238 ICE mutant (-238 ICEm LUC) are shown. The hatched box represents rat insulin II enhancer sequences (-238 to -91 bp), and the arrow indicates the site of the mutation. B, HIT[11.1E], HIT[11.1L], and HIT[0.8L] cells were transfected with either an insulin LUC expression construct or RSV LUC. For each HIT-T15 cell line, activity is expressed relative to RSV LUC ± SEM. The relative activity of the RSV LUC:-238 WT LUC:-238 Em LUC:-238 b1m LUC:-238 ICEm LUC in HIT[11.1E] cells was 100:96.1:17.8:11.8:1.4; HIT[11.1L], 100:6.6:3.3:2.7:0.25; HIT[0.8L], 100:137.4:28.3:35.4:10.4. The average activity of RSV LUC in HIT[11.1E], HIT[11.1L], and HIT[0.8L] cells was 3.9 × 10⁶, 1.3 × 10⁶, and 5.0 × 10⁶ RLU, respectively.

gene expression in HIT[11.1L] cells. This conclusion is based on our observation that the binding and activation potential of RIPE3b1 activator is decreased in HIT[11.1L] cells when compared with insulin-expressing HIT-T15 cell lines (i.e., HIT[11.1E] and HIT[0.8L]). The reduction in RIPE3b1 activator function paralleled the decline in STF-1 expression. In contrast, there was little or no difference in ICE activator or MLTF function in HIT[11.1L], HIT[11.1E], and HIT[0.8L] cells. These results strongly indicate that the reduction in insulin gene transcription mediated by glucose results from

specific and selective effects on the expression of the RIPE3b1 and STF-1 activators.

Insulin gene transcription is stimulated by glucose concentrations in early passage HIT-T15 cells. This response is mediated through the RIPE3b1 and ICE elements (23, 26). Thus, mutations in either of these elements severely reduces glucose-inducible expression from transfected insulin enhancer reporter constructs (23, 26). In contrast, mutating the STF-1-binding sites within these insulin constructs does not affect glucose-responsive expression in islets (23) or these HIT-T15 cells (26). In contrast, studies conducted by Melloul et al. (24) and Petersen et al. (25) with pancreatic islets indicates that STF-1 is involved in glucose-responsive expression. However, since German and Wang (23) were unable to reproduce this finding, it still remains to be determined whether STF-1 functions in islets. Importantly, our studies clearly indicate that glucose-stimulated insulin transcription is not mediated by STF-1 in HIT-T15 cells (26). Together with the studies in chronically passaged HIT-T15 cells, these results indicate that different transcriptional regulatory mechanisms are used by glucose to potentiate or repress expression. Interestingly, the targets of glucose regulation in both of these responses are activators that have an essential role in β-cell-specific expression of the insulin gene.

The RIPE3b1 activator appears to play a broad role in the regulation of insulin gene transcription by glucose. Thus, RIPE3b1 activity is potentiated under conditions that induce insulin transcription (26) and, as observed here, reduced when insulin expression is repressed by glucose. RIPE3b1-binding activity is directly affected in response to glucose. Our demonstration that RIPE3b1 binding is still sensitive to glucose stimulation in HIT[11.1L] cells suggests that the mechanisms implemented by glucose to positively and negatively regulate the activity of this key insulin gene transcription factor are distinct.

An understanding of the mechanisms involved in regulating RIPE3b1 function will require the isolation and characterization of the RIPE3b1 transcription factor. The block in STF-1 expression in HIT[11.1L] cells appears to result from an inability to process the 6.4-kilobase STF-1 pre-RNA to 2.3 and 1.9-kilobase STF-1 mRNAs (28). It is possible that the loss in RIPE3b1-binding activity also results from a splicing defect. However, other distinct and established control mechanisms can also be proposed, including a decrease in RIPE3b1 gene transcription and/or a post-translation modification of the RIPE3b1 protein that reduces its binding activity.

In conclusion, we have found that insulin gene transcription is inhibited after chronic exposure of HIT-T15 β-cells to high glucose concentrations as a result of effects on the function of two key transcriptional regulatory factors, STF-1 and RIPE3b1. Interestingly, we have proposed that chronic hyperglycemia in humans with type II diabetes may decrease glucose-induced insulin expression in β-cells (33). Whether the STF-1

and RIPE3b1 transcription factors are involved in the glucotoxic effects on insulin gene expression in pancreatic β-cells in vivo is under investigation.

MATERIALS AND METHODS

HIT T-15 Cell Cultures

HIT T-15 cells were normally cultured in RPMI-1640 medium (GIBCO/BRL, Gaithersburg, MD) containing 11.1 mm glucose supplemented with 10% (vol/vol) fetal bovine serum. Beginning at passage 70, the cells were split into medium containing either 11.1 mm or 0.8 mm glucose and continuously cultured through passage 140 as described previously (32). The HIT T-15 cells are defined as early (E, passage 70–81) or late (L, passage 126–140) and whether grown in the presence of 0.8 or 11.1 mm glucose.

DNA Constructs

All of the insulin-luciferase constructs have rat insulin II sequences from –238 (5′) to +2 (3′). The synthesis of the wild type [–238 WT LUC (9)], RIPE3b1 mutant [–238 b1m LUC (26)], and Flat-E mutant [–238 Em LUC (5)] were described previously. The ICE OVEC-1 LUC expression construct contains three copies of rat insulin II sequences from –102 to –87 bp cloned in the SalI site just upstream of the rabbit β-globin promoter in OVEC-1 LUC (30).

DNA Transfection and in Vivo Transcription Assays

HIT T-15 cells were subcultured for 2 days in RPMI-1640 containing 11.1 mm glucose at a density of 1.2 × 10⁶ cells per well (35 mm). Duplicate wells were transfected with either 2 μg of an insulin reporter construct or RSV LUC (34) using the liposome-mediated transfection technique (27). The cells were then grown in medium containing 0.8 or 11.1 mm glucose for 2 days. Luciferase (LUC) enzymatic assays were performed as described by De Wet et al. (34). The LUC activity is defined in arbitrary relative light units (RLU). The background LUC activity from mock-transfected cells was approximately 150 RLU.

Electrophoretic Mobility Shift Assays

Nuclear extracts were prepared from HIT-T15 cells as described previously (26). Double stranded oligonucleotide probes to the rat insulin II ICE (–104 TCTGGCCATCTGCT-GATCCT –85), RIPE3b1 (–126 TGGAAACTGCAGCT-TCAGCCCCCTCT –101), Flat-E (–213 CCTCTTAAGACTCTA-ATTACCCT –192), and Adenovirus type 2 major late transcription factor (MLTF) (–71 TAGGTGTAGGCCACGT-GACCGGGTGTTC –54) binding sites were end-labeled using [α-³²P]dATP (6,000 Ci/mmol) and the Klenow fragment of Escherichia coli DNA polymerase I. The binding reactions (20 μg extract/lane) with the ICE, RIPE3b1, Flat-E, and MLTF probes were performed as described earlier (5, 26). The competition experiments were performed with double stranded oligonucleotides corresponding to a RIPE3b1 (–126 TG-GAAACTGCAGCTCGAGCCCCTCT –101), FLAT-E –213 CCTCTTAAGACTCGACTGACCCT –192), ICE (–104 TCTG-GCCCTGCTGATCCT –85) mutant binding site. [The underlined nucleotides within the Flat-E (5, 26), RIPE3b1 (6, 26), and ICE (29) elements prevent element-driven expression in vivo and binding in vitro in the gel-shift assay.] After incubation of the binding mixtures at 4°C for 20 min, they were

loaded onto a 6% nondenaturing polyacrylamide gel (acryl-
amide-bisacrylamide ratio 29:1) and were run in TGE buffer
(50 mm Tris, 380 mm glycine, 2 mm EDTA, pH 8.5). The gel
was dried and subjected to autoradiography after
electrophoresis.

Acknowledgements

We are grateful to Drs. Mina Peshvaria and Gary Robinson for
their constructive criticism of the manuscript.

Received January 27, 1995. Revision received May 5,
1995. Accepted May 9, 1995.
Address requests for reprints to: Roland Stein, Department of
Molecular Physiology and Biophysics, Vanderbilt University
Medical School, Room 708, Light Hall, Nashville, Tennessee
37232.
This work was supported by Public Health Service Grants
DK-08742 (to L.K.O.), DK-50203 (to R.S.), DK R01–38325
(to R.P.R.) from the NIH. Partial support was also derived
from the Vanderbilt University Diabetes Research and Train-
ing Center Molecular Biology Core Laboratory (Public Health
Service Grant P60 DK-20593 from the NIH).

REFERENCES

1. Stein R 1993 Factors regulating insulin gene transcription.
Trends Endocrinol Metab 4:96–100
2. Crowe DT, Tsai M-J 1989 Mutagenesis of the rat insulin
II 5′-flanking region defines sequences important for
expression in HIT cells. Mol Cell Biol 9:1784–1789
3. German MS, Moss LG, Wang J, Rutter WJ 1992 The
insulin and islet amyloid polypeptide genes contain sim-
ilar cell-specific promoter elements that bind identical
β-cell nuclear complexes. Mol Cell Biol 12:1777–1788
4. Karlsson O, Edlund T, Moss JB, Rutter WJ, Walker MD
1987 A mutational analysis of the insulin gene transcrip-
tion control region: expression in beta cells is dependent
on two related sequences within the enhancer. Proc Natl
Acad Sci USA 84:8819–8823
5. Peshavaria M, Gamer L, Henderson E, Teitelman G,
Wright CVE, Stein R 1994 XLHbox 8, an endoderm-spe-
cific Xenopus homeodomain protein, is closely related to
a mammalian insulin gene transcription factor. Mol
Endocrinol 8:806–816
6. Shieh S-Y, Tsai M-J 1991 Cell-specific and ubiquitous
factors are responsible for the enhancer activity of the rat
insulin II gene. J Biol Chem 266:16708–16714
7. Whelan J, Poon D, Weil PA, Stein R 1989 Pancreatic
β-cell-type-specific expression of the rat insulin II gene is
controlled by positive and negative transcriptional
elements. Mol Cell Biol 9:3253–3259
8. Ohlsson H, Thor S, Edlund T 1991 Novel insulin promoter-
and enhancer-binding proteins that discriminate
between pancreatic α and β cells. Mol Endocrinol 5:
897–904
9. Robinson GLW G, Peshavaria M, Henderson E, Shieh
S-Y, Tsai M-J, Teitelman G, Stein R 1994 Analysis of
transcription regulatory signals of the insulin gene: ex-
pression of the trans-active factor that stimulates insulin
control element mediated expression precedes insulin
gene transcription. J Biol Chem 269:2452–2460
10. Leonard J, Peers B, Johnson T, Ferrere K, Lee S, Mont-
miny M 1993 Characterization of Somatostatin Transac-
tivating Factor-1, a novel homeobox factor that stimu-
lates somatostatin expression in pancreatic islet cells.
Mol Endocrinol 7:1275–1283
11. Miller CP, McGeehee Jr RE, Habener JF 1994 IDX-1: a
new homeodomain transcription factor expressed in rat
pancreatic islets and duodenum that transactivates the
somatostatin gene. EMBO J 13:1145–1156
12. Ohlsson H, Karlsson K, Edlund T 1993 IPF-1, a home-
domain-containing transactivator of the insulin gene.
EMBO J 12:4251–4259
13. Robinson GLW G, Cordle SR, Henderson E, Weil PA,
Teitelman G, Stein R 1994 Isolation and characterization
of a novel transcription factor that binds to and activates
insulin control element-mediated expression. Mol Cell
Biol 14:6704–6714
14. Cordle SR, Henderson E, Masuoka H, Weil PA, Stein R
1991 Pancreatic β-cell-type-specific transcription of
the insulin gene is mediated by basic helix-loop-helix
DNA-binding proteins. Mol Cell Biol 11:1734–1738
15. German MS, Blanar MA, Nelson C, Moss LG, Rutter WJ
1991 Two related helix-loop-helix proteins participate in
separate cell-specific complexes that bind the insulin
enhancer. Mol Endocrinol 5:292–299
16. Park CW, Walker MD 1992 Subunit structure of cell-
specific E box-binding proteins analyzed by quantita-
tion of electrophoretic mobility shift. J Biol Chem 267:
15642–15649
17. Peyton M, Moss LG, Tsai M-J 1994 Two distinct class A
helix-loop-helix transcription factors, E2A and BETA1,
form separate DNA binding complexes on the insulin
gene E box. J Biol Chem 269:25936–25941
18. Jonsson J, Carlsson L, Edlund T, Edlund H 1994 Insulin-
promoter-factor 1 is required for pancreas development
in mice. Nature 371:606–609
19. Guz Y, Montminy MR, Stein r, Leonard J, Gamer LW,
Wright CVE, Teitelman G 1995 Expression of murine
STF-1, a putative insulin gene transcription factor, in
β-cells of pancreas, duodenal epithelium and pancreatic
exocrine and endocrine progenitors during ontogeny.
Development 121:11–18
20. Efrat S, Surana M, Fleischer N 1991 Glucose induces
insulin gene transcription in a murine pancreatic β-cell
line. J Biol Chem 266:11141–11143
21. German MS, Moss LG, Rutter WJ 1990 Regulation of
insulin gene expression by glucose and calcium in
transfected primary islet cultures. J Biol Chem 265:
22063–22066
22. Goodison S, Kenna S, Ashcroft SJH 1992 Control of insulin
gene expression by glucose. Biochem J 285: 563–568
23. German MS, Wang J 1994 The insulin gene contains mul-
tiple transcriptional elements that respond to glucose. Mol
Cell Biol 14:4067–4075
24. Melloul D, Ben-Neriah Y, Cerasi E 1993 Glucose mod-
ulates the binding of an islet-specific factor to a con-
served sequence within the rat I and the human insulin
promoters. Proc Natl Acad Sci USA 90:3865–3869
25. Petersen HV, Serup P, Leonard J, Michelsen BK, Madsen
OD 1994 Transcriptional regulation of the human insulin
gene is dependent of the homeodomain proteins STF1/
IPF1 acting through the CT boxes. Proc Natl Acad Sci
USA 91:10465–10469
26. Sharma A, Stein R 1994 Glucose-induced transcription of
the insulin gene is mediated by factors required for β-cell-
type-specific expression. Mol Cell Biol 14: 871–879
27. Olson LK, Redmon JB, Towle HC, Robertson RP 1993
Chronic exposure of HIT cells to high glucose concentra-
tion paradoxically decreases insulin gene transcription.
J Clin Invest 92:514–519
28. Olson LK, Sharma A, Peshavaria M, Towle HC, Robert-
son RP, Stein R 1995 Reduction of insulin gene trans-
scription in HIT T- 15 β cells chronically exposed to
superphysiological glucose concentration is associated
with the loss of STF-1 transcription factor expression.
Proc Natl Acad Sci USA, in press
29. Whelan J, Cordle SR, Henderson E, Weil PA, Stein R
1990 Identification of a pancreatic β-cell insulin gene
transcription factor that binds to and appears to activate
cell-type-specific expression: its possible relationship to


other cellular factors that binds to a common insulin gene sequence. Mol Cell Biol 10:1564–1572

30. Henderson E, Stein R 1994 c-jun inhibits transcriptional activation by the insulin enhancer and the insulin control element is the target of control. Mol Cell Biol 14:655–662

31. Zhang H-J, Walseth TF, Robertson RP 1989 Insulin secretion and cAMP metabolism in HIT cells. Reciprocal and serial passage-dependent relationships. Diabetes 38:44–48

32. Robertson RP, Zhang H-J, Pyzdrowski KL, Walseth TF

1992 Preservation of insulin mRNA levels and insulin secretion in HIT cell by avoidance of chronic exposure to high glucose concentrations. J Clin Invest 90:320–325

33. Robertson RP, Olson LK, Zhang H-J 1994 Differentiating glucose toxicity from glucose desensitization: A new message from the insulin gene. Diabetes 43: 1085–1089

34. De Wet JR, Wood KV, DeLuca M, Helinski DR, Subramani S 1987 Firefly luciferase gene: structure and expression in mammalian cells. Mol Cell Biol 7:725–737
