
This article was downloaded by: [NorthWest University]
On: 19 December 2014, At: 07:18
Publisher: Taylor&Francis
Informa Ltd Registered in England and Wales Registered Number: 1072954 Registered office: Mortimer
House, 37-41 Mortimer Street, London W1T 3JH, UK

Bioscience, Biotechnology, and Biochemistry
Publication details, including instructions for authors and subscription information:
http://www.tandfonline.com/loi/tbbb20

Cloning and Nucleotide Sequence of the Gene
Responsible for Chlorination of Tetracycline

Tohru Dairi${}^{a}$, Tetsuo Nakano${}^{a}$, Kazuo Aisaka${}^{a}$, Ryoichl Katsumata${}^{a}$ & Mamoru Hasegawa${}^{a}$
${}^{a}$ Tokyo Research Laboratories of Kyowa Hakko Kogyo Co., Ltd., 3-6-6, Asahimachl,
Machida-shi, Tokyo 194, Japan
Published online: 12Jun 2014.

To cite this article: Tohru Dairi, Tetsuo Nakano, Kazuo Aisaka, Ryoichl Katsumata & Mamoru Hasegawa (1995) Cloning
and Nucleotide Sequence of the Gene Responsible for Chlorination of Tetracycline, Bioscience, Biotechnology, and
Biochemistry, 59:6, 1099-1106, DOI: 10.1271/bbb.59.1099
To link to this article: http://dx.doi.org/10.1271/bbb.59.1099

PLEASE SCROLL DOWN FOR ARTICLE

Taylor & Francis makes every effort to ensure the accuracy of all the information (the “Content”) contained
in the publications on our platform. However, Taylor & Francis, our agents, and our licensors make no
representations or warranties whatsoever as to the accuracy, completeness, or suitability for any purpose of
the Content. Any opinions and views expressed in this publication are the opinions and views of the authors,
and are not the views of or endorsed by Taylor & Francis. The accuracy of the Content should not be relied
upon and should be independently verified with primary sources of information. Taylor and Francis shall
not be liable for any losses, actions, claims, proceedings, demands, costs, expenses, damages, and other
liabilities whatsoever or howsoever caused arising directly or indirectly in connection with, in relation to or
arising out of the use of the Content.

This article may be used for research, teaching, and private study purposes. Any substantial or systematic
reproduction, redistribution, reselling, loan, sub-licensing, systematic supply, or distribution in any
form to anyone is expressly forbidden. Terms & Conditions of access and use can be found at http://
www.tandfonline.com/page/terms-and-conditions

Biosci. Biotech. Biochem., 59 (6), 1099–1106, 1995

# Cloning and Nucleotide Sequence of the Gene Responsible for Chlorination of Tetracycline

Tohru DAIRI,$^{\dagger}$ Tetsuo NAKANO, Kazuo AISAKA, Ryoichi KATSUMATA, and Mamoru HASEGAWA$^{++}$

**Tokyo Research Laboratories of Kyowa Hakko Kogyo Co., Ltd., 3-6-6, Asahimachi, Machida-shi, Tokyo 194, Japan**

Received December 8, 1994

Two cosmid clones containing distinct types of self-defense gene of a 6-demethylchlortetracycline producer, *Streptomyces aureofaciens* NRRL3203, were isolated. The gene responsible for chlorination of tetracycline (*chl* gene) was subcloned from one of the cosmid clones by complementation of a chlorination-deficient mutant, using a gene cloning system for strain NRRL3203 developed in this study. The nucleotide sequence analysis of a 4.4-kb *SacI-BamHI* fragment containing the *chl* gene showed that the predicted product of the *chl* gene is a polypeptide of 452 amino acids, and that the *chl* gene was preceded by two open reading frames, which could encode polypeptides of 50 kDa and 32 kDa, respectively. A search for sequences homologous to these ORFs found that the former product strongly resembles that of the 6-hydroxylation enzyme for oxytetracycline biosynthesis, and that the latter product has a weak but significant similarity to the hydroxyindole *O*-methyltransferase of bovine pineal gland. By Northern blot analysis, these three genes were suggested to be polycistrionically transcribed.

Many fungi and bacteria are known to produce a wide variety of halogenated compounds. ${ }^{1)}$ The incorporation of halogen into these compounds is believed to be catalyzed by haloperoxidases, which are divided into three groups with respect to their specificities for halide ions, chloroperoxidases ($\mathrm{Cl}^{-}, \mathrm{Br}^{-}$, and $\mathrm{I}^{-}$), bromoperoxidases ($\mathrm{Br}^{-}$ and $\mathrm{I}^{-}$), and iodoperoxidases ($\mathrm{I}^{-}$). ${ }^{1)}$ In many cases, the heme prosthetic group in these haloperoxidases is important in their catalytic activity. ${ }^{1)}$ However, recently, nonheme haloperoxidases have been detected in several organisms including bacteria. ${ }^{2-5)}$ Such haloperoxidases are expected to be valuable in use for bioconversion of industrial chemical compounds so that their detailed characterization and molecular cloning are of particular interest.

Some *Streptomyces* strains are reported to produce chlorinated antibiotics. Chloramphenicol and chlortetracycline, produced by *S*. venezuelae and *S*. aureofaciens, respectively, are representatives of these antibiotics. ${ }^{6,7)}$ Though the biosynthetic pathways of these antibiotics are well characterized, ${ }^{8,9)}$ only limited information is available about the enzymes participating in chlorination. Efforts to identify and purify the chloroperoxidases from these antibiotic producers have been made, and only bromoperoxidases could be isolated from these strains. ${ }^{3,5,10)}$ These results suggest that purifications of chloroperoxidases seemed to be difficult for some unknown reasons. Therefore, to analyze the primary structures of these enzymes, we decided to clone the gene(s) responsible for chlorination of tetracycline from *Streptomyces aureofaciens*. We tentatively named this gene the *chl* gene.

## Materials and Methods

### Bacterial strains and plasmids
*Streptomyces aureofaciens* NRRL3203, a 6-demethylchlortetracycline (Fig. 1) producer, was used as the DNA donor. Cosmid pHC79, ${ }^{11)}$ purchased from Boehringer Mannheim

---

$^{\dagger}$ Present address: Toyama Prefectural University, 5180 Kurokawa, Kosugi, Toyama 939-03, Japan.

$^{++}$ To whom correspondence should be addressed.

agar (Difco), 20 g; pH 7.6} and 2.0 ml of 5 M CaCl₂. Transformation was done with plasmid DNA in 10 μl of TE buffer to which 100 μl suspension of 10⁹–10¹⁰ protoplasts were added. After leaving the mixture for 1 min at room temperature, 440 μl of 20% (vol/vol) polyethylene glycol 4000 with T medium¹³⁾ was added and mixed gently by pipetting. The mixture was diluted with P medium, and 100-μl samples were plated on RA medium. These plates were incubated at 30°C for 18 h, then overlaid with 2.5 ml of nutrient soft agar¹³⁾ containing thiopeptin (final concentration, 20 μg/ml).

**DNA isolation and manipulation.** Plasmid and genomic DNA isolations were done by the method of Hopwood *et al.*¹³⁾ Plasmids from *E. coli* were prepared by using the Qiagen Plasmid Kit (Qiagen). All restriction enzymes, T4 ligase, and calf intestinal alkaline phosphatase were obtained from Takara (Kyoto, Japan). The procedure for cosmid library preparation has been described previously.¹⁸⁾

**RNA isolation and Northern blot analysis.** Total RNA was isolated from *Streptomyces aureofaciens* NRRL3203 grown at 28°C for 1, 4, or 7 days in the production medium. Cell samples from the 30-ml cultures were poured into a Falcon 2070 tube containing 10 g of ice that had been cooled at –80°C. After centrifugation at 4°C, 6000 × *g* for 10 min, the mycelia were resuspended in 10 ml of ice-cooled phenol-TNPS buffer {10% phenol (vol/vol), 10% sodium triisopropyl naphthalene sulfate (weight/vol), 3% sodium *p*-amino salicylic acid (weight/vol), 50 mM Tris-HCl (pH 8.0)}, and sonicated for 3 min with intervals of 30 s. The lysates were then extracted twice with phenol-chloroform (1:1, vol/vol). Nucleic acids were precipitated with ethanol and resuspended in 1 ml of NETS buffer {0.2% SDS (weight/vol), 10 mM Tris-HCl (pH 7.4), 10 mM EDTA, 25 mM NaCl}. RNAs were precipitated again by addition of 350 μl of 10 M lithium chloride. After centrifuging and washing once with 70% ethanol, the pellets were dissolved in 1 ml of TE buffer. Nucleic acids were measured by the absorbance at 260 nm. The procedures of Northern blot hybridization were done by the method of Tomas.¹⁹⁾

**Gene replacement.** The technique that was developed by Anzai *et al.*²⁰⁾ was used. The ORF1 mutated strain (NP7) was constructed as follows. Plasmid pCLSS1, which was constructed by inserting the *SacI* fragment (2.2 kb, Fig. 2, fragment A) from pGLA2 into the *SacI* site of pUC118, was digested with *XhoI*, treated with Klenow fragment of DNA polymerase I to fill-in the sticky ends of this site, and self-ligated to construct pCLFS1, in which ORF1 was inactivated by a frame-shift mutation. The *SacI* fragment of pCLFS1 was inserted into the *SacI* site of pIJ702, and the resultant plasmid pCLFS2 was introduced into *S. aureofaciens* NRRL3203. Protoplasts of thiopeptin-resistant colonies were prepared and regenerated on RA medium without antibiotic selection. In our experiments, about 10% of the regenerated colonies lost the plasmids, and the ORF1 inactivated strains emerged from the plasmid-cured colonies at the frequency of 5%. The frame-shift mutations in the chromosome of the mutant was confirmed by the Southern blot analysis of the genomic DNA digested with *XhoI* using the 2.2-kb *SacI* fragment (Fig. 2, fragment A) as a probe.

The procedures for construction of ORF2 or ORF3 mutated strains were essentially the same as that described above. The former (NP6) and latter (NP4) strains were constructed by using the *SacI* fragment (1.8 kb, Fig. 2, fragment B), the *NocI* site in which was destroyed by frame-shift mutation and the *SphI* fragment (2.2 kb, Fig. 2, fragment C), the *PvuII* site which was also destroyed by the insertion of *EcoRI* linker, respectively.

**Sequence analysis.** The 2.6-kb *SacI-BamHI* fragment inserted into pCLSB1 and the 1.8-kb *SacI* fragment neighboring the 2.6-kb *SacI-BamHI* fragment were prepared from pGLA2 and cloned into the *SacI-BamHI* and *SacI* sites of pUC119, respectively. After construction of a series of plasmids subcloned from these plasmids, single-stranded DNAs were

prepared with helper phage M13KO7. Sequencing was done by the dideoxy chain termination method with Sequenase version 2 (U.S. Biochemical Corp.). The nucleotide sequence data reported in this paper will appear in the GSDB, DDBJ, EMBL, and NCBI nucleotide sequence databases with the accession number D38214.

**Purification of 6-demethylhydrochlortetracycline.** Frozen stock mycelia of the ORF2 mutated strain (NP6) in 20% glycerol were inoculated into 20 ml of the seed medium, and were grown at 30°C by shaking for 4 days. Fifteen milliliters of the seed cultures were transferred into 300 ml of the production medium in a 2-liter Erlenmeyer flask and grown at 30°C for 7 days on a rotary shaker. The fermentation broth was adjusted to pH 1.0 with 12 N HCl. The supernatant obtained by centrifugation was put on a column (300 ml) packed with Cosmosil 75C18-OPN (Nacalai Tesque, Kyoto, Japan). After this was washed with 500 ml of deionized water, antibiotics were eluted with 10–30% aqueous acetonitrile. The fractions containing the yellow pigments were collected, and evaporated at 40°C *in vacuo*. The products were then put on reverse phase preparative HPLC {column: YMC D-ODS-5, 20 × 250 mm, 5 μm, Yamamura Chemical Lab. Ltd., mobile phase: CH₃CN–0.1 M citrate buffer (25 : 75, pH 2.0), flow rate: 10 ml/min, detection: UV absorption at 280 nm). The yellow eluate was concentrated *in vacuo* to an aqueous solution, which was desalted by Diaion HP-20 chromatography with 50% methanol elution to afford a pure solid. This product was analyzed by FAB-MS and ¹H-NMR analysis.

---

### Results

#### Cloning of cosmid clones containing the self-defense genes of *S. aureofaciens* NRRL3203

Total DNA of *S. aureofaciens* NRRL3203 was digested with *BamHI* or *BclI* and ligated to pIJ702, which had been digested previously with *BglII* and treated with calf intestinal alkaline phosphatase. The ligation mixtures were used to transform protoplasts of *Streptomyces lividans* TK23, a chlortetracycline-sensitive strain, by the conventional procedure.¹³⁾ Thiopeptin-resistant colonies were selected on regeneration medium R5,¹³⁾ and replica-plated to modified ATCC5 plate¹⁸⁾ containing chlortetracycline (100 μg/ml). Each of the ligation mixtures yielded several chlortetracycline-resistant transformants which carried the recombinant plasmids. Two representative plasmids, pTCRBA1 and pTCRBC1, were those obtained from *BamHI* digests and *BclI* digests, respectively. The DNA inserts of these two plasmids did not have any overlapping regions as judged by restriction enzyme analysis (Fig. 2), indicating that *S. aureofaciens* NRRL3203 had at least two kinds of self-defense genes.

To isolate the flanking regions of these self-defense genes, a cosmid library in *E. coli* HB101 of strain NRRL3203 DNA partially digested with Sau3AI was screened by colony hybridization with pTCRBA1 and pTCRBC1 as probes. Positive clones carrying a cosmid hybridizable with either of the probes but not with both were obtained. Two cosmids, pGLA2 (38 kb insert) and pGLA11 (37 kb insert), which were hybridized with pTCRBA1 and pTCRBC1, respectively, were chosen for further analysis (Fig. 2). These two inserts were shown by restriction enzyme analysis to contain a common DNA stretch. Genomic Southern analysis confirmed that these two inserts were not structurally rearranged during the cloning procedure. Thus, the two self-defense genes were located approximately 40 kb apart in the genome of the strain NRRL3203.

---

### Construction of gene-cloning system for *S. aureofaciens* NRRL3203

To identify the *chl* gene by complementation, it is necessary to construct an efficient self-cloning system for

![Structure of 6-Demethylchlortetracycline](https://i.imgur.com/structure.png)

**Fig. 1.** Structure of 6-Demethylchlortetracycline.

The Gene Responsible for Chlorination of Tetracycline

S. aureofaciens NRRL3203. In our preliminary experiments, the protocols for Streptomyces strains ${ }^{13)}$ were not applicable to this strain because no protoplast formation was observed even when mycelia were cultivated in the presence of 1% glycine, which is known to increase the yield of protoplasts. ${ }^{13)}$ Therefore, at first, we have optimized the conditions for the preparation and regeneration of protoplasts of S. aureofaciens NRRL3203. Previously we had found that the protoplast formation of Micromonospora strains was dramatically accelerated when the strain was grown in the presence of D-mannitol. ${ }^{17)}$ Therefore, application of these protocols to S. aureofaciens NRRL3203 was attempted. When S. aureofaciens NRRL3203 was cultivated in the presence of D-mannitol (5%), this strain also became highly sensitive to lysozyme. Besides D-mannitol, the addition of glycine (0.5%) made hyphae more sensitive. Under the conditions, about $1 \times 10^{10}$ protoplasts were obtained from 10 ml of culture.

Many regeneration media for protoplasts of Streptomyces strains have been reported. ${ }^{21)}$ In the preliminary experiments, the protoplasts of S. aureofaciens NRRL3203 were found to regenerate on A101 medium, which had been established for Streptomyces sannanensis. ${ }^{22)}$ However, the regeneration ratio of the strain NRRL3203 on A101 medium was very low (below 0.1%). The most critical factors for the regeneration were found to be Fe²⁺ and glucose concentrations. Using the optimized medium described in Materials and Methods, about 30% of the protoplasts were able to regenerate on this improved medium.

The standard transformation protocol for Streptomyces strains ${ }^{13)}$ was applied to S. aureofaciens NRRL3203. From approximately $1 \times 10^{10}$ protoplasts of S. aureofaciens NRRL3203 transformed with pIJ702 which had been prepared from S. lividans, about $1 \times 10^{4}$ of thiopeptin resistant colonies were obtained. Moreover, an increase of the transformation efficiency ($>1 \times 10^{6}$ transformants/µg DNA) was obtained by using the plasmid prepared from S. aureofaciens NRRL3203, and by tuning of the type and the concentration of polyethylene glycol as described in Materials and Methods. The plasmids isolated from thiopeptin resistant colonies had the same structures as that of the original pIJ702 as judged by the restriction enzyme analysis.

Cloning of the gene responsible for chlorination of tetracycline

By treating the spores of S. aureofaciens NRRL3203 with NTG, about 140 mutants, which did not form E. coli growth-inhibitory zone in the stick bioassay ${ }^{18)}$ or produced unusual pigments, were isolated from approximately 20,000 colonies. Characterization of these mutants showed that a mutant NP385 lacked the chlorinating activity. The mutant accumulated non-chlorinated antibiotic in the culture broth. When a series of plasmids, which were constructed by

![Diagram]

Fig. 2. Physical Map of the Genome of S. aureofaciens NRRL3203 Covered with the Two Cosmid Clones.

Solid bars under the upper physical map show the DNA fragments carried on cosmids pGLA2 and pGLA11 and those obtained by shotgun cloning and subcloning experiments. The fragment used for nucleotide sequence analysis is enlarged. Open and hatched boxes indicate the fragments used for gene replacement experiments (A, B and C) and for Northern blot analysis (D, E, F and G), respectively. The restriction enzyme sites used for frame-shift mutations are marked with asterisk.

10 20 30 40 50 60 70 80 90 100 1410 1420 1430 1440 1450 1460 1470 1480 1490 1500
GAGCTCGGGGTTTGTACTGGAAATGGCATGGGTCCTCGGGTCTGGGGTCAAGCCCCACGGTGCCCCCCCCGGGCTCCACGGTGGCCAGGGGGGCGAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGTGGGAGGCAAGGGCCAGGGT
2710 2720 2730 2740 2750 2760 2770 2780 2790 2800 4110 4120 4130 4140 4150 4160 4170 4180 4200
GGGGGCTGTCGGGCACCATCGGTCCGGCTCGATCCTCGGCCAAGCACGGCTTCCGGGATCATGGCTGCGGAGGACTGGGAATGGCTGGGCCCCCTGGGCTTGCTGGGGCCCTGGGCTTCAACGGACCCCGGACAAACCCACATCAACCCACACCCACATCAAGGAGGCAAGATGATCCGG

2810 2820 2830 2840 2850 2860 2870 2880 2890 2900 4220 4230 4240 4250 4260 4270 4280 4290 4300
GTCCACCATCCGGGAGACGGCTGGTTGGTGGCTGGGCTTGATCCGGGACCGGTTCCAGGGAGATCCGCAACACTGGGAGACTCCAGGACCTCCAGGACCTCCGGCAA GluGlnArgValArgGluSerAspTrpMetLeuProAlaLeuGlyPheAsnAspProAspThrHisHisIleAsnProThrAlaAspLysMetIleArg

2910 2920 2930 2940 2950 2960 2970 2980 2990 3000 4210 ATCGGGGAGTGGGGCCACCGGTCAACACCGGGAGATCCGGGAGCCAGGGCCGAGGGTCAAGGGGCGAGGGGCTCAAGGCGGATCCGGGAGACTCCGAGACTGGGA 11e Ala Glu Trp Ala Thr Gly His His Arg Pro Glu Ile Arg Glu Leu Ser Ala Glu Glu Val Arg Ala Ala Met Arg Val Lys Pro***

3010 3020 3030 3040 3050 3060 3070 3080 3090 3100 4310 4320 4330 4340 4350 4360 4370 4380 4390 4400
TCGATCGTGGCAACGGGCCCACTTCCGGCACCGGACACCAGACTTCTACATGTTCCAAGGCGGGGTGGGAGAGGGAGGAACGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAGGGAGAG
subcloning various internal segments of the insert of pGLA2 and pGLA11, were introduced into NP385 by the self-cloning system described above, one subclone pCLBB1 with the 10.5 kb BamHI fragment (Fig. 2) from pGLA2 could confer the chlorinating activity on the mutant. A further subcloning experiment found that the essential region was limited to the 2.6-kb SacI-BamHI fragment (pCLSB1, Fig. 2). The gene on the fragment functioning in chlorination was named *chl*.

### Nucleotide sequence of *chl* gene

To figure out the primary structure of the *chl* gene, the nucleotide sequence of the SacI-BamHI fragment inserted into pCLSB1 was analyzed (from positions 1834 to 4422, as numbered in Fig. 3). Two ORFs were predicted by Frame Analysis (data not shown)²⁴ in the same direction. One was a complete open reading frame (ORF1, from positions 2941 to 4296), which could encode a protein of 452 amino acids (deduced molecular weight 59,321), and the other was a truncated ORF (ORF3) extending beyond the analyzed sequence. To confirm that ORF1 encodes the enzyme participating in chlorination, we constructed a derivative strain (NP7) of *S. aureofaciens* NRRL3203 in which ORF1 was replaced with the insertionally inactivated sequence by the gene replacement technique. The ORF1-inactivated strain accumulated non-chlorinating antibiotic in the culture broth, showing that ORF1 codes for a gene involved in chlorination. The *chl* gene has a high G+C content (69%) and a marked bias in codon usage (Table 1). Codons with G or C in the third positions are very frequent (98%), as has been described in other structural genes of Streptomycetes. The stop codon of ORF1 is followed by a GC-rich region containing long palindromic sequences (Fig. 3, underlines), which might act as a transcriptional terminator. When compared with all sequences in the GenBank protein sequence database, the deduced amino acid sequence of ORF1 had no strong similarity to any proteins.

### Transcriptional analysis of the *chl* gene

To analyze the transcriptional unit and temporal expression of the *chl* gene, Northern blot hybridization was done. Total RNA was isolated from the mycelia on the 1st, 4th, and 7th day, and analyzed by Northern blotting using the probe SalI-SacI fragment (Fig. 2, fragment D) containing a part of the *chl* gene. A single transcript, about 4.4 kb long, appeared at 1 day, had a peak of expression on the 4th day, and decreased by 7th day (Fig. 4A), suggesting that the *chl* gene is polycistrionically transcribed with other genes, and is expressed throughout the cultivation.

### Analysis of the genes transcribed polycistrionically with the *chl* gene

Considering that the *chl* gene (ORF1) was preceded by ORF3, and that it might be polycistrionically transcribed, the *chl* gene was thought to be cotranscribed with ORF3. To explore this possibility, a detailed Northern blot analysis was done. Three DNA fragments, the SphI fragment (Fig. 2, fragment E), the SacI-SalI fragment which precede the *chl* gene (fragment F), and the BamHI fragment, which follows the *chl* gene (fragment G), were used as probes. The BamHI fragment was hybridized to the 1.5-kb mRNA. On the other hand, both the SphI fragment and the SacI-SalI fragment were hybridized to the 4.4-kb mRNA (Fig. 4B), showing that the *chl* gene was polycistrionically transcribed with ORF3. To discover the function of ORF3, the DNA sequence of the 1.8-kb SacI fragment (from positions 1 to 1833, as numbered in Fig. 3) neighboring the 2.6-kb SacI-BamHI fragment was also analyzed. The results predicted the presence of complete ORF3 (from positions 1738 to 2623). Moreover, one additional ORF (ORF2, from positions 298 to 1706) was also found. ORF2 and ORF3 could encode 469 and 293 amino acid proteins, respectively, and have also high G+C contents (74 and 72%, respectively) and a marked bias in codon usage (Table 1). Codons with G or C in the third positions are 95 and 98%, respectively. A search for sequences homologous to these ORFs found that the N-terminus amino acid sequence of the ORF2-specifying protein strongly resembles that of the 6-hydroxylation enzyme for oxytetracycline biosynthesis²⁴ and that the ORF3-specifying protein has a weak but significant similarity to hydroxyindole O-methyltransferase of bovine pineal gland²⁵ (Fig. 5). These results suggest that

**Fig. 4. Northern Blot Analysis of the *chl* Gene and Flanking Region.**

A: Total RNA was isolated from *S. aureofaciens* NRRL3203 grown at 28°C for 1 (lane 1), 4 (lane 2), 7 (lane 3) days in the production medium, and subjected to Northern blot analyses using the SalI-SacI fragment containing a part of the *chl* gene (Fig. 2, fragment D).

B: Total RNA was isolated from *S. aureofaciens* NRRL3203 grown at 28°C for 4 days, and analyzed by Northern blotting using the SphI fragment (fragment E, lane 4), SacI-SalI fragment (fragment F, lane 5) and BamHI fragment (fragment G, lane 6). The arrowheads indicate the positions of 23S and 16S rRNAs and the size of hybridized fragments. A weak background hybridization signal to the 16S and 23S RNAs occurred with some probes.

**Fig. 5. Alignment of the ORF2 and ORF3 Products with the 6-Hydroxylation Enzyme for Oxytetracycline Biosynthesis and the Hydroxyindole O-Methyltransferase.**

A: ORF2 (upper) and 6-hydroxylation enzyme of *S. rimosus* (lower).

B: ORF3 (upper) and hydroxyindole O-methyltransferase of bovine pineal gland.

The Gene Responsible for Chlorination of Tetracycline

Table Codon Usage in ORF1, ORF2, and ORF3

| TTT 0 | TCT 0 | TAT 0 | TGT 1 | TTT 0 | TCT 0 | TAT 0 | TGT 0 | TTT 0 | TCT 0 | TAT 0 | TGT 0 |
|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|
| TTC 22 | TCC 8 | TAC 11 | TGC 6 | TTC 13 | TCC 5 | TAC 9 | TGC 3 | TTC 12 | TCC 4 | TAC 9 | TGC 0 |
| TTA 0 | TCA 0 | TAA 1 | TGA 0 | TTA 0 | TCA 0 | TAA 0 | TGA 1 | TTA 0 | TCA 0 | TAA 0 | TGA 1 |
| TTG 3 | TCG 9 | TAG 0 | TGG 12 | TTG 1 | TCG 3 | TAG 0 | TGG 6 | TTG 1 | TCG 5 | TAG 0 | TGG 3 |
| CTT 2 | CCT 0 | CAT 0 | CGT 1 | CTT 1 | CCT 1 | CAT 0 | CGT 1 | CTT 0 | CCT 0 | CAT 0 | CGT 0 |
| CTC 5 | CCC 5 | CAC 19 | CGC 21 | CTC 12 | CCC 4 | CAC 14 | CGC 28 | CTC 5 | CCC 3 | CAC 12 | CGC 16 |
| CTA 0 | CCA 0 | CAA 0 | CGA 1 | CTA 0 | CCA 2 | CAA 1 | CGA 0 | CTA 0 | CCA 0 | CAA 0 | CGA 0 |
| CTG 25 | CCG 21 | CAG 11 | CGG 17 | CTG 40 | CCG 20 | CAG 15 | CGG 11 | CTG 22 | CCG 9 | CAG 8 | CGG 10 |
| ATT 0 | ACT 0 | AAT 0 | AGT 0 | ATT 0 | ACT 0 | AAT 0 | AGT 1 | ATT 0 | ACT 0 | AAT 0 | AGT 0 |
| ATC 20 | ACC 14 | AAC 17 | AGC 10 | ATC 8 | ACC 27 | AAC 5 | AGC 5 | ATC 11 | ACC 13 | AAC 4 | AGC 3 |
| ATA 0 | ACA 0 | AAA 0 | AGA 0 | ATA 0 | ACA 0 | AAA 0 | AGA 0 | ATA 0 | ACA 0 | AAA 0 | AGA 0 |
| ATG 12 | ACG 6 | AAG 8 | AGG 2 | ATG 13 | ACG 2 | AAG 6 | AGG 1 | ATG 5 | ACG 4 | AAG 6 | AGG 1 |
| GTT 0 | GCT 0 | GAT 0 | GGT 1 | GTT 0 | GCT 0 | GAT 1 | GGT 2 | GTT 0 | GCT 1 | GAT 0 | GGT 3 |
| GTC 12 | GCC 25 | GAC 34 | GGC 19 | GTC 17 | GCC 40 | GAC 31 | GGC 47 | GTC 11 | GCC 31 | GAC 24 | GGC 14 |
| GTA 0 | GCA 2 | GAA 1 | GGA 1 | GTA 0 | GCA 3 | GAA 9 | GGA 3 | GTA 0 | GCA 0 | GAA 0 | GGA 1 |
| GTG 13 | GCG 16 | GAG 34 | GGG 5 | GTG 21 | GCG 11 | GAG 19 | GGG 7 | GTG 9 | GCG 12 | GAG 16 | GGG 4 |

ORF1 (7-CI)  
ORF2 (6-OH)  
ORF3  

ORF2 and ORF3 code for 6-hydroxylation enzyme and 4- or 6-methylation enzyme, respectively, in 6-demethylchlortetracycline biosynthesis. To explore these possibilities, we constructed the derivative strains of *S. aureofaciens* NRRL3203 in which ORF2 or ORF3 was replaced with the mutated sequence by the gene replacement technique. These strains were cultivated in the production medium, and the fermentation products were analyzed by HPLC. In the culture broth of the ORF2-inactivated strain (NP6), an unknown large peak which was eluted after 6-demethylchlortetracycline was detected. The compound was isolated and identified as 6-demethylanhdrochlortetracycline, known to be an intermediate just before 6-hydroxylation, confirming that ORF2 codes for 6-hydroxylation enzyme. On the other hand, ORF3-inactivated strain (NP4) produced no intermediates detectable on HPLC under the conditions used.

### Discussion

We have isolated cosmid clones containing 6-demethylchlortetracycline biosynthetic genes of *S. aureofaciens* NRRL3203 by taking advantage of the close linkage between antibiotic biosynthetic genes and self-defense genes. On the basis of our results, it is probable that two self-defense genes involved in 6-hydroxylation, 7-chlorination, and possibly in methyltransfer are clustered in a genomic DNA region of 40 kb. Recently, Rotovin et al. have reported the isolation of a recombinant shuttle cosmid clone (LP²127) replicable in both *E. coli* and *S. lividans*, which contains the chlortetracycline biosynthetic gene cluster from *S. aureofaciens* ATCC13899, a chlortetracycline producer. The structure of their inserted DNA fragment is similar to that of our pGLA2 as judged by the restriction enzyme analysis. Though they did not locate each biosynthetic gene on the cosmid, they showed that LP²127 could confer on *S. lividans* TK54 the ability to produce chlortetracycline, suggesting that all or almost all genes responsible for the antibiotic biosynthesis exist in the DNA fragment cloned in LP²127 and in our pGLA2.

In this work, we have developed a self-cloning system for *S. aureofaciens* NRRL3203. Besides pIJ702, pIJ41, and pGM9 were also able to transform *S. aureofaciens* NRRL3203 with the efficiency of more than \(10^6\) trans-

chlorination of tetracycline. Further experiments, e.g., overproduction of this gene product and enzymatic studies, are necessary.

By the Northern blot analysis, the *chl* gene was shown to be polycistrionically transcribed together with the 6-hydroxylation gene and the possible methyltransferase gene. This suggests that a putative promoter of this transcript precedes the 6-hydroxylation gene. On the other hand, pCLSB1, which has only the 2.6-kb insert and does not contain the 6-hydroxylation gene, could complement the chlorinating activity of NP385. This could be explained by one of the following: (i) the expression of the *chl* gene on pCLSB1 might result from a read through transcription from a promoter on pIJ702, or (ii) there might be a weak promoter preceded the *chl* gene, the activity of which was very low as the transcript could not be detected by the Northern blot analysis. The following facts support the latter assumption. pCLSB1 was constructed by inserting the *SacI-BamHI* fragment into pIJ702 digested with *SacI* and *BglII*, the sites of which exist in the *mel* gene, a selective marker of the vector. The *mel* gene was known to be transcribed in the *BglII* to *SacI* direction.³¹⁾ On the other hand, the *chl* gene was thought to be transcribed from *SacI* to *BamHI* direction, which was in the opposite direction relative to the *mel* gene as judged by the deduced amino acid sequence, suggesting that a read through transcription from a promoter on the vector is improbable. The detection of the promoter activity on the *SacI-BamHI* fragment and the location of the transcriptional starting site would help in understanding this.

Acknowledgments. We thank Mrs. Michiko Nakagawa for her excellent technical assistance. We are grateful to W. Wohlleben for providing us with pGM9.

### References

1) S. L. Neidleman and J. Geigert, *Biohalogenation: Principles, Basic Roles and Applications*, Ellis Horwood Limited, England, 1986.
2) W. Wiesner, K. H. van Pee, and F. Lingens, *FEBS Lett.*, **209**, 321–324 (1986).
3) K. H. van Pee, G. Sury, and F. Lingens, *Biol. Chem. Hoppe-Seyler*, **368**, 1225–1232 (1987).
4) R. Zeiner, K. H. van Pee, and F. Lingens, *J. Gen. Microbiol.*, **134**, 3141–3149 (1988).
5) M. Weng, O. Pfeifer, S. Krauss, F. Lingens, and K. H. van Pee, *J.*

6) J. Ehrlich, Q. R. Bartz, R. M. Smith, D. A. Joslyn, and P. R. Burkholder, *Science*, **106**, 417 (1947).
7) B. M. Duggar, *Ann. N.Y. Acad. Sci.*, **51**, 177–181 (1948).
8) L. C. Vining and D. W. S. Westlake, in “Biotechnology of Industrial Antibiotics,” ed. by E. J. Vandamme, New York, Marcel Dekker, 1984, pp. 387–411.
9) Z. Hostalek and Z. Vanek, in “Biotechnology,” Vol. 4, Chap. 13, ed. by H. J. Rehm and G. Reed, VCH, Germany, 1986, pp. 393–429.
10) M. Knoch, K. H. van Pee, L. C. Vining, and F. Lingens, *J. Gen. Microbiol.*, **135**, 2493–2502 (1989).
11) B. Hohn and J. Collins, *Gene*, **11**, 291–298 (1980).
12) T. Maniatis, E. F. Fritsch, and J. Sambrook, “Molecular Cloning, A Laboratory Manual,” Cold Spring Harbor Laboratory, Cold Spring Harbor, New York (1982).
13) D. A. Hopwood, M. J. Bibb, K. F. Chater, T. Kieser, C. J. Bruton, H. M. Kieser, D. J. Lydiate, C. P. Smith, J. M. Ward, and H. Schrempf, “Gene Manipulation of Streptomyces, A Laboratory Manual,” The John Innes Foundation, Norwich, UK.
14) G. Muth, B. Nussbaumer, W. Wohlleben, and A. Puhler, *Mol. Gen. Genet.*, **219**, 341–348 (1989).
15) T. Dairi and M. Hasegawa, *J. Antibiot.*, **42**, 934–943 (1989).
16) I. Vancurova, J. Volc, M. Flieger, J. Neuzil, J. Novotna, J. Vlach, and V. Behal, *Biochem. J.*, **253**, 263–267 (1988).
17) M. Hasegawa, T. Dairi, T. Ohta, and E. Hashimoto, *J. Bacteriol.*, **173**, 7004–7011 (1991).
18) T. Dairi, T. Ohta, E. Hashimoto, and M. Hasegawa, *Mol. Gen. Genet.*, **236**, 39–48 (1992).
19) P. S. Tomas, *Proc. Natl. Acad. Sci. U.S.A.*, **77**, 5201–5205 (1980).
20) H. Anzai, Y. Kumada, O. Hara, T. Murakami, T. Itoh, E. Takano, S. Imai, A. Satoh, and K. Nagaoka, *J. Antibiot.*, **41**, 226–233 (1988).
21) D. A. Hopwood, *Ann. Rev. Microbiol.*, **35**, 237–272 (1981).
22) T. Ohta, E. Hasegawa, T. Dairi, and M. Hasegawa, in “Trends in Actinomycetology,” ed. Y. Koyama, Society for Actinomycetes, Japan, 1989, pp. 101–102.
23) C. Binnie, M. Warren, and M. J. Butler, *J. Bacteriol.*, **171**, 887–895 (1989).
24) M. J. Bibb, P. R. Findlay, and M. W. Johnson, *Gene*, **30**, 157–166 (1984).
25) I. Ishida, M. Obinata, and T. Deguchi, *J. Biol. Chem.*, **262**, 2895–2899 (1987).
26) J. A. Lotvin, M. J. Ryan, and N. Strthy, EP Patent 0468 220 A2 (1992).
27) J. Kormanec, B. Rezuchova, and M. Farkasovsky, *J. Gen. Microbiol.*, **139**, 2525–2529 (1993).
28) K. H. van Pee, *J. Bacteriol.*, **170**, 5890–5894 (1988).
29) O. Pfeifer, I. Pelletier, J. Altenbuchner, and K. H. van Pee, *J. Gen. Microbiol.*, **138**, 1123–1131 (1992).
30) I. Pelletier, O. Pfeifer, J. Altenbuchner, and K. H. van Pee, *Microbiology*, **140**, 509–516 (1994).
31) V. Bernan, D. Filpula, W. Herber, M. Bibb, and E. Katz, *Gene*, **37**, 101–110 (1985).
