
JOURNAL OF BACTERIOLOGY, May 1995, p. 2695–2706
0021-9193/95/$04.00+0
Copyright © 1995, American Society for Microbiology

# Oxygen-Insensitive Synthesis of the Photosynthetic Membranes  
**of Rhodobacter sphaeroides: a Mutant Histidine Kinase**

**JESUS M. ERASO AND SAMUEL KAPLAN***  
*Department of Microbiology and Molecular Genetics, The University of Texas Medical School, Houston, Texas 77030*

Received 28 December 1994/Accepted 10 March 1995

Two new loci, *prrB* and *prrC*, involved in the positive regulation of photosynthesis gene expression in response to anaerobiosis, have been identified in *Rhodobacter sphaeroides*. *prrB* encodes a sensor histidine kinase that is responsive to the removal of oxygen and functions through the response regulator PrrA. Inactivation of *prrB* results in a substantial reduction of photosynthetic spectral complexes as well as in the inability of cells to grow photosynthetically at low to medium light intensities. Together, *prrB* and *prrA* provide the major signal involved in synthesis of the specialized intracytoplasmic membrane (ICM), harboring components essential to the light reactions of photosynthesis. Previously, J. K. Lee and S. Kaplan (J. Bacteriol. 174:1158–1171, 1992) identified a mutant which resulted in high-level expression of the *puc* operon, encoding the apoproteins giving rise to the B800-850 spectral complex, in the presence of oxygen as well as in the synthesis of the ICM under conditions of high oxygenation. This mutation is shown to reside in *prrB*, resulting in a leucine-to-proline change at position 78 in mutant PrrB (PRRB78). Measurements of mRNA levels in cells containing the *prrB78* mutation support the idea that *prrB* is a global regulator of photosynthesis gene expression. Two additional mutants, PRRB1 and PRRB2, which make two truncated forms of the PrrB protein, possess substantially reduced amounts of spectral complexes. Although the precise role of *prrC* remains to be determined, evidence suggests that it too is involved in the regulatory cascade involving *prrB* and *prrA*. The genetic organization of the photosynthesis response regulatory (PRR) region is discussed.

---

*Rhodobacter sphaeroides* is a purple, nonsulfur photoheterotrophic bacterium whose mode of growth depends on the concentration of oxygen in the surrounding environment (49). At high oxygen concentrations, *R. sphaeroides* grows chemoheterotrophically, and morphologically it resembles a typical gram-negative bacterium (3). Under low oxygen or anaerobiosis, the photosynthetic membrane system, designated the intracytoplasmic membrane (ICM), is induced; the ICM fills the interior of the cell, and its abundance is inversely related to the incident light intensity (25). The ICM contains all of the structural and functional components of the photosynthetic apparatus. Included are three distinct bacteriochlorophyll (Bchl) a pigment-protein complexes: the photochemical reaction center (RC) (36) and the two light-harvesting complexes, designated the B800-850 and B-875 spectral complexes (7, 9). When grown anaerobically in the dark, in the presence of alternative electron acceptors, the gratuitous synthesis of the ICM occurs (53).

Lee and Kaplan (28) provided the first demonstration that a relatively simple switching mechanism must be involved in the repression of photosynthesis gene expression by oxygen in the genus *Rhodobacter* when they isolated the spontaneous mutant T<sub>1a</sub>, which accumulates photosynthetic membranes in the presence of high oxygen. Subsequently, PrrA, the response regulator of a two-component regulatory system which mediates the cellular response following the removal of oxygen, was identified in *R. sphaeroides* (17). PrrA positively regulates the expression of the *puc* and *puf* operons, which encode the α and β polypeptides of the B800-850 and B875 complexes, respectively, and the L (RC-L) and M (RC-M) polypeptides of the RC. *puhA*, encoding the RC-H polypeptide of the RC, as well as *cycA*, encoding cytochrome *c*<sub>2</sub>, are also positively regulated by PrrA (17, 25). A similar gene, *regA*, had previously been identified in the related bacterium *Rhodobacter capsulatus*, in which it positively regulates the expression of *puf::lacZ*, *puh::lacZ*, and *puc::lacZ* translational fusions. *regA* was reported not to be involved in regulation of the genes which encode the enzymes involved in the synthesis of the carotenoid (Crt) or Bchl pigments (40). The *prrA* equivalent from *R. sphaeroides* NCIB 8253, designated *regA*, has also been cloned and sequenced (38).

Reported here is the cloning and sequencing of the photosynthesis response regulatory (PRR) region from the wild-type and the *trans*-acting, oxygen-insensitive mutant T<sub>1a</sub> (28), identifying other genes involved in oxygen control of photosynthesis gene expression. One of these genes, designated *prrB*, and its mutant allele *prrB78* are shown to encode the histidine kinase responsive to anaerobiosis. This gene is similar to the recently identified *regB* gene in *R. capsulatus* (34). A mutant form of *prrB* responsible for its insensitivity to oxygen results in synthesis of the ICM in the presence of oxygen, providing the basis for the initial observation of Lee and Kaplan (28). Thus, *prrB* encodes a global positive regulator of photosynthesis gene expression, including the accumulation of Crt and Bchl. A third gene, *prrC*, whose gene product belongs to a family of highly conserved proteins which are anchored in the membrane, was identified.

## MATERIALS AND METHODS

### Bacterial strains, plasmids, and growth conditions.
The bacterial strains and plasmids used are listed in Table 1. *Escherichia coli* strains were grown at 37°C in LB (43). *R. sphaeroides* strains were grown at 30°C in Sistrom’s minimal medium A supplemented with succinate as the carbon source (45). Tetracycline, streptomycin, spectinomycin, and kanamycin were added at 1, 50, 50, and 25 μg/ml, respectively, to *R. sphaeroides* cultures when required and at 20, 50, 50, and 25 μg/ml, respectively, for *E. coli* cultures. Kanamycin was used at 450 μg/ml

* Corresponding author. Mailing address: Department of Microbiology and Molecular Genetics, The University of Texas Medical School, 6431 Fannin, Houston, TX 77030. Phone: (713) 794-1742. Fax: (713) 794-1782.

TABLE 1. Bacteria and plasmids used

| Strain | Genotype and phenotype$^a$ | Reference or source |
|--------|-----------------------------|---------------------|
| $E. coli$ | DH5$\alpha$phe | F$^{-}$ $\phi$80dlacZ$\Delta$M15 $\Delta$(lacZYA-argF) U169 recA1 endA1 hsdR17(rK$^{-}$ mK$^{+}$) supE44 $\lambda^{-}$ thi-1 gyrA relA1 phe::Tn10dCm | 17 |
| GM2163 | dam-13::Tn9 (Cm$^{r}$) dcm-6 as relevant markers | New England Biolabs |
| $R. sphaeroides$ | 2.4.1 | Wild type | W.R. Sistrom |
| 2.4.1$\Omega$ | 2.4.1::$\Omega$ Sm$^{r}$ Sp$^{r}$ at SmaI site downstream from prrB | This study |
| PRRA1 | 2.4.1prrA::$\Omega$ Sm$^{r}$ Sp$^{r}$ PS$^{-}$ RC$^{-}$ B875$^{-}$ B800-850$^{-}$ Car$^{-}$ | 17 |
| PRRA2 | 2.4.1prrA$\Delta$BstBI$^{b}$-PstI$^{c}$::$\Omega$ Sm$^{r}$ Sp$^{r}$ PS$^{-}$ RC$^{-}$ B875$^{-}$ B800-850$^{-}$ Crt$^{-}$ | This study |
| PRRB1 | 2.4.1prrB$\Delta$NruI-RsrII$^{b}$::$\Omega$ Sm$^{r}$ Sp$^{r}$ PS$^{-}$ | This study |
| PRRB1Km | 2.4.1prrB$\Delta$NruI-RsrII$^{b}$::$\Omega$ Km$^{r}$ PS$^{-}$ | This study |
| PRRB2 | 2.4.1prrB::$\Omega$ Sm$^{r}$ Sp$^{r}$ at EcoRI PS$^{-}$ | This study |
| PRRC1 | 2.4.1prrC$\Delta$Eco0109I$^{b}$-HincII::$\Omega$ Sm$^{r}$ Sp$^{r}$ | This study |
| PRRC2 | 2.4.1prrC$\Delta$HincII-HincII::$\Omega$ Sm$^{r}$ Sp$^{r}$ | This study |
| PRRCA1 | 2.4.1prrCA$\Delta$Eco0109I-Eco0109I$^{b}$::$\Omega$ Sm$^{r}$ Sp$^{r}$ PS$^{-}$ | This study |
| PRRBC1 | 2.4.1prrBC$\Delta$EcoRI$^{b}$-StuI::$\Omega$ Sm$^{r}$ Sp$^{r}$ PS$^{-}$ | This study |
| PRRBCA1 | 2.4.1prrBCA$\Delta$BspEI-Tth111I$^{b}$::$\Omega$ Sm$^{r}$ Sp$^{r}$ PS$^{-}$ | This study |
| PRRB78 | 2.4.1 derivative with the mutation T1020C which results in the change Leu-78$\rightarrow$Pro in PrrB and $\Omega$ Sm$^{r}$ Sp$^{r}$ at SmaI site downstream from prrB | This study |
| $T_{la}$ | Oxygen-insensitive expression of the ICM and puc operon | 28 |
| Plasmids | pRK415 | Tc$^{r}$ | 23 |
| pBluescriptII | Ap$^{r}$, with T3 and T7 promoters | Stratagene |
| pSUP203 | pBR325 derivative; Mob$^{+}$Ap$^{r}$Cm$^{r}$Tc$^{r}$ | 44 |
| pLK-1 | pLV106 derivative ($\Delta$tet) + 4.2-kb PvuII fragment of pPXK-1 with $\Omega$Sm$^{r}$Sp$^{r}$-puc (0.8-kb PstI-XmnI) -1.07-kb XhoI-EcoRIKm$^{r}$DNA | 26a, 27, 28 |
| pSL301 | 3.2-kb superlinker vector containing an extended polylinker | Invitrogen |
| pUI624 | pBS/HincII, 0.53-kb XmaIII fragment from puc; Ap$^{r}$ | 29 |
| pUI655 | pBS/HincII, 0.47-kb StyI fragment of pufBA; same orientation as T3 promoter; Ap$^{r}$ | 26a |
| pUI660 | pBS/SphI-SalI, approximately 0.18-kb SphI-XhoI fragment of puhA; same orientation as T3 promoter; Ap$^{r}$ | 26a |
| pUI661 | pBS/HincII, internal 0.35-kb StuI fragment of cycA; same orientation as T3 promoter; Ap$^{r}$ | 26a |
| pUI1621 | pRK415 derivative with $\Omega$Sm$^{r}$Sp$^{r}$ (HindIII) plus approximately 0.8 kb from cosmid 533, containing only prrA; Tc$^{r}$ | 17 |
| pUI1637 | Source of the $\Omega$Km$^{r}$ | 17 |
| pUI1638 | Source of the $\Omega$Sm$^{r}$Sp$^{r}$ | 17 |
| pUI1639 | pSL301/NcoI, approximately 10.4-kb NcoI fragment containing an $\Omega$Sm$^{r}$Sp$^{r}$ at the PstI site in prrA cloned from chromosome I of PRRA1; Ap$^{r}$ | This study |
| pUI1640 | pBS/PstI-BamHI, approximately 4.9-kb PstI-BglII fragment cloned from chromosome I of PRRA1; Ap$^{r}$ | This study |
| pUI1641 | pRK415/HindIII,$^{b}$ 3-kb NruI-HindIII$^{b}$ fragment containing prrB transcribed opposite to the vector tet and lac promoters; Tc$^{r}$ | This study |
| pUI1642 | pRK415/HindIII,$^{b}$ 3-kb NruI-HindIII$^{b}$ fragment containing prrB in the same orientation as the vector tet and lac promoters; Tc$^{r}$ | This study |
| pUI1643 | pBS/BamHI-HindIII, 4-kb BamHI-HindIII fragment cloned from chromosome I of 2.4.1; Ap$^{r}$ | This study |
| pUI1644 | pBS/BamHI-HindIII, 4-kb BamHI-HindIII fragment cloned from chromosome I of $T_{la}$; Ap$^{r}$ | This study |
| pUI1645 | pRK415/Ecl136II, XhoI$^{b}$ 1.3-kb fragment containing prrC from 2.4.1 transcribed opposite to the vector tet and lac promoters; Tc$^{r}$ | This study |
| pUI1649 | pRK415/BamHI-Asp718, 1.9-kb SmaI-ClaI fragment containing prrB from 2.4.1 transcribed opposite to the vector tet and lac promoters; Tc$^{r}$ | This study |
| pUI1650 | pRK415/BamHI-Asp718,$^{b}$ 1.9-kb SmaI-ClaI fragment containing prrB from $T_{la}$ transcribed opposite to the vector tet and lac promoters; Tc$^{r}$ | This study |
| pUI1651 | pBS/BamHI-HindIII, 6.2-kb BamHI-HindIII fragment cloned from chromosome I of 2.4.1 with $\Omega$Sm$^{r}$Sp$^{r}$ at SmaI; Ap$^{r}$ | This study |
| pUI1652 | pBS/BamHI-HindIII, 6.2-kb BamHI-HindIII fragment cloned from chromosome I of $T_{la}$ with $\Omega$Sm$^{r}$Sp$^{r}$ at SmaI; Ap$^{r}$ | This study |
| pUI1653 | pBS derivative, approximately 2.0-kb StuI-HindIII from pUI1643$\Delta$1037-bp NruI-RsrII in prrB with $\Omega$Sm$^{r}$Sp$^{r}$; Ap$^{r}$ | This study |
| pUI1654 | pBS derivative, approximately 2.0-kb StuI-HindIII from pUI1643$\Delta$1037-bp NruI-RsrII in prrB with $\Omega$Km$^{r}$, Ap$^{r}$ | This study |
| pUI1655 | pBS derivative, approximately 2.0-kb PstI-RsrII from pUI1643$\Delta$475-bp HincII-Eco0109I in prrC with $\Omega$Sm$^{r}$Sp$^{r}$; Ap$^{r}$ | This study |
| pUI1656 | pBS derivative, approximately 2.4-kb PstI-RsrII from pUI1643$\Delta$77-bp HincII-HincII in prrC with $\Omega$Sm$^{r}$Sp$^{r}$; Ap$^{r}$ | This study |
| pUI1657 | pBS derivative, approximately 1.9-kb PstI-RsrII from pUI1643$\Delta$583-bp Eco0109I-Eco0109I in prrCA with $\Omega$Sm$^{r}$Sp$^{r}$; Ap$^{r}$ | This study |
| pUI1658 | pUI1643$\Delta$475-bp StuI-EcoRI in prrBC with $\Omega$Sm$^{r}$Sp$^{r}$; Ap$^{r}$ | This study |
| pUI1659 | pBS derivative plus approximately 5.2 kb containing PRR region with a 2,070-bp $\Delta$Tth111I-BspEIprrBCA$\Delta$ with $\Omega$Sm$^{r}$Sp$^{r}$; Ap$^{r}$ | This study |

$a$ PS$^{-}$, cells are photosynthetically incompetent; RC$^{-}$, cells do not have RC pigment-protein complexes.

$b$ The 5' overhangs were made blunt with Klenow fragment of DNA polymerase I before cloning.

$c$ The 3' overhangs were made blunt with T4 DNA polymerase before cloning.

VOL. 177, 1995                                                                 OXYGEN REGULATION OF GENE EXPRESSION IN R. SPHAEROIDES 2697

![Diagram]

FIG. 1. Restriction map of the 10.4-kb NcoI fragment cloned from chromosome I of *R. sphaeroides* PRRA1 as well as the structure of the PRR region. The 10.4-kb NcoI fragment from pUI1639 is indicated at the top. The locations of *prrB*, *prrC*, and *prrA* and the Ω cartridge within the fragment are indicated by the shaded areas. Unique restriction sites as well as sites used for subcloning are marked. pUI1640 contains an approximately 4.9-kb *PstI-BglII* fragment. pUI1641 and pUI1642 contain the same *HindIII-NruI* fragment in opposite orientations. A map of the approximately 4-kb *HindIII-BamHI* fragment containing the PRR region from wild-type and T<sub>la</sub> cells in pUI1643 and pUI1644 is shown expanded at the bottom. The locations of *prrA*, *prrB*, and *prrC* are indicated by the shaded areas. The arrows indicate the direction of transcription. Relevant restriction sites are marked. Ω represents an Ω Sm/Sp cartridge. Scale markers are placed for the 10.4- and 4-kb fragments.

for growth of PRRB78(pLK-1). Ampicillin was used at 150 μg/ml and chloramphenicol was used at 25 μg/ml for *E. coli* cultures. Valinomycin was used at 2 μM. *R. sphaeroides* cultures were grown as described previously (12, 14). *E. coli* GM2163 was used to isolate unmethylated DNA for subcloning.

Cloning of the PRR region from chromosome I of PRRA1, wild-type, and T<sub>la</sub> cells. Chromosomal DNA containing an Ω cartridge encoding resistance to streptomycin and spectinomycin (Ω Sm/Sp) in *prrA* was isolated from PRRA1 cells (17) and used to transform *E. coli* by direct selection for antibiotic resistance. A 10.4-kb NcoI fragment that hybridized with a probe specific to the Ω cartridge was identified by Southern hybridization. Cloning of this 10.4-kb DNA fragment into the NcoI site of pSL301 created plasmid pUI1639 (Fig. 1). This insert contains the PRR region.

Chromosomal DNA was also isolated from wild type as well as T<sub>la</sub> cells (28) and digested with BamHI and HindIII, and following electrophoresis, a band corresponding to the 4-kb region was cloned into pBS which had been previously digested with BamHI and HindIII. Clones containing the PRR region were identified by restriction mapping. pUI1643 contains the PRR region from wild-type cells, and pUI1644 contains the PRR region from T<sub>la</sub> cells. The map of this region, with the locations of *prrB*, *prrC*, and *prrA*, is shown in Fig. 1. pUI1641 and pUI1642 were subsequently constructed by subcloning pUI1639. They contain an approximately 3-kb *NruI-HindIII* fragment containing the region immediately upstream from *prrA* from wild-type cells in pRK415, in both orientations. The two plasmids were used for DNA sequencing.

Construction of mutations in the PRR region. The 4-kb *BamHI-HindIII* insert in pUI1643 (Fig. 1) was used to construct mutations in the PRR region. A diagrammatic representation of all strains constructed is shown in Fig. 2. pUI1653 was used to construct a chromosomal deletion of the *prrB* gene. This plasmid contains a 1,037-bp *RsrII-NruI* deletion internal to the *prrB* gene, with an Ω Sm/Sp cartridge in its place. pUI1654 is similar to pUI1653, but it contains a kanamycin resistance (Ω Km) cartridge instead. The modified inserts from pUI1653 and pUI1654 were cloned into pSUP203 and crossed into chromosome I of wild-type *R. sphaeroides* 2.4.1, to create PRRB1 and PRRB1Km, respectively, as were all other chromosomally derived mutations, using techniques described previously (17). A similar strategy was used to construct pUI1655, which was used to inactivate *prrC* with an Ω Sm/Sp cartridge in place of a 475-bp *Eco0109I-HincII* fragment to create PRRC1.

Since *prrA* may be cotranscribed with *prrC* as well as being expressed from its own promoter, the presence of an Ω cartridge in *prrC* prompted the construction of two additional mutations. In the first, a 77-bp *HincII-HincII* deletion with an Ω Sm/Sp cartridge in pUI1656 was used to construct a pSUP203 derivative which in turn was used to generate PRRC2. The 77-bp deletion is near the 3′ end of *prrC* but does not affect the promoter for *prrA* (17). In the second construction, portions of both *prrC* and *prrA* were deleted by using plasmid pUI1657, containing a 583-bp *Eco0109I-Eco0109I* deletion substituted with an Ω Sm/Sp cartridge. This construction is designated PRRCA1.

PRRBC1 was made by crossing the insert from pUI1658, which contains a 475-bp *StuI-EcoRI* deletion of both *prrB* and *prrC* substituted with an Ω Sm/Sp cartridge. PRRCA1 was made using the insert from pUI1659, in which an Ω Sm/Sp cartridge is substituted for a 2,070-bp *Tth111I-BspEI* fragment containing most of *prrB*, all of *prrC*, and most of *prrA* in a 5,232-bp fragment containing the PRR region from pUI1643.

Additionally, an Ω Sm/Sp cartridge was inserted into the unique *EcoRI* site in *prrB* of a 4.9-kb *PstI-BglII* insert present in pUI1640 and crossed into the chromosome (Fig. 1). The mutant strain was designated PRRB2. All mutations were confirmed by Southern hybridization.

Subcloning of *prrB*, *prrC*, and *prrA* from wild-type and T<sub>la</sub> cells and construction of PRRB78 and 2.4.1Ω. *prrB*, *prrC*, and *prrA* were individually subcloned from either the wild type via pUI1643 or from T<sub>la</sub> via pUI1644 (Fig. 1) into pRK415. See Table 1 for plasmid designations, genotypes, and origin of the insert DNA.

pUI1651 and pUI1652 (Table 1) contain the approximately 4-kb *BamHI-HindIII* fragment from pUI1643 and pUI1644, respectively (Fig. 1), with an Ω Sm/Sp cartridge cloned at the *SmaI* site immediately downstream from the *prrB* gene. PRRB1Km (Fig. 2), which is unable to grow photosynthetically (PS<sup>−</sup>) at low to medium light intensities, was used as recipient in crosses with the pSUP203 derivatives containing either the wild-type DNA insert from pUI1651 or the *prrB*78 allele from T<sub>la</sub> cloned into pUI1652. Exconjugants were selected for both Sm<sup>r</sup> Sp<sup>r</sup> and tested for the ability to grow photosynthetically. Subsequently they were scored for Tc<sup>s</sup> and Km<sup>s</sup>. The results of these crosses created strains 2.4.1Ω, which contains *prrB*<sup>+</sup>, and PRRB78, which contains *prrB*78. The 6.2-kb *HindIII-BamHI* fragment from the chromosome of 2.4.1Ω and PRRB78 was subsequently subcloned into pBS and sequenced to confirm the presence of the wild-type and mutated sequences, respectively.

Molecular techniques. Standard techniques were used for plasmid isolation, restriction endonuclease digestion, isolation of DNA fragments from gels, ligations, and other molecular biological methods (2, 32, 43). Deletion clones for sequencing were prepared in both orientations with respect to the insert, using the Exo/Mung Bean nuclease deletion kit from Stratagene (La Jolla, Calif.). The linear plasmids were purified by using Geneclean (Bio101, La Jolla, Calif.) and/or the Wizard DNA cleanup kit (Promega, Madison, Wis.). A series of deletions differing by approximately 200 nucleotides was made from pUI1641 and pUI1642 (Fig. 1) as instructed by the manufacturer. These clones were used to determine the DNA sequences of pUI1641 and pUI1642. The −21 primer (5′-TGTA AAAACGAGCGGCCAGT-3′) was used to sequence the deletion clones. Sequence determination was performed with an ABI 373A automatic DNA sequencer, with a Taq DyeDeoxy terminator cycle sequencing kit (Perkin-Elmer, Applied Biosystems Division, Foster City, Calif.) at the DNA core facility of the Department of Microbiology and Molecular Genetics. Eleven oligonucleotides were designed to sequence *prrB*78 and *prrB* as a control.

DNA sequence analysis. Sequence analyses were performed by using the

2698 ERASO AND KAPLAN J. BACTERIOL.

Wild-type

PRRB1

PRRB2

PRRC1

PRRC2

PRRCA1

PRRBC1

PRRBCA1

FIG. 2. Chromosomal mutations in the PRR region. Shaded areas indicate the genes *prrB*, *prrC*, and *prrA*. Arrows indicate the direction of transcription of the genes. Relevant restriction enzyme sites are marked. Δ indicates the extent of deletions made. In all cases, Ω Sm/Sp cartridges were cloned and are indicated as Ω. Strain PRRB2 does not contain a deletion but contains an Ω Sm/Sp cartridge at the EcoRI site of *prrB*. All constructions are described in Materials and Methods and in Table 1.

computer programs SeqEd 675 DNA Sequence Editor (Perkin-Elmer, Applied Biosystems Division) and DNA Strider (Institut de Recherche Fondamentale, Commissariat à l’Energie Atomique, Paris, France), and the Genetics Computer Group software package from the University of Wisconsin. The program Pep-tidestructure was used to predict the folded structures of *prrB* and *prrC*. Sequence similarity requests as well as secondary structure prediction and motif searches were also performed.

Genetic techniques. Plasmid DNA was mobilized into *R.* *sphaeroides* by conjugation, using techniques previously described (12). The need to supplement Sistrom’s minimal medium with K₂TeO₃ could be bypassed by using the auxotroph *E.* *coli* DH5αphe as the donor in conjugations (17).

RNA isolation and Southern, Northern (RNA), and slot blot hybridization techniques. RNA was isolated from cells grown both chemoheterotrophically and anaerobically in the dark. RNA was isolated and assayed as described previously (8, 17, 54). Quantitation of the level of rRNA present in each slot or lane was used as a normalization factor for each transcript. QuickHyb solution from Stratagene was used for certain hybridizations as instructed by the manufacturer.

Radioactive probes were made by using highly purified plasmid DNA containing *puf* (pUI655), *puc* (pUI624) (29), *cycA* (pUI661), and *puhA* (pUI660) (26a) (Table 1). All procedures have been described previously (14).

Analytical techniques. Protein determinations were done by the Pierce assay (Pierce, Rockford, Ill.). The amount of Bchl present in the B875 light-harvesting complex can be measured at 875 minus 820 nm (*ε* = 73 ± 2.5 mM⁻¹ cm⁻¹), normalized for 2 mol of Bchl a per complex, whereas the concentration in the B800-850 complex can be measured at 849 minus 900 nm (*ε* = 96 ± 4 mM⁻¹ cm⁻¹), normalized for 3 mol of Bchl a, as determined previously (25, 33). Cells grown chemoheterotrophically at 30% oxygen were broken in a French press and suspended in 50 mM morpholine propane sulfonic acid (MOPs)-100 mM KCl

(pH 7.0) containing 2 μM valinomycin and 2 mM potassium ferrocyanide and ferricyanide for the measurement of the RC spectrum change (5). Flash photolysis used an neodymium-yttrium aluminum garnet pulse laser (6-ns pulse width, 532 nm, 40 mJ) in a modification of a procedure described for the measurement of flash-induced absorbance changes (37). The flashing rate was 0.017 Hz. Each trace was acquired as 3,968 datum points, and the acquisition rate was 500 μs per point. In all cases, initial points of flash artifact were removed. The sample was at ambient temperature. The method used to calculate the amounts of Bchl and Cts has been described elsewhere (10).

Materials. 5-Bromo-4-chloro-3-indoly-β-D-galactoside (X-Gal) and the Kle-now fragment of DNA polymerase I were purchased from Boehringer Mann-heim Biochemicals, Indianapolis, Ind. The p-toluidine salt of 5-bromo-4-chloro-3-indoly phosphate was purchased from Sigma Chemical Co., St. Louis, Mo. Restriction endonucleases and nucleic acid-modifying enzymes were purchased from Bethesda Research Laboratories Life Technologies, Inc., Gaithersburg, Md., New England Biolabs, Inc., Beverly, Mass., or Promega. [α-³²P]dCTP (3,000 Ci/mmol) and [α-³²P]CTP (800 Ci/mmol) were purchased from Amer-sham Corp., Arlington Heights, Ill. All other chemicals used in this work were reagent grade.

Nucleotide sequence accession number. The nucleotide sequence of the region containing *prrB* and *prrC* has been assigned GenBank accession number U22347.

RESULTS

Identification of a regulatory locus upstream of *prrA*. In *R.* *sphaeroides*, PrrA was shown to be involved in the positive transcriptional regulation of photosynthesis gene expression in

response to anaerobiosis (17). Since the kinase and regulator genes are often physically adjacent in two-component systems (1, 4, 35, 47), we assumed that the putative kinase gene would be physically close to *prrA*. Sequencing of the region downstream from *prrA* did not reveal the presence of a gene with the capacity to encode a histidine kinase (17). Therefore, the region upstream from *prrA* on chromosome I was studied further. In a separate but related approach, the corresponding region of chromosome I from mutant T<sub>1a</sub> was also isolated. We reasoned that the kinase was a likely candidate for the regulatory mutation in T<sub>1a</sub>. This conclusion was based on our observation that secondary mutations which silenced the original T<sub>1a</sub> phenotype, i.e., ICM production in the presence of oxygen, were invariably localized to *prrA*. Therefore, it seemed reasonable that the mutation in T<sub>1a</sub> must be upstream of *prrA* in the regulatory pathway.

The 4.9-kb *PstI-BglII* insert in pUI1640 containing an incomplete *prrA* gene and upstream sequence (Fig. 1) was cloned from the chromosome of PRRA1 as described in Materials and Methods. This region was mutagenized by placing Ω cassettes in several locations, and it was found that interruption at the *EcoRI* site located approximately 1.2 kb upstream from *prrA* (Fig. 1) gave rise to decreased spectral complex formation when crossed into the wild-type genome. This mutant was designated PRRB2 (Fig. 2). Moreover, when plated anaerobically at 10-W/m² light intensity, these cells were unable to grow photosynthetically. By contrast, cells with an interruption at the *HindIII* site located approximately 3.1 kb upstream from *prrA* (Fig. 1) had a wild-type phenotype and were photosynthetically competent.

Plasmids pUI1641 and pUI1642 contain approximately 3.1-kb *NruI-HindIII* fragments in opposite orientations in pRK415 (Fig. 1). When placed in trans, these subclones restored both wild-type coloration and photosynthetic competence to PRRB2. Therefore, the insertion mutation in PRRB2 was not polar to *prrA* and thus defined another coding sequence required for photosynthesis gene expression. Additionally, when multiple copies of *prrA* were placed in trans in PRRB2, the photosynthetic defect was bypassed, as might be anticipated since we have shown that *prrA* can be made active by at least one noncognate kinase, notably HupT (19). Therefore, the locus upstream of *prrA* was designated *prrB*, for photosynthetic response, regulator B.

A 4-kb *BamHI-HindIII* DNA fragment was subcloned from mutant T<sub>1a</sub> to form pUI1644 (Fig. 1) as described in Materials and Methods. A corresponding fragment was subcloned from wild-type cells as a control (pUI1643).

Sequencing of the *prrA* upstream region. The sequence of DNA comprising the approximately 3.1-kb *HindIII-NruI* fragment upstream from *prrA* (Fig. 1) was determined as described in Materials and Methods and is shown in Fig. 3. The Genetics Computer Group software package was used in conjunction with an *R. sphaeroides* codon usage table (16) to identify an open reading frame (ORF) of 1,389 bp, within which the *EcoRI* site used to interrupt this ORF was located. This ORF was designated *prrB* (Fig. 1). Additionally, a second ORF was found, which was upstream of and divergently transcribed from *prrB*. When this sequence was assembled with the sequence obtained previously from the 2.2-kb fragment containing *prrA* (17), a stop codon for this second ORF was found, resulting in an ORF of 696 bp which overlapped *prrA* by 8 bp. This new ORF was designated *prrC*. Structures of both *prrB* and *prrC* as well as *prrA* are shown in Fig. 1.

*prrB* and *prrC* encode proteins of 462 and 231 amino acids, respectively (Fig. 3). Arrows at the end of the sequence for *prrB* show a GC-rich stem with a theoretical ΔG of −11.9 kcal

(ca. −49.8 kJ), which is followed by a relatively AT-rich region, similar to a rho-independent terminator structure in *E. coli* (18). Potential Shine-Dalgarno sequences (15, 41) upstream of each ORF are doubly underlined. The match of the suspected Shine-Dalgarno sequence for *prrB* is poor, but that for *prrC* is very good. An in-frame GTG codon located 15 codons upstream of the presumed ATG initiation codon for *prrC* also was found but was not selected as the initiation codon for *prrC* (i) since there is no obvious Shine-Dalgarno sequence nearby and (ii) on the basis of sequence alignment with similar genes (see below).

Similarly, after subcloning the PRR region from both the wild type and mutant T<sub>1a</sub>, it was found that in contrast to *prrB* from wild-type cells, *prrB* from T<sub>1a</sub> cells conferred an oxygen-insensitive phenotype to wild-type cells. Thus, *prrB* was mutated in T<sub>1a</sub>, and the mutated allele was dominant over the wild-type gene. The DNA sequence for *prrB* from T<sub>1a</sub> was determined and revealed a single base pair mutation of a T to a C, which changes codon 78 from CTG (Leu) to CCG (Pro). This allele was designated *prrB78*.

Alignment of *prrB* and *prrC*. In the case of *prrB*, the carboxy-terminal end of the putative protein was found to be similar to that of members of the histidine kinase family of regulators associated with two-component regulatory systems (Fig. 4). The conserved histidine residue, known to be the site of autophosphorylation in other systems (1, 4, 35, 47), is at position 221 in *prrB*. Shaded areas represent the conserved regions 1, 2, 3.1, 3.3, and 3.5 (35). Like other members of the histidine kinase family of proteins, the amino-terminal region encoded by *prrB* was not found to be similar to that of any particular family of proteins. However, similarities to regions of genes which encode the membrane-spanning domains of several proteins were found, providing suggestive evidence that PrrB is anchored in the membrane, with six possible transmembrane domains. The *regB* gene, which encodes a histidine kinase with 58% identity and 75% similarity to *prrB*, has recently been identified in *R. capsulatus* (34). The amino terminus of PrrB is 70% similar and 48% identical to the amino terminus of RegB.

In the case of *prrC*, the strongest similarity was found with two genes from *Saccharomyces cerevisiae*, *sco1* and its homolog *sco2*, which encode mitochondrial inner membrane proteins (Fig. 5) that are involved in the assembly of cytochrome *c* oxidase subunits (6). Three other genes were identified as members of the same family, but their functions are unknown.

Phenotypes of mutants in *prrB* and *prrC*. A series of chromosomal mutations in both *prrB* and *prrC* were constructed as described in Materials and Methods and are depicted in Fig. 2. Mutants PRRB1 and PRRB2 (Fig. 2) were PS⁻ when grown at low to medium light intensities, and the levels of spectral complexes were severely reduced, as determined in cells growing anaerobically in the dark in the presence of dimethyl sulfoxide (DMSO) (Table 2, experiment A). Compared with the control, the levels of B800-850 were reduced dramatically and the level of B875 was reduced by approximately 60%. Thus, mutations in *prrB* behave similarly, although not identically, to those in *prrA* (17). On the other hand, mutations in *prrC* are more subtle. PrrC mutants grow photosynthetically, yet the cellular levels of spectral complexes (Table 2, experiment A) are reduced substantially. In combination with a mutation in *prrB*, the PrrC mutant strain PRRBC1 reveals a synergistic effect on spectral complex formation (Table 2, experiment A) and, as expected, does not grow photosynthetically. Mutations affecting *prrA* are, as anticipated, the most severely affected.

Each of the mutant strains listed in Table 2 was complemented in trans with each of the genes in question from wild-type and the PrB mutant PRRB78, designated *prrB78*, under

GTA prrA

TCAGCCATGA TTGTGCCCCA TCTGCTTTCC CCTGGATGCT TGATAGGACT TCTGCCGCCC G AGCGTCAAAG GAAGTTGACA CATGGCTGTC ATTCGCGAAG 100
231 STOP G H N D G M Q K G R S A Q Y S K Q A R A D F S T S V H S D N A F 200
CACGAAATTC GCTCGGCGAT CTGTTCCGGC GTGTCTTCGC GCTTGAAAGA GTCGACGAAAG CCCGTGCCGG GCAGCATGAA GTAGGTGAAG GTCGAATGGT 200
199 C S I R E A I Q E P T D E R K F F D V F G T G P L M F Y T F T S H 167
CGATGAGGTA ATAGTCGTCG TCGGACTCCT GCACCCGGTA GAAGGTCTTG TAGGCCCTGTG AGGCGGCTTT CACCTGGGCC TCGGTGCCGG TCAGCGCGAT 300
166 D I L Y Y D D D S E Q V R Y F T K Y A Q S A A K V Q A E T G T L A I 133
CGTGTCGGGA TGGATCGCCT CGGCGAAGAA CTTCAGCTGT TCGGGCGTGT CGCGCTTGGG ATCGATCGAG ATGAAGACCG GCGTGACCCT GATGCCCCCA 400
132 T D P H I A E A F F K L Q E P T D R K P D I S I F V P T V E I G W 100
TCCGTCAAGA TGTCACACGGC CTGCGCATTG CGGCCATGTG CGAAGGGGCA GACGTCCCCG CAGAAGGTAT AGCCGAAATA GACGAGCGAG GGCTTTGCCA 500
99 E T L I D V A Q A N R A M D F P C V D P C F T Y G F Y V L S P K A 67
GAACCTCGCG GTCCGTCACG GTGCGGCCCT CCTGATCGAC CAGCGTGAAG GGCCCGCCGA TGCGCGCCGC CGCCACCTGA TTGGCGCCGC AACCGGCGAA 600
66 L V E R D T V T R G E Q D V L T F P G G I A G G A V Q N A G C G A F 33
CCGCTCTTCG GATCTGCCGA GGAAAGACCA CGCCGCCGAG CCCGCAAGAA GTGCTGGGAT GGCCGCGCCC GCGACCCCGG CATAGAGCTT CGTCATCAGA 700
32 R E E S R G L F V A A S G A L L A A I A A A A V G A Y L K T M 1 prrC
TGATCCTCCT GCTTGCGGTT GCCGATTGAT CCCGCGGGCA CGGCAACCTA ACAAGGTTCC GATACAGGCA AAGTGACATG AGCTGCAATG ATACTCGGTC 800
M I L G P 5 prrB
CCGACGGCAT TCTGAAACCGT GACACCCCGCG GCGACTGGGT GCGACTGCGC ACCCTGATCC TCCTGCGCTG GATGGCGGTG GCGGGGCAGC TCGCCGCCAT 900
6 D G I L N R D T R G D W V R L R T L I L L R W M A V A G Q L A A I 38
CGTCGTGACC GACTGGTATC TGGGGTCCG CCTGCCGATG GGGCTCTGCT TCATGGCCGT CGGCGCCTCG GTCATCGCGA ACGTGATCGC GACCTTCGTC 1000
39 V V T D W Y L G V R L P M G L C F M A V G A S V I A N V I A T F V 71
TTCCCGCAGA ACCCCCGCCT GACCGAGTTC CAGGCGCTGA TGATCCTGCT CTTCGACCTT AGCAGCTGCT CGTTCCCTGCT GTTCCCTGACC GGGGGGCTCA 1100
72 F P Q N R R L T E F Q A L M I L L F D L T Q L S F L L F L T G G L T 105
CCAACCCGTT CGGCGTGCTG ATCCTCGCGC CCGTCACCAT CTCGGCGCTC GCGCTCGAGC TGCGCACGAC CGTCATTCTC GGGGCCATCG CGATCGGGTC 1200
106 N P F A L L I L A P V T I S A L A L E L R T T V I L G A I A I G L 138
GCTGACCTTC ACGGCCTATT TCCACCTGCC GCTGATCCTC GCGGACGGGT CGAGCCTGTC CGTCCCGGCC ATGTTCGAAT TCGGCTTCTG GCTCGCCATC 1300
139 L T F T A Y F H L P L I L A D G S S L S V P R M F E F G F W L A I 171
GTCATCGGCA TCCGTGTTCC T CGGGCTCTAT TCCCGGCGCG TCGCCATCGA GATCCGTTCG ATGTCGGATG CGCTTCTGGC CACGCAGATG GGCCTCGGAC 1400
172 V I G I L F L G L Y S R R V A I E I R S M S D A L L A T Q M A L D R 205
GCGAGCAGAA GCTGACGGAT CTGGGTGGGG TGGTGGCCGC GGCCGCGCAC GAGCTCGGCA CGCCGCTGGG CACGATCAAG CTGGTGAGCT CGGAGCTTGC 1500
206 E Q K L T D L G G V V A A A A A H E L G T P L A T I K L V S S E L A 238
CGAGGAGCCT TCCGAGCAGC CCGCGCTCGC CGACGATGCC GAGCTGATCC GCGAGCAGGC CGACCGCTGC CGCGACATCC TCCGCTCGAT GGGCCGGGCC 1600
239 E E L S E Q P A L R D D A E L I R E Q A D R C R D I L R S M G R A 271
GGGAAGGACG ATCTTTCAGAT GCGGCAGGCG CCCTTGGGCG AGGTGCTCGG CGAGGCCGCC GAGCCCCATG TCGGGCGCGG CAAGCGGGTC GAGTTCGACC 1700
272 G K D D L Q M R Q A P L G E V L R E A A E P H V G R G K R V E F D L 305
TCTACCCCAG CCGGGCGGGC GACGAGCGCC AGCCGGTGAT CCTGCGCCGT CCCGAGGTGA TCCACGGGCT GCGCAACCTC ATTCAAGAATG CCCTCGATTT 1800
306 Y P S R G G D E R Q P V I L R R P E V I H G L R N L I Q N A V D F 338
CGCCCAGTCC ACGGTCTGGA TCGACGGCGA ATGGACGGGA GACCGGATCG CGATCCGGAT TGTGGACGAT GGCGAGGGCT ATCCGCCCCG GATCATCGGC 1900
339 A R S T V W I D G E W T G D R I A I R I V D D G E G Y P P A I I G 371
CGCATCGGCG ATCCCTTCGT GCGGCAGCGC CGGCGCCGAGG AAAGCCAGTC GCGCCGGCCG GGCTATGAGG GCATGGGGCC CGGGCTCTTC ATCGCCAAGA 2000
372 R I G D P F V R Q R R A E E S Q S R R P G Y E G M G L G L F I A K T 405
CGCTTCTGGA GCGGTCCGGT GCCGAACCTGA GCTTTGCCAA TGCCCGCGAT CCCTTCCCTGC GGAGCCACCGA GCGCCCCGAA CGCTGGGGCG CCATCGTCGA 2100
406 L L E R S G A E L S F A N A A D P F L R S H E R P E R C G A I V E 438
GGTGATCTGG CCGGTGCGACA GGCTGGTGTT GGTCGGCAAC GCGCCCTTGG GCGAGAACGT CCTGATCCAG ACCTGATCCG GCGCCGGGATG CCGCGTCCGA 2200
439 V I W P V D R L V V V R N A P L G E N V L I Q T STOP 462
CCTTCCCCGG GGCGGGCCGCC CCGGCATTCG GGCTCCATTA A 2241

FIG. 3. Nucleotide sequence of *prrB* and *prrC*. Putative coding sequences are in boldface. Arrows throughout indicate the direction of transcription of the *prrB*, *prrC*, and *prrA* genes. The two arrows under nucleotides 2206 to 2211 and 2224 to 2219 represent the stem of a potential rho-independent transcription terminator, which is followed by a region relatively rich in A and T. The protein sequences are indicated below. Sequences underlined between 133 and 138 correspond to a potential Shine-Dalgarno sequence. The three nucleotides in boldface in the intergenic region between *prrB* and *prrC* represent a potential translation start GTG codon for *prrC*, in frame with the gene. Amino acid sequences underlined represent putative membrane-spanning domains. Nucleotide sequences double underlined represent potential Shine-Dalgarno sequences for *prrB* and *prrC*. Amino acids are numbered in the translational direction of the genes, and nucleotide sequences are numbered in boldface.

anaerobic, dark, DMSO conditions. As anticipated, *prrA* in trans restores expression of the light-harvesting complexes to all of the mutants tested. However, effects of the original mutations in *prrB* and *prrC* are still apparent. Similarly *prrB78*, although highly effective in promoting spectral complex formation in the wild type and in the PRRB mutants, has no effect when the *prrA* mutation is present and is only partially effective when the *prrC* mutations are present. Whereas the wild-type *prrB* allele can restore the PRRB mutants to near control levels for the B875 spectral complex and approximately 75% for the B800-850 complex, its presence in trans in the PRRC mutants reproducibly reduces spectral complex formation compared with the mutants alone. Again it is obvious that the *prrC* gene product is somehow involved, either directly or indirectly in this or a convergent regulatory pathway.*prrC* in trans, like *prrB*, appears to diminish the levels of spectral complexes that are otherwise observed.

Under semiaerobic conditions (Table 2, experiment B), *prrB78* is more effective than the wild-type *prrB* in restoring both spectral complexes. However, the effect of *prrB78* in trans

under anaerobic conditions in the PRRB1 mutant background is quite different than when these cells are grown semiaerobically. It appears that the *prrB1* allele is partially dominant to *prrB78*, whereas the wild-type *prrB* allele is recessive under conditions of low oxygen. Under anaerobic conditions, both alleles are recessive to *prrB78*. The effect of *prrC* is again subtle, actually diminishing the levels of complexes otherwise observed as well as being insensitive to the *prrB78* allele. Therefore, as judged by both photosynthetic growth and spectral complex formation, *prrB*, like *prrA* (17), is involved in photosynthesis gene expression. On the other hand, *prrC* appears to be epistatic to this process, although its involvement can be observed.

The phenotypic effect of *prrB78*. We have already shown that in mutant T<sub>la</sub> (28), the *prrB* gene contains a T-to-C change in codon 78, resulting in a Leu-to-Pro alteration. Further, when this *prrB* allele is placed in trans in wild-type and mutant strains, its effect on spectral complex formation was readily documented. Therefore, it was essential that we reconstruct the *prrB78* allele in single copy in place of the normal *prrB*

VOL. 177, 1995                                                                 OXYGEN REGULATION OF GENE EXPRESSION IN R. SPHAEROIDES                          2701

Region 1

| EcoEnvZ 225 | AGVKQLADDRITLLMAGVSSEERTEPTRE ... RLATEMMSEQDGYLAESINKDIIECNATIEQFIDYL ... RTGQ.EMPMEMADLNAVLGEVIAA.ES | 313 |
|-------------|-------------------------------------------------------------------------------------------------------------------------------|-----|
| RmelFixL 267 | LARLARLNEMGEMASTLAHEDNGELSAEANYSHGCTRLLLRDMDDAVATRIREALBEVASQSILRAGQIIKHLREFVTKGETEKAPEDIRKLVEESAALALV | 366 |
| RcapDctS 422 | LARTGRLVTLGEMASTLAHEDNGELSAEASYAAGGLNLF.DQPEPNLTMRLRQAEKMGAQARRAGLVIRRVQDFVKKRTPQLAALDLSEVLABELSITAP | 520 |
| RsphPrrB 203 | LDREQKLTDLGGVVA AAAHELCTRELAAKLVSSEL ... AEELSEQPALRDDAELIREQADRCRDILRSM.GRAGKDDLQMRQAPLGEVLRRAAEPHVG | 296 |

Region 2

| EcoEnvZ 314 | GYEREIETA..LYPGSIEVKM..HBLSIKRAVZMNWNSAR ... YGNGWIKVSSGTEPNRAWFQVR ... G | 393 |
|-------------|-------------------------------------------------------------------------------------------------------------------------------|-----|
| RmelFixL 367 | GSREQGVRTVFEYL.PGAEMVLVDRICQQVLEINMRQTEAMRHV..DR..RELTIRTMPADPGEVAVVVV ... TT | 454 |
| RcapDctS 520 | VAREHRVKLASLIE.GRIPGVQADRILJEQVIMMEJRNVEAMAEGPRTG..DDLTIVRLARAGAA.VTIEVR ... ST | 609 |
| RsphPrrB 296 | RGKRVEFDLYPSRGDERQPVILRPEVTHGLRMLZCRVD ... FARSTVWIDGEWTGDRIAIRKAIIGVRORRAEESQ | 387 |

Spacer R. 3.1 Spacer R. 3.3

| EcoEnvZ 394 | DSARTIQRIVDNHNGMLELGTSERGGLSIR ... AWLPVPVTRAQGTTKEG | 450 |
|-------------|-------------------------------------------------------------------------------------------------------------------------------|-----|
| RmelFixL 455 | KASKRIVEAHGGEMTVSKNEAG ... GATFRFTLPAYLDERIVAND | 505 |
| RcapDctS 610 | KSRSIVEMEHGSLSHGPRAGG ... GTVFTVTLPVPQEGAPA | 657 |
| RsphPrrB 388 | SRRPGYAKTLLERSGAELSFANAADPFLRSHERPERCGAIVEIWPDRLVVVRNAPLGENVLIQT | 462 |

FIG. 4. Alignment of PrrB with other sensor histidine kinases. The putative amino acid sequence encoded by the *prrB* gene was aligned with the sequences of other histidine kinases. Only the carboxy-terminal ends of the proteins, which contain the conserved kinase domains, were similar to PrrB. Thus, only these regions are shown in the alignment. Amino acids which are conserved in all cases are in boldface and boxed. Shaded boxed areas denote regions conserved among the kinases (35). The sequences are the following: RsphPrrB, PrrB (*R. sphaeroides*); EcoEnvZ, EnvZ (*E. coli*) (22); RmelFixL, FixL (*Rhizobium meliloti*) (30); RcapDctS, DctS (*R. capsulatus*) (20). Dots are spaced every 10 residues. Amino acid numbers are indicated for each protein. Spaces throughout were left by the alignment program.

allele in an otherwise wild-type background to determine its precise role in photosynthesis gene expression. The construction of mutant strain PRRB78 (Table 1) is described in Materials and Methods. This strain contains an Ω Sm/Sp cartridge at the *SmaI* site immediately downstream from *prrB* (Fig. 1), which overlaps with the putative transcription terminator for *prrB*. The isogenic wild-type strain 2.4.1Ω was also constructed as the basis for comparison with PRRB78.

Because PRRB78 produces ICM under highly aerobic conditions, it is very unstable, segregating PS⁻ derivatives, similar to what was originally observed with mutant T₁ₐ (28). Therefore, the introduction of the *puc::aph* fusion in pLK-1 (Table 1) in the presence of 450 μg of kanamycin per ml into PRRB78 provides for stabilization of the mutant strain.

When PRRB78(pLK-1) cells were grown chemoheterotrophically to low cell densities in the presence of 30% oxygen, approximately 1.4 nmol of B800-850 and 11.5 nmol of B875 per mg of protein were found. Both wild-type and 2.4.1Ω cells showed no detectable spectral complexes under identical conditions (Fig. 6A). Additionally, when these cells were grown anaerobically in the dark with DMSO, 55 and 21 nmol per mg of protein of B800-850 and B875, respectively, were present, whereas the corresponding value for 2.4.1 and 2.4.1Ω were 29 and 9 nmol/mg of protein and 31 and 11 nmol/mg of protein, respectively. Thus, PRRB78(pLK-1) cells produce spectral complexes aerobically, although the level of B800-850 is considerably lower than the level of B875. However, under anaerobic conditions, the levels of both B800-850 spectral complex and B875 increase, in both cases approaching levels of these complexes found in low light grown cells.

The photopigments were extracted from PRRB78(pLK-1) and 2.4.1Ω and quantitated as described in Materials and Methods (Fig. 6B). There was over 100-fold more Bchl produced in PRRB78(pLK-1) cells than in 2.4.1Ω cells, in which the level was practically nonexistent (10). This difference is similar to previously observed values when aerobic and anaer-

obically grown cells are compared (21) and clearly reveals that the synthesis and/or accumulation of Bchl is also under the control of the *prrB* gene.

Measurement of cellular carotenoid levels revealed a low basal level in 2.4.1Ω, also consistent with previous data (10), and an approximately 19-fold increase in PRRB78(pLK-1). Interestingly, no spheroidene (yellow) appeared to be present in these cells. High-pressure liquid chromatography HPLC analysis of the cellular carotenoids derived from aerobically grown PRRB78(pLK-1) confirmed this result.

Therefore, PRRB78(pLK-1) produces pigments and pigment-protein complexes in the presence of high oxygen, additionally supporting the hypothesis that the histidine kinase sensor containing the Leu-78→Pro alteration in these cells is defective in transducing the oxygen signal, resulting in the synthesis of the ICM in the presence of oxygen.

Presence of photosynthetic RCs in PRRB78(pLK-1) under aerobic conditions of growth. Previous data have shown that the presence of Bchl is essential for the stability of the photosynthetic RCs in *R. sphaeroides* and *R. capsulatus* (10, 48, 50). In the absence of Bchl, the apoproteins quickly turn over. Additionally, in the original T₁ₐ mutant strain containing the *prrB*78 mutation, it was shown that the rate of synthesis of both RC-M and RC-H polypeptides was similar to that observed in cells growing photosynthetically, despite the presence of high oxygen (50). It was further shown that a substantial fraction of these apoproteins remained stable and the remainder turned over. It was reasoned that the availability of Bchl was responsible for stabilizing these apoproteins by forming functional RCs. Therefore, we decided to investigate whether PRRB78 contains assembled photochemical RC Bchl dimers in the presence of oxygen, as well as B800-850 and B875 polypeptides.

As described in Materials and Methods and previously (5), the spectrum of the RC change was obtained by flash photolysis of cellular extracts of PRRB78 and 2.4.1Ω at high redox potential (Fig. 7). Whereas 2.4.1Ω did not show any absor-

2702 ERASO AND KAPLAN

Putative trans-membrane domain

Sco2 1 MLNSSRKYACRSLFRQANVSIKGLFYNGGAY..RRGFSTGCCILRSNDKESP SARQPLDRLQLGDEINEPEPIRTRFFQFSRWKATIALLLLSGGTYAYLS 98

Sco1 1 MLKLSRS...ANLRLVQLPAARLSGNGAKLLTQRGFTVTTRLWQSNKKPLSRVPVG...GTPIKDNGKVREGSIEFSTGKAIALFLAVGGALSYFFN 92

PrrC 1 MTKLYAGVAAAAIAALLAGSAAWYFLGR 28

Msp5 1 MRIFKIVSNLLLFAAVFLGYSYVN 25

Imm 1 MEHIMKAIKFILNLCLLEAAIFLGYSYIT 29

Orf193 1 MPVLNTLRRSLLILALACTLPSLAM 25

helix turn turn turn strand turn helix turn strand turn helix

Sco2 99 RKRRRETEKEADANRAYGSVALGGRF.NLTBEENKPFTENEKKCKRSSSECFSHGPDIP6PEELDRWTYWISEL66KDHIKIQPERKTS6DBA3ETPDVTE 197

Sco1 93 REKRRLLETQKEABANRGYCKPSLGGRF.HLEDYGNEFTEKNEXGXKSSXKNGFSNCBDICPDELDKLGLWLNTLSSKYGIITLQPEBTCDPARESPAVAL 191

PrrC 29 SEERFAGC....BANQVAGGAIGCPF.TLVDOECRTVTDREVLAKPSTIRFGYTHCPMECPDMARNAQAVDIL.TEWGIENVTVASIDBKRDPEQL 121

Msp5 26 KKGIFS KIGERFTTSEVVSEGIASA SFNNLVNHEGVTVSSGDFGGBKMNARGFSACKY46PTELGMASQLLSKLGDHAD.KLQMV3HTVBPRNDTVAKL 124

Imm 30 KQGIFQPKLHDSPDVNISNKADINTSF.SLINQDGTTISSKDFEGKHMFV13FSQCKL16PMELGLASTILDQLGNESQ.KLQSYSBTIDBPXKETVETL 127

Orf193 26 DRNALLDQANVILLPRERAITQ.....LELVDDNDQPES TETLKGR88116SFGFTAGPDI2PTTLSEMRRRLFGLPAETREQLQ5X13TABEABRFPOQL 120

helix turn helix strand turn strand helix helix

Sco2 198 KEYLSDFHPAII66BEGIYDQVKSVCKKKKSKSSTPRDVKPNQJXYMHSTEEYX3 DPEGQFIDALGRNYDEQSG.LEKIREQTQAYVPKEERERRSKKWY 296

Sco1 192 KEYLSDFHPSILGLTGTFDEVKNACKKXRSXEPSSTPPNVKPGQEXMVRHRYLMNDPEGQFVDALGRNYDEKTG.VDKIVEHVKS YVPAEQRANKKEAWY 290

PrrC 122 KFFAEAIHPDTIALTGETAQVKAASQAYKTFYR..VQESDDDXXKXCHSTETYEMLPGTGEVDFFKRE....DT.PEQUIERISC FANDSHVSTSFARA 214

Msp5 125 KEYHKSF DARIQM LIGBEADIKSVVENYRVVGD..KKPSEGD...GHMKRMLNGKG RYVGHFAPDFNASEGQGEELFKFVSGHMLNS........... 210

Imm 128 KEFHKNFDSRIQMLTGNIEAINQIVQGYKWWQ...QFENDN.OINHSK21M33DKKGEYLTHEVPDLKSKEPQVDKLLSLIKQYL.............. 209

Orf193 121 KTYLSYYRAGFIGLTGIMEQLQRLSKALGEP .VPATEISED.XS2SMSNJALSGPDGSLRGHI.RAPLKLDG.LQ................. 193

Sco2 297 SFIFN............ 301

Sco1 291 SFLFK............ 295

PrrC 215 QKSYQASRGKQMGDNHG 231

Msp5

Imm

Orf193

FIG. 5. Alignment of prrC. Computer searches described in Materials and Methods found proteins similar to PrrC in the data bases, and the putative amino acid sequence encoded by the prrC gene was aligned with their sequences. Amino acids which are conserved in all cases are in boldface and boxed. Shaded boxed areas denote regions with similar secondary structure. These are indicated as helix, turn, and strand. Amino acids in boldface and underlined denote putative transmembrane domains. Secondary structure prediction of the region in Sco1 and Sco2 prior to the transmembrane domain was not considered relevant for this figure. The sequences are the following: Sco2 (S. cerevisiae) (46), Sco1 (S. cerevisiae) (39), PrrC (R. sphaeroides), Msp5 (Anaplasma marginale) (51), Imm (Cowdria ruminantium) (31), and Orf193 (Pseudomonas stutzeri) (11). Dots were spaced every 10 residues. Amino acid numbers are indicated for each protein. Spaces throughout were left by the alignment program.

balance changes over the range monitored, consistent with the absence of RCs in these cells under aerobic conditions, PRRB78(pLK-1) showed a profile characteristic of the RC (5). The isosbestic point was at 584 nm, and the ratio of the absorption change at 605 and 540 (∆ε<sub>redox</sub><sup>605</sup>/∆ε<sub>redox</sub><sup>540</sup>) was -2.3. This ratio is in accord with previously published values (5) for functional RCs, and the magnitude of the change is consistent with high cellular levels of RCs in amounts comparable to those determined by Varga and Kaplan (50).

Analysis of mRNA levels in 2.4.1, 2.4.1Ω, and PRRB78 cells. From the data presented so far, it is evident that the prrB gene is involved in the ability of R. sphaeroides to respond to the removal of oxygen through the formation of all three spectral complexes associated with photosynthetically growing cells. In the case of the prrB78 allele, the PrrB78 gene product behaves as if there is either no or low oxygen in the environment. Therefore, the next logical question is to determine at what level of gene regulation PrrB is effective.

Slot blot hybridization values are presented in Table 3 for puf, puhA, and cycA. In the case of puc, the presence of the puc::aph fusion in pLK-1 complicated the results obtained for slot blot hybridization; therefore Northern hybridization values are presented. Since we have shown previously (17) that minimal levels of these mRNAs are produced in a prrA background under aerobic conditions of growth, values for each message obtained in PRRA2 (prrA mutation) were subtracted from the values obtained for each of the strains shown in Table 3. Because only the large transcript of cycA is affected by PrrA (17), this was not done. Values were normalized to those for the wild-type cells grown anaerobically in the dark in the presence of DMSO. The results are shown in Table 3.

What is immediately obvious from the data presented in Table 3 is that the mutation from Leu to Pro at position 78 in PrrB78 results in high transcript levels in the presence of oxygen of all of the genes encoding important structural elements of the photosynthetic apparatus. These transcript levels are normally observed in cells growing photosynthetically at moderate to low light intensity (13, 14, 24, 26). Thus, these measurements are compatible with observations made in T<sub>la</sub> (28) with respect to light-harvesting complex formation, or RC assembly (50), as well as those described above. What is of equal significance is that even in cells grown in the absence of oxygen with DMSO in the dark, increased transcription of puf, puhA, and puc is obvious.

TABLE 2. Spectral complex levels in PRR mutants$^a$

| Expt | Strain     | pRK415   | $prrC^{+}$    | $prrB^{+}$    | $prrA^{+}$    | $prrB78$      |
|------|------------|----------|---------------|---------------|---------------|---------------|
|      |            | B800-850 | B875          | B800-850      | B875          | B800-850      | B875          |
| A    | 2.4.1      | 43.4     | 14.8          | 32.0          | 11.6          | 33.7          | 12.6          | 44.4          | 11.2          | 61.3          | 20.7          |
|      | PRRC1      | 8.2      | 7.9           | 4.1           | 6.0           | 1.7           | 5.6           | 25.6          | 8.0           | 30.3          | 11.4          |
|      | PRRC2      | 13.1     | 7.6           | 11.7          | 6.9           | 7.7           | 6.6           | 32.7          | 8.9           | 36.0          | 14.2          |
|      | PRRCA1     | ND       | 0.3           | ND            | 1.4           | 0.1           | 2.4           | 33.8          | 10.0          | ND            | 1.8           |
|      | PRRB1      | 0.4      | 5.9           | ND            | 5.4           | 29.9          | 11.0          | 28.3          | 11.9          | 50.1          | 20.8          |
|      | PRRB2      | 2.9      | 5.5           | —             | —             | 29.9          | 11.5          | 31.3          | —             | —             | —             |
|      | PRRBC1     | ND       | 1.1           | —             | —             | —             | —             | —             | —             | —             | —             |
|      | PRRBCA1    | ND       | 1.2           | —             | —             | —             | —             | —             | —             | —             | —             |
| B    | 2.4.1      | 1.9      | 11.4          | 1.7           | 4.8           | 1.0           | 4.1           | —             | —             | 71.5          | 22.3          |
|      | PRRC1      | 0.2      | 2.4           | 0.03          | 0.8           | 0.05          | 1.7           | —             | —             | 1.0           | 4.0           |
|      | PRRC2      | 0.8      | 3.3           | 0.4           | 2.3           | 0.7           | 3.8           | —             | —             | 4.7           | 6.0           |
|      | PRRCA1     | ND       | ND            | ND            | ND            | ND            | ND            | —             | —             | ND            | ND            |
|      | PRRB1      | ND       | 0.8           | ND            | 1.5           | 0.9           | 4.2           | —             | —             | 7.2           | 8.5           |

$a$ Cells were grown anaerobically in the dark in DMSO (experiment A) or chemoheterotrophically, sparged with a mixture of 97% N₂, 2 to 3% O₂, and 1% CO₂ (experiment B). The amounts of light-harvesting complexes were determined as described previously (25) and expressed as nanomoles of spectral complex per milligram of crude membrane protein. ND, none detected; —, not done.

DISCUSSION

Cloning of a 10.4-kb fragment from chromosome I of the R. sphaeroides PRRA1 mutant (17) identified two loci, $prrB$ and $prrC$. The former is clearly involved in the positive regulation of photosynthesis gene expression in response to the removal of oxygen. $prrC$, on the other hand, appears to be involved, but its role is more subtle (see below). Sequence comparisons identified these genes as encoding a potential sensor histidine kinase of a two-component regulatory system ($prrB$), involved in prokaryotic signal transduction (1, 4, 35, 47), and in the case of $prrC$, a gene having strong similarity to a family of genes containing one putative membrane-spanning domain, among which were the $sco1$ and $sco2$ genes from S. cerevisiae. These genes are believed to play a role in cytochrome c oxidase subunit assembly in the mitochondria of this organism (39). The amino terminus of PrrB, which may comprise the sensing domain of the histidine kinase, was not found to be similar to any protein in GenBank except RegB, a histidine kinase from R. capsulatus whose sequence has been published recently (34). This domain contains six putative membrane-spanning regions, making PrrB most likely a protein anchored in the membrane at its amino-terminal half and with the kinase domain in the cytoplasm. Similarly, PrrC appears to be anchored in the membrane, but unlike PrrB, the carboxy-terminal end of the protein appears to be located in the periplasmic space, as concluded from recent data for PhoA fusions to PrrC (16).

Although the PrrB/PrrA regulatory system appears to be analogous to the RegB/RegA system of R. capsulatus in that both appear to be involved in regulation of the aerobic-to-anaerobic transition with respect to photosynthesis gene expression, there are important differences. The PrrB/PrrA system appears to lead to a more stringent regulation of those operons encoding the major structural apoproteins than does the RegB/RegA system. Both Bchl and Crt synthesis and/or accumulation are controlled by PrrB/PrrA. This may not be true for RegB/RegA. Finally, the structure of the DNA region linked to $prrB/prrA$ appears to be different from that linked to $regB/regA$ (data not shown).

Cloning of the PRR region from the oxygen-insensitive regulatory mutant Tla identified a Leu-to-Pro change at position 78 of PrrB (see Fig. 3 for coordinates). This mutation disrupts a predicted hydrophilic helix located between two putative hydrophobic membrane-spanning domains. Mutant PRRB78, which contains only this regulatory mutation, has the ability to synthesize both light-harvesting and RC components of a functional photosynthetic apparatus under highly aerobic conditions of growth. Additionally, the amounts of RNA specific for $puhA$ and $cycA$, although higher in PRRB78 than in the wild type under aerobic and anaerobic conditions of growth, were not as high as the increased levels observed for $puf$ and $puc$. This finding is consistent with previous data found for the regulation of these messages by $prrA$ (17), and it indicates that $puf$, $puc$, $puhA$, and $cycA$ are regulated by PRRB78 at the level of transcription. Together, the data clearly indicate that the sensor histidine kinase encoded by $prrB$ in R. sphaeroides is a global regulator of photosynthesis gene expression in response to the removal of oxygen. The fact that substantial levels of Crt and Bchl are produced aerobically in PRRB78 indicates that in R. sphaeroides, the accumulation of these pigments and spectral complexes is under the direct control of $prrB$. In R. capsulatus, using $bchC::lacZ$ and $bchH::lacZ$ translational fusions (34, 40), the role of $regA$ and $regB$ on pigment synthesis was not as apparent as observed here for R. sphaeroides.

PRRB78 grown in a high oxygen concentration had no appreciable levels of spheroidene compared with spheroidenone, which is consistent with previous observations from semiaerobically grown cells (49). Spheroidene is present in higher amounts than spheroidenone in B800-850 spectral complexes found in cells grown semiaerobically (42). Although more information is required, it may be reasonable to hypothesize that the B800-850 spectral complex is not assembled in aerobically grown PRRB78 because of a lack of spheroidene, despite the presence of both apoproteins and Bchl.

Several lines of evidence indicate that PrrB and PrrA are in the same signal-transducing pathway. The $prrB78$ mutation in trans in PRRCA1 (Table 2) shows clearly that the presence of an intact $prrA$ gene is absolutely necessary for the PrrB78 protein to have any effect on photosynthesis gene expression. Similarly, a $prrA$ mutation in the presence of the $prrB78$ allele results in a defect in photosynthesis gene expression. Likewise, expression of PrrA from a multicopy plasmid in the $prrB$ strains results in complementation of the $prrB$ phenotype, indicating that PrrA acts downstream from PrrB. Inactivation of either $prrB$ or $prrA$ leads to the same PS⁻ phenotype. Finally, whereas

2704 ERASO AND KAPLAN

A.

![Graph A](#)

B.

![Graph B](#)

FIG. 6. Spectral complex formation and organic extraction of pigments from PRRB78 and 2.4.1Ω grown aerobically. Cells were grown sparged with a mixture 30% O₂, 69% N₂, and 1% CO₂ in Sistrom's medium. (A) Cells were harvested at a cell density of approximately 20 Klett units and sonicated, and cell-free lysates were prepared for spectral analysis. All samples contained 1,570 μg of protein per ml. PRRB78(pLK-1) was grown in the presence of 450 μg of kanamycin and 50 μg of streptomycin-spectinomycin per ml. 2.4.1Ω was grown on 50 μg of streptomycin-spectinomycin per ml. 2.4.1 is the *R. sphaeroides* wild-type strain. The bar represents an absorbance (Abs) value of 0.08. (B) Cells were grown as for panel A to a density of approximately 25 Klett units. Optical densities were recorded, and approximately 9.6 × 10⁶ cells were pelleted, and the pellets were extracted with an acetone-methanol mixture in the dark as described previously (10). Spectral analysis was performed on the supernatants. The bar represents an absorbance value of 0.146.

The presence of *prrA* in multicopy in aerobic cells is responsible for the appearance of spectral complexes (17). PRRB78 cells also show complexes aerobically, and like *prrA* in trans, it is also the B875 complex which is preferentially made.

On the basis of data obtained with PRRB78, we suggest that the α-helical domain which contains Leu-78 is either (i) directly involved in the sensing of oxygen or of the redox state of some membrane component(s) and the positive regulation of the kinase activity, (ii) involved in the transfer of the signal to yet another domain of the PrrB protein, or (iii) a part of a phosphatase domain which is regulated by oxygen and normally rendered more active aerobically. The fact that the presence of PrrB in trans leads to a reduction in the amount of spectral complexes in wild-type cells as well as, more markedly, in PRRC mutant cells suggests that there is a phosphatase activity associated with PrrB. It is noteworthy that under anaerobic, dark, DMSO conditions of growth, the negative effect of *prrB* in trans in the wild type seems to be marginal, whereas under low-oxygen conditions, a greater negative effect is

FIG. 7. Presence of functional RCs in PRRB78 grown aerobically. Cells were grown as described in the legend to Fig. 6 in 1-liter bottles. The cells were harvested at a cell density of approximately 25 Klett units, resuspended in 1× ICM buffer (25), and broken in a French press. The protein concentration was approximately 5 mg/ml. Cell-free lysates were prepared for spectral analysis as described previously (5) and in Materials and Methods. Open circles represent 2.4.1Ω; closed circles represent PRRB78(pLK-1). The values at 530 nm might have an inherent error, since the wavelength of the flash light used in the experiment was 532 nm.

observed in both B800-850 and B875 spectral complexes, indicating that the negative effect of *prrB* in trans is more marked when oxygen is present. In contrast, the *prrB78* allele in trans in the wild type shows an increase in both spectral complexes anaerobically and a pronounced increase in the B800-850 and B875 semiaerobically. Thus, in the presence of oxygen, PrrB78 does not have an inhibitory effect on spectral complex formation. Further, the ability of the truncated protein (PRRB1) to be influenced by oxygen is also apparent; therefore, if a sensing mechanism exists, it must at least partially reside in the first 63 amino acids. The analysis of suppressors of the PRRB78 mutant will no doubt help in determining mechanistically how PrrB functions.

The role of *prrC* is less clear, although the available evidence indicates that it is either part of this sensory transducing pathway or convergent upon the pathway. We know that *prrA* has its own promoter located in the terminal portion of the *prrC* gene (17), and thus interruption of *prrC*, even if it forms an operon with *prrA*, cannot be completely polar. This is readily apparent in mutant PRRCAl containing *prrA* in trans.

TABLE 3. Analysis of photosynthesis gene-specific mRNAs

| Strain | $puf$ | $puhA$ | $cycA$ | $puc$ (small transcript) |
|--------|-------|--------|--------|--------------------------|
|        | +O₂   | -O₂    | +O₂    | -O₂                     | +O₂    | -O₂    | +O₂    | -O₂                     |
| 2.4.1  | 4     | 100    | ND     | 100                     | 58     | 100    | 44     | 100                      |
| 2.4.1Ω | 30    | 126    | ND     | 105                     | 81     | 89     | 68     | 156                      |
| PRRB78 | 246   | 679    | 94     | 242                     | 85     | 116    | 1,482  | 1,311                    |

$a$ Values were normalized to 100 for wild-type cells grown anaerobically as described below. Approximately 120 ng of RNA was loaded per slot ($puf$, $puhA$, and $cycA$), and 4 μg of RNA was loaded per lane ($puc$). Relative RNA concentrations were normalized by using rRNA probes as described previously (17). -O₂, anaerobic growth in the dark in the presence of DMSO. +O₂, aerobic growth, sparged with 30% O₂–69% N₂–1% CO₂. ND, none detected.

ever, whenever *prrB*, *prrB78*, or *prrA* is placed in trans in PRRC mutants, complete restoration of spectral complexes is not observed; i.e., the effect of the *prrC* lesion is apparent. This is particularly true in the case of mutant PRRBC1, which is severely affected in spectral complex abundance; in this case, a mutated *prrC* allele appears to behave synergistically to a mutated *prrB* allele.

Thus, our current model for the derepression of photosynthesis gene expression in response to either a lowering of oxygen tension or the introduction of anaerobiosis indicates that the PrrB protein senses such changes within the amino-terminal region either directly or indirectly, and the PrrC protein either stabilizes this change or directs yet another signal to this region. The alteration in PrrB is then manifest by either an increase in histidine kinase activity or a decrease in phosphatase activity or any combination of the two, leading to the increase steady-state phosphorylated state of PrrA. Additionally, at least one (19) and possibly other (52) histidine kinases are capable of phosphorylating PrrA, which can normally be rendered noneffective in the presence of the PrrB-mediated phosphatase activity when oxygen is present.

### ACKNOWLEDGMENTS

We thank Limin Gong for assistance with Northern hybridizations and John Spudich for assistance with flash photolysis and for helpful discussions. We also thank Alexei Yeliseev for assistance with the HPLC.

This work was supported by Public Health Service grant GM15590 to S. K. from the National Institutes of Health.

### REFERENCES

1. Albright, L. M., E. Huala, and F. M. Ausubel. 1989. Prokaryotic signal transduction mediated by sensor and regulator protein pairs. Annu. Rev. Genet. **23**:311–336.
2. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. D. Seidman, J. A. Smith, and K. Struhl (ed.). 1987. Current protocols in molecular biology. Greene Publishing Associates and Wiley Interscience, New York.
3. Baltscheffsky, M. 1978. Photosynthetic phosphorylation, p. 595–614. In R. K. Clayton and W. R. Sistrom (ed.), The photosynthetic bacteria. Plenum Publishing Corp., New York.
4. Bourret, R. B., K. A. Borkovich, and M. I. Simon. 1991. Signal transduction pathways involving protein phosphorylation in prokaryotes. Annu. Rev. Biochem. **60**:401–441.
5. Bowyer, N. R., S. W. Meinhardt, G. V. Tierney, and T. R. Crofts. 1981. Resolved difference spectra of redox centers involved in photosynthetic electron flow in *Rhodopseudomonas capsulata* and *Rhodopseudomonas sphaeroides*. Biochim. Biophys. Acta **635**:167–186.
6. Buchwald, P., G. Krummeck, and G. Rodel. 1991. Immunological identification of yeast SCO1 protein as a component of the inner mitochondrial membrane. Mol. Gen. Genet. **229**:413–420.
7. Chory, J., and S. Kaplan. 1983. Light-dependent regulation of the synthesis of soluble and intracytoplasmic membrane proteins of *Rhodopseudomonas sphaeroides*. J. Bacteriol. **153**:465–474.
8. Clark, J. M., and R. L. Switzer. 1977. Experimental biochemistry, p. 207–218. W. H. Freeman & Co., San Francisco.
9. Cogdell, R. J., and J. P. Thornber. 1980. Light-harvesting pigment-protein complexes of purple photosynthetic bacteria. FEBS Lett. **122**:1–8.
10. Cohen-Bazire, G., W. R. Sistrom, and R. Y. Stanier. 1957. Kinetic studies of pigment synthesis by non-sulfur purple bacteria. J. Cell. Comp. Physiol. **49**:25–68.
11. Cuypers, H., and W. G. Zumft. 1993. Anaerobic control of denitrification in *Pseudomonas stutzeri* escapes mutagenesis of an *fnr*-like gene. J. Bacteriol. **175**:7236–7246.
12. Davis, J., T. J. Donohue, and S. Kaplan. 1988. Construction, characterization, and complementation of a *puf* mutant of *Rhodobacter sphaeroides*. J. Bacteriol. **170**:320–329.
13. Donohue, T. J., J. H. Hoger, and S. Kaplan. 1986. Cloning and expression of the *Rhodobacter sphaeroides* reaction center H gene. J. Bacteriol. **168**:953–961.
14. Donohue, T. J., A. G. McEwan, and S. Kaplan. 1986. Cloning, DNA sequence, and expression of the *Rhodobacter sphaeroides* cytochrome *c*~2~ gene. J. Bacteriol. **168**:962–972.
15. Dryden, S. C., and S. Kaplan. 1990. Localization and structural analysis of the ribosomal RNA operons of *Rhodobacter sphaeroides*. Nucleic Acids Res. **18**:7267–7277.
16. Eraso, J., and S. Kaplan. 1994. Unpublished data.
17. Eraso, J. M., and S. Kaplan. 1994. *prrA*, a putative response regulator involved in oxygen regulation of photosynthesis gene expression in *Rhodobacter sphaeroides*. J. Bacteriol. **176**:32–43.
18. Farham, P. J., and T. Platt. 1980. A model for transcription termination suggested by studies on the *trp* attenuator in vitro using base analogs. Cell **20**:739–748.
19. Gomelsky, M., and S. Kaplan. Isolation of regulatory mutants in photosynthesis gene expression in *Rhodobacter sphaeroides* 2.4.1 and partial complementation of a PrrB mutant by HupT histidine kinase. Submitted for publication.
20. Hamblin, M. J., J. G. Shaw, and D. J. Kelly. 1993. Sequence analysis and interposon mutagenesis of a sensor-kinase (DctS) and response-regulator (DctR) controlling synthesis of the high affinity C4-dicarboxylate transport system in *Rhodobacter capsulatus*. Mol. Gen. Genet. **237**:215–224.
21. Hunter, C. N., and S. A. Coomber. 1988. Cloning and oxygen-regulated expression of the bacteriochlorophyll biosynthesis genes *bchE, B, A*, and *C* of *Rhodobacter sphaeroides*. J. Gen. Microbiol. **134**:1491–1497.
22. Igo, M. M., A. J. Ninfa, J. B. Stock, and T. J. Silhavy. 1989. Phosphorylation and dephosphorylation of a bacterial transcriptional activator by a transmembrane receptor. Genes Dev. **3**:1725–1734.
23. Keen, N. T., S. Tamaki, D. Kobayashi, and D. Trollinger. 1988. Improved broad-host-range plasmids for DNA cloning in Gram-negative bacteria. Gene **70**:191–197.
24. Kiley, P. J., and S. Kaplan. 1987. Cloning, DNA sequence, and expression of the *Rhodobacter sphaeroides* light-harvesting B800-850-α and B800-850-β genes. J. Bacteriol. **169**:3268–3275.
25. Kiley, P. J., and S. Kaplan. 1988. Molecular genetics of photosynthetic membrane biosynthesis in *Rhodobacter sphaeroides*. Microbiol. Rev. **52**:50–69.
26. Lee, J. K., B. S. DeHoff, T. J. Donohue, R. I. Gumpert, and S. Kaplan. 1989. Transcriptional analysis of *puf* operon expression in *Rhodobacter sphaeroides* 2.4.1 and an intercistronic transcription terminator mutant. J. Biol. Chem. **264**:19354–19365.
26a. Lee, J. K., and S. Kaplan. Unpublished data.
27. Lee, J. K., and S. Kaplan. 1992. *cis*-acting regulatory elements involved in oxygen and light control of *puc* operon transcription in *Rhodobacter sphaeroides*. J. Bacteriol. **174**:1146–1157.
28. Lee, J. K., and S. Kaplan. 1992. Isolation and characterization of *trans*-acting mutations involved in oxygen regulation of *puc* operon transcription in *Rhodobacter sphaeroides*. J. Bacteriol. **174**:1158–1171.
29. Lee, J. K., P. J. Kiley, and S. Kaplan. 1989. Posttranscriptional control of *puc* operon expression of B800-850 light-harvesting complex formation in *Rhodobacter sphaeroides*. J. Bacteriol. **171**:3391–3405.
30. Lois, A. F., G. S. Ditta, and D. R. Helinski. 1993. The oxygen sensor FixL of *Rhizobium meliloti* is a membrane protein containing four possible transmembrane segments. J. Bacteriol. **175**:1103–1109.
31. Mahan, S. M., T. C. McGuire, F. Jongejan, F. R. Rurangirwa, and A. F. Barbet. 1993. Unpublished data.
32. Maniatis, T., E. F. Fritsch, and J. Sambrook. 1982. Molecular cloning: a laboratory manual. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.
33. Meinhardt, S. W., P. J. Kiley, S. Kaplan, A. R. Crofts, and S. Harayama. 1985. Characterization of light-harvesting mutants of *Rhodopseudomonas sphaeroides*. I. Measurement of the efficiency of energy transfer from light-harvesting complexes to the reaction center. Arch. Biochem. Biophys. **236**:130–139.
34. Mosley, C. S., J. Y. Suzuki, and C. E. Bauer. 1994. Identification and molecular genetic characterization of a sensor kinase responsible for coordinately regulating light harvesting and reaction center gene expression in response to anaerobiosis. J. Bacteriol. **176**:7566–7573.
35. Ninfa, A. J. 1991. Protein phosphorylation and the regulation of cellular processes by the homologous two-component regulatory systems of bacteria, p. 39–66. In J. K. Setlow (ed.), genetic engineering. Plenum Press, New York.
36. Okamura, M. Y., L. A. Steiner, and G. Feher. 1974. Characterization of reaction centers from photosynthetic bacteria. I. Subunit structure of the protein mediating the primary photochemistry in *Rhodopseudomonas sphaeroides* R-26. Biochemistry **13**:1394–1403.
37. Olson, K. D., and J. L. Spudich. 1993. Removal of the transducer protein from sensory rhodopsin I exposes sites of proton release and uptake during the receptor photocycle. Biophys. J. **65**:2578–2585.
38. Phillips-Jones, M. K., and C. N. Hunter. 1994. Cloning and nucleotide sequence of *regA*, a putative response regulator gene of *Rhodobacter sphaeroides*. FEMS Microbiol. Lett. **116**:269–276.
39. Schulze, M., and G. Rodel. 1989. Accumulation of the cytochrome c oxidase subunits I and II in yeast requires a mitochondrial membrane-associated protein, encoded by the nuclear *scol* gene. Mol. Gen. Genet. **216**:37–43.
40. Sganga, M. W., and C. E. Bauer. 1992. Regulatory factors controlling photosynthetic reaction center and light-harvesting gene expression in *Rhodobacter capsulatus*. Cell **68**:945–954.
41. Shine, J., and L. Dalgarno. 1974. The 3′-terminal sequence of *Escherichia coli* 16S ribosomal RNA: complementarity to nonsense triplets and ribosome

binding sites. Proc. Natl. Acad. Sci. USA 71:1342–1346.

2706 ERASO AND KAPLAN

242. Siefermann-Harms, D. 1985. Carotenoids in photosynthesis. I. Location in photosynthetic membranes and light-harvesting function. Biochim. Biophys. Acta 811:325–355.
243. Silhavy, T. J., M. L. Berman, and L. W. Enquist. 1984. Experiments with gene fusions. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.
244. Simon, R., U. Priefer, and A. Pühler. 1983. A broad host range mobilization system for in vivo genetic engineering: transposon mutagenesis in Gram negative bacteria. Bio/Technology 1:37–45.
245. Sistrom, W. R. 1962. The kinetics of the synthesis of photopigments in *Rhodopseudomonas sphaeroides*. J. Gen. Microbiol. 28:607–616.
246. Smits, P. H. M., M. de Haan, C. Maat, and L. A. Grivell. 1994. The complete sequence of a 33-kb fragment on the right arm of chromosome II from *Saccharomyces cerevisiae* reveals 16 open reading frames, including ten new open reading frames, five previously identified genes and a homologue of the *sco1* gene. Yeast 10:S75–S80.
247. Stock, J. B., A. J. Ninfa, and A. M. Stock. 1989. Protein phosphorylation and regulation of adaptive responses in bacteria. Microbiol. Rev. 53:450–490.
248. Takemoto, J., and J. Lascelles. 1973. Coupling between bacteriochlorophyl and membrane protein synthesis in *Rhodopseudomonas sphaeroides*. Proc.

Natl. Acad. Sci. USA 70:799–803.

J. BACTERIOL.

249. van Neil, C. B. 1944. The culture, general physiology, and classification of the non-sulfur purple and brown bacteria. Bacteriol. Rev. 8:1–118.
250. Varga, A., and S. Kaplan. 1993. Synthesis and stability of reaction center polypeptides and implications for reaction center assembly in *Rhodobacter sphaeroides*. J. Biol. Chem. 268:19842–19850.
251. Visser, E. S., T. C. McGuire, G. H. Palmer, W. C. Davis, V. Shkap, E. Pipano, and D. P. Knowles. 1992. The *Anaplasma marginale msp5* gene encodes a 19-kilodalton protein conserved in all recognized *Anaplasma* species. Infect. Immun. 60:5139–5144.
252. Wanner, B. L. 1992. Is cross regulation of two-component response regulator proteins important in bacteria? J. Bacteriol. 174:2053–2058.
253. Yen, H.-C., and B. L. Marrs. 1977. Growth of *Rhodopseudomonas capsulata* under anaerobic dark conditions with dimethyl sulfoxide. Arch. Biochem. Biophys. 181:411–418.
254. Zhu, Y. S., and S. Kaplan. 1985. Effects of light, oxygen, and substrates on steady-state levels of mRNA coding for ribulose-1,5-bisphosphate carboxylase and light-harvesting and reaction center polypeptides in *Rhodopseudomonas sphaeroides*. J. Bacteriol. 162:925–932.
