
Molecular Microbiology (1994) 13(6), 1065–1070

# Glycopeptide resistance mediated by enterococcal transposon Tn1546 requires production of VanX for hydrolysis of D-alanyl-D-alanine

Peter E. Reynolds, ${ }^{1}$ Florence Depardieu, ${ }^{2}$ Sylvie Dutka-Malen, ${ }^{2}$ Michel Arthur ${ }^{2}$ and Patrice Courvalin ${ }^{2}$

${ }^{1}$ Department of Biochemistry, University of Cambridge, Tennis Court Road, Cambridge CB2 1QW, UK.

${ }^{2}$ Unité des Agents Antibactériens, Centre National de la Recherche Scientifique EP J0058, Institut Pasteur, 75724 Paris Cedex 15, France.

## Summary

Cloning and nucleotide sequencing indicated that transposon Tn1546 from Enterococcus faecium BM4147 encodes a 23 365 Da protein, VanX, required for glycopeptide resistance. The vanX gene was located downstream from genes encoding the VanA ligase and the VanH dehydrogenase which synthesize the depsipeptide D-alanyl-D-lactate (D-Ala-D-Lac). In the presence of ramoplanin, an Enterococcus faecalis JH2-2 derivative producing VanH, VanA and VanX accumulated mainly UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Lac (pentadepsipeptide) and small amounts of UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Ala (pentapeptide) in the ratio 49:1. Insertional inactivation of vanX led to increased synthesis of pentapeptide with a resulting change in the ratio of pentadepsipeptide to less than 1:1. Expression of vanX in E. faecalis and Escherichia coli resulted in production of a D,D-dipeptidase that hydrolysed D-Ala-D-Ala. Pentadepsipeptide, pentapeptide and D-Ala-D-Lac were not substrates for the enzyme. These results establish that VanX is required for production of a D,D-dipeptidase that hydrolyses D-Ala-D-Ala, thereby preventing pentapeptide synthesis and subsequent binding of glycopeptides to D-Ala-D-Ala-containing peptidoglycan precursors at the cell surface.

## Introduction

Inducible resistance to high levels of the glycopeptide antibiotics vancomycin and teicoplanin in clinical isolates of enterococci is mediated by Tn1546 or related transposons (Arthur and Courvalin, 1993; Arthur et al., 1993). Tn1546 carries a five-gene cluster sufficient for inducible expression of resistance (Fig. 1). The vanR and vanS genes encode a two-component regulatory system that activates transcription of vanH and vanA in response to the presence of vancomycin (Arthur et al., 1992c). The VanH dehydrogenase reduces pyruvate to D-lactate (D-Lac) (Bugg et al., 1991b; Arthur et al., 1992a). The VanA ligase (Dutka-Malen et al., 1990; Bugg et al., 1991a) catalyses synthesis of the depsipeptide D-alanyl-D-lactate (D-Ala-D-Lac), which replaces the dipeptide D-Ala-D-Ala in the assembly pathway of peptidoglycan precursors (Bugg et al., 1991b; Messer and Reynolds, 1992; Allen et al., 1992; Handwerger et al., 1992). In susceptible cells, binding of glycopeptides to the peptidyl-D-Ala-D-Ala termini of peptidoglycan precursors at the cell surface blocks the transglycosylation and transpeptidation reactions (Reynolds, 1989). The precursors terminating in D-Ala-D-Lac bind vancomycin with reduced affinity (Bugg et al., 1991b) and allow cell wall synthesis to proceed in the presence of glycopeptides.

As the homofermentative pathway of hexose dissimilation in Enterococcus faecalis leads to the production of L-Lac from pyruvate, synthesis of D-Ala-D-Lac should only require VanH and VanA (Arthur et al., 1992a). Furthermore, the depsipeptide D-Ala-D-Lac is a substrate for the enzyme that ligates D-Ala-D-Ala to tripeptide UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys in susceptible cells (Bugg et al., 1991b). Thus, production of VanA and VanH, in addition to enzymes encoded by the host chromosome, should be sufficient for synthesis of depsipeptide-containing precursors (Arthur and Courvalin, 1993). However, cloning experiments indicated that expression of an additional gene of Tn1546, designated vanX, was also required for resistance (Arthur et al., 1992c). We report here the nucleotide sequence of vanX and the role of the corresponding gene product in the metabolism of peptidoglycan precursors.

## Results and Discussion

### Nucleotide sequence of the vanX gene

Plasmid pAT85 was constructed by inserting the aphA-1 kanamycin-resistance gene downstream from the vanA gene of pAT80 (Fig. 1). The insertion abolished

1066 P. E. Reynolds et al.

![Diagram](#)

Fig. 1. Maps of the inserts in recombinant plasmids. The vectors are indicated in parentheses. Open arrows represent coding sequences. Genes *vanH*, *vanA* and *vanX* are cotranscribed from a promoter located in the *vanS-vanH* intergenic region (Arthur et al., 1992c). Restriction sites: B, BamHI; Bg, BglII; E, EcoRI; Hc, HindIII; P, PstI; X, XbaI.

glycopeptide resistance mediated by pAT80 (Arthur et al., 1992c). Sequencing of this portion of pAT80 revealed the presence of a single large open reading frame (ORF) (Fig. 2). The proposed translation initiation codon for VanX was preceded by a putative ribosome binding site (RBS) (5′TTAAAGGGGT N7 ATG) that displays limited complementarity (underlined) to the 3′ extremity of *Bacillus subtilis* 16S ribosomal RNA (3′-OH UCUUUCCUCC). An alternative translation initiation codon was located 9 bp upstream from this ATG but was not preceded by a puta-

tive RBS. The proximity of the *vanA* stop codon and of the *vanX* ATG (Fig. 2) combined with the absence of an RBS typical of Gram-positive bacteria (Moran et al., 1982) indicate that translational coupling might contribute to *vanX* expression.

The putative product of *vanX* is an acidic protein of 202 amino acids with a calculated *M*<sub>r</sub> of 23,365 that was designated VanX. The protein does not contain any cluster of hydrophobic amino acids that could correspond to a transmembrane region. Screening of the NBRF protein

C-terminus of VanA N-terminus of VanX
A G I A L P E L I D R L I V L A L K G * M E I G F T F L 8
CTGCAGGTATTGCACCTCCC GAAACTGATTGACC GCTTGATCGTATTAGCGTTAAAGGGGTGATAAGCATGGAAAATAGGATTTTACTTTTTT 90
PstI
D E I V H G V R W D A K Y A T W D N F T G K P V D G Y E V N 38
AGATGAAATAGTACACGGTGTTCGGGACGCTAAATATGCCACTTGGGATAATTT CACCGGAAAAACGGTTTGACGGTTATGAAGTAAA 180
R I V G T Y E L A E S L L K A K E L A A T Q G Y G L L L W D 68
TCGCATTGTAGGGACATACGGAGTTGGCTGAATCGCTTTTGAAGGCAAAAGAAC TGGGCTGTTCCAAGGGTACGGATTGGCTTCTATGGGA 270
G Y R P K R A V N C F M Q W A A Q P E N N L T K E S Y Y P N 98
CGGTTACC GT CCTA AG CGT GCT GTA ACT GTTTT ATG CA ATGGG CTG CAC AGC CGG AAA ATA ACCTG ACA AAAGG AAAGTT ATT AT CCC AA 360
I D R T E M I S K G Y V A S K S S H S R G S A I D L T L Y R 128
TATTG ACCG GA ACT GAG ATG AT TTC AAA AGG ATA CAG TG GG CT TC AAA AT CA AG CC CAT AG CC GCG GC AG TG GCC ATT GA TCT TAC GCT TT TAT CG 450
L D T G E L V P M G S R F D F M D E R S H H A A N G I S C N 158
ATTAG ACAC GGG GT GAG CTT GT ACC AA TGG GG AG CC GA TTT TAT GG AT GA ACG CT CT CAT CAT GCG GCAA AT GGA AT AT CAT GCAA 540
E A Q N R R R L R S I M E N S G F E A Y S L E W W H Y V L R 188
TGA AG CG CAAA AT CG CAG AC GT TT GG CTC CAT GAT G GAAA A CAG TGG G TTT TGA AG CAT AT AG CC TCG AA AT GGT GGC ACT AT GT TAT TA AG 630
D E P Y P N S Y F D F P V K * AGACGA ACC CAT ACC CCA A TAG CT AT TTT TG AT TT TCCC GT TA AA ATA AA ACT TTT A ACC GT TG CA CGG ACA AA ACT AT AT AA AG CT AA CT CT TTT C 720
GGC AG GAAA ACC CG AC GT AT GT AA CT GG TT CTT AG GG AAT TT AT AT AT AG TAT GT AA TT GA AG GAT GTA AG GC AG CG GA TAT TG CG GT C 810
ATT AT CTG CG GT GCG GCT GCG GCA AG GA TAG CC TG ATA ATA AAG ACT GAT CG CA TAG AG GGG GT GT TTT CA CAC CG GCC CATT GT CA AC AG GCA 900
GTTCAGCCTCGTTAAATTCAGCATGGGTATCACTTATGAAAATTCATCTACATTGGTGATAATAGTAAATCCAGTAGGGCGAAATAATG 990
ACTGTAATTTACGGGGCAAAACGGCACAAATCTCAACGGAGATTGTGCCGTTTAAGGGGAAGATTCTAGA 1059
XbaI

Fig. 2. Nucleotide sequence of the 1059 bp PstI–XbaI fragment of Tn1546 encoding VanX. The HincII restriction site at position 161 was used to inactivate the *vanX* gene of pAT85. The putative ribosome binding site of *vanX* is indicated in bold. The deduced amino acid sequence of VanX and of the C-terminus of VanA is shown above the nucleotide sequence.
sequence data base did not reveal any significant similarity between VanX and other proteins.

### Effect of vanX expression on the cytoplasmic pool of peptidoglycan precursors

Cytoplasmic peptidoglycan precursors were extracted from bacteria incubated with ramoplanin, which is unrelated to glycopeptides and blocks the addition of N-acetylglucosamine to the membrane-bound lipid intermediate I (C55-undecaprenyl-P-P-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Ala) at the inner surface of the membrane (Sommer and Reynolds, 1990). Inhibition of this reaction leads to the accumulation of pentapeptide UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Ala, the last soluble cytoplasmic precursor in the assembly pathway of peptidoglycan (Sommer and Reynolds, 1990). As expected, E. faecalis JH2-2 and JH2-2 harbouring vector pAT78 accumulated only pentapeptide (Table 1) in addition to small amounts of UDP-GlcNAc and UDP-MurNac (data not shown). The latter precursors represented approximately 3% and 5% respectively of total N-acetyhexosamine-containing precursors in all the strains studied. The amount of pentapeptide accumulated in the presence of ramoplanin was consistent with its normal rate of synthesis and incorporation into the cell wall (data not shown).

Plasmid pAT80, which confers high-level resistance to glycopeptides, was constructed by inserting vanR, vanS, vanH, vanA and vanX into the multicopy vector pAT78 (Fig. 1). Vancomycin was not added to the culture medium of E. faecalis JH2-2/pAT80 because the vanH, vanA and vanX genes of pAT80 are transcribed at a high level in the absence of induction by glycopeptides (Arthur et al., 1992c). E. faecalis JH2-2/pAT80 accumulated mainly pentadepsipeptide UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Lac (98% of total precursors) and small amounts of pentapeptide (2%). Thus, in this strain D-Ala-D-Lac almost completely replaced D-Ala-D-Ala in the

---

Mechanism of glycopeptide resistance 1067

assembly pathway of peptidoglycan precursors. Tripeptide UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys was not detected indicating that D-Ala-D-Lac was not limiting for pentadepsipeptide synthesis.

Clinical isolates of glycopeptide-resistant enterococci produce a vancomycin-inducible D,D-carboxypeptidase that cleaves the C-terminal residue of pentapeptide and pentadepsipeptide (Al-Obeid et al., 1990; Gutmann et al., 1992; Wright et al., 1992). Tetrapeptide UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala was detected in these isolates (Messer and Reynolds, 1992). Production of this D,D-carboxypeptidase requires expression of the Tn1546 gene vanY, which is not essential for glycopeptide resistance (Arthur et al., 1992b). As expected, tetrapeptide did not accumulate in E. faecalis JH2-2/pAT80, which does not contain vanY.

The aphA-1 kanamycin-resistance gene was inserted into the vanX gene of pAT80 giving rise to plasmid pAT85, which does not confer glycopeptide resistance (Arthur et al., 1992c). In comparison to JH2-2/pAT80, insertional inactivation of vanX in pAT85 resulted in the accumulation in the same host of pentapeptide (54%) in addition to pentadepsipeptide (46%). Therefore, VanX was not required for pentadepsipeptide synthesis although this precursor was partially replaced by pentapeptide when VanX was absent. This change could result from the hydrolysis of the dipeptide D-Ala-D-Ala (Gutmann et al., 1992) or from negative regulation of the chromosomal gene encoding the host D-Ala:D-Ala ligase (Bugg et al., 1991b).

#### Hydrolysis of peptidoglycan precursors

The dipeptide D-Ala-D-Ala was hydrolysed by cytoplasmic extracts from E. faecalis JH2-2/pAT80 but not from JH2-2 and JH2-2/pAT85 (Table 1). Thus, vanX was required for production of a D,D-dipeptidase. The majority of the activity (97%) was recovered in the cytoplasmic fraction of E. faecalis JH2-2/pAT80 suggesting that the enzyme

---

**Table 1. Peptidoglycan precursors and D,D-dipeptidase activity in extracts from derivatives E. faecalis JH2-2 harbouring various plasmids.**

| Plasmid (relevant characteristics) | total<sup>a</sup> | type of precursors<sup>b</sup> (%) | D,D-dipeptidase activity<sup>c</sup> |
| --- | --- | --- | --- |
|  | (μmole) | penta | pentadepsi | tetra | tri | (nmole min<sup>-1</sup> mg<sup>-1</sup>) |
| None | 12.8 | 100 | ND<sup>d</sup> | ND | ND | < 2 |
| pAT78 (vector) | 10.4 | 100 | ND | ND | ND | Not done |
| pAT80 (vanR, vanS, vanH, vanA and vanX in pAT78) | 10.1 | 2 | 98 | ND | ND | 1200 |
| pAT85 (insertion of aphA-1 in vanX of pAT80) | 14.1 | 54 | 46 | ND | ND | < 2 |

**a.** Total amount of precursors recovered from 1.25 l of culture grown to an OD600 of 1.0 prior to the addition of ramoplanin. The early precursors of the pathway UDP-MurNAc and UDP-GlcNAc were not included.

**b.** Penta, pentapeptide UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Ala; pentadepsi, pentadepsipeptide UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Lac; tetra, tetrapeptide UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala; tri, tripeptide UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys.

**c.** Nanomoles of D-Ala released from D-Ala-D-Ala per min and per mg of protein in cytoplasmic extract.

**d.** ND, not detected. The minimum amount of precursor detectable by the N-acetylhexosamine assay was approximately 10 nmole.
was not associated with the membrane to a significant extent. SDS–PAGE analysis of this cytoplasmic fraction revealed a protein with a mass of approximately 22 kDa that was not present in JH2-2/pAT85 (data not shown). This protein band could be the product of vanX, which has a calculated mass of 23 365 Da.

Expression of vanX under the control of the lac promoter of pUC18 resulted in D,D-dipeptidase activity in Escherichia coli JM83/pAT389 suggesting that vanX is the structural gene for the D,D-dipeptidase (data not shown). As in E. faecalis, the activity was mainly present in the cytoplasmic fraction. D,D-peptidase activity was not detected in extracts of the host strain E. coli JM83. An ammonium sulphate precipitate of the extract of E. coli JM83/pAT389 was used to analyse the substrate specificity of the enzyme. The D,D-dipeptidase did not cleave ester bonds but exclusively hydrolysed dipeptides with free NH₂ and COOH termini (Table 2). The contribution of the enzyme to glycopeptide resistance is therefore due to the hydrolysis of D-Ala-D-Ala which decreases pentapeptide synthesis and thus favours the production of pentadepsipeptide. The D,D-dipeptidase appears to be unrelated to other peptidases including the D-stereospecific aminopeptidase of Ochrobactrum anthropi which hydrolyses tri- and tetrapeptides, amides and esters in addition to D-Ala-D-Ala (Asano et al., 1989). Furthermore, the D,D-dipeptidase was not inhibited by benzyl penicillin (Table 2) whereas the aminopeptidase of O. anthropi is inhibited by β-lactams (Asano et al., 1992). Full characterization of the kinetics and substrate specificity of the enzyme will be carried out on the purified protein.

### Conclusion

Resistance to glycopeptides mediated by pAT80 required production of three enzymes: VanA and VanH for synthesis of D-Ala-D-Lac; and VanX for hydrolysis of D-Ala-D-Ala. Their combined action led to efficient synthesis of pentadepsipeptide which represented 98% of the late cytoplasmic precursors (Table 1). Insertional inactivation of vanX resulted in increased synthesis of pentapeptide which partially replaced pentadepsipeptide. As vanX was required for production of a dipeptidase, hydrolysis of D-Ala-D-Ala prevented synthesis of pentapeptide. Co-production of pentapeptide and pentadepsipeptide in E. faecalis JH2-2/pAT85 resulted in glycopeptide susceptibility, probably because binding of the antibiotics to D-Ala-D-Ala-containing precursors blocked peptidoglycan polymerization on the outside surface of the cytoplasmic membrane. Thus, expression of glycopeptide resistance in enterococci requires reduced synthesis of pentapeptide.

As the catalytic efficiency of VanA for D-Ala-D-Ala synthesis is very low (Bugg et al., 1991a, b), pentapeptide accumulated by E. faecalis JH2-2/pAT85 should originate from dipeptide produced by the D-Ala:D-Ala ligase encoded by the host chromosome. Therefore, negative regulation of D-Ala:D-Ala ligase production, if any, was not sufficient for glycopeptide resistance. These observations suggest that hydrolysis of D-Ala-D-Ala is the main mechanism that prevents synthesis of pentapeptide in glycopeptide-resistant enterococci.

Identification of VanX as a dipeptidase indicates that the protein is not directly involved in metabolism of depsipeptide-containing cell wall precursors. Consequently, the fact that VanA, VanH and VanX are sufficient for resistance demonstrates that the incorporation of D-Ala-D-Lac into cell wall precursors and subsequent steps of cell wall synthesis are catalysed by enzymes encoded by the host chromosome.

### Experimental procedures

#### Strains, plasmids and growth conditions

The bacterial strains and plasmids used are described in Table 3 and Fig. 1. Enterococci were grown in brain–heart infusion (BHI) broth supplemented with glucose (1% w/v) or on BHI agar. Spectinomycin (60 μg ml⁻¹) was added to the medium for plasmid-containing derivatives of E. faecalis JH2-2. E. coli JM83 and E. coli JM83/pAT389 were grown in Luria (L) broth or in L broth containing ampicillin (100 μg ml⁻¹), respectively. All incubations were at 37°C.

#### Cloning of vanX

Cloning methods, restriction endonuclease digestion, plasmid isolation, and transformation were carried out as described (Ausubel et al., 1987). For construction of pAT389, the 1.1 kb PstI fragment of pAT80 was purified, treated with a DNA-blunting kit (Amersham) and ligated with pUC18 digested with Smal. Restriction analysis indicated that vanX was placed under the control of the lac promoter of pUC18.

---

### Table 2. Substrate specificity of the D,D-dipeptidase produced by E. coli JM83/pAT389(vanX).

| Substrate                          | Enzymic activity<sup>a</sup> (% rate of hydrolysis of D-Ala-D-Ala) |
|------------------------------------|---------------------------------------------------------------|
| D-Ala-D-Ala                        | 100                                                             |
| D-Ala-D-Ala + benzyl penicillin (100 μg ml⁻¹) | 100                                                             |
| D-Ala-Gly                          | 26                                                              |
| D-Ala-NH₂                          | 0                                                               |
| D-Ala-D-Ala-D-Ala                  | 0                                                               |
| D-Ala-D-Ala-NH₂                    | 0                                                               |
| D-Ala-D-Lac                        | 0                                                               |
| Ac-D-Ala-D-Ala                     | 0                                                               |
| Ac₂-L-Lys-D-Ala-D-Ala              | 0                                                               |
| UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Ala | 0                                                               |
| UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Lac | 0                                                               |

<sup>a</sup> Hydrolysis as low as 0.2% of the activity against D-Ala-D-Ala could be detected. The substrates were present at 10 mM, 10-fold greater than a saturating concentration of D-Ala-D-Ala for the enzyme.

Table 3. Strains and plasmids.

| Strain/Plasmid | Genotype/Plasmid description | Origin/Reference |
|----------------|-------------------------------|------------------|
| E. coli JM83    | ara Δ(lac-proAB) rpsL        | Messing (1979)   |
|                | φ80 F′(traΔ36 proA⁺          |                  |
|                | proB⁺ lacIq lacZΔM15)       |                  |
| E. faecalis JH2-2 | Fusᴿ Rifᴿ                   | Jacob and Hobbs (1974) |
| pAT78          | oriRpAMβ1 oriRpUC            | Arthur et al. (1992c) |
|                | oriTRK2 spc lacZα cat        |                  |
| pAT80          | vanR vanS vanH vanA          | Arthur et al. (1992c) |
|                | vanX cloned into             |                  |
|                | pAT78                        |                  |
| pAT85          | aphA-1 kanamycin-            | Arthur et al. (1992c) |
|                | resistance gene inserted     |                  |
|                | into the vanX gene of        |                  |
|                | pAT80                        |                  |
| pUC18          | Plasmid vector Apᴿ lacZα     | Norrander et al. (1983) |
| pAT389         | 1059 bp vanX fragment of     | This work       |
|                | Tn1546 in pUC18              |                  |

DNA sequence determination and analysis

Sequencing of both strands of DNA was performed by the dideoxynucleotide chain termination method (Sanger et al., 1977) using [α-³⁵S]-dATP (400 Ci mmol⁻¹; Amersham) and the Sequenase Version 2.0 DNA sequencing kit (United States Biochemical Corporation). The National Biomedical Research Foundation protein sequence database was screened using the FASTP algorithm developed by Lipman and Pearson (1985). The nucleotide sequence of Tn1546 has been deposited in GenBank (accession number M97297).

Analysis of peptidoglycan precursors

Enterococci were grown in BHI broth (1.25 l) to mid-exponential phase (A₆₀₀ = 1), ramoplanin (9 μg ml⁻¹) was added and incubation was continued for 0.66 mean generation time. Bacteria were harvested by centrifugation at 12 000 × g for 2 min at 4°C, resuspended in water and treated with 7% trichloroacetic acid (TCA) for 15 min at 0°C in a final volume of 30 ml. The extract was centrifuged (48 000 × g for 1 min at 4°C) and the supernatant fraction containing the cytoplasmic peptidoglycan precursors collected. The pellet was washed with 10 ml of 5% TCA and the supernatants combined. TCA was removed by four extractions with an equal volume of ether, the pH adjusted to 6.5 with NaOH and residual ether evaporated at 30°C. Peptidoglycan precursors were separated by ion-exchange chromatography on Dowex 1 Cl⁻ (Serva) using a linear chloride gradient (0–0.3 M NaCl) and desalted by gel filtration on G 10 Sephadex (Pharmacia). Peptidoglycan precursors were previously shown to elute in four peaks that were identified by u.v. absorption and determination of N-acetylhexosamine (Messer and Reynolds, 1992). Pentapeptide, tetrapeptide and tripeptide eluted at the same concentration of NaCl (0.12 M) and were collected as a single pool (peak 1). Depsipeptidetide (peak 2), UDP-GlcNAc (peak 3) and UDP-MurNAc (peak

4) eluted at 0.15, 0.22 and 0.23 M NaCl, respectively. The precursors in the different peaks were identified and measured in four ways as described (Messer and Reynolds, 1992): (i) automated amino acid analysis of samples hydrolysed in 6 M HCl for 4 and 24 h; (ii) determination of D-Ala or D-Lac released from pentapeptide and pentadepsipeptide by the D,D-carboxypeptidase from Actinomadura R39, respectively; (iii) analysis of mass spectra by positive ion electrospray mass spectroscopy; and (iv) high performance liquid chromatography on a C18 reverse-phase column using isocratic elution with 0.05 M ammonium acetate buffer, pH 5.03.

Preparation of enzyme extracts from E. faecalis

Strains were grown in BHI broth (100 ml) to an optical density of 1.0 at 600 nm. Bacteria were harvested by centrifugation (30 000 × g for 30 s at 4°C), washed in buffer A (50 mM Tris-HCl, pH 7.5) and resuspended in 12 ml of the same buffer. The bacteria were lysed by treatment with lysozyme (200 μg ml⁻¹) and M1 muramidase (30 μg ml⁻¹) at 37°C. When lysis was almost complete (c. 10–15 min), MgCl₂ (5 mM) and DNase (20 μg ml⁻¹) were added and incubation was continued for 5 min at 37°C. The extracts were centrifuged (48 000 × g for 20 min at 4°C) and the supernatant collected (cytoplasmic fraction). The pellet was washed in buffer A and resuspended in the same buffer at a concentration of 5 mg protein per ml (membrane fraction).

Preparation of enzyme extracts from E. coli

E. coli JM83 and JM83/pAT389 were grown to an optical density of 0.75 at 600 nm, IPTG (2 mM) was added and incubation continued for 2 h. Bacteria were collected by centrifugation (30 000 × g for 30 s at 0°C), washed in 10 mM phosphate buffer, pH 7.5, resuspended in 1 ml of the same buffer, and broken by sonication for 30 s. The extracts were centrifuged (48 000 × g for 20 min at 4°C) and the supernatant (cytoplasmic fraction) and pellet (membrane fraction) were collected. Substrate specificity was determined using a 47–60% ammonium sulphate precipitate of the cytoplasmic fraction that was resuspended in and dialysed against 50 mM Tris-HCl, pH 7.5.

Assay for D,D-dipeptidase activity

The routine assay contained in a final volume of 30 μl, 75 mM Tris-HCl, pH 7.5, 6 mM MgCl₂, 1.66 mM D-Ala-D-Ala and enzyme preparation, diluted to ensure that less than 20% of the substrate was hydrolysed in a 30 min incubation at 37°C. D-Ala released was assayed using D-amino acid oxidase (Messer and Reynolds, 1992). The concentration of substrate was increased to 10 mM for tests of substrate specificity. The substrates were obtained from the following sources: D-Ala-D-Ala, D-Ala-Gly, D-Ala-D-Ala-D-Ala (Sigma); D-Ala-D-Ala-NH₂ (prepared by I. Hutchinson, Cambridge Centre for Molecular Recognition); D-Ala-D-Lac, D-Ala-NH₂ (gift of G. Smith, University of Cambridge); Ac-D-Ala-D-Ala (gift of D. Williams, University of Cambridge); Ac₂-L-Lys-D-Ala-D-Ala (gift of J.-M. Frère, University of Liège, Belgium). UDP-MurNAc-pentapeptide and UDP-MurNAc-pentadepsipeptide were purified from Staphylococcus aureus and E. faecalis JH2-2/pAT80, respectively.

Acknowledgements

We thank J. Jacoby and P. Skelton (Cambridge Centre for Molecular Recognition) for amino acid analyses and mass spectrometry respectively, A. J. Maguire for HPLC analyses, J.-M. Frère for gifts of Actinomadura R39 D,D-carboxypeptidase, and R. Quintiliani, Jr for helpful discussions.

References

Al-Obeid, S., Collatz, E., and Gutmann, L. (1990) Mechanism of resistance to vancomycin in Enterococcus faecium D366 and Enterococcus faecalis A256. Antimicrob Agents Chemother 34: 252–256.

Allen, N.E., Hobbs, Jr, J.N., Richardson, J.M., and Riggin, R.M. (1992) Biosynthesis of modified peptidoglycan precursors by vancomycin-resistant Enterococcus faecium. FEMS Microbiol Lett 98: 109–116.

Arthur, M., and Courvalin, P. (1993) Genetics and mechanisms of glycopeptide resistance in enterococci. Antimicrob Agents Chemother 37: 1563-1571.

Arthur, M., Molinas, C., Bugg, T.D.H., Wright, G.D., Walsh, C.T., and Courvalin, P. (1992a) Evidence for in vivo incorporation of D-lactate into peptidoglycan precursors of vancomycin-resistant enterococci. Antimicrob Agents Chemother 36: 867–869.

Arthur, M., Molinas, C., and Courvalin, P. (1992b) Sequence of the vanY gene required for production of a vancomycin-inducible D,D-carboxypeptidase in Enterococcus faecium BM4147. Gene 120: 111–114.

Arthur, M., Molinas, C., and Courvalin, P. (1992c) The VanS-VanR two-component regulatory system controls synthesis of depsipeptide peptidoglycan precursors in Enterococcus faecium BM4147. J Bacteriol 174: 2582–2591.

Arthur, M., Molinas, C., Depardieu, F., and Courvalin, P. (1993) Characterization of Tn1546, a Tn3-related transposon conferring glycopeptide resistance by synthesis of depsipeptide peptidoglycan precursors in Enterococcus faecium BM4147. J Bacteriol 175: 117–127.

Asano, Y., Nakazawa, A., Kato, Y., and Kondo, K. (1989) Properties of a novel D-stereospecific aminopeptidase from Ochrobactrum anthropi. J Biol Chem 264: 14233–14239.

Asano, Y., Kato, Y., Yamada, A., and Kondo, K. (1992) Structural similarity of D-aminopeptidase to carboxypeptidase DD and β-lactamases. Biochemistry 31: 2316–2328.

Ausubel, F.M., Brent, R., Kingston, R.E., Moore, D.D., Seidman, J.G., Smith, J.A., and Struhl, K. (1987) Current Protocols in Molecular Biology. New York: John Wiley and Sons, Inc.

Bugg, T.D.H., Dutka-Malen, S., Arthur, M., Courvalin, P., and Walsh, C.T. (1991a) Identification of vancomycin resistance protein VanA as a D-alanine:D-alanine ligase of altered substrate specificity. Biochemistry 30: 2017–2021.

Bugg, T.D.H., Wright, G.D., Dutka-Malen, S., Arthur, M., Courvalin, P., and Walsh, C.T. (1991b) Molecular basis for vancomycin resistance in Enterococcus faecium BM4147: biosynthesis of a depsipeptide peptidoglycan precursor by vancomycin resistance proteins VanH and VanA. Biochemistry 30: 10408–10415.

Dutka-Malen, S., Molinas, C., Arthur, M., and Courvalin, P. (1990) The VANA glycopeptide resistance protein is related to D-alanyl-D-alanine ligase cell wall biosynthesis enzymes. Mol Gen Genet 224: 364–372.

Gutmann, L., Billot-Klein, D., Al-Obeid, S., Klare, I., Francoual, S., Collatz, E., and Van Heijenoort, J. (1992) Inducible carboxypeptidase activity in vancomycin-resistant enterococci. Antimicrob Agents Chemother 36: 77–80.

Handwerger, S., Pucci, M.J., Volk, K.J., Liu, J., and Lee, M.S. (1992) The cytoplasmic peptidoglycan precursor of vancomycin resistant Enterococcus faecalis terminates in lactate. J Bacteriol 174: 5982–5984.

Jacob, A.E., and Hobbs, S.J. (1974) Conjugal transfer of plasmid-borne multiple antibiotic resistance in Streptococcus faecalis var. zymogenes. J Bacteriol 117: 360–372.

Lipman, D.J., and Pearson, W.R. (1985) Rapid and sensitive protein similarity searches. Science 227: 1435–1440.

Messer, J., and Reynolds, P.E. (1992) Modified peptidoglycan precursors produced by glycopeptide-resistant enterococci. FEMS Microbiol Lett 94: 195–200.

Messing, J. (1979) A multipurpose cloning system based on the single-stranded bacteriophage M13. Recombinant DNA Tech Bull 1: 43–44.

Moran, C.P.J., Lang, N., LeGrice, S.F.J., Lee, G., Stephens, M., Sonenshein, A.L., Pero, J., and Losick, R. (1982) Nucleotide sequences that signal the initiation of transcription and translation in Bacillus subtilis. Mol Gen Genet 186: 339–346.

Norrander, J., Kempe, T., and Messing, J. (1983) Construction of improved M13 vectors using oligodeoxynucleotide-directed mutagenesis. Gene 26: 101–106.

Reynolds, P.E. (1989) Structure, biochemistry and mechanism of action of glycopeptide antibiotics. Eur J Clin Microbiol Inf Dis 8: 943–950.

Sanger, F., Nicklen, S., and Coulson, A.R. (1977) DNA sequencing with chain-terminating inhibitors. Proc Natl Acad Sci USA 74: 5463–5467.

Sommer, E.A., and Reynolds, P.E. (1990) Inhibition of peptidoglycan biosynthesis by ramoplanin. Antimicrob Agents Chemother 34: 413–419.

Wright, G.D., Molinas, C., Arthur, M., Courvalin, P., and Walsh, C.T. (1992) Characterization of VanY, a D,D-carboxypeptidase from vancomycin-resistant Enterococcus faecium BM4147. Antimicrob Agents Chemother 36: 1514–1518.

This document is a scanned copy of a printed document. No warranty is given about the accuracy of the copy. Users should refer to the original published version of the material.
