
MICROBIOLOGICAL REVIEWS, Sept. 1994, p. 491–562
0146-0749/94/$04.00+0
Copyright © 1994, American Society for Microbiology

The Alphaviruses: Gene Expression, Replication, and Evolution

JAMES H. STRAUSS* AND ELLEN G. STRAUSS

Division of Biology, California Institute of Technology, Pasadena, California 91125

INTRODUCTION ..... 492
- The Cast of Characters ..... 492
- Dispersal of Alphaviruses ..... 494
- Full-Length Clones of Alphaviruses ..... 494

STRUCTURE OF ALPHAVIRUSES ..... 495
- Structure of the Virion ..... 495
  - Nucleocapsid ..... 495
  - Virion envelope ..... 496
- Organization of the Alphavirus Genome ..... 498

REPLICATION OF ALPHAVIRUSES ..... 499
- Overview of the Alphavirus Life Cycle ..... 499
- Structural Proteins ..... 499
  - Capsid protein ..... 499
  - Synthesis of glycoproteins ..... 500
  - Addition of carbohydrate and lipids ..... 502
  - Folding and transport of glycoproteins ..... 502
  - Cleavage of PE2 during transport ..... 503
  - Reorientation of the E2 C-terminal domain ..... 504
- Virus Assembly ..... 505
  - Assembly of nucleocapsids ..... 505
  - Capsid-spike interaction during budding ..... 506
  - Domains involved in the E2-nucleocapsid interaction ..... 506
  - Lateral interactions between glycoproteins ..... 508
  - The 6K protein and ion gradients in assembly ..... 509
- Nonstructural Proteins ..... 509
  - Translation ..... 509
  - Functions of nsP1 ..... 510
  - Functions of nsP2 ..... 510
  - Functions of nsP3 ..... 511
  - Functions of nsP4 ..... 511
- Nonstructural Proteinase ..... 512
  - Active site of the enzyme ..... 512
  - Nature of cleavage sites recognized by the enzyme ..... 512
  - Cleavage site preferences of the enzyme ..... 513
- Synthesis of Viral RNAs ..... 514
  - RNA replicase ..... 514
  - Control of minus-strand RNA synthesis ..... 514
  - Role of the opal termination codon ..... 517

cis-ACTING SEQUENCE ELEMENTS IN ALPHAVIRUS RNAs ..... 518
- Conserved Nucleotide Sequence Elements ..... 518
  - Identification of CSEs ..... 518
  - Mutagenesis of CSEs ..... 519
  - CSEs can be tissue specific ..... 519
  - Cellular proteins bind to CSEs ..... 519
  - Promoter for subgenomic RNA synthesis ..... 520
- Repeated Sequence Elements in the 3′ NTR ..... 521
- Sequences in DI RNAs Define Essential cis-Acting Elements ..... 522
- Packaging Signal in Alphavirus RNAs ..... 524
- Chimeric Alphaviruses ..... 525
- Alphaviruses as Vectors ..... 525

VIRUS-HOST INTERACTIONS ..... 526
- Host Cell Receptors for Alphaviruses ..... 526
  - Early studies of alphavirus receptors ..... 527
  - Anti-idiotypic antibodies as antireceptor antibodies ..... 527

* Corresponding author. Phone: (818) 395-4903. Fax: (818) 449-
0756. Electronic mail address: StraussJ@Starbase1.Caltech.edu.

Laminin receptor as a mammalian receptor ..... 528
Mosquito receptor for VEE ..... 529
Virus receptor-binding domain ..... 529
Virus Entry ..... 530
Early events in virus penetration ..... 530
Entry into endosomes ..... 531
Conformational changes in E1 and E2 ..... 532
Cholesterol requirement for entry by SF ..... 533
Early Events in Establishment of Infection ..... 533
Nucleocapsid disassembly ..... 533
Superinfection exclusion ..... 533
Inhibition of host protein synthesis ..... 534
Formation of replication factories ..... 535
Pathogenesis and Immunity ..... 536
Neutralization epitopes in E2 ..... 536
Neutralization epitopes in E1 ..... 536
Conformational dependence of epitopes ..... 536
Protection from disease ..... 537
Virus neurovirulence ..... 537
Clearance of virus from persistently infected neurons ..... 539
Induction of apoptosis ..... 539
Replication in the Mosquito Vector ..... 540
EVOLUTION OF ALPHAVIRUSES ..... 541
Divergence of Alphaviruses ..... 541
Rates of divergence ..... 541
Divergence of EEE and VEE ..... 542
Divergence of Sindbis viruses ..... 543
Alphavirus evolutionary trees ..... 543
Recombination among Alphaviruses ..... 544
Recombination in the laboratory ..... 544
Recombination to form WEE ..... 544
The Family Togaviridae ..... 545
The Sindbis-Like Supergroup of Viruses ..... 545
CONCLUDING REMARKS ..... 546
ACKNOWLEDGMENTS ..... 547
REFERENCES ..... 547

INTRODUCTION

The Cast of Characters

The alphaviruses have 26 currently recognized members (55). Each virus in turn consists of numerous geographical variants or strains. Inclusion in the genus *Alphavirus* has been based on serological cross-reaction with one or more of the existing members of the group (55, 409). Complete or partial sequences of the RNA genomes of more than 10 alphaviruses have now been reported, and these alphaviruses all share a minimum amino acid sequence identity of about 45% in the more divergent structural proteins and about 60% in the nonstructural proteins. Table 1 lists the currently recognized alphaviruses, as well as a number of strains that are referred to in this review, together with the geographic distribution of the viruses, their evolutionary relationships, and the abbreviations for the virus names that will be used throughout this review. The literature review for this paper was completed in May 1994.

As a group the alphaviruses have a very wide geographic distribution, isolates having been reported from all continents except Antarctica and from many islands (55), although each individual virus has a more limited distribution. The type virus, SIN, has a very wide distribution, strains of virus referred to as SIN having been isolated from throughout the Old World including Europe, Asia (including India and the Philippines), Australia, and many parts of Africa (both northern Africa and southern Africa). Viruses related to SIN have also been isolated from New Zealand (WHA) (484) and from South American (AURA) (434). The relationship of OCK to SIN deserves special mention. OCK has often been described as a distinct species because it causes a human disease different from that caused by SIN and it is serologically distinguishable from SIN. However, OCK is much more closely related to SIN AR339 and to South African strains of SIN than are strains called SIN that have been isolated from India, the Philippines, or Australia. In this review we will refer to OCK as a strain of SIN, on the basis of close sequence relationships, and use the term SIN-like viruses to include WHA and AURA.

Other Old World alphaviruses have more restricted distributions. RR, for example, is primarily an Australian virus. ONN virus caused a localized epidemic in Africa of about two million cases of human disease and has not been identified outside of this area. SF virus has been isolated from Africa, the USSR, and India.

Many of the New World viruses are widely distributed throughout the Americas. WEE is found from Canada to Argentina, and EEE and VEE occur in both North America and South America. Other New World viruses such as MAY or AURA have a more restricted distribution.

The alphaviruses can be grouped into seven complexes on the basis of serological cross-reactivity (55), and at least one member of six of these complexes has been sequenced in whole or in part. On the basis of these sequence data, the members of these six complexes fall naturally into three larger groups: the VEE/EEE group, the SF group, and the SIN group (10, 292,

TABLE 1. The alphaviruses

| Complex               | Abbreviation$^a$ | Virus name$^b$                          | Strains                | Distribution          | Sequence reference(s) |
|-----------------------|------------------|-----------------------------------------|------------------------|-----------------------|-----------------------|
| VEE/EEE group         | EEE             | EASTERN EQUINE ENCEPHALITIS            | North American        | North America        | 68, 569, 592          |
|                       |                 |                                         | South American        | South America        |                       |
|                       | VEE             | VENEZUELAN EQUINE ENCEPHALITIS         | Trinidad donkey       | South America        | 258                   |
|                       |                 |                                         | TC-83                 | Central America      |                       |
|                       | EVE             | Everglades                             |                      | Florida              | 589                   |
|                       | MUC             | Mucambo                                |                      | Brazil, Peru         | 589                   |
|                       | PIX             | Pixuna                                 |                      | Brazil               | 589                   |
| SF group              | SF              | SEMLIKI FOREST                         |                      | Africa, Eurasia      | 137, 138, 540         |
|                       | MID             | Middelburg                             |                      | Africa               | 516                   |
|                       | CHIK            | Chikungunya                            |                      | Africa               | 85                    |
|                       | ONN             | O'NYONG-NYONG                          |                      | Africa               | 292                   |
|                       | RR              | ROSS RIVER                             | Nelson Bay, T48       | Australia, Oceania   | 116                   |
|                       | BF              | Barmah Forest                          |                      | Australia            | 10                    |
|                       | GET             | Getah                                  |                      | Australia, Asia      |                       |
|                       | SAG             | Sagiyama                               |                      | Japan                |                       |
|                       | BEB             | Bebaru                                 |                      | Malaysia             |                       |
|                       | MAY             | Mayaro                                 |                      | South America        | 85                    |
|                       | UNA             | Una                                    |                      | South America        |                       |
| SIN group             | SIN             | SINDBIS                                | AR86, AR339, Girdwood, Karelian fever | Africa, Asia, Europe, Australia | 517 |
|                       |                 |                                         | OCKELBO (OCK)         | Scandinavia          | 485                   |
|                       | AURA            | AURA                                   |                      | Brazil, Argentina    | 434                   |
|                       | WHA             | WHATAROA                               |                      | New Zealand          | 484                   |
|                       | BAB             | Babanki                                |                      | West Africa          |                       |
|                       | KYZ             | Kyzylagach                             |                      | USSR                 |                       |
| Recombinant or        | WEE             | Western equine encephalitis            |                      | North America, South America | 177, 592 |
| uncertain             | HJ              | Highlands J                            |                      | Eastern U.S.A.       | 484, 587              |
|                       | FM              | Fort Morgan                            |                      | Western U.S.A.       |                       |
| Ungrouped             | NDU             | Ndumu                                  |                      | Africa               |                       |
|                       | ??              | Buggy Creek                            |                      | Western U.S.A.       |                       |

$a$ Standard abbreviations of the virus names which are used throughout the text.

$b$ Complete nucleotide sequences have been reported for the viruses shown in boldface capitals, and partial sequences have been reported for viruses in boldface, lowercase type. References to the sequences are given in the last column.

strains in South Africa cause a disease characterized by fever and rash (372). SF strains in the Central African Republic cause a disease characterized by exceptionally severe headache, fever, myalgia, and arthralgia (337).

### Dispersal of Alphaviruses

From a consideration of the sequence relationships among alphaviruses and their distributions, Levinson et al. (292) proposed that alphaviruses originated in the New World and were spread to the Old World twice, once to found the SIN group and once to found the SF group. Weaver et al. (592) presented a modified hypothesis in which the alphaviruses originated in either the New World or the Old World and were transmitted to the other hemisphere 2,000 to 3,000 years ago. This hypothesis requires that MAY and AURA must have been separately introduced (or reintroduced) into the New World after the divergence of the SF and SIN groups in the Old World. In either scenario, migratory birds are postulated to have distributed the viruses, although humans may have contributed to their spread.

The alphaviruses are arthropod borne, with mosquitoes being the usual vector. However, Fort Morgan virus and the Bijou Bridge strain of VEE have swallow bugs as their vector and alphaviruses have been isolated from other hematophagous arthropods. For example, SIN, whose primary or exclusive vectors are mosquitoes, has also been isolated from mites and ticks (55, 372, 481) and EEE has been isolated from chicken mites and lice that had been naturally infected (479); the chicken mites could transmit EEE but only very inefficiently. The distribution of the viruses thus reflects in part the distribution of their mosquito vectors, and the viruses are more abundant in tropical regions. Many of these viruses have birds as their primary vertebrate hosts, and bird-associated viruses such as SIN, WEE, and EEE tend to have the widest distributions. Many studies have been conducted in an effort to determine the importance of bird migration or dispersion in the spread of alphaviruses and other arboviruses (57, 95, 313, 331, 333, 334, 364, 499). These studies have shown that a large fraction of migratory birds may demonstrate evidence of past infection by alphaviruses and that alphaviruses can be isolated from the blood of migrating birds. Furthermore, it is clear that migrating birds are capable of transporting viruses long distances; South American strains of EEE have been isolated from the blood of migrating birds in the Mississippi delta (57), showing that these birds were infected in South America and arrived in the United States while still viremic.

The importance of birds in the dissemination of at least some alphaviruses is also suggested by comparisons of the genetic heterogeneity of RR, SIN, and the flaviviruses Murray Valley encephalitis virus and Kunjin virus in Australia. SIN and the flaviviruses are thought to have migratory waterfowl as major hosts in Australia, and minor variants of a single virus type are found throughout Australia. In contrast, the major vertebrate reservoirs of RR are small nonmigratory marsupials, and this virus has evolved into distinct strains in different parts of Australia (115a, 305, 309a, 310a).

Although avian hosts may be responsible for the global spread of alphaviruses, once introduced into an area most viruses evolve into an endemic strain that persists in a localized area, and continual mixing does not occur. As one example, EEE isolated in North America is distinguishable from virus isolated in South America (57, 62), even though, as described above, migrating birds are capable in principle of mixing these virus strains. Studies of other alphaviruses, including RR (332) and VEE (589), have similarly shown that strains isolated from

one geographic area are distinguishable from strains isolated from other areas. Strains of SIN-like viruses, which are almost worldwide in distribution, are for the most part related by location rather than by year of isolation (484, 485).

Humans are also capable of disseminating alphaviruses (as is the case for many other virus groups), particularly since the introduction of jet travel. RR is primarily an Australian virus which causes periodic outbreaks of polyarthritis in humans in many regions of Australia. RR recently caused an explosive epidemic of polyarthritis throughout the South Pacific (332), maintained probably by a human-mosquito-human transmission cycle. After infecting most of the human populations of a number of island groups, the virus died out, apparently failing to establish an endemic cycle in other animals. Because the first cases of disease occurred near the Nadi airport in Fiji, which has direct air connections with Australia, and then spread to other island groups having air connections with Nadi, and because all virus isolated during the epidemic appeared to be closely related if not identical (53, 115a), it seems very likely that a single infected air traveler introduced the virus to Fiji and that the spread was very rapid because the human populations on these islands had not been previously exposed to the virus.

### Full-Length Clones of Alphaviruses

Much sequence information has been reported for different alphavirus RNAs. Complete sequences have been determined for two strains of SIN (485, 517), for two strains of RR (116), for four strains of VEE and a vaccine derivative of one of these strains (258, 260, 261), for SF (137, 138, 540), for ONN (292), for EEE (68, 569, 592), for AURA (434), and for WHA (484); partial sequences have been determined for WEE (177, 592), MID (516), CHIK (85), MAY (85), BF (10), and HJ (484, 587). In addition, the results of extensive studies of the replication of these viruses in cultured cells have been reported, as have the results of many studies of temperature-sensitive (*ts*) mutants and of variants that differ in their neurovirulence or in their reactivity with monoclonal antibodies; many of these studies are reviewed below. An important development in these studies has been the construction of full-length cDNA clones corresponding to either a defective interfering (DI) RNA or the entire genome of an alphavirus, in which the viral sequences are positioned downstream of a promoter for an RNA polymerase, usually T7 or SP6 RNA polymerase. RNA transcribed in vitro from such clones can be transfected into susceptible cells. In the case of DI RNA, the cells are also infected with helper virus and DI particles can be rescued after several passages (295, 465, 467). In the case of full-length viral RNA, transfection leads to virus replication and production of infectious virions. Full-length clones have been reported for SIN (418), VEE (90) and a vaccine derivative of VEE (257), SF (303), and RR (273). Studies with these clones have allowed the identification of domains essential for the replication and packaging of the RNA, the mapping of genetic markers such as *ts* mutations or attenuating mutations in the genome, the construction of new mutations in regions of interest using site-directed mutagenesis, the construction of chimeric viruses to study the interactions of different domains of viral genomes with one another, and the use of alphaviruses as expression vectors to express genes of interest in the cytoplasm of an infected cell. These studies have required appropriate controls because it has been found that the RNA polymerases can make consistent mistakes and introduce errors into the transcribed RNA (272, 370, 371) and that suppressor mutations may arise rapidly, obscuring the effects of the mutation introduced into

FIG. 1. Stereoviews of the three-dimensional structure of RR virus. Surface-shaded representations of the three-dimensional structure of RR viewed along the icosahedral threefold axis are shown. The image was computed from 76 independent particle images obtained by cryoelectron microscopy (70). Bar, 200 Å (20 nm). Figure courtesy of R. J. Kuhn.

the genome (162, 312). It is possible to control for all these effects, however, and the results from many studies that rely on construction of mutants or of chimeric viruses are reported in detail in the various sections of this review. The alphaviruses now represent one of the best-studied of the animal virus groups, and new information is continually being reported on their evolution, the molecular biology of their replication, and their virulence. Within recent years, it has also become clear that the viruses interact with their hosts in a more intimate fashion than was previously suspected, and the necessity for the viruses to alternate between the mosquito vector and a vertebrate host has constrained their evolution in interesting ways. Studies of the interactions of the viruses with the host cell are leading alphavirologists into more detailed studies of cell biology, which are also important for our understanding of viral pathogenesis.

## STRUCTURE OF ALPHAVIRUSES

### Structure of the Virion

Alphaviruses contain an icosahedral nucleocapsid that is enveloped in a tight-fitting envelope whose glycoprotein components are present in an icosahedral lattice. The virion has a very regular structure, and crystals of SIN or SF have been obtained that diffract to about 3 nm, which implies a variability of no more than 0.5 to 0.8 nm in any part of the particle (191), and alphavirions have been described as icosahedral protein lattices with an associated membrane bilayer (391). The structures of SIN, SF, and RR have been solved to a resolution of ±3 nm by cryoelectron microscopy (70, 130, 391, 568). A reconstruction of the surface features of the RR virion is shown in Fig. 1 as a stereo pair. In Fig. 2 a reconstruction showing a cutaway view of the particle illustrates the surface lattice, the lipid bilayer, and the nucleocapsid within. Finally, the radial density distribution of the RR virion is shown in Fig. 3 at 14-Å (1.4-nm) intervals. The features illustrated by these three figures are described in detail below.

Nucleocapsid. The alphavirus nucleocapsid contains the single-stranded plus-sense RNA genome of approximately 11.7 kb complexed with multiple copies of a single species of capsid protein of about 30 kDa. It possesses icosahedral symmetry and a fenestrated structure such that the RNA within the

capsid is sensitive to RNase degradation. Early electron microscopy of isolated capsids suggested triangulation numbers (*T*) for the symmetry of the particle of 3, 4, or 9 (reviewed in references 189 and 529). Reconstruction of cryoelectron microscopic images of whole SIN virions by Fuller (130) suggested that *T* = 3. Subsequent work has clearly shown by several different methods that *T* = 4. First, the three-dimensional structure of the capsid protein of SIN has been solved to a resolution of 0.3 nm by X-ray diffraction (71, 546), and the dimensions of the folded capsid protein and of the nucleocapsid are compatible with a *T* = 4 particle containing 240 copies of the capsid protein but incompatible with a *T* = 3 structure made up of 180 copies of capsid protein. Second, electron microscopy of SIN capsids that were carefully prepared so as to prevent RNase degradation of the encapsidated RNA also led to the interpretation that *T* = 4 (77). Third, the mass of the SIN nucleocapsid was recently determined to be 10.9 MDa by scanning transmission electron microscopy, which predicts that 240 copies of capsid protein are present and therefore that *T* = 4 (392). Finally, recent reconstructions of SIN and RR virions from cryoelectron-microscopic images have clearly shown that for the nucleocapsid, *T* = 4 (70, 391) (Fig. 2 and 3). *T* = 4 means that there is a one-to-one relationship between capsid protein and envelope glycoproteins, which simplifies the interpretation of interactions that occur during virus assembly.

The reconstructions of RR in Fig. 2 and 3 clearly demonstrate the presence of five- and sixfold-coordinated subunits in the nucleocapsid arranged in a *T* = 4 icosahedral array. In these images the individual capsid protein subunits are visible and their interactions to form pentamers or hexamers can be seen. It is notable that these images give no sign of the presence of dimers of the nucleocapsid protein in the capsid, and the dimers observed in the crystal structure of the SIN capsid protein (71, 546) are thought not to be present in the capsid (70).

The diameter of the SIN nucleocapsid is 41 nm as determined by X-ray scattering by intact virus particles (189, 533), 40.4 nm as determined by X-ray diffraction of nucleocapsid crystals (191), or 34 to 38 nm as seen in the electron microscope after isolation from virions (77). Isolated nucleocapsids are stable and have a sedimentation coefficient of 140S. Electrostatic interactions are important for the stability of the

FIG. 2. Cutaway surface representation of RR revealing the multilayered structure of the virion. The envelope glycoproteins appear in the outer layer as thumb-like projections displayed in grey, while the inner layer of nucleocapsid protein is shown in blue. The inner and outer borders of the lipid bilayer membrane are displayed in orange and probably represent the polar head groups of the bilayer membrane. Interactions of the lipid bilayer membrane and the nucleocapsid are clearly visible. Figure courtesy of R. J. Kuhn.

capsid, and presumably for its assembly as well, because high salt concentrations (≥1 M) cause the capsid to disassemble (79), suggesting that the highly basic N-terminal domain of the capsid protein binds the viral RNA.

Virion envelope. The virion envelope consists of a lipid bilayer in which are embedded multiple copies of two virus-encoded glycoproteins (reviewed in references 189 and 529). The lipid bilayer is composed of lipids derived from the plasma membrane of the host cell, and its composition closely approximates that of the host plasma membrane. It has a thickness of 4.8 nm and is centered at a radius of 23.2 nm in SIN. Because of the high curvature of the bilayer, the outer leaflet has an area 40% larger than the inner leaflet.

The two glycoproteins, E1 and E2, each have a molecular mass of about 50 kDa and are anchored in the membrane by conventional membrane-spanning anchors in the C-terminal regions (135, 141, 189, 416, 459). E2 is 423 amino acids long in SIN and has a 33-residue cytoplasmic domain following the anchor. E1 is 439 residues long in SIN and has a cytoplasmic domain of only 2 residues. Both E1 and E2 are glycosylated,

FIG. 3. Projections of RR electron density at different radii. The RR image reconstruction is presented as projected views of the electron density truncated progressively toward lower radii starting from the outer surface of the envelope glycoproteins (345 Å) down to the capsomer region of the nucleocapsid (205 through 191 Å). The density shells move toward lower radii in increments of 14 Å. All representations are viewed along a radial direction midway between the axes of the pentavalent (toward the top) and hexavalent (toward the bottom) capsomer axes. Highest density is indicated by yellow, and low density is indicated by red. Figure courtesy of R. J. Kuhn.

but the number and position of the attached chains are not absolutely conserved among alphaviruses. E1 usually carries one or two chains, whereas E2 usually carries two or three chains. Both E1 and E2 of SIN and of SF also have covalently attached palmitic acids in or near the membrane-spanning anchors (474, 475; reviewed in reference 459), one attached to E1 and four to six attached to E2. Palmitoylation of the glycoproteins of other alphaviruses has not been studied.

E2 and E1 form a stable heterodimer that remains intact upon dissociation of the virus with mild detergents, and three E2-E1 heterodimers interact to form the spike that is found on the virus surface (420, 627; reviewed in reference 189). Cross-linking studies suggest that the trimers are maintained by E1-E1 interactions (7), a result consistent with the finding that upon fusion of the viral envelope with a cell membrane during viral infection, the E1-E2 heterodimer disassembles to give rise to E2 monomers and E1 homotrimers (575). In the spike the three heterodimers twist around one another in an anticlockwise direction to form the stalk of the spike and then separate to form a tripartite head (Fig. 1). At the tip of each heterodimer, E1 and E2 appear to separate (Fig. 1) (269). A model for the relative locations of E1 and E2 in the spike that was based on differential cryoelectron-microscopic reconstructions of untreated and detergent-treated particles has been presented (567). The glycoprotein spikes extend to a radius of about 34.5 nm, so that the total diameter of the virion is 69 nm (190, 391). The glycoproteins of the virus form an icosahedral lattice with \( T = 4 \) (130, 391, 568, 570), and there are 240 heterodimers on the surface of the virus, assembled into 80 spikes.

The trimeric nature of the spikes is clearly seen in the radial density distribution maps in Fig. 3, as are capsid-glycoprotein interactions that are believed to result in the precise 1:1 ratio of capsid protein to envelope protein. Proceeding from the inside out, the sections at 191 and 205 Å show the capsid protein capsomers. The next three sections are in the region of the lipid bilayer, in which only light density can be seen where the membrane-spanning domains of the glycoproteins traverse it. The glycoproteins emerge from the lipid bilayer precisely over the capsid protein subunits, consistent with a one-to-one interaction between a glycoprotein heterodimer and a capsid protein subunit. These interactions are described in more detail below. At this point the glycoproteins exhibit the same icosahedral lattice as the capsid and the trimeric structure is not evident (sections at 261 and 275 Å). As they extend upward, the glycoproteins take a sharp bend, seen in the section at 289 Å, and form trimeric assemblies that are positioned between capsid subunits rather than over them (sections beyond 303 Å). The heterodimers twist around one another as the stalk elongates and then separate.

E2 is formed as a precursor called PE2 or p62 that is cleaved into E2 and a small glycoprotein called E3. E3 remains associated with the virion in some cases (e.g., SF) but not in others (e.g., SIN). There is also a small hydrophobic peptide

nsP1 nsP2 nsP3 nsP4  
(Plus strand RNA Replicase)  

P123 nsP4  
(Minus strand RNA Replicase)  

P1234  
Nonstructural  
Proteins  

nsP1 nsP2 nsP3  
nsP2 Proteinase  
P123  
Translation  

5' CAP-H Nonstructural ORF 26 S mRNA Aₙ 3'  
Readthrough  
Translation  
Replication  
Transcription  

CAP Structural ORF Aₙ  
Translation  

p130  
Capsid Autoprotease  
Signalase  

Structural  
Proteins  

Capsid PE2 6K E1  
Furin  

E3 E2  

FIG. 4. Genome organization of SIN. The 49S genomic RNA is illustrated schematically in the center, with its translated ORF shown as an open box. Small black boxes are conserved sequence elements; the open diamond denotes the leaky opal termination codon. The nonstructural polyproteins and their processed products are shown above. Termination at the opal codon produces P123, whose major function in replication is believed to be as a proteinase that acts in trans to process the polyproteins in active RNA replicases; this proteinase domain is found in the nsP2 region. Readthrough of the opal stop codon produces P1234, which can form an active replicase. The 26S subgenomic mRNA is expanded below to show the structural ORF and its translation products. Polypeptides present in the virion are shaded. vcRNA is the minus-strand complement of the genomic RNA.

that is produced as a linker between E2 and E1, called the 6K protein, that has been found to be associated with the virus in submolar quantities (7 to 30 molecules per virion) (133, 317). From its composition (reviewed in reference 529), the SIN virion has a molecular mass of 52 MDa and is composed of 29% lipid, 57% protein, 6% carbohydrate, and 7% RNA. It has a sedimentation coefficient of 280S and a density of 1.22 g/ml in sucrose-D₂O solutions.

There are conflicting reports of whether the envelope of the virus is permeable to hydrogen ions. Isolated nucleocapsids of SF have been found to undergo a marked contraction on treatment at low pH (497); treatment of intact SF virions at low pH also led to the contraction of the nucleocapsid, as determined by examination of nucleocapsids isolated from treated virions (458). It was suggested that the viral envelope proteins function as ion channels that allow penetration of H⁺ ions. In contrast, treatment of intact SIN virions at pH 5 was found not to affect the diameter of the capsid, as determined by X-ray solution scattering by the intact particles, although the glycoprotein spikes did undergo a conformational rearrangement such that their length increased by 40 Å (4 nm) and they now extended to a radius of 390 Å (39 nm) (533).

Organization of the Alphavirus Genome

The organization and expression of the genome of the type alphavirus SIN are illustrated in Fig. 4, and comparisons of the different regions of the SIN genome with those of other alphaviruses are shown in Table 2. The SIN genomic RNA is 11,703 nucleotides (nt) in length exclusive of the 5′-terminal cap and a 3′-terminal poly(A) tract (reviewed in reference 524). It is divided into two major regions: a nonstructural domain encoding the nonstructural or replicase proteins, which forms the 5′-terminal two-thirds of the RNA, and a structural domain encoding the three structural proteins of the virus, which forms the 3′-terminal one-third. The nonstructural proteins are translated as one or two polyproteins from the genomic RNA itself. These polyproteins are cleaved to produce nsP1, nsP2, nsP3, and nsP4, as well as a number of cleavage intermediates that possess important functions distinct from the final products. In most sequenced alphaviruses a termination codon (UGA) is present following nsP3 and two polyproteins (P123 and P1234) are produced, whereas in some alphaviruses this codon has been replaced by a sense codon and only P1234 is produced. The structural domain is translated as a polyprotein from a subgenomic mRNA, the 26S mRNA (4,103 nt in SIN). 26S RNA is also capped and

TABLE 2. Genome organization and translation products of alphaviruses

| Virus (total nt) | 5'NT (nt) | Nonstructural region (aa) | Structural region (aa) |
|------------------|-----------|--------------------------|------------------------|
|                  |           | nsP1 | nsP2 | nsP3 | Jun<sup>a</sup> | nsP4 | C | E3 | E2 | 6K | E1 | 3'NT<sup>a</sup> (nt) |
|                  |           |      |      | Cons./noncons.<sup>b</sup> | (nt) |      |    |    |    |    |    |                 |
|                  |           |      |      | Total<sup>c</sup> |       |      |    |    |    |    |    |                 |
| SIN (11,703)     | 59        | 540   | 807  | 325/231 | 48   | 610   | 264  | 64  | 423 | 55  | 439 | 322               |
| AURA (11,813)    | 66        | 539   | 806  | 325/219 | 53   | 610   | 267  | 61  | 424 | 54  | 438 | 465               |
| WHA (11,586)     | 60        | 540   | 807  | 325/189 | 52   | 610   | 266  | 63  | 423 | 55  | 439 | 323               |
| EEE (11,675)     | 46        | 532   | 794  | 324/235 | 66   | 608   | 260  | 63  | 420 | 57  | 441 | 361               |
| VEE (11,444)     | 44        | 535   | 794  | 329/228 | 38   | 607   | 275  | 59  | 423 | 55  | 442 | 121               |
| ONN (11,835)     | 79        | 535   | 798  | 324/246 | 48   | 611   | 260  | 64  | 423 | 61  | 439 | 425               |
| SF (11,442)      | 85        | 537   | 798  | 324/158 | 41   | 614   | 267  | 66  | 422 | 60  | 438 | 264               |
| RR (11,851)      | 78        | 533   | 798  | 324/214 | 47   | 611   | 270  | 64  | 422 | 60  | 438 | 524               |

<sup>a</sup> Defined as nontranslated nucleotides only; the termination codons of both the nonstructural and structural ORFs are considered to be nontranslated.

<sup>b</sup> Cons./noncons., conserved/nonconserved.

<sup>c</sup> Length of nsP3 to the nsP3/nsP4 cleavage site, not to the stop codon; nsP3 terminating at the stop codon will be 7 amino acids (aa) shorter.

within the active site (Fig. 6). Thus, once the capsid protein has cleaved itself, the proteinase is no longer active because the active site is occupied.

The capsid protein has an overall size that varies from 258 residues in MAY to 275 residues in VEE and is divided into two distinct domains. The amino-terminal 96 residues (SIN numbering) are not conserved as to sequence among alphaviruses (84, 419) and are not ordered in crystals (71). This domain is rich in lysine and arginine and probably binds electrostatically to the RNA. There are 30 positively charged residues (K or R) and only one negatively charged residue (E) in the first 108 amino acids of the SIN capsid; 240 capsid monomers in the nucleocapsid contain a net positive charge in this N-terminal region sufficient to neutralize about 60% of the negative charges on the viral RNA. The presence of 20 proline residues in the first 100 amino acids, together with the large positive charge, suggests that this domain has little structure and protrudes into the interior of the nucleocapsid, where it interacts electrostatically with the RNA.

In contrast, the domain from residue 97 to the C terminus is highly conserved in sequence among alphaviruses and residues C-terminal to residue 114 are ordered in crystals (Fig. 6). In addition to its function as a proteinase, this domain probably contains regions that promote capsid protein-capsid protein interactions during nucleocapsid assembly and regions that lead to interaction with the cytoplasmic domains of the glycoproteins during budding.

There are regions in the capsid protein that bind specifically to RNAs. Newly synthesized capsid protein is bound by the large ribosomal subunit, and this binding reaction could be active in disassembling nucleocapsids upon entry into the cell (reviewed in reference 490). Wengler et al. (612) have presented evidence that the sequence KPGKRQRMALKLEAD (positions 99 to 113) in the SIN capsid protein binds to ribosomes. This element is conserved in alphavirus capsid proteins (7 of 15 residues are invariant in 10 sequenced alphaviruses) and is predicted to be present on the surface of the capsid (71), where it acts as a linker to join the basic N terminus in the interior of the capsid with the conserved and ordered C terminus.

Capsid protein also binds specifically to the viral genomic RNA, and this binding is believed to be important for the initiation of nucleocapsid formation and possibly for the stimulation of RNA synthesis (599). Deletion analysis of the capsid protein identified a region of 32 amino acids (residues 76 to 107) that was essential for specific binding and a region of 68 amino acids (residues 1 to 10 and 75 to 132) that possessed almost full binding activity (148). Thus, the genomic RNA-binding region overlaps the region required for binding to the large ribosomal subunit.

Synthesis of glycoproteins. A schematic overview of the synthesis and processing of the glycoproteins is shown in Fig. 7. Once the nucleocapsid protein is released from the nascent polypeptide chain, an N-terminal signal sequence functions in a conventional manner and leads to the insertion of glycoprotein PE2 into the endoplasmic reticulum (ER). Translation in cell-free systems has shown that microsomal membranes are required for insertion and processing of the envelope proteins (38, 142), that signal recognition particles are required for this process (39), and that the signal sequence is not removed (37). The signal sequence has not been mapped precisely, but the N-terminal 40 residues of PE2 have been shown to act as a signal for translocation of two different reporter molecules (139), and the signal is thought to be contained within the first 19 residues. The signal sequence has a carbohydrate attachment site at an asparagine residue between residues 11 and 14

FIG. 5. Growth curve of SIN in chicken embryo fibroblasts at 30°C. (A) Release of infectious virus into the culture fluid. At the times shown, the medium was harvested and replaced with fresh medium, and the titer was determined by plaque assay on monolayers of chicken cells. (B) Cells infected as in panel A were pulsed with radioactive uridine for 1 h at the times specified. Symbols: ○, total radioactivity incorporated into acid-insoluble form; ●, incorporation into minus-strand RNA (note that the scale is 10-fold less than that for total RNA). Adapted from reference 524 with the permission of the publisher.

studies have shown that the catalytic triad forming the active site of the SIN capsid proteinase consists of H-141, D-163, and S-215 (reviewed in reference 531). Boege et al. (31) first suggested that S-215 formed part of the active site of a serine proteinase because of the context surrounding this residue. Mapping of mutations that rendered the proteinase temperature sensitive led to the hypothesis that S-215, H-141, and either D-147 or D-163 formed the catalytic triad (179, 532). Site-specific mutagenesis experiments confirmed the importance of S-215 (180, 345) and H-141 (180) but were unable to identify an active-site aspartic acid. It is of interest that enzymes with C-215 or T-215 retained partial activity, whereas enzymes with A-215 or I-215 were inactive (180). It was hypothesized that the very rapid kinetics of this autoproteolytic event led to residual activity with these particular amino acids in the active site of the enzyme.

Protein-modeling studies suggested that D-163 formed the third component of the SIN catalytic triad (156). This was confirmed from the three-dimensional structure of the capsid protein (71, 546) which revealed that the SIN capsid protein C-terminal to residue 114 has a fold homologous to that of chymotrypsin, with S-215, H-141, and D-163 forming the catalytic triad and W-264, the C-terminal residue, present

FIG. 6. Three-dimensional structure of the SIN capsid protein (residues 114 to 264). The ribbon structure of the SIN capsid protein as determined by Choi et al. (71) is shown with β-strands represented by the large arrows. The fold of the protein is similar to that of chymotrypsin. R-114 is the N terminus of the ordered region of the capsid protein, and W-264 is the C terminus of the capsid protein; these residues are boxed. The residues that form the catalytic triad of the serine protease (S-215, H-141, and D-163) are shown in underlined italics, and the positions in the structure are indicated by the open diamonds; note that W-264 is present within the active site. The seven residues that have changed from the EEE sequence to the SIN sequence in WEE (M-137, N-172, G-201, L-231, A-234, T-238, and I-254) are also indicated by solid circles. These seven residues form a roughly linear trace across the capsid protein and are proposed to interact with the C-terminal cytoplasmic domain of glycoprotein E2 during virus assembly. Figure courtesy of R. J. Kuhn.

in all alphaviruses examined, and it has been hypothesized that addition of carbohydrate to this site may be responsible for the retention of the signal sequence in PE2. The carbohydrate chain is added early, while the polypeptide is nascent, suggesting that the N terminus of PE2 is released from the ER membrane into the lumen very soon after translation and thus that the signal sequence plays no role in further translocation of the protein (139, 480).

Near the C terminus of PE2 is a hydrophobic stretch that functions as a stop transfer signal and membrane anchor for E2 (Fig. 8). In common with other such signals, the luminal and cytoplasmic boundaries are marked by charged residues in

FIG. 7. Model for the production of glycoproteins PE2 and E1. Stop-transfer signals are indicated by cylinders, and signal sequences are indicated by rectangular blocks. Stalked symbols show the locations of carbohydrate chains, and the proteolytic cleavage sites are indicated by arrows (↓, signalase cleavage site; ↓, furin cleavage site). The model is not drawn to scale, in order to emphasize certain features.

FIG. 8. The hydrophobic membrane anchor of alphavirus E2 glycoproteins. The 26 residues which constitute the anchor domain are shown, numbered from the N terminus of E2. Charged residues are shaded: gray, basic amino acids; diagonal lines, acidic amino acids. Note that the cytoplasmic face is positively charged with respect to the luminal face of the anchor. Boxed residues in WEE and EEE are those in which the WEE amino acid differs from the SIN residue but is identical to the EEE residue (see text).

which the cytoplasmic charge cluster is positive with respect to
the luminal boundary (25). Site-specific mutagenesis of the
charge cluster at the cytoplasmic boundary of SF PE2 showed
that changing the charge from +2 to 0 or to –1 did not change
the topology of PE2 in the membrane or inhibit the subsequent
transport of the protein to the plasma membrane (82, 83).
These changes did lower the stability of the anchoring of the
protein in cellular membranes, however, as the mutant pro-
teins were more readily released from cell membranes upon
washing with alkaline sodium carbonate.

Following the anchor sequence, PE2 contains 33 residues (in
SIN) which act as a signal for insertion of the following
peptide, referred to as 6K (Fig. 7). The central part of this
sequence is uncharged and traverses the membrane such that
the N terminus of 6K is external. Liljeström and Garoff (301)
have shown that if 6K is deleted, the signal at the end of E2
leads to the translocation of E1, and thus that this signal
sequence can act independently of the 6K sequence. The
cleavage between PE2 and 6K is almost certainly effected by
signalase in the lumen of the ER since the site cleaved is
characteristic of signalase sites, cleavage does not require
expression of nonstructural proteins or of the capsid protein,
and mutations that alter the C-terminal alanine of PE2 abolish
cleavage in a fashion consistent with signalase recognition of
the site (301, 345, 524). Cleavage by signalase signifies that the
C terminus of PE2 is present at least transiently in the lumen
of the ER.

The C-terminal domain of the 6K polypeptide acts as a
signal sequence for the translocation of E1. Translocation of
PE2 is not required for this sequence to function as an internal
signal for the translocation of E1, and this sequence will also
function as an N-terminal signal for the translocation of E1 or
of a recorder gene (193, 301, 344, 355). Cleavage between 6K
and E1 is, again, almost certainly effected by signalase (301,
345, 524). Insertion of 15 amino acids into the SIN 6K protein
following F-29 was found to interfere with proteolytic process-
ing of the glycoprotein precursor and, as a result, with trans-
port of the glycoproteins, leading to decreased amounts of
mature E2 and E1 in the cell (462).

Synthesis of glycoprotein E1 continues until a stop codon is
reached (Fig. 7). Just upstream of the stop codon is a stop-
transfer sequence that acts as an anchor for E1. This stop-
transfer signal also follows the rule that the cytoplasmic
boundary is more positively charged than the luminal bound-
ary, and in most alphaviruses, the cytoplasmic domain of E1
consists of only two arginine residues. Deletion of the cyto-
plasmic domain or substitution of the arginine residues by
neutral amino acids did not affect the topology of E1 or the
subsequent transport of the protein to the cell surface (21).
Thus, both PE2 and E1 are type I integral membrane proteins
that have a membrane-spanning anchor at or near the C
terminus (135, 141, 189, 416, 459).

Addition of carbohydrate and lipids. The addition of carbo-
hydrate chains to alphavirus PE2 and E1 has been extensively
studied; high-mannose chains are added en bloc to the nascent
proteins in the lumen of the ER, and the carbohydrate chains
are subsequently modified during transit through the Golgi
apparatus (reviewed in reference 459). It appears that all
potential sites in the ectodomains of alphavirus glycoproteins
are glycosylated and carry either complex or simple oligosac-
charide chains. It has been postulated that if a simple oligo-
saccharide chain is accessible to cellular processing enzymes
during transport through the Golgi apparatus, the chain is
modified to the complex type, whereas if the folding of the
protein renders the chain inaccessible, it remains simple (215).
Both PE2 and E1 are glycosylated in all alphaviruses stud-

ied, but the number, position, and type of the attached chains
are not absolutely conserved among different alphaviruses. In
most strains of SIN, for example, both E1 (N-139 and N-245)
and E2 (N-196 and N-318) have two N-linked carbohydrate
chains, whereas SF E1 carries one chain (N141) and SF E2
carries two (N200 and N262). For SIN, the N-terminal chains
of both E1 and E2 are complex whereas the C-terminal chain
of E1 can be either complex or simple depending upon the host
cell (214, 340) and the C-terminal chain of E2 is simple. A SIN
strain that has three chains in E2 and that appears to grow
normally in cultured cells has been isolated (59); from this
variant, another variant that had four E2 chains was selected by
passage in mosquito cells (105, 107). The variant with four E2
chains grew well in mosquito cells but was temperature sensi-
tive in vertebrate cells. As another example of variability, a
variant of SIN selected to be resistant to a particular mono-
clonal antibody was found to have acquired resistance by
addition of a new carbohydrate chain on E2 (520).

Palmitoylation of the glycoproteins of SIN and of SF (459,
474, 475) occurs in a compartment reached approximately 15
min after completion of the polypeptide chain and is catalyzed
by a protein acyltransferase that uses palmitoyl coenzyme A as
a donor (27, 476). It has been shown that palmitoylation of
PE2 in vertebrate cells occurs after exit from the ER but
probably before arrival in the cis Golgi (40). However, palmi-
toylation in mosquito cells was found to occur primarily on E2
rather than on PE2 (454). This difference probably results
because the cleavage of PE2 occurs in an early compartment in
mosquito cells but in a late compartment in vertebrate cells.

The one palmitic acid in SF E1 was shown by direct
biochemical analysis to be attached to C-433 which is present
in the transmembrane segment at a location corresponding to
the middle of the inner leaflet of the bilayer (473). The one
palmitic acid chain attached to SIN E1 is presumed to be
attached to C-430 (the only cysteine in the region). Interest-
ingly, two alphaviruses, EEE and VEE, have no cysteine
residue in this region of E1 and are presumably not palmitoy-
lated, although this has not been examined directly.

E2 carries four to six palmitic acid chains in SIN and
probably the same number in SF. Site-specific mutagenesis has
shown that C-396, C-416, and C-417 are palmitoylated in SIN
E2 (134, 221). All three sites are in the cytoplasmic domain of
E2 (see below), and these three cysteines are conserved in all
alphaviruses sequenced to date. In addition, there is at least
one palmitic acid residue attached to the membrane anchor of
E2, possibly in a location similar to the E1 site, as shown by the
fact that E2 is palmitoylated even when the cytoplasmic tail is
deleted (432).

The 6K protein is also palmitoylated. The SIN 6K protein is
55 residues in length, and site-specific mutagenesis has shown
that C-35, C-36, C-39, and probably one other cysteine are
palmitoylated (132, 133). Mutagenesis to reduce the amount of
palmitoylation did not interfere with the transport of the
glycoproteins to the cell surface, but virus budding was affected
(132).

Folding and transport of glycoproteins. The folding and
oligomerization of viral glycoproteins have been studied in a
number of virus systems (reviewed in reference 99), and the
available data for alphaviruses suggest that the details of
folding and oligomerization are similar to those for other
viruses (325, 365). Folding of viral glycoproteins normally
begins immediately upon entry into the ER, while the protein
is nascent, and requires molecular chaperones, folding en-
zymes, energy, and the formation of disulfide bonds (44).
Furthermore, the addition of carbohydrate chains is usually
required for proper folding; it is believed that the function of

the carbohydrates at this stage is to increase the solubility of the protein and prevent aggregation or side reactions from occurring. It has been shown that inhibitors of glycosylation interfere with the transport and function of the proteins in SIN (341, 464), presumably because they fail to fold properly. Once folded, the proteins oligomerize in a specific fashion. Proteins that fail to fold properly are not transported out of the ER, usually aggregate into nonspecific complexes, and are slowly degraded. While in the ER, misfolded proteins may be rescued and induced to fold properly as a result of the activities of folding enzymes and chaperones; such rescue may be dramatic when conditions are changed, such as a shift from a nonpermissive to a permissive condition for a ts mutant (442).

Both PE2-E1 heterodimers and E2-E1 heterodimers have been demonstrated in the cell (235, 420, 628). Heterodimers can form in trans from proteins translated from two different mRNAs (309), consistent with early observations that ts defects in E1 and E2 are complementable (reviewed in reference 521) and with our general understanding that folding of protein monomers into their proper conformation must occur before oligomerization can occur (99). PE2-E1 heterodimers are much more stable than E2-E1 heterodimers. Wahlberg et al. (574) found that PE2-E1 heterodimers remain associated after exposure to pH 5.8, whereas the E2-E1 heterodimers are dissociated by exposure to pH 6.4. Related studies have shown that alterations in the conformation of E1 and E2 in the dimers that occur upon exposure to acidic pH require much lower pH for PE2-E1 dimers than for E2-E1 dimers; these changes in conformation are manifested by the exposure of new antigenic epitopes, by an alteration in the susceptibility of the glycoproteins to proteases, or by the activation of the fusion activity presumed to be present in E1 (224, 252, 308). Furthermore, Lobigs et al. (309) found that E2-E1 dimers did not form, or were of such low stability they could not be demonstrated, when E2 was produced as E2 rather than as PE2 (in this experiment a synthetic signal peptide replaced E3 and was removed by signalase after insertion). Thus, dimerization is a process that normally requires interaction of PE2 and E1.

After formation, the PE2-E1 dimer is transported to the Golgi apparatus and ultimately to the cell surface. It is unknown whether the unit transported is a single PE2-E1 heterodimer or a trimerized spike unit. The structure and function of the Golgi apparatus and the transport of proteins through it have been intensively studied (reviewed in reference 346). Where studied, the transport of alphavirus glycoproteins is similar in detail to that of other glycoproteins, and the alphavirus glycoproteins have been used as probes for Golgi function (91, 274, 442, 443).

Efficient transport of alphavirus glycoproteins beyond the ER normally requires dimerization (and possibly higher-order oligomerization) of the glycoproteins but does not require expression of capsid protein or of nonstructural proteins (140, 220, 264, 265, 309, 417, 423, 432, 603). Monomeric E2 of SF was transported inefficiently but detectably to the cell surface, whereas monomeric E1 was not detectably transported beyond the ER. In contrast, SIN E1 was found to be efficiently transported to the cell surface in the absence of PE2 or E2 (355), and the necessity for oligomerization may depend upon the virus.

Removal or alteration of the cytoplasmic domain of SF PE2 or of the intramembrane region was found not to prevent the transport of the protein to the cell surface (140, 423) or its polar localization at the basolateral surface of polarized cells (432), indicating that whatever signals exist for transport do not lie in the C-terminal region of the protein. Similarly, removal or replacement of the two Arg residues that constitute

the cytoplasmic domain of E1 had only a minor effect on transport of the complex (21). Deletion of the 6K protein, which is normally transported in association with the glycoprotein complex (317), also did not interfere with the transport of the glycoprotein complex to the cell surface (303).

A number of ts mutations of both SF (444) and SIN (9), which interfere with the transport of the glycoproteins, have been described. These mutations lead to reversible defects in which transport of the proteins is inhibited at the nonpermissive temperature, and the proteins accumulate in the ER or in the Golgi apparatus. Upon shift to the permissive temperature, transport of the previously formed proteins occurs. SIN mutations causing such a phenotype were mapped to changes at E1 residues 106, 176, and 267 in the ectodomain of the protein. It is assumed that the protein fails to fold properly at the nonpermissive temperature and that this leads to the failure to transport the protein complex; the misfolded protein is present in large aggregates (420).

Following transport through the Golgi complex, the glycoproteins are transported from the trans Golgi network to the cell plasma membrane, where they are used to assemble viruses. In polarized cells, sorting of proteins to the apical or basolateral surface occurs at this stage (91, 131, 338, 339, 346), and the transport of alphavirus glycoproteins to the cell surface is polarized. During SIN or SF infection of the FRT line of thyroid epithelial cells, the glycoproteins accumulate at the apical surface (and virus budding occurs only at the apical surface in such cells) (374, 629). However, following SF infection of Madin-Darby canine kidney cells (432) or SIN or SF infection of CaCo-2 cells (629), the glycoproteins are transported to the basolateral surface (and budding occurs from this surface). It has also been found that in SF-infected hippocampal cells in culture the viral glycoproteins were present exclusively in dendrites and cell body but not in axons (100). Differences in the surface used for virus budding in different tissues could be of importance in the spread of the virus in the organism.

Cleavage of PE2 during transport. During transport of the PE2-E1 complex, PE2 is cleaved to form E2 and E3. Cleavage occurs after the consensus sequence R-X-(K/R)-R↓, where X is any amino acid and cleavage occurs at the arrow (reviewed in reference 523). Cleavage of PE2 is effected by a host cell proteinase and is a late event in vertebrate cells, occurring after the proteins reach the trans Golgi network but before arrival at the cell surface (91), but has been reported to be an early and continuous event in mosquito cells (366, 454). There is now known to exist a family of cellular proteinases that cleave after basic residues and process a large number of cellular proproteins including, among others, insulin, pro-opiomelanocortin, and the insulin receptor precursor. These enzymes, which are Ca²⁺-dependent serine proteinases related to bacterial subtilisin, include furin (also called PACE), PC2, PC3-PC1, and PACE 4 in mammals and kex2 protease in Saccharomyces cerevisiae (reviewed in references 19, 129, 492, and 506). PC2 and PC3 are restricted to neuroendocrine cells, lack a membrane anchor, and process insulin and pro-opiomelanocortin, among others, only after packaging of the proproteins into secretory vesicles (16, 383, 493, 547). They prefer to cleave following the sequence (K/R)-R↓ (see, for example, reference 97). In contrast, furin has a wide tissue distribution, has a membrane anchor, is localized in the Golgi apparatus, and has a cleavage site preference for the sequence R-X-(K/R)-R↓. This enzyme is thought to cleave a number of proproteins as they traverse the Golgi apparatus, such as the insulin receptor precursor, which is cleaved following the sequence RKRR; mutation to RKRS leads to failure to cleave

the receptor precursor and to insulin-resistant diabetes in humans (624). Because of its properties, furin is believed to be the enzyme that cleaves alphavirus PE2 as well as glycoprotein precursors of several other virus families.

Although the cleavage of PE2 is normally extremely efficient and only E2 is found in virus particles, uncleaved PE2 can be efficiently incorporated into virus particles, at least when virus is grown in vertebrate cells, showing that cleavage of PE2 is not required for transport of the glycoprotein dimer to the cell surface or for virus assembly. Several approaches have been devised in which cleavage of PE2 was blocked, and such experiments routinely produced yields of PE2-containing particles that were similar to the yields of E2-containing particles in control experiments, although the specific infectivity of the PE2-containing particles was usually low. Russell et al. (437) isolated a mutant of strain AR89 of SIN in which no cleavage of PE2 occurred. PE2-containing particles were produced in high yield when the virus was grown in vertebrate cells but not when it was grown in mosquito cells. In the case of SIN AR339, treatments were devised that resulted in the partial incorporation of PE2 into virus (410), but with this approach it was not possible to completely replace E2 with PE2. Watson et al. (585) isolated a strain of CHO cells that was resistant to *Pseudomonas* exotoxin A, which requires cleavage after basic residues for activation. This cell line was defective in the proteinase that cleaves alphavirus PE2; SIN infection of these cells led to the production of high yields of virus particles that contained only uncleaved PE2. Dubuisson and Rice (102) studied SIN variants containing random insertions in PE2 or E1 and identified variants in which an insertion near the PE2 cleavage site led to a failure to cleave PE2; uncleaved PE2 was efficiently incorporated in virions. Finally, in the case of SF, cleavage-defective mutants of PE2 were constructed by changing the conserved arginine in the P1 position to lysine, leucine, or glutamic acid (308) or to phenylalanine (224). Uncleaved PE2 was efficiently transported and incorporated into virus particles (440).

Cleavage of PE2, which changes the stability of the glycoprotein dimer, is normally required to activate the virus for infection of a cell, and in almost all cases cited above in which PE2 completely replaced E2 in virus particles, the specific infectivity of the particles was quite low. When the cleavage-defective SF E2 glycoproteins were expressed by using vaccinia virus or simian virus 40 vectors, uncleaved PE2 was expressed at the cell surface in PE2-E1 dimers. No fusion of cells (by E1) occurred in cells transfected with cleavage-defective PE2 after exposure to pH 5.5, the optimal pH of fusion for the wild-type virus; instead, exposure to pH ≤ 5.0 was required for fusion, and exposure to this low pH led to dissociation of PE2-E1 dimers and alteration in trypsin sensitivity of the glycoproteins (310). SF virions containing PE2 (produced by introducing a cleavage mutation into the virus genome) were essentially noninfectious (440). SF virus with uncleaved PE2 bound to cells at pH 7.3 only one-fourth as well as did E2-containing virus, virus once bound was internalized only one-fifth as well, and internalized virus did not undergo the pH-dependent changes in E1 associated with fusion. Infectivity of the PE2-containing virions could be activated by trypsin treatment, which led to the cleavage of PE2 to E2, or by exposure of cells with bound virus to a pH of 4.5. Similarly, SIN containing uncleaved PE2 because it was grown in the mutant CHO cells (585) or because the virus contained an insertion preventing PE2 cleavage (102) was essentially noninfectious and was defective in binding to vertebrate cells; infectivity could be rescued by treating the virus with trypsin. The ability of uncleaved PE2 at the cell surface or in virus to be cleaved by

exogenously added proteinases to produce a protein very similar in size to E2, as well as the fact that this cleavage activates the heterodimer for fusion and infectivity, suggests that the sequence near the E3-E2 junction must be exposed in a form that makes it readily susceptible to proteolytic attack. Thus, the precise cleavage at this junction by furin must result not only from the specificity of the enzyme for its cleavage site but also from the exposure of this region to attack.

Although cleavage of PE2 is normally required to activate the infectivity of alphaviruses, it is possible to obtain virus with suppressor mutations in E3 or E2 that allow PE2-containing virus to retain infectivity. SIN AR86 and SIN AR339 that contain only PE2 are not infectious, but suppressor mutations that render PE2-containing virus infectious arise readily; all such suppressor mutations mapped to date lie in E3 or E2 (194, 437). It is probable that the compensating mutations allow PE2-containing virus to be infectious by destabilizing the PE2-E1 heterodimer. In addition, SIN AR339 in which PE2 partially replaced E2 in the virion was fully infectious (410), although in this case the remaining E2 in the virus could have activated the virus infectivity.

During transport to the cell surface, alphavirus glycoproteins are potentially exposed to slightly acidic pH which could trigger the low-pH-induced changes in the E2-E1 heterodimer, thereby inactivating it. The greater stability of the PE2-E1 heterodimer supports a model in which the stable PE2-E1 heterodimers are transported through the cell and the cleavage of PE2 just before arrival at the cell surface activates virus infectivity.

Although different in detail, the PE2 cleavage activation event is formally similar to those that occur in a number of other enveloped viruses in which glycoprotein precursors must be cleaved to activate virus infectivity. For flaviviruses, retroviruses, myxoviruses, and paramyxoviruses, cleavage occurs after a canonical sequence that is similar or identical to the cleavage sequence in alphavirus PE2, suggesting that these cleavages are all effected by the same cellular enzyme (reviewed in references 210, 523, and 618).

Reorientation of the E2 C-terminal domain. The C terminus of PE2 is thought to lie in the lumen of the ER when first synthesized (Fig. 7), with subsequent reorientation of the C-terminal region into the cytoplasm being required for virus budding (141, 221, 301, 419). A schematic of this reorientation is shown in Fig. 9. Liu and Brown (306) have recently presented evidence that the reorientation takes place during transport of the protein complex. A mutant was constructed with Y-420 in place of S-420 in E2, just upstream of the C-terminal A-423. Cells were infected with the mutant, and membrane vesicles were prepared by homogenization; such vesicles have the cytoplasmic side as the outside of the vesicle and the luminal side inside. The authors found that Y-420 could not be iodinated when PE2 was first synthesized and concluded that at this time it was protected because it was within the lipid bilayer. At some point during transport of the complex, after it had left the ER, this tyrosine became susceptible to iodination, showing that the E2 tail had reoriented to the cytoplasm.

In a second set of experiments, Liu and Brown (307) presented evidence that reorientation of the PE2 tail was associated with phosphorylation. In the presence of inhibitors of phosphorylation, when no phosphorylation of PE2 or of E2 was detectable, glycoproteins were synthesized and transported to the plasma membrane and PE2 was cleaved to E2. However, nucleocapsids were randomly distributed in the cytoplasm, and under these conditions there was no specific interaction between nucleocapsids and glycoproteins in the cell

VOL. 58, 1994

![Diagram](#)

FIG. 9. Schematic model of the configuration of glycoprotein PE2 (E3 + E2) in membranes. The upper panel shows the configuration of PE2 immediately after signalase cleavage from 6K, with the C terminus in the lumen. Open circles are potential sites for acylation. The lower panel shows the configuration after acylation and phosphorylation-dephosphorylation, with the C terminus in the cytoplasm. In this model the side chains of the fatty acids are shown penetrating into the membrane, resulting in the alignment of the C-terminal domain along the membrane. The model is not to scale.

surface. In contrast, in the presence of inhibitors of dephosphorylation, PE2 was also synthesized, transported, and cleaved to PE2, but in this case PE2 and/or E2 was detectably phosphorylated and nucleocapsids were aligned along membranes in the cell. Thus, in this case there was an interaction between nucleocapsids and glycoproteins in the cell surface, although no budding was seen. The authors hypothesized that the withdrawal of the E2 tail into the cytoplasm requires phosphorylation within the region; once in the cytoplasm, the E2 tail can interact with the nucleocapsid but cannot support budding until it is dephosphorylated.

The three conserved cysteine residues that are fatty acylated in SIN, and presumably in all alphaviruses, are present in the cytoplasm after reorientation (Fig. 9). It seems certain that the palmitic acid residues will be present within the lipid bilayer, rather than in the cytosol, and thus might serve to orient the E2 tail along the bilayer. It is possible that addition of palmitic acid is associated with the reorientation process.

**Virus Assembly**

Assembly of nucleocapsids. Viral nucleocapsids assemble as distinct entities in the cytoplasm of an infected cell. These nucleocapsids are easily visible in electron-microscopic images of infected cells and can be readily isolated from infected cells as stable structures (see, for example, references 52 and 511).

Capsid assembly is specific in that only genomic RNA is normally packaged into capsids and a packaging signal has been identified in SIN viral RNA that is required for efficient encapsidation (599). Capsids always have RNA associated with them, and there are no reports of the assembly of RNA-free subviral structures. These observations lead to the following model for nucleocapsid assembly. Assembly begins with a specific nucleation event in which a capsid protein molecule or an assemblage of molecules binds to the packaging signal. After initiation, additional capsid proteins are recruited into the structure in a reaction that probably involves both lateral interactions with proteins already in the partially assembled shell and electrostatic interactions between the highly charged N-terminal domain of the capsid protein and the viral RNA. The electrostatic binding of the N-terminal domain to the RNA is probably nonspecific and requires no specific protein structure. The very high charge density in this region could be responsible for the failure of empty capsids to assemble, because assembly may require neutralization of these charges. The lateral interactions with other capsid proteins, in contrast, are presumably specific and mediated by sequences in the conserved C-terminal part of the protein. These interactions determine the structure of the icosahedral shell, which closes on itself to produce a \(T = 4\) particle.

Although in most alphaviruses only the genomic RNA is packaged, in AURA virus the subgenomic 26S RNA is also packaged (436), suggesting that in this virus a packaging signal is found within the region of the genome transcribed into the subgenomic RNA. The genomic RNA is found only in \(T = 4\) particles, but the subgenomic RNA is packaged into both \(T = 4\) particles and a smaller particle believed to have \(T = 3\) symmetry (435). Smaller particles have also been found in SF (20) or SIN (233) preparations containing DI particles and in SIN virus preparations produced in mosquito cells (51), and thus the ability to form \(T = 3\) particles seems to be universal among alphaviruses. The results are consistent with a model in which the structure formed by the capsid protein is not predetermined. Genomic RNA is of a size that it can be encapsidated only in a \(T = 4\) (or larger) particle, and assembly is forced into a mode that results in a \(T = 4\) particle. Smaller RNAs that possess a packaging signal can be packaged into smaller particles, and assembly may proceed in such a way as to produce either a \(T = 3\) structure or a \(T = 4\) structure, perhaps depending on whether more than one RNA is recruited into the structure. Evidence exists that \(T = 1\) particles may also be formed under some conditions (51), and Lee and Brown (280) have recently found that a larger particle, probably \(T = 7\), is formed in high yield by a mutant of SIN having two amino acid changes in the capsid protein. Thus, it appears that particles with triangulation numbers of 1, 3, 4, and 7 can be formed by alphaviruses under different conditions.

During budding of the nucleocapsid to produce virions, the nucleocapsid appears to undergo a maturation event in which it contracts, or is triggered to contract upon isolation from the virus, and sediments more rapidly. Nucleocapsids isolated from mature SIN virus sediment more rapidly than and are more sensitive to RNase than nucleocapsids isolated from infected cells (76). This observation is corroborated by studies of a mutant of SIN (E2 A-344 → V) that is defective in envelopment and forms multicored particles (178, 511). Nucleocapsids isolated from multicored particles, which have failed to mature properly, sediment more slowly than do nucleocapsids isolated from wild-type virions or from mutant virions that sediment identically to wild-type virions and that therefore appear to be standard mature virions (511). We hypothesize that the signaling event for contraction is the

interaction of the capsid with E2 during maturation; the results with the envelopment-defective mutant suggest that envelopment must be complete or nearly complete before contraction occurs.

**Capsid-spike interaction during budding.** Nucleocapsids assembled in the cell cytoplasm are thought to diffuse freely to the plasma membrane, where they are bound by the viral glycoproteins present at the cell surface. It has long been proposed that the nucleocapsid binds specifically to the C-terminal cytoplasmic domains of the glycoprotein spikes (of which only the cytoplasmic domain of E2 is significant) and that this binding provides the free energy to propel the capsid through the plasma membrane, during which process it acquires a complete complement of glycoprotein spikes (141). Lateral interactions between the glycoproteins are also important for virus assembly (571).

Early studies demonstrated that the interactions of alphavirus capsids and glycoprotein were specific (reviewed in reference 523). Phenotypic mixing did not occur between alphaviruses and viruses belonging to different families (625), and no host protein was incorporated into the alphavirus virion (510). Expression and transport of the glycoproteins to the cell surface were found to be necessary for binding of nucleocapsids to the plasma membrane. In a study of ts mutants of SIN, Brown and Smith (52) found that for mutant ts23, which has two mutations in the ectodomain of E1 (A-106 → T and R-267 → Q) (9) that lead to a failure to transport the glycoproteins to the cell surface at the nonpermissive temperature, nucleocapsids were found distributed uniformly throughout the cytoplasm. In contrast, for SIN ts20, which has a single mutation in the ectodomain of E2 (H-291 → L) (304) but whose glycoproteins are transported to the cell surface at the nonpermissive temperature (although PE2 is not cleaved), the nucleocapsids were found to be aligned along the plasma membrane, apparently bound by the glycoprotein spikes, although no budding occurred at the nonpermissive temperature. Other studies showed that the location of the glycoproteins in the cell determined the site of virus budding, indicating a need for interaction between capsid and glycoproteins during budding. Saraste et al. (444) found that in a ts mutant of SF with a transport defect in the glycoproteins, no budding occurred at the nonpermissive temperature; shift down led to functional glycoprotein and virus budding into the Golgi apparatus where the glycoprotein spikes had accumulated, whereas normally the virus buds only from the plasma membrane. In related findings it was reported that treatment of cells with a mannosidase inhibitor (341) or with monensin (170, 229) could lead to limited budding at intracellular membranes. It has also been shown that transport of alphavirus glycoproteins to the cell surface is polarized in polarized cells and that virus budding occurs only at the surface containing the glycoproteins, whether it be basolateral or apical (374, 629). Finally, it has been shown recently that no virus assembly occurs in the absence of nucleocapsid formation or of glycoprotein expression (536), formally demonstrating the necessity for capsid-glycoprotein interactions for virus formation.

Several independent studies have led to the conclusion that most of the viral glycoprotein present at the cell surface is bound to nucleocapsids after the first few hours of infection. Johnson et al. (230) used fluorescent photobleaching to study the mobility of SIN glycoproteins on the cell surface and concluded that these proteins were mobile early after infection but not later in infection, suggesting that most of the SIN glycoproteins at the cell surface are immobilized later in infection by interactions with nucleocapsids. This finding contrasted with results obtained with vesicular stomatitis virus

glycoproteins in the same experiments, which were mobile both early and late after infection. The distribution of SIN glycoproteins on the surface of cells has also been studied by electron microscopy with freeze-fracture or surface replica techniques, and conclusions similar to those from the fluorescence-photobleaching experiments were drawn. Birdwell et al. (29) found that SIN glycoproteins could first be detected at the cell surface at 1.5 h after infection in a distributed pattern, with budding virus containing high concentrations of glycoproteins evident by 3 h after infection. Pavan et al. (394, 395) used both surface replica and freeze-fracture techniques together with gold labeling to detect virus glycoproteins. They found that after virus budding was first detected, almost all of the virus glycoprotein was associated with budding figures and there was no pool of free glycoprotein. Further, they found that budding virus figures excluded cell proteins. In related studies, Torrisi and Bonatti (548) used freeze-fracture to show that the viral glycoproteins partitioned differently in the plasma membrane from in internal membranes, suggesting that the glycoproteins in the plasma membrane, but not in intracellular membranes, are attached to nucleocapsids. In a different approach, Zhao and Garoff (626) reported that the SF glycoproteins are unstable at the cell surface in the absence of expression of the capsid protein and turn over rapidly. Degradation of E1 results in release of an E1 fragment into the culture fluid. The instability of the glycoproteins when expressed alone is consistent with the hypothesis that the viral membrane proteins are normally captured efficiently by budding nucleocapsids and that no large pool of free glycoprotein spikes exists at the plasma membrane.

Binding to the nucleocapsid by the glycoproteins has been directly demonstrated in studies in which SF virions were treated with octylglucoside. This treatment removed the lipid bilayer but left most of the glycoproteins attached to the nucleocapsid, suggesting that there is a noncovalent binding of the glycoprotein spikes by the capsid (196).

Domains involved in the E2-nucleocapsid interaction. Both the nucleocapsid and the glycoproteins are organized into a T = 4 lattice (70, 71, 391) such that there is a 1:1 molar ratio between nucleocapsid and glycoproteins, and the current model for virus budding is that there is a binding site on each nucleocapsid protein for part of or all of the cytoplasmic domain of E2 (Fig. 2 and 3). Metsikkö and Garoff (348) found that a synthetic peptide of 31 residues corresponding to the SF E2 tail would in fact bind to SF capsids, either in solution or after coupling the peptides to a solid matrix. The peptide was found to oligomerize, and it was the oligomers that interacted with the capsid.

Hahn et al. (177) sequenced the structural-protein region of the WEE genome and found that it is a recombinant virus in which the nucleocapsid protein was derived from EEE whereas the glycoproteins were derived from a SIN-like parent. The EEE capsid protein and the SIN glycoproteins in WEE appeared to have undergone selection after the recombination event that adapted these two proteins to one another (Fig. 10). Seven amino acids in the WEE nucleocapsid protein have changed from the EEE sequence to the SIN sequence, and Hahn et al. (177) suggested that these seven changes adapted the EEE nucleocapsid to the SIN glycoproteins. These amino acids, M-137, N-172, G-201, L-231, A-234, T-238, and I-254 (SIN numbering and residues), form a roughly linear trace in the capsid protein about 45 Å (4.5 nm) long (Fig. 6), and the cytoplasmic tail of E2 may lie along this trace (71). Conversely, four amino acids in the cytoplasmic domain of the SIN-like E2 of WEE have switched to the EEE amino acids, presumably to better adapt the glycoproteins to the EEE nucleocapsid.

VOL. 58, 1994

![Diagram]

FIG. 10. Evolution of the WEE capsid protein and glycoprotein E2 after formation of WEE by recombination. At the top of the figure is a schematic representation of the structural protein domain of WEE in which the capsid protein was donated by EEE and the glycoproteins were donated by a SIN-like virus. In the WEE capsid protein the seven amino acids (aa) whose positions are indicated by the tick marks have changed from the EEE residue to the SIN residue as shown below; these residues are marked on the structure of the capsid protein in Fig. 6. In addition, the four amino acids in the C-terminal domain of E2 whose locations are shown by the tick marks have changed from the SIN residue to the EEE residue as shown. Adapted from reference 527 with permission of the publisher.

SIN E2 cytoplasmic domain extends from K-391 to A-423, and these changes in WEE E2 at I-408, V-418, S-420, and A-421 (SIN numbering and residues) suggested that the C-terminal half of the tail, at a minimum, is important for the specificity of the capsid-E2 interaction.

Lopez et al. (312) extended this line of reasoning by constructing a chimeric virus in which the entire genome was derived from RR except for the capsid protein, which was derived from SIN. In this chimera, SIN nucleocapsid protein must interact with RR envelope proteins during virus assembly. The chimera was almost nonviable, producing 5 orders of magnitude less virus than the parental RR, as determined either by infectivity assays of released virus or by the release of radiolabeled virus particles into the culture medium. The number of nucleocapsids formed in cells transfected with RNA from the chimeric virus and from RR was the same, however, demonstrating that the defect in viral assembly was not in the formation of nucleocapsids. Other studies have shown that alphavirus glycoproteins are transported to the cell surface independently of the expression of other viral genes (626), and thus the very limited production of the chimeric virus must result from defective capsid-glycoprotein interactions that do not permit budding.

There are 11 amino acid differences between RR and SIN in the 33-amino-acid E2 tail (Fig. 11). Changing two of these residues in the chimera from the RR sequence to the SIN sequence, A-403 and N-405 (SIN numbering and residues), gave rise to a variant that produced >100-fold more virus than did the parental chimera. Changing nine of these residues to the SIN sequence, A-403, N-405, I-408, T-410, S-411, A-413, V-418, R-419, and S-420, resulted in a virus that grew 10⁴-fold better (Fig. 11). These results show that it is possible to adapt the RR E2 tail to the SIN nucleocapsid and directly demonstrate the importance of the E2 tail-capsid interaction for virus assembly, as distinct from possible effects of amino acid substitutions on glycoprotein transport or glycoprotein-glycoprotein interactions; they also show that the entire domain from residues 403 to 420 is important for the specificity of the interaction. Although the amino acid changes could affect the binding reaction by alterations in the conformation of the E2 tail at regions away from the actual site of change, the simplest

THE ALPHAVIRUSES 507

![Diagram]

FIG. 11. Mutational analysis of the chimeric virus RR(SINc). A schematic of the genome of RR(SINc) is shown, indicating that the entire genome is derived from RR with the exception of the capsid protein, which is derived from SIN. The E2 cytoplasmic domain is expanded below, with the residues numbered from the beginning of E2. The sequences of the two parental viruses and those of the mutated chimeras are shown. Single plaques from transfection with RNA were picked and used to infect one well of a six-well plate of BHK cells under liquid medium; the titer shown is the average of duplicate stocks produced in this way. Reprinted from reference 312 with permission of the publisher.

hypothesis is that the entire C-terminal region of the E2 tail is involved in binding to the nucleocapsid. Of interest was the finding that change of a 10th residue to the SIN sequence, K-394 (RR) → E-395 (SIN), was deleterious for the growth of the chimera (Fig. 11), suggesting that the N-terminal region of the tail might be involved in glycoprotein-glycoprotein interactions expressed in the membrane-spanning region or in the ectodomains of the proteins. The importance of the 11th difference, K-391 in SIN, was not tested.

Gaedigk-Nitschko and Schlesinger (134) and Ivanova and Schlesinger (221) also studied the effects of site-specific mutations in the E2 tail of SIN upon assembly. Changes at E-395, P-399, A-401, P-404, L-415, C-416, or C-417 were found to be deleterious for virus assembly, and for some mutants tested the extent of inhibition was found to depend on whether the virus was grown in mosquito cells or in vertebrate cells. Many of these mutations led to the formation of multicored particles, showing that the interactions among viral components required for virus assembly were defective. These results demonstrate the importance of the E2 cytoplasmic domain for efficient virus assembly, but because the effect of the mutation is to inhibit virus assembly, it cannot be determined from these results alone whether the inhibition results from interference with nucleocapsid-E2 tail interactions or with glycoprotein-glycoprotein interactions. From other results, however, it seems likely that interference is at the nucleocapsid-E2 binding step in assembly.

A different approach was used by Schlesinger and colleagues (73, 460) and by Kail et al. (239), who used short synthetic peptides as inhibitors of virus assembly. Schlesinger and colleagues used peptides 6 to 25 residues in length that were derived from the E2 tail. These were simply introduced into the culture fluid over cells infected with SIN, and a number of these peptides were found to inhibit virus assembly by up to 90%. The inhibitory peptides all contained sequence from the N-terminal part of the E2 tail, but because of the requirement that the peptides be taken up by the cells in an unknown fashion, it was not clear whether the peptides that failed to

inhibit were not involved in assembly or were simply taken up less efficiently. The peptides that interfered with virus assembly may have done so by disrupting nucleocapsid-glycoprotein E2 interactions or by disrupting glycoprotein-glycoprotein interactions or both.

Kail et al. (239) injected peptides 8 to 31 residues in length into cells infected with SF, which allowed them to study a variety of peptides without concern for the efficiency of uptake. They found that many of these peptides were extremely efficient inhibitors of virus production (inhibition, ≥95%), including an 8-mer representing the C-terminal 8 residues of the SF E2 tail. They also found that a 15-mer consisting of 14 residues derived from SIN E2 with an N-terminal lysine would inhibit assembly of SF.

Taken together, these various results suggest that all regions of the cytoplasmic tail of E2 are involved in the assembly process. The N-terminal region of the E2 tail may be involved in glycoprotein-glycoprotein interactions, whereas the entire C-terminal two-thirds appears to be involved in the binding of the spikes by the nucleocapsid.

Vaux et al. (566) and Kail et al. (239) used yet another approach to study nucleocapsid-E2 tail interactions. Antipeptide antibodies were made to a synthetic peptide representing the 31-residue SF E2 tail and were used to produce an anti-idiotypic antibody called F13. The anti-idiotypic antibody was reported to bind to the capsid and thus to reproduce the E2 tail-capsid interaction. Anti-anti-idiotypic antibodies (anti-F13) were then prepared; the anti-F13 antibodies recognized the 8 C-terminal residues of the E2 cytoplasmic domain, and the authors concluded that these 8 residues constitute a linear recognition signal for the binding of nucleocapsid. The significance of these observations has been called into question by a recent report that the anti-idiotypic antibody F13 does not in fact react with the nucleocapsid but instead reacts with some component of the RNA replication machinery (535), and if F13 does not in fact bind the capsid, it cannot be used to identify nucleocapsid-binding sites. Interaction of F13 with the RNA replication machinery rather than with the capsid might explain the finding that the F13 antibody did not display a specificity for a particular alphavirus and would, surprisingly, also interact with the very different flaviviruses.

It is clear from these results that the E2 cytoplasmic domain interacts with the nucleocapsid in a sequence-specific but cross-reactive fashion. The E2 tails of 10 sequenced alphaviruses are compared in Fig. 12, and conservation of sequence is evident. Of the 33 residues, 12 are invariant in the 10 viruses; these include 3 cysteines, 3 prolines, 2 alanines, 1 arginine, 1 leucine, and 1 each of threonine and tyrosine in the invariant tripeptide TPY. The three invariant cysteines have been found to be palmitoylated in SIN (134, 221) and are presumably palmitoylated in all alphaviruses. The conservation of the tripeptide TPY suggests that the phosphorylation of E2 that is believed to occur in this region (307) might be on the T or the Y residue of this tripeptide. The conservation of cysteines and prolines suggests that the E2 tail has a conserved structure that is important for binding to the capsid, and the presence of this conserved structure may explain the cross-reactivity observed. The results with RR-SIN chimeras then suggest that the precise interactions with the capsid are modified by the changes that have occurred in the changeable residues.

It is intriguing that in addition to the cluster of four changes in the WEE E2 tail from the SIN sequence to the EEE sequence, there are clusters of changes in the membrane-spanning domains of both WEE E2 and WEE E1 from the SIN sequence to the EEE sequence, six in E2 and four in E1. The sequences of the membrane-spanning anchors of 10 sequenced

SIN    KAARELLAAANVIITSLALLGVRSAN...COOH  
AURA   ...D...Q...TVFLVT.C.FQRST...COOH  
WHA    ...D...TV...V...I.PT...COOH  
WEE    ...D...TV.A.V.I.PT...COOH  
EEE    RTNLII.K..QVIL....IKPTR...COOH  
VEE    RSVA.R.T.R.FC..V.A.T.R...COOH  
ONN    C.R.I.E.T.G.T.FL.GC.A.T.K...COOH  
SF     A.SK...T.GAVWT.GI.APR.H...COOH  
RR     T.K...T.G.V.LT.G.APR...COOH  
MAY    V.TK...T.G.V.VTIGV.APK.H...COOH  

FIG. 12. Alignment of amino acid sequences of the cytoplasmic tails of 10 alphavirus E2 glycoproteins. The sequences shown are from the references cited in Table 1; amino acid numbering is that for SIN and begins with the N terminus of E2. The cytoplasmic tail of SIN E2 is 33 residues in length, and for convenience the last 33 amino acids of all E2s are shown. Invariant amino acid residues are shaded. The three invariant cysteine residues have been shown to be palmitoylated in SIN and are presumably palmitoylated in all alphaviruses. The invariant TPY is a possible site of phosphorylation.

alphaviruses were compared in Fig. 8, and in contrast to the E2 tail region, these anchors exhibit very little conservation in sequence. Since this region exhibits so little conservation among alphaviruses, the presence of six changes from the SIN to the EEE sequence (boxed in Fig. 8) suggests that these changes might also have been important for adapting the SIN glycoproteins to EEE, although it remains possible that they represent random fluctuations in a nonconserved domain. Mutagenesis of the transmembrane regions to test their importance in virus replication would be of interest.

Lateral interactions between glycoproteins. Lateral interactions between the glycoproteins are also important in virus assembly. von Bonsdorff and Harrison (571) found that SIN glycoproteins would form regular hexagonal arrays that were isomorphous in local packing to the viral surface lattice, showing that the \(T = 4\) surface lattice on the virus arises not just from interactions with a \(T = 4\) nucleocapsid but also from highly specific lateral interactions between the glycoproteins (Fig. 1 and 2). The importance of the lateral interactions derives further support from studies of mutations in the ectodomain of SIN E2. In one mutant, H-291 → L, glycoproteins are transported to the plasma membrane at the nonpermissive temperature, where they bind the nucleocapsid, but no budding occurs (52, 304). A second mutation in the ectodomain of E2, A-344 → V, results in budding to form multicoated particles (178, 511). The effect of this mutation can be partially suppressed by a compensating change in E1, K-227 → M. These results suggest that altered conformations of E2 can lead to a failure of correct glycoprotein-glycoprotein interactions during virus budding. A third mutation, I-277 → T, results in the formation of a new glycosylation site at N-275; the SIN strain in which this mutant was isolated had three glycosylation sites in E2, and the new site resulted in a total of four polysaccharide chains being attached to E2 (105, 107). In this mutant, glycoproteins were transported to the cell surface, where they interacted with the capsid, but no virus budding occurred in vertebrate cells at 34.5°C, although normal virus production occurred in mosquito cells at this temperature. In this mutant the extra bulk or charge contributed by the fourth carbohydrate chain at position 275 appears to block the lateral interactions between the glycoproteins required for virus assembly in vertebrate cells; in mosquito cells the carbohydrate chain added by position 275 does not interfere with virus budding, perhaps because it is less bulky or because it is

-10
AR339 R R T E Y Op L T G V G G UGA CUA ACC GGG GUA GGJ GGG
SIN
AR86 R R T E Y Op L T G V G G AGG AGG ACU GAA UAC UGA CUA ACC GGG GUA GGJ GGG
AURA G N R Q Y Op L T G V G G GGA AAC AGA CAA UAU UGA CUA ACC GGG GUA GGJ GGG
WEE R Q H S N Op R Y E A G A CGU CAA CAU UCC AAC UGA CGG UAU GAA GOG GGA GCG
EEE R R H S N Op R Y E A G A CGG AGG CAC UCG AAU UGA CGG UAC GAA GOG GGC GCG
VEE V A Q Q Q Op R F D A G A GUa GCA CAA CAA CAA UGA CGG UUU GAU GOG GGJ GCA
MID R L T S A Op L D R A G A CGU UUA ACG UCA GCA UGA CUA GAC CGG GOG GGG GCC
RR D F D Q F Op L G R A G A GAU UUU GAC CAA UUC UGA CUA GGG ACA GOG GGG GCG
SF F D D V L R L G R A G A UUC GAC GAC GUC CUG CGA CUA GGC CGC GOG GGUGCA
ONN T D E E L R L D R A G G ACA GAC GAA GAG UUA CGA CUA GAC AGA GCA GGG GGU

nsP4
+1
Y I F S T D T G UAC AUA UUU UCG ACG GAC ACA GGC
Y I F S T D T G UACAUAUUUUUCG ACG GAC ACA GGC
Y I F S S D T G UAC AUA UUC UCU UCUGAU ACA GGC
Y I F S S E T G UAU AUU UUC UCA UCG GAA ACA GGC
Y I F S S E T G UAC AUU UUC UCA UCC GAG ACG GGA
Y I F S S D T G UAC AUC UUU UCC UCC GAC ACC GGU
Y I F S S D T G UAC AUA UUC UCA UCG GAU ACA GGC
Y I F S S D T G UAC AUC UUC UCG UCUGAU ACC GGA
Y I F S S D T G UAU AUU UUC UCC UCG GAC ACU GGC
Y I F S S D T G UAC AUA UUC UCC UCUGAC ACU GGU

FIG. 13. Translated sequences of 10 alphaviruses around the termination codon (boxed) between nsP3 and nsP4. The top two lines are two strains of SIN which differ by only 2 nt (shaded) in this region. One of these changes alters the opal codon (UGA) to one encoding Cys (UGU). In the bottom two lines the opal codons have been replaced with CGA, encoding Arg, in SF and ONN. In every case the nucleotide following the opal codon (or its replacement) is C (underlined), which has been shown to be necessary for readthrough. This figure also illustrates that there is little sequence conservation upstream of the opal codon and that even where amino acid identity is high, such as the N-terminal region of nsP4, multiple codons are used such that the nucleotide sequences have diverged almost to the full extent allowed by the degeneracy of the code. Sequences for this figure are found in the references cited in Table 1, except for the SIN AR86 sequence, which is from reference 484.

uncharged (mosquito cells do not add sialic acid to oligosaccharide chains).

The 6K protein and ion gradients in assembly. The 6K polypeptide has been found to be important for virus assembly and to be incorporated into virions in small amounts (7 to 30 copies) relative to the glycoproteins (133, 317). Several alterations introduced into the 6K region, including mutations that reduced palmitoylation of 6K (132, 134) or deletion of the sequence encoding this peptide (303), resulted in greatly reduced virus yield. Many of the site-specific mutations tested resulted in multicored particles, demonstrating the failure of normal budding when mutant 6K is present. However, virus particles formed that were totally devoid of 6K appeared to be indistinguishable from wild-type particles, showing that although the 6K protein facilitates virus assembly, it is not essential for virus assembly, and that the incorporation of a few molecules of it into the virus is not required for infectivity. The mechanism by which the 6K protein affects transport and assembly is thus obscure.

It is also known that ion gradients are important in some way for virus assembly. SIN assembly has long been known to be sensitive to the ionic strength of the medium, and final maturation of the particles is very inefficient at low ionic strength (513, 577, 578). Shifting from low ionic strength to isotonic medium results in the rapid release of virus, but this release can be inhibited by furosemide, an inhibitor of the K⁺/Na⁺/Cl⁻ cotransporter. These results suggest that efficient virus maturation requires a high intracellular concentration of Na⁺ ions (560) and that the alterations in ion transport induced by virus infection (144, 561) may facilitate the efficient release of progeny virus particles. In like vein, verapamil and chloropromazine, which affect calcium levels in cells, were found to block virus formation at the final stages of assembly (461). It has also been shown that SF-infected cells become depolarized (24). The precise mechanism by which these ionic

changes promote virus assembly is not known (but see reference 287).

Nonstructural Proteins

Translation. The alphavirus nonstructural proteins are translated from the genomic RNA as one or two polyproteins, depending on the virus. For SIN AR339 and derivatives of it, translation begins at an AUG codon at nt 60 to 62 (386, 516, 517). Polyprotein P123, containing the sequences of nsP1, nsP2, and nsP3, is produced when translation terminates at an opal codon at codon 1897 of the nonstructural open reading frame (ORF), between nsP3 and nsP4. Polyprotein P1234, containing in addition the sequence of nsP4, is produced upon readthrough of this opal codon (Fig. 4). Readthrough of UGA codons in mammalian cells has been found to result from the incorporation of arginine, cysteine, or tryptophan in the position of the UGA codon (120). Readthrough of the opal codon in SIN occurs 10 to 20% of the time during translation in cell-free systems at 30°C and <5% of the time at 40°C (92, 300, 487) and presumably at about the same frequency in infected cells. Six other alphaviruses, MID (516), RR (515), VEE (258), AURA (434), WEE (587), and EEE (587), also have an opal codon positioned between nsP3 and nsP4 (Fig. 13). Polyprotein synthesis and processing in cells infected with these viruses is not as well understood as in the case of SIN, but the opal codon is assumed to function in the same way in each virus. In two alphaviruses, SF and ONN, the opal codon has been replaced by an arginine codon (CGA) such that there is no termination at this position and only polyprotein P1234 is produced (292, 540); in addition, in the AR86 strain of SIN the UGA codon is replaced by a cysteine codon (UGU) (484). The subsequent processing of the viral nonstructural proteins in viruses with or without a stop codon interrupting the ORF appears to be similar, although this should be examined in

greater detail. SIN (92) and SF (541) have been found to produce P123 and nsP4 early in infection and P12 and P34 later in infection, which are then further processed. The major differences seem to be that SF produces nsP4 and nsP4-containing polyproteins in greater amounts than does SIN, that P34 has a half-life of 30 min in SF-infected cells but is relatively stable in SIN-infected cells late in infection, and that relatively greater amounts of nsP4 are present in SF-infected cells than in SIN-infected cells.

The signal for translational readthrough of the UGA codon has been found to consist of a single C residue downstream of the UGA (283, 300). UGAC is leaky, whereas UGA followed by any other nucleotide terminates efficiently; no other upstream or downstream SIN nucleotides were required for the UGA to be leaky.

In every alphavirus sequence, C follows the opal codon or its replacement (Fig. 13). In the three sequenced alphaviruses in which the opal termination codon has been replaced by a sense codon, therefore, there is only a single nucleotide change from the tetranucleotide UGAC. Because alphavirus sequences drift rapidly in the absence of selection, we suggest that the selective advantage of a sense codon in these viruses may be transient and that sense codons may be in equilibrium with opal. Note from Fig. 13 that the N-terminal sequence of nsP4 is almost invariant among alphaviruses (and thus the amino acid sequence must be crucial for function) but the amino acids are encoded by different codons in the different viruses (and thus the exact nucleotide sequence is not crucial). For example, both possible codons are used for the N-terminal tyrosine and all three possible codons are used for the following isoleucine, and so forth. Figure 13 also shows that upstream of the opal codon there is little or no conservation of sequence at the amino acid or nucleotide level but the downstream sequence is conserved. Thus, the fact that the three viruses with sense codons differ by only one nucleotide from UGAC is striking. Conservation of the C is particularly notable given that both arginine and leucine, both of which can be encoded by six different codons, occur as the following amino acid and that the leucine in this position in the three viruses with sense codons could be encoded by UUA or UUG. The fact that one SIN isolate contains a sense codon but other closely related strains contain an opal codon also suggests that the advantage of a sense codon is transient.

Functions of nsP1. The four different nonstructural proteins function during infection as polyproteins as well as cleaved final products, and in any event the replicase activity is believed to reside in a complex of nonstructural proteins and host proteins. Functions of the individual protein components have been explored by examining the protein sequence for the presence of sequence motifs found in biochemically characterized enzymes, by performing biochemical assays with the different nonstructural proteins or polyproteins, and by studying the phenotypes of *ts* mutants or of mutants exhibiting other properties such as change in host range, RNA synthesis pattern, or protein processing, whose lesions have been determined to arise from specific changes in one of the nonstructural proteins. The mutants examined include site-specifically constructed mutants as well as mutants isolated on the basis of a selectable phenotype.

nsP1 appears to be specifically required for initiation of (or continuation of) synthesis of minus-strand RNA. The change A-348 → T gives rise to a mutant (*ts11*) that upon shift from permissive to nonpermissive temperatures stops making minus-strand RNA, whereas plus-strand synthesis continues (183, 450, 584). nsP1 is also thought to be the enzyme, or a component of the enzyme, that caps the genomic and sub-

genomic RNAs during transcription. A new methyltransferase is produced in virus-infected cells (80) that is thought to be responsible for the methylation of the caps of both genomic and subgenomic RNAs (81). Stollar and colleagues have shown that the methyltransferase activity resides in nsP1. They isolated variants of SIN that, unlike the parental virus, were able to replicate in mosquito cells in the absence of methionine in the medium (106). In one variant examined in detail, resistance to low methionine levels was due to two changes in nsP1, R-87 → L and S-88 → C (350), both of which were required for expression of the phenotype (351). These mutations decreased the \(K_m\) for *S*-adenosylmethionine, the methyl donor for the transferase, by 10-fold (456). Thus, the mutant nsP1 is able to function efficiently as a methyltransferase at the much lower *S*-adenosylmethionine concentrations present in cells grown in methionine-free medium. The mutant was also more sensitive to the effects of *S*-adenosylhomocysteine hydrolase inhibitors such as neplanocin A (104), which lead to an increase in the concentration of *S*-adenosylhomocysteine, an inhibitor of methyltransferase enzymes. The results are consistent with the hypothesis that the mutations alter the binding pocket for *S*-adenosylmethionine. Subsequently it was shown that SIN nsP1 expressed alone in *Escherichia coli* has methyltransferase activity (352).

A second activity required to cap RNA is guanylyltransferase. Scheidel et al. (455) isolated mutants of SIN that were resistant to mycophenolic acid and ribavirin. These drugs inhibit wild-type SIN replication by reducing the cellular GTP levels (322). One such mutant was studied in detail and found to contain three changes in nsP1, all of which are probably required for full expression of the resistance phenotype: Q-21 → K, S-23 → N, and V-302 → M. These results are consistent with the hypothesis that nsP1 is a guanylyltransferase and these mutations produce a mutant enzyme that is active at the reduced levels of GTP present in cells treated with mycophenolic acid.

A third activity of nsP1 is its modulation of the activity of the proteinase activity of nsP2. Polyproteins containing nsP1 cleave the bond between nsP2 and nsP3 very poorly (92).

Functions of nsP2. The N-terminal domain of nsP2 (residues 1 to 459 in SIN) is believed to be an RNA helicase. SIN nsP2 belongs to a superfamily of proteins containing nucleoside triphosphate (NTP)-binding domains that are characterized by two motifs, one represented by the sequence GSGKS (starting at residue 189) and the second represented by DEAD (starting at residue 252) (157, 161, 209). Gorbalena et al. (155, 158) have argued that this protein superfamily, with representatives from bacteria, eukaryotes, and many viruses, is monophyletic, suggesting that the core domain of the helicase is a very ancient one. A number of proteins belonging to this superfamily, including the *E. coli* RecBCD helicase and certain proteins belonging to the so-called DEAD box subset (159), have been shown biochemically to possess helicase activity. Because of its homology to these NTP-binding proteins, the N-terminal domain of nsP2 is thought to be a helicase involved in duplex unwinding during RNA replication and transcription.

The C-terminal domain of nsP2 (residues 460 to 807 in SIN) functions as the nonstructural proteinase and will be discussed in more detail below.

nsP2 has also been found to be specifically required for the synthesis of the 26S subgenomic mRNA, presumably for the initiation of its transcription. A number of mutations in the proteinase domain of nsP2 have been found to render the synthesis of 26S RNA temperature sensitive (183, 448, 449). It is not known at present whether the temperature sensitivity of subgenomic RNA synthesis is due to an altered proteinase activity at the nonpermissive temperature or whether nsP2 has

a function in 26S RNA synthesis distinct from its proteolytic role. Since nsP2 is known to have at least two distinct functions (helicase and proteinase), it is possible that it has additional functional domains whose signature amino acid motifs have not been identified.

In SF-infected cells, about 50% of nsP2 is found in the nucleus, principally in the nucleolus (398). Nuclear localization signals have been found in both the N-terminal and C-terminal regions of the protein, and a nucleolus-targeting signal is centered near residue 500 (424). The function served by nsP2 in the nucleus is unknown, but it is presumably distinct from its involvement in viral RNA replication.

**Functions of nsP3.** The functions of nsP3 in viral replication are not well understood. The protein has two distinct domains (258, 515) (Table 2). The N-terminal domain varies from 322 to 329 residues in length in sequenced alphaviruses and is conserved in sequence, exhibiting a minimum of 51% amino acid sequence identity between any two alphaviruses sequenced to date. In contrast, the C-terminal domain is not conserved in length or sequence. This domain varies in length from 134 residues in MID to 246 residues in ONN (Table 2), and no detectable sequence similarity has been identified between different alphaviruses. nsP3 is heavily phosphorylated on serine and threonine residues in SIN, probably by a cellular enzyme (297), and less heavily phosphorylated in SF (399), and it is suspected that much or all of the phosphorylation occurs within the nonconserved domain. Presumably these two domains play different roles in virus replication. It has been shown that SF nsP3 produced alone in COS cells is phosphorylated and localized to vesicular structures, demonstrating that phosphorylation does not require association with other viral proteins (397). The function of phosphorylation in the activity of nsP3 is not known, and studies in this area are needed. Barton et al. (23) found that replication complexes isolated from infected cells 6 h after infection contained nsP3 that was heavily phosphorylated, suggesting that the highly phosphorylated forms play an active role in RNA synthesis.

It has been found that large deletions within the nonconserved C-terminal domain of nsP3 are well tolerated in the case of SIN (quoted in reference 297) and VEE (90). In the case of VEE, a deletion of 34 amino acids in this domain gave rise to a virus that was indistinguishable from wild-type VEE in its growth properties in several cultured cell lines. The fact that large deletions in this domain may be tolerated is consistent with the lack of conservation in length or sequence of this domain. It is also of note that two forms of nsP3 are produced by viruses that contain an opal codon between nsP3 and nsP4, one upon termination at the opal codon and the second, 7 residues longer, from cleavage of the readthrough polyprotein.

A ts mutation has been mapped to nsP3, F-312 → S, that renders the virus RNA negative; that is, no RNA is synthesized upon infection of cells at 40°C (183). Thus, it is clear that nsP3 is required in some capacity for RNA synthesis, as is also shown by the requirement for the presence of the nsP3 sequence in polyprotein P123 for minus-strand RNA synthesis (285, 286, 487). The presence of nsP3 in polyproteins also affects the cleavage specificity of the nsP2 proteinase; nsP2 and P12 cleave the site between nsP3 and nsP4 very poorly, whereas both P123 and P1234 cleave this site efficiently (92).

**Functions of nsP4.** nsP4 is thought to be the RNA polymerase of the virus. It contains the GDD motif characteristic of viral RNA polymerases (241), and three ts mutations in nsP4 (G-153 → E, G-324 → E, Q-93 → A) that render the virus RNA negative are known (181). The first of these leads to total cessation of RNA synthesis upon shift to the nonpermissive temperature (22, 246, 450), consistent with the hypothesis that

nsP4 is the RNA polymerase. Other mutations in nsP4 (L-48 → V, D-142 → G, and P-187 → R) result in ts virus that is more restricted for growth in mosquito cells than in chicken cells (283). Finally, a mutation in nsP4 (Q-191 → K) renders the mutant unable to shut off the synthesis of minus strands at the nonpermissive temperature (446).

The concentration of nsP4 is tightly regulated in infected cells. In most alphaviruses, translation of nsP4 requires readthrough of a UGA stop codon, leading to its underproduction relative to other nonstructural proteins (516); in addition, excess nsP4 is rapidly degraded by an N-end rule pathway (93). Replacement of the opal codon in SIN with a sense codon gives rise to a virus that overproduces nsP4, but nsP4 does not accumulate, being rapidly degraded (299). However, increase of the termination efficiency of this codon, either by replacing it with an amber or ochre codon (299) or by changing the context around the opal codon (UGAC → UGAU) to decrease readthrough (283), gives rise to a virus that is impaired in its replication. In these cases the amount of nsP4 produced appears to be insufficient to support efficient RNA replication. In the last case the virus was also found to be ts and relatively more impaired for growth in mosquito cells than in chicken cells.

nsP4 is not only metabolically unstable in infected cells (186, 187, 247, 299) but also short-lived in rabbit reticulocyte lysates (92, 188, 486). Wellink and van Kammen (602) first pointed out that the amino terminus of nsP4 of all alphaviruses was tyrosine and that these proteins might therefore be degraded by the N-end rule pathway. The N-end rule pathway relates the half-life of a protein to the identity of its N-terminal residue (14, 154). As currently understood, the degradation signal, referred to as the N-degron, comprises at least two distinct determinants, a destabilizing N-terminal residue (14, 154) and one or more specific internal lysine residues that serve as multiubiquitination sites (15, 69, 231). After conjugation to ubiquitin, the protein is rapidly degraded by a large ATP-dependent proteasome complex. Amino termini can be divided into stabilizing (e.g., methionine and alanine) and destabilizing (e.g., tyrosine, phenylalanine, leucine, arginine, lysine, aspartic acid, and glutamic acid), depending on whether they confer a short half-life on a protein that contains the second (lysine) determinant. This pathway has been reviewed recently by Varshavsky (565).

deGroot et al. (93) studied the degradation of Sindbis nsP4 following translation in reticulocyte lysates and showed that nsP4 was in fact degraded by an N-end rule pathway. The formal proof involved the demonstration that (i) the metabolic stability of nsP4 in reticulocyte lysates was a function of its N-terminal residue, the half-life of nsP4 bearing different N termini decreasing in the order Met > Ala > Tyr ≥ Phe > Arg (this was the precise order determined by Gonda et al. [154] for a chimeric polypeptide that has been widely used in the characterization of the N-end rule pathway), and (ii) addition of the dipeptides Tyr-Ala, Trp-Ala, or Phe-Ala inhibited degradation of Tyr-nsP4 and Phe-nsP4 but not of Arg-nsP4, whereas addition of His-Ala, Arg-Ala, or Lys-Ala inhibited degradation of Arg-nsP4 but not of Tyr-nsP4 or Phe-nsP4. This was the predicted result for degradation by the N-end rule pathway (15, 154).

Although it is clear that nsP4 is degraded by an N-end rule pathway, de Groot et al. (93) were not able to identify a lysine residue required for degradation, and no attempt was made to demonstrate ubiquitin dependence. nsP4 was the first natural substrate shown to follow the N-end rule, and further study of the degradation of this protein in infected cells to examine the ubiquitin dependence of degradation and a requirement for a
lysine residue will be of interest. The recent discovery of an N-end rule degradation pathway in bacteria which is independent of ubiquitin suggests that the pathway in mammals should be reexamined with natural substrates (545).

Although the bulk of nsP4 is unstable and degraded rapidly, there is a resistant fraction that is stable. The fraction of nsP4 synthesized in a short pulse that is resistant to degradation is larger early in infection than late in infection in SIN (92, 186, 187) but does not appear to be as dependent upon the stage of the infection cycle in SF (541). We have suggested that nsP4 associated with the RNA replicase complex is protected from degradation, whereas free nsP4 is rapidly degraded (93). Such a model is consistent with the fact that replication complexes are formed early and are then stable, which would lead to protection of a higher proportion of SIN nsP4 synthesized early than synthesized late. Barton et al. (23) labeled SIN-infected cells with radioactive amino acids from 2 to 4 h after infection and then isolated replication complexes at 6 h after infection. These replication complexes contained significant levels of labeled nsP4, demonstrating that nsP4 in these complexes was stable during the 2-h chase period.

### Nonstructural Proteinase

#### Active site of the enzyme. Many animal viruses encode proteinases required for processing of viral polyproteins in the cytosol of the cell (526). In the case of alphaviruses, processing of the envelope glycoproteins is effected by cellular enzymes resident in the subcellular organelles, but removal of the capsid protein from the structural protein precursor and processing of the nonstructural polyproteins proceed in the cytosol and require viral enzymes (531). The capsid proteinase was described above. Here we describe what is known about the nonstructural proteinase.

Hahn et al. (183) mapped the causal mutations of four SIN ts mutants that were deficient in proteinase activity at the nonpermissive temperature. All such mutations were found to reside in the C-terminal half of nsP2 (F-509 → L, A-517 → T, D-522 → N, G-736 → S), and the authors concluded that this domain functioned as the nonstructural proteinase. To define the proteinase domain and the activities of the enzyme more precisely, the activity of the proteinase was studied after its translation in a cell-free system. RNA isolated from virions or RNA transcribed in vitro from cDNA constructs can be translated in rabbit reticulocyte lysates, and the resulting polyproteins are processed by the nonstructural proteinase to produce cleaved products that appear to be identical to the products produced in infected cells. In the first studies, the effects of truncations (96) or deletions (188) on the activity of the SIN enzyme were determined. The results clearly defined the C-terminal half of nsP2 as necessary and sufficient for proteolytic processing of the alphavirus nonstructural polyprotein. Deletion of virtually all of nsP1, nsP3, or nsP4 led to effectively normal processing of the deleted constructs. Deletions in the N-terminal half of nsP2 led to an active protease but altered processing pathways, presumably because of misfolding of the protein product. Deletions in the C-terminal half of nsP2 abolished processing, however; more precisely, residues 475 to 728 of nsP2 were required for proteolytic activity. As a control, it was shown that in cases when processing was abolished, cleavage would occur in trans when an active proteinase translated from wild-type viral RNA was supplied, demonstrating that the failure to cleave did not result from misfolding of the protein induced by the deletions such that the cleavage sites could not be recognized by the proteinase.

From an analysis of the amino acid sequences of the

C-terminal domains of alphavirus nsP2s, Hardy and Strauss (188) postulated that nsP2 is a thiol protease that resembles papain in which C-481 and one of five conserved histidine residues in the region defined by the deletion mapping form the catalytic dyad. This hypothesis has been tested by site-specifically changing this cysteine and all five histidines, as well as a number of other residues in the protease domain, and testing the mutagenized construct for activity (512). The change of C-481 → G, S, or R abolished all protease activity, consistent with the hypothesis that this is the active-site cysteine, whereas changes in C-525 or S-535, both of which are conserved, were tolerated. Of the five conserved histidines in this domain, substitution of A, Q, or Y for H-558 abolished all activity of the enzyme, whereas changes in the other four histidines were tolerated. The results are consistent with the hypothesis that nsP2 is a papain-like proteinase in which C-481 and H-558 form the catalytic dyad. A schematic diagram of SIN nsP2 that illustrates the different amino acid substitutions tested and their effects on the activity of the enzyme is shown in Fig. 14.

The possible importance of an asparagine residue for the activity of the enzyme was also examined (136, 202). All four conserved asparagine residues were changed to aspartic acid and to serine, and the effects on the proteinase activity were examined (512). No asparagine was absolutely required for activity, but the changes in N-609 markedly reduced the activity of the enzyme, and, remarkably, N-614 → D led to enhanced cleavage. This last change was lethal for the virus at 37°C, suggesting that the extent of cleavage is regulated and that polyprotein P123 has a specific function in virus replication.

It was also found that W-559, following the active-site H-558, was essential for proteolysis (512). A model was presented that W-559 might interact with W-482, following the active-site C-481, to maintain the two components of the catalytic dyad in the proper conformation for proteolysis.

The alphavirus proteinase is papain-like in that it has a catalytic cysteine upstream of a catalytic histidine, but no sequence identity with papain is discernible other than a few residues around the active site. It would be of considerable interest to determine if the protein has a fold similar to that of papain, indicating that the virus had in the past acquired a cellular proteinase for its own use, or if convergence of function led to the similarities observed. A statistically significant alignment of the (putative) papain-like proteinase of rubella virus with cellular papain-like proteinases has been reported (160), suggesting that the viral enzymes may have derived from cellular enzymes. Koonin and Dolja (267) have presented an alignment for the papain-like proteinases encoded by several families of RNA viruses that suggests that all of these proteinases may be related to one another.

Nature of cleavage sites recognized by the enzyme. There are three sites in the nonstructural polyproteins of alphaviruses that are cleaved. The sequences flanking these three sites in 10 alphaviruses are aligned in Fig. 15. The sites all share common features, consistent with the hypothesis that all three cleavages are effected by the same enzyme.

In all 10 viruses sequenced to date, the residue in the P2 position is invariably glycine in all three sites. The importance of glycine in the P2 position was examined by Shirako and Strauss (486) for SIN. Substitution of the P2 glycine in the nsP1/nsP2 site or the nsP2/nsP3 site with alanine led to slower cleavage, and substitution in any of the three sites with valine or glutamic acid made the site completely refractory to cleavage.

The residue in the P1 position of the cleavage sites is an amino acid with a small side chain, i.e., alanine, cysteine, or
nsP2

![Diagram](#)

FIG. 14. Map of SIN nonstructural protein nsP2, showing the location of the proteinase catalytic residues and of various mutations introduced into the proteinase. Both site-specific mutations (512) and mutations that were selected because they render the proteinase ts (183) are shown. Solid stalked symbols indicate reduced or absent proteolytic activity, and open symbols indicate wild-type enzymatic activity, as determined after translation of the constructs in cell-free lysates. Checkered symbols indicate that the mutants are ts for proteolysis in vivo (186). The shape of the symbol indicates the parental amino acid: circles for cysteines, squares for histidines, triangles for asparagines, and diamonds for other amino acids. Reprinted from reference 512 with permission of the publisher.

glycine, with only two exceptions (the P1 residue is Arg in the WEE and EEE nsP2/nsP3 site). The importance of alanine or glycine in the P1 position was examined in SIN by substituting the P1 glycine in the nsP3/nsP4 site with valine; the modified site was no longer cleavable (92). Thus, it is clear that efficient cleavage in SIN requires GA↓ or GG↓ (GC↓ has not been tested) and that substitution of either of these amino acids with bulky residues abolishes cleavage at the modified site.

The residue in the P3 position of all cleavage sites is alanine in all viruses examined except SIN and WHA, in which it is isoleucine or valine, and AURA, in which it is alanine, serine, or valine (Fig. 15). This suggests that the P3 position is also important for cleavage site recognition, although this has not been directly tested. The P4 residue exhibits only limited conservation, and the major recognition signal for cleavage may be contained, therefore, in residues P3-P2-P1.

The residue in the P1′ position is always alanine or glycine in

| nsP1/nsP2 | nsP2/nsP3 | nsP3/nsP4 |
| --- | --- | --- |
| $P_{4} P_{3} P_{2} P_{1}$ | $P_{4} P_{3} P_{2} P_{1}$ | $P_{4} P_{3} P_{2} P_{1}$ |
| SIN | D I G A | ALVE | G V G A | APSY | *LTG V G G | YIFS |
| AURA | D AGA | ALVE | G S G A | APSY | *LTG V G G | YIFS |
| WHA | D I G A | ALVE | G V G A | APSY | *RFD A G A | YIFS |
| VEE | E AGA | GSVE | E A G C | APSY | *RFD A G A | YIFS |
| EEE | E AGA | GSVE | E AGR | APAY | *RFD A G A | YIFS |
| WEE | E AGA | GSVE | E AGR | APAY | R LDR A G G | YIFS |
| ONN | R AGA | GI VE | R AGC | APSY | RLGR A G A | YIFS |
| SF | H AGA | GV VE | T AGC | APSY | *LGR A G A | YIFS |
| RR | R AGA | GV VE | T AGC | APSY | *LDR A G A | YIFS |
| MID | R AGA | GV VN | T AGC | APSY | *LDR A G A | YIFS |

FIG. 15. Cleavage sites in the nonstructural proteins. Amino acid sequences at the sites of proteolytic cleavage in the nonstructural polyproteins of 10 alphaviruses are shown. Boxed residues are those for which at least 7 out of the 10 viruses have the same amino acids. The glycine residue in the P2 position is shown in boldface type to emphasize that it is invariant in all three cleavage sites. The asterisks indicate opal codons upstream of the nsP3/nsP4 cleavage site. Sequences are from the references cited in Table 1.

the nsP1/nsP2 and nsP2/nsP3 sites but is invariably tyrosine in the nsP3/nsP4 site. This suggests that the enzyme is relatively insensitive to the P1′ position or to other residues C-terminal to the cleavage site and that the identity of the residues in these positions is determined by their function in the protein rather than by a requirement for cleavage site recognition. Consistent with this, de Groot et al. (93) used P34 constructs as a substrate for trans cleavage by the SIN nsP2 proteinase in which the N-terminal residue of nsP4 was tyrosine (the wild-type residue), phenylalanine, methionine, alanine, or arginine and found that all of these P34 polyproteins were cleaved with kinetics that differed by less than a factor of 3.

The mutagenesis studies described above, together with the mutagenesis studies of the active site of the nsP2 proteinase described in the previous section, the cleavage site preference studies described in the following section, and studies on the ability of the nsP2 proteinase to cleave in cis or in trans (188), make it clear that the nsP2 proteinase is the only proteinase involved in cleaving the nonstructural polyprotein of SIN (reviewed in reference 512). In contrast, polyprotein P34 of SF was reported to self-cleave by using a proteinase within nsP4 (541, 542). Comparable studies with SIN P34 have shown that this polyprotein is stable when translated as such in reticulocyte lysates but can be cleaved in trans by nsP2-containing polypeptides (93). Furthermore, in a reconstituted in vivo system that examined SIN RNA replication, a requirement for active nsP2 proteinase was found if the nsP4 required for RNA synthesis was to be produced from polyprotein P34 but the proteinase was not required if nsP4 itself was supplied (286), demonstrating that SIN nsP4 is produced by cleavage by the nsP2 proteinase. It is possible that SIN and SF differ in this regard, given the differences in producing nsP4 used by the two viruses, but it seems more likely that the nsP4 found in the SF experiments resulted from internal initiation within the constructs used.

Cleavage site preferences of the enzyme. Many experiments have shown that the SIN nonstructural proteinase cleaves primarily in trans in a bimolecular reaction, but certain bonds

can be cleaved in *cis* in an autocatalytic reaction (92, 188, 486). The major points that have been elucidated are as follows: (i) P1234 can cleave autoproteolytically to produce P123 and nsP4; (ii) P123 cannot cleave in *cis* but can be cleaved only in a bimolecular reaction; (iii) P12 can cleave autoproteolytically in a slow reaction (*t*<sub>1/2</sub> ≈ 9 min at 37°C) to produce nsP1 and nsP2; and (iv) all three sites can be cleaved in *trans*.

Both nsP2 and polyproteins containing nsP2 have been found to be active enzymes in *trans*, but the cleavage site preferences of different polyproteins are different. Shirako and Strauss (486) found that P123 would cleave in *trans* the nsP1/nsP2 bond of another P123 but not the nsP2/nsP3 bond. This observation was extended by de Groot et al. (92), who produced a nested set of nsP2-containing polyproteins whose own cleavage sites were blocked by substitutions at *P*1 or *P*2. These polypeptides were tested in an in vitro system for their ability to cleave in *trans* the three bonds of a labeled substrate P1234 whose own protease activity had been abolished by mutagenesis of the active-site cysteine (nsP2 C-481 → G). A clear set of rules emerged that described the *trans*-cleavage preferences of different polyprotein enzymes: (i) polyproteins containing the sequence of nsP1, that is, P12, P123, or P1234, were unable to cleave the nsP2/nsP3 site but cleaved the nsP3/nsP4 site fairly efficiently and the nsP1/nsP2 site inefficiently; (ii) only polyproteins containing the sequence of nsP3, that is, P123, P1234, P23, and P234, were able to cleave the nsP3/nsP4 site; (iii) polyproteins lacking nsP1, that is, P23 and P234, as well as nsP2 under certain circumstances, cleaved the nsP2/nsP3 bond very efficiently; and (iv) the presence or absence of nsP4 in the polyprotein did not affect cleavage specificity.

For SIN, these cleavage site preferences have been shown to result in different processing pathways for the virus nonstructural proteins early and late after infection (92). Very early in infection, P123 and P1234 are produced on translation of the genomic RNA. Cleavage of the nsP3/nsP4 bond in *cis*, coupled with the inability of P123 to autoproteolyze, results in the presence of P123 and nsP4. As infection proceeds and the concentration of P123 increases, cleavage in *trans* of the nsP1/nsP2 bond in P123 produces enzymes that are very active in cleaving the nsP2/nsP3 bond. By 3 to 4 h after infection, cleavage of the nsP2/nsP3 site is so rapid that it probably occurs in the nascent polyprotein. This has the effect that the major protein products produced after this point in the infection cycle are P12 and P34. P12 can be cleaved in *cis* or in *trans* to produce nsP1 and nsP2, although autoproteolysis appears to be kinetically favored. P34, however, is stable because there are no longer any enzymes in the cell that can cleave the nsP3/nsP4 site efficiently (i.e., no polyproteins containing both nsP2 and nsP3). Thus after 4 h postinfection, the major proteins found in the infected cell are nsP1, nsP2, nsP3 (produced by termination at the opal termination codon), and P34.

In SF virus, which lacks an opal termination codon between nsP3 and nsP4, nsP3 is produced throughout the infection cycle, implying that cleavage at the nsP3/nsP4 site continues, either in P34 or in P1234. Thus, there may be some differences in the processing pathways of the nonstructural polyproteins of SIN and SF. However, Takkinen et al. (541) have shown that during SF infection P123 is produced only early in infection and that later the predominant pathway is cleavage to produce P12 and P34, similar to the case for SIN.

---

### Synthesis of Viral RNAs

RNA replicase. After infection by alphaviruses, RNA replicase complexes are assembled on modified endosomal and lysosomal membranes (127). Attempts to purify the replicase complexes have met with only limited success: these partially purified complexes are membrane associated (72, 153, 335, 354, 412, 498); contain viral nonstructural proteins nsP1, nsP2, nsP3, and nsP4, as well as a cellular 120-kDa protein and other cellular proteins (22, 23); and are capable of limited RNA synthesis. These complexes are formed early in infection and are stable throughout the infection cycle. It has been shown that active complexes will form when the SIN nonstructural proteins are produced by vaccinia virus expression systems (298) or by baculovirus expression systems (54) and will function to replicate viral RNAs containing appropriate promoter elements, making possible the study of RNA replication following expression of engineered nonstructural proteins.

Cellular proteins are also thought to be required for RNA synthesis, possibly as components of the replicase complexes. A role for host proteins in viral RNA replication is suggested by studies of host range mutants of SIN (268) or of the effects of inhibitors of cellular RNA synthesis (17; reviewed in reference 524), by the finding that mutations in viral RNA promoter elements affect RNA replication differently in different cells (272, 370, 371), and by the finding that cellular proteins bind to the 3′ end of the minus strand (389, 390). The structural proteins also appear to be important in some way for alphavirus RNA synthesis. Kuhn et al. (270, 273) found that for chimeric SIN-RR alphaviruses in which the structural proteins were derived from one of the viruses and the replicase was derived from the second virus, RNA synthesis following infection was less efficient than RNA synthesis following infection by viruses whose nonstructural and structural proteins were derived from the same virus. The simplest hypothesis to explain this is that one or more of the structural proteins interacts in some way with the RNA replication machinery to regulate RNA synthesis. It is known that the capsid protein of SIN binds specifically to the viral RNA and that deletion of the RNA sequence to which the capsid binds results in less efficient RNA synthesis (599), so that capsid protein binding might promote RNA synthesis. It has also been found that for RR and for SIN a deletion of 7 amino acids in the ectodomain of E2 centered on residue 55 affects RNA synthesis following infection of some, but not all, cells (572, 598) and that substitutions at position 55 of SIN can affect RNA synthesis in mouse cells depending on the major histocompatibility complex (MHC) antigens expressed by the cell (175), suggesting that E2 may regulate RNA synthesis in some unknown way. It is of interest that Suomalainen and Garoff (535) postulated that the anti-idiotypic antibody F13, made to an antibody against the E2 cytoplasmic domain, bound to some component of the replication complex, which might imply that this domain of E2 is involved in some way with the replicase complex.

Control of minus-strand RNA synthesis. Alphaviruses regulate minus-strand RNA synthesis (447, 450, 453). Early in infection there is an exponential increase in the rate of RNA synthesis, with both plus-strand and minus-strand RNA being synthesized. At about 3 h after infection at 37 to 40°C or 6 h at 30°C (Fig. 5B), synthesis of minus-strand RNA ceases, after which plus-strand RNA synthesis continues at a constant rate throughout the remainder of the infection cycle (Fig. 5). Studies primarily by Sawicki and colleagues (23, 446, 449, 451, 452) have shown that replication complexes are formed early after infection and that these complexes are then stable. There appears to be a limit on the number of replicase complexes that

| Virus | Wild Type | Nonstructural Cleavage-Site Mutants |
| --- | --- | --- | --- | --- |
|  |  | 12V | 2V | 3V |
| Nonstructural Proteins Synthesized | Precursors | nsP1 nsP2 nsP3 nsP4 |  |  |  |
|  |  |  |  |  |
| Cleaved products |  | P123 | P123 | P123 |
|  |  | P234 | P234 | P234 |
|  |  | P12 | P12 | P12 |
|  |  | P23 | P23 | P23 |
|  |  | P34 | P34 | P34 |
|  |  | nsP1 | nsP1 | nsP1 |
|  |  | nsP2 | nsP2 | nsP2 |
|  |  | nsP3 | nsP3 | nsP3 |
|  |  | nsP4 | nsP4 | nsP4 |
| Growth Characteristics |  | 30° | 40° | 30° | 40° | 30° | 40° |
| RNA Synthesis | Minus strand | + | ++ | + | + | – | – |
|  | Plus strand | + | ++ | + | + | – | – |
| Plaque Size | Large | Large | Minute | Small | Minute | – | – |
| PFU/ml | $10^8$ | $10^8$ | $10^5$ | $10^7$ | $10^6$ | – | – |

FIG. 16. Characteristics of SIN mutants unable to cleave nonstructural polyproteins. Results for wild-type virus and three mutants are shown. In 12V the penultimate Gly in the cleavage site between nsP1 and nsP2 and that in the site between nsP2 and nsP3 have been changed to Val. In 2V only the Gly in the nsP2/nsP3 site has been changed to Val. In 3V the Gly in the nsP3/nsP4 site has been changed to Val. These changes abolish processing at the mutated site, and the nonstructural proteins produced by the mutants or the wild type are shown schematically at the top of the figure. The lower panels show the growth characteristics of these viruses following RNA transfection at either 30 or 40°C. Total incorporation of nucleic acid precursors was used to measure plus-strand synthesis, whereas minus-strand synthesis was measured by quantitative reverse transcriptase PCR. The plaque size produced by the mutants is also indicated, as is the titer of stocks produced upon infection of cells with each virus. Data from reference 487.

can be formed, possibly imposed by a limited number of cellular accessory proteins, because overexpression of minus-strand RNAs or of replicase proteins does not lead to an increase in the rate of RNA synthesis late in infection. The mechanism by which minus-strand RNA synthesis is controlled has been elucidated recently as the result of work in several laboratories.
As described above, P123 can be cleaved only in trans, and early after infection the predominant nonstructural proteins in the infected cell are P123 and nsP4. Later in infection when the concentration of P123 in the cell is high enough to support an efficient bimolecular reaction, P123 is further processed to nsP1, nsP2, and nsP3. This observation led Hardy and Strauss (188) to propose that P123 was active in the synthesis of minus-strand RNA. Further support for a role for P123 in RNA synthesis came from studies of Strauss et al. (512), who found that a mutation in nsP2 (N-614 → D) resulted in more efficient cleavage of P123, such that no uncleaved P123 was present during translation in vitro. This mutation was lethal for the virus at 37°C, leading the authors to propose that P123 (or an intermediate polyprotein) has a required role in RNA replication.
Shirako and Strauss (486, 487) examined the role of P123 in RNA synthesis by constructing mutants that were unable to cleave either the nsP1/nsP2 site or the nsP2/nsP3 site (mutant 12V), such that only P123 and nsP4 were produced in the

infected cell. Using a quantitative PCR assay to examine minus-strand RNA synthesis after infection by this mutant, they found that early minus-strand RNA synthesis proceeded normally but that subsequent production of plus strands was greatly impaired and temperature sensitive (Fig. 16). At 30°C small amounts of plus-strand RNA were produced (about 6% as much plus-strand RNA as produced by wild-type virus was present at 12 h after infection), allowing limited amplification of RNA (plus and minus strand) and production of tiny plaques. At 40°C no detectable plus-strand RNA was made and the minus-strand RNA produced early after infection was slowly degraded over the next 12 h. They also found an absolute requirement for nsP4; mutations introduced into the nsP3/nsP4 cleavage site that prevented cleavage to release nsP4 (e.g., mutant 3V) were lethal. These authors concluded that P123 and nsP4 functioned in minus-strand RNA synthesis but that cleaved products from P123 were required for efficient plus-strand RNA synthesis. Mutant 12V, in which cleavage of the nsP1/nsP2 and nsP2/nsP3 sites was blocked, could complement mutant 3V, in which cleavage of the nsP3/nsP4 site was blocked, leading to the synthesis of plus-strand RNA and production of large amounts of late-gene products.
Lemm et al. (284–286) used a reconstituted system to study the polypeptide requirements for SIN RNA replication. These researchers studied the ability of various SIN nonstructural proteins expressed transiently by a vaccinia virus system to

replicate a synthetic RNA template containing a reporter gene. P123 and nsP4 would support minus-strand RNA synthesis from a plus-strand template, but the four individual nonstructural proteins or combinations of polyproteins that did not include P123 would not support minus-strand RNA synthesis. It is possible that the extra N-terminal methionine present on nsP2 or nsP3, required for their expression as individual polypeptides, causes these proteins to be inactive, but further support for a required role for P123 came from constructs containing the nsP2 N-614 → D mutation. P123 containing D-614 was inactive in RNA synthesis unless the nsP1/nsP2 and nsP2/nsP3 cleavage sites were blocked, consistent with the hypothesis that the accelerated cleavage of these sites by the mutant proteinase is responsible for the failure of the mutant to make RNA and that uncleaved P123 is required for minus-strand RNA synthesis.

Lemm et al. (284–286) also found that nsP4 was required for synthesis of minus-strand RNA. nsP4 could be supplied by cleavage of P34 or truncated forms of P34 or could be supplied as a ubiquitin-nsP4 fusion product that is rapidly cleaved in the cell to produce authentic nsP4. Interestingly, nsP4 produced from a construct in which an N-terminal methionine was present was almost inactive in this system, leading to the synthesis of only small amounts of minus-strand RNA and no plus-strand RNA. The sensitivity of nsP4 to the identity of its N-terminal residue was also demonstrated by examining viruses in which the N-terminal tyrosine of nsP4 was replaced by phenylalanine, tryptophan, leucine, alanine, arginine, or methionine (434, 484). Viruses producing nsP4s containing an N-terminal aromatic residue (tyrosine, phenylalanine, or tryptophan) grew well; those producing nsP4 with N-terminal methionine grew poorly and formed small plaques (but demonstrating therefore at least some plus-strand RNA synthesis); and viruses producing nsP4 with N-terminal leucine, alanine, or arginine were not viable. The effect of the alanine substitution at the N terminus of nsP4 could be partially suppressed by a compensating change in nsP1 (484), suggesting that there is an interaction between nsP1 and the N terminus of nsP4 required for (minus-strand) RNA synthesis. This compensating mutation (T-349 → K) is immediately adjacent to the mutation in ts11 (A-348 → T) that renders the virus unable to make minus-strand RNA at 40°C (although it can make plus-strand RNA at 40°C), suggesting that this domain is involved specifically in minus-strand RNA synthesis. It is of interest that from a study of mutations that render the synthesis of RNA temperature sensitive, Hahn et al. (181) had also suggested that nsP2 and nsP4 interact during RNA synthesis and that the RNA replicase presumably functions as a complex in which the individual components bind to one another and interact.

One major difference between the results of Shirako and Strauss (487) and those of Lemm et al. (284–286) was in the importance of cleaved products from P123 for plus-strand RNA synthesis. Shirako and Strauss found that uncleaved P123 was very inefficient for plus-strand RNA synthesis at 30°C and totally lacked activity for plus-strand synthesis at 40°C, whereas Lemm et al. found that uncleaved P123, although preferring to synthesize minus strands, could synthesize plus-strand RNA fairly efficiently at 37°C in their system. The difference may arise from mass action considerations. The system used by Lemm et al. produced large numbers of replicase molecules and of RNA templates continuously, whereas Shirako and Strauss required amplification of the minus-strand RNA transcribed from the incoming RNA by the replicase translated from it for buildup of RNA. Under the latter conditions, the

inefficient production of plus-strand RNA by uncleaved P123 becomes apparent.

The model for RNA synthesis supported by these studies is diagramed in Fig. 17. Upon entry of the genomic RNA, it is translated into the nonstructural proteins. P123 and nsP4 are produced upon readthrough of the opal termination codon and self-cleavage of P1234 and form a replicase complex, probably in association with cellular proteins, that can use the genomic RNA as a template to synthesize minus strands. As the concentration of nonstructural proteins builds up, P123 becomes vulnerable to processing, which converts the replicase complex into one which is capable of synthesizing plus strands with high efficiency but can no longer synthesize minus strands. At 3 to 4 h after infection at 37°C, the concentration of proteinases in the infected cell becomes high enough that P123 cannot exist or be synthesized because it is cleaved while nascent. Once this stage is reached, only plus-strand RNAs can be synthesized.

In the simplest form of the model, P123 and nsP4 function to initiate minus strands and a different complex containing nsP1, nsP2, nsP3, and nsP4 functions to initiate plus strands. However, the possibility that a complex composed of nsP1, P23, and nsP4 also functions as a minus-strand replicase has not been ruled out. Complexes containing nsP1 and P23 are more efficient in plus-strand synthesis than P123-containing complexes but not as efficient as the complex containing the fully processed nonstructural proteins (Fig. 16), and it is possible that they represent an intermediate state for the replicase in which both plus and minus strands can be made. The first cleavage of P123 produces nsP1 plus P23 (486), and complexes containing these products would be expected to exist, at least transiently. It is also of note that, although the replicase complexes are formed early and appear to be stable, the ability of ts mutants containing lesions in different nonstructural proteins to complement one another (181, 183, 514) suggests that the cleaved components are capable of exchange.

The model in which cleavage of P123 results in shutdown of minus-strand RNA synthesis is supported by earlier results of Sawicki et al. (447, 450), who studied the sensitivity of RNA synthesis to protein synthesis inhibitors. During the early phase of the infection cycle, when minus-strand RNA is synthesized, continuing protein synthesis was found to be required for continuing synthesis of minus-strand RNA but not for continuing synthesis of plus-strand RNA. The vulnerability of P123 to processing, which converts the replicase complex to a complex capable of synthesizing only plus-strand RNAs, means that continuing synthesis of P1234 and formation of new minus-strand replicase complexes is required for continued synthesis of minus-strand RNA.

Two mechanisms are possible by which the cleavage of P123 could convert the replication complex from one that synthesizes minus strands to one that synthesizes plus strands. One possibility is that cleavage leads to the loss of a component which is required for minus-strand synthesis but not for plus-strand synthesis. A second possibility is that cleavage leads to changes in conformation such that the ability to initiate minus strands is lost. This second possibility is favored in studies by Sawicki et al. (446, 448, 449, 451, 453), in which mutations in nsP2 (A-517 → S and N-700 → K) or in nsP4 (Q-191 → K; not Q-195 → K as reported in reference 446) were found to result in temperature-sensitive shutdown of minus-strand RNA synthesis. The nsP2 mutations also inactivated the nonstructural proteinase (183, 186, 449), but the nsP4 mutation had no observable phenotype other than its effect on minus-strand RNA synthesis. Upon incubation at the nonpermissive temperature, minus-strand synthesis is not shut

A) Translation of 49S RNA

B) Cleavage of 3/4 bond

C) Minus-strand RNA synthesis

D) Cleavage of 1/2 and 2/3 bonds

E) Plus strand RNA synthesis

FIG. 17. Schematic representation of the formation of replicase complexes specific for the synthesis of SIN minus-strand and plus-strand RNA. (A) The uncleaved nonstructural polyprotein is shown, with different types of shading for the nsP1, nsP2, nsP3, and nsP4 domains. (B and C) Cleavage of the nsP3/nsP4 bond (and perhaps addition of host factor) leads to the formation of the active minus-strand replicase. (D and E) Cleavage of nsP1/nsP2 and nsP2/nsP3 bonds with a concomitant change in conformation leads to the formation of the plus-strand replicase and synthesis of plus strands. This model is based on the results of Shirako and Strauss (487) and of Lemm et al. (286).

off by these mutants, or if minus-strand synthesis has shut down, shift to a nonpermissive temperature leads to resumption. In these mutants, continuing protein synthesis is not required for minus-strand synthesis, in contrast to the situation for wild-type virus, and thus it appears that in these mutants cleaved P123 can make minus strands. It appears likely, therefore, that the cleavage of P123 alters its conformation in such a way as to render it incapable of initiating minus-strand synthesis and that conformational changes induced in nsP2 or nsP4 by these mutations allow the cleaved replication complex to mimic the conformation of the uncleaved complex and thus to synthesize minus strands as well as plus strands.

The cleavage site preferences exhibited by the SIN nonstructural proteinase lead not only to the disappearance of P123 at 3 to 4 h after infection but also to the appearance of a large, fairly stable pool of P34 at the same time. de Groot et al. (92) hypothesized that the changes effected by the cleavage site preferences found by them were responsible for the shutoff of minus-strand RNA synthesis and suggested that P34 might play an active role in RNA replication. It now seems probable that P34 arises simply as a by-product of the cleavage pathway and plays no direct role in RNA replication.

The work of Lemm and Rice (284, 285), as well as earlier studies of DI RNAs (see especially references 293 and 295), makes clear that a plus-strand RNA template can be supplied to a replicase in trans. On the other hand, attempts to supply minus-strand templates in trans have all failed (284, 463) for reasons that are not clear. It is possible that a minus-strand RNA is normally used only in cis by the replicase that transcribes it from a plus-strand template or that assembly of active replicase complexes occurs only on plus-strand templates, although the results of Sawicki and Sawicki (449) suggest that exchange of templates is possible, at least with certain mutant replicases. As a practical matter, the inability to supply minus-strand templates to a preformed replicase complex means that synthesis of plus-strand RNA cannot be studied independently of minus-strand synthesis and that the minimal requirements for synthesis of 26S subgenomic RNA versus 49S genomic RNA are not well understood. The 26S promoter differs from the 49S promoter, but it is not known whether the 26S transcription complex contains a different constellation of nonstructural proteins (and host components?) from the 49S replicase. A number of SIN mutations in nsP2 (C-304 → Y, F-509 → L, A-517 → T, N-700 → K, G-736 → S) result in the synthesis of reduced quantities of 26S RNA relative to 49S RNA following a shift to the nonpermissive temperature, suggesting that nsP2 could have a distinct role in the specific initiation of 26S subgenomic RNA transcription.

Role of the opal termination codon. The model for control of minus-strand RNA synthesis described above suggests a model

518 STRAUSS AND STRAUSS MICROBIOL. REV.

![Diagram](attachment:diagram.png)

CONSERVED STRUCTURES

FIG. 18. Schematic map of the alphavirus genome showing the locations of four CSEs. The four CSEs are shown as black boxes in the genome organization. Below are shown the structures for CSE 1 and CSE 2, for which it is believed that the structure is as important as or more important than the linear sequence. In the case of CSE 3 (the subgenomic promoter) and CSE 4 [the last 19 nt adjacent to the 3′ poly(A) tract], consensus sequences are shown.

for the function of the opal termination codon that is present between nsP3 and nsP4 in most alphaviruses. The readthrough rate of the SIN opal codon during cell-free translation has been found to be 10 to 20% at 30°C and <5% at 40°C (92, 300, 487). Assuming that the readthrough rate is comparable in infected cells, for alphaviruses having the opal codon there will be 5- to 20-fold more proteinase molecules present early in infection than there are nsP4-containing replicase complexes. This suggests that the function of the termination codon is to produce P123 proteinases that will act in trans to accelerate the processing of minus-strand replication complexes containing P123 and nsP4 into plus-strand replicases containing cleaved products from P123. It is intriguing that such acceleration appears to be particularly important at 40°C, a temperature at which the SIN P123-containing replication complex is totally unable to synthesize plus-strand RNA and at which readthrough is particularly inefficient, leading to a higher ratio of proteinase to replicase.

This model for the function of the opal termination codon is supported by results of Li and Rice (299), who studied the effect of replacing the opal codon in SIN with a sense codon. The sense codon-containing virus was found to grow well but to be delayed in total-RNA synthesis if the multiplicity of infection was moderate (≤20 PFU per cell), and under these conditions the wild-type virus containing an opal codon had a selective advantage. At high multiplicities (100 PFU/cell), the replication of the sense mutant was essentially indistinguishable from that of the wild type. These results are consistent with the hypothesis that at low multiplicities the conversion of P123-containing complexes into plus-strand replicases is delayed in the mutant but at high multiplicities many extra viral RNAs are translated, giving rise to proteinases active in cleaving P123. It seems probable from other studies that the total number of replicases inside the cell is limited, possibly because a host component is limiting, and that at multiplicities of 100 many of the translation products will not be able to form functional replicase complexes but could provide proteinases to cleave the functional complexes.

As described above, there is reason to postulate that sense and opal codons are in flux in any virus, and it would appear that regulation by the UGA codon is a subtle fine tuning of the virus replication cycle. The selective advantage of opal over a

sense codon could depend on the nature of the hosts and geographical constraints. It is intriguing that all three viruses containing a sense codon were isolated from the Ethiopian region and the conditions that give a selective advantage to a sense codon could be more prevalent in this region. It would be of interest to examine other alphavirus isolates for the presence or absence of the opal codon. It would also be of interest to determine whether P123-containing complexes from viruses that have a sense codon are able to synthesize plus-strand RNA more efficiently than P123-containing complexes from SIN containing an opal codon, thus compensating for the delay in cleavage of P123.

cis-ACTING SEQUENCE ELEMENTS IN ALPHAVIRUS RNAs

Conserved Nucleotide Sequence Elements

Identification of CSEs. One of the most active areas of alphavirus research over the last decade has been attempts to identify and characterize cis-acting nucleotide sequence elements in alphavirus RNAs required for replication of the genomic RNA, for transcription of the subgenomic RNA, and for RNA encapsidation. One of the earliest efforts to identify such elements utilized comparative nucleotide sequence analyses of alphavirus RNAs. In general, even regions of alphavirus genomes encoding conserved amino acid sequences have diverged significantly during evolution because codons encoding conserved residues have been randomized among those permitted by the degeneracy of the genetic code (524) (see, for example, Fig. 13). Thus, regions of nucleotide sequence conserved among alphaviruses must be of special significance. It seems clear that sequence elements that specifically interact with proteins will evolve slowly because any change in the sequence element would require compensating changes in the binding protein to maintain function. In the case of viral proteins that specifically recognize such viral nucleotide sequences, coevolution of the interacting components is possible but requires simultaneous changes in the element and the protein. In cases when cellular proteins recognize a viral sequence element, the amount of variation tolerated within the element may be severely restricted. In either case, these

interacting sequence elements should evolve more slowly than other viral sequences.

Four conserved sequence elements (CSEs) have been identified in the alphavirus genome (384–387), and their locations are illustrated schematically in Fig. 18. From their conservation and location, these elements were hypothesized to perform specific functions during viral RNA replication (reviewed in reference 524). A conserved stem-loop structure is found at the 5′ end of alphavirus RNAs, formed by the first 44 nt in the case of SIN. The complement of this sequence was hypothesized to serve as a promoter, at the 3′ end of the minus strand, for initiation of plus-strand RNAs from a minus-strand template. A 51-nt CSE found in the coding sequence for nsP1 is also capable of forming stem-loop structures. Although the function of this element is not clear, it has been suggested that it might serve as a copromoter for initiation of minus-strand synthesis on a genomic RNA template. The rationale for this hypothesis is that subgenomic 26S mRNA is not transcribed into minus strands in infected cells, suggesting that there is a sequence element not present in 26S RNA that is required for minus-strand synthesis. A 24-nt CSE found within the junction region was postulated to be the promoter for transcription of the subgenomic mRNA. The 19-nt CSE at the 3′ end of the RNA was hypothesized to serve as a promoter for initiation of minus-strand RNA synthesis on the plus-strand template.

Mutagenesis of CSEs. Mutational and mapping studies have been performed on the four CSEs described above. Here we consider mutational analyses of three of the conserved CSEs. The analyses of the promoter for the subgenomic RNA, which has progressed along different lines, will be considered separately below.

For the analysis of the 5′ and 3′ elements and the 51-nt element, many defined mutations, either point mutations or deletions, were introduced into the elements and the mutant viruses were examined for phenotypic defects. It was found that a simple examination of the plaque morphology of a mutant was not sufficient to determine if the mutation was deleterious. Although virus production by mutants which produced minute plaques or whose plaques took longer to develop was invariably impaired, many mutants were found that grew 3 or 4 orders of magnitude less well than the parental virus but formed plaques that were virtually indistinguishable from wild-type plaques (272, 370, 371). Thus, careful growth curves were required to determine if any given mutation was deleterious.

An analysis of the 5′ NTR in which the effects of 24 different deletions and substitutions were examined (370) revealed that the virus was very sensitive to changes in the first 44 nt, which are capable of forming a stem-loop structure, whereas the downstream nucleotides in the NTR could be deleted without effect. Deletion of nt 1, of nt 2 to 4, or of nt 5 was lethal; deletion of nt 8 or 36 resulted in ts virus; and numerous other deletions tested in the first 44 nt resulted in virus that grew poorly compared with the wild-type virus. These results suggested that it was the 3′-terminal structure in the minus strand that was important for promotion of plus-strand synthesis, as originally suggested by Ou et al. (386) and consistent with DI RNA findings described below. In a mutational analysis of the 51-nt CSE, 25 substitution mutants were examined, of which 21 did not involve a change in amino acid coding (371). Of the 21 silent changes, 19 were extremely deleterious for virus replication, resulting in a decrease in virus growth of 2 to 4 orders of magnitude. The 51-nt sequence is capable of forming two stem-loop structures that may be the functional elements; however, even changes that should not disrupt the structures were deleterious, and thus the linear nucleotide sequence is

(also?) important. Finally, a study of the 3′ NTR in which 42 different substitutions and deletions were examined (272) showed that the virus was particularly sensitive to changes in the 19-nt CSE and that substitutions or deletions in this element often decreased the ability of the virus to replicate by 2 orders of magnitude or more. The entire 3′ NTR was important for virus RNA replication, however, because deletion of large areas upstream of the 3′-terminal CSE was deleterious (but not lethal); such deletions were more deleterious in mosquito cells than in chicken cells.

From these three studies, certain common themes emerged. One was that most of the nucleotide substitutions attempted led to viable virus. Thus, although these sequence elements are highly conserved among alphaviruses, many changes are tolerated, although in virtually all cases the virus grew less well than the parental virus and would be selected against during passage in nature. Second, many mutants were affected differently in their growth in chicken cells and in their growth in mosquito cells (or in their growth in mouse cells [271]). As one example, deletion of nt 16 to 19 of the 3′ NTR resulted in a mutant which replicated to threefold-higher titer in mosquito cells than did the parental virus but which grew less well in chicken cells. In nature, SIN alternates between mosquito vectors and higher vertebrates, and birds serve as an important vertebrate reservoir. The virus has evolved to replicate efficiently in both invertebrate hosts and a range of vertebrate hosts, and it seems clear that the CSEs are a compromise between the sequences that best adapt it for growth in the different types of hosts. The fact that these mutations affect growth in these cell lines so differently suggests that host cell factors, presumably proteins, bind to these three sequence elements, and since these sequence elements are all quite disparate, it is likely that different cellular proteins interact with the three different elements.

CSEs can be tissue specific. The finding that mutations in CSEs had different effects on virus growth in chicken and mosquito cells was followed up with detailed studies on four mutants with changes in the 5′ NTR and four mutants with changes in the 3′ NTR (271). These mutants were tested for rates of RNA synthesis and virus production in cultured mouse cells, mosquito cells, and chicken cells and were examined for virulence and for virus production in different tissues in mice. All of the mutants had defects in RNA synthesis, and virus production was host cell dependent. Mouse, chicken, and mosquito cells responded differently to each change from the wild-type sequence, indicating that the wild-type sequence elements have been adapted to interact with all of the natural hosts for the virus but are not necessarily optimal for a particular host. Growth in cultured mouse cells was found to be a better predictor for the virulence of any particular variant in newborn mice than was growth in chicken or mosquito cells; viruses that grew poorly in mouse cells were usually attenuated in their neurovirulence. Of particular interest was the finding that a mutant in which nt 18 to 25 of the 3′ NTR (numbered 3′ to 5′) were deleted demonstrated tissue-specific differences. This mutant grew slightly less well than the parental virus did in cultured mouse cells but grew to ninefold higher titer than the parental virus did in mouse brain (and was correspondingly quite virulent for newborn mice). Thus, the host proteins that interact with these CSEs appear to be tissue specific as well as species specific, and this may serve to modulate the tissue tropism of the virus.

Cellular proteins bind to CSEs. Studies described above suggested that cellular factors, presumably proteins, bind to alphavirus CSEs, presumably to promote RNA replication. To test this, minus-sense riboprobes of 60 or 120 nt corresponding

to the 3′ end of the minus strand of SIN genomic RNA were mixed with extracts from uninfected or infected chicken cells. Cellular proteins were found to bind to these probes, resulting in a shift in the electrophoretic mobility of the probe (390). The affinity constants for the binding (Ks ≈ 10⁸) were similar to those found for the binding of sea urchin proteins to (DNA) sequence elements that lead to altered transcriptional states and result in embryonic development (58), and thus the binding reaction is potentially significant for virus RNA replication. One riboprobe with a single nucleotide deletion bound more tightly to the cellular proteins than did the wild-type probe, resulting in a threefold-longer half-life for the complex, indicating that the interaction is sequence specific.

Chicken cellular proteins binding to the RNA probes were labeled by being cross-linked to the bound ³²P-labeled riboprobe by irradiation of the complexes with UV light, followed by RNase treatment. The labeled proteins were then displayed on acrylamide gels. The major labeled products were proteins of 42 and 44 kDa, which could be two forms of the same protein.

Pardigon et al. (389) also identified proteins of 50 and 52 kDa from mosquito cells that bound specifically to the SIN probes described above with an affinity approximately equal to that of the chicken 42- and 44-kDa proteins. Using a number of different probes, these authors found that within the first 250 nt at the 3′ end of SIN minus-strand RNA there were three high-affinity binding sites and one low-affinity site for the mosquito proteins, which may interact cooperatively. High-affinity binding to the 3′-most binding site required the first 15 nt of the SIN sequence, suggesting that binding of these cellular proteins represents an important step in the initiation of RNA replication. The same mosquito proteins were also found to bind specifically to riboprobes from SF and RR, demonstrating that binding of this protein is a general phenomenon in alphavirus infection.

The binding protein from mosquito cells has been purified and partially characterized by amino acid sequence analysis (388). The sequence obtained to date suggests that this protein is the mosquito homolog of the La protein. This finding should make it possible to determine whether the mammalian or avian proteins are in fact La.

In parallel studies, Nakhasi et al. (367) found that a 3′-terminal minus-strand riboprobe from SIN was bound by proteins of 79 and 56 kDa in extracts of uninfected Vero cells and by these same two proteins plus a protein of 97 kDa in extracts of rubella virus-infected Vero cells. Similar results were obtained with riboprobes representing the 3′ minus strand of rubella virus. Competition experiments indicated that the probes competed for the same proteins, indicating that they probably serve a similar function in the replication of the two viruses. Why different proteins were found by Nakhasi et al. (367) and by Pardigon and Strauss (390) is not clear and requires further investigation; it may be that a number of cellular proteins are bound in the formation of the initiation complex and that differences in the method of preparing the extracts result in different amounts of these binding proteins. Nakhasi et al. (368) had previously reported that cellular proteins also bind to the 3′ end of the rubella virus plus strand, where they could play a role in the initiation of minus-strand transcription; one of these proteins appears to be the same as the 56-kDa protein identified as binding to the 3′ end of the minus strand (367).

Promoter for subgenomic RNA synthesis. The promoter for subgenomic RNA synthesis, originally postulated to be a 21-nt CSE found in the junction region of the RNA (384), has now been precisely defined in an elegant series of experiments.

SIN
↓
![Image Description](image.png)

AURA
↓
![Image Description](image.png)

WHA
↓
![Image Description](image.png)

WEE
↓
![Image Description](image.png)

EEE
↓
![Image Description](image.png)

VEE
↓
![Image Description](image.png)

ONN
↓
![Image Description](image.png)

RR
↓
![Image Description](image.png)

SF
↓
![Image Description](image.png)

MAY
↓
![Image Description](image.png)

NTs
0   100   200   300   400   500

FIG. 20. Schematic representation of repeated sequences in the 3′ NTRs of alphaviruses. For SIN, the repeated element shown is approximately 40 nt long and contains two motifs (shaded differently). This repeated element also appears in AURA, WHA, and WEE, and the individual motifs are found in EEE as separate elements. Among the remaining alphaviruses, the repeated units share no sequence similarity from virus to virus, as indicated by different types of shading. Both the lengths of the repeats and the overall lengths of the 3′ NTRs have been drawn to scale. The downward-pointing arrow indicates the termination codon at the end of the structural protein ORF. Sequence data for the construction of this figure are found in the references cited in Table 1.

rubella virus, and the relative efficiencies of the promoters were tested with the double subgenomic system. Almost all alphavirus promoters tested were recognized by the SIN RNA polymerase with various degrees of efficiency (Fig. 19). Interestingly, the promoter for RR was used more efficiently than that for SIN when the assays were performed in infected hamster (BHK-21) cells. One possible interpretation of the results is that the promoter strength is regulated by the virus because of the need to balance the production of genomic RNA and subgenomic RNA following infection and that an overly strong promoter is selected against. A second possible interpretation is that the optimal promoter sequence might depend on the normal host cell of the virus (RR is primarily a virus of mammals, whereas SIN has birds as its primary vertebrate reservoir), possibly because a host protein is involved in promoter recognition. It will be necessary to test the strengths of the promoters in cells derived from different tissues or from different animals to examine this hypothesis.

Involvement of host proteins in the recognition of the subgenomic promoter was suggested in studies by Durbin et al. (103). These authors found that a mutant of SIN with a nucleotide substitution in the subgenomic promoter led to decreased synthesis of 26S RNA in mosquito cells but not in chicken cells (the mutation is in the region encoding the C terminus of nsP4 but does not lead to an amino acid substitution). Interestingly, the decreased synthesis of 26S RNA in mosquito cells led to increased cytopathology and to the formation on mosquito cells of plaques that were clearer and better defined.

There is also evidence that viral proteins are required for the recognition of the subgenomic promoter. Grakoui et al. (162) inserted 3 nt into the wild-type SIN subgenomic promoter and found that transcription of subgenomic RNAs was dramatically reduced to the extent that the resulting virus was barely viable.

The authors found that second-site revertants which grew well although retaining the 3-nt insertion were easily obtained. At least two of these revertants recognized the mutant promoter more efficiently than they recognized the wild-type promoter, suggesting that the revertants contain suppressor mutations in viral proteins that promote subgenomic RNA synthesis by binding to the subgenomic promoter and that these suppressor mutations have adapted the viral protein to the mutant promoter. The finding that SIN recognizes the different promoters with differing efficiencies (note especially that SIN recognizes one of the two SF promoters very poorly) also indicates that a SIN protein is involved in promoter recognition (Fig. 19).

Repeated Sequence Elements in the 3′ NTR

There are repeated sequence elements 40 to 60 nt in length in the 3′ NTRs of almost all alphaviruses. These elements are not conserved among alphaviruses: although almost all alphaviruses have repeated sequences, their sequence and length are different in different viruses (387), as illustrated schematically in Fig. 20. These elements may be useful in virus classification.

In the 3′ NTR of SIN there are three copies of a 40-nt repeated element (387). Sequence analyses of a number of strains of SIN isolated from widely separated geographic areas showed that all isolates examined had three copies of the repeated element, whose sequences were much more highly conserved than the sequence in other regions of the 3′ NTR (with the notable exception of the 19-nt 3′-terminal sequence) (485). As an example, an Australian strain and an Egyptian strain of Sindbis virus diverged by 18% in the 3′ NTR sequence, but this overall divergence resulted from 6% divergence within the repeated elements and 24% divergence elsewhere. Furthermore, the SIN-related viruses WHA and AURA also have three copies of the same element, confirming

which are not translated but which do replicate vigorously (360, 362), and is consistent with observations that RNAs of at least some other virus families have (unrelated) sequence elements in the 3' NTRs that bind cellular proteins and regulate translation of the RNA (279).

Sequences in DI RNAs Define Essential *cis*-Acting Elements

DI RNAs replicate and are packaged in the presence of helper virus and thus retain all *cis*-acting elements required for this. One DI RNA of SIN and two DI RNAs of SF have been sequenced in their entirety, and partial sequences or sequence analyses by other techniques have also been reported, in attempts to define the *cis*-acting elements required for replication and packaging. The sequences found in these DI RNAs are illustrated schematically in Fig. 22 in comparison with the sequences present in the viral genomes.

All naturally occurring SIN and SF DI RNAs contain a minimum of 50 nt from the extreme 3' end of the genome (5, 362). This sequence contains the 3' CSE that is hypothesized to be required for initiation of minus-strand copies on the plus-strand RNA template. Deletion analysis of a SIN DI RNA showed a precise requirement for the 3'-terminal 19 nt of the genome; deletion of the entire sequence derived from the 3' NTR except for the 3'-terminal 19 nt resulted in active DI RNA in these experiments, but deletion of 2 additional bases (leaving a 17-nt element) or of the entire element resulted in an inactive RNA (295).

Unlike the 3' ends, the 5' ends of SIN DI RNAs are not homogeneous and three unrelated 5' termini have been found, all of which support DI RNA replication. Not surprisingly, the first of these is the normal 5' end of virion RNA (550), such that the DI RNA contains the 5' and 3' ends of the genome with deletions and rearrangements in the middle. Deletion of nt 16 to 26 in the 5' NTR abolished function (551), and by analogy with results with complete virus (370), it seems likely that the 5'-terminal 44-nt structure is required for DI RNA replication. A second 5' sequence element found in SIN DI RNA consisted of nt 10 to 75 of a cellular tRNA<sup>Asp</sup> sequence (361) (Fig. 22). This sequence must have originated by recombination with host cell RNA, and this sequence element must fulfill the functions of the 5' element required for RNA replication that is normally present in virion RNA. The fact that this element was found in more than one independent DI RNA isolate suggests that it supports RNA replication more efficiently than the normal 5' end of the RNA does. It is intriguing that tRNAs have a defined structure and that a structure at the 5' end of the genomic RNA, rather than a linear sequence, is thought to be important for function. The third 5' element found on SIN DI RNA was a sequence derived from the first 142 nt at the 5' end of the 26S subgenomic RNA (550); the 26S sequence had a deletion of nt 25 to 66, and this deletion was found to be necessary for amplification of the DI RNA (551). It appears that the 5' end of SIN RNA is not optimized for RNA replication, perhaps because of the requirement that the 5' end also function as a leader for translation. SIN DI RNAs are not translated and thus have the luxury of substituting the 5' end with one that is more efficient for replication but which may not support translation. It is also possible that the alternation of the virus in vertebrate and invertebrate cells constrains the type of 5' end of the viral RNA; DI RNAs sequenced to date have arisen in vertebrate cells, and studies of their replication in mosquito cells are limited.

The 5' ends of SF DI RNAs have not been as well characterized as those of SIN DI RNAs. The 5' ends of SF DI

A) Semliki Forest Virus DI RNAs

length 2286; complexity 1169

DIIb - [Diagram with sequences and annotations]

SF CAP - [Diagram with sequences and annotations]

DIIa - [Diagram with sequences and annotations]

length 1652; complexity 680

B) Sindbis Virus DI RNA

SIN CAP - [Diagram with sequences and annotations]

DI25 - [Diagram with sequences and annotations]

length 2310 (2244 + 66nt tRNA<sup>Asp</sup>); complexity 1557 (+ 66nt tRNA<sup>Asp</sup>)

FIG. 22. Diagram of the locations in the alphavirus genome of sequences found in DI RNAs. Maps are shown for SF and two SF DI RNAs (281, 282) and for SIN and its best-characterized DI (360, 362). (A) The SF genome is shown to scale; the two long ORFs are indicated by boxes, and the boundaries between the proteins are shown by vertical lines. Sequences found in two DI RNAs are mapped on the genome and illustrated above and below expanded fourfold. The maps of the DI RNAs are also expanded fourfold relative to the SF map. Some sequence elements are repeated three or four times; the exact 5′ sequence of the SF DI RNAs has not been reported. (B) A similar analysis of a SIN DI RNA. In this case, the 5′ end of the DI consists of a 66-nt sequence derived from a cellular tRNA<sup>Asp</sup>. Other conventions are as in panel A.

RNAs were found to be heterogeneous and different from the 5′ end of the genomic RNA (403), suggesting that the situation may be analogous to that for SIN DI RNAs.

In the middle of the DI genome are sequences derived from internal regions of the virus genome, mostly from the 5′ region, and there is little or no sequence present in DI RNA that is not derived from the virus. Early sequence studies revealed that DI RNAs of either SIN (360, 362) or SF (281, 282) contained repeated and rearranged genomic sequences of a complexity much lower than that of the parental virus; repeated elements are of particular interest and are presumably selected because they enhance the replication and packaging of the DI RNA. The sequences present in one SIN DI RNA sequenced in its entirety (Fig. 22) consisted of 5′ tRNA<sup>Asp</sup> followed by SIN nt 29 to 546 (which contain the 51-nt CSE at nt 154 to 205), an internal region containing SIN nt 647 to 1226 and 10239 to 10668, and the 3′-terminal sequence (nt 11655 to 11703). The internal region had two copies each of nt 350 to 490 and 746 to 1226. Deletion experiments suggested that the sequence from nt 746 to 1226 contains signals for both RNA replication and packaging (295, 599). At least one copy of the 5′ part of this element (nt 746 to 947) was required for efficient replication of the DI RNA, and one copy of the 3′ 279 nt (nt 948 to 1226) was required to allow packaging of DI RNA. This region was also shown to bind capsid protein in in vitro binding assays, and the two cis-acting functions of this 481-nt domain, encapsidation of the genomic RNA and promotion of RNA replication, could both require binding of capsid protein (599).

The 51-nt CSE was also found to be necessary for efficient DI RNA function. Deletion of this element gave rise to a viable DI RNA that was amplified and packaged (295) but which replicated more slowly and competed less efficiently than the parental DI RNA did (465).

Finally, deletion analysis showed that nt 647 to 745 and 10292 to 10668 were dispensable for DI RNA function and may simply represent sequence that has failed to be deleted or filler that increases the size of a minimal DI RNA (295).

Two SF DI RNAs have been sequenced in their entirety except for the extreme 5′ end (Fig. 22). One (DIIa) had a complexity of only 680 nt (total length, 1,652 nt): only SF RNA residues 11 to 317 and 2728 to 2994 and the 3′ 106 bases were present (281). nt 37 to 317, 2729 to 2857 and 2918 to 2993 were present in three copies in the DI RNA. The second DI RNA (DIIb) had a complexity of 1,169 nt (total length, 2,286 nt) and

contained residues 79 to 137, 164 to 684, 2628 to 3063, and 11216 to 11276 and the 3′ 103 bases; there were four copies of the sequence from 2737 to 3009 (282). Three additional SF DI RNAs have been examined by electron-microscopic heteroduplex analysis (5). Although this method is less precise than sequencing, the conclusion was reached that these three DI RNAs had three regions in common with one another and with the two sequenced DI RNAs. These were the 3′-terminal 80 nt, a sequence of 100 nt located around nt 200, and 90 nt near position 2740. The SF 51-nt CSE comprises residues 176 to 230. It is thus present in both sequenced DI RNAs (in three copies in DIa) and appears to be present in the three additional DI RNAs examined. The presence of this element in all SF and SIN DI RNAs examined suggests that this element is important for DI RNA function. The residues near nt 2740 may be required for packaging, and the importance of the 3′-terminal sequence has been commented upon above. It is of considerable interest that the only sequences in common between the SF and SIN DI RNAs are the 51-nt CSE and surrounding sequence and the 3′ end of the RNAs. Other signals for RNA replication and packaging in these two viruses (other than the 5′-end signals which differ in different DI RNAs) must therefore reside in nonidentical positions in the viral RNAs. It is of note that SIN DI RNAs have been found to cross-interfere with the replication of SF and that SF DI RNAs have been found to interfere with the replication of SIN, but interference was less efficient with the heterologous virus than with the homologous virus (600). Thus, some of the cis-acting signals are shared between the two viruses, consistent with the hypothesis that CSEs form such signals.

It is of interest to compare the sequence requirements for DI RNAs with those for the viral RNA (272, 370, 371). It seems surprising that deletion of the 51-nt CSE had such a limited effect on DI RNA function when single-nucleotide substitutions in this element had a profound effect upon virus replication, reducing the growth rate by 4 orders of magnitude. In like vein, certain deletions in the 3′ NTR reduced the virus growth rate by an order of magnitude or more but had no apparent effect on DI RNAs. Conversely, deletion of nt 16 to 26 in the 5′ sequence was lethal for DI RNA but had a less profound effect on the virus than did other substitutions or deletions which were dispensable for DI RNA function. It seems apparent that the functions required for DI RNA replication and packaging must be related to and overlap those for replication and packaging of virion RNA. However, it also seems clear that the replication of the viral RNA is more complicated and involved than is apparent from results with DI RNA replication.

### Packaging Signal in Alphavirus RNAs

Wengler et al. (606, 609) found that the SIN virus capsid protein would assemble into corelike particles in vitro in the presence of any RNA, including SIN 49S genomic RNA or 26S subgenomic RNA, tRNA, or even polyanionic polymers. This result was unexpected because SIN packages only the genomic RNA into virions, and other RNAs found in the cell, most notably 26S RNA, are not encapsidated. Recent results now suggest that the exclusivity of RNA packaging results from the presence of a specific encapsidation signal required for efficient formation of nucleocapsids in infected cells.

In vitro binding assays developed by Weiss et al. (599) showed that the RNA region between nt 746 and 1226 is specifically bound by the nucleocapsid protein of SIN, suggesting that this region contains a signal for encapsidation. One or more copies of this element have been found in all SIN DI

RNAs examined (360, 362). Weiss et al. (599) found that deletion of nt 948 to 1226 prevented packaging of DI RNA, and thus this region is functionally a packaging signal.

Although the sequence from nt 948 to 1226 was required for packaging of a DI RNA, alphavirus packaging appears to be more complicated than simply having or not having this packaging signal. Bredenbeek et al. (45) studied the packaging of a number of constructs with deletions between nt 746 and 1226 and that had either the SIN 5′ end or the tRNA<sup>Asp</sup> sequence found in a number of SIN DI RNAs. They found that the deleted constructs were packaged but that they were packaged less efficiently than constructs containing the region from nt 746 to 1226 and that the efficiency of packaging depended upon the particular deletion in the genome. Constructs with 5′ tRNA<sup>Asp</sup> were packaged more efficiently than were constructs having the 5′ SIN sequence, which may reflect differences in replication efficiency of the RNA. It appears that possession of the signal from nt 948 to 1226 is required for packaging of the RNA with the highest efficiency but that other sequences may influence packaging, that the size of the RNA is important, and that RNA lacking a signal can be packaged but with lowered efficiency.

It is of interest that the domain in SIN RNA that is required for efficient packaging of DI RNA is not present in the sequenced SF DI RNAs. The presence of a packaging signal in SF RNA has not been explicitly demonstrated, but expression of DI sequences in an SF-infected cell was shown to inhibit capsid assembly (226), and, as is the case for SIN, only the genomic RNA of SF is packaged in infected cells. Furthermore, SF DI RNAs that differ in their efficiency of packaging have been described, suggesting that these different DI RNAs contain packaging signals of different strengths, possibly as a result of differences in the number of copies of the packaging signal in the DI RNAs (237). It therefore seems most probable that the SF DI RNA does contain a packaging signal but that it is found in a different region of the SF genome from the packaging signal in the Sindbis genome. The sequences present in SF DI RNAs, described above, suggest that the SF packaging signal lies between nt 2737 and 2993.

As noted above, the putative packaging signals are present in multiple copies in DI RNAs. Smaller RNAs appear to be at a selective disadvantage during capsid assembly in vivo (237), and possession of multiple encapsidation binding sites might enable a DI RNA to be packaged more efficiently or at least to compete more effectively with genomic RNA during encapsidation.

In the case of AURA, a South American alphavirus that is serologically related to Sindbis virus, a packaging signal apparently resides in the 26S region of the genome (436) because 26S RNA is packaged into capsids and is found in virions. Packaging is fairly efficient, but, analogous to the case for packaging of DI RNAs, 26S RNA is packaged less efficiently than genomic RNA, possibly because of the size differences. Thus, it appears that for three different alphaviruses, SIN, SF, and AURA, the primary packaging signal is found in different positions in the RNA genome.

Although the primary encapsidation signals in different alphaviruses appear to be present in different positions in the genome, these signals appear to be related to one another. A chimeric alphavirus genome consisting of the RR genome in which the capsid protein gene has been replaced with that for the SIN capsid protein gene is efficiently encapsidated by the encoded SIN capsid protein, showing that the capsid protein of SIN can encapsidate the RNA of RR (270, 312).

It seems clear from these results that a packaging signal in alphavirus RNA is required for efficient packaging of RNA

into capsids in infected cells. In support of this, Suomalainen et al. (536) found that only very small numbers of virus-like particles were assembled in the absence of SF genomic RNA when SF capsid protein and glycoproteins were produced in large amounts in cells by using the vaccinia virus-T7 expression system. The fact that some particles were assembled, however, together with the results of Breedenbeck et al. (45) that SIN RNA lacking the putative packaging signal was packaged but with lowered efficiency, suggests that in the absence of a packaging signal, capsids will assemble with low efficiency, consistent with the results of Wengler et al. (606, 609).

### Chimeric Alphaviruses

Interactions between different domains of the alphavirus genome have been studied by constructing chimeric viruses in which part of the genome is derived from SIN and the remainder of the genome is derived from RR. In these chimeras, proteins encoded by one virus must interact with proteins or sequence elements encoded by the other virus. If these interactions are faulty, the virus will be attenuated and the nature of interacting components can be identified. Studies with chimeric viruses that probed the interactions between nucleocapsids and glycoproteins, between capsid protein and RNA, and between structural proteins and the RNA synthesis machinery were described above. Here we describe chimeras that probe interactions between the 5′ and 3′ NTRs and the rest of the genome.

Chimeric RR-SIN viruses in which the entire sequence was from one virus except for replacement of either the 5′ or 3′ NTR with that of the other virus were constructed (273). Replacement of the 3′ NTR of SIN with that of RR resulted in a chimeric virus whose growth was essentially indistinguishable from that of SIN in either mammalian cells or mosquito cells. The reciprocal chimera, RR having the SIN 3′ NTR, was indistinguishable from RR. Thus, although individual substitutions within the 3′ NTR of SIN led to virus that grew poorly (272), replacement of the whole unit resulted in virus that was essentially wild type. This finding is consistent with the hypothesis that a major function of this NTR, other than the 3′-terminal CSE, is to interact with cellular proteins to promote viral RNA replication or translation. Although the essential features of the 3′-terminal CSE may be conserved between SIN and RR to the extent that interaction with viral proteins might be normal, the remainder of the 3′ NTR is very different between the two viruses.

The converse chimeras in which the 5′ NTRs were exchanged were not wild type, however. Replacement of the entire 5′ NTR with that of the other virus gave rise to viruses that were viable but that grew less well, suggesting that there is an interaction between the 5′ NTR (or its complement in the minus strand) and a virus-encoded protein. Chimeras within the 5′ NTR such that part of the NTR was derived from SIN and part was derived from RR were not viable, suggesting that these chimeras might disrupt secondary features required for function.

### Alphaviruses as Vectors

The identification of *cis*-acting elements required for RNA replication and packaging makes it possible to design alphavirus expression vectors that can express any gene of interest in the cytoplasm of any cell permissive for virus replication (28, 45, 46, 217, 302, 623). Because of the toxicity of the virus for cells, these vectors will be useful primarily for transient expression, but an alphavirus expression system has several advantages. First, the high level of alphavirus replication and the interference of the viruses with host functions lead to the production of very large amounts of expressed product. Second, full-length clones of alphavirus cDNAs can be easily manipulated and infectious RNA can be produced by in vitro transcription, and the cloning steps required to produce the RNA vectors are straightforward. Third, alphaviruses replicate exclusively in the cytoplasm, and there is no possibility of adventitious splicing.

Several approaches have been explored to date. In the approaches illustrated schematically in Fig. 23, the gene to be expressed is placed downstream of the viral subgenomic promoter such that a subgenomic mRNA is transcribed and translated into the protein of interest. The RNA vector replicates to high copy number inside the cell, and large quantities of the mRNA are produced, leading to production of large amounts of the protein of interest. The factors to be considered in the design of the vector system are the method used to introduce the RNA into a susceptible cell, the nature of the vector carrying the gene to be expressed, and the design of a packaging system if it is desired to package the vector for high-efficiency infection.

The most effective system for expressing a foreign protein is one in which the structural protein genes of the virus (4.1 kb) have been deleted and replaced with the gene to be expressed (Fig. 23B); such systems have been described for both SIN (45, 623) and SF (28, 302). The RNA transcribed in vitro from the resulting construct can be introduced into a cell by lipofectin treatment (623) or electroporation (45, 302). This RNA is self-replicating since it encodes the viral replicase, and the vector leads to high rates of production of subgenomic mRNA encoding the protein to be expressed (reviewed in reference 46). In experiments with the SF vector, protein production continued for up to 75 h and resulted in the expressed protein constituting as much as 25% of total cell protein (302).

To package the RNA so as to enable it to be delivered with high efficiency to any cell sensitive to virus infection, cells can be coinfected with helper virus or cotransfected with a defective RNA that leads to production of the virus structural proteins. Xiong et al. (623) used coinfection of helper virus and a DI construct containing the heterologous gene to be expressed (Fig. 23A); the virus resulting from the mixed infection consisted of a mixture of wild-type virus and virions containing the vector RNA. In contrast, Liljeström and Garoff (302) and Berglund et al. (28) used a deleted form of the SF genome or of the SIN genome, respectively, as a helper (Fig. 23B). In this helper, the packaging signal for the RNA had been deleted such that only the vector RNA could be efficiently packaged, and the virions from the mixed infection contained almost exclusively vector RNA. Because infection of a large fraction of the cells is possible by using electroporation, coinfection with the two RNAs can be efficient and high yields of the packaged vector can be obtained. Bredenbek et al. (45) developed a number of deleted helpers for packaging of SIN that were packaged or not packaged, depending on the deletion, allowing the investigator to choose a packaged or non-packaged helper (Fig. 23B).

DI RNAs have also been used as packaging helpers. DI RNAs contain all of the *cis*-acting elements required for RNA replication and packaging. Engineering of a DI RNA to contain the structural protein-coding region downstream of the subgenomic promoter led to an efficient helper RNA. The virus population produced in this case consisted of packaged vector and packaged DI helper, and many virus particles contained both RNAs in the same particle, enabling them to form a plaque (149).

A. DI-CAT construct with wildtype helper virus

CAP ⭐ Replicase —— Structural —— Aₙ  
SG  

CAP ⭐ CAT —— Aₙ  
SG  

B. CAT replicon with either packaged (1) or nonpackaged (2) helper constructs

CAP ⭐ Replicase —— CAT —— Aₙ  
SG  

(1) CAP ⭐ Structural —— Aₙ  
SG  

(2) CAP ⭐ Structural —— Aₙ  
SG  

C. Packaged expression vectors with two subgenomic promoters

CAP ⭐ Replicase —— Structural —— Aₙ  
SG  

CAP ⭐ Replicase —— Structural —— Aₙ  
SG  

Replication promoters  
Subgenomic promoter  
Encapsidation signal  
Foreign Gene  

FIG. 23. Alphavirus expression vectors. (A) In one form of expression system, the heterologous gene is inserted into a packaged DI RNA under the control of an alphavirus subgenomic promoter. Cells are infected with wild-type helper virus and transfected with the DI RNA. The yield from such an infection is a mixture of wild-type virus and packaged DI RNAs. (B) The foreign gene replaces the genes for the structural proteins in a full-length alphavirus RNA construct. Transfection of this RNA leads to replication of the genome and transient expression of the foreign gene upon transcription of subgenomic RNA. The replicon can be packaged by cotransfection with a DI RNA construct that expresses the structural genes. DI RNA will also be packaged if it contains the packaging signal (construct 1) but not otherwise (construct 2). (C) The foreign gene can be inserted into a full-length nondefective construct under the control of a second subgenomic promoter. Because of size limitations on packaging, such constructs are unstable if the foreign gene is much larger than 2 kb. In these diagrams neither the SIN genomes nor the foreign inserts are drawn to scale.

Another approach was used by Hahn et al. (176, 182) (Fig. 23C). These investigators used infectious SIN as a vector to express small peptides derived from heterologous viruses, in order to map antigenic determinants of the second virus. For this, a second subgenomic RNA promoter was inserted into the 3′ NTR of the virus, positioned upstream of the gene for the peptide to be expressed. The resulting virus RNA was infectious and was packaged efficiently. During infection, two subgenomic RNAs were produced, the smaller of which was translated into the heterologous peptides to be studied. Although large inserts (4 kb) can be placed in the 3′ NTR and the resulting viral genome can be packaged, large inserts are unstable, and this system is effectively limited to expression of small inserts (<2 kb) (46). Raju and Huang (411) have also shown that this vector system can be used to express the CAT gene.

In a different approach to the use of SIN as a vector, London et al. (311) inserted a neutralization epitope from Rift Valley fever (RVF) virus randomly into the sequence for SIN glycoprotein PE2. They recovered infectious chimeras in which the RVF epitope had been inserted into E3 or into one of two positions in E2. The E2 chimeras stimulated a weak protective response in mice to RVF virus. Thus, the SIN glycoproteins can be used to express heterologous epitopes, but the potential of the system has yet to be explored in detail.

An interesting use of the SIN subgenomic promoter to express a foreign gene was described by Olivo et al. (377). The SIN subgenomic promoter was placed upstream of the luciferase gene and downstream of a Rous sarcoma virus promoter, and this construct was used to stably transform cells. Upon infection of the transformed cells with SIN or defective SIN variants that expressed a functional RNA replicase, high levels of luciferase were produced. Thus, such transformed cells produce significant amounts of the product under the control of the SIN promoter only upon infection with SIN. In addition to the potential value of such cells for the detection of SIN, such an approach could be useful in cases when expression of a gene product was desirable only after virus infection.

VIRUS-HOST INTERACTIONS

Host Cell Receptors for Alphaviruses

The alphaviruses have an enormous host range that comprises both invertebrate hosts (mosquitoes or other hemato-

phagous insects that serve as vectors) and vertebrate hosts (many species of birds and mammals [and some alphaviruses have also been isolated from amphibians or reptiles]) (66, 166, 372, 402). Within their hosts they replicate in a wide variety of cells including neurons and glial cells, striate and smooth muscle cells, cells of lymphoid origin, synovial cells, and brown fat cells. The question then arises whether the viruses are using the same receptor throughout this wide host range; if so, the same receptor must be expressed on the surface of many different mosquito, avian, and mammalian cells. Studies to address this issue (described below) have suggested that alphaviruses use protein receptors; that different alphaviruses may use the same receptor or different receptors; that more than one receptor can be used by one virus, leading to cases in which the major receptors used by one virus to enter different cells are different; that the nature of the receptors used determines in part the virulence of the virus; and that one or a few amino acid changes in the envelope glycoproteins can lead to utilization of different sets of receptors.

Early studies of alphavirus receptors. The number of receptors on the surface of a cell to which an alphavirus can bind has been examined by electron microscopy (30) and by quantitation of labeled virus bound to cells (125, 494). The number of receptors depended on the virus, the cell line, and even the individual cell, with values from \(4 \times 10^4\) to \(4 \times 10^6\) per cell being found. The apparent binding constant for SF was found to be \(10^{10}\) to \(10^{11} \mathrm{M}^{-1}\).

An early study that examined the nature of the receptors for alphaviruses reached the conclusion that the receptors for SIN were proteinaceous, at least in part, because treatment of cells with proteases, but not with phospholipases or neuraminidases, greatly reduced or abolished virus binding (494). Two strains of SIN were used, a neurovirulent strain, AR86, isolated from South Africa and a nonvirulent strain, AR339, isolated from Egypt. These two strains did not compete with one another for binding, and it was concluded that these viruses used completely different receptors. Two different neuronal cell lines were found to possess many more receptors for the neurovirulent strain than for the avirulent strain and in fact possessed more receptors for the neurovirulent strain than did nonneuronal cells for either strain. Thus, the nature of the receptors used appeared to determine at least in part the virulence of the virus. Finally, SIN AR86 and EEE partially competed with one another for binding, suggesting that these two neurovirulent alphaviruses shared some of their receptors; the fact that the competition was only partial suggested that each virus can use more than one receptor.

Maassen and Terhorst (319) attempted to identify receptors for SIN on lymphoblastoid cell lines, using two of the same lines utilized by Helenius et al. (200) in their studies of SF (see below). They showed that the SIN glycoproteins could be cross-linked to a cell surface protein of 90 kDa and concluded that this protein must be a SIN receptor or be found near the SIN-binding site.

Using a different approach, Sanders et al. (441) found that chicken fetal antigen determinant 9 served as a receptor for the binding of Sindbis virus to avian erythrocytes and is the receptor required for hemagglutination by the virus. Whether this receptor for hemagglutination is related to receptors used by the virus for entry into susceptible cells is not known.

Although these studies and other studies described below have suggested that SIN uses protein receptors, studies by Symington and Schlesinger (537, 538) suggested that charge is also important for binding to cells. They isolated a SIN variant that was more negatively charged and was able to bind to and infect mouse plasmacytoma cells 10-fold better than the wild-type virus. The increased binding appeared to be due to the increased charge of the virus because pretreatment of the plasmacytoma cells with heparin enabled the wild-type virus to bind almost as well as the variant. One interpretation of these results is that a charge repulsion must be overcome for the virus to bind tightly to a protein receptor.

Helenius et al. (200) suggested that antigens of the MHC, HLA-A and HLA-B in humans or H-2K and H-2D in mice, served as receptors for SF on the basis of their findings that these MHC antigens bound to SF glycoproteins. This binding was demonstrated in several different ways, including the isolation of specific complexes between the spike glycoproteins and MHC antigens. It is important to note, however, that MHC antigens are expressed in large amounts by the cells used by Helenius et al. (200) but that these cells do not serve as efficient host cells. Subsequently it was found that murine F9 and PCC4 teratoma cells, which do not express H-2 MHC antigens, as well as H-2-negative murine lymphoblastoid cells, are nonetheless sensitive to infection by SF, demonstrating that H-2 antigen expression is not required for SF infection (376). Daudi cells, which lack HLA antigens, were found to be resistant to SF not because of failure of the virus to bind to and enter cells but because of a subsequent block to replication. Thus, it is unclear whether the binding of SF to MHC is functional for virus entry. The many indications that alphaviruses can use more than one receptor make it possible that MHC antigens serve as only one of multiple receptors used by SF.

Anti-idiotypic antibodies as antireceptor antibodies. Anti-idiotypic antibodies can sometimes be used to identify receptors that bind to ligands, as reviewed by Gaulton and Greene (147). Two alphavirus studies used neutralizing monoclonal antibodies (MAbs) reactive with E2 of SIN to generate polyclonal anti-idiotypic antibodies in rabbits, and the anti-idiotypic antibodies obtained functioned as antireceptor antibodies (555, 582). Wang et al. (582) found that three such anti-idiotypic antibodies bound to chicken cells, as shown by fluorescence-activated cell-sorting assays, and blocked virus binding by about 50%. One of the antibodies, anti-Id 49 made to MAb 49, bound with much higher affinity than the other two. The finding that an anti-idiotypic antibody made to a purified MAb selected to be reactive with SIN E2 would bind to chicken cells and block virus binding is very suggestive evidence that it is functioning as an antireceptor antibody. Anti-Id 49 also immunoprecipitated a protein of about 63 kDa from chicken plasma membrane preparations, which is a putative receptor molecule. The results with the anti-idiotypic antibodies suggested that there are at least two receptors in chicken cells and that the anti-idiotypic antibody recognizes only one of these because (i) the antibody blocked virus binding by only 50% and (ii) a variant of SIN called v49, selected to be resistant to neutralization by MAb 49, was not blocked in its binding by anti-Id 49. This variant has a change in E2 (R-214 → P) that renders it unable to bind MAb 49; it presumably is unable to bind to the receptor recognized by anti-Id 49 and instead enters cells by the second receptor recognized by wild-type virus.

It was also found by Wang et al. (582) that anti-Id 49 did not bind to BHK cells and had no effect on virus binding to BHK cells. Evidently, BHK cells do not express a homolog of the chicken 63-kDa protein on their surface that reacts with the anti-idiotypic antibody, and the receptors used by the virus to enter hamster cells appear to be different from those used to enter chicken cells.

Related results were reported by Ubol and Griffin (555). These authors found that an anti-Id 209 bound to murine N18

neuroblastoma cells and blocked virus binding by 50%. This antibody immunoprecipitated proteins of 110 and 74 kDa from N18 cells; these proteins may therefore function as receptors for SIN. These proteins were present on mouse brain cells at birth, as shown by the binding of anti-Id 209 to these cells, but by 4 days after birth half of the cells had ceased expression of proteins reactive with the anti-idiotypic antibody. It was suggested that the age-dependent expression of this protein could be responsible in part for the age-dependent susceptibility of mice to fatal encephalitis caused by SIN (but see also the discussion of apoptosis below).

Laminin receptor as a mammalian receptor. Wang et al., observing that their antireceptor antibody was effective for chicken cells but not for BHK cells, set out to obtain an antireceptor MAb that would block SIN binding to BHK cells (580). Mice were immunized with whole BHK cells, hybridoma supernatants were screened for virus-blocking activity, and a MAb that blocked SIN binding to BHK cells by up to 80% was isolated. This MAb, called 1C3, bound to BHK cells, as expected, and immunoprecipitated a protein of apparent molecular size 67 kDa from membrane fractions of BHK cells. Screening of λgt11 libraries with MAb 1C3 revealed that it bound to the C-terminal domain of a cell surface protein referred to the high-affinity laminin receptor. The nature of this protein is controversial. It was originally isolated as a 67-kDa protein that was able to bind to laminin with high affinity. Screening of cDNA libraries by Wang et al. (580) and by others (174, 613) identified an ORF encoding a protein of only 295 amino acids. There is no evidence for an mRNA encoding a longer ORF (413, 434, 613), and considerable evidence has been presented that the 67-kDa protein and the 295-amino-acid ORF possess amino acid sequence in common (64, 65, 581, 613, 614). It has been proposed that the 295-residue protein is a precursor to the 67-kDa form (reviewed in reference 65). Glycosylation is not involved in the modification (64, 174), and Castronovo et al. (64, 65) have proposed that the 295-residue protein is covalently linked to another protein to produce the 67-kDa form. Grosso et al. (174), in contrast, have proposed that the 295-amino-acid protein is unrelated to the laminin receptor and that the mRNA for the 67-kDa protein is yet to be identified.

Castronovo et al. (65) found that antibodies made to synthetic peptides derived from the C-terminal part of the 295-residue ORF (specifically, residues C-terminal to position 107) would bind to intact cells, whereas antibodies to synthetic peptides from the region from positions 1 to 103 reacted only with permeabilized cells. This, together with a computer analysis (413), supported a model in which the N terminus of the 295-residue protein is intracellular, residues 86 to 101 form a membrane-spanning domain, and the N terminus is extracellular. This orientation is illustrated schematically in Fig. 24. Castronovo et al. (65) also found that a synthetic peptide comprising residues 161 to 180 bound to laminin and hypothesized that this sequence constituted the laminin-binding domain of the receptor. MAb 1C3 used by Wang et al. (580) reacted with λgt11 clones containing residues 248 to 295, suggesting that SIN binds to this C-terminal region of the protein.

Transformation of hamster cells with vectors that overexpressed the 295-amino-acid ORF gave rise to cells that bound up to 4.5 times as much radiolabeled virus, and the efficiency with which the virus formed plaques on such cells was increased by up to sevenfold (580). Binding was saturable for both the parental cells and the cells overexpressing the 295-residue ORF (Fig. 25). Expression of antisense RNA led to decreased sensitivity to the virus. Cells transformed with the

FIG. 24. Schematic model for the orientation in the cell surface of the 295-residue protein that is believed to form part of the high-affinity laminin receptor. The amino terminus of the molecule is thought to be in the cytoplasm, and amino acids (aa) 86 to 110 are believed to constitute a transmembrane domain (65). The laminin-binding domain (65) and the MAb 1C3-binding domain (580) are indicated.

plus-sense or antisense constructs were shown to bind more or less MAb 1C3, respectively. The amount of laminin receptor on the cell surface was assayed by the binding of 1C3, by the binding of radioactive virus, and by the sensitivity of the cell to virus infection as measured by a plaque assay. This quantitation showed that binding of the virus varied linearly with the concentration of laminin receptor on the cell surface whereas the sensitivity to the virus measured by a plaque assay varied

![Graph](https://i.imgur.com/ExampleGraph.png)

FIG. 25. Overexpression of the 295-residue laminin receptor ORF results in increased binding of SIN to the surface of hamster cells. CHO cells were transformed with the 295-residue laminin receptor ORF in a high-efficiency mammalian expression vector (CHO-LR) or transformed with the vector only (CHO). Monolayers of each resulting cell line were tested for their ability to bind purified ³⁵S-labeled SIN in a saturation binding assay. After 1.5 h of incubation at 8°C, the monolayers were washed to remove unbound virus, dissolved in 0.5% sodium dodecyl sulfate, and assayed for radioactivity. Reprinted from reference 580 with permission of the publisher.

with the 1.4 power of the receptor concentration, suggesting that interaction with more than one receptor aids virus penetration. Thus, the high-affinity laminin receptor is functionally a receptor for SIN in BHK cells.

MAb 1C3 was found to block SIN binding to a number of other cultured mammalian cells tested, including murine, monkey, and human cells. It thus appears that the high-affinity laminin receptor is a major receptor for SIN virus in all mammalian cells. This receptor, or at least the 295-residue ORF studied to date, is highly conserved among mammals. There are only two amino acid differences between the hamster sequence and a human sequence or between the hamster sequence and a bovine sequence, and the hamster sequence is identical to a sequence obtained from mice (580). Because the laminin receptor is highly conserved in amino acid sequence among mammals and is widely expressed, it appears that the ability of the virus to infect many different mammals and to infect many different tissues within those mammals results at least in part from utilization by the virus of a highly conserved protein receptor.

Although the antibody blocked virus binding to many different mammalian cells, it had only slight effects (≤10%) on the binding of SIN to chicken cells. These results complement those found by Wang et al. (582) with anti-Id 49, and the major receptor used by the virus to enter chicken cells must be different from that used to enter mammalian cells. The 63-kDa chicken protein precipitated by anti-Id 49 is different from the chicken laminin receptor, because MAb 1C3 was found to precipitate a protein of 71 kDa, presumably the chicken laminin receptor, from chicken membranes. The sequence of the chicken laminin receptor (i.e., the 296-residue ORF corresponding to the mammalian 295-residue ORF) is now known (434), and it is ~98% identical to the mammalian gene. Thus, there is reason to believe that the chicken laminin receptor could be used by the virus as a receptor, but apparently there is a higher-affinity virus receptor, possibly present in greater amounts, in chicken cells that is preferred by the virus. These results suggest that the very broad host range of the virus is achieved in part by utilizing more than one protein receptor.

The extreme conservation of the laminin receptor between birds and mammals, which diverged >200 million years ago, suggests that the laminin receptor is conserved in all multicellular animals. Wang et al. (580) found that MAb 1C3 partially blocked binding of SIN to mosquito cells as well as to mammalian cells, suggesting that the sequence of the laminin receptor is conserved in mosquito cells and that the receptor serves as a virus receptor in these cells.

Thus, the results from several lines of study suggest that the very wide host range of the alphaviruses results in part from the ability of the virus to use a receptor that is highly conserved and in part from the ability of the virus to use more than one receptor on the surface of the cell. The relationships between the different proteins studied in the different laboratories are unknown. In most cases only the protein molecular weights estimated from acrylamide gel electrophoresis were determined, and side-by-side comparisons of these proteins are necessary to determine if they are all different.

Mosquito receptor for VEE. Ludwig and Smith (314) have recently developed a direct binding assay to search for cellular receptors in mosquito cells. Proteins from plasma membrane preparations were separated on acrylamide gels and transferred to nitrocellulose, and the blot was probed with radiolabeled VEE. The virus bound to a protein of 32 kDa, which was expressed on the surface of mosquito cells. This protein bound laminin and cross-reacted immunologically with the high-affinity laminin receptor, consistent with the results of Wang et

al. (580) that a protein in mosquito cells reactive with MAb 1C3 acts as a receptor for some alphaviruses. The exact correspondence between the 32-kDa protein expressed in mosquito cells, the product expressed from the 295-residue ORF, and the 67-kDa high-affinity laminin receptor remains to be determined. Ludwig and Smith (314) found that VEE and SIN competed for binding, that higher-molecular-weight proteins from mosquito cells cross-reacted immunologically with the 32-kDa protein and bound VEE and laminin, and that VEE bound to 32- and 67-kDa proteins from BHK cells, consistent with the hypothesis that the 32- and 67-kDa proteins share sequence identity.

Virus receptor-binding domain. Evidence is developing that the set of cellular receptors that an alphavirus can use can be altered by relatively minor changes in E1 and E2. In one study with SIN, two strains of SIN that were identical except for a single amino acid difference in E2 (G-172 → R) were examined for their ability to bind to cells of neuronal origin in culture; it was found that the more neurovirulent of the two strains, possessing G-172, bound more readily to neuronal cells than did the less virulent strain, although the two strains bound identically to BHK cells (552). In another study, variants of SIN that differed in their neurovirulence for mice were sequenced and used to construct recombinant viruses; it was shown that amino acid substitutions in E1 and E2 affected the neurovirulence of the virus (318). For at least some of these mutants it was shown that the virus replicated less well in the brain if it was less virulent. Other studies, by Johnston and colleagues (described in a later section), have also demonstrated that changes in E1 and E2 can alter the neurovirulence of SIN for mice. Although these studies do not directly demonstrate a difference in binding to cells, it is suspected that at least some of these changes affect the ability of the virus to bind to receptors expressed on neurons.

Changes in E2 of RR have also been shown to affect the ability of the virus to bind to and infect different cells. RR, in contrast to SIN, is maintained in nature in small mammals, primarily marsupials, and the virus infects chicken cells in culture very poorly. In one study, variants of the Nelson Bay strain of RR were selected by passaging in chicken cells (249). The passaged virus had single changes in E2: in two of three independent variants sequenced the change was N-218 → K, and in the third variant the change was E-4 → K. Weir and Kuhn (598) examined the effect of changing E2 N-218 to T, K, or R by site-specific mutagenesis of a full-length cDNA clone of the T48 strain of RR, from which virus can be recovered (273). The parental T48 was found to infect only ~2% of chicken cells in culture, by using an immunofluorescence assay with anti-RR antibodies to determine which cells were infected. Virus with T-218 or K-218 infected ~6% of chicken cells, but virus with R-218 infected ~40% of chicken cells. Thus, the single change N-218 → R in E2 led to a 20-fold increase in the ability of RR T48 to infect chicken cells. Serial passage of the R-218 virus in chicken cells led to selection of a strain of RR that retained R-218 but also had two other changes, C-153 → R in E2 and D-323 → Y in E1. This strain infects almost all chicken cells in culture and will form plaques on chicken monolayers.

The importance of the domain of RR E2 near residue 218 for the host range of the virus is also suggested by studies of RR isolated during the epidemic of polyarthritis in the Pacific region described above. Although the virus is maintained in small mammals in Australia, it is believed to have been maintained by direct human-mosquito-human transmission during this epidemic. A single amino acid change occurred in E2 during the epidemic, T-219 → A, and it was suggested that

530 STRAUSS AND STRAUSS MICROBIOL. REV.

MAb 8  
MAb 51 Changed in  
MAb 23 MAb-resistant variants  

$HR$  
$\downarrow$  
RF$\mathbf{G}$PHAYT$\mathbf{S}$YL$\mathbf{E}$ESSGKVYAKPPSGKNITYECKC GDYKTGTVSTRTKITGC  

| | | | | | | | |  
170 180 190 200 210 220  

Domain expressed in $\lambda$ gt 11 expression library  
reactive with MAb 23  

FIG. 26. Domain of glycoprotein E2 containing important neutralization epitopes. The amino acid sequence of residues 170 to 220 of E2 of SIN AR339 is shown. Residues that have been found to vary in different SIN strains are boxed. This domain is very hydrophilic and contains an N-linked carbohydrate chain indicated by the circled CHO. Above the sequence are indicated the locations of changes in MAb-resistant variants (519) and of changes in other SIN strains (italics) (304, 485). Below the line, the arrow indicates the domain that is reactive with MAb 23 when expressed in a $\lambda$gt11 library (583).

this change may have been selected because it adapted the virus to humans (53).

It seems clear that the alphaviruses are a fairly plastic system in terms of host range and potential virulence. Limited changes, primarily in E2 but also in E1, allow the virus to recognize new sets of receptors, which can lead to an altered host range or altered potential to cause disease. The results with RR in chicken cells are particularly dramatic because three changes in the glycoproteins result in a virus that can infect all chicken cells in culture, whereas the parental virus infects only 2% of these cells. Although E1 and E2 form a heterodimer that functions as a unit, the E2 domain from about residue 170 to 220 (SIN numbering) appears to be particularly important in binding to cells. Other studies also indicate the importance of this region for cell binding. The anti-idiotypic antibodies that function as antireceptor antibodies in chicken cells were made to neutralizing anti-E2 antibodies that bind to this domain, as shown by two different methods. First, variants resistant to these antibodies possessed changes at E2 residue 214 (MAb 49), at residue 216 (MAb 23), or at residues 181, 190, and 205 (MAb 50) (520). Second, MAb 23 was shown to bind to $\lambda$gt11 clones expressing fusion proteins containing the E2 region between residues 173 and 220 (583). Thus, there is the presumption that this domain directly interacts with the cellular receptor (at least in chicken cells). This domain is illustrated schematically in Fig. 26. It is hydrophilic, and in SIN a polysaccharide chain is attached at N-196; thus, this domain would be predicted to be exposed on the surface of the particle.

Although the studies of mutants and anti-idiotypic antibodies show that E2 is important for binding to cells, E1 is also involved in binding and penetration. Omar and Koblet (381) prepared SF lacking E2 by treating the virus with trypsin in the presence of detergent. The resulting virus was infectious, showing that E1 alone can induce virus binding and penetration. The detergent treatment led to the loss of 90% of the viral infectivity, however, indicating that the structure of the virion had changed, and the relationship between binding and entry of detergent-treated virus and of untreated virus remains to be determined.

point fuse with a cellular membrane for the nucleocapsid to be deposited in the cytoplasm. The fusion activity is postulated to reside in a highly conserved hydrophobic stretch of 17 amino acids in E1 that starts at residue 80 (138, 419). Consistent with this hypothesis, it has been shown that cells that express alphavirus E1-E2 heterodimers but not E2 alone at their surface can be induced to fuse by exposure to low pH, showing that E1 is required for fusion (264, 265). This E1 putative

A.  
SIN CK V F G G V U P F M W G G A Q C F C DS  
AURA . . . T . . . . L . . . . . . . . . . . .  
WHA . . T . . . . . . . . . . . . . . . . . T  
WEE - R . . . . . . . . . . . . . . . . . . .  
EEE - Q . . T . . . . . . . . Y . . . . T  
VEE . . . T . . . . . . . . Y . . . . T  
ONN . . . T . . . . . . . . Y . . . . A  
SF . . . Y T . . . . . . . . Y . . . . . .  
RR . . . Y T . . . . . . . . Y . . . . . .  
MAY - A . . T . . . . . . . . Y . . . . . .  

B.  
Alphavirus  
glycoprotein E1 VFGGVYPFMWGGAQCFCA D  
Parainfluenza glycoprotein F2 NH₂-FFGGVIGTIALGVATSAQITAAVALV E  
Sendai NH₂-FFGAVIGTIALGVATSAQITAGIALA E  
Influenza glycoprotein HA2 NH₂-GLFGAIAGFI E  

FIG. 27. Putative fusion domain in glycoprotein E1 of alphaviruses. (A) The aligned sequences of E1 between residues 78 and 98 of E1 (SIN numbering). Residues identical to the SIN sequence in the top line are indicated by dashes. The hydrophobic domain between residues 80 and 96 is postulated to form the fusion domain. Data are taken from references cited in Table 1. (B) Comparison of the alphavirus fusion sequence with those of other enveloped viruses. In influenza virus the fusion sequence is found at the amino terminus of the HA2 protein (41), and in the paramyxoviruses it is at the N terminus of the F2 protein (363). PIV-3 is human parainfluenza virus type 3. Although the alphavirus sequence ends with a conserved acidic residue (boxed D) like the paramyxovirus sequences shown (boxed E) and the influenza virus sequence (boxed E), other paramyxovirus fusion sequences end with K or Q.

Virus Entry  

Early events in virus penetration. After binding to a receptor on the cell surface, the alphavirus envelope must at some

fusion domain is illustrated and compared with the fusion domains of a paramyxovirus and an influenza virus in Fig. 27. Disassembly of the alphavirus virion appears to begin at the surface of the cell, presumably induced by binding to a receptor and interactions with cell surface components. Flynn et al. (121) found that previously hidden epitopes in both E2 and E1, referred to as transitional epitopes, became exposed upon binding of SIN to a cell. These conformational alterations could be mimicked, albeit not precisely, by treatment of the virus with heat, reducing agents, or low pH (349). Heat-treated virus can be neutralized by MAbs reactive with transitional epitopes, and selection for resistance to neutralization by these MAbs gave viruses with changes at E2 position 200 or 202 (anti-E2 MAbs) or at E1 position 300 (anti-E1 MAbs). It is intriguing that the E2 domain centered near position 200 has been proposed to be important for receptor binding. Transitional epitopes have also been reported for SF, in which a MAb was isolated that reacted only with acid-treated SF E1 and inhibited virus penetration and fusion (576).

It is known that disulfide bonds play an important role in the stability of the virus (8, 240, 365, 382). Moreover, studies of certain toxins that require reduction of disulfide bonds for entry and toxicity have shown that reduction of disulfide bonds can occur at the cell surface and in endosomes, presumably catalyzed by cellular proteins (118, 438, 620). Abell and Brown (1) have presented evidence that destabilization of the virus structure by reduction or rearrangement of disulfide bonds is important for infection by SIN. These authors found that infection of cells by SIN was inhibited by the presence of the thiol-alkylating agent 5,5′-dithiobis(2-nitrobenzoic acid) during uptake. Because other studies had indicated that this reagent is active only at the cell surface, they concluded that these changes in disulfide bonds in the SIN glycoproteins must occur at the cell surface. These disulfide changes are assumed to be associated with the conformational changes that result in exposure of transitional epitopes, which could be the result of reduction of disulfide bonds or could result from interaction with a receptor and help to induce the reduction or rearrangement of disulfide bonds within the viral glycoproteins (50).

Entry into endosomes. Fan and Sefton (114) first presented evidence that alphaviruses might enter by an endocytic pathway when they showed that the viral glycoproteins do not remain on the cell surface following infection. In contrast, the glycoproteins of Sendai virus, which is known to fuse at the cell surface, did remain on the cell surface after infection. Helenius and colleagues have presented extensive data indicating that the normal pathway of entry for alphaviruses is endocytosis in clathrin-coated vesicles followed by transfer to endosomes, where the low pH leads to a conformational reorganization of the E1-E2 heterodimer such that the fusion domain in E1 is exposed and the virus envelope fuses with the endosomal membrane. This topic has been reviewed several times in the last few years (210, 250, 251, 326, 329, 504, 618).

Helenius and colleagues (101, 195, 197, 199, 327, 328, 330) used radiolabeled virus or electron microscopy to show that the bulk of SF bound to cells was endocytosed quickly and that virus could be seen to fuse to endosomal membranes but not to the cell surface. Anti-clathrin antibodies introduced into the cell inhibited endocytosis of SF. Lysosomotropic weak bases, which raise the pH of endosomes and lysosomes, blocked infection by SF if added before virus addition. However, addition of the bases even 5 min after virus led to detectable infection as assayed by the release of infectious virus 4.5 h after infection (197) or by the incorporation of \[^{3}\]H]uridine into virus nucleic acid from 2.5 to 5 h after infection (199), and addition 1 h after the virus had little effect on the establishment

of virus infection. The inhibition by the bases could be bypassed by briefly exposing cells with bound virus to low pH, which leads to fusion of the virus with the plasma membrane; establishment of infection after acid treatment was demonstrated by the increased incorporation of \[^{3}\]H]uridine into viral RNA at 1.5 to 4.5 h (616), by the finding that at 4.5 h after treatment 63% of acid-treated cells could be stained with fluorescent antibodies to the viral structural proteins versus <3% for the cells treated at pH 7 (197), and by a 10-fold increase in release of infectious virus (197). Thus, these agents block a very early stage of infection. These studies concluded that low pH in the endosome is essential for infection to occur; Helenius et al. (198) also found that the Na\(^+\) concentration in the medium was important for virus entry and proposed that fusion depends not only on acidic pH but also on the maintenance of a voltage potential.

The essential details of these experiments have been confirmed in other laboratories. Garoff and colleagues (440, 576) used a fluorescent-antibody assay to show that cells treated with lysosomotropic weak bases failed to synthesize viral structural proteins following attempts to infect them with SF and that the inhibition of infection could be bypassed by exposing cells with bound virus to low pH. Wengler and Wengler (608) found that treatment with chloroquine, a lysosomotropic weak base, prevented the transfer of SIN capsid protein to the large ribosomal subunit, one of the earliest steps to occur after nucleocapsid penetration into the cytoplasm.

Stegmann et al. (505) used a fluorescence-dequenching assay and a pyrene excimer assay to study the effects of lysosomotropic weak bases on fusion of SF with BHK cells. In these assays, fusion of the virus envelope with a cell membrane results in a dilution of a fluorescent probe incorporated into the viral membrane and this leads in turn to a detectable change in the fluorescence of the probe. Results with a fluorescence-dequenching assay, in which octadecylrhodamine (R18), which has been widely used to study fusion events, was incorporated into SF virions by addition as an ethanolic solution to a virus-containing solution, were ambiguous. The authors concluded that transfer of the added R18 probe did not depend on fusion under some circumstances and that the probe may have been associated with the exterior of the particle rather than incorporated into the lipid bilayer. However, the pyrene excimer assay gave the unambiguous result that fusion did not occur in the presence of 20 mM NH\(_4\)Cl, supporting the Helenius model of entry. In this assay, pyrene-phospholipids are metabolically incorporated into the virus and do not appear to undergo spontaneous exchange.

Support for the importance of low pH in the endosome for fusion of the viral membrane with that of the endosome has also come from the study of mutants that require lower pH to fuse with liposomal membranes (255, 256). Infection by such mutants was inhibited by lower concentrations of lysosomotropic weak bases, consistent with the hypothesis that these mutants require a lower endosomal pH for entry than does wild-type SF. These mutants were also delayed in their penetration of cells, suggesting that they required a longer period to reach endosomes of sufficiently low pH for fusion. Mutations that appear to be similar to these have been described for SIN and mapped to two locations in E1 (position 72, close to the putative fusion peptide in E1 at amino acids 80 to 96, and position 313); these two mutations, either separately or together, led to a requirement for a lower pH to fuse cells infected by the virus and expressing viral glycoproteins at their surface (so-called fusion from within) (36) and altered the neurovirulence of the virus for mice (318).

Mutants with site-specific mutations in SF E1 have been

constructed that led in some cases to a requirement for exposure to lowered pH for fusion of cells expressing the altered proteins at their surface (296). These site-specific SF mutations were constructed in or near the putative fusion domain of E1, and changes at residues 75, 83, and 91 led to a requirement for lowered pH for fusion; the effects of the mutations on the infectivity or entry of the virus have not been tested. Thus, changes in E1 near or within the fusion domain often lead to changes in pH required for fusion, but changes far from this domain (at least in the linear amino acid sequence) can also change the pH required. It is of interest that all of the changes described to date represent changes that appear to stabilize E1 at lowered pH, resulting in a requirement for lower pH for fusion.

Edwards and Brown (108) found that for cultured mosquito cells, the pH of fusion from without was determined by the medium in which the cells were grown. The specific infectivity of SIN in mosquito cells that required a lower pH for fusion was only one-third that in mosquito cells that were fused at higher pH, consistent with the hypothesis that in the cells requiring lower pH for fusion, virus would not reach endosomes of sufficiently low pH for fusion as efficiently.

Endosomes can be physically separated into two subpopulations that differ in their protein composition and that are kinetically related to one another such that an incoming ligand first encounters an early endosome and is then transferred to a late endosome (472). The early endosomes recycle receptors, whereas the late endosomes deliver the ligand to lysosomes. Schmid et al. (471) used wild-type SF and the mutant SF requiring a pH of <5.3 for fusion and for the accompanying changes in glycoprotein conformation described below in order to measure the kinetics of delivery of SF to these different endosome populations and to ascertain the pH encountered by the virus in these endosomes. These experiments were carried out with both wild-type CHO cells and ts mutant CHO cells defective in acidification of endosomes at a restrictive temperature. SF was found to first reside briefly in vesicles of neutral pH, but with a half time of 5 min it encountered a pH of <6.2 in early endosomes. A fraction of virus reached a pH of <5.3 in early endosomes, but most virus did not reach this pH until transfer to late endosomes, which occurred with a half time of 8 to 10 min from the time entry was initiated by warming the cells. The authors concluded that acidification of both endosomal populations was heterogeneous but that SF always passed through a less acidic early endosome before encountering a more acidic late endosome. CHO cells temperature sensitive in acidification of endosomes were found to be defective in acidification of both endosomal populations at the restrictive temperature.

Fusion of alphaviruses with artificial membranes (457, 615) or with the plasma membrane of cells (440, 576, 616), as well as fusion of cells expressing alphavirus glycoproteins on their surface to form heterokaryons, whether vertebrate cells (36, 242, 264, 323, 617) or mosquito cells (108, 262), can be accomplished by treatment with low pH. As described above, the pH required for these fusion reactions is correlated with the sensitivity of the virus to lysosomotropic agents when studied, and it is assumed that these assays measure the same phenomena that are involved in virus fusion with cellular membranes during entry. Although it seems certain that these phenomena are related, there are obvious differences between heterokaryon formation and initiation of a normal infection. For one, we know that virus bound to receptors at the cell surface undergoes conformational changes. Furthermore, cell-cell fusion induced by exposure of infected cells to low pH is a multiphasic phenomenon. Very soon after exposure to low pH,

infected mosquito cells demonstrate the presence of ion channels that have a pore size of about 1 nm, which are believed to be formed by the viral glycoproteins (277). During cell-cell fusion from without, similar-size pores form between two cells within about 1 min and then slowly expand in size (275, 276). Kempf et al. (243) found that the initial fusion of cells occurred very quickly and did not require energy, whereas the expansion of the initial pores to form heterokaryons required energy and considerable time. In the case of infected BHK cells, ATP was rapidly depleted upon exposure to low pH, which prevented heterokaryon formation until the cells were returned to neutrality and the ATP pools were reestablished (244). Abell and Brown (1) found that return to neutral pH was not required for heterokaryon formation if mercaptoethanol was present in the medium, suggesting that the energy requirement is related to shuffling of disulfide bonds. In addition, it has been shown that SF-induced cell-cell fusion will occur at neutral extracellular pH if the cytoplasmic pH is lowered (245).

The data supporting the model that alphavirus entry occurs by endocytosis followed by fusion with an endosomal membrane triggered by low pH are extensive, internally consistent, and convincing. Nevertheless, Brown and colleagues have proposed that alphaviruses infect cells by direct fusion to the cell surface, or at least that the low pH of the endosome is irrelevant to virus infection, and that the major effect of lysosomotropic weak bases is upon viral replication rather than upon endosomal pH (references 63, 78, and 109; reviewed in reference 50). They argue that their finding that infection of BHK cells in the presence of chloroquine or ammonium chloride allowed the establishment of superinfection exclusion (described in more detail below) when one virus was first added and then followed by the second must mean that infection had been established in the presence of the bases, because the establishment of exclusion requires the establishment of the infection process. They have reported that they have preliminary data suggesting that SIN nonstructural proteins are translated in the presence of lysosomotropic weak bases (48). They also found that a mutant CHO cell that is resistant to diphtheria toxin because of defects in the acidification of endosomes was completely sensitive to SIN and suggested that this indicated that low endosomal pH is not required for virus entry (109). However, Robbins et al. (425) found that two other such cell lines were in fact partially resistant to SIN and that the resistance could be overcome by exposing the cells to acidic pH, consistent with the model for endosome-mediated entry and pH-dependent fusion.

Conformational changes in E1 and E2. Several studies have shown that E1 and E2 undergo irreversible conformational changes when exposed to the fusion pH, and these changes are proposed to activate the fusion activity. In SF the change in E1 is accompanied by acquisition of resistance to digestion by trypsin (254) and changes in the reactivity of E1 with certain MAbs (252, 574, 576). These properties have been used to monitor the exposure of E1 to low pH during transit through the endosomal pathway (198, 256, 440, 471). The acid-induced change in SF E1 also increases the hydrophobicity of the virus (381) and exposes disulfide bonds that were otherwise not accessible (382). In the case of SIN, exposure of the virus to low pH was shown to cause an irreversible alteration in the density and sedimentation characteristics accompanied by conformational changes in the envelope proteins and inactivation of the virus (110).

Wahlberg et al. (575, 576) found that during infection by SF, the E2-E1 heterodimer dissociates on exposure to the acidic pH of the endosomes. E2 is then present as a monomer. E1

forms a trypsin-resistant homotrimer that may be derived from the trimeric spike, or that may form by interactions similar to the E1-E1 interactions that have been proposed to hold the spike together (7).

The kinetics of changes in E1 and E2 have been studied during acid-induced fusion of the virus with either liposomes (47) or with BHK plasma membranes (236). In the former system, binding to liposomes occurs only after exposure to low pH, whereas in the latter system, the virus is prebound to the membranes through receptor-mediated binding. The kinetics were found to be sensitive to pH and to temperature, and by working at a pH above the optimal fusion pH and at temperatures below 37°C, the different steps in the fusion process could be separated. Dissociation of the E1-E2 heterodimer occurred early, as did the acquisition of trypsin resistance and exposure of conformation-specific epitopes in E1 and the formation of E1 homotrimers. Justman et al. (236) reported that in their system the dissociation of E1-E2 heterodimers occurred first, followed by the conformational changes in E1 and then by trimerization of E1. Bron et al. (47) found that binding to liposomes occurred only after trimerization of E1 had occurred. Fusion occurs only after homotrimerization of E1 (and binding to liposomes in the liposome model system). Because there was a distinct lag between formation of homotrimers and fusion, it was suggested that multimers of E1 homotrimers are involved in fusion.

The changes in E1 are proposed to expose the fusion domain and allow fusion to occur by an unknown mechanism. In the case of influenza virus, the HA2 subunit, which contains the fusion domain at its N terminus, has been shown to insert into the target membrane prior to fusion (503). The presumption is that the fusion domain of E1, which is very hydrophobic, inserts into the target membrane and induces fusion of the two bilayers in some way.

Cholesterol requirement for entry by SF. In the case of SF, fusion to liposomes triggered by low pH has been found to require cholesterol (47, 615); other sterols that contain a 3-β-hydroxyl group can substitute for cholesterol, but sterols lacking this group cannot do so (253). The pH-dependent conformational changes observed in E1 also require cholesterol (252). Furthermore, cholesterol-depleted cells are less susceptible to infection by the virus, and the resistance to infection is reversed by addition of cholesterol (404). In addition to the requirement for cholesterol during entry, cholesterol is required for the assembly of SF; in cholesterol-depleted mosquito cells, production and transport of viral glycoproteins proceeded normally but production of progeny virions was severely depressed (324). It appears that the stability or conformation of SF glycoproteins is affected by binding to cholesterol. This dependence is curious because in nature alphaviruses replicate in mosquitoes that are unable to synthesize cholesterol, although a blood meal will contain it. Cholesterol-independent mutants of SF have now been isolated (324), which suggests that the requirement for it may come and go in nature depending on the host in which the virus replicates or that it may be a laboratory phenomenon that arose during passage of the virus in cells containing large amounts of cholesterol. Other alphaviruses have not been tested directly for a similar requirement, but it is known that mosquitoes that have never been exposed to cholesterol can be infected with SIN per os or by intrathoracic inoculation, and a complete infection cycle occurs with transfer of the virus to the brain and to the salivary glands (50).

---

Early Events in Establishment of Infection

Nucleocapsid disassembly. The mechanism by which the nucleocapsid disassembles when it reaches the cell cytoplasm has been a topic of interest (reviewed in reference 490). How is it that a nucleocapsid is unstable when it first enters a previously uninfected cell but newly assembled nucleocapsids are stable later in the infection process?

Newly synthesized SF capsid protein was found many years ago to bind to the large ribosomal subunit when synthesized in vitro (151, 609) or in vivo (557, 558). Wengler and Wengler (608) reported that capsid protein from infecting SIN virus was also associated with ribosomes and suggested that this binding to ribosomes resulted in nucleocapsid disassembly. In this model the ribosome competes with the viral RNA for capsid protein, and upon infection of the cell the large excess of unoccupied ribosome-binding sites leads to disassembly of the nucleocapsid. Wengler et al. further hypothesized that during virus replication, saturation of ribosomes with newly synthesized capsid protein leads to a switch from disassembly to assembly of nucleocapsids (605, 608, 609). Wengler et al. (612) and Singh and Helenius (491) have confirmed several aspects of this model. Both groups found that disassembly took place in cell-free lysates in a reaction requiring ribosomes. The capsid protein was transferred to the large ribosomal subunit in this in vitro reaction, and each ribosome could bind three to six capsid protein molecules. Wengler et al. (612) showed that pretreatment with capsid protein to saturate the binding sites resulted in a failure of the treated ribosomes to uncoat nucleocapsids, and they mapped the ribosome-binding site to residues 94 to 105 of the SIN capsid protein by examination of the binding of fragments of the capsid protein to ribosomes. However, Singh and Helenius (491) found that ribosomes in infected cells were not saturated with capsid protein and were still capable of disassembling capsids in an in vitro reaction, consistent with a previous report that well into infection only 20% of ribosomes were saturated with capsid protein (558). Thus, the shift from disassembly of capsids to assembly of capsids during infection does not appear to be due simply to saturation of ribosomes with capsid protein, and although ribosomes may uncoat the nucleocapsid there must be other factors at work that lead to the shift from instability to stability of the nucleocapsids.

As described above, one possibility arises from the observation that newly synthesized alphavirus nucleocapsids present in the infected cell appear to differ in size from mature capsids found in virions; this maturation event could trigger the capsid for disassembly in the cytoplasm. A second possibility is that the changes in Na⁺ and K⁺ concentrations inside the cell that occur about 3 h after infection (144) stabilize the capsids. A third possibility was proposed by Lanzrein et al. (277), who suggested that exposure of the virus to low pH in the endosome led to the formation of ion channels in the virus that allowed the entry of protons into the interior of the virus; they proposed that this facilitated the uncoating process.

Superinfection exclusion. Upon infecting a cell, many viruses establish a state such that the infected cell is resistant to superinfection by the same or closely related viruses. This phenomenon, called superinfection exclusion or homologous interference, can arise by a number of different mechanisms. Early studies on SIN by Johnston et al. (234) showed that superinfection exclusion could be established within 15 min after infection at moderate multiplicity (10 to 25 PFU/cell), could be established within 1 h after infection at very low multiplicity, and required the functioning of the incoming genome, as shown by the failure of UV-irradiated virus to

establish exclusion or by the failure of a *ts* mutant, *ts24*, to establish exclusion at the nonpermissive temperature. Establishment of exclusion was less sensitive to UV irradiation by a factor of 5 than was the ability to form a plaque, suggesting that expression of only 20% of the genome is required to establish exclusion. These authors also found that another RNA-negative mutant, *ts6*, could establish superinfection exclusion at the nonpermissive temperature under conditions where no detectable RNA was being synthesized, suggesting that a limited number of RNA genomes was sufficient to establish exclusion. It appears that exclusion requires translation and function of nonstructural proteins but can be established very early, before extensive replication of the RNA or production of the structural proteins.

Atkins (12) confirmed that superinfection by Sindbis virus can be established very early and that it can be established by RNA-negative mutants. Adams and Brown (3) also confirmed that RNA-negative mutants were capable of establishing superinfection exclusion and that the phenomenon occurred within 30 min of infection. These authors used *ts* mutants to establish exclusion at the nonpermissive temperature and superinfected the cells 30 min later with wild-type virus. They found that although the superinfecting wild-type virus did not replicate, it was able to complement the *ts* mutant, leading to increased production of *ts* virus. The simplest interpretation of these results is that under these conditions the superinfecting genome is uncoated and translated but the RNA does not replicate.

One model to explain this early superinfection exclusion is that a host protein or other cell component required for replication of the virus is present in limiting amounts and that once the excluding virus begins to replicate, it titrates out this factor. A second model is that the buildup of nonstructural proteinase limits production of minus-strand RNA by the superinfecting virus; as described above, P123 is required for minus-strand synthesis and accumulation of the viral nonstructural proteinase eventually prevents synthesis of minus-strand RNA. The conceptual problem with either of these models is that the first virus continues to synthesize minus-strand RNA for 3 h after infection, presumably forming new replication complexes in the process, and yet the second virus appears to be prevented from forming replication complexes after 0.5 to 1 h. It may be simply a kinetic phenomenon in which the second virus does replicate but is unable to overcome the advantage resulting from the exponential rise in minus-strand RNA synthesized by the first virus during the time that exclusion is being established, so that most of the yield is that of the first virus. The number of minus strands present in cells infected by SIN or SF has been found to be 4,000 to 7,000 (553a, 584). Starting from a multiplicity of 1, a doubling time of about 15 min would be required to produce this number of minus strands in 3 h. Thus, the advantage of a 1-h head start would be about 16-fold if replication were uniform. Exclusion has been found to lead to a more than 16-fold advantage in most cases, but nonuniform replication in which minus-strand RNAs are formed faster early in infection (which is quite reasonable) might lead to the observed exclusion without invoking anything other than the head start for the first virus and shutoff of minus-strand synthesis as the concentration of viral proteinase rises. It is also possible that there are factories established by the first virus into which cellular factors are recruited, making them unavailable to the superinfecting virus for establishment of a new factory.

Inhibition of host protein synthesis. Upon infection of vertebrate cells by alphaviruses, host macromolecular synthesis is inhibited (reviewed in references 238, 488a, 521, and 604).

The viral inhibition of host protein synthesis has been the most extensively studied. Inhibition sets in abruptly at about 3 h after infection, and after this time most protein synthesis is virus specific. Four mechanisms for inhibition of host protein synthesis by alphaviruses have been proposed which are not mutually exclusive and which could be synergistic in effect. These are (i) competition for translational machinery by viral mRNA, (ii) an altered intracellular environment that favors viral mRNA translation, (iii) direct inhibition by the viral capsid protein, and (iv) interference with initiation factors caused in some way by the nonstructural proteins.

Early studies led to the suggestion that inhibition of host protein synthesis resulted from competition with viral mRNAs for the translational machinery (607). This hypothesis received support from studies by Atkins (11), who used *ts* mutants to show that the inhibition of host protein synthesis was proportional to viral RNA synthesis. Because a number of the mutants used did not process the structural polyprotein at the nonpermissive temperature but nonetheless inhibited host protein synthesis but not host DNA synthesis, the conclusion was reached that viral structural proteins were not required for inhibition of host protein synthesis but were required for inhibition of host DNA synthesis. It seems unlikely that simple competition is the primary mechanism of inhibition, however, because total protein synthesis declines markedly in infected cells (528), and thus the decline in host protein synthesis does not seem to arise from a limitation in the capacity of the system.

A second model for inhibition of host protein synthesis was proposed by Carrasco (60) and by Garry et al. (144), who suggested that changes in the ionic environment within the infected cell favored translation of viral mRNA. Garry et al. (144) examined the concentrations of monovalent ions in cells infected with SIN and found that the intracellular K⁺ concentration decreased from 150 to about 50 mM and the Na⁺ concentration increased concomitantly from 20 to 50 mM at the same time after infection of chicken cells by SIN that host protein synthesis was inhibited. They proposed that the change in cation concentration resulted in inhibition of the initiation of host cell mRNA translation, consistent with known ionic requirements for initiation of translation (375, 439), and that viral RNAs were resistant to these altered ionic conditions. Garry et al. (146) found that treatment of cells with ouabain, which interferes with the Na⁺/K⁺ pump, also raised the intracellular Na⁺ concentration and reduced the K⁺ concentration and that this led to inhibition of host protein synthesis but not viral protein synthesis. They hypothesized that a viral component interferes with the Na⁺/K⁺ pump, leading to the observed alteration in cation concentration in the infected cells and resulting in inhibition of host protein synthesis. It is of interest that Garry and Bostick reported that heat shock also led to inhibition of the Na⁺/K⁺ pump (145).

Ulug et al. (561) measured the uptake of ⁸⁶Rb⁺, a tracer for K⁺, and found that the Na⁺/K⁺ pump was, in fact, inhibited 40 to 50% as a consequence of SIN infection but that the furosemide-sensitive Na⁺/K⁺/Cl⁻ cotransport system was not affected by virus infection. Binding studies revealed that there was no difference in the number of pump sites and no change in the *K*ₘ of the pump for K⁺ but that the *V*ₘₐₓ of the pump was lowered. In a follow-up paper, Ulug and Bose (559) reported that treatment of infected cells with tunicamycin, an inhibitor of glycosylation, prevented the inhibition of the Na⁺/K⁺ pump that would otherwise occur and also prevented the development of the obvious cytopathic effects that normally occur at this stage of the infection cycle. In the presence of tunicamycin the transport of viral proteins to the plasma

membrane was reduced and the transported proteins were not glycosylated. They concluded that insertion of glycosylated viral proteins into the plasma membrane was probably required for inhibition of the pump. Studies with *ts* mutants of SIN in which the glycoproteins are transported to the cell surface but no budding occurs showed that the pump was inhibited under these conditions and that virus maturation is therefore not required for inhibition (560).

Ulug and Bose (559) found that although tunicamycin prevented the virus-induced inhibition of the Na<sup>+</sup>/K<sup>+</sup> pump, it did not prevent the virus-induced inhibition of host protein synthesis. Inhibition of host protein synthesis was not as efficient in tunicamycin-treated cells, however, suggesting that the altered ionic conditions induced by the virus are partly responsible for inhibition of host protein synthesis but that other mechanisms must also be involved. Further support for this modified hypothesis came from Garry (143), who found that addition of K<sup>+</sup> to the medium over SIN-infected cells resulted in a rise in the intercellular K<sup>+</sup> concentration; this resulted in an increase in the overall rate of protein synthesis and a partial reversal of the virus-induced termination of host protein synthesis.

Gray et al. (163–165) confirmed that there are alterations in the concentration of monovalent cations in cells infected by SF. From studies with drugs, including the monovalent cation ionophore nigericin, they concluded that these changes could not account for the extent of inhibition of protein synthesis seen in infected cells, although they could contribute to inhibition, consistent with the results for SIN. Carrasco et al. (61) have reviewed the evidence that permeability changes occur following infection by many viruses, including alphaviruses, and that these changes may be responsible at least in part for inhibition of protein synthesis by altering the ionic conditions in the cell.

A third mechanism for inhibition of host protein synthesis was proposed by Van Steeg et al. (563), who found that SF capsid protein inhibited translation of cellular mRNA or of SF genomic RNA, but not of subgenomic 26S mRNA, when tested in a cell-free translation system. Inhibition appeared to be at the level of formation of 80S initiation complexes. They also confirmed that capsid protein was bound to ribosomes isolated from infected cells and could be removed in high-salt washes. They hypothesized that the inhibition of host protein synthesis following viral infection was the direct result of capsid protein interfering with initiation of translation of host mRNAs. In more extensive studies, Elgizoli et al. (112) reported that protein synthesis was markedly stimulated upon introduction of low concentrations of SF capsid protein (10<sup>3</sup> to 10<sup>4</sup> molecules per cell) into several mammalian cell lines, whereas protein synthesis was significantly inhibited (up to 60%) upon introduction of large amounts of capsid protein (10<sup>5</sup> to 10<sup>6</sup> molecules per cell). It was not determined whether the inhibition resulted from a direct interference with translation or whether it resulted from an indirect action, such as interference with transcription. The introduced capsid protein was found to be transported efficiently to the nucleus and to concentrate in the nucleolus (353), in agreement with earlier studies in which capsid protein was expressed from recombinant simian virus 40 (225). The inhibition of protein synthesis caused by capsid protein was transitory (353), and it seems unlikely that inhibition of host protein synthesis by the virus capsid protein is significant in infected cells. Schlesinger et al. (462), in fact, reported that a mutant of SIN in which capsid protein was synthesized in the same amounts as in wild-type-infected cells did not inhibit host protein synthesis and concluded that capsid protein was not responsible for the observed inhibition of protein synthesis in infected cells.

The fourth model for inhibition of host protein synthesis is interference with initiation factors required for protein synthesis, as has been found for other virus systems (111). In recent studies, Frolov and Schlesinger (126) found that SIN replicons that were unable to synthesize structural proteins or that synthesized only small amounts of subgenomic mRNA were able to inhibit host protein synthesis with the same kinetics as SIN virus, and they concluded that early steps in viral gene expression resulted in inhibition. In agreement with the results of Ulug and Bose, they found that replicons that did not express the glycoproteins did not induce cytopathologic changes. The initiation of alphavirus RNA translation differs from that of host mRNAs, and the initiation factors used could differ. van Steeg et al. (564) found that cell-free translation systems isolated from infected cells translated host mRNA or SF 49S RNA poorly while translating 26S RNA efficiently and that addition of cap-binding protein or of eIF-4B overcame the inhibition. 26S RNA requires lower concentrations of these initiation factors and binds these factors more tightly (26). It is also worth noting that the structure of the 5′ NTR is thought to control the efficiency with which an mRNA interacts with initiation factors, but alphavirus RNAs contain hypermethylated caps (m<sub>2</sub><sup>2,7</sup>G and m<sub>3</sub><sup>2,2,7</sup>G) as well as m<sup>7</sup>G caps (216), which could affect initiation of translation. It is known that SIN or SF 26S RNA requires a cap to be translated, as tested in cell-free systems, but translation of the RNA is insensitive to the presence of cap analogs that inhibit translation of host mRNAs (26, 539, 564). Tahara et al. (539) found that m<sup>2,7</sup>G caps were bound more tightly than m<sup>7</sup>G caps and supported translation of model mRNAs more efficiently in cell-free systems, whereas m<sup>2,2,7</sup>G caps are bound poorly and mRNAs with these caps are translated poorly. Nevertheless, SIN 26S RNA containing the authentic 5′ NTR was translated equally well whether it carried an m<sup>7</sup>G cap or an m<sup>2,7</sup>G cap but was translated poorly if it carried an m<sup>2,2,7</sup>G cap. van Duijn et al. (562) found that in SF-infected cells, polysomal 26S mRNA disproportionately possessed monomethylated caps and few had trimethylated caps.

In summary, the subject of inhibition of host protein synthesis has proven to be complex. It seems most likely that early viral gene functions are involved by mechanisms that have yet to be determined but that inhibition of the host is augmented by the alteration in intracellular [K<sup>+</sup>] induced by the virus. Inhibition could result in a lowered capacity of the system, with viral mRNAs being preferentially translated because they bind initiation factors more tightly. In addition to the alteration in intracellular ionic conditions, the onset of inhibition of host protein synthesis is accompanied by increased phospholipase activity that results in the release of choline and of arachidonic acid into the medium (401) and by the depolarization of the cell (24), and it is conceivable that these changes might (also?) contribute to inhibition of host protein synthesis.

Formation of replication factories. After infection of vertebrate cells, cytopathic vacuoles (CPVs) that serve as sites of RNA replication and perhaps as sites of RNA translation and nucleocapsid assembly are formed. CPVs are 0.6 to 2 μm in diameter and have 50-nm membrane spherules lining the vacuole membrane at regular intervals. They were long ago shown to be associated with RNA replication (2, 124, 171, 172). They form early (detectable after 1 h) following infection at a multiplicity of 200 but are not detectable until 5 h postinfection following infection at a multiplicity of 20. A recent immunoelectron-microscopic study by Froshauer et al. (127) has shown

that the CPVs are derived from endosomes and lysosomes and that the viral nonstructural proteins, presumably replicases active in RNA replication, are associated with the cytoplasmic face of the CPVs. Moreover, fibrous ribonucleoprotein structures extend from the base of the spherules on the cytoplasmic face of the CPVs and form connections with the rough ER. The association with the rough ER suggested that the CPVs are replication factories that constitute sites for RNA replication as well as containing sites for translation of structural proteins and assembly of nucleocapsids.

### Pathogenesis and Immunity

#### Neutralization epitopes in E2.
The epitopes on the surface of a virus that interact with antibodies, especially neutralizing antibodies, have attracted attention over the years because of their obvious importance in protection from viral disease. Studies with neutralizing MAbs have been reported for several alphaviruses including SIN, VEE, RR, SF, EEE, and WEE (reviewed in reference 426).

In general, antibodies directed against glycoprotein E2 are more often neutralizing than are antibodies reactive with E1. In the case of SIN, at least two apparently nonoverlapping domains in glycoprotein E2 have been found to contain epitopes that lead to neutralization by one or more MAbs. One of the E2 domains consists of a linear sequence in the ectodomain of the protein between amino acids 170 and 220, which is illustrated in Fig. 26. This domain is highly charged, is variable in sequence among strains of SIN, contains a bound oligosaccharide chain, and is almost certainly exposed on the surface of the virion. The importance of this region for virus neutralization has been shown in several ways. Strauss et al. (518, 520) found that variants resistant to neutralization by six different MAbs reactive with E2 (502) had amino acid substitutions within this domain (Fig. 26). Competition-binding studies showed that all six MAbs interfered with one another for binding to the virus; i.e., they reacted with overlapping epitopes, but the resistant variants could be divided into two groups by their pattern of cross-resistance to the six MAbs (502, 520). MAbs 49 and 50 define antigenic site A, and MAbs 23, 18, 30, and 51 define site B. It is clear from the location of amino acid changes in resistant variants (Fig. 26), as well as from the cross-competition experiments, that these sites are overlapping. Davis et al. (88) also mapped other variants resistant to MAb 23 and to two additional MAbs to substitutions at residue 216. Intriguingly, the last two MAbs, R8 and R10, require E-216, a negatively charged residue, for full reactivity, whereas MAbs 23, 18, and 51 require K-216, a positively charged residue. Almost all of the amino acid changes in the MAb-resistant variants resulted in a change in charge. Two exceptions to this general rule were resistant variants which had acquired a new site for N-linked glycosylation, K-216 → N (520) and T-213 → N (88). It is assumed that the presence of the new carbohydrate chain prevents binding of the antibody.

Residues that are changed in resistant variants have been shown in at least some cases to form part of the epitope bound by the MAb (278), and it is assumed that this is usually the case, although cases have been described in which single amino acid substitutions away from the antibody-binding domain result in a change in conformation such that the MAb no longer binds (94, 393). Most neutralizing antibodies recognize conformational or discontinuous epitopes, and, without performing structural analysis of the complex, it is difficult to ascertain where in a protein an antibody binds. However, SIN anti-E2 MAb 23 will bind to λgt11 phage that express residues

173 to 220 of E2 (583), and it is clear that for this antibody, at least, the domain illustrated in Fig. 26 contains all or substantially all of the antigenic epitope. It seems likely from the changes seen in the antibody-resistant variants that all of the E2 epitopes in sites A and B are contained largely within this domain.

A second neutralizing domain in SIN E2, called antigenic site C, is defined by reactivity with MAbs R6 and R13 and has been studied by Pence et al. (396). MAbs R6 and R13 were found to bind to but not to neutralize wild-type virus containing S-114 in E2 (379) but to neutralize an attenuated strain of SIN that contains R-114 (87, 378, 379). Variants resistant to MAbs R6 and R13 were found to have changes in E2 at position 62, 96, or 159. This epitope thus appears to be discontinuous and dispersed over E2.

Studies with VEE identified the E2 domain between residues 182 and 207 as important in virus neutralization. MAbs directed against VEE E2 have been grouped into eight antigenic sites, which have been shown by competitive-binding assays to fall into four overlapping domains (427, 431). Six of these sites led to neutralization of virus, but the antibodies differed in their ability to neutralize different strains of VEE. Five of these sites cluster into what the authors refer to as a critical neutralization domain of E2, and variants resistant to MAbs defining four of these have been isolated and sequenced. Changes found were S-182 → R, G-183 → V, E-199 → K, and I-207 → F (227). Thus, this critical neutralization domain appears to be essentially the same as the SIN E2 domain illustrated in Fig. 26.

Three neutralization epitopes in RR E2 that overlap in competitive-binding assays have been studied by isolation of antibody escape variants (573). One neutralizing antibody selected variants changed at T-216 and thus this epitope appears to be related to those in the major SIN and VEE domain described above. Two other MAbs selected changes at residues K-234 or H-232 or at D-246, T-248, or R-251.

Anti-E2 MAbs have also been described for SF and grouped into five antigenic determinants by competitive-binding assays (32, 35). Two determinants were associated with neutralization, but no information on antibody escape variants has been reported.

Neutralization epitopes in E1. E1-reactive MAbs have been studied for several alphaviruses including SIN (67, 428, 470, 500), VEE (427), SF (35), and WEE (219); for SIN, VEE, and SF some of the MAbs were neutralizing, but in only one case have resistant variants been obtained and sequenced. MAb 33 neutralizes the infectivity of SIN and is directed against E1. Two independent resistant variants were found to have substitutions at G-132 of E1, to R in one case and to E in the second (520). It is of considerable interest that this E1-specific MAb and the E2-specific MAbs belonging to sites A and B described above competed with one another in binding studies with intact virus, suggesting that the E1 domain reactive with MAb 33 and the E2 domain in Fig. 26 are spatially close in the virion. A related finding has been reported for VEE, in that a neutralizing anti-E1 MAb was found to compete with neutralizing anti-E2 MAbs reactive with the critical neutralization domain defined by Roehrig and collaborators (427). As described above, the domain of the E1-E2 heterodimer defined by these neutralizing antibodies may form, in whole or in part, the virus receptor-binding domain.

Conformational dependence of epitopes. Most epitopes that bind MAbs depend to a greater or lesser extent on the native conformation of the glycoprotein (or of the virus) for reactivity. In general it has been found that antibodies reactive with E1 are very sensitive to conformation, whereas those reactive

with E2 are less so and many of the MAbs studied will react with denatured E2 or with peptide fragments derived from E2 (219, 428, 431, 470). Some peptides derived from E2 of VEE (218), RR (248), or SF (173) reacted with polyclonal antisera produced against the respective virus; in the case of VEE and SF certain antipeptide antisera bound to the intact virus but did not neutralize virus infectivity, and immunization of mice with certain of these peptides protected against viral disease.

Cryptic epitopes that are nonreactive with antibody in the native protein but which become exposed after conformational changes in the proteins have been described for both E1 and E2 of alphaviruses. In an early study of SIN, certain anti-E1 MAbs were found that were nonneutralizing and that did not bind to the intact virus but became reactive after detergent treatment or treatment at low pH (470). These MAbs were capable of mediating complement-dependent lysis of infected cells, showing that these cryptic epitopes were expressed on the cell surface, possibly in immature forms of the glycoproteins. Cryptic epitopes called transitional epitopes that become exposed upon binding of SIN to a receptor on the surface of a cell or after exposure to heat, thiol-reducing agents, or low pH were described above. Cryptic epitopes have also been found in WEE E1 (219).

Stanley et al. (500) found that two SIN viruses that differed at positions 55 and 209 of E2 and at positions 72 and 313 of E1 (318) were differently neutralized by 3 of 11 MAbs directed against E1 and 8 of 12 MAbs directed against E2. Resistant variants have not been selected and mapped, and the relationship of these residues to the epitopes defined by resistant variants is unclear, but because differences in neutralization by so many different antibodies were observed it was suggested that the E1 and E2 changes led to conformational changes in the glycoprotein spikes.

Protection from disease. The most important mechanisms by which the immune system protects an animal from viral disease are usually considered to be neutralization of virus infectivity by humoral neutralizing antibodies, which prevents the spread of the virus, and lysis of infected cells by cytotoxic T cells, which prevents further production of the virus. The alphavirus epitopes defined by the various catalogs of MAbs appear to define the major neutralizing epitopes seen by the immune system. In the case of SIN, epitopes E2 A, E2 B, and the E1 epitope defined by the anti-E1 MAb 33 figure prominently in the antibody response of an infected mouse. Schmaljohn et al. (468, 470) found that serum from a mouse infected with wild-type virus neutralized wild-type virus 10-fold better than it neutralized a variant multiply resistant to MAbs 33, 50, and 23. Conversely, serum from a mouse infected with the triple variant neutralized the variant 10-fold more efficiently than it neutralized the wild type. This suggests that 90% of the neutralizing immune response in a mouse is directed against these three antigenic sites. Using a similar approach, Kerr and Weir (quoted in reference 573) found that a triply resistant variant of RR E2 was neutralized one-fourth as well as the wild type by an anti-RR polyclonal antiserum.

The mechanisms of neutralization vary. In some cases binding of the antibody has been shown to interfere with virus binding to a potential host cell, whereas in other cases virus with attached antibody could attach to cells but could not penetrate (430). In the case of the anti-SIN E2 A and E2 B sites, it has been suggested that the antibodies bind to the receptor-binding domain of the virus and block attachment to cells (582), and variants selected to be resistant to these antibodies have usually been found to lose the ability to bind the antibody (502). However, Flynn et al. (122) reported that MAb 49, which reacts with site A, would bind to many strains

of SIN and activate the infectivity of the virus for BHK cells. Antibodies that bind to the E2 C site bind to a number of SIN strains but neutralize only some of them (379).

Although neutralizing antibodies are clearly of major importance in protection from viral infection, studies of alphaviruses have repeatedly shown that nonneutralizing antibodies can be protective (35, 219, 336, 347, 431, 469, 470, 501, 621, 622). The mechanism(s) of protection is not known, but possible mechanisms include binding of antibody to epitopes expressed only in immature forms of the glycoprotein leading to inhibition of virus maturation, complement-mediated lysis of infected cells (34, 469), and curing of viral infection by a nonlytic mechanism (33, 290).

Stanley et al. (501) reported protection studies in which a distinction was made between prophylaxis (antibody passively administered before infection) and cure (antibody administered after establishment of an acute encephalitis that will result in death of the animal in the absence of an immune response). Most MAbs were protective when administered either prophylactically or after infection was established, but two MAbs were found that could cure an established infection but were not effective when administered prophylactically, and two MAbs were found that were effective prophylactically but not when administered after infection. These authors suggested that the mechanisms of prophylaxis and recovery might differ.

Rather than protecting animals from disease, the immune response sometimes leads to exacerbation of disease. Mokhtarian et al. (358, 359) found that alphaviruses can induce a paralytic disease resembling experimental autoimmune encephalomyelitis in certain strains of mice.

The complement system has also been shown to be important in host defense against alphaviruses. SIN has been shown to activate both the alternative and classical complement pathways, and depletion of complement results in prolonged viremia and a more severe central nervous system infection (203–205, 208). The relative ability of the virus to activate complement and to be cleared by an animal is inversely related to its sialic acid content (206). Thus, the natural immunity to SIN is higher in strains of mice that have less sialic acid in their tissues (207).

Virus neurovirulence. Many of the New World alphaviruses are capable of causing a serious encephalitis in humans or other animals; the Old World viruses are less neurovirulent and, in humans, lead to disease characterized by headache, fever, rash, and arthralgia (166, 402, 488). Old World viruses are often neurotropic in mice, however, and three of these, SIN, RR, and SF, have been used as models to investigate the molecular basis of neurovirulence.

The severity of encephalitis caused by alphaviruses is often age dependent. In humans, WEE and EEE lead to a more serious encephalitis in infants and young children than in adults (reviewed in reference 414). Similarly, SIN causes a more acute encephalitis in young mice than in adults. After infection with most strains of SIN, neonatal mice die of encephalitis whether the virus is administered peripherally or intracranially, whereas older mice survive and clear the infection. Two different approaches have been used to study SIN neurovirulence; in one approach, variants virulent for adult mice have been characterized, and in the other, variants attenuated for newborn mice have been examined (reviewed in references 167 and 168). Both virulent and attenuated strains replicate in neurons of the brain and spinal cord, and the pattern of virus replication is similar in fatal and nonfatal infections (222, 223, 483). However, neurovirulent strains of virus replicate to higher titers in brains of neonates or in brains

of adult mice and cause fatal infection; they are designated neurovirulent for neonates or adults, respectively. Furthermore, it appears that neurovirulent strains induce apoptosis and subsequent neuronal death; if apoptosis is not induced, the animal survives (556).

A neuroadapted strain of SIN referred to as NSV causes a fatal encephalitis in adult mice as well as in neonatal mice after ic inoculation. NSV was isolated after alternating passage in adult and suckling mice (169). It was shown by Lustig et al. (318) that alterations in the glycoproteins of the virus were largely responsible for the increased neurovirulence, although change(s) outside this region also contributed; changes in both E1 and E2 were required. Key changes associated with the increased virulence identified in this and in follow-up studies were Q-55 → H and R-172 → G in E2 and V-72 → A and G-313 → D in E1 (318, 552, 553). H-55 was particularly important for virulence, because five other amino acids tested in this position led to attenuation in 2-week-old mice. Studies of viral replication in mouse brain and in cultured cells led to the conclusion that virus with H-55 replicated more efficiently in both neuronal and nonneuronal cells (553). The identities of the residues at positions 55 and 172 in E2 and at positions 72 and 313 in E1, as well as other changes in both E1 and E2, were also important for the virulence of the virus in newborn mice.

The importance of H-55 in E2 for growth in neurons and for neurovirulence was also shown in a study in which scid mice were infected with AR339 SIN (288). Weanling scid/CB17 mice are persistently infected by SIN. The infection can be abrogated by treatment with antibodies, but in about half the mice the infection reactivates after 30 to 90 days (289, 290). Virus was isolated from the brains of mice that had been persistently infected for 39 days without treatment with antibody or from mice that had been treated with antibody and in which the infection had reactivated after 37 to 52 days. Of nine virus isolates, four had the change Q-55 → H in E2 and all four were virulent for weanling BALB/cJ mice. The remaining five isolates retained Q-55 and were avirulent for weanling BALB/cJ mice. Thus, extended growth in mouse brain selected for a virus containing H-55, and this virus was neurovirulent. Interestingly, the H-55-containing virus did not result in the death of the scid mice, suggesting that a host immunopathologic response may be involved in the severity of the encephalitis.

Johnston and collaborators have studied attenuation of SIN infection in neonatal mice. They selected variants that were able to penetrate cultured BHK cells rapidly and found that these variants were also attenuated in mice and altered in their reactivity with certain MAbs (18, 378, 379). The substitution responsible for all three changes in virus phenotype was S-114 → R in E2 (87, 406), and this substitution was found to reduce the stability of the virus on exposure to a number of inactivating agents, suggesting that the conformation of the glycoprotein spikes is altered (150). Although these three phenotypes covaried in selection procedures, it was later found that the attenuated phenotype could be separated from the rapid penetration phenotype (408) and from the MAb reactivity phenotype (396). In the course of these studies, other attenuating mutations in both E1 and E2 were characterized (406, 407).

Schoepp and Johnston (477, 478) examined mutants with site-specific mutations at E2 positions 62, 96, and 159, which form part of antigenic site E2c, and found many changes that reduced virulence and reduced binding of MAbs specific for site E2c. In general, binding of antibody was correlated with virulence, and the authors concluded that residues 62, 96, 114, and 159 are probably close together in the protein structure

and define an epitope that is important in some way for pathogenesis. They also found that changes at position 62 or 159 could render the virus virulent regardless of the residue at position 114.

Russell et al. (437) studied attenuation of the AR86 strain of SIN. This strain, unlike the AR339 strain used for most studies, is virulent for adult mice when inoculated intracranially. Rapid-penetration mutants were isolated and found to be attenuated. Two different changes were found to attenuate the virus. In one, S-114 → R in E2, the same change found to attenuate AR339 for neonatal mice, led to attenuation of AR86. In the second, S-1 → N in E2 gave rise to a new glycosylation site that resulted in failure of PE2 to be cleaved and in attenuation of virus virulence.

These various studies make clear the importance of the glycoproteins for SIN neurovirulence. Using a different approach, Kuhn et al. (271) examined the effect of deletions in the 5′ and 3′ NTRs of the SIN genome RNA. They found that the effect of any particular deletion was host cell specific. Several deletions that resulted in lowered RNA synthesis in infected cells and attenuated the neurovirulence of the virus in mice were found, and attenuation was correlated imperfectly with the replication efficiency of the virus in cultured mouse NIH 3T3 cells. In one case the virus grew better in the brain than did the parental virus and was increased in its neurovirulence but grew like the parental virus in cultured cells. Thus, as has been found for other viruses such as poliovirus (263), changes in nontranslated regions can be attenuating.

Variants of RR that are either enhanced or reduced in their virulence for mice have also been characterized. Passage of a strain of RR of low virulence for neonatal mice 10 times in mice resulted in a marked increase in virulence (342). It was concluded that early in passage, unmapped changes in the nonstructural proteins occurred that resulted in increased replication efficiency and increased virulence. Later during passage, changes in the E2 protein occurred that led to a still more efficient replication of the virus. Changes at nine different E2 residues were observed in the various variants sequenced. In another study, Vrati et al. (572) found that deletion of residues 55 to 61 of E2 in a virulent RR strain resulted in attenuation of the virulence of the virus for mice.

Attenuated variants of SF have been isolated, and the pathology of virus infection has been studied (13, 320, 321). Although no details of the molecular changes involved are known, several interesting studies on the pathologic changes caused by an attenuated variant of SF, called A7(74), have been reported. Growth of this mutant is restricted in the mouse brain; although the virus replicates in mouse neurons, little virus budding is observed (117). Infection of mice with the virus causes an acute inflammatory reaction in the central nervous system, which leads to demyelination; demyelination can be prevented by depletion of CD8⁺ T cells (534). Treatment of mice with gold sodium thiomalate results in the proliferation of membranes in the mouse central nervous system and leads to increased growth and increased virulence of A7(74), suggesting that the restriction in A7(74) growth in the brain is due at least in part to an inability of the virus to induce sufficient proliferation of membranes required for virus growth (343). Perhaps related to the hypothesis that membrane proliferation is required for SF growth in neurons is the report that synthesis of SF RNA requires continuous lipid synthesis (400).

The neurovirulence of VEE, a New World virus, has also been studied. VEE is neurovirulent for mice and can cause encephalitis in horses and humans. An attenuated strain that is avirulent for mice and has been used as a vaccine in horses and

in humans was isolated by passage in guinea pig heart cells. The complete sequence of this attenuated TC-83 strain has been obtained and compared with the sequence of the parental Trinidad donkey strain (228, 257–259). There are only 12 nucleotide changes between the two viruses, of which 5 lead to amino acid changes in E2 and 1 results in a change in E1, illustrating that passage in cultured cells often selects preferentially for changes in the envelope glycoproteins and that such changes are often attenuating. Mapping of the attenuating mutations in TC-83 by making recombinant viruses between the parental TRD strain and TC-83 and testing the virulence of the resulting viruses in mice showed that attenuation in mice resulted primarily from two changes. The amino acid change T-120 → R in E2 was the most important attenuating mutation, followed by the nucleotide substitution G-3 → A in the 5′ NTR of the virus; in combination, these two changes alone resulted in a completely attenuated virus (257).

Variants of VEE that are attenuated in their virulence for mice have also been isolated by selection for rapid penetration of cells in culture (232). Attenuation was due to amino acid changes in E2 at positions 3+4, 76, 120, or 209 (89). Combining these mutations gave rise to further attenuation of virus virulence. It is interesting that changes in the envelope of alphaviruses that allow them to penetrate BHK cells in culture more rapidly result in less efficient growth in mice.

Clearance of virus from persistently infected neurons. Most nucleated cells in the body produce class I MHC, and infected cells can be killed by cytotoxic T lymphocytes reactive with viral epitopes presented in the context of class I MHC. Neurons are terminally differentiated cells that do not express class I MHC and are therefore not a target for cytotoxic T cells. How can viral infections of neurons be cleared? Levine et al. (290) investigated the clearance of SIN infection from mouse brain and found that humoral antibody could inhibit virus replication in neurons by a noncytolytic mechanism. SIN infection of adult scid mice, which are unable to mount either a humoral or a T-cell immune response, led to persistent infection of brain and spinal cord neurons that could be cured by passive transfer of humoral antibody but not by passive transfer of T cells. Using MAbs, the authors found that three neutralizing MAbs directed against E2 were able to clear the infection but that other anti-E2 MAbs (including other neutralizing MAbs) and anti-E1 MAbs were not effective. There was no defined relationship between neutralizing activity, protective capacity upon prophylactic treatment, and ability to mediate complement-dependent lysis; this suggested that viral clearance from neurons involves a different humoral effector mechanism. These results could be reproduced in an in vitro system in which dorsal root ganglia were explanted and cultured. Infection of these neurons with SIN led to a persistent infection that could be cured by treatment with the same three MAbs that cured the infection in vivo, and the antibody treatment interfered with virus RNA transcription or translation (Fig. 28). Remarkably, continued presence of the MAb in the medium was not required; a 48-h treatment of the infected cells followed by removal of the MAb resulted in clearing of the infection over the next 5 days. The mechanism by which humoral antibodies clear SIN infection of neurons is unknown but is clearly of great interest.

Levine and Griffin (289) used a sensitive PCR assay to examine mice that had recovered from acute encephalitis caused by SIN for the persistence of viral RNA in brain cells; they found that viral RNA was still present in the brain up to 17 months after recovery from disease. In immunocompetent mice, no evidence for reactivation of viral infection was observed; however, in scid mice that had recovered following

FIG. 28. Effect of MAb therapy on SIN replication in primary rat neuron cultures. Dorsal root ganglions were dissected from embryonic rats and maintained in culture (290). Six-week-old cultures were infected with SIN at a multiplicity of 0.1. Two days later, cultures were treated with different neutralizing MAbs for 2 days (shaded area), after which the MAbs were washed from the cultures and medium without MAb was added. At 48-h intervals, aliquots of the supernatants were removed and assayed for virus by plaque assay on BHK monolayers. MAbs 209 (Δ) and R6 (○) are reactive with glycoprotein E2 epitope C, MAb 50 (□) reacts with E2 epitope A, MAb 202 (●) reacts with E2 epitope AB, and MAb 106 (▲) reacts with glycoprotein E1 epitope C. ▲, no MAb. Adapted from reference 290.

passive transfer of antibody, viral reactivation was seen in some of the mice treated with a low dose of antibody (Fig. 29). Reactivation was correlated with waning antibody titer in the animal, and the revived infection could be abrogated by fresh treatment with antibody. These authors concluded that SIN can persist for extended periods in a nonreplicating form in mouse brain and that the humoral response to virus infection plays an important role in preventing viral reactivation. The finding that a lytic RNA virus is able to persist in a latent state within neurons for long periods and that its replication is repressed by the continuing presence of humoral antibody was unexpected.

Induction of apoptosis. Alphavirus infection of vertebrate cells usually results in the death of the cell, with the notable exception of neuronal cells in which a persistent infection may be established (289, 290). Recent studies have suggested that alphaviruses kill the cell by inducing apoptosis, a cell suicide pathway characterized by DNA fragmentation and morphological changes (291). BHK cells, N18 mouse neuroblastoma cells, and AT-3 rat prostatic adenocarcinoma cells were found to exhibit symptoms characteristic of apoptosis following infection by SIN, including membrane blebbing, nuclear condensation, and fragmentation of DNA into nucleosome-sized pieces. Thus, there is the presumption that the virus induces apoptosis in some way and that this results in the death of the cell.

Production of the cellular oncogene *bcl-2* protects cells, in some unknown way, from apoptosis. One of the steps involved in the maturation of neurons is acquisition of the ability to express *bcl-2*, and this suggests that the age-dependent neurovirulence of alphaviruses may arise because mature neurons are protected from apoptosis. A number of treatments, such as withdrawal of nerve growth factor, induce apoptosis in immature neurons, which do not express *bcl-2*, but these treatments do not affect mature neurons, which do produce *bcl-2*. In immature neurons infection by wild-type SIN results in cell death, presumably by apoptosis, whereas in mature neurons a

540 STRAUSS AND STRAUSS

FIG. 29. Reactivation of SIN in the central nervous system of scid mice after antibody treatment. scid mice were inoculated intracranially with 1,000 PFU of SIN on day 0 and treated with 200 μg of MAb 209 (open circles) or anti-SIN hyperimmune serum (solid circles) on day 7; normal mice were also inoculated with 1,000 PFU on day 0 but had no antibody treatment (solid squares). At various times after infection, mice were sacrificed and assayed by plaque assay for infectious virus in the brain (A) and by enzyme-linked immunosorbent assay for the titer of anti-SIN antibody in the serum (B). Adapted from reference 289 with permission of the author and the publisher.

persistent infection is established, presumably because the cells are protected. Support for this model has come from studies in which immature rat dorsal root ganglia were explanted and grown in culture. These neurons do not express bcl-2 when first explanted, and infection by SIN results in the death of the cell. After 6 weeks in culture, bcl-2 is produced by the neurons, and they are no longer killed by the virus (291).

The importance of bcl-2 in protecting cells from virus-induced apoptosis has been shown by studies with nonneuronal cells. Infection by SIN of rat AT-3 cells leads to the induction of apoptosis and death of the cell. Transformation of AT-3 cells with the bcl-2 led to a cell line in which wild-type SIN no longer induces apoptosis and instead establishes a persistent infection (291), as illustrated in Fig. 30.

Strains of SIN that have histidine at position 55 of E2 can overcome the effects of bcl-2 expression (556). Strains with H-55 are not affected in their growth rate by the presence of the bcl-2 gene product in AT-3 cells that express bcl-2, and they induce apoptosis, whereas virus strains with Q-55 grow only about 10% as well in AT-3 cells expressing bcl-2 and do not induce apoptosis. The mechanism by which bcl-2 inhibits the replication of virus containing Q-55 is unknown. SIN strains containing H-55 are more neurovirulent in adult mice, consistent with the model that bcl-2 expression protects neurons against less neurovirulent strains of the virus.

### Replication in the Mosquito Vector

In nature, alphaviruses alternate between growth in arthropod hosts, almost always mosquitoes, and vertebrate hosts. Some arboviruses (arthropod-borne animal viruses) can be maintained by transovarial transmission in arthropods. For alphaviruses, transovarial transmission has been difficult to demonstrate and has been assumed to be rare (reviewed in reference 554). Natural vertical transmission has been demonstrated recently for WEE, however, leading to the suggestion that this is an important mechanism for the maintenance of the virus in nature (128). The possible importance of vertical transmission for other alphaviruses remains to be determined.

For horizontal transmission of an alphavirus, the vertebrate infection must produce a viremia of sufficient intensity and duration that the mosquito vector is infected when taking a blood meal, and the infection of the mosquito must lead to virus replication in the salivary glands of sufficient intensity that enough virus to infect a vertebrate is transmitted with the

FIG. 30. Replication of SIN in rat prostatic adenocarcinoma (AT-3) cells which express the cellular oncogene bcl-2. AT-3 cells were transfected with an expression vector that expresses the human bcl-2 gene (AT3Bcl2) or with the vector alone (AT3Neo). (A) Percentage of cells viable as a function of time after infection by SIN. Open symbols, mock-infected cells; solid symbols, cells infected with SIN at a multiplicity of 1 on day 0. Cells expressing bcl-2 remain viable after infection by SIN, whereas cells not expressing bcl-2 are killed by the virus. (B) Virus titer produced by these cells. Cells expressing bcl-2 (AT3Bcl2 cells) are persistently infected and continue to release virus, whereas control cells are acutely infected and die (AT3Neo cells). Adapted from reference 291 with permission of the authors and the publisher.

salivary fluid when the mosquito next feeds. These requirements have shaped the evolution of these viruses in interesting ways, and studies of the molecular biology of virus replication must take them into account. As one example, it has been found repeatedly that passage of alphaviruses in mosquitoes or in mosquito cells selects for variants that grow better in mosquito cells but less well in vertebrate cells (86, 482, 509, 544).

In vertebrate hosts alphaviruses often cause a serious illness, but the infection of the arthropod host is relatively benign. Early studies suggested that viral infection had no effect on the mosquito host, but reports of salivary gland abnormalities in *Aedes aegypti* following infection by SF suggested otherwise (reviewed in reference 185). Pathologic changes in the salivary glands of *Aedes albopictus* after infection by SIN have also been reported (42, 43), and recent experiments have demonstrated transient abnormalities in the midgut of the vector mosquito after infection by EEE (595) or WEE (593). More-detailed study of the effects of virus infection on the mosquito are needed, especially studies involving the normal vector for the virus in question.

Following ingestion of a blood meal, cells of the midgut epithelium of the mosquito are infected (reviewed in reference 185) and virus is released into the hemocele when virus buds from the basolateral surface of the midgut epithelial cells. In a small fraction of mosquitoes, the midgut appears to be “leaky,” such that virus has direct access to the hemocele (596). Once released into the hemocele, the virus eventually finds its way to the salivary glands, where a persistent infection is established. After an extrinsic incubation period of 3 days to more than 2 weeks, depending on the virus, the vector, and the environmental conditions, the mosquito is capable of transmitting the virus with any future blood meals. Strains of *Culex tarsalis*, a natural vector for WEE, have been identified in which WEE fails to replicate in the midgut (185), perhaps as a result of a lack of receptors for the virus (213). Other strains of WEE have been identified in which midgut infection occurs but infection of the salivary glands does not (185). In the case of VEE, mosquitoes that are very sensitive to infection by one strain of virus may fail to replicate closely related strains of VEE (reviewed in reference 479). Thus, the competence of any mosquito to serve as a vector for a particular alphavirus is affected by many factors.

Replication of alphaviruses in cultured mosquito cells has been studied as a proxy for the effects of virus replication in mosquitoes (reviewed in references 49, 356, 357, 509, and 543). A persistent infection is established in cultured mosquito cells, whereas a cytopathic infection occurs in most vertebrate cells. Although all mosquito cell lines examined can be persistently infected by alphaviruses, the mechanisms by which persistence is established appear to differ. In most cases virus is produced during the first 24 to 48 h at levels similar to those in cultured vertebrate cells. This is followed by a decline in virus production to low levels that are maintained indefinitely. In some cell lines and under certain conditions no cytopathic changes are noted, whereas in other lines or under different conditions there is extensive cell damage followed by establishment of a persistent infection (356, 445, 543). Increased RNA production has been found to be correlated with increased cytopathic changes, although the yield of virus was unchanged (509). Similarly, virus maturation has been reported to differ in different cell lines; in some lines maturation at the cell plasma membrane appears to be a major method of virus production, whereas in other lines virus appears to mature within intracellular vesicles (reviewed in reference 356). Maturation of virus

in intracellular vesicles has not been found in whole mosquitoes, and the role of vesicles in pathologic changes is unclear.

Following infection of cultured *A. albopictus* cells by SIN (422) or SF (369), a polypeptide factor is released that downregulates the production of virus. The factor is cell specific in that it affects only mosquito cells and virus specific in that the anti-SIN and anti-SF factors are distinct and relatively ineffective against the other virus. Only certain lines of cultured *A. albopictus* cells infected by SIN release this factor and are sensitive to its effects (74, 75). Treatment with the factor interferes with RNA synthesis following infection by SIN, and cells that produce the factor become resistant to superinfection by SIN. Full expression of the antiviral state requires 48 h of treatment with the factor. The anti-SIN factor has now been purified and found to be a hydrophobic polypeptide of 3,200 Da (315). Treatment of noninfected cells with the factor first leads to arrest of cell division, followed by recovery accompanied by acquisition of resistance to SIN and the constitutive expression of the factor (i.e., the factor induces its own synthesis). Also induced is the synthesis of a 55-kDa membrane protein that is associated with lysosomes which might be involved in the resistance to virus infection exhibited by these cells (316). Further characterization of this 55-kDa protein and comparison of the SIN antiviral factor with the anti-SF factor will clearly be of great interest for our understanding of the interactions of viruses and mosquitoes that lead to persistent infection and for our understanding of mosquito physiology and development. It is known, for example, that infection in mosquitoes is downregulated after about 7 days (reviewed in references 185 and 479); the mechanisms by which this occurs are unknown but could be related to the production of this antiviral factor.

---

### EVOLUTION OF ALPHAVIRUSES

#### Divergence of Alphaviruses

Rates of divergence. RNA viruses are capable of very rapid divergence because mutation rates are high (211, 212, 508, 530). Proofreading does not seem to occur during RNA replication (507), resulting in a high misincorporation rate. The free energy of base pair formation allows a discrimination of only \(10^{-4}\) to \(10^{-5}\) during RNA (or DNA) synthesis, and experimentally measured rates of misincorporation by DNA or RNA polymerases in the absence of proofreading are of this order (reviewed in reference 212).

Although the average rate of mutation is \(10^{-4}\) to \(10^{-5}\), there appear to be well-established examples in which the rate is higher (on the order of \(10^{-3}\)) or lower (on the order of \(10^{-7}\) to \(10^{-8}\)) at specific nucleotides in alphavirus genomes. In the case of SIN, apparent reversion rates of \(10^{-4}\) to \(10^{-5}\) have been found for most sequenced *ts* mutants that have been carefully examined (178, 181, 183). However, reversion rates of \(\sim10^{-8}\) have been found in three well-studied cases. Kuhn et al. (272) found that a SIN mutant having a C as residue 7 upstream of the poly(A) tract in the 3′ NTR was *ts*. Change of this residue to U, A, or G (or even deletion of the C residue) resulted in a temperature-insensitive virus. The *ts* virus reverted to temperature insensitivity at a frequency of \(<10^{-8}\), however. Similarly, Hahn et al. (183) found that *ts24* was *ts* because of a single nucleotide transition (G → A, leading to the amino acid substitution G-736 → S in nsP2); reversion of the virus to temperature insensitivity was \(<10^{-8}\), even when sequenced constructs were used to rule out the presence of other mutations that might contribute to temperature sensitivity. Finally, Durbin and Stollar (107) studied a mutant that was *ts* because

it had an extra glycosylation site in E2. They found that revertants arose by U-9455 → A, A-9454 → U, or C-9460 → U changes, but the reversion frequency was estimated to be < 5 × 10⁻⁷. In each of these cases, contribution of a second mutation to temperature sensitivity seems to have been ruled out. It appears either that misincorporation occurs very infrequently at certain positions in the viral RNA for reasons that are not understood at present or that limited proofreading does occur.

Although alphaviruses have the potential for rapid divergence, the actual rates of divergence measured are very low in comparison with those of other RNA viruses for which good data exist (reviewed in reference 594). RR virus isolated at the beginning and end of an 11-month epidemic that swept through the Pacific region differed at only 1 nt in 1,600 sequenced (1,200 sequenced in the E2 region and 400 sequenced in the 3′ NTR), and the measured rate of change during the epidemic was thus 6 × 10⁻⁴/nt/year (53). The rate of divergence over larger regions of the genome might be even lower, because it was hypothesized that the one change observed might have been selected by the change in vertebrate hosts that occurred during the epidemic. Sequencing of EEE isolated over a period of many years from different locations in North America gave an estimate of 1.4 × 10⁻⁴/nt/year for the rate of divergence (597). From oligonucleotide fingerprinting studies, the rates of divergence of VEE (421) and WEE (549) were estimated to be < 5 × 10⁻⁴ and 3 × 10⁻⁴/nt/year, respectively. Thus, all studies of alphavirus divergence give values of 1 × 10⁻⁴ to 6 × 10⁻⁴/nt/year for the rate of divergence, contrasting with rates of divergence of ~10⁻²/nt/year found for a number of other RNA viruses (495, 508, 530).

Weaver et al. (588) examined unpassaged EEE isolates from a single host and found that the diversity between different isolates appeared to be similar to those reported for other RNA viruses. They concluded that the low rate of virus evolution was not due to a decreased rate of generation of variants.

It seems probable that the low rate of divergence of alphaviruses results in large part from the requirement for alternation of growth in an arthropod host (at relatively low temperature) and in higher vertebrates (at ≥37°C). The virus must be highly adapted to persist in these alternating hosts, and changes that better adapt the virus to one host, which are readily selected upon passage in culture, or that might be neutral in one host may well be deleterious in the alternate host. In addition, infection of a mosquito when it takes a blood meal or of a vertebrate when an infected mosquito feeds is initiated by doses estimated to be >10² infectious viruses, and initiation of infection with such relatively large multiplicities minimizes founder effects.

Divergence of EEE and VEE. The molecular evolution of EEE and of VEE has been studied by partial or complete sequencing of many strains of virus isolated over periods of many years from the same or different locations, and the results for the two viruses differ in interesting and informative ways. In a study of 10 strains of EEE isolated in North America between 1933 and 1985, the virus was found to constitute a single lineage (429, 590, 597) in which all viruses were very closely related and in which the rate of divergence was 1.4 × 10⁻⁴ substitution per site per year. The fact that viruses isolated over a period of 55 years from locations as far apart as Massachusetts, Wisconsin, Florida, and the Dominican Republic constituted a single, slowly changing lineage implies that regular EEE dispersal and gene mixing occurs. The mechanisms for this have not been proven, but it is suspected that dissemination by infected birds is responsible. EEE is maintained in North America by transmission among passerine birds in freshwater swamps by the mosquito *Culiseta melanura* (479). Epizootics of disease are thought to arise when other mosquito species that feed on both birds and mammals become infected and spread the virus outside this narrow focus. The low rate of change in the virus may result in part from this narrow host range, and it is possible that infected birds regularly introduce or reintroduce the virus into temperate areas from overwintering areas in the subtropics, maintaining continual gene mixing. A more comprehensive study of EEE divergence in North America involving 24 additional isolates found that there are presently two closely related but distinguishable cocirculating lineages of EEE in North America (586, 591). The divergence of the North American group into two lineages probably occurred in the early 1970s. Prior to 1970, the rate of divergence of EEE was found to be only about 2 × 10⁻⁵. However, after 1970 the rate of divergence increased over 10-fold, to about 4 × 10⁻⁴.

Although the overall rate of EEE divergence in North America is low and gene mixing appears to occur, Weaver et al. (588) also found that within a given year genetic diversity was generally greater among geographically distant isolates than among those from the same transmission focus. These several studies suggest that gene mixing of EEE in North America requires several years to complete, during which time the virus diverges into multiple geographically separated lineages, acquiring base substitutions at a rate of about 4 × 10⁻⁴ per nucleotide per year. Over a long time period, however, frequent extinctions of lineages and reintroductions of virus by migratory birds maintain EEE as a single gene pool, and this gene pool evolves more slowly, with approximately 2 × 10⁻⁵ substitutions per site per year.

EEE in South America is distinct from the North American virus and has evolved into at least two subtypes that are geographically separated (586, 591). In South America the virus is probably maintained in enzootic foci and continual mixing of the viruses does not occur, in contrast to the situation for North American EEE. The rate of divergence in the South American strains was found to be about 4 × 10⁻⁴, comparable to rates for other alphaviruses and to the post-1970 rate for North American EEE. It was estimated that the North American and South American strains diverged about 1,000 years ago and that the two South American lineages diverged about 450 years ago.

The VEE complex, which is estimated to have diverged from EEE about 1,400 years ago, has evolved into a number of geographically isolated subtypes, thus resembling the case for South American EEE rather than North American EEE (421, 589). The virus is maintained in enzootic foci by transmission by a number of mosquito species to small mammals, and the relative lack of mobility of these mammals may be responsible for the geographic isolation and independent evolution of the many subtypes. The virus has erupted a number of times into epizootics that resulted in the infection of large numbers of horses and people and that subsequently died out. It is believed that variants of enzootic strains are the usual source of the epizootic viruses. However, the epizootic of 1969 to 1972, which spread from Ecuador through Central America and into Texas, almost certainly arose from an insufficiently inactivated vaccine in use throughout Central America. Virus isolated in Texas in 1972 was almost identical to the Trinidad Donkey strain of virus isolated during an epizootic in Trinidad in 1943 that was used as the source of virus for formalin-inactivated vaccines, whereas it was quite different in sequence from that of other VEE isolates (260, 261, 496).

VEE has been introduced into new areas on occasion, perhaps by infected birds. Of particular interest was the

TABLE 3. Percent amino acid identity among alphaviruses$^a$

| Virus | SIN | WHA | AURA | WEE | EEE | VEE | ONN | SF | RR | MAY |
|-------|-----|-----|------|-----|-----|-----|-----|----|----|-----|
| SIN   | —   | 79.7 | 61.9 | 70.7 | 50.1 | 47.4 | 44.6 | 47.7 | 48.9 | 45.5 |
| WHA   | 86.2 | —   | 61.4 | 69.0 | 51.7 | 48.8 | 44.8 | 46.1 | 47.6 | 45.3 |
| AURA  | 73.3 | 73.4 | —   | 58.0 | 49.0 | 46.8 | 44.7 | 45.7 | 46.4 | 44.1 |
| WEE   | NA$^c$ | NA | NA | —   | 56.4 | 50.6 | 43.4 | 45.3 | 46.2 | 44.1 |
| EEE   | 60.4 | 60.8 | 59.6 | NA | —   | 57.6 | 49.2 | 50.6 | 49.4 | 49.1 |
| VEE   | 60.8 | 61.3 | 59.5 | NA | 72.8 | —   | 46.8 | 47.7 | 47.5 | 46.6 |
| ONN   | 62.5 | 62.6 | 61.3 | NA | 62.9 | 62.6 | —   | 62.4 | 61.5 | 61.0 |
| SF    | 63.9 | 62.7 | 62.2 | NA | 63.3 | 61.8 | 73.4 | —   | 75.3 | 68.7 |
| RR    | 63.5 | 63.6 | 62.2 | NA | 63.3 | 63.0 | 73.6 | 76.8 | —   | 67.5 |

$a$ For each pairwise combination of viruses, the percent identical amino acids are shown for the structural region (C, E3, E2, 6K, and E1) (upper right half) and for the nonstructural domain (nsP1, nsP2, nsP3, and nsP4) (lower left). Sequences used for these comparisons are found in the references cited in Table 1. Pairs with ≥60% identity are boxed.

$b$ Within the structural protein domain, the N-terminal one-third of the capsid protein is not conserved and was not used in these comparisons. Within the nonstructural protein domain (nsP1 to nsP4) the C-terminal half of nsP3 is not conserved (see Table 2) and was not used in these comparisons.

$c$ NA, sequence not available.

CAPSID
E2

encoded in a DI RNA under control of a subgenomic promoter (Fig. 23B). When transfected into the same cell, these two RNAs complemented one another and infectious virions that contained copies of both RNAs were produced. Weiss and Schlesinger (601) found that these two RNAs underwent recombination at an easily detectable frequency to produce a nondefective RNA capable of a complete replication cycle. Recombination was not precise but took place anywhere within the overlap zone to give recombinant RNAs of different sizes. Some had two copies of the junction region between the nonstructural and structural domains and two subgenomic promoters, both of which were active. With continued passage, recombinants continued to evolve toward the wild-type configuration with a single junction sequence. These studies clearly demonstrate the inherent flexibility of RNA virus genomes and imply that recombination can and does play a role in the evolution of alphavirus genomes.

Recombination among Alphaviruses

Recombination in the laboratory. Recombination in alphaviruses occurs infrequently, so that it has been difficult to study or to quantify this phenomenon. It has taken the development of full-length clones that can be manipulated in order to demonstrate recombination in the laboratory. Geigenmüller-Gnirke et al. (149) developed a SIN derivative with a bipartite genome. The nonstructural proteins were encoded in a SIN genomic RNA in which the structural proteins had been replaced by the CAT gene, and the structural proteins were

FIG. 32. Alphavirus phylogenetic trees based on the structural proteins. Unrooted trees based upon the sequences of the capsid protein and of glycoprotein E2 are shown. Note that WEE clusters with EEE in the capsid protein tree but with SIN in the E2 tree. Adapted from reference 292.

Recombination to form WEE. Recombination has been shown to have occurred between alphaviruses in nature, producing a new virus (177, 592). WEE is a recombinant between EEE and a virus related to SIN in which most of the genome is derived from the EEE parent but the genes for glycoproteins E2 and E1 are derived from the SIN-like parent (Fig. 32 and 33). Two crossovers were required to produce WEE. The upstream crossover appears to have occurred within E3 between amino acid residues 5 and 24 of the SIN sequence. The downstream crossover appears to have occurred within the 3' NTR (nt 11381 to 11703 in SIN), after nt ~11440 of the SIN genome. There follow 160 nt in the WEE 3' NTR that are not homologous to either parental sequence and whose origin is not known, and the last 80 nt of the 3' NTR were then derived from EEE. The two crossovers required may not have occurred

![Diagram]

PERCENT SEQUENCE IDENTITY

| VIRUS | PROTEIN |
| --- | --- |
|  | nsP2 | nsP3 (1-324) | nsP4 (321-610) | Capsid (106-end) | Envelope |
| WEE/EEE | 82 | 80 | 89 | 85 | 50 |
| WEE/SIN | 54 | 54 | 77 | 67 | 71 |
| EEE/SIN | 55 | 53 | 78 | 61 | 46 |

FIG. 33. WEE is a recombinant virus. Genome organizations of SIN (shaded boxes) and of EEE (open boxes) are shown together with a schematic of the genome of WEE in which the genes derived from EEE are shown as open boxes and those from a SIN-like progenitor are shown shaded. Sequences are found in the references cited in Table 1; the areas with dotted outlines are the portions of WEE for which no sequence has been published. Below, the amino acid identities shared by WEE, EEE, and SIN in the various regions are tabulated. Modified from reference 527 with permission of the publisher.

ALPHAVIRUS

CAP- nsP1 nsP2 nsP3 nsP4 C E3 E2 6K E1 Aₙ
(MTR 1) (HEL 1) (P-PRO) X (POL 3)

CAP- nsP1 nsP2 nsP3 nsP4 C E2 E1 Aₙ
(MTR 1) X (P-PRO) (HEL 1) (POL 3)
→ 24S RNA

RUBELLA

FIG. 34. Genome organization of rubella (RUB) compared with that of SIN. The genomes of SIN and RUB are drawn to scale, and several functional domains are labeled according to the terminology of Koonin and Dolja (267). In the complete sequence of RUB (98, 123), a number of domains with sequence homology to SIN have been identified; these include the methyltransferase (MTR 1), the helicase (HEL 1), a papain-like proteinase (P-PRO), the X domain of unknown function, and the polymerase (POL 3). However, in the RUB genome these elements appear in a different order, suggesting that recombination has occurred during their divergence. The boxed 2 indicates the location of CSE 2 (see Fig. 18).

in one event. The crossover in the 3′ NTR could have been a second event required to produce an efficiently replicating virus, since the 3′ NTR contains promoter elements that interact with viral (and cellular) proteins to promote replication of the viral RNA. It seems likely that the recombination event occurred in mosquitoes in which the virus sets up a persistent, lifelong infection.

WEE is a widely distributed and successful virus. It is found from Canada discontinuously to Argentina and can be isolated every year over this very wide geographic range. The time that the recombination event occurred to give rise to WEE is not known. From a phylogenetic analysis of the alphavirus group, Weaver et al. (592) concluded that the EEE parent of WEE and the ancestor to the currently circulating EEE strains diverged more than 1,000 years ago, suggesting that WEE arose more than 1,000 years ago.

It seems clear that EEE served as one of the parents of WEE. The SIN-like parent is unknown, however. The New World SIN-like virus AURA is no more closely related to WEE than it is to SIN AR339 and could not have served as an immediate parent (Table 3); in fact, present-day African SIN and WHA from New Zealand are the most closely related in their glycoproteins to WEE of the known SIN-like viruses. It is possible that the SIN-like parent no longer exists, perhaps being replaced by WEE.

Examination of other alphavirus sequences for evidence of recombination suggests that recombination to produce new alphaviruses is rare. Weaver et al. (592) reported that the positions of ONN and of MID in phylogenetic trees of nsP3 and nsP4 differed, suggesting that one of these may be recombinant (Fig. 31). In no other case except for WEE, however, was evidence for recombination found. It appears that divergence of most alphavirus has resulted from linear descent from a common ancestor and that recombination among alphaviruses to produce viruses that persist in nature is a rare but significant event.

The Family Togaviridae

Alphaviruses are classified as a genus in the family Togaviridae, which contains a second genus with clear genome organization similarities to the alphaviruses, the genus Rubivirus. The 9,756-nt genome of rubella virus (RUB) has been sequenced in its entirety, and much is known about the organization and expression of the genome (reviewed in reference 123). The genome organization of RUB is very similar to that of alphaviruses (Fig. 34). The RUB genome, like alphaviruses, contains a 5′ nonstructural domain translated from the genomic RNA

and a 3′ structural domain translated from a subgenomic RNA. The nonstructural domain is translated into a polyprotein that is cleaved by a papain-like proteinase that appears to be related to the nonstructural protease of the alphaviruses. There are domains in the nonstructural polyprotein of RUB that share homology with domains in alphavirus nsP1 (the methyltransferase), nsP2 (both the helicase domain and the proteinase domain), nsP3 (function unknown, referred to as the X domain), and nsP4 (the RNA polymerase), although there have been rearrangements in the order of these domains between the RUB and alphavirus nonstructural polyproteins (123). The structural proteins of RUB, two glycoproteins and a capsid protein whose gene order is the same as in the alphaviruses, are translated from a 24S subgenomic mRNA. The promoter for the subgenomic RNA is similar to that of the alphaviruses, and, in fact, the RUB promoter can be used by the SIN replicase to make a subgenomic RNA, albeit 100-fold less efficiently than the homologous promoter (Fig. 19). A sequence element very similar to the 51-nt CSE of alphaviruses is present in RUB, indicated by the boxed 2 in Fig. 34, and there is a structural element present at the 5′ end of the RUB genome. We assume that these sequences are regulatory elements that perform the same function in RUB replication as do the corresponding sequences in alphavirus replication and that this function has been conserved during evolution of the viruses.

Despite the close similarity in the genome organizations of alphaviruses and RUB, phylogenetic analysis of the nonstructural sequences suggests that these viruses are only distantly related. The helicase proteins and the replicase proteins of RUB have been found to be closer to those of hepatitis E virus and of the furoviruses than they are to those of the alphaviruses, which in turn are more closely related to those of a number of plant viruses including the tobamoviruses, tobraviruses, ilarviruses, bromoviruses, and hordeiviruses than they are to those of RUB (266, 592). Recombination events within the Sindbis-like superfamily of viruses described below have led to reorganization and scrambling of the genomes of this large group of viruses during evolution (267).

The Sindbis-Like Supergroup of Viruses

One of the more interesting results from the complete sequencing of many viral genomes has been the close relationships that have been found between certain groups of plant and animal viruses and, more generally, between all positive-strand RNA viruses. The alphaviruses possess a genome organization that is similar to the genome organization of a number of plant

TOBAMOVIRUS

CAP-[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]—[]
HEL3 VPg C-PRO POL1
Picorna/Calici
Como/Nepoviruses

HEL2 VPg C-PRO POL1
Potyviruses

X P-PRO C-PRO POL1 HEL1
Coronaviruses

P-PRO S-PRO POL1 HEL1
Arteriviruses

S-PRO HEL2 MTR2 POL2
Phage/Carmo/
Tombusviruses

MTR1 P-PRO HEL1 POL3
Carla/Capillo/
Tymoviruses

MTR1 X P-PRO HEL1 POL3
Rubella

MTR1 HEL1 P-PRO X POL3
Alphaviruses

MTR1 HEL1 POL3
Tobamo/Tobravirus

MTR1 HEL1 POL3
Tricorna/Hordeivirus

FIG. 36. Genome organizations of plus-strand RNA viruses. These viruses can be grouped into three supergroups based on sequence relationships. The RNA polymerases (POL) can be divided into three groups, as shown; the proteinases (PRO) can be divided into three groups; the methyltransferases (MTR) can be divided into two groups; and the helicases (HEL) can be divided into two groups. Some viruses also have a VPg (genome-linked virus protein) or a domain labeled X (function unknown) that is shared by more than one virus. From a relatively small number of building blocks it is possible to arrive at the genomes of all of these viruses by divergence of individual domains and by recombination to reassemble them into different plans. Data for this figure are from reference 267.

group are known. Sequence elements required for RNA replication and encapsidation have been identified and characterized, and the function of these elements has been examined by site-specific mutagenesis and deletion mapping. Sequence comparisons have elucidated relationships between alphaviruses and plant viruses, leading to a better understanding of the evolution of these viruses in nature. Furthermore, full-length cDNA clones from which infectious RNA can be transcribed in vitro have been constructed for at least four alphaviruses, SIN, RR, SF, and VEE viruses, which has made possible a host of experiments. Such experiments include the mapping of ts mutants that help define the functions of many of the virus-encoded proteins, the mapping of mutations that lead to increased or decreased virulence, and the construction of site-specific mutants. Possession of clones for more than one virus also makes it possible to compare the effects of a mutation in one virus with the result found for the identical mutation in a second virus and permits the exploration of hybrid genomes in which the genomes are combined in various ways to ask which elements are able to interact with elements of a second virus, either at the protein level or at the RNA level. We have in our hands an extensively characterized genetic system for further study, and it is certain that over the next few years we will learn a great deal more about these viruses, their evolution, and details of their molecular biology.

ACKNOWLEDGMENTS

We are grateful to our many colleagues who shared ideas with us and who furnished us with material before publication. We are particularly grateful to R. H. Cheng, R. J. Kuhn, N. H. Olson, M. G. Rossmann, H.-K. Choi, and T. S. Baker for furnishing us with figures of RR virus structure prior to publication.

Our work is supported by grants AI 10793 and AI 20612 from NIH and grant DMB-9104054 from NSF.

REFERENCES

1. Abell, B. A., and D. T. Brown. 1993. Sindbis virus membrane fusion is mediated by reduction of glycoprotein disulfide bridges at the cell surface. J. Virol. 67:5496–5501.
2. Acheson, N. H., and I. Tamm. 1967. Replication of Semliki Forest virus: an electron microscopic study. Virology 32:128–143.
3. Adams, R. H., and D. T. Brown. 1985. BHK cells expressing Sindbis virus-induced homologous interference allow the translation of nonstructural genes of superinfecting virus. J. Virol. 54:351–357.
4. Ahlquist, P., E. G. Strauss, C. M. Rice, J. H. Strauss, J. Haseloff, and D. Zimmern. 1985. Sindbis virus proteins nsP1 and nsP2 contain homology to nonstructural proteins from several RNA plant viruses. J. Virol. 53:536–542.
5. Alanen, M., J. Wartiovaara, and H. Söderlund. 1987. Sequences conserved in the defective interfering RNAs of Semliki Forest virus: an electron microscopic heteroduplex analysis. Hereditas 106:19–29.

6. Aliperti, G., and M. J. Schlesinger. 1978. Evidence for an autoprotease activity of Sindbis virus capsid protein. Virology 90:366–369.

7. Anthony, R. P., and D. T. Brown. 1991. Protein-protein interactions in an alphavirus membrane. J. Virol. 65:1187–1194.

8. Anthony, R. P., A. M. Paredes, and D. T. Brown. 1993. Disulfide bonds are essential for the stability of the Sindbis virus envelope. Virology 190:330–336.

9. Arias, C., J. R. Bell, E. M. Lenches, E. G. Strauss, and J. H. Strauss. 1983. Sequence analysis of two mutants of Sindbis virus defective in the intracellular transport of their glycoproteins. J. Mol. Biol. 168:87–102.

10. Arthur, C. E., E. Lee, I. D. Marshall, R. C. Weir, and L. Dalgarno. Unpublished data.

11. Atkins, G. J. 1976. The effect of infection with Sindbis virus and its temperature-sensitive mutants on cellular protein and DNA synthesis. Virology 71:593–597.

12. Atkins, G. J. 1979. Establishment of persistent infection in BHK-21 cells by temperature-sensitive mutants of Sindbis virus. J. Gen. Virol. 45:201–207.

13. Atkins, G. J., B. J. Sheahan, and N. J. Dimmock. 1985. Semliki Forest virus infection of mice: a model for genetic and molecular analysis of viral pathogenicity. J. Gen. Virol. 66:395–408.

14. Bachmair, A., D. Finley, and A. Varshavsky. 1986. In vivo half-life of a protein is a function of its amino-terminal residue. Science 234:179–186.

15. Bachmair, A., and A. Varshavsky. 1989. The degradation signal in a short-lived protein. Cell 56:1019–1032.

16. Bailyes, E. M., K. I. J. Shennan, A. J. Seal, S. P. Smeeakens, D. F. Steiner, J. C. Hutton, and K. Docherty. 1992. A member of the eukaryotic subtilisin family (PC3) has the enzymatic properties of the type 1 proinsulin-converting endopeptidase. Biochem. J. 285:391–394.

17. Baric, R. S., L. J. Carlin, and R. E. Johnston. 1983. Requirement for host transcription in the replication of Sindbis virus. J. Virol. 45:200–205.

18. Baric, R. S., D. W. Trent, and R. E. Johnston. 1981. A Sindbis virus variant with a cell-determined latent period. Virology 110:237–242.

19. Barr, P. J. 1991. Mammalian subtilisins: the long-sought dibasic processing endoproteases. Cell 66:1–3.

20. Barrett, A. D. T., W. D. Cubitt, and N. J. Dimmock. 1984. Defective interfering particles of Semliki Forest virus are smaller than particles of standard virus. J. Gen. Virol. 65:2265–2268.

21. Barth, B. U., M. Suomalainen, P. Liljeström, and H. Garoff. 1992. Alphavirus assembly and entry: role of the cytoplasmic tail of the E1 spike subunit. J. Virol. 66:7560–7564.

22. Barton, D. J., S. G. Sawicki, and D. L. Sawicki. 1988. Demonstration in vitro of temperature-sensitive elongation of RNA in Sindbis virus mutant ts6. J. Virol. 62:3597–3602.

23. Barton, D. J., S. G. Sawicki, and D. L. Sawicki. 1991. Solubilization and immunoprecipitation of alphavirus replication complexes. J. Virol. 65:1496–1506.

24. Bashford, C. L., G. M. Alder, M. A. Gray, K. J. Micklem, C. C. Taylor, P. J. Turek, and C. A. Pasternak. 1985. Oxonol dyes as monitors of membrane potential: the effects of viruses and toxins on the plasma membrane potential of animal cells in monolayer culture and in suspension. J. Cell. Physiol. 123:326–336.

25. Beltzer, J. P., K. Fiedler, C. Fuhrer, I. Geffen, C. Handschin, H. P. Wessels, and M. Spiess. 1991. Charged residues are major determinants of the transmembrane orientation of a signal-anchor sequence. J. Biol. Chem. 266:973–978.

26. Berben-Bloemheuvel, G., M. A. M. Kasperaitis, H. van Heugten, A. A. M. Thomas, H. van Steeg, and H. O. Voorma. 1992. Interaction of initiation factors with the cap structure of chimeric mRNA containing the 5′-untranslated regions of Semliki Forest virus RNA is related to translational efficiency. Eur. J. Biochem. 208:581–587.

27. Berger, M., and M. F. G. Schmidt. 1984. Identification of acyl donors and acceptor proteins for fatty acid acylation in BHK cells infected with Semliki Forest virus. EMBO J. 3:713–719.

28. Berglund, P., M. Sjöberg, G. J. Atkins, B. J. Sheahan, and P. Liljeström. 1993. Semliki Forest virus expression system: produc-

tion of conditionally infectious recombinant particles. Bio/Technology 11:916–920.

29. Birdwell, C. R., E. G. Strauss, and J. H. Strauss. 1973. Replication of Sindbis virus. III. An electron microscopic study of virus maturation using the surface replica technique. Virology 56:429–438.

30. Birdwell, C. R., and J. H. Strauss. 1974. Distribution of the receptor sites for Sindbis virus on the surface of chicken and BHK cells. J. Virol. 14:672–678.

31. Boege, U., G. Wengler, G. Wengler, and B. Wittman-Liebold. 1981. Primary structure of the core proteins of the alphaviruses Semliki Forest virus and Sindbis virus. Virology 113:293–303.

32. Boere, W. A. M., B. J. Benaissa-Trouw, M. Harmsen, C. A. Kraaijeveld, and H. Snippe. 1983. Neutralizing and non-neutralizing monoclonal antibodies to the E₂ glycoprotein of Semliki Forest virus can protect mice from lethal encephalitis. J. Gen. Virol. 64:1405–1408.

33. Boere, W. A. M., B. J. Benaissa-Trouw, T. Harmsen, T. Erich, C. A. Kraaijeveld, and H. Snippe. 1985. Mechanisms of monoclonal antibody-mediated protection against virulent Semliki Forest virus. J. Virol. 54:546–551.

34. Boere, W. A. M., B. J. Benaissa-Trouw, T. Harmsen, T. Erich, C. A. Kraaijeveld, and H. Snippe. 1986. The role of complement in monoclonal antibody-mediated protection against virulent Semliki Forest virus. Immunology 58:553–559.

35. Boere, W. A. M., T. Harmsen, J. Vinje, B. J. Benaissa-Trouw, C. A. Kraaijeveld, and H. Snippe. 1984. Identification of distinct antigenic determinants on Semliki Forest virus by using monoclonal antibodies with different antiviral activities. J. Virol. 52:575–582.

36. Boggs, W. M., C. S. Hahn, E. G. Strauss, J. H. Strauss, and D. E. Griffin. 1989. Low pH-dependent Sindbis virus-induced fusion of BHK cells: differences between strains correlated with amino acid changes in the E₁ glycoprotein. Virology 169:485–488.

37. Bonatti, S., and G. Blobel. 1979. Absence of a cleavable signal sequence in Sindbis virus glycoprotein PE2. J. Biol. Chem. 254:12261–12264.

38. Bonatti, S., R. Cancedda, and G. Blobel. 1979. Membrane biogenesis: in vitro cleavage, core glycosylation, and integration into microsomal membranes of Sindbis virus glycoproteins. J. Cell Biol. 80:219–224.

39. Bonatti, S., G. Migliaccio, G. Blobel, and P. Walter. 1984. Role of signal recognition particle in the membrane assembly of Sindbis viral glycoproteins. Eur. J. Biochem. 140:499–502.

40. Bonatti, S., G. Migliaccio, and K. Simons. 1989. Palmitylation of viral membrane glycoproteins takes place after exit from the endoplasmic reticulum. J. Biol. Chem. 264:12590–12595.

41. Both, G. W., and M. J. Sleigh. 1981. Conservation and variation in the hemagglutinins of Hong Kong subtype influenza viruses during antigenic drift. J. Virol. 39:663–672.

42. Bowers, D. F., B. A. Abell, and D. T. Brown. Replication of the alphavirus Sindbis in the mosquito *Aedes albopictus*. Submitted for publication.

43. Bowers, D. F., and D. T. Brown. Sindbis virus tropism and virus-induced cytopathology is tissue specific in *Aedes albopictus*. Submitted for publication.

44. Braakman, I., J. Helenius, and A. Helenius. 1992. Role of ATP and disulphide bonds during protein folding in the endoplasmic reticulum. Nature (London) 356:260–262.

45. Bredenbeek, P. J., I. Frolov, C. M. Rice, and S. Schlesinger. 1993. Sindbis virus expression vectors: packaging of RNA replicons by using defective helper RNAs. J. Virol. 67:6439–6446.

46. Bredenbeek, P. J., and C. M. Rice. 1992. Animal RNA virus expression systems. Semin. Virol. 3:297–310.

47. Bron, R., J. M. Wahlberg, H. Garoff, and J. Wilschut. 1993. Membrane fusion of Semliki Forest virus in a model system: correlation between fusion kinetics and structural changes in the envelope glycoprotein. EMBO J. 12:693–701.

48. Brown, D. T. 1994. Personal communication.

49. Brown, D. T., and L. D. Condreay. 1986. Replication of alphaviruses in mosquito cells, p. 171–203. *In* S. Schlesinger and M. J. Schlesinger (ed.), The Togaviridae and Flaviviridae. Plenum Publishing Corp., New York.

50. Brown, D. T., and J. Edwards. 1992. Structural changes in alphaviruses accompanying the process of membrane penetration. Semin. Virol. 3:519–527.
51. Brown, D. T., and J. B. Gleidman. 1973. Morphological variants of Sindbis virus obtained from infected mosquito tissue culture cells. J. Virol. 12:1535–1539.
52. Brown, D. T., and J. F. Smith. 1975. Morphology of BHK-21 cells infected with Sindbis virus temperature sensitive mutants in complementation groups D and E. J. Virol. 15:1262–1266.
53. Burness, A. T., I. Pardoe, S. G. Faragher, S. Vrati, and L. Dalgarno. 1988. Genetic stability of Ross River virus during epidemic spread in nonimmune humans. Virology 167:639–643.
54. Buzan, J. M., and S. Schlesinger. 1992. Expression of the nonstructural proteins of Sindbis virus in insect cells by a baculovirus vector. Virus Res. 23:209–222.
55. Calisher, C. H., and N. Karabatsos. 1988. Arbovirus serogroups: definition and geographic distribution, p. 19–57. In T. P. Monath (ed.), The arboviruses: epidemiology and ecology. CRC Press, Inc., Baca Raton, Fla.
56. Calisher, C. H., N. Karabatsos, J. S. Lazuick, T. Monath, and K. L. Wolff. 1988. Reevaluation of the Western equine encephalitis antigenic complex of alphaviruses (family Togaviridae) as determined by neutralization tests. Am. J. Trop. Med. Hyg. 38:447–452.
57. Calisher, C. H., K. S. C. Maness, R. D. Lord, and P. H. Coleman. 1971. Identification of two South American strains of Eastern equine encephalomyelitis virus from migrant birds captured on the Mississippi delta. Am. J. Epidemiol. 94:172–178.
58. Calzone, F. J., N. Theze, P. Thiebaud, R. L. Hill, R. J. Britten, and E. D. Davidson. 1988. Developmental appearance of factors that bind specifically to *cis*-regulatory sequences of a gene expressed in the sea urchin embryo. Genes Dev. 2:1074–1088.
59. Canciedda, R., S. Bonatti, and A. Leone. 1981. One extra oligosaccharide chain of the high-mannose class in the E2 protein of a Sindbis virus isolate. J. Virol. 38:8–14.
60. Carrasco, L. 1977. The inhibition of cell functions after viral infection: a proposed general mechanism. FEBS Lett. 76:11–15.
61. Carrasco, L., M. J. Otero, and J. L. Castrillo. 1988. Modification of membrane permeability by animal viruses. Pharmacol. Ther. 40:171–212.
62. Casals, J. 1964. Antigenic variants of Eastern equine encephalitis virus. J. Exp. Med. 119:547–565.
63. Cassell, S., J. Edwards, and D. T. Brown. 1984. Effects of lysosomotropic weak bases on the infection of BHK-21 cells by Sindbis virus. J. Virol. 52:857–864.
64. Castronovo, V., A. P. Claysmith, K. T. Barker, V. Cioce, H. C. Krutzsch, and M. E. Sobel. 1991. Biosynthesis of the 67 kDa high affinity laminin receptor. Biochem. Biophys. Res. Commun. 177:177–183.
65. Castronovo, V., G. Taraboletti, and M. E. Sobel. 1991. Functional domains of the 67-kDa laminin receptor precursor. J. Biol. Chem. 266:20440–20446.
66. Chamberlain, R. W. 1980. Epidemiology of arthropod-borne togaviruses: the role of arthropods as hosts and vectors and of vertebrate hosts in natural transmission cycles, p. 175–227. In R. W. Schlesinger (ed.), The togaviruses: biology, structure, replication. Academic Press, Inc., New York.
67. Chanas, A. C., E. A. Gould, J. C. A. Clegg, and M. G. R. Varma. 1982. Monoclonal antibodies to Sindbis virus glycoprotein E1 can neutralize, enhance infectivity, and independently inhibit haemagglutination or haemolysis. J. Gen. Virol. 58:37–46.
68. Chang, G.-J. J., and D. W. Trent. 1987. Nucleotide sequence of the genome region encoding the 26S mRNA of Eastern equine encephalitis virus and the deduced amino acid sequence of the viral structural proteins. J. Gen. Virol. 68:2129–2142.
69. Chau, V., J. W. Tobias, A. Bachmair, D. Marriott, D. J. Ecker, D. K. Gonda, and A. Varshavsky. 1989. A multiubiquitin chain is confined to specific lysine in a targeted short-lived protein. Science 243:1576–1583.
70. Cheng, R. H., R. J. Kuhn, N. H. Olson, M. G. Rossmann, H.-K. Choi, and T. S. Baker. Three-dimensional structure of an enveloped alphavirus with T=4 icosahedral symmetry. Submitted for publication.

71. Choi, H.-K., L. Tong, W. Minor, P. Dumas, U. Boege, M. G. Rossmann, and G. Wengler. 1991. Structure of Sindbis virus core protein reveals a chymotrypsin-like serine proteinase and the organization of the virion. Nature (London) 354:37–43.
72. Clewley, J. P., and S. I. T. Kennedy. 1976. Purification and polypeptide composition of Semliki Forest virus RNA polymerase. J. Gen. Virol. 32:395–411.
73. Collier, N. C., S. P. Adams, H. Weingarten, and M. J. Schlesinger. 1992. Inhibition of enveloped RNA virus formation by peptides corresponding to glycoprotein sequences. Antiviral Chem. Chemother. 3:31–36.
74. Condrey, L. D., and D. T. Brown. 1986. Exclusion of superinfecting homologous virus by Sindbis virus-infected *Aedes albopictus* (mosquito) cells. J. Virol. 58:81–86.
75. Condrey, L. D., and D. T. Brown. 1988. Suppression of RNA synthesis by a specific antiviral activity in Sindbis virus-infected *Aedes albopictus* cells. J. Virol. 62:346–348.
76. Coombs, K., B. Brown, and D. T. Brown. 1984. Evidence for a change in capsid morphology during Sindbis virus envelopment. Virus Res. 1:297–302.
77. Coombs, K., and D. T. Brown. 1987. Topological organization of Sindbis virus capsid protein in isolated nucleocapsids. Virus Res. 7:131–149.
78. Coombs, K., E. Mann, J. Edwards, and D. T. Brown. 1981. Effects of chloroquine and cytochalasin B on the infection of cells by Sindbis virus and vesicular stomatitis virus. J. Virol. 37:1060–1065.
79. Coombs, K. M., and D. T. Brown. 1989. Form-determining functions in Sindbis virus nucleocapsids: nucleosomelike organization of the nucleocapsid. J. Virol. 63:883–891.
80. Cross, R. K. 1983. Identification of a unique guanine-7-methyltransferase in Semliki Forest virus (SFV) infected cell extracts. Virology 130:452–463.
81. Cross, R. K., and P. J. Gomatos. 1981. Concomitant methylation and synthesis *in vitro* of Semliki Forest virus (SFV) ssRNAs by a fraction from infected cells. Virology 114:542–554.
82. Cutler, D. F., and H. Garoff. 1986. Mutants of the membrane-binding region of Semliki Forest virus E2 protein. I. Cell surface transport and fusogenic activity. J. Cell Biol. 102:889–901.
83. Cutler, D. F., P. Melancon, and H. Garoff. 1986. Mutants of the membrane-binding region of Semliki Forest virus E2 protein. II. Topology and membrane binding. J. Cell Biol. 102:902–910.
84. Dalgarno, L., C. M. Rice, and J. H. Strauss. 1983. Ross River virus 26S RNA: complete nucleotide sequence and deduced sequence of the encoded structural proteins. Virology 129:170–187.
85. Dalrymple, J. D. 1992. Personal communication.
86. Davey, M. W., and L. Dalgarno. 1974. Semliki Forest virus replication in cultured *Aedes albopictus* cells: studies on the establishment of persistence. J. Gen. Virol. 24:453–463.
87. Davis, N. L., F. J. Fuller, W. G. Dougherty, R. A. Olmsted, and R. E. Johnston. 1986. A single nucleotide change in the E2 glycoprotein gene of Sindbis virus affects penetration rate in cell culture and virulence in neonatal mice. Proc. Natl. Acad. Sci. USA 83:6771–6775.
88. Davis, N. L., D. F. Pence, W. J. Meyer, A. L. Schmaljohn, and R. E. Johnston. 1987. Alternative forms of a strain-specific neutralizing antigenic site on the Sindbis virus E2 glycoprotein. Virology 161:101–108.
89. Davis, N. L., N. Powell, G. F. Greenwald, L. V. Willis, B. J. B. Johnson, J. Smith, and R. E. Johnston. 1991. Attenuating mutations in the E2 glycoprotein gene of Venezuelan equine encephalitis virus: construction of single and multiple mutations in a full-length cDNA clone. Virology 183:20–31.
90. Davis, N. L., L. V. Willis, J. F. Smith, and R. E. Johnston. 1989. *In vitro* synthesis of infectious Venezuelan equine encephalitis virus RNA from a cDNA clone: analysis of a viable deletion mutant. Virology 171:189–204.
91. de Curtis, I., and K. Simons. 1988. Dissection of Semliki Forest virus glycoprotein delivery from the trans-Golgi network to the cell surface in permeabilized BHK cells. Proc. Natl. Acad. Sci. USA 85:8052–8056.
92. de Groot, R. J., W. R. Hardy, Y. Shirako, and J. H. Strauss. 1990.

Cleavage-site preferences of Sindbis virus polyproteins containing the nonstructural proteinase: evidence for temporal regulation of polyprotein processing in vivo. EMBO J. 9:2631–2638.

93. de Groot, R. J., T. Rümenapf, R. J. Kuhn, E. G. Strauss, and J. H. Strauss. 1991. Sindbis virus RNA polymerase is degraded by the N-end rule pathway. Proc. Natl. Acad. Sci. USA 88:8967–8971.

94. Diamond, D. C., B. A. Jameson, J. Bonin, M. Kohara, S. Abe, H. Itoh, T. Komatsu, M. Arita, S. Kuge, A. Nomoto, A. D. M. E. Osterhaus, R. Crainic, and E. Wimmer. 1985. Antigenic variation and resistance to neutralization in poliovirus type 1. Science 229:1090–1093.

95. Dickerman, R. W., M. S. Martin, and E. A. Dipaola. 1980. Studies of Venezuelan encephalitis in migrating birds in relation to possible transport of virus from South to Central America. Am. J. Trop. Med. Hyg. 29:269–276.

96. Ding, M., and M. J. Schlesinger. 1989. Evidence that Sindbis virus nsP2 is an autoprotease which processes the virus nonstructural polyprotein. Virology 171:280–284.

97. Docherty, K., C. J. Rhodes, N. A. Taylor, K. I. J. Shennan, and J. C. Hutton. 1989. Proinsulin endopeptidase substrate specificities defined by site-directed mutagenesis of proinsulin. J. Biol. Chem. 264:18335–18339.

98. Dominguez, G., C.-Y. Wang, and T. K. Frey. 1990. Sequence of the genome RNA of rubella virus: evidence for genetic rearrangement during togavirus evolution. Virology 177:225–238.

99. Doms, R. W., R. A. Lamb, J. K. Rose, and A. Helenius. 1993. Folding and assembly of viral membrane glycoproteins. Virology 193:545–562.

100. Dotti, C. G., J. Kartenbeck, and K. Simons. 1993. Polarized distribution of the viral glycoproteins of vesicular stomatitis, fowl plague and Semliki Forest viruses in hippocampal neurons in culture: a light and electron microscopy study. Brain Res. 610:141–147.

101. Doxsey, S. J., F. M. Brodsky, G. S. Blank, and A. Helenius. 1987. Inhibition of endocytosis by anti-clathrin antibodies. Cell 50:453–463.

102. Dubuisson, J., and C. M. Rice. 1993. Sindbis virus attachment: isolation and characterization of mutants with impaired binding to vertebrate cells. J. Virol. 67:3363–3374.

103. Durbin, R., A. Kane, and V. Stollar. 1991. A mutant of Sindbis virus with altered plaque morphology and a decreased ratio of 26S:49S RNA synthesis in mosquito cells. Virology 183:306–312.

104. Durbin, R. K., E. deClercq, and V. Stollar. 1988. SVLM21, a mutant of Sindbis virus able to grow in *Aedes albopictus* cells in the absence of methionine, shows increased sensitivity to S-adenosylhomocysteine hydrolase inhibitors such as neplanocin. Virology 163:218–221.

105. Durbin, R. K., and V. Stollar. 1984. A mutant of Sindbis virus with a host-dependent defect in maturation associated with hyperglycosylation of E2. Virology 135:331–344.

106. Durbin, R. K., and V. Stollar. 1985. Sindbis virus mutants able to replicate in methionine-deprived *Aedes albopictus* cells. Virology 144:529–533.

107. Durbin, R. K., and V. Stollar. 1986. Sequence analysis of the E2 gene of a hyperglycosylated, host restricted mutant of Sindbis virus and estimation of mutation rate from frequency of revertants. Virology 154:135–143.

108. Edwards, J., and D. T. Brown. 1984. Sindbis virus induced fusion of tissue cultured *Aedes albopictus* (mosquito) cells. Virus Res. 1:705–711.

109. Edwards, J., and D. T. Brown. 1991. Sindbis virus infection of a Chinese hamster ovary cell mutant defective in the acidification of endosomes. Virology 182:28–33.

110. Edwards, J., E. Mann, and D. T. Brown. 1983. Conformational changes in Sindbis virus envelope proteins accompanying exposure to low pH. J. Virol. 45:1090–1097.

111. Ehrenfeld, E. (ed.). 1993. Translational regulation in virus-infected cells. Semin. Virol. 4:199–268.

112. Elgizoli, M., Y. Dai, C. Kempf, H. Koblet, and M. R. Michel. 1989. Semliki Forest virus capsid protein acts as a pleiotropic regulator of host cellular protein synthesis. J. Virol. 63:2921–2928.

113. Erwin, C., and D. T. Brown. 1983. Requirement of cell nucleus

for Sindbis virus replication in cultured *Aedes albopictus* (mosquito) cells. J. Virol. 45:792–799.

114. Fan, D. P., and B. M. Sefton. 1978. The entry into host cells of Sindbis virus, vesicular stomatitis virus and Sendai virus. Cell 15:985–992.

115. Faragher, S. G., and L. Dalgarno. 1986. Regions of conservation and divergence in the 3′ untranslated sequences of genomic RNA from Ross River virus isolates. J. Mol. Biol. 190:141–148.

115a. Faragher, S. G., I. D. Marshall, and L. Dalgarno. 1985. Ross River genetic variants in Australia and the Pacific islands. Aust. J. Exp. Biol. Med. Sci. 63:473–488.

116. Faragher, S. G., A. D. J. Meek, C. M. Rice, and L. Dalgarno. 1988. Genome sequences of a mouse-avirulent and a mouse-virulent strain of Ross River virus. Virology 163:509–526.

117. Fazakerley, J. K., S. Pathak, M. Scallan, S. Amor, and H. Dyson. 1993. Replication of the A7(74) strain of Semliki Forest virus is restricted in neurons. Virology 195:627–637.

118. Feener, E. P., W.-C. Shen, and H. J.-P. Ryser. 1990. Cleavage of disulfide bonds in endocytosed macromolecules. J. Biol. Chem. 265:18780–18785.

119. Feng, D.-F., and R. F. Doolittle. 1987. Progressive sequence alignment as a prerequisite to correct phylogenetic trees. J. Mol. Evol. 25:351–360.

120. Feng, Y.-X., T. D. Copeland, S. Oroszlan, A. Rein, and J. G. Levin. 1990. Identification of amino acids inserted during suppression of UAA and UGA termination codons at the gag-pol junction of Moloney murine leukemia virus. Proc. Natl. Acad. Sci. USA 87:8860–8863.

121. Flynn, D. C., W. J. Meyer, J. M. MacKenzie, and R. C. Johnston. 1990. A conformational change in Sindbis glycoprotein E1 and E2 is detected at the plasma membrane as a consequence of early virus-cell interaction. J. Virol. 64:3643–3653.

122. Flynn, D. C., R. A. Olmsted, J. J. Mackenzie, and R. E. Johnston. 1988. Antibody-mediated activation of Sindbis virus. Virology 166:82–90.

123. Frey, T. K. Molecular biology of rubella virus. Adv. Virus Res., in press.

124. Friedman, R. M., J. G. Levin, P. M. Grimley, and I. K. Berezesky. 1972. Membrane associated replication complex in arbovirus infection. J. Virol. 10:504–515.

125. Fries, E., and A. Helenius. 1979. Binding of Semliki Forest virus and its spike glycoprotein to cells. Eur. J. Biochem. 97:213–220.

126. Frolov, I., and S. Schlesinger. 1994. A comparison of the effects of Sindbis virus and Sindbis virus replicons on host cell protein synthesis and cytopathogenicity in BHK cells. J. Virol. 68:1721–1727.

127. Froshauer, S., J. Kartenbeck, and A. Helenius. 1988. Alphavirus RNA replicase is located on the cytoplasmic surface of endosomes and lysosomes. J. Cell Biol. 107:2075–2086.

128. Fulhorst, C. F., J. L. Hardy, B. F. Eldridge, S. B. Presser, and W. C. Reeves. 1994. Natural vertical transmission of Western equine encephalomyelitis virus in mosquitoes. Science 263:676–678.

129. Fuller, R. S., R. E. Sterne, and J. Thorner. 1988. Enzymes required for yeast prohormone processing. Annu. Rev. Physiol. 50:345–362.

130. Fuller, S. D. 1987. The T=4 envelope of Sindbis virus is organized by interactions with a complementary T=3 capsid. Cell 48:923–934.

131. Fuller, S. D., R. Bravo, and K. Simons. 1985. An enzymatic assay reveals that proteins destined for the apical or basolateral domains of an epithelial cell share the same late Golgi compartment. EMBO J. 4:297–307.

132. Gaedigk-Nitschko, K., M. Ding, and M. J. Schlesinger. 1990. Site-directed mutations in the Sindbis virus 6K protein reveal sites for fatty acylation and the underacylated protein affects virus release and virion structure. Virology 175:282–291.

133. Gaedigk-Nitschko, K., and M. J. Schlesinger. 1990. The Sindbis virus 6K protein can be detected in virions and is acylated with fatty acid. Virology 175:274–281.

134. Gaedigk-Nitschko, K., and M. J. Schlesinger. 1991. Site-directed mutations in Sindbis virus E2 glycoprotein’s cytoplasmic domain and the 6K protein lead to similar defects in virus assembly and

budding. Virology 183:206–214.

135. Gahmberg, C. G., G. Utermann, and K. Simons. 1972. The membrane proteins of Semliki Forest virus have a hydrophobic part attached to the viral membrane. FEBS Lett. 28:179–182.

136. Garavito, R. M., M. G. Rossmann, P. Argos, and W. Eventoff. 1977. Convergence of active site geometries. Biochemistry 16: 5065–5071.

137. Garoff, H., A. M. Frischauf, K. Simons, H. Lehrach, and H. Delius. 1980. The capsid protein of Semliki Forest virus has clusters of basic amino acids and prolines in its amino-terminal region. Proc. Natl. Acad. Sci. USA 77:6376–6380.

138. Garoff, H., A. M. Frischauf, K. Simons, H. Lehrach, and H. Delius. 1980. Nucleotide sequence of cDNA coding for Semliki Forest virus membrane glycoproteins. Nature (London) 288:236–241.

139. Garoff, H., D. Huylebroeck, A. Robinson, U. Tillman, and P. Liljeström. 1990. The signal sequence of the p62 protein of Semliki Forest virus is involved in initiation but not in completing chain translocation. J. Cell Biol. 111:867–876.

140. Garoff, H., C. Kondor-Koch, R. Petterson, and B. Burke. 1983. Expression of Semliki Forest virus proteins from cloned complementary DNA. II. The membrane-spanning glycoprotein E2 is transported to the cell surface without its normal cytoplasmic domain. J. Cell Biol. 97:652.

141. Garoff, H., and K. Simons. 1974. Location of the spike glycoproteins in the Semliki Forest virus membrane. Proc. Natl. Acad. Sci. USA 71:3988–3992.

142. Garoff, H., K. Simons, and B. Dobberstein. 1978. Assembly of the Semliki Forest virus membrane glycoproteins in the membrane of the endoplasmic reticulum in vitro. J. Mol. Biol. 124:587–600.

143. Garry, R. F. 1994. Sindbis virus-induced inhibition of protein synthesis is partially reversed by medium containing an elevated potassium concentration. J. Gen. Virol. 75:411–415.

144. Garry, R. F., J. M. Bishop, S. Parker, K. Westbrook, G. Lewis, and M. R. F. Waite. 1979. Na⁺ and K⁺ concentrations and the regulation of protein synthesis in Sindbis virus-infected chick cells. Virology 96:108–120.

145. Garry, R. F., and D. A. Bostick. 1987. Induction of the stress response: alterations in membrane associated transport systems and protein modification in heat shocked or Sindbis virus-infected cells. Virus Res. 8:245–259.

146. Garry, R. F., K. Westbrook, and M. R. F. Waite. 1979. Differential effects of ouabain on host- and Sindbis virus-specific protein synthesis. Virology 99:179–182.

147. Gaulton, G. N., and M. I. Greene. 1986. Idiotypic mimicry of biological receptors. Annu. Rev. Immun. 4:253–280.

148. Geigenmüller-Gnirke, U., H. Nitschko, and S. Schlesinger. 1993. Deletion analysis of the capsid protein of Sindbis virus: identification of the RNA binding region. J. Virol. 67:1620–1626.

149. Geigenmüller-Gnirke, U., B. Weiss, R. Wright, and S. Schlesinger. 1991. Complementation between Sindbis viral RNAs produces infectious particles with a bipartite genome. Proc. Natl. Acad. Sci. USA 88:3253–3257.

150. Gidwtz, S., J. M. Polo, N. L. Davis, and R. E. Johnston. 1988. Differences in virion stability among Sindbis virus pathogenesis mutants. Virus Res. 10:225–239.

151. Glanville, N., and J. Ulmanen. 1976. Biological activity of in vitro synthesized protein: binding of Semliki Forest virus capsid protein to the large ribosomal subunit. Biochem. Biophys. Res. Commun. 71:393–399.

152. Goldbach, R., O. Le Gall, and J. Wellink. 1991. Alpha-like viruses in plants. Semin. Virol. 2:19–25.

153. Gomatos, P. J., L. Kääriäinen, S. Keränen, M. Ranki, and D. L. Sawicki. 1980. Semliki Forest virus replication complex capable of synthesizing 42S and 26S nascent RNA chains. J. Gen. Virol. 49:61–69.

154. Gonda, D. K., A. Bachmair, I. Wünnig, J. W. Tobias, W. S. Lane, and A. Varshavsky. 1989. Universality and structure of the N-end rule. J. Biol. Chem. 264:16700–16712.

155. Gorbelenya, A. E., V. M. Blinov, A. P. Donchenko, and E. V. Koonin. 1989. An NTP-binding motif is the most conserved sequence in a highly diverged monophyletic group of proteins involved in positive strand RNA virus replication. J. Mol. Evol. 28:256–268.


156. Gorbelenya, A. E., A. P. Donchenko, E. V. Koonin, and V. M. Blinov. 1989. N-terminal domains of putative helicases of flaviviruses may be serine proteases. Nucleic Acids Res. 17:3889–3897.

157. Gorbelenya, A. E., E. V. Koonin, A. P. Donchenko, and V. M. Blinov. 1988. A conserved NTP-binding motif in putative helicases. Nature (London) 333:22.

158. Gorbelenya, A. E., E. V. Koonin, A. P. Donchenko, and V. M. Blinov. 1988. A novel superfamily of nucleoside triphosphate-binding motif containing proteins which are probably involved in duplex unwinding in DNA and RNA replication and recombination. FEBS Lett. 235:16–24.

159. Gorbelenya, A. E., E. V. Koonin, A. P. Donchenko, and V. M. Blinov. 1989. Two related superfamilies of putative helicases involved in replication, repair, and expression of DNA and RNA genomes. Nucleic Acids Res. 17:4713–4730.

160. Gorbelenya, A. E., E. V. Koonin, and M. M. C. Lai. 1991. Putative papain-related thiol proteases of positive-strand RNA viruses. FEBS Lett. 288:201–205.

161. Gorbelenya, A. E., E. V. Koonin, and Y. I. Wolf. 1990. A new superfamily of putative NTP-binding domains encoded by genomes of small DNA and RNA viruses. FEBS Lett. 262:145–148.

162. Grakoui, A., R. Levis, R. Raju, H. V. Huang, and C. M. Rice. 1989. A cis-acting mutation in the Sindbis virus junction region which affects subgenomic RNA synthesis. J. Virol. 63:5216–5227.

163. Gray, M. A., S. A. Austin, M. J. Clemens, L. Rodrigues, and C. A. Pasternak. 1983. Protein synthesis in Semliki Forest virus-infected cells is not controlled by permeability changes. J. Gen. Virol. 64:2361–2640.

164. Gray, M. A., K. J. Micklem, and C. A. Pasternak. 1983. Effect of vesicular stomatitis virus and Semliki Forest virus on uptake of nutrients and intracellular cation concentration. J. Gen. Virol. 64:1449–1456.

165. Gray, M. A., K. J. Micklem, and C. A. Pasternak. 1983. Protein synthesis in cells infected with Semliki Forest virus is not controlled by intracellular cation changes. Eur. J. Biochem. 135:299–302.

166. Griffin, D. E. 1986. Alphavirus pathogenesis and immunity, p. 209–250. In S. Schlesinger and M. J. Schlesinger (ed.), The Togaviridae and Flaviviridae. Plenum Publishing Corp., New York.

167. Griffin, D. E. 1989. Molecular pathogenesis of Sindbis virus encephalitis in experimental animals. Adv. Virus Res. 36:255–271.

168. Griffin, D. E., C. S. Hahn, A. C. Jackson, S. Lustig, E. G. Strauss, and J. H. Strauss. 1989. The basis of Sindbis virus neurovirulence. UCLA Symp. Mol. Cell. Biol. New Ser. 90:387–396.

169. Griffin, D. E., and R. Johnson. 1977. Role of the immune response in recovery from Sindbis virus encephalitis in mice. J. Immunol. 118:1070–1075.

170. Griffiths, G., P. Quinn, and G. Warren. 1983. Dissection of the Golgi complex. I. Monensin inhibits the transport of viral membrane proteins from medial to trans Golgi cisternae in baby hamster kidney cells infected with Semliki Forest virus. J. Cell Biol. 96:835–850.

171. Grimley, P. M., I. K. Berezesky, and R. M. Friedman. 1968. Cytoplasmic structures associated with an arbovirus infection: loci of viral ribonucleic acid synthesis. J. Virol. 2:1326–1338.

172. Grimley, P. M., J. G. Levin, I. K. Berezesky, and R. M. Friedman. 1972. Specific membranous structures associated with the replication of group A arboviruses. J. Virol. 10:492–503.

173. Grosfeld, H., B. Velan, M. Leitner, S. Cohen, S. Lustig, B. E. Lachmi, and A. Shafferman. 1989. Semliki Forest virus E2 envelope epitopes induce a nonneutralizing humoral response which protects mice against lethal challenge. J. Virol. 63:3416–3422.

174. Grosso, L. E., P. W. Park, and R. P. Mecham. 1991. Characterization of a putative clone for the 67-kilodalton elastin/laminin receptor suggests that it encodes a cytoplasmic protein rather than a cell surface receptor. Biochemistry 30:3346–3350.

175. Hahn, C. S. 1993. Personal communication.

176. Hahn, C. S., Y. S. Hahn, T. J. Braciale, and C. M. Rice. 1992. Infectious Sindbis virus transient expression vectors for studying

antigen processing and presentation. Proc. Natl. Acad. Sci. USA 89:2679–2683.

177. Hahn, C. S., S. Lustig, E. G. Strauss, and J. H. Strauss. 1988. Western equine encephalitis virus is a recombinant virus. Proc. Natl. Acad. Sci. USA 85:5997–6001.

178. Hahn, C. S., C. M. Rice, E. G. Strauss, E. M. Lenches, and J. H. Strauss. 1989. Sindbis virus ts103 has a mutation in glycoprotein E2 that leads to defective assembly of virions. J. Virol. 63:3459–3465.

179. Hahn, C. S., E. G. Strauss, and J. H. Strauss. 1985. Sequence analysis of three Sindbis virus mutants temperature-sensitive in the capsid autoprotease. Proc. Natl. Acad. Sci. USA 82:4648–4652.

180. Hahn, C. S., and J. H. Strauss. 1990. Site-directed mutagenesis of the proposed catalytic amino acids of the Sindbis virus capsid protein autoprotease. J. Virol. 64:3069–3073.

181. Hahn, Y. S., A. Grakoui, C. M. Rice, E. G. Strauss, and J. H. Strauss. 1989. Mapping of RNA⁻ temperature-sensitive mutants of Sindbis virus: complementation group F mutants have lesions in nsP4. J. Virol. 63:1194–1202.

182. Hahn, Y. S., C. S. Hahn, V. L. Braciale, T. J. Braciale, and C. M. Rice. 1992. CD8⁺ T cell recognition of an endogenously processed epitope is regulated primarily by residues within the epitope. J. Exp. Med. 176:1335–1341.

183. Hahn, Y. S., E. G. Strauss, and J. H. Strauss. 1989. Mapping of RNA⁻ temperature-sensitive mutants of Sindbis virus: assignment of complementation groups A, B, and G to nonstructural proteins. J. Virol. 63:3142–3150.

184. Hanson, R. P., S. E. Sulkin, E. L. Buescher, W. M. Hammon, R. W. McKinney, and T. H. Work. 1967. Arbovirus infections of laboratory workers. Science 158:1283–1286.

185. Hardy, J. L. 1988. Susceptibility and resistance of vector mosquitoes, p. 87–126. In T. P. Monath (ed.), The arboviruses: epidemiology and ecology. CRC Press, Inc., Boca Raton, Fla.

186. Hardy, W. R., Y. S. Hahn, R. J. de Groot, E. G. Strauss, and J. H. Strauss. 1990. Synthesis and processing of the nonstructural polyproteins of several temperature-sensitive mutants of Sindbis virus. Virology 177:199–208.

187. Hardy, W. R., and J. H. Strauss. 1988. Processing of the nonstructural polyproteins of Sindbis virus: study of the kinetics in vivo using monospecific antibodies. J. Virol. 62:998–1007.

188. Hardy, W. R., and J. H. Strauss. 1989. Processing the nonstructural polyproteins of Sindbis virus: nonstructural proteinase is in the C-terminal half of nsP2 and functions both in cis and in trans. J. Virol. 63:4653–4664.

189. Harrison, S. C. 1986. Alphavirus structure, p. 21–34. In S. Schlesinger and M. J. Schlesinger (ed.), The Togaviridae and Flaviviridae. Plenum Publishing Corp., New York.

190. Harrison, S. C., A. David, J. Jumblatt, and J. E. Darnell. 1971. Lipid and protein organization in Sindbis virus. J. Mol. Biol. 60:532.

191. Harrison, S. C., S. Schlesinger, M. J. Schlesinger, and R. K. Strong. 1992. Crystallization of Sindbis virus and its nucleocapsid. J. Mol. Biol. 226:277–280.

192. Haseloff, J., P. Goelet, D. Zimmern, P. Ahlquist, R. Dasgupta, and P. Kaesberg. 1984. Striking similarities in amino acid sequence among nonstructural proteins encoded by RNA viruses that have dissimilar genomic organization. Proc. Natl. Acad. Sci. USA 81:4358–4362.

193. Hashimoto, K., S. Erdei, S. Keränen, J. Saraste, and L. Kääriäinen. 1981. Evidence for a separate signal sequence for the carboxy-terminal envelope glycoprotein E1 of Semliki Forest virus. J. Virol. 38:34–40.

194. Heidner, H. W., K. L. McKnight, N. L. Davis, and R. E. Johnston. 1994. Lethality of PE2 incorporation into Sindbis virus can be suppressed by second-site mutations in E3 and E2. J. Virol. 68:2683–2692.

195. Helenius, A. 1984. Semliki Forest virus penetration from endosomes. Biol. Cell. 51:181–186.

196. Helenius, A., and J. Kartenbeck. 1980. The effects of octylglucoside on the Semliki Forest virus membrane: evidence for a spike-protein-nucleocapsid interaction. Eur. J. Biochem. 106:613–618.

197. Helenius, A., J. Kartenbeck, K. Simons, and E. Fries. 1980. On the entry of Semliki Forest virus into BHK-21 cells. J. Cell Biol. 84:404–420.

198. Helenius, A., M. Kielian, J. Wellstead, I. Mellman, and G. Rudnick. 1985. Effects of monovalent cations on Semliki Forest virus entry into BHK-21 cells. J. Biol. Chem. 260:5691–5697.

199. Helenius, A., M. Marsh, and J. White. 1982. Inhibition of Semliki Forest virus penetration by lysosomotropic weak bases. J. Gen. Virol. 58:47–61.

200. Helenius, A., B. Morrein, E. Fries, K. Simons, P. Robinson, V. Schirrmacher, C. Terhorst, and J. L. Strominger. 1978. Human (HLA-A and -B) and murine (H2-K and -D) histocompatibility antigens are cell surface receptors for Semliki Forest virus. Proc. Natl. Acad. Sci. USA 75:3846–3850.

201. Hertz, J. M., and H. Huang. 1992. Utilization of heterologous alphavirus junction sequences as promoters by Sindbis virus. J. Virol. 66:857–864.

202. Higaki, J. N., B. W. Gibson, and C. S. Craik. 1987. Evolution of catalysis in the serine proteases. Cold Spring Harbor Symp. Quant. Biol. 52:615–621.

203. Hirsch, R. L., D. E. Griffin, and J. A. Winkelstein. 1978. The effect of complement depletion on the course of Sindbis virus infection in mice. J. Immunol. 121:1276–1278.

204. Hirsch, R. L., D. E. Griffin, and J. A. Winkelstein. 1980. The role of complement in viral infections. II. The clearance of Sindbis virus from the bloodstream and central nervous system of mice depleted of complement. J. Infect. Dis. 141:212–217.

205. Hirsch, R. L., D. E. Griffin, and J. A. Winkelstein. 1980. Role of complement in viral infections: participation of terminal complement components (C5 to C9) in recovery of mice from Sindbis virus infection. Infect. Immun. 30:899–901.

206. Hirsch, R. L., D. E. Griffin, and J. A. Winkelstein. 1981. Host modification of Sindbis virus sialic acid content influences alternative complement pathway activation and virus clearance. J. Immunol. 127:1740–1743.

207. Hirsch, R. L., D. E. Griffin, and J. A. Winkelstein. 1983. Natural immunity to Sindbis virus is influenced by host tissue sialic acid content. Proc. Natl. Acad. Sci. USA 80:548–550.

208. Hirsch, R. L., J. A. Winkelstein, and D. E. Griffin. 1980. The role of complement in viral infections. III. Activation of the classical and alternative pathways by Sindbis virus. J. Immunol. 124:2507–2510.

209. Hodgman, T. C. 1988. A new superfamily of replicative proteins. Nature (London) 333:22–23.

210. Hoekstra, D., and J. W. Kok. 1989. Entry mechanisms of enveloped viruses. Implications for fusion of intracellular membranes. Biosci. Rep. 9:273–305.

211. Holland, J., K. Spindler, F. Horodyski, E. Grabau, S. Nichol, and S. VandePol. 1982. Rapid evolution of RNA genomes. Science 215:1577–1585.

212. Holland, J. J., J. C. De La Torre, and D. A. Steinhauer. 1992. RNA virus populations as quasispecies. Curr. Top. Microbiol. Immunol. 176:1–20.

213. Houk, E. J., Y. M. Arcus, J. L. Hardy, and L. D. Kramer. 1990. Binding of Western equine encephalomyelitis virus to brush border fragments isolated from mesenteronal epithelial cells of mosquitoes. Virus Res. 17:105–118.

214. Hsieh, P., M. R. Rosner, and P. W. Robbins. 1983. Host-dependent variation of asparagine-linked oligosaccharides at individual glycosylation sites of Sindbis virion envelope glycoproteins. J. Biol. Chem. 258:2548.

215. Hsieh, P., M. R. Rosner, and P. W. Robbins. 1983. Selective cleavage by endo-β-N-acetylglucosaminidase H at individual glycosylation sites of Sindbis virion envelope glycoproteins. J. Biol. Chem. 258:2555.

216. HsuChen, C.-C., and D. T. Dubin. 1976. Di- and trimethylated congeners of 7-methylguanine in Sindbis virus mRNA. Nature (London) 264:190–191.

217. Huang, H. V., C. M. Rice, C. Xiong, and S. Schlesinger. 1989. RNA viruses as gene expression vectors. Virus Genes 3:85–91.

218. Hunt, A. R., A. J. Johnson, and J. T. Roehrig. 1990. Synthetic peptides of Venezuelan equine encephalomyelitis virus E2 glycoprotein. I. Immunogenic analysis and identification of a pro-

219. Hunt, A. R., and J. T. Roehrig. 1985. Biochemical and biological characteristics of epitopes on the E1 glycoprotein of Western equine encephalitis virus. Virology 142:334–346.
220. Huth, A., T. A. Rapoport, and L. Kääriäinen. 1984. Envelope proteins of Semliki Forest virus synthesized in Xenopus oocytes are transported to the cell surface. EMBO J. 3:767–771.
221. Ivanova, L., and M. J. Schlesinger. 1993. Site-directed mutations in the Sindbis virus E2 glycoprotein identify palmitoylation sites and affect virus budding. J. Virol. 67:2546–2551.
222. Jackson, A. C., T. R. Moench, D. E. Griffin, and R. T. Johnson. 1988. The pathogenesis of spinal cord involvement in the encephalomyelitis of mice caused by neuroadapted Sindbis virus infection. Lab. Invest. 56:418–423.
223. Jackson, A. C., T. R. Moench, B. D. Trapp, and D. E. Griffin. 1988. Basis of neurovirulence in Sindbis virus encephalomyelitis of mice. Lab. Invest. 58:503–509.
224. Jain, S. K., S. DeCandido, and M. Kielian. 1991. Processing of the p62 envelope precursor protein of Semliki Forest virus. J. Biol. Chem. 266:5756–5761.
225. Jalanko, A. 1985. Expression of Semliki Forest virus capsid protein from SV40 recombinant virus. FEBS Lett. 186:59–64.
226. Jalanko, A., and H. Söderlund. 1985. The repeated regions of Semliki Forest virus defective-interfering RNA interfere with the encapsidation process of the standard virus. Virology 141:257–266.
227. Johnson, B. J. B., J. R. Brubaker, J. T. Roehrig, and D. W. Trent. 1990. Variants of Venezuelan equine encephalitis virus that resist neutralization define a domain of the E2 glycoprotein. Virology 177:676–683.
228. Johnson, B. J. B., R. M. Kinney, C. L. Kost, and D. W. Trent. 1986. Molecular determinants of alphavirus virulence: nucleotide and deduced protein sequence changes during attenuation of Venezuelan equine encephalitis virus. J. Gen. Virol. 67:1951–1960.
229. Johnson, D. C., and M. J. Schlesinger. 1980. Vesicular stomatitis virus and Sindbis virus glycoprotein transport to the cell surface is inhibited by ionophores. Virology 103:407–424.
230. Johnson, D. C., M. J. Schlesinger, and E. L. Elson. 1981. Fluorescence photobleaching recovery measurements reveal differences in envelopment of Sindbis and vesicular stomatitis virus. Cell 23:423–431.
231. Johnson, E. S., D. K. Gonda, and A. Varshavsky. 1990. cis-trans recognition and subunit-specific degradation of short-lived proteins. Nature (London) 346:287–291.
232. Johnston, R. E., and J. F. Smith. 1988. Selection for accelerated penetration in cell culture coselects for attenuated mutants of Venezuelan equine encephalitis virus. Virology 162:437–443.
233. Johnston, R. E., D. R. Tovell, D. T. Brown, and P. Faulkner. 1975. Interfering passages of Sindbis virus: concomitant appearance of interference, morphological variants and truncated viral RNA. J. Virol. 16:951–958.
234. Johnston, R. E., K. Wan, and H. R. Bose. 1974. Homologous interference induced by Sindbis virus. J. Virol. 14:1076–1082.
235. Jones, K. J., R. K. Scupham, J. A. Pfeil, K. Wan, B. P. Sagik, and H. R. Bose. 1977. Interaction of Sindbis virus glycoproteins during morphogenesis. J. Virol. 21:778–787.
236. Justman, J., M. R. Klimjack, and M. Kielian. 1993. Role of spike protein conformational changes in fusion of Semliki Forest virus. J. Virol. 67:7597–7607.
237. Kääriäinen, L., R. F. Pettersson, S. Keränen, P. Lehtovaara, H. Söderlund, and P. Ukkonen. 1981. Multiple structurally related defective-interfering RNAs formed during undiluted passages of Semliki Forest virus. Virology 113:686–697.
238. Kääriäinen, L., and M. Ranki. 1984. Inhibition of cell functions by RNA-virus infections. Annu. Rev. Microbiol. 38:91–109.
239. Kail, M., M. Hollinshead, W. Ansorge, R. Pepperkok, R. Frank, G. Griffiths, and D. Vaux. 1991. The cytoplasmic domain of alphavirus E2 glycoprotein contains a short linear recognition signal required for viral budding. EMBO J. 10:2343–2351.
240. Kaluza, G., and G. Pauli. 1980. The influence of intramolecular disulfide bonds on the structure and function of Semliki Forest virus membrane glycoproteins. Virology 102:300–309.

241. Kamer, G., and P. Argos. 1984. Primary structural comparison of RNA-dependent polymerases from plant, animal and bacterial viruses. Nucleic Acids Res. 12:7269–7282.
242. Kempf, C., U. Kohler, M. R. Michel, and H. Koblet. 1987. Semliki Forest virus-induced polykaryocyte formation is an ATP-dependent event. Arch. Virol. 95:111–122.
243. Kempf, C., M. R. Michel, U. Kohler, and H. Koblet. 1987. A novel method for the detection of early events in cell-cell fusion of Semliki Forest virus infected cells growing in monolayer cultures. Arch. Virol. 95:283–289.
244. Kempf, C., M. R. Michel, U. Kohler, and H. Koblet. 1988. Exposure of Semliki Forest virus-infected baby hamster kidney cells to low pH leads to a proton influx and a rapid depletion of intracellular ATP which in turn prevents cell-cell fusion. Arch. Virol. 99:111–115.
245. Kempf, C., M. R. Michel, A. Omar, P. Jentsch, and A. Morell. 1990. Semliki Forest virus induced cell-cell fusion at neutral extracellular pH. Biosci. Rep. 10:363–374.
246. Keränen, S., and L. Kääriäinen. 1979. Functional defects of RNA-negative temperature-sensitive mutants of Sindbis and Semliki Forest viruses. J. Virol. 32:19–29.
247. Keränen, S., and L. Ruohonen. 1983. Nonstructural proteins of Semliki Forest virus: synthesis, processing, and stability in infected cells. J. Virol. 47:505–515.
248. Kerr, P. J., S. Fitzgerald, G. W. Tregear, L. Dalgarno, and R. C. Weir. 1992. Characterization of a major neutralization domain of Ross River virus using anti-viral and anti-peptide antibodies. Virology 187:338–342.
249. Kerr, P. J., R. C. Weir, and L. Dalgarno. 1993. Ross River virus variants selected during passage in chick embryo fibroblasts: serological, genetic, and biological changes. Virology 193:446–449.
250. Kielian, M., and A. Helenius. 1986. Entry of alphaviruses, p. 91–119. In S. Schlesinger and M. J. Schlesinger (ed.), The Togaviridae and Flaviviridae. Plenum Publishing Corp., New York.
251. Kielian, M., and S. Jungerwirth. 1990. Mechanism of enveloped virus entry into cells. Mol. Biol. Med. 7:17–31.
252. Kielian, M., S. Jungerwirth, K. U. Sayad, and S. DeCandido. 1990. Biosynthesis, maturation, and acid activation of the Semliki Forest virus fusion protein. J. Virol. 64:4614–4624.
253. Kielian, M. C., and A. Helenius. 1984. Role of cholesterol in fusion of Semliki Forest virus with membranes. J. Virol. 52:281–283.
254. Kielian, M. C., and A. Helenius. 1985. pH-induced alterations in the fusogenic spike protein of Semliki Forest virus. J. Cell Biol. 101:2284–2291.
255. Kielian, M. C., S. Keränen, L. Kääriäinen, and A. Helenius. 1984. Membrane fusion mutants of Semliki Forest virus. J. Cell Biol. 98:139–145.
256. Kielian, M. C., M. Marsh, and A. Helenius. 1986. Kinetics of endosome acidification detected by mutant and wild type Semliki Forest virus. EMBO J. 5:3103–3109.
257. Kinney, R. M., G.-J. Chang, K. R. Tsuchiya, J. M. Sneider, J. T. Roehrig, T. M. Woodward, and D. W. Trent. 1993. Attenuation of Venezuelan equine encephalitis virus strain TC-83 is encoded by the 5′-noncoding region and the E2 envelope glycoprotein. J. Virol. 67:1269–1277.
258. Kinney, R. M., B. J. B. Johnson, V. L. Brown, and D. W. Trent. 1986. Nucleotide sequence of the 26S mRNA of the virulent Trinidad donkey strain of Venezuelan equine encephalitis virus and deduced sequence of the encoded structural proteins. Virology 152:400–413.
259. Kinney, R. M., B. J. B. Johnson, J. B. Welch, K. R. Tsuchiya, and D. W. Trent. 1989. The full-length nucleotide sequences of the virulent Trinidad donkey strain of Venezuelan equine encephalitis virus and its attenuated vaccine derivative, strain TC-83. Virology 170:19–30.
260. Kinney, R. M., K. R. Tsuchiya, J. M. Sneider, and D. W. Trent. 1992. Genetic evidence that epizootic Venezuelan equine encephalitis (VEE) viruses may have evolved from enzootic VEE subtype I-D virus. Virology 191:569–580.
261. Kinney, R. M., K. R. Tsuchiya, J. M. Sneider, and D. W. Trent.

1992. Molecular evidence for the origin of the widespread Venezuelan equine encephalitis epizootic of 1969–1972. J. Gen. Virol. 73:3301–3305.

262. Koblet, H., C. Kempf, U. Kohler, and A. Omar. 1985. Conformational changes at pH 6 on the cell surface of Semliki Forest virus-infected *Aedes albopictus* cells. Virology 143:334–336.

263. Kohara, M., T. Omata, A. Kameda, B. L. Semler, H. Itoh, E. Wimmer, and A. Nomoto. 1985. In vitro phenotypic markers of a poliovirus recombinant constructed from infectious cDNA clone of the neurovirulent Mahoney strain and the attenuated Sabin 1 strain. J. Virol. 53:786–792.

264. Kondor-Koch, C., B. Burke, and H. Garoff. 1983. Expression of Semliki Forest virus proteins from cloned complementary DNA. I. The fusion activity of the spike glycoprotein. J. Cell Biol. 97:644–651.

265. Kondor-Koch, C., H. Riedel, K. Soderberg, and H. Garoff. 1982. Expression of the structural proteins of Semliki Forest virus from cloned cDNA microinjected into the nucleus of the baby hamster kidney cells. Proc. Natl. Acad. Sci. USA 79:4525–4529.

266. Koonin, E. V. 1991. The phylogeny of RNA-dependent RNA polymerases of positive-strand RNA viruses. J. Gen. Virol. 72:2197–2206.

267. Koonin, E. V., and V. V. Dolja. 1993. Evolution and taxonomy of positive-strand RNA viruses: implications of comparative analysis of amino acid sequences. Crit. Rev. Biochem. Mol. Biol. 28:375–430.

268. Kowal, K. J., and V. Stollar. 1981. Temperature-sensitive host-dependent mutants of Sindbis virus. Virology 114:140–148.

269. Kuhn, R. J. 1994. Personal communication.

270. Kuhn, R. J., D. E. Griffin, H. G. M. Niesters, and J. H. Strauss. Unpublished data.

271. Kuhn, R. J., D. E. Griffin, H. Zhang, H. G. M. Niesters, and J. H. Strauss. 1992. Attenuation of Sindbis virus neurovirulence by using defined mutations in nontranslated regions of the genome RNA. J. Virol. 66:7121–7127.

272. Kuhn, R. J., Z. Hong, and J. H. Strauss. 1990. Mutagenesis of the 3′ nontranslated region of Sindbis virus RNA. J. Virol. 64:1465–1476.

273. Kuhn, R. J., H. G. M. Niesters, H. Zhang, and J. H. Strauss. 1991. Infectious RNA transcripts from Ross River virus cDNA clones and the construction and characterization of defined chimeras with Sindbis virus. Virology 182:430–441.

274. Kuisamanen, E., and J. Saraste. 1989. Low temperature-induced transport blocks as tools to manipulate membrane traffic. Methods Cell Biol. 32:257–274.

275. Lanzrein, M., N. Käsermann, and C. Kempf. 1992. Changes in membrane permeability during Semliki Forest virus induced cell fusion. Biosci. Rep. 12:221–236.

276. Lanzrein, M., N. Käsermann, R. Weingart, and C. Kempf. 1993. Early events of Semliki Forest virus-induced cell-cell fusion. Virology 196:541–547.

277. Lanzrein, M., R. Weingart, and C. Kempf. 1993. pH-dependent pore formation in Semliki Forest virus-infected *Aedes albopictus* cells. Virology 193:296–302.

278. Laver, W. G., G. M. Air, R. G. Webster, and S. J. Smith-Gill. 1990. Epitopes on protein antigens: misconceptions and realities. Cell 61:553–556.

279. Leathers, V., R. Tanguay, M. Kobayashi, and D. R. Gallie. 1993. A phylogenetically conserved sequence within viral 3′ untranslated RNA pseudoknots regulates translation. Mol. Cell. Biol. 13:5331–5347.

280. Lee, H., and D. T. Brown. 1994. Mutations in an exposed domain of Sindbis virus capsid protein result in the production of noninfectious virions and morphological variants. Virology 202:390–400.

281. Lehtovaara, P., H. Söderlund, S. Keränen, R. F. Pettersson, and L. Kääriäinen. 1981. 18S defective interfering RNA of Semliki Forest virus contains a triplicated linear repeat. Proc. Natl. Acad. Sci. USA 78:5353–5357.

282. Lehtovaara, P., H. Söderlund, S. Keränen, R. F. Pettersson, and L. Kääriäinen. 1982. Extreme ends of the genome are conserved and rearranged in the defective interfering RNAs of Semliki Forest virus. J. Mol. Biol. 156:731–748.

283. Lemm, J. A., R. K. Durbin, V. Stollar, and C. M. Rice. 1990. Mutations which alter the level or structure of nsP4 can affect the efficiency of Sindbis virus replication in a host-dependent manner. J. Virol. 64:3001–3011.

284. Lemm, J. A., and C. M. Rice. 1993. Assembly of functional Sindbis virus RNA replication complexes: requirement for coexpression of P123 and P34. J. Virol. 67:1905–1915.

285. Lemm, J. A., and C. M. Rice. 1993. Roles of nonstructural polyproteins and cleavage products in regulating Sindbis virus RNA replication and transcription. J. Virol. 67:1916–1926.

286. Lemm, J. A., T. Rümenapf, E. G. Strauss, J. H. Strauss, and C. M. Rice. 1994. Polypeptide requirements for assembly of functional Sindbis virus replication complexes: a model for the temporal regulation of minus-strand and plus-strand RNA-synthesis. EMBO J. 13:2925–2934.

287. Lerner, D. M., J. M. Deutsch, and G. F. Oster. 1993. How does a virus bud? Biophys. J. 65:73–79.

288. Levine, B., and D. Griffin. 1993. Molecular analysis of neurovirulent strains of Sindbis virus that evolve during persistent infection of *scid* mice. J. Virol. 67:6872–6875.

289. Levine, B., and D. E. Griffin. 1992. Persistence of viral RNA in mouse brains after recovery from acute alphavirus encephalitis. J. Virol. 66:6429–6435.

290. Levine, B., H. M. Hardwick, B. D. Trapp, T. O. Crawford, R. C. Bollinger, and D. E. Griffin. 1991. Antibody-mediated clearance of alphavirus infection from neurons. Science 254:856–860.

291. Levine, B., Q. Huang, J. T. Isaacs, J. C. Reed, D. E. Griffin, and J. M. Hardwick. 1993. Conversion of lytic to persistent alphavirus infection by the *bcl-2* cellular oncogene. Nature (London) 361:739–742.

292. Levinson, R., J. H. Strauss, and E. G. Strauss. 1990. Determination of the complete nucleotide sequence of the genomic RNA of O’Nyong-nyong virus and its use in the construction of phylogenetic trees. Virology 175:110–123.

293. Levis, R., H. Huang, and S. Schlesinger. 1987. Engineered defective interfering RNAs of Sindbis virus express bacterial chloramphenicol acetyltransferase in avian cells. Proc. Natl. Acad. Sci. USA 84:4811–4815.

294. Levis, R., S. Schlesinger, and H. V. Huang. 1990. The promoter for Sindbis virus RNA-dependent subgenomic RNA transcription. J. Virol. 64:1726–1733.

295. Levis, R., B. G. Weiss, M. Tsiang, H. Huang, and S. Schlesinger. 1986. Deletion mapping of Sindbis virus DI RNAs derived from cDNAs defines the sequences essential for replication and packaging. Cell 44:137–145.

296. Levy-Mintz, P., and M. Kielian. 1991. Mutagenesis of the putative fusion domain of the Semliki Forest virus spike protein. J. Virol. 65:4292–4300.

297. Li, G., M. W. LaStarza, W. R. Hardy, J. H. Strauss, and C. M. Rice. 1990. Phosphorylation of Sindbis virus nsP3 in vivo and in vitro. Virology 179:416–427.

298. Li, G., B. M. Prágai, and C. M. Rice. 1991. Rescue of Sindbis virus-specific RNA replication and transcription by using a vaccinia virus recombinant. J. Virol. 65:6714–6723.

299. Li, G., and C. M. Rice. 1989. Mutagenesis of the in-frame opal termination codon preceding nsP4 of Sindbis virus: studies of translational readthrough and its effect on virus replication. J. Virol. 63:1326–1337.

300. Li, G., and C. M. Rice. 1993. The signal for translational readthrough of a UGA codon in Sindbis virus RNA involves a single cytidine residue immediately downstream of the termination codon. J. Virol. 67:5062–5067.

301. Liljeström, P., and H. Garoff. 1991. Internally located cleavable signal sequences direct the formation of Semliki Forest virus membrane proteins from a polyprotein precursor. J. Virol. 65:147–154.

302. Liljeström, P., and H. Garoff. 1991. A new generation of animal cell expression vectors based on the Semliki Forest virus replicon. Bio/Technology 9:1356–1361.

303. Liljeström, P., S. Lusa, D. Huylebroeck, and H. Garoff. 1991. In vitro mutagenesis of a full-length cDNA clone of Semliki Forest virus: the small 6000-molecular-weight membrane protein modulates virus release. J. Virol. 65:4107–4113.

304. Lindqvist, B. H., J. DiSalvo, C. M. Rice, J. H. Strauss, and E. G. Strauss. 1986. Sindbis virus mutant ts20 of complementation group E contains a lesion in glycoprotein E2. Virology 151:10–20.

305. Lindsay, M. D. A., R. J. Coelen, and J. S. Mackenzie. 1993. Genetic heterogeneity among isolates of Ross River virus from different geographical regions. J. Virol. 67:3576–3585.

306. Liu, N., and D. T. Brown. 1993. Transient translocation of the cytoplasmic (endo) domain of a type I membrane glycoprotein into cellular membranes. J. Cell Biol. 120:877–883.

307. Liu, N., and D. T. Brown. 1994. Phosphorylation and dephosphorylation events play critical roles in Sindbis virus maturation. Virology 196:703–711.

308. Lobigs, M., and H. Garoff. 1990. Fusion function of the Semliki Forest virus spike is activated by proteolytic cleavage of the envelope glycoprotein precursor p62. J. Virol. 64:1233–1240.

309. Lobigs, M., Z. Hongxing, and H. Garoff. 1990. Function of Semliki Forest virus E3 peptide in virus assembly: replacement of E3 with an artificial signal peptide abolishes spike heterodimerization and surface expression of E1. J. Virol. 64:4346–4355.

309a. Lobigs, M., I. D. Marshall, R. C. Weir, and L. Dalgarno. 1986. Genetic differentiation of Murray Valley encephalitis virus in Australia and Papua New Guinea. Aust. J. Exp. Biol. Med. Sci. 64:571–585.

310. Lobigs, M., J. M. Wahlberg, and H. Garoff. 1990. Spike protein oligomerization control of Semliki Forest virus fusion. J. Virol. 64:5214–5218.

310a. Lobigs, M., R. C. Weir, and L. Dalgarno. 1986. Genetic analysis of Kunjin virus isolates using HaeIII and TaqI restriction digests of single-stranded cDNA to virion RNA. Aust. J. Exp. Biol. Med. Sci. 64:185–196.

311. London, S. D., A. L. Schmaljohn, J. D. Dalrymple, and C. M. Rice. 1992. Infectious enveloped RNA virus RNA chimeras. Proc. Natl. Acad. Sci. USA 89:207–211.

312. Lopez, S., J.-S. Yao, R. J. Kuhn, E. G. Strauss, and J. H. Strauss. 1994. Nucleocapsid-glycoprotein interactions required for alphavirus assembly. J. Virol. 68:1316–1323.

313. Lord, R. D., and C. H. Calisher. 1970. Further evidence of southward transport of arboviruses by migratory birds. J. Epidemiol. 92:73–78.

314. Ludwig, G. V., and J. F. Smith. 1994. Personal communication.

315. Luo, T., and D. T. Brown. 1993. Purification and characterization of a Sindbis virus-induced peptide which stimulates its own production and blocks virus RNA synthesis. Virology 194:44–49.

316. Luo, T., and D. T. Brown. 1994. A 55-kDa protein induced in *Aedes albopictus* (mosquito) cells by antiviral protein. Virology 200:200–206.

317. Lusa, S., H. Garoff, and P. Liljeström. 1991. Fate of the 6K membrane protein of Semliki Forest virus during virus assembly. Virology 185:843–846.

318. Lustig, S., A. Jackson, C. S. Hahn, D. E. Griffin, E. G. Strauss, and J. H. Strauss. 1988. Molecular basis of Sindbis virus neurovirulence in mice. J. Virol. 62:2329–2336.

319. Maassen, J. A., and C. Terhorst. 1981. Identification of a cell-surface protein involved in the binding site of Sindbis virus on human lymphoblastic cell lines using a heterobifunctional cross-linker. Eur. J. Biochem. 115:153–158.

320. Mabruk, M. J., A. M. Flack, G. M. Glasgow, J. M. Smyth, J. C. Folan, J. G. Bannigan, M. A. O’Sullivan, B. J. Sheahan, and G. J. Atkins. 1988. Teratogenicity of the Semliki Forest virus mutant ts22 for the foetal mouse: induction of skeletal and skin defects. J. Gen. Virol. 69:2755–2762.

321. Mabruk, M. J., G. M. Glasgow, A. M. Flack, J. C. Folan, J. G. Bannigan, J. M. Smyth, M. A. O’Sullivan, B. J. Sheahan, and G. J. Atkins. 1989. Effect of infection with the ts22 mutant of Semliki Forest virus on development of the central nervous system in the fetal mouse. J. Virol. 63:4027–4033.

322. Malinoski, F., and V. Stollar. 1980. Inhibition of Sindbis virus replication in *Aedes albopictus* cells by virazole (ribavirin) and its reversal by actinomycin: a correction. Virology 102:473–476.

323. Mann, E., J. Edwards, and D. T. Brown. 1983. Polycaryocyte formation mediated by Sindbis virus glycoproteins. J. Virol. 45:1083–1089.

324. Marquardt, M. T., T. Phalen, and M. Kiilian. 1993. Cholesterol

is required in the exit pathway of Semliki Forest virus. J. Cell Biol. 123:57–65.

325. Marquardt, T., and A. Helenius. 1992. Misfolding and aggregation of newly synthesized proteins in the endoplasmic reticulum. J. Cell Biol. 117:505–513.

326. Marsh, M. 1984. The entry of enveloped viruses into cells by endocytosis. Biochem. J. 218:1–10.

327. Marsh, M., E. Bolzau, and A. Helenius. 1983. Penetration of Semliki Forest virus from acidic prelysosomal vacuoles. Cell 32:931–940.

328. Marsh, M., and A. Helenius. 1980. Adsorptive endocytosis of Semliki Forest virus. J. Mol. Biol. 142:439–454.

329. Marsh, M., and A. Helenius. 1989. Virus entry into animal cells. Adv. Virus Res. 36:107–151.

330. Marsh, M., J. Wellstead, H. Kern, E. Harms, and A. Helenius. 1982. Monensin inhibits Semliki Forest virus penetration into baby hamster kidney (BHK-21) cells. Proc. Natl. Acad. Sci. USA 79:5297–5301.

331. Marshall, I. D., B. K. Brown, K. Keith, G. P. Gard, and E. Thibos. 1982. Variation in arbovirus infection rates in species of birds sampled in a serological survey during an encephalitis epidemic in the Murray Valley of south-eastern Australia, February 1974. Aust. J. Exp. Biol. Med. Sci. 60:471–478.

332. Marshall, I. D., and J. A. R. Miles. 1984. Ross River virus and epidemic polyarthritis. Curr. Top. Vector Res. 2:31–56.

333. Marshall, I. D., E. Thibos, and K. Clarke. 1982. Species composition of mosquitoes collected in the Murray Valley of south-eastern Australia during an epidemic of arboviral encephalitis. Aust. J. Exp. Biol. Med. Sci. 60:447–456.

334. Marshall, I. D., G. M. Woodroofe, and S. Hirsch. 1982. Viruses recovered from mosquitoes and wildlife serum collected in the Murray Valley of south-eastern Australia, February 1974, during an epidemic of encephalitis. Aust. J. Exp. Biol. Med. Sci. 60:457–470.

335. Martin, E. M., and J. A. Sonnabend. 1967. Ribonucleic acid polymerase catalyzing synthesis of double-stranded arbovirus ribonucleic acid. J. Virol. 1:97–109.

336. Mathews, J. H., and J. T. Roehrig. 1982. Determination of the protective epitopes on the glycoproteins of Venezuelan equine encephalomyelitis virus by passive transfer of monoclonal antibodies. J. Immunol. 129:2763–2767.

337. Mathiot, C. C., G. Grimaud, P. Garry, J. C. Bouquety, A. Mada, A. M. Daguisy, and A. J. Georges. 1990. An outbreak of human Semliki Forest virus infection in Central African Republic. Am. J. Trop. Med. Hyg. 42:386–393.

338. Matlin, K. S. 1986. The sorting of proteins to the plasma membrane in epithelial cells. J. Cell Biol. 103:2565–2568.

339. Matlin, K. S., and K. Simons. 1984. Sorting of apical plasma membrane glycoprotein occurs before it reaches the cell surface in cultured epithelial cells. J. Cell Biol. 99:2131–2139.

340. Mayne, J. T., J. R. Bell, E. G. Strauss, and J. H. Strauss. 1985. Pattern of glycosylation of Sindbis virus envelope proteins synthesized in hamster and chicken cells. Virology 142:121–133.

341. McDowell, W., P. A. Romero, R. Datema, and R. T. Schwarz. 1987. Glucose trimming and mannose trimming affect different phases of the maturation of Sindbis virus in infected BHK cells. Virology 161:37–44.

342. Meek, A. D. J., S. G. Faragher, R. C. Weir, and L. Dalgarno. 1989. Genetic and phenotypic studies on Ross River virus variants of enhanced virulence selected during mouse passage. Virology 172:399–407.

343. Mehta, S., S. Pathak, and H. E. Webb. 1990. Induction of membrane proliferation in mouse CNS by gold sodium thiamate with reference to increased virulence of the avirulent Semliki Forest virus. Biosci. Rep. 10:271–279.

344. Melancon, P., and H. Garoff. 1986. Reinitiation of translocation in the Semliki Forest virus structural polyprotein: identification of the signal for the E1 glycoprotein. EMBO J. 5:1551–1560.

345. Melancon, P., and H. Garoff. 1987. Processing of the Semliki Forest virus structural polyprotein: role of the capsid protease. J. Virol. 61:1301–1309.

346. Mellman, I., and K. Simons. 1992. The Golgi complex: in vitro veritas? Cell 68:829–840.

347. Mendoza, Q. P., J. Stanley, and D. E. Griffin. 1988. Monoclonal antibodies to the E1 and E2 glycoproteins of Sindbis virus: definition of epitopes and efficiency of protection from fatal encephalitis. J. Gen. Virol. 70:3015–3022.

348. Metsikkö, K., and H. Garoff. 1990. Oligomers of the cytoplasmic domain of the p62/E2 membrane protein of Semliki Forest virus bind to the nucleocapsid in vitro. J. Virol. 64:4678–4683.

349. Meyer, W. J., S. Gidwitz, V. K. Ayers, R. J. Schoepp, and R. E. Johnston. 1992. Conformational alteration of Sindbis virion glycoproteins induced by heat, reducing agents, or low pH. J. Virol. 66:3504–3513.

350. Mi, S., R. Durbin, H. V. Huang, C. M. Rice, and V. Stollar. 1989. Association of the Sindbis virus RNA methyltransferase activity with the nonstructural protein nsP1. Virology 170:385–391.

351. Mi, S., and V. Stollar. 1990. Both amino acid changes in nsP1 of Sindbis virus<sub>LM21</sub> contribute to and are required for efficient expression of the mutant phenotype. Virology 178:429–434.

352. Mi, S., and V. Stollar. 1991. Expression of Sindbis virus nsP1 methyltransferase activity in *Escherichia coli*. Virology 184:423–427.

353. Michel, M. R., M. Elgizoli, Y. Dai, R. Jakob, H. Koblet, and A.-P. Arrigo. 1990. Karyophilic properties of Semliki Forest virus nucleocapsid protein. J. Virol. 64:5123–5131.

354. Michel, M. R., and P. J. Gomatos. 1973. Semliki Forest virus-specific RNAs synthesized in vitro by enzyme from infected BHK cells. J. Virol. 11:900–914.

355. Migliaccio, G., M. C. Pascale, A. Leone, and S. Bonatti. 1989. Biosynthesis, membrane translocation, and surface expression of Sindbis virus E1 glycoprotein. Exp. Cell Res. 185:203–216.

356. Miller, M. L., and D. T. Brown. 1991. Alphavirus infection in cultured tissue cells. Adv. Dis. Vector Res. 8:107–142.

357. Miller, M. L., and D. T. Brown. 1993. The distribution of Sindbis virus proteins in mosquito cells as determined by immunofluorescence and immunoelectron microscopy. J. Gen. Virol. 74:293–298.

358. Mokhtarian, F., D. Grob, and D. E. Griffin. 1989. Role of the immune response in Sindbis virus-induced paralysis of SJL/J mice. J. Immunol. 143:633–637.

359. Mokhtarian, F., and P. Swoveland. 1987. Predisposition to EAE induction in resistant mice by prior infection with Semliki Forest virus. J. Immunol. 138:3264–3268.

360. Monroe, S. S., J.-H. Ou, C. M. Rice, S. Schlesinger, E. G. Strauss, and J. H. Strauss. 1982. Sequence analysis of cDNAs derived from the RNA of Sindbis virions and of defective interfering particles. J. Virol. 41:153–162.

361. Monroe, S. S., and S. Schlesinger. 1983. RNAs from two independently isolated defective interfering particles of Sindbis virus contain a cellular tRNA sequence at their 5′ ends. Proc. Natl. Acad. Sci. USA 80:3279–3283.

362. Monroe, S. S., and S. Schlesinger. 1984. Common and distinct regions of defective-interfering RNAs of Sindbis virus. J. Virol. 49:865–872.

363. Morrison, T., and A. Portner. 1991. Structure, function, and intracellular processing of the glycoproteins of *Paramyxoviridae*, p. 347–382. In D. W. Kingsbury (ed.), The paramyxoviruses. Plenum Publishing Corp., New York.

364. Mudge, P. R., R. S. H. Lim, B. Moore, and A. J. Radford. 1980. Epidemic polyarthritis in South Australia 1979–1980. Med. J. Aust. 2:626–627.

365. Mulvey, M., and D. T. Brown. 1994. Formation and rearrangement of disulfide bonds during maturation of the Sindbis virus E1 glycoprotein. J. Virol. 68:805–812.

366. Naim, H. Y., and H. Koblet. 1990. The cleavage of p62, the precursor of E2 and E3, is an early and continuous event in Semliki Forest virus-infected *Aedes albopictus* cells. Arch. Virol. 110:221–237.

367. Nakhasi, H. L., X.-Q. Cao, T. A. Rouault, and T.-Y. Liu. 1991. Specific binding of host cell proteins to the 3′-terminal stem-loop structure of rubella virus negative-strand RNA. J. Virol. 65:5961–5967.

368. Nakhasi, H. L., T. A. Rouault, D. J. Haile, T.-Y. Liu, and R. D. Klausner. 1990. Specific high-affinity binding of host cell proteins to the 3′ region of rubella virus RNA. New Biol. 2:255–264.

369. Newton, S. E., and L. Dalgarno. 1983. Antiviral activity released from *Aedes albopictus* cells persistently infected with Semliki Forest virus. J. Virol. 47:652–655.

370. Niesters, H. G. M., and J. H. Strauss. 1990. Defined mutations in the 5′ nontranslated sequence of Sindbis virus RNA. J. Virol. 64:4162–4168.

371. Niesters, H. G. M., and J. H. Strauss. 1990. Mutagenesis of the conserved 51 nucleotide region of Sindbis virus. J. Virol. 64:1639–1647.

372. Niklasson, B. 1988. Sindbis and Sindbis-like viruses, p. 167–176. In T. P. Monath (ed.), The arboviruses: epidemiology and ecology. CRC Press, Inc., Boca Raton, Fla.

373. Niklasson, B., A. Aspmark, J. W. LeDuc, T. P. Gargan, W. A. Ennis, R. B. Tesh, and J. A. J. Main. 1984. Association of a Sindbis-like virus with Ockelbo disease in Sweden. Am. J. Trop. Med. Hyg. 33:1212–1217.

374. Nitsch, L., D. Tramontano, and F.-S. Ambesi-Impiombato. 1985. Morphological and functional polarity of an epithelial thyroid cell line. Eur. J. Cell Biol. 38:57–66.

375. Nuss, D. L., H. O. Oppermann, and G. Koch. 1975. Selective blockage of initiation of host protein synthesis in RNA virus infected cells. Proc. Natl. Acad. Sci. USA 72:1258–1262.

376. Oldstone, M. B. A., A. Tishon, F. Dutko, S. I. T. Kennedy, J. J. Holland, and P. W. Lampert. 1980. Does the major histocompatibility complex serve as a specific receptor for Semliki Forest virus? J. Virol. 34:256–265.

377. Olivo, P. D., I. Frolov, and S. Schlesinger. 1994. A cell line that expresses a reporter gene in response to infection by Sindbis virus: a prototype for detection of positive strand RNA viruses. Virology 198:381–384.

378. Olmsted, R. A., R. S. Baric, B. A. Sawyer, and R. E. Johnston. 1984. Sindbis virus mutants selected for rapid growth in cell culture display attenuated virulence in animals. Science 225:424–427.

379. Olmsted, R. A., W. J. Meyer, and R. E. Johnston. 1986. Characterization of Sindbis virus epitopes important for penetration in cell culture and pathogenesis in animals. Virology 148:245–254.

380. Olson, K., and D. W. Trent. 1985. Genetic and antigenic variations among geographical isolates of Sindbis virus. J. Gen. Virol. 66:797–810.

381. Omar, A., and H. Koblet. 1988. Semliki Forest virus particles containing only the E1 envelope glycoprotein are infectious and can induce cell-cell fusion. Virology 166:17–23.

382. Omar, A., and H. Koblet. 1989. The use of sulfite to study the mechanism of membrane fusion induced by E1 of Semliki Forest virus. Virology 168:177–179.

383. Orci, L., M. Ravazzola, M.-J. Storch, R. G. W. Anderson, J.-D. Vassalli, and A. Perrelet. 1987. Proteolytic maturation of insulin is a post-Golgi event which occurs in acidifying clathrin-coated secretory vesicles. Cell 49:865–868.

384. Ou, J.-H., C. M. Rice, L. Dalgarno, E. G. Strauss, and J. H. Strauss. 1982. Sequence studies of several alphavirus genomic RNAs in the region containing the start of the subgenomic RNA. Proc. Natl. Acad. Sci. USA 79:5235–5239.

385. Ou, J.-H., E. G. Strauss, and J. H. Strauss. 1981. Comparative studies of the 3′-terminal sequences of several alphavirus RNAs. Virology 109:281–289.

386. Ou, J.-H., E. G. Strauss, and J. H. Strauss. 1983. The 5′-terminal sequences of the genomic RNAs of several alphaviruses. J. Mol. Biol. 168:1–15.

387. Ou, J.-H., D. W. Trent, and J. H. Strauss. 1982. The 3′-noncoding regions of alphavirus RNAs contain repeating sequences. J. Mol. Biol. 156:719–730.

388. Pardigon, N. 1994. Unpublished data.

389. Pardigon, N., E. Lenches, and J. H. Strauss. 1993. Multiple binding sites for cellular proteins in the 3′ end of Sindbis alphavirus minus sense RNA. J. Virol. 67:5003–5011.

390. Pardigon, N., and J. H. Strauss. 1992. Cellular proteins bind to the 3′ end of Sindbis virus minus strand RNA. J. Virol. 66:1007–1015.

391. Paredes, A. M., D. T. Brown, R. B. Rothnagel, W. Chiu, R. J. Schoepp, R. E. Johnston, and B. V. V. Prasad. 1993. Three-dimensional structure of a membrane-containing virus. Proc.

Natl. Acad. Sci. USA 90:9095–9099.

392. Paredes, A. M., M. Simon, and D. T. Brown. 1993. The mass of the Sindbis virus nucleocapsid suggests it has T=4 icosahedral symmetry. Virology 187:324–332.

393. Parry, N., G. Fox, D. Rowlands, F. Brown, E. Fry, R. Acharya, D. Logan, and D. Stuart. 1990. Structural and serological evidence for a novel mechanism of antigenic variation in foot-and-mouth disease virus. Nature (London) 347:569–572.

394. Pavan, A., E. Covelli, M. C. Pascale, G. Lucania, S. Bonatti, P. Pinto da Silva, and M. R. Torrisi. 1992. Dynamics of transmembrane proteins during Sindbis virus budding. J. Cell Sci. 102:149–155.

395. Pavan, A., L. V. Lotti, M. R. Torrisi, G. Migliaccio, and S. Bonatti. 1987. Regional distribution of Sindbis virus glycoproteins on the plasma membrane of infected baby hamster kidney cells. Exp. Cell Res. 168:53–62.

396. Pence, D. F., N. L. Davis, and R. E. Johnston. 1990. Antigenic and genetic characterization of Sindbis virus monoclonal antibody escape mutants which define a pathogenesis domain on glycoprotein E2. Virology 175:41–49.

397. Peränen, J. 1991. Localization and phosphorylation of Semliki Forest virus non-structural protein nsP3 expressed in COS cells from a cloned cDNA. J. Gen. Virol. 72:195–199.

398. Peränen, J., M. Rikkonen, P. Lileström, and L. Kääriäinen. 1990. Nuclear localization of the Semliki Forest virus-specific non-structural protein nsP2. J. Virol. 64:1888–1896.

399. Peränen, J., K. Takkinen, N. Kalkkinen, and L. Kääriäinen. 1988. Semliki Forest virus-specific non-structural protein nsP3 is a phosphoprotein. J. Gen. Virol. 69:2165–2178.

400. Pérez, L., R. Guinea, and L. Carrasco. 1991. Synthesis of Semliki Forest virus RNA requires continuous lipid synthesis. Virology 183:74–82.

401. Pérez, L., A. Irurzun, and L. Carrasco. 1993. Activation of phospholipase activity during Semliki Forest virus infection. Virology 194:28–36.

402. Peters, C. J., and J. M. Dalrymple. 1990. Alphaviruses, p. 713–761. In B. N. Fields, and D. M. Knipe (ed.), Virology. Raven Press, New York.

403. Pettersson, R. F. 1981. 5′-terminal nucleotide sequence of Semliki Forest virus 18S defective-interfering RNA is heterogeneous and different from the genomic 42S RNA. Proc. Natl. Acad. Sci. USA 78:115.

404. Phalen, T., and M. Kielian. 1991. Cholesterol is required for infection by Semliki Forest virus. J. Cell Biol. 112:615–623.

405. Phillips, D. A., J. R. Murray, J. G. Asakov, and M. A. Wiemers. 1990. Clinical and subclinical Barmah Forest virus infection in Queensland. Med. J. Aust. 152:463–466.

406. Polo, J. M., N. L. Davis, C. M. Rice, H. V. Huang, and R. E. Johnston. 1988. Molecular analysis of Sindbis virus pathogenesis in neonatal mice by using virus recombinants constructed in vitro. J. Virol. 62:2124–2133.

407. Polo, J. M., and R. E. Johnston. 1990. Attenuating mutations in glycoproteins E1 and E2 of Sindbis virus produce a highly attenuated strain when combined in vitro. J. Virol. 64:4438–4444.

408. Polo, J. M., and R. E. Johnston. 1991. Mutational analysis of a virulence locus in the E2 glycoprotein gene of Sindbis virus. J. Virol. 65:6358–6361.

409. Porterfield, J. S. 1980. Antigenic characteristics and classification of togaviridae, p. 13–46. In R. W. Schlesinger (ed.), The togaviruses: biology, structure, replication. Academic Press, Inc., New York.

410. Presley, J. F., and D. T. Brown. 1989. The proteolytic cleavage of PE2 to envelope glycoprotein E2 is not strictly required for the maturation of Sindbis virus. J. Virol. 63:1975–1980.

411. Raju, R., and H. V. Huang. 1991. Analysis of Sindbis virus promoter recognition in vivo, using novel vectors with two subgenomic mRNA promoters. J. Virol. 65:2501–2510.

412. Ranki, M., and L. Kääriäinen. 1979. Solubilized RNA replication complex from Semliki Forest virus-infected cells. Virology 98:298–307.

413. Rao, C. N., V. Castronovo, C. M. Schmitt, U. M. Wewer, A. P. Claysmith, L. A. Liotta, and M. E. Sobel. 1989. Evidence for a precursor of the high-affinity metastasis-associated murine laminin receptor. Biochemistry 28:7476–7486.

414. Rennels, M. B. 1984. Arthropod-borne virus infections of the central nervous system. Neurol. Clin. 2:241–254.

415. Rentier-Delrue, F., and N. A. Young. 1980. Genomic divergence among Sindbis virus strains. Virology 106:59–70.

416. Rice, C. M., J. R. Bell, M. W. Hunkapiller, E. G. Strauss, and J. H. Strauss. 1982. Isolation and characterization of the hydrophobic COOH-terminal domains of the Sindbis virus glycoproteins. J. Mol. Biol. 154:355–378.

417. Rice, C. M., C. A. Franke, J. H. Strauss, and D. E. Hruby. 1985. Expression of Sindbis virus structural proteins via recombinant vaccinia virus: synthesis, processing and incorporation into mature Sindbis virions. J. Virol. 56:227–239.

418. Rice, C. M., R. Levis, J. H. Strauss, and H. V. Huang. 1987. Production of infectious RNA transcripts from Sindbis virus cDNA clones: mapping of lethal mutations, rescue of a temperature sensitive marker, and in vitro mutagenesis to generate defined mutants. J. Virol. 61:3809–3819.

419. Rice, C. M., and J. H. Strauss. 1981. Nucleotide sequence of the 26S mRNA of Sindbis virus and deduced sequence of the encoded virus structural proteins. Proc. Natl. Acad. Sci. USA 78:2062–2066.

420. Rice, C. M., and J. H. Strauss. 1982. Association of Sindbis virion glycoproteins and their precursors. J. Mol. Biol. 154:325–348.

421. Rico-Hesse, R., J. T. Roehrig, D. W. Trent, and R. W. Dickerman. 1988. Genetic variation of Venezuelan equine encephalitis virus strains of the ID variety in Columbia. Am. Soc. Trop. Med. Hyg. 38:195–204.

422. Riedel, B., and D. T. Brown. 1979. Novel antiviral activity found in the media of Sindbis virus-persistently infected mosquito (Aedes albopictus) cell cultures. J. Virol. 29:51–60.

423. Riedel, H. 1985. Different membrane anchors allow the Semliki Forest virus spike subunit E2 to reach the cell surface. J. Virol. 54:224–228.

424. Rikkonen, M., J. Peränen, and L. Kääriäinen. 1992. Nuclear and nuclear targeting signals of Semliki Forest virus nonstructural protein nsP2. Virology 189:462–473.

425. Robbins, A. R., S. S. Peng, and J. L. Marshall. 1983. Mutant Chinese hamster ovary cells pleiotropically defective in receptor-mediated endocytosis. J. Cell Biol. 96:1064–1071.

426. Roehrig, J. T. 1986. The use of monoclonal antibodies in studies of the structural proteins of togaviruses and flaviviruses, p. 251–278. In S. Schlesinger and M. J. Schlesinger (ed.), The Togaviridae and Flaviviridae. Plenum Publishing Corp., New York.

427. Roehrig, J. T., J. W. Day, and R. M. Kinney. 1982. Antigenic analysis of the surface glycoproteins of a Venezuelan equine encephalomyelitis virus (TC-83) using monoclonal antibodies. Virology 118:269–278.

428. Roehrig, J. T., D. Gorski, and M. J. Schlesinger. 1982. Properties of monoclonal antibodies directed against the glycoproteins of Sindbis virus. J. Gen. Virol. 59:421–425.

429. Roehrig, J. T., A. R. Hunt, G. J. Chang, B. Sheik, R. A. Bolin, T. F. Tsai, and D. W. Trent. 1990. Identification of monoclonal antibodies capable of differentiating antigenic varieties of Eastern equine encephalitis viruses. Am. J. Trop. Med. Hyg. 42:394–398.

430. Roehrig, J. T., A. R. Hunt, R. M. Kinney, and J. H. Mathews. 1988. In vitro mechanisms of monoclonal antibody neutralization of alphaviruses. Virology 165:66–73.

431. Roehrig, J. T., and J. H. Mathews. 1985. The neutralization site on the E2 glycoprotein of Venezuelan equine encephalomyelitis virus is composed of multiple conformationally-stable epitopes. Virology 142:347–356.

432. Roman, L. M., and H. Garoff. 1986. Alteration of the cytoplasmic domain of the membrane-spanning glycoprotein p62 of Semliki Forest virus does not affect its polar distribution in established lines of Madin-Darby canine kidney cells. J. Cell Biol. 103:2607–2618.

433. Rozanov, M. N., E. V. Koonin, and A. E. Gorbalenya. 1992. Conservation of the putative methyltransferase domain: a hallmark of the ‘Sindbis-like’ supergroup of positive strand RNA viruses. J. Gen. Virol. 73:2129–2134.

434. Rümenapf, T. 1993. Unpublished data.

435. Rümenapf, T., D. T. Brown, E. G. Strauss, and J. H. Strauss. Unpublished data.
436. Rümenapf, T., E. G. Strauss, and J. H. Strauss. 1994. The subgenomic mRNA of Aura alphavirus is packaged into virions. J. Virol. 68:56–62.
437. Russell, D. L., J. M. Dalrymple, and R. E. Johnston. 1989. Sindbis virus mutations which coordinately affect glycoprotein processing, penetration and virulence in mice. J. Virol. 63:1619–1629.
438. Ryser, H. J.-P., R. Mandel, and F. Ghani. 1991. Cell surface sulfhydryls are required for the cytotoxicity of diphtheria toxin but not of ricin in Chinese hamster ovary cells. J. Biol. Chem. 266:18439–18442.
439. Saborio, J. L., S. S. Pong, and G. Koch. 1974. Selective and reversible inhibition of initiation of protein synthesis in mammalian cells. J. Mol. Biol. 85:195–211.
440. Salminen, A., J. M. Wahlberg, M. Lobigs, P. Liljeström, and H. Garoff. 1992. Membrane fusion process of Semliki Forest virus II: cleavage-dependent reorganization of the spike protein complex controls virus entry. J. Cell Biol. 116:349–357.
441. Sanders, B. G., K. M. Wan, K. Kline, R. F. Garry, and H. R. Bose. 1980. Chicken fetal antigens: role as cell surface receptors for Sindbis virus hemagglutinin. Virology 106:183–186.
442. Saraste, J., and K. Hedman. 1983. Intracellular vesicles involved in the transport of Semliki Forest virus membrane proteins to the cell surface. EMBO J. 2:2001–2006.
443. Saraste, J., and E. Kaismanen. 1984. Pre-Golgi and post-Golgi vacuoles operate in the transport of Semliki Forest virus membrane glycoproteins to the cell surface. Cell 38:535–549.
444. Saraste, J., C.-H. von Bonsdorff, K. Hashimoto, L. Kääriäinen, and S. Keränen. 1980. Semliki Forest virus mutant with temperature-sensitive transport defect of envelope proteins. Virology 100:229–245.
445. Sarver, N., and V. Stollar. 1977. Sindbis virus-induced cytopathic effect in clones of *Aedes albopictus* (Singh) cells. Virology 80:390–400.
446. Sawicki, D. L., D. B. Barkhimer, S. G. Sawicki, C. M. Rice, and S. Schlesinger. 1990. Temperature sensitive shut-off of alphavirus minus strand RNA synthesis maps to a nonstructural protein, nsP4. Virology 174:43–52.
447. Sawicki, D. L., and S. G. Sawicki. 1980. Short-lived minus-strand polymerase for Semliki Forest virus. J. Virol. 34:108–118.
448. Sawicki, D. L., and S. G. Sawicki. 1985. Functional analysis of the A complementation group mutants of Sindbis HR virus. Virology 144:20–34.
449. Sawicki, D. L., and S. G. Sawicki. 1993. A second nonstructural protein functions in the regulation of alphavirus negative-strand RNA synthesis. J. Virol. 67:3605–3610.
450. Sawicki, D. L., S. G. Sawicki, S. Keränen, and L. Kääriäinen. 1981. Specific Sindbis virus coded function for minus-strand RNA synthesis. J. Virol. 39:348–358.
451. Sawicki, S. G., and D. L. Sawicki. 1986. The effect of loss of regulation of minus-strand RNA synthesis on Sindbis virus replication. Virology 151:339–349.
452. Sawicki, S. G., and D. L. Sawicki. 1986. The effect of overproduction of nonstructural proteins on alphavirus plus-strand and minus-strand RNA synthesis. Virology 152:507–512.
453. Sawicki, S. G., D. L. Sawicki, L. Kääriäinen, and S. Keränen. 1981. A Sindbis virus mutant temperature-sensitive in the regulation of minus-strand RNA synthesis. Virology 115:161–172.
454. Schärer, C. G., H. Y. Naim, and H. Koblet. 1993. Palmitoylation of Semliki Forest virus glycoproteins in insect cells (C6/36) occurs in an early compartment and is coupled to the cleavage of the precursor p62. Arch. Virol. 132:237–254.
455. Scheidel, L. M., R. K. Durbin, and V. Stollar. 1987. Sindbis virus mutants resistant to mycophenolic acid and ribavirin. Virology 158:1–7.
456. Scheidel, L. M., R. K. Durbin, and V. Stollar. 1989. SVLM21, a Sindbis virus mutant resistant to methionine deprivation, encodes an altered methyltransferase. Virology 173:408–414.
457. Scheule, R. K. 1987. Fusion of Sindbis virus with model membranes containing phosphatidylethanolamine: implications for protein-induced membrane fusion. Biochim. Biophys. Acta 899: 185–195.

458. Schlegel, A., A. Omar, P. Jentsch, A. Morell, and C. Kempf. 1991. Semliki Forest virus envelope proteins function as proton channels. Biosci. Rep. 11:243–255.
459. Schlesinger, M., and S. Schlesinger. 1986. Formation and assembly of alphavirus glycoproteins, p. 121–148. In S. Schlesinger and M. J. Schlesinger (ed.), The Togaviridae and Flaviviridae. Plenum Publishing Corp., New York.
460. Schlesinger, M. J. 1989. Inhibition of Sindbis virus assembly by peptides that mimic the cytoplasmic domain of the virus glycoprotein, p. 73–76. In H.-G. Krausslich, S. Oroszlan, and E. Wimmer (ed.), Viral proteases as targets for chemotherapy. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.
461. Schlesinger, M. J., and D. Cahill. 1989. Verapamil and chlorpromazine inhibit the budding of Sindbis and vesicular stomatitis viruses from infected chicken embryo fibroblasts. Virology 168: 187–190.
462. Schlesinger, M. J., S. D. London, and C. Ryan. 1993. An in-frame insertion into the Sindbis virus 6K gene leads to defective proteolytic processing of the virus glycoproteins, a trans-dominant negative inhibition of normal virus formation, and interference in virus shutoff of host-cell protein synthesis. Virology 193:424–432.
463. Schlesinger, S. 1993. Personal communication.
464. Schlesinger, S., A. H. Koyama, C. Malfer, S. L. Gee, and M. J. Schlesinger. 1985. The effects of inhibitors of glucosidase I on the formation of Sindbis virus. Virus Res. 2:139–149.
465. Schlesinger, S., R. Lewis, B. G. Weiss, M. Tsiang, and H. Huang. 1987. Replication and packaging sequences in defective interfering RNAs of Sindbis virus. UCLA Symp. Mol. Cell. Biol. New Ser. 54:241–250.
466. Schlesinger, S., and M. J. Schlesinger (ed.). 1986. The Togaviridae and Flaviviridae. Plenum Publishing Corp., New York.
467. Schlesinger, S., and B. G. Weiss. 1986. Defective RNAs of alphaviruses, p. 149–166. In S. Schlesinger and M. J. Schlesinger (ed.), The Togaviridae and Flaviviridae. Plenum Publishing Corp., New York.
468. Schmaljohn, A. L. Unpublished results.
469. Schmaljohn, A. L., E. D. Johnson, J. M. Dalrymple, and G. A. Cole. 1982. Nonneutralizing monoclonal antibodies can prevent lethal alphavirus encephalitis. Nature (London) 297:70–72.
470. Schmaljohn, A. L., K. M. Kokubun, and G. A. Cole. 1983. Protective monoclonal antibodies define maturation and pH-dependent antigenic changes in Sindbis virus E1 glycoprotein. Virology 130:144–154.
471. Schmid, S., R. Fuchs, M. Kielian, A. Helenius, and I. Mellman. 1989. Acidification of endosome subpopulations in wild-type Chinese hamster ovary cells and temperature-sensitive acidification-defective mutants. J. Cell Biol. 108:1291–1300.
472. Schmid, S. L., R. Fuchs, P. Male, and I. Mellman. 1988. Two distinct subpopulations of endosomes involved in membrane recycling and transport to lysosomes. Cell 52:73–83.
473. Schmidt, M., M. F. G. Schmidt, and R. Rott. 1988. Chemical identification of cysteine as palmitoylation site in a transmembrane protein (Semliki Forest virus E1). J. Biol. Chem. 263: 18635–18639.
474. Schmidt, M. F. G. 1982. Acylation of viral spike glycoproteins, a feature of envelope RNA viruses. Virology 116:327–338.
475. Schmidt, M. F. G., M. Bracha, and M. J. Schlesinger. 1979. Evidence for covalent attachment of fatty acids to Sindbis virus glycoproteins. Proc. Natl. Acad. Sci. USA 76:1687–1691.
476. Schmidt, M. F. G., and M. J. Schlesinger. 1980. Relation of fatty acid attachment to the translation and maturation of vesicular stomatitis and Sindbis virus membrane glycoproteins. J. Biol. Chem. 255:3334–3339.
477. Schoepp, R. J., and R. E. Johnston. 1993. Directed mutagenesis of a Sindbis virus pathogenesis site. Virology 193:149–159.
478. Schoepp, R. J., and R. E. Johnston. 1993. Sindbis virus pathogenesis: phenotypic reversion of an attenuated strain to virulence by second-site intragenic suppressor mutations. J. Gen. Virol. 74:1691–1695.
479. Scott, T. W., and S. C. Weaver. 1989. Eastern equine encephalomyelitis virus: epidemiology and evolution of mosquito transmission. Adv. Virus Res. 37:277–328.

480. Sefton, B. M. 1977. Immediate glycosylation of Sindbis virus membrane proteins. Cell 10:659–668.
481. Shah, K. V., H. N. Johnson, T. R. Rao, P. K. Rajagopalan, and B. S. Lamba. 1960. Isolation of five strains of Sindbis virus in India. Ind. J. Med. Res. 48:300–308.
482. Shenk, T. E., K. A. Koshelnyk, and V. Stollar. 1974. Temperature-sensitive virus from *Aedes albopictus* cells chronically infected with Sindbis virus. J. Virol. 13:439–447.
483. Sherman, L. A., and D. E. Griffin. 1990. Pathogenesis of encephalitis induced in newborn mice by virulent and avirulent strains of Sindbis virus. J. Virol. 64:2041–2046.
484. Shirako, Y. 1994. Unpublished data.
485. Shirako, Y., B. Niklasson, J. M. Dalrymple, E. G. Strauss, and J. H. Strauss. 1991. Structure of the Ockelbo virus genome and its relationship to other Sindbis viruses. Virology 182:753–764.
486. Shirako, Y., and J. H. Strauss. 1990. Cleavage between nsP1 and nsP2 initiates the processing pathway of Sindbis virus nonstructural polyprotein P123. Virology 177:54–64.
487. Shirako, Y., and J. H. Strauss. 1994. Regulation of Sindbis virus RNA replication: uncleaved P123 and nsP4 function in minus strand RNA synthesis whereas cleaved products from P123 are required for efficient plus strand RNA synthesis. J. Virol. 68: 1874–1885.
488. Shope, R. E. 1980. Medical significance of togaviruses: An overview of diseases caused by togaviruses in man and in domestic and wild vertebrate animals, p. 47–82. In R. W. Schlesinger (ed.), The togaviruses: biology, structure, replication. Academic Press, Inc., New York.
488a. Simizu, B. 1984. Inhibition of host cell macromolecular synthesis following togavirus infection, p. 465–499. In H. Fraenkel-Conrat and R. R. Wagner (ed.), Comprehensive virology, vol. 19. Plenum Publishing Corp., New York.
489. Simmons, D. T., and J. H. Strauss. 1974. Translation of Sindbis virus 26S RNA and 49S RNA in lysates of rabbit reticulocytes. J. Mol. Biol. 86:397–409.
490. Singh, I., and A. Helenius. 1992. Nucleocapsid uncoating during entry of enveloped RNA viruses into cells. Semin. Virol. 3:511–518.
491. Singh, I., and A. Helenius. 1992. Role of ribosomes in Semliki Forest virus nucleocapsid uncoating. J. Virol. 66:7049–7058.
492. Smeeakens, S. P. 1993. Processing of protein precursors by a novel family of subtilisin-related mammalian endoproteases. Bio/Technology 11:182–186.
493. Smeeakens, S. P., A. G. Montag, G. Thomas, C. Albiges-Rizo, R. Carroll, M. Benig, L. A. Phillips, S. Martin, S. Ohagi, P. Gardner, H. H. Swift, and D. F. Steiner. 1992. Proinsulin processing by the subtilisin-related proprotein convertases furin, PC2, and PC3. Proc. Natl. Acad. Sci. USA 89:8822–8826.
494. Smith, A. L., and G. H. Tignor. 1980. Host cell receptors for two strains of Sindbis virus. Arch. Virol. 66:11–26.
495. Smith, D. B., and S. C. Inglis. 1987. The mutation rate and variability of eukaryotic viruses: an analytical review. J. Gen. Virol. 68:2729–2740.
496. Sneider, J. M., R. M. Kinney, K. R. Tsuchiya, and D. W. Trent. 1993. Molecular evidence that epizootic Venezuelan equine encephalitis (VEE) I-AB viruses are not evolutionary derivatives of enzootic VEE subtype I-E or II viruses. J. Gen. Virol. 74:519–523.
497. Söderlund, H., L. Kääriäinen, C.-H. V. Bonsdorff, and P. Weckstein. 1972. Properties of Semliki Forest virus nucleocapsid II: an irreversible contraction by acid pH. Virology 47:753–760.
498. Sreevalsan, T., and F. H. Yin. 1969. Sindbis virus induced viral ribonucleic acid polymerase. J. Virol. 3:599–604.
499. Stamm, D. D., and R. J. Newman. 1963. Evidence of southward transport of arboviruses from the U.S. by migratory birds. An. Microbiol. 11:123–133.
500. Stanley, J., S. J. Cooper, and D. E. Griffin. 1985. Alphavirus neurovirulence: monoclonal antibodies discriminating wild-type from neuroadapted Sindbis virus. J. Virol. 56:110–119.
501. Stanley, J., S. J. Cooper, and D. E. Griffin. 1986. Monoclonal antibody cure and prophylaxis of lethal Sindbis virus encephalitis in mice. J. Virol. 58:107–115.
502. Stec, D. S., A. Waddell, C. S. Schmaljohn, G. A. Cole, and A. L.

Schmaljohn. 1986. Antibody-selected variation and reversion in Sindbis virus neutralization epitopes. J. Virol. 57:715–720.
503. Stegmann, T., J. M. Delfino, F. M. Richards, and A. Helenius. 1991. The HA2 subunit of influenza hemagglutinin inserts into the target membrane prior to fusion. J. Biol. Chem. 266:18404–18410.
504. Stegmann, T., R. W. Doms, and A. Helenius. 1989. Protein-mediated membrane fusion. Annu. Rev. Biophys. Biophys. Chem. 18:187–211.
505. Stegmann, T., P. Schoen, R. Bron, J. Wey, I. Bartoldus, A. Ortiz, J.-L. Nieva, and J. Wilschut. 1993. Evaluation of viral membrane fusion assays: comparison of the octadecylrhodamine dequenching assay with the pyrene excimer assay. Biochemistry 32:11330–11337.
506. Steiner, D. F., S. P. Smeeakens, S. Ohag, and S. J. Chan. 1992. The new enzymology of precursor processing endoproteases. J. Biol. Chem. 267:23435–23438.
507. Steinhauer, D. A., E. Domingo, and J. J. Holland. 1992. Lack of evidence for proofreading mechanisms associated with an RNA virus polymerase. Gene 122:281–288.
508. Steinhauer, D. A., and J. J. Holland. 1987. Rapid evolution of RNA viruses. Annu. Revs. Microbiol. 41:409–433.
509. Stollar, V. 1987. Approaches to the study of vector specificity for arboviruses—model systems using cultured mosquito cells. Adv. Virus Res. 33:327–365.
510. Strauss, E. G. 1978. Mutants of Sindbis virus. III. Host polypeptides present in purified HR and *ts*103 virus particles. J. Virol. 28:466–474.
511. Strauss, E. G., C. R. Birdwell, E. M. Lenches, S. E. Staples, and J. H. Strauss. 1977. Mutants of Sindbis virus. II. Characterization of a maturation-defective mutant, *ts* 103. Virology 82:122–149.
512. Strauss, E. G., R. J. de Groot, R. Levinson, and J. H. Strauss. 1992. Identification of the active site residues in the nsP2 proteinase of Sindbis virus. Virology 191:932–940.
513. Strauss, E. G., E. M. Lenches, and M. A. Stamreich-Martin. 1980. Growth and release of several alphaviruses in chick and BHK cells. J. Gen. Virol. 49:297–307.
514. Strauss, E. G., E. M. Lenches, and J. H. Strauss. 1976. Mutants of Sindbis virus. I. Isolation and partial characterization of 89 new temperature-sensitive mutants. Virology 74:154–168.
515. Strauss, E. G., R. Levinson, C. M. Rice, J. M. Dalrymple, and J. H. Strauss. 1988. Nonstructural proteins nsP3 and nsP4 of Ross River and O’Nyong-nyong viruses: sequence and comparison with those of other alphaviruses. Virology 164:265–274.
516. Strauss, E. G., C. M. Rice, and J. H. Strauss. 1983. Sequence coding for the alphavirus nonstructural proteins is interrupted by an opal termination codon. Proc. Natl. Acad. Sci. USA 80:5271–5275.
517. Strauss, E. G., C. M. Rice, and J. H. Strauss. 1984. Complete nucleotide sequence of the genomic RNA of Sindbis virus. Virology 133:92–110.
518. Strauss, E. G., A. L. Schmaljohn, D. E. Griffin, and J. H. Strauss. 1987. Structure-function relationships in the glycoproteins of alphaviruses. UCLA Symp. Mol. Cell. Biol. New Ser. 54:365–378.
519. Strauss, E. G., A. L. Schmaljohn, D. S. Stec, and J. H. Strauss. 1990. Mapping of Sindbis virus neutralization epitopes, p. 305–308. In M. A. Brinton and F. X. Heinz (ed.), New aspects of positive-strand RNA viruses. American Society for Microbiology, Washington, D.C.
520. Strauss, E. G., D. S. Stec, A. L. Schmaljohn, and J. H. Strauss. 1991. Identification of antigenically important domains in the glycoproteins of Sindbis virus by analysis of antibody escape variants. J. Virol. 65:4654–4664.
521. Strauss, E. G., and J. H. Strauss. 1980. Mutants of alphaviruses: genetics and physiology, p. 393–426. In R. W. Schlesinger (ed.), The togaviruses: biology, structure, replication. Academic Press, Inc., New York.
522. Strauss, E. G., and J. H. Strauss. 1983. Replication strategies of the single stranded RNA viruses of eukaryotes. Curr. Top. Microbiol. Immunol. 105:1–98.
523. Strauss, E. G., and J. H. Strauss. 1985. Assembly of enveloped animal viruses, p. 205–234. In S. Casjens (ed.), Virus structure and assembly. Jones and Bartlett Publishers, Boston.

524. Strauss, E. G., and J. H. Strauss. 1986. Structure and replication of the alphavirus genome, p. 35–90. *In* S. Schlesinger and M. J. Schlesinger (ed.), The Togaviridae and Flaviviridae. Plenum Publishing Corp., New York.

525. Strauss, E. G., J. H. Strauss, and A. J. Levine. 1991. Viral evolution, p. 167–190. *In* B. N. Fields and D. M. Knipe (ed.), Fundamental virology. Raven Press, New York.

526. Strauss, J. H. (ed.). 1990. Viral Proteinases. Semin. Virol. 1:307–384.

527. Strauss, J. H. 1993. Recombination in the evolution of RNA viruses, p. 241–251. *In* S. S. Morse (ed.), Emerging viruses. Oxford University Press, New York.

528. Strauss, J. H., B. W. Burge, and J. E. Darnell. 1969. Sindbis virus infection of chick and hamster cells: synthesis of virus-specific proteins. Virology 37:367–376.

529. Strauss, J. H., and E. G. Strauss. 1976. Togaviruses, p. 111–166. *In* D. P. Nayak (ed.), The molecular biology of animal viruses. Marcel Dekker, Inc., New York.

530. Strauss, J. H., and E. G. Strauss. 1988. Evolution of RNA viruses. Annu. Rev. Microbiol. 42:657–683.

531. Strauss, J. H., and E. G. Strauss. 1990. Alphavirus proteinases. Semin. Virol. 1:347–356.

532. Strauss, J. H., E. G. Strauss, C. S. Hahn, Y. S. Hahn, R. Galler, W. R. Hardy, and C. M. Rice. 1987. Replication of alphaviruses and flaviviruses: proteolytic processing of polyproteins. UCLA Symp. Mol. Cell. Biol. New Ser. 54:209–225.

533. Stubbs, M. J., A. Miller, P. J. H. Sizer, J. R. Stephenson, and A. J. Crooks. 1991. X-ray solution scattering of Sindbis virus: changes in conformation induced at low pH. J. Mol. Biol. 221:39–42.

534. Subak-Sharpe, I., H. Dyson, and J. Fazakerley. 1993. In vivo depletion of CD8⁺ T cells prevents lesions of demyelination in Semliki Forest virus infection. J. Virol. 67:7269–7633.

535. Suomalainen, M., and H. Garoff. 1992. Alphavirus spike-nucleocapsid interaction and network antibodies. J. Virol. 66:5106–5109.

536. Suomalainen, M., P. Liljeström, and H. Garoff. 1992. Spike protein-nucleocapsid interactions drive the budding of alphaviruses. J. Virol. 66:4737–4747.

537. Symington, J., and M. J. Schlesinger. 1975. Isolation of a Sindbis virus variant by passage on mouse plasmacytoma cells. J. Virol. 15:1037–1041.

538. Symington, J., and M. J. Schlesinger. 1978. Characterization of a Sindbis virus variant with altered host range. Arch. Virol. 58:127–136.

539. Tahara, S. M., E. Darzynkiewicz, J. Stepinski, I. Ekiel, Y. Lin, C. S. Hahn, J. H. Strauss, T. Sujiwade, and D. Haber. 1990. Effects of hypermethylated and aberrant 5′ caps on translation of β-globin and Sindbis virus 26S mRNAs. UCLA Symp. Mol. Cell. Biol. New Ser. 128:67–82.

540. Takkinen, K. 1986. Complete nucleotide sequence of the nonstructural protein genes of Semliki Forest virus. Nucleic Acids Res. 14:5667–5682.

541. Takkinen, K., J. Peränen, and L. Kääriäinen. 1991. Proteolytic processing of Semliki Forest virus-specific non-structural polyprotein. J. Gen. Virol. 72:1627–1633.

542. Takkinen, K., J. Peränen, S. Keränen, H. Söderlund, and L. Kääriäinen. 1990. The Semliki Forest virus-specific nonstructural protein nsP4 is an autoproteinase. Eur. J. Biochem. 189:33–38.

543. Tatem, J., and V. Stollar. 1989. Effect of Sindbis virus infection on induction of heat shock proteins in *Aedes albopictus* cells. J. Virol. 63:992–996.

544. Theilmann, D., B. T. Eaton, and A. E. Downe. 1984. Properties of Sindbis virus variants from infected *Culex tarsalis* mosquitoes. J. Gen. Virol. 65:945–953.

545. Tobias, J. W., T. E. Shrader, G. Rocap, and A. Varshavsky. 1991. The N-end rule in bacteria. Science 254:1374–1377.

546. Tong, L., G. Wengler, and M. G. Rossmann. 1993. Refined structure of Sindbis virus core protein and comparison with other chymotrypsin-like serine proteinase structures. J. Mol. Biol. 230:228–247.

547. Tooze, J., M. Hollinshead, R. Frank, and B. Burke. 1987. An antibody specific for an endoproteolytic cleavage site provides evidence that pro-opiomelanocortin is packaged into secretory granules in AtT20 cells before its cleavage. J. Cell Biol. 105:155–162.

548. Torrisi, M. R., and S. Bonatti. 1985. Immunocytochemical study of the partition and distribution of Sindbis virus glycoproteins in freeze-fractured membranes of infected baby hamster kidney cells. J. Cell Biol. 101:300–306.

549. Trent, D. W., and J. A. Grant. 1980. A comparison of New World alphaviruses in the Western equine encephalitis virus complex by immunochemical and oligonucleotide fingerprint techniques. J. Gen. Virol. 47:261–282.

550. Tsiang, M., S. S. Monroe, and S. Schlesinger. 1985. Studies of defective interfering RNAs of Sindbis virus with and without tRNA<sup>Asp</sup> sequences at their 5′ termini. J. Virol. 54:38–44.

551. Tsiang, M., B. G. Weiss, and S. Schlesinger. 1988. Effects of 5′-terminal modifications on the biological activity of defective interfering RNAs of Sindbis virus. J. Virol. 62:47–53.

552. Tucker, P. C., and D. E. Griffin. 1991. Mechanism of altered Sindbis virus neurovirulence associated with a single-amino-acid change in the E2 glycoprotein. J. Virol. 65:1551–1557.

553. Tucker, P. C., E. G. Strauss, R. J. Kuhn, J. H. Strauss, and D. E. Griffin. 1993. Viral determinants of age-dependent virulence of Sindbis virus for mice. J. Virol. 67:4605–4610.

553a. Tuomi, K., L. Kääriäinen, and H. Söderlund. 1975. Quantitation of Semliki Forest virus RNAs in infected cells using <sup>32</sup>P equilibrium labeling. Nucleic Acids Res. 2:555–565.

554. Turell, M. J. 1988. Horizontal and vertical transmission of viruses by insect and tick vectors, p. 127–152. *In* T. P. Monath (ed.), The arboviruses: epidemiology and ecology. CRC Press, Inc., Boca Raton, Fla.

555. Ubol, S., and D. E. Griffin. 1991. Identification of a putative alphavirus receptor on mouse neural cells. J. Virol. 65:6913–6921.

556. Ubol, S., P. C. Tucker, D. E. Griffin, and J. M. Hardwick. Neurovirulent strains of alphavirus induce apoptosis in bcl-2-expressing cells: role of a single amino change in the E2 glycoprotein. Proc. Natl. Acad. Sci. USA, in press.

557. Ulmanen, I., H. Söderlund, and L. Kääriäinen. 1976. Semliki Forest virus capsid protein associates with the 60S ribosomal subunit in infected cells. J. Virol. 20:203–210.

558. Ulmanen, I., H. Soderlund, and L. Kaariainen. 1979. Role of protein synthesis in the assembly of Semliki Forest virus nucleocapsid. Virology 99:265–276.

559. Ulug, E. T., and H. R. Bose. 1985. Effect of tunicamycin on the development of the cytopathic effect in Sindbis virus-infected cells. Virology 143:546–557.

560. Ulug, E. T., R. F. Garry, and H. R. Bose. 1989. The role of monovalent cation transport in Sindbis virus maturation and release. Virology 172:42–50.

561. Ulug, E. T., R. F. Garry, M. R. F. Waite, and H. R. Bose. 1984. Alterations in monovalent cation transport in Sindbis virus-infected chick cells. Virology 132:118–130.

562. van Duijn, L. P., M. Kasperaitis, C. Ameling, and H. O. Voorman. 1986. Additional methylation at the N(2)-position of the cap of 26S Semliki Forest virus late mRNA and initiation of translation. Virus Res. 5:61–66.

563. Van Steeg, H., M. Kasperaitis, H. O. Voorma, and R. Benne. 1984. Infection of neuroblastoma cells by Semliki Forest virus: the interference of viral capsid protein with the binding of host messenger RNAs into initiation complexes is the cause of the shut-off of host protein synthesis. Eur. J. Biochem. 138:473–478.

564. van Steeg, H., A. Thomas, S. Verbeek, M. Kasperaitis, H. O. Voorma, and R. Benne. 1981. Shutoff of neuroblastoma cell protein synthesis by Semliki Forest virus: loss of ability of crude initiation factors to recognize early Semliki Forest virus and host mRNAs. J. Virol. 38:728–736.

565. Varshavsky, A. 1992. The N-end rule. Cell 69:725–735.

566. Vaux, D. J., A. Helenius, and I. Mellman. 1988. Spike-nucleocapsid interaction in Semliki Forest virus reconstructed using network antibodies. Nature (London) 336:36–42.

567. Vénien-Bryan, C., and S. D. Fuller. 1994. The organization of the spike complex of Semliki Forest virus. J. Mol. Biol. 236:572–583.

568. Vogel, R. H., S. W. Provencher, C. H. von Bonsdorff, M. Adrian, and J. Dubochet. 1986. Envelope structure of Semliki Forest virus reconstructed from cryoelectron micrographs. Nature (Lon-

don) 320:533–535.

569. Volchkov, V. E., V. A. Volchkova, and S. V. Netesov. 1991. Complete nucleotide sequence of the Eastern equine encephalomyelitis virus genome. Mol. Genet. Mikrobiol. Virusol. 5:8–15.

570. von Bonsdorff, C.-H., and S. C. Harrison. 1975. Sindbis virus glycoproteins form a regular icosahedral surface lattice. J. Virol. 16:141–145.

571. von Bonsdorff, C.-H., and S. C. Harrison. 1978. Hexagonal glycoprotein arrays from Sindbis virus membranes. J. Virol. 28:578–583.

572. Vrati, S., S. G. Faragher, R. C. Weir, and L. Dalgarno. 1986. Ross River virus mutant with a deletion in the E2 gene: properties of the virion, virus-specific macromolecule synthesis, and attenuation of virulence for mice. Virology 151:222–232.

573. Vrati, S., C. A. Fernon, L. Dalgarno, and R. C. Weir. 1988. Location of a major antigenic site involved in Ross River virus neutralization. Virology 162:346–353.

574. Wahlberg, J. M., W. A. M. Boere, and H. Garoff. 1989. The heterodimeric association between the membrane proteins of Semliki Forest virus changes its sensitivity to low pH during virus maturation. J. Virol. 63:4991–4997.

575. Wahlberg, J. M., R. Bron, J. Wilschut, and H. Garoff. 1992. Membrane fusion of Semliki Forest virus involves homotrimers of the fusion protein. J. Virol. 66:7309–7318.

576. Wahlberg, J. M., and H. Garoff. 1992. Membrane fusion process of Semliki Forest virus I: low pH-induced rearrangement in spike glycoprotein quaternary structure precedes virus penetration into cells. J. Cell Biol. 116:339–348.

577. Waite, M. R. F., D. T. Brown, and E. R. Pfefferkorn. 1972. Inhibition of Sindbis virus release by media of low ionic strength: an electron microscope study. J. Virol. 10:537–544.

578. Waite, M. R. F., and E. R. Pfefferkorn. 1970. Inhibition of Sindbis virus production by media of low ionic strength: intracellular events and requirements for reversal. J. Virol. 5:60–71.

579. Walton, T. E., and M. A. Grayson. 1989. Venezuelan equine encephalomyelitis, p. 203–231. *In* T. P. Monath (ed.), The arboviruses: epidemiology and ecology. CRC Press, Inc., Boca Raton, Fla.

580. Wang, K.-S., R. J. Kuhn, E. G. Strauss, S. Ou, and J. H. Strauss. 1992. High-affinity laminin receptor is a receptor for Sindbis virus in mammalian cells. J. Virol. 66:4992–5001.

581. Wang, K.-S., R. J. Kuhn, and H. Zhang. 1992. Unpublished data.

582. Wang, K.-S., A. L. Schmaljohn, R. J. Kuhn, and J. H. Strauss. 1991. Antiidiotypic antibodies as probes for the Sindbis virus receptor. Virology 181:694–702.

583. Wang, K.-S., and J. H. Strauss. 1991. Use of a λgt11 expression library to localize a neutralizing antibody-binding site in glycoprotein E2 of Sindbis virus. J. Virol. 65:7037–7040.

584. Wang, Y.-F., S. G. Sawicki, and D. L. Sawicki. 1991. Sindbis virus nsP1 functions in negative-strand RNA synthesis. J. Virol. 65: 985–988.

585. Watson, D. G., J. M. Moehring, and T. J. Moehring. 1991. A mutant CHO-K1 strain with resistance to *Pseudomonas* exotoxin A and alphaviruses fails to cleave Sindbis virus glycoprotein PE2. J. Virol. 65:2332–2339.

586. Weaver, S. C. Evolution of alphaviruses. *In* C. H. Calisher, A. J. Gibbs, and F. Garcia-Arenals (ed.), Molecular basis of viral evolution, in press. Cambridge University Press, Cambridge.

587. Weaver, S. C. 1994. Personal communication.

588. Weaver, S. C., L. A. Bellew, L. Gousset, P. M. Repik, T. W. Scott, and J. J. Holland. 1993. Diversity within natural populations of Eastern equine encephalomyelitis virus. Virology 195:700–709.

589. Weaver, S. C., L. A. Bellew, and R. Rico-Hesse. 1992. Phylogenetic analysis of alphaviruses in the Venezuelan equine encephalitis complex and identification of epizootic viruses. Virology 191:282–290.

590. Weaver, S. C., A. Hagenbaugh, L. A. Bellew, and C. H. Calisher. 1992. Genetic characterization of an antigenic subtype of Eastern equine encephalomyelitis virus. Arch. Virol. 127:305–314.

591. Weaver, S. C., A. Hagenbaugh, L. A. Bellew, L. Gousset, V. Mallampalli, J. J. Holland, and T. W. Scott. 1994. Evolution of alphaviruses in the Eastern equine encephalomyelitis complex. J. Virol. 68:158–169.

592. Weaver, S. C., A. Hagenbaugh, L. A. Bellew, S. V. Netesov, V. E. Volchkov, G.-J. J. Chang, D. K. Clarke, L. Gousset, T. W. Scott, D. W. Trent, and J. J. Holland. 1993. A comparison of the nucleotide sequences of Eastern and Western equine encephalomyelitis viruses with those of other alphaviruses and related RNA viruses. Virology 197:375–390.

593. Weaver, S. C., L. H. Lorenz, and T. W. Scott. 1992. Pathologic changes in the midgut of *Culex tarsalis* following infection with Western equine encephalomyelitis virus. Am. J. Trop. Med. Hyg. 47:691–701.

594. Weaver, S. C., R. Rico-Hesse, and T. W. Scott. 1992. Genetic diversity and slow rates of evolution in New World alphaviruses. Curr. Top. Microbiol. Immunol. 176:99–117.

595. Weaver, S. C., T. W. Scott, L. H. Lorenz, K. Lerdthusnee, and W. S. Romoser. 1988. Togavirus-associated pathologic changes in the midgut of a natural mosquito vector. J. Virol. 62:2083–2090.

596. Weaver, S. C., T. W. Scott, L. H. Lorenz, and P. M. Repik. 1991. Detection of Eastern equine encephalomyelitis virus deposition in *Culiseta melanura* following ingestion of radiolabeled virus in blood meals. Am. J. Trop. Med. Hyg. 44:250–259.

597. Weaver, S. C., T. W. Scott, and R. Rico-Hesse. 1991. Molecular evolution of Eastern equine encephalomyelitis virus in North America. Virology 182:774–784.

598. Weir, R. C., and R. J. Kuhn. 1993. Unpublished data.

599. Weiss, B., H. Nitschko, I. Ghattas, R. Wright, and S. Schlesinger. 1989. Evidence for specificity in the encapsidation of Sindbis RNAs. J. Virol. 63:5310–5318.

600. Weiss, B., and S. Schlesinger. 1981. Defective-interfering particles of Sindbis virus do not interfere with the homologous virus obtained from persistently infected BHK cells but do interfere with Semliki Forest virus. J. Virol. 37:840.

601. Weiss, B. G., and S. Schlesinger. 1991. Recombination between Sindbis virus RNAs. J. Virol. 65:4017–4025.

602. Wellink, J., and A. van Kammen. 1988. Proteases involved in the processing of viral polyproteins. Arch. Virol. 98:1–26.

603. Wen, D., and M. J. Schlesinger. 1986. Regulated expression of Sindbis and vesicular stomatitis virus glycoproteins in *Saccharomyces cerevisiae*. Proc. Natl. Acad. Sci. USA 83:3639–3643.

604. Wengler, G. 1980. Effects of alphaviruses on host cell macromolecular synthesis, p. 459–472. *In* R. W. Schlesinger (ed.), The togaviruses: biology, structure, replication. Academic Press, Inc., New York.

605. Wengler, G. 1987. The mode of assembly of alphavirus cores implies a mechanism for the disassembly of the cores in the early stages of infection. Arch. Virol. 94:1–14.

606. Wengler, G., U. Boege, G. Wengler, H. Bischoff, and K. Wahn. 1982. The core protein of the alphavirus Sindbis virus assembles into core-like nucleoproteins with the viral genome RNA and with other single-stranded nucleic acids *in vitro*. Virology 118: 401–410.

607. Wengler, G., and G. Wengler. 1975. Studies on the synthesis of viral RNA-polymerase-template complexes in BHK 21 cells infected with Semliki Forest virus. Virology 66:322–326.

608. Wengler, G., and G. Wengler. 1984. Identification of a transfer of viral core protein to cellular ribosomes during the early stages of alphavirus infection. Virology 134:435.

609. Wengler, G., G. Wengler, U. Boege, and K. Wahn. 1984. Establishment and analysis of a system which allows assembly and disassembly of alphavirus core-like particles under physiological condition *in vitro*. Virology 132:401–412.

610. Wengler, G., G. Wengler, and H. J. Gross. 1979. Replicative form of Semliki Forest virus RNA contains an unpaired guanosine. Nature (London) 282:754–756.

611. Wengler, G., G. Wengler, and H. J. Gross. 1982. Terminal sequences of Sindbis virus-specific nucleic acids: identity in molecules synthesized in vertebrate and insect cells and characteristic properties of the replicative form RNA. Virology 123: 273–283.

612. Wengler, G., D. Würkner, and G. Wengler. 1992. Identification of a sequence element in the alphavirus core protein which mediates interaction of cores with ribosomes and the disassembly of cores. Virology 191:880–888.

613. Wewer, U. M., L. A. Liotta, M. Jaye, G. A. Ricca, W. A. Drohan,

A. P. Claysmith, C. N. Rao, P. Wirth, J. Coligan, R. Albrechtsen, M. Mudry, and M. E. Sobel. 1986. Altered levels of laminin receptor mRNA in various human carcinoma cells that have different abilities to bind laminin. Proc. Natl. Acad. Sci. USA 83:7137–7141.

614. Wewer, U. M., G. Taraboletti, M. E. Sobel, R. Albrechtsen, and L. A. Liotta. 1987. Role of laminin receptor in tumor cell migration. Cancer Res. 47:5691–5698.

615. White, J., and A. Helenius. 1980. pH-dependent fusion between the Semliki Forest virus membrane and liposomes. Proc. Natl. Acad. Sci. USA 77:3273–3277.

616. White, J., J. Kartenbeck, and A. Helenius. 1980. Fusion of Semliki Forest virus with the plasma membrane can be induced by low pH. J. Cell Biol. 87:264–272.

617. White, J., K. Matlin, and A. Helenius. 1981. Cell fusion by Semliki Forest, influenza and vesicular stomatitis virus. J. Cell Biol. 89:674–679.

618. White, J. M. 1990. Viral and cellular membrane fusion proteins. Annu. Rev. Physiol. 52:675–697.

619. Willems, W. R., G. Kaluza, C. B. Boschek, H. Barrier, H. Hager, H. J. Schutz, and H. Feistner. 1979. Semliki Forest virus: cause of a fatal case of human encephalitis. Science 203:1127–1129.

620. Wright, H. T., A. W. Marston, and D. J. Goldstein. 1984. A functional role for cysteine disulfides in the transmembrane transport of diphtheria toxin. J. Biol. Chem. 259:1649–1654.

621. Wust, C. J., R. Crombie, and A. Brown. 1987. Passive protection across subgroups of alphaviruses by hyperimmune non-cross-neutralizing anti-Sindbis serum. Proc. Soc. Exp. Biol. Med. 184:56–63.

622. Wust, C. J., J. A. W. Nicholas, D. Fredin, D. C. Dodd, R. J. Brideau, M. E. Levely, and A. Brown. 1989. Monoclonal antibodies that cross-react with the E1 glycoprotein of different alphavirus serogroups: characterization including passive protection in vivo. Virus Res. 13:101–112.

623. Xiong, C., R. Levis, P. Shen, S. Schlesinger, C. M. Rice, and H. V. Huang. 1989. Sindbis virus: an efficient, broad host range vector for gene expression in animal cells. Science 243:1188–1191.

624. Yoshimasa, Y., S. Seino, J. Whittaker, T. Kakehi, A. Kosaki, H. Kuzuya, H. Imura, G. I. Bell, and D. F. Steiner. 1988. Insulin-resistant diabetes due to a point mutation that prevents insulin proreceptor processing. Science 240:784–787.

625. Zavadova, Z., J. Zavada, and R. Weiss. 1977. Unilateral phenotypic mixing of envelope antigens between togaviruses and vesicular stomatitis virus or avian RNA tumor virus. J. Gen. Virol. 37:557–567.

626. Zhao, H., and H. Garoff. 1992. Role of cell surface spikes in alphavirus budding. J. Virol. 66:7089–7095.

627. Ziemiecki, A., and H. Garoff. 1978. Subunit composition of the membrane glycoprotein complex of Semliki Forest virus. J. Mol. Biol. 122:259–269.

628. Ziemiecki, A., H. Garoff, and K. Simons. 1980. Formation of the Semliki Forest virus membrane glycoprotein complexes in the infected cell. J. Gen. Virol. 50:111–123.

629. Zurzolo, C., C. Polistina, M. Saini, R. Gentile, L. Aloj, G. Migliaccio, S. Bonatti, and L. Nitsch. 1992. Opposite polarity of virus budding and of viral envelope glycoprotein distribution in epithelial cells derived from different tissues. J. Cell Biol. 117: 551–564.
