
The Plant Journal (1994) 5(1), 69–80

# Cloning and structural analysis of the anthocyanin pigmentation locus Rt of Petunia hybrida: characterization of insertion sequences in two mutant alleles

Johan Kroon†, Erik Souer, Alja de Graaff, Yongbiao Xue‡, Joseph Mol and Ronald Koes*

Department of Genetics, Vrije Universiteit, Institute for Molecular Biological Sciences, Biocentrum Amsterdam, De Boelelaan 1087, 1081 HV Amsterdam, The Netherlands

## Summary

Anthocyanin biosynthesis in flowers of Petunia hybrida is controlled by the regulatory genes an1, an2 and an11. Seven classes of cDNA clones homologous to transcripts that are down-regulated in an1⁻, an2⁻ and an11⁻ mutants were isolated via differential cDNA cloning. Genetic mapping, antisense RNA experiments and analyses of mutant alleles demonstrated that one class of clones originated from the Rt locus. The rt gene has no introns and encodes a protein with homology to mammalian glucuronosyl transferases and flavonoid 3-O-glucosyltransferase (UF3GT) encoded by the bz1 gene from Zea mays. As the Rt locus controls the rhamnosylation of reddish anthocyanin-3-O-glucosides which is the first in a series of modifications that finally yield magenta or blue/purple coloured anthocyanins, this suggests that rt encodes an anthocyanin rhamnosyl transferase.

Molecular analysis of two mutant rt alleles showed that their expression is blocked by different DNA insertion elements. Mutability of the rt-vu15 allele results from the presence of a 284 bp transposable element (dTph1) in the rt promoter region, causing a block in transcription. The protein coding region of the rt-r27 allele contains a 442 bp insertion (dTph3) resulting in premature polyadenylation of rt transcripts. Although dTph3 cannot transpose, it has sequence characteristics of transposable elements, suggesting that it is a defective member of a new family of transposable elements.

Received 1 March 1993; revised 7 September 1993; accepted 14 September 1993.
*For correspondence (fax +31 20 644 2556).
†Present address: Department of Biological Sciences, University of Durham, Science Laboratories, South Road, Durham, DH1 3LE, UK.
‡Present address: John Innes Centre for Plant Science Research, Colney Lane, Norwich, NR4 7UH, UK.

## Introduction

The action of genes controlling the biosynthesis of anthocyanin pigments can be detected visually. This has facilitated the isolation of numerous mutants, in which pigmentation is blocked or spatially altered. Perhaps the largest collection of loci that control anthocyanin biosynthesis is available in Petunia hybrida. About 30 genes that control pigmentation of specific floral organs have been identified by mutations (see De Vlaming et al., 1984; Wiering and De Vlaming, 1984 for listings), and they can be divided in two different classes. One class of loci controls the activity of a single enzyme from the anthocyanin biosynthetic pathway and they appear to contain the structural gene encoding that enzyme. This includes the Po locus controlling chalcone isomerase (CHI) activity (Van Tunen and Mol, 1987), the An3 locus controlling flavanone 3-hydroxylase (F3H) activity (Britsch et al., 1991), the An6 locus controlling dihydroflavonol 4-reductase (DFR) activity (Beld et al., 1989) and the loci Mt1, Mt2, Mf1 and Mf2 controlling activity of four different anthocyanin methyltransferase (iso)enzymes (Jonsson et al., 1984a) (cf. Figure 1). Complementation experiments in which genes isolated from wild-type petunia lines were introduced into mutants proved that the Po and An6 locus contain the structural genes coding for the enzymes CHI and DFR respectively (Huits et al., unpublished data; Van Tunen et al., 1991).

A second class of loci controls the activity of multiple enzymes of the pathway, indicating that they encode a regulatory factor. This includes the An1, An2 and An11 loci controlling activity of DFR, UDP-glucose: flavonoid 3-O-glucosyltransferase (UF3GT), UDP-glucose: flavonoid 5-O-glucosyltransferase (UF5GT) and four anthocyanin methyl transferases (Beld et al., 1989; Gerats et al., 1984, 1985a; Jonsson et al., 1984b). In an1⁻, an2⁻ and an11⁻ mutants steady-state dfr mRNA levels (Beld et al., 1989) and dfr transcription rates (Quattrocchio et al., 1993) are reduced, indicating that these regulatory genes encode or control a transcription factor. Interestingly these regulatory mutants accumulate dihydroflavonols (Gerats et al., 1982) and express chs, chi and f3h genes (Beld et al., 1989; Quattrocchio et al., 1993), implying that in P. hybrida the first and the second part of the anthocyanin biosynthetic pathway are under a different transcriptional control. In flowers of Antirrhinum majus the

first and second part of the anthocyanin pathway are also under a differential control, but the regulatory division occurs here before *f3h* (Martin et al., 1991). In maize kernels however regulatory genes of the *R* and *C1* family from *Zea mays* appear to control the whole pathway as a single unit (see Dooner et al., 1991 for a review).

To study the molecular basis of this differential gene regulation it is necessary to clone both the regulatory *An* genes and a number of their target genes. Transposon-tagged alleles have been described for the regulatory genes *an1*, *an2* and *an11* as well as for some of their target genes, which can aid in their isolation (Gerats et al., 1989). One difficulty with this approach is that transposable elements in *P. hybrid*a are poorly characterized at the molecular level. Therefore we decided first to clone the target genes via a differential cDNA screening and to characterize mutant alleles. In a second step the transposable elements thus isolated can be used to clone the regulatory genes.

Here we report the isolation of seven classes of cDNA clones homologous to transcripts that are down-regulated in *an1⁻*, *an2⁻* and *an11⁻* mutants. Antisense RNA experiments, RFLP mapping and molecular analysis show that one of these originated from the *Rt* locus. The *Rt* locus controls rhamnosylation of anthocyanins and encodes a protein with homology to several sugar transferases suggesting that it is an anthocyanin-rhamnosyl transferase. A non-autonomous transposable element (*dTph1*) and a transposon-like insertion sequence (*dTph3*) were isolated from two mutant *rt* alleles.

### Results

#### Isolation of a set of *an1* regulated genes

In *Petunia hybrid*a lines with a mutation at the *An1* locus the activity of multiple enzymes from the anthocyanin biosynthetic pathway is strongly reduced (Gerats et al., 1984, 1985a; Jonsson et al., 1984b), as well as the steady-state level of *dfr* mRNA (Beld et al., 1989). Based on these findings we assumed that the *an1* gene regulates the mRNA levels for several anthocyanin biosynthetic genes and that it should be possible to clone the cDNAs for any of these genes using a differential cDNA screening strategy. Therefore a cDNA library was constructed based on mRNA from corolla limb tissue of the line V26 (*An1⁺*). Duplicate filters were taken and hybridized with first strand cDNA from an *An1⁺*(V26) and an *an1⁻* line (W162). Two hundred and seventy plaques out of 30 000 showing substantially stronger hybridization to the *An1⁺* cDNA probe than to the *an1⁻* cDNA probe were identified. Thirty five plaques, which did not hybridize to previously cloned pigmentation genes (*chs*, *chi* and *dfr*) were purified to homogeneity. Pairwise cross-

(a)  
(b)  

L A P T  

dif G  

R27 wt  
W162 an1  
W115 an2  
W134 an1  

dif G  

(c)  

1 2 3 4 5 6  

dif G  

1 2 3 4 5 6  

chs-A  

Figure 2. Organ specificity, temporal regulation and genetic control of difG mRNA levels.  
(a) RNA gel blot analysis of RNA extracted from different organs of the line V26. L = leaf; A = anther; P = petal limb; T = petal tube (pool of all developmental stages). (b) difG transcript levels in petals from different regulatory pigmentation mutants. (c) Accumulation of difG mRNA and chs mRNA in petals during flower development. The numbers above each lane correspond with the developmental stage of the buds (Van Tunen et al., 1988).

hybridizations demonstrated that these 35 clones represented seven distinct classes of genes. To verify if any of these clones is involved in anthocyanin biosynthesis, the largest cDNA clone from each class, further indicated as difA, difC, difE, difF, difG, difH and difI, was characterized with regard to spatial and temporal expression. As an example we show in Figure 2 the organ-specific and developmental expression of difG. The results show that the difG gene is expressed in the limb and tube of the petal, but not in leaves or anthers (Figure 2a). The accumulation of difG during development of the petal limb is similar to that of chs (Figure 2c). In addition, difG expression appears to be controlled by the regulatory pigmentation genes an1, an2 and an11 as the difG mRNA level was down-regulated in petal limbs of the corresponding mutants (Figure 2b). The expression of the six other classes (difA, difC, difE, difF, difH and difI) displayed a spatial, temporal and genetic control similar to difG (data not shown). This indicates that they may represent as yet unidentified clones from anthocyanin biosynthetic genes.

About 25 loci controlling anthocyanin biosynthesis in the flower of P. hybrida have been identified via mutations (De Vlaming et al., 1984; Wiering and De Vlaming, 1984). To test if the different classes of cDNA clones could have originated from one of these loci, we localized the position of the gene corresponding to each cDNA class using restriction fragment length polymorphisms (RFLPs). The F₁ hybrid of the two P. hybrida lines V23 and R51 (VR hybrid) used in these analyses is heterozygous for at least one marker on each of the seven chromosomes which allows chromosomal localizations in

(V23 x R51) x R51  
R51 VR V23  
- ± ± ± ± - - ± - - ± +  

Figure 3. Linkage of difG cDNA and the Rt locus. DNA gel blot of HindIII digested DNA from the lines V23 and R51, an F₁ plant (VR) and backcross progeny (V23xR51)xV23, probed with difG. The allelic status at the Rt locus is indicated above each lane (- = rtrt; ± = Rtrt and + = RtRt). Note that both V23 and R51 contain in addition to the strongly hybridizing difG fragment a second weakly hybridizing fragment of intermediate size. This weakly hybridizing band does not segregate, because it is not polymorphic. Backcross progeny plants in which the hybridization signal of the lowest (R51) band is at least equal to the upper (V23) band were scored heterozygous for the V23 and R51 difG allele. In plants homozygous for the V23 allele of difG strong hybridization is only seen to the uppermost (V23) band and at most a faint hybridization to the somewhat smaller non-polymorphic fragment.

one single backcross with either parent (Wallroth et al., 1986). Among a total of 75 plants from the two backcross progenies [V23xR51] × V23 (35 plants) and [V23xR51] × R51 (40 plants) we detected no cross-overs between the Rt locus on chromosome 6 and the RFLP generated by difG (Figure 3). Since two more genetically defined pigmentation genes, an1 and an2, map very close to this region of chromosome VI (De Vlaming et al., 1984), it is possible that difG originated from the An1, the An2 or the Rt locus. The availability of mutable alleles for an1, an2 as well as rt led us to investigate difG in more detail.

difG cDNAs originate from the Rt locus

One way to identify the function of a cryptic gene is to inactivate its expression in planta by antisense RNA (Van der Krol et al., 1988). To verify if difG encodes a protein that is involved in anthocyanin biosynthesis, an antisense difG RNA expressing construct under the control of the CaMV 35S promoter was introduced into P. hybrida line V26 by Agrobacterium tumefaciens mediated transformation. Figure 4 shows the flowers of some of these transformants. Eight transformants had flowers that were indistinguishable from normal V26 flowers (Figure 4; flower 1), in two other transformants alteration of the colour shade was observed in some sectors of the flower (flowers 2 and 3). Similar pigmentation patterns were observed previously in transgenic plants containing antisense or sense chs and dfr gene constructs (Napoli et al., 1990; Van der Krol et al., 1988, 1990). Analysis of the accumulated flavonoid-aglycones by thin layer chromatography (TLC) showed that corolla limbs of the flowers exhibiting a phenotypic change contained a mixture of delphinidin and petunidin whereas wild-type V26 corolla limbs (V26 is mf1⁻, mf2⁻) only contained petunidin

72 Johan Kroon et al.

(a)  
![Image](image1.png)

(b)  
![Image](image2.png)

Figure 4. Flower pigmentation patterns of transgenic *P. hybrida* V26 plants containing an antisense *difG* cDNA construct.  
(a) Flowers of transgenic *P. hybrida* V26 plants harbouring an antisense *difG* construct display phenotypes ranging from wild-type flowers (flower 1) to flowers with altered pigmentation in sectors varying in size (flowers 2 and 3).  
(b) Thin layer chromatography analysis shows that flowers exhibiting an antisense phenotype contain a mixture of petunidin (P) and delphinidin (D) (lane 1). In wild-type flowers only petunidin is detected (lane 2).

(Figure 4b). This indicates that the antisense *difG* construct blocks one of the reactions in between delphinidin and petunidin, which may be either the rhamnosylation, 5-O-glucosylation, acylation or methylation (cf. Figure 1). The antisense *difG* phenotype differs from that of *an1⁻* and *an2⁻* mutants, which completely lack anthocyanins in the flower limb due to interference with an earlier step in the pathway (De Vlaming et al., 1984; Wiering and De Vlaming, 1973). Therefore it is unlikely that *difG* originated from the *An1* or *An2* locus. The antisense phenotype is similar to that of *rt⁻*, *gf⁻* and *mt1⁻*, *mt2⁻* mutants. As the *Gf* and *Mt1* and *Mt2* loci map on other chromosomes than *difG*, it can be excluded that *difG* originated from these loci. However, both the antisense phenotype and the chromosomal position are consistent with the hypothesis that *difG* originated from the *Rt* locus.

To further test whether *difG* could have originated from the *Rt* locus, we made use of an *rt*-mutable allele originally described by Cornu (1977). This allele was maintained in our laboratory in the line Vu15 and is referred to as *rt-vu15*. We tested whether reversion of the *rt*-mutable phenotype correlated with the excision of a transposable element from the *difG* gene. Among 180 progeny plants, originating from a single seed capsule, obtained by selfing of an *rt*-mutable parent, we found six plants with evenly coloured reddish/purple flowers indicating that excision of an element at the *Rt* locus had occurred in sporogenic cells of the parent. Figure 5 shows a DNA gel blot analysis of DNA from two plants with an *rt*-mutable and two with an *Rt* revertant phenotype. At moderate stringency the *difG* probe detects three bands of 5.1 kb, 3.1 kb and 1.8 kb, respectively. High stringency washing of the blot results in melting of the hybrids with the 5.1 kb band in both the mutant and the revertants indicating that this fragment contains a gene that is related, but not identical, to *difG*. The identity of

![Image](image3.png)

Figure 5. DNA gel blot analysis of *rt*-mutable and revertant plants. Phenotypes of the flowers from the line Vu15 carrying an unstable *rt* allele (I) and a revertant thereof (R) are shown on the left. On the right a DNA gel blot of two plants with an *rt* mutable (I1 and I2) or a revertant *rt* allele (R1 and R2) probed with *difG*. The blot was initially washed at moderate stringency (2×SSC), exposed and subsequently washed at high stringency (0.1×SSC) and exposed again. The size of marker fragments are indicated in the margin.

this latter gene remains unknown at present. The gene from which the *difG* cDNA originated is cut at an internal *BclI* site, resulting in two hybridizing fragments of 1.8 kb and 3.1 kb (cf. Figure 7). Analysis of the two revertant plants reveals a novel band of 1.5 kb, which is smaller than the equivalent band of 1.8 kb present in the *rt*-mutable plants. The same size difference between the original mutable and the revertant allele was found with several other restriction digests and double digests of
the same DNAs (data not shown). Figure 5 shows that the signals of the original (mutable) 1.8 kb band and the novel (revertant) 1.5 kb band in the revertant plants (R1 and R2) are of approximately half of the intensity of the signal for the equivalent 1.8 kb band in the $rt$-mutable plants (I1 and I2). This can be accounted for by heterozygosity for the $Rt^{+}$ revertant allele with the mutable $rt$ allele on the other chromosome. Indeed, progeny obtained by selfing of revertant plants displayed a 3:1 segregation of the Rt-revertant versus the $rt$-mutable phenotype (not shown).

Taken together these data provide direct evidence that $difG$ originated from the $Rt$ locus and that mutability of the $rt-vu15$ allele is caused by excision of a mobile element of about 300 bp.

The $Rt$ locus encodes a protein with homology to mammalian glucuronosyl-transferases and to flavonoid 3-O-glucosyl transferase encoded by the $Bz1$-locus of Zea mays

To analyse the structure of the $Rt$ locus in more detail we screened genomic and cDNA libraries of the wild-type line V26 with $difG$ under high stringency conditions to obtain a full-size cDNA and genomic clones. From a genomic V26 library we obtained four positive clones of which one contained the whole gene colinear with the $rt$ cDNA and a 4.7 kb HindIII/Sall fragment was subcloned into pBluescript and used for further analyses.

Structural analysis of the $rt$ gene was performed by sequencing both strands of part of the genomic subclone and of a full length cDNA clone. Comparison of the genomic and cDNA sequences revealed that the V26 $rt$ gene contains no introns (Figure 6a). Using primer extension analysis we mapped the CAP site of the mRNA at a G residue 48 bp upstream of the translation start codon (data not shown). As the cDNA clone 5' end started at +15 and the region between nucleotide +1 and +15 does not contain a 3' splice site, we can exclude the presence of an intron and conclude that the upstream region represents the $rt$ promoter. The presence of a putative TATA box, TATATAAAT, at position –26 to –18 further supports this conclusion. The original $difG$ cDNA and the full size cDNA clone terminated at two distinct positions with a stretch of A residues, indicating that at least two polyadenylation sites are present in the $rt$ mRNA.

The cDNA sequence contains an open reading frame of 1422 bp which specifies a protein of 473 amino acids. To find out whether the $rt$ gene encodes an enzyme or a regulatory protein, we screened the Genbank data library with the amino acid sequence of the predicted RT polypeptide. Strikingly, the protein with the highest homology to RT was found to be UF3GT: (E.C. 2.4.1.91) encoded by the $bz1$ gene from Z. mays. Figure 6b shows the

region from amino acid 275 to 478 of the RT protein sequence in which the degree of homology was the highest. In addition, this region of RT shares homology with other sugar transferases; in particular the bilirubin UDP-glucuronosyl transferases from human, rat and mouse (Figure 6b). Three distinct sugar transferase are active in the anthocyanin pathway: UF3GT, ART and UF5GT (see Figure 1). We infer that $Rt$ encodes ART, as an $rt^{-}$ mutation blocks rhamnosylation of anthocyanins (Wiering and De Vlaming, 1973), but does not affect UF3GT nor UF5GT enzyme activity (Gerats et al., 1983; Jonsson et al., 1984b) (see Discussion for further details).

Molecular characterization of two mutant $rt$ alleles

Even though mutable alleles have been described for a large number of genes (Doodeman et al., 1984a, 1984b; Gerats et al., 1989), until now only one insertion element ($dTph1$) with sequence features of a transposable element has been identified in $P$. hybridia (Gerats et al., 1990). This led us to characterize the mutable $rt$ allele in the line Vu15 (Cornu, 1977, cf. Figure 5) in more detail. Using various combinations of primers (see Figure 7b) we amplified different regions of the $rt$ gene from plants harbouring the wild-type, the $rt$ mutable and the revertant allele and compared the size of the products. In this way the transposon insertion was mapped in the promoter region of the gene in between primers 1 and 3 (data not shown). Amplification products from the plants carrying the $rt$ mutable allele were cloned in plasmid vectors and sequenced. Comparison with the wild-type sequence (V26) shows that a 284 bp insertion is present in the TATA box of the $rt-vu15$ allele (Figure 7c). This insertion has typical characteristics of a transposable element, such as 12 bp terminal inverted repeats flanked by an 8 bp target site duplication. In addition the element is highly homologous to $dTph1$ (three mismatches out of 284 bp), a small insertion sequence isolated from the $dfrC$ locus (Gerats et al., 1990).

RNA gel blot analysis demonstrated that $rt$-mutable flowers contain severely reduced $rt$ mRNA levels compared with the $Rt^{+}$ revertants (Figure 7a), whereas the mRNA levels of two other anthocyanin biosynthetic genes, $chs$ and $f3h$ were similar (data not shown). At low intensity however, a slightly longer $rt$ mRNA is detected in the $rt$-mutable plants. It may be that this RNA originated from an alternative, less efficient, transcription start site within the $dTph1$ element. Alternatively it may have derived from the second gene with homology to the $rt$ probe.

A large number of $P$. hybridia lines are available which contain a stably mutated $rt$ allele. Because it was not clear whether they all harbour the same or different $rt^{-}$ alleles we screened several $P$. hybridia lines by RNA gel blot analysis. Figure 7a shows that in all $Rt^{+}$ lines, a tran-
74 Johan Kroon et al.

(a)

TCATTTATGTTTCTGTATATTTACCAACGAAATGTCACATACTCAT - 200  
TATATTTAATGTACATATTGGTTGCAGGTAATAGTGAAATATACGTATTATCAAATATAACTTTCTTTTTAAAAGAAGTACATAATATATGGATTGT - 151  
CTTTGTTGGTTGGTTGTCCAGCCAAAATGGATTTTTTCTCACTTCACATTTTCATGCCTACTCACCACTCAACTACATCCTGTCTATACTCTTTGTCA - 52  
ACTCCCACTTGACTTCTCATTTGTCQIATAFAAAATGGAAGTTCAATTTATTTTGCATAACCAATAATGAAACGTTGCTCGCAGTATTTAAACAAAGGATAT + 48  
ATG GAG AAT GAG ATG AAG CAC TCA AAT GAT GCA CTT CAT GTG GTA ATG TTC CCA TTT TTT GCT TTT GGC CAT ATT + 123  
M E N E M K H S N D A L H V V M F P F A F G H I 25  
AGT CCA TTT GTG CAG CTT GCT AAC AAG TTG TCC TCT TAT GGT GTC AAA GTT TCT TTC ACA GCA TCT GGC AAT + 198  
S P F V Q L A N K L S S Y G V K V S F F T A S G N 50  
GCC AGC AGA GTC AAA TCT ATG TTA AAT TCT GCT CCC ACT ACT CAT ATA GTC CCT ACA CTC CAT GTT GAA + 273  
A S R V K S M L N S A P T T H I V P L T L P H V E 75  
GGT CTA CCT CGT GCA GAA AGT ACT GCA GAA TTG ACA CCA GCT AGT GCT GAG CTT CTC AAG GTT GCT TTA GAC + 348  
G L P P G A E S T A E L T P A .S A E L L K V A L D 100  
CTA ATG CAA CCA CAA ATC AAG ACT TTA CTT TCC CAT CTC AAA CCC CAT TTT GTT CTC TTT GAT TTT GCT CAA GAA + 423  
L M Q P Q I K T L L S H L K P H F V L F D F A Q E 125  
TGG CTT CCT AAA ATG GCC AAT GGA TTG GGT ATC AAG ACT GTT TAT TAC TCT GTT GTT GCA CTT TCC ACT GCT + 498  
W L P K M A N G L G I K T V Y Y S V V A L S T A 150  
TTT CTT ACT TGT CCT GCT AGA GTT CTT GAA CCC AAA AAG TAT CCA AGT CTC GAA GAC ATG AAG AAA CCT CCA CTT + 573  
F L T C P A R V L E P K K Y P S L E D M K K P P L 175  
GGG TTT CCT CAG ACC TCT GTT ACC TCA GTC AGA ACC TTT GAG GCT AGA GAT TTT CTA TAT GTT TTC AAG AGT TTC + 648  
G F P Q T S V T S R T F E A R D F L Y V F K S F 200  
CAT AAT GGT CCT ACT TTA TAT GAC CGT ATA CAG TCA GGA CTC AGG GGG TGC TCA GCT ATA CTA GCA AAA ACT TGT + 723  
H N G P T L Y D R I Q S G L R G C S A I L A K T C 225  
TCA CAA ATG GAG GGT CCT TAT ATA AAA TAC GTA GAA GCA CAA TTC AAT AAA CCT GTT TTT TCT AAT CGG ACC CGT + 798  
S Q M E G P Y I K Y V E A Q F N K P V F S N R T R 250  
AGT TCC GGA CCC GCT TCG GGT AAA TTG GAA GAG AAA TGG GCT ACT TGG TTA AAC AAG TTT GAA GGT GGA ACA GTT + 873  
S S G P A S G K L E E K W A T W L N K F E G G T V 275  
ATT TAC TGT TCT TTT GGA AGT GAA ACT TTC TTG ACT GAT GAT CAG GTC AAA GAA CTG GCT TTA GGT TTG GAA CAG + 948  
I Y C S F G S E T F L T D D Q V K I A L G I E Q T G L D F F L V N F P A N V D V S A E E N R A glucuronosyltr rt  
ACA GGG CTT CCT TTC TTT GTG TTA AAT TTT CCT GCA AAT GTT GAT GTT TCA GCG GAG CTA AAC CGA GCT TTA + 1023  
T G L P F F L V N F P A N V D V A E L R R L A A G L E A S Q A P F L W S L R - - - - E D 9 W T L - - bz-1  
CCT GAA GGG TTT CTG GAA AGA GTG AAA GAC AAG GGG ATT ATT CAT TCA GGT TGG GTG CAA CAG CAG AAT ATA TTA + 1098  
P E G F L E H V K D R G I - H S G H V Q Q Q N I L A H S S V G C Y V C H A G F S S V I E A L V N glucuronosyltr rt  
GCT CAT TCT AGT GTA GGT TGT TAT GTA TGT CAT GCA GGG TTT AGT TCA GTT ATA GAG GCA CTG GTG AAT GAC TGT + 1173  
A H S S V G C Y V C H A G F S S V I E A L V N D C 375  
CAA GTA GTT ATG TTG CCC CAG AAA GTT GAC CAG ATT TTG AAT GCA AAG CTG GTG AGT GGT GAT ATG GAA GCT GGG + 1248  
Q V V M L P Q K G D Q I L N A K L V S G D M E A G V E I N R R D E D G Y F G K E D I K E A V E K glucuronosyltr rt  
GTG GAG ATT AAT AGG AGG GAT GAA GGT TAT TTT GGT AAA GAA GAT ATT AAG GAA GCT GTG GAA AAG GTG ATG + 1323  
V E I N R R D E G Y F G K E D I K E A V E K V M 425  
GTG GAT GTT GAA AAG GAC CCA GTT AAA TTA ATT AGG GAA AAT CAG AAG AAA TGG AAG GAG TTT CTG TTG AAC AAG + 1398  
V D V E K D P G K L I B E N Q K W K E F L L N K D I Q S K Y I - - G N L V N E M T A M A K V S glucuronosyltr rt  
GAT ATC CAG TCC AAA TAT ATT GGG AAT TTA GTT AAT GAA ATG ACA GCC ATG GCT AAG GTC TCG ACT ACA TAG + 1470  
D I Q S K Y I G N L V N E M T A M A K V S T T Z 474  
GAATCGATGTTCCCAGCATTCTGATGCAACAATTTAGTTAAAACATTATAGACATTATGCCCTATCCTTCCAAGCGAGTTTTTTAATTTAAATTTTTGTGG + 1569  
ACAAGTCC TGAAAGAATGTGGCTGAAAAATGCTGCTATGTGTCAGATAAGGTCACATTTTCATTACTTCTCAAGTTTTGTGGCACAAAATCAGCATAT + 1668  
GATTAATGAAGATGGTCTTTTACCAGAACATTTAAATAAAGGATGAGAGTTTCAGTTTCT + 1715  

(b)

K K A M A I A D A L G K I P Q T V L W glucuronosyltr  
D Q V K I A L G I E Q T G L D F F L V N F P A N V D V S A E E N R A rt  
B E L R R L A A G L E A S Q A P F L W S L R - - - - E D 9 W T L - - bz-1  
R Y T G T R P S N L A N N T - E L V K M L P Q N D L T G B F M T R A F I T H A G S H G V Y # S I C # glucuronosyltr  
L P E G F L E H V K D R G I - H S G H V Q Q Q N I L A H S S V G C Y V C H A G F S S V I E A L V N rt  
L P P G F L D # A A G T G S G L V # P H A P Q V A V L R H P S V G A F V T H A G W A S V L E G V S S bz-1  
G V P M V M M P L J G D Q M D N A X R M - E T K G A G V T L N V L E M T S E D L E N A L K A V I N D glucuronosyltr  
D C Q V V M L P Q K G D Q I L N A K L V S G D M E A G V E I N R R D E D G Y F G K E D I K E A V E K rt  
G V E M A C R P F G D Q R M N A R S V A H V W G F G A A F - - - - E G A M T S A G V A A B V E E bz-1  
K S Y K E N I M R L S S L H K D R P V E P L D L A V F W V E F V M R - - H K G A P H L R P A A H D L glucuronosyltr  
V M V D V E K D P G K L I B E N Q K W K E F L L N K D I Q S K Y I - - G N L V N E M T A M A K V S rt  
L L R G E E - - - G A R M B A R A K - - - - V L Q A L V A E A F G P G G E C R K N F D R F V E I V bz-1  

Figure 6. Nucleotide and deduced amino acid sequence of the gene encoded by the *Rt* locus of *P. hybrid*a line V26.

(a) Nucleotide position +1 represents the CAP site as determined by primer extension analysis and is marked by an arrowhead. The putative TATA sequence is boxed. The arrowheads in the 3' untranslated region mark the sites where poly(A) tails were present in two cDNA clones. The amino acid sequence corresponding to the open reading frame is presented beneath the nucleotide sequence. Numbers in bold refer to amino acid residues.

(b) Sequence alignment of part of the predicted V26 RT protein, a human bilirubin UDP-glucuronosyltransferase (Sato et al., 1990) and the protein encoded by the *Bz1* locus (Furtek et al., 1988; Ralston et al., 1988) from *Z. mays*. Sequence identity is indicated by bold face symbols and dark shading; sequence similarity is indicated by bold face symbols and a lighter shading.

The rt gene from Petunia hybrid 75

mum of about 10% of the intensity of the 1300 bp mRNA; see Discussion). To find out whether these aberrant mRNAs resulted from a rearrangement in the *rt* gene, we performed a PCR analysis on genomic DNA, using primers complementary to the ends of the *rt* coding sequence (primers 1 and 5; Figure 7b). From *Rt* lines the expected 1.5 kb fragment was amplified, whereas from all *rt* lines an approximately 2.0 kb fragment was amplified (data not shown). This indicates that all *rt* lines tested contain the same *rt* allele with an insertion of about 500 bp. The *rt* allele of line R27 was chosen for further analysis. PCR analyses with various combinations of primers showed that the insertion had occurred near the 3′ end of the gene in between primers 4 and 5 (cf. Figure 7b). This region of the R27 *rt* gene was amplified by PCR, cloned, and subsequently sequenced. The sequences at the ends of the insertion in the R27 *rt* gene and the flanking *rt* sequences display typical structural characteristics of transposable elements. Figure 7d shows that the inserted sequence has imperfect 12 bp inverted repeats at its termini and is flanked by 8 bp directly repeated *rt* target sequences. The 12 bp inverted repeats of the insertion element show homology to the termini of *dTph1* and to other transposons of the *Ac/Ds* family (Figure 8). However, the element does not have additional homology to *dTph1*, nor to another transposable element that was isolated recently from *P. hybrid* (dTph2: Van Houwelingen et al., unpublished data). Therefore, we named this insertion sequence *dTph3*.

To investigate how the shortened transcripts are generated by the *dTph3* insertion, we cloned and sequenced two PCR products amplified from reverse transcribed R27 petal RNA via the RACE procedure (Frohman et al., 1988). The sequence of both clones was

Figure 7. Analysis of mutant *rt* alleles.
(a) Left panel: gel blot analysis of RNA from the corolla limbs of *Rt* and stably mutated *rt* lines. The genetic status of the *Rt* locus is indicated above each lane (+ = *RtRt*, − = *rtrt*). Right panel: gel blot analysis of RNA from corolla limbs of mutable (*l*) and revertant flowers of the line Vu15 (*R*).
(b) Map illustrating the combined results of restriction, PCR, and sequence analysis of the *rt*-mutable and the recessive allele. Oligonucleotides used as PCR primers are indicated by numbered arrows (for details see Experimental procedures); the triangles indicate the position of the insertion sequences in the *Rt* gene from petunia Vu15 and R27. The translational start and stop codons and four restriction sites used in cloning procedures are indicated.
(c) Nucleotide sequence of the mobile element (*dTph1-rt*) inserted in the TATA box of the *rt* gene in petunia Vu15. Numbering of the *rt* flanking sequences is as in Figure 6.
(d) Nucleotide sequence of the non-mobile insertion sequence *dTph3* in the coding region of the *rt* gene of the line R27. The position at which a 3′ RACE product terminated in a series of A residues is indicated above the sequence. TSD = target site duplication, TIR = terminal inverted repeat.

script of about 1600 bp was present in petal limbs, whereas in all *rt* lines a shorter transcript of approximately 1300 bp was detected. We occasionally observed hybridizing bands of a larger size than 1600 bp in *rt* mutants. The intensity of this band was always low and it varied even between different RNA preparations of the same line (from undetectable levels (Figure 7) to a maximum of about 10% of the intensity of the 1300 bp mRNA; see Discussion).

Terminal inverted repeats of transposable elements with an 8 bp target site.

Terminal inverted repeat sequences (left column) of various elements (middle column) of different plant species (right column) were compared with *dTph1* and displayed in order of decreasing homology. Identical nucleotides are indicated by a gray shading. For terminal inverted repeat sequences see: Gerats et al. (1990) and Gierl and Saedler (1992) and references therein.

colinear with the R27 genomic sequence but terminated with a stretch of A residues within the *dTph3* sequence (Figure 7d). This indicates that the *dTph3* insertion creates a novel polyadenylation signal and explains the presence of the smaller transcripts found in *rt⁻* lines.

### Discussion

In this paper we report the cloning of seven classes of cDNA clones derived from genes that are under the control of the regulatory pigmentation genes *an1*, *an2* and *an11* from *P. hybrid*. Flowers from *an1⁻* lines contain little or no activity of the enzymes DFR, UF3GT, UF5GT and several anthocyanin methyltransferases, resulting in an absence of anthocyanins in the flower (Beld et al., 1989; Gerats et al., 1984, 1985a; Jonsson et al., 1984b). Since no additional abnormalities can be visually detected in *an1⁻*, *an2⁻* or *an11⁻* flowers it is likely that at least some, and possibly all, of the differential cDNA clones encode proteins that are involved in anthocyanin biosynthesis. For two classes of clones their involvement in anthocyanin biosynthesis has been further established. By genetic mapping (Figure 3), antisense-RNA experiments (Figure 4) and analysis of *rt* mutant alleles (Figures 5 and 6) we demonstrated that *difG* cDNAs originated from the *Rt* locus. Sequence analysis showed that *difA* cDNAs are homologous to the *candi* gene from *A. majus* and the *A2* gene from *Z. mays* (Weiss et al., 1993). Additional tests are in progress to establish the role of the other classes of differentially expressed cDNA clones in anthocyanin biosynthesis. A similar differential screening strategy was successfully used before, to isolate the *f3h* and *candi* genes from *A. majus* (Martin et al., 1991).

The RT protein predicted from the V26 *rt* cDNA sequence has homology with mammalian glucuronosyl transferases and UF3GT (E.C.2.4.1.91) encoded by the *bz1* gene from maize (Figure 6b). In vitro mutagenesis of histo blood group A and B sugar transferases showed that single amino acid substitutions can alter the nucleotide-sugar specificity of the enzyme (Yamamoto and Hakomori, 1990), implying that it is not possible to infer the nucleotide-sugar specificity from the protein sequence alone. Three sugar-transferase enzymes are active in the anthocyanin biosynthetic pathway, UF3GT, ART, UF5GT (Figure 1). Anthocyanins in *rt⁻* lines from *P. hybrid* are normally 3-O-glucosylated but lack both the rhamnose and the 5-O-glucose group and are not methylated (Wiering and De Vlaming, 1973, 1984). The same holds for plants in which *rt* expression was blocked via antisense RNA (Brugliera et al., 1994; this paper). In *gf⁻* mutants rhamnosylation is normal, but 5-O-glucosylation is blocked (Wiering and De Vlaming, 1973). As the levels of UF5GT enzyme activity in wild-type, *rt⁻* and *gf⁻* lines are similar (Jonsson et al., 1984b) this implies (i)

that 5-O-glucosylation is dependent on the presence of a rhamnose and an acyl group and (ii) that the *Rt* locus directly controls the addition of the rhamnose moiety and that its gene product is a rhamnosyltransferase. The molecular weight of the putative *P. hybrid* RT protein (53.4 kDa) corresponds well with that of a biochemically characterized rhamnosyl transferase (monomeric, 52 kDa) involved in production of bitter flavonoid compounds in grapefruit (*Citrus paradisi*) (Bar-Peled et al., 1991).

In DNA gel blot analyses a second gene with homology to the *rt* cDNA was detected (Figure 5). In RNA gel blot analyses of RNA from *Rt⁺* lines, the *rt* cDNA detects a 1.6 kb mRNA whereas in lines with a stably mutated *rt⁻* allele only a 1.3 kb mRNA (and sometimes an RNA larger than 1.6 kb) is detected (Figure 7). This indicates that all of the 1.6 kb mRNA was transcribed from the *Rt* locus and that the second gene is expressed in petal limbs at low level, or not at all. It may be that this second gene is a silent copy of the *rt* gene, like those found for other flavonoid biosynthetic genes in *P. hybrid* (Koes et al., 1989a, 1989b) or that it encodes another sugar-transferase enzyme.

In *P. hybrid* transposition has been extensively studied at the genetic level. Mutable alleles have been described for many genes among which are regulatory pigmentation genes and genes controlling plant development (Doodeman et al., 1984b; Gerats et al., 1989). Genetic analysis of instability at the *An1* locus of the line W138 indicated that a two-element system is involved (Huits et al., unpublished data; Wijsman, 1986). At least one other family of transposable elements appears to be active in *P. hybrid*, as mutability of the *an3* gene of the line W159 depends on a different activator element (Gerats et al., 1985b). Recently a small DNA insertion element (*dTph1*) with sequence characteristics of a defective transposable element was cloned from the *dfrc* gene of the *an1*-mutable line W138 (Gerats et al., 1990). Although the authors stated initially that *dfrc* and *an1* are identical, subsequent experiments clearly showed that they are not (Huits et al., unpublished data). This makes it difficult to assess whether the previously reported *dfrc* excision alleles (Gerats et al., 1990) arose from reversions or that they represent wild-type alleles. The finding of a *dTph1* insertion in the *rt*-mutable allele of *P. hybrid* line Vu15 (re)confirms that *dTph1* is indeed a functional transposable element by all criteria: it causes a variegated phenotype in somatic tissues of the plant as well as reversions among progeny plants due to excision of the element in sporogenic tissues (Figure 5). The insertion of *dTph1* in the *rt*-mutable allele causes a reduction of the steady-state *rt* mRNA level (Figure 7a). Since the element is inserted close to the TATA box, outside the transcribed region of the gene, we assume that the inser-

tion interferes with transcription of the *rt* gene rather than causing a destabilization of the *rt* mRNA.

The insertion sequence (*dTph3*) that is present in the R27 *rt* allele has sequence characteristics of a transposable element. It has inverted repeats at the ends and it is flanked by an 8 bp directly repeated target site duplication. Even though the internal sequences have completely diverged, there is some homology between the terminal inverted repeats of *dTph1* and *dTph3*. Therefore it is possible that transposition of both elements is dependent on the same activator element. *P. hybrida* was constructed around 1830 via interspecific crosses between the white flowering *P. axillaris*, *P. parodii* group and the coloured flowering *P. violacea*, *P. inflata*, *P. integrifolia* group (Koes *et al.*, 1987; Wijsman, 1983). Genetic analysis indicated the presence of a functional *rt* gene in each of these parental species (Wijsman *et al.*, 1983) and PCR analysis showed that none of them contained the *dTph3* insertion at the *Rt* locus (unpublished data). This indicates that transposition of *dTph3* into the *rt* gene presumably occurred after the interspecific crosses took place that gave rise to *P. hybrida*. The R27 line and related lines have been extensively used in many genetic studies and among tens of thousands of progeny plants instability of the *rt⁻* allele was never reported. This suggests that today the *dTph3* copy at the *Rt* locus can not transpose any longer. Presumably structural defects, like the non-perfect terminal inverted repeats, are the reason for immobility of this element. Nevertheless *dTph3* will be useful as a probe to clone genes that are tagged by insertion of an active transposable element from this family.

The *dTph3* insertion causes polyadenylation to occur at a new site upstream from the polyadenylation site used in a wild-type *rt* mRNA, which explains the smaller mRNA observed in *rt⁻* lines (Figure 7). Whether or not *dTph3* causes (occasionally) premature transcription termination can however not be concluded. Occasionally we observed some larger than wild-type RNA in gel blot analyses of *rt⁻* mutants. Brugliera *et al.* (1994) obtained similar results, but in their lines the abundance of the large RNA was much higher. As there appears to be only one stable *rt⁻* allele present in *P. hybrida* (Brugliera *et al.*, 1994; this paper) it is unlikely that this difference is due to the different lines that were used. As the long RNA hybridizes to the 3′ end of the *rt* coding region it most likely resulted from transcription through the *dTph3* element and polyadenylation at the normal *rt* polyadenylation site(s) (Brugliera *et al.*, 1994). One way to explain the variable levels of this mRNA is to assume that the (cryptic) polyadenylation in the *dTph3* sequence is relatively inefficient, taking into account that the mRNA levels (and presumably transcription) of flavonoid biosynthetic genes in plants do vary with greenhouse conditions (light

intensity, temperature, daylength etc.). This would result in an increased amount of the long *rt* mRNA under high transcription conditions.

Expression of *rt* in petal limbs is under the control of three regulatory genes, *an1*, *an2* and *an11* (Figure 2). No *rt* expression is detectable in anthers from the *an4⁻* line V26 (Figure 2), whereas anthers from *An4⁺* lines accumulate high levels of *rt* mRNA (Quattrocchio *et al.*, 1993) indicating that *rt* expression in anthers is controlled by the regulatory gene *an4*. Comparison of the *rt* promoter sequence, with that of two other *an1*, *an2*, *an4*, *an11* regulated genes (*chsJ* and *dfrA*; Huits *et al.*, unpublished data; Quattrocchio *et al.*, 1993) did not, however, reveal any clear homologies that may represent *an1*, *an2* or *an11* responsive elements. Possibly these elements are too small or too diverged to be detected by sequence comparison alone. Therefore elucidation of this interaction waits for the functional analysis of *cis*-acting elements in the *an1*, *an2*, *an4* and *an11* responsive promoters and the molecular isolation of the regulatory genes. Such experiments are currently in progress.

### Experimental procedures

#### Plant material

*Petunia hybrida* lines used in this study have been maintained as inbred stocks for several generations. Plants were grown under standard greenhouse conditions. For RFLP analysis a (*V23×R51*) F₁ hybrid was backcrossed to both parental strains and analysed for several phenotypic markers as described (Wallroth *et al.*, 1986). As R51 is *rt⁻* and V23 is *Rt⁺*, segregation of the *Rt* marker could be phenotypically scored in the backcross VR×R. The segregation for *RtRt* (homozygous dominant) versus *Rtrt* (heterozygous dominant plants) in the VR×V backcross was determined by selfing of each VR×V backcross plant (Gerats, unpublished data). If this yielded *rt⁻* progeny plants, the VR×V parental plant was scored as *Rtrt* (heterozygous). If no *rt⁻* plants were found the parent was scored as *RtRt*.

Thin layer chromatography (TLC) of flavonoids was performed as described (Wallroth *et al.*, 1986).

#### Construction and differential screening of cDNA libraries

A cDNA library in λZAPII was constructed using the RNase H procedure (Gübler and Hoffmann, 1983). Starting from 2 μg of poly(A)⁺ mRNA extracted from stage 3 flowerbuds of *P. hybrida* line V26 a library of 4 × 10⁶ p.f.u. was obtained, which was used for the differential screening. A directional λGEM4 cDNA library from V26 petal RNA was prepared according to procedures of the BRL Superscript Plasmid Cloning System and screened for a full size *rt* cDNA via plaque hybridization.

For cDNA labelling 1-2 μg of poly(A)⁺ mRNA and 1.0 μg of oligo-dT primer (Boehringer) was used to synthesize approximately 1.0 × 10⁸ c.p.m. ³²P-labelled first strand cDNA. The labelled cDNA/RNA hybrids were purified by chromatography over Sephadex G-50 columns. Prior to hybridization the probes were denatured by boiling, followed by a 15 min incubation at

78 Johan Kroon et al.

35°C with 10 μg of RNase A and H each (Boehringer) to remove the RNA.

For differential screening about \(3 \times 10^4\) p.f.u. (V26 floral cDNA library in λZAPII) were plated at 800 p.f.u./plate on to 90 mm plates and duplicate plaque lifts were taken using Hybond-N membranes (Amersham) according to the manufacturers instructions. About 20 filters were prehybridized at \(42^\circ \mathrm{C}\) for 3 h in 20 ml of 50% formamide; \(5 \times\) SSPE; \(5 \times\) Denhardt's; \(0.1 \%\) SDS; \(100 \mu \mathrm{g} \mathrm{ml}^{-1}\) herring sperm DNA. For hybridization \(1.0 \times 10^8\) c.p.m. \({ }^{32} \mathrm{P}\)-labelled first strand cDNA and \(100 \mu \mathrm{g}\) of poly(A) (Boehringer) were added and incubation was continued for 16–48 h at \(42^{\circ} \mathrm{C}\). Filters were washed in \(1.0 \times\) SSC/ \(0.1 \%\) SDS at \(60^{\circ} \mathrm{C}\) for 30 min and exposed for 3–4 days to X-AR film (Kodak). Plaques showing substantially stronger hybridization to the V26 cDNA probe than to the W162 cDNA probe were picked and retested. Inserts from selected λ clones were subcloned in pBluescript (Stratagene) via in vivo excision mediated by helper phage R408 (Stratagene).

the *rt* sequence and the 3' RACE primer respectively, see Figure 7) and ligated into *BamHI/Sall* digested pBluescript.

The primers used have the following sequences (their position in the *rt* gene is shown in Figure 7b):

primer 1: 5'-dTCATTTATGTTTCTGTTTAATATTTACC-3';

primer 2: 5'-dCGGAAAGCTTGCTCGCAGTAT-3';

primer 3: 5'-dCTGCACAAAATGGACTAATATGGCC-3';

primer 4: 5'-dCACCAGCTAGTGCTGAGCTTCTC-3';

primer 5: 5'-dACAGCCACATTCTTTCAGGCCCTTG-3';

3' RACE: 5'-dGACTCGAGTCGACATCGT\(_{17}\)-3'

Sequencing was performed using the dideoxy-chain termination method using \({ }^{[35} \mathrm{~S}\) ]ATP (Sänger et al., 1977), or using asymmetric PCR with fluorescent M13 primers or fluorescent dideoxynucleotides and employing an Applied Biosystems DNA sequencer model 370A. If necessary nested deletions were generated using the ExoIII/S1 method (Henikoff, 1984).

### RNA analysis

Flowers were harvested from plants and frozen in liquid nitrogen and total RNA was extracted as described (Van Tunen et al., 1988). Developmental stages of flowers have been described before (Van Tunen et al., 1988). Poly(A)\(^+\) mRNA was prepared by two passages over an affinity oligo-dT cellulose column (Pharmacia and Boehringer) and its integrity and purification factor were monitored via RNA gel blot analysis.

RNA gel blot analysis was performed according to Van Tunen et al. (1988), using \(10 \mu \mathrm{g}\) total RNA per sample. After electrophoretic separation in formaldehyde agarose gels, the RNA was blotted on to Hybond-N according to the suppliers protocol (Amersham). The gels were briefly stained with ethidium bromide prior to blotting to ensure equal loading and integrity of the RNA. After hybridization the blots were washed in \(0.5 \times\) SSC, \(1 \%\) SDS at \(60^{\circ} \mathrm{C}\).

Primer extension experiments using primer 3 and \(10 \mu \mathrm{g}\) of floral RNA from the line V26 were performed as described previously (Koes et al., 1989b).

### Construction of antisense genes and plant transformation

The orientation of the *difG* fragment in pBluescript was determined by sequencing both ends via double strand plasmid sequencing using the universal M13-21mer primer and the M13 reverse primer. The *difG* cDNA (1.0 kb) was inserted in an anti-sense orientation in between a CaMV 35S-promoter and a nopaline synthase 3' flanking region in a binary T-DNA vector (Van der Krol et al., 1988). This construct was mobilised into *A. tumefaciens* strain LBA 4404 by triparental mating (Ditta et al., 1980) and used to transform *P. hybrida* V26 leaf disks (Horsch et al., 1985). After shoot and root induction on kanamycin-containing medium transgenic plants were transferred to soil and kept in a greenhouse.

### DNA analyses

All plasmid- and λ-DNA manipulations were performed using standard methods. Isolations of plant DNA and DNA gel blot analyses were performed according to previously published procedures (Beld et al., 1989).

A V26 genomic library was constructed as previously described (Koes et al., 1989b) except that λGEM11 (Promega) was used as a vector. Approximately 150,000 clones were screened by stringent hybridization with a \({ }^{32} \mathrm{P}\)-nick-translated *rt* cDNA probe. Positive plaques were purified to homogeneity by two rounds of plaque purification. A 4.7 kb *HindIII/Sall* insert from one positive λ clone (λGEM11:1), containing the complete *Rt* gene, was subcloned in pBluescript vectors (Stratagene) using standard cloning procedures.

The region of the *rt* gene containing *dTph3*, was amplified from 300 ng genomic DNA, from the line R27, by polymerase chain reaction (PCR) using primers 2 and 5, cut at the internal *ClaI* and *PstI* sites (cf. Figure 7), and cloned into *ClaI/PstI* digested pBluescript. The *dTph1* insertion was cloned from the *P. hybrida* line Vu15 via amplification of the *rt* promoter region with primers 1 and 3, blunt ending with T4 DNA polymerase, phosphorylation with T4 DNA polynucleotide kinase and ligation into the *Smal* site of pBluescript. To amplify *rt* RNA products from R27 floral RNA we used the 3' RACE procedure described by Frohman et al. (1988) using primer 4 and the 3' RACE primer. Amplification products were digested with *BclI* and *SalI* (cut in

### Acknowledgements

We thank Olaf Mol for his contribution to the antisense RNA experiment and Pieter Hoogev een, Martina Meesters and Daisy Kloos for their care of the plants. We are indebted to Dr Tom Gerats for making genetically characterized VR backcross populations available and stimulating discussions. We thank Joop Meijer, Wim Bergenh enegouwen and Fred Schuurhof for their assistance with photographic work. This research was financed in part by Florigene BV, Rijnsburg, The Netherlands.

### References

Bar-Peled, M., Lewinsohn, E., Fluhr, R. and Gressel, J. (1991) UDP-rhamnose: flavanone-7-O glucoside 2"-O rhamnosyltransferase. Purification and characterization of an enzyme catalyzing the production of bitter compounds in citrus. *J. Biol. Chem.* 266, 20 953–20 959.

Beld, M., Martin, C., Huits, H., Stuitje, A.R. and Gerats, A.G.M. (1989) Flavonoid synthesis in *Petunia hybrida*: Partial characterization of dihydroflavonol 4-reductase genes. *Plant Mol. Biol.* 13, 491–502.
Britsch, L., Ruhnau-Brich, B. and Forkmann, G. (1991) Molecular cloning, sequence analysis and in vitro expression of flavanone 3-β-hydroxylase from Petunia hybrid. J. Biol. Chem. 267, 5380–5387.

Brugliera, F., Holton, T.A., Stevenson, T.W., Farcy, E., Lu, C.Y. and Cornish, E.C. (1993) Isolation and characterization of a cDNA clone corresponding to the Rt locus of Petunia hybrid. Plant J. 5, 81–92.

Cornu, A. (1977) Systèmes instables induits chez le petunia. Mut. Res. 42, 235–248.

De Vlaming, P., Cornu, A., Farcy, E., Gerats, A.G.M., Maizonnier, D., Wiering, H. and Wijsman, H.J.W. (1984) Petunia hybrid: A short description of the action of 91 genes, their origin and their map location. Plant Mol. Biol. Rep. 2, 21–42.

Ditta, G., Stanfield, S., Corbin, D. and Helinski, D.R. (1980) Broad host range DNA cloning sytem for gram negative bacteria: Construction of a genebank of Rhizobium meliloti. Proc. Natl Acad. Sci. USA, 77, 7347–7351.

Doedeman, M., Boersma, E.A., Koomen, W. and Bianchi, F. (1984a) Genetic analysis of instability in Petunia hybrid 1. A highly unstable mutation induced by a transposable element inserted at the An1 locus for flower colour. Theor. Appl. Genet. 67, 345–355.

Doedeman, M., Gerats, A.G.M., Schram, A.W., de Vlaming, P. and Bianchi, F. (1984b) Genetic analysis of instability in Petunia hybrid 2. Unstable mutations at different loci as the result of transpositions of the genetic element inserted at the An1 locus. Theor. Appl. Genet. 67, 357–366.

Dooner, H.K., Robbins, T. and Jorgensen, R. (1991) Genetic and developmental control of anthocyanin biosynthesis. Ann. Rev. Genet. 25, 173–199.

Frohman, M.A., Dush, M.K. and Martin, G.R. (1988) Rapid production of full-length cDNAs from rare transcripts: Amplification using a single gene-specific oligonucleotide primer. Proc. Natl Acad. Sci. USA, 85, 8998–9002.

Furtek, D., Schiefelbein, J.W., Johnston, F. and Nelson, O.E. (1988) Sequence comparison of three wild type Bronze-1 alleles from Zea mays. Plant Mol. Biol. 11, 473–481.

Gerats, A.G.M., De Vlaming, P., Doedeman, M. and Schram, A.W. (1982) Genetic control of the conversion of dihydro-flavonols into flavonols and anthocyanins in flowers of Petunia hybrid. Planta, 155, 364–368.

Gerats, A.G.M., Wallroth, M., Donker-Koopman, W., Groot, S.P.C. and Schram, A.W. (1983) The genetic control of the enzyme UDP glucose 3-O-flavonoid glucosyltransferase in flowers of Petunia hybrid. Theor. Appl. Genet. 65, 349–352.

Gerats, A.G.M., Farcy, E., Wallroth, M., Groot, S.P.C. and Schram, A. (1984) Control of anthocyanin biosynthesis in Petunia hybrid by multiple allelic series of the genes An1 and An2. Genetics, 106, 501–508.

Gerats, A.G.M., Vrijlandt, E., Wallroth, M. and Schram, A.W. (1985a) The influence of the genes An1, An2 and An4 on the activity of the enzyme UDP-glucose flavonoid 3-O-glucosyl-transferase. Biochem. Genet. 23, 591–598.

Gerats, A.G.M., Wallroth, M., de Vlaming, P. and Bianchi, F. (1985b) A two-element system controls instability at the An3 locus in Petunia hybrid. Theor. Appl. Genet. 70, 245–247.

Gerats, A.G.M., Beld, M., Huits, H. and Prescott, A. (1989) Gene tagging in Petunia hybrid using homologous and heterologous transposable elements. Dev. Genet. 10, 561–568.

Gerats, A.G.M., Huits, H., Vrijlandt, E., Marañá, C., Souer, E. and Beld, M. (1990) Molecular characterization of a

nonautonomous transposable element (dTph1) of Petunia. Plant Cell, 2, 1121–1128.

Gierl, A. and Saedler, H. (1992) Plant transposable elements and gene tagging. Plant Mol. Biol. 19, 39–49.

Gübler, U. and Hoffmann, B.J. (1983) A simple and very efficient method for generating cDNA libraries. Gene, 30, 195–200.

Henikoff, S. (1984) Unidirectional digestion with exonuclease III creates targeted breakpoints for DNA sequencing. Gene, 28, 351–359.

Horsch, R.B., Fry, J.E., Hoffmann, N.L., Eicholtz, D., Rogers, S.G. and Fraley, R.T. (1985) A simple and general method for transferring genes into plants. Science, 1229–1231.

Jonsson, L.M.V., Aarsman, M.E.G., Poulton, J.E. and Schram, A.W. (1984a) Properties and genetic control of four methyltransferases involved in methylation of anthocyanins in flowers of Petunia hybrid. Planta, 160, 174–179.

Jonsson, L.M.V., Aarsman, M.E.G., Van Diepen, J., Smit, N. and Schram, A.W. (1984b) Properties and genetic control of anthocyanidin 5-O-glucosyltransferase in flowers of Petunia hybrid. Planta, 160, 341–347.

Koes, R.E., Spelt, C.E., Mol, J.N.M. and Gerats, A.G.M. (1987) The chalcone synthase multigene family of Petunia hybrid (V30): Sequence homology, chromosomal localization and evolutionary aspects. Plant Mol. Biol. 10, 375–385.

Koes, R.E., Spelt, C.E. and Mol, J.N.M. (1989a) The chalcone synthase multigene family of Petunia hybrid (V30): Differential, light-regulated expression during flower development and UV light induction. Plant Mol. Biol. 12, 213–225.

Koes, R.E., Spelt, C.E., Van den Elzen, P.J.M. and Mol, J.N.M. (1989b) Cloning and molecular characterization of the chalcone synthase multigene family of Petunia hybrid. Gene, 81, 245–257.

Martin, C., Prescott, A., Mackay, S., Bartlett, J. and Vrijlandt, E. (1991) Control of anthocyanin biosynthesis in flowers of Antirrhinum majus. Plant J. 1, 37–49.

Napoli, C., Lemieux, C. and Jorgensen, R. (1990) Introduction of a chimeric chalcone synthase gene into petunia results in reversible co-suppression of homologous genes in trans. Plant Cell, 2, 279–289.

Quattrocchio, F., Wing, J.F., Leppen, H.T.C., Mol, J.N.M. and Koes, R.E. (1993) Regulatory genes controlling anthocyanin pigmentation are functionally conserved among plant species and have distinct sets of target genes. Plant Cell, 5 (in press).

Ralston, E.J., English, J.J. and Dooner, H.K. (1988) Sequence of three bronze alleles of maize and correlation with the genetic fine structure. Genetics, 119, 185–197.

Sänger, F., Nicklen, S. and Coulson, A.R. (1977) DNA sequencing with chain termination inhibitors. Proc. Natl Acad. Sci. USA, 74, 5463–5467.

Sato, H., Kowai, O., Tanabe, K. and Kashuwamata, S. (1990) Isolation and sequencing of rat liver bilirubin UDP-glucuronosyltransferase cDNA: Possible alternate splicing of a common primary transcript. Biochem. Biophys. Res. Comm. 169, 260–264.

Van der Krol, A.R., Lenting, P.E., Veenstra, J., Van der Meer, I.M., Koes, R.E., Gerats, A.G.M., Mol, J.N.M. and Stuitje, A.R. (1988) An anti-sense chalcone synthase gene in transgenic plants inhibits flower pigmentation. Nature, 333, 866-869.

Van der Krol, A.R., Mur, L.A., Beld, M., Mol, J.N.M. and Stuitje, A.R. (1990) Flavonoid genes in petunia: Addition of a limited number of gene copies may lead to a suppression of gene expression. Plant Cell, 2, 291–299.
Van Tunen, A.J. and Mol, J.N.M. (1987) A novel purification procedure for chalcone flavanone isomerase from Petunia hybrida and the use of its antibodies to characterise the Po mutation. Arch. Biochem. Biophys. 257, 85–91.

Van Tunen, A.J., Koes, R.E., Spelt, C.E., Van der Krol, R., Stuitje, A.R. and Mol, J.N.M. (1988) Cloning of the two chalcone flavanone isomerase genes from Petunia hybrida; coordinate, light-regulated and differential expression of flavonoid genes. EMBO J. 7, 1257–1263.

Van Tunen, A.J., Mur, L.A., Recourt, K., Gerats, A.G.M. and Mol, J.N.M. (1991) Regulation and manipulation of flavonoid gene expression in anthers of petunia: the molecular basis of the Po mutation. Plant Cell, 3, 39–48.

Wallroth, M., Gerats, A.G.M., Rogers, S.G., Fraley, R.T. and Horsch, R.B. (1986) Localization of foreign genes in Petunia hybrida. Mol. Gen. Genet. 202, 6–15.

Weiss, D., Van der Luit, A.H., Kroon, J.T.M., Mol, J.N.M. and Kooter, J.M. (1993) The petunia homolog of the Antirrhinum majus candi gene and Zea mays A2 flavonoid genes:

EMBL Data Library accession numbers X71059 (P. hybrid a mRNA for rhamnosyltransferase) and X71060 (P. hybrid a gene for rhamnosyltransferase).

homology to flavanone 3-hydroxylase and ethylene forming enzyme. Plant Mol. Biol. 22, 893–897.

Wiering, H. and De Vlaming, P. (1973) Glycosylation and methylation patterns of anthocyanins in Petunia hybrida. I. The gene Gf. Genen Phaenen, 16, 35–50.

Wiering, H. and De Vlaming, P. (1984) Genetics of flower and pollen colors. In Monographs on Theoretical and Applied Genetics 9: Petunia (Sink, K.C., ed.). Berlin, Springer-Verlag, pp. 49–67.

Wijsman, H.J.W. (1983) On the interrelationships of certain species of petunia II. Experimental data: crosses between different taxa. Acta Bot. Neerl. 32, 97–107.

Wijsman, H.J.W. (1986) Evidence for transposition in Petunia. Theor. Appl. Genet. 71, 291–296.

Yamamoto, F. and Hakomori, S. (1990) Sugar-nucleotide donor specificity of histo blood group A and B transferases is based on amino acid substitutions. J. Biol. Chem. 265, 19 257–19 262.
