
The EMBO Journal vol. 13 no. 5 pp. 1103–1113, 1994

# The Dbf2 and Dbf20 protein kinases of budding yeast are activated after the metaphase to anaphase cell cycle transition

Jeremy H. Toyn and Leland H. Johnston${}^{1}$

Laboratory of Yeast Genetics, National Institute for Medical Research,  
The Ridgeway, Mill Hill, London, NW7 1AA, UK  

${}^{1}$Corresponding author  

Communicated by J.J. Skehel  

Thermosensitive mutations in the $DBF2$ gene arrest the cell cycle during nuclear division. Although the chromatin has divided in arrested cells, an elongated mitotic spindle is present and Cdc28 protein kinase activity remains high, indicating that nuclear division is incomplete. By execution point analysis we show that Dbf2 carries out an essential cell cycle function after the metaphase to anaphase transition and is therefore required during anaphase and/or telophase. This cell cycle stage-specific requirement for the function of Dbf2 coincides with the cell cycle regulation of Dbf2/Dbf20 protein kinase activity, which can be detected in immunoprecipitates containing Dbf2 or Dbf20. The kinase activity is specific for serine/threonine residues and Dbf2 accounts for the bulk of the activity, with Dbf20 playing a minor role. Furthermore, Dbf2 is a phosphoprotein and, significantly, the dephosphorylated form appears with the same cell cycle timing as the kinase activity, suggesting a role for dephosphorylation in the activation mechanism. In addition, we show that the $DBF2$ transcript, which is under cell cycle control, is expressed in advance of the activation of the kinase, but that cell cycle-regulated expression of the mRNA is not required for activation of the Dbf2 kinase during M phase. Thus, Dbf2/Dbf20 kinase activity is precisely regulated in the cell cycle by a post-translational mechanism and phosphorylates its target substrates for an event that occurs during anaphase and/or telophase.

Key words: cell cycle/nuclear division/protein kinase/Saccharomyces cerevisiae/telophase

---

mutants have turned out to be protein kinases and phosphatases. The Cdc28 kinase is involved in START and also in the G₂/M transition, associating with different cyclins at each step (Nasmyth, 1993), whereas the Cdc7 kinase, in association with the Dbf4 protein, is activated early in the cell cycle where it is required specifically for initiation of DNA replication (Kitada et al., 1992; Jackson et al., 1993; Yoon et al., 1993). The Cdc5 kinase (Kitada et al., 1993) and Cdc15 kinase (Schweitzer and Philippsen, 1991) are required late in the cell cycle for completion of M phase. The Cdc5 kinase is a possible homologue of the *Drosophila* kinase encoded by the *polo* gene, which is activated during late anaphase to telophase (Fenton and Glover, 1993). Cell cycle proteins with homology to phosphatases include Sit4, which is required for expression of $SWI4$ and the G₁ cyclins during G₁ (Fernandez-Sarabia et al., 1992), and Cdc14 (Wan et al., 1992), which, like the Cdc5 and Cdc15 kinases, is required late in the cell cycle during M phase.

The $DBF2$ gene also functions in the cell cycle and encodes a putative protein kinase (Johnston et al., 1990). It has a close homologue, $DBF20$, which is >80% identical, and between them the two genes carry out at least one essential function. Either gene can be deleted alone, but deletion of both is lethal (Toyn et al., 1991). However, there is reason to believe that in the presence of Dbf2, Dbf20 does not carry out the essential function, because $dbf2$ loss-of-function mutations are conditionally lethal even in the presence of the wild-type $DBF20$ gene. This may be because Dbf2 sequesters the Spo12 protein, a high copy suppressor of $dbf2$ mutants, and which is required by Dbf20 if it is to replace functionally Dbf2 (Parkes and Johnston, 1992; Toyn and Johnston, 1993).

The precise function carried out by these two kinases is not yet clear, although, like $CDC28$, $DBF2$ seems to function twice in the cell cycle. It clearly affects initiation of S phase since at the restrictive temperature $dbf2$ thermosensitive (ts) mutants delay the onset of S phase by approximately one-third of a cell cycle (Johnston et al., 1990). Moreover, $DBF2$ also interacts with $SIT4$ (Parkes and Johnston, 1992), a protein phosphatase that functions in G₁ (Sutton et al., 1991). However, after the delay, S phase is completed and $dbf2$ mutant cells seem to block in late M phase (Johnston et al., 1990). This could be due to either arrest at a mitotic checkpoint in response to failure of the mutant to execute an earlier cell cycle event, or to an actual late mitotic defect in the mutant. One fact consistent with a late mitotic function is that $DBF2$ is expressed under cell cycle control in late M phase, at about the time mutant cells block in the cell cycle. Moreover, $DBF2$ interacts in some way with $CDC5$ and $CDC15$, the two late mitotic kinases, since multiple copies of these genes suppress $dbf2$ mutations (Kitada et al., 1992). Thus, in spite of being initially detected as a mutant affecting S phase function (Johnston and Thomas, 1982), $DBF2$ may have a primary function in mitosis.

In this study, we are principally concerned with the cell

© Oxford University Press

J.H.Toyn and L.H.Johnston

cycle timing of the function of Dbf2, as an indication of what its role might be. We demonstrate that Dbf2 carries out a vital cellular function during anaphase and/or telophase. An in vitro assay for Dbf2/Dbf20 is also described, which confirms that they are protein kinases and this kinase activity is precisely regulated in the cell cycle. The kinase is activated shortly after the metaphase to anaphase transition, consistent with the timing of the functional requirement for Dbf2. The mechanism of this kinase activation appears to involve primarily dephosphorylation of the Dbf2 molecule, although the *de novo* synthesized, dephosphorylated protein produced as a consequence of the cell cycle-regulated gene expression also contributes to activation. The timing of Dbf2/Dbf20 function and kinase activation, after the metaphase to anaphase transition, has implications for the roles of a number of other genes that interact with *DBF2* and which have phenotypes similar to the *dbf2* mutant phenotype, such as *cdc5*, *cdc14* and *cdc15*.

## Results

### Dbf2 is required after the metaphase to anaphase cell cycle transition

Preliminary experiments showed that Dbf2 had a role late in the cell cycle (Johnston *et al.*, 1990) and we have now carried out more detailed observations to address this question of cell cycle timing by further analysing both the terminal phenotype and the execution point of *dbf2*.

Thermosensitive (ts) mutations of *DBF2* cause an arrest of the cell cycle in which the cells have divided chromatin. In some of the cells, although the bulk of the chromatin has divided, a thin strand of chromatin can be seen stretching through the neck of the bud, suggesting that nuclear division is incomplete (Johnston *et al.*, 1990). This interpretation has been confirmed by two further observations. First, indirect immunofluorescence microscopy of *dbf2*-arrested cells showed that >90% of the cells contained elongated spindles, indicating a late M phase, possibly telophase, arrest (Figure 1). Second, Cdc28 kinase activity remains high in *dbf2*-arrested cells (Figure 2). The Cdc28 kinase was measured in immunoprecipitates made from cells in which the wild-type *CDC28* gene had been replaced by an epitope-tagged *CDC28* gene. A log phase culture of a *dbf2* mutant strain was shifted from the permissive to the restrictive temperature and samples were taken every hour up to 5 h, during which time >90% of the cells arrested with a dumbbell morphology. As a control, a wild-type culture was incubated with α-factor and samples were taken in the same way, during which time the cells all formed the characteristic unbudded morphology of α-factor-arrested cells. Cdc28 kinase activity remained high during arrest of the cell cycle by *dbf2*, whereas arrest of the cell cycle by α-factor abolished the kinase activity. Thus, M phase cannot be completed in the absence of functional *DBF2*. In conclusion, *dbf2* mutations cause arrest of the cell cycle either at the end of anaphase or during telophase.

This does not, however, prove that Dbf2 functions during nuclear division. It is possible that Dbf2 functions in advance of nuclear division and that an independent cell cycle checkpoint is responsible for the arrest during nuclear division. This question of precisely when Dbf2 is required during the cell cycle can be addressed by an execution point analysis. Using this approach, it had been shown previously that *dbf2* mutants have a late cell cycle execution point

Fig. 1. Elongated spindles are present in *dbf2*-arrested cells. A log phase culture of strain J2 (*dbf2*) was grown at the permissive temperature and then shifted for 3 h (~1.5 generation times) to the restrictive temperature for growth. The large budded cells that resulted from this treatment were then fixed and labelled by indirect immunofluorescence for spindles (upper panel) and with DAPI to detect the position of the chromatin (lower panel).

![Image Description: Figure 1](image1.png)

![Image Description: Figure 2](image2.png)

Fig. 2. Cdc28 kinase activity remains high in *dbf2*-arrested cells. A log phase culture of the strain L181-6B (*dbf2*), containing the epitope-tagged Cdc28, was grown at the permissive temperature and then shifted to the restrictive temperature for growth. In addition, a log phase culture of the strain CG378 (*DBF2* wild-type, w-t), containing the epitope-tagged Cdc28, was treated with α-factor (3.5 mg/ml). Samples were taken from both cultures every hour up to 5 h, as indicated, and immunoprecipitates were made to assess Cdc28 kinase activity.

(Johnston *et al.*, 1990). However, in order to determine whether the execution point was before or during nuclear division, it was necessary to compare the timing of the execution point with the timing of nuclear division itself. A convenient way to do this was by staining the cells with DAPI. Cells that have passed the metaphase to anaphase transition have divided chromatin, which can be seen as two fluorescent spots of DNA, one in the mother cell and one in the bud. Thus, a synchronous culture of the ts *dbf2* strain J114 was prepared at the permissive temperature using the α-factor method and samples were taken for three different measurements. First, to determine the proportion of cells that could complete the division cycle at the restrictive temperature (the execution point); cells that have passed the

Dbf2 and Dbf20 activation

A

1 2 3 4 5 6 7 8 9
= Dbf2
- Dbf20

B

1 2 3
= Dbf2
- Dbf20

C

1 2

Fig. 3. Dbf2 carries out a function during nuclear division after the metaphase to anaphase transition. A log phase culture of J114 (dbf2) was grown at the permissive temperature and synchronized by release from α-factor arrest. The culture was sampled at 10 min intervals to determine the percentage of cells with buds (circles) and the percentage of cells with divided chromatin (triangles). Part of each sample was also shifted to the restrictive temperature for growth and incubated for a further 3 h. The increase in cell number that resulted after 3 h was measured using a Coulter particle counter and the percentage of cells that had completed the cell division cycle was calculated (squares). This gives a measure of the proportion of cells that had passed the execution point (Johnston *et al.*, 1990).

execution point are able to divide at the restrictive temperature, cells that have not passed the execution point do not divide. Second, to estimate the proportion of cells that contained divided chromatin, and third, to measure the proportion of cells that had buds (a standard measure of synchrony). The cells in the culture started growing buds ~40 min after release from α-factor and started to undergo division of the chromatin after ~90 min (Figure 3). The execution point was >10 min after the division of the chromatin, which makes dbf2 the only cell cycle mutant to have an execution point demonstrably later than G₂ (Culotti and Hartwell, 1971). This shows that the Dbf2 kinase is functionally required after the metaphase to anaphase transition.

As independent confirmation of this result, the execution point in strain J2, a homozygous dbf2 diploid, was measured. The experiment had the same approach, except that the culture was synchronized by the dbf2 block and release method (which gives highly synchronous division in this strain), and samples were taken every 5 min. In this case the execution point was between 5 and 10 min after division of the chromatin (data not shown).

Dbf2 is a phosphoprotein

In order to study the molecular basis for the function of Dbf2 during nuclear division, an antiserum was raised against the synthetic peptide EHSDPFSTFY, corresponding to the C-terminus of the Dbf2 and Dbf20 proteins. The antiserum was used to visualize the Dbf2 and Dbf20 proteins by gel fluorography of immunoprecipitates from ³⁵S-labelled yeast extracts (Figure 4A). In yeast containing multiple copies of DBF2, the predominant band was a doublet at 68 kDa, whereas yeast containing multiple copies of DBF20 gave a single band at 60 kDa. The Dbf2 and Dbf20 proteins in yeast not overexpressing these genes could also be detected. As a further control to test the specificity of the antibodies, immunoprecipitations were carried out in the presence of

Fig. 4. Detection of the Dbf2 and Dbf20 molecules by gel fluorography of immunoprecipitates shows that Dbf2 is a phosphoprotein. (A) Strains derived from CG378 containing YRp12-DBF2 (lanes 1–3), YRp12-DBF20 (lanes 4–6) or the parental vector YRp12 (lanes 7–9) were pulse-labelled with [³⁵S]sulfate. Radiolabelled protein was extracted and immunoprecipitates were made using the anti-peptide antiserum in the absence (lanes 1, 4 and 7) or the presence of competing peptide (1 mg/ml, lanes 2, 5 and 8). Immunoprecipitates were also made using anti-glucose-6-phosphate dehydrogenase (lanes 3, 6 and 9). Immunoprecipitates were separated by SDS-PAGE (10% acrylamide) and radiolabelled protein was detected by gel fluorography. (B) Strains CG378 (DBF2 DBF20 wild-type, lane 1), S7-4B (dbf2Δ, lane 2) and J99 (dbf20Δ, lane 3) were pulse-labelled as above. Immunoprecipitates were made using the anti-peptide antiserum and the proteins were detected as above. (C) Strain CG378 containing YRp12-DBF2 was pulse-labelled and an immunoprecipitate was made using the anti-peptide antiserum. The immunoprecipitate was divided in half and treated in the absence (lane 1) or the presence (lane 2) of alkaline phosphatase. Proteins were detected as above.

the peptide to which the antibody had originally been raised. Under these conditions, neither Dbf2 nor Dbf20 could be detected. Similarly, yeast from which either DBF2 or DBF20 had been deleted lacked the corresponding proteins at 68 and 60 kDa, respectively (Figure 4B).

Dbf2 protein forms a doublet in the gel fluorograms, suggesting either that two different length proteins are made from the same gene, or that the protein is modified post-translationally. In fact, the upper band of the doublet is sensitive to phosphatase treatment (Figure 4C), indicating that the Dbf2 protein is a phosphoprotein. In contrast, phosphatase treatment of the Dbf20 protein gave no similar evidence of phosphorylation (data not shown).

Dbf2 accounts for most of the serine/threonine protein kinase activity of Dbf2 and Dbf20

To take the molecular study of Dbf2 and Dbf20 further, an assay was developed to measure their protein kinase activity. Immunoprecipitates of Dbf2 and Dbf20 prepared using the anti-peptide antibody under non-denaturing conditions catalysed phosphate transfer from ATP to the artificial

J.H.Toyn and L.H.Johnston

Fig. 5. Dbf2 immunoprecipitates contain serine/threonine protein kinase activity. (A) A gel autoradiogram of phosphohistone from kinase assays and corresponding phosphorimage quantitation is shown. Log phase cultures were harvested, protein was extracted and immunoprecipitates were made using the anti-peptide antibody in the absence (lanes 1–5 and 7) or the presence of 1 mg/ml peptide (lanes 6 and 8). The strains used were CG378 (DBF2 DBF20 wild-type, lane 1), S7-4B (dbf2Δ, lane 2), J99 (dbf20Δ, lane 3), V378 (spo12Δ lane 4), S7-4A (dbf2Δ, lanes 5 and 6), and S7-4A containing an integrated copy of DBF2 (dbf2Δ + pRS304-DBF2, lanes 7 and 8). (B) Phosphorimage quantitation of the following kinase assays is shown; 2 ml aliquots of anti-peptide antiserum were mixed with 1 ml of 3-fold serial dilutions of peptide, starting with a final concentration of 330 mM (lane 1) and ending with a final concentration of 0.46 mM (lane 7). Incubation was carried out for 10 min on ice and then extract from strain CG378 (DBF2 DBF20 wild-type) containing 250 mg of protein was added in order to make immunoprecipitates for kinase assays. A concentration of 4 mM peptide (lane 5) was sufficient to block the binding of kinase activity to the antibody. As controls, immunoprecipitates were made in the absence of peptide and kinase assays were subsequently carried out in the absence (lane 8) or the presence (lane 9) of 330 mM peptide. (C) An autoradiogram of a thin layer electrophoresis separation of phosphoamino acids and phosphopeptides is shown. Immunoprecipitates were made from an extract of CG378 using unblocked anti-peptide antiserum (lane 1), and anti-peptide antiserum that had been blocked by preincubation with 1 mM peptide (lane 2). Kinase assays were carried out, followed by phosphoamino acid analysis of the resulting radiolabelled phosphohistone. Arrowheads indicate the positions of phosphoamino acids, from top to bottom, phosphotyrosine, phosphothreonine and phosphoserine.

substrate histone H1. The reaction was visualized by using [γ-32P]ATP and gel autoradiography or phosphorimage analysis of the resulting phosphohistone.

The total kinase activity was largely dependent on the presence of Dbf2, since immunoprecipitates prepared from dbf2Δ strains had only 20% of the kinase activity (Figure 5A). Full kinase was restored in a dbf2Δ strain into which an integrated copy of DBF2 had been introduced (on the plasmid pRS304-DBF2) suggesting that Dbf2 accounts for ~80% of the kinase activity. Deletion of DBF20, or SPO12, a gene that interacts with DBF2 (Parkes and Johnston, 1992), had less of an affect on the kinase activity in these assays (Figure 5A). As a further control for the specificity of the kinase assay, immunoprecipitates were prepared using antibody that had been preincubated with a serial dilution of peptide and assayed for kinase activity (Figure 5B). A concentration of 4 mM peptide was sufficient to prevent immunoprecipitation of most of the kinase activity, consistent with the concentration of specific IgG expected in rabbit antiserum (Harlow and Lane, 1988). The assay itself was not inhibited by the presence of the peptide, because even the highest concentration of peptide could be added to the kinase assay, after formation of the immunoprecipitate, without inhibiting kinase activity (Figure 5B). Thus, the peptide can only affect the kinase assay by blocking the antibodies to which Dbf2 and Dbf20 bind. Immunoprecipitates were also prepared from ts dbf2 strains.

Surprisingly, none of the three mutant alleles of dbf2 contained ts kinase activity in this assay, even though the activity was peptide-competable (data not shown). Furthermore, even a dbf2 dbf20Δ strain, which is ts for growth, did not have ts kinase activity. Our interpretation of this is that the mutant alleles are affected for specificity or targeting of the protein kinase, rather than in the enzyme activity itself, and therefore, although they can catalyse phosphate transfer, they are unable to phosphorylate their normal substrates in vivo at the restrictive temperature.

From the sequence of the Dbf2 and Dbf20 proteins it was predicted that they would both phosphorylate target proteins on serine and threonine (Johnston et al., 1990; Toyn et al., 1991). To test this, the phosphorylated histone H1 from two kinase assays was broken down into the individual amino acids by acid hydrolysis and separated by thin layer electrophoresis (Figure 5C). One of the samples was prepared from an assay in which the immunoprecipitation had been carried out in the presence of peptide. Hence, the difference in the two samples would represent the Dbf2/Dbf20-specific kinase activity. Phosphothreonine and phosphoserine were detected, whereas no phosphotyrosine was detected. Using phosphorimage analysis, phosphothreonine was found to account for 60% of the phosphoamino acids and phosphoserine accounted for 40%. Phosphopeptides resulting from incomplete hydrolysis can also be seen.

Dbf2 and Dbf20 activation

A  
○ % cells with buds  
△ % cells with divided chromatin  

B  
Dbf2/20 kinase  

C  

1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22  

Fig. 6. Dbf2/Dbf20 kinase activation coincides with the stage of the cell cycle at which the chromatin divides. A synchronous culture of strain S7-4A (dbf2Δ), containing an integrated copy of the c-myc epitope-tagged DBF2 gene, was prepared by the α-factor method. Samples were taken at 10 min intervals up to 3 h and the following parameters were measured. (A) The percentage of cells with buds (circles) and the percentage of cells with divided chromatin (triangles). (B) Dbf2/Dbf20 kinase activity quantified by phosphorimage analysis was normalized to the amount of Dbf2 protein detected by densitometry of an immunoblot. (C) The upper row of bands shows the autoradiogram of phosphohistone H1 in the same gel that was quantified by phosphorimage analysis (lanes 4–22, corresponding to the 10 min timepoints shown in panels A and B) and the log phase culture before treatment with α-factor (lane 3). The lower row of bands shows immunoblots of the epitope-tagged Dbf2 in the same samples (lanes 3–22) and two control samples; one a mid-log phase sample of S7-4A containing Dbf2-myc protein (lane 1) and the other made from strain S7-4A containing an integrated copy of DBF2 without the c-myc epitope (lane 2).

Dbf2/Dbf20 kinase activity is maximal during nuclear division and coincides with the appearance of the dephosphorylated form of Dbf2

We have shown that Dbf2 is a phosphoprotein that has protein kinase activity and that it carries out a vital function during nuclear division. In order to find out whether the phosphorylation of Dbf2 and the kinase activity were regulated in a way consistent with the function during nuclear division, a cell cycle experiment was carried out. Cells were synchronized by the α-factor method and samples were taken every 10 min in order to measure four parameters: the Dbf2/Dbf20 kinase activity, the Dbf2 phosphorylation state, the proportion of cells with divided chromatin and the proportion of cells with buds (Figure 6). For the detection of the Dbf2 phosphorylation state, immunoblotting was used. However, rather than using the anti-peptide antiserum, a c-myc epitope-tagged Dbf2 and the monoclonal antibody 9E10 were used for immunoblotting. Thus, a yeast strain in which the wild-type Dbf2 had been replaced by an integrated copy of the epitope-tagged Dbf2 was used. The Dbf2/Dbf20 kinase activity was found to be strongly cell cycle-regulated and maximal activity occurred at the time of division of the chromatin, that is, the time at which approximately half of the cells had divided chromatin. In the immunoblots of Dbf2 in the same experiment, a doublet band was seen (Figure 6C), representing the different phosphorylation states of Dbf2, as demonstrated previously (Figure 4). The phosphorylation state of Dbf2 was under cell cycle control and the appearance of the lower dephosphorylated band coincided with the maximum kinase activity. Furthermore, the appearance of the dephosphorylated lower band was accompanied by a corresponding decrease in the amount of the phosphorylated upper band (Figure 6B and C), strongly suggesting that a dephosphorylation event might be involved in activation of Dbf2 kinase. However, a small amount of the dephosphorylated form in the three samples immediately following release from the α-factor arrest was not accompanied by high kinase activity (Figure 6, lanes 4 and 5). Otherwise, when the kinase activity was low, Dbf2 was in the less mobile phosphorylated form.

Similar cell cycle experiments were carried out with two other yeast strains, neither of which contained the epitope-tagged Dbf2. In both cases the Dbf2/Dbf20 kinase activity was strongly under cell cycle control, and, as above, coincided with division of the chromatin (data not shown). Furthermore, in one of these yeast strains the epitope-tagged Cdc28 was present, which allowed measurement of the Cdc28 kinase activity during the cell cycle. The Cdc28 kinase activity was under cell cycle control and peaked 20 min earlier in the cell cycle than Dbf2/Dbf20 kinase activity (data not shown). Since Cdc28 kinase is maximal during late G2 (Surana et al., 1991), this is consistent with a cell cycle timing of Dbf2/Dbf20 kinase activity in anaphase/telophase. From these three cell cycle experiments and the further experiments shown in Figures 7, 8 and 9, we conclude that activation of the Dbf2/Dbf20 kinase occurs after the

J.H.Toyn and L.H.Johnston

**Fig. 7. Activation of the Dbf2/Dbf20 kinase does not correlate with the expression of the DBF2 and DBF20 genes.** A synchronous culture of strain CG378 (DBF2 DBF20 wild-type) was made by the α-factor method. Samples were taken at 10 min intervals up to 2 h. (A) Northern blot and phosphorimage analyses were carried out for the DBF2 transcript (circles) and the actin transcript (squares) in each of the samples. (B) Kinase assays were quantified by phosphorimage analysis (circles) and the percentage of cells containing divided chromatin was determined (squares). (C) Kinase assays were repeated in the same samples by the microtitre well assay method using unblocked anti-peptide antiserum (squares) or antiserum that had been blocked by preincubation with peptide (triangles). The difference between these pairs of values (circles) represents the peptide-compatible, or Dbf2/Dbf20-specific kinase activity.

metaphase to anaphase transition and that therefore the essential function of Dbf2 during nuclear division is the phosphorylation, and hence regulation, of critical target proteins during this stage of the cell cycle. Furthermore, the activation of Dbf2 kinase potentially involves a dephosphorylation.

**The cell cycle-regulated expression of the DBF2 transcript precedes activation of Dbf2/Dbf20 kinase activity**

Expression of the DBF2 transcript is under cell cycle control (Johnston et al., 1990) whereas the DBF20 transcript is not under cell cycle control (Toyn et al., 1991). However, since Dbf2 accounts for the bulk of the kinase activity, periodic kinase activity might be influenced by periodic gene expression. This question was initially addressed by comparing the Dbf2/Dbf20 kinase activity with the level of DBF2 transcript in the same cell cycle experiment (Figure 7).

**Fig. 8. Cell cycle regulation of DBF2 mRNA expression is not required either for activation of the Dbf2 kinase or for the appearance of dephosphorylated Dbf2 protein.** A synchronous culture of strain S7-4A producing epitope tagged Dbf2 under the control of the MET3 promoter was prepared using the α-factor method in minimal medium. Methionine (1 mM) was added at time zero after release from α-factor arrest (marked by the vertical dashed line) and samples were taken during the course of the subsequent cell cycle. The values of all parameters in the culture before addition of the α-factor (3.5 h before release from α-factor) are shown to the left of the dashed line. (A) The percentage of cells with buds (circles) and the percentage of cells with divided chromatin (triangles) was estimated. (B) The level of DBF2 transcript (circles) was determined by phosphorimage analysis of a Northern blot and normalized to the level of actin transcript in each sample. All values were further normalized to a value of 100 arbitrary units for the sample taken from the culture before addition of α-factor. Dbf2/Dbf20 kinase activity (squares) was determined using the anti-peptide antibody; only the peptide compatible activity and hence Dbf2/Dbf20-specific activity is shown. The Dbf2 kinase activity was also determined using the 9E10 monoclonal antibody (triangles). Phosphorimage analysis was used to quantify kinase activity and the values were normalized to a value of 100 arbitrary units for the sample with maximal activity at 1 h 20 min. (C) Immunoblot of epitope tagged Dbf2 using the 9E10 monoclonal antibody corresponding to the timepoints shown in panels A and B.

Once again, a synchronous culture was made by the α-factor method and samples were taken every 10 min. Phosphorimage analysis of a Northern blot was carried out to measure the level of the Dbf2 transcript, and, as a control, the actin transcript. The DBF2 transcript was at a high level for the first 10 min of the experiment, when kinase activity was low,

although this could be accounted for if the kinase assay were less sensitive than the Northern hybridization used to detect the *DBF20* transcript (Toyn et al., 1991). Thus, elevated mRNA expression does not always lead to the activation of Dbf2/Dbf20 kinase activity.

The rapid activation of Dbf2/Dbf20 kinase during M phase occurs by a post-translational mechanism

To make a more thorough examination of the role of cell cycle regulation of *DBF2* mRNA expression in the activation of the kinase, *DBF2* was expressed under the control of a heterologous promoter that was not cell cycle-regulated. Thus, we could directly examine whether periodic activation of the kinase and the appearance of the dephosphorylated form of Dbf2 required periodic expression of the mRNA.

The c-myc epitope-tagged *DBF2* gene was placed under the control of the *MET3* promoter. The construct was then integrated into the strain S7-4A (*dbf2Δ*). It was then possible to repress expression of the *DBF2* gene, by the addition of methionine, and to detect the phosphorylation state of the Dbf2 protein by immunoblotting using the c-myc epitope tag (as used in Figure 6). A synchronously dividing culture of this strain was prepared by the α-factor method under inducing conditions of minimal medium in the absence of methionine. Methionine (1 mM) was added to the synchronized culture at time zero after release from α-factor to repress the *de novo* synthesis of Dbf2. Samples were then taken during the course of the ensuing cell cycle in order to detect *DBF2* mRNA, the phosphorylation state of Dbf2 protein and the level of Dbf2 kinase activity (Figure 8), as well as samples for determination of the percentage of cells with buds and divided chromatin, as described above. The amount of *DBF2* mRNA decreased by nearly 20-fold to a constant level within 30 min after addition of methionine. However, the repressed level of mRNA expression was still between four and five times higher than the average level of *DBF2* mRNA found in wild-type cells (data not shown). This probably explains the observation that even when the *MET3–DBF2* hybrid gene is repressed by methionine, it is still able to rescue growth of a *dbf2* ts mutant strain. Apparently, residual expression of *DBF2* from the *MET3* promoter provides enough Dbf2 for cell growth even when repressed by methionine; this was also true of *DBF2* expressed under the control of the *GAL1,10* promoter, suggesting that Dbf2 protein is required at a very low level for cell viability. This did not, however, prevent our investigating the activation of the Dbf2 kinase.

During the α-factor block the total amount of Dbf2 protein decreased, indicating that it was partly unstable during α-factor-mediated cell cycle arrest (Figure 8C). Two bands were detected, as expected, and the relative intensities of these bands changed during the course of the experiment. The samples taken at 10 min and between 1 h 10 min and 1 h 30 min contained the highest proportion of the lower band of dephosphorylated Dbf2. These changes in the relative proportions of the phosphorylation states of Dbf2 did not coincide with any change in the amount of *DBF2* mRNA, and therefore it is unlikely that *de novo* synthesis of Dbf2 can be the only mechanism that could account for the appearance of dephosphorylated Dbf2 protein. Figures 6C and 8C show a decrease in the amount of the upper, more phosphorylated, band of Dbf2 that coincided with the increase in the amount of the lower dephosphorylated band.

J.H.Toyn and L.H.Johnston

Thus, the simplest explanation for the increase in the amount of the lower band is that it results from a dephosphorylation event that converts the upper band into the lower band. Maximal kinase activity was also found in the samples taken between 1 h 10 min and 1 h 30 min, coinciding with the increase in the amount of dephosphorylated Dbf2 (Figure 8B). However, no sharp increase in kinase activity coincided with the presence of dephosphorylated Dbf2 during late G₁ (Figure 8C and also Figure 6C). Significantly, Dbf2 does have a function during G₁ (see Introduction) and therefore it is possible that the dephosphorylated state of Dbf2 during G₁ represents an active form of the kinase. If this form of the enzyme had a different substrate specificity it might account for the absence of detectable G₁ kinase activity with either of the two different kinase assays used. The timing of activation of Dbf2 kinase was confirmed by these two separate series of assays, one employing the usual anti-peptide antibody in the presence and absence of blocking peptide and the other in which the 9E10 monoclonal antibody was used (Figure 8B). This 9E10-based assay is specific for the c-myc epitope tagged Dbf2 and does not measure Dbf20 kinase. The maximum kinase activity occurred at 1 h 20 min for both kinase assays, but the 9E10-based assays showed a sharper peak of activation than the assays using the anti-peptide antibodies. The difference in the shapes of the peaks of kinase activity obtained with the two antibodies may reflect the contribution from Dbf20 kinase. Thus, cell cycle-regulated mRNA expression is clearly not required for kinase activation and therefore the change in the phosphorylation state of Dbf2 is likely to be a primary means of its activation during M phase. To date we have not been able to reconstitute the activation mechanism of Dbf2 in vitro despite testing the effect of dephosphorylation by commercially available phosphatases, presumably because a specific dephosphorylation is required. Of course, the kinase activity could also be activated by additional means such as complex formation.

One of the differences that can be seen between the experiments in Figures 6 and 8 is that the lower, dephosphorylated band of Dbf2 was present throughout the cell cycle when expressed under the control of the MET3 promoter (Figure 8) but present only transiently when expressed under the control of the wild-type DBF2 promoter (Figure 6). Since the MET3 promoter caused a higher level of expression of DBF2 mRNA throughout the cell cycle, even when repressed by the presence of methionine, it seems likely that under these conditions *de novo* synthesis of Dbf2 may well contribute to the appearance of the dephosphorylated form of Dbf2 in addition to the dephosphorylation mechanism proposed above. Furthermore, under the control of the MET3 promoter, a residual level of Dbf2 kinase activity appears to be detected in samples outside the time of maximal kinase activity using both kinase assays (Figure 8). This contrasts with the apparent lack of kinase activity during G₁ when Dbf2 was expressed under the control of its own promoter (Figure 7). Thus, overexpression of the DBF2 mRNA may lead directly to active kinase production by *de novo* synthesis of Dbf2, consistent with the idea that the dephosphorylated form leads to active kinase production.

To test this hypothesis, we constructed a strain in which DBF2 was highly overexpressed under the control of the constitutive phosphoglycerate kinase (PGK) promoter on a multicopy plasmid. In this strain, J211-2C (*dbf2Δ dbf20Δ*), we estimate, on the basis of immunoblotting, that the plasmid produces 100-fold or more of the wild-type level of Dbf2 protein (data not shown). A culture of this strain was synchronized by the α-factor method and samples were taken for measurement of the kinase activity through the cell cycle (Figure 9). As in Figures 7C and 8C, two sets of kinase assays were carried out in parallel using the anti-peptide antibody, one set in which the antibody had been blocked by addition of the synthetic C-terminal peptide and one set with antibody that had not been blocked. The difference between the two sets of measurements represents the Dbf2-specific kinase activity (Figure 9B). The kinase activity was now present throughout the cell cycle under these conditions of high overexpression, confirming our hypothesis that regulation of mRNA expression can play a role in production of active kinase. However, although kinase activity was present throughout the cell cycle, a marked increase in kinase activity took place with its normal timing during M phase. Taking these results together (Figures 7, 8 and 9), it seems that increased gene expression can lead to production of active kinase, especially when the gene is highly overexpressed, consistent with the dephosphorylated form of Dbf2 being required for active kinase production. However, the rapid peak of kinase activation during mitosis does not require increased expression of the mRNA at all, confirming that cell cycle regulation of the gene is not the principal means of regulating the kinase, which is likely to be post-translational.

### Discussion

To understand the role that Dbf2 carries out, it is important to know precisely when in the cell cycle it functions. We have shown, using *dbf2 ts* mutants, that in the absence of functional Dbf2, cells arrest in nuclear division with elongated spindles, divided chromatin and high Cdc28 mitotic kinase activity. The latest cell cycle stage that this arrest could represent is therefore telophase. However, by itself, these characteristics do not prove that Dbf2 normally functions during this stage of the cell cycle. It is formally possible that a cell cycle checkpoint (Hartwell and Weinert, 1989) causes arrest during nuclear division as a result of the absence of a molecular task that Dbf2 normally carries out in advance of nuclear division. This conjecture that Dbf2 functions before, but not after, nuclear division turned out not to be the case, since in dividing cells, functional Dbf2 was required for completion of the cell cycle even after the chromatin had already divided (Figure 3). The cells therefore required Dbf2 after the metaphase to anaphase transition. Thus, Dbf2 has an essential function during anaphase or telophase. This does not rule out an additional role for Dbf2 during other parts of the cell cycle and there is evidence for a role in late G₁ (Johnston *et al.*, 1990).

In order to study the molecular basis for the cell cycle function of the Dbf2/Dbf20 proteins, an assay for kinase activity was developed using antibody raised against a peptide corresponding to the C-terminus of Dbf2 and Dbf20. Immunoprecipitates of Dbf2/Dbf20, made using this antibody, catalysed transfer of phosphate from ATP to histone H1 in vitro. The assay was specific because extracts from *dbf2Δ* strains, which do not contain the Dbf2 protein, had much reduced kinase activity. In addition, assays carried out with peptide-blocked antibody gave strongly reduced activity, showing that the presence of Dbf2/Dbf20 in the immunoprecipitates was the cause of the kinase activity.

kinase activity of Dbf2 could also be measured using the c-myc epitope-tagged Dbf2 and the 9E10 monoclonal antibody. Further circumstantial evidence suggesting that we were detecting Dbf2 kinase specifically, was the coincidence of the activation of kinase activity with two Dbf2-specific cell cycle events; first, dephosphorylation of the Dbf2 protein, and second, the cell cycle requirement for Dbf2 function. In view of this, it was unexpected that the three ts alleles of DBF2 did not give rise to ts kinase activity in vitro. Thus, it is difficult to entirely eliminate the possibility that the kinase activity detected was co-immunoprecipitating with Dbf2. However, we feel it is more likely that the mutant alleles are affected for specificity or targeting of the kinase, rather than enzyme activity itself. So although the mutant proteins can catalyse phosphate transfer, they would be unable to phosphorylate their physiological substrates in vivo at the restrictive temperature. There is one other report of ts function of a kinase without ts kinase activity in vitro. This involves the cdc28-IN allele encoding the p36CDC28 mitotic kinase in budding yeast (Surana et al., 1991). Although yeast strains containing the cdc28-IN allele are ts for growth, the p36CDC28 kinase activity is not ts in vitro. The authors conjectured that the cdc28-IN allele was probably defective in its interaction with cyclins at the restrictive temperature.

In order to find out when in the cell cycle the Dbf2/Dbf20 kinase was activated, synchronous cultures were prepared and assays were carried out at different timepoints through the cell cycle. The timing of kinase activation was found to coincide approximately with the appearance of cells with divided chromatin, that is, when half of the cells in the synchronous culture have clearly observable divided chromatin, which presumably occurs shortly after the metaphase to anaphase transition. Therefore, the activation of Dbf2/Dbf20 kinase takes place during anaphase and/or telophase. Consistent with our interpretation of the timing of Dbf2/Dbf20 kinase activation was the observation that Cdc28 kinase, which is at a maximum during G2 (Surana et al., 1991), peaked 20 min earlier than the peak of Dbf2/Dbf20 kinase activity in the same experiment. The correlation of Dbf2/Dbf20 kinase activity with physiological activity, as determined by the execution point analysis, is also consistent with the Dbf2/Dbf20 kinase regulating its target protein substrates by phosphorylation during anaphase and/or telophase. The precise regulation of the Dbf2/Dbf20 kinase activity may therefore be an important factor in the regulation of cell cycle events during nuclear division after the metaphase to anaphase transition.

This regulation of the kinase activity might involve phosphorylation/dephosphorylation. Dbf2 protein migrated as a doublet during electrophoresis, the upper band being the more phosphorylated form. The appearance of the dephosphorylated form of Dbf2 coincided with activation of Dbf2 kinase during M phase (Figures 6 and 8). Since the appearance of the dephosphorylated form was accompanied by a corresponding decrease in the amount of the phosphorylated form, the simplest hypothesis is that a cell cycle-regulated dephosphorylation event is involved in activation of the Dbf2 kinase and that the dephosphorylated form is necessary for production of active kinase. However, the production of the dephosphorylated form is not by itself sufficient for maximal kinase activation, since it can be observed in the absence of detectable kinase activity near the beginning of the cell cycle (Figure 6, lanes 3, 4 and 5).

An alternative possibility is that the dephosphorylated form of Dbf2 in this case represents an active G1 form of the kinase, as proposed above, that we normally cannot detect using the in vitro kinase assay, perhaps because its substrate specificity is different. Alternative mechanisms for changes in the phosphorylation state and kinase activation could explain our observations, for example, cell cycle-regulated translational control of Dbf2 synthesis could account for the appearance of the dephosphorylated form involved in active kinase production and coincidental cell cycle-regulated degradation of the phosphorylated form could account for the decrease in the phosphorylated form. These alternative hypotheses cannot be ruled out but it seems more likely that the primary mechanism of activation of Dbf2 involves a dephosphorylation event after the metaphase to anaphase transition.

The DBF2 mRNA transcript is under cell cycle control (Johnston et al., 1990) and its expression is turned on in advance of the activation of the Dbf2/Dbf20 kinase (Figure 7). However, periodic expression of DBF2 was not required for the activation of the kinase activity, since expression of DBF2 under the control of the MET3 promoter abolished cell cycle regulation of the transcript, but did not affect activation of the kinase during M phase (Figure 8). Nevertheless, high overexpression of DBF2 under the control of the PGK promoter on a multicopy plasmid resulted in kinase activity throughout the cell cycle, although a peak of kinase activity was superimposed on this during M phase (Figure 9). Possibly, overexpression of the mRNA leads to production of active kinase because a post-translational deactivating mechanism becomes saturated. For example, the kinase that phosphorylates Dbf2 may be a limiting factor under these conditions, leading to a build-up of the dephosphorylated form. Thus, although de novo synthesis of Dbf2 can result in kinase activation, a post-translational mechanism is required for the maximal activation of kinase during M phase. This suggests a role for cell cycle-regulated expression of DBF2 mRNA in which it makes a small contribution to an essentially post-translational activation of kinase during M phase. Cells overexpressing DBF2, with resulting kinase activity throughout the cell cycle, do not appear to be greatly affected by it, although they do grow more slowly. Thus, taken at face value, the presence of kinase activity at the wrong time in the cell cycle apparently does not result in inappropriate execution of the M phase function of Dbf2. There are several possible explanations for this. First, it is likely that other levels of functional regulation may exist such as complex formation and intracellular targeting, which would mean that in spite of the activation of kinase, no function would be possible. This is consistent with the observation that the dbf2 ts mutants are conditional for function but not for kinase activity. Second, the kinase activity detected outside M phase may be a G1 form of the Dbf2 kinase, perhaps corresponding to the dephosphorylated form that is present during G1, and that we cannot normally detect using the in vitro assay. Functional evidence for a G1 form of Dbf2 kinase activity comes from the role of Dbf2 in the timing of S phase (Johnston et al., 1990) and the genetic interaction of DBF2 with SIT4, a phosphatase that is functionally required during G1 (Sutton et al., 1991; Parkes and Johnston, 1992). Thus, the high over-expression of Dbf2 may make this G1 form of the kinase more apparent.

In contrast to DBF2, expression of DBF20 is not cell cycle-
regulated and, in addition, we have not detected a gel mobility shift of the Dbf20 protein corresponding to different levels of phosphorylation, even in strains from which Dbf2 has been deleted (Figure 5). Therefore, although we cannot rule out the possibility that the same phosphorylation takes place on Dbf20, it is possible that Dbf20 is not regulated in exactly the same way as Dbf2. Certainly, there are a number of differences in the behaviour of the two proteins. First, deletion of the $DBF20$ gene did not decrease the kinase activity as severely as deletion of $DBF2$. Second, Dbf20 is not functionally identical to Dbf2, since, although $dbf2\Delta$ strains are viable, $dbf2$ ts strains are conditionally lethal, suggesting that Dbf20 can carry out the vital function only in the absence of Dbf2. Nevertheless, if Dbf20 does contribute kinase activity in the presence of Dbf2, it is under the same cell cycle control as Dbf2 (Figures 7 and 8). In order to address the question of whether Dbf20 is active during mitosis in wild-type yeast, the development of kinase assays that can distinguish Dbf20 from Dbf2 will be needed.

A number of cell cycle mutants have very similar cell cycle phenotypes to $dbf2$. These include $cdc5$ (Kitada et al., 1993) and $cdc15$ (Schweitzer and Philippsen, 1991), both of which encode protein kinases that can partially suppress the $dbf2-1$ allele when overexpressed in yeast (Kitada et al., 1993), and also $cdc14$ (Wan et al., 1992), which has homology to phosphotyrosine protein phosphatases. The Cdc28 kinase activity remains high in all these mutants; $dbf2$ (Figure 2), $cdc15$ (Surana et al., 1993), $cdc5$ and $cdc14$ (J.H.Toyn and L.H.Johnston, unpublished observations), suggesting that all of them are required for exit from M phase. Because of the similarity in cell cycle phenotypes and the genetic interactions between these genes and $dbf2$, it seems likely that they, like $dbf2$, are functioning in a process that takes place during anaphase and/or telophase. Furthermore, Cdc5 has a possible homologue in *Drosophila*, encoded by the *polo* gene, which is activated during late anaphase to telophase (Fenton and Glover, 1993). Thus, these genes could be involved in networks of regulatory phosphorylation that occur late in nuclear division and that are required for exit from M phase by destruction of cyclins and Cdc28 kinase activity. Of possible significance are the residues Tyr188 of Dbf2 and Tyr180 of Dbf20 (Toyn et al., 1991). These tyrosine residues are equivalent to Tyr15 of the p34${}^{cdc2}$ mitotic kinase of fission yeast, phosphorylation of which is involved in regulation of the function of p34${}^{cdc2}$ (Gould and Nurse, 1989). This raises the possibility that phosphorylation of this tyrosine in Dbf2 and Dbf20 might be important for their regulation, perhaps involving Cdc14, a potential tyrosine phosphatase as mentioned above. The purpose of this regulatory network would be to ensure the correct order of molecular events during nuclear division and surveillance of these events so that exit from M phase occurred at the appropriate time. In order to test this hypothesis, it will probably be necessary to identify the physiological substrates of these kinases and phosphatases and to make inducible dominant mutants to test the effect *in vivo* of changing the cell cycle regulation of the activities of these proteins.

## Materials and methods

### Yeast strains and media and growth conditions

The yeast strains used were CG378 ($MATa$ ade5 leu2-3,112 trp1-289 ura3-52), J99 (CG378 $dbf20\Delta::TRP1$), V378 (CG378 $spo12\Delta::TRP1$), J114 ($MATa$ $dbf2-2$ trp1 arg4), S7-4A ($MATa$ $dbf2\Delta::URA3$ his7 leu2-3,112 trp1-289 ura3-52), S7-4B ($MATa$ $dbf2\Delta::URA3$ ade5 leu2-3,112 trp1-289 ura3-52) and J211-2C ($MATa$ $dbf2\Delta::LEU2$ $dbf20\Delta::TRP1$ leu2 trp1 ura3 ade2-101). J211-2C requires the presence of a plasmid expressing $DBF2$ for viability. Yeast was grown in YPD broth (Sherman, 1991) or minimal medium made using yeast nitrogen base (Difco) according to the manufacturer’s instructions. The incubation temperature for yeast cultures was 30°C, except for the $dbf2$ strain, which was grown at a permissive temperature of 25°C or a restrictive temperature of 37°C. Synchronous cultures of $MATa$ cells were produced by the $\alpha$-factor method (Johnston et al., 1990).

### Plasmids and epitope tagging

For chromosomal integration of $DBF2$, the plasmid pRS304-$DBF2$ was used ($TRP1$, $DBF2$, AmpR), derived from pRS304 (Sikorski and Hieter, 1989) containing $DBF2$ on a 4.2 kb genomic restriction fragment cloned into the $BamHI$ site. For epitope-tagging of $DBF2$, a synthetic oligonucleotide with the sequence ATGGAAACAAAAATTAATCAGTGAAGAAGATTTAAAT, encoding the c-myc epitope recognized by the 9E10 monoclonal antibody (Evan et al., 1985), was cloned into the unique $StuI$ site of pRS304-$DBF2$, 37 codons from the 5′ ATG of $DBF2$. Both these plasmids were linearized using the unique $BglII$ site upstream of the $DBF2$ gene, before integration into the yeast genome (Rothstein, 1983). The epitope-tagged Dbf2 is functional in yeast, since an integrated copy of pRS304-$DBF2$-myc, expressing the epitope-tagged gene, complemented the growth defect in both $dbf2$ ts and $dbf2\Delta$ $dbf20\Delta$ double deletion yeast strains. To express the epitope-tagged Dbf2 under the control of the $MET3$ promoter, a 494 bp $XbaI-EcoRV$ fragment from pHAM8 (Mountain and Korch, 1991) containing the non-coding upstream region of the $MET3$ promoter was first cloned between the $XbaI$ and $SmaI$ sites of pRS306 (Sikorski and Hieter, 1989). The $MET3$ region was then reisolated from the resulting plasmid using the $SstI$ and $SalI$ sites of the pRS306 multiple cloning site and cloned into the plasmid pRS304-$DBF2$-myc between the $SalI$ site at the 5′ end of the $DBF2$ ORF and the $SstI$ site of the pRS304 multiple cloning site. The resulting plasmid, pRS304-$MET3$-$DBF2$-myc contains no DNA from the upstream region of the $DBF2$ gene, and can be put into yeast by integrative transformation at the $TRP1$ locus after linearization using the unique $Bsu36I$ site in the $TRP1$ gene. For expression of $DBF2$ under the control of the $PGK$ promoter, a 2 kb $HincII$ fragment containing the coding region of $DBF2$ was cloned downstream of the $PGK$ promoter of pMA91 (Mellor et al., 1983). The resulting plasmid, pMA-$DBF2$, has a 2 $\mu$ origin and the $LEU2$ selectable marker. For epitope-tagging of the $CDC28$ gene, the integrating plasmid pRD86 was used (obtained from R.Deshaises and P.Sorge). This is an integrating plasmid that contains $URA3$ and the C-terminal part of the $CDC28$ gene. The epitope tag, which is attached to the extreme C-terminus, is the hemagglutinin (HA) epitope recognized by the 12CA5 monoclonal antibody (Wilson et al., 1984). The plasmid was linearized using the unique $HindIII$ site in the $CDC28$ gene, before integrative transformation (Rothstein 1983), resulting in one complete epitope-tagged $CDC28$ gene and one N-terminally truncated $CDC28$ gene in the yeast genome. For overexpression of $DBF2$ and $DBF20$, the vectors YRp12-$DBF2$ (Johnston et al., 1991) and YRp12-$DBF20$ (Toyn et al., 1991) were used, respectively. Plasmids were introduced into yeast treated with lithium acetate (Ito et al., 1983).

### Anti-peptide antibody

A synthetic peptide with the amino acid sequence EHSDPFSTFY was covalently coupled to keyhole limpet hemocyanin (Calbiochem) using glutaraldehyde (Harlow and Lane, 1988) and unreacted peptide was removed by dialysis. Amino acid analysis of the product indicated that 0.17 mg of peptide were covalently bound per 1 mg of hemocyanin. A rabbit was immunized by subcutaneous injections of 1 mg of hemocyanin–peptide at three week intervals. The first injection contained Freund’s complete adjuvant; all subsequent boost injections contained Freund’s incomplete adjuvant. Test bleeds were taken 7 days after each boost injection.

### Cytological methods

Spindles were visualized by indirect immunofluorescence (Pringle et al., 1991). Cells were fixed and incubated successively with rat monoclonal anti-$\alpha$-tubulin (monoclonal YOL1/34 from Accurate Chemical and Scientific Corp.) and anti-rat FITC conjugate (Sigma). The cells were counterstained with DAPI (Sherman, 1991) and were observed and photographed using a Zeiss axiophot microscope.

### Detection of mRNA transcripts and proteins in yeast

The detection of the $DBF2$ and actin mRNA transcripts in yeast by Northern blot hybridization has been described (Johnston et al., 1990; Kitada et al., 1993). For detection of c-myc epitope-tagged $DBF2$ by immunoblotting,
an ECL kit (Amersham) was used according to the manufacturer's instructions, with a 1:5 dilution of 9E10 hybridoma supernatant as the first antibody (a gift from Dr A.Magee, NIMR). Pulse labelling with \[^{35}S\]sulfate, immunoprecipitation of denatured radiolabelled proteins and detection by gel fluorography has been described (Toyn *et al.*, 1988).

### Extraction of protein from yeast for kinase assays

At least \(5 \times 10^7\) cells per sample were harvested by centrifugation. The cell pellets were then frozen on dry ice in 1.5 ml Eppendorf tubes and could be stored without loss of kinase activity at \(-70^\circ \mathrm{C}\). Further manipulations were carried out on ice or at \(4^\circ \mathrm{C}\). To the cell pellet, \(100 \mu \mathrm{l}\) of lysis buffer [20 mM Tris pH 7.4, 100 mM NaCl, 10 mM EDTA, 1% Triton X-100, 5% glycerol, 1 mM each of \(\mathrm{NaF}, \beta\)-glycerophosphate, \(\mathrm{Na}_{3} \mathrm{VO}_{4}\), EGTA and sodium pyrophosphate, and \(5 \mu \mathrm{g} / \mathrm{ml}\) each of leupeptin, pepstatin A, chymostatin, aprotinin (Sigma) and AEBSF (Calbiochem)] were added and sufficient acid-washed glass beads (0.5 mm diameter, from Sigma) were added to fill up to the depth of liquid. Tubes were then vortexed vigorously for 1 min and fluid was removed from the beads, which were then washed once with another \(100 \mu \mathrm{l}\) of lysis buffer. The lysis buffer extract was combined in a fresh tube and centrifuged in a microfuge for 3 min to remove insoluble material. The protein concentration in the supernatant was measured (Lowry *et al.*, 1951). Protein extracts were stored at \(-70^\circ \mathrm{C}\).

### Dbf2/Dbf20 immunoprecipitation and kinase assay

Two microlitres of anti-peptide antiserum were added to yeast extract containing \(250 \mu \mathrm{g}\) of protein (typical volume \(20 \mu \mathrm{l}\)) and incubated for 1 h on ice. Twenty microlitres of a 1:1 slurry of protein A beads (protein A—Sepharose CL-4B, Pharmacia) were then added and incubation was continued, with mixing for a further 15 min. The beads were then washed by rounds of mixing and centrifugation successively in \(1 \mathrm{ml}\) each of lysis buffer, lysis buffer containing \(1 \mathrm{M} \mathrm{NaCl}\), lysis buffer again, and finally, assay buffer {25 mM 3-[N-morpholino]propanesulfonate (MOPS) pH 7.2, 60 mM b-glycerophosphate, 15 mM \(p\)-nitrophenylphosphate, 15 mM \(\mathrm{MgCl}_{2}, 5 \mathrm{mM}\) EGTA, \(1 \mathrm{mM}\) dithiothreitol, \(0.1 \mathrm{mM} \mathrm{Na}_{3} \mathrm{VO}_{4}, 10 \mu \mathrm{M}\) ATP and \(5 \mu \mathrm{g} / \mathrm{ml}\) each of leupeptin, pepstatin A, chymostatin and aprotinin}. The washed immunoprecipitate was then placed at \(30^{\circ} \mathrm{C}\), and \(10 \mu \mathrm{l}\) of assay buffer containing \(1 \mu \mathrm{Ci}\) of \([\gamma-{ }^{32} \mathrm{P}] \mathrm{ATP}\) and \(5 \mu \mathrm{g}\) of calf thymus H1 histone (type III-S, Sigma) were added. Incubation was continued for 30 min and stopped by addition of \(15 \mu \mathrm{l}\) SDS-PAGE sample buffer. A variation of this method utilized microtitre wells to which protein A (Pharmacia) had been adsorbed. Twenty-five microlitres of protein A solution (0.8 mg/ml in \(50 \mathrm{mM} \mathrm{Na}_{2} \mathrm{CO}_{3}\) ) was added to round-bottomed maxisorp wells (Nunc) and incubated for 4 h at room temperature. The wells were washed twice with \(50 \mathrm{mM}\) MOPS pH 7.2, and then incubated overnight at \(4^{\circ} \mathrm{C}\) with 300 \(\mu \mathrm{l} 5 \%\) bovine serum albumin (Sigma, fraction V) and washed again with MOPS. The immunoprecipitations were added to the microtitre wells at the same stage that protein A beads would otherwise have been added. Subsequent washes and kinase assays were carried out in the microtitre wells. For kinase assays of c-myc epitope-tagged Dbf2, \(1 \mu \mathrm{g}\) of affinity purified 9E10 monoclonal antibody was added to protein extracts prepared as described above. The subsequent procedure for assay of kinase activity was the same as described above, except that protein G beads (recombinant protein G—Sepharose 4B, Sigma) were used instead of protein A beads.

### Cdc28 immunoprecipitation and kinase assay

Cdc28 assays were similar to Dbf2/Dbf20 kinase assays; \(1 \mu \mathrm{l}(10 \mu \mathrm{g}\) protein) of 12CA5 monoclonal antibody (ascites fluid, from Babco) was added to yeast extract containing \(100 \mu \mathrm{g}\) of protein. The procedure was subsequently the same as for Dbf2/Dbf20 using protein A beads, except that ATP was used at a concentration of \(0.5 \mathrm{mM}\).

### Other methods

SDS-PAGE was carried out using \(10 \%\) acrylamide (Laemmli, 1973). Phosphoamino acid analysis of \({ }^{32} \mathrm{P}\)-labelled protein was carried out using acid hydrolysis and one-dimensional thin layer electrophoretic separation of the products. Unlabelled phosphoamino acids were added to the samples and their position of migration visualized by staining with ninhydrin (Cooper *et al.*, 1983).

## Acknowledgements

We thank Dr R.Deshaises (Berkeley) and Dr P.Sorge for providing the epitope-tagged Cdc28, Dr J.Diffley (ICRF, London) for helping us with the indirect immunofluorescence of spindles in yeast, Dr A.Magee (NIMR, UK) for providing 9E10 monoclonal antibody and Dr S.Sedgwick (NIMR, UK) for advice and suggestions during preparation of the manuscript.

---

### References

Cooper, J.A., Sefton, B.M. and Hunter, T. (1983) *Methods Enzymol.*, 99, 387–402.

Culotti, J. and Hartwell, L.H. (1971) *Exp. Cell Res.*, 67, 389–401.

Evan, G.I., Lewis, G.K., Ramsay, G. and Bishop, J.M. (1985) *Mol. Cell. Biol.*, 5, 3610–3616.

Fenton, B. and Glover, D.M. (1993) *Nature*, 363, 637–640.

Fernandez-Sarabia, M.J., Sutton, A., Zhong, T. and Arndt, K.T. (1992) *Genes Dev.*, 6, 2417–2428.

Gould, K.L. and Nurse, P. (1989) *Nature*, 342, 39–45.

Harlow, E. and Lane, D. (1988) *Antibodies: A Laboratory Manual*. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

Hartwell, L.H. and Weinert, T.A. (1989) *Science*, 246, 629–634.

Ito, H., Fukuda, Y., Murata, K. and Kimura, A. (1983) *J. Bacteriol.*, 153, 163–168.

Jackson, A.L., Pahl, P.M.B., Harrison, K., Rosamond, J. and Sclafani, R.A. (1993) *Mol. Cell. Biol.*, 13, 2899–2908.

Johnston, L.H. and Thomas, A.P. (1982) *Mol. Gen. Genet.*, 186, 439–444.

Johnston, L.H., Eberly, S.L., Chapman, J.W., Araki, H. and Sugino, A. (1990) *Mol. Cell. Biol.*, 10, 1358–1366.

Kitada, K., Johnston, L.H., Sugino, T. and Sugino, A. (1992) *Genetics*, 131, 21–29.

Kitada, K., Johnson, A.L., Johnston, L.H. and Sugino, A. (1993) *Mol. Cell. Biol.*, 13, 4445–4457.

Laemmli, U.K. (1970) *Nature*, 277, 680–685.

Lowry, O.H., Rosebrough, N.J., Farr, N.J. and Randall, R.J. (1951) *J. Biol. Chem.*, 193, 265–275.

Mellor, J., Dobson, M.J., Roberts, N.A., Tuite, M.F., Emtage, J.S., White, S., Lowe, P.A., Patel, T., Kingsman, A.J. and Kingsman, S.M. (1983) *Gene*, 24, 1–14.

Mountain, H.A. and Korch, C. (1991) *Yeast*, 7, 873–880.

Nasmyth, K. (1993) *Curr. Opin. Cell Biol.*, 5, 166–179.

Parkes, V. and Johnston, L.H. (1992) *Nucleic Acids Res.*, 20, 5617–5623.

Pringle, J.R., Adams, A.E.M., Drubin, D.G. and Haarer, B.K. (1991) *Methods Enzymol.*, 194, 565–602.

Rothstein, R. (1983) *Methods Enzymol.*, 101, 202–211.

Schweitzer, B. and Philippsen, P. (1991) *Yeast*, 7, 265–273.

Sherman, F. (1991) *Methods Enzymol.*, 194, 3–21.

Sikorski, R.S. and Hieter, P. (1989) *Genetics*, 122, 19–27.

Surana, U., Robitsch, H., Price, C., Schuster, T., Fitch, I., Futcher, A.B. and Nasmyth, K. (1991) *Cell*, 65, 145–161.

Surana, U., Amon, A., Dowzer, C., McGrew, J., Byers, B. and Nasmyth, K. (1993) *EMBO J.*, 12, 1969–1978.

Sutton, A., Immanuel, D. and Arndt, K.T. (1991) *Mol. Cell. Biol.*, 11, 2133–2148.

Toyn, J.H. and Johnston, L.H. (1993) *Genetics*, 135, 963–971.

Toyn, J.H., Hibbs, A.R., Sanz, P., Crowe, J. and Meyer, D.I. (1988) *EMBO J.*, 7, 4347–4353.

Toyn, J.H., Araki, H., Sugino, A. and Johnston, L.H. (1991) *Gene*, 104, 63–70.

Wan, J., Xu, H. and Grunstein, M. (1992) *J. Biol. Chem.*, 267, 11274–11280.

Wheals, A.E. (1987) In Rose, A. (ed.), *The Yeasts*. Academic Press, London, Vol. 1, pp. 283–390.

Wilson, I.A., Niman, H.L., Houghten, R.A., Cherenson, A.R., Connolly, M.L. and Lerner, R.A. (1984) *Cell*, 37, 767.

Yoon, H.-J., Loo, S. and Campbell, J.L. (1993) *Mol. Biol. Cell*, 4, 195–208.

Received on August 31, 1993; revised on December 6, 1993
