
JOURNAL OF BACTERIOLOGY, Nov. 1993, p. 7033–7044
0021-9193/93/217033-12$02.00/0
Copyright © 1993, American Society for Microbiology

Family of Glycosyl Transferases Needed for the Synthesis of Succinoglycan by Rhizobium meliloti

M. ALEXANDRA GLUCKSMANN,$^{\dagger}$ T. LYNNE REUBER,$^{\ddagger}$ AND GRAHAM C. WALKER$^{*}$

Department of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139

Received 19 April 1993/Accepted 27 August 1993

Rhizobium meliloti produces an acidic exopolysaccharide, termed succinoglycan or EPS I, that is important for invasion of the nodules that it elicits on its host, Medicago sativa. Succinoglycan is a high-molecular-weight polymer composed of repeating octasaccharide subunits. These subunits are synthesized on membrane-bound isoprenoid lipid carriers, beginning with a galactose residue followed by seven glucose residues, and modified by the addition of acetate, succinate, and pyruvate. Biochemical characterizations of lipid-linked succinoglycan biosynthetic intermediates from previously identified *exo* mutant strains have been carried out in our laboratory (T. L. Reuber and G. C. Walker, Cell 74:269–280, 1993) to determine where each mutation blocks the biosynthetic pathway. We have carried out a fine structure genetic analysis of a portion of the cluster of *exo* genes present on the second symbiotic megaplasmid of *R.* meliloti and have identified several new genes. In addition, the DNA sequence of 16 kb of the *exo* cluster was determined and the genetic map was correlated with the DNA sequence. In this paper we present the sequence of a family of glycosyl transferases required for the synthesis of succinoglycan and discuss their functions.

The plant symbiont Rhizobium meliloti produces an acidic exopolysaccharide, termed succinoglycan or EPS I, that plays an important role during the interaction with its host, Medicago sativa. Mutants of *R.* meliloti Rm1021 that fail to produce succinoglycan are deficient in the ability to invade alfalfa nodules (17, 36). Succinoglycan is a high-molecular-weight polymer composed of repeating octasaccharide subunits. Each subunit consists of seven glucose residues and one galactose residue and carries succinyl, acetyl, and pyruvyl modifications (1, 52) (Fig. 1). Recent evidence indicates that a low-molecular-weight form of succinoglycan is critical for nodule invasion and may function as a signal to the plant (2, 66). Interestingly, *R.* meliloti Rm1021 has a cryptic capacity to produce a second exopolysaccharide, with a structure quite different from that of succinoglycan, that is able to substitute for the symbiotic role of succinoglycan (19, 71). A requirement for exopolysaccharide during nodule invasion has also been demonstrated for other Rhizobium-legume symbioses (5, 11, 33).

Succinoglycan fluoresces under UV light when *R.* meliloti is grown on Calcofluor-containing agar plates (17, 36). By using this fluorescence as a screen, various classes of mutants that affect the production of succinoglycan have been isolated. Mutations in *exoA*, *exoB*, *exoC*, *exoF*, *exoL*, *exoM*, *exoP*, *exoQ*, *exoT*, and *exoY* completely abolish succinoglycan production (27, 36, 40, 44, 53, 70). The *exoB* and *exoC* mutations also affect the synthesis of other polymers (12, 19, 34) and have been shown to be defective in sugar nucleotide biosynthesis. *exoB* encodes a UDP-glucose-4-epimerase, which converts UDP-glucose to UDP-galactose (7, 9), and *exoC* codes for the enzyme phosphoglucosomutase, which synthesizes glucose-1-phosphate (67). The other *exo* genes listed above are apparently specific for succinoglycan production. Mutations in two other *exo* genes cause the production of altered forms of succinoglycan. Mutations in *exoH* cause production of EPS I that lacks the succinate modification (35), while mutations in a recently described locus, *exoZ* (7), cause the production of succinoglycan that lacks the acetyl modification (54). All these biosynthetic *exo* genes, except *exoC*, are clustered (27, 40, 44, 53, 70) on the second of two symbiotic megaplasmids present in *R.* meliloti (18, 23). *exoC* is located on the *R.* meliloti chromosome (18).

Succinoglycan is synthesized from uridine diphosphate-sugar precursors on membrane-bound isoprenoid lipid carriers (63–65). The octasaccharide subunits of succinoglycan are built up on these carriers, beginning with a galactose residue, followed by seven glucose residues, and modified by the addition of succinate, acetate, and pyruvate (55, 64). The octasaccharide subunits are then polymerized and finally secreted. Therefore, succinoglycan synthesis requires (i) enzymes for the production of each nucleotide sugar precursor, (ii) glycosyl transferases specific for each sugar linkage in the subunits, (iii) enzymes that add the modifications, and (iv) proteins involved in the polymerization and export of the polysaccharide. Biochemical characterizations of lipid-linked succinoglycan biosynthetic intermediates from previously identified *exo* mutant strains have been carried out in our laboratory in an effort to determine where each mutation blocks the biosynthetic pathway (55). As this biochemical characterization proceeded, it showed that the products of the *exoY*, *exoF*, *exoA*, *exoL*, and *exoM* genes are involved in the addition of the first four sugars of the subunit; and the products of the *exoP*, *exoQ*, and *exoT* genes appear to be needed for polymerization or transport of EPS I. However, the putative transferases that add the fifth, sixth, seventh, and eighth sugars of the subunit were unaccounted for. Therefore, it seemed likely that more *exo* genes remained to be identified.

One possible reason that genes encoding these late functions in the synthesis of the lipid-linked octasaccharide subunits were not identified in earlier studies is that the truncated succinoglycan subunits produced by strains carrying mutations in these genes could still be polymerized. In the case of Xanthomonas campestris, it has been shown that mutants that are blocked in the transfer of the fourth or fifth sugar to the lipid-linked subunit can produce altered forms of xanthan gum consisting of a three- or four-sugar repeat unit, respectively,

* Corresponding author.
† Present address: Millenium, Inc., Cambridge, MA 02139.
‡ Present address: Department of Molecular Biology, Massachusetts General Hospital, Boston, MA 02114.

[—Glc —β-1,4— Glc —β-1,4— Glc —β-1,3— Gal —β-1,4—]n

| β-1,6 |
| --- |
| acetyl |

Glc

| β-1,6 |
| --- |
| Glc |

| β-1,3 |
| --- |
| Glc⁶-succinyl (-) |

| β-1,3 |
| --- |
| Glc⁴-pyruvyl (-) |

FIG. 1. Structure of succinoglycan (EPS I) of *R. meliloti* Rm1021 (1, 52).

stimulated in part by the finding that a strain carrying a Tn5 insertion in this region, originally identified as no. 357 (40), fluoresced under UV light when grown on medium containing Calcofluor but was symbiotically defective. Indeed, this genetic search uncovered the presence of the genes *exoW*, *exoV*, and *exoU* (discussed below and in the accompanying paper [20]), all of which are required for the synthesis of succinoglycan (55). Our laboratory’s discovery of the biochemical function of most of the *exo* gene products motivated us to sequence 16 kb of the *exo* gene cluster present in the second megaplasmid in order to gain insight into the nature of the biosynthetic proteins of these *exo* gene products. This work revealed the presence of another previously unidentified *exo* gene, *exoO*. This work, together with the results of other studies (3, 7, 45, 50), made available the sequence of the entire 25-kb *exo* gene cluster from *exoP* to *exoB* (Fig. 2). In this paper we present the sequence of a novel family of glycosyltransferases needed for the synthesis of the octasaccharide subunits of succinoglycan (55). The members of this family are encoded by the previously identified *exoA*, *exoL*, and *exoM* genes (40) and by the newly identified *exoO*, *exoU*, and *exoW* genes. We will also discuss the *exoY* and *exoF* genes (45, 50), both of which we now know are needed for the addition of the first sugar to the lipid carriers (55). In the accompanying paper (20), we present further genetic analysis and sequences of other genes required for the biosynthesis of succinoglycan and propose a model for the synthesis of succinoglycan.

rather than the usual five-sugar repeat unit (4). If *R. meliloti* mutants blocked in the later stages of subunit assembly were able to produce altered forms of succinoglycan similar to the altered forms of xanthan gum, these mutants might still be somewhat fluorescent on Calcofluor medium. If this were the case, these strains might not have been identified in earlier screens for *exo* mutants, because these screens focused on the identification of colonies that were nonfluorescent on Calcofluor medium (36, 39, 53).

In hope of finding more *exo* genes required for the biosynthesis of succinoglycan, we carried out fine-structure genetic analysis on a region in the *exo* gene cluster (between *exoH* and *exoX* [Fig. 2B]) that had not been extensively characterized previously (40). Our decision to focus on this area was

---

**MATERIALS AND METHODS**

Strains, plasmids, and media. Bacterial strains and plasmids used in this study are listed in Table 1. Bacteria were grown in LB medium (41) with 2.5 mM MgSO₄ and 2.5 mM MgCl₂ added for *R. meliloti* cultures. For exopolysaccharide production, *R. meliloti* strains were grown in a modified M9 medium (41), with 1% mannitol as a carbon source and 0.1% glutamate

---

a

![Diagram](attachment:diagram.png)

b

Insertion number  
91 105 63 337 357  

Insertion  

Open reading frame  

exoW exoV exoU  

1 kb  

FIG. 2. (a) Map of the *exo* gene cluster on the second symbiotic megaplasmid of *R. meliloti* Rm1021. The boxes represent the ORFs, and the arrows indicate the orientation of transcription (3, 7, 40, 45, 50, 53). (b) Insertions in *exoU* and *exoW*. The large arrows indicate the orientation of the different complementation groups described in this paper and in the accompanying paper (20). The insertions were originally isolated on the plasmid pEX105 and then, when possible, were homogenized into the genome.

TABLE 1. Bacterial strains and plasmids

| Strain or plasmid | Relevant genotype or characteristic | Source or reference |
|-------------------|------------------------------------|---------------------|
| R. meliloti       |                                    |                     |
| Rm1021            | Su47, Sm'                         | F. Ausubel          |
| Rm8002            | Rm1021Pho-                        | 39                  |
| Rm8264            | Rm8002exoA264::TnphoA              | 53                  |
| Rm8401            | Rm1021exoA401::Tn5                | 40                  |
| Rm8407            | Rm1021exoA407::Tn5                | 40                  |
| Rm8431            | Rm1021exoL431::Tn5                | 40                  |
| Rm8457            | Rm1021exoM457::Tn5                | 40                  |
| Rm8545            | Rm1021exoO::Kan                   | This work           |
| Rm8292            | Rm1021exoU63::Tn5                 | This work           |
| Rm8337            | Rm1021exoU337::Tn5                | This work"          |
| Rm8357            | Rm1021exoU357::Tn5                | This work"          |
| Rm8285            | Rm1021exoW:28::Tn5                | This work           |
| Rm8284            | Rm1021exoW105::Tn5                | This work           |
| Rm8294            | Rm1021exoB294::Tn5-Tp             | J. Glazebrook       |
| E. coli MT614     | MM294malE::Tn5                    | T. Finan            |

| Plasmids         | Relevant genotype or characteristic                                                                 | Source or reference |
|------------------|------------------------------------------------------------------------------------------------------|---------------------|
| pEX105           | Tc', derivative of pRK404 carrying a 10.5-kb BglII fragment from pEX312                                  | This work           |
| pEX312           | Tc', pLAFR1 cosmid containing cloned exo genes                                                       | 40                  |
| pEX20            | Tc', derivative of pSUP104 carrying exoNexoOexoMexoA                                                 | 40                  |
| pRK600           | Cm', pRK2013npt::Tn9                                                                              | 18                  |
| pPH1JI           | Gm'Sp', IncP                                                                                      | 24                  |
| pH P45Ω:Kan      | Kan'Ap'                                                                                          | 16                  |
| pRK404           | Tc', IncP                                                                                        | 15                  |

"These strains are homogenotes of the corresponding Tn5 insertion in pD56, which were described by Long et al. (40)."

as a nitrogen source. The following antibiotics were used at the concentrations indicated: streptomycin (500 μg/ml), neomycin (200 μg/ml), tetracycline (10 μg/ml), gentamicin (20 μg/ml), and spectinomycin (100 μg/ml) for *R. meliloti* and chloramphenicol (20 μg/ml), kanamycin (25 μg/ml), and spectinomycin (100 μg/ml) for *Escherichia coli*. Calcofluor White M2R (Cellufluor [Polysciences] or Blue Brightener [Sigma]) were added to a final concentration of 0.02% in LB agar buffered to pH 7.4 with 10 mM HEPES (N-2-hydroxyethylpiperazine-N-2-ethanesulfonic acid).

Plant nodulation. *R. meliloti* strains were screened for their nodulation phenotypes on alfalfa (*Medicago sativa*) as described previously (36).

Genetic manipulations. Plasmids pRK404 and pLAFR1 and derivatives of these plasmids were transferred by triparental matings, by using the helper plasmid pRK600, as described previously (15). Tn5 mutagenesis of the region between *exoH* and *exoU* was accomplished by insertional mutagenesis of pEX105 (which contains a 10.5-kb BglII fragment from pEX312 [40] encompassing this region) by passaging this plasmid through the Tn5-carrying strain MT614, as described previously (51). The kanamycin insertion in the *exoO* gene was accomplished by blunt-end cloning a kanamycin cassette (of the plasmid pH P45Ω:Kan [16]) into the ESP I site of the plasmid pEX20 (40). Plasmid-borne Tn5 and kanamycin insertions were homogenotized into the *R. meliloti* genome by introduction of an incompatible plasmid, as has been described previously (58). All homogenotes were examined by Southern hybridization to confirm the presence of the Tn5 or kanamycin insertion in the appropriate position.

DNA manipulations and sequencing. Plasmid DNA was isolated from *E. coli* by the alkaline lysis method (41), and *R. meliloti* chromosomal DNA was isolated by the method of Marmur (43). Restriction enzyme digestions and ligations were performed according to the specifications of the suppliers (New England BioLabs, Bethesda Research Laboratories, or Boehringer Mannheim Biochemicals).

Plasmid pEX105, was constructed by cloning a 10.5-kb BglII fragment from pEX312 (40) into pRK404. Subclones of the *exo* gene cluster used for sequencing were made by cloning random EcoRI and BamHI fragments of pEX312 (40) into the EcoRI and BamHI sites of the plasmid Bluescript KSII+ (Stratagene). Smaller subclones were then made by further restricting the larger subclones. Sequencing was performed with Sequenase (U.S. Biochemical) and double-stranded or single-stranded miniprep DNA, as described in the Sequenase manual. Both strands were sequenced. Because of the high G+C content of *R. meliloti* DNA, extensive care was taken to ascertain that the sequence obtained was accurate. The program Codon Preference (Genetics Computer Group package [14]) was used to show the strong preference for G or C at the third position of the codons.

Fine mapping of the Tn5 insertions was performed by employing the polymerase chain reaction (PCR) to generate a DNA fragment containing one of the Tn5-chromosome junctions and then sequencing it. Each fragment was synthesized by PCR, using one oligonucleotide complementary to the inverted repeats of the Tn5 and another oligonucleotide complementary to the *exo* gene of interest. The double-stranded PCR fragment was then denatured with 0.2M NaOH, neutralized, precipitated, and used directly for sequencing. Oligonucleotides used for sequencing and for PCR were purchased from Operon.

Southern hybridization. Southern blotting and hybridization to GeneScreen Plus transfer membrane (New England Nuclear) were performed according to the manufacturer's instructions. DNA probes were labeled with [α-32P]dCTP (Amersham) by using a nick translation kit (Bethesda Research Laboratories).

Computer analysis of sequence data. The programs FASTA and BLAST (Genetics Computer Group package; National Center for Biotechnology Information), based on the method of Wilburn and Lipman (69), were used to search data banks for sequences with homology to the different open reading

A

1
ATG AGC TCA GAT GAA TTG ACG TCC AGC TCG AGC CTT ATC GTC ATA CCC TGT CTC AAC GAG GCC TCG CAT ATC GAG GCG CTG ATC GAA AAG CTG CGC CCG
M S D E L T S S L I V I P C L N E A S H I E A L I E K L R P
100
TCA CTC ACA CCG TTG AAC GCG CGG GTC GTC ATT GCC GAC GGC AGC ACG GAC GGA ACA CGG GAG ATC GCC CGG CGC CTT GCC ACT GAG GAT CCA CGG
S L T P L N A R V V I A D G G S T R E I A R R L A T E D P R
199
GTG CTC TTC CTC GAC AAT CCG AAG CGC ATA CAA AGC GCG GTC AAT CGT GCC GTC GCC GAA CTC GGC GCC GGC AGC GAC TAC CTG ATC CGC ATC GAC
V L F F L D N P K R I Q S A A V N R A V A E L G A G S D Y L I R I D
298
GCC CAC GGC ACC TAT CCG GAC GAT TAT TGC GAG CGG CTT GTC GAG GAT GCC TTG GCG ACC GGC GCG GAC TCG GTC GTG GTC GCC ATG CAG ACC GTC GGT
A H G T Y P D D Y C E R L V E D A L A T G A D S V V A M Q T V G
397
TTC AGC ACG TTC CAG AAG GCA ACG GCC TTC GCG CAG AAC TCC AAG CTC GGC AATGGC GGT TCG AAGCAC CGC ACC GGT GCC GTC GGT CAC TGG GCC GAG
F S T F Q K A T A F A Q N S K L G N G G S K H R T G A V G H W A E
496
CAC GGT CAC CAT GCA TTG ATG CGC ATC GAA GCC TTC AAG GCT GTC GGG GGC TAT GAC GAG AGC TTC AGC CAC AAC GAG GAC GCC GAA CTC GAC TAT CGC
H G H H A L M R I E A F K A V G G Y D E S F S H N E D A E L D Y R
595
CTC GGA AAG GCC GGC TAC CGG ATC TGG ATG ACC GAC AAG ACG ATG GTC TAC TAT CCG CGT GCC AAG CTC GTC CCG CTG TTC TGG CAA TAT TTC GGC
L G K A G Y R I W M T D K T S M V Y Y P R A K L V P L F W Q Y F G
694
TAC GGC CGC CGG GCA AAG AAC TTC AAG CTC GTC CCG CTG TTC TGG CAA TAT TTC GGC CTC AAG CAT CGC GCA ATG CCG GGG CTC AGG CAG ATG CTG
Y G R G R A K N F K L V P L F W Q Y F G L K H R A M P G L R Q M L
793
CCG CTT GCG GTG GCA CCC ATC GCT TTC GGC GCG CTT CTC GCG ATC GTC AAC TGG ATG GCC GTG GTG CCA GTC GGG GTT TGG GCT GCC GCA TGC CTC GGC
P L A V A P I A F G A L L A I V N W M A V V P V G V W A A A C L G
892 ▼ 264
TAT GGC GTC TGG ATG GCG CTC GGC CAG CGT AAT CCT TAT GGA CCG CTC GCC GGC GTT GCA GCC ATG GTC ATG CAC CTT GCC TGG TCC GCC GGG TTC TGG
Y G V W M A L G Q R N P Y G P L A A V A M V M H L A W S A G F W
991
CGG GAA CTC CTC GAC TTC CGC CGC AGG GTG GCC TAA
R E L L D F R R V A *

B

![Graph](#)

FIG. 3. (A) DNA sequence of the exoA ORF and deduced amino acid sequence. The arrowheads indicate the sequenced and mapped positions of the different Tn5 and TnphoA insertions described previously (40, 53). The amino acid residues that compose the potential membrane-spanning domains are underlined. (B) Hydrophobicity plot of ExoA. The most probable potential integral membrane segments are heavily shaded, and the possible integral membrane segments are lightly shaded (stippled). The positions of the insertions shown in panel A are indicated.

frames (ORFs) presented below. The program BESTFIT in the GCG package, which uses the “local homology” algorithm of Smith and Waterman (61), was used to perform sequence comparison. The percentages of identity and similarity given below were derived by using BESTFIT and the following parameters: a gap weight of 3.0 and a gap length weight of 0.10. A degree of similarity above 45% and a degree of identity above 21% were taken to be significant. The program ALOM based on the method of Klein et al. (29) and the program PepPlot (Genetics Computer Group package [14]), on the basis of the method of Kyte and Doolittle (31), were used to identify potential transmembrane segments. All the exo ORFs were also individually compared with the gum genes required for the synthesis of the extracellular polysaccharide of X. campestris (10, 68) and rfb genes needed for the synthesis of the lipopolysaccharide O-antigen of Salmonella typhimurium (26).

Exopolysaccharide analysis. Total hexose sugars were quantitated by the anthrone assay (38), and reducing sugars were quantitated by the method of Lever (37). For isolation of the exopolysaccharide, cultures were grown in M9-glutamate-mannitol medium (41) at 30°C for 7 days. Exopolysaccharide was precipitated from culture supernatants with 3 volumes of ethanol or with cetrimide (hexadecyltrimethyl ammonium bromide; Aldrich Chemical Co.), as described previously (36).

Precipitated exopolysaccharide was dialyzed against water and lyophilized. Nuclear magnetic resonance analysis of the exopolysaccharide isolated from different exo strains was performed by Russell Carlson, supported in part by funds from the Department of Energy-funded (DE-FG05-93ER20097) Center for Plant and Microbial Complex Carbohydrate Research in Athens, Georgia.

GenBank accession number. The accession number for the 16,260-bp sequence, extending from 150 bp downstream of exoP to 100 bp upstream of exoU, is L20758.

RESULTS

Sequence of glycosyl transferases needed for succinoglycan synthesis. We have sequenced 16 kb of the exo region of R. meliloti (Fig. 2) from exoP to exoU. In addition, we have undertaken a thorough genetic analysis of the central portion of the exo region. As a result of this effort we have identified 13 ORFs, including six genes encoding glycosyl transferases and six genes needed for other stages of succinoglycan synthesis. Below, the sequence of a family of six glycosyl transferases needed for the synthesis of succinoglycan will be discussed. Each member of this family utilizes UDP-glucose and transfers a glucose to the growing subunit. These glycosyl transferases will be discussed in the order in which they act in the synthesis

VOL. 175, 1993                                                                 GLYCOSYL TRANSFERASES FOR SUCCINOGLYCAN SYNTHESIS 7037

A


1
ATG CCG AAC GAA ACT CTT CAC ATC GAC ATC GGC GTC TGC ACG TAC CGC CGG GAG CTT GCC GAA ACG CTT CGC CTG GCT GCG ATG AAC GTG CCG
M P N E T L H I D I G V C T Y R R P E L A E T L R S , L A A M N V P

100
GAG CGC GCC AGG TTG CGC GTG ATC GTC GCC GAC AAC GCG GAA CCG AGC GCC AGG GCT CTG GTC GAA GGA CTT CGG CCG GAG ATG CCG TTC GAC ATC
E R A R L R V I V A D N A E P S A R A L V E G L R P E M P F D I

199
CTC TAC GTT CAC TGC CCT CAC TCG AAT ATC TCG ATC GCC CGC AAT TGC TGC CTC GAC AAC AGC ACC GGC GAC TTC CTC GCC TTC CTG GAT GAC GAC GAG
L Y V H C P H S N I S I A R N C C L D N S T G D F L A F L D D E

298
ACC GTT TCCGGC GAC TTG ACC CGC CTT CTG GAA ACG GCG CGC ACC ACG GGA GCC GCC GTG CTC GGC CCG GTC AGG GCG CAC TAC GGG CCT ACG
T V S G D W L T R L L E T A R T T G A A V L G P V R A H Y G P T

397
GCC CCC CGC TGG ATG CGC AGC GGC GAT TTC CAC TC G ACA CTG CCG GTC TGG GCA AAA GGT GAG ATC CGG ACC GGA TAC ACC TGC AAT GCG CTG CTC AGG
A P R W M R S G D F H S T L P V W A K G E I R T G Y T C N A L L R

496
CGC GAT GCG GCT TCG CTC CTT GGC CGG CGC TTC AAG CTT TCA CTC GGC AAG AGC GGC GAG GAC ACG GAT TTC TTC ACC GGG ATG CAC TGC GCC GGC
R D A A S L L G R R F K L S L G K S G G E D T D F F T G M H C A G

595
GGA ACG ATC GCC TTT TCG CCG GAA GCA TGG GTG CAC GAG CGG GTA CCG GAA AAC AGG GCC TCG CTT GCC TGG CTC GCG AAG CGT CGT TTC CGC TCC GGA
G T I A F S P E A W V H E P V P E N R A S L A W L A K R R F R S G

694
CAG ACC CAT GGA CGC CTT CTC GCC GAG AAA GCC CAT GGC TTG CGA CAG GCC TTG AAT ATC GCC CTT GCC GGG GCG AAA TCC GGC TTT TGC GCA ACT GCC
Q T H G R L L A E K A H G L R Q A W N I A L A G A K S G F C A T A

793
GCG GTC CTC TGC TTT CCT TCG GCC GCA AGG CGG AAC CGC TTT GCG CTA CGC GCC GTA CTG CAT GCC GGC GTC ATC AGC GGC CTG GGG CTT AAG GAA
A V L C F P S A A R R N R F A L R A V L H A G V I S G L L G L K E

893
ATC GAA CAA TAC GGA GCC CGC GAG GTG ACC TCG GCA TGA
I E Q Y G A R E V T S A *

B

![Graph](#)

FIG. 4. (A) DNA sequence of the exoM ORF and deduced amino acid sequence. The arrowhead indicates the sequenced and mapped position of the exoM457::Tn5 insertion described previously (40). The amino acid residues that compose the potential membrane-spanning domains are underlined. (B) Hydrophobicity plot of ExoM. The most probable potential integral membrane segments are heavily shaded, and the possible integral membrane segments are lightly shaded (stippled). The position of the exoM457::Tn5 insertion shown in panel A is indicated.

of the growing octasaccharide subunit. The products of the exoY and exoF genes are now known to be needed for the addition of galactose to the lipid carrier that initiates the synthesis of the succinoglycan subunit (55). The sequence of the exoY and exoF genes has already been published (45, 50) and will also be discussed below.

ExoA. The protein product of the exoA gene carries out the addition of the second sugar (glucose) of the succinoglycan subunit by catalyzing the formation of a β-1,3 linkage with the galactose on the lipid carrier (55). The deduced sequence of the exoA ORF is presented in Fig. 3A. The ORF for exoA codes for a 39-kDa protein. The hydrophobicity profile of the exoA ORF is presented in Fig. 3B, and the predicted transmembrane domains are highlighted. The hydrophobicity plots show that the protein is probably anchored at the carboxyl terminus to the cytoplasmic membrane. The exoA264::TnphoA insertion (53) maps 896 bp downstream of the start of the ORF (Fig. 3B) at the end of a DNA sequence encoding a transmembrane domain. The alkaline phosphatase moiety of the fusion protein produced by this mutant is predicted to be localized in the periplasm, which is consistent with the fact that TnphoA fusions are active only when the alkaline phosphatase portion is present in the periplasm (42). The exoA401::Tn5 and exoA407::Tn5 insertions (40) map 536 and 755 bp, respectively, downstream of the start of the ORF (Fig. 3). ExoA is partially related to ExoU (discussed below). No significant homology to the exoA ORF was found in the data base; some similarities at

the carboxyl end of the protein to various membrane-spanning proteins were found, but this is probably due to the hydrophobic nature of this region and is not necessarily related to any specific function.

ExoL. The protein product of the exoL gene is needed for the addition of the third sugar (glucose) and catalyzes the formation of a β-1,4 glucosidic linkage to the growing succinoglycan subunit on the lipid carrier (55). The sequence of this gene in *R. meliloti* has recently been published by Becker et al. (3) and matches the sequence that we obtained; therefore, it will not be presented here. We sequenced the junction of the exoL431::Tn5 insertion (40) and found that it maps 139 bp downstream from the start of the exoL ORF (data not shown). The sequence of the ExoL protein predicts it to be a cytoplasmic protein, with no stretches of hydrophobic amino acids. This is consistent with the analysis of 15 TnphoA-generated exoL insertions, none of which express alkaline phosphatase activity (32). The ExoL protein was found to be 21% identical and 50% similar at the amino acid level to the GlgB protein of *Synechococcus sp.* (28), a branching enzyme responsible for the formation of β-1,6 glycosidic linkages of glycogen. However, this degree of homology is very low and may be significant only with respect to the fact that both enzymes are glycosyl transferases.

ExoM. The ExoM protein is involved in the addition of the fourth sugar (glucose) to the growing succinoglycan subunit lipid-linked intermediates and catalyzes the formation of a

7038 GLUCKSMANN ET AL. J. BACTERIOL.

1
ATG AAC CCT CCG GCG ATC GAC AAG GTC CCC GAC GTC ACA TTC GTC GTG GCC GCC TAT AAT TCC GCG GAT ACG ATC GTG CGC GCG ATC GAA AGC GCG CTG
M N P P A I D K V P D V T F V V A A Y N S A D T I V R A I E S A L

100
GCG CAG GAA GGC GTG ACC GTC GAG GTC GTC GTC GAC GAT TGC TCA GCA GAT GCG ACG CCC GCG CTC GTT GCG GCG ATC CCG GAT CCC AGG GTC CGC
A Q E G V T V E V V D D C S A D A T P A L V A A I P D P R V R

199
CTG ATC GCC CTT GAT CGC AAC CGC GGT CCG GGC GGC GCA CGC AAT GCC GGC ATC GGG GCG GCG CGG TGG ATC GCC GTT CTC GAT TCG GAC GAT
L I A L D R N R G P G G A R N A G I G A A R G W I A V L D S D D

298
ACA GTC CGG CCG GAC CGG CTC AGG CGG ATG ATC GAG CGA GCG GAC GCG GCC GGG GCG CAG ATC GCC GTC GAC AAT CTC GAC GTC GTC CTG GAC GGA
T V R P D R L R R M I E R A A G A Q I A V D N L D V V S L D G

397
CGC AGT CTC AGG ATG TTC TCC GAG GCC GAA CTC GCC CGG CTG CCG CAG CTC ACG CTT CCC GCC TTC ATC GAG TCG AAC GTG CTC TTT CGA TCC GAG CAC
R S L R M F S E A L A R L P Q L T L P A F I E S N V L F R S E H

496
AAT TTC GGC TAC ATG AAA CGG ATC TTC GAG CGC CGC TTC CTG GAA AAC CAG CAG CTT CGC TTC GAC GAG GCG CTG CGC ATC GGC GAG GAC TAC ATC CTG
N F G Y M K P I F E R R F L E N Q Q L R F D E A L R I G E D Y I L

595
CTG GCA TCG GCC CTT GCC TGT GGC GGC CTG CGC GGT CGA GCC GTC CGC CGG CTA CAT TCA TAT CCG CGA GGG TTC CAT CTC GCG TGT GTT GAG
L A S A L A C G G A L R G R A V R R L H L S Y P R G F H L A C V E

694
GCT CGA TCA CAT AGA CGC GAT GAT CGC CGC CGA CGA GGC GTT CCT GCG CCG CTA CGC GCT CGA CGG CCT GGC TCA GAA AAT GCA GCA CAG GCG CAT GCG
A R S H R R D D R R R G V P A P L R A R R P G S E N A A Q A H A

793
CGG CTT CCG CGA AGC TCG CTC GTT CCT CGT CGA GCA GTT GAA AAA GAG ATC GTT GGC CGC GCT CAA GAC CGC TCT GGC CGA CCC GTT TGC
R L P R S S L V P G P R R A V E K E I V G R R A Q D R S G R P V C

892
GCT CCG GCA TCT CAG CAT GCC GAT CGC TGC CCG ATT GCG CCG CCT TGC CGC GCG TTT CGT GCA TCC CTC CCA TTC GGC CCC CGG TGC CGC CCC CGT
A P A S Q H A D R C P I A P P C R A F R A S L L P F G P P C R P R

991
GAC GGC GGC GGC TGA
D G G G *

FIG. 5. DNA sequence of the exoO ORF and deduced amino acid sequence. The arrowhead indicates the position of the kanamycin insertion described in Materials and Methods.

β-1,4 glucosidic linkage (55). The sequence of the exoM ORF and the predicted hydrophobicity plot are shown in Fig. 4. The junction of the exoM457::Tn5 insertion (40) was sequenced, and it maps 651 bp downstream from the start of the exoM ORF (Fig. 4A). The analysis of the probable membrane-spanning regions predicts that the 33.6-kDa ExoM protein is probably attached to the cytoplasmic membrane at the carboxyl terminus. No homology was found to any proteins in the database. Small regions of ExoM are homologous to other glycosyl transferases coded by exo genes and discussed below.

ExoO. When sequencing the exo gene cluster, a new ORF not corresponding to any known exo gene was identified between exoM and exoN and was named exoO. In order to determine whether the protein coded by exoO was involved in succinoglycan biosynthesis, a kanamycin cassette was inserted in the middle of the exoO ORF and homogenized into the second megaplasmid. The location of the kanamycin insertion was confirmed by Southern hybridization. The strain with a kanamycin insertion in the exoO gene had no visible phenotype (compared with the wild-type strain) on Calcofluor plates. exoO mutant strains produce a large amount of anthrone-positive material into culture supernatants, but 90% of it is insoluble. The nuclear magnetic resonance analysis of the material present in the dialyzed soluble fraction indicated that the mixture was complex but that all three substituents characteristic of succinoglycan (succinyl, acetyl, and pyruvyl) were present in an approximately 1:1:1 ratio. This observation indicated that the minor (soluble) component of the exopolysaccharide produced by an exoO strain is normal, fully modified succinoglycan. The nature of the major (insoluble) component has not yet been established directly. Nonetheless, the exoO::Kan homogenote was found to form ineffective nodules in symbiosis with alfalfa. Analysis of the biosynthetic succinoglycan intermediate produced by this strain has recently shown that this gene is required for the addition of the fifth sugar in the octasaccharide and therefore catalyzes the formation of a β-1,6 glucosidic linkage (55). Taken together, these results suggest that the insoluble carbohydrate that accumulates in an exoO mutant may consist of polymerized four-sugar subunits

1 Cel. syn. MPEVRSSSTQS ESGMSQWMGK ILSIRGAGLT IGVFGLCALI AATSVTLPPE
51 Cel. syn. QQLIVAFVCV VIFFIVGHKP SRRSQIFLEV LSGLVSLRYL TWRLTETLSF
101 Exoo . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
VOL. 175, 1993                                                                 GLYCOSYL TRANSFERASES FOR SUCCINOGLYCAN SYNTHESIS 7039


ATG ACC GCA GCC GCG CCG ACG GAC GTC TGC ATC ATC ATT TCC GCG AAG AAT GCC GCC GAC ACC ATA GCG CGC GCG GTC GCT TCG GCG CTT GCC GAA CCG
M   T   A   A   P   T   D   V   C   I   I   S   A   K   N   A   A   D   T   I   A   R   A   V   A   S   A   L   A   E   P
100
GAG GCG GCA GAA GTC GTC ATC GAT GAC GGC TCG ACC GAT GAC AGC GCT TCG GTG GCA CGC GCT GCG GAT GAC GGA ACG GGG CGG CTG AAT GTC GTT
E   A   A   E   V   V   V   I   D   D   G   S   T   D   D   S   A   S   V   A   R   A   A   D   D   G   T   G   R   L   N   V   V
199
CGC TTC GAA GAA AAC CGC GGC CCC GCG GCC CGC AAT CAT GCG ATC GCG ATC TCG CAT TCT CCG CTT ATC GGC GTG CTC GAC GCA GAT GAT TTC TTC
R   F   E   E   N   R   G   P   A   A   A   R   N   H   A   I   A   S   H   S   P   L   I   G   V   L   D   A   D   F   F
298
TTT CCG GGA CGC CTG GGC CAA TTG CTT TCG CAG GAC GGC TTG GAC TTC ATT GCG GAC AAC ATC GCG TTC ATC GAT GCC GCG CAG GCG GCA ACC GCG CAT
F   P   G   R   L   G   Q   L   L   S   Q   D   G   W   D   F   I   A   D   N   I   A   F   I   D   A   A   Q   A   A   T   A   H
397
GGC AGG ATC GAC CGA TTC GCC CCT ACC CCC CGG CTC ATC GAT CTC GTC GGC TTC GTC GAA GGA AAC ATA TCG CGA CGC GGC GTG CGG CGC GGC GAA ATA
G   R   I   D   R   F   A   P   T   P   R   L   I   D   L   V   G   F   V   E   G   N   I   S   R   R   G   V   R   R   G   E   I
496
GGC TTT CTG AAG CCG CTG ATG CGG CGC GCC TTC CTC GAC CAG CAT GGC CTG CGC TAC AAC GAG ACG CTG CGC CTC GGC GAA GAT TAC GAT CTC TAC GCT
G   F   L   K   P   L   M   R   R   A   F   L   D   Q   H   G   L   R   Y   N   F   T   L   R   L   G   E   D   Y   D   L   Y   A
595
CGC GCG CTC GCC AAT GGC GCA CGC TAC AAG ATC ATC CAC AGT TGC GGC TAT GCC GCC GTC GTC CGC GGC AAC TCG CTG AGC GGC AGT CAC CGG ACG ATC
R   A   L   A   N   G   A   R   Y   I   I   H   S   C   G   Y   A   A   V   V   R   G   N   S   L   S   G   S   H   R   T   I
694
GAC CTC AAG CGT CTC TAT GAG GCA GAT CGT GCA ATT CTC GCA GGG AGC CGA CTG AGC AGC GAC GCC GAA GCG GCG GTC CGC CGG CAC GAG CGC CAC ATT
D   L   K   R   L   Y   E   A   D   R   A   I   L   A   G   S   R   L   S   S   D   A   E   A   A   V   R   R   H   E   R   H   I
793
CGC GAC CGC CGC GAC CGC TAC GAA CTA CGG CAT TTT CTC GAT CTC AAG GGC TTT GGT CGT GCA TTC GGC TAC GCC CTG ACC CAC CGG GCG GCA CTG CCG
R   D   R   Y   E   L   R   H   F   L   D   L   K   N   Q   Q   G   F   G   R   A   F   G   Y   A   L   T   H   P   A   A   L   P
892
GCG ATC ATC GGC GGA ATC CTT GCG GAC AAG ACC GAA CGT TTT CGT CCG TCC GGT TCG CCG GCG CCC GTC GCT CTC GGT GGC AAG GGC GAC GTG CGC TAT
A   I   I   G   G   I   L   A   D   K   T   E   R   F   R   P   S   G   S   P   A   P   V   A   L   G   G   K   G   D   V   R   Y
991
CTG CTC GAG ACT CTT GCC GTG GAT CAG CCT CAA AAA TAG
L   L   E   T   L   A   V   D   Q   P   Q   K   *


FIG. 7. DNA sequence of the exoU ORF and deduced amino acid sequence. The arrowheads indicate the sequenced and mapped positions of the different Tn5 insertions described previously (40) and in the text.

and that some function in the cell is able to substitute inefficiently for the ExoO function so that a small amount of normal succinoglycan is synthesized.

The deduced sequence of the exoO ORF and the location of the kanamycin insertion are shown in Fig. 5. The ExoO protein has a molecular size of 38 kDa, and the analysis of its hydrophobicity suggests that it is a cytoplasmic protein. It is homologous with a number of proteins that are known to be involved in the formation of diverse glycosidic bonds, as well as with other proteins of unknown function. The ExoO protein is most homologous to the predicted product of an ORF downstream of the hetA gene of Anabaena sp. (21), as shown in Fig. 6. The overall degree of identity between this protein and ExoO is 29.7%. The HetA protein is involved in the formation of the heterocyst envelope polysaccharide of Anabaena sp., but the biochemical function of the downstream ORF is yet not known. Interestingly, ExoO is also 26.5% identical to cellulose synthetase from Acetobacter pasteurianus (59), which catalyzes the formation of β-1,4 glucosidic linkages. ExoO is also 23.8% identical to the NodC protein of R. meliloti (24) (Fig. 6). The NodC protein is itself homologous to chitin synthetases (8) and is thought to be involved in forming β-1,4 glycosidic bonds during the production of the Nod factor (13). The exoO gene product is also homologous to the ExoM protein described above and the ExoU and ExoW proteins described below. Each of these proteins adds a glucose moiety to the growing succinoglycan subunit on the lipid carrier but makes different β glucosidic linkages.

ExoU. The exoU gene, together with the exoW and exoV genes, was identified in a genetic analysis of the portion of the exo region between exoH and exoX (Fig. 2a). Strains carrying the exoU63::Tn5 mutation fluoresce with a blue, rather than a blue-green, hue on LB plates containing Calcofluor and lack the typical halo of fluorescence around the colonies. A similar phenotype was seen in strains Rm8337 and Rm8357, which were isolated in an earlier genetic analysis of the exo region (40) and carry insertions near the position of exoU63. These three mutations define a single complementation group. The exoU63 strain is Fix− on alfalfa. Originally, strains Rm8337

and Rm8357 were reported to be Fix+, but reexamination of these strains has shown that they are actually Fix−. Although the Calcofluor fluorescence of these strains would suggest that they produce some polymerized exopolysaccharide, we have been unable to isolate enough of this material from the supernatant of any exoU mutant strain for nuclear magnetic resonance analysis.

The predicted protein sequence of ExoU is presented in Fig. 7. Analysis of the hydrophobicity plot of the 32-kDa ExoU protein (data not shown) indicates that it is probably present in the cytoplasm. The exoU337::Tn5, exoU357::Tn5 (40), and exoU63::Tn5 mutations (20) map 138, 438, and 765 bp, respectively, downstream of the start of the exoU ORF (Fig. 7). Biochemical experiments have determined that the exoU gene product is most likely to be involved in adding the sixth sugar to the growing subunits on the lipid carriers and therefore catalyzes the formation of a β-1,6 glucosidic linkage (55). The ExoU protein is 37% identical to the ExoO protein (Fig. 8). This high degree of identity is not surprising, since both proteins are proposed to add glucose to closely related substrates by catalyzing the formation of β-1,6 linkages. The ExoU protein also shows an overall 26% identity with both the ExoA and ExoW proteins (discussed below), which catalyze the formation of β-1,3 bonds (Fig. 8). Because of its homology to the ExoO protein, the ExoU protein is also 25% identical to cellulose synthetase (59) and 28% identical to the ORF downstream of the hetA gene (21). Interestingly, both the ExoO and ExoU proteins are also 26 and 27% identical, respectively, to the Kps60 protein (56), a glycosyltransferase required for synthesis of the group II capsular exopolysaccharides of E. coli (56).

ExoW. The exoW complementation group was defined by three Tn5 insertions originally isolated in the plasmid pEX105: no. 91, 105, and 28 (Fig. 2B). Homogenote strains carrying the exoW28::Tn5 and exoW105::Tn5 mutations grow very slowly, taking 4 to 5 days to grow into colonies when streaked on plates, in comparison to 3 days for the wild-type strain. These homogenotes are unstable in the absence of selection and are initially nonfluorescent on Calcofluor medium, although slight

7040 GLUCKSMANN ET AL.

DISCUSSION

In this paper we present the sequence of a family of glycosyl transferases needed for the biosynthesis of succinoglycan. The results shown here, together with those of the accompanying paper (20) and the biochemical analysis (55), represent one of the most comprehensive studies on exopolysaccharide synthesis to date. This degree of understanding of exopolysaccharide synthesis for polysaccharides involving lipid-linked intermediates has been achieved only for the synthesis of xanthan gum of *X. campestris* (10, 68) and *Salmonella* lipopolysaccharide O-antigen biosynthesis (26). It is our hope that the sequence homologies among glycosyl transferases presented above may be informative for other systems.

None of the glycosyl transferases sequenced in this work were found to share homology with any of the gene products required for either xanthan gum or O-antigen biosynthesis. However, our laboratory first showed that the product of the *exoY* gene product is highly homologous to the *gumD* gene product of *X. campestris* (50). Müller et al. have recently shown that the *exoY* gene product is also homologous to the *rfbP* gene product of *Salmonella typhimurium* (45). The ExoY, GumB, and RfbP proteins are all thought to be glycosyl transferases required for the addition of the first sugar to the isoprenoid lipid carriers (26, 47, 55, 68). It seems likely that these proteins are homologous not only because they recognize a sugar moiety but also because they interact with the lipid carriers as they add the first sugar needed for the initiation of polysaccharide synthesis. Furthermore, we have recently found that ExoY is also 29% identical to the carboxyl half of CpsD, a galactosyl transferase needed for the synthesis type III capsule in *Streptococcus agalactiae* (57).

Mutations in the *exoF* gene also block the addition of first sugar to the lipid carriers (55). The sequence of the ExoF protein has been published recently (45), but the investigators reported no homology of this gene to any protein in the data base. We now report that the ExoF protein is 24.8% homologous to GumB of *X. campestris* (10) (Fig. 10), which has been postulated to be involved in polymerization of the repeating subunits of xanthan gum (68). Interestingly, we have also found the ExoF protein to be 23% homologous the periplasmic KpsD protein (56) needed for translocation of the capsular polysaccharide of *E. coli* (60). The subcellular localization of the ExoF protein is not known, but it contains a stretch of hydrophobic amino acids at its amino terminus (45) that probably anchors ExoF to the membrane. The rest of the protein is presumed to be in the periplasm, because of the activity of the *exoF369::TnphoA* fusion (39). Very little is known about the means of polymerization and export of exopolysaccharides across membranes. In *E. coli*, polymerization of the capsular polysaccharide is thought to occur on the inner face of the cytoplasmic membrane, followed by the transport to the cell surface through adhesion zones between the two cellular membranes (25, 30). In *Salmonella typhimurium*, O-antigen polymerization is thought to occur on the periplasmic side on the cytoplasmic membrane (46), but little is known about its export. Because of the large size of *R. meliloti* succinoglycan, it is tempting to consider that it is exported as it is being polymerized. If synthesis and export are indeed coupled during *R. meliloti* succinoglycan synthesis, one way to explain the phenotype of an *exoF* mutant strain would be to speculate that the absence of polymerization or translocation would somehow inhibit the first step in the synthesis of succinoglycan. ExoY is a small protein and is homologous only to the carboxyl half of GumD (68), RfbP (26), and CpsD (57). Another possible role for the ExoF protein is to play an accessory role which might be

FIG. 8. Alignment of ExoO, ExoU, ExoW, ExoA, and ExoM deduced amino acid sequences. Boldface letters represent amino acids that are identical between ExoO and ExoU and also to those of other proteins in the set. Underlined sequences indicate amino acids that are identical between two or more members of the set but are not identical between ExoO and ExoU. The asterisks indicate all the positions of identity.

Fluorescence is sometimes seen after a long incubation. However, no exopolysaccharide could be isolated from culture supernatants of these strains. The *exoW9I::Tn5* insertion could not be homogenotized in the wild-type background. However, all *exoW* insertions could be homogenotized without difficulty in an *exoB* background, and the homogenotes grew normally, indicating that these mutations are not deleterious if the cell is not able to make UDP-galactose and is therefore unable to initiate the synthesis of the succinoglycan subunit.

The deduced sequence of the ExoW protein is presented in Fig. 9A. The *exoW9I::Tn5* and *exoW28::Tn5* insertions map 8 and 700 bp, respectively, downstream of the start of the *exoW* ORF. Analysis of the hydrophobicity profile and of possible membrane-spanning regions (Fig. 9B) of the 26-kDa ExoW protein indicates that it is probably anchored at the carboxy terminal to the cytoplasmic membrane. The ExoW protein is thought to catalyze the addition of the seventh sugar to the subunits on the lipid carriers and therefore catalyzes the formation of a β-1,3 glucosidic bond (55). The ExoW protein shows no significant homology to any known protein in the database. As mentioned above, ExoW was found to be 26% identical to both the ExoO and ExoU proteins (Fig. 8).

VOL. 175, 1993 GLYCOSYL TRANSFERASES FOR SUCCINOGLYCAN SYNTHESIS 7041

A


1 ▼ 91  
ATG GCG AAG CTC ACA GTT GTC ATT CCG TAC TAC CAG AAG GAA CCC GGA ATT CTC CGG CGC GCG CTG GCA TCC GTT TTT GCT CAG AGC CTC GAG GAT TTC  
M A K L T V V I P Y Y Q K E P G I L R R A L A S V F A Q T L E D F  

100  
CAC GTT CTC GTG ATC GAT GAC GAA TCC CCC TAT CCG ATA GCG GAC GAG CTC GCC GGG CTC GCC CAG GAA GAA AGG GAG CGC ATC ACC GTC ATC CGC CAG  
H V L V I D D E S P Y P I A D E L A G L A Q E E R I T V I R Q  

199  
CCT AAC GGC GGG CCG GGT GGC GCG CGC AAT ACC GGT CTC GAT AAT GTT CCT GCC GAC AGC GAC TTC GTC GCT TTC CTC GAT TCC GAC GAC GTC TGG ACG  
P N G G P G G A R N T G L D N V P A D S D F V A F L D S D V W T  

298  
CCG GAT CAT CTC CTG AAC GCC TAT CAA AGC ATG AGC CGT TTC GAC GCC GAT TGC TAT TGG GCC TCG ATC ACG GGC GGC GAC GCT TTC CTC TAC TAT CAT TTC  
P D H L L N A Y Q S M T R F D A D C Y W A S I T G G D A F Y Y H F  

397  
GGG GTC GCG GAT CTC GAA AAG AGC GAG ACC GTC ACG CGC CTC TCG GAA AGT CCC CTC GTG GTC GAG CTC CCC GAA CTT CAG GAC GTC ATG CTG AAG AAC  
G V A D L E K S E T V T R L S E S P L V V E L Q D V M L K N  

496  
TTG AGC TTT CTC CAC ATG TCC TGC ATG GTG ATC GGG CGC AAG CTC TTC GAA AAG GTG CGT TTC GAG GCC ACG CTC AAG CTA GCG GCC GAG GAC GTG CTC  
W S F L H M S C M V I G R K L F E K V R F E A T L K L A A E D V_L  

595  
TTC TTC TGC GAT TGC GTG CTC GCA TCC AAG CGC GTG GTT CTT TGC GAT GCA GCG GGC GCG GTG GGC GAA GGG CTC AAC ATC TTT CAT TCG ATC GAT  
F F C D C V L A S K R V V L C D A A G A V R G E G L N I F H S I D  

694 ▼ 2θ  
AAC GAC TCG CCT CAG TTC CTG AAG CAG CAG TTC AAT ACC TGG GTT GCA CTC GAC ACG CTC GAA GGC CGA TAT CGT AAC CGG CCG AAC GGA TGG AGG CGA  
N D S P Q F L K Q Q F N T W V A L D T L E G R Y R N R P N G W R R  

793  
TCC GAT CCT ACA AGC ATA CCG CGC GGC AGG CGC TAT GGA GCC AGG CAC GCC GGA TAA  
S D P T S I P R G R R Y G A R H A G *  


B

![Graph](#)

FIG. 9. (A) DNA sequence of the exoW ORF and deduced amino acid sequence. The arrowheads indicate the sequenced and mapped positions of the different Tn5 insertions described in the text. The amino acid residues that compose the potential membrane-spanning domains are underlined. (B) Hydrophobicity plot of ExoW. The hydrophobicity was calculated by the method of Kyte and Doolittle (31). The computer program ALOM (29) was used to predict membrane-spanning domains. The most probable potential integral membrane segments are heavily shaded, and the possible integral membrane segments are lightly shaded (stippled). The positions of the insertions shown in panel A are indicated.

carried out by the amino half of the GumD, RfbP, and CpsD proteins.

Although at present we have no biochemical or genetic evidence for the formation of a multicomponent complex between enzymes required sequentially for the synthesis of succinoglycan in R. meliloti, few proteins exist free in the cell (22). The ExoA protein, the glucosyl transferase needed for the addition of the second sugar, appears to be anchored in the membrane at its carboxyl terminus. This may facilitate a possible interaction with ExoY and ExoF whose sequences (45, 50) and TnphoA fusions (39) predict them to be membrane bound. Of the glycosyl transferases that act subsequently, we predict that the ExoL (3), ExoO, and ExoU proteins are found in the cytoplasm. In contrast, the ExoM and ExoW proteins are anchored in the membrane. Both ExoH (3, 20) and ExoZ (7), which are required for the succinyl (35) and acetyl (54) modifications, respectively, appear to be transmembrane proteins (3, 7, 20). ExoM adds the fourth sugar to the succinoglycan subunit and ExoZ acetylates the third sugar, while ExoW adds the seventh sugar and ExoH succinylates the seventh sugar. It is possible that ExoM interacts with the ExoZ and that, likewise, ExoW interacts with ExoH and that the membrane localization of these four proteins has evolved to optimize this interaction.

The ExoL protein, which adds the third sugar to the

Percent Similarity: 47.887 Percent Identity: 24.883

ExoF 1 MQSNRRSGKSAGSRMVSCFTRLALLAALAASGASLARADEYRLGVMDKLR 50  
. : . | .|. .|.:.:. . || . . .|||:. . ||  
GumB 1 ...MSLGACSTGPEMASSLPHPDPPLAMSTV.......QPEYRLAPGDLLL 40  
. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
ExoM     R V I V A D N D A E P46 - - - - - - - - - -
ExoW     H V L V I D D E S P Y43 G P G G A R N T G77 L D S D D96
ExoA     R V V I A D G G S T D48 - - - - - - - - - -
ExoU     E V V V I D D G S T D47 G P A A A R N H A82 L D A D D97
ExoO     E V V V V D D G S A D51 G P G G A R N A G85 L D S D D99
Anab.ORF E I V I V D D G S T D85 G Q G G A F N A G123 L D A D D140
Kps60    E I L V C D D C S A D305 G P Y N I R N E L339 Q D A D D357
FBF15    S I V V V D D C S K D84 - - - - - - - - - -
NodC     Q V Y V V D D G S A N92 - - - - - - - - - -

FIG. 11. Alignment of regions of homolog between of ExoM, ExoW, ExoA, ExoU, ExoO, *Anabaena* sp. (*Anab.*) ORF (21), *E. coli* Kps60 (56), *Streptomyces aurantiacus* FBF15 protein (49), and *R. meliloti* NodC (24) deduced amino acid sequences. Identities between two or more sequences are shown in boldface. The positions of the different residues presented are indicated to the right of the homologous sequences.

lipid-linked subunit by catalyzing the formation of a β-1,4 linkage between two glucoses (the second and third sugars) (55), showed no significant degree of homology between any known protein or to any of the *exo* gene products sequenced, not even to ExoM, which also catalyzes the formation of a β-1,4 linkage between two glucoses (the third and fourth sugars). It is possible that the lack of homology between the ExoL and ExoM proteins is due to the fact the third sugar is already modified with an acetyl group before the ExoM protein makes the glycosidic bond, as observed in our studies of the intermediates accumulated in an *exoM* mutant (55). Therefore, the site of carbohydrate recognition of ExoM could be different from that of ExoL. A lack of homology between transferases with the same activities has also been observed in mammalian systems (48). In mammalian glycosyl transferase families, few regions of homology have been found within the catalytic domain, and no significant homology has been found with other proteins in GenBank (48).

The ExoO, ExoU, and ExoW proteins all are related. Because sequential enzymes handle the same substrate, they must have similar active sites. Srere proposed that the interaction of sequential enzymes could be postulated to have occurred by gene duplication and divergent evolution (62). It is conceivable that ExoO, ExoU, and ExoW indeed evolved by gene duplication and divergence. On the basis of our studies of succinoglycan biosynthetic intermediates (55), our laboratory has proposed that these proteins are required for the addition of the fifth, sixth, and seventh sugars, respectively. The regions with the higher degrees of homology are presented in Fig. 11. Interestingly, these regions of homology are also represented to various degrees in ExoA and ExoM and other gene products cited above, suggesting that they may have functional significance (Fig. 11). At this point, we cannot determine whether these conserved regions are required for substrate recognition or enzymatic activity.

As mentioned above, *exoU* mutant strains make very small amounts of Calcofluor-binding material, yet *exoO* mutant strains make a larger amount of exopolysaccharide, the majority of which is insoluble. The nuclear magnetic resonance analysis of the soluble fraction revealed that the exopolysaccharide recovered from an *exoO* mutant is normally modified by acetate, succinate, and pyruvate. Apparently, some other transferase can substitute for the ExoO function in vivo. The high degree of homology between ExoO and ExoU raises the possibility that ExoU might substitute for ExoO. However, because *exoU* mutant strains produce very little polysaccharide, the converse cannot be true. In contrast to *exoO* and *exoU* mutants, *exoW* mutant strains produce very little extracellular polysaccharide, even though they synthesize a lipid-linked subunit that lacks only the last two sugars in the subunit (55). It is possible that this six-sugar subunit is not polymerized.

Breedveld et al. (6) have recently characterized a mutant of *Rhizobium leguminosarum* that lacks the terminal sugar of the exopolysaccharide and also produces very little exopolysaccharide.

In general, enzymes of a multicomponent complex are linearly and closely located on a chromosome (62). To date, the genes required for the synthesis of bacterial exopolysaccharides have generally been found to be clustered in long operons (25, 26, 33, 68). As shown in Fig. 2, all the genes to the left of *exoT* are transcribed in one orientation (from right to left). There is an overlap of the start and stop codons of the protein downstream of *exoH*, except for the 100-bp gap between the *exoK* and *exoL* genes and the 48-bp gap between *exoO* and *exoN*. This observation raises the possibility that these genes are translationally coupled and may represent a mechanism by which the cell ensures that the various components of the succinoglycan biosynthetic machinery are expressed in equimolar ratios. It also suggests that they are transcribed as a single unit. We have preliminary evidence for a promoter upstream of *exoH* that may be responsible for the expression of all the genes to the left of *exoH*. The *exo* genes are expressed at very low levels (39, 53); thus, mapping of the site of transcription initiation is difficult. We are presently carrying out experiments to confirm the existence of the promoter upstream of *exoH* and to map other promoters on the right side of the *exo* region, where the genes do not seem to be organized in long operons.

ACKNOWLEDGMENTS

We thank I. S. Roberts for sharing both unpublished results and sequence data with us. We also thank M. C. Capage for helpful discussion of unpublished data.

This work was supported by Public Health Service grant GM31031 to G.C.W. and American Cancer Society Postdoctoral Fellowship PF3359 to M.A.G. T.L.R. was supported by predoctoral fellowships from the National Science Foundation, the National Institutes of Health, and Merck.

REFERENCES

1. Aman, P., M. McNeil, L.-E. Franzen, A. G. Darvill, and P. Albersheim. 1981. Structural elucidation, using HPLC-MS and GLC-MS, of the acidic exopolysaccharide secreted by *Rhizobium meliloti* strain Rm1021. Carbohydr. Res. **95**:263–282.

2. Battisti, L., J. C. Lara, and J. A. Leigh. 1992. Specific oligosaccharide form of the *Rhizobium meliloti* exopolysaccharide promotes nodule invasion in alfalfa. Proc. Natl. Acad. Sci. USA **89**:5625–5629.

3. Becker, A., A. Kleickmann, W. Arnold, and A. Pühler. 1993. Analysis of the *Rhizobium meliloti exoH, exoK, exoL* fragment: ExoK shows homology to excreted endo-β 1,3-1,4 glucanases and ExoH resembles membrane proteins. Mol. Gen. Genet. **238**:145–154.

4. Betlach, M. R., M. A. Capage, D. H. Doherty, R. A. Hassler, N. M.

Henderson, R. W., Vanderslice, J. D., Marrelli, and M. B. Ward. 1987. Genetically engineered polymers: manipulation of xanthan biosynthesis, p. 35–50. In M. Yalpani (ed.), Industrial polysaccharides: genetic engineering, structure/property relations and applications. Elsevier, Amsterdam.

5. Borthakur, D., C. E. Barber, J. W. Lamb, M. J. Daniels, J. A. Downie, and A. W. B. Johnston. 1986. A mutation that blocks exopolysaccharide synthesis prevents nodulation of peas by Rhizobium leguminosarum but not of beans by *R. phaseoli* and is corrected by cloned DNA from *Rhizobium* or the phytopathogen *Xanthomonas*. Mol. Gen. Genet. 203:320–323.

6. Breedveld, M. W., H. C. J. Canter-Cremers, M. Batley, M. A. Posthumus, L. P. T. M. Zevenhuizen, C. A. Wijffelman, and A. J. B. Zehnder. 1993. Polysaccharide synthesis in relation to nodulation behavior of *Rhizobium leguminosarum*. J. Bacteriol. 175:750–757.

7. Buendia, A. M., B. Enenkel, R. Köplin, K. Niehaus, W. Arnold, and A. Pühler. 1991. The *Rhizobium meliloti* exoZ/exoB fragment of megaplasmid 2: ExoB functions as a UDP-glucose-4-epimerase and ExoZ shows homology to NodX of *Rhizobium leguminosarum* biovar. *viciae* strain TOM. Mol. Microbiol. 5:1519–1530.

8. Bulawa, C. E., and W. Wasco. 1991. Chitin and nodulation. Nature (London) 353:710–711.

9. Canter-Cremers, H. C. J., M. Batley, J. W. Redmond, L. Eydems, M. W. Breedveld, L. P. T. M. Zevenhuizen, E. Pees, C. J. Wijffelman, and B. J. J. Lugtenberg. 1990. *Rhizobium leguminosarum* exoB mutants are deficient in the synthesis of UDP-glucose 4-epimerase. J. Biol. Chem. 265:21122–21127.

10. Capage, M. A., D. H. Doherty, M. R. Betlach, and R. van der Slice. October 1987. Recombinant-DNA mediated production of xanthan gum. Patent. International Publication Number WO87/05938.

11. Chen, H., M. Batley, J. W. Redmond, and B. G. Rolfe. 1985. Alteration of the effective nodulation properties of a fast-growing broad host range *Rhizobium* due to changes in exopolysaccharide synthesis. J. Plant Physiol. 120:331–349.

12. Clover, R. H., J. Kieber, and E. R. Signer. 1989. Lipopolysaccharide mutants of *Rhizobium meliloti* are not defective in symbiosis. J. Bacteriol. 171:3961–3967.

13. Dénarié, J., F. Debelle, and C. Rosenberg. 1992. Signal and host range variation in nodulation. Annu. Rev. Microbiol. 46:497–531.

14. Devereux, J., P. Haeberli, and O. Smithies. 1984. A comprehensive set of sequence analysis programs for the VAX. Nucleic Acids Res. 12:387–395.

15. Ditta, G., T. Schmidhauser, E. Yakobson, P. Lu, X.-W. Liang, D. R. Finlay, D. Guiney, and D. R. Helinski. 1985. Plasmids related to the broad host range vector, pRK290, useful for gene cloning and for monitoring gene expression. Plasmid 13:149–153.

16. Fellay, R., J. Frey, and H. Krisch. 1987. Interposon mutagenesis of soil and water bacteria: a family of DNA fragments designed for *in vitro* insertional mutagenesis of Gram-negative bacteria. Gene 52:147–154.

17. Finan, T. M., A. M. Hirsch, J. A. Leigh, E. Johansen, G. A. Kuldau, S. Deegan, G. C. Walker, and E. R. Signer. 1985. Symbiotic mutants of *Rhizobium meliloti* that uncouple plant from bacterial differentiation. Cell 40:869–877.

18. Finan, T. M., B. Kunkel, G. F. de Vos, and E. R. Signer. 1986. Second symbiotic megaplasmid in *Rhizobium meliloti* carrying exopolysaccharide and thiamine synthesis genes. J. Bacteriol. 167:66–72.

19. Glazebrook, J., and G. C. Walker. 1989. A novel exopolysaccharide can function in place of the Calcofluor-binding exopolysaccharide in nodulation of alfalfa by *Rhizobium meliloti*. Cell 56:661–672.

20. Glücksmann, M. A., T. L. Reuber, and G. C. Walker. 1993. Genes needed for the modification, polymerization, export, and processing of succinoglycan by *Rhizobium meliloti*: a model for succinoglycan biosynthesis. J. Bacteriol. 175:7045–7055.

21. Holland, D., and C. P. Wolk. 1990. Identification and characterization of *hetA*, a gene that acts early in the process of morphological differentiation of heterocysts. J. Bacteriol. 172:3131–3137.

22. Hrazdina, G., and R. A. Jensen. 1992. Spatial organization of enzymes in plant metabolic pathways. Annu. Rev. Plant Physiol. 43:241–267.

23. Hynes, M. F., R. Simon, P. Müller, K. Niehaus, M. Labes, and A. Pühler. 1986. The two megaplasmids of *Rhizobium meliloti* are involved in the effective nodulation of alfalfa. Mol. Gen. Genet. 202:356–362.

24. Jacob, T. W., T. T. Egelhoff, and S. R. Long. 1985. Physical and genetic map of a *Rhizobium meliloti* nodulation gene region and nucleotide sequence of *nodC*. J. Bacteriol. 162:469–476.

25. Jann, B., and K. Jann. 1991. Structure and biosynthesis of the capsular antigens of *Escherichia coli*. Curr. Top. Microbiol. Immunol. 150:19–42.

26. Jiang, X.-M., B. Neal, F. Santiago, S. J. Lee, L. K. Romana, and P. R. Reeves. 1991. Structure and sequence of the *rfb* (*O* antigen) gene cluster of *Salmonella* serovar *typhimurium* (strain LT2). Mol. Microbiol. 5:695–713.

27. Keller, M., P. Müller, R. Simon, and A. Pühler. 1988. *Rhizobium meliloti* genes for exopolysaccharide synthesis and nodule infection located on megaplasmid 2 are actively transcribed during symbiosis. Mol. Plant-Microbe Int. 1:267–274.

28. Kiel, J. A. K. W., J. M. Boels, G. Belman, and G. Venema. 1990. Nucleotide sequence of the *Synechococcus* sp. PCC7942 branching enzyme (GlgB): expression in *Bacillus subtilis*. Gene 89:77–84.

29. Klein, P., M. Kanehisa, and C. DeLisi. 1985. The detection and classification of membrane spanning proteins. Biochim. Biophys. Acta 815:468–476.

30. Kröncke, K.-D., G. Boulnois, I. Roberts, D. Bitter-Suermann, J. R. Golecke, B. Jann, and K. Jann. 1990. Expression of the *Escherichia coli* K5 capsular antigen: immunoelectron microscopic and biochemical studies with recombinant *E. coli*. J. Bacteriol. 172:1085–1091.

31. Kyte, J., and R. F. Doolittle. 1982. A simple method for displaying the hydropathic character of a protein. J. Mol. Biol. 157:105–132.

32. LeCour, L., T. L. Reuber, and G. C. Walker. Unpublished results.

33. Leigh, J. A., and D. L. Coplin. 1992. Exopolysaccharides in plant-bacterial interactions. Annu. Rev. Microbiol. 46:307–346.

34. Leigh, J. A., and C. C. Lee. 1988. Characterization of polysaccharides of *Rhizobium meliloti* exo mutants that form ineffective nodules. J. Bacteriol. 170:3327–3332.

35. Leigh, J. A., J. W. Reed, J. F. Hanks, A. M. Hirsch, and G. C. Walker. 1987. *Rhizobium meliloti* mutants that fail to succinylate their Calcofluor-binding exopolysaccharide are defective in nodule invasion. Cell 51:579–587.

36. Leigh, J. A., E. R. Signer, and G. C. Walker. 1985. Exopolysaccharide-deficient mutants of *R. meliloti* that form ineffective nodules. Proc. Natl. Acad. Sci. USA 82:6231–6235.

37. Lever, M. 1972. A new reaction for colorimetric determination of carbohydrates. Anal. Biochem. 47:273–279.

38. Loewus, F. A. 1952. Improvement in the anthrone method for determination of carbohydrates. Anal. Chem. 24:219.

39. Long, S., S. McCune, and G. C. Walker. 1988. Symbiotic loci of *Rhizobium meliloti* identified by random *TnphoA* mutagenesis. J. Bacteriol. 170:4257–4265.

40. Long, S., J. W. Reed, J. Himawan, and G. C. Walker. 1988. Genetic analysis of a cluster of genes required for synthesis of the Calcofluor-binding exopolysaccharide of *Rhizobium meliloti*. J. Bacteriol. 170:4239–4248.

41. Maniatis, T., E. F. Fritsch, and J. Sambrook. 1982. Molecular cloning: a laboratory manual. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

42. Manoil, C., and J. Beckwith. 1985. *TnphoA*: a transposon probe for protein export signals. Proc. Natl. Acad. Sci. USA 82:8129–8133.

43. Marmur, J. 1961. A procedure for the isolation of deoxyribonucleic acid from micro-organisms. J. Mol. Biol. 3:208–218.

44. Müller, P., M. Hynes, D. Kapp, K. Niehaus, and A. Pühler. 1988. Two classes of *Rhizobium meliloti* infection mutants differ in exopolysaccharide production and in coinoculation properties with nodulation mutants. Mol. Gen. Genet. 211:17–26.

45. Müller, P., M. W. M. Weng, J. Quandt, W. Arnold, and A. Pühler. 1993. Genetic analysis of *Rhizobium meliloti exoYFQ* operon: ExoY is homologous to sugar transferases and ExoQ represents a transmembrane protein. Mol. Plant-Microbe Int. 6:55–65.

46. Osborn, M. J. 1984. Biogenesis of the outer membrane of *Salmonella* Harvey Lect. 78:87–103.

47. Osborn, M. J., and R. Yuan Tze-Yuen. 1968. Biosynthesis of

bacterial lipopolysaccharide: enzymatic formation of the first intermediate in biosynthesis of the O-antigen of Salmonella typhimurium. J. Biol. Chem. 243:5145–5152.

48. Paulson, J. C., and K. J. Colley. 1989. Glycosyltransferases. J. Biol. Chem. 264:17615–17618.

49. Pospiech, R., B. Neumann, and H. Schairer. 1992. FBF15 protein from *Stigmatella aurantiaca*. EMBL reference no. S18962.

50. Reed, J. W., M. Capage, and G. C. Walker. 1991. *Rhizobium meliloti exoG* and *exoJ* mutations affect the ExoX-ExoY system for modulation of exopolysaccharide production. J. Bacteriol. 173: 3776–3788.

51. Reed, J. W., and G. C. Walker. 1991. The *exoD* gene of *Rhizobium meliloti* encodes a novel function needed for alfalfa nodule invasion. J. Bacteriol. 173:664–677.

52. Reinhold, B., S. Y. Chan, T. L. Reuber, A. Marra, G. C. Walker, and V. N. Reinhold. Detailed structural characterization of succinoglycan, the major symbiotically important exopolysaccharide of *Rhizobium meliloti* strain Rm1021. Submitted for publication.

53. Reuber, T. L., S. Long, and G. C. Walker. 1991. Regulation of *Rhizobium meliloti exo* genes in free-living cells and in planta examined by using TnphoA fusions. J. Bacteriol. 173:426–434.

54. Reuber, T. L., and G. C. Walker. 1993. The acetyl substituent of succinoglycan is not necessary for alfalfa nodule invasion by *Rhizobium meliloti* Rm1021. J. Bacteriol. 175:3653–3655.

55. Reuber, T. L., and G. C. Walker. 1993. Biosynthesis of succinoglycan, a symbiotically important exopolysaccharide of *Rhizobium meliloti*. Cell 74:269–280.

56. Roberts, I. S. Personal communications.

57. Rubens, C. E., L. H. Heggen, R. F. Haft, and M. R. Wessels. 1993. Identification of CpsD, a gene essential for type III capsule expression in group B *Streptococci*. GenBank no. L09116.

58. Ruvkun, G. B., V. Sundaresan, and F. M. Ausubel. 1982. Directed transposon Tn5 mutagenesis and complementation analysis of *Rhizobium meliloti* symbiotic nitrogen fixation genes. Cell 29:551–559.

59. Saxena, I. M., F. C. Lin, and R. M. Brown, Jr. 1990. Cloning and sequencing of the cellulose synthase catalytic subunit gene of *Acetobacter xylinum*. Plant. Mol. Biol. 15:673–683.

60. Silver, R. P., W. Aaronson, and W. F. Vann. 1987. Translocation of capsular polysaccharides in pathogenic strains of *Escherichia coli*

requires a 60-kilodalton periplasmic protein. J. Bacteriol. 169: 5489–5495.

61. Smith, T. F., and M. S. Waterman. 1981. Identification of the common molecular sequences. J. Mol. Biol. 147:195–197.

62. Srere, P. A. 1987. Complexes of sequential metabolic enzymes. Annu. Rev. Biochem. 56:89–124.

63. Sutherland, I. W. 1985. Biosynthesis and composition of gram-negative bacterial extracellular and wall polysaccharides. Annu. Rev. Microbiol. 39:243–270.

64. Tolmasky, M. E., R. J. Staneloni, and L. F. Leloir. 1982. Lipid-bound saccharides in *Rhizobium meliloti*. J. Biol. Chem. 257:6751–6757.

65. Tolmasky, M. E., R. J. Staneloni, R. A. Ugalde, and L. F. Leloir. 1980. Lipid-bound sugars in *Rhizobium meliloti*. Arch. Biochem. Biophys. 203:358–364.

66. Urzainqui, A., and G. C. Walker. 1992. Exogenous suppression of the symbiotic deficiencies of *Rhizobium meliloti exo* mutants. J. Bacteriol. 174:3403–3406.

67. Uttaro, A. D., G. A. Cangelosi, R. A. Geremia, E. W. Nester, and R. A. Ugalde. 1990. Biochemical characterization of avirulent *exoC* mutants of *Agrobacterium tumefaciens*. J. Bacteriol. 172:1640–1646.

68. Vanderslice, R. W., D. H. Doherty, M. A. Capage, M. R. Betlach, R. A. Hassler, N. M. Henderson, J. Ryan-Graniero, and M. Tecklenburg. 1988. Genetic engineering of polysaccharide structure in *Xanthomonas campestris*, p. 145–156. In V. Crescenzi, I. C. M. Dea, S. Paoletti, S. S. Stivala, and I. W. Sutherland (ed.), Biomedical and biotechnological advances in industrial polysaccharides. Gordon and Breach Science Publishers, New York.

69. Wilburn, W. J., and D. J. Lipman. 1983. Rapid similarity searches of nucleic acid and protein data banks. Proc. Natl. Acad. Sci. USA 80:726–730.

70. Zhan, H., and J. A. Leigh. 1990. Two genes that regulate exopolysaccharide production in *Rhizobium meliloti*. J. Bacteriol. 172: 5254–5259.

71. Zhan, H., S. B. Levery, C. C. Lee, and J. A. Leigh. 1989. A second exopolysaccharide of *Rhizobium meliloti* strain SU47 that can function in root nodule invasion. Proc. Natl. Acad. Sci. USA 86:3055–3059.
