
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 1993 by The American Society for Biochemistry and Molecular Biology, Inc.
Vol. 268, No. 35, Issue of December 15, pp. 26338–26345, 1993
Printed in U.S.A.

# Structure of the N-Linked Oligosaccharides That Show the Complete Loss of α-1,6-Polymannose Outer Chain from och1, och1 mnn1, and och1 mnn1 alg3 Mutants of Saccharomyces cerevisiae*

(Received for publication, April 8, 1993, and in revised form, July 9, 1993)

Yoko Nakanishi-Shindo†, Ken-ichi Nakayama‡, Atsushi Tanaka§, Yoshiko Toda§, and Yoshifumi Jigami¶

From the †National Institute of Bioscience and Human Technology, Tsukuba, Ibaraki 305, Japan and the §Asahi Chemical Industry Co., Ltd., Fuji, Shizuoka 416, Japan

The periplasmic invertase was purified from *Saccharomyces cerevisiae* och1::LEU2 disruptant cells (Δoch1), which have a defect in elongation of the outer chain attached to the N-linked core oligosaccharides (Nakayama, K., Nagasu, T., Shimma, Y., Kuromatsu, J., and Jigami, Y. (1992) EMBO J. 11, 2511–2519). Structural analysis of the pyridylaminated (PA) neutral oligosaccharides released by hydrazinolysis and N-acetylation confirmed that the och1 mutation causes a complete loss of the α-1,6-polymannose outer chain, although the PA oligosaccharides (Man₅GlcNAc₂-PA and Man₁₀GlcNAc₂-PA), in which one or two α-1,3-linked mannose(s) attached to the endoplasmic reticulum (ER)-form core oligosaccharide (Man₅GlcNAc₂) were also detected. Analysis of the Δoch1 mnn1 strain oligosaccharides released from total cell mannoprotein revealed that the Δoch1 mnn1 mutant eliminates the α-1,3-mannose attached to the core and accumulates predominantly a single ER-form oligosaccharide species (Man₅GlcNAc₂), suggesting a potential use of this strain as a host cell to produce glycoproteins containing mammalian high mannose type oligosaccharides. The Δoch1 mnn1 alg3 mutants accumulated Man₅GlcNAc₂ and Man₈GlcNAc₂ in total cell mannoprotein, confirming the lack of outer chain addition to the incomplete corelike oligosaccharide and the leaky phenotype of the alg3 mutation. All the results suggest that the OCH1 gene encodes an α-1,6-mannosyltransferase that is functional in the initiation of α-1,6-polymannose outer chain addition to the N-linked core oligosaccharide (Man₅GlcNAc₂ and Man₈GlcNAc₂) in yeast.

mutant, which accumulates the ER-form core oligosaccharide (Man₈GlcNAc₂) at the non-permissive temperature (2), but it was slightly smaller than that of mnn9 mutant, which contains two mannoses of outer chain attached to the core oligosaccharide (3). Cloning of the OCH1 gene, by complementation of ts growth phenotype, and subsequent characterization of the gene product demonstrated that the OCH1 gene encodes a novel membrane-bound mannosyltransferase (4). The OCH1 protein-overproducing cells exhibited about 5-fold greater mannose transfer activity with the mannoprotein prepared from the och1::LEU2 disruptant cells (Δoch1) as an acceptor for the in vitro mannose transfer reaction, while the smaller oligosaccharides, such as α-methyl-mannoside, α-1,6-mannobiose, and mannotetraose, used to measure conventional α-1,2- or α-1,6-mannosyl transfer, were inactive.

Therefore, further studies on the structure of acceptor molecule for the OCH1 protein-dependent mannose transfer are required to elucidate the role of OCH1 protein in outer chain biosynthesis in yeast. Here we report the structures of N-linked oligosaccharides of external invertase prepared from Δoch1 cells and those of cell wall mannoprotein prepared from Δoch1 mnn1 double mutant and from Δoch1 mnn1 alg3 triple mutant. The structural relationship indicated that the strains containing the Δoch1 mutation completely lack the α-1,6-polymannose outer chain attached to the core oligosaccharide (Man₈GlcNAc₂), not only in a specific glycoprotein, invertase, but also more generally in cell wall mannoprotein. The accumulation of predominantly a single oligosaccharide, Man₅GlcNAc₂, due to the disappearance of α-1,3-linked mannose addition, was also confirmed in the Δoch1 mnn1 mutant. Together with previous results on the properties of the OCH1 protein (4), the results reported herein indicate that the OCH1 gene encodes a membrane-bound α-1,6-mannosyltransferase that is functional in the initiation of the polymannose outer chain addition to the N-linked core oligosaccharide.

EXPERIMENTAL PROCEDURES

## Strains and Media

*S. cerevisiae* strain EHA-1C (MATa leu2–3 leu2–112 pep4–3) has been described in a previous paper (1). *S. cerevisiae* YS52-1-1B, in which the OCH1 gene is disrupted by LEU2 has been described previously (4). *S. cerevisiae* YS57-5A (MATa och1::LEU2 leu2 ura3 his1 his3), YS57-5C (MATa och1::LEU2 leu2 ura3 trp1 his1 his3), and YS57-2C (MATa och1::LEU2 leu2 ura3 trp1 his1 his3) were constructed by the standard genetic methods (5) as described (4). *S. cerevisiae* LB1-10B (MATa mnn1) (6) was obtained from the Yeast Genetic Stock Center, University of California, Berkeley. *S. cerevisiae* LB347-1C (MATa mnn9) (7), was provided by Dr. W. Tanner (University of Regensburg, Germany). *S. cerevisiae* PRY90 (MATa alg3-1 ade2-101 ura3-52) (8) was provided by Dr. P. W. Robbins, Massachusetts Institute of Technology. Cells were grown in YPD medium (2% Bacto-peptone, 1% yeast extract, 2% glucose), supplemented with 0.3 M sorbitol as an osmotic stabilizer.

---

* This work was supported in part by a grant-in-aid of Research and Development Project of Basic Technologies for Future Industries from Ministry of International Trade and Industry, Japan. The National Institute of Bioscience and Human Technology was formerly named the National Chemical Laboratory for Industry. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* To whom correspondence should be addressed: National Institute of Bioscience and Human Technology, 1-1 Higashi, Tsukuba, Ibaraki 305, Japan. Tel.: 81-298-54-4689; Fax: 81-298-54-5188.

26338

This is an Open Access article under the **CC BY** license.

# och1, och1 mnn1, and och1 mnn1 alg3 Yeast Oligosaccharides

## Purification of Invertase

YS52-1-1B (Δoch1) was grown in a 5-liter jar fermenter containing 3 liters of medium (YPD + 0.3 M sorbitol) at 25 °C. When glucose was exhausted by the cells, 0.5% sucrose was added to de-repress the invertase gene expression. The cells were incubated for an additional 4 h and then harvested by centrifugation. Invertase activity was assayed at 30 °C by measuring the rate of glucose released from sucrose with Glucose CII-Test Wako (Wako Chemical Co., Osaka, Japan) in sodium acetate buffer (pH 5.0) according to the supplier’s manual.

External (periplasmic) invertase was purified by Neumann and Lam-pen (9) with some modifications. All operations were done at 4 °C. The cells obtained from 12-liter cultures were suspended in 10 mM potassium phosphate buffer (pH 6.5) (buffer A) containing 1 mM phenylmethylsulfonyl fluoride and broken in a French pressure cell press (SLM, Aminco) at 12,000 p.s.i., and the homogenates were centrifuged to remove the cell debris. Streptomycin sulfate (1% final) was added to remove nucleic acids. Ammonium sulfate was then added to the supernatant until 75% saturation to precipitate nonglycosylated internal invertase and most of the contaminating proteins. After centrifugation, the supernatant was dialyzed against the buffer A. Ten grams of DEAE-Sephadex A-50 resin, pre-equilibrated with buffer A, was added to the dialysate. After the invertase activity remaining in solution was reduced to less than 10% of the initial, the liquid was removed, and the resin was packed in a column (2.6 × 35 cm) and washed with two volumes of the same buffer containing 0.1 M NaCl. External glycosylated invertase was then eluted with the buffer containing 0.2 M NaCl, and the fractions containing enzyme activity were pooled and dialyzed against deionized water. The dialysate was lyophilized, dissolved in deionized water, and applied to a Sephadex G-200SF column (2.0 × 95 cm) equilibrated with buffer A containing 0.2 M NaCl. The column was eluted with the same buffer, and the invertase-containing fractions were pooled and lyophilized. The sample thus obtained gave a discrete protein band on SDS-PAGE¹ (10) using a 4–20% gradient gel (SDS-PAGE Plate 4/20, 10 cm × 10 cm, Daiichi Pure Chemicals, Tokyo, Japan) by Coomassie Brilliant Blue staining. Table I summarizes the purification of external invertase from Δoch1 cells (YS52-1-1B strain) obtained from a 12-liter culture. The yield of purified glycosylated invertase was 2 mg from 180 g of wet cells. The above process was repeated four times to obtain sufficient invertase to analyze its oligosaccharide structure.

### Hydrazinolysis and N-Acetylation

About 8 mg of purified external invertase, isolated from Δoch1 cells according to the above procedure, was subjected to hydrazinolysis to release N-linked oligosaccharides (13, 14), using a gas-phase hydrazinolysis apparatus (Hydraclub S-204, Honen Oil Co., Tokyo, Japan). The glycoprotein samples (0.5 mg/vial) were treated for 60 min at 110 °C. After hydrazinolysis, the oligosaccharides were N-acetylated at room temperature for 30 min by adding 250 μl of 0.2 M ammonium acetate and 25 μl of acetic anhydride, and this process was repeated once to complete the reaction.

### Preparation of Mannoprotein

Cells were cultivated in YPD-sorbitol medium at 25 °C (YN3-1D, Δoch1 mnn1) or at 28 °C (YN5-2C, Δoch1 mnn1 alg3) and harvested at early stationary phase. Mannoproteins were obtained by hot citrate buffer extraction followed by precipitation with ethanol (15). The precipitates were further purified by a concanavalin A-agarose column (1.0 × 10 cm), which was equilibrated with Tris-HCl buffer (pH 7.3) containing 0.2 M NaCl, 1 mM MnCl₂, and 1 mM CaCl₂. The column was eluted by 0.1 M NaCl solution containing 0.1 M α-methyl-d-mannoside. Since the elution profile was not changed by using 0.2 M α-methyl-d-mannoside, we used 0.1 M concentration for the elution. The mannoprotein fraction was dialyzed and lyophilized.

### Glycopeptidase A Digestion

Glycopeptidase A from sweet almond (Seikagaku Kogyo Co., Tokyo, Japan) is specific for release of N-linked oligosaccharides from glycoprotein or glycopeptide (16), whereas hydrazinolysis releases both N- and O-linked oligosaccharides from a glycoprotein (17). When the oligosaccharides released from cell wall mannoprotein by hydrazinolysis were analyzed on HPLC after fluorescence labeling, several extra by-products were observed, which were probably formed from the O-linked

---

### TABLE I  
**Purification of the external invertase synthesized by Δoch1 cells at 25 °C**

One unit is defined as the enzyme activity that produces 1 μmol of glucose from sucrose for 1 min at 30 °C at pH 5.0.

| Step                          | Total protein<sup>a</sup> | Total activity | Specific activity | Recovery |
|-------------------------------|--------------------------|----------------|--------------------|----------|
|                               | mg                       | units × 10³    | units/mg          | %        |
| 1. Crude cell extract         | 5,600                    | 76.5           | 13                 | 100      |
| 2. Streptomycin sulfate       | 4,200                    | 66.5           | 16                 | 87       |
| 3. Ammonium sulfate           | 150                      | 57.6           | 384                | 78       |
| 4. DEAE-Sephadex              | 9                        | 33.3           | 3,700              | 44       |
| 5. Sephadex G-200SF           | 4.5                      | 20.6           | 4,560              | 27       |

<sup>a</sup> Protein was determined according to Bradford (11) using bovine serum albumin as a standard for steps 1–4. Protein concentration for step 5 was determined from the absorbance at 280 nm (ε<sub>1%</sub><sup>280 nm</sup> = 22.5) (12).

oligosaccharides by the chemical reaction (hydrazinolysis), because these by-products disappeared in samples from glycopeptidase A digestion. Hence, we used glycopeptidase A to release the N-linked oligosaccharides from total cell mannoprotein. Ten milliunits of glycopeptidase A was dissolved in 500 μl of 0.1 M sodium acetate buffer (pH 5.0). Each 3 μl of the above enzyme solution was added to samples containing 1 mg of mannoprotein in 10 μl of the same buffer. The mixture was incubated for 18 h at 37 °C, and the reaction was stopped by boiling for 5 min and centrifuged to remove insoluble material.

### Pyridylation

The oligosaccharides released from proteins, either by hydrazinolysis or by glycopeptidase A, were labeled with 2-aminopyridine to facilitate the sensitive detection by fluorescence (18, 19). Pyridylation was carried out by using the commercially available reagent kit (Takara Shuzo Co., Kyoto, Japan). The pyridylated derivatives (PA oligosaccharides) were loaded on a Toyopearl HW-40F column (1.0 × 40 cm) (Tosoh Co., Tokyo, Japan), which was pre-washed with 10 mM ammonium acetate buffer (pH 7.0), and eluted with the same buffer to remove residual 2-aminopyridine. PA oligosaccharides were detected by fluorescence (excitation wavelength (Ex) = 320 nm, emission wavelength (Em) = 400 nm).

### High Performance Liquid Chromatography (HPLC)

The separation of PA oligosaccharides was carried out by HPLC using a Shimadzu LC-6A chromatograph system equipped with a fluorescence spectrometer RF-530.

#### Anion Exchange HPLC—Since some of N-linked oligosaccharides on invertase are phosphorylated in *S. cerevisiae* (20), anion exchange Cosmogel QA column (0.8 × 7.5 cm) (Nakarai Tesque, Kyoto, Japan) was used to separate neutral oligosaccharides from the phosphorylated ones. The column was previously equilibrated with 10 mM ammonium acetate buffer (pH 8.0) at a flow rate of 0.5 ml/min. At 20 min after a sample injection, the buffer concentration was increased linearly up to 45 mM for 15 min. The neutral oligosaccharide fractions were eluted within 17 min after the sample injection and pooled for further analysis. PA oligosaccharides were detected by fluorescence (Ex = 310 nm and Em = 380 nm).

#### Size-fractionation HPLC—The retention time of oligosaccharide largely depends on the number of sugar residues in the amine-modified column chromatography (21). Size-fractionation HPLC was performed with Asahipak NH2P-50 (0.46 × 25 cm) (Asahi Chemical Co., Tokyo, Japan) at a flow rate of 1.0 ml/min. Two solvents, A and B, were used. Solvent A was composed of 200 mM acetic acid adjusted with triethylamine (pH 7.3) and acetonitrile (35:65, v/v). Solvent B was composed of 200 mM acetic acid adjusted with triethylamine (pH 7.3) and acetonitrile (50:50, v/v). The column was equilibrated with solvent A. After the sample injection, the proportion of solvent B was increased linearly up to 100% for 50 min. PA oligosaccharides were detected by fluorescence (Ex = 310 nm and Em = 380 nm).

#### Reverse-phase HPLC—The reverse-phase HPLC was performed with Cosmosil 5C18-P (0.46 × 15 cm) (Nakarai Tesque) at a flow rate of 1.0 ml/min. Two solvents, C and D, were used. Solvent C was 10 mM sodium phosphate buffer (pH 3.8), and solvent D was 10 mM sodium phosphate buffer (pH 3.8) containing 0.1% 1-butanol. The column was equilibrated with a mixture of solvent C and D (95:5, v/v). At 10 min after the sample injection, the ratio of solvent D was increased linearly up to 100% for 90 min. Samples were detected by fluorescence (Ex = 320 nm and Em = 400 nm).

---

¹ The abbreviations used are: PAGE, polyacrylamide gel electrophoresis; HPLC, high performance liquid chromatography; FAB-MS, fast atom bombardment mass spectroscopy.

26340

# och1, och1 mnn1, and och1 mnn1 alg3 Yeast Oligosaccharides

## α-1,2-Mannosidase Digestion

One microunit of α-1,2-mannosidase (from Aspergillus saitoi) (22) and 10 pmol of PA oligosaccharide were dissolved in 20 μl of 0.1 M sodium acetate buffer (pH 5.0). The reaction mixture was incubated for 18 h at 37 °C and then boiled to stop the reaction. After centrifugation, the supernatant was subjected to HPLC as described previously.

## Partial Acetolysis

Partial acetolysis of PA oligosaccharide (23) was carried out to release α-1,6-linked mannose specifically. A 5-nmol sample of PA oligosaccharide was peracetylated with a mixture (40 μl) of acetic anhydride and pyridine (1:1) at 100 °C for 15 min in a sealed glass tube. After removing excess reagent by evaporation, 20 μl of a mixture containing acetic anhydride-acetic acid-sulfuric acid (10:10:1) was added. The tube was resealed and incubated at 37 °C for 15 h. Four microliters of pyridine was added, and the solution was dried by evaporation. Then 0.4 ml of saturated sodium bicarbonate solution was added to the residue, and the products were extracted five times with 0.3 ml each of chloroform. After evaporation of the combined chloroform extract, the sample was subjected to hydrazinolysis and N-acetylation as described, and the PA oligosaccharide sample was analyzed by HPLC.

## FAB-MS

Fast atom bombardment mass spectroscopy (FAB-MS) was carried out using a JMS-HX110 mass spectrometer (JEOL Co., Tokyo, Japan) equipped with an FAB ion source operated with xenon at 10 kV of accelerating voltage. The matrix used was a mixture of nitrobenzyl alcohol and glycerol (6:4). Samples (1–2 μg of PA oligosaccharide) were desalted by HPLC using either an Asahipak NH2P-50 (0.46 × 25 cm) or a Tosoh TSKgel Amide-80 column (0.46 × 25 cm) at a flow rate of 0.5 ml/min with a mixture of acetonitrile and water (1:1).

## High Resolution <sup>1</sup>H NMR

<sup>1</sup>H NMR spectra of PA oligosaccharide were measured on a JNM-GX400 (JEOL Co.) at 50 °C. Samples (50–200 μg, except for 18 μg of authentic Man<sub>8</sub>GlcNAc<sub>2</sub>-PA) were dissolved in 99.96% D<sub>2</sub>O and lyophilized. After three repetitions of the above procedure, the samples were finally dissolved in 500 μl of 99.996% D<sub>2</sub>O. The chemical shifts (δ) are expressed in parts/million (ppm) downfield from internal sodium 4,4-dimethyl-4-silapentane-1-sulfonate, but they were actually measured by reference to internal acetone (δ 2.217 ppm). Authentic PA oligosaccharides were purchased from Takara Shuzo Co.

## Double and Triple Mutant Preparation

Double and triple mutants were constructed by sporulating the appropriate heterozygous diploids and isolating the haploid mutant clones after tetrad dissection (5). In cross between YS57-5C and LB1-10B, the sporulation ratio was about 10%, and the segregant carrying the och1 mutation was not obtained. Thus, among them, YN1-28A and YN1-28B, the haploid clones carrying mnn1 mutation were selected by the Western blot analysis using anti-α-1,3-mannan antibody (kindly supplied by Dr. R. Schekman, University of California, Berkeley) and these strains were crossed to the following Δoch1 strains. Two strains, YN3-1D and YN4-18A, carrying Δoch1 mnn1 double mutation, were obtained from the crosses between YN1-28A and YS57-5A, and between YN1-28B and YS57-2C, respectively. Three Δoch1 alg3 double mutants, designated YN2-12A, YN2-14A and YN2-15C, were isolated from the cross between PRY90 and YS57-2C. The size of invertase from each clone was analyzed on SDS-PAGE (10) using 4–20% gradient gel (SDS-PAGE Plate 4/20, 10 cm × 10 cm, Daiichi Pure Chemicals) by Western blotting (4) using anti-invertase polyclonal rabbit antibody. The phenotypes of Δoch1 mnn1 and Δoch1 alg3 mutants were confirmed by analyzing the length of oligosaccharides released from cell wall mannanprotein by glycopeptidase A on size-fractionation HPLC. The Δoch1 mnn1 alg3 triple mutant, YN5-2C (Δoch1 mnn1) and YN2-12A (Δoch1 alg3). The phenotype was confirmed by comparing the invertase size with those from each double mutant by Western blotting.

---

**FIG. 1. Comparison of electrophoretic mobility of external invertases on SDS-PAGE (10) using 4–20% gradient gel between the mutants EHA-1C (wild, 1), YS52-1-1B (Δoch1) (2), YN3-1D (Δoch1 mnn1) (3), PRY90 (alg3) (4), YN2-1-2A (Δoch1 alg3) (5), and YN5-2C (Δoch1 mnn1 alg3) (6).**

The invertase from Δoch1 mnn1 double mutant showed a few discrete bands at a smaller size than that from Δoch1 cells (Fig. 1, lanes 2 and 3). As demonstrated for the mnn1 mnn2 mnn9 triple mutant (24), each ladder band in Δoch1 mnn1 cells reflects the number of sugar chains attached to invertase protein, which has 13 potential N-glycosylation sites in wild type S. cerevisiae (25). The invertase from alg3 cells gave a broad smear (Fig. 1, lane 4), the migration range of which is similar to that of wild type cells, indicating an addition of polymannose outer chain. The mobility of invertase from Δoch1 alg3 double mutant cells is slightly faster than that from Δoch1 cells (Fig. 1, lanes 2 and 5), suggesting a shorter oligosaccharide length in Δoch1 alg3 cells than in Δoch1 cells. Furthermore, the Δoch1 mnn1 alg3 triple mutants produced a still smaller invertase molecule (Fig. 1, lanes 3, 5, and 6) as a result of accumulation of three independent mutations that affect N-linked oligosaccharide biosynthesis.

### N-Linked Neutral Oligosaccharide Length—The size of N-linked neutral oligosaccharides released by hydrazinolysis from the external invertase of Δoch1 cells was analyzed by size-fractionation HPLC. The PA oligosaccharides were separated into three major peaks (peaks 1–3) with some other minor peaks (Fig. 2A). The elution time of peak 1 (21.8 min) is identical to that of authentic Man<sub>8</sub>GlcNAc<sub>2</sub>-PA (PA Sugar Chain 019, Takara). Judging from the elution time of PA glucose oligomers (Honen Oil Co.), peaks 2 (25.1 min) and 3 (28.3 min) were estimated to correspond to Man<sub>9</sub>GlcNAc<sub>2</sub>-PA and Man<sub>10</sub>GlcNAc<sub>2</sub>-PA, respectively. The Δoch1 cells cultivated at the permissive temperature (25 °C) completely lacked the larger oligosaccharides that were observed in invertase from baker’s yeast and from the och1 ts mutant under the permissive temperature (1).

The PA oligosaccharides released by glycopeptidase A from the cell wall mannanprotein of Δoch1 cells showed a similar elution pattern on size fractionation HPLC, except for some minor peaks at 50–60 min (data not shown). Since the minor peaks were not observed in the PA oligosaccharide from external invertase of Δoch1 cells, but were seen in the total mannanprotein from the mnn9 mutant, these minor products may be derived from mannanprotein during the high temperature extraction of cell wall material.

The oligosaccharide from Δoch1 mnn1 double mutant mannanprotein contained predominantly a single species, corresponding to Man<sub>8</sub>GlcNAc<sub>2</sub>-PA (Fig. 2B, peak 4), suggesting the loss of α-1,3-mannose addition due to the mnn1 mutation. A minor component (10% of Man<sub>8</sub>GlcNAc<sub>2</sub>-PA) was observed at the elution time of 24.2–25.5 min, which corresponds to that of Man<sub>9</sub>GlcNAc<sub>2</sub>-PA (see “Discussion”).

The elution pattern of PA oligosaccharide from Δoch1 mnn1 alg3 triple mutant mannanprotein (Fig. 2C) reveals two major peaks (peaks 5 and 6). Judging from the elution time of PA glucose oligomers, peak 5 (12.3 min) corresponds to

---

## RESULTS

Comparison of Invertase Mobility between the Mutants—Fig. 1 shows the electrophoretic patterns of external invertases. The Δoch1 cells gave a few bands at a faster migration position than the wild type cells, indicating a deficiency of polymannose outer

och1, och1 mnn1, and och1 mnn1 alg3 Yeast Oligosaccharides 26341

a protonated molecular ion at $m/z = 1,313$, and the M8 product from the M9 sample showed a protonated molecular ion at $m/z = 1,800$, indicating the structure of Man₅GlcNAc₂-PA (calculated 1,312.5) and Man₈GlcNAc₂-PA (calculated 1,798.7), respectively. These data were in accordance with the estimated structure of the component 1 as an ER-form core PA oligosaccharide. Based on the structure of component 1, component 2 was estimated to be a Man₉GlcNAc₂-PA in which one mannose residue, not connected by α-1,2 linkage, is attached to the Man₈GlcNAc₂-PA core oligosaccharide at the mannose of position 11 in Fig. 3A. In addition, component 3 was estimated as Man₁₀GlcNAc₂-PA in which one additional mannose is attached to the above Man₈GlcNAc₂-PA component at the mannose of position 9 in Fig. 3A.

The NMR spectrum for authentic Man₈GlcNAc₂-PA (ER-form core PA oligosaccharide, PA Sugar Chain 019, Takara) and those for Man₈GlcNAc₂-PA, Man₉GlcNAc₂-PA, and Man₁₀GlcNAc₂-PA samples from invertase of Δoch1 cells are shown in Fig. 3 (panels A–D, respectively). Chemical shifts ($\delta$, ppm) for C1 and C2 protons are summarized in Table II, and linkage assignments (Fig. 3 and Table II) were based on the reported chemical shifts (26, 27). The spectrum for the Man₈GlcNAc₂-PA sample from Δoch1 cells (Fig. 3B) is identical to that for the authentic ER-form Man₈GlcNAc₂-PA (Fig. 3A). In the spectrum of the Man₈GlcNAc₂-PA sample, one additional C1-H signal was observed at 5.14 ppm, which is characteristic for the anomeric proton of the unsubstituted α-1,3-linked mannose (Fig. 3C). The signal at 5.14 ppm was doubled in intensity in the spectrum for the Man₁₀GlcNAc₂-PA sample, indicating the attachment of one additional α-1,3-linked mannose residue to the Man₈GlcNAc₂-PA structure. Since the α-1,2-mannosidase digestion of Man₉GlcNAc₂-PA yielded Man₈GlcNAc₂-PA, this α-1,3 linkage must be the mannose at position 11 in Fig. 3A, confirming the structure of Man₉GlcNAc₂-PA as shown in Fig. 3C. Considering the resistance of the Man₁₀GlcNAc₂-PA sample to α-1,2-mannosidase digestion, the oligosaccharide can be assigned the structure shown in Fig. 3D. The oligosaccharide structures shown in Fig. 3 (panels A–D) are in good agreement with the chemical shifts assigned in Table II.

Oligosaccharide Structure of Mannoproteins from Δoch1 mnn1 Double Mutant—The major PA oligosaccharide obtained from Δoch1 mnn1 cells on size-fractionation HPLC (peak 4 in Fig. 2B) showed a single peak on reverse-phase HPLC, designated as component 4 (data not shown). The elution time of this component 4 was identical to that of authentic Man₈GlcNAc₂-PA and the molecular mass was confirmed by FAB-MS, which gave $m/z$ 1,800, as estimated for Man₈GlcNAc₂-PA (calculated 1,798.7). After α-1,2-mannosidase digestion of component 4, the peak was shifted from M8 to M5 position on size-fractionation HPLC. The NMR spectrum of this component 4 (Fig. 3E and Table II) was identical to that of authentic Man₈GlcNAc₂-PA (Fig. 3A). From these data, we conclude that the Δoch1 mnn1 cells predominantly produced the ER-form Man₈GlcNAc₂ as a neutral oligosaccharide.

Oligosaccharide Structure of Mannoproteins from Δoch1 mnn1 alg3 Triple Mutant—Each PA oligosaccharide fraction, corresponding to peaks 5 and 6 in Fig. 2C, from Δoch1 mnn1 alg3 mutant cells showed a single peak on reverse-phase HPLC (components 5 and 6, respectively). The elution time on reverse-phase HPLC of component 6 was same as that of the authentic ER form of Man₈GlcNAc₂-PA. The ratio of components 5 and 6 was 1:1.4. It is reported that the alg3 sec18 mutant produced external invertase containing Man₅GlcNAc₂ and Man₈GlcNAc₂ N-linked oligosaccharides after the temperature shift (28). In accordance with the reported oligosaccharide structure of alg3 sec18 (28), we assumed that component 5 produced by Δoch1 mnn1 alg3 mutant cells must be the

FIG. 2. Elution profile on size fractionation HPLC of PA oligosaccharides released from YS52-1-1B (Δoch1) invertase by hydrazinolysis (A), YN3-1D (Δoch1 mnn1) mannoprotein by glycopeptidase A digestion (B), and YN5-2C (Δoch1 mnn1 alg3) mannoprotein by glycopeptidase A digestion (C). Each sample is about 500 pmol. The elution times of authentic PA glucose oligomers are indicated by arrows.

Man₅GlcNAc₂-PA and peak 6 (21.8 min) corresponds to Man₈GlcNAc₂-PA. Most of these two peaks disappeared in alg3 mutant cells, suggesting outer chain addition in the alg3 single mutant (data not shown).

Oligosaccharide Structure of Invertase from Δoch1 Cells—Each PA oligosaccharide fraction corresponding to peaks 1–3 in Fig. 2A from Δoch1 cells was collected individually and analyzed on reverse-phase HPLC to examine its purity. Each showed a single peak on reverse-phase HPLC, suggesting a single component (data not shown). The elution time of component 1 (31.6 min) was in accordance with that of the authentic ER-form Man₈GlcNAc₂-PA sample (PA Sugar Chain 019, Takara). Components 1–3 were analyzed by positive ion FAB-MS to give positive molecular ion peaks at $m/z$ 1,800, 1,962, and 2,123, respectively, which is consistent with the molecular mass of Man₈GlcNAc₂-PA (calculated 1,798.7), Man₉GlcNAc₂-PA (calculated 1,960.7), and Man₁₀GlcNAc₂-PA (calculated 2,122.8), respectively. These data confirmed the formation of PA derivatives and the estimation of each oligosaccharide length based on the elution time on HPLC.

α-1,2-Mannosidase removes only terminal α-1,2-linked mannose, and the size of each PA oligosaccharide component was analyzed on size-fractionation HPLC after the enzyme digestion. Component 1 was shifted from M8 (Man₈GlcNAc₂-PA) position to M5 position, the elution time of which is consistent with that of authentic Man₅GlcNAc₂-PA (PA Sugar Chain 017, Takara); component 2 was shifted from M9 position to M8 position; the elution time of component 3 was not shifted. The molecular mass of each digestion product was confirmed by FAB-MS. The M5 product derived from the M8 sample showed

26342 och1, och1 mnn1, and och1 mnn1 alg3 Yeast Oligosaccharides

A  
$\begin{aligned}
& M^{9} \stackrel{-2}{-} M^{6} \\
& M^{7} \stackrel{-3}{-} M^{6} \\
& M^{11} \stackrel{-2}{-} M^{8} \stackrel{-2}{-} M^{5}
\end{aligned}$  
$\begin{aligned}
& M^{6} \stackrel{-4}{-} GN \stackrel{-4}{-} GN \stackrel{-PA}{-} \\
& M^{3} \stackrel{-3}{-} M^{3}
\end{aligned}$  

B  
$\begin{aligned}
& M^{9} \stackrel{-2}{-} M^{6} \\
& M^{7} \stackrel{-3}{-} M^{6} \\
& M^{11} \stackrel{-2}{-} M^{8} \stackrel{-2}{-} M^{5}
\end{aligned}$  
$\begin{aligned}
& M^{6} \stackrel{-4}{-} GN \stackrel{-4}{-} GN \stackrel{-PA}{-} \\
& M^{3} \stackrel{-3}{-} M^{3}
\end{aligned}$  

C  
$\begin{aligned}
& M^{9} \stackrel{-2}{-} M^{6} \\
& M^{7} \stackrel{-3}{-} M^{6} \\
& M^{11} \stackrel{-2}{-} M^{8} \stackrel{-2}{-} M^{5}
\end{aligned}$  
$\begin{aligned}
& M^{6} \stackrel{-4}{-} GN \stackrel{-4}{-} GN \stackrel{-PA}{-} \\
& M^{3} \stackrel{-3}{-} M^{3}
\end{aligned}$  

D  
$\begin{aligned}
& M^{9} \stackrel{-2}{-} M^{6} \\
& M^{7} \stackrel{-3}{-} M^{6} \\
& M^{11} \stackrel{-2}{-} M^{8} \stackrel{-2}{-} M^{5}
\end{aligned}$  
$\begin{aligned}
& M^{6} \stackrel{-4}{-} GN \stackrel{-4}{-} GN \stackrel{-PA}{-} \\
& M^{3} \stackrel{-3}{-} M^{3}
\end{aligned}$  

E  
$\begin{aligned}
& M^{9} \stackrel{-2}{-} M^{6} \\
& M^{7} \stackrel{-3}{-} M^{6} \\
& M^{11} \stackrel{-2}{-} M^{8} \stackrel{-2}{-} M^{5}
\end{aligned}$  
$\begin{aligned}
& M^{6} \stackrel{-4}{-} GN \stackrel{-4}{-} GN \stackrel{-PA}{-} \\
& M^{3} \stackrel{-3}{-} M^{3}
\end{aligned}$  

Chemical shift (ppm)

FIG. 3. Comparison of the 400 MHz $^1$H NMR spectra for purified neutral PA oligosaccharide fractions from $\Delta och1$ invertase ($B-D$) and from $\Delta och1 mnn1$ mannoprotein ($E$). $A$, authentic ER-form core PA oligosaccharide, Man$_5$GlcNAc$_2$-PA, and PA oligosaccharide purified from the following: $B$, peak 1 in Fig. 2A; $C$, peak 2 in Fig. 2A; $D$, peak 3 in Fig. 2A; $E$, peak 4 in Fig. 2B. All signals in spectrum $B$ are identical to those in spectrum $A$, whereas $C$ and $D$ have an additional anomic proton signal at $\delta$ 5.14 equivalent to one and two protons, respectively, that is characteristic of the unsubstituted $\alpha$-1,3-linked mannose.

Man$_5$GlcNAc$_2$-PA shown in Fig. 4A, and component 6 may be the ER-form Man$_5$GlcNAc$_2$-PA shown in Fig. 3A. To confirm these structures, components 5 and 6 were subjected to $\alpha$-1,2-mannosidase digestion and partial acetolysis. After $\alpha$-1,2-mannosidase digestion, the component 5 peak was shifted from the M5 position (12.0 min) to the M3 position (7.2 min) (Fig. 4, A and B) on size-fractionation HPLC. The molecular masses of component 5 (M5) and enzyme digestion product (M3) were confirmed by FAB-MS, which gave $m/z$ 1,313, as estimated for Man$_5$GlcNAc$_2$-PA (calculated 1,312.5), and $m/z$ 989, estimated for Man$_3$GlcNAc$_2$-PA (calculated 988.4), respectively. After partial acetolysis, which specifically removes $\alpha$-1,6-linked mannose residues, the peak of component 5 (12.0 min) was shifted to the M4 position (9.4 min) (Fig. 4, C and D). This M4 product showed a protonated molecular ion at $m/z = 1,152$, indicating the formation of Man$_4$GlcNAc$_2$-PA (calculated 1,150.4). These data confirmed the structure of component 5 as the Man$_5$GlcNAc$_2$-PA shown in Fig. 4A. In contrast, component 6 was shifted from the M8 position (21.5 min) to the M5 position (12.0 min) after $\alpha$-1,2-mannosidase digestion, and from the M8 position (21.5 min) to the M4 position (9.4 min) after partial acetolysis (data not shown). The molecular mass of M5 product

och1, och1 mnn1, and och1 mnn1 alg3 Yeast Oligosaccharides 26343

**TABLE II**

${}^{1}$H NMR chemical shifts of authentic ER-form core oligosaccharide (Man$_{8}$GlcNAc$_{2}$-PA) and PA oligosaccharides obtained from $\Delta$och1 mnn1 cells

| Residue | Authentic M8 | $\Delta$och1 M8 | $\Delta$och1 M9 | $\Delta$och1 M10 | $\Delta$och1 mnn1 M8 |
| --- | --- | --- | --- | --- | --- |
|  | Shift | Intensity | Shift | Intensity | Shift | Intensity | Shift | Intensity | Shift | Intensity |
|  | ppm |  | ppm |  | ppm |  | ppm |  | ppm |  |
| C1-H |  |  |  |  |  |  |  |  |  |  |
| 3 | 4.748 | 1 | 4.753 | 1 | 4.755 | 1 | 4.753 | 1 | 4.753 | 1 |
| 4 | 4.863 | 1 | 4.867 | 1 | 4.867 | 1 | 4.863 | 1 | 4.863 | 1 |
| 5 | 5.323 | 1 | 5.322 | 1 | 5.328 | 1 | 5.324 | 1 | 5.322 | 1 |
| 6 | 5.115 | 1 | 5.120 | 1 | 5.118 | 1 | 5.116 | 1 | 5.118 | 1 |
| 7 | 5.091 | 1 | 5.094 | 1 | 5.096 | 1 | 5.091 | 1 | 5.092 | 1 |
| 8 | 5.283 | 1 | 5.283 | 1 | 5.283 | 1 | 5.279 | 1 | 5.283 | 1 |
| 9 | 5.039 | 1 | 5.043 | 1 | 5.038 | 1 | 5.030 | 1 | 5.043 | 1 |
| 11 | 5.039 | 1 | 5.043 | 1 | 5.038 | 1 | 5.030 | 1 | 5.043 | 1 |
| A |  |  |  |  |  |  |  |  |  |  |
| B |  |  |  |  | 5.140 | 1 | 5.141 | 1 | 5.141 | 1 |
| C2-H$^{\alpha}$ |  |  |  |  |  |  |  |  |  |  |
| 3 | 4.195 |  | 4.199 |  | 4.205 |  | 4.205 |  | 4.200 |  |
| 4 | 4.131 |  | 4.131 |  | 4.132 |  | 4.130 |  | 4.131 |  |
| 5 | 4.092 |  | 4.092 |  | 4.095 |  | 4.095 |  | 4.092 |  |
| 6 | 4.005 |  | 4.010 |  | 4.009 |  | 4.012 |  | 4.008 |  |
| 7 | 4.060 |  | 4.059 |  | 4.060 |  | 4.061 |  | 4.059 |  |
| 8 | 4.092 |  | 4.092 |  | 4.092 |  | 4.095 |  | 4.092 |  |
| 9 | 4.060 |  | 4.059 |  | 4.060 |  | 4.205 |  | 4.059 |  |
| 11 | 4.060 |  | 4.059 |  | 4.205 |  | 4.061 |  | 4.059 |  |
| A |  |  |  |  |  |  |  |  |  |  |
| B |  |  |  |  | 4.060 |  | 4.061 |  | 4.061 |  |

* Intensities for individual signals of C2-H could not be calculated due to the overlap of signals.

**FIG. 4. Elution profile on size fractionation HPLC of PA oligosaccharides released from $\Delta$och1 mnn1 alg3 mannanprotein. A and C, component 5 corresponding to peak 5 in Fig. 2C; B, after $\alpha$-1,2-mannosidase digestion; D, after partial acetolysis.**

from M8 sample after $\alpha$-1,2-mannosidase digestion and that of M4 product from M8 sample after partial acetolysis were also confirmed by FAB-MS (data not shown), demonstrating the ER form of Man$_{8}$GlcNAc$_{2}$-PA (Fig. 3A) as component 6 (Fig. 2C).

**DISCUSSION**

We have determined the structures of the $N$-linked neutral oligosaccharides attached to invertase in $\Delta$och1 cells, as well as those of the neutral oligosaccharides attached to total manno-protein from $\Delta$och1 mnn1 and $\Delta$och1 mnn1 alg3 mutants. The results demonstrate that the OCH1 gene disruption leads to loss of function to add the $\alpha$-1,6-polymannose outer chain to the ER-form core oligosaccharide. Previously (4), we reported that the OCH1 gene encodes a novel membrane-bound mannosyl-transferase that transfers mannose from GDP-[${}^{14}$C]mannose to the oligosaccharide isolated from mannanprotein from $\Delta$och1 cells. At that time, we assumed that the OCH1 protein might be functional not in the initiation of $\alpha$-1,6-polyman-nose addition to Man$_{8}$GlcNAc$_{2}$, but rather in the elongation of outer chain from Man$_{8}$GlcNAc$_{2}$, a conclusion that was based on the accumulation of a major PA oligosaccharide corresponding to Man$_{8}$GlcNAc$_{2}$-PA and the lack of mannose transfer to the "Man$_{8}$GlcNAc" in an in vitro reaction (4). The present results, however, are not consistent with the previous assumption, because all of the oligosaccharides identified in $\Delta$och1 cells lack $\alpha$-1,6-mannose attached to the Man$_{8}$GlcNAc$_{2}$ core oligosaccharide. Therefore, we have examined whether the OCH1 protein can use Man$_{8}$GlcNAc$_{2}$-PA and Man$_{8}$GlcNAc$_{2}$-PA as mannose acceptors. The microsomal membrane fraction prepared from OCH1 protein-overproducing cells was found to transfer mannose efficiently to Man$_{8}$GlcNAc$_{2}$-PA to form Man$_{9}$GlcNAc$_{2}$-PA and to Man$_{8}$GlcNAc$_{2}$-PA to form Man$_{10}$GlcNAc$_{2}$-PA, and this in vitro reaction was diminished in the membrane fraction from $\Delta$och1 cells.$^{2}$ In addition, in the previous study (4), we used Man$_{8}$GlcNAcOH (octamannosyl-N-acetylglucosaminitol) as acceptor, which was incorrectly called Man$_{8}$GlcNAc (octamannosyl-N-acetylglucosamine). Now, we have confirmed that Man$_{8}$GlcNAcOH does not serve as an acceptor for the OCH1 protein-dependent mannose transfer reaction, whereas Man$_{8}$GlcNAc does. Accordingly, it now becomes clear that the OCH1 gene encodes an $\alpha$-1,6-mannosyltransferase that is functional in the initiation of mannose outer chain addition to the ER-form core oligosaccharide, as shown in Fig. 5. Romero and Herscovics (29, 30) reported the purification and characterization of $\alpha$-1,6-mannosyltransferase, which is certain to be the same enzyme encoded by OCH1 gene. Although we cannot exclude the possibility that the OCH1 protein may also function in the subsequent elongation to form a polymannose outer chain, this seems unlikely because reaction products larger than one mannose addition were not observed in our in vitro mannose transfer experiments even with a prolonged incubation period.$^{2}$

Addition of $\alpha$-1,3-mannose is known to occur with the core-like oligosaccharide to which mannobiose (Man-$\alpha$-1, 2-Man-$\alpha$-1,6-) is added to the ER-form oligosaccharide (compound E in Fig. 5), in wild type invertase (26), carboxypeptidase Y (31), and

$^{2}$ K. Nakayama, Y. Nakanishi-Shindo, and Y. Jigami, manuscript in preparation.

26344 och1, och1 mnn1, and och1 mnn1 alg3 Yeast Oligosaccharides

![Diagram](attachment:diagram.png)

FIG. 5. N-Linked oligosaccharide biosynthesis in S. cerevisiae and deficient steps caused by the och1 and mnn1 mutations, together with the proposed steps caused by the mnn9 mutation (31).

mnn9 mannoproteins (32) to form compounds F and G in Fig. 5. It is noteworthy that the α-1,3-mannose addition to the core oligosaccharide occurs even in the absence of this mannose unit (see compounds B and C in Fig. 5). The first α-1,3-mannose addition occurs at the terminus of the α-1,3-linked mannotriose branch (position 11 of compound A in Fig. 5), and the subsequent α-1,3-mannose addition occurs at the end of α-1,6-linked mannotetraose branch (position 9 of compound A in Fig. 5), which is consistent with the previous results for the oligosaccharides from both wild type and mnn2 mnn9 mannoproteins (26, 32). This order of α-1,3-mannose addition to the core oligosaccharide may suggest the higher affinity of α-1,3-mannosyltransferase encoded by MNN1 gene (33) to add the mannose at position 11 of compound A in Fig. 5 rather than at position 9 of compound A in Fig. 5.

We have clarified that the Δoch1 mnn1 mutant cells produce predominantly the Man₈GlcNAc₂ core oligosaccharide due to the lack of α-1,3-mannosyltransferase activity, although about 10% of the oligosaccharide was observed at the elution time corresponding to the M9 PA oligosaccharide on size-fractionation HPLC (Fig. 2B). This fraction consisted mainly of two isomers. From the results of reverse-phase HPLC and α-1,2-mannosidase digestion (data not shown), one appears identical with the Man₈GlcNAc₂-PA from Δoch1 invertase and the other is estimated as the M9 structure in which one hexose is attached to the mannose at position 11 of compound A in Fig. 5. The former isomer suggests some residual α-1,3-mannosyltransferase activity in Δoch1 mnn1 mutant cells owing to a leaky phenotype of the mnn1 mutation. This may be resolved by oligosaccharide analysis of Δoch1 Δmnn1 double disruptant. The latter isomer was shifted from M9 position to M8 position after α-1,2-mannosidase digestion. It is possible that the

Glc₁Man₈GlcNAc₂-PA was formed as the M9 structure due to either the escape from glucosidase trimming or the reglucosylation of glucose-free core oligosaccharide, but further structure analysis was not performed. Regardless, we believe that the Δoch1 mnn1 strain is a novel host cell in which to produce a glycoprotein containing an ER-form oligosaccharide that is identical to the mammalian high mannose type oligosaccharide and that can function as a precursor to form a complex type sugar chain in mammalian cells by using recombinant DNA technology.

The Δoch1 mnn1 alg3 cells accumulated both the Man₈GlcNAc₂-PA ER-form core oligosaccharide (Fig. 3A) and Man₅GlcNAc₂-PA oligosaccharide (Fig. 4A). The alg3 mutant is known to accumulate lipid-linked Man₅GlcNAc₂ before transfer to the Asn residue of protein (8, 34). It is likely that the Man₅GlcNAc₂-PA, which lacks both the Man-α-1,2-Man-α-1,6-linked mannobiose and the Man-α-1,3-linked mannose of compound A in Fig. 5, was caused by transfer to proteins of the incomplete lipid linked Man₅GlcNAc₂. The formation of Man₈GlcNAc₂ is assumed to be caused by a leaky phenotype of the alg3 mutation, as reported for the oligosaccharide structure from alg3 sec18 invertase (28). The molar ratio of Man₅GlcNAc₂ to Man₈GlcNAc₂ was 1:1.4 in Δoch1 mnn1 alg3 mutant, whereas it was 5:1 in alg3 sec18 mutant. This may reflect the different growth conditions, because Δoch1 mnn1 mannoprotein was obtained from cells cultivated at 28 °C, whereas invertase from alg3 sec18 cells was obtained by a 3-h de-repression at 37 °C after cultivation at the permissive temperature due to the ts phenotype of the sec18 mutant (28). This suggests that the accumulation of the incomplete precursor of N-linked oligosaccharides may be temperature-dependent in the alg3 mutant, although the alg3 mutant itself is not ts for cell growth.

(8). The formation of mannose outer chain in the *alg3* mutant (Fig. 1, lane 4) was diminished in Δ*och1 mnn1 alg3* cells (Fig. 1, lane 6), which suggests that the OCH1 protein is functional in the initiation of outer chain elongation of Man₅GlcNAc₂ as well as Man₈GlcNAc₂. The *in vitro* experiment supports this assumption by demonstrating that extracts from OCH1 protein-overproducing cells have an activity to transfer mannose to Man₅GlcNAc₂-PA to form Man₆GlcNAc₂-PA.² In conclusion, our results suggest that the *OCH1* gene encodes an α-1,6-mannosyltransferase that is functional in the initiation of α-1,6-polymannose outer chain addition to the *N*-linked core oligosaccharide (Man₅GlcNAc₂ and Man₈GlcNAc₂) in yeast. Further studies on the substrate specificity and kinetic properties of OCH1 protein are in progress.

Acknowledgments—We thank Drs. W. Tanner and P. W. Robbins for providing LB347-1C and PRY90 yeast strains, respectively, and Dr. K. Kobata for supplying α-1,2-mannosidase. We express our appreciation to Dr. C. E. Ballou for critical reading of the manuscript, excellent comments, and improving the English.

**REFERENCES**

1. Nagasu, T., Shimma, Y., Nakanishi, Y., Kuromatsu, J., Iwama, K., Nakayama, K., Suzuki, K., and Jigami, Y. (1992) *Yeast* **8**, 535–547
2. Esmon, B., Esmon, P. C., and Schekman, R. (1984) *J. Biol. Chem.* **259**, 10322–10327
3. Ballou, L., Alvarado, E., Tsai, P.-K., Dell, A., and Ballou, C. E. (1989) *J. Biol. Chem.* **264**, 11857–11864
4. Nakayama, K., Nagasu, T., Shimma, Y., Kuromatsu, J., and Jigami, Y. (1992) *EMBO J.* **11**, 2511–2519
5. Sherman, F., and Hicks, J. (1991) *Methods Enzymol.* **194**, 21–37
6. Ballou, L., Cohen, R. E., and Ballou, C. E. (1980) *J. Biol. Chem.* **255**, 5986–5991

7. Tsai, P.-K., Frevert, J., and Ballou, C. E. (1984) *J. Biol. Chem.* **259**, 3805–3811
8. Huffaker, T. C., and Robbins, P. W. (1983) *Proc. Natl. Acad. Sci. U. S. A.* **80**, 7466–7470
9. Neumann, N. P., and Lampen, J. O. (1967) *Biochemistry* **6**, 468–475
10. Laemmli, U. K. (1970) *Nature* **227**, 680–685
11. Bradford, M. (1976) *Anal. Biochem.* **72**, 248–254
12. Trimble, R. B., and Maley, F. (1977) *J. Biol. Chem.* **252**, 4409–4412
13. Fukuda, M., Kondo, T., and Osawa, T. (1976) *J. Biochem. (Tokyo)* **80**, 1223–1232
14. Takasaki, S., Mizuochi, T., and Kobata, A. (1982) *Methods Enzymol.* **83**, 263–268
15. Peat, S., Whelan, W. J., and Edwards, T. E. (1961) *J. Chem. Soc.*, 29–34
16. Takahashi, N., and Nishibe, H. (1981) *Biochim. Biophys. Acta* **657**, 457–467
17. Kuraya, N., and Hase, S. (1992) *J. Biochem. (Tokyo)* **112**, 122–126
18. Kondo, A., Suzuki, J., Kuraya, N., Hase, S., Kato, I., and Ikenaka, T. (1990) *Agric. Biol. Chem.* **54**, 2169–2170
19. Hase, S., Ibuki, T., and Ikenaka, T. (1984) *J. Biochem. (Tokyo)* **95**, 197–203
20. Smith, W. L., and Ballou, C. E. (1974) *Biochemistry* **13**, 355–361
21. Blanken, W. M., Bergh, L. E., Koppen, P. L., and Van den Eijnden, D. H. (1985) *Anal. Biochem.* **145**, 322–330
22. Kobata, A., and Amano, J. (1987) *Methods Enzymol.* **138**, 779–785
23. Natsuka, S., Hase, S., and Ikenaka, T. (1987) *Anal. Biochem.* **167**, 154–159
24. Ballou, L., Gopal, P., Krummel, B., Tammi, M., and Ballou, C. E. (1986) *Proc. Natl. Acad. Sci. U. S. A.* **83**, 3081–3085
25. Ziegler, F. D., Maley, F., and Trimble, R. B. (1988) *J. Biol. Chem.* **263**, 6986–6992
26. Trimble, R. B., and Atkinson, P. H. (1986) *J. Biol. Chem.* **261**, 9815–9824
27. Cohen, R. E., and Ballou, C. E. (1980) *Biochemistry* **19**, 4345–4358
28. Verostek, M. F., Atkinson, P. H., and Trimble, R. B. (1991) *J. Biol. Chem.* **266**, 5547–5551
29. Romero, P. A., and Herscovics, A. (1989) *J. Biol. Chem.* **264**, 1946–1950
30. Romero, P. A., and Herscovics, A. (1992) *J. Cell. Biochem. Suppl.* **16D**, 175
31. Ballou, L., Hernandez, L. M., Alvarado, E., and Ballou, C. E. (1990) *Proc. Natl. Acad. Sci. U. S. A.* **87**, 3368–3372
32. Alvarado, E., Ballou, L., Hernandez, L. M., and Ballou, C. E. (1990) *Biochemistry* **29**, 2471–2482
33. Graham, T. R., Verostek, M. F., Mackey, V., Trimble, R., and Emr, S. D. (1992) *Yeast* **8**, S458
34. Orlean, P. (1990) *Mol. Cell. Biol.* **10**, 5796–5805
