
JOURNAL OF BACTERIOLOGY, July 1996, p. 4122–4130
0021-9193/96/$04.00+0
Copyright © 1996, American Society for Microbiology

# Cloning, Sequencing, and Characterization of the Bacillus subtilis Biotin Biosynthetic Operon

STANLEY BOWER, JOHN B. PERKINS, R. ROGERS YOCUM, C. L. HOWITT, PETER RAHAIM, AND JANICE PERO*

OmniGene Bioproducts, Inc., Cambridge, Massachusetts 02138

Received 8 February 1996/Accepted 2 May 1996

A 10-kb region of the *Bacillus subtilis* genome that contains genes involved in biotin biosynthesis was cloned and sequenced. DNA sequence analysis indicated that *B. subtilis* contains homologs of the *Escherichia coli* and *Bacillus sphaericus bioA*, *bioB*, *bioD*, and *bioF* genes. These four genes and a homolog of the *B. sphaericus bioW* gene are arranged in a single operon in the order *bioWAFDB* and are followed by two additional genes, *bioI* and *orf2*. *bioI* and *orf2* show no similarity to any other known biotin biosynthetic genes. The *bioI* gene encodes a protein with similarity to cytochrome P-450s and was able to complement mutations in either *bioC* or *bioH* of *E. coli*. Mutations in *bioI* caused *B. subtilis* to grow poorly in the absence of biotin. The bradytroph phenotype of *bioI* mutants was overcome by pimelic acid, suggesting that the product of *bioI* functions at a step prior to pimelic acid synthesis. The *B. subtilis bio* operon is preceded by a putative vegetative promoter sequence and contains just downstream a region of dyad symmetry with homology to the *bio* regulatory region of *B. sphaericus*. Analysis of a *bioW-lacZ* translational fusion indicated that expression of the biotin operon is regulated by biotin and the *B. subtilis birA* gene.

---

Biotin biosynthesis in *Escherichia coli* and *Bacillus sphaericus* has been studied extensively at both the biochemical and molecular biological levels (9, 14, 17, 29). The enzymes involved in the conversion of pimeloyl coenzyme A (CoA) to biotin have been isolated from both of these bacterial species and characterized (2, 14, 16, 23, 29, 42). The analogous pairs of enzymes from the two species are similar, although some of the components involved in the last step in biotin synthesis remain to be elucidated (6, 15, 25, 26, 37, 46). 8-Amino-7-ketopelargonic acid (KAPA), the product of *bioF*, catalyzes the conversion of pimeloyl-CoA and alanine to KAPA (Fig. 1). 7,8-Diaminopelargonic acid (DAPA) aminotransferase, the product of *bioA*, then uses S-adenosylmethionine as a donor to transfer an amino group to KAPA, yielding DAPA. Dethiobiotin (DTB) synthetase (*bioD*) catalyzes the closure of the ureido ring to produce DTB, and finally the product of *bioB*, biotin synthase, functions together with a number of other components, including flavodoxin (6, 26), S-adenosylmethionine (6, 15, 25, 37, 46), and possibly cysteine (6, 15, 47), to convert DTB to biotin.

In *E. coli* the genes that encode these enzymes are located in two divergently transcribed operons, controlled by a single operator that interacts with the BirA repressor (1, 9). In *B. sphaericus*, the genes are located in two separate, unlinked operons (17). The early steps of the pathway, those involved in the synthesis of pimeloyl-CoA, are less well understood (27, 48). *B. sphaericus* contains an enzyme, pimeloyl-CoA synthetase (*bioW*), that converts pimelic acid to pimeloyl-CoA (17, 43). *E. coli* lacks this enzyme and cannot use pimelic acid as an intermediate in biotin synthesis (17, 27, 48). *E. coli* contains two genes, *bioC*, which is located in the *bio* operon, and *bioH*, which is unlinked to the other *bio* genes, that appear to be involved in the early steps of biotin biosynthesis leading up to pimeloyl-CoA, but their exact roles are unknown (14, 32).

Although there are no obvious homologs of *bioC* or *bioH* in the two sequenced *bio* operons of *B. sphaericus*, Lemoine et al. (32) have suggested that both the BioC protein of *E. coli* and the BioX protein of *B. sphaericus* may function as acyl carrier proteins involved in pimeloyl-CoA synthesis. Like most acyl carrier proteins, BioX possesses a consensus sequence for a phosphopantetheine attachment site. BioC does not possess such an attachment site; however, Lemoine et al. (32) proposed that BioC functions in a way similar to that of chalcone synthase, an enzyme which does not require the 4′-phosphopantetheine group. They have also identified a consensus sequence in BioH protein which is characteristic of acyltransferase and thioesterase proteins.

Prior to this work, little was known about the biotin biosynthetic genes in *Bacillus subtilis*. Pai (40) had isolated a collection of biotin auxotrophs and shown that they all map at the same locus on the chromosome (262°) and are weakly linked to *aroG* by transformation. On the basis of nutritional requirements and excreted products, the mutants could be divided into three classes that appeared to correspond to *E. coli* mutations in *bioB*, *bioA*, and *bioF* (17, 40). Here we report that the *bio* genes of *B. subtilis* are located in a single operon and that genes with similarity to *bioW*, *bioA*, *bioF*, *bioD*, and *bioB* are found in this operon. In addition, the *B. subtilis* operon contains two other genes that correspond to no other known *bio* genes.

---

## MATERIALS AND METHODS

Bacterial strains, plasmids, and growth conditions. Bacterial strains used in this study are listed in Table 1. Plasmids pUC9 (57), pUC19 (61), pCL1921 (33), pJGP44, and pTR264 (31) were used for cloning into *E. coli*. pJGP44 is a derivative of pBR322 that contains an 82-bp polylinker with multiple restriction sites inserted between the filled *EcoRI* site and the *NruI* site of pBR322 (6a). *E. coli* strains were grown on Luria-Bertani medium without glucose. Competent *E. coli* was prepared by the method of Inoue (28) or purchased from Bethesda Research Laboratories, Inc. *E. coli* cells transformed by electroporation were prepared, stored, and transformed as described by Dower et al. (12). *B. subtilis* cells were grown on Tryptose Blood Agar Base (Difco) plates or in veal infusion broth-yeast extract (VY) broth (7). Competent *B. subtilis* was prepared, stored, and transformed as described by Dubnau and Davidoff-Abelson (13). Plasmid

* Corresponding author. Mailing address: OmniGene Bioproducts, Inc., 763D Concord Ave., Cambridge, MA 02138. Phone: (617) 576-1966, ext. 224. Fax: (617) 547-9256.

B. sphaericus

B. subtilis

E. coli

FIG. 1. Biotin biosynthesis pathways in *E. coli*, *B. subtilis*, and *B. sphaericus*. The question marks indicate that the pathways for the synthesis of the intermediates pimelyl-CoA in *E. coli* and pimelic acid in *B. subtilis* and *B. sphaericus* are not known. The last reaction is catalyzed by the *bioB* gene product; the potential sulfur donor cysteine (Cys-[S]) and the additional proteins and cofactors listed are based on in vitro studies using *E. coli* cell extracts (6, 15, 25, 26). asp, aspartate; asn, asparagine; glu, glutamate; ser, serine; L-Ala, alanine; SAM, S-adenosyl-L-methionine.

DNA from *E. coli* was prepared by using purification kits purchased from Qiagen, Inc.

Cloning of the biotin operon. The positive selection vector pTR264 (31) was used to construct a library of ~8- to 10-kb fragments of *B. subtilis* GP208 DNA in *E. coli*. Clones with inserts were selected by plating transformants on Luria-Bertani plates with tetracycline (10 μg/ml).

pTR264, prepared in *E. coli dam* mutant strain GM48 and digested with *BclI*, was ligated with chromosomal DNA from *B. subtilis* GP208 which had been partially digested with Sau3A and fractionated on a sucrose gradient (8- to 12-kb fragments). *E. coli* biotin mutants R879 (*bioA*), R875 (*bioB*), R878 (*bioC*), R877 (*bioD*), and R872 (*bioF*) were each transformed with the ligated DNA by electroporation, and Bio<sup>+</sup> colonies were selected on BIOS medium (7). Bio<sup>+</sup> transformants that were also Tc<sup>r</sup> were analyzed for plasmid content.

Cloning of a *B. subtilis* fragment containing the 5′ end of the *bio* operon. Analyses of restriction maps and Southern blot data using a *bioW*-containing fragment from pBIO100 as a probe indicated that a 5.5-kb *PstI* fragment would contain a complete *bioA* gene and ~2.7 kb of upstream sequences (data not shown). A plasmid, pBIO116, containing this fragment was subsequently recovered when a mini plasmid library of 4.4- to 6.6-kb *PstI* fragments of *B. subtilis* chromosomal DNA was transformed into *E. coli* BI259 (*bioA pcnB*) and Bio<sup>+</sup> colonies were selected. pBIO116 transformed BI259 again to biotin prototrophy at a high frequency but did not transform R879 (*bioA pcnB<sup>+</sup>*) to either biotin prototrophy or ampicillin resistance.

Only limited quantities of pBIO116 were recovered from the *pcnB* strain. The *pcnB80* allele which was used in this cloning experiment is reported to reduce the copy number of pBR322 replicons to about 6% of wild-type yields (34). To improve plasmid yields without impairing plasmid stability, the unique *BamHI* site in the 3′ end of *bioW* was used to subclone a 2.8-kb *BamHI-PstI* fragment from pBIO116 into a low-copy-number plasmid, pCLI921 (33). A plasmid, pBIO350, that contained the correct 2.8-kb *BamHI-PstI* fragment was recovered. The quantity of pBIO350 recovered from this strain was significantly higher than that of pBIO116 isolated from the *pcnB80* strain.

Construction of deletions in the biotin operon. The 10-kb *EcoRI*-to-*BamHI* fragment that contained most of the *bio* operon (except for part of *bioW* and the promoter) was cloned from *B.* *subtilis* GP275 (an isogenic strain of GP208) into *EcoRI*- and *BamHI*-digested pJGP44 to give pBIO201. Several deletion mutants and subclones were made from pBIO201 in order to roughly locate the *B.* *subtilis* *bio* genes corresponding to the known *E.* *coli bio* genes by complementation. Deletions were made by cutting with the appropriate restriction enzyme, filling in overhangs with Klenow fragment when necessary, and religating. Subclones were made into pUC9.

The 1.5-kb *EcoRI*-to-*ClaI* fragment of pBIO201 was removed to give pBIO202, the 1.6-kb *EcoRI*-to-*XhoI* deletion gave pBIO203, the 4.5-kb *EcoRI*-to-*Asp* 718 deletion gave pBIO204, the 5.2-kb *EcoRI*-to-*SmaI* deletion gave pBIO205, and the 7-kb deletion from *EcoRI* to the rightmost *EcoRV* gave pBIO206. The 4.3-kb *BamHI*-to-*SmaI* deletion gave pBIO207, the 3.6-kb insert *HindIII*-to-polylinker *HindIII* deletion gave pBIO208, and the 3.9-kb *BglII*-to-*BglII* deletion gave pBIO209. The 2.6-kb central *PstI* subclone gave pBIO210, the central 4.1-kb *EcoRV* subclone (into the *SmaI* site of pUC9) gave pBIO211, and the 3.3-kb *EcoRI*-to-*EcoRV* subclone (into the *EcoRI*-to-*SmaI* backbone of pUC9) gave pBIO212.

Construction of clones of *bioI* and/or *orf2*. Copies of *bioI* and *orf2* were generated by PCR using a Boehringer Mannheim PCR kit. A *HindIII* site was introduced at the 5′ end of each gene, a *BamHI* site was introduced at the 3′ end of *bioI*, and an *Asp718I* site was introduced at the 3′ end of *orf2*. The PCR-generated fragments were each cloned into three plasmids with different copy numbers, i.e., the low-copy-number plasmid pCL1921; a medium-copy-number plasmid, pJGP44; and the high-copy-number plasmid pUC19. In two of these recombinant plasmids expression of *bioI* and *orf2* is under the control of the *lac* promoter (pCL1921 and pUC19).

DNA sequencing. The *B.* *subtilis bio* genes contained on clones pBIO100 and pBIO350 were sequenced by the Sanger dideoxy sequencing method using Sequenase kits, version 2.0 (United States Biochemicals, Cleveland, Ohio) as instructed by the manufacturer. The strategy used to obtain the DNA sequence of the 8- to 10-kb region was to divide the region into four plasmid subclones of approximately 2 to 3 kb and then make nested sets of deletions progressing through each subclone. To generate the nested deletions, the exonuclease III-endonuclease S1 method was used; the reagents were purchased as a Generase kit (instructions included; Promega, Madison, Wis.). Nested deletions were made from both ends for three of the subclones and from one end for the fourth. Sequencing both sets of nested deletions for three of the subclones gave the sequence of both strands of each subclone. For pBIO350, one strand was determined similarly and the opposite strand was determined by synthesizing sequencing primers at intervals of approximately 150 bp. The junctions between non-overlapping subclones were confirmed by sequencing from synthetic primers using pBIO201 or pBIO100 (or subclones thereof) as a template. The sequences were aligned and compared with the DNASTAR computer program (DNASTAR, Inc., Madison, Wis.).

Construction of *cat* insertions. A *cat* cassette, encoding chloramphenicol resistance, derived from pMI1101 (62) was inserted by ligation into the coding region of *bioW* by using a *BamHI* site; between the *BspEI* and *PmlI* sites, deleting 260 bp of *bioB*; into *bioI* by using a *SmaI* site; between a pair of *SstI* sites, deleting 457 bp of *orf2* plus 149 bp of downstream sequences; into *orf3* by using an *XmnI* site; into *orf6* by using an *EcoRV* site; and between the pair of *BstBI* sites, deleting *orf4*. The *cat* cassette was also used to entirely replace the *bio* promoter region by ligating it between the *HpaI* sites. In each of the *orf2-SstI*, *orf4-BstBI*, and *bioB–BspEI-PmlI* constructions, the *cat* gene was inserted in only one direction. In all other constructions, two different plasmid derivatives, in which the *cat* cassette was inserted in either possible orientation, were generated. Each of these mutations was then integrated into the *bio* locus by first linearizing the *cat*-containing plasmid by a restriction enzyme cut outside of the *bio* DNA; then transforming this cut DNA into a competent prototrophic *B.* *subtilis* strain, PY79; and then selecting for chloramphenicol resistance (*Cm<sup>r</sup>*) at a final chloramphenicol concentration of 5 μg/ml.

Construction of a *bioW-lacZ* translational fusion. To construct a *bioW-lacZ* translational fusion, a 3.1-kb *BamHI*-to-*BglII* fragment containing most of the coding region of *E.* *coli lacZ* (amino acid residues 24 to 1021) was ligated into the *BamHI* site of pBIO350, to give pBIO397. The *bioW-lacZ* fusion was then used in the construction of a second plasmid to allow integration of the fusion into the modified SPβ prophage SPβc2del2::Tn917::pSK10Δ6 (63). To bring about this integration, the following four DNA fragments were ligated together to generate plasmid pBIO407: a 6-kb *PstI*-to-*KpnI* fragment of pBIO397 containing the *bioW-lacZ* fusion, a PCR-generated 2-kb *KpnI*-to-*BamHI* fragment containing the *oriC* and *repA* region of pCL1921, a PCR-generated 1.2-kb *PstII*-to-*SalI* fragment containing the *cat* gene of pC194 (22), and a PCR-generated *SalI*-to-*BglII* fragment containing the pUC9 *bla* gene. pBIO407 contains the *bla*, *lacZ*, and selectable *cat* genes in the appropriate orientation to allow integration of the *bioW-lacZ* fusion into the SPβc2del2::Tn917::pSK10Δ6 prophage of ZB493 (63). A specialized transducing lysate containing SPβ::*bioW-lacZ* was obtained by heat induction at 50°C.

TABLE 1. Bacterial strains used in cloning, complementation, and analysis of *B. subtilis bio* genes

| Strain | Relevant genotype or description | Source or reference(s) |
|--------|----------------------------------|------------------------|
| *B. subtilis* |  |  |
| PY79 | SP$\beta^c$ prototroph | 62 |
| BI421 | *birA* | 7 |
| JKB3173 | *bioA173 aroG932* | 17, 40 |
| BGSC1A92 | *bioB141 aroG932 sacA321 argA2* | Bacillus Genetic Stock Center |
| JKB3112 | *bioF112 aroG932* | 17, 40 |
| GP208 | *leu amyE $\Delta apr \Delta npr \Delta isp-1$ (Met$^{-}$)* | 49 |
| GP275 | *leu amyE $\Delta apr \Delta npr \Delta isp-1$ (Met$^{-}$) $\Delta epr \Delta bpr \Delta mpr \Delta hpr$* | 50 |
| ZB493 | *trpC2 pheA1 abrB703 SP$\beta c2del2::Tn917::pSK10\Delta 6$* | 63 |
| *E. coli* |  |  |
| YMC9 | $\Delta lacU169 endA1 hsdR17 supE44 thi-1$ | 4 |
| DH5$\alpha$ | $F^- (f80dlacZ\Delta M15) \Delta lacU169 recA1 endA1 hsdR17 supE44 thi-1 gyrA relA1$ | Bethesda Research Laboratories |
| GM48 | $F^- thr leu thi lacY galK galT ara fhyA tsx dam dcm supE44$ | New England Biolabs |
| R872 | *bioF3* | 8 |
| R875 | *bioB17* | 8 |
| R877 | *bioD19* | 8 |
| R878 | *bioC18* | 8 |
| R879 | *bioA24* | 8 |
| BM7086 | $\Delta(mal-bioH) gal$ | 19 |
| BI259 | *bioA24 pcnB80* | This study |

Partial diploids were generated by transforming the appropriate Bio$^+$ *B. subtilis* strain to Cm$^r$ with the *cat*-containing transducing phage. These partial diploids were then grown in Spizizen's minimal salts medium (52) containing 0.4% glucose and 0.04% sodium glutamate in the presence or absence of biotin (10 $\mu$g/liter). Samples were harvested at mid-exponential phase for o-nitrophenyl-$\beta$-galactoside assay (35).

Nucleotide sequence accession number. The DNA sequence of 10.2 kb including the *bio* operon has been submitted to GenBank under accession number U51868.

**RESULTS AND DISCUSSION**

Cloning of the *B. subtilis* biotin genes. A plasmid library of random *B. subtilis* partial Sau3A fragments ($\sim$8 to 12 kb) was constructed in *E. coli* by using the positive selection vector pTR264 as described in Materials and Methods. The library was used to transform *E. coli bio* mutants R879 (*bioA24*), R875 (*bioC18*), R877 (*bioD19*), R872 (*bioF3*), and BM7086 ($\Delta malA-bioH$) (8, 19). Bio$^+$ transformants containing plasmids that complemented each *E. coli bio* mutation were recovered. Plasmids pBIO100 and pBIO101 were isolated by complementation in R879 (*bioA*); plasmids pBIO102 and pBIO103 were isolated by complementation in R877 (*bioD*); plasmid pBIO104 was isolated by complementation in R872 (*bioF*); plasmids pBIO109 and pBIO110 were isolated by complementation in BM7086 ($\Delta bioH$); and plasmids pBIO111 and pBIO112 were isolated by complementation in R878 (*bioC*). Initial restriction analysis of the isolated plasmids indicated significant overlap of the cloned DNA fragments, suggesting that the *B. subtilis* biotin locus contains genes functionally equivalent to the *E. coli* genes *bioA*, *bioC*, *bioD*, *bioF*, and *bioH* (Fig. 2). pBIO100 extended the farthest to the right,

![Diagram](attachment:diagram.png)

FIG. 2. Physical map of the *B. subtilis bio* operon and flanking DNA. The locations of the structural genes, the putative promoter, the regulatory regions, and the transcription termination sites were determined from the nucleotide sequence of the 10.2-kb *BstXI-PstI* DNA region. Assignment of the *bio* genes is described in the text. Complementation of *E. coli bio* mutants by plasmids containing cloned fragments of the *B. subtilis bio* operon and flanking regions is indicated by plus signs; no complementation is indicated by minus signs. Mutations not listed were not tested. Endpoints of DNA segments carried by pBIO100, pBIO109, and pBIO112 are approximate. Symbols: □, ORF; ■, *Bacillus* RBS; Ω, putative rho-independent transcription termination site; $\uparrow$, possible start site of transcription for a $\sigma^A$-recognized promoter.

~300 bp beyond the unique *Bam*HI site at the right end of the restriction map of the *bio* locus shown in Fig. 2. pBIO110 extended the farthest to the left, ~1,100 bp beyond the *Eco*RI site at the other end of the restriction map (data not shown). Southern blots indicated that the insert DNA of pBIO100 was derived from a single continuous segment of the *B.* subtilis chromosome (data not shown).

**Complementation and marker rescue of *B.* subtilis and *E.* coli *bio* mutants with plasmids containing *B.* subtilis *bio* genes.** To confirm that the cloned DNA of pBIO100 contained *B.* subtilis *bio* genes, pBIO100 was tested for the ability to marker rescue *B.* subtilis *bio* mutations (40). The plasmid restored biotin prototrophy to *bioA*, *bioB*, and *bioF* mutants at high frequencies, indicating that the cloned DNA contained all or part of each of these *B.* subtilis *bio* genes. Several of the pBIO plasmids were also examined for their ability to complement *E.* coli strains with mutations in *bioA*, *bioB*, *bioC*, *bioD*, *bioF*, or *bioH*. Most plasmids complemented more than one *E.* coli biotin mutation (Fig. 2). The isolate pBIO112 complemented *E.* coli mutations in *bioA*, *bioB*, *bioC*, *bioD*, *bioF*, and *bioH* (Fig. 2); however, pBIO112 did not complement the *E.* coli Δ(*gal-uvrB*) mutation, which removes the entire *E.* coli *bio* locus.

The 9.9-kb *Eco*RI-to-*Bam*HI fragment containing most of the *bio* locus was cloned into a derivative of pBR322, pJGP44, resulting in plasmid pBIO201. To perform complementation experiments with plasmids with defined endpoints, a series of deletions was generated from pBIO201 as described in Materials and Methods. Each deletion-carrying plasmid was introduced into five *E.* coli *bio* mutants (*bioA*, *bioC*, *bioD*, *bioF*, and *bioH*), and complementation was scored. As shown in Fig. 2, the *B.* subtilis *bio* genes complementing these *E.* coli genes were located in the 8-kb fragment of DNA from *Bam*HI to *Xho*I. The removal of 5.4 kb from the left of the pBIO201 insert (pBIO205) eliminated the ability to complement *bioC* and *bioH* mutants. pBIO206 contained only the rightmost 2.5 kb of the biotin cluster and complemented only *bioA* and *bioF* mutants. One clone, pBIO208, in which the rightmost 4.0 kb of insert DNA was deleted complemented *E.* coli *bioC* and *bioH* mutants but failed to complement *E.* coli *bioA*, *bioD*, or *bioF* mutants. These results suggested the gene order (*bioC*, *bioH*)-*bioD-bioF-bioA*.

**Cloning of a *B.* subtilis fragment containing the 5′ end of the *bio* operon.** As described below, DNA sequences of the rightmost end of the cloned insert (pBIO100) that extended furthest to the right revealed about 300 bp of an open reading frame (ORF) that was homologous to *B.* sphaericus *bioW*, the gene encoding pimeloyl-CoA synthase (17, 43), followed immediately by genes with strong similarity to *bioA*, *bioF*, *bioD*, and *bioB* from *E.* coli and *B.* sphaericus (Fig. 2). The 5′ end of *bioW* and the promoter of the *bio* operon were not present on any of the originally cloned DNA fragments. Suspecting that it might be difficult to clone this region in high-copy-number plasmids, we cloned DNA fragments containing *bioA* and the adjacent upstream region by complementation in an *E.* coli strain containing a *bioA* mutation and a *pcnB* mutation to reduce plasmid copy number (34) as described in Materials and Methods.

**Identification and organization of *bio*-specific coding regions and transcriptional regulatory signals.** Analysis of ~10 kb of the DNA sequence from pBIO100 and pBIO350 indicated that many or all of the *B.* subtilis biotin biosynthetic genes are located in a single operon containing seven coding regions (Fig. 2). The *bioW* gene appears to be the first gene in the operon. Approximately 84 bp upstream from *bioW* is a putative vegetative (*σ*^A^) promoter sequence (TTGACA—17

![Figure 3](#fig3)

FIG. 3. Comparison of the nucleotide sequences of the *B.* sphaericus *bioDAYB* regulatory region and the putative *B.* subtilis *bio* promoter and regulatory region. The upper sequence represents the putative *B.* subtilis *bio* promoter and regulatory region. The lower sequence represents the *B.* sphaericus *bioDAYB* regulatory region (17). The sequence shown spans nucleotides 1995 to 2072 of the nucleotide sequence submitted to GenBank (accession no. U51868 [see Materials and Methods]). Symbols: double bold underline, 15-bp putative regulatory region of *B.* sphaericus *bioDAYB*; dashed lines, regions of dyad symmetry; single bold underlines, –35 and –10 regions of a possible promoter. RBS, putative *Bacillus* RBS. The nucleotides above or below the sequences were displaced to facilitate sequence alignment.

bp—TATATT [36]). This probable promoter sequence was followed by a 33-bp segment with strong sequence homology to the “regulatory” sites of the *B.* sphaericus *bio* operons and lesser similarity to the *E.* coli *bio* operator site. Comparison of the nucleotide sequences of this region with those of the 5′ noncoding region of the *B.* sphaericus *bioDAYB* operon (51) revealed two clusters of conserved nucleotides (13 and 11 bp) separated by a nonconserved 9-bp segment (Fig. 3).

The *bioW* gene (259 amino acids) is followed by ORFs with homology to *bioA* (448 amino acids), *bioF* (389 amino acids), *bioD* (231 amino acids), and *bioB* (335 amino acids) (Table 2). The next two ORFs, *bioI* (395 amino acids) and *orf2* (253 amino acids), showed no sequence similarity to *bioC* or *bioH* or to any other known *bio* gene (Fig. 2 and Table 2). Comparison with the protein database of GenBank, however, indicated significant similarity of the deduced amino acid sequence of *bioI* to those of cytochrome P-450 enzymes from *Bacillus megaterium* (P-450~BM-1~ [21]), *Saccharopolyspora erythraea* (EryF [20] and EryK [53]), and other organisms (53). Cytochrome P-450s include monooxygenases known to catalyze hydroxylation of many different kinds of substrates, including fatty acids. Since synthesis of pimelic acid, a precursor to biotin, might involve hydroxylation and/or further oxidation of a fatty acid, *bioI* may be involved in an early step in biotin synthesis (see below). Although similar protein database searches did not reveal a specific function for the *orf2* gene product, significant similarity between the N-terminal end of the deduced protein and putative NAD or NADH binding sites of short-chain alcohol dehydrogenases (e.g., BphB [3, 55]), dehydratases (e.g., RfbB [30]), and the β-ketoreductase domain of EryA~II~ of *S.* erythraea (11) was detected. Since this region of Orf2 also contains a GXGXXG motif, which is characteristic of a FAD or NAD binding site (60), it is conceivable that *orf2* encodes an NADH- or NADPH-dependent enzyme.

Each gene in the *bio* operon is preceded by a ribosome binding site (RBS), with calculated ΔGs ranging from –10.8 to –18.6 kcal (ca. –45.2 to –77.8 kJ)/mol (Table 2). All genes are oriented in the same transcriptional direction (right to left). In addition, the 5′ ends of *bioA*, *bioF*, *bioD*, and *bioB* overlapped the 3′ ends of the genes preceding them, suggesting that expression of these genes could be regulated, in part, by translational coupling. *bioI* and *orf2* are separated from the genes that precede them by 68- and 67-bp intercistronic regions, respectively.

*orf2* appears to be the last gene in the *bio* operon, as it is immediately followed by a region of dyad symmetry resembling a rho-independent transcription termination site (ΔG = –15.4 kcal [ca. –64.4 kJ]/mol). Another stem-loop structure with terminator-like features was detected in the region between

TABLE 2. Enzymes, genes, and regulatory elements of the *B. subtilis bio* operon and flanking DNA.

| Gene   | RBS ΔG (kcal/mol)${}^{a}$ | Predicted start codon | Enzyme or function                | Calculated no. of amino acids | Estimated $M_{r}$ | % Amino acid identity to corresponding gene product from: |
|--------|-----------------------------|-----------------------|-----------------------------------|-----------------------------|--------------------|---------------------------------------------------------|
|        |                             |                       |                                   |                             |                    | $E. coli{ }^{b}$                                      |
|        |                             |                       |                                   |                             |                    | $B. sphaericus{ }^{c}$                                |
|        |                             |                       |                                   |                             |                    | Other                                                |
| $bioW$ | $-10.8$                     | ATG                   | Pimeloyl-CoA synthase             | 259                         | 29,633              | 44                                                    |
| $bioA$ | $-15.8$                     | ATG                   | DAPA aminotransferase            | 448                         | 50,118              | 34                                                    |
| $bioF$ | $-11.6$                     | TTG                   | KAPA synthase                    | 389                         | 42,567              | 35                                                    |
| $bioD$ | $-18.6$                     | TTG                   | DTB synthetase                   | 231                         | 25,114              | 29                                                    |
| $bioB$ | $-12.2$                     | ATG                   | Biotin synthase                  | 335                         | 36,931              | 71                                                    | 22${ }^{d}$ |
| $bioI$ | $-18.4$                     | GTG                   | Cytochrome P-450                 | 395                         | 44,838              |                                                       | 30${ }^{e}, 33{ }^{f}$ |
| $orf2$ | $-17.6$                     | GTG                   | Unknown                          | 253                         | 28,204              |                                                       |           |
| $orf3$ | $-20.0$                     | GTG                   | Unknown membrane-associated       | >258                        | >28,600             | 53${ }^{g}, 24{ }^{h}, 23{ }^{i}$                      |           |
|        |                             |                       | transport protein                |                             |                    |                                                       |           |
| $orf4$ | $-10.0$                     | ATG                   | Unknown                          | 299                         | 33,780              |                                                       |           |
| $orf6$ | $-17.4$                     | ATG                   | Unknown regulatory protein       | >266                        | >29,200             | 30${ }^{j}, 26{ }^{k}$                               |           |

${ }^{a}$ Calculated according to the method of Tinoco et al. (56). One kilocalorie equals 4.184 kJ.

${ }^{b}$ Identity to $E. coli bio$ gene products (38).

${ }^{c}$ Identity to $B. sphaericus bio$ gene products (17).

${ }^{d}$ Identity to $E. coli lipA$ product (44).

${ }^{e}$ Identity to $B. megaterium$ cytochrome P-450${ }_{BM-1}$ (21).

${ }^{f}$ Identity to $S. erythraea eryF$ product (20).

${ }^{g}$ Identity to $B. subtilis lplC$ product (18).

${ }^{h}$ Identity to $E. coli malG$ product (10).

${ }^{i}$ Identity to $E. coli ugpE$ product (39).

${ }^{j}$ Identity to $E. coli ebgR$ product (54).

${ }^{k}$ Identity to $E. coli purR$ product (45).

TABLE 3. Biotin-regulated expression of SPβ::bioW-lacZ translational fusion

| Relevant genotype | β-Galactosidase sp act (Miller units)${}^{a}$ |
|-------------------|---------------------------------------------|
|                   | With biotin${}^{b}$                          | Without biotin |
| bioW-lacZ bio${}^{+}$ | $0.05 \pm 0.01$                           | $0.5 \pm 0.06$ |
| bioW-lacZ bio${}^{+}$ birA | $0.8 \pm 0.15$                          | $0.9 \pm 0.07$ |

${}^{a}$ Data are averages ± standard deviations for two isolates and two assays each (calculated according to the method of Miller [35]).

${}^{b}$ Biotin was present at 100 μg/liter.

activity in the birA strain was only somewhat higher than the levels observed in PY79 containing SPβ::bioW-lacZ and grown under nonrepressing conditions. Similar results were obtained when a bioW-lacZ fusion was introduced by integration of a circular plasmid (pBIO397cat) by Campbell-like recombination at the bio locus (data not shown). These results suggest that the B. subtilis bio promoter is regulated by birA and biotin, as is the case for the divergent bio promoters of E. coli. In future work, it will be interesting to establish whether the B. subtilis bio operon is regulated by the B. subtilis BirA by a repressor-operator mechanism similar to that used in E. coli for the regulation of biotin biosynthesis.

The B. subtilis bioI gene complements both E. coli bioC and bioH mutants. The presence of two genes, bioI and orf2, with homology to neither bioC nor bioH of E. coli, raised the issue of which gene(s) was complementing which E. coli mutant. Complementation studies using plasmid subclones that contained either bioI or orf2 alone under the transcriptional control of the lacZ promoter (see Materials and Methods) indicated that bioI alone was sufficient to complement both E. coli BM7086 (∆bioH) and E. coli R878 (bioC). Plasmids containing orf2 did not give normal complementation of either E. coli BM7086 or E. coli R878. The cytochrome P-450-like product of the bioI gene of B. subtilis can apparently supply an activity needed for biotin synthesis that can substitute for, or bypass, the activity missing in either bioC or bioH mutants of E. coli.

Insertional mutagenesis of the bio operon and flanking coding regions. To verify the boundaries of the bio operon predicted from the nucleotide sequence and to confirm the roles of previously unidentified bio genes, a cat cassette (chloramphenicol resistance gene) was used to construct insertions or deletions in bioW, bioB, bioI, orf2, the bio promoter region, orf3, orf4, and orf6. First, plasmid derivatives containing these mutations were constructed in E. coli, and then the cat insertions were transferred to the bio locus of B. subtilis by DNA transformation (see Materials and Methods). The locations of these mutations are diagrammed in Fig. 4. As summarized in Table 4, insertions into orf3 and orf6 and deletion of orf4, which represent mutation of the coding regions located outside of the predicted bio operon, generated Cm${}^{r}$, prototrophic colonies. Insertions and deletions in the bio operon gave results that generally supported the conclusions from the nucleotide sequence data. Replacement of the region upstream of bioW containing the putative $P_{bio}$ promoter with the cat gene oriented opposite to the biotin operon and interruption of bioW with the cat gene oriented in either direction relative to the bio operon generated an unambiguous Bio${}^{-}$ phenotype. However, replacement of the putative $P_{bio}$ promoter region with the cat gene inserted in the same transcriptional direction as the biotin operon generated Bio${}^{-}$ cells that reverted to Bio${}^{+}$ at a high frequency (0.1%). Bioassay experiments indicated that biotin vitamer production from such a Bio${}^{+}$ revertant was increased in the presence of low concentrations of chloramphenicol, suggesting that expression of the biotin operon resulted from read-through transcription from the chloramphenicol-inducible cat promoter. We also observed that the bioB gene was expressed when the cat gene was inserted into the biotin operon upstream of bioB and oriented in the same transcriptional direction, as judged by growth of such bacteria on DTB (Table 4).

Deletion (PmlI to BspBI) of the 3' end of bioB also generated a Bio${}^{-}$ phenotype, confirming that bioB was required for biotin biosynthesis. However, the 3' end of the operon could not be definitively identified by this genetic method. Insertions into bioI resulted in Cm${}^{r}$ colonies that were partially deficient in biotin production, i.e., that grew poorly on biotin-free medium but grew as well as wild-type colonies in the presence of biotin (33 μg/ml), whereas the orf2::cat mutation gave Bio${}^{+}$ colonies. These results suggested that bioI is not absolutely required for biotin production and that the orf2 gene product is dispensable for biotin biosynthesis. The question of whether orf2 encodes a redundant enzyme that functions in biotin synthesis or simply an unrelated protein awaits further experimentation. The bioA gene of E. coli is also located in an operon with another ORF (orfI) also of unknown function. However,

![Figure 4](#fig4)

FIG. 4. Locations of cat-containing insertions and deletions within the B. subtilis bio operon and flanking DNA. As described in Materials and Methods, in vivo mutations of the bio genes and flanking open reading frames were generated either by inserting a 1.5-kb cat-containing cassette into the indicated restriction site (A, XmnI; B, SmaI; C, BamHI; or D, EcoRV) or by replacing the indicated region with the cat cassette (E, replacement of a 606-bp SstI fragment; F, replacement of a 260-bp PmlI-BspEI fragment; G, replacement of three adjoining HpaI fragments totalling 313 bp; H, replacement of a 966-bp BstBI fragment). Not all restriction sites are shown. B. subtilis strains containing these mutations were examined for their biotin phenotypes, and the results are tabulated in Table 4.

TABLE 4. Characterization of insertion and deletion derivatives of the biotin operon

| Biotin operon derivative (mutation) and cat gene orientation$^{a}$ | Biotin phenotype$^{b}$ | Growth on$^{c}$: |  |  |
| --- | --- | --- | --- | --- |
|  |  | Minimal medium$^{d}$ | DTB$^{e}$ | Pimelic acid$^{f}$ |
| Wild-type bio operon A ($\Omega orf3$) | + | + | + | + |
| R | + | + |  |  |
| L | + | + |  |  |
| B ($\Omega bioI$) |  |  |  |  |
| R | $+/-$ | $+/-$ | + | + |
| L | $+/-$ | $+/-$ |  | + |
| C ($\Omega bioW$) |  |  |  |  |
| R | – | – | – | – |
| L | – | – | $+/-$ |  |
| D ($\Omega orf6$) |  |  |  |  |
| R | + | + |  |  |
| L | + | + |  |  |
| E ($\Delta orf2$) |  |  |  |  |
| R | + | + |  |  |
| F ($\Delta bioB$) |  |  |  |  |
| R | – | – |  |  |
| G ($\Delta P_{bio}$) |  |  |  |  |
| R | – | – | – | – |
| L | $+^{g}$ | + | $+^{g}$ |  |
| H ($\Delta orf4$) |  |  |  |  |
| L | + | + |  |  |

$a$ See Fig. 4 for a map of $cat$ insertions within the biotin operon. Insertion derivatives having the $cat$ gene in either orientation were obtained; R (right) and L (left) identify the transcriptional orientation of the inserted $cat$ gene when the bio operon is oriented as shown in Fig. 4.

$b$ Biotin phenotype determined by patching bacteria on biotin-free agar plates. +, biotin prototroph; $+/-$, biotin bradytroph; –, biotin auxotroph.

$c$ +, prototrophic; $+/-$, bradytrophic; –, auxotrophic.

$d$ Growth of bacteria on Spizizen’s minimal medium agar plates.

$e$ Growth of bacteria on biotin-free agar plates containing 33 $\mu$g of DTB per liter.

$f$ Growth of bacteria on biotin-free agar plates containing 33 $\mu$g of pimelic acid per ml.

$g$ Appearance at a frequency of 0.1% of Bio$^{+}$ bacteria in which biotin synthesis is inducible by chloramphenicol.

homolog, and $bioC$ or $bioH$ mutants of $E$. coli cannot be rescued for growth on biotin-free medium by the addition of pimelic acid. However, both $\Delta bioH$ and $bioC$ mutants of $E$. coli containing pBIO403 grew in the absence of biotin when, and only when, pimelic acid (30 $\mu$g/ml) was added to the medium. This result suggests that $bioW$ encodes a pimelyl-CoA synthase that, in the presence of pimelic acid, can bypass $bioH$ and $bioC$ in $E$. coli.

Early steps in biotin biosynthesis. The early steps in biotin biosynthesis appear to be different in the gram-negative bacteria, such as $E$. coli and Serratia marcescens, and the gram-positive bacteria, such as $B$. subtilis and $B$. sphaericus, two distantly related Bacillus species. $E$. coli cannot use free pimelic acid as a precursor for biotin synthesis (14), and $^{13}$C labeling experiments indicate that free pimelic acid is not an intermediate in biotin biosynthesis (48). On the other hand, $B$. subtilis and $B$. sphaericus readily use pimelic acid, which is converted to pimelyl-CoA by pimelyl-CoA synthase, the product of the $bioW$ gene. When supplied with the $bioW$ gene from $B$. subtilis or $B$. sphaericus (17, 43), $E$. coli can use pimelic acid to bypass the biotin auxotrophy of $bioC$ or $bioH$ mutants.

Is pimelyl-CoA synthase an obligatory part of the biotin biosynthetic pathway in $B$. subtilis, or is it part of an alternative pimelic acid salvage pathway? While the answer to this question is not clear, preliminary experiments indicate that the $bioW$ gene product is required for biotin synthesis in $B$. subtilis. Insertion of the $cat$ gene in place of the promoter region of the biotin operon, oriented in the same direction as the bio operon, yielded Bio$^{-}$ colonies that reverted to Bio$^{+}$ at a frequency of 0.1%. Insertion of the same $cat$ gene in $bioW$, also oriented in the same direction as the bio operon, yielded a nonreverting Bio$^{-}$ phenotype. However, such mutants were able to grow weakly on DTB or DAPA, indicating that the downstream $bioB$ and $bioD$ genes were being expressed. Furthermore, cells of $B$. subtilis containing an in-frame deletion within the chromosomal $bioW$ gene were also Bio$^{-}$ but were able to grow well on DTB or DAPA (unpublished results). We cannot rule out the possibility that both of these $bioW$ mutations exert a polar effect on $bioF$ or $bioA$ that is more deleterious than the effect on $bioD$ or $bioB$. However, it appears most likely that the pimelyl-CoA synthase is required for biotin synthesis in $B$. subtilis and that pimelic acid is a bona fide intermediate in biotin synthesis in $B$. subtilis.

On the basis of the cytochrome P-450-like structure of the BioI protein, we hypothesize that $B$. subtilis synthesizes pimelic acid by a pathway different from that of $E$. coli. Since other cytochrome P-450s are capable of oxidizing unsaturated fatty acid (59), we suggest that BioI may function to oxidize the double bond of an unsaturated fatty acid. Since BioI will complement an $E$. coli $bioC$ or $bioH$ mutant in the absence of pimelyl-CoA synthase, we further speculate that the BioI protein can use either a free fatty acid or the CoA thioester of a fatty acid as a substrate to produce pimelic acid or pimelyl-CoA, respectively.

there is no sequence similarity between the $B$. subtilis $orf2$ gene product and the $E$. coli $orf1$ gene product.

The biotin bradytroph phenotype generated by the $bioI::cat$ mutation appeared to be caused by inactivation of $bioI$ rather than by a polar effect because strains with mutations disrupting the downstream gene $orf2$ or $orf3$ were Bio$^{+}$. To determine whether the $bioI$ gene product was involved in formation of pimelic acid, we examined whether the $bioI::cat$ mutation could be bypassed by feeding pimelic acid. Derivatives of PY79 containing $bioI::cat$ with either orientation of the $cat$ gene grew as well as wild-type strains on biotin-free medium containing pimelic acid (Table 4). These results confirmed that the $bioI$ gene product is involved early in the biotin pathway.

$E$. coli cells expressing the $bioW$ gene of $B$. subtilis can utilize pimelic acid to synthesize biotin. On the basis of homology with the $B$. sphaericus $bioW$ gene, we hypothesized that the $B$. subtilis $bioW$ gene encodes a pimelyl-CoA synthase (43). To further examine this gene-enzyme relationship, we tested whether $B$. subtilis $bioW$ expression in $E$. coli could be utilized to synthesize biotin from pimelic acid as reported for the $B$. sphaericus $bioW$ (17). First, a fragment containing the $B$. subtilis $bioW$ gene and its promoter was cloned into plasmid pCL1921, generating pBIO403. Next, pBIO403 was introduced into $E$. coli $\Delta bioH$ or $bioC$ mutants and the resulting strains were tested for complementation. $E$. coli does not have a $bioW$

ACKNOWLEDGMENTS

We thank Alan Campbell for providing $E$. coli biotin mutants and Jim Hoch for providing $B$. subtilis biotin mutants.

REFERENCES

1. Abbott, J., and D. Beckett. 1993. Cooperative binding of the Escherichia coli repressor of biotin biosynthesis to the biotin operator sequence. Biochemistry 32:9649–9656.

2. Alexeev, D., S. M. Bury, C. W. G. Boys, M. A. Turner, L. Sawyer, A. J. Ramsey, H. C. Baxter, and R. L. Baxter. 1994. Sequence and crystallization of Escherichia coli dethiobiotin synthetase, the penultimate enzyme of biotin biosynthesis. J. Mol. Biol. 235:774–776.

3. Asturias, J. A., L. D. Eltis, M. Prucha, and K. N. Timmis. 1994. Analysis of three 2,3-dihydroxybiphenyl 1,2-dioxygenases found in *Rhodococcus globulus* P6. J. Biol. Chem. **269**:7807–7815.

4. Backman, K., Y.-M. Chen, and B. Magasanik. 1981. Physical and genetic characterization of the *glnA-glnG* region of the *Escherichia coli* chromosome. Proc. Natl. Acad. Sci. USA **78**:3743–3747.

5. Barker, D. F., and A. M. Campbell. 1981. Genetic and biochemical characterization of the *birA* gene and its product: evidence for a direct role of biotin holoenzyme synthetase in repression of the biotin operon in *Escherichia coli*. J. Mol. Biol. **146**:469–492.

6. Birch, O. M., M. Fuhrmann, and N. M. Shaw. 1995. Biotin synthase from *Escherichia coli*, an investigation of the low molecular weight and protein components required for activity *in vitro*. J. Biol. Chem. **270**:19158–19165.

6a. Bower, S. Unpublished data.

7. Bower, S., J. Perkins, R. R. Yocum, P. Serror, A. Sorokin, P. Rahaim, C. L. Howitt, N. Prasad, S. D. Ehrlich, and J. Pero. 1995. Cloning and characterization of the *Bacillus subtilis birA* gene encoding a repressor of the biotin operon. J. Bacteriol. **177**:2572–2575.

8. Cleary, P., and A. Campbell. 1972. Deletions and complementation analysis of the biotin gene cluster of *Escherichia coli*. J. Bacteriol. **112**:830–839.

9. Cronan, J. E., Jr. 1989. The *E. coli bio* operon: transcriptional repression by an essential protein modification enzyme. Cell **58**:427–429.

10. Dassa, E., and M. Hofnung. 1985. Sequence of gene *malG* in *E. coli* K12: homologies between integral membrane components from binding protein-dependent transport systems. EMBO J. **4**:2287–2293.

11. Donadio, S., and L. Katz. 1992. Organization of the enzymatic domains in the multifunctional polyketide synthase involved in erythromycin formation in *Saccharopolyspora erythraea*. Gene **111**:51–60.

12. Dower, W. J., J. F. Miller, and C. R. Ragsdale. 1988. High efficiency transformation of *E. coli* by high voltage electroporation. Nucleic Acids Res. **16**:6127–6145.

13. Dubnau, D., and R. Davidoff-Abelson. 1971. Fate of transforming DNA following uptake by competent *Bacillus subtilis*. J. Mol. Biol. **56**:209–221.

14. Eisenberg, M. A. 1987. Biosynthesis of biotin and lipoic acid, p. 544–550. In F. C. Neidhardt, J. L. Ingraham, B. Magasanik, K. B. Low, M. Schaechter, and H. E. Umbarger (ed.), *Escherichia coli* and *Salmonella typhimurium*: cellular and molecular biology. American Society for Microbiology, Washington, D.C.

15. Florentin, D., B. T. S. Bui, A. Marquet, T. Ohshiro, and Y. Izumi. 1994. On the mechanism of biotin synthetase of *Bacillus sphaericus*. C. R. Acad. Sci. Ser. III **317**:485–488.

16. Gibson, K. J., G. H. Lorimer, A. R. Rendina, W. S. Taylor, G. Cohen, A. A. Gatenby, W. G. Payne, D. C. Roe, B. A. Lockett, A. Nudelman, D. Marcovici, A. Nachum, B. A. Wexler, E. L. Marsilii, S. I. M. Turner, L. D. Howe, C. E. Kalbach, and H. Chi. 1995. Dethiobiotin synthetase: the carbonylation of 7,8-diaminononanoic acid proceeds regiospecifically via the N7-carbamate. Biochemistry **34**:10976–10984.

17. Gloeckler, R., I. Ohsawa, D. Speck, C. Ledoux, S. Bernard, M. Zinsius, D. Villeval, T. Kisou, K. Kamogawa, and Y. Lemoine. 1990. Cloning and characterization of the *Bacillus sphaericus* genes controlling the bioconversion of pimelate into dethiobiotin. Gene **87**:63–70.

18. Gomez, A., D. Ramon, and P. Sanz. 1993. GenBank accession no. L19164.

19. Hatfield, D., M. Hofnung, and M. Schwartz. 1969. Genetic analysis of the maltose A region in *Escherichia coli*. J. Bacteriol. **98**:559–567.

20. Haydock, S., J. A. Dowson, N. Dhillon, G. A. Roberts, J. Cortes, and P. F. Leadlay. 1991. Cloning and sequence analysis of genes involved in erythromycin biosynthesis in *Saccharopolyspora erythraea*: sequence similarities between EryG and a family of S-adenosylmethionine-dependent methyltransferases. Mol. Gen. Genet. **230**:120–128.

21. He, J.-S., R. T. Reuttinger, H.-M. Liu, and A. J. Fulco. 1989. Molecular cloning, coding nucleotides and the deduced amino acid sequence of *P-450BM-1* from *Bacillus megaterium*. Biochim. Biophys. Acta **109**:301–303.

22. Horinouchi, S., and B. Weisblum. 1982. Nucleotide sequence and functional map of pC194, a plasmid that specifies inducible chloramphenicol resistance. J. Bacteriol. **150**:815–825.

23. Huang, W., J. Jia, K. J. Gibson, W. S. Taylor, A. R. Rendina, G. Schneider, and Y. Lindqvist. 1995. Mechanism of an ATP-dependent carboxylase, dethiobiotin synthetase, based on crystallographic studies of complexes with substrates and a reaction intermediate. Biochemistry **34**:10985–10995.

24. Hueck, C. J., and W. Hillen. 1995. Catabolite repression in *Bacillus subtilis*: a global regulatory mechanism for the gram-positive bacteria? Mol. Microbiol. **15**:395–401.

25. Ifuku, O., J. Kishimoto, S. Haze, M. Yanagi, and S. Fukushima. 1992. Conversion of dethiobiotin to biotin in cell-free extracts of *Escherichia coli*. Biosci. Biotechnol. Biochem. **56**:1780–1785.

26. Ifuku, O., N. Koga, S. Haze, J. Kishimoto, and Y. Wachi. 1994. Flavodoxin is required for conversion of dethiobiotin to biotin in *Escherichia coli*. Eur. J. Biochem. **224**:173–178.

27. Ifuku, O., H. Miyaoka, N. Koga, J. Kishimoto, S. Haze, Y. Wachi, and M. Kajiwara. 1994. Origin of carbon atoms of biotin: **13**C-NMR studies on biotin biosynthesis in *Escherichia coli*. Eur. J. Biochem. **220**:585–591.

28. Inoue, H., H. Nojima, and H. Okayama. 1990. High efficiency transformation

of *Escherichia coli* with plasmids. Gene **96**:23–28.

29. Izumi, Y., Y. Kano, K. Inagaki, N. Kawase, Y. Tani, and H. Yamada. 1981. Characterization of biotin biosynthetic enzymes of *Bacillus sphaericus*: a dethiobiotin producing bacterium. Agric. Biol. Chem. **45**:1983–1989.

30. Jiang, X. M., B. Neal, F. Santiago, S. J. Lee, L. K. Romana, and P. R. Reeves. 1991. Structure and sequence of the *rfb* (O antigen) gene cluster of *Salmonella* serovar typhimurium (strain LT2). Mol. Microbiol. **5**:695–713.

31. Lauer, G., E. A. Rudd, D. L. McKay, A. Ally, D. Ally, and K. C. Backman. 1991. Cloning, nucleotide sequence, and engineered expression of *Thermus thermophilus* DNA ligase, a homolog of *Escherichia coli* DNA ligase. J. Bacteriol. **173**:5047–5053.

32. Lemoine, Y., A. Wach, and J. M. Jeltsch. 1996. To be free or not: the fate of pimelate in *Bacillus sphaericus* and in *Escherichia coli*. Mol. Microbiol. **19**:645–647.

33. Lerner, C. G., and M. Inouye. 1990. Low copy number plasmids for regulated low-level expression of cloned genes in *Escherichia coli* with blue/white insert screening capability. Nucleic Acids Res. **18**:4631.

34. Lopilato, J., S. Bortner, and J. Beckwith. 1986. Mutations in a new chromosomal gene of *Escherichia coli* K-12, *pcnB*, reduce plasmid copy number of pBR322 and its derivatives. Mol. Gen. Genet. **205**:285–290.

35. Miller, J. H. 1972. Experiments in molecular genetics, p. 352–355. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

36. Moran, C. P., Jr., N. Lang, S. F. J. LeGrice, G. Lee, M. Stephens, A. L. Sonenshein, J. Pero, and R. Losick. 1982. Nucleotide sequences that signal the initiation of transcription and translation in *Bacillus subtilis*. Mol. Gen. Genet. **186**:339–346.

37. Ohshiro, T., M. Yamamoto, Y. Izumi, B. T. S. Bui, D. Florentin, and A. Marquet. 1994. Enzymatic conversion of dethiobiotin to biotin in cell-free extracts of a *Bacillus sphaericus bioB* transformant. Biosci. Biotechnol. Biochem. **58**:1738–1741.

38. Otsuka, A. J., M. R. Buoncristiani, P. K. Howard, J. Flamm, C. Johnson, R. Yamamoto, K. Uchida, C. Cook, J. Ruppert, and J. Matsuzaki. 1988. The *Escherichia coli* biotin biosynthetic enzyme sequences predicted from the nucleotide sequence of the *bio* operon. J. Biol. Chem. **263**:19577–19585.

39. Overduin, P., W. Boos, and J. Tommassen. 1988. Nucleotide sequence of the *ugp* genes of *Escherichia coli* K12: homology to the maltose system. Mol. Microbiol. **2**:767–775.

40. Pai, C. H. 1975. Genetics of biotin biosynthesis in *Bacillus subtilis*. J. Bacteriol. **121**:1–8.

41. Perkins, J. B., S. Bower, C. L. Howitt, R. R. Yocum, and J. Pero. Unpublished data.

42. Ploux, O., and A. Marquet. 1992. The 8-amino-7-oxoate pelargonate synthase from *Bacillus sphaericus*. Purification and preliminary characterization of the cloned enzyme overproduced in *Escherichia coli*. Biochem. J. **283**:327–321.

43. Ploux, O., P. Soularue, A. Marquet, R. Gloeckler, and Y. Lemoine. 1992. Investigations of the first step of biotin biosynthesis in *Bacillus sphaericus*. Purification and characterization of the pimeloyl-CoA synthetase, and uptake of pimelate. Biochem. J. **287**:685–690.

44. Reed, K. E., and J. J. E. Cronan. 1993. Lipoic acid metabolism in *Escherichia coli*: sequencing and functional characterization of the *lipA* and *lipB* genes. J. Bacteriol. **175**:1325–1336.

45. Rolfes, R. J., and H. Zalkin. 1988. *Escherichia coli* gene *purR* encoding a repressor protein for purine nucleotide synthesis: cloning, nucleotide sequence, and interaction with the *purF* operator. J. Mol. Biol. **263**:19653–19661.

46. Sanyal, I., G. Cohen, and D. H. Flint. 1995. Biotin synthase: purification, characterization as a [2Fe-2S] cluster protein, and *in vitro* activity of the *Escherichia coli bioB* gene product. Biochemistry **33**:3625–3631.

47. Sanyal, I., K. Gibson, and D. Flint. 1996. *Escherichia coli* biotin synthase: an investigation into the factors required for its activity and its sulfur donor. Arch. Biochem. Biophys. **326**:48–56.

48. Sanyal, I., S.-L. Lee, and D. Flint. 1994. Biosynthesis of pimeloyl-CoA, a biotin precursor in *Escherichia coli*, follows a modified fatty acid synthesis pathway: **13**C-labeling studies. J. Am. Chem. Soc. **116**:2637–2638.

49. Sloma, A., A. Ally, D. Ally, and J. Pero. 1988. Gene encoding a minor extracellular protease in *Bacillus subtilis*. J. Bacteriol. **170**:5556–5563.

50. Sloma, A., G. A. Rufo, Jr., K. A. Theriault, M. Dwyer, S. W. Wilson, and J. Pero. 1991. Cloning and characterization of the gene for an additional extracellular serine protease of *Bacillus subtilis*. J. Bacteriol. **173**:6889–6895.

51. Speck, D., I. Ohsawa, R. Gloeckler, M. Zinsius, S. Bernard, C. Ledoux, T. Kisou, K. Kamogawa, and Y. Lemoine. 1991. Isolation of *Bacillus sphaericus* biotin synthesis control mutants: evidence for transcriptional regulation of the *bio* genes. Gene **108**:39–45.

52. Spizizen, J. 1958. Transformation of biochemically deficient strains of *Bacillus subtilis* by deoxyribonucleate. Proc. Natl. Acad. Sci. USA **44**:1072–1078.

53. Stassi, D., S. Donadio, M. J. Staver, and L. Katz. 1993. Identification of a *Saccharopolyspora erythraea* gene required for the final hydroxylation step in erythromycin biosynthesis. J. Bacteriol. **175**:182–189.

54. Stokes, H. W., and B. G. Hall. 1985. Sequence of the *ebgR* gene of *Escherichia coli*: evidence that the EBG and LAC operons are descended from a common ancestor. Mol. Biol. Evol. **2**:478–483.

55. Taira, K., J. Hirose, S. Hayashida, and K. Furukawa. 1992. Analysis of *bph* operon from the polychlorinated biphenyl-degrading strain of *Pseudomonas*

pseudoalcaligenes KF707. J. Biol. Chem. 267:4844–4853.

56. Tinoco, I., Jr., P. N. Borer, B. Dengler, M. D. Levine, O. C. Uhlenbeck, D. M. Crothers, and J. Gralia. 1973. Improved estimation of secondary structure in ribonucleic acids. Nature (London) New Biol. 246:40–41.

57. Vieira, J., and J. Messing. 1982. The pUC plasmids, an M13mp7-derived system for insertion mutagenesis and sequencing with synthetic universal primers. Gene 19:259–268.

58. Weickert, M. J., and G. H. Chambliss. 1990. Site-directed mutagenesis of a catabolite repression operator sequence in *Bacillus subtilis*. Proc. Natl. Acad. Sci. USA 87:6238–6242.

59. Wen, L.-P., and A. Fulco. 1987. Cloning of the gene encoding a catalytically self-sufficient cytochrome P-450 fatty acid monooxygenase induced by barbiturates in *Bacillus megaterium*. J. Biol. Chem. 262:6676–6682.

60. Wierenga, R. K., P. Terpstra, and W. G. J. Hol. 1986. Prediction of the occurrence of the ADP-binding βαβ-fold in proteins, using an amino acid sequence fingerprint. J. Mol. Biol. 187:101–107.

61. Yanisch-Perron, C., J. Vieira, and J. Messing. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119.

62. Youngman, P. J., J. B. Perkins, and R. Losick. 1984. Construction of a cloning site near one end of Tn917 into which foreign DNA may be inserted without affecting transposition in *Bacillus subtilis* or expression of the transposon-borne *erm* gene. Plasmid 12:1–9.

63. Zuber, P., and R. Losick. 1987. Role of AbrB in Spo0A- and Spo0B-dependent utilization of a sporulation promoter in *Bacillus subtilis*. J. Bacteriol. 169:2223–2230.
