
CARBOHYDRATE  
RESEARCH  

Carbohydrate Research 287 (1996) 59–76  

---

### Structure of amylovoran, the capsular exopolysaccharide from the fire blight pathogen *Erwinia amylovora*

**Manfred Nimtz** ${}^{a,*}$, **Andrew Mort** ${}^{b}$, **Tobias Domke** ${}^{a}$, **Victor Wray** ${}^{a}$, **Yongxiang Zhang** ${}^{c}$, **Feng Qiu** ${}^{b,1}$, **David Coplin** ${}^{d}$, **Klaus Geider** ${}^{c}$

${}^{a}$ GBF, Gesellschaft für Biotechnologische Forschung mbH, Mascheroder Weg 1, D-38124 Braunschweig, Germany  
${}^{b}$ Oklahoma State University, 246 Noble Research Center, Stillwater, OK 74074, USA  
${}^{c}$ Max-Planck-Institut für medizinische Forschung, Jahnstrasse 29, D-69120 Heidelberg, Germany  
${}^{d}$ Department of Plant Pathology, Ohio State University, Columbus, OH 43210, USA  

Received 7 December 1995; accepted 1 March 1996  

---

#### Abstract

The acidic exopolysaccharide (EPS) of *Erwinia amylovora*, amylovoran, was purified from culture supernatants of bacteria in minimal medium and cleaved chemically either by treatment with trifluoroacetic acid or hydrofluoric acid, and enzymatically by digestion with depolymerase from *E. amylovora* phage φ-Ealh. Structural characterization of the resulting oligosaccharides was performed by a combination of mass spectrometric and NMR [one- and two-dimensional (1D and 2D)] spectroscopic techniques. A branched repeating unit with five monosaccharide residues and various substituents was determined:

---

${}^{*}$ Corresponding author.  
${}^{1}$ Present address: Department of Chemistry, College of Staten Island, CUNY, 2800 Victory Blvd., NY 10301, USA.

0008-6215/96/$15.00 © 1996 Elsevier Science Ltd. All rights reserved  
PII S0008-6215(96)00070-5

α-d-Pyr4,6,Ac<sub>1,3</sub>Gal p
↓
4
β-d-Glc pA
↓
4
[ → 3)-α-d-Gal p-(1 → 6)-β-d-Gal p-(1 → 3)-β-d-Gal p-(1 → ]<sub>n</sub>
↑
6
1
{ β-d-Glc p}<sub>0,1</sub>

The terminal monosaccharide of the side branch, which bears a 4,6-bound pyruvate residue in the R-configuration, was found to be modified with 2-linked (26%), 3-linked (24%), 2-,3-linked (40%) O-acetyl groups, or these were absent (10%). An additional glucose residue is linked to approximately 10% of the core α-galactose of the repeating unit. © 1996 Elsevier Science Ltd.

Keywords: Exopolysaccharide, structure; Erwinia amylovora; Fire blight

---

### 1. Introduction

Like many plant-pathogenic bacteria, *Erwinia amylovora*, the causative agent of fire blight on apple and pear trees, produces large amounts of exopolysaccharide (EPS). Mutant strains without synthesis of the acidic capsular EPS, called amylovoran, are non-pathogenic [1–4]. The EPS capsules apparently protect the pathogen against plant defense reactions, and bind water and nutrients released from damaged plant cells [5]. Biosynthesis of amylovoran is encoded by the large *ams*-region, where 12 genes are transcribed as an operon [4]. Its expression is continued into the stationary growth phase of *E. amylovora* [6] which reflects the importance of amylovoran synthesis in the bacterial life cycle. The homopolymer levan is synthesized by the constitutively expressed enzyme levansucrase [7], when sucrose is present in the medium [8].

Heterologous polymeric EPS species are often synthesized as repeating units, which are transported through the bacterial membranes and polymerized on the cell surface. Amylovoran consists of one glucuronic acid and four galactose residues in the repeating unit, and a preliminary structure for their linkage has been proposed [9]. This structure is revised in the work presented here, and compared to other EPS structures.

---

### 2. Experimental

Strains, plasmids and isolation of amylovoran.—*E. amylovora* strain Eal/79 has been described in ref. [10]. Plasmid pJH94 with the depolymerase gene of phage φ-Ealh was generously provided by Dr. J. Hartung and has been described elsewhere [11]. Amylovoran was purified from suspension cells of strain Eal/79Sm (Eal/79 with spontaneous resistance to streptomycin) in minimal medium MM2 [6]. After two days of shaking at 28 °C, the bacteria were removed by centrifugation (Sorvall rotor GSA, 20 min at 13,000 rpm), and the supernatant was concentrated in a Millipore filter apparatus.
The concentrated solution was centrifuged in a preparative ultracentrifuge (Beckman rotor 60Ti, 4 h at 40,000 rpm), and the supernatant was extensively dialyzed against water. The dialysate was lyophilized and the EPS stored at room temperature.

Chemical cleavage of amylovoran and isolation of the resulting tetrasaccharide.—The native polysaccharide (20–70 mg) was degraded in anhydrous liquid HF (10 mL) at –40 °C, using a specially constructed apparatus designed to allow strict control of the reaction temperature and to retain the volatile HF [12,13]. The reaction was terminated by immersing the reaction vessel in liquid N₂ to freeze the HF. Anhydrous ether cooled with liquid N₂ was then added and the vessel was allowed, with occasional swirling, to warm to room temperature. The oligosaccharides were recovered from the mixture by retention on a teflon filter, or the entire mixture was evaporated. In either case, the sample was dissolved in 50 mM NH₄OAc buffer, pH 5.2, and then fractionated on a 500′ 25 mm stainless steel column packed with Toyopearl HW-40S gel filtration medium. Two major and several minor peaks were observed. The first major peak eluted in the region expected for a tetrasaccharide, and the second one eluted close to the included volume. When the oligosaccharides were isolated from the HF reaction mixture by filtration, the second major peak was smaller.

Enzymatic depolymerization of amylovoran and isolation of penta- and deca-saccharides.—The depolymerase was partially purified by a method described previously [11] for strain DH5α(pJH94). The enzymatic degradation of amylovoran was performed by mixing 200 μL EPS (5 mg/mL), 10 μL depolymerase extract (1 mg protein/mL) and 90 μL 0.2 M NaOAc–HOAc buffer, pH 5.0. The mixture was incubated for 3 days at 37 °C. HPLC separation was performed on a column (1 × 100 cm), packed with beads of Toyopearl HW-40S, and eluted with 0.1 M HOAc at a flow rate of 0.5 mL/min. The eluate was monitored with a refractive index (RI)-detector. Fractions at the expected position of the monomers and dimers of the repeating unit were collected and dried in a Speed Vac centrifuge.

Compositional analysis.—Monosaccharides were analyzed [14] as the corresponding methyl glycosides after methanolysis and trimethylsilylation on a Carlo Erba Mega Series gas chromatograph equipped with a 30 m DB1 capillary column. The absolute configuration of the monosaccharides was determined by GLC analysis of the trimethylsilylated (+)-2-butyl glycosides [15] on the same column.

Analysis by GLC–MS.—For methylation analysis the oligosaccharides were reduced with NaBD₄ (for the intact polysaccharide, this first reduction step was omitted) and permethylated according to ref. [16]. In order to detect hexuronic acids, the permethylated sample was reduced a second time, with NaBD₄ leading to the conversion of glucuronic acids to the respective glucose derivatives with two deuterium atoms incorporated at C-6. Purification of the permethylated sample, hydrolysis, reduction, and peracetylation were performed as described [17]. The oligosaccharides resulting from a partial acid hydrolysis (each of 100 μg amylovoran; 2 h, 100 °C; 0.1, 1, or 4 M trifluoroacetic acid) were reduced and permethylated as described above and analyzed directly by GLC–MS. Analyses were performed on a Kratos MS 50 fast-scan mass spectrometer connected to a Carlo Erba Mega Series gas chromatograph equipped with a 30 m DB1 capillary column.

600-MHz ¹H NMR spectroscopy.—Prior to ¹H NMR spectroscopic analysis, the
samples were repeatedly exchanged in D₂O (Fluka, > 99.95 atom% D) at pD 7 and ambient temperature. \({}^{1}\)H NMR spectra at 600 MHz were recorded at 300 K on a Bruker AVANCE DMX 600 NMR spectrometer incorporating a gradient unit. A 1.3 s presaturation pulse was employed prior to the experimental pulse sequence in order to suppress the signal of the residual HOD peak. Chemical shifts are expressed in ppm downfield from the signal for internal sodium 4,4-dimethyl-4-silapentane-1-sulfonate, but were actually measured by reference to free acetate (\(\delta\) 1.908 in D₂O at pD 6–8 and 27 °C), with an accuracy of 0.002 ppm. \({}^{1}\)H 2D correlated spectroscopy (COSY) [18], total correlated spectroscopy (TOCSY) [19], and rotating frame Overhauser effect spectroscopy (ROESY) [20] were performed on the same instrument, with mixing times of 80 and 500 ms, respectively, for the two latter techniques. All 1D and 2D spectra were recorded using modified Bruker software. The heteronuclear multiple quantum coherence (HMQC) [21] and heteronuclear multiple-bond coherence (HMBC) [22] experiments were performed on a Varian Unity Plus 600 MHz spectrometer using standard software.

Electrospray ionization tandem mass spectrometry (ESI-MSMS).—A Finnigan MAT TSQ 700 triple quadrupole mass spectrometer equipped with a Finnigan electrospray ion source (Finnigan MAT corp., San Jose, CA) was used for ESI-MS. The reduced and permethylated samples were dissolved in acetonitrile, saturated with NaCl (concentrations approximately 10 pmol/µL), and injected at a flow rate of 1 µL/min into the electrospray chamber. In the positive-ion mode, a voltage of +5.5 kV was applied to the electrospray needle. For collision-induced dissociation (CID) experiments, parent ions were selectively transmitted by the first mass analyzer and directed into the collision cell (Ar was used as collision gas) with a kinetic energy set around –60 eV. Native oligosaccharides were dissolved in MeOH containing 0.5% NH₃ and subjected to negative-ion mode ESI-MS. For the detection of negative ions, all voltages were reversed, i.e. –4.5 kV was applied to the needle and the collision energy was set at approximately +40 eV for MSMS experiments.

---

3. Results

Purification and properties of amylovoran released from suspension cells.—Amylovoran is loosely attached to the cell surface of *E. amylovora* as a capsule and is released into the environment especially by mechanical stress [6,23]. From supernatants of suspension cultures, the EPS was concentrated by pressure dialysis on a nitrocellulose membrane. The average molecular mass of purified amylovoran was recently determined to be 1 MDa. Taking into account the present study, this is equivalent to about 1000 repeating units per EPS molecule [24]. Contaminations of lipopolysaccharide were largely removed by high-speed centrifugation. Preparations of amylovoran, however, always contained another polymer (up to 20%) of lower molecular mass, when analyzed by HPLC gel filtration.

Carbohydrate compositional analysis.—The high-molecular-mass EPS was analyzed for its sugar composition and found to contain galactose and glucuronic acid in a molar ratio of about 4:1. The additional detection of various amounts of glucose in different

Table 1  
Methylation analyses data of the polysaccharide and the reduced oligosaccharide fragments produced by chemical or enzymatic cleavage. Uncorrected molar ratios of the derivatives are given

| Peracetylated derivative of | Substituted Amylovoran ${ }^{a}$ | Tetrasaccharide ${}^{a}$ | Pentasaccharide ${}^{a}$ | Decasaccharide ${}^{a}$ |
|-----------------------------|----------------------------------|--------------------------|--------------------------|--------------------------|
| in position                 |                                  |                          |                          |                          |
|                             | 11                              | 12                       | 13                       |                          |
| **Galactitol**              |                                  |                          |                          |                          |
| 1,2,4,5,6-Penta-O-methyl-   | 3 ${}^{b}$                      | -                        | 0.6                      | 0.6                      |
| 2,3,4,6-Tetra-O-methyl-     | term.                           | -                        | 1.0                      | -                        |
| 1,2,5,6-Tetra-O-methyl-     | 3:4 ${}^{b}$                    | -                        | 0.9                      | -                        |
| 2,4,6-Tri-O-methyl-          | 3                               | 0.6                      | 1.0                      | 0.7                      |
| 1,2,5-Tri-O-methyl-          | 3;4:6 ${}^{b}$                  | -                        | < 0.1                    | -                        |
| 2,3,6-Tri-O-methyl-          | 4                               | -                        | -                        | 0.5                      |
| 2,3,4-Tri-O-methyl-          | 6                               | 1.0                      | -                        | 2.0                      |
| 2,6-Di-O-methyl-             | 3;4                             | 0.4                      | -                        | 0.3                      |
| 2,3-Di-O-methyl-             | 4;6                             | 0.5                      | -                        | 1.3                      |
| 2-Mono-O-methyl-             | 3;4:6                           | < 0.1                    | -                        | < 0.1                    |
| **Glucitol**                |                                  |                          |                          |                          |
| 2,3,4,6-Tetra-O-methyl-      | term.                           | 0.2                      | 0.1                      | 0.2                      |
| 2,3,4-Tri-O-methyl-(6-d$_{2}$)- | term.                           | -                        | 0.2                      | -                        |
| 2,3-Di-O-methyl-(6-d$_{2}$)- | 4                               | 0.5                      | -                        | 0.8                      |

${}^{a}$ In these samples additionally some 3,4,6-tri-O-methyl- and 3,4-di-O-methyl-glucitol (substituted at positions 2 and 2;6) were detected, which probably originate from a contaminating glucan.

${}^{b}$ Derivatives of reduced monosaccharide residues.

${}^{c}$ These derivatives were obtained by an additional NaBD$_{4}$ reduction of the permethylated sample.

preparations could be mainly explained by the presence of a low-molecular-mass contaminant in the EPS preparations, consisting of glucose, possibly a glucan-like compound. The determination of the absolute configuration of all monosaccharide constituents was performed by GLC analysis of the respective trimethylsilylated (+)-2-butyl glycosides, and D-enantiomers were found exclusively. IR investigation of EPS preparations from E. amylovora showed the presence of acetyl groups, which could be removed by treatment with alkali.

Linkage analysis.—Methylation analysis of the native EPS (Table 1) showed the presence of five major partially methylated alditol acetates representing the presence of 3-, 6-, 4,6-di- and 3,4-di-substituted galactose residues, while an additional 3,4,6-trisubstituted derivative of galactose was observed in trace amounts. A 4-substituted gluconic acid residue was detected as 1,4,5,6-tetra-O-acetyl-2,3-di-O-methyl-glucitol-6-$d_{2}$ after additional NaBD$_{4}$ reduction of the permethylated polysaccharide. The presence of galactose and glucuronic acid residues in their furanose forms could be excluded from the NMR and MS analyses of polysaccharide fragments (see below). Whereas small amounts of terminal glucose were detected in all amylovoran preparations, 2-substituted and 2,6-disubstituted glucose were found in varying amounts depending on the preparation analyzed. This confirms the presence of an impurity and suggests a contamination by a glucan, probably with a similar structure to the gluco-oligosaccharide produced exclusively by E. amylovora under special fermentation conditions [25].

Analysis of derivatized hydrolytic fragments by GLC–MS.—In order to obtain
sequence information of the monosaccharide constituents described above, the native polysaccharide was partially hydrolyzed with trifluoroacetic acid at three different concentrations. Each of the resulting oligosaccharide mixtures was reduced with NaBD₄. After permethylation, these derivatized oligosaccharide-alditols-1-d were directly analyzed by GLC–MS. Detailed information regarding the sequence, linkage, and branching pattern of the intact polysaccharide was obtained from μg amounts of rather impure material as described below: the retention time on the GLC column and high-mass fragments of the EI mass spectrum indicate the number of monosaccharide residues in each hydrolytic fragment, whereas the EI mass spectra of the individual oligosaccharides allow their partial structural characterization. Diagnostic ions characteristic of the presence of terminal hexose or hexuronic acid residues are, e.g., the fragment-ion series at *m/z* 219, 187 (−MeOH), and 155 (−2MeOH), or 233 and 201, respectively. The signals at *m/z* 236 and 172 (−2MeOH) unequivocally show the presence of a monosubstituted deuterium-labelled reduced hexose unit, whereas the fragments generated by its unsubstituted, i.e. methoxylated carbon atoms \[*m/z* 45 (46, deuteriumlabelled), 89 (90), 133 (134), etc.\] indicate the substitution position. Data on ten different reduced and permethylated oligosaccharides obtained from native amylovoran by partial hydrolysis are summarized below (compare also Table 2 and Fig. 2).

In the region of permethylated monosaccharide-alditols, in addition to galactitol and some glucitol, a 4,6-pyruvated hexitol derivative eluted, which was identified by its characteristic fragmentation pattern (compound 1, Table 2). This is compatible with the detection of 4,6-disubstituted galactose in the methylation analysis, suggesting a terminal pyruvate-capped galactose unit as a structural motif of amylovoran (residue A of Fig. 1).

Three disaccharide-alditols could be identified by GLC–MS. The compound eluting first was shown to incorporate a 3-substituted hexitol moiety linked to a hexose residue confirming the detection of 3-substituted galactose by methylation analysis. This compound (2) could arise from residues D–E of Fig. 1. The second disaccharide-alditol (3, Table 2) incorporates a terminal HexA residue linked to a 4-substituted hexitol moiety. Since methylation analysis of native amylovoran excludes a 4-substituted hexose

$\begin{aligned} & \text { A } \\ & \alpha-\mathrm{d}-\operatorname{Pyr} 4,6, \mathrm{Ac}_{1,3} \mathrm{Gal} p \\ & \downarrow \\ & 4 \\ & \beta-\mathrm{d}-\mathrm{Glc} p \mathrm{~A} \quad \mathrm{B} \\ & \downarrow \\ & 4 \quad \mathrm{C} \quad \mathrm{D} \quad \mathrm{E} \\ & {[}-3)-\alpha-\mathrm{d}-\mathrm{Gal} p-(1 \rightarrow 6)-\beta-\mathrm{d}-\mathrm{Gal} p-(1 \rightarrow 3)-\beta-\mathrm{d}-\mathrm{Gal} p-(1 \rightarrow]_{n} \\ & \uparrow \\ & 6 \\ & \uparrow \\ & 1 \\ & \{\beta-\mathrm{d}-\mathrm{Glc} p\}_{0.1} \\ & \mathrm{F} \end{aligned}$

Fig. 1. Structure of the repeating unit of amylovoran. Residue A is not O-acetylated (10%), 2-mono-O-acetylated (26%), 3-mono-O-acetylated (24%), or 2,3-di-O-acetylated (40%); residue F is present in approximately 10% of the repeating units.
Table 2  
EI mass spectral data of the reduced (NaBD₄) and permethylated oligosaccharides obtained by partial hydrolysis or HF-treatment from amylovoran (diagnostic fragments have been set italic, relative abundance in parentheses). The exact structure of all compounds is given. The identity of the individual monosaccharide units and the anomeric configuration of the linkages is inferred from methylation analyses and NMR data.

| Compound | $m/z$ |
|----------|-------|
| 1 (Pyr4,6-Gal-ol) | 43 (100), 45 (95), 46 (53), 59 (45), 75 (68), 90 (55), 99 (37), 101 (85), 117 (85), 131 (19), 134 (17), 143 (24), 264 (43), 277 (3), 291 (<1), 323 ($M^+$) n.d. |
| 2 (Gal p-$\beta$(1 $\rightarrow$ 3)-Gal-ol) | 88 (82), 90 (27), 90 (17), 101 (100), 111 (48), 172 (15), 187 (45), 219 (12), 236 (30), 296 (3), 350 (2), 382 (1), 471 ($M^+$) n.d. |
| 3 (Glc pA-$\beta$(1 $\rightarrow$ 4)-Gal-ol) | 45 (73), 46 (53), 59 (47), 75 (72), 88 (73), 89 (67), 90 (38), 101 (89), 134 (38), 141 (43), 172 (37), 201 (100), 233 (55), 236 (37), 245 (3), 296 (3), 363 (3), 395 (2), 408 (1), 420 (1). |
| 4 (Gal p-$\alpha$(1 $\rightarrow$ 6)-Gal-ol) | 435 (1), 495 ($M^+$) n.d. |
| 5 (Glc pA-$\beta$(1 $\rightarrow$ 4)[Gal p-$\beta$(1 $\rightarrow$ 3)]-Gal-ol)${}^{a}$ | 88 (85), 89 (30), 90 (40), 101 (100), 111 (28), 134 (20), 146 (35), 172 (15), 178 (12), 187 (42), 190 (6), 219 (11), 222 (2), 236 (52), 250 (3), 337 (3), 471 ($M^+$) n.d. |
| 6 (Glc pA-$\beta$(1 $\rightarrow$ 4)[Glc p-$\beta$(1 $\rightarrow$ 6)]-Gal-ol) | 88 (77), 89 (24), 90 (9), 101 (77), 111 (41), 172 (28), 187 (44), 201 (100), 219 (13), 233 (8), 250 (4), 306 (<1), 319 (<1), 440 (2), 454 (8), 500 (<1), 514 (<1), 643 (<1), 644 (<1), 689 ($M^+$) n.d. |
| 7 (Glc pA-$\beta$(1 $\rightarrow$ 4)-Gal p-$\alpha$(1 $\rightarrow$ 6)-Gal-ol)${}^{b}$ | 88 (87), 89 (23), 90 (12), 101 (92), 111 (35), 134 (8), 172 (28), 187 (24), 201 (100), 219 (6), 233 (8), 250 (2), 317 (2), 349 (2), 440 (1), 454 (10), 500 (<1), 689 ($M^+$) n.d. |
| 8 (Pyr4,6-Gal p-$\alpha$(1 $\rightarrow$ 4)-Gal-ol) | 88 (42), 90 (13), 101 (60), 116 (22), 133 (20), 134 (6), 146 (23), 172 (29), 178 (10), 190 (7), 201 (100), 222 (1), 233 (25), 236 (90), 250 (3), 291 (2), 319 (3), 376 (2), 405 (<1), 436 (<1), 500 (<1), 523 (<1), 555 (<1), 598 (<1), 612 (<1), 689 ($M^+$) n.d. |
| 9 (Glc pA-$\beta$(1 $\rightarrow$ 4)[Glc p-$\beta$(1 $\rightarrow$ 3)]-Gal p-$\alpha$(1 $\rightarrow$ 6)-Gal-ol) | 88 (55), 89 (43), 90 (25), 101 (100), 111 (46), 127 (78), 134 (21), 141 (91), 155 (33), 172 (51), 201 (36), 215 (17), 236 (84), 243 (26), 247 (80), 275 (29), 296 (10), 502 (10), 685 (10), 745 ($M^+$) n.d. |
| 10 (Glc pA-$\beta$(1 $\rightarrow$ 4)-Gal p-$\alpha$(1 $\rightarrow$ 6)-Gal-ol) | 88 (43), 90 (9), 101 (64), 111 (27), 133 (10), 134 (4), 146 (11), 172 (15), 178 (4), 187 (34), 190 (2), 201 (100), 219 (13), 222 (1), 233 (10), 236 (90), 319 (3), 376 (2), 405 (<1), 409 (<1), 454 (<1), 704 (<1), 718 (<1), 759 (<1), 893 ($M^+$) n.d. |
| 11 (Gal p-$\beta$(1 $\rightarrow$ 3)-Gal p-$\beta$(1 $\rightarrow$ 4)[Glc pA-$\beta$(1 $\rightarrow$ 3)]Glc pA-$\beta$(1 $\rightarrow$ 4)]-Gal-ol) | 88 (82), 89 (20), 90 (13), 101 (100), 133 (9), 172 (9), 201 (69), 233 (8), 236 (12), 296 (2), 394 (1), 436 (<1), 440 (3), 640 (<1), 893 ($M^+$) n.d. |
|  | 88 (61), 89 (25), 90 (9), 101 (95), 111 (63), 127 (30), 159 (37), 172 (28), 187 (88), 201 (100), 219 (32), 233 (10), 250 (8), 423 (7), 454 (13), 500 (<1), 566 (<1), 644 (2), 686 (<1), 704 (<1), 718 (<1), 848 (<1), 893 ($M^+$) n.d. |

${}^{a}$ See Fig. 2a.  
${}^{b}$ See Fig. 2b.

residue, this monosaccharide unit must be originally 3,4-disubstituted, and the described compound is generated by the cleavage of three glycosidic bonds: residues B–C of Fig. 1. The last eluting disaccharide-alditol (4, Table 2) was identified as a hexose linked to O-6 of a hexitol residue, corroborating the detection of 6-substituted galactose by methylation analysis, i.e. this disaccharide arises from residues C–D of Fig. 1.

Four compounds eluted in the trisaccharide-alditol region. The fragmentation pattern of the first (5, Table 2, Fig. 2a) clearly indicated a branched structure, since characteristic fragments of both terminal Hex and terminal HexA were observed, but no fragment indicating a monosubstituted alditol residue. Since the HexA unit was shown above to be linked to O-4 of a hexose (compound 3), the second substituent on this hexose residue should be bound to O-3 according to the methylation analysis data. This is indeed the case as was deduced from an in-depth analysis of the fragmentation pattern (compare Fig. 2a). Therefore, compound 5 represents residues B–(E)C of Fig. 1. The second trisaccharide-alditol (6) (see Table 2), present in trace amounts only, again has a branched structure with both terminal hexose and terminal HexA residues. Since the O-1,2,3 of its reduced monosaccharide unit were shown to be unsubstituted (detection of the respective fragment ion in its mass spectrum) and HexA is linked to O-4, a hexose residue must be bound to O-6. Hence, the detection of this compound substantiates the presence of small amounts of modified repeating units of amylovoran incorporating an additional Hex residue at O-6 of the core galactose (residues B–(F)C of Fig. 1). This is confirmed by the detection of trace amounts of 3,4,6-trisubstituted galactose in the methylation analysis. Compound 7, eluting next, is a linear structure terminating with HexA. Substitution at O-6 of the hexitol moiety was clearly indicated by the fragment-ion series generated by the methylated carbon units 1–5 of this residue. Hence, compound 7 represents residues B–C–D of Fig. 1 (see Fig. 2b). The final trisaccharide-alditol (8), present only in very small amounts, again is a linear structure terminating with a pyruvated hexose residue. The hexitol moiety was substituted at O-4 with a HexA residue and is compatible with fragment A–B–C of Fig. 1. The presence of this structure in rather small amounts and our failure to detect further compounds incorporating a pyruvated Hex residue (with the exception of the respective monosaccharide 1) can be explained by the high hydrolytic susceptibility of the glycosidic linkage between residues A and B of Fig. 1.

Two major peaks eluted in the tetrasaccharide-alditol region. The fragmentation pattern of the first (9) revealed the presence of terminal HexA, terminal Hex, and monosubstituted Hex-ol, suggesting a branching at the monosaccharide unit linked to the proximal hexitol residue. The presence of characteristic fragment ions showed the hexitol residue to be substituted at O-6. These data, and comparison with the trisaccharides discussed above, indicate that residues B–(E)C–D of Fig. 1 generated compound 9. The second peak (10) contained a linear structure, since only fragments characteristic of terminal HexA and monosubstituted Hex-ol were detected in the lower-mass range. Since substitution was at O-3 of the hexitol moiety, this tetrasaccharide constitutes residues B–C–D–E of Fig. 1.

It may be evident that the basic structure of amylovoran could already be deduced from these GLC–MS data. The identity of the individual hexose residues can be assigned in combination with the compositional and methylation analysis data; residue

M. Nimtz et al. / Carbohydrate Research 287 (1996) 59–76

Fig. 2. EI mass spectra and fragmentation schemes of the reduced (NaBD₄) and permethylated trisaccharides 5 (a) and 7 (b). See text for further explanation.

F, present in trace amounts only, must be a glucose, since no terminal galactose was detected by methylation analysis of native amylovoran. However, no information regarding the configuration of the individual glycosidic linkages and the presence of
labile substituents, e.g., $O$-acyl groups, which would not survive the derivatization procedure, could be obtained by this technique. Therefore, for a rigorous structural characterization of the polysaccharide, specific fragments were produced either by chemical means or enzymatically. After purification by HPLC, the structures of these oligosaccharides were elucidated using 1D and 2D NMR methods in addition to various MS techniques.

**Structural elucidation of a tetrasaccharide produced by HF-treatment of amylovoran.**

Treatment of amylovoran with liquid HF at $-40\,^{\circ}\mathrm{C}$, conditions which are expected to cleave specifically $\alpha$-linkages of hexoses but neither $\alpha$- nor $\beta$-linkages of uronic acids [26], led to the production of predominantly mono- and tetra-saccharides. ${}^{1}\mathrm{H}$ NMR spectroscopy indicated a slow conversion of the initially formed sugar fluorides into the corresponding free saccharides. The monosaccharides appeared to be a mixture of variously substituted galactose residues, containing a combination of pyruvic and acetyl substituents. Negative-ion mode ESI-MS of the purified tetrasaccharide **11** afforded a molecular ion at $m/z\ 679$, corresponding to $[\mathrm{Hex}_{3}\mathrm{HexA}-\mathrm{H}]^{-}$. Additionally, small amounts (about 10%) of two oligosaccharides plus ($m/z\ 841$) and minus ($m/z\ 517$) one hexose residue were detected. Methylation analysis (Table 1) of reduced **11** was compatible with the structural element D–E–(B)C of Fig. 1. This finding was confirmed by the EI mass spectrum (Table 2). In particular, the presence of both terminal Hex and HexA residues, and the additional detection of a terminal Hex–Hex fragment combined with the absence of a monosubstituted hexitol moiety support the branched arrangement of the monosaccharide residues.

The detection of small amounts of the 3,4,6-trisubstituted reduced galactose derivative in combination with equal amounts of terminal glucose in the methylation analysis strongly suggests, that the pentasaccharide present in small amounts incorporates the residues D–E–(F)(B)C of Fig. 1. GLC–MS analysis of the reduced and permethylated sample showed that the minor trisaccharide component was identical to **5**.

The structure of **11** was confirmed by NMR spectroscopy. Three pyranose monosaccharide spin systems, all with $\beta$-anomeric configurations, and the $\alpha$ and $\beta$ anomers of the reducing residue C, also a pyranose, were completely or partially assigned (up to H-5) using 2D COSY and TOCSY NMR measurements (Table 3). Interresidual cross-peaks in the 2D ROESY spectrum were then used to determine the linkages between the monosaccharide residues (Fig. 3). Thus, two cross-peaks on the H-1 track of residue B corresponding to H-4 of the $\alpha$ and $\beta$ anomer of residue C were observed, demonstrating unequivocally the $(1 \rightarrow 4)$-linkage between these residues. Due to signal overlap of C H-1$\beta$ and E H-1 (the presence of two anomic protons was unequivocally demonstrated from an inverse ${}^{1}\mathrm{H}$-detected one-bond ${}^{13}\mathrm{C}-{ }^{1}\mathrm{H}$ correlation, that showed a cross-peak at 97 ppm for C C-1$\beta$ and at 105 ppm for E C-1), only the $\beta(1 \rightarrow 3)$-linkage between residue E and the $\alpha$ anomer of residue C could be clearly demonstrated. The $\beta(1 \rightarrow 3)$-linkage connecting residues D and E was confirmed by a weak cross-peak between D H-1 and E H-4 and a stronger one between D H-1 and E H-3. ${}^{13}\mathrm{C}$ chemical shifts for all carbon atoms in the tetrasaccharide were determined from a HMQC spectrum in conjunction with the proton assignments described above. The HMBC spectrum showed the expected long-range ${}^{13}\mathrm{C}-{ }^{1}\mathrm{H}$ correlations for the proposed structure. H-1 (d, 4.90 ppm) of residue B showed correlations to C-4 of residue C($\alpha$) at 75.8
Table 3  
${}^{1}$H and ${}^{13}$C chemical shifts of the tetrasaccharide produced by HF-treatment (a) and the enzymatically produced pentasaccharide (b) from amylovoran

---

**a) Compound 11**  
$\beta$-D-Gal $p-(1 \rightarrow 3)-\beta$-D-Gal $p-(1 \rightarrow 3)-\alpha / \beta$-D-Gal $p$

**b) Compound 12**  
$\alpha$-D-Pyr4,6,Ac$_{1,3}$Gal $p-(1 \rightarrow 4)-\beta$-D-Glc $pA$  


        B
        |
        4
β-D-Glc pA
        |
        4
α-D-Gal p-(1 → 6)-β-D-Gal p-(1 → 3)-α / β-D-Gal p
    C       D           E


---

**a) Compound 11**  
**Residue C ($\alpha$)**  
- H-1: 5.282  
- H-2: 4.10  
- H-3: 4.08 (R: E H-1)  
- H-4: 4.505 (R: B H-1)  
- H-5: 3.28  
- H-6a: 3.51  
- H-6b: 3.53  
- C-1: 93.1  
- C-2: 68.4  
- C-3: 79.7  
- C-4: 75.8  
- C-5: 75.4  
- C-6: 60.65  

**Residue E**  
- H-1: 4.658  
- H-2: 3.79  
- H-3: 3.84 (R: D H-1)  
- H-4: 4.201 (R: D H-1)  
- H-5: 3.70  

**Residue C ($\beta$)**  
- H-1: 4.657  
- H-2: 3.75  
- H-3: 3.89  
- H-4: 4.447 (R: B H-1)  
- H-5: 3.72  
- C-1: 97.0  
- C-2: 71.9  
- C-3: 82.8  
- C-4: 75.0  
- C-5: 74.7  
- C-6: 60.56  

**Residue D**  
- H-1: 4.620  
- H-2: 3.61  
- H-3: 3.66  
- H-4: 3.91  
- H-5: 3.74  

**Residue B**  
- H-1: 4.903  
- H-2: 3.377  
- H-3: 3.56  
- H-4: 3.50  
- H-5: 3.70  
- C-1: 102.4  
- C-2: 74.1  
- C-3: 76.4  
- C-4: 72.7  
- C-5: 75.8  

---

**b) Compound 12**  
**Residue E ($\alpha$)**  
- H-1: 5.279  
- H-2: 3.99  
- H-3: 3.96 (R: D H-1)  
- H-4: 4.20  
- H-5: 3.92  

**Residue A²**  
- H-1: 5.555  
- H-2: 5.041  
- H-3: 4.127  
- H-4: 4.262  
- H-5: 3.85  

**Residue E ($\beta$)**  
- H-1: 4.614  
- H-2: 3.65  
- H-3: 3.72 (R: D H-1)  
- H-4: 4.16  

**Residue A³**  
- Ac: 2.176  
- Pyr: 1.440  
- H-1: 5.529  
- H-2: 4.12  
- H-3: 5.111  
- H-4: 4.338  
- H-5: 3.85  

**Residue D**  
- H-1: 4.628  
- H-2: 3.65  
- H-3: 3.68  
- H-4: 3.95  
- H-5: 3.89  

**Residue A⁴**  
- Ac: 2.195  
- Pyr: 1.459  
- H-1: 5.599  
- H-2: 5.248  
- H-3: 5.333  
- H-4: 4.406  
- H-5: 3.91  

**Residue C**  
- H-1: 4.984  
- H-2: 3.92  
- H-3: 3.83  
- H-4: 4.15 (R: B H-1)  
- H-5: 3.74  

**Residue B**  
- H-1: 4.634  
- H-2: 3.37  
- H-3: 3.67  

**Residue A¹**  
- H-4: 3.75 (R: A¹⁻⁴ H-1)  
- H-1: 5.478  
- H-2: 3.92  
- H-3: 3.92  
- H-4: 4.16  
- Pyr: 1.435  

---

**R: ROESY.**

* An additional weak signal of a $\beta$-glucosyl system probably corresponding to residue F of Fig. 1 was observed: H-1, 4.471; H-2, 3.26; H-3, 3.51; H-4, 3.40.
* The ratio of the differently O-acetylated residues was determined by peak integration of the anomeric proton signals and the pyruvate signals in the 1D ${}^{1}$H NMR spectrum.
* These signal assignments may have to be interchanged.

CβH-1  
EH-1 DH-1  
BH-1  
CaH-1  

ROESY  

DH-1/EH-3  

EH-1/CaH-3  

DH-1/EH-4  

BH-1/CβH-4  
BH-1/CaH-4  

ppm 5.0     4.5     4.0     3.5  
TOCSY ppm  

Fig. 3. Part of the 600-MHz ¹H 2D ROESY (upper) and TOCSY (lower) spectra of the tetrasaccharide 11 produced by HF-treatment of amylovoran. The spin systems belonging to the various units are indicated in the TOCSY spectrum and the various linkage cross-peaks are indicated in the ROESY spectrum. The cross-peak corresponding to the linkage E H-1/Cβ H-3 cannot be observed because of the overlap of Cβ H-1 and E H-1.

and C(β) at 75.0 ppm. H-1 (d, 4.66 ppm) of residue E showed correlations to C-3 of residue C(α) at 79.7 and C(β) at 82.8 ppm. H-1 (d, 4.62 ppm) of residue D showed a correlation to C-3 of residue E at 82.8 ppm.

Structural elucidation of the pentasaccharide produced by enzymatic depolymerization of amylovoran.—Extensive cleavage of purified amylovoran with phage φ-Ealh-depolymerase and size fractionation of the degradation products gave two major oligosaccharides with a monosaccharide composition of glucuronic acid and galactose expected for amylovoran. Judged from calibration of the gel permeation column with oligomeric dextrans, the degradation products eluted in the size range of decasaccharides and pentasaccharides, respectively. When subjected to negative-ion mode ESI-MS, the pentasaccharide fraction (compound 12) yielded molecular ions at *m/z* 911 (10% rel abundance), 953 (45%), and 995 (45%), corresponding to [PyrHexHex₃HexA – H]⁻ with 0, 1, or 2 additional acetyl groups. Weaker signals were observed at *m/z* 1115 and 1157 compatible with structures enlarged by a further Hex residue. The most intense molecular ions at *m/z* 953 and 995 were subjected to negative-ion mode ESI-MSMS experiments. The respective daughter-ion spectra both showed an intense fragment ion generated by the loss of one Hex residue and had a base-peak at *m/z* 291 or 333, respectively, corresponding to a pyruvated mono- or di-*O*-acetylated Hex residue, unequivocally demonstrating the exclusive *O*-acetylation of the pyruvated galactose residue of amylovoran.

Positive-ion mode ESI-MS of the reduced and permethylated sample yielded an intense molecular ion at *m/z* 1176 [PyrHexHex₂HexAHex-ol + Na]⁺ corresponding to the Na-adduct of the pyruvated repeating pentasaccharide of amylovoran (the various acetyl groups do not survive the permethylation procedure). A weaker signal (ca. 10% rel abundance) at *m/z* 1380 [PyrHexHex₃HexAHex-ol + Na]⁺ is compatible with the presence of small amounts of modified repeating units bearing an additional Glc residue at O-6 of the core galactose C. MSMS of the parent ion at *m/z* 1176 clearly showed a linear arrangement of the molecule and the sequence PyrHex–HexA–Hex–Hex–Hex-ol (Fig. 4a). Most of the fragment ions are generated by cleavage of the glycosidic bonds (*Y*- or *B*-type according to the nomenclature of Domon et al. [27]). One relatively intense fragment at *m/z* 807, however, is generated by fragmentation of the sugar ring of the 4th monosaccharide residue from the non-reducing end between C-3/C-4 and C-5/O-5 (³,⁵A₄) excluding a (1 → 2)- or (1 → 3)-linkage between the 3rd and 4th monosaccharide unit. This is compatible with the sequence A–B–C–D–E of Fig. 1 and the α(1 → 6)-linkage between residues C and D. The daughter-ion spectrum of the hexasaccharide present in smaller amounts is depicted in Fig. 4b. The fragmentation pattern, i.e., the presence of both fragments [M – PyrHex] as well as [M – Hex] clearly indicates a branched structure. Since the fragments incorporating the deuterium-marked reduced proximal Hex residue are identical up to the second Hex residue and then show a characteristic mass shift of 204 Da, another Hex residue must be linked to the third Hex residue of the molecule indicating a sequence PyrHex–HexA–(Hex)Hex–Hex–Hex-ol compatible with the sequence A–B–(F)C–D–E of Fig. 1.

Methylation analysis of 12 (Table 1) afforded only monosubstituted alditol derivatives, except for 4,6-disubstituted galactose characteristic of terminal pyruvated galactose. This finding confirmed a linear arrangement of the monosaccharide unit and suggests an enzymatic cleavage of the β(1 → 3)-linkage between residues C and E. The structure of the hexasaccharide present in smaller amounts could not be verified from the methylation analysis data, since the presence of a Glc residue linked to O-6 of the

M. Nimtz et al. / Carbohydrate Research 287 (1996) 59–76

![Diagram](attachment:diagram.png)
core galactose would generate the 4,6-disubstituted galactose derivative already produced by the terminal pyruvated galactose residue. Terminal Glc was, as expected, observed, but could be due to contaminants.

Compound 12 was also investigated by NMR spectroscopy along similar routes as described for 11. In particular, the anomeric configurations of all monosaccharide residues could be unequivocally determined, most of the linkages could be corroborated from the data from the TOCSY and ROESY spectra, and four sub-species A^{1-4} differing in the degree of O-acetylation and/or the position of the O-acetyl residues were distinguished. The exact nature of all four species was readily deduced from the COSY spectrum (Fig. 5) as O-acetylation causes large downfield shifts for geminal skeleton protons. Quantitation was achieved by integration of the respective anomeric proton signals of the residues A^{1-4} and of the methyl singlet of the pyruvate group, which was completely resolved for each sub-species and indicative of a single pyruvate configuration. Comparison of its chemical shift with data published in the literature [28,29] indicates an equatorial conformation of the methyl group and the R configuration of the pyruvic substituent. This was confirmed by the absence of ROESY cross-peaks to this methyl group. The influence of the heterogeneity of residue A on the neighbouring monosaccharide unit was negligible and allowed a relatively straightforward assignment of the remaining spin systems (Table 2). The linkage between residues B and C and between D and βE could be unequivocally confirmed by cross-peaks in the ROESY spectrum. The NMR data are compatible with an α(1 → 3) or α(1 → 4) bond between residues A and B due to the similarity of the chemical shifts of the respective protons of residue B and the heterogeneity of residue A. Fortunately, methylation analysis data resolve this ambiguity, definitively excluding a substitution at O-3 of the GlcA residue. The linkage between residues D and αE and especially between C and D, however, could not be verified in the same way due to signal overlap. The chemical shifts of D H-2,3,4 do not exhibit any low field shift compared to terminal galactose, and therefore a substitution at O-2,3,4 is unlikely. Since residue A was already shown to be terminal, residue D probably is substituted at O-6, which is in full agreement with the methylation analysis data.

The detection of additional weak signals (relative intensity about 10%) of a β-glucosyl spin system is compatible with the presence of small amounts of the glucose residue F β(1 → 6)-linked to the core galactose C, as suggested by the negative-ion mode ESI-MS of the native and the positive-ion mode ESI-MSMS of the reduced and permethylated sample.

**Structural elucidation of the decasaccharide produced by enzymatic depolymerization of amylovoran.**—Additionally to the pentasaccharide discussed above, a larger oligosaccharide 13 was obtained by the enzymatic depolymerization procedure. It was identified as the dimer of the pentasaccharide by negative-ion mode ESI-MS. Deprotonated doubly

Fig. 4. Daughter-ion mass spectra with affixed fragmentation schemes of reduced (NaBD₄) and permethylated oligosaccharides obtained by enzymatic depolymerization from amylovoran. (a) Repeating unit of amylovoran; (b) modified repeating unit bearing an additional glucose residue (approximately 10% of material); (c) dimer of the repeating unit showing the site of polymerization.
Fig. 5. Part of the 600-MHz ${}^{1}$H 2D COSY contour plot of the pentasaccharide obtained from amylovoran by enzymatic depolymerization. Trace a shows the cross-peaks for H-1 to H-5 of the 2,3-di-$O$-acetyl, b those of H-1 to H-4 of the 2-$O$-acetyl, c those of H-1 to H-4 of the 3-$O$-acetyl, and d those of H-1 to H-2 of the non-$O$-acetylated terminal pyruvated galactose moiety. Each trace begins at the H-1 signal in the region 5.45 to 5.65 ppm.
charged molecular ions were detected at $m/z$ 923 (40% rel abundance), 944 (100%), 965 (100%), and 986 (63%), corresponding to $\left[\text{PyrHex}_2\text{Hex}_6\text{HexA}_2 - 2\text{H}\right]^{2-}$ with 1–4 additional $O$-acetyl groups. Furthermore, less intense signals of undecasaccharides at $m/z$ 1025 (20%), 1046 (42%), and 1067 (15%) were observed, corresponding to $\left[\text{PyrHex}_2\text{Hex}_7\text{HexA}_2 - 2\text{H}\right]^{2-}$ with 2–4 additional $O$-acetyl groups. This is in good agreement with the data obtained for the monomeric structure 12 and confirms the presence of approximately 10% of repeating units enlarged by one hexose residue.

Positive-ion mode ESI-MS of the reduced and permethylated sample showed signals at $m/z$ 2267 $\left[\text{PyrHex}_2\text{Hex}_5\text{HexA}_2\text{Hex-ol} + \text{Na}\right]^+$ and 2271 $\left[\text{PyrHex}_2\text{Hex}_6\text{HexA}_2\text{Hex-ol} + \text{Na}\right]^+$ (about 20%), corresponding to the sodiated molecular ions of the dimer of the repeating unit and its enlarged variant. MSMS of the disodiated, i.e., doubly charged molecular ion yielded the daughter-ion spectrum depicted in Fig. 4c. A complete $Y_i$-fragment-ion series, incorporating the reduced (deuterium-labelled) end of the molecule, was observed, and was confirmed by a partial $B_j$ series generated by fragmentation from the non-reducing end of the molecule. The data demonstrated the expected sequence of monosaccharide units, and allowed easy identification of the single branching point in the molecule. The two residues substituted at O-6 again gave rise to cleavages within the monosaccharide ring leading to fragments at $m/z$ 807 ($^3.5A_4$) and 1898 ($^3.5A_7$), in complete agreement with the sequence A–B–C–D–E–(A′–B′)C′–D′–E′ of Fig. 1.

Methylation analysis (Table 1), in particular the detection of 3,4-disubstituted galactose in addition to the 4-substituted derivative characteristic for the monomer, clearly defined the site of polymerization at O-3 of the core galactose C.

MSMS of the doubly charged parent of the undecasaccharide at $m/z$ 1247 yielded a fragmentation pattern explicable only by the assumption of the presence of two isomeric structures. The ratio of intensities of the monosaccharide ring cleavage fragments $^3.5A_4$ at both $m/z$ 807 (weak) and 1011 (strong), however, indicated that the structure A–B–(F)C–D–E–(A′–B′)C′–D′–E′ is dominant compared to the isomeric A–B–C–D–E–(A′–B′)(F′)C′–D′–E′. It is evident, that the enzyme cleaves only one specific linkage of the polysaccharide chain. However, it seems to prefer cleavage at core galactoses C modified by an additional hexose residue.

## 4. Discussion

The isolation of overlapping fragments produced by enzymatic or chemical procedures and the application of complementary MS and NMR methods for structural elucidation allowed the complete determination of the linkage arrangement of the repeating unit of amylovoran. Interestingly, the presence of various $O$-acetyl groups in the repeating unit is the source of some heterogeneity. Although all $O$-acetyl groups are linked to the terminal monosaccharide residue of the side branch, their exact position is variable. More heterogeneity is introduced by an additional glucose residue linked to the branched central galactose present in approximately 10% of all repeating units.

Although glucuronic acid and pyruvic acid as acidic components of bacterial EPS are rather common (see ref. [30] for a review), amylovoran is similar only to the structure of stewartan, the capsular EPS from the closely related bacterium *Erwinia stewartii*(unpublished results), whereas clear differences exist to the EPS of other bacteria. Colanic acid, e.g., of *Escherichia coli* also carries glucuronic acid and galactose in the side chain, but the backbone consists of fucose and glucose [31]. EPS from *E. chrysanthemi* strain SR260 contains predominantly rhamnose, and glucuronic acid is also part of the side chain [32] as in xanthan of *Xanthomonas campestris* or in EPS from *Rhizobium sp.* strain NGR234 (see ref. [5]). An intrinsic question is the specificity of these structures for the interaction of the bacteria with their hosts and the adjustment to changes in their environment and life cycle.

# Acknowledgements

We are grateful to Christel Kakoschke, Anke Meyer, and Christiane Proppe for excellent technical assistance.

# References

[1] E.M. Steinberger and S.V. Beer, *Mol. Plant-Microbe Interact.*, 1 (1988) 135–144.

[2] P. Bellemann and K. Geider, *J. Gen. Microbiol.*, 138 (1992) 931–940.

[3] F. Bernhard, D.L. Coplin, and K. Geider, *Mol. Gen. Genet.*, 239 (1993) 158–168.

[4] P. Bugert and K. Geider, *Mol. Microbiol.*, 15 (1995) 917–933.

[5] J.A. Leigh and D.L. Coplin, *Annu. Rev. Microbiol.*, 46 (1992) 307–346.

[6] P. Bellemann, S. Bereswill, S. Berger, and K. Geider, *Int. J. Biol. Macromolec.*, 16 (1994) 290–296.

[7] G. Geier and K. Geider, *Physiol. Molec. Plant Pathol.*, 42 (1993) 387–404.

[8] M. Gross, G. Geier, K. Rudolph, and K. Geider, *Physiol. Molec. Plant Pathol.*, 40 (1992) 371–381.

[9] A.R.W. Smith, R.A. Rastall, N.H. Rees, and R.C. Hignett, *Acta Horticult.*, 273 (1990) 211–219.

[10] H. Falkenstein, P. Bellemann, S. Walter, W. Zeller, and K. Geider, *Appl. Environm. Microbiol.*, 54 (1988) 2798–2802.

[11] J.S. Hartung, D.W. Fulbright, and E.J. Klos, *Mol. Plant-Microbe Interact.*, 1 (1988) 87–93.

[12] A.J. Mort, *Carbohydr. Res.*, 122 (1983) 315–321.

[13] A.J. Mort, F. Qiu, and N.O. Maness, *Carbohydr. Res.*, 247 (1993) 21–35.

[14] M.F. Chaplin, *Anal. Biochem.*, 123 (1982) 336–341.

[15] G.J. Gerwig, J.P. Kamerling, and J.F.G. Vliegenthart, *Carbohydr. Res.*, 62 (1978) 381–396.

[16] S. Hakomori, *J. Biochem. (Tokyo)*, 55 (1964) 205–207.

[17] D. Lobas, M. Nimtz, V. Wray, A. Schumpe, C. Proppe, and W.-D. Deckwer, *Carbohydr. Res.*, 251 (1994) 303–313.

[18] A. Bax and R. Freeman, *J. Magn. Reson.*, 44 (1981) 542–561.

[19] L. Braunschweiler and R.R. Ernst, *J. Magn. Reson.*, 53 (1983) 521–528.

[20] D. Morrison and K. Wüthrich, *Biochem. Biophys. Res. Commun.*, 113 (1983) 967–970.

[21] A. Bax and S. Subramanian, *J. Magn. Reson.*, 67 (1986) 565–569.

[22] M.F. Summers and A. Bax, *J. Am. Chem. Soc.*, 108 (1986) 2093–2094.

[23] D.J. Politis and R.N. Goodman, *Appl. Environ. Microbiol.*, 40 (1980) 596–607.

[24] K. Jumel et al., unpublished results.

[25] A.R.W. Smith, R.A. Rastall, P. Blake, and R.C. Hignett, *Microbios*, 83 (1995) 27–39.

[26] Y.A. Knirel, E.V. Vinogradov, and A.J. Mort, *Adv. Carbohydr. Chem. Biochem.*, 47 (1989) 167–202.

[27] B. Domon and C.E. Costello, *Glycoconjugate J.*, 5 (1988) 397–409.

[28] P.J. Garegg, P.-E. Jansson, B. Lindberg, F. Lindh, J. Lönngren, I. Kvarnström, and W. Nimmich, *Carbohydr. Res.*, 78 (1980) 127–132.

[29] P.-E. Jansson, J. Lindberg, and G. Widmalm, *Acta Chem. Scand.*, 47 (1993) 711–715.

[30] I.W. Sutherland, *Biotech. Adv.*, 12 (1994) 393–448.

[31] P.J. Garegg, B. Lindberg, T. Onn, and I.W. Sutherland, *Acta Chem. Scand.*, 25 (1971) 2103–2108.

[32] J.S.S. Gray, J.M. Brand, T.A.W. Koerner, and R. Montgomery, *Carbohydr. Res.*, 245 (1993) 271–287.