
Molecular Microbiology (1996) 21(5), 1087–1099

# Molecular characterization of group A streptococcal (GAS) oligopeptide permease (Opp) and its effect on cysteine protease production

Andreas Podbielski,${}^{1*}$ Barbara Pohl,${}^{1}$ Markus Woischnik,${}^{1}$ Christiane Körner,${}^{1}$ Karl-Hermann Schmidt,${}^{2}$ Eva Rozdzinski${}^{3}$ and Bettina A. B. Leonard${}^{4}$

${}^{1}$Institute of Medical Microbiology, Hospital of the Technical University (Klinikum RWTH), Pauwelsstrasse 30, D-52057 Aachen, Germany.

${}^{2}$Institute of Experimental Microbiology, University Jena, Winzerlaer Strasse 10, D-07745 Jena, Germany.

${}^{3}$Institute of Microbiology and Immunology, Hospital of the University, Robert-Koch-Strasse 8, D-89081 Ulm, Germany.

${}^{4}$Department of Microbiology, Box 196 UMHC, University of Minnesota, 420 Delaware Street SE, Minneapolis, Minnesota 55455, USA.

## Summary

Bacterial oligopeptide permeases are membrane-associated complexes of five proteins belonging to the ABC-transporter family, which have been found to be involved in obtaining nutrients, cell-wall metabolism, competence, and adherence to host cells. A lambda library of the strain CS101 group A streptococcal (GAS) genome was used to sequence 10192 bp containing the five genes *oppA* to *oppF* of the GAS *opp* operon. The deduced amino acid sequences exhibited 50–84% homology to pneumococcal AmiA to AmiF sequences. The operon organization of the five genes was confirmed by transcriptional analysis and an additional shorter *oppA* transcript was detected. Insertional inactivation was used to create serotype M49 strains which did not express either the *oppA* gene or the ATPase genes, *oppD* and *oppF*. The mutation in *oppA* confirmed that the additional shorter *oppA* transcript originated from the *opp* operon and was probably due to an intra-operon transcription terminator site located downstream of *oppA*. While growth kinetics, binding of serum proteins, and attachment to eukaryotic cells were unaffected, the *oppD/F* mutants showed reduced production of the cysteine protease, SpeB, and a change in the pattern of secreted proteins. Thus, the

Received 23 January, 1996; revised 25 June, 1996; accepted 1 July, 1996. *For correspondence. E-mail andreas.p@imib.rwth-aachen.de; Tel. (241) 8089512; Fax (241) 8888483.*

GAS *opp* operon appears to contribute to both protease production and export/processing of secreted proteins.

## Introduction

Bacterial oligopeptide permeases (Opps) have been identified in several Gram-negative (Hiles *et al.*, 1987; Fleischmann *et al.*, 1995) and Gram-positive species (Alloing *et al.*, 1990; (*ami*); Rudner *et al.*, 1991; Perego *et al.*, 1991; Tynkkynen *et al.*, 1993). Oligopeptide permeases belong to the ABC-transporter family (Tam and Saier, 1993) and comprise a complex of five membrane-associated proteins. OppA/AmiA are substrate-binding proteins which are periplasmic proteins in Gram-negative bacteria and membrane-associated lipoproteins in Gram-positive bacteria. OppB/AmiC and OppC/AmiD are transmembrane proteins which form a channel for passage of the substrate, and OppD/AmiE and OppF/AmiF are inner membrane-associated ATPases which provide energy for transport of the peptide substrate. Except for *Lactococcus lactis*, the encoding genes are arranged in the order *oppA/amiA* to *oppF/amiF* on the chromosome. In the case of *Salmonella typhimurium*, *Bacillus subtilis*, and *L. lactis* the *opp* operon products have been directly demonstrated to transport tripeptides and oligopeptides.

A variety of roles for Opp have been defined in bacteria. An involvement of Opp in cell-wall turnover has been shown for *Escherichia coli* (Tame *et al.*, 1994; Park, 1995). *L. lactis* uses the Opp to import peptides produced by an extracellular protease which are utilized as an amino acid source (Kunji *et al.*, 1995). Recently, investigations have focused on potential regulatory roles of Opp in bacteria. In *Bacillus*, Opp was found to play a role in the induction of competence and sporulation possibly by transporting a peptide signal, CSF, into the cell (Magnuson *et al.*, 1994; Solomon *et al.*, 1995). Similarly, the Opp of *Enterococcus faecalis* was reported to bind the enterococcal sex pheromone and transport it into the cell, thereby inducing intercellular aggregation (Leonard *et al.*, 1996). In *Streptococcus gordonii* and *Streptococcus pneumoniae*, the first evidence for a role of Opp in virulence regulation was discovered when mutations in Opp were found to affect adherence to substrates, other bacteria or eukaryotic cells (Jenkinson and Easingwood, 1990; Jenkinson, 1992; Cundell *et al.*, 1995).

1088 A. Podbielski et al.

Bacteria of the species *Streptococcus pyogenes* (group A streptococci, GAS) are important human pathogens. They express a variety of cell-surface and secreted virulence factors (recently reviewed by Ferretti *et al.*, 1991; Kehoe, 1994). Among the secreted factors, the cysteine protease SpeB has been suggested as contributing to the invasiveness of GAS (Talkington *et al.*, 1993; Raeder and Boyle, 1995; Shanley *et al.*, 1996). Although initially characterized as a mitogenic factor (Hauser and Schlievert, 1990), the molecule has recently been shown to be a cysteine protease (Chaussee *et al.*, 1993; Ohara-Nemoto *et al.*, 1994; Musser *et al.*, 1996). It was found to cleave human serum proteins like pre-IL-1β (Kapur *et al.*, 1993a), fibronectin (Kapur *et al.*, 1993b), human cell-surface proteins like the urokinase-receptor (Wolf *et al.*, 1994), and streptococcal surface proteins, M protein and C5a peptidase (Berge and Björck, 1995). The mitogenic activity has been reassigned to a molecule which co-purified with SpeB in the initial studies (Gerlach *et al.*, 1994).

We postulated that GAS also possess an *opp* operon and that this operon could be involved in regulation of bacterial pathogenesis, possibly by affecting adherence or virulence-factor production. A fragment of the GAS *oppD* gene was obtained by degenerate polymerase chain reaction (PCR) and used as an initial probe to sequence the entire GAS *opp* operon from a lambda library. After examination of the transcriptional organization of the *opp* operon, isogenic mutants were created in *oppD/F* and *oppA*. Upon analysis of the Opp mutants for impact on eukaryotic cell adhesion and secretion of virulence factors, we found that disruption of the *opp* operon affected production of the secreted cysteine protease, SpeB, and protein-secretion patterns of M49 GAS.

---

### Results

#### Analysis of the GAS *opp* operon sequence

Five predicted open reading frames in the 10192 bp of sequenced GAS genome (data not shown, EMBL Data Library Accession Number X89237) showed significant sequence identity with known Opps. Homology comparisons with five full or partial oligopeptide permease operons from *S. pneumoniae* (*amiA–F*), *L. lactis* (*oppA–F*), *B. subtilis* (*oppA–F*), *S. gordonii* (*hppAB*) and *S. typhimurium* (*oppA–F*) revealed that the GAS *opp* operon contained, from 5′ to 3′, the *oppA* to *oppF* genes. The highest homology (Table 1) was found for *oppF* (84% to *S. pneumoniae amiF* and 43.3% to *S. typhimurium oppF*). The *oppA* genes showed the least homology (with 49.7% homology to *S. pneumoniae amiA* and 19.7% to *S. typhimurium oppA*). The deduced amino acid sequences result in proteins of 656 (OppA), 500 (OppB), 308 (OppC), 356 (OppD), and 307 (OppF) residues. The putative extracellular OppA was found to contain an N-terminal leader peptide and subsequent cysteine residue typical of Gram-positive, bacterial lipoprotein, cell-membrane anchors (Fig. 1A). The proteins OppB and OppC were predicted to contain six and four transmembrane domains (Fig. 1A). Both OppD and OppF included a F/YSGGM/QRQRIV/GA box identified as a signature sequence for ATPases of ABC transporters (Fig. 1A). While the genes for OppB to OppF were either directly adjacent to each other or overlapping by a few nucleotides, there was a 64 bp stretch of non-coding sequence between *oppA* and *oppB* (Fig. 1B). The intervening sequence has the potential to form a stem-loop structure (−63.1 kJ mol⁻¹) followed by a run of three dT nucleotides which could function as a Rho-independent transcriptional terminator structure (Fig. 1B). In contrast,

---

### Table 1. Homology comparison of the deduced amino acid (aa) sequences of *opp*-like genes from selected bacterial species.

| Streptococcus pyogenes | Streptococcus pneumoniae (%) | Bacillus subtilis (%) | Streptococcus gordonii (%) | Lactococcus lactis (%) | Salmonella typhimurium (%) |
|-------------------------|------------------------------|------------------------|----------------------------|------------------------|-----------------------------|
|                         | AmiA                        | AliA                   | AliB                       | HppA                   | HppG                        | HppH                      |
| OppA                    | id.                          | 49.7                   | 51.1                       | 49.1                   | 22.2                        | 49.5                      | 46.2                     | 53.2                     | 11                        | 19.7                      |
|                         | sim.                        | 12.8                   | 11.3                       | 13.2                   | 12.8                        | 12.2                      | 12.6                     | 10                       | 12.2                      | 12                        |
| OppB                    | id.                          | 63.4                   |                            |                        | 22.8                        | 66                        |                          |                          | 24.2                      | 20                        |
|                         | sim.                        | 13.6                   |                            |                        | 24.7                        | 13                        |                          |                          | 17                        | 24                        |
| OppC                    | id.                          | 72.4                   |                            |                        | 32.1                        |                          |                          |                          | 26.2                      | 24.8                      |
|                         | sim.                        | 13.6                   |                            |                        | 21                          |                          |                          |                          | 15.7                      | 21.5                      |
| OppD                    | id.                          | 78.8                   |                            |                        | 53.2                        |                          |                          |                          | 43.5                      | 42.7                      |
|                         | sim.                        | 7.3                    |                            |                        | 11                          |                          |                          |                          | 11                        | 13.7                      |
| OppF                    | id.                          | 84                     |                            |                        | 52.7                        |                          |                          |                          | 39.8                      | 43.3                      |
|                         | sim.                        | 5.8                    |                            |                        | 13                          |                          |                          |                          | 13                        | 16.6                      |

Homology comparisons were performed by using the PC GENE (IntelliGenetics) PALIGN program for pairwise analyses. The results are shown as per cent identical (id.) and per cent similar (sim.) aa residues. The *opp* sequence data used were taken from Alloing *et al.* (1990; 1994; *S. pneumoniae amiA–F* and *aliAB*), Perego *et al.* (1991; *B. subtilis oppA–F*), Jenkinson *et al.* (1996; *S. gordonii hppABGH*), Tynkkynen *et al.* (1993; *L. lactis oppA–F*) and Hyles *et al.* (1987; *S. typhimurium oppA–F*).

A | 10 | 20 | 30 | 40 | 50 | 60 | 70 | B
--- | --- | --- | --- | --- | --- | --- | ---
OppA* |  |  |  |  |  |  | 
MKKSKNLAAVSTALLSYSALAAA CGNKNASGGSEATAKTYKYVFVNDPKSILDYILTNGGGTTDVITQMVDGL |  |  |  |  |  |  | GTTGAAAAATTAACGATGTTCAAAAGAACTTTCTCGTTTATGGAGGGAGTTCTTTTGAACATTATGGAGG
LENDEYGNIKVPSLAKDWKVSKDGLTYYTIRLDGVSWYTADGEEYAPVTAEDFVTGLKHAVDDKSDALYVV 140 |  |  |  |  |  |  | AIAACTATGAAAAAAAAACAT
EDSIKNLKAYONGEVDFKEVGVALDDKFQYTLNKPESYWSKTFYSVLFPVNAKELKSKGKDEGTTPD |  |  |  |  |  |  | 
SSILVNGAYFLSAFTSKSSMEFHKNEYWDAKNVGIESVKLTYSDGDPGSFYKNEDKGEEFVARLYLPND 280 |  |  |  |  |  |  | 
PTYKSACKKNYADNITYGMLTGDIRDHLTWNLNRTSFKNTKKDP AQQDAGKKAINNKDERQAIQEA FDRASE |  |  |  |  |  |  | TA
OATQAGQDAKT KALRNMLVPPTFVTIGESDFGEVEKEMAKLGDEWKDVNLADAQDFYNPEKAKAEFAK 420 |  |  |  |  |  |  | TT
AKEALTAEGVTFPVQLDYPVDQANAATVQEAQSFKQSVVEASLGKENIVNVLEETSTHEAQGFYAETPE |  |  |  |  |  |  | TG
QQDYDISSWWGP DYQDPRTYLDIMSPVGGGSVIQKLGIKAGQNKD VVAAAGLDTYQTLIDEAAAITDDN 560 |  |  |  |  |  |  | G
DARYKAYAKAQAYLTDNAVDIPVV ALGGTPRVTKAVPFSGGFSWAGSKGPLAYKG MKLQDKPVTAQYEK |  |  |  |  |  |  | T
AKEKWMKAKSNAKYAEKLADHVEK |  |  |  |  |  |  | C-G
IKLSTKKGGK |  |  |  |  |  |  | T-A
OppB |  |  |  |  |  |  | 
MKYIILNRIMRSLVSVVLVFALT YTVYFLVPSSLIFKODPENYKMTTPD KKVA YENLTFORMGYVNYE |  |  |  |  |  |  | T-A
SSKELKDNASKVDS SVTTEATSANKAVYEKYIDS LGNGWQVKREPTSKQVYAIRNIP IYERVWNFE SNLV 140 |  |  |  |  |  |  | C-G
GIDHPWKIQDKDNPKLARYIRIE KD KS GGWSIVGSGTKHKYLLYINGKEPYLHQNFVTLLNIGTSYPTYSN |  |  |  |  |  |  | A-T
IPVLQVISQQGQR TALQDVTFPSGVTKKSSVDIYSRSYKNPKSILDDITKVNYKGDSYTKTINNYVDPSM 280 |  |  |  |  |  |  | A-T
IHNSFVIGFFGVMESHIVGLPLGLFMAREKN TYFDRFESTATMFMLALPSI AVIYYYVRFLGGMVGLPDSE |  |  |  |  |  |  | G-C
PMLGASDPKSYILPALILGILN IPTVIVWFRRYLVDLQASDWRFARSKGLISESEIYRGHLFKNAMVP IV 420 |  |  |  |  |  |  | A-T
SGVPASI ILAIGGATL TETVFAFPGMGKM LIDSIKSANNSMIVGLIFFFTVLSIVSILLGDIVMTLV DPR |  |  |  |  |  |  | A-T
IKLSTKKGGK |  |  |  |  |  |  | C-G
OppC |  |  |  |  |  |  | 
MESIDKS KFRFVERDSEA SEVIDTPAYSYWSVFRQFFSKKSTV EMLVILVTVLMMSFIYPMFANYDFND |  |  |  |  |  |  | T-A
VSNINDEF SKRYIPNAEYWEGTDKNGQS LFDGVGYGARN SILLISVIATLINITGVVIGAIWGVSKA FDK 140 |  |  |  |  |  |  | C-G
VMIEIYNNI ISNIPSMLIIIVLTYSLGAGEWNLILAF C ITGWIGVAYSIRVQILYRDLEYNLASQTLGTP |  |  |  |  |  |  | A-T
MYKIAVK NLLPQLVSVMITMLSQMLP VYVSSETFLSFFGIGLPTTTPSILGR LIANYSNLT TNAYLFWIP 280 |  |  |  |  |  |  | A-T
IVTLLIVSLPLYIVGQN LADASDPPSRHR |  |  |  |  |  |  | A-T
OppD |  |  |  |  |  |  | 
MTKENNVIL TAKDVVVEFDVRDRVITAIRNVSLLEVEGEVLA FVGGSSSSVLTKTFTGML ESN GR IAN |  |  |  |  |  |  | 3'
GSI VYRGQELTDLKTNKEWAKIRGSKIAF IODPMTLSPIK TIGSQITEVIIKHQKVSHAKEMALDY 140 |  |  |  |  |  |  | Start OppB
MN KVGINPAKKRFEDYPELYSGGMRQ RIVI AALACRPDILICDEPTTALDVT IQA QIVELLKS LQREYH |  |  |  |  |  |  | stop OppA
FLI IFITHDLGVVASIADKV AMMYAGEIVEFGTV EEIFYDP RH PYTWSLLSSLPQLADESGELYAIPGTP 280 |  |  |  |  |  |  | 
PSLYSPIIGDAF ALRSEYAMVLD FEKAPPAINVSETHWAKTWLLHPEAPKVQKPEVIOGLHQKILRK MSQ |  |  |  |  |  |  | 
QEEGNV |  |  |  |  |  |  | 
OppF |  |  |  |  |  |  | 
MSEKLVEVK DLEISFGEGKKKFGAVKNAFFEIKKGETFSLV GESSGSGKTI GRAIIGINDTSSGQI LYDG |  |  |  |  |  |  | -10
KVIN GKRKSSEANELIRKIQMIFQDPAASLNERATVDYIISEGLYNFNLFKTEEERKEKIKNMMAEVG LL 140 |  |  |  |  |  |  | -35
SEHLTRYPHELSGGQORIGIARAJVMNPEFVIADEPISALDVSVRAQVLNLLKKRMQAEKGLTYLEIAHD |  |  |  |  |  |  | 
LSV VRFISDRIAVIHKGVIVEVAETEELFN NPIHPYTSLLISAVPIPDPILERQKELVYYPDQHDYTL D 280 |  |  |  |  |  |  | +1
KPSMVEIKPNHFVWANQAEIKYQK EL |  |  |  |  |  |  | S/D

1090 A. Podbielski et al.

Fig. 2. A. Transcript analysis of GAS serotypes of the M49 wild-type strain and isogenic oppD/F mutants. Total RNA (10 μg) from the M49 wild-type (wt) strain (lane 1) and from isogenic M49 OppDF⁻ mutants (lanes 2 and 3) were subjected to denaturing agarose gel electrophoresis and Northern blot hybridization. Specific transcripts were visualized with oppA- (left panel) and oppF-specific probes (right panel) prepared by directly labelling PCR products with Dig-dUTP (for details see the Experimental procedures). An RNA size reference (in kb) was run in parallel. With both probes, a 7.4 kb band could be detected in the wild-type RNA corresponding to the polycistronic transcript of the entire opp operon. Smaller bands are assumed to result from breakdown products of the polycistronic transcript, except for a distinct 2.3 kb band detected with the oppA probe. This band corresponds to an oppA-only transcript, started at the regular opp promoter and terminated at the stem-loop structure between oppA and oppB (B). As expected, this band could also be visualized in the oppD/F mutants, while in these strains the polycistronic opp transcript could not be detected by the oppF probe.

B. Southern blot hybridizations utilizing an oppA-specific probe and genomic DNA of the M49 wild-type (wt) strain and an isogenic oppA replacement mutant. The genomic DNA of the M49 wild type and the oppA replacement mutant strain (oppA⁻) was digested with SpeI, subjected to 1% agarose gel electrophoresis, and Southern blotted. A probe specific for the oppA-internal sequence replaced in the mutant strain by the aad9 resistance gene was used for hybridization at different levels of stringency achieved by diverse hybridization temperatures.

no terminator sequence could be identified in the 2352 bp sequenced downstream of oppF. Approximately 70 bp upstream of oppA, a potential promoter was identified including the sequences for -35 and -10 boxes separated by 17 bp (Fig. 1C). The putative -10 box also exhibited excellent homology to pneumococcal extended -10 boxes known to direct transcription in the absence of -35 boxes (Sabelnikov et al., 1995). In the 2352 bp downstream of oppF no additional open reading frames (ORFs) were identified on either strand. A 975 bp ORF was identified on the opposite strand 330 bp upstream of oppA. This putative gene exhibited the highest degree of homology to penicillin-binding protein 3 (PBP3) of S. pneumoniae (Schuster et al., 1990) and its potential promoter also had perfect homology to pneumococcal extended -10 boxes and did not overlap the putative opp promoter (Fig. 1C).

Transcriptional analysis of the GAS opp operon

In order to confirm the operon organization of the GAS opp genes and to gain information necessary for opp-gene mutagenesis, an analysis of specific GAS gene transcripts was conducted. Probes directed to the oppA and oppF genes were used for a Northern blot hybridization with RNA preparations from the M49 serotype. With each probe, a 7.4 kb band was detected corresponding to a polycistronic transcript of five genes, thus confirming the operon structure (Fig. 2A). If it is assumed that the predicted promoter for the GAS operon is correct, it appears that these transcripts would not extend for more than 900 bp downstream of oppF. Because of the absence of symmetrical transcription terminators in this region, opp transcription termination may depend on Rho or another RNA polymerase co-factor. Additional smaller bands were detected with both opp-directed probes (Fig. 2A). The smaller smeared bands (< 1.5 kb) are probably a result of mRNA degradation or processing. In contrast, the distinct 2.3 kb band detected with the oppA probe corresponded in size to an oppA message which could begin at the same putative promoter as the larger transcript (Fig. 1C) and is terminated at the stem-loop structure in the non-coding region between the oppA and oppB genes. On the basis of densitometrically quantified band intensity with the oppA probe, the 2.3 kb oppA-only transcript outnumbers the entire opp transcript by a factor of 10. No differences in the amount of opp message per total RNA could be detected in RNA preparations from early and late log-phase cells (data not shown).

# Generation of GAS opp-operon mutants

Isogenic Opp mutants were created to address the following questions: (i) was the small 2.3 kb *oppA* band due to the presence of additional *oppA*-like genes as seen for the *ami* locus (*aliA*, *aliB*) or due to higher expression of the *oppA* gene; and (ii) does the *opp* operon play a role in protein expression of GAS. Isogenic replacement of *oppA* was performed by using the pFW6 vector to insert a spectinomycin-resistance gene in place of the wild-type *oppA*.

To study the molecular and biological functions of the GAS *opp* operon, the Opp-associated ATPase function was disrupted. The mutation was designed to affect both ATPases, since it had been demonstrated that a single ATPase is sufficient for Opp activity in *B. subtilis*. Insertional mutagenesis utilizing the vector pSF152 was performed which resulted in a truncated *oppD* gene and eliminated *oppF* expression. When specific transcripts in the isogenic pairs were compared by Northern blot hybridization with the *oppA* probe, two transcripts were detected in the M49-derived OppD/F mutants (*OppDF⁻*), neither of which corresponded to full-length transcripts (Fig. 2A). The first was the mRNA containing only the *oppA* gene, which was not affected in the mutants. A new band was also detected (~7 kb) which consisted of the polycistronic *oppA-D'* transcript fused to 850 bp of the pSF152 vector (Fig. 2A). When a probe directed to the *oppF* portion of the transcripts was used, as expected no hybridization signal could be visualized with the RNA of the mutant (Fig. 2A).

## Table 2. Minimum inhibitory concentrations (MIC) of toxic peptide derivatives for GAS M49 wild type (wt) and *OppDF⁻* mutant strains.

| Strain       | Bialaphos | Aminopterin |
|--------------|-----------|-------------|
| Wild type    | 10        | 1           |
| *OppDF⁻*     | 20        | >20         |

MIC values (g l⁻¹) were determined according to NCCLS standards (1993) using TH broth as the culture medium. The peptide derivatives were obtained from Sigma.

---

### Fig. 3. Detection and measurement of protease activity of GAS wild-type and Opp mutant strains. GAS M49 wild-type (A) and isogenic Opp mutant (B) bacteria were grown on Ca-caseinate indicator agar for 24 h and inspected for cleared zones of proteolysis. Protein preparations from culture supernatants of both wild-type (wt) and Opp mutant bacteria (*OppDF⁻*) were measured for protease activity (C) utilizing a quantitative assay with FITC-labelled casein as substrate (for details see the Experimental procedures). Only the results of 48 h assays, each performed with three independent protein preparations, are shown. The proteolytic activity is given as U h⁻¹ mg⁻¹ protein. Cysteine protease-specific activity was assessed by adding the specific inhibitor E-64 (at a final concentration of 28 mM) to the assays.

**A**

![M49 wt](image.png)

**B**

![M49 oppD⁻](image.png)

**C**

![Bar chart](image.png)

© 1996 Blackwell Science Ltd, *Molecular Microbiology*, **21**, 1087–1099

1092 A. Podbielski et al.

**Absence of oppA-like genes in GAS**

To determine whether additional *oppA*-like genes were encoded by the GAS chromosome, Southern blot analysis of the *oppA* allelic replacement strain was carried out with the *oppA*-specific probe at hybridization temperatures of 65°C, 50°C, and 40°C. The replacement of *oppA* within the *opp* operon eliminated all *oppA*-hybridizing bands present in the wild type (Fig. 2B). The *oppA* allelic replacement mutant was shown to contain only a single copy of the *aad9* spectinomycin-resistance gene per genome, and thus multiple allelic replacements in other potential *oppA*-like genes were ruled out (see the Experimental procedures).

**Characterization of biological properties of the GAS Opp mutants**

As Opp mutations have been found to affect adherence and protein secretion, a number of biological properties known to contribute to virulence were examined in the *oppD/F* insertion mutant Opp49 strain.

The peptide derivatives bialaphos and aminopterin, which are toxic when transported into cells, were used to assess Opp function. The OppDF⁻ strain showed significantly higher resistance to aminopterin, indicating that its transport had been reduced (Table 2). However, relatively high concentrations of the toxic peptides had to be used before they affected growth of the wild type. This could be a reflection of a high natural resistance of GAS to these peptides or a reflection of the fact that the assays had to be performed in complex media since the chemically defined media for GAS did not give reliable results with this assay.

Bacteria can use oligopeptides as nutrients and although bacteria possess additional transport systems for di- and tripeptides (Hagting *et al.*, 1994; Foucaud *et al.*, 1995), deficiency of Opp function could affect the growth characteristics of the mutants. Therefore, the consequences of Opp mutations on growth of GAS were investigated. The growth of GAS under laboratory conditions in either Todd-Hewitt (TH) broth, tryptone soy (CASO) broth, or THB-supplemented CDM (CDM is Chemically Defined Medium; van de Rijn and Kessler, 1980) was unaffected by the OppDF⁻ mutation, indicating that, at least in vitro on partially complex media, GAS does not obtain essential nutrients through transport of oligopeptides (data not shown).

To determine whether the mutation in *oppD/F* affected the adherence of GAS or secreted virulence factors, a number of parameters were measured. No changes in the ability of OppDF⁻ strains to bind to peroxidase-labelled human serum albumin, fibrinogen, fibronectin, vitronectin, IgG and IgA was detected (see the Experimental procedures, data not shown). Unlike Opp-mutated pneumococci, the OppDF⁻ isolates showed no significant difference in eukaryotic cell binding (see the Experimental procedures, data not shown). However, protease production was found to be significantly reduced as measured by degradation of casein.

Relative to wild-type bacteria, the M49 Opp mutants showed reduced proteolytic clearing when grown on Caseinate indicator agar (Fig. 3, A and B). Quantitatively, there was a 50–70% reduction in protease activity in supernatants collected from 48 h cultures of M49 wild-type and isogenic mutant bacteria (Fig. 3C). The cysteine protease inhibitor E-64 was able to reduce wild-type and

![Figure 4](https://i.imgur.com/yourimage.png)

Fig. 4. SDS-PAGE and Western immunoblot analysis of cysteine protease from GAS wild-type and Opp mutant bacteria. Equal amounts of protein preparations from culture supernatants of M49 wild-type (wt) and Opp mutant (*oppD/F⁻*) bacteria as well as purified mature cysteine protease (SpeB) were subjected to SDS-PAGE followed by Coomassie blue staining (A) and subsequent Western immunoblotting using a polyclonal rabbit anti-SpeB antiserum (B). The molecular mass of the size-reference proteins is given in kDa. The protein preparations were again subjected to SDS-PAGE (C). An aliquot of the wild-type-derived proteins was treated with DTT (wt + thiol), a known inducer of SpeB auto-activation, and run in parallel.

remaining OppDF⁻ activity to a basal level of non-cysteine protease activity which represents only about 3–7% of all protease activity (Fig. 3C).

As SpeB has been reported to have cysteine protease activity (Chaussee *et al.*, 1993), production of SpeB was measured. Secreted proteins were isolated from the supernatants of 48 h GAS cultures and analysed by SDS-PAGE. A major band corresponding to the molecular weight of mature SpeB and reacting with polyclonal SpeB antisera was significantly reduced in the protein preparation from OppDF⁻ (Fig. 4, A and B). A second anti-SpeB reactive 40 kDa band also was found to be reduced in the Opp mutant (Fig. 4, A and B). The band was the same size as pro-SpeB. As incubation of pro-SpeB at 37°C in the presence of 10 mM DTT has been reported to convert pro-SpeB to mature SpeB (Berge and Björck, 1995), protein preparations from wild-type cells were incubated under these conditions and re-analysed by SDS-PAGE. The higher molecular-weight band was converted to mature SpeB, confirming its identity as pro-SpeB (Fig. 4C). To determine which forms of SpeB had proteolytic activity, fluorescein isothiocyanate (FITC)-labelled casein was incorporated into an SDS-polyacrylamide gel, proteins were electrophoresed through the gel, and renatured upon removal of the SDS. Protein bands containing active proteases were able to remove the FITC label from the casein and appeared as dark bands on a background of fluorescent casein. The mature SpeB, but not pro-SpeB, showed proteolytic activity in this assay (Fig. 5).

It is interesting to note that the Opp mutation also affected the pattern of secreted proteins (compare

| size | oppD⁻ | wt | ref. | SpeB |
|------|-------|----|------|------|
| 80   |       |    |      |      |
| 49.5 |       |    |      |      |
| 32.5 |       |    |      |      |
| 27.5 |       |    |      |      |
| 18.5 |       |    |      |      |

Fig. 5. SDS-PAGE analysis of GAS wild-type and Opp mutant cysteine protease activity in the presence of FITC-casein. Protein preparations from culture supernatants of M49 wild-type (wt) and Opp mutant (*oppD/F⁻*) bacteria as well as purified cysteine protease (SpeB) were subjected to SDS-PAGE in the presence of 0.05% FITC-labelled casein. The molecular masses of the prestained size-reference proteins are given in kDa. Upon renaturation of the proteins at the end of the run, proteases become active and cleave the FITC-labelled casein. The breakdown products are washed out of the gel. Under illumination with UV light, the gel shows dark bands at the sites of protease activity.

Group A streptococcal oligopeptide permease 1093

Fig. 4A, lanes 1 and 2) where bands were lost, gained or altered in intensity.

### Discussion

Oligopeptide permease-transport systems have recently been shown to contribute to a number of functions in bacteria, including nutrient uptake, adherence and cell signalling. The present investigation had two major objectives: first, to identify and characterize the oligopeptide permease of GAS, and second, to determine whether the GAS Opp is involved in expression or production of any known GAS virulence factors.

Using degenerative PCR in conjunction with a lambda library, ≈10 kb of genomic sequence of GAS was obtained which contained the putative *opp* genes. Mutants in this operon showed increased resistance to toxic peptide derivatives, suggesting a transport role for this region. In addition, both the transcription organization and ORFs within this region show clear homologies to previously identified genes from *opp/ami* operons from *S. pneumoniae*, *S. gordonii*, *B. subtilis*, *L. lactis* and *S. typhimurium*. Five ORFs were identified which share increasing degrees of homology to the five *oppA* to *oppF* genes. The homologies were particularly high for *S. pneumoniae ami* genes, reflecting the phylogenetic relationship of this organism to GAS.

The operon structure of these five genes was confirmed by Northern blotting. During these experiments a second, shorter mRNA was detected which was ≈10 times more abundant than the full-length transcript. The shorter transcript does not represent the presence of *oppA*-like genes in the GAS chromosome, because after allelic replacement of the *oppA* gene in the *opp* operon no detectable hybridization of the chromosome to an *oppA* probe could be found. Therefore, the short transcript probably originates upstream of *oppA* and appears to terminate in an intergenic region between *oppA* and *oppB* which shows a stem-loop and dT motif consistent with a Rho-independent transcriptional terminator. This suggests a 10:1 ratio of OppA to the rest of the Opp proteins.

As OppA is assumed to be a lipoprotein which is not covalently bound to the rest of the permease machinery, this may suggest that it can move freely within the membrane to collect substrate molecules more efficiently. In addition, the excess of OppA molecules could facilitate more efficient substrate transport since OppA binding could be predicted to be the rate-limiting step of the transport process.

This relative abundance of the substrate-binding protein to the rest of the transport machinery may not be limited solely to GAS. *E. coli* contains a similar intergenic region between *oppA* and *oppB* including the inverted repeats which could function to increase the

© 1996 Blackwell Science Ltd, *Molecular Microbiology*, **21**, 1087–1099

1094 A. Podbielski et al.

ratio of *oppA* to downstream *opp* genes in this organism (Kashiwagi *et al.*, 1990). In addition, unlike GAS, other streptococci appear to have multiple *oppA*-like genes distributed throughout the chromosome (*aliA* and *aliB* of *S. pneumoniae* (Alloing *et al.*, 1994) and *hppG* and *hppH* of *S. gordonii* (Jenkinson *et al.*, 1996)). These *oppA*-like genes may either increase the ratio of substrate-binding proteins to the rest of the transport machinery or allow for some degree of substrate specificity. The latter has been demonstrated for the pheromone-specific *oppA*-like gene encoded on pheromone-inducible plasmid pCF10 of *E. faecalis* (Leonard *et al.*, 1996). The absence of additional *oppA*-like genes in GAS suggests that the substrate binding of the oligopeptide transport system of GAS may not be as evolved as that of the closely related *S. pneumoniae*.

In *L. lactis*, Opp has been shown to be necessary for transport of peptides into the cell, where they can be utilized as amino acid sources (Kunji *et al.*, 1995). As GAS are also auxotrophic for 14 amino acids (Davies *et al.*, 1965), it was important to assess the effect of Opp mutation on bacterial growth. The loss of OppD/F ATPase activity did not affect the growth rate of the mutants in complex media or in a chemically defined medium supplemented with small amounts of ultra-filtered complex medium, suggesting that the mutants were still able to obtain amino acids. No evidence of a second *opp* operon was found during the experiments aimed at identifying *oppA*-like proteins. However, it is possible that, like *L. lactis*, GAS have additional di- and tripeptide transport systems by which they can obtain essential nutrients (Hagting *et al.*, 1994; Foucaud *et al.*, 1995).

To determine whether the mutation in Opp affected the expression of any known GAS virulence factors, ATPase (OppDF⁻) mutants were created and a number of established virulence parameters were measured. Among these, only protease production was found to be significantly reduced as measured by degradation of casein. The ability of the cysteine protease inhibitor E-64 to inhibit over 90% of protease activity in both the wild-type and mutant GAS indicated that cysteine proteases represent the major protease activity in GAS. While the Opp mutants demonstrated an overall reduction in protease activity (50–70%), the small amount of E-64-resistant protease activity remained unaffected. This suggests that the Opp mutation reduced cysteine protease production but not general protease production.

SpeB is the major protease of GAS, and the loss of cysteine protease activity in the OppDF⁻ mutants corresponded to a reduction in SpeB production. This loss did not appear to be a consequence of altered or inhibited activation of SpeB, since the reduction was equivalent for both proteolytically inactive pro-SpeB and mature SpeB protease. The co-ordination of protease expression

and peptide-transport function has also been observed for the dipeptide permease in *L. lactis*. Unlike the Opp-dependent increase in SpeB expression, in *L. lactis* the expression of proteinases is downregulated in response to transport of dipeptides from the medium (Marugg *et al.*, 1995). The co-ordination of protease production and Opp function also may have been observed in *L. lactis* where the mutation of multiple intracellular peptidases appears to cause an upregulation of Opp production (Mierau *et al.*, 1996).

The OppDF⁻ mutants also had an altered protein pattern in their culture supernatants. These proteins represented secreted polypeptides and, potentially, also fragments of GAS surface proteins. SpeB has been shown to cleave the human surface protein urokinase receptor, serum proteins pre-IL-1β and fibronectin as well as the streptococcal surface proteins M and C5a peptidase (Kapur *et al.*, 1993a,b; Wolf *et al.*, 1994; Berge and Björck, 1995). Whether the altered pattern of secreted proteins is due to the reduction of SpeB proteolytic activity or a direct effect of Opp mutation awaits further study.

In pneumococci and *S. gordonii*, the reduction of adhesion of Opp mutants has been suggested to result from either the loss of OppA expression, which can act directly as an adhesin, or by loss of other adhesins due to regulation of adhesin expression by Opp (Jenkinson and Easingwood, 1990; Jenkinson, 1992; Cundell *et al.*, 1995). However, even though the GAS Opp mutants showed a difference in secreted protein expression, the serum protein- and eukaryotic cell-binding characteristics of the mutants remained unchanged. While it is still possible that OppA itself serves directly as an adhesin in the OppDF⁻ mutants, these data suggest that loss of Opp function, the resulting decrease in SpeB production, and changes in secreted protein patterns have no effect on adherence in GAS.

A role for Opps in signalling and gene regulation has been established in *E. faecalis* (Leonard *et al.*, 1996), *B. subtilis* (Magnuson *et al.*, 1994), *E. coli* (Tame *et al.*, 1994; Park, 1995), and *Agrobacterium tumefaciens* (Tam and Saier, 1993). Therefore, elucidation of the mechanism by which Opp mutation leads to reduced SpeB expression could also provide further insight into the relationship between proteases, Opps, and Opp-regulated gene expression.

## Experimental procedures

### Bacterial strains and culture conditions

GAS serotype M49 strain CS101 was provided by P. Cleary, Minneapolis, USA. GAS serotype M2 strain T2/44RB4/119 was obtained from D. Johnson, WHO Reference Laboratory, Minneapolis, USA. *E. coli* strain DH5α (Gibco BRL) served as the Gram-negative host for recombinant pSF152 plasmids.

and E. coli strains Blue MRF and XLOLR (Stratagene) were used as hosts for phages Lambda ZAP Express and ExAssist, respectively.

E. coli DH5α isolates transformed with pSF-derivatives were grown on disk sensitivity testing agar (Unipath) supplemented with 100 mg l⁻¹ spectinomycin. E. coli Blue MRF strains infected with recombinant lambda phages were grown in NZ casamino acids/yeast extract (NZY) agar and E. coli XLOLR strains transfected with recombinant ExAssist phagemids were grown on Luria–Bertani (LB) medium supplemented with 50 mg l⁻¹ kanamycin, both according to the instructions of the supplier (Stratagene). All E. coli cultures were grown at 37°C in ambient air.

GAS isolates were cultured in tryptone soy (CASO) broth (Unipath) or on TH agar (Unipath) at 37°C in a 5% CO₂ / 20% O₂ atmosphere. Mutant strains of GAS were maintained in medium containing 60 mg l⁻¹ spectinomycin. Growth rates of strains were determined by measuring optical density at 600 nm (OD₆₀₀) throughout the growth curve of triplicate cultures in CASO- or THY-broth as well as CDM (van de Rijn and Kessler, 1980) supplemented with 0.1–1% glucose and 0.2–1% 1 kDa exclusion-size filtrate (Diaflo chamber and YM1 ultrafilters, Amicon) from TH broth. To exclude a potential influence of spectinomycin on the growth rate, multiplication of GAS Opp mutants was compared with wild-type strains in the absence of spectinomycin, and with another GAS strain carrying an irrelevant pSF-based mutation in the presence of spectinomycin.

### Vectors

E. coli phages Lambda Zap Express (BamHI-arms, CIP treated) and ExAssist were purchased from Stratagene and used according to the instructions of the manufacturer.

Plasmid pSF152 (Tao et al., 1992) was used for insertional mutagenesis as described by Podbielski et al. (1995; 1996a).

Plasmid pFW6 was employed for allelic replacement mutagenesis following the protocol of Podbielski et al. (1996b).

Plasmid pUC18 (Merck) was utilized for initial cloning of PCR products obtained with degenerate oligonucleotide primers carrying BamHI sites as 5′ extensions.

Recombinant pBK plasmids resulted from ExAssist phage-mid transfections and were used according to the supplier's instructions (Stratagene).

### Conventional DNA techniques

Chromosomal GAS DNA was prepared by the method of Martin et al. (1990). Preparations of plasmid DNA from E. coli and conventional techniques for DNA manipulations were performed as described by Ausubel et al. (1990). Transformation of GAS strains by electroporation was according to the protocol of Caparon and Scott (1991).

### Specialized DNA techniques

For preparation of the lambda library, 200 μg of serotype M49 streptococcal genomic DNA was digested with 0.2 U of Sau3A (Boehringer) in a total volume of 200 μl for 30 min at 37°C. The resulting DNA fragments were separated according to size by a salt-gradient technique (Fink, 1991). Fractions

Group A streptococcal oligopeptide permease 1095

containing 2–9 kb fragments were ligated with BamHI-digested lambda arms and packaged using a Gigapack II packaging kit (Stratagene). Further processing and characterization of the lambda library as well as plaque-lifting onto Bio-dyne A membranes (Pall) were according to the instructions of the manufacturer (Stratagene).

For sequencing of the GAS opp operon, the amino acid sequences of the four published oligopeptide permeases were compared for the presence of highly conserved sequences. In addition to the Walker-motifs comprising the ATP-binding cassettes in the OppD- and OppF-ATPase sequences, four other short stretches of conserved sequence were identified in OppB, -D, and -F which were used for the design of degenerate oligonucleotides (Table 3). When utilized for touch-down PCR (Don et al., 1991) on the GAS strain CS101 genomic DNA template, only oligonucleotides OppD1 and OppD3 resulted in a single product. The 440 bp product was cloned into pUC18 and sequenced with M13 standard primers. Its nucleotide sequence exhibited 75.2% homology to the corresponding pneumococcal internal oppD sequence.

**Table 3. List of oligonucleotides used in this work.**

| Designation | Sequence (5′ to 3′)                          | Position numbers |
|-------------|----------------------------------------------|------------------|
| A.          |                                              |                  |
| oppD1       | GTT GGW GAR TCA GGW TCW GGK AA              | 5983–6005        |
| oppD3       | ARM GCA GTT WTY GGC TCA TC                   | 6422–6403        |
| B.          |                                              |                  |
| opp40       | TGT GGA TGA TTT CTT AGT A                   | 7–25             |
| opp51       | ACG GCT GAT GGT GAA GAA                     | 1703–1720        |
| opp54       | TAG ATG ATT TTG AGG TGA AGG                 | 2069–2049        |
| opp33       | TTA TCT TTA AAC AAG ACC CTA                 | 3520–3540        |
| opp26       | TAG GTA TTT TTA AAC CGA GC                  | 4354–4335        |
| opp27       | TGC AGT TAA AAA TGC TAA TTT C               | 6985–7005        |
| opp18       | CGC GTC AAA TGC TCT GAT A                   | 7353–7335        |
| C.          |                                              |                  |
| opp mut FOR | TGC TTA TTA TCA TTG TTT TGA                 | 5382–5403        |
| opp mut REV | GTA TAG GGG TGT CTT GGA                     | 6620–6603        |
| D.          |                                              |                  |
| oppA for MCS1 | GGA CGG TAT TTT TGG TCT                    | 30–47            |
| oppA rev MCS1 | AAT TGC TAA ATG AAA TTG GT                 | 1344–1325        |
| oppA for MCS2 | CCT CAA AAT CAT CTA TGG AA                 | 2055–2074        |
| oppA rev MCS2 | AAT TGC TGC TTC ATC                         | 3049–3032        |

Oligonucleotides were used as degenerate primers for PCR-amplification of an oppD-internal fragment (A), as primers to PCR-amplify probes directed to opp-derived mRNA (B), an oppCD-derived fragment to be cloned into pSF152 for insertional inactivation of the oppDF genes (C), or oppA-flanking fragments to be cloned into pFW6 for allelic replacement of an oppA portion (D). Primer pairs oppD1/oppD3 and opp mut FOR/opp mut REV were 5′-extended by BamHI and EcoRI sites, respectively, to facilitate forced cloning of the resulting PCR products. Primer pairs oppA for/rev MCS1 and oppA for/rev MCS2 were 5′-extended by BamHI/Xbal and EcoRI/PinAI sites, respectively, for a forced cloning of the PCR products into the multiple cloning sites of pFW6 (Podbielski et al., 1996b). Primer pairs opp51/opp54 and opp27/opp18 were directed to oppA and oppF gene-internal sequences, respectively. The position numbers refer to the GAS opp operon sequence as submitted to the EMBL Data Library.

1096 A. Podbielski et al.

New primers designed to anneal within the sequenced GAS oppD were used to generate a Dig-dUTP-labelled PCR product (Podbielski et al., 1995). Applying this PCR product as a probe, hybridizing recombinant lambda phages were lifted off the CS101 genomic lambda library and, together with ExAssist helper phages, were used to transfect E. coli XLOLR according to the Stratagene manual.

The template for PCR sequencing was prepared by suspending the recombinant lambda phages in extraction buffer, boiling for 5 min, sedimenting the debris by centrifugation, and using 5 μl of the supernatant in PCR reactions. The fragments were initially amplified by PCR and sequenced using the improved T3 and T7 primers (Stratagene) directed to conserved lambda-phage sequences adjacent to the recombinant fragments. Sequencing was continued on recombinant pBK plasmids by a 'genome-walking' strategy.

Oligonucleotides were designed with the aid of OLIGO 5.0 (National Biosciences), synthesized on an OLIGO 1000 DNA Synthesizer (Beckman) and desalted through NAP5 columns (Pharmacia). The parameters of PCR assays, direct labelling of PCR products with Dig-dUTP, analysis of PCR products, and parameters for direct sequencing of PCR products were as previously described (Podbielski et al., 1995).

DNA sequences were compiled and analysed with PC GENE 6.8 (IntelliGenetics). Sequence comparisons were performed using the BLAST program and databases of the GenBank Data Library.

transcripts were analysed by Northern blot hybridization using directly labelled PCR products as probes (for primers, see Table 3).

Allelic replacement of an oppA-internal fragment was performed by using a double-recombinant pFW6 plasmid according to the description of Podbielski et al. (1996b). The replacement of the oppA sequence by the spectinomycin-resistance gene aad9 in the resulting mutant was confirmed by Southern blot hybridization utilizing its Spel- and Xhol-digested genomic DNA as well as the oppA- and aad9-specific probes (Table 3). Further confirmatory PCR assays (Podbielski et al., 1996b) were conducted using primers directed to the aad9 gene (Table 3) and to flanking genomic sequences not affected by the double cross-over event (opp40 and opp26, Table 3).

Binding of labelled serum proteins to whole GAS cells

Peroxidase labelling of human serum albumin, fibrinogen, fibronectin, vitronectin, IgG, and IgA (Sigma) was done according to the protocol of Schmidt and Wadström (1990). GAS cells grown to an OD<sub>600</sub> of 0.7 were washed twice in saline and adjusted to a final concentration of 10<sup>8</sup> cells ml<sup>-1</sup> as determined by colony counts. Serial twofold dilutions (100 μl aliquots) were dot-blotted onto nitrocellulose strips. Binding of labelled serum proteins, visualization, and quantification were carried out as outlined by Schmidt et al. (1993).

RNA preparation and analysis

RNA was prepared from serotype M2 and M49 GAS strains at OD<sub>600</sub> values of 0.2 and 0.7, which correspond to early and late logarithmic growth phases, respectively. RNA preparation, denaturing agarose gel electrophoresis, and Northern blotting with Dig-dUTP PCR product probes were performed as previously described (Podbielski et al., 1995). Hybridization band intensities on X-ray film exposures were quantified by densitometry with an Epson GT-9000 flat-bed scanner and the Biodoc analysis system (Biometra). The results reported are the result of three independent experiments, each utilizing a new RNA preparation to minimize the influence of preparation parameters on semi-quantitative results. The primers used for generation of the PCR probes are shown in Table 3.

Adherence assay

Eukaryotic cells used in these studies included the cell lines HEP-2 (human larynx epidermoid carcinoma), HCT-8 (human ileocaecal adenocarcinoma), LLC-PK1 (porcine renal tubular cells) (ATCC) and endothelial cells isolated from human umbilical veins (HUVECs; Müller and Weigl, 1992). The eukaryotic cells were cultured to confluence in 96-well, flat-bottom, tissue-culture plates (Falcon) and were fixed with 1.25% glutaraldehyde for 15 min at room temperature. For testing bacterial adherence to immobilized fibronectin, 96-well plates were coated with 50 mg l<sup>-1</sup> fibronectin from human plasma (Sigma), reconstituted in phosphate-buffered saline (PBS). Plates were incubated with fibronectin for 24 h at room temperature and subsequently blocked with 5% bovine serum albumin (Sigma) for 3 h at 37°C. All wells, fibronectin coated or confluent monolayers, were washed with PBS and 100 μl of PBS containing 10<sup>7</sup> streptococci grown to an OD<sub>600</sub> of 0.7 were added to each well. Since the streptococci spontaneously clump, the bacterial suspensions were sonicated to obtain single cells. After incubation for 60 min at 37°C, the non-adherent bacteria were removed by washing with PBS. The number of adherent bacteria was determined by using the enzyme-linked immunosorbent assay (ELISA) with 2,2-acino-bis(3-ethylbenz-thiazoline-6-sulphonic acid) (Sigma) as substrate. The colour reaction was allowed to develop for 20 min and was read at 405 nm with an ELISA plate reader (Milenia Kinetic Analyser, Molecular Devices). Each strain was tested in at least four independent experiments each containing six replicate wells. Statistical evaluation was performed using the Student's t test.

DNA-mutagenesis experiments

Insertional inactivation of the oppD/F genes was performed using a recombinant pSF152 plasmid following the strategy and specific methods previously described (Podbielski et al., 1995; 1996a). The specific primers used to generate the PCR product cloned into pSF152 in this study are listed in Table 3. Specific integration of the plasmid was confirmed by Southern blot hybridization utilizing BamHI-, Spel-, and Xbal-digested genomic DNA and probes specific for the integrated antibiotic-resistance marker (aad9; Tao et al., 1995) as well as oppA- and oppF-specific probes (Table 2). In addition, mutant chromosomal DNA was subjected to confirmatory PCR using primers Seq7 (specific to the integrated vector; Podbielski et al., 1995) and opp33 (Table 3). Specific

© 1996 Blackwell Science Ltd, *Molecular Microbiology*, **21**, 1087–1099

Group A streptococcal oligopeptide permease 1097

solution (30% v/v ethanol, 10% v/v acetic acid) removed protein products and FITC label liberated by any protease activity from the gel. The gels were then incubated in 40 mM Tris, pH 8.5, for 30 min at room temperature to enhance FITC-derived fluorescence and examined on a UV transilluminator (305 nm). Any protein bands demonstrating proteolytic activity for casein would appear dark on a uniform light background of FITC-labelled casein.

### Acknowledgements

We would like to thank A. Flosdorff and J. Weber-Heynemann for expert technical assistance, as well as W. Bollmann for typing. The work of A.P., K.-H.S., and E.R. was supported by DFG grants Po391/6–1, Sch877/1–2, and Ro977/2–1, respectively, and that of B.A.B.L. by Public Health Service Grant 1F32 AI08742.

### References

Alloing, G., Trombe, M.C., and Claverys, J.P. (1990) The *ami* locus of the Gram-positive bacterium *Streptococcus pneumoniae* is similar to binding protein-dependent transport operons of Gram-negative bacteria. *Mol Microbiol* **4**: 633–644.

Alloing, G., de Philip, P., and Claverys, J.P. (1994) Three highly homologous membrane-bound lipoproteins participate in oligopeptide transport by the Ami system of the Gram-positive *Streptococcus pneumoniae*. *J Mol Biol* **241**: 44–58.

Ausubel, F.M., Brent, R., Kingston, R.E., Moore, D.D., Seidman, J.G., Smith, J.A., and Struhl, K. (1990) *Current Protocols in Molecular Biology*. New York: Greene Publishing Associates and Wiley-Interscience.

Berge, A., and Björck, L. (1995) Streptococcal cysteine proteinase releases biologically active fragments of streptococcal surface proteins. *J Biol Chem* **270**: 9862–9867.

Caparon, M.G., and Scott, J.R. (1991) Genetic manipulation of pathogenic streptococci. *Meth Enzymol* **204**: 556–586.

Chaussee, M.S., Gerlach, D., Yu, C.E., and Ferretti, J.J. (1993) Inactivation of the streptococcal erythrogenic toxin B gene (*speB*) in *Streptococcus pyogenes*. *Infect Immun* **61**: 3719–3723.

Cundell, D.R., Pearce, B.J., Sandros, J., Naughton, A.M., and Masure, H.R. (1995) Peptide permeases from *Streptococcus pneumoniae* affect adherence to eukaryotic cells. *Infect Immun* **63**: 2493–2498.

Davies, H.C., Karush, F., and Rudd, J.H. (1965) Effect of amino acids on steady-state growth of a group A hemolytic streptococcus. *J Bacteriol* **89**: 421–427.

Don, R.H., Cox, P.T., Wainwright, B.J., Baker, K., and Mattick, J.S. (1991) ‘Touchdown’ PCR to circumvent spurious priming during gene amplification. *Nucl Acids Res* **19**: 4008.

Ferretti, J.J., Huang, T.T., Hynes, W.L., Malke, H., Simon, D., Suvorov, A., and Yu, C.E. (1991) Extracellular product genes of group A streptococci. In *Genetics and Molecular Biology of Streptococci, Lactococci, and Enterococci*. Dunn, G.M., Cleary, P.P., and McKay, L.L. (eds). Washington, DC: American Society for Microbiology, pp. 201–205.

© 1996 Blackwell Science Ltd, *Molecular Microbiology*, **21**, 1087–1099

Fink, P.S. (1991) Using sodium chloride step gradients to fractionate DNA fragments. *BioTechniques* 10: 447–449.

Fleischmann, R.D. *et al.* (1995) Whole-genome random sequencing and assembly of *Haemophilus influenzae* Rd. *Science* 269: 496–512.

Foucaud, C., Kunji, E.R.S., Hagting, A., Richard, J., Konings, W.N., Desmazeaud, M., and Poolman, B. (1995) Specificity of peptide transport systems in *Lactococcus lactis*: evidence for a third system which transports hydrophobic di- and tripeptides. *J Bacteriol* 177: 4652–4657.

Gerlach, D., Reichardt, W., Fleischer, B., and Schmidt, K.H. (1994) Separation of mitogenic and pyrogenic activities from so-called erythrogenic toxin type B (streptococcal proteinase). *Zbl Bakt* 280: 507–514.

Hagting, E., Kunji, E.R.S., Leenhouts, K.J., Poolman, B., and Konings, W.N. (1994) The di- and tripeptide transport protein of *Lactococcus lactis*: a new type of bacterial transporter. *J Biol Chem* 269: 11391–11399.

Hauser, A.R., and Schlievert, P.M. (1990) Nucleotide sequence of the streptococcal pyrogenic exotoxin type B gene and relationship between the toxin and the streptococcal proteinase precursor. *J Bacteriol* 172: 4536–4542.

Hiles, I.A., Gallagher, M.P., Jamieson, D.J., and Higgins, C.F. (1987) Molecular characterization of the oligopeptide permease of *Salmonella typhimurium*. *J Mol Biol* 195: 125–142.

Jenkinson, H.F. (1992) Adherence, coaggregation, and hydrophobicity of *Streptococcus gordonii* associated with expression of cell surface lipoproteins. *Infect Immun* 60: 1225–1228.

Jenkinson, H.F., and Easingwood, R.A. (1990) Insertional inactivation of the gene encoding a 76-kilodalton cell surface polypeptide in *Streptococcus gordonii* Challis has a pleiotropic effect on cell surface composition and properties. *Infect Immun* 58: 3689–3697.

Jenkinson, H.F., Baker, R.A., and Tannock, G.W. (1996) A binding-lipoprotein-dependent oligopeptide transport system in *Streptococcus gordonii* essential for uptake of hexa- and heptapeptides. *J Bacteriol* 178: 68–77.

Kapur, V., Majesky, M.W., Li, L.L., Black, R.A., and Musser, J.M. (1993a) Cleavage of interleukin 1β (IL-1β) precursor to produce active IL-1β by a conserved extracellular cysteine protease from *Streptococcus pyogenes*. *Proc Natl Acad Sci USA* 90: 7676–7680.

Kapur, V., Topouzis, S., Majesky, M.W., Li, L.L., Hamrick, M.R., Hamill, R.J., Patti, J.M., and Musser, J.M. (1993b) A conserved *Streptococcus pyogenes* extracellular cysteine protease cleaves human fibronectin and degrades vitronectin. *Microb Pathogen* 15: 327–346.

Kashiwagi, K., Yamaguchi, Y., Sakai, Y., Kobayashi, H., and Igarashi, K. (1990) Identification of the polyamine-induced protein as a periplasmic oligopeptide binding protein. *J Biol Chem* 265: 8387–8391.

Kehoe, M.A. (1994) Cell-wall-associated proteins in Gram-positive bacteria. *New Comph Biochem* 27: 217–261.

Kunji, E.R.S., Hagting, A., de Vries, C.J., Juillard, V., Haandrikman, A.J., Poolman, B., and Konings, W.N. (1995) Transport of β-casein-derived peptides by the oligopeptide transport system is a crucial step in the proteolytic pathway of *Lactococcus lactis*. *J Biol Chem* 270: 1569–1574.

Leonard, B.A.B., Podbielski, A., Hedberg, P.J., and Dunny, G.M. (1996) *Enterococcus faecalis* pheromone binding protein, PrgZ, recruits a chromosomal oligopeptide permease system (Opp) to import sex pheromone pCF10 for induction of conjugation. *Proc Natl Acad Sci USA* 93: 260–264.

Magnuson, R., Solomon, J., and Grossman, A.D. (1994) Biochemical and genetic characterization of a competence pheromone from *B. subtilis*. *Cell* 77: 207–216.

Martin, N.J., Kaplan, E.L., Gerber, M.A., Menegus, M.A., Randolph, M., Bell, K., and Cleary, P.P. (1990) Comparison of epidemic and endemic group G streptococci by restriction enzyme analysis. *J Clin Microbiol* 28: 1881–1886.

Marugg, J.D., Meijer, W., van Kranenburg, R., Laverman, P., Bruinenberg, P.G., and De Vos, W.M. (1995) Medium-dependent regulation of proteinase gene expression in *Lactococcus lactis*: control of transcription initiation by specific dipeptides. *J Bacteriol* 177: 2982–2989.

Mierau, I., Kunji, E.R.S., Leenhouts, K.J., Hellendoorn, M.A., Haandrikman, A.J., Poolman, B., Konings, W.N., Venema, G., and Kok, J. (1996) Multiple-peptidase mutants of *Lactococcus lactis* are severely impaired in their ability to grow in milk. *J Bacteriol* 178: 2794–2803.

Müller, W.A., and Weigl, S.A. (1992) Monocyte-selective transendothelial migration: dissection of the binding and transmigration phases by an in vitro assay. *J Exp Med* 176: 819–828.

Musser, J.M., Stockbauer, K., Kapur, V., and Rudgers, G.W. (1996) Substitution of cysteine 192 in a highly conserved *Streptococcus pyogenes* extracellular cysteine protease (interleukin 1β convertase) alters proteolytic activity and ablates zymogen processing. *Infect Immun* 64: 1913–1917.

National Committee for Clinical Laboratory Standards (1993) *Methods for Dilution Antimicrobial Susceptibility Tests for Bacteria that Grow Aerobically*, 3rd edn. Approved Standard M7-A3. Villanova, PA.: National Committee for Clinical Laboratory Standards.

Ohara-Nemoto, Y., Sasaki, M., Kaneko, M., Nemoto, T., and Ota, M. (1994) Cysteine protease activity of streptococcal pyrogenic exotoxin B. *Can J Microbiol* 40: 930–936.

Park, J.T. (1995) Why does *Escherichia coli* recycle its cell wall peptides. *Mol Microbiol* 17: 421–426.

Perego, M., Higgins, C.F., Pearce, S.R., Gallagher, M.P., and Hoch, J.A. (1991) The oligopeptide transport system of *Bacillus subtilis* plays a role in the initiation of sporulation. *Mol Microbiol* 5: 173–185.

Podbielski, A., Flosdorff, A., and Weber-Heynemann, J. (1995) The group A streptococcal *virR49* gene controls expression of four structural *vir* regulon genes. *Infect Immun* 63: 9–20.

Podbielski, A., Schnitzler, N., Beyhs, P., and Boyle, M.D.P. (1996a) M-related protein (Mrp) contributes to group A streptococcal resistance to phagocytosis by human granulocytes. *Mol Microbiol* 19: 429–441.

Podbielski, A., Spellerberg, B., Woischnik, M., Pohl, B., and Lütticken, R. (1996b) Novel series of plasmid vectors for gene inactivation and expression analysis in group A streptococci (GAS). *Gene*, in press.

Raeder, R., and Boyle, M.D.P. (1995) Properties of IgG-binding proteins expressed by *Streptococcus pyogenes*

isolates are predictive of invasive potential. *Clin Diagn Lab Immunol* **2**: 484–486.

Rudner, D.Z., LeDeaux, J.R., Ireton, K., and Grossman, A.D. (1991) The *spoOK* locus of *Bacillus subtilis* is homologous to the oligopeptide permease locus and is required for sporulation and competence. *J Bacteriol* **173**: 1388–1398.

Sabelnikov, A.G., Greenberg, B., and Lacks, S.A. (1995) An extended –10 promoter alone directs transcription of the *DpnII* operon of *Streptococcus pneumoniae*. *J Mol Biol* **250**: 144–155.

Schmidt, K.H., and Wadström, T. (1990) A secreted receptor related to M1 protein of *Streptococcus pyogenes* binds to fibrinogen, IgG, and albumin. *Zbl Bakt* **273**: 216–228.

Schmidt, K.H., Wiesner, J., Gerlach, D., Reichardt, W., Ozegowski, J.H., and Köhler, W. (1993) Susceptibility of chicken embryos to group A streptococci: correlation with fibrinogen binding. *FEMS Immunol Med Microbiol* **7**: 231–240.

Schuster, C., Dobrinski, B., and Hakenbeck, R. (1990) Unusual septum formation in *Streptococcus pneumoniae* mutants with an alteration in the D,D-carboxypeptidase penicillin-binding protein 3. *J Bacteriol* **172**: 6499–6505.

Shanley, T.P., Schrier, D., Kapur, V., Kehoe, M., Musser, J.M., and Ward, P.A. (1996) Streptococcal cysteine protease augments lung injury induced by products of group A streptococci. *Infect Immun* **64**: 870–877.

Solomon, J.M., Magnuson, R., Srivastava, A., and Grossman, A.D. (1995) Convergent sensing pathways mediate response to two extracellular competence factors in *Bacillus subtilis*. *Gene Dev* **9**: 547–558.

Talkington, D.F., Schwartz, B., Black, C.M., Todd, J.K.,

Elliott, J., Breiman, R.F., and Facklam, R.R. (1993) Association of phenotypic and genotypic characteristics of invasive *Streptococcus pyogenes* isolates with clinical components of streptococcal toxic shock syndrome. *Infect Immun* **61**: 3369–3374.

Tam, R., and Saier, M.T. (1993) Structural, functional, and evolutionary relationships among extracellular solute-binding receptors of bacteria. *Microbiol Rev* **57**: 320–346.

Tame, J.R.H., Murshudov, G.N., Dodson, E.J., Neil, T.K., Dodson, G.G., Higgins, C.F., and Wilkinson, A.J. (1994) The structural basis of sequence-independent peptide binding by OppA protein. *Science* **264**: 1578–1581.

Tao, L., LeBlanc, D.J., and Ferretti, J.J. (1992) Novel streptococcal-integration shuttle vectors for gene cloning and inactivation. *Gene* **120**: 105–110.

Twining, S.S. (1984) Fluorescein isothiocyanate-labeled casein assay for proteolytic enzymes. *Anal Biochem* **143**: 30–34.

Tynkkynen, S., Buist, G., Kunji, E., Kok, J., Poolman, B., Venema, G., and Haandrikman, A. (1993) Genetic and biochemical characterization of the oligopeptide transport system of *Lactococcus lactis*. *J Bacteriol* **175**: 7523–7532.

van de Rijn, I., and Kessler, R.E. (1980) Growth characteristics of group A streptococci in a new chemically defined medium. *Infect Immun* **27**: 444–448.

Wolf, B.B., Gibson, C.A., Kapur, V., Hussaini, I.M., Musser, J.M., and Gonias, S.L. (1994) Proteolytically active streptococcal pyrogenic exotoxin B cleaves monocytic cell urokinase receptor and releases an active fragment of the receptor from the cell surface. *J Biol Chem* **269**: 30682–30687.
