
Glycopeptide resistance in enterococci

Michel Arthur, Peter Reynolds and Patrice Courvalin

This review is dedicated to the memory of Sandra Handwerger.

Glycopeptide antibiotics such as vancomycin and teicoplanin inhibit peptidoglycan synthesis and are used to treat severe infections caused by Gram-positive bacteria. Glycopeptides do not interact with cell wall biosynthetic enzymes but form complexes with peptidoglycan precursors and prevent their incorporation into the wall¹. Consequently, the activity of glycopeptides is not determined by the affinity of target enzymes for the molecules but by the substrate specificity of the enzymes that determine the structure of peptidoglycan precursors. Acquisition of glycopeptide resistance results from the transfer of mobile genetic elements that encode enzymes for synthesis of low-affinity precursors and elimination of the high-affinity precursors normally produced by the host. We shall first review the mode of action and the mechanism of resistance, as exemplified by VanA-type glycopeptide resistance mediated by transposon Tn1546 which is widely spread in enterococci. The diversity, origin and evolution of glycopeptide resistance will then be discussed.

Target of glycopeptides

Inhibition of cell wall synthesis by glycopeptides results from the formation of complexes between the antibiotics and the carboxy-terminal D-alanine (D-Ala) residues of peptidoglycan precursors¹ (Fig. 1). Glycopeptides do not penetrate into the cytoplasm and interaction with the target can only take place after translocation of the precursors bound to the undecaprenol lipid carrier to the outer surface of the cytoplasmic membrane. Binding of the drugs at this step blocks incorporation of disaccharide pentapeptide subunits into nascent peptidoglycan by transglycosylation and leads to accumulation of cytoplasmic precursors (Fig. 2a). Binding of glycopeptides to peptide stems ending in D-Ala-D-Ala, if present in nascent peptidoglycan, is also expected to inhibit the reactions catalysed by D,D-transpeptidases and D,D-carboxypeptidases (Fig. 2b).

Glycopeptide resistance in enterococci results from the production of peptidoglycan precursors with low affinity for these antibiotics. The mobility of the resistance genes by transposition and conjugation and the ability of the resistance proteins to interfere with synthesis of normal precursors in different hosts indicate that dissemination into other bacterial species should be anticipated.

M. Arthur* and P. Courvalin are in the Unité des Agents Antibactériens, Centre National de la Recherche Scientifique EP J0058, Institut Pasteur, 75724 Paris, Cedex 15, France; P. Reynolds is in the Dept of Biochemistry, University of Cambridge, Cambridge, UK CB2 1QW, but temporarily at the Institut Pasteur. *tel: +33 1 40 61 35 88, fax: +33 1 45 68 83 19, e-mail: Michel.Arthur@pasteur.fr

Synthesis of the target and basis for resistance

The D-Ala residues that interact with glycopeptides are incorporated into peptidoglycan precursors as a dipeptide synthesized by D-alanine:D-alanine ligases² (Fig. 3). The substrate specificity of ligases determines the nature of the residues incorporated at the carboxy-terminal end of peptidoglycan precursors and the possibility for subsequent interaction with glycopeptides at the cell surface (Table 1).

Inducible resistance to high levels of vancomycin and teicoplanin (VanA phenotype) in enterococci is mediated by Tn1546 or closely related elements³. The transposon encodes a dehydrogenase (VanH) that reduces pyruvate to D-lactate (D-Lac), and a ligase of broad substrate specificity (VanA) that catalyses the formation of an ester bond between D-Ala and D-Lac⁴,⁵ (Table 1). The resulting D-Ala-D-Lac depsipeptide replaces the dipeptide D-Ala-D-Ala in the pathway of peptidoglycan synthesis (Fig. 3). The substitution eliminates a hydrogen bond critical for antibiotic binding⁵ (Fig. 1 and Table 1). The normal complement of cell wall biosynthetic enzymes encoded by the host chromosome tolerates the substitution of D-Lac for D-Ala⁶. However, penicillin-binding proteins (PBPs) that catalyse the final transpeptidation reactions in the assembly pathway may display different activity for D-Ala- and D-Lac-terminating precursors which would account for modification of the extent of cross-linking⁷,⁸. Induction of glycopeptide resistance is associated with an increase in susceptibility to β-lactams in certain strains of enterococci⁹. It has been proposed that PBP5 cannot perform cross-linking of D-Lac-terminating precursors⁹. Consequently, other PBPs would be required for cell wall synthesis leading to an increase in the susceptibility to β-lactams since these PBPs are inhibited by lower concentration of β-lactams than PBP5 (Ref. 9).

Elimination of D-Ala-containing precursors

Coproduction of precursors ending in D-Ala or D-Lac does not result in resistance¹⁰. Under these conditions, binding of glycopeptides to D-Ala-D-Ala-containing precursors bound to the lipid carrier at the external

Copyright © 1996 Elsevier Science Ltd. All rights reserved. 0966 842X/96/$15.00 PII: S0966-842X(96)10063-9

TRENDS IN MICROBIOLOGY 401 VOL. 4 NO. 10 OCTOBER 1996

Fig. 1. Structure of the glycopeptide: peptidyl-D-Ala-D-Ala complex. (a) Binding of a glycopeptide to the D-Ala-D-Ala extremity of peptidoglycan precursors involves five hydrogen interactions indicated by dashed lines. The NH group of the amide D-Ala-D-Ala linkage that is substituted by an oxygen in D-Ala-D-Lac-containing peptidoglycan precursors is indicated in red. The methyl (CH₃) side chain of the carboxy-terminal D-Ala that is substituted by a hydroxymethyl (CH₂OH) side chain in precursors ending in D-Ala-D-Ser is indicated in blue. (b) Space-filling models showing the bulky glycopeptide molecule bound to the precursor. Formation of the complex is thought to inhibit transglycosylases by steric hindrance, and penicillin binding proteins (PBPs) by directly preventing interaction with the carboxy-terminal D-Ala residues. Adapted, with permission, from Ref. 41.

surface of the membrane (Fig. 3) is likely to be critical for the inhibition of peptidoglycan synthesis. Formation of the glycopeptide–D-Ala-D-Ala complex would sequester the lipid carrier and thereby prevent translocation of additional precursors, including those terminated by D-Lac. Such inhibition is prevented by production of a D,D-dipeptidase encoded by Tn1546 (VanX) that hydrolyses D-Ala-D-Ala and drastically reduces synthesis of precursors ending in D-Ala⁶.

**Accessory proteins**

Tn1546 also encodes two accessory proteins (VanY and VanZ) that are not essential for resistance but their production increases the level of resistance to vancomycin and teicoplanin, respectively¹⁰. VanY is a D,D-carboxypeptidase that hydrolyses the terminal D-Ala residue of late peptidoglycan precursors that are produced if elimination of D-Ala-D-Ala by VanX is not complete¹¹. VanZ alone confers teicoplanin resistance by an unknown mechanism that does not involve incorporation of a substituent of D-Ala-D-Ala into peptidoglycan precursors¹².

**Regulation of glycopeptide resistance**

Synthesis of VanH, VanA and VanX is regulated at the transcriptional level by the VanR–VanS two-component regulatory system¹³. VanS is a putative membrane-associated sensor that controls the level of phosphorylation of VanR. VanR is an activator required for

(a)

D-Ala
|
D-Ala
|
X-L-Lys
|
D-Glu
|
L-Ala

P
P

D-Ala
|
D-Ala
|
X-L-Lys
|
D-Glu
|
L-Ala

P
P

Transglycosylase

D-Ala
|
D-Ala
|
X-L-Lys
|
D-Glu
|
L-Ala

P
P

C₅₅ lipid carrier (undecaprenol)
N-acetylglucosamine
N-acetylmuramic acid

Cytoplasmic membrane

(b)

D-Ala
|
HC-CH₃
|
NH
|
O=C
|
HC-CH₃
|
NH
|
H₂N-[X]-L-Lys
|
D-Glu
|
L-Ala
|
R

PBP
Ser
O
O=C
HC-CH₃
NH
H₂N-[X]-L-Lys
D-Glu
L-Ala
R

Carboxypeptidase
Tetrapeptide

COOH
|
HC-CH₃
|
NH
|
O=C-NH-[X]-L-Lys
|
HC-CH₃
|
NH
|
D-Glu
|
L-Ala
|
R

Transpeptidase
H₂N-[X]-L-Lys
|
D-Glu
|
L-Ala
|
R

Fig. 2. (a) Reaction catalysed by the transglycosylases at the outer surface of the cytoplasmic membrane. The new subunit is depicted in red, the existing nascent peptidoglycan in blue. (b) Reactions catalysed by the penicillin-binding proteins (PBPs). The first step involves nucleophilic attack of the carboxyl group of the penultimate D-Ala residue by the hydroxyl group of the active site serine, leading to formation of the acyl enzyme and release of the carboxy-terminal D-Ala. Note that acyl enzymes derived from D-Ala- and D-Lac-ending precursors are identical. If the PBP has D,D-carboxypeptidase activity, the acyl enzyme is hydrolysed and tetrapeptide is released. If the PBP has transpeptidase activity, there is nucleophilic attack of the acyl enzyme by the NH₂ group of the amino-terminal amino acid in the interpeptide bridge (X) leading to the formation of a peptide bond. The latter reaction results in peptidoglycan cross-linking. The amino acids of the interpeptide bridge are linked to the ε-NH₂ group of lysine and differ between species⁴⁰. The interpeptide bridge of *Enterococcus faecalis* and *E. faecium* consists of one D-asparagine or two L-Ala residues, respectively⁷,⁸.

transcription of an operon encoding VanH, VanA and VanX. Several phosphotransfer reactions are catalysed *in vitro* by purified VanR and VanS, including ATP-dependent autophosphorylation of a histidine residue in VanS, transfer of the phosphate group to an aspartate residue of VanR and spontaneous hydrolysis of phospho-VanR¹⁴. The latter reaction is stimulated by VanS¹⁴. Because phosphorylation increases the affinity of VanR for the DNA of the promoter of the *vanHAX* operon, it is likely that the actual transcriptional activator is the phosphorylated form of VanR, as found for related two-component regulatory systems¹⁵. This would imply that the VanS sensor displays increased kinase and/or reduced phosphatase activity in response to the presence of glycopeptides in the culture medium. The nature of the signal recognized by VanS remains unknown. The fact that moenomycin, a transglycosylase inhibitor that is structurally unrelated to glycopeptides, also induces glycopeptide resistance suggests that accumulation of peptidoglycan precursors rather than direct interaction with the drug could be responsible for induction¹⁶.

Relationship between the *vanA* and *vanB* gene clusters

Acquired resistance to glycopeptides in enterococci is mediated by two classes of related gene clusters that confer inducible resistance to high levels of vancomycin and teicoplanin (*vanA*) or inducible resistance to various levels of vancomycin (*vanB*) (Fig. 4). In the latter case, the strains remain susceptible to teicoplanin since this antibiotic is not an inducer¹⁰,¹⁷. Both types of gene clusters mediate glycopeptide resistance by synthesis of peptidoglycan precursors ending in D-Lac¹⁸,¹⁹.

Comparison of the *vanA* and *vanB* gene clusters indicates a high level of amino acid sequence identity in the dehydrogenases (VanH and VanHₐ), ligases (VanA and VanB) and D,D-dipeptidases¹⁷ (VanX and VanXₐ; Fig. 4). The VanA and VanB ligases display similar catalytic properties²⁰ (Table 1). In contrast, the VanR–VanS and VanRₐ–VanSₐ two-component regulatory systems are only distantly related and the putative sensor domains of VanS and VanSₐ are not related in sequence, raising the possibility that these proteins sense the presence of glycopeptides by different mechanisms.

REVIEW S

Wall

Membrane

Cytoplasm

Cross-linking  
(position b) or  
hydrolysis  
(a, c, and d)

Vancomycin binding inhibits  
transglycosylation. This inhibition  
leads in turn to phosphorylation of  
VanR by VanS and activation of  
transcription of vanH, vanA and  
vanX. Note the sequestration of  
the lipid carrier

Incorporation of disaccharide  
pentadepsipeptide subunits in  
nascent peptidoglycan by  
transglycosylation

Acylenzyme  
formation  
(release of D-Lac)

Vancomycin does not  
bind

VanS

VanR

VanH VanA

Elimination  
of  
D-Ala-D-Ala

Production of  
VanX  
VanH  
VanA

(activator)  
VanR-P

Turned on

PP-○○○○■  
UDP  
UDP-○  
UMP  
P  
UDP-○○○○■  

PP-○○○○■  
UDP  
UDP-○  
UMP  
P  
UDP-○○○○■  

PP-○○○○■  
UDP  
UDP-○  
UMP  
P  
UDP-○○○○■  

Pi

TR ENDS IN MICROBIOLOGY 404 VOL. 4 NO. 10 OCTOBER 1996

Fig. 3. Peptidoglycan synthesis in glycopeptide-resistant enterococci. The D-Ala residues that interact with vancomycin are incorporated into peptidoglycan precursors as a dipeptide synthesized by D-alanine:D-alanine ligases (Ddl). Formation of the complex between glycopeptides and peptidoglycan precursors at the cell surface prevents transglycosylation and leads to the accumulation of cytoplasmic peptidoglycan precursors. VanR and VanS act as a signal transducing system that couples inhibition of peptidoglycan synthesis to induction of the transcription of the *vanHAX* operon in the cytoplasm. Production of the corresponding enzymes turns on the pathway for synthesis of D-Lac-containing precursors (VanH and VanA) and turns off the pathway for synthesis of D-Ala-ending precursors by elimination of D-Ala-D-Ala (VanX). Analysis of the structure of the wall from resistant *Enterococcus faecium* and *E. faecalis* did not reveal any peptide stems ending in D-Lac, indicating that this residue is probably cleaved by the D,D-carboxypeptidases encoded by the host chromosome or by the resistance gene cluster<sup>7,8</sup> (*vanY* or *vanY<sub>B</sub>*). Resistance proteins are indicated in red and D-Lac in green.

The *vanA* and *vanB* clusters encode a D,D-carboxypeptidase (VanY or VanY<sub>B</sub>) but the proteins are not closely related and the genes are located at different positions of the clusters. VanZ (teicoplanin resistance) and VanW (unknown function), encoded respectively by the *vanA* and *vanB* clusters, do not seem to have counterparts in the other cluster. These observations and the variations in the base composition within the clusters suggest that genes from different origins have been recruited.

### Mobility of *vanA* and *vanB* gene clusters

The *vanA* gene cluster was originally detected on the nonconjugative transposon Tn1546, which belongs to the Tn3 family<sup>3</sup>. VanA-type resistance in clinical isolates of enterococci is mediated by genetic elements closely related to Tn1546 that are generally carried by self-transferable plasmids<sup>21</sup> and, occasionally, by the host chromosome as part of larger conjugative elements<sup>22</sup>. Tn1546-like elements are highly conserved except for the presence of insertion sequences that have transposed into intergenic regions not essential for the expression of glycopeptide resistance<sup>22</sup>. Conjugal transfer of plasmids that have acquired Tn1546-like elements by transposition appears to be responsible for the spread of glycopeptide resistance in enterococci<sup>3</sup>.

Gene clusters related to *vanB* are generally carried by large (90–250 kb) elements that are transferable by conjugation from chromosome to chromosome<sup>23</sup>. One of these elements (250 kb) was found to contain the composite transposon Tn1547 (64 kb) delineated by insertion sequences belonging to the IS256 family<sup>24</sup>. Plasmid-borne *vanB*-related gene clusters have also been detected in clinical isolates of enterococci<sup>25</sup>.

#### Intrinsic resistance

Enterococci belonging to the species *Enterococcus gallinarum*, *E. casseliflavus* and *E. flavescens* are intrinsically resistant to low levels of vancomycin but remain susceptible to teicoplanin. Resistance results from the production of peptidoglycan precursors ending in D-serine<sup>18,26</sup> (Table 1). Substitution of D-Ala by D-Ser at the carboxyl terminus of peptidoglycan precursor analogues lowers the affinity of the precursors for vancomycin with a relatively small change in the affinity for teicoplanin<sup>27</sup>. Intrinsically resistant enterococci appear to produce two types of ligases that synthesize D-Ala-D-Ser and D-Ala-D-Ala<sup>28</sup>. Production of peptidoglycan precursors ending in D-Ser was reported to be inducible by vancomycin at least in certain strains<sup>18</sup>.

Intrinsic resistance to high levels of vancomycin and teicoplanin due to production of peptidoglycan precursors ending in D-Lac is frequent in lactic acid bacteria including certain species belonging to the genera *Lactobacillus*, *Leuconostoc* and *Pediococcus*<sup>18,29</sup>. Comparison of the amino acid sequences of D-alanine:D-lactate and D-alanine:D-serine ligases from these organisms has not revealed any close relationship with the VanA or VanB ligases<sup>30</sup>.

### Table 1. Substrate specificity of ligases and affinity of glycopeptides for the corresponding precursors

| Enzyme       | Substrate (carboxy-terminus) | $K_{\text{m}}$ (mm) | $K_{\text{cat}}/K_{\text{m}}$ ($M^{-1}s^{-1}$) | Decrease in affinity |
|--------------|-------------------------------|---------------------|-----------------------------------------------|-----------------------|
| Ddl (*E. coli*) | D-Ala X=NH<sub>2</sub> Y=CH<sub>3</sub> | 1.2 | 14000 | Not applicable |
|              | (D-Lac is not a substrate)    |                     |                                               |                       |
| VanA         | D-Ala X=NH<sub>2</sub> Y=CH<sub>3</sub> | 38 | 129 | Not applicable |
|              | D-Lac X=OH Y=CH<sub>3</sub>   | 7.1 | 220 | >1000-fold |
|              | D-HBut X=OH Y=CH<sub>2</sub>CH<sub>3</sub> | 0.6 | 3000 | No detectable binding |
| VanB         | D-Ala X=NH<sub>2</sub> Y=CH<sub>3</sub> | 34 | 120 | Not applicable |
|              | D-Lac X=OH Y=CH<sub>3</sub>   | 11 | 41 | >1000-fold |
|              | D-HBut X=OH Y=CH<sub>2</sub>CH<sub>3</sub> | 3.0 | 84 | No detectable binding |
| VanC         | D-Ser X=NH<sub>2</sub> Y=CH<sub>2</sub>OH | Not determined | Not determined | 7-fold |

The ligases catalyse peptide or ester bond formation between D-Ala and D-amino acids (X=NH<sub>2</sub>) or D-2-hydroxy acids (X=OH), respectively (D-HBut, D-2-hydroxybutyrate). Differences in specificity also involve the side chain (Y) of the substrate. These differences account for resistance, since substitution of NH in a peptide bond by O in an ester bond prevents formation of one of the five hydrogen interactions, whereas bulky side chains destabilize the complex<sup>5,27,38</sup>.

(a)

Tn1546 (vanA)

![Diagram](#)

IR_L
ORF1    ORF2    vanR    vanS    vanH    vanA    vanX    vanY    vanZ
37      34      42      41      45      45      44      34      29

Tn1547 (vanB)

![Diagram](#)

IS256-like    vanR_B    vanS_B    vanY_B    vanW    vanH_B    vanB    vanX_B    IS16
47            47         47        49        46      51       49      47

(b)

Regulation
Response regulator (34%)
(VanR or VanR_B)
Sensor (23%)
(VanS or VanS_B)

Synthesis of d-Ala-d-Lac
Dehydrogenase (67%)
(VanH or VanH_B)
Ligase (76%)
(VanA or VanB)

Hydrolysis of precursors
D,D-Dipeptidase (71%)
(VanX or VanX_B)
D,D-Carboxypeptidase (30%)
(VanY or VanY_B)

Fig. 4. Comparison of the vanA and vanB gene clusters. (a) Transposon Tn1546 (10,851 bp) carries the vanA gene cluster and is delineated by 38-bp imperfect inverted repeats represented by arrowheads (IR_L and IR_R). Transposon Tn1547 (65 kb) carries the vanB gene cluster and is delineated by insertion sequences IS256-like and IS16 in direct orientation (represented by boxes). Open arrows indicate open reading frames (ORF1, transposase; ORF2, resolvase). The percentage of guanosine plus cytosine is indicated below each open reading frame. (b) Amino acid identity (%) between homologous proteins encoded by the vanA and the vanB gene clusters.

cocci have become important as nosocomial pathogens and as a reservoir of resistance genes^{31,32}. This is associated with increased difficulty in treating enterococcal infections and a risk of dissemination of glycopeptide resistance to more pathogenic bacteria such as staphylococci and streptococci (Table 2). There is no barrier to heterospecific expression or transfer of glycopeptide resistance genes among these bacteria in laboratory conditions, and the mobility of the vanA and vanB gene clusters by conjugation and transposition is expected to facilitate such transfer in natural conditions^{35,36}. The threat of dissemination of glycopeptide resistance in human pathogens has stimulated many debates on measures intended to limit the selective pressure resulting from the unnecessary use of glycopeptides in clinical medicine, especially oral administration of the drugs, since glycopeptides are not found in the intestinal tract following parenteral administration^{37}. The use of the glycopeptide antibiotic avoparcin as a growth promoter for pigs and chickens is also a cause for concern (Table 2).

The threat of dissemination of glycopeptide resistance

Glycopeptides, alone or in combination with amino-glycosides, often constitute the only therapeutic treatment for multiresistant strains of staphylococci, streptococci and enterococci^{31,32}. The emergence and dissemination of high-level resistance to glycopeptides in enterococci in the past decade has resulted in clinical isolates resistant to all antibiotics of proven efficacy^{33,34}. Although enterococci are not highly pathogenic, the incidence of glycopeptide resistance among clinical isolates is increasing and the enterococci have become important as nosocomial pathogens and as a reservoir of resistance genes^{31,32}. This is associated with increased difficulty in treating enterococcal infections and a risk of dissemination of glycopeptide resistance to more pathogenic bacteria such as staphylococci and streptococci (Table 2). There is no barrier to heterospecific expression or transfer of glycopeptide resistance genes among these bacteria in laboratory conditions, and the mobility of the vanA and vanB gene clusters by conjugation and transposition is expected to facilitate such transfer in natural conditions^{35,36}. The threat of dissemination of glycopeptide resistance in human pathogens has stimulated many debates on measures intended to limit the selective pressure resulting from the unnecessary use of glycopeptides in clinical medicine, especially oral administration of the drugs, since glycopeptides are not found in the intestinal tract following parenteral administration^{37}. The use of the glycopeptide antibiotic avoparcin as a growth promoter for pigs and chickens is also a cause for concern (Table 2).

Acknowledgements

P.R. was supported by a Fellowship from the Ciba Fellowship Trust.

References

1 Reynolds, P.E. (1989) Eur. J. Clin. Microbiol. Infect. Dis. 8, 943–950  
2 Wright, G.D. and Walsh, C.T. (1992) Acc. Chem. Res. 25, 468–473  
3 Arthur, M. et al. (1993) J. Bacteriol. 175, 117–127  
4 Dutka-Malen, S. et al. (1990) Mol. Genet. 224, 364–372  
5 Bugg, T.D.H. et al. (1991) Biochemistry 30, 10408–10415  
6 Reynolds, P.E. et al. (1994) Mol. Microbiol. 13, 1065–1070  
7 Boudewijn, L. et al. (1996) Antimicrob. Agents Chemother. 40, 863–869  
8 Billot-Klein, D. et al. (1996) Biochem. J. 313, 711–715  
9 Al-Obeid, S. et al. (1992) FEMS Microbiol. Lett. 91, 79–84  
10 Arthur, M. et al. (1996) Mol. Microbiol. 21, 33–44  
11 Arthur, M. et al. (1994) Antimicrob. Agents Chemother. 38, 1899–1903  
12 Arthur, M. et al. (1995) Gene 154, 87–92  
13 Arthur, M. et al. (1992) J. Bacteriol. 174, 2582–2591  
14 Wright, G.D. et al. (1993) Biochemistry 32, 5057–5063  
15 Holman, T.R. et al. (1994) Biochemistry 33, 4625–4631  
16 Handwerger, S. and Kolokathis, A. (1990) FEMS Microbiol. Lett. 70, 167–170  
17 Evers, S. and Courvalin, P. (1996) J. Bacteriol. 178, 1302–1309  
18 Billot-Klein, D. et al. (1994) J. Bacteriol. 176, 2398–2405  
19 Evers, S. et al. (1994) Gene 140, 97–102  
20 Meziane-Cherif, D. et al. (1994) FEBS Lett. 354, 140–142  
21 Leclercq, R. et al. (1988) New Engl. J. Med. 319, 157–161  

Questions for future research

- How do the VanS and VanS_B sensors recognize the presence of glycopeptides?
- What is the origin of the vanA and vanB gene clusters?
- Why has resistance to glycopeptides by enzymatic detoxification never developed?
- Is there a selective advantage associated with the production of d-Lac-ending peptidoglycan precursors in addition to glycopeptide resistance and why do lactobacilli, leuconostocs and pediococci use this pathway?
- What is the mechanism of VanZ-mediated teicoplanin resistance?
- Are the resistance proteins encoded by the vanA and vanB clusters potential targets for new antibiotics?
- How do glycopeptide-producing organisms protect themselves against the action of these molecules?

TRENDS IN MICROBIOLOGY 406 VOL. 4 NO. 10 OCTOBER 1996

Table 2. Glycopeptide-resistant enterococci:  
increase in incidence and measures to prevent spread of resistance

| Source of GRE | Selective pressure likely to result in acquisition | Measures to avoid or reduce incidence of GRE |
|---------------|-----------------------------------------------|--------------------------------------------|
| Hospitals<sup>a</sup> |  |  |
| New York: | 1989 | (a) Intensive use of glycopeptides: | (a) Limit use of glycopeptides to the treatment |
| 1 hospital | 1 case | (i) In immunosuppressed | of severe, life-threatening infections, e.g. |
|  | 1993 | patients | MRSA |
| USA: |  | (ii) In neutropenic patients | Examples of the increased use of |
| intensive care | 1989 |  | vancomycin: |
|  | 0.4%<sup>b</sup> | (iii) For decontamination of gut | (i) 1976–1985; 23-fold increase in sales |
|  | 1993 |  | worldwide |
|  | 13.6%<sup>b</sup> | (iv) For prophylactic use | (ii) 1982–1995; 23-fold increase in value |
|  |  |  | of sales in USA |
| USA: | 1993 |  | (iii) 1978–1992; 160-fold increase in use |
| general |  |  | in 1 USA hospital |
| European: | 1993 | (b) Use of long-term, broad-spectrum antibiotics | (b) Improve hygiene to prevent clonal spread |
| general | 0.7–1.0%<sup>b</sup> |  |  |
| Community (limited number of studies) |  |  | Any measure to reduce the reservoir of |
| UK: | 2%<sup>b</sup> (184 people investigated) | Not from previous treatment with glycopeptides: possibly from eating meat products | glycopeptide resistance genes (see above and below) |
| Belgium: | (i) 3.5%<sup>b</sup> (636 people investigated) |  |  |
|  | (ii) 27%<sup>b</sup> (40 people investigated) |  |  |
| Germany: | 12%<sup>b</sup> (100 people investigated) |  |  |
| Animal (limited number of studies)<sup>39</sup> |  |  | Avoid use of avoparcin or other glycopeptides |
| Broiler chickens (manure) |  | Inclusion of non-clinical glycopeptides in animal feed. | (cross-resistance to vancomycin and teicoplanin) as animal food additives for growth promotion. No definitive evidence for transfer of resistance from animal to human strains but use of avoparcin may increase the reservoir of glycopeptide resistance genes |
| Chicken carcasses |  | NB During 1993 in Denmark, more than 19000 kg avoparcin was used in animal feed compared with 22 kg vancomycin in human medicine |  |
| Chickens from retail outlets |  |  |  |
| Pigs |  |  |  |
| Not isolated from cattle or sheep |  |  |  |
| Meat products<sup>39</sup> | Minced pork: 5 of 13 samples from butchers in Germany | Contamination with faecal material at slaughter | Improvement of hygiene in slaughter houses. Avoidance of contamination of meat products with faecal material |

<sup>a</sup>Examples given of numerous investigations.  
<sup>b</sup>Percentages of enterococci examined that were resistant to glycopeptides.  
GRE, glycopeptide-resistant enterococci; MRSA, methicillin-resistant Staphylococcus aureus.

---

22 Handwerger, S. and Skoble, J. (1995) Antimicrob. Agents Chemother. 39, 2446–2453  
23 Quintiliani, R., Jr and Courvalin, P. (1994) FEMS Microbiol. Lett. 119, 359–364  
24 Quintiliani, R., Jr and Courvalin, P. (1996) Gene 172, 1–8  
25 Woodford, N. et al. (1995) J. Antimicrob. Chemother. 35, 179–184  
26 Reynolds, P.E. et al. (1994) Biochem. J. 301, 5–8  
27 Billot-Klein, D. et al. (1994) Biochem. J. 304, 1021–1022  
28 Navarro, F. and Courvalin, P. (1994) Antimicrob. Agents Chemother. 38, 1788–1793  
29 Handwerger, S. et al. (1994) J. Bacteriol. 176, 260–264  
30 Evers, S. et al. (1996) J. Mol. Evol. 42, 706–712  
31 Murray, B.E. (1990) Clin. Microbiol. Rev. 3, 46–65  

32 Woodford, N. et al. (1995) Clin. Microbiol. Rev. 8, 585–615  
33 Handwerger, S. et al. (1992) Clin. Infect. Dis. 14, 655–661  
34 Handwerger, S. et al. (1993) Clin. Infect. Dis. 16, 750–755  
35 Brisson-Noël, A. et al. (1990) Antimicrob. Agents Chemother. 34, 924–927  
36 Noble, W.C. et al. (1992) FEMS Microbiol. Lett. 93, 195–198  
37 Van der Auwera, P. et al. (1996) J. Infect. Dis. 173, 1129–1136  
38 Nieto, M. and Perkins, H.R. (1971) Biochem. J. 123, 789–803  
39 Klare, I. et al. (1995) Microb. Drugs Resist. 1, 265–272  
40 Schleifer, K.H. and Kandler, O. (1972) Bacteriol. Rev. 36, 407–477  
41 Jeffs, P.W. and Nisbet, L.J. (1988) in Antibiotic Inhibition of Bacterial Cell Surface Assembly and Function (Actor, P. et al., eds), pp. 509–530, American Society for Microbiology
