
The Plant Journal (1996) 10(5), 859–868

# Nuclear localization activity of phytochrome B

Koji Sakamoto${}^{1,\dagger}$ and Akira Nagatani${}^{1,2,*}$

${}^{1}$Laboratory for Photoperception and Signal Transduction, Frontier Research Program, Institute of Physical and Chemical Research (RIKEN), Hirosawa 2-1, Wako, Saitama 351-01, Japan, and ${}^{2}$Molecular Genetics Research Laboratory, The University of Tokyo, Hongo, Tokyo 113, Japan

## Summary

Phytochromes are soluble red/far-red-light photoreceptor proteins which mediate various photomorphogenic responses of plants. Despite much effort, the signal transduction mechanism of phytochrome has remained obscure. Phytochromes are encoded by a small multigene family in *Arabidopsis*. Among the members of the family, phytochrome A (*phyA*) and B (*phyB*) are the best characterized. *PhyB* contains putative nuclear localization signals within its C-terminal region. Transgenic *Arabidopsis* plants were produced which expressed a fusion protein consisting of GUS and C-terminal fragments of *phyB*. GUS staining from the fusion protein in these transgenic plants was observed in the nucleus, which suggests that the nuclear localization signal of the fragment is functional. Next, it was examined whether the endogenous *phyB* was detected in the nucleus. Nuclei were isolated from the light-grown wild-type *Arabidopsis* leaves and subjected to the immunoblot analysis. The result indicated that a substantial fraction of total *phyB* was recovered in the isolated nuclei. This result was further confirmed by the immunocytochemical analysis of the protoplasts. Finally, the effects of light treatments on the levels of *phyB* in the isolated nuclei were examined. Dark adaptation of the plants before the nuclear isolation reduced the levels of *phyB*. The reduction was accelerated by irradiation of plants with far-red light before the transfer to darkness. Thus, nuclear localization of *phyB* was suggested to be light-dependent.

## Introduction

It is essential for plants to monitor and adapt to environmental conditions in order to survive. Light is one of the most important environmental stimuli that plants must perceive and respond to. Plants utilize light signals to regulate various developmental processes such as seed germination, de-etiolation and floral induction (Kendrick and Kronenberg, 1994). For this purpose, plants have evolved several different photoreceptors. Phytochrome, the best-characterized plant photoreceptor, is a soluble chromoprotein which consists of an apoprotein of about 120 kDa and a covalently attached linear tetrapyrrole chromophore. The holoprotein undergoes photoreversible conversion between two spectrally distinct forms, a red light-absorbing form (*Pr*) and a far-red light-absorbing form (*Pfr*). Only the *Pfr* form is believed to be biologically active. Light acts as a switch between the two forms and phytochrome-mediated responses induced with red light can be reversed with far-red light (Butler et al., 1959). The phytochrome protein is comprised of two domains (Quail et al., 1995). The N-terminal portion, to which the chromophore is attached, confers the spectral property characteristic to phytochrome (Yamamoto, 1987). The C-terminal portion is involved in dimerization of the molecule (Edger- ton and Jones, 1992; Nakasako et al., 1990). It has been shown that a short segment in the C-terminal domain is critical for transferring the light signal to downstream components (Wagner and Quail, 1995; Xu et al., 1995).

Phytochrome is encoded by a small multigene family (Sharrock and Quail, 1989). In *Arabidopsis*, the complete family consists of five members, *PHYA–E* (Clack et al., 1994). Although all five members are conserved in terms of their basic structure throughout the molecules, they display a marked diversity in amino acid sequences. These observations, taken together with physiological data, suggest that each member might have a distinct mode of action and/or mediate a distinct set of biological responses (Furuya, 1993). Putative null mutants for the *PHYB* (Koornneef et al., 1980; Reed et al., 1993) and *PHYA* (Dehesh et al., 1993; Reed et al., 1994; Whitelam et al., 1993) genes have been identified in *Arabidopsis*. Although the *phyB* mutant exhibits phenotypes such as elongated appearance in the light (Koornneef et al., 1980; Reed et al., 1993), *phyA* mutant plants are almost indistinguishable from the wild-type plants (Nagatani et al., 1993; Parks and Quail, 1993; Whitelam et al., 1993). However, *phyA* is essential for a highly sensitive, light-induced germination in the dark-incubated seeds (Shinomura et al., 1996). Thus, it is possible that *phyA* and *phyB* act through different mechanisms, despite the fact that they share common spectral characteristics *in vitro* (Abe et al., 1989).

It has been shown recently by microinjection experiments that heterotrimeric G-proteins, calcium-calmodulin and cGMP are involved in the signal transduction pathway of *phyA* (Bowler et al., 1994; Neuhaus et al., 1993). However, it is not known if all phytochromes act through the same

Received 4 June 1996; revised 6 August 1996; accepted 16 August 1996.
*For correspondence (fax +81 35684 0785;
e-mail nagatani@is.s.u-tokyo.ac.jp).
†Present address: Mitsui Plant Biotechnology Research Institute, TCI D-21 Senegen 2-1-6, Tsukuba, Ibaraki 305, Japan.

860 Koji Sakamoto and Akira Nagatani

biochemical pathways. As discussed above, there might be a less sensitive signal transduction pathway through which phyB transduces light signals. Immunochemical analysis has indicated that in dark-grown tissue phyA is a cytosolic protein (McCurdy and Pratt, 1986). Intracellular localization of other phytochromes has never been examined. Interestingly, phyB and other phytochromes including phyA contain multiple weak nuclear localization signals (NLS) within their C-terminal regions.

In the present study, we have examined the nuclear localization of phyB with two different methods. First, we found that fusion proteins comprised of C-terminal fragments of phyB and GUS were localized to the nucleus in transgenic Arabidopsis. Secondly, nuclei isolated from light-grown plants of Arabidopsis contained a substantial fraction of total cellular phyB. Immunocytochemical staining of protoplasts further confirmed this observation. Interestingly, dark adaptation of the plants before preparation of nuclei greatly reduced the level of phyB. Taking these results together, we propose that phyB is localized to the nucleus in a light-dependent manner.

Results

The achromophoric C-terminal of phyB targets a GUS reporter protein to the nucleus in transgenic Arabidopsis

Nuclear localization signals (NLS) are grouped into several categories; SV40, MAT α2 and bipartite (Raikhel, 1992). Weak homology to bipartite type sequences is found in the C-terminal region of the Arabidopsis phyB sequence (Figure 1a). Alignment of known phyB/D sequences (Arabidopsis, potato, tobacco and rice) shows that some regions of bipartite NLS homology (sequences 1 and 3) are conserved among all the phyB/D sequences, whereas others (sequences 2 and 4) are less conserved (Figure 1a). We therefore examined whether the achromophoric C-terminal region of Arabidopsis phyB (residues 594–1172, designated as PHYB<sub>594-1172</sub>) has nuclear targeting activity. The PHYB<sub>594-1172</sub> domain was translationally fused to the GUS reporter gene and expressed under the control of the CaMV 35S promoter in transgenic Arabidopsis (Figure 1b).

Expression of the GUS-PHYB<sub>594-1172</sub> fusion protein was confirmed. Soluble proteins were extracted from seedlings and analyzed by immunoblot analysis. As shown in Figure 1(c), a major band at the predicted size (130 kDa) was detected with an anti-phyB monoclonal antibody mBA2. This result was further confirmed by detection with another anti-phyB monoclonal antibody, which is raised against an N-terminal fragment of phyB (Shinomura et al., 1996). As expected, mBA1 reacted only with the endogenous phyB (Figure 1c, right). These results indicated that the transgenic plants accumulate the full-length fusion protein. Subcellular localization of the fusion protein was

(a)

|  | 876 |  | 919 |
| --- | --- | --- | --- |
| Arabidopsis | RNGKFVQALL TANKRVSLEG |  | RRQDTECFTK AK |
| Potato | RNGKYVQALL TRNKRVNMEG |  | RQQEKKCYSQ MK |
| Tobacco | RNGKYVQALL TANKRVNMEG |  | RQQEKKCYSQ MK |
| Rice | RNGKYVQALL TANTRSRMDG |  | RHHEKKCYAR MK |

Sequence 1 Sequence 2

Sequence 4

|  | 1132 |  |  |
| --- | --- | --- | --- |
| Arabidopsis | RKILKLMNGE VQYIRESERS YFLIILELPV PRKRPLS |  |  |
| Potato | RKMLKLMNGE IQYIRESERC YFLIILDLPM TRKGPKS |  |  |
| Tobacco | RKILKLMNGD IQYIRESERC YFLIILDLPM TRRGSKS |  |  |
| Rice | RKILKLMGGE VQYIRESERS FFHIVLELPQ PQQAASR |  |  |

Sequence 3

(b)

|  |  |  |  |
| --- | --- | --- | --- |
| GUS-PHYB<sub>594-1172</sub> | 594 | 1172 |  |
|  | 35Sp | GUS | phyB | NOSt |
| GUS-PHYB<sub>898-1172</sub> | 898 | 1172 |  |
|  | 35Sp | GUS | phyB | NOSt |
| GUS |  |  |  |
|  | 35Sp | GUS | NOSt |

: Sequence 1  
: Sequence 2  
: Sequence 3,4  

(c)

|  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
|  | 1 | 2 | 3 | 4 | 5 | 6 |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |
in the nucleus in two independent transgenic line (Figure 2, and data not shown). In contrast, transgenic plants that expressed a control GUS gene without phyB sequence did not exhibit nuclear staining even after prolonged exposure to the substrate (Figure 2). To confirm the nuclear localization of the fusion protein in another type of cell, epidermal peels of hypocotyls were stained in which inhibition of cell elongation by phytochrome is observed (Gaba and Black, 1983). As expected, most GUS activity was detected in the nuclei (Figure 3). Thus, the nuclear targeting activity of PHYB<sub>594–1172</sub> was demonstrated in both cotyledon and stem.

We then examined whether the nuclear localization of the achromophoric GUS–PHYB<sub>594–1172</sub> fusion protein is affected by light. Plants were treated with far-red light for 20 min and then kept in darkness for 24 h before protoplasts preparation. As shown in Figure 4, GUS activity was predominantly detected in the nuclei from leaf protoplasts even if the plants were dark adapted. This result indicates that the nuclear localization of the achromophoric fusion protein is not light-dependent.

The C-terminal 275 amino acid residues of phyB are sufficient to target a GUS reporter protein to the nucleus

We further localized NLS within the C-terminal region of phyB. For this purpose, we prepared transgenic Arabidopsis lines expressing a shorter phyB fragment, PHYB<sub>898–1172</sub>, which contains only sequences 2–4 (Figure 1a). As shown in Figures 2 and 3, the transgenic plant expressing this fusion protein exhibited nuclear staining in both leaf protoplasts and epidermal cells. This result indicates that these 275 amino acid residues of Arabidopsis phyB are sufficient to target the protein to the nucleus.

Endogenous phyB is detected in isolated nuclei and protoplasts

To test whether endogenous phyB is also localized to the nucleus, nuclei were isolated from leaves of light-grown Arabidopsis plants. The nuclei were prepared with buffers adjusted to pH 6.0 and 7.0. Measurements of the pH after the homogenization indicated that the homogenate was not acidified. Although the nuclei were more stable and easier to handle at pH 6.0, phyA is known to artificially associate with the particulate fraction at this pH (Quail et al., 1973). Microscopic evaluation of the isolated nuclei indicated that the preparations using both buffers were free of visible contamination and that the nuclei were intact (representative data are shown in Figure 5a). The marked diversity in the size of nuclei probably reflects the multiploidy in Arabidopsis tissues as reported by Arumugananathan and Earle (1991). The average DNA content per nucleus in the preparations was about 0.9 pg, which is consistent

Nuclear localization activity of phytochrome B 861

with the reported value (Arumugananathan and Earle, 1991). Assessment of enzyme markers (Schaller and DeWitt, 1995) for endoplasmic reticulum (antimycin-insensitive NADH cytochrome c oxidase) and mitochondria (cytochrome c oxidase) indicated that contamination by these organelles was less than 5%. Immunoblot detection of the large subunit of Rubisco indicated that only about 0.1% of the total cellular accumulation was associated with the nuclear preparation. We did not see any significant differences in the purity of the preparation due to the difference in pH of the isolation buffer.

The nuclei were lysed with SDS and subjected to immunoblot analysis. Immunoblots probed with a monoclonal anti-Arabidopsis phyB antibody, mBA2, clearly demonstrates the presence of phyB in the nuclei (Figure 5b). No significant difference was seen in the levels of phyB between the nuclei prepared at pH 6.0 (lane 1) and pH 7.0 (lane 2). Since the nuclei from the putative phyB null mutant did not show staining (lane 3), the staining appears to be specific to phyB. Similar results were obtained with two other monoclonal anti-phyB antibodies, mAT1 raised against a tobacco PHYB fragment (residues 465–1132) (López-Juez et al., 1992) and mBA1 raised against an Arabidopsis PHYB fragment (residues 1–598) (Shinomura et al., 1996) (data not shown). The level of nuclear phyB relative to that of total cellular phyB was estimated on the basis of DNA content. As shown in Figure 5(b), the intensity of the phyB signal in whole-cell extracts (lane 4) was similar to that in isolated nuclei, indicating that a substantial part of the total cellular phyB exists in the nucleus. The detection of phyB in the crude extract was not affected by the presence of other cellular proteins as mixing cellular extract with the nuclear preparation did not alter the results (lanes 5 and 6).

The nuclear localization of endogenous phyB was examined by immunocytochemistry as well. Protoplasts were prepared from the light-grown leaves of the wild-type and the phyB null mutant and immunostained for phyB with the monoclonal antibody mBA2. As shown in Figure 6, strong immunoreactivity of phyB was detected in the nucleus in wild-type protoplasts whereas the phyB mutant ones showed little staining under the same conditions. Thus, nuclear localization of phyB was confirmed by both immunoblot and immunocytochemical methods.

Nuclear phyB is extracted to soluble fraction on disruption of nuclei

PhyB is extracted to the soluble fraction when tissue is homogenized in phytochrome extraction buffer (Abe et al., 1989). Therefore, we examined whether phyB in isolated nuclei can be similarly extracted. The nuclei were gently resuspended in the extraction buffer at pH 8.0, sonicated for a few seconds, and separated by centrifugation. Micro-

plants were determined. Plants were treated with red or far-red light for 20 min and then kept in darkness for 0, 4 or 12 h before the homogenization. In the case of the 0 h samples, leaves harvested in the light were placed on ice and irradiated with red or far-red light immediately before the nuclear isolation. For other samples, the whole plants were irradiated and transferred to darkness. Isolation of nuclei was performed under dim green safe light or in total darkness until the end of the first Percoll gradient step. All the remaining steps were done under white light. As a control, nuclei were prepared from light-grown plants in the light. As shown in Figure 7(a), the level of nuclear phyB is significantly reduced during the course of dark adaptation (lanes 3, 4, 6 and 7). The reduction was much faster and more complete if plants were treated with far-red light. However, treatment of leaves with either red or far-red light immediately before the isolation did not reduce the levels of nuclear phyB (lanes 2 and 3). This observation indicates that the levels of phyB in isolated nuclei were independent of the spectral forms of phyB during the nuclear isolation. We also examined if the dark adaptation altered the levels of cellular total phyB. As shown in Figure 7(b), the phyB level was not altered by these treatments.

## Discussion

### Nuclear localization of GUS-PHYB fragment fusion proteins

GUS reporter fusion proteins have been successfully used to demonstrate the presence of functional NLS in various plant proteins (Howard et al., 1992). Using this approach, we have shown that phyB contains functional NLS(s) within the C-terminal 275 amino acid residues which contain three putative bipartite type sequences (sequences 2–4 in Figure 1a). However, it should be noted that the localization could be due to a cryptic localization signal not used by the full-length molecule. Studies with a full-length construct would help to clarify this problem.

Since each putative NLSs found in the phyB sequence is rather weak compared with other known NLSs (Raikhel, 1992), these sequences might function in combination. It has been reported that a protein containing a weak NLS repeat can be localized to the nucleus (Varagona and Raikhel, 1994). Sequences 2 and 3 (Figure 1a) are conserved in Arabidopsis phyA, phyB and phyD, and in phyA, phyB, phyD and phyE, respectively. Although phyC appears to lack the same sequences found in phyB, different NLS-like sequences are found in the C-terminal region of phyC. Thus, it remains to be determined whether or not phytochromes other than phyB contain a functional NLS. It is noteworthy that a chimeric protein between the N-terminal half of phyB and the C-terminal half of phyA exhibits phyB activity

### Light regulation of the level of nuclear phyB

To examine if nuclear localization of the endogenous phyB is light-dependent, levels of nuclear phyB in dark-adapted

Nuclear localization activity of phytochrome B 863

Figure 3. Cellular GUS staining in epidermal peels of hypocotyls of transgenic Arabidopsis expressing the GUS-PHYB fragment fusion proteins. The epidermal peels prepared from the transgenic Arabidopsis expressing GUS-PHYB<sub>594-1172</sub> (a–d), GUS-PHYB<sub>898-1172</sub> (e and f) and GUS (g and h) were stained with X-Gluc (a, c, e and g) or DAPI (b, d, f and h). The scale bar represents 20 μm.

in transgenic plants (Wagner et al., 1996). This result might reflect the presence of a NLS in the C-terminal region of phyA.

Nuclear localization of endogenous phytochrome B

Together with the GUS-phyB fusion protein staining (Figures 2 and 3), our immunochemical analysis of isolated nuclei (Figure 5) suggests that the endogenous phyB is localized to the nucleus. In the history of phytochrome studies, the intracellular localization of phytochrome has been a matter of controversy for a long time (Pratt, 1994). Immunocytochemical analysis has revealed that phyA is principally distributed in the cytosol in dark-grown seedlings (McCurdy and Pratt, 1986). Although associations of phytochrome (probably phyA) with various organelles such

864 Koji Sakamoto and Akira Nagatani

Figure 4. Cellular GUS staining in leaf protoplasts prepared from dark-adapted transgenic Arabidopsis expressing the GUS-PHYB<sub>594-1172</sub> fusion protein.

The protoplasts were prepared from dark-adapted transgenic Arabidopsis expressing GUS-PHYB<sub>594-1172</sub> (a and b) and GUS (c and d), and were stained with X-Gluc (a and c) or DAPI (b and d). The scale bar represents 20 μm.

as amyloplasts, etioplasts, mitochondria, or plasma membrane have been reported in dark-grown plants, only a small portion of the total phyA present in the cell is recovered in these organelles or membrane fractions (Pratt, 1994). It is also well known that the Pfr form of phyA tends to associate with particulate material under certain conditions. However, no association with any particular organelle has been observed (Quail, 1975). Thus, there has been no convincing evidence that indicates a biologically relevant association of phytochrome with a particular organelle.

By a number of criteria, our results are distinct from all of these classical observations. First, staining for cellular GUS activity has revealed the presence of a functional targeting signal (Figures 2, 3 and 4), which has never previously been shown for any phytochromes. Secondly, a substantial part of the total cellular phyB was shown to be localized to the nucleus both by immunoblot detection (Figure 5) and immunocytochemical staining (Figure 6). Thirdly, phyB was detected in the isolated nuclei regardless of the spectral forms of phyB during isolation (lanes 2 and 5 of Figure 7). Finally, phyB was extracted from isolated nuclei only after disruption of the nucleus (Figure 5c). Therefore, we conclude that our observations are not due to artifacts, and that phyB is indeed present in the nucleus in the light. However, our data do not exclude the possibility that some amount of the Pfr form of phyB exists and acts in the cytosol. Functional analysis, such as microbeam

Figure 5. See facing page for legend.

Figure 6. See facing page for legend.

irradiation of the nucleus, will help to clearly demonstrate the biological significance of phyB in the nucleus.

### Possible mechanism of PhyB signal transduction

The level of nuclear phyB was greatly reduced by dark adaptation of plants, whereas the total level of cellular

(a)
L R0 R4 R12 F0 F4 F12
α-phyB
CBB
1 2 3 4 5 6 7

(b)
L F12 R12
1 2 3

Figure 7. Effects of light treatments on the levels of nuclear phyB.
(a) Immunoblot detection of phyB in isolated nuclei. Nuclei were isolated from light-grown plants (L), from plants treated with either red (R0) or far-red (F0) light immediately before the nuclear preparation, and from plants treated with red (R4, R12) or far-red (F4, F12) light before the dark adaptation for 4 (R4, F4) or 12 (R12, F12) h. Each lane contains a sample from 4.0×10⁵ nuclei. The blots were stained with a monoclonal anti-Arabidopsis phyB antibody mBA2 (upper). As a loading control, 8.0×10⁵ nuclei of each sample were analyzed by histone SDS-PAGE (lower).
(b) Immunoblot detection of phyB in the whole-cell extracts. Whole-cell extracts were prepared from light-grown plants (L), from plants treated with either red (R12) or far-red (F12) light and then dark-adapted for 12 h. Each lane contains a sample corresponding to 3.4 mg fresh weight tissue. The blot was stained with a monoclonal anti-Arabidopsis phyB antibody mBA2.

---

Figure 5. Immunoblot detection of phyB in isolated nuclei.
(a) Isolated nuclei were photographed in bright field (left), or under epifluorescence to show DNA staining with DAPI (right). The scale bar represents 20 μm.
(b) Immunoblot detection of phyB in nuclei isolated at pH 6.0 (lane 1) or 7.0 (lanes 2 and 3) and whole-cell extracts. Each lane contains a sample from 4×10⁵ nuclei (lanes 1–3), whole-cell extract corresponding to 2.4 mg fresh weight tissue (lanes 4 and 5) or 4×10⁵ wild-type nuclei plus an extract from 2.4 mg phyB mutant tissue (lane 6). All the samples are equivalent to 0.36 μg DNA. The blots stained with a monoclonal anti-Arabidopsis phyB antibody, mBA2 (α-phyB) and histone SDS-PAGE gel stained with Coomassie brilliant blue (CBB) are shown. WT, wild-type; phyB, Arabidopsis phyB-5 mutant.
(c) Extraction of phyB from isolated nuclei. Nuclei prepared at pH 7.0 were treated in extraction buffer at pH 8.0 with (+sonic) or without (−sonic) sonication and separated by centrifugation to supernatant (S) and pellet (P). Each lane contains the sample from 3.2×10⁵ nuclei. The blots probed with a monoclonal anti-Arabidopsis phyB antibody mBA2 (upper) and histone SDS-PAGE gel stained with CBB (lower) are shown.

Figure 6. Immunocytochemistry of wild-type and phyB mutant.
The localization of phyB was determined by mBA2 with protoplasts prepared from wild-type (a) and phyB mutant (b). The nuclei of wild-type (c) and phyB mutant (d) were stained with DAPI. The scale bar represents 20 μm.

---

Nuclear localization activity of phytochrome B 865

phyB remained fairly constant (Figure 7). However, the reduction was slower if plants were irradiated with red light before dark adaptation. This observation coincides with a photophysiological observation that short red light pulses given every 4 h are as effective as continuous red light for inhibition of hypocotyl elongation (Reed et al., 1994). Furthermore, the reduction in the levels of nuclear phyB was greatly accelerated by irradiation of plants with far-red light before dark adaptation (Figure 7). Since the achromophoric GUS-PHYB₅₉₄–₁₁₇₂ fusion protein did not change its cellular localization in response to a similar treatment (Figure 4), we propose that the spectral forms of phyB are important for the light-dependent nuclear localization and/or retention to occur. In mammalian cells, several transcription factors and proteins that modify these factors are known to change their nucleocytoplasmic partitioning in response to extracellular signals (Hill and Treisman, 1995). Thus, phyB might act in the same way to relay light signals to the nucleus.

Although targets of phyB in the nucleus are unknown, it is an attractive hypothesis that phyB regulates gene expression (Batschauer et al., 1994) through interactions with nuclear master regulators of photomorphogenesis. There are several Arabidopsis mutants that show constitutive photomorphogenic responses in the dark (Chory et al., 1989; Deng et al., 1991). A few of these mutants such as det1 (Chory, 1992) and cop1 (Ang and Deng, 1994) are known to be epistatic to photoreceptor mutants, suggesting that DET1 and COP1 act downstream of phytochrome. Interestingly, DET1 (Pepper et al., 1994) and COP1 (von Arnim and Deng, 1994) proteins are known to be nuclear-localized. Furthermore, nucleocytoplasmic partitioning of COP1 is regulated by light (von Arnim and Deng, 1994). Thus, phyB might modify the activity and/or localization of these nuclear factors in cytosol and/or within the nucleus. It is also noteworthy that protein phosphorylation is reported to be regulated by phytochrome in isolated nuclei (Datta et al., 1985; Romeo et al., 1991). Hence, protein phosphorylation might play a role in the signal transduction pathway acting downstream of phyB in the nucleus. However, it should be noted that nuclei were isolated from dark-grown plants for these phosphorylation experiments, and that the level of phyB in such nuclei has never been determined.

---

Other molecular species of phytochrome

Although it would be reasonable to hypothesize that phyB acts in the nucleus, the same model might not be applicable to other phytochromes. It has been shown that microinjection of agonists of heterotrimeric G-proteins or phyA to the cells of a phytochrome-deficient tomato mutant (aurea) complements its deficiency in the expression of phytochrome-regulated genes and in chloroplast development

(Neuhaus *et al.*, 1993). Calcium ions and cGMP have been shown to function as second messengers for these responses (Bowler *et al.*, 1994). Thus, the site of phyA action is probably not in the nucleus. It is noteworthy here that extremely weak light with which only 0.01% of the total Pr is converted to Pfr is sufficient to induce the phyA-dependent seed germination (Shinomura *et al.*, 1996). Thus, it is reasonable to suggest that the phyA response is amplified through cytosolic and/or plasma membrane signal transduction components, such as G-proteins. In contrast, the phyB response requires much higher fluence of light (Shinomura *et al.*, 1996), indicating that such amplification steps might be omitted in the phyB signal transduction pathway. It should also be noted that some of the phytochrome-mediated responses observed in lower plants are difficult to explain with phytochrome present in the nucleus. For example, polarization of functional phytochrome on the plasma membrane has been shown in a green algae and fern protonemata by microbeam irradiation experiments (Wada and Kadota, 1989). Thus, different phytochromes might act in totally different ways, even though they share a common overall structure and basic molecular properties.

### Experimental procedures

#### Plant materials
Arabidopsis thaliana (ecotype, Wassilewskija) was used for the transformation experiments. A. thaliana (ecotype Landsberg *er*) and the *phyB-5* (formerly known as *hy3-8-36*) mutant (Reed *et al.*, 1993) were used for the nuclear isolation and immunocytochemical experiments.

#### Plasmid construction and transformation
The binary vectors that express GUS fused to phyB C-terminal sequences were constructed by ligating the C-terminal fragments of the *PHYB* gene to the end of the GUS gene (Figure 1b). The stop codon of the GUS gene in the binary vector pBIH1-IG (Akama *et al.*, 1992) was replaced with a *SmaI* restriction endonuclease site by PCR. The plasmid so constructed (pGΔT8) was used to express the GUS-PHYB<sub>594–1172</sub> and the GUS-PHYB<sub>898–1172</sub> fusion proteins. The C-terminal fragments of *PHYB* gene were amplified by PCR with primers that generate a *SmaI*-ATG sequence at the 5′ end and a TAG-*SacI* sequence at the 3′ end. The *SmaI*-SacI fragment PCR products were ligated into *SmaI*-SacI-digested pGΔT8. Both of these plasmids were introduced into Arabidopsis by Agrobacterium-mediated transformation (Akama *et al.*, 1992).

#### Isolation of protoplasts and analysis of GUS localization
Protoplasts were prepared from transgenic Arabidopsis as follows. The cotyledons were harvested from approximately 10-day-old seedlings and cut into several pieces in a petri dish containing digestion buffer (0.4 M mannitol, 8 mM CaCl<sub>2</sub>, 0.5% w/v cellulase 'Onozuka' RS, 0.05% pectinase Y23). The tissues were then vacuum-infiltrated and incubated at room temperature with gentle agitation for 2–3 h. The protoplasts were collected by centrifugation at 60 g. The protoplasts were incubated with GUS assay mixture (0.3 M mannitol, 50 mM Na-phosphate buffer at pH 7.0, MS medium salt mixture, 10 mM EDTA, 0.5 mM K<sub>3</sub>Fe(CN)<sub>6</sub>, 0.5 mM K<sub>4</sub>Fe(CN)<sub>6</sub>, 2 mM X-Gluc) at 37°C for 3–4 h. After the development of blue color, 4′,6-diamidino-2-phenylindole dihydrochloride (DAPI) was added at 50 ng ml<sup>−1</sup> to visualize DNA.

#### Histochemical analysis of GUS localization
The epidermal layer was peeled from the hypocotyl of the transgenic plants with forceps, transferred to the GUS assay mixture (see above) and vacuum-infiltrated. After incubation at 37°C for 0.5–1 h, DAPI was added as described above.

#### Isolation of nuclei
Arabidopsis plants were grown under continuous white light for 3–4 weeks. For some experiments, plants were treated with 3.0 W m<sup>−2</sup> far-red or red light (Nagatani *et al.*, 1993) for 20 min and then transferred into darkness. To isolate nuclei, 2–5 g of leaves were harvested and homogenized in 80 ml of the nuclear isolation buffer at pH 6.0 (10 mM Mes-KOH, 10 mM NaCl, 5 mM EDTA, 0.15 mM spermine, 0.5 mM spermidine, 0.6% Triton X-100, 0.25 M Sucrose, pH 6.0 (Jensen *et al.*, 1988)) or at pH 7.0 (10 mM Pipes-KOH, 10 mM MgCl<sub>2</sub>, 0.2% Triton X-100, 1 M hexylene glycol, 5 mM 2-mercaptoethanol, 0.8 mM phenylmethylsulfonyl fluoride (PMSF), pH 7.0 (Foster *et al.*, 1992)) with a blender equipped with razor blades. The crude homogenate was then filtered through stainless mesh and the nuclei in the filtrate were collected by centrifugation at 1500 g. The nuclei were further purified twice on 0–50% discontinuous Percoll (Pharmacia Biotech) gradient (Datta *et al.*, 1985). With this method, 1–2 × 10<sup>6</sup> nuclei per gram tissue were reproducibly obtained. All the procedures were performed under white light at 4°C unless otherwise stated. The DNA contents in the preparations were determined according to Baer *et al.* (1982). Assays of enzyme markers for endoplasmic reticulum (antimycin-insensitive NADH cytochrome c oxidase) and mitochondria (cytochrome c oxidase) were carried out by standard procedures (Schaller and DeWitt, 1995). A monoclonal antibody against the large subunit of Rubisco was prepared as described (Nagatani *et al.*, 1984).

#### Immunoblot detection of phyB
The isolated nuclei were solubilized in the SDS-PAGE sample buffer (Laemmli, 1970) at 0.8 × 10<sup>5</sup> nuclei μl<sup>−1</sup> at 95°C for 2 min and were then subjected to the immunoblot analysis. For the preparation of nuclear extracts, 2 × 10<sup>6</sup> nuclei were resuspended in 25 μl of the extraction buffer (0.15 M NaCl, 10 mM Tris-HCl, 1 mM PMSF, 5 mM 2-mercaptoethanol, pH 8.0), sonicated at the lowest possible intensity for a few seconds (Sonicator 5202, Ohtake Works, Tokyo), and centrifuged at 18 000 g for 10 min. The supernatant and the pellet were then subjected to the immunoblot analysis. As a loading control, nuclear proteins from each sample were subjected to SDS-PAGE modified for histone separation and stained with Coomassie brilliant blue (Thomas and Kornberg, 1975). Four major bands presumed to be core histones were detected in the lower molecular-weight region.

The whole-cell extracts were prepared as follows. Leaves were homogenized in SDS-PAGE sample buffer (Laemmli, 1970) on ice with a small glass homogenizer. The homogenate was then heated at 95°C for 5 min. Debris were removed by centrifugation and

subjected to SDS-PAGE. The DNA contents in the tissue were determined to be 150 μg DNA per gram tissue by the method of Baer et al. (1982). The entire comparison between the nuclear fractions and the whole-cell extracts was based on this value. Immunoblot detection was done with ProtoBlot Western Blot AP Systems (Promega) using a monoclonal anti-phyB antibody mBA2, which is raised against an Arabidopsis PHYB fragment (residues 594–1172) (Shinomura et al., 1996) unless otherwise stated.

### Immunocytochemical detection of phyB

The protoplasts were washed with washing solution (0.4 M mannitol, 4 mM Mes-KOH, 20 mM KCl, pH 5.7) and laid on polylysine-coated slides. The protoplasts were fixed with 2% paraformaldehyde for 15 min and permeabilized with 0.4% Triton X-100 for 4 min. The permeabilized cells on the slides were blocked with 2% bovine serum albumin (BSA) for 1 h and then incubated with the mBA2 antibody for 1 h. The glass plates were washed with PBS and FITC linked anti-mouse Ig (Amersham International) was added and incubated for 1 h. The plates were washed with PBS to remove excess FITC and mounted in an anti-fade mount solution containing DAPI.

### Acknowledgments

We thank Dr Kenzo Nakamura for providing the binary vector pBIH1-IG and Dr Xing-Wang Deng for protocols for immunocytochemical staining of protoplasts. We also thank Drs Kazuhiro Akama, Azusa Yano, Takatoshi Kagawa, Hiroshi Takeda, Richard E. Kendrick for helpful discussions and Drs Stephen J. Streatfield and Shannon Frances for comments on the manuscript. This work was supported in part by a Grant-in-Aid for General Scientific Research to K.S. and A.N. (06454018) and a Grant-in-Aid for Scientific Research on Priority Areas 'The Molecular Basis of Flexible Organ Plans in Plants' to A.N. (0727022, 08262203) from the Ministry of Education, Science and Culture of Japan.

### References

Abe, H., Takio, K., Titani, K. and Furuya, M. (1989) Amino-terminal amino acid sequences of pea phytochrome II fragments obtained by limited proteolysis. *Plant Cell Physiol.* 30, 1089–1097.

Akama, K., Shiraishi, H., Ohta, S., Nakamura, K., Okada, K. and Shimura, Y. (1992) Efficient transformation of Arabidopsis thaliana: Comparison of efficiencies with various organ, plant ecotypes and Agrobacterium strain. *Plant Cell Rep.* 12, 7–11.

Ang, L.-H. and Deng, X.-W. (1994) Regulatory hierarchy of photomorphogenic loci: allele-specific and light-dependent interaction between the HY5 and COP1 loci. *Plant Cell*, 6, 613–628.

von Arnim, A.G. and Deng, X.-W. (1994) Light inactivation of Arabidopsis photomorphogenic repressor COP1 involves a cell-specific regulation of its nucleocytoplasmic partitioning. *Cell*, 79, 1035–1045.

Arumuganathan, K. and Earle, E.D. (1991) Nuclear DNA content of some important plant species. *Plant Mol. Biol. Rep.* 9, 208–219.

Baer, G.B., Meyers, S.P., Molin, W.T. and Scharader, L.E. (1982) A simple and sensitive DNA assay for plant extracts. *Plant Physiol.* 70, 999–1003.

Batschauer, A., Gilmartin, P.M., Nagy, F. and Schäfer, E. (1994) The molecular biology of photoregulated genes. In *Photomorphogenesis in plants*, 2nd Edn. (Kendrick, R.E. and Kronenberg, G.H.M., eds). Dordrecht: Kluwer Academic Publishers, pp. 559–599.

Bowler, C., Neuhaus, G., Yamagata, H. and Chua, N.-H. (1994) Cyclic GMP and calcium mediate phytochrome phototransduction. *Cell*, 77, 73–81.

Butler, W.L., Norris, K.H., Siegelman, H.W. and Hendricks, S.B. (1959) Detection, assay, and preliminary purification of the pigment controlling photoresponsive development of plants. *Proc. Natl Acad. Sci. USA*, 45, 1703–1708.

Chory, J. (1992) A genetic model for light-regulated seedling development in Arabidopsis. *Development*, 115, 337–357.

Chory, J., Peto, C., Feinbaum, R., Pratt, L. and Ausubel, F. (1989) Arabidopsis thaliana mutant that develops as a light-grown plant in the absence of light. *Cell*, 58, 991–999.

Clack, T., Mathews, S. and Sharrock, R.A. (1994) The phytochrome apoprotein family in Arabidopsis is encoded by five genes: The sequences and expression of PHYD and PHYE. *Plant Mol. Biol.* 25, 413–427.

Datta, N., Chen, Y.-R. and Roux, S.J. (1985) Phytochrome and calcium stimulation of protein phosphorylation in isolated pea nuclei. *Biochem. Biophys. Res. Commun.* 128, 1403–1408.

Dehesh, K., Tepperman, J., Christensen, A.H. and Quail, P.H. (1991) phyB is evolutionarily conserved and constitutively expressed in rice seedling shoots. *Mol. Gen. Genet.* 225, 305–313.

Dehesh, K., Franci, C., Parks, B.M., Seeley, K.A., Short, T.W., Tepperman, J.M. and Quail, P.H. (1993) Arabidopsis HY8 locus encodes phytochrome A. *Plant Cell*, 5, 1081–1088.

Deng, X.-W., Caspar, T. and Quail, P.H. (1991) COP1: a regulatory locus involved in light-controlled development and gene expression in Arabidopsis. *Genes Devel.* 5, 1172–1182.

Edgerton, M.D. and Jones, A.M. (1992) Localization of protein-protein interactions between subunits of phytochrome. *Plant Cell*, 4, 161–171.

Foster, R., Gasch, A., Kay, S. and Chua, N.-H. (1992) Analysis of protein/DNA interactions. In *Methods in Arabidopsis research* (Koncz, C., Chua, N.-H. and Schell, J., eds). Singapore: World Scientific Publishing, pp. 378–392.

Furuya, M. (1993) Phytochromes — their molecular species, gene families, and functions. *Ann. Rev. Plant Physiol. Plant Mol. Biol.* 44, 617–645.

Gaba, V. and Black, M. (1983) The control of cell growth by light. In *Photomorphogenesis: Encyclopedia of Plant Physiology*. New Ser. 16A. (Shropshire, Jr, W. and Mohr, H., eds), Berlin: Springer Verlag, pp. 358–400.

Heyer, A. and Gatz, C. (1992) Isolation and characterization of a cDNA-clone coding for potato type-B phytochrome. *Plant Mol. Biol.* 20, 589–600.

Hill, C.S. and Treisman, R. (1995) Transcriptional regulation by extracellular signals: Mechanisms and specificity. *Cell*, 80, 199–211.

Howard, E.A., Zupan, J.R., Citovsky, V. and Zambryski, P.C. (1992) The VirD2 protein of A. tumefaciens contains a C-terminal bipartite nuclear localization signal: Implications for nuclear uptake of DNA in plant cells. *Cell*, 68, 109–118.

Jensen, E.Ø., Marcker, K.A., Schell, J. and Bruijn, F.J. (1988) Interaction of a nodule specific, trans-acting factor with distinct DNA elements in the soybean leghaemoglobin *lbcl3 5′* upstream region. *EMBO J.* 7, 1265–1271.

Kendrick, R.E. and Kronenberg, G.H.M. (eds) (1994) *Photomorphogenesis in plants*, 2nd Edn. Dordrecht: Kluwer Academic Publishers.

Kern, R., Gasch, A., Deak, M., Kay, S.A. and Chua, N.-H. (1993) PhyB of tobacco, a new member of the phytochrome family. *Plant Physiol.* 102, 1363–1364.

Koornneef, M., Rolff, E. and Spruit, C.J.P. (1980) Genetic control of light-inhibited hypocotyl elongation in *Arabidopsis thaliana* (L.) Heynh. Z. Pflanzenphysiol. 100S, 147–160.

Laemmli, U.K. (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature, 227, 680–685.

López-Juez, E., Nagatani, A., Tomizawa, K.I., Deak, M., Kern, R., Kendrick, R.E. and Furuya, M. (1992) The cucumber long hypocotyl mutant lacks a light-stable PHYB-like phytochrome. Plant Cell, 4, 241–251.

McCurdy, D.W. and Pratt, L.H. (1986) Immunogold electron microscopy of phytochrome in *Avena*: Identification of intracellular sites responsible for phytochrome sequestrating and enhanced pelletability. J. Cell Biol. 103, 2541–2550.

Nagatani, A., Yamamoto, K.T., Furuya, M., Fukumoto, T. and Yamashita, A. (1984) Production and characterization of monoclonal antibodies which distinguish different surface structures of pea (*Pisum sativum* cv. Alaska) phytochrome. Plant Physiol. 25, 1059–1068.

Nagatani, A., Reed, J.W. and Chory, J. (1993) Isolation and initial characterization of *Arabidopsis* mutants that are deficient in phytochrome A. Plant Physiol. 102, 269–277.

Nakasako, M., Wada, M., Tokutomi, S., Yamamoto, K.T., Sakai, J., Kataoka, M., Tokunaga, F. and Furuya, M. (1990) Quaternary structure of pea phytochrome I dimer studied with small-angle X-ray scattering and rotary-shadowing electron microscopy. Photochem. Photobiol. 52, 3–12.

Neuhaus, G., Bowler, C., Kern, R. and Chua, N.-H. (1993) Calcium/calmodulin-dependent and calcium/calmodulin-independent phytochrome signal transduction pathways. Cell, 73, 937–952.

Parks, B.M. and Quail, P.H. (1993) *hy8*, a new class of *Arabidopsis* long hypocotyl mutants deficient in functional phytochromeA. Plant Cell, 5, 39–48.

Pepper, A., Delaney, T., Washburn, T., Poole, D. and Chory, J. (1994) *DET1*, a negative regulator of light-mediated development and gene expression in *Arabidopsis*, encodes a novel nuclear-localized protein. Cell, 78, 109–116.

Pratt, L.H. (1994) Distribution and localization of phytochrome within the plant. In *Photomorphogenesis in Plants*, 2nd Edn (Kendrick, R.E. and Kronenberg, G.H.M., eds). Dordrecht: Kluwer Academic Publishers, pp. 163–185.

Quail, P.H. (1975) Particle-bound phytochrome: Association with a ribonucleoprotein fraction from *Cucurbita pepo* L. Planta, 123, 223–234.

Quail, P.H., Marmé, D. and Schäfer, E. (1973) Particle-bound phytochrome from maize and pumpkin. Nature New Biol. 245, 189–191.

Quail, P.H., Boylan, M.T., Parks, B.M., Short, T.W., Xu, Y. and Wagner, D. (1995) Phytochromes: Photosensory perception and signal transduction. Science, 268, 675–680.

Raikhel, N. (1992) Nuclear targeting in plants. Plant Physiol. 100, 1627–1632.

Reed, J.W., Nagpal, P., Poole, D.S., Furuya, M. and Chory, J.
(1993) Mutations in the gene for the red far-red light receptor phytochrome B alter cell elongation and physiological responses throughout *Arabidopsis* development. Plant Cell, 5, 147–157.

Reed, J.W., Nagatani, A., Elich, T.D., Fagan, M. and Chory, J. (1994) Phytochrome A and phytochrome B have overlapping but distinct functions in *Arabidopsis* development. Plant Physiol. 104, 1139–1149.

Romeo, L.C., Sommer, D., Gotor, C. and Song, P.-S. (1991) Protein phosphorylation in isolated nuclei from etiolated *Avena* seedlings. Effects of red/far-red light and cholera toxin. FEBS Lett. 282, 347–350.

Schaller, G.E. and DeWitt, N.D. (1995) Analysis of the H⁺-ATPase and other proteins of the *Arabidopsis* plasma membrane. Methods Cell Biol. 50, 129–148.

Sharrock, R.A. and Quail, P.H. (1989) Novel phytochrome sequences in *Arabidopsis thaliana*: Structure, evolution, and differential expression of a plant regulatory photoreceptor family. Genes Devel. 3, 1745–1757.

Shinomura, T., Nagatani, A., Hanzawa, H., Kubota, M., Watanabe, M. and Furuya, M. (1996) Action spectra for phytochrome A- and B-specific photoinduction of seed germination in *Arabidopsis thaliana*. Proc. Natl Acad. Sci. USA, 93, 8129–8133.

Thomas, J.O. and Kornberg, R.D. (1975) An octamer of histones in chromatin and free in solution. Proc. Natl Acad. Sci. USA, 72, 2626–2630.

Varagona, M.J. and Raikhel, N.V. (1994) The basic domain in the bZIP regulatory protein Opaque2 serves two independent functions: DNA binding and nuclear localization. Plant J. 5, 207–214.

Wada, M. and Kadota, A. (1989) Photomorphogenesis in lower green plants. Ann. Rev. Plant Physiol. Plant Mol. Biol. 40, 169–191.

Wagner, D. and Quail, P.H. (1995) Mutational analysis of phytochrome B identifies a small COOH-terminal-domain region critical for regulatory activity. Proc. Natl Acad. Sci. USA, 92, 8596–8600.

Wagner, D., Fairchild, C.D., Kuhn, R.M. and Quail, P.H. (1996) Chromophore-bearing NH₂-terminal domains of phytochromes A and B determine their photosensory specificity and differential light lability. Proc. Natl Acad. Sci. USA, 93, 4011–4015.

Whitelam, G.C., Johnson, E., Peng, J.R., Carol, P., Anderson, M.L., Cowl, J.S. and Harberd, N.P. (1993) Phytochrome A null mutants of *Arabidopsis* display a wild-type phenotype in white light. Plant Cell, 5, 757–768.

Xu, Y., Parks, B.M., Short, T.W. and Quail, P.H. (1995) Missense mutations define a restricted segment in the C-terminal domain of phytochrome A critical to its regulatory activity. Plant Cell, 7, 1433–1443.

Yamamoto, K.T. (1987) Structure-function relationship of pea phytochrome as deduced from amino-terminal sequence analysis of phytochrome chromopeptides. In *Phytochrome and Photoregulation in Plants* (Furuya, M., ed.). Tokyo: Academic Press, pp. 63–81.
