
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 1997 by The American Society for Biochemistry and Molecular Biology, Inc.

**The Hepatitis B Virus X-associated Protein, XAP3, Is a Protein Kinase C-binding Protein**

(Received for publication, November 14, 1996, and in revised form, March 31, 1997)

Yu-Sheng Cong, Ya-Li Yao, Wen-Ming Yang, Nadarajan Kuzhandai Velu, and Edward Seto*

From the Moffitt Cancer Center & Research Institute, Department of Medical Microbiology/Immunology, College of Medicine, University of South Florida, Tampa, Florida 33612

The hepatitis B virus X protein induces transcriptional activation of a wide variety of viral and cellular genes. In addition to its ability to interact directly with many nuclear transcription factors, several reports indicate that the X protein stimulates different cytoplasmic kinase signal cascades. Using the yeast two-hybrid screen, we have isolated a clone designated X-associated protein 3 (XAP3) that encodes a human homolog of the rat protein kinase C-binding protein. One of the activation domains of X (amino acids 90–122) is required for binding to XAP3, while the NH₂-terminal part of XAP3 is necessary for binding to X. Both X and XAP3 bound specifically to the η PKC isoenzyme synthesized in rabbit reticulocyte lysates. Overexpression of XAP3 enhanced X transactivation activity. These results support earlier findings that one of the mechanisms of transactivation by X is through involvement with the cellular protein kinase C pathway.

Transcriptional activation is a widespread phenomenon among mammalian viral systems. Mammalian viral proteins that increase the rate of transcription can be divided into two groups based on whether they exhibit sequence-specific DNA binding. For example, the herpes simplex virus 1 (HSV-1)¹ Vmw175 (1, 2), the Epstein-Barr virus BZLF1 (3), the papilloma virus E2 (4), and the simian virus 40 and polyoma virus large T antigens (5) bind specific DNA sequence motifs, whereas HSV-1 VP16 (6, 7), the pseudorabies virus immediate early protein (8, 9), and the adenovirus E1A protein (10, 11) do not. During the last decade, many studies have shown that non-DNA binding viral transactivators achieve their task by direct interaction with different cellular sequence-specific DNA binding transcription factors. For example, VP16 interacts with the Oct-1 protein, thereby positioning the VP16 activating domain at a promoter to enhance transcription (7, 12–17). Similarly, E1A interacts with a number of cellular proteins including ATF-2 (18).

The hepatitis B virus (HBV) X protein is a promiscuous transcriptional transactivator (reviewed in Ref. 19). This conclusion is derived from a large number of studies using mostly transient cotransfection of the bacterial chloramphenicol acetyltransferase (CAT) gene under control of a potential target promoter/enhancer and the X gene under the control of a heterologous promoter in mammalian cells. Induction of transcription by X usually ranges from 2- to 20-fold depending on the target promoter and cell type; whether this transactivation activity contributes to viral function, however, remains to be determined. Fusion of the X protein to the DNA-binding domain of the bacterial LexA repressor resulted in a protein that can activate transcription from a reporter plasmid bearing lexA operator sequences fused to a minimal promoter (20). Similarly, fusion of the X protein to the DNA-binding domain of transcription factor C/EBP increased the ability of X to activate a reporter containing C/EBP binding sites (21). Attempts to demonstrate sequence-specific DNA binding by the X protein so far have not been successful, and, therefore, it is believed that the X protein belongs to the non-DNA binding viral transactivator family, which is brought into a transcription complex by association with cellular DNA-binding factors. So far, a number of transcription factors, including ATF-2 (22), CREB (22, 23), RPB5 (24), TATA-binding protein (25), and p53 (26), have been shown to interact with the X protein. However, unlike most viral activators, X appears to operate through additional mechanisms. For example, X appears to function as a serine protease inhibitor and regulates the turnover of different cellular transcription factors (27). Additionally, increasing evidence suggests that X may use signal transduction pathways to activate transcription. In this regard, both the protein kinase C (PKC) signaling pathway and mitogen-activated kinases have been shown to be involved in X-mediated transactivation (28–34).

To understand the mechanism of X transactivation, we and others have previously used the yeast two-hybrid screen to identify cellular proteins that interact with X (35, 36). Among the clones analyzed were genes encoding a human homolog of a DNA repair protein (XAP1) (35) and a cellular inhibitor of X (XAP2) (36). We have now sequenced and characterized an additional clone from our two-hybrid screen for encoded proteins capable of binding X. One protein, termed XAP3, appears to have strong homology to a rat PKC-binding protein.

PKC is a large family of phospholipid-dependent kinases involved in cell growth, differentiation, and carcinogenesis (37–41). The mammalian PKC enzyme family consists of at least 10 members that are divided into three groups based on enzymatic properties and common structural features (reviewed in Refs. 42 and 43). The group A (cPKC) (α, βI, βII, and γ) are calcium-dependent kinases whose activities are stimulated by diacylglycerol (DAG) or phorbol esters. The group B (nPKC) (δ, ε, η, and θ) and group C (aPKC) (ζ and λ) are different from those of

* This work was supported by National Cancer Institute Grant R01-CA61257, National Science Foundation Grant MCB-9631067, and American Cancer Society Grant F97USF-1. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) U67322.
* To whom correspondence should be addressed: Molecular Oncology Program, Moffitt Cancer Center & Research Institute, University of South Florida, 12902 Magnolia Dr., Tampa, FL 33612. Tel.: 813-979-6754; Fax: 813-979-6700 or 813-979-3893; E-mail: setoe@moffitt.usf.edu.
* The abbreviations used are: HSV-1, herpes simplex virus 1; HBV, hepatitis B virus; CAT, chloramphenicol acetyltransferase; PKC, protein kinase C; HTLV-1, human T-cell lymphotropic virus type 1; DBD, DNA-binding domain; ONPG, o-nitrophenyl-β-d-galactoside; DAG, diacylglycerol; GST, glutathione S-transferase.

group A in their regulatory domains in that a putative Ca²⁺-binding region is absent. The group C PKCs are different from groups A and B in that they possess a single cysteine-rich Zn²⁺ finger motif in the conserved C1 region. The group B PKCs are calcium-independent but can be stimulated by DAG, whereas the group C PKCs are neither calcium- nor DAG-dependent. A number of proteins have previously been identified that associate with PKC (44–48). Some of these PKC-binding proteins serve as substrates for PKC, while interaction of the various PKC isozymes with other cellular proteins may confer the unique localization of each of the PKC enzymes. Recently, it was demonstrated that the human T-cell lymphotropic virus type 1 (HTLV-1) transactivator Tax stimulates PKC translocation to the particulate cellular membrane fraction, suggesting that Tax activates PKC *in vivo* (49). Further, Tax bound specifically to the α, δ, and η PKC isoenzymes, and the addition of Tax to *in vitro* kinase reaction mixtures led to the phosphorylation of Tax and an increase in the autophosphorylation of PKC.

In this report, we demonstrate that X binds XAP3 *in vivo* and *in vitro*. Second, we found that X and XAP3 can both interact with PKCη. Finally, we show that an overexpression of XAP3 enhances X transactivation. These results strongly suggest the involvement of PKC as one of the mechanisms in X transactivation.

---

**MATERIALS AND METHODS**

**Plasmids—**The following plasmids have been described previously. pGST-X (36) contains the entire HBV-X ORF subcloned into the pGSTag vector (50) in-frame with the glutathione S-transferase (GST) polypeptide. pAS-X (36) contains the HBV (subtype adw2; Ref. 51) fragment from nucleotide 1375 to 1853 (*NcoI-AflIII*) in the Gal4 DNA-binding domain (DBD)-tagged plasmid, pAS1 (52). This plasmid expresses a fusion protein containing the Gal4DBD and the full-length wild type X protein. pSP72-PKCα, pSP72-PKCη, and pSP72-PKCδ (49) contain the different PKC isoform cDNAs under the control of T7 or SP6 phage promoters. pECE-X (53) contains the HBV fragment from nucleotide 1355 to 1987 in the SV40-derived expression vector pECE (54). Reporter plasmids pRSVCAT (55) and pSV2CAT (56) contain the Rous sarcoma virus and simian virus 40 promoter/enhancers upstream of the CAT gene.

Plasmids pGEM-XAP3 was constructed by subcloning an *XhoI* fragment from the X4 clone into the *XhoI* site of a modified pGEM7Zf plasmid (pGEM7Zf-3X; Ref. 57) such that XAP3 mRNA is expressed from a T7 promoter. Different XAP3 mutants were generated by restriction enzyme digestion and religation of the pGEM-XAP3 plasmid. pGEM-TBP1 was constructed by inserting the TAT-binding protein 1 cDNA (58) into the pGEM7Zf vector (Promega). Different X deletion mutants in pAS1 vector were constructed by first subcloning the X coding sequence (*a BglII* fragment from pECE-X) into the *BamHI* site of pGEM7Zf-3X, then using different restriction enzymes to subdivide the X coding region, and finally subcloning each individual X mutant into the *NcoI/BamHI* site of pAS1. pGST-XAP3 was constructed by subcloning an XAP3 cDNA fragment from pGEM-XAP3 into pGSTag. pCMV-XAP3 was constructed by cloning an *XhoI* fragment from clone X4 into the *SalI* site of pcDNAI/Amp (Invitrogen). All plasmid constructions were verified by dideoxy sequencing (59).

**In Vitro Transcription/Translation and in Vitro Binding Assays—**To generate ³⁵S-labeled XAP3, TBP-1, PKCα, PKCη, or PKCδ proteins; pGEM-XAP3, pGEM-TBP1, pSP72PKCα, pSP72PKCη, or pSP72PKCδ were transcribed and translated with T7 or SP6 RNA polymerase and [³⁵S]methionine in a transcription/translation coupled system (Promega). Bacterially expressed GST, GST-X, or GST-XAP3 fusion proteins were purified according to Frangioni *et al.* (60). Briefly, DH5α cells harboring either the pGSTag, pGST-X, or pGST-XAP3 plasmids were grown to log phase and induced with isopropyl-1-thio-β-D-galactopyranoside for 4 h. After sonication in STE buffer (10 mM Tris-HCl (pH 8), 150 mM NaCl, 1 mM EDTA, and 5 mM dithiothreitol) containing 1% sarcosyl (w/v, final concentration), solubilized proteins were recovered by centrifugation and incubated with glutathione-agarose beads in the presence of 3% Triton X-100 (final concentration) for 30 min at 4 °C, and washed several times with ice-cold phosphate-buffered saline.

For binding assays, beads were mixed with *in vitro* translated, ³⁵S-labeled proteins for 1 h at room temperature. Unbound proteins were washed extensively with STE buffer containing 0.1% Nonidet P-40, and

bound proteins were eluted from the beads by boiling in SDS loading buffer (50 mM Tris-HCl (pH 6.8), 100 mM dithiothreitol, 2% SDS, 0.1% bromphenol blue, and 10% glycerol). Final products were analyzed on a 12.5% SDS-polyacrylamide gel and detected by autoradiography.

**Yeast Two-hybrid Interaction Experiments—**To determine XAP3 interacting domains within X, library-derived X4 plasmid was transformed into Y153 alone or Y153 containing either pAS-X or pAS-X mutants. Transformants were assayed for the presence of β-galactosidase activity.

**Screening of cDNA Libraries and DNA Sequencing—**X4, a positive clone from a two-hybrid screen that showed specific interaction with X but not to other unrelated proteins were subcloned into pGEM7Zf-3X, and nucleotide sequences of the subcloned cDNA were obtained using dideoxy sequencing. To obtain a full-length XAP3 cDNA, a HeLa λgt11 cDNA library (CLONTECH) was screened with a ³²P-labeled, random-primed 368-base pair probe (*XhoI/BglII* fragment from clone X4) using standard protocols (61). Inserts from positive clones were subcloned into pGEM7Zf-3X, and sequences were determined. The final sequence of the XAP3 cDNA was determined from both DNA strands. Homology searches for DNA and deduced amino acid sequences were performed at the National Center for Biotechnology Information with the BLAST network service.

**β-Galactosidase Assays—**Filter lift assays were performed essentially as described (62). Briefly, transformants were allowed to grow at 30 °C for 2–4 days, transferred onto nitrocellulose filters, and frozen under liquid nitrogen. Filters were then placed on Whatman no. 3MM paper presoaked with 5-bromo-4-chloro-3-indolyl β-D-galactopyranoside solution, incubated at 30 °C, and checked periodically for production of blue color.

For quantitation of β-galactosidase activity in yeast, liquid culture assays were done using o-nitrophenyl-β-D-galactoside (ONPG) as described (63). Individual transformants were inoculated into the appropriate medium and incubated at 30 °C until the cultures reached mid-log phase (*A*₆₀₀ of 1.0). For each culture, 0.1 ml of culture was mixed with 0.9 ml of Z buffer (60 mM Na₂HPO₄, 40 mM NaH₂PO₄, 10 mM KCl, 1 mM MgSO₄, and 50 mM β-mercaptoethanol). The cells were permeabilized with 0.05 ml of CHCl₃ and 0.05 ml of 0.1% sodium dodecyl sulfate (SDS). After addition of 0.2 ml of ONPG solution (4 mg/ml ONPG in 0.1 M phosphate buffer, pH 7.0), reactions were incubated at 30 °C for 1 h and quenched by the addition of 0.5 ml of 1 M Na₂CO₃. β-Galactosidase activities were determined by measuring absorbances at 420 nm.

**Western Blot Analysis—**Standard protocols were followed (64). For each sample, yeast transformants were grown at 30 °C in 1 ml of selective SC medium containing 2% dextrose to an *A*₆₀₀ of 1–2. Cells were collected by centrifugation, and lysates were prepared according to a standard protocol (61). Briefly, cells were washed in ice-cold 50 mM Tris-HCl (pH 8.0) and resuspended in lysis buffer (50 mM Tris-HCl (pH 8.0), 0.1% Triton X-100, and 0.5% SDS) containing 1 mM phenylmethylsulfonyl fluoride, 1 μg/ml pepstatin A, and 1 μg/ml leupeptin. After addition of acid-washed glass beads (Sigma), samples were vortexed and cell extract recovered. One-tenth of each sample was resolved on a 12.5% SDS-polyacrylamide gel and transferred onto polyvinylidene difluoride membrane. After blocking with nonfat dried milk, the membrane was treated with 1:1000 diluted Gal4-DNA-binding domain polyclonal antiserum (Santa Cruz Biotechnology) followed by 1:7500 diluted alkaline phosphatase-conjugated rabbit anti-mouse IgG. Subsequently, the blot was developed by 5-bromo-4-chloro-3-indolyl phosphate and nitro blue tetrazolium.

**Purification of RNA and Northern Blots—**Total RNA was purified using the acid guanidinium thiocyanate-phenol-chloroform extraction method (65), separated on a formaldehyde agarose gel, and transferred onto a Hybond membrane (Amersham). Multiple human tissue Northern blot was obtained from CLONTECH. Prehybridization, hybridization, and high stringency washes were performed as described (61). To control for the relative amount of RNA in each lane, after hybridization with XAP3, the blots were stripped by incubation in 0.5% SDS at 95 °C, and reprobed with the human β-actin cDNA.

**Transfection and CAT Assay—**HepG2 cells were grown in minimal essential medium supplemented with 10% fetal bovine serum. Transfections were done using the calcium phosphate method (66), which included a 1-min glycerol shock 4 h after the addition of DNA precipitate. Forty-eight h after transfection, cells were harvested and CAT activity was determined (56) in 30-min reactions. All transfections were normalized to equal amounts of DNA with pECE or pcDNAI/Amp plasmids.

---

**RESULTS**

**Sequence Analysis of a cDNA Clone Encoding an X-associated Protein—**Previously, we have used the yeast two-hybrid

16484

**HBV X Interacts with a PKC-binding Protein**

| input | GST | GST-X |
| --- | --- | --- |
|  |  |  |

| MW (KDa) |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| 69 |  |  |  |  |  |  |
| 46 |  |  |  |  |  |  |
| 30 |  |  |  |  |  |  |
| 14 |  |  |  |  |  |  |

|  | 1 | 2 | 3 | 4 | 5 | 6 | 7 |
| --- | --- | --- | --- | --- | --- | --- | --- |

FIG. 1. Specific interaction between X and XAP3 as determined by in vitro binding assays. Input lanes (lanes 2 and 3) were loaded with one-fourth the amount of proteins used in the binding reactions. Equal amounts of GST or GST-X fusion protein were used in each binding assay. Bound proteins were eluted and analyzed in a 12.5% SDS-polyacrylamide gel. ${}^{14}$C-Labeled protein molecular weight markers are indicated in lane 1.

screen to detect human cDNAs coding for products that interact with the X protein (36). Of $5.6 \times 10^6$ colonies screened, 12 of the clones were active only when cotransformed with plasmid encoding Gal4DBD fused to X, but not to Rb, SNF1, or several unrelated transcription factors (36). Preliminary DNA sequence analysis revealed seven novel sequences and five that were highly homologous to known sequences. Of the seven novel sequences, six of them were overlapping clones of cDNA encoding XAP2 (36). Of the five known sequences, we have now completely sequenced one cDNA insert designated X4 (we will refer to the product encoded by X4 as X-associated protein-3 (XAP3)).

**X Binds XAP3 Specifically in Vitro—** To determine whether XAP3 binds X in vitro, we expressed the X protein as a fusion to GST and used it to test for its ability to bind specifically to in vitro ${}^{35}$S-labeled XAP3. As shown in Fig. 1, XAP3 binds to GST-X but not GST (lanes 4 and 6). An unrelated protein, TBP-1, was used as an additional negative control; and, as expected, TBP-1 did not bind GST-X in this assay (lane 7). Taken together, this suggests that X and XAP3 interact both in vivo and in vitro and the interaction is specific.

**Amino Acids 90–122 of X Are Required for Binding to XAP3—** The X protein sequences required for binding XAP3 were examined to determine whether they coincided with previously defined activation or regulatory domains. A series of carboxyl-terminal X deletion mutants, as well as internal deletion mutants previously generated and subcloned into pAS1 were tested for their abilities to interact with XAP3 in the two-hybrid system. As shown in Fig. 2A, deletion of the X protein carboxyl-terminal from amino acids 122–154 had no effect on $\beta$-galactosidase activities, as measured by filter lift or liquid assays. However, deletion of amino acids 109–154 eliminated $\beta$-galactosidase activities. Further amino-terminal and internal deletions revealed that amino acids 90–122 of X were important for binding XAP3. Western blot analysis indicated that mutants which did not bind XAP3 were expressed efficiently in yeast cells (Fig. 2B), indicating that the loss of binding property of these mutants is not a reflection of loss in protein expressions.

**Isolation and Analysis of a Full-length XAP3 cDNA—** The predicted amino acid sequence of XAP3 was determined by theoretical translation of the cDNA clone open reading frame.

|  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  | 
HBV X Interacts with a PKC-binding Protein

A

1 gaattccgggcccgggacccgctgggaccccgcactcggcgctccggcccgtagccggcagttggagggtccccggatg 80  
81 aggcgacaattttttccggccccccctccagtcgcccgcccaacttccgggggcacactttcactttctctttccgccaag 160  
161 ccgctcccccttgtcggaagaactggggccctccgggagagagggcttttgcaaaccgggacgccagggcgctcc 240  
241 cgcaagtggggggtcctccgggaacctgggaaacgcccccggctgggtgdtccgggectctttccccgcttctcccaactc 320  
321 ggctggtcccgttttcctcctgccccagtgccgacctgtctcggcgcccgctgccctctcacccgcccacgcaggatccc 400  
401 ggccctggteatcggggcagtgtgatgcttcccgaactgccgggggaacagcgaggaacacacacagggctttgggcgcgga 480  
481 ggccacacgggctgggctgagttgctcctggctcccgccctctcccaaggcgacccggaggtagcatttccccagggcacg 560  
561 gtccccccccaggggg ATG GGC ACA GCC ACG CCA GAT GGA CGA GAA GAC CAA GAA AGG CTG TGG 623  
1 M G T A T P D G R E D Q E R L W V S V E D A Q M H T V T I W L T V R P D M T V A S L K D M V F L D Y 80  
17 V S V E D A Q M H T V T I W L T V R P D 36  
624 GTG AGC GTG GAG GAT GCT CAG ATG CAC ACC GTC ACC ATC TGG CTC ACA GTG CGC CCT GAT 683  
37 M T V A S L K D M V F L D Y G F P P V L 56  
684 ATG ACA GTG GCG TCT CTC AAG GAC ATG GTT TTT CTG GAC TAT GGC TTC CCA CCA GTC TTG 743  
57 Q Q W V I G Q R L A R D Q O T L H S H G V R Q N G D S A Y L Y L L S A R N T S L N P Q E 76  
744 CAG CAG TGG GTG ATT GGG CAG CTG GCA CGA GAC CAG GAG ACC CTG CAC TCC CAT GGG 803  
77 V R Q N G D S A Y L Y L S A R N T S L 96  
804 GTG CGG CAG AAT GGG GAC AGT GCC TAC CTC TAT CTG TCA GCC CGC AAC ACC TCC CTC 863  
97 N P Q E L Q R L R M L E D L G F K 116  
924 GAC CTC ACG CTG CAG CCG CGG GGC CCT CTG GAG CCA GGC CCC CCA AAG CCC GGC CCC 983  
117 D L T L Q P R G P L E P G P P K P G V P 136  
984 CAG GAA CCC GGA CGG GGG CAG CCA GAT GCA GTG CCT GAG CCC CCA CCG GTG GGC TGG CAG 1043  
137 Q E P G R Q P D A V P E P P V G W Q 156  
1044 TGC CCC GGG TGC ACC TTC ATC AAC AAG CCC ACG CGG CCT GGC TGT GAG ATG TGC TGC CGG 1103  
157 C P G C T F I N K P T R P G G A G G A G G A G G A G G C G 176  
1104 GCG CGC CCC GAG GCC TAC CAG GTC CCC GCC TCA TAC CAG CCC GAC GAG GAG GAG CGA GCG 1163  
177 A R P E A Y Q V P A S Y Q P D E E R A 196  
1164 CGC CTG CGC GGC GAG GAG GCG CTG CGT CAG TAC CAG CAG AAG CAG CAG CAG CAG 1223  
197 R L A G E E A L R Q Y Q R Q Q 216  
1224 GAG GGG AAC TAC CTG CAG CAC GTC CAG GAC CAG AGG AGC CTG GTG CTG AAC ACG GAG 1283  
217 E G N Y L Q H V Q L D Q R S L V L N T E 236  
1284 CCC GCC GAG TGC CCC GTG TGC TAC TCG GTG CCC GGC GAG GCC GTG GTG CTG CGT 1343  
237 P A E C P V C Y S V L A P G E A V V L R 256  
1344 GAG TGT CTG CAC ACC TTC TGC AGG GAG TGC CTG CAG GGC ACC ATC CGC AAC AGC CAG GAG 1403  
257 E C L H T F C R E C L O G T I R N S Q E A E V S C P F I D N T Y S C P G K L L E R E I R 276  
1404 GCG GAG GTC TCC TGC CCC TTC ATT GAC AAC ACC TAC TCG TGC TCG GGC AAG CTG CTG GAG 1463  
277 A E V S C P F I D N T Y S C S G K L L E 296  
1464 AGG GAG ATC AAG GCG CTC CTG ACC CCT GAG GAT TAC CAG CGA TTT CTA GAC CTG GGC ATC 1523  
297 R E I K A L L T P E D Q T L H S G I R R N G D S A Y X Y L L S A R N T S L N P Q E 316  
1524 TCC ATT GCT GAA AAC CGC AGT GCC TTC AGC TAC CAT TGC AAG ACC CCA GAT TGC AAG GGA 1583  
317 S I A E N R S A F S Y H C K T P D C K G W C F F E D D V N E F T C P 336  
1584 TGG TGC TTC TTT GAG GAT GAT GTC AAT GAG TTC ACC TGC CCT GTG TGT TTC CAC GTC AAC 1643  
337 W C F F E D V N E F T C P V C F H V N C L L C K A I H E O M N C K E Y O E D L A L R A Q N D V A A R Q T T E M L K V M L Q Q G 356  
1644 TGC CTG CTC TGC AAG GCC ATC CAT GAG CAG ATG AAC TGC AAG GAG TAT CAG GAG GAC CTG 1703  
357 C L L C K A I H E Q M N C K E Y O E D L A L R A Q N D V A A R Q T T E M L K V M L Q Q G 376  
1704 GCC CTG CGG GCT CAG AAC GAT GTG GCT GCC CGG CAG ACG ACA GAG ATG CTG AAG GTG ATG 1763  
377 A L R A Q N D V A A R Q T T E M L K V M 396  
1764 CTG CAG CAG GGC GAG GCC ATG CGC TGC CCC CAG TGC CAG ATC GTG GTA CAG AAG AAG GAC 1823  
397 L Q Q G E A M R C P Q C Q I V V Q K K D 416  
1824 GGC TGC GAC TGG ATC CGC TGC ACC GTC TGC CAC ACC GAG ATC TGC TGG GTC ACC AAG GGC 1883  
417 G C D W I R C T V C H T E I C W V T K G 436  
1884 CCA CGC TGG GGC CCT GGG GGC CCA GGA GAC ACC AGC GGG GGC TGC CGC TGC AGG GTA AAT 1943  
437 P R W G P G G D T S G G C R V N 456  
1944 GGG ATT CCT TGC CAC CCA AGC TGT CAG AAC TGC CAC TGA gctaaagatggtggggccacatgctgac 2010  
457 G I P C H P S C Q N C H * 469  
2011 ccagccccacatccacattctgtttagaatgtagctcagggagcttcgtggacgggcccttgcttgctgtagcgttgtgagggg 2090  
2091 ccctgctgactgctgggtgtccacggtcacatctgcccacagtgcctttgtcctttccctgggcttgccgggcagactt 2170  
2171 ctctccctcgggctcccaacctctgcctgaccccaagcttaaacatagccccctgggagcccttgctgggggagacct 2250  
2251 ctgtgtgactccatactcctcccaaccaacactatotgtcaaacaccaagcactctcagcctcccccgcttcaagctgt 2330  
2331 cagcttctcgggggctaacttctctgccccttgtggggagccctgagggcctcttggaactcttgctaaacctgttcagagc 2410  
2411 caggaaggagactgcacagtttgtgaaaacacagcccgtdaggccggctctgcgtctccctctcgagcctgtgtaago 2490  
2491 tattataattaaaaatggttttccggccacgaaggccggccttcgtgggcctcgag 2543  

B

XAP3 MGTATPDGREDQERLWVSVEDAQMHTVTIWLTVRPDMTVASLKDMVFLDY 50  
rPKCBP VQVKPEVSPTQDIRLCVSVEDAYMHTVTIWLTVRPDMTVASLKDMVFLDY 82  
XAP3 GFPPVLQQWIGQRLARDQETLHSHGVQRNGDSAYLYLLSARNTSLN PQE 100  
rPKCBP GFXPSLQQWVGQRLARDQETLHSGIRRNGDSAYXYLLSARNTSLN PQE 132  
XAP3 LQERQLRMLEDLGFKDLTLOPRGPLEPGPPKPGVPQEPGRGQPDAVPEP 150  
rPKCBP LQRQRQWXCW.KFGLQGPHPAATGPLNLSSP....TQTHQRAANAAPES 177  
XAP3 PPVGWQCPGCTFINKPTRPGEMCCRARPEAYQVPASYQPDEEERARLAG 200  
rPKCBP PPVGWQCPGCTFINKPTRPGEMCCRASLRPTRSLPRTSLMRRKRARLAG 227  
XAP3 EEEALRQYQRKQQQEGNYLQHVQLDQRSVLNTEPAECPVCYSVLAPG 250  
rPKCBP EGGGLRQYEQRKQQQEGNYLQHVQLEQRSVLNTEPAECPVCYSVLAPG 277  
XAP3 EAVVLRECLHTFCRECLQGTIRNSQEAEVSCPFIDNTYSCGKLLEREIK 300  
rPKCBP EAVVLRECLHTFCRECLQGTIRNSQEAEVSCPFIDNTYSCPGKLLEREIR 327  
XAP3 ALLTPEDYQRFLLDLGISIAENRSAFSYHCKTPDCKGWCFEEDDVNEFTCP 350  
rPKCBP ALLSPEDYQRFLLDGVSIAENRSTLSYHCKTPDCRGWCFFEEDDVNEFTCP 377  
XAP3 VCFHVNCLLCKAIHEQMNCKEYOEDLALRAQNDDVAARQTTEMLKVMLQQG 400  
rPKCBP VCTRNVNCLLCKAIHERMNCREYQDDLAHRARNDVAA...........S 414  
XAP3 EAMRC 405  
rPKCBP RQLRC 419

16486

**HBV X Interacts with a PKC-binding Protein**

XAP3  
![image](https://via.placeholder.com/200x200)

$\beta$-actin  
![image](https://via.placeholder.com/200x200)

1 2 3 4 5 6 7 8

FIG. 4. Northern blot analysis of XAP3 RNA from human tissues. Northern blot analysis of poly(A)$^+$ mRNA were performed on multiple normal human tissues. Hybridization to a human $\beta$-actin probe is shown as a control.

Northern blot analysis was performed to determine the expression pattern of XAP3. We found that a message of approximately 2.5–3.0 kilobases was present in high levels in human brain, placenta, and pancreas; intermediate levels in the heart and skeletal muscle; and low levels in the lung, liver, and kidney (Fig. 4).

**X-binding Domain in XAP3**—To localize a domain within XAP3 that binds the X protein, we in vitro synthesized different $^{35}$S-labeled XAP3 mutants and used them to test for binding to GST-X. As shown in Fig. 5, carboxyl-terminal deletion of amino acids 212–468 or internal deletions of amino acids 97–262 or 123–347 had little or no effect on the binding of XAP3 to the X protein (lanes 3–10, 12, and 13). However, deletion of amino acids 13–74 completely eliminated XAP3’s ability to bind X (lane 11). Taken together, these data suggest that the amino-terminal part of the XAP3 protein (amino acids 13–74 in particular) is essential for binding to the X protein. This finding is consistent with the observation that amino acids 13 to 74 is included in the original X4 clone derived from the initial two-hybrid screen.

**X and XAP3 Interact with PKC in Vitro**—GST binding assays were performed to analyze X protein binding to $^{35}$S-labeled PKC isoenzymes translated in vitro with rabbit reticulocyte lysates. Equal quantities of bacterial expressed GST or GST-X fusion protein coupled to glutathione-Sephrose beads were incubated with PKC$\alpha$, PKC$\eta$, or PKC$\delta$. Interestingly, PKC$\eta$ but not PKC$\alpha$ or PKC$\delta$ bound specifically to GST-X but not to GST (Fig. 6, lanes 4–9).

To determine which PKC isoenzyme XAP3 binds, similar experiments were performed using GST-XAP3 fusion protein. Intriguingly, like the X protein, XAP3 also bound specifically to PKC$\eta$ but not PKC$\alpha$ or PKC$\delta$ (lanes 10–12). This result, therefore, suggests a functional significance of the interaction between X and XAP3.

![image](https://via.placeholder.com/200x200)

binding to X  
+ + + + + + + + + + + +  

GST  
MW  
(KDa)  
69  
46  
29  
18.4  
14.3  

1 2 3 4 5 6 7 8 9 10 11 12 13  

FIG. 5. X-binding domain in the XAP3 protein. Top panel, schematic drawing of wild type and XAP3 deletion mutants. The abilities of wild type and mutant XAP3 to bind the X protein is indicated (+). Bottom panel, in vitro transcribed and translated XAP3 wild type and mutant proteins were incubated with equal amount of GST or GST-X, washed, eluted, and analyzed by polyacrylamide gels as described under “Materials and Methods.” The size of molecular weight marker is indicated to the left.

![image](https://via.placeholder.com/200x200)

input GST GST-X GST-XAP3  
MW  
(KDa)  
97  
69  
46  
29  

1 2 3 4 5 6 7 8 9 10 11 12  

FIG. 6. Interactions between PKC and X or XAP3 proteins. Different in vitro transcribed and translated PKC proteins were incubated with either GST, GST-X or GST-XAP3 as described under “Materials and Methods.” Input lanes (lanes 1, 2, and 3) were loaded with one-fourth the amount of proteins used in the binding reactions. The size of molecular weight marker is indicated to the left.

**Stimulation of X Transactivation by XAP3**—To determine the role of XAP3 in X transactivation, we tested the ability of XAP3 to influence transcription directed by an X-responsive promoter in the presence of the X protein. We constructed a plasmid that expresses XAP3 under the control of the human cytomegalovirus immediate early promoter, cotransfected it into HepG2 cells together with a plasmid that expresses the X protein and a reporter. While the expression of XAP3 had no effect on the reporter plasmid (Fig. 7A, lane 2), the X protein activated the Rous sarcoma virus promoter in HepG2 cells about 10-fold as expected (Fig. 7A, compare lanes 1 and 3). Interestingly, when XAP3 expression plasmid was added to the transfections, transcriptional activation by X was further enhanced 7-fold (Fig. 7A, compare lanes 3 and 4). This enhance-

HBV X Interacts with a PKC-binding Protein

a probe to demonstrate that the increase in activity seen in cotransfection of X and XAP3 is due to overexpression of XAP3. As shown in Fig. 7C, XAP3 RNA recovered from HepG2 cells after transfection with X and XAP3 expression plasmids (lane 4) was much higher than from nontransfected cells (lane 1) or cells transfected with plasmids expressing either XAP3 or X alone (lanes 2 and 3, respectively).

DISCUSSION

The identification and characterization of proteins in a cell with which a given protein interacts is often helpful for understanding the function and mechanisms of action of that protein. To gain insight into the function and mechanisms of the X protein, we and others have previously attempted to identify cellular proteins that interact with X. Not surprisingly, many of these X-associated proteins identified were nuclear transcription factors. Therefore, like many viral transcription activators, X may be directed to a promoter to activate transcription by interaction with these nuclear factors. Additionally, X might activate transcription by increasing the activity of these X-associated factors. However, there is ample evidence that the X protein may function by additional mechanisms. In particular, there are well documented reports that the X protein acts through cytoplasmic signaling pathways involving the serine/threonine kinases, PKC, and Raf-1. Our current finding that X interacts with a PKC-binding protein, XAP3, strengthens previous studies suggesting that X is a dual cytoplasmic activator of signal transduction and nuclear activator of transcription factors (32).

X and XAP3 interaction occurred in yeast cells, as well as in a cell-free extract. A detailed analysis of X mutants indicated that the interaction with XAP3 occurred through X residues 90–122. Segments of the X protein required for transactivation have been analyzed previously by many laboratories (67–73). Although there are some disagreements on the exact amino acids essential for X transactivation, most reports suggest that amino acids 105–115 are required for X transactivation function. Therefore, it appears that XAP3 binding correlates with X transactivating activity. Our result is also consistent with an earlier study suggesting that amino acids 105–115 of the X protein interact with cellular factors (74).

XAP3 contains a C3HC4 (RING finger) zinc-binding domain that is also present in a number of other transcription factors and proto-oncogenes (reviewed in Ref. 75). Currently, the exact function of the RING finger motif is unclear. Although RING finger-containing proteins, such as BRCA-1 (76) and PML (77), have been strongly implicated in cell growth regulation and transcription. Extensive efforts to identify sequence-specific DNA binding activity for such domains have been unsuccessful, and it appears most likely that RING finger domains participate in protein-protein interactions. Our data suggest that X and XAP3 interaction occurs through the extreme amino-terminal region of XAP3, outside the RING finger domain. Thus, it is possible that XAP3 uses its RING finger domain to interact with PKC or other as yet unidentified proteins.

Previous studies suggest that the HTLV-1 transactivator Tax protein binds specifically to the α, δ, and η PKC isoenzymes (49). Here, we demonstrate that X and XAP3 interact selectively with PKCη, suggesting that the three proteins may form a trimeric complex *in vivo*. It is not known at this time whether X can directly bind PKCη, since the extract used for the binding assay probably contains XAP3. In addition to the three PKC isoenzymes we have tested, there exist at least six other PKC subunits. Currently, we do not know whether these other subunits also bind X and XAP3 or whether they are important for the regulation of transcription by X. Our results showing interaction of one of the PKC isoenzymes with X and

FIG. 7. Enhancement of X transactivation by the XAP3 protein. A, representative CAT assays with RSVCAT or SV2CAT reporter plasmids (10 μg each) transfected into HepG2 cells in the presence or absence of plasmids encoding X (3 μg) and XAP3 (10 μg). B, CAT assay results with Rous sarcoma virus promoter CAT construct (10 μg each) transfected into HepG2 cells in the presence of the indicated amount of XAP3 and X expression plasmids. The extent of acetylation in various reactions was determined relative to that for the CAT reporter, and results are presented as the mean of three independent transfections with standard deviations. C, comparison of XAP3 expression in transfected HepG2 cells with Northern blot.

ment of X transactivation was not due to a higher expression levels of the X protein, since XAP3 has no effect on the SV40 promoter used to produce X (Fig. 7A, lanes 5 and 6). Activation by X and XAP3 was dose-dependent with an optimum concentration of 10–15 μg of transfected XAP3 plasmid (Fig. 7B). Northern blot analysis was carried out with XAP3 cDNA as

16488

**HBV X Interacts with a PKC-binding Protein**

XAP3 do not exclude the possibility that other isoenzymes may also be important for X activation. Further experiments will address these issues.

How might X-XAP3 interaction influence X's ability to activate transcription? An inspection of the X protein sequence revealed that it possess recognition motifs for PKC (78, 79) (amino acids 54–56 and 75–77), and by metabolic labeling, it has been shown that the X protein is capable of being phosphorylated (80). Therefore, it is tempting to speculate that XAP3 may serve as a bridging protein connecting X and PKC, which then facilitates the phosphorylation of X. The consequence of X phosphorylation could be a redistribution of the X protein to a different cellular compartment, resulting in up-regulation of transcription. This would be similar to the transcription factor NF-κB whose ability to activate transcription is regulated by phosphorylation and association and dissociation with the cytoplasmically located inhibitor IκB (81, 82). Our data showing that overexpression of XAP3 enhances X transactivation fits well with this model. Studies are now under way to determine the ability of PKC to phosphorylate X in the presence of XAP3.

Another possibility is that perhaps by interaction with XAP3, X is drawn to the signaling cascade to activate PKC. This model should be interpreted with caution, however, as there is only one report that X causes membrane translocation of PKC, elevation of the endogenous PKC activator DAG, and subsequent activation of PKC (29). In contrast, more studies have shown that the X protein actually does not directly activate PKC. Therefore, it is more reasonable to speculate that through interaction with XAP3, X might not directly activate PKC but instead acts on a subsequent step after PKC activation. One possibility is that perhaps by association with XAP3, X can then cause an increase of autophosphorylation of different PKC subunits or a decrease in PKC turnover. Additionally, it is conceivable that XAP3 may facilitate X to target PKC to a particular subcellular compartment such as the nucleus, where PKC can phosphorylate and activate other transcription factors that regulate gene expression.

It is worth mentioning that a casual inspection of the XAP3 sequence revealed regions that are characteristic of transcriptional activation domains (reviewed in Ref. 83): proline-rich (amino acids 122–179; 29%) and glutamine-rich (amino acids 183–228; 26%). This raises an additional possibility that an alternative, although not mutually exclusive, model is that X recruits an activation domain on XAP3 to a promoter region to activate transcription. In this scenario, a DNA sequence-specific binding protein recruits X to a promoter, which in turn recruits XAP3 to the same promoter to activate transcription.

Numerous transcriptional regulatory sequences from heterologous viral and cellular promoters have been reported as targets for transactivation by the X protein. However, in many cases conflicting results have been obtained for the capacity of X to transactivate a certain regulatory sequence. It has been suggested that this discrepancy may be due to differences of cell lines used for each assay, and that X might transactivate through cellular proteins that are cell type-specific (84). Our previous finding (36) that a X-associated factor, XAP2, inhibits X activation, coupled with the current results that XAP3 enhances X activation, may partially explain the cell type-specific activation phenomenon by X. Perhaps different cell types contain different levels of XAP2 and XAP3, and the net effect on X's transactivation ability is dependent on the delicate concentration of these cellular proteins.

Finally, as mentioned before, PKC is involved in regulation of cell growth, differentiation, and carcinogenesis. Overexpression of wild type or mutant PKC may lead to disordered cell growth and transformation. It will be interesting to further dissect the interaction between X, PKC, and XAP3 and deduce the biological significance of these interactions. In addition to its transactivation ability, X has been shown to transform cells (85–87) and to cause liver cancer in some transgenic mice (88, 89). It is, therefore, possible that X targets some particular PKC isoforms and their binding proteins to form a complex that mediates the control of cell growth.

Acknowledgments—We thank John Brady and Paul Lindholm for PKC plasmids and Denise Cooper, Rich Jove, Nancy Olashaw, and Jerry Wu for discussion and critical reading of the manuscript.

---

**REFERENCES**

1. Faber, S. W., and Wilcox, K. W. (1986) *Nucleic Acids Res.* **14**, 6067–6083
2. Michael, N., Spector, D., Mavromara-Nazos, P., Kristie, T. M., and Roizman, B. (1988) *Science* **239**, 1531–1534
3. Farrell, P. J., Rowe, D. T., Rooney, C. M., and Kouzarides, T. (1989) *EMBO J.* **8**, 127–132
4. Spalholz, B. A., Byrne, J. C., and Howley, P. M. (1988) *J. Virol.* **62**, 3143–3150
5. Fried, M., and Prives, C. (1986) *Cancer Cells* **4**, 1–16
6. Kristie, T. M., and Roizman, B. (1984) *Proc. Natl. Acad. Sci. U.S.A.* **81**, 4065–4069
7. McKnight, J. L. C., Kristie, T. M., and Roizman, B. (1987) *Proc. Natl. Acad. Sci. U.S.A.* **84**, 7061–7065
8. Abmayr, S. M., Feldman, L. D., and Roeder, R. G. (1985) *Cell* **43**, 821–829
9. Abmayr, S. M., Workman, J. L., and Roeder, R. G. (1988) *Genes Dev.* **2**, 542–553
10. Ferguson, B., Kripl, B., Andrisani, O., Jones, N., Westphal, H., and Rosenberg, M. (1985) *Mol. Cell. Biol.* **5**, 2653–2661
11. Chatterjee, P. K., Bruner, M., Flint, S. J., and Harter, M. L. (1988) *EMBO J.* **7**, 835–841
12. Gerster, T., and Roeder, R. G. (1988) *Proc. Natl. Acad. Sci. U.S.A.* **85**, 6347–6351
13. O'Hare, P., and Goding, C. R. (1988) *Cell* **52**, 435–445
14. O'Hare, P., Goding, C. R., and Haigh, A. (1988) *EMBO J.* **7**, 4231–4238
15. Preston, C. M., Frame, M. C., and Campbell, M. E. M. (1988) *Cell* **52**, 425–434
16. Kristie, T. M., LeBowitz, J. H., and Sharp, P. A. (1989) *EMBO J.* **8**, 4229–4238
17. Stern, S., Tanaka, M., and Herr, W. (1989) *Nature* **341**, 624–630
18. Liu, F., and Green, M. R. (1994) *Nature* **368**, 520–525
19. Rossner, M. T. (1992) *J. Med. Virol.* **36**, 101–117
20. Seto, E., Mitchell, P. J., and Yen, T. S. B. (1990) *Nature* **344**, 72–74
21. Unger, T., and Shaul, Y. (1990) *EMBO J.* **9**, 1889–1895
22. Maguire, H. F., Hoeffler, J. P., and Siddiqui, A. (1991) *Science* **252**, 842–844
23. Williams, J. S., and Andrisani, O. M. (1995) *Proc. Natl. Acad. Sci. U.S.A.* **92**, 3819–3823
24. Cheong, J., Yi, M., Lin, Y., and Murakami, S. (1995) *EMBO J.* **14**, 143–150
25. Qadri, I., Maguire, H. F., and Siddiqui, A. (1995) *Proc. Natl. Acad. Sci. U.S.A.* **92**, 1003–1007
26. Feitelson, M. A., Zhu, M., Duan, L. X., and London, W. T. (1993) *Oncogene* **8**, 1109–1117
27. Takada, S., Kido, H., Fukutomi, A., Mori, T., and Koike, K. (1994) *Oncogene* **9**, 341–348
28. Cross, J. C., Wen, P., and Rutter, W. J. (1993) *Proc. Natl. Acad. Sci. U.S.A.* **90**, 8078–8082
29. Kekulé, A. S., Lauer, U., Weiss, L., Luber, B., and Hofschneider, P. H. (1993) *Nature* **361**, 742–745
30. Lucito, R., and Schneider, R. J. (1992) *J. Virol.* **66**, 983–991
31. Benn, J., and Schneider, R. J. (1994) *Proc. Natl. Acad. Sci. U.S.A.* **91**, 10350–10354
32. Doria, M., Klein, N., Lucito, R., and Schneider, R. J. (1995) *EMBO J.* **14**, 4747–4757
33. Benn, J., Su, F., Doria, M., and Schneider, R. J. (1996) *J. Virol.* **70**, 4978–4985
34. Wang, H.-D., Yuh, C.-H., Dang, C. V., and Johnson, D. (1995) *Mol. Cell. Biol.* **15**, 6720–6728
35. Lee, T. H., Elledge, S. J., and Butel, J. S. (1995) *J. Virol.* **69**, 1107–1114
36. Kuzhandai velu, N., Cong, Y.-S., Inouye, C., Yang, W.-M., and Seto, E. (1996) *Nucleic Acids Res.* **24**, 4741–4750
37. Housey, G. M., Johnson, M. D., Hsiao, W. L. W., O'Brian, C. A., Murphy, J. P., Kirschmeier, P., and Weinstein, I. B. (1988) *Cell* **52**, 343–354
38. Stabel, S. (1994) *Semin. Cancer Biol.* **5**, 277–284
39. Nishizuka, Y. (1989) *J. Am. Med. Assoc.* **262**, 1826–1833
40. Jaken, S. (1990) *Curr. Opin. Cell Biol.* **2**, 192–197
41. Glazer, R. I. (1994) in *Protein Kinase C* (Kuo, J. F., ed) pp. 171–198, Oxford University Press, New York
42. Kikkawa, U., Kishimoto, A., and Nishizuka, Y. (1989) *Annu. Rev. Biochem.* **58**, 31–44
43. Parker, P. J. (1994) in *Protein Kinase C* (Kuo, J. F., ed) pp. 3–15, Oxford University Press, New York
44. Chapline, C., Ramsay, K., Klauck, T., and Jaken, S. (1993) *J. Biol. Chem.* **268**, 6858–6861
45. Dong, L., Chapline, C., Mousseau, B., Fowler, L., Ramsay, K., Stevens, J. L., and Jaken, S. (1995) *J. Biol. Chem.* **270**, 25534–25540
46. Mochly-Rosen, D., Khaner, H., and Lopez, J. (1991) *Proc. Natl. Acad. Sci. U.S.A.* **88**, 3997–4000
47. Mochly-Rosen, D., Khaner, H., Lopez, J., and Smith, B. L. (1991) *J. Biol. Chem.* **266**, 14866–14868
48. Wolf, M., and Sayhoun, N. (1986) *J. Biol. Chem.* **261**, 13327–13332
49. Lindholm, P. F., Tamami, M., Makowski, J., and Brady, J. N. (1996) *J. Virol.* **70**, 2525–2532
50. Ron, D., and Dressler, H. (1992) *Biotechniques* **13**, 866–869

51. Valenzuela, P., Quiroga, M., Zaldivar, J., Gray, W., and Rutter, W. J. (1980) in *Animal Virus Genetics* (Fields, B. N., Jaenisch, R., and Fox, C. F., eds) pp. 57–70, Academic Press, New York

52. Durfee, T., Becherer, K., Chen, P. L., Yeh, S. H., Yang, Y., Kilburn, A. E., Lee, W. H., and Elledge, S. J. (1993) *Genes Dev.* **7**, 555–569

53. Seto, E., Yen, T. S. B., Peterlin, B. M., and Ou, J. H. (1988) *Proc. Natl. Acad. Sci. U. S. A.* **85**, 8286–8290

54. Ellis, L., Clauser, E., Morgan, D. O., Edery, M., Roth, R. A., and Rutter, W. J. (1986) *Cell* **45**, 721–732

55. Gorman, C. M., Merlino, G. T., Willingham, M. C., Pastan, I., and Howard, B. H. (1982) *Proc. Natl. Acad. Sci. U. S. A.* **79**, 6777–6781

56. Gorman, C. M., Moffat, L. F., and Howard, B. H. (1982) *Mol. Cell. Biol.* **2**, 1044–1051

57. Yang, W.-M., Inouye, C., Zeng, Y., Bearss, D., and Seto, E. (1996) *Proc. Natl. Acad. Sci. U. S. A.* **93**, 12845–12850

58. Nelbock, P., Dillon, P. J., Perkins, A., and Rosen, C. A. (1990) *Science* **248**, 1650–1653

59. Sanger, F., Nicklen, S., and Coulson, A. R. (1977) *Proc. Natl. Acad. Sci. U. S. A.* **74**, 5463–5467

60. Frangioni, J. V., and Neel, B. G. (1993) *Anal. Biochem.* **210**, 179–187

61. Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) *Molecular Cloning: A Laboratory Manual*, 2nd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, NY

62. Breeden, L., and Nasmyth, K. (1985) *Cold Spring Harbor Symp. Quant. Biol.* **50**, 643–650

63. Guarente, L. (1983) *Methods Enzymol.* **101**, 181–191

64. Harlow, E., and Lane, D. (1988) *Antibodies: A Laboratory Manual*, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY

65. Chomczynski, P., and Sacchi, N. (1987) *Anal. Biochem.* **162**, 156–159

66. Graham, F., and van der Eb, A. (1973) *Virology* **52**, 456–457

67. Faktor, O., and Shaul, Y. (1990) *Oncogene* **5**, 867–872

68. Ritter, S. E., Whitten, T. M., Quets, A. T., and Schloemer, R. H. (1991) *Virology* **182**, 841–845

69. Murakami, S., Cheong, J., and Kaneko, S. (1994) *J. Biol. Chem.* **269**, 15118–15123

70. Levrero, M., Balsano, C., Natoli, G., Avantaggiati, M. L., and Elfassi, E. (1990) *J. Virol.* **64**, 3082–3086

71. Balsano, C., Avantaggiati, M. L., Natoli, G., DeMarzio, E., Will, H., Perricaudet, M., and Levrero, M. (1991) *Biochem. Biophys. Res. Commun.* **176**, 985–992

72. Arii, M., Takada, S., and Koike, K. (1992) *Oncogene* **7**, 397–403

73. Kumar, V., Jayasuryan, N., and Kumar, R. (1996) *Proc. Natl. Acad. Sci. U. S. A.* **93**, 5647–5652

74. Takada, S., and Koike, K. (1994) *Virology* **205**, 503–510

75. Berg, J. M., and Shi, Y. (1996) *Science* **271**, 1081–1085

76. Miki, Y., Swensen, D., Shattuck-Eidens, P. A., *et al.* (1994) *Science* **266**, 66–71

77. Kakizuka, A., Miller, W. H., Umesono, K., Warrell, R. P., Frankel, S. R., Murty, V. V., Dmitrovsky, E., and Evans, R. M. (1991) *Cell* **66**, 663–674

78. Mahoney, C. W., and Huang, K.-P. (1994) in *Protein Kinase C,* (Kuo, J. F., ed) pp. 16–63, Oxford University Press, New York

79. Pearson, R. B., and Kemp, B. E. (1991) *Methods Enzymol.* **200**, 62–81

80. Schek, N., Bartenschlager, R., Kuhn, C., and Schaller, H. (1991) *Oncogene* **6**, 1735–1744

81. Baeuerle, P. A., and Baltimore, D. (1988) *Cell* **53**, 211–217

82. Baeuerle, P. A., and Baltimore, D. (1988) *Science* **242**, 540–546

83. Mitchell, P. J., and Tjian, R. (1989) *Science* **245**, 371–378

84. Seto, E., Zhou, D. X., Peterlin, B. M., and Yen, T. S. B. (1989) *Virology* **173**, 764–766

85. Shirakata, Y., Kawada, M., Fujiki, Y., Sano, H., Oda, M., Yaginuma, K., Kobayashi, M., and Koike, K. (1989) *Jpn. J. Cancer Res.* **80**, 617–621

86. Höhne, M., Schaefer, S., Seifer, M., Feitelson, M. A., Paul, D., and Gerlich, W. H. (1990) *EMBO J.* **9**, 1137–1145

87. Schaefer, S., and Gerlich, W. H. (1995) *Intervirology* **38**, 143–154

88. Kim, C. M., Koike, K., Saito, I., Miyamura, T., and Jay, G. (1991) *Nature* **351**, 317–320

89. Koike, K. (1995) *Intervirology* **38**, 134–142

NUCLEIC ACIDS, PROTEIN
SYNTHESIS, AND MOLECULAR
GENETICS:
The Hepatitis B Virus X-associated Protein,
XAP3, Is a Protein Kinase C-binding
Protein

Yu-Sheng Cong, Ya-Li Yao, Wen-Ming
Yang, Nadarajan Kuzhandai Velu and Edward
Seto
J. Biol. Chem. 1997, 272:16482-16489.
doi: 10.1074/jbc.272.26.16482

Access the most updated version of this article at http://www.jbc.org/content/272/26/16482

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 83 references, 39 of which can be accessed free at
http://www.jbc.org/content/272/26/16482.full.html#ref-list-1
