
Molecular Microbiology (1997) 24(5), 1049–1060

# 'Locked-on' and 'locked-off' signal transduction mutations in the periplasmic domain of the Escherichia coli NarQ and NarX sensors affect nitrate- and nitrite-dependent regulation by NarL and NarP

**Robin C. Chiang,** Ricardo Cavicchioli† and Robert P. Gunsalus*

Department of Microbiology and Molecular Genetics, and the Molecular Biology Institute, 1602 Molecular Sciences Building, University of California, Los Angeles, California 90095-1489, USA.

## Summary

The *Escherichia coli* NarX, NarQ, NarL and NarP proteins comprise a two-component regulatory system that controls the expression of many anaerobic electron-transport and fermentation-related genes in response to nitrate and nitrite. Either of the two sensor-transmitter proteins, NarX and NarQ, can activate the response-regulator proteins, NarL and NarP, which in turn are able to bind at their respective DNA regulatory sites to modulate gene expression. NarX contains a conserved 17 amino acid sequence, designated the 'P-box' element, that is essential for nitrate sensing. In this study we characterize *narQ* mutants that also confer altered nitrate control of NarL-dependent nitrate reductase (*narGHJI*) and fumarate reductase (*frdABCD*) gene expression. While some *narQ* mutations cause the constitutive activation or repression of reporter-gene expression even when the cells are grown in the absence of the nitrate signal (i.e. a 'locked-on' phenotype), other mutations abolish nitrate-dependent control (i.e. a 'locked-off' phenotype). Interestingly the *narQ* (A42→T) and *narQ* (R50→Q) mutations along with the analogous *narX18* (A46→T) and *narX902* (R54→E) mutations also confer a 'locked-on' or a 'locked-off' phenotype in response to nitrite, the second environmental signal detected by NarQ and NarX. Furthermore, these *narQ* and *narX* mutations also affect NarP-dependent gene regulation

of nitrite reductase (*nrfABCDEFG*) and *aeg-46.5* gene expression in response to nitrite. We therefore propose that the NarQ sensor-transmitter protein also detects nitrate and nitrite in the periplasmic space via its periplasmic domain. A signal transduction model, which we previously proposed for NarX, is now extended to NarQ, in which a nitrate- or nitrite-detection event in the periplasmic region of the cell is followed by a signal transduction event through the inner membrane to the cytoplasmic domain of NarQ and NarX proteins to modulate their protein kinase/phosphatase activities.

## Introduction

Nitrate- and nitrite-dependent gene expression in *Escherichia coli* is mediated by the NarX/NarQ/NarL/NarP two-component regulatory system that consists of two homologous sensor-transmitter proteins, NarX and NarQ (encoded by *narX* and *narQ*), and two homologous response-regulator proteins, NarL and NarP (encoded by *narL* and *narP*) (reviewed in Darwin and Stewart, 1996; Gunsalus, 1992; Stewart, 1993). NarX and NarQ can independently detect the presence of nitrate and nitrite in the cell environment (Chiang *et al.*, 1992; Kalman and Gunsalus, 1989; Rabin and Stewart, 1992; Stewart and Parales, 1988). This signal transduction process involves the autophosphorylation of the NarQ and NarX sensor-transmitter proteins, and a subsequent transfer of phosphate to the response regulators, NarL and NarP (Cavicchioli *et al.*, 1995; Schröder *et al.*, 1994; Walker and DeMoss, 1993). Activated NarL-phosphate and NarP-phosphate can then recognize and bind to their respective DNA target sites located within the regulatory regions of the genes that each controls by either activating or repressing gene expression (reviewed in Darwin and Stewart, 1996).

Genes in the NarL-regulated family include the nitrate reductase (*narGHJI*), fumarate reductase (*frdABCD*), and nitrite export (*narK*) genes (Stewart and Berg, 1988; Kalman and Gunsalus, 1988; Kolesnikow *et al.*, 1992). Genes regulated by both NarL and NarP include the nitrite reductase (*nrfABCDEFG*), and an anaerobically expressed gene

Received 19 August, 1996; revised 28 March, 1997; accepted 11 April, 1997. Present addresses: †University of Southern California Law School, University Park, Los Angeles, California 90089-0071, USA; ‡Department of Microbiology and Immunology, University of New South Wales, Sydney, New South Wales 2052, Australia. *For correspondence. E-mail robg@microbio.lifesci.ucla.edu; Tel. (310) 2068201; Fax (310) 2065231.

© 1997 Blackwell Science Ltd

1050 R. C. Chiang, R. Cavicchioli and R. P. Gunsalus

| Phenotype | "Locked-off" | Wild-type | "Locked-on" |
| --- | --- | --- | --- |
| Signal (nitrate or nitrite) | ± | – | + | ± |

![Diagram](attachment:diagram.png)

Dominant Activity  
Phosphatase Phosphatase Kinase Kinase

Fig. 1. Proposed model for nitrate and nitrite response by the NarX protein by activation of its kinase activity. The absence or presence of a nitrate or nitrite signal (indicated by ‘○’) dictates the conformation of the wild-type NarX protein and its resulting biochemical activities. In the absence of nitrate, the wild-type NarX protein assumes a conformation that does not allow the transfer of a phosphate to NarL or NarP to activate them. However, in the presence of nitrate or nitrite signal, the wild-type NarX protein assumes a conformation that results in autophosphorylation and the transfer of a phosphate to NarL or NarP to generate NarL-phosphate or NarP-phosphate, respectively. The phosphorylated response regulators can then bind at their respective DNA-regulatory sites to effect nitrate- and nitrite-dependent gene expression. The NarX* (star) protein has a conformation that resembles the conformation of the wild-type NarX protein even in the absence of nitrate and/or nitrite: this results in the ‘locked-on’ phenotype (i.e. constitutive activation or repression). The NarX ‘locked-off’ mutant protein has an alternative conformation that resembles the inactive conformation of the wild-type NarX even in the presence of nitrate or nitrite: this results in the ‘locked-off’ phenotype (i.e. lack of signal-dependent regulation). The N- and C-termini of the NarX proteins are indicated by N and C, respectively.

called aeg-46.5 that encodes a putative periplasmic nitrate reductase (Choe and Reznikoff, 1991; Rabin and Stewart, 1993).

The NarX and NarQ proteins are 598 and 566 amino acids in size, respectively. Each contains two transmembrane-spanning regions located near the N-terminal end (Chiang et al., 1992; Rabin and Stewart, 1992). Between the two transmembrane regions of each protein is an 115 amino acid domain which is proposed to be exposed to the periplasmic space. Comparison of the NarX and NarQ domains revealed a conserved 17 amino acid sequence, designated the ‘P-box’ element, that was proposed to be involved in nitrate sensing (Chiang et al., 1992). Single amino acid substitutions in the P-box element of NarX confer ‘locked-on’ (‘star’) and ‘locked-off’ phenotypes (Fig. 1) that are nitrate insensitive for NarL-regulated genes (Cavicchioli et al., 1996). In addition, several ‘locked-on’ mutations that map to the NarX linker region were previously identified (Kalman and Gunsalus, 1990) that are intramolecularly dominant over the narX P-box ‘locked-off’ mutation (Cavicchioli et al., 1996). Therefore, signal transduction is proposed to proceed from the initial event of nitrate detection in the periplasm region of

NarX followed by transfer across the inner membrane via the NarX linker region to subsequently activate the cytoplasmic kinase domain which in turn mediates protein phosphorylation of NarL (Cavicchioli et al., 1996; Schröder et al., 1994). Consequently, we proposed that similar mutations in narQ would cause similar defects in NarQ signal reception and/or signal transduction (Cavicchioli et al., 1996). While NarX and NarQ are proposed to function by similar mechanisms, little is known about the individual properties of the two sensor-transmitter proteins. They appear to have different roles in regulating the various target genes (Darwin and Stewart, 1996). Biochemical studies with soluble cytoplasmic domains of ‘NarX and ‘NarQ show that ‘NarQ-phosphate has a higher phosphotransferase activity to NarL, while ‘NarX possesses a higher phosphatase activity (Schröder et al., 1994).

In this study, we report the construction and analysis of 24 narQ mutations that encode single amino acid substitutions in the P-box region of NarQ. Approximately one third of the mutations resulted in altered nitrate control of NarL-regulated genes as monitored using narG–lacZ and frdA–lacZ reporter fusions. A subset of the P-box mutations in narQ as well as in narX also conferred altered phenotypes

© 1997 Blackwell Science Ltd, *Molecular Microbiology*, **24**, 1049–1060

Table 1. Effect of amino acid substitutions in NarQ on NarL-dependent gene expression in response to nitrate availability.

| Plasmid$^a$ | Relevant genotype | $narG-lacZ^b$ | $frdA-lacZ^b$ | Phenotype |
|--------------|-------------------|---------------|---------------|-----------|
| pACYC184     | None              | $-NO_3^-$     | $+NO_3^-$     |           |
|              |                   | 137           | 170           | 2360      | 2320      |
| pRCXD1       | $narL^+$          |               |               |           |
|              |                   | 2280          | 1430          | 2270      | 2020      |
| pRCQ3        | $narQ^+$          |               |               |           |
|              |                   | 246           | 169           | 2170      | 1910      |
| pRCQ3/pRCXD1 | $narQ^+ narL^+$   |               |               |           |
|              |                   | 1850          | 7080          | 2135      | 45        | Wild type |
| pRCQ61/pRCXD1| $narQ (R38 \rightarrow Q) narL^+$ |               |           |           |
|              |                   | 1860          | 6960          | 2070      | 43        | Wild type |
| pRCQ72/pRCXD1| $narQ (A40 \rightarrow N) narL^+$ |               |           |           |
|              |                   | 393           | 4550          | 2320      | 1160$^c$  | Impaired$^d$ |
| pRCQ71/pRCXD1| $narQ (A40 \rightarrow I) narL^+$ |               |           |           |
|              |                   | 1850          | 1140          | 2070      | 1980      | Locked-off |
| pRCQ73/pRCXD1| $narQ (A40 \rightarrow S) narL^+$ |               |           |           |
|              |                   | 1890          | 7740          | 2110      | 48        | Wild type |
| pRCQ59/pRCXD1| $narQ (A40 \rightarrow T) narL^+$ |               |           |           |
|              |                   | 867           | 6640          | 2250      | 46        | Wild type |
| pRCQ65/pRCXD1| $narQ (E41 \rightarrow P) narL^+$ |               |           |           |
|              |                   | 19100         | 7760          | 35        | 45        | Locked-on |
| pRCQ69/pRCXD1| $narQ (E41 \rightarrow H) narL^+$ |               |           |           |
|              |                   | 2000          | 6420          | 2130      | 46        | Wild type |
| pRCQ52/pRCXD1| $narQ (E41 \rightarrow R) narL^+$ |               |           |           |
|              |                   | 1900          | 6550          | 2140      | 51        | Wild type |
| pRCQ68/pRCXD1| $narQ (E41 \rightarrow L) narL^+$ |               |           |           |
|              |                   | 2080          | 6870          | 2040      | 41        | Wild type |
| pRCQ54/pRCXD1| $narQ (A42 \rightarrow T) narL^+$ |               |           |           |
|              |                   | 19400         | 6700          | 45        | 43        | Locked-on |
| pRCQ64/pRCXD1| $narQ (A42 \rightarrow S) narL^+$ |               |           |           |
|              |                   | 10300         | 7210          | 720       | 41        | Locked-on |
| pRCQ63/pRCXD1| $narQ (A42 \rightarrow I) narL^+$ |               |           |           |
|              |                   | 1620          | 6710          | 2010      | 43        | Wild type |
| pRCQ58/pRCXD1| $narQ (A46 \rightarrow T) narL^+$ |               |           |           |
|              |                   | 302           | 7550          | 2160      | 53        | Impaired |
| pRCQ70/pRCXD1| $narQ (A46 \rightarrow S) narL^+$ |               |           |           |
|              |                   | 2330          | 6740          | 2110      | 45        | Wild type |
| pRCQ66/pRCXD1| $narQ (A46 \rightarrow I) narL^+$ |               |           |           |
|              |                   | 318           | 280           | 2220      | 2270      | Locked-off |
| pRCQ67/pRCXD1| $narQ (A46 \rightarrow N) narL^+$ |               |           |           |
|              |                   | 307           | 266           | 2180      | 2290      | Locked-off |
| pRCQ75/pRCXD1| $narQ (S48 \rightarrow E) narL^+$ |               |           |           |
|              |                   | 2090          | 7570          | 2270      | 57        | Wild type |
| pRCQ76/pRCXD1| $narQ (S48 \rightarrow G) narL^+$ |               |           |           |
|              |                   | 1550          | 6850          | 2340      | 56        | Wild type |
| pRCQ62/pRCXD1| $narQ (S48 \rightarrow V) narL^+$ |               |           |           |
|              |                   | 456           | 6240          | 2050      | 42        | Impaired |
| pRCQ57/pRCXD1| $narQ (R50 \rightarrow Q) narL^+$ |               |           |           |
|              |                   | 206           | 247           | 2210      | 2380      | Locked-off |
| pRCQ60/pRCXD1| $narQ (S53 \rightarrow I) narL^+$ |               |           |           |
|              |                   | 1030          | 6540          | 2230      | 40        | Wild type |
| pRCQ74/pRCXD1| $narQ (S53 \rightarrow L) narL^+$ |               |           |           |
|              |                   | 953           | 7010          | 2270      | 52        | Wild type |
| pRCQ77/pRCXD1| $narQ (S53 \rightarrow V) narL^+$ |               |           |           |
|              |                   | 415           | 7340          | 2260      | 51        | Impaired |
| pRCQ56/pRCXD1| $narQ (R55 \rightarrow Q) narL^+$ |               |           |           |
|              |                   | 1900          | 6910          | 2130      | 46        | Wild type |

---

**a.** Strain RC11 containing the indicated plasmids was grown anaerobically in buffered LB with 30 $\mu$g ml$^{-1}$ chloramphenicol and 100 $\mu$g ml$^{-1}$ ampicillin when needed. Where indicated, nitrate was added at an initial concentration of 40 mM. The genotype of RC11 is $\Delta(narXL)$ $\Delta narQ$: it contained $\lambda$PC51 ($narG-lacZ$) or $\lambda$LK1 ($frdA-lacZ$) as indicated.

**b.** Units of $\beta$-galactosidase activity are given as nmol ONPG hydrolysed per min per mg protein of $narG-lacZ$ and $frdA-lacZ$ expression.

**c.** This mutation causes the partially ‘locked-off’ phenotype for $frdA-lacZ$ expression.

**d.** The ‘impaired’ phenotype retains nitrate responsiveness for both activation of $narG-lacZ$ and repression of $frdA-lacZ$ expression; however the level of $narG-lacZ$ expression is four- to sixfold lower than $narQ^+$ in the absence of nitrate.

1052 R. C. Chiang, R. Cavicchioli and R. P. Gunsalus

NarQ position 35 40 45 50 55 60
S S L R D A E A I N I A G S L R M Q S Y R L G Y D L Q

Phenotype
"Locked-on"
"Locked-off"
WT (nitrate responsive)
Q
↓
I
↓
P
↓
S
↓
T
↓
I
↓
N
↓
S
↓
H
↓
I
↓
S
↓
T
↓
R
↓
L
↓
E
↓
G
↓
V
↓
I
↓
L
↓
V
↓
Q

Fig. 2. Amino acid sequence of the wild-type NarQ, the ‘locked-on’ NarQ* (star) and the ‘locked-off’ NarQ P-box mutants. The periplasmic region of NarQ from amino acid residue 35 to 61 is represented by the single-letter amino acid codes. The numbering of the sequence is relative to the N-terminal methionine (Chiang et al., 1992). The position of each NarQ amino acid replacement is indicated at the bottom of the figure. The ‘locked-on’ row indicates substitutions that result in the nitrate ‘locked-on’ phenotype (i.e. constitutive activation/repression). The ‘locked-off’ row indicates substitutions that result in the ‘locked-off’ phenotype (i.e. abolished nitrate-dependent activation/repression). This row includes the replacement of Arg-40 with asparagine which confers a partially ‘locked-off’ phenotype. The WT (nitrate responsive) row indicates NarQ substitutions that retain the wild-type phenotype (i.e. normal activation/repression in response to the presence of nitrate). This row includes the substitutions which confer impaired but still nitrate-responsive phenotype.

(Cavicchioli et al., 1996). Ala-42 of NarQ was replaced with a threonine and serine (Fig. 2); each change conferred a similar ‘locked-on’ phenotype for NarL-dependent activation of narG-lacZ, and repression of frdA-lacZ expression which was nitrate independent (Table 1). However, when Ala-42 was substituted by an isoleucine, a wild-type nitrate response was retained. Thus, replacing Ala-42 with an amino acid with a similar hydrophobic side-chain does not disrupt the ability of NarQ to detect nitrate whereas the introduction of a hydroxyl group does. When the narQ mutation that replaced Arg-50 by glutamine was tested, a ‘locked-off’ phenotype was seen for NarL-dependent activation of narG-lacZ and repression of frdA-lacZ expression (Table 1). Thus, these narQ changes behave similarly to the corresponding narX mutants.

Analysis of additional narQ periplasmic domain mutations

A number of additional narQ mutations were made to further define the role of the P-box element. Interestingly, replacing Glu-41 by proline conferred a ‘locked-on’ phenotype. However, a substitution with histidine, arginine, or leucine did not (Table 1; Fig. 2). A substitution at Ala-46 with either an isoleucine or asparagine conferred a ‘locked-off’ phenotype (Table 1). The narQ (A40→I) allele also conferred a ‘locked-off’ phenotype: frdA-lacZ expression was fully derepressed but narG-lacZ expression was only weakly induced. The narQ (A40→N) allele, whereby a hydrophobic residue is replaced by a bulky charged residue, was also impaired for narG-lacZ expression but conferred a partial ‘locked-off’ phenotype for frdA-lacZ expression (Table 1; Fig. 2). The narQ (A46→T), narQ (S48→V) and narQ (S53→V) alleles conferred a nitrate-responsive phenotype that was slightly impaired for narG-lacZ expression but wild type for repression of frdA-lacZ expression.

Substitutions were made at Arg-38 and Arg-55 to test if these positively charged side-chains might be involved in ligand binding (i.e. narQ (R38→Q) and narQ (R55→Q), respectively (Fig. 2). As each change gave a wild-type phenotype, this suggests that the bulky charged side-chains at these positions are not critical to nitrate sensing by NarQ. Of the remaining narQ periplasmic domain mutants tested (i.e. narQ (A40→S), narQ (A40→T), narQ (E41→H), narQ (E41→R), narQ (E41→L), narQ (A46→S), narQ (S48→E), narQ (S48→G), narQ (S53→I), and narQ (S53→L)), none was altered for the activation of narG-lacZ and repression of frdA-lacZ expression in response to nitrate availability (Table 1).

Effect of narQ P-box mutations on nitrite-dependent activation of narG-lacZ and repression of frdA-lacZ by NarL

Several of the narQ P-box mutations were also evaluated for their effect on the regulation of narG-lacZ and frdA-lacZ expression in response to the second environmental

© 1997 Blackwell Science Ltd, Molecular Microbiology, 24, 1049–1060

NarQ and NarX periplasmic domain mutations 1053

Table 2. Effect of NarQ P-box mutations on NarL-dependent gene expression in response to nitrate or nitrite availability.

| Plasmid<sup>a</sup> | Relevant genotype | narG-lacZ<sup>b</sup> | frdA-lacZ<sup>b</sup> |
|--------------------|-------------------|---------------------|---------------------|
|                    |                   | no addition         | +NO₃⁻               | +NO₂⁻               | +NO₃⁻+NO₂⁻           | no addition         | +NO₃⁻               | +NO₂⁻               | +NO₃⁻+NO₂⁻           |
| pACYC 184          | None              | 137                 | 170                 | 520                 | 417                 | 2360                | 2320                | 1910                | 2050                |
| pRCQ 3/pRCXD 1     | narQ⁺narL⁺       | 1850                | 7080                | 8740                | 6710                | 2140                | 45                  | 55                  | 57                  |
| pRCQ 54/pRCXD 1    | narQ(A 42→T) narL⁺ | 19400               | 6700                | 9430                | 6980                | 45                  | 43                  | 51                  | 45                  |
| pRCQ 67/pRCXD 1    | narQ(A 46→N) narL⁺ | 206                 | 247                 | 540                 | 497                 | 2210                | 2380                | 2560                | 2520                |
| pRCQ 57/pRCXD 1    | narQ(R 50→Q) narL⁺ | 307                 | 266                 | 530                 | 469                 | 2180                | 2290                | 2580                | 2530                |

a. Cultures were grown anaerobically in buffered LB with 30 μg ml⁻¹ chloramphenicol and 100 μg ml⁻¹ of ampicillin. Where indicated, nitrate and nitrite were added at initial concentrations of 10 mM and 5 mM, respectively.
b. Units of β-galactosidase activity are given as nmol ONPG hydrolysed per min per mg protein of narG-lacZ and frdA-lacZ expression.

signal, nitrite. In the narQ⁺ control strain, narG-lacZ expression was induced by fivefold while frdA-lacZ expression was repressed 39-fold when nitrite was added to the culture medium (Table 2). Thus, this plasmid-assay system also allowed us to examine nitrite-dependent signal transduction. Interestingly, the narQ (A42→T) mutation resulted in a constitutive activation of narG-lacZ and constitutive repression of frdA-lacZ expression regardless of nitrite availability (i.e., the 'locked-on' phenotype). In contrast, the narQ (A46→N) and narQ (R50→Q) alleles abolished nitrite-dependent regulation of narG-lacZ and repression of frdA-lacZ expression (i.e., the 'locked-off' phenotype; Table 2).

Effect of narQ P-box mutations on NarP-dependent nitrate and nitrite activation of nrfA-lacZ and aeg-46.5-lacZ expression

To determine if NarQ interacts with the NarP response regulator in a manner similar to that with NarL, we tested how the narQ P-box mutations affected nitrate- and nitrite-dependent activation of two NarP-regulated reporter fusions, nrfA-lacZ and aeg-46.5-lacZ (Table 4). Plasmids containing narQ P-box alleles were introduced into strains containing either the nrfA-lacZ or aeg-46.5-lacZ fusion (i.e., strains RCC113-93 and RCC113-46, respectively). When the narQ⁺ gene was tested along with a single copy of narP⁺ located on the chromosome, nrfA-lacZ expression was induced fourfold by the addition of either nitrate or nitrite. Under the same conditions, expression of the aeg-46.5-lacZ fusion was induced six-to 10-fold under all growth conditions relative to the strain that lacked narQ⁺ (Table 4). Thus, NarQ and NarP were sufficient to induce expression of the aeg-46.5-lacZ fusion regardless of whether nitrate or nitrite was present. The narQ (A42→T) allele caused a constitutive high level of

allele exhibited constitutive activation of narG-lacZ and repression of frdA-lacZ expression, regardless of whether nitrite was present (i.e., the 'locked-on' phenotype). The narX902 (R54→E) allele abolished nitrite-dependent regulation of narG-lacZ and frdA-lacZ expression and gave the 'locked-off' phenotype (Table 3).

Effect of narX P-box mutations on NarL-dependent nitrite activation of narG-lacZ and repression of frdA-lacZ expression

As the three narQ P-box mutations described above affected nitrite-dependent gene expression, we tested whether the analogous narX P-box mutations also had a similar effect on nitrite-dependent expression of the narG-lacZ and frdA-lacZ reporter fusions (Table 3). In the narX⁺ control, narG-lacZ expression was induced sixfold and frdA-lacZ expression repressed 1.4-fold with the addition of nitrite (Table 3). Thus, NarX is unable to confer the same level of nitrite-dependent control of the frdA-lacZ reporter as does NarQ, although the biochemical basis of this is not yet known. The narX18 (A46→T)

Table 3. Effect of NarX P-box mutations on NarL-dependent gene expression in response to nitrate or nitrite availability.

| Plasmid<sup>a</sup> | Relevant genotype | narG-lacZ<sup>b</sup> | frdA-lacZ<sup>b</sup> |
|--------------------|-------------------|---------------------|---------------------|
|                    |                   | no addition         | +NO₃⁻               | +NO₂⁻               | +NO₃⁻+NO₂⁻           | no addition         | +NO₃⁻               | +NO₂⁻               | +NO₃⁻+NO₂⁻           |
| pLK63             | narX⁺narL⁺        | 408                 | 9100                | 2580                | 9360                | 2340                | 7                   | 1690                | 15                   |
| pAXE              | narX18 (A46→T) narL⁺ | 8620                | 9750                | 8280                | 10200               | 878                 | 39                  | 492                 | 81                   |
| pRCX33            | narX902 (R54→E) narL⁺ | 192                 | 188                 | 486                 | 496                 | 2060                | 2050                | 2640                | 2730                |

a. Cultures were grown anaerobically in buffered LB with 30 μg ml⁻¹ chloramphenicol. Where indicated, nitrate and nitrite were added at initial concentrations of 10 mM and 5 mM, respectively.
b. Units in β-galactosidase activity are given as nmol ONPG hydrolysed per min per mg protein of narG-lacZ and frdA-lacZ expression.

© 1997 Blackwell Science Ltd, Molecular Microbiology, 24, 1049–1060

1054 R.C. Chiang, R. Cavicchioli and R.P. Gunsalus

Table 4. Effect of NarQ P-box mutations on NarP-dependent gene expression in response to nitrate or nitrite availability.

| Plasmid<sup>a</sup> | Relevant genotype<sup>b</sup> | $nrfA-lacZ^c$ | $aeg-46.5-lacZ^c$ |
| --- | --- | --- | --- |
|  |  | no addition | +NO₃⁻ | +NO₂⁻ | +NO₃⁻+NO₂⁻ | no addition | +NO₃⁻ | +NO₂⁻ | +NO₃⁻+NO₂⁻ |
| pACYC 184 | None, narP⁺ | 84 | 124 | 196 | 202 | 150 | 147 | 148 | 164 |
| pRCQ 3 | narQ⁺narP⁺ | 118 | 479 | 452 | 463 | 1350 | 1420 | 1010 | 1040 |
| pRCQ 54 | narQ(A 42→T) narP⁺ | 398 | 438 | 410 | 480 | 2920 | 2540 | 2630 | 2610 |
| pRCQ 71 | narQ(A 40→I) narP⁺ | 74 | 127 | 194 | 174 | 147 | 149 | 172 | 185 |
| pRCQ 57 | narQ(R 50→Q) narP⁺ | 98 | 107 | 203 | 200 | 138 | 133 | 170 | 171 |

a. Cultures were grown anaerobically in buffered LB with 30 μg ml⁻¹ chloramphenicol. Where indicated, nitrate and nitrite were added at initial concentrations of 10 mM and 5 mM, respectively.
b. The narP⁺ gene is located on the chromosome.
c. Units in β-galactosidase activity are given as nmol ONPG hydrolysed per min per mg protein of nrfA-lacZ and aeg-46.5-lacZ expression.

nrfA-lacZ and aeg-46.5-lacZ expression regardless of nitrate or nitrite availability (i.e. the 'locked-on' phenotype). The narQ (R 50→Q) allele abolished both NarQ-dependent activation of nrfA-lacZ and aeg-46.5-lacZ expression (i.e. the 'locked-off' phenotype; Table 4). These results also demonstrate that the narQ (R 50→Q) allele blocks the ability of the resulting NarQ protein to induce nrfA-lacZ expression even when either nitrate or nitrite is present. Thus, the NarQ P-box mutant protein can interact with the NarP response regulator in a manner that is similar to the interaction with NarL.

relieved of the nitrate- and nitrite-dependent activation of nrfA-lacZ and aeg-46.5-lacZ expression (i.e. a 'locked-off' phenotype; Table 5). These results demonstrate that the NarX protein can interact with NarP in a similar manner to NarL. The narX 902 product is synthesized and is stable as evidenced by its high phosphatase activity (Cavicchioli et al., 1996).

Dominance/recessiveness analysis of the NarQ and NarXP-box mutants

The narQ P-box mutants were tested for trans dominance or trans recessiveness to the analogous narX P-box mutations using the frdA-lacZ reporter fusion (Table 6). For these studies, the narQ P-box allele was present on plasmid pBR 322 (ampicillin resistant; Ap<sup>R</sup>) while the narX P-box allele was on plasmid pACYC 184 (chloramphenicol resistant; Cm<sup>R</sup>). The ability of these strains to express frdA-lacZ was screened on MacConkey-lactose-fumarate (MLF) indicator plates with or without nitrate present, and with or without nitrite present. The strain containing narX⁺ and narL⁺ only gave red colonies on the MLF-nitrite plates (i.e. indicated by 'OFF' in Table 6), while the strain containing narQ⁺ and narL⁺ gave white colonies on the same plate (i.e. the wild-type phenotype; indicated by 'WT' in Table 6). However, the strain containing narX⁺, narQ⁺ and narL⁺ gave red colonies on the MLF-nitrite plate (i.e. also indicated by 'OFF'). In this screening

Effect of narX P-box mutations on NarP-dependent nitrate and nitrite activation of nrfA-lacZ and aeg-46.5-lacZ expression

As several of the narQ P-box mutations were shown to alter NarP-dependent gene expression, the analogous narX P-box mutations were also similarly tested (Table 5). When narX⁺ was present, along with a single copy of the narP⁺ gene located on the chromosome, nrfA-lacZ expression was induced approx. threefold by the addition of either nitrate or nitrite. However, aeg-46.5-lacZ expression was induced 21-to 27-fold in all growth conditions relative to the isogenic strain lacking narX⁺ (Table 4). The narX 18 (A 46→T) strain showed uniformly high nrfA-lacZ and aeg-46.5-lacZ expression regardless of nitrate or nitrite availability. The narX 902 (R 54→E) strain was

Table 5. Effect of NarX P-box mutations on NarP-dependent gene expression in response to nitrate or nitrite availability.

| Plasmid<sup>a</sup> | Relevant genotype<sup>b</sup> | $nrfA-lacZ^c$ | $aeg-46.5-lacZ^c$ |
| --- | --- | --- | --- |
|  |  | no addition | +NO₃⁻ | +NO₂⁻ | +NO₃⁻+NO₂⁻ | no addition | +NO₃⁻ | +NO₂⁻ | +NO₃⁻+NO₂⁻ |
| pLK 633 | narX⁺narP⁺ | 122 | 299 | 384 | 362 | 3100 | 3250 | 3930 | 4120 |
| pRCC 39 | narX 18 (A 46→T) narP⁺ | 694 | 709 | 704 | 775 | 2530 | 2850 | 3200 | 3760 |
| pRCC 38 | narX 902 (R 54→E) narP⁺ | 141 | 200 | 203 | 202 | 300 | 315 | 328 | 352 |

a. Cultures were grown anaerobically in buffered LB with 30 μg ml⁻¹ chloramphenicol. Where indicated, nitrate and nitrite were added at initial concentrations of 10 mM and 5 mM, respectively.
b. The narP⁺ gene is located on the chromosome.
c. Units in β-galactosidase activity are given as nmol ONPG hydrolysed per min per mg protein of nrfA-lacZ and aeg-46.5-lacZ expression.

Table 6. Phenotypes of nitrate- and nitrite-dependent repression of *frdA-lacZ* expression when different NarX and NarQ P-box alleles are combined.$^a$

| Relevant *narX* and *narL* alleles on pACYC184 | Relevant *narQ* allele on pBR322 |
| --- | --- |
|  | none | *narQ*<sup>+</sup> wild type | *narQ* (R50→Q) | *narQ* (A42→T) |
| None | OFF/OFF<sup>b</sup> | OFF/OFF | OFF/OFF | OFF/OFF |
| *narL*<sup>+</sup> | OFF/OFF | WT/WT | OFF/OFF | ON/ON |
| *narX*<sup>+</sup>*narL*<sup>+</sup> | WT/OFF | WT/OFF | WT/OFF | WT/WT |
| *narX902* (R54→E) *narL*<sup>+</sup> | OFF/OFF | WT/WT | OFF/OFF | WT/WT |
| *narX18* (A46→T) *narL*<sup>+</sup> | ON/ON | ON/ON | ON/ON | ON/ON |

a. RC11λLK1 cells harbouring the indicated two plasmids were grown on MacConkey–lactose–fumarate indicator media, with 30 μg ml<sup>−1</sup> chloramphenicol and 100 μg ml<sup>−1</sup> ampicillin, in either the absence or presence of 40 mM nitrate or 5 mM nitrite.

b. Results to the left of the diagonal line indicate the phenotype in response to the presence of nitrate, while results to the right indicate the phenotype in response to the presence of nitrite. ‘OFF’ indicates a ‘locked-off’ phenotype (i.e. constitutive red colony colour on media in the presence or absence of nitrate/nitrite); ‘WT’ indicates a nitrate/nitrite responsive ‘wild-type’ phenotype (i.e. red colony colour on media in the absence of nitrate/nitrite and white colony colour on media in the presence of nitrate/nitrite); and ‘ON’ indicates a ‘locked-on’ phenotype (i.e. constitutive white colony colour on media in the presence or absence of nitrate/nitrite).

system, *narX*<sup>+</sup> is dominant over *narQ*<sup>+</sup>. This observation is consistent with the lack of nitrite-dependent repression of *frdA-lacZ* expression observed in a wild-type strain or when NarX is the sole sensor-transmitter, while repression is observed when NarQ is the sole sensor-transmitter (Rabin and Stewart, 1993).

Interestingly, the *narQ*<sup>+</sup> gene is dominant over the *narX902* (R54→E) allele when tested in the presence of either nitrate or nitrite (indicated by ‘WT/WT’ in Table 6). However, *narQ*<sup>+</sup> is recessive to the *narX18* (A46→T) allele in the absence of nitrate and nitrite (i.e. both conditions gave a ‘locked-on’ phenotype; indicated by ‘ON/ON’ in Table 6). The *narQ* (R50→Q) allele, which confers the ‘locked-off’ phenotype, is trans recessive to both the *narX*<sup>+</sup> and *narX18* (A46→T) alleles. The *narQ* (A42→T) allele, which confers the ‘locked-on’ phenotype, is dominant over the *narX*<sup>+</sup> gene in the presence of nitrite. It is also dominant over the *narX902* (R54→E) allele in the presence of either nitrate or nitrite. However, in the absence of nitrate or nitrite, the *narQ* (A42→T) allele is recessive to both the wild-type *narX* and *narX902* (R54→E) alleles (thus giving rise to the ‘WT/WT’ result in both cases). These results suggest that NarQ (R50→Q) has a loss of function (as it is recessive to wild-type NarX and NarX (A46→T)), while NarQ (A42→T) is a gain of function (as it is dominant over NarX in the presence of nitrite). In addition, NarX phosphatase activity appears to be dominant over the kinase/phosphotransferase activity of NarQ.

### Discussion

We recently demonstrated that the P-box element of the NarX sensor-transmitter protein of *E. coli* is required for nitrate detection and subsequent activation of the NarL response-regulator protein, which in turn activates nitrate reductase (*narGHJI*) gene expression and represses fumarate reductase (*frdABCD*) gene expression (Cavicchioli et al., 1996). It was proposed that the homologous sensor-transmitter protein, NarQ, detects signal by a similar mechanism. It was not known whether the P-box elements of either NarQ or NarX are also required for sensing the second effector molecule, nitrite. Finally, it was not determined whether the NarQ and NarX P-box mutations similarly affected NarP-dependent gene regulation.

The first question was tested by introducing amino acid substitutions into the P-box element of *narQ* and determining the *in vivo* effect the resultant mutant NarQ proteins had on gene regulation. As previously shown for NarX (Cavicchioli et al., 1996), two distinct classes of *narQ* P-box mutations were identified that altered control of NarL-dependent nitrate regulation of *narG-lacZ* and *frdA-lacZ* gene expression (Table 1; Fig. 3). One class is represented by the *narQ* (A42→T) and *narQ* (A42→S) alleles (analogues to mutations in the conserved Ala-46 residue of NarX) which confer a ‘locked-on’ phenotype for nitrate regulation (Cavicchioli et al., 1996). The second class is represented by the *narQ* (R50→Q) allele (analogous to the Arg-54 mutant of NarX), which confers a ‘locked-off’ phenotype for nitrate regulation of *narG-lacZ* and *frdA-lacZ* expression (Cavicchioli et al., 1996). NarQ thus acts like NarX, as initially proposed.

The second question concerning whether NarQ or NarX P-box elements are required for nitrite sensing was resolved. The *narQ* (A42→T) and *narQ* (R50→Q) P-box alleles were found to confer the ‘locked-on’ and ‘locked-off’ phenotypes, respectively, on nitrite-dependent regulation of *narG-lacZ* and *frdA-lacZ* expression (Table 2). The same phenotypes were also observed with the analogous NarX P-box mutants (Table 3). This indicates that the P-box elements of both NarQ and NarX are required for detection of the second environmental signal, nitrite, because the altered phenotype conferred by each P-box

1056 R. C. Chiang, R. Cavicchioli and R. P. Gunsalus

NarQ


TM1 P TM2
H C N G1


Fig. 3. Summary of the single amino acid substitution mutants constructed for the NarX and NarQ to date. These include the mutations in the periplasmic domain of NarX (Cavicchioli et al., 1996) and NarQ (this study), within the linker region of NarX (Collins et al., 1992; Kalman and Gunsalus, 1990), of the conserved His-399 of NarX (Cavicchioli et al., 1995; Rabin and Stewart, 1992), and of the conserved histidines and asparagine of NarX and NarQ (Cavicchioli et al., 1995). The solid boxes represent the locations of the hydrophobic transmembrane regions (labelled TM1 and TM2) that are each 20–22 amino acids in length. The shaded boxes represent the conserved regions of each protein that are common to other bacterial two-component proteins. They are designated the H, N, and G1 blocks. The hatched boxes denote the locations of the conserved P-box and C-box elements (labelled P and C, respectively) unique to the NarX and NarQ proteins (Chiang et al., 1992). The positions of the amino acid residues in each protein are indicated and numbered. The open boxed regions indicate amino acid identity between NarX and NarQ. Each amino acid replacement that confers a ‘locked-on’ (or a partially ‘locked-on’) phenotype is enclosed by a diamond, while each amino acid replacement that confers a ‘locked-off’ (or a partially ‘locked-off’) phenotype is enclosed by a circle. Amino acids that confer either a wild-type or an impaired (but still nitrate-responsive) phenotype are not enclosed in any symbol.

mutant is the same for nitrate and nitrite. Therefore we propose that the mechanism for nitrite detection and signal transfer is similar in the two proteins, and perhaps also to nitrate sensing (Fig. 1). It is not yet clear whether the P-box element is directly involved in binding either ligand, or whether it participates in a subsequent signal transduction step once nitrate or nitrite is detected. As the individual P-box mutations affect both nitrate and nitrite signalling, this supports the latter model. However, it is also possible that the P-box element could be part of the signal detection site for both nitrate and nitrite, given their related chemical structures. Results of preliminary studies suggest that certain amino acid residues in the P-box element of NarQ and NarX are important for distinguishing between nitrate and nitrite, as substitution of these residues confers the ‘locked-off’ phenotype in response to nitrite but not nitrate (R. C. Chiang, unpublished). This is consistent with in vitro studies that demonstrate

that the soluble cytoplasmic forms 'NarQ and 'NarX, both of which lack their periplasmic domains, are insensitive to the presence of either nitrate or nitrite (Schröder et al., 1994).

We also addressed the question of whether the NarQ and NarX P-box regions have any role in NarP-dependent gene regulation. The results of the NarQ P-box mutations on NarP-dependent regulation of *nrfA-lacZ* and *aeg-46.5-lacZ* expression (Table 4) demonstrate that the NarQ P-box mutations affect NarP-dependent gene regulation in a manner similar to that for NarL. A similar process was also observed with the NarX P-box mutations (Table 5). This suggests that the periplasmic domain does not dictate response-regulator recognition. Consequently, we propose that the cytoplasmic domain, not the periplasmic domain, specifies how the sensor-transmitter proteins discriminate between NarL and NarP. The model proposed here for NarQ and NarX signal transduction is consistent with

© 1997 Blackwell Science Ltd, Molecular Microbiology, 24, 1049–1060

those for the sensor-transmitters of the chemotaxis and osmoregulation systems. For these regulatory systems, the periplasmic/transmembrane portions of the aspartate-binding Tar and sugar-binding Trg chemotaxis proteins were fused to the cytoplasmic portion of the EnvZ osmo-sensor involved in porin synthesis. The resultant hybrid fusion proteins regulated osmoregulation in response to aspartate and maltose-bound protein (Baumgartner *et al.*, 1994; Utsumi *et al.*, 1989).

From previous *in vitro* studies, the known major distinction between NarQ and NarX is the differential activities of phosphotransferase and phosphatase of the proteins (Schröder *et al.*, 1994; Cavicchioli *et al.*, 1995). A comparison of the P-box mutant data of NarQ from this study and NarX from a previous study (Cavicchioli *et al.*, 1996) shows that all the analogous NarQ and NarX P-box mutations tested so far always confer the same phenotype (Fig. 3). Substituting Ala-42 of NarQ and Ala-46 of NarX with either threonine or serine, both containing a hydroxyl group capable of hydrogen bonding to other amino acids or to water, confers a ‘locked-on’ phenotype. Also, substituting Arg-50 of NarQ and Arg-54 of NarX with other amino acids (e.g. glutamine and glutamate, respectively) confers a ‘locked-off’ phenotype. We propose the ‘locked-on’ mutants ‘lock’ or ‘jam’ the protein switch in a conformation whereby kinase is the dominant activity, while the ‘locked-off’ mutants ‘lock’ the protein in a conformation whereby phosphatase is the dominant activity (Fig. 1). Consequently, we propose that NarQ and NarX detect nitrate and perhaps nitrite by similar mechanisms. However, a mechanistic understanding of the ligand interactions and their effect on kinase/phosphatase activity must await future biochemical and biophysical studies with purified proteins.

The dominance/recessiveness analysis of NarQ and NarX (Table 6) allowed us, by genetic methods, to further define the hierarchy involved for the different biochemical activities of NarQ and NarX. The NarX* (A46→T) mutation is dominant over wild-type NarX while the ‘locked-off’ NarX (R54→E) mutation is recessive to the wild-type NarX protein (Cavicchioli *et al.*, 1996). The NarX (A46→T) mutation was also dominant over wild-type NarQ and the ‘locked-off’ NarQ (R50→Q) mutant. However, the converse was not true for the NarQ (A42→T) ‘locked on’ mutant. In the absence of an environmental effector, the NarQ* (A42→T) mutant was recessive to wild-type NarX and the ‘locked-off’ NarX (R54→E) mutant. These results demonstrate several features of NarQ and NarX. First, NarX phosphatase activity is dominant over NarQ kinase activity. Second, the NarX (R54→E) protein, which confers the ‘locked-off’ phenotype, having lost its biochemical ability to activate NarL still retains its phosphatase activity to inactivate NarL-phosphate. Third, either one or both of the NarX (R54→E) or NarQ (A42→T) proteins retain their ability

© 1997 Blackwell Science Ltd, *Molecular Microbiology*, **24**, 1049–1060

to sense nitrate, as the overall phosphatase activity is nitrate dependent. Fourth, this *in vivo* study demonstrates that the phosphatase activity of NarX is greater than that of NarQ: this is consistent with *in vitro* results obtained for cytoplasmic ’NarX and ’NarQ (Schröder *et al.*, 1994).

By our model (Fig. 1), when nitrate or nitrite is present in the periplasm of the cell, it interacts at or near the P-box element of NarQ and NarX, and causes a conformational change of the periplasmic domain which in turn causes a conformational change to the cytoplasmic portion of each protein. This switch converts the sensor-transmitter proteins into their ‘active’ forms, which allows either the autophosphorylation at the conserved histidines at position 399 in NarX and at position 370 in NarQ, or the subsequent transfer of this phosphate moiety to NarL or NarP to activate them (Cavicchioli *et al.*, 1995; Schröder *et al.*, 1994; S. K. Hemschemeier, unpublished). When ligand is absent, NarQ and NarX remain in the ‘inactive’ conformations which do not permit significant autophosphorylation and/or phosphotransfer to occur. Ligand detection thus modulates the interconversion of the proteins from the ‘inactive’ to the ‘active’ form. Based on the mutant phenotype of the *narQ* (E41→P) allele, we also propose that replacing His-45 of NarX with a proline will also confer a ‘locked-on’ mutant. Two amino acid residues, at positions 41 and 42 of NarQ and positions 45 (by analogy) and 46 of NarX, are clearly important in the switching mechanism and specific mutations in either position can ‘lock’ the membrane protein into the ‘active’ form. In addition, based on the mutant phenotype of the *narQ* (A40→I), *narQ* (A40→N), *narQ* (A46→I), and *narQ* (A46→N) alleles, we propose that replacing Ala-44 or Ala-48 of NarX with any amino acid with a side-chain bulkier than that of threonine will ‘lock’ the conformation of the protein in the ‘inactive’ form. *In vivo* and *in vitro* studies to test these predictions are in progress.

The periplasmic domains of NarQ and NarX appear to function like certain other bacterial sensor-transmitters which have been shown to require their periplasmic domains for environmental signal reception. The *Agrobacterium tumefaciens* VirA protein has been shown to interact with plant-wall-derived sugars at its periplasmic region (Ankenbauer and Nester, 1990; Chang and Winans, 1992). The periplasmic domain of the *E. coli* EnvZ protein is required for detecting osmotic changes, and amino acid substitutions in this region result in an altered osmotic response (Russo and Silhavy, 1991; Tokishita *et al.*, 1991; 1992). Specific amino acid residues in the periplasmic domains of the *E. coli* Tar and Tsr chemoreceptors are required for chemotaxis in response to aspartate and serine (Milligan and Koshland, 1988; Wolff and Parkinson, 1988; Lee and Imae, 1990). Biochemical studies are in progress to better characterize the nitrate and nitrite ligand-binding sites within NarQ and NarX.

| Strain/Phage/Plasmid | Parent | Genotype or phenotype | Source/Reference |
|----------------------|--------|-----------------------|------------------|
| MC4100              |        | $F^{-} \Delta(argF-lac) U 169$<br>$araD139 rpsL150 deoC1$<br>$relA1 flbB5301 rbsR ptsF 25$ | Silhavy et al. (1984) |
| IS1                 | MC4100 | $\Delta(narXL)$ | Schröder et al. (1993) |
| M2                  | IS1    | $\Delta(narXL) narQ1, \lambda LK1$ | Chiang et al. (1992) |
| RC11                | RC1    | $\Delta(narXL) \Delta narQ::kan recA 56$<br>$srl::Tn10$ | Cavicchioli et al. (1995) |
| RZ4546.5            |        | $aeg-46.5-lacZ::\lambda placMu 53$ | Choe and Reznikoff (1991) |
| RZ4593              |        | $nrfA-lacZ::\lambda placMu 53 (aeg-$<br>$93-lacZ::\lambda placMu 53)$ | Choe and Reznikoff (1991) |
| RCC101              | M2     | $narQ1 purC::Tn10, \lambda LK1$ | This work |
| RCC113              | IS1    | $\Delta(narXL) narQ1 purC::Tn10$ | This work |
| RCC113-46           | RCC113 | $\Delta(narXL) narQ1 purC::Tn10$<br>$aeg-46.5::\lambda placMu 53$ | This work |
| RCC113-93           | RCC113 | $\Delta(narXL) narQ1 purC::Tn10$<br>$nrfA::\lambda placMu 53$ | This work |
| Phage               |        |                       |                  |
| $\lambda LK1$       | $\lambda RS45$ | $\Phi(frdA-lacZ)(Hyb) lacY^{+}$<br>$lacA^{+}$ | Kalman and Gunsalus (1988) |
| $\lambda PC51$      | $\lambda RS45$ | $\Phi(narG-lacZ)(Hyb) lacY^{+}$<br>$lacA^{+}$ | Cotter et al. (1992) |
| Plasmid             |        |                       |                  |
| pACYC184            |        | $Cm^{R} Tc^{R}$ | Chang and Cohen (1978) |
| pBR322              |        | $Ap^{R} Tc^{R}$ | Lab. stock |
| pGem7zf$^{-}$       |        | $F1-ori Ap^{R}$ | Promega |
| pRS528              |        | $lacZ lacY^{+} lacA^{+} Ap^{R}$ | Schröder et al. (1994) |
| pAXE                | pLK634 | $narX18(A46 \rightarrow T) narL^{+} Cm^{R}$ | Cavicchioli et al. (1996) |
| pLK63               | pACYC184 | $narX^{+} narL^{+} Cm^{R}$ | Kalman and Gunsalus (1989) |
| pLK633              | pLK63  | $narX^{+} Cm^{R}$ | Kalman and Gunsalus (1989) |
| pLK634              | pLK63  | $narL^{+} Cm^{R}$ | Kalman and Gunsalus (1989) |
| pRCC38              | pRCX33 | $narX902(R54 \rightarrow E) Cm^{R}$ | This work |
| pRCC39              | pAXE   | $narX18(A46 \rightarrow T) Cm^{R}$ | This work |
| pRCC70              | pBR322 | $narQ^{+} Ap^{R}$ | This work |
| pRCC71              | pBR322 | $narQ(R50 \rightarrow Q) Ap^{R}$ | This work |
| pRCC72              | pBR322 | $narQ(A42 \rightarrow T) Ap^{R}$ | This work |
| pRCX33              | pLK634 | $narX(R54 \rightarrow E) narL^{+} Cm^{R}$ | Cavicchioli et al. (1996) |
| pRCXD1              | pRS528 | $narX narL^{+} \Delta lacZ-lacA Ap^{R}$ | Cavicchioli et al. (1995) |
| pRCQ3               | pACYC184 | $narQ^{+} Cm^{R}$ | Cavicchioli et al. (1995) |
| pRCQ52              | pACYC184 | $narQ(E41 \rightarrow R) Cm^{R}$ | This work |
| pRCQ54              | pACYC184 | $narQ(A42 \rightarrow T) Cm^{R}$ | This work |
| pRCQ56              | pACYC184 | $narQ(R55 \rightarrow Q) Cm^{R}$ | This work |
| pRCQ57              | pACYC184 | $narQ(R50 \rightarrow Q) Cm^{R}$ | This work |
| pRCQ58              | pACYC184 | $narQ(A46 \rightarrow T) Cm^{R}$ | This work |
| pRCQ59              | pACYC184 | $narQ(A40 \rightarrow T) Cm^{R}$ | This work |
| pRCQ60              | pACYC184 | $narQ(S53 \rightarrow I) Cm^{R}$ | This work |
| pRCQ61              | pACYC184 | $narQ(R38 \rightarrow Q) Cm^{R}$ | This work |
| pRCQ62              | pACYC184 | $narQ(S48 \rightarrow V) Cm^{R}$ | This work |
| pRCQ63              | pACYC184 | $narQ(A42 \rightarrow I) Cm^{R}$ | This work |
| pRCQ64              | pACYC184 | $narQ(A42 \rightarrow S) Cm^{R}$ | This work |
| pRCQ65              | pACYC184 | $narQ(E41 \rightarrow P) Cm^{R}$ | This work |
| pRCQ66              | pACYC184 | $narQ(A46 \rightarrow I) Cm^{R}$ | This work |
| pRCQ67              | pACYC184 | $narQ(A46 \rightarrow N) Cm^{R}$ | This work |
| pRCQ68              | pACYC184 | $narQ(E41 \rightarrow L) Cm^{R}$ | This work |
| pRCQ69              | pACYC184 | $narQ(E41 \rightarrow H) Cm^{R}$ | This work |
| pRCQ70              | pACYC184 | $narQ(A46 \rightarrow S) Cm^{R}$ | This work |
| pRCQ71              | pACYC184 | $narQ(A40 \rightarrow I) Cm^{R}$ | This work |
| pRCQ72              | pACYC184 | $narQ(A40 \rightarrow N) Cm^{R}$ | This work |
| pRCQ73              | pACYC184 | $narQ(A40 \rightarrow S) Cm^{R}$ | This work |
| pRCQ74              | pACYC184 | $narQ(S53 \rightarrow L) Cm^{R}$ | This work |
| pRCQ75              | pACYC184 | $narQ(S48 \rightarrow E) Cm^{R}$ | This work |
| pRCQ76              | pACYC184 | $narQ(S48 \rightarrow G) Cm^{R}$ | This work |
| pRCQ77              | pACYC184 | $narQ(S53 \rightarrow V) Cm^{R}$ | This work |
| pRQ1000R            | pGem7zf$^{-}$ | $narQ^{+} Ap^{R}$ | Chiang et al. (1992) |

# Experimental procedures

## Bacterial strains, bacteriophages, and plasmids

The genotypes of the *E. coli* strains, phages and plasmids used in this study are listed in Table 7. Strain RC11 is a MC4100-derived strain containing Δ(*narXL*) Δ*narQ* deletions. It was lysogenized with either λPC51 (*narG-lacZ*) or λLK1 (*frdA-lacZ*) in single copy at the *attλ* locus of the *E. coli* chromosome in order to monitor *in vivo* gene expression. Strain RCC101 was constructed by transducing the *purC::Tn10* selectable marker (Chiang *et al.*, 1992) into strain M2 using the procedure described by Silhavy *et al.* (1984). Strain RCC113 was constructed by transducing *narQ1*, from strain RCC101, into strain IS1, using the adjacent *purC::Tn10* as a selectable marker. Strains RCC113-93 and RCC113-46.5 were constructed by transducing the *nrfA-lacZ::λplacMu53* fusion and the *aeg-46.5-lacZ::λplacMu53* fusion, respectively, into strain RCC113.

Plasmids pRCC38 and pRCC39 were constructed by deleting the 2.0 kb *BglII-BamHI* DNA fragments of plasmids pRCX33 and pAXE, respectively. To construct the pBR322-based plasmids containing the *narQ* alleles, the *ClaI-SalI* 2.7 kb DNA fragments from plasmids pRCQ3, pRCQ57 and pRCQ54 were cloned into pBR322 to generate plasmids pRCC70, pRCC71 and pRCC71, respectively.

## Site-directed mutagenesis

Single-stranded DNA templates, containing uracil, of pRQ1000R (Chiang *et al.*, 1992) were amplified from strain CJ236 based on the method previously described (Kunkel, 1985). Plasmid pRQ1000R contains a 2.6 kb *KpnI-SphI* fragment that encodes the *narQ* gene and flanking sequences in the phagemid vector pGem7zf−. Mutations were identified by DNA sequencing and 2.6 kb *ClaI-SalI* fragments were cloned into pACYC184. Mutations in *narQ* were confirmed by double-stranded DNA-sequencing analysis of each pRCQ construct.

## Cell growth

Cells were grown in Luria–Bertani (LB) broth or on solid media. For β-galactosidase assays, cells were grown in phosphate-buffered (pH 7.0) LB under anaerobic conditions as previously described (Cotter *et al.*, 1990). The phenotypes of the *narX* and *narQ* mutants were determined using MLF indicator plates (as previously described in Kalman and Gunsalus, 1988; 1990; Chiang *et al.*, 1992; Cavicchioli *et al.*, 1996). Where indicated, nitrate and nitrite were each added at initial concentrations of 40 mM and 5 mM, respectively.

## DNA sequence analysis

DNA sequencing was performed by the Sanger dideoxy chain-terminating method (Sanger *et al.*, 1977) using Sequenase, with appropriate synthetic oligonucleotides that annealed with *narQ*, and either single- or double-stranded phagemid templates. Urea (7 M) denaturing polyacrylamide gels (5%) were used to separate the chain-termination reaction products.

---

# Beta-galactosidase assay

The β-galactosidase assays were performed as previously described (Cotter *et al.*, 1990). Beta-galactosidase values represent the average of at least three experiments with a variation of no more than 10% from the mean.

---

# Acknowledgements

We thank William Reznikoff for providing strains and Joan McCabe for technical assistance. This work was supported by Public Health Service grant AI21678 from the National Institutes of Health.

---

# References

Ankenbauer, R.G., and E.W. Nester. (1990) Sugar-mediated induction of *Agrobacterium tumefaciens* virulence genes: structural specificity and activities of monosaccharides. *J Bacteriol* **172**: 6442–6446.

Baumgartner, J.W., Kim, C., Brissette, R.E., Inouye, M., Park, C., and Hazelbauer, G.L. (1994) Transmembrane signalling by a hybrid protein: communication from the domain of chemoreceptor Trg that recognizes sugar-binding proteins to the kinase/phosphatase domain of osmosensor EnvZ. *J Bacteriol* **176**: 1157–1163.

Cavicchioli, R., Schröder, I., Constanti, M., and Gunsalus, R.P. (1995) The *Escherichia coli* NarQ and NarX regulatory proteins contain two conserved histidines that are required for nitrate dependent signal transduction to NarL. *J Bacteriol* **177**: 2416–2424.

Cavicchioli, R., Chiang, R.C., Kalman, L.V., and Gunsalus, R.P. (1996) Role of the periplasmic domain of the *Escherichia coli* NarX sensor-transmitter protein in nitrate-dependent signal transduction and gene regulation. *Mol Microbiol* **21**: 901–911.

Chang, A.C.Y., and Cohen, S.N. (1978) Construction and characterization of amplifiable multicopy DNA cloning vehicles derived from the P15A cryptic miniplasmid. *J Bacteriol* **134**: 1141–1156.

Chang, C.-H., and Winans, S.C. (1992) Functional roles assigned to the periplasmic, linker, and receiver domains of the *Agrobacterium tumefaciens* VirA protein. *J Bacteriol* **174**: 7033–7039.

Chiang, R.C., Cavicchioli, R., and Gunsalus, R.P. (1992) Identification and characterization of *narQ*, a second nitrate sensor for nitrate-dependent gene regulation in *Escherichia coli*. *Mol Microbiol* **6**: 1913–1923.

Choe, M.H., and Reznikoff, W.S. (1991) Anaerobically expressed *Escherichia coli* genes identified by operon fusion techniques. *J Bacteriol* **173**: 6139–6146.

Collins, L.A., Egan, S.E., and Stewart, V. (1992) Mutational analysis reveals functional similarity between NARX, a nitrate sensor in *Escherichia coli* K-12, and the methyl-accepting chemotaxis proteins. *J Bacteriol* **174**: 3667–3675.

Cotter, P.A., Chepuri, V., Gennis, R.B., and Gunsalus, R.P. (1990) Cytochrome o (*cyoABCDE*) and d (*cydAB*) oxidase gene expression in *Escherichia coli* is regulated by oxygen, pH and the *fnr* gene product. *J Bacteriol* **172**: 6333–6338.

Darwin, A.J., and Stewart, V. (1996) The NAR modulon

© 1997 Blackwell Science Ltd, *Molecular Microbiology*, **24**, 1049–1060

1060 R. C. Chiang, R. Cavicchioli and R. P. Gunsalus

systems: nitrate and nitrite regulation of anaerobic gene expression. In *Regulation of Gene Expression in Escherichia coli*. Lin, E.C.C., and Lynch, A.S. (eds). New York: Chapman and Hill, pp. 343–359.

Gunsalus, R.P. (1992) Control of electron flow in *Escherichia coli*: coordinated transcription of respiratory pathway genes. *J Bacteriol* **174**: 7069–7074.

Kalman, L.V., and Gunsalus, R.P. (1988) The *frdR* gene of *Escherichia coli* globally regulates several operons involved in anaerobic growth in response to nitrate. *J Bacteriol* **170**: 623–629.

Kalman, L.V., and Gunsalus, R.P. (1989) Identification of a second gene involved in global regulation of fumarate reductase and other nitrate-controlled genes for anaerobic respiration in *Escherichia coli*. *J Bacteriol* **171**: 3810–3816.

Kalman, L.V., and Gunsalus, R.P. (1990) Nitrate- and molybdenum-independent signal transduction mutations in *narX* that alter regulation of anaerobic respiratory genes in *Escherichia coli*. *J Bacteriol* **172**: 7049–7056.

Kolesnikow, T., Schröder, I., and Gunsalus, R.P. (1992) Regulation of *nark* gene expression in *Escherichia coli* in response to anaerobiosis, nitrate, iron and molybdenum. *J Bacteriol* **174**: 7104–7111.

Kunkel, T.A. (1985) Rapid and efficient site-specific mutagenesis without phenotypic selection. *Proc Natl Acad Sci USA* **82**: 488–492.

Lee, L., and Imae, Y. (1990) Role of threonine residue 154 in ligand recognition of Tar chemoreceptor in *Escherichia coli*. *J Bacteriol* **172**: 377–382.

Milligan, D.L., and Koshland, Jr, D.E. (1988) Purification and characterization of the periplasmic domain of the aspartate chemoreceptor. *J Biol Chem* **268**: 1991–1997.

Rabin, R.S., and Stewart, V. (1992) Either of two functionally redundant sensor proteins, NarX and NarQ, is sufficient for nitrate regulation in *Escherichia coli* K-12. *Proc Natl Acad Sci USA* **89**: 8419–8423.

Rabin, R.S., and Stewart, V. (1993) Dual response regulators (NarL and NarP) interact with dual sensors (NarX and NarQ) to control nitrate- and nitrite-regulated gene expression in *Escherichia coli* K-12. *J Bacteriol* **175**: 3259–3268.

Russo, F.D., and Silhavy, T.J. (1991) EnvZ controls the concentration of phosphorylated OmpR to mediate osmoregulation of the porin genes. *J Mol Biol* **222**: 567–580.

Sanger, F., Nicklen, O., and Coulson, A.R. (1977) DNA sequencing with chain terminating inhibitors. *Proc Natl Acad Sci USA* **74**: 5463–5467.

Schröder, I., Darie, S., and Gunsalus, R.P. (1993) Activation of the *Escherichia coli* nitrate reductase (*narGHJI*) operon by NarL and Fnr requires Integration Host Factor. *J Biol Chem* **268**: 771–774.

Schröder, I., Wolin, C.D., Cavicchioli, R., and Gunsalus, R.P. (1994) Phosphorylation and dephosphorylation of the NarX, NarL, and NarQ proteins of the nitrate dependent two-component regulatory system of *Escherichia coli*. *J Bacteriol* **176**: 4985–4992.

Silhavy, T.J., Berman, M.L., and Enquist, L.W. (1984) *Experiments with Gene Fusions*. Cold Spring Harbor, New York: Cold Spring Harbor Laboratory Press.

Stewart, V. (1993) Nitrate regulation of anaerobic respiratory gene expression in *Escherichia coli*. *Mol Microbiol* **9**: 425–434.

Stewart, V., and Berg, B.L. (1988) Influence of *nar* (nitrate reductase) genes on nitrate inhibition of formate-hydrogen lyase and fumarate reductase gene expression in *Escherichia coli* K-12. *J Bacteriol* **170**: 4437–4444.

Stewart, V., and Parales, Jr, J. (1988) Identification and expression of genes *narL* and *narX* of the *nar* (nitrate reductase) locus in *Escherichia coli* K-12. *J Bacteriol* **170**: 1587–1597.

Tokishita, S.-I., Kojima, A., Aiba, H., and Mizuno, T. (1991) Transmembrane signal transduction and osmoregulation in *Escherichia coli*. *J Biol Chem* **266**: 6780–6785.

Tokishita, S.-I., Kojima, A., and Mizuno, T. (1992) Transmembrane signal transduction and osmoregulation in *Escherichia coli*. II. The osmotic sensor, EnvZ, located in the isolated cytoplasmic membrane displays its phosphorylation and dephosphorylation abilities as to the activator protein, OmpR. *J Biochem* **111**: 707–713.

Utsumi, R., Brissette, R.E., Rampersaud, A., Forst, S.A., Oosawa, K., and Inouye, M. (1989) Activation of bacterial porin gene expression by a chimeric signal transducer in response to aspartate. *Science* **245**: 1246–1249.

Walker, M.S., and DeMoss, J.A. (1993) Phosphorylation and dephosphorylation catalyzed *in vitro* by purified components of the nitrate sensing system, NarX and NarL. *J Biol Chem* **268**: 8391–8393.

Wolff, C., and Parkinson, J.S. (1988) Aspartate taxis mutants of the *Escherichia coli* Tar chemoreceptor. *J Bacteriol* **170**: 4509–4515.
