
the biological effects of the JNK signaling pathway. We investigated the effect of JIP-1 on the promotion of apoptosis by JNK after the withdrawal of nerve growth factor (NGF) from PC12 cells (6). Expression of the JBD of JIP-1 (residues 127 to 281) reduced the NGF withdrawal-induced apoptosis by more than 90% (13). These data demonstrate that JIP-1 suppresses the biological actions of the JNK signal transduction pathway.

We also examined the effects of JIP-1 on oncogenic transformation. The Bcr-Abl leukemia oncogene causes activation of JNK, but not ERK (5). Thus, transformation by Bcr-Abl may be mediated, in part, by the JNK signaling pathway. Expression of Bcr-Abl or the viral oncogene v-Abl caused constitutive activation of JNK (approximately fivefold), which was blocked by co-expression of JBD (Fig. 4A). We prepared bi-cistronic retroviruses expressing Bcr-Abl alone or together with JBD in the sense and antisense orientations (Fig. 4B). Immunoblot analysis of Bcr-Abl and JBD confirmed that the retroviral constructs expressed the appropriate proteins. The recombinant retroviruses were used to infect primary marrow cells, and the transformation of precursor B cells (pre-B cells) was monitored in culture. Bcr-Abl caused transformation of pre-B cells (Fig. 4C). JBD inhibited transformation when it was expressed in the sense but not in the anti-sense orientation. These data implicate the JNK pathway in pre-B cell transformation by Bcr-Abl. Because Bcr-Abl has a role in human disease, both JNK and JIP-1 are candidate targets for the design of therapeutic strategies for the treatment of chronic myeloid leukemia.

The results of this study demonstrate that one role of JIP-1 may be to suppress signal transduction by the JNK pathway. For example, JIP-1 may compete with substrates that bind JNK. Alternatively, JIP-1 may have a more direct role in targeting JNK to specific regions of the cell or to specific substrates. Indeed, overexpression of JIP-1 caused retention of JNK in the cytoplasm. Therefore, in addition to other possible physiological functions, JIP-1 may act as a cytoplasmic anchor for JNK.

### REFERENCES AND NOTES

1. A. J. Whitmarsh and R. J. Davis, *J. Mol. Med.* **74**, 589 (1996).
2. H. K. Sluss, Z. Han, T. Barrett, R. J. Davis, T. Ip, *Genes Dev.* **10**, 2745 (1996); J. R. Riesgo-Escovar, M. Jenni, A. Fritz, E. Hafen, *ibid.*, p. 2759.
3. B. Su *et al.*, *Cell* **77**, 727 (1994); M. Rincón, B. Dérijard, C.-W. Chow, R. J. Davis, R. A. Flavell, *Genes Funct.* **1**, 51 (1997).
4. X. Xu *et al.*, *Oncogene* **13**, 153 (1996).
5. A. B. Raitano, J. R. Halpern, T. M. Hambuch, C. L. Sawyers, *Proc. Natl. Acad. Sci. U.S.A.* **92**, 11746 (1995).
6. Z. Xia, M. Dickens, J. Raingeaud, R. J. Davis, M. E. Greenberg, *Science* **270**, 1326 (1995).
7. B. W. Zanke *et al.*, *Curr. Biol.* **6**, 606 (1996); M. Verheij *et al.*, *Nature* **380**, 75 (1996); Y.-R. Chen, G. F. Meyer, T.-H. Tan, *J. Biol. Chem.* **271**, 631 (1996).
8. M. Cavigelli, F. Dolfi, F.-X. Claret, M. Karin, *EMBO J.* **14**, 5957 (1995).
9. I. M. Verma, J. K. Stevenson, E. M. Schwarz, D. Van Antwerp, S. Miyamoto, *Genes Dev.* **9**, 2723 (1995); R. B. McNeill and R. J. Colbran, *J. Biol. Chem.* **270**, 10043 (1995); M. C. Faux and J. D. Scott, *Trends Biochem. Sci.* **21**, 312 (1996).
10. P. Cohen, *Annu. Rev. Biochem.* **58**, 453 (1989); M. J. Hubbard and P. Cohen, *Trends Biochem. Sci.* **18**, 172 (1993); J. Inglese, N. J. Freedman, W. J. Koch, R. J. Lefkowitz, *J. Biol. Chem.* **268**, 23735 (1993); V. M. Coghlan *et al.*, *Science* **267**, 108 (1995); D. Mochly-Rosen, *ibid.* **268**, 247 (1995); F. Shibasaki, E. R. Price, D. Lian, F. McKeon, *Nature* **382**, 370 (1996).
11. K.-Y. Choi, B. Satterberg, D. M. Lyons, E. A. Elion, *Cell* **78**, 499 (1994); T. M. Klauck *et al.*, *Science* **271**, 1589 (1996); M. C. Faux and J. D. Scott, *Cell* **85**, 9 (1996).
12. A JIP-1 cDNA fragment was isolated by a two-hybrid screen of a mouse embryo cDNA library in the yeast strain L40 [Z. Galcheva-Gargova *et al.*, *Science* **272**, 1797 (1996)]. The bait plasmid (pLexA-JNK1) was constructed by the insertion of JNK1 in the polylinker of pBTM116. Full-length JIP-1 clones were obtained from a mouse brain λZAPII cDNA library (Stratagene), and expression vectors were constructed with the vectors pCMV5, pCDNA3 (Invitrogen), and pGEX-3X (Pharmacia LKB Biotechnology). Mutations were constructed with the polymerase chain reaction. The sequence of the JIP-1 cDNA has been deposited in GenBank with the accession number AF003115.
13. M. Dickens, J. Rogers, J. Cavanagh, Z. Xia, unpublished data.
14. S. Gupta *et al.*, *EMBO J.* **15**, 2760 (1996).
15. B. Dérijard *et al.*, *Cell* **76**, 1025 (1994); H. K. Sluss, T.

Barrett, B. Dérijard, R. J. Davis, *Mol. Cell. Biol.* **14**, 8376 (1994); T. Kallunki *et al.*, *Genes Dev.* **8**, 2996 (1994); T. Dai *et al.*, *Oncogene* **10**, 849 (1995).

16. R. Marais, J. Wynne, R. Treisman, *Cell* **73**, 381 (1993); A. J. Whitmarsh, P. Shore, A. D. Sharrocks, R. J. Davis, *Science* **269**, 403 (1995); H. Gilie, T. Strahl, P. E. Shaw, *Curr. Biol.* **5**, 1191 (1995); J. Raingeaud, A. J. Whitmarsh, T. Barrett, B. Dérijard, R. J. Davis, *Mol. Cell. Biol.* **16**, 1247 (1996).

17. J. Raingeaud *et al.*, *J. Biol. Chem.* **270**, 7420 (1995).

18. Single-letter abbreviations for the amino acid residues are as follows: A, Ala; G, Gly; K, Lys; L, Leu; S, Ser; and T, Thr.

19. Luciferase reporter assays in CHO cells were performed 48 hours after transfection with the use of β-galactosidase (β-Gal) to measure transfection efficiency (14). The cells were activated by treatment with fetal calf serum (10%). The data are presented as the relative luciferase activity [mean ± SEM of the activity ratio of luciferase/β-Gal (*n* = 3)].

20. We performed bone marrow transformation assays using recombinant retroviruses [C. L. Sawyers, J. McLaughlin, O. N. Witte, *J. Exp. Med.* **181**, 307 (1995)] that were packaged with 293T cells. We constructed the bi-cistronic retroviruses expressing the JBD and *Bcr-Abl* by subcloning p185*Bcr-Abl* in the Cla I site and the JBD in the Eco RI site of pSRαTK.

21. We thank J. Cooper and S. Hollenberg for two-hybrid reagents; D. Schmidt and F. Fay for assistance with immunofluorescence microscopy; M. Wysk, R. Tizard, and J. Demarinis for Maxam-Gilbert sequencing; I.-H. Wu for technical assistance; T. Barrett for automated DNA sequencing; and K. Gemme for secretarial assistance. Supported by NIH grants CA43855 (M.E.G.) and CA65861 (R.J.D.) and by a grant from the James S. McDonnell Foundation (C.L.S.). R.J.D. is an investigator of the Howard Hughes Medical Institute. C.L.S. is a scholar of the Leukemia Society of America.

---

### Analysis of a Chemical Plant Defense Mechanism in Grasses

Monika Frey, Paul Chomet, Erich Glawischnig, Cornelia Stettner, Sebastian Grün, Albert Winklmair, Wolfgang Eisenreich, Adelbert Bacher, Robert B. Meeley, Steven P. Briggs, Kevin Simcox, Alfons Gierl*

In the Gramineae, the cyclic hydroxamic acids 2,4-dihydroxy-1,4-benzoxazin-3-one (DIBOA) and 2,4-dihydroxy-7-methoxy-1,4-benzoxazin-3-one (DIMBOA) form part of the defense against insects and microbial pathogens. Five genes, *Bx1* through *Bx5*, are required for DIBOA biosynthesis in maize. The functions of these five genes, clustered on chromosome 4, were demonstrated in vitro. *Bx1* encodes a tryptophan synthase α homolog that catalyzes the formation of indole for the production of secondary metabolites rather than tryptophan, thereby defining the branch point from primary to secondary metabolism. *Bx2* through *Bx5* encode cytochrome P450–dependent monooxygenases that catalyze four consecutive hydroxylations and one ring expansion to form the highly oxidized DIBOA.

A substantial number of secondary metabolites in plants are dedicated to pathogen defense. These include the cyclic hydroxamic acids, which are found almost exclusively in Gramineae. For example, DIMBOA and its precursor DIBOA are present in maize. DIMBOA confers resistance to first-brood European corn borer (*Ostrinia nubilalis*), northern corn leaf blight (*Helminthosporium turcicum*), maize plant louse (*Rhophalosiphum maydis*), and stalk rot (*Diplodia maydis*), as well as to the herbicide atrazine (1). DIBOA is the main hydroxamic acid in rye, whereas DIMBOA is the predominant form in wheat and maize (1). The DIMBOA and tryptophan biosynthetic

pathways share certain intermediates. Labeled tryptophan precursors such as anthranilic acid and indole are incorporated into DIMBOA, although labeled tryptophan is not incorporated (2). The maize mutation *bxl* (benzoxazineless) abolishes DIMBOA synthesis (3). Plants homozygous for *bxl* grow normally but are extremely susceptible to the pathogens mentioned above.

To clone the *Bxl* gene, we used the Mutator (*Mu*) transposon tagging system (4). Approximately 150,000 seeds were produced from a cross of a *Mu* female line with the pollen from plants homozygous for the recessive *bxl* mutant allele (3). Seventeen putative mutants were identified and outcrossed to an inbred *Bxl/Bxl* line. The segregation of the *bxl* alleles in the progeny of the crosses was followed by a cleaved amplified polymorphic sequence (CAPS) marker (5) derived from the linked *Bx4* gene (6). One of the putative mutants showed the expected 1:1 segregation for the *bxl* allele and the newly *Mu*-induced recessive *bxl* allele with respect to the linked marker. From this material, the *Mu* element cosegregating with the new *bxl::Mu* allele was identified and a flanking genomic DNA fragment was isolated by a polymerase chain reaction (PCR)-based method (7). This fragment was used to isolate the wild-type *Bxl* and recessive *bxl* (3) alleles from genomic λ libraries (8) as well as a full-length cDNA clone (Fig. 1). DNA sequence analysis revealed that the *bxl* allele harbors a deletion of 924 base pairs (bp) comprising 355 bp of the 5′ nontranscribed region, the first exon, the following intron, and 53 bp of the second exon. The position of the *Mu* element in the *bxl::Mu* allele isolated in the transposon-tagging experiment was determined by PCR amplification of the flanking genomic sequences.

DNA sequence analysis revealed the exact *Mu* insertion site and the characteristic 9-bp host sequence duplication associated with integration of the transposon (Fig. 1). The exon sequences of *Bxl* were found to be identical to a gene previously described (9) that is homologous to tryptophan synthase α (TSA) and, when expressed in *Escherichia coli*, complemented a tryptophan synthase α mutation. TSA catalyzes the conversion of indole-3-glycerol phosphate to indole, the penultimate reaction in tryptophan biosynthesis. However, indole was also implicated as an intermediate in DIMBOA biosynthesis (2). Consequently, the *bxl* mutant should also be defective in the production of free indole.

The immersion of shoots of *bxl* seedlings (4 days after imbibition) into a 1 mM solution of indole in the dark for 1 day restored the formation of DIMBOA (Fig. 2A). Hence, the biosynthetic block in the *bxl* mutant cannot be downstream of indole formation. Addition of tryptophan did not result in DIMBOA accumulation (Fig. 2B). When [3-¹³C]indole was administered to maize shoots, [2-¹³C]DIMBOA (75% incorporation) was recovered (Table 1). This

![Figure 1](#fig1)

**Fig. 1.** Structure and chromosomal location of the *Bx* genes. (A) Schematic representation of the *Bx* gene cluster on chromosome 4. Genetic distances are indicated (in centimorgans). (B) Exon-intron structure of *Bx1* through *Bx5*. Exons are represented by boxes. Translation start and stop codons and polyadenylate addition sites are shown. Arrows represent insertion of a *Mu* element in the *bx1::Mu* and *bx3::Mu* alleles. The deletion in the *bx1* standard allele is indicated; it comprises nucleotides 1366 to 2289 of the published sequence (9). The distance from *Bx1* to *Bx2* (2490 bp) is not drawn to scale. The complete sequences of the genes have been deposited in the European Molecular Biology Laboratory data bank [accession numbers X76713 (*Bx1*), Y11368 (*Bx2*), Y11404 (*Bx3*), X81828 (*Bx4*), and Y11403 (*Bx5*)]. (C) Insertion sites of *Mu* in *bx1::Mu* and *bx3::Mu*. The characteristic 9-bp host sequence duplication associated with *Mu* insertion is underlined with an arrow. The insertion occurred at position 2826 of the genomic DNA sequences in *bx1::Mu* and at position 1260 in *bx3::Mu*.

![Figure 2](#fig2)

**Fig. 2.** Detection of metabolites of the DIMBOA pathway by HPLC. Metabolites are indicated at the position of chromatographic peaks. S represents the solvent peak. (A and B) Feeding of *bx1* standard mutant seedling shoots with 1 mM indole (A) or 1 mM tryptophan (B). Seedling material (1 g) was extracted (21) and analyzed on a Merck LiChroCART RP-18 HPLC column (4 × 125 mm). Elution was for 5 min under isocratic conditions with solvent A (H₂O/acetic acid, 9:1) followed by a linear gradient from 100% solvent A to 100% solvent B (methanol/H₂O/acetic acid, 70:27:3) over 7 min. (C to F) Analysis of maize P-450 enzymes expressed in yeast microsomes. Reaction mixtures of 0.2 ml contained 50 mM potassium phosphate buffer (pH 7.5), 0.8 mM NADPH, 0.1 to 0.5 mM of the respective substrates, and 1 mg of microsomal protein. Incubation was for 30 min at 25°C. HPLC analysis was as described above. Shown are results for (C) BX2 microsomes incubated with indole, (D) BX3 microsomes incubated with indolin-2-one, (E) BX4 microsomes incubated with 3-hydroxyindolin-2-one, and (F) BX5 microsomes incubated with HBOA.

M. Frey, E. Glawischnig, C. Stettner, S. Grün, A. Winklmair, A. Gierl, Institut für Genetik, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany.
P. Chomet, DeKalb Genetics Corporation, 62 Maritime Drive, Mystic, CT 06355, USA.
W. Eisenreich and A. Bacher, Institut für Organische Chemie und Biochemie, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany.
R. B. Meeley and S. P. Briggs, Pioneer Hi-Bred International, Box 7300, Johnston, IA 50131, USA.
K. Simcox, Department of Horticulture and Crop Science, Ohio State University, Wooster, OH 44691, USA.

*To whom correspondence should be addressed.*

confirms that indole is an intermediate in DIMBOA biosynthesis (2).

In bacteria, TSA activity is almost completely dependent on formation of an active α₂β₂ complex (10) with tryptophan synthase β (TSB), and indole is usually not released during tryptophan synthesis. An analogous heterosubunit complex exists in Arabidopsis (11). If the TSA homolog BX1 catalyzes the formation of free indole from indole-3-glycerol phosphate, BX1 should function independently of TSB. To test this assumption, we expressed BX1 in E. coli, then purified and assayed it for steady-state kinetic constants (12). We determined a Michaelis constant \( K_m \) indole-3-glycerol phosphate of 0.013 mM and a catalytic rate constant \( k_{cat} \) of 2.8 s⁻¹. Comparison of these values with the constants for conversion of indole-3-glycerol phosphate to indole by E. coli α₂β₂ complex (\( K_m \) indole-3-glycerol phosphate = 0.027 mM, \( k_{cat} = 0.2 \) s⁻¹) (13) demonstrates that BX1, independent of TSB, is ~30 times as efficient as the bacterial complex in catalyzing the production of indole. Tryptophan is essential for the maize plant (14). Because the bx1 mutants are viable, Bx1 cannot be the only maize gene encoding TSA activity. We suggest that there are at least two TSA genes in maize: one that is involved in tryptophan biosynthesis, forming the α₂β₂ complex, and a second gene, Bx1, that is required for the production of free indole and secondary DIMBOA synthesis.

Four maize cytochrome P-450-dependent monooxygenase genes, one of which was isolated by subtractive cDNA cloning from high versus low DIMBOA-accumulating lines (6), are in the CYP71C subfamily of plant cytochrome P-450 genes. These genes are strongly expressed in young maize seedlings, share an overall amino acid identity of 45 to 60%, and are clustered on the short arm of chromosome 4 (Fig. 1). The finding that all oxygen atoms of DIMBOA are incorporated from molecular oxygen (15) led to the speculation that these cytochrome P-450 enzymes might be involved in this pathway. The genes encoding these enzymes are here designated Bx2, Bx3, Bx4, and Bx5 (6).

Direct evidence for the involvement of Bx3 in DIMBOA biosynthesis is provided by a mutant allele (Bx3::Mu) isolated by a reverse genetic approach to screen for Mu insertions in the P-450 genes (16). Sequencing of the PCR-amplified Mu-flanking genomic DNA fragments showed that Bx3::Mu has a Mu transposon inserted in the second exon of the gene (Fig. 1). In maize seedlings homozygous for the recessive mutant allele, no DIMBOA could be detected by high-performance liquid chromatography (HPLC) analysis. In contrast, DIMBOA was detected in seedlings that were either heterozygous or homozygous wild-type. Cosegregation of the recessive mutant phenotype was established by genomic blotting analysis of 27 F₂ individuals (four homozygous recessives). These results demonstrate that an intact Bx3 gene is required for DIMBOA biosynthesis.

Genomic clones of Bx2 through Bx5 were isolated from a λ library and sequenced (8), and the exon-intron structure of the genes was determined (Fig. 1). The position of one intron is identical for Bx2 through Bx5, indicating a common evolutionary origin. The position of a second intron is conserved in Bx3 and Bx5. DNA sequence comparison showed that Bx1 and Bx2 are separated by only 2490 bp.

To investigate the function of the four P-450 enzymes in DIMBOA biosynthesis, we used a yeast expression system (17). The cDNAs of Bx2 through Bx5 were inserted into the pYeDP60 expression vector (18). These constructs were used to transform the WAT11 yeast strain. In WAT11, a galactose-inducible Arabidopsis thaliana microbial NADPH (reduced form of nicotinamide adenine dinucleotide phosphate)-P-450 reductase (ATR1) replaces the yeast reductase (19).

Microsomes were isolated from the transgenic yeast strains and tested for enzymatic activity (17, 19). Indole was converted to DIBOA by the stepwise action of the four cytochrome P450 enzymes (Fig. 2, C to F). When [3-¹³C]indole was incubated with yeast microsomes containing BX2 protein, [3-¹³C]indolin-2-one was produced in the reaction assay. A sufficient amount of [3-¹³C]indolin-2-one was produced by this enzyme-catalyzed reaction to test for subsequent enzymatic conversions. Incubation of [3-¹³C]indolin-2-one with microsomes containing BX3 resulted in the production of [3-¹³C]hydroxyindolin-2-one. For further analysis, unlabeled 3-hydroxyindolin-2-one was obtained by reduction of isatin (20). The conversion of 3-hydroxyindolin-2-one to 2-hydroxy-1,4-benzoxazin-3-one (HBOA) was catalyzed by microsomes containing BX4. The reaction of HBOA with microsomes containing BX5 resulted in the production of DIBOA.

Table 1. ¹H-NMR analysis of indole-derived enzymatic products in maize (δ, chemical shift; \( J_{HH} \), coupling constant). The signal assignments are based on two-dimensional NMR analysis (28). We used ¹H-NMR spectroscopy to monitor ¹³C enrichment from [¹³C]indole and subsequent metabolites. The ¹³C label from [3-¹³C]indole was incorporated into position 3 of indolin-2-one (100% incorporation) and position 2 of DIMBOA (75% incorporation). The label from [3-¹³C]indolin-2-one was incorporated into position 3 of 3-hydroxyindolin-2-one (100% incorporation). Some values of \( J_{HH} \) are dual because coupling acts on both protons over scalar bonds; others are missing because the signals were characterized by singlet multiplicity and \( J_{HH} \) could not be extracted.

| Position | δ (ppm) | \( J_{HH} \) (Hz) | Position | δ (ppm) | \( J_{HH} \) (Hz) | Position | δ (ppm) | \( J_{HH} \) (Hz) |
|----------|---------|------------------|----------|---------|------------------|----------|---------|------------------|
| 4        | 7.22    | 7.5              | 4        | 7.42    | 7.5              | 5        | 7.26    | 8.8              |
| 6        | 7.18    | 7.8, 0.9         | 6        | 7.26    | 7.7              | 6        | 6.67    | 8.8, 2.6         |
| 5        | 6.97    | 7.5, 0.8         | 5        | 7.07    | 7.5              | 8        | 6.63    | 2.6              |
| 7        | 6.86    | 7.8              | 7        | 6.84    | 7.7              | 2        | 5.67    |                  |
| 3        | 3.49    |                  | 3        | 5.05    |                  | O-CH₃    | 3.75    |                  |

Fig. 3. Biosynthetic pathways to DIMBOA and tryptophan. Names of gene products are indicated for each of the reactions. BX1 represents a tryptophan synthase α activity. BX2 through BX5 represent cytochrome P-450-dependent monooxygenases of the CYP71C subfamily. The sequence of N-hydroxylation and introduction of the methoxy group at C-7 of DIMBOA has not yet been elucidated.

the DIMBOA pathway. Indole-3-glycerol phosphate represents the branch point from tryptophan biosynthesis, and BX1 enzyme would catalyze the committing step.

The synthesis of several other secondary metabolites in plants, such as the indole glucosinates, anthranilate-derived alkaloids, and tryptamine derivatives (24, 25), depends on the tryptophan pathway. Indole-3-glycerol phosphate was also proposed as a branch point from the tryptophan pathway for the synthesis of the indolic phytoalexin camalexin (3-thiazol-2′-yl-indole) in *Arabidopsis thaliana* (26, 27). It will be interesting to see whether a BX1 homologous enzyme also catalyzes the first specific step in camalexin synthesis. Such an observation would shed light on the role of indole-3-glycerol phosphate as an intermediate in a wide range of secondary metabolites in plants.

**REFERENCES AND NOTES**

1. H. M. Niemeyer, *Phytochemistry* **27**, 3349 (1988).
2. S. R. Desai, P. Kumar, W. S. Chilton, *Chem. Commun.* **1996**, 1321 (1996).
3. R. H. Hamilton, *Weeds* **12**, 27 (1964).
4. P. Chomet, in *The Maize Handbook*, M. Freeling and V. Walbot, Eds. (Springer-Verlag, New York, 1994), pp. 243–248.
5. A. Konieczny and F. M. Ausubel, *Plant J.* **4**, 403 (1993).
6. M. Frey, R. Kliem, H. Saedler, A. Gierl, *Mol. Gen. Genet.* **246**, 100 (1995). The new gene designation is as follows: *Bx2 = Cyp71C4*, *Bx3 = Cyp71C2*, *Bx4 = Cyp71C1*, and *Bx5 = Cyp71C3*.
7. Individual plants were analyzed by PCR amplification of DNA restriction fragments ligated to an adapter, similar to the amplified fragment length polymorphism (AFLP) protocol [P. Vos et al., *Nucleic Acids Res.* **23**, 4407 (1995)]; however, a Mu-specific biotinylated primer [AGAGAAGCCAACGGCA(A/T)CGCCTCCATT] was used to isolate Mu-sequences.
8. The maize inbred line C31A was the source for all wild-type cDNA and genomic clones. Cloning was performed as described (6).
9. V. C. Kramer and M. G. Koziel, *Plant Mol. Biol.* **27**, 1183 (1995).
10. C. C. Hyde, S. A. Ahmed, E. A. Padlan, E. W. Miles, D. R. Davies, *J. Biol. Chem.* **263**, 17857 (1988).
11. E. R. Radwanski, J. Zhao, R. L. Last, *Mol. Gen. Genet.* **248**, 657 (1995).
12. Isolation of indole-3-glycerol phosphate and fluorometric enzymatic assay were done as described [T. E. Creighton, *Eur. J. Biochem.* **13**, 1 (1970)]. A reaction volume of 1 ml containing 4 μg of BX1 protein, 50 μg of glyceraldehyde phosphate dehydrogenase, and 0.5 mM nicotinamide adenine dinucleotide (oxidized form) was incubated at 22°C for 2 min. Indole-3-glycerol phosphate concentration varied from 0 to 50 μM. The identity of the product indole was proven by Ehrlich’s reagent [W. K. Lim et

al., *J. Bacteriol.* **173**, 1886 (1991)]. BX1 protein was expressed in *E. coli* by inserting the cDNA into a modified pET3a vector [A. H. Rosenberg et al., *Gene* **56**, 125 (1987)] and was purified to homogeneity by means of a six-nucleotide oligomer COOH-terminal histidine tag (Qiagen Ni-NTA purification system).

13. W. O. Weischet and K. Kirschner, *Eur. J. Biochem.* **65**, 375 (1976).
14. A. D. Wright, C. A. Moehlenkamp, G. H. H. Perrot, M. G. Neuffer, K. C. Cone, *Plant Cell* **4**, 711 (1992).
15. E. Glawischnig, W. Eisenreich, A. Bacher, M. Frey, A. Gierl, *Phytochemistry* **45**, 715 (1997).
16. The Pioneer Hi-Bred collection of 42,300 F₁ maize plants mutagenized by means of a Robertson’s Mutator element was screened for Mu-containing alleles of the P-450 genes by a reverse genetics-based technology [R. J. Benson et al., *Plant Cell* **7**, 75 (1995)]. PCR amplifications were done as described [M. Mena et al., *Science* **274**, 1537 (1996)].
17. G. Truan et al., *Gene* **125**, 49 (1993).
18. P. Urban et al., *Eur. J. Biochem.* **222**, 843 (1994). Full-size P-450 cDNAs were inserted in the expression vector, beginning with the AUG translation start codon. This was accomplished by PCR amplification of the relevant cDNA sequences.
19. D. Pompon, B. Louerat, A. Bronine, P. Urban, *Methods Enzymol.* **272**, 51 (1996).
20. Commercially available isatin (indole-2,3-dione) was reduced to 3-hydroxyindolin-2-one in a WATM 11 yeast culture; 1 mM isatin was added and the culture was grown overnight. 3-Hydroxyindolin-2-one was recovered from the cell-free media by ethyl acetate extraction and purified by HPLC. The structure of 3-hydroxyindolin-2-one was confirmed by ¹H NMR (Table 1).
21. B. A. Bailey and R. L. Larson, *Plant Physiol.* **95**, 792 (1991).
22. M. D. Woodward, L. J. Corcuera, H. K. Schnoes, J. P. Helgeson, C. D. Upper, *ibid.* **63**, 9 (1979). We determined (trimethylsilyl)₂ (TMS₂)-HBOA mass-to-charge ratios (*m/e*) of 309 (100%), 294 (29%), 266 (28%), 220 (12%), 208 (14%), 193 (17%), 192 (35%), 191 (15%), and 147 (95%). For TMS₂-DIBOA, *m/e* values were 325 (31%), 310 (84%), 297 (17%), 208 (43%), 192 (54%), 191 (30%), 179 (36%), 164 (72%), 151 (24%), 150 (23%), 147 (100%), and 136 (73%).
23. P. Kumar, D. E. Moreland, W. S. Chilton, *Phytochemistry* **36**, 893 (1994).
24. E. R. Radwanski and R. L. Last, *Plant Cell* **7**, 921 (1995).
25. T. M. Kutchan, *ibid.*, p. 1059.
26. J. Tsuji et al., *Physiol. Mol. Plant Pathol.* **43**, 221 (1993).
27. J. Zhao and R. L. Last, *Plant Cell* **8**, 2235 (1996).
28. We used gradient-enhanced heteronuclear multiple-quantum coherence, heteronuclear multiple-bond correlation, and double-quantum filtered correlation spectroscopy techniques (M. Frey et al., data not shown).
29. We thank R. Hüttl and A. O’Donnell for technical assistance; K. Barstad for screening of the Mu-population; M. H. Zenk for [3-¹³C]indole; M. H. Zenk, T. M. Kutchan, and K. Kirschner for discussions and suggestions; J. Winkler and H. Krause for GC-MS analysis; and P. Urban and D. Pompon for the yeast expression system. Supported by Deutsche Forschungsgemeinschaft grant SFB 369 and by Fonds der Chemischen Industrie.

5 March 1997; accepted 6 June 1997
