
JOURNAL OF BACTERIOLOGY, Sept. 1997, p. 5482–5493

0021-9193/97/$04.00+0

Copyright © 1997, American Society for Microbiology

# TRAP Transporters: a New Family of Periplasmic Solute Transport Systems Encoded by the *dctPQM* Genes of *Rhodobacter capsulatus* and by Homologs in Diverse Gram-Negative Bacteria

**JASON A. FORWARD, MARK C. BEHRENDT, NEIL R. WYBORN, RICHARD CROSS, AND DAVID J. KELLY**

*Krebs Institute, Department of Molecular Biology and Biotechnology, University of Sheffield, Sheffield S10 2UH, United Kingdom*

Received 13 May 1997/Accepted 1 July 1997

The *dct* locus of *Rhodobacter capsulatus* encodes a high-affinity transport system for the C₄-dicarboxylates malate, succinate, and fumarate. The nucleotide sequence of the region downstream of the previously sequenced *dctP* gene (encoding a periplasmic C₄-dicarboxylate-binding protein) was determined. Two open reading frames (ORFs) of 681 bp (*dctQ*) and 1,320 bp (*dctM*) were identified as additional *dct* genes by insertional mutagenesis and complementation studies. DctQ (24,763 Da) and DctM (46,827 Da) had hydrophobic profiles consistent with the presence of 4 and 12 potential transmembrane segments, respectively, and were localized in the cytoplasmic membrane fraction after heterologous expression of the *dctQM* ORFs in *Escherichia coli*. DctP, DctQ, and DctM were found to be unrelated to known transport proteins in the ABC (ATP-binding cassette) superfamily but were shown to be homologous with the products of previously unidentified ORFs in a number of gram-negative bacteria, including *Bordetella pertussis*, *E. coli*, *Salmonella typhimurium*, *Haemophilus influenzae*, and *Synechocystis* sp. strain PCC6803. An additional ORF (*rypA*) downstream of *dctM* encodes a protein with sequence similarity to eukaryotic protein-tyrosine phosphatases, but interposon mutagenesis of this ORF did not result in a Dct⁻ phenotype. Complementation of a *Rhizobium meliloti dctABD* deletion mutant by heterologous expression of the *dctPQM* genes from *R. capsulatus* demonstrated that no additional structural genes were required to form a functional transport system. Transport via the Dct system was vanadate insensitive, and in uncoupler titrations with intact cells, the decrease in the rate of succinate transport correlated closely with the fall in membrane potential but not with the cellular ATP concentration, implying that the proton motive force, rather than ATP hydrolysis, drives uptake. It is concluded that the *R. capsulatus* Dct system is a new type of periplasmic secondary transporter and that similar, hitherto-unrecognized systems are widespread in gram-negative bacteria. The name TRAP (for tripartite ATP-independent periplasmic) transporters is proposed for this new group.

---

Bacterial binding-protein-dependent solute transport systems are structurally complex, consisting of both periplasmic and membrane-bound components (3). To date, all such systems which have been sequenced and characterized functionally possess, in addition to the periplasmic binding protein and one or two integral membrane proteins, a highly conserved energy-coupling protein containing an ATP-binding cassette (the ABC protein) (28). In contrast, secondary transport systems, which are coupled to an electrochemical ion gradient rather than ATP hydrolysis, are much simpler and usually consist of a single integral membrane protein only. A number of families of related symporters, antiporters, and uniporters have been identified by primary sequence comparisons (18, 23, 40). Periplasmic binding proteins always appear to be a component of uptake, but not efflux, systems in the ABC transporter family but have never been found to be associated with secondary transporters, although there appears to be no thermodynamic reason why such systems should not exist (44). In this paper, evidence is presented which suggests that the high-affinity, binding-protein-dependent, C₄-dicarboxylate transport system of the purple photosynthetic bacterium *Rhodobacter capsulatus* is an example of such a novel system.

*R. capsulatus* is used widely as a model for the study of photosynthesis, nitrogen fixation, and the regulation of gene expression in response to environmental factors. It has an adaptable physiology and is able to utilize a wide range of carbon sources for chemoheterotrophic and photoheterotrophic growth. Particularly good carbon sources are the C₄-dicarboxylates malate and succinate (60). Aerobic growth on these substrates is mediated by a single osmotic-shock-sensitive, binding-protein-dependent transport system (21, 53) encoded by the *dct* locus (21, 22). Thus far, three genes essential for transport have been identified, one structural (*dctP*) and two regulatory (*dctS* and *dctR*). *dctP* encodes the periplasmic C₄-dicarboxylate-binding protein (DctP) which has been purified (54) and characterized extensively with respect to its ligand-binding kinetics and conformational thermodynamics (64, 65). DctP is very distantly related to the citrate-binding protein (TctC) of the TctI citrate transport system in *Salmonella typhimurium* (61). The *dctSR* operon is transcribed divergently from *dctP* (Fig. 1) and encodes a two-component sensor-regulator system which controls expression of the *dctP* operon (22). The *dct* locus has been physically mapped at high resolution on the *R. capsulatus* chromosome (16).

In this study, sequence analysis and mutagenesis have identified two additional genes (*dctQ* and *dctM*) essential for C₄-dicarboxylate transport activity, which encode integral membrane proteins. However, the *R. capsulatus dct* locus does not contain a gene encoding an ABC protein. This, combined with

* Corresponding author. Mailing address: Department of Molecular Biology and Biotechnology, Firth Court, Western Bank, Sheffield S10 2TN, United Kingdom. Phone: 44 114 222 4414. Fax: 44 114 272 8697. E-mail: d.kelly@sheffield.ac.uk.

the finding that expression of only the known *dct* genes results in the functional complementation of a *Rhizobium meliloti dct* mutant and with bioenergetic evidence against a role for ATP in transport, has led us to propose that this high-affinity transporter represents a novel type of system which utilizes a periplasmic binding protein in the transport mechanism. Partly as a result of recent genome-sequencing efforts, it has become clear that homologous systems are also present in *Escherichia coli*, *S. typhimurium*, *Bordetella pertussis*, *Haemophilus influenzae*, *Synecohcystis spp.*, and *Thiosphaera pantotropha*, indicating a widespread distribution among gram-negative bacteria.

### MATERIALS AND METHODS

**Bacterial strains, plasmids, and growth conditions.** Strains, vectors, and recombinant plasmids used in this work are described in Table 1. *R. capsulatus* was grown routinely in RCV medium (27, 66) with either D,L-malate, succinate, fumarate, or D,L-lactate as the sole carbon source (0.4% [wt/vol]) at 30°C under aerobic conditions in the dark in shake flasks. RCVYE media contained 0.05% (wt/vol) yeast extract in addition. Photoheterotrophic growth rates were determined by using cultures in completely filled test tubes capped with Suba-Seals, which fitted into the sample compartment of a Corning model 254 photometer equipped with a 710-nm filter. The tubes were illuminated by a 100-W tungsten bulb at a distance of 30 cm. Growth rates under chemoheterotrophic conditions were determined in 25-ml cultures contained in 250-ml shake flasks with side arms for turbidity measurements as described above. The flasks were shaken at 250 rpm at 30°C in the dark. *Rhizobium meliloti* was grown aerobically at 30°C in Luria-Bertani (LB) or RCV media, the latter with glucose or a C₄-dicarboxylate (0.4% [wt/vol]) as the carbon source. *E. coli* strains were grown aerobically at 37°C in LB media with the following concentrations of antibiotics: tetracycline, 10 μg ml⁻¹; ampicillin, 100 μg ml⁻¹; carbenicillin, 100 μg ml⁻¹; kanamycin, 25 μg ml⁻¹; rifampin, 200 μg ml⁻¹; spectinomycin, 25 μg ml⁻¹. For *R. capsulatus*, the following concentrations were used: spectinomycin, 25 μg ml⁻¹; streptomycin, 50 μg ml⁻¹; kanamycin, 30 μg ml⁻¹; rifampin, 25 μg ml⁻¹; and tetracycline, 1 μg ml⁻¹. For *Rhizobium meliloti*, rifampin was used at 25 μg ml⁻¹ and tetracycline was used at 1 μg ml⁻¹.

**DNA isolation and manipulation.** Standard techniques were used for the cloning, transformation, preparation, and restriction analysis of plasmid DNA from *E. coli* (51). Southern hybridizations were performed after transfer of restriction fragments from 0.5% agarose gels to nitrocellulose filters with a nonradioactive DNA labelling and detection kit (Boehringer Mannheim) under conditions specified by the manufacturer. Large-scale chromosomal DNA preparations from *R. capsulatus* were made by lysozyme-sodium dodecyl sulfate (SDS) treatment followed by CsCl gradient centrifugation.

**Nucleotide sequencing and analysis.** Restriction fragments from pDCT205 (Fig. 1 and Table 1) were cloned into pBluescript II SK(+) and KS(+) phagemid vectors (to give pDCT215, pDCT216, and pDCT220 to pDCT223) and transformed into *E. coli* XL1-Blue. Unidirectional deletions of phagemid inserts were constructed with an exonuclease III deletion kit (Pharmacia), and selected deletion clones were transfected with the VCSM13 helper phage to produce single-stranded DNA templates. These were sequenced completely on both strands by the dideoxy chain termination method by using a combination of custom-made and universal oligonucleotide primers with a commercially available kit (Sequenase; United States Biochemical Corp.). Compressions were resolved by using dITP in place of dGTP in the termination reactions. DNA sequences were compiled and analyzed with the programs devised by Staden (57, 58). Probable coding regions were identified with a codon preference plot (59) generated from the known codon usage of 16 genes from *R. capsulatus* sequenced previously (a total of 3,979 codons), as compiled by Armstrong et al. (5). Sequence similarities were searched for by using FASTA (48) with the OWL composite database (2) on the SEQNET facility of the Daresbury Laboratory (Warrington, United Kingdom). Hydropathy profiles of amino acid sequences were constructed by the method of Kyte and Doolittle (35). The statistical significance of sequence alignments was tested with the University of Wisconsin Genetics Computer Group BESTFIT program (11).

**Expression and localization of DctQ and DctM.** The *dctQ* and *dctM* genes were coamplified from pDCT205 by PCR. The forward primer (5′ GGCCCA TATGCTGCGCATCCTCGACCGG 3′) contained an *NdeI* restriction site at the *dctQ* initiation codon (underlined). The reverse primer (5′ TAGCAGATCT GGGGGCCCAAAGGCCCGGA 3′) was designed to be downstream of the *dctM* stop codon and contained a *BgII* site (underlined). The resulting 2.0-kb product was cloned into T7 expression vector pET-21a to give pJAF400, which was transformed into *E. coli* BL21(DE3). BL21(DE3, pJAF400) was grown to mid-exponential phase in 2 liters of Terrific Broth plus carbenicillin, harvested, and washed once in M9 medium (51). The cells were resuspended in 200 ml of M9 medium plus carbenicillin, the mixture was divided into equal volumes, and isopropyl-β-D-thiogalactopyranoside (IPTG) was added to one of the cultures to a 1 mM final concentration. The two cultures were incubated with shaking at 37°C for 15 min. Rifampin was then added, and incubation was continued for a further 90 min before 740 kBq of \[^{35}S\]methionine (>37 TBq mmol⁻¹; ICN) was added to each culture. After incubation for a further 1 h, yeast extract was added (0.1% [wt/vol] final concentration), and the cells were harvested, washed twice, and resuspended in M9 medium.

The cells were converted to spheroplasts and lysed by the method of Osborn and Munson (46), and total membrane pellets were obtained by differential centrifugation (200,000 × g, 2 h, 4°C). The membrane pellets were resuspended in a small volume of 10 mM Tris-HCl (pH 8.0) containing 25% (wt/vol) sucrose–5 mM EDTA and layered onto 12-ml discontinuous 30 to 55% (wt/wt) sucrose gradients which were centrifuged at 178,000 × g for 16 h at 4°C. The gradients were fractionated into 0.5-ml aliquots, which were assayed for absorbance at 280 nm and NADH oxidase activity (rate of decrease in absorbance at 340 nm in an assay mixture containing 10 mM Tris-HCl [pH 8.0] and 0.15 mM NADH). Samples from the cytoplasmic membrane fraction (containing NADH oxidase activity) were denatured in SDS sample buffer (15 min at 55°C) and analyzed by SDS-polyacrylamide gel electrophoresis. Gels were stained with Coomassie blue, destained, and then fluorographed by soaking in Amplify (Amershham) according to the manufacturer’s instructions, followed by drying and exposure to X-ray film for 48 h at –70°C.

**Transfer of plasmids to *Rhizobium meliloti*.** For the transfer of pDCT205 to *Rhizobium meliloti*, a spontaneous rifampin-resistant mutant of RmF726 was first selected and designated RmF726-1. Matings between *E. coli* S17-1(pDCT205) and RmF726-1 were set up on the surface of nitrocellulose filters on LB plates and were incubated at 30°C for 5 h before serial dilutions were plated onto RCVYE-glucose-rifampin agar (viable count) and RCVYE-glucose-rifampin-tetracycline agar (plasmid selection). Transconjugants were purified and stored in 20% (vol/vol) glycerol at –20°C.

**Insertional mutagenesis.** The 2.0-kb Ω interposon encoding a spectinomycin resistance gene flanked by transcriptional and translational termination signals (49) was cloned between the two *PstI* sites in pDCT225, resulting in a 240-bp deletion towards the 3′ end of *dctP* (Fig. 1). The resulting plasmid (pDCT225Ω) was transformed into XL1-Blue and conjugated into *R. capsulatus* PAS100 by a triparental mating with pRK2013 as the helper plasmid. Spectinomycin-resistant colonies were selected on RCV-pyruvate plates and were tested for growth on RCV-malate. Those unable to grow on the latter carbon source were purified, chromosomal DNA was prepared from them by the rapid method of Kranz (34), and DNA dot blots were prepared and probed with digoxigenin-labelled pARO180. Colonies which did not contain plasmid DNA were retained, and large-scale genomic DNA preparations were made and subjected to restriction endonuclease digestion and Southern hybridization with digoxigenin-labelled pHP45Ω as the probe. The *dctP::Ω* strain was designated MCB1. The Ω fragment was also inserted into a *SmaI* site towards the 3′ end of the *dctM* gene (Fig. 1). The 3.9-kb *BamHI-EcoRI* fragment from pDCT223 was first cloned into pARO181 to give pDCT226, which was cut with *SmaI* and ligated to the *SmaI*-cut Ω fragment from pHP45Ω to give pDCT227. After transformation into *E. coli*, conjugation into *R. capsulatus* PAS100, and screening as described above, the final *dctM::Ω* mutant selected was designated MCB200. For disruption of *rypA*, pDCT226 was cut at a unique *MluI* site within *rypA* (Fig. 1), end filled, and blunt-end ligated to the *SmaI*-cut Ω fragment from pHP45Ω to give pDCT228. This was transformed into XL1-Blue and mated triparentally into PAS100 as described above. Spectinomycin-resistant transconjugants were selected on RCV-lactate plates, and those which were kanamycin sensitive (and thus not likely to have retained the vector) were purified and their DNA was subjected to Southern hybridization to verify the construction. The final *rypA::Ω* mutant selected was designated MCB301.

**Mutagenesis of *dctQ* was achieved with the Tn5-derived kanamycin resistance cartridge from pUC4-KIXX.** This contains an outward-reading promoter which results in the expression of downstream genes when inserted in the correct orientation, thus producing a nonpolar mutation (7). The 1.8-kb cartridge was cloned in both orientations into the unique *dctQ BamHI* site in pUB3 to give pUB4 and pUB5 (Table 1), which were conjugated into PAS100 as described above. Kanamycin-resistant colonies that grew on RCV-lactate but not on RCV-succinate arose in both cases. These were purified, and genomic DNA was extracted and used as the template in a diagnostic PCR. The forward primer (5′ CGAAGAGGTGCTGATTGCCG 3′) was designed 5′ to the *BamHI* site in *dctQ*, and the reverse primer (5′ TTCGGTGTCGACGCCATCGA 3′) was designed 3′ to this site, so that, for the wild type, PCR would yield a single product of 0.53 kb, while mutants resulting from the correct allelic exchange should yield a single PCR product of 2.32 kb (0.53 kb plus 1.8 kb). Two such mutants with the cartridge in each orientation were thus identified and designated NRW100 and NRW200 (Fig. 1).

**Chemotaxis assay.** Chemotaxis was evaluated by the plug-plate technique, which measures the ability of cells to exhibit an accumulation response in a nutrient gradient. Cells were harvested from exponential phase RCV cultures containing lactate (40 mM) and D,L-malate (5 mM) as the carbon sources, washed once, and resuspended in RCV medium lacking any carbon source (RCV-C). Hard agar (2% [wt/vol]) plugs containing the desired chemoattractant (40 mM in RCV salts) were placed in the centers of petri dishes, and RCV-C containing 0.3% (wt/vol) agar and cells (to give an optical density at 710 nm of about 0.2) was poured around the plugs. The plates were incubated at 30°C for up to 5 h, and the formation of chemotactic rings was noted.

TABLE 1. Strains and plasmids used in this study

| Strain or plasmid | Genotype or description | Phenotype$^a$ | Source or reference |
|-------------------|-------------------------|---------------|---------------------|
| R. capsulatus     |                         |               |                     |
| SB1003           | rif-10                  | Dct$^{+}$Rf$^{r}$ | 62                  |
| PAS 100          | str-2hsd-1              | Dct$^{+}$Sm$^{r}$ | 62                  |
| MCB 1            | str-2hsd-1$\Delta$dctP::Q | Dct$^{-}$Sp$^{r}$Sm$^{r}$ | This study        |
| NRW 100          | str-2hsd-1dctQ::km      | Dct$^{-}$Km$^{r}$Sm$^{r}$ | This study        |
| NRW 200          | str-2hsd-1dctQ::km      | Dct$^{-}$Km$^{r}$Sm$^{r}$ | This study        |
| MCB 200          | str-2hsd-1dctM::Q       | Dct$^{-}$Sp$^{r}$Sm$^{r}$ | This study        |
| MCB 301          | str-2hsd-1rypA::Q       | Dct$^{+}$Sp$^{r}$Sm$^{r}$ | This study        |
| JGS 52           | str-2hsd-1dctS::Q       | Dct$^{-}$Sp$^{r}$Sm$^{r}$ | 22                 |
| MJH 240          | str-2hsd-1dctR::Q       | Dct$^{-}$Sp$^{r}$Sm$^{r}$ | 22                 |
| Rhizobium meliloti |                         |               |                     |
| Rm1021           | SU47str-21 (wild type)  | Dct$^{+}$Sm$^{r}$ | 43                  |
| RmF726           | SU47$\Delta\Omega$5149-5079::Tn5-233 | Dct$^{-}$Sp$^{r}$Sm$^{r}$ | 8                   |
| RmF726-1         | SU47$\Delta\Omega$5149-5079::Tn5-233rif-1 | Dct$^{-}$Sp$^{r}$Rf$^{r}$Sm$^{r}$ | This study        |
| E. coli          |                         |               |                     |
| XL1-Blue         | recA endA1 gyrA96 thi-1 hsdR-17 supE44 relA1 lac [F' proAB lacI$^{q}$Z$\Delta$M15 Tn10] | Tc$^{r}$ | Stratagene       |
| S17-1            | hsdR prorecA RP4-2(Tc::Mu; Km::Tn7) in chromosome | Tp$^{r}$Sp$^{r}$ | 55                  |
| BL21(DE3)        | F$^{-}$ompTr$^{-}$m$^{-}$ (DE3) | Novagen       |                     |
| Plasmids         |                         |               |                     |
| pBluescript II SK/KS(+) | Ap$^{r}$ | Stratagene       |                     |
| pET-21a          | T7 expression vector    | Ap$^{r}$ | Novagen        |
| pUC4-KIXX        | Source of Tn5-derived nptII gene | Km$^{r}$ | Pharmacia       |
| pARO 180         | Mobilizable pUC18 derivative | Ap$^{r}$ | 47                 |
| pARO 181         | As pARO180 but with the bla gene replaced by the nptII gene from Tn5 | Km$^{r}$ | 47                 |
| pCHB 500         | PycA cloned into pRK415 | Tc$^{r}$ |                     |
| pRK415           | Broad-host-range cloning vector | Tc$^{r}$ | 33                 |
| pHIP45$\Omega$   | 2.0-kb $\Omega$ fragment in pHIP45 | Ap$^{r}$Sp$^{r}$ | 49                 |
| pRK2013          | Helper plasmid          | Km$^{r}$Mob$^{+}$Tra$^{+}$ | 12                 |
| pDCT 203         | 4.3-kb EcoRI fragment containing dctQM and rypA in pRK415 | Tc$^{r}$ | 21                 |
| pDCT 205         | 8.3-kb partial EcoRI subclone containing dctPQMSR and rypA in pRK415 | Tc$^{r}$ | 21                 |
| pDCT 207         | 5.5-kb HindIII fragment from pDCT205in pRK415 | Tc$^{r}$ | 21                 |
| pDCT 208         | 4.5-kb EcoRI-BamHI fragment from pDCT205in pRK415 | Tc$^{r}$ | 21                 |
| pDCT 215/216     | 1.56-kb HindIII-BamHI fragment from pDCT205in pBluescript KS (215)/SK(216) | Ap$^{r}$ | This study        |
| pDCT 220/221     | 0.7-kb EcoRI-PstI fragment from pDCT205in pBluescript KS (220)/SK(221) | Ap$^{r}$ | This study        |
| pDCT 222/223     | 3.9-kb BamHI-EcoRI fragment from pDCT205in pBluescript KS (222)/SK(223) | Ap$^{r}$ | This study        |
| pDCT 225         | 1.56-kb HindIII-BamHI fragment in pARO180 | Ap$^{r}$ | This study        |
| pDCT225$\Omega$  | 2.0-kb $\Omega$ fragment inserted between the PstI sites in dctP in pDCT225 | Ap$^{r}$Sp$^{r}$ | This study        |
| pDCT 226         | 3.9-kb BamHI-EcoRI fragment in pARO181 | Km$^{r}$ | This study        |
| pDCT 227         | 2.0-kb $\Omega$ fragment inserted at the SmaI site in dctM in pDCT226 | Km$^{r}$Sp$^{r}$ | This study        |
| pDCT 228         | 2.0-kb $\Omega$ fragment inserted at the MluI site in rypA in pDCT226 | Km$^{r}$Sp$^{r}$ | This study        |
| pDCT 229         | 4.5-kb EcoRI fragment from pDCT205in pCHB500 in reverse orientation (PycA-dctM) | Tc$^{r}$ | This study        |
| pDCT 230         | 4.5-kb EcoRI fragment from pDCT205in pCHB500 in forward orientation (PycA-dctP) | Tc$^{r}$ | This study        |
| pUB3             | 2.8-kb HindIII-SmaI fragment (dctPQM') in pARO180 | Ap$^{r}$ | This study        |
| pUB4             | nptII in the dctQBamHI site of pUB3 (forward) | Km$^{r}$Ap$^{r}$ | This study        |
| pUB5             | nptII in the dctQBamHI site of pUB3 (reverse) | Km$^{r}$Ap$^{r}$ | This study        |
| pJAF 400         | 2.0-kbdctQM insert in pET21a | Ap$^{r}$ | This study        |
| pUII 106         | lacY from E. coli in RSF1010 | Sm$^{r}$ | 1                    |

$^a$ Abbreviations of antibiotics: Ap, ampicillin; Tc, tetracycline; Rf, rifampin; Km, kanamycin; Sp, spectinomycin; Gm, gentamicin; Sm, streptomycin; Tp, trimethoprim.

Transport assays. For comparisons of wild-type and mutant transport activities, cells were grown aerobically in the dark to late exponential phase in RCV medium with pyruvate (30 mM) and D,L-malate (1 mM) as the carbon sources. For bioenergetic studies (see below), cells were grown phototrophically in standard RCV-D,L-malate medium. After being harvested by centrifugation, cells were washed once in RCV-C, resuspended in the same medium at a concentration of 20 to 30 mg of cell protein ml$^{-1}$ and kept on ice for no longer than 6 h. Transport of 6 $\mu$M [$^{14}$C]succinate (1.8 to 2.2 GBq mmol$^{-1}$) was measured in a Clarke type oxygen electrode assembly at 30°C under aerobic conditions (53). Cells were added to 2 ml of RCV-C and allowed to equilibrate 5 min before the addition of the radiolabelled substrate. Samples (0.1 ml) were taken every 10 s, added to stop buffer (64 mM phosphate buffer [pH 7.0], 10 mM succinate, 0.2 mM sodium fluoroacetate), and filtered rapidly onto Whatman GF/F filters, which were dried and scintillation counted. Uncouplers were added from concentrated methanolic solutions 5 min before the addition of the isotope. For vanadate inhibition studies, cells were washed twice and finally resuspended in 20

1 kb

E P H P E BP S P M P E
205 | | | | | | | | |
< R > < S > < P > < Q > < M > < rypA > < draG >
Ω Ω Ω Ω Ω
MJH240 JGS52 MCB1 MCB200 MCB301
NRW100 nptII
NRW200 nptII
208
207 PycyA
230
229

FIG. 1. Gene organization at the *R.* *capsulatus dctRSPQM* locus, restriction map, location of insertion mutations, and complementation analysis. The region sequenced in this study is indicated by the thick black line above the restriction map. Complementation tests were carried out by mating pDCT plasmids (the inserts of which are numbered) into the strains indicated, with selection on both lactate and malate plates. Complementation was scored positive if the numbers of CFU were similar on both sets of plates (note that as the mutants are all *recA*⁺, repair of the mutations through recombination could occur at low frequency). E, EcoRI; P, PstI; H, HindIII; B, BamHI; S, SmaI; M, MluI. ND, not done.

227-residue protein of molecular mass 24,763 Da (DctQ) and a 440-residue protein of molecular mass 46,827 Da (DctM). Both of these ORFs were preceded by potential Shine-Dalgarno sequences (Fig. 2). The second ORF (*dctM*) was terminated by an ochre stop codon, which was followed 8 bp downstream by a GC-rich region of dyad symmetry with the potential to form a stable stem-loop structure in mRNA (∆G = −40.8 kcal mol⁻¹; calculated as described in reference 63). Thus, the sequence data indicate a probable *dctPQM* operon.

A third complete ORF was also identified, starting 63 bp downstream from the putative stem-loop structure. The translation initiation region contains a potential Shine-Dalgarno sequence (GAGG) followed 7 bp downstream by an ATG codon (Fig. 2). A database search revealed that the deduced protein encoded by this ORF (419 residues; 43,937 Da) showed some similarity to the members of the eukaryotic protein-tyrosine phosphatase family (21 to 30% amino acid sequence identity over 100 to 120 residues). These proteins are characterized by the possession of a diagnostic sequence motif at the active site (I/VHCXA/GGXGRS/TG) containing an essential cysteine residue which forms a thiol-phosphate intermediate during catalysis (14, 20). This active-site sequence, including the conserved cysteine, is present in the predicted protein (Fig. 2). The gene was thus designated *rypA* (for *Rhodobacter* tyrosine phosphatase). The *dctM-rypA* intergenic region was found to contain a sequence (TTGGGC-N₁₇-TATCCT) similar to those of promoter elements upstream of the *Rhodobacter sphaeroides cycA* gene (39). Finally, the 3′ end of an additional ORF was found to overlap *rypA* by 31 bp (Fig. 2). This encodes the dinitrogenase reductase-activating glycohydrolase (DraG) of *R.* *capsulatus*, which has been sequenced previously and which mapped close to *dctP* (41).

Insertional mutagenesis of the *dctP* operon; *dctP, dctQ,* and *dctM* are essential for C₄-dicarboxylate transport. In order to analyze the function of the genes in the putative *dctPQM* operon, null mutants were constructed in each ORF and a complementation analysis was performed (Fig. 1). An insertion-deletion mutant in *dctP* (MCB1) was constructed by replacing a 240-bp internal *PstI* fragment with the 2.0-kb spectinomycin-resistant Ω cartridge (49). The Ω cartridge was also inserted at unique *SmaI* and *MluI* sites in the *dctM* and *rypA* genes, respectively, to give strains MCB200 and MCB301, respectively. Finally, a kanamycin resistance cartridge, carrying its own promoter, was inserted in both orientations at the unique *BamHI* site in *dctQ* to give strains NRW100 and -200. The correct construction of all mutants was verified by Southern blotting of chromosomal DNA with cartridge, gene, and vector probes or by PCR (data not shown).

All of these mutants except MCB301 were unable to transport [¹⁴C]succinate at an external concentration of 6 μM (Table 2) and did not grow on D,L-malate, succinate, or fumarate as the sole carbon source under aerobic conditions in the dark. Slow photoheterotrophic growth on these substrates was observed (Table 2) as found previously for *dct* mutants made by transposon mutagenesis (21). MCB301 grew normally on lactate, D,L-malate, succinate, and fumarate under both chemoheterotrophic and photoheterotrophic conditions with doubling times similar to that of its isogenic parent, PAS100 (Table 2). Uptake experiments confirmed that MCB301 could transport [¹⁴C]succinate at rates similar to those for PAS100 (Table 2). These data clearly show that *rypA* is not needed for high-affinity C₄-dicarboxylate transport and is not an additional *dct* gene. By using the plug-plate technique, the chemotactic response of MCB1, MCB200, and MCB301 towards C₄-dicarboxylates was tested (Table 2), in order to determine whether transport is necessary for chemotaxis to these substrates. For MCB1 and MCB200,

mM HEPES–10 mM KCl (pH 8.0) and incubated with 1 mM sodium orthovanadate at 30°C for 15 min. The transport of 6 μM [¹⁴C]succinate, 10 μM 3-[¹⁴C]aminoisobutyrate (AIB; 1.8 to 2.2 GBq mmol⁻¹), or 20 μM [¹⁴C]lactose (0.9 to 1.1 GBq mmol⁻¹) was measured as described above in 2 ml of 20 mM HEPES–10 mM KCl–0.2% (wt/vol) glucose (pH 8.0). For AIB or lactose transport, the stop buffer contained either 10 mM AIB or lactose, respectively.

Measurement of membrane potential, ATP, intracellular pH, and cell volume. The measurement of succinate uptake and membrane potential and the ATP determination were all performed under similar conditions with the same cell suspension. The membrane potential in intact cells was measured by the carotenoid band shift technique (42). Cells were added to air-saturated RCV-C in a 3-ml glass cuvette, and the mixture was placed in the thermostated (30°C) sample holder of a Cary 219 spectrophotometer equipped with a magnetic stirrer. The carotenoid absorbance was measured at a single wavelength (528 nm) and was allowed to reach a steady value before a particular concentration of uncoupler was added. After 5 min, a high concentration (10 μM) of FCCP (trifluorothoxycarbonylcyanide phenylhydrazone) was added to collapse the membrane potential completely and thus establish a ∆Ψ = 0 baseline, from which values of the uninhibited and partially inhibited ∆Ψ were measured. A separate cuvette was prepared for each concentration of uncoupler used. For ATP determinations, cells were incubated in the oxygen electrode in exactly the same way as for succinate transport measurements and samples (0.3 ml) were manually injected into an equal volume of ice-cold 0.4 M formic acid–17 mM EDTA (37). After centrifugation at 13,800 × g for 5 min, ATP concentrations in the supernatants were determined by the luciferin-luciferase technique with a kit (Boehringer Mannheim). Intracellular pH (inside acid) was determined by using a [¹⁴C]methylamine distribution. Then, 16 kBq of [¹⁴C]methylamine (specific activity, 1.85 to 2.2 GBq mmol⁻¹) was added to a 2-ml cell suspension (containing 2 to 3 mg of cell protein) in RCV-C (pH 8.0) and incubated at 30°C. After addition of the required concentration of uncoupler, samples (0.3 ml) were taken, added to 0.3 ml of a silicone oil mixture (Dow Corning 200/1 grade/Dow Corning 550 grade ratio, 1:4 [vol/vol]), and centrifuged at 13,800 × g for 2 min. The radioactivity levels in samples (50 μl) of the aqueous supernatant and in the entire cell pellet were then determined by scintillation counting. Cell volumes were determined on the same cell suspension with tritiated water and [¹⁴C]dextran, as described previously (53).

Nucleotide sequence accession number. The nucleotide sequence reported in this paper has been submitted to the EMBL, GenBank, and DDJB databases under the accession number X63974.

---

**RESULTS**

Nucleotide sequence of the *dctP* 3′ region. Figure 1 shows the physical and genetic map of the *dct* locus of *R.* *capsulatus*. The nucleotide sequence of the region 3′ to *dctP* is shown in Fig. 2. Two open reading frames (ORFs) of 681 and 1,320 bp (designated *dctQ* and *dctM*, respectively; see below) were identified immediately downstream of *dctP*; these ORFs could encode a

5486 FORWARD ET AL. J. BACTERIOL.

$dctP \rightarrow$

| A E R I G A E T I A A V K A A T A E * |
| --- |
| GCCGAGCGGATCGGCGCCGAAACCATCGCCGCGGTCAAGGCGCGACGGCGGAATAAGAC |
| 970 980 990 1000 1010 1020 |

| CTGGGGCGGCGCCGACGGCGCCCCTTCCCTTCCACGACCTAAGCGGGGGGCC |
| --- |
| 1030 1040 1050 1060 1070 1080 |

$dctQ \rightarrow$

| M L R I L D R A E E V L I A A L I A T |
| --- |
| GCCATGCTGCGCATCCTCGACCGGGCGAAGAGGTGCTGATTGCGGCGCTCATCGCGACG |
| 1090 1100 1110 1120 1130 1140 |

| A T V L I F V S V T H R F T L G F V A D |
| --- |
| GCGACGGTGCTGATCTTCGTGTCGGTCACCCATCGTTTCACCTTGGCTTCGTCGCGGAT |
| 1150 1160 1170 1180 1190 1200 |

| F V G F F R G H G M T G A A A A K S L |
| --- |
| TTCGTGGGCTTCTTCCGGCGCCACGGCATGACCGGTGCGCCGCGCGCGCAAAGCCTT |
| 1210 1220 1230 1240 1250 1260 |

| Y T T L R G I N L V W A Q E L C I I L F |
| --- |
| TACACGACGCTGCGGCGCATCAACCTCGTCTGGGCGCAGGAGCTGTGCATCATCCTGTTTC |
| 1270 1280 1290 1300 1310 1320 |

| V W M A K F G A A Y G V R T G I H V G I |
| --- |
| GTCTGGATGGCGAAATTCGGCGCGGCTTACGGCGTGCGCACCGGATCCATGTCGGCATC |
| 1330 1340 1350 1360 1370 1380 |

| D V L I N R L D A P K R R F F I L L G L |
| --- |
| GACGTGCTGATCAACGGCTTGACGGCGCAAAGGGCGTTCTTCATCCTGCTGGGGTCTG |
| 1390 1400 1410 1420 1430 1440 |

| G A G A L F T G I I A T L G A N F V L H |
| --- |
| GGGGCCGGGGCGCTTCCAACGGCATCATCGGAGACGCTGGGCGCAAATTCGTGCTGAC |
| 1450 1460 1470 1480 1490 1500 |

| M Y H A S T S P D L E L P M W L V Y L |
| --- |
| ATGTATCACGGCTTCTCGACCTCGCCGATCTGGAACTGGCGATGTGGCTGGTTTATCTG |
| 1510 1520 1530 1540 1550 1560 |

| A I P M G S S L M C F R F L Q V A F G F |
| --- |
| GCGATCCCGATGGGGTGGTCGCTGATGTGCTTCCGCTTCGAGGTGGCCTTCGGCTTT |
| 1570 1580 1590 1600 1610 1620 |

| A R T G E L P H H D H G H V D G V D T E |
| --- |
| GCCCGCACCGGCGAGCTGCGGATCACGACCACGGCATGTGGATGGCGTGGACACCGAA |
| 1630 1640 1650 1660 1670 1680 |

| N E G I D A E G D V L L H S P L T P R D |
| --- |
| AACGAGGGCATCGATGGGGCGAGGGCGACGTGCTGCTGCAATTCGCGCTGACGGCGCGAC |
| 1690 1700 1710 1720 1730 1740 |

$dctM \rightarrow$

| L V E K P_K_D N * M S A L I I F G L L I |
| --- |
| CTTGTCGAAAAACCGAAGGACA ACTGACATGAGCGCCCTCATCATCTTTGGTCTGCTCAT |
| 1750 1760 1770 1780 1790 1800 |

| A L M L T G M P I S I S L G L T V L T F |
| --- |
| CGCGCTGATGCTGACCGGCA TGCCGATCTCGATCTCGCTTGGCCTGACGGTGCTGACCTT |
| 1810 1820 1830 1840 1850 1860 |

| L F T M T Q V P I D T V A L K L F T G I |
| --- |
| CCTGTTCA CCATGACCCAGGTGCCGATCGACACGGTGGCGCTCAAGCTCTTCA CCGGCAT |
| 1870 1880 1900 1910 1920 |

| E K F E I M A I P F F I A L G N F L T H |
| --- |
| CGAGAAATTCGAGATCATGGCGATCCCGTTCTTCA TCCTGGGGGAACCTCCTGACCCA |
| 1930 1940 1950 1960 1970 1980 |

| G G V A K R M I N F A T A M V G H W H G |
| --- |
| TGCGGGGGTGGCGAAGCGGATGATCAACTCGCCACGGGATGGTCCGTCACTGGGACGG |
| 1990 2000 2010 2020 2030 2040 |

| G L G L A G V I A C A L F A A V S G S S |
| --- |
| CGGGCTTGGCCTTTCGGGGCGGTGATCGCCTGCGCGCTTTTCGCCGCGGTGTCGGGCTCCTC |
| 2050 2060 2070 2080 2090 2100 |

| P A T V V A I G S V I L P A M V N Q G F |
| --- |
| GCCCGCCACCGTCGTAGCCATCGGCTCGGGTGA TCCTGCCCCGATGGTCAATCAGGGCCTT |
| 2110 2120 2130 2140 2150 2160 |

| P K Q F G A G V I T T S G A L G I L I P |
| --- |
| TCCGAAGCAATTCGGCGCGGCGGTCA TCACCCTCGGGGCCCTTGGGCATTCTCATTCC |
| 2170 2180 2190 2200 2210 2220 |

| P S I V M V M Y A V A T S G M V V T G P |
| --- |
| GCCGTCCATCGTGTATGGGTGATSTATGCGGTCGCA CCTCGGGGATGGTGGTGA CCGGGCC |
| 2230 2240 2250 2260 2270 2280 |

| D G Q P V S S A S V G E L F M A G V V P |
| --- |
| GGATGGCCAGCCGTTTCCCTCGG CCTCGGTGGGGAACTCTT CATGGCGGGGGTCGTGCC |
| 2290 2300 2310 2320 2330 2340 |

| G L M L A G F L A F T T W N R A R K F G |
| --- |
| CGGTCTGATGCTGGCGGGCTTCC TCGCCTTCA CCCTGGAACCGGGCGCGCAAATTCGG |
| 2350 2360 2370 2380 2390 2400 |

| Y P R L E K A S L R Q R W T A F R E A A |
| --- |
| CTATCCGCGTCIGGAAAAGCCCA GCGCTGCGCCAGCGCTGGACCGCCTTCCGCGAGGCCG |
| 2410 2420 2430 2440 2450 2460 |

| W G L M L I V V V I G G I Y A G I F T P |
| --- |
| CTGGGGGCTGATGCTGCTGCTG CATCGGGGATCTATGCGGGGATCTTCA CGCC |
| 2470 2480 2490 2500 2510 2520 |

FIG. 2. Nucleotide and deduced amino acid sequence of the $dctQM-rypA$ region. The nucleotide coordinates follow on from those given in reference 54. Convergent single-headed arrows indicate potential stem-loop structures capable of forming in mRNA. Short double-overlined regions upstream of initiation codons indicate potential Shine-Dalgarno sequences. Dashed lines between $dctM$ and $rypA$ indicate sequences similar to those of the $-35$ and $-10$ promoter regions of the $cycA$ gene of $R$. sphaeroides (39). The amino acid sequence in RypA corresponding to the active-site motif of the protein-tyrosine phosphatase family (14) is underlined, and the essential cysteine residue is indicated by a solid circle.

an accumulation response to lactate was clearly observable with each mutant after 4 to 5 h of incubation, but no such response could be detected with D,L-malate, succinate, or fumarate as the attractant. In contrast, the chemotactic response of MCB301 to $C_4$-dicarboxylates appeared normal (Table 2). MCB1 was complemented by pDCT207, a plasmid containing intact copies of both $dctP$ and the 3' ORFs, but was not complemented by pDCT208, containing the entire $dctP$ gene but lacking sequences beyond the unique BamHI site located in the coding region of the first downstream ORF (Fig. 1). Clearly, insertions in $dctP$ are polar on the downstream ORF(s), indicating that one or more of these genes are part of the same operon. Evidence that $dctP$, $dctQ$, and $dctM$ are each essential for $C_4$-dicarboxylate transport is provided by the following results: (i) pDCT230 (expressing $dctQ$ and $dctM$ under the control of the $R$. capsulatus cytochrome $c_2$ promoter in pCHB500) complemented MCB200 but not MCB1, (ii) insertions of the kanamycin resistance cartridge in $dctQ$ in either orientation (producing a polar or a nonpolar effect) resulted in an identical Dct phenotype (Table 2), and (iii) insertions in $dctM$ but not the downstream ORF ($rypA$) resulted in a Dct phenotype.

DctQ and DctM are hydrophobic integral membrane proteins. The amino acid composition of the deduced DctQ and DctM proteins reveals that they are very hydrophobic, with 62 and 73% apolar residues (i.e., MIVLPGAFW), respectively. This suggests strongly that they are both integral membrane proteins. The hydropathy profile of DctQ (Fig. 3) shows that there are four particularly hydrophobic regions of at least 20 amino acids which are potential membrane-spanning sequences. In contrast, the profile of DctM implies a total of 12 potential membrane-spanning segments, arranged as two groups of 6 separated by a markedly hydrophilic region (Fig. 3). To confirm that the $dctQ$ and $dctM$ genes encode the predicted membrane proteins, a PCR product containing both genes was cloned into a T7 expression vector, $E$. coli BL21 (DE3) was transformed with the construct (pJAF400), and a culture was induced with IPTG. This resulted in a marked insert-specific growth inhibition (data not shown). Membrane proteins synthesized under these conditions were detected by labelling cells with ${}^{[35]}$S]methionine in the presence of rifampin. NADH oxidase activity was used as a marker for the identification of fractions containing the cytoplasmic membrane. After a crude total membrane pellet was loaded onto a 30 to 55% sucrose gradient, two peaks in the $A_{280}$ protein profile were apparent (centered at about 32 and 42% sucrose). The denser of these was identified as the cytoplasmic membrane band, as it contained all of the NADH oxidase activity and was yellowish-brown in color. The peak of radioactivity in the gradient also correlated with this lower band. Two IPTG-

VOL. 179, 1997 TRAP TRANSPORTERS IN DIVERSE GRAM-NEGATIVE BACTERIA 5487

T E A A A M S A V Y A F F I S V F V Y K  
GACCGAAGCCGCCGGATGAGCGCCTCTACGGTTTTTCATCTCGGTCTTCGTCTACAAA  
2530 2540 2550 2560 2570 2580  

D L T L R D V P R V L L S S A N M S A M  
GGACCTGACCCCTGCGCGACGTGCCCCGGGTGCTGCTGCTCCGCAAACATGTCGGCGAT  
2590 2600 2610 2620 2630 2640  

L L Y I I T N A V L F S F L M A H E G I  
GCTGCTTTACATCATCACCAATGCGGTGCTGTTCAAGTTCCCTGATGGCGCATGAGGGCAT  
2650 2660 2670 2680 2690 2700  

P Q A L G E W M V N A G L S W W M F L I  
CCCGCAGGCCTGGGGCGAATGGTGAAACGGGGTCTGCTCCTGGGTGATGTTCCCTGAT  
2710 2720 2730 2740 2750 2760  

I V N I L L L A A G N F M E P S S I V L  
CATCGTCAAACATCCTGCTGCTGGCGGGCAACTTCATGGAGCCCCTCCGATCGTCTT  
2770 2780 2790 2800 2810 2820  

I M A P I L F P V A V R L G I D P V H F  
GATCATGGCGCCATCCTGTTTCGGTGCGGTGCGGCTGGGCATCGACC CGGTGCATT  
2830 2840 2850 2860 2870 2880  

G I M I V V N M E V G M C H P P V G L N  
CGGCATCATGATCGTGCATCAACATGGAAAGTCGGCATGTGCCACCCGCGGGTGGGGCTGAA  
2890 2900 2910 2920 2930 2940  

L Y V A S G I T K M G I T E L T V A V W  
CCTTTACGTCG CCTCGGGGATCCAAGATGGGCATCACCGAACTGACGGTGGCGGTCTG  
2950 2960 2970 2980 2990 3000  

P W L L T M L A F L V L V T Y V P A I S  
GCCCTGGCTTCTGACCATGCTCGCCCTT CCTGTCGTGGTGACCTATGTCCCCGGCGATCAG  
3010 3020 3030 3040 3050 3060  

L A L P N L L G M *  
TCTGGCCTT GCCAAATCTGCTTGGCGCAGGGCCGGGTTTCCCCGGC  
3070 3080 3090 3100 3110 3120  

=  
CCTGCGACGTTCCGGGCTTGGGCCCCCCTGCGCCGCCCTATCCTGCCCCGGACCACG  
3130 3140 3150 3160 3170 3180  

rypA →  
M A S A R T S L T H P L Q I A E V  
AGGGGGATCATGGCCAGCGCCCGCACCAGGCTTACCCATCCGCTGCAGATCGCCGAGGTT  
3190 3200 3210 3220 3230 3240  

P A G P G L G R I G I T F C P G K H D R  
CCGGCCGGGCCCGGCTGGGTGCGCATCGGCATCACCTTTTGGCCCGGCAAGCACGACCGC  
3250 3260 3270 3280 3290 3300  

A A M S G A W A R D L G L D L D A I L D  
GCCGCGATGTCGGGCGCCTGGGGCGCGATCTGGGCGCCTGATCGATGCGGATTTGCGGAT  
3310 3320 3330 3340 3350 3360  

FIG. 2—Continued.

inducible radiolabelled proteins of about 30 and 26 kDa, cor-  
responding to DctM and DctQ, respectively, were present in  
the cytoplasmic membrane fraction (Fig. 4). These proteins  
were synthesized in small amounts, since neither was visible  
after Coomassie blue staining (data not shown).  
Homologs of DctP, DctQ, and DctM exist in diverse gram-  
negative bacteria: evidence for a new family of solute trans-  
porters. Database searches revealed no significant sequence  
similarity between the Dct proteins of *R. capsulatus* and the  
proteins of other well-characterized ABC or secondary trans-  
port systems. Many integral membrane proteins of diverse  
periplasmic binding protein-dependent transport systems in  
the ABC superfamily share a conserved hydrophilic sequence  
with the consensus EAA X₃GX₉IFLP, which is located about  
80 to 90 residues from the C terminus (9) and which may  
interact with the ABC protein (25). Neither DctQ nor DctM  
contains sequences conforming to this consensus. However,  
ORFs sequenced previously were identified in *E. coli* (56), *B.*  
*pertussis* (67, 68), *H. influenzae* (15), and *Synechocystis* sp. strain  
PCC 6803 (32), which encode products with significant se-  
quence similarity to DctP, DctQ, and DctM (Table 3). DctM  
homologs are also present in *S. typhimurium* (38) and in the  
denitrifying bacterium *T. pantotropha* (6). From a statistical  
evaluation of the sequence alignments it can be concluded that  
the Rhodobacter DctP and DctM proteins are homologous with  
all of the corresponding proteins listed in Table 3 (i.e., the  
alignments scored > 9 standard deviation [SD] units above the  

W G A A H V L T L V E P Q E L G M L K V  
TG GGGCGCGGCCCATGTGCTGACGCTGGTCGAGCCGCAGGA ACTGGGCATGCTCAAGGTG  
3370 3380 3390 3400 3410 3420  

P D L G T Q G P R A G M D W H P L P I A  
CCGGACCTTGGGCAGCGAGGGTCCGGCGCGGGGATGGACTGGCATCC CCTGCGGATTTGCC  
3430 3440 3450 3460 3470 3480  

D Y S V P T P A F E A R W Q A E G R V I  
GATTATTCCGTGCCGAA CCCGGGCTTTGAAGCGGCGTGGCAGGGCGAGGGGGGTGATC  
3490 3500 3510 3520 3530 3540  

R A A L R A G A D V V V H C K G G L G R  
CGCGCCGCCCTGCGTGGGGCGCCGATGTCGTGTCGTGCATTGCA AGGGCGGGCTTGGCCGG  
3550 3560 3570 3580 3590 3600  

A_G M I A A A R L L L V E L G A D P K A A  
GGGGGGATGATCGCGGCGGGCGGCTGGTGGAACTGGGCGCCATCCGAAAAGCGGC  
3610 3620 3630 3640 3650 3660  

V N A V R T A R P G A I E T P A Q L A L  
GTGAA CGCCGTGCGGACGGCCCCGGCCCGCGCGATCGAGACGCCTGCGCAA CTGGCCCTT  
3670 3680 3690 3700 3710 3720  

V R A T L P L A E P A R V D P A Q L T R  
GTCCGCGCGAGCGCTGCGGCTGGCCGAGCCCGCGGGTGCATCCGGCGACGTGACACGC  
3730 3740 3750 3760 3770 3780  

V G G R L G S N P G G I W A D A A G G R  
GTCCGCGGGCGGCTCGGCTCGGAATCCCGGCGGGATCTGGGCGATGGCGGGCGGGCGG  
3790 3800 3810 3820 3830 3840  

L Y V K E L E S P A H A R N E Y L A A A  
CTTTACGTC AAGGAGCTGGA AAAGCCCCGGCCATGCCCGGAACGAATATCTGGCGGGCGG  
3850 3860 3870 3880 3890 3900  

L Y R L A G A P V L S Y L P C A A P D Q  
CTTTACC GTCTGGCCGGGGCGCGGTGCTGCTTACCTGCCCC TGCGCCGCCCCGATCAG  
3910 3920 3930 3940 3950 3960  

V A T V F V D L E K S R L S Q L D P A E  
GTGGCGACGGTCTTCGTGATCTGGAAAAATCCCGCCTGTGCGACGCTGGGACCCGGGAG  
3970 3980 3990 4000 4010 4020  

R A Q A W R W F G V H A W A L N W D A A  
CGGGCGCAGG CCTGGGCGCTGGTTCCGGGTGCATGCTGGGCTGGGA CGGCC  
4030 4040 4050 4060 4070 4080  

G F Q G D N Q G V I G G V V T T L D V G  
GGGTTTCA GGGCGACAATCAGGGCGTGATCGGGCGGTGGTGACGACGCTCGATGTCGGC  
4090 4100 4110 4120 4130 4140  

G A L D F R A Q G D P K G R A F G P T V  
GGCGCGCTGG ACTTCCGGGCGCAGGGCGACCCCAAGGGCGCGCCTT CGGGCCCCACCGTG  
4150 4160 4170 4180 4190 4200  

P E V D R L R T D P D N P F A V A L F G  
CCCGAGGTTGGA CC GGCTGCGCCACCGATCCCGACAACCC TTTCGCGGTCGCGCTT CGGG  
4210 4220 4230 4240 4250 4260  

P M P P D A L R A A L M V V I G L P E A  
CCGATGCCGCCCGATGCCCTGCGTGGCGGCTGATGGTGGTGATCGGCCTGCCC GAGGCC  
4270 4280 4290 4300 4310 4320  

A I R I V V A R F G G R A A L A E T L L  
GCAATCCGCATTGTGCTGGCCCCGGTTCGGCGGCGGCGGCTGGCCGAGACGCTTCT  
4330 4340 4350 4360 4370 4380  

A R K A D L A R Q L T Q M P E S A S S F  
GCCCGCAAGCCGATCTGGCCCGGCAGCTCAGATGCCC GAAAGCGCCACGAGCTTC  
4390 4400 4410 4420 4430 4450  

CGGGCGTTCCGGCTAGACCGGGCGTCGAGTGGATCTACGGGCTTT CGCGGTCGCGAAG  
* I G S L A L L K  

G T *  
GGCACC TTGAGCGCGGATCTCGGCCGTCACCGCGGGGTC AAGCGCCGCAAGCCAGCGTTCC  
4460 4470 4480 4490 4500 4510  

CCGTGGACTCGCGCCTAGACCGGGCAGTGGCGCCCCAAGTTCGCGGGCGTT CGCGCAAGG  
P V Q A R I E A T V A P D L A A L W R E  

← 'draG'

mean for a randomized sequence comparison) (10) and that  
the DctQ alignments are highly significant (most SD values are  
between 6 and 8 units above the mean). In *E. coli*, the three  
ORFs (*orf157a*, *orf424*, and *orf328*, encoding products corre-  
sponding to DctQ, DctM, and DctP, respectively) are located  
in the 80-min region sequenced by Sofia et al. (56) and appear  
to be part of a large operon containing genes which are known  
to be involved in pentose sugar metabolism and which are  
required for growth of *E. coli* on L-lyxose (52). In *B. pertussis*,  
the product of an ORF located at the 3′ end of the fimbrial  
gene cluster (*orfC*) (67, 68) is homologous with DctM from *R.*  
*capsulatus* and the Orf424 DctM homolog in *E. coli* (Table 3).  
However, examination of the region upstream of *orfC* revealed  
two other ORFs, here designated *orf279* (nucleotide coordi-

TABLE 2. C4-dicarboxylate transport, growth, and chemotaxis phenotypes of isogenic null mutants$^a$

| Strain (relevant genotype) | Succinate uptake rate (nmol min⁻¹ mg of protein⁻¹) | Growth rate (doubling time; h) under$^b$: | Aerobic chemotaxis response$^d$ to: |
|----------------------------|-------------------------------------------------|------------------------------------------|-------------------------------------|
|                            |                                             | Aerobic, dark conditions on:              | Anaerobic, light conditions on:      | lac | mal | succ | fum |
|                            |                                             | lac                                      | mal                                | succ | fum | lac | mal | succ | fum |
| PAS 100 ($dct^{+}$)         | 12.4                                         | 5.5                                      | 7.5                                | 7.3  | 7.0 | 6.3 | 8.4 | 6.0  | 7.5  | +   | +   | +    | +    |
| MCB 1 ($dctP::\Omega$)      | 0                                            | 6.5                                      | NG$^c$                             | NG   | NG  | 5.8 | 11.8 | 11.0 | 18.8 | +   | –   | –    | –    |
| NRW 100 ($dctQ::km$)        | 0                                            | 3.4                                      | NG                                 | NG   | NG  | 3.0 | 10.0 | 17.5 | 15.0 | ND  | ND  | ND   | ND   |
| NRW 200 ($dctQ::km$)        | 0                                            | 3.2                                      | NG                                 | NG   | NG  | 3.1 | 16.0 | 8.5  | 20.3 | ND  | ND  | ND   | ND   |
| MCB 200 ($dctM::\Omega$)    | 0                                            | 3.3                                      | NG                                 | NG   | NG  | 5.0 | 14.3 | 17.5 | 23.5 | +   | –   | –    | –    |
| MCB 301 ($rypA::\Omega$)    | 11.0                                         | 4.8                                      | 7.0                                | 6.5  | 6.5 | 6.8 | 6.8 | 8.0  | 8.5  | +   | +   | +    | +    |

$a$ Cells were assayed for succinate uptake and chemotaxis as described in Materials and Methods.

$b$ lac, D,L-lactate; mal, D,L-malate; succ, succinate; fum, fumarate.

$c$ NG, no growth.

$d$ +, positive response; –, no response; ND, not determined.

TABLE 3. Proteins with sequence similarity to DctPQM are present in a diverse range of bacteria$^a$

| Organism               | Database entry (location)          | Reference | Gene$^g$ (homolog$^h$)      | Amino acid identity (%) | Overlap (no. of residues) | SD$^b$ |
|------------------------|------------------------------------|-----------|-------------------------------|-------------------------|--------------------------------|--------|
| E. coli                | U 00039 (GenBank)                 | 56        | orf157a(Q)                   | 23.0                    | 188                            | 7.8    |
|                        |                                    |           | orf424(M)                    | 32.0                    | 440                            | 42.0   |
|                        |                                    |           | orf328(P)                    | 28.3                    | 333                            | 24.6   |
| S. typhimurium         | U 09309 (GenBank)                 | 38        | ygiK(M)                      | 34.6                    | 440                            | 40.0   |
| B. pertussis           | X 64876 (EMBL)                    | 67, 68    | orf279(P)$^c$                | 25.7                    | 280$^c$                        | 10.5$^c$ |
|                        |                                    |           | orf166(Q)                    | 26.0                    | 196                            | 6.7    |
|                        |                                    |           | orfC(M)$^d$                  | 29.0                    | 405                            | 17.6   |
| Synechocystis sp. strain PCC 6803 | cyanobase(www)$^i$ | 32        | Slr1102(Q)                   | 24.7                    | 204                            | 7.7    |
|                        |                                    |           | Slr1103(M)                   | 29.7                    | 452                            | 18.0   |
|                        |                                    |           | Slr1104(glnH)                |                         |                                |        |
|                        |                                    |           | Slr11314(P)                  | 25.0                    | 368                            | 13.5   |
| H. influenzae          | L 42023 (GSDB)$^j$               | 15        | HI0052(P)                    | 28.0                    | 338                            | 16.5   |
|                        |                                    |           | HI0051(Q)                    | 16.5                    | 181                            | 4.5    |
|                        |                                    |           | HI0050(M)                    | 34.8                    | 418                            | 35.9   |
| H. influenzae          | L 42023 (GSDB)                   | 15        | HI0146(P)                    | 29.3                    | 331                            | 18.8   |
|                        |                                    |           | HI0147(QM)$^e$              |                         | $-$                            | $-$    |
| H. influenzae          | L 42023 (GSDB)                   | 15        | HI1028(Q)                    | 21.8                    | 187                            | 6.5    |
|                        |                                    |           | HI1029(M)                    | 31.5                    | 439                            | 38.2   |
|                        |                                    |           | HI1030(P)                    | 28.9                    | 333                            | 27.1   |
| T. pantotropha         | Z 36773 (EMBL)                   | 6         | dctM(M)$^f$                  | 39.6$^f$                | 205$^f$                        | 22.9$^f$ |

$a$ Comparisons were made with the Genetics Computer Group BESTFIT program (10) with a gap penalty of 0.3 and a gap length penalty of 0.1. The entire protein sequences were used for the alignments.

$b$ The statistical significance of the alignments was evaluated by using the RAN option of BESTFIT, set to produce 100 randomizations. The numbers of standard deviations (SD) that the real alignments scored above the mean for the randomized sequence comparison are shown.

$c$ Orf279 is incompletely sequenced, with approximately 50 residues missing from the N terminus.

$d$ The start codon of OrfC was assumed to be at nucleotide coordinate 8289 for this comparison.

$e$ HI0147 may contain a frameshift making a single long ORF. The product was excluded from this comparison.

$f$ The T. pantotropha DctM homolog is incompletely sequenced; only the last 205 C-terminal residues were available for this comparison.

$g$ Genes are listed in order for each organism, with the most 5' gene at the top.

$h$ The gene from R. capsulatus whose product is homologous to that of the listed gene. Q, dctQ; M, dctM; P, dctP.

$i$ www, World Wide Web.

$j$ GSDB, Genome Sequence Database.

known to be sensitive to alterations in intracellular pH (1, 3), and we have shown previously that the activity of the Dct system is markedly reduced at intracellular pH values below 7.0 (53). However, at least in the case of FCCP, measurements using \[^{14}C\]methylamine showed that the internal pH was not lowered significantly over the range of uncoupler concentrations used (data not shown).

C₄-dicarboxylate transport via the Dct system is vanadate insensitive. If indeed an ABC protein is not part of the Dct system and ATP hydrolysis is not the energy source for uptake, high-affinity succinate transport should be insensitive to inhibition by orthovanadate, which is a known potent inhibitor of P-type ATPases and has also been shown to inhibit classical binding-protein-dependent (ABC type) transporters (50). Figure 7 shows experiments with *R. capsulatus* cells in which the effect of 1 mM sodium orthovanadate on the uptake of 3-AIB (a nonmetabolizable analog of alanine), succinate, and lactose was determined. AIB uptake and lactose uptake are used here as positive and negative controls, respectively, for inhibition by vanadate. The alanine transporter in *R. sphaeroides* appears to be a typical binding-protein-dependent system which is known to be vanadate sensitive (1). In keeping with this, AIB uptake in *R. capsulatus* was inhibited by about 80% at 1 mM vanadate (Fig. 7). In contrast, lactose uptake [mediated in SB1003 (pUI06) by expression of the *E. coli* LacY protein] is clearly vanadate insensitive, as would be expected for this Δp-driven secondary transporter. In comparison to these controls, it is evident from Fig. 7 that succinate uptake via the Dct system is unequivocally vanadate insensitive.

## DISCUSSION

Several lines of evidence indicate that the Dct system in *R. capsulatus* is distinct both structurally and mechanistically from known periplasmic binding protein-dependent transport systems.

FIG. 6. Measurement of membrane potential, intracellular ATP concentration, and the rate of succinate transport at different concentrations of three uncouplers, FCCP, SF6847, and 2,4-dinitrophenol (2,4-DNP). The rate of \[^{14}C\]succinate uptake (●) into PAS100 cells at pH 8.0 was determined as described in Materials and Methods at each concentration of uncoupler. Cells from the same batch were incubated under identical conditions, and 5 min after addition of the required concentration of uncoupler, samples (0.3 ml) were extracted into formic acid-EDTA for ATP determination, as described in Materials and Methods (○). The initial intracellular ATP concentrations in the absence of uncoupler were determined to be 1.8 mM (a), 3.7 mM (b), and 3.3 mM (c). The dependence of membrane potential on uncoupler concentration was determined under conditions as similar as possible to those used for the uptake and ATP assays by measuring carotenoid electrochromism at 528 nm (□). As the carotenoid band shift has a linear dependence on membrane potential (29) and only relative values were necessary for the purposes of this experiment, the absorbance changes were calculated as a percentage of that in the absence of uncoupler, with a Δψ = 0 baseline established by the addition of 10 μM FCCP.

are part of the B800-850 light-harvesting complex in the cytoplasmic membrane (29). Unlike the use of probe ions, carotenoid electrochromism is a noninvasive method and has a linear dependence on the magnitude of the membrane potential (29). With increasing concentrations of each of the three uncouplers used, the membrane potential was progressively decreased (Fig. 6) and was collapsed completely by 0.2 μM SF6847 and by 2 μM FCCP. Importantly, the same correlation was apparent between the decrease in the rate of succinate transport and the decrease in membrane potential with each uncoupler. In contrast, although the intracellular ATP concentration was reduced by high concentrations of SF6847—to about 50% of the value in the absence of uncoupler—this was not evident with FCCP or 2,4-dinitrophenol; high intracellular ATP concentrations were maintained at titers of uncoupler at which the membrane potential and rate of succinate transport were zero (FCCP) or severely lowered (2,4-dinitrophenol). Treatment of intact cells with an uncoupler can lower the intracellular pH, and this could be responsible for the effects observed. Binding-protein-dependent transport systems are

![Figure 7](https://i.imgur.com/placeholder.png)

FIG. 7. Effect of orthovanadate on transport of AIB, succinate, and lactose in *R. capsulatus* SB1003(pUI106). Rates of solute uptake were measured as described in Materials and Methods at time zero (A), 15 min after incubation at 30°C in the absence of sodium orthovanadate (B), or after 15 min in the presence of 1 mM sodium orthovanadate (C). Actual transport rates corresponding to the initial time zero (*t* = 0) samples (A) were as follows: AIB, 10.3 ± 1.5 nmol min⁻¹ (mg of protein)⁻¹; succinate, 38.9 ± 3.9 nmol min⁻¹ (mg of protein)⁻¹; and lactose, 0.41 ± 0.2 nmol min⁻¹ (mg of protein)⁻¹. These rates represent the averages of at least three independent experiments.

tems in the ABC superfamily and represents a new type of bacterial solute transport system. Insertional mutagenesis and complementation analysis have shown that the three ORFs *dctP*, *dctQ*, and *dctM* are essential for C₄-dicarboxylate transport and are part of the same transcriptional unit. It was suggested previously (54) that the stem-loop structure in the *dctP-dctQ* intergenic region may act as a partial terminator, allowing some readthrough into *dctQ* and *dctM*, thus regulating the relative abundances of the periplasmic and membrane-bound gene products. The more stable stem-loop structure following the *dctM* gene and the presence of a promoter-like sequence upstream of *rypA* also argue for an operon structure containing *dctP*, *dctQ*, and *dctM*. Mutagenesis of *rypA* did not result in a Dct⁻ phenotype, and we have, as yet, been unable to assign any phenotype to MCB301. The function of the putative product of this ORF, which has the typical active-site motif of the protein-tyrosine phosphatase family, is thus currently unknown. The overlapping convergent arrangement of *rypA* with *draG* is unusual and may indicate some regulatory connection between the two genes.

From the present study and from previous mutagenesis and complementation data (21, 22) we conclude that *dctP*, *dctQ*, and *dctM* are the sole structural genes of the high-affinity C₄-dicarboxylate transport system in *R. capsulatus*, with no evidence for a gene encoding an ABC protein. We considered the possibility that such a gene is located elsewhere on the chromosome, although this type of organization has not been found for any other periplasmic solute uptake system (24–26). Two pieces of evidence suggest that all of the genetic information required for the synthesis of a functional transport system is encoded by the *dctPQMSR* genes. First, all of the Tn5 mutants isolated by Hamblin et al. (21) were located in this region. Second, plasmid pDCT205, containing the entire *dct* locus, was able to complement a strain of *Rhizobium meliloti* carrying a deletion of the *dctABD* genes. The possibility remains that an ABC protein in the rhizobial host is able to interact functionally with the *R. capsulatus* Dct system, although this seems unlikely in view of the specificity of a given ABC protein for a particular transport system.

The deduced products of the *dctQ* and *dctM* genes are not related in terms of primary sequence to the integral membrane proteins of other periplasmic transport systems, and they do not possess the conserved sequence in such proteins first described by Dassa and Hofnung (9). DctQ is unusual in its predicted topology, and its role in the transport system is unknown, but there are examples of small membrane transport proteins in which four transmembrane α-helices are predicted (19). The hydrophobic profile of DctM is, however, remarkably similar to that of the so-called 12-transmembrane helix transporters exemplified by many families of membrane transport proteins in bacteria and eukaryotes (23, 40). This is consistent with its having a role in solute translocation across the cytoplasmic membrane, even though no significant sequence similarity could be demonstrated with known proteins in any of these families. These considerations, combined with the recognition that DctPQM homologs clearly exist in *E. coli*, *B. pertussis*, *H. influenzae*, and *Synechocystis*, indicate that we have discovered a novel family of transporters which is distributed widely in gram-negative bacteria. We propose the name TRAP (tripartite ATP-independent periplasmic) transporters for this new family. In addition, a partially sequenced ORF encoding a protein with sequence similarity to DctM is present in the denitrifier *T. pantotropha* (6), and we have also recently cloned functionally homologous *dct* genes from *R. sphaeroides* (45). As in *R. capsulatus*, no ORFs encoding ABC proteins are present in any of the other gene clusters.

Only in the case of *Rhodobacter* has the nature of the transported substrates of one of these systems been established unequivocally, although it seems likely that the three-gene cluster in *Synechocystis* encodes a glutamine transport system. Recently, a binding-protein-dependent glutamate transport system from *R. sphaeroides* which has been reconstituted in membrane vesicles and which is clearly ionophore sensitive but vanadate insensitive has been described (30). Although no sequence data were reported, some of the characteristics of this system appear very similar to those of the Dct system. In *E. coli*, genetic defects in aerobic C₄-dicarboxylate transport have been mapped to the 78- to 80-min region (36, 56), and an ORF at 78 min (designated *dctA* in reference 56) has been identified as encoding a product homologous to the DctA proton:C₄-dicarboxylate transporter of *Rhizobium*. In contrast, the ORFs encoding the DctPQM homologs in the same region are apparently part of a large gene cluster related to rare pentose sugar utilization (52).

Given that a number of bacteria appear to possess binding-protein-dependent transport systems in the absence of ABC protein subunits, the question of the mechanism of energy coupling is clearly important. The Dct system is clearly vanadate insensitive, and the results of the uncoupler titrations clearly showed a close correlation of the rate of succinate transport with the magnitude of the membrane potential, but not with the intracellular ATP concentration, which is consistent with the membrane potential being the driving force for succinate transport. The use of uncouplers to investigate the energetics of periplasmic transport systems in intact cells can obviously be problematic, due to effects on the internal pH and the possibility that the reagent can directly inhibit the particular transport system under investigation (4). Under the conditions used here, we found that no decrease in intracellular pH was measurable over the range of FCCP concentrations employed. The use of three different uncouplers of widely differing potency resulted in a very close correlation of the succinate uptake rate with the membrane potential in each case, suggesting that there were no secondary effects on the transport system itself. These data suggest that the Dct system is a novel type of secondary transporter with an associated binding protein. Unfortunately, EDTA and sucrose (necessary for lysosome-mediated cell lysis) appear to cause irreversible inactivation of the transporter (17, 69), and this may hamper attempts to reconstitute this system in membrane vesicles for further bioenergetic studies.

A major conclusion of this work is that periplasmic binding proteins can be integral components of secondary transporters that are clearly not in the ABC/traffic-ATPase superfamily. This has implications for understanding the evolution of transport systems, the essentiality of the binding protein in these novel systems, and the role it plays in the overall transport mechanism. The function of the DctQ protein (the most divergent in terms of sequence similarity) and how interactions between DctP, -Q, and -M result in solute translocation remain to be established.

---

**ACKNOWLEDGMENTS**

This work was supported by a Biotechnology and Biological Sciences Research Council grant to D.J.K. (for N.R.W.) and by studentships to M.C.B., R.C., and J.A.F.

The Krebs Institute is a designated center for molecular recognition studies. This work benefitted from the use of the SEQNET facility of the EPSRC Daresbury Laboratory.

ADDENDUM IN PROOF

Additional database searches of recently released microbial genome sequences have revealed DctP and/or DctM homologs to be present in *Bacillus subtilis*, *Enterococcus faecalis*, *Deinococcus radiodurans*, *Vibrio cholerae*, *Treponema pallidum*, *Thermotoga maritima*, and *Archaeoglobus fulgidus*. Thus, TRAP transporters are present in at least some gram-positive bacteria and also in archaea, indicating a ubiquitous distribution in prokaryotes.

### REFERENCES

1. Abee, T., F.-J. van der Wal, K. J. Hellingwerf, and W. N. Konings. 1989. Binding-protein-dependent alanine transport in *Rhodobacter sphaeroides* is regulated by the internal pH. J. Bacteriol. 171:5148–5154.
2. Akrigg, D., A. J. Bleasby, N. I. M. Dix, J. B. C. Findlay, A. C. T. North, D. Parry-Smith, J. C. Wooton, T. L. Blundell, S. Gardner, F. Hayes, S. Islam, M. J. E. Sternberg, J. M. Thornton, I. J. Tickle, and P. Murray-Rust. 1988. A protein sequence/structure database. Nature 335:745–746.
3. Ames, G. F.-L. 1988. Structure and mechanism of bacterial periplasmic permeases. J. Bioenerg. Biomembr. 20:1–18.
4. Ames, G. F.-L., and A. K. Joshi. 1990. Energy coupling in bacterial periplasmic permeases. J. Bacteriol. 172:4133–4137.
5. Armstrong, G., M. Alberti, F. Leach, and J. E. Hearst. 1989. Nucleotide sequence, organisation and nature of the protein products of the carotenoid biosynthesis gene cluster of *Rhodobacter capsulatus*. Mol. Gen. Genet. 216: 254–268.
6. Berks, B. C., D. J. Richardson, A. Reilly, A. C. Willis, and S. J. Ferguson. 1995. The *napEDABC* gene cluster encoding the periplasmic nitrate reductase system of *Thiosphaera pantotropha*. Biochem. J. 309:983–992.
7. Bott, M., M. Meyer, and P. Dimroth. 1995. Regulation of anaerobic citrate metabolism in *Klebsiella pneumoniae*. Mol. Microbiol. 18:533–546.
8. Charles, T. C., and T. M. Finan. 1991. Analysis of a 1600-kilobase *Rhizobium meliloti* megaplasmid using defined deletions generated *in vivo*. Genetics 127:5–20.
9. Dassa, E., and M. Hofnung. 1985. Sequence of gene *malG* in *E. coli* K12: homologies between integral membrane components from binding-protein-dependent transport systems. EMBO J. 4:2287–2293.
10. Dayhoff, M. O., W. C. Barker, and L. T. Hunt. 1983. Establishing homologies in protein sequences. Methods Enzymol. 91:524–545.
11. Devereux, J., P. Haeberli, and O. Smithies. 1984. A comprehensive set of sequence analysis programs for the VAX. Nucleic Acids Res. 12:387–395.
12. Ditta, G., S. Stanfield, D. Corbin, and D. R. Helinski. 1980. Broad-host range DNA cloning system for gram-negative bacteria: construction of a gene bank of *Rhizobium meliloti*. Proc. Natl. Acad. Sci. USA 77:7347–7351.
13. Engelke, T., D. Jording, D. Kapp, and A. Pühler. 1989. Identification and sequence analysis of the *Rhizobium meliloti dctA* gene encoding the $C_4$-dicarboxylate carrier. J. Bacteriol. 171:5551–5560.
14. Fischer, E. H., H. Charbonneau, and N. K. Tonks. 1991. Protein tyrosine phosphatases: a diverse family of intracellular and transmembrane enzymes. Science 253:401–406.
15. Fleischmann, R. D., M. D. Adams, O. White, R. A. Clayton, E. F. Kirkness, A. R. Kerlavage, C. J. Bult, J.-F. Tomb, B. A. Dougherty, J. M. Merrick, K. McKenney, G. Sutton, W. FitzHugh, C. Fields, J. D. Gocayne, J. Scott, R. Shirley, L.-I. Liu, A. Glodek, J. M. Kelley, J. F. Weidman, C. A. Phillips, T. Spriggs, E. Hedblom, M. D. Cotton, T. R. Utterback, M. C. Hanna, D. T. Nguyen, D. M. Saudek, R. C. Brandon, L. D. Fine, J. L. Fritchman, J. L. Furhmann, N. S. M. Geoghagen, C. L. Gnehm, L. A. McDonald, K. V. Small, C. M. Fraser, H. O. Smith, and J. C. Venter. 1995. Whole genome random sequencing and assembly of *Haemophilus influenzae* Rd. Science 269:496–512.
16. Fonstein, M., and R. Haselkorn. 1993. Chromosomal structure of *Rhodobacter capsulatus* SB1003: cosmid encyclopedia and high-resolution physical and genetic map. Proc. Natl. Acad. Sci. USA 90:2522–2526.
17. Gibson, J. 1975. Uptake of $C_4$ dicarboxylates and pyruvate by *Rhodopseudomonas sphaeroides*. J. Bacteriol. 123:471–480.
18. Griffith, J. K., M. E. Baker, D. A. Rouch, M. G. P. Page, R. A. Skurray, I. T. Paulsen, K. F. Chater, S. A. Baldwin, and P. J. F. Henderson. 1992. Membrane transport proteins: implications of sequence comparisons. Curr. Opin. Cell Biol. 4:684–695.
19. Grinius, L., G. Dreguniene, E. B. Goldberg, C.-H. Liao, and S. J. Projan. 1992. A staphylococcal multidrug resistance gene product is a member of a new protein family. Plasmid 27:119–129.
20. Guan, K., and J. E. Dixon. 1990. Protein tyrosine phosphatase activity of an essential virulence determinant in *Yersinia*. Science 249:553–556.
21. Hamlin, M. J., J. G. Shaw, J. P. Curson, and D. J. Kelly. 1990. Mutagenesis, cloning and complementation analysis of $C_4$-dicarboxylate transport genes from *Rhodobacter capsulatus*. Mol. Microbiol. 4:1567–1574.
22. Hamlin, M. J., J. G. Shaw, and D. J. Kelly. 1993. Sequence analysis and interposon mutagenesis of a sensor-kinase (DctS) and response-regulator (DctR) controlling synthesis of the high-affinity $C_4$-dicarboxylate transport system in *Rhodobacter capsulatus*. Mol. Gen. Genet. 237:215–224.
23. Henderson, P. J. F. 1993. The 12-transmembrane helix transporters. Curr. Opin. Cell Biol. 5:708–721.
24. Higgins, C. F. 1992. ABC-transporters—from microorganisms to man. Annu. Rev. Cell Biol. 8:67–113.
25. Higgins, C. F., M. P. Gallagher, S. C. Hyde, M. L. Mimmack, and S. R. Pearce. 1990. Periplasmic binding-protein-dependent transport systems: the membrane-associated components. Philos. Trans. R. Soc. Lond. Ser. B 326: 353–365.
26. Higgins, C. F., S. C. Hyde, M. L. Mimmack, U. Gileadi, D. R. Gill, and M. P. Gallagher. 1990. Binding-protein-dependent transport systems. J. Bioenerg. Biomembr. 22:571–592.
27. Hillmer, P., and H. Gest. 1977. $H_2$ metabolism in the photosynthetic bacterium *Rhodospseudomonas capsulata*: $H_2$ production by growing cultures. J. Bacteriol. 129:724–731.
28. Hyde, S. C., P. Elmsley, M. J. Hartshorn, M. L. Mimmack, U. Gileadi, S. R. Pearce, M. P. Gallagher, D. R. Gill, R. E. Hubbard, and C. F. Higgins. 1990. Structural model of ATP binding-proteins associated with cystic fibrosis, multidrug resistance and bacterial transport. Nature 346:362–365.
29. Jackson, J. B. 1988. Bacterial photosynthesis, p. 317–375. *In* C. Anthony (ed.), Bacterial energy transduction. Academic Press, London, United Kingdom.
30. Jacobs, M. H. J., T. van der Heide, A. J. M. Driessen, and W. N. Konings. 1996. Glutamate transport in *Rhodobacter sphaeroides* is mediated by a novel binding-protein dependent secondary transport system. Proc. Natl. Acad. Sci. USA 93:12786–12790.
31. Jording, D., and A. Pühler. 1993. The membrane topology of the *Rhizobium meliloti* $C_4$-dicarboxylate permease (DctA) as derived from protein fusions with *Escherichia coli* K12 alkaline phosphatase (PhoA) and $\beta$-galactosidase (LacZ). Mol. Gen. Genet. 241:106–114.
32. Kaneko, T., S. Sato, H. Kotani, A. Tanaka, E. Asamizu, Y. Nakamura, N. Miyajima, M. Hirosawa, M. Sugiura, S. Sasamoto, T. Kimura, T. Hosouchi, A. Matsuno, A. Muraki, N. Nakazaki, K. Naruo, S. Okumura, S. Shimpo, C. Takeuchi, T. Wada, A. Watanabe, M. Yamada, M. Yasuda, and S. Tabata. 1996. Sequence analysis of the genome of the unicellular cyanobacterium *Synechocystis sp.* strain PCC6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions. DNA Res. 3:109–136.
33. Keen, N. T., S. Tamaki, D. Kobayashi, and D. Trollinger. 1988. Improved broad host-range plasmids for DNA cloning in gram-negative bacteria. Gene 70:191–197.
34. Kranz, R. G. 1989. Isolation of mutants and genes involved in cytochromes $c$ biosynthesis in *Rhodobacter capsulatus*. J. Bacteriol. 171:456–464.
35. Kyte, J., and R. F. Doolittle. 1982. A simple method for displaying the hydropathic character of a protein. J. Mol. Biol. 157:105–132.
36. Lo, T. C. Y., and B. D. Sanwal. 1975. Genetic analysis of mutants of *Escherichia coli* defective in dicarboxylate transport. Mol. Gen. Genet. 140:303–307.
37. Lundin, A., and A. Thore. 1975. Comparison of methods for extraction of bacterial adenine nucleotides determined by firefly assay. Appl. Microbiol. 30:713–721.
38. Luttinger, A. L., A. L. Springer, and M. B. Schmid. 1991. A cluster of genes that affects nucleotide segregation in *Salmonella typhimurium*. New Biol. 3:687–697.
39. MacGregor, B. J., and T. J. Donohue. 1991. Evidence for two promoters for the cytochrome $c_2$ gene (*cycA*) of *Rhodobacter sphaeroides*. J. Bacteriol. 173:3949–3957.
40. Marger, M. B., and M. H. Saier, Jr. 1993. A major superfamily of transmembrane facilitators that catalyse uniport, symport and antiport. Trends Biochem. Sci. 18:13–20.
41. Masepohl, B., R. Krey, and W. Klipp. 1993. The *draTG* gene region of *Rhodobacter capsulatus* is required for post-translational regulation of both the molybdenuim and the alternative nitrogenase. J. Gen. Microbiol. 139: 2667–2675.
42. McEwan, A. G., S. J. Ferguson, and J. B. Jackson. 1983. Electron flow to dimethylsulphoxide or trimethylamine-N-oxide generates a membrane potential in *Rhodospseudomonas capsulata*. Arch. Microbiol. 136:300–305.
43. Meade, H. M., S. R. Long, G. B. Ruvkun, S. E. Brown, and F. M. Ausubel. 1982. Physical and genetic characterization of symbiotic and auxotrophic mutants of *Rhizobium meliloti* induced by transposon Tn5 mutagenesis. J. Bacteriol. 149:114–122.
44. Nicholls, D. G., and S. J. Ferguson. 1992. Bioenergetics, 2nd ed. Academic Press, London, United Kingdom.
45. Omrani, M. D., and D. J. Kelly. Unpublished results.
46. Osborn, M. J., and R. Munson. 1974. Separation of the inner (cytoplasmic) and outer membranes of gram-negative bacteria. Methods Enzymol. 31:642–653.
47. Parke, D. 1990. Construction of mobilizable vectors derived from plasmids RP4, pUC18 and pUC19. Gene 93:135–137.
48. Pearson, W. R. 1990. Rapid and sensitive sequence comparison with FASTP and FASTA. Methods Enzymol. 183:63–97.

49. Prentki, P., and H. M. Krisch. 1984. *In vitro* insertional mutagenesis with a selectable DNA fragment. Gene **29**:303–313.

50. Richarme, G., A. Elyaagoubi, and M. Kohiyama. 1993. The MglA component of the binding-protein dependent galactose transport system of *Salmonella typhimurium* is a galactose stimulated ATPase. J. Biol. Chem. **268**:9473–9477.

51. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

52. Sanchez, J. C., R. Gimenez, A. Schneider, W. D. Fessner, L. Baldoma, J. Aguilar, and J. Badia. 1994. Activation of a cryptic gene encoding a kinase for L-xylose opens a new pathway for the utilisation of L-lyxose by *Escherichia coli*. J. Biol. Chem., **269**:29665–29669.

53. Shaw, J. G., and D. J. Kelly. 1991. Binding-protein dependent transport of C4-dicarboxylates in *Rhodobacter capsulatus*. Arch. Microbiol. **155**:466–472.

54. Shaw, J. G., M. J. Hamblin, and D. J. Kelly. 1991. Purification, characterisation and nucleotide sequence of the periplasmic C4-dicarboxylate binding-protein (DctP) from *Rhodobacter capsulatus*. Mol. Microbiol. **5**:3055–3062.

55. Simon, R., U. Priefer, and A. Pühler. 1983. A broad host range mobilisation system for *in vivo* genetic engineering: transposon mutagenesis in gram-negative bacteria. Bio/Technology **1**:784–791.

56. Sofia, H. J., V. Burland, D. D. Daniels, G. Plunkett III, and F. R. Blattner. 1994. Analysis of the *Escherichia coli* genome. V. DNA sequence of the region from 76.0 to 81.5 minutes. Nucleic Acids Res. **22**:2576–2586.

57. Staden, R. 1980. A new computer method for the storage and manipulation of DNA gel reading data. Nucleic Acids Res. **8**:3673–3694.

58. Staden, R. 1982. An interactive graphics program for comparing and aligning nucleic acid and amino-acid sequences. Nucleic Acids Res. **10**:2951–2961.

59. Staden, R., and A. D. McLachlan. 1982. Codon preference and its use in identifying protein coding regions in long DNA stretches. Nucleic Acids Res. **10**:141–156.

60. Stahl, C. L., and G. A. Sojka. 1973. Growth of *Rhodopseudomonas capsulata* on L- and D-malic acid. Biochim. Biophys. Acta **299**:241–245.

61. Tam, R., and M. H. Saier, Jr. 1993. Structural, functional, and evolutionary relationships among extracellular solute-binding receptors of bacteria. Microbiol. Rev. **57**:320–346.

62. Taylor, D. P., S. N. Cohen, W. G. Clark, and B. L. Marrs. 1983. Alignment of the genetic and restriction maps of the photosynthesis regions of the *Rhodopseudomonas capsulata* chromosome by a conjugation-mediated marker rescue technique. J. Bacteriol. **154**:580–590.

63. Tinoco, I., P. N. Borer, B. Dengler, M. D. Levire, O. C. Uhlenbeck, D. M. Crothers, and J. Gralla. 1973. Improved estimation of secondary structure in ribonucleic acids. Nature New Biol. **246**:40–41.

64. Walmsley, A. R., J. G. Shaw, and D. J. Kelly. 1992. The mechanism of ligand-binding to the periplasmic C4-dicarboxylate binding-protein (DctP) from *Rhodobacter capsulatus*. J. Biol. Chem. **267**:8064–8072.

65. Walmsley, A. R., J. G. Shaw, and D. J. Kelly. 1992. Perturbation of the equilibrium between open and closed conformations of the periplasmic C4-dicarboxylate binding-protein from *Rhodobacter capsulatus*. Biochemistry **31**:11175–11181.

66. Weaver, P. F., J. D. Wall, and H. Gest. 1975. Characterisation of *Rhodopseudomonas capsulata*. Arch. Microbiol. **105**:207–216.

67. Willems, R. J. L., H. G. J. van der Heide, and F. R. Mooi. 1992. Characterisation of a *Bordetella pertussis* fimbral gene cluster which is located directly downstream of the filamentous haemagglutinin gene. Mol. Microbiol. **6**:2661–2671.

68. Willems, R. J. L., C. Geuijen, H. G. J. van der Heide, G. Renauld, P. Bertin, W. M. R. van den Akker, C. Locht, and F. R. Mooi. 1994. Mutational analysis of the *Bordetella pertussis fim/fha* gene cluster: identification of a gene with sequence similarities to haemolysin accessory genes involved in export of FHA. Mol. Microbiol. **11**:337–347.

69. Wyborn, N. R., and D. J. Kelly. Unpublished results.
