
Biochem. J. (1998) 330, 121–129 (Printed in Great Britain)

# Effect of mutations in the transmethylase and dehydrogenase/chelatase domains of sirohaem synthase (CysG) on sirohaem and cobalamin biosynthesis

Sarah C. WOODCOCK*, Evelyne RAUX*, Florence LEVILLAYER†, Claude THERMES†, Alain RAMBACH‡ and Martin J. WARREN*¹

*Department of Molecular Genetics, Institute of Ophthalmology, University College London, Bath Street, London EC1V 9EL, U.K., †Centre de Génétique Moléculaire, Laboratoire associé à l'Université Pierre et Marie Curie, Centre National de la Recherche Scientifique, 91198 Gif-sur-Yvette, France, and ‡CHROMagar, Rue Lecourbe, Paris, 75015, France

The *Escherichia coli* CysG protein (sirohaem synthase) catalyses four separate reactions that are required for the transformation of uroporphyrinogen III into sirohaem, initially two S-adenosyl-L-methionine-dependent trans methylations at positions 2 and 7, mediated through the C-terminal, or CysG<sup>A</sup>, catalytic domain of the protein, and subsequently a ferrochelation and dehydrogenation, mediated through the N-terminal, or CysG<sup>B</sup>, catalytic domain of the enzyme. This report describes how the deletion of the NAD<sup>+</sup>-binding site of CysG, located within the first 35 residues of the N-terminus, is detrimental to the activity of CysG<sup>B</sup> but does not affect the catalytic activity of CysG<sup>A</sup>, whereas the mutation of a number of phylogenetically conserved residues within CysG<sup>A</sup> is detrimental to the transmethylation reaction but does not affect the activity of CysG<sup>B</sup>. Further studies have shown that CysG<sup>B</sup> is not essential for cobalamin biosynthesis because the presence of the *Salmonella typhimurium* CobI operon with either *cysG<sup>A</sup>* or the *Pseudomonas denitrificans* *cobA* are sufficient for the synthesis of cobyrinic acid in an *E. coli cysG* deletion strain. Evidence is also presented to suggest that a gene within the *S. typhimurium* CobI operon might act as a chelatase that, at low levels of cobalt, is able to aid in the synthesis of sirohaem.

## INTRODUCTION

Sirohaem and cobalamin are related macrocyclic structures derived from the ubiquitous tetapyrrolic primogenitor uroporphyrinogen III (uro'gen III) by a number of enzymic transformations (Scheme 1) [1–4]. Sirohaem is required as a prosthetic group in the six-electron reduction processes of both sulphite and nitrite reductases [5,6]. As coenzymes, cobalamins are essential in several aspects of metabolism with enzymes involved in rearrangement and methylation reactions [7].

Sirohaem and cobalamin are derived from uro'gen III by C-methylation of the tetapyrrole framework (Scheme 1). Thus two S-adenosyl-L-methionine (AdoMet)-dependent trans methylations of uro'gen III at positions 2 and 7 produce precorrin-2 (dihydrosirohydrochlorin) [8]. Sirohaem is synthesized via sirohydrochlorin by dehydrogenation of the pyrrocorphin precorrin-2, which is subsequently ferrochelated. However, methylation of precorrin-2 at position 20, as opposed to macrocyclic oxidation, yields precorrin-3 [9], a transformation that ensures that the substrate is destined for corrin manufacture. Hence precorrin-2 represents the last common intermediate of the sirohaem and cobalamin pathways.

The enzyme that catalyses the transformation of uro'gen III into precorrin-2 is the AdoMet-dependent uro'gen III transmethylase (also referred to as SUMT: AdoMet uroporphyrinogen methyltransferase) [10–12]. The uro'gen III transmethylases can be categorized as follows: (1) uro'gen III transmethylases with a molecular mass of approx. 30 kDa, which are able only to perform the transmethylation of uro'gen III (e.g. the CobA from *Pseudomonas denitrificans* [10]); (2) multifunctional enzymes with a molecular mass of approx. 50 kDa capable of converting uro'gen III into sirohaem (e.g. CysG sirohaem synthase [13–15]); (3) fusion proteins between CobA (single domain uro'gen III transmethylase) and HemD (uro'gen III synthase), the preceding enzyme in the tetapyrrole biosynthetic pathway (e.g. in *Clostridium josuii* [16]); (4) as CobA fusions with proteins of no known function (e.g. in *Saccharomyces cerevisiae* [17] and *Bacillus subtilis* [18]). A sequence comparison of some of the type (1) and type (2) enzymes is given in Figure 1.

As a type (2) enzyme, described above, the CysG protein from *Escherichia coli* has been shown to be a multifunctional enzyme capable of converting uro'gen III into sirohaem [14,15] and is similar in primary structure and function to the CysG proteins from *Salmonella typhimurium* [19] and *Neisseria meningitidis* (M. J. Warren, S. E. Wilkie, T. McVeigh, S. Peters, E. Raux and S. C. Woodcock, unpublished work). CysG is able to catalyse the two AdoMet trans methylations, an NAD<sup>+</sup>-dependent dehydrogenation as well as ferrochelation. All the isolated *cysG* genes encode for proteins with molecular masses of approx. 50 kDa, and are associated with operons of sulphite or nitrite metabolism. It is thought that these proteins might have arisen as a result of a gene fusion between a CobA protein (AdoMet-dependent transmethylase) and an oxidase/chelatase. With this view in mind the N-terminal domain of CysG is referred to as CysG<sup>B</sup> (oxidase/dehydrogenase domain) and the C-terminal domain as CysG<sup>A</sup> (transmethylase domain) [15].

With respect to the chelation activity of *E. coli* CysG, it was reported that the enzyme is able to accept a range of divalent metal ions as substrate, including Fe<sup>2+</sup>, Co<sup>2+</sup> and Ni<sup>2+</sup> [14]. It has been suggested that the observed cobalt chelation activity might reflect a function required for corrin biosynthesis [14,20] and it

Abbreviations used: AdoMet, S-adenosyl-L-methionine; CysG<sup>A</sup>, uroporphyrinogen III transmethylase domain of CysG; CysG<sup>B</sup>, dehydrogenase and ferrochelatase domain of CysG; uro'gen III, uroporphyrinogen III.

¹ To whom correspondence should be addressed.

UROPORPHYRINOGEN III

![Diagram](https://i.imgur.com/1234567.png)

**Scheme 1 Biosynthesis of sirohaem and cobyrinic acid from uro'gen III**

Uro'gen III is converted into precorrin-2 by two AdoMet-dependent transmethylation at positions 2 and 7 by the action of uro'gen III transmethylases. Oxidation of precorrin-2, such as that afforded by the NAD⁺-dependent dehydrogenation catalysed by CysGᴮ, yields sirohydrochlorin; a subsequent ferrochelation gives rise to sirohaem. Methylation of precorrin-2, as opposed to oxidation, produces precorrin-3, which directs the intermediate towards corrin biosynthesis, a process that to make cobyrinic acid, for instance, requires the action of a further 10 enzymes.

was hypothesized that CysG could insert cobalt into siro-
hydrochlorin, which is subsequently reduced to give cobalt-
precorrin-2. Furthermore, because mutations in *S.* typhimurium
*cysG* that affect sirohaem synthesis also seem to affect cobalamin
biosynthesis in parallel, it has been argued that the two domains
of CysG must be essential for both activities [21]. This paper
investigates the contribution made by the two functional domains
of CysG towards sirohaem and cobalamin biosynthesis and
whether the two regions of the protein act independently.

### MATERIALS AND METHODS

#### Chemicals

Most chemicals, reagents and antibiotics were obtained from
Sigma Chemical Company (Poole, Dorset, U.K.). Tryptone,
yeast extract and agar were purchased from Difco Laboratories
(Detroit, MI, U.S.A.). Mimetic Green I affinity chromatography
resin was purchased from Affinity Chromatography (Cambridge,
U.K.). Restriction endonucleases and PCR requisites were pur-
chased from Promega (Southampton, Hampshire, U.K.). *In vitro*
mutagenesis kit, Sequenase kit, \([α-^{35}S]\) dATP, \([methyl/-^{3}H]\) AdoMet
(80 Ci/mmol) and Hyperfilm-MP were purchased from
Amersham International (Little Chalfont, Bucks., U.K.). Auto-
fluor was purchased from National Diagnostics (Hull, East
Yorkshire, U.K.). Oligonucleotide primers were purchased from
Genosys (Cambridge, U.K.).

#### Plasmids and bacteria

All the strains and plasmids used in this study are described in

Table 1. Standard methods for molecular biology were used, as
described previously [22].

For ease of further cloning, the *cysG* expression strain pSW95
was generated. This was accomplished by amplification of the *E.*
*coli cysG* (1374 nt) by the PCR method of MacFerrin [23], with
universal primer and a 5′ primer containing a GC clamp, a
*BamHI* site, a ribosome-binding site, a transcriptional spacer
element and an ATG start codon. The fragment was purified on
a 1% (w/v) agarose gel, extracted and digested with the
restriction enzymes *BamHI/EcoRI* and ligated into pUC19,
which had been restricted previously in a similar fashion.

A CysG derivative (*CysG^NAD^*) lacking the N-terminally
located NAD⁺-binding site was expressed from a truncated *cysG*
from which the first 34 codons had been deleted. This was
accomplished by amplification of nt 103–1374 from pSW95 by
using universal primer and a primer specific for the new 5′ end of
the truncated gene, which contained the expression features
described above. The fragment was amplified by standard PCR
methods, purified on a 1% (w/v) agarose gel, extracted and
digested with the restriction enzymes *BamHI* and *EcoRI* and then
ligated into pUC19 that had been restricted in a similar fashion.
This procedure generated *pcysG^NAD^*, which expressed a trunc-
ated protein with the N-terminus starting M-GARLTV. The
clone was fully sequenced to ensure that no further mutations
had been introduced during the PCR procedure.

Point mutants of CysG were generated by using the techniques
described in [24]. The mutants were generated in M13mp18
containing the *BamHI/EcoRI* fragment from pSW95 containing
the *cysG* gene. The *cysG* mutants G21D, G224A, D227A, D248A,
K270I, R298L, D303A and R309L were generated from the
following primers: G21D, GTCGGCGCTGATGATGTC;

Table 1 List of strains and plasmids used in this study

| Plasmid strain | Description                                                                                   | Reference |
|----------------|---------------------------------------------------------------------------------------------|-----------|
| Strains        |                                                                                             |           |
| TB1            | JM83 $h s d R(r_{k}-m_{k}+)$                                                                | [39]      |
| 302Δa          | Nir$^{s}$ Lac$^{+}$ Cys$^{-}$                                                              | [30]      |
| AR3612         | Leu$^{+}$ Sm$^{r}$ cysG metE                                                               | [27]      |
| AR8906         | 302Δa/pAR8827cysG deletion, Cbl$^{+}$                                                       | [27]      |
| Plasmids       |                                                                                             |           |
| pAR8414        | pKK: CysG cysG cloned into pKK223-3                                                         | [27]      |
| pEB1           | pUC19: CysG$^{A}$ truncated cysG (cysG$^{A}$, codons 202–457) cloned into pUC19 on a BamHI–EcoRI fragment. Overexpresses CysG$^{A}$ | [15]      |
| pAR8834        | pKK: CysG$^{A}$ cysG$^{A}$ from pEB1 cloned into EcoRI site of pKK223-3                        | This study |
| pcysG$^{G21D}$ | pUC19: CysG$^{G21D}$ site directed mutant of pSW95                                            | This study |
| pKKcysG$^{G21D}$ | pKK: CysG$^{G21D}$ mutant cysG from pcysG$^{G21D}$ cloned into EcoRI–HindIII site of pKK223-3 | This study |
| pcysG$^{G224A}$ | pUC19: CysG$^{G224A}$ site directed mutant of pSW95                                          | This study |
| pKKcysG$^{G224A}$ | pKK: CysG$^{G224A}$ mutant cysG from pcysG$^{G224A}$ cloned into EcoRI–HindIII site of pKK223-3 | This study |
| pcysG$^{D227A}$ | pUC19: CysG$^{D227A}$ site directed mutant of pSW95                                          | This study |
| pcysG$^{D248A}$ | pUC19: CysG$^{D248A}$ site directed mutant of pSW95                                          | This study |
| pcysG$^{K270I}$ | pUC19: CysG$^{K270I}$ site directed mutant of pSW95                                          | This study |
| pcysG$^{R298L}$ | pUC19: CysG$^{R298L}$ site directed mutant of pSW95                                          | This study |
| pcysG$^{D303A}$ | pUC19: CysG$^{D303A}$ site directed mutant of pSW95                                          | This study |
| pcysG$^{R309L}$ | pUC19: CysG$^{R309L}$ site directed mutant of pSW95                                          | This study |
| pKKcysG$^{R309L}$ | pKK: CysG$^{R309L}$ EcoRI–BamHI deletion of pKKSW104, removes cysG$^{A}$                    | This study |
| pcysG$^{\Delta NAD}$ | pUC19: CysG$^{\Delta NAD}$ truncated cysG lacking NAD$^{+}$-binding site (residues 1–35), cloned into BamHI–EcoRI of pUC19 | This study |
| pKKcysG$^{\Delta NAD}$ | pKK: CysG$^{\Delta NAD}$ BamHI–EcoRI of pcysG$^{\Delta NAD}$ cloned into pKK223-3       | This study |
| pACYC184       | Carries origin of replication from plasmid p15A                                              | [27]      |
| pSW95          | pUC19: CysG cysG cloned into BamHI–EcoRI site of pUC19                                        | This study |
| pSW97          | pUC18: CysG$^{A}$ BamHI–EcoRI fragment from pEB1, filled and cloned into SmaI site of pUC18. Expresses CysG$^{A}$ | This study |
| pSW98          | pKK: CysG$^{A}$ EcoRI–PstI fragment of pSW97 cloned into pKK223-3                              | This study |
| pSW99          | pUC19: CysG$^{A}$ HindIII fragment of pSW97 cloned into HindIII cut pUC19. Expresses CysG$^{A}$ | This study |
| pSW100         | pACYC184: CysG$^{A}$ SphI–HindIII fragment of pSW98 (containing Ptac and cysG$^{A}$) cloned into pACYC184 | This study |
| pSW101         | pUC19: CysG$^{G224A}$ + CysG$^{A}$ BamHI–EcoRI of pcysG$^{G224A}$ cloned into pSW99              | This study |
| pSW102         | pUC19: CysG$^{D248A}$ + CysG$^{A}$ BamHI–EcoRI of pcysG$^{D248A}$ cloned into pSW99              | This study |
| pSW103         | pUC19: CysG$^{R298L}$ + CysG$^{A}$ BamHI–EcoRI of pcysG$^{R298L}$ cloned into pSW99              | This study |
| pSW104         | pUC19: CysG$^{R309L}$ + CysG$^{A}$ BamHI–EcoRI of pcysG$^{R309L}$ cloned into pSW99              | This study |
| pKKSW104       | pKK: CysG$^{R309L}$ + CysG$^{A}$ EcoRI–HindIII fragment from pSW104, filled and cloned into SmaI cut pKK223-3. Expresses CysG$^{R309L}$ + CysG$^{A}$ | This study |
| pAR8827        | pACYC184: Ptac-cbiA-C-D-E-T-F-G-H-J-K-L-M-N-Q-O-P                                             | [27]      |
| pER119         | pKK: CobA$^{P}$ P. denitrificans cobA sub-cloned from pCR395                                  | This study; [40] |
| pAR8086        | lacI$^{q}$ on pACYC184                                                                      | [27]      |

---

G224A, GGTGCAGCTCCGGGCG; D227A, TCCGGGGCGC  
TGCCGGGGCT; D248A, GGTCTACGCCCCGTCTGGT;  
K270I, CGTCGGGCATACGGCGGGG; R298L, CGTGGT-  
GCTGCTGAAAGG; D303A, GGTGGCGCTCCGTTTA;  
R309L, TTTGGGCCTGGTGGCGGA. The mutants were verified by DNA sequencing and were cloned subsequently into the BamHI/EcoRI sites of pUC19 to create the following plasmids: pCysG$^{G21D}$, pCysG$^{G224A}$, pCysG$^{D227A}$, pCysG$^{D248A}$, pCysG$^{K270I}$, pCysG$^{R298L}$, pCysG$^{D303A}$ and pCysG$^{R309L}$. Transformation of these plasmids into TB1 afforded expression of the mutant protein.

Subcloning of cysG, cysG$^{A}$ and the cysG point mutants were undertaken with standard techniques [22] to generate the list of plasmids described in Table 1. A brief outline of the cloning procedure is also contained within Table 1.

---

### Enzyme purification

The proteins CysG, CysG$^{G21D}$, CysG$^{K270I}$, CysG$^{D303A}$, CysG$^{R309L}$ and CysG$^{A}$ were purified by previously published methods [15]. The remaining enzymes, CysG$^{\Delta NAD}$, CysG$^{G224A}$, CysG$^{D248A}$ and CysG$^{R298L}$ were overexpressed largely in the form of inclusion bodies and therefore initially had to be resolubilized before isolation. This was accomplished as follows: a 3 litre culture of the relevant strain was grown overnight and harvested by centrifugation. The cell pellet (10 g) was resuspended in 50 ml of 50 mM Tris/HCl, pH 7.8, and the resulting suspension was sonicated for four 1 min bursts with 1 min of cooling between each burst in an MSE ultrasonicator (8–10 μm). The solution was centrifuged at 12000 g for 10 min and the pellet was resuspended in ‘inclusion body wash buffer’ [50 mM Tris/HCl (pH 7.8) / 100 mM NaCl/15 mM EDTA/0.5% (v/v) Triton X-100] and then centrifuged at 12000 g for 10 min. The supernatant was discarded and the pellet was resuspended in 6 M urea in 20 mM Tris/HCl, pH 7.8, and incubated at room temperature for 30 min. This was centrifuged at 12000 g for 10 min, by which stage most of the protein was found in solution. This solution was dialysed overnight against 5 litres of 50 mM Tris/HCl, pH 7.8, containing 13 mM 2-mercaptoethanol. Throughout the purification the protein was detected by SDS/PAGE and was assayed with a rapid AdoMet binding assay [25, 26].

---

### Complementation experiments

The E. coli cysteine auxotroph 302Δa was transformed with plasmids containing cysG or the cysG mutants; these were then grown on minimal medium in the presence or absence of cysteine. The ability of the recombinant strains to synthesize sirohaem was measured by their capacity to grow in the absence of exogenous cysteine.

a
200 249 350 399
A.nidulans MTGKVY LVGAGPGDPE YLTLQAQQCL QSAEVLIYDA A.nidulans TDPVLSRGFA VYTVEHP...DALNWEAL AQ.LETLLILL MGSRHLLTIG
Synecocystis sp. RRESNLIFTM LSTDSRAKVY IMGAGLGPVA YLTQRAIAVL GRANVVIYDA Synecocystis sp. TAKDSGAGFF VMDGHDP....HRWPWPAL AQ.LPTLVIL MGTKNLSLLV
A.thaliana LKEKRVKGCK GGDCPGPDVY LVGTGPGDPE LLTLKAVRVI QSADLLLLYDR A.thaliana THRGVATSVR FLTGHSRKGG TDPLFVAENA ADPDTTLVVY MGLGTLPSLA
Zea mays LRASRGRGAE EGG.GPGEVA LVGTGPGDPE LLTLKAVRAI EAADLVLYDR Zea mays THRGVATSVR FLTGHSRNGG TDPLYVAGNA ADPDTTLVVY MGLSTLPSLA
Pa.denitrificans .....GAV RGKAGMGRVD LIGAGPGDPE LLTLRALRLL QQADVVVHDR Pa.denitrificans THRGLATGLR HVTGHRARDA ALDLDWAS.L ADPQTTLAIY MGAANMAEIA
M.ivanovii .............MVVY LVGAGPGDPE LITLKAVNVL KKADVLYDK M.ivanovii THRGVATSFT VVTHGEDPTK CCKQVGW.DF..KADTIVIL MGIGNLAENT
M.jannaschii ...MPKNINI SGEIMTGKVI LVGAGPGDBE LITIKGLKAI KEADVVVYDD M.jannaschii THRKVATSFT VVTGHEAEDK KEKQVDLSKL ..NADTIVIL MGITNLENLV
B.subtilis ...........MIMKNGIVY FVGAGPGDPG LLLTKGKQAL KEADVILYDR B.subtilis THRDFASSFA MITAHDKSLK GTPNLDWEGI ARSVQTLVFY MGVKNLSYIC
B.megaterium ...........MGKVY LVGAGPGDPD LITLKGLKAI QQADVILYDR B.megaterium THREYSASFA FVAGHRKDSK HDA.IKWDIS AKGVDTLAIY MGVRNLPYIC
B.stearothermophilus ....MTIGKVY LVGAGPGDEK LITVYGRECL EPADVIYYDR B.stearothermophilus PRWRWSAGIG V........QRIDWEAL TRGCDTILVV MGAANLPDIC
E.coli ITETTEQLIN EPLDHRGEVV LVGAGPGDAG LLTLKGLQQI QQADVVVYDR E.coli THRDAYQSVR LITGH...LK TGGELDWENL AAEKQTILVFY MGLNQAATIQ
P.freudenreichii .......M TTLLLPGTVT LVGAGPGDPE LVTVAGLRAV QQAEVILYDR P.freudenreichii THRHLVQGFT VVSGHVSPSD ERSEVPWRQL AKDRLTLVIL MGVAHMRDIA
P.denitrificans ..MIDDLFAG LPALEKGSVW LVGAGPGDPG LLLTHAANAL RQADVIVHDA P.denitrificans THEREVNHAVT FLTGHDSSSL VPDRINWQGI ASGSPVIVMY MAMKHIGAIT
Consensus -----------------G-G-G---- --T-------- ----------D-
* * *

250 299 400 449
A.nidulans LIDPRILDLV PANCDRIAVG KRGAESTPQ ATINQLLVEH CQQGRKVIRL A.nidulans HELIRHG.RS PDSPVALIQW AGHPQQTVLE SSLSRMAQQW GDQPL.SPCV
Synecocystis sp. LVSQELFDLL PPDCERIFVG KRGQPSTPQ AKINQLLVDH YRRGKQVVRL Synecocystis sp. NELVRAG.KS PHTPMAVIKN AGRPEQQTWE GTLADMVEKT QQSL.APAV
A.thaliana LVSNDVLELV APDARLLYVG KTAGYHSRTQ EEIHELLNNF AEAGATVVRL A.thaliana QKLMDHG.LP SDTPAVER GTTPLQRTVF AELKDFATEI QSAGLVSPTL
Zea mays LVSNDVLDLV GEGARLLYVG KTAGYHSRTQ EEIHELLSSF AEAGANVVRL Zea mays PKLMKHG.LP PDTPAVER GTTPQQRTVF ASLKDLVDEV KSADLVSPTL
Pa.denitrificans LVSDEVMACI PAHVRRIPVG KAAGFHPVPQ EQINALLVEL GLSCLTVARL Pa.denitrificans RELIRHG.MP ADLPVLAVSQ ASTPQEQR LH ATLQDIAAAL ARKPLPAPVL
M.ivanovii PANEEILKYA .EGAKLIYVG KQAGHHYKSQ NEINTLLVEE AKENDLVVRL M.ivanovii AEIMKH..KD PETPVCVIEN GTMEGQR IIT GTLENITAGKD IK....PPAL
M.jannaschii LISKELLNYA KKDAELIYVG KRKGKHSFKQ EEINKILVEK AEGKLVVVRL M.jannaschii KELLQNPKRS KETPVAIIME GTTKNQRVIK GTLGDIVEKA KKENARPPGV
B.subtilis LANPKILLEFA SPDCQFIYCG KLPNRHFMKQ KEINALLVEK ALNGLTVVRL B.subtilis QQLISYG.KS PSVPVIVIQW GTWGRQRSVK GTLENIQQKV QEHQITNPAl
B.megaterium LVNKDLLEYA KSDADIICYG KLPNYHTLKQ ETINNFLVKF AKKGKIVTRL B.megaterium QQLMKHG.KT SATPALIHW GTCADQRTVT GTLGTVDIV KEEQIENPSM
B.stearothermophilus LINRKLLRYA KPGAELVYCG KEPGKHDTVQ EQIHELLVWH AQOQKTVVRL B.stearothermophilus GRLMAAG.KP SDTPAAIIEW GTTERQRTVA ATLSTLSDEA EQAGISHPAI
E.coli LVSDDIMNLV RRADRVFVG KRAGYHCVPQ EEINQILLRE AQQGKRVVRL E.coli OKLIEHG.MP GEMPVAIVEN GTA VTQRVID GTLTQIGELA QQ..MNSPSL
P.freudenreichii LAPQDLLSEA SDDAELVPVG KIPRGHYVPQ EEINQLLVAH AREGRKVVRL P.freudenreichii PELMA.GGLP ADTPVRVVSN ASLASQESWR TTLGDVADAM DAHHVRPPAL
P.denitrificans LVNEDCLKLA RPGAVLEFAG KRGGKPSPKQ RDISLRLLVEL ARAGRNVLR Consensus ANLIA.GGRS PDEPVAFVCN AATPQQAVLE TTLARAEADV AAAGLEPPAI
Consensus ------------------G K---------Q --------L--- -------V-RL
* * 

450
A.nidulans IVIGEVV.
Synecocystis sp. IVIGDVVS
A.thaliana IIIGKVVE
Zea mays IIIGKVVS
Pa.denitrificans FIVGHVAA
M.ivanovii VVLEMLSM
M.jannaschii IVIGEVVN
B.subtilis IVIGDIVN
B.megaterium IIVGEVNN
B.stearothermophilus IIIGDVVR
E.coli IIIGRVVG
P.freudenreichii VVVGTLAG
P.denitrificans VVVGEVVR

300 349
A.nidulans KSGDPFVFR AASELDALER SGCDYAVLPG LSTALAAPLL A.....GIPI
Synecocystis sp. KSGDPWIFGR IMPELATLVT HHCAYEVVPG ISSAIAGAGL A.....SIPL
A.thaliana KGGDPLVFR GGEEMDFLQQ QGIRVQVIPG ITAASGIAAE L.....GIPL
Zea mays KGGDPLVFR GGEEMDFLQQ QGIRVQVIPG ITAASGIAAE L.....GIPL
Pa.denitrificans KGGDPTIFGR GGEEFEA VTR AGIPCDYVPG ITAAQGAAVS A.....RFPL
M.ivanovii KGGDPFVFR GGEEILALVE EGIDFELVPG VTSAIGVPTT I.....GLPV
M.jannaschii KGGDPFVFR GGEEILELKK HNIPYEVIPEG ITSIAIVPEV A.....GIPv
B.subtilis KGGDPSVFR VGEEADALHE HGIRYEMVPG ITSGIAAPLY A.....GIPv
B.megaterium KGGDPFVFR GGEEEAELVQ QGISFEIVPG ITSGIAAAAAY A.....GIPv
B.stearothermophilus KGGDPCVFR VGEEAEVLAK AGIPFEIVPG VTSGIAVPAY AFRSPTVITP
E.coli KGGDPFIFGR GGEELETLCN AGIPFSVVPG ITAASGCSAY S.....GIPL
P.freudenreichii KGGDSFVFR GGEEWQACAE AGIPVRVIPG VSSATAGPAL A.....GIPL
P.denitrificans KGGDPFVFR GGEEALT LVE HQVPFRIVPG ITAGIGGLAY A.....GIPv
Consensus K-GDP--FGR ---E------- -------------PG

b
1 21 50
E.coli VDHLPIFCQL RDRDCLIVGG GDVAEPKARL LLDAGARLTV NALAFIPQF
S.typhimurium VDHLPIFCQL RDRDCLIVGG GDVAERKARL LLEAGARLTV NALTFIPQF
V.anguillarium MQYFPLFLNL LGKPVLVVGG GEVACRKVEA LIRAGAQVTV VSPKIERYL
C.josuii .GRFPLFIDL SGKKVLVVGG GEIATRRVKT LLRFGADIYL VAPHLTSEL
Consensus ----P-F---L------L-VGG G--A------ L---GA------

51 100
E.coli AWADAGMLTL VEGPFDESLL DTCWLAI.AA TDDDALNQRV RQAAEARRI
S.typhimurium VWANE GMLTL VEGPFDETLL DSCWLAI.AA TDDDTV NQRV SDAAESRRI
V.anguillarium ELVEQNRCQW VQNFYSSELL TKEYWQWWAT TDNP SLNHQV HKDARKTGV
C.josuii EMLNCKLINY REGYYESQDI QNMFLVI.AA TNDRET NHKV YLDAKEKG
Consensus ---------------------A-T------ ---A-------

101 150
E.coli CNVVDAPKAA SFIMPSIIDR SPLMVA.VSS GGTS PVLARL LREKLESSLL
S.typhimurium CNVVDAPKAA SFIMPSIIDR SPLMVA.VSS GGTS PVLARL LREKLESSLL
V.anguillarium VN VDDQPYC DFITPSMVNR GRIQIA.ISS GGASPVLVRN IRETLEAVL
C.josuii MSIADCREEC SYFP AIFEF DGIVGGLVSQ NGDNHSLVKS VAEQI....
Consensus ----D----- -F---P----- ---------S- -G-----L--- --E-------

151 200
E.coli LHLGQVAKYA GQLRGRVKQQ FATMGERRRF WEKLFVNDRL AQSLANNDQ
S.typhimurium QHLGQVARYA GQLRARVKQQ FATMGERRRF WEKFFVNDRL AQSLANADE
V.anguillarium QNLSLLADFA ASKRNSIKQS LNSVSARRKF WEAFLSDSKT ENVVSREE.
C.josuii RKIGQATD*. ..................
Consensus -----------------------------

201 250
E.coli AITETTEQLI NEPLDHRGEV VLVGAGPGDA GLLLTKGLQQ IQQADVVYY
S.typhimurium AVNATTERLF SEPLDHRGEV VLVGAGPGDA GLLLTKGLQQ IQQADIVYY
V.anguillarium .LERLYQQAL LVPIEEESAQC TWISCG.DDV EMLPIKALRY MQRAEVVFY
C.josuii ...
Consensus -----------------------------

Figure 1 Sequence alignment of a number of proteins showing similarity to either the uro'gen III transmethylase domain of E. coli CysG (CysG^A domain; residues 200–457) or to the dehydrogenase and ferrochelatase domain (CysG^B domain; residues 1–200)

The alignment was generated using the PILEUP program from the UWGCG package [41]. (a) Alignment of the predicted amino acid sequences of the uro'gen III transmethylase sequences (CysG^A) from Anacystis nidulans [42], Synechocystis sp. (accession number ss110378), Arabidopsis thaliana [35], Zea mays [43], Paracoccus denitrificans [44], Methanobacterium ivanovii [45], Methanococcus jannaschii [46], Bacillus subtilis [18], Bacillus megaterium [12], Bacillus stearothermophilus [47], Escherichia coli [13], Propion freudenreichii [48] and Pseudomonas denitrificans [10]. The consensus sequence is shown and the mutated residues are marked by an asterisk. (b) Alignment of the predicted amino acid sequences of the dehydrogenase/ferrochelatase domains (CysG^B) from Escherichia coli [13], Salmonella typhimurium [19], Vibrio anguillarum [49] and Clostridium josuii [16]. The consensus sequence is shown and the predicted NAD⁺-binding site is underlined. The mutated residue Gly-21 is marked by an asterisk.

AdoMet binding assay and fluorography

The AdoMet binding assays were performed as described previously [25, 26] by analysis of the profile of a \([methyl-3H]\)-AdoMet/protein mixture after chromatography on a Sephadex G-25 column (1 cm × 20 cm). Briefly, protein (1 mg) was preincubated with \(0.5 \mu Ci\) of \([methyl-3H]\) AdoMet (specific radioactivity 80 Ci/mmol) in a final volume of \(200 \mu l\). The mixture was applied to the Sephadex G-25 column; 1 ml fractions were collected. AdoMet binding was identified by the presence of a significant quantity of radioactivity in the protein-containing fractions. Fluorograms were taken of SDS/PAGE gels containing 10–20 \(\mu g\) of protein that had been premixed with the labelled AdoMet as described previously [26].

Quantification of cobyrinic acid synthesis by bioassay

Cobyrinic acid levels were determined by the modification of the agar plate bioassay system as described by Raux et al. [27]. The indicator strain used for this assay was AR3612, *S. typhimurium cysG metE*. Exogenously added cobyrinic acid promoted growth of the strain on minimal medium plates supplemented with cysteine. The growth dependence of the strain allowed a standard curve to be constructed.

Preparation of bacteria for estimation of cobyrinic acid synthesis

The levels of cobyrinic acid in the strains of interest were estimated in extracts of the bacteria after they had been prepared as follows: bacteria were grown under anaerobic conditions in minimal medium [27] supplemented with antibiotics. After harvesting of the cells by centrifugation, the cells were resuspended in water to a \(D_{600}\) of 8, then sonicated. A \(10 \mu l\) droplet of the extract was then bioassayed.

RESULTS

Construction of CysG AdoMet-binding site variants and conserved amino acids found within CysG<sup>A</sup>

The AdoMet-binding site of the transmethylase domain (CysG<sup>A</sup>) was subjected to mutagenesis by changing residues 224 (Gly) and 227 (Asp) to Ala, which are found in the putative AdoMet-binding motif GXGXGD of the uro'gen III methyltransferases (Figure 1a) [25, 28, 29]. The mutations were performed with the techniques described in the Materials and methods section and were verified by sequencing. The mutated genes were cloned into both pUC19 and pKK223-3 vectors (Table 1). The mutant proteins expressed from these plasmids were analysed initially for their ability to complement the *E. coli cysG* deletion mutant 302Δa [15, 30] and to bind AdoMet [25, 26].

Other conserved charged amino acids in the CysG<sup>A</sup> domain include D248, K270, R298, D303 and R309, which are located in regions of the primary structure that are thought to be involved in binding and catalysis [25, 29, 31] (Figure 1a). The residues were mutated with the techniques described in the Materials and methods section and were verified and analysed as described above.

Construction of NAD<sup>+</sup>-binding site variants within CysG<sup>B</sup>

The contribution of the CysG<sup>B</sup> domain to sirohaem synthase activity was investigated by altering the NAD<sup>+</sup>-binding site. Initially this was performed by making a truncated version of *cysG*, lacking the first 34 codons from the 5′ end of the gene. The truncated gene, *cysG<sup>ΔNAD</sup>*, was cloned into both pUC19 and

**Figure 2** Rapid AdoMet binding assay of CysG and a number of mutant variants

The purified protein was incubated with \(0.5 \mu Ci\) of \([methyl-3H]\) AdoMet and passed down a small gel-filtration column. The protein was eluted in fractions 4–6, whereas the smaller molecules were eluted in fractions 9–14. If the protein was co-eluted with some of the radioactivity it was deemed to be capable of binding AdoMet. Control experiments were performed with BSA. (a) The AdoMet binding profile of CysG and BSA in comparison with those of CysG<sup>A</sup>, CysG<sup>G21D</sup> and CysG<sup>ΔNAD</sup>. The graphs indicate that all the CysG variants with altered CysG<sup>B</sup> domains are able to bind AdoMet. (b) The AdoMet binding profile of CysG in comparison with those of CysG<sup>G224A</sup>, CysG<sup>D248A</sup>, CysG<sup>K270I</sup>, CysG<sup>R298L</sup>, CysG<sup>D303A</sup> and CysG<sup>R309L</sup>. The graphs suggest that the CysG<sup>G224A</sup> and CysG<sup>R298L</sup> variants are unable to bind AdoMet.

pKK223-3 to yield pcysG<sup>ΔNAD</sup> and pKKcysG<sup>ΔNAD</sup> respectively. When these plasmids were transformed into *E. coli* TB1 the recombinant strains were found to express a shortened protein (45 kDa as opposed to 50 kDa [11]), which was clearly seen when crude cell extracts were analysed by SDS/PAGE (results not shown). Furthermore the cells that expressed CysG<sup>ΔNAD</sup> were very fluorescent when viewed under UV light, a phenotypic characteristic shared with cells overexpressing CysG<sup>A</sup> [15]. The fluorescence is due to the accumulation of sirohydrochlorin and trimethylpyrrocorphin within the cell [15, 32].

As well as the NAD<sup>+</sup>-binding site-deficient protein, CysG<sup>ΔNAD</sup>, another NAD<sup>+</sup>-binding site variant was generated through the introduction of a point mutation at Gly-21 (Gly-21 → Asp). Glycine-21 makes up part of the GXGXXG nucleotide-binding site of CysG [29, 33] (Figure 1b) and the inclusion of a charged polar amino acid in a key part of the putative Rossman fold [34] was envisaged to disrupt the structure and prevent the binding of NAD<sup>+</sup>. When *E. coli* TB1 was transformed with pcysG<sup>G21D</sup>, the cells also exhibited a strong fluorescence under UV light. Thus all CysG derivatives with a missing or altered NAD<sup>+</sup>-binding site

Table 2 Complementation tests of the E. coli cysG deletion strain 302Δa

CobA<sup>pd</sup> is uro'gen III transmethylase from P. denitrificans; Cbi-ACDETFGHJKLMNQOP is the S. typhimurium CobI genes that are required to convert precorrin-2 into cobyrinic acid. In the complementation column, Y signifies growth, and N signifies no growth.

| Plasmid used to transform 302Δa (E. coli cysG) | Complementation | Properties and observations |
| --- | --- | --- |
| pKK223-3 | N | — |
| pAR8414 (CysG<sup>A</sup>) | Y | Wild-type protein has transmethylase, dehydrogenase and chelatase functions |
| pAR8834 (CysG<sup>A</sup>) | N | Has transmethylase activity only |
| pcysG<sup>G21D</sup> (CysG<sup>G21D</sup>) | Y | Has transmethylase and ferrochelatase activity but greatly reduced dehydrogenase activity |
| pKKcysG<sup>G21D</sup> (CysG<sup>G21D</sup>) | Y | Has transmethylase and ferrochelatase activity but greatly reduced dehydrogenase activity |
| pcysG<sup>G224A</sup> (CysG<sup>G224A</sup>) | N | No transmethylase activity. Has dehydrogenase and chelatase activity |
| pKKcysG<sup>G224A</sup> (CysG<sup>G224A</sup>) | N | No transmethylase activity. Has dehydrogenase and chelatase activity |
| pcysG<sup>D227A</sup> (CysG<sup>D227A</sup>) | Y | Has all activities of CysG |
| pcysG<sup>D248A</sup> (CysG<sup>D248A</sup>) | N | No transmethylase activity. Has dehydrogenase and chelatase activity |
| pcysG<sup>K270L</sup> (CysG<sup>K270L</sup>) | Y | Has all activities of CysG |
| pcysG<sup>R298L</sup> (CysG<sup>R298L</sup>) | N | No transmethylase activity. Has dehydrogenase and chelatase activity |
| pcysG<sup>D303A</sup> (CysG<sup>D303A</sup>) | Y | Has all activities of CysG |
| pcysG<sup>R309L</sup> (CysG<sup>R309L</sup>) | N | No transmethylase activity. Has dehydrogenase and chelatase activity |
| pKKcysG<sup>R309L</sup> (CysG<sup>R309L</sup>) | N | No transmethylase activity. Has dehydrogenase and chelatase activity |
| pcysG<sup>ΔNAD</sup> (CysG<sup>ΔNAD</sup>) | N | Has transmethylase activity but no dehydrogenase or chelatase activity |
| pKKcysG<sup>ΔNAD</sup> (CysG<sup>ΔNAD</sup>) | N | Has transmethylase activity but no dehydrogenase or chelatase activity |
| pSW100 (CysG<sup>A</sup>) | N | Has transmethylase activity only |
| pSW101 (CysG<sup>G224A</sup>+CysG<sup>A</sup>) | Y | — |
| pSW102 (CysG<sup>D248A</sup>+CysG<sup>A</sup>) | Y | — |
| pSW103 (CysG<sup>R298L</sup>+CysG<sup>A</sup>) | Y | — |
| pSW104 (CysG<sup>R309L</sup>+CysG<sup>A</sup>) | Y | — |
| pKKSW104 (CysG<sup>R309L</sup>+CysG<sup>A</sup>) | Y | — |
| pER119 (CobA<sup>pd</sup>) | N | Has transmethylase activity only |
| pcysG<sup>G224A</sup>+pSW100 (CysG<sup>G224A</sup>+CysG<sup>A</sup>) | Y | — |
| pcysG<sup>D248A</sup>+pSW100 (CysG<sup>G224A</sup>+CysG<sup>A</sup>) | Y | — |
| pcysG<sup>R298L</sup>+pSW100 (CysG<sup>R298L</sup>+CysG<sup>A</sup>) | Y | — |
| pcysG<sup>R309L</sup>+pSW100 (CysG<sup>R309L</sup>+CysG<sup>A</sup>) | Y | — |
| pAR8827 (Cbi-ACDETFGHJKLMNQOP) | N | Strain does not grow in presence or absence of exogenous cobalt (4.2 μM) |
| pAR8827+pAR8834 (Cbi-ACDETFGHJKLMNQOP+CysG<sup>A</sup>) | Y | Complementation is inhibited in the presence of exogenous cobalt (4.2 μM) |
| pAR8827+pER119 (Cbi-ACDETFGHJKLMNQOP+CobA<sup>pd</sup>) | Y | Complementation is inhibited in the presence of exogenous cobalt (4.2 μM) |
| pAR8827+pAR8414 (Cbi-ACDETFGHJKLMNQOP+CysG) | Y | Strain is able to grow in presence or absence of cobalt |

seem to have this illuminating characteristic. As for the mutations in the CysG<sup>A</sup> domain, the alterations to the CysG<sup>B</sup> domain were verified by DNA sequencing, and the effect of the mutations on cobalamin biosynthesis and sirohaem synthesis was investigated.

### AdoMet-binding studies on CysG variants

To assess whether the CysG variants described above were able to form a tightly bound complex with AdoMet as described previously [26], the purified proteins were analysed for their ability to bind AdoMet during gel filtration. In this case the assay was used as a simple qualitative test to determine the presence of an AdoMet-binding protein. The result of the binding assay with CysG<sup>A</sup> is shown in Figure 2a. The rapid binding assay demonstrated that a large amount of the label was co-eluted with the CysG<sup>A</sup> fraction, whereas in control experiments with BSA the label was eluted exclusively in the small molecules fractions. Further experiments clearly demonstrated that along with CysG<sup>A</sup>, CysG<sup>G21D</sup>, CysG<sup>ΔNAD</sup>, CysG<sup>D248A</sup>, CysG<sup>K270L</sup>, CysG<sup>D303A</sup> and CysG<sup>R309L</sup> were also able to bind AdoMet (Figures 2a and 2b). In contrast, CysG<sup>G224A</sup> and CysG<sup>R298L</sup> were unable to bind AdoMet, implying that these proteins might not be able to perform the transmethylation reaction (Figures 2a and 2b). The AdoMet-binding capacity of the modified versions of CysG was confirmed by fluorography. This was achieved by incubation of the enzyme with [methyl-<sup>3</sup>H]AdoMet, as described in the the Materials and methods section, and subsequently subjecting the protein sample to SDS/PAGE. After the gel had been dried and placed next to X-ray film, the fluorogram was developed to reveal that CysG<sup>A</sup>, CysG<sup>G21D</sup>, CysG<sup>ΔNAD</sup>, CysG<sup>D303A</sup> and CysG<sup>R309L</sup> were all radioactive (results not shown).

#### Functional complementation of the E. coli cysG deletion strain 302Δa

In earlier studies it had been shown that the E. coli cysteine auxotrophic strain 302Δa was deficient in all sirohaem synthase activity and could not be complemented by CysG<sup>A</sup> [15]. Similarly when the strain was transformed with a plasmid carrying the cobA gene of P. denitrificans (pER119) (Table 2) complementation was not observed. We also investigated the effect of the mutant cysG variants as well as other modified cysG constructs on their ability to complement the cysteine auxotrophy of the E. coli cysG deletion strain 302Δa; the results of these tests are shown in Table 2. Point mutations in CysG at positions 227 (D→A), 270 (K→I) and 303 (D→A) did not prevent complementation when plasmids harbouring these mutant CysG derivatives were transformed into the E. coli cysG deletion strain 302Δa. However, point mutations in CysG at positions 224 (G→A), 248 (D→A), 298 (R→L) and 309 (R→L) gave proteins that were unable to complement the auxotrophic mutant (Table 2).

The effect of the mutation at position 21 (G→D) and the removal of the NAD<sup>+</sup>-binding site in the CysG<sup>B</sup> domain were also assessed for their effect on sirohaem synthesis. Complementation of 302Δa by the plasmid pCysG<sup>G21D</sup> was observed even though this plasmid overexpresses a protein with greatly decreased dehydrogenase activity (less than 1% of wild-type

strain which produced cobyric acid at a level of 190 pmol per attenuation unit (Table 3).

Rather surprisingly, AR8906 (*E. coli* *cysG* deletion, CobI⁺) transformed with pcysGᴬ was found to be able to grow on minimal medium in the absence of cysteine, albeit quite slowly (Table 2). When exogenous cobalt was added to the minimal medium plates, complementation was prevented.

## DISCUSSION

### Rationale for the construction of CysG variants

Previously it had been shown that it is possible to express the CysGᴬ domain of sirohaem synthase as a truncated protein by dissecting *cysG* into separate *cysGᴬ* and *cysGᴮ* entities [15]. However, attempts to express the N-terminal domain, CysGᴮ, proved to be unsuccessful. To assess the roles of the two functional domains CysGᴬ and CysGᴮ, a number of mutations and a truncation of CysG were made. Variations to CysG were made in the putative AdoMet-binding site of CysGᴬ (G224A and D227A) (Figure 1a), a number of highly conserved residues in CysGᴬ (D248A, K270L, D303A, R298L and R309L) (Figure 1a) and the NAD⁺-binding site of CysGᴮ (G21D, and a NAD⁺-binding site-deleted enzyme, CysGᴬᴺᴬᴰ) (Figure 1b). The replacement residues were selected for their lack of functionality, with consideration given to spatial and chemical properties, and also to the ease with which it was possible to change the codon.

### Effect of mutation on AdoMet binding

The transmethylation reaction catalysed by CysG seems to proceed via an enzyme–AdoMet covalent complex [26], although the precise nature of the interaction between the enzyme and substrate has not been determined. All the mutant proteins were tested for their ability to bind AdoMet. Alterations to the CysGᴮ domain had no effect on AdoMet binding: CysGᴬ, CysGᴳ²¹ᴰ and CysGᴬᴺᴬᴰ all associated with AdoMet on gel filtration. However, as expected, mutations within the transmethylation functional domain (CysGᴬ) had a more pronounced effect on AdoMet binding. The mutant CysGᴳ²²⁴ᴬ was unable to bind AdoMet significantly. Glycine-224 makes up part of the glycine-rich motif, which is thought to constitute a Rossman-type fold associated with binding both AdoMet and dinucleotide cofactors. The mutation of Arg-298 to Leu also prevented AdoMet binding and thus Arg-298 must presumably have a role in AdoMet recognition. The mutants, CysGᴰ²⁴⁸ᴬ and CysGᴿ³⁰⁹ᴸ, did not seem to affect AdoMet binding. However, from the results presented in Figure 2(b) it is apparent that CysGᴷ²⁷⁰ᴸ and CysGᴰ³⁰³ᴬ bind less AdoMet than do CysG and some of the other variants. It is not possible from this result to determine whether this is due to a lower affinity for the substrate, whether the mutant enzyme has more bound unlabelled AdoMet, or whether, proportionally, the enzyme preparations from which these mutant proteins were obtained contain less active enzyme. This is due to the lack of a suitable active-site titrant for CysG.

### Effect of mutations on complementation of the *E. coli cysG* deletion strain 302Δa

The *E. coli cysG* deletion strain 302Δa is deficient in all CysG functions, that is the strain has no measurable transmethylation, dehydrogenase or ferrochelatase activity. In general, for the transmethylation mutants, the complementation of this cysteine auxotrophic strain followed the results observed in the AdoMet binding assay. Thus the mutants that did not bind AdoMet,

**Table 3 Amount of cobyric acid synthesized in *E. coli* strain AR8906 after transformation with a range of plasmids**

| Plasmid used to transform AR8906 (302Δa/pAR8827; *E. coli cysG*, CobI⁺) | Cobyric acid synthesis (pmol per $D_{600}$ unit) |
|---------------------------------------------------------------------|--------------------------------------------------|
| pKK223-3                                                          | 0                                                |
| pAR8414 (CysG)                                                    | 580                                              |
| pAR8834 (CysGᴬ)                                                   | 12                                               |
| pER119 (CobAᴾᵈ)                                                   | 500                                              |
| pKKSW104 (CysGᴬ + CysGᴿ³⁰⁹ᴸ)                                      | 506                                              |
| pKKcysGᴿ³⁰⁹ᴸ (CysGᴿ³⁰⁹ᴸ)                                          | 0                                                |
| pKKcysGᴳ²¹ᴰ (CysGᴳ²¹ᴰ)                                            | 190                                              |

CysG; results not shown). However, complementation of the *E. coli cysG* deletion strain 302Δa was not observed by the plasmid carrying the NAD⁺-deleted protein, pcysGᴬᴺᴬᴰ (Table 2).

To investigate whether the CysGᴬ and CysGᴮ regions could function independently, dual plasmid complementation studies were undertaken. The *E. coli cysG* deletion strain 302Δa was transformed with one of the mutant *cysG* derivatives (pcysGᴳ²²⁴ᴬ, pcysGᴰ²⁴⁸ᴬ, pcysGᴿ²⁹⁸ᴸ or pcysGᴿ³⁰⁹ᴸ) harboured on pUC19, together with pSW100, which contains the truncated gene for the functional transmethylase domain of CysG, *cysGᴬ*, on a pACYC184-derived plasmid. In each case, complementation of the auxotrophic strain was attained (Table 2). Identical results were obtained when the experiment was repeated with the mutated *cysG* genes cloned on the same plasmid with *cysGᴬ*. Thus plasmids pSW101, pSW102, pSW103 and pSW104 all complemented the *E. coli cysG* deletion strain 302Δa (Table 2).

### Cobalamin biosynthesis

To find a definitive answer to the question of whether all or part of CysG is required for cobalamin biosynthesis, the *E. coli cysG* deletion mutant 302Δa was transformed with pAR8827, a plasmid that contains all the *S. typhimurium* genes required for the transformation of precorrin-2 into cobyric acid, and was designated AR8906. Levels of cobyric acid, a late corrin intermediate, can be accurately determined by using a very sensitive bioassay system [27]; this method was used to determine the levels of cobyric acid synthesized in this study. When grown anaerobically in minimal medium in the presence of added cobalt, 5-aminolaevulinic acid, cysteine and isopropyl β-D-thiogalactoside, AR8906 was unable to synthesize any cobyric acid (Table 3). However, when AR8906 (302Δa/pAR8827; *cysG* deletion, CobI⁺) was further transformed with pAR8414, a plasmid that harbours the *E. coli cysG*, cobyric acid was produced to levels of 580 pmol of cobyric acid per attenuation unit of bacteria (Table 3). These results indicate that the *E. coli cysG* is required for corrin biosynthesis and that its absence, as expected, prevents corrin production because the bacterium is unable to transform uro'gen III into precorrin-2.

Strain AR8906 (302Δa/pAR8827) was further transformed with plasmids containing *cysGᴬ* (pAR8834), *cysGᴳ²¹ᴰ* (pKKcysGᴳ²¹ᴰ), *cysGᴿ³⁰⁹ᴸ* (pKKcysGᴿ³⁰⁹ᴸ) or *cobAᴾᴰ* (pER119). The resulting strains were examined for their ability to grow on minimal medium and to produce cobyric acid (Table 3). When AR8906 was transformed with a plasmid containing *cysGᴬ* it produced approx. 12 pmol of cobyric acid per attenuation unit (Table 3). When the *P. denitrificans cobA* gene (pER119) was transformed into AR8906 (*E. coli cysG* deletion, CobI⁺) cobyric acid was produced to a level of 500 pmol per attenuation unit (Table 3). Transformation of AR8906 with pcysGᴳ²¹ᴰ gave a
CysG<sup>G224A</sup> and CysG<sup>R298L</sup>, did not complement. Of particular interest from this study are mutants CysG<sup>R309L</sup> and CysG<sup>D248A</sup>, which are able to bind AdoMet but do not complement the *E. coli* *cysG* deletion strain. These mutations must be specific for the transmethylation reaction because they did not affect the ferrochelation or dehydrogenation ability of the protein, a conclusion that is deduced from the dual-plasmid complementation experiments with *cysG*<sup>A</sup>. Presumably the mutations at positions 248 and 309 either prevent the binding of the other substrate, uro'gen III, or inhibit the transmethylation reaction.

All the other transmethylase domain mutants (D227A, K270I and D303A) were able to complement the cysteine auxotrophic strain. This indicates that these residues are not essential in the catalytic process and that the change did not lead to a detrimental loss of structure.

The effects of the mutations and alterations within CysG<sup>B</sup> on the ability to complement the *E. coli* *cysG* deletion strain were varied. As reported before, complete loss of the CysG<sup>B</sup> domain does not allow complementation [15]. However, a point mutation within the dinucleotide-binding fold, G21D, does allow complementation even though the mutant protein has a barely measurable dehydrogenase activity. As CysG<sup>G21D</sup> is able to make precorrin-2, the complementation can be explained on the basis that a certain amount of spontaneous oxidation of precorrin-2 occurs within the cell, producing sirohydrochlorin [15]. This in turn is ferrochelated by the CysG<sup>G21D</sup> variant by the insertion of a ferrous ion into the chemically oxidized precorrin-2.

The modified enzyme lacking the NAD<sup>+</sup>-binding site, CysG<sup>ΔNAD</sup>, was unable to complement the *E. coli* *cysG* deletion strain. This suggests that some of the first 34 amino acid residues are probably important for the correct folding of the ferrochelation active site and that the deletion of the N-terminal arm prevents the proper folding of the CysG<sup>B</sup> domain. Alternatively, the N-terminal arm, which probably adopts a β–α–β nucleotide-binding fold, might also harness some critical chelatase functional residues.

### CysG functional domains (CysG<sup>A</sup> and CysG<sup>B</sup>) seem to operate independently

As alluded to earlier, complementation of the *E. coli* *cysG* strain can be attained by expression of a transmethylase-functional protein (either CysG<sup>A</sup> or CobA) and a transmethylase-deficient CysG (e.g. CysG<sup>G224A</sup>, CysG<sup>D248A</sup>, CysG<sup>R298L</sup> or CysG<sup>R309L</sup>). Such results suggest that the intermediate precorrin-2 is diffusible between CysG<sup>A</sup> and the transmethylase-deficient CysG mutants. Thus CysG<sup>A</sup> synthesizes precorrin-2, which is transformed into sirohaem by the CysG<sup>B</sup> functional domain of the mutant CysG. This is an important point as it could be argued that the reason for the CysG<sup>BA</sup> architecture is to facilitate substrate channelling and that direct metabolite transfer within the multifunctional enzyme might have evolved because of the chemical lability of precorrin-2. The results described here show that the biosynthesis of sirohaem can be accomplished with a two-enzyme system. Two-enzyme systems for sirohaem biosynthesis have been inferred from previous work on *Bacillus megaterium* [12] and in the higher plant *Arabidopsis thaliana* [35]. Recently sirohaem synthesis from uro'gen III in *S. cerevisiae* has been shown to require two proteins, Met1 and Met8 [17], although the functions of the two proteins have not been accurately ascribed.

#### CysG and cobalamin biosynthesis

Although *E. coli* does not possess the genes that encode for the enzymes required to convert precorrin-2 into cobinamide (*CobI*) genes), it is able to convert cobinamide into both adenosylcobalamin and methylcobalamin, i.e. it contains the *CobII* and *CobIII* genes [36]. However, the closely related enteric species *Salmonella typhimurium* possesses all the genes for cobalamin biosynthesis *de novo*, although cobalamin synthesis is attained only under anaerobic conditions [37]. It has previously been shown that the *S. typhimurium cysG*, which is more than 90% identical with the *E. coli cysG*, is required for cobalamin biosynthesis [37]. Nevertheless, it is not known whether the whole CysG protein, both the CysG<sup>B</sup> and the CysG<sup>A</sup> domains, is necessary for cobalamin biosynthesis or whether merely the CysG<sup>A</sup> (transmethylase) domain is required.

#### Only the transmethylase function of CysG (CysG<sup>A</sup>) is required for promotion of cobalamin biosynthesis

When the *E. coli cysG* deletion strain 302Δa is transformed with a plasmid carrying the *S. typhimurium* CobI operon (pAR8827), producing AR8906, the strain is unable to synthesize cobyric acid because it does not contain a functional *cysG*. When AR8906 (*E. coli cysG*, CobI<sup>+</sup>) is transformed with a compatible plasmid harbouring *cysG*<sup>A</sup>, cobyric acid is synthesized, albeit at 2% of the level observed when the strain is transformed with a plasmid carrying the complete *cysG*. The fact that CysG<sup>A</sup> is able to make any corrin suggests that CysG<sup>B</sup> is not essential for cobalamin biosynthesis and it seems that the biosynthesis of precorrin-2 is a prerequisite for cobalamin synthesis. However, corrin production in AR8906 transformed with a plasmid carrying *cysG*<sup>A</sup> is only 2% of that measured in AR8906 transformed with *cysG* (Table 3). There are two possible explanations for this low level of synthesis. First, CysG<sup>A</sup> is known to produce a large quantity of the overmethylated compound, a 2,7,12-trimethylpyrrocorphin, which has no known physiological function [8]. By making such large quantities of this compound the enzyme is lowering the precorrin-2 concentration, effectively removing the substrate from the cobalamin biosynthetic pathway. Secondly, if CysG<sup>B</sup> is required for the facilitation of cobalt chelation of precorrin-2, then in its absence the ‘chemical’ or non-enzymic chelation of precorrin-2 would be expected to be dependent on the presence of endogenous cobalt in the strain AR8906/pcysG<sup>A</sup>. No such dependence was observed (results not shown).

The former possibility described above, that the reduced level of cobyric acid is due to the overmethylating activity of CysG<sup>A</sup>, was investigated by looking at the effect of a uro'gen III transmethylase that does not overmethylate to the same extent as CysG<sup>A</sup>. Such a uro'gen III transmethylase is CobA from *P. denitrificans* [10]. Transformation of AR8906 (*E. coli cysG*, CobI<sup>+</sup>) with a plasmid harbouring *cobA* results in the production of cobyric acid at levels observed with *cysG*. This would suggest that it is the overmethylating activity of CysG<sup>A</sup> that is responsible for the lower levels of cobyric acid rather than the lack of a CysG<sup>B</sup> domain.

However, CysG<sup>B</sup> might have some role in regulating the activity of CysG<sup>A</sup> because transformation of AR8906 with a plasmid containing both CysG<sup>A</sup> and CysG<sup>R309L</sup> (pKKSW104) produced a strain that made cobyric acid at 506 pmol per attenuation unit (Table 3). Thus, under physiological conditions, CysG<sup>B</sup> might regulate and stabilize precorrin-2 synthesis and prevent the excessive accumulation of the overmethylated trimethylpyrrocorphin.

#### S. typhimurium CobI gene with an equivalent function to CysG<sup>B</sup>

Unexpectedly, the cysteine auxotrophy of *E. coli* 302Δa was complemented by the presence of the *S. typhimurium* CobI
operon and $cysG^{A}$. This would suggest that sirohaem can be made by a mechanism that does not require Cys$G^{B}$. Similarly, complementation of the cysteine auxotrophy was observed when the *P. denitrificans cobA* gene was used to transform AR8906. A possibility is that an enzyme involved in cobalamin biosynthesis within the CobI operon is able to chelate iron instead of cobalt at the level of precorrin-2 on the minimal medium plates. To test this theory, the minimal medium plates on which the strain was able to grow were supplemented with exogenous cobalt. The presence of the exogenous cobalt prevented growth on minimal medium of AR8906/pcys$G^{A}$ but did not affect growth of AR8906/pAR8414 (CysG). Thus the observed growth on minimal medium in the absence of cobalt might be due to iron acting as a surrogate metal at the level of precorrin-2. Evidence has previously been provided that strongly suggests that during corrin biosynthesis cobalt is added at the level of precorrin-2 [27]. Spontaneous oxidation of this compound would then yield sirohaem. Recently the *cbi* gene responsible for the Cys$G^{B}$-like activity within the *S. typhimurium* CobI operon has been shown to be *cbiK* [38].

In summary, a number of *cysG* variants have been constructed with changes in either the transmethylase or oxidase/chelatase regions of the protein. These have been investigated for their ability to bind AdoMet, complement an *E. coli cysG* deletion strain, and contribute to cobalamin biosynthesis. The results provide evidence that the two domains of *cysG* can act independently. In terms of cobalamin biosynthesis, only the Cys$G^{A}$ (transmethylase) domain of CysG is required because corrin biosynthesis is promoted in the absence of Cys$G^{B}$. However, evidence is presented that a gene within the *S. typhimurium* CobI operon encodes a protein with Cys$G^{B}$ functionality.

We thank Professor A. I. Scott and Dr. C. A. Roessner (Texas A & M University, College Station, TX, U.S.A.) for the gift of the *P. denitrificans cobA* harboured within pCR395. Financial support from the BBSRC is gratefully acknowledged.

**REFERENCES**

1. Warren, M. J. and Scott, A. I. (1990) Trends Biochem. Sci. **180**, 486–491
2. Debussche, L., Thibaut, D., Cameron, B., Crouzet, J. and Blanche, F. (1993) J. Bacteriol. **175**, 7430–7440
3. Scott, A. I. (1993) Angew. Chem. Int. Ed. Engl. **32**, 1223–1243
4. Battersby, A. R. (1994) Science **264**, 1551–1557.
5. Murphy, M. J., Siegel, L. M., Tove, S. R. and Kamen, H. (1974) Proc. Natl. Acad. Sci. U.S.A. **71**, 612–616
6. Crane, B. R., Siegel, L. M. and Getzoff, E. D. (1995) Science **270**, 59–67
7. Schneider, Z. and Stroinski, A. (1987) in Comprehensive B$_{12}$ (Schneider, Z. and Stroinski, A., eds.), pp. 259–266, Walter de Gruyter, Berlin
8. Warren, M. J., Stolowich, N. J., Santander, P. J., Roessner, C. A., Sowa, B. A. and Scott, A. I. (1990) FEBS Lett. **261**, 76–80
9. Warren, M. J., Roessner, C. A., Ozaki, S., Santander, P. J., Stolowich, N. J. and Scott, A. I. (1992) Biochemistry **31**, 603–609
10. Blanche, F., Debussche, L., Thibaut, D., Crouzet, J. and Cameron, B. (1989) J. Bacteriol. **171**, 4222–4231
11. Warren, M. J., Roessner, C. A., Santander, P. J. and Scott, A. I. (1990) Biochem. J. **265**, 725–729
12. Robin, C., Blanche, F., Cauchois, L., Cameron, B., Coouder, M. and Crouzet, J. (1991) J. Bacteriol. **173**, 4893–4896

13 Peakman, T., Crouzet, J., Mayaux, J. F., Busby, S., Mohan, S., Harborne, N., Wootton, J., Nicolson, R. and Cole, J. (1990) Eur. J. Biochem. **191**, 315–323
14 Spencer, J. B., Stolowich, N. J., Roessner, C. A. and Scott, A. I. (1993) FEBS Lett. **335**, 57–60
15 Warren, M. J., Bolt, E. L., Roessner, C. A., Scott, I. A., Spencer, J. B. and Woodcock, S. C. (1994) Biochem. J. **302**, 837–844
16 Fujino, E., Fujino, T., Karita, S., Sakka, K. and Ohmiya, K. (1995) J. Bacteriol. **177**, 5169–5175
17 Hansen, J., Muldbjerg, M., Chérest, H. and Surdin-Kerjan, Y. (1997) FEBS Lett. **401**, 20–24
18 Ogasawara, N., Fujita, Y., Kobayashi, Y., Sadaie, Y., Tanaka, T., Takahashi, H., Yamane, K. and Yoshikawa, H. (1995) Microbiology **141**, 257–259
19 Wu, J. Y., Siegel, L. M. and Kredich, N. M. (1991) J. Bacteriol. **173**, 325–333
20 Fazio, T. G. and Roth, J. R. (1996) J. Bacteriol. **178**, 6952–6959
21 Goldman, B. S. and Roth, J. R. (1993) J. Bacteriol. **175**, 1457–1466
22 Sambrook, J., Fritsch, E. F. and Maniatis, T. (1989) Molecular Cloning: A Laboratory Manual, 2nd edn., Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y
23 MacFerrin, K., Terranova, M., Schriever, S. and Verdine, G. (1990) Proc. Natl. Acad. Sci. U.S.A. **87**, 1937–1941
24 Nakamaye, K. L. and Eckstein, F. (1986) Nucleic Acids Res. **14**, 9679–9698
25 Crouzet, J., Cameron, B., Cauchois, L., Rigault, S., Rouyez, M.-C., Blanche, F., Thibaut, D. and Debussche, L. (1990) J. Bacteriol. **172**, 5980–5990
26 Woodcock, S. C. and Warren, M. J. (1996) Biochem. J. **313**, 415–421
27 Raux, E., Lanois, A., Levillayer, F., Warren, M. J., Brody, E., Rambach, A. and Thermes, C. (1996) J. Bacteriol. **178**, 753–767
28 Haydock, S. F., Dowson, J. A., Dhillon, N., Roberts, G. A., Cortes, J. and Leadlay, P. F. (1991) Mol. Gen. Genet. **230**, 120–128
29 Warren, M. J., Bolt, E. and Woodcock, S. C. (1994) in The Biosynthesis of the Tetrapyrrole Pigments (Ciba Foundation Symposium 180) (Chadwick, D. J. and Ackrill, K., eds.), pp. 26–40, Wiley, Chichester
30 Griffiths, L. and Cole, J. A. (1987) Arch. Microbiol. **147**, 364–369
31 Roth, J. R., Lawrence, J. G., Rubenfield, M., Kieffer-Higgins, S. and Church, G. M. (1993) J. Bacteriol. **175**, 3303–3316
32 Roessner, C. A. and Scott, A. I. (1995) Biotechniques **19**, 760–764
33 Hanukoglu, I. and Gutfinger, T. (1989) Eur. J. Biochem. **180**, 479–484
34 Rossman, M. G., Liljas, A., Brändén, C. I. and Banaszak, L. J. (1975) in The Enzymes, vol. 11 (Boyer, P. D., ed.), pp. 61–102, Academic Press, New York
35 Leustek, T., Smith, M., Murillo, M., Pal Singh, D., Smith, A. G., Woodcock, S. C., Awan, S. J. and Warren, M. J. (1997) J. Biol. Chem. **272**, 2744–2752
36 Lawrence, J. G. and Roth, J. R. (1995) J. Bacteriol. **177**, 6371–6380
37 Jeter, R., Olivera, B. M. and Roth, J. R. (1984) J. Bacteriol. **159**, 206–216
38 Raux, E., Thermes, C., Heathcote, P., Rambach, A. and Warren, M. J. (1997) J. Bacteriol. **179**, 3202–3212
39 Focus (1984) **6**, 4–7
40 Roessner, C. A., Spencer, J. B., Ozaki, S., Min, C. H., Atshaves, B. P., Nayar, P., Anousis, N., Stolowich, N. J., Holderman, M. T. and Scott, A. I. (1995) Protein Express. Purif. **6**, 155–163
41 UWGCG (1994) Program Manual for the Wisconsin Package, Version 8, September 1994, Genetics Computer Group, 575 Science Drive, Madison, WI 53711, U.S.A.
42 Jones, M. C., Jenkins, J. M., Smith, A. G. and Howe, C. J. (1994) Plant Mol. Biol. **24**, 435–448
43 Sakakibara, H., Takei, K. and Sugiyama, T. (1996) Plant J. **10**, 883–892
44 Deboer, A. P. N., Reijnders, W. N. M., Kuenen, J. G., Southamer, A. H. and Vanspanning, R. J. M. (1994) Antonie van Leeuwenhoek Int. J. Gen. Molec. Microbiol. **66**, 111–127
45 Blanche, F., Robin, C., Coulder, M., Faucher, D., Cauchois, L., Cameron, B. and Crouzet, J. (1991) J. Bacteriol. **173**, 4637–4645
46 Bult, C. J., White, O., Olsen, G. J., Zhou, L., Fleischmann, R. D., Sutton, G. G., Blake, J. A., FitzGerald, L. M., Clayton, R. A., Gocayne, J. D. et al. (1996) Science **273**, 1058–1073
47 Chou, P. L., Ohtsuka, M., Minowa, T., Yamasato, K., Sakano, Y., Matsuzawa, H., Ohta, T. and Sakai, H. (1995) Biosci. Biotechnol. Biochem. **59**, 1817–1824
48 Sattler, I., Roessner, C. A., Stolowich, N. J., Hardin, S. H., Harris-Haller, L. W., Yokobaitis, N. T., Murooka, Y., Hashimoto, Y. and Scott, A. I. (1995) J. Bacteriol. **177**, 1564–1569
49 Milton, D. L., Norquist, A. and Wolf-Wartz, H. (1995) Gene **164**, 95–100

Received 17 April 1997/12 September 1997; accepted 6 October 1997
