
Molecular and Biochemical Parasitology 91 (1998) 51–66

**Multiple mechanisms of immune evasion by African trypanosomes**

John E. Donelson *, Kent L. Hill, Najib M.A. El-Sayed

*Department of Biochemistry, University of Iowa, Iowa City, Iowa 52242, USA*

---

### Abstract

During infection of a mammalian host, African trypanosomes are in constant contact with the host’s immune system. These protozoan parasites are infamous for their ability to evade the immune responses by periodically switching their major variant surface glycoprotein (VSG), a phenomenon called antigenic variation. Antigenic variation, however, is likely to be only one of several mechanisms enabling these organisms to thrive in the face of the immune defenses. The ability to grow in high levels of interferon-gamma (IFN-γ) and to avoid complement-mediated destruction may also facilitate the parasite’s survival. In this review we summarize (i) the activation of trypanosome genes for three different VSGs during antigenic variation, (ii) the secretion of a trypanosome protein that induces host CD8⁺ T cells to produce IFN-γ, and (iii) the evidence for trypanosome protein similar to a surface protease of Leishmania that plays a role in resistance to complement-mediated lysis. © 1998 Francqui Foundation. Published by Elsevier Science B.V. All rights reserved.

**Keywords:** Trypanosomes; Recombinant cloning; VSG genes; T cell; Leishmania

---

### 1. Introduction

The molecular mechanisms used by African trypanosomes to evade the immune responses of their mammalian hosts have been an object of interest and investigation for most of this century. As early as 1907, the Italian scientist, A. Massaglia, concluded that African trypanosomes in the bloodstream ‘escape destruction because they become used or habituated to the action of antibodies’ (quoted in [1]), a remarkable deduction given the understanding of immunology at the time. The discovery in 1976 that different populations of trypanosomes in the mammalian bloodstream possess different proteins on their surface [2] drew attention to the phenomenon of antigenic variation in African trypanosomes, the process whereby these organisms periodically change the

**Abbreviations:** BC, basic copy; EC, expression copy; ELC, expression linked extra copy; ESAG, expression site-associated gene; GP63, glycoprotein of 63 kilodaltons; IFN-γ, interferon gamma; MVAT, metacyclic variant antigen type; TLTF, T lymphocyte triggering factor; VSG, variant surface glycoprotein.

*Corresponding author. Tel.: +1 319 3357889; fax: +1 319 3356764; e-mail: john-donelson@uiowa.edu

0166-6851/98/$19.00 © 1998 Francqui Foundation. Published by Elsevier Science B.V. All rights reserved.
PII S0166-6851(97)00209-0

major protein on their surface enabling them to potentially keep one step ahead of the immune system's response. The first recombinant cloning of the trypanosome genes for these variant surface glycoproteins (VSGs) in 1979–1980 [3,4] ushered in the use of molecular biology techniques to study the events at the DNA and RNA level in trypanosomes that are responsible for antigenic variation.

The 1980’s and 1990’s have seen the publication of a large amount of data documenting that bloodstream African trypanosomes switch from the expression of one VSG to another and that this switch is sometimes associated with duplicative transpositions or other rearrangements of VSG genes within the trypanosome genome (reviewed by several articles in this issue). The intricacies of these VSG gene switching events and the success in elucidating their molecular mechanisms have focused attention on antigenic variation as the dominant immune defense mechanism of these parasites. However, antigenic variation is not the sole determinant of the amazingly successful ability of this parasite to evade the immune defenses. It is not known to be directly responsible for the resistance of African trypanosomes to complement-mediated lysis or for the elevated level of interferon-gamma (IFN-γ) and the profound suppression of many immune responses that occur in the infected host. Thus, antigenic variation appears to be but one of several mechanisms that African trypanosomes use to evade the immune system of their mammalian hosts.

antigens are not successful in clearing the infection because the invariant proteins are inaccessible inside the living trypanosomes and the readily accessible VSGs periodically switch. Thus, many of the immune events that occur during an African trypanosome infection are likely the result of the perpetual presence of ever-changing VSGs mixed with many invariant antigens, a situation similar to that of successive infections by related but non-identical organisms (reviewed by [6]).

The immune responses to an African trypanosome infection are dominated by two overwhelming phenomena: a massive non-specific polyclonal activation of B cells and a generalized suppression of some humoral (B cell) and cellular (T cell) immune functions [69]. The polyclonal B cell activation results in a large production of IgM, the first class of antibodies to be generated by the appearance of new foreign antigens. This activation is not triggered entirely by the continually changing epitopes of the different VSGs, however, since the newly synthesized antibodies do not react solely with the VSGs and other trypanosome antigens. They frequently are heterogeneous in their reactivity and can be autoantibodies directed against the proteins and nucleic acids of the host. It has been proposed that an unknown non-VSG molecule of trypanosomes [6,7], or perhaps even the VSGs themselves [8], serve as the mitogen causing this massive non-specific expansion of B cells and the subsequent increase in immunoglobulin concentration. The greatly elevated levels of IgM and resultant antigen–antibody complexes in turn cause hyperplasia of the reticuloendothelial system, especially the spleen and lymph nodes, and are likely to be responsible for many of the pathogenic characteristics of the infection.

The other striking immune feature of African trypanosome infections, a dramatic suppression of immune responses other than the initial B cell activation, results in an increased susceptibility to opportunistic infections. This generalized immune suppression has been reported to affect a large variety of both B cell and T cell functions and seems to inhibit many secondary immune events (reviewed in [6]). For example, the enormous IgM production is not followed by the normally con-
comitant increase in IgG and the other antibody classes, and T cell proliferation is severely suppressed. However, the concentration of IFN-γ is greatly increased in experimental animals infected with *Trypanosoma brucei*. Macrophage activity is enhanced, perhaps by the increase in IFN-γ, and the elimination of trypanosomes by antibodies is thought to be mediated by opsonization and destruction by liver macrophages, rather than by complement-mediated lysis [9,10]. The levels of some cytokines such as IL-2 decrease during trypanosome infections which may contribute to the lack of T cell proliferation. Although the coexistence of massive polyclonal B cell expansion and profound immunosuppression appears to be a paradox, both phenomena obviously generate an environment conducive to perpetuation of the infection.

This review summarizes recent progress in our laboratory on three of the above issues related to the general question of how African trypanosomes in the mammalian bloodstream avoid destruction by the immune system: (a) characterizations of the duplication and transcription of VSG genes involved in antigenic variation; (b) elucidation of a trypanosome protein that induces host CD8⁺ T cells to produce IFN-γ; and (c) identification of a protease that the trypanosome may utilize to evade complement-mediated lysis.

### 3. The expression of three telomere-linked VSG genes

The final developmental stage of African trypanosomes in the tsetse fly is a quiescent form of the parasites in the insect’s salivary glands called the metacyclic stage. Metacyclic trypanosomes can express only one of 10–15 different VSGs on their surface [11–13], in contrast to bloodstream trypanosomes which sequentially express hundreds of different VSGs. The metacyclic VSGs are used to define the metacyclic variant antigen types (MVATs) of the parasite. After metacyclic parasites enter the mammalian bloodstream, they continue to express one of the MVAT VSGs for 5–7 days and then change to the expression of a bloodstream VSG followed by the periodic VSG switching. Late in the infection MVAT VSGs are occasionally re-expressed as bloodstream VSGs.

The expressed genes for both bloodstream and metacyclic VSGs are invariably located near chromosomal telomeres. Only a few expressed bloodstream VSG genes have been examined in detail, but all have been found to be transcribed from telomere-linked expression sites that typically consume 45–60 kb and contain (in a 5′ to 3′ direction) a promoter, seven or more expression site-associated genes (ESAGs), a variable number of 70–76-bp repeats, and the VSG gene followed by subtelomeric and telomeric DNA repeats (see this issue for reviews). Transcription of these bloodstream VSG gene expression sites generate polycistronic precursor RNAs containing eight or more protein-coding regions that are subsequently processed into mature, monocistronic mRNAs. Likewise, only a few telomere-linked expression sites for VSG genes expressed during the metacyclic stage have been characterized in detail, but these MVAT VSG gene expression sites are much smaller in size than those for bloodstream VSG genes. These telomere-linked MVAT VSG expression sites lack many or all of the ESAGs, do not contain the multiple 70–76-bp repeats and are transcribed into monocistronic precursor RNAs from a proximal promoter located within 2 kb of the VSG gene [14,15].

Several years ago our laboratory began to investigate bloodstream trypanosome clones that re-express VSGs normally present during the metacyclic stage. Our goal was to elucidate the molecular mechanisms responsible for the activation of these MVAT VSG genes in bloodstream trypanosomes. For these investigations we have focused on trypanosome clones that initially appeared to have on their surface one of three metacyclic VSGs named MVAT4, MVAT5 and MVAT7. Bloodstream trypanosomes expressing one of these MVAT VSGs were initially identified in immunofluorescence assays of bloodstream trypanosome populations using monoclonal antibodies specific for each of these three MVAT VSGs [16]. When a trypanosome population containing a few organisms that reacted with these monoclonal antibodies was found, the parasites were serially diluted to either one or a few organisms
J.E. Donelson et al. / Molecular and Biochemical Parasitology 91 (1998) 51–66

![Diagram](attachment:diagram.png)

Fig. 1. Summary of the duplicative gene transpositions (MVAT 5 and MVAT 7) and the in situ gene activation (MVAT4) associated with the re-expression in bloodstream trypanosomes of the MVAT5, MVAT7 and MVAT4 VSG genes. Rectangles are VSG genes (1.5 kb in length) or ESAG I family members. Black circles are telomeres. Black flags are the active promoters in the bloodstream re-expressor trypanosomes. White flags are promoters not active in bloodstream re-expressor trypanosomes but thought to be active in metacyclic trypanosomes. BC and ELC are the loci containing the basic copy and expression linked copy, respectively, of the VSG gene. Zigzag lines in the MVAT5 ELC denotes 70–76-bp repeats. The BC locus of the MVAT4 VSG gene is also shown as an expressed copy (EC) locus since there is no duplication event to create an ELC. Crossed lines between the BC and ELC loci are the 5' boundaries of the duplicative gene transposition. Wavy lines with an arrowhead are precursor RNAs derived from their respective promoters. Adapted from reference [19].

that were used to infect immunocompromised mice and the new population assayed with the monoclonal antibodies again. This process of dilution and subsequent growth in immunocompromised mice was repeated several times until a pure population (>99%) of bloodstream trypanosomes was obtained that expressed the VSG recognized by the MVAT monoclonal antibodies. These bloodstream trypanosome clones expressing either the MVAT4, MVAT5 or MVAT7 VSG became the basis of our subsequent studies. DNA and RNA were isolated from each of these three sets of trypanosome clones for construction of genomic DNA and cDNA libraries. In addition, the same libraries were prepared from a bloodstream trypanosome clone of the same serodeme,

called the WRATat 1.1 clone, which expresses an unrelated bloodstream VSG gene. The MVAT VSG chromosomal genes and their corresponding cDNAs were isolated from these libraries, and used for sequencing studies and as probes in Southern and Northern blots to determine the positions and organization of these genes in their expressed and non-expressed states.

We found that in WRATat 1.1 trypanosomes each of the three MVAT VSGs is encoded by a silent, single copy gene already located near a telomere [indicated in Fig. 1 as the basic copy (BC) MVAT VSG genes]. In Southern blots probed with unique portions of each of the three MVAT VSG cDNAs, no other isogene that cross-hybridizes under high hybridization stringency

J.E. Donelson et al. / Molecular and Biochemical Parasitology 91 (1998) 51–66

| 10 | 20 | 30 | 40 | 50 | 60 | 70 bp |
| --- | --- | --- | --- | --- | --- | --- |
| MVAT5 | TCAGAAGGCGGACGCTCTAGGCATGCCAAATGTGGTTAAAGCAGGATTTCGAGGCATAAATTCAAGATTTATTTCAAG | | | | | ↓ |
| MVAT4 | GTCCGGGACCACCCCTATAACCCGACCAAATACATGTAAGCGAGGTTTCGTGGAAAAAACCATCAGTTGCACCCATA | * | * * * * * | * | * * * * * | * * * * * | * * * * * | * * | * * |
| MVAT7 | GTCCGGTGCCACCC-TGTAAGCCGACCAAATATGTGTAAGGCGAGGTTTCGTGGCATAAAACCCCCGATTTAAGCCAAG | * * * * * | * * * * * | * * * | * * * * * * * * * * * * * * * | * * * * * | * * * * * * * * * * * * * * * | * * * * * | * * | * * * * |

CONSENSUS | C | C | T TA | C | CCAAAT | TAA GC | G TTCG GG A AAA | TT | A

Fig. 2. Sequence alignment of the promoters located about 2 kb upstream of the basic copy genes for the MVAT5, MVAT7 and MVAT4 VSGs. The sequences correspond to the flags shown in Fig. 1. Asterisks indicate nucleotide identity. The arrow indicates transcription start sites mapped in the MVAT7 and MVAT4 re-expressor trypanosomes. CONSENSUS shows those 42% of the positions that are identical in the three promoters. Adapted from reference [19].

was found to occur in the WRATat 1.1 genome, either at a chromosome-interior location or near a telomere, suggesting that these MVAT VSG genes are not closely related members of isogene families as are some bloodstream VSG genes. Consistent with this observation is the finding that none of these three telomere-linked, basic copy MVAT VSG genes is preceded by a long stretch of 70–76-bp repeats which are thought to be involved in some bloodstream VSG gene duplications and rearrangements [17]. Two of these three basic copy MVAT VSG genes, however, are preceded by a member of the ESAG I family located 5–7 kb upstream. An ESAG I is not located within 5–7 kb upstream of the other basic copy MVAT VSG gene, although it is not known if it might be positioned still further upstream. Members of the other ESAG families found in bloodstream VSG gene expression sites have not been detected in the vicinity of the MVAT VSG genes.

Each of these three basic copy MVAT VSG genes is, however, preceded about 2 kb upstream by a 70-bp sequence that acts as a promoter when placed in front of a reporter gene on a plasmid used in transient transfections of trypanosomes [14, 18, 19]. These promoter sequences are likely to be activated during the metacyclic stage of the life cycle, although we have not conclusively proven this possibility. These three MVAT VSG gene promoters possess 42% positional identity (Fig. 2) and we have conducted limited mutagenesis studies to demonstrate that specific nucleotides shared within the 70-bp are crucial for the promoter activity. We have also determined at least 5 kb of sequence upstream of each of the three basic copy MVAT VSG genes and find that, other than the aforementioned ESAG I, the only regions of similarity among these three upstream regions are the 70-bp promoter sequences. A comparison of these three MVAT promoter sequences with the sequences previously determined for the promoters of three bloodstream VSG promoters [20–22] revealed that relatively few nucleotides are shared among all six promoters. The longest stretch of identity among all six promoters is three adjacent nucleotides and only about one in five positions in the 70-bp are shared by the same nucleotide in all six promoters (see Fig. 6 of [19]). Thus, there is still much to be learned about the sequences that serve as the promoters at VSG expression sites.

Our early analyses of the bloodstream trypanosome clones re-expressing these three MVAT VSGs revealed that indeed the silent, basic copy MVAT VSG gene served as the donor gene for the bloodstream re-expression event. However, each of the three basic copy MVAT VSG genes was found to be re-expressed in bloodstream trypanosomes via a different molecular mechanism. Each of these three mechanisms is discussed in turn.

### 3.1. Re-expression of MVAT5 VSG

Southern blots of genomic DNA from three independently derived bloodstream trypanosome clones re-expressing the MVAT5 VSG demonstrated that in each case the telomere-linked, basic

copy VSG gene is duplicated and the duplicated copy is located adjacent to another telomere. The upstream (5') boundary of the duplicated segment occurs 844-bp in front of the start codon of the VSG gene in two of the three cases, and only 234-bp in front of the start codon in the other case [16,23]. Thus, none of these three duplications of the basic copy MVAT5 VSG gene includes the promoter sequence located about 2 kb upstream of the gene (Fig. 1). The downstream (3') boundaries of these three independent gene duplications could not be identified from Southern blots but they must be located more than 300-bp downstream of the VSG gene stop codon, and it is possible that the duplicated segments extend all the way down to the telomere itself, although this possibility has not been confirmed.

In each of the three cases the upstream (5') boundary of the newly duplicated segment containing the MVAT5 VSG gene is positioned immediately 3' of a region of at least 20 kb that possesses tandem 70–76-bp repeats (jagged line in Fig. 1). Thus, it is likely that these repeats participated in the recombination events that led to the translocation of the duplicated segment into the expression site despite the fact that the basic copy MVAT5 VSG donor gene is not preceded by a sequence resembling a consensus of the 70–76-bp repeat.

We have been unable to identify genomic DNA clones containing the region located upstream of the tandem 70–76-bp repeats. Probably, the 20 kb of 70–76-bp repeats is unstable in recombinant DNA clones. Thus, we do not know if the three independent duplications were translocated into the same expression site or if they went to different expression sites. The results of Southern blots are consistent with the same expression site being used for each of the three duplications, but since the restriction sites preceding the 70–76-bp repeats are more than 20 kb upstream of the probe sequence, it is difficult to deduce an accurate restriction map. Nuclear RNA run-on assays and UV-irradiation inactivation experiments demonstrated that, in each of the three cases, the promoter for the expression site lies far upstream of the expressed MVAT5 VSG gene and almost surely is located upstream of the 70–76-bp repeats. Thus, the duplicative activation of the basic copy MVAT5 VSG gene in bloodstream trypanosomes is similar to that described for several bloodstream VSG genes, i.e. the duplicated gene is inserted immediately downstream of 70–76-bp repeats of an expression site and the transcription unit begins many kb upstream of the VSG gene itself. The monocistronic MVAT5 VSG mRNA is then derived from this large precursor RNA.

Although most features of these MVAT5 VSG gene duplications and expression are similar to those of a 'conventional' bloodstream VSG gene duplication/expression, one common feature of the three independent MVAT5 VSG gene duplications is decidedly unconventional. In each of the three duplications, the duplicated gene of about 1500-bp is not a faithful copy of the MVAT5 VSG donor gene [16,23]. Point mutations distinguishing the duplicated gene from the donor gene are distributed throughout the coding region, with the greatest concentration of mutations occurring in the middle half of the gene. These point changes were first detected on Southern blots in which restriction enzymes that cleave in the coding region of the basic copy gene were found to not cleave in the duplicated gene. The presence of the mutations was subsequently confirmed by sequence determinations of multiple cDNA and genomic DNA clones derived from the bloodstream trypanosome re-expressor clones. Fig. 3 summarizes the distribution of the mutations in the three duplicated gene copies. One duplicated gene copy (Rx1) has 35 point mutations, another (Rx2) has 11 and the third (Rx3) has 28 point changes, all relative to the original basic copy MVAT5 donor gene. The sequence of this donor gene is unchanged in each of these three trypanosome re-expressor clones and in the WRATat 1.1 trypanosome clone expressing an unrelated bloodstream VSG. Thus, the mechanism responsible for the mutations does not introduce changes into the donor gene of the duplication. Likewise, the region of the duplicated segment located upstream of the MVAT5 VSG gene (844-bp in the Rx1 and Rx2 duplications, and 234-bp in the Rx3 duplication) does not possess any mutations relative to the corresponding region preceding the donor gene. Since mutations would be expected to occur

J.E. Donelson et al. / Molecular and Biochemical Parasitology 91 (1998) 51–66

BC
Rx 1
Rx 2
Rx 3

Fig. 3. Distribution of point mutations in the duplicated VSG gene of three independent bloodstream trypanosome clones re-expressing the basic copy MVAT5 VSG gene [23]. The large rectangle represents the 518 codons of the basic copy (BC) MVAT5 VSG donor gene. The duplicated VSG gene in MVAT5 trypanosome re-expressor clones 1, 2 and 3 (Rx 1, Rx 2 and Rx 3) contains 35, 11 and 28 point mutations, respectively, when compared to the BC donor gene. Vertical lines represent the point mutations that cause an amino acid change; black dots are the point mutations that do not cause an amino acid change.

in this upstream region of the duplicated segment if they were introduced in a completely random manner during the duplicative transposition event, the events responsible for them must be confined to the coding region, or perhaps to the region between the spliced leader and the polyadenylation addition sites. Another clue that the mutations are not introduced randomly during the duplication event is the fact that in the sense strand of the coding region, 54% of the mutations are purine transitions and only 7% are pyrimidine transitions (out of the 74 total mutations in the three duplications). If the mutations were introduced randomly by, for example, a DNA polymerase involved in the duplication, both strands would be expected to have an equal distribution of purine and pyrimidine transitions. Despite the indications that the mutations are not randomly generated, the molecular mechanism responsible for them remains unclear. We have discussed some of the possible alternatives previously [23, 24], so will not repeat them here except to note that the data on the mutations are consistent with, but do not prove, the existence of an RNA intermediate in the events leading to expression of the duplicated MVAT5 VSG genes. This possibility might account for the strand bias of the mutations and their locations between the spliced leader and polyadenylation addition sites in the duplicated segment. Also supporting this possibility is the presence in the trypanosome genome of more than 400 copies of a retrotransposon-like element called ingi, each of which contains a reverse transcriptase gene whose sequence varies slightly from one ingi to another [25–28]. If some of these ingi elements encode a reverse transcriptase with a very low proof-reading activity, point mutations could be introduced during the reverse transcription of an mRNA whose DNA product could become integrated into the genome at a VSG gene expression site.

Finally, about two-thirds of the point mutations lead to changes in the encoded amino acids (Fig. 3). The observation that most of these mutations are located in the middle of the gene may suggest that mutations which change the amino acids in the N- and C-termini interfere with the ability of the VSG to fold or target properly and result in death of the organism. Monoclonal antibodies directed against the VSG from each of the MVAT5 re-expressor clones can distinguish among the three VSGs, indicating that at least some of the mutations alter the epitopes of these closely related VSGs [23]. Thus, the introduction of point mutations in duplicated VSG genes is a way for the trypanosome to expand its repertoire of immunologically distinct VSGs.

### 3.2. Re-expression of MVAT7 VSG

Southern blots of genomic DNA from a bloodstream trypanosome clone re-expressing the MVAT7 VSG revealed that this telomere-linked, basic copy MVAT7 VSG gene is also duplicated and inserted near another telomere prior to its expression in bloodstream trypanosomes, similar to that of the MVAT5 VSG gene (Fig. 1). In this MVAT7 case, however, sequence determinations of the duplicated gene and its corresponding cDNA revealed that the sequence of duplicated gene is an exact replica of the basic copy donor gene [19]. No mutations occurred during the duplication.

lication. It is not clear why the duplication of one telomere-linked, basic copy MVAT VSG gene would not involve mutations (MVAT7), whereas three independent duplications of another telomere-linked, basic copy MVAT VSG gene would each have mutations (MVAT5), unless the duplication of these two donor genes occurs via different molecular mechanisms. In the original isolation of bloodstream trypanosomes re-expressing MVAT VSGs, the few parasites re-expressing the MVAT7 VSG were detected about as frequently as the few organisms re-expressing the MVAT5 VSG, so duplications of both donor genes likely occur at about the same low rate.

In addition to these mutations, several other differences exist between the duplications of the basic copy MVAT7 and MVAT5 VSG genes. First, the 5′ boundary of the duplicated MVAT7 VSG gene occurs about 3 kb upstream of the start codon, which is more than 2 kb further upstream than the 5′ boundaries of the duplicated MVAT5 VSG genes. Secondly, the 70–76-bp repeats in the bloodstream MVAT5 VSG expression site are not present in the expression site of the re-expressed MVAT7 VSG gene. Likewise, no members of the ESAG I family have been found upstream of the either the MVAT7 VSG donor gene or its expressed duplicated copy. Thirdly, and most interestingly, the MVAT7 duplicated segment includes the promoter sequence preceding the MVAT7 VSG donor gene (Fig. 1). Thus, this promoter is co-duplicated with its MVAT VSG gene. Primer extension experiments and nuclear RNA run on assays demonstrate that this promoter is used for the transcription of the expressed MVAT7 VSG gene and that the region immediately preceding it in the expression site is not transcribed. Therefore, the co-duplicated promoter is the promoter that is activated at the expression site, in contrast to the MVAT5 VSG gene duplications in which the duplicated gene is subsequently transcribed from a far upstream promoter already present in the expression site.

### 3.3. Re-expression of MVAT4 VSG

Southern blots of genomic DNA from a bloodstream trypanosome clone re-expressing the

MVAT4 VSG indicated that the single telomere-linked, basic copy MVAT4 VSG gene was not duplicated or otherwise rearranged prior to its re-expression in the bloodstream. Likewise, careful quantification of the radioactive signals of slot blots containing genomic DNAs from several trypanosome clones probed with the MVAT4 VSG cDNA failed to provide evidence for a gene duplication [14]. Thus, the bloodstream re-expression of the MVAT4 VSG gene is an example of the in situ activation of a VSG gene already located near a telomere (Fig. 1). Primer extension experiments and nuclear run on assays indicate that the bloodstream transcription of this gene is initiated at the sequence about 2 kb upstream of the gene that was shown via transient transfections with a reporter gene to possess promoter activity. The 2 kb region between this promoter and the VSG start codon does not possess an open translation reading frame of any significant length and precursor RNA derived from this promoter sequence does not have a spliced leader at its 5′ end. Thus, this precursor RNA is a monocistronic RNA that is processed via 3′ polyadenylation and 5′ spliced leader addition to a site about 20 nucleotides in front of the VSG start codon to yield the MVAT4 mRNA.

Fig. 1 shows that an ESAG I is located about 3 kb upstream of this MVAT4 promoter sequence (5 kb upstream of the VSG gene). However, when ESAG I cDNAs from a cDNA library of this MVAT4 re-expressor trypanosome clone were isolated and examined, none was found to have exactly the same sequence as this specific ESAG I family member [29]. Of the 25 ESAG I cDNA clones from this library whose sequences were determined, at least 20 different ESAG I members were identified, none of which are identical to the ESAG I gene preceding the MVAT4 promoter. Furthermore, nuclear RNA run on assays in the presence of α-amanitin demonstrated that these expressed ESAG I members are transcribed by an RNA polymerase II (α-amanitin sensitive) activity, rather than the α-amanitin resistant activity known to transcribe VSG genes, including the re-expressed MVAT4 VSG gene [29]. Thus, although ESAG families were first identified as genes whose expression is associated with that of

telomere-linked VSG genes (hence, the name of expression site associated genes), in this bloodstream trypanosome clone re-expressing the MVAT4 VSG, the expressed ESAG I family members are not transcribed by the same RNA polymerase as the VSG gene, nor are these expressed ESAG I members in the vicinity of the VSG gene.

To summarize the information available on the re-expression of the MVAT4, MVAT5 and MVAT7 VSGs in bloodstream trypanosomes, the activation of these three basic copy VSG genes share some common features and differ in other respects. For example, the MVAT4 and MVAT7 VSG genes are both transcribed into monocistronic precursor RNAs whose transcription is initiated from the same promoters thought to be used in the metacyclic expression of these genes. Interestingly, these two promoters are much more similar in sequence to each other than either is to the MVAT5 promoter (Fig. 2). In contrast, the basic copy MVAT5 VSG gene is not re-expressed from its own promoter. It is re-expressed via duplication to a 'conventional' bloodstream expression site, containing 70–76-bp repeats, in which transcription initiation occurs far upstream to generate a large precursor RNA that is cleaved into individual mRNAs. On the other hand, the MVAT5 and MVAT7 VSG genes are similar in that they are both re-expressed in the bloodstream via duplicative transposition, whereas the MVAT4 VSG gene is re-expressed via in situ activation (Fig. 1). Finally, MVAT5 VSG gene duplications are associated with mutations in the VSG coding region (Fig. 3), whereas the MVAT7 VSG gene duplication generates a faithful copy of the donor gene.

pleted of CD8⁺ T cells by injection of anti-CD8 antisera or in knockout animals possessing a deletion in the CD8 gene, the trypanosomes do not induce as much IFN-γ, the parasitemia is decreased and the infected animals survive longer [30]. In contrast, in rodents depleted of CD4⁺ T cells, trypanosomes induce about the same level of IFN-γ as they do in wild type animals of the same strain. These results are consistent with the possibility that a constituent(s) of bloodstream trypanosomes stimulates, either directly or indirectly, the production of IFN-γ from CD8⁺ T cells but not from CD4⁺ T cells. Several years ago bloodstream trypanosomes were shown to release a protein that possesses this property, i.e. its presence induces CD8⁺, but not CD4⁺, T cells to secrete IFN-γ [31–33]. This protein, named T lymphocyte triggering factor or TLTF, was subsequently shown to bind directly to the CD8 molecule on T cells [30]. IFN-γ, in turn, induces the production of a mitogen-activated protein (MAP) kinase in African trypanosomes [34] which might be a factor contributing to proliferation of trypanosomes in the bloodstream. Consistent with this possibility, a monoclonal antibody directed against TLTF reduces parasite levels in, and increases survival of, mice infected with African trypanosomes [35,36].

In collaboration with Moiz Bakhiet, Tomas Olsson, Krister Kristensson and their colleagues, who first identified and characterized TLTF, we immunoscreened a trypanosome cDNA library with antisera directed against affinity purified TLTF. A partial length cDNA clone whose protein product reacted with the antisera was used to identify full length cDNAs containing the 5′ spliced leader and 3′ poly(A). The corresponding full length recombinant protein was produced in Escherichia coli as a fusion protein using two different expression plasmids in which the partner protein was either thioredoxin or glutathione-S-transferase [37]. Both versions of the fusion protein were used to raise antisera and to test for TLTF activity. In blind assays the antisera against the recombinant fusion proteins were found to inhibit the ability of native TLTF to induce the production of IFN-γ from CD8⁺ T cells. Likewise, the recombinant fusion proteins themselves

4. A trypanosome protein that triggers IFN-γ production

One of the hallmarks of an African trypanosome infection is an elevated level of IFN-γ in the bloodstream. When rodents are experimentally infected with African trypanosomes, IFN-γ production in the spleen increases markedly within a few hours [6]. However, in rodents de-

Fig. 4. Targeting of a fusion protein of TLTF and the green fluorescent protein (GFP) to vesicles localized near the trypanosome flagellar pocket. The trypanosomes were transiently transfected with an expression plasmid containing the GFP gene alone (left panel) or a plasmid encoding a TLTF-GFP fusion protein (right panel). The trypanosome on the right was stained with ethidium bromide to reveal the nucleus (large red area) and the kinetoplast (small red area). The kinetoplast serves as a marker for the subcellular location of the flagellar pocket.

were shown to possess the TLTF activity of inducing CD8+ T cells to produce IFN-γ, whereas unrelated fusion proteins produced from the same expression plasmids did not have this activity. These results are consistent with the interpretation that the cDNA encodes TLTF.

The protein sequence deduced from the TLTF cDNA is 453 amino acids in length. It bears no obvious similarity to IGIF (IFN-γ-inducing factor), a protein recently identified in mouse liver that induces IFN-γ production by CD4+ T cells [38]. The lack of similarity between these two proteins could be because one targets CD8+ T cells and the other acts on CD4+ T cells. However, TLTF is similar in sequence to two mammalian sequences that have been deposited in GenBank without further characterization. One is a mouse protein of 489 amino acids designated as a growth arrest-specific protein. The other protein is encoded by a randomly sequenced human cDNA from a fetal lung cDNA library. In each case TLTF displays about 35% identity and 58% similarity to the mammalian protein [37]. Although the properties and functions of these mouse and human proteins are unknown, it is tempting to speculate that they may possess an activity towards CD8+ cells that resembles that of TLTF. If so, then TLTF, a trypanosome-encoded protein, mimics the activity of an immunomodulatory protein of its host. In recognition of this possibility, we have suggested that TLTF is an example of a group of proteins that we have designated as trypanokines, i.e. factors secreted by trypanosomes that modulate the cytokine network of the host immune system for the benefit of the parasite [37]. Another potential trypanokine is a secreted cysteine protease [39,40] since cysteine proteases of other microorganisms have been shown to modulate cytokine activity [41,42].

For TLTF to gain access to host T lymphocytes, it must first be delivered to the exterior of the trypanosome cell. However, the deduced TLTF amino acid sequence does not contain a hydrophobic region that might serve as a signal for membrane transport. Since it specifically lacks a classical N-terminal hydrophobic signal sequence similar to those found in most secreted eukaryotic proteins [43], we decided to investigate how TLTF is targeted for export from the trypanosome cell. For these experiments the luciferase gene in the trypanosome expression plasmid pHD1 [44] was replaced with a mutated version of the gene for the green fluorescent protein (GFP) of *Aequorea victoria* that gives 30–100-times more fluorescence than wild type GFP.

[45]. Trypanosomes transiently transfected with this plasmid display a bright green fluorescence several hours after electroporation (Fig. 4, left). The fluorescence is distributed uniformly throughout the trypanosome cell including the nucleus and the flagellum. Thus, the amino acid sequence of GFP itself does not restrict it from, nor direct it to, any specific compartment within the trypanosome.

When the TLTF coding sequence is placed either in front of, or behind, the GFP gene to generate a TLTF-GFP fusion protein and the resultant plasmid is transiently transfected into trypanosomes, the result shown in the right panel of Fig. 4 is obtained. The green fluorescence is now confined to vesicles in the cytoplasm of the trypanosome, most of which are located near a region of the trypanosome called the flagellar pocket. This surface pocket of trypanosomes has long been thought to be the site of endocytosis and secretion for the organism. It is derived from a specialized invagination of the cell membrane that forms at the location where the flagellum emerges from the cell. Thus, since TLTF is a secreted protein, it is reasonable to find that vesicles containing the TLTF-GFP fusion protein would congregate around the flagellar pocket. The fact that the same vesicular localization of the fluorescence is obtained when TLTF is positioned either in front of or behind GFP indicates that the location of the fusion protein is not an artifact of the relative order of the TLTF and GFP sequences. Thus, TLTF appears to be a member of the group of secreted eukaryotic proteins for which the amino acid sequence provides no indication of the targeting signal. Interestingly, some of the other secreted proteins in this group are mammalian immunoregulatory proteins such as IL-1b and adult T cell leukemia-derived factor (ADF) [46, 47].

Many questions remain about TLTF. Is there a unique secretory pathway for TLTF in trypanosomes? What is the sequence in TLTF that targets it to the cytoplasmic vesicles? How do TLTF and CD8⁺ cells interact? What is the signaling pathway within CD8⁺ T cells that stimulates INF-g production? Does vaccination with recombinant TLTF reduce susceptibility to trypanosome infection? These questions are currently being addressed experimentally.

5. A trypanosome protein similar to the metalloprotease GP63 of *Leishmania*

While conducting a preliminary project to determine the efficiency of sequencing random fragments of trypanosome genomic DNA for discovering new trypanosome genes, we identified a fragment that potentially encodes a protein with substantial similarity to a surface protease called GP63 that occurs in all *Leishmania* species investigated [48]. This fragment was used to screen trypanosome cDNA and genomic DNA libraries, and to probe Southern blots of trypanosome genomic DNA. These experiments revealed that African trypanosomes possess several non-identical, tandemly arrayed genes encoding GP63-like proteins [70]

GP63s and their genes in different *Leishmania* species have studied extensively [49–52]. These studies have included characterizations of the zinc protease activity and its substrate specificity, elucidation of the post-translation modifications of this glycosylphosphatidylinositol (GPI)-anchored protein, and determination of the chromosomal organization and differential expression of the multicopy GP63 genes. Depending on the *Leishmania* species, the infective metacyclic promastigote form of *Leishmania* can have as many as one-half million GP63 molecules on its surface [50, 53]. These *Leishmania* GP63s have been hypothesized to have a number of different functions, several of which involve the interaction between the infective promastigote and host macrophages. GP63s have been reported to participate in the entry of promastigotes into macrophages [54–56] and to contribute to the survival of the amastigote form of *Leishmania* inside the macrophage [57]. The protease activity of GP63 on the surface of the promastigotes has also been shown to contribute to the resistance of promastigotes to complement-mediated lysis [58]. Finally, immunization with recombinant GP63 has been found to partially protect experimental animals against *Leishmania* infections [59–62].

Tbr GP63-1 MAV----------------IMFPRYIIPCLLGLISCGDVTEGNIPPHRDFGKLMKNM--------- 43
Lc GP63-S1 MSVDSSSTHRHRSVAARLVRLAAAAGAAVIAAVGTAAAWAHAGAVQHRJHDAMQARVRQSVARH 64
* *
Tbr GP63-1 --------SMRDLVVDEPPVPKGDLVHAI--VTSSMAGWQPIRFKVFKSDIKNKKYGNVGETRSNFREGIYYKKTESV 113
Lc GP63-S1 HTAPGAVSAVGLDYVTLDTAAAAADRRPGSAPTVVRAANW GALRIAVSTEDLTDPAYHARVGQHIKRLGGVDITAEDI 144
* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
panosome protein with the *Leishmania* and *Crithidia* GP63s is in the region known to be the catalytic site of the zinc protease activity of *Leishmania* GP63. Two histidines and one glutamic acid in this region, which are conserved in the trypanosome protein, have been shown to be essential for the protease activity [64]. These amino acid similarities of the trypanosome protein and the *Leishmania* and *Crithidia* GP63s indicates that it shares structural properties and likely protease activity with GP63s of these other organisms.

Nuclear RNA run on assays indicate that the trypanosome GP63 genes are transcribed to an equal extent in bloodstream and procyclic organisms, whereas Northern blots show that their mRNAs accumulate to a 50-fold higher steady state level in bloodstream trypanosomes than in procyclic organisms. In bloodstream trypanosomes the ratio of mRNAs for GP63 and the VSG is about 1:150, as determined by cDNA library screenings. Since the VSG mRNA represents about 5% of the total mRNA population of bloodstream trypanosomes, about 0.03% of the mRNA encodes GP63.

The function of GP63 in bloodstream trypanosomes has not been determined yet. However, one attractive possibility is that the surface protease activity of GP63 might contribute to evasion of complement-mediated lysis of bloodstream trypanosomes, a function consistent with one of its proposed roles in *Leishmania* [58]. Bloodstream trypanosomes are known to activate the alternative pathway of complement in human serum, but appear not to lyse because the complement cascade does not go beyond the association of C3 convertase with the parasite surface [65]. Procyclic trypanosomes, on the other hand, have a much lower amount of GP63 and are highly susceptible to complement-mediated cytolysis [66]. This model for resistance to complement-mediated lysis by African trypanosomes can be tested by comparing the susceptibility to complement of bloodstream and procyclic trypanosomes transfected with plasmids over expressing GP63 versus that of untransfected organisms. In addition, it will be interesting to see if GP63 is highly expressed during the metacyclic stage of the African trypanosome life cycle, the stage that encounters

the complement system first upon infection from the tsetse fly. In this respect, a preliminary report has occurred of a membrane-bound metalloprotease in infective metacyclic trypomastigotes of *Trypanosoma cruzi* [67], an intracellular parasite that causes Chagas disease in Latin America and must also evade the complement system after transmission from the insect.

## Acknowledgements

The research in our laboratory was supported by NIH grants AI32135 and AI40591. KLH was supported by NIH grants AI09872 and AI07511.

## References

[1] Ross R, Thomson D. A case of sleeping sickness studied by precise enumerative methods: further observations. Proc R Soc London 1910;83:187–205.

[2] Bridgen PJ, Cross GAM, Bridgen J. N-terminal amino acid sequences of variant-specific surface antigens from *Trypanosoma brucei*. Nature 1976;263:613–4.

[3] Williams RO, Young JR, Majiwa PAO. Genomic rearrangements correlated with antigenic variation in *Trypanosoma brucei*. Nature 1979;282:847–9.

[4] Hoeijmakers JH, Frasch AC, Bernards A, Borst P, Cross GA. Novel expression-linked copies of the genes for variant surface antigen in trypanosomes. Nature 1980;284:78–80.

[5] Vickerman K. Developmental cycles and biology of pathogenic trypanosomes. Br Med Bull 1985;41:105–14.

[6] Sileghem M, Flynn JN, Darji A, De Baetselier P, Naessens J. African Trypanosomiasis. In: Kierszenbaum F, editor. Parasite Infections and the Immune System. San Diego: Academic Press, 1994:1–51.

[7] Urquhart GM, Murray M, Murray PK, Jennings FW, Bate E. Immunosuppression in *Trypanosoma brucei* infections in rats and mice. Trans R Soc Trop Med Hyg 1973;67:528–35.

[8] Diffl ey P. Trypanosomal surface coat variant antigen causes polyclonal lymphocyte activation. J Immunol 1983;131:1983–6.

[9] Jokiranta TS, Jokipii L, Meri S. Complement resistance of parasites. Scand J Immunol 1995;42:9–20.

[10] Dempsey WA, Mansfield JM. Lymphocyte function in experimental African trypanosomiasis. V. Role of antibody and the mononuclear phagocyte system in variant-specific immunity. J Immunol 1983;130:405–11.

[11] Esser KM, Schoenbecher MJ, Gingrich JB. *Trypanosoma rhodesiense* blood forms express all antigen

specificities relevant to protection against metacyclic (in-
sect form) challenge. J Immunol 1982;129:1715–8.
[12] Crowe JS, Barry JD, Luckins AG, Ross CA, Vickerman
K. All metacyclic variable antigen types of Trypanosoma
congolense identified using monoclonal antibodies. Na-
ture 1983;306:389–91.
[13] Turner CM, Barry JD, Maudlin I, Vickerman K. An
estimate of the size of the metacyclic variable antigen
repertoire of Trypanosoma brucei rhodesiense. Parasitol-
ogy 1988;97:269–76.
[14] Alarcon CM, Son HJ, Hall T, Donelson JE. A mono-
cistronic transcript for a trypanosome variant surface
glycoprotein. Mol Cell Biol 1994;14:5579–91.
[15] Graham SV, Barry JD. Transcriptional regulation of
metacyclic variant surface glycoprotein gene expression
during the life cycle of Trypanosoma brucei. Mol Cell
Biol 1995;15:5945–56.
[16] Lu Y, Hall T, Gay LS, Donelson JE. Point mutations
are associated with a gene duplication leading to the
bloodstream reexpression of a trypanosome metacyclic
VSG. Cell 1993;72:397–406.
[17] McCulloch R, Rudenko G, Borst P. Gene conversions
mediating antigenic variation in Trypanosoma brucei can
occur in variant surface glycoprotein expression sites
lacking 70-base-pair repeat sequences. Mol Cell Biol
1997;17:833–43.
[18] Nagoshi Lu Y, Alarcon CM, Donelson JE. The putative
promoter for a metacyclic VSG gene in African try-
panosomes. Mol Biochem Parasitol 1995;72:33–45.
[19] Kim KS, Donelson JE. Co-duplication of a variant
surface glycoprotein gene and its promoter to an expres-
sion site in African trypanosomes. J Biol Chem
1997;272:24637–45.
[20] Pays E, Coquelet H, Tebabi P, Pays A, Jefferies D,
Steinert M, Koenig E, Williams RO, Roditi I. Try-
panosoma brucei: constitutive activity of the VSG and
procyclin gene promoters. EMBO J 1990;9:3145–51.
[21] Gottesdiener K, Chung H-M, Brown SD, Lee MG-S,
Van der Ploeg LHT. Characterization of VSG gene
expression site promoters and promoter-associated DNA
rearrangement events. Mol Cell Biol 1991;11:2467–80.
[22] Zomerdijk JCBM, Kieft R, Shiels PG, Borst P. Alpha-
amanitin-resistant transcription units in trypanosomes: a
comparison of promoter sequences for a VSG gene
expression site and for the ribosomal RNA genes. Nu-
cleic Acids Res 1991;19:5153–8.
[23] Lu Y, Alarcon CM, Hall T, Reddy LV, Donelson JE. A
strand bias occurs in point mutations associated with
VSG gene conversion in Trypanosoma rhodesiense. Mol
Cell Biol 1994;14:3971–80.
[24] Donelson JE. Mechanisms of antigenic variation in Bor-
relia hermsii and African trypanosomes. J Biol Chem
1995;270:7783–6.
[25] Kimmel BE, Ole-MoiYoi OK, Young JR. Ingi, a 5.2 kb
dispersed sequence element from Trypanosoma brucei
that carries half of a smaller mobile element at either end
and has homology with mammalian lines. Mol Cell Biol
1987;7:1465–75.

[26] Lodes MJ, Smiley BL, Stadnyk AW, Bennett JL, Myler
PJ, Stuart K. Expression of a retroposon-like sequence
upstream of the putative Trypanosoma brucei variant
surface glycoprotein gene expression site promoter. Mol
Cell Biol 1993;13:7036–44.
[27] Aksoy S, Lalor TM, Martin J, Van der Ploeg LH,
Richards FF. Multiple copies of a retroposon interrupt
spliced leader RNA genes in the African trypanosome,
Trypanosoma gambiense. EMBO J 1987;6:3819–26.
[28] Pays E, Murphy NB. DNA-binding fingers encoded by a
trypanosome retroposon. J Mol Biol 1987;197:147–8.
[29] Morgan RW, El-Sayed NMA, Kepa JK, Pedram M,
Donelson JE. Differential expression of the expression
site-associated gene I family in African trypanosomes. J
Biol Chem 1996;271:9771–7.
[30] Olsson T, Bakhiet M, Hojeberg B, Ljungdahl A, Edlund
C, Andersson G, Ekre H-P, Fung-Leung W-P, Mak T,
Wigzell H, Fiszer U, Kristensson K. CD8 is critically
involved in lymphocyte activation by a T. brucei brucei-
released molecule. Cell 1993;72:715–27.
[31] Bakhiet M, Olsson T, Van der Meide P, Kristensson K.
Depletion of CD8+ T cells suppresses growth of Try-
panosoma brucei brucei and interferon-gamma produc-
tion in infected rats. Clin Exp Immunol 1990;81:195–9.
[32] Olsson T, Bakhiet M, Kristensson K. Interactions be-
tween Trypanosoma brucei and CD8+ T cells. Parasitol
Today 1992;8:237–9.
[33] Olsson T, Bakhiet M, Edlund C, Hojeberg B, Van der
Meide PH, Kristensson K. Bidirectional activating sig-
nals between Trypanosoma brucei and CD8+ T cells: a
trypanosome-released factor triggers interferon-γ pro-
duction that stimulates parasite growth. Eur J Immunol
1991;21:2447–54.
[34] Hua SB, Wang CC. Interferon-gamma activation of a
mitogen-activated protein kinase, KFR1, in the blood-
stream form of Trypanosoma brucei. J Biol Chem
1997;272:10797–803.
[35] Bakhiet M, Olsson T, Edlund C, Hojeberg B, Holmberg
K, Lorentzen J, Kristensson K. A Trypanosoma brucei
brucei-derived factor that triggers CD8+ lymphocytes to
interferon-γ secretion: Purification, characterization and
protective effects in vivo by treatment with a mono-
clonal antibody against the factor. Scand J Immunol
1993;37:165–78.
[36] Bakhiet M, Olsson T, Mhlanga J, Buscher P, Lycke N,
Van der Meide PH, Kristensson K. Human and rodent
interferon-γ as a growth factor for Trypanosoma brucei.
Eur J Immunol 1996;26:1359–64.
[37] Vaidya T, Bakhiet M, Hill KL, Olsson T, Kristensson
K, Donelson JE. The gene for a trypanosome triggering
factor from African trypanosomes. J Exp Med
1997;186:433–8.
[38] Okamura H, Tsutsui H, Komatsu T, Yutsudo M,
Hakura A, Tanimoto T, Torigoe K, Okura T, Nukada
Y, Hattori K, Akita K, Namba M, Tanabe F, Konishi
K, Fukuda S, Kurimoto M. Cloning of a new cytokine
that induces IFN-γ production by T cells. Nature
1995;378:88–91.

[39] McKerrow JH. The proteases and pathogenicity of parasitic protozoa. Annu Rev Microbiol 1993;47:821–53.

[40] Lonsdale-Eccles JD, Mpimbaza WN, Nkhungulu RM, Olobo J, Smith L, Tosomba OM, Grab DJ. Trypanosomatid cysteine protease activity may be enhanced by a kininogen-like moiety from host serum. Biochem J 1995;305:549–56.

[41] Kapur V, Majesky MW, Li L-L, Black RA, Musser JM. Cleavage of interleukin 1β (IL-1β) precursor to produce active IL-1β by a conserved extracellular cysteine protease from *Streptococcus pyogenes*. Proc Natl Acad Sci USA 1993;90:7676–80.

[42] LaMarre J, Wollenberg GK, Gonias SL, Hayes MA. Cytokine binding and clearance properties of proteinase-activated α2-macroglobulins. Lab Invest 1991;65:3–14.

[43] Schatz G, Dobberstein B. Common principles of protein translocation across membranes. Science 1996;271:1519–26.

[44] Hug M, Carruthers VB, Hartmann C, Sherman DS, Cross GAM, Clayton C. A possible role for the 3′-untranslated region in developmental regulation in *Trypanosoma brucei*. Mol Biochem Parasitol 1993;61:87–96.

[45] Cormack BP, Valdivia RH, Falkow S. FACS-optimized mutants of the green fluorescent protein (GFP). Gene 1996;173:33–8.

[46] Muesch A, Hartmann E, Rohde K, Rubartelli A, Sitia R, Rapoport TA. A novel pathway for secretory proteins? TIBS 1990;15:86–8.

[47] Tagaya Y, Maeda Y, Mitsui A, Kondo N, Matsui H, Hamuro J, Brown N, Arai K, Yokota T, Wakasugi H, Yodoi J. ATL-derived factor (ADF), an IL-2 receptor/Tac inducer homologous to thioredoxin; possible involvement of dithiol-reduction in the IL-2 receptor induction. EMBO J 1989;8:757–64.

[48] El-Sayed NMA, Donelson JE. A survey of *Trypanosoma brucei rhodesiense* genome using shotgun sequencing. Mol Biochem Parasitol 1997;84:167–78.

[49] Bordier C. The promastigote surface protease of *Leishmania*. Parasitol Today 1987;3:151–3.

[50] Bouvier J, Schneider P, Etges R. Leishmanolysin: Surface metalloproteinase of *Leishmania*. Methods Enzymol 1995;248:614–33.

[51] Chang K-P, Chaudhuri G. Molecular determinants of *Leishmania* virulence. Annu Rev Microbiol 1990;44:499–529.

[52] Roberts SC, Swihart KG, Agey MW, Ramamoorthy R, Wilson ME, Donelson JE. Sequence diversity and organization of the *msp* gene family encoding gp63 of *Leishmania chagasi*. Mol Biochem Parasitol 1993;62:157–72.

[53] Medina-Acosta E, Beverley SM, Russell DG. Evolution and expression of the *Leishmania* surface proteinase (gp63) gene locus. Infect Agents Dis 1993;2:25–34.

[54] Chang C-S, Chang K-P. Monoclonal antibody affinity purification of a leishmania membrane glycoprotein and its inhibition of *Leishmania*-macrophage binding. Proc Natl Acad Sci USA 1986;83:100–4.

[55] Liu X, Chang K-P. Extrachromosomal genetic complementation of surface metalloproteinase (gp63)-deficient *Leishmania* increases their binding to macrophages. Proc Natl Acad Sci USA 1992;89:4991–5.

[56] Soteriadou KP, Remoundos MS, Katsikas MD, Tzinia AK, Tsikaris V, Sakarellos D, Tzartos SJ. The Ser-Arg-Tyr-Asp region of the major surface glycoprotein of *Leishmania* mimics the Arg-Gly-Asp-Ser cell attachment region of fibronectin. J Biol Chem 1992;267:13980–5.

[57] McGwire B, Chang K-P. Genetic rescue of surface metalloproteinase (gp63)-deficiency in *Leishmania amazonensis* variants increases their infection of macrophages at the early phase. Mol Biochem Parasitol 1994;66:345–7.

[58] Brittingham A, Morrison CJ, McMaster WR, McGwire BS, Chang K-P, Mosser DM. Role of the *Leishmania* surface protease gp63 in complement fixation, cell adhesion, and resistance to complement-mediated lysis. J Immunol 1995;155:3102–11.

[59] Olobo JO, Anjili CO, Gicheru MM, Mbati PA, Kariuki TM, Githure JI, Koech DK, McMaster WR. Vaccination of vervet monkeys against cutaneous leishmaniosis using recombinant *Leishmania* 'major surface glycoprotein' (gp63). Vet Parasitol 1995;60:199–212.

[60] Gheorghiu M, Dellagi K, Gicquel B. Recombinant BCG expressing the leishmania surface antigen Gp63 induces protective immunity against *Leishmania major* infection in BALB/c mice. Microbiology 1995;141(Pt 7):1585–92.

[61] Xu D, McSorley SJ, Chatfield SN, Dougan G, Liew FY. Protection against *Leishmania major* infection in genetically susceptible BALB/c mice by gp63 delivered orally in attenuated *Salmonella typhimurium* (AroA- AroD-). Immunology 1995;85:1–7.

[62] Connell ND, Medina-Acosta E, McMaster WR, Bloom B, Russell DG. Effective immunization against cutaneous leishmaniasis with recombinant bacille Calmette-Guerin expressing the *Leishmania* surface proteinase gp63. Proc Natl Acad Sci USA 1993;90:11473–7.

[63] Inverso JA, Medina-Acosta E, O'Connor J, Russell DG, Cross GAM. *Crithidia fasciculata* contains a transcribed leishmanial surface proteinase (gp63) gene homologue. Mol Biochem Parasitol 1993;57:47–54.

[64] McGwire BS, Chang K-P. Posttranslational regulation of a *Leishmania* HEXXH metalloprotease (gp63). J Biol Chem 1996;271:7903–9.

[65] Devine DV, Falk RJ, Balber AE. Restriction of the alternative pathway of human complement by intact *Trypanosoma brucei* subsp. *gambiense*. Infect Immun 1986;52:223–9.

[66] Mosser DM, Roberts JF. *Trypanosoma brucei*: Recognition in vitro of two developmental forms by murine macrophages. Exp Parasitol 1982;54:310–6.

[67] Salles JM, Thomas N, Carreira Fragoso MA, Goldenberg S. Purification of a metalloproteinase constitutively expressed during *Trypanosoma cruzi* metacyclogenesis. Mem Inst Oswaldo Cruz 1996;91:269.

[68] Ramamoorthy R, Donelson JE, Maybodi M, Panitz S, Paetz KE, Wilson ME. Three distinct RNA species code for the major surface protease (gp63) of Leishmania donovani chagasi. J Biol Chem 1992;267:1888–95.

[69] Pepin J, Donelson JE. Human African trypanosomiasis. In: Guerrant RL, Krogstad DJ, Maguire JH,

Walker KH, Weller P, editors. Tropical Infectious Diseases: Principles, Pathogens and Practice. New York: Churchill Livingstone (in press)

[70] El-Sayed NMA, Donelson JE. African trypanosomes have differentially expressed genes encoding homologues of the Leishmania GP63 surface protease. J Biol Chem 1997;272:26742–8.
