
ENZYMIOLOGY:

Characterization of the N-Acetylglucosaminyltransferase Activity Involved in the Biosynthesis of the Staphylococcus epidermidis Polysaccharide Intercellular Adhesin

Christian Gerke, Angelika Kraft, Roderich Süssmuth, Oliver Schweitzer and Friedrich Götz

J. Biol. Chem. 1998, 273:18586-18593.

doi: 10.1074/jbc.273.29.18586

---

Access the most updated version of this article at [http://www.jbc.org/content/273/29/18586](http://www.jbc.org/content/273/29/18586)

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 43 references, 25 of which can be accessed free at [http://www.jbc.org/content/273/29/18586.full.html#ref-list-1](http://www.jbc.org/content/273/29/18586.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 1998 by The American Society for Biochemistry and Molecular Biology, Inc.

**Characterization of the N-Acetylglycosaminyltransferase Activity Involved in the Biosynthesis of the Staphylococcus epidermidis Polysaccharide Intercellular Adhesin**

(Received for publication, November 26, 1997, and in revised form, April 23, 1998)

Christianne Gerke‡, Angelika Kraft§, Roderich Süßmuth¶, Oliver Schweitzer‡, and Friedrich Götz‡

From the ‡Mikrobielle Genetik, Universität Tübingen, Waldhäuser Strasse 70/8, the §Max-Planck-Institut für Entwicklungsbiologie, Abteilung Biochemie, Spemannstrasse 35, and the ¶Institut für Organische Chemie, Universität Tübingen, Auf der Morgenstelle 18, 72076 Tübingen, Germany

The polysaccharide intercellular adhesin (PIA) is an important factor in the colonization of medical devices by *Staphylococcus epidermidis*. The genes encoding PIA production are organized in the *icaADBC* (intercellular adhesion) operon. To study the function of the individual genes, we have established an *in vitro* assay with UDP-N-acetylglucosamine, the substrate for PIA biosynthesis, and analyzed the products by thin-layer chromatography and mass spectrometry. IcaA alone exhibited a low N-acetylglycosaminyltransferase activity and represents the catalytic enzyme. Coexpression of *icaA* with *icaD* led to a significant increase in activity. The newly identified *icaD* gene is located between *icaA* and *icaB* and overlaps both genes. N-Acetylglucosamine oligomers produced by IcaAD reached a maximal length of 20 residues. Only when *icaA* and *icaD* were expressed together with *icaC* were oligomer chains that react with PIA-specific antiserum synthesized. IcaA and IcaD are located in the cytoplasmic membrane, and IcaC also has all the structural features of an integral membrane protein. These results indicate a close interaction between IcaA, IcaD, and IcaC. Tunicamycin and bacitracin did not affect the *in vitro* synthesis of PIA intermediates or the complete PIA biosynthesis *in vivo*, suggesting that a undecaprenyl phosphate carrier is not involved. IcaAD represents a novel protein combination among β-glycosyltransferases.

contribute to primary attachment of *S.* epidermidis cells to a polymer surface include unspecific hydrophobic interactions (5, 6), a capsular polysaccharide/adhesin (PS/A) (7, 8), proteinaceous cell-surface antigens (SSP-1 and SSP-2) (9, 10), and the autolysin AtlE identified by our group (11).

A characteristic feature of the second phase of biofilm formation is intercellular adhesion, which results in the formation of large cell clusters by clinical *S.* epidermidis strains. This reaction is associated with the production of the polysaccharide intercellular adhesin (PIA)¹ located at the cell surface (12). PIA consists of two structurally related homoglycans, polysaccharides I and II, composed of at least 130 2-deoxy-2-amino-D-glucopyranosyl residues that are mostly (>80%) N-acetylated. The residues are β-1,6-linked (13), a type of linkage for N-acetylglucosamine polymers that has not been previously described. In addition, a 140-kDa extracellular protein that is missing in an accumulation-negative mutant of *S.* epidermidis RP62A is thought to contribute to intercellular adhesion (14, 15).

Recently, we cloned and sequenced three *S.* epidermidis RP62A genes (*icaABC*) that are involved in intercellular adhesion and PIA production (16). Transposon insertion mutants in the *ica* operon of *S.* epidermidis O-47 (class B mutants) (17, 18) are unable to form a biofilm on various surfaces. It has also been shown that the majority of *S.* epidermidis strains isolated from septicemic patients were biofilm-positive, whereas skin and mucosal isolates were not, even though the genome of the biofilm-forming strains contained the *ica* genes (19).

Here, we report the identification of a fourth gene (*icaD*) that is located within the *ica* operon and that is necessary for PIA synthesis. Furthermore, we demonstrate that IcaA and IcaD are located in the membrane and together exhibit N-acetylglycosaminyltransferase activity, in which IcaA represents the catalytic enzyme that requires IcaD for full activity. Finally, we show that in addition to *icaA* and *icaD*, *icaC* is essential for long-chain PIA synthesis.

EXPERIMENTAL PROCEDURES

Bacterial Strains and Growth Conditions—DNA containing *ica* was isolated from PIA-producing and biofilm-positive *S.* epidermidis RP62A (ATCC 35984). The *Staphylococcus carnosus* wild-type strain TM300 (20) was used as cloning host for all staphylococcal plasmids and for heterologous expression of *ica* genes. Cloning in *Escherichia coli* and the production of maltose-binding protein fusions were performed in *E.* coli strain TB1 (ara Δ(*lac proAB*) *rpsL* (f80 *lacZΔM15*) *hsdR*) obtained from New England Biolabs (Schwalbach, Germany). Staphylococci were

* This work was supported by Grant BMBF 01KI9451/7 and by a Deutsche Forschungsgemeinschaft Graduiertenkolleg Mikrobiologie fellowship (to C. G.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* || To whom correspondence should be addressed. Tel.: 49-7071-74636; Fax: 49-7071-5937; E-mail: friedrich.goetz@uni-tuebingen.de.

1 The abbreviations used are: PIA, polysaccharide intercellular adhesin; MBP, maltose-binding protein; HPTLC, high performance thin-layer chromatography; HPLC, high performance liquid chromatography; MS, mass spectrometry.
grown in tryptic soy broth (Difco, Augsburg, Germany) or in LB medium (1% Tryptone (Difco), 0.5% yeast extract (Difco), and 0.5% NaCl), which was also used to cultivate *E. coli*. When appropriate, the media were supplemented with tetracycline (10 μg/ml), chloramphenicol (10 μg/ml), or ampicillin (100 μg/ml).

**DNA Techniques and Transformation—** DNA manipulations, plasmid DNA isolation, and transformation of *E. coli* were performed using standard procedures (21). To isolate DNA from staphylococci, cells were lysed by adding 0.1 mg/ml lysostaphin (Sigma, Deisenhofen, Germany), and the alkaline lysis method was subsequently applied. Chromosomal staphylococcal DNA was prepared according to the procedure of Mar-mur (22). *S. carnosus* was transformed with plasmid DNA by protoplast transformation (23). Polymerase chain reaction experiments were performed with Vent DNA polymerase (New England Biolabs) as recommended by the manufacturer. DNA sequences were determined using the Sequenase protocol (Version 2.0, Amersham, Braunschweig, Germany) or a LI-COR DNA sequencer (MWG-Biotech, Ebersberg, Germany). The primers for polymerase chain reaction were as follows: primer 4, ATGCATGTATTTAACTTTTACTTTTC; primer 5, TCGACGCGTG-ATATAGTAATAAATAAGCTCTCCC; primer 18, CGGGATCCAGGAG-GTGAAAAATGCA GTATT; primer 23, TCGACGCGTTTGAAAGG-TTTCATATGTCACG; primer 25, CGGGATCCAAGAAAAGGTTGG-CTATGC; primer 33, GAAGATCTCATGGTCAAGCCCAGA; primer 34, GAAGATCTCATCAATGCGCTAATAAAG; primer S1, AAAAGGCGC-CGGGACTTTTTTATTAATTCCAGTTAGGC; primer S2, TTTTAGA-TCTCGCGTTTCAAAAATATAAGAAAGGTCGTG; primer S8, AATAA-GGGACGCGTTTTATTAATTCCAG; primer S12, GCTTTAGATCTTA-ACTAACGAAAGGTAGG; and primer S15, CGATGATTATGACACAA-ATAAACACGCGTAAATACAC. Restriction sites are underlined.

**Vectors and Plasmid Constructions—** In this study, the *ica* genes were amplified by polymerase chain reaction from chromosomal DNA of *S. epidermidis* RP62A and cloned independently or in various combinations in the inducible staphylococcal expression vectors pTX15 (24) and pCX19, a derivative of pCX15 (25). Gene expression was induced by adding 0.5% xylose (final concentration) to the growth medium (LB medium) at $A_{578 \mathrm{~nm}} = 0.4$ and further incubation of the strains for 6 h. The restriction sites used for the construction of the following plasmids are indicated in parentheses: pTXicaA carries *icaA* amplified using primers 18 and 5 (*BamHI-MluI*); pCXicaD contains *icaD* amplified using primers 25 and 23 (*BamHI-SmaI*); pTXicaAD harbors *icaA* and *icaD* amplified using primers 18 and 23 (*BamHI-MluI*); pTXicaADBC contains the entire *ica* operon amplified using primers S12 and S8 (*BglII-MluI*); pTXicaADB carries *icaA*, *icaD*, and *icaB* amplified using primers S12 and S15 (*BglII-MluI*); and pTXicaBC contains *icaB* and *icaC* amplified using primers S2 and S1 (*BglII-NarI*). pTXicaADC was obtained by generating a frameshift deletion mutation of *icaB* in pTX-icaADBC as described for the inactivation of *icaB* in pCN27 (16). The plasmid pTX16 (24) was used as a negative control.

Fusions of *icaA* and *icaD* with *malE* were constructed using the vector pIH902 (New England Biolabs), a derivative of pMAL-c2. The *icaA* gene was amplified using primers 4 and 5, cleaved with AccI, treated with Klenow enzyme, and inserted into the *StuI* site of pIH902, yielding the plasmid pIHicaA300, encoding the first 306 amino acids of IcaA. pIHicaD carries *icaD* amplified using primers 33 and 34 and blunt end-inserted into the *StuI* site of pIH902.

**Preparation of Cell Extracts and Membranes—** Staphylococcus and *E. coli* cultures were harvested by centrifugation, and the cell pellets were resuspended in buffer A (50 mM Tris-HCl, pH 7.5, 10 mM MgCl₂, and 4 mM dithiothreitol; 2 μl/mg of cell wet weight). The cells were disrupted by 3 × 1-min vortexing in Corex tubes with glass beads (diameter of 0.15–0.30 mm; 4.5 times the weight of the cell pellet). DNase I (20 μg/ml; Boehringer Mannheim, Mannheim, Germany) was added before breaking the cells. Unbroken cells and glass beads were sedimented (10 min, 2000 × g), and the supernatant was saved. The procedure was repeated one to three times; all supernatants were combined. Mem-branes were sedimented from the crude extract by ultracentrifugation (20 min, 200,000 × g) and resuspended in buffer A at a protein concentration of 5 mg/ml (5-fold concentration of the membrane proteins over the crude extract). The collected membranes were called “crude membranes.” For further purification, the crude membranes were extracted with 2% (w/v) Triton X-100 (in buffer A) for 2 h with gentle shaking, sedimented again, washed once with buffer A, and resuspended in the same volume of buffer A as the crude membranes. These preparations were designated “Triton-extracted membranes.” The membranes were stored at −70 °C. Protein concentrations were determined by the method of Bradford (47).

**Preparation of Anti-Ica and Anti-IcaD Antisera—** *E. coli* TB1 carrying pIHicaA300 or pIHicaD was used to express the *malE* gene fusions.

The soluble maltose-binding protein (MBP) fusion protein MBP-IcaA300 was purified according to Ref. 45. The very poorly produced fusion protein MBP-IcaD was located in the membrane fraction, from where it was solubilized as described above using 2% (w/v) Triton X-100 and 500 mM NaCl. The MBP-IcaD-containing fraction was diluted to a final Triton X-100 concentration of 0.5% and incubated for 12 h with amylose resin equilibrated with column buffer (50 mM Tris-HCl, pH 7.5, 500 mM NaCl, and 10 mM EDTA) containing 0.5% Triton X-100. The amylose material was collected by centrifugation (10 min, 2000 × g) and washed. MBP-IcaD was eluted with 10 mM maltose in column buffer. Rabbits were immunized with the purified MBP fusion proteins at Eurogentech (Seraing, Belgium) according to their standard immunization program. The antiserum obtained against MBP-IcaA300 was designated anti-IcaA; the antiserum obtained against MBP-IcaD was designated anti-IcaD.

**Immunoblot Analyses of Proteins and PIA Production—** To block unspecific binding of the antisera and secondary conjugates used in immunoblot analyses, these were diluted 1:250 in Tris-buffered saline, 0.1% bovine serum albumin, and 1 mM sodium azide and incubated with a crude extract from *S. carnosus* pTX16 (final protein concentration of 0.5 mg/ml). After 12 h of gentle shaking, precipitated material was sedimented by centrifugation (30 min, 30,000 × g).

Proteins were separated by SDS-polyacrylamide gel electrophoresis according to Laemmli (46) and either stained with Coomassie Brilliant Blue R-250 (Sigma) or electrophoretically transferred to a nitrocellulose membrane (BA83, Schleicher & Schuell, Dassel, Germany) using a semidry apparatus. IcaA was detected by the anti-IcaA antiserum and IcaD by the anti-IcaD antiserum (1:1000). Subsequently, the nitrocel-lulose membranes were incubated with anti-rabbit immunoglobulin G-biotin conjugate (1:5000; Sigma) and with streptavidin-horseradish peroxidase complex (1:5000; Amersham). Immunoreactive proteins were visualized by enhanced chemiluminescence (ECL, Amersham) with Fuji HR-E30 films. Reprobing of a blot with another primary antiserum was performed according to the ECL Western blotting protocol.

PIA production *in vivo* was analyzed in crude extracts and in cell-surface extracts obtained by incubating the cells in 0.5 M EDTA, pH 8.0, for 5 min at 100 °C. For production experiments in the presence of antibiotics, tunicamycin (5–20 μg/ml) and bacitracin (50–400 μg/ml) were added to cultures of *S. carnosus* pTXicaADBC at the time of induction. PIA production *in vitro* was determined in the enzymatic assays. Extracts (1 μl each) were applied to a nitrocellulose membrane (BA83) using the Bio-Dot Microfiltration apparatus (Bio-Rad, München, Germany). PIA was detected with the PIA-specific antiserum (1:5000) (16). Bound PIA antibodies were visualized by anti-rabbit immunoglobulin G-alkaline phosphatase conjugate (1:5000; Sigma) and color reaction.

**In Vitro Enzymatic Assay (N-Acetylglucosaminyltransferase Assay)—** *In vitro* reactions to analyze glycosyltransferase activity were performed by incubating crude extracts (see above) with 0.4 mM UDP-*N*-acetylglucosamine or crude membranes or Triton-extracted membranes (see above) with 2 mM UDP-*N*-acetylglucosamine. *In vitro* synthesis of peptidoglycan was repressed by adding 50 μg/ml D-cycloserine (26). For radiolabeling, 10 μM UDP-*N*-acetyl-β-[U-¹⁴C]glucosamine (final radioactive concentration of 84 Bq (2.3 nCi)/μl of reaction volume; Amersharm) was added. Analytical reactions were carried out in a total volume of 10–50 μl. Reaction mixtures were incubated for 12 h at 20 °C. To test other substrates, UDP-glucose, UDP-galactose, and UDP-glucuronic acid (¹⁴C-labeled substrates from Amersham) were used in equimolar amounts. For experiments in the presence of tunicamycin and bacitracin, the antibiotics were added in the above-mentioned concentrations.

**Analytical Methods—** For thin-layer chromatography, in each case, 1 μl of *in vitro* reaction assays containing ¹⁴C-labeled UDP-*N*-acetylglucosamine was applied to NH₂-HPTLC plates (Merck, Darmstadt, Germany). The chromatogram was developed twice using acetonitrile/water (60:40 or 65:35, v/v). Radioactive spots were detected by phosphoimaging with a Fuji BAS 1500 (Raytest, Straubenhardt, Germany) or by autoradiography. Fuji HR-E30 films were exposed for 6–10 weeks. As a reference compound, *N*-acetyl-β-[1-¹⁴C]glucosamine (25 Bq) or unlabeled *N*-acetylglucosamine (10 μg) was used. If appropriate, 0.005% Triton X-100 was added. The unlabeled standard was visualized by spraying with 0.2% orcinol in H₂SO₄ and subsequent heating at 100 °C for 15 min.

For HPLC analysis of the *in vitro* synthesized products, the pH of the reaction assays was lowered to 4.5 with phosphoric acid; the samples were heated for 5 min at 100 °C; and the precipitated material was sedimented by centrifugation (10 min, 13,000 × g). Acetonitrile was added to the supernatant to a final concentration of 75%. The products
18588

**N-Acetylglucosaminyltransferase in PIA Biosynthesis**

![Diagram](https://i.imgur.com/yourimage.png)

FIG. 1. Organization of *icaD* in the *ica* operon of *S. epidermidis*. *icaD* is located between *icaA* and *icaB*. *icaD* overlaps with the 3′-end of *icaA* by 37 nucleotides and with the start of *icaB* by 4 nucleotides. The DNA sequence of the overlapping region of *icaA* and *icaD* is shown. The putative start codon and ribosomal binding site (Shine-Dalgarno (*SD*)) of *icaD* are indicated by boldface letters; the stop codon of *icaA* is underlined. The numbering refers to the sequence of the *ica* operon available from GenBank™ under accession number U43366. Relevant restriction sites are indicated. The *BseAI* site was used to introduce a frameshift deletion mutation in *icaD*. kb, kilobase.

were separated on a Nucleosil 120-7-NH₂-HPLC column (4.6 × 250 mm; Bischoff, Leonberg, Germany) and eluted with an acetonitrile gradient in pure water from 75 to 50% acetonitrile over 45 min at a flow rate of 1 ml/min. Subsequently, the column was washed for 10 min with pure water. Elution was monitored by determining A₂₀₅ₙₘ. Radiolabeled products were detected by measuring ¹⁴C using a flow counter (Packard Instrument Co.).

Electrospray mass spectra were recorded on a triple-quadrupole mass spectrometer Model API III (mass range of *m/z* 10–2400) equipped with a pneumatic supplied electrospray (“ion spray”) interface (Sciex, Thornhill, Ontario, Canada). The mass spectrometer was operated in positive ion mode under unit mass resolution conditions for all determinations. The potential of the spray needle was kept at 4.8 kV; the orifice voltage was +80 V. For mass calibration, a solution of polypropylene glycol was used. Unlabeled IcaAD-dependent products synthesized *in vitro* were applied to the mass spectrometer by HPLC/MS coupling injecting 100 μl/min into the mass spectrometer.

---

### RESULTS

#### Identification and Characterization of *icaD*
To analyze the biosynthesis of the N-acetylg glucosamine polymer PIA, the *ica* genes were expressed under the control of the xylose-inducible promoter of the pTX15 expression vector (24) independently and in various combinations in *S. carnosus*. Glycosyltransferase activity was studied by incubating crude extracts or cell fractions of these clones with UDP-N-acetylg lucosamine, the putative substrate for N-acetylg lucosamine oligomer synthesis. In these experiments, evident transferase activity was detected when *icaA* and the intergenic region between *icaA* and *icaB* were expressed in *S. carnosus*. This region harbors a small open reading frame whose relevance was, until then, unclear. The gene was designated *icaD* (Fig. 1).

The *icaD* gene is 306 nucleotides long and is transcribed in the same direction as the *icaABC* genes. The start of *icaD* overlaps with the end of *icaA* by 37 nucleotides (Fig. 1), and the end of *icaD* overlaps with the start of *icaB* by 4 nucleotides. The deduced polypeptide of *icaD* consists of 101 amino acids with a calculated molecular mass of 11,953 Da and a pI of 9.88. IcaD contains 54.5% hydrophobic amino acids, and two transmembrane helices are predicted. Similarity searches in data bases revealed no sequence similarity of IcaD to known proteins. Inactivation of *icaD* in a plasmid containing the complete *ica* operon led to the loss of cell aggregation and PIA production in *S. carnosus* (data not shown), indicating that, in addition to *icaABC*, *icaD* plays an essential role in intercellular adhesion and PIA production *in vivo*.

#### Localization of IcaA and IcaD
To determine the localization of IcaA and IcaD, antisera against maltose-binding protein fusions of IcaA and IcaD were raised in rabbits. By Western blot analyses, the presence of IcaA and IcaD was analyzed in various cell fractions from induced *S. carnosus* pTXicaAD cells. For detailed studies, the membrane fraction was divided into

crude membranes (membranes derived from the crude extract by ultracentrifugation) and Triton-extracted membranes (crude membranes purified by Triton X-100 extraction). The anti-IcaA antiserum reacted with a 38-kDa protein present in the crude extract, the crude membranes, and the Triton-extracted membranes, but not with any protein in the cytosolic fraction or the soluble fraction obtained from Triton X-100 extraction (Fig. 2A). No reaction of the antiserum was observed with Triton-extracted membranes from uninduced *S. carnosus* pTXicaAD or from *S. carnosus* pTX16 (24) carrying the vector alone, indicating that the reaction of the antiserum was IcaA-specific. The IcaD-specific antiserum reacted with a 34-kDa protein that was present in the same fractions as IcaA (Fig. 2B). Thus, both IcaA and IcaD are membrane-bound. IcaA and IcaD exhibited very similar apparent molecular masses in the SDS-polyacrylamide gel electrophoresis analyses. We therefore reprobed the blot that was analyzed with the IcaD-specific antiserum with the anti-IcaA antiserum to confirm that IcaA and IcaD are independent proteins. The result (Fig. 2C) clearly showed that the two antisera recognize different proteins.

#### In Vitro Synthesis and Characterization of PIA Oligomers
The glycosyltransferase activity of IcaAD was analyzed in crude extracts and membranes from *S. carnosus* pTXicaAD. As a negative control, extracts from *S. carnosus* pTX16 were used. Products synthesized *in vitro* from UDP-N-acetylg lucosamine (5–25% of the applied substrate was ¹⁴C-labeled) were analyzed by HPTLC on NH₂-HPTLC plates (Fig. 3). With crude extract, crude membranes, and Triton-extracted membranes from induced *S. carnosus* pTXicaAD, but not with the cytosolic fraction or the soluble fraction obtained from Triton X-100 extraction, radiolabeled products were formed that were separated in a ladder-like series. This ladder strongly suggested that the products represent oligomers with an increasing number of residues. No such products were synthesized with extracts from *S. carnosus* pTX16 cells (Fig. 3) or extracts from uninduced *S. carnosus* pTXicaAD cells (data not shown), indicating that the observed transferase activity was due to *icaA* and *icaD*. When UDP-glucose, UDP-galactose, and UDP-glucuronic acid, which often function as substrates for the biosynthesis of exopolymers, were tested, we obtained no ladder of products (data not shown).

The migration rates of the products synthesized with crude membranes and Triton-extracted membranes were different. Since the migration rate of the reference compound N-acetylglucosamine differed as well after addition of Triton X-100 (0.005%, w/v), as indicated in Fig. 3, the deviation is probably due to the Triton X-100 remaining in the Triton-extracted membrane preparation. These data were supported by mass spectrometric analyses, in which the oligomers synthesized

N-Acetylglucosaminyltransferase in PIA Biosynthesis

A
CE C M TM TM<sup>n</sup> TS TM<sup>h</sup> TM
kDa
120-
76-
46-
32-
26-
pTXicaAD
pTX16
M
TM
- 1 — GlcNAc (T)
- 2 → 2-mer
- 3 → 3-mer
- 4 → 4-mer
- 5 → 5-mer
- 6 → 6-mer
- 7 → 7-mer
- 8 → 8-mer
9
10
11
12
B
C
CE C M TM TM<sup>n</sup>
kDa
120-
76-
46-
32-
26-
18-
pTXicaAD
IcaA →
← IcaD →
pTXicaAD
CE M
GlcnAc — 2 -
(GlcNAc)<sub>2</sub> ← 4 -
(GlcNAc)<sub>3</sub> ← 6 -
(GlcNAc)<sub>4</sub> ← 8 -
(GlcNAc)<sub>5</sub> ← 9 -
(GlcNAc)<sub>6</sub> ← 10 -
(GlcNAc)<sub>7</sub> ← 11 -
(GlcNAc)<sub>8</sub> ← 12 -
AD - AD - AD -
CE M TM

FIG. 2. Western blot analyses of IcaA and IcaD in various cell fractions using the anti-IcaA serum (A) and the anti-IcaD serum (B) and superimposition of the blot shown in B stripped and reprobed with the anti-IcaA serum (C). The presence of IcaA and IcaD (indicated by arrows) was analyzed in the following extracts from induced (unless otherwise stated) cells of S. carnosus pTXicaAD: lane CE, crude extract (1 μg); lane C, cytoplasmic fraction (1 μg); lane M, crude membranes (0.5 μg); lane TM, Triton-extracted membranes (0.5 μg); lane TM<sup>n</sup>, Triton-extracted membranes of uninduced cells (0.5 μg); lane TS, soluble fraction obtained from Triton extraction (same volume as Triton-extracted membrane fraction); lane TM<sup>h</sup>, Triton-extracted membranes heated for 3 min at 100 °C (0.5 μg) before loading on gel (all other samples were not heated). Triton-extracted membranes of S. carnosus pTX16 were applied as a negative control (0.5 μg) (lane pTX16 TM). For separation of the proteins, 11% acrylamide were used in A and 12.5% in B and C. The localization and sizes of the marker proteins are indicated on the left.

with Triton-extracted membranes were found to be associated with Triton X-100 molecules (data not shown).

The mass spectrometric analyses were performed to further characterize the in vitro synthesized reaction products. For these analyses, the products were separated by HPLC. The elution profile of the IcaAD-dependent products from Triton-extracted membranes, recorded by analyzing 10 μl of radiolabeled assay mixtures, is shown in Fig. 4. HPLC separation was performed on an NH<sub>2</sub>-HPLC column using an acetonitrile gradient. The HPLC separation system was comparable to the NH<sub>2</sub>-HPTLC system. By isolating the eluted compounds and analyzing them again by HPTLC, we found that the succession of the eluted IcaAD-dependent products was the same in HPLC and HPTLC analyses (data not shown). The numbers of the spots in the high performance thin-layer chromatogram (Fig. 3) and the peaks in the high performance liquid chromatogram (Fig. 4) correspond to identical substances.

In the electrospray mass spectrometric analyses of nonradioactive products synthesized with Triton-extracted membranes, spectra of the substances eluting as peaks 1–8 were recorded. They revealed molecular ions at m/z 204, 407, 610, 813, 1016, 1219, 1422, and 1625 (Fig. 4). These values correspond to the [M + H]<sup>+</sup> pseudo-molecular ions of a monomer to octamer of N-acetylglucosamine lacking one water residue/sugar chain since the displayed ions were 18 mass units smaller than the calculated [M + H]<sup>+</sup> pseudo-molecular ions of the fully hydrated N-acetylglucosamine monomer to octamer (m/z 222, 425, 628, 831, 1034, 1237, 1440, and 1643).

The mass spectrometric analyses of the products synthesized with crude membranes revealed that different kinds of oligomers were formed with this membrane preparation and that the TLC and HPLC systems used were not capable of separating all of the formed products. In the analyses of the products M2, M4, M6, and M8–12 (Fig. 3), ions were obtained that corresponded to two different N-acetylglucosamine oligomers. The ions at m/z 204, 407, 610, 813, 1016, 1219, 1422, and 1625 represent the [M + H]<sup>+</sup> pseudo-molecular ions of the putative anhydrous monomer to octamer of N-acetylglucosamine that were also formed with Triton-extracted membranes; the ions at m/z 222, 425, 628, 831, 1034, 1237, 1440, and 1643 correspond to the fully hydrated N-acetylglucosamine monomer to octamer. The ladder of these products is indicated in Figs. 3 and 6. The ion of product M1 was detected at m/z 325, and in substances M3–9, additional ions at m/z 528, 731, 934, 1137, 1340, 1543, and 1746 were detected. These ions probably represent the [M + H]<sup>+</sup> pseudo-molecular ions of the monomer to octamer of N-acetylglucosamine carrying a modification with a mass of 103. None of the MS analyses of the in vitro products from

FIG. 3. TLC of in vitro synthesized oligomers derived from UDP-N-acetylglucosamine. Synthesis of radiolabeled oligomers was analyzed in crude extracts (CE), crude membranes (M), and Triton-extracted membranes (TM) from induced S. carnosus pTXicaAD (AD) or the negative control S. carnosus pTX16 (-). Reaction assays were analyzed on an NH<sub>2</sub>-HPTLC plate. The results were visualized by autoradiography. The unreacted UDP-N-acetylglucosamine of the reaction assays is contained in the smeared spots directly above the origin. The substances produced by the crude membranes and the Triton-extracted membranes are numbered, and the degree of oligomerization of the products, as determined by mass spectrometry, is indicated. 2-mer to 8-mer on the right represent the dimer to octamer of N-acetylglucosamine lacking one water residue/oligosaccharide. The products marked with (GlcNAc)<sub>2</sub> to (GlcNAc)<sub>8</sub> on the left contain the fully hydrated dimer to octamer of N-acetylglucosamine and the corresponding oligomer lacking one water residue. The position of N-acetylglucosamine, solubilized in pure water (GlcNAc) or containing 0.005% (w/v) Triton X-100 (GlcNAc (T)) is indicated by bars. The arrowhead indicates the origin of TLC.

18590 N-Acetylglucosaminyltransferase in PIA Biosynthesis

FIG. 4. ${ }^{14} \mathrm{C}$ HPLC elution profile and MS analysis of IcaAD-dependent products synthesized by Triton-extracted membranes. In vitro synthesis was performed using Triton-extracted membranes of induced $S$. carnosus pTX-icaAD. Prepurified radiolabeled products were loaded onto a Nucleosil $\mathrm{NH}_{2}$-HPTLC column and eluted with an acetonitrile/water gradient. To specifically detect IcaAD-dependent products, the elution was followed by measuring ${ }^{14} \mathrm{C}$ using a flow counter. The numbered elution peaks and the substances they represent correspond to the numbered TLC spots in Fig. 3. By HPLC/MS analysis of unlabeled reaction products, ions of the substances eluting in peaks 1–8 were displayed at $m / z$ 204, 407, 610, 813, 1016, 1219, 1422, and 1625, respectively. These values correspond to the [M + H]${ }^{+}$ pseudo-molecular ions of the monomer to octamer of N-acetylglucosamine lacking one water residue ($M_{r} 18$)/oligomer ([M + H]${ }^{+}$ of the fully hydrated oligomers: 222, 425, 628, 831, 1034, 1237, 1440, and 1643).

crude membranes and Triton-extracted membranes revealed deacetylated N-acetylglucosamine residues, which compose 15–20% of the major polysaccharide I of PIA (13).

Bacterial exopolysaccharides are often synthesized on an undecaprenyl phosphate lipid carrier (27). We therefore investigated whether a lipid carrier is involved in the synthesis of the IcaAD metabolites. Attempts to isolate lipid-linked intermediates in vitro or in vivo according to the method of Bligh and Dyer (28) failed. Furthermore, the antibiotics tunicamycin and bacitracin, which affect undecaprenyl phosphate-mediated syntheses, had no effect on either in vitro oligomer synthesis or in vivo PIA production (data not shown).

Analysis of the Individual Functions of icaA and icaD—To dissect the individual functions of IcaA and IcaD in N-acetylglucosaminyltransferase activity, the corresponding genes were inducibly expressed in $S$. carnosus using two compatible plasmids. icaA was expressed from pTXicaA, and icaD from pCXicaD. N-Acetylglucosaminyltransferase assays performed with Triton-extracted membranes of induced cells (Fig. 5) showed that IcaD alone has no transferase activity, whereas IcaA alone has very low activity. A similar low activity was obtained when the membrane preparations from $S$. carnosus pTXicaA and $S$. carnosus pCXicaD were combined. However, when icaA and icaD were expressed in trans in $S$. carnosus pTXicaA + pCXicaD, transferase activity was increased, although to a lower extent than by cis-expressed icaA and icaD in $S$. carnosus pTXicaAD.

In Vitro Analysis of the Effect of icaB and icaC on the Synthesis of the PIA Sugar Chain—The oligosaccharides synthesized by IcaAD have a maximal chain length of 20 residues as determined by TLC analysis. Full-length PIA, however, consists of at least 130 residues (13). To determine whether the additional Ica proteins IcaB and IcaC have an effect on the polymerization degree of the synthesized oligomers, extracts of $S$. carnosus expressing icaC and/or icaB in addition to icaAD from xylose-inducible vectors were analyzed for transferase activity. In these assays (Fig. 6), the expression of icaADC resulted in the synthesis of new radiolabeled products that did not migrate on TLC plates and that were formed in addition to the oligomers from IcaAD, which are less concentrated in the presence of IcaC. In the absence of IcaAD, IcaC had no transferase activity. IcaB produced no detectable effect in any combination with other Ica proteins. The nonmigrating IcaADC-dependent products reacted with the PIA-specific antiserum, in contrast to the products of IcaAD. We therefore surmise that the IcaADC-dependent products represent sugar chains with a higher degree of polymerization than those produced by IcaAD alone. These products were not synthesized when membrane preparations of different $S$. carnosus clones expressing either icaAD or icaC were mixed.

DISCUSSION

In previous work (16), we showed that the ica operon is responsible for the production of PIA. Since PIA consists of a β-1,6-linked homoglycan composed of N-acetylglucosamine (13), we expected activated N-acetylglucosamine to be a substrate for the glycosyltransferase reaction in PIA biosynthesis. Indeed, with UDP-N-acetylglucosamine, we obtained oligomer synthesizing activity in vitro with crude extracts and membranes of an $S$. carnosus clone expressing the ica operon. UDP-glucose, UDP-galactose, and UDP-glucuronic acid each revealed no activity, and nonactivated N-acetylglucosamine and glucosamine also failed to function as substrates in the in vitro assay (data not shown).

To allocate the N-acetylglucosaminyltransferase activity, the ica genes were introduced independently or in various combinations into $S$. carnosus, and cell extracts were tested for glycosyltransferase activity. We could thus demonstrate that IcaA alone revealed a weak N-acetylglucosaminyltransferase activity (Fig. 5, lane A). Although the activity was very weak, there is no doubt that IcaA represents the actual N-acetylglucosaminyltransferase since no transferase activity was obtained with IcaD, IcaB, and/or IcaC in the absence of IcaA. This is in accordance with the sequence similarity of IcaA to processive glycosyltransferases, which add multiple monosaccharides to a growing sugar chain (29). In the N-acetylglucosaminyl

N-Acetylglucosaminyltransferase in PIA Biosynthesis

---

- GlcNAc (T)
  - 2-mer
  - 3-mer
  - 4-mer
  - 5-mer
  - (GlcNAc)₂ -
  - (GlcNAc)₃ -
  - (GlcNAc)₄ -
  - (GlcNAc)₅ -
  - (GlcNAc)₆ -
  - (GlcNAc)₇ -
  - (GlcNAc)₈ -

- A D A+D AD AD
mix 2pl. 1pl.

FIG. 5. TLC of reaction products of IcaA and/or IcaD. In vitro reactions containing \[^{14}C\]-labeled UDP-N-acetylglucosamine were performed with Triton-extracted membranes from induced *S. carnosus* carrying pTX16 (lane –), pTXicaA (lane A), pCXicaD (lane D), pTXicaA and pCXicaD (lane AD 2pl.), or pTXicaAD (lane AD 1pl.). In addition, a 1:1 mixture (lane A+D mix) of the Triton-extracted membranes of *S. carnosus* pTXicaA and *S. carnosus* pCXicaD was analyzed. The samples were analyzed by NH₂-HPTLC. The results were visualized by autoradiography. The arrowhead indicates the origin of TLC. The arrows indicate the products of IcaA alone; their degree of oligomerization is indicated on the right. The position of N-acetylglucosamine (containing 0.005% (w/v) Triton X-100 (GlcNAc (T))) is marked by a bar.

Glc-
NAc AD AD AD BC UDP-
BC AD + BC

FIG. 6. Analysis of the influences of IcaB and IcaC on the degree of polymerization of the IcaAD-dependent products. The in vitro effect of IcaB and IcaC was analyzed using crude membranes of induced *S. carnosus* harboring pTXicaAD (lane AD), pTXicaADB (lane ADB), pTXicaADC (lane ADC), pTXicaADBC (lane ADBC), pTXicaBC (lane BC), or the negative control plasmid pTX16 (lane –) and, in addition, a mixture of crude membranes from *S. carnosus* pTXicaAD and *S. carnosus* pTXicaBC (lane AD+BC). The assays performed with \[^{14}C\]-labeled substrate were analyzed by NH₂-HPTLC. N-Acetyl-d-[1-\[^{14}C\]]glucosamine and UDP-N-acetyl-d-[U-\[^{14}C\]]glucosamine (25 Bq each) were used as reference compounds. The results were visualized by autoradiography. The monomer to octamer of N-acetylglucosamine as determined by mass spectrometry are marked. The unreacted UDP-N-acetylglucosamine of the reaction assays did not correspond in its migration to that of the pure substance, but it was contained in the spot smearing directly above the origin. The arrowhead indicates the origin of TLC.

Transferase NodC of rhizobia (30, 31), the hyaluronan synthase HasA of *Streptococcus pyogenes* (32), bacterial cellulose synthases (33), chitin synthases of fungi (34), and the DG42 protein of *Xenopus laevis*, which synthesizes Nod-like oligomers during embryogenesis (35), five amino acids are conserved that are thought to function as catalytic residues (36). These amino acids are also conserved in IcaA (Asp¹³⁴, Asp²²⁷, Gln²⁷³, Arg²⁷⁶, and Trp²⁷⁷) as shown in Fig. 7.

Full glycosyltransferase activity is only achieved when *icaD* is cotranscribed with *icaA* (Fig. 5, lane AD 1pl.). The newly identified *icaD* gene is located between *icaA* and *icaB* and overlaps both of these genes. Since the start of *icaD* overlaps with the end of *icaA* by 37 nucleotides (Fig. 1), we were at first uncertain whether this small open reading frame was functional or whether it was expressed as an IcaA-IcaD fusion protein due to –1 translational frameshifting. In the Western blots (Fig. 2), IcaD exhibited a size of 34 kDa, instead of the calculated 12.0 kDa, similar to that of the IcaA protein (38 kDa). However, reprobing of the blot that was analyzed with the IcaD-specific antiserum with the anti-IcaA antiserum clearly showed that IcaA and IcaD represent separate proteins with no cross-reactivity. The deviations in the apparent molecular masses of IcaA and IcaD from their calculated masses (IcaA, 38 instead of 47.8 kDa; IcaD, 34 instead of 12.0 kDa) might result from performing all SDS-polyacrylamide gel electrophoresis analyses without heating the protein samples. When heated, IcaA could be detected only as a high molecular mass smear (Fig. 2A, lane \(TM^h\)), possibly due to aggregation of the IcaA molecules. Without heating, a complete denaturation of the proteins is not guaranteed, thus possibly causing an unusual migration of both IcaA and IcaD. The 34-kDa IcaD protein could also correspond to a trimer of the 12-kDa monomer. IcaA and IcaD are both membrane-bound (Fig. 2), which correlates with the finding that essentially membrane fractions showed transferase activity (Fig. 3). The membrane location was not unexpected since the hydrophobicity plots suggested the presence of four transmembrane helices in IcaA (one located at the N terminus and three at the C terminus) (16) and two transmembrane domains in IcaD.

To exhibit its function, IcaD probably has to interact with IcaA prior to membrane integration since mixing of extracts from an *S. carnosus* clone expressing *icaA* and a clone expressing *icaD* did not increase the N-acetylglucosaminyltransferase activity over that of IcaA alone (Fig. 5, lane A+D mix), whereas in trans expression of *icaA* and *icaD* did (lane AD 2pl.). The lower activity obtained with *icaA* and *icaD* expressed in trans compared with *icaAD* expressed in cis probably resulted from the different copy numbers of the vectors used (high copy pTX-icaA and medium copy pCXicaD).

The concrete function of IcaD is not yet known. The interaction of IcaA and IcaD might be necessary due to the unusual β-1,6-linkage of the N-acetylglucosamine residues in PIA. Sugar chains composed of β-1,6-linked monomers differ greatly in their three-dimensional structure from the straight β-1,4-sugar chains synthesized by most of the processive glycosyltransferases. Another possibility is that the IcaA protein needs IcaD to obtain an active conformation. Since IcaD is also a

18592

**N-Acetylglucosaminyltransferase in PIA Biosynthesis**

IcaA 121 GIKQASYEYVMCLDADTVIDDDAP  
NodC 127 AVRRSCGDLVLNVSDTILARDVV  
HasA 121 AFERSDADVFLTVSDTYITYPNAL  
Chs4 828 GISPDFYEIVLMYDADTKVFPDSL  
AcsA 223 AIKRTSGDYILILDCDHIPTRAFL  
DG42 227 QAIGTSVDYVQVCSDTKLDELAT  

IcaA 220 DMITEDIAVS  
NodC 234 SDFGEDRHLT  
HasA 228 VSIGDDRCLT  
Chs4 943 LLLGEDRYLS  
AcsA 328 ETVTEDAHTA  
DG42 335 CTLGDDRHLT  

IcaA 258 GGLWKQRVRWAQGGH  
NodC 272 GPYLROQLRWARSTF  
HasA 265 KSYLKQQNRWNSFF  
Chs4 983 MVLLSQRRRWINSTI  
AcsA 364 TTHIGORMRWARGMI  
DG42 373 LRWLNQQTRWTKSYY  

**FIG. 7. Partial alignment of IcaA with processive glycosyltransferases.** The deduced IcaA protein sequence (412 amino acids) was compared with the sequences of NodC from *Rhizobium loti* (424 amino acids; Swiss-Prot P17862), HasA from *S. pyogenes* (395 amino acids; PIR A48755), the class IV chitin synthase Chs4 from *Neurospora crassa* (1195 amino acids; GenBank™ U25097), the cellulose synthase AcsA from *A. xylinum* (754 amino acids; Swiss-Prot P19449), and the DG42 protein from *X. laevis* (588 amino acids; Swiss-Prot P13563) in the regions containing the five amino acids that are predicted to function as catalytic residues (36). The strictly conserved amino acids (in IcaA, Asp<sup>134</sup>, Asp<sup>227</sup>, Gln<sup>273</sup>, Arg<sup>276</sup>, and Trp<sup>277</sup>) are indicated by boldface letters. The surrounding less strictly conserved motifs are indicated by asterisks.

important for a further elongation of the oligomers without a new activation. Anhydro bonds, for example, also exist in the peptidoglycan of *E. coli*, in which the sugar chains do not contain a reducing end, but carry 1,6-anhydromuramic acid (41).

No oligomers containing *N*-unsubstituted residues, which compose 15–20% of the major polysaccharide I of PIA *in vivo* (13), were detected by MS analysis. Since the known pathways for the synthesis of amino sugar-containing polysaccharides utilize the nucleotide-linked *N*-acetylamino sugars as precursors, it seems most likely that the *N*-unsubstituted amino sugar residues of PIA are produced through enzymatic deacetylation subsequent to the formation of the polymer chain. The MS analyses were only performed with products synthesized with membranes, and therefore, the putative deacetylase activity may have escaped if it is located in another cell fraction. Known deacetylating enzymes are localized in different cell fractions (42–44). However, it is also possible that the percentage of deacetylated IcaAD-dependent oligomers was too low to be detected or that only full-length PIA is a substrate for deacetylation. It was not possible to test whether the *N*-unsubstituted residues in PIA might be derived from the direct incorporation of glucosamine because UDP-glucosamine is not commercially available.

IcaAD synthesized oligomers of a maximal length of 20 residues as determined by TLC analysis, but did not form polymers comprising full-length PIA, which was shown to be at least 130 residues long (13). By expressing *icaADC* in *S. carnosus*, products were synthesized that did not migrate on TLC plates and that reacted with the PIA-specific antiserum. We therefore assume that the nonmigrating products represent longer oligosaccharide chains. We cannot rule out that these products might represent modified or protein-bound oligomers. However, an *S. carnosus* clone expressing *icaADC* is capable of forming cell aggregates.² This speaks against the possibility that the IcaADC-dependent products simply represent PIA precursors and suggests that the genes *icaA*, *icaD*, and *icaC* are sufficient to direct the biosynthesis of PIA. The genes *icaA*, *icaD*, and *icaC* must, however, be coexpressed in one strain to obtain synthesis of the longer oligomers (Fig. 6). Thus, IcaA, IcaD, and IcaC, which is predicted to be an integral membrane protein (16), probably form a complex and assemble in the membrane in a coordinated manner. This would explain why mixing of crude extracts from strains expressing *icaAD* and *icaC*, respectively, did not lead to PIA biosynthesis.

The IcaB protein, predicted to be a secreted protein (16), does not seem to be involved in the biosynthesis of the PIA sugar chain; it had no detectable effect in the *in vitro* assays. Its function in the PIA-mediated cell aggregation is being investigated.

The data presented here show that IcaA, IcaD, and IcaC are involved in the biosynthesis of the sugar backbone of PIA. IcaAD constitutes *N*-acetylglucosaminyltransferase activity in which IcaA represents the actual transferase that requires IcaD for full activity. This is the first enzymatic activity identified to be involved in intercellular adhesion in *S. epidermidis*.

Acknowledgments—We thank Elisabeth Knorpp for excellent technical assistance and Sarah E. Cramton and Karen A. Brune for critical reading of the manuscript.

**REFERENCES**

1. Christensen, G. D., Baldassarri, L., and Simpson, W. A. (1994) in *Infections Associated with Indwelling Medical Devices* (Bisno, A. L., and Waldvogel, F. A., eds) 2nd Ed., pp. 45–78, American Society for Microbiology, Washington, D. C.
2. Rupp, M. E., and Archer, G. D. (1994) *Clin. Infect. Dis.* **19**, 231–245

² C. Gerke, O. Schweitzer, and F. Götz, unpublished data.

3. Ludwicka, A., Uhlenbruck, G., Peters, G., Seng, P. N., Gray, E. D., Jeljaszewicz, J., and Pulverer, G. (1984) *Zentralbl. Bakteriol. Mikrobiol. Hyg. Ser. A* 258, 256–267

4. Peters, G., Locci, R., and Pulverer, G. (1981) *Zentralbl. Bakteriol. Abt. 1 Orig. B Hyg. Krankenhaushyg. Betriebshyg. Praev. Med.* 172, 293–299

5. Ludwicka, A., Jansen, B., Wadström, T., and Pulverer, G. (1984) *Zentralbl. Bakteriol. Mikrobiol. Hyg. Ser. A* 256, 479–489

6. Pascual, A., Fleer, A., Westerdaal, N. A. C., and Verhoef, J. (1986) *Eur. J. Clin. Microbiol.* 5, 518–522

7. Tojo, M., Yamashita, N., Goldmann, D. A., and Pier, G. B. (1988) *J. Infect. Dis.* 157, 713–722

8. Müller, E., Hübner, J., Gutierrez, N., Takeda, S., Goldmann, D. A., and Pier, G. B. (1993) *Infect. Immun.* 61, 551–558

9. Timmerman, C. P., Fleer, A., Besnier, J. M., de Graaf, L., Cremers, F., and Verhoef, J. (1991) *Infect. Immun.* 59, 4187–4192

10. Veenstra, G. J. C., Cremers, F. F. M., van Dijk, H., and Fleer, A. (1996) *J. Bacteriol.* 178, 537–541

11. Heilmann, C., Hussain, M., Peters, G., and Götz, F. (1997) *Mol. Microbiol.* 24, 1013–1024

12. Mack, D., Nedelmann, M., Krokotsch, A., Schwarzkopf, A., Heesemann, J., and Laufs, R. (1994) *Infect. Immun.* 62, 3244–3253

13. Mack, D., Fischer, W., Krokotsch, A., Leopold, K., Hartmann, R., Egge, H., and Laufs, R. (1996) *J. Bacteriol.* 178, 175–183

14. Hussain, M., Herrmann, M., von Eiff, C., Perdreau-Remington, F., and Peters, G. (1997) *Infect. Immun.* 65, 519–524

15. Schumacher-Perdreau, F., Heilmann, C., Peters, G., Götz, F., and Pulverer, G. (1994) *FEMS Microbiol. Lett.* 117, 71–78

16. Heilmann, C., Schweitzer, O., Gerke, C., Vanittanakom, N., Mack, D., and Götz, F. (1996) *Mol. Microbiol.* 20, 1083–1091

17. Heilmann, C., Gerke, C., Perdreau-Remington, F., and Götz, F. (1996) *Infect. Immun.* 64, 277–282

18. Heilmann, C., and Götz, F. (1998) *Zentralbl. Bakteriol.* 287, 69–83

19. Ziebuhr, W., Heilmann, C., Götz, F., Meyer, P., Wilms, K., Straube, E., and Hacker, J. (1997) *Infect. Immun.* 65, 890–896

20. Götz, F. (1990) *J. Appl. Bacteriol. Symp. Suppl.* 49–53

21. Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) *Molecular Cloning: A Laboratory Manual*, 2nd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, NY

22. Marmur, J. (1961) *J. Mol. Biol.* 3, 202–218

23. Götz, F., and Schumacher, B. (1987) *FEMS Microbiol. Lett.* 40, 285–288

24. Peschel, A., and Götz, F. (1996) *J. Bacteriol.* 178, 531–536

25. Wieland, K.-P., Wieland, B., and Götz, F. (1995) *Gene (Amst.)* 158, 91–96

26. Lugtenberg, E. J. J., de Haas-Menger, L., and Ruyters, W. H. M. (1972) *J. Bacteriol.* 109, 326–335

27. Bugg, T. D. H., and Brandish, P. E. (1994) *FEMS Microbiol. Lett.* 119, 255–262

28. Bligh, E. G., and Dyer, W. J. (1959) *Can. J. Biochem. Physiol.* 37, 911–917

29. Saxena, I. M., Brown, R. M. J., Fevre, M., Geremia, R. A., and Henrissat, B. (1995) *J. Bacteriol.* 177, 1419–1424

30. Spaink, H. P., Wijfjes, A. H. M., van der Drift, K. M. G. M., Haverkamp, J., Thomas-Oates, J. E., and Lugtenberg, B. J. J. (1994) *Mol. Microbiol.* 13, 821–831

31. Geremia, R. A., Mergaert, P., Geelen, D., Van Montagu, M., and Holsters, M. (1994) *Proc. Natl. Acad. Sci. U. S. A.* 91, 2669–2673

32. DeAngelis, P. L., Papaconstantinou, J., and Weigel, P. H. (1993) *J. Biol. Chem.* 268, 19181–19184

33. Wong, H. C., Fear, A. L., Calhoun, R. D., Eichinger, G. H., Mayer, R., Amikam, D., Benziman, M., Gelfand, D. H., Meade, J. H., Emerick, A. W., Bruner, R., Ben-Bassat, A., and Tal, R. (1990) *Proc. Natl. Acad. Sci. U. S. A.* 87, 8130–8134

34. Din, A. B., Specht, C. A., Robbins, P. W., and Yarden, O. (1996) *Mol. Gen. Genet.* 250, 214–222

35. Semino, C. E., and Robbins, P. W. (1995) *Proc. Natl. Acad. Sci. U. S. A.* 92, 3498–3501

36. Nagahashi, S., Sudoh, M., Ono, N., Sawada, R., Yamaguchi, E., Uchida, Y., Mio, T., Takagi, M., Arisawa, M., and Yamada-Okabe, H. (1995) *J. Biol. Chem.* 270, 13961–13967

37. Mergaert, P., D’Haeze, W., Geelen, D., Promé, D., Van Montagu, M., Geremia, R., Promé, J.-C., and Holsters, M. (1995) *J. Biol. Chem.* 270, 29217–29223

38. Sugahara, K., Schwartz, N. B., and Dorfmann, A. (1979) *J. Biol. Chem.* 254, 6252–6261

39. Orlean, P. (1987) *J. Biol. Chem.* 262, 5732–5739

40. Ross, P., Mayer, R., and Benziman, M. (1991) *Microbiol. Rev.* 55, 35–58

41. Höltje, J. V., Mirelman, D., Sharon, N., and Schwarz, U. (1975) *J. Bacteriol.* 124, 1067–1076

42. Araki, Y., Oba, S., Araki, S., and Ito, E. (1980) *J. Biochem. (Tokyo)* 88, 469–479

43. John, M., Röhrig, H., Schmidt, J., Wieneke, U., and Schell, J. (1993) *Proc. Natl. Acad. Sci. U. S. A.* 90, 625–629

44. Tokuyasu, K., Ohnishi-Kameyama, M., and Hayashi, K. (1996) *Biosci. Biotech. nol. Biochem.* 60, 1598–1603

45. New England Biolabs (1990) *Protein Fusion and Purification System Manual*, New England Biolabs, Schwalbach, Germany

46. Laemmli, U. K. (1970) *Nature* 227, 680–685

47. Bradford, M. M. (1976) *Anal. Biochem.* 72, 248–254
