
Sirohaem sulfite reductase and other proteins encoded by genes at the *dsr* locus of *Chromatium vinosum* are involved in the oxidation of intracellular sulfur

Andrea S. Pott and Christiane Dahl

Author for correspondence: Christiane Dahl. Tel: +49 228 732119. Fax: +49 228 737576.  
e-mail: ChDahl@uni-bonn.de

Institut für Mikrobiologie & Biotechnologie,  
Rheinische Friedrich-Wilhelms-Universität Bonn,  
Meckenheimer Allee 168,  
D-53115 Bonn, Germany

The sequence of the *dsr* gene region of the phototrophic sulfur bacterium *Chromatium vinosum* D (DSMZ 180ᵀ) was determined to clarify the *in vivo* role of 'reverse' sirohaem sulfite reductase. The *dsrAB* genes encoding dissimilatory sulfite reductase are part of a gene cluster, *dsrABEFHCMK*, that encodes four small, soluble proteins (*DsrE*, *DsrF*, *DsrH* and *DsrC*), a transmembrane protein (*DsrM*) with similarity to haem-*b*-binding polypeptides and a soluble protein (*DsrK*) resembling [4Fe–4S]-cluster-containing heterodisulfide reductase from methanogenic archaea. Northern hybridizations showed that expression of the *dsr* genes is increased by the presence of reduced sulfur compounds. The *dsr* genes are not only transcribed from a putative promoter upstream of *dsrA* but primary transcripts originating from (a) transcription start site(s) downstream of *dsrB* are also formed. Polar insertion mutations immediately upstream of *dsrA*, and in *dsrB*, *dsrH* and *dsrM*, led to an inability of the cells to oxidize intracellularly stored sulfur. The capability of the mutants to oxidize sulfide, thiosulfate and sulfite under photolithoautotrophic conditions was unaltered. Photoorganoheterotrophic growth was also unaffected. ‘Reverse’ sulfite reductase and *DsrEFHCMK* are, therefore, not essential for oxidation of sulfide or thiosulfate, but are obligatory for sulfur oxidation. These results, together with the finding that the sulfur globules of *C*. *vinosum* are located in the extracytoplasmic space whilst the *dsr* gene products appear to be either cytoplasmic or membrane-bound led to the proposal of new models for the pathway of sulfur oxidation in this phototrophic sulfur bacterium.

Keywords: *Chromatium vinosum*, sulfur oxidation, reverse sirohaem sulfite reductase, extracytoplasmic sulfur globules, heterodisulfide reductase

INTRODUCTION

A wide variety of enzymes catalysing sulfur redox reactions have been isolated from phototrophic sulfur bacteria (Trüper & Fischer, 1982; Fischer, 1989; Brune, 1989; Dahl & Trüper, 1994; Brune, 1995a). However, to some of these, specific *in vivo* functions could not be assigned. Flavocytochrome *c* (Dolata *et al.*, 1993; Chen *et al.*, 1994) and sirohaem sulfite reductase (Schedel *et al.*, 1979) are only two examples of biochemically well-characterized proteins whose roles in the oxidation of reduced sulfur compounds are still unclear. On the other hand, the enzymes responsible for several of the sulfur transformations catalysed by phototrophic sulfur bacteria, among them the oxidation of sulfide and sulfur, have not conclusively been identified.

Two possibilities have been discussed for the initial step in sulfide oxidation, the first of which is oxidation to elemental sulfur catalysed by a sulfide dehydrogenase. The sulfide dehydrogenase has been proposed to be flavocytochrome *c*, other *c*-type cytochromes or sulfide:quinone oxidoreductase, depending on the organism studied (reviewed by Brune, 1989, 1995a). It has also been proposed that H₂S might, at least in part, be oxidized directly to sulfite by a sirohaem sulfite reductase

The GenBank accession number for the sequence reported in this paper is U84760.

A. S. POTT and C. DAHL

Table 1. Bacterial strains and plasmids

| Bacterial strain or plasmid | Genotype or phenotype | Source or reference |
|----------------------------|-----------------------|---------------------|
| Escherichia coli strains    |                       |                     |
| DH5α                        | $F^{-} \phi 80 d l a c Z \Delta M 15 \Delta(l a c Z Y A-a r g F) U 169 r e c A 1 e n d A 1 h s d R 17(r_{K}^{+} m_{K}^{+}) s u p E 44$ $\lambda^{-} t h i-1 g y r A r e l A 1$ | Hanahan (1983) |
| SM10                        | $K m^{r}, s u p E 44 t h i-1 t h r-1 r e c A l e u B 6 l a c Y 1 t o n A 21 R P 4-2-T c::M u-K m::T n 7$ in chromosome | Simon et al. (1983) |
| Chromatium vinosum strains   |                       |                     |
| SM50                        | $S m^{r}$, spontaneous streptomycin-resistant mutant of $C$. $v i n o s u m D S M 180^{T}$ | Dahl (1996) |
| 31D                         | $S m^{r} K m^{r}, d s r B::K m \Omega$ ($K m^{r}$ in the same orientation as $d s r B$) | This work |
| 21D                         | $S m^{r} K m^{r}, d s r B::K m \Omega$ ($K m^{r}$ in the opposite orientation to $d s r B$) | This work |
| 34D                         | $S m^{r} K m^{r} d s r H::K m \Omega$ | This work |
| 35D                         | $S m^{r} K m^{r}, d s r M::K m \Omega$ | This work |
| 34S                         | $S m^{r} K m^{r}, A p^{r}, d s r M::p A P 34$ | This work |
| 37D                         | $S m^{r} K m^{r}, 236 b p$ upstream of $d s r A::K m \Omega$ | This work |
| Plasmids                    |                       |                     |
| pBluescript SK II(+)        | $A p^{r} l a c Z^{\prime} f 1 o r i$ | Stratagene |
| pSUP202                     | $A p^{r} C m^{r} T c^{r}, R P 4 o r i T C o l E 1 o r i$ | Simon et al. (1983) |
| pSUP301                     | $A p^{r} K m^{r}, R P 4 o r i T p 15 A o r i$ | Simon et al. (1983) |
| pAP1a and pAP1b             | $A p^{r}, 4 \cdot 8 k b X h o I$ fragment in $X h o I$ of pBluescript SK II(+), orientations A and B | This work |
| pAP10                       | $A p^{r}, 3 \cdot 9 k b S a l I$ fragment in $S a l I$ of pBluescript SK II(+) | Hipp et al. (1997) |
| pAP50a and pAP50b           | $A p^{r}, 4 \cdot 06 k b B g l I I$ fragment in $B a m H I$ of pBluescript SK II(+), orientations A and B | This work |
| pAP2                        | $A p^{r}, 3 \cdot 1 k b E c o R I$ fragment in $E c o R I$ of pUC18 | Hipp et al. (1997) |
| pAP20a                      | $T c^{r} A p^{r}, 3 \cdot 1 k b E c o R I$ fragment in $E c o R I$ of pSUP202 | This work |
| pAP21                       | $K m^{r} T c^{r} A p^{r}, K m^{r}$ cartridge ($E c o R I$) from pHPP45Ω in $S m a I$ of pAP20a | This work |
| pAP30                       | $A p^{r}, 3 \cdot 1 k b E c o R I$ fragment in $S m a I$ of pSUP301 | This work |
| pAP31                       | $K m^{r} A p^{r}, K m^{r}$ cartridge ($E c o R I$) from pHPP45Ω in $S m a I$ of pAP30 | This work |
| pAP33a and pAP33b           | $A p^{r}, 2 \cdot 3 k b E c o 109 I$ fragment from pAP1b in $S m a I$ of pSUP301, orientations A and B | This work |
| pAP34                       | $K m^{r} A p^{r}, K m^{r}$ cartridge ($E c o R I$) from pHPP45Ω in $E c o R I$ of pAP33a | This work |
| pAP35                       | $K m^{r} A p^{r}, K m^{r}$ cartridge ($E c o R I$) from pHPP45Ω in $N c o l$ of pAP33a | This work |
| pAP36a and pAP36b           | $A p^{r}, 3 \cdot 9 k b S a l I$ fragment from pAP10 in $S m a I$ of pSUP301, orientations A and B | This work |
| pAP37                       | $K m^{r} A p^{r}, K m^{r}$ cartridge ($E c o R I$) from pHPP45Ω in $E c o R V$ of pAP36a | This work |
| pHPP45Ω                     | $K m^{r} A p^{r}$ | Fellay et al. (1987) |

operating in reverse, an enzyme found in Chromatium vinosum D (Schedel et al., 1979). This protein is closely related to the dissimilatory sulfite reductases from sulfate-reducing bacteria and archaea (Hipp et al., 1997), enzymes which are generally located in the cytoplasm. During the oxidation of sulfide and thiosulfate, phototrophic sulfur bacteria of the family Chromatiaceae form sulfur as an intermediate which is stored inside the cell. In the light of the recent finding that the sulfur globules in $C$. $v i n o s u m$ are located extracytoplasmically (Pattaragulwanit et al., 1998) it appears impossible that cytoplasmic sulfite reductase can directly oxidize sulfur as has been proposed by Trüper (1984). It has also been hypothesized that the sulfur globules of phototrophic sulfur bacteria are formed on a side branch of the main sulfide oxidation pathway and must be reduced back to sulfide prior to further oxidation (Trüper, 1984). A cytoplasmic sulfite reductase could be essential for this pathway.

As experimental evidence has not been obtained for any of the suggested functions of sulfite reductase, we cloned and sequenced $5^{\prime}-$ and $3^{\prime}$-flanking regions of the $C$. $v i n o s u m d s r A B$ genes encoding this protein. Analysis of the sequence data and phenotypic characterization of strains with mutations at the $d s r$ locus allowed us to propose new models for the sulfur oxidation pathways in phototrophic sulfur bacteria.

METHODS

Bacterial strains and plasmids, media and growth conditions. The strains and plasmids used in this study are listed in Table 1.

Escherichia coli strains were cultured in LB medium (Sambrook et al., 1989). IPTG and X-Gal were included in solid media to identify recombinant plasmids containing inserts in the α portion of lacZ. C. vinosum was grown and harvested as described earlier (Dahl & Trüper, 1994; Pattaragulwanit & Dahl, 1995; Dahl, 1996). Antibiotics were used at the following concentrations (in μg ml⁻¹): for E. coli, ampicillin, 100; kanamycin, 50; tetracycline, 5; chloramphenicol, 100; for C. vinosum, kanamycin, 5; streptomycin, 50; ampicillin, 10 (solid media) or 100 (liquid media).

Recombinant DNA techniques. Standard methods were used for molecular techniques (Sambrook et al., 1989; Wilkinson, 1991; Ausubel et al., 1997). Chromosomal DNA of C. vinosum strains was obtained by Sarkosyl lysis (Bazaral & Helinski, 1968) and purified by ethidium bromide/caesium chloride density-gradient centrifugation or by phenol/chloroform extraction and dialysis against water. Southern hybridizations were performed overnight at 68 °C as described previously (Dahl et al., 1994). PCR with Taq DNA polymerase was done essentially as described by Dahl (1996). DNA probes for Southern and Northern hybridization experiments and screening of libraries were digoxigenin-labelled by PCR and colony hybridizations were performed following the protocol of Boehringer. Double-stranded DNA was sequenced with the Silver Sequence system (Promega) and universal and de-novo-synthesized oligonucleotide primers. Nucleotide sequences were compiled and analysed with the DNASIS software (Pharmacia).

Cloning of the dsr locus. The region downstream of dsrB was cloned by using a PCR-amplified fragment from nucleotide 1298 to 3037 of the sequence presented in Fig. 1 as a probe to identify several positive clones with a 4.8 kb Xhol insert in a library of 4.5–4.8 kb Xhol fragments of chromosomal C. vinosum DNA in the pBluescript vector. Sequencing of the positive clone pAP1a revealed the presence of several complete ORFs and one ORF that was truncated at the 3′ end. To analyse the missing part of this ORF, a clone with a 4 kb BglII fragment was isolated from a library of 3.5–4.5 kb BglII fragments in the pBluescript vector which had previously been digested with BamHI. A PCR-amplified fragment covering nucleotides 4086–4636 of the complete sequence was used as a probe in this step.

Conjugative plasmid transfer and gene replacement. The plasmids used for the construction of C. vinosum strains with mutations in several of the dsr genes are listed and described in Table 1. Mobilizable plasmids were transferred from E. coli SM10 to C. vinosum SM50 by conjugation (Pattaragulwanit & Dahl, 1995). Transconjugants were selected on RCV (Weaver et al., 1975) plates with 0.3% malic acid and 0.5 g yeast extract l⁻¹ containing the appropriate antibiotics under anaerobic conditions in the light. Growth could be enhanced by addition of 20% PFAT (Pattaragulwanit & Dahl, 1995) to the RCV plates. Approximately 95% of the recombinants analysed had lost the vector-encoded antibiotic resistance indicating that a double-crossover recombination had occurred. The genotypes of the C. vinosum recombinants used in this study were confirmed by Southern hybridization and PCR.

RNA isolation and Northern hybridizations. Total RNA from C. vinosum was isolated following the protocol for rapid RNA isolation from Gram-negative bacteria described by Ausubel et al. (1997). After precipitation of the RNA with ethanol a red pellet was obtained. The RNA was further purified using the RNeasy Total RNA kit (Qiagen). After addition of 0.8–1.0 U RNasin ribonuclease inhibitor (Promega) µl⁻¹ the RNA was stored at –70 °C. RNA was electrophoresed in agarose/formaldehyde gels (0.9% agarose, 6.3% formaldehyde, 1 × MOPS buffer (40 mM MOPS, 10 mM sodium acetate, 2 mM EDTA, pH 7.0). Samples of 10 µg RNA in a total volume of 8 µl were mixed with 0.5 U RNase-free DNase (Promega) and incubated for 5 min at 37 °C. Sixteen microlitres of freshly prepared sample buffer [75 µl deionized formamide, 30 µl 37% formaldehyde, 30 µl 5 × MOPS, 15 µl 10 × dye (50% glycerol, 1 mM EDTA, 0.25% bromphenol blue) and 1 µl 1% ethidium bromide] were added, followed by incubation at 65 °C for 10 min and chilling on ice. The fragments of a 0.24–9.5 kb RNA ladder (Life Technologies) were used as RNA size markers. The RNA was transferred and fixed to positively charged Nylon membranes (Boehringer) via capillary blotting and UV cross-linking. Hybridization with digoxigenin-labelled DNA fragments was performed essentially as described by Boehringer. The membranes were prehybridized in 0.75 M sodium chloride, 0.075 M sodium citrate, pH 7.0, 50% deionized formamide, 0.1% sodium lauroylsarcosine, 0.02% SDS and 2% blocking reagent at 40 °C for 4–6 h. PCR-amplified DNA probes were purified by gel electrophoresis and isolation of the DNA with the QiaQuick Gel Extraction kit (Qiagen). After denaturation of the probe in prehybridization solution at 100 °C for 10 min, the prehybridization solution was discarded, the hybridization solution with the probe DNA was added and hybridization was continued for 12–16 h at 40 °C. Membranes were washed twice for 5 min in 2 × SSC, 0.1% SDS at room temperature and twice for 15 min in 0.1 × SSC, 0.1% SDS at 40 °C. Chemiluminescent detection was performed following the instructions of Boehringer with the following modifications: anti-DIG Fab fragments were diluted 1:7500, CSPD was applied to a final concentration of 125 µM.

Turnover of reduced sulfur compounds. This was measured in batch culture and sulfur compounds were determined as described by Dahl (1996).

**RESULTS**

Nucleotide sequence of the dsr locus

Recently, we reported the cloning and sequencing of the genes, dsrA and dsrB, encoding the two subunits of sirohaem sulfite reductase from C. vinosum (Hipp et al., 1997). As a further step in the systematic examination of this region, we cloned and sequenced upstream and downstream flanking regions and obtained a contiguous sequence of 6976 bp. Downstream of dsrB six complete ORFs are present, which we termed dsrEFHCMK. These are closely spaced and all are preceded by putative ribosome-binding sites (Fig. 1). All six ORFs show a codon usage typical for C. vinosum (not shown) and a G + C content ranging from 59.6 to 63.6 mol %, which is slightly lower than the 64.3 % found for total C. vinosum DNA (Pfennig & Trüper, 1989). A putative promoter region conforming to the consensus sequence of E. coli σ⁷⁰ promoters (Lisser & Margalit, 1993) was found upstream of dsrA (Fig. 1). Two inverted repeats with a potential for formation of hairpin loop structures (free energy of formation –92.8 and –164.8 kJ mol⁻¹) were found between dsrB and dsrE. These structures could function as transcription terminators; however, poly(T) sequences located directly downstream of such hairpin loops in typical eubacterial rho-independent transcription terminators (Reynolds et al., 1992) are not present in the C. vinosum sequence.

A. S. POTT and C. DAHL

1 CTGC GGAAGGCTCC AGGCACTGGC GTATATCTCC CAAAGGGATAG AGGTGCGAGGA GGATATCCCC
65 CCCCGATCGT GGAATGGGCG CGCATCCTGT CCATGGGAT ATTGAAAAAC GCCGATGGGG TTGAGGGTCC GGCTCGATAC GCACGCCGAG CGTGTCGGTA
165 CCACCGGCTC CACACACAGA CCCGACGCGA TGCAACGATC CGGCCTGTCC CGGCAAAGCG GTCGAGACAG GTTCGTGCAC GCGCACAAAAC CATATCCGTT
265 CCTGGTTATA TCGATAGGAC AGACCACCGCA ATG dsrA → 1248 nt TGA * dsrB → 54 nt ATG 1065 nt TGA TCCGGAG CGGACAGGGG GAGGCCGATC
2701 CGGCACAGCA CGGACGCGCC ATGATCGAAC GACAATCCAT CGCCTCGGGT CGGACGCGCG GCACGACCGA CCCGAGAGCGG CGATGACTGG ATTGGATAG
rbS dsrE → * rbS dsrF → * rbS dsrH → *
2801 CCAAGAGGTC TGCATG 387 nt TGAGGGA GCGCCAACAT G 405 nt TAAGGGAGG GTGGAATG 303 nt TAA TTTTCGATT TTCTTTACCG
rbS dsrC → *
3971 AAACCAAGAG AGGAAGATTC CAATG 333 nt TAAGCG GACGAGCGTC CGGCGCGCGC GGTTCCTTTTG AACCTGTGGC GCGCAGGATT CCATCAAGCT
rbS dsrM → *
4395 CCACCGAGGG TGCGGAAACA TG 726 nt TGAGCCGCCC TGATTCGAGC GCCCAGTCG TCCGCTTACA GAGGTTGATC CATG 1569 nt TGACCG
6771 GGGCCATCGA AGCGCGGCGT CTCCGGATCA ATCCACTAGA GCCGGCTGCG CGACCTGATA CGAATCCCCCT GATCCCCGCG TCAGGGTTCA ACTGGTACGG
6771 AGAACGATTG AGATGGCGAC TTCCAGCGAC GAGATGAAAGA TGAAAGCCGAC CTGGCGTCGA TTCGAGGACG CGGAAACACGT CTGGGACAAC CTGGACCGACA
6871 AGATCT

Fig. 1. Partial nucleotide sequence of the dsr locus of C. vinosum. The nucleotide sequence of the dsrA and dsrB genes (nt 295–2673) has been published elsewhere (Hipp et al., 1997). Putative ribosome-binding sites and promoter regions are boxed. Translation stop signals are marked by asterisks. Inverted repeats with the ability to form stable hairpin structures are indicated by arrows. The nucleotide sequence printed white on black conforms to the consensus sequence for RNase E cleavage sites published by Ehretsmann et al. (1992) and is located in the vicinity of two probable hairpin-loop-forming sequences.

dsrE, dsrF and dsrH encode similar proteins

dsrE, dsrF and dsrH encode polypeptides of 130, 136 and 102 amino acids with calculated molecular masses of 14·6, 15·6 and 12·6 kDa, respectively. The theoretical isoelectric points for these proteins lie at pH 5·7, 4·2 and 4·7, respectively. All three deduced proteins are predicted to be soluble and cytoplasmic due to the lack of regions able to form transmembrane helices and the lack of potential signal peptide sequences. A comparison of the derived protein sequences with sequences deposited in the databases was done with the Ψ-BLAST program (Altschul et al., 1997). The only matches found were with hypothetical proteins from E. coli, Haemophilus influenzae and methanogenic archaea (Table 2). Like dsrEFH, the matching E. coli and H. influenzae genes are juxtapositioned. In Methanobacterium thermoautotrophicum, the matching genes MTH1357, 1358 and 1360 are interrupted by only one gene MTH1359. Surprisingly, all three C. vinosum gene products showed low but significant similarity (P-values ranging from 0·036 to 0·14) to the hypothetical proteins MJ0989 (pir E64423) and ORF1 (gb Y10251) from Methanococcus jannaschii and Methanopyrus kandleri, respectively, indicating that DsrE, DsrF and DsrH are similar to each other. The alignment of their amino acid sequences (Fig. 2a) verified this assumption. The presence of related genes in E. coli, H. influenzae and methanogenic archaea, organisms which do not contain dissimilatory sulfite reductases but can synthesize assimilatory sirohaem sulfite and nitrite reductases under appropriate growth conditions, may hint at a function of the dsrEFH-encoded polypeptides in assembly, folding or stabilization of sirohaem proteins. This conclusion is, however, inconsistent with the observation that the genome

of the sulfate reducer Archaeoglobus fulgidus, an organism that contains a dissimilatory sirohaem sulfite reductase (Dahl et al., 1993) does not contain similar genes (Klenk et al., 1997).

The dsrC-encoded protein is similar to DsrC from dissimilatory sulfate- and sulfite reducers

dsrC encodes a soluble, cytoplasmic, 112 amino acid, 12·6 kDa polypeptide with a theoretical isoelectric point of pH 4·97. A homology search identified a significant degree of sequence identity with the dsv(r)C gene products of Desulfovibrio vulgaris (Karkhoff-Schweizer et al., 1993), Pyrobaculum islandicum (Molitor et al., 1998) and A. fulgidus (Klenk et al., 1997) and to the products of the potential yccK genes from E. coli and H. influenzae (Table 2). A conspicuously high sequence similarity is found at the C terminus of these proteins (Fig. 2b). The strict conservation of a single cysteine residue in this region may hint at a catalytic function of DsrC in the metabolism of sulfur compounds. The dsvC-encoded gene product is co-purified with the dissimilatory sulfite reductases from some sulfate-reducing bacteria (Pierik et al., 1992; Arendsen et al., 1993; Steuber et al., 1995). For Desulfovibrio desulfuricans, a possible function of DsrC in thiosulfate and trithionate reduction has been pointed out (J. Steuber, personal communication, cited in Voordouw, 1995).

dsrM exhibits similarity to membrane-bound b-type cytochromes

DsrM is deduced to be a protein of 243 amino acids with a molecular mass of 27·9 kDa and a pI of 9·85. As shown in Table 2, optimum sequence similarity is shared with

Sulfur oxidation in Chromatium vinosum

Table 2. Amino acid sequence similarity of DsrE, -F, -H, -C, -M and -K to other proteins

Up to a threshold of \(1 \times 10^{-3}\) for the P-value, the seven best matches obtained with the Ψ-BLAST program are shown.

| Dsr protein | Organism               | Gene           | Function                                                                 | P-value*         | Accession no. |
|-------------|------------------------|----------------|--------------------------------------------------------------------------|------------------|---------------|
| DsrE        | E. coli                | yheN           | Hypothetical 13·6 kDa protein                                              | \(1 \times 10^{-26}\) | sp P45532     |
|             | H. influenzae          | HI0576         | Hypothetical protein                                                      | \(3 \times 10^{-18}\) | sp P44762     |
|             | Mb. thermoautotrophicum | MTH1357        | Conserved protein                                                         | \(5 \times 10^{-4}\) | gb AE000898   |
| DsrF        | E. coli                | yheM           | Hypothetical 13·0 kDa protein                                              | \(8 \times 10^{-9}\) | sp P45531     |
|             | H. influenzae          | HI0576.1       | Hypothetical protein                                                      | \(2 \times 10^{-7}\) | gb U32739     |
|             | Mb. thermoautotrophicum | MTH1358        | Conserved protein                                                         | \(1 \times 10^{-5}\) | gb AE000898   |
| DsrH        | E. coli                | yheL           | Hypothetical 10·7 kDa protein                                              | \(2 \times 10^{-8}\) | gb U18997     |
|             | Mb. thermoautotrophicum | MTH1360        | Conserved protein                                                         | \(3 \times 10^{-3}\) | gb AE000898   |
| DsrC        | E. coli                | yccK           | Hypothetical protein                                                       | \(2 \times 10^{-28}\) | gb AE000199   |
|             | H. influenzae          | yccK           | Hypothetical protein HI1371                                                | \(2 \times 10^{-27}\) | sp P45184     |
|             | A. fulgidus            | dsvC           | Homologue of D. vulgaris DsvC                                              | \(2 \times 10^{-26}\) | gb AE000951   |
|             | D. vulgaris            | dsvC           | Proposed γ subunit of dissimilatory sulfite reductase                        | \(5 \times 10^{-21}\) | sp P45573     |
|             | P. islandicum          | dsrC           | 12·7 kDa protein in dsr operon                                             | \(7 \times 10^{-19}\) | gb U75249     |
| DsrM        | A. fulgidus            | AF0501         | Putative γ subunit of nitrate reductase                                    | \(3 \times 10^{-34}\) | gb AE001069   |
|             | A. fulgidus            | AF0546         | Putative γ subunit of nitrate reductase                                    | \(2 \times 10^{-13}\) | gb AE001066   |
|             | A. fulgidus            | AF0545         | Hypothetical protein                                                       | \(4 \times 10^{-7}\) | gb AE001067   |
|             | E. coli                | narI           | γ subunit of nitrate reductase 1                                           | \(6 \times 10^{-6}\) | sp P11350     |
|             | Th. thermophilus       | narV           | γ subunit of nitrate reductase 2                                           | \(2 \times 10^{-4}\) | gb Y10124     |
|             | Ms. barkeri            | hdrE           | 29·7 kDa subunit, heterodisulfide reductase                                | \(2 \times 10^{-4}\) | gb Y09870     |
|             | D. vulgaris            | hmc5           | 25·3 kDa protein in hmc operon                                             | \(8 \times 10^{-3}\) | sp P33392     |
| DsrK        | A. fulgidus            | AF0502         | Heterodisulfide reductase, putative subunit D                              | \(1 \times 10^{-105}\) | gb AE001069   |
|             | A. fulgidus            | AF0547         | Iron–sulfur binding reductase                                              | \(1 \times 10^{-65}\) | gb AE001066   |
|             | A. fulgidus            | AF0543         | Conserved hypothetical protein                                             | \(4 \times 10^{-47}\) | gb AE001067   |
|             | A. fulgidus            | AF0544         | Conserved hypothetical protein                                             | \(2 \times 10^{-35}\) | gb AE001067   |
|             | D. vulgaris            | hmc6           | 52·7 kDa protein in hmc operon                                             | \(2 \times 10^{-28}\) | sp P33393     |
|             | Ms. barkeri            | hdrD           | 46 kDa subunit, heterodisulfide reductase                                  | \(3 \times 10^{-18}\) | gb Y09870     |
|             | A. fulgidus            | AF0755         | Putative heterodisulfide reductase, subunits E and D                       | \(5 \times 10^{-12}\) | AE001052      |

*Derived from Ψ-BLAST score; P-value represents the probability that the observed similarity had occurred by chance. Low P-values indicate a high probability of two proteins being structurally related.

A. S. POTT and C. DAHL

(a)
| CvDsrF | 1 | MSEWKKFMYLNKAPYGTIYAWEALEVVLIGAAFDQDVC-VLFLLDDGVYQLTRGQDTKG |
| CvDsrH | 1 | MSILHTVN-KSPFERNISLESLKFATEGAS--------VLLFEDGIIYAALAGTR-VE |
| CvDsrE | 1 | MKFALQIN-EGPYQHQASDSAYQFAKALEKGHEIFRVFFYHDGVNNSTRLTTPO |

| CvDsrF | 60 | IGMKNESPTYRTLGDYEVRRJIYORDSLEARGLTQQDLVEIAFEDMETEEEFDNIVEVID |
| CvDsrH | 48 | --------SQVTEALGKLKL------YLGPDLKARGFSDE------------------RVIPGISWD |
| CvDsrE | 57 | DDRHIVNRWAEELAEQYELD-MVVCVAAAQRRGIVDEGEASRNGKD---ATNIHPKFRISG |

| CvDsrF | 120 | SARVSELMNESDAVFSF* |
| CvDsrH | 85 | YAGFVDLTTEDTIVQAWL* |
| CvDsrE | 113 | LGQLVEAAIQADRLVVFGD* |

(b)
| CvDsrC | 1 | MADTIEVDGKQFAVDEEGYLSNLNDWVPGVADVMAKQDNL--------ELTEEHWDIINF |
| EcYccK | 13 | LIKVVIIMLIFE GK E IETDTEGYLKESSQWEPLAVVAENEGI--------SLSPEHWEVVRF |
| HaeYccK | 1 | MLNINGIEVETDKDGYLHSQQWNEDAAARAI AQLESI--------ELTDAHWEVIYF |
| AfDsvC | 1 | MPELEVKGKLRIDEDGEFLQDWEEWDEEVAEALAKDTRFSPQPIELTEEHWKIIRY |
| DvDsvC | 1 | MAEVTYKGKSFEVEDGEFLRFDDWCPEWVEYVKESEGIS--------DISPDHQKLIIDF |
| PiDsrC | 1 | MPVKCPGEYKVDDVTIIILEECFLSNPEWNEKVAEWMARELEG--------IQKMTEDHWKVVKY |

| CvDsrC | 56 | LREYYEEYQIAPA VRVLTKAVGKKLGKEKGNSKYLYSLEFPYSEAKQACREFAGLPKPTGCV* |
| EcYccK | 72 | VRDFYLEFNTSPAIRM LVKAMANKFGEEEKGNSRYLYRLFPPKGP AKQATKLAGLPKPVKCI* |
| HaeYccK | 53 | VRDFYQEYNTSPAIRM LVKAMAEKLGADKGNSSYLOHLEPPEGPAKQATKLAGLPKFAKCL* |
| AfDsvC | 60 | LR DYBIKYGVAEPVRMLVKHCKKEV-RPDCNLQYIYKIFPQGPAKDACRIAGLPKPTGCV* |
| DvDsvC | 56 | LQDYYKKNGIAPMVRIILSKNTGFKL--------KEVYELFPSGBP KGACKMAGLPKPTGCV* |
| PiDsrC | 62 | LREYWETYGVQPPIKMLLKETGFTL--------EQIYQLEEPSGBANGACKMAGAPKPTGCV* |

(c)
| CvDsrM | 4 | LTNEYAAPLFYFAAFVLVAGLARKIIQYALTPAPLKIPPTPAPTTGRGVVLRMMREVVLFE |
| EcNarI | 4 | LNMEFEDIYPYJAGAVFLIGSWLRYDYGGQTWRAAS-------------------------- |
| DvHmc5 | 7 | GRMLNASLLLVFFGGILLARVIWYIRGLDWRLDRVAYKPHLAI GLQGAVQSALKWLVPFGT- |

Helix a

↓ Helix b ↓

| CvDsrM | 64 | SLFRGNKWTWIFGWMFHFGIL----FLVTLRHLRYFIEPVW--------FPIQLVQFGIYGG-- |
| EcNarI | 40 | SQMLDRKGMINIASNILPHICGI----LGIFVGHFFGMLTPHM MYEAWLPIEVKQKAMFFAGGA |
| DvHmc5 | 66 | YSWRQQPFTTVAFFLFHIGAVLVPLFLAGHN-VILEERF--GFSLPALPMGVADTLT---- |

Helix c                      Helix d

| CvDsrM | 114 | ---IAMVVGLAGLWARRFLVDRVKYITSTDHIILGILLVAIGCTGLMKFV FHVDVVEIKT |
| EcNarI | 97 | SGVLCLIGGVLLLKRRLFSPRVRATTTGADIILLSILLVTOCALGLLTIPESAQHMDGSEM |
| DvHmc5 | 120 | ---VLAITTGLVMIALRRIALTEVRILTTGYDWELLAVSAAPFVTGF LA--------- |

↓ Helix e ↓

| CvDsrM | 172 | ----------FFLGLYSYDIEPLPADELLLLLALLLVALLMIIFPYSKLLHMPGLFFSP--T |
| EcNarI | 157 | MKLVGWAQS VVTFHG GASQHLDGVAEIFRLHLVLGMTLFLLFPE SRLIHWSVPVEY--L |
| DvHmc5 | 174 | ----------RLHVG DYDTWLLAHIIITGELFLIVAPETKL SHIVLFFMSRGQL |

| CvDsrM | 221 | RNQVDNPREQRHIAPWARKLEQS* |
| EcNarI | 215 | TRKYQLVRA RH* |
| DvHmc5 | 208 | GMDYAIKR GGA* |

Fig. 2. For legend see facing page.

(d) CvDsrK 1 MAKATFEVPELTQYTEVPSVTPGA MAHSAPFKSKEDFQTPLGFG--ELVDDWQTKAIDK
     DvHmc6 1 MPEGKFCNRKPVNTEEDLKALLGDKGGAQYYKEEMEELEVDO
     MsbHdrD 1 MAKRTPSIDT

CvDsrK 59 MGELLGKYRSLR----VYMDACVRGGACTDKCHYFLGTG-DPKNMVAR-QDLMRKVYRR
     DvHmc6 42 EALWANIEKTCQSRTKTWLEICARCGMCADSCLYRVNDRDPKQVPAYLIQSTLGEIIRR
     MsbHdrD 11 KNLTAVQLMELD--------ACVRCGECVKWCPTYAASGGHQGLAPRDK-ILRWRQYMNK

CvDsrK 113 YFTLSGKLFPKLVGAEDETEEVLDDWYSYFHQCSCRRCSVFCPYGMDTAEISMAAREIM
     DvHmc6 102 KGKVD----------TQFMNHAMEVAWSQCTCNNRCGQYCPHGDMGVMSYLRLGLL
     MsbHdrD 62 SYGLKAKLFGP----QEVSPELEEFKDDVHGCITTSGVCATVCEAGINTVEIWEAIRTNL

CvDsrK 173 DSIGVGQKYCNEIIGKVYKIGNNLGLPGPALVOTLEGLEEDVYDQTGVKVRYPIDEEGAE
     DvHmc6 149 YSQGFVPWELKIGSGMHRVYCAQMDVTTEDWVETCEWMAEEQQEQ-WPGLSIPVDVENAD
     MsbHdrD 118 VKKGIGPYGK--QSAFPKLVCQYNPYMKDQKRLAWVPDVKIEDKADIYFTGCTAGY

CvDsrK 233 ILLVTPSADFFAEPHVDGLIGYGKVFHEAGVSWTLSYASEAANFGMFIGSYDNMRRVAQ
     DvHmc6 208 IMYVLNAREPKHYP--EDVAEAAIIIFHIAGENWTVSEGWEQTSLAMFAGDWAACKMQVE
     MsbHdrD 176 NQLALAFATSRLN---KLGIKFAMLGEE---EWCCGS-------ALIRTGQVHVDDVARELA

CvDsrK 293 RIREAAIKLVKRIVFGECHAWRVAYSFLNTLAGPWDFLDPRYPVP-QHICEFTYE LIQ
     DvHmc6 266 RVYAAIERLPKCVVGTECGHAHRASAIEGPYWAG---YEDGKTPAPWLHYVEWVAMALR
     MsbHdrD 226 RHINVEALQKKGAKKVLFAAGCFRAAKIDWPRLLG--------KELPFEVIHITQFLADLIQ

CvDsrK 352 QGKLNLDSENDDKVLTYHDSONVARASRMGSTPGGQFEIPRAIRATCNNFYMDDEDTI
     DvHmc6 323 TGKIKIDPEKRIKEPVTLQSCNYIRNHGLAKCT---------REIMSYIADBREMTPN--
     MsbHdrD 280 ADKIKWEKP--INKTITYHDRCHLGRHVGVFNAPR--------YVISHIPGVKFVEMDRS--

CvDsrK 412 RDRTECSSGGGCLLTDDLMELRVKGAKPRLDALNNVIQEKGVTHMAATCAICKSQLTKVL
     DvHmc6 374 REHNYCSSGGGEE--FNGIGKFRKQRNKAL-QTKRDQILATGAKLVVAPSHNCWDAIRDLE
     MsbHdrD 330 KEFORCGGABSS-VKAGMPDLAVAMGESR----VKDALETNADILSSACPFCKRNISDGR

CvDsrK 472 PYYGLEMDQIVSVHQIVSNAIMLTPGPDDDEDEDDEPDLDDPNEEEEETDEAA*
     DvHmc6 431 EEYRIGRWESFLKPLIKMAIIDPEHLRPEEE*
     MsbHdrD 385 DALKSDIVVEDIIIELVAEALGLSTS*

Fig. 2. Alignment of amino acid sequences of proteins encoded in the C. vinosum dsr locus with similar polypeptides. The alignments were produced using the program CLUSTAL W (Higgins & Sharp, 1989) using the Mutation Data Matrix (Dayhoff et al., 1978; George et al., 1990). In all alignments strictly conserved residues are printed white on black and similar amino acids (similarity groups after Dayhoff et al., 1978) are boxed. (a) Alignment of DsrF, DsrH and DsrE. (b) Alignment of DsrC with similar proteins from E. coli (Ec), H. influenzae (Hae), A. fulgidus (Af), D. vulgaris (Dv) and P. islandicum (Pi). (c) Alignment of DsrM with the γ subunit of E. coli nitrate reductase (EcNarI) and the hmc5-encoded polypeptide from D. vulgaris (DvHmc5). Potential membrane-spanning hydrophobic sequences (helices a–e) are boxed. Potential haem-b-ligating histidine residues are highlighted by arrows. This alignment was done such that the hydrophobic stretches were first aligned and fixed. (d) Alignment of DsrK with the D. vulgaris hmc6-encoded protein (DvHmc6) and the HdrD subunit of Ms. barkeri heterodisulfide reductase (MsbHdrD). Strictly conserved cysteine residues are highlighted by arrows.

HdrD of heterodisulfide reductase from Ms. barkeri (Künkel et al., 1997), to the hmc6 gene product from D. vulgaris (Rossi et al., 1993) and to several hypothetical proteins from A. fulgidus (Table 2, Fig. 2d). Like dsrM and dsrK in C. vinosum, hdrE/hdrD, hmc5/hmc6 and the respective A. fulgidus genes are juxtapositioned. The

N-terminal parts of DsrK, HdrD and Hmc6 have similarity to HdrC of heterodisulfide reductase from Mb. thermoautotrophicum (Hedderich et al., 1994) while their C termini are similar to the HdrB subunit of the latter enzyme. In methanogenic archaea, heterodisulfide reductases catalyse the reduction of

A.S. POTT and C. DAHL

37D  
21D/31D  
34D  
35D 34S  

dsrA  
EcoRV Ncol Xhol EcoRI  

dsrB  
Ncol BglII Smal SalI  

dsrE dsrF dsrH dsrC  
BglII SalI EcoRI Smal  

dsrM dsrK'  
Ncol SalI SalI Ncol SalI Ncol SalI SalI  
Xhol Ncol  

Bglll  

500 bp  

Fig. 3. Schematic representation of the *dsr* locus in *C. vinosum*. The positions of insertion sites of the KmΩ cartridge in the *C. vinosum* single-crossover mutant 34S and the double-crossover mutants 37D, 21D/31D, 34D and 35D are indicated.

(a)  
![Graph](https://i.imgur.com/placeholder.png)  

(b)  
![Graph](https://i.imgur.com/placeholder.png)  

Fig. 4. Sulfur accumulation and oxidation by the wild-type and *dsr* mutants during photolithoautotrophic growth on sulfide (a) and thiosulfate (b). (a) Sulfur content of cultures of the wild-type and *dsr* mutants. The representative experiments shown were performed in completely filled 250 ml glass bottles on mineral salts medium with an initial sulfide concentration of 5 mM. Sulfide is not completely recovered as sulfur due to loss of gaseous H₂S during sampling. Sulfide and sulfate concentrations were also measured but are not shown for clarity. Note that the time axis is not to scale between 36 and 120 h. The protein content of the cultures was 120·0, 76·3, 132·5, 95·0 and 65·0 μg ml⁻¹ for SM50, 37D, 21D, 34D, 35D and 34S, respectively. (b) Thiosulfate oxidation by the wild-type compared to thiosulfate utilization by mutant 31D. The representative experiments shown were performed in pH-controlled fermenter cultures as described by Dahl (1996). •, Sulfur; □, thiosulfate; ▲, sulfate; △, tetrathionate. Note that the time axis is not to scale between 30 and 74 h for mutant 31D.

Co-M-S-S-B-Co to reduced coenzyme M and reduced coenzyme B (Künkel et al., 1997). The related thiol: fumarate reductases from *Mb. thermoautotrophicum* and *Mc. jannaschii* catalyse the reduction of fumarate with coenzymes M and B as electron donors (Bobik & Wolfe, 1989; Bult et al., 1996; Künkel et al., 1997). The N-terminal parts of the proteins similar to DsrK contain two copies of the sequence motif CX₂CX₂CX₃C indicating the presence of two [4Fe–4S] clusters. In pyridine-nucleotide-dependent thioredoxin reductases the two redox-active cysteine residues are separated by two amino acids in the sequence VXXCATCD (Russel & Model, 1988; Lübbbers & Andresen, 1993; Cohen et al., 1993; Jacquot et al., 1994). A comparable arrangement is found near the C terminus of DsrK,HdrD and Hmc6 (Fig. 2d) and could constitute the redox-active disulfide of these proteins. It is furthermore interesting to note that amino acids 188–214 of *C. vinosum* DsrK share similarity with amino acids 7–33 of glycine reductase

selenoprotein PA from *Eubacterium acidaminophilum* and several *Clostridium* species. After its oxidation during the reductive deamination of glycine this protein is re-reduced by the thioredoxin system (Dietrichs et al., 1991).

Construction and characterization of *C. vinosum* strains with mutations in *dsr* genes

*C. vinosum* strains with defined mutations in the *dsr* genes were constructed by insertion of a kanamycin Ω interposon (Fellay et al., 1987) (Fig. 3), which leads to premature termination of transcription of the affected gene and downstream genes in the same transcription unit. As expected for a classical purple sulfur bacterium (Brune, 1989), the wild-type oxidized sulfide first to intracellular sulfur, which was further oxidized to sulfate when sulfide was depleted. Mutants 37D, 31D, 34D and 35D, with polar mutations upstream of *dsrA*,

Sulfur oxidation in Chromatium vinosum

oxidize it. Sulfite oxidation was not affected in the *dsr* mutants (not shown). All mutants also grew normally under photoorganoheterotrophic conditions (not shown).

**Sulfide induction experiments and mRNA analysis**

For *C. vinosum* SM50 (wild-type), two different distinct mRNA bands of 2.5 and 1.4 kb were detected with a *dsrAB* (nucleotides 1298–2500) probe (Fig. 5). Distinct bands over 2.6 kb that would have indicated a common transcript *dsrABEFHCMK* were not detected, even after prolonged film exposure. The formation of a common transcript cannot, however, be excluded, because such a long mRNA may be very unstable. As hybridization with the *dsrAB* probe led to the detection of a smear reaching up to a size of approximately 6.0 kb, degradation of RNA may indeed have occurred and the bands at 2.5 kb and 1.4 kb could represent specific or even unspecific degradation products. Nevertheless, hybridization with the *dsrAB* probe yielded important information when RNA from photoheterotrophically grown and from sulfide-induced cells was compared. The latter cells were harvested when they had just begun to oxidize stored sulfur. Photoheterotrophically growing *C. vinosum* contained far lesser amounts of mRNA hybridizing with *dsrAB* than sulfur-oxidizing cells (Fig. 5). The finding that *dsrAB* are transcribed at a low level in the absence of reduced sulfur compounds is not in agreement with earlier results of Schedel *et al.* (1979) who were unable to detect sulfite reductase in malate-grown cells. The level of sulfite reductase in these cells might, however, have been below the detection limit or further regulation mechanisms may work at the post-transcriptional level.

A *dsrC* probe (nucleotides 4086–4401) reacted with an mRNA of approximately 0.9 kb. In addition, the wild-type showed a faint smear with a maximum size of 2.5 kb and very faint signals resulting from RNAs of even bigger sizes. In contrast to the hybridizations with *dsrAB*, the intensity of the signal was independent of the presence of sulfide in the medium. Mutant 21D contained the same 0.9 kb band that was found for the wild-type (Fig. 5).

**DISCUSSION**

The nucleotide sequence of the *C. vinosum dsr* locus at first sight suggests that the *dsrABEFHCMK* genes form a single transcriptional unit. A sequence resembling σ⁷⁰ promoters was found upstream of *dsrA* and the following genes are separated by short intergenic sequences in which further putative promoter sequences are not apparent. A distinct transcript long enough to cover the whole sequenced *dsr* region was, however, not detected and mRNAs of different sizes reacted with *dsrAB* and *dsrC* probes. As the occurrence of these mRNAs is not co-ordinately regulated in *C. vinosum*, they cannot result from the degradation of a common, longer transcript covering *dsrAB* as well as *dsrC* unless the

and in *dsrB*, *dsrH* and *dsrM*, respectively, oxidized sulfide to sulfur with rates comparable to those of the wild-type; however, the ability of the cells to oxidize intracellular sulfur was completely abolished (Fig. 4a). Mutant 34S exhibited a greatly reduced rate of intracellular sulfur oxidation. This mutant arose by single crossover recombination with plasmid pAP34 and should be able to form DsrA, -B, -E, -F, -H and -C. The very C-terminal end of DsrM is altered probably not changing the protein’s properties, and DsrK cannot be formed unless the respective gene is transcribed from a promoter downstream of the Km Ω insertion site in the second, inactivated copy of *dsrH*. The effect of a mutation in the *dsr* genes on thiosulfate oxidation is illustrated for mutant 31D in Fig. 4(b). Typical for *C. vinosum* at pH 7.0 (Smith, 1966), the wild-type and the mutant oxidized part of the thiosulfate to tetrathionate, which was not further metabolized. The other part of the thiosulfate is split to sulfide and sulfite. Whilst sulfite is directly oxidized to sulfate, sulfide first appears as intracellular sulfur and is then oxidized to sulfate (Trüper & Fischer, 1982; Brune, 1989, 1995a). The wild-type showed this typical oxidation pattern, whereas the mutant accumulated sulfur but was not able to further

A.S. POTT and C. DAHL

Model A                                                                 Model B

Periplasm

H₂S                                                                 H₂S
Flavocytochrome c or                                              Flavocytochrome c or
Sulfide:quinone oxidoreductase                                      Sulfide:quinone oxidoreductase
2e⁻ + 2H⁺                                                            2e⁻ + 2H⁺
2e⁻ + 1H⁺                                                            XSS⁻
S⁰                                                                  S⁰
HS⁻                                                                 XS⁻
6 H⁺                                                                2 H⁺
3 QH₂                                                               3 QH₂
DsrM                                                                DsrM
Cyt bₗ                                                              Cyt bₗ
↑                                                                   ↓
Cyt bₕ                                                              Cyt bₕ
SO₃²⁻                                                               SO₃²⁻
HS⁻                                                                 XSS⁻
7 H⁺                                                                H⁺
3 H₂O                                                               6 H⁺
DsrK                                                                DsrK
[4Fe-4S]                                                            [4Fe-4S]
[4Fe-4S]                                                            [4Fe-4S]
DsrA                                                                DsrA
Sirohaem-                                                           Sirohaem-
[4Fe-4S]                                                            [4Fe-4S]
[4Fe-4S]                                                            [4Fe-4S]
DsrB                                                                DsrB
[4Fe-4S]                                                            [4Fe-4S]
DsrC                                                                DsrC
DsrE                                                                DsrE
DsrF                                                                DsrF
DsrH                                                                DsrH

Cytoplasm

Fig. 6. Models for sulfur oxidation in *C. vinosum* and involvement of the proteins encoded in the *dsr* locus. A sulfur globule is schematically represented with its envelope consisting of the three different proteins SgpA, SgpB and SgpC (Brune, 1995b; Pattaragulwanit et al., 1998). By analogy to the enzymes from sulfate-reducers, *C. vinosum* sirohaem sulfite reductase is depicted as an α₂β₂-structured protein binding two sirohaem prosthetic groups. Model A allows electron flow from cytoplasmic sulfite reductase to the integral membrane protein DsrM via the iron-sulfur protein DsrK. Model B proposes the shuttling of sulfur from the extracytoplasmic sulfur globule to the cytoplasm via a perthiol acting as a carrier molecule. DsrM and DsrK are involved in the regeneration of a thiol and sulfide from the hypothetical perthiol in the cytoplasm.

stability of the processed transcripts is different under different growth conditions. As the same 0.9 kb RNA reacts with *dsrC* in the wild-type and in mutant 21D, we conclude that transcription of the *dsr* genes can start at least two different promoters, one of which must be located downstream of *dsrB*, the insertion site of the interposon in mutant 21D (Fig. 2). In this respect it is interesting to note that the sequence between *dsrH* and *dsrC* has a conspicuously low G+C content of only 35.7 mol %, which may indicate a function in transcription initiation or regulation. The smear of longer RNAs detected in the wild-type with the *dsrC* and the *dsrAB* probes indicates that transcripts covering the whole sequenced *dsr* locus may in addition be formed in vivo. The 2.5 kb mRNA detected with *dsrAB* represents the length expected for a transcript originating from the putative promoter upstream of *dsrA* and ending at the hairpin loop structures downstream of *dsrB*. A sequence conforming to the proposed consensus for RNase E cleavage sites (G/A)′A′TT(A/T) (Ehretsmann et al.,

1992) occurs immediately downstream of these inverted repeats (Fig. 1). It has been suggested that RNase E requires a hairpin loop structure in close proximity to its recognition site for maximal cleavage rates. Further experimentation is necessary to clarify whether the 2.5 kb transcript arises from transcription termination at the hairpin loops or from degradation of a longer transcript.

Strains with interposon mutations in the *dsr* genes grew normally under photoheterotrophic conditions. The oxidation of sulfide to sulfur, of thiosulfate to tetra-thionate, of sulfite to sulfate, and the initial splitting reaction during thiosulfate oxidation to sulfate were not affected in the *dsr* mutants but all mutant strains showed a reproducible inability to oxidize intracellularly stored sulfur under photolithoautotrophic growth conditions. This provides strong evidence for the participation of the *dsrABEFHCMK*-encoded proteins in sulfur oxidation. As already discussed, Northern hybridizations

1890
with the $dsrC$ probe indicate that transcription of $dsr$ genes can start at (a) promoter(s) downstream of $dsrB$. The failure of mutant 21D to oxidize sulfur can, therefore, be attributed to the lack of sulfite reductase and is not due to a polar effect of the interposon in $dsrB$ on the downstream genes. Mutant 37D carries an insertion 20 bp upstream of the putative promoter sequence upstream of $dsrA$. Although the putative promoter is intact, this mutant is not able to oxidize internal sulfur. This could either be explained by the fact that the true promoter lies upstream of the Km' insertion site or be due to the interruption of a region essential for transcriptional regulation. In future, construction of strains with mutations upstream and downstream of the sequenced $dsr$ genes will show whether further proteins involved in sulfur oxidation are encoded in this region.

Amino acid sequence analysis suggests that the $dsrABEFHCMK$ gene products are all cytoplasmic or membrane-bound. The sulfur globules of C. vinosum are, however, located extracytoplasmically (Pattaragulwanit et al., 1998), implying that sulfur in some way has to be transported across the membrane – probably in a reductively activated form – before it can serve as a substrate for the $dsr$-encoded proteins. The idea that sulfur has to be reductively activated before further oxidation is supported by the finding of Hurlbert (1968) that thiol-binding reagents like N-ethylmaleimide inhibit sulfur oxidation in C. vinosum.

As pointed out above, DsrM is similar to the membrane-bound haem-$b$-binding $\gamma$ subunits of dissimilatory nitrate reductases, the direct electron donor of which is ubiquinol or menaquinol (Craske & Ferguson, 1986; Berks et al., 1995). In the course of quinol oxidation, protons are released to the periplasm by these poly-peptides. In C. vinosum, DsrM could either function as a quinol oxidase like the nitrate reductase $\gamma$ subunits or operate in the reverse direction as a quinone reductase. For both cases, models for the pathway of sulfur oxidation can be envisaged and are illustrated in Fig. 6.

In model A it is assumed that extracytoplasmic sulfur is made available for cytoplasmic sulfite reductase by reduction to the level of sulfide which either diffuses into or is actively transported to the cytoplasm. The [4Fe–4S]-cluster-containing DsrK could function in electron transport from sulfite reductase to DsrM. DsrM could then feed these electrons into the photosynthetic electron transport chain. This speculation on the function of DsrM and DsrK was prompted by a proposal of Rossi et al. (1993) that the related proteins Hmc5 and Hmc6 function in the opposite direction in D. vulgaris, transferring electrons from periplasmic hydrogenase to the cytoplasmic sulfate-reducing enzymes. In strict analogy to the nitrate reductase $\gamma$ subunits, DsrM working as quinone reductase would take up protons for QH$_2$ generation from the periplasm, i.e. it would work against the photosynthetic $\Delta p$. Furthermore, electron transfer from cytochrome $b_{\mathrm{H}}$ to cytochrome $b_{\mathrm{L}}$ would have to be proposed. So far, model A does not consider the significant similarity of DsrK to archaeal

heterodisulfide reductases, which implies a function in the oxidation of two thiols or the reduction of a di- or persulfide. One possibility is that the electrons from sulfide oxidation are used to reduce a hypothetical disulfide and that the resulting two thiols are oxidized by DsrK, regenerating the disulfide. DsrC contains a single conserved cysteine residue which could play a role in such a scenario. An additional weakness of model A is that it does not explain how the reductive activation of extracytoplasmic sulfur is achieved. Furthermore, it is unlikely that sulfur is indeed reduced back to hydrogen sulfide. First, the problem arises as to how the cell would be able to prevent immediate reoxidation of periplasmically developing sulfide to sulfur. Second, it is difficult to explain why oxidation of sulfide to sulfite catalysed by sulfite reductase should not occur in parallel to sulfide oxidation resulting in sulfur globule formation. If the components and compounds involved in sulfur formation and those functioning in sulfur oxidation were different, the cell would be able to perform and regulate each step independently.

Model B in Fig. 6 takes these considerations into account and suggests that DsrK plays an indirect role in the reductive activation of the relatively inaccessible stored sulfur. DsrM could function as quinol oxidase and donate electrons to DsrK, which in turn could reduce a hypothetical disulfide. We have depicted the simplest case, i.e. that the disulfide is a perthiol which upon reduction yields a thiol and sulfide. Sulfide could then be directly oxidized by cytoplasmic sulfite reductase whilst the thiol could be transported to the extracytoplasmic space and nucleophilically attack sulfur, thereby regenerating the perthiol. After transport of this compound to the cytoplasm, the cycle could start again. This model is supported by findings of Bartsch et al. (1996) who reported that C. vinosum specifically accumulates the persulfide form of glutathione amide as well as three further as yet unidentified perthiols when growing photoautotrophically on sulfide. Bartsch et al. (1996) speculated that the observed perthiols could act as carrier molecules of sulfur to and from the sulfur globules. To our knowledge, information about transport of thiols and perthiols across membranes is not available for phototrophic bacteria, but such a transport is feasible and could be mediated by specialized transport proteins.

Working as a quinol oxidase, DsrM would extrude protons to the periplasm. When C. vinosum fixes CO$_2$ in the presence of reduced sulfur compounds, photosynthetic electron flow can be considered essentially linear and the proposed system could contribute to $\Delta p$. On the other hand, a high $\Delta p$ generated by cyclic electron flow, especially under conditions where CO$_2$ assimilation is limited, could exert a thermodynamic back pressure on the system and lower the rate of sulfur oxidation. Without knowledge about the pool sizes, redox potentials and regulation of the components involved in such a system, it is at present not possible to decide if it could function in vivo. Model B does not
A. S. POTT and C. DAHL

answer the question of how the electrons resulting from sulfide oxidation by sulfite reductase are fed into the electron transport chain.

In both models it remains unclear which reaction could yield the electrons for reductive activation of stored sulfur. Once sulfur oxidation gets started, the electrons could indirectly be supplied by the sulfite reductase reaction or by the further oxidation of sulfite to sulfate (E₀′ = −516 mV). The Ms. barkeri heterodisulfide reductase and probably also D. vulgaris Hmc6 receive electrons from periplasmic hydrogenase. hmc5- and hmc6-homologous genes are intercalated between the hupSL genes encoding the small and large subunits of the stable, constitutively expressed hydrogenase from Thiocapsa roseopersicina BBS (Rákhely et al., 1998). Taken together, these findings may point to a connection between sulfur metabolism and hydrogenase in purple sulfur bacteria. In these organisms, a function has not yet been assigned to hydrogenase (Albracht, 1994).

We are far from clearly understanding the functional mechanism of sulfur oxidation in phototrophic sulfur bacteria and much more work is needed to exactly define the structural and catalytic properties of the dsr-encoded polypeptides in C. vinosum. However, the results presented in this paper allowed important insights into the oxidation of intracellularly stored sulfur, a pathway that has so far been very difficult to approach experimentally. The presented hypotheses are speculative but should provide incentives for the design of further experiments to study this interesting pathway.

## ACKNOWLEDGEMENTS

This work was supported by grant Da 351/1-1 from the Deutsche Forschungsgemeinschaft. Excellent technical assistance by Dorit Zischka is gratefully acknowledged. We are indebted to Hans G. Trüper, Daniel C. Brune and Werner Klipp for stimulating discussions. Thomas Brüser is gratefully acknowledged for helpful suggestions concerning model B. We thank Daniel Burdziak for help with DNA sequencing. A. S. Pott was supported by a scholarship of the Studienstiftung des Deutschen Volkes.

## REFERENCES

Albracht, S. P. J. (1994). Nickel hydrogenases: in search of the active site. *Biochim Biophys Acta* 1188, 167–204.

Altschul, S. F., Gish, W., Miller, W., Myers, E. W. & Lipman, D. J. (1997). Gapped BLAST and Psi-BLAST: a new generation of protein database search programs. *Nucleic Acids Res* 25, 3389–3402.

Arendsen, A. F., Verhagen, M. F. J. M., Wolbert, R. B. G., Pierik, A. J., Stams, A. J. M., Jetten, M. S. M. & Hagen, W. R. (1993). The dissimilatory sulfite reductase from Desulfosarcina variabilis is a desulforubidin containing uncoupled metalated sirohemes and S = 9/2 iron-sulfur clusters. *Biochemistry* 32, 10323–10330.

Ausubel, F. A., Brent, R., Kingston, R. E., Moore, D. D., Seidman,

J. G., Smith, J. A. & Struhl, K. (1997). *Current Protocols in Molecular Biology*. New York: John Wiley.

Bartsch, R. G., Newton, G. L., Sherrill, C. & Fahey, R. C. (1996). Glutathione amide and its perthiol in anaerobic sulfur bacteria. *J Bacteriol* 178, 4742–4746.

Bazaral, M. & Helinski, D. R. (1968). Circular DNA forms of colicinogenic factors E1, E2 and E3 from Escherichia coli. *J Mol Biol* 36, 185–194.

Berks, B. C., Page, M. D., Richardson, D. J., Reilly, A., Cavill, A., Outen, F. & Ferguson, S. J. (1995). Sequence analysis of subunits of the membrane-bound nitrate reductase from a denitrifying bacterium: the integral membrane subunit provides a prototype for the dihaem electron-carrying arm of a redox loop. *Mol Microbiol* 15, 319–331.

Bobik, T. A. & Wolfe, R. S. (1988). Physiological importance of the heterodisulfide of coenzyme M and 7-mercaptoheptanoyl-threonine phosphate in the reduction of carbon dioxide to methane in Methanobacterium. *Proc Natl Acad Sci USA* 85, 60–63.

Brune, D. C. (1989). Sulfur oxidation by phototrophic bacteria. *Biochim Biophys Acta* 975, 189–221.

Brune, D. C. (1995a). Sulfur compounds as photosynthetic electron donors. In *Anoxygenic Photosynthetic Bacteria*, pp. 847–870. Edited by R. E. Blankenship, M. T. Madigan & C. E. Bauer. Dordrecht: Kluwer.

Brune, D. C. (1995b). Isolation and characterization of sulfur globule proteins from Chromatium vinosum and Thiocapsa roseopersicina. *Arch Microbiol* 163, 391–399.

Bult, C. J., White, O., Olsen, G. J. & 37 other authors (1996). Complete genome sequence of the methanogenic archaeon, Methanococcus jannaschii. *Science* 273, 1058–1073.

Chen, Z. W., Koh, M., van Driessche, G., van Beeumen, J. J., Bartsch, R. G., Meyer, T. E., Cusanovich, M. A. & Mathews, F. S. (1994). The structure of flavocytochrome c sulfide dehydrogenase from a purple phototrophic bacterium. *Science* 266, 430–432.

Cohen, G., Yanko, M., Mislovati, M., Argaman, A., Schreiber, R., Av-Gay, Y. & Aharonowitz, Y. (1993). Thioredoxin-thioredoxin reductase system of Streptomyces clavuligerus: sequences, expression and organization of the genes. *J Bacteriol* 175, 5159–5167.

Craske, A. L. & Ferguson, S. J. (1986). The respiratory nitrate reductase of Paracoccus denitrificans. Molecular characterization and kinetic properties. *Eur J Biochem* 158, 429–436.

Dahl, C. (1996). Insertional gene inactivation in a phototrophic sulfur bacterium: APS-reductase-deficient mutants of Chromatium vinosum. *Microbiology* 142, 3363–3372.

Dahl, C. & Trüper, H. G. (1994). Enzymes of dissimilatory sulfide oxidation in phototrophic bacteria. *Methods Enzymol* 243, 400–421.

Dahl, C., Kredich, N. M., Deutzmann, R. & Trüper, H. G. (1993). Dissimilatory sulphite reductase from Archaeoglobus fulgidus: physico-chemical properties of the enzyme and cloning, sequencing and analysis of the reductase genes. *J Gen Microbiol* 139, 1817–1828.

Dahl, C., Speich, N. & Trüper, H. G. (1994). Enzymology and molecular biology of sulfate reduction in the extremely thermophilic archaeon Archaeoglobus fulgidus. *Methods Enzymol* 243, 331–349.

Dayhoff, M. O., Schwartz, R. M. & Orcutt, B. C. (1978). A model of evolutionary change in proteins. In *Atlas of Protein Sequence and Structure*, pp. 345–352. Edited by M. O. Dayhoff. Washington, DC: National Biochemical Research Foundation.

Dietrichs, D., Meyer, M., Rieth, M. & Andresen, J. R. (1991). Interaction of selenoprotein Pₐ and the thioredoxin system, components of the NADPH-dependent reduction of glycine in Eubacterium acidaminophilum and Clostridium litoralis. *J Bacteriol* 173, 5983–5991.

Dolata, M. M., van Beeumen, J. J., Ambler, R. P., Meyer, T. E. & Cusanovich, M. A. (1993). Nucleotide sequence of the heme subunit of flavocytochrome c from the purple phototrophic bacterium, Chromatium vinosum. A 2·6 kilobase pair DNA fragment contains two multiheme cytochromes, a flavoprotein, and a homolog of human ankyrin. *J Biol Chem* 268, 14426–14431.

Ehretsmann, C. P., Carpousis, A. J. & Krisch, H. M. (1992). Specificity of Escherichia coli endoribonuclease E: in vivo and in vitro analysis of mutants in a bacteriophage T4 mRNA processing site. *Genes Dev* 6, 149–159.

Fellay, R., Frey, J. & Krisch, H. M. (1987). Interposon mutagenesis of soil and water bacteria: a family of DNA fragments designed for in vivo insertional mutagenesis of Gram-negative bacteria. *Gene* 52, 147–154.

Fischer, U. (1989). Enzymatic steps in dissimilatory sulfur metabolism by whole cells of anoxyphotobacteria. In *Biogenic Sulfur in the Environment*, pp. 262–279. Edited by E. Saltzman & W. Cooper. Washington, DC: American Chemical Society.

George, D. G., Barker, W. C. & Hunt, L. T. (1990). Mutation data matrix and its uses. *Methods Enzymol* 183, 333–351.

Hanahan, D. (1983). Studies on transformation of Escherichia coli with plasmids. *J Mol Biol* 166, 557–580.

Hedderich, R., Koch, J., Linder, D. & Thauer, R. K. (1994). The heterodisulfide reductase from Methanobacterium thermoautotrophicum contains sequence motifs characteristic of pyridine-nucleotide-dependent thioredoxin reductases. *Eur J Biochem* 225, 253–261.

Higgins, D. G. & Sharp, P. M. (1989). CLUSTAL: a package for performing multiple sequence alignment on a microcomputer. *Gene* 73, 237–244.

Hipp, W. M., Pott, A. S., Thum-Schmitz, N., Faath, I., Dahl, C. & Trüper, H. G. (1997). Towards the phylogeny of APS reductases and sirohaem sulfite reductases in sulfate-reducing and sulfur-oxidizing prokaryotes. *Microbiology* 143, 2891–2902.

Hofman, K. & Stoffel, W. (1993). TMbase—a database of membrane spanning protein segments. *Biol Chem Hoppe-Seyler* 347, 166–171.

Hurlbert, R. E. (1968). Effect of thiol-binding reagents on the metabolism of Chromatium vinosum D. *J Bacteriol* 95, 1706–1712.

Jacquot, J. P., Rivera-Madrid, R., Marinho, P., Kollarova, M., Le Marechal, P., Miginiac-Maslow, M. & Meyer, Y. (1994). Arabidopsis thaliana NADPH thioredoxin reductase cDNA characterization and expression of the recombinant protein in Escherichia coli. *J Mol Biol* 235, 1357–1363.

Karkhoff-Schweizer, R. R., Bruschi, M. & Voordouw, G. (1993). Expression of the γ-subunit gene of desulfoviridin-type dissimilatory sulfite reductase and of the α- and β-subunit genes is not coordinately regulated. *Eur J Biochem* 211, 501–507.

Klenk, H.-P., Clayton, R. A., Tomb, J.-F. & 48 other authors (1997). The complete genome sequence of the hyperthermophilic, sulphate-reducing archaeon Archaeoglobus fulgidus. *Nature* 390, 364–370.

Künkel, A., Vaupel, M., Heim, S., Thauer, R. K. & Hedderich, R. (1997). Heterodisulfide reductase from methanol-grown cells of

Methanosarcina barkeri is not a flavoenzyme. *Eur J Biochem* 244, 226–234.

Lisser, S. & Margalit, H. (1993). Compilation of E. coli mRNA promoter sequences. *Nucleic Acids Res* 21, 1507–1516.

Lübbers, M. & Andresen, J. R. (1993). Components of glycine reductase from Eubacterium acidaminophilum. Cloning, sequencing and identification of the genes for thioredoxin reductase, thioredoxin and selenoprotein Pₐ. *Eur J Biochem* 217, 791–798.

Molitor, M., Dahl, C., Molitor, I., Schäfer, U., Speich, N., Huber, R., Deutzmann, R. & Trüper, H. G. (1998). A dissimilatory sirohaem-sulfite reductase-type protein from the hyperthermophilic archaeon Pyrobaculum islandicum. *Microbiology* 144, 529–541.

Pattaragulwanit, K. & Dahl, C. (1995). Development of a genetic system for a purple sulfur bacterium: conjugative plasmid transfer in Chromatium vinosum. *Arch Microbiol* 164, 217–222.

Pattaragulwanit, K., Brune, D. C., Trüper, H. G. & Dahl, C. (1998). Molecular genetic evidence for extracytoplasmic localization of sulfur globules in Chromatium vinosum. *Arch Microbiol* (in press).

Pfennig, N. & Trüper, H. G. (1989). Anoxygenic phototrophic bacteria. In Bergey’s Manual of Systematic Bacteriology, vol. 3, pp. 1635–1653. Edited by J. T. Staley, M. P. Bryant, N. Pfennig & J. G. Holt. Baltimore: Williams & Wilkins.

Pierik, A. J., Duyvis, M. G., van Helvoort, J. M. L. M., Wolbert, R. B. G. & Hagen, W. R. (1992). The third subunit of desulfovirdin-type dissimilatory sulfite reductases. *Eur J Biochem* 205, 111–115.

Rákhely, G., Colbeau, A., Garin, J., Vignais, P. N. & Kovács, K. L. (1998). Unusual gene organization of HydSL, the stable [NiFe]hydrogenase in the photosynthetic bacterium Thiocapsa roseopersicina. *J Bacteriol* 180, 1460–1465.

Reynolds, R., Bermudez-Cruz, R. M. & Chamberlin, M. J. (1992). Parameters affecting transcription termination by Escherichia coli RNA. I. Analysis of 13 rho-independent terminators. *J Mol Biol* 224, 31–51.

Rossi, M., Pollock, B. R., Reiji, M. W., Keon, R. G., Fu, R. & Voordouw, G. (1993). The hmc operon of Desulfovibrio vulgaris subsp. vulgaris Hildenborough encodes a potential transmembrane redox protein complex. *J Bacteriol* 175, 4699–4711.

Russel, M. & Model, P. (1988). Sequence of thioredoxin reductase from Escherichia coli. Relationship to other flavoprotein disulfide oxidoreductases. *J Biol Chem* 263, 9015–9019.

Sambrook, J., Fritsch, E. F. & Maniatis, T. (1989). *Molecular Cloning: a Laboratory Manual*, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Schedel, M., Vanselow, M. & Trüper, H. G. (1979). Siroheme sulfite reductase from Chromatium vinosum. Purification and investigation of some of its molecular and catalytic properties. *Arch Microbiol* 121, 29–36.

Simon, R., Priefer, U. & Pühler, A. (1983). A broad host range mobilization system for in vivo genetic engineering: transposon mutagenesis in gram negative bacteria. *Bio/Technology* 1, 784–791.

Smith, A. J. (1966). The role of tetrathionate in the oxidation of thiosulphate by Chromatium sp. strain D. *J Gen Microbiol* 42, 371–380.

Steuber, J., Arendsen, A. F., Hagen, W. R. & Kroneck, P. M. H. (1995). Molecular properties of the dissimilatory sulfite reductase from Desulfovibrio desulfuricans (Essex) and comparison with the enzyme from Desulfovibrio vulgaris (Hildenborough). *Eur J Biochem* 233, 873–879.

A. S. POTT and C. DAHL

Trüper, H. G. (1984). Phototrophic bacteria and their sulfur metabolism. In *Sulfur, its Significance for Chemistry, for the Geo-, Bio-, and Cosmosphere and Technology*, pp. 367–382. Edited by A. Müller & B. Krebs. Amsterdam: Elsevier.

Trüper, H. G. & Fischer, U. (1982). Anaerobic oxidation of sulphur compounds as electron donors for bacterial photosynthesis. *Philos Trans R Soc Lond Ser B Biol Sci* 298, 529–542.

Voordouw, G. (1995). The genus *Desulfovibrio*: the centennial. *Appl Environ Microbiol* 61, 2813–2819.

Weaver, P. F., Wall, J. D. & Gest, H. (1975). Characterization of *Rhodopseudomonas capsulata*. *Arch Microbiol* 105, 207–216.

Wilkinson, M. (1991). Purification of RNA. In *Essential Molecular Biology*, vol. 1, pp. 69–87. Edited by T. A. Brown. Oxford: Oxford University Press.

Received 6 October 1997; revised 19 January 1998; accepted 9 March 1998.
