
JOURNAL OF BACTERIOLOGY, Nov. 1998, p. 5875–5884
0021-9193/98/$04.00+0
Copyright © 1998, American Society for Microbiology. All Rights Reserved.

# Accumulation of the Enterobacterial Common Antigen Lipid II Biosynthetic Intermediate Stimulates *degP* Transcription in *Escherichia coli*

**PAUL N. DANESE,**${}^{1\dagger}$ **GEORGE R. OLIVER,**${}^{1\ddagger}$ **KATHLEEN BARR,**${}^{2}$ **GREGORY D. BOWMAN,**${}^{1}$  
**PAUL D. RICK,**${}^{2}$ AND **THOMAS J. SILHAVY**${}^{1*}$

*Department of Molecular Biology, Princeton University, Princeton, New Jersey 08544,*${}^{1}$ *and Department of Microbiology and Immunology, Uniformed Services University of the Health Sciences, Bethesda, Maryland 20814-4799*${}^{2}$

Received 7 July 1998/Accepted 18 September 1998

In *Escherichia coli*, transcription of the *degP* locus, which encodes a heat-shock-inducible periplasmic protease, is controlled by two parallel signal transduction systems that each monitor extracytoplasmic protein physiology. For example, the heat-shock-inducible sigma factor, $\sigma^E$, controls *degP* transcription in response to the overproduction and folded state of various extracytoplasmic proteins. Similarly, the CpxA/R two-component signal transduction system increases *degP* transcription in response to the overproduction of a variety of extracytoplasmic proteins. Since *degP* transcription is attuned to the physiology of extracytoplasmic proteins, we were interested in identifying negative transcriptional regulators of *degP*. To this end, we screened for null mutations that increased transcription from a strain containing a *degP-lacZ* reporter fusion. Through this approach, we identified null mutations in the *wecE*, $rmlA_{ECA}$, and *wecF* loci that increase *degP* transcription. Interestingly, each of these loci is responsible for synthesis of the enterobacterial common antigen (ECA), a glycolipid situated on the outer leaflet of the outer membrane of members of the family *Enterobacteriaceae*. However, these null mutations do not stimulate *degP* transcription by eliminating ECA biosynthesis. Rather, the *wecE*, $rmlA_{ECA}$, and *wecF* null mutations each impede the same step in ECA biosynthesis, and it is the accumulation of the ECA biosynthetic intermediate, lipid II, that causes the observed perturbations. For example, the lipid II-accumulating mutant strains each (i) confer upon *E. coli* a sensitivity to bile salts, (ii) confer a sensitivity to the synthesis of the outer membrane protein LamB, and (iii) stimulate both the Cpx pathway and $\sigma^E$ activity. These phenotypes suggest that the accumulation of lipid II perturbs the structure of the bacterial outer membrane. Furthermore, these results underscore the notion that although the Cpx and $\sigma^E$ systems function in parallel to regulate *degP* transcription, they can be simultaneously activated by the same perturbation.

---

In *Escherichia coli*, transcription of the *degP* gene, which encodes a heat-shock-inducible periplasmic protease, is modulated by at least two signal transduction systems that function in parallel with respect to each other. The Cpx signal transduction pathway and the $\sigma^E$ regulatory system each control *degP* transcription in response to extracytoplasmic signals (4, 6, 15, 19, 21, 22). For example, overproduction of outer membrane proteins increases $\sigma^E$ activity, while mutations that decrease the production of outer membrane proteins concomitantly decrease $\sigma^E$ activity. The signal transduction system that senses these extracytoplasmic changes has recently been defined, and it consists of an inner membrane anti-sigma factor (RseA) and a periplasmic protein (RseB) that both monitor extracytoplasmic protein physiology (7, 18). A second signal transduction system, comprised of the CpxA-CpxR two-component proteins, also regulates *degP* transcription in response to extracytoplasmic protein physiology (4, 6, 9, 19, 32).

To further characterize the transcriptional regulation of *degP* by Cpx and $\sigma^E$, we screened for negative regulators of *degP* transcription. Through this approach, we identified null mutations in three genes of the *wec* gene cluster that stimulate *degP* transcription.

The *wec* gene cluster is required for the synthesis of the enterobacterial common antigen (ECA), a glycolipid found in the outer leaflet of the outer membrane in all species of the family *Enterobacteriaceae* (reviewed in references 11, 12, and 24). The polysaccharide portion of ECA consists of three sugar moieties: N-acetyl-D-glucosamine (GlcNAc), N-acetyl-D-mannosaminuronic acid (ManNAcA), and 4-acetamido-4,6-dideoxy-D-galactose (Fuc4NAc). These sugars form a linear trimeric repeat with the following structure: →3)-$\alpha$-D-Fuc4NAc-(1→4)-$\beta$-D-ManNAcA-(1→4)-$\alpha$-D-GlcNAc-(1→) (Fig. 1) (13). Each polysaccharide chain is ultimately linked to phosphatidic acid via a phosphodiester linkage.

The ECA polysaccharide trimer is synthesized in stepwise fashion at the inner membrane by the successive transfer of the component amino sugars to the lipid carrier, undecaprenyl monophosphate (Fig. 2a). The first step in ECA synthesis is the transfer of GlcNAc-1-phosphate from UDP-GlcNAc to undecaprenyl monophosphate (C${}_{55}$-P) to yield C${}_{55}$-PP-GlcNAc, also known as lipid I. ManNAcA is then transferred from UDP-ManNAcA to lipid I to yield C${}_{55}$-PP-GlcNAc-ManNAcA (lipid II). Finally, Fuc4NAc is transferred from TDP-Fuc4NAc to lipid II to yield a complete lipid-linked trimer, C${}_{55}$-PP-GlcNAc-ManNAcA-Fuc4NAc (lipid III) (1, 16, 25). Subse

* Corresponding author. Mailing address: Department of Molecular Biology, Princeton University, Princeton, NJ 08544. Phone: (609) 258-5899. Fax: (609) 258-2957. E-mail: tsilhavy@molbio.princeton.edu.
† Present address: Department of Molecular and Cellular Biology, Harvard University, Cambridge, MA 02138.
‡ Present address: University of Texas Southwestern Medical School, Dallas, TX 75235.

Fuc4NAc                      ManNAcA                     GlcNAc

![Diagram](attachment:diagram.png)

FIG. 1. The trisaccharide repeat unit of the ECA polysaccharide moiety. This repeat unit, α-d-Fuc4NAc-(1→4)-β-d-ManNAcA-(1→4)-α-d-GlcNAc, constitutes the amino sugar polymer of ECA. N-Acetylglucosamine (GlcNAc) also serves as the attachment site for the lipid anchor. N-Acetyl-d-mannosaminuronic acid and 4-acetamido-4,6-dideoxy-d-galactose are abbreviated as ManNAcA and Fuc4NAc, respectively. The dashed line representing the bond between the number 6 oxygen and the acetyl group of GlcNAc indicates that the acetyl group is not present in stoichiometric amounts in ECA.

quent steps involve polymerization, transfer of the polymer to a phospholipid aglycone, and translocation to the outer membrane. The nature and chronology of these last three events have not yet been established.

The mutations identified in this study all interfere with the conversion of lipid II to lipid III (Fig. 2). Indeed, the results described here indicate that the accumulation of lipid II stimulates *degP* transcription. Specifically, the lipid II-accumulating mutants can (i) confer upon *E. coli* a sensitivity to bile salts, (ii) confer a sensitivity to the synthesis of the outer membrane protein LamB, and (iii) stimulate both the Cpx pathway and σ^E^ activity. These phenotypes suggest that the *wec* mutations perturb the structure of the bacterial outer membrane. Moreover, analysis of these mutations underscores the notion that although the Cpx and σ^E^ systems function in parallel to regulate *degP* transcription, they can be simultaneously activated by the same perturbation (4, 9, 19).

## MATERIALS AND METHODS

### Media and reagents
Media were prepared as described by Silhavy et al. (28). Liquid cultures were grown in Luria broth. Unless specifically noted, the final concentrations of antibiotics used in growth media were as follows: ampicillin, 50 μg/ml; spectinomycin, 50 μg/ml; and chloramphenicol, 20 μg/ml. Standard microbiological techniques were used for strain construction and bacterial growth (28). 5-Bromo-4-chloro-3-indoly-l-galactoside (X-Gal) was purchased from Fischer.

### Strains and phage
All strain genotypes are provided in the text and figure legends. λRS88, λRS45, and λNK1324 have been described elsewhere (10, 29). The *degP-lacZ* and *rpoH~P3~-lacZ* operon fusions have been described elsewhere (6, 15).

### Screening for negative regulators of *degP* transcription
Nine independent cultures of MC4100 were infected with λNK1324 (which delivers the Tn10cam transposon) as described by Kleckner et al. (10). Infected cells were then plated on Luria agar containing chloramphenicol to select for cells carrying Tn10cam insertions. The resulting transductants were grouped into nine independent pools, with each pool containing at least 1,200 colonies.

P1vir lysates were prepared on each pool as described by Kleckner et al. (10). The P1 lysates were then used to transduce the Tn10cam insertions into PND257 (MC4100; *ompR::Tn10 λRS88[degP-lacZ]*). Transductants were plated on Luria agar containing chloramphenicol and 1.4 μg of X-Gal per ml. The parent strain (PND257) is phenotypically Lac^-^ (white) on this medium, providing a simple screen for mutants with increased *degP* transcription. Eight Lac^+^ (blue) colonies were isolated from each of the nine pools and further analyzed.

The Tn10cam insertions within each Lac^+^ mutant were reintroduced into the parent strain (PND257) to determine whether the increase in Lac activity was due to the specific Tn10cam insertion. Those mutant strains whose increased Lac activity resulted from the Tn10cam insertion were further analyzed. These rebuilt strains were used for subsequent analyses. Each insertion mutation (except insertion 5) was numbered XY. The X value indicates the pool number (1 to 9) from which the mutation was isolated, while the Y value indicates the isolate number (1 to 8) from the given pool. Insertion 5 was isolated in a preliminary pilot experiment, and as a consequence, it was not given a designated pool number.

#### Determination of β-galactosidase activity
Cells were grown overnight in Luria broth, then subcultured (1:40) into 2 ml of the same medium, and grown to mid-exponential phase. β-Galactosidase activities were determined by a microtiter plate assay (29). β-Galactosidase activities are expressed as (units/A~600~) × 10³, where units = micromoles of product formed per minute. Assays were performed on a minimum of four independent isolates of each strain, and the results were averaged to obtain the indicated activities. Error bars indicate the standard deviations. The absence of error bars indicates that the standard deviation fell below the resolution limit of the graphing program.

#### Passive hemagglutination assay for the presence of ECA
Determination of the presence of ECA was performed as described elsewhere (24).

#### Determination of chromosomal insertion sites of the Tn10cam insertions
The precise sites of Tn10cam insertions were determined in the following manner. Strains carrying the *wecF::cam*, *rmlA~ECA~::cam*, and *wecE::cam* mutations were subjected to arbitrarily primed PCR (2) using the CAM-5′ primer (5′ CTG ACG GGG TGG TGC GTA ACG GC 3′) and the ARB1 primer (5′ GG CCA CGC GTC GAC TAG TAC NNN NNN NGA TAT 3′). The PCR products generated by these two primers were subjected to a secondary amplification step with the CAM-5′ primer and the ARB2 primer (5′ GGC CAC GCG TCG ACT AGT AC 3′). PCR products generated from the second amplification step were sequenced, and the junction between the Tn10cam sequence and the host chromosomal DNA was determined from this sequence information.

Specifically, the Tn10cam of mutant 5 (*wecF::cam*) is inserted between nucleotides 7863 and 7864 of the published *wec* gene cluster sequence (accession no. AE000455). The Tn10cam of mutant 31 (*wecE::cam*) is inserted between nucleotides 5707 and 5708 of the published *wec* gene cluster sequence, while the Tn10cam of insertion 22 (*rmlA~ECA~::cam*) is situated between nucleotides 3894 and 3895 of the same sequence.

#### Assay for lipid II accumulation
The incorporation of [³H]GlcNAc into lipid II was determined as previously described (26). Briefly, bacteria were grown with vigorous aeration at 37°C in 60 ml of Proteose Peptone-beef extract medium supplemented with glucose (0.2% final concentration) to an A~600~ of 0.4. The cells were then harvested by centrifugation, resuspended in fresh Proteose Peptone-beef extract–0.2% glucose (6 ml), and incubated at 37°C with [³H]GlcNAc (75 μCi, 5.2 Ci/mmol) for 30 min. The labeled cells were subsequently poured over crushed ice, harvested by centrifugation, and washed with cold 0.9% saline. The washed cells were then successively extracted with 95% ethanol (6 ml) and acetone (6 ml) and then dried in vacuo. The dried cells were extracted with chloroform-methanol (3:2 [vol/vol]), and the extracts were analyzed by ascending paper chromatography on EDTA-treated silica-gel-impregnated paper with chloroform-methanol-water-concentrated ammonium hydroxide (88:48:10:1 [vol/vol/vol/vol]) as the developing solvent. The material in the region of the chromatogram corresponding to lipid II was eluted, and the amount of radioactivity in this material was determined. In addition, the identity of this material as lipid II disaccharide was verified by treatment of the material with mild acid followed by gel filtration chromatography of the water-soluble fraction on a Bio-Gel P2 column.

#### Maltose sensitivity disc assays
The maltose sensitivity disc assays, whose results are shown in Fig. 6, were performed as follows. Each strain was grown to saturation overnight at 37°C in 5 ml of Luria broth. Three milliliters of molten Luria top agar (55°C) was mixed with 100 μl of an overnight culture and immediately spread onto Luria agar (warmed to 23°C). The top agar was allowed to solidify for 2 min. A Schleicher and Schuell analytical paper filter disc (7-mm diameter) was then placed in the middle of the Luria agar plate. Ten microliters of 20% (wt/vol) maltose was placed on the filter disc, and the plates were incubated overnight at 37°C. The zone of clearing, which is defined as the diameter of inhibited growth minus the diameter of the filter disc, was measured 18 h after the inception of incubation. Each value shown in Fig. 6 is the average of four replicate experiments. The error bars represent the standard deviations from each average.

#### Genetic nomenclature
Genes and gene products involved in ECA and O-antigen synthesis and assembly are designated in accordance with the recently formulated bacterial polysaccharide gene nomenclature scheme (23). The following are the relevant new designations, each of which is accompanied by the former designation in parentheses: *wbb (rfb)*, *wec (rfe/rff)*, *wecA (rfe)*, *wzzECA? (o349)*, *wecB (rffE)*, *wecC (rffD)*, *rmlB (o355)*, *rmlA (o292)*, *wecD (rffC)*, *wecE (rffA)*, *wzx~ECA~? (o416)*, *wecF (rffT)*, and *wecG (rffM)*. In addition, the subscripts ECA and Oag have been included where appropriate to distinguish between homologs involved in ECA and O-antigen synthesis, respectively.

## RESULTS

### Rationale and mutant isolation
We sought to identify negative regulators of *degP* transcription by generating null mutations that increased transcription from a *degP-lacZ* reporter fusion. To facilitate this analysis, we created strain PND257 (MC4100; *ompR::Tn10 λRS88[degP-lacZ]*), which contains the *degP-lacZ* fusion as well as the *ompR::Tn10* null mutation.

a)

TTP + Glc-1-P
↓
rmlA<sub>Oag</sub> rmlA<sub>ECA</sub>
PPi ←
TDP-Glc
↓
rmlB<sub>Oag</sub> rmlB<sub>ECA</sub>
TDP-4-keto-6-deoxy-D-Glc
Glutamate
wecE
α-KG

wecB
UDP-GlcNAc
UDP-ManNAc
wecC
UDP-ManNAcA
C55~PP-GlcNAc (Lipid I)
wecA
UMP ←
C55~P
wecG
UDP ←
C55~PP-GlcNAc-ManNAc (Lipid II)
wecD
TDP-Fuc4NH<sub>2</sub>
Acetyl-CoA
Co-ASH
TDP-Fuc4NAc
TDP ←
C55~PP-GlcNAc-ManNAc-Fuc4NAc (Lipid III)
wecF
polymerization
transfer of ECA chains to phospholipid
translocation to outer membrane
ECA

b)

5′ wecA wzz<sub>ECA</sub>? wecB wecC rmlB<sub>ECA</sub> rmlA<sub>ECA</sub> wecD wecE wzx<sub>ECA</sub>? wecF wecG 3′
insertion 22 insertion 31 insertion 5
5,000 bp 10,000 bp

FIG. 2. The ECA polysaccharide biosynthetic pathway and the wec gene cluster. (a) The ECA polysaccharide biosynthetic pathway. Gene names are listed in italics adjacent to the reactions that their products catalyze. The genes affected by the mutational analysis employed in this study are enclosed within rectangles. Abbreviations: C<sub>55</sub>-P, undecaprenyl monophosphate; GlcNAc, N-acetylglucosamine; ManNAcA, N-acetyl-D-mannosaminuronic acid; Fuc4NAc, 4-acetamido-4,6-dideoxy-D-galactose; Glc-1-P, glucose-1-phosphate; PP<sub>i</sub>, pyrophosphate; TDP-Glc, TDP-glucose; TDP-4-keto-6-deoxy-D-Glc, TDP-4-keto-6-deoxy-D-glucose; α-KG, α-ketoglutarate; TDP-FucN, TDP-fucosamine; acetyl-CoA, acetyl coenzyme A; Co-ASH, coenzyme A; ManNAc, N-acetyl mannose. (b) The wec gene cluster. The open reading frames of this cluster are shown in shaded boxes. All open reading frames are transcribed from 5′ to 3′ as shown. The positions of the insertions described in this study are also depicted. A 13,000-bp scale is depicted below the gene cluster for reference.

compared to the parent strain. For example, insertions 5 and 31 increase $degP$ transcription approximately sixfold (Fig. 3, compare lanes 2 and 4 with lane 1), while insertion 22 increases $degP$ transcription approximately threefold (Fig. 3, compare lanes 1 and 3). The six remaining transposon-generated mutations increase $degP$ transcription to the same extent as that observed with insertions 5 and 31 (data not shown).

Null mutations within the $wec$ gene cluster stimulate $degP$ transcription. The transposon insertions in each of the MacConkey agar-sensitive isolates were all tightly linked to the $wec$ gene cluster, which is located at approximately 85 min on the $E$. coli chromosome. Since the $wec$ gene cluster is involved in the biosynthesis of ECA, we were interested in determining whether the Tn10cam insertions affected the biosynthesis of ECA. Using a passive hemagglutination assay for ECA biosynthesis (25), we determined that eight of the nine insertions did not produce ECA ($ECA^-$). Only one insertion (no. 22) remained $ECA^+$. Interestingly, insertion 22 causes the weakest induction of $degP$ transcription (Fig. 3).

Fine-structure mapping of each insertion using previously described Tn10 insertions within the $wec$ gene cluster (16) indicated that five of the nine Tn10cam insertions (no. 5, 41, 45, 47, and 51) were tightly linked to the $wecE$ locus. In addition, three of the nine Tn10cam insertions (no. 31, 32, and 33) were tightly linked to the $wecE$ locus, while the remaining insertion (no. 22) was tightly linked to the $wecD$ locus (Fig. 2b).

Using this linkage information, we then determined the precise sites of insertion of three representative Tn10cam insertions (no. 5, 22, and 31). The Tn10cam of mutant 5 is inserted within the 140th codon of the $wecF$ open reading frame. The Tn10cam of mutant 31 is situated within the 215th codon of the $wecE$ coding sequence, while the Tn10cam of mutant 22 is inserted within the 123rd codon of $rmlA_{ECA}$ (see Materials and Methods for the precise nucleotide insertion sites). Thus, our screen identified three distinct loci in the $wec$ gene cluster, $wecF$, $wecE$, and $rmlA_{ECA}$. Accordingly, Tn10cam insertions 5, 22, and 31 are referred to as $wecF::cam$, $rmlA_{ECA}::cam$, and $wecE::cam$, respectively.

The $rmlA_{ECA}::cam$ mutation. The $rmlA_{ECA}::cam$ mutation was anomalous in two major regards. First, unlike the other eight mutations, the $rmlA_{ECA}::cam$ mutation did not confer an $ECA^-$ phenotype, although it mapped within the $wec$ gene cluster. Second, the $rmlA_{ECA}::cam$ mutation conferred a relatively weak increase in $degP$ transcription (approximately 50% of that observed with the other Tn10cam insertions).

A potential explanation of the phenotypes conferred by the $rmlA_{ECA}::cam$ mutation was provided by previously described biochemical analysis of the $rmlA_{ECA}$ gene product (14). Specifically, the predicted $rmlA_{ECA}$ gene product has 65% identity with RmlA$_{Oag}$, a glucose-1-phosphate thymidylyltransferase that plays a role in the initial stages of TDP-Fuc4NAc biosynthesis (33). Marolda and Valvano (14) have demonstrated that the $rmlA_{ECA}$ gene product does indeed display glucose-1-phosphate thymidylyltransferase activity, thus confirming what the sequence homology between RmlA$_{ECA}$ and RmlA$_{Oag}$ had suggested. Given the activity of the $rmlA_{ECA}$ gene product, we predicted that a lesion in this locus would reduce production of TDP-glucose and, hence, slow Fuc4NAc synthesis (Fig. 2a). As a consequence, the $rmlA_{ECA}::cam$ mutation would slow the production of ECA by impeding the conversion of lipid II to lipid III. However, since a functional $rmlA_{Oag}$ gene product is still present in the $E$. coli chromosome (33), Fuc4NAc synthesis and ECA synthesis would be reduced but not abolished. According to this model, a lesion in $rmlA_{ECA}$ would (i) not confer an $ECA^-$ phenotype and (ii) display an attenuated increase in $degP$ transcription compared with the $wecF$ and

FIG. 3. The Tn10cam insertions stimulate $degP-lacZ$ transcription. The $\beta$-galactosidase activities of PND257 (MC4100; $ompR::Tn10$ $\lambda$RS88[$degP-lacZ$]) (lane 1), PND788 (PND257; Tn10cam insertion 5) (lane 2), PND789 (PND257; Tn10cam insertion 22) (lane 3), and PND790 (PND257; Tn10cam insertion 31) (lane 4) were determined. Tn10cam insertions 5 and 31 each stimulate $degP-lacZ$ transcription approximately sixfold, while insertion 22 stimulates $degP-lacZ$ transcription approximately threefold compared to the parent strain. All strains were grown as described in Materials and Methods.

$ompR$ null strains do not synthesize the major outer membrane proteins OmpC and OmpF (30), and consequently, $\sigma^E$ activity is reduced in such strains (15). Because of the reduction in $\sigma^E$ activity, PND257 is phenotypically Lac$^-$ (white) on Luria agar containing 1.4 $\mu$g of X-Gal per ml. This Lac$^-$ phenotype provides a simple screen for mutants with increased $degP$ transcription. By generating null mutations throughout the chromosome of PND257, we hoped to identify mutants with increased Lac activity, and by extension, mutations that impaired the function of negative regulators of $degP$ transcription.

We used $\lambda$NK1324, which delivers the Tn10cam transposon (10), to perform transposon mutagenesis on strain PND257. The chloramphenicol-resistant colonies generated by this procedure were screened for those with increased $degP$ transcription. Approximately 15,000 chloramphenicol-resistant colonies were screened, and 11 colonies with increased $degP$ transcription were analyzed. Of these 11 isolates, 9 grew poorly on lactose-MacConkey agar, while the remaining two grew as well as the parent (PND257) on this medium. Since sensitivity to MacConkey agar is often an indicator of a structural perturbation in the outer membrane (20), and since $degP$ transcription is attuned to such perturbations (15, 22, 27), we chose to analyze the nine MacConkey agar-sensitive mutants.

Figure 3 shows that each transposon-generated insertion increases $degP$ transcription approximately three- to sixfold

TABLE 1. Accumulation of lipid II in Tn10cam insertion mutants defective in the synthesis of lipid III

| Strain | Relevant genotype | Accumulation of lipid II$^a$ |
|--------|-------------------|------------------------------|
| PND257 | Parent            | $-$                         |
| PND788 | wecF::Tn10cam     | $+$                         |
| PND789 | rmlA${}_{ECA}$::Tn10cam | $+$                         |
| PND790 | wecE::Tn10cam     | $+$                         |

$^a$ $+$, >1,000 cpm of ${}^{[3]}$H]GlcNAc-labeled lipid II was recovered from extracts prepared from approximately $10^8$ cells. $-$, ${}^{[3]}$H]GlcNAc-labeled lipid II was not detectable in extracts prepared from approximately $10^8$ cells.

wecE mutations, which completely abolish the synthesis of lipid III. These predictions are borne out in the data described above (Fig. 3).

One additional prediction is that all of the Tn10cam insertions should impede the conversion of lipid II to lipid III (Fig. 2). Indeed, analysis of the PND788 (PND257; wecF::cam), PND789 ($rmlA_{ECA}$::cam), and PND790 (wecE::cam) strains shows that they accumulate significant amounts of lipid II, while their parent strain, PND257, does not accumulate lipid II in detectable quantities (Table 1). This information indicates that (i) the mutations in wecF and wecE are loss-of-function mutations and (ii) this study provides the first evidence indicating that the $rmlA_{ECA}$ gene product is actually involved in ECA biosynthesis.

The accumulation of the lipid II ECA biosynthetic intermediate increases degP transcription and confers sensitivity to bile salts. From the data presented, it is clear that all of the mutations identified in our screen affect the same general step in the biosynthesis of ECA. Specifically, the insertions within wecF and wecE cause the accumulation of lipid II while the $rmlA_{ECA}$::cam mutation impedes the conversion of lipid II to lipid III (Fig. 2a).

What is not clear, however, is whether it is the absence of ECA or the accumulation of the lipid II intermediate that stimulates degP transcription and confers sensitivity to MacConkey agar. Various tests of epistasis were performed to distinguish between these two possibilities. For example, the wecA::Tn10 mutation was introduced into the wecF::cam, wecE::cam, and $rmlA_{ECA}$::cam strains, and the resulting amounts of degP transcription were determined (Fig. 4). The wecA::Tn10 mutation blocks ECA synthesis at its earliest step, preventing the transfer of GlcNAc phosphate from UDP-GlcNAc to undecaprenyl monophosphate (Fig. 2a). If the lack of ECA is the cause of the observed phenotypes, the wecA::Tn10 wecE::cam double mutant strain should display at least the same degree of degP transcriptional induction and MacConkey agar sensitivity as observed with the wecE::cam mutation alone. If the accumulation of lipid II is responsible for the transcriptional induction of degP, then the double mutant strain should not display the transcriptional induction observed with the single chloramphenicol-resistant insertion mutations. These predictions also hold true for the wecF::cam and $rmlA_{ECA}$::cam mutant strains.

Figure 4 indicates that the accumulation of lipid II is responsible for the observed phenotypes. For example, when the

degP-lacZ, ompR101

![Diagram](https://i.imgur.com/yourimageurl.png)

FIG. 4. The accumulation of lipid II stimulates transcription of degP-lacZ. β-Galactosidase activities were determined for the following strains: PND423 (MC4100; ompR101 λRS88[degP-lacZ]) (lane 1), SP173 (PND423; wecE::cam) (lane 2), SP190 (PND423; wecE::cam wecA::Tn10) (lane 3), SP172 (PND423; $rmlA_{ECA}$::cam) (lane 4), SP185 (PND423; $rmlA_{ECA}$::cam wecA::Tn10) (lane 5), SP188 (PND423; $rmlA_{ECA}$::cam wecE::Tn10) (lane 6), SP171 (PND423; wecF::cam) (lane 7), and SP179 (PND423; wecF::cam wecA::Tn10) (lane 8). The wecA::Tn10 mutation blocks the transcriptional induction of degP-lacZ that is conferred by the wecE::cam, $rmlA_{ECA}$::cam, and wecF::cam insertions (compare lanes 2 and 3, 4 and 5, and 7 and 8). The weak transcriptional induction of degP-lacZ conferred by the $rmlA_{ECA}$::cam mutation can be increased by the wecE::Tn10 mutation (compare lanes 4 and 6). The ompR101 mutation is a deletion within the ompR open reading frame, rendering the strains described in this figure null for ompR. All strains were grown as described in Materials and Methods.

wecA::Tn10 mutation is introduced into strains containing either the wecE::cam, rmlA<sub>ECA</sub>::cam, or wecF::cam mutation, the double mutant strain shows only a minimal induction of degP transcription (Fig. 4, compare lanes 2 and 3, 4 and 5, and 7 and 8). In addition, these double mutant strains are no longer MacConkey agar sensitive.

Finally, based on the biochemical analysis of RmlA<sub>ECA</sub> (14) and on the homology between the rmlA<sub>ECA</sub> gene product and the RmlA<sub>Oag</sub> protein, we have suggested that the rmlA<sub>ECA</sub>::cam mutation attenuates, but does not abolish, the conversion of lipid II to lipid III. This notion explains the weak induction of degP transcription conferred by this mutation, and it also explains why the rmlA<sub>ECA</sub>::cam lesion does not confer an ECA<sup>-</sup> phenotype. If this model is correct, introduction of the wecE::Tn10 mutation into a strain that contains the rmlA<sub>ECA</sub>::cam insertion should raise degP transcription to the level observed with the wecE mutation alone. Again, this prediction is verified. Specifically, the rmlA<sub>ECA</sub> wecE double mutant strain displays the same degree of transcriptional induction of degP as the wecE mutation does in isolation (compare lanes 2, 4, and 6 in Fig. 4).

There are two general conclusions that can be drawn from Fig. 4. First, the accumulation of lipid II stimulates degP transcription and confers MacConkey agar sensitivity in these strains. Second, the rmlA<sub>ECA</sub>::cam mutation impedes, but does not abolish, the conversion of lipid II to lipid III in the biosynthesis of ECA.

The induction of degP transcription by accumulation of the lipid II intermediate is decreased in an ompR<sup>+</sup> background. Since the experiments described above have all utilized strains that were ompR null, we were also interested in determining the effects of lipid II accumulation in an ompR<sup>+</sup> background. Figure 5 shows that while the wec mutations still stimulate degP transcription in the ompR<sup>+</sup> background, the magnitude of the induction of degP transcription conferred by these mutations is significantly reduced compared to the analogous ompR strains. These ompR<sup>+</sup>, lipid II-accumulating strains are also no longer as sensitive to growth on MacConkey agar as their ompR counterparts.

Since the major proteins whose synthesis is controlled by OmpR are the outer membrane proteins OmpF and OmpC, it seemed likely that the enhanced degP transcriptional induction observed in the ompR background was due to the lack of these proteins. Consistent with this observation, production of OmpC in an ompR, lipid II-accumulating background reduced the magnitude of transcriptional induction of the degP locus (data not shown). Thus, at a minimum, it is the lack of OmpC that heightens the MacConkey agar sensitivity and the increase in degP transcription that is observed in the lipid II-accumulating strains.

Accumulation of the lipid II intermediate can confer a sensitivity to high-level synthesis of the outer membrane protein LamB. During the course of this study, we noted that the lipid II-accumulating mutant strains were exquisitely sensitive to growth on maltose and maltodextrins. As a consequence, we wanted to determine if these strains had difficulty growing in the presence of all types of sugars or whether their sensitivity was restricted to growth in the presence of maltose and its oligomers. To this end, we assayed the growth of the lipid II-accumulating strains on Luria agar in the presence of high concentrations of maltose, lactose, and glucose. The lipid II-accumulating strains were sensitive only to maltose, indicating that this sensitivity is not simply a sugar-mediated effect (data not shown).

Since the various phenotypes described for these strains appear to be associated with perturbations in the outer membrane, the observed maltose sensitivity might be due to high-level synthesis and export of the outer membrane porin LamB (LamB synthesis is induced in the presence of maltose). To address this issue, the lamBΔ60 mutation, which deletes a portion of the LamB signal sequence (and prevents its export across the inner membrane [8]), was introduced into the lipid II-accumulating strains. We then determined if these lamBΔ60 strains were also sensitive to high levels of maltose. Figure 6 indicates that the lamBΔ60 mutation abolishes the sensitivity of the lipid II-accumulating strains to high concentrations of maltose (compare lanes 2 and 6, 3 and 7, and 4 and 8). Thus, the maltose sensitivity observed with these strains is due to high-level export of wild-type LamB, further suggesting that the accumulation of lipid II perturbs the physiology of the outer membrane.

Accumulation of lipid II does not interfere with incorporation or assembly of LamB in the outer membrane. Because of the observed toxicity associated with high-level synthesis of LamB in the lipid II-accumulating mutants, we wanted to determine if these strains displayed defects in the incorporation and assembly of LamB into the outer membrane. To address this issue, we performed two experiments. (i) We examined the amount of LamB protein associated with membrane fractions of a parental strain, PND257 (MC4100; ompR::Tn10 λRS88 [degP-lacZ]), as well as derivative strains that accumulate lipid II, PND788 (PND257; wecF::cam), PND789 (PND257;

Zone of Clearing (mm)
25-
12.5-
0

| Lane | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 |
|------|---|---|---|---|---|---|---|---|
| Wild-type | wecF::cam | rmlA<sub>ECA</sub>::cam | wecE::cam | lamB<sup>+</sup> | Wild-type | rmlA<sub>ECA</sub>::cam | wecF::cam | lamBΔdelta60 |

FIG. 6. High-level export of the LamB maltoporin in lipid II-accumulating strains is toxic. Ten microliters of 20% maltose was added to filter discs that had been placed on lawns of the following strains: SP299 (MC4100; ompR101 zja::Tn10 linked to lamB<sup>+</sup> λRS88[degP-lacZ]) (lane 1), SP301 (SP299; wecF::cam) (lane 2), SP303 (SP299; rmlA<sub>ECA</sub>::cam) (lane 3), SP305 (SP299; wecE::cam) (lane 4), SP298 (MC4100; ompR101 zja::Tn10 linked to lamBΔ60 λRS88[degP-lacZ]) (lane 5), SP300 (SP298; wecF::cam) (lane 6), SP302 (SP298; rmlA<sub>ECA</sub>::cam) (lane 7), and SP304 (SP298; wecE::cam) (lane 8). The ompR101 allele is a deletion within the ompR open reading frame, rendering the strains described in this figure null for ompR. The lamBΔ60 mutation deletes part of the coding sequence for the LamB signal sequence (8). Thus, strains containing the lamBΔ60 mutation cannot export LamB across the inner membrane. The values displayed along the y axis (zone of clearing) represent the amount of growth inhibition caused by the addition of maltose. The zone of clearing value is defined as the diameter of growth inhibition around the maltose-saturated filter disc, minus the diameter of the filter disc itself (7 mm). All strains were grown at 37°C as described in Materials and Methods.

rmlA<sub>ECA</sub>::cam), and PND790 (PND257; wecE::cam). (ii) We also examined the kinetics of LamB trimerization in PND257, PND788, PND789, and PND790. None of the lipid II-accumulating strains displayed defects in the incorporation of LamB into membrane fractions, nor did these mutants display defects in the trimerization of LamB (data not shown). Thus, the toxicity associated with high-level synthesis of LamB in lipid II-accumulating mutants is not the result of a gross structural defect in LamB assembly.

The wecF::cam, wecE::cam, and rmlA<sub>ECA</sub>::cam mutations increase σ<sup>E</sup> activity. Finally, we were interested in the mechanism(s) by which degP transcription was being stimulated as a result of lipid II accumulation. Two regulatory pathways are known to modulate degP transcription. As mentioned elsewhere, degP transcription can be increased by increasing σ<sup>E</sup> activity, through the RseA/B signal transduction system (15). In addition, degP transcription is also regulated by the Cpx two-component signal transduction pathway (6, 22), which functions in parallel with the σ<sup>E</sup> signal transduction pathway (4, 6). To determine the specific route by which the wecF::cam, rmlA<sub>ECA</sub>::cam, and wecE::cam mutations functioned to increase degP transcription, we introduced each insertion into SP245 (MC4100; ompR::Tn10 rpoH<sub>P3</sub>-lacZ). The rpoH<sub>P3</sub> pro-

beta-galactosidase activity
40-
20-
0

Lane 1 2 3 4

FIG. 7. The wecF::cam, rmlA<sub>ECA</sub>::cam, and wecE::cam mutations stimulate σ<sup>E</sup> activity. The β-galactosidase activities of SP245 (MC4100; ompR::Tn10 λRS45[rpoH<sub>P3</sub>-lacZ]) (lane 1), SP282 (SP245; wecF::cam) (lane 2), SP283 (SP245; rmlA<sub>ECA</sub>::cam) (lane 3), and SP284 (SP245; wecE::cam) (lane 4) were determined. The wecF::cam and wecE::cam mutations stimulate rpoH<sub>P3</sub>-lacZ transcription approximately 2.6-fold. The rmlA<sub>ECA</sub>::cam mutation stimulates rpoH<sub>P3</sub>-lacZ transcription approximately 2.3-fold. All strains were grown as described in Materials and Methods.

motor is recognized solely by RNA polymerases containing the σ<sup>E</sup> subunit. Hence, this fusion provides an assay for only σ<sup>E</sup> activity (6, 15). Figure 7 shows that rpoH<sub>P3</sub>-lacZ transcription is induced by the Tn10cam insertions in a qualitatively similar fashion as that observed with the induction of degP transcription. For example, the wecF::cam and wecE::cam insertions stimulate rpoH<sub>P3</sub>-lacZ transcription approximately 2.6-fold (Fig. 7, compare lane 1 with lanes 2 and 4). The rmlA<sub>ECA</sub>::cam insertion stimulates rpoH<sub>P3</sub>-mediated transcription approximately 2.3-fold (compare lanes 1 and 3 of Fig. 7).

The wecF, wecE, and rmlA<sub>ECA</sub> mutations activate the Cpx signal transduction pathway. The results presented in Fig. 7 demonstrate that the wecF, rmlA<sub>ECA</sub>, and wecE mutations activate degP transcription, at least in part by stimulating σ<sup>E</sup> activity. However, we were also interested in determining if these mutations stimulated degP transcription via the Cpx pathway as well. Accordingly, the cpxR null mutation was introduced into the parent strain (PND257) as well as the wecF, rmlA<sub>ECA</sub>, and wecE mutant strains (PND788, PND789, and PND790, respectively), and the amount of degP-lacZ transcription generated from these strains was quantified. Interestingly, elimination of the Cpx pathway by a cpxR null mutation decreases, but does not abolish, the transcriptional induction of degP conferred by the wecF, rmlA<sub>ECA</sub>, and wecE mutations (Fig. 8). For example, in a cpxR background, the wecF::cam and wecE::cam mutations stimulate degP transcription only 1.6- to 1.7-fold (Fig. 8, compare lane 5 with lanes 6 and 8). Similarly, the rmlA<sub>ECA</sub>::cam mutation stimulates degP transcription approximately 1.4-fold in the cpxR background (Fig. 8, compare lanes 5 and 7). Although the transcriptional induction of degP is qualitatively similar in the cpxR<sup>+</sup> and cpxR

also suggesting a perturbation in outer membrane physiology. Third, these mutations increase *degP* transcription by stimulating both the σ<sup>E</sup> modulatory signal transduction system (15) and the Cpx signal transduction system (6, 22). Since *degP* encodes a periplasmic protease that destroys misfolded extracytoplasmic proteins (see reference 17), this is also an indicator of a perturbation in the physiology of periplasmic and/or outer membrane proteins. Taken together, these results imply that lipid II accumulation perturbs the physiology of envelope proteins, thus causing an increase in *degP* transcription.

Accumulation of lipid II. The results presented in Fig. 4 demonstrate that it is the accumulation of lipid II that stimulates *degP* transcription. Moreover, previous studies have also noted a toxicity associated with the accumulation of lipid II. For example, Rick et al. (26) have observed that *Salmonella typhimurium* strains containing a lesion in *rmlA*<sub>Oag</sub> (which encodes a homolog of RmlA<sub>ECA</sub>) accumulate the lipid II intermediate and display sensitivity to sodium dodecyl sulfate (SDS). This SDS-sensitive phenotype can be suppressed by mutations that block the accumulation of lipid II.

Despite the body of evidence indicating that lipid II accumulation confers a host of envelope-associated defects, it is unclear why lipid II accumulation exerts these effects. While ECA is not essential for the viability of *E. coli*, the undecaprenyl carrier lipid that is used to synthesize the ECA trisaccharide is essential. We considered the possibility that lipid II accumulation indirectly impeded the synthesis of the peptidoglycan layer and/or lipopolysaccharide by sequestering undecaprenyl monophosphate (*C*<sub>55</sub>-P). However, overproduction of the BacA protein, which is believed to increase the pool of free *C*<sub>55</sub>-P (3), had no ameliorative effect on the lipid II-associated phenotypes (6a). Thus, it seems unlikely that the accumulation of lipid II exerts its effects by sequestering *C*<sub>55</sub>-P.

An alternative model posits that the partially completed ECA trisaccharide may interfere with the biogenesis of envelope proteins, thus altering the permeability of the outer membrane and signaling for increased levels of the envelope protease, DegP. This interference would most likely occur at or near the inner membrane since the lipid II disaccharide remains attached to its undecaprenyl carrier lipid in the inner membrane. According to this view, the accumulation of lipid II in the inner membrane affects some process(es) that is important for outer membrane biogenesis.

The involvement of *rmlA*<sub>ECA</sub> in ECA biosynthesis. The results presented here also represent the first mutational analysis of *rmlA*<sub>ECA</sub>. Previous studies that have sought mutations in genes involved in ECA biosynthesis have uncovered several loci in the *wec* and *wbb* gene clusters. For example, mutations in *wecB*, *wecC*, *wecD*, *wecE*, *wecF*, and *wecG* have all been identified because of their abolition of ECA biosynthesis (Fig. 2) (16).

Indeed, *rmlA*<sub>Oag</sub> mutations (Fig. 2a) have also been shown to radically reduce (but not abolish) ECA biosynthesis in *S. typhimurium* (26). However, despite the homology between RmlA<sub>ECA</sub> and RmlA<sub>Oag</sub>, no mutations were ever identified in *rmlA*<sub>ECA</sub>. Based on the previous analyses and on the results presented here, we suggest that RmlA<sub>ECA</sub> and RmlA<sub>Oag</sub> perform redundant functions for the biosynthesis of ECA. The reasons for suggesting this are threefold. First, RmlA<sub>ECA</sub> and RmlA<sub>Oag</sub> have 65% amino acid sequence identity, and they each display glucose-1-phosphate thymidylyltransferase activity (14). Second, *rmlA*<sub>Oag</sub> null strains do not completely abolish ECA biosynthesis in *S. typhimurium*. Finally, Fig. 4 clearly demonstrates that the accumulation of lipid II stimulates *degP* transcription. If RmlA<sub>ECA</sub> partially contributes to the conversion of lipid II to lipid III, then inactivation of this locus should

FIG. 8. The *wecF::cam*, *rmlA*<sub>ECA</sub>::*cam*, and *wecE::cam* mutations activate the Cpx signal transduction pathway. The β-galactosidase activities of PND257 (MC4100 *ompR::Tn10 λRS88[degP-lacZ]*) (lane 1), PND788 (PND257 *wecF::cam*) (lane 2), PND789 (PND257 *rmlA*<sub>ECA</sub>::*cam*) (lane 3), PND790 (PND257 *wecE::cam*) (lane 4), SP150 (PND257 *cpxR::spec*) (lane 5), SP231 (SP150 *wecF::cam*) (lane 6), SP232 (SP150 *rmlA*<sub>ECA</sub>::*cam*) (lane 7), and SP233 (SP150 *wecE::cam*) (lane 8) were determined. The *wecF::cam* and *wecE::cam* mutations stimulate *degP-lacZ* transcription 1.6- to 1.7-fold in the absence of CpxR. Similarly, the *rmlA*<sub>ECA</sub>::*cam* mutation stimulates *degP-lacZ* transcription approximately 1.4-fold in the absence of CpxR. This is in contrast to the three- to sixfold stimulation conferred by these mutations in the presence of CpxR (lanes 1 to 4). All strains were grown as described in Materials and Methods.

background, CpxR is clearly responsible for the majority of the *degP* transcriptional induction in the parental background (PND257). For instance, the *wecF::cam* mutation stimulates *degP* transcription approximately 6-fold in the *cpxR*<sup>+</sup> background, but only 1.7-fold in the *cpxR* background (Fig. 8, compare lanes 1 and 2 with lanes 5 and 6).

Since the transcriptional induction of *degP* is partially blocked by the *cpxR* null mutation, and since the *wecF*, *rmlA*<sub>ECA</sub>, and *wecE* mutations also increase transcription of *rpoH*<sub>P3</sub>*-lacZ*, these mutations must activate both signal transduction pathways that are known to control *degP* transcription.

DISCUSSION

In our search for negative regulators of *degP* transcription, we have shown that mutations that cause the accumulation of the lipid II intermediate in the pathway for ECA biosynthesis cause several envelope-associated perturbations. First, these mutations confer a sensitivity to growth in the presence of bile salt detergents, which is a classic indicator of an outer membrane permeability defect (20). Second, these mutations confer a sensitivity to high-level export of the wild-type LamB porin,

display an attenuated increase in *degP* transcription and should remain ECA<sup>+</sup>. Moreover, when the *rmlA*<sub>ECA</sub> null mutation is combined with a second mutation that completely abolishes the conversion of lipid II to lipid III (i.e., *wecE*), the second mutation should raise the transcriptional induction of *degP* from the attenuated response observed with only the *rmlA*<sub>ECA</sub> mutation to the strong induction observed with the *wecE* mutation. All of these predictions are verified by the results presented here.

Thus, although the *rmlA*<sub>ECA</sub> locus appears to be involved in the biosynthesis of ECA, its identification in this role has been lacking because mutational inactivation of this locus is phenotypically masked by its functional homolog, *rmlA*<sub>Oag</sub>.

The lack of porin enhances the susceptibility of *E. coli* to lipid II accumulation. From the results of Fig. 3 and 5, it is clear that strains lacking the outer membrane porins OmpF and OmpC are more susceptible to the toxic effects of lipid II accumulation. The reasons for this enhanced susceptibility are at present unclear. It is possible that the lack of OmpF and OmpC directly alters the structure of the outer membrane, perhaps making this membrane more susceptible to the perturbations caused by the accumulation of lipid II. For example, lipid II accumulation may increase the ability of the outer membrane to be solubilized by bile salts when OmpF and OmpC are absent.

Alternatively, we note that the lack of porins in an *ompR* background decreases the expression of the σ<sup>E</sup> regulon (including *degP*) by approximately fourfold (15). Since σ<sup>E</sup> regulation is involved in responding to extracytoplasmic protein stresses, *ompR* strains (i.e., σ<sup>E</sup> attenuated) may not be equipped to properly cope with large-scale perturbations, such as those caused by the accumulation of lipid II. However, since the precise biochemical effect(s) of lipid II accumulation is not known, we cannot at present distinguish among these possibilities.

The export-associated toxicity of LamB. To our knowledge, this study describes the first instance in which export of the wild-type LamB protein is toxic. The requirement for export of LamB is specific, as expression of the nonexportable LamBΔ60 mutant is not toxic. Interestingly, this toxicity is not due to a gross structural defect in the assembly of LamB. However, we cannot exclude the possibility that a minor structural alteration in LamB assembly (not detectable by membrane fractionation or trimerization studies) confers this toxicity. Alternatively, the transit of large amounts of LamB protein en route to the outer membrane may confer the toxicity observed in the lipid II-accumulating strains. Suppressor analysis may ultimately be informative as to the precise molecular nature of this toxicity.

Transcriptional induction of *degP*. Finally, we note that the lipid II-mediated induction of *degP* transcription is mediated by increases in both Cpx and σ<sup>E</sup> activity (Fig. 7 and 8).

While lipid II accumulation manifests several phenotypes, all of these phenotypes intimate a perturbation in the structure of the outer membrane (e.g., SDS sensitivity, bile salt sensitivity, toxicity of export of LamB, and increased transcription of *degP*). The fact that lipid II accumulation stimulates both signal transduction pathways that control *degP* transcription further supports the hypothesis that both of these signal transduction pathways monitor extracytoplasmic stress (5, 6, 9, 15, 32). Moreover, the activation of both pathways indicates that under some circumstances both Cpx and σ<sup>E</sup> are affected by the same types of extracytoplasmic stresses.

### ACKNOWLEDGMENTS

We thank Susan DiRenzo for manuscript preparation and members of the Silhavy laboratory (especially Scott Hande) for comments and suggestions throughout the course of this work.

P.N.D. gratefully acknowledges support from a National Institutes of Health (NIH) training grant (GM07388). This work was supported by an NIGMS grant to T.J.S. (GM34821) and to P.D.R. (GM52882).

#### REFERENCES

1. Barr, K., and P. D. Rick. 1987. Biosynthesis of enterobacterial common antigen in *Escherichia coli*. In vitro synthesis of lipid-linked intermediates. J. Biol. Chem. **262**:7142–7150.
2. Caetano-Anolles, G. 1993. Amplifying DNA with arbitrary oligonucleotide primers. PCR Methods Appl. **3**:85–92.
3. Cain, B. D., P. J. Norton, W. Eubanks, H. S. Nick, and C. M. Allen. 1993. Amplification of the *bacA* gene confers bacitracin resistance to *Escherichia coli*. J. Bacteriol. **175**:3784–3789.
4. Connolly, L., A. de las Peñas, B. M. Alba, and C. A. Gross. 1997. The response to extracytoplasmic stress in *Escherichia coli* is controlled by partially overlapping pathways. Genes Dev. **11**:2012–2021.
5. Cosma, C. L., P. N. Danese, J. H. Carlson, T. J. Silhavy, and W. B. Snyder. 1995. Activation of the Cpx two-component signal transduction pathway in *Escherichia coli* suppresses envelope-associated stresses. Mol. Microbiol. **18**:491–505.
6. Danese, P. N., W. B. Snyder, C. L. Cosma, L. J. B. Davis, and T. J. Silhavy. 1995. The Cpx two-component signal transduction pathway of *Escherichia coli* regulates transcription of the gene specifying the stress-inducible periplasmic protease, DegP. Genes Dev. **9**:387–398.
6a. Danese, P. N., G. R. Oliver, and T. J. Silhavy. Unpublished observation.
7. de las Peñas, A., L. Connolly, and C. A. Gross. 1997. The σ<sup>E</sup>-mediated response to extracytoplasmic stress in *Escherichia coli* is transduced by RseA and RseB, two negative regulators of σ<sup>E</sup>. Mol. Microbiol. **24**:373–385.
8. Emr, S. D., and T. J. Silhavy. 1980. Mutations affecting localization of an *Escherichia coli* outer membrane protein, the bacteriophage lambda receptor. J. Mol. Biol. **141**:63–90.
9. Jones, C. H., P. N. Danese, J. S. Pinkner, T. J. Silhavy, and S. J. Hultgren. 1997. The chaperone-assisted membrane release and folding pathway is sensed by two signal transduction systems. EMBO J. **16**:6394–6406.
10. Kleckner, N., J. Bender, and S. Gottesman. 1991. Uses of transposons with emphasis on Tn10. Methods Enzymol. **204**:139–180.
11. Kuhn, H. M., U. Meier-Dieter, and H. Mayer. 1988. ECA, the enterobacterial common antigen. FEMS Microbiol. Rev. **4**:195–222.
12. Mäkelä, P. H., and H. Mayer. 1976. Enterobacterial common antigen. Bacteriol. Rev. **40**:591–632.
13. Männel, D., and H. Mayer. 1978. Isolation and chemical characterization of the enterobacterial common antigen. Eur. J. Biochem. **86**:361–370.
14. Marolda, C. L., and M. A. Valvano. 1995. Genetic analysis of the dTDP-rhamnose biosynthesis region of the *Escherichia coli* VW187 (O7:K1) *rfb* gene cluster: identification of functional homologs of *rfbB* and *rfbA* in the *rfb* cluster and correct location of the *rfbE* gene. J. Bacteriol. **177**:5539–5546.
15. Mecsas, J., P. E. Rouvière, J. W. Erickson, T. J. Donohue, and C. A. Gross. 1993. The activity of σ<sup>E</sup>, an *Escherichia coli* heat-inducible σ-factor, is modulated by expression of outer membrane proteins. Genes Dev. **7**:2618–2628.
16. Meier-Dieter, U., R. Starman, K. Barr, H. Mayer, and P. D. Rick. 1990. Biosynthesis of enterobacterial common antigen in *Escherichia coli*. Biochemical characterization of Tn10 insertion mutants defective in enterobacterial common antigen synthesis. J. Biol. Chem. **265**:13490–13497.
17. Misra, R., A. Peterson, T. Ferenci, and T. J. Silhavy. 1991. A genetic approach for analyzing the pathway of LamB assembly into the outer membrane of *Escherichia coli*. J. Biol. Chem. **266**:13592–13597.
18. Missiakas, D., M. P. Mayer, M. Lemaire, C. Georgopoulos, and S. Raina. 1997. Modulation of the *Escherichia coli* σ<sup>E</sup> (RpoE) heat-shock transcription-factor activity by the RseA, RseB and RseC proteins. Mol. Microbiol. **24**:355–371.
19. Missiakas, D., and S. Raina. 1997. Signal transduction pathways in response to protein misfolding in the extracytoplasmic compartments of *E. coli*: role of two new phosphoprotein phosphatases PrpA and PrpB. EMBO J. **16**:1670–1685.
20. Nikaido, H., and M. Vaara. 1985. Molecular basis of bacterial outer membrane permeability. Microbiol. Rev. **49**:1–32.
21. Pogliano, J., A. S. Lynch, D. Belin, E. C. Lin, and J. Beckwith. 1997. Regulation of *Escherichia coli* cell envelope proteins involved in protein folding and degradation by the Cpx two-component system. Genes Dev. **11**:1169–1182.
22. Raina, S., D. Missiakas, and C. Georgopoulos. 1995. The *rpoE* gene encoding the sigma E (sigma 24) heat shock sigma factor of *Escherichia coli*. EMBO J. **14**:1043–1055.
23. Reeves, P. R., M. Hobbs, M. A. Valvano, M. Skurnik, C. Whitfield, D. Coplin, N. Kido, J. Klena, D. Maskell, C. Raetz, and P. D. Rick. 1996. Bacterial polysaccharide synthesis and gene nomenclature. Trends Microbiol. **4**:495–503.
24. Rick, P. D., and R. P. Silver. 1996. Enterobacterial common antigen and capsular polysaccharides, p. 104–122. In F. C. Neidhardt, R. Curtiss III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter, and H. E. Umbarger (ed.), *Escherichia coli* and *Salmonella*: cellular and molecular biology, 2nd ed. ASM Press, Washington, D.C.

5884 DANESSE ET AL.

25. Rick, P. D., H. Mayer, B. A. Neumeyer, S. Wolski, and D. Bitter-Suermann. 1985. Biosynthesis of enterobacterial common antigen. J. Bacteriol. 162:494–503.
26. Rick, P. D., S. Wolski, K. Barr, S. Ward, and L. Ramsay-Sharer. 1988. Accumulation of a lipid-linked intermediate in enterobacterial common antigen synthesis in mutants lacking dTDP-glucose pyrophosphorylase. J. Bacteriol. 170:4008–4014.
27. Rouvière, P. E., A. de las Peñas, J. Mecsas, C. Z. Lu, K. E. Rudd, and C. A. Gross. 1995. *rpoE*, the gene encoding the second heat-shock sigma factor, σ<sup>E</sup>, in *Escherichia coli*. EMBO J. 14:1032–1042.
28. Silhavy, T. J., M. L. Berman, and L. W. Enquist. 1984. Experiments with gene fusions. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.
29. Simons, R. W., F. Houman, and N. Kleckner. 1987. Improved single and

multicopy *lac*-based cloning vectors for protein and operon fusion. Gene 53:85–96.
30. Slach, J. M., and T. J. Silhavy. 1991. *cis*-acting *ompF* mutations that result in OmpR-dependent constitutive expression. J. Bacteriol. 173:4039–4048.
31. Slach, J. M., S. Garrett, D. E. Jackson, and T. J. Silhavy. 1988. EnvZ functions through OmpR to control porin gene expression in *Escherichia coli* K-12. J. Bacteriol. 170:439–441.
32. Snyder, W. B., L. J. B. Davis, P. N. Danese, C. L. Cosma, and T. J. Silhavy. 1995. Overproduction of NlpE, a new outer-membrane lipoprotein, suppresses the toxicity of periplasmic LacZ by activation of the Cpx signal transduction pathway. J. Bacteriol. 177:4216–4223.
33. Stevenson, G., B. Neal, D. Liu, M. Hobbs, N. H. Packer, M. Batley, J. W. Redmond, L. Lindquist, and P. Reeves. 1994. Structure of the O antigen of *Escherichia coli* K-12 and the sequence of its *rfb* gene cluster. J. Bacteriol. 176:4144–4156.
