
Molecular and Biochemical Parasitology 96 (1998) 111–123

**Cloning, expression and reconstitution of the trypanothione-dependent peroxidase system of *Crithidia fasciculata*¹**

Emmanuel Tetaud, Alan H. Fairlamb*

Department of Biochemistry, Wellcome Trust Building, University of Dundee, Dundee DD1 5EH, Scotland, UK

Received 10 June 1998; received in revised form 24 July 1998; accepted 3 August 1998

---

### Abstract

As a consequence of aerobic metabolism, trypanosomatids are exposed to reactive oxygen intermediates such as superoxide, hydrogen peroxide and the hydroxyl radical. Metabolism of hydrogen peroxide in *Crithidia fasciculata* is accomplished by three distinct proteins, tryparedoxin, tryparedoxin peroxidase and trypanothione reductase, working in concert with the substrates NADPH and trypanothione. Here, we report the cloning and characterisation of the tryparedoxin (*TryX*) and tryparedoxin peroxidase (*TryP*) genes from *C. fasciculata*. Both genes are multicopy and organized in distinct tandem arrays in the genome. *TryX* encodes a 16 kDa protein, which belongs to the thioredoxin superfamily, sharing the WCPPC motif, whereas *TryP* encodes a 21 kDa protein belonging to a new class of peroxidases called 2-Cys peroxidoxins. Both *TryX* and *TryP* were expressed in *Escherichia coli* and the purified recombinant proteins shown to utilise hydrogen peroxide in the presence of NADPH, trypanothione and trypanothione reductase, similar to the native proteins. TryX is rapidly reduced by trypanothione, but weakly by glutathionylspermidine, glutathione or ovothiol A. TryP shows a broad substrate specificity and can reduce hydrogen peroxide, *t*-butyl hydroperoxide and cumene hydroperoxide with equal efficiency. © 1998 Elsevier Science B.V. All rights reserved.

---

### Keywords: Thioredoxin; Peroxidoxin; Thiol-specific antioxidant; Drug design; Trypanothione peroxidase system

---

### Abbreviations: T(SH)₂, trypanothione; T(S)₂, trypanothione disulphide; GspdSH, glutathionylspermidine; TryR, trypanothione reductase; GR, glutathione reductase; GSH, glutathione; TryX, tryparedoxin; TryP, tryparedoxin peroxidase; PCR, polymerase chain reaction.

* Corresponding author. Tel.: +44 1382 345155; fax: +44 1382 345542; e-mail: ahfairlamb@bad.dundee.ac.uk

¹ Note: Nucleotide sequence data reported in this paper are available in the EMBL, GenBank and DDJB databases under the GenBank accession numbers AF055913 (tryparedoxin) and AF055914 (tryparedoxin peroxidase).

0166-6851/98/$ - see front matter © 1998 Elsevier Science B.V. All rights reserved.
PII S0166-6851(98)00120-0

1. Introduction

    Like other organisms living in an aerobic environment, trypanosomatids are exposed to reactive oxygen intermediates such as superoxide anion ($\mathrm{O}_{2}^{-}$), hydrogen peroxide ($\mathrm{H}_{2}\mathrm{O}_{2}$) and hydroxyl radical ($\mathrm{OH}^{\bullet}$). The most reactive species, the hydroxyl radical, is generated from $\mathrm{O}_{2}^{-}$ and $\mathrm{H}_{2}\mathrm{O}_{2}$ by the transition metal ion-catalysed Haber–Weiss reaction [1]. The general strategy to minimize formation of $\mathrm{OH}^{\bullet}$ in cells is to maintain the levels of the precursors $\mathrm{O}_{2}^{-}$ and $\mathrm{H}_{2}\mathrm{O}_{2}$ as low as possible. The first line of defence involves removal of $\mathrm{O}_{2}^{-}$ by superoxide dismutase producing oxygen and hydrogen peroxide [2,3]. Trypanosomatids contain appreciable superoxide dismutase activity [4,5] and genes for iron-containing superoxide dismutases have been recently cloned for *Trypanosoma cruzi* and *Leishmania* [6–9]. The second line of defence involves removal of $\mathrm{H}_{2}\mathrm{O}_{2}$. In most cells this is normally achieved by catalase and various peroxidases [10,11]. However, medically important trypanosomatids do not contain catalase or the glutathione peroxidase/glutathione reductase (GR) system for detoxification of $\mathrm{H}_{2}\mathrm{O}_{2}$ [4,12–14]. Instead, metabolism of $\mathrm{H}_{2}\mathrm{O}_{2}$ is achieved via trypanothione ($N^{1}, N^{8}$-bis(glutathionyl)spermidine, T(SH)$_{2}$ [15] and trypanothione reductase (TryR) [16] together with a trypanothione-dependent peroxidase activity [17–20]. Although the dithiol (reduced) form of T(SH)$_{2}$ will reduce $\mathrm{H}_{2}\mathrm{O}_{2}$ non-enzymatically, the rate is too low to account for the activity observed in crude extracts in *T. brucei* and *Crithidia fasciculata* [18], however, this may be the case for *T. cruzi* [20]. Another unusual thiol identified in *C. fasciculata* and *L. donovani* by Steenkamp et al. [21,22] as ovothiol A ($N'$-methyl-4-mercaptohistidine) has also been proposed as contributing to $\mathrm{H}_{2}\mathrm{O}_{2}$ metabolism. However, its role has not been assessed quantitatively so far. This rather puzzling and confusing situation has been recently clarified for *C. fasciculata* by Nogoecke et al. [23]. In an elegant study, the trypanothione peroxidase system was shown to be comprised of two distinct proteins [Cf16, called tryparedoxin (TryX) and Cf21, called tryparedoxin peroxidase (TryP)] which work in concert with T(SH)$_{2}$ and TryR to metabolise $\mathrm{H}_{2}\mathrm{O}_{2}$ and other peroxides [23]. TryX which is a member of the thioredoxin superfamily [23,24], catalyses the reduction of TryP using T(SH)$_{2}$ as electron donor [23,25]. TryP is related to a new family of antioxidants, recently classified as peroxiredoxins [26], which are widely distributed in protozoan and helminth parasites [27].

    We report here the cloning and characterisation of both the tryparedoxin (TryX) and tryparedoxin peroxidase (TryP) genes from *C. fasciculata*, their expression in *Escherichia coli*, and their functional reconstitution with T(SH)$_{2}$ and TryR as the trypanothione peroxidase system originally identified by Henderson et al. [18]. Our findings confirm and extend those by Montemartini et al. [28] on the cloning and expression of the TryP from *C. fasciculata* and also provide novel sequence information on the electron donor, TryX.

2. Materials and methods

2.1. Organisms and reagents

    *C. fasciculata* clone HS6 [29] was used as source for DNA and RNA. Genomic DNA was isolated from *C. fasciculata* as described [30]. Routine DNA manipulations were performed in *E. coli* strain JM109 and XL1-Blue. Overexpression in *E. coli* was performed in strain BL21(DE3). All chemicals were of the highest grade available and from Sigma or BDH. T(S)$_{2}$ and glutathionylspermidine (GspdSH) were purchased from Bachem and DL-ovothiol A was a gift from Dr Paul Hopkins of Washington State University (Seattle, USA). Restriction enzymes and DNA modifying enzymes were from Promega.

2.2. Amplification of a tryparedoxin gene fragment

    Two degenerate oligonucleotides were designed based on the published sequence of two trypsin peptides from Cf16 [23] (KLVFFYF, TryX-5': 5'-AAR YTN GTN TTY TAY TT-3' and KDPEGEQ; TryX-3': 5'-YTG YTC NCC YTC NGG RTC YTT-3'). The polymerase chain reaction (PCR) was performed in a 50 $\mu$l volume

containing 0.2 mM each dNTP, 500 ng each primer, 100 ng *C. fasciculata* total genomic DNA and 2.5 units Taq DNA polymerase with buffer (Bioline). Before adding the enzyme, the PCR mixture was heated for 10 min at 95°C, then cooled rapidly to 30°C. Following centrifugation, the enzyme was added and the reaction mixture covered with oil and subjected to the following for 30 cycles: denaturation for 10 s at 95°C; ramping at 2°C s⁻¹ to 40°C; annealing for 1 s at 40°C; and elongation for 30 s at 72°C. A final 10 min extension at 72°C was included. A 336 bp DNA fragment was identified following agarose gel electrophoresis of the PCR products. The DNA fragment was cloned into the SmaI site of pUC18 (SureClone kit, Pharmacia) giving the plasmid pTryX.

### 2.3. Amplification of a trypanothione reductase gene fragment

Two degenerate oligonucleotides were designed based on the peptide sequences of two proteolytic fragments of Cf21 [23] (APEFDDM, TryP-5′: 5′-GCN CCN GAR TTY GAY GAY ATG-3′ and WKKGDAA, TryP-3′: 5′-NGC NGC RTC NCC YTT YTT CCA-3′). PCR was performed as described above and a 516 bp DNA fragment was cloned into the SmaI site of pUC18 giving the plasmid pTryP.

### 2.4. cDNA synthesis

cDNA was synthesized from either total or poly(A)+ RNA with a first-strand cDNA synthesis kit (Pharmacia) using an oligo(dT) as primer and then amplified directly by PCR using oligonucleotides within the coding sequence of TryX and TryP (TryX-int5′: 5′-GTG GAA CTT GTC GTA GAA CTC-3′ or TryP-int5′: 5′-CTT GTA GGA CGA CAG GCT GAC-3′) together with an oligonucleotide corresponding to the splice leader sequence of *C. fasciculata* (MiE: 5′-ATC ACT TTC TGT ACT TTA TTG-3′). Products were separated by agarose gel electrophoresis and sequenced after cloning into plasmid pUC18.

---

### 2.5. Amplification of the complete coding sequence and the intergenic regions of TryX and TryP

Complete coding sequences of TryX and TryP were obtained by PCR amplification using native Pfu DNA polymerase (Stratagene) and oligonucleotides corresponding to the start and stop codon (underlined) of the coding sequence of TryX and TryP, respectively (ExpTryX-5′: 5′-TTC CAT **ATG** TCC GGG CTT GACAAGTAC-3′ and ExpTryX-3′: 5′-AGC GGA TCC **TTA** CGG AGC GTC CTT CCA CGG GAA C-3′; ExpTryP-5′: 5′-GCA CAT **ATG** TCC TGC GGT GCC GCC AAG-3′ and ExpTryP-3′: 5′-GCT GGA TCC **CTA** GTG GCC CTC CTT CTT GGC-3′). The DNA fragments were cloned into the SmaI site of pUC18 and sequenced on both strands. Three different clones of TryX and 5 of TryP were sequenced.

The intergenic regions were obtained by PCR using oligonucleotides localized in the coding sequence of TryX and TryP, respectively (TryX-int5′: 5′-GTG GAA CTT GTC GTA GAA CTC-3′ and TryX-int3′: 5′-ACT TCA ACG TGG AGT CGA TCC-3′; TryP-int5′: 5′-CTT GTA GGA CGA CAG GCT GAC-3′ and TryP-int3′: 5′-GCC AAC TGG AAG AAG GGC GAC-3′). The DNA fragments were cloned into the SmaI site of pUC18 and 4 different clones of the TryX and 4 of the TryP intergenic regions were sequenced.

---

### 2.6. DNA sequencing

Sequencing of double-stranded DNA was accomplished by the dideoxynucleotide chain termination method [31] by automated cycle sequencing using the dye terminator method (ABI PRISM dye terminator kit, Perkin Elmer). DNA and amino acid sequences were analysed with the Lasergene system (DNASTAR) and with the Pileup programme from the GCG Wisconsin Sequence Analysis Software Package.

---

### 2.7. Expression of trypanothione reductase and trypanothione peroxidase in *E. coli*

*E. coli* strain BL21(DE3) expressing TryX was obtained by transformation with TryX cloned in

the expression vector pET-15b (Novagen) which adds an N-terminal 6-His-tag to facilitate purification. The gene was obtained by PCR amplification of the *C. fasciculata* TryX open reading frame using oligonucleotide ExpTryX-5′: 5′-TTC CAT ATG TCC GGG CTT GAC AAG TAC-3′ for the sense strand containing an *NdeI* cloning site (underlined) incorporating an initiation codon (ATG). For the antisense strand, oligonucleotide ExpTryX-3′: 5′-AGC GGA TCC TTA CGG AGC GTC CTT CCA CGG GAA C-3′ was designed which includes a stop codon with a *BamHI* restriction site (underlined) just downstream. The PCR product was blunt-end ligated into the *SmaI* site of pUC18 (SureClone kit from Pharmacia) giving the plasmid pUC18-*TryX*. The insert was removed from this subclone by digestion with *NdeI-BamHI* and ligated with the pET-15b vector creating plasmid pET-*TryX*.

E.coli strain BL21(DE3) expressing TryP was obtained by transformation with *TryP* cloned in the expression vector pET-15b. The gene was obtained in a similar fashion to *TryX* using the following primers: ExpTryP-5′: 5′-GCA CAT ATG TCC TGC GGT GCC GCC AAG-3′ for the sense strand and ExpTryP-3′: 5′-GCT GGA TCC CTA GTG GCC CTC CTT CTT GGC-3′ for the antisense strand. The PCR product was blunt-end ligated into *SmaI* site of pUC18 (SureClone kit from Pharmacia) giving the plasmid pUC18-*TryP*. The insert was removed from this subclone by digestion with *NdeI-BamHI* and ligated with the pET-15b vector creating plasmid pET-*TryP*.

BL21(DE3)/pET-*TryX* or BL21(DE3)/pET-*TryP* were grown at 37°C with vigorous aeration in Luria-Bertani broth containing 100 μg ml⁻¹ ampicillin. When cultures reached an OD₆₀₀ of ≈0.6, isopropyl-β-D-thiogalactopyranoside was added to a concentration of 0.4 mM to induce expression of recombinant TryX or TryP protein. Cultures were grown an additional 3–4 h and then harvested by centrifugation.

### 2.8. Purification of recombinant His-tagged trypanothione reductase and trypanothione peroxidase by affinity chromatography

All procedures were performed at 4°C. After 3–4 h induction, cells were resuspended in 50 ml binding buffer [5 mM imidazole, 500 mM NaCl in 20 mM Bis-Tris propane (pH 7.5)] and lysed by flash-freezing in liquid nitrogen, followed by rapid thawing, then sonication (6 × 45 s pulses interrupted with cooling on ice). Cell debris was separated by centrifugation at 15000 × g.

The supernatant was applied to a Ni²⁺-resin column (Poros 4.6 mm/100 mm) pre-equilibrated with binding buffer at a flow rate of 8 ml min⁻¹. The column was washed extensively of 10 mM imidazole in Bis-Tris propane pH 7.5 (15 column volumes). TryX and TryP eluted with 60 mM and 350 mM imidazole in Bis-Tris propane pH 7.5, respectively. Fractions were analysed by SDS-PAGE. Fractions of about 70% or greater purity were pooled, dialysed against loading buffer and subject to a second purification on the Ni²⁺-resin column. Fractions were analysed by SDS-PAGE and pure TryX and TryP were pooled, dialysed against 50 mM Hepes pH 7.5 and concentrated.

### 2.9. Enzyme activities

Enzyme activities were determined at 25°C exactly as described in Nogoceke et al. [23] except in our study T(S)₂ was generated enzymatically from trypanothione disulphide [T(S)₂]. Briefly, the assay contained 0.15 mM NADPH, 1 mM EDTA, 50 μM H₂O₂, 50 μM T(S)₂, 17 nM pure recombinant *T. cruzi* TryR, 300 nM TryX and 300 nM TryP in 50 mM Hepes pH 7.5. NADPH consumption was measured spectrophotometrically at 340 nm. In some experiments, TryR and T(S)₂ were replaced by 20 nM human GR and 50 μM glutathione disulphide (GSSG), respectively, using a buffer containing 80 mM KCl in 50 mM Hepes pH 7.0. Protein concentration was determined using the Bio-Rad protein assay reagent with bovine serum albumin as standard.

### 3. Results

#### 3.1. Isolation and characterisation of the trypanothione genes (*TryX*)

Degenerate oligonucleotide primers TryX-5′ and TryX-3′ complementary to internal peptide se-

(A1 - TryX)


taagaaatgcgcaacgaagaagagaacggcggagggtggcgactgctgcccggctgggttaccg 44
tgccctgccacgctggtggagtgttgtgactgtggtgctgttggtgctgtttggtgctgtct 115
ttgtttgcttggtggcgactgtgtttggtgctgtttggtgctgtctccccctgtctccccct 186
ccttcttcttcttcttcttcttcttcttcttcttcttcttcttcttcttcttcttcttcttct 257
ctgcgacccccgtgcccctcccgccgcaagggacagtgcgcccaaactgcgctactcacgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctgctg
quences amplified a 336 bp fragment from *C.*  
*fasciculata* genomic DNA, which was subsequently  
cloned into pUC18 giving the plasmid pTryX.  
Sequence analysis verified that this DNA fragment  
encoded an open reading frame with sequence  
similarity to proteins of the thioredoxin family [24].  

*C. fasciculata* genomic DNA was digested with  
several restriction enzymes and probed with the  
labelled TryX-5'/TryX-3' PCR fragment (data not  
shown). The probe hybridised to several fragments,  
suggesting that *TryX* genes are organized in a  
tandem repeat containing at least 6 copies as shown  
schematically in Fig. 1B1. Based on this observation, we decided to amplify by PCR the 5' and 3'  
coding sequences along with the intergenic regions  
(see Section 2.5). Four different clones were isolated  
containing a DNA fragment of ≈ 1.1 kb. Sequence  
analysis verified that these DNA fragments encoded  
the *TryX* 5' and 3' coding sequence along with the  
intergenic region (Fig. 1A1). Several differences  
were observed in the intergenic sequence, suggesting  
that we had amplified different regions of a tandem  
repeat (Fig. 1A1). The complete coding sequence of  
*TryX* was obtained by PCR amplification using  
primers corresponding to the start and stop codon,  
respectively (see Section 2.5). Three different clones  
were sequenced and all yielded the same coding  
sequence (Fig. 1A1). Reconstitution of a complete  
repeat unit is presented Fig. 1A1. The repeat unit  
(1292 bp) contains a 438 bp open reading frame  
encoding a 146-amino acid protein with a predicted  
molecular mass of 16479 Da (Fig. 2A), which  
correlates well with the previously determined  
molecular mass (16396 ± 10 Da) [23]. Each of the  
six peptide sequences determined from the 16 kDa  
protein [23] were found in the predicted protein  

product. cDNA synthesis followed by PCR amplification using internal and mini-exon primers (see  
Section 2.4) were performed to map the AG consensus splice leader acceptor site. Sequence analysis of  
these PCR products confirmed the splicing site as  
nucleotide 354 (Fig. 1A1).

### 3.2. Isolation and characterisation of the  
tryparedoxin peroxidase genes (*TryP*)

A 516 bp fragment was amplified from *C. fasciculata* genomic DNA using the primers TryP-5'/  
TryP-3' [designed from two of the above peptide  
sequences (see Section 2.3)] and subsequently  
cloned into pUC18 giving the plasmid pTryP.  
Sequence analysis verified that this DNA fragment  
encoded an open reading frame with sequence  
similarity to *T. brucei rhodesiensis* thiol-specific  
antioxidant gene [32] and several peroxidoxins from  
various organisms [26,27].

*C. fasciculata* genomic DNA was digested with  
several restriction enzymes and probed with the  
labelled TryP-5'/TryP-3' PCR fragment (data not  
shown). As for *TryX*, the probe hybridised to  
several fragments, suggesting that the *TryP* genes  
were also organized in a tandem repeat containing  
at least five copies as shown schematically in Fig.  
1B2. Again, PCR amplification using internal  
primers gave a PCR product containing the 5' and  
3' coding sequences along with the intergenic regions (see Section 2.5). Four different clones were  
isolated containing a DNA fragment of ≈ 1.1 kb.  
Sequence analysis verified that these DNA fragments encoded the *TryP* 5' and 3' coding sequence (Fig. 1A2).  
Several differences were observed in the intergenic

---

Fig. 2. Sequence alignment of *TryX* with other thioredoxin proteins (A), and *TryP* with several peroxidoxin proteins (B). (A),  
alignment of *TryX* from *C. fasciculata* (*Cf-TryX*) with the putative thioredoxin from *C. elegans* (*Ce-Trx*; AC: Q09489), *A. thaliana*  
(*At-Trx*; AC: Q38880) and three authentic thioredoxins from human (*Hu-Trx*; AC: P10599), *S. cerevisiae* (*Sc-Trx*; AC: P22803,  
Trxl) and *E. coli* (*Ec-Trx*; AC: P00274, Trxl). Residues shown in bold and underlined highlight the identity between *C. fasciculata*  
thioredoxin and the putative thioredoxin from *C. elegans*. The sequences shown white on black highlight the active site of known  
thioredoxins. (*) and (+) indicate the identity and similarity, respectively. (B), alignment of *TryP* from *C. fasciculata* (*Cf-TryP*) with  
several putative or identified peroxidoxin proteins from various organisms: *T. brucei rhodesiensis* (*Tb-TryP*; AC: Q26695; Tbr-TSA);  
*F. hepatica* (*Fh-Tpx*; AC: U88577); *D. immitis* (*Di-Tpx*; AC: AF004167); *S. cerevisiae* (*Sc-Tpx*; AC: P34760, TSA1); *B. malayi*  
(*Bm-Tpx*; AC: Q17172, Tpx2); *O. volvulus* (*Ov-Tpx*; AC: AF029247, Tpx2); human (*Hu-Tpx*; AC: Q06830, Tpx2); *C. elegans*  
(*Ce-Tpx*; AC: S43598); *E. coli* (*Ec-Tpx*; AC: P26427, AHPC); *E. histolytica* (*Eh-Tpx*; AC: P19476, Eh29) and *A. thaliana* (*At-Tpx*;  
AC: Q96291, BAS1 precursor). (*) indicate identity in all the sequence and (~) indicate identity in 8 of 12 sequences. Sequences shown white on black indicate the two conserved (VCP) sequence implicated in the active site. Sequences are aligned in order of  
similarity.

(A)

| Cf-TryX | MSGLDKYLPG IEKLRRGDGE VEVK.SLAGK LVFFYFSASW | CPPCRGFTPQ LIEFYDKFHE SKNFEVVFCT WDEEEDGFAG | 79 |
| Ce-Trx | MSLLAG VKLEKRDKTL VDATEALAGK AVGFYSAHW | CPPCRGFTPI LKDFYEEVED ..EFVVVS FDRSESDLKM | 74 |
| At-Trx | MAAEEGQVIG CHTNDVWTVO LDKAKE.SNK LIVIDFTASW | CPPCRMIAPI FNDLAKKFMS SAIFFKV... | 66 |
| Hu-Trx | VKQIESKTAF QEALDAAGDK LVVVDFSATW | CGPCKMIKPF FHSLSEKY.S NVIFLEV... | 56 |
| Sc-Trx | VTQLKSASEY DSALAS.GDK LVVVDFFATW | CGPCKMIAPM IEKFAEQY.S DAAFYKL... | 55 |
| Ec-Trx | SDKII HLTDDSFDTD VLKADG.A.. .ILVDFWAEW | CGPCKMIAPI LDEIADEYQG KLTVAKL... | 58 |

+ * + * * * * + +


| Cf-TryX | YFAKMP.WLA VPFAQSEAVQ KLSKHFNVES IPTLIGVDAD | SGDVVT.TRA RATLVKDPEG EQFPWKDAP | 146 |
| Ce-Trx | YMSEHGDWYH IPYGN.DAIK ELSTKYGVSG IPALIIVKPD | GTEVTKDGRN DVQNGKDPKA TVAWRA | 140 |
| At-Trx | ...DVDELQ SVAKEFGVEA MPTFVFIKAG | EVVDKLVGAN .KEDLQAKI | 110 |
| Hu-Trx | ...DVDDCQ DVASECEVKC MPTFQFFKKG | QKVGEFSGAN .KEKLEATIN ELV | 104 |
| Sc-Trx | ...DVDEVS DVAQKAEVSS MPTLIFYKGG | KEVTRVVGAN PAAIKQAIAS NV | 103 |
| Ec-Trx | ...NIDQNP GTAPKYGIRG IPTLLLLFKNG | EVAATKV GAL SKGQLKEFLD ANLA | 108 |

+ + + + ++ +


(B)

| Ce-Tpx | MFSSA VRALCRTVPT VATRQLSTSR | 25 |
| Eh-Tpx | MSCNQ QKECCKKECQ EKECCKECCC PRIKAFKKFI | 35 |
| At-Tpx | MASVA SSTTLISSPS SRVFPAKSSL SSPSVSFLRT LSSPSASASL RSGFARRSSL SSTSRRSFAV | 65 |
| Cf-TryP | MSCGAAKL NHPAPEFDDM ALMPNGTFKK VSLSSYKGK. YVVLFFYPMD | FTFVCPT EII QFSDDAKRFA EI..NTEVIS | 77 |
| Tb-TryP | MSCGDAKL NHPAPHFNEV ALMPNGTFKK VDLASYRGK. WVVLFFYPLD | FTFVCPT EIC QFSDRVKEFN DV..DCEVIA | 77 |
| Fh-Tpx | MLQP NMPAPNFSGQ AWVGK.EFET ISLSDYKGK. WVILAFYPLD | FTFVCPT EII AISDQMEQFA QR..NCAVIF | 72 |
| Di-Tpx | MTLAGSKAFI GQPAPNFKTT AVV.NGDFKE ISLCQFKGK. YVVLFFYPLD | FTFVCPT EII AFSDRIAEFK KL..DVAVMA | 78 |
| Sc-Tpx | MVAQV QKQAPTFKKT AVV.DGVFDE VSLDKYKGK. YVVLAFIPLA | FTFVCPT EII AFSEAAKKFE EQ..GAQVLF | 73 |
| Bm-Tpx | MTLAGSKAFI GQPAPNFKTT AVV.NGDFKE ISLGQFKGK. YVLLFYPLD | FTFVCPT EII AFSDRIAEFK QL..DVAVMA | 78 |
| Ov-Tpx | MTLAGSKAFI GQPAPNFKTT AVV.NGDFKE ISLNQFKGK. YVVLFFYPLD | FTFVCPT EII AFSDRISEFK KL..DVAVMA | 78 |
| Hu-Tpx | MSSGNAKI GHPAPNFKAT AVMPDGQFKD ISLSDYKGK. YVFFFFYPLD | FTFVCPT EII AFSDRAEEFK KL..NCQVIG | 77 |
| Ce-Tpx | ALLSLRPLGP KNTVPAFKGT AVV.DGDFKV ISDQDYKGK. WLVMFFYPLD | FTFVCPT EII AYGDRANEFR SL..GAEVVA | 103 |
| Ec-Tpx | MSLIN TKIKPKFNQA FK..NGEFIE ITEKDTEGR. WSVFFFYPAD | FTFVCPT ELG DVADHYEELQ KL..GVDVYA | 72 |
| Eh-Tpx | NTFEKAQIGK FTEAPEFKAP AYCPCGSIKE IDINEYRGK. YVLLFYPLD | WTFVCPT EMI GYSELAGQLK EIFTNCEVIG | 114 |
| At-Tpx | KAQADDLPLV GNKAPDFKAE AVF.DQEFIK VKLSDYNGKK YVILFFYPLD | FTFVCPT EIT AFSDRHSEFE KL..NTEVLG | 144 |

~* ~~ ~~~ ~**~ ~~~~~~~~ **~~~~~ ~~~ *


| Cf-TryP | CSCDSEYSHL QWTSVDRKKG GLGPMAIPML ADKTKAIARA YGVLDEDSGV A..YRGVFI I DPNGKLRQII INDMPIGRNV | 157 |
| Tb-TryP | CSMDSEFSHL AWTNVERKKG GLGTMNIPI L ADKTKSIMKA YGVLKEEDGV A..YRGLFI I DPQQNLRQIT INDLPVGRNV | 157 |
| Fh-Tpx | CSTDSVYSHL QWTKMDRKVG GIGQLNFPLL ADKNMSVSRA FGVLDEEQGN T..YRGNFLI DPKGVLRQIT VNDDPVGRSV | 152 |
| Di-Tpx | CSTDSHFSHL AWVNTDRKMG GLGQMNIPI L ADTNHTISRA YGVLKEDDGI A..YRGLFI I DPKGILRQIT INDLPVGRNV | 158 |
| Sc-Tpx | ASTDSEYSL L AWTNIPRKEG GLGPINIPLL ADTNHLSRD YGVLIEEEGV A..LRGLFI I DPKGVI RHIT INDLPVGRNV | 153 |
| Bm-Tpx | CSTDSHFSHL AWVNTDRKMG GLGQMNIPI L AYT NHVISRA YGVLKEDDGI A..YRGLFI I DPKGIL LGQIT INDLPVGRSV | 158 |
| Ov-Tpx | CSTD SHFSHL AWVNTDRKMG GLGQMNIPI L ADT NHAISKA YGVLKEDEGI A..YRGLSI I DSKGILRQIT VNDLPVGRSV | 158 |
| Hu-Tpx | ASVDSHFCHL AWVNTPKKQG GLGPMNIPLV SDPKRTIAQD YGVLKADEGI S..FRGLFI I DD KGILRQIT VNDLPVGRSV | 157 |
| Ce-Tpx | CSCDSHFSHL AWVNTPRKDG GLGDMDIPLL ADFNKKIADS FGVLDKESGL S..YRGLFLI DPSGTVRHTT CNDLPVGRSV | 183 |
| Ec-Tpx | VSTDTHFTHK AWHSSSE... TIAKIKYAMI GDPTGALTRN FDNMREDEGL A..DRATFVV DPQGI IQAIE VTAEGIGRDA | 149 |
| Eh-Tpx | VSVDSVYCHO AWCEADKSKG GVGKLTFPLV SDIKRCISIK YGMLNVEAGI ARRFTGYVII DDKGKVRYIQ MNDDGIGRST | 194 |
| At-Tpx | VSVDSVFSHL AWVQTDRKSG GLGDLNYPLI SDVTKSISKS FGVLIH DQGI A..LRGLFI I DKEGVIQHST INNLGIGRSV | 224 |

* *~ ~~~~ ~* ~~~ ~~~ ~ ~ ~~~~ *~ ~ ~~~~ **~~~


| Cf-TryP | EEVIRLVEAL QFVEEH.GEV | CPANWKKGDA KKKEGH | 188 |
| Tb-TryP | DETLRLVKAF QFVEKH.GEV | CPANWKPGSK TMKADPNGSQ DYFSSMN | 199 |
| Fh-Tpx | EEALRLLD AF IFHEEH.GEV | CPANWKPKSK TIVPTPDGSK AYFSSAN | 194 |
| Di-Tpx | DETRLLIQAF QFVDNH.GEV | CPANWQPGSE AIKPGVKESK AYFEKH | 199 |
| Sc-Tpx | DEALRLVEAF QWTDKN.GTV | LPCNWTPGAA TIKPTVEDSK EYFEAANK | 196 |
| Bm-Tpx | DETRLLIQAF QFVDKH.GEV | CPANWHPGSE TIKPGVKESK AYFEKH | 199 |
| Ov-Tpx | DETLRLVQAF QFVDNH.GEV | CPANWQPGSE TIKPEVKESK EYFGKH | 199 |
| Hu-Tpx | DETLRLVQAF QFTDKH.GEV | CPAGWKPGSD TIKPDVQKSK EYFSKQK | 199 |
| Ce-Tpx | DETLRLVKAF QFSDKH.GEV | CPADWHEDSP TIKPGVATS K EYFNKV NK | 226 |
| Ec-Tpx | SDLLRKIKAA QYVASHPGEV | CPAKWKEGEA TLAPSLDLVG KI | 187 |
| Eh-TPx | EETIRIVKAI QFSDEH.GAV | CPLNWKP GKD TIEPTFTPDG IKKYLT AH | 241 |
| At-Tpx | DETMRTLQAL QYTGNP.DEV | CPAGWKSGEK SMKPDPKLSK EYFSAI | 265 |

~~~~*~ ~~~~*~ ~~~~

Fig. 2.

sequences, suggesting that we had amplified different intergenic regions of a tandem repeat (Fig. 1A2). The complete coding sequence was obtained by PCR amplification using primers corresponding to the start and stop codon, respectively (see Section 2.5). Five different clones were sequenced and all yielded the same coding sequence (Fig. 1A2). Reconstitution of a complete repeat unit is presented in Fig. 1A2. The repeat unit (1518 bp) contained a 564 bp open reading frame encoding a 188 amino acid protein with a predicted molecular mass of 20957 Da (Fig. 2B), which correlates well with the previously determined molecular mass (21028 ± 120 Da) [23]. All of the eight peptide sequences determined from the 21 kDa protein [23] were found in the predicted protein product. As described for TryX, reverse transcription-PCR was performed in order to map the AG consensus spliced leader acceptor site of TryP to nucleotide 390 (Fig. 1A2).

### 3.3. Sequence analysis and comparison

The predicted translation products of TryX present the best homology with a translated sequence from *Caenorhabditis elegans*, *Arabidopsis thaliana* and with several other thioredoxin proteins (Fig. 2A). TryX shares a strong homology with the putative *C. elegans* protein, which also possess an insertion of 26 amino acids in the middle of the protein which is not found in the other thioredoxin proteins (Fig. 2A). Another interesting feature arising from the alignment of these different proteins is that the amino acid sequence of TryX shares a motif WCPPCR with the *C. elegans* and *A. thaliana* sequences, which is similar to the WCGPC motif of *Saccharomyces cerevisiae*, human and *E. coli* (Fig. 2A). This motif has been implicated in the active site of thioredoxins, where both cysteine residues are responsible for redox activity [33].

The predicted translation product of TryP is strongly homologous (34–75%) to several peroxidoxins from various organisms including recently characterised thioredoxin peroxidases from various parasites (Fig. 2B). The two conserved motifs (VCP) implicated in the active site of the peroxidoxins [26,34] were also conserved in TryP. Interestingly, our TryP possess an alanine in position 111 instead of a glycine as recently reported by Montemartini et al. [28], suggesting that all the copies in tandem array are not identical.

### 3.4. Overexpression of TryX and TryP in *E. coli* and purification

TryX was cloned into an expression vector (pET-15b) for overproduction of the protein in *E. coli*. Growth of BL21(DE3)/pET-TryX strain in the presence of isopropyl-β-D-thiogalactopyranoside resulted in overproduction of a protein which migrated with an apparent molecular mass of ≈ 19 kDa (Fig. 3), which correlates with the molecular mass predicted for TryX (16.6 kDa) plus the 2180 Da corresponding to the N-terminal His-tag sequence. The addition of six histidine residues to the N-terminus allowed a fast and simple purification of the recombinant protein on a Ni²⁺-resin column. After elution with 60 mM imidazole, the protein was found not to be homogenous by SDS-PAGE. Therefore, a second purification was performed on the same column, resulting in a product that was ≈ 95% pure (Fig. 3, lane 2).

*E. coli* strain BL21(DE3)/pET-TryP strain overproduced a protein which migrated with an apparent molecular mass of about 23 kDa (Fig. 3) which also correlates with the mass predicted from the primary amino acid sequence (20.9 kDa) plus the His-tag. TryP was purified exactly as described above for TryX, only 350 mM imidazole was required to elute the recombinant protein. Again, a product of about 95% purity was obtained (Fig. 3, lane 4).

### 3.5. Reconstitution of the trypanothione peroxide system

Peroxide metabolism has been recently characterized by Nogoceke et al. [23], and shown to be the result of the action of three distinct proteins, TryX, TryP and TryR, working in concert with

oxide and cumene hydroperoxide are also substrates for TryP (Fig. 4H,I) with a similar activity to H₂O₂. The small decreases in absorbance observed after addition of the substrates correspond mainly to the dilution of the NADPH (Fig. 4A), but also to the reduction of TryX by T(SH)₂ and TryP by TryX. In fact, TryX and TryP were found to be in the reduced form after purification, but TryP was, after few days, found to be partially oxidized. Surprisingly, using E. coli thioredoxin reductase and Spirulina sp. thioredoxin as substrates for the reduction of TryP, we observed a consumption of NADPH. This suggests that TryP can also be reduced by thioredoxin, although the rate was 10-fold lower than with TryX and T(SH)₂/TryR (data not shown).

## 4. Discussion

The 'thioredoxin-dependent peroxidase system' of yeast and mammalian cells [35] involves the sequential transfer of reducing equivalents from NADPH via three distinct proteins to a final peroxide acceptor. Thioredoxin reductase, thioredoxin and thioredoxin peroxidase all contain reducible disulphide groups that are thought to undergo cyclical reduction and oxidation during catalysis [26]. The 'trypanothione-dependent peroxidase system' is analogous in many respects to the thioredoxin-dependent system, only our reconstitution studies show that TryR is unable to directly reduce TryX, and that T(SH)₂ is an essential intermediary in the process. Moreover, the inability of GSH and GR to substitute for T(SH)₂ and TryR in the overall reaction clearly distinguishes the trypanothione peroxidase system from other glutathione-dependent peroxidases.

Our sequence analysis of *TryX* has revealed a number of important features. Alignment of TryX with other thioredoxins strongly supports the earlier proposal based on peptide sequence information that TryX is a homologue of thioredoxin [23]. Moreover, the specific requirement for T(SH)₂ as reducing agent supports the proposed nomenclature of tryparedoxins as a new

class of thioredoxins [23]. In addition, in contrast to classical thioredoxins, which possess the active site sequence WCGPC, tryparedoxin contains a WCPPC motif that also occurs in thioredoxin-like proteins from *A. thaliana* [36], *Paracoccus denitrificans* (AC: P52236), *C. elegans* [37] and in the nucleoredoxin of mice [38]. TryX shares the strongest homology with a thioredoxin-like protein from *C. elegans*, both of which possess a similar insertion just after the active site. The *C. fasciculata* insertion contains several negatively charged amino acids which could be involved in the specific recognition of T(SH)₂, (which is positively charged) and exclusion of GSH, (which is negatively charged). However, this would not account for the low activity observed with GspdSH and other functions such as recognition by its cognate peroxidase are also possible. Sequence analysis of TryX's from other trypanosomatids together with mutagenesis and structural studies would help clarify this problem. Thioredoxins from other organisms play a number of other important roles in metabolism and homeostasis, including the important role as an electron donor for ribonucleotide reductase or as hydrogen donor in the sulphatase assimilation pathway [33]. In addition, members of the thioredoxin family can facilitate refolding [39], activate receptors [40], or modulate the DNA-binding activity of transcription factors [41]. Similar roles in trypanosomatids deserve investigation, since ribonucleotide reductase has been recently cloned from *L. mexicana amazonensis* and *T. brucei* [42,43].

Peroxidoxins have been found in various organisms, from prokaryotes to mammals, al-

city of TryP using *t*-butyl hydroperoxide (H) and cumene hydroperoxide (I); (J–K) TryR and T(S)₂ dependent hydroperoxide metabolism, the high activity observed at the beginning of the reaction (J–K) reflects the immediate reduction of T(S)₂, with the second rate reflecting the reduction of hydrogen peroxide. T(S)₂, TryR, trypanothione reductase; GR, glutathione reductase; GSSG, glutathione disulphide; (GspdS)₂, glutathionylspermidine disulphide; t-BOOH, *t*-butyl hydroperoxide; CuOOH, cumene hydroperoxide. The rate of reduction of H₂O₂, expressed as μmol NADPH oxidised per min per ml, is indicated above each curve.
though is it not known if all of these proteins catalyse the reduction of hydroperoxides. TryP from *C. fasciculata* belongs to this growing family of proteins. Sequence alignment of peroxidoxins reveals the existence of two different groups, based on the presence of one or two very conserved cysteine residues [27]. TryP belongs to the 2-Cys peroxidoxin family which have two conserved VCP motifs in positions 52 and 173 of the TryP amino acid sequence (Fig. 2B). The best characterized 2-Cys peroxidoxin is yeast Tpx, which functions as a homodimer and contains two cysteine residues, Cys⁴⁷ and Cys¹⁷⁰, which are essential for the thioredoxin-coupled peroxidase activity [44]. In contrast to the well characterized peroxidoxins (*E. coli*, yeast and mammal), TryP exists as an oligomeric complex [23], suggesting a different mechanism for the reduction of H₂O₂. Moreover, TryP is able to reduce different hydroperoxides, including organic or lipid hydroperoxides [23], with similar activities.

Some peroxidoxins have been implicated in redox regulation of gene transcription in mammals [45]. For example, by regulating the intracellular concentration of H₂O₂, they are believed to participate in the signalling cascades of certain growth factors and tumor necrosis factor-α [46], thereby regulating the activity of transcription factor (NF-κB) [47], and inhibiting apoptosis [48]. Moreover, five different human peroxidoxin genes have been identified, suggesting multiple functions for these proteins [47].

Whether redox-mediated signalling pathways exist in parasites is not clear at present. However, all parasites have to cope with the removal of reactive oxygen intermediates not only as a result of aerobic metabolism but also as a consequence of the host immune response. The conservation of all these proteins and their abundance in parasites suggest a important role in defence against oxidant stress and possibly other house-keeping functions.

### Acknowledgements

We particularly thank M.S. Alpehy for technical assistance with the purification of the recombinant enzymes and M. Berriman for purified trypanothione reductase. This work was supported by the Wellcome Trust.

### References

[1] Aust SD, Morehouse LA, Thomas CE. Role of metals in oxygen radical reactions. J Free Radic Biol Med 1985;1:3–25.

[2] Bannister JV, Bannister WH, Rotillo G. Aspects of the structure, function, and applications of superoxide dismutase. Crit Rev Biochem 1987;22:111–80.

[3] Fridovich I. Superoxide dismutases: an adaptation to a paramagnetic gas. J Biol Chem 1989;264:7761–4.

[4] Boveris A, Stoppani AO. Hydrogen peroxide generation in *Trypanosoma cruzi*. Experientia 1977;33:1306–8.

[5] Le Trang N, Meshnick SR, Kitchener K, Eaton JW, Cerami A. Iron-containing superoxide dismutase from *Crithidia fasciculata*: purification, characterization, and similarity to leishmanial and trypanosomal enzymes. J Biol Chem 1983;258:125–30.

[6] Temperton NJ, Wilkinson SR, Kelly JM. Cloning of an Fe-superoxide dismutase gene homologue from *Trypanosoma cruzi*. Mol Biochem Parasitol 1996;76:339–43.

[7] Ismail SO, Paramchuk W, Skeiky YAW, Reed SG, Bhatia A, Gedamu L. Molecular cloning and characterization of two iron superoxide dismutase cDNAs from *Trypanosoma cruzi*. Mol Biochem Parasitol 1997;86:187–97.

[8] Ismail SO, Skeiky YAW, Bhatia A, Omaraopyene LA, Gedamu L. Molecular-cloning, characterization, and expression in *Escherichia coli* of iron superoxide-dismutase cDNA from *Leishmania donovani chagasi*. Infect Immun 1994;62:657–64.

[9] Paramchuk WJ, Ismail SO, Bhatia A, Gedamu L. Cloning, characterization and overexpression of two iron superoxide dismutase cDNAs from *Leishmania chagasi*: role in pathogenesis. Mol Biochem Parasitol 1997;90:203–21.

[10] Chance B, Sies H, Boveris A. Hydroperoxide metabolism in mammalian organs. Physiol Rev 1979;59:527–605.

[11] Flohé L. The selenoprotein glutathione peroxidase. In: Dolphin D, Pulsion R, Avramovic O, editors. Glutathione, Chemical, Biochemical, and Medical aspects. New York: Wiley J and Sons, 1989:643–731.

[12] Docampo R. Sensitivity of parasites to free radical damage by antiparasitic drugs. Chem Biol Interact 1990;73:1–27.

[13] Leid RW, Suquet CM, Tanigoshi L. Oxygen detoxifying enzymes in parasites: a review. Acta Leiden 1989;57:107–14.

[14] Fairlamb AH, Cerami A. Metabolism and functions of trypanothione in the kinetoplastida. Annu Rev Microbiol 1992;46:695–729.
[15] Fairlamb AH, Blackburn P, Ulrich P, Chait BT, Cerami A. Trypanothione: a novel bis(glutathionyl)spermidine cofactor for glutathione reductase in trypanosomatids. Science 1985;227:1485–7.

[16] Shames SL, Fairlamb AH, Cerami A, Walsh CT. Purification and characterization of trypanothione reductase from *Crithidia fasciculata*, a newly discovered member of the family of disulphide-containing flavoprotein reductases. Biochemistry 1986;25:3519–26.

[17] Penketh PG, Klein RA. Hydrogen peroxide metabolism in *Trypanosoma brucei*. Mol Biochem Parasitol 1986;20:111–21.

[18] Henderson GB, Fairlamb AH, Cerami A. Trypanothione-dependent peroxide metabolism in *Crithidia fasciculata* and *Trypanosoma brucei*. Mol Biochem Parasitol 1987;24:39–45.

[19] Penketh PG, Kennedy WPK, Patton CL, Sartorelli AC. Trypanosomatid hydrogen peroxidase metabolism. FEBS lett 1987;221:427–31.

[20] Carnieri EGS, Moreno SNJ, Docampo R. Trypanothione-dependent peroxide metabolism in *Trypanosoma cruzi* different stages. Mol Biochem Parasitol 1993;61:79–86.

[21] Steenkamp DJ, Spies HSC. Identification of a major low-molecular-mass thiol of the trypanosomatid *Crithidia fasciculata* as ovothiol A: facile isolation and structural analysis of the bimane derivative. Eur J Biochem 1994;223:43–50.

[22] Spies HSC, Steenkamp DJ. Thiols of intracellular pathogens: identification of ovothiol A in *Leishmania donovani* and structural analysis of a novel thiol from *Mycobacterium bovis*. Eur J Biochem 1994;224:203–13.

[23] Nogoceke E, Gommel DU, Kiess M, Kalisz HM, Flohé L. A unique cascade of oxidoreductases catalyses trypanothione-mediated peroxide metabolism in *Crithidia fasciculata*. Biol Chem 1997;378:827–36.

[24] Loferer H, Hennecke H. Protein disulphide oxidoreductases in bacteria. TIBS 1994;19:169–71.

[25] Gommel DU, Nogoceke E, Morr M, Kiess M, Kalisz HM, Flohé L. Catalytic characteristics of tryparedoxin. Eur J Biochem 1997;248:913–8.

[26] Chae HZ, Robinson K, Poole LB, Church G, Storz G, Rhee SG. Cloning and sequencing of thiol-specific antioxidant from mammalian brain: alkyl hydroperoxide reductase and thiol-specific antioxidant define a large family of antioxidant enzymes. Proc Natl Acad Sci USA 1994;91:7017–21.

[27] McGonigle S, Dalton JP, James ER. Peroxidoxins: a new antioxidant family. Parasitol Today 1998;14:139–45.

[28] Montemartini M, Nogoceke E, Singh M, Steinert P, Flohé L, Kalisz HM. Sequence analysis of the tryparedoxin peroxidase gene from *Crithidia fasciculata* and its functional expression in *Escherichia coli*. J Biol Chem 1998;273:4864–71.

[29] Shim H, Fairlamb AH. Levels of polyamines, glutathione and glutathione-spermidine conjugates during

[30] Bernards A, Van der Ploeg LHT, Frasch CA, et al. Activation of trypanosome surface glycoprotein genes involves a duplication-transposition leading to an altered 3′ end. Cell 1981;27:497–505.

[31] Sanger F, Nicklen S, Coulson AR. DNA sequencing with chain terminating inhibitors. Proc Natl Acad Sci USA 1977;74:5463–7.

[32] El-Sayed NMA, Alarcon CM, Beck JC, Sheffield VC, Donelson JE. cDNA expressed sequence tags of *Trypanosoma brucei rhodesiense* provide new insights into the biology of the parasite. Mol Biochem Parasitol 1995;73:75–90.

[33] Holmgren A. Thioredoxin. Ann Rev Biochem 1985;54:237–71.

[34] Chae HZ, Kim IH, Kim K, Rhee SG. Cloning, sequencing, and mutation of thiol-specific antioxidant gene of *Saccharomyces cerevisiae*. J Biol Chem 1993;268:16815–21.

[35] Chae HZ, Chung SJ, Rhee SG. Thioredoxin-dependent peroxide reductase from yeast. J Biol Chem 1994;269:27670–8.

[36] Sahrawy M, Hecht V, Lopez Jaramillo J, Chuec AA, Chartier Y, Meyer Y. Intron position as an evolutionary marker of thioredoxins and thioredoxin domains. J Mol Evol 1996;42:422–31.

[37] Wilson R, Ainscough R, Anderson K, et al. 2.2 Mb of contiguous nucleotide sequence from chromosome III of *C. elegans*. Nature 1994;368:32–8.

[38] Kurooka H, Kato K, Minoguchi S, et al. Cloning and characterization of the nucleoredoxin gene that encodes a novel protein related to thioredoxin. Genomics 1997;39:331–9.

[39] Lundstrom J, Holmgren A. Protein disulfide isomerase is a substrate for thioredoxin reductase and has thioredoxin-like activity. J Biol Chem 1990;265:9114–20.

[40] Tagaya Y, Maeda Y, Mitsui A, et al. ATL derived factor (ADF), an IL2 receptor tac inducer homologous to thioredoxin—possible involvement of dithiol-reduction in IL2 receptor induction. EMBO J 1989;8:757–64.

[41] Matthews JR, Wakasugi N, Virelizier JL, Yodoi J, Hay RT. Thioredoxin regulates the DNA-binding activity of NF-κB by reduction of a disulfide bond involving cysteine 62. Nucleic Acids Res 1992;20:3821–30.

[42] Lye LF, Hsieh YH, Su KE, Lee ST. Cloning and functional analysis of the ribonucleotide reductase small subunit from hydroxyurea-resistant *Leishmania mexicana amazonensis*. Mol Biochem Parasitol 1997;90:353–8.

[43] Hofer A, Schmidt PP, Graslund A, Thelander L. Cloning and characterization of the R1 and R2 subunits of ribonucleotide reductase from *Trypanosoma brucei*. Proc Natl Acad Sci USA 1997;94:6959–64.

[44] Chae HZ, Uhm TB, Rhee SG. Dimerization of thiol-specific antioxidant and the essential role of cysteine 47. Proc Natl Acad Sci USA 1994;91:7022–6.

[45] Sen CK, Packer L. Antioxidant and redox regulation of  
     gene transcription. FASEB J 1996;10:709–20.  
[46] Kang SW, Chae HZ, Seo MS, Kim K, Baines IC, Rhee  
     SG. Mammalian peroxiredoxin isoforms can reduce hy-  
     drogen peroxide generated in response to growth factors  
     and tumor necrosis factor-α. J Biol Chem 1998;273:6297–  
     302.  

[47] Jin DY, Chae HZ, Rhee SG, Jeang KT. Regulatory role  
     for a novel human thioredoxin peroxidase in NF-κB  
     activation. J Biol Chem 1997;272:30952–61.  
[48] Zhang P, Liu B, Kang SW, Seo MS, Rhee SG, Obeid  
     LM. Thioredoxin peroxidase is a novel inhibitor of apop-  
     tosis with a mechanism distinct from that of Bcl-2. J Biol  
     Chem 1997;272:30615–8.
