
JOURNAL OF BACTERIOLOGY, Feb. 1999, p. 965–972
0021-9193/99/$04.00+0
Copyright © 1999, American Society for Microbiology. All Rights Reserved.

# Biochemical and Genetic Evidence for meta-Ring Cleavage of 2,4,5-Trihydroxytoluene in Burkholderia sp. Strain DNT

BILLY E. HAIGLER,† GLENN R. JOHNSON, WEN-CHEN SUEN,‡ AND JIM C. SPAIN*

Air Force Research Laboratory, Tyndall Air Force Base, Florida 32403-5323

Received 13 July 1998/Accepted 9 November 1998

2,4,5-Trihydroxytoluene (THT) oxygenase from *Burkholderia* sp. strain DNT catalyzes the conversion of THT to an unstable ring fission product. Biochemical and genetic studies of THT oxygenase were undertaken to elucidate the mechanism of the ring fission reaction. The THT oxygenase gene (*dntD*) was previously localized to the 1.2-kb DNA insert subcloned in the recombinant plasmid designated pJS76 (W. C. Suen and J. C. Spain, J. Bacteriol. 175:1831–1837, 1993). Analysis of the deduced amino acid sequence of DntD revealed the presence of the highly conserved residues characteristic of the catechol 2,3-dioxygenase gene family I. The deduced amino acid sequence of DntD corresponded to a molecular mass of 35 kDa. The native molecular masses for the THT oxygenase estimated by using gel filtration chromatography and nondenaturing gel electrophoresis were 67.4 and 77.8 kDa, respectively. The results suggested that the native protein consists of two identical subunits. The colorless protein contained 2 mol of iron per mol of protein. Stimulation of activity in the presence of ferrous iron and ascorbate suggested a requirement for ferrous iron in the active site. The properties of the enzyme are similar to those of the catechol 2,3-dioxygenases (*meta*-cleavage dioxygenases). In addition to THT, the enzyme exhibited activity towards 1,2,4-benzenetriol, catechol, 3- and 4-methylcatechol, and 3- and 4-chlorocatechol. The chemical analysis of the THT ring cleavage product showed that the product was 2,4-dihydroxy-5-methyl-6-oxo-2,4-hexadienoic acid, consistent with extradiol ring fission of THT.

---

Bacterial degradation of aromatic compounds under aerobic conditions involves the conversion of the initial substrates into intermediates with two or more hydroxyl groups on the aromatic ring. The intermediates are subject to oxidative attack by the ring cleavage dioxygenases. The ring cleavage enzymes are classified into two groups based on the site of ring fission (23). Enzymes catalyzing ring fission between two hydroxyl groups are designated intradiol (*ortho-*) cleavage enzymes. Enzymes that cleave at a bond proximal to one of the two adjacent hydroxyl groups are designated extradiol (*meta-*) cleavage enzymes. The primary structures of many ring cleavage dioxygenases have been determined, and comparison of their amino acid sequences has revealed that they can be divided further into distinct gene families. The *ortho*-cleavage enzymes are divided into three families designated catechol 1,2-dioxygenase I, catechol 1,2-dioxygenase II, and protocatechuate 3,4-dioxygenase (22). The *meta*-cleavage enzymes constitute at least three different groups, including the catechol 2,3-dioxygenase I, the protocatechuate 4,5-dioxygenase, and the homoprotocatechuate dioxygenase families (22).

The interest in bacterial metabolism of aromatic compounds has motivated a great amount of research on the biochemistry and molecular genetics of the *meta*-cleavage enzymes. Harayama et al. provided an early review on the molecular biology of the ring cleavage dioxygenases (22). Since the publication of that review, numerous studies characterizing other *meta*-ring cleavage enzymes show the broad range of catabolic pathways that use the extradiol dioxygenases. More recent reports have provided additional information and discussion on the phylogeny of the *meta*-ring cleavage enzymes (14, 36). The molecular genetic analysis can be coupled with the information gained from determinations of the three-dimensional structures for the 2,3-dihydroxybiphenyl dioxygenases from *Burkholderia cepacia* LB400 (21) and *Pseudomonas* sp. strain KKS102 (50) to provide a comprehensive overview of the relationships between the primary protein structure and the biochemistry of the *meta*-ring cleavage enzymes.

*Burkholderia* sp. strain DNT mineralizes 2,4-dinitrotoluene (2,4-DNT) via an oxidative pathway (Fig. 1) (52). The initial step in the pathway is the dioxygenation of the aromatic ring to form 4-methyl-5-nitrocatechol (MNC), with concomitant release of the first nitrite from the substrate. Subsequently, MNC monooxygenase catalyzes the oxidation of MNC to yield 2-hydroxy-5-methylquinone (HMQ) and nitrite. A quinone reductase then catalyzes the conversion of HMQ to 2,4,5-trihydroxytoluene (THT). THT is oxidized by a ring cleavage enzyme, THT oxygenase. The mode of ring cleavage has not been determined rigorously. Previous results suggested that THT oxygenase catalyzes an extradiol cleavage reaction; however, the preliminary evidence was not sufficient to distinguish among three possible mechanisms of ring fission (Fig. 1) (19).

The cloning and preliminary characterization of the genes encoding DNT dioxygenase (*dntA*), MNC monooxygenase (*dntB*), and THT oxygenase (*dntD*) from strain DNT have been described previously (55). Additional reports outlined analysis of the nucleotide sequences for *dntA* and *dntB* (20, 54). The present paper describes biochemical and genetic characterization of the THT oxygenase along with experiments to determine the mechanism of ring fission and identify the product of the reaction catalyzed by THT oxygenase.

(A preliminary report of this work has been presented previously [53].)

---

**MATERIALS AND METHODS**

Bacterial strains, plasmids, and culture conditions. *Burkholderia* sp. strain DNT was grown and maintained as previously described (19). *Escherichia coli*

966 HAIGLER ET AL. J. BACTERIOL.

![Chemical Reaction Diagram](attachment:chemical-reaction-diagram.png)

FIG. 1. Biodegradation of 2,4-DNT by *Burkholderia* sp. strain DNT (19). The shaded area indicates the ring cleavage step catalyzed by the THT oxygenase and the potential products of the reaction. The gene designations for the enzymes in the pathway are as follows: *dntA*, DNT dioxygenase; *dntB*, MNC monooxygenase; *dntC*, HMQ reductase; *dntD*, THT oxygenase.

JM109(DE3) (Promega Corp., Madison, Wis.) was used as the host for recombinant plasmids in this work. Plasmid pT7-5 was used as the expression vector (57). *E. coli* strains were grown at 37°C unless otherwise indicated. Recombinant *E. coli* strains were cultivated in complex medium containing 20 g of tryptone, 10 g of yeast extract, 5 g of NaCl, and 0.2% glycerol per liter of 50 mM KH₂PO₄ (pH 7.2). Ampicillin was added at a concentration of 50 μg/ml when required to select for the presence of plasmids.

**Enzyme assays.** Bacterial cell extracts of strain DNT and *E. coli*(pJS77) were prepared from frozen cells suspended in an equal volume of 0.05 M Tris-HCl (pH 7.5) containing 0.5 mg of DNase I. Cell lysis and ultracentrifugation to remove insoluble material were done as previously described (51).

THT oxygenase activity was determined by measuring the rate of oxygen consumption in the presence of THT as previously described (19). For routine purification and analysis, THT oxygenase activity was determined spectrophotometrically by measuring the conversion of 4-methylcatechol (4MC), a structural analog of THT, to the 2-hydroxy-(4- and/or 5-)methylmuconic semialdehyde (19). The assay mixture contained 4MC (10⁻⁴ M) and appropriate amounts of protein in 1 ml of KH₂PO₄ buffer (0.02 M, pH 7.0). The rate of hydroxymuconic semialdehyde accumulation was determined spectrophotometrically at 382 nm. The extinction coefficient of the hydroxymethylmuconic semialdehyde (ε₃₈₂ = 20,000 at pH 7.0) was determined spectrophotometrically after complete conversion of 4MC by THT oxygenase under standard assay conditions. The activity of THT oxygenase towards alternate substrates was initially demonstrated spectrophotometrically. Transformation rates for alternate substrates were also estimated by measuring oxygen consumption in the presence of the test substrate.

Pyruvate formation from THT by cell extracts of *Burkholderia* sp. strain DNT was estimated by a modification of the method of Friedemann and Haugen (16) and by measuring NADH oxidation in the presence of lactate dehydrogenase. Assays (3-ml mixtures) were performed in 0.02 M KH₂PO₄ buffer (pH 6.65) with 0.66 mg of cell extract obtained as previously described (20). Reactions were initiated by the addition of 5 × 10⁻⁵ to 10 × 10⁻⁵ M THT. After a 15- to 20-min incubation at room temperature, 1 ml of the reaction mixture was transferred to a cuvette and pyruvate was quantified by measuring NADH oxidation in the presence of lactate dehydrogenase. Another 1 ml of the assay mixture was transferred to a tube containing 10 μl of phosphoric acid and incubated for an additional 5 min, and then 0.333 ml of 2,4-dinitrophenylhydrazine reagent (16) was added. The precipitated protein was removed by centrifugation at 15,800 × g rpm, and the assay mixture was allowed to incubate for an additional 5 min. After the addition of 2.5 N NaOH (1.667 ml), the assay mixtures were incubated for an additional 10 min and the absorbance at 520 nm was determined.

**Protein purification.** All procedures were carried out at 4°C unless stated otherwise. A rapid detection method was used to assay THT oxygenase activity during chromatographic procedures. Samples (10 to 20 μl) of the column fractions were transferred to tubes containing 4-MC (0.1 mM) in 1 ml of KH₂PO₄ (20 mM, pH 7.0). The presence of the THT oxygenase was indicated by the appearance of the yellow hydroxymethylmuconic semialdehyde resulting from oxidation of 4MC.

High-level expression of *dntD* was accomplished by subcloning the gene into expression vector pT7-5 and then transferring the plasmid construct to *E. coli* JM109. Expression of *dntD* was induced by the addition of IPTG (isopropyl-β-d-thiogalactopyranoside) to the culture medium at 100 μM by standard methods (2).

For protein purification, cell extracts were prepared from 10 g (wet weight) of *E. coli*(pJS77). The volume of the extract was adjusted to three times the volume of the frozen cell suspension, and ammonium sulfate was added to a final concentration of 25% at 0°C. The solution was subjected to ultracentrifugation at 100,000 × g for 30 min to remove precipitated protein. The supernatant was loaded onto a phenyl-Sepharose CL-4B fast-flow column (5 by 5 cm) (Pharmacia Biotech Inc., Piscataway, N.J.) previously equilibrated with 25% ammonium sulfate in Tris-HCl buffer. Sample loading and elution were at a flow rate of 4 ml/min. The column was washed with 200 ml of Tris-HCl buffer containing 25% ammonium sulfate, and proteins were eluted with a decreasing linear ammonium sulfate gradient (25 to 0%) in Tris-HCl (400 ml). The fractions (6 ml each) that contained maximum THT oxygenase activity were combined and then loaded directly onto a Q-Sepharose fast-flow column (1.6 by 19.5 cm) (Pharmacia). Proteins were eluted with 120 ml of Tris-HCl buffer followed by a 240-ml linear KCl gradient (0 to 0.4 M). The fractions (4 ml each) that contained THT oxygenase activity were pooled and concentrated by using a YM5 filter (Amicon Inc., Beverly, Mass.) under nitrogen at 4°C to yield a final volume of 1 to 2 ml. The solution was loaded in two equal portions onto a Sephacryl S300 HR column (1.6 by 91 cm) (Pharmacia), and proteins were eluted with Tris-HCl buffer at 1

VOL. 181, 1999                                                                                       BIOCHEMISTRY AND GENETICS OF THT OXYGENASE 967


GAATTCTCCGAAATACGGTTGTTGATGAGGCCTGATTGGCCGGTTGATTATTGAAATTTGGTTCAAAAAATAGGAGATGTC CATGTCCGTTGCAGACATT


| 10 | 20 | 30 | 40 | 50 | 60 | 70 | 80 | 90 | 100 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| M | S | V | A | D | I | 6 |


110 120 130 140 150 160 170 180 190 200
GCTTATGTGGTTTTTAACCATGTTGATTGGCTGGAAGCTGAGCGCTTCTATACCGACTTTGGTCTC ACTGTGCTTACCGTAATGAGTTCGAGGTGGCCT
7 A Y V V F N H V D L L E A E R F Y T D F G L T V A Y R N E F E V A 40
210 220 230 240 250 260 270 280 290 300
TTCGACCTGCAATGGAAGCTATTGCTATGTTGCTCGTAAGGCAGATAAGCCCCGGCCTGGTGCTAT TGGGCTGAGCGCCGCGAGTCAAGAAGACCT
40 F R P A M E R S Y C Y V A R K A D K P G L V S I G L S A A S Q E D L 73
310 320 330 340 350 360 370 380 390 400
GGTGCTTGCTGCGCAGTTTCCC GAAAGCTCACCGATTC AAGCCATTGAGCGACCCGGCGGAGGACAGGTTGTACGTTTGACCAGTCAGAT GGTTGGAAT
73 V V A A Q'F P E A S P I Q A I E R P G G G Q V V R L T S P D G V N 106
410 420 430 440 450 460 470 480 490 500
TTCGAGTTGGTTCA CGGCATTGATGTGTATCCGGCGCGACCA GTACGAGATGCACTGAGCATCAATACAGGTATCGTAAAGACACGTCA CGGTCAATGGC
107 F E L V H G I D V Y P A R P V R D A L S I N T G I V K T R H G Q W 140
510 520 530 540 550 560 570 580 590 600
AGCGTCCGGAAC TTGAGCCTGCGAGCATT TGGCTTAGGCCATGTGGCATTGGCTGACGGGTGACTTCAAGACCAATGCGCAATGGTTGCGGTCACGCTT
140 Q R P E L E P A S I L R L G H V A L L T G D F K T N A Q W L R S R F 173
610 620 630 640 650 660 670 680 690 700
TGGTTTTATGCCTTCGGACGTGCTATTTGCGCAGCACCCGAAAACCAAGTGGGTGCCTTCTCCAT TGCACGGGGGGGAATCCATGGGACC GATCACCAT
173 G F M P S D V L F A D A P E N Q V G A F F H C T G G N P W T D H H 206
710 720 730 740 750 760 770 780 790 800
ACCCTGGCGCTCTTCCCCAGCGATGGTTCCC GCA TGCATATTGGCTT CGAAATACAGGATCTTGACGCACAGTTCTGGGTAACAAGTGGTGGCAT
207 T L A L F P S D G S R M H H C S F E I Q D L D A Q F L G N K W L A 240
810 820 830 840 850 860 870 880 890 900
CGCGCGGTTG GAAACCTGGTTGGGGTGGGAGCCCATATTTTGGTAGCCAGATTTTCGATTATTGGTTTGATCCC GCTGGGAAACCTGGTTGAACACTT
240 S R G W K P G W G V G R H I L G S Q I F D Y W F D P A G N L V E H F 273
910 920 930 940 950 960 970 980 990 1000
TACCGACGGCGATCTTATCCAAGCAGGTGCTGCCCGGGATGCCCCCTCTGGATGACGTGCTTTGCC CATGTGGGGGCCACC ACTTACTGCATCCAAA
273 T D G D L I Q A G A A P G M H P L D D A L A M W G P P L T A S K 306
1010 1020 1030 1040 1050 1060 1070 1080 1090 1100
TTCGTTGCCATGGCGGCCAGGCACA ACTGAACCCAGTAGCAGGGTCAAAGTTTCTTTTTGAACCATTGCGAGTGC GCAATTCACGGAATATCTTCA TCA
307 F V A M A A R H N
1110 1120 1130 1140 1150 1160 1170 1180 1190 1200
TGTCAGATCAAAAAATCTCTACC GTA CCCACAGTCAAAC TGTTGATCAA CGGTGCAITTCGTTGAGTCGAAAACCACCCAGTGGCGCAATGTGATCAATCC
1210 1220
TGCGACACAGATGTCCTGCGCTCGAG


FIG. 2. Nucleotide sequence of the 1.2-kb DNA insert of pJS76 and deduced amino acid sequence of DntD. The putative promoter regions are underlined twice. The Shine-Dalgarno sequence is underlined.

---

ml/min. Fractions (2 ml each) containing THT oxygenase activity were pooled and loaded onto a DEAE-Sephrose fast-flow column (1.6 by 10 cm) (Pharmacia). Proteins were eluted with a 40-ml buffer wash followed by a continuous gradient of KCl (0 to 0.4 M) in Tris buffer at 2 ml/min. The fractions (1 ml each) that contained maximum activity were combined, and the KCl was removed by dialysis prior to storage at −70°C.

Isolation and identification of THT reaction products. Purified THT oxygenase (280 μg) was incubated with THT (added in four equal portions at 2-min intervals to give a final concentration of 0.2 mM) in 1 ml of deionized water. Two minutes after the final addition of THT, 100 μl of the assay mixture was transferred to a tube containing 100 μl of acetonitrile. The solution was then analyzed by liquid chromatography (LC)-mass spectrometry (MS) (chemical ionization). Products formed from 1,2,4-benzenetriol and 4MC were analyzed similarly, except that higher concentrations of THT oxygenase (560 and 1,050 μg/ml, respectively) were used in the reactions.

Analytical methods. Protein determinations, UV-visible spectral analyses, high-pressure LC (HPLC), sodium dodecyl sulfate-polyacrylamide gel electrophoresis, and gel filtration molecular weight determinations were done by standard methods (2) and as described previously (20, 54). Gel filtration was done by both standard LC and HPLC. An S-200 HR column (1.6 by 88 cm) (Pharmacia) with a mobile phase consisting of 0.1 M NaCl and 0.05 M Tris-HCl (pH 7.5 at 4°C) was used for LC. HPLC was done by using a GF-250 gel filtration column (9.4 mm by 25 cm) (Dupont Instruments, Wilmington, Del.) with a mobile phase of 0.3 M Na₂H₂PO₄ (pH 7.0) at 0.5 ml/min. Nondenaturing gel electrophoresis was done in slab gels by the procedure described by the reagent manufacturer (Sigma, St. Louis, Mo.). The iron content of THT oxygenase was determined with o-phenanthroline and Ferene S as described previously (18, 41).

DNA sequencing was done at the genetic engineering core facility at the University of Illinois-Champaign, Urbana. Sequence similarity searches were done with the BLAST programs (1) via the National Center for Biotechnology Information web site. Additional sequence analysis was carried out with the CLUSTAL W software package (61). Multiple-alignment comparisons of amino acid sequences were done by the method of Feng and Doolittle (15), and phylogenetic trees were inferred from the alignments by using the neighbor-joining method of Saitou and Nei (47). Graphics for phylogenetic trees were produced by using the TreeView program designed by Page (45).

Analysis of ring cleavage products was done with a Hewlett-Packard (HP) model 1050 liquid chromatograph coupled with an HP model 5987 mass spectrometer. Reaction products were separated with a Spherisorb C₈ column (Altech Bioscience, Deerfield, Ill.) with an acetonitrile-water (70:30) mobile phase at a flow rate of 0.7 ml/min. The LC eluent was passed through an HP model 59980A particle beam interface prior to introduction into the mass spectrometer.

Negative chemical ionization was done by using methane as a carrier gas. Nuclear magnetic resonance (NMR) spectra were obtained with a Bruker AC-300 Fourier transform NMR spectrometer by Stephen A. Aubert (Eglin Air Force Base, Fla.) and with a Bruker WM-360 spectrometer by Sol M. Resnick (University of Iowa, Iowa City).

Chemicals. DNase I, o-phenanthroline, ferrene S, 4-MC, catechol, and 1,2,4-benzenetriol were purchased from Sigma Chemical Co. THT was kindly provided by R. Spanggord (SRI International, Menlo Park, Calif.). 3-Methylmaleyl-acetate was derived from 3-methyl-cis-dienelactone as previously described (46). 3-Methyl-cis-dienelactone and 3-methyl-trans-dienelactone were generous gifts from D. Pieper (Gesellschaft fur Biotechnologische Forschung, Braunschweig, Germany).

Nucleotide sequence accession number. The GenBank accession number of *dntD* from strain DNT is AF076848.

---

**RESULTS**

Nucleotide sequence of *dntD*. Analysis of the nucleotide sequence of the 1.2-kb DNA insert from pJS75 (55) revealed a 945-bp open reading frame preceded by a ribosome binding site (AGGAG, nucleotides 72 to 76) (Fig. 2). The open reading frame (*dntD*) encoded a 35-kDa polypeptide composed of 315 amino acids, with a calculated isoelectric point of 5.53. Analysis of the upstream sequence and comparisons with upstream regions from related genes revealed a potential promoter region at positions 22 to 51. The region resembled the canonical sequence motif (TTGACA-17±1-TATAAT) required for recognition by σ⁷⁰ RNA polymerase holoenzyme (12).

Comparisons of the deduced amino acid sequence of DntD to sequences stored in the available databases revealed that DntD was similar to previously described extradiol ring cleavage dioxygenases (Fig. 3) used in pathways for the degradation of many aromatic compounds. The most closely related gene products had only 40% identity with DntD. The multiple alignment of THT oxygenase and other ring cleavage enzymes summarizes the sequence similarity among the class I dioxygenases.

968 HAIGLER ET AL. J. BACTERIOL.

FIG. 3. Phylogenetic tree showing the relationships among THT oxygenase and other extradiol ring cleavage enzymes. The scale at the lower right corresponds to a genetic distance of 0.1 substitution/position (10% difference). References for the peptide sequences are as follows: CatA from *Rhodococcus rhodochrous*, 34; BphC from strain BN6, 27; CmtC from *P. putida*, 13; BphC from *P. stutzeri*, 59; BphC from *P. mendocina*, 59; BphCII from strain BN6, 26; MpcII from *Ralstonia eutropha*, 29; CdoA from *R. rhodochrous*, 5; PheB from *Bacillus stearothermophilus*, 25; TdnC from *P. putida*, 39; C23OII from TOL plasmid, 30; XylE from *Sphingomonas yanoikuyae*, 31; CmpE 63; NahH from PpG7, 17; BphE, 6; XylE<sub>J1104-1</sub> from *P. aeruginosa* JI104, 32; XylE<sub>J1104-2</sub> from *P. aeruginosa* JI104, 32; XylE<sub>TOL</sub> from *P. aeruginosa* JI104, 33; C23O from *P. putida* KF715, 38; XylE from TOL plasmid, 42; XylE from *Bacillus subtilis*, 43; XylEII from TOL plasmid, 4; PhlH from *P. putida* H, 28; C23O from *P. putida* CF600, 3; and AtdA, 60.

(Fig. 4). Like the other *meta*-ring cleavage enzymes, the THT oxygenase peptide retained the residues which form the active site (His-154, His-220, and Glu-271) and coordinate the iron cofactors (His-206, His-252, and Tyr-261) (14). In their previous study of extradiol dioxygenases, Eltis and Bolin (14) showed the strict conservation of glycine, leucine, and proline residues which were located along the interface of the N- and C-terminal domains of the protein. For DntD, as well as the most recently described BphC proteins (59), only the Gly-27

and Pro-265 residues are conserved; at position 173, phenylalanine was substituted for the leucine residue.

Construction of an *E. coli* strain for overexpression of THT oxygenase. To facilitate purification of the THT oxygenase, the 1.2-kb EcoRI-XhoI restriction fragment from pJS76 (55) containing *dntD* was subcloned into the expression vector pT7-5 to yield pJS77. Subsequently, the plasmid was transferred to *E. coli* JM109(DE3). The initial induction trials carried out at 37°C led to formation of inclusion bodies that could not be

DntD - - - - - - - - - - - - - - - - - - - - MSVADIAYVVFNHVDLLLEAER-FYTD FG LTVAYRNEFEVAFRPAM--ERSYCYVARKA 56
BphC P.S.MAKQTATAILATPQPARHPQPTVRAQALAHLI FERPDL EQADR-FLGDFGLQTAQR--DAATLYMRGTGSAPYCYRIHH- 77
BphC P.m.-- - - - - - - - - - - - - - - MVKAQALAHLIFERPDLDEAER-FLVDFGLQLAHA--KRTVCTCAAAPARLTALRVQR- 56
CdoA - - - - - - - - - - - - - - - - - - - - MSNITSDSNIVEVSVPRVHNLHHVELLTPKNESLDDFFTRVL GLHETHR-EGQSVYLRGSGEWGQYSTILTES 73
CatA - - - - - - - - - - - - - - - - - - - - MALVTGIGYIGIGVSDLPAWEE-FAETI G FQIRERGEDGTLYL RMD--KAHHRVAVHPT 57
OrpE - - - - - - - - - - - - - - - - - - - - MALTGVIRPGYVQLRVLDDLDEAI I HYRDRIG LNFVNR-EGDRAFFQA FDEFDRHSIILREA 61
DmpB - - - - - - - - - - - - - - - - - - - - MKKGVMRPGHVQLRVLNLESALAHYR DLL GLIEMDRDEQGRVYLKAWTEVDKFSVVLREA 61
PheB - - - - - - - - - - - - - - - - - - - - MSKNFQEPIFDVAQLAHVELLSPKLEESIVFFT K YLG MEVTAR-AGNSVYLRAYEDFYHNTLKITES 67
XylE - - - - - - - - - - - - - - - - - - - - MNKGVMRPGHVQLRVLDMSKALEHYVELL GLIEMDRDDQGRVYLKAWTEVDK FSLVLREA 61
At dA - - - - - - - - - - - - - - - - - - - - MKKGVMRPGHVQIRVLDMDEAVKHYKDLL GLIEMDRDDQGRVYLKAWTEVDKFSVVLREA 61

DntD DKPGLVSIG LSAA SQE-- - - - DLVVAAQFPEASPIQ--AIERPGGGQVVRLTS PB DG VNF ELVHGIDVYPARPVRDAL SIN 129
BphC P.S.-AKRSRFLG FGLALAS-RDDLERIAALPG--ASGIE--PLEGGGGERVRLTD PS GFLL EA IHGQQAAEAMAHRAPLPLN 151
BphC P.m.-ASRARFVG FGLTLAS-RADLQPLTALPG--ASPIE--TLHTPGGECVQLTD PSGFRF EAVFGQQSVAALPHRPQLPVN 130
CdoA PTAGLGHMG WQVAEPEHVQGWGRRRLKDAGVEHRFEA--GGSSFAQGDTVSFEGP YGHKM ELFYDERFVPEDRSKL LSQP 151
CatA GEDDLTYVG WQVADENGFD ELERTLRAAGVPVEMAGEDDAELRGVARLMRFED PSGIKS EAYGLVSEPE---VPYV 131
OrpE DQAGMDVMG FKVAKDADLDHFTERLLDIGHV DVIP--AGEDPGVGRKIRFNT PT QHVFE LYAEIMALSATGP AVKNPD VW 139
DmpB DQPGMDFMG FKVIDEDCLNRLTQDLLNYGCLIETIP--AGELKGCGRRVRFQT PSGHFF ELYADKEYTGKWGLEEINPEA 139
PheB AEAGLGHVG WRASSPQALER RVLELEKSG--LGRGW--IDGDIHGKAYQFTT BDGHQM EIFFEVEYYKP OEQTKLLN 143
XylE DEPGMDFMG FKVVDEDALRQLERDLMAYGCAVEQLP--AGELNSCGRRVRFQA PSGHHF ELYADKEYTGKWGLNDVNPEA 139
At dA DEPGMDFMG FKCLS EEVVDQLRGE LA AFGCEIEDVP--EDELKDCGRRVRFIA PT GH SFE IYATKKQTGKWGVGNHNPEA 139

DntD TGI VKTRHGQWRPELEPASILRLGH VALLTGD FKT N-- - - - - - AQWLRSRFGFMPSDVLFADAPE-NQVGAFFHCTGG 200
BphC P.S.LADETPRINATQRPPIA PPEVIKLGH VVLEVADYQAT-- - - - - - C AWYTHHFGFI PSDVQVLPDG--SPA VTFMRLLDLG 221
BphC P.m.RVDELPRFNATQRPPVA APEIIKLGH VVMEVADYQAT-- - - - - - C AWYTGHFGFI PSDVQVLPDG--SPA VA FMRLVLG 200
CdoA VKFPTPGIGAR-- - - - - - - - - - - - - - - RLDHLNITAKDVD DA-- - - - - - - RNWSDVLGFKLREALRTPDG--DLGAWMSVSSQ 209
CatA SPYAVDFVTEDQ-- - - - - - - - - - - - - - - GFGHI VVMVDDYDET-- - - - - - - MRFYREVLGLQTSDLVKVGAGGVQTRMAFMRCNPR 193
OrpE VVEPRGM RAT-- - - - - - - - - - - - - - - RFDH CALNGVDIA SS-- - - - - - - AKIFVDALDFSVAEELVDE TSG-ARLGI FLSCSNK 198
DmpB WPRNLKGMRAV-- - - - - - - - - - - - - - - RFDH CLLYGDELQAT-- - - - - - - YALFTEVLGFYLAEQV IDDDG--TRVAQFLSLS TK 198
PheB RPSKRPAQGVPVR-- - - - - - - - - - - - - - - RLDH INLMTSNPGV D-- - - - - - - TQFMIDTLGFR LREQIRD KGK--ILGSWISVS NL 203
XylE WPRDLKGMAAV-- - - - - - - - - - - - - - - RFDH ALMYGDELPAT-- - - - - - - YDLFTKV LGFYLAEQVL DENG--TRVAQFLSLS TK 198
At dA WPRGLEGMKAT-- - - - - - - - - - - - - - - RFDH CLLYGPNLDET-- - - - - - - LNLF RDVLGFDLAEQVMAPDG--KRVSQFLT VS TK 198

DntD NPWTDH H TLA ALFPSDG-- - - - - - - - - - - - - - - SR-MH HC SFEIQDLDAQFLG NKWLASRGWKPGWG VGR-H ILGSQIFDYWFDPAGNLVEHF 274
BphC P.S.DTPTDH H TLA AIAQGFM-- - - AA-YSH SAYEVVDADA VGMGQRVLR EKGWKHAWGIGR-H ILGSQIFDYWQDPWGDKHEHY 295
BphC P.m.DTPADH H TLA AIAQGFM-- - - AT-YSH SAYEVVDADA VGMGQRVLRQ NGWRHAWGIGR-H ILGSQIFDYWQDPWGDKHEHY 274
CdoA -- - - - V H DMA IMRDGME--EGRLH HIAYWLETPETLLRAVDTFVEEGTPIDVGP GK-H GLTQA FYVYVFEPGGNRVELF 281
CatA -- - - - Q H SLA FWAGDS--TTRLN HF MLQTQTLDMQTLDRCF-HGGIPATNL GR-H VNDYAVSFYITTPSGFMIEYG 262
OrpE -- - - - A HDVA FLGYPE--DGKIH HTSFNLESHDVGAADIISRYDISLDIGPTR-H GITRG QT IYFFDPSGNRNETF 268
DmpB -- - - - A HDVA FIHCPE--KGKFH HV SFFLETWEDVLRAADLISM TDTSIDIGPTR-H GLTHG KT IYFFDPSGNRN EVF 268
PheB -- - - - V HE IA FMQEPNQE--KGKLH HLCYWYGIPQNLYDLADLKDHEYFIEVPPNK-H GISQAFCMYVYEPGGNRIELF 275
XylE -- - - - A HDVA FIHHE--KGRLH HVSFHLETWEDLLRAADLISM TDTSIDIGPTR-H GLTHG KT IYFFDPSGNRN EVF 268
At dA -- - - - A HDIA FIHHEE--PGKFH HASFFLETWQDVLKAADLLSMTDTSIDIGPTR-H GLTHG QT IYFFDPSGNRN EVF 268

DntD TD-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
TABLE 2. Substrate specificity of THT oxygenase

| Substrate           | Relative activity$^a$ (%) | $\lambda_{\text{max}}$ of product |
|---------------------|--------------------------|----------------------------------|
| THT                 | 100                      | 280, 320–380                     |
| 1,2,4-Benzenetriol  | 17                       | 280, 320–380                     |
| Catechol            | 1                        | 375                              |
| 3-MC                | 0.5                      | 318, 340–420                     |
| 4-MC                | 3                        | 325, 382                         |
| 3-Chlorocatechol    | 0.7                      | 367                              |
| 4-Chlorocatechol    | 0.8                      | 377                              |
| 4,5-Dichlorocatechol| ND$^b$                   |                                  |
| 4-Nitrocatechol     | ND                       |                                  |
| MNC                 | ND                       |                                  |
| Protocatechuate     | ND                       |                                  |

$a$ Oxygen consumption in the presence of $5 \times 10^{-5} \mathrm{M}$ substrate was measured, with the activity for THT ($8.633 \mu \mathrm{mol}$ of oxygen consumed/min/mg of protein) taken as $100 \%$ activity.

$b$ ND, none detected.

The substrate range of the THT oxygenase was determined by measuring the oxygen consumption stimulated by various aromatic compounds (Table 2). THT oxygenase exhibited low activity with 1,2,4-benzenetriol and 4-MC. Traces of activity toward catechol, 3-MC, and 3- and 4-chlorocatechol were also detected. The spectra and calculated extinction coefficients of the yellow products formed from 1,2,4-benzenetriol, catechol, and 4-MC were consistent with those of the respective hydroxymuconic semialdehydes. No metabolites that corresponded to intradiol cleavage products were detected.

Identification of the THT reaction product. Preliminary attempts to identify the product of the THT reaction provided misleading results (19). The compound obtained by extraction of the reaction medium with ethyl acetate provided an apparent molecular ion of 154. The proton NMR spectrum of the extracted product gave three sets of peaks at 1.81, 6.81, and 8.23 ppm in a ratio of 3:1:1. The ${ }^{13} \mathrm{C}$ NMR yielded seven resonances at 10.35, 116.6, 126.0, 152.8, 153.8, 161, and 178.5 ppm and indicated that the chemical shifts of the carbonyl resonance are in the general region for acids, esters, and lactones (45a). The chemical shifts are consistent with those of a 3-methylidenelactone; however, the spectra were different from those reported by Prucha et al. for 3-methyl-trans-dienelactone and 3-methyl-cis-dienelactone (46). The extracted compound was not transformed by cell extracts or intact cells of strain DNT. The lack of further transformation suggested that the compound was not an intermediate of the 2,4-DNT pathway but an artifact of the extraction procedure.

A second approach was used to identify the products formed by the ring fission reaction. THT oxygenase was incubated with THT, as well as the structural analogs 1,2,4-benzenetriol and 4-MC. Following incubation, the reaction mixtures were analyzed directly by using LC-MS. The analyses revealed molecular ion peaks at $m / z$ 172 (THT), 158 (1,2,4-benzenetriol), and 156 (4MC). The mass-to-charge ratios of the molecular ion peaks corresponded to the respective hydroxymuconic semialdehydes identified in the spectrophotometric analyses of reaction products. Authentic 3-methylmaleylacetate, which would be produced by intradiol ring fission of THT, gave a molecular ion peak at $m / z$ 128 (172 – 44 due to decarboxylation during chemical ionization) (46). Cell extracts from strain DNT transformed the ring cleavage product to pyruvate with 1:1 stoichiometry. The subsequent transformation of the 2,4-dihydroxy-5-methyl-6-oxo-2,4-hexadienoic acid indicated that the compound was an intermediate of the DNT pathway and not an experimental artifact like 3-methyldienelactone.

identity (11, 64). The deduced amino-terminal sequence of THT oxygenase had no significant identity to any of the benzenetriol oxygenases (<15%). In addition, THT oxygenase did not catalyze the conversion of 1,2,4-benzenetriol to maleylacetate. Thus, the overlap of substrates for the benzenetriol and THT oxygenases was not reflected in the amino acid sequences of the enzymes.

Our results indicate that THT oxygenase cleaves THT by a mechanism analogous to the extradiol cleavage of 1,2,4-benzenetriol and 2,3,5-trihydroxytoluene by the 2,3,5-trihydroxytoluene 1,2-dioxygenase present in *Pseudomonas putida* ORC and *P. putida* O1OC (8, 9). Because of the similarities in the reactions catalyzed by THT oxygenase and 2,3,5-trihydroxytoluene 1,2-oxygenase, it would be interesting to compare the amino acid sequences of these two enzymes. However, the sequence for 2,3,5-trihydroxytoluene 1,2-oxygenase is not available.

In previous work, gas chromatography-MS analysis of the THT ring cleavage provided a compound that corresponded to the lactonized 2,4-dihydroxy-5-methyl-6-oxo-2,4-hexadienoic acid (19). However, at that time, the possibility that the isolated product was a 3-methyldienelactone could not be ruled out. The 3-methyldienelactone could result from acid-catalyzed dehydration of 3-hydroxy-4-methylmuconate, the product resulting from intradiol cleavage of THT. Recent evidence indicates that the 3-hydroxy-4-methylmuconate does not undergo dehydration but instead undergoes tautomerization to form 4-hydroxy-3-methylmuconolactone and/or 3-methylmaleylacetate (46). The mass spectra of the 3-methyl-*cis*- and the 3-methyl-*trans*-dienelactones (46) are different from that of the product formed from the THT ring fission product after acid extraction (19). In addition, the compound formed from THT prior to acid extraction is clearly not 3-methylmaleylacetate (46). The mass spectral results, coupled with the stoichiometric production of pyruvate from THT in reactions with cell extracts from strain DNT, provide strong evidence that ring fission occurs between positions 5 and 6 of the aromatic ring to yield 2,4-dihydroxy-5-methyl-6-oxo-2,4-hexadienoic acid (Fig. 1). We are currently working to determine the subsequent metabolism of the ring cleavage product.

### ACKNOWLEDGMENTS

We thank R. J. Spanggord for providing THT, D. Pieper for kind gifts of 3-methyl-*cis*- and 3-methyl-*trans*-dienelactones, Mike Henley for gas chromatography and LC-MS spectral determinations, Stephen A. Aubert and Sol Resnick for NMR spectral determinations, and G. J. Zylstra for helpful discussions.

This research was supported in part by an appointment to the Research Participation Program at the Air Force Engineering and Service Laboratory administered by Oak Ridge Associated Universities through an interagency agreement between the U.S. Department of Energy and Tyndall Air Force Base. Funding was provided by the Air Force Office of Scientific Research and the Strategic Environmental Defense Program.

### REFERENCES

1. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, and D. J. Lipman. 1990. Basic local alignment search tool. J. Mol. Biol. **215**:403–410.
2. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A. Smith, and K. Struhl (ed.). 1998. Current protocols in molecular biology. John Wiley and Sons, Inc., New York, N.Y.
3. Bartilson, M., and V. Shingler. 1989. Nucleotide sequence and expression of the catechol 2,3-dioxygenase-encoding gene of phenol-catabolizing *Pseudomonas* CF600. Gene **85**:233–238.
4. Benjamin, R. C., J. A. Voss, and D. A. Kunz. 1991. Nucleotide sequence of *xylE* from the TOL pDK1 plasmid and structural comparison with isofunctional catechol-2,3-dioxygenase genes from TOL, pWW0, and NAH7. J. Bacteriol. **173**:2724–2728.
5. Candidus, S., K. H. van Pee, and F. Lingens. 1994. The catechol 2,3-dioxygenase gene of *Rhodococcus rhodochrous* CTM: nucleotide sequence, comparison with isofunctional dioxygenases and evidence for an active-site histidine. Microbiology **140**:321–330.
6. Carrington, B., A. Lowe, L. E. Shaw, and P. A. Williams. 1994. The lower pathway operon for benzoate catabolism in biphenyl-utilizing *Pseudomonas* sp. strain IC and the nucleotide sequence of the *bphE* gene for catechol 2,3-dioxygenase. Microbiology **140**:499–508.
7. Chapman, P. J. 1976. Microbial degradation of halogenated compounds. Biochem. Soc. Trans. **4**:463–476.
8. Chapman, P. J., and D. W. Ribbons. 1976. Metabolism of resorcinyl compounds by bacteria: alternative pathways for resorcinol catabolism in *Pseudomonas putida*. J. Bacteriol. **125**:985–998.
9. Chapman, P. J., and D. W. Ribbons. 1976. Metabolism of resorcinyl compounds by bacteria: orcinol pathway in *Pseudomonas putida*. J. Bacteriol. **125**:975–984.
10. Daubaras, D. L., C. E. Danganan, H. Anette, R. W. Ye, W. Hendrickson, and A. M. Chakrabarty. 1996. Biodegradation of 2,4,5-trichlorophenoxycetic acid by *Burkholderia cepacia* strain AC1100: evolutionary insight. Gene **179**:1–8.
11. Daubaras, D. L., K. Saito, and A. M. Chakrabarty. 1996. Purification of hydroxyquinol 1,2-dioxygenase and maleylacetate reductase: the lower pathway of 2,4,5-trichlorophenoxycetic acid metabolism by *Burkholderia cepacia* AC1100. Appl. Environ. Microbiol. **62**:4276–4279.
12. Deretic, V., W. M. Konyecsni, C. D. Mohr, D. W. Martin, and N. S. Hibler. 1989. Common denominators of promoter control in *Pseudomonas* and other bacteria. Bio/Technology **7**:1249–1254.
13. Eaton, R. W. 1996. *p*-Cumate catabolic pathway in *Pseudomonas putida* F1: cloning and characterization of DNA carrying the *cmt* operon. J. Bacteriol. **178**:1351–1362.
14. Eltis, L. D., and J. T. Bolin. 1996. Evolutionary relationships among extradiol dioxygenases. J. Bacteriol. **178**:5930–5937.
15. Feng, D.-F., and R. F. Doolittle. 1987. Progressive sequence alignment as a prerequisite to correct phylogenetic trees. J. Mol. Evol. **25**:351–361.
16. Friedemann, T. E., and G. E. Haugen. 1942. Pyruvic acid. II. The determination of keto acids in blood and urine. J. Biol. Chem. **147**:415–442.
17. Ghosal, D., I. S. You, and I. C. Gunsalus. 1987. Nucleotide sequence and expression of gene *nahH* of plasmid NAH7 and homology with gene *xylE* of TOL pWWO. Gene **55**:19–28.
18. Haigler, B. E., and D. T. Gibson. 1990. Purification and properties of NADH-ferredoxin_NAP_ reductase, a component of naphthalene dioxygenase from *Pseudomonas* sp. strain NCIB 9816. J. Bacteriol. **172**:457–464.
19. Haigler, B. E., S. F. Nishino, and J. C. Spain. 1994. Biodegradation of 4-methyl-5-nitrocatechol by *Pseudomonas* sp. strain DNT. J. Bacteriol. **176**:3433–3437.
20. Haigler, B. H., W.-C. Suen, and J. C. Spain. 1996. Purification and sequence analysis of 4-methyl-5-nitrocatechol oxygenase from *Burkholderia* sp. strain DNT. J. Bacteriol. **178**:6019–6024.
21. Han, S., L. D. Eltis, K. N. Timmis, S. W. Muchmore, and J. T. Bolin. 1995. Crystal structure of the biphenyl-cleaving extradiol dioxygenase from a PCB-degrading pseudomonad. Science **270**:976–980.
22. Harayama, S., M. Kok, and E. L. Neidle. 1992. Functional and evolutionary relationships among diverse oxygenases. Annu. Rev. Microbiol. **46**:565–601.
23. Harayama, S., and M. Rekik. 1989. Bacterial aromatic ring-cleavage enzymes are classified into two different gene families. J. Biol. Chem. **264**:15328–15333.
24. Harayama, S., M. Rekik, A. Wasserfallen, and A. Bairoch. 1987. Evolutionary relationships between catabolic pathways for aromatics: conservation of gene order and nucleotide sequences of catechol oxidation genes of pWW0 and NAH7 plasmids. Mol. Gen. Genet. **210**:241–247.
25. He, Z. Q., Y. M. Mao, Z. J. Sheng, and R. Q. Shen. Swiss protein database, accession no. P31003.
26. Heiss, G., C. Muller, J. Altenbuchner, and A. Stolz. 1997. Analysis of a new dimeric extradiol dioxygenase from a naphthalenesulfonate-degrading sphingomonad. Microbiology **143**:1691–1699.
27. Heiss, G., A. Stolz, A. E. Kuhm, C. Muller, J. Klein, J. Altenbuchner, and H. J. Knackmuss. 1995. Characterization of a 2,3-dihydroxybiphenyl dioxygenase from the naphthalenesulfonate-degrading bacterium strain BN6. J. Bacteriol. **177**:5865–5871.
28. Herrmann, H., C. Muller, I. Schmidt, J. Mahnke, L. Petruschka, and K. Hahnke. 1995. Localization and organization of phenol degradation genes of *Pseudomonas putida* strain H. Mol. Gen. Genet. **247**:240–246.
29. Kabisch, M., and P. Fortnagel. 1990. Nucleotide sequence of the metapyrocatechase II (catechol 2,3-oxygenase II) gene *mpcII* from *Alcaligenes eutrophus* JMP 222. Nucleic Acids Res. **18**:5543.
30. Keil, H., M. R. Lebens, and P. A. Williams. 1985. TOL plasmid pWW15 contains two nonhomologous, independently regulated catechol 2,3-dioxygenase genes. J. Bacteriol. **163**:248–255.
31. Kim, E., and G. J. Zylstra. 1995. Molecular and biochemical characterization of two *meta*-cleavage dioxygenases involved in biphenyl and *m*-xylene degradation by *Beijerinckia* sp. strain B1. J. Bacteriol. **177**:3095–3103.
32. Kitayama, A., T. Achioku, T. Yanagawa, K. Kanou, M. Kikuchi, H. Ueda, E. Suzuki, H. Nishimura, T. Nagamune, and Y. Kawakami. 1996. Cloning and

characterization of extradiol aromatic ring-cleavage dioxygenases of *Pseudomonas aeruginosa* JI104. J. Ferment. Bioeng. **82**:217–223.

33. Kitayama, A. Swiss protein database, accession no. P27887.

34. Ksenzenko, V. N., A. N. Kulakova, M. J. Larkin, and L. A. Kulakov. GenBank, accession no. L77225.

35. Kuhn, A. E., A. Stolz, K.-L. Ngai, and H.-J. Knackmuss. 1991. Purification and characterization of a 1,2-dihydroxynaphthalene dioxygenase from a bacterium that degrades naphthalenesulfonic acids. J. Bacteriol. **173**:3795–3802.

36. Kukor, J. J., and R. H. Olsen. 1996. Catechol 2,3-dioxygenases functional in oxygen-limited (hypoxic) environments. Appl. Environ. Microbiol. **62**:1728–1740.

37. Latus, M., H.-J. Seitz, J. Eberspacher, and F. Lingens. 1995. Purification and characterization of hydroxyquinol 1,2-dioxygenase from *Azotobacter* sp. strain GP-1. Appl. Environ. Microbiol. **61**:2453–2460.

38. Lee, J., J. Oh, K. R. Min, C. K. Kim, K. H. Min, K. S. Lee, Y. C. Kim, J. Y. Lim, and Y. Kim. 1996. Structure of catechol 2,3-dioxygenase gene encoded in chromosomal DNA of *Pseudomonas putida* KF715. Biochem. Biophys. Res. Commun. **224**:831–836.

39. McClure, N. C., C. P. Saint, and A. S. Weightman. EMBL database, accession no. S31402.

40. Moon, J., H. Chang, K. R. Min, and Y. Kim. 1995. Cloning and sequencing of the catechol 2,3-dioxygenase gene of *Alcaligenes* sp. KF711. Biochem. Biophys. Res. Commun. **208**:943–949.

41. Nakai, C., K. Hori, H. Kagamiyama, T. Nakazawa, and M. Nozaki. 1983. Purification, subunit structure, and partial amino acid sequence of metapyrocatechase. J. Biol. Chem. **258**:2916–2922.

42. Nakai, C., H. Kagamiyama, M. Nozaki, T. Nakazawa, S. Inouye, Y. Ebina, and A. Nakazawa. 1983. Complete nucleotide sequence of the metapyrocatechase gene on the TOL plasmid of *Pseudomonas putida* mt-2. J. Biol. Chem. **258**:2923–2928.

43. Ng, L. C., C. C. Sze, and C. L. Poh. GenBank, accession no. A14852.

44. Ng, L. C., C. C. Sze, and C. L. Poh. GenBank, accession no. G540972.

45. Page, R. D. M. 1996. TREEVIEW: an application to display phylogenetic trees on personal computers. Comput. Appl. Biosci. **12**:357–358.

45a. Pretsch, E., T. Clerc, J. Seibel, and W. Simon. 1989. Tables of spectral data for structure determinations of organic compounds. Springer-Verlag, Berlin, Germany.

46. Prucha, M., V. Wray, and D. H. Pieper. 1996. Metabolism of 5-chlorosubstituted muconolactones. Eur. J. Biochem. **237**:357–366.

47. Saitou, N., and M. Nei. 1987. The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol. Biol. Evol. **4**:406–425.

48. Schein, C., and M. Noteborn. 1988. Formation of soluble recombinant proteins in *Escherichia coli* is favored by lower growth temperature. Bio/Technology **4**:291–294.

49. Schmid, A., B. Rothe, J. Altenbuchner, W. Ludwid, and K.-H. Engesser. 1997. Characterization of three distinct extradiol dioxygenases involved in mineralization of dibenzofuran by *Terrabacter* sp. strain DPO360. J. Bacteriol. **179**:53–62.

50. Senda, T., H. Sugiyama, H. Narita, T. Yamamoto, K. Kimbara, M. Fukuda, M. Sato, K. Yano, and Y. Mitsui. 1996. Three dimensional structure of free form and two substrate complexes of an extradiol ring-cleavage type dioxygenase, the *BphC* enzyme from *Pseudomonas* sp. strain KKS102. J. Mol. Biol. **255**:735–752.

51. Spain, J. C., and S. F. Nishino. 1987. Degradation of 1,4-dichlorobenzene by a *Pseudomonas* sp. Appl. Environ. Microbiol. **53**:1010–1019.

52. Spanggord, R. J., J. C. Spain, S. F. Nishino, and K. E. Mortelmans. 1991. Biodegradation of 2,4-dinitrotoluene by a *Pseudomonas* sp. Appl. Environ. Microbiol. **57**:3200–3205.

53. Suen, W.-C., B. E. Haigler, and J. C. Spain. 1993. Evidence that 2,4,5-trihydroxytoluene oxygenase is a *meta*-ring cleavage enzyme, p. 179. Abstracts of the IV International Congress on *Pseudomonas*.

54. Suen, W. C., B. E. Haigler, and J. C. Spain. 1996. 2,4-Dinitrotoluene dioxygenase from *Burkholderia* sp. strain DNT: similarity to naphthalene dioxygenase. J. Bacteriol. **178**:4926–4934.

55. Suen, W. C., and J. C. Spain. 1993. Cloning and characterization of *Pseudomonas* sp. strain DNT genes for 2,4-dinitrotoluene degradation. J. Bacteriol. **175**:1831–1837.

56. Sze, I. S.-Y., and S. Dagley. 1984. Properties of salicylate hydroxylase and hydroxyquinol 1,2-dioxygenase purified from *Trichosporon cutaneum*. J. Bacteriol. **159**:353–359.

57. Tabor, S., and C. C. Richardson. 1985. A bacteriophage T7 RNA polymerase/promoter system for controlled exclusive expression of specific genes. Proc. Natl. Acad. Sci. USA **82**:1074–1078.

58. Taira, K., N. Hayase, N. Arimura, S. Yamashita, T. Miyazaki, and K. Furukawa. 1988. Cloning and nucleotide sequence of the 2,3-dihydroxybiphenyl dioxygenase from the PCB-degrading strain of *Pseudomonas paucimobilis* Q1. Biochemistry **27**:3990–3996.

59. Takami, H., T. Kudo, and K. Horikoshi. 1997. Isolation of extradiol dioxygenase genes that are phylogenetically distant from other *meta*-cleavage dioxygenase genes. Biosci. Biotechnol. Biochem. **61**:530–532.

60. Takeo, M. GenBank, accession no. D86080.

61. Thompson, J. D., D. G. Higgins, and T. J. Gibson. 1994. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice. Nucleic Acids Res. **22**:4673–4680.

62. Whiting, A. K., Y. R. Boldt, M. P. Hendrich, L. P. Wackett, and L. Que, Jr. 1996. Manganese (II)-dependent extradiol-cleaving catechol dioxygenase from *Arthrobacter globiformis* CM-2. Biochemistry **35**:160–170.

63. Yrjala, K., L. Paulin, S. Kilpi, and M. Romantschuk. 1994. Cloning of *cmpE*, a plasmid-borne catechol 2,3-dioxygenase-encoding gene from the aromatic- and chloroaromatic-degrading *Pseudomonas* sp. HV3. Gene **138**:119–121.

64. Zaborina, O., M. Latus, J. Eberspächer, L. A. Golovleva, and F. Lingens. 1995. Purification and characterization of 6-chlorohydroxyquinol 1,2-dioxygenase from *Streptomyces rochei* 303: comparison with an analogous enzyme from *Azotobacter* sp. strain GP1. J. Bacteriol. **177**:229–234.
