
Biochemical and Biophysical Research Communications 257, 50–56 (1999)

Article ID bbrc.1999.0411, available online at http://www.idealibrary.com on IDEAL®

# Cloning and Expression of a Novel Lysophospholipase Which Structurally Resembles Lecithin Cholesterol Acyltransferase

Yoshio Taniyama,*¹ Sachio Shibata,* Shunbun Kita,* Kenichi Horikoshi,* Hiromitsu Fuse,* Hideo Shirafuji,* Yasuhiro Sumino,* and Masahiko Fujinot

*Discovery Research Laboratories II and †Discovery Research Laboratories I, Pharmaceutical Discovery Research Division, Takeda Chemical Industries Ltd., 10 Wadai, Tsukuba, Ibaraki 300-4293, Japan

Received February 4, 1999

---

Lecithin cholesterol acyltransferase (LCAT) is the key enzyme in the esterification of plasma cholesterol and in the reverse cholesterol transport on high-density lipoprotein (HDL). We have found a novel LCAT-related gene among differentially expressed cDNA fragments between two types of foam cells derived from THP-1 cells, which are different in cholesterol efflux ability, using a subtractive PCR technique. The deduced 412-amino-acid sequence has 49% amino acid sequence similarity with human LCAT. In contrast to the liver-specific expression of LCAT, mRNA expression of the gene was observed mainly in peripheral tissues including kidney, placenta, pancreas, testis, spleen, heart, and skeletal muscle. The protein exists in human plasma and is probably associated with HDL. Moreover, we discovered that the recombinant protein hydrolyzed lysophosphatidylcholine (lysoPC), a proatherogenic lipid, to glycerophosphorylcholine and a free fatty acid. We have therefore named this novel enzyme LCAT-like lysophospholipase (LLPL), through which a new catabolic pathway for lysoPC on lipoproteins could be elucidated.

lipase activity, which is not observed with LCAT. LysoPC is produced in reactions catalyzed by LCAT on HDL and has been reported to be partially transferred from HDL to serum albumin [3]. However, until now the catabolism of the resulting lysoPC has remained unclear. Moreover, lysoPC is also produced by LDL-associated phospholipase A₂ (PLA2) [4] after the oxidation of LDL. LysoPC is believed to be proatherogenic because it serves as a chemoattractant for monocytes, induces monocyte adhesion to arterial endothelial cells, impairs endothelium-dependent arterial relaxation and promotes macrophage proliferation [5]. Based on our data that LLPL exists in the plasma and is probably associated with HDL, we propose a model by which lysoPC produced by LCAT on HDL is degraded by LLPL, and lysoPC on oxidized LDL can also be a substrate for LLPL in the circulation.

## EXPERIMENTAL PROCEDURES

### Preparation of foam cells
Two types of foam cells were prepared fundamentally according to the previous method [6] with minor modifications. Namely, THP-1 cells (Dainippon Pharmaceutical, Osaka, Japan) were induced to differentiate into macrophage-like cells by 3 days of treatment with 400 ng/ml PMA followed by exposure to 0.2 mg/ml total cholesterol of rabbit β-VLDL (Biomedical Technologies Inc., Stoughton, MA) for 1 day. Similarly, THP-1 cells were treated with 5 ng/ml PMA for 5 days, and then treated with β-VLDL.

### Subtraction of mRNA between the two types of foam cells
For each cell type, the total RNA was extracted with guanidine isothiocyanate, and the poly(A)⁺RNA was purified using an oligo-dT cellulose column (Pharmacia). A subtractive PCR was carried out using a PCR-select cDNA Subtraction Kit (Clontech) according to the instructions of the manufacturer.

### cDNA cloning of human LLPL
Cloning of the cDNA was performed using a Gene Trapper Positive Selection System (Gibco BRL) according to the manufacturer’s manual. The following synthetic oligonucleotide specific for clone 4s-086 (Fig. 1) was used as a probe for the screening of the human heart and kidney cDNA plasmid libraries (Gibco BRL): 5′-GCTGCTGCCCTACAACACTACACAT-3′. Sequence analysis of the cDNA clones was done using a Dye Terminator.

---

¹ To whom correspondence should be addressed: Fax: 81-298-64-6308. E-mail: Taniyama_Yoshio@takeda.co.jp).

The sequence for human LLPL has been deposited in DDBJ, EMBL and GenBank nucleotide sequence databases with the accession number: AB017494.

0006-291X/99 $30.00

Copyright © 1999 by Academic Press  
All rights of reproduction in any form reserved.

**Vol. 257, No. 1, 1999**

**BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS**

tor Cycle Sequence FS Ready Reaction Kit and a Prism sequencer 377 (PE Applied Biosystems).

**Polyclonal antibodies against human LLPL peptide and recombinant LLPL produced by E. coli.** The partial peptide (H-PVIGPLKIREQQRSAVSTC-NH2) of human LLPL was chemically synthesized as LLPL antigen. To prepare the protein with His peptide (7mer) fused at the C-terminus (LLPL/His), a primer: 5′-CAACGAGGAACCCGGGGAGCTCGCC-3′, which was complementary to the upper strand and consisting of the XhoI linker sequence added at the 5′-end and a primer: 5′-ACATATGGCTAGCGCC-GGACGTCACCCC-3′, for the upper strand involving the start codon and its upstream of the NdeI site were constructed. PCR was carried out to obtain a DNA fragment. Purification and production of the protein in *E. coli* was performed using pET system (Novagen) by the instructions of the manufacturer. Immunization was performed by standard procedures [7]. Rabbit antiserum PCL 33 against the peptide was purified by affinity column. Two kinds of antisera to LLPL/His, PCL01 and PCL02 were purified with High-Trap protein G column (Amersham-Pharmacia).

**Expression of LLPL cDNA in COS-7 cells.** The EcoRI/XbaI digest of LLPL cDNA was subcloned into the same sites of a mammalian expression vector pCI (Promega). COS-7 cells were transfected by the plasmid using TransFast Transfection Reagent (Promega).

**Expression of the LLPL/FLAG and LLPL genes in insect cells.** To prepare the protein with the FLAG peptide (DYKDDDDK) fused at the C-terminus (LLPL/FLAG), a primer (5′-CCGCTCGAGTCACT-TGTCATCGTCGTCCTTGTAGTCGGGCCCAAAGGAGCACACG-TTTCAG-3′) which was complementary to the upper strand and consisting of the sequence encoding the FLAG peptide and the XhoI linker sequence added at the 5′-end and a primer (5′-GGAGACAACCAACCGGATCCC-AGTCATCGGG-3′) for the upper strand upstream of the BamHI site were constructed. PCR was carried out using the positive clone from heart as the template to obtain a DNA fragment, and its nucleotide sequence was confirmed. Then a DNA fragment encoding the N-terminal sequence was prepared, and the two fragments were ligated into pFAST BacI (Gibco BRL). Recombinant virus was acquired from the insect cell line Sf9 according to the manual of the Bac-To-Bac Baculovirus Expression System (Gibco BRL).

**Purification and determination of the N-terminal amino acid sequence of LLPL/FLAG protein.** High Five cells (Gibco BRL) were grown to a density of 2.0 × 10⁶ cells/ml in Ex-Cell 405 medium (JRH Biosciences, Lenexa, KS), and then infected at a multiplicity of infection of 1. The culture was incubated at 27°C for 3 days. The supernatant was applied to an anti-FLAG M2 affinity column. After washing with TBS buffer, elution was carried out with solutions of FLAG peptide in TBS buffer. A portion of the above solution was subjected to SDS-PAGE and transferred to a PVDF membrane (Millipore), and the N-terminal amino acid sequence was determined using a HP G1005A protein sequencing system (Hewlett Packard, Palo Alto, CA).

**Preparation of hydrophobic proteins in lipoprotein deficient plasma (LPDP).** Ten ml of fresh human plasma was separated by ultra-centrifugation, and the resulting LPDP pool (d > 1.21 g/ml, 40 krpm, 40 h) was dialyzed against 10 mM sodium phosphate buffer (pH 7.0) containing 0.5 M NaCl and 0.1% NaN₃, and loaded onto a 10 × 60 mm Phenyl Sepharose CL-4B (Pharmacia) column equilibrated with the same buffer. The column was washed with 100 ml of the same buffer and eluted with 50 ml of deionized water. The eluent was concentrated with YM-10 membrane (Amicon) to 5 ml, and used as PS eluent.

**Northern blot analyses.** The multiple tissue northern blots of human tissues (Clontech) were hybridized with a clone 4S-086 labeled with [α-³²P]dCTP by the random priming method. The membrane was autoradiographed with an image analyzer (BAS 2000, Fuji Film).

---

**Detection of lysophospholipase activity.** \[^{14}C\]LysoPC (40 μM) was incubated with the enzyme in 10 mM Tris-HCl (pH 7.5)-150 mM NaCl solution at 37°C for 1 h. After extraction from the mixture by CHCl₃/MeOH (1:2, v/v), the release of the free fatty acid was measured by TLC using successive development with chloroform/methanol/H₂O (65/35/5; v/v/v) and hexane/diethylether/acetic acid (90/20/1; v/v/v) and an image analyzer.

**RESULTS**

**cDNA Cloning of LLPL**

To isolate the genes related to the lipid metabolism, the nucleotide sequences of the differentially expressed cDNA fragments between two types of foam cells (see Experimental Procedures) were determined and homology searches undertaken in the GenBank database for them using the BLAST N program [8]. We found clone 4s-086 (Fig. 1) had a novel nucleotide sequence which is similar to that of chicken LCAT. Four clones were selected from human heart and kidney cDNA libraries, three of which were found to encode a predicted 412-amino-acid protein (Fig. 1). The other clone from kidney encoded a predicted 444-amino-acid protein, corresponding to a variant with a 32-amino-acid insert in the above form (Fig. 1), suggesting that this long form may be generated by alternative splicing. The presence of the long form was confirmed by PCR in other human cDNA libraries. The sequence of the three clones revealed only one possible open reading frame with a typical signal peptide of 33 residues and a mature protein of 379 amino acids (Fig. 1).

**Primary Structure of LLPL**

Compared with the amino acid sequence of human LCAT [9], the two potential disulfide bonds and three potential N-linked glycosylation sites are conserved (Fig. 2). While the putative active site serine is present in the LLPL protein, the GXSXG motif commonly found in lipases is replaced by the AXSXG motif [10]. Recently, a surface region consisting of 25 amino acids linked by a disulfide bond (C₇₄-C₉₈ in Fig. 2) in LCAT has been reported to be involved in the binding of the enzyme to lipoproteins [11]. The corresponding region of LLPL demonstrates a high sequence similarity with that of LCAT (Fig. 2), suggesting that this region of LLPL is functional, as in the case of LCAT.

**Production of Recombinant LLPL Protein**

To study the properties of the LLPL protein, the LLPL/FLAG as well as native LLPL protein were produced in insect cells using the baculovirus system. The culture supernatants were then harvested and analyzed by Western blotting. In the case of LLPL/FLAG, a specific band reacting with both the anti-peptide antibody and an anti-FLAG M2 monoclonal antibody (Eastman Kodak, New Haven, CT) was identified at a position corresponding to 47 kDa (Fig. 4e). This protein

Vol. 257, No. 1, 1999

BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS

CATCCCGGACCTGCGGCGACC GTGCTACACCATGGGCC TCCA CCTCCGCCCCCTACC GTGTGGGGCCTGCTCCC GGA TG GCCCTCCTGTT CCT M G L H L R P Y R V G L L P D G L L F L 90

CTTGCTGCTGCTAATGCTGCTCGCGGACCCAGCGCTCCC GGCCGGACGTCACCCCCCA GTGGTGC TGGTCCCTGGTGATTTGGTA ACCA L L L L M L L A D P A L P A G R H P P V V L V P G D L G N Q 180

ACTGGAAGCCAAGCTGGACAAGCCGACAGTGGTGCACT ACCTCTGCTCCAAGAAAGAC CGAAAGCTACTTCACAAT CTGGCTGAACCTGGA L E A K L D K P T V V H Y L C S K K T E S Y F T I W L N L E 270

ACTGCTGCTGCC TGTCATCATTGACTGCTGGATTGACA AATATCAGGCTGGTTTACAACAAAAACATCCAGGGCCACCCAGTTT CCTGATGG L L L P V I I D C W I D N I R L V Y [N K T] S R A T Q F P D G 360

TGTGGATGTACGTGTCCTGGCTTTGGGAAGACCTTCTC ACTGGAGTTCCTGGACCCCAGCAA AAAGCAGGTGGGTTCTC TATTTCCACAC V D V R V P G F G K T F S L E F L D P S K S S V G S Y F H T 450

CATGGTGGAGAGCCTTGTGGGCTGGGGCTACACACGGGG TGAGGATGTCGAGGGGCTCCCT ATGACTGGCGCCGAGCCCCAAATGAAA M V E S L V G W G Y T R G E D V R G A P Y D W R R A P N E N 540

CGGGCCCTACTT CCTGGCCCTCCGCGAGATGATCGAGGAGATGTACCAGCTGTATGGGGGCCCCGTGGTGCTGGTTGCCCACAGTATGGG G P Y F L A L R E M I E E M Y Q L Y G G P V V L V [A H S M G] 630

CAACATGTACACGCTCTACTTTCTGCAGCGGCAGCCGC AGGCCCTGGAAGACAAGTATAT CCGGGCCTTCTGTGTC ACTGGGTGCGCCCTG N M Y T L Y F L Q R Q P Q A W K D K Y I R A F V S L G A P W 720

GGGGGGCGTGGCCAAGACCCCTGCGCGT CCTGGCTCAGGAGACAACAACCGGATCC CAGT CATCGGGCCCCTGAAGATCCGGGAGCAGCA G G V A K T L R V L A S G D N N R I P V I G P L K I R E Q Q 810

GCGGTCAGCTGTCTCCACCAGCTGGCTGCCC TACA ACTACACATGGTCACCTGAGAAGGTGTT CGTGCAGACACCCCA CAATCACTA R S A V S T S W L L P Y [N Y T] W S P E K V F V Q T P T I [N Y] 900

CACACTGCGGGGACTACCGCAAGTTCTTCCAGGACATCG GCTTTGAA GATGGCTGGCTCATGGGCAGGACACAGAAGGGCTGGT GGAAGC T J L R D Y R K F F Q D I G F E D G W L M R Q D T E G L V E A 990

CACGATGCCACCTGGCGTGCAGCTGC ACTGGCGTCCCCACACCAGACTCCTTCTACTATGAGAGC TTTCCCTGACC G T M P P G V Q L H C L Y G T G V P T P D S F Y Y E S F P D R 1080

TGACCCTAAAATCTGCTTTGGTGACGGCGATGGTACTGT GAACTTGAAGAGTGCCCCTGCA GTGCCAGG CCTGGCAGAGCCGCCAGGAGCA D P K I C F G D G D G T V N L K S A L Q C Q A W Q S R Q E H 1170

CCAAGTGT TGCTGCAGGAGCTGCCAGCGAGCACATCGAGATGCTGGCCAACGCCACCCCTGGCCTATCTGAAACGTGTGCTCCT Q V L L Q E L P G S E H I E M L A [N A T] T L A Y L K R V L L 1260

TGGGCCCTGACTCCTGTGCCACAGGACTCCTGTGGCTCG GCCGTGGACCTGCTGTTGGCCTCTGGGGCTGTCA TGGCCCACGCGTTTTTGC G P * 1350

FIG. 1. Nucleotide and deduced amino-acid sequences of a cDNA encoding human LLPL. Amino acids are numbered beginning with the presumed initiating methionine. The N-terminal amino acid sequence of purified LLPL/FLAG experimentally determined, is **double underlined**. The consensus sequences for N-linked glycosylation sites and for the conserved lipase active-site serine motif are boxed by a dashed line and solid line, respectively. Amino acid sequence of the peptide for antigen is indicated by underlining. Nucleotide sequence of the clone 4S-086 is indicated by **dashed underlining**. Nucleotide sequence of the probe for Gene Trapper is indicated by an **underlined arrow**. The site for insertion of the amino acid sequence LECSGAISAHYTSASQAQALLLPQT PDNW DYR in a variant is indicated by an upward arrow. The stop codon is indicated by an asterisk.

---

was purified using an anti-FLAG M2 affinity column (Eastman Kodak) from the culture supernatants (ca. 6 mg/l). Sequence analysis of the purified epitope-tagged protein gave a partial N-terminal amino acid sequence of AGRHPPVVLV, indicating that the N-terminal sequence of this mature protein starts from the 34th amino acid of the precursor protein shown in Fig. 1. Similarly, the native LLPL was analyzed by Western blotting, giving a specific band with a molecular mass of about 45 kDa.

Biological Activities of the Recombinant Proteins

The esterase activity of the recombinant LLPL/FLAG was assayed using p-nitrophenyl butyrate (PNPB) as a substrate [12]. LLPL/FLAG hydrolyzed PNPB with a Km of 310 μM and a Vmax of 3.3 nmol/min in the presence of 3.4 μg/ml of the protein, calculated from the Lineweaver-Burk plot. These kinetic parameters are in good accordance with those of human plasma LCAT [12]. The partially purified recombinant native LLPL also had the esterase activity. Although we tried to determine whether the recombinant proteins have lecithin cholesterol acyltransferase activity (LCAT activity) [13], none was detectable under the conditions. We studied the phospholipase activity of the LLPL/FLAG protein, since LCAT activity consists of PLA2 and acyltransferase activities. When 1,2-di [¹⁴C] oleoyl-phosphatidylcholine was used as a substrate, a low level of radioactivity was detected in spots corresponding to the free fatty acid, suggesting that the enzyme has weak phospholipase activity toward phosphatidylcholine (0.4 ± 0.1 nmd/min/mg). Finally, we examined the lysophospholipase activity of this enzyme and found that it released a fatty acid at the sn-1 position of lysoPC (Fig. 3) with an activity of 12 ± 0.2 nmol/min/mg, which is ca. 30 times higher than the phospholipase activity described above. In addition, the lyso-

Vol. 257, No. 1, 1999

BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS

LLPL MGLHLRPY-RV----G-LLPDGLLF-LL-LLLMLLADPALPAGRH-PPVVLVPGDLGQNLEAKLDKPTV 60  
LCAT MGPPGSPWQWVTLLLGLLLPPAEPFWLLNVLFPPHTTPKAELSNHTRPVILVPGCLGNQLEAKLDKPDV 69  
** * * ** * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
Vol. 257, No. 1, 1999

BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS

![Figure](https://i.imgur.com/yourimage.png)

FIG. 4. Western blot analyses of the plasma LLPL in LPDP fraction. Five μl of PS eluent was separated by 12.5% SDS-PAGE under reducing conditions and transferred to a Hybond-P membrane (Amersham). Lane 1, recombinant LLPL secreted by COS-7 cells; lane 2, PS eluent of LPDP; lane F, recombinant LLPL/FLAG secreted by insect cells. Blots recognized by anti-human LLPL-peptide antibody, PCL33 (a), and anti-human LLPL/His antibody, PCL01 (b) were detected with alkaline phosphatase conjugated goat anti-(rabbit IgG) polyclonal antibody absorbed with human serum protein (Sigma, MO). Blots recognized by biotinylated antibody PCL02 in the absence (c) or presence (e) of excess amounts of rabbit preimmune IgG, and presence of excess amounts of non-biotinylated PCL02 (d), were all detected with Streptavidin alkaline phosphatase conjugate (Boehringer). No bands were detected without first antibodies in these systems.

plasma LLPL as assessed by Western blot analyses using three different antibodies.

**Expression of LLPL**

Northern blot analysis of human tissues revealed a high level of LLPL mRNA expression in peripheral tissues (Fig. 5). The low transcription level of LLPL in liver is surprising because LCAT mRNA has been reported to be expressed most abundantly in liver [2].

**DISCUSSION**

We have reported here the molecular cloning and characterization of a new human lysophospholipase that we have designated as LLPL, which contains a unique sequence motif, AXSXG, found in Bacillus lipases [10]. We have confirmed that this unique motif is also conserved in the counterpart molecules of mouse and rabbit, suggesting that the motif in LLPL has been conserved among mammalian species during evolution. To our knowledge, an enzyme containing this motif has not yet been reported in mammalian species. In addition, the catalytic residues, D369 and H401, and the possible second oxyanion hole residue F127 of LCAT in Fig. 2 are conserved in LLPL protein, suggesting that the tertiary structure of the active site in LLPL is similar to that of LCAT [14].

LLPL has sequence homology with LCAT, which associates with lipoproteins. To confirm the existence of mildly associated LLPL with lipoproteins, human plasma precipitated with dextran sulfate/MnCl₂ [15] was fractionated [16] and analyzed by Western blotting. As a result, the bands with a molecular weight of about 57 kDa appeared mainly in the HDL fractions, suggesting that this 57-kDa band involves the plasma LLPL protein associated with HDL (data not shown). Although the source of LLPL found in the plasma remains unknown, data on the tissue distribution of the mRNA and its structural homology with LCAT protein suggest that the peripheral cells secrete LLPL protein, which then associates or interacts with HDL, and the LLPL protein affects the metabolism of lipoproteins in the plasma.

A 5′-flanking region of the human LLPL gene has been cloned and found to contain a sequence TCCAGGTCA, a potential responsive element for nuclear receptor SF-1, suggesting that their ligands such as oxysterols [17] are involved in the regulation of LLPL gene transcription. In fact, the expression of the LLPL transcript in macrophage-like THP-1 cells was increased by about 2-fold with stimulation by oxidized LDL, indicating that the expression is inducible. The expression of LCAT gene, however, appears to be relatively resistant to dietary or drug challenge *in vivo* [18].

HDL is believed to have antiatherogenic properties. This assumption is supported by epidemiologic evidence, specifically, an inverse correlation between the incidence of coronary artery disease and the plasma HDL level [19]. Such properties of HDL may be ascribed to some of the enzymes located in HDL complexes, for instance, paraoxonase, PAF-acetylhydrolase, and LCAT. An important activity of HDL is to stimulate the efflux of cholesterol from peripheral cells and to accommodate the cholesterol via its esterification. LCAT on HDL is thought to be responsible in a large part for this activity of HDL. Since LLPL has similarity with LCAT and exists in the plasma (Fig. 2, Fig. 4), it may also contribute to the activity of HDL of accumulating free cholesterol released by peripheral cells. LLPL is also likely to regulate the activity of LCAT by reducing the level of lysoPC because the end-product, lysoPC, has been reported to inhibit LCAT activity as a feed-back regulator [3]. LLPL may catalyze the transfer of a fatty acid from the 1-position of lysoPC to the sn-2 position of another lysoPC or to the hydroxyl group of some cholesterol derivatives *in vivo*. Furthermore, several lines of evidence [20] suggest a role for HDL in the protection of LDL against oxidative modification. Recently, LCAT has been reported to hydrolyze oxidized phosphatidylcholine to lysoPC and be involved in the protective activity against oxidation of LDL [21]. Similarly, PAF-acetylhydrolase hydrolyzes oxidized phosphatidylcholine to lysoPC [4]. Although lysoPC is

Vol. 257, No. 1, 1999

BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS

![Figure](https://i.imgur.com/yourimage.png)

FIG. 5. Tissue distribution in human LLPL mRNA. The mRNA was detected as a band of 2.8 kb. To normalize the amounts of mRNA loaded, blots were reprobed with cDNA encoding glyceraldehyde-3-phosphate dehydrogenase (GAPDH).

proatherogenic, both enzymes apparently have only PLA2 activity, with lysoPC as a reaction end product. In this paper, LLPL structurally resembling LCAT was shown to play a role in degrading the proatherogenic lipid, lysoPC and to exist in the plasma. Therefore, we could sufficiently explain the antiatherogenic property of HDL, the protection of LDL against oxidative modification, with this novel enzyme. Recently, bile-salt-stimulated lipase in the aorta has been postulated to protect against atherosclerosis by degrading the lysoPC on oxidized LDL [22]. Thus, the regulation of lysophospholipase activity may be used for therapeutic applications. From this point of view, the presence of the LLPL variant is interesting because its activity and localization may be different from native LLPL and the profile of its existence in individuals may be associated with susceptibility to atherosclerosis. Although the physiological roles of LLPL remain to be determined, our results strongly suggest that LLPL, a novel enzyme in the plasma, plays several important roles in the antiatherogenic properties of HDL.

### ACKNOWLEDGMENTS

We thank H. Okazaki, K. Kitano, Z. Terashita, M. Nakamura and T. Wada for useful discussion throughout this study and T. Komiyama for performing the purification and production of the LLPL/His protein in *E. coli*.

---

### REFERENCES

1. Kuivenhoven, J. A., Pritchard, H., Frohlich, J., Assmann, G., and Kastelein, J. (1997) *J. Lipid Res.* **38**, 191–205.
2. Warden, C. H., Langner, C. A., Gordon, J. I., Taylor, B. A., McLean, J. W., and Lusis, A. J. (1989) *J. Biol. Chem.* **264**, 21573–21581.
3. Nakagawa, M., and Nishida, T. (1973) *J. Biochem.* **74**, 1263–1266.
4. Tew, D. G., Southan, C., Rice, S. Q. J., Lawrence, G. M. P., Li, H., Boyd, H. F., Moores, K., Glogger, I. S., and Macphee, C. H. (1996) *Arterioscler. Thromb. Vasc. Biol.* **16**, 591–599.
5. Prokazova, N. V., Zvezdina, N. D., and Korotaeva, A. A. (1998) *Biochemistry (Moscow)* **63**, 38–46.
6. Rodriguez, A., Kafonek, S. D., Georgopoulos, A., and Bachorik, P. S. (1994) *J. Lipid Res.* **35**, 1909–1917.
7. Fielding, C. J., and Fielding, P. E. (1981) *Proc. Natl. Acad. Sci. USA* **78**, 3911–3914.
8. Altschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990) *J. Mol. Biol.* **215**, 403–410.
9. McLean, J., Fielding, C., Drayna, D., Dieplinger, H., Baer, B., Kohr, W., Henzel, W., and Lawn, R. (1986) *Proc. Natl. Acad. Sci. USA* **83**, 2335–2339.
10. Schmidt-Dannert, C., Rua, M. L., and Schmid, R. D. (1997) *Methods Enzymol.* **284**, 194–220.
11. Adimoolam, S., and Jonas, A. (1997) *Biochem. Biophys. Res. Commun.* **232**, 783–787.
12. Bonelli, F. S., and Jonas, A. (1989) *J. Biol. Chem.* **264**, 14723–14728.
13. Gillett, M. P. T., and Owen, J. S. (1992) *Lipoprotein Analysis*

Vol. 257, No. 1, 1999

**BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS**

(Converse, C. A., and Skinner, E. R., Eds.), pp. 187–201, IRL Press, Oxford.

14. Peelman, F., Vinaimont, N., Verhee, A., Vanloo, B., Verschelde, J.-L., Labeur, C., Seguret-Mace, S., Duverger, N., Hutchinson, G., Vandekerckhove, J., Tavernier, J., and Rosseneu, M. (1998) *Protein Sci.* **7**, 587–599.

15. Burstein, M., Scholnick, H. R., and Morfin, R. (1970) *J. Lipid Res.* **11**, 583–595.

16. Yokode, M., Hammer, R. E., Ishibashi, S., Brown, M. S., and Goldstein, J. L. (1990) *Science* **250**, 1273–1275.

17. Lala, D. S., Syka, P. M., Lazarchik, S. B., Mangelsdorf, D. J., Parker, K. L., and Heyman, R. A. (1997) *Proc. Natl. Acad. Sci. USA* **94**, 4895–4900.

18. Ly, H., Francone, O. L., Fielding, C. J., Shigenaga, J. K., Moser, A. H., Grunfeld, C., and Feingold, K. R. (1995) *J. Lipid Res.* **36**, 1254–1263.

19. Stampfer, M. J., Sacks, F. M., Salvini, S., Willett, W. C., and Hennekens, C. H. (1991) *New Engl. J. Med.* **325**, 373–381.

20. Klimov, A. N., Gurevich, V. S., Nikiforova, A. A., Shatilina, L. V., Kuzmin, A. A., Plavinsky, S. L., and Teryukova, N. P. (1993) *Atherosclerosis* **100**, 13–18.

21. Goyal, J., Wang, K., Liu, M., and Subbaiah, P. V. (1997) *J. Biol. Chem.* **272**, 16231–16239.

22. Shamir, R., Johnson, W. J., Morlock-Fitzpatrick, K., Zolfaghair, R., Li, L., Mas, E., Lombardo, D., Morel, D. W., and Fisher, E. A. (1996) *J. Clin. Invest.* **97**, 1696–1704.
