
# Vitamin B₆ biosynthesis: formation of pyridoxine 5′-phosphate from 4-(phosphohydroxy)-L-threonine and 1-deoxy-D-xylulose-5-phosphate by PdxA and PdxJ protein

**Bernd Laber*, Wolfgang Maurer, Sandra Scharf, Karin Stepusin, Frank S. Schmidt**

*Hoechst Schering AgrEvo GmbH, Biochemical Research, Werk Höchst, H872N, D-65926 Frankfurt am Main, Germany*

Received 1 February 1999; received in revised form 15 March 1999

---

### Abstract
In *Escherichia coli* the coenzyme pyridoxal 5′-phosphate (PLP) is synthesised de novo by a pathway that is thought to involve the condensation of 4-(phosphohydroxy)-L-threonine and 1-deoxy-D-xylulose, catalysed by the enzymes PdxA and PdxJ, to form either pyridoxine (vitamin B₆) or pyridoxine 5′-phosphate (PNP). Here we show that incubation of PdxJ with PdxA, 4-(phosphohydroxy)-L-threonine, NAD and 1-deoxy-D-xylulose-5-phosphate, but not 1-deoxy-D-xylulose, results in the formation of PNP. The PNP formed was characterised by (i) cochromatography with an authentic standard, (ii) conversion to pyridoxine by alkaline phosphatase treatment, and (iii) UV and fluorescence spectroscopy. Furthermore, when [2-¹⁴C]1-deoxy-D-xylulose-5-phosphate was used as a substrate, the radioactivity was incorporated into PNP. These results clarify the previously unknown role of PdxJ in the de novo PLP biosynthetic pathway. The sugar used as substrate by PdxJ is 1-deoxy-D-xylulose-5-phosphate rather than the previously assumed 1-deoxy-D-xylulose. The first vitamin B₆ vitamer synthesised is PNP, and not pyridoxine.

© 1999 Federation of European Biochemical Societies.

**Keywords:** 1-Deoxy-D-xylulose-5-phosphate; Pyridoxal 5′-phosphate; Pyridoxine 5′-phosphate; PdxJ; Vitamin B₆

---

### 1. Introduction
Pyridoxal 5′-phosphate (PLP) is the biocatalytically active form of vitamin B₆ (pyridoxine, pyridoxol; PN), which acts as the essential coenzyme in many metabolic conversions of amino acids. Recent genetic and biochemical evidence indicates that the entire C₈N skeleton of PN is derived from two precursors. A modified pentose, 1-deoxy-D-xylulose (1; Scheme 1), yields C-2′, 2, 3, 4, and 4′ of PN, while the non-proteinogenic amino acid 4-(phosphohydroxy)-L-threonine (HTP; 2) is destined to become N-1, C-6, 5, and 5′ of PN [1–4]. In *Escherichia coli* only two genes, *pdxA* and *pdxJ*, have been implicated in the intramolecular condensation reaction between these two substrates to yield pyridoxine 5′-phosphate (PNP; 3). PNP is then oxidised by the PdxH oxidase to form PLP, the active coenzyme [5, 6]. The exact roles, however, played by the *pdxA* and *pdxJ* gene products, PdxA and PdxJ, have long been undetermined. Cane et al. [7] recently demonstrated that PdxA is an NAD-dependent dehydrogenase which oxidises HTP. Here we present evidence that purified PdxJ catalyses the in vitro formation of PNP in the presence of purified PdxA, HTP, NAD and 1-deoxy-D-xylulose-5-phosphate, but not 1-deoxy-D-xylulose.

---

### 2. Materials and methods

#### 2.1. Materials
HTP was prepared as described previously [8]. PNP was synthesised by the method of Argoudelis [9]. Pyridoxine, DL-glyceraldehyde 3-phosphate and alkaline phosphatase (type VII-S from bovine intestinal mucosa) were purchased from Sigma. [2-¹⁴C]Pyruvate (0.59 GBq/mmol, 1.85 MBq/ml) was obtained from New England Nuclear. 1-Deoxy-D-xylulose was a gift from Dr. Michael Hoffmann, Hoechst Schering AgrEvo GmbH.

#### 2.2. Overexpression and purification of PdxA and PdxJ
The two *pdx* genes encoding the PdxA and PdxJ polypeptides were amplified by PCR (20 cycles, 56°C annealing temperature) with Pfu DNA polymerase (Stratagene) according to the manufacturer's recommendations using phosphorothioate-protected primers [10]. PCR mixtures contained approximately \(10^6\) cells of *E. coli* JM83 [11] as a source for the template DNA. *pdxA* was amplified using the oligonucleotides 5′-GCACTAGGTCTCGAATGGTTAAAACCCAACGTG-3′ and 5′-GCACTAGGTCTCAGCGCTTTGGGGTGTAAACAAT-CATTTTG-3′ as primers. *pdxJ* was amplified using the oligonucleotides 5′-GCACTAGGTCTCGAATGGCTGAATTACTGTTTAGG-3′ and 5′-GCACTAGGTCTCAGCGCTGCCACGCGCTTCCAGCA-T-3′ as primers. The *pdxA* and *pdxJ* PCR products were digested by *BsaI* and cloned in the *BsaI* sites of the *E. coli* expression vector pASK-IBA3 (IBA-Institut für Bioanalytik GmbH, Göttingen, Germany) yielding the plasmids pPDXA1 and pPDXJ1, respectively. pPDXA1 and pPDXJ1 encode PdxA and PdxJ polypeptides, respectively, which are tagged at their C-termini by the affinity peptide Strep-tag II [12]. The sequences of the cloned PCR products were identical with the respective CDS regions of the GenBank/EMBL accession numbers M68521 (*pdxA*) and M74526 (*pdxJ*). The pPDXA1 and pPDXJ1 constructs were used to transform *E. coli* JM83. For overexpression, the *E. coli* strains JM83/pPDXA1 and JM83/pPDXJ1 were grown at 37°C in LB medium containing 100 μg/ml ampicillin to a cell density of \(A_{550}=0.5\) and induced with 200 μg/l anhydrotetracycline [13] for 16 h. Cells were harvested by centrifugation, resuspended in CP buffer (100 mM Tris-HCl pH 8.0, 1 mM EDTA) and disrupted by two passages through a French pressure cell at 15,000 psi. The cell debris was removed by centrifugation (18,000 × g for 30 min) and the supernatant subjected to 0–35% ammonium sulphate fractionation. The precipitate was resuspended in CP buffer, desalted, and affinity-purified on a StrepTactin column (IBA).

#### 2.3. Synthesis of DXP
The DXP synthase overproducing *E. coli* strain JM109/pUCBM20dxs was a gift from Dr Georg A. Sprenger, Forschungszentrum Jülich, Institut für Biotechnologie 1, Jülich, Germany. DXP synthase was purified by ammonium sulphate fractionation and Q-Sepharose anion-exchange chromatography [14]. The reaction mixture for the enzymatic synthesis of DXP consisted of 100 mM HEPES buffer pH 8.3, 2 mM thiamine diphosphate, 10 mM MgCl₂, 45 mM DL-glyceraldehyde 3-phosphate, 15 mM pyruvate and 0.25 mg of DXP synthase in a final volume of 5 ml. After incubation for 5 h at 37°C, the reaction was terminated by heating at 100°C for 5 min. Denatured

---

*Corresponding author. Fax: (49) (69) 305 17207.  
E-mail: bernd.laber@agrevo.com*

**Abbreviations:** DXP, 1-deoxy-D-xylulose-5-phosphate; HTP, 4-(phosphohydroxy)-L-threonine; PN, pyridoxine; PLP, pyridoxal 5′-phosphate; PNP, pyridoxine 5′-phosphate

0014-5793/99/$20.00 © 1999 Federation of European Biochemical Societies. All rights reserved.  
PII: S0014-5793(99)00393-2

#### Text Content

**Page Content:**

protein was removed by centrifugation (13,000 × g, 5 min) and the amount of DXP formed was calculated after determination of the residual pyruvate with lactate dehydrogenase. DXP was used without further purification. \[^{14}C\]DXP (15 Bq/nmol) was prepared in 1 ml of the above reaction mixture containing 30 μg DXP synthase and supplemented with 74 kBq \[^{14}C\]pyruvate. Radiochemical purity of \[^{14}C\]DXP was determined after DC separation [15] with a thin-layer scanner (Berthold, Germany) and found to be >95%.

#### 2.4. Assay for PNP biosynthesis

For the enzymatic synthesis of PNP, a reaction mixture consisting of 20 mM Tris-HCl pH 7.8, 1 mM NAD, 1 mM HTP, and 67 μg PdxA in a total volume of 1 ml was incubated at 37°C. After 30 min, when about 50% of HTP had been oxidized (as judged by the increase in absorbance at 340 nm), a 0.6 ml aliquot of the reaction mixture was supplemented with 138 μg PdxJ, 0.5 mM DXP, and water to yield a final volume of 0.9 ml. After another 80 min incubation at 37°C, 0.4 ml of the assay mixture was mixed with 0.16 ml of 1 N HClO₄ to terminate the reaction, while another 0.4 ml aliquot of the assay mixture was mixed with 0.18 ml 0.33 M Tris-HCl pH 8.8, 3.3 mM MgSO₄, 0.33 ZnSO₄, and 20 μl (100 units) of alkaline phosphatase. After 60 min at 37°C, the reaction was terminated by the addition of 1 N HClO₄. Precipitated protein was removed by centrifugation (13,000 × g, 5 min), and the samples were analyzed for the presence of vitamin B₆ vitamers by isocratic reversed-phase HPLC on a Merck Hibar RT Lichrospher RP-18 (250 × 4 mm, 5 μm) column. The effluent was monitored fluorometrically (emission, 390 nm; excitation, 290 nm) [16].

#### 3. Results and discussion

Recombinant PdxJ protein was produced in *E. coli* and purified by ammonium sulfate fractionation and StrepTactin affinity chromatography. This protocol yielded about 20 mg of purified enzyme per liter cell culture. The apparent molecular mass of the affinity-purified protein was determined as 30.0 kDa by SDS-polyacrylamide gel electrophoresis, which is close to the 27,510 Da calculated from the nucleic acid sequence. Overexpression of cloned *pdxA* in *E. coli* yielded about 10 mg of purified enzyme per liter cell culture. The purified protein showed a \(k_{cat}\) of 1.4 s⁻¹ and its apparent molecular mass was determined as 35.9 kDa (calculated molecular mass: 36,240 Da).

It is unclear at present whether the product of the PdxA reaction, 2-amino-3-oxo-4-(phosphohydroxy)butyric acid (4; Scheme 2), or 1-amino-3-(phosphohydroxy)propan-2-one (5), which is assumed to be generated by spontaneous decarboxylation of 4, is the substrate for PdxJ. Therefore, all assays for PdxJ activity were carried out in the presence of PdxA, HTP, and NAD to ensure in situ formation of 4 and/or 5.

HPLC analysis of a reaction mixture containing PdxJ and DXP as well as PdxA, HTP, and NAD revealed the formation of a fluorescent compound that was eluted from the HPLC column with a retention time identical to that of authentic PNP (Fig. 1). Further evidence for the formation of PNP under our experimental conditions was obtained by treatment of the reaction mixture with alkaline phosphatase prior to HPLC analysis. This treatment increased the retention time of the fluorescent compound to such an extent that it coeluted with authentic PN (Fig. 1). When aliquots of the untreated and alkaline phosphatase-treated reaction mixtures were supplemented with PNP and PN standards, the fluorescent compound was identified as PNP.

---

### Schemes

#### Scheme 1: Biosynthesis of pyridoxine 5′-phosphate (3)

![Scheme 1](https://i.imgur.com/XYZ.png)

**Description:**
- **(1):** Dihydroxyacetone phosphate
- **(2):** 2-Amino-3-ketobutyrate
- **(3):** Pyridoxine 5′-phosphate
- **PdxA:** Catalyzes the conversion of (1) to (2)
- **PdxJ:** Catalyzes the conversion of (2) to (3)

#### Scheme 2: Proposed reaction mechanism for the final steps in pyridoxine 5′-phosphate biosynthesis

![Scheme 2](https://i.imgur.com/ABC.png)

**Description:**
- **(2):** 2-Amino-3-ketobutyrate
- **(4):** 2-Amino-3-oxo-4-(phosphohydroxy)butyric acid
- **(5):** 1-Amino-3-(phosphohydroxy)propan-2-one
- **(6):** Dihydroxyacetone phosphate
- **(7):** Intermediate in the pathway
- **(3):** Pyridoxine 5′-phosphate

---

### Footer

**FEBS 21861 12-4-99**

---

# Protein Removal and DXP Formation

Protein was removed by centrifugation (13,000 × g, 5 min) and the amount of DXP formed was calculated after determination of the residual pyruvate with lactate dehydrogenase. DXP was used without further purification. \[^{14}C\]DXP (15 Bq/nmol) was prepared in 1 ml of the above reaction mixture containing 30 μg DXP synthase and supplemented with 74 kBq \[^{14}C\]pyruvate. Radiochemical purity of \[^{14}C\]DXP was determined after DC separation [15] with a thin-layer scanner (Berthold, Germany) and found to be >95%.

## 2.4. Assay for PNP biosynthesis

For the enzymatic synthesis of PNP, a reaction mixture consisting of 20 mM Tris-HCl pH 7.8, 1 mM NAD, 1 mM HTP, and 67 μg PdxA in a total volume of 1 ml was incubated at 37°C. After 30 min, when about 50% of HTP had been oxidized (as judged by the increase in absorbance at 340 nm), a 0.6 ml aliquot of the reaction mixture was supplemented with 138 μg PdxJ, 0.5 mM DXP, and water to yield a final volume of 0.9 ml. After another 80 min incubation at 37°C, 0.4 ml of the assay mixture was mixed with 0.16 ml of 1 N HClO₄ to terminate the reaction, while another 0.4 ml aliquot of the assay mixture was mixed with 0.18 ml 0.33 M Tris-HCl pH 8.8, 3.3 mM MgSO₄, 0.33 ZnSO₄, and 20 μl (100 units) of alkaline phosphatase. After 60 min at 37°C, the reaction was terminated by the addition of 1 N HClO₄. Precipitated protein was removed by centrifugation (13,000 × g, 5 min), and the samples were analyzed for the presence of vitamin B₆ vitamers by isocratic reversed-phase HPLC on a Merck Hibar RT Lichrospher RP-18 (250 × 4 mm, 5 μm) column. The effluent was monitored fluorometrically (emission, 390 nm; excitation, 290 nm) [16].

## 3. Results and discussion

Recombinant PdxJ protein was produced in *E. coli* and purified by ammonium sulfate fractionation and StrepTactin affinity chromatography. This protocol yielded about 20 mg of purified enzyme per liter cell culture. The apparent molecular mass of the affinity-purified protein was determined as 30.0 kDa by SDS-polyacrylamide gel electrophoresis, which is close to the 27,510 Da calculated from the nucleic acid sequence. Overexpression of cloned *pdxA* in *E. coli* yielded about 10 mg of purified enzyme per liter cell culture. The purified protein showed a \(k_{cat}\) of 1.4 s⁻¹ and its apparent molecular mass was determined as 35.9 kDa (calculated molecular mass: 36,240 Da).

It is unclear at present whether the product of the PdxA reaction, 2-amino-3-oxo-4-(phosphohydroxy)butyric acid (4; Scheme 2), or 1-amino-3-(phosphohydroxy)propan-2-one (5), which is assumed to be generated by spontaneous decarboxylation of 4, is the substrate for PdxJ. Therefore, all assays for PdxJ activity were carried out in the presence of PdxA, HTP, and NAD to ensure in situ formation of 4 and/or 5.

HPLC analysis of a reaction mixture containing PdxJ and DXP as well as PdxA, HTP, and NAD revealed the formation of a fluorescent compound that was eluted from the HPLC column with a retention time identical to that of authentic PNP (Fig. 1). Further evidence for the formation of PNP under our experimental conditions was obtained by treatment of the reaction mixture with alkaline phosphatase prior to HPLC analysis. This treatment increased the retention time of the fluorescent compound to such an extent that it coeluted with authentic PN (Fig. 1). When aliquots of the untreated and alkaline phosphatase-treated reaction mixtures were supplemented with PNP and PN standards, the fluorescent compound was identified as PNP.

### Schemes

#### Scheme 1: Biosynthesis of pyridoxine 5′-phosphate (3)

![Scheme 1](https://i.imgur.com/XYZ.png)

- **(1):** Dihydroxyacetone phosphate
- **(2):** 2-Amino-3-ketobutyrate
- **(3):** Pyridoxine 5′-phosphate
- **PdxA:** Catalyzes the conversion of (1) to (2)
- **PdxJ:** Catalyzes the conversion of (2) to (3)

#### Scheme 2: Proposed reaction mechanism for the final steps in pyridoxine 5′-phosphate biosynthesis

![Scheme 2](https://i.imgur.com/ABC.png)

- **(2):** 2-Amino-3-ketobutyrate
- **(4):** 2-Amino-3-oxo-4-(phosphohydroxy)butyric acid
- **(5):** 1-Amino-3-(phosphohydroxy)propan-2-one
- **(6):** Dihydroxyacetone phosphate
- **(7):** Intermediate in the pathway
- **(3):** Pyridoxine 5′-phosphate

---

### Footer

**FEBS 21861 12-4-99**

pounds coeluted with the respective standards without any sign of peak splitting. No significant amounts of fluorescent compounds were formed when DXP was replaced by DX in the reaction mixture. These data are taken as evidence for the formation of PNP by PdxJ from DXP and 2-amino-3-oxo-4-(phosphohydroxy)butyric acid (4) or 1-amino-3-(phosphohydroxy)propan-2-one (5). Under the experimental conditions described about 20% of the DXP supplied was transformed into PNP.

To further characterise the enzymatically formed PNP and PN, 200 μl aliquots of the untreated and alkaline phosphatase-treated reaction mixtures were subjected to HPLC and the fluorescent compounds collected. UV and fluorescence spectroscopy at different pH values revealed a close correlation of the absorption maxima and minima of the biosynthetic compounds with the corresponding values determined from authentic material and with literature data (Table 1).

Finally, the experiment was repeated on a small scale with [2-¹⁴C]DXP as the substrate. When the untreated and alkaline phosphatase-treated reaction mixtures were subjected to HPLC, radioactivity was detected by liquid scintillation counting in the fractions containing PNP and PN, respectively. Based on the specific radioactivity of the [2-¹⁴C]DXP used as the substrate, 39 nmol of PNP was formed from 150 nmol of DXP in an 0.3 ml reaction mixture upon an 80 min incubation in the presence of 0.14 mg PdxJ. Thus, a 26% conversion of the DXP employed into PNP was achieved under the described experimental conditions.

In summary, we could demonstrate for the first time that incubation of PdxJ with 1-deoxy-d-xylulose-5-phosphate in the presence of PdxA, 4-(phosphohydroxy)-L-threonine, and NAD results in the formation of PNP. 1-Deoxy-d-xylulose, on the other hand, is not used as a substrate.

On the basis of the above results, the reaction mechanism shown in Scheme 2 is proposed for the last steps in the biosynthesis of PNP (3). Schiff base formation between DXP and 5, followed by elimination of water yields the carbinolamine (7). The ring closure reaction between the carbon atoms destined to become C-4 and C-5 of PNP is facilitated by elimination of the good leaving group inorganic phosphate from 7. Elimination of water finally yields PNP (3).

---

**Fig. 1. Fluorescence elution profiles of HPLC separations of (A) 0.16 nmol of PNP and 0.31 nmol PN standards, (B) 5 μl of the reaction mixture for PNP biosynthesis and (C) 25 μl of the alkaline phosphatase-treated reaction mixture for PNP biosynthesis.**

---

**Table 1: Comparison of the absorption maxima (nm) of enzymatically synthesised and HPLC-purified PNP and PN with authentic material and literature values as determined by (A) UV spectroscopy and (B) fluorescence emission spectroscopy**

| PNP       | This work | Authentic | Literature | PN        | This work | Authentic | Literature |
|-----------|-----------|-----------|------------|-----------|-----------|-----------|------------|
| A         |           |           |            |           |           |           |            |
| pH 3.5    | 290       | 291       | 290        |           | 290       | 290       | 291        |
| pH 7.5    | 325       | 325       | 325        |           | 324       | 324       | 324        |
| pH 10     | 253       | 251       | 253        |           | 255       | 253       | 252        |
|           | 307       | 309       | 310        |           | 312       | 310       | 309        |
|           | 244       | 244       | 245        |           | 244       | 244       | 244        |
|           | 209       |           |            |           | 216       | 209       | 209        |
| B         |           |           |            |           |           |           |            |
| pH 3.5    | 394.5     | 395.5     |            |           | 394.0     | 392.0     |            |
| pH 7.5    | 392.5     | 394       |            |           | 392.5     | 392.0     |            |
| pH 10     | 375.0     | 375.0     |            |           | 374.5     |           | 375.5      |

Literature values are from [9] and [17] for PNP and PN, respectively. Emission spectra were recorded at excitation wavelengths of 290, 325, and 307 nm for pH 3.5, 7.5, and 10, respectively.

References

[1] Kennedy, I.A., Hill, R.E., Pauloski, R.M., Sayer, B.G. and Spenser, I.D. (1995) J. Am. Chem. Soc. 117, 1661–1662.

[2] Hill, R.E., Himmeldirk, K., Kennedy, I.A., Pauloski, R.M., Sayer, B.G., Wolf, E. and Spenser, I.D. (1996) J. Biol. Chem. 271, 30426–30435.

[3] Hill, R.E., Sayer, B.G. and Spenser, I.D. (1989) J. Am. Chem. Soc. 111, 1916–1917.

[4] Zhao, G.S. and Winkler, M.E. (1996) FEMS Microbiol. Lett. 135, 275–280.

[5] Notheis, C., Drewke, C. and Leistner, E. (1995) Biochim. Biophys. Acta 1247, 265–271.

[6] Zhao, G. and Winkler, M.E. (1995) J. Bacteriol. 177, 833–891.

[7] Cane, D.E., Hsiung, Y.J., Cornish, J.A., Robinson, J.K. and Spenser, I.D. (1998) J. Am. Chem. Soc. 120, 1936–1937.

[8] Laber, B., Gerbling, K.P., Harde, C., Neff, K.H., Nordhoff, E. and Pohlenz, H.D. (1994) Biochemistry 33, 3413–3423.

[9] Argoudelis, C.J. (1986) J. Agric. Food Chem. 34, 995–998.

[10] Skerra, A. (1992) Nucleic Acids Res. 20, 3551–3554.

[11] Yanisch-Perron, C., Vieira, J. and Messing, J. (1985) Gene 33, 103–119.

[12] Schmidt, T.G.M., Koepke, J., Frank, R. and Skerra, A. (1996) J. Mol. Biol. 255, 753–766.

[13] Skerra, A. (1994) Gene 151, 131–135.

[14] Sprenger, G.A. et al. (1997) Proc. Natl. Acad. Sci. USA 94, 12857–12862.

[15] Lois, L.M., Campos, N., Putra, S.R., Danielsen, K., Rohmer, M. and Boronat, A. (1998) Proc. Natl. Acad. Sci. USA 95, 2105–2110.

[16] Tsuge, H. (1997) Methods Enzymol. 280, 3–12.

[17] Johnson, R.J. and Metzler, D.E. (1970) Methods Enzymol. 18a, 433–471.
