
Molecular Cell, Vol. 3, 771–780, June, 1999, Copyright ©1999 by Cell Press

# The Crystal Structure of the Human Hepatitis B Virus Capsid

S. A. Wynne, R. A. Crowther, and A. G. W. Leslie*

Medical Research Council  
Laboratory of Molecular Biology  
Hills Road  
Cambridge CB2 2QH  
United Kingdom

## Summary

Hepatitis B is a small enveloped DNA virus that poses a major hazard to human health. The crystal structure of the T = 4 capsid has been solved at 3.3 Å resolution, revealing a largely helical protein fold that is unusual for icosahedral viruses. The monomer fold is stabilized by a hydrophobic core that is highly conserved among human viral variants. Association of two amphipathic α-helical hairpins results in formation of a dimer with a four-helix bundle as the major central feature. The capsid is assembled from dimers via interactions involving a highly conserved region near the C terminus of the truncated protein used for crystallization. The major immunodominant region lies at the tips of the α-helical hairpins that form spikes on the capsid surface.

## Introduction

Hepatitis B virus (HBV) is a human pathogen of major significance, as chronic infection can lead to cirrhosis and primary liver cancer, resulting in over a million deaths worldwide each year. The virus belongs to the *Hepadna* family and is composed of an outer envelope of host-derived lipid containing the surface proteins, and an inner protein capsid that contains the genomic DNA (for review, see Nassal and Schaller, 1993). The capsid is composed of a single polypeptide chain, molecular weight 20 kDa, known as the core antigen, HBcAg. When the gene for the core antigen is expressed in *Escherichia coli*, the expressed protein assembles into shells, which, when visualized by electron microscopy, are indistinguishable from core shells isolated from virally infected liver (Cohen and Richmond, 1982).

Previous work has established that the bacterially expressed protein assembles to give two different sizes of shell, composed of 180 or 240 subunits arranged with T = 3 or T = 4 icosahedral symmetry, respectively, and with subunits paired to produce dimeric spikes protruding from the surface of the shell (Crowther et al., 1994). The capsid protein contains a very basic C-terminal sequence that is believed to interact with the viral nucleic acid. Removal of this C-terminal sequence by truncation at residue 149 leads to higher levels of bacterial expression, and a predominance of the T = 4 capsid, with no obvious change in the morphology of the particle (Zlotnick et al., 1996).

The structure of the T = 4 capsid containing 240 copies of the bacterially expressed truncated protein (aa 1–149) has been determined at 7.4 Å resolution by single-particle electron cryomicroscopy (Böttcher et al., 1997). The polypeptide fold was apparent in the resulting map, showing that the truncated protein is largely α-helical. In particular, the dimer spikes visible on the surface of the capsid in studies at lower resolution were shown to be composed of a four-helix bundle (Böttcher et al., 1997; Conway et al., 1997). A tentative sequence assignment was made that placed the primary antigenic site at the tip of the spike and was consistent with other biochemical and immunological observations (Böttcher et al., 1997). Subsequently, cryomicroscopy and difference imaging have been used to locate the N terminus by a peptide extension (Böttcher et al., 1998a; Conway et al., 1998b) and the C terminus by gold labeling (Zlotnick et al., 1997).

The T = 4 capsids corresponding to a particular primary patient isolate (CW) but truncated after aa 149 have been crystallized (Wynne et al., 1999). The crystal structure has now been solved at 3.3 Å resolution, confirming the basic fold deduced by electron cryomicroscopy but adding a significant level of detail. In particular, it is apparent that a conserved hydrophobic core is responsible for stabilization of the monomer fold, while residues 120–143 form crucial interactions that stabilize the structure of the capsid. The precise location and conformation of the previously mapped major antigenic site have also been revealed.

## Results and Discussion

### Structure Determination

Crystals of the T = 4 HBV capsids belong to space group C2, with unit cell dimensions a = 538.4 Å, b = 354.8 Å, c = 370.1 Å, β = 132.3°. Diffraction data to 3.3 Å resolution were collected from cryocooled crystals at 100 K on beamline ID2 of the ESRF in Grenoble (Table 1). Data from 20 crystals were merged to give a data set that was 95% complete to 3.3 Å. The quality of the diffraction varied significantly from crystal to crystal, and a separate resolution limit was applied to each crystal based on the results of initial scaling and merging. Self-rotation functions showed the expected icosahedral symmetry operators and confirmed that an icosahedral two-fold axis was coincident with the crystallographic two-fold (Wynne et al., 1999). This is consistent with the presence of one half of the viral capsid (120 subunits) per asymmetric unit of the C2 unit cell, as deduced from packing arguments. As the choice of origin along the b axis is arbitrary in this space group, the problem of placing the particle in the unit cell is reduced to determining the azimuthal orientation of the shell around the crystallographic two-fold axis. This angle was obtained from the κ = 180° section of the self-rotation function and is consistent with the positions of the other

Molecular Cell
772

Table 1. Crystallographic Statistics

**Data Quality**
- Resolution (Å): 38.8–3.3
- Completeness (%): 94.9 (78.2)
- Multiplicity: 2.9 (2.0)
- $R_{\text{merge}}$ (%): 15.3 (79.3)
- Mean I/σ(I): 6.4 (0.9)

**Phase Extension Statistics**
- Correlation Coefficient
- R factor (%)
- Initial EM model (7.5 Å): 0.785, 31.5
- After extension to 5.5 Å: 0.954, 16.6
- After extension to 3.3 Å: 0.956, 19.6

**Refinement Statistics**
- Resolution (Å): 8–3.3
- Number of reflections: 597999
- Number of independent protein atoms: 4560
- R factor (%): 27.1
- rmsd bond length (Å): 0.016
- rmsd bond angles (degrees): 2.02
- Mean atomic temperature factor (Å²): 54.2

**Statistics as a Function of Resolution**
- Resolution (Å): 10.4, 7.4, 6.0, 5.2, 4.7, 4.3, 3.9, 3.7, 3.5, 3.3
- $R_{\text{merge}}$ (%): 7.5, 7.5, 11.1, 13.5, 15.0, 18.6, 24.6, 38.5, 56.7, 79.3
- Correlation coefficient: 0.938, 0.976, 0.949, 0.937, 0.926, 0.906, 0.876, 0.836, 0.759, 0.575

a Values in parentheses are for the highest resolution shell.
b For the initial EM model, the correlation is between observed native F's and those calculated from the Fourier transform of the EM model. During phase extension, the correlation is between native F's and those calculated from the Fourier transform of the icosahedrally averaged and solvent-flattened electron density.
c R factors are based on the same structure factor amplitudes used in calculating the correlation coefficient.

icosahedral operators. The 7.4 Å resolution electron cryomicroscopy map was used to generate initial phases to 8 Å resolution, and these were extended to 3.3 Å resolution by exploiting the 30-fold noncrystallographic symmetry (Table 1). The path of the polypeptide backbone in the resulting electron density map (Figure 1) was very clear for residues 1–142 in all subunits. No density was observed for the six C-terminal residues (144–149) present in the expressed protein, and there was a small break in the density at the tip of the spikes. It was apparent that the α helices in the initial electron density map were left handed, showing that the wrong hand had been selected for the EM map used for initial phasing (this hand was based on the more probable hand for the four-helix bundle). The hand of the crystallographic map was therefore inverted prior to model building.

![Figure 1. The Experimental Electron Density Map](<image>)

Stereo view of a portion of the experimental electron density map following phase extension to 3.3 Å, showing parts of the α-helical hairpins in subunits C and D, with the final refined model superimposed.

Structure of the Hepatitis B Capsid
773

ing and now agrees with the hand determined by Conway et al. (1998a).

The icosahedral repeat unit in the T = 4 HBV capsid is made up of four independent subunits. A model for amino acids 1–142 of subunits A and C, and 1–143 of B and D (in the subunit notation of Zlotnick et al. [1996]; see Figure 5), was built and refined with XPLOR, imposing strict icosahedral symmetry. Several rounds of manual rebuilding and refinement resulted in a final model with good stereochemistry and an R factor of 27.1%. Refinement statistics are presented in Table 1.

### Fold of the Monomer

The monomer fold (Figure 2) is in excellent agreement with that obtained by electron cryomicroscopy (Böttcher et al., 1997) except for the details of the C-terminal region. The residue assignment of the cryo EM structure is also remarkably accurate, being within two or three residues of the X-ray structure. As expected from both CD spectra and electron microscopy, the structure is largely helical. This is unusual for coat proteins of icosahedral viruses, which more typically form a jellyroll or β barrel structure, although the recently determined structures of HIV-1 capsid protein (Momany et al., 1996; Gamble et al., 1997) and the VP3 subunit of blue tongue virus (Grimes et al., 1998) are also largely helical. The structure of the HBV capsid protein is dominated by a long α-helical hairpin consisting of residues 50–73 (α3) and 79–110 (α4), but the α4 helix is kinked between residues 90 and 92, so that the lower part of this helix is splayed away from the α3 helix. The N-terminal region (aa 1–12) adopts an irregular structure, followed by a short helix, α1 (aa 13–17), and a longer helix, α2 (aa 27–43, kinked after residue 37), which together surround three sides of the α-helical hairpin. The C-terminal region consists of helix α5 (aa 112–127) followed by an irregular proline-rich loop (aa 128–136) and an extended strand (aa 137–142/3).

There is an extensive hydrophobic core made up principally of residues located on α1, the loop connecting α1 and α2, α4b, and α5 (Figure 3). These residues are highly conserved in human variants (Figure 2c), suggesting that they play an important role in maintaining the stability of the monomer fold.

### Dimer Formation

Two monomers associate to give a compact dimer (Figure 4) in which the two α-helical hairpins pack to form a four-helix bundle. As recognized in the electron cryomicroscopy model, it is this four-helix bundle that forms the characteristic spikes on the surface of the capsid. The overall shape of the dimer resembles a pickaxe, with the four-helix bundle corresponding to the shaft and the basal region corresponding to the head. The N-terminal arm of each monomer packs against the opposing subunit. Each helical hairpin is highly amphipathic, and the hydrophobic faces associate on dimer formation, burying a surface of 2000 Å² (63% hydrophobic), which helps to account for the high stability of the dimer unit.

At the dimer interface, there is a disulfide bridge between the Cys-61 residues of the two monomers, as identified previously by biochemical studies (Nassal et

![Figure 2](attachment://figure_2.png)

**Figure 2. Structure and Sequence of the HBV Capsid Monomer**

A schematic representation of the fold of the HBV capsid protein monomer derived from the crystal structure (a) and from electron cryomicroscopy (b). The cryo EM fold is taken directly from Böttcher et al. (1997), but the hand has been inverted to match that of the X-ray structure. The dimer interface is nearest the viewer. (c) The amino acid sequence of the CW variant of HBV capsid, truncated at amino acid 149 for this study (indicated by Δ). The positions of the helices are shown. The 94 full-length human HBV capsid sequences in the SWISS-PROT and TrEMBL databases (Bairoch and Apweiler, 1998) were aligned using CLUSTALW. Fully conserved residues (identical in all 94 sequences) are marked (*).

Molecular Cell
774

![Figure 3. Hydrophobic Core of the Monomer](image.png)

A stereo view of the monomer showing the hydrophobic core, formed by residues Tyr-6, Phe-9, Leu-15, Leu-16, Phe-18, Leu-19, Phe-23, Phe-24, Trp-102, Phe-103, Val-115, Tyr-118, Leu-119, Phe-122, Trp-125, and Leu-140, some of which are labeled. The dimer interface is away from the viewer.

has also been observed (Zheng et al., 1992), although high concentrations of oxidant are required to achieve complete formation of this cross-link (Nassal et al., 1992). However, Cys-48 is not located at the dimer interface, and the distance between Cys-48 residues is greater than 20 Å both for dimer-related subunits and for subunits related by icosahedral symmetry, precluding disulfide bond formation in the native shell.

Both independent dimers have almost exact two-fold symmetry. Superposition of the alpha carbons shows that the A and B subunits are related by a pure rotation of 178°, while C and D subunits are related by a 178° rotation and a translation of 0.3 Å. In the T = 3 capsid, 30 dimers lie on strict icosahedral two-fold axes and must have perfect diad symmetry, while the remaining 60 lie on local two-fold axes.

The four-helix bundle in the HBV capsid dimer is a more open structure than, for example, that found in the rop dimer (Banner et al., 1987). Only two of the four helices (α3 from the two monomers) are involved in extensive contacts, and there is almost no contact between the two α4 helices except at the very tip of the bundle. There is a closer similarity to the four-helix bundle formed by the dimerization of the Spo0B phosphotransferase (Varughese et al., 1998), although this bundle has the opposite hand and differs in detail from that in the HBV dimer. A database search using DALI (Holm and Sander, 1993) did not reveal any significant structural homology to other viral proteins (the closest structure was seryl tRNA synthetase). In particular, the similarity to HIV-1 capsid protein suggested by Zlotnick et al. (1998) was not apparent.

**Capsid Assembly**

In a T = 4 virus, there are four independent copies of the coat protein, which, on application of the icosahedral symmetry operations, generate a complete capsid (Figure 5). The capsid consists of a thin spherical protein shell, with an inner radius of 130 Å and a thickness of 20 Å, from which the four-helix bundles protrude, forming spikes approximately 25 Å in length and 20 Å in width. The extent of the interactions between different dimers, as judged by the buried surface area, is similar for the interactions around the icosahedral five-fold and two-fold (quasi six-fold) axes, with between 600 and 800 Å² buried surface in each case. There is no suggestion from the structure that there is any stable intermediate other than the dimer in capsid assembly, in agreement with biochemical studies (Zhou and Standring, 1992).

The interactions between subunits around both the five-fold and two-fold axes (Figure 6) primarily involve residues in helix α5, the proline-rich loop (aa 128–136), and the C-terminal arm (aa 137–142/3). In particular, Tyr-132, which is fully exposed in an isolated dimer, becomes fully buried in the capsid. Arg-127, Pro-129, and Ile-139 also play important roles in intersubunit packing. Site-directed mutagenesis of Arg-127 to either leucine or glutamine dramatically reduces the yield of intact capsids when the modified protein is expressed in *E. coli* (König et al., 1998). The side chain of Trp-125 is not involved in intersubunit contacts but may be important in stabilizing the conformation of the proline-rich loop region by packing against Arg-133 (on one face) and Ala-137 (on the other).

The length of the expressed protein affects both the ability to form capsids, with a reported minimum length of 140 aa required for particle formation (Birnbaum and Nassal, 1990), and the ratio of T = 3 to T = 4 particles, which decreases steadily as the chain length increases from 140 to 149 aa (Zlotnick et al., 1996). The minimum chain length of 140 aa required for capsid assembly may be influenced by the structural role of the side chain of Leu-140, which is packed between the side chains of Phe-110 and Tyr-118, forming part of the hydrophobic core that stabilizes the monomer fold (Figure 3). The loss of this interaction is likely to modify the conformation of

the loop region involved in intersubunit interactions (aa 128–139) and could also affect the folding or stability of the entire monomer. There is, however, no analogous explanation for the effect of residues 143–149 on the relative proportions of T = 3 and T = 4 particles. There is no electron density for any of these residues (except aa 143 in subunits B and D), suggesting that they are conformationally disordered. It is therefore difficult to rationalize why their presence would favor formation of the T = 4 capsids, although it is possible that their influence results from destabilization of the T = 3 shell rather than stabilization of the T = 4 shell. In assembly of the whole virus, as opposed to expressed core protein shells, the size of capsid is likely to be set by polymerization around an initial complex of viral polymerase and pregenomic RNA.

Quasi Equivalence

The four monomers (A, B, C, and D) in the icosahedral unit have very similar folds and superimpose with rms deviations of between 0.5 Å and 1.2 Å in α-carbon positions. The similarity is even more marked if the basal tips corresponding to the C-terminal region are excluded, with rms deviations of between 0.3 Å and 0.6 Å for residues 1–120. For residues 1–142, subunits A and D are the most similar (rms deviation 0.5 Å), while subunit C shows the largest deviation from other subunits, (rms deviations of 1.0, 1.3, and 1.0 Å to subunits A, B, and D, respectively). Thus, the largest differences in conformation occur in the regions involved in interdimer interactions (particularly residues 128–136), while the remainder of the subunit is essentially invariant. In agreement with the principle of quasiequivalence (Caspar and Klug, 1962), the intersubunit packing is similar around the icosahedral five-fold and two-fold (quasi six-fold) symmetry axes.

Fenestration of the Capsid

As observed in the electron cryomicroscopy studies, the capsid is fenestrated, with large pores around the icosahedral two-fold (quasi six-fold), three-fold, and quasi three-fold axes. The dimensions of the pores are similar, about 14 Å in diameter around the three-fold and quasi three-fold axes and between 12 Å and 15 Å wide around the two-fold. However, the nature of the side chains forming the pores is quite different. The three-fold and quasi three-fold channels are lined with polar residues (Asp-2, Lys-7, Arg-39, Glu-40, Glu-43, and Glu-46) while the two-fold channel is more hydrophobic, being formed primarily by residues Pro-135 and Asn-136 from the six contributing subunits. Similar channels have been observed in the capsids of ΦX174 (McKenna et al., 1992), FMDV (Acharya et al., 1989), and MS2 (Vallegård et al., 1990), although in all these cases the channels were centered on the icosahedral five-fold axes. In the HBV capsid, the close association of the C-terminal residues results in a pore of only 3 Å diameter at the position of the five-fold axis.

As reverse transcription of the viral pregenomic RNA to partially double-stranded DNA occurs within the assembled capsid, it has been proposed that the pores are essential to allow nucleotides access to the capsid interior during DNA synthesis. Equally, they may allow

Molecular Cell
776

![](image1.png)

Figure 5. Packing of Dimers in the T = 4 Capsid

(a) A schematic representation of the arrangement of the four independent subunits in the T = 4 capsid, showing one icosahedral face. The subunit nomenclature follows Zlotnick et al. (1996). The positions of the four-helix bundles are shown by black circles.

(b) Stereo view of the packing of the dimers in the T = 4 capsid in the region of a quasi three-fold axis. Subunits A, B, C, and D are shown in green, yellow, red, and blue as in Böttcher et al. (1997). The positions of the icosahedral five-fold (5), three-fold (3), quasi three-fold (q3), and two-fold (2) axes are shown.

(c) The capsid viewed down an icosahedral three-fold axis.

around the two-fold axes would be involved. The absence of interpretable electron density for residues 143/4-149 in the crystal structure supports the notion that this may be a flexible hinge region, although this could also be a consequence of truncation at aa 149. Some aspect of core protein flexibility may be important in signaling the maturation of the nucleocapsid prior to envelopment.

Antigenic Sites

Hepatitis B capsids are highly antigenic, eliciting a strong B cell, T helper cell, and cytotoxic T cell response. The major immunodominant region that elicits a strong antibody response in human infections has been mapped to a region around residue 80 (Salfeld et al., 1989; Sällberg et al., 1991). This region lies very close to the tips

of the four-helix bundle (Figure 7), where residues Asp-78, Pro-79, Ala-80, and Ser-81 form a compact array of side chains at the N-terminal end of helix α4a. The position of the cluster close to the local two-fold axis implies that, because of steric blocking, a dimer spike has the capacity for binding only one Fab fragment at a time. This has been shown to be the case by electron cryomicroscopy studies of hepatitis B capsids decorated with Fab fragments prepared from monoclonal antibodies (Conway et al., 1998a). Core shells are also very effective as carriers for foreign epitopes (reviewed in Ulrich et al., 1998; Pumpens and Grens, 1999). Insertions as large as 46 amino acids between residues 78 and 79 have been accommodated without significantly affecting particle formation (Borisova et al., 1993). This insertion site is displaced slightly to one side of the tip (Figure 7), which

Structure of the Hepatitis B Capsid
777

a

b
B
C
D
D
C
B

Figure 6. Dimer-Dimer Interactions in the Capsid
A stereo view from outside the capsid of the interactions between adjacent subunits around (a) the five-fold (A subunits) and (b) the two-fold (quasi six-fold) axes (B, C, and D subunits). In each case, the polypeptide chain is color ramped from blue (aa 115) to red (aa 142). Residues that play an important role in intersubunit contacts or in stabilizing the conformation of the proline-rich loop (aa 128–136) are indicated. For clarity, the side chains of residues 115–121 have been omitted.

may be advantageous as it could reduce steric hindrance between the inserted sequences. The four-helix bundle clearly provides a convenient and stable structural scaffold for the insertion and display of foreign epitopes.
A secondary epitope has been identified around residue 130 (Salfeld et al., 1989), and monoclonal antibodies against this region label well-preserved shells (Pushko et al., 1994; Kenney et al., 1995). This region corresponds to the loop between helix α5 and the C-terminal extended strand and is located close to the five-fold and two-fold icosahedral symmetry axes in the capsid (Figure 6). The side chain of Pro-130 is fully solvent-exposed, pointing away from the surface of the capsid and could contribute to an antigenic site. Steric considerations also suggest that any antigenic site on the surface of the capsid (other than on the tips) must lie close to the five-fold or two-fold axes, as the separation between

adjacent dimer spikes is only 50 Å and the tips are 20 Å in width. The size of a V<sub>L</sub>/V<sub>H</sub> dimer and the central position of the antibody-combining site would effectively preclude an interaction with the capsid surface at any site other than the relatively open regions around the five-fold and two-fold axes. The scale of the surface topography is such that only one or possibly two Fab fragments could bind around any particular five-fold or local six-fold axis.

Peptide Binding and Capsid Envelopment
Small peptides selected to bind to the core and that, in subsequent assay, block interaction with surface antigen (Dyson and Murray, 1995) have been shown by electron cryomicroscopy to bind at the tips of the spikes (Böttcher et al., 1998b). The peptides contain a common pattern of basic residues, and changing Glu-77 or Asp-78 of the core protein to alanine greatly reduced the

Molecular Cell
778

![Image Description](image1.png)

Figure 7. The Major Immunodominant Region Stereo view of the tip of the $\alpha$-helical hairpin of subunit C, with the dimer interface away from the viewer. The major immunodominant region has been mapped to residues 78–83. Foreign sequences of up to 46 aa can be inserted between residues 78 and 79 without preventing capsid assembly.

affinity of binding of the peptide (Böttcher et al., 1998b). The high-resolution structure shows that residues Glu-77 and Asp-78 form the extreme tip of the double-helical hairpin, with the acidic side chains fully exposed (Figure 7). It is thus likely that binding of peptide to one core protein monomer would sterically block binding of another copy of the peptide to the two-fold related subunit, resulting in binding of only one peptide per dimer spike, in accordance with the stoichiometry of chemical cross-linking (Böttcher et al., 1998b). The tips of the spikes are likely to form at least part of the specific interface between the core protein and the surface antigen. This is consistent with the observation of a cluster of conserved residues (Figure 2c) on the surface of the dimer spike, suggesting that it may play an important role both in the assembly of the virus and in the stability of the resulting particle.

## Experimental Procedures

### Protein Expression and Crystallization
Crystals were obtained as described in Wynne et al. (1999). The truncated core protein (aa 1–149) of a hepatitis B strain CW isolated at St. Mary’s Hospital, London, was expressed in *E. coli*. The protein was purified by ammonium sulfate precipitation and column chromatography, and the two sizes of capsid (with T = 3 and T = 4 icosahedral symmetry) were separated by ultracentrifugation on a sucrose gradient. Crystals were grown at 21°C by the hanging drop method. Two microliters of protein (15–20 mg/ml) in 5 mM Tris-HCl, 150 mM NaCl (pH 7.5) were added to an equal volume of 0.1 M MES (pH 6.5–7.0), 0.1–0.4 M $(\mathrm{NH}_{4})_{2} \mathrm{SO}_{4}$, 3.5%–4% PEG 20,000, and 20% butanediol and equilibrated against a reservoir of the same composition. The crystals belong to the space group C2 with unit cell dimensions $a = 538.4 \AA$, $b = 354.8 \AA$, $c = 370.1 \AA$, $\beta = 132.3^\circ$.

### Data Collection and Processing
Diffraction data were collected on beamline ID2 at the ESRF, Grenoble, using a Mar Research Mar345 image plate scanner and radiation of $\lambda = 0.9919 \AA$. The crystals were flash frozen to 100 K in the $\mathrm{N}_2$ gas stream. Twenty crystals were used to collect approximately $90^\circ$ of data to 3.3 Å resolution, using a rotation angle of $0.3^\circ$ per image. More than one position was exposed on the larger crystals, but the quality of diffraction was inferior to that obtained from a fresh crystal. Diffraction images were processed with MOSFLM (Leslie, 1992), and data reduction was performed with programs from the CCP4 suite (Collaborative Computational Project Number 4, 1994).

### Initial Phasing and Phase Extension
Self-rotation functions were calculated using the program GLRF (Tong and Rossmann, 1997) using an integration radius of 140 Å. The $\kappa = 180^\circ$ section confirmed that an icosahedral two-fold axis is coincident with the crystallographic two-fold and also gave the orientation of the particle in the C2 unit cell (Wynne et al., 1999). Initial phases to 8 Å resolution were generated using the 7.4 Å resolution map determined by electron cryomicroscopy (Böttcher et al., 1997), which was transformed into the crystallographic unit cell using a modified version of the CCP4 program MAPROT. The overall scale and orientation of the EM map, the level for the solvent density, and the solvent content were optimized by maximizing the correlation between the observed structure factor amplitudes and those calculated from Fourier inversion of the model, using data to 7.5 Å resolution. The phases were then extended by exploiting the 30-fold noncrystallographic symmetry initially to 5.5 Å (in 80 cycles of averaging and phase extension with a map grid interval of 2.1 Å) and then to 3.3 Å in a further 150 cycles (using a smaller map grid interval of 1.2 Å). The envelope used for averaging was derived from the 7.4 Å resolution cryo EM map and was not modified during the phase extension. The CCP4 program SOLOMON (Abrahams and Leslie, 1996) was used for averaging, and map coefficients were calculated using SIGMAA (Read, 1986). Details of the structure solution will be published elsewhere.

### Model Building, Refinement, and Assessment
A model for the four independent subunits was built into the 3.3 Å resolution map using O (Jones et al., 1991). This model was refined using torsion angle dynamics in XPLOR 3.8 (Rice and Brünger, 1994) with an initial temperature of 2500 K, followed by group temperature refinement and conventional positional refinement. Strict 30-fold icosahedral symmetry was applied during all stages of refinement. Several rounds of alternating rebuilding and refinement gave a final model with an R factor of 27.1% and good stereochemistry. The geometry of the model was assessed with PROCHECK (Laskowski et al., 1993), and the Ramachandran plot shows 78% of all residues in the most favored regions and none in disallowed regions. Accessible surface area calculations and coordinate superpositions were performed with programs from the CCP4 suite. Figures were produced using BOBSCRIPT (Esnouf, 1997).

### Acknowledgments
We thank Kerstin Braig and the staff at beamline ID2, ESRF, Grenoble for assistance with data collection, and colleagues at the Laboratory of Molecular Biology for critical reading of the manuscript.

Received March 3, 1999; revised April 19, 1999.

### References
- Abrahams, J.P., and Leslie, A.G.W. (1996). Methods used in the structure determination of bovine mitochondrial F1-ATPase. Acta Crystallogr. D 52, 30–42.
- Acharya, R., Fry, E., Stuart, D., Fox, G., Rowlands, D., and Brown, F. (1989). The three-dimensional structure of foot-and-mouth disease virus at 2.9Å resolution. Nature 337, 709–716.
- Bairoch, A., and Apweiler, R. (1998). The SWISS-PROT protein sequence data bank and its supplement TrEMBL in 1998. Nucleic Acids Res. 26, 38–42.
- Banner, D.W., Kokkinidis, M., and Tsernoglou, D. (1987). Structure of the ColE1 ROP protein at 1.7Å resolution. J. Mol. Biol. 3, 657–675.
- Birnbaum, F., and Nassal, M. (1990). Hepatitis B virus nucleocapsid

assembly: primary structure requirements in the core protein. J. Virol. 64, 3319–3330.

Borisova, G.P., Berzins, I., Pushko, P.M., Pumpen, P., Gren, E.J., Tsibinogin, V.V., Loseva, V., Ose, V., Ulrich, R., Siakkou, H., and Rosenthal, H.A. (1989). Recombinant core particles of hepatitis B virus exposing foreign antigenic determinants on their surface. FEBS Lett. 259, 121–124.

Borisova, G., Arya, B., Dislers, A., Borschukova, O., Tsibinogin, V., Skrastina, D., Eldarov, M.A., Pumpens, P., S kryabin, K.G., and Grens, E. (1993). Hybrid hepatitis B virus nucleocapsid bearing an immunodominant region from hepatitis B virus surface antigen. J. Virol. 67, 3696–3701.

Böttcher, B., Wynne, S.A., and Crowther, R.A. (1997). Determination of the fold of the core protein of hepatitis B virus by electron cryomicroscopy. Nature 386, 88–91.

Böttcher, B., Dyson, M.R., and Crowther, R.A. (1998a). Finding the small difference: a nine amino acid extension to the hepatitis B core protein. Proc. Intl. Congr. Electr. Microsc. 14, 737–738.

Böttcher, B., Tsuji, N., Takahashi, H., Dyson, M.R., Zhao, S., Crowther, R.A., and Murray, K. (1998b). Peptides that block hepatitis B virus assembly: analysis by cryomicroscopy, mutagenesis and transfection. EMBO J. 17, 6839–6845.

Caspar, D.L.D., and Klug, A. (1962). Physical principles in the construction of regular viruses. Cold Spring Harb. Symp. Quant. Biol. XXVII, 1–24.

CCP4 (1994). Programs for protein crystallography. Acta Crystallogr. D 50, 760–763.

Cohen, B.J., and Richmond, J.E. (1982). Electron microscopy of hepatitis B core antigen synthesized in *E. coli*. Nature 296, 677–678.

Conway, J.F., Cheng, N., Zlotnick, A., Wingfield, P.T., Stahl, S.J., and Steven, A.C. (1997). Visualization of a 4-helix bundle in the hepatitis B virus capsid by cryo-electron microscopy. Nature 386, 91–94.

Conway, J.F., Cheng, N., Zlotnick, A., Stahl, S.J., Wingfield, P.T., Belnap, D.M., Kannegiesser, U., Noah, M., and Steven, A.C. (1998a). Hepatitis B virus capsid: localization of the putative immunodominant loop (residues 78 to 83) on the capsid surface, and implications for the distinction between c and e-antigens. J. Mol. Biol. 279, 1111–1121.

Conway, J.F., Cheng, N., Zlotnick, A., Stahl, S.J., Wingfield, P.T., and Steven, A.C. (1998b). Localization of the N terminus of hepatitis B virus capsid protein by peptide-based difference mapping from cryoelectron microscopy. Proc. Natl. Acad. Sci. USA 95, 14622–14627.

Crowther, R.A., Kiselev, N.A., Böttcher, B., Berriman, J.A., Borisova, G.P., Ose, V., and Pumpens, P. (1994). Three-dimensional structure of hepatitis B virus core particles determined by electron cryomicroscopy. Cell 77, 943–950.

Dyson, M.R., and Murray, K. (1995). Selection of peptide inhibitors of interactions involved in complex protein assemblies: association of the core and surface antigens of hepatitis B virus. Proc. Natl. Acad. Sci. USA 92, 2194–2198.

Esnouf, R.M. (1997). An extensively modified version of Molscript that includes greatly enhanced colouring capabilities. J. Mol. Graph. 15, 132–134.

Gamble, T.R., Yoo, S., Vajdos, F.F., von Schwedler, U.K., Worthylake, D.K., Wang, H., McCutcheon, J.P., Sundquist, W.I., and Hill, C.P. (1997). Structure of the carboxyl-terminal dimerization domain of the HIV-1 capsid protein. Science 278, 849–853.

Grimes, J.M., Burroughs, J.N., Gouet, P., Diprose, J.M., Malby, R., Zientara, S., Mertens, P.P.C., and Stuart, D.I. (1998). The atomic structure of the bluetongue virus core. Nature 395, 470–478.

Holm, L., and Sander, C. (1993). Protein structure comparison by alignment of distance matrices. J. Mol. Biol. 233, 123–138.

Jones, T.A., Zhou, J.Y., Cowan, S.W., and Kjeldgaard, M. (1991). Improved methods for building protein models in electron density maps and the location of errors in these models. Acta Crystallogr. A 47, 110–119.

Kenney, J.M., von Bonsdorff, C.-H., Nassal, M., and Fuller, S.D. (1995). Evolutionary conservation in the hepatitis B virus core structure: comparison of human and duck cores. Structure 3, 1009–1019.

König, S., Beterams, G., and Nassal, M. (1998). Mapping of homologous interaction sites in the hepatitis B virus core protein. J. Virol 72, 4997–5005.

Laskowski, R.A., MacArthur, M.W., Moss, D.S., and Thornton, J.M. (1993). A program to check the stereochemical quality of protein structures. J. Appl. Crystallogr. 26, 283–291.

Leslie, A.G.W. (1992). Recent changes to the MOSFLM package for processing film and image plate data. In Joint CCP4 and ESF-EACMB Newsletter on Protein Crystallography No. 26 (Warrington, UK: SERC, Daresbury Laboratory).

McKenna, R., Xia, D., Willingmann, P., Ilag, L.L., Krishnaswamy, S., Rossmann, M.G., Olson, N.H., Baker, T.S., and Incardona, N.L. (1992). Atomic structure of single-stranded DNA bacteriophage ΦX174 and its functional implications. Nature 355, 137–143.

Momany, C., Kovari, L.C., Prongay, A.J., Keller, W., Gitti, R.K., Lee, B.M., Gorbelenya, A.E., Tong, L., McClure, J., Ehrlich, L.S., et al. (1996). Crystal structure of dimeric HIV-1 capsid protein. Nat. Struct. Biol. 3, 763–770.

Nassal, M., and Schaller, H. (1993). Hepatitis B virus replication. Trends Microbiol. 1, 221–228.

Nassal, M., Rieger, A., and Steinau, O. (1992). Topological analysis of the hepatitis B virus core particle by cysteine-cysteine cross-linking. J. Mol. Biol. 225, 1013–1025.

Pumpens, P., and Grens, E. (1999). Hepatitis B core particles as a universal display model: a structure-function basis for development. FEBS Lett. 442, 1–6.

Pushko, P., Sällberg, M., Borisova, G., Rudén, U., Bichko, V., Wahren, B., Pumpens, P., and Magnus, L. (1994). Identification of hepatitis B virus core protein regions exposed or internalized at the surface of HBcAg particles by scanning with monoclonal antibodies. Virology 202, 912–920.

Read, R.J. (1986). Improved Fourier coefficients for maps using phases from partial structures with errors. Acta Crystallogr. A 42, 140–149.

Rice, L.M., and Brünger, A.T. (1994). Torsion angle dynamics: reduced variable conformational sampling enhances crystallographic structure refinement. Proteins 19, 277–290.

Salfeld, J., Pfaff, E., Noah, M., and Schaller, H. (1989). Antigenic determinants and functional domains in core antigen and e antigen from hepatitis B virus. J. Virol. 63, 798–808.

Sällberg, M., Rudén, U., Magnus, L.O., Harthus, H.P., Noah, M., and Wahren, B. (1991). Characterization of a linear binding site for a monoclonal antibody to hepatitis B core antigen. J. Med. Virol. 33, 248–252.

Schödel, F., Moriarty, A.M., Peterson, D.L., Zheng, J., Hughes, J.L., Will, H., Leturcq, D.J., McGee, J.S., and Milich, D.R. (1992). The position of heterologous epitopes inserted in hepatitis B virus core particles determines their immunogenicity. J. Virol. 66, 106–114.

Seifer, M., and Standring, D.N. (1994). A protease-sensitive hinge linking the two domains of the hepatitis B virus core protein is exposed on the viral capsid surface. J. Virol. 68, 5548–5555.

Tong, L., and Rossmann, M.G. (1997). Rotation function calculations with GLRF program. Methods Enzymol. 276, 594–611.

Ulrich, R., Nassal, M., Meisel, H., and Krüger, D.H. (1998). Core particles of hepatitis B virus as carrier for foreign epitopes. Adv. Virus Res. 50, 141–182.

Valegård, K., Liljas, L., Fridborg, K., and Unge, T. (1990). The three-dimensional structure of the bacterial virus MS2. Nature 345, 36–41.

Varughese, K.I., Madhusudan, Zhou, X.Z., Whiteley, J.M., and Hoch, J.A. (1998). Formation of a novel four-helix bundle and molecular recognition sites by dimerization of a response regulator phosphotransferase. Mol. Cell 2, 485–493.

Wynne, S.A., Leslie, A.G.W., Butler, P.J.G., and Crowther, R.A. (1999). Crystallisation of hepatitis B virus core protein shells, determination of cryoprotectant conditions and preliminary X-ray characterisation. Acta Crystallogr. D 55, 557–560.

Zheng, J., Schödel, F., and Peterson, D.L. (1992). The structure of hepadnaviral core antigens. J. Biol. Chem. 267, 9422–9429.

Zhou, S., and Standring, D.N. (1992). Hepatitis B virus capsid particles are assembled from core-protein dimer precursors. Proc. Natl. Acad. Sci. USA 80, 10046–10050.

Molecular Cell
780

Zlotnick, A., Cheng, N., Conway, J .F., Booy, F.P., Steven, A.C., Stahl, S.J., and Wingfield, P.T. (1996). Dimorphism of hepatitis B virus capsids is strongly influenced by the C-terminus of the capsid protein. Biochemistry 35, 7412–7421.

Zlotnick, A., Cheng, N., Stahl, S.J., Conway, J .F., Steven, A.C., and Wingfield, P.T. (1997). Localization of the C terminus of the assembly domain of hepatitis B virus capsid protein: implications for morphogenesis and organization of encapsidated RNA. Proc. Natl. Acad. Sci. USA 94, 9556–9561.

Zlotnick, A., Stahl, S.J., Wingfield, P.T., Conway, J .F., Cheng, N., and Steven, A.C. (1998). Shared motifs of the capsid proteins of hepadnaviruses and retroviruses suggest a common evolutionary origin. FEBS Lett. 431, 301–304.

Protein Data Bank Accession Number

The accession number for the coordinates of the structure reported in this paper is 1QGT.
