
Structure of a bacterial  
30S ribosomal subunit  
at 5.5 Å resolution  

William M. Clemons Jr*†, Joanna L. C. May*†, Brian T. Wimberly*†, John P. McCutcheon†, Malcolm S. Capel‡ & V. Ramakrishnan§  

† Department of Biochemistry, University of Utah School of Medicine, Salt Lake City, Utah 84103, USA  
‡ Biology Department, Brookhaven National Laboratory, Upton, New York 11973, USA  
§ Structural Studies Division, MRC Laboratory of Molecular Biology, Cambridge CB2 2QH, UK  
* These authors contributed equally to this work.

The 30S ribosomal subunit binds messenger RNA and the anticodon stem-loop of transfer RNA during protein synthesis. A crystallographic analysis of the structure of the subunit from the bacterium *Thermus thermophilus* is presented. At a resolution of 5.5 Å, the phosphate backbone of the ribosomal RNA is visible, as are the α-helices of the ribosomal proteins, enabling double-helical regions of RNA to be identified throughout the subunit, all seven of the small-subunit proteins of known crystal structure to be positioned in the electron density map, and the fold of the entire central domain of the small-subunit ribosomal RNA to be determined.

Translation of the genetic code on mRNA into a polypeptide chain takes place on the ribosome, a large nucleoprotein complex that consists of two subunits, one large and one small. In bacteria, the large and small ribosomal subunits are designated 50S and 30S, respectively, and together they form the 70S ribosome. Each ribosomal subunit is about two-thirds RNA by mass and consists of many different proteins. In particular, the small ribosomal subunit of *Escherichia coli* has a relative molecular mass of about $0.9 \times 10^6 (M_r = 900K)$ and consists of a piece of RNA 1,542 nucleotides long, designated 16S RNA, and 21 different proteins¹. The ribosome is an intricate molecular machine that is actively involved in the various steps of protein synthesis. Peptidyltransferase activity is associated with the 50S subunit, whereas the 30S subunit provides the framework for decoding genetic information, which involves detecting that a match has been made between the anticodon of tRNA and the codon on mRNA. The 30S subunit also has a sophisticated proofreading mechanism to minimize translation errors. During elongation of the peptide chain and translocation of the ribosome along the mRNA, there is a concerted movement of the mRNA and bound tRNA by precisely one codon relative to the ribosome. This movement must involve breaking and reformation of precise contacts, and is particularly interesting in the context of the 30S subunit because of its role in codon recognition. Both large and small ribosomal subunits are targets for several clinically relevant antibiotics.

Crystals of the 50S subunit were obtained that diffracted to beyond 3 Å (ref. 2), indicating that the determination of an atomic-resolution structure of a ribosomal subunit was possible in principle. Using a modification of earlier crystallization conditions³, we have obtained crystals of the 30S ribosomal subunit from *T. Thermophilus* that diffract to 3.6 Å resolution. These crystals are a significant improvement over those reported previously which diffracted to resolutions of 9–12 Å (refs 4, 5), but they are comparable to an improved form obtained independently⁶.

We present our current analysis of the structure of the 30S ribosomal subunit, based on an electron-density map at 5.5 Å

Table 1 Summary of crystallographic data and analysis

**Data-collection statistics**

| Crystal | Estimated mosaicity (deg) | Resolution limit (Å) | Completeness (%) | Redundancy | $R_{\text{sym}}$ |
|---------|---------------------------|-----------------------|------------------|-------------|-------------------|
| Native  | 0.56                      | 6.76                  | Overall: 95      | Overall: 20 | Overall: 0.048    |
|         |                           |                       | Outer shell: 92  |             | Outer shell: 0.079 |
| Os      | 0.33                      | 5.26                  | Overall: 95      | Overall: 5  | Overall: 0.081    |
|         |                           |                       | Outer shell: 83  |             | Outer shell: 0.196 |
| Yb      | 0.41                      | 6.01                  | Overall: 97      | Overall: 5  | Overall: 0.080    |
|         |                           |                       | Outer shell: 95  |             | Outer shell: 0.223 |
| Lu      | 0.58                      | 6.50                  | Overall: 97      | Overall: 6  | Overall: 0.078    |
|         |                           |                       | Outer shell: 90  |             | Outer shell: 0.234 |
| Ta      | 0.45                      | 7.00                  | Overall: 99      | Overall: 5  | Overall: 0.098    |
|         |                           |                       | Outer shell: 100 |             | Outer shell: 0.278 |
| W12Si   | 0.26                      | 6.00                  | Overall: 96      | Overall: 4  | Overall: 0.113    |
|         |                           |                       | Outer shell: 79  |             | Outer shell: 0.336 |
| W17     | 0.64                      | 6.75                  | Overall: 96      | Overall: 4  | Overall: 0.099    |
|         |                           |                       | Outer shell: 88  |             | Outer shell: 0.251 |

**Phasing statistics**

| Derivative | Number of sites | $R_{\text{cullis}}$ (centric) | Phasing power |
|------------|-----------------|-------------------------------|---------------|
|            |                 |                               | Isomorphous   |
|            |                 |                               | Anomalous     |
| Ta         | 3               | 0.73                          | 0.48          | 0.67        |
| W12Si      | 7               | 0.72                          | 0.51          | 0.72        |
| W17        | 2               | 0.72                          | 0.61          | 0.80        |
| Os         | 23              | -                             | -             | 1.4         |
| Lu         | 11              | 0.74                          | 0.47          | 0.64        |
| Yb         | 10              | 0.74                          | 0.48          | 0.64        |

Overall mean figure of merit (before solvent flattening): 0.56  

Os, osmium hexamine chloride; Yb, ytterbium chloride; Lu, lutetium chloride; Ta, $\mathrm{Ta_6Br_{12}}$; W12Si, $\mathrm{H_4SiO_4[12WO]}$; W17, $\mathrm{Li_{10}[P_2W_{17}O_{61}]}$.  

$R_{\text{cullis}} = (\text{lack of closure}) / (|F_{\text{ph}} - F_{\text{ph}}|)$.  

Phasing power = $(|F_{\text{h(calc)}}| / \text{lack of closure})$.

articles

![](image1.png)

Figure 1 Electron-density map of the 30S subunit at 5.5 Å resolution. Stereo view of a surface contoured at 1 s.d. displayed using RIBBONS<sup>50</sup>. H, head; P, platform; S, shoulder; B, body. The intersubunit interface is towards the reader.

![](image2.png)

![](image3.png)

Figure 2 Features visible at 5.5 Å resolution. **a**, Double-helical RNA corresponding to the 1400/1500 stem-loop of 16S RNA. Left, density with a right-handed double-helical twist with clear major and minor grooves, individual phosphate strands and bumps at the right separation for phosphate groups. Right, stereo

view of a fit of short stretches of A-form double helix to the density. The tetraloop at the end was fitted to the structure determined by NMR<sup>10</sup>. **b**, **c**, Fits of proteins of known crystal structure into the electron density. The panels show stereo views of: **b**, S6 (ref. 12); **c**, S15 (refs 15, 18). **d**, Protein-RNA complexes. The region shows S6 (red) and S15 (cyan) bound to double-helical RNA. **e**, Putative structure of S20, for which no high-resolution structure exists. Figure made using O<sup>49</sup>.

NATURE | VOL 400 | 26 AUGUST 1999 | www.nature.com

© 1999 Macmillan Magazines Ltd

resolution. At this resolution, double-helical RNA and α-helices and β-sheets of proteins can be identified. As a result, all seven 30S-subunit ribosomal proteins of previously known structure and a large fraction of the double-stranded RNA helices have been fitted into density. α-Helices of many of the proteins of unknown structure are also evident, as well as their interactions with neighbouring RNA. We have also determined the fold of the central domain of 16S rRNA, a region which includes the platform. This part of the 30S subunit is particularly important in its decoding, large-subunit associating and proof-reading activities. The structure helps to rationalize a large body of biochemical data and establishes a new framework for mechanistic studies of translation.

### Analysis of the structure

The structure was solved using multiple isomorphous replacement including anomalous scattering. An osmium derivative was used as the 'native' or reference data set because it had the best diffraction. Crystallization, data collection and phasing were done as described (see Methods), and the crystallographic data are summarized in Table 1. The final resolution of the map is about 5.5 Å, as judged by both phasing statistics and visual inspection of the map.

### Overall shape

The crystallographic asymmetric unit contains a single small subunit. As shown in Fig. 1, the small subunit in the crystal clearly exhibits the characteristic head, platform, shoulder and body

familiar from electron-microscopy studies of 30S subunits from *E. coli*⁷ or *T. Thermophilus*⁸. The dimensions of the subunit quantitatively agree with the results from these earlier electron-microscopy studies.

#### Features visible at 5.5 Å resolution

A long, protein-free stretch of double-helical density is visible at the subunit surface, seen running roughly vertically in Fig. 1. This long double helix is the penultimate stem-loop of 16S RNA (the 1,400/1,500 region), believed to be located at the interface between the two subunits⁹. At a resolution of 5.5 Å, the two strands of RNA are clearly distinguishable (Fig. 2a), so that the deep major and shallow minor grooves can be seen. The tetraloop at the end of this stem agrees with the conformation determined by nuclear magnetic resonance (NMR) methods¹⁰. In well ordered double-helical regions, individual phosphates appear as bumps in the density along each strand. Elsewhere in the map, single-stranded RNA is visible as tubes of backbone density joining double-stranded helices.

α-Helices of proteins appear as tubes at this resolution. Weaker density is also generally seen for β-sheets and even for well ordered loops. We can therefore fit individual proteins of known structure into the map, as shown for proteins S6 and S15 (Fig. 2b, c). Protein-RNA complexes can be identified for proteins of known structure involving double-helical RNA. An example is shown in Fig. 2d, where S6 and S15 make contact with the same stretch of double-helical RNA.

![Figure 3](#)

**Figure 3** The central domain of 16S RNA. **a**, Left, secondary-structure diagram of 16S RNA, with central domain shaded; right, expanded view of the central domain with individual helices coloured and the 690 and 790 loops indicated. **b, c**, Stereo views of the tertiary fold of the central domain, using the colouring scheme for the helices shown in **a**. Insets (right) show the corresponding views in the context of the 30S subunit. Figure made with RIBBONS⁵⁰.

Placement of proteins in the 30S subunit

We have been able to place all seven of the proteins whose high-resolution structure is known, namely S4, S5, S6, S7, S8, S15 and S17 (refs 11–21). In the absence of other information, the localization of known protein structures in such a large asymmetric unit would have been problematic even at this resolution. However, there are extensive data on the approximate location of the proteins in the 30S subunit, as well as their interactions with 16S RNA. The neutron map of the centres of mass of the 30S proteins²² particularly facilitated and accelerated the identification of density for specific proteins. Data from footprinting studies²³ and extensive biochemical data (summarized in ref. 24) also provide clues about the location of proteins in the subunit. Several of these proteins are known to be adjacent to or to bind to neighbouring regions of 16S RNA.

The placement of these seven known structures in the 5.5 Å map was compared with the neutron map by calculating a least-squares superposition of the neutron map into the observed centre-of-mass of the coordinates of the positioned proteins. When S15 is excluded from the comparison, the neutron map is in generally good agreement with the actual location observed here, with an r.m.s. deviation (r.m.s.d.) of 11.8 Å, which is comparable to the mean coordinate error in the neutron map²². However, this superposition places S15 roughly 51 Å from its position in the neutron map. The discrepancy could arise from a single distance measurement (S11–S15) in the neutron data, which has poor statistics²⁵. A definitive assessment of the overall quality of the neutron map must await the identification of the remaining proteins in the subunit.

We have also identified proteins S2, S3, S11, S16 and S18, whose

structures have not yet been determined, on the basis of α-helical density in the regions predicted for these proteins from biochemical and neutron-scattering data. At this point, their identification must be considered tentative. Near the bottom of the subunit, there is a three-helix bundle (Fig. 2e). This protein could be modelled in some detail as all of the loops can be traced in the electron density, and we have identified it as S20 on the basis of biochemical data²³,²⁶. Our model for the protein represents a new protein structure, albeit at low resolution, and is consistent with secondary-structure predictions based on the amino-acid sequence. However, our location for S20, although it is consistent with both biochemical data and a previous model²⁴, is not in agreement with the neutron map, which places S20 in the head of the subunit.

### The central domain of 16S RNA

The central domain of 16S RNA spans nucleotides 565–885 (*E. coli* sequence). Figure 3a shows the secondary structure of the central domain using the scheme adopted in ref. 23 and the helix numbering of ref. 27. This portion of the 16S RNA and its associated proteins form the platform and part of the body of the 30S subunit. The platform is a flexible and functionally important structure containing the 690 and 790 stem-loops, which have been identified as crucial for binding of P-site tRNA²⁸ and for subunit association²⁹,³⁰. At the 3′ end of the central domain is another functionally important region, H27, which has been proposed to function as an essential conformational switch that affects translational accuracy³¹. The central domain is also one of the biochemically best-characterized parts of 16S RNA. It contains the binding sites for ribosomal proteins S6, S8, S11, S15 and S18 (refs 23, 32, 33).

**Figure 4** Stereo view of the three-way junction formed by helices 20, 21 and 22 of the central domain of 16S RNA. Inset (right) shows the structure in the context of the 30S subunit. Figure made with RIBBONS⁵⁰.

**Figure 5** Stereo view of the interactions made by helix 27 of the central domain with helices 24 and 44 of 16S RNA. Inset (right) shows the elements in the whole 30S subunit. Figure made with RIBBONS⁵⁰.

particular, the binding to ribosomal RNA of S8 (refs 34, 35) and S15 (refs 36, 37) has been studied extensively.

After independent fitting of double-stranded RNA and proteins into electron density, a number of protein–RNA complexes were apparent, such as the adjacent S6–RNA and S15–RNA complexes (Fig. 2d). The extensive biochemical data on protein–RNA interactions in the central domain, the high density of proteins of previously known structure, and the well ordered density in this region allowed us to determine the fold of the entire central domain of 16S RNA. It also allowed us to examine details of its interaction with three proteins of known structure in this region, S6, S8 and S15. Furthermore, because of the known association of S6 and S18 in the absence of RNA and the proximity of S11 to S6, we could identify the location and low-resolution structure *in situ* of parts of these proteins (data not shown).

The central-domain RNA is an elongated, curved structure that wraps partly around the body of the small subunit (Fig. 3b, c). As seen elsewhere in the structure, the domain contains many short RNA helices packed together. A common mode of RNA–RNA packing involves the insertion of a ridge of phosphates into the minor groove of an adjacent double helix (for example, H26–H22 and H23a–H26 in Fig. 3c). Internal loops are used to modulate groove widths and to present purine N1 or N2 functional groups for helix–helix docking (for example, H20); they are also often found at sharp bends (such as the bends in H23 and H24).

The three-way junction of RNA helices 20, 21 and 22 is the heart of the central domain (Fig. 4). Its conformation is stabilized by the binding of S15 to the minor grooves of H20 and H22, in excellent agreement with *in vitro* biochemical studies on the interaction of S15 with fragments of 16S RNA³⁶,³⁷. The tip of S15 is also close to H21 near the three-way junction, which is known to be required for S15 binding. The junction conformation is further stabilized by more tenuous interactions from S17 and S8, whose primary binding sites are on nearby helices. The S8–RNA interaction is in excellent agreement with binding studies that identify its interaction with a distorted minor groove in H21 (ref. 38) and a crosslink between Lys 55 of *E. coli* S8 and U653 (ref. 39), as well as footprinting studies²³ that show protection of both H21 identified in other studies and H25. In our structure, the amino-terminal helix of S8 makes contact with a minor groove of H25. Although S17 is very close to H21, and Lys 29 of *E. coli* S17 has been crosslinked to U632 (ref. 39), its interaction with this helix as seen in our map is slight, explaining a lack of footprint of S17 to this region. Rather, its RNA-binding regions interact with other double-helical RNA, presumably including H11 in the 5′ domain (nucleotides 240–290).

### The platform

The front half of the platform contains only the RNA helices H23 and H24, which terminate in the functionally important 690 and 790 loops. The 690 and 790 loops are indeed in close proximity (Fig. 3c), as has been predicted by on the basis of intra-site crosslinking⁴⁰ and footprinting experiments using P-site tRNA or antibiotics²⁸. Moreover, the 690 and 790 stems protrude into the intersubunit interface, with their minor grooves available for association with the 50S subunit, in agreement with hydroxy-radical footprinting data on subunit association³⁰.

H21 joins the platform to the back of the body, and H25, H19 and H27 are part of the body. Although H27, which is known to function in a switch, is well separated from the other helices, it makes a minor-groove packing interaction with H24. Moreover, the tip of H27 is close to ribosomal protein S5. Mutations in S5 have been implicated in the switch that is part of the role of H27 in accuracy³¹.

The platform appears to consist of two halves separated by a plane of lesser electron density, with this plane oriented roughly parallel to the intersubunit surface. Several observations indicate that the front and back halves of the platform may indeed exhibit some flexibility relative to each other. It is known that the platform is a flexible part of the subunit; electron-microscopy studies have shown that, upon subunit association or IF-3 binding, the platform moves upwards⁸,⁴¹. Moreover, the separation between the two halves coincides roughly with the maximum extent of the proteins in this region, as if there is a limit to how close to the subunit interface protein-mediated stabilization is permissible. In addition, our fold of the central domain RNA reveals that this region of lesser density consists of potentially flexible multi-stem junctions. The maximal extent of hydroxyl-radical footprinting from 50S subunit association³⁰ extends approximately to the same vertical plane of lesser electron density. All of this evidence indicates that the platform has two halves which may be able to move independently relative to one another during translation.

#### The H27 accuracy switch

This putative motion of the front of the platform may be coupled to conformation changes in the decoding site by the H27 accuracy switch. The hairpin loop of H27 packs against the minor groove of the proximal end of H24 in the platform (Figs 3c, 5). If the front half of the platform does move independently, then H27 may well move with it. The other end of H27 makes a very extensive minor-groove packing interaction with H44 (Fig. 5), just below the decoding site where the A-site tRNA binds. This end of H27 is also close to ribosomal protein S5, and to RNA nucleotides footprinted by streptomycin; both streptomycin and mutations in S5 affect translational accuracy³¹. Previously, it was unknown how the H27 switch affected translational accuracy; now it appears likely that the H27 switch determines translation accuracy by modulating the conformation of the decoding site by a direct interaction, including a possible movement of H44.

#### Protein–protein interactions

Although many of the interactions of ribosomal proteins are with RNA, even within just the subset of proteins identified there is an example of interesting protein–protein interactions. Protein S5 forms close interactions with both S4 and S8. As shown in Fig. 6, the helical domain of S4 is juxtaposed against a face of S5, while the carboxy-terminal end of S5 makes an interface with S8. The mutations on S4 and S5 that confer the *ram* phenotype⁴²,⁴³ lie at their interface, indicating that in this case the mutations may disrupt a crucial protein–protein interaction rather than a protein–RNA contact, as previously suggested¹¹. Similarly, the interface between S5 and S8 is entirely consistent with the observed crosslink between S8 and the C-terminal region of S5 (ref. 44).

![Figure 6](#)

Figure 6 Stereo view of the relative disposition of proteins S4, S5 and S8 as they occur in the 30S subunit. The sites of *ram* mutations in S4 and S5 are shown in green (see text); the spectinomycin-resistance mutations in S5 are shown in blue.

Inset (right) shows the structures in the envelope of the 30S subunit. Figure made with RIBBONS⁵⁰.
We have compared portions of our model with the corresponding parts of a 30S model based on cryo-electron microscopy and biochemical data (F. Mueller, I. Sommer and R. Brimacombe, personal communication). The r.m.s.d. of the positions of proteins S4, S5, S8 and S15 in a least-squares superposition is 17.5 Å. In addition, the location of H21 in the model is similar to ours. However, the orientation of S5 relative to S4 is different, so that the ram sites on S5 do not appear to be part of the interface with S4. Both the orientation and location of S15 are significantly different. Finally, the quasi-continuous helices formed by stacking at three-way junctions in our structure of the central domain do not appear to be observed, so that while the locations of the other RNA helices in the central domain are close, as can be expected from topological constraints, they do not agree in detail.

### Other regions of the 30S subunit

We have determined the locations of many double-helical RNA segments throughout the subunit. As ribosomal RNA double helices can be highly distorted because of bulges and non-canonical base pairs, this was done by visual inspection of the map rather than by automated search algorithms. These segments are shown in Fig. 7, along with the central domain, the proteins of known structure that have been identified in the map and our structure for S20. As shown in Fig. 7, our putative S20 is surrounded by RNA double helices, one of which is also close to S17. This result is consistent with data from hydroxyl-radical footprinting²³ and nuclease protection²⁶. In this view, S20 would participate in the folding of the bottom portion of the 30S subunit and be intimately associated with 16S RNA. Its dual identification as the 50S protein L26 is therefore puzzling.

The bottom part of the 30S is relatively free of proteins. Although the head has several ribosomal proteins, none of their structures are known, with the exception of S7. We have therefore postponed a detailed interpretation of other regions of the 30S subunit until we have further analysed the current map and improved the resolution.

#### Discussion

One of the most striking features of the structure as a whole is the almost complete absence of protein from the subunit interface (Fig. 7b). Nearly all of the proteins identified in our map are located on either the sides or the back of the body. Only a small portion of S7 and possibly part of S12 appear to lie close to the subunit interface. This observation is consistent with predictions that the subunit interface contains little or no protein⁴⁵. As the interface is likely to be functionally the most important part of the ribosome, it also supports speculations that a primordial ribosome could have contained no protein at all.

However, our 30S structure also provides evidence that at least a few ribosomal proteins may be more directly involved in ribosomal function than previously suspected, as noted above for S4 and S5. Thus, although a primordial ribosome may have consisted entirely

![Figure 7](#fig7)

**Figure 7** Two orthogonal stereo views of the 30S subunit. The central domain RNA is in the colouring scheme of Fig. 3, the 1,400/1,500 stem-loop of 16S RNA in red, the proteins of known structure that have been identified in the map, and protein S20 near the bottom of the subunit. Other regions that have been identified as double-helical RNA are shown in grey. **a**, Intersubunit interface side faces the reader; in **b** it faces left. Figure made using RIBBONS⁵⁰.

NATURE | VOL 400 | 26 AUGUST 1999 | www.nature.com   © 1999 Macmillan Magazines Ltd   839
of RNA, it is likely that not all of the moving parts and crucial surfaces in the ribosomes of today will be found to consist of RNA. Our results have revealed the details of protein and RNA architecture in the 30S subunit, and the principles of their organization in its central domain. They also provide a firm basis for future biochemical and crystallographic analysis of the small subunit and its functions.

### Methods

#### Preparation and crystallization of ribosomal subunits.
Ribosomes and 30S subunits were prepared from *T. Thermophilus* cells as described⁸. Crystals of 30S ribosomes were grown using the hanging-drop method with MPD as the precipitant, with minor modifications of the procedure of ref. 3. Crystals grew to a maximum size of $80 \times 80 \times 200 \mu$m, and were in space group $P4(1)2(1)2$ with cell dimensions of $a = b = 401.5 \, \text{\AA}$ and $c = 174.5 \, \text{\AA}$. The correct choice of space group between the two possible enantiomers was determined eventually by the handedness of RNA double helices in the electron-density map. Crystals were transferred to a stabilizing solution containing 26% MPD as a cryoprotectant (with or without heavy atom compounds) before flash-freezing in liquid nitrogen for data collection at 100 K.

#### Data collection.
Data were collected on beamlines X12B and X25 at the National Synchrotron Light Source (NSLS) at Brookhaven National Laboratory. We collected data on native and heavy atom derivatives of the crystals using $20 \times 20 \, \text{cm}$ CCD detectors, the ADSC Quantum 4 detector on X12B or the Brandeis B4 detector on X25. The final data used for phasing were all from beamline X25. All data were indexed, scaled and reduced using the programs Denzo and Scalepack⁴⁶.

#### Phasing and analysis of maps.
A Harker section of the difference Patterson map of data from a W17 cluster showed a clear strong peak of about 7 standard deviations above background, as well as a second minor peak. The positions of these sites and phases to low resolution were calculated using the program SOLVE⁴⁷. Difference Fouriers using these low-resolution phases revealed a substantial number of sites in the other heavy-atom derivatives. We used the program SOLVE both to determine independently and to confirm these sites, as well as to calculate phases by Bayesian phase refinement. The single native data set was not used in the phasing because of lack of isomorphism, and the osmium data set was used as the ‘native’ or reference data set in the phasing. Density modification using the program SOLOMON⁴⁸ was used to improve the initial phases obtained from SOLVE. All map visualization and interpretation was done using the program O⁴⁹.

Received 14 June; accepted 13 July 1999.

---

1. Alberts, B. *et al.* Molecular Biology of the Cell (Garland, New York, 1995).
2. von Böhlen, K. *et al.* Characterization and preliminary attempts for derivatization of crystals of large ribosomal subunits from *Haloarcula marismortui* diffracting to 3 Å resolution. *J. Mol. Biol.* **222**, 11–15 (1991).
3. Trakhanov, S. D. *et al.* Crystallization of 70 S ribosomes and 30 S ribosomal subunits from *Thermus thermophilus*. *FEBS Lett.* **220**, 319–322 (1987).
4. Yusupov, M. M., Tischenko, S. V., Trakhanov, S. D., Ryazantsev, S. N. & Garber, M. B. A new crystalline form of 30 S ribosomal subunits from *Thermus thermophilus*. *FEBS Lett.* **238**, 113–115 (1988).
5. Yonath, A. *et al.* Characterization of crystals of small ribosomal subunits. *J. Mol. Biol.* **203**, 831–834 (1988).
6. Yonath, A. *et al.* Crystallographic studies on the ribosome, a large macromolecular assembly exhibiting severe nonisomorphism, extreme beam sensitivity and no internal symmetry. *Acta Crystallogr. A* **54**, 945–955 (1998).
7. Lata, K. R. *et al.* Three-dimensional reconstruction of the *Escherichia coli* 30S ribosomal subunit in ice. *J. Mol. Biol.* **262**, 43–52 (1996).
8. McCutcheon, J. P. *et al.* Location of translational initiation factor IF3 on the small ribosomal subunit. *Proc. Natl Acad. Sci. USA* **96**, 4301–4306 (1999).
9. Mueller, F. & Brimacombe, R. A new model for the three-dimensional folding of *Escherichia coli* 16 S ribosomal RNA. I. Fitting the RNA to a 3D electron microscopic map at 20 Å. *J. Mol. Biol.* **271**, 524–544 (1997).
10. Allain, F. H. & Varani, G. Structure of the P1 helix from group I self-splicing introns. *J. Mol. Biol.* **250**, 333–353 (1995).
11. Ramakrishnan, V. & White, S. W. Structure of ribosomal protein S5 reveals sites of interaction with 16S RNA. *Nature* **358**, 768–771 (1992).
12. Lindahl, M. *et al.* Crystal structure of the ribosomal protein S6 from *Thermus thermophilus*. *EMBO J.* **13**, 1249–1254 (1994).
13. Jaishree, T. N., Ramakrishnan, V. & White, S. W. Solution structure of prokaryotic ribosomal protein S17 by high-resolution NMR spectroscopy. *Biochemistry* **35**, 2845–2853 (1996).
14. Davies, C., Ramakrishnan, V. & White, S. W. Structural evidence for specific S8–RNA and S8–protein interactions within the 30S ribosomal subunit; ribosomal protein S8 from *Bacillus stearothermophilus* at 1.9 Å resolution. *Structure* **4**, 1093–1104 (1996).
15. Berglund, H., Rak, A., Serganov, A., Garber, M. & Härd, T. Solution structure of the ribosomal RNA binding protein S15 from *Thermus thermophilus*. *Nature Struct. Biol.* **4**, 20–23 (1997).
16. Winberly, B. T., White, S. W. & Ramakrishnan, V. The structure of ribosomal protein S7 at 1.9 Å resolution reveals a β-hairpin motif that binds double-stranded nucleic acids. *Structure* **5**, 1187–1198 (1997).
17. Hosaka, H. *et al.* Ribosomal protein S7: a new RNA-binding motif with structural similarities to a DNA architectural factor. *Structure* **5**, 1199–1208 (1997).
18. Clemons, W. M. Jr, Davies, C., White, S. W. & Ramakrishnan, V. Conformational variability of the N-terminal helix in the structure of ribosomal protein S15. *Structure* **6**, 429–438 (1998).
19. Nevskaya, N. *et al.* Crystal structure of ribosomal protein S8 from *Thermus thermophilus* reveals a high degree of structural conservation of a specific RNA binding site. *J. Mol. Biol.* **279**, 233–244 (1998).
20. Davies, C., Gerstner, R. B., Draper, D. E., Ramakrishnan, V. & White, S. W. The crystal structure of ribosomal protein S4 reveals a two-domain molecule with an extensive RNA-binding surface: one domain shows structural homology to the ETS DNA-binding motif. *EMBO J.* **17**, 4545–4558 (1998).
21. Markus, M. A., Gerstner, R. B., Draper, D. E. & Torchia, D. A. The solution structure of ribosomal protein S4 delta41 reveals two subdomains and a positively charged surface that may interact with RNA. *EMBO J.* **17**, 4559–4571 (1998).
22. Capel, M. S. *et al.* A complete mapping of the proteins in the small ribosomal subunit of *Escherichia coli*. *Science* **238**, 1403–1406 (1987).
23. Powers, T. & Noller, H. F. Hydroxyl radical footprinting of ribosomal proteins on 16S rRNA. *RNA* **1**, 194–209 (1995).
24. Mueller, F. & Brimacombe, R. A new model for the three-dimensional folding of *Escherichia coli* 16 S ribosomal RNA. II. The RNA–protein interaction data. *J. Mol. Biol.* **271**, 545–565 (1997).
25. Ramakrishnan, V. *et al.* Position of proteins S6, S11 and S15 in the 30 S ribosomal subunit of *Escherichia coli*. *J. Mol. Biol.* **153**, 739–760 (1981).
26. Ungewickell, E., Garrett, R., Ehresmann, C., Stiegler, P. & Fellner, P. An investigation of the 16-S RNA binding sites of ribosomal proteins S4, S8, S15, and S20 from *Escherichia coli*. *Eur. J. Biochem.* **51**, 165–180 (1975).
27. Mueller, F., Stark, H., van Heel, M., Rinke-Appel, J. & Brimacombe, R. A new model for the three-dimensional folding of *Escherichia coli* 16 S ribosomal RNA. III. The topography of the functional centre. *J. Mol. Biol.* **271**, 566–587 (1997).
28. Moazed, D. & Noller, H. F. Binding of tRNA to the ribosomal A and P sites protects two distinct sets of nucleotides in 16 S rRNA. *J. Mol. Biol.* **211**, 135–145 (1990).
29. Lee, K., Varma, S., SantaLucia, J. Jr & Cunningham, P. R. *In vivo* determination of RNA structure–function relationships: analysis of the 790 loop in ribosomal RNA. *J. Mol. Biol.* **269**, 732–743 (1997).
30. Merryman, C., Moazed, D., McWhirter, J. & Noller, H. F. Nucleotides in 16S rRNA protected by the association of 30S and 50S ribosomal subunits. *J. Mol. Biol.* **285**, 97–105 (1999).
31. Lodmell, J. S. & Dahlberg, A. E. A conformational switch in *Escherichia coli* 16S ribosomal RNA during decoding of messenger RNA. *Science* **277**, 1262–1267 (1997).
32. Gregory, R. J. *et al.* Interaction of ribosomal proteins S6, S8, S15 and S18 with the central domain of 16 S ribosomal RNA from *Escherichia coli*. *J. Mol. Biol.* **178**, 287–302 (1984).
33. Greuer, B., Osswald, M., Brimacombe, R. & Stöffler, G. RNA-protein cross-linking in *Escherichia coli* 30S ribosomal subunits; determination of sites on 16S RNA that are cross-linked to proteins S3, S4, S7, S9, S10, S11, S17, S18 and S21 by treatment with bis-(2-chloroethyl)-methylamine. *Nucleic Acids Res.* **15**, 3241–3255 (1987).
34. Wu, H., Jiang, L. & Zimmermann, R. A. The binding site for ribosomal protein S8 in 16S rRNA and spc mRNA from *Escherichia coli*: minimum structural requirements and the effects of single bulged bases on S8–RNA interaction. *Nucleic Acids Res.* **22**, 1687–1695 (1994).
35. Moine, H., Cachia, C., Westhof, E., Ehresmann, B. & Ehresmann, C. The RNA binding site of S8 ribosomal protein of *Escherichia coli*: Selex and hydroxyl radical probing studies. *RNA* **3**, 255–268 (1997).
36. Batey, R. & Williamson, J. Interaction of the *Bacillus stearothermophilus* ribosomal protein S15 with 16 S rRNA: I. Defining the minimal RNA site. *J. Mol. Biol.* **261**, 536–549 (1996).
37. Serganov, A. A. *et al.* The 16S rRNA binding site of *Thermus thermophilus* ribosomal protein S15: comparison with *Escherichia coli* S15, minimum site and structure. *RNA* **2**, 1124–1138 (1996).
38. Kalurachchi, K., Uma, K., Zimmermann, R. A. & Nikonowicz, E. P. Structural features of the binding site for ribosomal protein S8 in *Escherichia coli* 16S rRNA defined using NMR spectroscopy. *Proc. Natl Acad. Sci. USA* **94**, 2139–2144 (1997).
39. Urlaub, H., Thiede, B., Müller, E. C., Brimacombe, R. & Wittmann-Liebold, B. Identification and sequence analysis of contact sites between ribosomal proteins and rRNA in *Escherichia coli* 30 S subunits by a new approach using matrix-assisted laser desorption/ionization-mass spectrometry combined with N-terminal microsequencing. *J. Biol. Chem.* **272**, 14547–14555 (1997).
40. Atmadja, J. *et al.* The tertiary folding of *Escherichia coli* 16S RNA, as studied by *in situ* intra-RNA cross-linking of 30S ribosomal subunits with bis-(2-chloroethyl)-methylamine. *Nucleic Acids Res.* **14**, 659–673 (1986).
41. Agrawal, R. K., Penczek, P., Grassucci, R. A. & Frank, J. Visualization of elongation factor G on the *Escherichia coli* 70S ribosome: the mechanism of translocation. *Proc. Natl Acad. Sci. USA* **95**, 6134–6138 (1998).
42. van Acken, U. Proteinchemical studies on ribosomal proteins S4 and S12 from ram (ribosomal ambiguity) mutants of *Escherichia coli*. *Mol. Gen. Genet.* **140**, 61–68 (1975).
43. Wittmann-Liebold, B. & Greuer, B. The primary structure of protein S5 from the small subunit of the *Escherichia coli* ribosome. *FEBS Lett.* **95**, 91–98 (1978).
44. Allen, G., Capasso, R. & Gualerzi, C. Identification of the amino acid residues of proteins S5 and S8 adjacent to each other in the 30 S ribosomal subunit of *Escherichia coli*. *J. Biol. Chem.* **254**, 9800–9806 (1979).
45. Agafonov, D. E., Kolb, V. A. & Spirin, A. S. Proteins on ribosomal surface: measurements of protein exposure by hot tritium bombardment technique. *Proc. Natl Acad. Sci. USA* **94**, 12892–12897 (1997).
46. Otwinowski, Z. & Minor, W. in *Methods in Enzymology* (eds Carter, C. W. J. & Sweet, R. M.) 307–325 (Academic, New York, 1997).
47. Terwilliger, T. & Berendzen, J. Automated MAD and MIR structure determination. *Acta Crystallogr. D* **55**, 849–861 (1999).
48. Abrahams, J. P. & Leslie, A. G. W. Methods used in the structure determination of bovine mitochondrial F1 ATPase. *Acta Crystallogr. D* **52**, 30–42 (1996).
49. Jones, T. A. & Kjeldgaard, M. Electron-density map interpretation. *Methods Enzymol.* **277B**, 173–207 (1997).
50. Carson, M. Ribbons 2.0. *J. Appl. Crystallogr.* **24**, 958–961 (1991).

### Acknowledgements.
This work was supported by grants from the US NIH (to S. W. White & V.R.), the UK MRC and the University of Utah. Beamlines X12B and X25 at the NSLS are supported by the Office of Basic Energy Research of the US Department of Energy and by the NIH. We thank L. Berman and H. Lewis for their help on beamline X25; T. Terwilliger for advice on using SOLVE; P. Reversi and C. Vonrhein for discussions on phasing strategies; M. Pope for gifts of compounds of polytungstate clusters; J. Löwe and G. Schneider for gifts of hexatantalum bromide; B. S. Brunschwig and M. H. Chou for synthesizing osmium hexamine chloride; and S. C. Harrison, K. Nagai and D. Rhodes for critical comments on the manuscript. Coordinates for our partial model fo the 30S have been deposited in the protein data bank, with accession number 1QD7.

Correspondence and requests for materials should be addressed to V.R. at MRC Cambridge (e-mail: ramak@mrc-lmb.cam.ac.uk).
