
Placement of protein and RNA structures into a 5 Å-resolution map of the 50S ribosomal subunit

Nenad Ban*, Poul Nissen*, Jeffrey Hansen†, Malcolm Capel‡, Peter B. Moore§ & Thomas A. Steitz*†§

Departments of *Molecular Biophysics & Biochemistry and §Chemistry, Yale University and the †Howard Hughes Medical Institute, New Haven, Connecticut 06520-8114, USA  
‡Department of Biology, Brookhaven National Laboratory, Upton, New York 11973, USA

We have calculated at 5.0 Å resolution an electron-density map of the large 50S ribosomal subunit from the bacterium Haloarcula marismortui by using phases derived from four heavy-atom derivatives, intercrystal density averaging and density-modification procedures. More than 300 base pairs of A-form RNA duplex have been fitted into this map, as have regions of non-A-form duplex, single-stranded segments and tetraloops. The long rods of RNA crisscrossing the subunit arise from the stacking of short, separate double helices, not all of which are A-form, and in many places proteins crosslink two or more of these rods. The polypeptide exit channel was marked by tungsten cluster compounds bound in one heavy-atom-derivatized crystal. We have determined the structure of the translation-factor-binding centre by fitting the crystal structures of the ribosomal proteins L6, L11 and L14, the sarcin–ricin loop RNA, and the RNA sequence that binds L11 into the electron density. We can position either elongation factor G or elongation factor Tu complexed with an aminoacylated transfer RNA and GTP onto the factor-binding centre in a manner that is consistent with results from biochemical and electron microscopy studies.

The large (50S) ribosomal subunit catalyses the peptidyl-transfer reaction of messenger-RNA-directed protein synthesis¹. In bacteria, it consists of a ~2,900-nucleotide 23S RNA, a 120-nucleotide 5S RNA and about 33 different proteins; it has a relative molecular mass of ~1.6 × 10⁶ (ref. 2). Peptide-bond synthesis occurs at the bottom of a deep cleft in the large ribosomal subunit where the aminoacylated acceptor stem of the tRNA carrying the next amino acid is brought together with the corresponding region of a

Table 1 Statistics for data collection and phase determination

| Space group: | C222₁ | P2₁ | C222₁ |
| --- | --- | --- | --- |
| Unit-cell dimensions (Å): | 212 × 301 × 576 | 184 × 570 × 184 | 210 × 300 × 540 |
| β = 108.5 |  |  |  |
| Asymmetric unit: | 1 molecule | 2 molecules | 1 molecule |
| Native 1 | Native 2 | W₁₈ | W₁₁ | TA | OsHx | HM50SP21 | HM50SD |
| Soaking time (days) | – | – | 6 | 2 | 5 | 2 | – | – |
| Wavelength (Å) | 1.21 | 1.34 | 1.21 | 1.21 | 1.25 | 1.14 | 1.14 | 1.21 |
| Resolution (Å) | 120–3.75 | 70–4.3 | 100–4.0 | 120–4.0 | 120–4.8 | 30–3.4 | 30–4.6 | 30–3.6 |
| Observations | 957,850 | 496,354 | 1,467,476 | 1,411,640 | 366,832 | 1,646,468 | 1,144,780 | 600,063 |
| Unique | 185,190 | 116,447 | 302,089 | 303,285 | 133,044 | 488,275 | 198,796 | 163,558 |
| Completeness | 98.6 | 93.6 | 99.9 | 99.6 | 74.3 | 97.9 | 100.0 | 83.2 |
| Rmerge | 10.3 | 21.8 | 13.4 | 14.3 | 10.4 | 14.2 | 21.1 | 17.0 |
| Rmerge (ano) | – | – | 8.7 | 9.1 | 6.9 | 7.4 | – | – |
| Cluster conc. (mM) | – | – | 1.0 | 1.0 | 1.0 | 4.5 | – | – |
| No. of sites# | – | – | 6 | 10 | 9 | 38 | – | – |
| Riso(30–5.0 Å) | – | – | 26.5 | 22.5 | 19.5 | 14.5 | – | – |
| I/σ (highest res.bin) | 13.0 (2.2) | 6.2 (1.8) | 10.4 (1.8) | 8.9 (1.6) | 10.6 (2.37) | 9.9 (2.08) | 8.6 (3.2) | 7.6 (1.6) |

MIRAS phasing statistics

| Resolution shells (Å) – ~34, 600 reflections per bin |
| --- |
| 60.0 | 6.9 | 5.8 | 5.2 | 5.0 | Total |
| W₁₈ |
| Phasing power | 1.05 | 0.32 | 0.46 | 0.54 | 1.00 |
| R<sub>culis</sub> (centric) | 0.60 | 0.73 | 0.71 | 0.70 | 0.62 |
| W₁₁ |
| Phasing power | 1.09 | 0.35 | 0.41 | 0.43 | 1.00 |
| R<sub>culis</sub> (centric) | 0.60 | 0.74 | 0.72 | 0.73 | 0.61 |
| TA |
| Phasing power | 0.92 | 0.30 | 0.10 | 0.10 | 0.81 |
| R<sub>culis</sub> (centric) | 0.69 | 0.83 | 0.90 | 0.97 | 0.72 |
| OsHx |
| Phasing power | 0.52 | 0.40 | 0.33 | 0.25 | 0.40 |
| R<sub>culis</sub> (centric) | 0.65 | 0.68 | 0.74 | 0.73 | 0.69 |
| (MIRAS/SAD) FOM | 0.65 | 0.31 | 0.27 | 0.22 | 0.37 |
| (Density modification) FOM | 0.94 | 0.62 | 0.32 | 0.18 | 0.51 |

R<sub>iso</sub> = Σ|F<sub>PH</sub> – F<sub>P</sub>| / ΣF<sub>PH</sub>, where F<sub>PH</sub> and F<sub>P</sub> are the derivative and the native structure factor amplitudes, respectively; R<sub>sym</sub> = ΣΣ<sub>i</sub>|I<sub>(h)</sub> – I<sub>(h')</sub>| / ΣΣ<sub>(h)</sub>I<sub>(h)</sub>, where I(h) is the mean intensity of reflections. Phasing power: r.m.s. isomorphous difference divided by the r.m.s. residual lack of closure. R<sub>culis</sub> = Σ(|F<sub>PH</sub> – F<sub>P</sub>| – |F<sub>H(calc)</sub>|) / Σ|F<sub>PH</sub> – F<sub>P</sub>|. The summation is valid only for centric reflections. W₁₈, W₁₁ and TA are the eighteen and eleven tungsten and six tantalum cluster compounds described earlier²¹ and OsHx is osmium hexamine. HM50SP21 is a monoclinic data set with a shorter c-axis.

articles

![](image1.png)

Figure 1 The 5Å-resolution electron-density map of the 50S ribosomal subunit. In **a** and **c**, some RNA helices and tetraloops of unknown assignment (in yellow) have been fitted into density to guide the viewer. The backbone of the SRL is red, the backbones of helices believed to be 96 and 97 are blue and yellow, respectively, and the backbone of the L11 RNA is yellow. All proteins are shown as ribbon structures. The structure of protein L1 (ref. 45) was positioned in a 12Å-resolution MIR map. **a**, A stereo representation of the subunit is viewed down the peptide synthesis active-site cleft. EF-G, in green, has been docked onto the fitted proteins and RNA. **b**, Solid-surface drawing of the large subunit in the crown view and in the same orientation as in **c**. The locations of proteins L1, L7/L12 and the central protuberance, as determined by electron microscopy, are labelled. The tunnel seen at 5 Å resolution, is shown in blue and is viewed at a 45° angle to its long axis. **c**, 'Crown' view of the 50S subunit, including ribbon representations of protein and RNA-fragment structures that have been fitted to the density on this side of the subunit. Identified structures are labelled.

![](image2.png)

Figure 2 Visualization of the tunnel in which tungsten clusters are located. **a**, Stereo view, rotated by 90° about the vertical axis from the crown view (Fig.1b, c) and showing the four tungsten clusters that mark the polypeptide exit tunnel. **b**, Close-up to the tunnel sliced on one side to show the four tungsten clusters. The surface is gold on the outside and blue on the inside and represents the low electron density (empty space) in the proximity of the clusters.

tions varying between 13 and 24 Å (refs 4, 19, 20). A 9Å-resolution electron-density map of the 50S subunit obtained by X-ray crystallography revealed density consistent with long RNA rods that crisscross the subunit²¹.

We have now calculated at 5 Å resolution an X-ray crystallographic electron-density map of the 50S subunit which has enabled us to make atomic-level structural interpretations of this large assembly. Familiar generic structures, such as A-form RNA duplex and RNA tetraloops, as well as the crystal structures of specific RNA fragments and proteins, can be positioned in this map. We can now plausibly position the major RNA and protein components of the factor-binding centre in the subunit’s structure and propose a model for the way in which elongation factors bind to it.

## Structure determination

The 5Å-resolution map was obtained using multiple isomorphous replacement (MIR)/single wavelength anomalous difference (SAD) phasing together with intercrystal and non-crystallographic averaging between three different crystal forms (Table 1). This was possible only after we discovered that the *H. marismortui* 50S subunit crystallizes both in an orthorhombic space group, $C222_1$, having unit-cell dimensions $a = 212$, $b = 301$ and $c = 576$ Å, and in a merohedrally twinned monoclinic space group, $P2_1$, whose cell dimensions and Laue symmetry are nearly indistinguishable from those of the orthorhombic form but whose diffraction intensities differ by about 30–35% (30 to 5.5 Å resolution data). The space-group change and the twinning were discovered by examination of difference Patterson maps of the two crystal forms derivatized with a tungsten cluster ($W_{18}$) compound. Twinning could not be detected by analysing intensity statistics, perhaps because of strong pseudo-$C222_1$ symmetry in each of the monoclinic crystals, or because only low-resolution data were available. Using a 20Å-resolution reconstruction of the *H. marismortui* large ribosomal subunit²¹, we found a molecular replacement solution for the two subunits in the asymmetric unit of the $P2_1$ crystals, which explains the $W_{18}$ difference-Patterson map in $P2_1$ and confirms that the $P2_1$ crystals are twinned. The subunit packing in the $P2_1$ and $C222_1$ crystals differs by a 10 Å translation, which eliminates two dyad axes of the orthorhombic crystals (see Methods).

All phasing by heavy-atom methods was performed in the orthorhombic unit cell. An osmium hexamine derivative was found, and by combining data from it with the data from the original three heavy-atom derivatives, MIRAS phases could be obtained that had a satisfactory figure of merit to 7 Å and contained significant phase information beyond that. Phases were improved beyond 7 Å by combining SAD phases obtained from each derivative with four-derivative MIRAS phases and by intercrystal averaging²²,²³ using two non-isomorphous orthorhombic data sets and two detwinned monoclinic data sets (Table 1).

## General structure at 5 Å resolution

The particle envelope of the 5Å-resolution map has the same overall shape (Fig. 1) as that obtained previously by cryo-electron microscopy at 20 Å resolution²¹. At the higher resolution, however, deep crevices and open spaces are seen to separate large domains. One of the most prominent features of the subunit is a very deep, wide cleft that forms the entrance to the peptidyltransferase active site. The ‘rim’ of this cleft consists largely of a long rod of RNA duplex constructed from a series of stacked double-stranded helices that apparently emanate from several different regions of the 23S RNA sequence.

At the bottom of the active-site cleft is a tunnel that runs straight through the subunit. In one of our heavy-atom derivatized crystals, the path of this tunnel is marked by the positions of four bound eleven-tungsten($W_{11}$)-cluster molecules, which have a van der Waals diameter of nearly 20 Å and lie on an almost straight line (Fig. 2). This tunnel is thought to be the path used by the nascent

Figure 3 A variety of interactions between RNA duplexes. **a**, An RNA of unknown sequence and containing an S-turn backbone structure with its tetraloop is seen to be docked into an RNA duplex. **b**, Many RNA duplexes show direct interactions, some of which may involve ribose-ribose contacts. **c, d**, The experimental electron density map (**c**) and one calculated at 5 Å resolution from the atomic coordinates with a B-factor of 70 (**d**) are very similar. Bumps in the electron density arising from the phosphoribose backbone are seen in both.

peptidyl-tRNA³,⁴. The elongating polypeptide is believed to exit through a tunnel that passes through the middle of the 50S subunit⁵,⁶. This subunit also contains a region termed the ‘GTPase-associated site’ or the ‘GTPase centre’, which we call the ‘factor-binding centre’: this region includes proteins L6, L11, L14 and the L7/L12 stalk, as well as the sarcin–ricin loop sequence and the 1,055–1,080-nucleotide region of 23S RNA⁷. The factor-binding centre binds the two translation-elongation factors EF-G and EF-Tu, as well as the initiation factor IF-2 and the release factor RF-3, all of which are G proteins: binding to the centre stimulates their GTPase activity.

All of the proteins in the small ribosomal subunit⁸ and some in the large subunit⁹ have been positioned by neutron scattering, and immune electron microscopy has provided more approximate localizations of proteins in both subunits¹⁰,¹¹. X-ray crystallographic and NMR studies have determined the structures at high resolution of ten proteins of the large subunit¹², two RNA fragments (the sarcin–ricin loop (SRL)¹³,¹⁴ and half of 5S RNA¹⁵,¹⁶), complexes of protein L11 with a 58-nucleotide RNA fragment¹⁷,¹⁸ and L25 complexed with a fragment of 5S RNA (M. Lu and T.A.S., unpublished results). Single-particle analysis of whole 70S ribosomes by cryo-electron microscopy has generated three-dimensional maps of the ribosome and its complexes with tRNA and factors, at resolu-

polypeptide chain to traverse from the peptidyltransferase centre to the point where it leaves the ribosome ${ }^{6,24}$, an idea supported by the location at the tunnel entrance of the aminoacyl end of formylmethionine (fMet)-tRNA bound at the P-site ${ }^{4}$ and the alignment of the tunnel exit on the back of the large subunit with the central pore of the Sec61 channel in yeast ${ }^{6}$.

The 5Å-resolution electron-density map of the 50S ribosomal subunit shows well resolved density for the backbone of the two strands of duplex RNA and higher density features at the positions of phosphate groups. Density for base pairs is continuous and smoothly joins the backbone of the two strands. The shape of the density corresponding to RNA seen in the experimental 5Å-resolution map is very similar to that observed in a perfect 5Å-resolution map produced using calculated phases (Fig. 3c), indicating that the experimental map is reasonably well phased to 5 Å resolution. Using the real-space correlation program ESSENS ${ }^{25}$ and visual inspection, density corresponding to more than 300 base pairs of A-form duplex RNA has been fitted to models.

Long rods of double-stranded RNA crisscross the subunit, and helices of A-form and non-A-form duplex RNA from different sequence regions stack to form these rods (Fig. 3). There are also regions where RNA helices lying side by side interact (Fig. 3), reminiscent of arrangements seen in the structure of the P4-6 domain of the group I ribozyme ${ }^{26}$, and some of these inter-RNA duplex interactions may involve contacts between backbone ribose groups (ribose 'zippers'). Another well known structural motif seen repeatedly in these maps is an RNA tetraloop of sequence guanine-any nucleotide-purine-adenine (GNRA) ${ }^{27}$, which is often found at the ends of duplex hairpins (Figs 3, 4). Although many regions of single-stranded RNA can be seen in these 5Å-resolution maps, they are difficult to trace unambiguously for long distances. Density arising from the protein can be differentiated from that attributed to RNA because of its different character. The rods of density arising from α-helices are the easiest to identify and the hardest are the β-sheet regions.

So far about 12 proteins have been located (although no systematic search has been undertaken) and the atomic coordinates of five proteins have been fitted to the density. Most of these proteins are evenly dispersed among RNA regions, consistent with IEM localization ${ }^{10,11}$ and crosslinking two or more RNA rods. They neither cover the RNA, as occurs in viruses, nor are they surrounded by nucleic acid, as in the nucleosome. Many examples of protein α-helices interacting with the minor or major grooves of duplex RNA have been found, as well as some β-strand interactions. Most of the proteins seem to fix the tertiary structure of 23S rRNA by cross-linking helical segments that are distant in the sequence, which indicates that they may be important in stabilizing ribosome conformation, consistent with biochemical observations ${ }^{28}$.

### The structure of the factor-binding centre

We have been able to position the crystal structures of three proteins and two RNA fragments that form a major part of the binding centre for G-protein factors. Using the program ESSENS, we first found the electron density corresponding to the sarcin–ricin loop (SRL), which then guided us to L6 (Fig. 4). The SRL was identifiable because of its unusual S-shaped backbone structure, which is close to a GNRA tetraloop. Only one other such electron-density region was found in the map (Fig. 3), but its location is inconsistent with other data on the position of the SRL ${ }^{9}$. Protein L6 was identified primarily by its two long α-helices, which are in different domains ${ }^{29}$. To position these helices in the long rods of density to which they correspond, the angle between the domains had to be altered by about $5^{\circ}$. The β-sheet regions then fitted well into regions of broad density (Fig. 4), an interpretation that would not have been possible in the absence of a high-resolution crystal structure.

Biochemical and genetic data on the interacting regions of the SRL and L6 are in agreement with the structure derived from the 5Å-resolution map. Dimethyl sulphate protection and crosslinking data show that L6 binds to domain VI of 23S rRNA, particularly to

---

**Figure 4** The fitting of previously solved protein and nucleic acid structures to the 5Å-resolution electron density map. **a**, The known crystal structure of a protein L2 fragment ${ }^{36}$ and an unknown α-helical protein are seen to be interacting with an RNA helix that forms the base of a stalk on which protein L1 (not shown) sits. **b**, Crystal structure of the protein L11 C-terminal domain complexed with 58 nucleotides of rRNA ${ }^{20,21}$ fitted into the electron density. **c**, The crystal structures of protein L6 (ref. 29) and the sarcin-ricin loop ${ }^{16,17}$, as well as two other RNA regions are fitted into density. **d**, The density identified as the SRL shows the characteristic S-turn in the RNA backbone. Proteins L14 (ref. 34) and L6 are seen to interact with the SRL on opposite sides.

helices 95 (SRL) and 97 (refs 30–32). The hydroxyl group of Tyr 156, which has been crosslinked to 23S rRNA³³, is less than 4 Å away from the backbone of a tetraloop that interacts with the SRL.

The structure of protein L14 (ref. 34) was subsequently found (Fig. 4c) by making use, in part, of the relative locations of L6 and L14 in the IEM map of the 50S subunit³⁵. Two crystal structures, one of L11 complexed with 58 nucleotides of RNA¹⁷ and the other of a single L11 domain bound to a similar RNA¹⁸, were then positioned (Fig. 4b) in the density, guided by the fact that this complex must be near the SRL in the L7/L12 stalk⁷. There is no electron density in our map for the N-terminal domain of L11, suggesting that this domain is not ordered in these crystals. Likewise, there is no density for the L7/L12 stalk, which is also expected to be intimately involved in factor binding²⁰. Nevertheless, the three located proteins and two RNA fragments constitute most of the factor-binding centre. The structure of an L2 protein fragment³⁶ was also positioned (Fig. 4a) using IEM data as a guide³⁵.

Docking aa-tRNA·EF-Tu·GTP and EF-G to 50S

Two types of data were used to position the elongation factors on the factor-binding centre of the 50S subunit. First, cryo-electron microscopy reconstructions of ribosomes complexed with either aa-

tRNA·EF-Tu·GTP or EF-G provide the approximate orientation of these two G proteins; both bind in a similar way at the base of the L7L12 stalk¹⁹,²⁰. Second, a site-directed RNA hydrolysis has been used to provide distance constraints between specific residues on EF-G that have been mutated to cysteines and specific nucleotides in the 23S rRNA that have been hydrolysed by a reagent that can generate hydroxyl radicals, BABE, attached to the cysteine residues³⁷. The mutated and derivatized EF-G was bound in the post-translocation state by addition of GTP and fusidic acid. By positioning the two residues of EF-G as closely as possible to the RNA residues that were hydrolysed (Fig. 5) and simultaneously maximizing the steric complementarity between EF-G³⁸,³⁹ and the 50S subunit, EF-G can be docked in a position that is broadly consistent with the structure of this complex obtained by cryo-electron microscopy¹⁹. The aa-tRNA·EF-Tu·GTP (ref. 40) was then positioned by superimposing the corresponding G-domain α-carbon atoms, assuming that the homologous G domains of these two enzymes bind in an identical way (Fig. 5). Even though the shape complementarity between both factors and the electron density of the 50S subunit is good, the accuracy of this factor-docking model and its degree of correspondence with the cryo-electron microscopic density is difficult to assess because we do not

| a                                                                 | b                                                                 | c                                                                 |
|-------------------------------------------------------------------|-------------------------------------------------------------------|-------------------------------------------------------------------|
| **L11:rRNA**<br>![](https://via.placeholder.com/150)<br>L6<br>SRL<br>L14 | **SRL**<br>![](https://via.placeholder.com/150)<br>#192<br>EF-G<br>IV<br>V<br>#650 | **L6**<br>![](https://via.placeholder.com/150)<br>SRL<br>L14<br>EF-G<br>IV |

| d                                                                 | e                                                                 |
|-------------------------------------------------------------------|-------------------------------------------------------------------|
| **L11:rRNA**<br>L6<br>SRL<br>G<br>EF-G<br>V<br>L14<br>IV           | **L11:rRNA**<br>L6<br>SRL<br>G<br>EF-G<br>V<br>L14<br>IV         |
| **L6**<br>![](https://via.placeholder.com/150)<br>SRL<br>G<br>EF-G<br>V<br>L14<br>IV | **L6**<br>![](https://via.placeholder.com/150)<br>SRL<br>L14<br>aa-tRNA:<br>EF-Tu:<br>GTP |

Figure 5 The proteins (in white backbone ribbons) and RNAs (in coloured ribbon) of the factor-binding site that have been fitted to the density. The docked structures of EF-G·GDP and aa-tRNA·EF-Tu·GTP are shown in green. a, The three proteins and three RNA fragments as fitted to the density are shown in the crown view (Fig. 1). b, Docking of EF-G on the SRL using site-directed RNA cleavage data³⁷. The backbone of EF-G, in blue, with residues 650 and 192, to which the BABE cleavage reagent was attached, is represented as red spheres. The SRL backbone cleaved by BABE bound to residues 650 and 192 is in red. The EF-G domains are labelled in green³⁹,⁴⁰. c, The factor-binding centre and the docked EF-G shown in a ‘top’ view (without L11:rRNA) demonstrate the shape complementarity between the centre and the EF-G. d, Another orientation of c, shown in stereo, that includes L11:rRNA, also shows the complementarity and proximity of structures. e, The centre proteins and rRNAs, shown and oriented as in c, but with aa-tRNA·EF-TU·GTP (ref. 40) replacing EF-G, shows the proximity of the SRL (red) to the G domain and of L14 to tRNA.

articles

a
area of weak interactions
Plane of projection (ab) for crystal packing analysis

b
C2221 (Laue class mmm)
$a=212$ $b=300$ $c=575$, $\alpha=\beta=\gamma=90$
1 molecule/ASU

c
P21 (Laue class 2/m)
$a=184$ $b=570$ $c=184$ $\alpha=90$ $\beta=108.5$ $\gamma=90$
C2221: $a=212$ $b=300$ $c=575$, $\alpha=\beta=\gamma=90$
2 molecules/ASU

d
Twinned P21 (Laue class mmm)
$a=212$ $b=300$ $c=570$, $\alpha=\beta=\gamma=90$
2+2 molecules/ASU

can be positioned, long stretches of rRNA backbone traced, and proteins of unknown structure located. The SRL, L6, L11, L14 and the 23S rRNA domain that binds L11, which together constitute most of the ribosome's factor-binding site, have all been located. The central position of the SRL in that site suggests it may stimulate the GTPase activity of factors like EF-G and EF-Tu when they are ribosome-bound. As data extending to 3Å resolution can be obtained from these crystals, computation of a map at significantly higher resolution and its fitting by a complete atomic model of the 50S ribosomal subunit may soon be possible.

Methods

Crystals. *H. marismortui* (ATCC 43049) was grown and 50S subunits prepared and crystallized as described ${}^{21,46}$. The $C222_1$ and merohedrally twinned $P2_1$ crystals grow under the same conditions and cannot be distinguished by gross morphology. $C222_1$ crystals with shorter $c$ axis ($540 \AA$) were obtained by increasing the concentration of PEG 6000. Crystals were stabilized by gradual transfer into a solution containing ethylene glycol in addition to other components of the crystallization buffer, and flash-frozen in liquid propane. Heavy atom derivatives were prepared by soaking crystals in stabilization solutions containing the heavy atom compound of interest.

X-ray data collection and processing. All data were collected at the National Synchrotron Light Source (Brookhaven) from crystals frozen at 100K, using beamlines X12b and X12c. For each heavy-atom derivative, anomalous diffraction data were recorded using either a 345-mm MAR imaging plate or a $2 \times 2$ CCD detector (Brandeis or Quantum). The beam size was $200 \times 200 \mu$m. Crystals were aligned along the long axis of the unit cell so that $1.0^\circ$ oscillations could be used to collect reflections out to $3.2 \AA$ resolution at the edge of the MAR detector. The crystal-to-detector distances varied between 450.0 mm and 550.0 mm depending on wavelength, crystal quality and beam divergence. Data sets were processed by using DENZO and SCALEPACK ${}^{47}$.

MIRAS phasing. The heavy-atom positions for the major binding sites were determined from combination difference anomalous and isomorphous Patterson maps at various resolution ranges, and minor sites and osmium hexamine sites were found by difference Fouriers; heavy-atom cluster compounds and the refinement of their parameters have been described ${}^{21}$. These phases were combined with SAD phases calculated using each derivative separately. All aspects of the isomorphous and anomalous phasing, including difference Patterson, difference Fourier map calculation and heavy-atom cluster rigid-body refinement against lack of closure, were done using the CNS program package ${}^{48}$.

Non-crystallographic averaging and detwinning. MIRAS/SAD combined phases were improved by intercrystal and non-crystallographic averaging between three different crystal forms: the $C222_1$ form, the twinned $P2_1$ form, and the $C222_1$ form with reduced $c$ axis (Fig. 6). The merohedrally twinned data were detwinned at the level of intensities when the twinning fraction was below 35% and by back-transforming the map when the twinning was complete ${}^{49}$. As the second method of detwinning is more accurate when the phase estimates improve, it was combined with averaging and applied every ten cycles. Phases were improved by non-crystallographic and intercrystal averaging between four views of the large ribosomal subunit in three crystal forms using DMMULTI ${}^{22}$ and RAVE ${}^{23}$ program packages. Several hundred cycles of averaging were performed starting from low-resolution reflections where average MIRAS/SAD figures of merit were greater than 0.5. The improved phases were combined with original MIRAS/SAD probabilities and subjected to solvent flipping using SOLOMON ${}^{50}$ as implemented in CNS ${}^{48}$. For the final map calculation, to compensate for the overestimation of phase accuracy, phase probabilities were 'blurred' by applying a $125 \AA^2$ temperature factor.

Received 2 July; accepted 19 July 1999.

1. Monro, R. E. Catalysis of peptide bond formation by 50S ribosomal subunits from *Escherichia coli*. *J. Mol. Biol.* **26**, 147–151 (1967).
2. Wittmann-Liebold, B., in *Structure, Function, and Genetics of Ribosomes* (eds Hardesty, B. & Kramer, G.) 326–361 (Springer, New York, 1986).
3. Frank, J. *et al.* A model for protein synthesis based on cryo-electron microscopy of the *E. coli* ribosome. *Nature* **376**, 441–444 (1995).
4. Stark, H. *et al.* Arrangement of tRNAs in pre- and posttranslocational ribosomes revealed by electron cryomicroscopy. *Cell* **88**, 19–28 (1997).

Summary

We have obtained a 5Å-resolution map of the large ribosomal subunit in which proteins and rRNA segments of known structure

© 1999 Macmillan Magazines Ltd
NATURE | VOL 400 | 26 AUGUST 1999 | www.nature.com

5. Milligan, R. A. & Unwin, P. N. T. Location of exit channel for nascent protein in 80S ribosome. *Nature* **319**, 693–695 (1986).

6. Beckmann, R. *et al.* Alignment of conduits for the nascent polypeptide chain in the ribosome–Sec61 complex. *Science* **278**, 2123–2128 (1997).

7. Liljas, A. in *The Ribosome. Structure, Function & Evolution* (eds Hill, W. E. *et al.*) 309–317 (Am. Soc. Microbiol., Washington DC, 1990).

8. Capel, M. S. *et al.* A complete mapping of the proteins in the small ribosomal subunit of *E. coli*. *Science* **238**, 1403–1406 (1987).

9. May, R. P., Nowotny, V., Nowotny, P., Voss, H. & Nierhaus, K. H. Inter-protein distances within the large subunit from *Escherichia coli* ribosomes. *EMBO J.* **11**, 373–378 (1992).

10. Oakes, M. Henderson, E., Scheinman, A., Clark, M. & Lake, J. A. in *Structure, Function and Genetics of Ribosomes* (eds Hardesty, B. & Kramer, G.) 47–67 (Springer, New York, 1986).

11. Stoeffler, G. & Stoeffler-Meilicke, M. in *Structure, Function, and Genetics of Ribosomes* (eds Hardesty, B. & Kramer, G.) 28–46 (Springer, New York, 1986).

12. Ramakrishnan, A. & White, S. W. Ribosomal protein structures: insights into the architecture, machinery and evolution of the ribosome. *Trends Biochem. Sci.* **23**, 208–212 (1998).

13. Szewczak, A. A. & Moore, P. B. The sarcin/ricin loop, a modular RNA. *J. Mol. Biol.* **247**, 81–98 (1995).

14. Correll, C. C. *et al.* Crystal structure of the ribosomal RNA loop essential for binding both elongation factors. *Proc. Natl Acad. Sci. USA* **95**, 13436–13441 (1998).

15. Correll, C. C., Freeborn, B., Moore, P. B. & Steitz, T. A. Metals, motifs and recognition in the crystal structure of a 5S rRNA domain. *Cell* **91**, 705–712 (1997).

16. Dallas, A. & Moore, P. B. The solution structure of the loop E/loop D region of *E. coli* 5S rRNA. *Structure* **5**, 1639–1653 (1997).

17. Wimberly, B. T., Guymon, R., McCutcheon, J. P., White, S. W. & Ramakrishnan, V. R. A detailed view of a ribosomal active site: the structure of the L11–RNA complex. *Cell* **97**, 491–502 (1999).

18. Conn, G. L., Draper, D. E., Lattman, E. E. & Gittis, A. G. Crystal structure of a conserved ribosomal protein RNA complex. *Science* **284**, 1171–1174 (1999).

19. Agrawal, R. K., Penczek, P., Grassucci, R. A. & Frank, J. Visualization of elongation factor G on the *Escherichia coli* 70S ribosome: the mechanism of translocation. *Proc. Natl Acad. Sci. USA* **95**, 6134–6138 (1998).

20. Stark, H. *et al.* Visualization of elongation factor Tu on the *Escherichia coli* ribosome. *Nature* **389**, 403–406 (1997).

21. Ban, N. *et al.* A 9 Å resolution X-ray crystallographic map of the large ribosomal subunit. *Cell* **93**, 1105–1115 (1998).

22. Cowtan, K. D. An automated procedure for phase improvement by density modification. *Joint CCP4 and ESF-EACBM Newsletter on Protein Crystallography* **31**, 34–38 (1994).

23. Jones, T. A. in *CCP4 Study Weekend, Molecular Replacement* (eds Dodson, E. J., Glover, S. & Wolf, W.) 91–105 (SSRC, Daresbury Laboratory, Warrington, Cheshire, UK, 1992).

24. Bernabeu, C. & Lake, J. A. Nascent polypeptide chains emerge from the exit domain of the large ribosomal subunit: Immune mapping of the nascent chain. *Proc. Natl Acad. Sci. USA* **79**, 3111–3115 (1982).

25. Kleywegt, G. J. & Jones, T. A. Template convolution to enhance or detect structural features in macromolecular electron-density maps. *Acta. Crystallogr. D* **53**, 179–185 (1997).

26. Cate, J. *et al.* Crystal structure of a group I ribozyme domain: principles of RNA packing. *Science* **273**, 1678–1685 (1996).

27. Jucker, F. M., Heus, H. A., Yip, P. F., Moors, E. H. M. & Pardi, A. A network of heterogeneous hydrogen bonds in GNRA tetraloops. *J. Mol. Biol.* **264**, 968–980 (1996).

28. Stern, S., Powers, T., Changchien, L.-I. & Noller, H. F. RNA–proteins interactions in 30S ribosomal subunits: folding and function of 16S rRNA. *Science* **244**, 783–790 (1989).

29. Golden, B. L., Ramakrishnan, V. & White, S. W. Ribosomal protein L6. Structural evidence of gene duplication from a primitive RNA binding protein. *EMBO J.* **12**, 4901–4908 (1993).

30. Leffers, H., Kjems, J., Ostergaard, L., Larsen, N. & Garrett, R. A. Evolutionary relationships amongst archaeabacteria: a comparative study of 23S ribosomal RNAs of a sulphur-dependent extreme thermophile, an extreme halophile, and a thermophilic methanogen. *J. Mol. Biol.* **195**, 43–61 (1987).

31. Leffers, H., Egeberg, J., Andersen, A., Christensen, T. & Garrett, R. A. Domain VI of *Escherichia coli* 23S ribosomal RNA structure, assembly and function. *J. Mol. Biol.* **204**, 507–522 (1988).

32. Uchiumi, T., Sato, N., Wada, A. & Hachimori, A. Interaction of the sarcin/ricin domain of 23S ribosomal RNA with proteins L3 and L6. *J. Biol. Chem.* **274**, 681–686 (1999).

33. Urlaub, H., Kruft, V., Bischof, O., Müller, E. C. & Wittmann-Liebold, B. Protein–rRNA binding features and their structural and functional implications in ribosomes as determined by crosslinking studies. *EMBO J.* **14**, 4578–4588 (1995).

34. Davies, C., White, S. W. & Ramakrishnan, V. The crystal structure of ribosomal protein L14 reveals an important organizational component of the translational apparatus. *Structure* **4**, 55–66 (1996).

35. Walleczek, J., Schuler, D., Stoeffler-Meilicke, M., Brimacombe, R. & Stoeffler, G. A model for the spatial arrangement of the proteins in the large subunit of the *Escherichia coli* ribosome. *EMBO J.* **7**, 3571–3576 (1988).

36. Nakagawa, A. *et al.* The three-dimensional structure of the RNA-binding domain of ribosomal protein L2; a protein at the peptidyl transferase center of the ribosome. *EMBO J.* **18**, 1459–1467 (1999).

37. Wilson, K. S. & Noller, H. H. Mapping the position of translational elongation factor EF-G in the ribosome by directed hydroxyl radical probing. *Cell* **92**, 131–139 (1998).

38. Aevazsson, A. *et al.* Three-dimensional structure of the ribosomal translocase: elongation factor G from *Thermus thermophilus*. *EMBO J.* **13**, 3669–3677 (1994).

39. Czworkowski, J., Wang, J., Steitz, T. A. & Moore, P. B. The crystal structure of elongation factor G complexed with GDP at 2.7 Å resolution. *EMBO J.* **13**, 3661–3668 (1994).

40. Nissen, P. *et al.* Crystal structure of the ternary complex of Phe-tRNAPhe, EF-Tu and a GTP analog. *Science* **270**, 1464–1472 (1995).

41. Moazed, D., Robertson, J. M. & Noller, H. F. Interaction of elongation factors EF-G and EF-Tu with a conserved loop in 23S RNA. *Nature* **334**, 362–364 (1988).

42. Traut, R. R. *et al.* in *Structure, Function, and Genetics of Ribosomes* (eds Hardesty, B. & Kramer, G.) 286–308 (Springer, New York, 1986).

43. Skold, S. E. Chemical crosslinking of elongation factor G to both subunits of the 70S ribosome from *E. coli*. *Eur. J. Biochem.* **127**, 225–229 (1982).

44. Skold, S. E. Chemical crosslinking of elongation factor G to the 23S rRNA in 70S ribosomes from *Escherichia coli*. *Nucleic Acids Res.* **11**, 4923–4932 (1983).

45. Nevskaya, N. *et al.* Structure of archaeal ribosomal protein L1 provides new insights into the RNA binding for the L1 protein family. *J. Mol. Biol.* (in the press).

46. von Bohlen, K. *et al.* Characterization and preliminary attempts for derivitization of crystals of large ribosomal subunits from *Haloarcula marismortui* diffracting to 3 Å resolution. *J. Mol. Biol.* **222**, 11–15 (1991).

47. Otwinowski, Z. in *Data Collection and Processing* (eds Sawyer, L., Isaacs, N. & Bailey, D.) 52–62 (SERC Daresbury Laboratory, Warrington, UK, 1993).

48. Brünger, A. T. *et al.* Crystallography and NMR system (CNS): a new software suite for macromolecular structure determination. *Acta Crystallogr. D* **54**, 905–921 (1998).

49. Yeates, T. O. Detecting and overcoming crystal twinning. *Methods Enzymol.* **276**, 344–358 (1997).

50. Abrahams, J. P. & Leslie, A. G. W. Methods used in the structural determination of bovine mitochondrial F1 ATPase. *Acta Crystallogr. D* **52**, 30–43 (1996).

Acknowledgements. We thank E. Freeborn for biochemical support; R. Sweet for helping us with data collection at the National Synchrotron Light Source; and I. Tanaka, M. Garber and S. Al-Karadaghi for providing coordinates before publication. This investigation was supported by grants from NIH to T.A.S. and P.B.M. N.B. was a Damon Runyon-Walter Winchell Postdoctoral Fellow during part of this study, and P.N. is supported by the Danish Research Council. The coordinates of all identified RNA and protein structures have been deposited in the protein data bank with accession number IC04.

Correspondence and requests for materials should be addressed to T.A.S.
