
links’ between the different loop domains. This provides a rationale for organization of, for example, the 4.5-Mb chromosome of *E. coli* into loop domains of approximately this size.^[15,16]

Received 22 December 1998; accepted 18 August 1999.

1. Rybenkov, V. V., Ullsperger, C., Vologodskii, A. V. & Cozzarelli, N. R. Simplification of DNA topology below equilibrium values by type II topoisomerases. *Science* **277**, 690–693 (1997).
2. Hopfield, J. J. Kinetic proofreading: a new mechanism for reducing errors in biosynthetic processes requiring high specificity. *Proc. Natl Acad. Sci. USA* **71**, 4135–4139 (1974).
3. Ninio, J. Kinetic amplification of enzyme discrimination. *Biochimie* **57**, 587–595 (1975).
4. Frank-Kamenetskii, M. D., Lukashin, A. V. & Vologodskii, A. V. Statistical mechanics and topology of polymer chains. *Nature* **258**, 398–402 (1975).
5. Klenin, K. V., Vologodskii, A. V., Anshelevich, V. V., Dykhne, A. M. & Frank-Kamenetskii, M. D. Effect of excluded volume on topological properties of circular DNA. *J. Biomol. Struct. Dyn.* **5**, 1173–1185 (1988).
6. Shaw, S. Y. & Wang, J. C. Knotting of a DNA chain during ring closure. *Science* **260**, 533–536 (1993).
7. Rybenkov, A. V., Cozzarelli, N. R. & Vologodskii, A. V. Probability of DNA knotting and the effective diameter of the DNA double helix. *Proc. Natl Acad. Sci. USA* **90**, 5307–5311 (1993).
8. Vologodskii, A. V. in RECOMB 98: Proceedings of the second annual international conference on computational molecular biology 266–269 (Association for Computing Machinery, New York, 1998).
9. Deguchi, T. & Tsurusaki, K. A statistical study of random knotting using the Vassiliev invariants. *J. Knot Theory Ramific.* **3**, 321–353 (1994).
10. Roca, J. & Wang, J. C. DNA transport by a type II DNA topoisomerase: evidence in favor of a two-gate mechanism. *Cell* **77**, 609–616 (1994).
11. Roca, J. & Wang, J. C. The capture of a DNA double helix by an ATP-dependent protein clamp: a key step in DNA transport by type II DNA topoisomerases. *Cell* **71**, 833–840 (1992).
12. Roca, J., Berger, J. M. & Wang, J. C. On the simultaneous binding of eukaryotic DNA topoisomerase II to a pair of double-stranded DNA helices. *J. Biol. Chem.* **268**, 14250–14255 (1993).
13. Harkins, T. T. & Lindsley, J. E. Pre-steady-state analysis of ATP hydrolysis by *Saccaromyces cerevisiae* DNA topoisomerase II. 1. A DNA-dependent burst in ATP hydrolysis. *Biochemistry* **37**, 7292–7299 (1998).
14. Harkins, T. T. & Lindsley, J. E. Pre-steady-state analysis of ATP hydrolysis by *Saccaromyces cerevisiae* DNA topoisomerase II. 2. Kinetic mechanism for the sequential hydrolysis of two ATP. *Biochemistry* **37**, 7299–7312 (1998).
15. Worcel, A. & Burgi, E. On the structure of the folded chromosome of *Escherichia coli*. *J. Mol. Biol.* **71**, 127–147 (1972).
16. Sinden, R. R. & Pettijohn, D. E. Chromosomes in living *Escherichia coli* cells are segregated into domains of supercoiling. *Proc. Natl Acad. Sci. USA* **78**, 224–228 (1981).

### Acknowledgements

We thank D. Chatenay, N. R. Cozzarelli, G. B. Mindlin, V. Rybenkov, E. D. Siggia, A. V. Vologodskii and E. L. Zechiedrich for discussions. J.F.M. and J.Y. acknowledge the support of the NSF, the Research Corporation, the Petroleum Research Fund, and the Whitaker Foundation. M.O.M. acknowledges support of the Sloan Foundation and the Mathers Foundation.

Correspondence and requests for materials should be addressed to J.F.M. (e-mail: jmarko@uic.edu).

---

Table 1 Crystallographic data and refinement statistics

| Data and refinement statistics (values in parentheses are for the resolution bin 2.53–2.40 Å) |
| --- |
| Space group | C2 |
| Cell dimensions | $a = 165.51 \, \text{\AA}, b = 95.87 \, \text{\AA}, c = 211.77 \, \text{\AA}$, $\beta = 106.83^\circ$ |
| Resolution range | 25–2.4 Å |
| No. of reflections | 103,327 (10,696) |
| Completeness | 84.3% (59.9%) |
| Multiplicity | 2.7 (1.7) |
| $R_{\text{merge}}$ | 0.132 (0.287) |
| $R$-factor | 0.232 (0.312) |
| $R_{\text{free}}$ value | 0.265 (0.365) |
| No. of atoms | 12,623 |
| No. of reflections used in refinement | 101,663 |
| No. of reflections used for $R_{\text{free}}$ | 1,662 |
| Solvent content | 73.1% |
| No. of protein atoms (6 × 264 residues) | 12,042 |
| No. of water molecules | 581 |
| $B$-value from Wilson plot (3.5–2.4 Å) | $35.3 \, \text{\AA}^2$ |
| Mean $B$-value | $35.2 \, \text{\AA}^2$ |
| Average protein $B$-value | $35.0 \, \text{\AA}^2$ |
| Average water $B$-value | $38.0 \, \text{\AA}^2$ |
| Ramachandran plot of non-glycine and non-proline residues | Most favourable regions | 1,102 (78.5%) |
| Additional allowed regions | 275 (19.6%) |
| Generously allowed regions | 17 (1.2%) |
| Disallowed regions | 10 (0.7%) |
| r.m.s. deviations from ideal values | Bond distances | $0.006 \, \text{\AA}$ |
| Angles | $1.4^\circ$ |

---

$R_{\text{merge}} = \Sigma_{hkl} \Sigma_i |I_{hkl}^i - \langle I_{hkl} \rangle| / \Sigma_{hkl} \Sigma_i \langle I_{hkl}^i \rangle$ where the sum $i$ is over all separate measurements of the unique reflections $hkl$.

$R$-factor $= \Sigma_{hkl} ||F_{\text{obs}}| - |F_{\text{calc}}|| / \Sigma_{hkl} |F_{\text{obs}}|$

$R_{\text{free}}$, as $R$-factor but summed only over the test reflections

$B$-value $= 8\pi^2 \langle u^2 \rangle$ where $\langle u^2 \rangle$ is the apparent mean square deviation from the atomic position.

---

shaft domain contains a repeating sequence motif with an invariant glycine or proline and a conserved pattern of hydrophobic residues. Here we describe the crystal structure at 2.4 Å resolution of a recombinant protein containing the four distal repeats of the adenovirus type 2 fibre shaft plus the receptor-binding head domain. The structure reveals a novel triple β-spiral fibrous fold for the shaft. Implications for folding of fibrous proteins (misfolding of shaft peptides leads to amyloid-like fibrils) and for the design of a new class of artificial, silk-like fibrous materials are discussed.

The human adenovirus serotype 2 (Ad2) fibre is a trimer of 582 residues per monomer, of which the head domain is essential for trimerization and autonomously trimerizes when expressed. The high-resolution structures of the heads of Ad2 and Ad5 are known and are very similar: each head monomer forms an eight-stranded anti-parallel β-sandwich structure and the three monomers interact to form a three-bladed propeller. The head domain is responsible for binding to the cell receptor, which has been identified to be a human protein of unknown function: the coxsackievirus and adenovirus receptor. This protein serves as the receptor for coxsackieviruses of subgroup B and adenoviruses of all subgroups except subgroup B (ref. 12). The primary sequence of the fibre shaft consists of 15-residue pseudo-repeats (22 of them for Ad2 and Ad5, ref. 7). Green et al. predicted that these repeats contain two β-strands and two turns (the cross-β model). Stouten et al. subsequently proposed a triple β-helical model, taking into account length measurements from electron microscopy and fibre diffraction patterns. The full-length fibre is very stable, resistant to heat (its melting temperature is 85°C, ref. 15) and detergents (at low temperatures), and the shaft domain is highly resistant to proteases.

The Ad2 fibre unfolds through a stable intermediate in which the C-terminal head and distal part of the shaft remain folded and trimeric. The stable domain has been identified to span residues 319–582 and has been cloned and expressed in *Escherichia coli* (M.J.v.R. et al., unpublished results). The recombinant protein was

---

### A triple β-spiral in the adenovirus fibre shaft reveals a new structural motif for a fibrous protein

**Mark J. van Raaij**, **Anna Mitraki**, **Gilles Lavigne** & **Stephen Cusack**

*European Molecular Biology Laboratory, Grenoble Outstation,*  
*c/o Institut Laue Langevin, BP 156, 38042 Grenoble Cedex 9, France*

*Institut de Biologie Structurale (CEA-CNRS), 41 rue Jules Horowitz,*  
*38027 Grenoble Cedex 1, France*

Human adenoviruses are responsible for respiratory, gastroenteric and ocular infections and can serve as gene therapy vectors. They form icosahedral particles with 240 copies of the trimeric hexon protein arranged on the planes and a penton complex at each of the twelve vertices. The penton consists of a pentameric base, implicated in virus internalization, and a protruding trimeric fibre, responsible for receptor attachment. The fibres are homo-trimeric proteins containing an amino-terminal penton base attachment domain, a long, thin central shaft and a carboxy-terminal cell attachment or head domain. The

trimeric and crystallized in space group C2. Although the crystals diffract well, diffraction patterns show strong diffuse scattering connecting spots along the c-axis. A dataset of 25–2.4 Å resolution was collected (Table 1) and the structure solved by molecular replacement using the high-resolution structure of the Ad2 head domain alone⁹. Into the resulting electron density map, a complete model for all the extra shaft residues could be built. The final model was refined with CNS programs¹⁸ and has reasonable stereochemistry (Table 1). The asymmetric unit contains two trimers and 581 water molecules.

The shaft structure reveals a new fold, which we call a ‘triple β-spiral’ (Fig. 1). The repeating motif of 15 residues (a–o) exists in two types, with either a proline or a glycine at position j (Fig. 2). The four repeats in our structure are all of the glycine type. Each repeat comprises an extended strand (residues b–h) which runs parallel to the fibre axis, followed by a type 2 β-turn, containing the conserved glycine with φ around 80° and ψ around 0° (φ and ψ are polypeptide chain backbone dihedral angles). A proline at this position may impose a type 1 turn (φ around –90 deg; and ψ around 0°). The turn is followed by another β-strand (residues k–n) which runs backwards at about 45° to the shaft axis. The repeats are joined together by a solvent-exposed variable loop (residues o–a). The intra-chain transformation from one repeat to the next is a displacement along the shaft axis of about 13 Å and clockwise rotation of just over 50° (as seen from the N to the C terminus); the axial displacement agrees with previous measurements by electron microscopy¹⁹. Three conserved hydrophobic residues in each repeat (residues c, e and k in Fig. 2; shown in orange in Fig. 3a) point towards the central three-fold axis, forming a longitudinal hydrophobic core with the equivalent residues from the two other chains. Two other residues, g and m (shown in green in Fig. 3a), form stabilizing hydrophobic patches at greater radius. The basic framework of the structure is maintained by conserved inter- and intra-chain main-chain hydrogen bonds as shown in Fig. 3b. These interactions are supplemented by non-conserved inter- and intra-chain hydrogen bonds and salt bridges involving side-chains (not shown).

The resulting structure is a complex but regular, highly cross-linked structure which, together with a high proportion of buried surface (one-third of solvent-accessible surface of a shaft monomer is buried upon trimer formation), accounts for the high rigidity and stability of the shaft. The average diameter of the shaft, excluding the surface loops, is 15 Å, and the radius of the large loop (residues 346–355) is 22 Å. Figure 4 shows a model of a fibre 300 Å long (the full length of the Ad2 fibre) to give an impression of the proportions of the molecule. The fold of the adenovirus shaft is unique amongst protein structures so far determined and different from both the cross-β model⁶ and the triple-helical model¹³. The difference lies largely in the fact that the β-strands are oriented more along the fibre axis than transverse to it, as previously assumed. Further, the intra-chain hydrogen bonding pattern shown in Fig. 3b suggests that the basic repeating structural module should be redefined as starting from residue g and ending at residue f. This redefinition naturally leads to a clear end of the regular shaft structure at residue Gly 392. This residue is followed by a linker region between the shaft and the head domains (residues 393–398), which was originally defined as part of repeat number 22 of the shaft. The electron density is not well defined for residues 394–396, suggesting that the linker is flexible. There is also a solvent-filled volume around the three-fold axis between the head and shaft, rather than a continuation of the longitudinal hydrophobic core (Fig. 1a). In the crystal,

![Figure 1](#fig1)

**Figure 1** The adenovirus fibre fold. The two parts of this figure are shown in stereo.  
**a**, Chain trace of the Cα atoms of one of the trimers present in the asymmetric unit. Chains A, B and C are shown in red, blue and green respectively. Every tenth residue of the shaft domain in chain A is numbered. **b**, The shaft domain as present in the crystal. The N and C termini of chain A are labelled.

| a b c d e f g h | i j k l m n o |
| --- | --- |
| β β β β β β | β β β |
| tail PPTVPFLTPPFVSPNGFQESPP |  |
| 1 45 G V L S L R V S . . . E P L D T S H . . . . . . . . |  |
| 2 60 G M L A L K M G . . . S G L T L D K A . . . . . . . . |  |
| 3 76 G N L T S Q N V TTVTQP L K K T K . . . . . . . . |  |
| 4 95 S N I S L D T S . . . A P L T I T S . . . . . . . . |  |
| 5 110 G A L T V A T T . . . A P L I V T S . . . . . . . . |  |
| 6 125 G A L S V Q S Q . . . A P L T V Q D . . . . . . . . |  |
| 7 140 S K L S I A T K . . . G P I T V S D . . . . . . . . |  |
| 8 155 G K L A L Q T S . . . A P L S G S D S . . . . . . . . |  |
| 9 171 D T L T V T A S . . . P P L T T A T . . . . . . . . |  |
| 10 186 G S L G I N M E . . . D P I Y V N N . . . . . . . . |  |
| 11 201 G K I G I K I S . . . G P L Q V A Q N S . . . . . . . . |  |
| 12 218 D T L T V V T G . . . P G V T V E Q . . . . . . . . |  |
| 13 233 N S L R T K V A . . . G A I G Y D S S . . . . . . . . |  |
| 14 249 N N M E I K T G . . . G G M R I N N . . . . . . . . |  |
| 15 264 N L L I L D V D . . . Y P F D A Q . . . . . . . . |  |
| 16 278 T K L R L K L G Q . . . G P L Y I N A S . . . . . . . . |  |
| 17 295 H N L D I N Y N . . . R G L Y L F N A S N N T . . . |  |
| 18 315 K K L E V S I K K S . . S G L N F D N . . . . . . . . |  |
| 19 332 T A I A I N A G . . . K G L E F D T N T S E S P D I |  |
| 20 355 N P I K T K I G . . . S G I D Y N E N . . . . . . . . |  |
| 21 371 G A M I T K L G . . . S G L S F D N S . . . . . . . . |  |
| 22 387 G A I T I GNKNDDKLT LWTT PDP head |  |
| P X X φ X φ X φ X . . . X P φ X φ X X . . . . . . . . |  |
| G X X φ X φ X φ G . . . X G φ X φ X X . . . . . . . . |  |
| a b c d e f g h | i j k l m n o |

**Figure 2** Alignment of the 22 repeats¹⁵ in the Ad2 shaft. The repeating residues are labelled a to o. Residues forming β-strands are indicated with β. The conserved glycines or prolines in the tight turn (purple, except the corresponding residue in repeat 13, which is an alanine), residues involved in the hydrophobic core (orange), and residues forming the peripheral hydrophobic patches (green) are shown. Some residues of the fibre tail domain (23–44) are at the top (italics). Repeat 3 (shown in italics) cannot be aligned reliably and may have a different structure. The spacer (393-NKNDDK-398) between the shaft and the head and the first nine residues of the head domain are also in italics. The consensus sequences of the proline and glycine repeats (bottom, P and G) are shown (φ is hydrophobic).
the three-fold shaft axis is tilted with respect to the three-fold axis running through the head domain by ~2°, further confirming this flexibility. This implies the possibility of relative movement between the head and shaft, which may be required to optimise receptor binding. Interestingly, this spacer region is not conserved in adenoviruses of subgroup B, which use a different, as yet unknown, primary receptor¹².

Comparisons of adenovirus serotypes⁷ show that the shaft sequence can accommodate sequence variations, insertions and deletions. The most variable element is the solvent-exposed loop, for which this structure shows examples of 4, 5 and 12-residue turns. This loop can contain insertions of up to eight residues and could possibly be used to insert foreign sequences (potentially useful for gene therapy), while preserving a correctly folded fibre. Whereas other fibrous proteins contain repetitive sequences, the presence of a variable loop appears to be a unique feature of the adenovirus fibre shaft fold. In the Ad2 sequence, the longer turns are concentrated at the C-terminal end of the shaft and may constitute serotype-specific antigenic regions, as the sequence of these loops varies between Ad2 and Ad5. The compact glycine or proline loops are more highly conserved four-residue turns, with some exceptions, such as the insertion of two residues (323-Lys–Ser-324) in the glycine turn of repeat 18 in this structure. Amplification of insertions in this region occurs in some non-human adenovirus shaft sequences²⁰,²¹. The breakdown of the consensus sequence in the third repeat (Fig. 2) has been correlated to a kink in the shaft at this point, as observed by electron microscopy¹⁹. Similar putative hinge regions correlated with aberrant repeats have been described in the very long bovine adenovirus fibre shaft²⁰ and in certain avian adenovirus fibres²¹. High stability, length and rigidity, together with well-defined hinge positions in the shaft or at the head–shaft

a

![Diagram](attachment:diagram_a.png)

b

![Diagram](attachment:diagram_b.png)

Figure 3 Interactions in the shaft. **a**, Hydrophobic core. The main chain of residues 356–389 of monomers A (red), B (blue) and C (yellow) is shown. Residues contributing to the central hydrophobic core (orange) and residues contributing to the peripheral hydrophobic patches (green) are also given. **b**, Diagram of the hydrogen-bonding network within one monomer and between adjacent monomers. Residues are labelled according to Fig. 2. Intra-chain (dotted lines) and inter-chain hydrogen bonds (solid black lines) that are conserved in each of the four repeats of the structure are shown.

boundary, are clearly desirable properties for an extra-cellular viral protein required to bind to particular receptors amongst many other cell surface molecules.

Most known fibrous protein structures are based on α-helical coiled coils or the collagen triple helix²². Some elongated proteins such as the bacteriophage P22 tailspike²³ and pectate lyase²⁴ are based on a β-helix. The triple β-spiral is quite distinct from these. It is also very different from the crystalline structural region in silk²⁵. However, sequence similarities shared with T4 bacteriophage short and long fibres indicate that this novel structural motif may exist in other proteins. The protein sequences of the bacteriophage T4 short and long fibre shafts show consensus repeats of 17 residues (xTxxxxGφφXφxTxxxφ and xGxxxxφφTPxxφxxx; x is any amino acid, φ is hydrophobic), which contain two predicted β-strands connected by a predicted β-turn in which a glycine or proline is conserved²⁶. We note that the positions of the conserved hydrophobic residues relative to the conserved glycine or proline are not the same as in the adenovirus shaft motif, so some structural differences must also exist. It remains to be seen how many members there are of this new structural class of fibrous proteins.

To assemble into the triple β-spiral, the shaft repeats need to be brought into the correct registration. Synthetic peptides containing shaft residues 355–395 fail to assemble correctly, but aggregate,

![Image](attachment:image_1.png)

Figure 4 Space-filled representation of a model containing 21 repeats of the shaft. The model contains residues 321–582 of chains A (red), B (blue) and C (green), to which copies of residues 361–376 (one repeat) were added by repeated application of the inter-motif transformation to obtain a shaft length of 21 repeats or 265 Å (about the length of the entire Ad2 fibre shaft). We note that seven inter-motif transformations provide a rotation through 360°.
probably through out-of-register interactions, and form fibrils similar to amyloid fibrils (A.M. *et al.*, unpublished results). The head domain probably acts as the registration signal for correct folding, as do the C-terminal domains of T4 fibritin²⁷ and pro-collagen peptides. Our structural information, combined with folding studies, could be used to clarify the mechanisms that control correct assembly or misassembly in fibrous β-sheet proteins, as was done for the P22 tailspike²⁸. Because of its stability and morphology, the adenovirus fibre has served as a model for synthetic fibre design. O'Brien *et al.*²⁹ have designed recombinant polymers using simplified consensus adenovirus shaft repeats as building blocks, which were bacterially expressed as inclusion bodies, refolded, purified and spun into fibres. In one case, using the sequence NAL-RIKLGSGLDFN as a 26-fold repeating motif, they obtained ordered fibres with mechanical properties comparable to several commercial textile fibres. Our structural information on the fibre shaft may stimulate further design of synthetic fibres with improved or novel properties, for example by engineering a simple trimerization domain coupled to a certain number of shaft repeats to obtain high tensile strength fibres of a defined length.

### Methods

#### Crystallization and structure solution

A C-terminal part of the adenovirus type 2 fibre protein containing residues 319–582 was expressed, purified and crystallized (M.J.v.R. *et al.*, unpublished data). Crystals were obtained using the sitting-drop vapour-diffusion method by combining 1 μl of protein solution (14 mg ml⁻¹) and 1 μl precipitant (0.8 M ammonium sulphate, 0.1 M sodium citrate, pH 4.0) and equilibrating at 20°C against the precipitant solution. A cluster of plate-like crystals was obtained after six months; a single crystal (0.4 × 0.15 × 0.05 mm) was obtained by transferring the cluster to the precipitant solution containing 25% by volume glycerol and separating it with a glass hair probe. It was flash-frozen in a cryostream at 100 K. Data was collected on station ID14eh3 at the European Synchrotron Radiation Facility. Diffraction images were processed using CCP4 programs³⁰. The structure was solved by molecular replacement (AMORE³⁰) using the Ad2 fibre head structure⁸. Apart from a side-chain flip of Phe 480, the structure of the head domain does not change in the presence of the shaft, although it is now apparent that residues 388–395, which form the putative last repeat of the shaft, were forced into an unnatural conformation by crystal packing in the structure of the head alone. For calculation of the free R-factor, a subset of reflections was selected in thin shells of resolution. No non-crystallographic symmetry restraints were used in the refinement as crystal contacts result in the six monomer chains being slightly different.

Received 18 June; accepted 26 August 1999.

1. Schenk, T. in *Fields Virology* 2111–2148 (Lippincott-Raven, Philadelphia, 1996).
2. Horwitz, M. S. in *Fields Virology* 2149–2171 (Lippincott-Raven, Philadelphia, 1996).
3. Robbins, P.D., Tahara, H. & Ghivizzani, S.C. Viral vectors for gene therapy. *Trends Biotechnol.* **16**, 35–40 (1998).
4. Wickham, T.J., Mathias, P., Cheresh, D.A. & Nemerow, G.R. Integrins αvβ3 and αvβ5 promote adenovirus internalization but not virus attachment. *Cell* **73**, 309–319 (1993).
5. Philipson, L., Lonberg-Holm, K. & Petterson, U. Virus-receptor interaction in an adenovirus system. *J. Virol.* **2**, 1064–1075 (1986).
6. Green, N.M., Wrigley, N.G., Russel, W.C., Martin, S.R. & McLachlan, A. Evidence for a repeating cross β structure in the adenovirus fibre. *EMBO J.* **2**, 1357–1365 (1983).

7. Chroboczek, J., Ruigrok, R.W.H. & Cusack, S. Adenovirus fiber. *Curr. Top. Microbiol. Immunol.* **199**, 163–200 (1995).
8. Hong, J.S. & Engler, J.A. Domains required for assembly of adenovirus type 2 fiber trimers. *J. Virol.* **70**, 7071–7078 (1996).
9. van Raaij, M.J., Louis, N., Chroboczek, J. & Cusack, S. Structure of the human adenovirus serotype 2 fiber head domain at 1.5 Å resolution. *Virology* **262**, 333–343 (1999).
10. Xia, D., Henry, L.J., Gerard, R.D. & Deisenhofer, J. Crystal structure of the receptor-binding domain of adenovirus type 5 fiber protein at 1.7 Å resolution. *Structure* **2**, 1259–1270 (1994).
11. Bergelson, J.M. *et al.* Isolation of a common receptor for coxsackie B viruses and adenoviruses 2 and 5. *Science* **275**, 1320–1323 (1997).
12. Roelvink, P.W. *et al.* The coxsackievirus-adenovirus receptor protein can function as a cellular attachment protein for adenovirus serotypes from subgroups A, C, D, E, and F. *J. Virol.* **72**, 7909–7915 (1998).
13. Stouten, P.F.W., Sander, C., Ruigrok, R.W.H. & Cusack, S. New triple-helical model for the shaft of the adenovirus fibre. *J. Mol. Biol.* **226**, 1073–1084 (1992).
14. Devaux, C., Adrian, M., Berthet-Colominas, C., Cuasck, S. & Jacrot, B. Structure of adenovirus fibre. I. Analysis of crystals of fibre from adenovirus serotypes 2 and 5 by electron microscopy and X-ray crystallography. *J. Mol. Biol.* **215**, 567–588 (1990).
15. Louis, N. *Etude de la structure de la fibre de l'adénovirus humain de la sérotype 2; son devenir et son rôle au cours de l'infection*. PhD thesis, Univ. Joseph-Fourier, Grenoble (1994).
16. Mitraki, A. *et al.* Unfolding studies of human adenovirus type 2 fiber trimers: evidence for a stable domain. *Eur. J. Biochem.* **264**, 599–606 (1999).
17. Devaux, C., Caillet-Boudin, M.L., Jacrot, B. & Boulanger, P. Crystallization, enzymatic cleavage, and the polarity of the adenovirus type 2 fiber. *Virology* **161**, 121–128 (1987).
18. Brünger, A.T. *et al.* Crystallography & NMR System: A new software suite for macromolecular structure determination. *Acta Crystallogr. D* **54**, 905–921 (1998).
19. Ruigrok, R.W.H., Barge, A., Albiges-Rizo, C. & Dayan, S. Structure of adenovirus fibre. II. Morphology of single fibres. *J. Mol. Biol.* **215**, 589–596 (1990).
20. Ruigrok, R.W.H., Barge, A., Mittel, S.K. & Jacrot, B. The fibre of bovine adenovirus type 3 is very long but bent. *J. Gen. Virol.* **75**, 2069–2073 (1994).
21. Hess, M., Cuzange, A., Ruigrok, R.W.H., Chroboczek, J. & Jacrot, B. The avian adenovirus penton: two fibres and one base. *J. Mol. Biol.* **252**, 379–385 (1995).
22. Beck, B. & Brodsky, B. Supercoiled protein motifs: The collagen triple-helix and the α-helical coiled coil. *J. Struct. Biol.* **122**, 17–29 (1998).
23. Steinbacher, S. *et al.* Crystal structure of P22 tailspike protein: interdigitated subunits in a thermostable trimer. *Science* **265**, 383–386 (1994).
24. Yoder, M.D., Keen, N.T. & Jurnak, F. New domain motif: Structure of pectate lyase C, a secreted plant virulence factor. *Science* **260**, 1503–1507 (1993).
25. Marsh, R.E., Corey, R.B. & Pauling, L. The structure of Tussah silk fibroin. *Acta Crystallogr.* **8**, 710–715 (1955).
26. Cerritelli, M.E, Walls, J.S., Simon, M.N., Conway, J.F. & Steven, A.C. Stoichiometry and dominant organization of the long-fiber of bacteriophage T4: A hinged viral adhesion. *J. Mol. Biol.* **260**, 767–780 (1996).
27. Tao, Y., Strelkov, S.V., Mesyanzhinov, V. & Rossmann, M.G. Structure of bacteriophage T4 fibritin: A segmented coiled coil and the role of the C-terminal domain. *Structure* **5**, 789–798 (1997).
28. Betts, S. & King, J. There's a right and a wrong way: in vivo and in vitro folding, misfolding and subunit assembly of the P22 tailspike. *Structure* **7**, R131–R139 (1999).
29. O'Brien, J.P. *et al.* in *Silk Polymers, Materials Science and Biotechnology* 104–117 (ACS Symposium Series 544, American Chemical Society, Charlottesville, Virginia, 1993).
30. Collaborative Computational Project Number 4. The CCP4 suite: Programs for protein crystallography. *Acta Crystallogr. D* **50**, 760–763 (1994).

### Acknowledgements

We thank P. Goeltz and N. Cohet for help with cloning, expression and purification and J. Gagnon for comments. M.J.v.R. was supported by an EU Biotech II fellowship. The EMBL-ESRF Joint Structural Biology Group provided synchrotron radiation facilities. Correspondence and requests for materials should be addressed to S.C. (e-mail: cusack@embl-grenoble.fr). Atomic co-ordinates and structure factors are available from the Protein Data Bank under accession numbers 1QIU and R1QIUSF, respectively.
