
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2000 by The American Society for Biochemistry and Molecular Biology, Inc.

**Expression Cloning of a New Member of the ABO Blood Group Glycosyltransferases, iGb₃ Synthase, That Directs the Synthesis of Isoglobo-glycosphingolipids**

Received for publication, March 28, 2000, and in revised form, June 13, 2000  
Published, JBC Papers in Press, June 14, 2000, DOI 10.1074/jbc.M002629200

Jeremy J. Keusch‡§, Stephen M. Manzella‡¶, Kwame A. Nyame∥, Richard D. Cummings∥, and Jacques U. Baenziger‡

From the ‡Department of Pathology, Washington University School of Medicine, St. Louis, Missouri 63110 and the ∥Department of Biochemistry and Molecular Biology, University of Oklahoma Health Sciences Center, Oklahoma City, Oklahoma 73190

The large array of different glycolipids described in mammalian tissues is a reflection, in part, of diverse glycosyltransferase expression. Herein, we describe the cloning of a UDP-galactose: β-D-galactosyl-1,4-glucosylceramide α-1,3-galactosyltransferase (iGb₃ synthase) from a rat placental cDNA expression library. iGb₃ synthase acts on lactosylceramide, LacCer (Galβ1,4Glcβ1Cer) to form iGb₃ (Gala1,3Galβ1,4Glcβ1Cer) initiating the synthesis of the isoglobo-series of glycosphingolipids. The isolated cDNA encoded a predicted protein of 339 amino acids, which shows extensive homology (40–50% identity) to members of the ABO gene family that includes: murine α1,3-galactosyltransferase, Forssman (Gb₅) synthase, and the ABO glycosyltransferases. In contrast to the murine α1,3-galactosyltransferase, iGb₃ synthase preferentially modifies glycolipids over glycoprotein substrates. Reverse transcriptase-polymerase chain reaction revealed a widespread tissue distribution of iGb₃ synthase RNA expression, with high levels observed in spleen, thymus, and skeletal muscle. As an indirect consequence of the expression cloning strategy used, we have been able to identify several potential glycolipid biosynthetic pathways where iGb₃ functions, including the globo- and isoglobo-series of glycolipids.

based on their carbohydrate structure, into six major series in vertebrates: gangliosides, lacto-, neolacto-, muco-, isoglobo-, and globo-series GSL (1). The common precursor to these GSL is LacCer (Galβ1,4Glcβ1Cer). Recently, the vital importance of GSL has been demonstrated through the null mutation of glucosylceramide synthase (2), the first step in the synthesis of the majority of GSL. The significance of individual glycan structures within the major GSL series arising from LacCer remains to be established. To date, animals deficient in glycosyltransferases involved in the terminal modifications have not yielded the prominent or lethal phenotypes associated with the loss of the ability to synthesize core regions of glycans. Targeting null mutations in glycosyltransferases acting at key branch points in GSL biosynthesis will help reveal the contributions of the different series of GSL.

As a first step to this approach we have cloned Gb₃ synthase and iGb₃ synthase, two transferases that act on LacCer and initiate the synthesis of the globo- and isoglobo-series of GSL. Following the synthesis of Gb₃ (Gala1,4Galβ1,4Glcβ1Cer) or iGb₃ (Gala1,3Galβ1,4Glcβ1Cer) (*boldface* is used to show the linkage of saccharide transferred by Gb₃ synthase and/or iGb₃ synthase), sequential addition of GalNAc residues by Gb₄ synthase and Gb₅ synthase leads to the production of both Gb₄ and Gb₅ or iGb₄ and iGb₅, respectively. Using a mAb SMLDN1.1 that detects GalNAc on these downstream products, we have cloned Gb₃ and iGb₃ synthases from a rat placental cDNA expression library. In this paper we described the cloning of iGb₃ synthase, an α1,3-galactosyltransferase, that shares high homology with other glycosyltransferases in the histo-blood group ABO gene family but differs in its substrate specificity.

There are many diverse glycans found in nature that may be synthesized on different aglycone substrates, including proteins and lipids. The glycosphingolipids (GSL)¹ are organized,

* This work was supported by National Institutes of Health Grant R01-DK 41738 (to J. U. B.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) AF248543.
* § To whom correspondence should be addressed: Tel.: 314-362-8733; Fax: 314-362-8888; E-mail: jkeusch@pathbox.wustl.edu.
* ¶ Current address: Dade-Behring Inc., Newark, DE.
* ¹ The abbreviations for the glycosphingolipids are in accordance with the 1997 recommendations of the IUPAC-IUB Joint Commission of Biochemical Nomenclature. The abbreviations used are: GSL, glycosphingolipid; LacCer, lactosylceramide (Galβ1,4Glcβ1Cer); Cer, ceramide; Gal, galactose; GalNAc, N-acetylgalactosamine; Glc, glucose; Gb₃, Pk, CD77 or globotriaosylceramide (Gala1,4Galβ1,4Glcβ1Cer); iGb₃, isoglobotriaosylceramide (Gala1,3Galβ1,4Glcβ1Cer); Gb₄, globoside (GalNAcβ1,3Gala1,4Galβ1,4Glcβ1Cer); iGb₄, isogloboside (GalNAcβ1,3Gala1,4Galβ1,4Glcβ1Cer); Gb₅, Forssman (GalNAca1,3GalNAcβ1,3Gala1,4Galβ1,4Glcβ1Cer); iGb₅, isoForssman (GalNAca1,3GalNAcβ1,3Gala1,4Galβ1,4Glcβ1Cer); G<sub>M3</sub>, Neu5Aca2,3Galβ1,4Alcβ1Cer; mAb, monoclonal antibody; PPPP, 1-phenyl-2-hexadecanoylamino-3-pyrrolidino-1-propanol, hydrochloride; TLC, thin layer chromatography;

EXPERIMENTAL PROCEDURES

Rat Placental cDNA Library—RNA was isolated from normal rat placental tissue, gestation day 18, and used to prepare a cDNA library with the mammalian expression vector pCDM8. This cDNA library, RPL18, was generously provided by Dr. P. Smith, University of Michigan.

Cell Culture—Chinese hamster ovary (CHO) cells were grown in Ham’s F-12 medium with 10% fetal bovine serum + 2 μM 1-phenyl-2-hexadecanoylamino-3-pyrrolidino-1-propanol, HCl (DL-threo-PPPP, Calbiochem, CA), an inhibitor of glucosylceramide synthase (3).

SMLDN1.1 mAb Binding Specificity—The generation and characterization of this SMLDN1.1 mAb has been described previously (4). Subsequent analysis of SMLDN1.1 binding characteristics with additional carbohydrate structures indicates that the presence of a single terminal phy; FACS, fluorescence-activated cell sorting; RT-PCR, reverse transcriptase-polymerase chain reaction; PBS, phosphate-buffered saline; BSA, bovine serum albumin; FITC, fluorescein isothiocyanate; MES, 2-(N-morpholino)ethane sulfonic acid; HPLC, high performance liquid chromatography; Neu5Ac, sialic acid.

GalNAc residue in either an α or β linkage is sufficient for recognition. We have observed that SMLDN1.1 mAb efficiently recognizes the terminal GalNAc residues in both Gb₄ (GalNAcβ1,3) and Gb₅ (GalNAca1,3GalNAcβ1,3).

**Expression Cloning of a Rat iGb₃ Synthase cDNA—RPL18 plasmid cDNA was isolated from an overnight culture of transformed *Escherichia coli* MC1061/P3 cells selected in 50 μg/ml ampicillin and 10 μg/ml tetracycline. Ten 100-mm plates of CHO cells (2.5 × 10⁶ cells per plate) were cotransfected with 10 μg of RPL18 plasmid cDNA and 3 μg of pSVE1-PyE plasmid, harboring the polyoma large T cDNA (a kind gift from Dr. Minoru Fukuda, The Burnham Institute), using Lipofectamine (Life Technologies, Inc.). The pCDM8 vector requires the polyoma large T antigen for high levels of replication, but unlike the library vector the pSVE1-PyE plasmid is neither amplified nor selected in subsequent enrichment steps, because it lacks the *supF* gene (5). Cells were harvested using 0.02% EDTA 60 h post-transfection, stained with SMLDN1.1 mAb, and processed for flow cytometry (FACS). Plasmid cDNA was recovered, from the FACS-sorted cells, by Hirt extraction (6) and amplified in *E. coli* MC1061/P3. Two further rounds of transfection and FACS enrichment were performed. Plasmid cDNA obtained from the third FACS sort was transformed into *E. coli* MC1061/P3, and dilutions were plated. Plaque lifts from these dilution plates were used to prepare more plasmid cDNA for transfection into CHO cells and screened with the SMLDN1.1 mAb via FACS. A positive plate with 590 colonies was divided into 12 smaller pools of 50 colonies, which were subsequently screened for SMLDN1.1 binding. This process was repeated on the reactive smaller pool until a single plasmid was isolated whose expression in CHO cells conferred reactivity with SMLDN1.1 mAb. We refer to this cDNA clone as iGb₃ synthase.

**DNA Sequencing—Plasmid cDNA was isolated from clones of interest using Qiagen columns. The cDNA insert was sequenced using T7, a pCDM8 reverse primer and gene-specific primers, in a reaction mix containing ABI Big-Dye terminators with AmpliTaq DNA polymerase according to the manufacturer’s instructions (PE Applied Biosystems). Reaction products were run on an ABI 310 sequencer. Complete insert sequence was obtained in both directions and compared against sequences held in GenBank™ using BLAST (NCBI). Additional analysis was performed using Geneworks (version 2.5, Oxford Molecular, UK) and the transmembrane prediction program TMHMM (7).**

**Site-directed Mutagenesis—The ¹⁹⁸DVD²⁰⁰ sequence in the iGb₃ cDNA clone was mutated to ¹⁹⁸AVA²⁰⁰ by inverse PCR using high fidelity KlenTaq LA-polymerase mix (Sigma) and completely overlapping primers containing the point mutations (underlined), in the forward direction: 5′-CTATGTGTTCTGCCTGGCCGTGGCCCAGTACTT-CAGCGG-3′ and reverse direction: 5′-CCGCTGAAGTACTGGGCCAC-GGCCAGGCAGAACACATAG-3′. Following thermocycling and digestion of parental DNA template with DpnI, MC1061/P3 cells were transformed and mini-preps sequenced to verify the desired mutations.**

**FACS Analysis—Cells were harvested using 0.02% EDTA. After washing in phosphate-buffered saline (PBS), pH 7.2, cells were counted and resuspended at 10⁷ cells/ml in ice-cold PBS/1% BSA. Typically 10⁶ cells were incubated with 100 μl of mouse IgM mAb, SMLDN1.1, diluted to 5 μg/ml in blocking buffer (PBS/1% BSA/5% normal goat serum), for 45 min on ice. Cells were washed in PBS/1% BSA and incubated for 30 min on ice with goat anti-mouse IgM-FITC (Jackson ImmunoResearch Laboratories, PA) diluted 1:200 in blocking buffer. After washing, cells were resuspended in PBS and read directly on the FACS Calibur (Becton Dickinson) and analyzed using CellQuest version 3.1 software. Other mAbs, lectins, and secondary antibodies used in FACS analysis included rat IgM anti-Forssman mAb M1/22.21, used at 1:100; anti-SSEA-3 mAb MC631 (both reagents kindly provided by Dr. D. Haslam, Washington University, St. Louis, MO); anti-CD77 mAb, clone 38-13 diluted 1:5 (Biodesign International); *Griffonia simplicifolia* lectin I-B₄-biotin, used at 10 μg/ml with 0.1 mM CaCl₂ (Vector Laboratories); and streptavidin-FITC (PharMingen).**

**Assay of α1,3GalT Activity—Cell extracts from transfected CHO cells were prepared and assayed as described previously except 100 mM sodium cacodylate, pH 6.8 was used in place of MES buffer (8). Supernatants from the cultured cells were concentrated 10-fold, using Microcon-10 (Millipore) mini-spin columns, prior to assaying a 50-μl fraction. The reaction mixture contained the following in a final volume of 100 μl: 100 mM sodium cacodylate-HCl, pH 6.8, 0.5% Triton X-100, 5 mM ATP, protease inhibitors (50 μg/ml turkey egg white trypsin inhibitor, 20 μg/ml leupeptin, 20 μg/ml antipain, 20 μg/ml pepstatin, 20 μg/ml chymotrypsin, and 0.115 trypsin inhibitor units/ml aprotinin, Sigma), 250 μM UDP-galactose, UDP-[³H]galactose (800 cpm/pmol) (American Radiolabeled Chemicals, Inc.), 15 mM MnCl₂, 1 mM LacCer (Sigma). Reactions were started with the addition of 15- or 50-μg membrane fractions or 50 μl of concentrated culture supernatant and incubated for 90 min or overnight at 37 °C. The enzyme assay was linear over these times. Reaction products were isolated by reverse-phase chromatography using Sep-Pak C₁₈ cartridges (Waters, MA) and analyzed by thin layer chromatography (TLC) using silica gel aluminum-backed plates (Whatman) developed in chloroform:methanol:water (65:25:4, v/v). After drying, the plates were sprayed with En³Hance (NEN Life Science Products) then exposed to Kodak BioMax MR x-ray film at –80 °C. Neutral glycolipid standards from porcine blood (Calbiochem) were included, and this portion of the plate was cut-off and sprayed with orcinol.**

**Metabolic Labeling of CHO Cells—The day after transfection, cells were labeled with either 35 μCi/ml [³H]Gal or [³H]GlcNAc (NEN Life Science Products) for a further 24–48 h. Cells were scraped off the culture plate, and glycolipids were extracted in chloroform/methanol as described previously (8).**

**Glycolipid Digestion with Exoglycosidases—Glycolipid extracts (approximately 1–10 μg or 30,000 cpm) were isolated over a Sep-Pak C₁₈ cartridge and separated by HPLC using an Iatrobead column (kindly provided by Dr. R. Schnaar, John Hopkins School of Medicine) as described (9). Pooled peak fractions were digested overnight at 37 °C with exoglycosidases in the presence of 0.05% sodium taurodeoxycholate in a final volume of 10 μl. The glycosidases used included α-galactosidase (green coffee bean, Calbiochem); α1,3-galactosidase (recombinant *Escherichia coli*, Calbiochem); β1,3,4,6-galactosidase (bovine testes, Calbiochem); α-N-acetylgalactosaminidase (chicken liver, Oxford GlycoSciences, UK); and β-N-acetylhexosaminidase (*Streptomyces plicatus* recombinant in *E. coli*, New England BioLabs). The digested samples were re-isolated using a Sep-Pak C₁₈ cartridge prior to separation on TLC as described above.**

**Expression of iGb₃ Synthase in Rat Tissues Using RT-PCR—Tissues were taken from a 6-month-old female Long Evans rat and snap-frozen in liquid nitrogen. These tissues were homogenized in Trizol (Life Technologies), and total RNA was purified according to the manufacturer’s instructions. The quality and quantity of RNA was verified by gel electrophoresis. First strand cDNA synthesis was prepared from 1–5 μg of RNA using the reverse transcriptase enzyme, Superscript II, and oligo(dT) primer (Life Technologies). Gene-specific primer pairs were used in the PCR to yield an expected product size of 780 base pairs. Glyceraldehyde-3-phosphate dehydrogenase gene-specific primers were used as a control. Products were analyzed by gel electrophoresis.**

---

**RESULTS**

**Isolation of a Rat iGb₃ Synthase cDNA Clone—Phenotypic cloning has been used to isolate many glycosyltransferase genes, which are typically expressed at low levels in cells (10). The screening strategy used in this paper relies on the specificity of the SMLDN mAb, which recognizes terminal GalNAc residues. Positive clones identified in this cloning approach are involved in the cell surface expression of a GalNAc epitope. Despite expecting the isolated clone to express GalNAc transferase activity, no such activity was detected. These results were initially confusing until it became apparent that our read-out signal, GalNAc expression, was in fact the product of a key upstream biosynthetic step. Thus, the isolated clone synthesizes iGb₃, a product that Gb₄ and Gb₅ synthases, enzymes endogenous to the CHO host recipient cells, convert to iGb₄ and iGb₅, respectively. Using this expression cloning procedure, the iGb₃ synthase cDNA (accession number AF248543) and later Gb₃ synthase cDNA (accession number AF248544, see Keusch et al., companion paper (35)) were isolated from the rat placental library, RPL18.**

**cDNA Sequence Analysis—Sequencing of the iGb₃ synthase cDNA clone revealed an insert of 1496 base pairs. Translation from the first methionine in the open reading frame predicts a protein of 339 amino acids (Fig. 1). The TMHMM transmembrane prediction program (7) identifies a single transmembrane domain of 19 amino acids with a type II transmembrane topology typical of Golgi glycosyltransferases (11). Other hydrophobic regions are also apparent, including two stretches near the N terminus (residues 15–29 and 34–58) but are not predicted to be membrane spanning by the TMHMM program. Two N-linked glycosylation consensus sequences are present in**

25310

Expression Cloning of iGb₃ Synthase

CTAAAGCAAGTCATTTCTTCTATCTCTCTGGGCTGGAAAAAACTGCATGGCAGGTTTTCCACCAC TGACTTCTCCCC

1 M A L E G L R A K K R L L W R L F L S A F G L L L G L Y H Y W F K I F R L F E V F 40
1 ATGGCTCTGGAGGGACTCAGGGCCAAGAAGAGACTCCTGTGGGGACTCTT CCTGTCTGGCTTTGGTCTCCTAGGCCCTGTACCATTA CTGGTTCAAAGATTTTCAGGCTCTTTGAAGTGTTTC 120
41 I P M G I C P M A I M P L L K D N F T G V L R H W A R P E V L T C T S W G A P I 80
121 ATCCCCATGGGCATCTGCCCTATGGCCATCATGGCCTCTGCTAAAGGACAATTTCACCGGAGT GCTGGCTCACTGGGCCC GGCTGAAGTCTGTGACCTCTTTGGGAGCCCCAAATT 240
81 I W D E T F D P H V A E R A R R Q N L T I G L T V F A V G R Y L E K Y L E H F 120
241 ATCTGGGATGAGACCTTCGACCCACACGTAGCTGAGCGAGAGGCGAGACGGCAGAACCTTACC ATTTGGACTGACTGTCTTTGCGG TAGGCCAGGTACCTGGGAGAAGTACCTGGAGCAC TTC 360
121 L V S A E Q Y F M V G Q N V V Y Y V F T D R P E A V P H V A L G Q G R L L R V K 160
361 CTGGTATCGGCAGAGCAGTACTTCATGGTCGGCCAGAACGTGGGTATTATGTGTTTACC GACCGACCCGAAGCAGTGCCCCACGTGGGCTCTAGGCCAGGTCG CCTGCGGCGGTGAAA 480
161 P V R R E K R W Q D V S M A R M L T L H E A L G G Q L G R E A D Y V F C L D V D 200
481 CCAGTGAGAAGAGAGAAGCGCTGGCAGGACGTGTCATGGCTCGCATGCTCACGAGGCTCTGGG AGGGCAGCTGGGCCGAGAAGCTGACTATGTGTTCTG GCCCTGGACGTGGAC 600
201 Q Y F S G N F G P E V L A D L V A Q L H A W H F R W P R W M L P Y E R D K R S A 240
601 CAGTACTTCAGCGGTAACTTCGGACCCGAGGTGCTGGCACAGCTGGACGCCTGGCACTTCCGG CTCTGGGTGGATGCTGCCCTACGAGAGGGACAAGCGGTCGGCT 720
241 A A L S L S E G D F Y Y H A A V F G G S V A A L L K L T A H C A T G Q Q L D R R E 280
721 GCCGCGCTGTGCTTTAAGCGAAGGCGATTCTACTACCACGCGCGGGTGTTCGGGGGCAGTGTG GGCCTCTGCTCAAGCTGACGGCTCACTGTGCGACTGGCCAAACAGCTGGACC GTGAG 840
281 H G I E A R W H D E S H L N K F F W L S K P T K L L S P E F C W A E E I G W R P 320
841 CATGGCATTAGGACACGCTGGCACGAGAGGCCACTCAACAAGTTCTTCTGGCTGAGCAAGCCC ACCAAGCTGCTGTCGCCAGAGTTTTGCTGGGCA GAGGAAATTGGCTGGAGGCAA 960
321 E I H H P R L I W A P K E Y A L V R T * 339
961 GAGATCCACCACCCCTCGTCTGATCTGGGCACCCAAGGAGTACGCAC TGGTGCAGAACCTAGCAGGCCACAGGAACGTCTTTGGCACTGCGAGT TTTGAATGGA 1080
1081 TTGCAAATATGTTTCAAAAGCACGAAGTCA GAGCCAGTTTACCTTGGCCTTIGGAAGGGGA ACTCGGAAC TCGAGAGGTACTGTGCC TGGGGCCGTACACCCCTTGTCA GCTGGCCAGAGAGC 1200
1201 CTTTTTGCTCCAACCAGATGCAAATTATGGAAAGGCAAGAGTTTGTCA GCTTACTTTGGCTCA AAGGCTTCTTGGTAATGGTTGTGCTCAGCTTCTGTCCGGACCTTAG 1320
1321 GTTTGACAGCTGTCTGGAGCCATTCCCTGTGACCAAGGACCTCGCAG ATTCTCTTTCTCTT GCTTTCTACTTCAGCTTTAGACTTTAG 1420

FIG. 1. cDNA sequence analysis of rat iGb₃ synthase. The deduced amino acid sequence in the single-letter code is shown above the nucleotide sequence. The putative transmembrane domain is underlined, and two potential N-linked glycosylation sites are boxed. The ¹⁹⁸DXD²⁰⁰ motif is indicated by three black dots.

iGb3 S 1 M A L E G [L R A K K R L L W R L F L S A F G L L L G L Y H Y W F K I F R L F E V F 23
a1-3 GalT 1 M I T M L Q D L H V N K I S M S R S K S E T S L P S S R S G S Q E K I M N V K G K V I L 50
FS 1 M R C R R L A L G L G F S L L S G - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Expression Cloning of $iGb_3$ Synthase

Anti-GalNAc | Anti- | Anti-$Gb_5$ | Anti-$Gb_3$
--- | --- | --- | ---
| Gal$\alpha 1,3$Gal |

pCDM8

$iGb_3$

synthase

cDNA

$\alpha 1,3$GalT

cDNA

AVA $iGb_3$

synthase

cDNA

FIG. 3. Cell surface expression of isoglobo-GSL in CHO cells transfected with $iGb_3$ synthase cDNA is inhibited by PPPP. CHO cells transfected with either empty pCDM8 vector, $iGb_3$ synthase cDNA, murine $\alpha 1,3$GalT cDNA, or mutant ${}^{198}$AVA${}^{200}$ $iGb_3$ synthase cDNA were cultured in the absence (thick line) or presence of PPPP (thin line) and stained for surface expression of terminal carbohydrates on GSL: GalNAc (SMLDN1.1 mAb); Gal$\alpha 1,3$Gal ($G$. simplicifolia lectin I-B$_4$-biotin); $Gb_5$ (GalNAc$\alpha 1,3$GalNAc$\beta 1,3$Gal$\alpha 1,4$Gal$\beta 1,4$Glc$\beta 1$Cer) using anti-$Gb_5$ (Forssman) glycolipid M1/22.21 mAb; or $Gb_3$ (Gal$\alpha 1,4$Gal$\beta 1,4$Glc$\beta 1$Cer) using anti-CD77 mAb clone 38-13 prior to analysis by FACS. Control primary antibodies are shown using a dotted line. Secondary reagents included anti-IgM-FITC and streptavidin-FITC.

an $\alpha 1,3$-linkage to a $\beta$-linked Gal/GalNAc. $iGb_3$ synthase represents the third distinct $\alpha 1,3$-galactosyltransferase in the ABO gene family.

**Expression of $iGb_3$ Synthase in CHO Cells Leads to the Synthesis of the Isoglobo-series GSL—**FACS analysis shows that CHO cells transfected with $iGb_3$ synthase cDNA express high levels of terminal GalNAc on their cell surface. In contrast, CHO cells transfected with the pCDM8 vector did not express terminal GalNAc. The highest reactivity in the $iGb_3$ synthase cDNA transfectants is seen using the SMLDN1.1 mAb, with approximately 70% of the transfected cells expressing terminal GalNAc (Fig. 3). This positive staining is predominantly due to glycolipids, because reactivity to the SMLDN1.1 mAb is lost when transfectants are cultured in the presence of PPPP, an inhibitor of the major series of GSL (3). Apart from $G_{M3}$, parent CHO cells do not synthesize any GSL beyond LacCer (13). Therefore, we were surprised to see that transfection of $iGb_3$ synthase cDNA into CHO cells results in strong reactivity with the anti-Forssman ($Gb_5$) glycolipid mAb (Fig. 3). Forssman glycolipid, $Gb_5$ (GalNAc$\alpha 1,3$GalNAc$\beta 1,3$Gal$\alpha 1,4$Gal$\beta 1,4$Glc$\beta 1$Cer), and $iGb_5$ (GalNAc$\alpha 1,3$GalNAc$\beta 1,3$Gal$\alpha 1,3$Gal$\beta 1,4$Glc$\beta 1$Cer), represent the terminal structures in the globo- and isoglobo-series GSL, respectively (14, 15). Transfection of $Gb_5$ synthase cDNA into parent CHO cells does not result in $Gb_5$ or $iGb_5$ expression due to the absence of precursor substrates (35). Therefore, we reasoned that the $iGb_3$ synthase cDNA is able to initiate the synthesis of isoglobo-GSL in CHO cells. $Gb_4$ synthase and $Gb_5$ synthase but not $Gb_3$ synthase nor $iGb_3$ synthase are endogenous to CHO cells. Thus, the *de novo* production of $iGb_3$ in CHO cells, transfected with the $iGb_3$ synthase cDNA, provides substrate for the sequential addition of $\beta 1,3$- and $\alpha 1,3$GalNAc moieties by $Gb_4$ and $Gb_5$ synthases, respectively. Assuming similar expression levels, a catalytically inactive form of $iGb_3$ synthase produced by mutating the ${}^{198}$DVD${}^{200}$ sequence to ${}^{198}$AVA${}^{200}$ does not result in synthesis of any isoglobo-GSL (Fig. 3). Transfection of CHO cells with $iGb_3$

25312

**Expression Cloning of iGb₃ Synthase**

synthase cDNA results in the synthesis of isoglobo-GSL but not globo-GSL (Gb₃ core structures) as indicated by the lack of reactivity with anti-CD77/Gb₃ (Gala1,4Galβ1,4Glcβ1Cer) (Fig. 3) or anti-SSEA-3 (Galβ1,3GalNAcβ1,3Galα1,4Galβ1,4Glcβ1Cer) mAbs (data not shown).

Extensive homology seen between iGb₃ synthase and a previously cloned murine α1,3GalT prompted us to compare their ability to synthesize isoglobo-GSL. CHO cells transfected with either α1,3GalT cDNA or iGb₃ synthase cDNA express Gala1,3Gal epitopes on their cell surface that are reactive with the lectin G. simplicifolia lectin I-B₄ (Fig. 3). However, only 5% of CHO cells transfected with murine α1,3-galactosyltransferase cDNA express GSL with terminal GalNAc as determined with the SMLDN1.1 mAb + PPPP, whereas 70% of CHO cells transfected with iGb₃ synthase cDNA are reactive with the mAb (Fig. 3). This suggests that CHO cells transfected with α1,3GalT cDNA preferentially synthesize Gala1,3Gal on glycoprotein substrates, most likely using the preferred acceptor Galβ1,4GlcNAcβ1-R (16). In contrast, CHO cells transfected with iGb₃ synthase cDNA synthesize Gala1,3Gal on GSL, primarily LacCer (Galβ1,4Glcβ1Cer) (see Fig. 5 below), producing iGb₃ that is subsequently converted to iGb₄ and iGb₅ (reactive to SMLDN1.1 mAb).

**Isolation of iGb₅ from CHO Cells Transfected with iGb₃ cDNA—** CHO cells were transfected with iGb₃ synthase cDNA or Gb₃ synthase cDNA and metabolically labeled with [³H]GlcNAc. The Gb₃ synthase transfectants have been previously characterized as synthesizing the globo-series GSL, including Gb₄ and Gb₅ GSL (35) (Fig. 4, lane 2). Cells can incorporate [³H]GlcNAc into GlcNAc-, GalNAc-, and sialic acid-containing glycans. Glycolipids were extracted, isolated over a Sep-Pak C₁₈ column, and separated by TLC. The resulting autoradiographs show a small amount of material migrating as a doublet at the position of Gb₄ and a large amount of material migrating as a doublet at the position of Gb₅ produced by CHO cells transfected with iGb₃ synthase cDNA (Fig. 4, lane 1). This labeled GSL extract was purified on a Sep-Pak C₁₈ cartridge and separated by HPLC using an Iatrobead column. Fractions containing each of the four major peaks were pooled and reanalyzed by TLC. The two fractions containing material that migrates with Gb₅ (Fig. 4, lanes 3 and 6) were treated with exoglycosidases. Both bands migrating at the position of Gb₅ were resistant to digestion with β-N-acetylhexosaminidase (Fig. 4, lanes 5 and 8) but sensitive to α-N-acetylgalactosaminidase (Fig. 4, lanes 4 and 7). Following digestion with α-N-acetylgalactosaminidase, the two bands that had migrated at the position of Gb₅ now migrate at the position of Gb₄. Note that the loss in the intensity of the bands following digestion reflects the loss of one of the two [³H]GalNAc residues.

Glycolipids often migrate as doublets by TLC, reflecting a difference in the lipid tail rather than the carbohydrate structure (17). Digestion profiles similar to those described above were done using unlabeled glycolipids extracted from CHO cells transfected with iGb₃ synthase cDNA. Following separation on TLC, the undigested glycolipids migrating at Gb₅ and the α-N-acetylgalactosaminidase-digested glycolipids migrating at Gb₄ both stained with SMLDN1.1 mAb, indicating the presence of terminal GalNAc (not shown). These results show that the major isoglobo-GSL isolated from CHO cells transfected with iGb₃ synthase cDNA is iGb₅. Note that, like parent CHO cells, CHO cells transfected with either murine α1,3GalT cDNA (16) or the mutant ¹⁹⁹AVA²⁰¹ iGb₃ synthase cDNA only synthesized GlcCer, LacCer, and Gₘ₃ in significant amounts, as determined by metabolically labeling with [³H]Gal or [³H]GlcNAc (not shown). A number of additional GSL products from CHO cells transfected with iGb₃ synthase are observed

FIG. 4. Detection of isoglobo-GSL from CHO cells transfected with iGb₃ synthase cDNA. TLC separation of glycolipid extracts from metabolically [³H]GlcNAc-labeled CHO cells transfected with iGb₃ synthase cDNA (lanes 1, 3–8) or Gb₃ synthase cDNA (lane 2). Material migrating with standard Gb₄ and Gb₅ is indicated by a the brackets. The labeled GSL from iGb₃ synthase transfectants was separated by HPLC using an Iatrobead column (9), and pooled fractions of two peaks comigrating with Gb₅ were re-run on TLC before (lanes 3 and 6) and after digestion with α-N-acetylgalactosaminidase (lanes 4 and 7) or β-N-acetylhexosaminidase (lanes 5 and 8). The asterisk indicates Gₘ₃, which is also produced in parent CHO cells.

![Figure 4](https://i.imgur.com/yourimage.png)

FIG. 5. TLC of [³H]Gal GSL products from the in vitro galactosyltransferase assay. Cell extracts from parent CHO cells, or cells transfected with either murine α1,3GalT cDNA or iGb₃ synthase cDNA, are used as enzyme source. Lanes 13 and 14 show products using supernatant from iGb₃ synthase transfectants as the enzyme source. Products obtained using the exogenous acceptor substrates indicated along the bottom of the TLC were isolated following the enzyme reaction by Sep-Pak C₁₈ prior to separation by TLC in a chloroform:methanol:water solvent (65:25:4, v/v). Arrows indicate material comigrating with *a*, LacCer; *b*, Gb₃; and *c*, Gb₄ standards.

when cells are metabolically labeled with [³H]Gal, but the identity of these products has not yet been established (data not shown).

**In Vitro Activity of iGb₃ Synthase—** Culture supernatant or detergent extracts from CHO cells transfected with iGb₃ synthase cDNA were assayed for activity against several GSL acceptor substrates. An increase of approximately 100- to 200-fold in galactosyltransferase activity, using LacCer as acceptor, is seen when comparing extracts from CHO cells expressing iGb₃ synthase to those expressing murine α1,3-galactosyltransferase or mock-transfected with vector, respectively. A faint radioactive product migrating with Gb₃ standard is seen in extracts assayed from the CHO cells expressing murine α1,3GalT (Fig. 5, lane 3), but it is minor when compared with the product from CHO cells expressing iGb₃ synthase (Fig. 5, lane 11). Parent CHO cells express sufficient levels of endogenous LacCer to allow the detection of iGb₃ synthase activity in vitro. This accounts for the appearance of material migrating at the position of Gb₃ in all cell extracts expressing iGb₃ synthase, even in the absence of exogenous LacCer (Fig. 5, lane 9). Neither GlcCer nor Gb₅ (not shown) are able to act as acceptor substrates. Significant, but lower activity was observed in the iGb₃ synthase transfectants when using GalCer and Gb₃ (Fig. 5, lanes 10 and 12), which contain terminal β- and α-linked Gal, respectively. The predicted structure from the Gb₃ reaction is Gala1,3Galα1,4Galβ1,4Glcβ1Cer, a GSL found in the rat small intestine (18). This is also the underlying structure of

Expression Cloning of $iGb_{3}$ Synthase

III3Gala$Gb_{3}$Cer (Gala1,3Gala1,4Galβ1,4Glcβ1Cer), which is present in the cryptic cells and circular muscle of the rat small intestine (19). The proposed structure for the product from the Galβ1Cer reaction, Gala1,3Galβ1Cer, represents a novel GSL, to the best of our knowledge. CHO cells expressing $iGb_{3}$ synthase secrete an active form of the enzyme into the culture supernatant that acts on LacCer but not on GalCer (Fig. 5, lanes 13 and 14). Radioactive material near the origin of the TLC plates is seen in all cell extracts and is nonspecific.

CHO Cells Expressing $iGb_{3}$ Synthase Produce $iGb_{3}$ in Vitro—Cell extracts from CHO cells expressing $iGb_{3}$ synthase show high levels of galactosyltransferase activity toward LacCer, yielding a product that migrates at the position of $Gb_{3}$ standard. Digestion of this isolated [³H]Gal *in vitro* product with specific glycosidases revealed the terminal galactose to be in an α1,3-linkage (Fig. 6, lane 2). Hence, CHO cells expressing the $iGb_{3}$ synthase cDNA produce $iGb_{3}$ (Gala1,3Galβ1,4Glcβ1Cer). This confirms the FACS analysis, where CHO cells transfected with $iGb_{3}$ synthase are reactive to *G. simplicifolia* lectin I-B₄ (a lectin that recognizes α1,3-galactosyl residues) but not to anti-CD77 mAb (a monoclonal antibody that binds to $Gb_{3}$, Gala1,4GalβGlcβ1Cer) (Fig. 3). Similarly, total GSL extracts from CHO cells expressing $iGb_{3}$ synthase do not react with the B-subunit of Shiga toxin (35), which specifically binds $Gb_{3}$ (20). The globoside and Forssman structures produced in CHO transfected with $iGb_{3}$ synthase cDNA must, therefore, be synthesized on the $iGb_{3}$ glycolipid, Gala1,3Galβ1,4Glcβ1Cer rather than the more common $Gb_{3}$, Gala1,4Galβ1,4Glcβ1Cer (see Fig. 8).

**Expression of $iGb_{3}$ Synthase in Rat Tissues—RT-PCR using $iGb_{3}$ synthase-specific primers shows a widespread tissue expression of the $iGb_{3}$ synthase enzyme (Fig. 7). Highest levels of expression are seen in the spleen, thymus, and skeletal muscle. The lung, uterus, pituitary, and heart have intermediate $iGb_{3}$ synthase expression, whereas other tissues indicate low or undetectable amounts. Similar amounts of control glyceraldehyde-3-phosphate dehydrogenase gene expression is seen in all tissues (35).**

---

### DISCUSSION

We have described the cloning of a new member of the ABO gene family, $iGb_{3}$ synthase, an α1,3-galactosyltransferase that initiates the synthesis of isoglobo-GSL. LacCer (Galβ1, 4Glcβ1Cer) is the preferred acceptor substrate for $iGb_{3}$ synthase, however, alternative substrates include Galβ1Cer and, surprisingly, $Gb_{3}$ (Gala1,4Galβ1,4Glcβ1Cer). This suggests that $iGb_{3}$ synthase is able to utilize multiple substrates containing a terminal Gal residue, irrespective of its anomeric configuration (Fig. 8). Hence, $iGb_{3}$ synthase may be regarded as an exception to the one-linkage–one-enzyme rule. The ability to add α1,3-linked Gal to these other structures may result in the appearance of novel structures in tissues depending on the available substrates. Even though $iGb_{3}$ synthase acts principally on GSL, treatment of cells expressing $iGb_{3}$ synthase with PPPP did not completely abolish the expression of Gala1,3Gal seen using the lectin, *G. simplicifolia* lectin I-B₄ (data not shown). It remains to be determined if this PPPP-resistant expression of the Gala1,3Gal epitope is occurring on GalCer or glycoprotein acceptor substrates.

Previously cloned α1,3-glycosyltransferases (α1,3GalT and Forssman or i/$Gb_{5}$ synthase) are members of the ABO gene family. They have been described in numerous mammals but are nonfunctional in Old World primates, including humans (21, 22). These enzymes differ in their substrate specificity and may therefore have different functional significance. GSL containing the $iGb_{3}$ structure have been described in fish, rat, dog, pig, and horse (23–27). At present it is not known if homologues

---

### FIG. 6. Exoglycosidase digestion of the $iGb_{3}$ synthase *in vitro* product. The isolated [³H]Gal product from an *in vitro* enzyme assay using LacCer as acceptor substrate and cell extract from CHO cells transfected with $iGb_{3}$ synthase cDNA was treated with exoglycosidases. After digestion the products were purified on Sep-Pak C₁₈ and analyzed by autoradiography following TLC. Digests are: lane 1, mock-digested; lane 2, α1,3-galactosidase; and lane 3, β1,3,4,6-galactosidase.

---

### FIG. 7. Expression of $iGb_{3}$ synthase RNA in rat tissues. Total RNA was isolated from rat tissues and assayed for $iGb_{3}$ synthase RNA expression using RT-PCR and gene-specific primers. Gene-specific primers for glyceraldehyde-3-phosphate dehydrogenase RNA, a housekeeping gene, produced similar levels of product in each tissue (35). The expected $iGb_{3}$ synthase PCR product size of 780 base pairs is indicated by the arrow. pop. LN, popliteal lymph node; mes. LN, mesenteric lymph node; and skel. muscle, skeletal muscle.

---

### FIG. 8. The major mammalian GSL pathways. Major GSL series are noted in boxes. Structures with Gala1,3Gal represent the product catalyzed by $iGb_{3}$ synthase (*thick arrows*). Isoglobo-series are synthesized in CHO cells following transfection with $iGb_{3}$ synthase cDNA. *Black arrows* indicate reactions in the biosynthetic pathways where the pertinent glycosyltransferases are cloned. Digalacto GSL represents a novel glycolipid structure.

of the rat iGb₃ synthase exist in other species. The phenotypic cloning procedure used to clone iGb₃ synthase has revealed that iGb₃ may be efficiently capped with GalNAc residues to form iGb₄ and iGb₅. The presence of structures arising from iGb₃ may not be apparent in other species, because iGb₄ and iGb₅ react with many of the same immunological reagents as Gb₄ and Gb₅, respectively. The fixative procedures used in the detection of Galα1,3Gal epitopes on glycoproteins may eliminate or reduce the levels of iGb₃.

Humans do not normally express the Galα1,3Gal epitope and as a result have high circulatory levels of the natural anti-Galα1,3Gal antibody (28). These antibodies mediate acute rejection of xenotransplants from species that express the Galα1,3Gal epitope. The anomalous expression of Galα1,3Gal epitope in humans has been described in human cancer cells and in the thyroid from patients with the autoimmune disorder, Graves' disease (29, 30). Furthermore, small amounts of Galα1,3Gal have been reported on red cells (31). Low levels of Galα1,3Gal expression in humans may reflect the activity of an alternative α1,3-galactosyltransferase, possibly iGb₃ synthase. The accumulation of GSL containing iGb₃ structures has been noted in rat hepatomas, mammary adenocarcinomas, and tumors of colorectal origin (32–34). Furthermore, the presence of multiple α1,3-galactosyltransferases may have an important clinical consequence in animals currently being assessed for xenotransplantation. Should they express the Galα1,3Gal epitope on both glycoproteins and glycolipids, it may be essential to eliminate these epitopes from both proteins and lipids to prevent xenotransplant rejection.

In summary, we have cloned iGb₃ synthase, an α1,3-galactosyltransferase, that initiates the synthesis of the isogloboseries GSL. This enzyme also has the ability to act on two further substrates found in different GSL series, the globoseries and the galactosylceramides. With the recent cloning of the Gb₃ synthase (35), gene ablation studies will be possible to determine the functional significance of different GSL series.

Acknowledgments—We thank Dr. D. Haslam for his helpful suggestions and generous gifts of the anti-Forssman (Gb₅) and anti-SSEA-3 MC631 (GalGb₄) mAb. We are also indebted to Dr. P. Smith for providing the RPL18 cDNA library and to Dr. J. Lowe for supplying the α1,3GalT cDNA.

REFERENCES

1. Wiegandt, H. (ed) (1985) *Glycolipids*, pp. 1–99, Elsevier Science Publishers, Amsterdam, The Netherlands
2. Yamashita, T., Wada, R., Sasaki, T., Deng, C., Bierfreund, U., Sandhoff, K., and Proia, R. L. (1999) *Proc. Natl. Acad. Sci. U. S. A.* **96**, 9142–9147
3. Abe, A., Radin, N. S., Shayman, J. A., Wotring, L. L., Zipkin, R. E., Sivakumar, R., Ruggieri, J. M., Carson, K. G., and Ganem, B. (1995) *J. Lipid Res.* **36**, 611–621
4. Nyame, A. K., Leppanen, A. M., DeBose-Boyd, R., and Cummings, R. D. (1999) *Glycobiology* **9**, 1029–1035
5. Bierhuizen, M. F., and Fukuda, M. (1992) *Proc. Natl. Acad. Sci. U. S. A.* **89**, 9326–9330
6. Hirt, B. (1967) *J. Mol. Biol.* **26**, 365–369
7. Glasgow, J., Littlejohn, T., Major, F., Lathrop, R., Sankoff, D., and Sensen, C. (eds) (1998) in *Proceedings of the Sixth International Conference on Intelligent Systems for Molecular Biology*, pp. 175–182, AAAI Press, Menlo Park, CA
8. Haslam, D. B., and Baenziger, J. U. (1996) *Proc. Natl. Acad. Sci. U. S. A.* **93**, 10697–10702
9. Kannagi, R., Watanabe, K., and Hakomori, S. (1987) *Methods Enzymol.* **138**, 3–12
10. Fukuda, M., Bierhuizen, M. F., and Nakayama, J. (1996) *Glycobiology* **6**, 683–689
11. Paulson, J. C., and Colley, K. J. (1989) *J. Biol. Chem.* **264**, 17615–17618
12. Wiggins, C. A., and Munro, S. (1998) *Proc. Natl. Acad. Sci. U. S. A.* **95**, 7945–7950
13. Rosales Fritz, V. M., Daniotti, J. L., and Maccioni, H. J. (1997) *Biochim. Biophys. Acta* **1354**, 153–158
14. Siddiqui, B., and Hakomori, S. (1971) *J. Biol. Chem.* **246**, 5766–5769
15. Falk, P., Holgersson, J., Jovall, P. A., Karlsson, K. A., Stromberg, N., Thurin, J., Brodin, T., and Sjogren, H. O. (1986) *Biochim. Biophys. Acta* **878**, 296–299
16. Smith, D. F., Larsen, R. D., Mattox, S., Lowe, J. B., and Cummings, R. D. (1990) *J. Biol. Chem.* **265**, 6225–6234
17. Schnaar, R. L. (1994) *Methods Enzymol.* **230**, 371–389
18. Ångström, J., Breimer, M. E., Falk, K. E., Hansson, G. C., Karlsson, K. A., Leffler, H., and Pascher, I. (1982) *Arch. Biochem. Biophys.* **213**, 708–725
19. Kotani, M., Kawashima, I., Ozawa, H., Ugura, K., Ariga, T., and Tai, T. (1994) *Arch. Biochem. Biophys.* **310**, 89–96
20. Lindberg, A. A., Brown, J. E., Stromberg, N., Westling-Ryd, M., Schultz, J. E., and Karlsson, K. A. (1987) *J. Biol. Chem.* **262**, 1779–1785
21. Joziasse, D. H., Shaper, J. H., Jabs, E. W., and Shaper, N. L. (1991) *J. Biol. Chem.* **266**, 6991–6998
22. Xu, H., Storch, T., Yu, M., Elliott, S. P., and Haslam, D. B. (1999) *J. Biol. Chem.* **274**, 29390–29398
23. Ostrander, G. K., Levery, S. B., Eaton, H. L., Salyan, M. E., Hakomori, S., and Holmes, E. H. (1988) *J. Biol. Chem.* **263**, 18716–18725
24. Breimer, M. E., Hansson, G. C., Karlsson, K. A., and Leffler, H. (1982) *J. Biol. Chem.* **257**, 557–568
25. Sung, S. S., and Sweeley, C. C. (1979) *Biochim. Biophys. Acta* **575**, 295–298
26. Slomiany, B. L., Slomiany, A., and Horowitz, M. I. (1974) *Eur. J. Biochem.* **43**, 161–165
27. Yamamoto, H., Iida-Tanaka, N., Kasama, T., Ishizuka, I., Kushi, Y., and Handa, S. (1999) *J. Biochem. (Tokyo)* **125**, 923–930
28. Galili, U., Anaraki, F., Thall, A., Hill-Black, C., and Radic, M. (1993) *Blood* **82**, 2485–2493
29. LaTemple, D. C., Abrams, J. T., Zhang, S. Y., and Galili, U. (1999) *Cancer Res.* **59**, 3417–3423
30. Winand, R. J., Devigne, J. W., Meurisse, M., and Galili, U. (1994) *J. Immunol.* **53**, 1386–1395
31. Galili, U., Kobrin, E., Macher, B. A., and Shohet, S. B. (1989) *Prog. Clin. Biol. Res.* **319**, 225–241
32. Ariga, T., Kasai, N., Miyoshi, I., Yamawaki, M., Scarsdale, J. N., Yu, R. K., Kasama, T., and Taki, T. (1995) *Biochim. Biophys. Acta* **1254**, 257–266
33. Carlsen, S. A., Xie, Y., Whitfield, D. M., Pang, H., and Krepsky, J. J. (1993) *Cancer Res.* **53**, 2906–2911
34. Thurin, J., Brodin, T., Bechtel, B., Jovall, P. A., Karlsson, H., Stromberg, N., Teneberg, S., Sjogren, H. O., and Karlsson, K. A. (1989) *Biochim. Biophys. Acta* **1002**, 267–272
35. Keusch, J. J., Manzella, S. M., Nyame, K. A., Cummings, R. D., and Baenziger, J. U. (2000) *J. Biol. Chem.* **33**, 25315–25321

GLYCOBIOLOGY AND  
EXTRACELLULAR MATRICES:  
Expression Cloning of a New Member of  
the ABO Blood Group  
Glycosyltransferases, iGb₃ Synthase, That  
Directs the Synthesis of  
Isoglobo-glycosphingolipids  

Jeremy J. Keusch, Stephen M. Manzella,  
Kwame A. Nyame, Richard D. Cummings and  
Jacques U. Baenziger  
*J. Biol. Chem.* 2000, 275:25308-25314.  
doi:10.1074/jbc.M002629200 originally published online June 14, 2000  

Access the most updated version of this article at doi: 10.1074/jbc.M002629200  

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.  

Alerts:  
- When this article is cited  
- When a correction for this article is posted  

Click here to choose from all of JBC's e-mail alerts  

This article cites 33 references, 18 of which can be accessed free at  
http://www.jbc.org/content/275/33/25308.full.html#ref-list-1
