
Biochimica et Biophysica Acta 1459 (2000) 346–355

**Electron transfer during the oxidation of ammonia by the chemolithotrophic bacterium Nitrosomonas europaea**

Mark Whittaker, David Bergmann, David Arciero, Alan B. Hooper*

*Department of Biochemistry, Molecular Biology and Biophysics, University of Minnesota, St. Paul, MN 55108, USA

---

### Abstract

The combined action of ammonia monooxygenase, AMO, $(\mathrm{NH}_{3}+2 \mathrm{e}^{-}+\mathrm{O}_{2} \rightarrow \mathrm{NH}_{2} \mathrm{OH})$ and hydroxylamine oxidoreductase, HAO, $(\mathrm{NH}_{2} \mathrm{OH}+\mathrm{H}_{2} \mathrm{O} \rightarrow \mathrm{HNO}_{2}+4 \mathrm{e}^{-}+4 \mathrm{H}^{+})$ accounts for ammonia oxidation in *Nitrosomonas europaea*. Pathways for electrons from HAO to $\mathrm{O}_{2}$, nitrite, NO, $\mathrm{H}_{2} \mathrm{O}_{2}$ or AMO are reviewed and some recent advances described. The membrane cytochrome $c_{M} 552$ is proposed to participate in the path between HAO and ubiquinone. A $b c_{1}$ complex is shown to mediate between ubiquinol and the terminal oxidase and is shown to be downstream of HAO. A novel, red, low-potential, periplasmic copper protein, nitrocyanin, is introduced. Possible mechanisms for the inhibition of ammonia oxidation in cells by protonophores are summarized. Genes for nitrite- and NO-reductase but not $\mathrm{N}_{2} \mathrm{O}$ or nitrate reductase are present in the genome of *Nitrosomonas*. Nitrite reductase is not repressed by growth on $\mathrm{O}_{2}$; the flux of nitrite reduction is controlled at the substrate level. © 2000 Elsevier Science B.V. All rights reserved.

---

### 1. Introduction

Ammonia oxidation, electron transfer and energy transduction has been reviewed in [1–4]. The obligate aerobic chemolithotrophic bacterium *Nitrosomonas* derives energy from the oxidation of ammonia to nitrite (Fig. 1). Ammonia is oxidized to hydroxylamine by ammonia monooxygenase, AMO,

(NH₃ + 2e⁻ + O₂ → NH₂OH) [5]. Hydroxylamine is oxidized to nitrite by hydroxylamine oxidoreductase, HAO (NH₂OH + H₂O → HNO₂ + 4e⁻ + 4H⁺) [6]. HAO will also rapidly oxidize hydrazine to dinitrogen, an experimentally useful reaction which may be of biological significance in the microbial anaerobic transformation of ammonium and nitrite to dinitrogen [7]. Cytochrome P460, a monoheme c-cytochrome with hydroxylamine oxidizing activity is also found in small quantity and in the periplasm [8]. The contribution of this protein to inorganic nitrogen metabolism is not known.

A significant amount of NO and N₂O [9,10] and a trace of N₂ [11] are produced during oxidation of ammonia by *Nitrosomonas* at low concentrations of oxygen. In vitro, NO and N₂O can be products of the oxidation of hydroxylamine by HAO [12], however, production of N₂O in vivo appears to occur by the reduction of nitrite [13–15]. A soluble enzyme with nitrite reductase and dye oxidase activity has been isolated by several workers [16–18]. The enzyme

Abbreviations: AMO, ammonia monooxygenase; HAO, hydroxylamine oxidoreductase; DCCD, $N, N'$-dicyclohexylcarbodiimide; DMSO, dimethylsulfoxide; CCCP, chlorocarbonylcyanidephenyl hydrazone; ASN, ammonium sulfate supernate; ASP, ammonium sulfate precipitate; PCR, polymerase chain reaction; EPR, electron paramagnetic resonance spectroscopy; M_L, low-potential metal center; M_H, high-potential metal center; pMMO, particulate methane monooxygenase; N, cytoplasmic (negative) side of membrane; P, periplasmic (positive) side of membrane

* Corresponding author. Fax: +1-612-625-5780; E-mail: hooper@biosci.umn.edu

0005-2728/00/$ – see front matter © 2000 Elsevier Science B.V. All rights reserved.
PII: S0005-2728(00)00171-7

BBABIO 44926 7-8-00

M. Whittaker et al. / Biochimica et Biophysica Acta 1459 (2000) 346–355

![Diagram](https://i.imgur.com/yourimage.png)

Fig. 1. Electron transport in *Nitrosomonas europaea*. Established electron transport pathways are represented with a solid line and hypothetical pathways are represented with a dashed line. AMO, ammonia monooxygenase; HAO, hydroxylamine oxidoreductase; C-P460, cytochrome P460; Q, Ubiquinone-8; Cu NiR, copper-containing nitrite reductase; CCP, cytochrome c peroxidase; NOR, nitric oxide reductase, Cu $aa_3$-type cytochrome c oxidase; $C_{c_M}552$, membrane cytochrome c552. ‘Alternative substrates’ are substrates other than ammonia [4] and do not produce hydroxylamine.

contains type II copper [17]. In keeping with physiological observations, probable genes for one NO reductase and two dissimilatory nitrite reductase enzymes appear in the genome whereas genes similar to a known nitrate or N₂O reductase are not found [19].

On average, for every four electrons removed from hydroxylamine by HAO and its electron acceptor, cytochrome c554 [20,21], 2 electrons must be returned to the AMO reaction, 1.65 electrons [1] pass to the terminal oxidase (or to a nitrite- or NO-reductase or diheme cytochrome c peroxidase, [22]) and 0.35 pass to NAD. It has been proposed [1] that NADH dehydrogenase, which is found in the genome [19], catalyzes ‘reverse electron transfer’ from ubiquinol to NAD. Cells contain ubiquinone-8 [23] in 13-fold excess relative to HAO and the electron carriers cytochromes c554, c552 and $aa_3$ [24]. Ubiquinol is thought to be an electron donor to the AMO system.

ATP synthase is driven by a proton gradient [25] which is generated by oxidation of hydroxylamine [26]. A value of Δp of 173 mV (inside negative) has been estimated for Nitrosomonas [27]. Invagination of the cytoplasmic membrane result in an extensive ‘periplasmic’ or ‘P-side’ compartment of this Gram-negative bacterium [28]. This compartment contains the HAO reaction [29] which thus contributes four protons to the membrane gradient. Assuming that the terminal oxidase is a proton pump, re-oxidation of ubiquinol by a $bc_1$ complex and cytochrome oxidase would yield the net translocation of six protons from N to P side.

Here we address several recent advances related to electron transfer in Nitrosomonas: (a) a survey of electron transfer components; (b) the role of the cytochrome $bc_1$ complex; (c) the role of cytochrome $c_M552$; (d) The inhibition by protonophores of ammonia oxidation in cells; and (e) control of the flux of nitrite oxidation.

### 2. Materials and methods

#### 2.1. Growth and harvesting of Nitrosomonas europaea

N. europaea (Schmidt strain) was grown in continuous culture as recorded previously [30] utilizing a 15-liter stainless steel and glass reactor chamber under high aeration. Cells were harvested by filtration through a Millipore Pellicon cassette using a 0.45-μm filter and resuspended to 20% w/v in 50 mM potassium phosphate at pH 7.8. The cell suspension was stored at 4°C and used within 24 h for whole cell experiments or frozen at −20°C for protein purification.

2.2. Preparation of Nitrosomonas for electron microscopy

To a 10-ml glass hypovial was added 1 ml Nitrosomonas which had been removed directly from growth tank, washed once and resuspended in 50 mM potassium phosphate at pH 7.8, and the container closed with an aluminum crimp cap and butyl rubber septum. In triplicate, 1-μl aliquots of DMSO, phenylacetylene in DMSO, or CCCP in DMSO were added to vials such that the final concentration of phenylacetylene or CCCP was 10 μM. Cells were incubated at room temperature with orbital shaking for 20 min, after which ammonium sulfate was added to a final concentration of 10 mM from a 1.25 M aqueous stock solution. Cells were incubated for an additional 1.5 h, after which samples were fixed for electron microscopy as described previously [28,31]. Inhibition of CCCP and phenylacetylene-treated cells was assayed by monitoring nitrite concentration as published previously [32]. In order to preserve osmotic balance, the buffer for the fixation procedure was replaced with the buffer used for cell resuspension.

2.3. General procedure for whole cell inhibition experiments

To 5 ml 50 mM potassium phosphate (pH 7.8) in a 10-ml hypovial, equipped with an aluminum crimp

cap with Teflon-backed butyl rubber septum, was added inhibitor in DMSO stock solution to final concentration followed by 20 μl (for ammonia assay) or 5 μl 20% w/v N. europaea cells (for hydroxylamine assay) from the above procedure for cell harvest, shaken orbitally for 20 min then the reaction initiated with the addition of aqueous solution of ammonium sulfate or hydroxylamine to final concentration of 10 mM or 100 μM, respectively. Nitrite production was monitored every minute for the first 10 min and subsequently every 10 min over 50 min.

3. Results and discussion

3.1. A survey of electron transfer components

A sample of 13.5 g wet cells was analyzed. An accounting of the identity and quantity of c-cytochromes in membranes was complicated by the presence of small amounts of HAO (thought to be in process of secretion) and large amounts of cytochrome c 554 (thought to be bound by electrostatic interaction to promote electron transfer [33], since they contain 21 and 4 c-hemes per molecule, respectively. Cytochrome c 554 was removed by salt washing of the membrane. Most heme c is found in the soluble fraction (Table 1). HAO and cytochrome c 552 account for nearly all of the c-heme in the 60–100% ammonium sulfate pellets (ASP) and are

Table 1  
Heme distribution in Nitrosomonas europaea  

|               | Heme c | Heme b | heme a |
|---------------|--------|--------|--------|
| Soluble fraction |        |        |        |
| 40 ASP       | 1.49   | 0      | 0      |
| 60 ASP       | 2.25   | 0      | 0      |
| 100 ASP      | 3.88   | 0      | 0      |
| 100 ASN      | 1.48   | 0      | 0      |
| Membrane fraction |        |        |        |
| KCl Wash     | 1.36   | 0      | 0      |
| HCl/Acetone wash | 0    | 0.89   | 0.91   |
| SDS wash     | 2.03   | —      | —      |

All values are given in μmol. Cells (13.5 g wet weight) of N. europaea were disrupted by a freeze-thaw procedure [34]. After pelleting of the membrane fraction, the soluble fraction was fractionated with ammonium sulfate at 40%, 60%, and 100% of saturation. The membranes were washed two times with buffer containing 1 M KCl [33]. After washing two times with buffer to remove KCl, the membranes were extracted two times with acidic acetone [35] to remove hemes a and b. The membrane pellet was then extracted two times with 0.15 M Tris/Cl, pH 8.5 buffer containing 2% SDS. The amount of heme was determined in alkaline pyridine [36].

present in a molar ratio of about 1:10 [18] (based on the HAO monomer and minimum catalytic unit (HAO(α)), the ratio is 1:3.5). On a heme basis, the ratio is about 2:1. Thus, based on the quantity of heme c in the 60 and 100% ammonium sulfate precipitates, the amount of HAO(α) and cytochrome c552 present in the sample is 0.58 and 2.04 μmol, respectively. The 100% ammonium sulfate supernatant (ASN) and the KCl wash contain almost exclusively the tetraheme cytochrome c554 hence the total amount of cytochrome c554 present is 0.71 μmol.

All of the heme a and b is found associated with the membrane. The membranes do not contain heme o or other heme types. The 0.91 μmol of heme a indicate the presence of 0.45 μmol of aa₃ oxidase. The 0.89 μmol of heme b may be associated with a total of 0.45 μmol of bc₁ complex and NO reductase, if it is expressed, though the presence of other b-heme containing proteins in the membrane cannot be excluded. Of the 2.03 μmol of c-heme in the membrane 0.45 μmol are presumably distributed between the bc₁ complex and NO reductase, if present, and membrane HAO and cytochrome c₄ might account for as much as 0.2 μmol. The remaining c-heme could account for as much as 0.35 μmol of the tetraheme cytochrome cM552. The relative amounts of electron transfer components are approximately 1.0 HAO α subunit:1.3 cytochrome c554:0.6 cytochrome cM552:13 ubiquinone-8:0.78 bc₁ complex (together with NO reductase):3.5 cytochrome c552:0.78 cytochrome aa₃.

Based on sequencing of the N-terminus (ESPA-AGDVEKGKEIAAGICAGCHNADGN) and two internal heme-containing tryptic peptides (HemePeptide I: IAAGICAGCHNADGNSAIPLYPILAGQY-PGYIA; HemePeptide II: LYGNGNLSENSIPAC-SSCHSPNGQGIPPHYTR), a cytochrome isolated from the membrane fraction of *N*. europaea by washing with H₂O was identified as a cytochrome c₄ and this was confirmed by the complete sequence from the Nitrosomonas genome [19]. Cytochrome c₄s have been reported in several *Pseudomonas* species [37], *Azotobacter vinlandii* [38], and *Thiobacillus* [39]. These are well characterized diheme cytochromes whose function may be as an immediate electron donor to terminal oxidation systems, possibly acting in parallel with cytochrome c₅ in donating electrons

to a cytochrome o oxidase [40]. The cytochrome c₄ from *Nitrosomonas europaea* has similar spectroscopic and electrochemical properties to other members of the cytochrome c₄ family. However, a cytochrome o terminal oxidase does not appear to be present in the organism. HPLC analysis for heme o has shown it is not present, and gene sequences for the polypeptides have not been found in the genome. Thus, an alternate role for cytochrome c₄ in *Nitrosomonas* must be contemplated. Since it is present at molar levels about 5–10% of the main proteins involved in nitrification, it is unlikely to play a major role in the process.

Two other c-heme containing polypeptides have recently been identified in minor amounts from the soluble fraction of *Nitrosomonas europaea*. They co-purify during all purification steps and may represent subunits of a single protein. N-terminal sequencing (Polypeptide 1: ASDDSFEHAERLYDTYCTQCH-GVNRDGNGVNS; Polypeptide 2: ADVPAVLQ-SKCASCHALTKPESN) and a subsequent BLAST search of the Nitrosomonas genome database [19] reveals that their genes are in the same operon and may also share an operon with an apparent outer membrane nirK, Cu-containing nitrite reductase. Although a soluble blue-copper nitrite reductase has been isolated from *N*. *europaea* [18], the N-terminal sequence that we have obtained does not match this nirK gene.

### 3.2. Nitrosocyanin

A new monocopper cupredoxin has recently been isolated and named nitrosocyanin [41]. It is a very unusual red copper protein whose concentration is equivalent to the other components of the ammonia oxidizing system (AMO, HAO and other c-cytochromes). The gene sequence has been obtained by protein chemistry and PCR (D. Arciero, B. Tessema, A.B. Hooper, unpublished data). Based on the genome sequence [19], the protein contains a leader sequence, thus the protein is periplasmic. The protein sequence exhibits some homology to plastocyanin, the electron donor copper region of N₂O reductase. The absorption band of the cupric form is at 390 nm rather than 600 or 450 nm peaks found in blue copper proteins or perturbed blue copper proteins, respectively. The redox potential (+85 mV) is low rel-

ative to other members of this family (whose values
are in the +300 mV range). The role of nitrosocyanin
in Nitrosomonas has not been determined.

### 3.3. A cytochrome bc₁ complex is 'downstream' from HAO

Genes for all proteins of a cytochrome bc₁ are
found in the genome of Nitrosomonas [19], however
the expression of the cytochrome bc₁ complex and its
participation in electron transfer between cyto-
chrome c554 and the terminal oxidase has not been
documented. Since the terminal oxidase is a CuA
type [42], the electron donor is presumably a peri-
plasmic c-cytochrome, possibly cytochrome c552
[20], rather than a quinol. Although the oxidation/
reduction potential for cytochrome c552 (+220 mV,
[20]) is in keeping with this role, genetic or kinetic
evidence has not established its significance. Since
two of the four electrons from hydroxylamine/
HAO/cytochrome c554 may reduce ubiquinone and
then pass to AMO, it is logical that the two electrons
bound for cytochrome c oxidase should also reduce
ubiquinone and pass through a bc₁ complex to cyto-
chrome oxidase. Acceptance of that route has been
complicated by the readily reproducible observation
that, in vitro, electron flow from substrate quantities
of hydroxylamine to substrate quantities of the solu-
ble cytochrome c552 is extremely rapid in the pres-
ence of catalytic amounts of HAO and cytochrome
c554 [20, 21]. This observation called for the sugges-
tion that electron flow from cytochrome c554 to the

terminal oxidase did not involve the cytochrome bc₁
complex [2, 3].
Here we present evidence that, in vivo, the bc₁
complex is, in fact, a component of the electron
transfer path, mediating electrons downstream of
HAO. As shown in Table 2, the aerobic oxidation
of either hydroxylamine or ammonia by suspensions
of cells of Nitrosomonas is inhibited by antimycin A
(50% inhibition at 10⁻⁴ M), myxothiazol (1 mM),
stigmatellin (10⁻⁴ M), all of which are known inhib-
itors of the bc₁ complex [43]. The data indicate that
the bc₁ complex is expressed and functions down-
stream from the HAO/cytochrome c554 pair. This
simplifies the model pathway for electron transfer
(Fig. 1). It also fits well with the structures of cyto-
chromes c554 and c552 and HAO; whereas a very
reasonable docking site for an HAO/cytochrome
c554 complex is observed, cytochromes c554 and
c552 do not appear to share a complementary sur-
face [44].

Cytochrome c552 is found in much higher concen-
trations than the other redox components of the N-
oxidation pathway. Further, its gene is present in a
single copy which is not in any of the three gene
clusters encoding HAO and cytochrome c554 [45].
Cytochrome c552 is also able to reduce cytochrome
c peroxidase [22] and possibly the copper nitrite re-
ductase [17]. Thus, distribution of electrons to alter-
nate carriers may be a function of that protein. It is
possible that, certain under circumstances in vivo,
the bc₁ complex can be circumvented. As seen here,
inhibition by antimycin A, myxothiazol or stigmatel-

---

**Table 2**

Effect of inhibitors of the bc₁ complex on ammonia and hydroxylamine oxidation

| Inhibitor       | Concentration (M) | Rate of oxidation |
|-----------------|--------------------|-------------------|
|                 |                    | NH₃ (% of control) | NH₂OH (% of control) |
| Antimycin A    | 10⁻⁵              | 41                | 45                   |
| Myxothiazol    | 10⁻⁴              | 67                | 75                   |
| Stigmatellin   | 10⁻⁴              | 23                | 18                   |
|                 | 10⁻⁵              | 34                | 46                   |
|                 | 10⁻⁶              | 41                | 49                   |
| Diphenylamine  | 10⁻⁴              | 5                 | 44                   |
|                 | 10⁻⁵              | 87                | 69                   |
|                 | 10⁻⁶              | 100               | 77                   |

Known inhibitors of the bc₁ ubiquinone reducing site (Qₙ, antimycin A) and ubihydroquinone oxidizing site (Qₚ, myxothiazol, stig-
matellin, diphenylamine). The data shown are the percent remaining ammonia or hydroxylamine oxidizing capability relative to con-
trol experiments containing no inhibitors. Hydroxylamine and cell concentrations were adjusted as described in Section 2 to prevent
inhibition of hydroxylamine oxidation in cells (observed at greater than 100 μM NH₂OH).

lin is not complete. The reasons are not understood but, in this case, could include the utilization of an alternate pathway such as the cytochrome c552 bypass.

### 3.4. Cytochrome $c_M$552; a possible electron donor to ubiquinone

The gene encoding cytochrome $c_M$552 is immediately downstream of the gene for cytochrome c554 (part of a gene cluster containing HAO) [46] and is co-transcribed with the latter gene [47]. The protein is predicted to have four $c$-heme attachment sites. The N-terminal sequence of the purified detergent-solubilized protein appears to be the only transmembrane domain [3]. Based on its amino acid sequence, this protein is a member of the 'NirT/NapC' family of tetraheme membrane-anchored cytochromes implicated by genetic experiments as electron transfer agents from membrane quinols to periplasmic terminal electron acceptors (nitrate reductase, nitrite reductase, DMSO reductase, etc.) [46,48]. However, we and others [49] suggest that cytochrome $c_M$552 of *Nitrosomonas* carries electrons from the oxidation of hydroxylamine by HAO/c554 in the periplasm to ubiquinone in the membrane. Although all members of the NirT/NapC family have similar N-terminal portions with four $c$-heme binding motifs [46], all differ in their C-terminal domains. Since they are all thought to react with membrane quinones and each reacts with a unique periplasmic electron acceptor (or donor), it is likely that the conserved portions of these cytochromes all interact with quinones in the membrane whereas the variable C-terminal regions interact with differing periplasmic electron transport proteins. The C-terminal domain of *Nitrosomonas* is particularly unique; 24 of the 60 amino acids are aspartate or glutamate.

The optical spectrum of this protein is shown below (Fig. 2).

The only other member of this family of proteins which has had substantial biochemical analysis is NapC, the putative mediator to the periplasmic nitrate reductase [48]. A contrast of properties and function between this protein and cytochrome $c_M$552 of *Nitrosomonas* is likely to prove interesting. One property of this protein may be relevant to the $\Delta P$ economy of *Nitrosomonas*. Since all of the

---

![Figure 2](#fig2)

**Fig. 2.** Optical spectra of $c_M$552. An aliquot of $c_M$552 purified through DEAE-Sephrose and gel filtration chromatography was diluted to 1.4 $\mu$M, based on heme absorbance at 408 nm, in 50 mM potassium phosphate (pH 7.0). The UV-visible spectrum of the resting oxidized (solid line) and potassium dithionite-reduced (dashed line) protein was recorded in a 1-cm path-length quartz cuvette.

protein except the transmembrane domain is likely to be periplasmic (or on the P face of the membrane), reductive protonation of ubiquinol anion would presumably occur at the P face of the membrane. If this is true, the reduction of two molecules of ubiquinone would be promoted by $\Delta P$ and would, in effect, consume the four protons yielded in the HAO reaction. This effect would aid in overcoming the thermodynamically unfavorable values of the oxidation-reduction potential for the 2 HAO-reducible hemes of cytochrome c554 (+47 mV, [50]) and ubiquinone ($-$90 mV [36]).

### 3.5. Inhibition of ammonia oxidation by protonophores and inhibitors of proton transport

Ammonia dependent- but not hydroxylamine dependent- oxygen uptake and nitrite production by cells of *Nitrosomonas* is reversibly inhibited by the protonophores chlorocarbonylcyanidephenyl hydrazone (CCCP), tetrachlorosalicylanilide, and $\alpha\alpha'$-bis(hexafluoroacetyl)cyclohexanone (duPont 1799) in the $10^{-5}$ M range and dinitrophenol in the $10^{-4}$ M range (Table 3, [51]). The inhibitory effect of CCCP is apparently on AMO or its electron donor system and not the HAO-to-terminal oxidase reaction and electron transfer pathway which is, in fact, stimulated by CCCP. The concentration dependence of

Table 3  
Effect of protonophores, ionophores, and inhibitors of proton transport on ammonia and hydroxylamine oxidation  

| Inhibitor       | Concentration (µM) | Rate of oxidation | NH₃ (% of control) | NH₂OH (% of control) |
|-----------------|--------------------|-------------------|--------------------|----------------------|
| CCCP            | 10                 |                   | 5                  | 128                  |
| FCCP            | 10                 |                   | 6                  | ND                   |
|                 | 10                 |                   | 19                 | ND                   |
|                 | 10                 |                   | 91                 | ND                   |
| DCCD            | 10                 |                   | 0                  | 29                   |
| Valinomycin     | 10                 |                   | 95                 | 82                   |
|                 | 10                 |                   | 100                | 95                   |
| Gramicidin A    | 10                 |                   | 109                | ND                   |
|                 | 10                 |                   | 115                | ND                   |
|                 | 10                 |                   | 97                 | ND                   |

Assays were performed using the procedure outlined in Table 1, with stock solutions of inhibitors dissolved in DMSO or ethanol.  
ND, not determined.

the inhibition of ammonia oxidation and stimulation  
of hydroxylamine oxidation by CCCP are very similar, suggesting that the two effects of CCCP share a  
common mechanistic basis. The ionophores valinomycin or gramicidin A are not inhibitory, hence an  
effect on a sodium or potassium gradient is not involved.  

As suggested by Wood [1], a simple mechanism for  
inhibition of ammonia oxidation by CCCP is that  
the dissipation of the proton gradient and the resulting  
increased flux of electrons through the bc₁/cytochrome oxidase pathway to dioxygen (Fig. 1) (reflected in increased rate of hydroxylamine  
oxidation) decreases the flux of electrons to AMO.  
The inhibition of ammonia oxidation by catalytic  
concentrations of phenazine methosulfate [50] is an  
example of this general mode of inhibition: PMS is  
rapidly reduced by HAO and reoxidized by O₂ [52],  
hence electrons would not reach AMO. A second  
mechanism is based on the apparent ability of  
CCCP to catalyze the decay of a reduced state of  
the reaction center in Photosystem II, 'ADRY' (Acceleration of Deactivation Reactions of the water splitting enzyme systems Y) [53]; in this way CCCP  
might cause electrons to bypass AMO.  

Inhibition by protonophores also raises the interesting possibility that ammonia oxidation is dependent on a proton gradient, possibly in order to transfer electrons from a redox carrier through a lower potential electron donor to AMO. Electron donors to the active site of many monooxygenase enzymes

have values of midpoint potentials which are  
lower than the redox potential of ubiquinol, e.g., flavins (-190 and -328 mV) of microsomal cytochrome P450 reductase [54,55], putidaredoxin (-230 mV) of cytochrome P450cam [56] or an Fe₂S₂ center (-220 mV) of soluble MMO [57]. Fig. 3 illustrates a plausible mechanism for reverse electron transfer within the AMO reaction (ubiquinol + NH₃ + O₂ → NH₂OH + H₂O + ubiquinone). If appropriately oriented in the membrane, the partial reactions (involving hypothetical low and high potential metal centers, M_L and M_H, respectively):  
(a) ubiquinol + 2M_H^{++} → ubiquinone + 2M_H^{++} + 2H⁺  
on the N face; (b) 2M_L^{++} + 2M_H⁺ → 2M_L⁺ + 2M_H^{++} occurring vectorally from N to P; and (c) 2H⁺ + O₂ + 2M_L⁺ + NH₃ → NH₂OH + 2M_L^{++} on the P face, would lead to utilization of two P protons, production of two N protons and vectoral movement of an electron from N to P side and could require a charge gradient for turnover of the enzyme. A pair of b-cytochromes similar to those which have been proposed to act as electron donors to the pMMO, particulate methane monooxygenase, of *Methylococcus capsulatus* [58] could fill the role of M_L and M_H. The need for proton gradient coupling would explain the observed difficulty in obtaining high levels of AMO (or pMMO) activity in crude extracts. Mechanistic coupling of ammonia oxidation to transmembrane movement of protons and/or electrons would also explain the presence of multiple transmembrane regions in AMO [59]. Possible precedent for reactions

M. Whittaker et al. / Biochimica et Biophysica Acta 1459 (2000) 346–355

HAO, cyt c554, cyt $c_{\mathrm{m}}$552

AMO

$\begin{aligned}
& \text { HAO, cyt c554, cyt } c_{\mathrm{m}} 552 \\
& \left.\begin{array}{l}
P \\
H_{2} O \\
N H_{2} O H \\
2 Q
\end{array}\right\} \rightarrow\left\{\begin{array}{l}
H N O_{2} \\
2 Q H_{2}
\end{array}\right\} \\
& \text { bc }_{1} \quad 2 Q \\
& N \\
& \text { AMO } \\
& \left.\begin{array}{l}
2 H^{+} \\
2 H^{+} \\
N H_{3} \\
O_{2} \\
2 M_{L}^{+}
\end{array}\right\} \rightarrow\left\{\begin{array}{l}
H_{2} O \\
N H_{2} O H \\
2 M_{L}^{++}
\end{array}\right\} \\
& 2 M_{L}^{+} \leftarrow 2 M_{L}^{++} \\
& 2 e^{-} \\
& 2 M_{H}^{++} \leftarrow 2 M_{H}^{+} \\
& \left.\begin{array}{l}
Q H_{2} \\
2 M_{H}^{++}
\end{array}\right\} \rightarrow\left\{\begin{array}{l}
Q \\
2 M_{H}^{+} \\
2 H^{+}
\end{array}\right\} \\
& 2 H^{+}
\end{aligned}$

Fig. 3. Hypothetical mechanism for proton gradient-driven ammonia oxidation activity. Electrons generated by HAO through oxidation of hydroxylamine are passed through $c 554$ and to ubiquinone-8 via $c_{\mathrm{M}} 552$ (left hand dashed box). Electrons from ubiquinol are passed either to the $b c_{1}$ complex (and cytochrome $c$ oxidase) or to AMO. AMO is represented in the right-hand tripartite dashed box. $M_{L}$ and $M_{H}$ are low and high potential metal centers, respectively. The AMO reaction is represented as containing (a) an N-side Ubiquinol-$M_{H}$ reductase, (b) a transmembrane $M_{H}-M_{L}$ reductase, and (c) an $M_{L}$-dependent oxygen activating center.

driven by an ion gradient include the reduction of CO₂ to formaldehyde [60], the oxidation of nitrite [61] and the reduction of menaquinone by succinate [62].

The presence of large amounts of closely-opposed membranes in ammonia-oxidizing bacteria [28] is consistent with the need for an ion gradient but might, alternatively, be needed for precise juxtaposition of proteins in adjacent membranes. If so, membrane movement caused by osmotic changes following collapse of the proton gradient might inhibit AMO activity. With this in mind, possible major changes in membrane arrangements induced by complete inhibition of ammonia or hydroxylamine oxidation activity by CCCP in *N. europaea* and *Nitrosococcus oceanus* were determined by examination of thin sections by transmission electron microscopy. *Nitrosococcus* was chosen because the detection of major changes was easier in the very large stacks of membrane discs. Photographs containing approximately 10 control cells typically showed a variety of membrane morphologies with the majority represented by the images that have been published [28]. Control and CCCP-treated cells were not noticeably different with respect to the amount of closely-opposed membranes. Hence the effect on the oxidation

of ammonia- or hydroxylamine cannot be attributed to major disruption of cell ultrastructure.

Inhibition at $10^{-4}$ M by $N, N'$-dicyclohexylcarbodiimide (DCCD), known to inhibit proton translocation of the $F_{1} F_{0}$-ATPase, cytochrome oxidases, anion- or divalent cation- translocases by covalent modification of the proton translocation channel, is complete and irreversible (Table 3; Ref. [51]). The oxidation of both ammonia and hydroxylamine are inhibited in an overlapping concentration range and can thus probably be accounted for by an effect on the terminal oxidase.

### 3.6. Control of nitrite reduction in Nitrosomonas

Some elements of the control of the flux of nitrite reduction are known. First, the fact that the nitrite reductase can easily be isolated from aerobically-grown cells indicates that expression the enzyme is not completely repressed by O₂. Given the low density to which *N. europaea* grows, we wished to rule out the slight possibility that a nitrite reductase could have been induced under anaerobic conditions late in the culture cycle or during the accumulation of cells prior to use. Cells from a chemostat culture were rapidly harvested and frozen in liquid N₂ prior to

BBABIO 44926 7-8-00

use, when they were broken in a French pressure cell and soluble extracts prepared by ultracentrifugation. We observed comparable nitrite reductase activity, assayed as *p*-phenylenediamine oxidation [24] in soluble extracts from cells grown at high levels of oxygen (100–40% saturation) and in cells grown at low levels of oxygen (about 1.0% saturation). The contrasting report of Miller and Nicholas [17], who purified substantial quantities of nitrite reductase from cultures grown at high aeration but obtained 10-fold greater quantities of the reductase from cultures grown at low rates of aeration, indicates that the situation is complex.

Secondly, nitrite reductase, though present, does not transform nitrite at concentrations up to 40 mM when the oxygen concentration is high: cultures routinely convert ammonia stoichiometrically into biomass and nitrite. When the oxygen concentration decreases to the 1% range, N₂O will account for a major fraction of the ammonia-N utilized [10]. Hence, in the presence of nitrite, nitrite reductase is present but reacts only when oxygen is diminished. This may account for the observation that in a nitrite-containing chemostat culture, N₂O production was initiated within 15 min after a decrease in oxygen concentration [63]. Hence, although increased N₂O production resulting from de novo synthesis of denitrifying enzymes cannot be ruled out, available evidence suggests that the catalytic turnover of the enzyme is apparently modulated at the metabolite level. Although dioxygen reacts with nitrite reductase and might outcompete nitrite [17], this flux of electrons would represent the loss to the cell of at least the proton gradient generated by cytochrome oxidase. Thus a more attractive hypothesis involves the obligatory participation of a low-potential electron donor in the electron transport pathway leading to nitrite reductase. The flux of electrons to nitrite reductase would be zero as long as this donor remained oxidized and re-initiate when the redox potential of the cell was decreased by limitation of oxygen.

Acknowledgements

Supported by Research Grants from the National Science Foundation (MCB9723608) and Department of Energy (DE-FG02-95FR20191).

References

[1] P.M. Wood, in: J.I. Prosser (Ed.), Nitrification, Spec. Pub. Soc. Gen. Microbiol. vol. 20, 1986, pp. 39–62.
[2] A.B. Hooper, in: H.G. Schlegel, B. Bowien (Eds.), Autotrophic Bacteria, Sci. Tech. Publishers, Madison, WI, 1989, pp. 239–265.
[3] T. Vannelli, D. Bergmann, D.M. Arciero, A.B. Hooper, in: M.E. Lidstrom, F.R. Tabita (Eds.), Proceedings of the 8th International Symposium on Microbial Growth on C₁ Compounds, Kluwer, Dordrecht, 1996, pp. 80–87.
[4] A.B. Hooper, T. Vannelli, D.J. Bergmann, D.M. Arciero, Antonie van Leeuwenhoek 71 (1996) 59–67.
[5] T.C. Hollocher, M.E. Tate, D.J.D. Nicholas, J. Biol. Chem. 256 (1981) 10834–10836.
[6] K.K. Andersson, A.B. Hooper, FEBS Lett. 164 (1983) 236–240.
[7] J. Schalk, S. de Vries, G. Kuenen, M.S.M. Jetten, Biochemistry 39 (2000) 5405–5412.
[8] D. Bergmann, A.B. Hooper, FEBS Lett. 353 (1994) 324–326.
[9] I.C. Anderson, J.S. Levine, Appl. Environ. Microbiol. 51 (1986) 938–945.
[10] T.J. Goreau, W.A. Kaplan, S.C. Wofsy, M.B. McElroy, F.W. Valois, S.W. Watson, Appl. Environ. Microbiol. 40 (1980) 526–532.
[11] M. Poth, Appl. Environ. Microbiol. 52 (1986) 957–959.
[12] A.B. Hooper, K.R. Terry, Biochim. Biophys. Acta 571 (1979) 12–20.
[13] M. Poth, D.D. Focht, Appl. Environ. Microbiol. 49 (1985) 1134–1141.
[14] A.B. Hooper, D.M. Arciero, A.A. DiSpirito, J. Fuchs, M. Johnson, F. LaQuier, G. Mundfrom, H. McTavish, in: P.M. Gresshof, W.E. Newton, W.E. Roth, G. Stacey (Eds.), Nitrogen Fixation: Achievements and Objectives, Chapman-Hall, New York, 1990, pp. 387–391.
[15] A. Remde, R. Conrad, Arch. Microbiol. 154 (1990) 187–191.
[16] D.J. Miller, P.M. Wood, J. Gen. Microbiol. 129 (1983) 1645–1650.
[17] D.J. Miller, D.J.D. Nicholas, J. Gen. Microbiol. 131 (1985) 2851–2854.
[18] A.A. Dispirito, L.R. Taaffe, J.D. Lipscomb, A.B. Hooper, Biochim. Biophys. Acta 827 (1985) 320–326.
[19] J. Lamerdin, F. Larimer, D. Arp, A. Hooper, J. Norton, M. Klotz, L. Sayavedra-Soto, D. Arciero, N. Hommes, M. Whittaker, American Society of Microbiology Annual Meeting, Los Angeles, CA, May 24, 2000, abstract H-141.
[20] T. Yamanaka, M. Shinra, J. Biochem. 75 (1974) 1265–1273.
[21] D. Arciero, C. Balny, A.B. Hooper, Biochemistry 30 (1992) 11466–11472.
[22] D.M. Arciero, A.B. Hooper, J. Biol. Chem. 269 (1994) 11878–11886.

[23] A.B. Hooper, R.H. Erickson, K.R. Terry, J. Bacteriol. 110 (1972) 430–438.
[24] A.A. DiSpirito, L.R. Taafe, A.B. Hooper, Biochim. Biophys. Acta 806 (1985) 320–330.
[25] S. Kumar, D.J.D. Nicholas, FEMS Microbiol. Lett. 14 (1982) 21–25.
[26] T.C. Hollocher, S. Kumar, D.J.D. Nicholas, J. Bacteriol. 149 (1982) 1013–1020.
[27] S. Kumar, D.J.D. Nicholas, J. Bacteriol. 154 (1983) 65–71.
[28] R.G.E. Murray, S.W. Watson, J. Bacteriol. 89 (1965) 1594–1609.
[29] A.B. Hooper, A.A. DiSpirito, T.C. Olson, K.K. Andersson, W. Cunningham, L. Taafe, in: R. Crawford, R. Hanson (Eds.), Microbial Growth on C₁ Compounds, 1984, pp. 53–58.
[30] M.S.P. Logan, A.B. Hooper, Biochemistry 34 (1995) 9257–9264.
[31] E. Kellenberger, A. Ryter, J. Sechaud, J. Biophys. Biochem. Cytol. 4 (1958) 671–676.
[32] D. Arciero, T. Vannelli, M. Logan, A.B. Hooper, Biochem. Biophys. Res. Commun. 159 (1989) 640–643.
[33] H. McTavish, D.M. Arciero, A.B. Hooper, Arch. Biochem. Biophys. 324 (1995) 53–58.
[34] A.B. Hooper, P.C. Maxwell, K. Terry, Biochemistry 17 (1978) 2984–2989.
[35] J.-H. Fuhrup, K.S. Smith, in: Laboratory Methods in Porphyrin and Metalloprotein Research, Elsevier, Amsterdam, 1975.
[36] E.A. Berry, B.L. Trumpower, Anal. Biochem. 161 (1987) 1–15.
[37] G.W. Pettigrew, K.R. Brown, Biochem. J. 252 (1988) 427–435.
[38] R.T. Swank, R.H. Burris, Biochim. Biophys. Acta 180 (1969) 473–489.
[39] C. Cavazza, M.T. Giudici-Orticoni, W. Nitschke, C. Appia, V. Bonnefoy, M. Bruschi, Eur. J. Biochem. 242 (1996) 308–314.
[40] T.C.N. Ng, A.N. Laheri, R.J. Maier, Biochim. Biophys. Acta 1230 (1995) 119–120.
[41] A.B. Hooper, D.M. Arciero, J. Inorgan. Biochem. 74 (1999) 166.
[42] A.A. DiSpirito, J.D. Lipscomb, A.B. Hooper, J. Biol. Chem. 261 (1986) 17048–17056.
[43] B.M. Geier, U. Haase, G. van Jagow, Biochem. Soc. Trans. 22 (1994) 203–209.
[44] T.M. Iverson, D.M. Arciero, B.T. Hsu, M.S.P. Logan, A.B. Hooper, D.C. Rees, Nat. Struct. Biol. 5 (1998) 1005–1012.
[45] H. McTavish, F. LaQuier, D. Arciero, M. Logan, G. Mundfrom, J. Fuchs, A.B. Hooper, J. Bacteriol. 175 (1993) 2445–2447.
[46] D. Bergmann, D. Arciero, A.B. Hooper, J. Bacteriol. 176 (1994) 3148–3153.
[47] D. Bergmann, Ph.D. dissertation, University of Minnesota, 1996.
[48] M.D. Roldán, H.J. Sears, M.R. Cheesman, S.J. Ferguson, A.J. Thomson, B.C. Berks, D.J. Richardson, J. Biol. Chem. 44 (1998) 28785–28790.
[49] D.J. Richardson, N.J. Watmough, Curr. Opin. Chem. Biol. 3 (1999) 207–219.
[50] D.M. Arciero, M. Collins, J. Haladjian, P. Bianco, A.B. Hooper, Biochemistry 30 (1991) 11459–11465.
[51] A. Hooper, K. Terry, J. Bacteriol. 115 (1973) 480–485.
[52] A. Hooper, A. Nason, J. Biol. Chem. 240 (1965) 4044.
[53] B. Hanssum, G. Dohnt, G. Renger, Biochim. Biophys. Acta 806 (1985) 210–220.
[54] T. Iyanagi, N. Makino, H.S. Mason, Biochemistry 13 (1974) 1701–1710.
[55] J.L. Vermilion, M.J. Coon, J. Biol. Chem. 253 (1978) 8812–8819.
[56] M. Sono, M.P. Roach, E.D. Coulter, J.H. Dawson, Chem. Rev. 96 (1996) 2841–2887.
[57] J. Lund, H. Dalton, Eur. J. Biochem. 147 (1985) 291–296.
[58] J.A. Zahn, A.D. DiSpirito, J. Bacteriol. 178 (1996) 1018–1029.
[59] H. McTavish, J. Fuchs, A.B. Hooper, J. Bacteriol. 175 (1993) 2436–2444.
[60] B. Kaesler, P. Schonheit, Eur. J. Biochem. 184 (1989) 223–232.
[61] J.G. Cobley, Biochem. J. 156 (1976) 493–498.
[62] J. Schirawski, G. Unden, Eur. J. Biochem. 257 (1998) 210–215.
[63] R.A. Kester, R. Laanbroek, Appl. Environ. Microbiol. 63 (1997) 3872–3877.
