
articles

Functional insights from the structure of the 30S ribosomal subunit and its interactions with antibiotics

Andrew P. Carter$^{*†}$, William M. Clemons$^{*†‡}$, Ditlev E. Brodersen$^{*†}$, Robert J. Morgan-Warren$^{†}$, Brian T. Wimberly$^{†}$ & V. Ramakrishnan$^{†}$

$^{†}$MRC Laboratory of Molecular Biology, Hills Road, Cambridge, UK  
$^{‡}$Department of Biochemistry, University of Utah School of Medicine, Salt Lake City, Utah 84132, USA  
$^{*}$These authors contributed equally to this work.

The 30S ribosomal subunit has two primary functions in protein synthesis. It discriminates against aminoacyl transfer RNAs that do not match the codon of messenger RNA, thereby ensuring accuracy in translation of the genetic message in a process called decoding. Also, it works with the 50S subunit to move the tRNAs and associated mRNA by precisely one codon, in a process called translocation. Here we describe the functional implications of the high-resolution 30S crystal structure presented in the accompanying paper, and infer details of the interactions between the 30S subunit and its tRNA and mRNA ligands. We also describe the crystal structure of the 30S subunit complexed with the antibiotics paromomycin, streptomycin and spectinomycin, which interfere with decoding and translocation. This work reveals the structural basis for the action of these antibiotics, and leads to a model for the role of the universally conserved 16S RNA residues A1492 and A1493 in the decoding process.

The ribosome is a large, template-directed enzyme with sophisticated proof-reading capabilities. Its ligands are tRNA and mRNA. There are three tRNA-binding sites, designated A (aminoacyl), P (peptidyl) and E (exit), after their respective tRNA substrates (reviewed in ref. 1). The anticodon stem-loops (ASL) of the A-and P-site tRNAs bind to the 30S where they are base paired with adjacent codons on mRNA. The E-site tRNA is bound in a similar orientation, but whether the E-site tRNA is base-paired to the E-site mRNA codon or not is a matter of controversy$^{2,3}$.

The decoding of mRNA into protein requires the correct recognition of each A-site codon by the anticodon of the corresponding aminoacyl-tRNA (aa-tRNA). The difference in binding energy between the codon and anticodon with correct (cognate) versus incorrect (non-cognate) tRNAs is too small to account for the high accuracy of translation$^4$. Initially, a ternary complex of elongation factor Tu (EF-Tu), aminoacyl tRNA and GTP binds to the A-site. The high level of accuracy of translation by the ribosome is thought to be a combination of the initial codon–anticodon interaction and a proof-reading step occurring after GTP hydrolysis and release of EF-Tu$^{5,6}$. The proof-reading step is considered to be especially important for discriminating cognate codon–anticodon interactions from energetically similar near-cognate ones. The mechanism by which proof-reading occurs remains unclear, although a number of models have been proposed$^{7,8}$. There are also models for decoding that do not invoke a separate proof-reading step$^9$.

It has been suggested that the 30S switches between at least two distinct states during translation, and that stabilizing one state over the other can affect accuracy$^{10,11}$. One genetically well characterized conformational switch in the 30S subunit is the helix 27 accuracy switch$^{11}$. Genetic and biochemical data support a model in which this switch may be part of a larger-scale conformational change that occurs between initial selection and proof-reading of the A-site tRNA, or it may have a role in translocation.

Until recently, there has been a large disparity between the high resolution of the genetic and biochemical data that define the RNA components of the active sites of the 30S subunit, and the relatively low resolution of the three-dimensional structures of ribosomes available. Here we analyse the high-resolution structure of the 30S

from *Thermus thermophilus* reported in the accompanying paper$^{12}$, and identify the molecular details of the interaction of mRNA and tRNA with the A-, P- and E-sites of the 30S. We also discuss the structural implications of the switch involving helix 27.

Both subunits are targets for antibiotics. By interfering with various aspects of their function, these antibiotics help to shed light on the mechanisms involved in translation. Although antibiotics were characterized several decades ago, their molecular mechanisms were not clear in the absence of a high-resolution structure. We describe here the crystal structure to 3 Å resolution of 30S from *Thermus thermophilus* complexed with streptomycin and paromomycin (which reduce translational accuracy), and spectinomycin (which interferes with translocation). Difference Fourier maps calculated after the refinement of our atomic model of the 30S against these data reveal the bound antibiotic molecules *in situ*. The structure allows us to rationalize much of the biochemical and genetic data on these antibiotics and propose models for their mode of action. In addition, the effect of paromomycin on the universally conserved bases A1492 and A1493 indicates a model for how they participate in the decoding process.

Results

A single round of refinement against the native 30S coordinates resulted in a model with $R/R_{\text{free}}$ of 0.231/0.266. During refinement, the 30S structure changed significantly only in the vicinity of the paromomycin-binding site. Elsewhere, the phosphate r.m.s. deviation between the antibiotic-bound and free 30S structures fluctuated by about 0.45 Å, which is comparable to the coordinate error in the original model. Electron density for the antibiotics was visible in both difference Fourier maps and $\sigma$-weighted $2mF_o - DF_c$ maps. At 3 Å resolution, it was straightforward to position each antibiotic in an unambiguous orientation in the density, so that precise conclusions could be made about the location and interactions of each antibiotic in the 30S subunit. A second round of refinement using a model that included the antibiotics reduced the $R/R_{\text{free}}$ to 0.224/0.258. Crystallographic statistics are shown in Table 1 of the Supplementary Information.

In the following discussion, we analyse the 30S structure with

respect to function. As described in the accompanying paper, we use the standard H1–H45 nomenclature¹³ to describe the helical elements in 16S RNA and the numbering used is for the *Escherichia coli* 16S RNA sequence¹².

**Mimics of P-site tRNA and mRNA**

Functionally, the most important parts of the 30S subunit are its substrate-binding A-, P- and E-sites. In our 30S crystal structure¹², the P-site is occupied by the tip of the spur stem-loop (H6) from a symmetry-related 30S subunit. The spur thus appears to mimic the P-site tRNA anticodon stem-loop. A superposition of the 7.8 Å 70S structure including mRNA and tRNA¹⁴ onto our 30S structure using the common elements H27 and H44 gave a phosphate r.m.s. deviation of only 2.3 Å after rejection of outliers. This superposition shows residues 27–43 of P-site tRNA almost exactly coinciding with residues 75–95 of the spur (Fig. 1a), indicating that it is a good mimic of the ASL. Our 30S maps also contain electron density for single-stranded RNA that mimics mRNA codons in the P- and E-sites, with base pairing between the P-site codon and residues that correspond to the anticodon of the spur. This mRNA mimic is of uncertain origin, but its location and sequence (UCU, inferred from the non Watson–Crick base-pairing geometries with the UUU of the spur) indicate that it corresponds to the 3′ end of 16S RNA. The last nucleotide of our 16S model is A1534, which implies that four disordered nucleotides 1535–1538 would span the 7 Å distance to the first nucleotide of the E-site codon mimic. Alternatively, it is possible that the 3′ end of 16S RNA has been cleaved somewhere between A1534 and U1539 before or during crystallization. Spur coordinates are available in the Supplementary Information.

---

**Definition of the 30S A-, P- and E-sites**

The 16S RNA-based superposition of the 70S tRNAs and mRNA onto the 30S structure also provides a basis for defining the A-, P- and E-sites on the 30S subunit (Fig. 1b–d). There are no serious clashes between the A-, P- and E-site tRNAs and the 30S subunit (Fig. 1b). The 30S A-, P- and E-site elements defined in this manner are shown in Fig. 1c and d. The most striking characteristic of the three binding sites is that they are all composed of RNA elements from at least two different domains (Fig. 1c). It is clear that independent motion of the head (3′ domain) or platform (central domain) observed in low-resolution cryo-EM studies¹⁵,¹⁶ would have important consequences for movement of the tRNA–mRNA complex, as must occur during translocation.

The P-site. The spur contacts several discrete regions of 16S RNA, and surprisingly, two proteins (Fig. 2a and b). Most of the interaction surface is found on the minor groove of the spur stem and RNA nucleotides 1338–1341, 1229–1230 and small portions of the carboxy-terminal tails of proteins S13 and S9. There are seven hydrogen bonds from the minor groove of spur residues C91–C92 and G78 to the minor groove surface of G1338–A1339. Only one of these hydrogen bonds appears to be sequence specific. Both G1338 and A1339 have been implicated in P-site binding¹⁷. A contact from K126 of S9 appears to help to stabilize this minor-groove-to-minor-groove packing interaction. A second area of contact, nearly continuous with the first, is the interaction between the 16S 1229–1230 sugar-phosphate backbone and spur residues G77 and G78. This region of contact is extended by the C-terminal tail of S13. The other areas of contact are less extensive. One interaction is the stacking of U82 on C1400, which rationalizes

---

**Figure 1 Overview of the 30S A-, P- and E-sites.**  
**a**, Superposition of the 70S P-site tRNA anticodon stem-loop (ASL)¹⁴ with its mimic in the 30S structure, the spur stem-loop.  
**b**, Model of the A-, P- and E-site mRNA and tRNA ASLs on the 30S surface.  
**c**, Secondary structure of 16S RNA. Magenta, A-site; red, P-site; yellow, E-site.  
**d**, Model of the A-, P- and E-site mRNA and tRNA ASLs with 30S components of the three sites coloured as in **c**. This and other figures were prepared with RIBBONS⁴⁶.

the ASL34–C1400 ultraviolet-induced crosslink¹⁸. The other region is a packing interaction between A790 and spur residues 88–89, with a single hydrogen bond present. Finally, if the spur anticodon– mRNA codon helix were a few angstroms wider, as it would be for a Watson–Crick-paired helix, it would make van der Waals contact with the base of G966, which has been implicated as part of the P-site by chemical modification and binding experiments¹⁷,¹⁹.

The P-site codon threads through the major groove of the upper portion of helix 44, in a universally conserved region of 16S RNA (Fig. 2a, b and e). There is a tight turn between nucleotides –1 and +1, between the last E-site and the first P-site codon nucleotides. This turn is stabilized by a hydrogen bond to the N1/N2 groups of the conserved residue G926, a residue previously implicated in P-site binding¹⁹. Additional hydrogen bonds are observed between

the 2′ OH of nucleotide +1 to the phosphate of C1498, and between the phosphate of nucleotide +2 and the 2′ OH of C1498. The phosphate of +2 also stacks on the base of C1498. The phosphate of +3 is within hydrogen-bonding distance of two conserved cytidine N4 groups from C1402 and C1403. The +3 base also stacks on the sugar of C1400. Finally, it appears probable that there are several Mg²⁺ ions that may help stabilize the location of the P-site codon in the major groove of H44. One such ion appears to interact with the N7 atom of G1401, in agreement with biochemical data implicating this residue in P-site function¹⁷,¹⁹. The contacts are also consistent with mRNA crosslinking data (summarized in ref. 20).

The A-site. The A-site is much wider and shallower than the P- or E-sites (Figs 1b and 2c), consistent with its much lower affinity for tRNA. The shallowness of the A-site may reflect the need to allow

![Figure 2](attachment:Figure_2.png)

**Figure 2** Detailed views of the 30S A-, P-, and E-sites with models of bound tRNA and mRNA ligands. **a, b,** Two roughly perpendicular views of the spur in the P-site. Dotted yellow lines indicate base pairing between the spur (red) and the codon mimic (dark blue). Dotted black lines indicate hydrogen bonds. RNA bases coloured black have been implicated in binding of P-site tRNA or mRNA (see text). **c,** Stereo view of the A-site, including the A-site ASL (magenta) and codon (green) modelled as in Fig. 1. Red balls

highlight RNA residues implicated in binding of A-site tRNA or mRNA by biochemical experiments (see text). **d,** Stereo view of the E-site, including the E-site codon from the 30S structure (purple) and the ASL (yellow) modelled as in Fig. 1. **e, f,** Two views of the mRNA model shown in **a–d** (see text). **g,** Inset showing the location of the A-, P- and E-sites on the 30S subunit.

rotation of the A-site codon–anticodon helix during or after GTP hydrolysis by EF-Tu. The RNA components of the A-site include portions of the 530 loop, H31 and H34 in the head, as well as residues 1492–1493 from the 3′minor domain (Fig. 2c), all of which have been implicated as elements of the decoding site (see articles in ref. 21). Interestingly, the electron density for A1492 and A1493 is not consistent with a single conformation for these residues in the absence of antibiotics. The only protein that appears even remotely likely to participate directly in decoding is S12, whose K47 loop is close to the A-site codon–anticodon helix. Other proteins are more distant: a poorly conserved part of the C-terminal tail of S13 lies between the A- and P-sites, and the conserved but disordered C-terminal 12 residues of S19 may be in approximately the right location to interact with the upper portion of the A-site ASL.

The E-site. Unlike the A- and P-sites, the E-site consists mostly of protein (Fig. 2d). Proteins S7 and S11 have a small interface that binds the minor groove of the E-site ASL. The position of the highly conserved β-hairpin of S7 indicates that it might help dissociate the E-site codon–anticodon. The RNA portion of the E-site makes relatively few interactions with the modelled E-site ASL, in accordance with the observed failure of the E-site tRNA to footprint 16S

RNA<sup>17</sup>. Several of the RNA residues footprinted by P-site tRNA, including G693, A794 and C795 are found to be in contact with the E-site codon<sup>17</sup>.

The helix 27 accuracy switch. In addition to the ligand-binding A-, P- and E- sites, the 30S subunit must also contain conformational switches that are important for its function. Only one of these, involving H27, has been well characterized genetically and biochemically<sup>11</sup>. H27 has been proposed to have two alternative base-pairing schemes during translation: a ribosomal ambiguity (ram) or ‘error-prone’ form with nucleotides 885–887 paired to 910–912, and an alternative hyperaccurate or ‘restrictive’ form with nucleotides 888–890 paired to 910–912. The ram form features an S-turn motif in H27, which is present in our structure (green in Fig. 3a). It packs against the minor groove of H44, just below the decoding site (Fig. 3b). Switching to the restrictive form must disrupt the S-turn, and change the packing of H27 and H44 and perhaps the structure of the decoding site. Indeed, strains containing S12 restrictive mutations exhibit altered chemical reactivity of RNA residues at the H27–H44 interface (orange and magenta in Fig. 3b)<sup>10</sup>. However, H27 also packs against several other helices: the conserved hairpin loop at the end of H27 packs against the platform (H24), and there is a long-range base pair between H27 and the root


![Figure 3](#fig3)

**Figure 3** The H27 switch and its environment in the 30S subunit. **a**, H27 with the triplet switch highlighted in red and green. In the *ram* conformation pictured here, there is an S-turn. **b**, The packing of H27 and H44. Residues whose chemical reactivity changes in restrictive S12 strains are labelled<sup>10</sup>. **c**, The environment of H27 in the 30S subunit.


Red balls highlight sites of restrictive mutations in S12; red sticks show A1492 and A1493. **d**, The H1–H18 interaction surface is contiguous with the S4–S5 surface. The locations of *ram* mutations in S4 and S5 are highlighted in red, as are the locations of residues 8 and 26 whose chemical reactivity changes in S4 *ram* mutants<sup>10</sup>.
of H11 (Fig. 3c). Thus, it seems probable that the switch also affects other helix–helix interactions, and in fact there could be a network of coupled switches among H1, H27, H18, H34 and H44. In support of this, chemical protection experiments show that a number of RNA residues in these helices have greater chemical reactivity in the restrictive state, which indicates that the restrictive state may be a more ‘open’ state of the 30S subunit$^{11}$.

ram mutations often occur in S4 or S5 (reviewed in ref. 22), all but one of which map to the interface between the two proteins (red amino acids in Fig. 3d). S4 ram strains exhibit increased chemical reactivity in two RNA sites at the interface between the central pseudoknot (H1) and the 530 pseudoknot (H18; red RNA sticks in Fig. 1d)$^{10}$. Several streptomycin resistance mutations are also found at this RNA–RNA interface. Interestingly, the S4–S5 and the H1–H18 interaction surfaces are contiguous, which indicates that this hybrid RNA–protein surface may move, and that an important function of the S4–S5 interaction may be to modulate the interaction of H1 with H18. It is probable that ram mutations preferentially destabilize the restrictive state, as with the S4–S5 interface and its RNA extension.

### Interaction of antibiotics with the 30S Spectinomycin

Spectinomycin inhibits elongation-factor-G-catalysed translocation of the peptidyl-tRNA from the A-site to the P-site$^{23}$. The fused ring system in spectinomycin makes it a rigid molecule. It binds in the minor groove at one end of H34, makes a single contact with a $2^{\prime}$OH and makes hydrogen bonds to a number of bases (Fig. 4). The most interactions are made with G1064 and C1192, consistent with protection studies$^{24}$ and mutagenesis data on spectinomycin resistance$^{25}$. These two bases are too far apart to form Watson–Crick base pairs, but are able to make a single hydrogen bond. A loop of S5 and part of H28 of 16S RNA are within 5 Å of the spectinomycin-binding site, but in this state do not make direct contacts with it. It is possible, however, that in other conformations of the 30S spectinomycin is in more direct contact with these regions.

Translocation of tRNA from one site to the next must involve movement of elements of the head (Fig. 2). It is probable that such movements involve H34 and a possible rearrangement of the connections between it and helices H35 and H38. The structure indicates that the rigid spectinomycin molecule binds near this pivot point of the head and sterically blocks movement, although it is also possible that it prevents changes in the conformation of H34. Mutations in S5 that cause resistance to spectinomycin$^{26}$ do not make direct contacts with the antibiotic. Rather, they map to a loop that stabilizes the interaction between the central pseudoknot, H28 and the H35–H36 region that is directly connected to H34. An attractive hypothesis is that the mutants destabilize this interaction and, by thus removing the network of interactions that stabilizes the conformation of the head to the body through S5, allows it to move even when spectinomycin is bound.

Streptomycin. Early experiments indicated that streptomycin made ribosomes error prone by affecting the proof-reading step$^{27}$. More recent data indicate interference with both the initial selection and proof-reading$^{28-30}$.

Streptomycin is tightly bound to the phosphate backbone of 16S RNA from four different parts of the molecule through both salt bridges and hydrogen bonds (Fig. 5). It also makes contact with K45 from protein S12. The four nucleotides of 16S RNA (13, 526, 915, and 1490) have been implicated in streptomycin binding on the basis of protection$^{24}$, crosslinking$^{31}$ and mutagenesis data$^{32-34}$.

The tight interactions observed for streptomycin indicate that it preferentially stabilizes the ram state seen in the crystal structure. The restrictive A-site has a low tRNA affinity, whereas the ram state has a higher affinity$^{7,11}$. Therefore, by stabilizing the ram state, streptomycin would be expected to increase initial binding of non-cognate tRNAs. The preferential stabilization of the ram state would also make the transition to the restrictive state more difficult, thereby also affecting proof-reading. Thus, our results offer a structural rationale for the observed properties of streptomycin.

This stabilization of the ram state by streptomycin indicated by our structure can also explain much of the genetic data on the

**Figure 4** Interaction of spectinomycin with the 30S ribosomal subunit. **a**, Difference Fourier maps showing the binding site of spectinomycin in helix 34. **b**, Chemical structure of spectinomycin, showing interactions of the various groups with specific residues of 30S. **c**, The spectinomycin-binding site, showing its location at a pivotal point in the head of the 30S subunit. **d**, Inset showing spectinomycin in a space-filling model, and the location of its binding site on the 30S.
antibiotic. Mutations in S12 lead to a hyperaccurate phenotype (reviewed in ref. 22). A weak phenotype manifests itself as streptomycin resistance, whereas a strong phenotype (often the result of multiple mutations) leads to streptomycin dependence. Most of these mutants are to varying degrees more hyperaccurate and slower than wild-type ribosomes, consistent with destabilization of the ram state with respect to the restrictive state.

All the mutations in S12 map to protein loops that connect and hold in place the 908–915 and 524–527 regions, with the exception of one mutant K56 (*E. coli* K53) which contacts H44 (Fig. 2). Thus, S12 stabilizes the same region that is stabilized by streptomycin. In the resistance mutations, the *ram* state is destabilized sufficiently so that the additional stabilization induced by streptomycin does not trap the ribosome in this state. In the streptomycin-dependent mutants, the *ram* state is so destabilized that the restrictive form predominates. Streptomycin can then help stabilize the *ram* state sufficiently to restore the balance between the two states and help restore translation.

This hypothesis is supported by analysis of the K45R (*E. coli* K42) mutant, which is the only known mutant that is resistant to streptomycin but not hyperaccurate²². K45 forms a salt bridge with phosphate A913 and thus contributes to stabilization of the *ram* state. It also makes direct hydrogen-bonding contacts to two OH groups on streptomycin (Fig. 2). Mutation of this lysine to arginine would disrupt the hydrogen bonding and thereby reduce the affinity of the 30S for streptomycin, leading to resistance. However, the mutation would leave the salt bridge intact, so that the *ram* form is not destabilized and thus translation remains normal.

A number of mutations in ribosomal RNA also lead to hyperaccuracy³²,³³,³⁵⁻³⁸. Some of these nucleotides are involved in hydrogen-bonding interactions in regions close to the streptomycin-binding site. Thus, the mutations disrupt interactions that help to stabilize the *ram* state. Others such as A915 make no contacts

with any other bases. It is possible that mutation of this base leads to more favourable contacts in the restrictive state, acting by stabilizing the restrictive state rather than destabilizing the *ram* state.

We propose that *ram* mutations in S4 and S5 preferentially destabilize the restrictive state, shifting the balance towards the *ram* state. The observation that *ram* mutations increase the affinity of ribosomes for streptomycin³⁹ is consistent with this model. Our results provide a structural basis for the notion that a delicate balance exists between the *ram* and restrictive states for optimal translation, and also explains how disruption of this balance leads to the various phenotypes observed. A definitive test of this model must await an atomic-resolution structure of the restrictive form.

Paromomycin. Paromomycin is a member of the aminoglycoside family of antibiotics, and increases the error rate of the ribosome. This family is thought to reduce the dissociation rate of A-site tRNA from the ribosome²⁸, but recent experiments also indicate that it increases the initial binding affinity of tRNA⁷.

Paromomycin binds in the major groove of H44 (Fig. 6) in a location that is in agreement with mutagenesis and protection data²⁴,⁴⁰. Paromomycin ring IV contacts the backbone of both sides of H44, and ring III makes only weak contacts with the RNA. Ring II forms tight interactions with both bases and backbone of the RNA, and ring I inserts into the RNA helix and helps to flip out bases A1492 and A1493, which are not well ordered in the structure of the antibiotic-free 30S¹². Ring I mimics a nucleotide base, stacking against G1491 and hydrogen-bonding with A1408. In addition it forms a tight hydrogen-bond interaction with the phosphate backbone of A1493 which helps lock the flipped-out bases in place. The structural basis for resistance mutations has been described from the NMR structure of a complex of paromomycin with an RNA fragment corresponding to its binding site⁴¹, which is generally similar to our structure. The main differences are that ring IV is dynamically disordered in the NMR structure, and A1492 and A1493 are flipped out to a far greater extent in our structure (Fig. 6).

![Figure 5](https://i.imgur.com/yourimage.png)

**Figure 5** Interaction of streptomycin with the 30S ribosomal subunit. **a**, Difference Fourier maps showing the binding site of streptomycin. Mutations in ribosomal protein S12 that confer resistance are shown in red. **b**, Chemical structure of streptomycin, showing interactions of the various groups with specific residues of the ribosome. **c**, The streptomycin-binding site, showing its interaction with H27, the 530 loop (H18), H44 and ribosomal protein S12. **d**, A view of the 30S showing streptomycin in a space-filling model, and the surrounding RNA and protein elements.

Figure 6 Interaction of paromomycin with the 30S ribosomal subunit. **a**, Difference Fourier maps showing paromomycin in a pocket formed by the major groove of H44. The bases A1492 and A1493 are flipped out from the starting model (also seen in the difference density). **b**, Chemical diagram for paromomycin, showing interactions of the various groups with specific residues of the 30S. **c**, The paromomycin-binding site, with a model for the A-site codon and anticodon as described in Fig. 1. The bases A1492 and A1493 flipped out by paromomycin interact with the minor groove of the codon–anticodon duplex. **d**, A view of the 30S showing paromomycin in a space-filling model, and H44.

Consequently we do not see the base pair between A1408 and A1493 that was observed in the NMR structure.

Rings I and II of paromomycin are found in a number of other antibiotics including gentamycin. An NMR structure of gentamycin bound to the same fragment of H44 showed that these two rings interact with RNA in the same way as in paromomycin⁴¹. This indicates that many other aminoglycosides that bind to the decoding centre on H44 may induce errors in translation by the same mechanism as paromomycin.

With respect to the model of the A-site codon and tRNA ASL described above, the flipped-out bases point directly into the A-site and are positioned to interact with the minor groove of the codon–anticodon helix (Fig. 1b). In this modelling, there are rather strict steric constraints on the A-site anticodon and especially the codon, which is also covalently attached to both the P-site codon and downstream message. Thus, despite the uncertainties associated with modelling, it appears unlikely that the A1492–A1493 could interact with any portion of the codon–anticodon helix other than its minor groove. This model provides clues as to how paromomycin increases the affinity of the A-site for tRNA. It seems probable that in the absence of paromomycin some energy is required to flip out A1492 and A1493 so they can contact the tRNA, and presumably this energetic cost is compensated by the formation of favourable interactions with tRNA. By binding to H44, paromomycin forms a structure in which these bases are already flipped out, thus reducing the energetic cost of both cognate and non-cognate tRNA binding and increasing tRNA affinity for the A-site.

### Implications for decoding in normal translation

The universally conserved residues A1492 and A1493 are required for viability in *E. coli*⁴², and are footprinted by the A-site codon and tRNA¹⁷. Experimental evidence for hydrogen bonding between the N1 atoms of A1492 and A1493 and 2′ OH groups on the message⁴² as well as modelling efforts⁴²,⁴³ have implicated these bases in the decoding process, but it has not been clear how hydrogen bonding to mRNA alone could discriminate sufficiently between cognate and non-cognate tRNAs. Moreover, in the structure of the 70S with tRNA and mRNA¹⁴, these bases were considered too distant from the codon–anticodon helix to have a direct role in decoding. A flipped-out conformation for A1492 and A1493 implies a decoding model that can reconcile these conflicting views.

The crux of decoding is the discrimination of near-cognate from cognate tRNAs. In the 30S structure, with the A-site codon and anticodon modelled as described above, the flipped-out A1492 and A1493 lie in the minor groove of the codon–anticodon helix, and form a portion of the decoding surface (Fig. 7a). We propose that the A1492 and A1493 adenine bases hydrogen-bond simultaneously to 2′ OH groups on both sides of the codon–anticodon helix, in a structural motif that is seen elsewhere in the 30S structure (Fig. 7b, c). Two adenines can monitor a large portion of the minor groove of three consecutive base pairs. As the distance between the 2′ OH groups across the groove is a function of the base-pairing geometry, the simultaneous formation of hydrogen bonds to both strands of the codon–anticodon helix should be sensitive to distortions arising from mispairing. We propose that the remainder of the decoding surface is composed of conserved RNA residues from the 530 pseudoknot and/or H34, which are nearby (Fig. 7a) and may monitor the shape of the correct anticodon–codon helix and contribute additional bonds. Significantly, the pairs of adenines that recognize minor grooves elsewhere in the 30S structure often recruit other RNA residues (blue in Fig. 7b and c) to complete an interaction surface.

There are at least two possible ways that adenines can hydrogen bond simultaneously to 2′ OH groups on both strands of a regular helix. The docking may involve hydrogen bonding of either adenine N1, N6 and N3 (Fig. 7b), or of adenine N1, N6 and N7 (Fig. 7c).

Figure 7 Model for the role of A1492 and A1493 in decoding. **a**, Overview of the 30S RNA elements surrounding the A-site helix of ASL and codon (magenta and blue, respectively), modelled as in Fig. 1. The RNA elements of the decoding site are the 530 loop (H18, green), H34 (dark blue) and the 1400–1500 region of H44 (cyan). A1492 and A1493 are represented by ball-and-stick models in the flipped-out conformation. Some of the highly conserved residues in H18 and H34 (red) that may help A1492–A1493 recognize the codon–anticodon duplex are highlighted. **b**, **c**, Two slightly different models for the structural role of A1492–A1493 in decoding, both taken from elsewhere in the 30S structure. In both cases two adenines measure cross-strand 2′-OH–2′-OH distances for three successive base pairs. Blue residues are other residues that help complete coverage of the minor groove. In **b**, the orientation of the adenines leaves the N7 atom pointing out and the C2 position pointing into the minor groove, and in **c** the adenine orientation leaves the N7 atom pointing in and the C2 position pointing out.

The former mode would result in a steric clash if the adenine were mutated to a G and the latter would not. The distinction is important because experiments supporting a model in which A1492 and A1493 hydrogen-bond to the 2′ OH of the codon made use of adenine to guanine mutants, which seem to rule out the N1–N6–N3 mode<sup>42</sup>. Another argument favouring the N1–N6–N7 mode is that it allows formation of a sequence-independent hydrogen bond between the adenine N6 atom and the O2 or N3 hydrogen-bond acceptor in the codon–anticodon helix<sup>43</sup> (Fig. 7c). However, the relative orientations of our flipped-out A1492–A1493 residues and the A-site codon–anticodon duplex appears to be more consistent with the N1–N6–N3 mode. The third or ‘wobble’ position of the codon–anticodon helix could be monitored less stringently by the A1492–A1493 in this scheme, thus allowing for the greater freedom for base pairing at the wobble position. A conclusive test of the decoding model will require the determination of the structure of cognate tRNA and mRNA ligands bound to the A-site in the restrictive form of the ribosome.

### Conclusions

In this work, we have analysed the functional implications of the 30S structure reported in the accompanying paper<sup>12</sup>. We have identified molecular details of the interactions made by A-, P- and E-site codons and tRNA with the 30S. We have also determined the structure of the 30S in complex with three antibiotics that target different regions of the 30S ribosomal subunit and work in different ways. The detailed knowledge of their binding sites from the structure of their complex with the 30S subunit should help the design of novel drugs that target bacterial protein synthesis. It is striking that all three antibiotics bind at the functional centre, with streptomycin and paromomycin being particularly close to each other. Another common theme is that all three antibiotics work by altering a delicate balance between conformational states of the 30S as it goes through the process of translation. Each of them sheds light on a different but fundamental process during translation: spectinomycin on translocation, streptomycin on the switch between the ram and restrictive states, and paromomycin on decoding. Finally, the structure indicates a direct role for A1492 and A1493 in the decoding process.

### Methods

Purification of ribosomal subunits, followed by crystallization, cryoprotection and flashcooling were carried out as described<sup>12</sup>, except that crystallization was carried out in the presence of a 10-fold molar excess of each of a mixture of streptomycin, paromomycin and spectinomycin. Also, no cobalt hexammine was included in the cryoprotectant, and antibiotics were present throughout the transfers into cryoprotectant solutions.

Data were collected on beamline ID14-4 at the ESRF in Grenoble, and integrated and scaled using HKL-2000 (ref. 44). The refined 3 Å structure of the 30S with the cobalt ions removed<sup>12</sup> was used as a starting model for refinement using CNS<sup>45</sup>. Positional refinement using energy minimization was followed by grouped B-factor refinement. For cross-validation, 5% of the reflections were left out of the refinement, and care was taken to

ensure that these corresponded to the same 5% that were omitted for the refinement of the original 30S model.

Received 14 July; accepted 10 August 2000.

1. Green, R. & Noller, H. F. Ribosomes and translation. *Annu. Rev. Biochem.* **66**, 679–716 (1997).
2. Agrawal, R. K. *et al.* Direct visualization of A-, P-, and E-site transfer RNAs in the *Escherichia coli* ribosome. *Science* **271**, 1000–1002 (1996).
3. Stark, H. *et al.* Arrangement of tRNAs in pre- and posttranslocational ribosomes revealed by electron cryomicroscopy. *Cell* **88**, 19–28 (1997).
4. Kurland, C. G. Translational accuracy and the fitness of bacteria. *Annu. Rev. Genet.* **26**, 29–50 (1992).
5. Pape, T., Wintermeyer, W. & Rodnina, M. V. Complete kinetic mechanism of elongation factor Tu-dependent binding of aminoacyl-tRNA to the A site of the *E. coli* ribosome. *EMBO J.* **17**, 7490–7497 (1998).
6. Pape, T., Wintermeyer, W. & Rodnina, M. Induced fit in initial selection and proof-reading of aminoacyl-tRNA on the ribosome. *EMBO J.* **18**, 3800–3807 (1999).
7. Pape, T., Wintermeyer, W. & Rodnina, M. V. Conformational switch in the decoding region of 16S rRNA during aminoacyl-tRNA selection on the ribosome. *Nature Struct. Biol.* **7**, 104–107 (2000).
8. Gabashvili, I. S. *et al.* Major rearrangements in the 70S ribosomal 3D structure caused by a conformational switch in 16S ribosomal RNA. *EMBO J.* **18**, 6501–6507 (1999).
9. Nierhaus, K. H. The allosteric three-site model for the ribosomal elongation cycle: features and future. *Biochemistry* **29**, 4997–5008 (1990).
10. Allen, P. N. & Noller, H. F. Mutations in ribosomal proteins S4 and S12 influence the higher order structure of 16S ribosomal RNA. *J. Mol. Biol.* **208**, 457–468 (1989).
11. Lodmell, J. S. & Dahlberg, A. E. A conformational switch in *Escherichia coli* 16S ribosomal RNA during decoding of messenger RNA. *Science* **277**, 1262–1267 (1997).
12. Wimberly, B. T. *et al.* Structure of the 30S ribosomal subunit. *Nature* **407**, 327–339 (2000).
13. Mueller, F. & Brimacombe, R. A new model for the three-dimensional folding of *Escherichia coli* 16S ribosomal RNA. I. Fitting the RNA to a 3D electron microscopic map at 20 Å. *J. Mol. Biol.* **271**, 524–544 (1997).
14. Cate, J. H., Yusupov, M. M., Yusupova, G. Z., Earnest, T. N. & Noller, H. F. X-ray crystal structures of 70S ribosome functional complexes. *Science* **285**, 2095–2104 (1999).
15. Frank, J. & Agrawal, R. K. A ratchet-like inter-subunit reorganization of the ribosome during translocation. *Nature* **406**, 319–322 (2000).
16. Stark, H., Rodnina, M. V., Wieden, H. J., van Heel, M. & Wintermeyer, W. Large-scale movement of elongation factor G and extensive conformational change of the ribosome during translocation. *Cell* **100**, 301–309 (2000).
17. Moazed, D. & Noller, H. F. Binding of tRNA to the ribosomal A and P sites protects two distinct sets of nucleotides in 16S rRNA. *J. Mol. Biol.* **211**, 135–145 (1990).
18. Prince, J. B., Taylor, B. H., Thurlow, D. L., Ofengand, J. & Zimmermann, R. A. Covalent crosslinking of tRNA1Val to 16S RNA at the ribosomal P site: identification of crosslinked residues. *Proc. Natl Acad. Sci. USA* **79**, 5450–5454 (1982).
19. von Ahsen, U. & Noller, H. F. Identification of bases in 16S rRNA essential for tRNA binding at the 30S ribosomal P site. *Science* **267**, 234–237 (1995).
20. Mueller, F., Stark, H., van Heel, M., Rinke-Appel, J. & Brimacombe, R. A new model for the three-dimensional folding of *Escherichia coli* 16S ribosomal RNA. III. The topography of the functional centre. *J. Mol. Biol.* **271**, 566–587 (1997).
21. Garrett, R. A. (eds) *The Ribosome. Structure, Function, Antibiotics and Cellular Interactions* (ASM, Washington DC, 2000).
22. Kurland, C. G., Hughes, D., Ehrenberg, M. in *Limitations of Translational Accuracy* (eds Neidhardt, F. C. *et al.*) 979–1003 (ASM, Washington DC, 1996).
23. Bilgin, N., Richter, A. A., Ehrenberg, M., Dahlberg, A. E. & Kurland, C. G. Ribosomal RNA and protein mutants resistant to spectinomycin. *EMBO J.* **9**, 735–739 (1990).
24. Moazed, D. & Noller, H. F. Interaction of antibiotics with functional sites in 16S ribosomal RNA. *Nature* **327**, 389–394 (1987).
25. Brink, M. F., Brink, G., Verbeet, M. P. & de Boer, H. A. Spectinomycin interacts specifically with the residues G1064 and C1192 in 16S rRNA, thereby potentially freezing this molecule into an inactive conformation. *Nucleic Acids Res.* **22**, 325–331 (1994).
26. Wittmann-Liebold, B. & Greuer, B. The primary structure of protein S5 from the small subunit of the

*Escherichia coli* ribosome. *FEBS Lett.* **95**, 91–98 (1978).

27. Ruusala, T. & Kurland, C. G. Streptomycin preferentially perturbs ribosomal proofreading. *Mol. Gen. Genet.* **198**, 100–104 (1984).
28. Karimi, R. & Ehrenberg, M. Dissociation rate of cognate peptidyl-tRNA from the A-site of hyperaccurate and error-prone ribosomes. *Eur. J. Biochem.* **226**, 355–360 (1994).
29. Karimi, R. & Ehrenberg, M. Dissociation rates of peptidyl-tRNA from the P-site of *E. coli* ribosomes. *EMBO J.* **15**, 1149–1154 (1996).
30. Bilgin, N. & Ehrenberg, M. Mutations in 23 S ribosomal RNA perturb transfer RNA selection and can lead to streptomycin dependence. *J. Mol. Biol.* **235**, 813–824 (1994).
31. Gravel, M., Melancon, P. & Brakier-Gingras, L. Cross-linking of streptomycin to the 16S ribosomal RNA of *Escherichia coli*. *Biochemistry* **26**, 6227–6232 (1987).
32. Montandon, P. E., Wagner, R. & Stutz, E. *E. coli* ribosomes with a C912 to U base change in the 16S rRNA are streptomycin resistant. *EMBO J.* **5**, 3705–3708 (1986).
33. Pinard, R., Payant, C., Melancon, P. & Brakier-Gingras, L. The 5′ proximal helix of 16S rRNA is involved in the binding of streptomycin to the ribosome. *FASEB J.* **7**, 173–176 (1993).
34. Melancon, P., Lemieux, C. & Brakier-Gingras, L. A mutation in the 530 loop of *Escherichia coli* 16S ribosomal RNA causes resistance to streptomycin. *Nucleic Acids Res.* **16**, 9631–9639 (1988).
35. Montandon, P. E., Nicolas, P., Schurmann, P. & Stutz, E. Streptomycin-resistance of *Euglena gracilis* chloroplasts: identification of a point mutation in the 16S rRNA gene in an invariant position. *Nucleic Acids Res.* **13**, 4299–4310 (1985).
36. Leclerc, D., Melancon, P. & Brakier-Gingras, L. Mutations in the 915 region of *Escherichia coli* 16S ribosomal RNA reduce the binding of streptomycin to the ribosome. *Nucleic Acids Res.* **19**, 3973–3977 (1991).
37. Melancon, P., Boileau, G. & Brakier-Gingras, L. Cross-linking of streptomycin to the 30S subunit of *Escherichia coli* with phenyldiglyoxal. *Biochemistry* **23**, 6697–6703 (1984).
38. Powers, T. & Noller, H. F. A functional pseudoknot in 16S ribosomal RNA. *EMBO J.* **10**, 2203–2214 (1991).
39. Bock, A., Petzet, A. & Piepersberg, W. Ribosomal ambiguity (ram) mutations facilitate dihydrotreptomycin binding to ribosomes. *FEBS Lett.* **104**, 317–321 (1979).
40. Spahn, C. M. & Prescott, C. D. Throwing a spanner in the works: antibiotics and the translation apparatus. *J. Mol. Med.* **74**, 423–439 (1996).
41. Fourmy, D., Recht, M. I., Blanchard, S. C. & Puglisi, J. D. Structure of the A site of *Escherichia coli* 16S ribosomal RNA complexed with an aminoglycoside antibiotic. *Science* **274**, 1367–1371 (1996).
42. Yoshizawa, S., Fourmy, D. & Puglisi, J. D. Recognition of the codon–anticodon helix by ribosomal RNA. *Science* **285**, 1722–1725 (1999).
43. VanLoock, M. S., Easterwood, T. R. & Harvey, S. C. Major groove binding of the tRNA/mRNA complex to the 16 S ribosomal RNA decoding site. *J. Mol. Biol.* **285**, 2069–2078 (1999).
44. Otwinowski, Z. & Minor, W. in *Methods in Enzymology* (eds Carter, C. W. J. & Sweet, R. M.) 307–325 (Academic, New York, 1997).
45. Brünger, A. T. *et al.* Crystallography & NMR system: A new software suite for macromolecular structure determination. *Acta Crystallogr D* **54**, 905–921 (1998).
46. Carson, M. Ribbons 2.0. *J. Appl. Cryst.* **24**, 958–961 (1991).

Supplementary information is available on Nature’s World-Wide Web site (http://www.nature.com) or as paper copy from the London editorial office of *Nature*.

### Acknowledgements

This work was supported by the Medical Research Council (UK) and a grant from the US National Institutes of Health to V.R. and S.W.White. D.E.B. was supported by an EMBO long-term fellowship, and W.M.C. by an NIH predoctoral fellowship. We thank R. Ravelli for help with data collection, and M. Pacold for help and discussions with the modelling of antibiotic structures.

Correspondence and requests for materials should be addressed to V.R. (e-mail: ramak@mrc-lmb.cam.ac.uk) or B.T.W. (e-mail: brianw@mrc-lmb.cam.ac.uk). Coordinates for the antibiotic–30S complex have been deposited in the Protein Data Bank, accession code 1FJG.
