
Virology 279, 439–446 (2001)  
doi:10.1006/viro.2000.0727, available online at http://www.idealibrary.com on IDEAL®

# Combined Results from Solution Studies on Intact Influenza Virus M1 Protein and from a New Crystal Form of Its N-Terminal Domain Show That M1 Is an Elongated Monomer

Steffi Arzt,* Florence Baudin,† Annie Barge,‡ Peter Timmins,§ Wim P. Burmeister,*§ and Rob W. H. Ruigrok†

*European Synchrotron Radiation Facility and §Forschungszentrum Jülich, Avenue des Martyrs, B.P. 220, F-38043 Grenoble Cedex 09, France;  
†European Molecular Biology Laboratory, Grenoble Outstation, B.P. 156, 38042 Grenoble Cedex 09, France; ‡Institut Laue Langevin, B.P. 156, 38042 Grenoble Cedex 09, France

Received September 6, 2000; returned to author for revision October 11, 2000; accepted October 25, 2000

The amino-terminal domain of influenza A virus matrix protein (residues 1–164) was crystallized at pH 7 into a new crystal form in space group P1. This packing of the protein implies that M1(1–164) was monomeric in solution when it crystallized. Otherwise, the structure of the M1 fragment in the pH 7 crystals was the same as the monomers in crystals formed at pH 4 where crystal packing resulted in dimer formation [B. Sha and M. Luo, 1997, *Nature Struct. Biol.* 4, 239–244]. Analysis of intact M1 protein, the N-terminal domain, and the remaining C-terminal fragment (residues 165–252) in solution also showed that the N-terminal domain was monomeric with the same dimensions as determined from the crystal structure. Intact M1 protein was also monomeric but with an elongated shape due to the presence of the C-terminal part. Circular dichroism showed that the C-terminal part of M1 contained helical structure. A model for soluble M1 is presented, based on the assumption that the C-terminal domain is spherical, in which the N- and C-terminal domains are connected by a linker sequence which is available for proteolytic attack.

## INTRODUCTION

Influenza virus is a segmented negative strand RNA virus. Inside the viral membrane, a layer of matrix M1 protein forms the contact between the membrane with the glycoprotein cytoplasmic tails on the one side and the internal viral genome in the form of eight ribonucleoprotein particles on the other side. The M1 protein in virus is a 60-Å (1 Å = 0.1 nm)-long rod (Ruigrok et al., 2000) that touches the membrane with one of its ends. By negative stain electron microscopy, in end-on view, the M1 molecules are seen to make up a network of lines (Wrigley, 1979; Ruigrok et al., 1989). The rods in the lines have a spacing of about 40 Å and the lines themselves are also about 40 Å apart. Since this network does not seem to possess any internal symmetry, its building blocks are probably monomers of M1.

The crystal structure of an 18-kDa amino-terminal fragment of M1 (28 kDa) has been determined from protein isolated from virus at low pH (Sha and Luo, 1997a). Although the crystallization drops were set up with intact M1 protein, cleavage occurred during the time it took for crystals to grow and only the 18-kDa fragment (amino acids 1–164) was found to be present in the crystals that formed at pH 4.0. This fragment forms a dimer in the crystal (Sha and Luo, 1997a) and the authors state that both the N-terminal fragment and the intact M1 protein elute as dimers from a gel-exclusion column (Sha and Luo, 1997b). However, the interpretation of the structure of the M1 layer in virus from EM pictures described above would not agree with a dimer as building block. Here we have crystallized at neutral pH the same amino-terminal fragment of M1 produced in *E. coli*. The protein crystallized in space group P1 with two monomers with almost the same orientation in the asymmetric unit. The space group P1 implies that the protein was a monomer in solution when it crystallized. The structure of the monomers in the pH 7 crystal is identical to the structure of the monomers in the pH 4 crystal, apart from differences in the protein-protein interfaces involved in crystal packing. Further analysis of this fragment and intact recombinant M1 protein at pH 4 and 7 shows that M1 is a monomer in solution with an elongated shape, which explains its aberrant elution from gel-exclusion columns.

## RESULTS AND DISCUSSION

### Crystal structure of recombinant M1(1–164) at neutral pH

Our initial crystallization experiments were performed with intact recombinant M1 protein. The polypeptide species that finally crystallized was the same M1(1–164) fragment as Sha and Luo (1997a) had obtained (F. Baudin and C. Müller, unpublished results). This means that both

1 To whom correspondence and reprint requests should be addressed at: EMBL Grenoble Outstation, c/o ILL, 6 Rue Jules Horowitz, 38000 Grenoble, France. Fax: +33.4.76.20.71.99. E-mail: ruigrok@embl-grenoble.fr.

ARZT ET AL.

![Image 1](image1.png)

FIG. 1. Superimposed worm diagrams of the backbones of the pH 7 structure (green) and the pH 4 structure (blue). Arrows indicate the N- and C-termini as well as residues where the backbones deviate.

![Image 2](image2.png)

FIG. 2. Plot of the rms deviations in the Ca positions of the residues of the pH 7 structure (two molecules mol1 and mol2 in the asymmetric unit) and the pH 4 structure [dimer consisting of mol1(dimer) and mol2(dimer)].

in viral and in recombinant protein, the region around residue 164 is particularly sensitive to protease attack. We then performed crystallization experiments directly with the recombinant N-terminal fragment and obtained crystals in space group P1 at pH 7 (see Table 1). Two monomers were found in the asymmetric unit which are related to each other by a translation of approximately [0 $\frac{1}{2}$ $\frac{1}{2}$]. There are no symmetry axes relating the molecules and this implies that the molecule that crystallized was a monomer in solution. Figure 1 shows a ribbon diagram of the pH 7 monomer structure with arrowed areas of difference between this structure and a monomer of the pH 4 dimer structure. The rms deviation of all Ca-positions between the pH 4 and pH 7 structures is 1.36 Å. The most significant differences between the main chains can be observed in the areas of protein-protein contacts that are different in the two crystals; in particular the conformation is different in the area where dimer contacts occur in the pH 4 structure, involving residues 36, 86–94, and 135–136 (Fig. 2).

INFLUENZA VIRUS M1 IS AN ELONGATE MONOMER

A
Surface Potential -9.483 -4.741 0.000 4.678 9.357 >-<

K95
K98
K102
R101
R105
K104
R134
R72
R76
R77
K47
R49
K57
R78

B
Surface Potential -9.483 -4.741 0.000 4.678 9.357 >-<

FIG. 3. The surface potential of the monomer of the pH 7 structure. The positively charged residues are indicated in (A) and the negatively charged surface area on the opposite side (rotation by 180°) is shown in (B).

On one side of the molecule the NLS/naked RNA binding site (Ye et al., 1995; Elster et al., 1997; Watanabe et al., 1996) involving the positively charged side chains Lys95, Lys98, Arg101, Lys102, Lys104, and Arg105, form a continuous positive surface together with Arg134, Arg76, Arg77, Lys47, Arg49, Lys57, Arg78, and Arg72 (Fig. 3a).

Remarkably, the other side of M1 opposite this positive area is negatively charged (Fig. 3b). The presence of the NLS is needed for the binding of M1 to naked RNA (Watanabe et al., 1996; Elster et al., 1997). However, as we have remarked before (Ruigrok et al., 2000), it is unlikely that M1 inside the virus is bound to naked viral

TABLE 1  
Structure Determination and Refinement  

Data collection  
(ESRF, ID 14-EH1)  
Space group  
P1  
Cell dimensions  
a (Å)  
36.86  
b (Å)  
44.15  
c (Å)  
47.69  
α (°)  
77.10  
β (°)  
67.83  
γ (°)  
77.60  
Wavelength (Å)  
0.934  
Resolution (Å)  
2.3  
Unique reflections  
11585  
Redundancy  
3.8 (3.8)  
Completeness (%)  
97.1 (97.1)  
Rsym (%)  
6.7 (21.2)  
I/sig(I)  
5.3 (2.6)  
Bfac (Wilson)  
37.44  

Refinement (15-2.3 Å)  
Number of reflections for refinement  
11376  
Rfactor (%)  
24.1  
Rfree (%)  
31.24  
Number of protein atoms  
2420  

Note. High-resolution shell 2.3-2.39 Å in brackets  

RNA. The influenza virus RNA inside the virus particle is present in the form of ribonucleoprotein particles in which the negatively charged backbone of the RNA is bound to the positively charged viral nucleoprotein (Baudin et al., 1994; Klumpp et al., 1997). We have recently shown that M1 can bind to negatively charged liposomes but not to neutral liposomes and that this binding can be prevented by salt (Ruigrok et al., 2000). These results suggest that positive charges on M1 are involved in lipid binding. On the other hand, it is also possible that the presence of positive and negative charged surfaces on M1 are used for M1-M1 interactions in building up a lattice through salt bridges, needed for stabilizing the structure of the virus particle. In the pH 7 crystal, sheets are formed with opposite positive and negative charges, which are oriented approximately parallel to the crystallographic (0 1 -1) planes (Fig. 4). These sheets do indeed pack onto each other with their positive side facing the negative face of the neighbor sheet. It is possible that at pH 4 the negative face is less charged, resulting in different crystal packing. Up to now, we have not been able to confirm hydrophobic interactions of M1 protein inside the virus particle or of recombinant M1 reconstituted with liposomes (Ruigrok et al., 2000). Earlier work by Bucher et al. (1980) and Gregoriades and Frangione (1981), however, did suggest that some part of M1 was embedded in the hydrophobic interior of the viral membrane. To explain

the fact that the M1 molecule in the pH 4 crystal was totally hydrophilic on its outside, Sha and Luo (1997a) proposed that the M1 molecule had to open up and expose its hydrophobic core for membrane interaction. M1 protein can easily be solubilized from virus at low pH (Zhirnov, 1992). Since the pH inside the virus, where M1 interacts with the viral membrane, is probably neutral, it would have been theoretically possible that at pH 4 the M1 protein was in a soluble conformation and at pH 7 in a membrane-bound conformation. However, the pH 4 and 7 structures of the M1 monomers are identical and the hydrophobic core of the pH 7 structure described here is the same as the one in the pH 4 structure and totally solvent excluded.

Gel filtration of intact M1, M1(1-164), and M1(165-252)

We then studied both intact M1(1-252) and its N-terminal M1(1-164) and C-terminal M1(165-252) domains in solution. Although we present here only the results on the recombinant proteins, we have also studied M1 protein isolated from influenza A/PR/8/34 virus and found in all studies the same results as with recombinant M1(1-252) protein. The only difference was that a fraction of the viral protein contains zinc (Elster et al., 1994), whereas the recombinant protein did not contain any zinc, as measured by atomic absorption (Waku and Ruigrok, unpublished results).

The last step in the purification protocol for the three recombinant proteins is a gel-exclusion column (Superdex 75). This column was calibrated with a low molecular weight gel filtration kit from Pharmacia. M1(1-252) (Mw 28 kDa) eluted from that column with a retention time corresponding to that of a globular protein of 38 kDa, the N-terminal domain M1(1-164) (Mw 18 kDa) eluted as a protein of 18 kDa, and the C-terminal domain M1(165-252) (Mw 10 kDa for the viral sequence plus 2.5 kDa for the poly-His tail) eluted from the column corresponding to a globular protein of 20 kDa. These results suggest that the N-terminal fragment is monomeric in solution and that the intact protein is an elongated monomer, with the C-terminal domain contributing much more to the size of the intact protein than its molecular weight would lead to expect. The N-terminal domain behaves as a regular globular protein, in agreement with its shape determined from crystallography. The gel filtration experiments were done at pH 5 and at pH 7.5 with the same results. For the following analyses we used only the intact protein and the N-terminal domain since the C-terminal domain had a tendency to aggregate.

Small-angle neutron scattering on intact M1 and M1(1-164)

We measured the shape and the mass of the molecular species in solution by small-angle neutron

INFLUENZA VIRUS M1 IS AN ELONGATE MONOMER

TABLE 2  
Results from Small Angle Neutron Scattering on M1(1–252)  
and M1(1–164)

| pH 4 | pH 7 |
|------|------|
| $R_{g}$ | $29 \pm 3 \AA (2)$ | N.D. |
| $M_{w}$ | $24 \pm 1 \mathrm{kDa} (2)$ | N.D. |
| $R_{g}$ | $15 \pm 3 \AA (2)$ | $15 \pm 2 \AA (1)$ |
| $M_{w}$ | $12 \pm 1 \mathrm{kDa} (2)$ | $16 \mathrm{kDa} (1)$ |

Note. $R_{g}$: Radius of gyration; N.D.: not determined. The values given are the averages plus or minus the standard deviation of the population with the number of experiments in brackets. For the single measurement of M1(1–164) at pH 7, the error in the $R_{g}$ is the statistical error in the data points of the single measurement.

structure that is attached to the N-terminal domain with a linker.

Model for the shape of M1(1–252) that agrees with the SANS data

If M1(165–252) were to form a compact globular domain flexibly linked to the M1(1–164), we could calculate the distance between the centers of mass of these domains. The C-terminal part represents 10 kDa, which corresponds to $12,120 \AA^{3}$, assuming a partial specific volume of $0.73 \mathrm{ml} / \mathrm{mg}$. This volume could represent a sphere with a radius of $14.3 \AA$ with a radius of gyration of $11.0 \AA$. Using this value for $R_{g}$ of the C-terminal part, the known value of $14.7 \AA$ for the N-terminal part and the value for M1(1–252) of $29 \AA$ determined by SANS, we can calculate that the distance between the centers of mass is about $53 \AA$. (Using the parallel axis theorem: $R_{g}^{2}=f_{1} R_{1}^{2}+f_{2} R_{2}^{2}+f_{1} f_{2} L^{2}$; with $R_{g}$ the radius of gyration of the whole protein; $R_{1}$ the radius of gyration of domain 1; $R_{2}$ the radius of gyration of domain 2; $f_{1}$ the mass fraction of protein in domain 1; $f_{2}$ the mass fraction of the protein in domain 2; and $L$ the distance between the centers of mass of the two domains.) If the two domains were spherical, this would imply a molecule with an overall length of some $80 \AA$ and a linker between the two domains of some $20 \AA$. However, if the domains are somewhat elongated, which we know is the case for the N-terminal domain, then the length of the linker would be shorter. In either case, it is likely that the two domains are joined by a linker polypeptide, which is exposed to proteolytic cleavage. Finally, note that the error in the radius of gyration of M1(1–252) is significant. If the actual value were not $29$ but $26 \AA$, the distance between the centers of mass would only be $45 \AA$.

Residues 159–164 are invisible in both M1(1–164) structures and could be part of the linker sequence. This sequence is also part of the zinc binding sequence 148-Cys-Xaa2-Cys-Xaa7-His-Xaa2-His-162 (Wakefield and Brownlee, 1989; Elster et al., 1994). As mentioned above, the recombinant M1(1–252) does not contain zinc and neither of the crystal structures has shown the presence of the metal. It would be very likely that the 5–15% of viral M1 that does contain zinc has another conformation in this linker sequence and that the presence of zinc could have an influence on the apposition of the two domains. Finally, the $80 \AA$ for the model of M1(1–252) that corresponds to the SANS data are longer than the $60 \AA$ rods observed under the viral membrane by EM (Ruigrok et al., 2000). This could mean that the relative position of the two domains inside the viral particle is not the same as in isolated, membrane-free M1.

scattering (SANS, see Table 2). Measurement of intact M1 protein at pH 4 resulted in a molecular weight of 24 kDa, close to the actual value of 28 kDa and in a radius of gyration ($R_{g}$) of $29 \AA$, much larger than expected for a compact globular protein of this weight, which would be in the range of $16-18 \AA$. For the N-terminal domain we found molecular weights of 12 and 16 kDa at pH 4 and 7, respectively, and a value for $R_{g}$ of $15 \AA$ at both pHs, close to the value of $14.7 \AA$ calculated from the crystal structure. The weights determined for the N-terminal domain are a bit low, possibly due to an overestimation of the protein concentration. The determined molecular weight is directly proportional to the absolute protein concentration. However, the SANS data confirm the data from the gel-exclusion experiments that the N-terminal domain behaves in solution as a globular, monomeric structure with the dimensions of the monomer from the crystal structure. The intact M1 is also clearly monomeric but has an elongated shape, caused by the addition of the C-terminal 165–252 residues to the globular domain formed by the first 164 amino acids.

Circular dichroism on intact M1 and M1(1–164)

It is possible that the intact recombinant protein has a structured N-terminal domain with an unfolded C-terminal domain. Therefore, we measured the CD spectra for the intact protein and the N-terminal domain, both proteins at $0.25 \mathrm{mg} / \mathrm{ml}$ (see Fig. 5). The two spectra were almost identical. Analysis of the data converted into $\Delta \epsilon$ with the CDnn program gave a value of 68% $\alpha$-helix for M1(1–164) and 58% $\alpha$-helix for M1(1–252), with virtually no $\beta$-structure. From the crystal structure we know that the N-terminal domain is 70% $\alpha$-helical, close to the 68% that was measured by CD. If the C-terminal domain were totally unstructured, the intact M1 protein would contain 46% $\alpha$-helix. Since we measured 58%, this means that the C-terminal part is structured with 38% $\alpha$-helix (31 of the 88 residues) and either forms an elongated structure attached to the N-terminal domain or a more globular

Surface Potential -9.352 -4.676 0.000 4.999 9.998 >-<

FIG. 4. Monomers in the pH 7 crystal arranged in sheets with a positively charged side and an opposite negatively charged surface. The sheets lie approximately parallel to the (0 1 -1) crystallographic plane.

MATERIALS AND METHODS

M1 protein from virus

M1 protein from influenza viruses A/PR/8/34, grown in embryonated hen's eggs, was isolated as described in Elster et al. (1994).

Recombinant proteins

M1(1–252). The gene for M1 (A/PR/8/34 strain), as described in Elster et al. (1997), was subcloned into Pet16b (Novagen) to remove the sequence coding for the poly-His tail. The new plasmid was expressed in E. coli BL21(DE3) pLysS after induction with 0.5 mM IPTG (iso-

propyl thiogalactoside) at 30°C. The bacteria were opened up by sonication and M1 was purified by a two-step FPLC protocol using first an ion exchange column (SP Sepharose FF) and second a gel-exclusion column (Superdex 75, Pharmacia). During purification, the protein was kept at pH 5.7 in a 20 mM MES-KOH, 250 mM NaCl, 10 mM β-mercaptoethanol buffer.

M1(1–164). The N-terminal part of M1, amino acids 1–164, was subcloned from the Pet16b plasmid expressing full-length M1 by removing the sequence coding for amino acids 165–252 by PCR. Expression and purification was the same as described for full-length M1 protein except that the buffer used for the last step in purification

delta absorbance


![Graph](attachment:graph.png)

Wavelength in nm

FIG. 5. Circular dichroism spectra of M1(1–252) and M1(1–164), both at 0.25 mg/ml.

was 100 mM NaCl, 20 mM Tris-HCl pH 7.5, 10 mM β-mercaptoethanol.

M1(165–252). The C-terminal part of M1, amino acids 165–252, was subcloned into Pet16b from the original plasmid coding for His-tagged M1 described in Elster *et al.* (1997) by removing the sequence coding for amino acids 1–164 by PCR. This C-terminal construct still contains the N-terminal poly-His tail. Transformed *E. coli* (BL21/DE3/pLysS) was grown at 30°C and induced with 0.5 mM IPTG. After 4 h induction, the cells were harvested by centrifugation and resuspended in ice-cold buffer (5 mM imidazole, 500 mM NaCl, 20 mM Tris-HCl pH 7.9). The cells were sonicated and M1(165–252) was purified using a Ni-chelation resin according to the manufacturer (Novagen) and subsequently a gel-exclusion column (Superose 75) in 100 mM NaCl, 20 mM Tris-HCl pH 7.5, 10 mM β-mercaptoethanol.

### Determination of protein concentration

The protein concentration was determined by triplicate amino acid analyses of two independent M1 preparations of known optical density (at 280 nm). Samples were hydrolyzed for 24 h under reduced pressure at 110°C in constant-boiling HCl containing 1% (w/v) phenol. Analyses were performed on acid hydrolysates with a Beckman 7300 amino acid analyzer using ninhydrin for detection. This method yielded an ε<sub>0.1%</sub> value at 280 nm of 0.50 (±0.01) cm²/mg for M1(1–252) and 0.53 (±0.01) cm²/mg for M1(1–164).

### Crystallization, structure determination, and model refinement

M1(1–164) in 10 mM Tris-HCl pH 7.5, 100 mM NaCl, 10 mM mercaptoethanol was concentrated to 14 mg/ml. Crystals were grown in hanging drops by vapor diffusion. Drops were mixed at a ratio of 1:1 with 0.1 M Hepes pH 7.5, 5% v/v isopropanol, 6–10% PEG4000. Native data were collected at the ESRF ID14-EH1 with a MARCCD detector. Crystals were flash frozen in a 90-K nitrogen stream after 20% glycerol was added to the well solution. The crystals have the space group P1 (for cell parameters, see Table 1) and diffracted to a resolution of 2.3 Å. Data were processed using the CCP4 suite (CCP4, 1994). The structure was solved by molecular replacement with the program AMORE using one molecule of the dimer of the same fragment of the M1 protein (Sha and Luo, 1997a; PDB entry 1AA7) as a search model. The refinement was carried out with the program CNS (Brünger *et al.*, 1998) alternating with model building with O (Jones *et al.*, 1991). The amino acids 2–71 and 75–158 could be fitted into the electron density map; no electron density was observed for the loop region with residues 72 to 74, so their occupancy was set to 0.0 in the refinement. No obvious continuation to residue 164 and the C-terminal domain is visible in the electron density map.

The Ramachandran plot shows 88% of the residues in the most favored regions.

### Small-angle neutron scattering

SANS was performed on the D22 instrument at the Institut Laue-Langevin (Grenoble, France) using a wavelength of 10 Å and a sample-detector distance of 4.0 or 5.0 m. Protein concentrations were between 2 and 5 mg/ml. The SANS method (for a review see Timmins and Zaccai, 1988) gives information on the shape and the mass of the molecular species in solution. The radius of gyration is derived from the Guinier approximation: \( I(Q) = I_0 \exp(-R_g^2 Q^2 / 3) \), where \( I_0 \) is the zero-angle scattered intensity, \( R_g \) the radius of gyration, and \( Q = 4\pi \sin\theta / \lambda \), with \( 2\theta \) being the scattering angle and \( \lambda \) the neutron wavelength. The radius of gyration can be calculated from the initial slope of the plot ln \( I(Q) \) vs \( Q^2 \). The mass was calculated from \( I_0 \) using the exact protein concentration (Jacrot and Zaccai, 1981).

### Circular dichroism

CD experiments were done on a JOBIN Yvon CD6 spectrophotometer, with a 1-mm cell. The spectra were collected from 190 to 260 nm with 1-nm steps and an integration time of 5 s/nm. Two spectra were done for each measurement and averaged. The proteins were in 5 mM MES-KOH pH 5.7, 150 mM KF at a concentration of 0.25 mg/ml. The measurements were corrected for buffer absorption and the resulting spectra expressed as ΔAbsorbance. The spectra were deconvoluted by CDnn software (Andrade *et al.*, 1993; Bohm *et al.*, 1992), available on http://bioinformatik.biochemtech.uni-halle.de/cdnn/index.html.

### ACKNOWLEDGMENTS

We thank Christoph Müller (EMBL, Grenoble) for generous help with the initial crystallization experiments and for discussions, Dominique Madern (IBS, Grenoble) for help with recording the CD spectra, and Nicolas Tarbouriech (ESRF, Grenoble) for help with the analysis of the CD spectra.

### REFERENCES

Andrade, M. A., Chacon, P., Merelo, J. J., and Moran, F. (1993). Evaluation of secondary structure of proteins from UV circular dichroism spectra using an unsupervised learning neural network. *Protein Eng.* **6**, 383–390.

Baudin, F., Bach, C., Cusack, S., and Ruigrok, R. W. H. (1994). Structure of influenza virus RNP. I. Influenza virus nucleoprotein melts secondary structure in panhandle RNA and exposes the bases to the solvent. *EMBO J.* **13**, 3158–3165.

Bohm, G., Muhr, R., and Jaenicke, R. (1992). Quantitative analysis of protein far UV circular dichroism spectra by neural networks. *Protein Eng.* **5**, 191–195.

Brünger, A. T., Adams, P. D., Clore, G. M., DeLano, W. L., Gros, P., Grosse-Kunstleve, R. W., Jiang, J.-S., Kuszewski, J., Nilges, N., Pannu, N. S., Read, R. J., Rice, L. M., Simonson, T., and Warren, G. L. (1998).

Crystallography and NMR system (CNS): A new software system for macromolecular structure determination. *Acta Cryst.* D54, 905–921.

Bucher, D. J., Kharitonkov, I. G., Zakomirdin, J. A., Grigoriev, V. B., Klimenko, S. M., and Davis, J. F. (1980). Incorporation of influenza virus M-protein into liposomes. *J. Virol.* 36, 586–590.

CCP4. (1994). The CCP4 suite: Programs for protein crystallography. *Acta Crystallogr.* Sect. D50, 760–763.

Elster, C., Fourest, E., Baudin, F., Larsen, K., Cusack, S., and Ruigrok, R. W. H. (1994). A small percentage of influenza virus M1 protein contains zinc but zinc does not influence in vitro M1-RNA interaction. *J. Gen. Virol.* 75, 37–42.

Elster, C., Larsen, K., Gagnon, J., Ruigrok, R. W. H., and Baudin, F. (1997). Influenza virus M1 protein binds to RNA through its nuclear localisation signal. *J. Gen. Virol.* 78, 1589–1596.

Gregoriades, A., and Frangione, B. (1981). Insertion of influenza M protein into the viral lipid bilayer and localization of site of insertion. *J. Virol.* 40, 323–328.

Jacrot, B., and Zaccai, G. (1981). Determination of molecular weight by neutron scattering. *Biopolymers* 20, 2413–2426.

Jones, T. A., Zhou, J. Y., Cowan, S. W., and Kjeldgard, M. (1991). Improved methods for building protein models in electron density maps and the location of errors in these models. *Acta Crystallogr.* Sect. A47, 110–119.

Klumpp, K., Ruigrok, R. W. H., and Baudin, F. (1997). Roles of the influenza virus polymerase and nucleoprotein in forming a functional RNP structure. *EMBO J.* 16, 1248–1257.

Ruigrok, R. W. H., Calder, L. J., and Wharton, S. A. (1989). Electron microscopy of the influenza virus submembranal structure. *Virology* 173, 311–316.

Ruigrok, R. W. H., Barge, A., Durrer, P., Brunner, J., Ma, K., and Whittaker, G. R. (2000). Membrane interaction of influenza virus M1 protein. *Virology* 267, 289–298.

Sha, B., and Luo, M. (1997a). Structure of a bifunctional membrane-RNA binding protein, influenza virus matrix protein M1. *Nature Struct. Biol.* 4, 239–244.

Sha, B., and Luo, M. (1997b). Crystallization and preliminary X-ray crystallographic studies of type A influenza virus matrix protein M1. *Acta Crystallogr.* Sect. D53, 458–460.

Timmins, P. A., and Zaccai, G. (1988). Low resolution structures of biological complexes studied by neutron scattering. *Eur. Biophys. J.* 15, 257–268.

Wakefield, L. and Brownlee, G. G. (1989). RNA-binding properties of influenza A virus matrix protein M1. *Nucl. Acid. Res.* 17, 8569–8580.

Watanabe, K., Handa, H., Mizumoto, K., and Nagata, K. (1996). Mechanism for inhibition of influenza virus RNA polymerase activity by matrix protein. *J. Virol.* 70, 241–247.

Wrigley, N. G. (1979). Electron microscopy of influenza virus. *Br. Med. Bull.* 35, 35–38.

Ye, Z., Robinson, D., and Wagner, R. R. (1995). Nucleus-targeting domain of the matrix protein (M1) of influenza virus. *J. Virol.* 69, 1964–1970.

Zhirnov, O. P. (1992). Isolation of matrix protein M1 from influenza viruses by acid-dependent extraction with nonionic detergent. *Virology* 186, 324–330.
