
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2001 by The American Society for Biochemistry and Molecular Biology, Inc.

**The Yeast ALG11 Gene Specifies Addition of the Terminal α1,2-Man  
to the Man₅GlcNAc₂-PP-dolichol N-Glycosylation Intermediate  
Formed on the Cytosolic Side of the Endoplasmic Reticulum*§**

Received for publication, December 4, 2000, and in revised form, February 13, 2001  
Published, JBC Papers in Press, February 15, 2001, DOI 10.1074/jbc.M010896200

John F. Cipollo‡§, Robert B. Trimble‡§, Jung Hee Chi¶, Qi Yan¶, and Neta Dean¶

From the ‡Department of Biomedical Sciences, State University of New York at Albany, Albany, New York 12201, the §Wadsworth Center, New York State Department of Health, Albany, New York 12201-0509, and the ¶Department of Biochemistry and Cell Biology, Institute for Cell and Developmental Biology, State University of New York at Stonybrook, Stonybrook, New York 11794-5215

The initial steps in N-linked glycosylation involve the synthesis of a lipid-linked core oligosaccharide followed by the transfer of the core glycan to nascent polypeptides in the endoplasmic reticulum (ER). Here, we describe *alg11*, a new yeast glycosylation mutant that is defective in the last step of the synthesis of the Man₅GlcNAc₂-PP-dolichol core oligosaccharide on the cytosolic face of the ER. A deletion of the *ALG11* gene leads to poor growth and temperature-sensitive lethality. In an *alg11* lesion, both Man₃GlcNAc₂-PP-dolichol and Man₄GlcNAc₂-PP-dolichol are translocated into the ER lumen as substrates for the Man-P-dolichol-dependent sugar transferases in this compartment. This leads to a unique family of oligosaccharide structures lacking one or both of the lower arm α1,2-linked Man residues. The former are elongated to mannan, whereas the latter are poor substrates for outerchain initiation by Ochlp (Nakayama, K.-I., Nakanishi-Shindo, Y., Tanaka, A., Haga-Toda, Y., and Jigami, Y. (1997) FEBS Lett. 412, 547–550) and accumulate largely as truncated biosynthetic end products. The *ALG11* gene is predicted to encode a 63.1-kDa membrane protein that by indirect immunofluorescence resides in the ER. The Alg11 protein is highly conserved, with homologs in fission yeast, worms, flies, and plants. In addition to these Alg11-related proteins, Alg11p is also similar to Alg2p, a protein that regulates the addition of the third mannose to the core oligosaccharide. All of these Alg11-related proteins share a 23-amino acid sequence that is found in over 60 proteins from bacteria to man whose function is in sugar metabolism, implicating this sequence as a potential sugar nucleotide binding motif.

Asparagine-linked (*N*-linked) glycosylation is an essential and highly conserved modification that is required for the function of glycoproteins in eukaryotic cells. Although *N*-linked oligosaccharides represent a diverse group of structures, all share a common biosynthetic pathway that begins with the synthesis of a core oligosaccharide consisting of Glc₃Man₉GlcNAc₂ (for reviews see Refs. 1 and 2). The assembly of the core oligosaccharide precursor occurs in two compartments. On the cytosolic face of the ER,¹ Man₅GlcNAc₂ is assembled stepwise from the nucleotide sugars UDP-GlcNAc and GDP-Man on to a lipid carrier, dolichylphosphate (3, 4). A “flipping” of the lipid-linked precursor translocates the oligosaccharide-lipid (OSL) into the lumen of the ER. Here, the next seven sugars (four Man and three Glc residues) are donated from Man-P-Dol and Glc-P-Dol, respectively, to form Glc₃Man₉GlcNAc₂. As a nascent polypeptide passes into the lumen of the ER, the preassembled oligosaccharide is transferred as a unit to selected asparagine residues in a reaction catalyzed by the enzyme oligosaccharyltransferase (OST). After its transfer, the oligosaccharide undergoes further processing in the ER by glucosidases I and II, and in *Saccharomyces* by an α1,2-mannosidase, Mns1p, that removes a single specific residue (5) as part of the editing mechanism that promotes the exit of correctly folded glycoproteins from the ER (6).

Mutants from the budding yeast, *Saccharomyces cerevisiae*, have been isolated that are defective in various steps involved in the synthesis of the Glc₃Man₉GlcNAc₂ core. Characterization of the *alg* (asparagine-linked glycosylation) mutants demonstrates that the early steps of core assembly are essential, whereas the later steps are not (7). Yeast with conditional mutations that interfere with the synthesis of the nucleotide sugar donors or the early biosynthetic steps that occur on the cytoplasmic face of the ER accumulate highly truncated core oligosaccharides and define essential genes, including *DPM1* (8), *SEC59* (9), *ALG1* (10, 11), *ALG2* (12), and *ALG4* (13). In contrast, mutations in genes that accumulate lipid-linked core oligosaccharides in which five or more mannose residues have been attached or that are defective in the later processing reactions, including *ALG3* (7, 14), *ALG5* (7, 15), *ALG6* (7, 16), *ALG8* (17), *ALG9* (18), and *ALG10* (19), show little or no growth phenotype. Truncated oligosaccharides are transferred by OST to proteins *in vivo* (11, 20) and *in vitro* (21–23), though the full-length, glucosylated Glc₃Man₉GlcNAc₂ is the preferred substrate (21, 24). Although the transfer of truncated oligosac-

* This work was supported in part by National Institutes of Health Grants GM48467 (to N. D.), GM33184 (to William J. Lennarz for the support of Q. Y.), and GM23900 (to R. B. T.) from the United States Public Health Service. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* § The on-line version of this article (available at http://www.jbc.org) contains supplemental material.
* The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) U12141.
* || To whom correspondence should be addressed. Tel.: 631-632-9309; Fax: 631-632-8575; E-mail: ndean@notes.cc.sunysb.edu.

---

¹ The abbreviations used are: ER, endoplasmic reticulum; OST, oligosaccharyltransferase; OSL(s), oligosaccharide-lipid(s); ORF(s), open reading frame(s); CPY, carboxypeptidase Y; PAGE, polyacrylamide gel electrophoresis; endo H, endoglycosidase H and endo-β-N-acetylglucosaminidase H (EC 3.2.1.96); Dol, dolichol; kb, kilobase pair; HA, hemagglutinin; TPI, triose phosphate isomerase; dansyl, 5-dimethylaminophthalene-1-sulfonyl; MALDI-TOF, matrix-assisted laser desorption ionization/time of flight; 1D, one-dimensional; 2D, two-dimensional; MS, mass spectrometry; LPS, lipopolysaccharide.

This paper is available on line at http://www.jbc.org

Role of ALG11 in Oligosaccharide-Lipid Synthesis

TABLE I  
Yeast strains used in the present study

| STRAIN    | GENOTYPE                                                                                   | SOURCE                          |
|-----------|---------------------------------------------------------------------------------------------|---------------------------------|
| SEY 6210  | MATα ura3-52 his3-Δ200 trp1-Δ901 lys2-801 suc2-Δ9 leu2-3, 112                                | S. Emr, UCSD                     |
| SEY 6211  | MATα ura3-52 his3-Δ200 trp1-Δ901 ade2-101 suc2-Δ9 leu2-3, 112                                | S. Emr, UCSD                     |
| JCY 19    | MATα ura3-52 his3-Δ200 trp1-Δ901 lys2-801 ADE2 suc2-Δ9 leu2-3, 112<br>alg11Δ::URA3             | This study                      |
|           | MATα ura3-52 his3-Δ200 trp1-Δ901 LYS2 ade2-101 suc2-Δ9 leu2-3, 112<br>ALG11                  |                                 |
| JCY 20-1c | MATα ura3-52 his3-Δ200 trp1-Δ901 lys2-801 suc2-Δ9 leu2-3, 112<br>alg11Δ::URA3                 | This study                      |
| JCY 20-1d | MATα ura3-52 his3-Δ200 trp1-Δ901 ade2-101 suc2-Δ9 leu2-3, 112<br>alg11Δ::URA3                 | This study                      |
| NDY 13.4  | MATα ade2-101 ura3-52 alg11-1                                                              | 27                              |
| NDY 41    | MATα ura3-52 his3-Δ200 trp1-Δ901 lys2-801 suc2-Δ9 leu2-3, 112 Δalg3::HIS3                    | This study                      |
| NDY 5     | MATα ura3-52 leu2-3, 112 vrg4-2                                                             | 35                              |
| NDY 17.4  | MATα ade2-101 ura3-52 ost4-2                                                               | 47                              |
| MCY 1094  | MATα ade2-101 ura3-52                                                                       | M. Carlson, Columbia University  |
| RSY 255   | MATα ura3-52 leu2-3, 112                                                                   | R. Schekman, UC Berkeley         |

charide structures is tolerated, mutations that block OST activity was subcloned into pRS316 (30) to generate a URA3, CEN-based plasmid, p6DB. This fragment contains the entire ALG11 gene and regulatory sequences, as well as an upstream ORF. The 5' and 3' ends of this fragment were sequenced by the dideoxy method (31) and were used to search the S. cerevisiae genome data base to obtain the entire nucleotide sequence corresponding to this SacI/ClaI fragment. This fragment contains two ORFs. Each of these ORFs was subcloned and tested for the ability to rescue the hygromycin B sensitivity of alg11 to identify the complementing activity. Searches of the data bases were made using the BLAST algorithm (32), and sequence alignments were made with the MegAlign program (DNASTAR) using the Clustal algorithm.

Plasmid Constructions—All DNA manipulations were carried out according to standard protocols (33). A 2.2-kb NheI/ClaI fragment was isolated from the plasmid p6DB. This fragment, which contains only the ALG11 ORF and 560 base pairs of upstream flanking sequence, was subcloned into pBR322 to produce pBRNhe/Cla. A 2.2-kb BamHI/ClaI fragment containing the ALG11 ORF and upstream regulatory sequences was subcloned into pRS316 (30) to produce pALG11-316 and into Bluescript SK− (Stratagene) to produce pALG11-SK−.

The deletion plasmid, palg11Δ::URA3, was constructed by replacing an internal 842-base pair EcoRV fragment within the ALG11 gene with a 2-kb SmaI/SalI fragment containing the URA3 gene. Replacement of the EcoRV fragment results in removal of approximately half of the ORF, after the first 95 amino acids.

The integrative plasmid, pALG11-306, was constructed by ligating the 2.2-kb BamHI/ClaI fragment from pALG11-316 into pRS306 (30). The plasmid was linearized at a unique EcoRI site in the ALG11 gene for integration at the ALG11 locus. The ALG11 overexpression plasmid, pYEp24-ALG11, was constructed by ligating the 2.2-kb BamHI/SalI fragment from pAlg11-316 into the URA3, 2μ plasmid, YEp24.

An epitope-tagged version of ALG11, pALG11-HA, was constructed by using polymerase chain reaction to introduce a HindIII site at the 5' end and an NsiI site at the 3'end of the gene that replaces the stop codon of the ALG11 ORF with a cysteine codon. The 1.6-kb HindIII/NsiI fragment containing the ALG11 ORF was cloned into pSK−P/X HA3 (34), resulting in the in-frame fusion of ALG11 to sequences encoding three copies of the HA epitope at the C terminus, in the PstI/XbaI site of pBlueScript SK− (Stratagene). For its expression in yeast, the HA-tagged ALG11 gene, on a HindIII/SacI fragment, was cloned into the HindIII/SacI site of pRS425-TPI. pRS425-TPI was constructed by ligating a 1.2-kb SalI/HindIII fragment that contains the TPI promoter into the SalI/HindIII site of pRS425 (30). This places ALG11 under the control of the strong, constitutive TPI promoter in a 2μ, LEU2 yeast shuttle vector.

Western Immunoblotting and Immunofluorescence—Overnight yeast cultures were diluted to 10^7 cells/ml and grown for 3 to 4 h prior to protein extraction. Whole cell protein extracts were prepared as described (35). For removal of N-linked oligosaccharides, extracts were resuspended in 0.1% SDS, 1% 2-mercaptoethanol, 50 mM sodium acetate and boiled for 3 min. After the addition of phenylmethylsulfonyl fluoride to 0.5 mM, 1 milliunit of endoglycosidase H (endo H; New England Biolabs) was added, and samples were incubated for 60 min at 37°C. Proteins were separated by 8% SDS-PAGE and immunoblotted as described (35). Anti-CPY antibodies were used at a 1:3000 dilution. Culture supernatants containing the monoclonal anti-HA antibody, 12CA5, were used at a 1:10 dilution. Anti-chitinase antibodies were

Although considerable progress has been made in the isolation of genes encoding proteins that mediate some of these stepwise reactions, the regulation of glycosylation in the ER is still not fully understood. This report describes the characterization of ALG11, a yeast gene that is required for normal glycosylation and is essential for growth at high temperatures. The alg11 mutant was originally recovered based on its resistance to sodium vanadate, a drug that enriches for glycosylation mutants (26, 27). Oligosaccharide structural studies define the alg11 defect to be the final step in Man₅GlcNAc₂-PP-Dol synthesis on the cytosolic face of the ER, specifically the addition of the terminal α1,2-linked Man to Man₄GlcNAc₂-PP-Dol. As a consequence of the elongation of Man₃,₄GlcNAc₂-PP-Dol in the ER lumen by subsequent Man-P-Dol-dependent transferases, Man₇GlcNAc₂-PP-Dol accumulates. Though the size of this oligosaccharide implies that it is the addition of the eighth mannose that is blocked in alg11 strains, other alg mutants that accumulate more highly truncated oligosaccharides display no growth phenotype. Thus, the temperature-sensitive lethality of alg11, as well as genetic epistasis analyses reported here, are fully consistent with a role for ALG11 during the earliest steps of N-linked glycosylation.

EXPERIMENTAL PROCEDURES

Yeast Strains, Media, and Genetic Methods—Yeast strains were grown in either YPAD (1% yeast extract, 2% peptone, 2% dextrose, 50 mg/liter of adenine sulfate) or synthetic medium that contained 0.67% yeast nitrogen base and 2% glucose, supplemented with the appropriate auxotrophic requirements (28). YPAD liquid medium was supplemented with 0.5 M KCl for the growth of alg11 mutant strains, which are osmotically sensitive. Hygromycin B (Roche Molecular Biochemicals) was added to YPAD agar after autoclaving to a final concentration of 30 μg/ml.

All yeast strains used in this study are listed in Table I. NDY 13.4, which carries the alg11-1 allele, was isolated as a spontaneous vanadate-resistant mutant on YPAD plates containing 7.5 mM sodium orthovanadate (Fisher) (27). NDY 41, containing a deletion of the ALG3 gene, was constructed by replacing the ALG3 gene in SEY 6210, using the linearized disruption plasmid pYBL0720::HIS3 (kindly provided by M. Aebi and S. te Heesen) as described (14). Yeast strains were transformed using the lithium acetate procedure (29).

Isolation of the ALG11 Gene—Strain NDY 13.4 was transformed with a yeast genomic CEN-based library in YCp50, carrying the URA3 selectable marker. Prototrophic transformants were selected on media lacking uracil. These transformants were replica plated on media containing 30 μg/ml of hygromycin B. Plasmid DNA from hygromycin B-resistant colonies was isolated, amplified in Escherichia coli, and retransformed into the alg11-1 mutant to confirm the hygromycin B resistance.

21830

**Role of ALG11 in Oligosaccharide-Lipid Synthesis**

used at a 1:1000 dilution. Secondary anti-rabbit or anti-mouse antibodies, conjugated to horseradish peroxidase (Amersham Pharmacia Biotech), were used at a 1:3000 dilution and detected by chemiluminescence (ECL; Amersham Pharmacia Biotech) followed by autoradiography. Indirect immunofluorescence of yeast cells expressing Alg11-HAp and data analyses were as described (36).

**Isolation of Lipid-linked and Whole Cell Pellet Oligosaccharides—** Mannosylated and glucosylated *in vivo alg11Δ* cells were grown overnight in YPD + 0.5 M KCl to a density of \(4 \times 10^7\) cells/ml and collected by centrifugation for 5 min at 3000 rpm and room temperature. The yeast were washed twice in YP + 0.1% glucose medium by centrifugation, and cells (\(1 \times 10^9\)) were resuspended in 200 μl of YP + 0.1% glucose containing 250 μCi of either [2-³H]Glc (15 Ci/mmol) or [2-³H]Man (20 Ci/mmol) purchased from American Radiolabeled Chemicals, St. Louis, MO. The labeling and preparation of OSLs were performed as described previously (11) with the following modifications. Pulse labeling was for 2 min, and steady-state labeling was for 15 min. Labeling was terminated by addition of 4 ml of CHCl₃/CH₃OH (3:2). To isolate N-linked oligosaccharides, whole cell pellets were solubilized in 1% SDS in 50 mM sodium phosphate buffer (pH 8.5) with heating, followed by pH adjustment to 5.5 with 1.0 M phosphoric acid, and endo H was then added to 50 milliunits/ml. The reactions were incubated at 37°C for 16 h, and endo H activity was verified by hydrolysis of Man₆GlcNAc₄-Asn-dansyl, followed by paper chromatography of the released GlcNAc-Asn-dansyl moiety. The released glycans were solvent-precipitated in 80% acetone as described previously (20).

**Chromatography—** Oligosaccharides, released from *alg11Δ* OSL or glycoproteins (see below), were sized on a calibrated Bio-Gel P-4 column (1.6 × 96-cm) eluted at room temperature with 0.1 N CH₃COOH/1% n-butanol (20). Fractions of 0.74 ml were collected and scanned for radioactivity by scintillation counting or for neutral hexose by a modification (5) of the phenol sulfuric acid assay (37).

**Isolation of Oligosaccharides from *alg11Δ* Cells for Structural Studies—** Two separate 200-g batches of logarithmic phase cells were suspended in two volumes of 50 mM sodium citrate, pH 5.5, broken with 0.5-mm glass beads in a Bead Beater (Bio-Spec Industries, Bartlesville, OK), and a crude cell extract was prepared as described (20). Solid ammonium sulfate was added to the extract to 50% saturation, stirred for 30 min at 4°C after the salt was in solution, and centrifuged at 30,000 × g for 15 min at 4°C. The supernatant fraction (400-ml) was dialyzed over 72 h at 4°C against three 6-liter changes of 5 mM sodium citrate, pH 5.5, and concentrated with an Amicon spiral cartridge concentrator to 80 ml. After lyophilization, the sample was taken up in 15 ml of dH₂O, and protein was determined by the Bearden assay (38). Glycoproteins were denatured with a 1.2-fold weight excess of SDS (39), and oligosaccharides were hydrolyzed with endo H (40) at 50 milliunits/ml overnight at 30°C. Protein and glycans were precipitated at –20°C by the addition of four volumes of cold acetone, and the oligosaccharides were recovered from the pellet by extraction with 60% aqueous methanol as described (39). Methanol was removed by rotary evaporation under reduced pressure, and the oligosaccharides were taken up in 1.5 ml of 0.1 N CH₃COOH and initially chromatographed on a preparative Bio-Gel P-4 column (2.6 × 67-cm) with a typical profile shown in Fig. 7. Fractions (1.35-ml) corresponding to Hex₅,₇,₉GlcNAc (odds) and Hex₆,₈GlcNAc (evens) from the two preparative columns were pooled separately, and each was rechromatographed into isolated glycan sizes on the calibrated 1.6 × 96-cm Bio-Gel P-4 column described above (not shown). The central 85% of the phenol-sulfuric acid assay color for each peak was collected as pools I–V corresponding to Hex₅–₉GlcNAc, respectively, and the glycan eluting in the column void-volume was collected as pool VI representing “mannan.”

**Mass Spectrometry—** MALDI-TOF mass spectrometry was performed on a Bruker reflex instrument. Samples of 25 to 50 pmol were crystallized with 2,5-dihydrobenzoic acid as the matrix. Data from 10 to 50 3-ns pulses of the 337-nm laser were averaged for each sample. Analyses were performed in linear and reflective mode.

**Methylation Linkage Analysis—** Samples were analyzed by methylation as described (41) using the NaOH/Me₂SO method. Briefly, the free hydroxyls of the oligosaccharides were deprotonated with NaOH/Me₂SO and then CH₃I was added to replace the free hydroxyls with methoxy groups. The methylated oligosaccharide was hydrolyzed in strong acid, evaporated under low pressure, and applied to Whatman silica gel 60A TLC plates. The plates were developed twice with MeCN-CHCl₃-MeOH, 3:9:1 (v/v/v), thoroughly dried between each ascent, and rapidly dipped into a solution containing 3 g of *N*-(naphthyl)ethylenediamine and 50 ml of concentrated H₂SO₄ in 1 liter of CH₃OH. The plates were dried and placed in an oven for 10 min at 120°C. All saccharide standards were purchased from Sigma.

¹H NMR Spectroscopy—Oligosaccharides (0.15–1.0 pmol) were exchanged three times by rotary evaporation from 99.8% D₂O and twice by lyophilization from 99.96% D₂O. Lyophilized samples were dried over P₂O₅ *in vacuo* for a day or more and then reconstituted in 0.5 ml of 99.996% D₂O (containing 1.35 mM acetone as an internal chemical shift reference) to a final concentration of 0.3–2 mM. Acetone protons were set at 2.225 and 2.217 ppm for spectra taken at 300 and 318 K, respectively. Samples were quickly transferred to 5-mm tubes (catalog number 535pp; Wilmad Co., previously washed and exchanged with 99.8% D₂O), flame-sealed, and examined at 300 and 318 K by 1D and 2D DQF-COSY phase-sensitive ¹H NMR spectroscopy at 500 MHz as described previously (42–44). Spectral width in the 11.7-tesla field was 1502 Hz for all experiments. For acquisition of 1D data, 1024 scans were collected over 4096 data points. The limit of resolution was 0.0045 ppm based on the ratio of the width of the widest peak at half-height (2.26 Hz) over the number of Hz per ppm (500.13 Hz/ppm). For homonuclear 2D DQF-COSY, 1.5 s of low power presaturation on residual HDO at 4.79 ppm was applied in the 300 K experiments. Data collection for the 2D experiments was 4096 data points in *t*₂ and 512 complex data points in the indirect *t*₁ dimension. The 2D relayed ROESY experiments were conducted at 318 K as described by Cipollo *et al.* (45).

---

**RESULTS**

**The *alg11* Mutation Affects the Synthesis of N-Linked Glycoproteins—** Resistance to sodium vanadate has been described previously as a way to isolate yeast glycosylation mutants (26, 35), and using this selection, a new mutant, *alg11*, was identified that affects an early step in glycosylation. The glycosylation state of the vacuolar proteinase, CPY, which undergoes a series of modifications as it transits the secretory pathway, is a useful probe to assess normal *versus* aberrant glycosylation (17–19). The 67-kDa ER form of CPY contains four core N-linked glycans, the 69-kDa Golgi form is further modified by the addition of sugars, and in the vacuole CPY is proteolytically processed to the mature 61-kDa form. Mutations that affect the assembly or transfer of oligosaccharides to CPY result in underglycosylation, which is readily observed as an increased mobility in which forms with 4, 3, 2, 1, or 0 *N*-glycans form a ladder of bands on SDS-PAGE (46). Lysates were prepared from *alg11–1* and wild-type cells, separated by SDS-PAGE, and assayed by immunoblotting using anti-CPY antiserum.

The *alg11–1* mutant accumulated CPY as a series of intermediates that migrated with an increased mobility compared with that of the mature wild-type form (Fig. 1A). The mobility of CPY in *alg11–1* was compared with that in the *ost4–2* mutant. *OST4* encodes a component OST (47) and when defective, reduces the transfer of N-linked oligosaccharides to CPY in the ER (47). Both *alg11–1* and *ost4–2* reduce the N-glycosylation of CPY, but the defect was more restrictive in *alg11–1*, as a higher proportion of forms with only one or two glycans was apparent (Fig. 1A, compare lanes 2 and 3). *ALG11* and *alg11–1* CPY treated with a low concentration of endo H resulted in identical deglycosylated products (Fig. 1B). This result confirms that the increased mobility of CPY in *alg11–1* extracts is because of underglycosylation and not proteolysis and that the majority of the oligosaccharides of CPY have the middle arm α1,3-linked Man added to the Man₅GlcNAc₂-PP-Dol core OSL by Alg3p in the ER lumen, a requirement for endo H sensitivity (48).

In addition to N-linked glycosylation, yeast also initiate O-linked glycosylation in the ER (reviewed in Ref. 49). To assay whether the *alg11* mutation affected O-linked carbohydrates, the glycosylation state of chitinase was examined. Because chitinase contains carbohydrates that are exclusively O-linked to Ser or Thr residues, an effect on O-linked glycosylation can be detected by an increase in its mobility by SDS-PAGE (50). A comparison of wild-type and *alg11–1* chitinase, assayed by immunoblotting cell extracts separated by SDS-PAGE with anti-chitinase antiserum, is shown in Fig. 1C, lanes 1 and 2. As a control, these chitinases were compared with that from *urg4–2* cells, a mutant affected in the luminal transport of

Role of ALG11 in Oligosaccharide-Lipid Synthesis

A.
![Image](image1.png)

B.
![Image](image2.png)

C.
![Image](image3.png)

FIG. 1. The *alg11* mutation affects the synthesis of N-linked glycoproteins. A, Western immunoblot analysis of carboxypeptidase Y in *alg11* mutant and wild-type cells. Proteins were extracted from the isogenic parental cells *ALG11* (MCY1094) (lane 1), *alg11-1* (NDY13.4) (lane 2), or *ost4-2* (NDY17.4) (lane 3), separated by 8% SDS-PAGE, and subjected to immunoblot analysis using anti-CPY antibodies as described under “Experimental Procedures.” The arrow denotes the mobility of mature CPY. B, protein extracts were prepared from *ALG11* (MCY1094) (lanes 1 and 2) or *alg11-1* (NDY13.4) (lanes 3 and 4) yeast strains and incubated in the presence (lanes 2 and 4) or absence (lanes 1 and 3) of endo H as described under “Experimental Procedures.” Proteins were separated by SDS-PAGE and subjected to immunoblot analysis with anti-CPY antibodies as in Panel A. The arrows denote the position of mature CPY (*mCPY*) or deglycosylated CPY (*dgCPY*). C, immunoblot analysis of chitinase in *alg11* mutant and wild-type cells. Proteins were extracted from the supernatants of *ALG11* (MCY1094) (lane 1), *alg11-1* (NDY13.4) (lane 2), or *vrg4-2* (NDY5) (lane 3) cultures, separated by 8% SDS-PAGE, and subjected to immunoblot analysis using anti-chitinase antisera as described under “Experimental Procedures.” The arrow denotes the mobility of chitinase.

GDP-mannose into the Golgi and therefore defective in O-glycan elongation (35, 36). Although an increase in chitinase was detected in the *vrg4-2* mutant (Fig. 1C, lane 3), no such difference was observed between *ALG11* and *alg11-1* chitinase (Fig. 1C, compare lanes 1 and 2). These results suggest that *alg11* does not affect O-linked glycosylation.

In addition to O-linked sugar addition, no effect of the *alg11* mutation was observed on the addition of glycosylphosphatidylinositol anchors, the steady-state level of Man-P-Dol, or in vitro OST activity (data not shown). Thus, these results suggest that *ALG11* is specific for a step in the N-linked glycosylation pathway.

Isolation of the *ALG11* Gene—The *alg11* mutant phenotype of severe under N-glycosylation of CPY, but with endo H-sensitive glycans, suggested the *ALG11* locus might encode one of the few OSL biosynthetic steps remaining to be identified.

A.
![Image](image4.png)

B.
![Image](image5.png)

FIG. 2. The cloned *ALG11* gene rescues both the glycosylation defect and the hygromycin B sensitivity of *alg11-1*. A, immunoblot analysis of CPY, using extracts prepared from wild-type (SEY6210), *alg11-1* (NDY13.4), or *alg11-1* cells harboring a plasmid bearing the wild-type *ALG11* gene (NDY13.4 + pALG11–316). Mature CPY (*mCPY*) is denoted by the arrow. B, strains MCY1094 (*ALG11*), NDY13.4 (*alg11-1*), and NDY13.4 harboring pALG11–316 were streaked on YPAD or YPAD plates containing hygromycin B at 30 μg/ml and incubated at 30 °C for 18 h.

fied. Like many other yeast glycosylation mutants, *alg11* growth is completely inhibited on medium containing hygromycin B, a condition that does not restrict the growth of wild-type cells (see Ref. 27 and Fig. 2). The glycosylation defect and hygromycin B sensitivity co-segregated through two sequential out crosses, and the mutation behaved as a single recessive allele (data not shown). Therefore, we cloned the wild-type *ALG11* gene by complementation of the hygromycin B sensitivity (see “Experimental Procedures”). The *alg11* complementing activity was localized to YNL048w, an ORF of previously unknown function, which we designate *ALG11* (for asparagine-linked glycosylation). A plasmid containing only this ORF expressed in the *alg11-1* mutant rescued the growth defect (data not shown), the glycosylation defect as assayed by CPY immunoblots (Fig. 2A), and the hygromycin B sensitivity (Fig. 2B).

To confirm that the cloned fragment contained the *ALG11* locus, rather than an extragenic suppressor, a fragment containing *ALG11* was cloned into an integrative plasmid that carries *URA3* as a selectable marker. The plasmid was linearized at a unique site within the putative *ALG11* gene to allow homologous recombination at the *ALG11* locus in an *ALG11 ura3-52* strain. Ura<sup>+</sup> transformants were crossed to an *alg11-1 ura3-52* strain to produce diploids that were sporulated and dissected. A 2:2 segregation pattern was observed for hygromycin-sensitive: hygromycin-resistant colonies, and all of the hygromycin-resistant spores were Ura<sup>+</sup> in the 25 tetrads analyzed (data not shown). These results confirm that the cloned DNA is very tightly linked to the *alg11* mutation and suggest that this DNA fragment does indeed contain the *ALG11* gene.

Alg11p Is Highly Conserved among Eukaryotes, Is Related to Alg2p, and Contains a Conserved Motif Found in Many Sugar Binding Enzymes—The nucleotide and predicted protein sequence of *ALG11* is deposited in GenBank™ (accession num-

21832

**Role of ALG11 in Oligosaccharide-Lipid Synthesis**

A.

ScAlg11p 48 SLNPFSDKSSLLNRRAVASC--GEKNVKVFGEFFHPYCNAGGGGEKVLWKAVDITLRKDAK-
S. pombe 31 RLKKIAPVKASLYKQVGVEPKLARTVGEFFHPYCNAGGGGERVLWTAIVKSVOTEFPN-
C. elegans 25 ALLIPFSLYSGFRRK-------SKTVAEFFHPYCNAGGGGERVLWAATRTMQKKFPD-
Drosophila1 23 LFLRQWLLGRKNKLHTS------SENGINVGIHHPYCNAGGGGERVLWCAIVALQEKEYQN-
Arabidopsis 22 FLS-----VINARKS--------RKRAVGFFHPYTNDGGGGERVLWCAVKAIQEENPD-
Leishmania 151 RMLTFTVVLVALALLCHCRKRAYARNTVGELHAAGAGGGGGERVLWVAALDGLQHADAAAR

ScAlg11p 105 ---NVIV[YSGDFVNGE-----NVT PEN[IL]NNVKAKEFDYDLDSDR-IF[E]IS[KLR]YLVDSS
S. pombe 88 ---VIS[VV]YTGDN-------VS[KAEF]ILRRVKNTFEIDLDSSK-IVFVY[LKR]FLVSAT
C. elegans 74 ---HKYFV[YSGDTD--------ATKEQ[ILL]LKARQRFGIELEDPNS-IQFIYLHWBTLVEAR
Drosophila1 77 ---ARMV[YTGDD----------ASPNS[LL]QKAKNVFNIAVDSDN-VKEVFLKQKRWHEAK
Arabidopsis 67 ---LDCVIFTGDHD--------SIJSDSLARRAVDRFGVHLQSP--PKVIHJNKRBKWIEES
Leishmania 211 GVKRQYVLFINEYKPADRILSAESSDQHLLSLVEKQFSLRLRPVRFIYLRRPALTRWLSGD

ScAlg11p 157 TWKHFTLLIGQAIS[EMI]LAFESEIIQCPRPDIWIDTMGYPF[SYPIIARFLRRIPIVTYTHY
S. pombe 135 TWHRFTLLLGQSLEG-SMIILGFEEAIYRFAPDIFIDTMGYAEFTFCVVKSFQN-IPVGAYYVHY
C. elegans 122 HYKHCTMLFQALA---GLIALLEAWFRMVPAVFIDSMGYPLSLPAFRLSG--SKVVAVYVHY
Drosophila1 125 NYPHFTLLLGSIG-SMVVGLEALECRFPPDIYIDTMGYAEFTYPLFYLAQ-SKVGCVVHY
Arabidopsis 114 TYPHFTMIGQSSLG----SIYYLAWEALRMFTPPLYFLDTSGYAEFTYPLARIFG--CKVCYTHY
Leishmania 271 AYPRLLTLLLLQTFWGGAAALFYEVAVANAVTPIVVEIVGVPEAYPLLRLLAG-CMWISYTHY

ScAlg11p 215 PIMSKDMLNKLFKMPKKGIKVYGKILY---------WKVFMLIYQSIGSKIDIVITNSS
S. pombe 192 PTISTDMCLKSLKQVSLLAKVKMAYWR--------WFAQLYSDAGSHADYVMTNSS
C. elegans 178 PTISCDMLDVVESRQETFNNSSSTIAQSNVLSWGKLTYYRLFACLYWLAGKAAHVG MVNGS
Drosophila1 182 PVISTDMJKRVQQRQMSHNNKKYVARNPFLTWTKLAYYRLFSRMYKWVGCCAETIMVNSS
Arabidopsis 170 PTISLDMISRVRQRNSMYNNDASIKA---------SWMYGMVGGSCTHLAMVNSS
Leishmania 330 PIVSSAMTQRVRSGEVSTNSPTVAWNPMILCAKVYYYGVFSLLYRCMGFFPNVVLTNSS

ScAlg11p 265 WTNNHIKQIWSQS-NTCKIIVPPCSSTEKLVD------------------WKQKFGTAKGER
S. pombe 239 WTRNHIASLWGKDIQLSVVPPPCNTS------------------ELEKIDINRKR
C. elegans 238 WTQRHITSIRSR-RDVSIYPPCDVEAFLN------------------ESVAESLLEDTK
Drosophila1 242 WTENHILQLWDVPFKTHRIVPPCEVS------------------HLKSLQHTEKG
Arabidopsis 215 WTKSHIEVLRIPERITRVYPPCDTSGLQVITFRQLESYDFLFAIDLGLLQAFPLERSSD
Leishmania 390 WTQNHVQSIFWP-RACIRLYPPCDVAG------------------FAAGSQPPALR

ScAlg11p 306 LNQAVLAAQFRPEKR-----HKLIIESFATFLKNLPDSVSPIKLIMAGGSTBRSKQDENYVKS
S. pombe 276 EPTLLYLAQYRPEKN-----HENVLRSFALYFEQHPDSP--AKLLLVGSVRQEEDMCFVNH
C. elegans 280 TVRLLSVGQIRPEKN-----HKLQLEVLDVKEPLEKMGNVELCIAGGCRNEEDQEREVKM
Drosophila1 280 EFIILLSVGQFRPEKD-HPLQLQAIELRTLLAQDEALWNQIKLIVIGSCRNEEDDYERLKN
Arabidopsis 275 PKKISVAQFRPEKVRDPAHMLQLEAFSLALEKLDAADVPRPKLQFVGSCCRNNSDEERLLQK
Leishmania 427 NNRIVSVGQFRPEKN-----HMLQLVAFHAAMPRLPRDDKLVMIGGARNADDRKBAEQ

ScAlg11p 362 LQDWSENVLKIPKHLISFEKNLPFDKIEILLNKSTFGVNAMWNNEHFGIAVVEYMASGLIP
S. pombe 330 LKTILATELN--LQSKVKFVVDAWPWKVVEYLGTCSIGVNYMWNEHFGIGVVEYMAAGLIP
C. elegans 336 LKNEAEKLD--ISEQLIWQLNVPIYEDLVVEJSKALISIHTMNNEHFGISVVEAMAASTII
Drosophila1 339 MQDLTKHLS--LENNVQFSVNVPYEDLLKYQTAHIGIHTMNNEHFGIGIVESMAAGLIM
Arabidopsis 335 LK德拉VELK--VDGDVQFYKNAMYRELELLLGNAVAGLHGMIDEHHFGISVVEYMAAGAIP
Leishmania 480 LHVRAKELG--IEEQVELLVNATVAEVQAEILGKCVIGLHTMRDEHHFGIVLLEYLAAGCIP

ScAlg11p 422 IVHASSAGPLLDDIVTPWDANGNIGKAPPQWEELQKKYFAKLEDDGETTTGFFFFKEPSDPDYN
S. pombe 388 VVNNSSGGPKFDIVIPWIGKPTGFHASSTISEYAAEAYHKALTLSPQEQLEMRINARSACARF
C. elegans 394 LSNDSSGGPRMDIVKDYEGHCVGYLSITKEEYVETILKIVEEGLKKRNDDTRKYARKSLTRF
Drosophila1 397 VAHKSSGGPLLDDIVETSAGSQNGFLATDAVEEYAEINILNIIVN-NSEMNGIRNAAR-----
Arabidopsis 393 IAHNSSAGPKMDIVLEEDGQKTGFLAETVEEYAAEAILEI VKMNETERLKMAESAARKAAARF
Leishmania 538 LGHRSSGGIVELDILNSPD---LGFLAVTAEEIYAAAMVEICEMRLRDPDRYVQFQKRGSE--

ScAlg11p 482 TKDPLRYPNLSDLFLQITKLDYDCLRVMG-
S. pombe 448 GEHVFMRDFG-NVFAKLLREDYSRT---
C. elegans 454 GEAAEEVNSLEQRNRKSAVNIFSFKLIIFS
Drosophila1 0 ------------------------------
Arabidopsis 453 SEQRFCED----EKTAIRPIFTGPLK---
Leishmania 593 HVKSFDDSSFTRREFELVSEYVYAC---

FIG. 3. ALG11 encodes a conserved protein. A, alignment of S. cerevisiae Alg11p-related proteins. Proteins were identified using the BLAST (32) program and aligned with DNASSTAR (MegAlign) using the Clustal algorithm. The GenBank™ accession number for each protein is as follows: S. cerevisiae Alg11p, U12141; S. pombe Alg11p, CAA20913 (51); L. major, AAF77213; D. melanogaster, AAF51756; C. elegans, P53993; A. thaliana, AAD25934. B, alignment of S. cerevisiae Alg11p with Alg2-related proteins from S. cerevisiae and C. elegans. The GenBank™ accession numbers for each of the proteins listed are as follows: S. cerevisiae Alg2p, Z72587; C. elegans Alg2p, U39649; S. cerevisiae Alg11p, U12141. C, proteins were identified using the BLAST program (32), in which the 23-amino acid sequence found to be conserved in Alg11p and Alg2p (EHFGIAVVVEAM-ACGTPVVAASGG) was used to search the data base, and proteins were aligned as in panel A (note that only 20 of more than 60 proteins containing this sequence are shown here). The conserved consensus is indicated above the alignment. GenBank™ accession numbers are as follows: Acetobacter xylinum mannosyltransferase, X94981; Anabaena HepB, U68035; Bacillus subtilis, L38424; Hemophilus influenzae lipopolysaccharide biosynthesis protein (LBP) Rd, U32842; Klebsiella pneumoniae, D21242; Methanococcus jannaschii LPS, U67601; Mycobacterium leprae LPS, U00018; Pisum sativum suc synthase, X98598; human PiGA, P37287; S. cerevisiae SPT14, Z73531x1; Salmonella choleraesuis mannosyltransferase, X61917; Synechocystis mannosyltransferase, D90901; Synechocystis LPS galactosyltransferase, D63999; Xanthomas campestris GumH, U22511; Y. enterocolitica trsE, Z47767.

All of the Alg11- and Alg2-related proteins share a sequence, beginning at position 404 of the S. cerevisiae Alg11p, that is particularly well conserved. When this sequence was used to search the data base, over 60 related proteins were identified, a subset of which is shown in Fig. 3C. These proteins, which include members from bacteria to man, perform functions relating to sugar metabolism. Included in this group of proteins are various glycosyltransferases, sucrose synthases, proteins that function in bacterial lipopolysaccharide biosynthesis, and proteins that function in glycosylphosphatidylinositol biosynthesis. This high degree of conservation implies that this sequence may represent a domain required for some aspect of carbohydrate binding or metabolism. Though speculative, these proteins all utilize nucleotide sugar substrates, suggesting that this motif may correspond to a nucleotide sugar recognition site.

Role of ALG11 in Oligosaccharide-Lipid Synthesis 21833

B.

|  | sc alg11 | sc alg2 | ce alg2 |
| --- | --- | --- | --- |
| 1 | MGSAWTNYN FEE-VKSHFGFKKYWS---SLVLVY---------GLIKVLIWFRQWVYSSLNPFSKKSSLLNRVASCGEKNVKV GFF---HPY | MI EKDKR--SIAFTHPDL---GIGG---AERLV----------DA--------ALGLO---------QSHSVIY-------TSH | MPSELK RYSVLSLYTCAL----FFSV---PLSLVIWFVRRELCTET-----------------SVAGE---------------------KQMHWIVHPEQWNGGSDR |
| 81 | CN---------------------------------AGGGGEKVLWKAJDITLRKOA KNVIV----------------------IYSGDFVNGENVDBENILNNVKAKFDYDLS- | OD-----------------------KS-----------HCREE--EVKNGQLKE-----------------------------VYG-------------------DELPT | GTVALIRHFVSQGHRVTWLTTMIDEYWKN-----------------HPBDVEIREVGLKLHPGDWWSQNVALGWHMVFS--------------------------NLNP |
| 140 ------------------------------------DRIFFTSIKLRYLVDSSTWKHF TLJGQATGSMILA FES---IIQCPPDIWDTMGYPFSMPII-ARFLR | 72 NFLGRFFIVFATIRQLYLVIQKLILQKKVNAYQT------------------RIDQLSTCIP L------------------------LH-I--FS | 131 ------------------------------DVA--------------------------LIDHSASCVPM--------------------------IKWR--FP |
| 205 RIPIVTYTHYPTMSKML-------------------NKLFKMPKKGIKVYCKILYNKFEM LIYQSIGSKIDIITTNSIWTNW H---IKQIWQSNF-CKIIYPPCSTE | 122 SATLMFYCHFPD---QLL-----------------AQRAGLLKK---------------IYRLPFDLIEQFSVSAADTVVVNSNFTKUFEHQTFKYL--SND-PDV IYPCVDLS | 151 QCKILFYCHFPQ---QLV----------------TPSRFFLYR-----------------WYAKLIGIVEELFGHIDQIFVNSNFATQFCKVMPNI--EKNKVRVYYPCCIDD |
| 290 KL--VDWKQKFGTAKGERLNQA------------IVLAQFRPEKRHKLTIESFA-TFLKNOPDSVSPIKLI----------MAGSTRS-----------------KQDENYV | 198 TTEIE---DIDKKFFKTVFNEGDRF-YLSINRF---EKKDVALAINGFAHSED--QINDNVKLVICGGYDERVAENVEYLKELQSLADEYELSHTTTY | 228 WI--VS----ASERPVSRAQRANKETYTFLSMNRFWEKRLDITEAAS-ILKOK--GYHFHVQLA----------GSVMPHI---------------------PBSRIYY |
| 360 KSLQDWSE--NVLKIPKHLISFEKNLPFDKI-EILLNKSTFGVANMNEHFGIAWVEYM ASGLIPIVHASAGPLLDDIVTPWDAN---------------------GNF | 289 QEIKR-VSDLSEFKANNSKII-FLTSISSSLKELLERTEMLLYT PAYEHFGIVPLEAMKLKPVLEAVNNCGGPL-ETIKSYVAG---------------------ENE | 299 EVLQR-MT--EELNVID-MVT-FIPSPIDKVFQOYYQCQDTALYTPPNEHFGTVPIEALDQRRPVICDSGGPA-ETVLEDITG---------------------TKI |
| 444 GK--------APP---QWEBQKY------------------FAKL--------------------------sc alg11 | 373 SS--------ATGWLK-PAVPIQ------------------WATAIDESRKILQNGSVN---ERNGPLRVKKY--------------------------sc alg2 | 380 AKFCGELLAEAMLH-HMNKRD------------------WPELDTDE------------------G---YAKQ-RHRL ETE--------------------------ce alg2 |
| 461 ----------------EDDGETIGF------------------FFKEPSDPDY------------------NTTK------------------LRYPNLS DL---sc alg11 | 417 ----------------FSREAMTQSF------------------BENJEK------------------VIWKEKKYYPWEI---F---GT---sc alg2 | 420 ----------------FSIRFCGNI------------------DRATAE------------------MM------------------GT---EI---ce alg2 |
| 495 ----FLQ-ITKLDYDCI-RVMGARN------------------QQYSLYKFS---DLKF D KD------------------WEN------------------F sc alg11 | 449 ----SFSNFILHIA--F-IKILPNWPWSYLWLLWYILRTTYGEFTGHLVSLSPTIM-KKY N------------------sc alg2 | 443 ----SSS-EPTTTTP--F-VDTIVHQ------------------PTATEVY SK---P-QQY N------------------ce alg2 |
| 534 VLN PIC------------------KLEE ER------------------G | 505 -----------------VYLNKQCIPPEEKIYIGEEISK------------------CK | 474 ------------------KAHSRRA------------------QA |

C.

|  | Consensus | Majority |
| --- | --- | --- |
| -E.FGL..VEAMA.G.PV.....GG--------- | WNEHFGIAVVEYMA3GLIPIVHASAGPLLDIVTPWD | S. cer Alg11 |
| --EGFGLVVVEAMACGLPVVAT--GG--------- | --EHFGISVVEAMAASTIILSNDSGGP | C. eleg Alg11 |
|  | --YEHFGIVPLEAMKLCKPVLA VNNGGPLETI | S. cer Alg2 |
|  | --NEHFGIVPIEALDORRPVTVCDSSGGPAE | C. eleg Alg2 |
|  | --EGFGLAAVEAMSAGLVEIL | A. xylinum ManTase |
|  | --FEGFLAITE SLACGTPVLC TPIG | Anabaena HepB |
|  | --ESFGLVLL EAMACGVPCIGTNIG | B. subtilis |
|  | --EGLPIVVEAMAFCLEIVAFNCSP | H. influenza LBP |
|  | --WESFGLVAJE AOLYCVPV I | K. pneumoniae |
|  | --YEPFGIVAL EAMARGIPVVSSVGGLMEII | M. jannaschii LPS |
|  | --ESFGLVAVEAQACGIPVWAAA VG | Myco leprae LPS |
|  | --YEA FGLIVVEAMAIIGLPIFEATLNGGPAEII | P. sativum suc synthase |
|  | --TEAF CMAIVEAASCLQVVSTRVGGIPEV | PIGA human |
|  | --TEAF GTILVEAASCNLLIVTTOVGGIPEV | S. cer SPT14 |
|  | --YE GFGLPVVEGMACGIPVITS | S. choleraesuis Mtase |
|  | LYEGFGLPVVEGMACGIPVL | S. typhimurium Mtase |
|  | --YEGFGLPVVEAMNFGGLPIITAGAGS----L | Synechocystis Mtase B |
|  | ----LKVEYMAAGLRIVA-SCI G | Synechocystis SPS LPS GTASE |
|  | --EGFGLAAVEAMSAGLIT PIL | X. campestris GumH |
|  | W-EGFPLVITEAMACKIIIVATD-AS | Y. enterocolitica trsE |

FIG. 3—continued

Disruption of the ALG11 Locus Results in Underglycosylation of Proteins and Temperature-sensitive Lethality—The *alg11–1* mutation leads to a severely impaired growth phenotype. Other glycosylation mutants that are blocked in the luminal steps of core biosynthesis display no obvious growth phenotype. For instance, deletion of *ALG3* and *ALG9*, which block addition of the sixth and seventh Man to Man₅GlcNAc₂-PP-Dol, respectively, and *ALG6*, which blocks addition of the first Glc to Man₉GlcNAc₂-PP-Dol, have no apparent effect on growth rate (14, 16, 18). The impaired *alg11–1* growth phenotype in conjunction with the sequence similarity between Alg11p and Alg2p suggested that Alg11p might be a mannosyltransferase involved with the cytosolic synthesis of Man₅GlcNAc₂-PP-Dol. To further investigate this mutant, the phenotype of an *alg11* null allele was determined. An 832-base pair *EcoRV* fragment within the *ALG11* gene was replaced with a fragment encoding the *URA3* gene. A standard one-step gene disruption (52) was performed to replace the chromosomal copy of *ALG11*

21834 Role of ALG11 in Oligosaccharide-Lipid Synthesis

A.

B.
25 C 37 C

FIG. 4. The ALG11 gene is essential for growth at 37°C. A, tetrad analysis of a diploid strain heterozygous for the alg11Δ::URA3 deletion allele. Diploids were sporulated, and dissected tetrads were analyzed for cell viability at 25°C. B, colonies were streaked onto YPAD plates and incubated for 5 days at 25°C or 37°C.

with the null allele in a diploid, as described under “Experimental Procedures.” The disruption of one allele was confirmed by polymerase chain reaction analysis of genomic DNA (data not shown). Heterozygous diploids were sporulated, and 15 dissected tetrads were analyzed for cell viability at 25°C (Fig. 4A). Colonies from spores carrying the null allele that were Ura⁺ could not be detected until 5 days after the wild-type colonies arose, indicating that the slow growth phenotype of the null allele was much more severe than the alg11-1 allele and that the severe growth defect is directly because of the alg11 mutation. Furthermore, no growth was observed when alg11Δ::URA3 colonies were grown at 37°C (Fig. 4B). Thus, the ALG11 gene is necessary for normal growth at room temperature and is essential for growth at high temperatures. This severe phenotype is in stark contrast to other glycosylation mutants that block lumenal ER steps in the synthesis of the core OSL, suggesting a role for ALG11 at an earlier stage.

ALG3 Is Not Epistatic to ALG11—As a further genetic test of whether ALG11 might function in the cytosolic assembly of Man₅GlcNAc₂-PP-Dol, the phenotype of an alg3Δ alg11Δ double mutant was analyzed. The alg3 mutant blocks the first mannose addition to the Man₅GlcNAc₂-PP-Dol once flipped into the ER lumen yet accumulates Man₅GlcNAc₂ OSL (7, 14, 20). Isogenic haploid strains of opposite mating types but containing the alg11Δ::URA3 and alg3Δ::HIS3 alleles were crossed, and the heterozygous diploids were sporulated. Of the 40 tetrads dissected, none contained viable spores that were both His⁺ and Ura⁺ (data not shown), demonstrating that the double haploid mutant is lethal. These results are consistent with ALG11 performing a function epistatic to ALG3, which in combination so severely limits downstream lumenal ER OSL glycan processing event(s) as to become synthetically lethal.

Alg11p Is Localized in the ER—To examine the physical properties of the Alg11 protein, an epitope-tagged ALG11 allele was constructed that encodes three tandem copies of the HA epitope at the C terminus. Expression of the Alg11-HAp fusion protein in an alg11Δ::URA3 strain rescued its glycosylation defect, hygromycin B sensitivity, and growth phenotype (data not shown), suggesting that the additional C-terminal HA sequences do not alter the normal function of the Alg11 protein. The ALG11-HA₃ fusion was expressed constitutively from the TPI promoter and could be detected by immunoblotting cell protein extracts separated by SDS-PAGE (Fig. 5A). Although the expected molecular mass of Alg11-HAp is about 68 kDa, the tagged protein migrated as a 59-kDa species on denaturing gels, which may be explained by its hydrophobicity. The protein is predicted to contain four N-linked glycosylation sites, but none appear to be utilized, because Alg11p was insensitive to

FIG. 5. Western immunoblot and cytological analysis of Alg11p. A, whole cell protein extracts were prepared from yeast cells (SEY6210) expressing Och1-HA₃p (lanes 1 and 2) or Alg11-HA₃p (lanes 3 and 4). Proteins were incubated in the presence or absence of endo H, fractionated by 8% SDS-PAGE, and subjected to Western blot analysis using anti-HA antibodies. B, indirect immunofluorescence of SEY6210 cells expressing HA3-tagged Alg11p. Fixed cells were treated with anti-HA antibodies followed by FITC-conjugated anti-mouse IgG and viewed by confocal microscopy.

digestion with endo H (Fig. 5A, compare lanes 3 and 4), whereas Och1-HAp, a known glycosylated Golgi sugar transferase, was sensitive to endo H (Fig. 5A, lanes 1 and 2).

The intracellular location of Alg11-HAp was analyzed by indirect immunofluorescence in wild-type cells. A perinuclear pattern of fluorescence (Fig. 5B) characteristic of the ER in yeast co-localized with DAPI-stained nuclei (not shown). In conjunction with fractionation experiments that demonstrated Alg11p is insoluble and associated with a membrane fraction (data not shown), these results suggest that Alg11p is a membrane protein that resides in the ER.

N-Linked Oligosaccharide Structures in alg11Δ OSL—In vivo labeling experiments employed [2-³H]Man or [2-³H]Glc (see “Experimental Procedures”), and the glycans released from OSL were chromatographed on Bio-Gel P-4 (Fig. 6). Panel A shows that the steady-state [³H]Man-labeled OSL glycans (open triangles) elute as Hex₃GlcNAc₂ (a), Hex₆GlcNAc₂ (b), Hex₇GlcNAc₂ (c), and Hex₁₁GlcNAc₂ (d). The central 85% of each peak was pooled, and their structural identities were investigated using endo H and α1,2-mannosidase digestion, as well as methylation linkage analysis with the results summarized in Table II. All glycans were α1,2-mannosidase- and endo H-sensitive with the exception of Hex₃GlcNAc₂. The lack of sensitivity to these two glycosidases defines the Hex₃GlcNAc₂ polysaccharide as the core Man₃GlcNAc₂ (48), and it was not investigated further. The endo H sensitivity of all other glycans confirms the presence of the Alg3p-directed middle arm α1,3-linked Man (see Ref. 47 and Table II) and defines the minimum structural component of this family of N-glycans as residues 2–5 and 7 (Scheme IA).

Role of ALG11 in Oligosaccharide-Lipid Synthesis

FIG. 6. P-4 profile of in vivo labeling of OSL and glycoprotein N-glycans in alg11Δ yeast. A, steady-state \[^3H\]Man-labeled (△) or \[^3H\]Glc-labeled (○) N-glycans released from OSL. The peaks are as follows: \(a\), \[^3H\]Man₃GlcNAc₂; \(b\), \[^3H\]Man₆GlcNAc₂; \(c\), \[^3H\]Man₇GlcNAc₂; \(d\), \[^3H\]Glc₃Man₈GlcNAc₂ and Glc₃\[^3H\]Man₈GlcNAc₂. B, \[^3H\]Glc-pulse-labeled N-glycans released from cellular proteins (●). The peaks are as follows: \(e\), \[^3H\]Glc₁Man₈GlcNAc; \(f\), \[^3H\]Glc₂Man₈GlcNAc; \(g\), \[^3H\]Glc₃Man₈GlcNAc. C, steady-state \[^3H\]Man-labeled (▲) N-glycans from cellular glycoproteins. The peaks are as follows: \(h\), \[^3H\]Man₆GlcNAc; \(i\), \[^3H\]Man₇GlcNAc; \(j\), Glc₁₋₂\[^3H\]Man₇₋₈GlcNAc. The internal raffinose standard centered at fraction number 184–186 is shown in all panels (■). The elution positions of Glc₃\[^3H\]Man₉GlcNAc₂, \[^3H\]Man₉GlcNAc₂, \[^3H\]Man₇GlcNAc₂, and \[^3H\]Man₅GlcNAc₂ released from authentic wild-type OSL and the internal raffinose marker are shown above relative to \(\log_{10}\) of their hexose size.

Digestion of Hex₆GlcNAc₂ (Fig. 6A, peak \(b\)) with α1,2-mannosidase gave rise to two products on the Bio-Gel P-4 that eluted as Hex₅GlcNAc₂ and Hex₄GlcNAc₂ indicating the presence of isomers with either one or two Man α1,2-linked residues. In the algΔ9 background residue 6 is not added to N-glycans because of the lack of Alg9p, the presumptive α1,2-mannosyltransferase responsible for adding residue 10, a required structural component for the addition of residue 6 (see Scheme IA and Refs. 6, 18, and 53). Once 6 is added, residue 9 is the final α1,2-Man added to OSL. This strict order of Man addition means that the Hex₆GlcNAc₂ containing one α1,2-Man must be missing both α1,2-linked residues 11 and 8, while having unsubstituted residue 6 and α1,2-Man 10. The other Hex₆GlcNAc₂ from OSL with two α1,2-linked Man residues must contain residues 8 and 10 while missing residue 6, because, as indicated above, the addition of 10 must precede that of 9. The existence of both isomers implies that both Man₃GlcNAc₂ and Man₄GlcNAc₂ OSLs are translocated into the ER lumen to be further elongated by the Man-P-Dol-dependent mannosyltransferases present in this compartment.

The \[^3H\]Man-labeled Hex₇GlcNAc₂ represented ~80% of the

TABLE II  
Characterization of alg11Δ N-glycans from OSL pulse-labeled for 2 min with \[^2^H\]Man.

| Peak\(a\) | Saccharide size | Endo H sensitivity | o1,2-mannosidase sensitivity (Hexoses lost)\(b\) | 3, 4, 6-Tri-O-methylmannose | 2, 4, Di-O-methylmannose | Structural designation |
|-----------|------------------|--------------------|------------------------------------------------|----------------------------|--------------------------|------------------------|
| a         | Hex₃GlcNAc₂      | –                 | 0                                                | NA                         | NA                       | Man₃GlcNAc₂            |
| b         | Hex₆GlcNAc₂      | +                 | 1,2                                             | +                          | +                        | Man₆GlcNAc₂            |
| c         | Hex₇GlcNAc₂      | +                 | 2                                               | ++                         | +                        | Man₇GlcNAc₂            |
| d         | Hex₁₁GlcNAc₂     | +                 | 2                                               | +++                        | ++                       | Glc₃Man₈GlcNAc₂        |

\(a\) From Fig. 6A.  
\(b\) Indicates number of Man removed from glycans by α1,2-mannosidase.  
\(c\) Not applicable.

21836 Role of ALG11 in Oligosaccharide-Lipid Synthesis

A

B

Man₇GlcNAc from translocated Man₃GlcNAc₂-PP-Dol and sites of terminal α1,3-Man addition in the Golgi

C

D

"Glycan" formed on translocated Man₄GlcNAc₂-PP-Dol after Mns1p trimming of residue 10 from 7. In Pool VI-I "mannan" (Fig. 7, inset), the outer chain repeat structure, X, is 2-5 units.

SCHEME I

glycans released from pulse-labeled alg11Δ OSL (Fig. 6A, peak c), hinting that this pool contained the rate-limiting dolichol-linked biosynthetic intermediate in the alg11Δ background. Endo H sensitivity, the loss of two α1,2-mannosidase-sensitive residues, and the methylation pattern of the Hex₇GlcNAc₂ pool (Table II) documents that Hex₇GlcNAc₂ contained the above-defined core residues 2–5 and 7 plus the upper arm α1,6-Man residue 6 and only two of the three α1,2-linked Man residues 8, 9, and 10 shown in Scheme IC. Because residue 10 is required for the addition of 6 (see above), only two of the three possible Man₇GlcNAc₂ isomers would be present in the OSL pool; the Man₇ derived from elongation of the Man₃GlcNAc₂-PP-Dol with α1,2-Man residues 9 and 10 (Scheme IB) and the Man₇ derived from the Man₄GlcNAc₂-PP-Dol with α1,2-Man residues 8 and 10 (see Scheme IC). Because α1,2-mannosidase removes two residues from each, and their methylation patterns are the same, we cannot specify the proportion of each in the pool.

The Hex₁₁GlcNAc₂ pool (Fig. 6A, peak d) lost the equivalent of two hexoses on α1,2-mannosidase digestion (Table II). In [³H]Glc pulse-labeling experiments (Fig. 6A, open circles) a peak that eluted in the same position as the [³H]Man-labeled Hex₁₁GlcNAc₂ showed the same endo H and α1,2-mannosidase sensitivities as the [³H]Man-labeled form implying that they are the same structure. The methylation analysis pattern generated by the [³H]Man-labeled Hex₁₁GlcNAc₂ (see Fig. 6, peak d and Table II) was consistent with this glycan being identical to the authentic Glc₃Man₉GlcNAc₂ with its lower arm shortened by the absence of residue 11 (see Scheme I, A and C for comparison).

³H-Glc- and ³H-Man-labeled N-Linked Glycan Structures on alg11Δ Glycoproteins—Bio-Gel P-4 chromatographic analysis of endo H-released glycans from [³H]Glc pulse-labeled alg11Δ glycoproteins revealed the presence of Glc₁₋₃Man₈GlcNAc oligosaccharides (Fig. 6B, peaks e–g). On steady-state [³H]Man labeling, Hex₆GlcNAc, Hex₇GlcNAc, and Hex₉GlcNAc were present in the P-4 profile (Fig. 6C, peaks h–j). α1,2-Mannosidase and endo H sensitivities of the peaks from Fig. 6C, summarized in Table III, are consistent with the following structural assignments: h, [³H]Man₆GlcNAc; i, [³H]Man₇GlcNAc; and j, Glc₁₋₂[³H]Man₇₋₈GlcNAc. Although Glc₁₋₂[³H]Man₇₋₈GlcNAc and [³H]Man₇GlcNAc are the expected partially and fully processed forms of Glc₃[³H]Man₈GlcNAc₂, the presence of [³H]Man₆GlcNAc in the endo H-released glycans may appear inconsistent. However, if both Man₃GlcNAc₂-PP-Dol and Man₄GlcNAc₂-PP-Dol forms were translocated into and elongated in the lumen of the ER as argued above, a [³H]Man₆GlcNAc would be expected on alg11Δ glycoproteins, and this would be among the endo H-released glycans. It would arise

Role of ALG11 in Oligosaccharide-Lipid Synthesis

TABLE III  
Characterization of steady-state \[^{2-3}H\]Man-labeled N-glycans from alg11Δ cellular glycoproteins

| Peak \(^a\) | Saccharide size | Endo H sensitivity | α1,2-Mannosidase sensitivity (Hexose loss) \(^b\) | Structural designation |
|-------------|------------------|--------------------|---------------------------------------------------|-------------------------|
| h           | Hex₆GlcNAc       | +                  | 1,2                                              | Man₆GlcNAc              |
| i           | Hex₇GlcNAc       | +                  | 1,2                                              | Man₇GlcNAc              |
| j           | Hex₉GlcNAc       | +                  | 1,2                                              | Glc₁₂Man₇₈GlcNAc        |

\(^a\) From Fig. 6C.  
\(^b\) Indicates number of Man removed from glycans by α1,2-mannosidase.

through Mns1p trimming of residue 10 from the \[^3H\]Man₇GlcNAc isomer defined in Scheme IB. Another \[^3H\]Man₆GlcNAc isomer on alg11Δ glycoproteins would be missing residue 9 but have residue 10 (Scheme IB) because of the inability of Msn1p to efficiently remove 10 in the absence of 9 (54). Both of these Man₆GlcNAc₂ forms were identified on OSL in the previous section and Table II and are shown to be present on alg11Δ glycoproteins by the NMR structural studies of the isolated glycans.

The above experiments with labeled Glc and Man indicate that two series of truncated N-glycans arise in the alg11Δ background, one from Man₃GlcNAc₂-PP-Dol and the other from Man₄GlcNAc₂-PP-Dol. Processing of these two glycan families would be quite different in the yeast ER and Golgi. The first, leading to Man₇GlcNAc in Scheme IB with no Glc residues and without residue 8, would only be processed in the ER by Mns1p's removal of residue 10 (54) and, quite likely, would not be expected to be a significant substrate for the early Golgi α1,6 branch-initiating enzyme Och1p (55, 56). The only Golgi addition to this series would be α1,3-Man residues to 5, 7, 9, and/or residual residue 10 that escaped Mns1p activity (54). The second series, containing Glc residues G₁, G₂, G₃ (Scheme IC), and Man 10, would be trimmed by Gls1p, Gls2p, and Mns1p. Because the series contains residue 8, it could act as a substrate, albeit weakly (56), for Och1p. Once acted upon by Och1p, the series would, presumably, follow a normal mannan elongation pattern (57).

Assigning alg11Δ Oligosaccharide Structures by \({}^{1}H\) NMR—To test the above predictions and determine the steady-state distribution of N-linked glycans in alg11Δ yeast, whole-cell glycoproteins were deglycosylated, and the oligosaccharides were sized on Bio-Gel P-4 (Fig. 7). The profile reveals Hex₅₋₉GlcNAc sizes, which were collected, rechromatographed on Bio-Gel P-4 (see "Experimental Procedures"), and assigned as pools I-V, respectively. The \(M_r\) of the final pools was confirmed by MALDI-TOF MS. Pools II and III were solely Hex₆GlcNAc and Hex₇GlcNAc, respectively, whereas pools I, IV, and V had minor (<5% MS signal intensity) Hexₓ₊₁GlcNAc components. As described in an earlier study (58), pools I-V were subjected to Dionex high pH anion-exchange chromatography (data not shown), and the number of peaks and their areas generally agreed with the proportions of major and minor branch isomers ultimately assigned in each pool. A fraction of the large \(V_0\) peak was collected as alg11Δ mannan (Fig. 7, inset), and by MALDI-TOF MS it was Hex₃₀₋₄₉GlcNAc with an average size of Hex₃₉GlcNAc (Fig. 8, inset).

To determine how N-glycans were processed in alg11Δ yeast, the structure of isomers present in each pool were determined by \({}^{1}\)H-NMR 1D and 2D DQF-COSY experiments. The 1D spectra (not shown) were used for anomeric proton chemical shift assignments and the integration of resonance intensities as in previous studies (20, 53, 57–59). The apportionment of intensities was assigned to structural isomers in each pool using relevant C1-, C2-, and C3-H chemical shifts. As noted above for labeled alg11Δ glycans (Table II and III), all structures deduced have as their minimum core the Man₄GlcNAc structure defined in Scheme IB, consisting of residues 2 α/β, 3–5, and 7,

FIG. 7. Preparative Bio-Gel P-4 chromatography of N-linked glycans released from alg11Δ glycoproteins. Oligosaccharides were sized on the 2.6 × 67-cm Bio-Gel P-4 column, and 1.35-ml fractions were collected and analyzed for neutral hexose by phenol-sulfuric acid assay (♦ and ▲) and the presence of a Man₃GlcNAc\[^3H\]ol internal marker by scintillation spectrometry (●). Peaks representing separated Hex₅₋₉GlcNAc were collected as pools I-V and rechromatographed on an analytical 1.6 × 96-cm Bio-Gel P-4 column as described under "Experimental Procedures." Inset, a portion of the \(V_0\) peak was collected as pool VI and rechromatographed on the preparative column. The smallest glycans of the \(V_0\) peak were collected as pool VI-I and represent alg11Δ mannan.

FIG. 8. One- and two-dimensional DQF-COSY spectra at 500 MHz and 318 K of the C1- and C2-H region of alg11Δ mannan. The average size of the glycans in pool VI-I by MALDI-TOF MS (Fig. 7, inset) was Hex₃₉GlcNAc, and the concentration in the NMR tube on the basis of this size was 2 mM.

accompanied by their normal anomeric proton chemical shifts. Additions of mannose in various linkages to this core diagnostically alters anomeric and ring proton chemical shifts allowing structural assignment on the basis of a large library of existing chemical shift data (5, 20, 53, 57–65). A summary of the altered N-glycosylation pathway in alg11Δ yeast defined in the current work is shown in Scheme II.

alg11Δ Mannan—Essentially all of the structural isomers assigned in the Hex₅₋₉GlcNAc pools from alg11Δ glycoproteins (Scheme II) are consistent with Man₃GlcNAc₂-PP-Dol being

21838

Role of ALG11 in Oligosaccharide-Lipid Synthesis

Cytosol

ER Lumen

Cytosol

Golgi Lumen

Cytosol

SCHEME II

Role of ALG11 in Oligosaccharide-Lipid Synthesis

translocated into the ER lumen and elongated maximally to the Hex₇GlcNAc₂-PP-Dol, whose glycan structure is shown in Scheme IB. However, the pulse and steady-state labeling studies of both *alg11Δ* OSL and glycoprotein glycans described in the text, and Fig. 6 indicated that the Hex₈GlcNAc core shown in Scheme IC, which has the single lower arm α1,2-linked Man (residue 8), was the principal N-linked glycan intermediate. It seems unlikely that all of the structures in Scheme II without residue 8 would be degradation products of this Hex₈ core intermediate (Scheme IC). This could result if Man₃GlcNAc₂-PP-Dol translocated into the ER lumen led to this family of poorly elongated dead-end products that accumulate on glycoproteins, whereas Man₄GlcNAc₂-PP-Dol was more efficiently processed to Glc₃Man₈GlcNAc₂-PP-Dol, transferred to protein, trimmed in the ER, and elongated in the Golgi to mannan (66).

If this occurred, we would expect to find essentially all of residue 5 substituted by α1,2-Man 8, allowing Och1p to initiate outer chain synthesis by adding α1,6-Man 12. Furthermore, we would expect Mns1p to remove the middle arm α1,2-Man 10 from 7, because the mannan chains whose origin was Glc₃-Man₈GlcNAc₂-PP-Dol contained residues 6 and 9, requisite for this activity (54). To test this prediction, pool VI, the more included portion of the *alg11Δ V₀* peak in Fig. 7, was isolated and rechromatographed on Bio-Gel P-4 (Fig. 7, inset). Again, the most included portion, identified as pool VI-I, was isolated and subjected to MS analysis and 1D DQF-COSY, 2D DQF-COSY, and relayed ROESY ¹H NMR experiments.

MALDI-TOF MS provided a distribution of Hex₃₀₋₄₉GlcNAc with an average size of Hex₃₉GlcNAc for the *alg11Δ* mannan (Fig. 8, inset). The Man anomeric proton region of the 1D and 2D DQF-COSY 318 K spectra is shown in Fig. 8. Core residues 3 (H₁/H₂ = 4.72/4.188 ppm) and 4 (H₁/H₂ = 4.862/4.131 ppm) are present at 1 mol each. Note that the signal at H₁/H₂ = 5.332/4.108 ppm for residue 5 when substituted by α1,2-Man residue 8 (see Scheme IC). Residue 5 at this chemical shift integrates to more than 0.92 mol, indicating that nearly all of this mannan fraction was derived from the Man₄GlcNAc₂-PP-Dol precursor. Essentially no signal could be found for residual glucose in this glycan pool. In addition, none of the anomeric proton signal of residue 7 is found at 5.403 ppm, where it resides when substituted by α1,2-Man 10 (67). Thus, we conclude that the Glc₃Man₈GlcNAc₂ transferred to *alg11Δ* glycoproteins is thoroughly processed by glucosidases I and II, as well as the ER α1,2-mannosidase, Mns1p, and is the precursor of mannan in this mutant.

DISCUSSION

In this report we describe the characterization of *alg11*, a yeast mutant that is deficient in the assembly of lipid-linked oligosaccharides, which was isolated as a sodium vanadate-resistant, hygromycin B-sensitive glycosylation mutant (27). Sodium vanadate is a drug that enriches for yeast glycosylation mutants (26, 27), although the molecular mechanism for this enrichment is not understood. The severely impaired growth phenotype associated with *alg11Δ* is incompatible with the idea that Alg11p functions as a mannosyltransferase that catalyzes the addition of the fourth or fifth mannose to OSL in the ER lumen. As expected for a protein that functions in OSL synthesis, immunofluorescence analysis demonstrated that Alg11p is localized in the ER (Fig. 5). Epistasis analysis of an *alg11Δ alg3Δ* double mutant demonstrates that *alg3* is not epistatic to *alg11*, strengthening the notion that *ALG11* does not simply function in the stepwise ER lumenal assembly pathway. Rather, the mutant phenotype more closely resembles that associated with mutants blocked in the early steps of core biosynthesis that occur on the cytoplasmic face of the ER, and indeed, analysis of lipid-linked oligosaccharides and those on

alg11 glycoproteins defines Alg11p to be involved in adding the final α1,2-linked Man to the Man₅GlcNAc₂-PP-Dol synthesized on the cytosolic face of the ER. An assay to show that Alg11p is indeed an α1,2-mannosyltransferase remains to be developed.

The absence of the terminal α1,2-Man residue so impairs the ER translocation of the Man₄GlcNAc₂-PP-Dol that Man₃GlcNAc₂-PP-Dol is also “flipped” into the ER lumen. Normalizing the short *N*-glycans in pools I-V derived from Man₃GlcNAc₂-PP-Dol and the *V₀* mannan derived from Man₄GlcNAc₂-PP-Dol (Fig. 7) for their respective mannose content provides a rough estimate that nearly equal amounts of both are translocated into the ER. Man₃,₄GlcNAc₂-PP-Dol both appear to be elongated by the Man-P-Dol-dependent mannosyltransferases in the ER lumen to Man₇,₈GlcNAc₂-PP-Dol. Of particular interest is that the Man₈ isomer missing terminal α1,2-linked Man residue 11 (Scheme IC) is a substrate for glucosylation (Fig. 6). Once transferred to protein, Glc₃Man₈GlcNAc₂ is a substrate for ER removal of the three Glc residues by Gls1p and Gls2p and the middle arm α1,2-Man 10 by Mns1p (54). This Man₈GlcNAc₂ is a substrate for the Golgi addition of the outer chain branch-initiating α1,6-Man 12 on core residue 5 (68) by Och1p (55). Indeed, residue 10 is completely absent in *alg11Δ* mannan, and residues 8 and outer chain-initiating 12 are essentially quantitatively present (see Scheme ID, and Supplemental Material, Fig. 1).

The accumulation of some Man₃GlcNAc₂-PP-Dol on steady-state labeling of *alg11Δ* OSL indicates that once Man₄GlcNAc₂-PP-Dol is formed it is translocated and elongated to at least Man₇GlcNAc₂-PP-Dol, the next larger OSL that accumulates (see Structure I and Fig. 6A). As discussed earlier, the proportion of the two possible isomers in the pool is not known, but Man₇B, the product of Man₃GlcNAc₂-PP-Dol elongation, would be the main candidate expected to accumulate prior to transfer to protein, because it is not glucosylated. Accumulation of Man₇A would imply that addition of α1,2-Man 9 to 6 was rate-limiting relative to the formation of Glc₃Man₈GlcNAc₂-PP-Dol, a small amount of which accumulates on steady-state mannos and glucose labeling (Fig. 6A).

The first α1,2-Man, residue 8, is absent on all short non-glucosylated oligosaccharides released from *alg11Δ* glycoproteins (Scheme II). These apparently accumulate as biosynthetic end products because of their failure to be elongated significantly by Och1p (55) and serve in the Golgi as substrates for peripheral α1,2-Man elongation (69), as well as capping with α1,3-Man by Mnn1p (70), and Man α1,3Man α1,3 elongation by additional α1,3-mannosyltransferases (71).

The Alg11 protein has been conserved through evolution. In addition to closely related proteins from other eukaryotes, Alg11p displays significant homology to Alg2p in *S. cerevisiae* and *C. elegans*. Mutants in *alg2* are defective in an early step of the OSL assembly pathway, accumulating lipid-linked Man-GlcNAc₂ and Man₂GlcNAc₂ (7). *ALG2* encodes a transferase that catalyzes mannose addition on the cytoplasmic face of the ER or a protein that regulates the activity of this transferase (12). Despite the sequence similarity between Alg11p and Alg2p, overexpression of *ALG2* fails to rescue the *alg11* muta-

tion and vice versa (data not shown) suggesting that these two proteins do not perform overlapping functions.

A 23-amino acid sequence that is particularly well conserved in the Alg2 and Alg11 proteins was observed. Strikingly, when this sequence was used to search the data base, we found that it was present in over 60 proteins, most of which have known functions in carbohydrate metabolism. These include different sucrose synthases (sucrose::UDP-glucosyltransferases) from a variety of organisms, proteins from yeast, mice, and humans that function in the synthesis of the glycosylphosphatidylinositol anchor, including PigA and the yeast Spt14, and interestingly, a large number of bacterial proteins that function in the biosynthesis of the lipopolysaccharide (LPS) coat. The functional significance and conservation of this motif to a family of glycosyltransferases was previously recognized. A comparison of sequences common to a number of bacterial α-mannosyltransferases and their alignment by hydrophobic cluster analysis identified several invariant residues. When this sequence was used to search the data bases, it was found to exist in a large family of proteins, including Alg2p (72), although Alg11p was not identified in those studies. One feature common to all of these proteins is the utilization of carbohydrates in the form of nucleotide sugars. The conservation of this sequence among such a wide spectrum of proteins implicates this peptide as a domain involved in the binding of nucleotide sugars or in the catalysis of their transfer, though not all proteins that utilize nucleotide sugars contain this sequence. If indeed this sequence reflects an active site domain that functions in nucleotide sugar binding, it may provide a useful target for the inactivation of these proteins, particularly in the case of the bacterial proteins that synthesize LPS, as these are the surface molecules used by pathogens to colonize and survive in their hosts.

**Acknowledgments—** We thank Niomi Peckham for early contributions to the ALG11 and OST4 projects. We thank Markus Aebi, Stephan te Heesen, and Maria Kukuruzinska for plasmids and members of the Dean laboratory for comments on the manuscript. We also gratefully acknowledge use of the Wadsworth biological mass spectrometry core facility and the Wadsworth structural NMR facility and thank Tracy Godfrey for help in preparing the manuscript.

**REFERENCES**

1. Kornfeld, R., and Kornfeld, S. (1985) *Annu. Rev. Biochem.* **54**, 631–664
2. Herscovics, A., and Orlean, P. (1993) *FASEB J.* **7**, 540–550
3. Hirschberg, C. B., and Snider, M. D. (1987) *Annu. Rev. Biochem.* **56**, 63–87
4. Abeijon, C., and Hirschberg, C. B. (1992) *Trends Biochem. Sci.* **17**, 32–36
5. Byrd, J. C., Tarentino, A. L., Maley, F., Atkinson, P. H., and Trimble, R. B. (1982) *J. Biol. Chem.* **257**, 14657–14666
6. Jakob, C. A., Burda, P., Roth, J., and Aebi, M. (1998) *J. Cell Biol.* **142**, 1223–1233
7. Huffaker, T. C., and Robbins, P. W. (1983) *Proc. Natl. Acad. Sci. U. S. A.* **80**, 7466–7470
8. Orlean, P., Albright, C., and Robbins, P. W. (1988) *J. Biol. Chem.* **263**, 17499–17507
9. Heller, L., Orlean, P., and Adair, W. L. (1992) *Proc. Natl. Acad. Sci. U. S. A.* **89**, 7013–7016
10. Albright, C. F., and Robbins, P. W. (1990) *J. Biol. Chem.* **265**, 7042–7049
11. Huffaker, T. C., and Robbins, P. W. (1982) *J. Biol. Chem.* **257**, 3203–3210
12. Jackson, B. J., Kukuruzinska, M. A., and Robbins, P. (1993) *Glycobiology* **3**, 357–364
13. Kepes, F., and Schekman, R. (1988) *J. Biol. Chem.* **263**, 9155–9161
14. Aebi, M., Gassenhuber, J., Domdey, H., and te Heesen, S. (1996) *Glycobiology* **6**, 439–444
15. te Heesen, S., Lehle, L., Weissman, A., and Aebi, M. (1994) *Eur. J. Biochem.* **224**, 71–91
16. Reiss, G. S., te Heesen, S., Zimmerman, J., Robbins, P. W., and Aebi, M. (1996) *Glycobiology* **6**, 493–498
17. Stagljar, I., te Heesen, S., and Aebi, M. (1994) *Proc. Natl. Acad. Sci. U. S. A.* **91**, 5977–5981
18. Burda, P., te Heesen, S., Brachat, A., Wach, A., Dusterhoft, A., and Aebi, M. (1996) *Proc. Natl. Acad. Sci. U. S. A.* **93**, 7160–7165
19. Burda, P., and Aebi, M. (1998) *Glycobiology* **8**, 455–462
20. Verostek, M. F., Atkinson, P. H., and Trimble, R. B. (1993) *J. Biol. Chem.* **268**, 12095–12103
21. Turco, S. J., Stetson, B. J., and Robbins, P. W. (1977) *Proc. Natl. Acad. Sci. U. S. A.* **74**, 4411–4414
22. Trimble, R. B., Maley, F., and Tarentino, A. L. (1980) *J. Biol. Chem.* **255**, 10232–10238
23. Lehle, L. (1980) *Eur. J. Biochem.* **109**, 589–601
24. Trimble, R. B., Byrd, J. C., and Maley, F. (1980) *J. Biol. Chem.* **255**, 11892–11895
25. Silberstein, S., and Gilmore, R. (1996) *FASEB J.* **10**, 849–858
26. Ballou, L., Hitzeman, R. A., Lewis, M. S., and Ballou, C. E. (1991) *Proc. Natl. Acad. Sci.* **88**, 3209–3212
27. Dean, N. (1995) *Proc. Natl. Acad. Sci. U. S. A.* **92**, 1287–1291
28. Sherman, F. (1991) *Methods Enzymol.* **194**, 3–21
29. Ito, H. Y., Fukuda, Y., Murata, K., and Kimura, J. (1983) *J. Bacteriol.* **153**, 163–168
30. Sikorski, R. S., and Hieter, P. (1989) *Genetics* **122**, 19–27
31. Sanger, F., Nicklen, S., and Coulson, A. R. (1977) *Proc. Natl. Acad. Sci. U. S. A.* **74**, 5463–5467
32. Altschul, S. F., Madden, T. L., Schaffer, A. A., Zhang, J., Zhang, Z., Miller, W., Lipman, D. J. (1997) *Nuc. Acids Res.* **25**, 3389–3402
33. Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) *Molecular Cloning: A Laboratory Manual*, Cold Spring Harbor Press, Cold Spring Harbor, New York
34. Neiman, A. M., Mhaiskar, V., Manus, V., Galibert, F., and Dean, N. (1997) *Genetics* **145**, 637–645
35. Poster, J. B., and Dean, N. (1996) *J. Biol. Chem.* **271**, 3837–3845
36. Dean, N., Zhang, Y. B., and Poster, J. B. (1997) *J. Biol. Chem.* **272**, 31908–31114
37. Dubois, M., Gilles, K. A., Hamilton, J. K., Rebers, P. A., and Smith, F. (1956) *Anal. Chem.* **28**, 350–356
38. Bearden, J. C., Jr. (1978) *Biochim. Biophys. Acta* **533**, 525–529
39. Verostek, M. F., Lubowski, C., and Trimble, R. B. (2000) *Anal. Biochem.* **278**, 111–122
40. Trumbly, R. J., Robbins, P. W., Belfort, M., Ziegler, F. D., Maley, F., and Trimble, R. B. (1985) *J. Biol. Chem.* **260**, 5683–5690
41. Ciucanu, F., and Kerek, F. (1984) *Carbohydr. Res.* **131**, 209–217
42. States, D. J., Haberkorn, R. A., and Ruban, D. J. (1982) *J. Magn. Reson.* **48**, 286–292
43. Stanley, P., and Atkinson, P. H. (1988) *J. Biol. Chem.* **263**, 11374–11381
44. Rance, M., Sorensen, O. W., Bodenhausen, G., Wagner, G., Ernst, R. R., and Wuthrich, K. (1983) *Biochem. Biophys. Res. Commun.* **117**, 479–485
45. Cipollo, J. F., Trimble, R. B., Rance, M., and Cavanagh, J. (2000) *Anal. Biochem.* **278**, 52–58
46. Trimble, R. B., Maley, F., and Chu, F. K. (1983) *J. Biol. Chem.* **258**, 2562–2567
47. Chi, J. H., Roos, J., and Dean, N. (1996) *J. Biol. Chem.* **271**, 3132–3140
48. Trimble, R. B., and Tarentino, A. L. (1991) *J. Biol. Chem.* **266**, 1646–1651
49. Lehle, L., and Tanner, W. (1995) in *New Comprehensive Biochemistry* (Montreuil, J., Vliegenthart, J. F. G., and Schachter, H., eds) Vol. 29a, pp. 475–509, Elsevier Science Publishing Co., Inc., New York
50. Kuranda, M. J., and Robbins, P. W. (1991) *J. Biol. Chem.* **266**, 19758–19767
51. Umeda, K., Yoko, O.-T., Nakayama, K.-I., Suzuki, T., and Jigami, Y. (2000) *Yeast* **16**, 1261–1271
52. Rothstein, R. J. (1983) *Methods Enzymol.* **101**, 202–211
53. Cipollo, J. F., and Trimble, R. B. (2000) *J. Biol. Chem.* **278**, 4267–4277
54. Ziegler, F. D., and Trimble, R. B. (1991) *Glycobiology* **1**, 605–614
55. Nakanishi-Shindo, Y., Nakayama, K.-I., Tanaka, A., Toda, Y., and Jigami, Y. (1993) *J. Biol. Chem.* **268**, 26338–26345
56. Nakayama, K.-I., Nakanishi-Shindo, Y., Tanaka, A., Haga-Toda, Y., and Jigami, Y. (1997) *FEBS Lett.* **412**, 547–550
57. Trimble, R. B., and Atkinson, P. H. (1992) *Glycobiology* **2**, 57–75
58. Trimble, R. B., Atkinson, P. H., Tschopp, J. F., Townsend, R. R., and Maley, F. (1991) *J. Biol. Chem.* **266**, 22807–22817
59. Verostek, M. F., Atkinson, P. H., and Trimble, R. B. (1993) *J. Biol. Chem.* **268**, 12104–12115
60. Cohen, R. E., and Ballou, C. E. (1980) *Biochemistry* **19**, 4345–4358
61. Carver, J. P., and Grey, A. (1981) *Biochemistry* **20**, 6607–6616
62. Vliegenthart, J. F. G., Dorland, L., and van Halbeek, H. (1983) *Adv. Carbohydr. Chem. Biochem.* **41**, 209–374
63. Trimble, R. B., and Atkinson, P. H. (1986) *J. Biol. Chem.* **261**, 9815–9824
64. Brisson, J. R., and Carver, J. P. (1983) *Biochemistry* **22**, 3680–3686
65. Winnik, F. M., Carver, J. P., and Krepinsky, J. J. (1982) *J. Org. Chem.* **47**, 2701–2707
66. Ballou, C. E. (1990) *Methods Enzymol.* **185**, 440–470
67. Dorland, L., van Halbeek, H., and Vliegenthart, J. F. G. (1981) *J. Biol. Chem.* **256**, 7708–7711
68. Hernandez, L. M., Ballou, L., Alvarado, E., Gillece-Castro, B. L., Burlingame, A. L., and Ballou, C. E. (1989) *J. Biol. Chem.* **264**, 11849–11856
69. Lussier, M., Sdicu, A. M., Camirand, A., and Bussey, H. (1996) *J. Biol. Chem.* **271**, 110001–110008
70. Verostek, M. F., and Trimble, R. B. (1995) *Glycobiology* **5**, 671–681
71. Romero, P. A., Lussier, M., Veronneau, S., Sdicu, A.-M., Herscovics, A., and Bussey, H. (1999) *Glycobiology* **9**, 1045–1051
72. Geremia, R. A., Petroni, E. A., Ielpi, L. and Henrissat, B. (1996) *Biochem. J.* **318**, 133–138

GLYCOBIOLOGY AND  
EXTRACELLULAR MATRICES:  
The Yeast ALG11 Gene Specifies Addition  
of the Terminal α1,2-Man to the Man₅  
GlcNAc₂-PP-dolicholN-Glycosylation  
Intermediate Formed on the Cytosolic Side  
of the Endoplasmic Reticulum  

John F. Cipollo, Robert B. Trimble, Jung Hee  
Chi, Qi Yan and Neta Dean  
J. Biol. Chem. 2001, 276:21828-21840.  
doi: 10.1074/jbc.M010896200 originally published online February 15, 2001  

Access the most updated version of this article at doi: 10.1074/jbc.M010896200  

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.  

Alerts:  
- When this article is cited  
- When a correction for this article is posted  

Click here to choose from all of JBC's e-mail alerts  

Supplemental material:  
http://www.jbc.org/content/suppl/2001/06/15/276.24.21828.DC1.html  

This article cites 71 references, 45 of which can be accessed free at  
http://www.jbc.org/content/276/24/21828.full.html#ref-list-1
