
# Identification of Genes Encoding Adenylate Isopentenyltransferase, a Cytokinin Biosynthesis Enzyme, in *Arabidopsis thaliana*

**Kentaro Takei†§**, **Hitoshi Sakakibara§‡**, and **Tatsuo Sugiyama‡§**

*From the †Department of Biological Mechanisms and Functions, Graduate School of Bioagricultural Sciences, Nagoya University, Chikusa, Nagoya 464-8601, Japan and §RIKEN (The Institute of Physical and Chemical Research), Plant Science Center, Hirosawa 2-1, Wako, Saitama 351-0198, Japan*

---

The initial step in the *de novo* biosynthesis of cytokinin in higher plants is the formation of isopentenyladenosine 5′-monophosphate (iPMP) from AMP and dimethylallylpyrophosphate (DMAPP), which is catalyzed by adenylate isopentenyltransferase (IPT). Although cytokinin is an essential hormone for growth and development, the nature of the enzyme for its biosynthesis in higher plants has not been identified. Herein, we describe the molecular cloning and biochemical identification of IPTs from *Arabidopsis thaliana*. Eight cDNAs encoding putative IPT, designated as AtIPT1 to AtIPT8, were picked up from *A. thaliana*. The *Escherichia coli* transformants expressing the recombinant proteins excreted cytokinin species into the culture medium except for that expressing AtIPT2 that is a putative tRNA IPT. A purified recombinant AtIPT1 catalyzed the formation of iPMP from DMAPP and AMP. These results indicate that the small multigene family contains both types of isopentenyltransferase, which could synthesize cytokinin and mature tRNA.

Cytokinin is a phytohormone involved in various processes of growth and development of plants, such as cell division, photosynthesis, chloroplast differentiation, senescence, and nutrient metabolism (1). Although phenylurea-type species are known (2), the most abundant cytokinins in plants are adenine-type species, which are adenosines substituted at N⁶ with an isoprenoid or aromatic side chain.

Multiple routes have been proposed in cytokinin biosynthesis. Transfer RNA degradation has been suggested to be a source of cytokinin (3), because some tRNA molecules contain an isopentenyladenosine (iPA)¹ residue at the site adjacent to the anticodon. The modification is catalyzed by tRNA isopentenyltransferase (tRNA IPT; EC 2.5.1.8), which has been identified in various organisms such as *Escherichia coli* (4–6), *Saccharomyces cerevisiae* (7, 8), *Lactobacillus acidophilus* (9), *Homo sapiens* (10), and *Zea mays* (11). However, from the calculated tRNA turnover rate, it is estimated that the degradation pathway is not a major source of cytokinin (12). Another possible route of cytokinin formation is *de novo* biosynthesis of iPMP by adenylate isopentenyltransferase (IPT; EC 2.5.1.27) with DMAPP and AMP as substrates. In the plant pathogenic crown gall-forming bacterium, *Agrobacterium tumefaciens*, the IPT gene on the Ti-plasmid (13) is integrated into the genome of host plant cells after infection. Overproduction of cytokinins by the transduced *IPT* causes abnormal cell proliferation. The gene has been identified in various bacterial species (13–17), and the translated product has been proved to biosynthesize iPMP, an active cytokinin, *in vitro* (18). On the other hand, there is little concrete evidence of the authentic biosynthesis of iPMP by IPT in higher plants. Cytokinin has been suggested to be synthesized in specific sites such as the root tip (19), immature kernel (20), and shoot apical meristem (21). However, the activity of plant IPT has been reported in only a few tissues such as immature maize kernels (20) and cytokinin autonomous tobacco callus (22, 23). As the enzyme seemed to be highly unstable and low in content, biochemical approaches to purify and characterize plant IPT have been hampered.

As the first step toward understanding the cytokinin biosynthetic pathway at the molecular level, we tried to identify IPT genes in *Arabidopsis thaliana*. Our study showed that *Arabidopsis* contains multiple IPT genes encoded by a small multigene family. To our knowledge, this is the first report on the identification of IPT in a higher plant at the molecular level.

---

## EXPERIMENTAL PROCEDURES

### Plant Materials—
*A. thaliana* ecotype Columbia was grown on vermiculite (24) at 22 °C under fluorescent light, at an intensity of about 100 μE m⁻² s⁻¹, in a growth chamber with a photoperiod of 16 h (day)/8 h (night).

### RT-PCR—
Total RNA was prepared by the guanidine thiocyanate procedure (25). Complementary DNAs were amplified with SuperScript One-step RT-PCR system (Life Technologies, Inc.) essentially as described by the supplier. Sequences of the primers for PCR were: for the AtIPT1, 5′-TCATGACAGAACTCAACTTCCACC-3′ and 5′-ATAAAGCTTCTAATTTTGCACCAAATGCCGC-3′; for the AtIPT2, 5′-CGCGGTACCGTCATGATGTTAAACCCTAGC-3′ and 5′-ATAGTCGACTGATATAAATCAATTACTTCTGC-3′; for the AtIPT3, 5′-CGCGGATCCATGATCATGAAGATATCTATGGC-3′ and 5′-ATAGTCGACGTGGTTACAAC TGATCACC-3′; for the AtIPT4, 5′-TCATGAAGTGTAATGACAAAATGG-3′ and 5′-ATAGTCGACGTTTGGCGGTGATATTAGTCC-3′; for the AtIPT5, 5′-GGGATCATGAAGCCATG GATGACGGC-3′ and 5′-GGTTCCTGCAGTACCTCACCGGG-3′; for the IPT6, 5′-CAACA ACTCATGACCTTGTTATCACC-3′ and 5′-GGCCAAGCTTGGAAAACAGACTAAACTTCC-3′; for the AtIPT7, 5′-GGCGGATCCTCAATCTCATC-3′ and 5′-GGCCTGCAGCTTTTCATATC-3′.

---

*This paper is available online at http://www.jbc.org*

---

*Vol. 276, No. 28, Issue of July 13, pp. 26405–26410, 2001*
*Printed in U.S.A.*

*Received for publication, March 9, 2001, and in revised form, April 4, 2001*
*Published, JBC Papers in Press, April 19, 2001, DOI 10.1074/jbc.M102130200*

---

*† This work was supported by Grants-in-aid for Scientific Research on Priority Areas (numbers 09274101 and 09274102 (to T. S.) and 12142202 (to H. S.)) from the Ministry of Education, Culture, Sports, Science and Technology, Japan. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.*

*‡ To whom all correspondence should be addressed: RIKEN (The Institute of Physical and Chemical Research), Plant Science Center, Hirosawa 2-1, Wako, Saitama 351-0198, Japan. Tel.: 81-48-467-6906; Fax: 81-52-467-6857; E-mail: sakaki@postman.riken.go.jp.*

*¹ The abbreviations and trivial names used are: iPA, isopentenyladenosine; DMAPP, dimethylallylpyrophosphate; DTT, dithiothreitol; ELISA, enzyme-linked immunosorbent assay; HPLC, high performance liquid chromatography; iP, isopentenyladenine; iPMP, isopentenyladenosine 5′-monophosphate; IPT, adenylate isopentenyltransferase; IPTG, isopropyl-β-D-thiogalactopyranoside; PAGE, polyacrylamide gel electrophoresis; PCR, polymerase chain reaction; RT, reverse transcription; tRNA IPT, tRNA isopentenyltransferase; X-Gal, 5-bromo-4-chloro-3-indolyl-β-D-galactoside; Z, *trans*-zeatin; FPLC, fast protein liquid chromatography.*
### TABLE I
**Structural features of AtIPT genes and proteins**

| Gene     | Accession number | Length$^a$ | Molecular mass$^b$ ($kDa$) | % identity to AtIPT1 | Chromosome |
|----------|------------------|------------|---------------------------|---------------------|------------|
| AtIPT1   | AAG52395         | 357        | 40.8                      | 100                 | I          |
| AtIPT2   | AAF00582         | 466        | 53.1                      | 34.7                | II         |
| AtIPT3   | CAB87756         | 336        | 37.9                      | 46.8                | III        |
| AtIPT4   | CAA22998         | 318        | 36.7                      | 56.9                | IV         |
| AtIPT5   | AC068809$^c$     | 330        | 37.4                      | 43.4                | V          |
| AtIPT6   | AAG50809         | 342        | 38.4                      | 55.6                | I          |
| AtIPT7   | BAB02782         | 329        | 37.0                      | 40.4                | III        |
| AtIPT8   | BAB02956         | 330        | 37.3                      | 60.6                | III        |

---

**Notes:**
- $^a$ Lengths are given in amino acids.
- $^b$ Calculated molecular mass.
- $^c$ AC068809 is an accession number for the nucleotide sequence. Reading frame encoding AtIPT5 is 30367–29375. The gene nomenclature was determined by discussion with T. Kakimoto.

---

### Enzyme Assays for IPT
Two methods were applied to measure the IPT activities:
1. **Radioisotope rapid assay**: The enzyme assay was carried out as essentially described by Blackwell and Horgan (18).
2. **Nonradioisotope assay**: Enzyme was incubated in a reaction mixture (1 M betaine, 20 mM triethanolamine, 50 mM KCl, 10 mM MgCl₂, 1 mM DTT, 1 mg/ml bovine serum albumin, pH 8.0) with 1 mM AMP and 340 μM DMAPP at 25°C for 20 min. The reaction was stopped by the addition of a quarter volume of 10% acetate and centrifuged at 18,000 × g for 20 min. The resulting supernatant was subjected to HPLC with an ODS column (Merck, Supersphere RP-select B; 4 mm inside diameter × 250 mm). Other conditions were as described previously (29). One unit of IPT activity was defined as the amount of enzyme that produced 1 μmol of iPMP/min under the condition of the reaction.

### Identification of Cytokinin Species by Mass Spectrometry
Liquid chromatography-mass spectrometry analysis of cytokinins was performed on a Platform II LC-MS (Jasco, Tokyo, Japan) with a C₁₈ column (Wakosil-II 5C18RS, 1 mm inside diameter × 250 mm) using a positive ion electrospray ionization. The cone voltage was 42 V, source temperature was 70°C, and capillary voltage was 3.0 V. Data were analyzed using Masslynx version 2.1 software.

### Others
Protein was quantitated by Bradford's method (30) with bovine serum albumin as the protein standard. The conventional techniques for manipulation of DNA were those described by Sambrook et al. (31). SDS-PAGE was performed by the method of Laemmli (32).

---

### RESULTS

#### Isolation of a Set of cDNAs Encoding IPT-like Proteins
To pick up plant gene(s) encoding IPT, we screened the genome sequence of *A. thaliana* in silico against the amino acid sequence of an *Agrobacterium* IPT, tmr (17), as queries. Consequently, eight candidates, designated as AtIPT1 to AtIPT8 having significant homology to tmr, were found. At the amino acid level, tmr has 37.3% similarity to AtIPT1, 32.8% to AtIPT2, 40.8% to AtIPT3, 44.7% to AtIPT4, 40.9% to AtIPT5, 41.2% to AtIPT6, 42.2% to AtIPT7, and 43.6% to AtIPT8. Table I summarizes the structural features of the sequences. The AtIPT genes are distributed all over five chromosomes of Arabidopsis. AtIPT2 was equivalent to a sequence registered as Arabidopsis tRNA IPT (GenBank accession numbers AAF00582 and AF109376 for the protein and the mRNA, respectively). Six of the eight (AtIPT1, AtIPT3, AtIPT4, AtIPT6, AtIPT7, and AtIPT8) have been deposited and annotated as “putative” tRNA IPT based on the sequence similarities. One was not annotated as an open reading frame, but we found a possible reading frame on chromosome V, which has homology to other AtIPTs, and designated it AtIPT5.

To obtain the cDNAs, total RNA was prepared, and RT-PCR was performed with specific primers as described under “Experimental Procedures.” Each PCR amplified a specific cDNA fragment with an expected length (data not shown), and the nucleotide sequences of the cDNAs were determined. Fig. 1A shows a sequence comparison of a set of deduced amino acid sequences of AtIPT proteins. The reading frames of AtIPTs deduced from the cDNA sequences consisted of 318–466 amino acids, which have 34.7–60.6% amino acid identities to AtIPT1.### Adenylate Isopentenyltransferase in Arabidopsis

#### FIG. 1. Multiple alignment of amino acid sequence of the predicted translation products of AtIPTs (A) and phylogenetic tree of representative tRNA IPTs and IPTs (B).

**A.**  
Gaps denoted by dashes were inserted to obtain maximum homology. The identical amino acid residues among all AtIPTs are indicated by white letters on a black background, and those with AtIPT1 are hatched. Marked stretches (Regions \(a\) and \(b\)) are discussed in the text.

**B.**  
The tree was generated using the CLUSTALW program at the DNA Data Bank of Japan. Relative branch lengths are approximately proportional to phylogenetic distance.

#### Sequence Alignment (Region \(a\))

| AtIPT1 | AtIPT2 | AtIPT3 | AtIPT4 | AtIPT5 | AtIPT6 | AtIPT7 | AtIPT8 |
|--------|--------|--------|--------|--------|--------|--------|--------|
| MTELNFHLLPIISDRFTTTTTTSPSFSHHSSSSSLLSFTRRRKHQPLVSSIRMEQSRSRKRDKVVVLILGTAGKSR | MMMLNPSNGGIEGEKMKKARKJVVIIMGPQSGKSK | MIMKISMAMCKQPLPPSPTLDFPPARFGPNMLTLN-PYGPKDKVVVIMGATCTGKSR | MKPCMTALRQVIQPLSLNFQGMVDVP-FFRRKDKVVFVMGATCTGKSR | MQQLMTLLSSPPLSHSSLPLPTTKFGSPRLVTTCMGHAGRKNIKDWWLITETTGKSR | MKFSISBLKOVQPILCFKNKLSKVNVNSFLHPKEKVIFVMGATCSGKSR | MQNLITSTFVSPEMIPITSPLRLRLPPERSVVPMTTVCMEQSYRQWVVIMGATTGSGKSC |  |

#### Sequence Alignment (Region \(b\))

| AtIPT1 | AtIPT2 | AtIPT3 | AtIPT4 | AtIPT5 | AtIPT6 | AtIPT7 | AtIPT8 |
|--------|--------|--------|--------|--------|--------|--------|--------|
| LSVDIAFRFP-SEIDNSDKIQVYEGLEITTRQITLQDRRGVPHHLGLVINPEHGELTAGEFRSAASN VKKEITSQKVP | LAVDIAASHFP-VEIDNADAMCIYSGLDVITNKVTVDEQKGVPHHLGLTVS-SDMEFTARDFRDFTVPLEEIVSRNHIPV | LSVEIAAFBR-AEILNEDKIQVHQGLDIVTNKITSEESCGVPHHLGLVLP-PEADLTAANYCHMANLSIESVLNGKLPI | LAIDIALHFK-AEILNEDKMCFYDGLDITROSTIEDRHGVPHHLGLINPEAGEVTAEEFRVMAAEISEITQKKLPI | LAIDIAFRFP-AEILNEDKIQVYKGOLDIVTNKVTPEESIGVPHHLGLTVHDITYEDFTAEDOREAIRAVESIVCQDRWPI | LAIDIAFRFP-AEILNEDKIYKGFEIVTNKLIPHLHEQGVPHHLGLQFHPDOGELTPAEFRSLATLSISKLISSKKLPI | LAIDIAFRFQ-GEILDNSDKICLYKGLOVITNKVTPECEGVPHHLGLVFDSEAGNITATQYSRLASQAISKLSANNKLPI | LSIDIAAKHS-GEILDNDKIQFMDGLKVTRMQMSILERCVPVHHLGLEPRDDSELTSEFRSLABRSISEITARGNILPI |  |

#### Sequence Alignment (Region \(c\))

| AtIPT1 | AtIPT2 | AtIPT3 | AtIPT4 | AtIPT5 | AtIPT6 | AtIPT7 | AtIPT8 |
|--------|--------|--------|--------|--------|--------|--------|--------|
| IAGGSNSFVHALLAQRFD | LVGGTHYIQAVVSKFLLDDAAEDTEECADVASVVDQDMVVESVFGRDDLSHGYYELLKELDPVAANRIHPNNHRKINQY | IVGGSNSYVEALVDD | IAGGSNSYIHALLAKSYD | IAGGSNSYIEALVND | VVGGNSFNHALLAERFD | VAGGSNSYIEALVNH | IAGGSNSFIHALLVDRFD |  |

#### Sequence Alignment (Region \(d\))

| AtIPT1 | AtIPT2 | AtIPT3 | AtIPT4 | AtIPT5 | AtIPT6 | AtIPT7 | AtIPT8 |
|--------|--------|--------|--------|--------|--------|--------|--------|
| PKFDPFSSG--SCLISSDLRYECCFIWWDVSETULYEYLRRUDEMMDSGMFEELSRFYD | LSLHASRGVLPSKLYQGKTAEVDSFLLTSRQNWCINAASRFDYCCLICMEAETAVLDYVECPNDAWMVDAGLLDZVYDIYK | KENKFRR--YDCCLWWDALFVLHGFSERUDKMVVEGEMVEEVREFFD | PENYPFDH--KGSICSELKYDCCFIWDQDSVLFKEYLSIRLIMMKSGMFEEIAEFHR | CVDFLLR--YECCLWWDVSRERVHSFVSERUDKMVDMGLVDEVRRIFD | PIDPFPSFSSLSTICSDLRYKCCILWWDVLEPVLFQHCMLDCMIESGLVEQIAELYD | SSGFLLNN--YDCFITWWDVSLFVLNSFVSKRUDRMHEAGLLEEVREVFH | PKTYPFSS--ETSISGLRYECCFLWWDSVSVSVLFEYLSKRUDOMMESGMFEELAGFYD |  |

#### Sequence Alignment (Region \(e\))

| AtIPT1 | AtIPT2 | AtIPT3 | AtIPT4 | AtIPT5 | AtIPT6 | AtIPT7 | AtIPT8 |
|--------|--------|--------|--------|--------|--------|--------|--------|
| PVKSGLETRF-GIRKAEGVEFDGYFKEYPEKK | PGAD----YTRGLROSIGVREFEDFLKIHLSETCAGHLTSLSNDDKVMKENLRKILNFPKDDKLRIMLEAIDRVKLNTR | FSNEO---YSRGIKKACIFEED----RFFRNEQ----------------------FLNVEDREELLSKYLEEKRTF | SKK--APKEPLGIWKAIGVQEFDDYLKMKWNDN--------------------MDKWDPMRKEAYEKAVRAIKENTF | PSSBD---YSAGIRRAIGVERLD----EFLRSEM--------------------NYPAAETTERLLETAIEKKEKNTC | PVVD----SGRRLGVRTIGVEEFDRYFRVPKEMD--------------------KGIWDLARKAAYEETVKGMKERTC | PKAN----YSVGIRRAICWPELH----EYLKNES--------------------LVDRATKSKMLDVAVKNIKKTE | PRYBOSAIRAHGIHKTGIPFEDRYTSLYPPERK--------------------KMSEWDQARKGAYDEAVOLIKENTW |  |

#### Sequence Alignment (Region \(f\))

| AtIPT1 | AtIPT2 | AtIPT3 | AtIPT4 | AtIPT5 | AtIPT6 | AtIPT7 | AtIPT8 |
|--------|--------|--------|--------|--------|--------|--------|--------|
| TLAKRVKKIEMDKDAG-WEIERVDATAEKKVMMKESS----EKKWRENMEQVLEPSVKIVKRHLVQG | RLLRCKRRVSRLETVFCWNIHYIDATEYILS--------------------------KSEESBNAGVVKPASEJIRCFLETETESGRDPTSG | ELACECREKIERDKVKWSIORVDATPVETKRRS--------------------------KMDANVAERLWAGPSTDTSRFLLDIASRRLPVEAS | QLTKQITKINKLRNAG-WDIKKDATASFREAIRAAKEGEGVAEMQRKINKEVLEPCVKIVRSHLDQPINYYYYYFYL | LLACECLQIQRLYKQWKWNMHVRDATERVELRR--------------------------GEEADEAODNSVAHPSALAWEKFLSYSDDHHLEGANI | RLVKCKEKMKLIRGGEWEIKRLDAIMAELNQSTAK-GEGKNGREIDEKHIVDESVEIVKKFILLEV | ILACECLKKIQRLLKKWKMSMHRVDATEVELKN--------------------------VEEQDEAENLVARPSEIRIVDKFYNNNNQLKNDDEH | RLAKKQIERIMKEXSSG-WDICRLDAIPSEG--------------------------RSSREIJDNTVLDESIKVVKFLVKDKV |  |

#### Sequence Alignment (Region \(g\))

| AtIPT2 | AtIPT3 | AtIPT4 | AtIPT5 | AtIPT6 | AtIPT7 |
|--------|--------|--------|--------|--------|--------|
| KSIERDLWTQYVCEACGNKILRGRHEWEHHKOGRTHRKRTRRHKNSQTYKNREVQEAEVN | TAVAAAMEREELSCLVA | LKRFSLN | LLPEISAWPPLPAVAAISR | CLAASYGGGSGSRAHNMI |  |

#### Phylogenetic Tree

![Phylogenetic Tree](#)

#### Species and Accession Numbers

- **Eukaryotic tRNA-IPT:**
  - *S. cerevisiae* (GenBank™ accession number P07884)
  - *Schizosaccharomyces pombe* (GenBank™ accession number CAB52278)
  - *H. sapiens* (GenBank™ accession number AF074918)
  - *C. elegans* (GenBank™ accession number T27538)

- **Prokaryotic tRNA-IPT:**
  - *Aquifex aeolicus* (GenBank™ accession number G70391)
  - *Borrelia burgdorferi* (GenBank™ accession number AAC-67163)
  - *Richettsia prowazekii* (GenBank™ accession number CAA14962)
  - *Mycobacterium leprae* (GenBank™ accession number S72942)
  - *Streptomyces coelicolor* (GenBank™ accession number T35111)
  - *A. tumefaciens* (GenBank™ accession number P38436)
  - *Deinococcus radiodurans* (GenBank™ accession number AAF11245)
  - *E. coli* (GenBank™ accession number AAC-77128)
  - *Pseudomonas putida* (GenBank™ accession number AAB69443)
  - *Thermotoga maritima* (GenBank™ accession number C72366)
  - *Bacillus subtilis* (GenBank™ accession number G69657)
  - *Chlamydia trachomatis* (GenBank™ accession number AAC-68361)
  - *Synechocystis sp.* PCC 6803 (GenBank™ accession number S75554)

- **Bacterial IPT (green):**
  - *Agrobacterium rhizogenes* pRiA4 (GenBank™ accession number S06738)
  - *A. tumefaciens* pTiC58 (GenBank™ accession number AAA27406)
  - *A. tumefaciens* pTi-SAKURA (17)
  - *Agrobacterium vitis* pTiS4 (GenBank™ accession number S30106)
  - *Pseudomonas syringae* pCK1 (GenBank™ accession number A24937)
  - *Pseudomonas solanacearum* (GenBank™ accession number S06739)
  - *Rhodococcus fascians* pFiD188 (GenBank™ accession number CAA82744)

#### Discussion

AtIPT2 encoded the longest reading frame. Multiple alignment of the AtIPT showed that AtIPT2 contains two inserted regions of about 80 and 20 amino acids. The carboxyl-terminal region of AtIPT2 also had an extra 40 amino acids. Fig. 1B shows a phylogenetic tree based on a comparison with other representative IPT and tRNA IPT sequences. This calculation implies that the divergence of the AtIPT2 gene occurred before that of other AtIPTs and bacterial IPT genes. These results suggest that the primary structure of putative AtIPT proteins, AtIPT1 and AtIPT3 to AtIPT8, are more closely related to bacterial IPT than tRNA IPT.

#### Expression of IPTs in E. coli

The eight cDNAs, AtIPT1 to AtIPT8, were expressed in E. coli. The expression levels and activities of the recombinant proteins were analyzed to confirm their functions as adenylate isopentenyltransferases. Further studies are needed to elucidate the specific roles of these enzymes in plant metabolism and development.
26408 Adenylate Isopentenyltransferase in Arabidopsis

**Figure 2. Excretion of cytokinin from recombinant E. coli harboring IPT genes and the IPT activity of the cell extracts.**

- **A:** Each transformant was cultured in the presence of 1 mM IPTG in modified M9 minimal medium for 4 h. The culture media of the transformants were collected, and the cytokinin fraction was purified and the molecular species determined by ELISA. Values shown are the means of three independent replicates.
- **B:** The culture of *E. coli* strain (ΔrcsC, cps::lacZ) harboring pIN-III-AHK4 (27) was mixed with that of *E. coli* expressing each *AtIPT*. The mixed cells were spotted on Luria agar plates supplemented with X-Gal and incubated at 25°C for 48 h.
- **C:** The crude extract of each transformant cell was used to measure the IPT activity by radioisotope rapid assay. The amount of each sample for assay was equivalent to 1 A<sub>600</sub> unit of cells. One A<sub>600</sub> unit is defined as the amount of cells obtained from 1 ml of cell culture whose A<sub>600</sub> value is 1.

AtIPT8, were expressed in *E. coli* under the control of the trc promoter, which can be driven by IPTG. First, to examine the ability of the expressed protein to synthesize cytokinin *in vivo*, we measured the cytokinin content in the culture medium. As the substrates of IPT, DMAPP, and AMP are provided by authentic metabolism in *E. coli*, we expected the cytokinins to be synthesized and excreted. As a control, *tmr* from *A. tumefaciens* pTi-SAKURA (17) was also expressed in *E. coli*. As shown in Fig. 2A, when *tmr* expression was induced, iP was predominantly accumulated in the culture medium. A small amount of Z was also detected. Expression of AtIPT1 and AtIPT3 to AtIPT8 also caused the accumulation of iP and Z in the media. In the culture of *E. coli* transformants of AtIPT1, AtIPT4, AtIPT7, and AtIPT8, Z content was relatively higher than those of AtIPT3, AtIPT5, and AtIPT6. iPMP, the possible reaction product, and other cytokinin species were below the detectable level in every culture (data not shown). On the other hand, in the culture media of transformants of AtIPT2, no significant accumulation of cytokinin species was detected.

**Figure 3. Purification of AtIPT1 produced in E. coli.** Samples (lanes 1–4, 20 μg; lanes 5 and 6, 5 μg) from various purification stages were subjected to SDS-PAGE. Lane 1, total extract of noninduced *E. coli*; lane 2, total extract of 4-h-induced *E. coli*; lane 3, soluble fraction of the induced *E. coli*; lane 4, supernatant of protamine sulfate precipitation; lane 5, Mono S column chromatography fraction; lane 6, Superdex 200-pg fraction. The gel was stained with Coomassie Brilliant Blue. The molecular masses of marker proteins are indicated in kilodaltons (kDa) on the left.

Using an *E. coli* system, we attempted to examine the cytokinin biosynthesis ability of the transformants of *AtIPT*. In Arabidopsis, a cytokinin receptor, AHK4 (identical to CRE1), has been identified recently (27, 33). The Rcs-phosphorelay system in *E. coli* (RcsC → YojN → RcsB) is involved in extracellular polysaccharide synthesis by activating the *cps* operon (27). In the *E. coli* strain having the ΔrcsC and *cps::lacZ* genetic background, AHK4 can function as a cytokinin-responsive sensory His-kinase through activating the *E. coli* YojN → RcsB → *cps::lacZ* pathway, thereby giving rise to blue colonies in the presence of external cytokinin and X-Gal (27). When each of the transformants of *AtIPT* was mixed with that of *AHK4* and grown in the presence of X-Gal, all those except for that of *AtIPT2* turned blue without externally added cytokinin (Fig. 2B). This was well consistent with the result shown in Fig. 2A.

To confirm the ability of the gene products to synthesize cytokinin, IPT activity was measured by the radioisotope rapid assay with the total extract of the *E. coli* transformants (Fig. 2C). Although the extent was different, IPT activity was detected in all extracts of the transformants of *AtIPTs* except for *AtIPT2*. These results suggest that the gene products of *AtIPTs* other than *AtIPT2* could synthesize cytokinin species in *E. coli* cells.

**Purification and Characterization of Recombinant AtIPTs—** To confirm further the catalytic reaction of the recombinant protein, the AtIPT1 polypeptide was purified as described under “Experimental Procedures.” SDS-PAGE of the final preparation showed it to be apparently homogeneous (Fig. 3, lane 6). The catalytic activity of IPT was determined by a nonradioisotope assay.

Purified AtIPT1 was incubated with AMP and DMAPP, and the products were loaded on HPLC (Fig. 4). The retention time of a peak of one of the reaction products (Peak A) was identical to that of a peak of iPMP (Fig. 4, A and B). When the reaction products were treated with alkaline phosphatase, the retention time of Peak A was shifted to that of Peak B whose retention time coincided with that of iPA (Fig. 4, A and C). Peaks A and B had an absorbance spectrum identical to that of iPMP and iPA, respectively (data not shown). To identify Peak B, mass spectrometry analysis was performed. Consequently, the mass of the product coincided with the iPA standard (Figs. 4, D and E). These results clearly demonstrated that recombinant AtIPT1 catalyzes the IPT reaction.

**Kinetic Parameters of AtIPT1—** The recombinant enzyme of AtIPT enabled us to analyze the kinetic parameters (Table II). The specific activity of AtIPT1 was 57 milliunits/mg of protein.

(Fig. 4). While all AtIPT sequences had been annotated as putative tRNA IPTs in the *A. thaliana* Annotation Database, our results demonstrated that all AtIPTs except for AtIPT2 have IPT activity.

Although AtIPTs did not show close similarity to tRNA IPTs and bacterial IPTs at the amino acid level (detailed alignment is not shown), some common structural features were found. The putative motif for DMAPP binding (34), which is similar to the ATP/GTP-binding motif at the amino-terminal region ((A, G)-X₄-G-K-(S, T); Fig. 1A, Region a), was conserved in both types of isopentenyltransferase. In tRNA IPT in *E. coli*, some nucleotides such as GTP, ATP, and CTP inhibit the activity in a competitive manner with respect to DMAPP (34). In this study, nucleotides such as ATP and ADP strongly inhibited the IPT activity (Table II), suggesting that these nucleotides inhibit the activity by competitive access to the DMAPP-binding site and that the energy status in the cell is involved in the regulation of the IPT activity. Another structural feature is that the carboxyl-terminal extension of AtIPT2 contained a putative zinc finger-like motif (C-X₂-C-X₁₂,₁₈-H-X₅-H; Fig. 1A, Region b), which is conserved in eukaryotic tRNA IPT such as *H. sapiens* (10), *Caenorhabditis elegans* (GenBank™ accession number T27538), and others. The motif is also found in the murine RNA-binding protein ZFR (35) and thought to play an important role in the expression and/or the retention of the activity of eukaryotic tRNA IPTs (10, 36, 37). The absence of IPT activity of AtIPT2 and the structural similarities between AtIPT2 and tRNA IPTs are well consistent with that AtIPT2 is registered as tRNA IPT.

The cytokinin species excreted from the *E. coli* transformants of AtIPTs did not coincide with that determined by chemical analysis of the *in vitro* reaction product (Figs. 2 and 4). This discrepancy of detected products between the culture medium and *in vitro* reaction is attributed to the metabolism of cytokinins in *E. coli* cells. Namely, iPMP synthesized by AtIPTs is metabolized to iP and Z and excreted to the culture medium. Nonpolar compounds such as iP and Z are expected to penetrate easily across the cell membrane. In fact, the *E. coli* transformants expressing AtIPT1 and AtIPT3 to AtIPT8 had a growth rate significantly slower than the AtIPT2 (data not shown) probably due to metabolic depletion of DMAPP in the *E. coli*. The tendency was more remarkable in those expressing AtIPT1, AtIPT4, AtIPT7, and AtIPT8 (data not shown), which excreted Z into the medium. Further metabolism of synthesized iPMP to iP and Z by the authentic dephosphorylation, hydroxylation, and deribosylation systems occur in the *E. coli* cells.

The existence of isoforms of AtIPT leads us to speculate the differentiation of the physiological function of each isoform in terms of the expression site and the regulatory manner. For instance, cytokinin has been suggested to be synthesized in restricted sites in which cell proliferation is active (19–21). In terms of gene regulation, iPMP has been shown to rapidly accumulate in roots in response to nitrate replenishment to the nitrogen-depleted maize (29). These data imply that the cytokinin biosynthesis genes are expressed differently in spatially and temporally specific areas and in response to various environmental stimuli. Further comparative analysis of the expression pattern of each gene should help elucidate the physiological function.

Recently, an alternative pathway for cytokinin biosynthesis has been proposed by Åstot et al. (38). They provided evidence that IPT could use an unknown compound of terpenoid origin as a side chain donor instead of DMAPP, and the initial product of the pathway is trans-zeatin 5′-monophosphate. When the possible donor compounds become available, we need to determine

---

### FIGURE 4: Identification of Reaction Products of Assay with Purified Recombinant AtIPT1
Patterns of elution of iPMP and iPA (A), reaction products of the nonradioisotope assay with AtIPT1 (B), dephosphorylated products after treatment with alkaline phosphatase (C) are shown. iPA standard (D) and Peak B fraction (E) were subjected to mass spectrometry.

---

### TABLE II: Enzymatic Properties of AtIPT1

| Property                          | Value           |
|-----------------------------------|-----------------|
| **Kinetic parameters**            |                 |
| $K_m$ for AMP ($\mu$M)            | $185 \pm 19^b$  |
| $K_m$ for DMAPP ($\mu$M)          | $50 \pm 3$      |
| Specific activity (units/mg protein) | $5.7 \times 10^{-2}$ |
| pH optimum                         | $8.0$           |
| **Effect of nucleotides**         |                 |
| + ATP (1 mM)                      | $4\%{^d}$       |
| + ADP (1 mM)                      | $4\%{^d}$       |
| + GTP (1 mM)                      | $8\%{^d}$       |
| + GDP (1 mM)                      | $29\%{^d}$      |

#### Footnotes:
- $^a$ Assay was performed by a nonradioisotope method.
- $^b$ Mean ± S.E. ($n = 3$).
- $^c$ Assay was performed by a radioisotope rapid method.
- $^d$ Relative activity with the value in the absence of compounds taken as 100.

---

and the $K_m$ values for AMP and DMAPP were 185 and 50 $\mu$M, respectively. The $K_m$ value for AMP of AtIPT1 was much higher than that of ipt1 (85.7 nM) in *A. tumefaciens* (18). Adenine, adenosine, and isopentenylpyrophosphate were not utilized as the substrates (data not shown). On the other hand, ATP, GTP, ADP, and GDP strongly inhibited the IPT activity. The optimum pH was around 8.0, and there was no activity at pH 6 (data not shown). Due to the unstableness of the other recombinant AtIPTs, the yields of the purified preparation were quite low (data not shown). The enzymatic property of them could not be determined.

---

### DISCUSSION

In this study, we identified genes encoding IPT, a cytokinin biosynthesis enzyme, in *A. thaliana*. The identity of the cDNA was established by determination of the catalytic activity of the translated products *in vivo* and *in vitro* (Fig. 2) and by chemical determination of the reaction product by mass spectrometry.

mine whether AtIPT can catalyze the alternative reaction to elucidate the biochemical entity of the alternative cytokinin biosynthesis pathway.

**Acknowledgments**—We deeply thank Dr. K. Suzuki and Dr. K. Yoshida for kindly providing us with pTi-SAKURA. We are also grateful to Dr. T. Mizuno for his advice and for providing the *E. coli* strain and pIN-III-AHK4.

**REFERENCES**

1. Mok, M. C. (1994) in *Cytokinins: Chemistry, Activity, and Function* (Mok, D. W. S., and Mok, M. C., eds) pp. 155–166, CRC Press, Boca Raton, FL
2. Shudo, K. (1994) in *Cytokinins: Chemistry, Activity, and Function* (Mok, D. W. S., and Mok, M. C., eds) pp. 35–42, CRC Press, Boca Raton, FL
3. Swaminathan, S., and Bock, R. M. (1977) *Biochemistry* **16**, 1355–1360
4. Bartz, J. K., and Soll, D. (1972) *Biochemie (Paris)* **54**, 31–39
5. Rosenbaum, N., and Gifter, M. L. (1972) *J. Biol. Chem.* **247**, 5675–5680
6. Caillet, J., and Droogmans, L. (1988) *J. Bacteriol.* **170**, 4147–4152
7. Kline, L. K., Fittler, F., and Hall, R. H. (1969) *Biochemistry* **8**, 4361–4371
8. Dihanich, M. E., Najarian, D., Clark, R., Gillman, E. C., Martin, N. C., and Hopper, A. K. (1987) *Mol. Cell. Biol.* **7**, 177–184
9. Holtz, J., and Klämbt, D. (1975) *Hoppe-Seyler's Z. Physiol. Chem.* **356**, 1459–1464
10. Golovko, A., Hjälms, G., Sitbon, F., and Nicander, B. (2000) *Gene (Amst.)* **258**, 85–93
11. Holtz, J., and Klämbt, D. (1978) *Hoppe-Seyler's Z. Physiol. Chem.* **359**, 89–101
12. Klämbt, D. (1992) in *Physiology and Biochemistry of Cytokinins in Plants* (Kaminek, M., Mok, D. W. S., and Zazímalová, E., eds) pp. 25–27, SPB Academic Publishing, The Hague
13. Akiyoshi, D. E., Klee, H., Amasino, R. M., Nester, E. W., and Gordon, M. P. (1984) *Proc. Natl. Acad. Sci. U. S. A.* **81**, 5994–5998
14. Powell, G. K., and Morris, R. O. (1986) *Nucleic Acids Res.* **14**, 2555–2565
15. Crespi, M., Messens, E., Caplan, A. B., van Montagu, M., and Desomer, J. (1992) *EMBO J.* **11**, 795–804
16. Lichter, A., Barash, I., Valinsky, L., and Manulis, S. (1995) *J. Bacteriol.* **177**, 4457–4465
17. Suzuki, K., Ohta, N., Hattori, Y., Uraji, M., Kato, A., and Yoshida, K. (1998)

**Biochim. Biophys. Acta** **1396**, 1–7

18. Blackwell, J. R., and Horgan, R. (1993) *Phytochemistry* **34**, 1477–1481
19. Feldman, L. J. (1975) in *The Development and Function of Roots* (Torrey, J. G., and Clarkson, D. T., eds) pp. 55–72, Academic Press, London
20. Blackwell, J. R., and Horgan, R. (1994) *Phytochemistry* **35**, 339–342
21. Koda, Y., and Okazawa, Y. (1980) *Physiol. Plant.* **49**, 193–197
22. Chen, C.-M., and Melitz, D. K. (1979) *FEBS Lett.* **107**, 15–20
23. Chen, C.-M., and Ertl, J. R. (1994) in *Cytokinins: Chemistry, Activity, and Function* (Mok, D. W. S., and Mok, M. C., eds) pp. 81–85, CRC Press, Boca Raton, FL
24. Fujiwara, T., Yokota-Hirai, M., Chino, M., Komeda, Y., and Naito, S. (1992) *Plant Physiol.* **99**, 263–268
25. Wadsworth, G. J., Redinbaugh, M. G., and Scandalios, J. G. (1988) *Anal. Biochem.* **172**, 279–283
26. Sanger, F., Nicklen, S., and Coulson, A. R. (1977) *Proc. Natl. Acad. Sci. U. S. A.* **74**, 5463–5467
27. Suzuki, T., Miwa, K., Ishikawa, K., Yamada, H., Aiba, H., and Mizuno, T. (2001) *Plant Cell Physiol.* **42**, 107–113
28. Masui, Y., Coleman, J., and Inoue, M. (1983) in *Experimental Manipulation of Gene Expression* (Inoue, M., ed) pp. 15–32, Academic Press, New York
29. Takei, K., Sakakibara, H., Taniguchi, M., and Sugiyama, T. (2001) *Plant Cell Physiol.* **42**, 85–93
30. Bradford, M. M. (1976) *Anal. Biochem.* **72**, 248–254
31. Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) *Molecular Cloning: A Laboratory Manual*, 2nd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
32. Laemmli, U. K. (1970) *Nature* **227**, 680–685
33. Inoue, T., Higuchi, M., Hashimoto, Y., Seki, M., Kobayashi, M., Kato, T., Tabata, S., Shinozaki, K., and Kakimoto, T. (2001) *Nature* **409**, 1060–1063
34. Leung, H.-C. E., Chen, Y., and Winkler, M. E. (1997) *J. Biol. Chem.* **272**, 13073–13083
35. Meagher, M., Schumacher, J. M., Lee, K., Holdcraft, R. W., Edelhoff, S., Disteche, C., and Braun, R. E. (1999) *Gene (Amst.)* **228**, 197–211
36. LaCasse, E. C., and Lefebvre, Y. A. (1995) *Nucleic Acids Res.* **23**, 1647–1656
37. Chong, S., Curnow, A. W., Huston, T. J., and Garcia, G. A. (1995) *Biochemistry* **34**, 3695–3701
38. Astot, C., Dolezal, K., Nordström, A., Wang, Q., Kunkel, T., Moritz, T., Chua, N.-H., and Sandberg, G. (2000) *Proc. Natl. Acad. Sci. U. S. A.* **97**, 14778–14783
