
# Characterization of a Three-Component Vanillate O-Demethylase from *Moorella thermoacetica*

**DEVENDRA NAIDU AND STEPHEN W. RAGSDALE***  
*Department of Biochemistry, Beadle Center, University of Nebraska—Lincoln, Lincoln, Nebraska 68588-0664*

Received 23 October 2000/Accepted 13 March 2001

The *Moorella thermoacetica* aromatic O-demethylase was characterized as an inducible three-component system with similarity to the methanogenic methanol, methylamine, and methanethiol methyltransferases and to the O-demethylase system from *Acetobacterium dehalogenans*. MtvB catalyzes methyl transfer from a phenylmethyl ether to the cobalt center of MtvC, a corrinoid protein. MtvA catalyzes transmethylation from MtvC to tetrahydrofolate, forming methyltetrahydrofolate. Cobalamin can substitute for MtvC.

---

Constituting about 25% of the earth’s biomass (18), lignin is a polymer composed of hydroxylated and methoxylated phenylpropanoid units linked by C–C and C–O–C bonds that undergo depolymerization to generate monomeric methoxy-lated aromatics (or phenylmethyl ethers) like vanillate and syringate. The anaerobic metabolism of phenylmethyl ethers (11) has been known since 1979 (13). Although acetogenic bacteria can be selectively isolated by growth on methoxylated aromatics like syringate (2), they do not metabolize the aromatic ring itself but use the O-methyl group as a one-carbon growth substrate (equation 1). Oxidation of one methyl group to CO₂ provides the six electrons required for conversion of three methyl groups to acetate. *Moorella thermoacetica* can metabolize at least 20 different methoxylated aromatics (5). A 22-kDa protein and aromatic O-demethylase activity are induced when *M. thermoacetica* is exposed to such compounds (6, 37).

(1)

4 vanillic acid (C₈H₈O₄) + 2 CO₂ →  
3 CH₃COOH + 2 H₂O + 4 protocatechuic acid (C₇H₆O₄)


The aromatic O-demethylases of *Acetobacterium woodii* (3), *Acetobacterium dehalogenans* (27), and *M. thermoacetica* (8) require H₄folate as the methyl acceptor, producing CH₃-H₄folate (8, 15, 27). Oxidation of one equivalent of CH₃-H₄folate to CO₂ and H₄folate generates six electrons (equation 2) which drive the synthesis of three equivalents of acetyl-coenzyme A (acetyl-CoA) by the Wood-Ljungdahl pathway of acetyl-CoA synthesis (Fig. 1 and reaction 3). This paper describes a three-component aromatic O-demethylase from *M. thermoacetica*.

(2)

CH₃-H₄folate + 2 H₂O →  
CO₂ + H₄folate + 6 electrons + 6 H⁺


(3)

CH₃-H₄folate + CO₂ + CoASH +  
2 H⁺ + 2 electrons → CH₃-CO-SCoA + H₄folate + H₂O


---

## MATERIALS AND METHODS

### Bacterial strains and growth conditions.
*M. thermoacetica* was cultivated as described earlier (1) but included a vitamin mixture (36). Dicamba (2 mM) was added at an optical density at 600 nm (OD₆₀₀) of 0.6. Cells were harvested at an OD₆₀₀ of 2.0 using a CEPA continuous centrifuge. The cells were frozen in liquid nitrogen and stored at −80°C.

### Analytical methods.
Corrinoid concentrations were measured by conversion to the dicyano derivative (23) using the extinction coefficients at 580 and 367 nm of \(10.13 \times 10^3 \, \text{M}^{-1} \, \text{cm}^{-1}\) and \(30.82 \times 10^3 \, \text{M}^{-1} \, \text{cm}^{-1}\). Metals were determined by plasma emission spectroscopy (14). Protein concentrations were determined by the Rose Bengal method (9). Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) was done by the method of Laemmli (20), and the gels were stained as described by Blum et al. (4).

Electron paramagnetic resonance (EPR) spectra were recorded on a Bruker ESP 300E spectrometer equipped with an Oxford ITC4 temperature controller, a model 5340 automatic frequency counter (Hewlett-Packard), and a Bruker gaussmeter. The molecular mass of MtvC was determined by electrospray ionization mass spectroscopy using a Micromass Platform II instrument (Manchester, United Kingdom). N-terminal sequences were determined by automated N-terminal Edman degradation using an ABI-494 Precise Sequencer after transferring the purified protein (30 nmol) from the SDS-PAGE gel to a polyvinylidene difluoride membrane. The protein band was stained with 0.1% Amido Black, excised, and sequenced.

### Enzymatic assays.
(i) **O-demethylase assay.**
O-demethylase activity was measured using 3 mM dicamba or 1 mM vanillate as substrate in a 400-μl reaction mixture at 55°C in the anaerobic chamber. The reaction mixture contained 50 mM 2-(N-morpholino)ethanesulfonate (MES) buffer (pH 6.6), 2 mM ATP, 3 mM tetrahydrofolate, 4 mM titanium (III) citrate, 5 mM MgCl₂, 5 mM NaCl, and enzyme components. The assay was quenched after 1 h by adding 0.2 M perchloric acid (final) and being centrifuged at 10,000 × g for 2 min, and substrates and products were quantified by thin-layer chromatography (TLC) and/or reverse-phase high-performance liquid chromatography (HPLC). For TLC analysis, after being quenched and centrifuged, the samples were extracted with diethyl ether (1:1 [vol/vol]), the organic phase was evaporated, and the residue was dissolved in 40 μl of diethyl ether. This solution was then chromatographed on a silica gel AL SIL G/UV (Whatman Ltd.) plate using the organic phase of a toluene-acetic acid-water (6:7:3, vol/vol/vol) mixture as mobile phase. Products were visualized with a UV lamp and by being sprayed with 1% FeCl₃. The \(R_f\) values of the aromatics under these conditions were as follows: dicamba, 0.7; dichlorosalicylate, 0.54; vanillate, 0.72; catechol, 0.3; and protocatechuic acid, 0.05. For HPLC, samples were quenched and centrifuged as above, and then the supernatant was neutralized with NaOH and a sample (30 to 50 μl) was injected onto a μ-Bondapak C18 3.5- by 150-mm column (Waters, Milford, Mass.). To assay dicamba metabolism, the column was developed with a linear gradient from 0 to 60% methanol in 50 mM KPᵢ buffer, pH 7.7. The 280-nm absorbance of dicamba and the 420-nm fluorescence emission (excitation wavelength, 310 nm) of dichlorosalicylate were monitored. For quantitation of vanillate and protocatechuic acid, the column was developed with a linear gradient from 0 to 60% methanol in 125 mM sodium acetate, pH 4.5 (solvent B), and the elution profile was monitored at 254 nm.

(ii) **Vanillate:hydroxocobalamin methyl transfer assay (MtvB).**
The vanillate-dependent methylation of hydroxocobalamin was performed in the dark at 55°C.

---

* Corresponding author. Mailing address: Department of Biochemistry, Beadle Center, University of Nebraska—Lincoln, Lincoln, NE 68588-0664. Phone: (402) 472-2943. Fax: (402) 472-7842. E-mail: sragsdale1@unl.edu.

#### Figure 1: Scheme Describing the Conversion of the Aromatic O-Methyl Group to the Methyl Group of Acetic Acid

![Scheme](https://i.imgur.com/yourimageurl.png)

**FIG. 1.** Scheme describing the conversion of the aromatic O-methyl group to the methyl group of acetic acid. MtvA, -B, and -C are the components of the O-demethylase system. Abbreviations: CFSP, the corrinoid iron-sulfur protein; MeTr, the CFSP methyltransferase; ACS, acetyl-CoA synthase.

---

#### Polyacrylamide Gel Electrophoresis of Purified MtvA, MtvB, and MtvC

![Gel Electrophoresis](https://i.imgur.com/yourimageurl2.png)

**FIG. 2.** Polyacrylamide gel electrophoresis of purified MtvA, MtvB, and MtvC. Proteins were visualized by silver staining. The gels contained 3 μg of MtvA, 2 μg of MtvB, and 6 μg of MtvC.

---

### Text Content

#### Inside the Anaerobic Chamber
The reaction mixture contained 0.3 μmol of hydroxocobalamin, 0.4 μmol of vanillate, 0.8 μmol of ATP, 1.6 μmol of titanium (III) citrate, 2 μmol of MgCl₂, and 2 μM NaCl in 50 mM MES buffer, pH 6.6 (final volume, 0.4 ml). The reaction was initiated by adding 0.21 nmol of MtvB. The reaction mixture was incubated for 16 h and quenched and processed for TLC analysis as described above.

#### (iii) Methylcobalamin:tetrahydrofolate Methyl Transfer Assay (MtvA)
The methylcobalamin-dependent methylation of tetrahydrofolate was monitored spectrophotometrically in the dark at 55°C by a modification of a previously described procedure (12). The reaction was carried out in a 0.4-cm-path-length cuvette capped with a rubber stopper to maintain an anaerobic atmosphere. The 0.75-ml reaction mixture contained 15 nmol of methylcobalamin, 112 nmol of tetrahydrofolate, 3.75 nmol of NaCl, and 0.36 nmol of MtvA in 50 mM MES buffer, pH 6.8. The reaction was initiated with H₄folate and monitored at 525 and 540 nm. In the absence of MtvA, there was no demethylation of methylcobalamin. The demethylation rate was calculated using an extinction coefficient for methylcobalamin at 540 nm of 7.7 mM⁻¹ cm⁻¹ (19).

#### (iv) Reduction of the Corrinoid Protein
Dithionite stock solutions (~1.1 M) were prepared in 0.2 M NaOH. The purified corrinoid protein was reduced chemically by adding 1 mM sodium dithionite to an argon-purged quartz cuvette containing 0.12 mg of corrinoid protein per ml in buffer A (25 mM KPi buffer, pH 7.6). The absorption spectrum was recorded 5 min after adding dithionite to the sample.

#### Purification of the Mtv O-Demethylase Components

##### (i) Separation of the Mtv Components
Protein purification was performed at 15°C in an anaerobic chamber (Coy Laboratory Products, Ann Arbor, Mich.) under an atmosphere of 90% nitrogen and 10% hydrogen. During purification, the O-demethylase activity was routinely determined by TLC analysis. Fifty grams (wet weight) of cells was sonicated in 100 ml of solution containing buffer A, 0.5 mg of lysozyme per ml, 2 mM dithiothreitol, and 2.5% glycerol. The sonicate was centrifuged at 30,000 × g for 90 min in a Type 35 rotor (Beckman Instruments Inc.).

The cell extract was heat treated at 70°C for 20 min in a water bath and centrifuged at 10,000 × g for 15 min. This step gave a modest increase in specific activity but appeared to stabilize the enzyme toward further purification. The supernatant was then subjected to ammonium sulfate fractionation at 4°C. The dicamba O-demethylase activity precipitated at 40 to 70% saturation. After being centrifuged at 10,000 × g for 20 min at 4°C, the 70% pellet was solubilized in 1.2 M ammonium sulfate in buffer A and fractionated by phenyl Sepharose chromatography, which resolved the O-demethylase activity into three components: MtvA, MtvB, and MtvC. To purify each component, the assays contained two components from the phenyl Sepharose fractionation step plus column fractions that were being screened for the third component. The three components were purified to homogeneity (Fig. 2).

The 40 to 70% ammonium sulfate fraction (above) was applied to a 2.5- by 25-cm phenyl Sepharose CL-6B (Pharmacia LKB) column equilibrated with the same solution. A linear gradient (1,200 ml) from 1.2 M to 0.3 M ammonium sulfate was run. Then, the column was washed with 5 column volumes of buffer A and further washed with 3 mM KPᵢ buffer, pH 7.6. The dicamba O-demethylase activity separated into three components, MtvA, MtvB, and MtvC. MtvC (450 mg/265 ml) eluted at 0.94 M ammonium sulfate, MtvA (436 mg/220 ml) eluted at approximately 0.7 M, and MtvB (484 mg/213 ml) eluted in the final 3 mM KPᵢ buffer wash. Active fractions were pooled and stored at 4°C.

##### (ii) MtvA Purification
The MtvA fraction from phenyl Sepharose was concentrated to 3 ml by ultrafiltration with a YM10 membrane (Amicon Division, Beverly, Mass.) and applied to a 1.5- by 68-cm Sephacryl S-100 (Amersham Pharmacia Biotech AB) gel filtration column equilibrated with buffer A containing 50 mM KCl. Active MtvA fractions (282 mg/15 ml) were pooled, applied to a 2.5- by 7-cm DEAE-Sepharose (Sigma Chemical Co., St. Louis, Mo.) column that was equilibrated with 50 mM KCl in buffer A and developed with a linear gradient from 50 to 375 mM KCl in buffer A. MtvA eluted at 130 mM KCl. The pooled fractions (12 mg/42 ml) were diluted threefold with buffer A and applied to a 2.5- by 3-cm hydroxyapatite fast-performance liquid chromatography (Calbiochem, La Jolla, Calif.) column. A linear gradient (60 ml) of 5 to 200 mM KPᵢ buffer, pH 7.6, was applied. MtvA eluted at approximately 70 mM KPᵢ. Active fractions were pooled (4.6 mg/10 ml) and stored at 4°C.

##### (iii) MtvC Purification
The phenyl Sepharose fraction containing MtvC was diluted 10-fold with buffer A containing 125 mM KCl and loaded on a 2.5- by 7-cm DEAE-Sepharose column equilibrated with the same buffer. A 150-ml linear gradient from 125 to 350 mM KCl was applied. Fractions with MtvC activity eluted at approximately 270 mM KCl. The active fractions were pooled (245 mg/53 ml) and concentrated to 3 ml using a YM10 membrane. This sample was then applied to a 1.5- by 68-cm Sephacryl S-100 gel filtration column equilibrated with buffer A containing 50 mM KCl. Fractions were collected at a flow rate of 0.2 ml/min. Active MtvC fractions were pooled (8.3 mg/13 ml) and diluted 10-fold with 2 mM KPᵢ buffer, pH 7.6. Active fractions were pooled and applied to a 2.5- by 3-cm hydroxyapatite high-performance column equilibrated with 5 mM KPᵢ buffer, pH 7.6, which was developed with a 100-ml linear gradient from 0 to 100 mM KPᵢ buffer, pH 7.6. MtvC activity eluted at approximately 30 mM KPᵢ. Active MtvC fractions were pooled (3.0 mg/17 ml) and stored at 4°C.

##### (iv) MtvB Purification
The MtvB pool from the phenyl Sepharose column was loaded on a 2.5- by 7-cm DEAE-Sepharose column that had been equilibrated with buffer A. In a linear gradient (400 ml) of 0 to 400 mM KCl, MtvB eluted at 125 mM KCl. The pooled fractions (220 mg/97 ml) were then concentrated with a YM10 ultrafiltration membrane to 3 ml and loaded on a 1.5- by 68-cm Sephacryl S-100 gel filtration column equilibrated with buffer A containing 50 mM KCl. Active fractions were pooled (55 mg/12 ml) and diluted 10-fold with 20 mM Tris-HCl buffer, pH 7.6. This sample was loaded on a 2.5- by 3-cm hydroxyapatite column equilibrated with the same buffer, and a 150-ml linear gradient from 0 to 200 mM KPᵢ buffer, pH 7.6, was applied. Active MtvB fractions eluting at 90 mM KPᵢ were pooled (3.6 mg/12 ml) and stored at 4°C.

---

**RESULTS AND DISCUSSION**

Dicamba demethylation by *M. thermoacetica*. *M. thermoacetica* cells were grown anaerobically with glucose and dicamba to yield approximately 55 g (wet weight) of cells. In the absence of glucose, the cells grew slowly, with a low yield. Glucose did not appear to repress the $O$-demethylase activity. Dicamba conversion to dichlorosalicylate was monitored by TLC and HPLC, and cells were harvested anaerobically when ~85% of the dicamba was metabolized.

Besides metabolizing dicamba, extracts from cells grown on dicamba could demethylate vanillate to protocatechuate and further decarboxylate protocatechuate to catechol under standard assay conditions. Similarly, extracts of cells grown in the presence of vanillate could demethylate dicamba to dichlorosalicylate as well as convert vanillate to protocatechuate and catechol (not shown). Daniel et al. also found that *M. thermoacetica* cells grown on different methoxylated aromatic compounds catalyze the $O$-demethylation of the same particular methoxylated compound, suggesting the presence of a broad specificity of $O$-demethylase (6). As shown below, the enzyme from dicamba-grown cells utilizes vanillate much more efficiently than dicamba. To conform to the nomenclature used by Krzycki and Thauer to describe the components of the methanogenic methyltransferases involved in demethylation of methylamines, methylsulfides, and pterins (reviewed in reference 32), we designate this as the Mtv system, i.e., methyltransferase for vanillate. The reaction catalyzed by the Mtv system of *M. thermoacetica* is most similar to that of the four-component system from *A. dehalogenans* (17).

Enzymatic properties of the $O$-demethylase.

(i) Overall reaction requirement of MtvA, MtvB, and MtvC. The three components required for $O$-demethylation of dicamba and vanillate were resolved and purified to homogeneity. Omission of MtvA, -B, or -C from the $O$-demethylase reaction mixture results in complete loss of $O$-demethylase activity; thus, all three protein components are required for dicamba $O$-demethylase activity. Cell extracts that were bubbled with air for 0.5 h and then with nitrogen for 15 min (so that $O_2$ was not introduced into the assay mixture) retained full $O$-demethylase activity. Therefore, the reaction, which involves Co(I), is $O_2$ sensitive, not the enzyme. The demethylation reaction requires a reductant. Although dithionite can reduce Co(II) to Co(I), it inhibits the $O$-demethylase. On the other hand, Ti(III) citrate reduces Co(II) and drives the $O$-demethylase reaction. The optimal temperature is 55°C and the optimal pH is 6.6. As described above, H₄folate is required as the acceptor of the methyl group from vanillate. The reaction rate is linear for up to 2 h at 55°C.

(ii) Steady-state kinetics. The apparent $K_m$ values for dicamba and vanillate are 9.0 mM and 85 μM, respectively, and the $V_{\text{max}}$ values with dicamba and vanillate as the methyl donors are 276 and 900 nmol mg⁻¹ h⁻¹, respectively (Fig. 3). The 350-fold (based on relative $V/K$ values) preference for vanillate is striking, since the cells were grown on dicamba. These results prompted us to name the enzyme vanillate, instead of dicamba, $O$-demethylase. The $K_m$ value for H₄folate is 0.23 mM.

As has been found for some other methyltransferases, hydroxocobalamin or methylcobalamin could substitute for MtvC, which allowed dissection of the roles of the individual components of the Mtv system. In the first partial reaction, MtvB is the only protein required to catalyze the vanillate- or dicamba-

FIG. 4. UV-visible spectra of purified MtvC (0.45 mg/ml).

#### Figure 5: EPR Spectra of the Co(II) in As-Isolated MtvC

**Figure Description:**
The spectrum of 90 μM MtvC in 25 mM KP<sub>i</sub> buffer, pH 7.6, was recorded using the following EPR conditions:
- Temperature: 100 K
- Gain: \(10^4\)
- Power: 100 mW
- Modulation amplitude: 10 G
- Microwave frequency: 9.47 GHz
- Scans: 16

The coupling constants calculated were:
- \(A_{z(\text{Co})} = 105\) G
- \(A_{z(\text{N})} = 20\) G

The inset was recorded under identical conditions, except that the temperature was 85 K.

---

#### Figure 6: Methylcobalamin-Tetrahydrofolate Methyl Transfer Catalyzed by MtvA

**Figure Description:**
The reaction was performed at 55°C under anaerobic conditions as described in Materials and Methods. Spectral changes occurring during the methyl group transfer from methylcobalamin (20 μM) to H<sub>4</sub>folate (140 μM) were recorded at 0, 5, 10, 20, 40, and 60 min after addition of H<sub>4</sub>folate.

---

### Text Content

#### Dependent Methylation of Cob(I)alamin
In the second partial reaction, MtvA is the only enzyme required for the methylcobalamin-dependent methylation of H<sub>4</sub>folate.

#### MtrB, MtvB: A Vanillate:Cobalamin Methyltransferase
The function of MtvB was established by studying an O-demethylase partial reaction: transfer of the O-methyl group of vanillate to the Co(I) state of B<sub>12</sub> to form methylcobalamin. MtvA and H<sub>4</sub>folate were not required for the demethylation of vanillate; B<sub>12</sub> was absolutely required (in the absence of MtvC). TLC analysis of the partial reaction showed protocatechuic acid formation as the concentration of vanillate decreased, demonstrating that MtvB participates in the first part of the overall O-demethylase reaction, transferring the methyl group from the methoxylated aromatic substrate to cobalamin. The enzymatic function of MtvB is similar to that of the 36-kDa component B from *A. dehalogenans* (15, 17).

#### Purified MtvB
Purified MtvB is a homodimer with an apparent monomeric molecular mass of 48 kDa. Like MtvA, MtvB is colorless and exhibits the typical UV-visible spectrum of a protein lacking chromophoric prosthetic groups. Metals, including zinc and iron, were found in less than stoichiometric amounts (<0.1 g-atm/mol). To assess the possibility that this is a Zn protein, we determined the effect of Zn addition on the demethylation of vanillate. Even at concentrations as high as 4 mM, there was no stimulation of activity. Either this is not a zinc protein or reconstitution requires different conditions. In methionine synthase and the methyltransferases that use coenzyme M, zinc is involved in coordinating and thus activating the thiol substrate (reviewed in references 25 and 32). Zinc is also found in the methyltransferases involved in the transfer of the methyl group of methanol to cobalamin (MtaB) (31).

#### MtvC: A Corrinoid Protein
Homogeneous preparations of MtvC contain a single subunit with an apparent molecular mass of 27 kDa based on SDS analysis and 22 kDa based on mass spectrophotometric analysis. Presumably, this is the 22-kDa protein identified by Daniel et al. to be induced when *M. thermoacetica* cells are exposed to syringate (6). Functionally, this enzyme is most similar to the 26-kDa corrinoid protein of *A. dehalogenans* called component A (15, 17). *A. dehalogenans* also contains a fourth component (called component C), which is an activating protein that couples the ATP-dependent reduction of Co(II) to Co(I) (17). The *M. thermoacetica* O-demethylase system was assayed using an artificial electron donor [Ti(III) citrate]; as was found with *A. dehalogenans*, with this assay the reductive activation system was not required. The N-terminal sequence of the protein, MLTDTL(S)KAMA ELEE EQ(V)LA, is not significantly similar to the sequence of the *A. dehalogenans* corrinoid protein (component A) (16).

The UV-visible spectrum (Fig. 4) of the reddish brown as-isolated protein is similar to that of Co(II) corrinoids, with a major absorption peak at 474 nm (22). MtvC was calculated to contain 0.8 mol of cobalamin per mol of protein. Cobalt is the only metal found in stoichiometric amounts. There were variable amounts of zinc and iron at levels of <0.2 g-atm per mol of protein.

Reduction of MtvC with 1 mM Ti(III) citrate or dithionite bleaches the 474-nm peak of cob(II)alamin as an absorption peak forms at 388 nm, which is characteristic of cob(I)alamin. However, the Co(I) state of MtvC, when generated with dithionite, was unable to demethylate dicamba or vanillate in the presence of MtvA or MtvB. Dithionite also inhibited the demethylation reaction in the standard assay. Therefore, Ti(III) citrate (4 mM) concentration was routinely used in the demethylation assays.

The EPR spectrum of purified as-isolated MtvC (Fig. 5) is

---

### Figure 5: EPR Spectra of the Co(II) in As-Isolated MtvC

**Figure Description:**
The spectrum of 90 μM MtvC in 25 mM KP<sub>i</sub> buffer, pH 7.6, was recorded using the following EPR conditions:
- Temperature: 100 K
- Gain: \(10^4\)
- Power: 100 mW
- Modulation amplitude: 10 G
- Microwave frequency: 9.47 GHz
- Scans: 16

The coupling constants calculated were:
- \(A_{z(\text{Co})} = 105\) G
- \(A_{z(\text{N})} = 20\) G

The inset was recorded under identical conditions, except that the temperature was 85 K.

---

### Figure 6: Methylcobalamin-Tetrahydrofolate Methyl Transfer Catalyzed by MtvA

**Figure Description:**
The reaction was performed at 55°C under anaerobic conditions as described in Materials and Methods. Spectral changes occurring during the methyl group transfer from methylcobalamin (20 μM) to H<sub>4</sub>folate (140 μM) were recorded at 0, 5, 10, 20, 40, and 60 min after addition of H<sub>4</sub>folate.

---

### Text Content

#### Dependent Methylation of Cob(I)alamin
In the second partial reaction, MtvA is the only enzyme required for the methylcobalamin-dependent methylation of H<sub>4</sub>folate.

#### MtrB, MtvB: A Vanillate:Cobalamin Methyltransferase
The function of MtvB was established by studying an O-demethylase partial reaction: transfer of the O-methyl group of vanillate to the Co(I) state of B<sub>12</sub> to form methylcobalamin. MtvA and H<sub>4</sub>folate were not required for the demethylation of vanillate; B<sub>12</sub> was absolutely required (in the absence of MtvC). TLC analysis of the partial reaction showed protocatechuic acid formation as the concentration of vanillate decreased, demonstrating that MtvB participates in the first part of the overall O-demethylase reaction, transferring the methyl group from the methoxylated aromatic substrate to cobalamin. The enzymatic function of MtvB is similar to that of the 36-kDa component B from *A. dehalogenans* (15, 17).

#### Purified MtvB
Purified MtvB is a homodimer with an apparent monomeric molecular mass of 48 kDa. Like MtvA, MtvB is colorless and exhibits the typical UV-visible spectrum of a protein lacking chromophoric prosthetic groups. Metals, including zinc and iron, were found in less than stoichiometric amounts (<0.1 g-atm/mol). To assess the possibility that this is a Zn protein, we determined the effect of Zn addition on the demethylation of vanillate. Even at concentrations as high as 4 mM, there was no stimulation of activity. Either this is not a zinc protein or reconstitution requires different conditions. In methionine synthase and the methyltransferases that use coenzyme M, zinc is involved in coordinating and thus activating the thiol substrate (reviewed in references 25 and 32). Zinc is also found in the methyltransferases involved in the transfer of the methyl group of methanol to cobalamin (MtaB) (31).

#### MtvC: A Corrinoid Protein
Homogeneous preparations of MtvC contain a single subunit with an apparent molecular mass of 27 kDa based on SDS analysis and 22 kDa based on mass spectrophotometric analysis. Presumably, this is the 22-kDa protein identified by Daniel et al. to be induced when *M. thermoacetica* cells are exposed to syringate (6). Functionally, this enzyme is most similar to the 26-kDa corrinoid protein of *A. dehalogenans* called component A (15, 17). *A. dehalogenans* also contains a fourth component (called component C), which is an activating protein that couples the ATP-dependent reduction of Co(II) to Co(I) (17). The *M. thermoacetica* O-demethylase system was assayed using an artificial electron donor [Ti(III) citrate]; as was found with *A. dehalogenans*, with this assay the reductive activation system was not required. The N-terminal sequence of the protein, MLTDTL(S)KAMA ELEE EQ(V)LA, is not significantly similar to the sequence of the *A. dehalogenans* corrinoid protein (component A) (16).

The UV-visible spectrum (Fig. 4) of the reddish brown as-isolated protein is similar to that of Co(II) corrinoids, with a major absorption peak at 474 nm (22). MtvC was calculated to contain 0.8 mol of cobalamin per mol of protein. Cobalt is the only metal found in stoichiometric amounts. There were variable amounts of zinc and iron at levels of <0.2 g-atm per mol of protein.

Reduction of MtvC with 1 mM Ti(III) citrate or dithionite bleaches the 474-nm peak of cob(II)alamin as an absorption peak forms at 388 nm, which is characteristic of cob(I)alamin. However, the Co(I) state of MtvC, when generated with dithionite, was unable to demethylate dicamba or vanillate in the presence of MtvA or MtvB. Dithionite also inhibited the demethylation reaction in the standard assay. Therefore, Ti(III) citrate (4 mM) concentration was routinely used in the demethylation assays.

The EPR spectrum of purified as-isolated MtvC (Fig. 5) is

MtvA                      1 MLIIIGEKLNSAIPSVRQIIN  
Component D               1 MIIGEKINGAIPATAKAI A  
AcSE                      1 MLIIGERINGMFGDIKRAIQ  
consensus                 1 MlIIIGEkiNgaip vrkaIn  

FIG. 7. Comparison of N-terminal sequences of MtvA and AcSE (the corrinoid iron-sulfur protein methyltransferase) from *M. thermoacetica* and component D from *A. dehalogenans*.

characteristic of a Co(II) corrinoid (29). The $g_{\|}$ resonance is split into eight lines by hyperfine interactions between the unpaired electrons and the cobalt nucleus, which has a nuclear spin, $I$, of 7/2. The strength of the interaction is described by the hyperfine coupling constant, $A_{Z(\text{Co})} \approx 105$ G. Each of these hyperfine lines is split into a triplet superhyperfine splitting pattern [$A_{Z(N)} \approx 20$ G] (Fig. 5, inset) that is characteristic of corrinoid proteins containing a coordinated axial nitrogen ligand. This ligand could be a histidine residue from the protein, as in methionine synthase, methanogenic corrinoid proteins involved in methyl transfer, and some adenosylcobalamin enzymes (reviewed in reference 24). Proteins that have the histidine ligand generally contain a conserved motif, DxHxG-(41-42)-gxSxL-(21/22)-GG (24), although there are exceptions (32). Component A of the *A. dehalogenans* O-demethylase contains this "his" binding motif (16). Alternatively, the ligand could be from the benzimidazole group appended to the corrin ring, as in ribonucleotide reductase (21) and diol dehydrase (35).

**Properties of MtvA, a methylcobalamin: H₄folate methyltransferase.** MtvA was found to catalyze the H₄folate-dependent demethylation of methyl-cob(III)alamin, which was followed photometrically at 525 nm (Fig. 6). The 525-nm peak decreased as a peak at 475 nm formed, which is characteristic of cob(II)alamin. Based on the published extinction coefficient for methylcobalamin ($\varepsilon_{540} = 7.70 \, \text{mM}^{-1} \, \text{cm}^{-1}$ [19]), the rate of demethylation is $72 \, \text{nmol min}^{-1} \, (\text{mg of MtvA})^{-1}$. Under the conditions of the assay, which lacked reductant, cob(I)alamin is rapidly oxidized to cob(II)alamin. MtvA showed no activity in the vanillate- or dicamba-dependent methylation of cob(I) alamin.

Functionally, MtvA is similar to the *A. dehalogenans* component D (15, 17). The reaction of MtvA (equation 4) is also analogous to that of AcSE (equation 5), the CH₃-H₄folate- and corrinoid-dependent methyltransferase involved in transferring the methyl group of the CH₃-H₄folate to the corrinoid iron-sulfur protein in the Wood-Ljungdahl pathway (33, 34). However, MtvA is unable to substitute for AcSE, and AcSE cannot substitute for MtvA. Similarly, in methanogens, the cognate corrinoid proteins and the methyltransferases involved in dimethylsulfide, trimethylamine, dimethylamine, and monomethylamine metabolism (28) are discrete gene products that cannot substitute for each other (10, 26, 28).

MtvA: methyl-Co-MtvC +  
H₄folate → CH₃-H₄folate + MtvC   (4)

AcSE: CH₃-H₄folate + Co-CFeSP →  
methyl-Co-CFeSP + H₄folate   (5)

Based on SDS-PAGE analysis, MtvA is a 29.5-kDa protein. Similarly, component D from *A. dehalogenans* and AcSE from *M. thermoacetica* have monomeric molecular masses of 30 kDa (15, 17) and 28.6 kDa (30), respectively. The UV-visible absorption spectrum of MtvA does not reveal any chromophores other than the amino acids. AcSE also lacks prosthetic groups (30).

The N-terminal sequence of MtvA is highly similar to that of the CH₃-H₄folate- and cobalamin-dependent methyltransferase from *M. thermoacetica* (AcSE) (Fig. 7), which folds into a TIM barrel structure containing the pterin binding site within the barrel (7). Given the strong sequence homology among the N termini of MtvA, AcSE, and component D, perhaps the O-demethylases adopt a similar fold.

---

**REFERENCES**

1. Andreesen, J. R., A. Schaupp, C. Neurater, A. Brown, and L. G. Ljungdahl. 1973. Fermentation of glucose, fructose, and xylose by *Clostridium thermoceticum*: effect of metals on growth yield, enzymes, and the synthesis of acetate from CO₂. J. Bacteriol. **114**:743–751.
2. Bache, R., and N. Pfennig. 1981. Selective isolation of *Acetobacterium woodii* on methoxylated aromatic acids and determination of growth yields. Arch. Microbiol. **130**:255–261.
3. Berman, M. H., and A. C. Frazer. 1992. Importance of tetrahydrofolate and ATP in the anaerobic O-demethylation reaction for phenylmethylethers. Appl. Environ. Microbiol. **58**:925–931.
4. Blum, H., H. Meier, and H. J. Gross. 1987. Improved silver staining method of plant proteins, RNA and DNA in polyacrylamide gels. Electrophoresis **8**:93–98.
5. Daniel, S. L., E. S. Keith, H. Yang, Y. S. Lin, and H. L. Drake. 1991. Utilization of methoxylated aromatic compounds by the acetogen *Clostridium thermoaceticum*: expression and specificity of the co-dependent O-demethylating activity. Biochem. Biophys. Res. Commun. **180**:416–422.
6. Daniel, S. L., Z. Wu, and H. L. Drake. 1988. Growth of thermophilic acetogenic bacteria on methoxylated aromatic acids. FEMS Microbiol. Lett. **52**:25–28.
7. Doukov, T., J. Seravalli, J. Stezowski, and S. W. Ragsdale. 2000. Crystal structure of a methyltetrahydrofolate and corrinoid dependent methyltransferase. Structure **8**:817–830.
8. El Kasmi, A., S. Rajasekharan, and S. W. Ragsdale. 1994. Anaerobic pathway for conversion of the methyl group of aromatic methyl ethers to acetic acid by *Clostridium thermoaceticum*. Biochemistry **33**:11217–11224.
9. Elliott, J. I., and J. M. Brewer. 1978. The inactivation of yeast enolase by 2,3-butanedione. Arch. Biochem. Biophys. **190**:351–357.
10. Ferguson, D. J., J. A. Krzycki, and D. A. Grahame. 1996. Specific roles of methylcobamide:coenzyme M methyltransferase isozymes in metabolism of methanol and methylamines in *Methanosarcina barkeri*. J. Biol. Chem. **271**:5189–5194.
11. Frazer, A. C. 1994. O-demethylation and other transformations of aromatic compounds by acetogenic bacteria, p. 445–483. In H. L. Drake (ed.), Acetogenesis. Chapman and Hall, New York, N.Y.

12. Goulding, C. W., D. Postigo, and R. G. Matthews. 1997. Cobalamin-dependent methionine synthase is a modular protein with distinct regions for binding homocysteine, methyltetrahydrofolate, cobalamin, and adenosylmethionine. Biochemistry **36**:8082–8091.

13. Healy, J. B., Jr., and L. Y. Young. 1979. Anaerobic biodegradation of eleven aromatic compounds to methane. Appl. Environ. Microbiol. **38**:84–89.

14. Jones, J. B., Jr. 1977. Elemental analysis of soil extracts and plant tissue by plasma emission spectroscopy. Commun. Soil Sci. Plant Anal. **8**:349–365.

15. Kaufmann, F., G. Wohlfarth, and G. Diekert. 1997. Isolation of O-demethylase, an ether-cleaving enzyme system of the homoacetogenic strain MC. Arch. Microbiol. **168**:136–142.

16. Kaufmann, F., G. Wohlfarth, and G. Diekert. 1998. O-demethylase from *Acetobacterium dehalogenans*—cloning, sequencing, and active expression of the gene encoding the corrinoid protein. Eur. J. Biochem. **257**:515–521.

17. Kaufmann, F., G. Wohlfarth, and G. Diekert. 1998. O-demethylase from *Acetobacterium dehalogenans*—substrate specificity and function of the participating proteins. Eur. J. Biochem. **253**:706–711.

18. Kirk, T. K., and R. L. Farrell. 1987. Enzymatic “combustion”: the microbial degradation of lignin. Annu. Rev. Microbiol. **41**:465–505.

19. Kreft, J. U., and B. Schink. 1994. O-demethylation by the homoacetogenic anaerobe *Holophaga foetida* studied by a new photometric methylation assay using electrochemically produced cob(I)alamin. Eur. J. Biochem. **226**:945–951.

20. Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature (London) **227**:680–685.

21. Lawrence, C. C., G. J. Gerfen, V. Samano, R. Nitsche, M. J. Robins, J. Retey, and J. Stubbe. 1999. Binding of Cob(II)alamin to the adenosylcobalamin-dependent ribonucleotide reductase from *Lactobacillus leichmannii*. Identification of dimethylbenzimidazole as the axial ligand. J. Biol. Chem. **274**:7039–7042.

22. Lexa, D., J.-M. Savéant, and J. Zickler. 1977. Electrochemistry of Vitamin B₁₂. 2. Redox and acid-base equilibria in the B₁₂a/B₁₂r system. J. Am. Chem. Soc. **99**:2786.

23. Ljungdahl, L. G., J. LeGall, and J.-P. Lee. 1973. Isolation of a protein containing tightly bound 5-methoxybenzimidazolycobamide (Factor IIIm) from *Clostridium thermoaceticum*. Biochemistry **12**:1802–1808.

24. Ludwig, M. L., and R. G. Matthews. 1997. Structure-based perspectives on B-12-dependent enzymes. Annu. Rev. Biochem. **66**:269–313.

25. Matthews, R. G., and C. W. Goulding. 1997. Enzyme-catalyzed methyl transfers to thiols: the role of zinc. Curr. Opin. Chem. Biol. **1**:332–339.

26. Maupin-Furlow, J., and J. G. Ferry. 1996. Characterization of the *cdhD* and *cdhE* genes encoding subunits of the corrinoid/iron-sulfur enzyme of the CO dehydrogenase complex from *Methanosarcina thermophila*. J. Bacteriol. **178**:340–346.

27. Meßmer, M., S. Reinhardt, G. Wohlfarth, and G. Diekert. 1996. Studies on methyl chloride dehalogenase and O-demethylase in cell extracts of the homoacetogen strain MC based on a newly developed coupled enzyme assay. Arch. Microbiol. **165**:18–25.

28. Paul, L., D. J. Ferguson, and J. A. Krzycki. 2000. The trimethylamine methyltransferase gene and multiple dimethylamine methyltransferase genes of *Methanosarcina barkeri* contain in-frame and read-through amber codons. J. Bacteriol. **182**:2520–2529.

29. Pilbrow, J. R. 1982. EPR of B₁₂-dependent enzyme reactions and related systems, p. 431–462. In D. Dolphin (ed.), Vitamin B₁₂, vol. 1. John Wiley & Sons, New York, N.Y.

30. Roberts, D. L., S. Zhao, T. Doukov, and S. W. Ragsdale. 1994. The reductive acetyl coenzyme A pathway: sequence and heterologous expression of active methyltetrahydrofolate:corrinoid/iron-sulfur protein methyltransferase from *Clostridium thermoaceticum*. J. Bacteriol. **176**:6127–6130.

31. Sauer, K., and R. K. Thauer. 1997. Methanol:coenzyme M methyltransferase from *Methanosarcina barkeri*—Zinc dependence and thermodynamics of the methanol:cob(I)alamin methyltransferase reaction. Eur. J. Biochem. **249**:280–285.

32. Sauer, K., and R. K. Thauer. 1999. The role of corrinoids in methanogenesis, p. 655–680. In R. Banerjee (ed.), Chemistry and biochemistry of B₁₂, vol. 1. John Wiley & Sons, New York, N.Y.

33. Seravalli, J., R. K. Shoemaker, M. J. Sudbeck, and S. W. Ragsdale. 1999. Binding of (6R,S)-methyltetrahydrofolate to methyltransferase from *Clostridium thermoaceticum*: role of protonation of methyltetrahydrofolate in the mechanism of methyl transfer. Biochemistry **38**:5736–5745.

34. Seravalli, J., S. Zhao, and S. W. Ragsdale. 1999. Mechanism of transfer of the methyl group from (6S)-methyltetrahydrofolate to the corrinoid/iron-sulfur protein catalyzed by the methyltransferase from *Clostridium thermoaceticum*: a key step in the Wood-Ljungdahl pathway of acetyl-CoA synthesis. Biochemistry **38**:5728–5735.

35. Shibata, N., J. Masuda, T. Tobimatsu, T. Toraya, K. Suto, Y. Morimoto, and N. Yasuoka. 1999. A new mode of B12 binding and the direct participation of a potassium ion in enzyme catalysis: X-ray structure of diol dehydratase. Struct. Fold Des. **7**:997–1008.

36. Wu, Z. R., S. L. Daniel, and H. L. Drake. 1988. Characterization of a CO-dependent O-demethylating enzyme system from the acetogen *Clostridium thermoaceticum*. J. Bacteriol. **170**:5747–5750.
