
Chemistry & Biology

ELSEVIER

Chemistry & Biology 8 (2001) 817–829

www.elsevier.com/locate/chembiol

Research Paper

Analysis of the prodigine biosynthesis gene cluster of Streptomyces coelicolor A3(2): new mechanisms for chain initiation and termination in modular multienzymes

Ana M. Cerdeño¹, Mervyn J. Bibb², Gregory L. Challis*

Departments of Molecular Microbiology and Genetics, John Innes Centre, Norwich Research Park, Colney, Norwich NR4 7UH, UK

Received 20 March 2001; revisions requested 18 May 2001; revisions received 12 June 2001; accepted 19 June 2001

First published online 10 July 2001

---

### Abstract

**Background:** Prodigines are a large family of pigmented oligopyrrole antibiotics with medicinal potential as immunosuppressants and antitumour agents that are produced by several actinomycetes and other eubacteria. Recently, a gene cluster in *Streptomyces coelicolor* encoding the biosynthesis of undecylprodigine and butyl-meta-cycloheptylprodigine has been sequenced.

**Results:** Using sequence comparisons, functions have been assigned to the majority of the genes in the cluster, several of which encode homologues of enzymes involved in polyketide, nonribosomal peptide, and fatty acid biosynthesis. Based on these assignments, a complete pathway for undecylprodigine and butyl-meta-cycloheptylprodigine biosynthesis in *S. coelicolor* has been deduced. Gene knockout experiments have confirmed the deduced roles of some of the genes in the cluster.

**Conclusions:** The analysis presented provides a framework for a general understanding of the genetics and biochemistry of prodigine biosynthesis, which should stimulate rational approaches to the engineered biosynthesis of novel prodigines with improved immunosuppressant or antitumour activities. In addition, new mechanisms for chain initiation and termination catalysed by hitherto unobserved domains in modular multienzyme systems have been deduced. © 2001 Elsevier Science Ltd. All rights reserved.

Keywords: Antibiotic; Fatty acid synthase; Modular polyketide synthase; Non-ribosomal peptide synthetase; *Streptomyces*

---

### 1. Introduction

Prodigines are a family of red-pigmented antibiotics produced by actinomycetes and other eubacteria, in particular species of *Serratia* [1]. The actinomycete prodigines include the linear tripyrrole undecylprodigine 1 and several isomeric, cyclic derivatives such as butyl-meta-cycloheptylprodigine 2, ethyl-meta-cyclononylprodigine 3, and methylcyclodecylprodigine 4 (Fig. 1) [2–4]. Although these antibiotics are not currently used clinically because of their toxicity, the recent finding that they are potent immunosuppressants at non-toxic doses has stimulated renewed interest [5–9]. The antitumour antibiotics BE185915 and roseophilin 6 (Fig. 1), isolated from *Streptomyces sp.* BA18591 and *Streptomyces griseoviridis* respectively, can also be considered as members of the prodigine family [10, 11]. Their cytotoxicity, and indeed the cytotoxicity of prodigines in general, may arise from their ability to bind to DNA and cause double-strand cleavage in the presence of molecular oxygen and copper(II) ions [12, 13]. Because of the unusual structures and potent biological activities of the macrocyclic prodigines (e.g. 2–4 and 6) they have also attracted attention recently as targets for total synthesis. The main challenge of these studies has been efficient construction of the unfunctionalised, ansa-bridged macrocycles of these metabolites [14–25].

The biosynthesis of prodigiosin (7, Fig. 1) by *Serratia marcescens* and of several actinomycete prodigines has been examined by the incorporation of labelled precursors

---

1 Current address: The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.

2 Current address: Diversa Corporation, 4955 Directors Place, San Diego, CA 92121, USA.

* Corresponding author. Current address: Department of Chemistry, University of Warwick, Coventry CV4 7AL, UK.  
E-mail address: g.l.challis@warwick.ac.uk (G.L. Challis).

1074-5521/01/$ – see front matter © 2001 Elsevier Science Ltd. All rights reserved.  
PII: S1074-5521(01)00054-0

CHBIOL 119 9-8-01  Cyaan Magenta Geel Zwart

Fig. 1. Structures of some prodiginines produced by actinomycetes 1–6 and the structure of prodigiosin 7.

[26–28]. These studies demonstrated that undecylprodigine is derived from one unit of proline, one unit of glycine, one unit of serine and several units of acetate, via a convergent pathway involving condensation of 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde (8, Fig. 2) and 2-undecylpyrrole (9, Fig. 2) at a late stage. The macrocyclic prodigines appear to be derived from undecylprodigine by oxidative cyclisation (Fig. 2).

A red-pigmented antibiotic produced by Streptomyces coelicolor A3(2) was first discovered during genetic studies

of actinorhodin biosynthesis [29]. It was subsequently shown to be a mixture of predominantly undecylprodigine 1 and butyl-meta-cycloheptylprodigine 2 in about a 2:1 ratio [30]. In early genetic studies, mutants defective in prodigine biosynthesis that had been generated by UV irradiation were grouped into five classes (redA–E) according to their cosynthesis behaviour [31]. Mapping of these mutations to the S. coelicolor chromosome indicated that they were clustered. Subsequently, a gene encoding an O-methyltransferase required for a late step in prodigine

Fig. 2. The biosynthetic origin of undecylprodigine and three cyclic derivatives deduced from the incorporation of labelled precursors.

biosynthesis was cloned by complementation of the redE mutation [32]. This was followed by the cloning of other biosynthetic genes covering > 21 kb of the chromosome by complementation of further classes of red mutants, eventually leading to the cloning of the entire prodigiosine biosynthesis cluster and its expression in a heterologous host [33, 34]. These studies provided the foundations for more recent work aimed at elucidating the regulation of prodigiosine production in S. coelicolor, which is growth phase-dependent [35–44].

Recently, the sequence of the entire red cluster for prodigiosine biosynthesis became available as part of the S. coelicolor genome-sequencing project (http://www.sanger.ac.uk/Projects/S_coelicolor). In this paper we describe the organisation of the red cluster and assign functions to most of the gene products by sequence comparisons. Among the proteins of assigned function, there are homologues of those involved in the biosynthesis of a diverse array of natural products including fatty acid synthases [45], type I modular polyketide synthases [46], nonribosomal peptide synthetases [47], and α-oxoamine synthases—key enzymes in porphyrin and biotin biosynthesis [48, 49]. The assignments have led to the deduction of a complete pathway for undecylprodigiosine and butyl-meta-cycloheptylprodigiosine biosynthesis in S. coelicolor, which is supported by gene deletion experiments.

with the map generated from the sequence data limits the right-hand end of the cluster. It seems unlikely, however, that the rightmost gene of the cloned insert in pIJ941 (i.e., the rightmost open reading frame (ORF) in Fig. 3) is involved in prodigiosine biosynthesis since it appears to constitute part of a separate operon with the downstream gene, which is not contained in the plasmid.

Two of the 23 genes in the cluster (redD and redZ, orange in Fig. 3) have been shown previously to encode pathway-specific regulators [36, 40, 42]. Of the remaining 21 genes, six are assigned to 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde biosynthesis (red in Fig. 3), eight are assigned to 2-undecylpyrrole biosynthesis (blue in Fig. 3), and two are assigned as housekeeping genes (green in Fig. 3). BLAST and PROSITE searches were used to assign biosynthetic functions for the products of these genes on the basis of their similarity to database proteins (Table 1) [50, 51]. Comparison of the chromosomal location of the genes assigned to 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde and 2-undecylpyrrole biosynthesis with the regions of the cluster previously deduced to be involved in the biosynthesis of these two moieties by complementation and cosynthesis tests [52] shows excellent agreement. The proteins encoded by the remaining five genes in the cluster (white in Fig. 3) showed no similarity to proteins of known function and their role, if any, in prodigiosine biosynthesis is currently unknown.

### 2. Results and discussion

#### 2.1. Organisation of the prodigiosine biosynthetic gene cluster

The red cluster consists of 23 genes organised into four transcription units (J. White and M.J. Bibb, in preparation) (Fig. 3). The left limit of the red cluster is defined by the trkA operon that encodes a potassium uptake system, which is not essential for prodigiosine production but which can influence titres in liquid media (A.M. Cerdeño and M.J. Bibb, unpublished observations). Comparison of the restriction map for pIJ941, which causes copious prodigiosine production when introduced into S. parvulus [34],

#### 2.2. Genes for 2-undecylpyrrole biosynthesis

Incorporation experiments with Streptomyces longisporus ruber have shown that the undecyl side chain and three of the ring carbon atoms constituting the 2-undecylpyrrole moiety of undecylprodigiosine are derived from seven acetates condensed head-to-tail, while the remaining ring carbon atom and the nitrogen atom are derived from glycine (Fig. 2) [27]. In conjunction with incorporation experiments into prodigiosines from other actinomycetes, these results led Gerber et al. to suggest that 2-undecylpyrrole is biosynthesised via condensation of β-ketomyristoyl coenzyme A and glycine with concomitant loss of carbon dioxide from the latter [28].

![Figure 3](https://example.com/image.png)

Fig. 3. Organisation of the prodigiosine biosynthetic gene cluster in S. coelicolor A3(2). Genes deduced to be involved in 2-undecylpyrrole biosynthesis are blue, genes deduced to be involved in 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde biosynthesis are red, putative housekeeping genes are green, regulatory genes are in orange and genes of unknown function are white. Black arrows illustrate the four mRNA molecules likely to be generated by transcription of the cluster. The regions of the cluster spanned by the cosmids SC2E9, SCF7 and SC10A5 from the S. coelicolor ordered cosmid library are indicated in grey (nucleotide sequence accession numbers: SC2E9: AL021530; SC3F7: AL021409; SC10A5: AL021529).

In line with these suggestions, the red cluster contains the genes *redP*, *redR* and *redQ*, which encode homologues of the *Escherichia coli* KASIHI and KASII enzymes [45], and an ACP, respectively (Table 1). Thus, RedP could generate an acetoacetyl thioester attached to RedQ. The carbonyl group would then be reduced and dehydrated to give butyryl ACP by the ketoreductase, dehydratase and enoylreductase enzymes of the S. coelicolor fatty acid synthase (FAS; no genes encoding these enzymes are present in the cluster, nor have they yet been identified elsewhere in the genome). RedR would subsequently extend butyryl ACP to a longer chain fatty acid, again with concomitant reduction and dehydration of the β-ketoacyl ACP intermediates by the reductive enzymes from the endogenous FAS. While it is possible that the final product of RedR could be β-ketomyristoyl ACP, this is considered unlikely for two reasons. Firstly, comparison of the residues in the substrate-binding pocket of RedR with those of KASII from *E. coli*, the structure of which has recently been determined with the irreversible inhibitor cerulenin bound [53], revealed no significant differences. Since *E. coli* KASII can catalyse the chain extension of acyl thioesters up to 16 carbons in length and mutations in the substrate-binding pocket residues are required to restrict the specificity to extension of only shorter substrates [54], it would seem unlikely that RedR has an appreciably different chain length specificity from KASII. Secondly, it is hard to imagine a mechanism whereby the reduction of the keto group of β-ketomyristoyl ACP by the ketoreductase from the endogenous FAS can be prevented, while shorter substrates are readily reduced. Thus, we suggest that the main role of RedP, RedQ and RedR is to provide a pool of fatty acyl ACPs from which dodecyl ACP is specifically selected for initiation of 2-undecylpyrrole biosynthesis.

The synthesis of the requisite β-ketomyristoyl ACP for prodigine biosynthesis is probably accomplished instead by a complex of the multienzymes encoded by the *redX* and *redN* genes. RedX consists of two contiguous ketosynthase domains showing 34% and 27% identities in 501 and 498 amino acid overlaps, respectively, to the ketosynthase (KS) domain in the first module of the myxothiazol PKS MtaB [55]. In addition, the two domains show 29% identity over a 480 amino acid overlap to each other. While the C-terminal ketosynthase domain in RedX (KS) contains an active site Cys 727 residue for covalent attachment of the acyl chain that undergoes chain extension, in the N-terminal domain (KSD) this residue is replaced by Asp 221. RedN consists of two contiguous ACP domains at its N-terminus, both of which contain the requisite serine residues (Ser 47 and Ser 145) for attachment of a phosphopantetheinyl arm, and a C-terminal glycyl transferase domain, which is 43% identical over a 355 amino acid overlap to the 2-amino-3-ketobutyrate synthase of *Bacillus subtilis* [56].

The organisation of RedX is reminiscent of the KSα/KSβ pair of enzymes encoded within type II polyketide synthase gene clusters from many actinomycetes [57], which are responsible for the iterative synthesis of the ACP-bound poly-β-keto precursors of aromatic polyketide antibiotics. In the KSα enzyme, an active site cysteine is present, whereas in the KSβ enzyme the cysteine residue is replaced by glutamine. The latter enzyme has been suggested to play a role in determining the chain length of the polyketide product [58] and, in addition, has recently been shown to act as a malonyl ACP decarboxylase, providing acetyl ACP for initiation of polyketide synthesis [59]. By analogy with bacterial type II iterative FASs, where the ketosynthase enzyme forms a homodimer, it has been suggested that KSα and KSβ form a heterodimer [60, 61].

---

**Table 1**

Deduced functions of gene products in the red cluster

| Gene   | Amino acids | Highest protein sequence similarity                          | Putative function                     |
|--------|-------------|-------------------------------------------------------------|---------------------------------------|
| *redF* | 226         | Prosite entry PS 50244                                      | oxidoreductase                        |
| *redG* | 395         | PrnD*P. fluorescens*                                        | dioxygenase                           |
| *redH* | 873         | PpsA *Synechocystis marinus*                                | phototransferase                      |
| *redI* | 362         | TcmO*S. glaucescens*                                       | O-methyltransferase                   |
| *redJ* | 280         | ORF 3 *Saccharopolyspora erythrea*                         | thioesterase                          |
| *redK* | 347         | Cdh *Nocardia sp.*                                         | oxidoreductase                        |
| *redL* | 2297        | multiple domains<sup>a</sup>                               | type I polyketide synthase            |
| *redM* | 532         | PltF*P. fluorescens*                                       | prolyl-PCP synthetase                 |
| *redN* | 640         | multiple domains<sup>a</sup>                               | pyrrolinone synthase                  |
| *redO* | 87          | FkbJ *Streptomyces hygroscopicus*                          | peptidyl carrier protein              |
| *redP* | 335         | FabH*E. coli*                                              | β-ketoacyl-ACP synthase III           |
| *redQ* | 81          | AcpP *Sinorhizobium meliloti*                              | acyl carrier protein                  |
| *redR* | 407         | FabF*E. coli*                                              | β-ketoacyl-ACP synthase II            |
| *redU* | 261         | JadM *Streptomyces venezuelae*                             | phosphopantetheinyl transferase       |
| *redZ* | 217         | [36, 42]                                                   | transcriptional regulator             |
| *redW* | 391         | PltE*P. fluorescens*                                       | prolyl-PCP dehydrogenase             |
| *redX* | 982         | MtaB *Stigmatella aurantiaca*                              | β-ketomyristoyl-ACP synthase          |
| *redD* | 350         | [36, 40]                                                   | transcriptional regulator             |

<sup>a</sup> For sequence homologies of the domains within these multienzymes and their deduced functions see the text in the relevant section.

where KS<sub>α</sub> catalyses the iterative condensation of malonyl ACP with the growing polyketide chain and KS<sub>β</sub> helps to ‘sense’ the length of the polyketide chain. This suggestion is supported by the X-ray crystal structure of the *E. coli* KASI enzyme with cerulenin covalently bound, which shows that the substrate-binding pocket adjacent to each active site in the homodimer is constructed of residues from both monomers [53,62]. A similar structural organisation of RedX can be envisaged where the two domains interact in an intramolecular manner by virtue of the intervening linker, which joins the domains to form a single protein. It is interesting to note, however, that the order of the KS and KS<sup>D</sup> domains in RedX is opposite to that of the genes encoding KS<sub>α</sub> and KS<sub>β</sub> in all known type II PKS gene clusters and that the domains show higher sequence homology to the KS domains of type I modular PKSs than type II ketosynthases.

By analogy to the type II KS<sub>α</sub>/KS<sub>β</sub> enzymes, we suggest that RedX specifically detects a dodecyl chain bound to the ACP RedQ and catalyses its condensation with a malonyl group attached to one or both of the N-terminal ACP domains of RedN to generate β-ketomyristoyl ACP. This β-ketoacyl thioester could be resistant to reduction by the ketoreductase from the endogenous FAS, because the ketoreductase may be unable to interact with RedN, thus circumventing the problems associated with direct synthesis of β-ketomyristoyl ACP by RedP, RedQ and RedR, discussed above. The ketosynthase and ACP encoded by the nodulation genes *nodE* and *nodF*, respectively, probably perform an analogous role in the biosynthesis of conjugated, polyunsaturated fatty acids that form the side chains of Nod factors in species of *Rhizobium* [63]. In this system, it has been proposed that NodE catalyses the condensation of fatty acyl thioesters of specified lengths that are synthesised by the endogenous FAS, with NodF-bound malonate. The resulting β-ketoacyl thioesters can undergo multiple rounds of ketoreduction and dehydration, but not enoylreduction, because NodE is unable to interact productively with the enoylreductase

from the endogenous FAS. It is interesting to contrast these biosynthetic systems, where an intermediate length fatty acyl ACP appears to be diverted from the endogenous fatty acid synthase into a specific biosynthetic pathway, with the mechanisms for priming of biosynthetic pathways by fatty acyl thioesters in other systems. For example, aflatoxin biosynthesis in species of *Aspergillus* utilises a dedicated type I FAS with altered chain length specificity that synthesises a hexanoyl ACP-bound starter unit, which is transferred directly onto the active site cysteine of the KS domain in the type I iterative aflatoxin PKS [64].

Subsequent cleavage of the β-ketomyristoyl chain from RedN, via pyridoxal-phosphate-mediated condensation of glycine with the ACP-bound thioester followed by decarboxylation of the resulting α-amino-β-keto acid, would yield 4-keto-2-undecylpyrrole (7, Fig. 4) after spontaneous cyclisation and aromatisation. This reaction, which bears mechanistic similarity to the biosynthesis of δ-aminolevulinic acid (an intermediate in porphyrin biosynthesis) [65], from glycine and succinyl CoA, would be catalysed by the C-terminal domain of RedN (Fig. 4).

To establish the involvement of *redX* (and the adjacent *redW* and *redY*) in prodiginine biosynthesis, the genes were deleted from the chromosome. Thus, the pSET151 [66] derivative pIJ8506 was constructed as described in Section 4 and transferred by conjugation from *E. coli* ET12567 (containing pUZ8002 [67], which bears the functions required to mobilise pSET151) to *S. coelicolor* strains M511 and M521 (derivatives of M145 and M600, respectively, that are unable to produce actinorhodin due to a deletion in the pathway-specific regulatory gene *actII-ORF4* [68]). The resulting double crossover mutants, M591 and M592 respectively, lacked the entire *redW* gene, and all but the 5′-terminus of *redX* and the 3′-terminus of *redY*, and did not produce any red pigment. Complementation of the deletion in these mutants by conjugal transfer of pIJ8534 (a pSET151 derivative containing the entire coding sequences for *redX*, *redW* and *redY*)

![RedQ RedX RedN](https://i.imgur.com/yourimage.png)

Fig. 4. Deduced pathway to 2-undecylpyrrole illustrating the domain structures of the multienzymes RedX and RedN. Ketosynthase (KS) domains are coloured light blue, ACPs are dark blue and the PLP-dependent glycyl transferase (GT) domain is green.

from *E. coli* ET12567/pUZ8002 restored red pigment production.

To examine the role of *redXWY* in 2-undecylpyrrole biosynthesis, the ability of the previously described *S. coelicolor redC* and *redE* mutants [31], which are blocked in bipyrrrole biosynthesis [52], to cross-feed M591 and M592 was tested. Both the *redC* and the *redE* mutants were able to cross-feed M591 and M592, thus implicating the *redXWY* locus in monopyrrole biosynthesis. In addition, the *S. marcescens* mutant OF, which is blocked in bipyrrrole biosynthesis, but not the *S. marcescens* mutant 9-3-3, which is deficient in 2-methyl-3-pentylpyrrole biosynthesis, also cross-fed M591 and M592 (Fig. 5). The role of the *redXWY* genes in monopyrrole biosynthesis was confirmed by HPLC analysis of ethyl acetate extracts of M591 and M592 mycelium, which revealed 4-methoxy-2,2′-bipyrrrole-5-carboxaldehyde but no prodigines (data not shown).

In similar experiments to examine the role of *redN* in 2-undecylpyrrole biosynthesis pIJ8544 was constructed as described in Section 4. Transfer of this plasmid from *E. coli* ET12567 to *S. coelicolor* M511 and M521 generated the corresponding double crossover mutants M595 and M596, respectively, containing an in-frame deletion

in *redN*. As expected, these mutants produced no red pigment and the red phenotype could be restored by complementation of the deletion via conjugal transfer of pIJ8566 (a pKC1250 [66] derivative containing the entire *redN* coding sequence) from *E. coli* ET12567/pUZ8002. Cross-feeding experiments with the *S. coelicolor redC* and *redE* mutants and the *S. marcescens* OF and 9-3-3 mutants gave the same results as for the *redXWY* deletion mutant (Fig. 6). HPLC analysis of ethyl acetate extracts of M595 and M596 confirmed that only 4-methoxy-2,2′-bipyrrrole-5-carboxaldehyde was being produced (data not shown), and that *redN* is therefore involved in 2-undecylpyrrole biosynthesis.

The last two steps in the biosynthesis of 2-undecylpyrrole require the reduction of the ketone in 4-keto-2-undecylpyrroline and elimination of water from the resulting hydroxyl compound. We suggest that the products of *redH* and *redF* catalyse these reactions (Table 1). Although RedF does not show significant similarity to other proteins in the databases, it does contain a sequence matching the Prosite entry PS50244, which contains the NAPDH-binding site within steroid 5-α-reductases, and is therefore assigned as an oxidoreductase. Thus, 4-keto-2-undecylpyrroline would undergo reduction by RedF-bound NADPH to give the corresponding hydroxyl compound. RedH appears to contain three distinct domains, with a central domain of low similarity to other proteins in the database flanked by N- and C-terminal domains homologous to the ATP-binding and phosphorylhistidine domains of phosphoenol pyruvate (PEP) synthases, respectively. By analogy with the swiveling domain mechanism of PEP synthase [69], we suggest that RedH catalyses the transfer of a phosphoryl group from ATP to the 4-hydroxy group in the product of RedF via a phosphoryl histidine intermediate. In line with this suggestion, RedH contains the universally conserved His869 residue in its putative C-terminal phosphorylhistidine domain for mediating this process. Finally, spontaneous (possibly syn-) elimination of phosphate from the product of RedH would yield 2-undecylpyrrole (Fig. 4).

### 2.3. Genes for 4-methoxy-2,2′-bipyrrrole-5-carboxaldehyde biosynthesis

Incorporation experiments in *S. longisporusuber* and other actinomycetes have shown that the monosubstituted pyrrole ring of undecylprodigine is derived from the pyrrolidine carbons of proline [27,28]. The formation of an analogous monopyrrole ring in the *Pseudomonas fluorescens* Pf-5 natural product pyoluteorin has been suggested to occur via initial conversion of L-proline to its coenzyme A thioester by PltF, an enzyme homologous to adenylation domains of non-ribosomal peptide synthetases (NRPSs) [70]. Subsequent oxidation of prolyl-CoA to the corresponding Δ-2-pyrroline derivative, which undergoes spontaneous oxidation to pyrrole-2-carboxyl-

Research Paper  *Streptomyces coelicolor* prodigiosine biosynthesis cluster  A.M. Cerdeño et al.  823

L-proline to a thioester of pyrrole-2-carboxylic acid, which primes the synthesis of the methoxybipyrrole aldehyde precursor of undecylprodigiosine (Fig. 7). A similar role in the conversion of L-proline to peptidyl carrier protein (PCP)-bound pyrrole-2-carboxylate has recently been proposed for the enzymes encoded by *proABC* in the coumermycin biosynthetic gene cluster [71]. In the prodigiosine biosynthetic pathway, L-proline is probably attached as a thioester to the PCP encoded by *redO*, although the use of coenzyme A as an alternative acyl carrier cannot be ruled out on the basis of the available data. The RedM/RedO system constitutes a rare example of a type II NRPS (consisting of individual enzymatic and carrier protein activities on distinct polypeptides as opposed to a multienzyme), which has thus far been found in only the bleomycin, coumermycin and pyoluteorin biosynthetic gene clusters [70–72]. Several other interesting natural products, such as indanomycin and calcimycin (and other pyrroletethers), appear to derive from pyrrole-2-carboxylic acid starter units [73–76]. It seems likely that the biosynthesis of the pyrrole moieties in these natural products would also utilise a type II NRPS system, similar to the one discussed above.

In support of the proposed roles for RedM, PltF and ProB in prodigiosine, pyoluteorin and coumermycin biosynthesis, respectively, comparison of the deduced substrate specificity pocket residues for these three adenylating enzymes shows that they are virtually identical [77,78]. This indicates that they probably have very similar substrate specificity.

At first sight the fact that deletion of the *redXWY* genes results in abolition of 2-undecylpyrrole production but not 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde production (vide supra) would appear to be at odds with the proposed

Fig. 6. Cosynthesis behaviour of the *redN* deletion mutants M595 and M596 with mutants of *S. coelicolor* A3(2) and *S. marcescens* blocked in prodigiosine biosynthesis. The plate on the left shows that 2-methyl-3-pentylpyrrole secreted by the *S. marcescens* OF mutant (blocked in 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde biosynthesis), but not 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde secreted by the WF and 9-3-3 mutants restores prodigiosine production in M595 and M596. The plate on the right shows that 2-undecylpyrrole secreted by *S. coelicolor redC* and *redE* mutants (blocked in 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde biosynthesis) restores prodigiosine production in M595 and M596.

CoA, is thought to be catalysed by an acyl-CoA dehydrogenase homologue (PltE) [70]. Pyrrole-2-carboxyl-CoA is then transferred to a ketosynthase domain of a type I modular polyketide synthase complex that catalyses the formation of the resorcinol ring of pyoluteorin using three units of malonyl CoA. Two genes in the *red* cluster, *redM* and *redW*, encode homologues of PltF (37% identity in a 506 amino acid overlap) and PltE (46% identity over a 381 amino acid overlap), respectively. Based on these similarities it is suggested that RedM and RedW catalyse analogous reactions to PltF and PltE in the conversion of

![Figure 7](https://i.imgur.com/yourimageurl.png)

Fig. 7. Deduced pathway to 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde illustrating the domain structure of the multienzyme RedL. The ketosynthase (KS) domain is light blue, ACPs are dark blue, the adenylation (A) domain is red, the acyltransferase (AT) domain is orange, and the PLP-dependent seryl transferase (ST) domain is green.

role for RedW in the biosynthesis of the latter. This discrepancy is explained, however, by analysis of the domains within the modular multienzyme encoded by *redL* (Fig. 7), which we believe is responsible for assembling the second pyrrole ring of 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde from pyrrole-2-carboxylic acid or its corresponding thioester, malonyl CoA and serine. RedL contains an N-terminal domain (A) homologous to the superfamily of adenylate-forming enzymes [79], followed by an ACP domain, a ketosynthase (KS) domain, an acyltransferase (AT) domain and another ACP domain, all of which are homologous to the corresponding domains in type I modular polyketide synthases [46]. At the C-terminus of RedL there is a domain showing 34% identity over a 447 amino acid overlap to the glycyl transferase domain of RedN (Fig. 7). This domain organisation suggests that the pyrrole-2-carboxyl starter unit can be supplied either by transfer from RedO (or CoA), or by transfer from the upstream ACP domain, to the active site cysteine in the KS domain of RedL. The upstream ACP domain would be acylated directly with pyrrole-2-carboxylic acid (which is a known catabolite of proline in many bacteria [80]) by the N-terminal adenylation domain of RedL, whereas RedO is acylated by RedM with L-proline, which undergoes subsequent oxidation to the corresponding pyrrole by RedW. Thus, the phenotype of the *redXWY* mutant can be explained because only one of the two routes for priming 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde biosynthesis in *S.* coelicolor is abolished by this mutation. In support of the N-terminal adenylation domain's proposed function, it lacks the universally conserved Asp residue in the substrate selectivity pocket of amino acid-selective A domains of NRPSs, and therefore is likely to activate aromatic acids, such as pyrrole-2-carboxylic acid, rather than an amino acid such as L-proline.

Once the KS domain of RedL has been acylated with pyrrole-2-carboxylic acid, subsequent condensation with a malonyl group, which would be transferred from CoA to the second ACP domain in RedL by the upstream AT domain, would give the corresponding β-ketoacyl thioester. This would then be cleaved from RedL by nucleophilic addition of the anion generated by PLP-mediated deprotonation of serine to the acyl thioester, catalysed by the C-terminal seryl transferase domain (ST). Decarboxylation,

followed by spontaneous dehydration of the resulting amino ketone would give the dihydroxybipyrrole (8, Fig. 7). To the best of our knowledge, PLP-dependent amino acid transferase domains have hitherto not been observed as part of modular multienzymes and the proposed mechanism of chain termination catalysed by this domain has no parallels in other known type I PKS or NRPS systems.

To convert 8 to 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde, methylation of the hydroxyl group bound to the pyrrole ring and oxidation of the hydroxymethyl group to the corresponding aldehyde are required. These reactions are thought to be catalysed by RedI, which shows 25% identity over 319 amino acids to the SAM-dependent O-methyl transferase TcmO from *Streptomyces glaucescens* [81], and RedK, which shows 26% identity over 319 amino acids to the NAD(P)-dependent cholesterol dehydrogenase Cdh from *Nocardia* sp. [82], respectively. The product of *redI* is probably identical to the previously characterised hydroxybipyrrolecarboxaldehyde O-methyltransferase that is inactive in the *S.* coelicolor *redE* and *redF* mutants [32,33].

### 2.4. Genes for formation of undecylprodiginine and butyl-meta-cycloheptylprodiginine

The last step in undecylprodiginine biosynthesis is condensation of 2-undecylpyrrole with 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde to complete formation of the extended system of conjugated double bonds responsible for the characteristic red colour of most prodiginines (Fig. 8). At present, we are unable to assign a gene product to the catalysis of this reaction, although one or more of the five unassigned ORFs in the cluster may be involved. It is notable, however, that synthetic studies have shown this reaction to be very facile in the presence of a Bronsted acid catalyst [25]. Therefore, this reaction may not require catalysis by a pathway-specific enzyme in vivo.

The conversion of undecylprodiginine to butyl-meta-cycloheptylprodiginine is formally a two-electron oxidation. The only enzyme encoded by the cluster which shows significant similarity to other proteins in the databases, that has not been assigned a function in the preceding discussion, is RedG. This protein is predicted to contain an

![Chemical Reaction Diagram](attachment:chemical_reaction.png)

Fig. 8. Coupling of 2-undecylpyrrole and 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde to form undecylprodiginine and oxidative cyclisation of undecylprodiginine to give *meta*-butylcycloheptylprodiginine.

N-terminal domain similar to the N-terminal Fe₂S₂-binding Rieske domains of many non-haem iron-dependent dioxygenases such as PrnD (33% identity in a 65 amino acid overlap), which is a pyrrolonitrin biosynthetic enzyme from *P. fluorescens* [83]. The C-terminal domain of such dioxygenases, which usually hydroxylate aromatic substrates (e.g. naphthalene), contains a non-haem iron coordinated by two His residues and an Asp residue. Although the C-terminal domain of RedG does not show significant sequence similarity to the C-terminal domain of these dioxygenases or any other proteins in the databases, it does contain the YHxxxxH and DxxxxE motifs, which encompass the universally conserved His and Asp iron-binding residues [84]. We therefore suggest that RedG is a non-haem iron-dependent dioxygenase capable of binding molecular oxygen, which undergoes two-electron reduction by an external reductant (e.g. a ferredoxin) via the Rieske domain, to give an iron (IV)oxene intermediate. Regioselective removal of a hydrogen atom from the saturated side chain of undecylprodigine to give either a radical or cationic intermediate would be followed by capture of this intermediate by the nucleophilic C-4 atom of the disubstituted pyrrole to give butyl-meta-cycloheptylprodigine via a direct oxidative cyclisation reaction (Fig. 8). It seems likely that macrocyclic prodiginines from other actinomycetes (e.g. 3 and 4) arise from undecylprodigine (or in some cases nonylprodigine) via similar oxidative cyclisations catalysed by homologues of RedG with different regiospecificities. While broadly similar reactions occur in penicillin biosynthesis (e.g. the oxidative cyclisation of ACV to isopenicillin N [85]), formation of macrocyclic rings via such a process has no parallel either in other known enzyme- or non-enzyme-catalysed reactions. Indeed RedG and its potential homologues may be of considerable use for the regio- and stereoselective functionalisation of hydrocarbons, which is one of the major, hitherto unachieved, goals of organic chemistry.

2.5. Genes encoding post-translational modification enzymes and editing enzymes

The formation of functional ACPs and PCPs both as discrete domains within multienzymes (e.g. in RedL and RedN) and as separate proteins (e.g. RedQ and RedO) requires them to be modified by post-translational attachment of the 4′-phosphopantetheinyl moiety of coenzyme A to a universally conserved serine residue. This reaction is catalysed by the superfamily of 4′-phosphopantetheinyl transferase enzymes [86]. The product of *redU* shows homology to enzymes in this superfamily and probably catalyses the post-translational phosphopantetheinylation of some or all of RedL, RedN, RedO and RedQ.

Although RedN and RedL contain novel glycyl and seryl transferase domains in place of a C-terminal thioesterase domain, which is the most common type of domain found in multienzyme synthases for catalysing release of

the fully assembled chain, the *red* cluster does contain a thioesterase homologue that is encoded by *redJ*. Such type II thioesterases have been found in other clusters containing genes that encode modular PKSs and NRPSs and have been suggested to play a role in editing by cleaving mis-cognate chains from the carrier protein domains of the multienzymes [87]. It seems probable that RedO performs a similar role in prodigine biosynthesis.

2.6. Genes of unassigned function

Five genes in the cluster cannot be assigned a function at the present time, because of their low similarity to genes of known function in the databases. Besides the condensation of 2-undecylpyrrole and 4-methoxy-2,2′-bipyrrole-5-carboxaldehyde, for which we have been unable to assign an enzyme activity, every other step in prodigine biosynthesis has been assigned a cluster-encoded enzyme catalyst. Further knockout experiments will be required to determine which of these genes are involved in prodigine biosynthesis and which are involved in transport of, or self-resistance to, undecylprodigine and butyl-meta-cycloheptylprodigine.

3. Significance

Prodiginines are a structurally interesting group of oligopyrroles produced by actinomycetes and other eubacteria that have attracted much recent interest from both synthetic and medicinal chemists because of their immunosuppressive and cytotoxic properties. A complete pathway for prodigine biosynthesis has been deduced by analysis of the *red* cluster in *S. coelicolor* A3(2), including assignment of putative enzymatic functions for the proteins encoded by 16 of the 21 previously uncharacterised genes in the cluster. Several new mechanisms both for chain initiation and termination in multienzyme synthases and synthetases, and for post-multienzyme tailoring steps have been revealed by this analysis. Confirmation of the role of some of the enzymes in the deduced pathway is provided by gene deletion experiments. The deduced pathway provides a framework for developing a detailed understanding of the mechanisms governing selectivity in key prodigine biosynthetic enzymes and suggests several strategies for engineering the biosynthesis of novel prodigines with improved immunosuppressive and antitumour activities.

4. Materials and methods

4.1. Bacterial strains, plasmids and cloning vectors

*E. coli* DH5α was used as the cloning host for construction of plasmids. *S. coelicolor* strains M511 and M521 [68] were used for

the construction of *red* gene deletion mutants. *E. coli* ET12567 [88] was used as the host for conjugal transfer of plasmids to *S. coelicolor*. *S. marcescens* strains NIMA, 9-3-3, OF and WF [89] were provided by Josep Lauren Egea and *S. coelicolor* strains B395, B701, B317 and JF4 were provided by David Hopwood, John Innes Centre. Cosmids SC2E9 and SC3F7 were obtained from Dr Helen Kieser, John Innes Centre, and pIJ4126, pIJ4136, pIJ4148 and pIJ6013 [38] containing portions of the *red* cluster were provided by Janet White and Eriko Takano, John Innes Centre. The vectors pIJ2925 [90] and pBLUESCRIPT SK+ (Stratagene) were used for subcloning in *E. coli*. Vectors pSET151 and pKC1132 [66] capable of conjugal transfer from *E. coli* to *Streptomyces* spp. were used for gene deletions in *S. coelicolor*.

### 4.2. Growth of strains and DNA manipulation

Standard procedures and media were used for the growth of *S. coelicolor* and the isolation of genomic DNA [91]. DNA was manipulated in vitro and in *E. coli* following established protocols [92]. *S. marcescens* was grown on PGA medium (0.5% Difco Bacto-peptone, 1% glycerol, 2.2% agar, pH 7.1) at 27 or 30°C [89].

### 4.3. Construction of pIJ8506 and deletion of *redXWY*

A 1 kb *SphI-BamHI* fragment from pIJ6013 encompassing the 3'-terminus of *redD* and a small portion of the 5'-terminus of *redX* was cloned into *BamHI-SphI*-digested pIJ2925. The resulting plasmid was digested with *EcoRI* and ligated with a 0.8 kb fragment from pIJ4136 containing the 5'-terminus of *redZ* and the 3'-terminus of *redY*, screening for clones with the same orientation of the *EcoRI* fragment relative to the *BamHI-SphI* fragment as in the chromosome. A 1.8 kb *BgII* fragment was subcloned from this plasmid into *BamHI*-digested pSET151 to generate pIJ8506, which was transformed into *E. coli* ET12567 containing the helper plasmid pUZ8002. Conjugal transfer of pIJ8506 from *E. coli* ET12567/pUZ8002 to *S. coelicolor* M511 and M521 according to the procedure of Flett et al. [67] using thiostrepton selection (50 μg ml⁻¹) yielded putative exconjugants that were screened for integration of pIJ8506 into the chromosome and Southern hybridisation. After four rounds of non-selective growth on MS agar colonies of the single crossover mutants were screened for thiostrepton resistance. Sensitive colonies that did not produce any red pigment were picked and screened by Southern hybridisation for a second homologous recombination resulting in deletion of the *redXWY* locus.

### 4.4. Construction of pIJ8544 and in-frame deletion of *redN*

A 2 kb *BamHI* blunt-ended *SnoI* fragment from pIJ4126 containing the 3'-terminus of *redN* and the 5'-terminus of *redM* was cloned into *BamHI-EcoRV*-digested pBLUESCRIPT SK+. The resulting plasmid was digested with *NotI* and ligated with a 1.7 kb *NotI* fragment from pIJ4148 containing *redP*, *redO* and the 5'-terminus of *redN*, screening for the same orientation of the *NotI* fragment relative to the *BamHI-SnoI* fragment as in the *S. coelicolor* chromosome. A 3.7 kb *HindIII-SacI* fragment was subcloned from this plasmid into *HindIII-EcoRV*-digested pKC1132 to yield pIJ8544. Conjugal transfer of pIJ8544 into *S. coelicolor* M511 and M521, and apramycin selection of exconjugants, followed by screening and manipulation as for the ex-

conjugants derived from pIJ8506 yielded the corresponding mutants M595 and M596 bearing an in-frame deletion in *redN*.

### 4.5. Analysis of cross-feeding between the *redXWY* and *redN* deletion mutants, and *S. coelicolor* and *S. marcescens* mutant strains blocked in prodigiosine biosynthesis

Spores of *S. coelicolor* strains M591 and M592, or strains M595 and M596, were streaked on PGA medium adjacent to *S. coelicolor* strains B395, B701, B317 and JF4, or *S. marcescens* strains NIMA, 9-3-3, OF and WF and grown at 30°C for 5 days. After this time, extracellular complementation of the *redXWY* and *redN* mutations was assessed by the extent of red pigment production in the mycelia of M591, M592, M595 and M596.

## Acknowledgements

Team 23 and Julian Parkill at the Sanger Centre are gratefully acknowledged for sequencing the cosmids described in the paper, and Janet White is also gratefully acknowledged for initial sequencing of part of the *red* cluster. We would like to thank Peter Leadlay, Jim Staunton, David Hopwood, Keith Chater and Janet White for helpful discussions, Flavia Marinelli for HPLC analysis of *S. coelicolor* mycelial extracts and Josep Loren for supplying *S. marcescens* strains. Financial support of this work by the European Union through a Marie Curie Fellowship TMR-B20 (to A.M.C., contract no. ERBFM-BICT 950464), by the Wellcome Trust through a postdoctoral fellowship (to G.L.C., grant no. 053086) and by a grant-in-aid to the John Innes Centre from the BBSRC is also gratefully acknowledged.

## References

[1] N.N. Gerber, Prodigiosin-like pigments, CRC Crit. Rev. Microbiol. 3 (1975) 469–485.

[2] H.H. Wasserman, G.C. Rodgers, D.D. Keith, Metacycloprodigiosin, a tripyrrole pigment from *Streptomyces longisporusrubber*, J. Am. Chem. Soc. 91 (1969) 1263–1264.

[3] H. Laatsch, M. Kellner, H. Weyland, Butyl-meta-cycloheptylprodigione - a revision of the structure of the former ortho-isomer, J. Antibiot. 44 (1991) 187–191.

[4] N.N. Gerber, Prodigiosin-like pigments from *Actinomadura (Nocardia) pelletieri*, J. Antibiot. 24 (1971) 636–640.

[5] J. Magae, M.W. Miller, K. Nagai, G.M. Shearer, Effect of metacycloprodigiosin, an inhibitor of killer T cells, on murine skin and heart transplants, J. Antibiot. 49 (1996) 86–90.

[6] T. Azuma, N. Watanabe, H. Yagisawa, K. Hirata, M. Iwamura, Y. Kobayashi, Induction of apoptosis of activated murine splenic T cells by cycloprodigiosin hydrochloride, a novel immunosuppressant, Immunopharmacology 46 (2000) 29–37.

[7] M.H. Lee, T. Kataoka, N. Honjo, J. Magae, K. Nagai, *In vivo* rapid reduction of alloantigen-activated CD8(+) mature cytotoxic T cells by inhibitors of acidification of intracellular organelles, prodigiosin 25-C and concanamycin B, Immunology 99 (2000) 243–248.

[8] A. Nakamura, J. Magae, R.F. Tsuji, M. Yamasaki, K. Nagai, Suppression of cytotoxic T cell induction in vivo by prodigiosin 25-C, Transplantation 47 (1989) 1013–1016.

[9] R.F. Tsuji, J. Magae, M. Yamashita, K. Nagai, M. Yamasaki, Immunomodulating properties of prodigiosin 25-C, an antibiotic which preferentially suppresses induction of cytotoxic T cells, J. Antibiot. 45 (1992) 1295–1302.

[10] K. Kojiri, S. Nakajima, H. Suzuki, A. Okura, H. Suda, A new anti-tumor substance, Be-18591, produced by a streptomyces. 1. Fermentation, isolation, physicochemical and biological properties, J. Antibiot. 46 (1993) 1799–1803.

[11] Y. Hayakawa, K. Kawakami, H. Seto, K. Furihata, Structure of a new antibiotic, roseophilin, Tetrahedron Lett. 33 (1992) 2701–2704.

[12] M.S. Melvin, D.C. Ferguson, N. Lindquist, R.A. Manderville, DNA binding by 4-methoxypyrrrolic natural products. Preference for intercalation at AT sites by tambjamine E and prodigiosin, J. Org. Chem. 64 (1999) 6861–6869.

[13] M.S. Melvin, J.T. Tomlinson, G.R. Saluta, G.L. Kucera, N. Lindquist, R.A. Manderville, Double-strand DNA cleavage by copper prodigiosin, J. Am. Chem. Soc. 122 (2000) 6333–6334.

[14] S.H. Kim, I. Figueroa, P.L. Fuchs, Syntheses via vinyl sulfones. 71. Application of the Grubbs ring-closing metathesis for the construction of a macrocyclic ansa-bridge. Synthesis of the tricyclic core of roseophilin, Tetrahedron Lett. 38 (1997) 2601–2604.

[15] P.E. Harrington, M.A. Tius, A formal total synthesis of roseophilin: Cyclopentannelation approach to the macrocyclic core, Org. Lett. 1 (1999) 649–651.

[16] A. Furstner, T. Gastner, H. Weintritt, A second generation synthesis of roseophilin and chromophore analogues, J. Org. Chem. 64 (1999) 2361–2366.

[17] A. Furstner, H. Krause, Flexible synthesis of metacycloprodigiosin and functional derivatives thereof, J. Org. Chem. 64 (1999) 8281–8286.

[18] A. Furstner, J. Grabowski, C.W. Lehmann, Total synthesis and structural refinement of the cyclic tripyrrole pigment nonylprodigiosin, J. Org. Chem. 64 (1999) 8275–8280.

[19] A. Furstner, H. Weintritt, Total synthesis of roseophilin, J. Am. Chem. Soc. 120 (1998) 2817–2825.

[20] A. Furstner, H. Weintritt, Total synthesis of the potent antitumor agent roseophilin: A concise approach to the macrotricyclic core, J. Am. Chem. Soc. 119 (1997) 2944–2945.

[21] M. D'Auria, E. De Luca, G. Mauriello, R. Racioppi, A short synthesis of a thienyl analogue of undecylprodigiosin, Synth. Comm. 29 (1999) 35–42.

[22] T. Mochizuki, E. Itoh, N. Shibata, S. Nakatani, T. Katoh, S. Terashima, Studies toward the total synthesis of antibiotic roseophilin: A novel synthesis of the macrotricyclic part, Tetrahedron Lett. 39 (1998) 6911–6914.

[23] J. Robertson, R.J.D. Hatley, Formal synthesis of roseophilin, Chem. Comm. (1999) 1455–1456.

[24] B.M. Trost, G.A. Doherty, An asymmetric synthesis of the tricyclic core and a formal total synthesis of roseophilin via an enyne metathesis, J. Am. Chem. Soc. 122 (2000) 3801–3810.

[25] H.H. Wasserman, D.D. Keith, J. Nadelson, The synthesis of metacycloprodigiosin, J. Am. Chem. Soc. 91 (1969) 1264–1265.

[26] H.H. Wasserman, R.J. Sykes, P. Peverada, C.K. Shaw, R.J. Cushley, C.R. Lipsky, Biosynthesis of prodigiosin. Incorporation patterns of C-labeled alanine, proline, glycine, and serine elucidated by fourier transform nuclear magnetic resonance, J. Am. Chem. Soc. 95 (1973) 6874–6875.

[27] H.H. Wasserman, C. Shaw, R. Sykes, The biosynthesis of metacycloprodigiosin and undecylprodigiosin, Tetrahedron Lett. 15 (1974) 2787–2790.

[28] N. Gerber, A. McInnes, D. Smith, J. Walter, J. Wright, L. Vining, Biosynthesis of prodiginines. 13C assignments and enrichment patterns in nonyl-, cyclononyl-, methylcyclodecyl-, and butylcycloheptyl-prodigine produced by actinomycete cultures supplemented with 13C-labeled acetate and 15N-labeled nitrate, Can. J. Chem. 56 (1978) 1155–1163.

[29] B. Rudd, D. Hopwood, Genetics of actinorhodin biosynthesis by Streptomyces coelicolor A3(2), J. Gen. Microbiol. 114 (1979) 35–43.

[30] S.W. Tsao, B.A.M. Rudd, X.G. He, C.J. Chang, H.G. Floss, Identification of a red pigment from Streptomyces coelicolor A3(2) as a mixture of prodigiosin derivatives, J. Antibiot. 38 (1985) 128–131.

[31] B.A.M. Rudd, D.A. Hopwood, A pigmented mycelial antibiotic in Streptomyces coelicolor: control by a chromosomal gene cluster, J. Gen. Microbiol. 119 (1980) 333–340.

[32] J.S. Feitelson, D.A. Hopwood, Cloning of a Streptomyces gene for an O-methyltransferase involved in antibiotic biosynthesis, Mol. Gen. Genet. 190 (1983) 394–398.

[33] J.S. Feitelson, F. Malpartida, D.A. Hopwood, Genetic and biochemical characterization of the red gene cluster of Streptomyces coelicolor A3(2), J. Gen. Microbiol. 131 (1985) 2431–2441.

[34] F. Malpartida, J. Niemi, R. Navarrete, D.A. Hopwood, Cloning and expression in a heterologous host of the complete set of genes for biosynthesis of the Streptomyces coelicolor antibiotic undecylprodigiosin, Gene 93 (1990) 91–99.

[35] T. Yonekawa, Y. Ohnishi, S. Horinouchi, Involvement of amfC in physiological and morphological development in Streptomyces coelicolor A3(2), Microbiology 145 (1999) 2273–2280.

[36] J. White, M.J. Bibb, bldA dependence of undecylprodigiosin production in Streptomyces coelicolor A3(2) involves a pathway-specific regulatory cascade, J. Bacteriol. 179 (1997) 627–633.

[37] T. Umeyama, Y. Tanabe, B.D. Aigle, S. Horinouchi, Expression of the Streptomyces coelicolor A3(2) ptpA gene encoding a phosphotyrosine protein phosphatase leads to overproduction of secondary metabolites in S. lividans, FEMS Microbiol. Lett. 144 (1996) 177–184.

[38] E. Takano, H.C. Gramajo, E. Strauch, N. Andres, J. White, M.J. Bibb, Transcriptional regulation of the redD transcriptional activator gene accounts for growth-phase-dependent production of the antibiotic undecylprodigiosin in Streptomyces coelicolor A3(2), Mol. Microbiol. 6 (1992) 2797–2804.

[39] H. Onaka, T. Nakagawa, S. Horinouchi, Involvement of two A-factor receptor homologues in Streptomyces coelicolor A3(2) in the regulation of secondary metabolism and morphogenesis, Mol. Microbiol. 28 (1998) 743–753.

[40] K.E. Narva, J.S. Feitelson, Nucleotide sequence and transcriptional analysis of the redD locus of Streptomyces coelicolor A3(2), J. Bacteriol. 172 (1990) 326–333.

[41] H. Ishizuka, S. Horinouchi, H.M. Kieser, D.A. Hopwood, T. Beppu, A putative two-component regulatory system involved in secondary metabolism in Streptomyces spp., J. Bacteriol. 174 (1992) 7585–7594.

[42] E.P. Guthrie, C.S. Flaxman, J. White, D.A. Hodgson, M.J. Bibb, K.F. Chater, A response-regulator-like activator of antibiotic synthesis from Streptomyces coelicolor A3(2) with an amino-terminal domain that lacks a phosphorylation pocket, Microbiology 144 (1998) 727–738.

[43] M.A. Fernandez-Moreno, A.J. Martin-Triana, E. Martinez, J. Niemi, H.M. Kieser, D.A. Hopwood, F. Malpartida, abaA, a new pleiotropic regulatory locus for antibiotic production in Streptomyces coelicolor, J. Bacteriol. 174 (1992) 2958–2967.

[44] R. Chakraburty, M. Bibb, The ppGpp synthetase gene (relA) of Streptomyces coelicolor A3(2) plays a conditional role in antibiotic production and morphological differentiation, J. Bacteriol. 179 (1997) 5854–5861.

[45] C.O. Rock, J.E. Cronan, Escherichia coli as a model for the regulation of dissociable (type II) fatty acid biosynthesis, Biochim. Biophys. Acta 1302 (1996) 1–16.

[46] J. Staunton, B. Wilkinson, Biosynthesis of erythromycin and rapamycin, Chem. Rev. 97 (1997) 2611–2629.

[47] D. Konz, M.A. Marahiel, How do peptide synthetases generate structural diversity?, Chem. Biol. 6 (1999) R39–R48.

[48] Z. Zaman, P.M. Jordan, M. Akhtar, Mechanism and stereochemistry of the 5-aminolaevulinate synthetase reaction, Biochem. J. 135 (1973) 257–263.

[49] O. Ploux, A. Marquet, The 8-amino-7-oxopelargonate synthase from *Bacillus sphaericus*. Purification and preliminary characterization of the cloned enzyme overproduced in *Escherichia coli*, Biochem. J. 283 (1992) 327–331.

[50] K. Hofmann, P. Bucher, L. Falquet, A. Bairoch, The PROSITE database, its status in 1999, Nucleic Acids Res. 27 (1999) 215–219.

[51] S.F. Altschul, T.L. Madden, A.A. Schaffer, J. Zhang, Z. Zhang, W. Miller, D.J. Lipman, Gapped BLAST and PSI-BLAST: a new generation of protein database search programs, Nucleic Acids Res. 25 (1997) 3389–3402.

[52] E.A. Coco, K.E. Narva, J.S. Feitelson, New classes of *Streptomyces coelicolor* A3(2) mutants blocked in undecylprodigiosin (Red) biosynthesis, Mol. Gen. Genet. 227 (1991) 28–32.

[53] M. Moche, G. Schneider, P. Edwards, K. Dehesh, Y. Lindqvist, Structure of the complex between the antibiotic cerulenin and its target, beta-ketoacyl-carrier protein synthase, J. Biol. Chem. 274 (1999) 6031–6034.

[54] D. Val, G. Banu, K. Seshadri, Y. Lindqvist, K. Dehesh, Re-engineering ketoacyl synthase specificity, Struct. Fold. Des. 8 (2000) 565–566.

[55] B. Silakowski, H.U. Schairer, H. Ehret, B. Kunze, S. Weinig, G. Nordsiek, P. Brandt, H. Blocker, G. Hofle, S. Beyer, R. Muller, New lessons for combinatorial biosynthesis from myxobacteria. The myxothiazol biosynthetic gene cluster of *Stigmatella aurantiaca* dw4/3-1, J. Biol. Chem. 274 (1999) 37391–37399.

[56] F. Kunst, N. Ogasawara et al., The complete genome sequence of the Gram-positive bacterium *Bacillus subtilis*, Nature 390 (1997) 249–256.

[57] D.A. Hopwood, Genetic contributions to understanding polyketide synthases, Chem. Rev. 97 (1997) 2465–2497.

[58] R. McDaniel, S. Ebert-Khosla, D.A. Hopwood, C. Khosla, Rational design of aromatic polyketide natural products by recombinant assembly of enzymatic subunits, Nature 375 (1995) 549–554.

[59] C. Bisang, P.F. Long, J. Cortes, J. Westcott, J. Crosby, A.L. Matharu, R.J. Cox, T.J. Simpson, J. Staunton, P.F. Leadlay, A chain initiation factor common to both modular and aromatic polyketide synthases, Nature 401 (1999) 502–505.

[60] D.H. Sherman, F. Malpartida, M.J. Bibb, H.M. Kieser, D.A. Hopwood, Structure and deduced function of the granaticin-producing polyketide synthase gene cluster of *Streptomyces violaceoruber* Tu22, EMBO J. 8 (1989) 2717–2725.

[61] M.J. Bibb, S. Biro, H. Motamedi, J.F. Collins, C.R. Hutchinson, Analysis of the nucleotide sequence of the *Streptomyces glaucescens tcmI* genes provides key information about the enzymology of polyketide antibiotic biosynthesis, EMBO J. 8 (1989) 2727–2736.

[62] K.K. Burson, C. Khosla, Dissecting the chain length specificity in bacterial aromatic polyketide synthases using chimeric genes, Tetrahedron 56 (2000) 9401–9408.

[63] T. Ritsema, B.J.J. Lugtenberg, H.P. Spaink, in: D.H.R. Barton, K. Nakanishi, O. Meth-Cohn (Eds.), Comprehensive Natural Products Chemistry, Vol. 1, Elsevier, Amsterdam, 1999, pp. 325–344.

[64] C.M. Watanabe, D. Wilson, J.E. Linz, C.A. Townsend, Demonstration of the catalytic roles and evidence for the physical association of type I fatty acid synthases and a polyketide synthase in the biosynthesis of aflatoxin B1, Chem. Biol. 3 (1996) 463–469.

[65] M.M. Abboud, M. Akhtar, Biosynthesis of 5-aminolevulinic acid: involvement of a retention-inversion mechanism, J. Chem. Soc. Chem. Commun. (1974) 643–644.

[66] M. Bierman, R. Logan, K. O'Brien, E.T. Seno, R.N. Rao, B.E. Schoner, Plasmid cloning vectors for the conjugal transfer of DNA from *Escherichia coli* to *Streptomyces* spp., Gene 116 (1992) 43–49.

[67] F. Flett, V. Mersinias, C.P. Smith, High efficiency intergeneric conjugal transfer of plasmid DNA from *Escherichia coli* to methyl DNA-restricting streptomycetes, FEMS Microbiol. Lett. 155 (1997) 223–229.

[68] B. Floriano, M. Bibb, *afsR* is a pleiotropic but conditionally required regulatory gene for antibiotic production in *Streptomyces coelicolor* A3(2), Mol. Microbiol. 21 (1996) 385–396.

[69] O. Herzberg, C.C. Chen, G. Kapadia, M. McGuire, L.J. Carroll, S.J. Noh, D. Dunaway-Mariano, Swiveling-domain mechanism for enzymatic phosphotransfer between remote reaction sites, Proc. Natl. Acad. Sci. USA 93 (1996) 2652–2657.

[70] B. Nowak-Thompson, N. Chaney, J.S. Wing, S.J. Gould, J.E. Loper, Characterization of the pyoluteorin biosynthetic gene cluster of *Pseudomonas fluorescens* Pf-5, J. Bacteriol. 181 (1999) 2166–2174.

[71] Z.X. Wang, S.M. Li, L. Heide, Identification of the coumermycin A(1) biosynthetic gene cluster of *Streptomyces rishiriensis* DSM 40489, Antimicrob. Agents Chemother. 44 (2000) 3040–3048.

[72] L. Du, C. Sanchez, M. Chen, D.J. Edwards, B. Shen, The biosynthetic gene cluster for the antitumor drug bleomycin from *Streptomyces verticillus* ATCC15003 supporting functional interactions between nonribosomal peptide synthetases and a polyketide synthase, Chem. Biol. 7 (2000) 623–642.

[73] C.M. Liu, T.E. Hermann, M. Liu, D.N. Bull, N.J. Palleroni, B.L. Prosser, J.W. Westley, P.A. Miller, X-14547A, a new ionophorous antibiotic produced by *Streptomyces antibioticus* NRRL 8167. Discovery, fermentation, biological properties and taxonomy of the producing culture, J. Antibiot. 32 (1979) 95–99.

[74] J.W. Westley, R.H. Evans Jr., L.H. Sello, N. Troupe, C.M. Liu, J.F. Blount, Isolation and characterization of antibiotic X-14547A, a novel monocarboxylic acid ionophore produced by *Streptomyces antibioticus* NRRL 8167, J. Antibiot. 32 (1979) 100–107.

[75] M.O. Chaney, N.D. Jones, M. Debono, The structure of the calcium complex of A23187, a divalent cation ionophore antibiotic, J. Antibiot. 29 (1976) 424–427.

[76] M.O. Chaney, P.V. Demarco, N.D. Jones, J.L. Occolowitz, The structure of A23187, a divalent cation ionophore, J. Am. Chem. Soc. 96 (1974) 1932–1933.

[77] T. Stachelhaus, H.D. Mootz, M.A. Marahiel, The specificity-conferring code of adenylation domains in nonribosomal peptide synthetases, Chem. Biol. 6 (1999) 493–505.

[78] G.L. Challis, J. Ravel, C.A. Townsend, Predictive, structure-based model of amino acid recognition by nonribosomal peptide synthetase adenylation domains, Chem. Biol. 7 (2000) 211–224.

[79] E. Conti, N.P. Franks, P. Brick, Crystal structure of firefly luciferase throws light on a superfamily of adenylate-forming enzymes, Structure 4 (1996) 287–298.

[80] E. Adams, L. Frank, Metabolism of proline and the hydroxyprolines, Annu. Rev. Biochem. 49 (1980) 1005–1061.

[81] R.G. Summers, E. Wendt-Pienkowski, H. Motamedi, C.R. Hutchinson, Nucleotide sequence of the *tcmII-tcmIV* region of the tetracenomycin C biosynthetic gene cluster of *Streptomyces glaucescens* and evidence that the *tcmN* gene encodes a multifunctional cyclase-dehydratase-O-methyl transferase, J. Bacteriol. 174 (1992) 1810–1820.

[82] S. Horinouchi, H. Ishizuka, T. Beppu, Cloning, nucleotide sequence, and transcriptional analysis of the NAD(P)-dependent cholesterol dehydrogenase gene from a *Nocardia* sp. and its hyperexpression in *Streptomyces* spp., Appl. Environ. Microbiol. 57 (1991) 1386–1393.

[83] S. Kirner, P.E. Hammer, D.S. Hill, A. Altmann, I. Fischer, L.J. Weislo, M. Lanahan, K.H. van Pee, J.M. Ligon, Functions encoded by pyrrolnitrin biosynthetic genes from *Pseudomonas fluorescens*, J. Bacteriol. 180 (1998) 1939–1943.

[84] B. Kauppi, K. Lee, E. Carredano, R.E. Parales, D.T. Gibson, H. Eklund, S. Ramaswamy, Structure of an aromatic-ring-hydroxylating dioxygenase-naphthalene 1,2-dioxygenase, Structure 6 (1998) 571–586.

[85] N.I. Burzlaff, P.J. Rutledge, I.J. Clifton, C.M. Hensgens, M. Pickford, R.M. Adlington, P.L. Roach, J.E. Baldwin, The reaction cycle of isopenicillin N synthase observed by X-ray diffraction, Nature 401 (1999) 721–724.

[86] R.H. Lambalot, A.M. Gehring, R.S. Flugel, P. Zuber, M. LaCelle, M.A. Marahiel, R. Reid, C. Khosla, C.T. Walsh, A new enzyme superfamily - the phosphopantetheinyl transferases, Chem. Biol. 3 (1996) 923–936.

[87] A.R. Butler, N. Bate, E. Cundliffe, Impact of thioesterase activity on tylosin biosynthesis in *Streptomyces fradiae*, Chem. Biol. 6 (1999) 287–292.

[88] D.J. MacNeil, K.M. Gewain, C.L. Ruby, G. Dezeny, P.H. Gibbons, T. MacNeil, Analysis of *Streptomyces avermitilis* genes required for avermectin biosynthesis utilizing a novel integration vector, Gene 111 (1992) 61–68.

[89] D.A. Morrison, Prodigiosin synthesis in mutants of *Serratia marcescens*, J. Bacteriol. 91 (1966) 1599–1604.

[90] G.R. Jansen, M.J. Bibb, Derivatives of pUC18 that have *BglII* sites

flanking a modified multiple cloning site and that retain the ability to identify recombinant clones by visual screening of *Escherichia coli* colonies, Gene 124 (1993) 133–134.

[91] T. Kieser, M.J. Bibb, M.J. Buttner, K.F. Chater, D.A. Hopwood, Practical *Streptomyces* Genetics, The John Innes Foundation, Norwich, 2000.

[92] J. Sambrook, E.F. Fritsch, T. Maniatis, Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.
