
Journal of Bacteriology

Functional Analysis of Genes for Biosynthesis of Pyocyanin and Phenazine-1-Carboxamide from Pseudomonas aeruginosa PAO1

Dmitri V. Mavrodi, Robert F. Bonsall, Shannon M. Delaney, Marilyn J. Soule, Greg Phillips and Linda S. Thomashow

J. Bacteriol. 2001, 183(21):6454. DOI: 10.1128/JB.183.21.6454-6465.2001.

---

Updated information and services can be found at:  
<http://jb.asm.org/content/183/21/6454>

These include:

**REFERENCES**  
This article cites 59 articles, 30 of which can be accessed free at: <http://jb.asm.org/content/183/21/6454#ref-list-1>

**CONTENT ALERTS**  
Receive: RSS Feeds, eTOCs, free email alerts (when new articles cite this article), [more](http://jb.asm.org/content/183/21/6454)

---

Information about commercial reprint orders: <http://journals.asm.org/site/misc/reprints.xhtml>  
To subscribe to to another ASM Journal go to: <http://journals.asm.org/site/subscriptions/>

---

Journals.ASM.org

JOURNAL OF BACTERIOLOGY, Nov. 2001, p. 6454–6465
0021-9193/01/$04.00+0 DOI: 10.1128/JB.183.21.6454–6465.2001
Copyright © 2001, American Society for Microbiology. All Rights Reserved.

# Functional Analysis of Genes for Biosynthesis of Pyocyanin and Phenazine-1-Carboxamide from *Pseudomonas aeruginosa* PAO1

**DMITRI V. MAVRODI**,${}^{1,2\ast}$ **ROBERT F. BONSALL**,${}^{1}$ **SHANNON M. DELANEY**,${}^{2}$ **MARILYN J. SOULE**,${}^{2}$ **GREG PHILLIPS**,${}^{3}$ AND **LINDA S. THOMASHOW**${}^{1,2,3}$

Department of Plant Pathology${}^{1}$ and USDA-ARS Root Disease and Biological Control Research Unit, Agricultural Research Service,${}^{3}$ Washington State University, Pullman, Washington 99164-6430, and School of Molecular Biosciences, Washington State University, Pullman, Washington 99164-4234${}^{2}$

Received 27 April 2001/Accepted 18 July 2001

Two seven-gene phenazine biosynthetic loci were cloned from *Pseudomonas aeruginosa* PAO1. The operons, designated *phzA1B1C1D1E1F1G1* and *phzA2B2C2D2E2F2G2*, are homologous to previously studied phenazine biosynthetic operons from *Pseudomonas fluorescens* and *Pseudomonas aureofaciens*. Functional studies of phenazine-nonproducing strains of fluorescent pseudomonads indicated that each of the biosynthetic operons from *P. aeruginosa* is sufficient for production of a single compound, phenazine-1-carboxylic acid (PCA). Subsequent conversion of PCA to pyocyanin is mediated in *P. aeruginosa* by two novel phenazine-modifying genes, *phzM* and *phzS*, which encode putative phenazine-specific methyltransferase and flavin-containing monooxygenase, respectively. Expression of *phzS* alone in *Escherichia coli* or in enzymes, pyocyanin-nonproducing *P. fluorescens* developed pyocyanin-deficient phenotypes. A third phenazine-modifying gene, *phzH*, which has a homologue in *Pseudomonas chlororaphis*, also was identified and was shown to control synthesis of phenazine-1-carboxamide from PCA in *P. aeruginosa* PAO1. Our results suggest that there is a complex pyocyanin biosynthetic pathway in *P. aeruginosa* consisting of two core loci responsible for synthesis of PCA and three additional genes encoding unique enzymes involved in the conversion of PCA to pyocyanin, 1-hydroxyphenazine, and phenazine-1-carboxamide.

Phenazine compounds produced by fluorescent *Pseudomonas* species are biologically active metabolites that function in microbial competitiveness (37), the suppression of soilborne plant pathogens (1, 11, 55, 56), and virulence in human and animal hosts (35).

The most widely studied phenazine-producing fluorescent pseudomonad is *P. aeruginosa*, a gram-negative opportunistic pathogen of animals, insects, nematodes, and plants (30, 33, 35, 46). In humans, *P. aeruginosa* infects immunocompromised, burned, or injured patients and can cause both acute and chronic lung disease. Strains of *P. aeruginosa* produce a variety of redox-active phenazine compounds, including pyocyanin, phenazine-1-carboxylic acid (PCA), 1-hydroxyphenazine (1-OH-PHZ), and phenazine-1-carboxamide (PCN) (7, 52, 57). From 90 to 95% of *P. aeruginosa* isolates produce pyocyanin (52), and the presence of high concentrations of pyocyanin in the sputum of cystic fibrosis patients has suggested that this compound plays a role in pulmonary tissue damage observed with chronic lung infections (64). This idea is supported by several recent studies which demonstrated that pyocyanin contributes in a variety of ways to the pathophysiological effects observed in airways infected by *P. aeruginosa*. Pyocyanin interferes with the regulation of ion transport, ciliary beat frequency, and mucus secretion in airway epithelial cells by altering the cytosolic concentration of calcium (15). It may interact with endothelium-derived relaxing factor or with nitric oxide (which plays a central role in the control of blood pressure, blood flow, and immune function) through the formation of a complex, or it may act by inhibition of nitric oxide synthase (29, 58, 59). Phenazines that are produced by *P. aeruginosa* also can stimulate alveolar macrophages to produce two neutrophil chemotaxins, IL-8 and leukotriene B₄, that attract neutrophils into airways, causing an inflammatory response and neutrophil-mediated tissue damage (14, 33).

The unusually broad range of biological activity associated with phenazines is thought to be due to their ability to undergo redox cycling in the presence of various reducing agents and molecular oxygen, which leads to the accumulation of toxic superoxide ($\mathrm{O}_{2}^{-}$) and hydrogen peroxide ($\mathrm{H}_{2}\mathrm{O}_{2}$) and eventually to oxidative cell injury or death (6, 25). It also has been shown that pyocyanin can interact synergistically with the siderophore pyochelin and with transferrin cleaved by proteases secreted by both *P. aeruginosa* and neutrophils in infected lungs to catalyze the formation of the highly cytotoxic hydroxyl radical ($\cdot \mathrm{OH}$), which damages pulmonary endothelial cells (6, 38). In model pathogenesis systems, phenazine synthesis by *P. aeruginosa* is required for the generation of disease symptoms in plants and for effective killing of the nematode *Caenorhabditis elegans* and the greater wax moth, *Galleria mellonella* (30, 35, 46). Phenazine compounds produced in the rhizosphere of plants contribute to the biological control activity of *P. aeruginosa* against *Fusarium* wilt of chickpea and *Pythium* damping-off of bean (1).

* Corresponding author. Mailing address: Department of Plant Pathology, P.O. Box 646430, Washington State University, Pullman, WA 99164-6430. Phone: (509) 335-3269. Fax: (509) 335-7674. E-mail: mavrodi@mail.wsu.edu.

Although the pathophysiological effects of phenazines produced by *P. aeruginosa* in host organisms are well studied (6, 14, 15, 33, 34, 38, 64) and pyocyanin-deficient phenotypes frequently have been described (18, 19, 26, 32, 35, 46, 54), the biochemistry and genetics of phenazine synthesis in *P. aeruginosa* have remained unclear. We describe here cloning and functional analysis of two seven-gene phenazine operons and three phenazine-modifying genes from *P. aeruginosa* PAO1. Our results show that *P. aeruginosa* contains a complex phenazine biosynthetic pathway consisting of two homologous core loci (*phzA1B1C1D1E1F1G1* and *phzA2B2C2D2E2F2G2*) responsible for synthesis of PCA and three additional genes (*phzM*, *phzS*, and *phzH*) encoding unique enzymes involved in the conversion of PCA to pyocyanin and PCN. We detected the core biosynthetic operon by Southern hybridization in 21 phenazine-producing pseudomonads, including strains of *P. aeruginosa*, *Pseudomonas fluorescens*, *Pseudomonas chlororaphis*, and *Pseudomonas aureofaciens*, but not in seven phenazine-producing isolates of *Burkholderia cepacia*, *Burkholderia phenazinium*, and *Brevibacterium iodinum*. Thus, the core biosynthetic pathway is highly conserved in fluorescent *Pseudomonas* spp. but differs significantly from that in other phenazine-producing bacterial genera.

**MATERIALS AND METHODS**

**Bacterial strains and plasmids.** All of the bacterial strains and plasmids used in this study are described in Table 1. Strains of *P. fluorescens*, *P. aeruginosa*, *P. chlororaphis*, *P. aureofaciens*, *B. cepacia*, *B. phenazinium*, and *B. iodinum* were grown at 28°C in Luria-Bertani (LB) broth (2). *Escherichia coli* strains were grown at 28 or 37°C in LB medium. To examine phenazine production, *P. aeruginosa* strains were grown on PIA plates (Difco Laboratories, Detroit, Mich.) or in PB medium (containing [per liter of distilled water] 20 g of Bacto Peptone [Difco Laboratories], 1.4 g of MgCl₂, and 10 g of K₂SO₄) (18) for 1 to 3 days at 28 or 37°C. Strains of *P. fluorescens* were tested for phenazine production in LB medium supplemented with 2% glucose. The antibiotics used in this study were gentamicin (300 μg/ml) and carbenicillin (500 μg/ml) in experiments with mutant derivatives of *P. aeruginosa*; tetracycline (15 to 20 and 100 μg/ml) in experiments with isolates of *P. fluorescens* and *P. aeruginosa*, respectively; and ampicillin (80 μg/ml) and tetracycline (12.5 μg/ml) in experiments with *E. coli*.

**DNA manipulations.** Standard methods were used for plasmid DNA isolation, restriction enzyme digestion, agarose gel electrophoresis, ligation, and transformation (2). Pseudomonad cells were electroporated with a Gene Pulser system (Bio-Rad Laboratories, Hercules, Calif.) by using the procedure of Enderle and Farwell (17). Total DNA from *P. aeruginosa* was isolated and purified by using a cetyltrimethylammonium bromide miniprep procedure (2).

A 6.4-kb DNA probe containing the entire *phz* locus from *P. fluorescens* 2-79 (GenBank accession number L48616) was generated by PCR performed with oligonucleotide primers PHZ-UP and PHZ-LOW (Table 1). The amplification was carried out by using a 50-μl reaction mixture containing 1× eLONGase buffer (Life Technologies, Inc., Rockville, Md.), 2 mM MgSO₄, 3.0% dimethyl sulfoxide (Sigma Chemical Co., St. Louis, Mo.), 200 μM (each) dGTP, dATP, dTTP, and dCTP (Perkin-Elmer, Norwalk, Conn.), 10 pmol of each primer, 0.7 μl of eLONGase enzyme mixture (Life Technologies, Inc.), and 20 ng of purified genomic DNA from strain 2-79. All amplifications were performed with a PTC-200 thermal cycler (MJ Research, Inc., Watertown, Mass.). The cycling program included a 30-s initial denaturation step at 94°C, followed by 30 cycles of 94°C for 30 s, 64°C for 30 s, and 72°C for 7 min. The PCR product was gel purified and labeled with a random primer biotin labeling kit (NEN Life Science Products Inc., Boston, Mass.). Other PCR amplifications were carried out by using 25-μl reaction mixtures that contained 1× *Taq* DNA polymerase buffer (Life Technologies, Inc.), 1.5 mM MgCl₂, 200 μM (each) dGTP, dATP, dTTP, and dCTP, 20 pmol of each primer, 1.2 U of Platinum *Taq* DNA polymerase (Life Technologies, Inc.), and 20 ng of target DNA.

DNA samples used for Southern hybridization were digested with restriction endonucleases, separated by electrophoresis in 0.8% agarose gels, and transferred to BrightStar-Plus nylon membranes (Ambion, Inc., Austin, Tex.) in 0.4 M NaOH, and there was subsequent cross-linking of the DNA by exposure of the

membranes to UV radiation (2). Hybridization analyses were carried out as described previously (13). DNA-DNA hybrids were detected with a BrightStar nonisotopic detection kit (Ambion, Inc.) by using the manufacturer’s protocol.

Insertional inactivation of *phzM* and *phzS* genes. *P. aeruginosa* PAO1 *phzM* and *phzS* knockout mutants were generated by using a gene replacement strategy previously described by Schweizer (50). To mutagenize *phzM*, the gene was amplified by PCR with oligonucleotide primers METHYL1 and METHYL2 (Table 1) from cosmid 1G5 DNA of *P. aeruginosa*. The 1.26-kb PCR product was digested with *EcoRI* and *BamHI*, cloned into pNOT19, and inactivated by insertion of an 879-bp *aacC1* cassette from pUCGM into a unique *EcoRV* site (Fig. 1A). The resulting plasmid, pNOT-ORF1-Gm, was digested with *NotI* and ligated with the 5.3-kb MOB3 *sacB* cassette (50).

To inactivate *phzS*, the gene was amplified from cosmid 1G5 target DNA with oligonucleotide primers OXY1 and OXY2 (Table 1). The 1.24-kb PCR product was digested with *XbaI* and *SphI*, gel purified, and cloned into pNOT-19. The *aacC1* cassette was then inserted into a unique *ScaI* site within *phzS* (Fig. 1A). The resulting plasmid, pNOT-ORF2-Gm, was ligated to the MOB3 *sacB* cassette and used for mutagenesis.

Plasmids containing the *sacB* cassette and the inactivated *phzM* or *phzS* gene were mobilized in *P. aeruginosa* PAO1 from *E. coli* S17-1. Following selection for double crossovers, isolates were screened by PCR for the presence of plasmid-borne *bla* and *sacB* genes in the genome. The β-lactamase gene was detected by PCR performed with primers BLA1 and BLA2 (Table 1), which amplified a 744-bp fragment of *bla*. The *sacB* cassette was detected by PCR performed with primers SAC1 and SAC2 (Table 1), which amplified a 1.05-kb DNA fragment. The cycling program included a 1-min initial denaturation step at 94°C, followed by 25 cycles of 94°C for 45 s, 53°C for 45 s, and 72°C for 1.25 min. The presence of mutated alleles of *phzS* and *phzM* was confirmed by PCR performed with oligonucleotide primers MET1 and MET2 and oligonucleotide primers ORF2UP and ORF2LOW (Table 1), respectively.

**DNA sequencing and analysis.** DNA was sequenced by using an ABI PRISM dye terminator cycle sequencing kit (Perkin-Elmer, Norwalk, Conn.). All oligonucleotides were obtained from Operon Technologies, Inc. (Alameda, Calif.). Sequence data were compiled and analyzed with the Genetics Computer Group package (22) and the OMIGA 2.0 software package (Oxford Molecular Ltd., Oxford, United Kingdom). A database search for similar protein sequences was carried out with OMIGA’s BLAST tool. A probable domain similarity search was performed by using the PROSITE (European Molecular Biology Laboratory, Heidelberg, Germany) (3) and ISREC ProfileScan (Swiss Institute for Experimental Cancer Research, Epalinges, Switzerland) (4) web servers. The significance of the similarity of a predicted protein to known proteins was determined by calculating the binary comparison score (*Z* score) as described elsewhere (13). Similarities with *Z* scores greater than 9 were considered significant.

Multiple sequence alignments were constructed with OMIGA’s ClustalW. Phylogenetic analyses were carried out with PHYLIP 3.57c software (20). Distance matrices were generated with PROTDIST. Trees were inferred from the distances by using NEIGHBOR and FITCH with global rearrangement. Protein parsimony analyses were carried out with the PROTPARS program of the PHYLIP package. PHYLIP’s SEQBOOT program was used for bootstrap analyses.

The present study was greatly facilitated by the use of the *P. aeruginosa* genome web site resources (http://www.pseudomonas.com).

**Phenazine transformation assays.** Cultures of *E. coli* JM109 bearing pUCP26, pUCP-M, pUCP-S, pUCP-H, or pUCP-phnAB (Table 1) were grown in 2× YT (2) supplemented with tetracycline and were induced with 0.5 mM isopropyl-β-d-thiogalactopyranoside (IPTG) at an optical density at 600 nm of 0.6. Immediately after induction, PCA was added to a final concentration of 0.3 mg/ml from a 25 mM stock solution in 5% (wt/vol) NaHCO₃, and the cultures were grown for 6 h. Samples were extracted with chloroform at 3-h intervals and analyzed for phenazine composition by reverse-phase high-performance liquid chromatography (HPLC).

**Cross-feeding experiments.** Overnight cultures of *E. coli* JM109 bearing pUCP26-based plasmids were diluted 1:100 in fresh 2× YT broth supplemented with tetracycline, grown with shaking to an optical density at 600 nm of 0.6, and induced with 0.5 mM IPTG. Immediately after induction, PCA was added to a concentration of 0.3 mg/ml, and the cultures were grown for 6 h to allow phenazine metabolites to accumulate. Finally, *E. coli* cells were removed by centrifugation and passage through a 0.22-μm-pore-size filter, and the filtrates were added to broth cultures of *E. coli* JM109 bearing pUCP26, pUCP-S, or pUCP-M. After 6 h of bacterial growth, phenazines were extracted with chloroform and analyzed by reverse-phase HPLC and mass spectrometry.

**Analyses of phenazine compounds.** Phenazine compounds were extracted from bacterial cultures with chloroform, filtered (pore size, 0.2 μm), and sub-

TABLE 1. Bacterial strains, plasmids, and primers used in this study

| Strain, plasmid, or primer | Description or sequence$^{a}$ | Reference(s) or source$^{b}$ |
|---------------------------|---------------------------------|------------------------------|
| **Brevibacterium iodinum strains** |  |  |
| ATCC 15728 | Phz$^+$, produces iodinin | 57 |
| ATCC 15729 | Phz$^+$, produces iodinin | 57 |
| ATCC 9897 | Phz$^+$, produces iodinin | ATCC |
| **Burkholderia cepacia strains** |  |  |
| ATCC 25416 | Phz$^+$, produces 4, 9-dihydroxyphenazine-1, 6-dicarboxylic acid dimethylester | 31 |
| 1324 | Phz$^+$, biocontrol strain | Lab collection |
| 5.5B#5 | Phz$^+$, biocontrol strain | Lab collection |
| 2257 | Phz(?), biocontrol strain | Lab collection |
| ATCC 52796 | Phz(?), biocontrol strain | 39, 40 |
| **Burkholderia phenazinium ATCC 33666** | Phz$^+$, produces iodinin | 49 |
| **Pseudomonas aeruginosa strains** |  |  |
| PAO1 | Phz$^+$, produces pyocyanin, PCA, PCN, 1-OH-PHZ | Pseudomonas Genetic Stock Center |
| PAOmxM | Phz$^+$phzM::Gm$^r$ | This study |
| PAOmxS | Phz$^+$phzS::Gm$^r$ | This study |
| ATCC 23993 | Phz$^+$, produces pyocyanin | ATCC |
| ATCC 25007 | Phz$^+$, produces pyocyanin | 49 |
| ATCC 25011 | Phz$^+$, produces aeruginosins A and B | 65 |
| PAKN1 | Phz$^+$, produces pyocyanin | Lab collection |
| PAKNP-1 | Phz$^+$, produces pyocyanin | Lab collection |
| PAKNP-2 | Phz$^+$, produces pyocyanin | Lab collection |
| **Pseudomonas aureofaciens strains** |  |  |
| 30-84 | Phz$^+$, produces PCA, 2-OH-PCA, 2-OH-PHZ | 42 |
| ATCC 17415 | Phz$^+$, produces PCA, 2-OH-PCA, 2-OH-PHZ | ATCC |
| ATCC 13985 | Phz$^+$, produces PCA, 2-OH-PCA, 2-OH-PHZ | 49 |
| TX-1 | Phz$^+$, produces PCA, 2-OH-PCA, 2-OH-PHZ | Eco-Soils |
| BS1391 | Phz$^+$, produces PCA, 2-OH-PCA, 2-OH-PHZ | V.Kotchekov |
| BS1393 | Phz$^+$, produces PCA, 2-OH-PCA, 2-OH-PHZ | V.Kotchekov |
| PGS12 | Phz$^+$, produces PCA, 2-OH-PCA, 2-OH-PHZ | 23 |
| **Pseudomonas chlororaphis strains** |  |  |
| ATCC 17411 | Phz$^+$, produces PCN | 57 |
| ATCC 17809 | Phz$^+$, produces PCN | ATCC |
| ATCC 9446 | Phz$^+$, produces PCN | 49 |
| **Pseudomonas fluorescens strains** |  |  |
| 2-79 | Phz$^+$Rif$^r$, produces PCA | 61 |
| UN15 | Phz$^+$, produces PCA | G.Botelho |
| UN4127 | Phz$^+$, produces PCA | G.Botelho |
| UQ112 | Phz$^+$, produces PCA | G.Botelho |
| M4-80R | Phz$^-$Rif$^r$ | 24 |
| **Escherichia coli strains** |  |  |
| JM109 | F'traD36proA$^+$proB$^+$lacI$^q$lacZ$\Delta$M15/recA1gyrA96thihsdR17supE44relA1 $\Delta$(lac-proAB)mcrA | Promega Corp. |
| S17-1 | thi pro hsdR hsdM recA rpsL RP4-2 (Tc$^r$::Mu) (Km$^r$::Tn7) | Lab collection |
| **Plasmids** |  |  |
| pUC19 | ColE1bla | 67 |
| pUCP26 | pUC18-derived broad-host-range vector, Tet$^r$ | 62 |
| pMOB3 | Kan$^r$cat sacBR | H.P.Schweizer |
| pUCGM | Source of gentamicin cassette ColE1aacC1 | H.P.Schweizer |
| pNOT19 | ColE1bla accessory plasmid | H.P.Schweizer |
| p137, p143 | pUC19 containing 2.4-kb PstI-EcoRI fragment with phzCDE genes | This study |
| 1G5 | s-cos containing approximately 15-kb fragment of P.aeruginosa PAO1 genomic DNA | S.Lory |
| pUCP-1G5 | pUCP26 containing 10-kb BglII-NotI fragment with phzA1B1C1D1E1F1G1 genes from 1G5 | This study |
| pUCP-A1G1 | pUCP26 containing 6.9-kb NdeI-EcoRV fragment with phzA1B1C1D1E1F1G1 genes | This study |
| 1H4 | s-cos containing approximately 25-kb fragment of P.aeruginosa PAO1 genomic DNA | S.Lory |
| pUCP-A2G2 | pUCP26 containing 7.4-kb HindIII fragment with phzA2B2C2D2E2F2G2 genes | This study |
| M13-10116 | M13mp18 containing 4.5-kb fragment of P.aeruginosa PAO1 genomic DNA | M.Banger |
| pUCP-phnAB | pUCP26 containing 2.8-kb SacI-KpnI fragment from M13-10116 with phnAB genes | This study |
| pUCP-M | pUCP26 containing PCR fragment with phzM | This study |
| pUCP-S | pUCP26 containing PCR fragment with phzS | This study |
| pUCP-H | pUCP26 containing PCR fragment with phzH | This study |
| pUCP-MS | pUCP26 containing tandem of phzM and phzS genes | This study |
| pNOT-M-Gm | pNOT19 containing phzM with Gm$^r$ cassette inserted into EcoRV site | This study |
| pNOT-S-Gm | pNOT19 containing phzS with Gm$^r$ cassette inserted into ScaI site | This study |
| pNOT-M-Gm-MOB | pUCP-M-Gm ligated with MOB cassette | This study |
| pNOT-S-Gm-MOB | pUCP-S-Gm ligated with MOB cassette | This study |
| **Primers** |  |  |
| PHZ-UP | TAAGGATCCGGTAGTTCCAAGCCCCAGAAAAC |  |
| PHZ-LOW | CACATTGGATCTAGATGGGTCA CGGCTATTCAG |  |
| METHYL1 | TTTTTGAATTCTTCCGGACGCAGGAAAAG |  |
| METHYL2 | TTTTTGGATCCGT TGAAAGTTCCGATTCA |  |
| OXY1 | TCGAACACTCTAGAAAAAGGAAGCACCC |  |
| OXY2 | TTTTTGGCATGCTAGCGTGGCCGTICC |  |
| phzHup | CGCACGGATCCTT CAGAATGTTC |  |
| phzHlow | GCCACGCCAAGCTTCACGCTCA |  |
| BLA1 | GGCCCCCAGTGCTGCAATGATA C |  |
| BLA2 | GAGTATCAACATTTCCGTGTCGC |  |
| SAC1 | GATGTTTTCTTGCCTTTGATGTTC |  |
| SAC2 | GTCTTTTGCATTAGCCGGAGATC |  |

$^{a}$Phz$^+$, strain produces phenazines; Phz$^-$, strain does not produce phenazines; Phz(?) strains was not tested for phenazine production; iodinin, 1, 6-dihydroxyphenazine 5, 10-di-N-oxide; 2-OH-PCA, 2-hydroxyphenazine-1-carboxylic acid; 2-OH-PHZ, 2-hydroxyphenazine; aeruginosin A, 5-methyl-7-amino-1-carboxyphenazinium betaine; aeruginosin B, 5-methyl-7-amino-1-carboxy-3-sulfophenazinium betaine; bla, β-lactamase; Rif$^r$, rifampin resistance; Tet$^r$, tetracycline resistance.

$^{b}$ATCC, American Type Culture Collection.

A

| 0.0kb | BglII | Ndel | EcoRI |
| --- | --- | --- | --- |
| EcoRV | 2.0kb | EcoRV | 4.0kb |
|  |  |  | EcoRI |
|  |  |  | Scal |
|  |  |  | EcoRV |
|  |  |  | 10.0kb |


ORF


RBS phzM


las-box


RBS phzA1


RBS phzB1


RBS phzC1


RBS phzD1


RBS phzE1


RBS phzF1


RBS phzG1


RBS phzS


repeats


pUCP-A1G1

pUCP-1G5

pUCP-M

pUCP-S

---

B

| 0.0kb | HindIII | BamHI | EcoRI |
| --- | --- | --- | --- |
| EcoRI | 2.0kb | BamHI | 4.0kb |
|  |  |  | EcoRI |
|  |  |  | 8.0kb |
|  |  |  | 10.0kb |


ORF


RBS repeat


RBS phzA2


RBS phzB2


RBS phzC2


RBS phzD2


RBS phzE2


RBS phzF2


RBS phzG2


RBS ORF


repeat


pUCP-A2G2

---

C

| 0.0kb | XbaI | SphI |
| --- | --- | --- |
| EcoRI | BamHI | 2.0kb |
| EcoRV | Scal |  |


RBS phzM


RBS phzS


pUCP-MS

---

D

| 0.0kb | HindIII |
| --- | --- |
| BamHI | 2.0kb |
| EcoRI |  |


RBS phzH


pUCP-H

---

E

| 0.0kb | KpnI |
| --- | --- |
| EcoRI | 2.0kb |
| BamHI |  |


RBS phnA


RBS phnB


pUCP-phnAB

---

FIG. 1. Restriction maps and locations of individual genes in regions of the *P. aeruginosa* PAO1 genome containing the *phzA1B1C1D1E1F1G1* (A) and *phzA2B2C2D2E2F2G2* (B) operons, related DNA fragments contained in plasmids used in this study, and physical maps of plasmids pUCP-MS (C), pUCP-H (D), and pUCP-phnAB (E). DNA fragments contained in plasmids used in this study are indicated by thick lines. ▷, positions of the lac promoter from pUCP26; ▼, positions of insertions of the Gm' cassette. The shaded arrows indicate the positions of open reading frames (ORF) that were not relevant to the present study. RBS, ribosome binding site.

jected to C18 reverse-phase HPLC. Analyses were performed by using a Waters HPLC Integrity system consisting of an Alliance 2690 separation module, a 996 photodiode array detector, and an electron ionization ThermoBeam mass detector. Phenazine compounds were separated on a Symmetry C18 column (5 μm; 3.0 by 150 mm). The solvent flow rate was 350 μl min−1, and the flow consisted of 2 min of 8% acetonitrile–25 mM ammonium acetate, followed by a 25-min linear gradient to 80% acetonitrile–25 mM ammonium acetate. A filtered 1 M ammonium acetate solution was used to prepare all solvents. HPLC gradient profiles were monitored at spectral peak maxima of 257.0 and 313.0 nm, which are characteristic of PCA and pyocyanin in the solvent system used. Mass spectrometry total-intensity chromatogram analyses were performed from m/z 100 to 355 at a rate of 1 scan s−1. The nebulizer, ion source, and expansion region temperatures were 84, 220, and 80°C, respectively, with a helium flow rate of 15 lb/in². The UV spectra and mass spectral characteristics of the major phenazine compounds detected are summarized in Fig. 2A.

Reference samples of pyocyanin and PCA were purchased from Colour Your Enzyme (Bath, Ontario, Canada). Mass spectrometry analyses of protonated (molecular weight, 211) and unprotonated (molecular weight, 210) forms of synthetic pyocyanin revealed the presence of an unknown m/z 224 ion peak [m/z (relative intensity) 224(13) 211(58) 210(100, M⁺) 197(11) 196(14) 182(60, M⁺-CO) 181(44, M⁺·CHO) 168(18) 167(29, M⁺-NCHO) 149(14) 102(7) 91(16, C₇H₇⁺) 79(13) 78(8) 77(16, C₆H₅⁺) 76(10) 75(5)] that was not detected (Fig. 2)

when the same reference samples were analyzed with a Voyager matrix-assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometer (PerSeptive Biosystems, Framingham, Mass.), in which no heating of the sample occurred. The predominant peak detected was at m/z 211, which corresponded to the protonated form of pyocyanin. The MALDI-TOF analyses were carried out in dihydrobenzoic and α-cyano-4-hydroxycinnamic acids with an acceleration voltage of 2,500 V (positive ion), a grid voltage of 60%, a guide wire voltage of 0.03%, and a delay of 150. We concluded that the m/z 224 peak was the product of a thermally promoted intermolecular reaction in the ThermoBeam mass spectral detector, which is consistent with the observations of Watson et al. (60).

Phenazine compounds extracted from *P. chlororaphis* ATCC 17809 and ATCC 17411 were used as a source of reference material for PCN because the synthetic compound was not available. Both of these strains are well characterized and are known to produce large amounts of PCN (52). The retention time, UV spectra, and mass spectral characteristics of the major phenazine compounds produced by each of these strains were identical to those of the other and to those of the putative PCN peak observed in extracts from *P. aeruginosa* or *E. coli* expressing *phzH* from PAO1 in the presence of PCA (data not shown).

Nucleotide sequence accession numbers. The GenBank accession numbers for the nucleotide sequence data for the *phzA1B1C1D1E1F1G1* and *phzA2B2C2D2E2F2G2* operons from *P. aeruginosa* PAO1 are AF005404 and AE004616, respectively.

6458 MAVRODI ET AL. J. BACTERIOL.

A

|  |  |  |  |
| --- | --- | --- | --- |
| PCA | PYO | PCN | 1-OH-PHZ |

B

| AU | P.aeruginosa PAO-1 | 257.0 nm |
| --- | --- | --- |
| 2.00 |  |  |
| 1.00 |  |  |
| 0.00 |  |  |

| AU | P.fluorescens 2-79/pUCP-26 | 257.0 nm |
| --- | --- | --- |
| 0.80 |  |  |
| 0.40 |  |  |
| 0.00 |  |  |

| AU | P.fluorescens 2-79/pUCP-MS | 257.0 nm |
| --- | --- | --- |
| 0.40 |  |  |
| 0.20 |  |  |
| 0.00 |  |  |

| AU | E.coli JM109/pUCP-A1G1 | 257.0 nm |
| --- | --- | --- |
| 0.40 |  |  |
| 0.20 |  |  |
| 0.00 |  |  |

| AU | Mixture of E.coli JM109/pUCP-A1G1 and E.coli JM109/pUCP-MS | 257.0 nm |
| --- | --- | --- |
| 0.10 |  |  |
| 0.05 |  |  |
| 0.00 |  |  |

| AU | E.coli JM109/pUCP-A2G2 | 257.0 nm |
| --- | --- | --- |
| 0.60 |  |  |
| 0.40 |  |  |
| 0.20 |  |  |
| 0.00 |  |  |

| AU | Mixture of E.coli JM109/pUCP-A2G2 and E.coli JM109/pUCP-MS | 257.0 nm |
| --- | --- | --- |
| 0.06 |  |  |
| 0.04 |  |  |
| 0.02 |  |  |
| 0.00 |  |  |

| Minutes | 2.00 | 4.00 | 6.00 | 8.00 | 10.00 | 12.00 | 14.00 | 16.00 | 18.00 | 20.00 | 22.00 | 24.00 | 26.00 | 28.00 | 30.00 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |

and *phzE2* overlap. A putative transcriptional terminator consisting of an 18-bp region of dyad symmetry (5′-ATAACCGC AAGCGGTTAT-3′) was identified 30 bp downstream of the *phzG2* stop codon. The *phzA2B2C2D2E2F2G2* gene cluster is located approximately 2.6 Mb from *phzA1B1C1D1E1F1G1* at positions 2,070,685 to 2,076,985 of the *P. aeruginosa* PAO1 genome. The two *phz* operons are 98.3% identical at the DNA level. Essentially all of the sequence divergence occurs in the first pair of genes, *phzA1B1* and *phzA2B2*; the regions containing *phzC* through *phzG* are nearly identical. Notably, the *lux* box in the putative promoter region upstream of *phzA1* was not found upstream of *phzA2*. A cosmid clone bearing the full-length *phzA2B2C2D2E2F2G2* gene cluster, clone 1H4, was identified by comparative Southern hybridization analyses performed with a 3.1-kb *ScaI* fragment containing *phzE1*, *phzF1*, and *phzG1* (data not shown).

**Identification of *phzM*, *phzS*, and *phzH* genes in the *P. aeruginosa* PAO1 genome database.** We identified two potential phenazine-modifying genes in *P. aeruginosa* PAO1 by analysis of the genomic region containing *phzA1B1C1D1E1F1G1*. This operon is flanked by genes designated PA4209 and PA4217 in the PAO1 database (53). PA4209, referred to in this paper as *phzM*, is preceded by a putative ribosome binding site, GAGAGA, and spans positions 4,713,098 to 4,712,094 in the PAO1 genome. The *phzA1B1C1D1E1F1G1* operon and *phzM* are transcribed divergently and are separated by 695 bp (Fig. 1A). *phzM* encodes a 334-residue protein with a calculated molecular mass of 36.4 kDa that most closely resembles O-demethylpuromycin-O-methyltransferase (DmpM) from *Streptomyces anulatus* (NCBI accession number P42712; 30.1% identity; Z score, 72.4), a putative O-methyltransferase (SC5C11.09c) from *Streptomyces coelicolor* (accession number CAB76315; 30.9% identity; Z score, 64.4), O-methyltransferase (MmcR) from *Streptomyces lavendulae* (accession number AAD32742; 29.6% identity; Z score, 58.2), and carminomycin 4-O-methyltransferase (DauK) from *Streptomyces sp.* (accession number AAB16938; 28.9% identity; Z score, 44.9). A ProfileScan database search revealed a generic methyltransferase motif (residues 234 to 274) and a conserved S-adenosyl-L-methionine (SAM) binding domain (residues 164 to 273) within PhzM.

PA4217, referred to in this paper as *phzS*, is located 236 bp downstream from *phzG1* and spans positions 4,720,300 to 4,721,508 of the *P. aeruginosa* PAO1 genome. *phzS* is preceded by a well-conserved ribosome binding site, AAGGAA, and encodes a 402-amino-acid protein with a molecular mass of 43.6 kDa. PhzS is similar to bacterial monooxygenases, including salicylate hydroxylase (NahW) from *Pseudomonas stutzeri* (NCBI accession number AAD02157; 27.7% identity; Z score, 38.9), a putative salicylate hydroxylase (SCE29.14c) from *S. coelicolor* (accession number T36193; 37.5% identity; Z score,

---

FIG. 2. UV and mass spectra of PCA, pyocyanin (PYO), PCN, and 1-OH-PHZ produced by *P. aeruginosa* PAO1 (A). (B) HPLC analyses of phenazine compounds produced by wild-type *P. aeruginosa* PAO1; *P. fluorescens* 2-79 harboring pUCP26; *P. fluorescens* 2-79 harboring pUCP-MS; *E. coli* JM109 harboring pUCP-A1G1, containing *phzA1B1C1D1E1F1G1*; a mixture of *E. coli* JM109 harboring pUCP-A1G1 and *E. coli* JM109 harboring pUCP-MS; *E. coli* JM109 harboring pUCP-A2G2, containing *phzA2B2C2D2E2F2G2*; and a mixture of *E. coli* JM109 harboring pUCP-A2G2 and *E. coli* JM109 harboring pUCP-MS. The identities of phenazine compound peaks were confirmed by spectral analysis and mass spectrometry. The retention times for PCA, pyocyanin, PCN, and 1-OH-PHZ with the solvent system used in this study were 9.4, 10.6, 16.4, and 18.2 min, respectively. The absorption maxima for PCA were at 251 and 364 nm; the absorption maxima for pyocyanin were at 237, 313, and 376 nm; the absorption maxima for PCN were at 246 and 364 nm; and the absorption maxima for 1-OH-PHZ were at 259 and 367 nm. AU, absorbance units.

50.6), and putative salicylate hydroxylases from *Bordetella pertussis* (accession number AAC46266; 32.1% identity; Z score, 48.2) and *Acinetobacter* sp. (accession number AAC27110; 28.7% identity; Z score, 43.4). A Pfam database search revealed a monooxygenase motif (residues 157 to 366) and a putative N-terminal NAD binding site (residues 7 to 36) within PhzS.

A third potential phenazine-modifying gene was identified in the *P. aeruginosa* genome by using the sequence of the recently described *phzH* gene from *P. chlororaphis* PCL1391 (10) as a query sequence in a BLAST search (10). This search identified the PA0051 gene, which exhibits 80.3% identity to *phzH* and spans positions 66,303 to 68,135 in the *P. aeruginosa* genome (53). This gene, referred to in this paper as *phzH*, is preceded by a well-conserved ribosome binding site, GAGG, and encodes a 610-amino-acid protein with a calculated molecular mass of 69.5 kDa. The deduced amino acid sequence encoded by *phzH* most closely resembles the sequence of phenazine-modifying enzyme PhzH from *P. chlororaphis* (NCBI accession number AAF17502; 80.3% identity; Z score, 556.5) and is similar to the sequences of bacterial asparagine synthetases, including TcsG from *Streptomyces aureofaciens* (accession number BAB12569; 46.1% identity; Z score, 267.9), PA2084 from *P. aeruginosa* (accession number AAG05472; 45.4% identity; Z score, 291.6), and AsnO from *Bacillus subtilis* (accession number O05272; 40.2% identity; Z score, 273.3). Two well-conserved Pfam protein motifs commonly associated with asparagine synthetases were found in PhzH: the glutamine amidotransferase class II signature (residues 43 to 153) and an asparagine synthetase signature (residues 199 to 608).

Functional analyses of the core phenazine loci and the *phnAB* genes from *P. aeruginosa* PAO1. To determine the function of the phenazine genes from *P. aeruginosa*, each of the seven-gene operons was cloned in pUCP26 under control of the *lac* promoter. The resulting plasmids, pUCP-A1G1 and pUCP-A2G2 (Fig. 1), and pUCP-1G5 were expressed in *P. fluorescens* M4-80, which does not produce phenazines, and in *P. fluorescens* 2-79, which produces a single phenazine compound, PCA. Extracts from each plasmid-bearing strain were analyzed by HPLC and mass spectrometry, and the data were compared to elution profiles and spectra generated with synthetic pyocyanin or extracts from wild-type *P. fluorescens* 2-79 and *P. aeruginosa* PAO1. Extracts from *P. aeruginosa* PAO1 contained a mixture of pyocyanin, 1-OH-PHZ, and PCA. Only PCA was detected in extracts from strain M4-80 containing either pUCP-A1G1 or pUCP-A2G2 (Table 2). Similar results were obtained for derivatives of strain 2-79, in which the presence of either plasmid led to increased levels of PCA accumulation (data not shown) but did not change the range of compounds synthesized (Table 2). In contrast, strain 2-79 bearing pUCP-1G5 produced large amounts of both PCA and the derivative 1-OH-PHZ (Table 2). Plasmid pUCP-1G5 contained the complete *phzA1B1C1D1E1F1G1* operon and 3.3 kb of additional downstream DNA.

Previous studies of tryptophan biosynthesis in *P. aeruginosa* identified the *phnAB* genes(18), which encode an anthranilate synthase homologue that complements *trpE* and *trpE(G)* mutants of *E. coli*. PhnAB also contributes to pyocyanin synthesis in *P. aeruginosa*, and pyocyanin production by *phnA* mutants is greatly reduced (18). To further evaluate the role of *phnAB* in

TABLE 2. Detection by HPLC and HPLC-mass spectrometry of phenazine compounds produced by *Pseudomonas* strains

| Strain/plasmid | Phenazine compound(s) detected |
|----------------|--------------------------------|
| *P. fluorescens* 2-79/pUCP26 | PCA |
| *P. fluorescens* 2-79/pUCP-1G5 | PCA, 1-OH-PHZ |
| *P. fluorescens* 2-79/pUCP-A1G1 | PCA |
| *P. fluorescens* 2-79/pUCP-A2G2 | PCA |
| *P. fluorescens* 2-79/pUCP-phnAB | PCA |
| *P. fluorescens* 2-79/pUCP-MS | PCA, pyocyanin |
| *P. fluorescens* M4-80/pUCP26 | None |
| *P. fluorescens* M4-80/pUCP-A1G1 | PCA |
| *P. fluorescens* M4-80/pUCP-A2G2 | PCA |
| *P. fluorescens* M4-80/pUCP-phnAB | None |
| *P. aeruginosa* PAO1 | PCA, pyocyanin, 1-OH-PHZ, PCN |
| *P. aeruginosa* PAOmxM | PCA, 1-OH-PHZ, PCN |
| *P. aeruginosa* PAOmxM/pUCP-M | PCA, pyocyanin, 1-OH-PHZ, PCN |
| *P. aeruginosa* PAOmxS | PCA, PCN, red pigment |
| *P. aeruginosa* PAOmxS/pUCP-S | PCA, pyocyanin, 1-OH-PHZ, PCN |

pyocyanin synthesis, the genes were excised from M13-10116 phage (the phage DNA was a kind gift from M. G. Bangera) by digestion with SacI and KpnI, treated with T4 DNA polymerase, and inserted into the SmaI site of pUCP26. The resulting plasmid, pUCP-phnAB, was introduced into *P. fluorescens* M4-80R and 2-79 by electroporation, and transformants were assayed for phenazine production by HPLC. In no case was pyocyanin produced, and strain 2-79 produced only PCA (Table 2) in quantities comparable to those detected in 2-79 containing pUCP26.

We also conducted phenazine transformation assays to determine whether *phnAB* conferred the ability to transform PCA, a precursor of pyocyanin and 1-OH-PHZ, into other phenazine compounds. This approach has proven to be useful in characterizing other genes encoding phenazine-modifying enzymes (13). Cultures of *E. coli* JM109 containing pUCP-phnAB or the control plasmid pUCP26 were induced with IPTG and incubated for 9 h in the presence of 0.3 mg of PCA ml⁻¹. HPLC analyses revealed that the extracts from *E. coli* JM109(pUCP-phnAB) were identical to those from the control strain, and no conversion of PCA to pyocyanin or any other phenazine derivative was detected (data not shown).

Functional analyses of *phzM*, *phzS*, and *phzH*. To determine whether *phzM*, *phzS*, and *phzH* encode phenazine-modifying enzymes, gene function was analyzed in *E. coli* transformed with pUCP-M, pUCP-S, pUCP-H, and pUCP-MS (Table 1). These plasmids were constructed by amplifying *phzM*, *phzS*, and *phzH* from PAO1 genomic DNA with oligonucleotide primers METHYL1 and METHYL2, oligonucleotide primers OXY1 and OXY2, and oligonucleotide primers phzHup and phzHlow, respectively (Table 1). The PCR products were digested with restriction endonucleases, gel purified, and cloned into pUCP26 under control of the *lac* promoter (Fig. 1). To construct pUCP-MS, pUCP-M was digested with XbaI and SphI, and the PCR fragment containing *phzS* was cloned immediately downstream of *phzM*. All plasmids were single-pass sequenced to confirm the integrity of cloned gene(s).

Induced cultures of *E. coli* bearing pUCP-MS converted PCA to pyocyanin within 6 h, whereas no such conversion occurred in control cultures harboring only pUCP26 (data not shown). We did not detect derivatives of PCA in supernatants

Vol. 183, 2001

PHENAZINE BIOSYNTHESIS IN *P. AERUGINOSA* 6461

**P. aeruginosa**  
PAOmxM  

**P. aeruginosa**  
PAO1  

**P. aeruginosa**  
PAOmxS  

FIG. 3. Production of phenazine compounds by wild-type *P. aeruginosa* PAO1 (center plate) and marker exchange mutants *P. aeruginosa* PAOmxM and PAOmxS (left and right plates, respectively) on PIA plates.

of *E. coli* JM109(pUCP-M) cultures incubated under similar conditions, although these cultures were an intense dark red. Induced cultures of *E. coli* JM109 bearing pUCP-S and *E. coli* bearing pUCP-H converted PCA to 1-OH-PHZ and PCN, respectively (data not shown). We were able to restore the complete pyocyanin biosynthetic pathway in *E. coli* by mixing induced cultures containing either pUCP-A1G1 or pUCP-A2G2 with a culture containing pUCP-MS. HPLC analysis revealed that in both cases the mixed cultures produced small but detectable amounts of pyocyanin (Fig. 2B).

Plasmid pUCP-MS also was introduced into *P. fluorescens* 2-79. Overnight broth cultures of 2-79(pUCP-MS) were intensely blue, which was consistent with synthesis of pyocyanin, and the results of HPLC and mass spectrometry analyses (Fig. 2B) confirmed that large amounts of this compound were present.

**Effect of *phzM* and *phzS* mutations on pyocyanin production.** The gene replacement approach was used to evaluate the role of the phenazine-modifying genes *phzM* and *phzS* in pyocyanin biosynthesis in *P. aeruginosa* PAO1. Mutant derivatives with mutations in *phzM* (strain PAOmxM) or *phzS* (strain PAOmxS) exhibited unusual pigment phenotypes when they were cultivated on PIA. Whereas cultures of wild-type PAO1 were blue due to production of pyocyanin, cultures of *P. aeruginosa* PAOmxM were yellow, and *P. aeruginosa* PAOmxS produced large amounts of a dark red water-soluble pigment (Fig. 3). Chloroform extracts prepared from cultures of these mutants grown in PB medium were analyzed by HPLC and mass spectrometry (Table 2). Both mutants were able to synthesize PCA and PCN but did not produce pyocyanin. In fact, the *phzM* mutant produced and secreted more PCN than the parent strain (data not shown). PAOmxM produced 1-OH-PHZ, although it produced markedly less than the parental strain, while no 1-OH-PHZ was detected in extracts from PAOmxS. Finally, complementation of PAOmxM and PAOmxS with pUCP-M and pUCP-S, respectively, restored production of pyocyanin (Table 2).

---

DISCUSSION

The present study demonstrated clearly that the opportunistic human pathogen *P. aeruginosa* PAO1 contains two complete copies of a seven-gene operon shown previously (13, 36, 43) to be responsible for the synthesis of PCA. We also screened a collection of 30 bacterial strains, including isolates of *P. aeruginosa*, *P. fluorescens*, *P. chlororaphis*, *P. aureofaciens*, *Pseudomonas putida*, *B. cepacia*, *B. phenazinium*, and *B. iodinum* (Table 1), for the presence of these core phenazine biosynthetic genes by performing Southern hybridization with probes spanning *phzA* through *phzG*. Similar sequences were not detected in phenazine-producing isolates of *B. cepacia*, *B. phenazinium*, and *B. iodinum* under moderately stringent conditions (6× SSC [1× SSC is 0.15 M NaCl plus 0.015 M sodium citrate], 55°C) despite the fact that the same mechanism of phenazine synthesis is thought to function in all of these genera (34, 52, 57). Only total DNA from 26 known phenazine-producing strains of *P. aeruginosa*, *P. aureofaciens*, *P. chlororaphis*, and *P. fluorescens* hybridized strongly to the phenazine gene probe (data not shown), indicating not only that the operon is conserved in phenazine-producing fluorescent pseudomonads but also that it may be unique to them.

Of the pseudomonads, only *P. aeruginosa* has been found to contain two copies of the phenazine operon. Both copies are homologous to phenazine biosynthetic loci from *P. aureofaciens*, *P. fluorescens*, and *P. chlororaphis* (13, 36, 43), and both are functional, as each enabled the non-phenazine-producing strain *P. fluorescens* M4-80R to synthesize PCA, which was implicated previously as a pyocyanin precursor (8). Although the two phenazine operons of *P. aeruginosa* PAO1 are 98% identical overall, they differ markedly in their promoter regions. In fluorescent *Pseudomonas* species, phenazine production occurs mainly during the stationary phase and is regulated by quorum sensing and the highly conserved, global, two-component signal transduction system, GacA-GacS (44, 47). The N-acyl-homoserine lactone-mediated regulation of phenazine

synthesis in *P. aeruginosa* is complex and involves at least two different quorum-sensing circuits, *las* and *rhl*, organized in a hierarchical cascade (41). Whiteley et al. (63) recently identified the *phzA1B1C1D1E1F1G1* operon among the genes controlled by quorum sensing in *P. aeruginosa*. This operon is preceded by a well-conserved putative promoter element required for quorum control, the *las* box (Fig. 1). Interestingly, no such element is present upstream of *phzA2B2C2D2E2F2G2*. On the other hand, analyses of the PAO1 genome have revealed a gene, *qscR*, that is similar to *lasR* and *rhlR* and is located upstream of *phzA2B2C2D2E2F2G2* (Fig. 1). According to Chugani et al. (12), *qscR* encodes a protein that negatively regulates expression of a number of quorum-sensing-controlled genes via repression of *lasI* and *lasI*-regulated target genes, including *phzA1B1C1D1E1F1G1*. It is tempting to speculate that the presence of two differentially regulated phenazine biosynthetic operons may give the bacterium greater flexibility in modulating the amount or range of phenazine compounds produced depending on the phase of growth or in response to environmental signals.

Recently, Mahajan-Miklos et al. (35) demonstrated that phenazines are involved in the killing of *C. elegans* by the clinical isolate *P. aeruginosa* PA14. The pathogenicities of two pyocyanin-deficient mutants of *P. aeruginosa* PA14, both containing a *TnphoA* insertion in a gene similar to *phzB1*, were reduced in the *Caenorhabditis* killing model, in *Arabidopsis* leaf infiltration assays, and in the burned mouse model. Interestingly, inactivation of *phzB* reduced pyocyanin production in strain PA14 but did not eliminate it entirely (35). These results are consistent with expression of a second, differently regulated copy of the *phz* genes in the PA14 genome. We compared the proteins encoded by *phzA* and *phzB* from *P. aeruginosa* PA14 with the corresponding products of the *P. aeruginosa* PAO1 genes. The results of cluster analyses (data not shown) indicated that the products of the *phzA* and *phzB* genes of strain PA14 most closely resemble PhzA2 and PhzB2 from *P. aeruginosa* PAO1 (all bootstrap values were ≥81%). These data suggest that PAO1 is not unique in containing two homologous *phz* operons, and the trait may be common among strains of *P. aeruginosa*. It remains to be determined whether this unusual feature is typical of both environmental and clinical isolates and if it is relevant to the role in pathogenesis of pyocyanin and other phenazines produced by *P. aeruginosa*.

Our results indicate that specific genes in *P. aeruginosa* are required for conversion of PCA to pyocyanin and PCN. At least two modifications would be expected in the conversion of PCA to pyocyanin: addition of the N-methyl group, converting PCA to 5-methylphenazine-1-carboxylate betaine; and hydroxylative decarboxylation of the betaine to form pyocyanin (8, 52, 57). Using DNA sequence information from the Pseudomonas Genome Project (www.pseudomonas.com), we identified two potential genes in the vicinity of the *phzA1B1C1D1E1F1G1* operon (Fig. 1) that could encode the required PCA-modifying enzymes. Several lines of evidence support the idea that *phzM* and *phzS* play essential roles in the pyocyanin biosynthetic pathway. Transformation of the PCA-producing strain *P. fluorescens* 2-79 with *phzM* and *phzS* triggered synthesis of large amounts of pyocyanin (Table 2). We also were able to reconstitute the full pathway for biosynthesis of pyocyanin in *E. coli* by mixing induced cultures expressing *phzMS* with cultures expressing *phzA1B1C1D1E1F1G1* or *phzA2B2C2D2E2F2G2* (Fig. 2B). Finally, *E. coli* expressing *phzM* and *phzS* efficiently converted exogenously supplied PCA to pyocyanin. These data are consistent with results of mutagenesis studies in which *P. aeruginosa* marker exchange mutants with mutations in *phzM* and *phzS* failed to produce detectable amounts of pyocyanin (Table 2).

Our results support the results of previous studies (8, 21, 51) that predicted the existence of a phenazine methyltransferase and a phenazine-specific hydroxylase in the pyocyanin biosynthetic pathway. Byng et al. (8) attempted to determine the exact relationship among the different phenazine compounds synthesized by *P. aeruginosa*. Their study yielded three different groups of mutant strains: one group that produced no phenazines; a second group that synthesized PCA and PCN; and a third group that produced PCA, PCN, and the dark red compounds aeruginosins A and B (8, 9). The members of the second group of mutants identified by Byng et al. appear to be phenotypically identical to the *phzM* knockout strain, while the members of the third group resemble the *phzS* knockout strain (Table 2). Our data raise questions concerning the identity of the dark red compound characterized previously as a complex of aeruginosin A (5-methyl-7-amino-1-carboxymethylphenazinium betaine) and aeruginosin B (5-methyl-7-amino-1-carboxy-3-sulfophenazinium betaine) and thought to be derived from PCA via a specific biosynthetic pathway (8). We were surprised to find that *E. coli* expressing *phzM* alone converted PCA to a dark red compound(s) similar in appearance to the product(s) synthesized by *phzS* mutants of *P. aeruginosa* (Fig. 3). Unfortunately, we were unable to identify this material by C<sub>18</sub> reverse-phase HPLC because the pigment is extremely hydrophilic and could not be extracted with the solvents commonly used to recover phenazines. These properties are similar to those of the aeruginosins described previously by Herbert and Holliman (27) and Holliman (28). We hypothesize that the dark red pigment synthesized by *P. aeruginosa* PAOmxS mutants and by *E. coli* expressing *phzM* is not a complex of aeruginosins A and B but rather is a colored intermediate (probably 5-methylphenazine-1-carboxylic acid betaine) formed through methylation of PCA by PhzM.

PhzM is a predicted 36.4-kDa protein similar to enzymes involved in the methylation of polyketide antibiotics by *Streptomyces* spp. Streptomycetes produce a variety of bioactive polyketides, many of which contain methyl groups introduced via the action of *N*- and *O*-methyltransferases (16, 66). PhzM most closely resembles *O*-methyltransferases from *Streptomyces* spp. and shares with them a putative catalytic domain (residues 137 to 177) that might be universal among SAM-dependent methyltransferases (4). The presence of a potential SAM binding site in the N-terminal part of PhzM (residues 67 to 176) further supports the idea that PhzM functions as a methyltransferase.

The second putative phenazine-modifying gene, *phzS*, is located immediately downstream of *phzA1B1C1D1E1F1G1* (Fig. 1) and encodes a predicted 43.6-kDa protein similar to putative salicylate hydroxylases from *S. coelicolor* (GenBank accession numbers T36193 and CAB95795), *Bordetella pertussis* (AAC46266), *Acinetobacter* sp. strain ADP1 (AAC27110), and *P. aeruginosa* (AAG06716) and NahW from *P. stutzeri* AN10 (AAD02157). NahW is of particular interest as it is the best-

Vol. 183, 2001

PHENAZINE BIOSYNTHESIS IN *P. AERUGINOSA* 6463

Steps controlled by  
phzA1B1C1D1E1F1G1 and  
phzA2B2C2D2E2F2G2  

![Diagram](https://i.imgur.com/yourimage.png)

FIG. 4. Proposed mechanism for the synthesis of pyocyanin, 1-OH-PHZ, and PCN in *P. aeruginosa* PAO1.

characterized enzyme related to PhzS. According to Bosch et al. (5), nahW encodes a flavoprotein monooxygenase with broad substrate specificity that catalyzes conversion of salicylic acid and its derivatives to catechol. In a detailed sequence analysis of NahW, they identified several conserved motifs present in bacterial salicylate hydroxylases. A sequence comparison of PhzS with NahW and other salicylate hydroxylases revealed that PhzS contains the consensus NADH binding domain (159-DVLVGADGIHSAVR-172), putative N-terminal flavin adenine dinucleotide binding site (11-GAGIGG-16), and substrate active site (303-GRITLLGDAAHLMYPM GANGA-323). Not only is PhzS clearly involved in the biosynthesis of pyocyanin, but it is also probably responsible for the production of 1-OH-PHZ by *P. aeruginosa*. *E. coli* expressing phzS efficiently converted exogenously supplied PCA to 1-OH-PHZ, and we also detected 1-OH-PHZ in extracts from the *P. aeruginosa* phzM mutant (Table 2). Thus, our results suggest that 1-OH-PHZ is formed from PCA via enzymatic synthesis and not through light-mediated decomposition of pyocyanin, as proposed previously (45). In fact, no traces of decomposition were found in solutions of synthetic pyocyanin after 5 days of incubation at room temperature under direct light (data not shown).

We suggest that two steps are involved in the synthesis of pyocyanin from PCA (Fig. 4). In the first step, catalyzed by the SAM-dependent methyltransferase PhzM, PCA is converted to 5-methylphenazine-1-carboxylic acid betaine. The second step, catalyzed by the NADH (or NADPH)-dependent flavoprotein monooxygenase PhzS, involves hydroxylative decarboxylation of 5-methylphenazine-1-carboxylic acid betaine to pyocyanin. In cross-feeding studies to determine the order of the two reactions, we detected pyocyanin only when cultures of *E. coli* expressing phzS were incubated with filtered extracts from cultures expressing phzM (data not shown). When cultures expressing phzM were incubated for 6 h with filtered extracts from cultures expressing phzS, only PCA and 1-OH-PHZ were detected. We speculate that the ability of PhzS to convert PCA to 1-OH-PHZ can be explained by the broad substrate specificity of the enzyme.

The gene for biosynthesis of PCN was identified in the *P. aeruginosa* database by using the recently discovered phzH gene from *P. chlororaphis* (10) as the query sequence. *E. coli* expressing phzH converted exogenously supplied PCA to PCN (data not shown), confirming that phzH from *P. aeruginosa* is functionally homologous to its counterpart from *P. chlororaphis*. The deduced product of these genes is similar to class II glutamine amidotransferases, namely, to asparagine synthases. The closest functionally characterized enzyme with similarity to PhzH is the asparagine synthetase AsnO from *B. subtilis*. Bacterial asparagine synthetases are grouped into two families (68); the first comprises enzymes similar to *E. coli* AsnA that accept only ammonia as the amino donor, whereas the members of the second family resemble *E. coli* AsnB and can use both ammonia and glutamine as amino donors (48). According to Yoshida et al. (68), asnO encodes an AsnB type of asparagine synthase crucial for sporulation in *B. subtilis*. Based on the sequence similarity and function data, we propose that phzH of *P. aeruginosa* encodes a glutamine-dependent, phenazine-specific amidotransferase that catalyzes the amidation of PCA to PCN (Fig. 4).

A question then arises concerning the role of the phnAB genes, which have been implicated previously in pyocyanin production. These genes initially were characterized by Essar et al. (18) and were found to encode products resembling the anthranilate synthase subunits TrpE and TrpG. However, unlike trpE and trpG, which encode a typical tryptophan-specific anthranilate synthase in *P. aeruginosa*, phnA and phnB are cotranscribed and expressed during the stationary phase (18). Essar et al. also demonstrated that phnAB could complement trpE and trpE(G) mutants of *E. coli* and that PhnAB was not feedback inhibited by tryptophan. Insertional inactivation of phnA significantly reduced pyocyanin synthesis, an observation confirmed later by other workers (1, 35, 46); however, production was not eliminated completely, and mutants synthesized pyocyanin at levels that were 22 to 34% of the wild-type levels, which was attributed to the activity of TrpEG (18). Essar et al. suggested that PhnAB functions as a phenazine-specific an-

thranilate synthase, providing anthranilic acid as a precursor for pyocyanin production.

To further evaluate the role of these genes, we expressed *phnAB* in *P. fluorescens* 2-79, which produces only PCA, and in M4-80R, which does not produce phenazine. The introduced genes neither enhanced nor enabled PCA production in these strains (Table 2), nor did *E. coli* JM109 expressing *phnAB* convert PCA to other phenazines. Recent studies of the mechanism of phenazine nucleus assembly indicate that PCA is synthesized via the conversion, catalyzed by PhzE and PhzD, of 2-amino-2-deoxyisochorismic acid to 2,3-dihydroxy-3-hydroxyanthranilate (M. G. McDonald, D. V. Mavrodi, L. S. Thomashow, and H. G. Floss, unpublished data). Thus, anthranilic acid is not a precursor in the formation of the phenazine nucleus. Therefore, it seems likely that PhzE1 and PhzE2 in *P. aeruginosa* are largely responsible for the conversion of chorismic acid to PCA via the same mechanism proposed (36) for PhzE in *P. fluorescens* 2-79. There is ample evidence that PhnA and PhnB influence pyocyanin production in *P. aeruginosa* (1, 18, 35, 46), but it seems unlikely that they participate directly in assembly of the phenazine nucleus. The precise role of these enzymes in *P. aeruginosa* metabolism, the identities of intermediates, and the products involved remain to be discovered. These uncertainties reinforce the fact that despite numerous biochemical studies of pyocyanin biosynthesis, the actual enzymology and genetic control are still not well understood and require further investigation.

### ACKNOWLEDGMENTS

We are grateful to Stephen Lory and M. G. Bangera for providing the cosmids used in this study and to William Siems of the Laboratory for Biotechnology and Bioanalysis at Washington State University for the MALDI-TOF analysis of pyocyanin samples. We thank David M. Weller for his help, valuable discussions, and critical review of the manuscript and Karen Hansen for technical assistance.

### REFERENCES

1. Anjaiah, V., N. Koedam, B. Nowak-Thompson, J. E. Loper, M. Hofte, J. T. Tambong, and P. Cornelis. 1998. Involvement of phenazines and anthranilate in the antagonism of *Pseudomonas aeruginosa* PNA1 and Tn5 derivatives toward *Fusarium* spp. and *Pythium* spp. Mol. Plant-Microbe Interact. 11: 847–854.
2. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A. Smith, and K. Struhl (ed.). 1995. Short protocols in molecular biology. J. Wiley and Sons, New York, N. Y.
3. Bairoch, A., P. Bucher, and K. Hofmann. 1995. The PROSITE database, its status in 1995. Nucleic Acids Res. 24:189–196.
4. Bateman, A., E. Birney, R. Durbin, S. R. Eddy, K. L. Howe, and E. L. Sonnhammer. 2000. The Pfam protein families database. Nucleic Acids Res. 28:263–266.
5. Bosch, R., E. R. B. Moore, E. Garcia-Valdes, and D. H. Pieper. 1999. NahW, a novel, inducible salicylate hydroxylase involved in mineralization of naphthalene by *Pseudomonas stutzeri* AN10. J. Bacteriol. 181:2315–2322.
6. Britigan, B. E., G. T. Rasmussen, and C. D. Cox. 1997. Augmentation of oxidant injury to human pulmonary epithelial cells by the *Pseudomonas aeruginosa* siderophore pyochelin. Infect. Immun. 65:1071–1076.
7. Budzikiewicz, H. 1993. Secondary metabolites from fluorescent pseudomonads. FEMS Microbiol. Rev. 104:209–228.
8. Byng, G. S., D. C. Eustice, and R. Jensen. 1979. Biosynthesis of phenazine pigments in mutant and wild-type cultures of *Pseudomonas aeruginosa*. J. Bacteriol. 138:846–852.
9. Carson, M., and R. A. Jensen. 1974. Phenotypic recognition of pyocyanin mutants in *Pseudomonas aeruginosa*. J. Bacteriol. 117:312–314.
10. Chin-A-Woeng, T. 2000. Molecular basis of biocontrol of tomato foot and root rot by *Pseudomonas chlororaphis* strain PCL1391. Ph.D. thesis. University of Leiden, Leiden, The Netherlands.
11. Chin-A-Woeng, T. F. C., G. V. A. Bloemberg, J. van der Bij, K. M. G. M. van der Drift, J. Schripsema, B. Kroon, R. J. Scheffer, C. Keel, P. A. H. M. Bakker, H.-V. Tichy, F. J. de Bruijn, J. E. Thomas-Oates, and B. J. J.
12. Chugani, S. A., M. Whiteley, K. M. Lee, D. D’Argenio, C. Manoil, and E. P. Greenberg. 2001. QscR, a modulator of quorum-sensing signal synthesis and virulence in *Pseudomonas aeruginosa*. Proc. Natl. Acad. Sci. USA 98:2752–2757.
13. Delaney, S. M., D. V. Mavrodi, R. F. Bonsall, and L. S. Thomashow. 2001. *phzO*, a gene for biosynthesis of hydroxylated phenazine compounds in *Pseudomonas aureofaciens* 30-84. J. Bacteriol. 183:318–327.
14. Denning, G. M., L. A. Wolleweber, M. A. Railsback, C. D. Cox, L. L. Stoll, and B. E. Britigan. 1998. *Pseudomonas* pyocyanin increases interleukin-8 expression by human airway epithelial cells. Infect. Immun. 66:5777–5784.
15. Denning, G. M., M. A. Railsback, G. T. Rasmussen, C. D. Cox, and B. E. Britigan. 1998. *Pseudomonas* pyocyanine alters calcium signaling in human airway epithelial cells. Am. J. Physiol. 274:L893–L900.
16. Dickens, M. L., J. Ye, and W. R. Strohl. 1995. Analysis of clustered genes encoding both early and late steps in daunomycin biosynthesis by *Streptomyces* sp. strain C5. J. Bacteriol. 177:536–543.
17. Enderle, P. J., and M. A. Farwell. 1998. Electroporation of freshly plated *Escherichia coli* and *Pseudomonas aeruginosa* cells. BioTechniques 25:954–958.
18. Essar, D. W., L. Eberly, A. Hadero, and I. P. Crawford. 1990. Identification and characterization of genes for a second anthranilate synthase in *Pseudomonas aeruginosa*: interchangeability of the two anthranilate synthases and evolutionary implications. J. Bacteriol. 172:884–900.
19. Evans, K., L. Passador, R. Srikumar, E. Tsang, J. Nezezon, and K. Poole. 1998. Influence of the MexAB-OprM multidrug efflux system on quorum sensing in *Pseudomonas aeruginosa*. J. Bacteriol. 180:5443–5447.
20. Felsenstein, J. 1989. PHYLIP—phylogeny inference package (version 3.2). Cladistics 5:164–166.
21. Flood, M. E., R. B. Herbert, and F. G. Holliman. 1972. Pigments of *Pseudomonas* species. V. Biosynthesis of pyocyanin and the pigments of *Ps. aureofaciens*. J. Chem. Soc. Perkin Trans. I 1972: 622–626.
22. Genetics Computer Group. 1994. Program manual for the Wisconsin package, version 8. Genetics Computer Group, Madison, Wis.
23. Georgakopoulos, D. G., M. Henderson, N. J. Panopoulos, and M. N. Schroth. 1994. Cloning of a phenazine biosynthetic locus of *Pseudomonas aureofaciens* PGS12 and analysis of its expression in vitro with the ice nucleation reporter gene. Appl. Environ. Microbiol. 60:2931–2938.
24. Hamdan, H., D. M. Weller, and L. S. Thomashow. 1991. Relative importance of fluorescent siderophores and other factors in biological control of *Gaeumannomyces graminis* var. tritici by *Pseudomonas fluorescens* 2-79 and M4-80R. Appl. Environ. Microbiol. 57:3270–3277.
25. Hassan, H. M., and I. Fridovich. 1980. Mechanism of the antibiotic action of pyocyanine. J. Bacteriol. 141:156–163.
26. Hassett, D. J., H. P. Schweizer, and D. E. Ohman. 1995. *Pseudomonas aeruginosa sodA* and *sodB* mutants defective in manganese- and iron-cofactored superoxide dismutase activity demonstrate the importance of the iron-cofactored form in aerobic metabolism. J. Bacteriol. 177:6330–6337.
27. Herbert, R. B., and F. G. Holliman. 1969. Pigments of *Pseudomonas* species. II. Structure of aeruginosin B. J. Chem. Soc. C:2517–2520.
28. Holliman, F. G. 1969. Pigments of *Pseudomonas* species. I. Structure and synthesis of aeruginosin A. J. Chem. Soc. C:2514–2516.
29. Hussain, A. S., J. Bozinovski, D. H. Maurice, B. E. McLaughlin, G. S. Marks, J. F. Brien, and K. Nakatsu. 1996. Inhibition of the action of nitric oxide prodrugs by pyocyanin: mechanistic studies. Can. J. Pharmacol. 75:398–406.
30. Jander, G., L. G. Rahme, and F. M. Ausubel. 2000. Positive correlation between virulence of *Pseudomonas aeruginosa* mutants in mice and insects. J. Bacteriol. 182:3843–3845.
31. Korth, H., A. Römer, H. Budzikiewicz, and G. Pulverer. 1978. 4,9-Dihydroxyphenazine-1,6-dicarboxylic acid dimethyl ester and the “missing link” in phenazine biosynthesis. J. Gen. Microbiol. 104:299–303.
32. Latifi, A., M. K. Wilson, M. Foglino, B. W. Bycroft, G. S. A. B. Stewart, A. Lazdunski, and P. Williams. 1995. Multiple homologues of LuxR and LuxI control expression of virulence determinants and secondary metabolites through quorum sensing in *Pseudomonas aeruginosa* PAO1. Mol. Microbiol. 17:333–343.
33. Lauredo, I. T., J. R. Sabater, A. Ahmed, Y. Botvinnikova, and W. N. Abraham. 1998. Mechanism of pyocyanin- and 1-OH-PHZ-induced lung neutrophilia in sheep airways. J. Appl. Physiol. 85:2298–2304.
34. Leisinger, T., and R. Margraff. 1979. Secondary metabolites of the fluorescent pseudomonads. Microbiol. Rev. 43:422–442.
35. Mahajan-Miklos, S., M.-W. Tan, L. G. Rahme, and F. M. Ausubel. 1999. Molecular mechanisms of bacterial virulence elucidated using a *Pseudomonas aeruginosa-Caenorhabditis elegans* pathogenesis model. Cell 96:47–56.
36. Mavrodi, D. V., V. N. Ksenzenko, R. F. Bonsall, R. J. Cook, A. M. Boronin, and L. S. Thomashow. 1998. A seven-gene locus for synthesis of phenazine-1-carboxylic acid by *Pseudomonas fluorescens*. J. Bacteriol. 180:2541–2548.
37. Mazzola, M., R. J. Cook, L. S. Thomashow, D. M. Weller, and L. S. Pierson III. 1992. Contribution of phenazine antibiotic biosynthesis to the ecological

competence of fluorescent pseudomonads in soil habitats. Appl. Environ. Microbiol. 58:2616–2624.

38. Miller, R., G. T. Rasmussen, C. D. Cox, and B. E. Britigan. 1996. Protease cleavage of iron-transferrin augments pyocyanin-mediated endothelial cell injury via promotion of hydroxyl radical formation. Infect. Immun. 64:182–188.

39. Parke, J. L. 1990. Population dynamics of *Pseudomonas cepacia* in the pea spermosphere in relation to biocontrol of *Pythium*. Phytopathology 80:1307–1311.

40. Parke, J. L., R. E. Rand, A. E. Joy, and E. B. King. 1991. Biological control of Pythium damping-off and Aphanomyces root rot of peas by application of *Pseudomonas cepacia* or *Pseudomonas fluorescens* to seed. Plant Dis. 75:987–992.

41. Pesci, E. C., J. P. Pearson, P. C. Seed, and B. H. Iglewski. 1997. Regulation of *las* and *rhl* quorum sensing in *Pseudomonas aeruginosa*. J. Bacteriol. 179:3127–3132.

42. Pierson, L. S., III, and L. S. Thomashow. 1992. Cloning and heterologous expression of the phenazine biosynthetic locus from *Pseudomonas aureofaciens* 30-84. Mol. Plant-Microbe Interact. 5:330–339.

43. Pierson, L. S., III, T. Gaffney, S. Lam, and F. Gong. 1995. Molecular analysis of genes encoding phenazine biosynthesis in the biological control bacterium *Pseudomonas aureofaciens* 30-84. FEMS Microbiol. Lett. 134:299–307.

44. Pierson, L. S., D. W. Wood, and E. A. Pierson. 1998. Homoserine lactone-mediated gene regulation in plant-associated bacteria. Annu. Rev. Phytopathol. 36:207–225.

45. Propst, C., and L. Lubin. 1979. Light-mediated changes in pigmentation of *Pseudomonas aeruginosa* cultures. J. Gen. Microbiol. 113:261–266.

46. Rahme, L. G., M.-W. Tan, L. Le, S. M. Wong, R. G. Tompkins, S. B. Calderwood, and F. M. Ausubel. 1997. Use of model plant hosts to identify *Pseudomonas aeruginosa* virulence factors. Proc. Natl. Acad. Sci. USA 94:13245–13250.

47. Reimmann, C., M. Beyeler, A. Latifi, H. Winteler, M. Foglino, A. Lazdunski, and D. Haas. 1997. The global activator GacA of *Pseudomonas aeruginosa* PAO positively controls the production of the autoinducer N-butyryl-homoserine lactone and the formation of the virulence factors pyocyanin, cyanide, and lipase. Mol. Microbiol. 24:309–319.

48. Reitzer, L. J., and B. Magasanik. 1987. Ammonia assimilation and the biosynthesis of glutamine, glutamate, aspartate, asparagine, L-alanine, and β-alanine, p. 302–320. In F. Neidhardt, J. L. Ingraham, K. B. Low, B. Magasanik, M. Schaechter, and H. E. Umbarger (ed.), *Escherichia coli and Salmonella typhimurium*: cellular and molecular biology. ASM Press, Washington, D.C.

49. Römer, A., and R. B. Herbert. 1982. Further observations on the source of nitrogen in phenazine biosynthesis. Z. Naturforsch. C 37:1070–1074.

50. Schweizer, H. P. 1992. Allelic exchange in *Pseudomonas aeruginosa* using novel ColE1-type vectors and a family of cassettes containing portable *oriT* and the counter-selectable *Bacillus subtilis sacB* marker. Mol. Microbiol. 6:1195–1204.

51. Sheikh, N. M., and J. C. MacDonald. 1964. Biogenesis of the N-methyl group of pyocyanine. Can. J. Microbiol. 10:861–866.

52. Smirnov, V. V., and E. A. Kipriyanova. 1990. Bacteria of *Pseudomonas* genus, p. 100–111. Naukova Dumka, Kiev, USSR.

53. Stover, C. K., X. Q. Pham, A. L. Erwin, S. D. Mizoguchi, P. Warrener, M. J. Hickey, F. S. L. Brinkmann, W. O. Hufnagle, D. J. Kowalik, M. Lagrou, R. L. Garber, L. Goltry, E. Tolentino, S. Westbrock-Wadman, Y. Yuan, L. L.

Brody, S. N. Coulter, K. R. Folger, A. Kas, K. Larbig, R. Lim, K. Smith, D. Spencer, G. K.-S. Wong, Z. Wu, I. T. Paulsen, J. Reizer, M. H. Saier, R. E. W. Hancock, S. Lory, and M. V. Olson. 2000. Complete genome sequence of *Pseudomonas aeruginosa* PAO1: an opportunistic pathogen. Nature 406:959–964.

54. Suh, S.-J., L. Silo-Suh, D. E. Woods, D. J. Hassett, S. E. H. West, and D. E. Ohman. 1999. Effect of *rpoS* mutation on the stress response and expression of virulence factors in *Pseudomonas aeruginosa*. J. Bacteriol. 181:3890–3897.

55. Thomashow, L. S., and D. M. Weller. 1996. Current concepts in the use of introduced bacteria for biological disease control: mechanisms and antifungal metabolites, p. 187–235. In G. Stacey and N. T. Keen (ed.), Plant-microbe interactions. Chapman and Hall, New York, N.Y.

56. Thomashow, L. S., D. W. Essar, D. K. Fujimoto, L. S. Pierson III, C. Thrane, and D. M. Weller. 1993. Genetic and biochemical determinants of phenazine antibiotic production in fluorescent pseudomonads that suppress take-all disease of wheat, p. 535–541. In E. W. Nester and D. P. S. Verma (ed.), Advances in molecular genetics of plant-microbe interactions—1993. Kluwer Academic Publishers, Dordrecht, The Netherlands.

57. Turner, J. M., and A. J. Messenger. 1986. Occurrence, biochemistry and physiology of phenazine pigment production. Adv. Microb. Physiol. 27:211–275.

58. Vukomanovic, D. V., D. E. Zoutman, J. A. Stone, G. S. Marks, J. F. Brien, and K. Nakatsu. 1997. Electrospray mass-spectrometric, spectrophotometric and electrochemical methods do not provide evidence for the binding of nitric oxide by pyocyanine at pH 7. Biochem J. 322:25–29.

59. Warren, J. B., R. Loi, N. B. Rendell, and G. W. Taylor. 1990. Nitric oxide is inactivated by the bacterial pigment pyocyanin. Biochem. J. 266:921–923.

60. Watson, D., J. MacDermot, R. Wilson, P. J. Cole, and G. W. Taylor. 1986. Purification and structural analysis of pyocyanin and 1-OH-PHZ. Eur. J. Biochem. 159:309–313.

61. Weller, D. M. 1983. Colonization of wheat roots by a fluorescent pseudomonad suppressive to take-all. Phytopathology 73:1548–1553.

62. West, S. E. H., H. P. Schweizer, C. Dall, A. K. Sample, and L. J. Runyen-Janecky. 1994. Construction of improved *Escherichia-Pseudomonas* shuttle vectors derived from pUC18/19 and sequence of the region required for their replication in *Pseudomonas aeruginosa*. Gene 148:81–86.

63. Whiteley, M., K. M. Lee, and E. P. Greenberg. 1999. Identification of genes controlled by quorum sensing in *Pseudomonas aeruginosa*. Proc. Natl. Acad. Sci. USA 96:13904–13909.

64. Wilson, R., D. A. Sykes, D. Watson, A. Rutman, G. W. Taylor, and P. J. Cole. 1988. Measurement of *Pseudomonas aeruginosa* phenazine pigments in sputum and assessment of their contribution to sputum sol toxicity for respiratory epithelium. Infect. Immun. 56:2515–2517.

65. Yabuuchi, E., and A. Ohyama. 1972. Characterization of “pyomelanin”-producing strains of *Pseudomonas aeruginosa*. Int. J. Syst. Bacteriol. 22:53–64.

66. Yang, S.-W., L.-J. Lin, G. A. Cordell, P. Wang, and D. G. Corley. 1999. O- and N-methylation in the biosynthesis of staurosporine. J. Nat. Prod. 62:1551–1553.

67. Yanisch-Perron, C., J. Viera, and J. Messing. 1985. Improved M13 phage vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119.

68. Yoshida, K.-I., Y. Fujita, and S. D. Ehrlich. 1999. Three asparagine synthetase genes of *Bacillus subtilis*. J. Bacteriol. 181:6081–6091.
