
Molecular Microbiology (2001) 42(3), 573–585

# CheC is related to the family of flagellar switch proteins and acts independently from CheD to control chemotaxis in *Bacillus subtilis*

John R. Kirby,$^{1\dagger}$ Christopher J. Kristich,$^{1}$  
Michael M. Saulmon,$^{1}$ Michael A. Zimmer,$^{1}$  
Liam F. Garrity,$^{1\dagger}$ Igor B. Zhulin$^{2}$ and George W. Ordal$^{1*}$  

$^{1}$ Department of Biochemistry, Colleges of Medicine and Liberal Arts and Sciences, University of Illinois, Urbana, IL 61801, USA.  
$^{2}$ School of Biology, Georgia Institute of Technology, 310 Ferst Drive, Atlanta, GA 30332-0230, USA.

## Summary

Chemotaxis by *Bacillus subtilis* requires the interacting chemotaxis proteins CheC and CheD. In this study, we show that CheD is absolutely required for a behavioural response to proline mediated by McpC but is not required for the response to asparagine mediated by McpB. We also show that CheC is not required for the excitation response to asparagine stimulation but is required for adaptation while asparagine remains complexed with the McpB chemoreceptor. CheC displayed an interaction with the histidine kinase CheA as well as with McpB in the yeast two-hybrid assay, suggesting that the mechanism by which CheC affects adaptation may result from an interaction with the receptor-CheA complex. Furthermore, CheC was found to be related to the family of flagellar switch proteins comprising FliM and FliY but is not present in many proteobacterial genomes in which CheD homologues exist. The distinct physiological roles for CheC and CheD during *B. subtilis* chemotaxis and the observation that CheD is present in bacterial genomes that lack CheC indicate that these proteins can function independently and may define unique pathways during chemotactic signal transduction. We speculate that CheC interacts with flagellar switch components and dissociates upon CheY-P binding and subsequently interacts with the receptor complex to facilitate adaptation.

Accepted 15 June, 2001. *For correspondence.* E-mail g-ordal@uiuc.edu; Tel. (+1) 217 333 9098; Fax (+1) 217 333 8868.  
Present addresses: $^{\dagger}$Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720, USA. $^{\ddagger}$Pierce Chemical Co., Rockford, IL 61105, USA.

© 2001 Blackwell Science Ltd

## Introduction

Chemotactic organisms respond behaviourally to chemo-effectors by altering the flux of phosphoryl groups through the two-component system that comprises the kinase CheA and its cognate response regulator CheY (for reviews, see Parkinson, 1993; Stock and Surette, 1996; Armitage, 1999; Hoch, 2000). In *Escherichia coli*, chemoreceptors (methyl-accepting chemotaxis proteins, MCPs) are coupled by CheW to the CheA kinase to regulate phosphoryl group flux through the system during the excitation response to repellent stimuli (Hess *et al.*, 1988; Liu and Parkinson, 1989; Borkovich and Simon, 1990; Eisenbach, 1991). Adaptation to a repellent stimulus depends on dephosphorylation of CheY-P, enhanced by oligomeric CheZ (Hess *et al.*, 1988; Blat and Eisenbach, 1996; Bren *et al.*, 1996; Wang and Matsumura, 1996) and net demethylation of the chemoreceptors (Goy *et al.*, 1977; 1978).

Chemotaxis by *Bacillus subtilis* deviates substantially from the *E. coli* paradigm in many ways. First, *B. subtilis* responds to all amino acids as attractants, whereas only certain amino acids are attractants while others, such as leucine, act as repellents for *E. coli* (Tso and Adler, 1974). Secondly, in *B. subtilis*, attractants (rather than repellents) enhance the flux of phosphoryl groups through CheA to CheY, resulting in smooth swimming rather than tumbling as in *E. coli*. Thirdly, adaptation to asparagine stimulation in *B. subtilis* is dependent on an apparent CheY-P feedback that is coupled to complex receptor methylation changes on McpB (Kirby *et al.*, 1999). Fourthly, site-specific methylation changes occur sequentially on McpB (Zimmer *et al.*, 2000) and are required to preset the ligand-bound McpB complex for a response to negative stimuli. CheB-mediated demethylation of ligand-bound McpB is necessary to generate a tumbling response when asparagine concentrations decrease (Kirby *et al.*, 2000). In *E. coli*, methylation changes are thought to affect adaptation and do not appear to be required for swimming or tumbling responses. Fifthly, *B. subtilis* lacks the CheZ phosphatase present in *E. coli* and other closely related γ-proteobacteria.

Yet another major difference between *E. coli* and *B. subtilis* is that *B. subtilis* possesses two additional proteins, CheC and CheD, that have been shown to affect

adaptation. CheC and CheD interact and regulate methylation of the chemoreceptors in *B. subtilis* by an unknown mechanism (Rosario *et al.*, 1995; Rosario and Ordal, 1996). A *cheC* mutant was shown to methylate its receptors well above wild-type levels, display a high counterclockwise (CCW) flagellar rotation bias (smooth swimming) and responded but did not adapt to positive stimulation by a proline analogue, azetidine-2-carboxylate. In contrast, a *cheD* mutant was shown to have severely undermethylated receptors, displayed an extremely low flagellar rotation bias (tumbly) and was unable to respond to azetidine-2-carboxylate. Because the *cheD* mutant did not respond to azetidine, it was inferred that CheD was required for all attractants to induce CheA activity during chemotaxis in *B. subtilis* (Rosario *et al.*, 1995). Moreover, it was inferred that the CheC–CheD interaction was the most important facet of this regulation of adaptation (Rosario and Ordal, 1996).

Subsequent analysis of a *cheCD* double mutant, however, demonstrated that the *cheCD* mutant cells were capable of producing methanol in response to asparagine (Kirby *et al.*, 1997). Methanol production results from CheA-dependent activation of the CheB methylesterase (Goldman *et al.*, 1984; Kirsch *et al.*, 1993). Thus, asparagine stimulation does not require CheD for the activation of the chemotaxis machinery in *B. subtilis*. It is now known that asparagine is sensed by McpB (Hanlon and Ordal, 1994) and that azetidine-2-carboxylate and proline are sensed by McpC (Müller *et al.*, 1997). Therefore, McpB- and McpC-mediated taxes in *B. subtilis* have different requirements for CheD. In this study, we characterize further the role of the interacting proteins CheC and CheD during chemotaxis in *B. subtilis*.

to pre-stimulus levels. After removal of the stimulus, the flagellar bias decayed more rapidly and returned to pre-stimulus levels. In contrast, none of the *cheCD* mutant cells responded to stimulation by proline (not shown) or its analogue azetidine-2-carboxylate (Fig. 1A). Analysis of the *cheD* mutant gave similar results. Altogether, 53% of the *cheD* mutant cells responded behaviourally to asparagine but not to proline (not shown) or azetidine-2-carboxylate (Fig. 1B). The data shown are those for cells that responded. It is worth noting that *cheC* and *cheD* lie at the distal end of the major *fla/che* operon in *B. subtilis* and that the mutants used in this study show no polar effects (Rosario *et al.*, 1995).

The data in Fig. 1 also indicate that the *cheD* mutant differs from the *cheCD* mutant. The *cheD* mutant partially adapts by lowering its flagellar rotation bias within 90 s after asparagine stimulation, whereas the *cheCD* mutant does not (compare with wild type in Fig. 5A). However, the *cheD* mutant does not complete adaptation, as the flagellar bias does not return to pre-stimulus levels and is observed to increase gradually after the initial partial adaptation. We attribute the adaptation difference between these two strains to the presence of CheC in the *cheD* mutant. We therefore conclude that CheC plays a role in generating the adapted state or in maintaining adaptation after stimulation by the chemoattractant asparagine. CheC-dependent adaptation is also demonstrated below (Fig. 5B). A similar phenomenon has been observed for several *B. subtilis* chemotaxis mutants, including the *cheB* mutant, in which the adapted state is not maintained after the addition of asparagine (M. Saulmon and G. Ordal, unpublished).

The results from Fig. 1 lead to the following conclusions: (i) because both the *cheD* and the *cheCD* mutants have low pre-stimulus flagellar rotation biases (indicating low CheA activity), whereas the *cheC* mutant has a high pre-stimulus flagellar rotation bias (Rosario *et al.*, 1995), *cheD* is epistatic over *cheC*; (ii) CheD is required to generate the wild-type pre-stimulus level of CheA activity by the signalling complex; and (iii) independent of CheD, CheC is involved in generating an adapted state with the asparagine-bound McpB receptor complex.

Results

*Bacillus subtilis cheD* mutants respond behaviourally to asparagine stimulation but not to proline stimulation

To assess the role of CheD in McpB- and McpC-mediated chemotaxis, both *cheD* (strain OI2934) and *cheCD* (strain OI3305) mutant cells were analysed by the tethered cell assay for their ability to exhibit behavioural responses to asparagine (via McpB) and proline (via McpC). Because the *cheCD* mutant cells were able to activate the methylesterase in response to asparagine stimulation (Kirby *et al.*, 1997), we predicted that the *cheCD* mutant cells would also phosphorylate CheY in response to asparagine stimulation and, therefore, display an observable behavioural response. Some 55% of the *cheCD* mutant cells analysed showed a strong response to the asparagine stimulus (Fig. 1A). However, the *cheCD* mutant cells did not adapt normally and displayed a gradual decrease in flagellar rotation bias without returning

*Bacillus subtilis cheD* mutants exhibit chemotaxis to asparagine but not to proline

A more sensitive assay for chemotaxis is the capillary assay. Do the above behavioural data from the tethered cell assay correlate with an ability of the cells to detect a gradient of attractant in the capillary assay? Assays were performed on the *cheC*, *cheD* and *cheCD* mutant strains (Fig. 2). It was reported previously that the *cheD* mutant was unable to migrate into capillary tubes in response to all amino acid attractants (Rosario *et al.*, 1995). We repeated

Fig. 1. Behaviour of *cheCD* and *cheD* tethered cells in response to asparagine and azetidine-2-carboxylate stimulation. The assay was performed as described in Experimental procedures. The cells were exposed to the effector for a 4 min window beginning at 240 s and ending at 480 s. The rotational data for at least 15 cells were averaged to generate each plot. The responses of the (A) *cheCD* and (B) *cheD* mutant cells are shown to 0.5 mM asparagine (open circles) or 10 μM azetidine-2-carboxylic acid (solid line), a non-metabolizable analogue of proline.

those experiments over a larger concentration range for both asparagine and proline. The results again show that the *cheD* and *cheCD* mutants have the same phenotype, indicating that *cheD* is epistatic to *cheC*. Furthermore, neither the *cheD* nor the *cheCD* mutant cells were able to exhibit chemotaxis to any concentration of proline tested (Fig. 2B). However, a small but significant number of *cheD* and *cheCD* mutant cells (~3–5% of wild type) were able to respond to asparagine at the highest concentrations tested (Fig. 2A). The ability of the *cheD* and *cheCD* mutant cells to accumulate in the capillary tube to this extent is consistent with the observation that the *cheD* and *cheCD* mutant cells were able to respond to asparagine in the tethered cell assay (Fig. 1). The large reduction relative to wild type in the total number of *cheD* and *cheCD* mutant cells accumulating in the capillary tube is attributable to the extreme tumbling bias as well as the incomplete adaptation displayed by these mutant cells in the tethered cell assay. In contrast, relatively high numbers of *cheC* mutant cells were able to accumulate in the capillary tubes containing asparagine (~30% of wild type) but displayed a large reduction in chemotaxis to proline (~2% of wild type). The observation that the *cheD* and *cheCD* mutant cells are unable to respond to proline in either the capillary assay (Fig. 2) or the tethered cell assay (Fig. 1) demonstrates that McpC-mediated signalling is critically dependent upon CheD. Because McpB-mediated signalling is not absolutely dependent upon CheD, McpB and McpC must transduce signals via partially distinct pathways during chemotaxis in *B. subtilis*. The observation that the *cheC* mutant displayed a greater reduction in chemotaxis to proline than asparagine (Fig. 2) also demonstrates this point.

Yet another test for chemotaxis is the swarm plate assay. Again, the same mutants were assayed for their ability to form chemotactic rings in swarm plates containing either asparagine or proline amino acid attractants (Fig. 3). Individual chemoreceptor mutants are shown as controls for signalling specificity; the *mcpB* mutant was unable to form a chemotactic ring in response to asparagine, and the *mcpC* mutant was unable to respond to proline. The results indicate that the *cheD* and *cheCD* mutants were unable to respond to proline but did form chemotactic rings in response to asparagine. The rings formed by the *cheD* and *cheCD* mutant cells in asparagine swarm plates are faint but observable (Fig. 3, white arrows) and are not as clearly defined as those formed by the wild type. This partial yet distinct response to asparagine by the *cheD* and *cheCD* mutant cells is consistent with the observations listed above from these same mutants in the capillary and tethered cell assays. Additionally, the *cheC* mutant formed a sharper

Fig. 2. Capillary assay for chemotaxis by the *cheC*, *cheCD* and *cheD* mutants. The assay was performed as described in Experimental procedures. The data represent the number of cells (after subtracting background) migrating into the capillary tube containing either (A) asparagine or (B) proline chemoattractants at various concentrations. Responses shown are by the wild type (OI1085; solid line), the *cheC* mutant (OI3135; open circles), the *cheD* mutant (OI2934; open squares) and the *cheCD* mutant (OI3305; closed circles).

© 2001 Blackwell Science Ltd, Molecular Microbiology, 42, 573–585

576 J. R. Kirby et al.

chemotactic ring in the swarm plate assay in response to asparagine than that formed in response to proline, again consistent with the results from the capillary assay.

Bacillus subtilis cheD mutants express wild-type levels of McpB and McpC

In *E. coli*, low levels of receptor methylation have been correlated with low levels of CheA activation (Borkovich et al., 1992). Previous data from our laboratory indicated that the *B. subtilis* cheD mutant cells had decreased levels of total methylation relative to the wild type (Rosario et al., 1995). The low level of receptor methylation in the cheD mutant was considered to be the most likely cause for the low level of CheA activation by that mutant. However, the possibility remains that receptor degradation could occur in the cheD mutant and account for the overall decrease in methylation and the lower levels of CheA activity. Furthermore, the lack of a behavioural response to proline and azetidine-2-carboxylate by the cheD and cheCD mutants in both the tethering assay (Fig. 1) and the capillary assay (Fig. 2) could result from the degradation of the proline receptor McpC. To test this possibility, immunoblot analysis of the cheD and cheCD mutants was performed using anti-McpB and anti-McpC polyclonal antibodies (Fig. 4). The results show that anti-McpB antibody cross-reacts with both McpA and McpB under the conditions tested (Fig. 4, top; compare lanes 2 and 3). The results also indicate that anti-McpC antibody is specific for McpC under these conditions (Fig. 4, bottom; compare lanes 1 and 4). Based on these specificities, it is apparent that the cheD mutant cells express near wild-type levels of McpB and McpC (Fig. 4, lanes 6). The cheCD mutant showed the same relative levels of both McpB and McpC (not shown). Therefore, the lack of response to proline or azetidine-2-carboxylate stimulation by the cheD and cheCD mutants in these assays is not caused by the absence of the McpC chemoreceptor. The results allow us to conclude that McpC-mediated responses to proline or azetidine-2-carboxylate are strictly dependent on CheD for the proper activation of the signalling complex. We also note that the cheD mutant appears to lack the cross-reacting material that constitutes McpA (Fig. 4, lanes 6).

A B. subtilis cheC mutant is deficient in adaptation to asparagine stimulation

The data above allow us to conclude that CheC is required for optimal chemotaxis to both proline and asparagine. Furthermore, the data in Fig. 1 indicate that CheC is required for adaptation after asparagine stimulation. However, both mutants tested in Fig. 1 lacked CheD. We therefore analysed the effect of a cheC mutation in a cheD+ background. The cheC mutant (OI3135) had

been analysed previously for a response to azetidine-2-carboxylate but not to asparagine (Rosario et al., 1995). Cells from strain OI3135 were analysed for their ability to respond to asparagine. The results show that the cheC mutant cells excite normally in response to the addition of asparagine (Fig. 5B). The cells then display partial

![Figure 3](attachment://image.png)

Fig. 3. Swarm plate assay of cheD mutants. The assay was performed as described in Experimental procedures.

A. A swarm plate containing 0.3 mM asparagine inoculated with (clockwise from top) the wild type (OI1085), the cheD mutant (OI2934), the cheCD mutant (OI3305), the cheC mutant (OI3135) and the mcpB mutant (OI3056).

B. A swarm plate containing 0.2 mM proline inoculated with (clockwise from top) the wild type (OI1085), the cheD mutant (OI2934), the cheCD mutant (OI3305), the cheC mutant (OI3135) and the mcpC mutant (OI3280). Faint chemotactic rings (indicated by the black arrow) are present around the cheD and cheCD mutants in response to asparagine.

© 2001 Blackwell Science Ltd, Molecular Microbiology, 42, 573–585

searches of public protein sequence databases (see Experimental procedures) using the full-length sequence of the *B. subtilis* CheC protein. CheC orthologues were identified in the genomes of *Thermotoga maritima*, Gram-positive bacteria *Bacillus stearothermophilus, Clostridium acetobutylicum, Clostridium difficile*, as well as in the Archaea *Archaeoglobus fulgidis, Pyrococcus horikoshii, Pyrococcus abyssi* and *Halobacterium salinarum* (Fig. 6A). CheD orthologues were also found in these genomes (Fig. 6B). In previous work, CheC was shown to interact and be translationally coupled with CheD in *B. subtilis* (Rosario et al., 1995; Rosario and Ordal, 1996). Similarly, the *cheC* and *cheD* genes appear to be present

adaptation, as indicated by the decrease in flagellar rotation bias to ≈75% CCW, whereas asparagine remains complexed with McpB. After removal of the asparagine, the *cheC* mutant cells immediately tumble and then adapt to the negative stimulus. The behavioural response to asparagine stimulation by wild-type cells is shown for comparison (Fig. 5A). From these data, we conclude that CheC is required to bring about adaptation to positive stimuli.

**CheC interacts with CheA in the yeast two-hybrid assay**

A reasonable assumption based on the above data is that CheC and CheD interact with the receptor ternary complex to affect both excitation and adaptation. To assess possible interactions, we generated constructs to assay the ability of CheC and CheD to interact with CheA or the cytoplasmic signalling domain (the carboxy-terminus) of McpB in the yeast two-hybrid system. The results indicate that CheA interacts with CheC but not with CheD under these conditions (Table 1). The data also show that both CheC and CheD are capable of interacting with McpB directly. The specific sites of interaction between these proteins remain unknown. By this same assay, it was shown previously that CheC and CheD interact (Rosario and Ordal, 1996), and this is shown as a positive control (Table 1). From these data, we speculate that CheC interacts directly with the receptor–CheA complex to lower CheA activity, thereby facilitating adaptation.

**cheC is absent from, but cheD is present in proteobacterial genomes**

No reports have been made describing the role of CheC or CheD in other chemotactic organisms. In order to identify CheC and CheD homologues and possibly other related proteins in different microbial species, we performed

© 2001 Blackwell Science Ltd, *Molecular Microbiology*, 42, 573–585

Fig. 4. Immunoblot analysis of the *cheD* mutant. The assay was performed as described in Experimental procedures. Anti-McpB antibody (top; 1:100 000 dilution) and anti-McpC antibody (bottom; 1:50 000 dilution) were used to probe extracts from: 1, the wild type (OI1085); 2, the *mcpA* mutant (OI3055); 3, the *mcpB* mutant (OI3056); 4, the *mcpC* mutant (OI3280); 5, the mutant that does not express *mcpA, mcpB, mcpC, tlpA, tlpB* or *tlpC* (OI3281); and 6, the *cheD* mutant (OI2934). The *cheD* mutant expresses both McpB and McpC.

Fig. 5. Behaviour of a *cheC* mutant expressing McpB in response to asparagine stimulation. The assay was performed as described in Experimental procedures. The cells were exposed to 0.5 mM asparagine beginning at the time indicated by the downward grey arrow and ending at the time indicated by the upward grey arrow. The rotational data for at least 15 cells were averaged to generate each plot. The responses by (A) the wild type (OI1085) and (B) the *cheC* mutant (OI3135) are shown.

578 J. R. Kirby et al.

Table 1. Yeast two-hybrid analysis.

| DNA-binding domain (insert) | GAL activation domain (insert) | Relative β-galactosidase activity$^a$ |
|----------------------------|------------------------------|----------------------------------------|
| cheC                       | –                            | –                                      |
| cheD                       | –                            | –                                      |
| –                          | cheD                         | –                                      |
| cheC                       | cheD                         | +++++ (2h)                             |
| –                          | cheA                         | –                                      |
| cheC                       | cheA                         | + (8h)                                 |
| cheD                       | cheA                         | –                                      |
| –                          | mcpB$^b$                     | –                                      |
| mcpB                       | –                            | –                                      |
| cheC                       | mcpB                         | +++ (3h)                               |
| mcpB                       | cheD                         | +++ (3h)                               |

a. The relative β-galactosidase expression level (+) was estimated by comparing the intensity of blue colour present on the filters after 23 h incubation at 30°C. Plasmid combinations that yielded no blue colour are indicated by –. The time when blue colour was first observed is indicated in parentheses. The interaction observed using plasmid combination pMR136 and pMR137 was reported previously (Rosario and Ordal, 1996) and was used as a positive control for these experiments.

b. Only the portion of the mcpB gene encoding the cytoplasmic (soluble) signalling domain of the receptor was cloned into either the activation domain or the binding domain construct.

within the same operon (usually chemotaxis operons) in most of the above-mentioned bacterial and archaeal species, as revealed by inspection of individual genomes at The Institute for Genomic Research Comprehensive Microbial Resource (http://www.tigr.org/tigr-scripts/CMR2/CMRHomePage.spl). However, our results here indicate that CheC and CheD function in part independently of each other. This conclusion is supported further by the genomic analysis. We have identified CheD homologues in all species in which CheC was present, as well as in many proteobacterial species in which CheC was absent, namely Sinorhizobium meliloti, Bordetella pertussis, Bordetella bronchiseptica, Shewanella putrefaciens, Pseudomonas aeruginosa, Vibrio cholerae and Caulobacter crescentus (Fig. 7). Because CheC and CheD are present in the most ancient organisms, we conclude that the evolutionarily young proteobacteria lost CheC but retained CheD. Both CheC and CheD are missing from the genomes of E. coli and Helicobacter pylori.

CheC is similar to the flagellar motor proteins

The observation that CheC interacts with several components of the chemotaxis signal transduction system, including CheA, CheD and McpB (Table 1), prompted further analysis of the protein sequence. A position-specific iterative (PSI) BLAST search (Altschul et al., 1997) with the full-length sequence of the B. subtilis CheC protein indicates that CheC is similar to several flagellar proteins (Fig. 6A). The initial search retrieved the B. subtilis FliY flagellar switch protein with a high degree of statistical significance ($E = 7 \times 10^{-7}$). Further iterations retrieved CheX, FliY and FliM proteins from a large number of microbial species with a confident statistical significance ($E < 1 \times 10^{-3}$). The function of CheX, which is found in T. maritima and three spirochaetal species, is unknown, whereas FliM is a switch protein of the bacterial flagellar motor (Bren and Eisenbach, 1998). No proteins with a known function other than CheC, CheX, FliM and FliY were found upon convergence of the PSI-BLAST

Fig. 6. Multiple alignments of CheC and CheC-like domains from flagellar proteins (A) and CheD proteins (B). Amino acid residues are coloured according to an 80% (A) or 90% (B) consensus (shown underneath the alignment): – indicates negatively charged residues (D and E, highlighted in red); + indicates positively charged residues (H, K and R, highlighted in blue); a indicates aromatic residues (F, H, W and Y, highlighted in yellow); c indicates charged residues (D, E, H, K and R, highlighted in purple); s indicates small residues (A, C, D, G, N, P, S, T and V, coloured in blue); l indicates aliphatic residues (I, L and V, highlighted in yellow); h indicates hydrophobic residues (A, C, F, G, H, I, L, M, T, V, W and Y, highlighted in yellow); p indicates polar residues (C, D, E, H, K, N, Q, S and T, coloured in red); t indicates turn-like residues (A, C, D, E, G, H, K, N, Q, R, S and T, coloured in green); u indicates tiny residues (A, G and S). Only a few FliM proteins are shown because of space limitation (A), whereas an 80% consensus was calculated for a complete alignment containing 42 sequences. Predicted secondary structure is indicated above the alignments: E indicates beta-strand and H indicates alpha-helix. Sequences included in the secondary structure prediction set are marked with an asterisk. Residue numbers and GenBank identifiers are shown at the end of each sequence. For sequences from unfinished microbial genomes, abbreviated sequencing centre and preliminary identifiers are shown: UOKNOR, University of Oklahoma; Sanger, The Sanger Centre; TIGR, The Institute for Genomic Research; PAGP, PathoGenesis; GTC, Genome Therapeutics. Species abbreviations: Afu, Archaeoglobus fulgidis; Atum, Agrobacterium tumefaciens; Bbro, Borrelia burgdorferi; Bper, Bordetella pertussis; Bste, Bacillus stearothermophilus; Bsub, Bacillus subtilis; Cace, Clostridium acetobutylicum; Ccre, Caulobacter crescentus; Cdif, Clostridium difficile; Ecol, Escherichia coli; Hpyl, Helicobacter pylori; Hsal, Halobacterium salinarum; Paby, Pyrococcus abyssi; Paer, Pseudomonas aeruginosa; Phor, Pyrococcus horikoshii; Sput, Shewanella putrefaciens; Smel, Sinorhizobium meliloti; Tden, Treponema denticola; Tmar, Thermotoga maritima; Tpal, Treponema pallidum; Vcho, Vibrio cholerae.

© 2001 Blackwell Science Ltd, *Molecular Microbiology*, **42**, 573–585

A 2D structure HH...HHHHHHHHHH.HH....EEEEEE....HHHHH.........EEEEEEEE...........EEEEEE.HHHHHHHHHHHH..............HHHHHHHHHHHHHHHHH
CheC1_Phor* ETFNIGASHAATALS-EMTGKEVNISVPDLKIIAIKNVPEVVG-----EDVKVAVIIELGKDFSSHAFFIADYEDALRMFDIIMGNPLGTTK---EMDKSSFMEVGNIL
CheC2_Paby ETFNIGASHAATALS-EMTGREVNISVPDLKVVEIKNVPEIVG-----EDKVVVYIELGKDFSSHAFFAADYDDVLRMFDVIMGNPPGTT---ELDDMVKSSEMEMGNIL
CheC2_Phor* EASNIAMSHAITALS-QMIGGPIEMEAPDVDIVSRVEFLKILAERG-VSKGFTVMFDITEGMSGTLTILQFPKHSALNISAVLGMMEPGSME---ELDEMGKSAIMEVGNIL
CheC1_Paby EASNIAMSHAITSLS-QMIGGPIEMEAPDVEIVSRAEFLKLLAERG-VSKGFTVMFDVTEGLSGLTILQFPKHSALNISAVLGMMEPGSME---ELDEMGKSAIMEVGNIL
CheC_Hsal* ELAREGASTVENMS-QLTGVETQMQLTKINVIDVEDLGAHLG-----AAKQGVSVPLKEQPYGSVLVLFDDESARRVAGTMGGIESEGG---GYSDMERSAIREVGNIM
CheC_Bsub* EVGNIGAGGHASSAMA-QLLNRKIDMBVPFAKLLSFDELVDFFGA---DVPVASIFLRMEGDLTGSMFFIMPFFAQEFIRELIGNPDFDIE-DLGEDHMSSSALHELGNIL
CheC_Bste EIGNIGAGNAATALS-TLLNKIEIMAVPRVQIATAFEAMMESIGGP--EQVVACYLRIEGEAPGNMMFFLPPEQAERFVRRMIGDDSFSFQ---GEALGC SALQELGNIL
CheC_Cace EVV NIGSGNAATALS-QLLNKKIDMTVPDINIVPDTTLSIEGE----EKVAVG VVVRLGDTPGNILVFVEKETAVEIVETLTGIKDDDILT------DLSSSVLCEIGNII
CheC_Tmar* EIGNIGAGNAATAIS-YMINKKEVISPVNEIVPISKVIFIAKDP---EEIVVGVKMPVTGDIEGSVLLIMGTTVKKILEILTRGAPDNLNN---LDEF SASALREIGNIM
CheC_Cdif EIGNIGSGNATALA-SMINSNEVLPIMVKILEYNEATNLLGGP----ENKVVCILLDMKGDNIGMFMLLDESITQLMILSSLFNKEEAFLD---ELEAEISEAIKEIGNM
CheC_Aful* EIGNIGVKAATLS-OMLGRRIEMSVEAVIVKIPELHKYINAEEVMAGVVTGLEDIENGHSGFLYITFPEKSSKKLAEILLGDS D----------EEMVDSTLMEIGNIL
CheX_Tden EYINPFSEAAYNILS-OVISEDIKRGDLYLKSTC-------MPVG-----VAAIVGLAGDVEGRVIFDMTLDTALKIASAMNEEELAEF------DELARATITELANLI
CheX_Tpal EYINPFSEAAYVVL-EVLAGETKRGDLYLKSTC-------MPVG-----VAAIVGLAGDVEGRVVFDMTLDTALKIASSMNEEKLAAF------DELARATITELANLI
CheX_Bbur DYIEPFLDAASSVLRDMLLVENIEMKGPKGLKSIN-----QKIKG-----VSIVGLAGSVEGSIIIDMDIETALFVASKLNFEYYDDFDD-EETKMVAATLTEVGNII
CheX_Tmar --MDARIVNALIGSVYETIRDVLGIEEPKTGKPSTVSHIEIPHS-----LVTVIGITGGIEGS LIYSFSSETALKVVSAMGGMEYN-----QLDEALSAIGELGNMT
FliY_Bsub* EIGNISFGSSATALS-TLLNQKVDITTPSVTVIPRSKISDAFP-----EPYVAIEVNYTEGFSGSNLLVVEQSDAAIADLMIGDGKGADP---SLGEIHLSAVQEAMNQM
FliY_Tpal SFCQANVGSPCSLE-SLTKGTVSISPPRVEISSKEEFRVIP-----DMIVATVIDFEGEFGADHLFLMAPEFAQKLVLNSHEDNDQI------DDMGLSVVSETITQY
FliY_Hpyl QDFIKIFIQEVVSTLEGVGKAPSVGLEKEISSSDESFLKLIS-----TPYARVVISAIBEKEESSIELLAPVVIVTSLSDLMLGGEGASKE---EMDNDLD AFKEMASNI
FliM_Tpal SLMHETFARLTTTSLSAQILRSMVHVHVASVDQLTYEEFIRSTP----TPSTLAVITMDPIKGNAVLEVDPSITFSIIDRLFGGT GQ-AAK--VQRDLTDIENSMEGVI
FliM_Hpyl RSTHDKMARNISSQVSSIMRSIVEIQLHSVDQM TYGEFLMSLP----SPTSFNVFSMKPMGGTVLEINPSIAFPMIDRLLGKGGS-AYD---QNREFSDIEINLLDTIL
FliM_Ecol EINERFARHERMGLFNNLLRRSPDITVGAIRIQPYHEFARNLP----VPTNINLIHLKPIRGTGLVVFSPLIVFIAVDNLFGGDGRFPTKV-EGREFTHTEQRVINRML
FliM_Bsub TRHDNFARLITTTHFSAQILRTYTHISVSSVDQVPYEEFIRSTP----NMTILNLFDVHPMEGRIMMEVNPTIAYTMMDRV MGGIGISHN---KVDSLTEIETKISNLF
FliM_Tmar QMIHENFGRALSTYLSGRLRTFDVEIS-IDQLTYEEFIRSTVM----IPSFIVFTGDVFEGSALIFEMRLDLFTMLDIIMGGPGENPP----NRPPTEIETSIMRKEV
FliM_Ccre EIVFDRIVRLMTTSLRNFTSDNEVESLDNISSTRFGDYINSIP----LPGILAVFRAEEILDNYGLLTVDSNL IYSIVDVLGGRRGTAAMRIEGRP YTTIERVLVQRM I
consensus/80% php.thsphp.....thhttp.hphths.lp.htthph.p.is........ssl.h.hhtshpGphhh.hs.p.sh.hhs.hhGtp.t.h.......c-hs..t.p.htthh

2D structure ........HHHHHHHHHHH....EE...........HHHHHHHHHHHHH...........EEEEEE...........HHHHHHHHH.........
CheC1_Phor ----ISAFANALSEFLGITIEQT PP--SLAIDFLPAILDEAL-ADIGKYCDYTILQTNIKISDV KFE---EHFL LFPKPED--MKKILEKLLGGFA--- (25-212) 3256892
CheC2_Paby ----ISAFANALSQFLGITIEQT PP--ALTIDFLPAILDAL-ADIGKFCDYTIIILQTDIKISGVEFE---EHFL LFPKPED--MKKILEKLLGLLA--- (25-212) 5458971
CheC2_Phor ----ISVYTDILSNLLIGEPVSL SPP--KPAESLYDIEKELGR-SDIRNVNS-VVFKSKFYKEDIRVE---SYFI IPTPES---FTKVKKLESQVSGEV (13-205) 3256893
CheC1_Paby ----ISAYTDILSNLLIGEPVSL SPP--KPAESLYDIEKELGR-PDIRNVNS-IVVFKSKFYKEDIGVE---SYFLVPTPES---FTKVKKLEQISEEV (13-205) 5458970
CheC_Hsal ----TSGFIDGWANVLGR--------TIDISTPOLIRASGE-DIA SHCVDPGHEIAMVFDAELHAPDANEAKIYSFD---IEAFVSMINSI---- (15-195) 1076149
CheC_Bsub ----AGSYLTALADLTKLQLYPSPV-EVSLDMFGAVISEGL-MELS QVGEHAIIVDTSIFDQSHQQELKAHMFMLPDYDS---FEKLFVALGASL--- (17-209) 732148
CheC_Bste ----AGSYLSALA--------------IDMIGAVLSFGL-LELS RVGDYAILIDTAIYDERRPDEISINGHFLLPDPDS--FSIIFRALG------UOKNOR_1422 1762
CheC_Cace ----SSSYMNAISKPTNLVITPSVP--AFSYDMLGAILTTSF-LEAQGQCDDN-VLDLETVFLDKTKN-ISGHFYIPMPGS---LEKILNILGVN---- GTC_1488 AE001437
CheC_Tmar ----CGTYVSALADF LGFKIDTLPP--QLVIDMISAI FAEASIELEDNSLDQIVFEETILLKVEEEEEPLTSYMMIPKPGY---LVKIFERMGIQE--- (13-205) 4981440
CheC_Cdif ----ASSYWNAIASMLNMTISVSIP--DICIDMGAVLNP M-IRESVDGDKVLFIBNKFKMSDNYFTS---HLMIPEMSS--KKILVRVLG------ Sanger_1496 433
CheC_Aful ----SSSFCDAAAEMLGIVLLPTPP--SFGRDFAI AVIDAVV-SQLAEBKSDYVVIFETKLEESBNEIEA---LVMLIPNEK---FVSYILKLLGMVE--- (20-205) 2649551
CheX_Tden ----TAQAVTKLH----------------DLGFKFDLTP-PALFTGEN---MKISNNDIBALIVP------MTAPQGR---VEINVAIRDRV---- (4-154) 3493637
CheX_Tpal ----TAKAVTTLH----------------ELGFKFDLTP-PALFTGDN---MEISSSDIBALIVP------METPQGK---VEINVAIRDKV---- (4-154) 1765975
CheX_Bbur ----AGNFVTTLH-----------------AKGFVDTTP-PAFIHGEN---MKISNKGSEALIVP------ISLPDGKI--IEVNIAIRERV---- (4-161) 2688605
CheX_Tmar ----AGKLAMKLE-----------------HLGHVDITP-PTVSGRD---LKIKSFGVILKLPIS-----VFSEEDFD---LHLSVKSGG------ (1-155) 4982190
FliY_Bsub ---MGSATSMSTVFSKKIDISPRVELLDVTEEKGTD RIPDDEMVLKVSFNKVGELIDSSIMQLYPLTFAKDLISSLMN--SES AEEETVQPEVTY (43-239) 585146
FliY_Tpal ---VS AOLSTLEARN-VQNVSVAP--AESVQVPKAMVRFP-PQLCVFTYTVELDGTSYTLWELVSEQLMVRIIERMGA---MQRPAQVGNSMQMGV (45-234) 2120517
FliY_Hpyl ---FGAIATSILKSQELLPKLNFTT--INA EIAKELPKKEDYAKAMVFSFKMEAIKEESQIILLTTAAFEGQFEKTHKEEKEE-TTEGVAEEVKTHDASLE (2-197) 2314175
FliM_Tpal ---VRILANVERESWTQVIDLRPRL--GQIETNPFAQIVPDP-SEMVLV TLETKVGEELGMMNFCIPYTIIEPILSKLSSQFWFSSVRRSSTTQYMGL (56-250) 6225364
FliM_Hpyl ---RQVMQILKEVWSPVVEMFTI---DAKESSANVQQIVAAQ-NEISIMVVLEIIIICH SRGMMNICYPVISIESILSKMGSRDLMLETSNKKSRNKLQ (57-251) 4154924
FliM_Ecol ---KLALEGYSDAWKAINPLEVEY--VRSEMQVKTNTTTS PNDIVVNTPFHVEIGNL TGGEFINCLPFSMIEPLRELLVNP-PLENSRNE-DQNRWDNL (55-250) 120345
FliM_Bsub ---ENALGNYKEAQSIA DIEPEM--TEFEVNPOFVQMVSP-NETVVVISLNTQIGEISGVINLCI PHIVLEPLIPKLSVHYWMQSDRNEPKPEETKSL (57-251) 120344
FliM_Tmar ---TNMLTLLAQAWSDFQYFIPSI--ENVETNPQFVQIVPP-NEIVLLVTASVSWGEFTSFINVCWPFSLEPLEKLSDRFWMMGRKPEKEERMEEL (56-248) 4981202
FliM_Ccre ---DVVLHDLKSAFEP LHPVSFSL--DRLETNPPFAAIARP-ANAAILVKLRI DMEDRGGRIBLLL PYATLEPIRKMLLQQ--FMGEKFGRDNIWEGHL (94-289) 462112
consensus/80% .......hh.shhthh...............thh...hs....phh.hs....h.htp...h.phhh........hh...st....hpt.htt.t.......

B 2D structure ................EEEEEEEEEEEEEE....EEEEEE...........EEEEEE................HHHHHHHHHH
CheD_Bsub -----------------MSTTEAVVTKVGIADV KIARFPDTIRTSGLSCVGLVLYDKEKQTAGIVVMLP DSTLSKTA---------ELNRAKYADTAVQT T
CheD_Bste -----------------QAVKIGIAEMEVVIAPN VIRTCGLSCGVVVIYDAGKAVAGMAHVMLPHSSMARGG---------VINAAKYADTAVEAL
CheD_Bper -TRRYFDNAFNSPA VKILPNEYV TNGEDVMLSTVLGSCVAACIHDPIVGGMNHFMLPEGDIHSP---------ASATMRYGAFAMEVL
CheD_Bbro -TRRYFDSA FNSPA VKILPNEYV TNGEDVMLSTVLGSCVAACIHDPIVGGMNHFMLPEGDIHSP---------ASATMRYGAFAMEVL
CheD1_Sput -----------------VKILPEFYATR-ENTMITVLGSCVACLYD PVLKIGGMNHFLLENDNVTAPN---------MMTESSARYGVFAMELL
CheD_Paer -----------------VKLLPEFYASG-EDLIVTVLGSCVSCLRD PLNGIAGMNHFMLBERGLGGDD---------ASPARYGSAMELL
CheD_Vcho -----------------FNHPSDHRHWVKVMPGGVYATS-DQEI IHTGLGSCVSACAWDIE MKVGGMNHFLLEFNNQFESQWH PQALLSDSSRYGCYAMEVL
CheD2_Sput -----------------VMIGPGEHYVTA-KNEVIKTLGLSCVACLYDPKAQVIGMNHFLLPADRRKFTH------FLDSRAGYYGV HAMEIL
CheD_Atum* -----------------MMEAAA KRVHI IQEKYVVSDPDVMTTILGSCVAACLRDPVAGLGGMNHFLLBGTGNT---------GGDATRYGVHME LL
CheD_Smel* -----------------MNT EEA GRRVHV IQEFKVNDPIDVLSTILGSCVAACMRDPAAGVGGMNHFLLGSA TSPSS---------GADATRYGVHME LL
CheD_Ccre* MT SFHHDDPERAVKVHVTQESHVTADPNVMTTVLGSCIAICRDPQSGVGGMNHFLBDSGDGRR---------DGDAVRYGAYAMEVL
CheD_Cdif -----------------EIVVNIADMKIA YRPNVIVTYAIGSCGVCLIDKVAGIGMLMVLPYSKDAIN--------IENKCKFADTG INEL
CheD_Cace -----------------ERENIKEIRVGIADINTAFSPNRIITVGLSCIGIAIYDSKNKLGLSHIMLPDSTQFSK---------VTNPYKFADLA INIL
CheD_Phor* -----------------MKEVKVGIQDAYVGKNGIIISTYGLGSCV GITLYDRVTKVGLLIHALLPEAARYGH--------RGNPAKYVDTGLQ LL
CheD_Paby -----------------M TREIKVGIQDAYVGKEGIIISTYGLGSCV GITLYDRVTKVGLLIHALLPEAARYGH--------RGNPAKYVDTGLQ LL
CheD_Tmar* -----------------MKKVIGIGEFRVAK-GAVLKTIGLGSCVGIALYDPKLRLLGGLAEBVMLPQSGNGTK---------RSAKYADHAVE MM
CheD_Aful* -----------------MGSEILVGIQGEFRVAK-GAVLKTIGLGSCVGIALYDPKLRLLGGLAEBVMLPQSGNGTK---------RSAKYADHAVE MM
CheD_Bbur* -MLNHFNFKLKR DVTTIIVPGEAFVSN--KRVISTILGSCVA AVLCDENNLIGMNHYVLVKSDLDIS--------PDQRGRYG IYAIAPML
consensus/90% ...............lhlh.s-h.hs...t.h...hhLGSCluhshhD...tlhGh.HhhLs.tt.............t.h+asshhhphh

2D JPRE D2 HHHHHHH..HHHHHHHHHH..HHHHH........HHHH.HHHHHHHHHHHH...EEEEEE........EEEEEE........
CheD_Bsub IDMLIEAGCRKFALKAIAGGSEMFKF KSTN--DLMKIGPRNVLAIKEISLFNIPPISED TGGSSSGRTIEFEPKS CMIHIRTVKQGEKT (1-165) 732149
CheD_Bste VQLVIAAGRKGMILKAKLAGGAQMFSFSTVGG-DDMMRIGARNVEEVKKQLERLHIPVVAEDVGGHSGRTIEFNPQTGVLSIRTA AQEIK- UOKNOR_1422 1762
CheD_Bper INELLKAGAVRERLEAKVFGGGAVLSAMQLMN-----GERNGQIVLNYFKTEGIPVR AQDLGDVHARRINYFPRDGRVMVRKMA----- Sanger_520 328
CheD_Bbro INELLKAGAVRERLEAKVFGGGAVLSAMQLMN-----GERNGQIVLNYFKTEGIPVR AQDLGDVHARRINYFPRDGRVMVRKMA----- Sanger_518 1983
CheD1_Sput INHVLKLGARRNAL EAKVFGGGNVLRGLTVQN-----GERNAEFVLDYLQMEQIPVIAADLLDIYPRKVYFFDETEGLVKVRKI----- TIGR s.ptr _5526
CheD_Paer INRMLALGASERL IQAKVFGGGSVLRQISAAA-----IGQRNVFEFTLDYLATERIPLLASDVLEAFPRKIYFFPRN GRV--------- PAGP_287 1
CheD_Vcho INRLLSMGAERERLKF KLFGGAAHLMGYQS LV-----GEKNVBFVLEYAKREKLNVVAQDLGGAO PRKLLFDPQTGQAWVKRIGFSSAH TIGR v.c._666_1741
CheD2_Sput INAMLKRG AQ RKYLQS KIFGGGANVLSLCADNILNHYDIGGMNIDFVRHFLQRERIPII SEDHGHCGRV IYFDPTDYSVYRSLI------ TIGR s.ptr _5567
CheD_Atum INGLLKQGARRDLREAKVFGGAKTIASFSNV-----GEQNAIFAMQFLKDEGIPVI SSTS TGGDHGRKIEFWPVSGRARQHPLSGAETQ (1-159) 3282796
CheD_Smel INGLLKQGARRDLREAKVFGGARTIARFSNV-----GEQNAFARQFLMDEGIRIVES TGGDHGKRLEYWPS SGRARQYALTGVEAQ (1-161) 2120809
CheD_Ccre INDLLKGARRERLEAKVFGGALKLFDGLSDV-----GASNAFAERFLRDEGIPIVS SSTGGVSARRVEFWPASGRVRQRLVAVDNAP (1-165) 3387368
CheD_Cdif IKSMENIANGRIYIKAKIAGGAQMFLGSSNS----VIASI GHRNVBEAVKKTLLGLNIPILAEDTGMNYGRTIRFYETBTEL--------- Sanger_1496 433
CheD_Cace IKKMEGMANIRNM KAKIAGGASMNFSDKN-MMNDIGNRNGISVKKVLKELNV PLLSQDIGGNGK GRTMIFNTLDGSVDIRTVGM----- GTC_1488 AE001437
CheD_Phor LKEVIKLGASKFRELAKLFGGAAQMFDNIRSE----ELKIGEKNVQVAKRELRKL GIRLVAEDTGGKGGR GTIYLDLS TGKVRMRKVTEGKV (1-158) 3256895
CheD_Paby LKEVKLGASKFRELAKLFGGAAQM FONIKSD----ELKIGERNVQOTAKRELLK KL GIRLVAEDTGGGRGGR GTIYLDLS TGKVRMRKVIGGQVI (1-159) 5458969
CheD_Tmar VEELKKMGAKVERLEAKTAGGASMFE SKGMN-----IGARNVEAVKKHLKDFGIGKLLAEDTGGGNARSVEYNIETGKLLVRKVGGGEOL (1-153) 4981439
CheD_Aful TEAMERLGS DRKRI VAKMAGGAQIFKHMTMD---MLRIGDRNAEAIRTILRDYGI RIVSEDLGGDEGRTVYFFTNDGRMLVKYSRGGELW (1-156) 2649552
CheD_Bbur INAMLENGASKSNL KAKLFGGTNFM AKGSVK-----VGLENS EFAVNTLNKYRIPILAKDFDQSKSRKIFA FPENF KVIVEYPDGTKVF (1-163) 2688536
consensus/90% lp.h.t.Gu.h..lpuKlhGGsth...t........G.pNs..hhp.h...tl.lhupshht..sRp l.h....sh.h.........

© 2001 Blackwell Science Ltd, Molecular Microbiology, 42, 573–585

separate functions during chemotaxis in *B. subtilis*:

(i) CheD is absolutely required for McpC-mediated signalling but not for McpB-mediated signalling;

(ii) CheD, but not CheC, is necessary for the generation of wild-type prestimulus CheA autophosphorylation levels;

(iii) CheC is necessary for adaptation to positive stimulation by asparagine via McpB; and

(iv) CheC interacts directly with CheA, whereas CheD does not.

The distinct physiological roles for CheC and CheD are supported further by the observation, based on the genomic analyses above, that these two proteins are not always present together in other chemotactic organisms. Together, these results allow us to conclude that, although CheC and CheD interact to facilitate chemotaxis, these proteins can act independently to affect chemotaxis in *B. subtilis*. The independent activity of these two proteins therefore defines at least two types of signalling pathways during *B. subtilis* chemotaxis. The presence of CheD in other chemotactic organisms may reflect the presence of unique chemosensing pathways in those organisms.

CheC and CheD have no known enzymatic activity and are therefore thought to facilitate chemotaxis by affecting the ternary receptor signalling complex. Because a majority of the *cheD* mutant cells can excite and partially adapt to asparagine stimulation (Fig. 1B), CheD is not required for coupling ligand-bound McpB to CheA. It is not known why some of the *cheD* mutant cells do not respond to asparagine stimulation. However, it is unlikely to result from variation in the level of CheC, because both the *cheD* and *cheCD* mutant subpopulations respond in the same proportions. Nevertheless, the level of CheA activity, as indicated by the prestimulus bias of both the *cheD* and *cheCD* mutants (about 5% CCW; Fig. 1), is well below that of the wild type (about 55% CCW; Fig. 5A). From this observation, we conclude that normal activation of the ternary signalling complex requires CheD. The low flagellar rotation bias of the *cheD* and *cheCD* mutants is probably the reason for poor taxis to asparagine in the capillary assay (Fig. 2) and the reduced chemotactic ring formation in the swarm plate assay (Fig. 3). Reduced receptor levels could have accounted for the low flagellar rotation bias and poor chemotaxis towards proline for these mutants, but was ruled out because receptor levels are normal, as indicated by the immunoblot assay (Fig. 4). One possibility is that CheD may influence higher order complexes within receptor clusters. Polar localization of the chemoreceptors appears to exist for *B. subtilis* (Kirby et al., 2000), and minor perturbations within a receptor array have been predicted to affect overall CheA activity levels (Bray et al., 1998; Levit et al., 1998).

CheC appears to function in part by interacting directly with CheA (Table 1). A direct interaction between CheC and CheA could prevent the kinase from becoming activated by the ligand-bound receptor complex, or

Discussion

Previous work has indicated that CheC and CheD interact to facilitate chemotaxis in *B. subtilis* (Rosario et al., 1995; Rosario and Ordal, 1996). In this study, however, we show that these two proteins display

lowering the overall level of CheA activation. However, the observation that the cheD mutant (in which CheC is present) adapts more effectively than the cheCD mutant (Fig. 1) and the cheC mutant displays incomplete adaptation (where CheD is present; Fig. 5) allows us to conclude that CheC plays a role in adaptation that is independent from CheD. CheC could interact with all proteins simultaneously within the ternary complex (McpB, CheA and CheD; Table 1) to help bring about and maintain an adapted state for B. subtilis. In this way, CheC could affect both CheA activity and receptor conformation such that altered methylation would occur as shown previously (Rosario et al., 1995).

The unexpected similarity of the CheC protein to FliM and FliY (Figs 6 and 8) raises the possibility that CheC may interact directly with components of the motor as well as with components of the signalling complex (Fig. 9A). This is a highly speculative suggestion. However, interactions have been shown to occur between proteins that contain similar sequences at those positions required for interaction. For example, this has been seen with the chemotaxis proteins CheA and CheW (Bilwes et al., 1999) and the flagellar proteins FliM and FliN (Mathews et al., 1998). Based on this analogy, CheC could associate with FliM or FliY and be released as a result of CCW rotation (CheY-P/FliM association; Bren et al., 1996) and subsequently interact with the receptor complex to facilitate adaptation (Fig. 9A). It is worth noting that CheC and CheY were assayed for a direct interaction by immunoprecipitation, but no interaction was observed (M. Rosario and G. Ordal, unpublished).

The extensive similarity between the CheC homologues and the flagellar motor proteins may reflect an unknown direct interaction between the flagellar switch and the receptor signalling complexes in many species. Indeed, the chemoreceptors in C. crescentus were shown to be localized to the cell pole that bears the single polar flagellum (Alley et al., 1992). As part of the mechanism for adaptation, feedback from the motor directly to the signalling complex could be achieved by direct protein interactions. This analogy can be extended to a common ancestor in which receptors were in immediate proximity to flagellar motors, either peritrichous or polar. The evolution of a soluble form of the interacting domain of a motor protein necessary for adaptation (e.g. CheC) would eliminate spatial constraints and thus allow for communication between the motor and a signalling complex not in relative proximity to the flagellar motor (Fig. 9A). In this way, a chemoreceptor array would be possible and would allow for integration of disparate signals over a wide dynamic range to achieve a co-ordinated response with all flagella.

Communication between the flagellar motor and upstream components has precedents. Results from our

© 2001 Blackwell Science Ltd, Molecular Microbiology, 42, 573–585

Fig. 9. Communication between the flagellar motor and upstream components.

A. Model for adaptation in *B. subtilis* by CheC and CheY-P feedback. After stimulation by asparagine via McpB, enhanced phosphorylation of CheA leads to elevated levels of CheY-P. In *B. subtilis*, CheY-P interacts with switch components to cause CCW rotation of the flagella or smooth swimming. As part of the adaptation process, both CheC and CheY-P feed back onto the CheA–receptor complex to lower the CheA autokinase activity to prestimulus levels. CheY-P could interact with either FliM or FliY or competitively inhibit CheC/switch binding, thereby dissociating CheC from the switch complex.

B. Model indicating viscosity-induced expression of the lateral flagella (*laf*) genes in *V. parahaemolyticus* (based on McCarter and Silverman, 1990). The single polar flagellum in *V. parahaemolyticus* acts as a mechanosensor for increased viscosity and initiates *laf* expression by an unknown mechanism.

laboratory have indicated that CheY-P feedback onto asparagine-bound McpB occurs to facilitate adaptation in *B. subtilis* (Fig. 9A; Kirby et al., 1999). Additionally, *Vibrio parahaemolyticus* swarmer cell differentiation occurs when the rotational rate of the single polar flagellum is slowed as a result of viscosity (Fig. 9B; McCarter and Silverman, 1990). The conclusion from that work is that the rotational information from the switch is ultimately responsible for regulating transcription of the lateral flagellar (*laf*) genes. In the same fashion, lateral flagella in *Azospirillum lipoferum* appear to regulate their own expression (Alexandre et al., 1999). A similar phenomenon may also be occurring in *E. coli*, which has recently been shown to undergo swarmer cell differentiation on solid surfaces (Burkart et al., 1998). That transition requires the chemotaxis machinery, although the system does not need to be functionally chemotactic. Thus, communication from the switch to other cellular components has been observed in several distantly related species.

Our speculation is that the state of rotation of the flagellar motor in *B. subtilis* might be sensed by a putative CheC–FliM or CheC–FliY interaction as well as by the total level of CheY-P, such that CheC and CheY-P could feed back onto the signalling complex to affect adaptation (Fig. 9A). However, a more simple explanation may be that CheC is the primary factor that feeds back onto the receptor complex in response to CheY-P interaction with the switch. Because CheC is known to affect methylation of the MCPs (Rosario et al., 1995), this putative

mechanism could account for the previous observation that cheY mutants were defective in methylation, which led to the hypothesis that CheY-P feeds back onto the signalling complex (Kirby et al., 1999). It is worth noting, however, that a *cheRBCDV* mutant has been shown partially to adapt to asparagine stimulation (E. Karatan, M. Saulmon and G. W. Ordal, unpublished) and, in that mutant, CheY-P feedback seems to be the only possibility left to account for adaptation. It seems, therefore, that during *B. subtilis* chemotaxis, multiple mechanisms contribute to bringing about adaptation. Integration of information from multiple aspects of the signalling complex as well as the behavioural status of the motor would enable the cells to adapt better to all types of stimuli. Because *B. subtilis* possesses one homologue of each of the known Che proteins (CheA, CheB, CheC, CheD, CheR, CheV, CheW and CheY) except CheZ, which is only found in *E. coli* and other closely related γ-proteobacteria, the underlying mechanism of chemotaxis in *B. subtilis* is likely to provide the basis for future studies in many other organisms.

### Experimental procedures

#### Bacterial strains and growth

All bacterial strains and plasmids used in this study are listed in Table 2. All plasmids were propagated in *E. coli* strain TG-1 (Amersham). Luria–Bertani (LB) medium is 1% tryptone, 0.5% yeast extract and 1% NaCl. Tryptone broth (Tbr) is 1% tryptone and 0.5% NaCl. TBAB is tryptose blood agar base

© 2001 Blackwell Science Ltd, Molecular Microbiology, **42**, 573–585

Table 2. *B. subtilis* strains and plasmids used in this study.

| Strain or plasmid | Relevant genotype$^a$ or description | Reference |
|-------------------|--------------------------------------|-----------|
| pGBT9             | Plasmid encoding GAL4 DNA-binding domain | Bartel et al. (1993) |
| pGAD424           | Plasmid encoding GAL4 activation domain | Bartel et al. (1993) |
| pLG109            | pGAD424::cheA                       | This work |
| pMR136            | pGAD424::cheD                       | Rosario and Ordal (1996) |
| pMR137            | pGBT9::cheC                         | Rosario and Ordal (1996) |
| pSF2              | pGBT9::mcpB cterm                   | This work |
| pSF3              | pGAD424::mcpB cterm                 | This work |
| pUNK200           | 2.8 kb Smal–Xbal fragment from pUNK101 subcloned into Smal–XbaI of pHL007, Amp<sup>R</sup>, Kan<sup>R</sup> containing mcpB | Kirby et al. (1999) |
| OI1085            | che<sup>+</sup>, ilvC1 leu-1 trpF7 hisH2 metC1 | Ullah and Ordal (1981) |
| OI2934            | cheD1::cat                          | Rosario et al. (1995) |
| OI3055            | mcpA1::cat                          | Hanlon and Ordal (1994) |
| OI3056            | mcpB1::cat                          | Hanlon and Ordal (1994) |
| OI3135            | ΔcheC1                              | Rosario et al. (1995) |
| OI3280            | mcpC4::erm                          | Müller et al. (1997) |
| OI3281            | Δ(mcpA mcpB tlpA tlpB)101::cat mcpC4::erm ΔtlpC | Müller et al. (1997) |
| OI3305            | ΔcheC1 cheD1::cat                   | Kirby et al. (1997) |

a. All mutants in this study are derived from the parent strain OI1085.

(Difco). Chemotaxis buffer (CB) is 0.1 mM EDTA, 50 μM CaCl<sub>2</sub>, 0.05% glycerol, 5 mM sodium lactate, 0.3 mM ammonium sulphate and 20 mM potassium phosphate, pH 7.0 (Ordal and Goldman, 1975). Minimal medium is 50 mM potassium phosphate, 1 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 1.2 mM MgCl<sub>2</sub>, 140 μM CaCl<sub>2</sub>, 10 μM MnCl<sub>2</sub>, 50 μg ml<sup>−1</sup> required amino acids and 20 mM sorbitol, pH 7.0 (Ordal and Goldman, 1975). Protoplast buffer (PB) is 20% sucrose, 25 mM potassium phosphate, 10 mM MgCl<sub>2</sub>, 30 mM sodium lactate and 1.0 mM EDTA, pH 7.0 (Ullah and Ordal, 1981).

Chemicals

All chemicals were of reagent grade.

Assay for behaviour of tethered cells

The method of Kirby et al. (1999) was used. In this method, bacteria are sheared in a Warring blender to remove most flagella. The cells are then tethered to a glass coverslip using anti-flagellin antibody. The coverslip is inverted to form the ceiling of a laminar flow chamber. The effect of flowing attractant (or its removal) on the rotating cells is followed using a microscope, recorded on a VCR and analysed using the HOBSON tracking programs.

Capillary assay for chemotaxis

The capillary assay for chemotaxis in *B. subtilis* has been described previously (Ullah and Ordal, 1981). The strain to be analysed was grown overnight at 37°C on TBAB. Cells were then resuspended in 250 μl of Tbr, diluted 1:100 into minimal media and grown at 37°C with vigorous shaking (250 r.p.m.) until A<sub>600</sub> = 32.5 Klett units (red filter). The culture was then supplemented to a final concentration of 0.05% glycerol (v/v) and 5 mM sodium lactate and incubated for 15 min before harvesting. The culture was then pelleted and resuspended to A<sub>600</sub> = 0.001 and assayed for a chemotactic response to chemotaxis buffer (CB) or varying concentrations of asparagine, proline or azetidine-2-carboxylate diluted in CB. The contents of the capillary tubes were plated on Tbr, and the resulting colonies were counted. The number of colonies from the tube containing CB was used as a blank for background subtraction.

Swarm plate assay for asparagine and proline taxis

Strains were grown overnight on tryptose blood agar base plates. A single colony was transferred to a minimal asparagine (0.3 mM) or proline (0.2 mM) swarm plates [0.1× Spizizen’s salts (Spizizen, 1958), 0.7 mM sorbitol, 5 μg ml<sup>−1</sup> required amino acids and 0.25% agar] and incubated at 30°C for 12–16 h.

Immunoblot analysis

Samples were electrophoresed at 15 V cm<sup>−1</sup> on 10% SDS–polyacrylamide gels, pH 8.8. The proteins were then transferred to polyvinylidene difluoride (PVDF) using a Bio-Rad semi-dry Trans-blot apparatus at 25 V for 1 h. Before transfer, the gel and PVDF were equilibrated in Bjerrum and Schafer-Nielson buffer (48 mM Tris, 39 mM glycine, 0.0375% SDS) with 20% methanol. After transfer was complete, the PVDF was blocked with 10% powdered milk Tris-HCl pH 7.5 (PM) for 3 h, incubated with anti-McpB or anti-McpC antibody (diluted 1:100 000 or 1:50 000 respectively) in Tris-PM overnight, washed with 0.5% Tween 20 in Tris-PM, incubated with a 1:8000 dilution of secondary antibody in Tris-PM for 2 h and washed again with Tween–Tris-PM. The PVDF membrane was then incubated with SuperSignal (Pierce) for 10 min. Visualization of the antigen–antibody complex required only 5 s to 1 min exposures with Kodak X-Omat AR film.

Yeast two-hybrid analysis

The C-terminal fragment of *mcpB* encoding the entire

cytoplasmic signalling domain was amplified by polymerase chain reaction (PCR) from pAIN700 using cloned Pfu polymerase (Stratagene). pAIN700 is a derivative of pUNK101 (Kirby *et al.*, 1999), in which the sole EcoRI site internal to *mcpB* has been removed by a silent mutation. PCR amplification was performed using primers that incorporated a BamHI site at the 5′ end of the fragment and a SalI site at the 3′ end of the fragment. The primers were designed such that cloning of the BamHI–SalI-digested PCR fragment into the BamHI–SalI sites of pGBT9 and pGAD424 generated an in frame fusion between the GAL4 DNA-binding domain or activation domain and the carboxy-terminus of McpB respectively. The *mcpB* C-terminus codes for a 359-amino-acid fragment that begins with residue R304 of McpB.

**Bioinformatics**

Iterative searches of the non-redundant database were performed using the PSI BLAST program (Altschul *et al.*, 1997). CheC and CheD of *B. subtilis* were used as queries in initial searches. Searches of the unfinished microbial genomes database at NCBI (http://www.ncbi.nlm.nih.gov/BLAST/unfinishedgenome.html) and TIGR (http://www.tigr.org) were performed using the gapped BLAST program (Altschul *et al.*, 1997). Multiple alignments were constructed using the CLUSTAL X program (Thompson *et al.*, 1997). Consensus for multiple alignments was calculated using http://www.bork.embl-heidelberg.de/Alignment/consensus.html (N. Brown and J. Lai, unpublished results). Secondary structure prediction was performed using the JNET consensus program of the JPRED server (Cuff and Barton, 1999). Phylogenetic trees were built from CLUSTAL W multiple alignments by the neighbour-joining method (Saitou and Nei, 1987). The bootstrapping procedure (Felsenstein, 1985) was applied for the PHYLIP format tree output. Trees were drawn with the TREEVIEW 1.5 program (Page, 1996).

**Acknowledgements**

We thank Shelby Feinberg for construction of plasmids used in the yeast two-hybrid assay, and Stacey Schiel for generating antibodies against McpC. Preliminary sequence data were obtained from The Institute for Genomic Research, the Sanger Centre, University of Oklahoma, PathoGenesis and Genome Therapeutics via the TIGR website at http://www.tigr.org. Support for this research was provided by Public Health Service grants AI20336 and GM54365 to G.W.O. from the National Institutes of Health. I.B.Z. was supported by start-up funds from the Georgia Institute of Technology. J.R.K. was supported by Postdoctoral Fellowship GM19676 from the National Institutes of Health.

**References**

Alexandre, G., Rohr, R., and Bally, R. (1999) A phase variant of *Azospirillum lipoferum* lacks a polar flagellum and constitutively expresses mechanosensing lateral flagella. Appl Environ Microbiol **65**: 4701–4704.

Alley, M.R.K., Maddock, J.R., and Shapiro, L. (1992) Polar

localization of a bacterial chemoreceptor. Gen Dev **6**: 825–836.

Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., and Lipman, D.J. (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res **25**: 3389–3402.

Armitage, J.P. (1999) Bacterial tactic responses. Adv Microb Physiol **41**: 229–289.

BarTEL, P.L., Chien, C.T., Sternglaz, R., and Fields, S. (1993) Using the two-hybrid system to detect protein–protein interactions. In Cellular Interactions in Development. A Practical Approach. Hartely, D.A. (ed.). Oxford: Oxford University Press, pp. 153–179.

Bilwes, A.M., Alex, L.A., Crane, B.R., and Simon, M.I. (1999) Structure of CheA, a signal-transducing histidine kinase. Cell **96**: 131–141.

Blat, Y., and Eisenbach, M. (1996) Mutants with defective phosphatase activity show no phosphorylation-dependent oligomerization of CheZ. The phosphatase of bacterial chemotaxis. J Biol Chem **271**: 1232–1236.

Borkovich, K.A., and Simon, M.I. (1990) The dynamics of protein phosphorylation in bacterial chemotaxis. Cell **63**: 1339–1348.

Borkovich, K.A., Alex, L.A., and Simon, M.I. (1992) Attenuation of sensory receptor signaling by covalent modification. Proc Natl Acad Sci USA **89**: 6756–6760.

Bray, D., Levin, M.D., and Morton-Firth, C.J. (1998) Receptor clustering as a cellular mechanism to control sensitivity. Nature **393**: 85–88.

Bren, A., and Eisenbach, M. (1998) The N terminus of the flagellar switch protein, FliM, is the binding domain for the chemotactic response regulator, CheY. J Mol Biol **278**: 507–514.

Bren, A., Welch, M., Blat, Y., and Eisenbach, M. (1996) Signal termination in bacterial chemotaxis: CheZ mediates dephosphorylation of free rather than switch-bound CheY. Proc Natl Acad Sci USA **93**: 10090–10093.

Burkart, M., Toguchi, A., and Harshey, R.M. (1998) The chemotaxis system, but not chemotaxis, is essential for swarming motility in *Escherichia coli*. Proc Natl Acad Sci USA **95**: 2568–2573.

Cuff, J.A., and Barton, G.J. (1999) Evaluation and improvement of multiple sequence methods for protein secondary structure prediction. Proteins **34**: 508–519.

Eisenbach, M. (1991) Signal transduction in bacterial chemotaxis. In Modern Cell Biology. Spudich, J.L., and Satir, B.H. (eds). New York: Wiley-Liss, pp. 137–208.

Felsenstein, J. (1985) Confidence limits on phylogenies: an approach using the bootstrap. Evolution **39**: 783–791.

Goldman, D.J., Nettleton, D.O., and Ordal, G.W. (1984) Purification and characterization of chemotactic methylesterase from *Bacillus subtilis*. Biochemistry **23**: 675–680.

Goy, M.F., Springer, M.S., and Adler, J. (1977) Sensory transduction in *Escherichia coli*: role of a protein methylation reaction in sensory adaptation. Proc Natl Acad Sci USA **74**: 4964–4968.

Goy, M.F., Springer, M.S., and Adler, J. (1978) Failure of sensory adaptation in bacterial mutants that are defective in a protein methylation reaction. Cell **15**: 1231–1240.

Hanlon, D.W., and Ordal, G.W. (1994) Cloning and

© 2001 Blackwell Science Ltd, Molecular Microbiology, **42**, 573–585

characterization of genes encoding methyl-accepting chemotaxis proteins in *Bacillus subtilis*. J Biol Chem 269: 14038–14046.

Hess, J.F., Oosawa, K., Kaplan, N., and Simon, M.I. (1988) Phosphorylation of three proteins in the signaling pathway of bacterial chemotaxis. Cell 53: 79–87.

Hoch, J.A. (2000) Two-component and phosphorelay signal transduction. Curr Opin Microbiol 3: 165–170.

Kirby, J.R., Kristich, C.J., Feinberg, S.L., and Ordal, G.W. (1997) Methanol production during chemotaxis to amino acids in *Bacillus subtilis*. Mol Microbiol 24: 869–878.

Kirby, J.R., Saulmon, M.M., Kristich, C.J., and Ordal, G.W. (1999) CheY-dependent methylation of the asparagine receptor, McpB, during chemotaxis in *Bacillus subtilis*. J Biol Chem 274: 11092–11100.

Kirby, J.R., Niewold, T.B., Maloy, S., and Ordal, G.W. (2000) CheB is required for behavioural responses to negative stimuli during chemotaxis in *Bacillus subtilis*. Mol Microbiol 35: 44–57.

Kirsch, M.L., Peters, P.D., Hanlon, D.W., Kirby, J.R., and Ordal, G.W. (1993) Chemotactic methylesterase promotes adaptation to high concentrations of attractant in *Bacillus subtilis*. J Biol Chem 268: 18610–18616.

Levit, M.N., Liu, Y., and Stock, J.B. (1998) Stimulus response coupling in bacterial chemotaxis: receptor dimers in signalling arrays. Mol Microbiol 30: 459–466.

Liu, J.D., and Parkinson, J.S. (1989) Role of CheW protein in coupling membrane receptors to the intracellular signaling system of bacterial chemotaxis. Proc Natl Acad Sci USA 86: 8703–8707.

McCarter, L., and Silverman, M. (1990) Surface-induced swarmer cell differentiation of *Vibrio parahaemolyticus*. Mol Microbiol 4: 1057–1062.

Mathews, M.A., Tang, H.L., and Blair, D.F. (1998) Domain analysis of the FliM protein of *Escherichia coli*. J Bacteriol 180: 5580–5590.

Müller, J., Schiel, S., Ordal, G.W., and Saxild, H.H. (1997) Functional and genetic characterization of *mcpC*, which encodes a third methyl-accepting chemotaxis protein in *Bacillus subtilis*. Microbiology 143: 3231–3240.

Ordal, G.W., and Goldman, D.J. (1975) Chemotaxis away

from uncouplers of oxidative phosphorylation in *Bacillus subtilis*. Science 189: 802–805.

Page, R.D.M. (1996) TREEVIEW: an application to display phylogenetic trees on personal computers. Comput Appl Biosci 12: 357–358.

Parkinson, J.S. (1993) Signal transduction schemes of bacteria. Cell 73: 857–871.

Rosario, M.M.L., and Ordal, G.W. (1996) CheC and CheD interact to regulate methylation of *Bacillus subtilis* methyl-accepting chemotaxis proteins. Mol Microbiol 21: 511–518.

Rosario, M.M.L., Kirby, J.R., Bochar, D.A., and Ordal, G.W. (1995) Chemotactic methylation and behavior in *Bacillus subtilis*: role of two unique proteins, CheC and CheD. Biochemistry 34: 3823–3831.

Saitou, N., and Nei, M. (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol 4: 406–425.

Spizizen, J. (1958) Transformation of biochemically deficient strains of *Bacillus subtilis* by deoxyribonucleate. Proc Natl Acad Sci USA 44: 1072–1078.

Stock, J.B., and Surette, M.G. (1996) Chemotaxis. In *Escherichia coli and Salmonella: Cellular and Molecular Biology*. Neidhardt, F.C. *et al.* (eds). Washington, DC: American Society for Microbiology Press, pp. 1103–1129.

Thompson, J.D., Gibson, T.J., Plewniak, F., Jeanmougin, F., and Higgins, D.G. (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res 25: 4876–4882.

Tso, W.W., and Adler, J. (1974) Negative chemotaxis in *Escherichia coli*. J Bacteriol 118: 560–576.

Ullah, A.H.J., and Ordal, G.W. (1981) In vivo and in vitro chemotactic methylation in *Bacillus subtilis*. J Bacteriol 145: 958–965.

Wang, H., and Matsumura, P. (1996) Characterization of the CheAS/CheZ complex: a specific interaction resulting in enhanced dephosphorylating activity on CheY-phosphate. Mol Microbiol 19: 695–703.

Zimmer, M.A., Tiu, J., Collins, M.A., and Ordal, G.W. (2000) Selective methylation changes on the *Bacillus subtilis* chemotaxis receptor McpB promote adaptation. J Biol Chem 275: 24264–24272.
