
JOURNAL OF BACTERIOLOGY, Jan. 2002, p. 302–306  
0021-9193/02/$04.00+0 DOI: 10.1128/JB.184.1.302–306.2002  
Copyright © 2002, American Society for Microbiology. All Rights Reserved.

NOTES

**Utilization of L-Ascorbate by Escherichia coli K-12: Assignments of Functions to Products of the yif-sga and yia-sgb Operons**

Wen Shan Yew and John A. Gerlt*

Departments of Biochemistry and Chemistry, University of Illinois, Urbana, Illinois 61801

Received 4 April 2001/Accepted 2 October 2001

Escherichia coli K-12 can ferment L-ascorbate. The operon encoding catabolic enzymes in the utilization of L-ascorbate (ula) has been identified; this operon of previously unknown function had been designated the yif-sga operon. Three enzymes in the pathway that produce D-xylulose 5-phosphate have been functionally characterized: 3-keto-L-gulonate 6-phosphate decarboxylase (UlaD), L-xylulose 5-phosphate 3-epimerase (UlaE), and L-ribulose 5-phosphate 4-epimerase (UlaF). Several products of the yia-sgb operon were also functionally characterized, although the substrate and physiological function of the operon remain unknown: 2,3-diketo-L-gulonate reductase (YiaK), 3-keto-L-gulonate kinase (LyxK), 3-keto-L-gulonate 6-phosphate decarboxylase (SgbH), and L-ribulose 5-phosphate 4-epimerase (SgbE).

Workers in this laboratory study divergent evolution of enzyme function, with focus on groups of homologous proteins that catalyze different chemical reactions (5). In defining the range of reactions catalyzed by homologous proteins, we have assigned functions to proteins annotated in genome sequencing projects as having unknown, uncertain, or alternate functions. For example, we identified a homologue of enoyl coenzyme A (CoA) hydratase (crotonase) encoded by an operon in the Escherichia coli genome as methylmalonyl CoA decarboxylase (6) and homologues of muconate lactonizing enzyme encoded by both the E. coli and Bacillus subtilis genomes as D-Ala-D/L-Glu epimerases (15). We now provide functional assignments of proteins encoded by the yif-sga and yia-sgb operons in the E. coli genome, one of which encodes enzymes for the utilization of L-ascorbate.

The literature on the utilization of L-ascorbate by bacteria, including E. coli, is limited. In 1939 (4) and again in 1942 (18), L-ascorbate was reported to be decomposed under anaerobic conditions by enteric bacteria, including E. coli. In 1962, a strain of Aerobacter aerogenes was reported to ferment L-ascorbate; 3-keto-L-gulonate was implicated as an intermediate in the dissimilation (16). In 1988, L-ascorbate was reported to stimulate the rate and extent of anaerobic but not aerobic growth of E. coli B (13). However, to the best of our knowledge, whether E. coli can catabolize L-ascorbate has not been clarified.

Herein we report that L-ascorbate can be fermented by E. coli K-12, identify the operon of previously unknown function that encodes the catabolic enzymes in utilization of L-ascorbate (the ula operon; previously designated the yif-sga operon), provide functional assignment of three enzymes in the pathway encoded by the ula operon, and provide functional characterization of four enzymes in the pathway encoded by the yia-sgb operon.

E. coli K-12 can ferment L-ascorbate. In our initial experiments, L-ascorbate failed to support growth of E. coli K-12 strains MG1655 (1) and BW25113 (2) either aerobically or anaerobically at a concentration of 100 mM in M9 minimal medium. However, given the early reports that L-ascorbate is unstable anaerobically in the presence of E. coli only at concentrations of <50 mM (4, 18), we tested lower concentrations. At concentrations of L-ascorbate of ≤20 mM, both MG1655 and BW25113 fermented L-ascorbate but could not use it for aerobic respiration; the growth curves for BW25113 are presented in Fig. 1. Concentrations of L-ascorbate of >20 mM also prevented fermentation of D-glucose, suggesting that the inability of E. coli to utilize L-ascorbate at high concentrations may be the result of alteration of the internal redox state of the cells. The same strains of E. coli anaerobically respire L-ascorbate in the presence of fumarate, trimethylamine N-oxide, or nitrate, with the cell yields exceeding those obtained in fermentation (data not shown). Irrespective of the identity of the electron acceptor, similar cell yields are obtained with L-ascorbate and D-glucose.

Possible pathway for utilization of L-ascorbate by E. coli. We sought to identify the operon that encodes the catabolic pathway for L-ascorbate. The catabolism of L-ascorbate can be hypothesized to involve loss of carbon-1 (via decarboxylation) and epimerization of carbons-4 and -5. The structure of L-ascorbate is compared with those of pentoses in a possible catabolic sequence in Fig. 2.

In the L-arabinose catabolic pathway, L-ribulose 5-phosphate is 4-epimerized to D-xylulose 5-phosphate, an intermediate in the pentose phosphate pathway; L-ribulose 5-phosphate 4-epimerase (AraD) catalyzes this reaction. L-Ribulose 5-phosphate also could be an intermediate in the catabolism of L-ascorbate: conversion of L-ascorbate to L-xylulose by hydrolysis

* Corresponding author. Mailing address: Departments of Biochemistry and Chemistry, University of Illinois, 600 South Mathews Avenue, Urbana, IL 61801. Phone: (217) 333-3945. Fax: (217) 265-0385. E-mail: j-gerlt@uiuc.edu.

min. The amplified genes were cloned into a modified pET15-b (Novagen) vector in which the N-terminal His tag contains 10 instead of the usual 6 His residues.

The proteins were expressed in *E. coli* strain BL21(DE3). For all proteins except YiaK, transformed cells were grown at 37°C in Luria-Bertani (LB) broth (supplemented with ampicillin at 100 μg/ml) to an optical density at 600 nm (*OD*~600~) of 2 and harvested by centrifugation. No IPTG (isopropyl-β-D-thiogalactopyranoside) was added to induce protein expression. The cells were resuspended in binding buffer (5 mM imidazole, 0.5 M NaCl, 20 mM Tris-HCl, pH 7.9, and 5 mM MgCl₂) and lysed by sonication. The lysates were cleared by centrifugation.

The His-tagged proteins were purified with columns of chelating Sepharose Fast Flow (Pharmacia Biotech) charged with Ni²⁺. Cell lysates were applied to the column in binding buffer, washed with 15% elute buffer (1 M imidazole, 0.5 M NaCl, 20 mM Tris-HCl, pH 7.9, and 5 mM MgCl₂)—85% wash buffer (60 mM imidazole, 0.5 M NaCl, 20 mM Tris-HCl, pH 7.9, and 5 mM MgCl₂), and eluted with 50% binding buffer–50% strip buffer (100 mM EDTA, 0.5 M NaCl, 20 mM Tris-HCl, pH 7.9, and 5 mM MgCl₂). The N-terminal His tags (except for YiaK and LyxK) were removed with thrombin (Pharmacia Biotech) according to the manufacturer’s instructions, and the proteins were purified to homogeneity on a ResourceQ column (Pharmacia Biotech) equilibrated with binding buffer (10 mM Tris-HCl, pH 7.9, 5 mM MgCl₂) and eluted with a linear gradient of 0 M to 0.5 M elution buffer (1 M NaCl, 10 mM Tris-HCl, pH 7.9, 5 mM MgCl₂). For YiaK, transformed cells were grown at 37°C in M9 minimal medium supplemented with 0.1 M glucose and ampicillin (100 μg/ml) to an *OD*~600~ of 0.6 and induced with 0.5 mM IPTG for 2 h; the protein was purified as described above.

SgaE and SgbE are L-ribulose 5-phosphate 4-epimerases. Both SgaE and SgbE catalyzed the epimerization of L-ribulose 5-phosphate to D-xylulose 5-phosphate, using the established coupled enzyme assay for AraD (transketolase, triose phosphate isomerase, and α-glycerophosphate dehydrogenase [9–11]). The kinetic constants for the SgaE- and SgbE-catalyzed reactions, presented in Table 2, are comparable to those reported for AraD (9–11) (values not shown), so we conclude that SgaE and SgbE are L-ribulose 5-phosphate 4-epimerases. SgbE was previously reported to catalyze the L-ribulose 5-phosphate 4-epimerase reaction, although that study was not performed with purified protein (8).

Operon responsible for utilization of L-ascorbate by *E. coli*. Does either SgaE or SgbE participate in the catabolism of L-ascorbate? The method described by Datsenko and Wanner (2) was used to generate two mutants of strain BW25113 in which the six genes in the operon encoding SgaE and the nine genes in the operon encoding SgbE were separately deleted by insertion of an antibiotic resistance gene. The operon encoding SgaE (Fig. 3A) was replaced with the kanamycin resistance gene (for neomycin phosphotransferase) from plasmid pKD13; the kanamycin resistance gene was inserted between codon 134 of *sgaT* (405 codons in the intact gene) and codon 137 of *sgaE* (229 codons in the intact gene). The operon encoding SgbE (Fig. 3B) was replaced with the chloramphenicol resistance gene (for chloramphenicol acetyltransferase) from plasmid pKD3; the chloramphenicol resistance gene was inserted between codon 101 of *yiaK* (333 codons in the intact gene) and

FIG. 1. Growth of BW25113 in M9 minimal medium with several concentrations of L-ascorbate in the absence of an exogenous electron acceptor. •, 5 mM; ■, 10 mM; ○, 20 mM; □, 50 mM.

of the lactone followed by decarboxylation would retain the configurations of carbons-4 and -5; 3-epimerization of L-xylulose 5-phosphate would produce L-ribulose 5-phosphate. Finally, L-ribulose 5-phosphate would be 4-epimerized to D-xylulose 5-phosphate, presumably by a homologue of AraD.

The *E. coli* genome (1) encodes two homologues of AraD, SgaE and SgbE, each located in the product of a multigene operon of unknown function. The *sga* and *sgb* mnemonics for the encoding operons were assigned by Reizer and coworkers in an analysis of operons proposed to be involved in sugar metabolism, albeit with unknown substrates and products (12). Based on the intergenic spacings, SgaE is encoded by an operon containing six genes (Fig. 3A). SgbE is known to be encoded by an operon containing nine genes (Fig. 3B) (7). SgaE and SgbE show 61% sequence identity with each other and 76 and 60% sequence identity, respectively, with AraD.

Expression and purification of enzymes. The genes encoding SgaE and SgbE, as well as those encoding SgaH, SgbH, SgaU, SgbU, YiaK, and LyxK, also discussed in this paper, were PCR amplified from genomic DNA isolated from *E. coli* strain MG1655 using Platinum Pfx DNA polymerase (Life Technologies). A typical PCR mixture (100 μl) contained 1 ng of genomic DNA, 10 μl of 10× Pfx amplification buffer (Life Technologies), 1 mM MgSO₄, 0.4 mM concentrations of each of the four deoxynucleotide triphosphates (dNTPs), 40 pmol of each primer (Table 1), and 5 U of Platinum Pfx DNA polymerase. The genes were amplified using a PTC-200 gradient thermal cycler (MJ Research), with the following parameters: 94°C for 2 min; followed by 40 cycles of 94°C for 1 min, a gradient annealing temperature range of 45 to 60°C for 1 min 15 s and 68°C for 2 min; and a final extension of 68°C for 10

L-Ascorbic Acid → L-Xylulose → L-Ribulose → D-Xylulose

FIG. 2. Possible pathway for utilization of L-ascorbate by *E. coli*.

A

ulaA  
sgaT  
yjfS  

ulaB  
sgaB  
yjfT  

ulaC  
ptxA  
yjfU  

ulaD  
sgaH  
yjfV  

ulaE  
sgaU  
yjfW  

ulaF  
sgaE  
yjfX  

B

yiaK  
yiaL  
yiaM  
yiaN  
yiaO  
yiaP  
lyxK  
yiaQ  
sgbH  
yiaR  
sgbU  
yiaS  
sgbE  

C

![Chemical Structure](https://i.imgur.com/1234567.png)

FIG. 3. Operons in the E. coli genome that encode SgaH (A) and SgbH (B); the first operon is located in locus AE000491, and the second is located in locus AE000435 of the E. coli genome. (C) Pathway for the utilization of L-ascorbate.

codon 179 of sgbE (232 codons in the intact gene). The positions of the deletions were confirmed by DNA sequence analysis of PCR-amplified regions of the mutant genomes using locus-specific primers flanking the intended deletion sites.

The mutant in which the operon encoding SgaE was deleted failed to ferment L-ascorbate; fermentation of L-ascorbate was unaffected by disruption of the operon encoding SgbE (data not shown). We assign utilization of L-ascorbate to the operon encoding SgaE.

Pathway for L-ascorbate utilization. We then sought to verify our predicted pathway for L-ascorbate utilization by assigning specific reactions (Fig. 3C) to other proteins encoded by the operon encoding SgaE (Fig. 3A). Hydrolysis of the L-ascorbate lactone and tautomerization would produce a 3-keto acid (e.g., 3-keto-L-gulonate). The operon encodes three components of a phosphotransferase system (PTS; SgaT, SgaB, and PtxA [12]), so the 3-keto acid derived from L-ascorbate could be imported as its 6-phosphate. Alternatively, L-ascorbate could be imported as its 6-phosphate which would be hydrolyzed and tautomerized intracellularly. Irrespective of these details, decarboxylation of the 3-keto acid 6-phosphate would yield L-xylulose 5-phosphate.

Conveniently, a useful 3-keto acid was obtained from 2,3-diketo-L-gulonate using YiaK (Fig. 3B) and NADH; the kinetic constants for the YiaK-catalyzed reaction are included in Table 2. Based on the homology of YiaK with other known NADH-dependent reductases, we assign 3-keto-L-gulonate as the product of the YiaK reaction (Fig. 4). A dehydrogenase catalyzing the formation of 3-keto-L-gulonate from 2,3-diketo-L-gulonate was isolated from a strain of Aerobacter aerogenes grown anaerobically with L-ascorbate as the carbon source

TABLE 1. PCR primers$^a$

| Gene          | Forward primer                                                                                   | Reverse primer                                                                                          |
|---------------|-----------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
| $sgaH$        | GAAGGAATTGGCATATGTGCATTACCGATG ($NdeI$)                                                        | GGACAACATATCGGATCCTTAGCCCC ($BamHI$)                                                                  |
| $sghH$        | GGAGCACACCATATGAGCCGACCAC ($NdeI$)                                                            | CATAAATCCCTAACGGATCCTTACGGCACGCG ($BamHI$)                                                           |
| $sgaU$        | GGCTAAGGACATATGATGTTGTCCAAAC ($NdeI$)                                                         | TTAGCTTTTGGATCCTGGCCGCCCTCCAC ($BamHI$)                                                              |
| $sgbU$        | GCTGGCGCATATGATGCCGCAAATCG ($NdeI$)                                                           | GCTTTCACTCGAGCTAAACATATAAATCC ($XhoI$)                                                               |
| $sgaE$        | GGAGGCCATATGATGCAAAAGCTAAAAC ($NdeI$)                                                         | GCAGCGGGATCCCTACTTCTGCC ($BamHI$)                                                                    |
| $sgbE$        | GGAGGCTGGCATATGATGTTAGAGCAAC ($NdeI$)                                                         | GGCGTGGATCCTTACTGCCCGTAATAGG ($BamHI$)                                                               |
| $yiaK$        | CAAGGAAGCCTCATATGAAAAGTGACATTTGAGC ($NdeI$)                                                   | TGACTTCTCGAGTCATAACGCCTGGATTTTGG ($XhoI$)                                                            |
| $lyxK$        | GAGGTGCAAGTCGACATGACGCAATAC ($SalI$)                                                          | GCAGAAGTGGGATCCTCATAATGTGTG ($XhoI$)                                                                 |
| $sga$ operon deletion | CGTCGCATTACCGGCATTCGCACAAATCATGTTGACC GGCCACGTGTA GGCTGGAGCTGCTTC (pKD13 site 1) | GCTTAACCCGCGCGTACACGGAATGTGCGCCAAGAAG TAGTCATTCCGGGGATCCGTCGACC (pKD13 site 4)                     |
| $sgb$ operon deletion | GCCGATCACGGTTATGGTCTGGTGGCACTACGTAAT GCCAACGTGTA GGCTGGAGCTGCTTC (pKD3 site 1) | CGGGCCGTGAGAATGCACCAGCACCGGGGGATTTGT GCCGGCATATGAATATCCTCCTTAG (pKD3 site 2)                      |

$^a$ Restriction sites and priming sites are shown in bold and indicated in parentheses.

L-ascorbate via a β-keto acid decarboxylation reaction catalyzed by SgaH and successive epimerization reactions catalyzed by SgaU and SgaE to yield D-xylulose 5-phosphate.

Based on our functional assignment of this operon, we propose that the operon be designated the utilization of L-ascorbate operon, $ula$. We propose that the genes encoding the components of the PTS system be renamed (Fig. 3A): $sgaT$

TABLE 2. Kinetic constants obtained from spectrophotometric coupled-enzyme assays

| Enzyme       | $k_{cat}$ ($s^{-1}$) | $k_{cat}/K_m$ ($M^{-1}s^{-1}$) |
|--------------|----------------------|----------------------------------|
| $SgaE^a$     | 25                   | $3.8 \times 10^4$               |
| $SgbE^a$     | 12                   | $1.7 \times 10^4$               |
| $SgaH^b$     | 51                   | $7.7 \times 10^4$               |
| $SgaU^c$     | 2.9                  | $5.5 \times 10^3$               |
| $YiaK^d$     | 110                  | $1.4 \times 10^5$               |
| $LyxK^e$ (3-keto-L-gulonate) | 48                   | $8.9 \times 10^4$               |
| $LyxK^e$ (L-xylulose) | 120                  | $1.1 \times 10^5$               |
| $SgbH^b$     | 64                   | $1.0 \times 10^5$               |

$^a$ The assay mixtures (1 ml at $25^\circ C$) contained L-ribulose 5-phosphate (0.1 to 10 mM), 50 mM glycoaldehyde, 0.1 mM thiamine pyrophosphate, 50 mM K$^+$HEPES (pH 7.5), 10 mM MgCl$_2$, 10 U of α-glycerophosphate dehydrogenase, 1 U of transketolase, 45 U of triose phosphate isomerase per ml, and 0.16 mM NADH. The change in absorbance at 340 nm was quantitated.

$^b$ The assay mixtures (1 ml at $25^\circ C$) contained β-keto-L-gulonate 6-phosphate (0.2 to 2.9 mM), 50 mM K$^+$HEPES (pH 7.5), and 10 mM MgCl$_2$. The enzyme was removed by centrifugation through a Biomax-10 (10,000 Da) Millipore centrifugal filter device. After incubation of an aliquot (800 μl) with 30 U of calf intestine alkaline phosphatase at $37^\circ C$ for 5 min, 300 μl of a 5% solution of ZnSO$_4 \cdot 7H_2O$ and 300 μl of 0.15 M Ba(OH)$_2$ were added; the precipitate was removed by centrifugation. An aliquot (800 μl) of the supernatant was assayed for L-xylulose by reaction with L-cysteine and carbazole (17). The absorbance at 540 nm was measured; the molar extinction coefficient for the chromophore was determined to be $1,367 M^{-1} cm^{-1}$. A stock solution of β-keto-L-gulonate 6-phosphate was generated by incubating 3.25 mM L-diketogulonate, 10 mM MgCl$_2$, 6 mM ATP, 60 U of YiaK per ml, 50 U of LyxK per ml, and a molar equivalent of NADH for 20 min at $25^\circ C$.

$^c$ The assays (1 ml at $25^\circ C$) contained L-xylulose (0.2 to 16.7 mM), 1.5 mM ATP, 15 mM acetyl phosphate, 50 mM glycoaldehyde, 0.1 mM thiamine pyrophosphate, 50 mM K$^+$HEPES (pH 7.5), 10 mM MgCl$_2$, 22 U of acetate kinase, 110 U of LyxK, 10 U of α-glycerophosphate dehydrogenase, 1 U of transketolase, 45 U of a triose phosphate isomerase, 5 U of SgaE per ml, and 0.16 mM NADH. The change in absorbance at 340 nm was quantitated. The assays were incubated for 30 min at $25^\circ C$ to generate L-xylulose 5-phosphate prior to addition of SgaU.

$^d$ The assay mixtures (1 ml at $25^\circ C$) contained L-diketogulonate (0.050 to 3.4 mM), 50 mM Tris-HCl (pH 7.5), and 0.16 mM NADH. The change in absorbance at 340 nm was quantitated.

$^e$ The assay mixtures (1 ml at $25^\circ C$) contained 3-keto-L-gulonate (0.08 to 9.2 mM; prepared in situ with YiaK) or L-xylulose (0.066 to 10.7 mM), 1.5 mM phosphoenolpyruvate, 1.5 mM ATP, 50 mM K$^+$HEPES (pH 7.5), 10 mM MgCl$_2$, 9 U of pyruvate kinase, 9 U of lactate dehydrogenase per ml, and 0.16 mM NADH. The change in absorbance at 340 nm was quantitated.

lases. As we will discuss in detail elsewhere, 3-keto-L-gulonate 6-phosphate decarboxylase and orotidine 5′-phosphate decarboxylase utilize conserved active-site functional groups to catalyze mechanistically distinct decarboxylation reactions in different metabolic pathways.

This work was supported by grants GM-40570 and GM-52594 from the National Institutes of Health.

### REFERENCES

1. Blattner, F. R., G. Plunkett, C. A. Bloch, N. T. Perna, V. Burland, M. Riley, J. Collado-Vides, J. D. Glasner, C. K. Rode, G. F. Mayhew, J. Gregor, N. W. Davis, H. A. Kirkpatrick, M. A. Goeden, D. J. Rose, B. Mau, and Y. Shao. 1997. The complete genome sequence of *Escherichia coli* K-12. Science 277:1453–1474.
2. Datsenko, K. A., and B. L. Wanner. 2000. One-step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. Proc. Natl. Acad. Sci. USA 97:6640–6645.
3. Dworsky, P., and O. Hoffmann-Stenhof. 1967. 3-Keto-L-gulonate Decarboxylase aus *Schwanniomyces occidentalis*. Monatsh. Chem. 98:993–996.
4. Esselen, W. B., and J. E. Fuller. 1939. The oxidation of ascorbic acid as influenced by intestinal bacteria. J. Bacteriol. 37:501–521.
5. Gerlt, J. A., and P. C. Babbitt. 2001. Divergent evolution of enzymatic function: mechanistically diverse superfamilies and functionally distinct superfamilies. Annu. Rev. Biochem. 70:209–246.
6. Haller, T., T. Buckel, J. Retey, and J. A. Gerlt. 2000. Discovering new enzymes and metabolic pathways: conversion of succinate to propionate by *Escherichia coli*. Biochemistry 39:4622–4629.
7. Ibanez, E., E. Campos, L. Baldoma, J. Aguilar, and J. Badia. 2000. Regulation of expression of the *yiaKLMNOPQRS* operon for carbohydrate utilization in *Escherichia coli*: involvement of the main transcriptional factors. J. Bacteriol. 182:4617–4624.
8. Ibanez, E., R. Gimenez, T. Pedraza, L. Baldoma, J. Aguilar, and J. Badia. 2000. Role of the *yiaR* and *yiaS* genes of *Escherichia coli* in metabolism of endogenously formed L-xylulose. J. Bacteriol. 182:4625–4627.
9. Johnson, A. E., and M. E. Tanner. 1998. Epimerization via carbon-carbon bond cleavage. L-Ribulose-5-phosphate 4-epimerase as a masked class II aldolase. Biochemistry 37:5746–5754.
10. Lee, L. V., R. R. Poyner, M. V. Vu, and W. W. Cleland. 2000. Role of metal ions in the reaction catalyzed by L-ribulose-5-phosphate 4-epimerase. Biochemistry 39:4821–4830.
11. Lee, L. V., M. V. Vu, and W. W. Cleland. 2000. ¹³C and deuterium isotope effects suggest an aldol cleavage mechanism for L-ribulose-5-phosphate 4-epimerase. Biochemistry 39:4808–4820.
12. Reizer, R., A. Charbit, A. Reizer, and M. H. Saier. 1996. Novel phosphotransferase system genes revealed by bacterial genome analysis: operons encoding homologues of sugar-specific permease domains of the phosphotransferase system and pentose catabolic enzymes. Genome Sci. Technol. 1:53–75.
13. Richter, H. E., J. Switala, and P. C. Loewen. 1988. Effect of ascorbate on oxygen uptake and growth of *Escherichia coli* B. Can. J. Microbiol. 34:822–825.
14. Sanchez, J. C., R. Gimenez, A. Schneider, W. D. Fessner, L. Baldoma, J. Aguilar, and J. Badia. 1994. Activation of a cryptic gene encoding a kinase for L-xylulose opens a new pathway for the utilization of L-xylose by *Escherichia coli*. J. Biol. Chem. 269:29665–29669.
15. Schmidt, D. Z., B. K. Hubbard, and J. A. Gerlt. Evolution of enzymatic activities in the enolase superfamily: functional assignment of unknown proteins in *Bacillus subtilis* and *Escherichia coli* as L-Ala-D/L-Glu epimerases. Biochemistry, in press.
16. Volk, W. A., and J. L. Larsen. 1962. β-Keto-L-gulonic acid as an intermediate in the bacterial metabolism of ascorbic acid. J. Biol. Chem. 237:2454–2457.
17. Winkelman, J., and G. Ashwell. 1961. Enzymic formation of L-xylulose from β-keto-L-gulonic acid. Biochim. Biophys. Acta 52:170–175.
18. Young, R. M., and L. H. James. 1942. Action of intestinal microorganisms on ascorbic acid. J. Bacteriol. 44:75–84.
