
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2002 by The American Society for Biochemistry and Molecular Biology, Inc.
Vol. 277, No. 16, Issue of April 19, pp. 13421–13429, 2002
Printed in U.S.A.

# Identification of Coenzyme M Biosynthetic Phosphosulfolactate Synthase

## A NEW FAMILY OF SULFONATE-BIOSYNTHESIZING ENZYMES*

Received for publication, January 30, 2002  
Published, JBC Papers in Press, February 5, 2002, DOI 10.1074/jbc.M201011200

### David E. Graham, Huimin Xu, and Robert H. White‡

*From the Department of Biochemistry, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061-0308*

The hyperthermophilic euryarchaeon *Methanococcus jannaschii* uses coenzyme M (2-mercaptoethanesulfonic acid) as the terminal methyl carrier in methanogenesis. We describe an enzyme from that organism, (*2R*)-phospho-3-sulfolactate synthase (ComA), that catalyzes the first step in coenzyme M biosynthesis. ComA catalyzed the stereospecific Michael addition of sulfite to phosphoenolpyruvate over a broad range of temperature and pH conditions. Substrate and product analogs moderately inhibited activity. This enzyme has no significant sequence similarity to previously characterized enzymes; however, its Mg²⁺-dependent enzyme reaction mechanism may be analogous to one proposed for enolase. A diverse group of microbes and plants have homologs of ComA that could have been recruited for sulfolactate or sulfolipid biosyntheses.

Sulfonate-containing natural products are believed to be biosynthesized either by oxidizing thiol groups or by adding nucleophilic sulfite at unsaturated carbon bonds (7). Taurine and cysteate are produced from cysteine by the oxidative mechanism (8). The nonenzymatic sulfonation of α,β-unsaturated carbonyl compounds is also well known (9, 10). Yet, only recently have the biosyntheses of sulfoquinovose, sulfolactate, and CoM been demonstrated to occur by the sulfite addition mechanism. Plants and photosynthetic bacteria produce UDP-sulfoquinovose, a sulfolipid head group precursor, using SQD1/SqdB proteins to add sulfite to UDP-glucose (11, 12). This reaction may proceed via a UDP-4-ketoglucose-5-ene intermediate (13). *Bacillus subtilis* produces sulfolactate during sporulation (14), apparently using homologs of CoM biosynthesis proteins (15).

The proposed pathway for the biosynthesis of coenzyme M from phosphoenolpyruvate (P-enolpyruvate) was deduced by analysis of labeling patterns of CoM purified from methanogens grown on stable isotope-labeled acetates (Fig. 1) (15, 16). Incubation of P-enolpyruvate and bisulfite with cell extracts of *Methanobacterium formicum* produced sulfolactate, sulfopyruvate, and sulfoacetalddehyde, establishing the role of these sulfonated intermediates in the pathway (17). Recently, three of the enzymes involved in this pathway (ComB, ComC, and ComDE) have been identified and characterized in a hyperthermophilic marine methanogen, *Methanococcus jannaschii* (18–20). In this work, we describe the first dedicated enzyme required for CoM biosynthesis in *M. jannaschii*, (*2R*)-phospho-3-sulfolactate synthase (ComA). This novel enzyme catalyzes the stereospecific Michael addition of sulfite to P-enolpyruvate, forming L-2-phospho-3-sulfolactate (PSL). Analyses of metal cofactor requirements, substrate specificity, the proton exchange reaction, a site-directed mutant, and reaction inhibitors suggest that the ComA-catalyzed reaction is analogous to those reactions catalyzed by β-elimination enzymes that proceed through an enolate intermediate (21).

---

Each year, archaeal methanogens produce >400 million tons of methane by anaerobically reducing acetate or single carbon compounds (1). Regardless of which carbon substrate these cells use for methanogenesis, all methyl intermediates are transiently bound to the terminal methyl carrier coenzyme M (CoM¹; 2-mercaptoethanesulfonic acid) (2). Methyl-CoM reductase releases methane from the methylated coenzyme, forming a CoM–S–S–coenzyme B heterodisulfide, which is reduced in a key energy-yielding step that makes methanogenesis a viable physiology (3). Alkene-oxidizing bacteria also use CoM as a cofactor (4).

CoM is unique among coenzymes in having a strong anionic sulfonate group. This sulfonate moiety solubilizes CoM, makes it membrane-impermeable, and provides a handle for enzymes such as methyl-CoM reductase to bind (5). Like glutathione, CoM is a soluble scavenger of reactive chemicals, and its simple chemical synthesis has fostered its therapeutic use in humans as a mucolytic and uroprotective agent during chemotherapy (6).

---

* This work was supported by National Science Foundation Grant MCB 9985712 (to R. H. W.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
‡ To whom correspondence should be addressed: Dept. of Biochemistry (0308), Engel Hall, Virginia Polytechnic Inst. and State University, Blacksburg, VA 24061-0308. Tel.: 540-231-6605; Fax: 540-231-9070; E-mail: rhwhite@vt.edu.
¹ The abbreviations used are: CoM, coenzyme M (2-mercaptoethanesulfonic acid); P-enolpyruvate, phosphoenolpyruvate; PSL, (*2R*)-phospho-3-sulfolactate (L-phosphosulfolactate); GC-MS, gas chromatographymass spectrometry; TSP, sodium [2,2,3,3-²H₄]trimethylsilylpropionate; bis-Tris, 2-[bis(2-hydroxyethyl)amino]-2-(hydroxymethyl)propane-1,3-diol; Mes, 4-morpholineethanesulfonic acid; Aces, 2-[(2-amino-2-oxoethyl)amino]ethanesulfonic acid; Caps, 3-(cyclohexylamino)propanesulfonic acid; Hepps, 4-(2-hydroxyethyl)-1-piperazinepro-panesulfonic acid.

---

### EXPERIMENTAL PROCEDURES

#### Materials—All reagents and synthetic precursors were purchased from Sigma unless otherwise specified. (RS)-2-Phosphonoxy-3-sulfopropionic acid (racemic phosphosulfolactate), both enantiomers of 2-phosphonoxypropionic acid (phospholactate), and both enantiomers of 2-phosphonoxybutanedioic acid (phosphomalate) were synthesized as described previously (18). Hydroxycarbamoylmethylphosphonic acid (phosphonoaceto hydroxyamic acid) (22) and 2-phosphonomethylacrylic acid (23) were also prepared as described previously. D-Cysteic acid was prepared by the oxidation of D-cystine with bromine (24).

#### Stereospecific Syntheses of Sulfolactic Acids—The scheme used to synthesize sulfolic acid enantiomers was modified from that described previously (25), which in our hands was found to form 2-chloro-3-sulfopropionic acid. To a solution of D- or L-cysteic acid (5 mmol, 0.93 g) dissolved in 6.25 ml of 3.3 M trifluoroacetic acid, isoamyl nitrite (15.6 mmol, 2.1 ml) was added dropwise with stirring at room temper-

13422

**Phosphosulfolactate Synthase from M. jannaschii**

![Diagram](attachment:diagram.png)

* Phosphoenolpyruvate → (R)-Phosphosulfolactate → (R)-Sulfolactate → Sulfopyruvate → Sulfoacetaldehyde → CoM

**FIG. 1. Biosynthetic pathway for coenzyme M.** Phosphosulfolactate synthase (ComA) produces (2R)-phospho-3-sulfolactate, which phosphosulfolactate phosphatase (ComB) hydrolyzes to form (R)-sulfolactate (L-sulfolactate) (18). An NAD⁺-dependent dehydrogenase (ComC) oxidizes (R)-sulfolactate to form sulfopyruvate (19). Sulfopyruvate decarboxylase (ComDE) produces sulfoacetaldehyde (20), which is reductively thiolated to yield coenzyme M.

After 24 h of stirring at room temperature, the organic phase was removed, and the aqueous layer was extracted two times with 6 ml of diethyl ether. The aqueous layer was then heated for 30 min at 100 °C, evaporated under vacuum, and dissolved in 3 ml of acetone. Dicyclohexylamine (3.5 mmol, 0.7 ml) was added dropwise at 0 °C with stirring. After overnight crystallization at 3 °C, 1.0 g of white crystals was isolated by filtration. The specific rotation of the D-cysteine-derived dicyclohexylamine (S)-sulfolactate salt (D-sulfolactate) was found to be [α]²⁵ = +3.5°, and that for the L-cysteine-derived (R)-sulfolactate salt (L-sulfolactate) was found to be [α]²⁵ = −3.1°. Analysis by gas chromatography of the (S)-sulfolactate for chiral purity as described below showed the presence of only the S-isomer, whereas the (R)-sulfolactate had an enantiomeric ratio of 83:17 R-isomer:S-isomer.

**Analysis of Sulfolactic Acids—** Sulfolactate samples were converted into volatile methyl derivatives for gas chromatography (17). Samples (10–100 μg) were dissolved in 100 μl of water and passed through a Dowex AG 50-H⁺ (200–400 mesh) column (1 × 4 mm; Bio-Rad) to form the free acids. The effluent was evaporated to dryness with a stream of nitrogen gas and then dissolved in 50 μl of methanol. An ether solution of diazomethane was added until the yellow color of the diazomethane persisted for 1 min, and the sample was evaporated to dryness as described above and then dissolved in 20 μl of methylene chloride. Gas chromatography-mass spectrometry (GC-MS) analyses were performed using a VG-70–70EHF GC-MS apparatus operating at 70 eV and equipped with an HP-5 column (0.32 mm × 30 m) programmed from 70 to 280 °C at 10 °C/min. Under these conditions, two different sulfolactate derivatives were detected. The first peak to elute was the dimethyl ester of O-methyl sulfolactate (M⁺ = 212 m/z, with a base peak at M⁺ – 59 = 153 m/z). The second was the dimethyl ester of sulfolactate (M⁺ = 198 m/z, with a base peak at M⁺ – 59 = 139 m/z and a strong fragment at M⁺ – 32 = 107 m/z). The mass spectrum of the compound comprising the first peak has been previously reported (17).

GC-MS analysis of the resulting mixture of methyl ester derivatives on a chiral column was used to establish the stereochemical purity of the samples. Samples were separated on a β-DEX 120 column (0.25 mm × 30 m × 0.25-μm film thickness; Supelco Inc.) programmed from 75 to 230 °C at 3 °C/min. On this column, the trimethyl derivatives eluted as a single peak, but the dimethyl derivatives were separated into two peaks, with the first corresponding to the S-isomer and the second corresponding to the R-isomer. Analysis of a racemic mixture of the two isomers gave two peaks with a ratio of the first to the second of 0.74. Analytical data were corrected to account for this unexplained deviation from the expected ratio of 1.0.

**Syntheses of ComA Inhibitor Candidates—** 3-Sulfo-2-sulfomethylpropionic acid was synthesized by the addition of sulfite to 2-bromomethylacrylic acid. 2-Bromomethylacrylic acid (1 mmol, 165 mg) was suspended in 1 ml of water and mixed with sodium sulfite (3 mmol, 378 mg; Mallinckrodt Chemical Works) dissolved in 1 ml of water, and the resulting clear colorless solution was heated at 110 °C for 12 h in a sealed vial. ¹H NMR spectroscopy was performed using a 500-MHz Jeol Eclipse 500 NMR spectrometer at the Virginia Polytechnic Institute NMR Facility. ¹H NMR analysis of a portion of the solution, after replacement of the water with ²H₂O, showed a quantitative production of the 3-sulfo-2-sulfomethylpropionate with resonances δ 3.26 ppm (4H, m, H-3, and H-3′) and 3.12 ppm (1H, m, H-2). The product concentration was determined from the ratios of the sample peaks to the sodium [2,2,3,3-²H₄]trimethylsilylpropionate (TSP) standard peak (δ = 0). The product was converted to the acid form by passage through a Dowex AG 50W-X8-H⁺ column (Bio-Rad) and evaporation of the solution to dryness.

2-Phosphonomethylacrylic acid (23) at a concentration of 0.18 M was heated for 1 h at 110 °C with a 3 M excess of sodium sulfite and, after cooling, passed though a Dowex AG 50-H⁺ column. ¹H NMR analysis established that the 2-phosphonomethylacrylate was quantitatively converted into 2-phosphonomethyl-3-sulfopropionic acid: resonances were δ 3.56 (1H, dd, $J_{3\rightarrow2} = 2.1$ Hz, $J_{3\rightarrow3'} = 14.1$ Hz, H-3), 3.22 (1H, dd, $J_{3'\rightarrow2} = 11.6$ Hz, $J_{3'\rightarrow3} = 14.1$ Hz, H-3'), 2.88 (1H, m, $J_{2\rightarrow3} = 2.1$ Hz, $J_{2\rightarrow\alpha} = 2.7$ Hz, $J_{2\rightarrow\alpha'} = 11.6$ Hz, $J_{2\rightarrow3'} = 11.6$ Hz, H-2), 1.71 (1H, ddd, $J_{\alpha\rightarrow2} = 2.7$ Hz, $J_{\alpha\rightarrow\alpha'} = 14.8$ Hz, $J_{\alpha\rightarrow P} = 11.6$ Hz, H-$\alpha$), and 1.53 (1H, ddd, $J_{\alpha'\rightarrow2} = 11.6$ Hz, $J_{\alpha'\rightarrow\alpha} = 14.8$ Hz, $J_{\alpha'\rightarrow P} = 14.3$ Hz, H-$\alpha'$).

**Cloning and Recombinant Expression of the MJ0255 Gene in E. coli—** The *M. jannaschii* gene at locus MJ0255 (encoding the protein submitted to Swiss Protein Database under Swiss-Prot accession number Q57703 (34)) was amplified by PCR from genomic DNA using oligonucleotide primers synthesized by Invitrogen. Primer MJ0255-Fwd (5'-GGTGAGCATATGAAAGCATTTG-3') introduced an *NdeI* restriction site at the 5'-end of the amplified DNA, whereas primer MJ0255-Rev (5'-GATCGGATCCTAAACCTTCC-3') introduced a *BamHI* site at the 3'-end. PCRs contained 1× GeneAmp PCR buffer (Applied Biosystems), 1 μM each primer, 200 μM each dNTP, 1 μg of *M. jannaschii* JAL-1 chromosomal DNA, and 5 units of AmpliTaq LD DNA polymerase (Applied Biosystems) in a volume of 100 μl. DNA was amplified during 35 cycles, and each cycle included incubation at 95 °C for 1 min, 45 °C for 2 min, and 72 °C for 3 min. PCR product DNA was purified using a QIAquick spin column (QIAGEN Inc.) and then digested with *NdeI* and *BamHI* restriction enzymes (Invitrogen). DNA fragments were ligated into compatible sites in plasmid pET19b (Novagen) or pT7-7 (26) using bacteriophage T4 DNA ligase (Invitrogen). Recombinant plasmids were transformed into *Escherichia coli* NovaBlue (Novagen) and *E. coli* BL21-CodonPlus(DE3)-RIL (Stratagene). Sequences of cloned DNA were confirmed by dye terminator cycle sequencing (Virginia Bioinformatics Institute Core Laboratory Facility).

Transformed *E. coli* cells were grown in Luria-Bertani/Miller broth (1 liter; Difco) supplemented with 100 mg/liter ampicillin. Cultures were shaken at 37 °C until they reached an absorbance at 600 nm of 1.0. Recombinant protein production was then induced with 28 mM lactose. After an additional 4-h incubation with shaking at 37 °C, the cells were harvested by centrifugation (6000 × g, 10 min) and frozen at −20 °C.

**Purification of the ComA Protein—** Recombinant ComA protein was purified by heat treatment and chromatography of soluble cell-free extract. *E. coli* cells (7.4 g, wet weight) expressing ComA from the pT7-7-derived plasmid were suspended in 20 ml of buffer A (20 mM bis-Tris-HCl (pH 6.5)) and lysed by sonication. Soluble cell-free extract was obtained after centrifugation at 20,000 × g for 10 min at 4 °C. Native *E. coli* proteins were denatured by heating soluble extract at 70 °C for 20 min, and then insoluble material was removed by centrifugation at 20,000 × g for 15 min at 4 °C. Heat-soluble cell-free extract (17 ml) was sealed in $M_r$ 6000–8000 cutoff membranes (Arthur H. Thomas) and dialyzed for 10 h at 4 °C against 1 liter of buffer A. Dialyzed solutions were concentrated in the membranes using polyethylene glycol 8000 and then applied to a MonoQ HR anion-exchange column (10 mm × 8 cm; Amersham Biosciences) equilibrated in buffer A. Pumps attached to the column were controlled by a BioLogic HR workstation (Bio-Rad). Protein bound to the column was washed with 0.33 M NaCl in buffer A and then eluted with 0.55 M NaCl in buffer A at a flow rate of 1 ml/min. Fractions (1 ml) were collected automatically, and those containing PSL synthase activity were pooled and dialyzed for 10 h at 4 °C against 1 liter of buffer B (20 mM Tris-HCl (pH 7.6)).

Dialyzed protein was concentrated using polyethylene glycol 8000 and then applied to a DEAE-Sepharose FF column (16 mm × 5.2 cm; Amersham Biosciences) equilibrated in buffer B. Protein was eluted with a 30-ml linear gradient of 0–1 M NaCl in buffer B at a flow rate of 0.5 ml/min. Fractions (1.0 ml) were assayed for PSL synthase activity. Fractions containing activity (which elute at ~0.35 M NaCl) were pooled and concentrated in an N₂-pressurized stirred cell with a YM-10 ultrafiltration membrane (Millipore Corp.).

Concentrated protein solutions were heated for 20 min at 70 °C,

cooled on ice, and centrifuged for 10 min at 16,000 × g at room temperature. The supernatant was applied to a Superose 12HR column (10 mm × 31 cm; Amersham Biosciences) equilibrated in 50 mM Tris-HCl (pH 8.0), 150 mM NaCl, and 1 mM MgCl₂. Chromatography was performed in this buffer at a flow rate of 0.5 ml/min, and fractions of 0.75 ml were collected. Fractions containing PSL synthase activity were pooled and dialyzed against 1 liter of 20 mM Tris-HCl (pH 8.0) and 1 mM MgCl₂ for 10 h at 4 °C. Dialyzed protein was concentrated in the stirred cell ultrafiltration unit.

Protein purity was evaluated by SDS-PAGE with silver staining (Bio-Rad). Total protein concentrations of samples were measured using the Bradford method (Bio-Rad) with bovine serum albumin as a standard. The final concentration of purified protein was verified using the BCA total protein assay (Pierce) with bovine serum albumin as a standard.

Recombinant proteins expressed with an amino-terminal polyhistidine tag (ComAHis and ComAHis-K137N) were purified by nickel affinity column chromatography. *E. coli* cells (0.8 g, wet weight) expressing recombinant fusion proteins from pET19b-derived plasmids were suspended in buffer C (50 mM Hepes/NaOH (pH 7.6) and 0.5 M NaCl) and then processed as described for ComA. Heat-soluble cell extracts were loaded onto a HiTrap chelating column (1.5 × 2.5 cm; Amersham Biosciences) charged with NiCl₂ and equilibrated in buffer C with 10 mM imidazole. Protein bound to the column was washed and eluted using a stepwise isocratic elution (5 ml of 0.05, 0.1, 0.4, 0.6, and 1 M imidazole in buffer C (pH 7.5)) at a flow rate of 1 ml/min. Fractions (1 ml) were collected and then analyzed by SDS-PAGE. Fractions containing recombinant protein, eluted at 1 M imidazole, were pooled and desalted using a Sephadex G-25 column (1.6 × 14.5 cm; Amersham Biosciences) equilibrated in 50 mM Hepes/NaOH (pH 7.1). Purified, desalted protein preparations produced a single band on SDS-polyacrylamide gels stained with Coomassie Blue R-250 dye.

Analytical Size-exclusion Chromatography—Protein quaternary structure was analyzed using a Superose 12HR column (10 mm × 30 cm). The column was equilibrated and operated at a flow rate of 0.5 ml/min in buffer containing 50 mM Hepes/NaOH (pH 7.5), 150 mM NaCl, and 1 mM MgCl₂. Protein standards used to calibrate the sizing column were horse spleen apoferritin, potato β-amylase, yeast alcohol dehydrogenase, hen egg white conalbumin, bovine erythrocyte carbonic anhydrase, and horse heart cytochrome c. Eluted protein was detected by its absorbance at 280 nm and PSL synthase activity (for wild-type enzyme). Molecular masses and elution volumes of standards were successfully fit to an exponential equation using nonlinear least-squares regression (SigmaPlot 2000, SPSS Inc.).

Measurement of Phosphosulfolactate Synthase Activity—One unit of PSL synthase activity incorporates 1 μmol/min sulfite into phosphosulfolactate. Standard activity assays included 50 mM Hepes/NaOH (pH 7.5), 5 mM MgCl₂, 1 mM NaHSO₃ (Eastman Kodak Co.) or 1 mM Na₂SO₃, and 1 mM P-enolpyruvate in 50 μl. Sulfite stock solutions were prepared daily. Assay mixtures were preincubated at 70 °C for 10 min before the addition of enzyme. After 7 min of incubation at 70 °C, activity was terminated by the addition of 5 μl of 10× stop solution (0.5 M arginine and 0.1 M EDTA adjusted to pH 12.8 with NaOH). Enzyme was diluted in buffer containing 100 mM Tris-HCl (pH 8.0) and 10 mM MgCl₂.

Enolase activity was measured using a continuous assay of P-enolpyruvate absorbance at 230 nm (27). Yeast enolase (Sigma) was used as a control (at 25 °C). Reactions (1 ml) containing 50 mM KCl, 50 mM Hepes/NaOH (pH 7.5), 1 mM MgCl₂, and 0.5 mM P-enolpyruvate or 1 mM D-2-phosphoglycerate were initiated by the addition of enzyme. ComA enolase activity was measured at 60 °C.

Measurement of Sulfite—Monobromobimane (Calbiochem) was used to measure sulfite concentrations (28). Terminated enzymatic activity assays (55 μl) were reacted with 3 μl of 50 mM monobromobimane dissolved in acetonitrile. Reactions were incubated away from light for 15 min at room temperature and then diluted to 1 ml with 50 mM glycine and 10 mM EDTA adjusted to pH 10 with NaOH. Fluorescence of the sulfite-monobromobimane adduct was measured in methacrylate cuvettes using a PerkinElmer Life Sciences 650-40 fluorescence spectrophotometer with excitation at 410 nm and emission at 480 nm. Sulfite concentrations were calculated using linear regression results from a standard curve consisting of 0–1 mM NaHSO₃ or Na₂SO₃ (50 μl) in enzyme reaction buffer. Standards were processed as described for the activity assays.

Temperature and Cation Effects on Phosphosulfolactate Synthase Activity—Phosphosulfolactate synthase activity was optimized by varying reaction conditions, including temperature, pH, and inorganic cation concentrations. All reactions were carried out under enzymatic activity-limited conditions. Effects of reaction temperature were studied in standard assays initiated by the addition of sulfite to reaction mixtures pre-equilibrated in water baths. Reactions were incubated at temperatures from 23 to 90 °C for 5–10 min and then terminated with alkaline stop solution and cooled on ice. The influence of pH on ComA activity was studied using approximately constant ionic strength buffers (29). Buffer mixtures containing 0.5 M acetic acid (Fisher), 0.5 M Mes, and 1 M Tris were adjusted to pH 4.0–8.5 at room temperature using NaOH or HCl. Buffers containing 1 M Aces, 0.5 M Tris, and 0.5 M ethanolamine HCl (Calbiochem) were similarly adjusted to pH 6.0–10.0. Single component buffers containing 0.5 M glycine/NaOH (pH 10.0), 0.5 M Caps/NaOH (pH 10–11), 0.5 M L-arginine/NaOH (pH 11.5–12), or 0.5 M guanidine/NaOH (pH 12) were used to test activity at alkaline pH values. Buffers were diluted 10-fold in standard assays, and reactions were initiated by adding sulfite. Effects of KCl (Fisher), NaCl (Fisher), LiCl (J. T. Baker Inc.), or NH₄Cl (Fisher) were also tested in standard assays. Divalent cation requirements were tested using Hepes and P-enolpyruvate reagents purified by passage through a 7.5 mm × 7.5-cm column of Chelex 100-Na⁺ (Bio-Rad) (30). ComA was purified on a 0.5 × 6.5-cm column containing Chelex 100-Na⁺ and quantified by its absorbance at 280 nm. Metal replacement reactions included 5 mM concentrations of MgCl₂·6H₂O (Fisher), MnCl₂·4H₂O (Fisher), CaCl₂·6H₂O (Fisher), NiCl₂·6H₂O, Zn(C₂H₃O₂)₂·2H₂O, BaCl₂·2H₂O (Fisher), CuCl₂ (Fisher), CoCl₂·6H₂O (J. T. Baker Inc.), Fe(NH₄)₂(SO₄)₂·6H₂O, or EDTA.

Kinetic Analysis of ComA Activity—For each substrate (P-enolpyruvate, sulfite, and Mg²⁺), initial rates of ComA-catalyzed phosphosulfolactate synthase activity were measured varying a single substrate concentration, with the other two substrates present in excess. All reactions were preincubated at 70 °C. Assays (120 μl) varying the P-enolpyruvate concentration contained 10–20 mM MgCl₂, 50 mM Hepes/NaOH (pH 7.2), and 8 μg/ml ComA and were initiated with 1.5 mM NaHSO₃. Assays (120 μl) varying the sulfite concentration contained 10 mM MgCl₂, 2 mM P-enolpyruvate, and 50 mM Hepps/NaOH (pH 8.0) and were initiated with 8 μg/ml ComA. The initial rate at each P-enolpyruvate or sulfite concentration was calculated from measurements of residual sulfite in a 20-μl reaction volume at 0.25, 2.5, and 5.0 min. Fixed time point assays at varying MgCl₂ concentrations contained 16 μg/ml Chelex-purified ComA, 50 mM Hepes/NaOH (pH 7.6), and 2.6 mM P-enolpyruvate. Reactions were initiated with 1 mM sulfite and stopped after 5 min. Initial rate data from low substrate concentrations were successfully fitted to the hyperbolic Michaelis-Menten equation, and kinetic parameters were calculated using the Levenberg-Marquardt method of nonlinear least-squares regression (SigmaPlot 2000). Initial rate data obtained over the full range of concentrations tested were fitted to a simple hyperbolic equation for substrate inhibition: \( v = V \cdot A / ((K_M + A + A^2)/K_i) \) (31).

Analysis of 2-Phosphosulfolactate Deuterium Exchange Activity—(RS)-Phosphosulfolactate (4.2 μmol) was mixed with 1 μmol of MgCl₂ and 10 μmol of Tris-HCl (pH 8.0), lyophilized using a SpeedVac apparatus (Savant Instruments, Inc.), and then suspended in 0.5 ml of ²H₂O (99.8 atom % ²H) with TSP. Enzyme (25 μg) was mixed with 5 nmol of MgCl₂, dried by lyophilization, and then suspended in 40 μl of ²H₂O. The enzyme solution was added to the (RS)-phosphosulfolactate and incubated at 70 °C for 60 min. Reactions were cooled to room temperature and then analyzed by ¹H NMR at 23 °C.

Stereochemical Analysis of the ComA Reaction Product—Phosphosulfolactate was hydrolyzed by bacterial alkaline phosphatase (0.2 units), and then the resulting sulfolactate was purified using Dowex AG 1-X2-Cl⁻ resin (Bio-Rad) (17). Sulfolactate was derivatized and analyzed by chiral GC-MS as described above.

Inhibition of ComA PSL Synthase Activity—Potential inhibitors of ComA activity were tested in reactions (50 μl) containing 1.2 mM P-enolpyruvate, 1 mM Na₂SO₃, 50 mM Hepes/NaOH (pH 7.5), 5 mM MgCl₂, and 15 μg/ml ComA. Fixed time point reactions were initiated, incubated, and terminated after 6 min as described for standard assays. To account for interferences with the monobromobimane detection of sulfite, residual sulfite concentrations in samples were subtracted from those of inhibitor control reactions incubated without ComA. Net specific activities were compared with those of ComA reactions without inhibitor to calculate relative activities.

Site-directed Mutagenesis—To test the function of a conserved lysine residue in ComA, Lys¹³⁷ was replaced with asparagine. The QuikChange™ site-directed mutagenesis kit (Stratagene) was used according to the manufacturer’s instructions with template pMjcomA-His (*comA* in pET19b). Oligodeoxynucleotide primers were synthesized by Invitrogen: K137N-Fwd (5′-GGTTTTAACAGAAGTTGGTAAAAAC-ATGCCAGATAAGGATAAACAGC-3′) and K137N-Rev (5′-GCTGTTT-

13424 Phosphosulfolactate Synthase from *M. jannaschii*

**TABLE I**

Purification of *M. jannaschii* ComA

ComA was purified from 7.4 g of recombinant E. coli cell paste as described under “Experimental Procedures.”

| Purification step       | Volume ($ml$) | Total protein ($mg$) | Total activity ($units$) | Specific activity ($units/mg$) | Yield (%) |
|-------------------------|---------------|----------------------|--------------------------|---------------------------------|-----------|
| Soluble extract         | 22            | 579                  | 482                      | 0.83                            | 100       |
| Heat treatment          | 17            | 183                  | 159                      | 0.87                            | 33        |
| MonoQ pool              | 15            | 125                  | 216                      | 1.73                            | 45        |
| DEAE pool               | 3             | 81                   | 159                      | 1.96                            | 33        |
| Superose 12HR pool      | 1             | 58                   | 138                      | 2.37                            | 29        |

$a$ Total protein was measured using the Bio-Rad protein assay.

$b$ Phosphosulfolactate synthase activity was quantified by measuring the monobromobimane derivative of the remaining sulfite substrate.

ATCCTTATCTGGCATGTTTTACCAACTTCTGTAAACC-5'). DNA sequences were verified by dye terminator sequencing at the University of Iowa DNA Facility.

Identification of ComA Homologs, Sequence Alignment, and Phylogenetic Inference—The translated sequence of the *M. jannaschii* MJ0255 gene (Swiss-Prot accession number Q57703) was used to query the non-redundant protein data base at the National Center for Biotechnology Information using the BLASTP program (Version 2.2.1) (32) with the BLOSUM62 matrix and default gap costs for existence—11 and extension—1. Homologs were identified in the archaeon *Methanobacterium thermoautotrophicum* (GenBank™/EBI accession number AAB86146.1); in the bacteria *Xanthobacter sp.* Py2 (GenBank™/EBI accession number AAG61256.1) and *B. subtilis* (EMBL accession number CAB12935.1); and in the eukaryotes *Arabidopsis thaliana* (EMBL accession number CAB79131.1), *Lycopersicon esculentum* (cDNAs; GenBank™/EBI accession numbers BE436489.1 and AW930285.1), and *Glycine max* (cDNA GenBank™/EBI accession number BE346999.1). Additional homologs were identified in incomplete genomic sequences from *Ferroplasma acidarmanus*, *Cytophaga hutchinsonii*, and *Phanerochaete chrysosporium* (www.jgi.doe.gov); *Methanococcus maripaludis* (www.genome.washington.edu/Methanococcus.html); *Bordetella bronchiseptica* (www.sanger.ac.uk); and *Aspergillus fumigatus*, *Entamoeba histolytica*, and *Coxiella burnetii* (www.tigr.org).

Amino acid sequences were aligned automatically using the ClustalW program (Version 1.82) (33). From the alignment of 15 protein sequences, 223 positions were deemed to be confidently aligned. These were analyzed by protein maximum likelihood methods using the ProML program (PHYLIP phylogeny inference package), Version 3.6a2.1) (53) with the JTT amino acid substitution model. Bootstrap proportions were calculated using Seqboot, ProML, and Consense programs from PHYLIP.

**RESULTS**

Identification, Expression, and Purification of *M. jannaschii* ComA—The *comA* gene was identified based on its physical proximity to previously identified CoM biosynthetic genes in *M. jannaschii* (19, 20, 34) and in *Xanthobacter sp.* Py2 (35). Homologs of *comA* are found in complete genome sequences of all organisms known to synthesize CoM or sulfolactate, except for *Methanosarcina* spp. In *E. coli* cells that recombinantly expressed the *M. jannaschii comA* gene, the ComA protein composed 35% of the cells' total soluble protein. Protein purification by heating (to denature most native *E. coli* proteins) and anion-exchange chromatography removed most contaminating proteins and nucleic acids. Table I shows the purification of 58 mg of ComA protein from 7.4 g of recombinant *E. coli* (wet mass) at a concentration of 54 mg/ml and a specific activity of 2.4 units/mg.

Purified ComA preparations showed a single band on a silver-stained SDS-polyacrylamide gel with an apparent molecular mass of 32,000 Da, close to its predicted molecular mass of 28,370 Da. From a Sepharose 12HR size-exclusion column, ComA eluted with a Stokes radius of 37 Å, corresponding to an apparent molecular mass of 86,700 Da. This elution profile suggests that ComA forms a trimer; a minor peak (<5% total protein) corresponded to a hexameric form of ComA. Diluted in phosphate-buffered saline (pH 7.4), ComA had a single absorbance maximum at 276 nm and an extinction coefficient ($\epsilon_{280}$) of

0.84 ml/mg/cm. The enzyme retained full activity when stored for 2 months at 4°C in 20 mM Tris-HCl and 1 mM MgCl₂ (pH 8.0).

Phosphosulfolactate Synthase Activity—(R)-Phosphosulfolactate produced by ComA was identified by GC-MS as described below. However, phosphosulfolactate is not readily quantified; therefore, enzymatic activity was determined by measuring residual sulfite substrate in discontinuous assays. Standard reactions were initiated by the addition of sulfite to a mixture of ComA, P-enolpyruvate, buffer, and MgCl₂ preincubated at 70°C. Reactions were stopped with a solution of L-arginine and EDTA (pH 12.8). At alkaline pH, sulfite reacts quantitatively with monobromobimane to form a fluorescent compound that has an excitation maximum at 410 nm and an emission maximum at 480 nm (28, 36). Phosphoenolpyruvate also reacts spontaneously with excess sulfite to form (RS)-phosphosulfolactate (18, 37); however, this non-enzyme reaction was insignificant at the low sulfite concentrations and short incubation times used in the enzymatic assays described here.

The ComA enzyme is metal-activated: it requires several molar equivalents of Mg²⁺/P-enolpyruvate for maximum activity. In the absence of Mg²⁺ or in the presence of a molar excess of EDTA versus Mg²⁺, this enzyme is inactive. To test metal ion requirements, ComA, P-enolpyruvate, and Hepes solutions were passed through a Chelex 100 column. The metal-free enzyme had no phosphosulfolactate synthase activity, but activity was restored by the addition of 5 mM MgCl₂ (3.3 units/mg for Chelex-treated ComA) versus 4.0 units/mg for originally purified ComA). When incubated with 1 mM bisulfite and 1 mM P-enolpyruvate, ComA required 5 mM MgCl₂ for maximum activity. No other divalent cation restored full activity at a 5 mM concentration (CaCl₂ (25% relative activity), NiCl₂ (18%), MnCl₂ (15%), and CuCl₂ (10%)), whereas CoCl₂, Fe(NH₃)₂(SO₄)₂, Zn(C₂H₃O₂)₂, and BaCl₂ supported no activity.

ComA is active over a wide pH range (pH 6–11), with maximum activity at pH 8.5. At alkaline pH, the SO₃²⁻ species (pKa = 7.2 at 25°C) rather than bisulfite appears to be the relevant substrate. LiCl was a weak inhibitor of ComA activity (~50% activity at 100 mM LiCl₂), but other monovalent cations (K⁺, Na⁺ and NH₄⁺) had no effect on activity at 100 mM concentrations. ComA activity increased with temperature up to 90°C (the practical limit of the assay). Enzyme heated at 125°C for 15 min prior to assaying at 70°C showed no activity.

To estimate the efficiency of ComA in synthesizing phosphosulfolactate, initial reaction rates were measured at varying P-enolpyruvate, sulfite, or MgCl₂ concentrations, with other substrates present in excess (38). Fig. 2 shows that high concentrations of each substrate inhibited activity. ComA was also inhibited by phosphosulfolactate; however, concentrations of sulfite converted to PSL in inhibited reactions (measured after 5 min) were <10% of the concentration of racemic phosphosulfolactate (4 mM) required to reduce ComA activity by 50%. Activities measured over the full range of tested P-enolpyruvate concentrations were fit to a simple model for substrate

Phosphosulfolactate Synthase from M. jannaschii 13425

FIG. 2. Initial rate plots of ComA activity at varying substrate concentrations of phosphoenolpyruvate (A), sulfite (B), or MgCl₂ (C). One unit of phosphosulfolactate synthase activity incorporates 1 μmol/min sulfite into phosphosulfolactate. Initial rate data are fit to a simple model of substrate inhibition.

13426

**Phosphosulfolactate Synthase from M. jannaschii**

**aci-carboxylates**

P-enolpyruvate

sulfite

$(R)$-PSL

FIG. 3. Stepwise model for reactions in $(R)$-phosphosulfolactate formation by ComA. To form $(R)$-phosphosulfolactate, P-enolpyruvate must be protonated on the $re$-face.

inhibition. The pseudo first-order apparent kinetic parameters for P-enolpyruvate were $V_{\text{max}} = 39$ units/mg, $K_m = 9.8 \, \mathrm{mM}$, and $K_i = 1.9 \, \mathrm{mM}$. When activities measured only at low P-enolpyruvate concentrations ($0–3.6 \, \mathrm{mM}$) were fit to the Michaelis-Menten-Henri equation, the apparent kinetic parameters were significantly lower: $V_{\text{max}} = 9 \pm 0.6$ units/mg and $K_m = 0.9 \pm 0.2 \, \mathrm{mM}$. Fitting the activities at the full range of tested sulfite concentrations to the inhibition model produced apparent values of $V_{\text{max}} = 33$ units/mg, $K_m = 2 \, \mathrm{mM}$, and $K_i = 3 \, \mathrm{mM}$. Considering only activities at low sulfite concentrations ($0–2 \, \mathrm{mM}$) produced lower apparent values: $V_{\text{max}} = 18 \pm 2$ units/mg and $K_m = 1 \pm 0.2 \, \mathrm{mM}$. Finally, a model of activities at all tested $\mathrm{MgCl}_2$ concentrations produced apparent values of $V_{\text{max}} = 13$ units/mg, $K_m = 4.2 \, \mathrm{mM}$, and $K_i = 1 \, \mathrm{mM}$. At low $\mathrm{MgCl}_2$ concentrations ($0–5 \, \mathrm{mM}$), apparent values were lower: $V_{\text{max}} = 8$ units/mg and $K_m = 2 \, \mathrm{mM}$.

When incubated with $(RS)$-phosphosulfolactate, ComA does not produce sulfite or affect the chemical shift or intensities of the C-3 proton resonances, as observed by ${}^1\mathrm{H}$ NMR. Therefore, the phosphosulfolactate synthase reaction is effectively irreversible. In addition, ComA does not have detectable enolase activity when incubated with P-enolpyruvate or D-2-phosphoglycerate in the absence of sulfite (<1% specific activity of yeast enolase).

**Phosphosulfolactate Proton Isotope Exchange—**Abstraction of an $\alpha$-proton from a carboxylic acid typically proceeds stereospecifically through an enolate (aci-carboxylate) intermediate (39). Proton isotope exchange experiments were performed to test this proposed intermediate in a stereospecific partial reaction of ComA (Fig. 3). ComA${}^{\mathrm{His}}$ was incubated with $(RS)$-phosphosulfolactate and $\mathrm{MgCl}_2$ in ${}^2\mathrm{H}_2\mathrm{O}$ at $70^\circ \mathrm{C}$ for $1 \, \mathrm{h}$. The ${}^1\mathrm{H}$ NMR spectroscopy data in Fig. 4 show that ComA${}^{\mathrm{His}}$ exchanged $50\%$ of the racemic phosphosulfolactate C-2 protons for deuterons. No exchange of C-3 protons was observed: the intensities of these resonances relative to the TSP internal standard remained unchanged.

**Stereochemistry of the Phosphosulfolactate Product—**The deuterium-exchanged sample was treated with alkaline phosphatase, and the resulting sulfolactate was converted into a dimethyl ester derivative and analyzed by GC-MS with a chiral column. By selective ion monitoring of the $m/z \, 139$ and $m/z \, 140$ ions in the eluting peaks, the extent of deuterium incorporation into each of the stereoisomers could be measured. The results in Fig. 5 show an equal mixture of the two isomers, with deuterium incorporated only in the $R$-isomer. The extent of labeling was essentially quantitative considering the amount of non-deuterated water in the sample. A similar analysis of the phosphosulfolactate produced from P-enolpyruvate and sulfite by ComA showed an enantiomeric ratio of $84:16 \, R$-isomer:$S$-isomer. The presence of $(S)$-sulfolactate can be attributed to its nonenzymatic formation during the $3 \, \mathrm{h}$ incubation period used in this experiment.

**Inhibitors of ComA Activity—**In reactions containing $1.2 \, \mathrm{mM}$ P-enolpyruvate, $1 \, \mathrm{mM} \, \mathrm{Na}_2\mathrm{SO}_3$, $5 \, \mathrm{mM} \, \mathrm{MgCl}_2$, and $26 \, \mu \mathrm{M}$ ComA, a number of substrate and product analogs were tested for their ability to inhibit ComA-catalyzed PSL synthesis. Less than $50\%$ relative PSL synthase activity was measured in the presence of $4 \, \mathrm{mM} \, (RS)$-phosphosulfolactate, $3 \, \mathrm{mM} \, (R)$-sulfolactate, $2 \, \mathrm{mM} \, (R)$-phosphomalate, $2 \, \mathrm{mM} \, (S)$-phosphomalate, $200 \, \mathrm{mM} \, \mathrm{NH}_4\mathrm{SO}_4$, $200 \, \mathrm{mM} \, \mathrm{Na}_2\mathrm{SO}_4$, $10 \, \mathrm{mM}$ phosphonoacetic acid, $10 \, \mathrm{mM}$ 2-carboxyethylphosphonate, $3 \, \mathrm{mM}$ phosphoglycolate, $5 \, \mathrm{mM}$ sulfopyruvate, or $3 \, \mathrm{mM}$ 2-phosphono-3-sulfopropionate. Conversely, ComA retained at least $50\%$ relative activity in the presence of $2 \, \mathrm{mM}$ 3-bromopropanesulfonic acid, $1 \, \mathrm{mM} \, \mathrm{CoM}$, $50 \, \mathrm{mM} \, \mathrm{NH}_4\mathrm{SO}_4$, $100 \, \mathrm{mM} \, \mathrm{NH}_4\mathrm{Cl}$, $5 \, \mathrm{mM} \, \mathrm{NaF}$ (with or without $5 \, \mathrm{mM} \, \mathrm{NaHPO}_4$), $5 \, \mathrm{mM} \, \mathrm{BaCl}_2$, $10 \, \mathrm{mm}$ oxalate, $2 \, \mathrm{mM} \, \mathrm{Na}_2\mathrm{SeO}_3$, $10 \, \mathrm{mM} \, \mathrm{NaNO}_3$, $5 \, \mathrm{mM} \, \mathrm{NaHPO}_4$, $10 \, \mathrm{mM} \, \mathrm{NaHCO}_3$, $5 \, \mathrm{mM} \, (S)$-phospholate, $3 \, \mathrm{mM} \, (R)$-phospholate, $5 \, \mathrm{mM} \, (S)$-lactate, $5 \, \mathrm{mM} \, (R)$-lactate, $5 \, \mathrm{mM} \, (R)$-2-phosphoglycerate, $5 \, \mathrm{mM} \, (R)$-glycerate, $5 \, \mathrm{mM}$ 3-bromopropanesulfonate, $5 \, \mathrm{mM}$ methyl phosphate, $5 \, \mathrm{mM}$ sodium acrylate, $5 \, \mathrm{mM} \, (S)$-sulfolactate, $5 \, \mathrm{mM}$ 2-phosphonomethylacrylate, $3 \, \mathrm{mM}$ phosphonoacetohydroxamate, or $5 \, \mathrm{mM}$ 3-sulfo-2-sulfomethylpropionate.

**Characterization of a K137N Mutant—**Site-directed mutagenesis was used to change the amino acid at position 137 in ComA${}^{\mathrm{His}}$ from lysine to asparagine. The resulting ComA${}^{\mathrm{His}}$-K137N mutant had no measurable phosphosulfolactate synthase activity (<4% activity relative to ComA${}^{\mathrm{His}}$) and did not catalyze the isotope exchange of the C-2 proton of either phosphosulfolactate stereoisomer (Fig. 4). Purified wild-type and mutant ComA${}^{\mathrm{His}}$ both formed single bands on SDS-polyacrylamide gels corresponding to their expected molecular masses, and both eluted from a size-exclusion column in peaks corresponding to trimeric and hexameric forms.

**Sequence Analysis of ComA Homologs and Phylogenetic Inference—**Fig. 6 shows an alignment of the $M. \, \mathrm{jannaschii}$ ComA amino acid sequence with several homologous sequences. Most amino acid positions are not conserved. However, several conserved acidic residues (Asp${}^{40}$, Glu${}^{103}$, Glu${}^{133}$, Glu${}^{171}$, and Glu${}^{205}$) could function as ligands for $\mathrm{Mg}^{2+}$, and two basic residues (Lys${}^{43}$ and Lys${}^{137}$) may function as general acids to protonate an enolate intermediate (Fig. 3) or to bind sulfite.

The phylogeny of $comA$ homologs, inferred from an alignment of 15 highly diverged sequences, shows two distinct groups (Fig. 7). The first group comprises the archaeal, bacterial, and *Entamoeba* homologs, which likely function as phosphosulfolactate synthases in the context of sulfolactate or sulfopyruvate synthesis. Most of the organisms represented in this group have at least one other sulfolactate/CoM biosynthesis gene. Plants and some fungal species contain $comA$ homologs in the second group; these organisms do not have homologs of other CoM biosynthesis genes. Bootstrap resamplings support the major groupings, suggesting that the plant homolog is widespread and was present early in the evolution of the plant lineage.

Phosphosulfolactate Synthase from M. jannaschii

FIG. 4. ${}^{1}$H NMR (500 MHz) spectra from (RS)-phosphosulfolactate proton isotope exchange experiments. ComA${}^{\text{His}}$ exchanged 50% of the C-2 hydrogens for deuterons (A), whereas ComA${}^{\text{His}}$-K137N had no proton exchange activity (B). The spectrum for substrate in the absence of enzyme is equivalent to that shown in B. Multiplets for phosphosulfolactate (in ${}^{2}$H$_{2}$O) were measured at $\delta$ 4.66 ppm (H-2) and 3.37 ppm (H-3 and H-3'). Additional peaks were produced by buffer salts and HDO.

Parts per million

FIG. 5. GC-MS ion profiles of a characteristic sulfolactate mass spectral fragment. Deuterium-exchanged (RS)-phosphosulfolactate by ComA${}^{\text{His}}$ (see Fig. 4A) was hydrolyzed and derivatized to form dimethyl sulfolactate derivatives. GC-MS analysis using a chiral $\beta$-DEX 120 column resolved two peaks. The first peak (eluting at 37:17 min) corresponds to the (S)-sulfolactate derivative, whereas the second peak (eluting at 37:24 min) corresponds to the (R)-sulfolactate derivative. Ion profiles at M${}^{+}$ - 59 = 139 m/z (A) and M${}^{+}$ - 59 = 140 m/z (B) show that ComA${}^{\text{His}}$ exchanged the C-2 proton of only (R)-phosphosulfolactate and not (S)-phosphosulfolactate.

DISCUSSION

Phosphosulfolactate synthase is one of the first biosynthetic enzymes demonstrated to use sulfite. Because of its high nucleophilicity and rapid reactivity in vitro with unsaturated carbon atoms in nucleotides (40), fatty acids (41), and aldehydes (42), sulfite has been discounted as a short-lived intermediate in sulfur oxidation or sulfate reduction. Nevertheless, UDP-sulfoquinovose synthase (SQD1) from A. thaliana (12)

13428

**Phosphosulfolactate Synthase from M. jannaschii**

| M. jannaschii (ComA) | 1 --MKAEFEFLY--EDFORGLTVLIDKG--LPPKFVEDYLKVCGDYIDFKFGWGSTSAVIDRDVVKEKINYYKDWGIKVYPGTLFEYA |
| M. thermoauto. (ComA) | 1 --MNAFDFTPPRSGKPRKNGITMVLDKG--MGPASARDLMEISSDYVDFIKFGWGTLPPLHRRDTVKEKVDMYRSFDVEPYPGGTLLFEIA |
| Xanthobacter (XecG)   | 1 MQARSDRPWRGVLALDAALDRRVSRPKRGITMVIDKG--IGPAAMADIAAAVAPYIDHWKLAEGTSALMPPQLADKLAFRLERGVLTYPGGTLLLEAA |
| B. subtilis (YitD)    | 1 --MNDFSLELPVRTNKPRETGQSILIDNG--YPLQFFKDAAIAGASDYIDFVKFGWGSTSLLT-KDLEEKISTLKEHDITFFEGGTLFEKY |
| A. thaliana           | 1 --MAAYYRWKSFEENED-RPEKPRRYGVTEMRGPHYSVLSQNLLQEIFESMGQEVVDGLKFSGGSNSLIPKSFIKQAIEMAHEHGYYVSTGDWAEHML |

| M. jannaschii (ComA) | 82 YS-KGKFDEFLNECEKLGFEBEISDGSSDISLEERKNAIKRAKDNGFMVLTEVGKKMPDKDKQLTTIDDRIKLINFDLDAG---ADYVIIEGR---ES |
| M. thermoauto. (ComA) | 87 HL-NDKVEEYFQEARSLGFETLEISNGTVEIETEEKCRLIEMAVDEGFMLSSEVGKKDPERDRILLEPDDRVRLVRADLRAG---ASMVLMEAR---ES |
| Xanthobacter (XecG)   | 98 IV-QQHCRVFMRQARDLGFSABEISDGTIDLPRDRRRRIIDCAREAGLVVITEVGKKDPQ--CQPEAAELAAEQALDDLKWG---SSFIVVEAR---ES |
| B. subtilis (YitD)    | 85 VS-QKKVNEFHRYCTYFGCEYIEISNGTLPMTNKEKAAYIADFSDE-FLVLSEVGSKDAELASRQSSSEEWLEYIVEDMEAG---AEKVITEAR---ES |
| A. thaliana           | 95 RSGPSAFKDYVEECKQLGFDTIELNANLLEVPEETLLRYVRLIKNGGLRAKPMFAVKFNKSDIPGRNRAFGSYVPEPRSSEFVEDIDLLIRKAEERCLEA |

| M. jannaschii (ComA) | 173 GKGIGLFDKE--GKVKENELDVLAKNV--DINKVIFEAPQKSQQVAFILKFSSSVNLANIADFDEISETLRRGLRGDTFGKV---------- |
| M. thermoauto. (ComA) | 178 GQNIGIYDER--GNIREDEFNHHLTDRL--PMDRIWEAPQKSQQVYFILKIGPDVNLAGNIPEEEITAEETIRRGLRGDTLGKVNL---------- |
| Xanthobacter (XecG)   | 187 GRGIGIYDKT--GELRSSFLEEIANLLGDKIDQLWEAPQKEQQAAVVARFCANSGLNVAVNEVLAEALRAGLRFETLAAVADREKASGQWDPD |
| B. subtilis (YitD)    | 175 GTG-GICSSS--GDVRFQIVDDIISSD-IDINRLIFEAPNKTLQQGFIQKIGPNLANIPFHDAILAETLRLGLRSDTFFL---------- |
| A. thaliana           | 195 GADTIMIDADDVCKYADSLRADIIAKVIGRL--GIEKTMEASDAKLVEWEIKRYCPNWEADTLVLISIIFNLFEWSPSQDSYLEVLV---------- |

**FIG. 6. Alignment of M. jannaschii ComA sequence with homologs from M. thermoautotrophicum, Xanthobacter sp., B. subtilis, and A. thaliana. Positions of identically conserved residues are in shown in white on black, and regions of similarly conserved residues are boxed.**

![FIG. 7. Phylogeny of ComA homologs inferred by a protein maximum likelihood method. Bootstrap percentages are indicated for branches supported by a plurality of bootstrap replicates. The scale bar represents 10 amino acid replacements/100 positions.](https://i.imgur.com/placeholder.png)

and now phosphosulfolactate synthase (ComA) from *M. jannaschii* both add sulfite to unsaturated carbon atoms to form stable sulfonates. A kinetic analysis of SQD1 found the apparent \( K_m \) for sulfite to be 10 μM, with a turnover number (\( k_{cat} \)) of 0.1 min\(^{-1}\) (12). By comparison, ComA has a higher apparent \( K_m \) (1 mM) and a much higher turnover number (510 min\(^{-1}\)). ComA is not homologous to SQD1, an NAD\(^+\)-dependent member of the short-chain dehydrogenase/reductase or tyrosine-dependent oxidoreductase family (43). Structural classification of ComA must await is crystal structure determination; however, the set of evolutionarily conserved residues in ComA homologs is inconsistent with its being a member of the short-chain dehydrogenase/reductase family.

In contrast to ComB, the CoM biosynthetic phosphosulfolate phosphate, ComA is stereospecific. ComB was previously shown to hydrolyze both enantiomers of pseudo-symmetric 2-hydroxycarboxylic acid monophosphate esters (18). Yet, ComC, the third enzyme in the CoM biosynthetic pathway (sulfolate dehydrogenase), specifically oxidizes only L-hydroxy acids (such as (R)-sulfolate) (19). Therefore, the stereospecificity of ComA is consistent with the rest of the characterized pathway. Considered independently from the rest of cellular metabolism, there is no compelling reason to prefer L-sulfolactate versus D-sulfolactate: enolase catalyzes a reaction analogous to ComA using D-2-phosphoglycerate, and D-lactate

dehydrogenases analogous to ComC are widespread. Perhaps D-phosphosulfolactate or D-sulfolactate inhibits key enzymes of central metabolism.

The kinetic analyses presented here demonstrate that high concentrations of all three substrates, especially P-enolpyruvate, can inhibit ComA. Although the inhibitory concentrations are much higher than those expected for substrates *in vivo*, this phenomenon may be useful in future efforts to elucidate the enzyme’s reaction mechanism. Such effects are consistent with an ordered substrate binding mechanism (31). However, we cannot rule out alternative explanations such as negative cooperativity and allosteric inhibition. Despite technical limitations of the sulfite assay, CoM was not observed to inhibit ComA activity. Therefore, feedback inhibition by CoM is not likely to regulate the coenzyme’s biosynthesis. Other intermediates in CoM biosynthesis ((R)-phosphosulfolactate, (R)-sulfolactate, and sulfopyruvate) are moderate inhibitors and may be relevant to regulation of the pathway.

Analogs of CoM have proved to be good inhibitors of methyl-CoM reductase *in vitro* (44) and anti-methanogenic agents *in vivo* (45). To find other inhibitors of methanogenesis and CoM biosynthesis, we screened substrate and product analogs as inhibitors of ComA PSL synthase activity. The most effective ComA inhibitors identified were the reaction product (2-phosphosulfolactate) and a phosphonate analog (2-phosphono-3-sulfopropionate). In the future, even more potent inhibitors could be designed using mechanistic and structural information about the ComA active site.

The catalytic mechanism of ComA likely proceeds through an enolate intermediate (46). Similar to enolase, ComA requires P-enolpyruvate and Mg\(^{2+}\), has an essential lysine residue, and can exchange the C-2 proton of a 2-hydroxycarboxylic acid phosphate ester. Further investigations are required to elucidate the structural and mechanistic relationship of ComA to other P-enolpyruvate-utilizing enzymes in the enolase superfamily (47).

Many of the organisms that have homologs of *comA* also have homologs of other CoM biosynthetic genes. *B. subtilis* has a gene cluster (*yisZ-yitABCD*) that encodes an adenylylsulfate kinase, a sulfate adenyltransferase, a sulfite-producing 3′-phosphoadenosine-5′-phosphosulfate sulfotransferase, a phosphosulfate phosphatase (ComB), and a phosphosulfate synthase (ComA), respectively. These enzymes are probably used to synthesize sulfolate from sulfate and P-enolpyruvate during *B. subtilis* sporulation (14, 48). This organism may also produce sulfolate from cysteic acid (49). The *C. burnetii* genome contains two unlinked gene clusters (*yit-BDC* and *yitA-yisZ*), with the genes in different orientations from *B. subtilis*. *C. hutchinsonii* has homologs of *comABC*, although the genes are unlinked on its chromosome. A related species, *Cytophaga johnsonae*, produces N-acylcapnine sulfolipids. Carbon atoms in the capnine head group are derived

from L-cysteate, but not from L-cysteine (50, 51). Rather, Com-
ABC proteins in Cytophaga spp. could produce sulfopyruvate,
which is then transaminated to form cysteate. Another species,
*F. acidarmanus*, a euryarchaeon not known to have CoM, has
a *yitB-comA* cluster and an unlinked *comB* homolog, which
could be used to produce sulfolactate. *A. fumigatus* and white
rot fungus could have acquired the plant-type *comA* homolog by
horizontal gene transfer.

Sequence similarity alone does not link any of the CoM
biosynthesis genes to universally conserved genes in central
metabolic pathways. Yet, mechanistic similarities and con-
served structural features join these proteins to large super-
families of enzymes. Just as catabolic pathways have evolved
by stringing together familiar reactions to consume unfamiliar
compounds, so have biosynthetic pathways drawn upon a sig-
nificant unexplored and ancient gene pool (52).

Acknowledgments—We thank Kim Harich for assistance with
GC-MS and Tom Glass for NMR analysis.

**REFERENCES**

1. Neue, H. (1993) *Bioscience* **43**, 466–473
2. McBride, B. C., and Wolfe, R. S. (1971) *Biochemistry* **10**, 2317–2324
3. Thauer, R. K. (1998) *Microbiology (Read.)* **144**, 2377–2406
4. Allen, J. R., Clark, D. D., Krum, J. G., and Ensign, S. A. (1999) *Proc. Natl.*
   *Acad. Sci. U. S. A.* **96**, 8432–8437
5. Grabarse, W., Mahlert, F., Duin, E. C., Goubeaud, M., Shima, S., Thauer,
   R. K., Lamzin, V., and Ermler, U. (2001) *J. Mol. Biol.* **309**, 315–330
6. Shaw, I. C., and Graham, M. I. (1987) *Cancer Treat. Rev.* **14**, 67–86
7. Cook, A. M., Laue, H., and Junker, F. (1999) *FEMS Microbiol. Rev.* **22**,
   399–419
8. Huxtable, R. J. (2000) *Adv. Exp. Biol.* **483**, 1–25
9. Morton, M., and Landfield, H. (1952) *J. Am. Chem. Soc.* **74**, 3523–3526
10. Gilbert, E. E. (1977) *Sulfonation and Related Reactions*, Robert E. Krieger
    Publishing Co., Huntington, NY
11. Benning, C. (1998) *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **49**, 53–75
12. Sanda, S., Leustek, T., Theisen, M. J., Garavito, R. M., and Benning, C. (2001)
    *J. Biol. Chem.* **276**, 3941–3946
13. Pugh, C. E., Roy, A. B., Hawkes, T., and Harwood, J. L. (1995) *Biochem. J.*
    **309**, 513–519
14. Bonsen, P. P., Spudich, J. A., Nelson, D. L., and Kornberg, A. (1969) *J.*
    *Bacteriol.* **91**, 62–68
15. Graham, D. E., and White, R. H. (2002) *Nat. Prod. Rep.* **19**, in press
16. White, R. H. (1985) *Biochemistry* **24**, 6487–6493
17. White, R. H. (1986) *Biochemistry* **25**, 5304–5308
18. Graham, D. E., Graupner, M., Xu, H., and White, R. H. (2001) *Eur. J. Biochem.*
    **268**, 5176–5188
19. Graupner, M., Xu, H., and White, R. H. (2000) *J. Bacteriol.* **182**, 3688–3692
20. Graupner, M., Xu, H., and White, R. H. (2000) *J. Bacteriol.* **182**, 4862–4867
21. Anderson, V. E. (1998) in *Comprehensive Biological Catalysis* (Sinnott, M., ed)
    Vol. 2, pp. 115–133, Academic Press, New York
22. Anderson, V. E., Weiss, P. M., and Cleland, W. W. (1984) *Biochemistry* **23**,
    2779–2786
23. McFadden, H. G., Harris, R. L., and Jenkins, C. L. D. (1989) *Australian*
    *J. Chem.* **42**, 301–314
24. White, R. H. (1987) *J. Labelled Cmpds. Radiopharm.* **24**, 323–330
25. Izumi, Y., Yajima, S., Okubo, K., and Babievskii, K. K. (1971) *Bull. Chem. Soc.*
    *Jpn.* **44**, 1416–1417
26. Tabor, S., and Richardson, C. C. (1985) *Proc. Natl. Acad. Sci. U. S. A.* **82**,
    1074–1078
27. Westhead, E. W. (1966) *Methods Enzymol.* **9**, 670–679
28. Togawa, T., Ogawa, M., Nawata, M., Ogasawara, Y., Kawanabe, K., and
    Tanabe, S. (1992) *Chem. Pharm. Bull. (Tokyo)* **40**, 3000–3004
29. Ellis, K. J., and Morrison, J. F. (1982) *Methods Enzymol.* **87**, 405–427
30. Poyner, R. R., Cleland, W. W., and Reed, G. H. (2001) *Biochemistry* **40**,
    8009–8017
31. Cleland, W. W. (1979) *Methods Enzymol.* **63**, 500–513
32. Altschul, S. F., Madden, T. L., Schaffer, A. A., Zhang, J., Zhang, Z., Miller, W.,
    and Lipman, D. J. (1997) *Nucleic Acids Res.* **25**, 3389–3402
33. Thompson, J. D., Higgins, D. G., and Gibson, T. J. (1994) *Nucleic Acids Res.*
    **22**, 4673–4680
34. Bult, C. J., White, O., Olsen, G. J., Zhou, L., Fleischmann, R. D., Sutton, G. G.,
    Blake, J. A., FitzGerald, L. M., Clayton, R. A., Gocayne, J. D., Kerlavage,
    A. R., Dougherty, B. A., Tomb, J.-F., Adams, M. D., Reich, C. I., Overbeek,
    R., Kirkness, E. F., Weinstock, K. G., Merrick, J. M., Glodek, A., Scott, J. L.,
    Geoghagen, N. S. M., Smith, H. O., Woese, C. R., and Venter, J. C. (1996)
    *Science* **273**, 1017–1140
35. Krum, J. G., and Ensign, S. A. (2001) *J. Bacteriol.* **183**, 2172–2177
36. Fahey, R. C., and Newton, G. L. (1987) *Methods Enzymol.* **143**, 85–96
37. Lehmann, J., and Benson, A. A. (1964) *J. Am. Chem. Soc.* **86**, 4469–4472
38. Morrison, J. F. (1979) *Methods Enzymol.* **63**, 257–294
39. Gerlt, J. A. (1998) in *Bioorganic Chemistry: Peptides and Proteins* (Hecht, S.,
    ed) pp. 279–311, Oxford University Press, New York
40. Tuazon, P. T., and Johnson, S. L. (1977) *Biochemistry* **16**, 1183–1188
41. Southerland, W. M., Akogyeram, C. O., Toghrol, F., Sloan, L., and Scherrer, R.
    (1982) *J. Toxicol. Environ. Health* **10**, 479–491
42. Wedzicha, B. L. (1992) *Food Addit. Contam.* **9**, 449–459
43. Jornvall, H., Persson, B., Krook, M., Atrian, S., Gonzalez-Duarte, R., Jeffery,
    J., and Ghosh, D. (1995) *Biochemistry* **34**, 6003–6013
44. Gunsalus, R. P., Romesser, J. A., and Wolfe, R. S. (1978) *Biochemistry* **17**,
    2374–2377
45. Balch, W. E., and Wolfe, R. S. (1979) *J. Bacteriol.* **137**, 256–263
46. Cleland, W. W. (1998) in *Comprehensive Biological Catalysis* (Sinnott, M., ed)
    Vol. 2, pp. 1–6, Academic Press, New York
47. Babbitt, P. C., Hasson, M. S., Wedekind, J. E., Palmer, D. R., Barrett, W. C.,
    Reed, G. H., Rayment, I., Ringe, D., Kenyon, G. L., and Gerlt, J. A. (1996)
    *Biochemistry* **35**, 16489–16501
48. Wood, D. A. (1971) *Biochem. J.* **123**, 601–605
49. Koshikawa, T., Nakashio, S., Kusuyama, K., Ichikawa, T., and Kondo, M.
    (1981) *J. Gen. Microbiol.* **124**, 415–423
50. Gilmore, D. F., Godchaux, W., III, and Leadbetter, E. R. (1989) *Biochem.*
    *Biophys. Res. Commun.* **160**, 535–539
51. White, R. H. (1984) *J. Bacteriol.* **159**, 42–46
52. Teichmann, S. A., Rison, S. C., Thornton, J. M., Riley, M., Gough, J., and
    Chothia, C. (2001) *J. Mol. Biol.* **311**, 693–708
53. Felsenstein, J. (2001) *PHYLIP*, Version 3.6a2.1, University of Washington,
    Seattle
