
Molecular Mechanisms of Inherited Cardiomyopathies

DIANE FATKIN AND ROBERT M. GRAHAM

*Molecular Cardiology Unit, Victor Chang Cardiac Research Institute, Sydney, New South Wales, Australia*

---

I. Introduction                                                                                       945
II. Cardiomyocyte Cellular Physiology                                                                946
   A. Cardiac myocyte structure                                                                      946
   B. Mechanisms of contraction                                                                      948
III. Hypertrophic Cardiomyopathy                                                                     949
   A. Clinical manifestations                                                                        949
   B. Chromosomal loci and disease genes                                                             951
   C. Functional consequences of gene mutations                                                     953
   D. Triggers and effectors of left ventricular hypertrophy                                         958
IV. Dilated Cardiomyopathy                                                                           961
   A. Clinical manifestations                                                                        961
   B. Chromosomal loci and disease genes                                                             962
   C. Pathophysiological mechanisms                                                                  968
   D. Pathophysiology of HCM and DCM: is there a unifying hypothesis?                                970
V. Arrhythmogenic Right Ventricular Dysplasia                                                       971
   A. Clinical manifestations                                                                        971
   B. Chromosomal loci and disease genes                                                             972
VI. Restrictive Cardiomyopathy                                                                       973
   A. Clinical manifestations                                                                        973
   B. Chromosomal loci and disease genes                                                             974
VII. Conclusion                                                                                      974

---

Fatkin, Diane, and Robert M. Graham. Molecular Mechanisms of Inherited Cardiomyopathies. *Physiol Rev* 82: 945–980, 2002; 10.1152/physrev.00012.2002.—Cardiomyopathies are diseases of heart muscle that may result from a diverse array of conditions that damage the heart and other organs and impair myocardial function, including infection, ischemia, and toxins. However, they may also occur as primary diseases restricted to striated muscle. Over the past decade, the importance of inherited gene defects in the pathogenesis of primary cardiomyopathies has been recognized, with mutations in some 18 genes having been identified as causing hypertrophic cardiomyopathy (HCM) and/or dilated cardiomyopathy (DCM). Defining the role of these genes in cardiac function and the mechanisms by which mutations in these genes lead to hypertrophy, dilation, and contractile failure are major goals of ongoing research. Pathophysiological mechanisms that have been implicated in HCM and DCM include the following: defective force generation, due to mutations in sarcomeric protein genes; defective force transmission, due to mutations in cytoskeletal protein genes; myocardial energy deficits, due to mutations in ATP regulatory protein genes; and abnormal Ca²⁺ homeostasis, due to altered availability of Ca²⁺ and altered myofibrillar Ca²⁺ sensitivity. Improved understanding that will result from these studies should ultimately lead to new approaches for the diagnosis, prognostic stratification, and treatment of patients with heart failure.

---

I. INTRODUCTION

Cardiomyopathies are diseases of heart muscle that are associated with cardiac dysfunction. Heart failure due to cardiomyopathy represents a major health problem in our society. Although increased emphasis on prevention programs and new therapeutic agents have resulted in improved survival from acute coronary artery disease,

there has been a dramatic increase in morbidity and mortality from heart failure, which has been described as the emerging health epidemic of the 21st century. Cardiomyopathies are classified traditionally according to morphological and functional criteria into four categories: hypertrophic cardiomyopathy, dilated cardiomyopathy, arrhythmogenic right ventricular dysplasia, and restrictive cardiomyopathy (135). These cardiomyop-

athies can be primary myocardial disorders or develop as a secondary consequence of a variety of conditions, including myocardial ischemia, inflammation, infection, increased myocardial pressure or volume load, and toxic agents. The pathogenesis of primary cardiomyopathies has been poorly understood. Over the last decade, the importance of gene defects in the etiology of primary cardiomyopathies has been recognized. Autosomal dominant, autosomal recessive, X-linked, and maternal patterns of inheritance have been observed. Families with inherited cardiomyopathies have provided a unique resource for studies of the genetic basis of these disorders. Molecular genetic studies performed to date have focused largely on monogenic inherited cardiomyopathies, i.e., caused by mutations in a single gene. Although relatively uncommon, these monogenic disorders enable pathological processes applicable to a wide range of more commonly occurring heart diseases to be evaluated. The usefulness of studying monogenic disorders was first demonstrated by Brown and Goldstein (19) in their pioneering work on the role of the low-density lipoprotein receptor in atherosclerosis. In 1989, Christine and Jon Seidman and co-workers (63) reported the first association between an inherited gene defect and a primary cardiomyopathy. In a subsequent study, they showed that mutations in the β-myosin heavy chain gene were responsible for familial hypertrophic cardiomyopathy (40). Since then, substantial progress has been made in elucidating further gene defects in hypertrophic cardiomyopathy. More recently, disease-causing genes have been identified in dilated cardiomyopathy, arrhythmogenic right ventricular dysplasia, and restrictive cardiomyopathy. Although these genetic studies have enabled molecular triggers of cardiomyopathies to be identified, the functional consequences of gene mutations and precise details of the signaling pathways that lead to hypertrophy, dilation, and contractile failure remain to be elucidated. Current concepts of the molecular genetic basis of inherited cardiomyopathies are summarized in this review.

intercalated discs. The intercalated discs are comprised of adherens junctions (containing N-cadherin, catenins, and vinculin), desmosomes (containing desmin, desmoplakin, desmocollin, desmoglein), and gap junctions (containing connexins).

2. Sarcomeres and the sarcomeric cytoskeleton

Cardiac myocytes are surrounded by a thin membrane, the sarcolemma. The interior of each myocyte contains bundles of longitudinally arranged myofibrils that have a characteristic striated appearance, similar to that of skeletal muscle, formed by repeating sarcomeres. The sarcomere is the fundamental structural and functional unit of cardiac muscle that is comprised of interdigitating thick and thin filaments (Fig. 1). The thick filaments are composed predominantly of myosin and also contain the myosin binding proteins C, H, and X. The thin filaments are composed of cardiac actin, α-tropomyosin, and troponins C, I, and T. Each sarcomere contains an A band (comprised of overlapping thick filaments and thin filaments), with a central M line (comprised of thick filaments only). At each side of the A band are I bands (comprised of thin filaments only) bounded by Z lines.

The sarcomeric cytoskeleton, comprised of titin and the myomesins, provides a scaffolding for the thick and thin filaments. Titin is a giant protein that spans from the Z line to the M line. As well as contributing to sarcomere assembly and organization, titin is a major determinant of the elastic properties of the cardiac myofibril. Myomesins 1 and 2 are titin-associated proteins. The Z disc (at the Z line) is formed by a lattice of interdigitating proteins that maintain myofilament organization by cross-linking antiparallel titin and thin filaments from adjacent sarcomeres. The Z-disc proteins include α-actinin, filamin, nebulette, telethonin, and myotilin.

3. Extrasarcomeric cytoskeleton

The extrasarcomeric cytoskeleton is a complex network of proteins linking the sarcomere with the sarcolemma and the extracellular matrix. It provides structural support for subcellular structures and transmits mechanical and chemical signals within and between cells. The extrasarcomeric cytoskeleton has intermyofibrillar and subsarcolemmal components (Fig. 1).

The intermyofibrillar cytoskeleton is comprised of intermediate filaments, microfilaments, and microtubules. Desmin intermediate filaments form a three-dimensional scaffold throughout the extrasarcomeric cytoskeleton. Desmin filaments surround the Z discs and form longitudinal connections to adjacent Z discs and lateral connections to subsarcolemmal costameres. Microfilaments composed of nonsarcomeric actin (mainly γ-actin) also form a complex network linking the sarcomere (via α-actinin) to various components of the costameres. Tubulin

INHERITED CARDIOMYOPATHIES

EXTRACELLULAR MATRIX

SARCOLEMMA
- Integrins
- Sarcoglycans
  - Spectrin
  - Dystroglycans
    - α
    - δ, γ, α, β, β
- Costameres
- Desmin
- Cytoplasmic Actin
- Dystrophin
- T-tubule
- DHPR
- RyR
- Ca²⁺
- SERCA2a
- Phospholamban
- Na⁺/Ca²⁺ Exchanger

MITOCHONDRIA
- ATP

NUCLEUS
- Nuclear Lamina

Z disc
- I band
- A band
- I band
- Z line
- M line
- Z line

SARCOMERE

FIG. 1. Schematic of the components of cardiac myocyte structure. The sarcomere, comprised of interdigitating thick and thin filaments, is the fundamental structural and functional unit of cardiac muscle. Sarcomeres are linked by a complex network of cytoskeletal proteins with the sarcolemma, extracellular matrix, and nucleus. The intermyofibrillar cytoskeleton is comprised of desmin intermediate filaments, actin-containing microfilaments, and microtubules. The subsarcolemmal cytoskeleton is comprised of costameres, sites of interconnection between the various pathways linking sarcomeres to sarcolemmal transmembrane proteins. Muscle contraction is an energy-requiring process that utilizes ATP supplied by mitochondria. The intracellular free Ca²⁺ concentration ([Ca²⁺]i) critically regulates muscle contraction and relaxation. Muscle contraction is initiated by an increase in [Ca²⁺]i. This results from depolarization-induced influx of a small amount of Ca²⁺ via voltage-gated dihydropyridine receptors (DHPR) in t tubules, which in turn triggers the release of a large quantum of Ca²⁺ from the sarcoplasmic reticulum (SR), a process called Ca²⁺-induced Ca²⁺ release. Muscle relaxation is initiated by the uptake of cytosolic Ca²⁺ into the SR via the high energy-dependent pump, the SR Ca²⁺-ATPase (SERCA2a) and by extrusion through the sarcolemmal Na⁺/Ca²⁺ exchanger. Pathophysiological mechanisms that have been implicated in hypertrophic and dilated cardiomyopathies include the following: defective force generation, due to mutations in sarcomeric protein genes; defective force transmission, due to mutations in cytoskeletal protein genes; myocardial energy deficits, due to mutations in ATP regulatory protein genes; and abnormal Ca²⁺ homeostasis, due to altered availability of Ca²⁺ and altered myofibrillar Ca²⁺ sensitivity. RyR, ryanodine receptor.

Physiol Rev • VOL 82 • OCTOBER 2002 • www.prv.org

is a cytosolic protein that is present in a polymerized form (as microtubules) and in an unpolymerized form. Changes in the total amount of tubulin and the proportion of the polymerized form are thought to influence cytoskeletal stiffness and hence contractile function.

Costameres are subsarcolemmal domains that are located in a periodic, gridlike pattern, flanking the level of the Z lines and overlying the I bands, along the cytoplasmic side of the sarcolemma. The costameres are sites of interconnection between various cytoskeletal networks linking the sarcomere and the sarcolemma. They are thought to function as anchor sites for stabilization of the sarcolemma and for integration of pathways involved in mechanical force transduction. Costameres contain three principal components: the focal adhesion-type complex, the spectrin-based complex, and the dystrophin/dystrophin-associated glycoprotein complex. The focal adhesion-type complex is comprised of cytoplasmic proteins, including vinculin, talin, tensin, paxillin, and zyxin, that connect with cytoskeletal actin filaments and with the transmembrane proteins α- and β-integrin. The extracellular domains of the integrins interact with collagens, laminin, and fibronectin in the extracellular matrix. The spectrin-based complex contains ankyrin, that cross-links actin and spectrin, and spectrin, a component of the sarcolemma that interacts with actin, ankyrin, and desmin. The dystrophin/dystrophin-associated glycoprotein complex is made up of the cytoskeletal protein dystrophin, which binds to actin filaments, and the dystrophin-associated glycoprotein complex (α- and β-dystroglycans; α-, β-, γ-, δ-sarcoglycans; dystrobrevin; and syntrophin). The dystrophin-associated glycoprotein complex has cytoplasmic, transmembrane, and extracellular components. The cytoplasmic domain binds to the COOH-terminal region of dystrophin; the extracellular domain binds to laminin. Several actin-associated proteins are located at sites of attachment of cytoskeletal actin filaments with costameric complexes, including α-actinin and the muscle LIM protein MLP.

The rod contains an elastic element and also binds myosin light chains (MLC). The “swinging cross-bridge” model was a subsequent modification of the sliding filament model, which hypothesized that a swinging motion of the myosin head was a critical factor for progression along the thin filament (60, 84). In the first step of this model, myosin is bound strongly to actin, with the myosin head orientated in a 45° position relative to its tail. ATP binding at the ATPase site causes rapid dissociation of myosin from actin and the formation of ADP and P<sub>i</sub>. Actin combines weakly with this myosin-ADP-P<sub>i</sub> complex, with the myosin head orientated at a 90° angle. Release of ADP and P<sub>i</sub> enables the strong actin binding position to be regained. The power stroke is achieved in this last step by a rowing-like movement of the myosin head as it “walks” down the actin filament. Each cross-bridge cycle is associated with hydrolysis of one ATP molecule. The “lever-arm” hypothesis was a subsequent modification of the swinging cross-bridge model, in which structural changes in the catalytic domain of the myosin head are amplified by rotation of the myosin tail, acting as a lever arm (57, 59, 131). In this model, the power stroke is produced not by movement of the myosin head (at the point of actin attachment) but by the pivoting movement of the myosin tail (at the hinge region). The lever arm rotation has been attributed to the transition between two conformations of the myosin head, open and closed, that influence nucleotide affinity and hydrolysis and that are determined by actin binding. It has subsequently been proposed that altered orientation of both the myosin head and the α-helical tail may contribute to the overall displacement of actin. Conflicting data have been reported for the extent of actin displacement, with estimates ranging from 4–5 nm to 15–20 nm (59, 178). The number of attached states may be an important determinant of actin displacement. It is possible that a single myosin head may attach at two points on an actin molecule or that one or two myosin heads may be attached at any time point. Differences in compliance and load in the various model systems studied may also contribute to this range of data.

B. Mechanisms of Contraction

1. Actin-myosin interaction

Currently, the universally accepted concept for the process of muscle contraction is the “sliding filament” theory, proposed by Huxley in 1957 (58). In this model, force generation is achieved by the sliding movement of the thick filaments relative to thin filaments, which is mediated by the cyclical attachment and detachment of myosin “cross-bridges” to actin. These cross-bridges consist of myosin heavy chain (MHC) molecules that project from the thick filament. Muscle myosin contains two MHC. Each MHC contains a head, with an actin binding site and ATPase site. The heads are linked at a “hinge” region to a long rod; that is an α-helical coiled-coil dimer.

2. Regulation by the troponin-tropomyosin complex

The troponin-tropomyosin complex is a Ca<sup>2+</sup>-sensitive switch that regulates actin-myosin interaction. The backbone of the thin filament is formed by a double helical array of globular actin molecules. Tropomyosin proteins assemble as α-helical coiled-coil dimers that lie in a head-to-tail orientation within the major groove of the actin filaments, spanning seven actin monomers. The troponin complex (troponins T, I, and C) is anchored to tropomyosin predominantly by troponin T, and to a lesser extent, by troponin I. Troponin C interacts with both troponins T and I. During diastole, actin-myosin interac-

tion is inhibited by the binding of troponin I to actin-tropomyosin. Ca²⁺ binding to troponin C induces a conformational change that weakens the interaction between troponin I and actin-tropomyosin and strengthens the interaction between troponin I and troponin C. These changes release the thin filament from its inhibitory state, promoting actin-myosin interaction and force generation. A reduction in the intracellular Ca²⁺ concentration ([Ca²⁺]i) causes dissociation of Ca²⁺ from troponin C and restores the relaxed state (148).

### 3. Role of calcium

The importance of Ca²⁺ in regulation of contraction in normal myocytes and in cardiomyopathies has increasingly been recognized (Fig. 1). During the cardiac action potential, [Ca²⁺]i is increased initially by direct Ca²⁺ entry into cells, via voltage-gated L-type channels and to a lesser extent via Na⁺/Ca²⁺ exchange. This influx of Ca²⁺ triggers Ca²⁺ release from stores in the sarcoplasmic reticulum via ryanodine receptors and inositol 1,4,5-trisphosphate receptors. Increases in [Ca²⁺]i promote Ca²⁺ binding to multiple cytosolic buffers, including troponin C. The magnitude of the systolic rise in [Ca²⁺]i is determined not only by Ca²⁺ entry from the extracellular fluid and the sarcoplasmic reticulum, but also by the buffering capacity of the cell. During relaxation, Ca²⁺ is removed from the cytosol by the sarcoplasmic reticulum Ca²⁺-ATPase (SERCA2a), sarcolemmal Na⁺/Ca²⁺ exchange, sarcolemmal Ca²⁺-ATPase, and mitochondrial Ca²⁺ uniporter. A complex network of positive and negative feedback mechanisms maintains the [Ca²⁺]i (8). The Ca²⁺ dependence of force is generally expressed as a function of the free [Ca²⁺]i. This relationship does not directly indicate the total amount of Ca²⁺ required to activate myofilaments, since changes in the sensitivity to Ca²⁺ may be present. For example, myofilament Ca²⁺ sensitivity is decreased by protein kinase A phosphorylation of troponin I, low pH, and reduced sarcomere length (8). Ca²⁺ activation may not just exert an “on-off” action on actin-myosin interaction. It has been proposed that the thin filament exists in three activation states. In the “blocked” (off) state, Ca²⁺ is not bound to troponin C, and cross-bridge binding is unable to occur; in the “closed” (off) state, Ca²⁺ is bound to troponin C but there are no strong cross-bridges; in the “open” (on) state, Ca²⁺ is bound to troponin C and cross-bridges are strongly bound. According to the three-state model, Ca²⁺ may determine both whether or not cross-bridges bind and the strength of binding. It has also been proposed that changes in Ca²⁺ activation may alter the number of cross-bridges attached, as well as the kinetics of cross-bridge attachment and detachment (97, 148, 171).

#### 4. Other regulatory factors

MLC and cardiac myosin binding protein C (cMyBP-C) have been implicated as regulatory factors in muscle contraction. Essential and regulatory MLC bind to the α-helical lever arm of the myosin cross-bridge and influence force production by modulating cross-bridge kinetics (125, 140). cMyBP-C is thought to participate in the adrenergic regulation of cardiac contractility (38). The MyBP-C motif, a highly conserved 100-residue region in the NH₂-terminal region of cMyBP-C, binds to the S2 segment near the lever arm of the myosin head (48). Ultrastructural studies have shown that phosphorylation of the MyBP-C motif by cAMP-dependent protein kinase extends myosin cross-bridges from the backbone of the thick filament and changes their orientation (176). Functional studies indicate that the phosphorylation status of cMyBP-C is a determinant of the stiffness and attachment rates of cross-bridges and Ca²⁺ sensitivity of force production (74, 176). In addition to these intrinsic sarcomere components, numerous extrinsic factors, including neurohumoral, endocrine, and hemodynamic factors, influence cardiac contractility in vivo.

### III. HYPERTROPHIC CARDIOMYOPATHY

#### A. Clinical Manifestations

##### 1. Disease characteristics

Hypertrophic cardiomyopathy (HCM) is a primary myocardial disorder with an autosomal dominant pattern of inheritance that is characterized by hypertrophy of the left (±right) ventricles with histological features of myocyte hypertrophy, myofibrillar disarray, and interstitial fibrosis. HCM is one of the most common inherited cardiac disorders, with a prevalence in young adults of 1 in 500 (91). Various names have been given to this disorder including hypertrophic obstructive cardiomyopathy and idiopathic subaortic stenosis. These names reflect “textbook” features of asymmetric septal hypertrophy and left ventricular outflow tract obstruction. This description of the disease is based primarily on patients with severe symptoms seen in tertiary hospital referral centers. Epidemiological studies now suggest that a wide spectrum of clinical manifestations of varying severity and prognosis is present in community populations. The generic name HCM is thus more appropriate and is used in this review. The first clinical description of HCM was reported in 1869 (51). HCM was recognized to be a genetic disorder in the late 1950s. Since then, numerous clinical and pathological studies of HCM have been performed. During the last decade, molecular genetic studies have given important insights into the pathogenesis of HCM and provide a new

perspective for the diagnosis and management of patients with this disorder.

2. Diagnosis of HCM

Affected individuals with HCM exhibit significant variability in their clinical presentation. Genotype-positive individuals may be asymptomatic or present with symptoms ranging from palpitations and dizziness to syncope and sudden death. Genotype-phenotype studies have shown that the age of onset of symptoms varies between different HCM disease genes. For example, individuals with β-MHC mutations typically present in the first two decades of life, whereas those with cMyBP-C mutations may be asymptomatic until the fifth or sixth decades (116).

The primary modality for the diagnosis of HCM is transthoracic echocardiography. The hallmark diagnostic feature of HCM is asymmetric hypertrophy of the interventricular septum, with or without left ventricular outflow tract obstruction and systolic anterior motion of the mitral valve. It is now recognized that the obstructive form of HCM occurs in <25% of affected individuals. Studies of kindreds with HCM have shown that the distribution and severity of left ventricular hypertrophy may vary considerably and that asymmetric hypertrophy is no longer an essential requirement for the diagnosis of this disorder. The diagnosis of HCM generally requires exclusion of secondary causes of hypertrophy, such as hypertension or aortic stenosis; however, in some individuals, particularly in older age groups, these conditions may coexist. The differentiation of HCM from physiological left ventricular hypertrophy may be difficult, particularly in competitive athletes. The extent of left ventricular hypertrophy varies between different genes. For example, individuals with β-MHC gene mutations usually develop moderate or severe hypertrophy with a high disease penetrance, whereas those with cardiac troponin T gene mutations generally have only mild or clinically undetectable hypertrophy (106, 173). Unusual forms of hypertrophy have been reported, localized to the left ventricular apex (cardiac troponin I mutations) (71) or midcavity (cardiac actin and MLC gene mutations) (119, 125). The extent of left ventricular hypertrophy may also vary between members of a single family with the same gene mutation. These observations may be explained by a modifying role of additional genetic and environmental factors, such as blood pressure, exercise, diet, and body mass.

3. Natural history of HCM

The natural history of HCM is variable; some individuals remain asymptomatic throughout life, and others may develop progressive symptoms with or without heart failure or experience sudden death. Longitudinal echocardiographic studies have documented left ventricular remodeling with age. Progressive increases in left ventricular wall thickness have been reported in ungenotyped individuals during adolescence and early adult life. In some individuals, including those with cMyBP-C mutations, left ventricular wall thickness may increase in later life (116). Age-related reductions in left ventricular wall thickness, associated with myocyte loss and fibrosis, have also been described in individuals with long-standing disease ("burnt out" HCM). Ten to 20% of individuals with HCM may develop dilated cardiomyopathy. Ten to 16% of affected individuals develop atrial fibrillation. The risk of atrial fibrillation is increased in those with left atrial enlargement.

HCM is a frequent cause of sudden death, particularly in young individuals and competitive athletes. Estimates of the prevalence of sudden death vary according to the population studied, ranging from <1% in the general community to 3–6% in tertiary hospital referral centers. Various mechanisms for sudden death have been proposed, including ventricular bradyarrhythmias due to sinus node and atrioventricular conduction abnormalities and tachyarrhythmias triggered by reentrant depolarization pathways related to myofibrillar disarray and fibrosis, abnormal Ca²⁺ homeostasis, myocardial ischemia, left ventricular diastolic dysfunction, or left ventricular outflow tract obstruction. Various risk stratification algorithms based on clinical parameters have been proposed to identify individuals with an increased propensity for sudden death. Given the complexity of mechanisms that may precipitate sudden death, it is not surprising that no single risk factor has been identified. Conflicting results have been found for the positive predictive value of young age at diagnosis, history of syncope, severity of symptoms, left ventricular wall thickness, left ventricular outflow tract gradient, left atrial size, and atrial fibrillation. The majority of clinical risk factor studies have been performed in ungenotyped populations. In genotyped individuals, it has been demonstrated that prognosis varies considerably between different HCM genes and between different mutations in the same gene. For example, the Arg403Gln and Arg453Cys β-MHC mutations, cardiac troponin T mutations, and some α-tropomyosin mutations (Ala63Val, Lys70Thr) are "high risk" mutations with reduced life expectancy and high rates of sudden death, whereas the Val606Met β-MHC mutation and cMyBP-C mutations have a relatively benign course (106, 116, 173, 174). The mechanisms whereby HCM gene mutations influence prognosis are unknown. Although some HCM mutations that alter the charge of the encoded amino acid have been associated with a poor outcome, other mutations that alter charge have a good prognosis (170, 174). Electrophysiological studies in mouse models may provide important insights into the differential propensity for sudden death between different HCM gene mutations.

INHERITED CARDIOMYOPATHIES 951

B. Chromosomal Loci and Disease Genes

### 1. Chromosomal loci

HCM is a genetically heterogeneous disorder, with 12 distinct chromosomal loci mapped to date. Disease-causing genes have now been identified in all 12 loci: the first 10 of these genes encoded protein components of the cardiac sarcomere. Mutations have been found in four genes that encode components of the thick filament: β-MHC (40), essential MLC (125), regulatory MLC (125), and cMyBP-C (14, 172); in five genes that encode thin filament proteins: cardiac actin (119), cardiac troponin T (165), cardiac troponin I (71), cardiac troponin C (56), and α-tropomyosin (165); and in the sarcomeric cytoskeletal protein titin (143) (Table 1). Recently, mutations in two genes encoding nonsarcomeric proteins have been reported to cause HCM. Mutations in the γ₂-regulatory subunit of an AMP-activated protein kinase (AMPK) were found to be responsible for a variant of HCM associated with ventricular preexcitation (Wolff-Parkinson-White syndrome) at the chromosome 7q36 locus (11). Mutations in the gene encoding the cytoskeletal muscle LIM protein have also been identified (39).

For each disease gene, a variety of different mutations have been reported. Single nucleotide substitutions (“missense” mutations) and deletion or insertion of nucleotides have been identified. In some cases, the encoded protein is of normal size. In other cases, the mutation may result in a premature termination codon or cause a shift of the reading frame with truncation of the encoded protein. Mutations located at intron-exon boundaries can result in abnormal splicing. In general, each affected family has a unique mutation, although several mutation “hot spots” have been found. Methylated CpG dinucleotides within the genome are particularly prone to point mutations (26).

Mutations in the 12 known disease genes account for ~50–70% of all cases of HCM. It is possible that families in which mutations have not been found have undiscovered abnormalities in one of the known disease-causing genes. Alternatively, a significant number of novel genes may remain to be identified. Generally, individuals with HCM-causing mutations are heterozygous at the disease locus, i.e., one copy (allele) of the gene is mutated and the other allele has the normal DNA sequence. Two cases of homozygous HCM mutations have recently been reported; one of these was an Arg869Gly point mutation in the β-MHC gene (55), and the other was a Ser179Phe mutation in the cardiac troponin T gene (136). Both mutations caused a particularly severe phenotype with onset in childhood and premature death.

#### 2. β-Myosin heavy chain gene (MYH7)

Myosin exists as a hexameric protein with two heavy chains and two sets of light chains. Cardiac MHC exists as two isoforms: β- and α-MHC. In humans, β-MHC is present in the embryonic heart and the adult atrium and is the predominant isoform expressed in the adult ventricle. β-MHC is also expressed in slow skeletal muscles, such as the soleus. In rodents, β-MHC is the principal isoform expressed in the embryonic ventricle with a switch to predominance of α-MHC in the adult ventricle. The MYH7 gene (encoding β-MHC) and the MYH6 gene (encoding α-MHC) are located in tandem on chromosome 14, ~4 kb apart. MYH7 is comprised of 23 kb of genomic DNA, with 41 exons, 38 of which encode a protein of 1,935 amino acids. Seventy-three mutations in the MYH7 gene have been reported, accounting for ~30–35% of cases of HCM (32, 40). The majority of mutations are missense mutations. Deletions and premature termination codons have also been identified. MYH7 mutations appear to be distributed throughout the gene-coding sequence. Three-dimensional structural studies of the MHC molecule have demonstrated, however, that these mutations cluster predominantly in four domains in the MHC head: 1) the actin binding surface, 2) the nucleotide binding pocket, 3) adjacent to two reactive cysteines in the hinge region, and 4) in the α-helical tail near the essential MLC binding site (130). A small number of mutations occur in the MHC rod. Although the functional consequences have not been defined precisely, it has been proposed that rod mutations may result in altered transmission of force from the head to the body of the thick filament (170).

#### 3. cMyBP-C gene (MYPBC3)

Myosin binding protein C has three isoforms: slow skeletal, fast skeletal, and cardiac. The cardiac MyBP-C isoform is expressed exclusively in cardiac tissue. It is located in the sarcomere A bands and forms a series of seven to nine transverse bands spaced at 43-nm intervals.

---

**TABLE 1. HCM chromosomal loci and disease genes**

| Chromosomal Locus | Gene       | Protein                          |
|-------------------|------------|----------------------------------|
| 1q32              | TNNT2      | Cardiac troponin T              |
| 2q31              | TTN        | Titin                           |
| 3p21              | MYL3       | Essential myosin light chain    |
| 3p21-p14          | TNNC1      | Cardiac troponin C              |
| 7q36              | PRKAG2*    | AMP-activated protein kinase     |
|                   |            | (γ₂-regulatory subunit)         |
| 11p11             | MYBPC3     | Cardiac myosin binding protein C |
| 11p15             | CLP        | Cardiac muscle LIM protein      |
| 12q23-q24         | MYL2       | Regulatory myosin light chain   |
| 14q12             | MYH7       | β-Myosin heavy chain            |
| 15q14             | ACTC       | Cardiac actin                   |
| 15q22             | TPM1       | α-Tropomyosin                   |
| 19q13             | TNNI3      | Cardiac troponin I              |

*Recent data suggest that families with PRKAG2 mutations may have a novel myocardial metabolic storage disease rather than hypertrophic cardiomyopathy (HCM) (see sect. mD).

MYBPC3 is comprised of 24 kb of genomic DNA with 37 exons that encode a protein of 1,274 amino acids. The protein has multiple immunoglobulin C2-like and fibronectin type 3 domains, as well as a cardiac-specific region, a phosphorylation region, and overlapping myosin and titin binding sites. *MYBPC3* mutations are found in ~15–20% individuals with HCM. Thirty mutations have been reported (14, 32, 172). The majority of mutations are insertions, deletions, or splice site mutations that result in truncation of the cMyBP-C protein with loss of the myosin and titin binding sites. Missense mutations that preserve the myosin and titin binding sites have also been found. None of these mutations has been located in the cardiac-specific region.

---

### 4. Cardiac troponin T gene (*TNNT2*)

Cardiac troponin T is expressed in embryonic and adult heart and in developing skeletal muscle. *TNNT2* is comprised of 17 kb of genomic DNA and has 17 exons. A number of different cardiac troponin T isoforms are produced by alternate splicing. The principal isoform in the adult heart consists of 288 amino acids and has two major domains: an NH₂-terminal domain that interacts with tropomyosin and a COOH-terminal domain that binds to tropomyosin, troponin C, and troponin I. *TNNT2* mutations account for ~5–10% of cases of HCM. Fourteen mutations have been reported: 12 missense mutations located in both the NH₂-terminal and COOH-terminal domains, 1 splice site mutation, and 1 single codon deletion (32, 165).

---

### 5. Cardiac troponin I gene (*TNNI3*)

Cardiac troponin I is expressed solely in cardiac tissue. *TNNI3* is comprised of 6.2 kb of genomic DNA and has 8 exons that encode a protein of 210 amino acids. Cardiac troponin I has an inhibitory region (residues 129–149) that, at low Ca²⁺ concentrations, depresses contraction by inhibition of actomyosin ATPase (mediated by protein kinase C phosphorylation). Increased binding of cardiac troponin I to the thin filament (mediated by protein kinase A) also contributes to inhibition of contraction. Under activating conditions, the inhibitory region of cardiac troponin I binds to troponin C-Ca²⁺, permitting actin-myosin interaction. Two binding sites for actin-tropomyosin and a second troponin C site are located more proximal to the COOH terminus. Troponin I forms an antiparallel dimer with troponin C. Eight *TNNI3* mutations have been reported (<5% cases of HCM): seven missense mutations and one single codon deletion (32, 71). Two mutations are located in the inhibitory region, with one mutation found in each of the second troponin C and actin-tropomyosin sites, respectively.

---

### 6. Cardiac troponin C gene (*TNNC1*)

Troponin C has two isoforms that are expressed in cardiac and skeletal muscle. The *TNNC1* gene encodes the isoform that is present in the heart and in slow skeletal muscle. *TNNC1* consists of 3 kb of genomic DNA with 6 exons that encode a protein of 161 amino acids. Ca²⁺ binding to cardiac troponin C induces conformational changes in the troponin-tropomyosin complex that initiate muscle contraction. Cardiac troponin C has two Ca²⁺ binding sites, only one of which is functional. Recently, one missense mutation in the *TNNC1* gene was reported in an individual with HCM (56).

---

### 7. α-Tropomyosin gene (*TPM1*)

α-Tropomyosin is expressed in ventricular myocardium and in fast skeletal muscle. *TPM1* consists of 15 exons, with multiple isoforms resulting from alternate splicing. Five exons are present in all α-tropomyosin transcripts with the remaining 10 exons variably present in different tissues. The cardiac α-tropomyosin isoform is comprised of 10 exons and 284 amino acids. α-Tropomyosin has two binding sites for troponin T, one of which is Ca²⁺ sensitive and the other Ca²⁺ insensitive. Six *TPM1* missense mutations have been reported; three of these are located in the Ca²⁺-sensitive troponin T site (32, 165).

---

### 8. Regulatory MLC gene (*MYL2*)

The MLC belong to a superfamily of Ca²⁺-binding proteins, which is characterized by helix-loop-helix of Ca²⁺-binding sites (EF hands). Other members of this family included troponin C and calmodulin. Cardiac muscle has two regulatory (or phosphorylatable) MLC isoforms. The MLC-2 slow isoform is expressed in the ventricle and in slow skeletal muscle; the MLC-2 atrial isoform is expressed in atrial myocardium. *MYL2* encodes the MLC2 slow isoform. It has seven exons that encode a protein of 166 amino acids. Eight *MYL2* missense mutations have been reported (32, 125).

---

### 9. Essential MLC gene (*MYL3*)

Two of the five essential (or alkali) MLC isoforms are present in cardiac muscle. The MLC-1 slow/ventricular isoform is expressed in the ventricle and slow skeletal muscle. *MYL3* is comprised of 7 exons, 6 of which encode a protein of 195 amino acids. Two *MYL3* missense mutations have been reported (32, 125).

---

### 10. Cardiac actin gene (*ACTC*)

Twenty actin genes are present in the human genome, four of which are found in cardiac, skeletal, and smooth muscle. The cardiac and skeletal actin isoforms are both expressed in the heart and skeletal muscle.

α-Skeletal actin is the predominant actin isoform in the embryonic heart but is downregulated in the adult heart. ACTC has 6 exons that encode 375 amino acids. The NH₂-terminal domain of cardiac actin is the site of myosin cross-bridge attachment; the COOH-terminal domain has binding sites for α-actinin and dystrophin. Five ACTC mutations have been identified; two of these were missense mutations located close to the myosin binding region (32, 119).

### 11. Titin gene (TTN)

Titin is a giant (3 kDa) protein that is the largest known polypeptide. It comprises 10% of vertebrate striated muscle. Titin spans half sarcomeres, with an extensible portion within the I band and a stiffer portion within the A band. Ninety percent of titin’s mass is comprised of up to 298 repeating immunoglobulin and fibronectin 3 domains. The I-band region contains tandemly arranged immunoglobulin domains, as well as a PEVK segment, composed of a sequence rich in proline, glutamine, valine, and lysine residues, and a cardiac-specific N2B region. Titin contributes to the maintenance of sarcomere organization and myofibrillar elasticity. Titin may also participate in myofibrillar cell signaling. Tissue-specific expression of various titin isoforms results in differential tissue elasticity. One HCM-causing TTN missense mutation has been identified in a single individual (143). This mutation was located in the Z-disc binding region of titin.

### 12. γ₂-Regulatory subunit of AMPK gene (PRKAG2)

AMPK is a heterotrimeric protein comprised of a catalytic subunit (α) and two regulatory subunits (β and γ). The γ-subunit has three isoforms (γ₁, γ₂, γ₃) that vary in length and tissue expression. The PRKAG2 gene encodes the γ₂-subunit, which is the predominant γ-isoform present in the heart. PRKAG2 is comprised of >280 kb of genomic DNA. Two isoforms have been identified: a longer transcript (PRKAG2b) with 16 exons that encodes 569 amino acids, and a shorter transcript (PRKAG2a) with 12 exons that encodes tissue-specific proteins of 352 and 328 amino acids, respectively. The shorter transcript results from an alternate transcription initiation site in intron 4. The γ₂-AMPK protein consists primarily of four consecutive cystathionine-β-synthase (CBS) domains. AMPK acts as a “metabolic sensor” in cells, responding to ATP depletion by regulating diverse intracellular pathways that utilize and generate ATP. In the absence of metabolic stress, AMPK activity is suppressed by an autoinhibitory region on the α-subunit that blocks the catalytic site. During periods of hypoxic or metabolic stress, consumption of ATP results in an increase in the AMP/ATP ratio. Rising levels of AMP activate AMPK by interactions with both the autoinhibitory region and the γ-subunit, as well as activating an upstream kinase, AMPKK. In addition to its protein kinase activity, AMPK is postulated to have a transcriptional regulatory role, since it is homologous to the SNF1 transcription factor complex that regulates glucose metabolism in yeast. Five PRKAG2 mutations have recently been reported in families with HCM associated with preexcitation (Wolff-Parkinson-White syndrome); four missense mutations and one in-frame single codon insertion (2, 11, 47).

### 13. Cardiac muscle LIM protein gene (CLP)

Cardiac muscle LIM protein is expressed in cardiac and slow-twitch skeletal muscle. CLP has 4 exons that encode a protein of 194 amino acids. Cardiac muscle LIM protein consists of 2 LIM domains, linked by a spacer of 50 residues. LIM domains are highly conserved cysteine-rich structures that contain two zinc fingers. Cardiac muscle LIM protein is present in embryonic muscle and has been shown to be an important regulator of myogenic differentiation. Cardiac muscle LIM protein is also thought to act as a scaffold for protein assembly in the actin-based cytoskeleton. The first of the two LIM domains interacts with α-actinin, a component of the Z discs, whereas the second LIM domain interacts with actin filaments and spectrin. Four CLP missense mutations have been reported in individuals with HCM (39).

### C. Functional Consequences of Gene Mutations

Since the majority of HCM disease genes encode protein components of the sarcomere, it has been widely proposed that left ventricular hypertrophy is not a primary manifestation but develops as compensatory response to sarcomere dysfunction. Characterization of the fundamental deficit resulting from HCM-causing gene mutations has been a major focus of research over the last decade. A variety of techniques have been used to examine the effects of mutations on sarcomere structure and function, ranging from in vivo studies of myocardial performance in genetically engineered mouse models to in vitro studies of interactions between single actin and myosin molecules. It is clear, as detailed below, that the type of strategy employed to investigate the effects of sarcomere protein mutations greatly influences the outcome, with conflicting results found for the same mutation in many cases. Investigators have sought to answer questions such as whether the various sarcomere protein mutations cause similar or diverse effects on sarcomere structure and function and whether sarcomere protein mutations act by a dominant negative mechanism or alter function by causing haploinsufficiency. In the dominant negative model, both wild-type and mutant proteins are present in equivalent proportions; the mutant peptide is stably incorporated into the sarcomere but acts as a “poison polypeptide” and perturbs wild-type protein function.

Alternatively, mutations may result in null alleles or cause a reduction in the amount of wild-type protein, leading to an imbalance of sarcomere protein stoichiometry. Mutations that truncate the encoded protein are thought to act by haploinsufficiency. Understanding the consequences of sarcomere protein mutations is an essential prerequisite for determining the stimulus for hypertrophy in HCM. For example, if HCM-causing mutant proteins merely induced an imbalance in the stoichiometry of the protein components involved in sarcomere assembly, in vitro analysis of the mutant proteins per se would have little merit.

### 1. β-Myosin heavy chain gene mutations

A) ARG403GLN β-MHC STUDIES. Based on their structural location in the myosin head (see sect. mB2), the majority of MYH7 gene mutations can be predicted to disrupt both mechanical and catalytic components of actin-myosin interaction leading to a reduction of force generation. The Arg403Gln missense mutation in β-MHC has been studied the most extensively. The Arg-403 residue is located in the myosin head at the base of a loop that interacts with actin. A Glu substitution at this residue results in a loss of charge. Studies of the structural and functional consequences of the Arg403Gln mutation in β-MHC have been performed. Sarcomere assembly was disrupted when human Arg403Gln β-MHC was transfected into adult feline cardiomyocytes (87) but was unchanged when this mutation was introduced into neonatal rat cardiomyocytes (7). Functional studies of the Arg403Gln β-MHC mutation using soleus muscle biopsy specimens from patients with HCM have shown that mutant muscle fibers have depressed velocity of shortening, reduced force/stiffness ratio, and reduced power output when compared with control muscle (77). A number of in vitro motility assays performed with human and recombinant mutant β-MHC have shown reduced velocity of actin translocation (23, 142, 155).

B) ARG403GLN α-MHC MOUSE MODEL. Geisterfer-Lowrance et al. (41) generated a mouse model of HCM in which an Arg403Gln point mutation was introduced into the murine α-MHC gene using the “hit-and-run” technique. In contrast to transgenic models in which the location and level of expression of a mutant transgene can be highly variable, the hit-and-run technique results in precise targeting of a single allele at a specific locus, analogous to human heterozygous HCM mutations. Heterozygous Arg403Gln α-MHC (designated α-MHC<sup>403/+</sup>) mutant mice demonstrated progressive left ventricular hypertrophy on transthoracic echocardiography, together with histological evidence of myocyte hypertrophy, myofibrillar disarray, and fibrosis (35, 41). Left ventricular sections from α-MHC<sup>403/+</sup> mice showed normal sarcomere structure on electron microscopy (12).

Systolic function in α-MHC<sup>403/+</sup> mouse hearts has been evaluated in detail using several techniques. Serial in vivo hemodynamic studies demonstrated that young (6 wk) α-MHC<sup>403/+</sup> mice had normal myocardial histology but altered contraction kinetics with accelerated systolic pressure rise. By 20 wk of age, α-MHC<sup>403/+</sup> mice had developed myocardial histopathology as well as hyperdynamic left ventricular contraction, increased end-systolic chamber stiffness, increased left ventricular outflow tract pressure gradient, and lower cardiac index (43). It was proposed that the fast systolic kinetics in young mice represented primary effects of the Arg403Gln mutation with later exacerbation of left ventricular dysfunction attributable to altered left ventricular wall composition and chamber geometry. Two groups of investigators have studied isolated papillary muscles from α-MHC<sup>403/+</sup> mice. In one study, α-MHC<sup>403/+</sup> and control left ventricular trabeculae and papillary muscles were able to generate similar peak force during twitch contractions, but mutant muscle required higher intracellular Ca<sup>2+</sup> concentrations to achieve equivalent force. Increased Ca<sup>2+</sup> mobilization in mutant muscle was insufficient to maintain force, however, at high stimulation rates (37). In a second study, sinusoidal length perturbation analysis was used to generate oscillatory work and evaluate cross-bridge function in α-MHC<sup>403/+</sup> papillary muscle strips. Ca<sup>2+</sup> dependence of α-MHC<sup>403/+</sup> muscle was demonstrated. At maximal or near-maximal Ca<sup>2+</sup> activation, peak isometric tension and oscillatory power were reduced in mutant muscle strips; at submaximal Ca<sup>2+</sup> concentrations, tension and power output were shown to have increased Ca<sup>2+</sup> sensitivity. Compared with wild-type muscle strips, α-MHC<sup>403/+</sup> muscle strips had depressed cross-bridge kinetics (12). α-MHC<sup>403/+</sup> myocytes have been shown to have depressed velocity of contraction but equivalent sarcomere length, fractional shortening, and peak amplitude of Ca<sup>2+</sup> transients when compared with wild-type myocytes (70). Finally, elegant single molecule studies of MHC isolated from α-MHC<sup>403/+</sup> mouse hearts have shown dose-related increases in actin-activated ATPase activity, force generation, and actin filament sliding velocity (169).

Does the Arg403Gln MHC mutation cause hypo- or hypercontractile function? Although several studies have suggested that Arg403Gln MHC augments cross-bridge kinetics and force generation, a large body of data derived from in vitro studies has suggested that Arg403Gln MHC perturbs actin-myosin interaction and depresses motor function. Several pathophysiological mechanisms for these findings have been proposed, including altered myosin binding affinity for actin, reductions in cross-bridge cycling rates, reduction in the extent of actin displacement per cross-bridge cycle, a drag effect on normal cross-bridges by mutant cross-bridges, or abnormal interactions between heterodimeric myosin heads. Observations of depressed contractile function, at the cellular

level, however, cannot be readily reconciled with the clinical finding of preserved, or enhanced, systolic function in patients with HCM. These apparently conflicting findings highlight the importance of the methods used to evaluate contractile performance. Studies of the interaction between single myosin and actin molecules might intuitively appear to be the optimal technique for determining the fundamental consequences of a mutant protein. However, these studies do not take into account the effects of loading or the regulatory influences of other protein components of the sarcomere. In the intact heart, the presence of hypertrophy, myofibrillar disarray, fibrosis, and hemodynamic changes will further influence left ventricular systolic and diastolic performance. Differences in contractile “environment” may explain the seemingly paradoxical findings related to effects of mutant protein “dose”: in single molecule studies, increasing amounts of mutant myosin cause progressive increases in systolic function, whereas in vivo studies in homozygous Arg403Gln α-MHC mice demonstrate reduced systolic function with a rapidly progressive dilated cardiomyopathy (33).

Studies of left ventricular diastolic function in α-MHC<sup>403/+</sup> mouse hearts have uniformly shown prolongation of relaxation, analogous to that observed in patients with HCM (41, 43, 149). It has been proposed that diastolic dysfunction may be a direct mechanical consequence of slowed cross-bridge cycling rates and thus a fundamental consequence of HCM mutations. The detection of prolonged decay of Ca<sup>2+</sup> transients in α-MHC<sup>403/+</sup> myocytes raises the additional possibility that Ca<sup>2+</sup> dysregulation or altered Ca<sup>2+</sup> sensitivity may contribute to impaired relaxation (70). Myocardial energetic studies suggest that an energy-requiring process may also prolong diastolic relaxation (149). The development with age of left ventricular structural changes may increase diastolic stiffness and further exacerbate left ventricular diastolic filling abnormalities. Further studies are required to determine the molecular and cellular consequences of diastolic dysfunction and its relationship with the development of hypertrophy.

### 2. cMyBP-C gene mutations

cMyBP-C is thought to have structural and regulatory roles in the sarcomere. Experimental studies have focused on the effects of cMyBP-C mutations, in particular, those that result in truncated proteins, on sarcomere assembly. Transfection of a range of COOH-terminal truncation mutants into skeletal myoblasts demonstrated a minimal region of 372 amino acids was required for correct localization into the sarcomere A band; one construct, which lacked the major myosin binding domain, strongly inhibited myofibril assembly. It was proposed that truncated protein might compete with endogenous cMyBP-C at the myosin binding site. The truncated proteins were all detected by Western blot (45). In another study, truncated cMyBP-C protein in cardiac tissue from a patient with a splice donor site mutation was unable to be detected by Western blot, despite the presence of an mRNA transcript. It was suggested that the truncated protein may have undergone rapid proteolysis and that a reduction in cMyBP-C protein might alter the stoichiometry of sarcomere proteins with consequent impairment of sarcomere assembly (138).

Western blot analyses of myocardium from a transgenic mouse expressing a truncated cMyBP-C lacking myosin and titin binding domains demonstrated endogenous regulation of total cMyBP-C levels with overexpression of the mutant protein compensated by a reduction in wild-type protein (180). Histological analyses of myocardial sections from 25-wk-old mice showed foci of degenerate myocytes and sarcomeric disorganization. The mutant protein was incorporated into the sarcomere but was also present diffusely throughout the cytoplasm. A second mouse model with a similar cMyBP-C truncation was generated using homologous recombination techniques (95). In contrast to α-MHC<sup>403/+</sup> mice, heterozygous mice (MyBP-C<sup>t/+</sup>) did not develop left ventricular hypertrophy until after 125 wk of age. This late onset of hypertrophy is analogous to that observed in patients with HCM caused by cMyBP-C mutations. Total cMyBP-C protein expression in the left ventricle of MyBP-C<sup>t/+</sup> mice was slightly reduced (90% wild-type levels). In contrast to the transgenic mouse studies, myocardial histology of MyBP-C<sup>t/+</sup> mice aged 125 wk was indistinguishable from age-matched wild-type mice. In particular, sarcomere assembly was normal. Differences between these in vitro and in vivo studies in the effects of truncated cMyBP-C on sarcomere organization and protein localization may result from differences in the relative proportions of mutant and wild-type protein, with abnormalities detected only with overexpression models. Endogenous regulation of protein levels or posttranslational modification may reduce the levels of wild-type protein below a threshold level required for maintenance of normal sarcomere relationships. Alternatively, truncated protein may competitively inhibit wild-type protein function. Dose-related effects are suggested by the findings of normal myocardial histology in heterozygous MyBP-C<sup>t/+</sup> mice but prominent left ventricular hypertrophy, disarray, and fibrosis in homozygous MyBP-C<sup>t/t</sup> mice (95, 96).

Few studies have examined the functional consequences of cMyBP-C truncations. Hemodynamic analyses in isolated heart preparations from transgenic cMyBP-C mice aged 25 wk showed no differences in contraction or relaxation parameters between mutant and wild-type hearts (180). Similarly, in vivo hemodynamic studies showed no differences in systolic or diastolic parameters between MyBP-C<sup>t/+</sup> and wild-type mice (95). Studies of

ventricular fibers from transgenic mice demonstrated a leftward shift in the pCa-force relationship and a reduction of maximum power (180). It is not clear whether these changes represent primary effects of the mutant protein on sarcomere function or are a consequence of the deranged sarcomere structure present in this model.

3. Cardiac troponin T gene mutations

A) ARG92GLN, ILE79ASN MUTATIONS. The Arg-92 and Ile-79 residues are located in the NH₂-terminal domain of cardiac troponin T. The functional consequences of the Arg92Gln and Ile79Asn missense mutations have varied according to the techniques employed. In one study, transfection of Arg92Gln and Ile79Asn cardiac troponin T into quail myotubes resulted in increased myotube shortening velocity. The Ile79Asn mutation, but not Arg92Gln, reduced maximum force (154). In a second study, in vitro motility assays performed using recombinant embryonic rat cardiac troponin T bearing an Ile79Asn substitution demonstrated a 50% increase in velocity of translocation of thin filaments over myosin (83). In contrast, transfection of Arg92Gln cardiac troponin T in adult feline cardiac myocytes showed reductions in fractional shortening and peak velocity of shortening (88). Ca²⁺ sensitivity of force production was increased in Arg92Gln and Ile79Asn cardiac troponin T in skinned cardiac muscle preparations (108, 157) but was decreased in transfected rat adult cardiac myocytes (139) and quail myotubes (154). Ca²⁺ sensitivity of myofibrillar ATPase was increased in transfected rabbit cardiac myofibrils (177). No abnormalities of sarcomere structure have been reported in any of these studies.

An Arg92Gln cardiac troponin T transgenic mouse model with transgene expression levels ranged from 1 to 10% has been characterized. Adult mice exhibited systolic and diastolic left ventricular dysfunction with variable myocyte degeneration, necrosis, myofibril disarray, and fibrosis (82, 117). Another group of investigators have found that Arg92Gln cardiac troponin T transgenic mice with 30, 67, and 92% transgene expression levels exhibited dose-related myocardial disarray and fibrosis (162). Surprisingly, mutant hearts in the latter study were smaller than control hearts, with a reduction in both the number and size of myocytes. Isolated working hearts from the mice with 67% transgene expression were hypercontractile with stepwise increases in cardiac work load. Single myocytes from these mice had reduced shortening and prolonged contraction times. Both isolated heart and myocyte studies showed impaired diastolic relaxation. Skinned papillary muscle fibers from transgenic mice expressing Ile79Asn cardiac troponin T showed increased Ca²⁺ sensitivity of ATPase activity and force development with accelerated kinetics of force activation and relaxation. A computer simulation of intracellular Ca²⁺ and

force generation predicted that the Ile79Asn cardiac troponin T mutation altered cross-bridge kinetics and increased cardiac troponin T Ca²⁺ affinity (103).

B) INT15G TO A SUBSTITUTION. The Int15G to A substitution results in truncation of the COOH terminus of the cardiac troponin T protein. Transfection of a truncated cardiac troponin T in quail myotubes resulted in decreased Ca²⁺ sensitivity of force production, similar to the findings in the Arg92Gln experiments (175). In recombinant cardiac muscle preparations, this truncation mutant reduced Ca²⁺ activation and inhibition of force and reduced activation and inhibition of actin-tropomyosin-activated myosin ATPase activity (157). In in vitro motility assays, truncated cardiac troponin T increased the velocity of actin-tropomyosin filaments to a greater extent than wild-type cardiac troponin T at low pCa (activating conditions) but failed to decrease the velocity of filaments and had negligible inhibition of actin-tropomyosin-activated myosin ATPase at high pCa (relaxing conditions). Varying proportions of wild-type and mutant cardiac troponin T significantly influenced the results: at pCa 5, the inhibitory effect of wild-type protein on thin filament velocity was enhanced at lower protein levels (10–50%) (132). A transgenic mouse with a truncated cardiac troponin T exhibited similar findings to the Arg92Gln cardiac troponin T transgenic mouse, with myocyte disarray, fibrosis, and a reduction in the number and size of myocytes. Echocardiographic studies in the truncated cardiac troponin T mouse showed a mild reduction in systolic contraction and a relatively greater extent of impaired diastolic relaxation (161).

Cardiac troponin T mutations would be predicted to influence the inhibitory regulatory effect of the tropomyosin-troponin complex. In vitro studies showing increased contraction kinetics and increased Ca²⁺ sensitivity of force generation are consistent with this concept. The observation of hypocontractility in some studies has also been attributed to increased Ca²⁺ sensitivity, with higher basal levels of sarcomeric activation resulting in initiation of contraction at shorter, less optimal sarcomere lengths against a stronger passive restoring force (162). This may also partly explain the small size of myocytes. Myocyte degeneration and loss may further depress contractile performance. Despite myocardial histopathology and functional changes, none of the cardiac troponin T mouse models has exhibited left ventricular hypertrophy. Individuals with HCM caused by cardiac troponin T mutations typically have minimal left ventricular hypertrophy.

4. Cardiac troponin I gene mutations: Arg145Gly mutation

The Arg-145 residue is located in the inhibitory region of cardiac troponin I that is required for inhibition of actin-tropomyosin-activated myosin ATPase and Ca²⁺-

mediated binding to troponin C. In vitro functional analyses have shown that the Arg145Gly cardiac troponin I mutation results in reduced inhibition of actin-tropomyosin-activated myosin ATPase and increased Ca²⁺ sensitivity of ATPase regulation (30, 158). A transgenic mouse overexpressing Arg145Gly cardiac troponin I (1.2-fold) had normal life expectancy and no overt phenotype, with the exception that females stressed by pregnancy developed histological evidence of cardiomyocyte hypertrophy and patchy myocardial fibrosis. Isolated working heart preparations from unstressed mutant mice showed significantly enhanced contractility (+dP/dt) and prolonged relaxation (-dP/dt). Mice with 3.5-fold transgene overexpression died by 17 days. At 10 days after birth, the ventricles of these mice showed myocyte disarray, nuclear degeneration, and interstitial fibrosis. Skinned papillary muscle strips from 10-day-old mutant mice showed no differences in unloaded shortening velocity, maximum shortening velocity, or maximum relative power but increased Ca²⁺ sensitivity and reduced maximum tension (61). It has been proposed that mutations in the inhibitory region of cardiac troponin I disrupt the Ca²⁺-sensitive switch and cause the thin filament to remain in an “on” position analogous to the changes observed with cardiac troponin T gene mutations. The differences in contractile performance between mice with 1.2- and 3.5-fold transgene expression may reflect dose-related differences in basal sarcomere length or more severe myocardial histopathology.

tant fibers demonstrated no differences in shortening velocity or maximal force generation but did have increased Ca²⁺ sensitivity of force production, compared with controls. Transgenic mice expressing Asp175Asn α-tropomyosin exhibited varying extent of myocyte hypertrophy, disarray, and fibrosis by 20 wk of age. Both endogenous and mutant α-tropomyosin protein incorporated into myofibrils. Normal α-tropomyosin stoichiometry was maintained with overexpression of the mutant protein compensated by a reciprocal reduction in the level of endogenous protein. Transthoracic echocardiography showed no significant differences between transgenic and control mice in left ventricular diameters, fractional shortening, or wall thickness. After an 8-wk swimming program, left ventricular fractional shortening increased in control mice but was unchanged in transgenic mice. In isolated heart preparations from mice expressing 60% Asp175Asn protein, both the rates of contraction and relaxation were reduced. These functional changes were not observed in mice expressing <40% mutant protein. Skinned fiber preparations from transgenic mouse hearts showed a leftward shift in the pCa²⁺-force relationship (112).

These in vitro and in vivo data indicate that the Asp175Asn mutation increases the basal level of activation of α-tropomyosin. This may result from structural changes, such as altered protein conformation, or functional changes mediated by changes in binding of the troponin complex or altered Ca²⁺ affinity. Variability in functional assays has been observed, as found with cardiac troponins T and I. Downregulation of endogenous α-tropomyosin suggests that mutations may act by haploinsufficiency. However, similar (50%) levels of mutant and endogenous protein have been found in human HCM, consistent with a dominant negative mechanism.

5. α-Tropomyosin gene mutations:  
   Asp175Asn mutation

Asp175 is located in the Ca²⁺-sensitive troponin T binding domain of α-tropomyosin. The substitution of Asp to Asn at this residue causes the loss of one negative charge, which alters the mobility of α-tropomyosin on gel electrophoresis (16) and results in local unfolding of the encoded protein. In in vitro motility assays, Asp175Asn α-tropomyosin had a relatively greater increase in velocity than wild-type α-tropomyosin, following addition of troponin. All other parameters were unchanged, suggesting that the effects of the mutation were due predominantly to altered interaction with troponin T (9). Addition of N-ethylmaleimide-conjugated myosin subfragment-1 (NEM-S1), a strongly binding myosin analog that cooperatively enhances thin filament activation, failed to further increase thin filament velocity, indicating that the Asp175Asn substitution switched α-tropomyosin to a fully on state (133).

Mechanical properties of skeletal muscle fibers from patients with HCM due to Asp175Asn α-tropomyosin have been studied (16). The levels of mutant and wild-type α-tropomyosin protein were similar (~50%), and other α-tropomyosin isoforms were not upregulated. The mu-

6. Myosin essential and regulatory light chain gene mutations: Met149Val essential MLC, Glu22Lys regulatory MLC mutations

MLC are thought to influence the mechanical efficiency of cross-bridge cycling and the speed of contraction. No specific functional domains in the MLC have been recognized. In vitro motility assays have shown that actin sliding velocities were increased by the Met149Val essential MLC substitution but were unchanged by the Glu22Lys regulatory MLC substitution (125). These two missense mutations have also been evaluated in transgenic mouse models (141). Mice with Met158Val essential MLC (analogous to human Met149Val essential MLC) exhibited dose-related myofibril disarray and fibrosis. While this mutation is associated with the development of papillary muscle hypertrophy in humans, transgenic mice had lower left ventricular mass and smaller myocytes than observed in control mice. Functional analyses in isolated

working heart preparations showed hypercontractility and impaired relaxation in the mutant mice. Studies of skinned ventricular fibers and in vitro motility assays showed no differences in shortening velocity or actin translocation, respectively, between mutant and wild-type mice. Met158Val essential MLC fibers did demonstrate leftward shifts in the ATPase activity and pCa-force curves and reduced power output. Histological, cellular, and molecular analyses of transgenic mice with the Glu22Lys RLC mutation were indistinguishable from wild-type mice. Skeletal muscle fibers from an individual with a Glu22Lys regulatory MLC mutation were found to have a leftward shift in the pCa-force relationship (78). These data suggest that the essential MLC and regulatory MLC may regulate power output through a Ca²⁺-dependent mechanism.

D. Triggers and Effectors of Left Ventricular Hypertrophy

1. Triggers of left ventricular hypertrophy

The development of left ventricular hypertrophy has been generally considered to be an adaptive response to biomechanical stress that enables cardiac work to be maintained. Despite these generally accepted principles of hypertrophy development and its consequences, it is of note that a recent study using genetically engineered mice with a markedly blunted growth response to pressure overload questions the adaptive value of load-induced hypertrophy (31). Surprisingly, despite failing to correct wall stress after acute partial transthoracic aortic constriction, cardiac function in these animals with either inhibition of Gq signaling or disruption of catecholamine synthesis was well maintained. Indeed, function was even better maintained than in wild-type mice whose wall stress was normalized as a result of hypertrophy development. This study also highlights the primacy, not only of the Gq signaling pathway in load-induced hypertrophy, but somewhat more surprisingly, that of the sympathetic nervous system.

Left ventricular hypertrophy may be “physiological” in elite athletes or occur in pathological states, such as hypertension, myocardial infarction, cardiomyopathies, valvular heart disease, and a variety of systemic disorders. The hypertrophic process is thought to be initiated by factors extrinsic and intrinsic to the cardiac myocyte. Extrinsic stimuli include vasoactive peptides (e.g., angiotensin II, endothelin-1), α₁-adrenergic agonists (e.g., norepinephrine, epinephrine, phenylephrine), activators of protein kinase C (e.g., tumor-producing phorbol esters), peptide growth factors (e.g., insulin-like growth factor, fibroblast growth factor), cytokines (e.g., cardiotrophin-1), arachidonate metabolites (e.g., prostaglandin F₂α), mechanical stretch, and cell contact. Intrinsic stimuli include elevated [Ca²⁺]i, the heterotrimeric G protein Gq, as well as activated small G proteins, kinases, phosphatases, and transcriptional factors (151). These extrinsic and intrinsic factors trigger a complex cascade of intracellular pathways, termed the “hypertrophic response,” which results in increased myocardial mass, altered spatial relationships between myocytes and other cellular and extracellular components of the myocardium, reprogramming of myocardial gene expression, and apoptosis. A rapidly growing list of genes has been found to elicit cardiac hypertrophy when overexpressed in transgenic mice. Determination of which of these genes are clinically relevant in human hypertrophy will be important.

A) HCM. Despite a decade of research, the stimulus for hypertrophy in HCM has not been definitively identified. Collectively, data from in vitro and in vivo studies demonstrate that sarcomere protein gene mutations perturb sarcomere structure and/or function and thus are likely to be disease causing. The precise consequences of sarcomere protein gene mutations differ according to the type of model studied with both reduced and augmented motor function described for individual HCM disease genes. Mutations in the various sarcomere protein genes also have diverse effects on motor function. It appears, however, that mechanical dysfunction of the sarcomere is a common feature and, hence, a potential stimulus for hypertrophy (Fig. 2). Whether a threshold level of sarcomere dysfunction is required before the hypertrophic response is triggered is unknown. Observations of hypo- and hypercontractility in different studies of a single gene highlight the importance of the model used and the effects of the contractile “milieu,” i.e., the presence or absence of thin filament regulatory elements, myocyte loading, myocardial histopathology, etc. What is the most physiologically relevant method for studying contractile function of mutant sarcomere proteins? This will remain unanswered until we understand the origin (i.e., in the intact organ or at the cellular or molecular level) and the nature of the signaling processes that initiate the hypertrophic response.

B) INTRACELLULAR CALCIUM. Several years ago, the observation that transgenic mice expressing constitutively activated calcineurin developed left ventricular hypertrophy that was prevented by administration of the calcineurin inhibitors cyclosporin and FK506 initiated intense interest in the role of elevated [Ca²⁺]i and the calcineurin signaling pathway in the pathogenesis of left ventricular hypertrophy (105). Calcineurin is a cytoplasmic protein phosphatase that is activated by sustained [Ca²⁺]i. Activated calcineurin dephosphorylates NFAT3 (nuclear factor of activated T cells) transcription factors, which causes their translocation to the nucleus where they combine with the cardiac-restricted zinc finger transcription factor GATA4, resulting in synergistic activation of embryonic cardiac genes and a hypertrophic response. Although the trans-

A Hypertrophic cardiomyopathy

Sarcomeric/intrasarcomeric  
protein mutation  
↓  
↑ Ca²⁺-force & Ca²⁺-ATPase  
↓  
LV diastolic dysfunction  
↓  
LV hypertrophy  
↓  
Heart failure  

LV remodelling  
(apoptosis, necrosis  
fibrosis)  

↑ SNSA  
↓  
LV outflow tract  
obstruction  

AMP kinase  
mutation  
↓  
↓ ATP production  
↓  
↑ ATP  
consumption  
↓  
↓ ATP availability  
↓  
↓ SERCA2a activity  
↓  
↑ cytosolic free Ca²⁺  
↓  
Ca²⁺ overload  
↓  
Sudden death  

B Dilated cardiomyopathy  

Sarcomeric/intrasarcomeric  
protein mutation  
↓  
↓ Ca²⁺-force & Ca²⁺-ATPase  
↓  
↓ Force generation  
↓  
LV systolic dysfunction  
↓  
Heart failure  
↓  
Death  

Cytoskeletal protein  
mutation  
↓  
↓ Force transmission  
↓  
LV chamber dilation  
↓  
↑ SERCA2a inhibition  

LV remodelling  
(apoptosis, necrosis, fibrosis)  

FIG. 2. Potential pathophysiological mechanisms underlying hypertrophic cardiomyopathy (A) and dilated cardiomyopathy (B). LV, left ventricular; SERCA2a, sarcoplasmic reticulum Ca²⁺-ATPase; SNSA, sympathetic nervous system activity.

genic mouse studies suggested that the calcineurin pathway might be critical in the development of myocardial hypertrophy, studies in a variety of different models have suggested that multiple signaling pathways are likely to be involved (118, 151). Recently, the effects of calcineurin inhibition in HCM were examined in α-MHC^{403/+} mice (35). Paradoxically, cyclosporin and FK506 administration both dramatically increased the severity of left ventricular hypertrophy and accelerated sudden death in the mutant mice. Cyclosporin-induced hypertrophy was pre-
vented by coadministration of the L-type Ca²⁺ channel blocker diltiazem. Studies of α-MHC⁴⁰³/⁺ and wild-type myocytes showed no differences in Ca²⁺ transients at baseline. Diastolic Ca²⁺ levels increased in wild-type myocytes but were unchanged in α-MHC⁴⁰³/⁺ myocytes after acute administration of cyclosporin, and after chronic oral treatment with cyclosporin. Similar results were also found in response to acute administration of minoxidil, a K⁺ channel agonist that has been shown to cause left ventricular hypertrophy in humans and rodents. These data suggest first that abnormal Ca²⁺ regulation may play an important role in the hypertrophic response in HCM, and second, that administration of calcineurin inhibitors such as cyclosporin may actually be harmful to patients with HCM.

What is the basis for the abnormalities of Ca²⁺ regulation observed in HCM? The inability to elevate diastolic Ca²⁺ in response to cyclosporin in α-MHC⁴⁰³/⁺ myocytes might be indicative of a relative depletion of intracellular Ca²⁺ reserves that might result from sequestration of Ca²⁺ by the contractile apparatus due to reciprocal feedback of cross-bridge formation on troponin C/Ca²⁺ binding affinity. This might also explain the increased Ca²⁺ sensitivity observed in mutant myofibrils and contribute to delayed diastolic relaxation (Fig. 2). These data raise intriguing questions about the mechanisms by which intracellular Ca²⁺ might influence hypertrophic signaling pathways. In contrast to the traditional concept, cytoplasmic free [Ca²⁺] did not appear to be the stimulus for left ventricular hypertrophy in α-MHC⁴⁰³/⁺ mice. It is possible that the [Ca²⁺] in another intracellular compartment or total cellular Ca²⁺ might be the critical Ca²⁺ sensor. Calcium may provide a critical link between mechanical dysfunction of the sarcomere and induction of the hypertrophic gene program. Despite these considerations, it should be noted that recent studies of mice in which calcineurin was inhibited not by drugs, which can affect unrelated pathways as well as being toxic, but by genetic engineering, provide compelling evidence for an important role of the calcineurin/NFAT3/GATA4 pathway in load-induced hypertrophy (28, 137, 182). It will be of interest to see if inhibition of this pathway also prevents the hypertrophy observed in the various genetically induced HCM models, which should be evident when they are crossed with the calcineurin-inhibited animals.

c) MYOCARDIAL ENERGETICS. The recent finding of mutations in the PRKAG2 gene (see sect. III.B12) was somewhat unexpected, given the traditional dogma that sarcoplasmic dysfunction was the fundamental defect in HCM. These PRKAG2 mutations raise the intriguing possibility that defective myocardial energetics may be a common feature of HCM mutations. Alterations of myocardial metabolism had been observed previously in patients with HCM (66) and in α-MHC⁴⁰³/⁺ mice (149). Although these energetic changes might result from factors such as myocardial ischemia due to increased oxygen demands of the hypertrophic myocardium, it is also possible that alterations of myocardial metabolism may occur as primary effects of mutant proteins (Fig. 2). For example, if sarcoplasmic dysfunction due to contractile protein mutations created a mechanically unfavorable and energy-requiring state, a relative myocardial energy deficit may then provide a stimulus for hypertrophy. Although defective myocardial energetics remain an attractive hypothesis, the clinical diagnosis of families with PRKAG2 gene mutations has very recently been questioned. The demonstration of vacuoles containing glycogen-associated granules within the myocardium of affected individuals, together with in vitro experiments indicating constitutive activation of AMPK, suggested that PRKAG2 gene mutations might not cause familial HCM, but rather a novel myocardial metabolic storage disease (2).

### 2. Effectors of left ventricular hypertrophy

To date, research interest in HCM has been directed toward characterization of the consequences of gene mutations on sarcomere structure and function and identification of factors that might trigger the hypertrophic response. The cascade of signaling pathways that constitute the effectors of hypertrophy in HCM is largely unexplored. It has not been established whether the various HCM gene mutations stimulate the same or different hypertrophic pathways. It is also not known to what extent hypertrophic response pathways in HCM might differ from those in other pathological states.

The severity of left ventricular hypertrophy varies considerably between individuals with different sarcomere protein gene mutations and also between individuals in a family with the same gene mutations. Are these differences due to varying severity of the stimulus for hypertrophy or do other factors influence the expression of the hypertrophic phenotype? Observations in mouse models do not support a direct correlation between the extent of sarcomere dysfunction and left ventricular hypertrophy. For example, although cardiac function is impaired in α-MHC⁴⁰³/⁺ mice and normal in mice with truncated cMyBP-C, both mouse models develop similar severity of left ventricular hypertrophy (95). In both of these mouse models, heterozygous mice have left ventricular hypertrophy but homozygous mice exhibit dilated cardiomyopathy rather than severe hypertrophy (33, 95, 96). These findings suggest that the extent to which left ventricular hypertrophy can compensate for a cardiac motor deficit is dose related and that severe dysfunction is less likely to be “rescued.” Further studies are required to determine the effects of additional genetic factors and environmental factors as modifiers of the hypertrophic phenotype. In the α-MHC⁴⁰³/⁺ mice, left ventricular hypertrophy is observed in 100% of mice with a 129SvEv background.
ground but only in 50% of mice with a Black Swiss background (95). These observations are best explained by the presence of modifier gene(s) in these mouse strains. Exacerbation of left ventricular hypertrophy in α-MHC<sup>403/+</sup> mice by calcineurin inhibitors and the subsequent prevention of hypertrophy with Ca<sup>2+</sup> blockade demonstrated, for the first time, that environmental factors (i.e., pharmacological agents) could also modify the hypertrophic phenotype in HCM.

IV. DILATED CARDIOMYOPATHY

A. Clinical Manifestations

1. Disease characteristics

Dilated cardiomyopathy (DCM) is a myocardial disorder characterized by cardiac dilatation and contractile dysfunction of the left and/or right ventricles. DCM may result from a diverse variety of etiologic agents that cause cardiomyocyte dysfunction or injury. In ~50% of cases, a specific precipitating cause is unable to be identified (“idiopathic” DCM). The incidence of idiopathic DCM has been estimated to be 5–8 cases per 100,000 per year (27). Although DCM has traditionally been regarded as a sporadic nongenetic disorder, recent studies of large families with DCM suggest that inherited gene defects are an important cause of idiopathic DCM (“familial” DCM).

2. Prevalence of familial DCM

The reported prevalence of familial DCM has varied according to the methods of detection and diagnostic criteria employed. Studies based solely on a positive clinical history of DCM in relatives of probands have yielded a relatively low prevalence (<10%) of familial disease. In contrast, studies in which first-degree relatives of probands have been evaluated systematically, irrespective of the presence of symptoms, with physical examination, 12-lead electrocardiography, and transthoracic echocardiography, suggest a prevalence of up to 35% (49, 69, 102). These data may underestimate the true prevalence of familial DCM. In the majority of studies, the diagnosis of DCM has been based on the World Health Organization criteria, i.e., the presence of both ventricular dilation and contractile dysfunction (135). Abnormalities of either ventricular dimensions or contractile function have been observed commonly, however, in asymptomatic relatives of probands with DCM. It is not yet clear whether these findings represent early disease or a more benign form of disease. Age-related penetrance (i.e., absence of disease manifestations in genotype-positive individuals until after a particular age) and nonpenetrance (i.e., genotype-positive but phenotype-negative throughout life) may contribute to underestimation of the prevalence of familial dis-

ease. The diagnosis of familial DCM may also be confounded by the concurrent presence of conditions known to promote DCM, such as myocardial ischemia, viral infection, or alcohol excess. The extent to which genetic factors may increase susceptibility to the recognized DCM precipitating factors has not been evaluated.

3. Diagnosis of familial DCM

Studies of families with DCM have revealed autosomal dominant, autosomal recessive, X-linked, and maternal modes of inheritance, with the autosomal dominant pattern observed most frequently. Familial DCM exhibits marked clinical variability, with several distinct phenotypic groups: DCM alone, DCM with conduction-system disease (sinus bradycardia, atrioventricular conduction block, atrial tachyarrhythmias), DCM with skeletal myopathy with or without conduction-system disease, or DCM with sensorineural deafness. In some families, DCM may occur as part of a multisystem inherited disorder. The relative prevalence of each of these phenotypic groups has not been determined.

Apart from family history, no single clinical parameters have been found to reliably distinguish familial from nonfamilial DCM. In some cases, genetically affected individuals may be asymptomatic, with an incidental finding of cardiomegaly on a routine chest X-ray. The majority of individuals present with symptoms attributable to left ventricular failure or cardiac arrhythmias, including weakness, fatigue, dyspnea, orthopnea, palpitations, or exercise intolerance. Symptoms of right heart failure, such as peripheral edema and abdominal distension due to hepatic congestion and ascites, are likely to occur with disease progression.

It has been suggested that individuals with familial DCM are diagnosed at an earlier age than individuals with nonfamilial DCM (101). This may reflect the presence of a relatively malignant phenotype in some families, or more intensive screening of individuals deemed to be at high risk due to their family history.

The principal modality for the diagnosis of familial and nonfamilial DCM is transthoracic echocardiography. Two-dimensional echocardiographic imaging is used to quantitatively assess ventricular structure and systolic function and to evaluate valvular pathology and atrial size. Doppler echocardiographic studies are used to assess ventricular diastolic function as well as the severity of mitral and tricuspid valve regurgitation. Gated radionuclide scans may be used to quantify ventricular volumes and contractile performance, particularly in individuals in whom echocardiography is technically difficult. Cardiac catheterization may also be used for assessment of ventricular function, particularly in individuals concurrently at risk for coronary artery disease. Endomyocardial biopsy generally shows nonspecific findings such as myo-

TABLE 2. DCM chromosomal loci and disease genes

| Chromosomal Locus | Gene       | Protein                          |
|-------------------|------------|----------------------------------|
| **Autosomal dominant inheritance** |
| DCM only          |            |                                  |
| 1q32              | TNNT2      | Cardiac troponin T              |
| 2q31              | TTN        | Titin                           |
| 2q35              | DES        | Desmin                          |
| 5q33              | SGCD       | δ-Sarcoglycan                   |
| 6q12-q16          | ?          | ?                               |
| 9q13-q22          | ?          | ?                               |
| 9q22-q31          | ?          | ?                               |
| 10q22-q23         | VCL        | Metavinculin                    |
| 11p11             | MYBPC3     | Cardiac myosin binding protein C |
| 14q12             | MYH7       | β-Myosin heavy chain            |
| 15q14             | ACTC       | Cardiac actin                   |
| 15q22             | TPM1       | α-Tropomyosin                   |
| DCM + conduction-system disease |
| 1p1-q21           | LMNA       | Lamins A and C                  |
| 2q14-q22          | ?          | ?                               |
| 3p22-p25          | ?          | ?                               |
| DCM + skeletal myopathy ± conduction-system disease |
| 1p1-q21           | LMNA       | Lamins A and C                  |
| 6q23              | ?          | ?                               |
| DCM + sensorineural deafness |
| 6q23-q24          | ?          | ?                               |
| **X-linked inheritance** |
| Xp21 (X-linked DCM) | DMD       | Dystrophin                      |
| Xq28 (X-linked infantile DCM) | G4.5    | Tafazzin                      |

DCM, dilated cardiomyopathy.

candidate gene approach in small families. The first disease gene identified was cardiac actin (chromosome 15q14) (123). Five further sarcomere protein genes that have been associated with HCM have now been shown to also cause DCM: β-MHC (chromosome 14q12) (67), cMyBP-C (chromosome 11p11) (134), cardiac troponin T (chromosome 1q32) (67), α-tropomyosin (chromosome 15q22) (122), and titin (chromosome 2q31) (44). DCM-only mutations have also been found in the cytoskeletal proteins desmin (chromosome 2q35) (80), δ-sarcoglycan (chromosome 5q33) (168), and metavinculin (chromosome 10q22-q23) (120).

Linkage studies performed in families with autosomal dominant DCM with conduction system disease have identified three loci: on chromosomes 1p1-q21 (68), 2q14-q22 (65), and 3p22-p25 (121); the disease-causing gene has been found in one of these loci (lamin A/C gene at chromosome 1p1-q21) (34). Autosomal dominant DCM with conduction-system disease and skeletal myopathy has been mapped to two loci [chromosomes 1p1-q21 (15, 109) and 6q23 (100)]; mutations in the lamin A/C gene have been associated with two skeletal myopathies with cardiac involvement, Emery-Dreifuss muscular dystrophy (15), and autosomal dominant limb girdle muscular dystrophy 1B (109). Autosomal dominant DCM with sensorineural deafness has been mapped to chromosome 6q23-q24 (120).
neural hearing loss has been mapped to chromosome 6q23-q24 (144).

Autosomal recessive inheritance has been associated with infantile forms of DCM. No chromosomal loci or disease genes for autosomal dominant infantile DCM have been reported. Two disease genes have been found to cause X-linked DCM. Mutations in the gene encoding dystrophin have been found in Duchenne and Becker muscular dystrophy but have also been shown to cause an adult-onset X-linked DCM without skeletal myopathy (167). Mutations in G4.5, encoding tafazzin, cause Barth's syndrome as well as a childhood-onset X-linked DCM (24). Mutations in mitochondrial DNA have been found in maternally inherited mitochondrial myopathies (153).

### 2. Cardiac actin gene (ACTC)

The ACTC gene encodes the sarcomere thin filament protein cardiac actin. Characteristics of the ACTC gene have been described in section mB10. ACTC was the first gene associated with adult-onset autosomal dominant DCM. Because cardiac actin has an integral role in cardiac muscle contraction, it was considered to be a promising candidate for DCM. Olson et al. (123) described two missense mutations in the ACTC gene in two small unrelated families with DCM. Subsequently, ACTC mutations were found in several families with HCM (119). The pathophysiological basis for differences in phenotypic expression of ACTC mutations has not been established. It has been proposed that the location of mutations in distinct functional domains of the ACTC molecule might account for these differences: HCM-causing mutations identified to date have been located in regions that are involved in actin-myosin interaction and force generation, whereas DCM-causing mutations have been located in the immobilized end of the actin molecule at the Z line that is thought to be involved in transmission of force from the sarcomere to the extrasarcomeric cytoskeleton.

### 3. β-Myosin heavy chain gene (MYH7)

The MYH7 gene encodes the sarcomere thick filament protein β-MHC. Characteristics of the MYH7 gene have been described in section mB2. The chromosome 14q12 DCM locus was identified by Kamisago et al. (67) in linkage analysis performed in a multigeneration family characterized by early-onset (<25 years age) DCM in ~50% of affected members. The MYH7 gene, associated previously with HCM, was located within the genetically determined disease interval. Subsequent screening of MYH7 in probands from the index family and from 20 additional families with DCM revealed 2 missense mutations: one mutation in the large linked family and a second mutation in a small unlinked family (67). The phenotype of the second family was also characterized by an early onset of ventricular dilation and dysfunction, including the sudden death of an infant aged 2 mo. HCM-causing MYH7 mutations have been clustered predominantly in four regions of the molecule and are thought to perturb actin-myosin interaction and force generation. One of the two DCM-causing MYH7 mutations (Ser532Pro), located in a conserved α-helical structure of the lower 50-kDa myosin domain, is predicted to disrupt actin-myosin binding. The other mutation (Phe764Leu), located in a “hinge” region between the myosin head and neck, is predicted to alter the magnitude or polarity of cross-bridge movement and hence the efficiency of contraction.

### 4. cMyBP-C gene (MYBPC3)

The MYBPC3 gene encodes the sarcomere thick filament protein cMyBP-C. Characteristics of the MYBPC3 gene have been described in section mB3. The MYBPC3 gene was selected as a potential candidate gene for DCM subsequent to the finding that a number of HCM-causing genes could also cause DCM. One missense MYBPC3 mutation has recently been reported in an individual with DCM (134).

### 5. Cardiac troponin T gene (TNNT2)

The TNNT2 gene encodes the sarcomere thin filament protein cardiac troponin T. Characteristics of the TNNT2 gene have been described in section mB4. The TNNT2 gene was selected as a potential candidate gene for DCM due to its previous association as a disease gene for HCM. The TNNT2 gene was screened by Kamisago et al. (67) in 19 probands from families with DCM in whom MYH7 mutations had not been identified. A deletion in cardiac troponin T (ΔLys210) was identified in two unrelated families in this cohort. No mutations in the genes encoding cardiac troponin I or α-tropomyosin were found. HCM-causing TNNT2 mutations have been located in both the NH₂- and COOH-terminal domains of cardiac troponin T. The DCM-causing TNNT2 deletion was located in a COOH-terminal region (residues 207–234) implicated in Ca²⁺-sensitive troponin C binding. Because none of the HCM-causing mutations identified has been found in this region, it has been suggested that the pathophysiological mechanisms responsible for HCM and DCM may differ.

### 6. α-Tropomyosin gene (TPM1)

The TPM1 gene encodes the sarcomere thin filament protein cMyBP-C. Characteristics of the TPM1 gene have been described in section mB7. The TPM1 gene was selected as a potential candidate gene for DCM due to its previous association as a disease gene for HCM. The TPM1 gene was screened in 350 individuals with familial and sporadic DCM. Two missense mutations were identified in probands with familial disease (122). HCM-causing

Physiol Rev • VOL 82 • OCTOBER 2002 • www.prv.org
TPM1 mutations have been found to occur in regions of the molecule that would be expected to impair troponin T binding and Ca²⁺ sensitivity and, hence, force generation. In contrast, the two DCM-causing TPM1 mutations could be predicted to alter the surface charge of α-tropomyosin, which might be expected to influence the structural integrity of the α-tropomyosin filament or impair its interaction with actin.

7. Titin gene (TTN)

The TTN gene encodes the giant sarcomeric cytoskeletal protein titin. Characteristics of the TTN gene have been described in section III.B.11. The 2q31 locus had been identified by linkage analysis in a multigeneration family with DCM (147). Screening of more than 340 exons of the TTN gene in 2 large families that mapped to this locus resulted in the finding of a two novel mutations: a 2-bp insertion that caused a frameshift and premature stop codon predicted to truncate the titin protein and a missense mutation in an immunoglobulin sequence at the Z-disc/I-band transition zone (44).

8. Desmin gene (DES)

The DES gene encodes desmin, a cytoskeletal protein that belongs to the intermediate filament family. Desmin is a muscle-specific protein that is expressed in all muscle tissues. Desmin is one of the earliest known myogenic markers to be expressed in cardiac and skeletal muscle, suggesting that it might play a modulating role in myogenic commitment and differentiation. In mature muscle, the desmin intermediate filament network is thought to contribute to force transmission from the sarcomere and resistance to mechanical stress. The DES gene is a single-copy gene comprised of 8.4 kb of genomic DNA with 9 exons that encode a protein of 470 amino acids. The structure of the desmin protein resembles other intermediate filaments, with a central α-helical coiled-coil rod domain flanked by nonhelical NH₂-terminal head and COOH-terminal tail domains.

Missense mutations in the DES gene have been found to cause desmin-related myopathy, a familial disorder characterized by skeletal myopathy, cardiac conduction-system abnormalities, restrictive cardiomyopathy, and intracytoplasmic accumulation of desmin-reactive deposits. These DES mutations have been clustered in a highly conserved COOH-terminal rod region. On the basis of its expression pattern and the identification of mutations in desmin-related myopathy, the DES gene was considered to be a possible candidate gene for DCM. A missense mutation (Ile451Met) in the DES gene was recently reported in one family with autosomal dominant DCM without conduction-system disease or skeletal myopathy. This mutation was located in the desmin tail domain (80). Mice completely lacking desmin exhibit marked disruption of myofibril organization and myocyte degeneration with progressive ventricular dilation and impaired systolic contraction (104). Observations in an affected family (80) and in desmin knockout mice suggest that a reduction or absence of normal desmin function can result in DCM. The significance of cytoplasmic desmin deposits in desmin-related myopathies is not clear, since these may occur in the absence of DES mutations and accumulations of other myofibrillar proteins may concurrently be present. The mechanism by which some DES mutations cause restrictive cardiomyopathy and others cause DCM is unknown.

9. δ-Sarcoglycan gene (SGCD)

The SGCD gene encodes the sarcolemmal transmembrane glycoprotein, δ-sarcoglycan, one of four proteins (α, β, γ, δ) that associate in stoichiometrically equal proportions to form the sarcoglycan complex, a component of the dystrophin-associated glycoprotein transmembrane complex. δ-Sarcoglycan is expressed in striated and smooth muscles, with the highest levels of expression found in skeletal and cardiac muscle. The SGCD gene is comprised of at least 100 kb of genomic DNA with 8 exons encoding a protein of 290 amino acids. δ-Sarcoglycan has a small intracellular domain, a single transmembrane hydrophobic domain, and a large extracellular COOH terminus.

Mutations in the SGCD gene were first reported in families with autosomal recessive limb girdle muscular dystrophy (LGMD2F) (114). With the exception of electrocardiographic evidence of left ventricular hypertrophy in one individual, no cardiac abnormalities have been reported in this disorder. A deletion of the first exon of the SGCD gene, resulting in the complete absence of δ-sarcoglycan protein, was found to be the cause of autosomal recessive cardiomyopathy observed in the Syrian hamster (115). These animals exhibit histological features of muscular dystrophy with progressive myocardial necrosis, ventricular hypertrophy and dilation, and premature death due to heart failure. On the basis of these findings, SGCD gene mutations in human DCM were sought. In a series of individuals with familial and sporadic DCM, one missense mutation in the SGCD gene was found in a family with autosomal dominant DCM and a high prevalence of sudden death at an early age, and 2 different mutations that both resulted in a deletion of the Lys238 codon were identified in 2 of 50 individuals with sporadic DCM who had presented with heart failure at <20 yr of age. None of these individuals had clinical evidence of skeletal myopathy (168).

10. Metavinculin gene (VCL)

The VCL gene encodes the cytoskeletal proteins vinculin and its splice variant, metavinculin. Vinculin is ubiqui

uitously expressed, while metavinculin is coexpressed with vinculin exclusively in cardiac, skeletal, and smooth muscle. In the heart, vinculin and metavinculin are localized to subsarcolemmal costameres where they interact with α-actinin, talin, and γ-actin to form a microfilamentous network linking the cytoskeleton to the sarcolemma. Vinculin and metavinculin are also present in adherens junctions in intercalated discs, where they participate in cell-cell adhesion. The specific function of metavinculin in the heart is unknown. The VCL gene is comprised of >75 kb of genomic DNA and has 22 exons that encode the 2 isoforms vinculin (1,066 amino acids) and metavinculin (1,134 amino acids). The larger isoform, metavinculin, is generated by alternate splicing of exon 19 and has an additional 68 amino acids in the COOH-terminal region. Cross-species analyses have shown that the first half of the inserted sequence is variable, whereas the second half is highly conserved. Vinculin and metavinculin are comprised of a globular head and a rod-shaped tail. Four to six single molecules assemble into multimers by head-to-tail and tail-to-tail interactions. The head region contains talin-binding domains, while the tail region interacts with γ-actin and paxillin.

Because vinculin is an “immediate early response gene,” increased levels of expression may be induced by a variety of growth-inducing factors. In one study, vinculin levels were increased 1.2-fold in heart tissue explanted from 19 patients with end-stage heart failure due to nonischemic DCM (53). In contrast, absent metavinculin protein and disorganized intercalated discs were found in 1 of 23 individuals with idiopathic DCM (85). Vinculin but not metavinculin mRNA was detected in the diseased heart tissue. These observations suggest that altered vinculin and metavinculin expression might be a consequence of, or contribute to, heart failure. Very recently, 2 VCL mutations were identified in a cohort of 350 individuals with familial and sporadic DCM: 1 missense mutation and 1 3-bp deletion. Heart tissue from the individual with an Arg975Trp VCL mutation showed disruption of intercalated discs. In vitro assays demonstrated that both the Arg975Trp substitution and the Leu-95 deletion significantly altered metavinculin-mediated cross-linking of actin filaments (120).

### 11. Lamin A/C gene (LMNA)

The LMNA gene encodes the intermediate filament proteins lamins A and C. The lamins are located in the nuclear lamina, at the nucleoplasmic side of the inner nuclear membrane. Lamins are classified broadly as A type and B type. The A-type lamins include four transcripts produced by alternate splicing of the LMNA gene: lamins A, C, AΔ10, and C2. The B-type lamins include lamins B1 and B2, transcribed from the LMNB1 and LMNB2 genes, respectively. B-type lamins are expressed throughout development and are the only lamins present in the embryo. A-type lamins are expressed in differentiated cells in a wide range of tissues. Lamins A, C, and B2 are expressed in heart and skeletal muscle. The functions of lamins A and C have not been determined precisely. The lamins are presumed to have a structural role in maintaining the integrity of the nuclear membrane. In dividing cells, the binding of lamins to M-phase chromatin is important for reassembly of nuclear membranes in daughter cells during mitosis. Lamin binding to interphase chromatin may maintain chromatin architecture and contribute to regulation of the cell cycle and gene transcription (150). The coding sequence of the LMNA gene spans 24 kb of genomic DNA with 12 exons. The principal transcripts, lamins A and C, have identical sequence for the first 566 amino acids (exons 1–10) but differ in their COOH-terminal regions, which contain 98 and 6 unique amino acids, respectively. Lamins A and C consist of a central rod domain flanked by globular head (NH₂-terminal) and tail (COOH-terminal) domains. The rod domain is formed by an α-helical coiled-coil dimer.

Nineteen LMNA mutations have been found in families with autosomal dominant DCM with conduction-system disease (34, 42). These include missense mutations, deletions, and frameshift mutations that result in premature termination codons. These LMNA mutations have been located predominantly in the central rod domain common to lamins A and C. Families with autosomal dominant DCM with conduction-system disease caused by mutations in the lamin A/C rod typically have no clinical evidence of skeletal myopathy and normal creatine kinase levels. In one family with a mutation in the lamin C-specific region of the tail domain, there were no clinical features of skeletal myopathy, but creatine kinase levels were mildly elevated in some family members (34).

Over 30 LMNA mutations have also been found to cause two forms of autosomal dominant skeletal myopathy (Emery-Dreifuss muscular dystrophy and limb girdle muscular dystrophy type 1B) (15, 42, 109) and familial partial lipodystrophy (20, 146). Emery-Dreifuss muscular dystrophy is characterized by a childhood onset of contractures of the elbows, Achilles tendons, and postcervical muscles, together with slowly progressive wasting and weakness of humeroperoneal muscles. In later life, affected individuals may develop conduction-system disease, most commonly, heart block. Left ventricular dilation and contractile dysfunction occur infrequently. Creatine kinase levels are typically elevated. The clinical manifestations of the autosomal dominant form of Emery-Dreifuss muscular dystrophy are similar to those of the X-linked form, which results from mutations in the gene encoding emerin, a protein in the inner nuclear membrane. Mutations causing autosomal dominant Emery-Dreifuss muscular dystrophy were first identified in the globular head and tail domains of lamins A and C (15).

Recent studies have found that Emery-Dreifuss muscular dystrophy-causing mutations may also be located in the rod domain of lamins A and C. Autosomal dominant limb girdle muscular dystrophy type 1B is a slowly progressive skeletal myopathy without contractures and with age-related conduction disturbances. Mutations causing this disorder have been located in the rod and tail domains of lamins A and C (109). Familial partial lipodystrophy is an autosomal dominant disorder characterized by regional fat loss in the peripubertal period as well as insulin resistance. Mutations causing partial lipodystrophy have all been located in the tail domain of lamins A and C with mutation hot spots at codons 482 and 486 (20, 146). Cardiac and skeletal muscle involvement have not been reported in any of the families with this disorder.

frequent than in Duchenne muscular dystrophy. In general, individuals with Duchenne muscular dystrophy have frameshift or nonsense *DMD* mutations that result in premature termination of translation and a reduction or absence of dystrophin protein. Individuals with Becker muscular dystrophy usually have *DMD* deletions that result in truncation or reduced levels of expression of dystrophin.

*DMD* mutations have also been found to cause X-linked DCM, a disorder in which affected males present with congestive cardiac failure in adolescence or early twenties with rapid progression to death within 1–2 yr (167). Although creatine kinase levels may be elevated, clinical evidence of skeletal myopathy is absent. Female carriers may develop a slowly progressive late-onset DCM. *DMD* gene defects associated with X-linked DCM have been clustered in two regions: at the NH₂-terminal end (including the promoter, first exon, and a “hinge” region between NH₂-terminal actin-binding domain and the central rod domain) and mid-rod domain exons. Point mutations, deletions, insertions, and inversions have been reported. A nonsense mutation subsequently deleted by alternate splicing has been described (36). These *DMD* mutations have resulted in quantitative and qualitative abnormalities of dystrophin protein (36, 111). In one study, deletions in NH₂-terminal exons were associated with a more severe clinical course than deletions in exons in the rod domain (5).

12. **Dystrophin gene (*DMD*)**

The *DMD* gene encodes dystrophin, a large cytoskeletal protein that is expressed predominantly in skeletal, cardiac, and smooth muscle, and to a lesser extent, the brain. Dystrophin interacts with both cytoskeletal actin and the dystrophin-associated glycoprotein complex in the sarcolemma. Dystrophin is thought to contribute to intracellular organization, force transduction, and membrane stability. The *DMD* gene is the largest known gene (2.4 Mb). It contains 79 exons that encode a protein of 3,685 amino acids. Transcription regulation of the *DMD* gene is complex, with at least eight independent, tissue-specific promoters. Alternate splicing gives rise to a range of different transcripts and protein isoforms. Four domains in the dystrophin protein have been identified: a 240-amino acid NH₂-terminal domain that is homologous to the actin-binding domain of α-actinin, a large rod-shaped domain comprised of 25 triple-helical segments similar to the repeat domains of spectrin, a cysteine-rich segment similar to the COOH-terminal domain of *Dictyostelium* α-actinin, and a 420-amino acid COOH-terminal domain with no homology to known proteins. The dystrophin protein forms a rod shape and is ~150 nm in length. Mutations in the *DMD* gene were first identified in Duchenne and Becker muscular dystrophies (52, 72). Duchenne muscular dystrophy is an X-linked recessive disorder, affecting 1 in 3,500 live-born males. Affected males present in childhood with gait disturbances; develop progressive weakness of pelvic femoral, shoulder girdle, neck flexor, and trunk muscles; and may be wheelchair-bound by adolescence. Pseudohypertrophy of the calf muscles and markedly raised creatine kinase levels are characteristic features. Cardiomyopathy and conduction abnormalities may occur late in the disease. The severity of cardiac disease is unrelated to the severity of the skeletal muscle disease. Becker muscular dystrophy is a milder, allelic form. Affected males present later in life and have a milder course. Cardiac involvement is less

13. **Tafazzin gene (*G4.5*)**

The *G4.5* gene encodes the tafazzins, a group of proteins of unknown function with no similarities to other proteins. They are expressed ubiquitously with high levels present in cardiac and skeletal muscle. *G4.5* is a single-copy gene comprised of ~11 kb of genomic DNA with 11 exons. Up to 10 different mRNAs can be produced by alternate splicing, resulting in tafazzin proteins ranging from 129 to 292 amino acids in length, that differ at the NH₂-terminal and central regions (exons 5–7). Two putative functional domains have been identified: a highly hydrophobic segment of 30 residues at the NH₂ terminus that might act as a membrane anchor and a hydrophilic segment in the central region that might form an exposed loop interacting with other proteins.

Mutations in *G4.5* were first associated with Barth’s syndrome, an X-linked disorder characterized by infantile-onset DCM, skeletal myopathy, short stature, neutropenia, and abnormal mitochondria (10). Histopathological studies of the ventricular myocardium variably show hypertrophy, dilation, and/or endocardial fibroelastosis. On electron microscopic examination, mitochondria typically have concentric, tightly packed cristae with occasional inclusion bodies. Increased urinary levels of 3-methylglutaconic acid and 2-ethylhydracrylic acid may be

present. G4.5 mutations reported to cause Barth's syndrome include splice site mutations, insertions, deletions, and nonsense and missense mutations. These mutations have been located throughout the G4.5 gene. Mutations in G4.5 have subsequently been associated with three other myocardial disorders: X-linked endocardial fibroelastosis (24), an X-linked form of noncompaction of the left ventricle (13), and X-linked infantile DCM (24). X-linked endocardial fibroelastosis is characterized by DCM with infantile death, neutropenia, and mitochondrial abnormalities. The same missense mutation in a conserved region of exon 10 of the G4.5 gene was found in two unrelated families (24). X-linked noncompaction of the left ventricle is a recessive disorder with infantile DCM, cardiac arrhythmias, and sudden death. Cardiac pathology typically shows numerous prominent trabeculations and deep intertrabecular recesses in the ventricular myocardium as well as ventricular hypertrophy, dilation, and endocardial fibroelastosis. A missense mutation in a conserved region of exon 8 of the G4.5 gene has been reported in one family (13). X-linked DCM is a recessive disorder in which male infants develop DCM and sudden death. A deletion in exon 8 of the G4.5 gene was identified in one family, resulting in the complete absence of all tafazzin proteins (24).

### 14. Mitochondrial DNA

Cardiac myocytes contain numerous mitochondria, scattered throughout the cytoplasm between myofibrils, that have an essential role in energy generation and contractile function. Mitochondria contain their own DNA, transcription, and translation systems. The mitochondrial genome covers 16.6 kb and contains 13 structural genes encoding proteins involved in oxidative phosphorylation, 22 transfer RNAs, and 2 ribosomal RNAs. Mitochondrial DNA is maternally inherited. A large number of mitochondrial proteins are not encoded by mitochondrial DNA but by nuclear DNA. These include proteins involved in enzyme complexes, mitochondrial DNA replication and transcription, as well as structural proteins and transport proteins in the mitochondrial membranes. These nuclear genes may have autosomal dominant, autosomal recessive, or sex-linked patterns of inheritance.

Point mutations and deletions in mitochondrial DNA have been found in a number of multisystem disorders that may have associated cardiac abnormalities, such as adult-onset hypertrophy and childhood-onset DCM (89). In particular, mitochondrial abnormalities have been implicated in a fatal infantile form of DCM (153). The role of mitochondrial DNA mutations in the pathogenesis of adult-onset DCM is not entirely clear. Although several studies have shown that point mutations and deletions in mitochondrial DNA occur more frequently in individuals with DCM than in control subjects, it is not clear whether these abnormalities exert a primary effect or occur as a secondary consequence of established DCM (4, 81, 90). Primary mitochondrial DNA mutations might directly cause DCM or increase susceptibility to myocardial damage or stress. It is notable that mitochondrial DNA has a high rate of spontaneous mutation and that deletions occur with normal aging and with alcohol and ischemic injury. It is feasible that the presence of congestive heart failure might accelerate these processes. Progressive mitochondrial impairment would be expected to significantly impair myocardial energy generation and further exacerbate contractile dysfunction.

### 15. Additional disease genes

Further additions to this growing list of DCM disease genes can be expected. Of the 15 chromosomal loci mapped for autosomal dominant DCM ± conduction-system disease, 10 disease-causing genes have been identified. It is important to note that in 9 of these 10 genes, only a very small number of mutations (1–3) in each gene have been reported. Furthermore, current data suggest that these known disease genes may only account for a relatively small proportion of all cases of DCM. For example, Olson et al. (123) found only 2 mutations in the ACTC gene in a cohort of 350 patients with familial and sporadic DCM. Three studies in other populations with autosomal dominant DCM have failed to identify further ACTC mutations: 136 Japanese patients, of whom 30 had familial and 106 had sporadic disease (159); 86 European patients, of whom 43 had familial and 43 had sporadic disease (163); and 57 South African (56% black) patients (93). In the cohort of 350 patients screened for ACTC mutations, Olson and colleagues (120, 122) subsequently identified 2 mutations in each of the TPM1 and VCL genes, respectively. No mutations in the DES gene were found in 63 European patients with autosomal dominant DCM, of whom 41 had familial and 22 had sporadic disease (163). These observations suggest that either a large number of different genes may cause DCM or that important disease-causing genes remain to be discovered. Linkage studies in large families with DCM will continue to be an invaluable method for identification of DCM disease genes. A major limitation of the candidate gene approach alone, without linkage or functional analyses, is that apparent mutations may in fact be rare polymorphisms.

In the past few years, there has been a proliferation of mouse models generated by overexpression or deletion of a variety of cardiac proteins, including sarcomeric, cytoskeletal, and cell signaling proteins, that have exhibited a DCM phenotype. These studies demonstrate that perturbation of the normal function of diverse proteins contributing to cardiac structure and function may cause DCM. Until disease-causing mutations are identified, how-

ever, the relevance of these models to human DCM remains speculative.

### C. Pathophysiological Mechanisms

#### 1. "Defective force transmission" hypothesis

Subsequent to the finding that HCM was caused predominantly by mutations in genes encoding sarcomere proteins, investigators sought to identify an equivalent unifying paradigm for the pathogenesis of DCM. The "defective force transmission" hypothesis has been widely promoted. This hypothesis is based on the premise that the cytoskeleton provides an intracellular scaffolding that is important for transmission of force from the sarcomere to the extracellular matrix, and for protection of the myocyte from external mechanical stress. Hence, defects in cytoskeletal proteins are presumed to predispose to DCM by reducing force transmission and/or resistance to mechanical stress. Identification of mutations in the genes encoding cardiac actin, desmin, metavinculin, the δ-sarcoglycan subunit of the dystrophin-associated glycoprotein complex, and dystrophin have provided support for the defective force transmission hypothesis for DCM (Fig. 2). Cardiac dysfunction may be related to effects of the protein itself or from abnormal interactions with other proteins. For example, a dystrophin gene mutation has been described that encodes a truncated protein with an abnormal conformation that alters the assembly of the sarcoglycan complex (36). It has also been demonstrated that the absence of the δ-sarcoglycan component of the sarcoglycan complex results in loss of the entire complex (21). Interestingly, in δ-sarcoglycan-null mice, it has been proposed that cardiac and skeletal myopathy may not result directly from the loss of the sarcoglycan complex in myocytes but may be mediated by ischemic damage due to a sarcoglycan deficit in vascular smooth muscle (21).

At least two other cytoskeletal proteins are potential candidate genes for DCM. The muscle LIM protein MLP promotes protein assembly along the actin cytoskeleton. MLP deficiency has been associated with the onset and progression of heart failure. In mice, the absence of MLP results in severe DCM and heart failure (3). In individuals with DCM, reduced levels of MLP expression in myocardial tissue have been demonstrated (181). In the latter case, it is not known, however, whether MLP deficiency is the cause of, or a consequence of, heart failure. Recently, MLP gene mutations have been found in individuals with HCM (39). α-Actinin and various other constituents of the dystrophin-associated glycoprotein complex have been implicated in limb girdle muscular dystrophies but have not as yet been shown to cause DCM.

#### 2. "Defective force generation" hypothesis

The identification of mutations in the genes encoding the two sarcomere proteins β-MHC and cardiac troponin T first raised the alternative hypothesis that defects in force generation could be responsible for familial DCM (67) (Fig. 2). The β-MHC gene mutations were predicted to disrupt actin-myosin binding and cross-bridge function, respectively, while the cardiac troponin T mutation was located in the Ca²⁺-sensitive troponin C binding domain. Although DCM can be a late complication of chronic HCM, individuals with these β-MHC gene and cardiac troponin T gene mutations appeared to have a primary DCM with no clinical or histological evidence of antecedent HCM. Potential mechanism(s) by which sarcomere protein gene mutations might cause either HCM or DCM are discussed below.

#### 3. Role of nuclear lamina proteins

The discovery that mutations in the LMNA gene caused DCM raised intriguing questions about the role of nuclear lamina proteins in the heart and in the pathogenesis of cardiac and skeletal myopathies. Currently, there are no data to suggest that lamins A and C participate directly in either force transmission or force generation in cardiac myocytes. Although connections between lamin B and the desmin intermediate filament network have been proposed, no interactions between lamins A and C and any of the cytoskeletal or sarcomeric proteins have been described.

One possibility is that LMNA mutations might promote DCM by disruption of nuclear function. Mutations that alter the charge or hydrophobicity of lamins A and C might alter dimer formation, head-to-tail assembly of dimers, or lateral assembly of complex filaments. Derangement of nuclear structure might result from impaired lamin filament assembly or from other altered properties of mutant lamin proteins. Derangement of nuclear function might occur as a generalized consequence of altered nuclear structure or specific lamin dysfunction. Because lamins bind to chromatin and several transcription factors, LMNA mutations might result in altered transcription regulation. Potential downstream target genes regulated by LMNA have not been identified. It is notable that DCM, skeletal myopathies, and partial lipodystrophy caused by LMNA mutations are not present at birth but develop later in life. These observations would be consistent with the hypothesis that mutant lamins might confer an increased susceptibility to degenerative processes such as apoptosis or biomechanical stress (e.g., repetitive contractions, high energy requirements). Dysregulated apoptosis has been implicated in several cardiovascular disorders, including heart failure (113), cardiac conduction defects (62), myocardial infarction (160), and arrhythmogenic right ventricular dysplasia (86). Whether

apoptosis has a primary or secondary role in these disorders has not been established.

Sullivan et al. (152) recently characterized homozygous null *LMNA* mice. By 3–4 wk of age, these mice exhibited clinical and histological features of skeletal myopathy, similar to Emery-Dreifuss muscular dystrophy. Myocardial tissue specimens showed patchy myocyte degeneration and atrophy. Histological analyses also showed thymic atrophy, a reduction of spleen size, and absent white fat. Heterozygous *LMNA* knockout mice were phenotypically normal. These mice will provide a useful model to further evaluate the role of lamins A and C in the heart. Although the majority of reported *LMNA* mutations have been missense mutations, if the major consequence of mutant peptide is absence of normal lamin filament assembly, then missense alleles may be functionally equivalent to null alleles. Hence, observations in *LMNA* null mice may be directly relevant to studies of the pathogenesis of DCM.

### 4. Role of altered gene expression in progression of DCM

RNA microarray analyses of myocardial tissue from patients with DCM have shown that expression of a large number of genes may be upregulated or downregulated as a secondary response to cardiac failure. This molecular response is comprised of factors that may exacerbate progression of heart failure and as well as factors that form part of compensatory mechanisms. In one study, genes with altered expression profiles were clustered into several groups: 1) cytoskeletal and myofibrillar genes (e.g., striated muscle LIM protein-1, myomesin, nonsarcomeric regulatory MLC-2, β-actin), 2) genes responsible for degradation and disassembly of myocardial proteins (e.g., α₁-antichymotrypsin, ubiquitin, gelsolin), 3) genes involved in metabolism (e.g., ATP synthase α-subunit, succinate dehydrogenase flavoprotein subunit, aldose reductase), 4) genes responsible for protein synthesis (elongation factor-2, eukaryotic initiation factor-4AII), and 5) genes encoding stress proteins (*αβ*-crystallin and *μ*-crystallin) (179). Cumulative mutations in mitochondrial genes with a progressive myocardial energy deficit may also contribute significantly to cardiac decompensation (4, 81, 90).

### 5. Variable phenotypic expression of sarcomere protein gene mutations

It is notable that mutations in six genes encoding sarcomere proteins have now been shown to result in either HCM or DCM. The factors that determine the phenotypic expression of sarcomere protein gene mutations have not as yet been identified. One possibility is that the HCM-causing and DCM-causing mutations may occur in distinct functional domains of the encoded proteins. In support of this proposal, it has been noted that HCM-causing mutations in cardiac actin occur in regions involved in force generation, whereas DCM-causing mutations have been identified in regions putatively involved in force transmission (123). A DCM-causing mutation, but not HCM-causing mutations, was found in the troponin C-binding domain of cardiac troponin T, suggesting that these two disease phenotypes might arise by different pathophysiological mechanisms (67). In contrast, both HCM- and DCM-causing mutations in the β-MHC gene occurred in regions involved in actin-myosin interaction and force generation (67). These latter observations suggest a second possibility, that is, whether a mutation results in HCM or DCM is determined by the extent of contractile deficit incurred. If myocardial hypertrophy is a compensatory response to a certain degree of contractile dysfunction, it is feasible that this compensatory response might be inadequate to rescue more severe dysfunction and DCM may ensue. A “dose effect” of mutant protein has been observed in mouse models. For example, heterozygous mice expressing Arg403Gln α-MHC develop HCM, whereas homozygous mice develop rapidly progressive DCM and neonatal death (33, 35, 41). Similarly, heterozygous mice expressing a truncated cMyBP-C exhibit HCM; homozygous mice have a normal life span but develop DCM (95, 96). Further studies are required to determine the correlation between the “dose” of mutant protein and quantitative differences in sarcomere function. Although these observations in mice suggest that there might be a continuum between the extent of sarcomere dysfunction and the presence of hypertrophy or dilation, a contrasting hypothesis is that independent pathways are triggered that lead either to HCM or DCM. If this is the case, then determining the components of each pathway and determining which of these pathways is activated may have important implications for the therapeutic management of individuals with sarcomere protein gene mutations.

### 6. Mechanisms of tissue specificity

Mutations in the genes encoding desmin, lamins A and C, δ-sarcoglycan, and dystrophin have been found to cause DCM without skeletal myopathy, as well as skeletal myopathy with or without cardiac disease. Various possible explanations for these tissue differences in phenotypic expression have been proposed. First, if different domains within the protein have distinct functions in different tissues, then the location of the mutation within the peptide might be important. Observations supporting this proposal include the finding of a DCM-causing mutation in the *DES* gene in the desmin tail region (80) with a skeletal myopathy-causing mutation located in the rod (25, 46). In the *LMNA* gene, initial observations suggested that DCM-causing mutations occurred predominantly in

the lamin A/C rod (34), while mutations associated with Emery-Dreifuss muscular dystrophy were localized in the globular head and tail regions (15). Subsequent studies have shown, however, that mutations causing Emery-Dreifuss muscular dystrophy may also occur throughout the rod region (42). The importance of location per se is challenged by observations of marked phenotypic variability within family members bearing the same mutation. For example, a Leu-320 deletion in the LMNA gene in one family resulted in DCM with conduction defects; some family members had clinical features of skeletal myopathy, whereas others did not (18). A second possibility is that tissue-specific differences in gene regulation might be present. This may result from factors such as differing effects of a mutation on tissue-specific promoter sequences or tissue differences in promoter activity. In one family with X-linked DCM associated with a deletion in the muscle-specific promoter, differences between cardiac and skeletal muscle dystrophin expression were attributed to upregulation of nonmuscle isoforms in the skeletal muscle but not cardiac tissue (110). A third possibility is that other genetic factors may modify the phenotypic expression of a mutant protein. Finally, it is possible that susceptibility to environmental factors may differ between tissues. For example, it has been proposed that muscle tissues bearing mutant desmin may have increased susceptibility to biomechanical stress (25). Further studies are required in affected individuals and in mouse models to determine whether quantitative or qualitative differences in mutant protein are present in cardiac and skeletal muscle. These studies might provide a clue for subsequent mechanistic analyses.

### 7. Conclusions

Collectively, current data suggest that mutations in a number of individual genes can cause DCM by multiple distinct pathophysiological mechanisms. In adult-onset forms of familial DCM, affected individuals appear phenotypically normal at birth and subsequently acquire disease manifestations in later life. These observations suggest that the mutant proteins do not prevent normal cardiac development. The progressive onset of disease may result from increased susceptibility to factors such as biomechanical stress, apoptosis, or other degenerative processes. Given the clinical variability of autosomal dominant DCM both between and within families, it is likely that phenotypic expression of DCM disease genes can be modified by other genetic factors and/or environmental factors. In determining the role of genetic factors in the pathogenesis of DCM, it will be important to establish whether genetic variants (polymorphisms) in various genes increase susceptibility to other genetic and/or environmental factors. Finally, multiple changes in gene expression have been found to occur in response to heart failure. Identification and characterization of all the genes responsible for the onset and progression of DCM will be an important goal of future studies.

#### D. Pathophysiology of HCM and DCM: Is There a Unifying Hypothesis?

Despite more than a decade of research, as indicated above, the mechanisms by which disease-causing mutations impair cardiac function and result in either left ventricular hypertrophy or chamber dilation remain elusive. Functionally, the chief defect in DCM is impaired systolic performance, whereas in HCM it is diastolic dysfunction. That HCM is caused by mutations in genes that encode sarcomeric proteins (β-MHC; regulatory and essential MLC; cMyBP-C; troponins T, I, and C; α-tropomyosin; and actin) or intrasarcomeric cytoskeletal proteins (titin) has led to the hypothesis that the left ventricular hypertrophy in this disorder develops as a compensatory response to impaired force generation. However, this hypothesis requires further consideration, given that impaired force generation cannot be readily reconciled with the clinical finding of preserved or enhanced systolic function. Along the same lines, the finding that single gene defects that cause DCM involve extrasarcomeric cytoskeletal proteins (desmin, δ-sarcoglycan, metavinculin, and dystrophin) has led to the suggestion that this disorder results from defective force transmission. This follows given that efficient myocyte contraction necessitates cell shortening, i.e., efficient mechanical coupling, and that the sarcomere is anchored to the cell wall by cytoskeletal proteins. Again, however, no experimental evidence has been generated to support this hypothesis, and, as yet, the functional consequences of DCM-causing cytoskeletal protein mutations have not been investigated. Furthermore, it is now known that DCM can also be caused by mutations of sarcomeric and intrasarcomeric proteins (actin, β-MHC, MyBP-C, cardiac troponin T, α-tropomyosin, and titin) and even nuclear proteins (lamins A and C). Involvement of the former has been suggested to indicate that some proteins, such as cardiac actin, may not only function as important components of the sarcomere, critical to force generation, but may also function as intrasarcomeric cytoskeletal proteins that contribute to force transmission. Whether involvement of all DCM-causing sarcomeric proteins, let alone the nuclear lamins, can be reconciled on this basis remains unclear.

Very recent findings from studies of both HCM- and DCM-causing troponin mutations do, however, provide novel insights into their pathophysiology. Despite some conflicting observations, detailed above, there is generally good agreement that the functional defect associated with HCM-linked mutations in cardiac troponin T (Ile79Asn,

Arg92Gln, Phe110Ile, Glu163Lys, Glu244Asp, Arg278Cys, and Int15G1→A) (157, 177) and cardiac troponin I (Arg145Gly, Arg162Trp) (30, 158) is a sensitization of the cardiac contractile apparatus and myofibrillar ATPase activity to Ca²⁺. In contrast, a DCM-causing troponin T mutation, ΔLys210, has now been shown to result in desensitization of the Ca²⁺-force relationship, with no change in either the slope of this relationship or in maximal force generation. In addition, sensitivity of myofibrillar ATPase to Ca²⁺ is also impaired, but the stoichiometry of the three troponin components (troponin T, troponin I, and troponin C) within the sarcomere is unaltered (107). The structural basis for these alterations in Ca²⁺ sensitivity must await resolution of sarcomere assembly in atomic detail, although it is of interest that Lys-210 is in a domain of troponin T that interacts strongly and in a Ca²⁺-sensitive manner with troponin C, but very weakly with tropomyosin and troponin I. Whether the Ca²⁺ insensitivity of this troponin T mutation is due to a decrease in Ca²⁺ avidity or to a decrease in the conformational changes following Ca²⁺ binding remains unclear. Nevertheless, although maximal force generation is unaltered, given that in intact cardiac muscle the contractile apparatus is never activated beyond the half-maximal level, even small changes in Ca²⁺ sensitivity can be expected to markedly alter force generation. In terms of the DCM-causing troponin T mutation, impaired Ca²⁺ sensitivity, as with the impaired force transmission of cytoskeletal protein mutations, could well be expected to result in ventricular dilation as a compensatory response to the decrease in stroke volume. Moreover, impaired Ca²⁺ sensitivity is likely to enhance diastolic relaxation, although it is unclear if such a lusiotropic effect occurs in the in vivo setting. Together, these altered responses at the cellular level, if also applicable to other causes of DCM, could explain the impaired systolic and preserved diastolic function in this disorder. The recent finding of a progressive increase in phospholamban-mediated inhibition of Ca²⁺ uptake via the Ca²⁺ pump, in an animal model of DCM, potentially provides also a cogent explanation for progression of DCM to overt heart failure (50). This defect would reduce sarcoplasmic reticulum Ca²⁺ stores and limit availability of Ca²⁺ during systole. Together with impaired myofibrillar Ca²⁺ sensitivity, this would lead to systolic function being further depressed. In contrast to the defects in DCM, the Ca²⁺ sensitization of HCM-linked mutations is likely to promote enhanced systolic contractility but impaired cardiac muscle relaxation. The resulting diastolic dysfunction could contribute to the high incidence of sudden death, due either to ventricular arrhythmia generation or to an increase in cytostolic free Ca²⁺, the latter potentially being exacerbated by exercise- or stress-induced enhancement of cardiac sympathetic drive. Increased cytostolic free Ca²⁺, particularly during diastole, would also promote hyper-

trophy development as a result of calcineurin/NFAT/GATA4 and/or calmodulin kinase II activation.

Studies of animal models of HCM, as well as clinical disorders that mimic the HCM phenotype (e.g., Friedreich's ataxia, mitochondrial mutations, and very-long-chain acyl-CoA dehydrogenase deficiency) have been found to cause disordered cardiac energetics (either inefficient ATP utilization or generation) (11, 149). Moreover, it is likely that the resulting relative or absolute ATP depletion contributes to diastolic dysfunction, as well as to impaired Ca²⁺ uptake via the high energy-dependent pump of the sarcoplasmic reticulum (SERCA2a). Although the latter would also deplete sarcoplasmic reticulum Ca²⁺ stores and, thus, availability of Ca²⁺ for cardiac contraction, this may be offset, and systolic function maintained, by the increased Ca²⁺ sensitivity of the HCM-linked mutations.

Taken together, these recent findings suggest the exciting possibility that energy compromise and mutation-induced alterations in myofibrillar Ca²⁺ sensitivity, acting either separately or in concert, may underlie a unifying pathophysiology for both HCM and DCM (Fig. 2).

V. ARRHYTHMOGENIC RIGHT VENTRICULAR DYSPLASIA

A. Clinical Manifestations

1. Disease characteristics

Arrhythmogenic right ventricular dysplasia (ARVD) is a primary heart muscle disease characterized by myocyte loss due to necrosis and/or apoptosis, with fatty or fibrofatty replacement. The pathological process predominantly affects the right ventricle and may be focal or diffuse with progressive changes extending from the subepicardium to the endocardium, resulting in chamber dilation and wall thinning. Extensive wall thinning may give rise to a parchment appearance (Uhl anomaly). Single or multiple aneurysms of the right ventricular free wall have been reported in 50% cases. Left ventricular involvement has been found in up to 76% of cases (22). In advanced disease, ARVD may be difficult to distinguish clinically from DCM. The prevalence of ARVD has been estimated to be 1/5,000, although higher rates (4.4/1,000) have been reported in some areas of northern Italy (127, 164). ARVD is thought to be familial in at least 30% cases, with autosomal dominant inheritance observed most commonly (164). An autosomal recessive form of ARVD has been reported in association with palmoplantar keratoderma and woolly hair (Naxos disease).

2. Diagnosis of ARVD

A definitive diagnosis of ARVD may be difficult to establish using clinical criteria. The age of onset, degree

of penetrance, and clinical features vary both between
and within families. Affected individuals present most
commonly with ventricular and supraventricular arrhythmias, which may be well-tolerated or result in syncope or
sudden death. Sudden death is frequently precipitated by
exercise. ARVD is a major cause of death in the young,
occurring at a rate of 2.5%/year (164). In Italy, ARVD has
a relatively high prevalence and accounts for 20% of sudden
deaths in individuals aged less than 35 yr and over
20% deaths in competitive athletes (164). Electrocardiographic studies typically show T-wave inversion in the
precordial leads. Late potentials may be present on signal-
averaged electrocardiography. Sick sinus syndrome and
complete atrioventricular conduction block are late complications observed in some older patients with severe
disease. Heart failure may occur with disease progression
in up to 20% cases.

B. Chromosomal Loci and Disease Genes

1. Chromosomal loci

Eight chromosomal loci have been identified in families with ARVD (Table 3). Seven of these loci have been
found in autosomal dominant ARVD, on chromosomes
14q23-q24 (ARVD1) (127), 1q42-q43 (ARVD2) (128), 14q12-
q22 (ARVD3) (145), 2q32 (ARVD4) (129), 3p23 (ARVD5)
(1), and 10p12-p14 (ARVD6) (79); the seventh locus (chromosome 10q22) (99) was mapped in a family with autosomal dominant myofibrillar myopathy characterized
mainly by skeletal muscle disease with ARVD occurring in
some family members. Muscle biopsy specimens showed
myopathic changes with accumulation of desmin, dystrophin, and γ-sarcoglycan. A disease-causing gene (cardiac ryanodine receptor gene) has recently been identified at the ARVD2 locus (166). Autosomal recessive
ARVD (Naxos disease) has been mapped to chromosome
17q21, with mutations in the plakoglobin gene recently
identified (98).

TABLE 3. ARVD chromosomal loci and disease genes

| Chromosomal Locus | Gene   | Protein                |
|-------------------|--------|------------------------|
| 1q42-q43 (ARVD2)  | RYR2   | Cardiac ryanodine receptor |
| 2q32 (ARVD4)      | ?      | ?                      |
| 3p23 (ARVD5)      | ?      | ?                      |
| 10p12-p14 (ARVD6) | ?      | ?                      |
| 10q22 (myofibrillar myopathy) | ? | ? |
| 14q12-q22 (ARVD3) | ?      | ?                      |
| 14q23-q24 (ARVD1) | ?      | ?                      |
| 17q21 (Naxos disease) | JUP | Plakoglobin           |

ARVD, arrhythmogenic right ventricular dysplasia.

2. Cardiac ryanodine receptor gene (RYR2)

The RYR2 gene encodes the cardiac ryanodine receptor, which is the major Ca²⁺ release channel in cardiac
muscle. The cardiac ryanodine receptor is one of three
ryanodine receptor isoforms, each of which is encoded by
a separate gene. RYR2 is expressed in the heart and, to a
lesser extent, in the brain and gestational myometrium.
RYR1 and RYR3 are expressed in skeletal muscle and in
brain, respectively. RYR2-encoded proteins associate as homotetramers with four FK506-binding proteins
(FKBP12.6) to form a membrane-spanning complex in the
sarcoplasmic reticulum. Stimulation of voltage-sensitive
L-type Ca²⁺ channels on the outer myocardial cell membrane permits small amounts of Ca²⁺ to enter the cell.
This Ca²⁺ influx activates the ryanodine receptors in the
sarcoplasmic reticulum resulting in the release of large
quanta of Ca²⁺ that is responsible for initiating myocardial contraction. Phosphorylation of RYR2 by protein kinase A dissociates FKBP12.6 and regulates the channel
open probability (92). Administration of FK506 disinhibits
coordinated Ca²⁺ flux through the ryanodine receptor, an
effect due to its interaction with FKBP12.6. Ca²⁺ overload-induced cardiac failure has been reported in children
receiving FK506 as an immunosuppressive following
heart transplantation (6). RYR2 is comprised of 105 exons
that encode a protein of 4,967 amino acids. Several distinct transcripts of RYR2 arise by alternate splicing. The
cardiac ryanodine receptor has a large NH₂-terminal cytoplasmic domain that contains interaction sites for
FKBP12.6, ATP, and calmodulin and a COOH-terminal
transmembrane domain that is anchored to the sarcoplasmic reticulum by 4–10 hydrophobic motifs.

Four missense mutations in RYR2 have been identified in four unrelated families that map to the ARVD2
locus (166). These four mutations were located in two
clusters in the cytosolic portion of the cardiac ryanodine
receptor: one cluster at the NH₂-terminal end and a second cluster in the central FKBP12.6 interacting domain.
RYR2 mutations have also recently been reported in
families and sporadic cases of catecholamine-sensitive
polymorphic ventricular tachycardia, indicating that this
disorder is allelic to ARVD (76, 126). Catecholamine-sensitive polymorphic ventricular tachycardia is characterized by exercise-induced, bidirectional polymorphic ventricular tachycardia that occurs in the absence of
structural heart disease. RYR2 mutations in this disorder
have been found in the two clusters: in the cytosolic
central FKBP12.6-interacting domain and in the COOH-
terminal transmembrane region, respectively. The three
clusters of mutations in the RYR2 gene (proximal NH₂
terminal, central FKBP12.6 interacting domain, COOH terminal) correspond with the location of mutations in the
RYR1 gene, the skeletal muscle homolog of RYR2, that
have been reported to cause malignant hyperthermia and

central core disease (94). Malignant hyperthermia is a skeletal myopathy characterized by muscle rigidity and contractures that is triggered by administration of anesthetics and depolarizing muscle relaxants; central core disease is a nonprogressive skeletal myopathy, characterized by hypotonia and proximal muscle weakness, that is associated with an increased risk of hyperthermia during anesthetic administration.

3. Plakoglobin gene (JUP)

The JUP gene encodes the cytoplasmic protein plakoglobin. This protein is a key component of desmosomes and adherens junctions in many tissues, including the heart, skin, and hair. It belongs to the catenin family of proteins. Plakoglobin associates with desmoglein, one of the transmembrane desmosomal proteins, and is a component of the cadherin-catenin complex in fasciae adhesions. It is thought to be important for tight adhesion and signaling between cells (54). The JUP gene is comprised of ~17 kb of genomic DNA with 13 exons that encode a protein of 744 amino acids. The plakoglobin protein contains a distinct repeating amino acid repeat, the armadillo repeat. Several protein isoforms may arise by alternate splicing, not all of which have been fully characterized.

A homozygous two-base pair deletion in the JUP gene was found in all affected members of a family with autosomal recessive ARVD (Naxos disease) that mapped to the chromosome 17q21 locus (98). Family members that were heterozygous for this deletion were clinically unaffected. The deletion caused a frameshift and premature stop codon, resulting in truncation of the encoded protein. COOH-terminal truncations of plakoglobin have been shown in vitro to produce marked alterations of desmosome morphology (124).

4. Pathophysiological mechanisms

Linkage studies have shown that ARVD is a genetically heterogeneous disorder. Although only two disease-causing genes have been identified to date, it is clear that diverse pathophysiological pathways may culminate in the ARVD phenotype. Mutations in the cardiac ryanodine receptor are presumed to alter Ca²⁺ channel properties with either hyperactivation or hypersensitivity to activating levels of Ca²⁺. The subsequent imbalance of intracellular Ca²⁺ homeostasis would be expected to disrupt normal excitation-contraction coupling, promote cardiac arrhythmias, and potentially trigger apoptosis and/or necrosis. In contrast, it has been proposed that mutant plakoglobin might render cells more susceptible to the effects of mechanical stress, resulting in detachment of intercalated discs and subsequent cellular necrosis. A common feature of these two pathophysiological mechanisms is the endpoint of cell death. These putative mechanisms would both be consistent with the characteristic

myocardial histopathological findings in patients with ARVD (22). Cumulative cell death in the heart, skin, and hair might account for the progressive changes with age in these tissues. The cause of right ventricular predominance in ARVD has not been established. One possible explanation is that disease-causing genes might be expressed to a greater extent in the right rather than the left ventricle. Alternatively, differences between the right and left ventricular “environment,” such as pressure and loading conditions, may determine phenotypic expression. Further insights into the pathogenesis of this disorder will be gained with the identification and characterization of additional disease genes.

VI. RESTRICTIVE CARDIOMYOPATHY

A. Clinical Manifestations

1. Disease characteristics

Restrictive cardiomyopathy (RCM) is a myocardial disorder characterized by impaired diastolic filling of the left ventricle, with rapid early filling and slow late filling. RCM is the least common of the four categories of cardiomyopathies. It is most frequently caused by pathological conditions that stiffen the myocardium by infiltration or fibrosis, such as eosinophilic endomyocardial disease, hemochromatosis, amyloidosis, sarcoidosis, scleroderma, carcinoid, Gaucher’s disease, Fabry disease, glycogen storage disease, metastatic malignancy, anthracycline toxicity, or radiation damage. These pathophysiological processes may be confined to the heart or affect multiple organs. Several of the infiltrative diseases that result in RCM may be inherited, including familial amyloidosis, hemochromatosis, Gaucher’s disease, and glycogen storage disease.

In some individuals, RCM may occur in the absence of a precipitating condition (“idiopathic” RCM). Idiopathic RCM is not generally recognized to have a familial predisposition; however, several small families have been reported (75). The phenotypes in these families have been variable: isolated RCM, RCM with atrioventricular conduction block and skeletal myopathy, and RCM with skeletal myopathy. Autosomal dominant and autosomal recessive patterns of inheritance have been observed.

2. Diagnosis of RCM

Individuals with RCM may present with dyspnea, weakness, or exercise intolerance due to the inability to increase cardiac output by tachycardia without further reducing ventricular filling. In severe cases, there may be signs of right heart failure, including hepatomegaly, ascites, and peripheral edema. The jugular venous pressure may be elevated with an inspiratory increase in height
(Kussmaul sign). Transthoracic echocardiography may show thickening of the left (±right) ventricular walls, with variable effects on chamber diameter and contractility. A classic feature of cardiac amyloidosis is the presence of sparkling echodense spots in the myocardial walls giving a speckled “ground-glass” appearance. The mitral and aortic valve leaflets may also be thickened. Transmural Doppler interrogation typically shows increased peak E-wave velocity, producing an increase in the E/A wave ratio, and shortened E-wave deceleration time. These Doppler findings are not specific to RCM and may occur in other disorders that increase left ventricular diastolic stiffness, such as HCM and coronary artery disease. The characteristic hemodynamic feature of RCM is the “dip and plateau” or “square root” sign, consisting of a marked rapid decline in ventricular pressure at the onset of diastole (dip), followed by a rapid rise to a plateau in early diastole (plateau). These changes are reflected in atrial pressure tracings as a prominent $y$-descent (rapid atrial emptying in early diastole) followed by a rapid rise and plateau (abrupt cessation of atrial filling due to restriction). Pulmonary venous and arterial pressures and systemic venous pressures are generally elevated.

The principal differential diagnosis of RCM is constrictive pericarditis. Although these conditions have similar clinical and hemodynamic features, differentiation between them has important management implications, since patients with constrictive pericarditis may benefit from surgical intervention. Both conditions cause impairment of diastolic filling with elevation of left and right ventricular pressures and reduced stroke volume and cardiac output. In RCM, the diastolic filling pressure in the left ventricle usually exceeds that in the right ventricle by at least 5 mmHg; in constrictive pericarditis, diastolic filling pressures in the left and right ventricles are equivalent. Marked respiratory variations in left ventricular isovolumic relaxation time and peak mitral valve velocity in early diastole are observed in constrictive pericarditis, but not in RCM. Endomyocardial biopsy can be useful to detect myocardial histopathology in RCM; transthoracic or transesophageal echocardiography, computed tomography scanning, and magnetic resonance imaging may demonstrate pericardial thickening or calcification in constrictive pericarditis.

3. **Natural history of RCM**

The prognosis in RCM varies according to the underlying cause, but the majority of individuals experience progressive deterioration due to congestive cardiac failure with a high incidence of premature mortality. Symptomatic improvement in RCM may be achieved with diuretic or vasodilator therapy. Specific therapies are indicated in some cases; for example, iron chelation and immunosuppressive drugs may benefit individuals with

hemochromatosis and primary amyloidosis, respectively. Cardiac transplantation may be performed in individuals with disease pathologies localized to the heart.

### B. Chromosomal Loci and Disease Genes

#### 1. Chromosomal loci

No genetic linkage studies of large families with idiopathic RCM have been described. Only one disease-causing gene, $DES$, on chromosome 2q35, has been identified to date (46).

#### 2. Desmin gene ($DES$)

Desmin-related myopathies are a heterogeneous group of disorders characterized predominantly by skeletal myopathy and cardiac conduction disturbances with accumulation of desmin deposits in skeletal and cardiac muscle. RCM is a relatively infrequent manifestation of desmin-related myopathy. Missense mutations in the $DES$ gene have been found in several families with desmin-related myopathy (with and without RCM) (25, 46). Desmin accumulation in skeletal and cardiac muscle may be a relatively nonspecific finding and has been observed in the absence of $DES$ gene mutations. As noted in section IVB8, mutations in the $DES$ gene may cause DCM. The mechanism for these differences in cardiac phenotypic expression has not been studied. Further studies of those families with suspected familial RCM are required to explore the potential genetic basis of this disorder.

### VII. CONCLUSION

Over the last decade, substantial progress has been made in defining single gene mutations that can cause inherited cardiomyopathies. Disease-causing genes have yet to be identified for a number of mapped chromosomal loci, and it is likely that additional loci remain to be discovered. Further genetic studies of families with inherited myopathies are required to compile a comprehensive list of disease genes and mutations. The availability of data from the Human Genome Project should reduce the need for time-consuming and laborious positional cloning strategies and greatly facilitate identification of candidate genes in mapped loci. Identification of all disease-causing genes may be required to fully understand the pathogenesis of inherited cardiomyopathies.

Elucidating the normal function of these genes in the heart and the mechanisms by which gene mutations initiate disease pathways are major goals of future research studies. The generation of genetically engineered mouse models with tissue-restricted gene expression will be an invaluable resource for these studies. Dissecting out the components of intracellular signaling cascades triggered

**Physiol Rev • VOL 82 • OCTOBER 2002 • www.prv.org**by a gene defect that ultimately results in cardiac hypertrophy, dilation, and contractile dysfunction is a further research objective. The use of RNA microarray technology will facilitate identification of genes that participate in these pathways. A detailed understanding of these molecular and cellular events may lead to the identification of new targets for therapeutic intervention. In contrast to current therapies directed predominantly toward alleviation of symptoms, targeted therapies should influence the onset and progression of the disease process itself.

Disease manifestations in inherited cardiomyopathies are likely to be a complex interaction of both genetic and environmental factors. The potential role of genetic factors in modifying a disease phenotype, increasing susceptibility to secondary causes of disease and influencing responses to therapy, needs to be evaluated. Determination of the role of various environmental factors, such as exercise, alcohol, and pharmacological agents, will be important for counseling and managing patients with these disorders.

Currently, a major rate-limiting step in translating the results of molecular genetic discoveries to clinical practice is the inability to efficiently perform genotype analyses in large numbers of individuals. Since each family tends to have a unique mutation, all the known disease genes need to be evaluated in every new proband. With the use of conventional techniques, this approach is both time-consuming and expensive. The development of automated methods for performing large-scale genotyping is urgently required. The ability to perform genotype analyses will enable young individuals genetically at risk to be identified before the onset of symptoms and signs of disease. These genotype-positive, phenotype-negative individuals will be an important group to target in clinical trials of preventative pharmacological therapy. Genotype analyses will also enable clinical surveillance of the offspring of affected individuals to be rationalized, i.e., serial diagnostic investigations, counseling on life-style modifications, etc., need only be performed in genotype-positive individuals. Genotype-phenotype correlations will enable “high-risk” and “low-risk” mutations to be determined. This prognostic information will assist physicians in selection of appropriate therapies, timing of follow-up visits, pregnancy counseling, etc. Molecular genetic studies have provided a new framework for understanding the pathogenesis of inherited cardiomyopathies. Ultimately, it is hoped that this knowledge will lead to new approaches to clinical management, including diagnosis, genetic counseling, prognostic stratification, and treatment.

Address for reprint requests and other correspondence: D. Fatkin or R. M. Graham, Molecular Cardiology Unit, Victor Chang Cardiac Research Institute, Level 6, 384 Victoria St., Darlinghurst, New South Wales 2010, Australia (E-mail: d.fatkin@victorchang.unsw.edu.au or b.graham@victorchang.unsw.edu.au).

---

**REFERENCES**

1. AHMAD F, LI D, KARIBE A, GONZALEZ O, TAPSCOTT T, HILL R, WEILBAECHER D, BLACKIE P, FUREY M, GARDNER M, BACHINSKI LL, AND ROBERTS R. Localization of a gene responsible for arrhythmogenic right ventricular dysplasia to chromosome 3p23. *Circulation* 98: 2791–2795, 1998.
2. ARAD M, BENSON DW, PEREZ-ATAYDE AR, McKENNA WJ, SPARKS EA, KANTOR RJ, MCGARRY K, SEIDMAN JG, AND SEIDMAN CE. Constitutively active AMP kinase mutations cause glycogen storage disease mimicking hypertrophic cardiomyopathy. *J Clin Invest* 109: 357–362, 2002.
3. ARBER S, HUNTER JJ, Ross J, HONGO M, SANSIG G, BORG J, PERRIARD JC, CHIEN KR, AND CARONI P. MLP-deficient mice exhibit a disruption of cardiac cytoarchitectural organization, dilated cardiomyopathy, and heart failure. *Cell* 88: 393–403, 1997.
4. ARBUSTINI E, DIEGOLI M, FASANI R, GRASSO M, MORBINI P, BANCHIERI N, BELLINI O, DAL BELLO B, PILOTTO A, MAGRINI G, CAMPANA C, FORTINA P, GAVAZZI A, NARULA J, AND VIGANO M. Mitochondrial DNA mutations and mitochondrial abnormalities in dilated cardiomyopathy. *Am J Pathol* 153: 1501–1510, 1998.
5. ARBUSTINI E, DIEGOLI M, MORBINI P, DAL BELLO B, BANCHIERI N, PILOTTO A, MAGANI F, GRASSO M, NARULA J, GAVAZZI A, VIGANO M, AND TAVAZZI L. Prevalence and characteristics of dystrophin defects in adult male patients with dilated cardiomyopathy. *J Am Coll Cardiol* 35: 1760–1768, 2000.
6. ATKISON P, JOUBERT G, BARRON A, GRANT D, PARADIS K, SEIDMAN E, WALL W, ROSENBERG H, HOWARD J, WILLIAMS S, AND STILLER C. Hypertrophic cardiomyopathy associated with tacrolimus in paediatric transplant patients. *Lancet* 34: 894–896, 1995.
7. BECKER KD, GOTTSCHALD KR, HICKEY R, PERRIARD JC, AND CHIEN KR. Point mutations in human β cardiac myosin heavy chain have differential effects on sarcomeric structure and assembly: an ATP binding site change disrupts both thick and thin filaments, whereas hypertrophic cardiomyopathy mutations display normal assembly. *J Cell Biol* 137: 131–140, 1997.
8. BERS DM. Calcium fluxes involved in control of cardiac myocyte contraction. *Circ Res* 87: 275–281, 2000.
9. BING W, REDWOOD CS, PURCELL IF, ESPOSITO G, WATKINS H, AND MARSTON SB. Effects of two hypertrophic cardiomyopathy mutations in α-tropomyosin, Asp175Asn and Glu180Gly, on Ca²⁺ regulation of thin filament motility. *Biochem Biophys Res Commun* 236: 760–764, 1997.
10. BIONE S, D'ADAMO P, MAESTRINI E, GEDEON AK, BOLHUIS PA, AND TONIOLO D. A novel X-linked gene, G4.5, is responsible for Barth syndrome. *Nat Genet* 12: 385–389, 1996.
11. BLAIR E, REDWOOD C, ASHRAFIAN H, OLIVEIRA M, BROXHOLME J, KERR B, SALMON A, OSTMAN-SMITH I, AND WATKINS H. Mutations in the γ2 subunit of AMP-activated protein kinase cause familial hypertrophic cardiomyopathy: evidence for the central role of energy compromise in disease pathogenesis. *Hum Mol Genet* 10: 1215–1220, 2001.
12. BLANCHARD E, SEIDMAN C, SEIDMAN JG, LEWINTER M, AND MAUGHAN D. Altered crossbridge kinetics in the αMHC⁴⁰³⁺ mouse model of familial hypertrophic cardiomyopathy. *Circ Res* 84: 475–483, 1999.
13. BLEYL SB, MUMFORD BR, THOMPSON V, CAREY JC, PYASHER TJ, CHIN TK, AND WARD K. Neonatal lethal noncompaction of the left ventricular myocardium is allelic with Barth syndrome. *Am J Hum Genet* 61: 868–872, 1997.
14. BONNE G, CARRIER L, BERCOVICI J, CRUAUD C, RICHARD P, HAINQUE B, GAUTEL M, LABEIT S, JAMES M, BECKMANN J, WEISSENBACH J, VOSBERG HP, FISZMAN M, KOMAJDA M, AND SCHWARTZ K. Cardiac myosin binding protein-C gene splice acceptor site mutation is associated with familial hypertrophic cardiomyopathy. *Nat Genet* 11: 438–440, 1995.
15. BONNE G, DI BARLETTA MR, VARNOUS S, BECANE HM, HAMMOUDA EH, MERLINI L, MUNTONI F, GREENBERG CR, GARY F, URTIZBEREA JA, DUBOC D, FARDEAU M, TONIOLO D, AND SCHWARTZ K. Mutations in the gene encoding lamin A/C cause autosomal dominant Emery-Dreifuss muscular dystrophy. *Nat Genet* 21: 285–288, 1999.
16. BOTTINELLI R, COVIELLO DA, REDWOOD CS, PELLEGRINO MA, MARON BJ, SPIRITO P, WATKINS H, AND REGGIANI C. A mutant tropomyosin that causes hypertrophic cardiomyopathy is expressed in vivo and as—associated with an increased calcium sensitivity. *Circ Res* 82: 106–115, 1998.

17. BOWLES KR, GAJARSKI R, PORTER P, GOYTIA V, BACHINSKI L, ROBERTS R, PIGNATELLI R, AND TOWBIN JA. Gene mapping of familial autosomal dominant dilated cardiomyopathy to chromosome 10q21-q23. *J Clin Invest* 98: 1355–1360, 1996.

18. BRODSKY GL, MUNTONI F, MIOCIC S, SINAGRA G, SEWRY C, AND MESTRONI L. Lamin A/C gene mutation associated with dilated cardiomyopathy with variable skeletal muscle involvement. *Circulation* 101: 473–476, 2000.

19. BROWN MS AND GOLDSTEIN JL. A receptor-mediated pathway for cholesterol homeostasis. *Science* 232: 34–47, 1986.

20. CAO H AND HEGELE RA. Nuclear lamin A/C R482Q mutation in Canadian kindreds with Dunnigan-type familial partial lipodystrophy. *Hum Mol Genet* 9: 109–112, 2000.

21. CORAL-VAZQUEZ R, COHN RD, MOORE SA, HILL JA, WEISS RM, DAVISSON RL, STRAUB V, BARRESI R, BANSAI D, HRSTKA RF, WILLIAMSON R, AND CAMPBELL KP. Disruption of the sarcoglycan-sarcospan complex in vascular smooth muscle: a novel mechanism for cardiomyopathy and muscular dystrophy. *Cell* 98: 465–474, 1999.

22. CORRADO D, BASCO C, THIENE G, McKENNA WJ, DAVIES MJ, FONTALIRAN F, NAVA A, SILVESTRI F, BLOMSTROM-LUNDQVIST C, WLODARSKA EK, FONTAINE G, AND CAMERINI F. Spectrum of clinicopathologic manifestations of arrhythmogenic right ventricular cardiomyopathy/dysplasia: a multicenter study. *J Am Coll Cardiol* 30: 1512–1520, 1997.

23. CUDA G, FANANAPAZIR L, EPSTEIN ND, AND SELLERS JR. The in vitro motility activity of β-cardiac myosin depends on the nature of the β-myosin heavy chain gene mutation in hypertrophic cardiomyopathy. *J Muscle Res Cell Motil* 18: 275–283, 1997.

24. D'ADAMO P, FASSONE L, GEDEON A, JANSSEN EAM, BIONE S, BOLHUIS PA, BARTH PG, WILSON M, HAAN E, ORSTAVIK KH, PATTON MA, GREEN AJ, ZAMMARCHI E, DONATI MA, AND TONIOLO D. The X-linked gene G4.5 is responsible for different infantile dilated cardiomyopathies. *Am J Hum Genet* 61: 862–867, 1997.

25. DALAKAS MC, PARK KY, SEMINO-MORA C, LEE HS, SIVAKUMAR K, AND GOLDFARB LG. Desmin myopathy, a skeletal myopathy with cardiomyopathy caused by mutations in the desmin gene. *N Engl J Med* 342: 770–780, 2000.

26. D'CRUZ LG, BABOONIAN C, PHILLIMORE HE, TAYLOR R, ELLIOTT PM, VARNAVA A, DAVISON F, McKENNA WJ, AND CARTER ND. Cytosine methylation confers instability on the cardiac troponin T gene in hypertrophic cardiomyopathy. *J Med Genet* 37: e18, 2000. (http://jmedgenet.com/cgi/content/full/37/9/e18)

27. DEC GW AND FUSTER V. Idiopathic dilated cardiomyopathy. *N Engl J Med* 331: 1564–1575, 1994.

28. DE WINDT LJ, LIM HW, BUENO OF, LIANG Q, DELLING U, BRAZ JC, GLASCOCK BJ, KIMBALL TF, DEL MONTE F, HAJJAR RJ, AND MOLKENTIN JD. Targeted inhibition of calcineurin attenuates cardiac hypertrophy in vivo. *Proc Natl Acad Sci USA* 98: 3322–3327, 2001.

29. DURAND JB, BACHINSKI LL, BIELING LC, CZERNUSZEWCZ GZ, ABCHEE AB, YU QT, TAPSCOTT T, HILL R, IFEGWU J, MARIAN AJ, BRUGADA R, DAIGER S, GREGORITCH JM, ANDERSON JL, QUINONES M, TOWBIN JA, AND ROBERTS R. Localization of a gene responsible for familial dilated cardiomyopathy to chromosome 1q32. *Circulation* 92: 3387–3389, 1995.

30. ELLIOTT K, WATKINS H, AND REDWOOD CS. Altered regulatory properties of human cardiac troponin I mutants that cause hypertrophic cardiomyopathy. *J Biol Chem* 275: 22069–22074, 2000.

31. ESPOSITO G, RAPACCIUOLO A, PRASAD SVN, TAKAOKA H, THOMAS SA, KOCH WJ, AND ROCKMAN HA. Genetic alterations that inhibit in vivo pressure-overload hypertrophy prevent cardiac dysfunction despite increased wall stress. *Circulation* 105: 85–92, 2002.

32. FAMILIAL HYPERTROPHIC CARDIOMYOPATHY MUTATION DATABASE. http://www.angis.org.au/Databases/Heart/.

33. FATKIN D, CHRISTE ME, ARISTIZABAL O, MCCONNELL BK, SRINIVASAN S, SCHOEN FJ, SEIDMAN CE, TURNBULL DH, AND SEIDMAN JG. Neonatal cardiomyopathy in mice homozygous for the Arg403Gln mutation in the α cardiac myosin heavy chain gene. *J Clin Invest* 103: 147–153, 1999.

34. FATKIN D, MACRAE C, SASAKI T, WOLFF MR, PORCU M, FRENNEAUX M, ATHERTON J, VIDAILLET HJ, SPUDICH S, DE GIROLAMI U, SEIDMAN JG, AND SEIDMAN CE. Missense mutations in the rod domain of the lamin

A/C gene as causes of dilated cardiomyopathy and conduction-system disease. *N Engl J Med* 341: 1715–1724, 1999.

35. FATKIN D, MCCONNELL BK, MUDD JO, SEMSARIAN C, MOSKOWITZ IGP, SCHOEN FJ, GIEWAT M, SEIDMAN CE, AND SEIDMAN JG. An abnormal Ca²⁺ response in mutant sarcomere protein-mediated familial hypertrophic cardiomyopathy. *J Clin Invest* 106: 1351–1359, 2000.

36. FRANZ WM, MULLER M, MULLER OJ, HERRMANN R, ROTHMANN T, CREMER M, COHN RD, VOIT T, AND KATUS HA. Association of nonsense mutation of dystrophin gene with disruption of sarcoglycan complex in X-linked dilated cardiomyopathy. *Lancet* 355: 1781–1785, 2000.

37. GAO WD, PEREZ NG, SEIDMAN CE, SEIDMAN JG, AND MARBAN E. Altered cardiac excitation-contraction coupling in mutant mice with familial hypertrophic cardiomyopathy. *J Clin Invest* 103: 661–666, 1999.

38. GAUTEL M, ZUFFARDI O, FREIBURG A, AND LABEIT S. Phosphorylation switches specific for the cardiac isoform of myosin binding protein-C: a modulator of cardiac contraction? *EMBO J* 14: 1952–1960, 1995.

39. GEIER C, OEZCELIK C, PERROT A, BIT-AVRAGIM N, SCHEFFOLD T, AND OSTERZIEL KJ. Muscle LIM protein: a novel disease gene for hypertrophic cardiomyopathy? *Circulation* 104 Suppl II: II-521, 2001.

40. GEISTERFER-LOWRANCE AAT, KASS S, TANIGAWA G, VOSBERG HP, McKENNA W, SEIDMAN CE, AND SEIDMAN JG. A molecular basis for familial hypertrophic cardiomyopathy: a β cardiac myosin heavy chain missense mutation. *Cell* 62: 999–1006, 1990.

41. GEISTERFER-LOWRANCE AAT, CHRISTE M, CONNER DA, INGWALL JS, SCHOEN FJ, SEIDMAN CE, AND SEIDMAN JG. A mouse model of familial hypertrophic cardiomyopathy. *Science* 272: 731–734, 1996.

42. GENSCHEL J AND SCHMIDT HHJ. Mutations in the LMNA gene encoding lamin A/C. *Hum Mutat* 16: 451–459, 2000.

43. GEORGAKOPOULOS D, CHRISTE ME, GIEWAT M, SEIDMAN CE, SEIDMAN JG, AND KASS DA. The pathogenesis of familial hypertrophic cardiomyopathy: early and evolving effects from an α-cardiac myosin heavy chain missense mutation. *Nat Med* 5: 327–330, 1999.

44. GERULL B, GRAMLICH M, ATHERTON J, MCNABB M, TROMBITAS K, SASE-KLAASSEN S, SEIDMAN JG, SEIDMAN CE, GRANZIER H, LABEIT S, FRENNEAUX M, AND THIERFELDER L. Mutations of *TTN*, encoding the giant muscle filament titin, cause familial dilated cardiomyopathy. *Nat Genet* 30: 201–204, 2002.

45. GILBERT R, KELLY MG, MIKAWA T, AND FISCHMAN DA. The carboxyl terminus of myosin binding protein C specifies incorporation into the A-band of striated muscle. *J Cell Sci* 109: 101–111, 1996.

46. GOLDFARB LG, PARK KY, CERVENAKOVA L, GOROKHOVA S, LEE HS, VASCONCELAS O, NAGLE JW, SEMINO-MORA C, SIVAKUMAR K, AND DALAKAS MC. Missense mutations in desmin associated with familial cardiac and skeletal myopathy. *Nat Genet* 19: 402–403, 1998.

47. GOLLOB MH, GREEN MS, TANG ASL, GOLLOB T, KARIBE A, HASSAN AS, AHMAD F, LOZADO R, SHAH G, FANANAPAZIR L, BACHINSKI L, AND ROBERTS R. Identification of a gene responsible for familial Wolff-Parkinson-White syndrome. *N Engl J Med* 344: 1823–1831, 2001.

48. GRUEN M AND GAUTEL M. Mutations in the β-myosin S2 that cause familial hypertrophic cardiomyopathy abolish the interaction with the regulatory domain of myosin binding protein-C. *J Mol Biol* 286: 933–949, 1999.

49. GRUNIG E, TASMAN JA, KUCHERER H, FRANZ W, KUBLER W, AND KATUS HA. Frequency and phenotypes of familial dilated cardiomyopathy. *J Am Coll Cardiol* 31: 186–194, 1998.

50. HAGHIGHI K, SCHMIDT AG, HOIT BD, BRITTSAN AG, YATANI A, LESTER JW, ZHAI J, KIMURA Y, DORN GW, MACLENNAN DH, AND KRANIAS EG. Superinhibition of sarcoplasmic reticulum function by phospholamban induces cardiac contractile failure. *J Biol Chem* 276: 24145–24152, 2001.

51. HALLOPEAU M. Retrecissement ventriculoaortique. *Gazette Medicale Paris* 24: 683–684, 1869.

52. HART KA, HODGSON S, WALKER A, COLE CG, JOHNSON L, DUBOWITZ V, AND BOBROW M. DNA deletions in mild and severe Becker muscular dystrophy. *Hum Genet* 75: 281–285, 1987.

53. HELING A, ZIMMERMANN R, KOSTIN S, MAENO Y, HEIN S, DEVAUX B, BAUER E, KLOVEKORN WP, SCHLEPPER M, SCHAPER W, AND SCHAPER J. Increased expression of cytoskeletal, linkage and extracellular proteins in failing human myocardium. *Circ Res* 86: 846–853, 2000.

54. HERTIG CM, BUTZ S, KOCH S, EPPENBERGER-EBERHARDT M, KEMLER R,
AND EPPENBERGER HM. N-cadherin in adult rat cardiomyocytes in culture. II. Spatio-temporal appearance of proteins involved in cell-cell contact and communication. Formation of two distinct N-cadherin/catenin complexes. *J Cell Sci* 109: 11–20, 1996.

55. Ho CY, LEVER HM, DESANCTIS R, FARVER CF, SEIDMAN JG, AND SEID-MAN CE. Homozygous mutation in cardiac troponin T. Implications for hypertrophic cardiomyopathy. *Circulation* 102: 1950–1955, 2000.

56. HOFFMANN B, SCHMIDT-TRAUB H, PERROT A, OSTERZIEL KJ, AND GEBNER R. First mutation in cardiac troponin C, L29Q, in a patient with hypertrophic cardiomyopathy. *Hum Mutat* 17: 524, 2001.

57. HOLMES KC AND GEEVES MA. The structural basis of muscle contraction. *Philos Trans R Soc Lond* 355: 419–431, 2000.

58. HUXLEY AF. Muscle structure and theories of contraction. *Prog Biophys Phys Chem* 7: 255–318, 1957.

59. HUXLEY AF. Mechanics and models of the myosin motor. *Philos Trans R Soc Lond* 355: 433–440, 2000.

60. HUXLEY HE. The mechanism of muscular contraction. *Science* 164: 1356–1366, 1969.

61. JAMES J, ZHANG Y, OSINSKA H, SANBE A, KLEVITSKY R, HEWETT TE, AND ROBBINS J. Transgenic modeling of a cardiac troponin I mutation linked to familial hypertrophic cardiomyopathy. *Circ Res* 87: 805–811, 2000.

62. JAMES TN, ST. MARTIN E, WILLIS PW, AND LOHR TO. Apoptosis as a possible cause of gradual development of complete heart block and fatal arrhythmias associated with absence of the AV node, sinus node and intermodal pathways. *Circulation* 93: 1424–1438, 1996.

63. JARCHO JA, MCKENNA W, PARE JAP, SOLOMON SD, HOLCOMBE RF, DICKIE S, LEVI T, DONIS-KELLER H, SEIDMAN JG, AND SEIDMAN CE. Mapping a gene for familial hypertrophic cardiomyopathy to chromosome 14q1. *N Engl J Med* 321: 1372–1378, 1989.

64. JHA S, KAO A, DESAI S, St. JOHN SUTTON MG, JESSUP M, AND KEATING MT. Identification of a new locus for autosomal dominant dilated cardiomyopathy on chromosome 9q22–q31. *Circulation* 104 Suppl II: II-135, 2001.

65. JUNG M, POEPPING I, PERROT A, ELLMER AE, WIENKER TF, DIETZ R, REIS A, AND OSTERZIEL KJ. Investigation of a family with autosomal dominant dilated cardiomyopathy defines a novel locus on chromosome 2q14–q22. *Am J Hum Genet* 65: 1068–1077, 1999.

66. JUNG WI, SIEVERDING L, BREUER J, HOESS T, WIDMAIER S, SCHMIDT O, BUNSE M, VAN ERCKELENS F, APITZ J, LUTZ O, AND DIETZE GJ. 31P NMR spectroscopy detects metabolic abnormalities in asymptomatic patients with hypertrophic cardiomyopathy. *Circulation* 97: 2536–2542, 1998.

67. KAMISAGO M, SHARMA SD, DEPALMA SR, SOLOMON S, SHARMA P, MC-DONOUGH B, SMOOT L, MULLEN MP, WOOLF PK, WIGLE ED, SEIDMAN JG, AND SEIDMAN CE. Mutations in sarcomere protein genes as a cause of dilated cardiomyopathy. *N Engl J Med* 343: 1688–1696, 2000.

68. KASS S, MACRAE C, GRABER HL, SPARKS EA, MCNAMARA D, BOUDOULAS H, BASSON CT, BAKER PB, CODY RJ, FISHMAN MC, COX N, KONG A, WOOLEY CF, SEIDMAN JG, AND SEIDMAN CE. A gene defect that causes conduction system disease and dilated cardiomyopathy maps to chromosome 1p1–1q1. *Nat Genet* 7: 546–551, 1994.

69. KEELING PJ, GANG Y, SMITH G, SEO H, BENT SE, MURDAY V, CAFORIO ALP, AND MCKENNA WJ. Familial dilated cardiomyopathy in the United Kingdom. *Br Heart J* 73: 417–421, 1995.

70. KIM SJ, IZUKA K, KELLY RA, GENG YJ, BISHOP SP, YANG G, KUDEJ A, MCCONNELL BK, SEIDMAN CE, SEIDMAN JG, AND VATNER SF. An α-cardiac myosin heavy chain gene mutation impairs contraction and relaxation function of cardiac myocytes. *Am J Physiol Heart Circ Physiol* 276: H1780–H1787, 1999.

71. KIMURA A, HARADA H, PARK JE, NISHI H, SATOH M, TAKAHASHI M, HIROI S, SASAOKA T, OHBUCHI N, NAKAMURA T, KOYANAGI T, HWANG TH, CHOO JA, CHUNG KS, HASEGAWA A, NAGAI R, OKAZAKI O, NAKAMURA H, MATSUZAKI M, SAKAMOTO T, TOSHIMA H, KOGA Y, IMAIZUMI T, AND SASAZUKI T. Mutations in the cardiac troponin I gene associated with hypertrophic cardiomyopathy. *Nat Genet* 16: 379–382, 1997.

72. KOENIG M, HOFFMAN EP, BERTELSON CJ, MONACO AP, FEENER C, AND KUNKEL LM. Complete cloning of the Duchenne muscular dystrophy (DMD) cDNA and preliminary genomic organization of the DMD gene in normal and affected individuals. *Cell* 50: 509–517, 1987.

73. KRAJINOVIC M, PINAMONTI B, SINAGRA G, VATTA M, SEVERINI GM,

MILASIN J, FALASCHI A, CAMERINI F, GIACCA M, AND MESTRONI L. Linkage of familial dilated cardiomyopathy to chromosome 9. *Am J Hum Genet* 57: 846–852, 1995.

74. KUNST G, KRESS KR, GRUEN M, UTTEWEILER D, GAUTEL M, AND FINK RHA. Myosin binding protein C, a phosphorylation-dependent force regulator in muscle that controls the attachment of myosin heads by its interaction with myosin S2. *Circ Res* 86: 51–58, 2000.

75. KUSHWAHA SS, FALLON JT, AND FUSTER V. Restrictive cardiomyopathy. *N Engl J Med* 336: 267–276, 1997.

76. LAITINEN PJ, BROWN KM, HIPPO K, SWAN H, DEVANEY JM, BRAHMBHATT B, DONARUM EA, MARINO M, TISO N, VIITASALO M, TOIVONEN L, STEPHAN DA, AND KONTULA K. Mutations of the cardiac ryanodine receptor (RyR2) gene in familial polymorphic ventricular tachycardia. *Circulation* 103: 485–490, 2001.

77. LANKFORD EB, EPSTEIN ND, FANANAPAZIR L, AND SWEENEY HL. Abnormal contractile properties of muscle fibers expressing β-myosin heavy chain gene mutations in patients with hypertrophic cardiomyopathy. *J Clin Invest* 95: 1409–1414, 1995.

78. LEVINE RJC, YANG Z, EPSTEIN ND, FANANAPAZIR L, STILL JT, AND SWEENEY HL. Structural and functional responses of mammalian thick filaments to alterations in myosin regulatory light chains. *J Struct Biol* 122; 149–161, 1998.

79. LI D, AHMAD F, GARDNER MJ, WEILBAECHER D, HILL R, KARIBE A, GONZALEZ O, TAPSCOTT T, SHARRATT GP, BACHINSKI LL, AND ROBERTS R. The locus of a novel gene responsible for arrhythmogenic right-ventricular dysplasia characterised by early onset and high penetrance maps to chromosome 10p12–p14. *Am J Hum Genet* 66: 148–156, 2000.

80. LI D, TAPSCOTT T, GONZALEZ O, BURCH PE, QUINONES MA, ZOGHIBI WA, HILL R, BACHINSKI LL, MANN DL, AND ROBERTS R. Desmin mutation responsible for idiopathic dilated cardiomyopathy. *Circulation* 100: 461–464, 1999.

81. LI YY, MAISCH B, ROSE ML, AND HENGSTENBERG C. Point mutations in mitochondrial DNA of patients with dilated cardiomyopathy. *J Mol Cell Cardiol* 29: 2699–2709, 1997.

82. LIM DS, OBERST L, MCCLUGGAGE M, YOUKER K, LACY J, DEMAYO F, ENTMAN ML, ROBERTS R, MICHAEL LH, AND MARIAN AJ. Decreased left ventricular ejection fraction in transgenic mice expressing mutant cardiac troponin T-Q(92), responsible for human hypertrophic cardiomyopathy. *J Mol Cell Cardiol* 32: 365–374, 2000.

83. LIN D, BOBKOVA A, HOMSHER E, AND TOBACMAN LS. Altered cardiac troponin T in vitro function in the presence of a mutation implicated in familial hypertrophic cardiomyopathy. *J Clin Invest* 97: 2842–2848, 1996.

84. LYMN RW AND TAYLOR EW. Mechanism of adenosine triphosphate hydrolysis by actomyosin. *Biochemistry* 10: 4617–4624, 1971.

85. MAEDA M, HOLDER E, LOWES B, VALENT S, AND BIES RD. Dilated cardiomyopathy associated with deficiency of the cytoskeletal protein metavinculin. *Circulation* 95: 17–20, 1997.

86. MALLAT Z, TEDGUI A, FONTALIRAN F, FRANK R, DURIGON M, AND FONTAINE G. Evidence of apoptosis in arrhythmogenic right ventricular dysplasia. *N Engl J Med* 335: 1190–1196, 1996.

87. MARIAN AJ, YU QT, MANN DL, GRAHAM FL, AND ROBERTS R. Expression of a mutation causing hypertrophic cardiomyopathy disrupts sarcomere assembly in adult feline cardiac myocytes. *Circ Res* 77: 98–106, 1995.

88. MARIAN AJ, ZHAO G, SETA Y, ROBERTS R, AND YU Q. Expression of a mutant (Arg92Gln) human cardiac troponin T, known to cause hypertrophic cardiomyopathy, impairs adult cardiac myocyte contractility. *Circ Res* 81: 76–85, 1997.

89. MARIN-GARCIA J AND GOLDENTHAL MJ. Mitochondrial cardiomyopathy: molecular and biochemical analysis. *Pediatr Cardiol* 18: 251–260, 1997.

90. MARIN-GARCIA J, GOLDENTHAL MJ, ANANTHAKRISHNAN R, PIERPONT MEM, FRICKER FJ, LIPSHULTZ SE, AND PEREZ-ATAYDE A. Specific mitochondrial DNA deletions in idiopathic dilated cardiomyopathy. *Cardiovasc Res* 31: 306–313, 1996.

91. MARON BJ, GARDIN JM, FLACK JM, GIDDING SS, KUROSAKI TT, AND BILD DE. Prevalence of hypertrophic cardiomyopathy in a general population of young adults. Echocardiographic analysis of 4111 subjects in the CARDIA study. *Circulation* 92: 785–789, 1995.

92. MARX SO, REIKEN S, HISAMATSU Y, JAYARAMAN T, BURKHOFF D, ROSEM-BLIT N, AND MARKS AR. PKA phosphorylation dissociates FKBP12.6

from the calcium release channel (ryanodine receptor): defective regulation in failing hearts. *Cell* 101: 365–376, 2000.

93. MAYOSI BM, KHOGALI S, ZHANG B, AND WATKINS H. Cardiac and skeletal actin gene mutations are not a common cause of dilated cardiomyopathy. *J Med Genet* 36: 796–797, 1999.

94. MCCARTHY TV, QUANE KA, AND LYNCH PJ. Ryanodine receptor mutations in malignant hyperthermia and central core disease. *Hum Mutat* 15: 410–417, 2000.

95. MCCONNELL BK, FATKIN D, SEMSARIAN C, JONES KA, GEORGAKOPOULOS D, MAGUIRE CT, HEALEY MJ, MUDD JO, MOSKOWITZ IPG, CONNER DA, GIEWAT M, WAKIMOTO H, BERUL CI, SCHÖEN FJ, KASS DA, SEIDMAN CE, AND SEIDMAN JG. Comparison of two murine models of familial hypertrophic cardiomyopathy. *Circ Res* 88: 383–389, 2001.

96. MCCONNELL BK, JONES KA, FATKIN D, ARROYO LH, LEE RT, ARISTIZABAL O, TURNBULL DH, GEORGAKOPOULOS D, KASS D, BOND M, NIIMURA H, SCHÖEN FJ, CONNER D, FISCHMAN DH, SEIDMAN CE, AND SEIDMAN JG. Dilated cardiomyopathy in homozygous myosin-binding protein-C mutant mice. *J Clin Invest* 104: 1235–1244, 1999.

97. MCDONALD KS AND Moss RL. Strongly binding myosin crossbridges regulate loaded shortening and power output in cardiac myocytes. *Circ Res* 87: 768–773, 2000.

98. MCKOY G, PROTONOTARIOS N, CROSBY A, TSATSOPOULOU A, ANASTASAKIS A, COONAR A, NORMAN M, BABOONIAN C, JEFFERY S, AND MCKENNA WJ. Identification of a deletion in plakoglobin in arrhythmogenic right ventricular cardiomyopathy with palmoplantar keratoderma and woolly hair (Naxos disease). *Lancet* 355: 2119–2124, 2000.

99. MELBERG A, OLDFORS A, BLOMSTROM-LUNDQVIST C, STALBERG E, CARLS-SON B, LARSSON E, LIDELL C, EEG-OLOFSSON KE, WIKSTROM G, HENRIKSSON KG, AND DAHL N. Autosomal dominant myofibrillar myopathy with arrhythmogenic right ventricular cardiomyopathy linked to chromosome 10q. *Ann Neurol* 46: 684–692, 1999.

100. MESSINA DN, SPEER MC, PERICAK-VANCE MA, AND MCNALLY EM. Linkage of familial dilated cardiomyopathy with conduction defect and muscular dystrophy to chromosome 6q23. *Am J Hum Genet* 61: 909–917, 1997.

101. MESTRONI L, ROCCO C, GREGORI D, SINAGRA G, DI LENARDA A, MIOCIC S, VATTA M, PINAMONTI B, MUNTONI F, CAFORIO ALP, MCKENNA WJ, FALASCHI A, GIACCA M, AND CAMERINI F. Familial dilated cardiomyopathy: evidence for genetic and phenotypic heterogeneity. *J Am Coll Cardiol* 34: 181–190, 1999.

102. MICHELS VV, MOLL PP, MILLER FA, TAJIK AJ, CHU JS, DRISCOLL DJ, BURNETT JC, RODEHEFFER RJ, CHESEBRO JH, AND TAZELAAR HD. The frequency of familial dilated cardiomyopathy in a series of patients with idiopathic dilated cardiomyopathy. *N Engl J Med* 326: 77–82, 1992.

103. MILLER T, SZCZESNA D, HOUSMANS PR, ZHAO J, DE FREITAS F, GOMES AV, CULBREATH L, MCCUE J, WANG Y, XU Y, KERRICK WG, AND POTTER JD. Abnormal contractile function in transgenic mice expressing a familial hypertrophic cardiomyopathy-linked troponin T (I180) mutation. *J Biol Chem* 276: 3743–3755, 2001.

104. MILNER DJ, WEITZER G, TRAN D, BRADLEY A, AND CAPETANAKI Y. Disruption of muscle architecture and myocardial degeneration in mice lacking desmin. *J Cell Biol* 134: 1255–1270, 1996.

105. MOLKENTIN JD, LU JR, ANTOS C, MARKHAM B, RICHARDSON J, ROBBINS J, GRANT SR, AND OLSON EN. A calcineurin-dependent transcriptional pathway for cardiac hypertrophy. *Cell* 93: 215–228, 1998.

106. MOOLMAN JC, CORFIELD VA, POSEN B, NGUMBELA K, SEIDMAN CE, BRINK PA, AND WATKINS H. Sudden death due to troponin T mutations. *J Am Coll Cardiol* 29: 549–555, 1997.

107. MORIMOTO S, LU QW, HARADA K, TAKAHASHI-YANAGA F, MINAKAMI R, OHTA M, SASAGURI T, AND OHTSUKI I. Ca²⁺-desensitizing effect of a deletion mutation δK210 in cardiac troponin T that causes familial dilated cardiomyopathy. *Proc Natl Acad Sci USA* 99: 913–918, 2002.

108. MORIMOTO S, YANAGA F, MINAKAMI R, AND OHTZUKI I. Ca²⁺-sensitizing effects of the mutations at Ile-79 and Arg-92 of troponin T in hypertrophic cardiomyopathy. *Am J Physiol Cell Physiol* 275: C200–C207, 1998.

109. MUCHIRI A, BONNE G, VAN DER KOOI AJ, VAN MEEGEN M, BAAS F, BOLHUIS PA, DE VISSER M, AND SCHWARTZ K. Identification of mutations in the gene encoding lamins A/C in autosomal dominant limb girdle muscular dystrophy with atrioventricular conduction disturbances (LGMD1B). *Hum Mol Genet* 9: 1453–1459, 2000.

110. MUNTONI F, MELIS MA, GANAU A, AND DUBOWITZ V. Transcription of the dystrophin gene in normal tissues and in skeletal muscle of a family with X-linked dilated cardiomyopathy. *Am J Hum Genet* 56: 151–157, 1995.

111. MUNTONI F, WILSON L, MARROSU G, MARROSU MG, CIANCHETTI C, MESTRONI L, GANAU A, DUBOWITZ V, AND SEWRY C. A mutation in the dystrophin gene selectively affecting dystrophin expression in the heart. *J Clin Invest* 96: 693–699, 1995.

112. MUTHUCHAMY M, PIEPLES K, RETHINASAMY P, HOIT B, GRUPP IL, BOIVIN GP, WOLSKA B, EVANS C, SOLARO RJ, AND WIECZOREK DF. Mouse model of a familial hypertrophic cardiomyopathy mutation in α-tropomyosin manifests cardiac dysfunction. *Circ Res* 85: 47–56, 1999.

113. NARULA J, HAIDER N, VIRMANI R, DiSALVO TG, KOLOGDIE FD, HAJJAR RJ, SCHMIDT U, SEMIGRAN MJ, DEC GW, AND KHAW BA. Apoptosis in myocytes in end-stage heart failure. *N Engl J Med* 335: 1182–1189, 1996.

114. NIGRO V, DE SA MOREIRA E, PILUSO G, VAINZOF M, BELSITO A, POLITANO L, PUCA AA, PASSOS-BUENO MR, AND ZATZ M. Autosomal recessive limb-girdle muscular dystrophy, LGMD2F, is caused by a mutation in the delta-sarcoglycan gene. *Nat Genet* 14: 195–198, 1996.

115. NIGRO V, OKAZAKI Y, BELSITO A, PILUSO G, MATSUDA Y, POLITANO L, NIGRO G, VENTURA C, ABBONDANZA C, MOLINARI AM, ACAMPORA D, NISHIMURA M, HAYASHIZAKI Y, AND PUCA GA. Identification of the Syrian hamster cardiomyopathy gene. *Hum Mol Genet* 6: 601–607, 1997.

116. NIIMURA H, BACHINSKI LL, SANGWATANAROJ S, WATKINS H, CHUDLEY AE, MCKENNA W, KRISTINSSON A, ROBERTS R, SOLE M, MARON BJ, SEIDMAN JG, AND SEIDMAN CE. Mutations in the gene for cardiac myosin-binding protein C and late-onset familial hypertrophic cardiomyopathy. *N Engl J Med* 338: 1248–1257, 1998.

117. OBERST L, ZHAO G, PARK JT, BRUGADA R, MICHAEL LH, ENTMAN ML, ROBERTS R, AND MARIAN AJ. Dominant-negative effect of a mutant cardiac troponin T on cardiac structure and function in transgenic mice. *J Clin Invest* 102: 1498–1505, 1998.

118. OLSON EN AND MOLKENTIN JD. Prevention of cardiac hypertrophy by calcineurin inhibition. Hope or hype? *Circ Res* 84: 623–632, 1999.

119. OLSON TM, DOAN TP, KISHIMOTO NY, WHITBY FG, ACKERMAN MJ, AND FANANAPAZIR L. Inherited and de novo mutations in the cardiac actin gene cause hypertrophic cardiomyopathy. *J Mol Cell Cardiol* 32: 1687–1694, 2000.

120. OLSON TM, ILLENBERGER S, KISHIMOTO NY, HUTTELMAIER S, KEATING MT, AND JOCKUSCH BM. Metavinculin mutations alter actin interaction in dilated cardiomyopathy. *Circulation* 105: 431–437, 2002.

121. OLSON TM AND KEATING MT. Mapping a cardiomyopathy locus to chromosome 3p22–p25. *J Clin Invest* 97: 528–532, 1996.

122. OLSON TM, KISHIMOTO NY, WHITBY FG, AND MICHELS VV. Mutations that alter the surface charge of alpha-tropomyosin are associated with dilated cardiomyopathy. *J Mol Cell Cardiol* 33: 723–732, 2001.

123. OLSON TM, MICHELS VV, THIBODEAU SN, TAI YS, AND KEATING MT. Actin mutations in dilated cardiomyopathy, a heritable form of heart failure. *Science* 280: 750–752, 1998.

124. PALKA HL AND GREEN KJ. Roles of plakoglobin end domains in desmosome assembly. *J Cell Sci* 110: 2359–2371, 1997.

125. POTTER K, JIANG H, HASSANZADEH S, MASTER SR, CHANG A, DALAKAS MC, RAYMENT I, SELLERS JR, FANANAPAZIR L, AND EPSTEIN ND. Mutations in either the essential or regulatory light chains of myosin are associated with a rare myopathy in human heart and skeletal muscle. *Nat Genet* 13: 63–69, 1996.

126. PRIORI SG, NAPOLITANO C, TISO N, MEMMI M, VIGNATI G, BLOISE R, SORRENTINO V, AND DANIELI GA. Mutations in the cardiac ryanodine receptor gene (*hRyR2*) underlie catecholaminergic polymorphic ventricular tachycardia. *Circulation* 103: 196–200, 2001.

127. RAMPAZZO A, NAVA A, DANIELI GA, BUJA G, DALIENTO L, FASOLI G, SCOGNAMIGLIO R, CORRADO D, AND THIENE G. The gene for arrhythmogenic right ventricular cardiomyopathy maps to chromosome 14q23–q24. *Hum Mol Genet* 3: 959–962, 1994.

128. RAMPAZZO A, NAVA A, ERNE P, EBERHARD M, VIAN E, SLOMP P, TISO N, THIENE G, AND DANIELI GA. A new locus for arrhythmogenic right ventricular cardiomyopathy (ARVD2) maps to chromosome 1q42–q43. *Hum Mol Genet* 4: 2151–2154, 1995.

129. RAMPAZZO A, NAVA A, MIORIN M, FONDERICO P, POPE B, TISO N, LIVOLSI B, ZIMBELLO R, THIENE G, AND DANIELI GA. ARVD4, a new locus for arrhythmogenic right ventricular cardiomyopathy, maps to chromosome 2 long arm. *Genomics* 45: 259–263, 1997.

130. RAYMENT I, HOLDEN HM, SELLERS JR, FANANAPAZIR L, AND EPSTEIN ND. Structural interpretation of the mutations in the $\beta$-cardiac myosin that have been implicated in familial hypertrophic cardiomyopathy. *Proc Natl Acad Sci USA* 92: 3864–3868, 1995.

131. RAYMENT I, HOLDEN HM, WHITTAKER M, YOHN CB, LORENZ M, HOLMES KC, AND MILLIGAN RA. Structure of the actin-myosin complex and its implications for muscle contraction. *Science* 261: 58–65, 1993.

132. REDWOOD C, LOHMANN K, BING W, ESPOSITO GM, ELLIOTT K, ABDUL-RAZZAK H, KNOTT A, PURCELL I, MARSTON S, AND WATKINS H. Investigation of a truncated cardiac troponin T that causes familial hypertrophic cardiomyopathy. *Circ Res* 86: 1146–1152, 2000.

133. REDWOOD CS, PURCELL IF, ESPOSITO G, WATKINS H, MARSTON SB, AND BING W. Functional effects of hypertrophic cardiomyopathy mutations in $\alpha$-tropomyosin on Ca${}^{2+}$ regulation of thin filaments (Abstract). *Biophys J* 74: A354, 1998.

134. REGITZ-ZAGROSEK V, DAEHMLOW S, KNEUPPEL T, KOTSCH K, GILLE C, FROEMMEL C, ELSNER A, HUMMEL M, AND HETZER R. Novel mutations in the $\beta$ myosin heavy chain and myosin binding protein C gene are associated with dilated cardiomyopathy. *Circulation* 104 Suppl II: II-572, 2001.

135. REPORT OF THE 1995 WORLD HEALTH ORGANIZATION/INTERNATIONAL SOCIETY AND FEDERATION OF CARDIOLOGY TASK FORCE. The definition and classification of cardiomyopathies. *Circulation* 93: 841–842, 1996.

136. RICHARD P, CHARRON P, LECLERCQ C, LEDEUIL C, CARRIER L, DUBOURG O, DESNOS M, BOUHOUR JB, SCHWARTZ K, DAUBERT JC, KOMAJDA M, AND HAINQUE B. Homozygotes for a R869G mutation in the beta-myosin heavy chain gene have a severe form of familial hypertrophic cardiomyopathy. *J Mol Cell Cardiol* 32: 1575–1583, 2000.

137. ROTHERMEL BA, MCKINSEY TA, VEGA RB, NICOL RL, MAMMEN P, YANG J, ANTOS CL, SHELTON JM, BASSEL-DUBY R, OLSON EN, AND WILLIAMS RS. Myocyte-enriched calcineurin-interacting protein, MCIP1, inhibits cardiac hypertrophy in vivo. *Proc Natl Acad Sci USA* 98: 3328–3333, 2001.

138. ROTTBAUER W, GAUTEL M, ZEHELEIN J, LABEIT S, FRANZ WM, FISCHER C, VOLLRATH B, MALL G, DIETZ R, KUBLER W, AND KATUS HA. Novel splice donor site mutation in the cardiac myosin-binding protein-C gene in familial hypertrophic cardiomyopathy. *J Clin Invest* 100: 475–482, 1997.

139. RUST EM, ALBAYYA FP, AND METZGER JM. Identification of a contractile deficit in adult cardiac myocytes expressing hypertrophic cardiomyopathy-associated mutant troponin T proteins. *J Clin Invest* 103: 1459–1467, 1999.

140. SANBE A, GULICK J, HAYES E, WARSHAW D, OSINSKA H, CHAN CB, KLEVITSKY R, AND ROBBINS J. Myosin light chain replacement in the heart. *Am J Physiol Heart Circ Physiol* 279: H1355–H1364, 2000.

141. SANBE A, NELSON D, GULICK J, SETSER E, OSINSKA H, WANG X, HEWETT TE, KLEVITSKY R, HAYES E, WARSHAW DM, AND ROBBINS J. In vivo analysis of an essential myosin light chain mutation linked to familial hypertrophic cardiomyopathy. *Circ Res* 87: 296–302, 2000.

142. SATA M AND IKEBE M. Functional analysis of the mutations in the human cardiac $\beta$-myosin that are responsible for familial hypertrophic cardiomyopathy. *J Clin Invest* 98: 2866–2873, 1996.

143. SATOH M, TAKAHASHI M, SAKAMOTO T, HIROE M, MARUMO F, AND KIMURA A. Structural analysis of the titin gene in hypertrophic cardiomyopathy: identification of a novel disease gene. *Biochem Biophys Res Commun* 262: 411–417, 1999.

144. SCHONBERGER J, LEVY H, GRUNIG E, SANGWATANAROJ S, FATKIN D, MACRAE C, STACKER H, HALPIN C, EAVEY R, PHILBIN EF, KATUS H, SEIDMAN JG, AND SEIDMAN CE. Dilated cardiomyopathy and sensorineural hearing loss. A heritable syndrome that maps to chromosome 6q23–q24. *Circulation* 101: 1812–1818, 2000.

145. SEVERINI GM, KRAJINOVIC M, PINAMONTI B, SINAGRA G, FIORETTI P, BRUNAZZI MC, FALASCHI A, CAMERINI F, GIACCA M, AND MESTRONI L. A new locus for arrhythmogenic right ventricular dysplasia on the long arm of chromosome 14. *Genomics* 31: 193–200, 1996.

146. SHACKLETON S, LLOYD DJ, JACKSON SNJ, EVANS R, NIERMEIJER MF, SINGH BM, SCHMIDT H, BRABANT G, KUMAR S, DURRINGTON PN, GREGORY S, O'RAHILLY S, AND TREMBATH RC. LMNA, encoding lamin A/C is mutated in partial lipodystrophy. *Nat Genet* 24: 153–156, 2000.

147. SIU BL, NIIMURA H, OSBORNE JA, FATKIN D, MACRAE C, SOLOMON S, BENSON DW, SEIDMAN JG, AND SEIDMAN CE. Familial dilated cardiomyopathy locus maps to chromosome 2q31. *Circulation* 99: 1022–1026, 1999.

148. SOLARO RJ AND VAN EYK J. Altered interactions among thin filament proteins modulate cardiac function. *J Mol Cell Cardiol* 28: 217–230, 1996.

149. SPINDLER M, SAUPE KW, CHRISTE ME, SWEENEY HL, SEIDMAN CE, SEIDMAN JG, AND INGWALL JS. Diastolic dysfunction and altered energetics in the $\alpha$MHC${}^{403/+}$ mouse model of familial hypertrophic cardiomyopathy. *J Clin Invest* 101: 1775–1783, 1998.

150. STUURMAN N, HEINS S, AND AEBI U. Nuclear lamins: their structure, assembly and interactions. *J Struct Biol* 122: 42–66, 1998.

151. SUGDEN PH. Signaling in myocardial hypertrophy. Life after cal-cineurin? *Circ Res* 84: 633–646, 1999.

152. SULLIVAN T, ESCALANTE-ALCALDE D, BHATT H, ANVER M, BHAT N, NAGASHIMA K, STEWART CL, AND BURKE B. Loss of A-type lamin expression compromises nuclear envelope integrity leading to muscular dystrophy. *J Cell Biol* 147: 913–919, 1999.

153. SUOMALAINEN A, PAETAU A, LEINONEN H, MAJANDER A, PELTONEN L, AND SOMER H. Inherited idiopathic dilated cardiomyopathy with multiple deletions of mitochondrial DNA. *Lancet* 340: 1319–1320, 1992.

154. SWEENEY HL, FENG HS, YANG Z, AND WATKINS H. Functional analyses of troponin T mutations that cause hypertrophic cardiomyopathy: insights into disease pathogenesis and troponin function. *Proc Natl Acad Sci USA* 95: 14406–14410, 1998.

155. SWEENEY HL, STRACESKI AJ, LEINWAND LA, TIKUNOV BA, AND FAUST L. Heterologous expression of a cardiomyopathic myosin that is defective in its actin interaction. *J Biol Chem* 269: 1603–1605, 1994.

156. SYLVIOUS N, TESSON F, GAYET C, CHARRON P, BENAICHE A, MANGIN L, PEUCHMAURD M, DUBOSQ-BIDOT L, FEINGOLD J, BECKMANN JS, BOUCHIER C, AND KOMAJDA K. A new locus for autosomal dominant dilated cardiomyopathy identified on chromosome 6q12–q16. *Am J Hum Genet* 68: 241–246, 2001.

157. SZCZESNA D, ZHANG R, ZHAO J, JONES M, GUZMAN G, AND POTTER JD. Altered regulation of cardiac muscle contraction by troponin T mutations that cause familial hypertrophic cardiomyopathy. *J Biol Chem* 275: 624–630, 2000.

158. TAKAHASHI-YANAGA F, MORIMOTO S, AND OHTZUKI I. Effect of Arg145Gly mutation in human cardiac troponin I on the ATPase activity of cardiac myofibrils. *J Biochem* 127: 355–357, 2000.

159. TAKAI E, AKITA H, SHIGA N, KANAZAWA K, YAMADA S, TERASHIMA M, MATSUDA Y, IWAI C, KAWAI K, YOKOTA Y, AND YOKOYAMA M. Mutational analysis of the cardiac actin gene in familial and sporadic dilated cardiomyopathy. *Am J Med Genet* 86: 325–327, 1999.

160. TAKEMURA G, OHNO M, HAYAKAWA Y, MISAO J, KANOH M, OHNO A, UNO Y, MINATOGUCHI S, FUJIWARA T, AND FUJIWARA H. Role of apoptosis in the disappearance of infiltrated and proliferated interstitial cells after myocardial infarction. *Circ Res* 82: 1130–1138, 1998.

161. TARDIFF JC, FACTOR SM, TOMPKINS BD, HEWETT TE, PALMER BM, MOORE RL, SCHWARTZ S, ROBBINS J, AND LEINWAND LA. A truncated cardiac troponin T molecule in transgenic mice suggests multiple cellular mechanisms for familial hypertrophic cardiomyopathy. *J Clin Invest* 101: 2800–2811, 1998.

162. TARDIFF JC, HEWETT TE, PALMER BM, OLSSON C, FACTOR SM, MOORE RL, ROBBINS J, AND LEINWAND LA. Cardiac troponin T mutations result in allele-specific phenotypes in a mouse model for hypertrophic cardiomyopathy. *J Clin Invest* 104: 469–481, 1999.

163. TESSON F, SYLVIOUS N, PILOTTO A, DUBOSQ-BIDOT L, PEUCHMAURD M, BOUCHIER C, BENAICHE A, MANGIN L, CHARRON P, GAVAZZI A, TAVAZZI L, ARBUSTINI E, AND KOMAJDA M. Epidemiology of desmin and cardiac actin gene mutations in a European population of dilated cardiomyopathy. *Eur Heart J* 21: 1872–1876, 2000.

164. THIENE G, BASSO C, DANIELI G, RAMPAZZO A, CORRADO D, AND NAVA A. Arrhythmogenic right ventricular cardiomyopathy. *Trends Cardiovasc Med* 7: 84–90, 1997.

165. THIERFELDER L, WATKINS H, MACRAE C, LAMAS R, MCKENNA W, VOSBERG HP, SEIDMAN JG, AND SEIDMAN CE. $\alpha$-Tropomyosin and cardiac troponin T mutations cause familial hypertrophic cardiomyopathy: a disease of the sarcomere. *Cell* 77: 710–712, 1994.

166. TISO N, STEPHAN DA, NAVA A, BAGATTIN A, DEVANEY JM, STANCHI F, LARDERET G, BRAHMBHATT B, BROWN K, BAUCE B, MURIAGO M, BASSO C, THIENE G, DANIELLI GA, AND RAMPAZZO A. Identification of mutations in the cardiac ryanodine receptor gene in families affected with arrhythmogenic right ventricular cardiomyopathy type 2 (ARVD2). *Hum Mol Genet* 10: 189–194, 2001.

167. TOWBIN JA, HEJTMANCIK JF, BRINK P, GELB B, ZHU XM, CHAMBERLAIN JS, MCCABE ERB, AND SWIFT M. X-linked dilated cardiomyopathy. Molecular genetic evidence of linkage to the Duchenne muscular dystrophy (dystrophin) gene at the Xp21 locus. *Circulation* 87: 1854–1865, 1993.

168. TSUBATA S, BOWLES KR, VATTA M, ZINTZ C, TITUS J, MUHONEN L, BOWLES NE, AND TOWBIN JA. Mutations in the human δ-sarcoglycan gene in familial and sporadic dilated cardiomyopathy. *J Clin Invest* 106: 655–662, 2000.

169. TYSKA MJ, HAYES E, GIEWAT M, SEIDMAN CE, SEIDMAN JG, AND WARSHAW DM. Single-molecule mechanics of R403Q cardiac myosin isolated from the mouse model of familial hypertrophic cardiomyopathy. *Circ Res* 86: 737–744, 2000.

170. VIKSTROM KL AND LEINWAND LA. Contractile protein mutations and heart disease. *Curr Opin Cell Biol* 8: 97–105, 1996.

171. WANNENBURG T, HELJNE GH, GEERDINK JH, VAN DEN DOOL HW, JANSEN PML, AND DE TOMBE PP. Cross-bridge kinetics in rat myocardium: effect of sarcomere length and calcium activation. *Am J Physiol Heart Circ Physiol* 279: H779–H790, 2000.

172. WATKINS H, CONNER D, THIERFELDER L, JARCHO JA, MACRAE C, McKENNA W, MARON BJ, SEIDMAN JG, AND SEIDMAN CE. Mutations in the cardiac myosin binding protein-C gene on chromosome 11 cause familial hypertrophic cardiomyopathy. *Nat Genet* 11: 434–437, 1995.

173. WATKINS H, MCKENNA WJ, THIERFELDER L, SUK HJ, ANAN R, O'DONOUGHUE A, SPIRITO P, MATSUMORI A, MORAVEC CS, SEIDMAN JG, AND SEIDMAN CE. Mutations in the genes for cardiac troponin T and α-tropomyosin in hypertrophic cardiomyopathy. *N Engl J Med* 332: 1058–1064, 1995.

174. WATKINS H, ROSENZWEIG A, HWANG DS, LEVI T, MCKENNA W, SEIDMAN CE, AND SEIDMAN JG. Characteristics and prognostic implications of

myosin missense mutations in familial hypertrophic cardiomyopathy. *N Engl J Med* 326: 1108–1114, 1992.

175. WATKINS H, SEIDMAN CE, SEIDMAN JG, FENG HS, AND SWEENEY HL. Expression and functional assessment of a truncated cardiac troponin T that causes hypertrophic cardiomyopathy. *J Clin Invest* 98: 2456–2461, 1996.

176. WEISBERG A AND WINEGRAD S. Relation between crossbridge structure and actomyosin ATPase activity in rat heart. *Circ Res* 83: 60–72, 1998.

177. YANAGA F, MORIMOTO S, AND OHTSUKI I. Ca²⁺ sensitization and potentiation of the maximum level of myofibrillar ATPase activity caused by mutations of troponin T found in familial hypertrophic cardiomyopathy. *J Biol Chem* 274: 8806–8812, 1999.

178. YANAGIDA T, ESAKI S, IWANE AH, INOUE Y, ISHIJIMA A, KITAMURA K, TANAKA H, AND TOKUNAGA M. Single-motor mechanics and models of the myosin motor. *Philos Trans R Soc Lond* 355: 441–447, 2000.

179. YANG J, MORAVEC CS, SUSSMAN MA, DI PAOLA NR, FU D, HAWTHORN L, MITCHELL CA, YOUNG JB, FRANCIS GS, MCCARTHY PM, AND BOND M. Decreased SLIM1 expression and increased gelsolin expression in failing human hearts measured by high-density oligonucleotide arrays. *Circulation* 102: 3046–3052, 2000.

180. YANG Q, SANBE A, OSINSKA H, HEWETT TE, KLEVTISKY R, AND ROBBINS J. A mouse model of myosin binding protein C human familial hypertrophic cardiomyopathy. *J Clin Invest* 102: 1292–1300, 1998.

181. ZOLK O, CARONI P, AND BOHM M. Decreased expression of the cardiac LIM domain protein MLP in chronic human heart failure. *Circulation* 101: 2674–2677, 2000.

182. ZOU Y, HIROI Y, UOZUMI H, TAKIMOTO E, TOKO H, ZHU W, KUDO H, MIZUKAMI M, SHIMOYAMA M, SHIBASAKI F, NAGAI R, YAZAKI Y, AND KOMURO I. Calcineurin plays a critical role in the development of pressure overload-induced cardiac hypertrophy. *Circulation* 104: 97–101, 2001.
