
Microbiology (2002), 148, 3317–3326

**Methyltransferase genes in Streptomyces rishiriensis: new coumermycin derivatives from gene-inactivation experiments**

Shu-Ming Li,$^{1}$ Lucia Westrich,$^{1}$ Jürgen Schmidt,$^{2}$ Christine Kuhnt$^{2}$ and Lutz Heide$^{1}$

Author for correspondence: Shu-Ming Li. Tel: +49 7071 2976995. Fax: +49 7071 295250.  
e-mail: shuming.li@uni-tuebingen.de

---

$^{1}$ Eberhard-Karls-Universität Tübingen, Pharmazeutische Biologie, Auf der Morgenstelle 8, D-72076 Tübingen, Germany  

$^{2}$ Institut für Pflanzenbiochemie, Weinberg 3, 06120 Halle (Saale), Germany  

The coumarin antibiotic coumermycin A₁ contains at least eight methyl groups, presumably derived from S-adenosylmethionine. Two putative methyltransferase genes, *couO* and *couP*, of the coumermycin A₁ biosynthetic gene cluster were inactivated by in-frame deletion. In the resulting mutants, coumermycin A₁ production was abolished. New coumermycin derivatives were accumulated instead, and were identified by HPLC-MS using selected reaction monitoring via electrospray ionization. *couO* mutants accumulated a coumermycin derivative lacking the methyl groups at C-8 of the characteristic aminocoumarin rings, whereas in the *couP* mutant a coumermycin derivative lacking the methyl groups at the 4-hydroxyl groups of the two deoxysugar moieties was identified. These results provided evidence that *couO* encodes a C-methyltransferase responsible for the transfer of a methyl group to C-8 of the aminocoumarin ring, and *couP* an O-methyltransferase for methylation of 4-OH of the sugar in the biosynthesis of coumermycin A₁, respectively. C-methylation of the aminocoumarin ring is considered as an early step of coumermycin biosynthesis. Nevertheless, the intermediates with the non-methylated aminocoumarin ring were accepted by the enzymes catalysing the subsequent steps of the pathway. The new, demethylated secondary metabolites were produced in an amount at least as high as that of coumermycin A₁ in the wild-type.

---

Keywords: aminocoumarin antibiotics, biosynthesis, methyltransferase

---

### INTRODUCTION

Aminocoumarin antibiotics, such as novobiocin, clorobiocin and coumermycin A₁ (Berger & Batcho, 1978), are produced by various *Streptomyces* strains and are very potent against Gram-positive bacteria, including methicillin-resistant *Staphylococcus* strains. Novobiocin is licensed in the US as an antibiotic for use in humans (Albamycin; Pharmacia & Upjohn).

The mechanism of action of the aminocoumarin antibiotics is well studied. Bacterial DNA gyrase is their target (Hooper *et al.*, 1982; Maxwell, 1999); X-ray crystallographic examinations demonstrated that the aminocoumarin moiety and the substituted deoxysugar moiety are essential for the binding of these compounds to the B subunit of gyrase (Ali *et al.*, 1993; Lewis *et al.*, 1996; Maxwell, 1993; Tsai *et al.*, 1997). The affinity of coumermycin A₁ for intact gyrase is extremely high: 50% inhibition of gyrase is reportedly achieved by coumermycin A₁ in a concentration of only 0·004 μM, compared to 0·1 μM for novobiocin, 1·8 μM for norfloxacin and 110 μM for nalidixic acid (Peng & Marians, 1993). Correspondingly, coumermycin A₁ has been found to exhibit a much higher antibacterial activity than novobiocin (Ryan, 1979). These features make coumermycin A₁ a most interesting starting compound for the development of new aminocoumarin antibiotics, which may serve as anti-infective agents against multi-resistant Gram-positive bacteria.

Coumermycin A₁ (1, Fig. 1), produced by *Streptomyces*

Abbreviations: CID, collision-induced dissociation; ESI, electrospray ionization.

The GenBank accession number for the sequence reported in this paper is AF235050.

S.-M. Li and others

(a)

Glucose-1-
phosphate

Tyrosine

C-methyltransferase
= CouU

C-methyltransferase
= CouO

Proline

C-methyltransferase
= CouN6

Coumermycin A₁ 1

(b)

Compound | R₁ | R₂ | Produced by:
--- | --- | --- | ---
2 Coumermycin LW1 | H | CH₃ | couO mutant
3 Coumermycin LW2 | CH₃ | H | couP mutant

Fig. 1. (a) Proposed biosynthetic pathway of coumermycin A₁ in S. rishiriensis. (b) Coumermycin A₁ analogues produced by couO and couP mutants.

rishiriensis DSM 40489, contains two 3-amino-4,7-dihydroxy-8-methylcoumarin moieties, which are attached via amide bonds to a central pyrrole unit, i.e. 3-methylpyrrole 2,4-dicarboxylic acid. Two unusual 5-gem-dimethyl sugar units are linked to the 7-hydroxyl groups of the two aminocoumarin rings via glycosidic bonds. Both deoxysugars are acylated with 5-methyl-pyrrole-2-carboxylic acid at position 3.

Methyltransferases of coumermycin biosynthesis

In the structurally similar novobiocin, it has been shown that the aminocoumarin moiety is derived from L-tyrosine, and the deoxysugar (called noviose) is formed from glucose (Bunton *et al.*, 1963; Li *et al.*, 1998). The pyrrole moieties of coumermycin A₁ have been reported to be derived from L-proline (Scannell & Kong, 1969).

Coumermycin A₁ contains at least eight methyl groups, presumably derived from S-adenosylmethionine. Feeding experiments on novobiocin biosynthesis, using \[^{14}CH_3\]-labelled methionine (Birch *et al.*, 1960) and \[1-\^{13}C\]glucose (Li *et al.*, 1998), have shown that the methyl groups at C-8 of the aminocoumarin ring, the methyl groups at the 4-OH of the deoxysugar and one of the two methyl groups at C-5 of the deoxysugar probably originated from S-adenosylmethionine. The other methyl group at C-5 of the deoxysugar is derived from C-6 of glucose after reduction. Coumermycin A₁ contains two further methyl groups at C-5 of the terminal pyrrole moieties. Coumermycin derivatives lacking one or both of these methyl groups were identified in the coumermycin-producing strains (Claridge *et al.*, 1984). It might therefore be expected that these methyl groups at the terminal pyrrole units are also derived from S-adenosylmethionine. The origin of the methyl group at C-3 of the central pyrrole unit has not been established, but this group also may be derived from S-adenosylmethionine. Therefore, at least four different methyltransferases are expected to be involved in the biosynthesis of coumermycin A₁.

None of the methyltransferases involved in novobiocin or coumermycin biosynthesis has been functionally identified. Our group has recently cloned and sequenced the biosynthetic gene clusters of novobiocin from Streptomyces spheroides NCIMB 11891 (Steffensky *et al.*, 2000b), of coumermycin A₁ from *S*. rishiriensis DSM 40489 (Wang *et al.*, 2000) and of clorobiocin from Streptomyces roseochromogenes DS 12.976 (unpublished results). The three clusters share extensive similarities. To create a consistent nomenclature for the corresponding biosynthetic genes of these clusters, we have recently revised the nomenclature of the coumermycin biosynthetic genes (see GenBank entry AF235050).

Sequence analysis of the coumermycin biosynthetic gene cluster (Wang *et al.*, 2000) led to the identification of three putative methyltransferase genes, i.e. *couU*, *couO* and *couP* (formerly designated as *cumW*, *cumM* and *cumN*, respectively). The predicted gene product of *couU*, as well as that of the very similar *novU* of the novobiocin cluster, showed high sequence similarity to enzymes that have been functionally identified as C-methyltransferases carrying out the C-methylation of deoxysugar during antibiotic biosynthesis. *CouU*, for example, shows 48% identity to MtmC, which is responsible for the 3-C-methylation reaction in the biosynthesis of D-mycarose, a deoxysugar moiety of mithramycin (Gonzalez *et al.*, 2001). Likewise, *CouU* shows 37% identity to TylCIII and 35% to EryBIII, found in the biosynthetic gene clusters of tylosin and erythromycin A, respectively. Like MtmC, TylCIII and EryBIII catalyse the methylation of dTDP-L-mycarose at C-3 during the biosynthesis of these two antibiotics (Bate *et al.*, 2000; Gaisser *et al.*, 1998). *CouU* shows also 35% identity to AviG1, a 3-C-methyltransferase involved in the biosynthesis of the deoxysugar 2-deoxy-D-evalose present in the antibiotic avilamycin (Weitnauer *et al.*, 2002). Therefore *couU* was assigned to the C-methyltransferase reaction involved in the biosynthesis of the deoxysugar noviose (Wang *et al.*, 2000).

Recently, a new subgroup of radical S-adenosylmethionine proteins has been identified by bioinformatic techniques (Sofia *et al.*, 2001). The gene *couN6* of the coumermycin cluster (formerly designated as *cumK*) shows sequence similarity to these enzymes and is therefore likely to represent a methyltransferase. Since *couN6* is contained in a putative transcription unit with genes of pyrrole biosynthesis, and since no homologue for *couN6* is found in the novobiocin cluster, it appears likely that *couN6* may be responsible for the introduction of the methyl groups into the pyrrole moieties of coumermycin A₁.

In this study, we report the functional identification of two further methyltransferase genes, *couO* and *couP*, from the coumermycin A₁ cluster by means of gene inactivation and the identification of the new secondary metabolites accumulated in the defective mutants.

## METHODS

### Bacterial strains and plasmids

*S*. rishiriensis DSM 40489 [DSMZ (Deutsche Sammlung von Mikroorganismen und Zellkulturen), Braunschweig, Germany] was routinely cultivated at 28 °C for 2 days in HA medium (Steffensky *et al.*, 2000b) containing 1·0% malt extract, 0·4% yeast extract, 0·4% glucose and 1·0 mM CaCl₂ (pH 7·3).

Escherichia coli XL-1 Blue MRF′ (Stratagene) was grown in liquid or solid Luria–Bertani medium at 37 °C (Sambrook & Russell, 2001). pGem-3Zf(−) and pUC18 were purchased from Promega and Amersham Biosciences, respectively. pKC1132, a non-replicative vector carrying an apramycin resistance determinant, was described by Bierman *et al.* (1992). Carbenicillin (50 μg ml⁻¹) and apramycin (50 μg ml⁻¹) were used for selection of recombinant plasmids and strains.

### Genetic procedures

Standard methods for DNA isolation and manipulation were performed as described by Sambrook & Russell (2001) and Kieser *et al.* (2000). DNA fragments were isolated from agarose gels using a NucleoSpin 2 in 1 extraction kit (Macherey-Nagel). Genomic DNA was isolated from Streptomyces strains by lysozyme treatment and phenol/chloroform extraction as described elsewhere (Kieser *et al.*, 2000).

Southern blot analysis was performed on Hybond-N membranes (Amersham Biosciences) with digoxigenin-labelled probes by using DIG high prime DNA labelling and detection kit II (Roche Molecular Biochemicals).

### Construction of the vector pLW3 for in-frame gene inactivation of *couO*

A *SphI–PstI* fragment of 3·89 kb containing genes *couN6*, *couN7*, *couO* and *couP* was isolated from cosmid 4-2H (Wang *et al.*, 2000) and cloned into the same sites of pGem-3Zf(−), resulting in vector pLW1. Vector pLW2, containing a deletion of 366 bp within *couO*, was constructed by ligation of two restriction fragments of pLW1, i.e. a 5·09 kb *EcoRI–SfoI* fragment and a 0·98 kb *SmaI–EcoRI*

S.-M. Li and others

(a)
Wild-type (WT)


SphI
BamHI
SfoI
Smal
BamHI
PstI
couN6
couN7
couO
couP



[SfoI/Smal]



SphI
HindIII
BamHI
BamHI
PstI Xbal /EcoRI
couO*



apra



double cross-over
recombination



[SfoI/Smal]



SphI
BamHI
BamHI
PstI
couO*


(b)
Wild-type
LW-O11, SCO
LW-O41, DCO
LW-O61, RV
LW-O316, DCO
LW-O318, RV
kb
3·6
2·8
1·9
1·48
1·07 kb
1·0

Fig. 2. Inactivation of the gene couO of the coumermycin biosynthetic gene cluster. (a) Schematic presentation of the gene-inactivation experiment. apra, apramycin-resistance gene. (b) Southern blot analysis of wild-type and mutants. Genomic DNA was restricted by BamHI. The indicated 1·43 kb BamHI fragment containing couO was used as probe. SCO, single-crossover mutant; DCO, double-crossover mutant; RV, reversion to wild-type.

(a)
Wild-type (WT)


NarI
HincII BclI
678 bp
BclI PstI
HincI
NarI
couO
couP
couR1



XbaI
NarI
HincII
BclI
PstI
HincI
NarI HindIII
couP*



apra



double cross-over
recombination



NarI
HincII
BclI
PstI
HincI
NarI
couP*


(b)
Wild-type
LW-P10, SCO
LW-P634, DCO
LW-P582, RV
kb
3·6
2·8
1·9
2·40 kb
1·72 kb
1·48

Fig. 3. Inactivation of the gene couP of the coumermycin biosynthetic gene cluster. (a) Schematic presentation of the gene-inactivation experiment. (b) Southern blot analysis of wild-type and mutants. Genomic DNA was restricted by HincII. The indicated 1·1 kb PstI–HincII fragment containing couR1 was used as probe. Abbreviations are as given in Fig. 2.

fragment. pLW3 was obtained by cloning of a 3·52 kb HindIII–PstI fragment from pLW2 into the same sites of vector pKC1132 (Fig. 2a).

Construction of the vector pLW9 for in-frame gene inactivation of couP. A NarI fragment of 3·08 kb containing couO, couP and couR1 was isolated from cosmid 4-2H and cloned into the AccI sites of pUC18 to give vector pLW7. pLW7 was restricted with BclI, and the 5·09 kb fragment was religated, resulting in a deletion of 678 bp within the couP

gene. The vector obtained was termed pLW8. The 2·56 kb XbaI–HindIII fragment was released from pLW8 and ligated into the same sites of pKC1132, resulting in the inactivation vector for couP, pLW9 (Fig. 3a).

Transformation of S. rishiriensis and selection for recombinant mutants. Transformation of the S. rishiriensis wild-type with pLW3 and pLW9 was carried out by polyethylene glycol-mediated protoplast transformation (Kieser et al., 2000). For preparation of protoplasts, mycelia of S.

Methyltransferases of coumermycin biosynthesis

Coumermycin A₁ (1, Mᵣ=1109), positive-ion reactions: m/z 1110 ([M+H]⁺)→m/z 108, m/z 1110→m/z 282 and m/z 1110→m/z 622; negative-ion reactions: m/z 1108 ([M−H]⁻)→m/z 206 and m/z 1108→m/z 487.

Coumermycin LW1 (2, Mᵣ=1081): positive-ion reactions: m/z 1082 ([M+H]⁺)→m/z 108 and m/z 1082→m/z 282; negative-ion reactions: m/z 1080 ([M−H]⁻)→m/z 192 and m/z 1080→m/z 473.

Coumermycin LW2 (3, Mᵣ=1081): positive-ion reactions: m/z 1082 ([M+H]⁺)→m/z 108 and m/z 1082→m/z 268; negative-ion reactions: m/z 1080 ([M−H]⁻)→m/z 206 and m/z 1080→m/z 473.

Assay for antibacterial activity. The antibacterial activities of coumermycin derivatives were tested using *Bacillus subtilis* ATCC 14893. Cultures of *S. rishiriensis* (wild-type and mutant strains) were cultured and extracted as described for HPLC analysis. For the bioassays, 30 μl of an ethanolic solution of the ethyl acetate extracts was applied to filter-paper discs (5 mm diameter) and air-dried for 30 min. The discs were then placed on Difco nutrient agar plates (Kieser et al., 2000) containing approximately 2 × 10⁵ spores of *B. subtilis* per ml agar medium. After culture overnight at 37 °C, the diameter of the growth-inhibition zone was determined.

**RESULTS**

**Sequence analysis of couO and couP**

Two adjacent ORFs, *couO* and *couP*, encoding putative methyltransferases were found in the coumermycin A₁ biosynthetic gene cluster (Wang et al., 2000). Two very similar genes, *novO* and *novP*, are found in the novobiocin biosynthetic cluster. The functions of these genes have not been established.

The predicted gene product of *couO* comprises 230 amino acids and shows only low sequence similarity to putative methyltransferases in the database, e.g. 29% identity (at amino acid level) to the C-methyltransferase UbiE (GenBank accession no. AE010901) of the ubiquinone/menaquinone biosynthesis in *Methanosarcina acetivorans* (Galagan et al., 2002). Conserved motif III [LL(R/K)PGG(R/I/L)(L/I)(L/F/I/V)(I/L)] for S-adenosylmethionine-dependent methyltransferases (Kagan & Clarke 1994) is found from amino acid 135 to 144 of CouO (LVKPGGAILN).

In the biosynthesis of coumermycin A₁, C-methyltransferases may be involved in the methylation of C-8 of the aminocoumarin ring, of C-5 of the terminal pyrrole, and possibly of C-3 of the central pyrrole unit, besides the C-methylation reaction at C-5 of the deoxysugar.

The predicted gene product of *couP* consists of 276 amino acids. In contrast to CouO, CouP shows high similarities to known deoxysugar O-methyltransferases in the database, e.g. 57% identity to 3-O-methyltransferase MycF catalysing the methylation at the 3-position of the deoxysugar mycinose of mycinamicin III (Inouye et al., 1994), 59% to ElmMIII responsible for 4-O-methylation of the permethylated L-rhamnose in the biosynthesis of elloramycin A (Patallo et al., 2001), and 53% to 3-O-methyltransferase TylF converting macrocin to tylosin by O-methylation at C-3 of the deoxysugar

S.-M. Li and others

mycinose (Fouces *et al.*, 1999). Conserved motif I [(V/I/L)(L/V)(D/E)(V/I)G(G/C)G(T/P)G] for S-adenosylmethionine-dependent methyltransferases (Kagan & Clarke, 1994) was found in the predicted CouP from amino acid 105 to 113 (LVETGVWRG).

### Construction of the inactivation vectors

To provide experimental evidence for the functions of *couO* and *couP*, inactivation vectors were prepared for both genes. A vector containing an in-frame deletion within the coding sequence of *couO* was obtained by the cloning of a *SphI-SfoI* fragment of 1926 bp and a *SmaI-EcoRI* fragment of 983 bp from a cosmid containing the coumermycin cluster into the non-replicative vector pKC1132 (containing an apramycin resistance determinant), via the cloning vector pGem-3Zf(−). The resulting inactivation vector was termed pLW3 (Fig. 2a). The predicted product of *couO* is thereby expected to be shortened from 230 (wild-type) to 108 (mutants) amino acids.

The inactivation vector for *couP* (pLW9) (Fig. 3a) contained an in-frame deletion within the structural gene of *couP*. This could be achieved by excision of a 678 bp *BclI* fragment from a subclone of the coumermycin gene cluster, and ligation of the resulting fragment into pKC1132. The predicted product of the shortened *couP* consists of only 50 amino acids, in comparison to 276 amino acids in the wild-type.

### Transformation of *S. rishiriensis* with pLW3, and selection of *couO*-defective mutants

Vector pLW3 was transformed into *S. rishiriensis* protoplasts (Kieser *et al.*, 2000; Wang *et al.*, 2000), and apramycin-resistant colonies were selected. Southern blotting confirmed that pLW3 had been integrated into these mutants via a single-crossover recombination event (Fig. 2b). One of these single-crossover mutants (LW-O11) was grown in the absence of apramycin, sporulated, and examined for loss of resistance as a consequence of double-crossover events. Four sensitive colonies, LW-O41, LW-O61, LW-O316 and LW-O318, were obtained and examined further. Chromosomal DNA from these four strains and from the *S. rishiriensis* wild-type was digested with *BamHI* and hybridized with a probe containing the complete sequence of the *couO* gene. A band at 1·43 kb, corresponding to the intact *couO*, was detected in the *S. rishiriensis* wild-type, while chromosomal DNA from mutants LW-O41 and LW-O316 showed a band at 1·07 kb, corresponding to the inactivated *couO* (Fig. 2b). The other two sensitive strains, LW-O61 and LW-O318, showed a band of the same size as the wild-type (1·43 kb) and therefore represented reversion to the wild-type.

### Transformation of *S. rishiriensis* with pLW9, and selection of *couP*-defective mutants

The introduction of vector pLW9 into the *S. rishiriensis* genome and the selection of double-crossover mutants were carried out as described above for pLW3. This procedure yielded the single-crossover mutant LW-P10 and, subsequently, the antibiotic-sensitive, double-crossover mutants LW-P582 and LW-P634. Southern blotting with chromosomal DNA, restricted with *HincII* and hybridized with a 1·1 kb *PstI-HincII* fragment containing the region just downstream of *couP* as probe, showed that LW-P634 was the desired *couP* mutant, showing a band at 1·72 kb. LW-P582 represented a reversion to the wild-type, with a band at 2·40 kb, as for the wild-type (Fig. 3b).

### Identification of secondary metabolites in the defective mutants

*S. rishiriensis* wild-type, the *couO* mutants (LW-O41 and LW-O316) and the *couP* mutant (LW-P634) were cultured under the conditions described by Scannell & Kong (1969). After extraction of the cultures with ethyl acetate, secondary metabolites were analysed by HPLC. As shown in Fig. 4, the production of coumermycin A₁ was abolished in all mutants. The HPLC chromatogram of strain LW-O316 was identical to that of LW-O41 (data not shown). Instead, the *couO* mutant LW-O41 showed a dominant peak at 19·7 min (2) and the *couP* mutant showed a dominant peak at 17·5 min (3). The

Fig. 4. HPLC analysis of secondary metabolites produced by *S. rishiriensis* DSM 40489 wild-type (a), the *couO*-defective mutant, LW-O41 (b), and the *couP*-defective mutant, LW-P634 (c). The samples were analysed with an RP C-18 column. Detection was at 345 nm.

two peaks 2 and 3 showed UV spectra similar to that for coumermycin A₁ with two maxima at 275 and 345 nm (data not shown). HPLC-MS showed both substances to have identical molecular ions: \( m/z \) at 1082 \([M+H]^+\) in the positive mode and \( m/z \) at 1080 \([M-H]^{-}\)in the negative mode. Therefore substances 2 and 3 have a molecular mass of 1081, corresponding to a loss of two methyl groups relative to coumermycin A₁ (\( M_r \) 1109).

To determine the positions of the remaining methyl groups in 2 and 3, we employed MS using selected reaction monitoring via ESI during positive and negative ionization. Fragmentation of coumermycin A₁ occurred at the amide, the glycosidic and the ester bonds, producing ions at \( m/z \) 487 (ESI−), \( m/z \) 282 (ESI+) and \( m/z \) 108 (ESI+), respectively (Fig. 5a). In addition, the ion at \( m/z \) 206 (ESI−), representing the methylated aminocoumarin ring (4), was observed. Fig. 5 shows, as examples, the selected reaction monitoring of the reactions of \( m/z \) 1110 \([M+H]^+\)→\( m/z \) 282 as well as \( m/z \) 1108 \([M-H]^{-}\)→\( m/z \) 206.

In compound 2 produced by the couO mutant, reaction of \( m/z \) 1082 \([M+H]^+\)→\( m/z \) 282 was observed, confirming the presence of the same sugar and the same terminal pyrrole units as from coumermycin A₁ 1. In addition, a reaction of \( m/z \) 1080 \([M-H]^{-}\)→\( m/z \) 192 (rather than \( m/z \) 1080→\( m/z \) 206) was observed, indicating that the aminocoumarin moiety now lacked a methyl group, as shown in structure 5 (Fig. 5). This proved that the couO mutant produced a new coumermycin, lacking the 8-methyl groups at both aminocoumarin rings (Fig. 5, 2). This new coumermycin was designated coumermycin LW1.

In compound 3 produced by the couP mutant, selected reaction monitoring showed the reaction confirming the presence of the methylated aminocoumarin moiety 4. However, instead of the \( m/z \) 1082 → \( m/z \) 282 reaction, a reaction of \( m/z \) 1082 \([M+H]^+\)→\( m/z \) 268 was now observed, indicating the loss of a methyl group from either the deoxysugar or the terminal pyrrole unit. However, since the fragment at \( m/z \) 108 (ESI+) was still observed, it is obvious that the pyrrole moiety is unchanged in comparison to coumermycin A₁, and that therefore a methyl group is lacking from the deoxysugar in comparison to coumermycin A₁.

The S. rishiriensis wild-type produced about 5 mg coumermycin A₁ per litre medium under the culture conditions described. Judging from the peak areas in the HPLC chromatograms, the couO and couP mutants produced at least as much of the new metabolites, i.e. coumermycin LW1 and LW2, respectively (Fig. 4).

Antibiotic activity of culture extracts from the couO and the couP mutants

Cultures of S. rishiriensis wild-type as well as of the couO and the couP mutants were extracted with ethyl acetate and assayed (against B. subtilis) for antibiotic activity (Fig. 6). All three strains showed similar antibacterial activity, indicating that neither the methyl

group at C-8 of the aminocoumarin ring nor the methyl group at 4-OH of the deoxysugar was essential for the antibiotic activity of the coumermycins.

## DISCUSSION

In the present study, we have inactivated the putative methyltransferase genes couO and couP by in-frame deletion, and have identified the products formed in the resulting mutants by HPLC-MS analysis, using selected reaction monitoring for structure elucidation.

Inactivation of couO led to a new product (coumermycin LW1) which lacked the methyl groups of the aminocoumarin moieties. This provides strong evidence that couO encodes the C-methyltransferase responsible for the methylation of C-8 of these aminocoumarin rings. However, it is still unclear at which stage of coumermycin biosynthesis (or correspondingly of novobiocin biosynthesis) this reaction occurs. The aminocoumarin ring is derived from tyrosine, and the methylation reaction may take place prior to, during, or after the conversion of L-tyrosine to 3-amino-4,7-dihydroxycoumarin (Chen & Walsh, 2001). We have previously inactivated an early step of deoxysugar biosynthesis in novobiocin formation, and the mutant accumulated large amounts of novobiocic acid, the aglycone of novobiocin (Steffensky et al., 2000b). This indicates that methylation of C-8 takes place prior to glycosylation, and probably before acylation of the amino group of the aminocoumarin (Steffensky et al., 2000a). Nevertheless, the unmethylated aminocoumarin produced by the couO mutant was accepted as substrate by all subsequent enzymes of coumermycin biosynthesis (Fig. 1), and the resulting product, coumermycin LW1, was accumulated in an amount at least as high as that of coumermycin A₁ in the wild-type. This proves that there is no strict substrate specificity for the substituent in this position of the aminocoumarin. This is also indicated by the natural occurrence of novobiocin analogues lacking the 8-methyl group, as recently reported by Sasaki et al. (2001), and by the simultaneous occurrence of simocyclones with and without a substituent at the corresponding position of the aminocoumarin ring (Theobald et al., 2000).

Inactivation of another methyltransferase gene, couP, in the present study also led to the formation of a new coumermycin (termed coumermycin LW2) lacking two methyl groups. Mass spectroscopic analysis with selected reaction monitoring proved that these methyl groups were lacking from the deoxysugar moiety. Since C-methylation at C-5 had been assigned to couU, this result suggests that couP encodes a methyltransferase responsible for the methylation of the 4-hydroxyl groups of the deoxysugar moieties of coumermycin. This is also in accordance with the sequence similarity of couP to known deoxysugar O-methyltransferases (see Results).

In novobiocin biosynthesis, acylation of 3-OH and methylation of 4-OH of the deoxysugar moiety have been suggested as the two final steps of the biosynthetic

S.-M. Li and others

(a)

![Chemical Structure 1](chemical_structure_1.png)

negative ESI  
m/z 1108 → m/z 206  

positive ESI  
m/z 1110 → m/z 282  

(b)

![Chemical Structure 2](chemical_structure_2.png)

negative ESI  
m/z 1080 → m/z 192  

positive ESI  
m/z 1082 → m/z 282  

(c)

![Chemical Structure 3](chemical_structure_3.png)

negative ESI  
m/z 1080 → m/z 206  

positive ESI  
m/z 1082 → m/z 268  

Fig. 5. Fragmentation scheme for coumermycin derivatives, and the corresponding LC-MS chromatograms with detection by selected reaction monitoring.

Methyltransferases of coumermycin biosynthesis

hydrolyzes ATP and binds coumarin drugs. *Biochemistry* 32, 2717–2724.

Bate, N., Butler, A. R., Smith, I. P. & Cundliffe, E. (2000). The mycarose-biosynthetic genes of *Streptomyces fradiae*, producer of tylosin. *Microbiology* 146, 139–146.

Berger, J. & Batcho, A. D. (1978). Coumarin-glycoside antibiotics. *J Chromatogr Libr* 15, 101–158.

Bierman, M., Logan, R., O’Brien, K., Seno, E. T., Rao, R. N. & Schoner, B. E. (1992). Plasmid cloning vectors for the conjugal transfer of DNA from *Escherichia coli* to *Streptomyces* spp. *Gene* 116, 43–49.

Birch, A. J., Cameron, D. W., Holloway, P. W. & Rickards, R. W. (1960). Further examples of biological C-methylation. Novobiocin and actinomycin. *Tetrahedron Lett.*, 26–31.

Bunton, C. A., Kenner, G. W., Robinson, M. J. T. & Webster, B. R. (1963). Experiments related to the biosynthesis of novobiocin and other coumarins. *Tetrahedron* 19, 1001–1010.

Chen, H. & Walsh, C. T. (2001). Coumarin formation in novobiocin biosynthesis: beta-hydroxylation of the aminoacyl enzyme tyrosyl-S-NovH by a cytochrome P450 NovI. *Chem Biol* 8, 301–312.

Claridge, C. A., Elander, R. P. & Price, K. E. (1984). The coumermycins: properties, biosynthesis, and fermentation. *Drugs Pharm Sci* 22, 413–425.

Fouces, R., Mellado, E., Díez, B. & Barredo, J. L. (1999). The tylosin biosynthetic cluster from *Streptomyces fradiae*: genetic organization of the left region. *Microbiology* 145, 855–868.

Gaisser, S., Böhm, G. A., Doumith, M., Raynal, M. C., Dhillon, N., Cortés, J. & Leadlay, P. F. (1998). Analysis of *eryBI*, *eryBIII* and *eryBVII* from the erythromycin biosynthetic gene cluster in *Saccharopolyspora erythraea*. *Mol Gen Genet* 258, 78–88.

Galagan, J. E., Nusbaum, C., Roy, A. & 52 other authors (2002). The genome of *M. acetivorans* reveals extensive metabolic and physiological diversity. *Genome Res* 12, 532–542.

González, A., Remsing, L. L., Lombó, F. & 7 other authors (2001). The *mtmVUC* genes of the mithramycin gene cluster in *Streptomyces argillaceus* are involved in the biosynthesis of the sugar moieties. *Mol Gen Genet* 264, 827–835.

Hooper, D. C., Wolfson, J. S., McHugh, G. L., Winters, M. B. & Swartz, M. N. (1982). Effects of novobiocin, coumermycin A₁, clorobiocin, and their analogs on *Escherichia coli* DNA gyrase and bacterial growth. *Antimicrob Agents Chemother* 22, 662–671.

Inouye, M., Suzuki, H., Takada, Y., Muto, N., Horinouchi, S. & Beppu, T. (1994). A gene encoding mycinamicin III O-methyltransferase from *Micromonospora griseorubida*. *Gene* 141, 121–124.

Kagan, R. M. & Clarke, S. (1994). Widespread occurrence of three sequence motifs in diverse S-adenosylmethionine-dependent methyltransferases suggests a common structure for these enzymes. *Arch Biochem Biophys* 310, 417–427.

Kieser, T., Bibb, M. J., Buttner, M. J., Chater, K. F. & Hopwood, D. A. (2000). *Practical Streptomyces Genetics*. Norwich: John Innes Foundation.

Kominek, L. A. & Sebek, O. K. (1974). Biosynthesis of novobiocin and related coumarin antibiotics. *Dev Ind Microbiol* 15, 60–69.

Lewis, R. J., Singh, O. M., Smith, C. V., Skarzynski, T., Maxwell, A., Wonacott, A. J. & Wigley, D. B. (1996). The nature of inhibition of DNA gyrase by the coumarins and the cyclothialidines revealed by X-ray crystallography. *EMBO J* 15, 1412–1420.

Li, S.-M., Hennig, S. & Heide, L. (1998). Biosynthesis of the dimethylallyl moiety of novobiocin via a non-mevalonate pathway. *Tetrahedron Lett* 39, 2717–2720.


Fig. 6. Antibacterial activity of culture extracts from *S. rishiriensis* wild-type (1), the *couO*-defective mutant (2) and the *couP*-defective mutant (3). Extracts from non-inoculated culture medium (4) and a coumermycin A₁ standard (5) were used for comparison. Antibiotic activity against *B. subtilis* was determined as described in Methods.


pathway, since novobiocin derivatives lacking either or both substituent(s) have been identified in mutants of the novobiocin producer *Streptomyces niveus* (Kominek & Sebek, 1974). At first sight, the very high accumulation of coumermycin LW2 in the *couP* mutant may appear to indicate that 4-O-methylation is the last step of coumermycin A₁ biosynthesis. In the study by Kominek & Sebek (1974) mentioned above, however, it was shown by mutation and biotransformation experiments that 4-O-methylation of the deoxysugar could be achieved before or after acylation of the 3-hydroxyl group. It has been shown repeatedly that several enzymes of coumarin antibiotic biosynthesis are not strictly substrate specific. The flexibility for the substituent at the position 8 of the aminocoumarin discussed above provides one example. Another is the occurrence of a novobiocin derivative lacking one of the 5-methyl groups of noviose, detected as a side-product from a novobiocin producer (Sasaki *et al.*, 2001). Therefore, the exact sequence of the biosynthetic reactions depicted in Fig. 1 remains tentative. The flexibility of the enzymes of coumarin antibiotic biosynthesis for different substrates presents a very useful feature in experiments for the production of new antibiotics by combinatorial biosynthesis (Tang & McDaniel, 2001; Yoon *et al.*, 2002), and this study has provided two examples of the production of new antibiotics by genetic methods.


**ACKNOWLEDGEMENTS**

This work was supported by a grant from the Deutsche Forschungsgemeinschaft (to L.H. and S.-M.Li) and by the Fonds der Chemischen Industrie.


**REFERENCES**

Ali, J. A., Jackson, A. P., Howells, A. J. & Maxwell, A. (1993). The 43-kilodalton N-terminal fragment of the DNA gyrase B protein

MacNeil, D. J., Gewain, K. M., Ruby, C. L., Dezeny, G., Gibbons, P. H. & MacNeil, T. (1992). Analysis of *Streptomyces avermitilis* genes required for avermectin biosynthesis utilizing a novel integration vector. *Gene* 111, 61–68.

Maxwell, A. (1993). The interaction between coumarin drugs and DNA gyrase. *Mol Microbiol* 9, 681–686.

Maxwell, A. (1999). DNA gyrase as a drug target. *Biochem Soc Trans* 27, 48–53.

Oh, S. H. & Chater, K. F. (1997). Denaturation of circular or linear DNA facilitates targeted integrative transformation of *Streptomyces coelicolor* A3(2): possible relevance to other organisms. *J Bacteriol* 179, 122–127.

Patallo, E. P., Blanco, G., Fischer, C., Braña, A. F., Rohr, J., Méndez, C. & Salas, J. A. (2001). Deoxysugar methylation during biosynthesis of the antitumor polyketide elloramycin by *Streptomyces olivaceus*. Characterization of three methyltransferase genes. *J Biol Chem* 276, 18765–18774.

Peng, H. & Marians, K. J. (1993). *Escherichia coli* topoisomerase IV. Purification, characterization, subunit structure, and subunit interactions. *J Biol Chem* 268, 24481–24490.

Ryan, M. J. (1979). Novobiocin and coumermycin A₁. In *Antibiotics*, vol. 5, part 1. *Mechanism of Action of Antibacterial Agents*, pp. 214–234. Edited by F. E. Hahn. Berlin, Heidelberg, New York: Springer-Verlag.

Sambrook, J. & Russell, D. W. (2001). *Molecular Cloning: a Laboratory Manual*, 3rd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Sasaki, T., Igarashi, Y., Saito, N. & Furumai, T. (2001). TPU-0031-A and B, new antibiotics of the novobiocin group produced by *Streptomyces* sp. TP-A0556. *J Antibiot* 54, 441–447.

Scannell, J. & Kong, Y. L. (1969). Biosynthesis of coumermycin A₁: incorporation of L-proline into the pyrrole groups. *Antimicrob Agents Chemother* 9, 139–143.

Sofia, H. J., Chen, G., Hetzler, B. G., Reyes-Spindola, J. F. & Miller, N. E. (2001). Radical SAM, a novel protein superfamily linking unresolved steps in familiar biosynthetic pathways with radical mechanisms: functional characterization using new analysis and information visualization methods. *Nucleic Acids Res* 29, 1097–1106.

Steffensky, M., Li, S. M. & Heide, L. (2000a). Cloning, overexpression, and purification of novobiocic acid synthetase from *Streptomyces spheroides* NCIMB 11891. *J Biol Chem* 275, 21754–21760.

Steffensky, M., Mühlenweg, A., Wang, Z. X., Li, S. M. & Heide, L. (2000b). Identification of the novobiocin biosynthetic gene cluster of *Streptomyces spheroides* NCIB 11891. *Antimicrob Agents Chemother* 44, 1214–1222.

Tang, L. & McDaniel, R. (2001). Construction of desosamine containing polyketide libraries using a glycosyltransferase with broad substrate specificity. *Chem Biol* 8, 547–555.

Theobald, U., Schimana, J. & Fiedler, H. P. (2000). Microbial growth and production kinetics of *Streptomyces antibioticus* Tü 6040. *Antonie Leeuwenhoek* 78, 307–313.

Tsai, F. T., Singh, O. M., Skarzynski, T. & 9 other authors (1997). The high-resolution crystal structure of a 24-kDa gyrase B fragment from *E. coli* complexed with one of the most potent coumarin inhibitors, clorobiocin. *Proteins* 28, 41–52.

Wang, Z. X., Li, S. M. & Heide, L. (2000). Identification of the coumermycin A1 biosynthetic gene cluster of *Streptomyces rishiriensis* DSM 40489. *Antimicrob Agents Chemother* 44, 3040–3048.

Weitnauer, G., Gaisser, S., Kellenberger, L., Leadlay, P. F. & Bechthold, A. (2002). Analysis of a C-methyltransferase gene (*aviG1*) involved in avilamycin biosynthesis in *Streptomyces viridochromogenes* Tü57 and complementation of a *Saccharopolyspora erythraea eryBIlll* mutant by *aviG1*. *Microbiology* 148, 373–379.

Yoon, Y. J., Beck, B. J., Kim, B. S., Kang, H. Y., Reynolds, K. A. & Sherman, D. H. (2002). Generation of multiple bioactive macro-lides by hybrid modular polyketide syntheses in *Streptomyces venezuelae*. *Chem Biol* 9, 203–214.

Received 14 May 2002; revised 18 June 2002; accepted 20 June 2002.
