
7. N. Shigesada, K. Kawasaki, *Biological Invasions: Theory and Practice* (Oxford Univ. Press, New York, 1997).

8. A. Hastings, S. Harrison, K. McCann, *Proc. R. Soc. London Ser. B* **264**, 1837 (1997).

9. J. L. Maron, S. Harrison, *Science* **278**, 1619 (1997).

10. W. G. Wilson, S. P. Harrison, A. Hastings, K. McCann, *J. Anim. Ecol.* **68**, 94 (1999).

11. E. Ranta, V. Kaitala, J. Lindström, *Ecography* **20**, 454 (1997).

12. D. T. Haydon, *Theor. Popul. Biol.* **58**, 239 (2000).

13. P. J. Bacon, *Population Dynamics of Rabies in Wildlife* (Academic Press, London, 1985).

14. B. T. Grenfell, O. N. Bjørnstad, J. Kappey, *Nature* **414**, 716 (2001).

15. W. Baltensweiler, *Oecologia* **94**, 62 (1993).

16. , D. Rubli, *Forest Snow Landscape Res.* **74**, 1 (1999).

17. C. Auer, *Eidg. Anst. Forstl. Versuchswes. Mitt.* **53**, 71 (1977).

18. See supplemental data on Science Online.

19. W. Baltensweiler, *Eidg. Anst. Forstl. Versuchswes. Ber.* **270**, 215 (1985).

20. We analyzed outbreak data that originated from maps of visually detected defoliation in the Central European Alps between 1961 and 1998 (16). The surveys were made in cooperation with the forest administration agencies of the Alpine countries (France, Italy, Switzerland, and Austria). Maps were digitized to show the presence or absence of defoliation in each 1 km × 1 km cell. These data were aggregated to the proportion of defoliation in 20 km × 20 km grid cells as a surrogate for local abundance (18). Grid cells were excluded from further analyses if the maximum defoliation was less than 1% or if >90% of the years were completely without defoliation. This was done to prevent problems arising from analysis of time series dominated by zero values.

21. O. N. Bjørnstad, J. Bascompte, *J. Anim. Ecol.* **70**, 924 (2001).

22. O. N. Bjørnstad, W. Falck, *Environ. Ecol. Statist.* **8**, 53 (2001).

23. The cross-correlation, $w_{ij,\Delta}$, between two time series of abundance at location $H_i$ and at location $i$ and $H_j$ at location $j$ (either at similar or different locations) lagged by $\Delta$ years relative to each other (either in the same year, $\Delta = 0$, or in lagged years), is given by

    $$
    w_{ij,\Delta} = \left(H_i - \langle H_i \rangle\right) \times \left(H_{j,\Delta} - \langle H_j \rangle\right)^T / \sigma_{H_i} \sigma_{H_j}
    $$

    where $\langle H. \rangle$ represents the mean of the time series, $\sigma_H.$ represents the standard deviations, underscored symbols represent vectors (time series), $\times$ denotes matrix multiplication, and the superscript $T$ denotes matrix transposition. The correlation depends on the spatial distance and direction separating the populations, $r_{ij,\theta}$. We thus measure the distance as the projections onto a sequence of cardinal directions ($\theta$: $0, \pi/16, \ldots, 31\pi/32$) (28) according to $r_{ij,\theta} = x_i \sin(\theta) + y_i \cos(\theta) - x_j \sin(\theta) - y_j \cos(\theta)$. By extension of existing methods (21), we estimate the LCCF as a function of direction and time lag as the kernel regression of cross-correlation on directional distance:

    $$
    \tilde{C}(r_\theta, \Delta) = \frac{\sum_{i=1}^n \sum_{j=i+1}^n K(r_{ij,\theta}/h) w_{ij,\Delta}}{\sum_{i=1}^n \sum_{j=i+1}^n K(r_{ij,\theta}/h)}
    $$

    Hence, we estimate spatial correlation for time lags of 0, 1, and 2 years and distance lags up to ±600 km. An R/S-plus library for calculating the metrics is available upon request (onb1@psu.edu).

24. M. P. Hassell, *The Dynamics of Arthropod Predator-Prey Systems* (Princeton Univ. Press, Princeton, NJ, 1978).

25. P. Turchin, *Complex Population Dynamics: A Theoretical/Empirical Synthesis* (Princeton Univ. Press, Princeton, NJ, 2002).

26. The local dynamics of location $i$ of the moth, $H_i$, and its parasitoid, $P_i$, are given as a density-dependent version of the Nicholson-Bailey model

    $$
    H'_{i,t+1} = H_{i,t} \exp\left\{r_{i,0}\left[1 - \left(\frac{H_{i,t}}{K}\right)\right]\right\} \exp\left(\frac{-aP_{i,t}}{1 + awP_{i,t}}\right)
    $$

    where $H_{i,t}$ is the density of hosts at generation $t$, $P_{i,t}$ is the density of parasitoids at generation $t$, $r_{i,0}$ is per capita rate of population change at location $i$, $K$ is carrying capacity, $a$ is parasitoid search efficiency, and $w$ is a conversion constant. We used the following parameters (25): $a = 2.5$, $w = 0.17$, and $K = 250$. In the habitat homogeneous case, $r_{i,0} = 2.5$ for all populations. In the heterogeneous case, $r_{i,0}$ is distributed along a gradient across the lattice from 2.3 to 2.5. A fraction $\mu_h$ of adult hosts and a fraction $\mu_p$ of adult parasitoids are assumed to disperse into the eight surrounding patches (3). The equations for the dispersal stage can be written as

    $$
    H_{i,t+1} = (1 - \mu_h)H'_{i,t+1} + \mu_h \overline{H'}_{i,t+1}
    $$

    $$
    P_{i,t+1} = (1 - \mu_p)P'_{i,t+1} + \mu_p \overline{P'}_{i,t+1}
    $$

    where $H_{i,t+1}$ and $P_{i,t+1}$ denote the post-dispersal abundance of moths and parasitoids at patch $i$, and $H'_{i,t}$ and $P'_{i,t}$ denote predispersal abundances. The quantities $\overline{H}$ and $\overline{P}$ are the neighborhood host and parasitoid densities (over the eight nearest neighboring patches). The lattice is assumed to have absorbing boundaries. We discarded the first 2000 generations of each simulation; the spatiotemporal patterns from the last 500 generations were then analyzed using the same method as used in the larch budmoth data analysis. The local dynamics for the moth-plant quality model are identical to those in the model proposed by Turchin (25). The moth dispersal stage is again modeled according to Eq. 5.

27. O. N. Bjørnstad, R. A. Ims, X. Lambin, *Trends Ecol. Evol.* **14**, 427 (1999).

28. X. Lambin, D. A. Elston, S. J. Petty, J. L. MacKinnon, *Proc. R. Soc. London Ser. B* **265**, 1491 (1998).

29. Supported by USDA, the Academy of Finland, and the National Center for Ecological Analysis and Synthesis (funded by NSF, the University of California, and UC Santa Barbara). We thank A. Satake, P. Tobin, and three anonymous reviewers for comments on earlier versions of the manuscript.

**Supporting Online Material**

www.sciencemag.org/cgi/content/full/298/5595/1020/DC1

Figs. S1 to S4

18 June 2002; accepted 22 August 2002

# Microbial Dehalorespiration with 1,1,1-Trichloroethane

Baolin Sun,$^{1}$ Benjamin M. Griffin,$^{1,2}$ Héctor L. Ayala-del-Río,$^{1,2}$ Syed A. Hashsham,$^{1,3}$ James M. Tiedje$^{1,2,*}$

1,1,1-Trichloroethane (TCA) is a ubiquitous environmental pollutant because of its widespread use as an industrial solvent, its improper disposal, and its substantial emission to the atmosphere. We report the isolation of an anaerobic bacterium, strain TCA1, that reductively dechlorinates TCA to 1,1-dichloroethane and chloroethane. Strain TCA1 required H₂ as an electron donor and TCA as an electron acceptor for growth, indicating that dechlorination is a respiratory process. Phylogenetic analysis indicated that strain TCA1 is related to gram-positive bacteria with low DNA G+C content and that its closest relative is *Dehalobacter restrictus*, an obligate H₂-oxidizing, chloroethene-respiring bacterium.

TCA is a synthetic organic solvent widely used in industrial processes and is a major environmental pollutant commonly found in soil (1), groundwater (2), and the atmosphere (3). TCA is present in at least 696 of the 1430 National Priorities List sites identified by the U.S. Environmental Protection Agency (EPA) (1). Because of TCA's adverse effects on human health, the EPA has set a maximum contaminant level of 200 μg/liter in drinking water (4). TCA is also listed as an ozone-depleting substance by the United Nations Environment Programme (5). Even when released to soil or leached to groundwater, the primary environmental fate of TCA is volatilization to the atmosphere, where it interacts with ozone and contributes to the erosion of the ozone layer (1, 5). TCA is often a co-contaminant in aquifers with chlorinated ethenes, especially tetrachloroethene (PCE) and trichloroethene (TCE), because they have similar industrial uses. Although in situ bioremediation processes for the chloroethenes are known (6), TCA remediation remains problematic and can prevent site restoration.

TCA undergoes slow abiotic degradation to acetic acid and 1,1-dichloroethene, an EPA priority pollutant (7). Biotransformation of TCA has been observed under aerobic and anaerobic conditions only in cometabolic processes (8–11). A growth-linked, or dehalorespiratory, process would be more effective for in situ bioremediation of TCA-contaminated sites, because reaction rates would be faster and natural selection would ensure growth in situ.

Although bacterial growth by dehalorespiration of chloroethenes, chlorobenzenes, 3-chlorobenzoate, and 2-chlorophenol has been well documented (12–15), bacterial growth by reductive dechlorination of TCA has not been reported until now. We describe the isolation of a bacterium capable of energy

$^1$Center for Microbial Ecology, $^2$Department of Microbiology and Molecular Genetics, $^3$Department of Civil and Environmental Engineering, Michigan State University, East Lansing, MI 48824–1325, USA.

*To whom correspondence should be addressed. Email: tiedjej@msu.edu*

conservation for growth through the reductive dechlorination of TCA.

Isolation was initiated from a sediment microcosm that reductively dechlorinated TCA (16). Single colonies were subcultured and dechlorination activity was maintained in deep agarose shake cultures until a pure culture was obtained. The isolate is designated strain TCA1 and is a motile, short rod with a diameter of 0.4 to 0.6 μm and a length of 1.0 to 2.0 μm (Fig. 1). Cells stain gram-negative. No spores were observed in starved cultures. Although strain TCA1 did not grow or dechlorinate in the presence of oxygen, exposure of the active culture to aerobic conditions for up to 3 days did not result in the loss of dechlorinating activity (16). Dechlorination of TCA was sequential with the accumulation of 1,1-dichloroethane (DCA) before conversion to chloroethane (CA) (Fig. 2). Dechlorination of TCA occurred at a faster rate than that of DCA. The temperature range for

reductive dechlorination was from 12°C to 30°C with the optimum at 25°C. No dechlorination occurred at 37°C. Growth yield from reductive dechlorination was 5.60 ± 1.26 g (dry weight) (mean ± SD, n = 3 cultures) of cells per mole of chloride released. TCA, H₂, and acetate were essential for growth of strain TCA1. Formate could replace H₂ as an electron donor, which resulted in a similar dechlorination rate. Acetate alone did not support TCA dechlorination, suggesting that it was used only as a carbon source and not as an electron donor for reductive dechlorination. Because H₂ oxidation does not support substrate-level phosphorylation, growth in a defined medium with TCA as the electron acceptor and H₂ as the electron donor indicated that strain TCA1 conserves energy in a respiratory process.

Strain TCA1 did not dechlorinate 1,1,1,2-

tetrachloroethane, 1,1,2-trichloroethane, 1,2-dichloroethane, 1,2-dichloropropane, PCE, or TCE when they were added as potential electron acceptors. No growth occurred in liquid media amended with 4 mM H₂ as an electron donor, 5 mM acetate as a carbon source, and either 5 mM sulfate, sulfite, thiosulfate, nitrate, or fumarate as an electron acceptor. Strain TCA1 did not use lactate, pyruvate, propionate, fumarate, butyrate, benzoate, phenol, glucose, ethanol, or methanol as electron donors. No fermentative growth was observed. Sulfite or thiosulfate at 5 mM concentration completely inhibited TCA dechlorination, whereas 5 mM sulfate, nitrate, or fumarate had no effect. Because these compounds did not serve as electron acceptors for strain TCA1, inhibition was not due to competition for electron donors. Instead, the re-

![Figure 1](#fig1)

**Fig. 1.** Scanning electron micrograph of strain TCA1. The rod-shaped morphology and dividing cells are shown. Scale bar, 1 μm.

![Figure 2](#fig2)

**Fig. 2.** Stoichiometry of the dechlorination of TCA to DCA and CA by strain TCA1. Samples were incubated at 25°C and analyzed every 3 days over a 2-month period for depletion of TCA and production of DCA and CA. Error bars (shown if larger than the symbols) represent standard deviations of triplicate cultures.

---

**Fig. 3.** 16S rDNA-based phylogenetic tree of strain TCA1 and representative bacteria. The sequence of strain TCA1 was aligned against the most similar sequence in the Ribosomal Database Project (25) and GenBank using the automated aligner of the ARB software package (26). The alignment was corrected manually based on both primary and secondary structural considerations. The tree was generated by a maximum-likelihood method with the program fastDNAml (27), with 1375 nucleotide positions unambiguously aligned. Values in each node are the percentage of 100 bootstrap trees. Scale bar, 0.1 substitutions per nucleotide position. T, type strain.


Escherichia coli, Z83205
Dehalospirillum multivorans str. K, X82931
Desulfomonile tiedjei str. DCB-1, M26635
Bacillus cereus IAM 12605 (T), D126266
Syntrophobotulus glycolicus str. FLGlyR, X99706
Clone SHA-67, from a 1,2-dichloropropane-dechlorinating microbial consortium, AJ249096
Clone SJA-19, from a trichlorobenzene-dechlorinating microbial consortium, AJ009454
Dehalobacter sp. str. TCA1, AF507944
Dehalobacter restrictus str. TEA, Y10164
Clone SJA-47, from a trichlorobenzene-dechlorinating microbial consortium, AJ009464
Dehalobacter restrictus str. PER-K23, U84497
Desulfitobacterium sp. str. PCE1, X81032
Desulfitobacterium chlororespirans str. Co23, U68528
Desulfitobacterium frappieri str. PCP1, U40078


1 NOVEMBER 2002 VOL 298 SCIENCE www.sciencemag.org

active sulfur oxyanions may directly inhibit the components of the electron transport chain or even the reductive dehalogenase, as observed in *Desulfomonile tiedjei* (17).

Comparative analysis of the nearly full-length 16S ribosomal DNA (rDNA) sequence of strain TCA1 and available 16S rDNA sequences revealed that strain TCA1 is related to gram-positive bacteria with low G+C content in their DNA. Strain TCA1, *Dehalobacter restrictus* (18, 19), and three clones from trichlorobenzene- and 1,2-dichloropropane-dechlorinating consortia (20, 21) form a phylogenetic cluster (Fig. 3) defined by 16S rDNA sequence similarities of 97% and higher. The closest relative of strain TCA1 is *D. restrictus* strain TEA, with a sequence similarity of 99%. *D. restrictus* strains PER-K23 and TEA are strict anaerobes capable of coupling PCE and TCE dechlorination to H₂ oxidation for growth in a respiratory process. However, we did not detect TCA dechlorination by strain PER-K23. These bacterial strains seem to be obligate H₂-oxidizing dechlorinators that only grow by reductive dechlorination of specific chlorinated ethanes or ethenes in anaerobic respiration. The physiology, morphology, and 16S rDNA sequence of strain TCA1 suggest that it is a *Dehalobacter* and perhaps represents a new species based on its unique features of TCA dechlorination and formate oxidation. The isolation of strain TCA1 further suggests an important role for *Dehalobacter* species in polluted anoxic environments.

To determine the potential of strain TCA1 to attenuate TCA in the natural environment, we bioaugmented anoxic aquifer sediments from Bachman and Schoolcraft (both in Michigan, USA) contaminant plumes. Both sites are contaminated with PCE and daughter products, and the Schoolcraft plume G site is also contaminated with TCA, DCA, and chromium. TCA was completely converted to CA within 2 months in both aquifer sediment samples amended with strain TCA1, whereas no dechlorination was observed in samples without the inoculum. These results suggest that bioaugmentation with strain TCA1 could ensure and speed the degradation of TCA, especially if naturally occurring populations are patchy or absent.

CA, rather than ethane, appears to be the terminal TCA product from our culture, and studies have shown that both DCA and CA can be degraded under aerobic conditions (22, 23). Because the aerobic transformation of DCA is much slower than that of CA (24), complete conversion to CA would result in more reliable removal of chloroethanes on the aerobic fringes of a plume.

The discovery of an anaerobic dehalorespiring *Dehalobacter* that couples reductive dechlorination of TCA to growth not only may lead to a better understanding of the

**References and Notes**

1. Agency for Toxic Substances and Disease Registry, ToxFAQs for 1,1,1-Trichloroethane (September 1996; www.atsdr.cdc.gov/tfacts70.html).
2. J. J. Westrick, J. W. Mello, R. F. Thomas, *J. Am. Water Works Assoc.* **76**, 52 (1984).
3. S. A. Montzka *et al.*, *Science* **288**, 500 (2000).
4. Federal Register, *Fed. Regist.* **54**, 22062 (1989).
5. United Nations Environment Programme, *Production and Consumption of Ozone Depleting Substances 1986–1998* (October 1999; www.unep.org/ozone/dataport99.shtml).
6. M. D. Lee, J. M. Odom, R. J. Buchanan Jr., *Annu. Rev. Microbiol.* **52**, 423 (1998).
7. W. W. McNab Jr., T. N. Narasimhan, *Environ. Sci. Technol.* **28**, 769 (1994).
8. Y. Kim, D. J. Arp, L. Semprini, *J. Environ. Eng.* **126**, 934 (2000).
9. C. E. Aziz, G. Georgiou, G. E. Speitel Jr., *Biotechnol. Bioeng.* **65**, 100 (1999).
10. D. T. Adamson, G. F. Parkin, *Water Res.* **33**, 1482 (1999).
11. J. H. de Best *et al.*, *Appl. Microbiol. Biotechnol.* **48**, 417 (1997).
12. X. Maymó-Gatell, Y. Chien, J. M. Gossett, S. H. Zinder, *Science* **276**, 1568 (1997).
13. L. Adrian, U. Szewzyk, J. Wecke, H. Görisch, *Nature* **408**, 580 (2000).
14. W. W. Mohn, J. M. Tiedje, *Arch. Microbiol.* **157**, 1 (1991).
15. B. Sun, J. R. Cole, R. A. Sanford, J. M. Tiedje, *Appl. Environ. Microbiol.* **66**, 2408 (2000).
16. Materials and methods are available as supporting material on Science Online.
17. G. T. Townsend, J. M. Suflita, *Appl. Environ. Microbiol.* **63**, 3594 (1997).
18. C. Holliger *et al.*, *Arch. Microbiol.* **169**, 313 (1998).
19. A. Wild, R. Hermann, T. Leisinger, *Biodegradation* **7**, 507 (1996).
20. F. von Wintzingerode, B. Selent, W. Hegemann, U. B. Göbel, *Appl. Environ. Microbiol.* **65**, 283 (1999).
21. C. Schlötelburg, F. von Wintzingerode, R. Hauck, W. Hegemann, U. B. Göbel, *Int. J. Syst. Evol. Microbiol.* **50**, 1505 (2000).
22. R. Oldenhuis, R. L. J. M. Vink, D. B. Janssen, B. Witholt, *Appl. Environ. Microbiol.* **55**, 2819 (1989).
23. R. Scholtz, A. Schmuckle, A. M. Cook, T. Leisinger, *J. Gen. Microbiol.* **133**, 267 (1987).
24. T. M. Vogel, C. S. Criddle, P. L. McCarty, *Environ. Sci. Technol.* **21**, 722 (1987).
25. B. L. Maidak *et al.*, *Nucleic Acids Res.* **29**, 173 (2001).
26. O. Strunk *et al.*, Ed., ARB: A Software Environment for Sequence Data (Department of Microbiology, Technical University of Munich, Muenchen, 1998; www.mikro.biologie.tu-muenchen.de).
27. G. J. Olsen, H. Matsuda, R. Hagstrum, R. Overbeek, *Comput. Appl. Biosci.* **10**, 41 (1994).
28. Supported by the Michigan Department of Environmental Quality and the Center for Microbial Ecology at Michigan State University.

Supporting Online Material  
www.sciencemag.org/cgi/content/full/298/5595/1023/DC1  
Materials and Methods  

4 June 2002; accepted 19 August 2002

---

**Toll-Like Receptor 4–Dependent Activation of Dendritic Cells by β-Defensin 2**

Arya Biragyn,$^{1\ast}$ Pier Adelchi Ruffini,$^{1\dagger}$ Cynthia A. Leifer,$^{2}$ Elena Klyushnenkova,$^{3}$ Alexander Shakhov,$^{3}$ Oleg Chertov,$^{4}$ Aiko K. Shirakawa,$^{5}$ Joshua M. Farber,$^{5}$ David M. Segal,$^{2}$ Joost J. Oppenheim,$^{6}$ Larry W. Kwak$^{1}$

β-Defensins are small antimicrobial peptides of the innate immune system produced in response to microbial infection of mucosal tissue and skin. We demonstrate that murine β-defensin 2 (mDF2β) acts directly on immature dendritic cells as an endogenous ligand for Toll-like receptor 4 (TLR-4), inducing up-regulation of costimulatory molecules and dendritic cell maturation. These events, in turn, trigger robust, type 1 polarized adaptive immune responses in vivo, suggesting that mDF2β may play an important role in immunosurveillance against pathogens and, possibly, self antigens or tumor antigens.

Activation of innate immunity through pattern recognition receptors for ligands derived from evolutionarily distant pathogens provides essential signals for initiation of the adaptive immune response (1–3). Microbial infection activates the TLR signaling cascade (4), which results in expression of various proinflammatory cytokines, chemokines, and large quantities of small antimicrobial peptides, such as defensins (5–7). Recently, it was reported that β-defensins, epithelial antibacterial peptides with six conserved cysteine residues, might have an additional function as potential chemoattractants of immature dendritic cells (iDCs) through chemokine receptor CCR6 (8, 9). In the course of our studies using β-defensins and chemokines to target delivery of nonimmunogenic antigens to iDCs in vivo as vaccines, we unexpectedly observed that the resulting immune responses differed substantially depending on the type of chemoattractant moiety used. In particular, murine β-defensin 2 (mDF2β)-based vaccines elicited modest lev

www.sciencemag.org SCIENCE VOL 298 1 NOVEMBER 2002 1025

Microbial Dehalorespiration with 1,1,1-Trichloroethane
Baolin Sun et al.
Science 298, 1023 (2002);
DOI:10.1126/science.1074675

This copy is for your personal, non-commercial use only.

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by clicking here.

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines here.

The following resources related to this article are available online at www.sciencemag.org (this information is current as of February 1, 2015):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:
http://www.sciencemag.org/content/298/5595/1023.full.html

Supporting Online Material can be found at:
http://www.sciencemag.org/content/suppl/2002/10/30/298.5595.1023.DC1.html

A list of selected additional articles on the Science Web sites related to this article can be found at:
http://www.sciencemag.org/content/298/5595/1023.full.html#related

This article cites 22 articles, 10 of which can be accessed free:
http://www.sciencemag.org/content/298/5595/1023.full.html#ref-list-1

This article has been cited by 35 article(s) on the ISI Web of Science

This article has been cited by 12 articles hosted by HighWire Press; see:
http://www.sciencemag.org/content/298/5595/1023.full.html#related-urls

This article appears in the following subject collections:
Microbiology
http://www.sciencemag.org/cgi/collection/microbio

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2002 by the American Association for the Advancement of Science; all rights reserved. The title Science is a registered trademark of AAAS.
