
Arch Microbiol (2002) 178:506–516  
DOI 10.1007/s00203-002-0487-2  

**ORIGINAL PAPER**

Ralf Rabus · Michael Kube · Alfred Beck  
Friedrich Widdel · Richard Reinhardt  

**Genes involved in the anaerobic degradation of ethylbenzene in a denitrifying bacterium, strain EbN1**

Received: 11 June 2002 / Revised: 20 August 2002 / Accepted: 30 August 2002 / Published online: 3 October 2002  
© Springer-Verlag 2002  

**Abstract** Genes involved in anaerobic degradation of the petroleum hydrocarbon ethylbenzene in the denitrifying Azoarcus-like strain EbN1 were identified on a 56-kb DNA contig obtained from shotgun sequencing. Ethylbenzene is first oxidized via ethylbenzene dehydrogenase to (S)-1-phenylethanol; this is converted by (S)-1-phenylethanol dehydrogenase to acetophenone. Further degradation probably involves acetophenone carboxylase forming benzoylacetate, a ligase forming benzoylacetyl-CoA, and a thiolase forming acetyl-CoA and benzoyl-CoA. Genes of this pathway were identified via N-terminal sequences of proteins isolated from strain EbN1 and by sequence similarities to proteins from other bacteria. Ethylbenzene dehydrogenase is encoded by three genes (ebdABC), in accordance with the heterotrimeric enzyme structure. Binding domains for a molybdenum cofactor (in subunit EbdA) and iron/sulfur-clusters (in subunits EbdA and EbdB) were identified. The previously observed periplasmic location of the enzyme was corroborated by the presence of a twin-arginine leader peptide characteristic of the Tat system for protein export. A fourth gene (ebdD) was identified, the product of which may act as an enzyme-specific chaperone in the maturation of the molybdenum-containing subunit. A distinct gene (ped) coding for (S)-1-phenylethanol dehydrogenase apparently forms an operon with the ebdABCD genes. The ped gene product with its characteristic NAD(P)-binding motif in the N-terminal domain belongs to the short-chain dehydrogenase/reductase (SDR) superfamily. A further operon apparently contains five genes (apc1–5) suggested to code for subunits of acetophenone carboxylase. Four of the five gene products are similar to subunits of acetone carboxylase from Xanthobacter autotrophicus. Upstream of the apc genes, a single gene (bal) was identified which possibly codes for a benzoylacetate CoA-ligase and which is co-transcribed with the apc genes. In addition, an apparent operon containing almost all genes required for β-oxidation of fatty acids was detected; one of the gene products may be involved in thiolytic cleavage of benzoylacetyl-CoA. The DNA fragment also included genes for regulatory systems; these were two sets of two-component systems, two LysR homologs, and a TetR homolog. Some of these proteins may be involved in ethylbenzene-dependent gene expression.

**Keywords** Ethylbenzene · Denitrifying bacterium · Anaerobic degradation · Gene prediction · Gene annotation · Genomics

---

**Introduction**

Hydrocarbons belong to the chemically least reactive organic compounds. Their biodegradation requires specific biochemical reactions to activate the apolar C-H bonds. Under oxic conditions, reactive oxygen species (from O₂) are used in oxygenase-catalyzed reactions to activate hydrocarbons and form hydroxylated products. Since such activation mechanisms are in principle excluded under anoxic conditions, anaerobic degradation of hydrocarbons has long been doubted. During the past decade, however, numerous pure cultures of bacteria have been isolated that degrade hydrocarbons anaerobically (for overview see Spormann and Widdel 2000; Widdel and Rabus 2001), and three distinct novel biochemical reaction principles for hydrocarbon activation have been recognized. (1) Methylarenes such as toluene (for overview see: Heider et al. 1999; Spormann and Widdel 2000) as well as *n*-alkanes (Kropp et al. 2000; Rabus et al. 2001) are presumably activated by glycyl-radical-catalyzed addition to fumarate yielding arylmethyl- or alkyl-substituted succinates, respectively. (2) Anaerobic degradation of naphthalene is

R. Rabus (✉) · F. Widdel  
Max-Planck-Institut für Marine Mikrobiologie,  
Celsiusstrasse 1, 28359 Bremen, Germany  
e-mail: rrabus@mpi-bremen.de,  
Tel.: +49-421-2028736, Fax: +49-421-2028580  

M. Kube · A. Beck · R. Reinhardt  
Max-Planck-Institut für Molekulare Genetik,  
Ihnestr. 73, 14195 Berlin, Germany

Periplasm                                                                                     Cytoplasm

![Diagram](attachment:diagram.png)

Ethylbenzene dehydrogenase  
$ebdABCD$

(S)-1-Phenylethanol dehydrogenase  
$ped$

Acetophenone carboxylase  
$apc1-5$

Benzoylacetate CoA-ligase  
$bal$

Ethylbenzene  
(S)-1-Phenylethanol  
Acetophenone  
Benzoylacetate  
Benzoylacetyl-CoA  
(3-Oxo-3-phenylpropionyl-CoA)  
Benzoyl-CoA  

Fig. 1 Proposed reaction sequence for the anaerobic degradation of ethylbenzene in denitrifying strain EbN1 to the level of benzoyl-CoA (modified from: Rabus and Widdel 1995; Ball et al. 1996; Rabus and Heider 1998; Kniemeyer and Heider 2001a). Benzoyl-CoA is further oxidized via ring cleavage to carbon dioxide (not shown). Reducing equivalents are used for the reduction of nitrate to dinitrogen. Coding genes identified in this study (see Fig. 2) are indicated in **bold**

suggested to involve carboxylation to 2-naphthoate (Zhang and Young 1997; Meckenstock et al. 2000). (3) Ethylbenzene in the Azoarcus-like β-Proteobacteria strains EbN1, PbN1 (Rabus and Widdel 1995) and EB1 (Ball et al. 1996), the currently known denitrifiers able to utilize this hydrocarbon, is degraded via dehydrogenation to (S)-1-phenylethanol.

Based on physiological and biochemical studies with strains EbN1 and EB1 (Rabus and Widdel 1995; Ball et al. 1996; Rabus and Heider 1998; Johnson and Spormann 1999; Kniemeyer and Heider 2001a), a complete pathway (Fig. 1) has been suggested that allows the anaerobic conversion of ethylbenzene to benzoyl-CoA, the central intermediate in the anaerobic degradation of aromatic compounds (Harwood et al. 1999). The enzymes of the first two reactions, ethylbenzene dehydrogenase and (S)-1-phenylethanol dehydrogenase, have been characterized. Ethylbenzene dehydrogenase is unique in that it catalyzes dehydrogenation of a hydrocarbon, a reaction previously unknown in biochemistry. The enzyme is a heterotrimeric molybdenum/iron-sulfur/heme protein (Johnson et al. 2001; Kniemeyer and Heider 2001a) with a periplasmic location in strain EbN1. The genes ($ebdABC$) encoding the three subunits of ethylbenzene dehydrogenase of strain EB1 have been identified, and the derived gene products show sequence similarity to the subunits of selenate reductase from *Thauera selenatis* (Johnson et al. 2001). The product of the initial reaction, (S)-1-phenylethanol, is further oxidized to acetophenone by an NAD⁺-dependent homodimeric alcohol dehydrogenase (Kniemeyer and Heider 2001b). Further degradation of acetophenone has been suggested to proceed via carboxylation to benzoylacetate (3-oxo-3-phenylpropionate), which has been confirmed indirectly by demonstrating CO₂-dependence of ethylbenzene and acetophenone degradation (Ball et al. 1996; Champion et al. 1999). Subsequent reactions have been suggested to involve benzoylacetate CoA-ligase, forming benzoylacetyl-CoA, and a benzoylacetyl-CoA thiolase,

thiotically cleaving the substrate to form acetyl-CoA and benzoyl-CoA.

A unique characteristic of the Azoarcus-like strain EbN1 is that it can also utilize toluene as a hydrocarbon substrate in addition to ethylbenzene (Rabus and Widdel 1995). Toluene is degraded via the activation mechanism with fumarate that is common to toluene-degrading anaerobes. Hence, strain EbN1 employs completely different reaction sequences for the conversion of two chemically similar aromatic hydrocarbons to benzoyl-CoA. Moreover, the two pathways are specifically induced by the respective substrate, as demonstrated by adaptation studies with whole cells (Champion et al. 1999) and in vitro measurement of the initial reactions (Rabus and Heider 1998); accordingly, 2-dimensional gel electrophoresis revealed substrate-specific protein patterns (Champion et al. 1999). Strain EbN1 can, therefore, be regarded as a useful model organism to study various enzymes as well as aspects of catabolic regulation in the anaerobic degradation of aromatic hydrocarbons. The goal of the present study was to identify genes involved in anaerobic ethylbenzene degradation. Analysis of a 56-kb DNA contig (contiguous sequence) assembled from shotgun DNA sequencing allowed the identification of genes and gene clusters required for the postulated enzymes of anaerobic ethylbenzene degradation to benzoyl-CoA; furthermore, several genes coding for regulatory proteins were identified that may be involved in the differential expression of genes for anaerobic degradation of aromatic hydrocarbons.

---

### Materials and methods

#### Bacterial strain, growth conditions and isolation of genomic DNA

The denitrifying bacterium Azoarcus-like strain EbN1 (β-Proteobacteria) was isolated from anoxic freshwater mud sampled in Bremen, Germany (Rabus and Widdel 1995). Strain EbN1 was grown in 5-l bottles containing 4 l of a defined, ascorbate-reduced medium with nitrate (10 mM) as electron acceptor and benzoate (4 mM) as electron donor and organic carbon source (Rabus and Widdel 1995). Cells were harvested during the exponential growth phase as described before (Rabus and Widdel 1995).

The procedure for isolation of genomic DNA was modified from previously described methods (Ausubel et al. 1992; Zhou et al. 1996). For isolation of genomic DNA, about 3 g (wet mass) of freshly harvested cells were resuspended in 17 ml of lysis buffer (pH 8) containing Tris, sodium ethylenediaminetetraacetate, sodium phosphate (each 100 mM), NaCl (87 g/l) and hexadecyltrimethylammonium bromide (10 g/l). For lysis, cell suspen-

Table 1 Assignment of score values in the additive scoring tool of the HTGA (High Throughput Genome Annotation) system. For each gene analyzed, the sum of the three determined score values is calculated. The resulting additive score value is used as a measure of the reliability of the automated annotation. Supplementary information is available under http://www.microgenomes.mpg.de/ebn1/

| Evaluated categories of ORF analysis                                                                 | Score value |
|-----------------------------------------------------------------------------------------------------|-------------|
| Lowest e-value among BLAST hits (e-value of best BLAST hit)                                          | <10⁻⁵       | 2–5<sup>a</sup> |
|                                                                                                     | ≥10⁻⁵       | 0             |
| INTERPRO hits were obtained:                                                                         |             |               |
| Without significant BLAST hit for ORF                                                                |             | 1             |
| In addition to BLAST hit for ORF, the hit ratio<sup>b</sup> being                                    | ≥0.8        | 3             |
|                                                                                                     | <0.8        | 0             |
| Ratio between lengths of best BLAST hit and ORF:                                                     |             |               |
| is >2<sup>c</sup>                                                                                    |             | –4            |
| is ≤2, the ORF length covered by BLAST hit being                                                    | 0.8–1       | 2             |
|                                                                                                     | <0.8        | 0             |

<sup>a</sup>The lower the e-value, the higher the score value  
<sup>b</sup>Number of common INTERPRO hits divided by number of total INTERPRO hits for ORF and for gene from best BLAST hit  
<sup>c</sup>If this criterion is applicable, a lower limit of 2 is set for the additive score value

sions were incubated on a rotary shaker (225 rpm) at 37°C for 30 min. Then 1.5 ml of sodium dodecyl sulfate (20 g/l) were added and samples were incubated at 65°C for 2 h. After removal of cell debris by centrifugation, samples were treated with chloroform/isoamylalcohol (24:1, v/v) at least two times. DNA was precipitated from the aqueous phase with 0.11% (v/v) sodium acetate (3 M; pH 5.2) and 0.56% (v/v) 2-propanol, washed with ethanol and stored at 4°C.

Construction of shotgun libraries and DNA sequencing

Two shotgun libraries with average insert sizes of 1.5 kb and 3.5 kb were generated for DNA sequencing. DNA was fragmented by sonication. The resulting fragments were end-repaired, size-selected, and ligated into the *SmaI*-digested and dephosphorylated pUC19 vector (Fermentas, St. Leon-Rot, Germany). Cloned DNA was transferred into *Escherichia coli* (strain DH10B; Gibco, Karlsruhe, Germany) via electroporation. Obtained clones were isolated and cultured in 384-well multititer plates. From a small amount of each bacterial culture, cloned DNA fragments were amplified by PCR using minor modifications of the method described by Radeloff et al. (1998). DNA was sequenced using Dye Terminator Chemistry (ABI, Weiterstadt, Germany) and 3700 ABI capillary sequencer systems (ABI, Weiterstadt, Germany). Regions of weak quality within the analyzed contig were improved by resequencing and primer walking so as to achieve finished sequence quality (three independent reads, both strands sequenced; Hattori et al. 2000). The nucleotide sequence has been deposited at EMBL under the accession number AL844858.

Sequence assembly and gene prediction

The quality of raw sequence data was determined with PHRED (Ewing et al. 1998; Ewing and Green 1998). Sequences were then assembled with phrap2gap (http://www.sanger.ac.uk/Software/sequencing/docs/phrap2gap/) using PHRAP (Rieder et al. 1998; http://bozeman.genome.washington.edu/). GAP4 of the Staden Package (Staden et al. 1999) was used for final editing of the sequence.

ORPHEUS (Frishman et al. 1998) was used for the prediction of genes on the finished 56-kb contig. The program was adjusted such that only open reading frames (ORFs) ≥105 bp were predicted. Overlapping ORFs without functional assignment or BLAST hit were removed from the list of predicted genes. To identify ORFs that remained unnoticed during initial prediction, the nucleotide sequence of the contig was screened against nucleotide databases (EMBL and EMBLNEW). Furthermore, BLAST searches (Altschul et al. 1997) were performed between six-frame translations of the contig and of nucleotide sequences in the databases.

Functional assignment

Similarity searches were carried out with the BLAST programs (Altschul et al. 1997). Translated amino acid sequences of the predicted ORFs were screened against a non-redundant protein database (compiled from SWISSPROT, TREMBL and PIR). ORFs with no similarity to any entry in the protein database were screened against translated sequences from nucleotide databases (EMBL and EMBLNEW).

Functional assignment was done with the INTERPRO system (Apweiler et al. 2001) using the modules (for references, see Baxevanis 2002) PROSITE, Pfam, PRINTS, ProDom, SMART, TIGRFAMs and SIGNALP (Nielsen et al. 1997). The INTERPRO system includes direct links to the gene ontology system (The Gene Ontology Consortium 2001). In addition, the predicted ORFs were screened against the Clusters of Orthologous Groups of proteins (COGs; for references, see Baxevanis 2002).

To evaluate the reliability of gene annotation (i.e. functional assignment), an additive scoring tool was developed. For each identified ORF, score values were determined as a means to evaluate: (1) the e-value of the BLAST hit, (2) the correlation between INTERPRO and BLAST hits, and (3) the length of the alignment with the BLAST hit (for details see Table 1). The sum of the three obtained score values is termed the additive score value and used as an indicator of the reliability of annotation; reliability increases from 0 to 10.

Results from the automated ORF prediction and functional assignment were manually controlled for the entire contig (56 kb).

Data management

The methods used for functional assignment (BLAST, INTERPRO and the additive scoring tool) were all implemented in the annotation platform HTGA (High Throughput Genome Annotation), which allowed automated annotation of the ORFs predicted on the 56-kb contig. As an additional feature of the HTGA system, all information generated during functional assignment was compiled into HTML tables by a set of Perl scripts. More details on the design of the HTGA system are provided on the Web page http://www.micro-genomes.mpg.de/ebn1/.

Results and discussion

The present shotgun DNA sequencing approach was used to identify genes in strain EbN1 (related to *Azoarcus*, β-Proteobacteria), that are involved in the anaerobic conversion of ethylbenzene to benzoyl-CoA via the reaction sequences depicted in Fig. 1. DNA fragments were screened using N-terminal amino acid sequences of proteins isolated from strain EbN1; these were: (1) purified ethylben-

Fig.2 Scale model of gene organization in the investigated contig from denitrifying strain EbN1. Putative functions of the depicted ORFs are listed in Table 2. Blue Genes related to anaerobic oxidation of ethylbenzene to benzoyl-CoA (hues indicate distinct functional proteins as explained in the text and Fig. 1); orange regulatory proteins; green genes encoding proteins involved in metabolism of fatty acids; gray ORFs with assigned putative functions; white hypothetical proteins. Proteins demonstrated to be induced in strain EbN1 by ethylbenzene or acetophenone are marked by a dot (●). The scale indicates nucleotide position on the DNA contig

Genes of ethylbenzene dehydrogenase (ebdABCD)

Three genes, ebdA (orf65), ebdB (orf63) and ebdC (orf62), were identified that apparently code for the α-subunit (EbdA), the β-subunit (EbdB) and the γ-subunit (EbdC), respectively, of ethylbenzene dehydrogenase, which has been previously shown to be a heterotrimer. Identification was based on N-terminal sequences of the purified subunits of the enzyme from strain EbN1 (Kniemeyer and Heider 2001a) and sequences of the genes encoding these subunits in strain EB1 (Johnson et al. 2001). The ebd operon contains a fourth gene, ebdD (orf60), which is described below. The genes from both strains are almost identical; the three genes ebdABC, with a total length of 4.63 kb, differ only in four nucleotide positions, resulting in a difference of merely six amino acids. This high conservation is in agreement with the close relationship of the two strains with respect to their 16S rRNA genes (Ball et al. 1996). Similarly, also the subunits of the related dissimilatory nitrate reductases (NarGHI) of the two *E. coli* strains K12 and O157:H7 (Blattner et al. 1997; Perna et al. 2001) differ merely by a total of five amino acids.

The closest relatives of EbdA are the α-subunit (SerA) of selenate reductase from *Thauera selenatis* (Krafft et al. 2000), α-subunits of membrane-bound nitrate reductases (e.g. NarG from *E. coli* and *Haloarcula marismortui*; Blasco et al. 1989; Yoshimatsu et al. 2002), and the α-subunit (DdhA) of dimethyl sulfide dehydrogenase from *Rhodovulum sulfidophilum* (McDevitt et al. 2002). EbdA starts with a signal peptide of 54 amino acid residues con-

Table 2 Annotated ORFs of studied contig from denitrifying strain EbN1

| Open reading frame<sup>a</sup> | Length (amino- acids) | INTERPRO/COG references<sup>b</sup> | BLASTP hit used for annotation<sup>c</sup> | Putative function |
| --- | --- | --- | --- | --- |
| Gene | Organism<sup>d</sup> | E-value | Accession number<sup>e</sup> |  |
| 203 | 423 | COG 2067 | vc1042 | Vibch | 1e-44 | B82250 | Long-chain fatty acid transport protein; fadL |
| 206 | 136 | — | cc0190 | Caurc | 2e-24 | AAK 22177 | Acyl-CoA thioesterase |
| 208 | 678 | IPR 001753, IPR 002135/COG 1250 | vc1047 | Vibch | 5e-76 | AAF 94206 | α-Subunit of fatty acid oxidation complex; fadB |
| 212 | 426<sup>f</sup> | IPR 002155/COG 0183 | cg4581 | Drome | 4e-64 | NP_477378 | β-Subunit of fatty acid oxidation complex; 3-keto-acyl-CoA-thiolase; fadA |
| 18 | 829 | IPR 001552/COG 1960 | yafh | Ecoli | 0 | Q47146 | Long chain acyl-CoA dehydrogenase; fadE |
| 220 | 233 | IPR 001647/COG 1309 | pa3006 | Pseae | 3e-25 | AAG 06394 | Transcriptional regulator; TetR family; regI |
| 25 | 288 | COG 1561 | nmb0711 | Neime | 2e-48 | F81168 | Hypothetical protein |
| 29 | 272 | IPR 000719/COG 0515 | dra0332 | Deira | 2e-25 | AAF 12449 | Protein kinase |
| 229 | 305<sup>f</sup> | IPR 001932/COG 0631 | bh2505 | Bachd | 2e-20 | BAB 06224 | Phosphatase 2C domain |
| 232 | 239 | IPR 001247/COG 0689 | rph | Legpn | 1e-77 | AAD 28218 | Ribonuclease PH; rph |
| 234 | 198 | IPR 002637/COG 0127 | nmb0639 | Neime | 6e-52 | AAF 41062 | Protein of Haml family |
| 39 | 407<sup>f</sup> | IPR 003401/COG 0635 | nmb0665 | Neime | 1e-121 | AAF 41083 | Oxygen-independent coproporphyrinogen III oxidase; hemN |
| 237 | 613 | IPR 002142/COG 0616 | vc1994 | Vibch | 1e-110 | AAF 95142 | Protease IV |
| 43 | 259<sup>f</sup> | IPR 003358, IPR 004395/COG 0220 | nmb1328 | Neime | 2e-75 | AAF 41703 | Methyltransferase |
| 46 | 261 | IPR 003009/COG 2022 | thig | Pseae | 1e-101 | Q9I6B4 | Thiamine biosynthesis protein; thiG |
| 49 | 641 | IPR 002104/COG 0582 | int | Psepu | 1e-174 | CAA 06238 | Integrase; int |
| 250 | 216 | COG 0583 | bphr | Psep | 6e-19 | AAD 37845 | Regulatory protein; LysR-family; reg2 |
| 53 | 300 | IPR 000847/COG 0583 | bphr | Alceu | 8e-69 | CAB 72138 | Regulatory protein; LysR-family; reg3 |
| 56 | 104 | COG 0644 | etfdh | Human | 3e-25 | Q16134 | Hypothetical protein; possible electron transfer protein |
| 57 | 356 | IPR 003495/COG 0523 | pa4604 | Pseae | 4e-53 | AAG 07992 | Hypothetical protein; putative GTPase |
| 58● | 249 | IPR 00128/COG 1028 | mlr6807 | Rhilo | 3e-36 | BAB 53033 | (S)-1-Phenylethanol dehydrogenase; ped |
| 60 | 221<sup>f</sup> | COG 3381 | serd | Thase | 3e-10 | CAB 53374 | δ-Subunit of ethylbenzene dehydrogenase; ebdD |
| 62● | 214 | — | ebdc | Azosp | 1e-114 | AAK 76389 | γ-Subunit of ethylbenzene dehydrogenase; ebdC |
| 63 | 352 | IPR 001450/COG 1140 | ebdb | Azosp | 0 | AAK 76388 | β-Subunit of ethylbenzene dehydrogenase; ebdB |
| 65● | 976 | IPR 001467/COG 0243 | ebda | Azosp | 0 | AAK 76387 | α-Subunit of ethylbenzene dehydrogenase; ebdA |
| 68 | 473 | — | pa0383 | Pseae | 8e-06 | AAG 03772 | Hypothetical protein |
| 70 | 76 | COG 2801 | tISRso8b | Ralso | 2e-14 | NP_518699 | Hypothetical protein; transposon related function |
| 74 | 316 | IPR 002559/COG 3039 | tnpa | Pseae | 6e-88 | AAD 38927 | Is2000 transposase |
| 75 | 421 | IPR 000014, IPR 003594/COG 0642 | pa0600 | Pseae | 1e-41 | AAG 03989 | Two-component sensor; tcs1 |
| 77 | 206 | IPR 000792, IPR 001789/COG 2197 | pa0601 | Pseae | 2e-42 | AAG 03990 | Two-component response regulator; tcr1 |
| 81 | 622 | IPR 000014, IPR 003594/COG 0642 | tdis | Thaar | 2e-79 |CAA 05048 | Two-component sensor; tcs2 |
| 82 | 208 | IPR 000792, IPR 001789/COG 2197 | tdir | Thaar | 1e-42 |CAA 05049 | Two-component response regulator; tcr2 |

Table 2 (continued)

| Open reading frame<sup>a</sup> | Length (amino-acids) | INTERPRO/COG references<sup>b</sup> | BLASTP hit used for annotation<sup>c</sup> | Putative function |
|--------------------------|----------------------|-------------------------------|-------------------------------------|--------------------|
| Gene                     | Organism<sup>d</sup> | E-value                      | Accession number<sup>e</sup>         |                    |
| 283                      | 120                  | IPR 000390/COG 2076           | all0285                            | Noss p             | 3e-18                              | NP_484329          | Hypothetical protein; putative efflux system |
| 84                       | 275                  | IPR 000051/COG 0500           | –                                  | Arath              | 2e-10                              | AAL 16223          | Hypothetical protein; putative methyltransferase |
| 87                       | 723                  | IPR 001687                   | –                                  | –                  | –                                  | –                 | Hypothetical protein; putative cox2 cytochrome oxidase subunit 2 |
| 90                       | 808                  | IPR 000873,                   | –                                  | Pseae              | 9e-46                              | AAK 01517          | Hypothetical protein; putative transport protein |
|                           |                      | IPR 002114/COG 1033            |                                     |                     |                                    |                    |                                      |
| 92                       | 325                  | IPR 002860                   | pa3080                             | Pseae              | 5e-09                              | AAG 06468          | Hypothetical protein               |
| 95                       | 464                  | –                            | paak                               | Azoev              | 6e-34                              | AAF 26285          | Related to phenylacetyl-CoA ligase; possible benzoylacetate CoA-ligase; bal |
| 97●                      | 303                  | –                            | –                                  | –                  | –                                  | –                 | Putative subunit of acetophenone carboxylase; apc5 |
| 100●                     | 684                  | IPR 003692/COG 0146           | acxB                               | Xanau              | 1e-35                              | AAL 17711          | Related to α-subunit of acetone carboxylase; possible subunit of acetophenone carboxylase; apc4 |
| 102●                     | 732                  | IPR 002821,                   | acxA                               | Xanau              | 6e-85                              | AAL 17710          | Related to β-subunit of acetone carboxylase; possible subunit of acetophenone carboxylase; apc3 |
|                           |                      | IPR 001092/COG 0145            |                                     |                     |                                    |                    |                                      |
| 200                      | 129                  | –                            | acxC                               | Xanau              | 1.7                                | AAL 17712          | Related to γ-subunit of acetone carboxylase; possible subunit of acetophenone carboxylase; apc2 |
| 105●                     | 658                  | IPR 001687,                   | acxA                               | Xanau              | 2e-37                              | AAL 17710          | Related to β-subunit of acetone carboxylase; possible subunit of acetophenone carboxylase; apcl |
|                           |                      | IPR 002821/COG 0145            |                                     |                     |                                    |                    |                                      |

* <sup>a</sup> Open reading frames for which expression in strain EbN1 has been demonstrated on the protein level (Champion et al. 1999; Kniemeyer and Heider 2001a, b) are marked by a dot (●)
* <sup>b</sup> References relate to INTERPRO (http://www.ebi.ac.uk/interpro/; Apweiler et al. 2001) and COG databases (http://www.ncbi.nlm.nih.gov/COG/; Tatusov et al. 2001)
* <sup>c</sup> Hits were obtained from BLASTP-comparison of predicted proteins from strain EbN1 with SWISS-PROT and TrEMBL-databases (Bairoch and Apweiler 2000)
* <sup>d</sup> Abbreviations of names of organisms is according to the list of organism identification codes (SWISS-PROT). Alceu, Alcaligenes eutrophus; Arath, Arabidopsis thaliana; Azoev, Azoarcus evansii; Azosp, Azoarcus sp. strain EB1; Bachd, Bacillus halodurans;
* <sup>e</sup> Accession numbers allow retrieval of sequences from NCBI database (http://www.ncbi.nlm.nih.gov/Entrez/index.html)
* <sup>f</sup> ORFs were manually shortened to improve alignments and to avoid overlaps with upstream genes

taining the twin-arginine motif (S/T)-R-R-x-F-L-K. The presence of this motif is characteristic of cofactor-containing proteins that are folded in the cytoplasm before they are exported into the periplasm via the Tat (twin-arginine translocation) pathway (Berks et al. 2000; Sargent et al. 2002). Presence of the twin-arginine motif and export via the Tat pathway have also been reported for other molybdenum-containing enzymes, such as selenate reductase from *T. selenatis* (Krafft et al. 2000), TMAO (trimethylamine N-oxide) reductase, and FDH<sub>N</sub> (nitrate-inducible formate dehydrogenase) from *E. coli* (Berg et al. 1991; Mejean et al. 1994). A periplasmic location of these enzymes (Silvestro et al. 1989; Rech and Macy 1992; Jormakka et al. 2002) and of ethylbenzene dehydrogenase from strain EbN1 (Kniemeyer and Heider 2001a) has been demonstrated experimentally. Ethylbenzene dehydrogenase from strain EB1 has been described as membrane-associated (Johnson and Spormann 1999). EbdA from strain EbN1 contains domains with motifs that have been suggested to be involved in binding of molybdopterin cofactors in enzymes of the DMSO reductase family (Fig. 3; Blasco et al. 1989; Schneider et al. 1996; Dias et al. 1999). Furthermore, EbdA contains a motif with four cysteines close to the N-terminus of the mature protein. Such a motif coordinates an Fe<sub>4</sub>S<sub>4</sub> cluster in the α-subunit of formate dehydrogenases (Boyington et al. 1997; Jormakka et al. 2002), and is also present in SerA and NarG. Structural considerations suggest that the first cysteine of the motif is not necessarily involved in coordination of the cluster; coordination may be also accomplished by a conserved histidine close to the cysteine motif in EbdA, SerA and NarG. Nevertheless, it remains to be shown whether this motif actually binds an iron/sulfur cluster, which would be a second metal center besides

Protein

| I | II | III | IV | V |
|---|---|---|---|---|
| EbdA_EbN1 | 151 GKRGEGKWRVSWDEA | 301 GSDAAFWLGLSQVMIDEK | 413 TPEKASAKCGVPVSLIRELGR | 639 MSSSAMYADIVLPCAWYYEK | 880 DAAELGIKDGD |
| SerA_Thase | 139 GERGERKWRISWDEA | 287 GTDSHLAAAIVNVLIADK | 401 TPEATHTTGGIHPSVVRQLAG | 589 LNSGAMYADIVLPAATNLEK | 822 AMARKGIKDGD |
| NarG_Halma | 146 GERGEGQWERISWDEA | 296 GSDTALALGMAQTIVDEG | 423 TPDVVHEETGVGENTYQRVAR | 603 MDSTAMYSDIVLPAASHYEK | 847 DAQQRGIEDGD |
| NarG_Ecoli | 150 QARGRGGFVRSSWQEV | 300 GTDAAMALAMGHVMLREF | 503 TPAWAEQITGVRSQIIRIAR | 775 LSSTCLYSDIILPTATWYEK | 1123 DAKDLGIADND |
| NarG_Bacsu | 155 QARGKGGFVRAEWPEV | 305 GTDGALAMAMGHVILQEF | 495 TPAWQEOMTGIKKEAVVKIAR | 767 MAGTALYSDIVLPAATWYEK | 1116 DAEDTDIKDND |
| BisC_Ecoli | 75 GIRGQDEFVRVSWDEA | 244 GTDVALLMLGIAHTLVENG | 292 TAEWAAEICGVGAKIRELAA | 458 WTAAAKHADIVLPATTSFER | 674 DAQERGIQDGD |
| DmsA_Ecoli | 128 GARGEGBKFERISWEEA | 291 GTDAALVNGLAYVMITEN | 352 TPEWASQITGVPADKIILKAR | 516 MTSSAKYADILLPDCTASEQ | 730 DAQKRGIHNGD |
| FdhF_Ecoli | 69 .RQRGGKLEPVSWDEA | 221 GSNIALLNAMGHVIIIEEN | 263 TPESVEDITGVSAEIRQAAR | 432 MTKTASAADVILPSTSWGEH | 614 DAKRLGIEDEA |
| TorA_Ecoli | 119 SQRGDNRFVRVSWDEA | 292 QTDPVPLQLALAHTLYSEN | 340 DAAWAEKLTGIDAETIRGLAR | 514 WTSTCRFADIVLPATTQFER | 729 DASARGIRNGD |
| NapA_Alceu | 110 KYDKNGDFAPVTWDQA | 264 QTDLAMLNYIANYIIRNN | 340 DADYVSKLSAVPKAKLDQLAE | 515 PTVTALAADLVLPSAMWVEK | 755 DAKALGLRRGV |
| PhsA_Salty | 107 GPRGDGEWQVISWQQA | 252 GGDLAVLLAMCHVMIDEQ | 294 TPEWAAAQADVPADVIVRVTR | 466 LSESAAYADYLLPECTYLER | 693 TAQEKGIKTGD |
| PsrA_Wolsu | 110 GERGEGBKVESWDEA | 255 GHDLPEVLGFMHTLIFEN | 297 TPEKMALECDIPADTIKRLAR | 477 VSDTAWFADVLPDTTYLER | 678 TASKLGIKNGD |

Fig. 3 Domains with binding motifs for molybdopterin in α-subunits of ethylbenzene dehydrogenase and related molybdenum enzymes of the DMSO family. Conserved amino acids and conservative substitutes were selected according to the proposed consensus sequences (Blasco et al. 1989) and the amino acid substitution matrix (Staden 1982), respectively; all such amino acids are indicated in bold. Abbreviated designations of α-subunits pertain to the following enzymes (accession numbers in parentheses): EbdA, ethylbenzene dehydrogenase (AL844858, EbN1); SerA, selenate reductase (CAB53372); NarG, dissimilatory nitrate reductase (AJ277440, Halma; P09152 (Ecoli) and P42175(Bacsu)); BisC, biotin sulfoxide reductase (P20099); DmsA, dimethylsulfoxide (DMSO) reductase (P18775); FdHF, formate dehydrogenase (P07658); TorA, trimethylamine N-oxide (TMAO) reductase (P33225); NapA; periplasmic nitrate reductase (P39185); PhsA, thiosulfate reductase (P37600); PsrA, polysulfide reductase (CAA46176). Abbreviations of bacterial species (and strains) are as explained in the footnote of Table 2; further abbreviations are: EbN1, denitrifying Azoarcus-like strain EbN1; Bacsu, Bacillus subtilis; Halma, Haloarcula marismortui; Salty, Salmonella typhimurium; Wolsu, Wolinella succinogenes

Protein

|  |  |  |
|---|---|---|
| EbdD_EbN1 | 192 KEWPDHICIQLEFLHYLTFLEA |  |
| SerD_Thase | 102 REYPDHLVTELEFLAWLCLOEH |  |
| NarJ_Halma | 141 RDNHDALTYELEFAGYLARREA |  |
| NarJ_Ecoli | 108 RELPDHLPYLEYLAQLPQSEA |  |
| NarJ_Salch | 108 RELPDHLPYLEYLAQLPKSEA |  |
| NarJ_Pyrae | 110 RELPDYLPVMLEFLAATSEAEG |  |
| TorD_Ecoli | 120 NEPTDHLAIYLELLSHLFSLG |  |
| TorD_Vibch | 130 NEPEDHLAIQLDFLAHLAISAN |  |

Fig. 4 Domain characteristic of putative enzyme-specific chaperones for ethylbenzene dehydrogenase and other molybdenum enzymes. Conserved amino acids and conservative substitutes are indicated in bold. Abbreviations and accession numbers of the putative enzyme-specific chaperones are as follows: EbdD, AL844858 (EbN1); SerD, Q9S1G8 (Tause); NarJ, AJ277440 (Halma), P11351 (Ecoli), AB0649 (Salch) and AAL65041 (Pyrae); TorD, P36662 (Ecoli) and AAF94870 (Vibch). Abbreviations of bacterial species (and strains) are as explained in the footnote of Table 2 and for Fig. 3; further abbreviations are: Salch, Salmonella cholerae, formerly Salmonella enterica; Pyrae, Pyrobaculum aerophilum

molybdopterin in these subunits. Sequence similarities of EbdA with other molybdenum-containing enzyme subunits suggest an affiliation with the DMSO reductase family (Kisker et al. 1997). Ethylbenzene dehydrogenase has recently been suggested to form a subfamily together with membrane-bound nitrate reductases and periplasmic selenate and DMSO reductases (McDevitt et al. 2002). NarG-containing periplasmic nitrate reductase from Haloarcula marismortui (Yoshimatsu et al. 2002) may represent yet another member of this subgroup.

Gene of (S)-1-phenylethanol dehydrogenase (ped)

Using the N-terminal sequence of purified (S)-1-phenylethanol dehydrogenase from strain EbN1 (Kniemeyer and Heider 2001b), the coding gene, *ped* (*orf58*), was identified. The *ped* gene is located directly downstream of the *ebdABCD* genes, suggesting co-transcription. The INTERPRO entry for the *ped* gene product indicates affiliation with the large family of so-called short-chain dehydrogenases/reductases (SDR; alcohol dehydrogenases with short peptide chain; Jörnvall et al. 1981). An NAD⁺-binding motif characteristic of the SDR family G-x₃-G-x-G (Jörnvall et al. 1995) was identified in the N-terminal region (positions 15–20) of Ped. This is in agreement with the experimentally demonstrated NAD⁺-dependence of the enzyme (Kniemeyer and Heider 2001b).

Genes of putative acetophenone carboxylase (*apcl–5*)

N-terminal sequences of several proteins that are specifically formed during growth on ethylbenzene or acetophenone were separated by 2-dimensional gel electrophoresis (Champion et al. 1999) matched to the amino acid sequences translated from *orf97* (*apc5*), *orf100* (*apc4*), *orf102* (*apc3*) and *orf105* (*apcl*). PSI-BLAST analysis indicates sequence similarity of Orf100 to the α-subunit, of Orf105 and Orf102 to the β-subunit, and of Orf200 (Apc2) to the γ-subunit of acetone carboxylase from aerobic *Xanthobacter autotrophicus* strain Py2 (Sluis et al. 2002). This enzyme, which catalyzes the ATP-dependent carboxylation of acetone to acetylacetate, has an α₂β₂γ₂ structure (Sluis and Ensign 1997). ATP-dependent acetone carboxylases have also been reported in anaerobic bacteria (Janssen and Schink 1995; Birks and Kelly 1997; Ensign et al. 1998). The presence of an ATP-binding motif in Orf105 agrees with these observations. We therefore suggest that the gene products of *orf100*, *orf102*, *orf200* and *orf105* constitute four subunits of the postulated acetophenone carboxylase. Since *orf97* is apparently co-transcribed with the other four *orfs*, the *orf97* gene product might constitute another subunit of this enzyme. Hence, acetophenone and acetone carboxylases may differ with respect to subunit composition (five versus three) of the holoenzymes, despite similar enzymatic reactions. BLAST and PSI-BLAST analysis of the gene products of *orf100*, *orf102* and *orf105* indicated high similarities to *N*-methylhydantoinases, which catalyze ATP-dependent hydrolysis of cyclic amide bonds (Ogawa et al. 1995). It may be suspected that some of the genes currently annotated as putative *N*-methylhydantoinases in complete genome sequences actually code for carboxylases.

from *Azoarcus evansii* (Mohamed 2000), but only weak similarity with BadA, a benzoyl-CoA-ligase from *Rhodopseudomonas palustris*; the latter enzyme is highly specific for benzoate (Egland et al. 1995). The apparent co-transcription of *orf95* with the genes for acetophenone carboxylase (*apc*) suggests a concerted function in the degradation pathway; *orf95* may, therefore, code for the proposed benzoylacetate CoA-ligase (Bal).

Gene of a putative transporter

Gene *orf90* is located upstream of the genes suggested to code for acetophenone carboxylase (*apc*) and benzoylacete CoA-ligase (*bal*). The gene product of *orf90* shows highest sequence similarities to transport proteins, some of which affiliate with the major facilitator superfamily (MFS) and resistance-nodulation-cell division superfam-ily (RND). Members of these transporter superfamilies are involved in transport (including also efflux) of aromatic compounds and organic solvents (Saier 2000). One may speculate that Orf90 transports (S)-1-phenylethanol, the product of the periplasmic ethylbenzene dehydrogenase, into the cytoplasm; even though (S)-1-phenylethanol as an uncharged, hydrophobic substance may freely diffuse through the cytoplasmic membrane, a specific transporter may concentrate this metabolite in the cytoplasm and minimize loss to the environment.

Genes of a fatty acid metabolism

Genes *orf203*, *orf206*, *orf208*, *orf212* and *orf18* encode proteins which are probably all involved in the degradation of fatty acids. The operon-like arrangement of these *orfs* contrasts with the scattered organization of the orthologous genes in *E. coli* (Clark and Cronan 1996). The gene product of *orf203* is highly similar to the FadL protein, an outer membrane protein for the uptake of long-chain fatty acids. Genes *orf208* and *orf212* are likely to code for the α-subunit (FadB) and β-subunit (FadA), respectively, of the fatty acid oxidation complex. The gene product of *orf18* shows high similarity to the long-chain acyl-CoA dehydrogenase, FadE (Campbell and Cronan 2002), which is sometimes also referred to as FadF (Black and DiRusso 1994). *Orf206* possibly codes for a thioesterase acting on long-chain acyl-CoA. However, the function of this type of thioesterase is presently unknown (Cho and Cronan 1993).

Since fatty acid oxidation complexes are known to exhibit relaxed substrate specificity, one may envision that the gene product of *orf212*, which is homologous to 3-ketoacyl-CoA thiolase (FadA), could possess thiolase activity towards benzoylacetyl-CoA and form benzoyl-CoA and acetyl-CoA. Nevertheless, it cannot be excluded that a specific thiolase for benzoylacetyl-CoA is actually encoded outside of the studied contig.

Gene of putative benzoylacetate CoA-ligase (*bal*)

The gene product of *orf95* belongs to the family of AMP-forming CoA-ligases. PSI-BLAST analysis resolved strong sequence similarity to aerobic phenylacetate CoA-ligase

Regulatory proteins

The product of *orf220* resembles transcriptional regulators of the TetR family, which encompasses a large group of DNA-binding regulatory proteins blocking transcription upon binding with their helix-turn-helix motifs to the operator (Saenger et al. 2000). The location of genes related to fatty acid oxidation directly upstream of *orf220* (*reg1*) suggests a possible involvement in the regulation of these genes. However, PSI-BLAST analysis did not reveal similarity of Orf220 to the known repressor of the β-oxidation pathway, FadR, which belongs to the GntR family of transcriptional regulators (Haydon and Guest 1991).

Proteins encoded by *orf250* (*reg2*) and *orf53* (*reg3*) are related to the LysR family of transcriptional regulators that are known to regulate many aerobic and anaerobic pathways for the degradation of aromatic compounds, usually by activating transcription (Gerischer 2002).

Gene products of the pairs *orf75/orf77* (*tcs1/tcr1*) and *orf81/orf82* (*tcs2/tcr2*) resemble typical two-component regulatory systems, with Orf75 and Orf81 representing the sensors and Orf77 and Orf82 the response regulators. The two sensor proteins contain so-called PAS domains, which are widespread sensor motifs assumed to monitor stimuli (Taylor and Zhulin 1999). The product of *tcs1/tcr1* shows highest similarities to two-component regulatory systems of plant-pathogenic bacteria, e.g., HprXY of *Pantoea stewartii*. The two-component system VirAG of *Agrobacterium tumefaciens* is known to sense phenolic compounds such as p-hydroxyacetophenone (Lee et al. 1995) and to regulate transcription of the virulence (*vir*) genes. In analogy, the Tcs1/Tcr1 system of strain EbN1 might sense acetophenone and regulate expression of the corresponding section of the ethylbenzene degradation pathway. This assumption is supported by the detection of specific proteins by 2-dimensional gel electrophoresis upon growth with acetophenone (Champion et al. 1999) and the tentative identification of these proteins as acetophenone carboxylase (see concerning section of this study). For the *tcs2/tcr2* system, PSI-BLAST analysis revealed sequence similarity with two-component systems involved in the regulation of alkylbenzene catabolism. These are: (1) the *tdiSR* systems of *T. aromatica* strain K172 and *Azoarcus*-like strain T (Leuthner and Heider 1998; Achong et al. 2001) and the *tutBC* system of *T. aromatica* strain T1 (Coscigano and Young 1997), all involved in the transcriptional regulation of benzylsuccinate synthase, the initial enzyme in anaerobic toluene degradation; (2) the *todST* system regulating aerobic toluene catabolism in *Pseudomonas putida* strain F1 (Lau et al. 1997); and (3) the *stySR* system controlling the expression of the aerobic styrene pathway in *Pseudomonas sp.* strain Y2 (Velasco et al. 1998). In all these cases, the genes coding for the two-component regulatory system are located in close proximity to the structural genes of the regulated catabolic route. Due to the proximity to the *ebdABCD* genes, the *tcs2/tcr2* system may code for the sensor for ethylbenzene and regulator of the “upper” part of the degradative pathway in strain EbN1.

---

Conclusion

The present study has revealed tentative genes for anaerobic ethylbenzene degradation which are organized in close proximity to each other. Genes for the degradation pathway of toluene are probably located outside of the presently studied contig. The independent induction of each of these pathways (Rabus and Heider 1998; Champion et al. 1999) as well as the presently observed modular organization of structural genes and tentative two-component regulatory systems for the catabolism of ethylbenzene suggest a complex regulatory system for the degradation of aromatic compounds in strain EbN1. For instance, ethylbenzene may not induce the entire pathway for its degradation, but only the initial two dehydrogenases. Only the product acetophenone may trigger synthesis of enzymes for subsequent reactions. Such sequential induction may not only allow a finely tuned regulation of the pathway for ethylbenzene mineralization, but also a versatile response to the availability of different aromatic substrates in the environment. The roles of individual regulatory proteins with respect to specific substrate recognition and signal transduction are of particular interest for future investigations.

Acknowledgements We wish to thank Katja Heitmann (Berlin) for help in finishing the DNA sequence, S. Kühner, D. Lange and J. Wulf (Bremen) for technical assistance, J. Heider (Freiburg) for valuable discussions, and R. Amann (Bremen) for general support of genomic work at our institute. This work was supported by the Max-Planck-Society and the Deutsche Forschungsgemeinschaft.

---

References

Achong GR, Rodriguez AM, Spormann AM (2001) Benzylsuccinate synthase of *Azoarcus* sp. strain T: cloning, sequencing, transcriptional organization, and its role in anaerobic toluene and m-xylene mineralization. J Bacteriol 183:6763–6770

Altschul SF, Madden TL, Schäffer AA, Zhang J, Zhang Z, Miller W, Lipman DJ (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 25:3389–3402

Apweiler R, Attwood TK, Bairoch A, Bateman A, Birney E, Biswas M, Bucher P, Cerutti L, Corpet F, Croning MDR, Durbin R, Falquet L, Fleischmann W, Gouzy J, Hermjakob H, Hulo N, Jonassen I, Kahn D, Kanapin A, Karavidopoulou Y, Lopez R, Marx B, Mulder NJ, Oinn TM, Pagni M, Servant F, Sigrist CJA, Zdobnov EM (2001) The InterPro database, an integrated documentation resource for protein families, domains and functional sites. Nucleic Acids Res 29:37–40

Ausubel FM, Brent R, Kingston RE, Moore DD, Seidman JG, Smith JA, Struhl K (1992) Current protocols in molecular biology. Greene/Wiley-Interscience, New York

Bairoch A, Apweiler R (2000) The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000. Nucleic Acids Res 28:45–48

Ball HA, Johnson HA, Reinhard M, Spormann AM (1996) Initial reactions in anaerobic ethylbenzene oxidation by a denitrifying bacterium, strain EB1. J Bacteriol 178:5755–5761

Baxevanis AD (2002) The molecular biology database collection: 2002 update. Nucleic Acids Res 30:1–12

Berg BL, Li J, Heider J, Stewart V (1991) Nitrate-inducible formate dehydrogenase in *Escherichia coli* K-12. I. Nucleotide sequence of the *fdnGHI* operon and evidence that opal (UGA) encodes selenocysteine. J Biol Chem 266:22380–22385

Berks BC, Sargent F, Palmer T (2000) The Tat protein export pathway. Mol Microbiol 35:260–274

Birks SJ, Kelly DJ (1997) Assay and properties of acetone carboxylase, a novel enzyme involved in acetone-dependent growth and CO₂ fixation in *Rhodobacter capsulatus* and other photosynthetic and denitrifying bacteria. Microbiology 143: 755–766

Black PN, DiRusso CC (1994) Molecular and biochemical analyses of fatty acid transport, metabolism, and gene regulation in *Escherichia coli*. Biochim Biophys Acta 1210:123–145

Blasco F, Iobbi C, Giordano G, Chippaux M, Bonnefoy V (1989) Nitrate reductase of *Escherichia coli*: completion of the nucleotide sequence of the *nar* operon and reassessment of the role of the α and β subunits in iron binding and electron transfer. Mol Gen Genet 218:249–256

Blasco F, Dos Santos J-P, Magalon A, Frixon C, Guigliarelli B, Santini C-L, Giordano G (1998) NarJ is a specific chaperone required for molybdenum cofactor assembly in nitrate reductase A of *Escherichia coli*. Mol Microbiol 28:435–447

Blattner FR, Plunkett III G, Bloch CA, Perna NT, Burland V, Riley M, Collado-Vides J, Glasner JD, Rode CK, Mayhew GF, Gregor J, Davis NW, Kirkpatrick HA, Goeden MA, Rose DJ, Mau B, Shao Y (1997) The complete genome sequence of *Escherichia coli* K-12. Science 277:1453–1474

Boyington JC, Gladyshev VN, Khangulov SV, Stadtman TC, Sun PD (1997) Crystal structure of formate dehydrogenase H: catalysis involving Mo, molybdopterin, selenocysteine, and an Fe₄S₄ cluster. Science 275:1305–1308

Campbell JW, Cronan Jr JE (2002) The enigmatic *Escherichia coli fadE* gene is *yafH*. J Bacteriol 184:3759–3764

Champion KM, Zengler K, Rabus R (1999) Anaerobic degradation of ethylbenzene and toluene in denitrifying strain EbN1 proceeds via independent substrate-induced pathways. J Mol Microbiol Biotechnol 1:157–164

Cho H, Cronan Jr JE (1993) *Escherichia coli* thioesterase I, molecular cloning and sequencing of the structural gene and identification as a periplasmic enzyme. J Biol Chem 268:9238–9245

Clark DP, Cronan Jr JE (1996) Two-carbon compounds and fatty acids as carbon sources. In: Neidhardt FC, Curtiss III R, Ingraham JL, Lin ECC, Low KB, Magasanik B, Reznikoff WS, Riley M, Schaechter M, Umbarger HE (eds) *Escherichia coli* and *Salmonella*. Cellular and molecular biology, vol I, 2nd edn. ASM, Washington, pp 343–357

Coschigano PW, Young LY (1997) Identification and sequence analysis of two regulatory genes involved in anaerobic toluene metabolism by strain T1. Appl Environ Microbiol 63:652–660

Dias JM, Than ME, Humm A, Huber R, Bourenkov GP, Bartunik HD, Bursakov S, Calvete J, Caldeira J, Carneiro C, Moura JJG, Moura I, Romão MJ (1999) Crystal structure of the first dissimilatory nitrate reductase at 1.9 Å solved by MAD methods. Structure 7:65–79

Egland PG, Gibson J, Harwood CS (1995) Benzoate-coenzyme A ligase, encoded by *badA*, is one of three ligases able to catalyze benzoyl-coenzyme A formation during anaerobic growth of *Rhodopseudomonas palustris* on benzoate. J Bacteriol 177: 6545–6551

Ensign SA, Small FJ, Allen JR, Sluis MK (1998) New role of CO₂ in the microbial metabolism of aliphatic epoxides and ketones. Arch Microbiol 169:179–187

Ewing B, Green P (1998) Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res 8:186–194

Ewing B, Hiller L, Wendl MC, Green P (1998) Base-calling of automated sequencer traces using phred. I. Accuracy assessment. Genome Res 8:175–185

Frishman D, Mironov A, Mewes H-W, Gelfand M (1998) Combining diverse evidence for gene recognition in completely sequenced bacterial genomes. Nucleic Acids Res 26:2941–2947

Gerischer U (2002) Specific and global regulation of genes associated with the degradation of aromatic compounds in bacteria. J Mol Microbiol Biotechnol 4:111–121

Harwood CS, Burchardt G, Hermann H, Fuchs G (1999) Anaerobic metabolism of aromatic compounds via the benzoyl-CoA pathway. FEMS Microbiol Rev 22:439–458

Hattori M, Fujiyama A, Taylor TD, Watanabe H, Yada T, Park H-S, Toyoda A, Ishii K, Totoki Y, Choi D-K, Soeda E, Ohki M, Takagi T, Sakaki Y, Taudien S, Blechschmidt K, Polley A, Menzel U, Delabar J, Kumpf K, Lehmann R, Patterson D, Reichwald K, Rump A, Schillhabel M, Schudy A, Zimmermann W, Rosenthal A, Kudoh J, Shibuya K, Kawasaki K, Asakawa S, Shintani A, Sasaki T, Nagamine K, Mitsuyama S, Antonarakis SE, Minoshima S, Shimizu N, Nordsiek G, Hornischer K, Brandt P, Scharfe M, Schön O, Desario A, Reichelt J, Kauer G, Blöcker H, Ramser J, Beck A, Klages S, Hennig S, Riesselmann L, Dagand E, Haaf T, Wehrmeyer S, Borzym K, Gardiner K, Nizetic D, Francis F, Lehrach H, Reinhardt R, Yaspo M-L (2000) The DNA sequence of human chromosome 21. Nature 405:311–319

Haydon DJ, Guest JR (1991) A new family of bacterial regulatory proteins. FEMS Microbiol Lett 63:291–295

Heider J, Spormann AM, Beller HR, Widdel F (1999) Anaerobic bacterial metabolism of hydrocarbons. FEMS Microbiol Rev 22:459–473

Janssen PH, Schink B (1995) ¹⁴CO₂ exchange with acetoacetate catalyzed by dialyzed cell-free extracts of the bacterial strain BunN grown with acetone and nitrate. Eur J Biochem 228: 677–682

Johnson HA, Spormann AM (1999) In vitro studies on the initial reactions of anaerobic ethylbenzene mineralization. J Bacteriol 181:5662–5668

Johnson HA, Pelletier DA, Spormann AM (2001) Isolation and characterization of anaerobic ethylbenzene dehydrogenase, a novel Mo-Fe-S enzyme. J Bacteriol 183:4536–4542

Jormakka M, Törnroth S, Byrne B, Iwata S (2002) Molecular basis of proton motive force generation: structure of formate dehydrogenase-N. Science 295:1863–1868

Jörnvall H, Persson M, Jeffery J (1981) Alcohol and polyol dehydrogenases are both divided into two protein types, and structural properties cross-relate the different enzyme activities within each type. Proc Natl Acad Sci USA 78:4226–4230

Jörnvall H, Persson B, Krook M, Atrian S, González-Duarte R, Jeffery J, Ghosh D (1995) Short-chain dehydrogenases/reductases (SDR). Biochemistry 34:6003–6013

Kisker C, Schindelin H, Rees DC (1997) Molybdenum-cofactor-containing enzymes: structure and mechanism. Annu Rev Biochem 66:233–267

Kniemeyer O, Heider J (2001a) Ethylbenzene dehydrogenase, a novel hydrocarbon-oxidizing molybdenum/iron-sulfur/heme enzyme. J Biol Chem 276:21381–21386

Kniemeyer O, Heider J (2001b) (S)-1-Phenylethanol dehydrogenase of *Azoarcus* sp. strain EbN1, an enzyme of anaerobic ethylbenzene catabolism. Arch Microbiol 176:129–135

Krafft T, Bowen A, Theis F, Macy JM (2000) Cloning and sequencing of the genes encoding the periplasmic-cytochrome b-containing selenate reductase of *Thauera selenatis*. DNA Sequence 10:365–377

Kropp KG, Davidova IA, Suflita JM (2000) Anaerobic oxidation of n-dodecane by an addition reaction in a sulfate-reducing bacterial enrichment culture. Appl Environ Microbiol 66: 5393–5398

Lau PCK, Wang Y, Patel A, Labbé D, Bergeron H, Brousseau R, Konishi Y, Rawlings M (1997) A bacterial basic region leucine zipper histidine kinase regulating toluene degradation. Proc Natl Acad Sci USA 94:1453–1458

Lee Y-W, Jin S, Sim W-S, Nester EW (1995) Genetic evidence for direct sensing of phenolic compounds by the VirA protein of *Agrobacterium tumefaciens*. Proc Natl Acad Sci USA 92: 12245–12249

Leuthner B, Heider J (1998) A two-component system involved in regulation of anaerobic toluene metabolism in *Thauera aromatica*. FEMS Microbiol Lett 166:35–41

Liu X, DeMoss JA (1997) Characterization of NarJ, a system-specific chaperone required for nitrate reductase biogenesis in *Escherichia coli*. J Biol Chem 272:24266–24271

McDevitt CA, Hugenholtz P, Hanson GR, McEwan AG (2002) Molecular analysis of dimethyl sulphide dehydrogenase from *Rhodovulum sulfidophilum*: its place in the dimethyl sulphoxide reductase family of microbial molybdopterin-containing enzymes. Mol Microbiol 44:1575–1587

Meckenstock RU, Annweiler E, Michaelis W, Richnow HH, Schink B (2000) Anaerobic naphthalene degradation by a sulfate-reducing enrichment culture. Appl Environ Microbiol 66: 2743–2747

Méjean V, Iobbi-Nivol C, Lepelletier M, Giordano G, Chippaux M, Pascal M-C (1994) TMAO anaerobic respiration in *Escherichia coli*: involvement of the *tor* operon. Mol Microbiol 11:1169–1179

Mohamed MES (2000) Biochemical and molecular characterization of phenylacetate-coenzyme A ligase, an enzyme catalyzing the first step in aerobic metabolism of phenylacetic acid in *Azoarcus evansii*. J Bacteriol 182:286–294

Nielsen H, Engelbrecht J, Brunak S, von Heijne G (1997) Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites. Protein Engineering 10:1–6

Ogawa J, Min Kim J, Nirdnoy W, Amano Y, Yamada H, Shimizu S (1995) Purification and characterization of an ATP-dependent amidohydrolase, *N*-methylhydantoin amidohydrolase, from *Pseudomonas putida* 77. Eur J Biochem 229:284–290

Perna NT, Plunkett III G, Burland V, Mau B, Glasner JD, Rose DJ, Mayhew GF, Evans PS, Gregor J, Kirkpatrick HA, Posfai G, Hackett J, Klink S, Boutin A, Shao Y, Miller L, Grotbeck EJ, Davis NW, Lim A, Dimalanta ET, Potamousis KD, Apodaca J, Anantharaman TS, Lin J, Yen G, Schwartz DC, Welch RA, Blattner FR (2001) Genome sequence of enterohaemorrhagic *Escherichia coli* O157:H7. Nature 409:529–533

Pommier J, Méjean V, Giordano G, Iobbi-Nivol C (1998) TorD, a cytoplasmic chaperone that interacts with the unfolded trimethylamine *N*-oxide reductase enzyme (TorA) in *Escherichia coli*. J Biol Chem 273:16615–16620

Rabus R, Widdel F (1995) Anaerobic degradation of ethylbenzene and other aromatic hydrocarbons by new denitrifying bacteria. Arch Microbiol 163:96–103

Rabus R, Heider J (1998) Initial reactions of anaerobic metabolism of alkylbenzenes in denitrifying and sulfate-reducing bacteria. Arch Microbiol 170:377–384

Rabus R, Wilkes H, Behrends A, Armstroff A, Fischer T, Pierik AJ, Widdel F (2001) Anaerobic initial reaction of *n*-alkanes in a denitrifying bacterium: evidence for (*l*-methylpentyl)succinate as initial product and for involvement of an organic radical in *n*-hexane metabolism. J Bacteriol 183:1707–1715

Radelf U, Hennig S, Seranski P, Steinfath M, Ramser J, Reinhardt R, Poustka A, Francis F, Lehrach H (1998) Preselection of shotgun clones by oligonucleotide fingerprinting: an efficient and high throughput strategy to reduce redundancy in large-scale sequencing projects. Nucleic Acids Res 26:5358–5364

Rech SA, Macy JM (1992) The terminal reductases for selenate and nitrate respiration in *Thauera selenatis* are two distinct enzymes. J Bacteriol 174:7316–7320

Rieder MJ, Taylor SL, Tobe VO, Nickerson DA (1998) Automating the identification of DNA variations using quality-based fluorescence re-sequencing: analysis of the human mitochondrial genome. Nucleic Acids Res 26:967–973

Saenger W, Orth P, Kisker C, Hillen W, Hinrichs W (2000) The tetracycline repressor – a paradigm for a biological switch. Angew Chem Int Ed 39:2042–2052

Saier Jr MH (2000) A functional-phylogenetic classification system for transmembrane solute transporters. Microbiol Mol Biol Rev 64:354–411

Sargent F, Berks BC, Palmer T (2002) Assembly of membrane-bound respiratory complexes by the Tat protein-transport system 178:77–84

Schneider F, Löwe J, Huber R, Schindelin H, Kisker C, Knäblein J (1996) Crystal structure of dimethyl sulfoxide reductase from *Rhodobacter capsulatus* at 1.88 Å resolution. J Mol Biol 263: 53–69

Silvestro A, Pommier J, Pascal M-C, Giordano G (1989) The inducible trimethylamine *N*-oxide reductase of *Escherichia coli* K12: its localization and inducers. Biochim Biophys Acta 999:208–216

Sluis MK, Ensign SA (1997) Purification and characterization of acetone carboxylase from *Xanthobacter* strain Py2. Proc Natl Acad Sci USA 94:8456–8461

Sluis MK, Larsen RA, Krum JG, Anderson R, Metcalf WW, Ensign SA (2002) Biochemical, molecular and genetic analyses of the acetone carboxylases from *Xanthobacter autotrophicus* strain Py2 and *Rhodobacter capsulatus* strain B10. J Bacteriol 184:2969–2977

Spormann AM, Widdel F (2000) Metabolism of alkylbenzenes, alkanes, and other hydrocarbons in anaerobic bacteria. Biodegradation 11:85–105

Staden R (1982) An interactive graphics program for comparing and aligning nucleic acid and amino acid sequences. Nucleic Acids Res 10:2951–2961

Staden R, Beal KF, Bonfield JK (1999) The Staden Package, 1998. Computer Methods in Molecular Biology. In: Misener S, Krawetz SA (eds). Bioinformatics methods and protocols. Humana, New Jersey, pp 115–130

Tatusov RL, Natale DA, Garkavtsev IV, Tatusova TA, Shankavaram UT, Rao BS, Kiryutin B, Galperin MY, Fedorova ND, Koonin EV (2001) The COG database: new developments in phylogenetic classification of proteins from complete genomes. Nucleic Acids Res 29:22–28

Taylor BL, Zhulin IB (1999) PAS domains: internal sensors of oxygen, redox potential and light. Microbiol Mol Biol Rev 63:479–506

The Gene Ontology Consortium (2001) Creating the gene ontology resource: design and implementation. Genome Res 11: 1425–1433

Velasco A, Alonso S, García JL, Perera J, Díaz E (1998) Genetic and functional analysis of the styrene catabolic cluster of *Pseudomonas* sp. strain Y2. J Bacteriol 180:1063–1071

Widdel F, Rabus R (2001) Anaerobic biodegradation of saturated and aromatic hydrocarbons. Curr Opin Biotechnol 12:259–276

Yoshimatsu K, Iwasaki T, Fujiwara T (2002) Sequence and electron paramagnetic resonance analyses of nitrate reductase NarGH from a denitrifying halophilic euryarchaeote *Halorcula marismortui*. FEBS Letters 516:145–150

Zhang X, Young LY (1997) Carboxylation as an initial reaction in the anaerobic metabolism of naphthalene and phenanthrene by sulfidogenic consortia. Appl Environ Microbiol 63:4759–4764

Zhou J, Bruns MA, Tiedje JM (1996) DNA recovery from soils of diverse composition. Appl Environ Microbiol 62:316–322
