
PHYTOCHEMISTRY

PERGAMON Phytochemistry 62 (2003) 127–137 www.elsevier.com/locate/phytochem

A flavonol O-methyltransferase from Catharanthus roseus performing two sequential methylations

Sabrina Cacace${}^{a,1}$, Gudrun Schröder${}^{a,1}$, Elke Wehinger${}^{a,1}$, Dieter Strack${}^{b}$, Jürgen Schmidt${}^{b}$, Joachim Schröder${}^{a,*}$

${}^{a}$Institut für Biologie II, Universität Freiburg, Schänzlestr. 1, D-79104 Freiburg, Germany  
${}^{b}$Institut für Pflanzenbiochemie, Weinberg 3, D-06120 Halle (Saale), Germany  

Received 22 July 2002; received in revised form 20 September 2002

---

### Abstract

Protein extracts from dark-grown cell suspension cultures of *Catharanthus roseus* (Madagascar periwinkle) contained several $O$-methyltransferase (OMT) activities, including the 16-hydroxytabersonine $O$-methyltransferase (16HT-OMT) in indole alkaloid biosynthesis. This enzyme was enriched through several purification steps, including affinity chromatography on adenosine agarose. SDS-PAGE of the purified protein preparation revealed a protein band at the size expected for plant OMTs (38–43 kDa). Mass spectrometry indicated two dominant protein species of similar mass in this band, and sequences of tryptic peptides showed similarities to known OMTs. Homology-based RT-PCR identified cDNAs for four new OMTs. Two of these cDNAs (*CrOMT2* and *CrOMT4*) encoded the proteins dominant in the preparation enriched for 16HT-OMT. The proteins were closely related (73% identity), but both shared only 48–53% identity with the closest relatives found in the public databases. The enzyme functions were investigated with purified recombinant proteins after cDNA expression in *Escherichia coli*. Unexpectedly, both proteins had no detectable 16HT-OMT activity, and *CrOMT4* was inactive with all substrates investigated. *CrOMT2* was identified as a flavonoid OMT that was expressed in dark-grown cell cultures and copurified with 16HT-OMT. It represented a new type of OMT that performs two sequential methylations at the 3′-and 5′-positions of the B-ring in myricetin (flavonol) and dihydromyricetin (dihydroflavonol). The resulting methylation pattern is characteristic for *C. roseus* flavonol glycosides and anthocyanins, and it is proposed that *CrOMT2* is involved in their biosynthesis.

© 2003 Elsevier Science Ltd. All rights reserved.

Keywords: *Catharanthus roseus*; Apocynaceae; Madagascar periwinkle; Homology based cDNA cloning; Recombinant protein expression; $O$-Methyltransferase; Flavonoids

---

### 1. Introduction

S-Adenosyl-L-methionine (SAM) dependent methylations are important in the biosynthesis of many plant secondary products, e.g. phenylpropanoids (Gang et al., 2001; Dixon et al., 2001), flavonoids (Dixon, 1999; Forkmann and Heller, 1999; Harborne and Williams, 2000), alkaloids (Frick and Kutchan, 1999; Morishige et al., 2000), and other substances (Ibrahim and Muzac, 2000). The DNA databases contain at present (June 2002) more than a 100 entries for plant sequences labelled as $O$-methyltransferases (OMTs), and many of the assignments were only based on similarity with functionally identified OMTs.

A previous study (Schröder et al., 2002) addressed the question whether protein sequences obtained via homology-based cloning strategies provide sufficient information for predictions on the substrates of new OMTs. One of the conclusions was that even 85% or higher identity scores may be misleading, and the large subfamily containing most of the known caffeic acid OMTs (COMTs) provided a convincing example: it contained several multifunctional enzymes (Gauthier et al., 1998; Frick and Kutchan, 1999; Chiron et al., 2000), and also enzymes with little or no COMT activity (Gauthier et al., 1996; Seguin et al., 1998; Wang and Pichersky, 1999; Muzac et al., 2000).

* Corresponding author. Tel.: +49-761-203-2691; fax: +49-761-203-2601.  
E-mail address: joachim.schroeder@biologie.uni-freiburg.de (J. Schröder).  
${}^{1}$ These authors contributed equally to the experiments.

Our subsequent work therefore combined molecular and biochemical strategies. In parallel to homology-based PCR cloning, we purified OMT activities to obtain peptide sequences that would facilitate the identification of cloned sequences. The guide for the enzyme purification was an OMT in the indole alkaloid pathway from tabersonine to vindoline, the 16-hydroxytabersonine *O*-methyltransferase (16HT-OMT, EC 2.1.1.94) (Fahn et al., 1985; De Luca et al., 1986) that synthesizes methoxytabersonine (Fig. 1A). Several additional OMT activities were also monitored during the purification to follow the separation of 16HT-OMT from other OMTs. Here we describe the results of the combined approach. They show that the purified enzyme preparation contained two dominant protein species, but both were not 16HT-OMT, as demonstrated by assays with recombinant proteins expressed from the corresponding cDNAs. Investigations with other substrates unexpectedly revealed that one of the dominant proteins was a flavonoid OMT that was expressed in dark-grown cells synthesizing no flavonoids.

cultures, and this was confirmed with the cell line maintained in our laboratory as suspension culture growing in continuous dark. Various conditions known to enhance alkaloid production led at most to a 2-fold activity increase, and therefore the purifications were carried out with cultures grown under standard conditions. The use of dark-grown cultures also appeared a preferable choice because they do not synthesize flavonoids (Knobloch et al., 1982), and thus appeared unlikely to express the OMTs that could be expected from the structures of the *C. roseus* flavonoids (Fig. 1B) (Carew and Krueger, 1976; Knobloch et al., 1982; Milo et al., 1985; Brun et al., 1999).

A typical purification is summarized in Table 1 and Fig. 2. The 16HT-OMT activity was purified through three steps (fractionated ammonium sulfate precipitation, DEAE sepharose, and affinity chromatography on adenosine agarose), resulting in a ca. 40-fold increase of the specific activity. The activity proved to be very unstable, and the final yield was only ca. 1% in the purified preparation. One reason for the low yield was that only ca. 18% of the enzyme activity bound to the adenosine agarose, but this was the only step leading to an active enzyme preparation containing only a few major proteins suitable for protein sequencing. Several other procedures were investigated, but they did not improve the purification, either because of even more extensive losses of activity (size fractionation; chromatography on hydroxyapatite; chromatofocusing; hydrophobic interaction chromatography) or because the enzyme failed to bind to the column material (several

### 2. Results and discussion

#### 2.1. Purification of OMTs and peptide sequences

Previous experiments (St-Pierre and De Luca, 1995) had reported the presence of 16HT-OMT activity in cell

![Figure 1](#fig1)

**Fig. 1.** Methylated secondary products in *C. roseus*. A. Methylation reaction in the pathway from tabersonine to vindoline. B. Methylated flavonoids identified from *C. roseus*. Syringetin 3-*O*-robinobioside = syringetin 3-*O*-α-*L*-rhamnopyranosyl-(1-6)-β-*D*-galactopyranoside (Brun et al., 1999). The anthocyanin glycosylation patterns have not been analyzed (Carew and Krueger, 1976; Milo et al., 1985).

![Figure 2](#fig2)

**Fig. 2.** Purification of 16HT-OMT from *C. roseus* cell suspension cultures. Lanes: 1, crude extract (35 μg); 2, ammonium sulfate precipitation (60–80% saturation) (35 μg); 3, DEAE-cellulose, elution with 0.2 M NaCl (25 μg); 4, adenosine agarose, elution with 4 mM SAM (10 μg protein).

S. Cacace et al./ Phytochemistry 62 (2003) 127–137

Table 1  
Typical purification of OMT activities from C. roseus cell suspension cultures. The total activities (nmol/min per total protein in the fractions) are given in square brackets

| Fraction               | Protein (mg) | Specific activity (pmol/min/mg) $^a$ |
|------------------------|--------------|--------------------------------------|
|                        |              | 16HT-OMT                            |
|                        |              | Myricetin                           |
|                        |              | Caffeic acid                        |
|                        |              | Catechol                            |
| Crude extract          | 2700         | 4[10.8]                             |
| Ammonium sulfate       |              |                                     |
| 30–60%                 | 1920         | 1[1.9]                              |
| 60–80%                 | 420          | 14[5.9]                             |
| DEAE sepharose         |              |                                     |
| 0.1M NaCl              | 275          | ≤1                                  |
| 0.2M NaCl              | 65           | 40[2.6]                             |
| Adenosine agarose      |              |                                     |
| Flow through           | 25           | 21[0.53]                            |
| KCl wash               | 0.3          | 221[0.07]                           |
| SAM eluate             | 0.8          | 158[0.13]                           |

$^a$ Average from triplicate assays.

dye affinity chromatographies, including Reactive Red, Brown, Green, or Cibacron Blue).

Incubations of the crude extracts with other substrates showed that they also contained activities methylating caffeic acid (COMT), catechol, and the flavonol myricetin (Table 1). All of these could be explained by the presence of a single enzyme, a COMT, because caffeic acid and catechol are the prototype COMT substrates, and myricetin had been shown to be methylated by the recombinant CrCOMT from C. roseus (Schröder et al., 2002). The reason for the large increase of myricetin activity after ammonium sulfate precipitation remained unclear, but it seemed possible that endogenous inhibitors were removed in this step. The ammonium sulfate fractionation and the 0.2 M NaCl DEAE sepharose step separated almost all of this activity from 16HT-OMT. However, a residual myricetin activity copurified with 16HT-OMT through the adenosine agarose step (Table 1). Activities with caffeic acid or catechol were no longer detectable at this stage, and therefore these data provided a first suggestion that the dark-grown cells contained a flavonoid-specific OMT. Its purification could not be quantified because it could not be distinguished from the CrCOMT activities with the flavonoid substrate. Its presence was unexpected because these suspension cultures do not synthesize flavonoids in the dark (Knobloch et al., 1982), and it is also known that the expression of the key enzyme in the precursor formation, chalcone synthase, is strictly light-dependent (Kaltenbach et al., 1999).

The SDS gel electrophoretic analysis of the enzyme preparation obtained from the SAM-eluate of the adenosine affinity column revealed two dominant protein bands (Fig. 2, lane 4). The 70 kDa band showed a strong cross-reaction with antiserum against HSP 70 from tomato (Neumann et al., 1987), suggesting that a chaperonin type protein copurified with the OMT. In the size range expected for plant OMTs (38–43 kDa),

Table 2  
Amino acid sequences from tryptic peptides obtained with the purified proteins (SAM eluate of adenosine agarose column) $^a$

| Tryptic peptide (Da) | Amino acid sequence $^b$ |
|----------------------|--------------------------|
| Peak 1               |                          |
| Pep1                 | 2956                     | MLIPEFNYLFEGLDSLVDVGGGxxT |
| Pep2                 | 2682                     | AWNAMSEWFQNE                |
| Pep3                 | 1083                     | HENEEAVK                   |
| Pep4                 | 1228                     | NFWDFGAEDK                 |
| Pep5                 | 1038                     | [i,h,s,g,e]FPMIDFR         |
| Peak 2               |                          |
| Pep6                 | 1557                     | SVLHDWkde[e,d]             |
| Pep7                 | 1138                     | NVYSLTSL[s,d][r,k]         |
| Pep8                 | 1422                     | ILVAAGYFSEEPK              |

$^a$ The total mass of the proteins in peaks 1 and 2 were 39,191±3 Da and 38,835±2 Da, respectively.  
$^b$ x, Residue not identified; small letters, identification not certain; residues in square brackets, possible alternatives.

the preparation revealed a dominant band at ca. 39 kDa and a very weak band at ca. 41 kDa that was detectable only with high protein amounts applied to the gels. The stained gels also showed a weak band at ca. 27 kDa. The size was characteristic for caffeoyl-CoA dependent OMTs (Schmitt et al., 1991; Ibrahim, 1997), and the corresponding activity could be demonstrated, but was not further investigated.

This preparation was used for the further analysis (Environment- und Biotechnologie Laboratorium GmbH, Freiburg). The 39 kDa band contained two protein species with masses of 39,191±3 Da (Peak 1) and 38,835±2 Da (Peak 2), respectively, and a ratio of ca. 4 to 1 in favor of Peak 1. Tryptic digests of this polypeptide and sequencing identified the five peptides shown in Table 2. A second purification with slight variations led to a more balanced ratio of the two protein species. The second analysis of tryptic peptides focused on Peak 2, and this identified sequences for

three additional peptides (Table 2, Pep 6 to 8). All eight
peptides revealed significant similarities with OMTs
from other plants. None of them belonged to the pre-
viously cloned CrCOMT (Schröder et al., 2002), as
would be expected from the absence of significant
COMT activity in the analyzed protein fraction. The
results indicated that the purified enzyme preparation
contained two dominant proteins that were candidates
for novel OMT activities.

2.2. cDNA clones for the purified proteins

The molecular strategy was not based on the peptides
obtained from the purified proteins because one of our
interests was a general overview of OMTs expressed in
the dark-grown suspension culture cells. The first partial
clones were obtained by RT-PCR with a new degenerate
primer (see Experimental). The DNA sequences showed
that the cells contained mRNAs for at least four differ-
ent proteins related to known OMTs, in addition to the
previously characterized CrCOMT. Although it was
difficult to exclude that the experiments missed other
OMTs, the identification of the cDNAs corresponding

to the purified proteins (see below) argued that the iso-
lated cDNAs represented at least the dominantly
expressed OMTs. Overlapping clones and then cDNAs
for the complete coding regions were obtained by PCR
with gene-specific primers in 5'-and 3'-RACEs.
The protein deduced from cDNA CrOMT2 (Fig. 3)
corresponded precisely to the peptides obtained from
Peak 1 of the purified protein preparation, not only in
the amino acid sequences but also in the measured and
deduced sizes of the tryptic peptides. The calculated
mass (39,145 Da) was slightly smaller than that
obtained from the purified protein (39,191±3Da). The
difference was as expected from N-terminal acetylation
(+42Da), and this was consistent with the failure to
obtain N-terminal sequences of the polypeptide. A sec-
ond cDNA (CrOMT4) predicted a protein containing
the peptides obtained from Peak 2 of the purified
enzyme preparation (Fig. 3), and again the match was
not only in the sequences, but also in the sizes of the
analyzed and the predicted tryptic peptides. Like
CrOMT2, CrOMT4 isolated from the plant tissue was
most likely acetylated because the predicted mass
(38,789Da) was smaller than the experimentally deter-

CrOMT2 MELQSSEIRNAQAHFFTQVFSFTSMSSLKCAVQLGIPDAIHSHGKPMALS 50
CrOMT4 MDLETAEIRKAQAQRSRHVFSFVSTSCCLKCAIQLEIPEAIHNHGKPMTLS 50
CrOMT2 DLTNSLPINPSKAPYIYRLMILVAAGYFSEEEKNVYSLTPFTRLLLLKND 100
CrOMT4 DLTNSLPINPSKAPYIHRLMILVAAGYFSEEPKNVYSLTLSRILVKNO 100
| ---Pep8--- | | --Pep7-- |
| ---Pep2--- | | --Pep4-- |
CrOMT2 PLNSISMVLGVNQIAELAWNAMSEWFQNEDLTAFETAHGKNFWDEGAED 150
CrOMT4 PLNLREFVLSANEIAEVEGWNALSEWFQNDVATAFQTAHGKTWYEYLSQD 150
| ----------Pep1---------- |
CrOMT2 KYGKNFDGVMAADSIILVSMLIPEFNYLFEGLDSLVDVGGGTGTIAKAI A 200
CrOMT4 KYGKNFDQLMATDSLLISKLIPDYNYLFEGLISLVDVGGGTGTLAGAVA 200
CrOMT2 KSFPDLKCTVFDLPHVVANLESTENLEFVGGDMEKIPSANAILLKWILH 250
CrOMT4 KAFPNLKCTVFEQPHVIADLEAKGNLEFVGGDMEKIPSANAILLKSVLH 250
| --- |
| -Pep3- |
CrOMT2 DWKDEECVKVLKMCRKAIPEKEKGKVILIETVLMDSKHENEEAVKAI 300
CrOMT4 DWKDEDSVKILKNCKKAIPEKEKGKVIVIDIVLMDSKHDN-PLVKSQI 299
Pep6-
| -Pep5- |
CrOMT2 SSDIDMMVFFTAKERTEEEWATLFREAGFSGYIFPMIDFRSPIEVYP* 348
CrOMT4 SGDMDMMVSMGAERTEEEWAALFKEAGFSGYIFPMIDFRSPIEVYP* 347

Fig. 3. Proteins encoded in CrOMT2 and CrOMT4 from *C. roseus*. The position of the peptide sequences are indicated (bold), and the residues distinguishing between the two proteins are underlined. The cleavage sites (K or R) leading to the tryptic peptides are boxed. Sizes of the predicted trypsin digestion fragments corresponding to the peptide sequences: Pep1, 2957Da; Pep2, 2684Da; Pep3, 1083Da; Pep4, 1228Da; Pep5, 1038Da; Pep6, 1558Da; Pep7, 1139Da; Pep8, 1424Da. ▼, position of the single intron. The GenBank accession numbers for the DNA sequences are AY127568 (CrOMT2) and AY127569 (CrOMT4).

determined size (38,835 ± 2 Da) by the expected mass. The results showed that CrOMT2 and CrOMT4 represented the cDNAs for the dominant OMT-related proteins in the purified enzyme preparation.

CrOMT2 and CrOMT4 were closely related (73% identity), and the analysis of PCR-amplified genomic sequences showed that both possessed a single intron splitting the protein coding region at the same position (Fig. 3). However, the intron sequences revealed no significant similarity, and the sizes were different (CrOMT2 = 380 bp; CrOMT4 = 369 bp). Genomic Southern blots revealed different hybridization patterns for the two genes and in both cases only a few strongly hybridizing bands, suggesting the presence of only one or at most a few gene copies. Interestingly, the cDNA and also the genomic clone for CrOMT4 isolated from the cell culture contained, located 132 bp upstream of the CrOMT4 start codon, sequences from the start of the CrOMT2 coding region. Although we have no evidence that the cell culture used in our experiments contained extensive DNA rearrangements, this appears to be common with other cells maintained in culture, and therefore the experiments were repeated with DNA isolated from leaves of *C. roseus* plants. The genomic fragment analyzed after PCR amplification with appropriate primers revealed the same sequences, indicating that the unusual genomic arrangement was not an aberration that was present only in the cell culture. It was not investigated further whether the cells contained another CrOMT4 gene without CrOMT2 sequences. The significance of this observation remains to be explored, but it is not excluded that the expression of the identified CrOMT4 gene may be under control of sequences from CrOMT2. A Northern blot analysis indicated indeed that all tissues containing CrOMT2

transcripts also contained transcripts from CrOMT4 (not shown).

Searches of the databases (June 2002) showed that no closely related OMTs had been described from other plants. The highest scores (48–53% identity) for both of the new OMT proteins were obtained with OMTs from *Prunus amygdalus* (AJ223151) (Suelves and Puigdomènech, 1998) and *Prunus armeniaca* (U82011) (Mbéguié-A-Mbéguié et al., 1997). In both cases a role in flavonoid biosynthesis was suspected, but no functional evidence was provided. Similar scores were obtained with *Rosa* hybrid orcinol OMTs (e.g. AF502433, AF502434) that are involved in floral scent production (Lavid et al., 2002; Scalliet et al., 2002). Much lower scores were detected with typical COMTs (ca. 35%). Fig. 4 summarizes the overall relationships to other plant OMTs in a tree in which the large number of proteins very closely related to known COMTs was reduced to a few representative examples. CrOMT2 and CrOMT4 formed a distinct side branch in the main cluster containing a heterogeneous group that consisted of functionally identified OMTs (alkaloid, flavonoid, isoflavonoid, phenylpropanoid, orcinol OMTs), but also of examples identified solely by sequence similarity. The overall relations provided no clues with respect to the substrates of the two proteins purified from *C. roseus* cell cultures.

### 2.3. Identification of a flavonoid 3′,5′-O-dimethyltransferase

CrOMT2 and CrOMT4 were recloned for recombinant protein expression in *Escherichia coli*, and the purified proteins (see Fig. 5 for an example) were tested for enzyme activity. We first looked for 16HT-OMT,

![Diagram](attachment:diagram.png)

Fig. 4. Relationship of CrOMT2 and CrOMT4 with other plant OMTs. The numbers at the forks are bootstrap values that indicate the percentage values for obtaining this particular branching in 1000 repetitions of the analysis; only the values above 65% are shown.

early stages of this work, because it was effectively masked by the activity of CrCOMT with flavonoid substrates, and only the last purification step had suggested that the flavonoid OMT activity was an enzyme distinct from COMT (Table 1).

Both CrOMT2 and CrCOMT were active with myricetin, and thus it was an interesting question whether CrOMT2 was simply an unusual COMT. The most important results of an extensive comparison are summarized in Table 3. A remarkable difference was the low specific activity of CrOMT2 when compared to CrCOMT, even with its best substrate myricetin. CrOMT2 had no activity with phenylpropanoids (5-hydroxyferulate, caffeate, and their CoA-esters) which were the best substrates of CrCOMT, and this by itself excluded a COMT function for CrOMT2. The experiments also showed that CrOMT2 was active with dihydroflavonols, but CrCOMT was not (Table 3). Another difference was that CrCOMT accepted flavones and flavanones possessing two or three B-ring hydroxyl groups (Schröder et al., 2002), while CrOMT2 was inactive with these compounds (not shown). The activity of CrOMT2 was indeed strictly confined to flavonols and dihydroflavonols, and it required at least two B-ring hydroxyl groups. The TLC analysis revealed two product peaks from dihydromyricetin, suggesting that CrOMT2 synthesized a mono- and a dimethylated product. This point was investigated in more detail with myricetin, the best substrate for CrOMT2. The mass-spectrometric analysis identified a compound whose fragmentation pattern was identical to that of authentic syringetin (3′,5′-dimethylmyricetin, Table 4), indicating that the enzyme performed two methylations. A monomethylated form was also detected, and it seems most likely that it represented the intermediate to syringetin.

Fig. 5. SDS gel electrophoretic analysis of the purification of recombinant CrOMT2 synthesized in *E. coli*. Lanes: M, marker proteins; 1, crude extract; 2, flow-through of Ni–NTA-column; 3, elution with 0.1 M imidazole; 4, elution with 0.25 M imidazole. The gel was stained with Coomassie Brilliant Blue.

but no such activity could be detected. Incubations with myricetin, however, revealed that recombinant CrOMT2, but not CrOMT4, methylated myricetin. This provided definite evidence that the enzyme purification designed for 16HT-OMT led to extensive copurification of a flavonoid OMT, and that this enzyme actually was one of the two dominant protein species in the preparation. This result was unexpected. The presence of such activity could not even be suspected before in the

Table 3  
Activities of the purified recombinant CrOMT2 with various substrates and comparison with CrCOMT$^a$

| Structures | Compounds | Activity (%) |
|------------|-----------|---------------|
|            | Flavonols |               |
|            | Kaempferol ($R^1 = R^2 = \text{H}$) | $\leq 1$ | $\leq 1$ |
|            | Quercetin ($R^1 = \text{OH}, R^2 = \text{H}$) | $13$ | $5$ |
|            | Myricetin ($R^1 = R^2 = \text{OH}$) | $100$ | $13$ |
|            | Dihydroflavonols |               |
|            | Dihydrokaempferol ($R^1 = R^2 = \text{H}$) | $\leq 1$ | $\leq 1$ |
|            | Dihydroquercetin ($R^1 = \text{OH}, R^2 = \text{H}$) | $22$ | $\leq 1$ |
|            | Dihydromyricetin ($R^1 = R^2 = \text{OH}$) | $54$ | $\leq 1$ |
|            | Caffeate and CoA-ester |               |
|            | Caffeate ($R = \text{OH}$) | $\leq 1$ | $100$ |
|            | Caffeoyl-CoA ($R = \text{CoAS}$) | $\leq 1$ | $31$ |

$^a$ With CrOMT2, the values were normalized to myricetin ($100\% = 0.5 \, \text{nmol/min/mg}$); with CrCOMT they were normalized to caffeate ($100\% = 57.5 \, \text{nmol/min/mg}$).
We also analyzed the products synthesized from myricetin by CrCOMT, and the results indicated mono- and dimethylated products, as obtained from CrOMT2.

The comparison of the enzyme activities showed that CrOMT2 was a flavonoid-specific OMT clearly distinct from COMT. The preference for flavonols suggested that myricetin may be the physiological substrate, in line with the presence of a syringetin derivative in *C. roseus* (Fig. 1B) (Brun et al., 1999). The same B-ring 3′,5′-methylation pattern was found in the anthocyanidins malvidin and hirsutidin in *C. roseus* (Carew and Krueger, 1976; Knobloch et al., 1982; Milo et al., 1985). It seems possible that these methylations were also carried out at the flavonol level, but this needs to be shown by other experiments. Interestingly, the single methyl group in petunidin (Fig. 1B) could be explained by another enzyme carrying out only one methylation, but it is equally likely that it is the result of a partial CrOMT2 reaction. An interesting feature of CrOMT2 is in fact that it carries out two methylations of the same substrate. Multifunctional OMTs (acting on different substrate classes, e.g. phenylpropanoids and flavonoids or alkaloids) are well-known (Gauthier et al., 1998; Frick and Kutchan, 1999; Chiron et al., 2000), but we are not aware of other flavonoid-specific OMTs performing a similar dimethylation of the same substrate. It should be noted, however, that this was rarely investigated, and to our knowledge it is also a new finding that a COMT is capable of transferring two methyl groups to the same substrate. Interestingly, it was recently reported that the orcinol OMTs involved in rose scent biosynthesis carried out two sequential methylations (Lavid et al., 2002; Scalliet et al., 2002). Sequential methylations of the same substrate have been reported for certain *N*-methyltransferases, but these proteins possess an unusual domain structure (Nuccio et al., 2000; Bolognese and McGraw, 2000; Charron et al., 2002).

CrOMT4 was the second dominant protein species in the purified enzyme preparation. The recombinant protein was tested with all the substrates investigated in the comparison of CrCOMT and CrOMT2, but no activity could be detected. The methylation pattern of the anthocyanidin hirsutidin (Fig. 1B) predicted an OMT

specific for the position 7 in the A-ring, and thus we also tested cyanidin, malvidin and delphinidin, but the protein showed no activity with these compounds. It may be possible, however, that these experiments were unsuccessful because the correct substrates were not available. Although not investigated with *C. roseus*, experiments with other plants had indicated that some flavonoid methylations probably occur at the glycoside stage (Jonsson et al., 1982, 1984; Knogge and Weissenböck, 1984). Fig. 6 summarizes the reactions proposed for the methylation pattern found in *C. roseus* flavonoids. It is remarkable that only two OMTs are sufficient to explain the methyl groups in all methylated flavonoids known from *C. roseus*.

### 2.4. Concluding remarks

The failure to identify 16HT-OMT was disappointing. One possible explanation was that the enzyme had a very high specific activity and that consequently the amount was too low to be detected among the bulk of CrOMT2 and CrOMT4. It also seems possible that the minor band at 41 kDa not investigated in detail represented the 16HT-OMT. These points need to be reinvestigated. On the other hand it may be worthwhile to reconsider the basic assumptions underlying the biochemical and the molecular approaches. Both assumed as working hypothesis that the protein belongs to the known family of OMTs, but assumptions do not necessarily reflect reality. Regardless of this, the biochemical and molecular characterization of a new type of flavonoid OMT adds an interesting member to the few flavonoid OMTs cloned up to now (Maxwell et al., 1993; Gregersen et al., 1994; Gauthier et al., 1995, 1996, 1998; Haga et al., 1997; Wu et al., 1997; He et al., 1998).

## 3. Experimental

### 3.1. Plant material

The cell suspension culture of Madagascar periwinkle (*C. roseus* L.G. Don, line CP3a) and its maintenance in MX growth medium in continuous dark with sub-

### Table 4

HPLC data for myricetin and its methylation products, and MS data after trimethylsilylation

| Sample          | Compound                          | RT<sup>a</sup> (min) | EI mass spectra, *m/z* (relative abundance, %) |
|-----------------|-----------------------------------|---------------------|------------------------------------------------|
|                 |                                   |                     | M<sup>+</sup>                                     |
|                 |                                   |                     | [M–Me]<sup>+</sup>                                |
|                 |                                   |                     | Other ions                                        |
| CrOMT2          | Myricetin (substrate)             | 29.58               | 750 (2)                                          | 735 (100)                                         | 647 (27), 575 (6), 395 (2), 147 (10), 73 (87)       |
|                 | Monomethylmyricetin               | 30.50               | 692 (−)                                          | 677 (100)                                         | 647 (17), 589 (12), 575 (3), 393 (2), 147 (6), 73 (75) |
|                 | Dimethylmyricetin                 | 31.16               | 634 (−)                                          | 619 (100)                                         | 589 (12), 547 (8), 517 (2), 302 (3), 295 (5), 73 (32) |
| Syringetin      | 3′,5′-*O*-Dimethylmyricetin        | 31.18               | 634 (−)                                          | 619 (100)                                         | 589 (11), 547 (7), 517 (3), 302 (6), 295 (10), 73 (48) |
| (authentic)     | (syringetin)                      |                     |                                                   |                                                  |

<sup>a</sup> RT = retention time.
gel EMD DEAE M, bed vol. 98 ml) with the column equilibrated with buffer B. Proteins were eluted with a step gradient (0.1 M NaCl steps) in buffer B and tested for enzyme activity. The fraction containing the 16HT-OMT activity (0.2 M NaCl) was concentrated by ammonium sulfate precipitation (80% saturation). The precipitate was dissolved in buffer C (20 mM Tris-HCl, pH 7.8, 1 mM EDTA, 10% glycerol), and applied to adenosine agarose (vol. 3.6 ml) prepared from 5'-AMP agarose by established procedures (Attieh et al., 1995; Rakwal et al., 2000) at a flow rate of 0.3 ml/min. After extensive washing (50 ml each) with buffer C and with 0.2 M KCl in buffer C the bound proteins were eluted with 4 mM SAM dissolved in buffer C containing 0.2 M KCl. They were concentrated by ammonium sulfate precipitation and dissolved in buffer D (50 mM Tris-HCl, pH 8, 10% glycerol), and dialyzed against the same buffer.

Other materials investigated for purification of 16HT-OMT included size fractionation (Fractogel EMD BioSEC(S), Merck), hydroxyapatite (CHT 5I, BioRad), chromatofocusing (MonoP HR 5/20, Pharmacia), hydrophobic interaction chromatography (Fractogel EMD Butyl, Merck), Reactive Red 120, Brown 10, Green 19, Cibacron Blue 3GA (Sigma Biochemicals), and Procion Red He-3B (Amersham Biosciences).

The mass spectrometric analysis of the proteins, the tryptic digests and the microsequencing was carried out by EBL—Environment-und Biotechnologie Laboratorium GmbH, BioTechPark, Freiburg (Germany).

### 3.3. Enzyme assays

The 16HT-OMT activity was determined with a test system in which the OMT substrate 16-hydroxytabersonine was synthesized by recombinant cytochrome P450-dependent tabersonine hydroxylase during the incubations, and the assays were carried out as described (Schröder et al., 1999), except that the glucose 6-phosphate concentration of the NADPH regenerating system was reduced to 1.88 mM. The assays with the various flavonoids, caffeic acid, and caffeoyl-CoA were performed as described (Schröder et al., 2002) with appropriate amounts of the recombinant proteins or plant enzyme preparations. Briefly, the assays contained 50 mM Tris-HCl (pH 7.5), 2 mM DTT, 0.1 mM substrate, 40 μM unlabelled SAM, 9.3 μM S-adenosyl-L-[methyl-¹⁴C]methionine (55,000 dpm, 54 μCi/μmol), and protein in a final volume of 50 μl. The reactions were stopped after 30 or 60 min by acidification (2 μl 1 M HCl), and the EtOAc-extracted products were quantified after TLC separation. The product of catechol methylation was quantified by counting the EtOAc extract in a liquid scintillation spectrometer.

S. Cacace et al. / Phytochemistry 62 (2003) 127–137

### 3.4. Analytical techniques

The detailed identification of myricetin and its methylation products was carried out by GC-EIMS. Both the retention time and the EIMS of the detected dimethylmyricetin after trimethylsilylation were in very good agreement with an authentic sample of syringetin (Table 4). Syringetin (3′,5′-O-dimethylmyricetin; Reference 12.39 S, batch 03) was purchased from Extrasynthese (Genay, France). Table 4 summarizes the results with CrOMT2; the same data were obtained with CrOMT1.

GC-MS details (Voyager, ThermoQuest): 70 eV EI, source temperature 200 °C; column DB-5MS (J&W, 30 m × 0.25 mm, 0.25 μm film thickness), injection temperature 250 °C, interface temperature 300 °C, carrier gas He, flow rate 0.6 ml/min, splitless injection, column temperature program: 60 °C for 1 min, then raised to 110 °C at a rate of 25 °C min⁻¹, isothermally at 110 °C for 1 min, then raised to 300 °C at a rate of 10 °C min⁻¹, isothermally at 300 °C for 25 min. The trimethylsilylation of the flavonoid samples was carried out with N-methyl-N-trimethylsilylfluoroacetamide.

### 3.5. Homology based PCR

The preparation of cDNA and libraries in phage lambda NM1149 followed published methods (Schröder et al., 1999). The PCR reactions for OMT specific sequences were carried out with a degenerate primer (5′ T-[TG]-G-[AC]-I-[CT]-A-T-G-T-T-G-G-[AT]-G-G-I-GAT-A-T-G-T-T-T-G 3′) designed from the motif 3 conserved in OMTs (Ibrahim, 1997; Ibrahim et al., 1998). The PCR reactions were carried out either with phage lysates with phage primers flanking the cDNA inserts, or with a 5′/3′-RACE kit (Roche Diagnostics, Germany). Overlapping clones for the complete coding regions were obtained by 5′ and 3′-RACEs with specific primers designed from the known sequences. The DNAs were sequenced on both strands.

### 3.6. Expression and purification of recombinant proteins

For expression in *E. coli*, the protein coding regions of *CrOMT2* and *CrOMT4* were amplified with 5′-primers providing a *BamHI* site directly before the start codon (*CrOMT2*: 5′ AA **GGA TCC** ATG GAA TTG CAA TCT TCT GAG ATT CG 3′; *CrOMT4*: 5′ AA **GGA TCC** ATG GAC TTG GAA ACT GCT GAG ATT CG 3′), and 3′-primers inserting a *SalI* site after the stop codon (*CrOMT2*: 5′ **AAGTCGAC** TCA AGG ATA TAC TTC AAT AGG AC 3′; *CrOMT4*: 5′ **AAGTCGAC** TT TCA TGG ATA AAC TTC AAT GG 3′). The *BamHI/SalI* fragments were inserted into vector pHis8-3 (Ferrer et al., 1999) which provided the expressed protein with a His-tag at the N-terminal. The proteins were affinity purified with the His-trap™ purification kit from Pharmacia Biotech (Freiburg, Germany) as recommended by the manufacturer. The elution was in most cases with 0.5 M imidazole that was subsequently removed by passing the proteins through a PD10 column (Amersham Biosciences). Fig. 5 shows as example one of the CrOMT2 purifications. The COMT from *C. roseus* and the recombinant protein expression have been reported (Schröder et al., 2002).

### 3.7. Relationship tree

The protein sequences were aligned with CLUSTAL W built into the sequence analysis program OMIGA 2.0 (Oxford Molecular, UK), and the alignment was improved by visual inspection. The tree was developed with the program TREECON for Windows (Van de Peer and De Wachter, 1994), using the matrix for amino acid sequences and the neighbour-joining method. The outgroup for the plant sequences was the *O*-demethylpuromycin-*O*-methyltransferase from *Streptomyces anulatus* (SwissProt DMPM_STRLP).

### Acknowledgements

This work was supported by a grant from the Deutsche Forschungsgemeinschaft to J.S. (Schr 21/5). We thank C. Kuhnt (Halle) for the GC-MS measurements, S. Martens (München) for samples of cyanidin, malvidin, and delphinidin, and L. Nover (Frankfurt) for a gift of HSP70 antiserum.

### References

Attieh, J.M., Hanson, A.D., Saini, H.S., 1995. Purification and characterization of a novel methyltransferase responsible for biosynthesis of halomethanes and methanethiol in *Brassica oleracea*. Journal of Biological Chemistry 270, 9250–9257.

Bolognese, C.P., McGraw, P., 2000. The isolation and characterization in yeast of a gene for *Arabidopsis S*-adenosylmethionine:phospho-ethanolamine *N*-methyltransferase. Plant Physiology 124, 1800–1813.

Brun, G., Dijoux, M.G., David, B., Mariotte, A.M., 1999. A new flavonol glycoside from *Catharanthus roseus*. Phytochemistry 50, 167–169.

Carew, D.P., Krueger, R.J., 1976. Anthocyanidins of *Catharanthus roseus* callus cultures. Phytochemistry 15, 442.

Charron, J.B.F., Breton, G., Danyluk, J., Muzac, I., Ibrahim, R.K., Sarhan, F., 2002. Molecular and biochemical characterization of a cold-regulated phosphoethanolamine *N*-methyltransferase from wheat. Plant Physiology 129, 363–373.

Chiron, H., Drouet, A., Claudot, A.C., Eckerskorn, C., Trost, M., Heller, W., Ernst, D., Sandermann Jr., H., 2000. Molecular cloning and functional expression of a stress-induced multifunctional *O*-methyltransferase with pinosylvilin methyltransferase activity from Scots pine (*Pinus sylvestris* L.). Plant Molecular Biology 44, 733–745.

De Luca, V., Balsevich, J., Tyler, R.T., Eilert, U., Panchuk, B.D., Kurz, W.G.W., 1986. Biosynthesis of indole alkaloids. Develop-

mental regulation of the biosynthetic pathway from tabersonine to vindoline in *Catharanthus roseus* cultivar *Little delicata*. Journal of Plant Physiology 125, 147–156.

Dixon, R.A., 1999. Isoflavonoids: biochemistry, molecular biology, and biological functions. In: Sankawa, U. (Ed.), Polyketides and Other Secondary Metabolites Including Fatty Acids and Their Derivatives, Vol. 1. Elsevier, Amsterdam, pp. 773–823.

Dixon, R.A., Chen, F., Guo, D.J., Parvathi, K., 2001. The biosynthesis of monolignols: a “metabolic grid”, or independent pathways to guaiacyl and syringyl units? Phytochemistry 57, 1069–1084.

Fahn, W., Laussermair, E., Deus-Neumann, B., Stöckigt, J., 1985. Late enzymes of vindoline biosynthesis. S-Adenosyl-L-methionine:11-O-demethyl-17-O-deacetylvin-doline 11-O-methyltransferase and unspecific acylesterase. Plant Cell Reports 4, 337–340.

Ferrer, J.-L., Jez, J.M., Bowman, M.E., Dixon, R.A., Noel, J.P., 1999. Structure of chalcone synthase and the molecular basis of plant polyketide biosynthesis. Nature Structural Biology 6, 775–784.

Forkmann, G., Heller, W., 1999. Biosynthesis of flavonoids. In: Sankawa, U. (Ed.), Polyketides and Other Secondary Metabolites Including Fatty Acids and Their Derivatives, Vol. 1. Elsevier, Amsterdam, pp. 713–748.

Frick, S., Kutchan, T.M., 1999. Molecular cloning and functional expression of O-methyltransferases common to isoquinoline alkaloid and phenylpropanoid biosynthesis. Plant Journal 17, 329–339.

Gang, D.R., Wang, J.H., Dudareva, N., Nam, K.H., Simon, J.E., Lewinsohn, E., Pichersky, E., 2001. An investigation of the storage and biosynthesis of phenylpropenes in sweet basil. Plant Physiology 125, 539–555.

Gauthier, A., Gulick, P.J., Ibrahim, R.K., 1995. A flavonol 3′,5′-O-methyltransferase cDNA clone (GenBank U16794) from *Chrysosplenium*. Plant Physiology 108, 1341.

Gauthier, A., Gulick, P.J., Ibrahim, R.K., 1996. cDNA cloning and characterization of a 3′/5′-O-methyltransferase for partially methylated flavonols from *Chrysosplenium americanum*. Plant Molecular Biology 32, 1163–1169.

Gauthier, A., Gulick, P.J., Ibrahim, R.K., 1998. Characterization of two cDNA clones which encode O-methyltransferases for the methylation of both flavonoid and phenylpropanoid compounds. Archives of Biochemistry and Biophysics 351, 243–249.

Gregersen, P.L., Christensen, A.B., Sommer-Knudsen, J., Collinge, D.B., 1994. A putative O-methyltransferase from barley is induced by fungal pathogens and UV light. Plant Molecular Biology 26, 1797–1806.

Haga, M., Akashi, T., Aoki, T., Ayabe, S.-I., 1997. A cDNA for S-adenosyl-L-methionine: isoliquiritigenin/licodione 2′-O-methyltransferase (Accession No. D88742) from cultured licorice (*Glycyrrhiza echinata* L.) cells. Plant Physiology 113, 663.

Harborne, J.B., Williams, C.A., 2000. Advances in flavonoid research since 1992. Phytochemistry 55, 481–504.

He, X.Z., Reddy, J.T., Dixon, R.A., 1998. Stress responses in alfalfa (*Medicago sativa* L). XXII. cDNA cloning and characterization of an elicitor-inducible isoflavone 7-O-methyltransferase. Plant Molecular Biology 36, 43–54.

Ibrahim, R.K., 1997. Plant O-methyltransferase signatures. Trends in Plant Sciences 2, 249–250.

Ibrahim, R.K., Bruneau, A., Bantignies, B., 1998. Plant O-methyltransferases: molecular analysis, common signature and classification. Plant Molecular Biology 36, 1–10.

Ibrahim, R.K., Muzac, I., 2000. The methyltransferase gene superfamily: a tree with multiple branches. In: Romeo, J.T., Ibrahim, R.K., Varin, L., De Luca, V. (Eds.), Evolution of Metabolic Pathways, Vol. 34. Pergamon, pp. 349–384.

Jonsson, L.M.V., Aarsman, M.E.G., Poulton, J.E., Schram, A.W., 1984. Properties and genetic control of four methyltransferases involved in methylation of anthocyanins in flowers of *Petunia hybrida*. Planta 160, 174–179.

Jonsson, L.M.V., Aarsman, M.E.G., Schram, A.W., Bennink, G.J.H.,

1982. Methylation of anthocyanins by cell-free extracts of flower buds of *Petunia hybrida*. Phytochemistry 21, 2457–2459.

Kaltenbach, M., Schröder, G., Schmelzer, E., Lutz, V., Schröder, J., 1999. Flavonoid hydroxylase from *Catharanthus roseus*: cDNA, heterologous expression, enzyme properties, and cell-type specific expression in plants. Plant Journal 19, 183–193.

Knobloch, K.-H., Bast, G., Berlin, J., 1982. Medium- and light-induced formation of serpentine and anthocyanins in cell suspension cultures of *Catharanthus roseus*. Phytochemistry 21, 591–594.

Knogge, W., Weissenböck, G., 1984. Purification, characterization, and kinetic mechanism of S-adenosyl-L-methionine:vitexin 2″-O-rhamnoside 7-O-methyltransferase of *Avena sativa* L. European Journal of Biochemistry 140, 113–118.

Lavid, N., Wang, J., Shalit, M., Guterman, I., Bar, E., Beuerle, T., Menda, N., Shafir, S., Zamir, D., Adam, Z., Vainstein, A., Weiss, D., Pichersky, E., Lewinsohn, E., 2002. O-Methyltransferases involved in the biosynthesis of volatile phenolic derivatives in rose petals. Plant Physiology 129, 1899–1907.

Maxwell, C.A., Harrison, M.J., Dixon, R.A., 1993. Molecular characterization and expression of alfalfa isoliquiritigenin 2′-O-methyltransferase, an enzyme specifically involved in the biosynthesis of an inducer of *Rhizobium meliloti* nodulation genes. Plant Journal 4, 971–981.

Mbéguié-A-Mbéguié, D., Gomez, R.M., Fils-Lycaon, B., 1997. Sequence of an O-methyltransferase from Apricot fruit (Accession No. U82011). Gene expression during fruit ripening. Plant Physiology 114, 1569.

Milo, J., Levy, A., Akavia, N., Ashri, A., Palevitch, D., 1985. Inheritance of corolla color and anthocyanin pigments in periwinkle (*Catharanthus roseus* [L.] G. Don). Zeitschrift für Pflanzenzüchtung 95, 352–360.

Morishige, T., Tsujita, T., Yamada, Y., Sato, F., 2000. Molecular characterization of the S-adenosyl-L-methionine:3′-hydroxy-N-methylcoclaurine 4′-O-methyltransferase involved in isoquinoline alkaloid biosynthesis in *Coptis japonica*. Journal of Biological Chemistry 275, 23398–23405.

Muzac, I., Wang, J., Auzellotti, D., Zhang, H., Ibrahim, R.K., 2000. Functional expression of an *Arabidopsis* cDNA clone encoding a flavonol 3′-O-methyltransferase and characterization of the gene product. Archives of Biochemistry and Biophysics 375, 385–388.

Neumann, D., Zur Nieden, U., Manteuffel, R., Walter, G., Scharf, K.-D., Nover, L., 1987. Intracellular localization of heat-shock proteins in tomato cell cultures. European Journal of Cell Biology 43, 71–81.

Nuccio, M.L., Ziemak, M.J., Henry, S.A., Weretilnyk, E.A., Hanson, A.D., 2000. cDNA cloning of phosphoethanolamine N-methyltransferase from spinach by complementation in *Schizosaccharomyces pombe* and characterization of the recombinant enzyme. Journal of Biological Chemistry 275, 14095–14101.

Rakwal, R., Agrawal, G.K., Yonekura, M., Kodama, O., 2000. Nar-ingienin 7-O-methyltransferase involved in the biosynthesis of the flavanone phytoalexin sakuranetin from rice (*Oryza sativa* L.). Plant Science 155, 213–221.

Scalliet, G., Journot, N., Jullien, F., Baudino, S., Magnard, J.-L., Channelière, S., Vergne, P., Dumas, C., Bendahmane, M., Cock, J.M., Hugueney, P., 2002. Biosynthesis of the major scent components 3,5-dimethoxytoluene and 1,3,5-trimethoxybenzene by novel rose O-methyltransferases. FEBS Letters 523, 113–118.

Schmitt, D., Pakusch, A.-E., Matern, U., 1991. Molecular cloning, induction, and taxonomic distribution of caffeoyl-CoA 3-O-methyltransferase, an enzyme involved in disease resistance. Journal of Biological Chemistry 266, 17416–17423.

Schröder, G., Unterbusch, E., Kaltenbach, M., Schmidt, J., Strack, D., De Luca, V., Schröder, J., 1999. Light-induced cytochrome P450-dependent enzyme in indole alkaloid biosynthesis: tabersonine 16-hydroxylase. FEBS Letters 458, 97–102.

Schröder, G., Wehinger, E., Schröder, J., 2002. Predicting the substrates of cloned plant O-methyltransferases. Phytochemistry 59, 1–8.

Seguin, J., Muzac, I., Ibrahim, R.K., 1998. Purification and immunological characterization of a recombinant trimethylflavonol 3′-O-methyltransferase. Phytochemistry 49, 319–325.

St-Pierre, B., De Luca, V., 1995. A cytochrome P-450 monooxygenase catalyzes the first step in the conversion of tabersonine to vindoline in *Catharanthus roseus*. Plant Physiology 109, 131–139.

Suelves, M., Puigdomènech, P., 1998. Specific mRNA accumulation of a gene coding for an O-methyltransferase in almond (*Prunus amygdalus*, Batsch) flower tissues. Plant Science 134, 79–88.

Van de Peer, Y., De Wachter, R., 1994. TREECON for Windows: a software package for the construction and drawing of evolutionary trees for the Microsoft Windows environment. Computational Applications in Biosciences 10, 569–570.

Vetter, H.-P., Mangold, U., Schröder, G., Marner, F.-J., Werck-Reichhart, D., Schröder, J., 1992. Molecular analysis and heterologous expression of an inducible cytochrome P-450 protein from periwinkle (*Catharanthus roseus* L.). Plant Physiology 100, 998–1007.

Wang, J.H., Pichersky, E., 1999. Identification of specific residues involved in substrate discrimination in two plant O-methyltransferases. Archives of Biochemistry and Biophysics 368, 172–180.

Wu, Q., Preisig, C.L., VanEtten, H.D., 1997. Isolation of the cDNAs encoding (+)-6a-hydroxymaackiain 3-O-methyltransferase, the terminal step for the synthesis of the phytoalexin pisatin in *Pisum sativum*. Plant Molecular Biology 35, 551–560.
