
# Invited Review

## Bacterial degradation of aromatic compounds via angular dioxygenation

Hideaki Nojiri, Hiroshi Habe, and Toshio Omori*

Biotechnology Research Center, The University of Tokyo,  
Bunkyo-ku, Tokyo 113–8657, Japan

(Received September 4, 2001; Accepted September 26, 2001)

Dioxygenation is one of the important initial reactions of the bacterial degradation of various aromatic compounds. Aromatic compounds, such as biphenyl, toluene, and naphthalene, are dioxygenated at lateral positions of the aromatic ring resulting in the formation of *cis*-dihydrodiol. This “normal” type of dioxygenation is termed lateral dioxygenation. On the other hand, the analysis of the bacterial degradation of fluorene (FN) analogues, such as 9-fluorenone, dibenzofuran (DF), carbazole (CAR), and dibenzothiophene (DBT)-sulfone, and DF-related diaryl ether compounds, dibenzo-*p*-dioxin (DD) and diphenyl ether (DE), revealed the presence of the novel mode of dioxygenation reaction for aromatic nucleus, generally termed angular dioxygenation. In this atypical dioxygenation, the carbon bonded to the carbonyl group in 9-fluorenone or to heteroatoms in the other compounds, and the adjacent carbon in the aromatic ring are both oxidized. Angular dioxygenation of DF, CAR, DBT-sulfone, DD, and DE produces the chemically unstable hemiacetal-like intermediates, which are spontaneously converted to 2,2′,3-trihydroxybiphenyl, 2′-aminobiphenyl-2,3-diol, 2′,3′-dihydroxybiphenyl-2-sulfinate, 2,2′,3-trihydroxydiphenyl ether, and phenol and catechol, respectively. Thus, angular dioxygenation for these compounds results in the cleavage of the three-ring structure or DE structure. The angular dioxygenation product of 9-fluorenone, 1-hydro-1,1a-dihydroxy-9-fluorenone is a chemically stable *cis*-diol, and is enzymatically transformed to 2′-carboxy-2,3-dihydroxybiphenyl. 2′-Substituted 2,3-dihydroxybiphenyls formed by angular dioxygenation of FN analogues are degraded to monocyclic aromatic compounds by *meta* cleavage and hydrolysis. Thus, after the novel angular dioxygenation, subsequent degradation pathways are homologous to the corresponding part of that of biphenyl. Compared to the bacterial strains capable of catalyzing lateral dioxygenation, few bacteria having angular dioxygenase have been reported. Only a few degradation pathways, CAR-degradation pathway of *Pseudomonas resinovorans* strain CA10, DF/DD-degradation pathway of *Sphingomonas wittichii* strain RW1, DF/DD/FN-degradation pathway of *Terrabacter* sp. strain DBF63, and carboxylated DE-degradation pathway of *P. pseudoalcaligenes* strain POB310, have been investigated at the gene level. As a result of the phylogenetic analysis and the comparison of substrate specificity of angular dioxygenase, it is suggested that this atypical mode of dioxygenation is one of the oxygenation reactions originating from the relaxed substrate specificity of the Rieske nonheme iron oxygenase superfamily. Genetic characterization of the degradation pathways of these compounds suggests the possibility that the respective genetic elements constituting the entire catabolic pathway have been recruited from various other bacteria and/or other genetic loci, and that these pathways have not evolutionary matured.

---

* Address reprint requests to: Dr. Toshio Omori, Biotechnology Research Center, The University of Tokyo, 1–1–1 Yayoi, Bunkyo-ku, Tokyo 113–8657, Japan.  
E-mail: aseigyo@mail.ecc.u-tokyo.ac.jp

Key Words—angular dioxygenase; carbazole; dibenzo-p-dioxin; dibenzofuran; dibenzothiophene; diphenyl ether; fluorene; microbial degradation

---

### Introduction

Soil microorganisms are able to metabolize an enormous range of natural and synthetic organic compounds, and are important agents in the degradation and recycling of organic materials (Gibson, 1988). The initial step in the aerobic microbial degradation of aromatic compounds is usually the introduction of two hydroxyl groups into the benzene ring, forming *cis*-dihydrodiols. The enzymes catalyzing the incorporation of both atoms of dioxygen into aromatic substrates are called dioxygenase. The degradation pathways for polycyclic aromatic hydrocarbons (PAHs), such as naphthalene and phenanthrene, and biphenyl have been intensively analyzed as reviewed by Cerniglia (1992) and Furukawa (2000). As the initial degradation reaction, naphthalene and biphenyl are converted to *cis*-1,2-dihydro-1,2-dihydroxynaphthalene and *cis*-2,3-dihydro-2,3-dihydroxybiphenyl, respectively, as shown in Fig. 1A. This type of dioxygenation is termed the lateral dioxygenation or *cis*-dihydroxylation. *Cis*-Dihydrodiols are stereoselectively dehydrogenated by *cis*-dihydrodiol dehydrogenase, which rearomatize the benzene nucleus to form a catechol moiety. Because the fission of the aromatic ring occur at the catechol moiety in a *meta* or *ortho* fashion, the formation of the catechol moiety via lateral dioxygenation followed by dehydrogenation is a critical point in the degradation pathways of various aromatic compounds.

On the other hand, microbial degradation of fluorene (FN) and its heteroatomic analogues, dibenzofuran (DF), carbazole (CAR), and dibenzothiophene (DBT) were investigated, because these compounds are components of creosote, crude oils, and shale oils (Later et al., 1981). In addition, due to the environmental significance of their chlorinated derivatives, the metabolism of DF and dibenzo-*p*-dioxin (DD) has been studied intensively in the last decade. The microbial degradation of diaryl ether compounds, such as diphenyl ether (DE) and its 3- or 4-carboxylated derivatives, has also been investigated. Figure 2 illustrates the chemical structure and numbering system of the above-mentioned compounds. The elucidation of the bacterial degradation pathways of these compounds

---

revealed a new type of oxidative attack with high regioselectivity and specificity for the angular position (Fig. 1B). In this dioxygenation, the carbon atom bonded to the carbonyl group in 9-fluorenone or to heteroatoms in the analogues, and the adjacent carbon atom in the aromatic ring are both oxidized. This reaction, termed angular dioxygenation, is catalyzed by the Rieske nonheme iron oxygenases, termed angular dioxygenases.

This review summarizes the degradation pathways of DF, DD, FN, DE, CAR, and DBT via angular dioxygenation. In addition, this review shows the present situations of biochemical and genetic studies on their degradation pathways.

---

**Figure 1. Lateral dioxygenation for naphthalene and biphenyl (A) and angular dioxygenation for dibenzofuran (DF) and carbazole (CAR) (B).**

Products of angular dioxygenation shown in brackets are unstable, and have not been observed directly.

Aromatics degradation with angular attack

Dibenzofuran (DF)
![Structure of Dibenzofuran](https://i.imgur.com/structure1.png)

Dibenzo-$p$-dioxin (DD)
![Structure of Dibenzo-p-dioxin](https://i.imgur.com/structure2.png)

Diphenyl ether (DE)
![Structure of Diphenyl ether](https://i.imgur.com/structure3.png)

Carbazole (CAR)
![Structure of Carbazole](https://i.imgur.com/structure4.png)

Fluorene (FN)
![Structure of Fluorene](https://i.imgur.com/structure5.png)

Dibenzothiophene (DBT)
![Structure of Dibenzothiophene](https://i.imgur.com/structure6.png)

Fig. 2. Chemical structure of fluorene (FN) analogues and dibenzofuran (DF)-related diaryl ether compounds, and their numbering conventions.

---

### 1. Degradation Pathways Containing Angular Dioxygenation

Angular dioxygenation has been found to be involved in the degradation pathways for DF, DD, FN, DE, carboxylated DE, CAR, and DBT. Of course, other biodegradation pathways for these compounds via lateral dioxygenation or monooxygenation have been well investigated and reviewed (Bressler and Fedorak, 2000; Ohshiro and Izumi, 1999; Wittich, 1998a, b). For instance, the lateral dioxygenation at the peripheral aromatic ring of FN and its structural analogues, followed by a *meta* cleavage analogous to the naphthalene metabolism, lead to the accumulation of substituted 2-oxo-3-butenonic acids (Casellas et al., 1997; Grifoll et al., 1992; Kodama et al., 1973).

Table 1 shows a list of bacterial strains having the capacity to catalyze angular dioxygenation.

#### 1.1. Microbial degradation of DF, DD, and FN

Polychlorinated DDs and DFs, unintentionally formed as contaminating by-products during the manufacture of pesticides, incineration of industrial and domestic wastes, and bleaching of paper pulp, have thus become widespread contaminants of the environment. The fact that most of these compounds are present in the environment at low but toxicologically relevant concentrations renders the use of thermic or chemical remediation as well as removal and storage procedures inappropriate. Because of this, attention has tended to be focused on the biodegradation of these xenobiotics. Over the last decade, a number of bacteria capable of mineralizing DF and/or DD have been isolated by enrichment culture methods using the carbon-free mineral medium supplemented with DF or DD as a sole source of carbon and energy, as listed in Table 1.

Engesser et al. (1989) and Fortnagel et al. (1989) established that the initial angular attack could occur on DF at the 4 and 4a carbon atoms (Fig. 3). This angular dioxygenation for DF produces highly unstable hemiacetal products, which have not been observed directly. Analysis of the chemical structure of dead-end product accumulated in the reaction mixture of the analogous compound of DF, 9-fluorenone, incubated with DF-grown bacteria by mass spectroscopy (MS) and nuclear magnetic resonance (NMR) spectroscopy, revealed 1-hydro-1,1a-dihydroxy-9-fluorenone to be the resultant product (the chemical structure is shown in Fig. 4; Engesser et al., 1989; Fortnagel et al., 1989). Thus, formation of hemiacetals from DF/DD was inferred. In addition, Wilkes et al. (1992) revealed that dioxygenase uses molecular oxygen during substrate transformation based on the identification of $^{18}$O-labeled hydroxyl groups of products transformed in an atmosphere of $^{18}$O₂.

Although the formation of trihydroxyl compound has not been observed in the culture supernatant of various DF-degrading bacteria, Fortnagel et al. (1990) reported that the UV-derived mutant strain, which could no longer grow with DF as a carbon source (designated strain HH69-II), accumulates 2,2′,3-trihydroxybiphenyl as a metabolic intermediate of DF (Fig. 3). 2,2′,3-Trihydroxybiphenyl is a compound analogous to 2,3-dihydroxybiphenyl, which is a metabolic intermediate of biphenyl, and is converted to benzoic acid by

| Bacterium | Initial dioxygenase | Substrate $a$ | Reference |
| --- | --- | --- | --- |
| Brevibacterium sp. strain DPO 220 | DF 4, 4a-dioxygenase | DF, FN | Engesser et al., 1989; Strubel et al., 1989 |
| Terrabacter sp. strain DPO 1361 | DF 4, 4a-dioxygenase | DF, FN | Engesser et al., 1989; Schmid et al., 1997; Strubel et al., 1989; Trenz et al., 1994 |
| Terrabacter sp. strain DPO 360 | DF 4, 4a-dioxygenase | DF | Schmid et al., 1997 |
| Sphingomonas sp. strain HH69 | DF 4, 4a-dioxygenase | DF, DD | Fortnagel et al., 1990; Harms et al., 1990, 1991, 1995; Schreiner et al., 1997 |
| Sphingomonas wittichii strain RW1 | DF 4, 4a-dioxygenase (Dioxin dioxygenase) | DF, DD, 9-Fluorenone | Armengaud et al., 1998; Armengaud and Timmis, 1997, 1998; Bünz et al., 1993; Keim et al., 1993; Halden et al., 1999; Megharaj et al., 1997; Schläfli et al., 1995; Wilkes et al., 1996; Wittich et al., 1992; Yabuuchi et al., 2001 |
| Terrabacter sp. strain DBF 63 | DF 4, 4a-dioxygenase | DF, FN | Habe et al., 2001; Kasuga et al., 1997, 2001; Monna et al., 1993 |
| Sphingomonas sp. strain HH19k | DF 4, 4a-dioxygenase | DF | Harms and Zehnder, 1994, 1995 |
| Sphingomonas sp. strain RW16 | DF 4, 4a-dioxygenase | DF | Wittich et al., 1999 |
| Pseudomonas mendocina strain F274 | 9-Fluorenone 1,1a-dioxygenase | FN, DF | Grifoll et al., 1994, 1995; Selifonov et al., 1993 |
| Sphingomonas sp. strain SS3 | DE 1,2-dioxygenase | 9-Fluorenone | Casellas et al., 1998 |
| Pseudomonas sp. strain NSS2 | Phenoxybenzoic acid 1,6-dioxygenase | DE | Schmidt et al., 1992a,b, 1993 |
| Pseudomonas pseudoalcaligenes strain POB 310 | Phenoxybenzoic acid 1,6-dioxygenase | 3-Carboxy DE | Wittich et al., 1990 |
| Ralstonia sp. strain RJGII.123 | CAR 1, 9a-dioxygenase | 3-or 4-Carboxy DE | Dehmel et al., 1995; Engesser et al., 1990; Halden et al., 2000 |
| Pseudomonas sp. strain CA06 | CAR 1, 9a-dioxygenase | CAR | Grosser et al., 1991; Schneider et al., 2000 |
| Pseudomonas resinovorans strain CA10 | CAR 1, 9a-dioxygenase | CAR | Ouchiyama et al., 1993 |
| Pseudomonas stutzeri strain ATCC 31258 | CAR 1, 9a-dioxygenase | CAR, DF, DD | Habe et al., 2001a,b; Kimura et al., 1996; Nam et al., 2001; Nojiri et al., 1999, 2001 (in press); Ouchiyama et al., 1993; Sato et al., 1997a,b |
| Pseudomonas sp. strain LD2 | CAR 1, 9a-dioxygenase | CAR | Hisatsuka and Sato, 1994 |
| Pseudomonas stutzeri strain OM1 | CAR 1, 9a-dioxygenase | CAR | Gieg et al., 1996 |
| Sphingomonas sp. strain CDH-7 | CAR 1, 9a-dioxygenase | CAR | Ouchiyama et al., 1998 |
| Brevibacterium sp. strain DO | DBT sulfone 4, 4a-dioxygenase | DBT sulfone, 9-Fluorenone | Kimura et al., 1999 |
| Arthrobacter sp. strain DBTS2 (strain K3b) | DBT sulfone 4, 4a-dioxygenase | 9-Fluorenone | van Afferden et al., 1988, 1990, 1993 Dahlberg et al., 1993; Ochman et al., 1990; Sato and Clark, 1995 |

$a$ Only the compounds found to be metabolized via angular dioxygenation are shown. Substituted derivatives of the parental compounds shown in Fig. 2 are not shown except for 9-fluorenone, 3-or 4-carboxy DE, and DBT-sulfone.

Aromatics degradation with angular attack

Dibenzofuran
![Chemical Structure](chemical_structure_1.png)

Dibenzo-$p$-dioxin
![Chemical Structure](chemical_structure_2.png)

Angular dioxygenation

Spontaneous cleavage

ortho Cleavage

2,2',3-Trihydroxybiphenyl
![Chemical Structure](chemical_structure_3.png)

2,2',3-Trihydroxydiphenyl ether
![Chemical Structure](chemical_structure_4.png)

2-Hydroxyacetophenone
![Chemical Structure](chemical_structure_5.png)

meta Cleavage

Hydrolysis

Nucleophilic attack of the hydroxyl oxygen on the ester carbon

3-(Chroman-4-on-2-yl)pyruvic acid
![Chemical Structure](chemical_structure_6.png)

Salicylic acid
![Chemical Structure](chemical_structure_7.png)

2-Hydroxypenta-2,4-dienoic acid
![Chemical Structure](chemical_structure_8.png)

Catechol
![Chemical Structure](chemical_structure_9.png)

2-Hydroxymuconic acid
![Chemical Structure](chemical_structure_10.png)

Catechol
![Chemical Structure](chemical_structure_11.png)

2-Pyrone-6-carboxylic acid
![Chemical Structure](chemical_structure_12.png)

Gentisic acid
![Chemical Structure](chemical_structure_13.png)

Catechol
![Chemical Structure](chemical_structure_14.png)

TCA cycle

Fig. 3. Overview of bacterial degradative pathways for dibenzofuran (DF) and dibenzo-$p$-dioxin (DD) via angular dioxygenation.

The structures shown in brackets have not been characterized. The production of catechol and 2-pyrone-6-carboxylic acid, which are shown in open box, from meta cleavage compound of 2,2',3-trihydroxydiphenyl ether was proposed based on the results reported by Pfeifer et al. (1993). The arrows with solid and broken lines indicate the enzymatic and spontaneous conversions, respectively.

meta cleavage and hydrolysis by various biphenyl-degrading bacteria. From the culture supernatant of DF-utilizing bacteria, salicylic acid that corresponds to the benzoic acid in the biphenyl degradation pathway, was identified (Fortnagel et al., 1990; Monna et al., 1993; Strubel et al., 1989). This indicates that 2,2',3-trihydroxybiphenyl is meta cleaved, and the resultant meta cleavage compound is hydrolyzed to salicylic acid and 2-hydroxypenta-2,4-dienoic acid (Bünz et al., 1993; Strubel et al., 1991). In the DF-degradation pathway reported in *Pseudomonas* sp. strain HH69 (Fortnagel et al., 1990), which was reassigned to the genus Sphingomonas (Harms et al., 1995), *S. wittichii* strain RW1 (Wittich et al., 1992), and *Staphylococcus auriculans* strain DBF63 (Monna et al., 1993), which was re-assigned to the genus Terrabacter (Kasuga et al., 1997), formed salicylic acid was branched into catechol and gentisic acid degradation pathways. After the meta cleavage of 2,2',3-trihydroxybiphenyl, a spontaneous chemical side reaction occurs by the Michael-type addition of a phenolic hydroxyl group of the meta cleavage compound of 2,2',3-trihydroxybiphenyl, 2-hydroxy-6-(2-hydroxyphenyl)-6-oxo-2,4-dienoic acid, to the double bond of the aliphatic side chain, yielding a

Nojiri, Habe, and Omori

Fluorene
↓
9-Fluorenol

9-Fluorenone
↓
1-Hydro-1,1a-dihydroxy-9-fluorenone

Dehydrogenation

8-Hydroxy-3,4-benzocoumarin
↔
2'-Carboxy-2,3-dihydroxybiphenyl

meta Cleavage

Hydrolysis

Phthalic acid
+
2-Hydroxypenta-2,4-
dienoic acid

Protocatechuic acid

TCA cycle

Fig. 4. Overview of bacterial degradative pathways for fluorene (FN) via angular dioxygenation.

The structures shown in brackets have not been characterized. The arrows with solid and broken lines indicate the enzymatic and spontaneous conversions, respectively.

3-(chroman-4-on-yl)pyruvic acid (Kohler et al., 1993; Strubel et al., 1991) (Fig. 3). This chroman-4-one-related product and its derivatives have been observed in the culture supernatant and reaction mixture of a transformation experiment using several DF-degrading bacteria (Fortnagel et al., 1990; Strubel et al., 1991; Wilkes et al., 1996). 2-Hydroxyacetophenone has also been identified in the culture supernatant of DF-grown Sphingomonas sp. strain HH69 (Fortnagel et al., 1990)

(Fig. 3). The occurrence of acetophenone in microbial degradation of PCBs is well documented. Seah et al. (2000) showed a satisfactory mechanism for their nonenzymatic formation from the meta cleavage compound of 2,3-dihydroxybiphenyl. The formation of 2-hydroxyacetophenone from DF is proposed in accordance with the mechanism suggested by Seah et al. (2000).

As a result of initial angular dioxygenation follow-

ed by spontaneous ring cleavage, 2,2′,3-trihydroxy-
diphenyl ether is formed from DD (Wittich et al., 1992)
(Fig. 3). Catechol has also been identified by gas chro-
matography (GC)-MS analysis as a metabolite of DD.
Because the meta cleavage activity for 2,3-dihydroxy-
diphenyl ether could be detected, Wittich and cowork-
ers proposed that catechol was formed from 2,2′,3-tri-
hydroxydiphenyl ether via meta cleavage and subse-
quent hydrolysis (Wittich et al., 1992). However, the
possible product, 2-hydroxymuconic acid, of hydrolysis
of the meta cleavage compound has not been identi-
fied as a metabolite of DD. Pfeifer et al. (1993) re-
ported that the meta cleavage of 2,3-dihydroxy-
diphenyl ether by 2,3-dihydroxybiphenyl 1,2-dioxygen-
ase leads to the formation of phenol and 2-pyrone-6-
carboxylate as products of ring fission and ether cleav-
age without participation of free intermediates. Thus, it
is likely that only meta cleavage enzyme is necessary
for the production of catechol from 2,3-dihydroxy-
diphenyl ether (Fig. 3). Similar to the catechol metabo-
lism in the DF-degradation pathway, catechol formed
from DD was mineralized through meta and/or ortho
cleavage pathways. In addition to the above-men-
tioned DD-degradation pathway via meta cleavage,
unproductive transformation to the dead-end product is
the fortuitous ortho cleavage of 2,2′,3-trihydroxy-
diphenyl ether, resulting in 2-(2-hydroxyphenoxy)mu-
conic acid (Harms et al., 1990) (Fig. 3).

FN is an abundant constituent of coal tar and cre-
osote and one of the simplest PAHs, and is one of the
16 PAHs on the list of priority pollutants compiled by
the U.S. Environmental Protection Agency (Kieth and
Teillard, 1979). FN is highly toxic to fish, aquatic algae,
and zooplankton (Hsieh et al., 1980; Stay et al., 1988;
Thomas et al., 1981) and 9-fluorenone has been
shown to be toxic in several bioassays (Richardson
and Gangolli, 1994). Both FN and 9-fluorenone have
carcinogenic (Danz et al., 1978) and mutagenic poten-
tial (Vasilieva et al., 1990).

FN-utilizing bacteria, *Pseudomonas* sp. strain F274
(Grifoll et al., 1994), and 9-fluorenone-utilizing bacte-
ria, *Pseudomonas mendocina* strain MC2 (Casellas et
al., 1998), were isolated from soil severely contami-
nated by creosote and PAHs, respectively. Some DF-
utilizing strains also grow on FN (Monna et al., 1993;
Trenz et al., 1994).

Based on the identification of 9-fluorenel and 9-fluo-
renone as metabolic intermediates of FN, monooxy-
genation at C9 to give 9-fluorenel followed by the de-

hydrogenation to 9-fluorenone was shown to be the
initial conversion before angular dioxygenation of aro-
matic ring (Grifoll et al., 1994; Monna et al., 1993;
Trenz et al., 1994) (Fig. 4). Because 1-hydro-1,1a-di-
hydroxy-9-fluorenone was identified as the degrada-
tion product of FN or 9-fluorenone, angular dioxygena-
tion has been established as the crucial step in FN
degradation (Casellas et al., 1998; Engesser et al.,
1989; Selifonov et al., 1993) (Fig. 4). Other than in the
case of the hemiacetal intermediates produced by an-
gular dioxygenation of DF and DD, the angular dioxy-
genation product of 9-fluorenone is chemically stable.
Dehydrogenation of 1-sec-hydroxyl group of 1-hydro-
1,1a-dihydroxy-9-fluorenone would yield a β-diketone-
type compound whose hydrolysis at the C9-C1a bond
would give 2′-carboxy-2,3-dihydroxybiphenyl. While 8-
hydroxy-3,4-benzocoumarin was identified as a
metabolite of FN, this compound is in fact the δ-lac-
tone of 2′-carboxy-2,3-dihydroxybiphenyl and could be
formed as a result of a reversible dehydration reaction
similar to that described for 8-hydroxy-naphthalene-1-
acetic acid (Danforth et al., 1976). meta Cleavage of
2′-carboxy-2,3-dihydroxybiphenyl at C1-C2 bond and
subsequent hydrolysis result in the formation of
phthalic acid and 2-hydroxypenta-2,4-dienoic acid as
shown in Fig. 4. Phthalic acid is further degraded by
*Pseudomonas* sp. strain F274 via protocatechuic acid.

### 1.2. Microbial degradation of DE and 3- or 4-carboxy
DE

Substituted DEs constitute an important class of pre-
and postemergent herbicides. Industrially, DEs are
finding increased uses as components of heat transfer
media, as solvents, and in plasticizers (Francis, 1986).
Polychlorinated DEs are manufactured for use as
flame retardants and herbicides (Becker et al., 1991).
Due to the marked toxicity of several congeners of the
halogenated diaryl ethers and their general resistance
to degradation, because of their physical, chemical,
and biological stabilities (Hoffman et al., 1991), the
study of mechanisms whereby such compounds may
be degraded are of interest. Bacterial degradation of
DE by *Pseudomonas* species has been shown to pro-
ceed by lateral 2,3-dioxygenation; the unproductive
ortho cleavage of the respective diol yielded 2-phe-
noxymuconic acid (Takase et al., 1986). meta Cleav-
age of 2,3-dihydroxydiphenyl ether and subsequent in-
tramolecular transesterification of 2-hydroxy-6-phe-
noxymuconic acid semialdehyde yielded the dead-


Fig. 5. Overview of bacterial degradative pathways for diphenyl ether (DE) and carboxylated DE via angular dioxygenation.

The structures shown in brackets have not been characterized. The arrows with solid and broken lines indicate the enzymatic and spontaneous conversions, respectively.

end product 2-pyrone-6-carboxylic acid, as described above (Pfeifer et al., 1993). In both cases, solely phenol is utilized for growth.

On the other hand, 3- or 4-carboxy DE-degrading bacteria, *Pseudomonas* sp. strain NSS2 and *P. pseudoalcaligenes* strain POB310, were isolated from soil samples as a microorganism capable of growing on the 3- or 4-carboxy DE as a sole source of carbon and energy (Engesser et al., 1990; Wittich et al., 1990). Accumulation of phenol and protocatechuic acid has been detected in the culture supernatant of strain NSS2 (Wittich et al., 1990). In addition, Engesser et al. (1990) reported that the resting cells of strain POB310 co-oxidize a DE analogue, 4-carboxybenzophenone, yielding its angular diol, 1,2-dihydroxy-2-hydro-4-carboxybenzophenone. The degradation pathway of 3- or 4-carboxy DE proposed based on these findings is shown in Fig. 5. The initial 1,6-dioxygenation of 3- and 4-carboxy DE yields an unstable hemiacetal which, in the course of chemical rearrangement, spontaneously generates phenol and protocat-

echuic acid. The phenol formed is converted to catechol by monooxygenation, and then degraded via meta and ortho cleavage in strains POB310 and NSS2, respectively (Engesser et al., 1990; Wittich et al., 1990). Protocatechuic acid is degraded via ortho cleavage.

DE-degrading bacterium, *Sphingomonas* sp. strain SS3, was isolated from a soil sample by an enrichment culture using 4-fluoro DE as a sole source of carbon and energy (Schmidt et al., 1992a). During DE degradation in batch culture experiments, phenol and catechol were identified as metabolic intermediates of DE, indicating that strain SS3 attacks at angular (C1) and adjacent (C2) positions as shown in Fig. 5. When strain SS3 was grown on DE, ortho cleavage enzyme (catechol 1,2-dioxygenase) and phenol hydroxylase were induced, although meta cleavage enzyme (catechol 2,3-dioxygenase) was not induced. In addition, 4-fluorocatechol was shown to be converted to muconolactone (4-carboxymethylenebut-2-en-4-olide). These results indicate that phenol is converted to catechol,

and that catechol is degraded via ortho cleavage pathway in strain SS3. Considering that DD was oxidized by DE-grown strain SS3 cells to 2-(2-hydroxyphenoxymuconic acid upon ortho cleavage of the intermediate, 2,2′,3-trihydroxydiphenyl ether, it is possible that DE-degrading enzymes degrade DD (Schmidt et al., 1992a), although strain SS3 might have another enzyme(s) for DD degradation.

### 1.3. Microbial degradation of CAR

CAR is a nitrogen heterocycle released into the environment with fossil fuels or their products such as creosote (Santodonato and Howard, 1981). While CAR is used as a chemical feedstock for the production of dyes, medicines, and plastics, CAR is known to possess mutagenic and toxic activities (Arcos and Argus, 1968) and also to be a recalcitrant molecule. As shown in Table 1, several CAR-utilizing bacteria were isolated by the enrichment culture method using CAR as a sole source of carbon and energy, or of carbon, nitrogen, and energy. Ouchiyama et al. (1993) screened soil and activated sludge samples for CAR-utilizing bacteria as a sole source of carbon, nitrogen, and energy. They isolated *Pseudomonas* sp. strain CA06 from the 202 soil samples, while *P. resinovorans* strain CA10 was obtained from one of the 4 activated sludge samples. Anthranilic acid and catechol were identified as the main metabolites of CAR by high-performance liquid chromatography (HPLC) and GC-MS analyses. In addition, meta cleavage activity for 2,3-dihydroxybiphenyl was strongly induced in CAR-grown strain CA10 cells. These results suggested that strain CA10 degrades CAR to anthranilic acid through the angular dioxygenation, meta cleavage, and hydrolysis. This pathway of CAR degradation by strain CA10 is homologous to the pathway of DF degradation by DF-degrading bacteria (Figs. 3 and 6). From the extract of culture medium of various CAR-utilizing bacteria isolated, anthranilic acid was identified as the main metabolite of CAR (Gieg et al., 1996; Hisatsuka and Sato, 1994; Kirimura et al., 1999; Ouchiyama et al., 1998; Schneider et al., 2000), suggesting that these CAR-utilizing bacteria degrade CAR through a pathway similar to that of strain CA10. Gieg et al. (1996) identified 3a,4-dihydro-2-hydroxypyrrolo[1,2-a]-quinoline-1,5-dione from the acidified culture supernatant extracts of *Pseudomonas* sp. strain LD2. This compound is likely an artifact formed by Michael-type addition of an amino group of the meta cleavage compound of 2′-aminobiphenyl-2,3-diol, 2-hydroxy-6-(2-aminophenyl)-6-oxo-2,4-dienoic acid, to the double bond of the aliphatic side chain, followed by the elimination of water under acidic conditions (Fig. 6). Intramolecular cyclization of meta cleavage compounds by Michael-type addition has been reported in DF- and 2-hydroxybiphenyl-degradation pathways as described above (Kohler et al., 1993; Strubel et al., 1991). The presence of this cyclized compound strongly suggested that CAR is indeed being attacked by CAR-utilizers via angular dioxygenation followed by meta cleavage. The mass spectrum obtained for this compound was virtually identical to that of the dimethyl ether of 2′-aminobiphenyl-2,3-diol (Ouchiyama et al., 1993). Because the analytical data reported by Gieg et al. (1996) indicated that this compound is unequivocally 3a,4-dihydro-2-hydroxypyrrolo[1,2-a]-quinoline-1,5-dione, it is quite possible that the metabolite identified from culture medium of strain CA10 by Ouchiyama et al. (1993) was this cyclized compound.

When anthranilic acid was used for a growth substrate by strain CA10, catechol and a small amount of cis,cis-muconic acid was detected (Ouchiyama et al., 1993). These results indicate the possibility that anthranilic acid is converted to catechol by anthranilate 1,2-dioxygenase, and that the catechol formed is mineralized by the β-ketoadipate pathway via ortho cleavage by strain CA10. On the other hand, *P. stutzeri* strain OM1 was revealed to metabolize catechol via meta cleavage (Ouchiyama et al., 1998).

### 1.4. DBT degradation via angular dioxygenation

Crude oil and its distillates contain significant amounts of low-molecular-mass organosulfur compounds such as alkyl- and cycloalkylthiols, alkyl- and arylthioethers and aromatic heterocycles based on thiophene. This last group includes thiophene itself, benzothiophene, DBT, and their alkylated derivatives. Combustion of these compounds results in the release into the atmosphere of sulfur dioxide and hence in acid rain. Thus there is considerable interest in the development of petrochemical desulfurization technologies. One approach involves the application of microorganisms selected for their ability to degrade the organosulfur component. In the investigation of biodesulfurization of petroleum, DBT has been used as a model compound of organic sulfur compounds in petroleum (McFarland, 1999; Ohshiro and Izumi, 1999). The release of sulfur atom by specific cleavage of the car-

Carbazole

$\downarrow$

Angular dioxygenation

$\downarrow$

Spontaneous cleavage

2'-Aminobiphenyl-2,3-diol

$\downarrow$

meta Cleavage

Michael-type cyclization

Hydrolysis

Anthranilic acid + 2-Hydroxypenta-2,4-dienoic acid

Catechol

ortho Cleavage

meta Cleavage

TCA cycle

Fig. 6. Overview of bacterial degradative pathways for carbazole (CAR) via angular dioxygenation.

The structures shown in brackets have not been characterized. The arrows with solid and broken lines indicate the enzymatic and spontaneous conversions, respectively.

bon-sulfur bond, rather than degradation/mineralization of organic sulfur compounds, is preferred for the microbial desulfurization process so that sulfur is removed but carbon and calorific values remain intact. Therefore, the microbial desulfurization pathway without a cleavage of carbon-carbon bond has been extensively investigated (McFarland, 1999; Ohshiro and Izumi, 1999). On the other hand, DBT degradation by breakdown of the carbon skeleton via angular dioxygenation is less investigated.

Brevibacterium sp. strain DO was isolated on the basis of being able to grow on DBT as a sole source of sulfur, carbon, and energy (van Afferden et al., 1988). By co-chromatography, UV spectroscopy, and GC-MS,

DBT-sulfoxide, DBT-sulfone, and benzoic acid were isolated and identified as a metabolic intermediates of DBT by strain DO (van Afferden et al., 1990). During DBT utilization by strain DO, sulfite was released in a stoichiometrical amount and was further oxidized to sulfate. When culture of strain DO was supplemented with both DBT-sulfone and 9-fluorenone, 9-fluorenone was converted to angular dioxygenation product, 1-hydro-1,1a-dihydroxy-9-fluorenone (van Afferden et al., 1993). This result provides strong support for an analogous hydroxylation of DBT-sulfone at the angular position, and implies the formation of chemically unstable hemimercaptal (S-oxidized form) which spontaneously decays and forms 2',3'-dihydroxybiphenyl-2-

Aromatics degradation with angular attack

Dibenzothiophene
![Structure](https://i.imgur.com/1234567.png)

Dibenzothiophene sulfone
![Structure](https://i.imgur.com/8901234.png)
Angular dioxygenation

Spontaneous cleavage

2',3'-Dihydroxybiphenyl-2-sulfinate
![Structure](https://i.imgur.com/5678901.png)

meta Cleavage
![Structure](https://i.imgur.com/2345678.png)

SO₃²⁻ → SO₄²⁻

Benzoic acid
![Structure](https://i.imgur.com/9012345.png)

TCA cycle

Fig. 7. Overview of bacterial degradative pathways for dibenzothiophene (DBT) via angular dioxygenation. The structures shown in brackets have not been characterized. The arrows with solid and broken lines indicate the enzymatic and spontaneous conversions, respectively.

sulfinate (Fig. 7). Additionally, 2,3-dihydroxybiphenyl serves as a substrate for the growth of strain DO, and meta cleavage activity for 2,3-dihydroxybiphenyl was detected in cell-free extract prepared from DBT-grown strain DO cells. These results support the formation of 2',3'-dihydroxybiphenyl-2-sulfinate as a metabolite in the degradation pathway of DBT (Fig. 7). This dihydroxy compound is further metabolized by meta cleavage, and the ring fission product is degraded to benzoate and sulfite.

DBT-sulfone degrader, *Arthrobacter* sp. strain DBTS2 which was designated strain K3b by Dahlberg et al. (1993), was isolated by the enrichment culture method using DBT-sulfone as a sole source of carbon, sulfur, and energy (Ochman et al., 1990). Based on the results that this bacterium can co-oxidize 9-fluorenone to 1-hydro-1,1a-dihydroxy-9-fluorenone just as *Brevibacterium* sp. strain DO did, and that benzoic acid was identified as the metabolite of DBT, the DBT-sulfone-degradation pathway of strain DBTS2 was proposed to be identical to that of strain DO. However, in sharp contrast, strain DBTS2 can not grow on DBT itself (Dahlberg et al., 1993). This bacterium can grow on DBT-sulfoxide as a sole source of carbon, sulfur, and energy. Accordingly, strain DBTS2 possibly lacks the enzyme required for initial sulfoxidation of DBT.

2. Genetic and Biochemical Characterization of Degradation Pathways with Angular Dioxygenation

Compared to the degradation pathways via lateral dioxygenation of aromatic compounds such as

biphenyl, toluene, phenol, and naphthalene, the genetic and enzymatic analyses of angular dioxygenation are less reported, because fewer bacteria having the angular dioxygenase has been isolated so far. To our knowledge, six genes encoding angular dioxygenase have been reported (Armengaud et al., 1998; Dehmel et al., 1995; Kasuga et al., 2001; Ouchiyama et al., 1998; Sato et al., 1997a; Shepherd and Lloyd-Jones, 1998). The genes encoding CAR-degradation enzymes isolated from *P. stutzeri* strain OM1 were nearly identical to those from *P. resinovorans* strain CA10 (Ouchiyama et al., 1998), suggesting that the gene cluster isolated from strain OM1 is a different allele of the CAR-degrading *car* gene cluster isolated from strain CA10. From CAR-degrading bacteria, *Sphingomonas* sp. strain CB3, Shepherd and Lloyd-Jones (1998) reported the isolation of the gene encoding the possible angular dioxygenase for CAR, whose deduced amino acid sequence was divergent from that of strain CA10. However, the authors did not find clear evidence that the dioxygenase isolated could catalyze the angular dioxygenation for CAR (Shepherd and Lloyd-Jones, 1998). In addition, no metabolic intermediate was identified, while strain CB3 was shown to grow on anthranilic acid (Shotbolt-Brown et al., 1996). Shepherd and Lloyd-Jones (1998) reported that the expression of this *car* gene cluster was induced in CAR-grown strain CB3 cells. However, the enzymes encoded by *car* gene cluster of strain CB3 were related to biphenyl-degrading enzymes, and the metabolic intermediates have a closely related chemical structure between the degradation pathways of CAR and biphenyl as shown above. Together with the fact that the meta cleavage compound of 2,3-dihydroxybiphenyl is an inducer of the biphenyl-degrading *bph* operon of *Pseudomonas* sp. strain KKS102 (Ohtsubo et al., 2000), we can not exclude the possibility that the *car* gene cluster of strain CB3 is fortuitously induced by a CAR-degradation intermediate. Under this consideration, we exclude the *car* genes isolated from strain CB3 from our consideration.

flanking region from strain TD2 and the partial determination of its nucleotide sequence, revealed that meta cleavage enzyme was encoded in this locus, and that the genes encoding the enzymes involved in the CAR degradation were clustered (Kimura et al., 1996). Based on these results, we cloned the CAR-degrading *car* gene cluster from the wild-type strain CA10 by means of shot-gun cloning using the meta cleavage activity for 2,3-dihydroxybiphenyl as an indicator (Sato et al., 1997a,b). As a result of the nucleotide sequence analysis of the cloned DNA fragment, homology search analyses, and determination of the enzyme activities, the genes encoding angular dioxygenase for CAR (CAR 1,9a-dioxygenase; CarAaAcAd), meta cleavage enzyme (2′-aminobiphenyl-2,3-diol 1,2-dioxygenase, CarBaBb), and meta cleavage compound hydrolase (2-hydroxy-6-oxo-6-(2-aminophenyl)hexa-2,4-dienoate hydrolase, CarC) were identified. Consequently, the genetic structure of the *car* gene cluster was revealed to be *carAaAaBaBCAc*(ORF7)*Ad* as shown in Fig. 8A, although the function of ORF7 is unknown. It was revealed that the 1,263-bp DNA region containing the entire *carAa* gene was tandemly duplicated except for one base, and that there were only three bases (GGC) between these tandemly-linked DNA regions (Sato et al., 1997a). To our knowledge, such a gene structure has never been observed in other degradative gene clusters. The significance of this duplication of *carAa* genes in strain CA10 cells remains to be clarified. Recently, we succeeded in isolating several CAR-degrading bacteria, some of which have a homologous *car* gene cluster containing a single copy of *carAa* gene (Widada, J., Nojiri, H., Shintani, M., Nakai, S., Yoshida, T., Habe, H., and Omori, T., unpubl.). Comparison of the CAR-degrading capacity between the bacteria having *car* gene clusters with a different copy number of *carAa* gene might provide us an insight into the significance of this gene duplication. In addition, the genes encoding terminal oxygenase component (CarAa) of CAR 1,9a-dioxygenase are separated from the genes encoding ferredoxin component (CarAc) and ferredoxin reductase component (CarAd) by 2-kb DNA fragment encoding meta cleavage enzyme (CarBaBb) and meta cleavage compound hydrolase (CarC) (Fig. 8A).

The phylogenetic tree of the large (*α*) subunits of terminal oxygenases showed that CAR 1,9a-dioxygenase is an evolutionarily novel enzyme (Fig. 9). The Rieske nonheme iron oxygenases were classified into

2.1. CAR degradation by *P. resinovorans* strain CA10

To isolate the genes involved in CAR degradation, transposon mutagenesis of *P. resinovorans* strain CA10 was performed (Kimura et al., 1996). One Tn5-Mob inserted mutant, strain TD2, was capable of using anthranilic acid but not CAR as a sole source of carbon, nitrogen, and energy. Cloning of the Tn5-Mob-

2001 Aromatics degradation with angular attack 291

A
5 kb

Symbols
- car gene cluster
  - involved in the conversion of CAR to anthranilic acid (carAaAcAdBaBbC) and 2-hydroxypenta-2,4-dienoic acid to TCA cycle intermediates (carDEF)
- ant gene cluster
  - encoding anthranilate 1,2-dioxygenase
- cat gene cluster
  - involved in the metabolism of catechol
- Other genes
- Insertion sequence

Fig. 8. Physical maps of the 44,266-bp EcoRV-EcoRI DNA region containing car and ant genes (A) and 8,918-bp EcoRI-NotI DNA region containing catRBCA genes (B), that are involved in the carbazole (CAR) degradation by P. resinovorans strain CA10 (Nojiri et al., 2001, in press; Sato et al., 1997a, b).

The pentagons and triangles in the physical map indicate the size, location, and the direction of transcription of the ORFs derived from the DNA sequence analysis. Restriction site designations are: A, Apal; C, Clal; EI, EcoRI; EV, EcoRV; H, HindIII; K, KpnI; N, NotI; P, PstI; SI, SalI; Xh, Xhol.

three classes in terms of the number of constituent components and the nature of the redox centers (Batie et al., 1991; Mason and Cammack, 1992). Class I consists of two-component enzymes in which the first protein is a reductase containing both a flavin and a [2Fe-2S] redox center and the second component is the terminal oxygenase; class II consists of three-component enzymes in which the flavin and [2Fe-2S] redox centers of the reductase are on the separate flavoprotein (ferredoxin reductase) and ferredoxin, respectively; and class III consists of three-component enzymes in which the reductase contains both a flavin and [2Fe-2S] redox center but also requires a second [2Fe-2S] center on a ferredoxin for electron transfer to the terminal oxygenase. Further subdivision is based on the type of flavin (FMN or FAD) in the reductase, the coordination of the [2Fe-2S] center in the ferredoxin, and the number of terminal oxygenase subunits. The consensus sequence of Rieske-type iron-sulfur proteins for the binding of a [2Fe-2S] cluster (Kauppi et al., 1998) was found in the deduced amino acid sequences of the terminal oxygenase (CarAa) and ferredoxin (CarAc) of CAR 1,9a-dioxygenase. The ferredoxin reductase (CarAd) possesses the consensus sequence of the plant-type iron-sulfur proteins for the binding of a [2Fe-2S] cluster (Otaka and Ooi, 1989) and the putative FAD- and NAD-binding site (Neidle et al., 1991). Based on these results, CAR 1,9a-dioxygenase seems to have a class III electron transport chain like the well-known naphthalene dioxygenase. However, based on the phylogenetic study (Fig. 9), terminal oxygenase component (CarAa) of CAR 1,9a-dioxygenase is shown to have a closer relationship to the corresponding proteins contained in class IA dioxygenase systems (Nam et al., 2001; Sato et al., 1997a). This discrepancy between the phylogenetic affiliation of terminal oxygenase component and the feature of electron transport components indicates the possibility that some of these three components have been recruited from the other oxygenase system.

Recently, we determined the nucleotide sequence of the 27,939-bp-long upstream and 9,448-bp-long downstream regions of carAaAaBaBbCAc(ORF7)Ad genes (Nojiri et al., 2001). Thirty-two ORFs were identified, and the car gene cluster was consequently revealed to consist of ten genes (carAaAaBaBbCAcAdDFE), as shown in Fig. 8A, encoding the enzymes for the first three-step conversions of CAR to anthranilic acid and

OxoO (P. putida 86, Y12655)
CarAa (P. resinovorans CA10, D89064)
CarAa (P. stutzeri OM1, AB001723)

PobA (P. pseudoalcaligenes POB310, X78823)
Phi3 (P. putida, D13229)
CbaA (Ralstonia sp. BR60, U18133)
TsaM (C. testosteroni T-2, U18133)
VanA (Acinetobacter sp. ADP1, AF009672)
XylX (P. putida mt-2, M64747)
TfIA (B. cepacia AC1100, U11420)
BenA (Acinetobacter sp. ADP1, AF009224)
ChdA (B. cepacia 2CBS, X79076)
AntA (Acinetobacter sp. F1, U24215)
CmtAb (P. putida F1, U24215)
TdnA1 (P. putida UCC22, X59790)
AddA3 (Acinetobacter sp. YAA, D86080)
OhbB (P. aeruginosa JB2, AF087482)
NagG (Pseudomonas sp. U2, AF036940)
NarAa (Rhodococcus sp. NCIMB12038, AF082663)
NidaA (Rhodococcus sp. IZ2, AF121905)
PhdA (Nocardoides sp. KPT, AB017794)
DbfA1 (Terrabacter sp. DBF63, AB054975)
PahA3 (P. aeruginosa Pak1, D84146)
NagAc (Pseudomonas sp. U2, AF036940)
NtdAc (Pseudomonas sp. JS42, U49504)
DmAc (Burkholderia sp. DNT, U62430)
NahAc (P. putida G7, M86949)
NdoC2 (P. fluorescens ATCC17483, AF004283)
PahAc (P. putida OUS82, D16629)
NdoB (P. putida NCB19816, M23914)
NdoC (P. putida ATCC17484, AF004284)
NahA3 (P. putida BS202, AF010471)
DoxB (Pseudomonas sp. C18, M60405)
NahAc (P. putida NCIB9816-4, U49496)
HcaA1 (E. coli K-12, Z37966)
BphAlf (S. aromaticivorans F199, AAD03858)
BphA1 (R. wittichii RW1, X72850)
CarAa (Sphingomonas sp. CB3, AF060489)
BphA1 (Pseudomonas sp. KKS102, D17319)
BphA1 (C. testosteroni B-356, U47637)
CumA1 (P. fluorescens IP01, D37828)
IpbA1 (Pseudomonas sp. JR1, U53507)
BphA1 (Burkholderia sp. JB1, AJ010057)
BphA1 (P. cepacia LB400, M86348)
BphA1 (Pseudomonas sp. B4, U95054)
BphA1 (Pseudomonas sp. KF707, M83673)
BphA1 (R. erythropolis TA421, D88021)
BphA1 (R. erythropolis BD2, U24277)
BpdC1 (Rhodococcus sp. M5, U27591)
TecA1 (Burkholderia sp. PS12, U78099)
BedC1 (P. putida ML2, L04642)
TodC1 (P. putida FJ04996)
MeaAa (Ralstonia sp. JS705, AI006307)

Fig. 9. Phylogenetic tree of large (α) subunits of terminal oxygenase components of Rieske nonheme iron oxygenases based on the multiple alignment of related proteins.
The names of bacterial strains and the DDBJ, EMBL, and GenBank or Swiss-Prot and PIR accession numbers are indicated in parentheses. The scale bar denotes 0.1 substitution per site.
The tree was constructed by neighbor-joining method, and bootstrap values above 80 from 100 resampling are shown for each node. Bold characters indicate angular dioxygenases.

2001 Aromatics degradation with angular attack 293

![Diagram](attachment:diagram.png)

Fig. 10. Proposed model of the developmental process of novel structure in the car gene cluster and its flanking region of P. resinovorans strain CA10 (Nojiri et al., 2001).

Shaded boxes represent the car genes and several neighboring ORFs. Shaded boxes with triangles indicate the ISs. Striped boxes represent the ant genes or the transposed DNA region form antA gene within ORF9. The arrows with solid lines indicate the transposition (recombination). The arrows with broken lines indicate the duplication or homologous recombination.

the degradation of 2-hydroxypenta-2,4-dienoic acid. The high identities (68–83%) with the enzymes involved in 3-(3-hydroxyphenyl)propionic acid degradation were observed only in CarFE. Together with the fact that two ORFs are inserted between carD and carFE, it is quite likely that carFE were recruited from other loci. In the 21-kb upstream region from carAa, the Rieske nonheme iron oxygenase genes (ORF26, ORF27, and ORF28) were found (Fig. 8A). Inductive expression in CAR-grown cells and the results of homology searching indicate that these genes encode the anthranilate 1,2-dioxygenase involved in CAR degradation. Therefore, these ORFs were designated antABC.

Four homologous insertion sequences (ISs) were identified in neighboring regions of car and ant genes (Fig. 8A) (Nojiri et al., 2001). Based on the features of inverted repeats and direct repeats, and the homology of the transposase, these ISs were classified into the IS5 subgroup of the IS5 family (Mahillon and Chandler, 1998). Thus, these ISs were designated IS5car1 to IS5car4. The nucleotide sequence of IS5car1 is identical to that of IS5car4, and nearly identical to that of IS5car2 with one base pair mismatch. On the other hand, the nucleotide sequence of IS5car3 showed only 81.6% identity with that of IS5car1, although it showed 98% identity with the IS-like sequence containing tnpA1 from P. stutzeri strain AN10 (Bosch et al., 1999). Comparison of the nucleotide sequence revealed that one-ended transposition of IS5car2 together with the 5′-portion of antA into the immediate upstream of carAa had resulted in the formation of IS5car1 and ORF9 (Fig. 10). In addition to the IS-dependent recombination, gene duplications (ORFs 11 and 12, and two copies of carAa gene) were shown to occur (Fig. 10). The G+C content of the total DNA of strain CA10 is 61.9% (Ouchiyama et al., 1993). While the G+C contents of carAaAaBaBbCAc(ORF7)Ad genes is 48.0–53.3%, those of 3′-portion (1,006 bp) of ORF9, ORF11, and ORF12 are 49.2–50.7%. Similarly, the G+C contents of ORFs located in the downstream region of the car gene cluster except for IS ranged from 38.5 to 52.3%. On the other hand, the G+C contents of the 5′-portion of ORF9 and ORFs located in the upstream region of ORF13 except for ISs are similar to or higher than that of total DNA of strain CA10.

ORF13 located between the high and low G+C content regions showed an intermediate value of G+C content (55.0%). These results suggest that the gene fusion had occurred in or around ORF13 (Fig. 10), although the molecular mechanism has not been determined.

In addition, the car and ant gene clusters were found to be located on the 200-kb-long circular plasmid, pCAR1 (Nojiri et al., 2001). Catechol formed from CAR is degraded by the ortho cleavage pathway encoded by the cat operon (Fig. 8B) (Nojiri et al., in press). Considering that the cat operon were encoded on chromosome of strain CA10 (Nojiri et al., 2001), it can be considered that strain CA10 had acquired metabolic capacity of CAR by recruitment of pCAR1. In newly-identified CAR-degrading bacteria, we have also detected the pCAR1-like plasmid containing the car gene cluster homologue, although chromosomally encoded car gene clusters were similarly found (Widada, J. et al., unpubl.). This phenomenon suggests the possibility that pCAR1 plays an important role for distribution of CAR-degrading activity in the environment. The existence of a chromosomally encoded car gene cluster also implies the possibility that the entire or a portion of pCAR1 is able to integrate into the chromosome or other replicon as reported in 2,4-dichlorophenoxyacetic acid-degradative plasmid, pKA2 (Ka and Tiedje, 1994) and toluene/xylene-degradative plasmid, pWW0 (Jeenes and Williams, 1982; Meulien and Broda, 1982; Sinclair et al., 1986). Together with the fact that all ISs identified in strain CA10 are located on pCAR1 (Nojiri et al., 2001), it might also be possible that the ISs specifically function in the integration of car and ant gene clusters and their flanking regions into the other replicon.

### 2.2. DF/DD-degradation pathway of S. wittichii strain RW1

Two isofunctional meta cleavage compound hydrolases (hydrolase H1 and H2) were purified from strain RW1 cell by Bünz et al. (1993). Each hydrolase was able to hydrolyze 2-hydroxy-6-oxo-6-(2-hydroxyphenyl)hexa-2,4-dienoate (meta cleavage compound in DF-degradation pathway) and 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate (meta cleavage compound in biphenyl-degradation pathway) to produce salicylic acid and benzoic acid, respectively. These hydrolases were presumed to be monomeric. There was a 50% identity between N-terminal amino acid sequences of

the two hydrolases, but no similar protein sequences were observed with database search analysis.

The dbfB gene encoding 2,2′,3-trihydroxybiphenyl 1,2-dioxygenase, the meta cleavage enzyme involved in DF degradation, was cloned from the genomic cosmid library of strain RW1 (Happe et al., 1993). Comparison of the amino acid sequence of DbfB revealed significant homologies (>30% identity) to the meta cleavage enzyme of Burkholderia sp. strain LB400 (Hofer et al., 1993), P. pseudoalcaligenes strain KF707 (Furukawa et al., 1987), and P. putida KF715 (Hayase et al., 1990). These significant homologies to meta cleavage enzyme from biphenyl/polychlorinated biphenyl-degrading bacteria are perhaps not surprising, given the structural relatedness of the substrates of these enzymes. Unlike most meta cleavage enzymes, which have an oligomeric quaternary structure, the 2,2′,3-trihydroxybiphenyl 1,2-dioxygenase of strain RW1 is a monomeric protein. ¹H NMR analysis of purified DbfB revealed that the high spin Fe(II) iron present in the active form of the enzyme is coordinated by at least two histidine residues (Bertini et al., 1995).

The initial angular dioxygenation for DF and DD is carried out by a three-component dioxin dioxygenase (DF 4,4a-dioxygenase) system, consisting of a terminal oxygenase which has a heterodimeric structure (α₂β₂) and an electron-transfer system (Bünz and Cook, 1993). The electrons obtained at the expense of the oxidation of NADH by ferredoxin reductase are channeled to the terminal oxygenase component by ferredoxin. An atypical electron transfer chain interacting with the terminal oxygenase component of the dioxin dioxygenase system, consisting of a 12-kDa putidaredoxin-type [2Fe-2S] ferredoxin and two iso-functional monomeric flavoproteins (ferredoxin reductase A1 and A2), has been purified, and the sequences of the N-termini of the polypeptides have been determined (Bünz and Cook, 1993). Based on the characteristics of the electron supply system, the dioxin dioxygenase system of strain RW1 is proposed to be a class IIA dioxygenase system according to the classification scheme of Batie et al. (1991).

Using the N-terminal sequence of purified putidaredoxin-type [2Fe-2S] ferredoxin and the conserved stretch of only four amino acids at the C-terminal of the ferredoxin, Armengaud and Timmis (1997) designed a nested-PCR strategy in order to obtain a specific probe for the ferredoxin gene. With this probe, the authors identified the ferredoxin gene, fdx1. Ferredoxin

2001 Aromatics degradation with angular attack 295

**dxn locus**


dbfB dxnA1 A2 dxnB dxnC fdx3 dxnD dxnE F G H I transposase


**redA2 locus**


ORF R1 redA2


**fdxl locus**


ORF E1 ORF E3
fdx1 ORF E2 ORF E4


| Symbol | Gene product or putative gene product |
|--------|---------------------------------------|
| ▲      | Component of dioxin dioxygenase       |
| ▶      | meta Cleavage enzyme                 |
| ▷      | meta Cleavage compound hydrolase     |
| ⬇      | 4-Hydroxysalicylic acid/hydroxyquinol degradative enzyme |
| ▷      | Other enzyme or unknown function     |

Fig. 11. Genetic organization of loci encompassing the genes involved in dibenzofuran (DF) and dibenzo-p-dioxin (DD) degradation by S. wittichii strain RW1 (Armengaud and Timmis, 1997, 1998; Armengaud et al., 1998, 1999, 2000).

The pentagons and triangles in the physical map indicate the size, location, and the direction of transcription of the ORFs derived from the DNA sequence analysis.

Fdx1 of strain RW1 shares about 40% identity with several [2Fe-2S] ferredoxins involved in electron transfer to bacterial monooxygenases. UV/visible spectrophotometry and electron paramagnetic resonance spectroscopy using the hyperexpressed Fdx1 revealed that Fdx1 contains a putidaredoxin-type [2Fe-2S] cluster. The Fdx1 is thus clearly related to the group of ferredoxins which typically donate electrons to monooxygenase, trivially designated by the generic name cytochrome P-450, and thus constitute an atypical electron transfer system for the Rieske nonheme iron oxygenase system. The redA2 gene, which encodes the ferredoxin reductase A2 described by Bünz and Cook (1993), was also successfully cloned by using the PCR technique (Armengaud and Timmis, 1998). From the nucleotide and deduced amino acid sequences, as well as from the biochemical data obtained from a recombinant form of this reductase, it is evident that this flavoprotein is similar to class-I cytochrome P-450-type reductase (Munro and Lindsay, 1996). Indeed, the two proteins of the electron transport system, RedA2 and Fdx1, are similar to their counterparts supplying electrons to the well-characterized cytochrome P450cam, a monooxygenase from P. putida able to oxidize camphor (Peterson et al., 1990). As in the cases with fdx1 and redA2, the genes encoding the terminal oxygenase component of the dioxin dioxygenase system were isolated by PCR-based technique (Armengaud et al., 1998). The authors designed the degenerated primers based on the determined polypeptide sequences of the dioxin dioxygenase (Bünz and Cook, 1993) and hydrolase (Bünz et al., 1993), and on the consensus regions of related enzymes. The dxnA1 and dxnA2 cistrons, encoding the large (α) and small (β) subunits of the terminal oxygenase component of the dioxin dioxygenase system, were shown to be located just upstream of the hydrolase gene [dxnB encoding hydrolase H1 termed by Bünz et al. (1993)] (Fig. 11). At the 4.5-kb upstream region of dxnA1 gene, the dbfB gene encoding 2,2',3-trihydroxybiphenyl 1,2-dioxygenase which catalyzes the second step in the DF/DD-degradation pathway was located in the opposite direction (Fig. 11). Most catabolic pathways in bacteria that have been studied so far have a genetic organization characterized by clustering of the genes of entire pathways or independently functioning pathway segments in single transcriptional units transcribed from highly regulated promoters. The above results show that the respective loci encoding dxnA1A2B, fdx1, redA2, and dbfB are physically distinct (Fig. 11). The genetic organization of the DD/DF-degradation pathway in strain RW1 is thus

296

Nojiri, Habe, and Omori

Vol. 47

**Fig. 12. Genetic organization of loci encompassing the genes involved in dibenzofuran (DF) degradation by Terrabacter sp. strain DBF63 (Kasuga et al., 1997, 2001).**

The pentagons and triangles in the physical map indicate the size, location, and the direction of transcription of the ORFs derived from the DNA sequence analysis.

---

**Symbol** | **Gene product or putative gene product**
--- | ---
Terminal oxygenase component of DF 4,4a-dioxygenase
meta Cleavage enzyme
meta Cleavage compound hydrolase
Transcriptional regulator
Other enzyme or unknown function
Possible insertion sequence

---

clearly different from those of other catabolic pathways characterized in bacteria.

Recently, a genetic analysis of the downstream region of *dxnB* gene was carried out (Armengaud et al., 1999). In this region, there exist 10 additional genes, *dxnC* to *dxnl*, constituting the same operon with *dxnA1A2B* (Fig. 11). Based on the homology searches and the detection of enzymatic activities, DxnDFEGHI was revealed to form a complete 4-hydroxysalicylic acid/hydroxyquinol-degradative pathway. Unlike the DD/DF-degradation upper pathway to catechol/salicylate, the genes consisting the 4-hydroxysalicylic acid/hydroxyquinol-degradative pathway of strain RW1 were well clustered (Fig. 11). A homology search analysis revealed the presence of an additional gene (*fdx3*) encoding ferredoxin that showed significant similarity to Fdx1 (Armengaud and Timmis, 1997), and putidaredoxin (Peterson et al., 1990). Fdx3 was shown to have biochemical features similar to Fdx1, and to function as an electron transport protein from RedA2 to DxnA1A2 in the dioxin dioxygenase system (Armengaud et al., 2000).

---

**2.3. DF/DD/FN degradation by Terrabacter sp. strain DBF63**

Two genes encoding meta cleavage enzymes were cloned from Terrabacter sp. strain DBF63 by shot-gun cloning on the basis of the expressed activity of meta cleavage enzyme for 2,3-dihydroxybiphenyl (Kasuga et al., 1997). One of the two clones, Escherichia coli MV1184 (pLN1), exhibited a higher level of activity for 2,3-dihydroxybiphenyl than 3-methylcatechol, while the other, E. coli MV1184 (pKN1) exhibited a low level of activity for both catechol derivatives. In addition, as a result of Southern hybridization analysis, the gene carried on pLM1 was revealed to be lost in strain DBF63W, which is a mutant of strain DBF63 deficient in the ability to utilize DF and obtained by a continuous culture on nutrient broth. On the other hand, clear hybridization was detected on blots of total DNAs from both strains in Southern hybridization analysis using the gene carried by pKN1 as a probe. These results suggest that the gene carried on pLM1 encodes the meta cleavage enzyme involved in the DF degradation by strain DBF63. Nucleotide sequence analysis of 8.6-kb KpnI-Stul DNA fragment containing pLM1 insert and its flanking region revealed the existence of *dbfB* and *dbfC* genes encoding meta cleavage enzyme and meta cleavage compound hydrolase, respectively (Fig. 12). The gene encoding a possible phenol hydroxylase and IS401-like sequence were found in the flanking region of the *dbfBC* cistron, as shown in Fig. 12. Co-expression in E. coli cells of *dbfBC* genes and *carA*-AcAd genes encoding CAR 1,9a-dioxygenase, that can catalyze the angular dioxygenation for DF/DD as discussed below, revealed that DbfB and DbfC were capable of degrading 2,2′,3-trihydroxybiphenyl to salicylic acid. In addition, based on the formation of the yellow meta cleavage compound from DD, it was shown that DbfB could function as a meta cleavage enzyme for 2,2′,3-trihydroxydiphenyl ether in the DD-degradation pathway.

Recently, Kasuga et al. (2001) succeeded in cloning the *dbfA1* and *dbfA2* genes, which encode the large

(α) and small (β) subunits of the terminal oxygenase component of DF 4,4a-dioxygenase system, respectively, from strain DBF63 genome by a PCR-based method (Fig. 12). DbfA1 and DbfA2 showed moderate homology to the large (α) and small (β) subunits of other ring-hydroxylating dioxygenases (less than 40%), respectively, and some motifs such as the Fe(II) binding site and the Rieske-type [2Fe-2S] cluster ligands (Kauppi et al., 1998) were conserved in DbfA1. DF 4,4a-dioxygenase activity was confirmed in *E. coli* cells containing the cloned *dbfA1* and *dbfA2* genes with the complementation of nonspecific ferredoxin and ferredoxin reductase component of *E. coli*. Phylogenetic analysis revealed that DbfA1 formed a branch with a recently reported large (α) subunit of PAH dioxygenase from gram-positive bacteria (Fig. 9). Just downstream and upstream of *dbfA1A2* cistron, a putative hydrolase gene (ORF3) and dehydrogenase gene (ORF4) were located, respectively, in the same direction to *dbfA1A2* cistron, suggesting that *dbfA1A2* genes constitute the gene cluster with operonic structure (Fig. 12). The putative transcriptional regulator genes were found in the upstream region of this gene cluster. In addition, putative ferredoxin reductase gene (ORF8) was isolated in the vicinity of the *dbfA1* and *dbfA2* genes, although the nucleotide sequence of 5′-terminal region has not been determined completely. Although there is a possibility that DbfA1A2 is complemented with the product of ORF8, the putative regulatory gene (ORF5) that had the opposite direction of transcription exists between *dbfA1A2* genes and ORF8 (Fig. 12). This indicates that *dbfA1A2* genes and ORF8 are located on different transcriptional units. In addition, another gene set encoding the terminal oxygenase component of the Rieske nonheme iron oxygenase exists in the upstream region of ORF8, based on the Southern hybridization analysis, suggesting that the product of ORF8 functions with another terminal oxygenase component.

2.4. Angular dioxygenation for carboxylated DE by *P. pseudoalcaligenes* strain POB310

*P. pseudoalcaligenes* strain POB310 has two 40-kb-long plasmids that could be transferred by conjugation into the gram-negative bacterium (Dehmel et al., 1995). Transfer of one of the plasmids resulted in formation of transconjugants that used 4-carboxy DE as a growth substrate, suggesting that the genes involved in 4-carboxy DE degradation in strain POB310 are located on this plasmid. This plasmid was designated pPOB, and a 12.8-kb EcoRI fragment of pPOB was cloned to the broad host-range-cloning vector. Transfer of the resultant plasmid to the carboxylated DE degradation-negative mutant strain POB310spMu resulted in the formation of transconjugants able to utilize carboxylated DE. Consequently, Dehmel et al. (1995) determined the nucleotide sequence of 4.5-kb EcoRI-KpnI fragment responsible for angular dioxygenation for carboxylated DE.

Subsequent analyses revealed the presence of two ORFs (*pobA* and *pobB*). Homology search analysis of the deduced amino acid sequence of *pobA* gene revealed the presence of binding sites of Rieske-type [2Fe-2S] cluster and mononuclear Fe(II) atom (Kauppi et al., 1998). PobB was shown to have a binding site for plant-type [2Fe-2S] cluster, and showed the 36–40% homologies with reductase components of phthalate dioxygenases from *P. cepacia* (Correll et al., 1992) and *P. putida* (Nomura et al., 1992), and vanilate demethylase (VanB) from *Pseudomonas* sp. strain ATCC19151 (Brunel and Davison, 1988). These observations suggested that *pobA* and *pobB*, respectively, encode the oxygenase and reductase components of carboxylated DE dioxygenase (POB dioxygenase). Based on the feature of electron transport chain, the POB dioxygenase system was assigned to class IA Rieske nonheme iron oxygenase (Batie et al., 1991; Mason and Cammack, 1992). Phylogenetic analysis of the catalytic subunit of the terminal oxygenase component (Fig. 9) revealed that POB dioxygenase is an evolutionarily novel dioxygenase.

2.5. meta Cleavage enzyme involved in DF mineralization by *Terrabacter* sp. strain DPO360

Schmid et al. (1997) revealed that two 2,3-dihydroxybiphenyl 1,2-dioxygenases (BphC1 and BphC2) and one catechol 2,3-dioxygenase were expressed in *Terrabacter* sp. strain DPO360 growing with DF as a sole source of carbon and energy. These enzymes were purified by anion exchange, hydrophobic interaction, and gel filtration column to apparent homogeneity as homodimers (BphC1 and BphC2) and as homotetrameric catechol 2,3-dioxygenase. Based on the specific constants *k*~cat~/K~m~ of these meta cleavage enzymes, both BphC1 and BphC2 were shown to be responsible for the meta cleavage of 2,2′,3-trihydroxybiphenyl. The DNA region containing the *bphC1* gene was cloned and sequenced (Schmid et al., 1997).

this region, *meta* cleavage compound hydrolase (bphD) is located directly downstream of bphC1 gene, although the nucleotide sequence of 3′-terminal region has not been determined completely. Interestingly, no other gene encoding the biphenyl-degrading enzyme could be detected upstream of bphC1, indicating that this bphC1D cistron is not part of a complete bph operon, as found in other bacteria (Furukawa, 2000; Higson, 1992). Upstream of the bphC1 gene, there is a DNA region with 26.4% identity to ISAE1, an insertion element of Alcaligenes eutrophus strain H1 (Kung et al., 1992), including a 33-bp-long inverted repeat.

3. Variation of Angular Dioxygenase for FN Analogues

As mentioned above, four angular dioxygenase systems (CAR 1,9a-dioxygenase from *P.* resinovorans strain CA10, dioxin dioxygenase from *S.* wittichii strain RW1, DF 4,4a-dioxygenase from *Terrabacter* sp. strain DBF63, and POB dioxygenase from *P.* pseudoalcaligenes strain POB310) have been isolated and characterized. While the detailed analysis for substrate specificity of POB dioxygenase has not been reported, the substrate specificity of cloned or purified angular dioxygenase for FN analogues has been determined. Although the gene encoding angular dioxygenase has not been cloned, angular dioxygenases of several bacteria have been characterized by the analyses of oxygen uptake and co-metabolic capacity. These angular dioxygenases can be characterized by their substrate specificity for possible substrates.

Table 2 summarizes the oxygenations catalyzed by several angular dioxygenases specified for FN analogues. Based on the identification of oxygenation product after biotransformation experiment, CAR 1,9a-dioxygenase from *P.* resinovorans strain CA10 was shown to catalyze the angular dioxygenation for CAR, DF, DD, xanthene, and phenoxathiin, but not for 9-fluorenone (Nojiri et al., 1999; Sato et al., 1997a). Thus, when FN-analogous compounds served as its substrate, the angular dioxygenation was shown to occur at the angular position adjacent to the oxygen or nitrogen atom, but not to sulfur or carbon atom. The above results might indicate that the substrate specificity of CAR 1,9a-dioxygenase is different from those of angular dioxygenases from *S.* wittichii strain RW1, *Terrabacter* sp. strain DBF63, *Terrabacter* sp. strain DPO1361, and *Pseudomonas* sp. strain F274. Dioxin dioxygenase from *S.* wittichii strain RW1 was reported to catalyze the angular dioxygenation of DF and DD, but not of CAR (Armengaud et al., 1998; Bünz and Cook, 1993). This angular dioxygenase was also presumed to catalyze the angular dioxygenation of 9-fluorenone to yield 1-hydro-1,1a-dihydroxy-9-fluorenone (Bünz and Cook, 1993). Using the *E.* coli electron transport components that can fortuitously donate electrons to the DbfA1A2 complex, DbfA1A2 could catalyze the angular dioxygenation of DF and DD, and monooxygenation of FN to 9-fluorenone, but not angular dioxygenation of CAR, xanthene, or phenoxathiin (Kasuga et al., 2001). Although the chemical structure of the oxygenation product from 9-fluorenone has not been determined, it is quite possible that this compound is the angular diol of 9-fluorenone, 1-hydro-1,1a-dihydroxy-9-fluorenone (Kasuga et al., 2001). It was also reported that *Pseudomonas* sp. strain F274 could catalyze angular dioxygenation for DF and 9-fluorenone but not for CAR, although this transformation was performed by the whole cell (Grifoll et al., 1994, 1995). Based on the above results, the above angular dioxygenases could be divided into at least two types. Type-I is CAR 1,9a-dioxygenase of strain CA10, which can catalyze the angular dioxygenation for CAR and DF, but for 9-fluorenone. The representative of Type-II is dioxin dioxygenase of strain RW1, which catalyzes the angular dioxygenation for DF and 9-fluorenone, but for CAR.

The above angular dioxygenase is presumed not to be able to catalyze the angular dioxygenation for DBT-sulfone, or has not been tested for the oxygenation capacity for this compound. On the other hand, *Brevibacterium* sp. strain DO was presumed to be able to catalyze the angular dioxygenation for DBT-sulfone and 9-fluorenone but not for DF, although this transformation was performed by the whole cell (van Afferden et al., 1993). This tendency is markedly different from the above-mentioned angular dioxygenases, because DF can be oxygenated at an angular position by these dioxygenases. To characterize the DBT-sulfone 4,4a-dioxygenase, more detailed investigation of substrate specificity is necessary. In addition, the determination of dioxygenase activity for DBT-sulfone of other angular dioxygenases is indispensable.

Recently, Bressler and Fedorak (2000) pointed out that angular dioxygenation seems to be favored by a high electronegativity of one of the neighbors of the angular carbons, suggesting that formation of a nega-

Table 2. Capacity to catalyze the oxygenation for aromatic compounds of angular dioxygenases or angular dioxygenase-bearing bacteria.

| Angular dioxygenase or angular dioxygenase-bearing bacteria | Angular dioxygenation | Monoxygenation | Lateral dioxygenation |
|----------------------------------------------------------|-----------------------|-----------------|-----------------------|
| CAR 1,9a-dioxygenase (CA10)                              | DF                    | CAR             | 9-Fluorenone          | DBT-sulfone           | Naphthalene           | Biphenyl              |
| Dioxin dioxygenase (RW1)                                  | +                     | +               | +                     | +                     | -                     | +                     |
| DF 4,4a-dioxygenase (DBF63)                               | +                     | +               | -                     | +                     | -                     | +                     |
| Biotransformation using whole cell of wild-type strain    | +                     | +               | -                     | +                     | N.T.$^a$              | N.T.$^a$              |
| Pseudomonas sp. strain F274 (9-Fluorenone 1,1a-dioxygenase) | +                     | N.T.$^a$        | -                     | -                     | N.T.$^a$              | N.T.$^a$              |
| Terrabacter sp. strain DPO1361 (DF 4,4a-dioxygenase)       | +                     | N.T.$^a$        | N.T.$^a$              | +                     | N.T.$^a$              | N.T.$^a$              |
| Brevibacterium sp. strain DO (DBT sulfone 4,4a-dioxygenase) | -                     | N.T.$^a$        | N.T.$^a$              | +                     | +                     | -                     |
| Arthrobacter sp. strain DBTS2 (DBT sulfone 4,4a-dioxygenase) | N.T.$^a$              | N.T.$^a$        | N.T.$^a$              | +                     | -                     | N.T.$^a$              |

**Notes:**

$a$ N.T.: Not tested.

$b$ Inability was proposed, because no hydroxylation product other than DBT-sulfone and sulfone was detected by biotransformation experiments of DBT.

$c$ Biphenyl was transformed to an unidentified product.

$d$ The product is proposed to be 1-hydro-1,1a-dihydroxy-9-fluorenone or 1,2-dihydro-1,2-dihydroxy-9-fluorenol.

tively charged reaction intermediate is favored by electron-withdrawing groups at this position. In fact, the methylene carbon atom in FN and sulfur atom in DBT must be oxidized prior to attack by angular dioxygenases as shown above (Figs. 4 and 7). The addition of the oxygen atoms increases the electron-withdrawing capability of the central ring, facilitating the addition of electrons by the angular dioxygenases. In addition to the electric effect from the hetero atoms or functional groups in a five-membered ring, the difference in three-dimensional structure can result in the differences in the “fit” of the substrates to the substrate pocket of angular dioxygenase. While DF, CAR, and DBT have a nearly planar triple-ring structure, two aromatic rings in FN are twisted with each other. The volumes of the heteroatoms or corresponding functional groups might affect the reactivity of respective substrates in relation to angular dioxygenase.

As is well known, various Rieske nonheme iron oxygenases have relaxed substrate specificity, and can catalyze various oxygenation for various aromatic compounds (Gibson and Parales, 2000; Hudlicky et al., 1999; Resnick et al., 1996). In addition to the angular dioxygenation, CAR 1,9a-dioxygenase catalyzed lateral dioxygenation of PAHs and biphenyl (Nojiri et al., 1999) (Table 2). Naphthalene and biphenyl are converted to *cis*-1,2-dihydro-1,2-dihydroxynaphthalene and *cis*-2,3-dihydro-2,3-dihydroxybiphenyl by CAR 1,9a-dioxygenase, respectively. On the other hand, CAR 1,9a-dioxygenase also catalyzes the monooxygenation of sulfur heteroatoms in DBT and of benzyllic methylenic group in FN to yield DBT-sulfoxide and 9-fluorenol, respectively (Nojiri et al., 1999). These findings indicate that CAR 1,9a-dioxygenase of *P.* resinovorans strain CA10 has a broad substrate range, and can catalyze diverse oxygenation: Angular dioxygenation, lateral dioxygenation, and monooxygenation. The diverse oxygenation catalyzed by CAR 1,9a-dioxygenase for several aromatic compounds might reflect the differences in the “fit” of the substrates to the reaction center of CAR 1,9a-dioxygenase.

Sulfoxidation of DBT to DBT-sulfoxide and DBT-sulfone was presumed to be catalyzed by dioxin dioxygenase from strain RW1, while no oxygenation product from FN by dioxin dioxygenase was detected (Bünz and Cook, 1993) (Table 2). *Pseudomonas* sp. strain F247 has been reported to catalyze sulfoxidation of DBT to sulfoxide and sulfone, and monooxygenation of fluorene to 9-fluorenol (Grifall et al., 1994), though

there has been no evidence that the angular dioxygenase from strain F247 actually catalyzed the monooxygenation. *Arthrobacter* sp. strain DBTS2 and *P.* mendocina strain MC2 can grow on DBT-sulfone and 9-fluorenone, respectively, via angular dioxygenation (Casellas et al., 1998; Dahlberg et al., 1993). These two bacteria can not oxidize DBT or FN, respectively. Considering these facts, it might be possible that the initial oxidation of sulfur and carbon atom in DBT and FN, respectively, was catalyzed by different oxygenases from angular dioxygenases in DBT- and/or FN-degrading bacteria.

### Conclusion

In the last decade, extensive work on microbial degradation of FN analogues and their related compounds revealed degradation pathways containing a novel mode of dioxygenation, called angular dioxygenation. Except for 9-fluorenone, angular dioxygenation for FN analogues, such as DF, DD, CAR, and DBT-sulfone, results in the spontaneous cleavage of the three-ring structure. Namely, the single step enzymatic oxidation destroys the planar structures of these compounds. Resultantly formed 2′-substituted 2,3-dihydroxybiphenyls are degraded to monocyclic aromatic compounds by *meta* cleavage and hydrolysis. Thus, after the novel angular dioxygenation, subsequent degradation pathways of FN analogues are homologous to corresponding parts of that of biphenyl.

Recently, biphenyl 2,3-dioxygenase, the initial dioxygenase in the biphenyl degradation pathway of *Burkholderia* sp. strain LB400, has been reported to catalyze the angular dioxygenation for DD and DF (Seeger et al., 2001). Suenaga et al. (2001) reported that point mutation-introduced biphenyl 1,2-dioxygenase of *P.* pseudoalcaligenes strain KF707 could catalyze angular dioxygenation for DD and DF. These results clearly indicate that the mode of reaction catalyzed by angular dioxygenase is novel, but that this novel dioxygenation is one of the oxygenation reactions originating from the “relaxed” substrate specificity of the Rieske nonheme iron oxygenase super family. This hypothesis is in accordance with the fact that the angular dioxygenases do not constitute a cluster in the phylogenetic tree as shown in Fig. 9. Further investigation of the substrate specificity of various dioxygenases including angular dioxygenases is considered to be indispensable for better understanding of the char-

acteristics of angular dioxygenase. Recently, the three-dimensional structure of the terminal oxygenase component of naphthalene 1,2-dioxygenase has been reported (Carredano et al., 2000; Kauppi et al., 1998). The structure of angular dioxygenase will provide us useful information on the substrate binding and activation of angular dioxygenation.

It was revealed that CAR-degrading car gene cluster of *P. resinovorans* strain CA10 have a novel mosaic structure, and that the genes responsible for DD/DF catabolism by *S. wittichii* strain RW1 and *Terrabacter* sp. strain DBF63 are dispersed on the genome. These results suggest the possibility that the respective genetic elements constituting the entire catabolic pathways have been recruited from various other bacteria and/or other genetic loci, and that these pathways have not been evolutionarily matured.

### References

- van Afferden, M., Schacht, S., Beyer, M., and Klein, J. (1988) Microbial desulfurization of dibenzothiophene. *Am. Chem. Soc. Div. Fuel Chem.*, **33**, 561–572.
- van Afferden, M., Schacht, S., Klein, J., and Trüper, H. G. (1990) Degradation of dibenzothiophene by *Brevibacterium* sp. DO. *Arch. Microbiol.*, **153**, 324–328.
- van Afferden, M., Tappe, D., Beyer, M., Trüper, H. G., and Klein, J. (1993) Biochemical mechanisms for the desulfurization of coal-relevant organic sulfur compounds. *Fuel*, **72**, 1635–1643.
- Arcos, J. C. and Argus, M. F. (1968) Molecular geometry and carcinogenic activity of aromatic compounds. New perspectives. *Adv. Cancer Res.*, **11**, 305–471.
- Armengaud, J., Gaillard, J., and Timmis, K. N. (2000) A second [2Fe-2S] ferredoxin from *Sphingomonas* sp. strain RW1 can function as an electron donor for the dioxin dioxygenase. *J. Bacteriol.*, **182**, 2238–2244.
- Armengaud, J., Happe, B., and Timmis, K. N. (1998) Genetic analysis of dioxin dioxygenase of *Sphingomonas* sp. strain RW1: Catabolic genes dispersed on the genome. *J. Bacteriol.*, **180**, 3954–3966.
- Armengaud, J. and Timmis, K. N. (1997) Molecular characterization of Fdx1, a putidaredoxin-type [2Fe-2S] ferredoxin able to transfer electrons to the dioxin dioxygenase of *Sphingomonas* sp. RW1. *Eur. J. Biochem.*, **247**, 833–842.
- Armengaud, J. and Timmis, K. N. (1998) The reductase RedA2 of the multi-component dioxin dioxygenase system of *Sphingomonas* sp. RW1 is related to class-I cytochrome *P*~450~-type reductases. *Eur. J. Biochem.*, **253**, 437–444.
- Armengaud, J., Timmis, K. N., and Wittich, R.-M. (1999) A functional 4-hydroxysalicylate/hydroxyquinol degradative pathway gene cluster is linked to the initial dibenzo-*p*-dioxin pathway genes in *Sphingomonas* sp. strain RW1. *J. Bacteriol.*

- riol., **181**, 3452–3461.
- Batie, C. J., Ballou, D. P., and Correll, C. C. (1991) Phthalate dioxygenase reductase and related flavin-iron-sulfur containing electron transferases. *In Chemistry and Biochemistry of Flavoenzymes*, Vol. 3., ed. by Müller, F., CRC Press, Boca Raton, Fla., pp. 543–556.
- Becker, M., Phillips, T., and Safe, S. (1991) Polychlorinated diphenyl ethers—A review. *Toxicol. Environ. Chem.*, **33**, 189–200.
- Bertini, I., Capozzi, F., Dikiy, A., Happe, B., Luchinat, C., and Timmis, K. N. (1995) Evidence of histidine coordination to the catalytic ferrous ion in the ring-cleaving 2,2′,3-trihydroxybiphenyl dioxygenase from the dibenzofuran-degrading bacterium *Sphingomonas* sp. strain RW1. *Biochem. Biophys. Res. Commun.*, **215**, 855–860.
- Bosch, R., García-Valdés, E., and Moore, E. R. B. (1999) Genetic characterization and evolutionary implications of a chromosomally encoded naphthalene-degradation upper pathway from *Pseudomonas stutzeri* AN10. *Gene*, **236**, 149–157.
- Bressler, D. C. and Fedorak, P. M. (2000) Bacterial metabolism of fluorene, dibenzofuran, dibenzothiophene, and carbazole. *Can. J. Microbiol.*, **46**, 397–409.
- Brunel, F. and Davison, J. (1988) Cloning and sequencing of *Pseudomonas* genes encoding vanillate demethylase. *J. Bacteriol.*, **170**, 4924–4930.
- Bünz, P. V. and Cook, A. M. (1993) Dibenzofuran 4,4a-dioxygenase from *Sphingomonas* sp. strain RW1: Angular dioxygenation by a three-component enzyme system. *J. Bacteriol.*, **175**, 6467–6475.
- Bünz, P. V., Falchetto, R., and Cook, A. M. (1993) Purification of two isofunctional hydrolases (EC 3.7.1.8) in the degradative pathway for dibenzofuran in *Sphingomonas* sp. strain RW1. *Biodegradation*, **4**, 171–178.
- Carredano, E., Karlsson, A., Kauppi, B., Choudhury, D., Parales, R. E., Parales, J. V., Lee, K., Gibson, D. T., Eklund, H., and Ramaswamy, S. (2000) Substrate binding site of naphthalene 1,2-dioxygenase: Functional implications of indole binding. *J. Mol. Biol.*, **296**, 701–712.
- Casellas, M., Grifoll, M., Bayona, J. M., and Solanas, A. M. (1997) New metabolites in the degradation of fluorene by *Arthrobacter* sp. strain F101. *Appl. Environ. Microbiol.*, **63**, 819–826.
- Casellas, M., Grifoll, M., Sabaté, J., and Solanas, A. M. (1998) Isolation and characterization of a 9-fluorenone-degrading bacterial strain and its role in synergistic degradation of fluorene by a consortium. *Can. J. Microbiol.*, **44**, 734–742.
- Cerniglia, C. E. (1992) Biodegradation of polycyclic aromatic hydrocarbons. *Biodegradation*, **3**, 351–368.
- Correll, C. C., Batie, C. J., Ballou, D. P., and Ludwig, M. L. (1992) Phthalate dioxygenase reductase: A modular structure for electron transfer from pyridine nucleotides to [2Fe-2S]. *Science*, **258**, 1604–1610.
- Dahlberg, M. D., Rohrer, R. L., Fauth, D. J., Sprecher, R., and Olson, G. J. (1993) Biodesulfurization of dibenzothiophene

sulfone by *Arthrobacter* sp. and studies with oxidized Illinois No. 6 coal. *Fuel*, **72**, 1645–1649.

Danforth, C., Nicholson, A. W., James, J. C., and Loudon, G. M. (1976) Steric acceleration of lactonization reaction: An analysis of “stereopopulation control.” *J. Am. Chem. Soc.*, **98**, 4275–4281.

Danz, M., Urban, H., and Schmidt, A. (1978) A possible short-term prediction of potential carcinogenicity of chemical compounds in vivo by means of promoting activity test (PAT). *Exp. Pathol.*, **16**, 109–120.

Dehmel, U., Engesser, K. H., Timmis, K. N., and Dwyer, D. F. (1995) Cloning, nucleotide sequence, and expression of the gene encoding a novel dioxygenase involved in metabolism of carboxydiphenyl ethers in *Pseudomonas pseudoalcaligenes* POB310. *Arch. Microbiol.*, **163**, 35–41.

Engesser, K. H., Fietz, W., Fischer, P., Schulte, P., and Knackmuss, H.-J. (1990) Dioxygenolytic cleavage of aryl ether bonds: 1,2-Dihydro-1,2-dihydroxy-4-carboxybenzophenone as evidence for initial 1,2-dioxygenation in 3- and 4-carboxy biphenyl ether degradation. *FEMS Microbiol. Lett.*, **69**, 317–322.

Engesser, K. H., Strubel, V., Christglou, K., Fischer, P., and Rast, H. G. (1989) Dioxygenolytic cleavage of aryl ether bonds: 1,10-Dihydro-1,10-dihydroxyfluoren-9-one, a novel arene dihydrodiol as evidence for angular dioxygenation of dibenzofuran. *FEMS Microbiol. Lett.*, **65**, 205–210.

Fortnagel, P., Harms, H., Wittich, R.-M., Krohn, S., Meyer, H., Sinnwell, V., Wilkes, H., and Francke, W. (1990) Metabolism of dibenzofuran by *Pseudomonas* sp. strain HH69 and the mixed culture HH27. *Appl. Environ. Microbiol.*, **56**, 1148–1156.

Fortnagel, P., Wittich, R.-M., Harms, H., Schmidt, S., Franke, S., Sinnwell, V., Wilkes, H., and Francke, W. (1989) New bacterial degradation of the biaryl ether structure. *Naturwissenschaften*, **76**, 523–524.

Francis, B. M. (1986) Teratogenicity of bifenox and nitrofen in rodents. *J. Environ. Sci. Health*, **B21**, 303–317.

Furukawa, K. (2000) Biochemical and genetic bases of microbial degradation of polychlorinated biphenyls (PCBs). *J. Gen. Appl. Microbiol.*, **46**, 283–296.

Furukawa, K., Arimura, N., and Miyazaki, T. (1987) Nucleotide sequence of the 2,3-dihydroxybiphenyl dioxygenase gene of *Pseudomonas pseudoalcaligenes*. *J. Bacteriol.*, **169**, 427–429.

Gibson, D. T. (1988) Microbial metabolism of aromatic hydrocarbons and the carbon cycle. *In Microbial Metabolism and the Carbon Cycle*, ed. by Hagedorn, S. R., Hudson, R. S., and Kunz, D. A., Harwood Academic Publishers, New York, pp. 33–58.

Gibson, D. T. and Parales, R. E. (2000) Aromatic hydrocarbon dioxygenases in environmental biotechnology. *Curr. Opin. Biotechnol.*, **11**, 236–243.

Gieg, L. M., Otter, A., and Fedrak, P. M. (1996) Carbazole degradation by *Pseudomonas* sp. LD2: Metabolic characteristics and the identification of some metabolites. *Environ.*

Sci. Technol., **30**, 575–585.

Grifoll, M., Casellas, M., Bayona, J. M., and Solanas, A. M. (1992) Isolation and characterization of fluorene-degrading bacterium: Identification of ring oxidation and ring fission products. *Appl. Environ. Microbiol.*, **58**, 2910–2917.

Grifoll, M., Selifonov, S. A., and Chapman, P. J. (1994) Evidence for a novel pathway in the degradation of fluorene by *Pseudomonas* sp. strain F274. *Appl. Environ. Microbiol.*, **60**, 2438–2449.

Grifoll, M., Selifonov, S. A., and Chapman, P. J. (1995) Transformation of substituted fluorenes and fluorene analogs by *Pseudomonas* sp. strain F274. *Appl. Environ. Microbiol.*, **61**, 3490–3493.

Grosser, R. J., Warshawsky, D., and Vestal, J. R. (1991) Indigenous and enhanced mineralization of pyrene, benzo[a]-pyrene, and carbazole in soils. *Appl. Environ. Microbiol.*, **57**, 3462–3469.

Habe, H., Chung, J.-S., Lee, J.-H., Kasuga, K., Yoshida, T., Nojiri, H., and Omori, T. (2001a) Degradation of chlorinated dibenzofurans and dibenzo-*p*-dioxins by two types of bacteria having angular dioxygenases with different features. *Appl. Environ. Microbiol.*, **67**, 3610–3617.

Habe, H., Ide, K., Yotsumoto, M., Tsuji, H., Hirano, H., Widada, J., Yoshida, T., Nojiri, H., and Omori, T. (2001b) Preliminary examinations for applying a carbazole-degrader, *Pseudomonas* sp. strain CA10, to dioxin-contaminated soil remediation. *Appl. Microbiol. Biotechnol.*, **56**, 788–795.

Halden, R. U., Halden, B. G., and Dwyer, D. F. (1999) Removal of dibenzofuran, dibenzo-*p*-dioxin, and 2-chlorodibenzo-*p*-dioxin from soils inoculated with *Sphingomonas* sp. strain RW1. *Appl. Environ. Microbiol.*, **65**, 2246–2249.

Halden, R. U., Peters, E. G., Halden, B. G., and Dwyer, D. F. (2000) Transformation of mono- and dichlorinated phenoxybenzoates by phenoxybenzoate-dioxygenase in *Pseudomonas pseudoalcaligenes* POB310 and a modified diarylether-metabolizing bacterium. *Biotechnol. Bioeng.*, **69**, 107–112.

Happe, B., Eltis, L. D., Poth, H., Hedderich, R., and Timmis, K. N. (1993) Characterization of 2,2′,3-trihydroxybiphenyl dioxygenase, an extradiol dioxygenase from the dibenzofuran- and dibenzo-*p*-dioxin-degrading bacterium *Sphingomonas* sp. strain RW1. *J. Bacteriol.*, **175**, 7313–7320.

Harms, H., Wilkes, H., Sinnwell, V., Wittich, R.-M., Figge, K., Francke, W., and Fortnagel, P. (1991) Transformation of 3-chlorodibenzofuran by *Pseudomonas* sp. strain HH69. *FEMS Microbiol. Lett.*, **81**, 25–30.

Harms, H., Wilkes, H., Wittich, R.-M., and Fortnagel, P. (1995) Metabolism of hydroxydibenzofurans, methoxydibenzofurans, acetoxydibenzofurans, and nitrodibenzofurans by *Sphingomonas* sp. strain HH69. *Appl. Environ. Microbiol.*, **61**, 2499–2505.

Harms, H., Wittich, R.-M., Sinnwell, V., Meyer, H., Fortnagel, P., and Francke, W. (1990) Transformation of dibenzo-*p*-dioxin by *Pseudomonas* sp. strain HH69. *Appl. Environ. Microbiol.*, **56**, 1157–1159.

Harms, H. and Zehnder, A. J. B. (1994) Influence of substrate diffusion on degradation of dibenzofuran and 3-chlorodibenzofuran by attached and suspended bacteria. Appl. Environ. Microbiol., 60, 2736–2745.

Harms, H. and Zehnder, A. J. B. (1995) Bioavailability of sorbed 3-chlorodibenzofuran. Appl. Environ. Microbiol., 61, 27–33.

Hayase, N., Taira, K., and Furukawa, K. (1990) *Pseudomonas putida* KF715 *bphABCD* operon encoding biphenyl and polychlorinated biphenyl degradation: Cloning, analysis, and expression in soil bacteria. J. Bacteriol., 172, 1160–1164.

Higson, F. K. (1992) Microbial degradation of biphenyl and its derivatives. Adv. Appl. Microbiol., 37, 135–164.

Hisatsuka, K. and Sato, M. (1994) Microbial transformation of carbazole to anthranilic acid by *Pseudomonas stutzeri*. Biosci. Biotechnol. Biochem., 58, 213–214.

Hofer, B., Eltis, L. D., Dowling, D. N., and Timmis, K. N. (1993) Genetic analysis of a *Pseudomonas* locus encoding a pathway for biphenyl/polychlorinated biphenyl degradation. Gene, 130, 47–55.

Hoffman, D. J., Spann, J. W., LeCaptain, L. J., Bunck, C. M., and Rattner, B. A. (1991) Developmental toxicity of diphenyl ether herbicides in nestling American kestrels. J. Toxicol. Environ. Health, 34, 323–336.

Hsieh, Y., Thompson, M. B., and Ward, C. H. (1980) Toxicity of water soluble extracts of No. 2 fuel oil to the freshwater alga *Selenastrum capricornutum*. Dev. Ind. Microbiol., 21, 401–409.

Hudlicky, T., Gonzalez, D., and Gibson, D. T. (1999) Enzymatic dihydroxylation of aromatics in enantioselective synthesis: Expanding asymmetric methodology. Aldrichimica Acta, 32, 35–62.

Jeenes, D. J. and Williams, P. A. (1982) Excision and integration of degradative pathway genes from TOL plasmid pWW0. J. Bacteriol., 150, 188–194.

Ka, J. O. and Tiedje, J. M. (1994) Integration and excision of a 2,4-dichlorophenoxyacetic acid-degradative plasmid in *Alcaligenes paradoxus* and evidence of its natural intergeneric transfer. J. Bacteriol., 176, 5284–5289.

Kasuga, K., Habe, H., Chung, J.-S., Yoshida, T., Nojiri, H., Yamane, H., and Omori, T. (2001) Isolation and characterization of the genes encoding a novel oxygenase component of angular dioxygenase from the gram-positive dibenzofuran-degrader *Terrabacter* sp. strain DBF63. Biochem. Biophys. Res. Commun., 283, 195–204.

Kasuga, K., Nojiri, H., Yamane, H., Kodama, T., and Omori, T. (1997) Cloning and characterization of the genes involved in the degradation of dibenzofuran by *Terrabacter* sp. strain DBF63. J. Ferment. Bioeng., 84, 387–399.

Kauppi, B., Lee, K., Carredano, E., Parales, R. E., Gibson, D. T., Eklund, H., and Ramaswamy, S. (1998) Structure of an aromatic ring-hydroxylating dioxygenase-naphthalene 1,2-dioxygenase. Structure, 6, 571–586.

Keim, T., Francke, W., Schmidt, S., and Fortnagel, P. (1999) Catabolism of 2,7-dichloro- and 2,4,8-trichlorodibenzofuran by

Sphingomonas sp strain RW1. J. Ind. Microbiol. Biotechnol., 23, 359–363.

Kieth, L. H. and Teillard, W. A. (1979) Priority pollutants I—A perspective view. Environ. Sci. Technol., 13, 416–423.

Kimura, T., Zhang, Y., Kodama, T., and Omori, T. (1996) Isolation and characterization of Tn5-induced mutants deficient in carbazole catabolism. FEMS Microbiol. Lett., 135, 65–70.

Kirimura, K., Nakagawa, H., Tsuji, K., Matsuda, K., Kurane, R., and Usami, S. (1999) Selective and continuous degradation of carbazole contained in petroleum oil by resting cells of *Sphingomonas* sp. CDH-7. Biosci. Biotechnol. Biochem., 63, 1563–1568.

Kodama, K., Umehara, K., Shimizu, K., Nakatani, S., Minoda, Y., and Yamada, K. (1973) Identification of microbial products from dibenzothiophene and its proposed oxidation pathway. Agric. Biol. Chem., 37, 45–50.

Kohler, H.-P. E., Schmid, A., and van der Maarel, M. (1993) Metabolism of 2,2′-dihydroxybiphenyl by *Pseudomonas* sp. strain HBP1: Production and consumption of 2,2′,3-trihydroxybiphenyl. J. Bacteriol., 175, 1621–1628.

Kung, S.-S., Chen, J., and Chow, W.-Y. (1992) Molecular and genetic characterization of an *Alcaligenes eutrophus* insertion element. J. Bacteriol., 174, 8023–8029.

Later, D. W., Lee, M. L., Bartle, K. D., Kong, R. C., and Vassilaros, D. L. (1981) Class separation and characterization of organic compounds in synthetic fuels. Anal. Chem., 53, 1612–1620.

Mahillon, J. and Chandler, M. (1998) Insertion sequences. Microbiol. Mol. Biol. Rev., 62, 725–774.

Mason, J. R. and Cammack, R. (1992) The electron-transport proteins of hydroxylating bacterial dioxygenases. Annu. Rev. Microbiol., 46, 277–305.

McFarland, B. L. (1999) Biodesulfurization. Curr. Opin. Microbiol., 2, 257–264.

Megharaj, M., Wittich, R.-M., Blasco, R., Pieper, D. H., and Timmis, K. N. (1997) Superior survival and degradation of dibenzo-p-dioxin and dibenzofuran in soil by soil-adapted *Sphingomonas* sp. strain RW1. Appl. Microbiol. Biotechnol., 48, 109–114.

Meulien, P. and Broda, P. (1982) Identification of chromosomally integrated TOL DNA in cured derivatives of *Pseudomonas putida* PAW1. J. Bacteriol., 152, 911–914.

Monna, L., Omori, T., and Kodama, T. (1993) Microbial degradation of dibenzofuran, fluorene, and dibenzo-p-dioxin by *Staphylococcus auricularans* DBF63. Appl. Environ. Microbiol., 59, 285–289.

Munro, A. W. and Lindsay, J. G. (1996) Bacterial cytochromes P-450. Mol. Microbiol., 20, 1115–1125.

Nam, J.-W., Nojiri, H., Yoshida, T., Habe, H., Yamane, H., and Omori, T. (2001) New classification system for oxygenase components involved in ring-hydroxylating oxygenations. Biosci. Biotechnol. Biochem., 65, 254–263.

Neidle, E. L., Hartnett, C., Ornston, L. N., Bairoch, A., Rekik, M., and Harayama, S. (1991) Nucleotide sequences of the

Acinetobacter calcoaceticus benABC genes for benzoate 1,2-dioxygenase reveal evolutionary relationships among multicomponent oxygenases. J. Bacteriol., 173, 5385–5395.

Nojiri, H., Maeda, K., Sekiguchi, H., Urata, M., Shintani, M., Yoshida, T., Habe, H., and Omori, T. Organization and transcriptional characterization of catechol degradation genes involved in carbazole degradation by Pseudomonas resinovorans strain CA10. Biosci. Biotechnol. Biochem. (in press).

Nojiri, H., Nam, J.-W., Kosaka, M., Morii, K., Takemura, T., Furihata, K., Yamane, H., and Omori, T. (1999) Diverse oxidations catalyzed by carbazole 1,9a-dioxygenase from Pseudomonas sp. strain CA10. J. Bacteriol., 181, 3105–3113.

Nojiri, H., Sekiguchi, H., Maeda, K., Urata, M., Nakai, S., Yoshida, T., Habe, H., and Omori, T. (2001) Genetic characterization and evolutionary implications of a car gene cluster in the carbazole degrader Pseudomonas sp. strain CA10. J. Bacteriol., 183, 3663–3679.

Nomura, Y., Nakagawa, M., Ogawa, N., Harashima, S., and Oshima, Y. (1992) Genes in PHT plasmid encoding the initial degradation pathway of phthalate in Pseudomonas putida. J. Ferment. Bioeng., 74, 333–344.

Ochman, M., Klubek, B., Boydstun, J., Clark, D., and Nabe, S. (1990) Mineralization of S from dibenzothiophene, dibenzothiophene sulphone and benzene sulphonic acid by soil isolates. Microbios, 63, 79–91.

Ohshiro, T. and Izumi, Y. (1999) Microbial desulfurization of organic sulfur compounds in petroleum. Biosci. Biotechnol. Biochem., 63, 1–9.

Ohtsubo, Y., Nagata, Y., Kimbara, K., Takagi, M., and Ohta, A. (2000) Expression of the bph genes involved in biphenyl/PCB degradation in Pseudomonas sp. KKS102 induced by the biphenyl degradation intermediate, 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoic acid. Gene, 256, 223–228.

Otaka, E. and Ooi, T. (1989) Examination of protein sequence homologies. V. New perspectives on evolution between bacterial and chloroplast-type ferredoxins inferred from sequence evidence. J. Mol. Evol., 29, 246–254.

Ouchiyama, N., Miyachi, S., and Omori, T. (1998) Cloning and nucleotide sequence of carbazole catabolic genes from Pseudomonas stutzeri strain OM1, isolated from activated sludge. J. Gen. Appl. Microbiol., 44, 57–63.

Ouchiyama, N., Zhang, Y., Omori, T., and Kodama, T. (1993) Biodegradation of carbazole by Pseudomonas spp. CA06 and CA10. Biosci. Biotechnol. Biochem., 57, 455–460.

Peterson, J. A., Lorence, M. C., and Amarneh, B. (1990) Putaredoxin reductase and putiredoxin. Cloning, sequence determination, and heterologous expression of the proteins. J. Biol. Chem., 265, 6066–6073.

Pfeifer, F., Trüper, H. G., Klein, J., and Schacht, S. (1993) Degradation of diphenylether by Pseudomonas cepacia Et4: Enzymatic release of phenol from 2,3-dihydroxydiphenylether. Arch. Microbiol., 159, 323–329.


Resnick, S. M., Lee, K., and Gibson, D. T. (1996) Diverse reactions catalyzed by naphthalene dioxygenase from Pseudomonas sp. strain NCIB 9816. J. Ind. Microbiol., 17, 438–457.

Richardson, M. L. and Gangolli, M. L. (1994) The Dictionary of Substances and Their Effects. The Royal Society of Chemistry, Cambridge, U.K.

Santodonato, J. and Howard, P. H. (1981) Azaarenes: Sources, distribution, environmental impact, and health effects. In Hazard Assessment of Chemicals: Current Developments, ed. by Saxena, J. and Fisher, F., Academic Press, New York, pp. 421–440.

Sato, H. and Clark, D. P. (1995) Degradation of dibenzothiophene sulphoxide and sulphone by Arthrobacter strain DBTS2. Microbios, 83, 145–159.

Sato, S., Nam, J.-W., Kasuga, K., Nojiri, H., Yamane, H., and Omori, T. (1997a) Identification and characterization of genes encoding carbazole 1,9a-dioxygenase in Pseudomonas sp. strain CA10. J. Bacteriol., 179, 4850–4858.

Sato, S., Ouchiyama, N., Kimura, T., Nojiri, H., Yamane, H., and Omori, T. (1997b) Cloning of genes involved in carbazole degradation of Pseudomonas sp. strain CA10: Nucleotide sequences of genes and characterization of meta-cleavage enzymes and hydrolase. J. Bacteriol., 179, 4841–4849.

Schläfli, H. R., Baker, D. P., Leisinger, T., and Cook, A. M. (1995) Stereospecificity of hydride removal from NADH by reductases of multicomponent nonheme iron oxygenase systems. J. Bacteriol., 177, 831–834.

Schmid, A., Rothe, B., Altenbuchner, J., Ludwig, W., and Engesser, K. H. (1997) Characterization of three distinct extradiol dioxygenases involved in mineralization of dibenzofuran by Terrabacter sp. strain DPO360. J. Bacteriol., 179, 53–62.

Schmidt, S., Fortnagel, P., and Wittich, R.-M. (1993) Biodegradation and transformation of 4,4′- and 2,4-dihalodiphenyl ethers by Sphingomonas sp. strain SS33. Appl. Environ. Microbiol., 59, 3931–3933.

Schmidt, S., Wittich, R.-M., Erdmann, D., Wilkes, H., Francke, W., and Fortnagel, P. (1992a) Biodegradation of diphenyl ether and its monohalogenated derivatives by Sphingomonas sp. strain SS3. Appl. Environ. Microbiol., 58, 2744–2750.

Schmidt, S., Wittich, R.-M., Fortnagel, P., Erdmann, D., and Francke, W. (1992b) Metabolism of 3-methyl diphenyl ether by Sphingomonas sp. SS31. FEMS Microbiol. Lett., 96, 253–258.

Schneider, J., Grosser, R. J., Jayasimhulu, K., Xue, W., Kinkle, B., and Warshawsky, D. (2000) Biodegradation of carbazole by Ralstonia sp. RJGII.123 isolated from a hydrocarbon contaminated soil. Can. J. Microbiol., 46, 269–277.

Schreiner, G., Wiedmann, T., Schimmel, H., and Ballschmitter, K. (1997) Influence of the substitution pattern on the microbial degradation of mono- to tetrachlorinated dibenzo-p-dioxins and dibenzofurans. Chemosphere, 34, 1315–1331.

Seah, S. Y. K., Labbé, G., Nerdinger, S., Johnson, M. R., Snieckus, V., and Eltis, L. D. (2000) Identification of a serine hydrolase as a key determinant in the microbial degradation of polychlorinated biphenyls. *J. Biol. Chem.*, **275**, 15701–15708.

Seeger, M., Cámara, B., and Hofer, B. (2001) Dehalogenation, denitration, dehydroxylation, and angular attack on substituted biphenyls and related compounds by a biphenyl dioxygenase. *J. Bacteriol.*, **183**, 3548–3555.

Selifonov, S. A., Grifoll, M., Gurst, J. E., and Chapman, P. J. (1993) Isolation and characterization of (+)-1,1a-dihydroxy-1-hydrofluoren-9-one formed by angular dioxygenation in the bacterial catabolism of fluorene. *Biochem. Biophys. Res. Commun.*, **193**, 67–76.

Shepherd, J. M. and Lloyd-Jones, G. (1998) Novel carbazole degradation genes of *Sphingomonas* CB3: Sequence analysis, transcription, and molecular ecology. *Biochem. Biophys. Res. Commun.*, **247**, 129–135.

Shotbolt-Brown, J., Hunter, D. W. F., and Aislabie, J. (1996) Isolation and description of carbazole-degrading bacteria. *Can. J. Microbiol.*, **42**, 79–82.

Sinclair, M. I., Maxwell, P. C., Lyon, B. R., and Holloway, B. W. (1986) Chromosomal location of TOL plasmid DNA in *Pseudomonas putida*. *J. Bacteriol.*, **168**, 1302–1308.

Stay, F. S., Katko, A., Rohm, C. M., Feix, M. A., and Larsen, D. P. (1988) Effects of fluorene on microcosms developed from natural communities. *Environ. Toxicol. Chem.*, **7**, 635–644.

Strubel, V., Engesser, K. H., Fischer, P., and Knackmuss, H.-J. (1991) 3-(2-Hydroxyphenyl)catechol as substrate for proximal meta ring cleavage in dibenzofuran degradation by *Brevibacterium* sp. strain DPO 1361. *J. Bacteriol.*, **173**, 1932–1937.

Strubel, V., Rast, H. G., Fietz, W., Knackmuss, H.-J., and Engesser, K. H. (1989) Enrichment of dibenzofuran utilizing bacteria with high co-metabolic potential towards dibenzodioxin and other anellated aromatics. *FEMS Microbiol. Lett.*, **58**, 233–238.

Suenaga, H., Goto, M., and Furukawa, K. (2001) Emergence of multifunctional oxygenase activities by random priming recombination. *J. Biol. Chem.*, **276**, 22500–22506.

Takase, I., Omori, T., and Minoda, Y. (1986) Microbial degradation products from biphenyl-related compounds. *Agric. Biol. Chem.*, **50**, 681–686.

Thomas, P., Wofford, H. W., and Neff, J. M. (1981) Biochemical stress responses of striped mullet (*Mugil cephalus*) to fluorene analog. *Aquatic Toxicol.*, **1**, 329–342.

Trenz, S. P., Engesser, K. H., Fischer, P., and Knackmuss, H.-J. (1994) Degradation of fluorene by *Brevibacterium* sp. strain DPO 1361: A novel C-C bond cleavage mechanism via 1,10-dihydro-1,10-dihydroxyfluoren-9-one. *J. Bacteriol.*, **176**, 789–795.

Vasilieva, S., Tanirbergenov, B., Abilev, S., Migatchev, G., and Huttunen, M. T. (1990) A comparative study of mutagenic and SOS-inducing activity of biphenyls, phenanthrene-quinones, and fluorenones. *Mutat. Res.*, **244**, 321–330.

Wilkes, H., Francke, W., Wittich, R.-M., Harms, H., Schmidt, S., and Fortnagel, P. (1992) Mechanistic investigations on microbial degradation of diaryl ethers. Analysis of isotope-labeled reaction products. *Naturwissenschaften*, **79**, 269–271.

Wilkes, H., Wittich, R.-M., Timmis, K. N., Fortnagel, P., and Francke, W. (1996) Degradation of chlorinated dibenzofurans and dibenzo-p-dioxins by *Sphingomonas* sp. strain RW1. *Appl. Environ. Microbiol.*, **62**, 367–371.

Wittich, R.-M. (1998a) Aerobic degradation by bacteria of dibenzo-p-dioxin, dibenzofurans, diphenyl ethers and their halogenated derivatives. In *Biodegradation of Dioxins and Furans*, ed. by Wittich, R.-M., Springer-Verlag, Heidelberg, Germany, pp. 1–28.

Wittich, R.-M. (1998b) Degradation of dioxin-like compounds by microorganisms. *Appl. Microbiol. Biotechnol.*, **49**, 489–499.

Wittich, R.-M., Schmidt, S., and Fortnagel, P. (1990) Bacterial degradation of 3- and 4-carboxybiphenyl ether by *Pseudomonas* sp. NSS2. *FEMS Microbiol. Lett.*, **67**, 157–160.

Wittich, R.-M., Strömpl, C., Moore, E. R. B., Blasco, R., and Timmis, K. N. (1999) Interaction of *Sphingomonas* and *Pseudomonas* strains in the degradation of chlorinated dibenzofurans. *J. Ind. Microbiol. Biotechnol.*, **23**, 353–358.

Wittich, R.-M., Wilkes, H., Sinnwell, V., Francke, W., and Fortnagel, P. (1992) Metabolism of dibenzo-p-dioxin by *Sphingomonas* sp. strain RW1. *Appl. Environ. Microbiol.*, **58**, 1005–1010.

Yabuuchi, E., Yamamoto, H., Terakubo, S., Okamura, N., Naka, T., Fujiwara, N., Kobayashi, K., Kosako, Y., and Hiraishi, H. (2001) Proposal of *Sphingomonas wittichii* sp. nov. for strain RW1<sup>T</sup>, known as a dibenzo-p-dioxin metabolizer. *Int. J. Syst. Evol. Microbiol.*, **51**, 281–292.
