
Molecular Microbiology (2003) 47(3), 793–805

# A new *Escherichia coli* metabolic competency: growth on fatty acids by a novel anaerobic β-oxidation pathway

John W. Campbell,$^{1,2}$ Rachael M. Morgan-Kiss$^1$ and John E. Cronan, Jr$^{1,3*}$

$^1$ Department of Microbiology, University of Illinois at Urbana-Champaign, 601 S. Goodwin Ave., Urbana, IL 61801, USA.

$^2$ Integrated Genomics, Inc. Chicago, IL 60612, USA.

$^3$ Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA.

## Summary

*Escherichia coli* uses fatty acids as a sole carbon and energy source during aerobic growth by means of the enzymes encoded by the *fad* regulon. We report that this bacterium can also grow on fatty acids under anaerobic conditions provided that a terminal respiratory electron acceptor such as nitrate is available. This anaerobic utilization pathway is distinct from the well-studied aerobic pathway in that (i) it proceeds normally in mutant strains lacking various enzymes of the aerobic pathway; (ii) it functions with fatty acids (octanoate and decanoate) that cannot be used by wild-type *E. coli* strains under aerobic conditions; and (iii) super-repressor mutants of the *fadR* regulatory locus that block aerobic growth on fatty acids fail to block the anaerobic pathway. We have identified homologues of the FadA, FadB and FadD proteins required for aerobic fatty acid utilization called YfcY, YfcX and YdiD, respectively, which are involved in anaerobic growth on fatty acids. A strong FadR binding site was detected upstream of the *yfcY* gene consistent with microarray analyses, indicating that *yfcYX* expression is negatively regulated by FadR under aerobic growth conditions. In contrast, transcriptional regulation of *ydiD* appears to be independent of FadR, and anaerobic growth on fatty acids is not under FadR control. These three genes are conserved in the available genome sequences of pathogenic *E. coli*, *Shigella* and *Salmonella* strains.

Accepted 28 October, 2002. *For correspondence at the first address.* E-mail j-cronan@life.uiuc.edu; Tel. (+1) 217 333 7919; Fax (+1) 217 244 6697.

© 2003 Blackwell Publishing Ltd

## Introduction

Research in *Escherichia coli* has provided the most complete view of lipid metabolism in any organism. Aerobic fatty acid catabolism by this Gram-negative facultative anaerobe was originally described over 30 years ago (Overath and Raufuss, 1967; Overath et al., 1969). Since that time, many molecular details of how *E. coli* carries out and regulates this β-oxidation pathway have been described (Clark and Cronan, 1996). Under aerobic conditions, *E. coli* can use fatty acids of various chain lengths as sole carbon and energy sources. However, long-chain fatty acids (LCFAs) of at least 12 carbon atoms are required for induction of the catabolic enzymes. After uptake, fatty acids can either be degraded via the β-oxidation pathway or used as precursors in membrane phospholipid biosynthesis. The aerobic degradation pathway is catalysed by the enzymes encoded by the *fad* regulon, which are responsible for transport, activation and β-oxidative cleavage of LCFAs to acetyl-CoA (Fig. 1). Briefly, LCFAs are transported across the cell membrane via a transport/acyl-activation mechanism involving an outer membrane protein, FadL, and an acyl-CoA synthetase, FadD. Although eukaryotic systems possess multiple acyl-CoA synthetases with different fatty acid chain length specificities, *E. coli* has been reported to have a single acyl-CoA synthetase, FadD, of broad substrate specificity (Overath et al., 1969; Kameda and Nunn, 1981). The first step in the aerobic bacterial β-oxidation cycle involves conversion of acyl-CoA to enoyl-CoA via the FadE acyl-CoA dehydrogenase. Unlike mammalian mitochondria, which possess several chain length-specific acyl-CoA dehydrogenases, *E. coli* appears to have only one enzyme capable of catalysing the dehydrogenation step of the aerobic β-oxidation pathway (Campbell and Cronan, 2002). The remaining steps in fatty acid degradation are performed by a tetrameric complex consisting of two copies each of FadB and FadA (Pramanik et al., 1979). This complex has five separate enzymatic activities including the hydration, oxidation and thiolytic cleavage functions. The β-oxidation pathway acts in a cyclic fashion, in which each cycle results in shortening the input acyl-CoA by two carbon atoms to give acetyl-CoA (Fig. 1), which can be used via the TCA and glyoxylate cycles in

source by the aerobic pathway seems unlikely to occur. However, alternative electron acceptors, such as nitrate, fumarate and trimethylamine oxide, which support anaerobic respiration, may be available in the lumen. We therefore tested the ability of *E. coli* K-12 to grow on fatty acids in the presence of nitrate, fumarate or trimethylamine oxide and readily obtained growth. Although anaerobic β-oxidation of fatty acids has long been known for various obligate anaerobic rumen bacteria (Mackie et al., 1991), there is little information regarding either the enzymology or the regulation of fatty acid degradation during anaerobic growth of a facultative anaerobe such as *E. coli*. Indeed, Iuchi and Lin (1988) reported that expression of 3-hydroxyacyl-CoA dehydrogenase and, to a lesser extent, acyl-CoA dehydrogenase activity, as well as expression of isocitrate lyase, a major enzyme of the glyoxylate cycle, was repressed under anaerobic conditions via the ArcAB system, suggesting that *E. coli* may be unable to catabolize fatty acids under anaerobic conditions. Here, we report a novel fatty acid degradation pathway that functions under anaerobic growth conditions in the presence of nitrate as a terminal electron acceptor and have identified three genes that are involved in this new pathway. We also show that anaerobic growth on fatty acids is not strongly regulated by FadR, the *fad* regulon control protein that represses β-oxidation under aerobic conditions.

Fig. 1. The aerobic β-oxidation pathway of fatty acid degradation in *E. coli.*

the presence of electron acceptors, such as oxygen, nitrate, fumarate or triethylamine N-oxide (Clark, 1989).

Transcriptional control of the aerobic fatty acid degradation genes occurs via the FadR regulatory protein. In the absence of LCFAs, the FadR protein binds directly to operator DNA sequences, where it acts to repress transcription of the *fad* genes. When LCFAs are present in the growth medium, these are converted to long-chain acyl-CoAs, which bind to FadR resulting in release of the regulatory protein from the operator and thus derepression of transcription (Henry and Cronan, 1991; 1992). In addition, the genes encoding the glyoxylate shunt enzymes are in an operon that is indirectly regulated by FadR (Maloy et al., 1980; Maloy and Nunn, 1981; Gui et al., 1996). Although wild-type *E. coli* requires fatty acids that are 12 or more carbons long for the induction of metabolic enzymes, *fadR* mutants grow on shorter fatty acids such as decanoate and octanoate (Overath et al., 1969). In addition, many *fad* genes are positively regulated by the Crp/cyclic-AMP system (Pauli and Overath, 1974).

In the mammalian gut, a natural environment of *E. coli*, a variety of carbon and energy sources are readily available, but the oxygen supply is thought to be severely limited. Therefore, although the intestinal lumen may be rich in a variety of fatty acids, utilization of this carbon

Results

Two distinct sets of β-oxidative enzymes in *E. coli*

Our previous microarray experiments (Campbell and Cronan, 2001) showed that expression of two adjacent open reading frames (ORFs) of unknown function, *yfcX* and *yfcY*, were markedly increased upon aerobic growth of a wild-type strain in the presence of fatty acids. Moreover, a *fadR* null mutant strain showed constitutive expression of these genes. Examination of the proteins encoded by these ORFs showed that *yfcX* and *yfcY* have high levels of homology to *fadB* and *fadA* respectively. Other laboratories subsequently confirmed our expression results (Zhang et al., 2002) and also proposed that *yfcX* and *yfcY* function in β-oxidation (Haller et al., 2000). We therefore sought physiological evidence for the function of *yfcX* and *yfcY* in β-oxidation.

The first clue to the function of the *yfcX* and *yfcY* genes was the leaky phenotype associated with strains carrying the classical *fadA751::Tn10* mutation. After prolonged incubation (>5–7 days) on oleate plates, a low level of growth was seen (Table 1; Fig. 3A, centre). Initially, we ascribed this behaviour to insertion of the Tn10 element upstream of the *fadB* gene such that it was incompletely polar on the *fadBA* promoter. However, Nicols et al. (1998)

Table 1. Aerobic and anaerobic growth of *fad* strains on fatty acids of varying chain length.

| Strain   | Relevant genotype | Aerobic       | Anaerobic (+25 mM NO₃) |
|----------|-------------------|---------------|------------------------|
|          |                   | GLU | GLY | ACE | C4 | C6 | C8 | C10 | C18:1 | GLU | GLY | ACE | C4 | C6 | C8 | C10 | C18:1 |
| BW25113  | Wild type         | 6+  | 6+  | –   | –  | –a | –a | 6+  | 6+    | 6+  | 6+  | –   | +  | +  | +  | +   | 3+    |
| MG1655   | Wild type         | 6+  | 6+  | –   | –  | –a | –a | 6+  | 6+    | 6+  | 6+  | –   | +  | +  | +  | +   | 3+    |
| RMK6     | *fadR*            | 6+  | 6+  | –a  | –  | 3+ | 6+ | 6+  | 6+    | 6+  | 6+  | –   | +  | +  | +  | +   | 3+    |
| CAG18557 | *fadA751*         | 6+  | 6+  | –   | –  | –  | –  | –b | 6+    | 6+  | 6+  | –   | +  | +  | +  | +   | 3+    |
| FB20456  | *fadD*            | 6+  | 6+  | –   | –  | –  | –  | –  | 6+    | 6+  | 6+  | –   | +  | +  | +  | +   | 3+    |
| JWC266   | *fadE*            | 6+  | 6+  | –   | –  | –  | –  | –  | 6+    | 6+  | 6+  | –   | +  | +  | +  | +   | 3+    |
| EM66     | *aceA1*           | 6+  | 6+  | –   | –  | –  | –  | –  | –     | 6+  | 6+  | –   | –  | –  | –  | –   | –     |
| JWC281   | *yfcYX*           | 6+  | 6+  | –   | –  | –a | –a | 6+  | 6+    | 6+  | 6+  | –   | –b | –b | –  | –   | 3+    |
| JWC294   | *fadA yfcYX*      | 6+  | 6+  | –   | –  | –  | –  | –  | –     | 6+  | 6+  | –   | –  | –  | –  | –   | –     |
| RMK13    | *ydiD*            | 6+  | 6+  | –   | –  | –a | –a | 5+  | 6+    | 6+  | 6+  | –   | +  | +  | +  | +   | 2+    |
| RMK58    | *fadD ydiD*       | –   | +   | +   | –  | –  | –  | –  | –     | +   | ++  | –   | –  | –  | –  | –   | –     |

a. Appearance of isolated colonies after extended incubation (>4 days).  
b. Slow growth after extended incubation (>5 days)

Cultures were grown on M9 plates supplemented with 0.2% glucose (GLU), 0.4% glycerol (GLY) or acetate (ACE) or 1 g l⁻¹ butyrate (C4), hexanoate (C6), octanoate (C8), decanoate (C10) or oleate (C18:1). The plates were supplemented with 25 mM nitrate for anaerobic growth. Growth was scored + if discernible colonies were present, or – when no colonies were apparent after 7 days incubation under aerobic or anaerobic conditions. Growth was usually obvious after overnight aerobic growth or 2–4 days of anaerobic growth. Aerobic growth (after 2 days) on a given carbon source was given a score of 6+.

796 J. W. Campbell, R. M. Morgan-Kiss and J. E. Cronan, Jr

**Fig. 3. Growth of - and fad mutant strains on Sgal minimal M9 plates supplemented with 1 g l⁻¹ oleate under aerobic (Ole + O₂) versus anaerobic conditions in the presence of 25 mM NO₃. The plates were incubated for either 2 or 4 days under aerobic conditions and for 4 days under anaerobic conditions. The strains tested were: (A) MG1655 (wt), CAG18557 (fadA), JWC281 (yfcYX) and JWC294 (fadA yfcYX); (B) MG1655 (wt) (fadD), RMK13 (ydiD), RMK58 (ydiD fadD); (C) TH181 (wt), TH182 [fadR250(S)], RMK54 [fadR250(S) fadD], RMK53 [fadR250(S) ydiD].**

---

**A**
- **Oleate+O₂ (2 days)**
  - wt
  - yfcYX
  - fadA
  - yfcYX
- **Oleate+O₂ (4 days)**
  - wt
  - yfcYX
  - fadA
  - yfcYX
- **Oleate+NO₃ (4 days)**
  - wt
  - yfcYX
  - fadA
  - yfcYX

**B**
- **wt**
  - ydiD
  - fadD
  - ydiD
- **wt**
  - ydiD
  - fadD
  - ydiD
- **wt**
  - ydiD
  - fadD
  - ydiD

**C**
- **Oleate+O₂ (2 days)**
  - wt
  - fadR(S)
  - fadD
  - ydiD
  - fadR(S)
- **Oleate+NO₃ (4 days)**
  - wt
  - fadR(S)
  - fadD
  - ydiD
  - fadR(S)

---

YfcYX-catalysed fatty acid degradation proceeds by the classical β-oxidation pathway, a series of intermediates differing by an acetate unit should be observed. The strains used in these experiments contained a fabA mutation that blocked unsaturated fatty acid synthesis and resulted in a requirement for unsaturated fatty acids for growth. The unsaturated fatty acids normally found in the membranes of *E. coli* are 16 or 18 carbons long; however, fabA mutants incorporate unsaturated fatty acids as short as 14 and as long as 20 or 22 carbons into membrane phospholipids (Silbert et al., 1968), thus indicating that phospholipid synthesis can intercept and incorporate chains shortened by the β-oxidation. To test whether the new yfcYX-dependent pathway also produced chains progressively shortened by two carbon atoms, we grew various fabA mutant strains on cis-11-eicosenoic acid (a monounsaturated fatty acid of 20 carbon atoms, C20:1) and determined the masses of the unsaturated fatty acids incorporated into membrane phospholipids by negative-ion cone-voltage degradation mass spectrometry (Fig. 4).

---

**Fig. 4. CVD-MS analysis of fabA mutant strains grown in minimal glycerol medium supplemented with 100 μg ml⁻¹ cis-11-eicosenoic acid (C20:1). Phospholipids were isolated from the following strains: (A) JWC290 (fabA fadBA); (B) JCW291 (fabA yfcYX); (C) JWC292 (fabA fadA yfcYX) and degraded to their fatty acid moieties in the mass spectrometer.**

- **A**
  - fabA fadA751
  - C20:1 (309.4)
  - C18:1 (281.4)
  - C16:0 (255.4)
  - C16:1 (253.3)
  - C14:0 (227.3)
- **B**
  - fabA yfcYX
  - C20:1 (309.4)
  - C18:1 (281.4)
  - C16:0 (255.4)
  - C16:1 (253.3)
  - C14:0 (227.3)
- **C**
  - fabA fadA751
  - yfcYX
  - C20:1 (309.4)
  - C18:1 (281.4)
  - C16:0 (255.4)
  - C16:1 (253.3)
  - C14:0 (227.3)

---

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, 47, 793–805

was confirmed by the observed complete lack of chain shortening of cis-11-eicosenoic acid in the fabA fadA yfcYX strain (Fig. 4C).

Purified FadR binds to DNA sequences adjacent to yfcYX in vitro

The observed FadR regulation of yfcYX predicts the presence of a FadR binding site located upstream of yfcY (Fig. 2A). Indeed, a 21 bp sequence upstream of the yfcY start codon showed a good match to known FadR binding sites. Alignment of this sequence with those of the known FadR binding sites listed at http://arep.med.harvard.edu/ecoli_matrices/dat/fadR.dat (Robinson et al., 1998) is shown in Fig. 5. The sequence of this putative yfcY FadR binding site closely matched the other negatively regulated genes of fatty acid degradation. To determine whether the putative yfcY FadR site was indeed a functional FadR binding site, in vitro gel mobility shift assays were conducted. A 528 bp 32P-labelled DNA fragment containing the putative FadR binding site was produced from chromosomal DNA. The chromosomal location of the DNA fragment is shown in Fig. 2A. FadR protein purified by nickel chelate chromatography of a His-tagged fusion protein known to be fully functional in vivo was bound by the DNA fragment (Fig. 6). The specificity of FadR DNA binding was demonstrated by the fact that, when the yfcY DNA was digested with the restriction enzymes AlwNI and

Fig. 6. Gel mobility shift assays of FadR and the putative yfcY-associated binding site. The presence or absence of FadR in the binding assay is denoted by + or – at the top of each lane. Lanes 1 and 2, uncut labelled fragment. Lanes 3 and 4, AlwNI digest of the labelled fragment to produce 134 and 394 bp products. Lanes 5 and 6, BglI digest to produce products of 158 and 370 bp. Estimated sizes of the fragments are indicated on the left. The location of the 528 bp PCR product used in the binding assay is indicated in the insert located at the lower left and shows relative positions of the AlwNI; BglI sites and the yfcY start codon are given at the lower left. The putative FadR binding site is denoted as a solid rectangle.

© 2003 Blackwell Publishing Ltd, Molecular Microbiology, 47, 793–805

798 J. W. Campbell, R. M. Morgan-Kiss and J. E. Cronan, Jr

BglI, only the DNA fragment that contained the putative binding site was shifted by FadR (Fig. 6). This result effectively rules out non-specific protein–DNA binding and localized the *yfcY* FadR binding site to an ≈ 236 bp region at the beginning of the *yfcY* gene.

**Anaerobic growth of *E. coli* on fatty acids**

Although the YfcYX-mediated fatty acid degradation pathway acts as an alternative to the FadBA pathway under aerobic conditions, the low activity of YfcYX that we observed made it seem unlikely that this was the primary role of YfcYX. It seemed more likely that YfcYX degraded fatty acids under physiological conditions that differed from those we had tested. Degradation of fatty acids under anaerobic conditions seemed a possible function for YfcYX, but it was unclear whether or not anaerobic growth on fatty acids was possible. We therefore tested several wild-type strains of *E. coli* for anaerobic growth on plates of minimal medium supplemented with oleic acid in carbon source levels. Anaerobic growth was tested both in the presence and in the absence of an alternative electron acceptor. The wild-type strains MG1655 (Richmond et al., 1999) and BW25113 (Datsenko and Wanner, 2000) grew well under anaerobic conditions on plates containing nitrate as electron acceptor and with glucose, glycerol or oleate (but not acetate) as sole carbon sources. Neither strain grew anaerobically on either fatty acids or glycerol in the absence of nitrate, indicating that no other potential electron acceptors were present at concentrations sufficient to support growth. Not surprisingly, the rate of anaerobic growth on nitrate plates with oleate as a sole carbon source proceeded more slowly than aerobic growth on oleate (Fig. 3). With oleate as sole carbon source, colonies of ≈ 2 mm in diameter were formed in 2 days under aerobic conditions, whereas 5 days of incubation were required to obtain colonies of comparable size under anaerobic conditions on oleate-nitrate plates. As expected, nitrate was the most efficient anaerobic electron acceptor and yielded the highest growth rates under these conditions, although trimethylamine N-oxide and fumarate also supported growth on fatty acids, albeit at reduced rates relative to nitrate. Note that the anaerobic conditions used were sufficiently stringent to permit growth of a methanogen in the same enclosure (Metcalf et al., 1997).

**Anaerobic growth of *E. coli* on fatty acids is not blocked by mutations that block aerobic growth and require glyoxylate cycle function**

It seemed possible that the enzymes used for aerobic degradation of fatty acids may have little or no role in the anaerobic pathway. To test this possibility, we examined

growth of *fadD*, *fadE* and *fadA751::Tn10* mutant strains on fatty acid-nitrate media under either aerobic or anaerobic conditions. As expected, *fadD* and *fadE* strains completely failed to grow aerobically on plates containing any of the fatty acids tested (Table 1), whereas the *fadA* strain showed the slow aerobic growth on oleate discussed above. Thus, each of these mutations blocks (or, in the case of the *fadA* strain, severely limits) aerobic growth on fatty acids. In contrast, under anaerobic conditions, each of the mutant strains grew as well as the isogenic wild-type strains on all the fatty acids tested. These results indicated that several of the major enzymes of the *fad* regulon (Fig. 1) were not required to degrade fatty acids under anaerobic conditions. Thus, in addition to the well-characterized aerobic pathway of fatty acid degradation, a distinct fatty acid degradation system operates under anaerobic conditions. Furthermore, we expected that the glyoxylate pathway would be required for anaerobic growth on fatty acids in the presence of nitrate and found that the *aceA1* mutant strain, EM66, grew well on glucose or glycerol but failed to grow either aerobically or anaerobically in the presence of nitrate on any of the fatty acids tested (Table 1). These results indicated that the glyoxylate pathway is required for the utilization of fatty acids as a sole carbon and energy source by the anaerobic pathway as well as by the aerobic pathway.

**Anaerobic growth on fatty acids is not strongly regulated by *fadR***

Under anaerobic plus nitrate conditions, both wild-type strains grew on the shorter chain fatty acids, butyrate, hexanoate, octanoate and decanoate, as sole carbon sources under anaerobic plus nitrate conditions, although growth on these fatty acids was significantly slower than on oleate (Table 1). This finding argued that, unlike the aerobic pathway, the anaerobic pathway was not strongly regulated by the FadR regulatory protein. Indeed, under anaerobic conditions, cultures of wild-type and *fadR* strains grew equally well on fatty acids of chain lengths C4 to C10 (Table 1). In contrast and as expected from previous reports (Overath and Raufuss, 1967; Overath et al., 1969), both wild-type strains grew well under aerobic conditions on the long-chain unsaturated fatty acid, oleate, but failed to grow on the shorter chain fatty acids (Table 1), although the expected spontaneous *fadR* mutants arose in the wild-type strains after an incubation period of >4 days on octanoate or decanoate. As also expected, a *fadR::Tn10* strain grew well aerobically on decanoate and slowly on octanoate (Table 1).

To investigate further whether FadR plays a role in anaerobic *fad* gene regulation, we tested anaerobic growth of *fadR* super-repressor [*fadR(S)*] strains (Hughes et al., 1988) on oleate plus nitrate medium. The *fadR(S)* strains

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **47**, 793–805

Table 2. Effect of *fadR* super-repressor mutations on growth of *fad* and *fad* homologue mutants.

| Strain | Relevant genotype | Aerobic growth | Anaerobic growth |
|--------|-------------------|----------------|------------------|
|        |                   | GLY            | C18:1            | GLY            | C18:1          |
| TH181  | Wild type         | 6+             | 6+               | 6+             | 3+             |
| RMK57  | *fadR*            | 6+             | 6+               | 6+             | 3+             |
| TH182  | *fadR250(S)*      | 6+             | _a_              | 6+             | 3+             |
| TH183  | *fadR251(S)*      | 6+             | _a_              | 6+             | 3+             |
| RMK56  | *fadR250(S) fadA* | 6+             | –                | 6+             | _b_            |
| RMK55  | *fadR250(S) yfcYX*| 6+             | _a_              | 6+             | _b_            |
| RMK54  | *fadR250(S) fadD* | 6+             | –                | 6+             | 3+             |
| RMK53  | *fadR250(S) ydiD* | 6+             | _a_              | 6+             | +              |

a. Appearance of isolated colonies after extended incubation (>4 days).

b. Very slight growth after extended incubation (>5 days).

Cultures were grown on M9 plates supplemented with 0.4% glycerol (GLY) or 1 g l⁻¹ oleate (C18:1). The plates were supplemented with 25 mM nitrate for anaerobic growth. Growth was scored + if discernible colonies were present, or – when no colonies were apparent after 7 days incubation under aerobic or anaerobic conditions. Growth was usually obvious after overnight aerobic growth or 2–4 days of anaerobic growth. Aerobic growth (after 2 days) on a given carbon source was given a score of 6+.

were isolated as dominant *fadR* mutants unable to grow on fatty acids as a sole aerobic carbon source (Hughes et al., 1988). These strains are non-inducible for the *fadBA* enzyme activities and have lower basal levels of expression than wild-type strains (hence their designation as super-repressor mutants). Presumably, these mutant FadR proteins bind the *fad* operators more tightly than the wild-type protein and either fail to bind acyl-CoA inducers or bind these ligands but fail to dissociate from the *fad* regulon operators. As reported previously (Hughes et al., 1988), strains carrying the *fadR250(S)* or the *fadR251(S)* mutations blocked aerobic growth on oleate, although isolated colonies appeared after extended growth periods (>4 days), which are presumably spontaneous *fadR* mutants that have lost FadR function. In contrast, the *fadR(S)* strains grew as well as either the isogenic wild-type strain or the *fadR* strain on oleate plus nitrate plates incubated under anaerobic conditions (Fig. 3C; Table 2). Therefore, the effects of FadR regulation on anaerobic fatty acid degradation, if any, are minimal.

The role of *fadR* in the transcriptional regulation of *yfcYX*

We first detected the *yfcYX* genes as genes showing a marked increase in aerobic expression in a *fadR* strain or in a wild-type strain supplemented with oleate (Campbell and Cronan, 2001) (http://www.life.uiuc.edu/~jwcampbe). Similar results were recently reported by Zhang et al. (2002). The *yfcYX* genes are located within a putative operon located at min 52.9 on the *E. coli* K-12 genetic map. A potential promoter as well as several CRP binding

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **47**, 793–805

protein) and therefore seemed a plausible acyl-CoA synthetase candidate. A *ydiD* mutant strain of *E. coli* was constructed by replacing the entire coding sequence of the gene with a chloramphenicol resistance gene to produce strain RMK13 (Fig. 2B). Under aerobic conditions, strain RMK13 grew well on acetate, glucose, glycerol or oleate and failed to grow on any of the short fatty acids (Table 1; Fig. 3B). In contrast, under anaerobic conditions, strain RMK13 grew well on glucose and glycerol, but grew relatively poorly on all fatty acids tested in comparison with the isogenic wild-type strain, BW25113 (Table 1; Fig. 3B). The residual anaerobic growth of strain RMK13 seems to result from FadD activity, as a *fadD ydiD* double mutant strain failed to grow on any of the fatty acids tested under either aerobic or anaerobic plus nitrate conditions (Table 1; Fig. 3B). Although the observed partial functional replacement of YdiD function by FadD is consistent with YdiD being an acyl-CoA synthetase, proof of this hypothesis will require direct enzymatic analyses.

### Regulation of *ydiD* expression

In our microarray experiments (see above), we observed no induction of *ydiD* expression upon addition of fatty acids or inactivation of FadR. However, these experiments were done under aerobic conditions, where the level of *ydiD* expression is presumably very low (as *fadD* mutants show a complete lack of growth on fatty acids). We therefore turned to the *fadR(S)* mutation to determine whether FadR regulates *ydiD*. Inactivation of both *fadD* and *ydiD* is required to block anaerobic growth on oleate plus nitrate medium (Table 1; Fig. 3B) and, thus, efficient FadR repression of *ydiD* expression should also result in defective growth of a *fadR(S) fadD* strain under these conditions. However, although the *fadR(S) fadD* strain failed to grow aerobically on oleate, when placed under anaerobic conditions, this strain grew as well as the isogenic *fadR(S)* strain on oleate with nitrate as the electron acceptor (Table 2; Fig. 3C). Thus, expression of the *ydiD* gene seems to be independent of FadR.

Note that the *fadR(S) ydiD* strain showed slow growth on oleate under anaerobic conditions (Table 2; Fig. 3C), presumably because of the relatively low affinity of the *fadD* operator sites for FadR (Black et al., 1992), which permits an appreciable expression of *fadD*. As expected, the introduction of a *fadR* null mutation into the *fadD, ydiD* and *fadD ydiD* strains had no effect on the growth of these strains on oleate under either aerobic or anaerobic conditions (data not shown).

### Discussion

The evidence presented here indicates that *E. coli* is capable of anaerobically respiring fatty acids. This is not surprising given the life style of the organism. The usual habitat for *E. coli*, the mammalian bowel, is likely to have abundant levels of fatty acids derived from both the host diet and the cell membranes of other bacteria and the host. However, this environment is microaerobic at best and, therefore, some form of anaerobic respiration would be required to support efficient utilization of fatty acids. Previous studies of fatty acid degradation in *E. coli* have invariably been under aerobic conditions and, although there have been suggestions of a second set of fatty acid degradation enzymes in other bacteria (Olivera et al., 2001), to date evidence of an alternative β-oxidative pathway in *E. coli* has been lacking. Our finding that many *fad* mutant strains blocked in aerobic growth on fatty acids grow anaerobically on fatty acids as a sole carbon and energy source in the presence of an alternative electron acceptor indicates that *E. coli* possesses an alternative set of enzymes for using these substrates under these conditions. The functions of these two pathways have some overlap. The YfcYX proteins play a minor role in aerobic β-oxidation, whereas either the FadBA or the YfcYX multienzyme complexes can degrade oleic acid anaerobically in the presence of nitrate. However, only the YfcYX complex can use short-chain acids anaerobically. In contrast, YdiD plays no role in aerobic growth on fatty acids; however, either the FadD or the YdiD proteins can support anaerobic oleic acid utilization in the presence of nitrate.

The first clue to the genes of this new pathway was the leaky phenotype associated with strains carrying the classical *fadA751::Tn10* mutation that was eliminated by deletion of the *yfcYX* genes. The proteins encoded by the *yfcX* and *yfcY* genes each showed 30–35% sequence homology over the full lengths of the proteins to FadB and FadA, respectively, with retention of all the known active site residues (Haller et al., 2000). Like the *fadBA* genes, the *yfcYX* genes appear to be transcribed as an operon (Fig. 2A). However, the order of the genes relative to the promoter is reversed, and the DNA sequence homologies among the four genes are not significant, suggesting the possibility of separate evolutionary origins for these two systems. Our finding that *yfcYX* is responsible for the residual growth and fatty acid chain shortening of strains carrying the *fadA751::Tn10* mutation, together with the microarray data, indicates that these genes are expressed during aerobic growth on oleate. Anaerobic expression of *yfcYX* is demonstrated by the ability of YfcYX to permit anaerobic growth on fatty acids in the absence of FadBA activity. Shortly after the submission of this paper for publication, Snell et al. (2002) reported that YfcX possesses both enoyl-CoA hydratase and 3-hydroxyacyl-CoA dehydrogenase activities, and *yfcY* encodes thiolase activity. This provides strong evidence in support of our findings that YfcX and YfcY encode functional homologues of FadB.
and FadA respectively. These workers also found that a $f a d B$ null mutant retained aerobic $\beta$-oxidation activity *in vivo*, whereas a strain with null mutations in both $f a d B$ and $y f c X$ had no detectable activity. These results (which were obtained using a completely different assay for $\beta$-oxidation) are in excellent agreement with those that we obtained with our $f a d A y f c Y X$ strain.

An unexpected result was the finding that, although the presence of a FadR super-repressor protein failed to prevent anaerobic growth of an otherwise wild-type strain on oleate plus nitrate, loss of activity of either FadBA or YfcYX in this strain background resulted in an inability to grow under these conditions (Table 2). This result suggests that, under repressing conditions where the activities are low, both multienzyme complexes are required for growth. This could result from a simple additive effect of the two complexes but, given the levels of FadR repression reported previously (10- to 15-fold) for the two enzymes (Campbell and Cronan, 2001; Zhang *et al.*, 2002), slow growth rather than the robust growth observed was expected. For this reason, we favour a model in which the substrate specificities of the two complexes complement one another in order to achieve more efficient utilization of fatty acid chains. It is energetically more favourable to produce a given amount of acetyl-CoA by complete degradation of an acyl chain rather than through partial degradation of multiple acyl chains. This is because each acyl chain requires activation by ATP hydrolysis in order to enter the first $\beta$-oxidation cycle, but subsequent cycles require no further activation. Klein *et al.* (1971) have reported that aerobic cultures of wild-type cells (in which FadBA is the dominant enzyme complex) perform incomplete oxidative degradation of fatty acid chains. These workers assayed the release of ${ }^{14} \mathrm{C}$ from oleate labelled at various positions within the acyl chain. Relative to the carboxyl carbon, release of carbon atoms 10 and 18 was only 0.4 and 0.33 respectively. Therefore, an accumulation of short- and medium-chain intermediates would be expected in these cells. Klein *et al.* (1971) also examined the pattern of carbon release in a strain ($f a d-5$) that lacks all the $f a d B A$-encoded enzyme activities. From our data, we expect that the remaining $\beta$-oxidation activity found in the $f a d-5$ strain is that of the YfcYX complex. Cultures of the $f a d-5$ strain released the distal carbons at much slower rates than the wild-type strain. Relative to the carboxyl carbon, the rates of release of carbon atoms 10 and 18 were only 0.07 and 0.03 respectively. Therefore, it seems that the FadBA complex degrades long-chain fatty acids with fair efficiency, but releases appreciable quantities of short- and medium-chain length intermediates that are excellent substrates for the YfcYX complex. In support of this hypothesis, $y f c Y X$ null mutant strains failed to grow on the shorter chain fatty acids (Table 1), indicating that anaerobic growth on short- or

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **47**, 793–805

medium-chain fatty acids requires YfcYX activity. This ‘efficient utilization’ model provides a rationale for retention of the two $\beta$-oxidation complexes. It should be noted that the results of Snell *et al.* (2002) disagree somewhat with this model. *In vitro*, the enzyme activities assayed had a preference for medium-chain rather than short-chain substrates. However, only a limited range of substrates was examined, which did not include long-chain substrates and, thus, further analyses are needed to test the efficient utilization model. Although the $y f c Y X$ operon is regulated by FadR both aerobically and anaerobically, anaerobic transcription of these genes appears to involve factors in addition to FadR. The fact that wild-type strains are capable of growth on medium-chain fatty acids anaerobically indicates that FadR does not play a major role in the regulation of anaerobic fatty acid degradation.

The $y d i D$ gene was chosen as a candidate to encode a protein with acyl-CoA synthetase activity based on several criteria. First, the N-terminal two-thirds of YdiD aligns with FadD over the entire length of the latter protein. Although the homology score is low (25% identical residues) and alignment required many small gaps, $y d i D$ possesses both a putative ATP/AMP-binding motif and a fatty acyl-CoA-binding (FACS) motif, the two essential components of acyl-CoA synthetases. Furthermore, our confidence in these comparisons was increased by alignment of YdiD with two putative acyl-CoA synthetases, Fat1p of yeast (Faergeman *et al.*, 1997) and mouse FATP (Hirsch and Lodish, 1998). Both these proteins align with YdiD over the entire lengths of the proteins but, again, the alignments were of modest quality (24–27% identical residues) and required many small gaps. However, as expression of mouse FATP complements growth of a yeast strain carrying a deletion of $f a t 1$, the two proteins (which have only 33% identical residues) are functional homologues (DiRusso *et al.*, 2000). Based on these sequence analyses, we proceeded to delete $y d i D$ and thus test its function in fatty acid utilization. Strains carrying the $y d i D$ deletion grew normally on aerobic oleate plates, but grew relatively poorly on oleate plus nitrate plates under anaerobic conditions. We suspected that the residual growth observed under the latter condition resulted from FadD and therefore constructed a $y d i D f a d D$ double mutant strain to test this possibility. This strain proved unable to grow on oleate plus nitrate plates under anaerobic conditions. As $f a d D$ mutant strains are unable to grow on fatty acids under aerobic conditions, it is clear that YdiD is either insufficiently expressed or poorly functional under aerobic conditions. However, the converse is not true; FadD is expressed under anaerobic conditions and can partially replace YdiD. It seems clear that $y d i D$ expression is not regulated by FadR based on our microarray analyses and the finding that a $f a d R(S) f a d D$ strain grows normally on oleate plus nitrate plates under
anaerobic conditions (Table 2; Fig. 3C). Furthermore, no obvious FadR binding sites were apparent upstream of the putative *ydi* operon containing *ydiD* (Fig. 2B), and a *fadR* derivative of the *ydiD* mutant grew equally poorly under anaerobic conditions on medium- or long-chain fatty acids (data not shown). However, other transcription factors may be involved in *ydiD* expression. Several putative CRP binding sites are present directly upstream of *ydiQ* (Fig. 2B). A promoter as well as an associated Fnr binding site are predicted upstream of *ydiP*, which encodes a putative AraC-like regulator. Thus, we suspect that *ydiD* expression is controlled by one or more global regulators such as ArcAB and Fnr that govern the transition between aerobic and anaerobic growth but, as yet, we have only preliminary evidence that *fnr* may be involved. An *fnr* mutant strain grows very poorly on fatty acids anaerobically (data not shown). It should be noted that *ydiD* is the last gene of a putative operon. The downstream gene (*pps*) is transcribed in the opposite direction and, thus, polarity is precluded (Fig. 2B). The putative *ydi* operon contains several ORFs that encode proteins that could play a role in anaerobic β-oxidation. Proceeding upstream from *ydiD*, the genes encode a putative ferredoxin (YdiT), a flavoprotein (YdiS), which could accept electrons and use these to reduce a quinone, and two proteins (YdiR and YdiQ) that seem likely to form a heterodimeric electron transport flavoprotein (that could transfer electrons to YdiS). There is also a putative acyl-CoA dehydrogenase, *ydiO*, located immediately upstream of the *ydiP* gene. These genes (*ydiQRST*) have high sequence homology to the *fixABCX* operon of anaerobic carnitine metabolism (Buchet et al., 1998; Walt and Kahn, 2002) and may function as an electron transport chain linking the β-oxidation of fatty acids with the respiratory chain. Most of these genes would be involved in electron transfer from the multienzyme complexes under anaerobic conditions, as FadE performs this function under aerobic conditions (Klein et al., 1971; Campbell and Cronan, 2002). However, as *fadE* null mutant strains grow normally on oleate plus nitrate under anaerobic conditions (Table 1), *E. coli* must possess another route of electron transfer from the multienzyme complexes, and the products of the *ydiO-QRST* genes seem prime candidates for this function. This model resembles fatty acid metabolism in mammalian mitochondria, in which several electron transfer flavoproteins link β-oxidation of fatty acids with the respiratory chain. Together with *yfcXY*, the genes of this putative operon could form an alternative complete β-oxidation system.

Close homologues of the *yfcXY* and *ydiD* genes are found in the currently available genomes of pathogenic *E. coli* strains as well as the *Salmonella* serovar genomes and the *Shigella flexneri* genome. It should be noted that *fadD* (Lucas and Lee, 2001) and *fadE* (which was called

$fadF$) (Spector et al., 1999) have been reported to play roles in the pathogenesis pathways of *Salmonella enterica* serovar Typhimurium. As some environments encountered by this organism during pathogenesis are believed to have very low oxygen tensions, it seems possible that the genes involved in anaerobic utilization of fatty acids may likewise be involved in pathogenesis.

Finally, as the *yfcY*, *yfcX* and *ydiD* genes clearly function in fatty acid degradation, we propose that these genes be renamed *fadJ* (replaces *yfcX*), *fadI* (replaces *yfcY*) and *fadK* (replaces *ydiD*).

### Experimental procedures

#### Microbial methods

Rich broth (RB) contained (g l⁻¹) tryptone, 10; NaCl, 5; and yeast extract, 1. Minimal medium was M9 medium (Miller, 1992). Glucose was used at a final concentration of 0.2%, and glycerol and acetate at 0.4%. Butyrate, hexanoate, octanoate, decanoate and oleate were used at a final concentration of 1 g l⁻¹ (w/v) plus 4 g l⁻¹ Tergitol NP-40 detergent to aid in solubilization. Antibiotics were used at the following concentrations in μg ml⁻¹: tetracycline HCl, 12; kanamycin sulphate, 50; chloramphenicol, 50; and Timentin, 25. Sgal (3,4-cyclohexenoesculatin-β-D-galactopyranoside) was used at a final concentration of 300 mg l⁻¹ in the presence of 500 mg l⁻¹ ferric ammonium citrate. IPTG was used at 100 μM final concentration. Detergent, most antibiotics and most bulk chemicals were obtained from Sigma. Timentin was from SmithKline Beecham. Solid media contained 1.5% (w/v) Bacto agar (Difco).

Aerobic cultures were inoculated under anaerobic conditions and cycled into a Coy anaerobic chamber containing an atmosphere of 5% H₂, 75% N₂ and 20% CO₂. Within the anaerobic chamber, the plates were stored in hermetically sealed jars under pure N₂. The plates were periodically removed from the jars within the anaerobic chamber and visually inspected for growth. To verify the absence of oxygen, strains were routinely grown in the absence of alternative electron acceptors. These same growth conditions are sufficiently anaerobic to allow growth of *Methanosarcina acetivorans* C2A, a methanogen that is highly sensitive to the presence of molecular oxygen (Metcalf et al., 1997). All growth experiments were done at 37°C.

#### Construction of bacterial strains

All bacterial strains were derivatives of *E. coli* K-12 (Table 3). Unless otherwise indicated, strains were obtained from either local laboratory stocks or the *E. coli* Genetic Stock Center (CGSC; Yale University, New Haven, CT, USA). Transduction with phage P1vir and other basic genetic techniques were generally carried out as described by Miller (1992). Strains CAG18496, CAG18497 and CAG18544 are from the collection of ordered Tn10 insertions of Singer et al. (1989). The phage λ Red-mediated recombination method of Datsenko and Wanner (2000) was used to produce strains JWC280, JWC281 and JWC282 from BW25113. The phage λ Red-

Table 3. Bacterial strains.

| Strain       | Genotype                                                                                   | Reference or source                  |
|--------------|-------------------------------------------------------------------------------------------|-------------------------------------|
| MG1655       | $rph-1$, $fnr(?)$                                                                         | Richmond et al. (1999)              |
| BW25113      | $(araD-araB)567, lacZ4787\Delta::rrnB-4, lacI^{Q}, rpoS396, rph-1, \Delta(rhaD-rhaB)568, rrnB-4, hsdR514$ | Datsenko and Wanner (2000)          |
| CAG18557     | $MG1655, fadA751::Tn10$ (formerly $fadBA::Tn10$)                                           | Singer et al. (1989)                |
| CAG18497     | $MG1655, fadR613::Tn10$                                                                  | Singer et al. (1989)                |
| CAG18544     | $MG1655, fadR3115::Tn10kan$                                                              | Singer et al. (1989)                |
| FB20456      | $MG1655, fadD::Tn5$                                                                      | F. Blattner                        |
| RMK6         | $MG1655, fadR::Tn10kan$                                                                  | This work                          |
| RMK13        | $BW25113, ydiD::cat$                                                                     | This work                          |
| RMK58        | $BW25113, ydiD::cat fadD::Tn5$                                                           | This work                          |
| JWC266       | $BW25113, fadE::kan$                                                                     | Campbell and Cronan (2002)          |
| JWC280       | $BW25113, fabA::cat$                                                                     | This work                          |
| JWC281       | $BW25113, yfcYX::kan$                                                                    | This work                          |
| JWC294       | $MG1655, fadA751::Tn10yfcYX::cat$                                                        | This work                          |
| JWC290       | $BW25113, fabA::cat fadA751::Tn10$                                                       | This work                          |
| JWC291       | $BW25113, fabA::cat yfcYX::kan$                                                          | This work                          |
| JWC292       | $BW25113, fabA::cat fadA751::Tn10yfcYX::kan$                                              | This work                          |
| TH181        | $zcf-2039:Tn10$                                                                         | Hughes et al. (1988)               |
| TH182        | $TH181, fadR250(S)$                                                                      | Hughes et al. (1988)               |
| TH183        | $TH181, fadR251(S)$                                                                      | Hughes et al. (1988)               |
| RMK53        | $TH181, fadR250(S) ydiD::cat$                                                            | This work                          |
| RMK54        | $TH181, fadR250(S) fadD::Tn5$                                                            | This work                          |
| RMK55        | $TH181, fadR250(S) yfcYX::kan$                                                           | This work                          |
| RMK56        | $TH181 fadR250(S) fadA751::Tn10kan$                                                      | This work                          |
| RMK57        | $TH181, fadR3115:Tn10kan$                                                                | This work                          |
| EM66         | $aceA1zja::Tn10$                                                                         | CSGC                               |

mediated recombination system of Yu et al. (2000) was used to construct strain RMK13.

Strain JWC280 was constructed by amplification of the chloramphenicol acetyltransferase (cat) gene from plasmid pKD3 (Datsenko and Wanner, 2000) using primers specific to the P1 and P2 regions of the plasmid plus an additional 42 bases of perfect homology to sequences flanking $fabA$. This polymerase chain reaction (PCR) product was used to replace the coding sequence of the $fabA$ gene of BW25113. The upstream primer was $5^{\prime}$-GCTTCAATAAAAATAAGGC TACAGAGAACATGGTACATAAAACTGTGTAGGCTGGAGCT, and the downstream primer was $5^{\prime}$-GGTTTCGCC TTTTG ATACTCTGTCTGATTATAATCAGAAGGTGTGTAGGCTGGAGCT GCT. The $fabA$ genotype was confirmed by the finding that strain JWC280 required unsaturated fatty acids for growth.

Strain JWC281 was constructed by amplification of the aminoglycoside 3′-phosphotransferase (kan) gene of plasmid pKD4 (Datsenko and Wanner, 2000) using primers specific to the P1 and P2 regions of the plasmid plus an additional 30 bases of sequence flanking $yfcYX$. The upstream primer was $5^{\prime}$-ATGGGTCAGGTTTTACC GCTGGTTACCCGCCAGG GCGATCGTATCCATATGAATATCCTCCTTAG, and the downstream primer was $5^{\prime}$-TTATTGCAGGTCAGTTGCAGTTGT TTCCA AAAACTTTCCCCACGTGTGTAGGCTGGAGCT CTTCG. The same primers were used to amplify the cat gene from pKD3 in order to produce strain JWC282. Replacement of the $yfcYX$ genes with either the kanamycin (strain JWC281) or chloramphenicol (strain JWC282) resistance elements resulted in no apparent growth defect under normal laboratory conditions.

Strain RMK13 was constructed by replacement of the entire coding sequence of the $ydiD$ gene with the cat gene in strain DY330 (Yu et al., 2000). P1 lysates were grown on the resultant $Cm^{R}$ mutants, and the $ydiD::cat$ allele was transduced into BW25113 to produce RMK13. The upstream

primer was $5^{\prime}$-CGCTACGGCTAATCATG CATCCCACAGGC CCGCATCTCGGGGCCTG, and the downstream primer was $5^{\prime}$-GCCTGGCTGA ACTGAA GAAAATAAAATCC CGGC GGCGTTTA. Replacement of the $ydiD$ gene with the cat gene resulted in no apparent growth defect under normal laboratory conditions.

In aid of improved visualization of growth on minimal plates, lactose-positive derivatives RMK13, RMK58 and JWC281 were constructed via standard genetic techniques to allow for a positive phenotype to be observed on Sgal plates (see above).

Genomic expression profiling analysis

The methods and procedures involved in cell growth, RNA isolation, cDNA library production, hybridization and interpretation of the genomic array experiments have been described previously (Campbell and Cronan, 2001). Genomic profiling used the Sigma-Genosys Panorama array system that is composed of full-length ORF targets spotted onto nylon membranes probed with radiolabelled cDNA. The results of these experiments are available online at http://www.life.uiuc.edu/~jwcampbe.

Cone-voltage degradation mass spectrometry (CVD-MS)

Structural determination and analysis of individual phospholipid species in cellular membranes is possible using electrospray ionization mass spectrometry (Han and Gross, 1994). Briefly, small cultures were grown overnight in minimal plus glycerol medium in the presence of $100 \mu g \mathrm{ml}^{-1}$ cis-11-eicosenoic acid (C20:1). At this concentration, the long-chain fatty acid was not completely soluble but could still support growth of unsaturated fatty acid requiring $fabA$ strains. After

804 J. W. Campbell, R. M. Morgan-Kiss and J. E. Cronan, Jr

overnight growth, cells were pelleted in a microfuge tube and washed three times with fresh media lacking fatty acid to remove unincorporated fatty acid. The final cell pellet was resuspended in 1 ml of a 1:1 mixture of chloroform–methanol and vortexed for at least 1 h. Insoluble debris was removed by centrifugation, and the organic phase was transferred to a fresh tube. Samples were placed in a fume hood and allowed to dry overnight. The non-volatile residue was resuspended in a small volume of chloroform–methanol and subjected to CVD-MS. Negative ion ES-MS analysis was performed on a Micromass Quatro I equipped with a Nano-probe ESI sample probe at the Mass Spectrometry Laboratory (School of Chemical Sciences, University of Illinois at Urbana-Champaign, Urbana, IL, USA). The in-source cone voltage was routinely set to 90 V, which resulted in almost complete degradation of phospholipids to their constituent fatty acids.

**Gel mobility shift analyses**

A 528 bp DNA fragment including ≈ 265 bp of DNA on either side of the *yfcY* start codon (see Fig. 2A) was produced by PCR amplification. In this case, a standard PCR amplification was carried out in the presence of a small quantity of [α-³³P]-dCTP (3000 Ci mmol⁻¹) to generate a labelled DNA product. The primers were 5′-CACCCCGACTTTCACTGAAGAGTC-3′ on the 5′ side and 5′-GAACAAATTTCAACGCGCAATGTTGG-3′ on the 3′ side. The 528 bp product was purified on a Qiagen PCR purification column. Purification of FadR and the gel retardation conditions used were described in detail previously (Henry and Cronan, 1992; Subrahmanyam and Cronan, 1998; Campbell and Cronan, 2001).

**References**

van Aalten, D.M., DiRusso, C.C., Knudsen, J., and Wierenga, R.K. (2000) Crystal structure of FadR, a fatty acid-responsive transcription factor with a novel acyl coenzyme A-binding fold. *EMBO J* **19**: 5167–5177.

Black, P.N., DiRusso, C.C., Metzger, A.K., and Heimert, T.L. (1992) Cloning, sequencing, and expression of the *fadD* gene of *Escherichia coli* encoding acyl coenzyme A synthetase. *J Biol Chem* **267**: 25513–25520.

Blattner, F.R., Plunkett, G., III, Bloch, C.A., Perna, N.T., Burland, V., Riley, M., et al. (1997) The complete genome sequence of *Escherichia coli* K-12. *Science* **277**: 1453–1474.

Buchet, A., Eichler, K., and Mandrand-Berthelot, M.A. (1998) Regulation of the carnitine pathway in *Escherichia coli*: investigation of the *cai-fix* divergent promoter region. *J Bacteriol* **180**: 2599–2608.

Campbell, J.W., and Cronan, J.E., Jr (2001) *Escherichia coli* FadR positively regulates transcription of the *fabB* fatty acid biosynthetic gene. *J Bacteriol* **183**: 5982–5990.

Campbell, J.W., and Cronan, J.E., Jr (2002) The enigmatic *Escherichia coli fadE* gene is *yafH*. *J Bacteriol* **184**: 3759–3764.

Clark, D.P. (1989) The fermentation pathways of *Escherichia coli*. *FEMS Microbiol Lett* **63**: 223–234.

Clark, D.P., and Cronan, J.E., Jr (1996) Two-carbon compounds and fatty acids as carbon sources. In: *Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology*. Neidhardt, F.C., Curtiss, R., Ingraham, J.L., Lin, E.C.C., Low, K.B., Magasanik, B., et al. (eds). Washington, DC: American Society for Microbiology Press, pp. 343–357.

Datsenko, K.A., and Wanner, B.L. (2000) One-step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. *Proc Natl Acad Sci USA* **97**: 6640–6645.

DiRusso, C.C., Connell, E.J., Faergeman, N.J., Knudsen, J., Hansen, J.K., and Black, P.N. (2000) Murine FATP alleviates growth and biochemical deficiencies of yeast *fat1Δ* strains. *Eur J Biochem* **267**: 4422–4433.

Faergeman, N.J., DiRusso, C.C., Elberger, A., Knudsen, J., and Black, P.N. (1997) Disruption of the *Saccharomyces cerevisiae* homologue to the murine fatty acid transport protein impairs uptake and growth on long-chain fatty acids. *J Biol Chem* **272**: 8531–8538.

Gui, L., Sunnarborg, A., and LaPorte, D.C. (1996) Regulated expression of a repressor protein: FadR activates *iclR*. *J Bacteriol* **178**: 4704–4709.

Haller, T., Buckel, T., Retey, J., and Gerlt, J.A. (2000) Discovering new enzymes and metabolic pathways: conversion of succinate to propionate by *Escherichia coli*. *Biochemistry* **39**: 4622–4629.

Han, X., and Gross, C.A. (1994) Electrospray ionization mass spectroscopic analysis of human erythrocyte plasma membrane phospholipids. *Proc Natl Acad Sci USA* **91**: 10635–10639.

Henry, M.F., and Cronan, J.E., Jr (1991) *Escherichia coli* transcription factor that both activates fatty acid synthesis and represses fatty acid degradation. *J Mol Biol* **222**: 843–849.

Henry, M.F., and Cronan, J.E., Jr (1992) A new mechanism of transcriptional regulation: release of an activator triggered by small molecule binding. *Cell* **70**: 671–679.

Hirsch, D.A., S., and Lodish, H.F. (1998) A family of fatty acid transporters conserved from mycobacterium to man. *Proc Natl Acad Sci USA* **95**: 8625–8629.

Hughes, K.T., Simons, R.W., and Nunn, W.D. (1988) Regulation of fatty acid degradation in *Escherichia coli*: *fadR* superrepressor mutants are unable to utilize fatty acids. *J Bacteriol* **170**: 1666–1671.

Iuchi, S., and Lin, E.C.C. (1988) *arcA* (*dye*), a global regulatory gene in *Escherichia coli* mediating repression of enzymes in aerobic pathways. *Proc Natl Acad Sci USA* **85**: 1888–1892.

Kameda, K., and Nunn, W.D. (1981) Purification and characterization of acyl coenzyme A synthetase from *Escherichia coli*. *J Biol Chem* **256**: 5702–5707.

Klein, K., Steinberg, R., Fiethen, B., and Overath, P. (1971) Fatty acid degradation in *Escherichia coli*. An inducible system for the uptake of fatty acids and further characterization of *old* mutants. *Eur J Biochem* **19**: 442–450.

Lucas, R.L., and Lee, C.A. (2001) Roles of *hilC* and *hilD* in regulation of *hilA* expression in *Salmonella enterica* serovar Typhimurium. *J Bacteriol* **183**: 2733–2745.

Mackie, R.I., White, B.A., and Bryant, M.P. (1991) Lipid metabolism in anaerobic ecosystems. *Crit Rev Microbiol* **17**: 449–479.

Maloy, S.R., and Nunn, W.D. (1981) Role of gene *fadR* in *Escherichia coli* acetate metabolism. *J Bacteriol* 148: 83–90.

Maloy, S.R., Bohlander, M., and Nunn, W.D. (1980) Elevated levels of glyoxylate shunt enzymes in *Escherichia coli* strains constitutive for fatty acid degradation. *J Bacteriol* 143: 720–725.

Metcalf, W.W., Zhang, J.K., Shi, X., and Wolfe, R.S. (1997) Molecular, genetic, and biochemical characterization of the *serC* gene of *Methanosarcina barkeri* Fusaro. *J Bacteriol* 178: 5797–5802.

Miller, J.H. (1992) *A Short Course in Bacterial Genetics*. Cold Spring Harbor, NY: Cold Spring Harbour Laboratory Press.

Nicols, B.P., Shafiq, O., and Meiners, V. (1998) Sequence analysis of Tn10 insertion sites in a collection of *Escherichia coli* strains used for genetic mapping and strain construction. *J Bacteriol* 180: 6408–6411.

Olivera, E.R., Carnicero, D., Garcia, B., Minambres, B., Moreno, M.A., Canedo, L., *et al.* (2001) Two different pathways are involved in the beta-oxidation of n-alkanoic and n-phenylalkanoic acids in *Pseudomonas putida* U: genetic studies and biotechnological applications. *Mol Microbiol* 39: 863–874.

Overath, P., and Raufuss, E.M. (1967) The induction of the enzymes of fatty acid degradation in *Escherichia coli*. *Biochem Biophys Res Commun* 29: 28–33.

Overath, P., Pauli, G., and Schairer, H.U. (1969) Fatty acid degradation in *Escherichia coli*. An inducible acyl-CoA synthetase, the mapping of old-mutations, and the isolation of regulatory mutants. *Eur J Biochem* 7: 559–574.

Pauli, G., and Overath, P. (1974) Fatty acid degradation in *Escherichia coli*: requirement of cyclic adenosine monophosphate and cyclic adenosine monophosphate receptor protein for enzyme synthesis. *J Bacteriol* 117: 1178–1183.

Pramanik, A., Pawar, S., Antonian, E., and Schulz, H. (1979) Five different enzymatic activities are associated with the multienzyme complex of fatty acid oxidation in *Escherichia coli*. *J Bacteriol* 137: 469–473.

Richmond, C.S., Glasner, J.D., Mau, R., Jin, H., and Blattner, F.R. (1999) Genome-wide expression profiling in *Escherichia coli* K-12. *Nucleic Acids Res* 27: 3821–3835.

Robinson, K., McGuire, A.M., and Church, G.M. (1998) A comprehensive library of DNA-binding site matrices for 55

proteins applied to the complete *Escherichia coli* K-12 genome. *J Mol Biol* 284: 241–254.

Silbert, D.F., Ruch, F., and Vagelos, P.R. (1968) Fatty acid replacements in a fatty acid auxotroph of *Escherichia coli*. *J Bacteriol* 95: 1658–1665.

Singer, M., Baker, T.A., Schnitzler, G., Deischel, S.M., Goel, M., Dove, W., *et al.* (1989) A collection of strains containing genetically linked alternating antibiotic resistance elements for genetic mapping of *Escherichia coli*. *Microbiol Rev* 53: 1–24.

Snell, K.D., Feng, F., Zhong, L., Martin, D., and Madison, L.L. (2002) YfcX enables medium-chain-length poly(3-hydroxyalkanoate) formation from fatty acids in recombinant *Escherichia coli fadB* strains. *J Bacteriol* 184: 5696–5705.

Spector, M.P., DiRusso, C.C., Pallen, M.J., Garcia del Portillo, F., Dougan, G., and Finlay, B.B. (1999) The medium/long-chain fatty acyl-CoA dehydrogenase (*fadF*) gene of *Salmonella typhimurium* is a phase 1 starvation-stress response (SSR) locus. *Microbiology* 145: 15–31.

Spratt, S.K., Black, P.N., Ragozzino, M.M., and Nunn, W.D. (1984) Cloning, mapping, and expression of genes involved in the fatty acid-degradative multienzyme complex of *Escherichia coli*. *J Bacteriol* 158: 535–542.

Subrahmanyam, S., and Cronan, J.E., Jr (1998) Overproduction of a functional fatty acid biosynthetic enzyme blocks fatty acid synthesis in *Escherichia coli*. *J Bacteriol* 180: 4596–4602.

Walt, A., and Kahn, M.L. (2002) The *fixA* and *fixB* genes are necessary for anaerobic carnitine reduction in *Escherichia coli*. *J Bacteriol* 184: 4044–4047.

Xu, Y., Heath, R.J., Li, Z., Rock, C.O., and White, S.W. (2001) The FadR: DNA complex. Transcriptional control of fatty acid metabolism in *Escherichia coli*. *J Biol Chem* 276: 17373–17379.

Yu, D., Ellis, H.M., Lee, E.-C., Jenkins, N.A., Copeland, N.G., and Court, D.L. (2000) An efficient recombination system for chromosome engineering in *Escherichia coli*. *Proc Natl Acad Sci USA* 97: 5978–5983.

Zhang, Y.M., Marrakchi, H., and Rock, C.O. (2002) The FabR (YijC) transcription factor regulates unsaturated fatty acid biosynthesis in *Escherichia coli*. *J Biol Chem* 277: 15558–15565.
