
Annu. Rev. Biochem. 2003.72:137-174
doi: 10.1146/annurev.biochem.72.121801.161712
Copyright © 2003 by Annual Reviews. All rights reserved
First published online as a Review in Advance on January 16, 2003

# THE ENZYMES, REGULATION, AND GENETICS
OF BILE ACID SYNTHESIS

David W. Russell  
Department of Molecular Genetics, University of Texas Southwestern Medical Center,  
5323 Harry Hines Blvd., Dallas, Texas 75390-9046; email: david.russell@utsouthwestern.edu  

Key Words cholesterol, catabolism, cytochrome P450, nuclear receptors, genetic disease

■ Abstract The synthesis and excretion of bile acids comprise the major pathway of cholesterol catabolism in mammals. Synthesis provides a direct means of converting cholesterol, which is both hydrophobic and insoluble, into a water-soluble and readily excreted molecule, the bile acid. The biosynthetic steps that accomplish this transformation also confer detergent properties to the bile acid, which are exploited by the body to facilitate the secretion of cholesterol from the liver. This role in the elimination of cholesterol is counterbalanced by the ability of bile acids to solubilize dietary cholesterol and essential nutrients and to promote their delivery to the liver. The synthesis of a full complement of bile acids requires 17 enzymes. The expression of selected enzymes in the pathway is tightly regulated by nuclear hormone receptors and other transcription factors, which ensure a constant supply of bile acids in an ever-changing metabolic environment. Inherited mutations that impair bile acid synthesis cause a spectrum of human disease; this ranges from liver failure in early childhood to progressive neuropathy in adults.

CONTENTS

INTRODUCTION . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Retinoid X Receptor . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
# PATHWAYS OF BILE ACID SYNTHESIS

Cholesterol is converted into bile acids by pathways that involve 17 different enzymes, many of which are preferentially expressed in the liver. The immediate products of these pathways are referred to as primary bile acids; the structures of which vary widely between different vertebrate species. For example, in humans and rats, cholic acid and chenodeoxycholic acid are the primary bile acids, whereas in mice cholic acid and $\beta$-muricholic acid predominate. The chemical diversity of the bile acid pool is further expanded by the actions of anaerobic bacteria in the gut, which convert primary bile acids into dozens of secondary and tertiary bile acids (3). The plethora of different bile acids in the enterohepatic circulation ensures complete solubilization of hydrophobic nutrients in the small intestine. In addition, individual bile acids differ in their abilities to regulate the synthesis of bile acids, which allows an organism to fine-tune output from the biosynthetic pathways.

The steps leading to synthesis of a primary bile acid include: $(a)$ the initiation of synthesis by $7\alpha$-hydroxylation of sterol precursors (Figure 1), $(b)$ further modifications to the ring structures (Figure 2), $(c)$ oxidation and shortening of the side chain (Figure 3), and $(d)$ conjugation of the bile acid with an amino acid (Figure 4). The 17 enzymes that participate in these four steps are listed in Table 1. The exact order of steps in the biosynthetic pathway remains unclear because many of the intermediates serve as substrates for more than one biosynthetic enzyme.

## Initiation

The synthesis of bile acids begins by one of several routes. In the classic pathway, cholesterol is converted into $7\alpha$-hydroxycholesterol by cholesterol $7\alpha$-hydroxylase (Figure 1, reaction 1), a microsomal cytochrome P450 enzyme (CYP7A1) expressed only in the liver (Table 1). P450 enzymes are mixed function mono-oxidases located in the microsomal and mitochondrial compartments of cells. Six members of the P450 superfamily participate in bile acid synthesis. Genes and cDNAs encoding cholesterol $7\alpha$-hydroxylase have been isolated from many species (4), and sequence comparisons between these reveal a hydrophobic enzyme of $\sim 500$ amino acids. When purified from rat liver (5–7), or expressed in cultured cells (7), *Escherichia coli* (8–10), or transgenic mice (11), the cholesterol $7\alpha$-hydroxylase enzyme has a low turnover number and a preference for cholesterol as substrate. Expression of the cholesterol $7\alpha$-hydroxylase gene and possibly the activity of the enzyme are highly regulated (see Regulation of Bile Acid Synthesis).

Mice deficient in the cholesterol $7\alpha$-hydroxylase gene (*Cyp7a1*) have a high incidence of postnatal lethality due to liver failure, vitamin deficiencies, and lipid malabsorption (12–14). The bile acid pool size in these animals is reduced by 75% (15), and the reduction in bile acid synthesis is not compensated for by increased expression of other bile acid biosynthetic enzymes (16). Intestinal

CYP7A1,  
Cholesterol  
7α-Hydroxylase  

5-Cholest-3β-ol (Cholesterol)  

CYP46A1,  
Cholesterol  
24-Hydroxylase  

5-Cholest-3β,24(S)-diol  

CYP25H,  
Cholesterol  
25-Hydroxylase  

5-Cholest-3β,25-diol  

CYP39A1,  
Oxysterol  
7α-Hydroxylase  

5-Cholest-3β,7α-diol  

CYP7B1,  
Oxysterol  
7α-Hydroxylase  

5-Cholest-3β,7α,25-triol  

CYP27A1,  
Sterol  
27-Hydroxylase  

5-Cholest-3β,27-diol  

Ring Structure Modification

cholesterol absorption is reduced to <5% of normal in the mutant mice, which results in a 200% increase in hepatic cholesterol synthesis. These alterations maintain cholesterol homeostasis (15). About 90% of the cholesterol 7α-hydroxylase deficient mice die within the first three weeks of birth. These short-lived mice produce only small amounts of hepatotoxic monohydroxy bile acids (14). Animals that survive this period begin synthesizing normal bile acids. Although the bile acid pool size never exceeds 25% of normal, this amount is sufficient to reverse the pathologic phenotype (13). The mice begin to synthesize bile acids via a different pathway in which oxysterols rather than cholesterol serve as substrates for 7α-hydroxylation (Figure 1).

Hydroxylation of cholesterol at three different positions on the side chain produces 24-hydroxycholesterol, 25-hydroxycholesterol, and 27-hydroxycholesterol. That these oxysterols could serve as substrates for bile acid synthesis was discovered in the 1960s (17). The finding that cholesterol 7α-hydroxylase deficient mice produced bile acids from an alternate pathway led to the identification of the three enzymes required to synthesize the various oxysterols and two enzymes that hydroxylate these intermediates at the 7α-position (Figure 1).

The first of these enzymes is cholesterol 24-hydroxylase, a microsomal cytochrome P450 (CYP46A1, Table 1) that synthesizes the oxysterol 24(S)-hydroxycholesterol (Figure 1, reaction 2). This enzyme is expressed in neurons of the mouse brain and at much lower levels in the liver (18). The 24-hydroxylase mRNA and protein are detectable in nerve cells of the normal human brain (18), and they are induced in glial cells of the central nervous system in subjects with Alzheimer's disease (18, 19).

Mice lacking the cholesterol 24-hydroxylase gene (*Cyp46a1*) have a 40% reduction in de novo cholesterol synthesis in the brain but no detectable alterations in bile acid metabolism (E.G. Lund et al., unpublished information). These findings are consistent with the observations of Björkhem et al. that 24-hydroxylation represents an important pathway by which cholesterol is secreted from the brain (21), and they indicate that a decrease in cholesterol catabolism in the central nervous system is compensated for by a reduction in synthesis. The rate of 24(S)-hydroxycholesterol synthesis in mouse brain is estimated to be 0.85 mg per day per 100 kg body weight, which pales in comparison to the hepatic synthesis of bile acids (~100 mg per day per 100 kg body weight) (E.G. Lund et al., unpublished information). Humans produce 6–7 mg of 24(S)-hydroxycholesterol per day (22), of which ~3.5 mg is catabolized

---

**Figure 1** Biochemical steps involved in the initiation of bile acid synthesis. Circled numbers designate individual reactions, and the enzymes that catalyze each step are indicated next to the arrows. The chemical modification introduced by a particular enzyme is indicated in red on the product of the reaction. Cumulative changes to sterol intermediates are indicated in green. The products of the sterol 7α-hydroxylase enzymes are substrates for ring structure modification.

| Reaction$^a$ | Enzyme | Mass$^b$ (Da) | Subcellular Localization | Comments | cDNA Accession Numbers$^c$ |
| --- | --- | --- | --- | --- | --- |
| 1 | Cholesterol 7α-hydroxylase | 57,660 | Endoplasmic reticulum | P450 (CYP7A1)$^d$, regulated, liver specific | (h) M93133<br>(m) L23754 |
| 2 | Cholesterol 24-hydroxylase | 56,821 | Endoplasmic reticulum | P450 (CYP46A1), brain | (h) AF094480<br>(m) AF094479 |
| 3 | Cholesterol 25-hydroxylase | 31,700 | Endoplasmic reticulum | Diiron cofactor, many tissues | (h) AF059214<br>(m) AF059213 |
| 4 | Sterol 27-hydroxylase | 56,900 | Mitochondria | P450 (CYP27A1), many tissues | (h) M62401<br>(m) AK004977 |
| 5 | Oxysterol 7α-hydroxylase | 54,129 | Endoplasmic reticulum | P450 (CYP39A1), many tissues | (h) AF237982<br>(m) AF237981 |
| 6 | Oxysterol 7α-hydroxylase | 58,255 | Endoplasmic reticulum | P450 (CYP7B1), many tissues | (h) AF029403<br>(m) U36993 |
| 7 | 3β-Hydroxy-Δ5-C27 steroid oxidoreductase | 40,929 | Endoplasmic reticulum | Selective for C27 substrates, many tissues | (h) AF277719<br>(m) AF277718 |
| 8 | Sterol 12α-hydroxylase | 58,078 | Endoplasmic reticulum | P450 (CYP8B1), regulated, liver specific | (h) AF090320<br>(m) AF090317 |
| 9 | Δ4-3-Oxosteroid 5β-reductase | 37,377 | Cytoplasm | Aldo-keto reductase (AKR1D1), many tissues | (h) Z28339<br>(m) BF234820/AI931261 |
| 10 | 3α-Hydroxysteroid dehydrogenase | 37,095 | Cytoplasm | Aldo-keto reductase (AKR1C4), many tissues, many isozymes | (h) S68287<br>(m) NM_030611 |

| 11 | Sterol 27-hydroxylase | 56,900 | Mitochondria | P450 (CYP27A1), many | See above |
| --- | --- | --- | --- | --- | --- |
| 12 | Bile acid CoA ligase | 70,312 | Endoplasmic reticulum, peroxisome | Also uses very long chain fatty acids as substrates | (h) AF096290 (m) AF033031 |
| 13 | 2-Methylacyl-CoA racemase | 42,359 | Peroxisome, mitochondria | Also acts on 2-methyl fatty acids, liver and kidney | (h) AF158378 (m) U89906 |
| 14 | Branched-chain acyl-CoA oxidase | 76,826 | Peroxisome | ACox2, also acts on 2-methyl fatty acids, many tissues | (h) X95190 (m) AJ238492 |
| 15 | D-Bifunctional protein | 79,686 | Peroxisome | Multiple enzyme activities, many tissues | (h) X87176 (m) X89998 |
| 16 | Peroxisomal thiolase 2 | 58,993 | Peroxisome | Multiple isoforms (SCP2, SCPx), liver enriched | (h) U11313 (m) M91458 |
| 17 | Bile acid CoA:amino acid N-acyltransferase | 46,296 | Peroxisome | Liver selective | (h) L34081 (m) U95215 |

a Reaction numbers refer to Figures 1, 2, 3, and 4.

b Exact molecular weights are derived from cDNA sequences and are for the mature human enzymes.

c (h), human; (m), mouse GenBank accession numbers (3a).

d Cytochrome P450 enzymes are named and numbered according to the convention described in (199).
to bile acids (23). This amount is again much smaller than the mass of bile acids synthesized each day (~500 mg). Together these findings indicate that the 24-hydroxylase enzyme contributes little to overall bile acid synthesis but is important in the turnover of cholesterol in the brain.

The microsomal enzyme cholesterol 25-hydroxylase (Figure 1, reaction 3) is not a P450 but rather is a member of a family of lipid metabolizing enzymes that utilize oxygen and a diiron-oxygen cofactor to hydroxylate, desaturate, epoxidate, or acetylate substrates (24). The mouse cholesterol 25-hydroxylase cDNA was isolated by expression cloning (25). The mRNA is present at low levels in most tissues and at a higher level in the lung. Expression of recombinant cholesterol 25-hydroxylase in cultured cells results in the synthesis of 25-hydroxycholesterol (25), and as expected (26), the concomitant repression of cholesterol biosynthetic enzymes. Cholesterol 25-hydroxylase knockout mice have no alterations in bile acid synthesis or cholesterol metabolism, and to date there is no in vivo evidence that this enzyme catalyzes the formation of 25-hydroxycholesterol in the whole animal (G. Liang, J. Li-Hawkins, D. W. Russell, unpublished observations). Cholesterol 25-hydroxylase may play a role in cholesterol catabolism in a tissue-specific fashion, as does the cholesterol 24-hydroxylase.

27-Hydroxycholesterol is the most abundant oxysterol in the plasma of the mouse (27) and human (28), and it is synthesized from cholesterol by sterol 27-hydroxylase (Figure 1, reaction 4), a mitochondrial cytochrome P450 (CYP27A1, Table 1). This enzyme can also hydroxylate cholesterol at carbons 24 and 25 to form 24-hydroxycholesterol and 25-hydroxycholesterol, respectively (29). Unlike the 24-hydroxylase and 25-hydroxylase enzymes, which contribute only modestly to bile acid synthesis in the mouse, about 25% of the bile acid pool originates from oxysterols produced by sterol 27-hydroxylase. This percentage is derived from the observations that the 24- and 25-hydroxylase knockout mice have no discernible alterations in bile acid synthesis and that the cholesterol 7α-hydroxylase deficient mice, which cannot convert cholesterol directly into bile acids, have a bile acid pool size that is 25% of normal (15, 16). The relative contributions of the oxysterol biosynthetic enzymes to bile acid synthesis in humans are difficult to assess. Analysis of the bile acids in normal individuals (22, 30, 31) and in a subject with cholesterol 7α-hydroxylase deficiency (32) suggest that the oxysterol pathways together are responsible for only 5% to 10% of bile acid production. The consequences for loss of the sterol 27-hydroxylase gene in mice and in humans are discussed below in Side Chain Oxidation.

To be converted into bile acids, oxysterols must undergo 7α-hydroxylation. Two microsomal cytochrome P450 enzymes catalyze this step. The CYP39A1 oxysterol 7α-hydroxylase acts on 24(S)-hydroxycholesterol (Figure 1, reaction 5). The mRNA encoding this enzyme is abundantly and constitutively expressed in mouse and human liver as well as in the nonpigmented epithelium of the eye (33–35). The CYP39A1 oxysterol 7α-hydroxylase contains ~470 amino acids,
and when expressed in cultured cells, it demonstrates a rank order preference for oxysterol substrates of 24(S)-hydroxycholesterol (1.0) > 24,25-epoxycholesterol (0.16) > 25-hydroxycholesterol (0.08) > 27-hydroxycholesterol (0.06) > 22-hydroxycholesterol (0.05) (33). The contributions of this enzyme to bile acid synthesis in the mouse, or any other species, have not been determined.

The conversion of 25-hydroxycholesterol and 27-hydroxycholesterol to bile acid intermediates is catalyzed by the CYP7B1 oxysterol 7α-hydroxylase (Figure 1, reaction 6). The cDNA for CYP7B1 was isolated originally from brain by differential hybridization (36), but it was only later shown to encode an enzyme with oxysterol 7α-hydroxylase activity (37–39). The deduced sequence of the protein is ~40% identical to that of cholesterol 7α-hydroxylase and ~30% identical to the CYP39A1 oxysterol 7α-hydroxylase (33). The CYP7B1 oxysterol 7α-hydroxylase is expressed at high levels in the adult liver and at lower levels in the kidney, brain, and prostate. In mice, hepatic expression of this enzyme is induced during the third week of life and thereafter exhibits a sexually dimorphic expression pattern in this and other tissues in which expression is higher in the male (13, 27). The CYP7B1 oxysterol 7α-hydroxylase utilizes 27-carbon oxysterols and 19-carbon steroids, such as dehydroepiandrosterone (37), as substrates. Mice that lack the encoding gene (*Cyp7b1*) have elevated plasma levels of 25-hydroxycholesterol and 27-hydroxycholesterol but not 24-hydroxycholesterol (27, 39a). These animals also have increased levels of cholesterol 7α-hydroxylase, presumably to compensate for the reduced bile acid biosynthetic capacity (27). This increase (~30%) is roughly equal in size to that of the residual bile acid pool in cholesterol 7α-hydroxylase-deficient mice (15), which confirms that the CYP7B1 oxysterol 7α-hydroxylase pathway synthesizes 25% to 30% of all bile acids in this species. As noted previously, 5% to 10% of the bile acid pool in humans originates from oxysterols, but the fraction that derives from the CYP7B1 versus the CYP39A1 oxysterol 7α-hydroxylase is unknown.

## Ring Structure Modification

The 7α-hydroxylated intermediates derived from cholesterol and the oxysterols are next converted into their 3-oxo, Δ⁴ forms (Figure 2, reaction 7) by a microsomal 3β-hydroxy-Δ⁵-C₂₇-steroid oxidoreductase (C₂₇ 3β-HSD, Table 1). There is only one C₂₇ 3β-HSD, and loss of this enzyme blocks the synthesis of all bile acids. The human and mouse C₂₇ 3β-HSDs are membrane-bound enzymes of 369 amino acids that share ~34% sequence identity with the C₁₉ and C₂₁ 3β-HSDs involved in steroid hormone biosynthesis (40). The reaction catalyzed by these enzymes is complex and involves isomerization of the double bond from the 5 to the 4 position and the oxidation of the 3β-hydroxyl to a 3-oxo group (Figure 2, reaction 7). The C₂₇ 3β-HSD enzyme will act only on sterols with a 7α-hydroxyl group (40–42), and thus this step lies downstream of the cholesterol and oxysterol 7α-hydroxylases.

The products of the C₂₇ 3β-HSD enzyme take one of two routes in subsequent steps of bile acid synthesis. If the intermediate is acted upon by sterol 12α-

HSD3B7,  
3β-Hydroxy-Δ5-C₂₇  
Steroid Oxidoreductase  

5-Cholesten-3β,7α-diol  

4-Cholesten-7α-ol-3-one  

AKR1D1, Δ⁴-3-Oxosteroid  
5β-Reductase  

5β-Cholestane-7α-ol-3-one  

AKR1C4,  
3α-Hydroxysteroid  
Dehydrogenase  

5β-Cholestane-3α,7α-diol  

Side chain oxidation,  
chenodeoxycholic acid  
synthesis  

CYP8B1, Sterol  
12α-Hydroxylase  

4-Cholesten-7α,12α-diol-3-one  

AKR1D1,  
Δ⁴-3-Oxosteroid  
5β-Reductase  

5β-Cholestane-7α,12α-diol-3-one  

AKR1C4,  
3α-Hydroxysteroid  
Dehydrogenase  

5β-Cholestane-3α,7α,12α-triol  

Side chain  
oxidation,  
cholic acid  
synthesis  
hydroxylase (Figure 2, reaction 8), a microsomal cytochrome P450 (CYP8B1, Table 1), then the resulting product will be converted ultimately into cholic acid. In the absence of 12α-hydroxylation, a metabolic fate of conversion to chenodeoxycholic acid or another bile acid is met. Two primary bile acids are produced by most vertebrate species; one of which is usually cholic acid and the other a bile acid like chenodeoxycholic acid (rat, human, and hamster), muricholic acid (mouse), ursodeoxycholic acid (bear), or hyodeoxycholic acid (pig). The level of sterol 12α-hydroxylase in the liver determines the relative amounts of the two primary bile acids, and in some species like the mouse, the ratio of one bile acid to another controls output from the biosynthetic pathway because cholic acid mediates feedback regulation of the pathway.

Complementary DNAs and genes encoding sterol 12α-hydroxylase have been isolated from the rabbit (43), human (44), and mouse (45). Transcription of the gene in rodents is subject to many of the same regulatory inputs as the cholesterol 7α-hydroxylase gene (46, 47). Loss of sterol 12α-hydroxylase in mice eliminates cholic acid from the bile acid pool and leads to an increase in the synthesis of muricholates (45). The altered composition of the pool affects the absorption of dietary sterols and the regulation of bile acid synthesis. With no cholic acid and an excess of muricholates, the overall hydrophilicity of the bile acid pool is increased, which causes a decrease in the absorption of dietary cholesterol and a compensating increase in hepatic cholesterol synthesis. In normal mice, cholic acid mediates feedback regulation of bile acid synthesis, and its loss from the sterol 12α-hydroxylase knockout mice causes a derepression of cholesterol 7α-hydroxylase and a corresponding increase in the synthesis of bile acids (see Regulation of Bile Acid Synthesis).

The 12α-hydroxylated intermediates, and those produced by the C₂₇-3β-HSD enzyme that escape 12α-hydroxylation, are subject to reduction of the double bond in the A-ring by the enzyme Δ⁴-3-oxosteroid 5β-reductase (Figure 2, reaction 9). This cytosolic enzyme of 326 amino acids utilizes NADH as a cofactor and is a member (AKR1D1, Table 1) of a large family of proteins, the aldo-keto reductases. These enzymes are present in all species and catalyze oxidation-reduction reactions on endogenous as well as xenobiotic substrates

---

Figure 2 Modifications to the sterol ring structure in bile acid synthesis. Circled numbers designate individual reactions, and the enzymes that catalyze each step are indicated next to the arrows. The chemical modification introduced by a particular enzyme is indicated in red on the product of the reaction. Cumulative changes to sterol intermediates are indicated in green. The products of the 3α-hydroxysteroid dehydrogenase enzyme are substrates for side chain oxidation. Intermediates not acted upon by sterol 12α-hydroxylase (left arm of pathway scheme) are converted ultimately into chenodeoxycholic acid as indicated or into another primary bile acid depending on the species (see text for further details). Intermediates that are acted upon by sterol 12α-hydroxylase (right arm of pathway scheme) are converted ultimately into cholic acid.(48). The enzymes prior to 5β-reductase in the bile acid biosynthetic pathway are located in microsomal or mitochondrial membranes, and thus the cell faces the challenge at this step of transporting intermediates from a hydrophobic environment (the membrane) to a hydrophilic environment (the cytosol). It is not known whether specific transport proteins or 5β-reductase itself mediate this transfer or whether the actions of the prior membrane-bound enzymes increase the hydrophilicity of intermediates to the point that they spontaneously enter the cytosol (see Subcellular Itinerary of Bile Acid Synthesis).

The final step of ring modification involves reduction of the 3-oxo group to an alcohol in the alpha stereochemical configuration and is catalyzed by 3α-hydroxysteroid dehydrogenase (Figure 2, reaction 10). Rat cDNAs encoding an enzyme with this activity specify a soluble protein that is a member (AKR1C4, Table 1) of the aldo-keto reductase family (49, 50). While the rat AKR1C4 enzyme will utilize bile acid intermediates as substrates in vitro and is abundant in the liver, other enzymes also may have this activity in vivo. The aldo-keto reductase family is large and includes multiple members with sequence identity to AKR1C4 that differ in number and enzymatic properties depending on the species (49). The rat AKR1C4 enzyme also reduces C₁₉ and C₂₁ steroid hormones, and thus it is possible that a 3α-hydroxysteroid dehydrogenase with specificity for C₂₇ intermediates in the bile acid pathways remains to be identified.

## Side Chain Oxidation

The products of ring modification next undergo progressive oxidation and shortening of the sterol side chain. The first few steps in this portion of the pathway are performed by sterol 27-hydroxylase (Figure 3, reaction 11), the same mitochondrial cytochrome P450 (CYP27A1) that initiates bile acid synthesis through the formation of 27-hydroxycholesterol (Figure 1, reaction 4). The enzyme introduces a hydroxyl group at carbon 27 and then oxidizes this group to an aldehyde and then to a carboxylic acid. At one time, it was thought that separate aldehyde and alcohol dehydrogenases catalyze the oxidation of the initial hydroxyl group (2), but studies in transfected cells and with purified sterol 27-hydroxylase are consistent with one enzyme performing all three reactions (51–53).

The participation of sterol 27-hydroxylase in the initiation and side chain oxidation steps of the biosynthetic pathways means that all intermediates,

---

**Figure 3** Side chain oxidation in bile acid synthesis. Circled numbers designate individual reactions, and the enzymes that catalyze each step are indicated next to the arrows. The chemical modification introduced by a particular enzyme is indicated in red on the product of the reaction. Cumulative changes to sterol intermediates are indicated in green. The products of side chain oxidation are substrates for conjugation. Sterol 27-hydroxylase catalyzes three sequential oxidation steps in this portion of the pathway.
BILE ACID SYNTHESIS

CYP27A1, Sterol  
27-Hydroxylase  

5β-Cholestane-3α,7α,12α-triol  

CYP27A1, Sterol  
27-Hydroxylase  

5β-Cholestane-3α,7α,12α-triol  

CONJUGATION  

3α,7α,12α-Trihydroxy-5β-  
cholestan-27-al  

3α,7α,12α-Trihydroxy-5β-  
cholestanoic acid  

2-Methylacyl-  
CoA Racemase  

25(R) 3α,7α,12α-Trihydroxy-5β-  
cholestanyl-CoA  

ACOX2,  
Branched-chain  
Acyl-CoA Oxidase  

D-Bifunctional  
Protein  

3α,7α,12α-Trihydroxy-5β-  
cholest-24-enoyl-CoA  

Peroxisomal  
Thiolase 2  

3α,7α,12α-Trihydroxy-5β-  
cholan-24-one-CoA  

25(S) 3α,7α,12α-Trihydroxy-5β-  
cholest-24-enoyl-CoA  

25(S) 3α,7α,12α-Trihydroxy-5β-  
cholest-24-enoyl-CoA

regardless of their origin, must be acted upon by the enzyme prior to formation of a bile acid. As a consequence of the multiple reactions catalyzed by sterol 27-hydroxylase, inactivation of the gene in mice has severe effects on bile acid synthesis (54–57). The bile acid pool size is 25% of normal in the mutant mice; this results in decreased intestinal cholesterol absorption and increased hepatic and peripheral tissue synthesis of cholesterol. Cholesterol 7α-hydroxylase mRNA and activity are greatly increased in these mice (54, 55). The composition of the bile acid pool is not altered (55).

Unexpectedly, plasma cholesterol and triglyceride concentrations are elevated in sterol 27-hydroxylase knockout mice, and both the liver and adrenal are enlarged (55). The dislipidemia appears to have two causes. First, the expression of sterol regulatory element binding proteins 1 and 2 (SREBP-1 and SREBP-2) is elevated in these mice. These transcription factors increase the expression of genes involved in lipid biosynthesis leading to a greater rate of triglyceride synthesis. Second, the reduced bile acid pool causes a decrease in the activity of the farnesoid X receptor (FXR) and a corresponding decline in the expression of apolipoprotein C-II, which is required for the hydrolysis of triglycerides (58). Dietary bile acids normalize plasma lipids and liver size in the mutant mice but have no effects on the enlarged adrenal gland (55).

Humans lacking sterol 27-hydroxylase accumulate cholestanol, a 5α-reduced derivative of cholesterol that is neurotoxic (59). Cholestanol is synthesized by an alternate catabolic pathway that is active when bile acid synthesis is disrupted. In contrast, mice with an induced mutation in the sterol 27-hydroxylase gene do not accumulate cholestanol and consequently do not develop the severe neuropathy characteristic of the human disease (54). Protection in the mouse is due in part to the induction of another cytochrome P450, CYP3A11, which hydroxylates sterol intermediates on carbon 25. This induction reduces the buildup of sterols and compensates for the loss of the sterol 27-hydroxylase in the mutant mice (56, 57). The corresponding enzyme in humans [CYP3A4, (60)] is not induced in patients with sterol 27-hydroxylase deficiency, which may explain the phenotypic difference between mice and humans.

The oxidized bile acid intermediates arising from sterol 27-hydroxylase exit the mitochondria and are next subject to shortening of the side chain. The terminal three carbon atoms are removed in peroxisomes by a series of reactions analogous to those involved in the β-oxidation of fatty acids (61). The first reaction is catalyzed by bile acid coenzyme A ligase (Figure 3, reaction 12), which activates the sterol intermediate by conjugation with coenzyme A. Two enzymes have been identified with this activity, very long-chain coenzyme A synthetase [a 620 amino acid protein of the endoplasmic reticulum and peroxisome (62–64)] and very long-chain acyl-coenzyme A synthetase homolog 2 (65) [alternatively bile acyl-coenzyme A synthetase (66), a related (~45% identical) 690 amino acid protein of the endoplasmic reticulum]. These enzymes also activate very long chain fatty acids containing 18 or more carbons. The very long-chain coenzyme A synthetase is found in liver and kidney and appears to be
largely responsible for the activation of C₂₇ intermediates of bile acid biosynthesis (Figure 3, reaction 12). The very long-chain acyl-coenzyme A synthetase homolog 2 is present only in liver and is involved in the activation of C₂₄ bile acids that are deconjugated in the small intestine and returned to the liver via the enterohepatic circulation (66).

The carbon atom at position 25 of intermediates in the bile acid biosynthetic pathways is prochiral, having two apparently equivalent methyl groups (C26 and C27) as substituents. Sterol 27-hydroxylase, like most enzymes, recognizes the nonequivalence of the two methyl groups and catalyzes a stereospecific hydroxylation event, producing almost exclusively the 25(R) isomer (67, 68). After activation of intermediates with coenzyme A by the bile acid ligase, the 25(R) isomers must be converted into 25(S) isomers before subsequent steps in side chain shortening can take place. This reaction is catalyzed by 2-methylacyl-coenzyme A racemase (Figure 3, reaction 13), an enzyme located in both mitochondria and peroxisomes (69, 70). This protein can be purified in the absence of detergents (71), which suggests that it is not membrane bound. The 2-methylacyl-coenzyme A racemase also acts on branched chain fatty acids like phytanic acid derived from the catabolism of isoprenoids. Consequently, mutations in the encoding gene result in the accumulation of both bile acid intermediates and phytanic acid (see Genetics of Bile Acid Synthesis).

The sterol products of the racemase enzyme are next subject to dehydrogenation catalyzed by the FAD-containing peroxisomal enzyme branched chain acyl-coenzyme A oxidase to yield 24,25-trans-unsaturated derivatives (Figure 3, reaction 14). During the course of this reaction, the enzyme transfers electrons to molecular oxygen and produces hydrogen peroxide as a byproduct. There are two related (45% sequence identity) acyl-coenzyme A oxidase enzymes in humans and mice, abbreviated as ACOX1 and ACOX2. ACOX1 is a peroxisomal proliferator activated receptor-α (PPARα) target gene that dehydrogenates straight chain fatty acids and eicosanoids (72, 73). Mice that lack the ACOX1 gene accumulate long chain fatty acids, have reduced numbers of peroxisomes, and develop fatty livers but do not appear to have defects in bile acid synthesis (74, 75). A similar phenotype is observed in humans with ACOX1 deficiency (76). ACOX2 also may be regulated by PPARα and acts on intermediates in the bile acid pathway and 2-methyl-branched chain fatty acids such as pristanic acid (77, 78). The consequences of an ACOX2 knockout in mice have not yet been reported. Three ACOX enzymes exist in the rat, and these have substrate specificities that are different from those of the orthologous human and mouse enzymes (73).

The next step in the biosynthetic pathway involves hydration and oxidation at the Δ²⁴ bond and is catalyzed by the D-bifunctional protein (Figure 3, reaction 15). This remarkably complex peroxisomal enzyme of 736 amino acids catalyzes both a hydration step in which a molecule of water is added across the double bond to form a C₂₄-alcohol intermediate and the subsequent oxidation of the alcohol to form the C₂₄-oxo product shown in Figure 3.Different domains in the amino-terminal half of the enzyme catalyze these two reactions, while the carboxyl-terminal half of the enzyme contains a domain postulated to be involved in sterol transport (76, 79). Mice deficient in the D-bifunctional protein gene accumulate unsaturated C₂₇ bile acid intermediates and very long chain fatty acids such as pristanic and phytanic acid (77). They continue to synthesize some C₂₄ bile acids; this indicates that another enzyme can also perform this step in the pathway. The compensating enzyme is thought to be the related L-bifunctional protein, which normally utilizes long chain fatty acids as substrates (77).

The last step in the oxidation of the side chain of bile acid intermediates is catalyzed by peroxisomal thiolase 2 (Figure 3, reaction 16), which cleaves the C₂₄-C₂₅ bond to form propionyl-coenzyme A and a C₂₄-coenzyme A bile acid intermediate. This enzyme is also referred to as sterol carrier protein-chi (SCPχ) in the literature (80) and is encoded by a gene with two promoters (81). Transcription from sequences flanking exon 1 of the gene produces a mRNA that encodes a 547 amino acid peroxisomal thiolase 2 precursor protein, which following import into the peroxisome is cleaved after amino acid 424 to produce the mature active enzyme. The carboxyl-terminal 123 amino acid product of this cleavage event is referred to as sterol carrier protein 2. X-ray crystallography studies show that this fragment is capable of binding a variety of lipids (82). The second promoter is located in intron 11 and produces a mRNA that encodes only sterol carrier protein 2 (81). Mice deficient in the peroxisomal thiolase 2 gene accumulate bile acid intermediates, C₂₃ bile acids, and branched chain fatty acids (83, 84), which indicates that both the full-length thiolase enzyme and the truncated sterol carrier protein 2 are involved in bile acid synthesis and the metabolism of branched chain fatty acids (85). The knockout mice contain some normal C₂₄ bile acids; this suggests the presence of another thiolase with activity towards sterols (84).

## Conjugation

The terminal step in bile acid synthesis involves the addition of an amino acid, usually glycine or taurine, in amide linkage to carbon 24 (Figure 4, reaction 17). The reaction is catalyzed by the bile acid coenzyme A:amino acid N-acyltransferase enzyme (Table 1). The N-acyltransferase is a remarkably efficient enzyme as more than 98% of bile acids excreted from the liver are amidated. This enzyme, like the preceding four enzymes in the pathway, is located in the peroxisomes (86). The substrates of the N-acyltransferase are a bile acid coenzyme A thioester and either taurine [mice (87)] or glycine and taurine [humans (88)]. The ratio of glycine to taurine conjugated bile acids in humans is solely dependent on the relative abundance of the two amino acids and appears to have no functional or regulatory consequences. The ratio of free to conjugated bile acids may be regulated by PPARα, which activates a peroxisomal enzyme, coenzyme A thioesterase 2, that catalyzes the hydro-
Figure 4 Conjugation of bile acids. A circled number designates the reaction, and the enzyme that catalyzes this step is indicated next to the arrow. The chemical modifications introduced by the enzyme are indicated in red on the products of the reaction. Cumulative changes to sterol intermediates are indicated in green. Depending on the species of origin, the bile acid CoA:amino acid N-acyltransferase will utilize either glycine or taurine or both amino acids as substrates for conjugation. Conjugated bile acids are secreted from the liver into the bile by ABC transporters and other proteins located in the canalicular membrane.

lysis of bile acid coenzyme A thioesters into free bile acids and coenzyme A (89). The biological consequences arising from changes in this ratio are not known.

Conjugation of bile acids increases the amphipathicity and enhances the solubility of the molecules, which makes them impermeable to cell membranes. Oxygens on the sulfur of taurocholic acid and the terminal carbon of glycocholic acid are ionized at physiological pH, which together with the planar structure of the bile acid and the hydroxyl groups on the rings renders the bile salt (the ionized form of the bile acid) very amphipathic. Conjugation of cholic acid with glycine reduces the pK from 6.4 to 4.4 ensuring that the bile acid is completely ionized and highly soluble. Before conjugation, the monohydrate form of cholic acid is soluble to a concentration of 0.28 g/l in water at 15°C, whereas the sodium salt of free cholic acid is soluble to >569 g/l (90). The sodium salt of glycocholic acid is somewhat less soluble at 274 g/l, which may protect the gut and liver from the exceptionally strong detergent properties of the free bile acid. Because conjugated and free bile

acids do not cross cell membranes, dedicated transport systems such as the ileal bile acid transporter and members of the ABC family of transporters are required to move bile acids into and out of cells (91, 92). The need for a transport system increases the half-lives of bile acids in the enterohepatic circulation and protects cells that are otherwise ill equipped to handle the detergent properties of these molecules.

# SUBCELLULAR ITINERARY OF BILE ACID SYNTHESIS

As intermediates progress down the bile acid biosynthetic pathways they gradually increase in hydrophilicity to the point that the final conjugated bile acids are exceptionally soluble in water. When these changes in chemical properties are considered together with the distribution of the biosynthetic enzymes in the endoplasmic reticulum, cytosol, mitochondria, and peroxisome (Table 1) and the mass of cholesterol converted into bile acids each day (~500 mg), it is clear that the hepatocyte faces an enormous challenge in transporting intermediates and products of the pathway throughout its interior, and finally, to the exterior of the cell.

How intermediates are moved from one compartment to another remains uncharacterized. In the case of steroid hormone biosynthesis, the rate-limiting step is the movement of cholesterol to the first enzyme in the pathway, the CYP11A1 side chain cleavage enzyme. This transport is accomplished by the steroidogenic acute regulatory (StAR) protein, whose expression is restricted to tissues that synthesize large quantities of steroids (93). Although the expression of StAR in cultured cells stimulates the formation of 27-hydroxycholesterol by sterol 27-hydroxylase (Figure 1, reaction 4) (94, 94a), the StAR transporter is not detectable in the liver. StAR is but one member of a family of proteins referred to as STARTS that are thought to be involved in intracellular cholesterol movement (95). It is possible that one or more START family members is involved in the intracellular transport of bile acid intermediates.

Many steps in the oxidation and shortening of the side chain of bile acid intermediates resemble similar reactions in the fatty acid oxidation pathway. Moreover, the enzymes that catalyze these steps in bile acid synthesis also utilize branched chain fatty acids as substrates (Table 1). This analogy suggests that a transport system similar to the carnitine/carnitine palmitoyl transferase machinery used to import fatty acids into the mitochondria may exist to shuttle bile acid intermediates between intracellular compartments; however, no covalently modified derivatives of bile acid intermediates that could represent a transport form have been reported in mass spectrometry studies.

Adrenoleukodystrophy, a human disease, is caused by mutations in the ABCD1 gene, which encodes a member of the ABC transporter family that is implicated in the import of very long chain fatty acids into peroxisomes (96). Similarly, genetic studies in yeast (97–99) and Arabidopsis (100) indicate that

several different ABC transporters are required for the metabolism of fatty acids in these species. With respect to sterol movement, mutations in yeast genes encoding ABC transporters influence the uptake of sterols (100a) and the ability of glucocorticoids to modulate the activity of an ectopically expressed glucocorticoid receptor in this species (101, 102). The ABC transporters active in the latter regard (LEM gene products) pump glucocorticoids out of the cell, thereby reducing the activity of the glucocorticoid receptor. By analogy, ABC transporters may export bile acid intermediates out of the various subcellular compartments and into the cytosol en route to another organelle.

The availability of cDNAs encoding all of the enzymes in the bile acid pathway will enable the development of in vitro systems that reconstitute the pathway and allow the biochemical purification of proteins involved in transport. Alternatively, expression cloning approaches in cells that do not normally synthesize bile acids may identify candidate importers and exporters of bile acid intermediates.

# REGULATION OF BILE ACID SYNTHESIS

## Role of Nuclear Receptors

It has been known since the late 1960s that the amount of bile acid synthesized by the liver is regulated precisely (103, 104). When bile acids accumulate, synthesis is reduced by a negative feedback mechanism that decreases the expression of two enzymes in the biosynthetic pathway, cholesterol 7α-hydroxylase (Figure 1, reaction 1) and sterol 12α-hydroxylase (Figure 2, reaction 8). Conversely, cholesterol accumulation induces bile acid synthesis by activating cholesterol 7α-hydroxylase in some but not all species. The regulatory responses of cholesterol 7α-hydroxylase were shown to be mediated at the transcriptional level in the late 1980s (2). Since that time, transcription factors that mediate negative and positive feedback regulation of bile acid synthesis have been identified, and the mechanisms by which these proteins act have been elucidated.

A striking finding from this body of work is that many of the transcription factors regulating the expression of the cholesterol 7α-hydroxylase and sterol 12α-hydroxylase genes are nuclear receptors (Figure 5). Suppression is triggered by the binding of bile acids (when in excess) to the farnesoid X receptor (FXR, NR1H4), which then activates transcription of the short heterodimeric partner (SHP, NR0B2) gene, a second nuclear receptor (105, 106). SHP binds to and inhibits a third receptor, the liver receptor homologue-1 (LRH-1, NR5A2), which normally activates the genes encoding cholesterol 7α-hydroxylase and sterol 12α-hydroxylase (47, 107, 108). In the absence of LRH-1 activity, transcription from these two genes decreases, and consequently, the synthesis of bile acids declines. The increase in bile acid synthesis that occurs when cholesterol accumulates in rodents is mediated by the liver X receptor α (LXRα, NR1H3),

Cholesterol
SHP Independant
Pathways

LXR
LRH-1
T
SHP
FXR
Bile Acids
Oxysterols

Cholesterol
7α-Hydroxylase
Sterol 12α-
Hydroxylase

Other Nuclear
Receptors

Figure 5 Regulation of bile acid synthesis by nuclear hormone receptors and other inputs. Black arrows indicate the different pathways by which cholesterol is converted into bile acids. Green arrows indicate positive regulatory inputs, and red brakes indicate negative inputs. The nuclear receptor acronyms used are: LXR, liver X receptor α; LRH-1, liver receptor homologue-1; SHP, short heterodimer partner; FXR, farnesoid X receptor.

a nuclear receptor that is activated on binding oxysterols (109–111). In conjunction with LRH-1, LXRα stimulates transcription from the cholesterol 7α-hydroxylase gene (105, 106). Three retinoid X receptors (RXRα, β, and γ; NR2B1–3) serve as coreceptors for FXR and LXR, which brings the basal number of receptors that activate and suppress bile acid synthesis to seven.

The individual contributions of nuclear receptors to the regulation of bile acid synthesis are described below.

## Farnesoid X Receptor

Suppression of bile acid synthesis is mediated by FXR, which binds bile acids and activates the transcription of genes involved in bile acid and lipid metabolism (112–114). Target genes include those encoding the ileal bile acid binding protein (112), SHP (105, 106), the phospholipid transfer protein (115), several ABC transporters (116, 117), the organic anion transporting polypeptide 8 (118), and apolipoprotein C-II (58). Mice deficient in FXR express high levels of cholesterol 7α-hydroxylase and sterol 12α-hydroxylase mRNAs; this results in increased bile acid synthesis (119). The mutant mice accumulate bile acids in plasma due to markedly decreased levels of ABCB11, which normally transfers bile acids from the hepatocyte into the bile (92). FXR knockout mice also are hypertriglyceridemic, due in part to decreased expression of apolipoprotein C-II, which is required for the metabolism of triglycerides (58). High concentrations of dietary bile acids cause death in these mice, most likely due to liver failure associated with the loss of ABCB11 expression (119).

FXR agonists that mimic the ability of bile acids to activate the receptor have been identified. The best characterized of these is GW4064, an isoxazole derivative with high selectivity and nanomolar affinity for the receptor (120). When administered to rats, the compound decreases hepatic cholesterol 7α-hydroxylase mRNA levels and reduces serum triglyceride levels (106). A naturally occurring FXR antagonist, guggulsterone, also has been identified. Guggulsterone inhibits transcription from FXR-responsive genes in cultured cells (121, 122) and, when administered to cholesterol-fed mice, prevents the accumulation of cholesterol in the liver (121). The mechanism responsible for the reduction in hepatic cholesterol associated with the administration of guggulsterone remains to be defined.

The bile acid pool of an organism contains twenty or more different bile acids that vary in their abilities to activate FXR, in their susceptibilities to metabolism by the gut flora and in their half-lives in the enterohepatic circulation. In vitro studies indicate that some bile acids, such as chenodeoxycholic acid, are potent FXR ligands, whereas others such as cholic acid, ursodeoxycholic acid, and β-muricholic acid are less active (112–114). Differences in potency can be due to the failure of hydrophilic bile acids to gain entry into cells lacking appropriate transporters (114) or due to differences in the relative affinities of the various bile acids for the nuclear receptor. An example of the latter phenomenon is observed in knockout mice lacking sterol 12α-hydroxylase, which cannot make cholic acid and have unregulated synthesis of bile acids. This derepression is attributable to a failure of the bile acids made in the mutant mice, chiefly muricholic acid derivatives, to activate FXR and hence to mediate feedback regulation (45). Structural differences between receptors isolated from different species also determine the effective concentration of a given bile acid (123); this suggests that a bile acid active in one species may be inactive in another.

Ligand potency is a function of many variables that include the relative affinities of the ligand for the nuclear receptor, of the nuclear receptor for the DNA response element, and of the receptor for auxiliary proteins, such as chaperones, coactivators, and corepressors. The half-life of the ligand also affects its potency, and in the case of bile acids, this parameter is determined by how long the bile acid persists in the enterohepatic circulation and its susceptibility to metabolism by bacteria in the intestinal tract. The relative effects of each of these mechanisms on the regulation of bile acid synthesis remains to be determined, but clearly the possible means by which output from the biosynthetic pathways can be regulated are numerous.

## Short Heterodimer Partner

Most nuclear receptor response elements are positive; this means transcription of the target gene is increased when it is bound by a ligand-activated receptor. Although negative response elements are known, the mechanisms by which these act often involve the participation of a second transcription factor rather than the receptor acting as a classic repressor like those defined in bacteria. In agreement
with this general rule, FXR suppresses the expression of bile acid synthesis indirectly by activating the transcription of SHP, an unusual nuclear receptor that lacks a DNA binding domain (124). The promoter for the SHP gene contains an FXR response element composed of inverted repeats separated by a single base pair (105, 106), which explains the ability of FXR to stimulate SHP transcription. SHP in turn suppresses bile acid synthesis by binding to and inhibiting LRH-1, which is required for expression of the cholesterol 7α-hydroxylase and sterol 12α-hydroxylase genes (46, 47, 105, 106).

In addition to the FXR binding site in the SHP promoter, there is a functional LRH-1 response element (105, 106, 125). This arrangement provides an autoregulatory mechanism by which the SHP-dependent inhibition of bile acid synthesis can be attenuated (Figure 5). In this loop, the accumulation of SHP protein results in a gradual titration of LRH-1 activity, which in turn decreases SHP transcription and allows the system to return to baseline.

Chow-fed mice deficient in SHP have elevated levels of cholesterol 7α-hydroxylase and sterol 12α-hydroxylase mRNAs, increased synthesis of bile acids, and a larger bile acid pool size (126, 127), which are phenotypic traits consistent with SHP repressing bile acid synthesis. Nevertheless, cholesterol 7α-hydroxylase and sterol 12α-hydroxylase mRNA levels fall when SHP knockout mice are fed bile acids but not when a specific FXR agonist (GW4064) is administered to the mice. These results indicate that additional, SHP-independent, pathways exist that are capable of suppressing bile acid synthesis (see Additional Regulatory Responses).

SHP also inhibits the activity of several other nuclear receptors in transfected cells via direct protein-protein interactions (124). One of these nuclear receptor targets may be hepatic nuclear factor-4α (HNF-4α, NR2A1). In vitro studies indicate that HNF-4α activates transcription from the sterol 12α-hydroxylase promoter (128), and consistent with these findings, HNF-4α-deficient mice express low levels of the 12α-hydroxylase mRNA. The increased expression of sterol 12α-hydroxylase in the SHP-deficient mice may thus result from derepression of either of the positive transcription factors, HNF-4α or LRH-1.

## Liver Receptor Homologue-1

Cholesterol 7α-hydroxylase is expressed exclusively in the liver (7). This tissue-specific pattern of expression is mediated by *cis*-acting regulatory sequences in the gene (129, 130), which bind the transcription factor LRH-1 (107). LRH-1 has been isolated several times and each time given a different name. The initial description appeared as a submission (M813985) to GenBank in 1991 with the name liver receptor homologue-1 (LRH-1). Orthologous cDNAs were isolated thereafter from several sources and given various names, including from human (PHR-1) (131), *Xenopus* (xFF1rA) (132), rat (FTF) (133), and human again (CPF) (107). The term LRH-1 is used here to denote the protein's christened name.
LRH-1 is expressed in the liver, ovary, small intestine, pancreas, and colon of the adult mouse (134) and shares sequence identity with steroidogenic factor-1 (SF-1, NR5A1), a nuclear receptor that directs the tissue-specific expression of genes involved in gonadal and adrenal steroidogenesis (135). SF-1 was shown previously to be antagonized by another member of the nuclear receptor family termed DAX-1 (136), which prompted the discovery that antagonism of LRH-1 by SHP was the mechanism by which bile acids mediated feedback regulation (105). The finding that hepatocytes treated with bile acids or the FXR agonist GW4064 induced the expression of SHP was the impetus for this discovery in another laboratory (106). Inhibition involves the binding of SHP to the carboxy-terminal *trans*-activation domain of LRH-1, which prevents the latter from interacting with coactivator proteins (108).

Conventional elimination of LRH-1 from mice causes early embryonic lethality (T.A. Kerr and D.W. Russell, unpublished observations), and thus the expectations for bile acid synthesis arising from loss of this factor cannot be tested in this manner. The application of conditional knockout strategies is underway and should provide both confirmation and further insight into the biological roles of LRH-1.

## Liver X Receptor α

That cholesterol could induce the expression of cholesterol 7α-hydroxylase and hence bile acid synthesis was initially controversial; however, the isolation of cDNAs encoding this enzyme allowed an unambiguous demonstration that this is the case in rats (7) and mice (137). Substrate-mediated induction occurs via LXRα, which binds oxysterol ligands and induces transcription of the cholesterol 7α-hydroxylase gene (109–111). An LXRα response element is located in the 5′-flanking region of the mouse and rat 7α-hydroxylase genes (110), but this element is either missing in species that do not respond to cholesterol (e.g., human and rabbit) or present but nonfunctional in other nonresponding species (e.g., hamster) (138).

Mice deficient in LXRα appear phenotypically normal until challenged with diets high in cholesterol, which cause dramatic accumulation of the sterol in the liver and eventually death (139). The mutant mice fail to induce cholesterol 7α-hydroxylase and thus are unable to convert excess cholesterol into bile acids. These results explain in part the resistance of plasma cholesterol levels to dietary cholesterol in rodents, and conversely, the sensitivity of other species such as rabbits and humans that fail to induce cholesterol 7α-hydroxylase when a high cholesterol diet is consumed (140, 141). There are two LXR genes in mammals, one encoding LXRα and the other encoding LXRβ (NR1H2) (142). LXRα is more highly expressed in the liver than LXRβ (134), and in agreement with this expression pattern, mice lacking LXRβ do not exhibit dislipidemia when fed cholesterol (143). Mice that lack both LXRα and -β have a more severe phenotype than those without LXRα, which suggests that LXRβ is able to compensate partially for LXRα (144).

LXR activates expression from the cholesterol 7α-hydroxylase gene but does not induce transcription from the sterol 12α-hydroxylase gene. In contrast to 7α-hydroxylase, sterol 12α-hydroxylase is suppressed by cholesterol feeding in murine species (145). As a consequence, the bile acid profile changes so proportionally less cholic acid is synthesized. Cholic acid is a potent ligand for FXR, which means that a decrease in the production of this bile acid attenuates the negative feedback regulation of bile acid synthesis and facilitates cholesterol disposal (45). Moreover, cholic acid enhances the solubilization and absorption of cholesterol in the small intestine, and a reduction in its level decreases the delivery of dietary cholesterol to the liver in the face of a cholesterol challenge.

LXR plays a crucial role in integrating the pathways of cholesterol supply and catabolism by regulating the expression of the sterol response element binding protein-1c (SREBP-1c) gene. SREBP-1c is a transcription factor that activates many genes involved in cholesterol and lipid biosynthesis (146). As cholesterol accumulates, LXR induces the expression of SREBP-1c, which in turn activates the synthesis of enzymes that produce fatty acids (144). These fatty acids are then utilized in the formation of cholesteryl esters, the intracellular storage form of cholesterol. LXR also stimulates the transcription of genes encoding cholesterol efflux proteins [the ABCA1, ABCG5, and ABCG8 transporters (147, 148)], and in this manner enhances the direct secretion of the sterol from the liver. Numerous other genes involved in lipid metabolism are targets of LXR (149); these include the cholesterol ester transport protein, ABCG1, apolipoprotein E, lipoprotein lipase, and the phospholipid transfer protein (150), and the activation of these contributes to the integration of the supply and catabolism pathways.

Synthetic agonists that activate LXR have been developed, initially as potential therapeutic agents to treat hyperlipidemias. These compounds activate LXR in vitro and induce the expected transcriptional responses of the receptor in vivo (147, 151, 152). Treatment of mice and hamsters with one of these agonists (T0901317), however, is reported to increase plasma cholesterol and triglyceride levels (151). Although the mechanism responsible for these increases is not known, it may be that the effects of stimulation of SREBP-1c by LXR and the ensuing increase in lipid synthesis predominate over the stimulation of cholesterol 7α-hydroxylase and bile acid synthesis. An added challenge in the development of LXR therapeutics is that activation of this receptor in human hepatocytes causes a decrease in cholesterol 7α-hydroxylase expression (153). Thus, species-specific regulatory responses must be taken into account. LXR agonists may find application as antiatherogenic agents in macrophages where they promote the excretion of cholesterol from the vessel wall (154, 155); alternatively, receptor antagonists may be useful in lowering plasma lipid levels.

## Retinoid X Receptor

LXR and FXR form obligate heterodimers with RXR proteins. The activities of the heterodimeric LXR and FXR receptors with respect to the regulation of bile acid synthesis are modulated by ligands that interact with the RXR subunit (147).
Mice deficient in hepatic RXRα express high levels of cholesterol 7α-hydroxylase, which is consistent with the loss of FXR-mediated feedback inhibition, and they fail to induce expression of the 7α-hydroxylase gene in response to dietary cholesterol, which is consistent with reduced LXRα input (156). Two additional RXR isoforms, β and γ, are expressed in the liver (157), but the above results suggest that RXRα is the predominant heterodimerization partner for nuclear receptors in this tissue.

# Other Nuclear Receptors

The expression of cholesterol 7α-hydroxylase is increased while that of sterol 12α-hydroxylase is decreased in rats administered thyroid hormone (145, 158–160). Thyroid hormone response elements in the rat cholesterol 7α-hydroxylase promoter have not yet been defined, but a negative element has been identified in the human gene (161), which suggests that induction of this gene may result from the alleviation of inhibition rather than ligand-dependent stimulation of transcription. The mechanism by which thyroid hormone represses 12-hydroxylase expression is not known (160). Analyses in receptor knockout mice suggest that thyroid hormone receptor β (NR1A2) is responsible for the activation of the cholesterol 7α-hydroxylase gene (162).

Mice with an induced hepatic deficiency of the nuclear receptor HNF-4α (NR2A1) have markedly reduced expression of cholesterol 7α-hydroxylase (163). Many genes involved in lipid metabolism have altered expression patterns in these animals, and thus it is not clear whether the response of cholesterol 7α-hydroxylase is direct or indirect. The expression of the CYP7B1 oxysterol 7α-hydroxylase and sterol 12α-hydroxylase genes is reduced to very low levels in the HNF-4α-deficient mice, and in vitro studies indicate that these two genes are direct targets of the receptor. The effects of these changes on bile acid metabolism in the mutant mice remain to be determined, but the prediction based on results from other knockout animals is that oxysterols will accumulate and that cholic acid levels will be very low in the HNF-4α-deficient mice (27, 45). Certain cytokines decrease the expression of cholesterol 7α-hydroxylase (see Additional Regulatory Responses), and in vitro studies indicate that these act by decreasing the activity of HNF-4α (164).

Agonists of PPARα (NR1C1) such as fibrates have multiple effects on bile acid metabolism (165). In rats and mice, dietary fibrates decrease the expression of cholesterol 7α-hydroxylase resulting in a reduction in bile acid secretion (166). In vitro studies indicate that PPARα antagonizes the positive effects of HNF4-α on the promoter of the cholesterol 7α-hydroxylase gene (167, 168). PPARα-deficient mice express normal levels of cholesterol 7α-hydroxylase suggesting that the input of this receptor is modest under physiological conditions (168). However, these animals are unresponsive to dietary fibrates (166, 168), and this indicates the response of bile acid synthesis to these agents is receptor mediated. The expression of sterol 12α-hydroxylase is increased by fibrates in the mouse (169), which would have the net effect of elevating the levelof cholic acid in the bile acid pool and of reducing the expression of cholesterol 7α-hydroxylase via activation of the FXR-SHP pathway (45).

When certain bile acids, such as lithocholic acid, accumulate in an organism, they induce a number of responses that reduce bile acid synthesis in an effort to protect the liver. These responses are mediated *inter alia* by three nuclear receptors, the pregnane X receptor (PXR, NR1I2), the constitutive androstane receptor (CAR, NR1I3), and the vitamin D receptor (VDR, NR1I1). Lithocholic acid is an agonist for PXR and VDR, and through these receptors stimulates the transcription of cytochrome P450 and sulfotransferase genes whose products detoxify the bile acid (170–172). In addition, activated PXR reduces the expression of cholesterol 7α-hydroxylase by an as yet undefined mechanism (170, 173). The regulatory contributions of CAR to bile acid synthesis remain to be determined, but the ligand specificity of this receptor overlaps with that of PXR (174), which suggests an important role in this regard.

## Additional Regulatory Responses

The actions of nuclear receptors underlie many of the responses of bile acid synthesis to various dietary and physiological inputs. There are clearly other regulatory pathways that impinge on synthesis and for which the identities of the participating proteins are more or less well defined. Underlying transcription factors have been identified for pathways that involve hepatic nuclear factor-1α (HNF-1α) and two proteins involved in circadian rhythm, the D site binding protein (DBP) and the adenovirus E4 promoter ATF site binding protein (E4BP4). Mice that lack HNF-1α have elevated levels of cholesterol 7α-hydroxylase and increased bile acid synthesis (175). The loss of feedback regulation in these animals is due in part to decreased expression of FXR and a corresponding decline in SHP expression. Cholesterol 7α-hydroxylase is subject to diurnal regulation mediated by DBP and E4BP4 (176, 177), and the expression of DBP itself is regulated by circadian rhythm in the liver as well as regions of the brain involved in an organism's response to light (177, 178). This level of control integrates the expression of cholesterol 7α-hydroxylase and bile acid synthesis with food intake and exposure to light, which are postulated to be at the ancestral heart of metabolic gene regulation (179).

Evidence for additional regulatory pathways with as yet undefined mediators comes from the analysis of mice that lack SHP, the central repressor in the scheme shown in Figure 5. Based on this model, the removal of SHP should result in the constitutive expression of cholesterol 7α-hydroxylase and sterol 12α-hydroxylase, and it should eliminate the response of these genes to dietary bile acids. Experiments in two independent lines of knockout mice reveal that SHP-deficient mice exhibit only modest elevations in hydroxylase gene expression (~twofold) and respond to dietary bile acids by decreasing expression of cholesterol 7α-hydroxylase and sterol 12α-hydroxylase (126, 127). The mutant animals do not respond to an FXR agonist (GW4064), which indicates that FXR mediates suppression via activation of SHP as predicted by the model.These data indicate the existence of one or more SHP-independent pathways that negatively regulate bile acid synthesis. The modest elevation in hydroxylase gene expression in the mutant mice is ascribed to a tonic suppression of synthesis mediated by the bile acid pool size. Two observations support this interpretation. First, the pool size is increased by ~30% in these mice and, as a consequence of elevated sterol 12α-hydroxylase expression, has more cholic acid in it. These alterations cause suppression of bile acid synthesis in wild-type mice and in the knockout mice serve to limit derepression of the biosynthetic enzymes. Second, the SHP knockout mice induce cholesterol 7α-hydroxylase and sterol 12α-hydroxylase in response to cholestyramine, a drug whose only known mechanism of action is to reduce the bile acid pool size (126, 127).

A second SHP-independent pathway comes into play when the SHP knockout animals consume bile acids. This damages the liver and induces a stress-response pathway that decreases the expression of bile acid biosynthetic enzymes (126). The c-Jun N-terminal kinase (JNK) is thought to mediate this repression (127, 180).

Additional lines of evidence support the notion that kinases and other intracellular signaling molecules regulate bile acid synthesis. For example, certain strains of mice fail to repress cholesterol 7α-hydroxylase in response to bile acid feeding, and this outcome is correlated with alterations in cytokine secretion, in particular with the levels of tumor necrosis factor and interleukin-1, which repress 7α-hydroxylase expression (181). In addition, mice deficient in the fibroblast growth factor 4 receptor (FGFR4), a cell surface tyrosine kinase, express high levels of cholesterol 7α-hydroxylase and have markedly elevated bile acid pool sizes and fecal excretion rates (182). When fed bile acids and cholesterol, they develop massive liver enlargement, which is correlated with the induction of a corepressor of nuclear hormone receptors termed receptor interacting peptide 140 (RIP140). This induction in turn is postulated to antagonize the positive actions of LXR on the cholesterol 7α-hydroxylase promoter and thus to mimic the phenotype associated with loss of LXR (182).

The involvement of cytokines and a tyrosine kinase receptor in regulating bile acid synthesis means that any number of intracellular signaling molecules may affect the system, which include intracellular kinases, phosphatases, and adaptor proteins. The identification of these regulatory proteins is an active area of current research.

## GENETICS OF BILE ACID SYNTHESIS

Inherited genetic diseases that affect bile acid synthesis provide a unique window on the biological roles of bile acids and of individual enzymes in the pathways. Seven defects involving enzymes that catalyze both early and late steps in the pathways are known (Table 2), and several more have been reported in abstract form. In general, those that affect early biosynthetic steps cause disease in
| Reaction$^a$ | Enzyme | Gene | Chromosome | Phenotype | OMIM (182a) |
| --- | --- | --- | --- | --- | --- |
| 1 | Cholesterol 7$\alpha$-hydroxylase | CYP7A1 | 8q11-q12 | Hypercholesterolemia | 118455 |
| 4 | Sterol 27-hydroxylase | CYP27A1 | 2p23.3-p24.1 | Cerebrotendinous xanthoma-tosis, progressive CNS neuropathy, cholestanol, and bile alcohol accumulation | 606530 |
| 6 | Oxysterol 7$\alpha$-hydroxylase | CYP7B1 | 8q21.3 | Hyperoxysterolemia, neonatal liver failure | 603711 |
| 7 | 3$\beta$-hydroxy-$\Delta^5$-$C_{27}$-steroid oxidoreductase | HSD3B7 | 16p11.2-p12 | Neonatal liver failure, hepato-toxic bile acid intermediate accumulation | 231100 |
| 9 | $\Delta^4$-3-Oxosteroid 5$\beta$-reductase | AKR1C4 | 7q32-q33 | Neonatal liver failure, hepato-toxic bile acid intermediate accumulation | 604741 |
| 13 | 2-Methylacyl-CoA racemase | AMACR | 5pl3.2-q11.1 | Adult onset sensory motor neuropathy, pristanic acid accumulation, neonatal liver disease | 604489 |
| 15 | D-Bifunctional protein | EHHADH | 3q27 | Hypotonia, liver enlargement, developmental defects, pristanic acid, and $C_{27}$ bile acid accumulation | 261515 |

$a$ Reaction numbers refer to Figures 1, 2, and 3.

newborns, while the consequences of those affecting later steps are varied. Several disorders, including Zellweger syndrome, neonatal adrenoleukodystrophy, and infantile Refsum disease, affect peroxisome assembly and thereby impair bile acid synthesis. These diseases are not considered further here, as they do not involve mutations in genes encoding biosynthetic enzymes.

## Cholesterol 7α-Hydroxylase Deficiency

A prismatic pedigree of cholesterol 7α-hydroxylase deficiency is known in which affected individuals present with elevated plasma cholesterol levels, decreased bile acid excretion, and accumulation of cholesterol in the liver (32). A two base pair deletion in exon 6 of the gene (CYP7A1) in these subjects causes a shift in the translational reading frame and the synthesis of a truncated protein with no enzymatic activity. Individuals heterozygous for the mutant allele have modest elevations in serum cholesterol, which suggests that the disorder is inherited in a codominant fashion. The hyperlipidemic phenotype associated with cholesterol 7α-hydroxylase deficiency in humans contrasts with the normolipidemic phenotype of mice that lack this enzyme (12). This difference may reflect alterations in how bile acid synthesis is regulated in the two species, for example, via LXR (153). The hyperlipidemia of human cholesterol 7α-hydroxylase deficiency is resistant to treatment with hydroxymethylglutaryl CoA reductase inhibitors (statins) (32). The effects of bile acid therapy have not yet been tested in these individuals.

## Sterol 27-Hydroxylase Deficiency

Loss of sterol 27-hydroxylase (Figure 1, reaction 4 and Figure 3, reaction 11) causes the neuopathological disorder cerebrotendinous xanthomatosis (CTX) (59). CTX is characterized by the synthesis of abnormal bile alcohols, a reduced synthesis of normal bile acids, and the accumulation of cholesterol and the 5α-reduced derivative of cholesterol, cholestanol, in the blood and tissues of affected individuals. The buildup of the latter sterols in the myelin sheaths surrounding neurons in the brain gradually disrupts the ordered structure of this tissue and causes progressive neurological dysfunction and eventual death. Since an initial description of the molecular basis of this disease in 1991 (183), more than 43 mutations have been described in the CYP27A1 gene that cause CTX (184), and the consequences of an induced mutation in the mouse are well characterized (54–56). There are marked differences between the phenotypes of the mutant mice and human subjects with this disorder, which, as described above, appear to be due to alterations in transcriptional regulatory mechanisms and biosynthetic enzyme expression. CTX is treated effectively with oral bile acid therapy if diagnosed by mass spectrometry or molecular methods at an early age (59).

# Oxysterol 7α-Hydroxylase Deficiency

As with cholesterol 7α-hydroxylase deficiency, one family with CYP7B1 oxysterol 7α-hydroxylase deficiency is known (185). The affected individual presented with liver failure as a newborn, a marked elevation of oxysterols in the serum, and the hyperexcretion of 3β-hydroxy-5-cholestenoic and 3β-hydroxy-5-cholenic acids in the urine. 3β-Hydroxy-5-cholestenoic acid is the expected product arising from repeated side chain oxidations of cholesterol by sterol 27-hydroxylase (Figure 3, reaction 11), and 3β-hydroxy-5-cholenoic acid is derived from cholesterol after undergoing all of the side chain oxidation steps shown in Figure 3. Molecular analysis of this subject's genomic DNA revealed a nonsense mutation in exon 5 of the CYP7B1 oxysterol 7α-hydroxylase gene (CYP7B1) that caused premature truncation of the protein and eliminated enzymatic activity (185).

The chemical phenotype in this individual is consistent with a generalized failure to 7α-hydroxylate oxysterols produced by the alternate pathways of bile acid synthesis (Figure 1, reactions 2, 3, and 4), while the liver failure is consistent with the known hepatotoxic properties of oxysterols and the unsaturated mono-hydroxy bile acids listed above. Cholesterol 7α-hydroxylase protein was present in the liver of this subject as determined by immunoblotting; however, no corresponding enzyme activity was detectable (185). Presumably, the accumulated oxysterols inhibited this enzyme and effected a complete loss of bile acid synthesis. This subject did not respond to oral bile acid therapy and was treated subsequently by liver transplantation. The phenotype of the CYP7B1 oxysterol 7α-hydroxylase deficient individual, together with data showing that the contributions of the alternate pathways to bile acid synthesis are modest in humans (22, 30, 31), suggest that the most important function of this enzyme is to inactivate oxysterols by facilitating their conversion to bile acids.

# 3β-Hydroxy-Δ⁵-C₂₇-Steroid Oxidoreductase Deficiency

Patients with a defect in the C₂₇ 3β-HSD enzyme were first described in 1987 (186). They present in the clinic with neonatal jaundice, liver enlargement, fat-soluble vitamin deficiency, and lipid malabsorption. Mass spectrometry reveals an accumulation in blood and urine of hepatotoxic C₂₄ and C₂₇ steroids with 3β-hydroxy-Δ⁵ structures. Oral bile acid therapy is curative and is thought to cause suppression of cholesterol 7α-hydroxylase and thus prevent further synthesis of the cholestatic intermediates. The encoding C₂₇ 3β-HSD gene (HSD3B7) was isolated in 2000 (40), and since then 12 different mutations that inactivate the enzyme have been identified in affected individuals (J. B. Cheng et al., unpublished observations). Very little, if any, cholic acid and chenodeoxycholic acid are detected by sensitive chemical methods in these subjects, which indicates that the C₂₇3β-HSD enzyme is unique in the bile acid biosynthetic pathway.

$\Delta^{4}$-3-Oxosteroid 5$\beta$-Reductase Deficiency

Loss of this enzyme activity (Figure 2, reaction 9) causes an accumulation of C$_{24}$ bile acids that retain the $\Delta^{4}$-3-oxo structure in the A-ring and C$_{24}$ bile acids that have a 5$\alpha$-reduced configuration (*allo*-bile acids) (187). These sterols are hepatotoxic and over time cause liver failure. The $\Delta^{4}$-3-oxosteroid 5$\beta$-reductase enzyme has been purified to homogeneity, and a putative cDNA cloned (188). Antibodies raised against the protein failed to detect a signal in liver samples from affected individuals (2); however, no formal description of the molecular basis of this disease has appeared to date. With respect to bile acid synthesis, the chemical phenotype of these individuals indicates that the 5$\beta$-reductase acts preferentially on intermediates in the pathway, but when these accumulate in the absence of the enzyme, the structurally unrelated, membrane-bound steroid 5$\alpha$-reductase enzymes (189) are capable of reducing the $\Delta^{4}$ bond to produce *allo*-bile acids. Steroid 5$\beta$-reductase deficiency is effectively treated with oral bile acid therapy.

### 2-Methylacyl-CoA Racemase Deficiency

Ferdinandusse et al. reported two patients with adult-onset sensory motor neuropathy who had elevated plasma levels of pristanic acid, a polyisoprenoid fatty acid, and of two intermediates in the bile acid biosynthetic pathway, 25(R)3$\alpha$,7$\alpha$,12$\alpha$-trihydroxy-5$\beta$-cholestanoic acid and 25(R)3$\alpha$,7$\alpha$-dihydroxy-5$\beta$-cholestanoic acid (190). The accumulation of these compounds was consistent with defects in the 2-methylacyl-CoA racemase (Figure 3, reaction 13), and two point mutations were identified in the encoding gene (AMACR) that inactivated the enzyme. Cultured fibroblasts from these subjects exhibited a reduced ability to oxidize pristanic acid, but the activities of downstream enzymes involved in $\beta$-oxidation, which included the D-bifunctional protein and peroxisomal thiolase 2, were normal. Liver function also was apparently normal in these adult individuals. This suggested that the above C$_{27}$ intermediates were efficient as bile acids. In agreement with this outcome, several species, including alligators and frogs, synthesize 3$\alpha$,7$\alpha$,12$\alpha$-trihydroxy-5$\beta$-cholestanoic acid and 3$\alpha$,7$\alpha$-dihydroxy-5$\beta$-cholestanoic acid as primary bile acids (191). A similar phenotype of progressive peripheral neuropathy is characteristic of infantile Refsum disease in which branched chain fatty acids alone accumulate (192). These observations together suggest that the buildup of C$_{27}$ bile acid intermediates in the racemase deficient subjects does not contribute to the symptoms of the disease.

An infant with 2-methylacyl-CoA racemase deficiency presented a very different clinical picture (192a). In contrast to the adult subjects, this child had coagulopathy, vitamin D and E deficiencies, and mild liver impairment. No neurologic disease was noted. Chemical analysis revealed an accumulation of C$_{27}$ racemase substrates and pristanic acid. A point mutation specifying a missense substitution (S52P) was present in exon 1 of the AMACR gene. Treatment with cholic acid restored liver function and reduced levels of bile acid intermediates. Together, these findings suggest that racemase deficiency causes liver failure in children that resolves prior to adulthood, at which time the progressive accumulation of phytanic acid leads to

neuropathy. The ontological changes in the phenotype of affected individuals suggests that cholic acid therapy is appropriate in childhood and that diets low in branched chain fatty acids may be palliative in adulthood.

# D-Bifunctional Protein Deficiency

Loss of this enzyme activity is associated with the accumulation of C₂₇ bile acid intermediates and pristanic acid, an enlarged liver, developmental defects, hypotonia, and seizures (193–195). Early studies ascribed the biochemical defect in these patients to the absence of the L-bifunctional protein (193); however, later studies showed that mutations in the structurally related D-bifunctional protein were the cause of this genetic disease (196–198). Some, but not all, patients manifest symptoms of liver failure although all show the associated neurological deficiencies (76). This presentation makes it difficult to determine whether the accumulation of bile acid intermediates or pristanic acid is the major contributor to the observed phenotype. An effective treatment for D-bifunctional protein deficiency is not reported.

## FUTURE DIRECTIONS

In conclusion, interest in the synthesis and metabolism of bile acids has continued to grow over the past decade due in large part to advances in the biochemistry, regulation, and genetics of the enzymes involved. Much remains to be learned about the transport of intermediates between subcellular organelles, the roles of nuclear receptors and their various spliced forms and ligands, the identities of other transcription factors and intracellular signaling enzymes that regulate bile acid synthesis, and the physiological importance of tissue specific cholesterol catabolism. The next several years should be equally exciting.

## ACKNOWLEDGMENTS

I thank Trevor Penning for insight into the aldo-keto reductase family of enzymes, and Mike Brown, Joe Goldstein, Helen Hobbs, Steve Kliewer, and David Mangelsdorf for critical review of the manuscript. Research described here from the author's lab is supported by grants from the National Institutes of Health (HL20948, HD38127).

The Annual Review of Biochemistry is online at http://biochem.annualreviews.org

## LITERATURE CITED

1. Carey MC. 1982. *The Liver, Biology and Pathobiology*, pp. 429–65. New York: Raven
2. Russell DW, Setchell KDR. 1992. *Biochemistry* 31:4737–49
3. Hylemon PB, Harder J. 1998. *FEMS Microbiol. Rev.* 22:475–88
3a. Natl. Cent. Biotechnol. Inf. 2002. GenBank. http://www.ncbi.nlm.nih.gov/

4. Chiang JYL. 1998. Front. Biosci. 3:D176–93
5. Ogishima T, Deguchi S, Okuda K. 1987. J. Biol. Chem. 262:7646–50
6. Li YC, Wang DP, Chiang JYL. 1990. J. Biol. Chem. 265:12012–19
7. Jelinek DF, Andersson S, Slaughter CA, Russell DW. 1990. J. Biol. Chem. 265:8190–97
8. Li YC, Chiang JYL. 1991. J. Biol. Chem. 266:19186–91
9. Karam WG, Chiang JYL. 1994. J. Lipid Res. 35:1222–31
10. Nakayama K, Puchkaev A, Pikuleva IA. 2001. J. Biol. Chem. 276:31459–65
11. Miyake JH, Doung X-DT, Strauss W, Moore GL, Castellani LW, et al. 2001. J. Biol. Chem. 276:23304–11
12. Ishibashi S, Schwarz M, Frykman PK, Herz J, Russell DW. 1996. J. Biol. Chem. 271:18017–23
13. Schwarz M, Lund EG, Setchell KDR, Kayden HJ, Zerwekh JE, et al. 1996. J. Biol. Chem. 271:18024–31
14. Arnon R, Yoshimura T, Reiss A, Budai K, Lefkowitch JH, Javitt NB. 1998. Gastroenterology 115:1223–28
15. Schwarz M, Russell DW, Dietschy JM, Turley SD. 1998. J. Lipid Res. 39:1833–43
16. Schwarz M, Russell DW, Dietschy JM, Turley SD. 2001. J. Lipid Res. 42:1594–603
17. Javitt NB. 2000. Biochim. Biophys. Acta 1529:136–41
18. Lund EG, Guileyardo JM, Russell DW. 1999. Proc. Natl. Acad. Sci. USA 96:7238–43
19. Bogdanovic N, Bretillon L, Lund EG, Diczfalussy U, Lannfelt L, et al. 2001. Neurosci. Lett. 314:45–48
20. Deleted in proof
21. Björkhem I, Diczfalussy U, Lutjohann D. 1999. Curr. Opin. Lipidol. 10:161–65
22. Björkhem I, Lutjohann D, Diczfalussy U, Stahle L, Ahlborg G, Wahren J. 1998. J. Lipid Res. 39:1594–600

23. Björkhem I, Andersson U, Ellis E, Alvelius G, Ellegard L, et al. 2001. J. Biol. Chem. 276:37004–10
24. Shanklin J, Cahoon EB. 1998. Annu. Rev. Plant Physiol. Plant Mol. Biol. 49:611–41
25. Lund EG, Kerr TA, Sakai J, Li W-P, Russell DW. 1998. J. Biol. Chem. 273: 34316–48
26. Schroepfer GJ. 2000. Physiol. Rev. 80:361–554
27. Li-Hawkins J, Lund EG, Turley SD, Russell DW. 2000. J. Biol. Chem. 275: 16536–42
28. Dzeletovic S, Breuer O, Lund E, Diczfalussy U. 1995. Anal. Biochem. 225: 73–80
29. Lund E, Björkhem I, Furster C, Wikvall K. 1993. Biochim. Biophys. Acta 1166: 177–82
30. Babiker A, Andersson O, Lindblom D, van der Linden J, Wiklund B, et al. 1999. J. Lipid Res. 40:1417–25
31. Duane WC, Javitt NB. 1999. J. Lipid Res. 40:1194–99
32. Pullinger CR, Eng C, Salen G, Shefer S, Batta AK, et al. 2002. J. Clin. Invest. 110:109–17
33. Li-Hawkins J, Lund EG, Bronson AD, Russell DW. 2000. J. Biol. Chem. 275: 16543–49
34. Li-Hawkins J, Meunch C, Russell DW. 2000. Pathways of bile acid synthesis. Presented at Biol. Bile Acids Health Dis. XVI Int. Bile Acid Meet., ed. GP van Berge Henegouwen, D Keppler, U Leuschner, G Paumgartner, A Stiehl., Dordrecht, Neth.: Kluwer Acad.
35. Ikeda H, Ueda M, Ikeda M, Kobayashi H, Honda Y. 2003. Lab. Investig. 83:349–55
36. Stapleton G, Steel M, Richardson M, Mason JO, Rose KA, et al. 1995. J. Biol. Chem. 270:29739–45
37. Rose KA, Stapleton G, Dott K, Kiény MP, Best R, et al. 1997. Proc. Natl. Acad. Sci. USA 94:4925–30
38. Schwarz M, Lund EG, Lathe R,

Björkhem I, Russell DW. 1997. J. Biol. Chem. 272:23995–24001

39. Martin KO, Reiss AB, Lathe R, Javitt NB. 1997. J. Lipid Res. 38:1053–58

39a. Rose K, Allan A, Gauldie S, Stapleton G, Dobbie L, et al. 2001. J. Biol. Chem. 276:23937–44

40. Schwarz M, Wright AC, Davis DL, Nazer H, Björkhem I, Russell DW. 2000. J. Clin. Investig. 106:1175–84

41. Wikvall K. 1981. J. Biol. Chem. 256: 3376–80

42. Furster C, Zhang J, Toll A. 1996. J. Biol. Chem. 271:20903–7

43. Eggertsen G, Olin M, Andersson U, Ishida H, Kubota S, et al. 1996. J. Biol. Chem. 271:32269–75

44. Gafvels M, Olin M, Chowdhary BP, Raudsepp T, Andersson U, et al. 1999. Genomics 56:184–96

45. Li-Hawkins J, Gafvels M, Olin M, Lund EG, Andersson U, et al. 2002. J. Clin. Investig. 110:1191–200

46. Castillo-Olivares A, Gil G. 2000. J. Biol. Chem. 275:17793–99

47. Castillo-Olivares A, Gil G. 2001. Nucleic Acids Res. 29:4035–42

48. Jez JM, Penning TM. 2001. Chem.-Biol. Interact. 130(1–3):499–525

49. Penning TM. 1997. Endocr. Rev. 18:281–305

50. Usui E, Okuda K, Kato Y, Noshiro M. 1994. J. Biochem. 115:230–37

51. Andersson S, Davis DL, Dahlback H, Jornvall H, Russell DW. 1989. J. Biol. Chem. 264:8222–29

52. Dahlback H, Holmberg I. 1990. Biochem. Biophys. Res. Commun. 167: 391–95

53. Pikuleva IA, Babiker A, Waterman MR, Björkhem I. 1998. J. Biol. Chem. 273:18153–60

54. Rosen H, Reshef A, Maeda N, Lippoldt A, Shpizen S, et al. 1998. J. Biol. Chem. 273:14805–12

55. Repa JJ, Lund EG, Horton JD, Leitersdorf E, Russell DW, et al. 2000. J. Biol. Chem. 275:39685–92

56. Honda A, Salen G, Matsuzaki Y, Batta AK, Xu G, et al. 2001. J. Biol. Chem. 276:34579–85

57. Honda A, Salen G, Matsuzaki Y, Batta AK, Xu G, et al. 2001. J. Lipid Res. 42:291–300

58. Kast HR, Nguyen CM, Sinal CJ, Jones SA, Laffitte BA, et al. 2001. Mol. Endocrinol. 15:1720–28

59. Björkhem I, Boberg KM, Leitersdorf E. 2001. In The Metabolic and Molecular Bases of Inherited Disease, ed. CR Scriver, AL Beaudet, WS Sly, D Valle, B Childs, et al., 2:2961–88. New York: McGraw-Hill. 8th ed.

60. Furster C, Wikvall K. 1999. Biochim. Biophys. Acta 1437:46–52

61. Reddy JK, Hashimoto T. 2001. Annu. Rev. Nutr. 21:193–230

62. Uchiyama A, Aoyama T, Kamijo K, Uchida Y, Kondo N, et al. 1996. J. Biol. Chem. 271:30360–65

63. Berger J, Truppe C, Neumann H, Forss-Petter S. 1998. FEBS Lett. 425:305–9

64. Steinberg SJ, Wang SJ, Kim DG, Mihalik SJ, Watkins PA. 1999. Biochem. Biophys. Res. Commun. 257:615–21

65. Steinberg SJ, Wang SJ, McGuinness MC, Watkins PA. 1999. Mol. Genet. Metab. 68:32–42

66. Mihalik SJ, Steinberg SJ, Pei Z, Park J, Kim DG, et al. 2002. J. Biol. Chem. 277:24771–79

67. Gustafsson J, Sjöstedt S. 1978. J. Biol. Chem. 253:199–201

68. Shefer S, Cheng FW, Batta AK, Dayal B, Tint GS, et al. 1978. J. Biol. Chem. 253:6386–92

69. Kotti TJ, Savolainen K, Helander HM, Yagi A, Novikov DK, et al. 2000. J. Biol. Chem. 275:20887–95

70. Amery L, Fransen M, De Nys K, Man-naerts GP, van Veldhoven PP. 2000. J. Lipid Res. 41:1752–59

71. Schmitz W, Albers C, Fingerhut R, Conzelmann E. 1995. Eur. J. Biochem. 231:815–22

72. Lee SS, Pineau T, Drago J, Lee EJ, Owens JW, et al. 1995. *Mol. Cell. Biol.* 15:3012–22
73. VanVeldhoven PP, Vanhove G, Asselberghs S, Eyssen HJ, Mannaerts GP. 1992. *J. Biol. Chem.* 267:20065–74
74. Fan C-Y, Pan J, Chu R, Lee D, Kluckman KD, et al. 1996. *J. Biol. Chem.* 271:24698–710
75. Infante JP, Tschanz CL, Shaw N, Michaud AL, Lawrence P, Brenna JT. 2002. *Mol. Genet. Metab.* 75:108–19
76. Wanders RJA, Barth PG, Heymans HSA. 2001. See Ref. 59, Vol. II, pp. 3219–56
77. Baes M, Huyghe S, Carmeliet P, Declercq PE, Collen D, et al. 2000. *J. Biol. Chem.* 275:16329–36
78. Vanhove GF, Van Veldhoven PP, Fransen M, Denis S, Eyssen HJ, et al. 1993. *J. Biol. Chem.* 268:10335–44
79. Haapalainen AM, van Aalten DMF, Merilainen G, Jalonen JE, Pirila P, et al. 2001. *J. Mol. Biol.* 313:1127–38
80. Seedorf U, Assmann G. 1991. *J. Biol. Chem.* 266:630–36
81. Ohba T, Holt JA, Billheimer JT, Strauss III. 1995. *Biochemistry* 34: 10660–68
82. Choinowski T, Hauser H, Piontek K. 2000. *Biochemistry* 39:1897–902
83. Seedorf U, Rabbe M, Ellinghaus P, Kannenberg F, Fobker M, et al. 1998. *Genes Dev.* 12:1189–201
84. Kannenberg F, Ellinghaus P, Assmann G, Seedorf U. 1999. *J. Biol. Chem.* 274:35455–60
85. Seedorf U, Ellinghaus P, Nofer JR. 2000. *Biochim Biophys Acta* 1486: 45–54
86. Solaas K, Ulvestad A, Soreide O, Kase BF. 2000. *J. Lipid Res.* 41:1154–62
87. Falany CN, Fortinberry H, Leiter EH, Barnes S. 1997. *J. Lipid Res.* 38: 1139–48
88. Falany CN, Johnson MR, Barnes S, Diasio RB. 1994. *J. Biol. Chem.* 269: 19375–79

89. Hunt MC, Solaas K, Kase BF, Alexson SEH. 2002. *J. Biol. Chem.* 277: 1128–38
90. Budavari S, ed. 1989. *The Merck Index*. Rahway, NJ: Merck & Co. 11th ed.
91. Love MW, Dawson PA. 1998. *Curr. Opin. Lipidol.* 9:225–29
92. Borst P, Elferink RO. 2002. *Annu. Rev. Biochem.* 71:537–92
93. Stocco DM. 2001. *Annu. Rev. Physiol.* 63:193–213
94. Sugawara T, Lin D, Holt JA, Martin KO, Javitt NB, et al. 1995. *Biochemistry* 34:12506–12
94a. Pandak WM, Ren S, Marques D, Hall E, Redford K, et al. 2002. *J. Biol. Chem.* 277:48, 158–64. 10.1074/jbc.M205244200
95. Soccio RE, Adams RM, Romanowski MJ, Sehayek E, Burley SK, Breslow JL. 2002. *Proc. Natl. Acad. Sci. USA* 99:6943–48
96. Moser HW, Smith KD, Watkins PA, Powers J, Moser AB. 2001. See Ref. 59, Vol. II, pp. 3257–3301
97. Shani N, Watkins PA, Valle D. 1995. *Proc. Natl. Acad. Sci. USA* 92:6012–16
98. Shani N, Valle D. 1996. *Proc. Natl. Acad. Sci. USA* 93:11901–6
99. Swartzmann EE, Viswanathan MN, Thorner J. 1996. *J. Cell Biol.* 132: 549–63
100. Zolman BK, Silva ID, Bartel B. 2001. *Plant Physiol.* 127:1266–78
100a. Wilcox LJ, Balderes DA, Wharton B, Tinkelenberg AH, Rao G, et al. 2002. *J. Biol. Chem.* 277:32466–72
101. Kralli A, Bohen SP, Yamamoto KR. 1995. *Proc. Natl. Acad. Sci. USA* 92:4701–5
102. Sitcheran R, Emter R, Kralli A, Yamamoto KR. 2000. *Genetics* 156: 963–72
103. Danielsson H, Einarsson K, Johansson G. 1967. *Eur. J. Biochem.* 2:44–49
104. Shefer S, Hauser S, Mosbach EH. 1968. *J. Lipid Res.* 9:328–33
105. Lu TT, Makishima M, Repa JJ,

Schoonjans K, Kerr TA, et al. 2000. Mol. Cell. 6:507–15

Silva FV, Ordentlich P, et al. 2002. Science 296:1703–6

106. Goodwin B, Jones SA, Price RR, Watson MA, McKee DD, et al. 2000. Mol. Cell. 6:517–26

122. Wu J, Xia CS, Meier J, Li SZ, Hu X, Lala DS. 2002. Mol. Endocrinol. 16:1590–97

107. Nitta M, Ku S, Brown C, Okamoto AY, Shan B. 1999. Proc. Natl. Acad. Sci. USA 96:6660–65

123. Cui J, Heard TS, Yu JH, Lo JL, Huang L, et al. 2002. J. Biol. Chem. 277: 25963–69

108. Lee YK, Moore DD. 2001. J. Biol. Chem. 277:2463–67

124. Seol W, Choi HS, Moore DD. 1996. Science 272:1336–39

109. Janowski BA, Willy PJ, Devi TR, Falck JR, Mangelsdorf DJ. 1996. Nature 383: 728–31

125. Lee YK, Parker KL, Choi HS, Moore DD. 1999. J. Biol. Chem. 274: 20869–73

110. Lehmann JM, Kliewer SA, Moore LB, Smith- Oliver TA, Oliver BB, et al. 1997. J. Biol. Chem. 272:3137–40

126. Kerr TA, Saeki S, Schneider M, Schaefer K, Berdy S, et al. 2002. Dev. Cell 2:713–20

111. Forman BM, Ruan B, Chen J, Schroepfer GJ Jr, Evans RM. 1997. Proc. Natl. Acad. Sci. USA 94:10588–93

127. Wang L, Lee Y-K, Bundman D, Han YQ, Thevananther S, et al. 2002. Dev. Cell 2:721–31

112. Makishima M, Okamoto AY, Repa JJ, Tu H, Learned RM, et al. 1999. Science 284:1362–65

128. Zhang M, Chiang JY. 2001. J. Biol. Chem. 276:41690–99

113. Parks DJ, Blanchard SG, Bledsoe RK, Chandra G, Consler TG, et al. 1999. Science 284:1365–68

129. Goodhart SA, Huynh C, Chen W, Coop er AD, Levy-Wilson B. 1999. Biochem. Biophys. Res. Commun. 266: 454–59

114. Wang HB, Chen J, Hollister K, Sowers LC, Forman BM. 1999. Mol. Cell. 3:543–53

130. Agellon LB, Drover VAB, Cheema SK, Gbaguidi GF, Walsh A. 2002. J. Biol. Chem. 277:20131–34

115. Urizar NL, Dowhan DH, Moore DD. 2000. J. Biol. Chem. 275:39313–17

131. Becker-Andre M, Andre E, DeLamarter JF. 1993. Biochem. Biophys. Res. Commun. 194:1371–79

116. Ananthanarayanan M, Balasubramanian N, Makishima M, Mangelsdorf DJ, Suchy FJ. 2001. J. Biol. Chem. 276: 28857–65

132. Ellinger-Ziegelbauer H, Hihi AK, Laude t V, Keller H, Wahli W, Dreyer C. 1994. Mol. Cell. Biol. 14:2786–97

117. Kast HR, Goodwin B, Tarr PT, Jones SA, Anisfeld AM, et al. 2002. J. Biol. Chem. 277:2908–15

133. Galarneau L, Pare JF, Allard D, Hamel D, Levesque L, et al. 1996. Mol. Cell. Biol. 16:3853–65

118. Jung D, Podvinec M, Meyer UA, Mangelsdorf DJ, Fried M, et al. 2002. Gas troenterology 122:1954–66

134. Repa JJ, Mangelsdorf DJ. 1999. Curr. Opin. Biotechnol. 10:557–63

119. Sinal CJ, Tohkin M, Miyata M, Ward JM, Lambert G, Gonzalez FJ. 2000. Cell 102:731–44

135. Parker KL, Rice DA, Lala DS, Ikeda Y, Luo X, et al. 2002. Recent Prog. Horm. Res. 57:19–36

120. Maloney PR, Parks DJ, Haffner CD, Fivush AM, Chandra G, et al. 2000. J. Med. Chem. 43:2971–74

136. Ito M, Yu R, Jameson JL. 1997. Mol. Cell. Biol. 17:1476–83

121. Urizar NL, Liverman AB, Dodds DT,

137. Duel S, Drisko J, Graf L, Machleder D, Lusis AJ, Davis RA. 1993. J. Lipid Res. 34:923–31

138. Chiang JY, Kimmel R, Stroup D. 2001. *Gene* 262:257–65
139. Peet DJ, Turley SD, Ma W, Janowski BA, Lobaccaro J-MA, et al. 1998. *Cell* 93:693–704
140. Rudel L, Deckelman C, Wilson M, Scobey M, Anderson R. 1994. *J. Clin. Investig.* 93:2463–72
141. Xu G, Salen G, Shefer S, Ness GC, Nguyen LB, et al. 1995. *J. Clin. Invest.* 95:1497–504
142. Alberti S, Steffensen KR, Gustafsson J-A. 2000. *Gene* 243:93–103
143. Alberti S, Schuster G, Parini P, Feltkamp D, Diczfalusy U, et al. 2001. *J. Clin. Investig.* 107:565–73
144. Repa JJ, Liang G, Ou J, Bashmakov Y, Lobaccaro J-MA, et al. 2000. *Genes Dev.* 14:2819–30
145. Vlahcevic ZR, Eggertsen G, Björkhem I, Hylemon PB, Redford K, Pandak WM. 2000. *Gastroenterology* 118:599–607
146. Horton JD, Goldstein JL, Brown MS. 2002. *J. Clin. Investig.* 109:1125–31
147. Repa JJ, Turley SD, Lobaccaro J-MA, Medina J, Li L, et al. 2000. *Science* 289:1524–29
148. Repa JJ, Berge KE, Pomajzl C, Richardson JA, Hobbs H, Mangelsdorf DJ. 2002. *J. Biol. Chem.* 277:18793–800
149. Edwards PA, Kast HR, Anisfeld AM. 2002. *J. Lipid Res.* 43:2–12
150. Cao G, Beyer TP, Yang XP, Schmidt RJ, Zhang Y, et al. 2002. *J. Biol. Chem.* 277:39561–65
151. Schultz JR, Tu H, Luk A, Repa JJ, Medina JC, et al. 2000. *Genes Dev.* 14:2831–38
152. Menke JG, Macnaul KL, Hayes NS, Baffic J, Chao Y-S, et al. 2002. *Endocrinology* 143:2548–58
153. Goodwin B, Watson MA, Kim H., Miao J, Kemper JK, Kliewer SA. 2002. *Mol. Endocrinol.* 17:386–94
154. Claudel T, Leibowitz MD, Fievet C, Tailleux A, Wagner B, et al. 2001. *Proc. Natl. Acad. Sci. USA* 98:2610–15
155. Tangirala RK, Bischoff ED, Joseph SB,
Wagner BL, Walczak R, et al. 2002. *Proc. Natl. Acad. Sci. USA* 99:11896–901
156. Wan Y-JY, An D, Cai Y, Repa JJ, Hung-Po Chen T, et al. 2000. *Mol. Cell. Biol.* 20:4436–44
157. Mangelsdorf DJ, Borgmeyer U, Heyman RA, Zhou JY, Ong ES, et al. 1992. *Genes Dev.* 6:329–44
158. Ness GC, Pendleton LC, Li YC, Chiang JY. 1990. *Biochem. Biophys. Res. Commun.* 172:1150–56
159. Pandak WM, Heuman DM, Redford K, Stravitz RT, Chiang JY, et al. 1997. *J. Lipid Res.* 38:2483–91
160. Andersson U, Yang YZ, Björkhem I, Einarsson C, Eggertsen G, Gafvels M. 1999. *Biochim. Biophys. Acta* 1438:167–74
161. Drover VA, Wong NC, Agellon LB. 2002. *Mol. Endocrinol.* 16:14–23
162. Gullberg H, Rudling M, Forrest D, Angelin B, Vennstrom B. 2000. *Mol. Endocrinol.* 14:1739–49
163. Hayhurst GP, Lee Y-H, Lambert G, Ward JM, Gonzalez FJ. 2001. *Mol. Cell. Biol.* 21:1393–403
164. de Fabiani E, Mitro N, Anzulovich AC, Pinelli A, Galli G, Crestani M. 2001. *J. Biol. Chem.* 276:30708–16
165. Grundy SM, Vega GL. 1987. *Am. J. Med.* 83:9–20
166. Post SM, Duez H, Gervois PP, Staels B, Kuipers F, Princen HM. 2001. *Arterioscler. Thromb. Vasc. Biol.* 21:1840–45
167. Marrapodi M, Chiang JY. 2000. *J. Lipid Res.* 41:514–20
168. Patel DD, Knight BL, Soutar AK, Gibbons GF, Wade DP. 2000. *Biochem. J.* 351:747–53
169. Hunt MC, Yang Y-Z, Eggertsen G, Carneheim CM, Gafvels M, et al. 2000. *J. Biol. Chem.* 275:28947–53
170. Staudinger JL, Goodwin B, Jones SA, Hawkins-Brown D, MacKenzie KI, et al. 2001. *Proc. Natl. Acad. Sci. USA* 98:3369–74
171. Xie W, Radominska-Pandya A, Shi Y,

Simon CM, Nelson MC, et al. 2001. Proc. Natl. Acad. Sci. USA 98:3375–80

172. Makishima M, Lu TT, Xie W, Whitfield GK, Domoto H, et al. 2002. Science 296:1313–16

173. Staudinger J, Liu Y, Madan A, Habeebu S, Klaasen CD. 2001. Drug Metab. Dispos. 29:1467–72

174. Moore LB, Maglich JM, McKee DD, Wisely B, Willson TM, et al. 2002. Mol. Endocrinol. 16:977–86

175. Shih DQ, Bussen M, Sehayek E, Anantharayanan M, Schneider BL, et al. 2001. Nat. Genet. 27:375–82

176. Lavery DJ, Schibler U. 1993. Genes Dev. 7:1871–84

177. Mitsui S, Yamaguchi S, Matsuo T, Ishida Y, Okamura H. 2001. Genes Dev. 15:995–1006

178. Ripperger JA, Shearman LP, Reppert SM, Schibler U. 2000. Genes Dev. 14:679–89

179. Rutter J, Reick M, McKnight SL. 2002. Annu. Rev. Biochem. 71:307–31

180. Gupta S, Stravitz RT, Dent P, Hylemon PB. 2001. J. Biol. Chem. 276:15816–22

181. Miyake JH, Wang S-L, Davis RA. 2000. J. Biol. Chem. 75:21805–8

182. Yu C, Wang F, Kan M, Jin C, Jones RB, et al. 2000. J. Biol. Chem. 275: 15482–89

182a. Natl. Cent. Biotechnol. Inf. 2002. Online Mendelian Inheritance in Man. http://www.ncbi.nlm.nih.gov/omim/

183. Cali JJ, Hsieh C, Francke U, Russell DW. 1991. J. Biol. Chem. 266:7779–83

184. Lee M-H, Hazard S, Carpten JD, Yi S, Cohen J, et al. 2001. J. Lipid Res. 42:159–69

185. Setchell KDR, Schwarz M, O'Connell NC, Lund EG, Davis DL, et al. 1998. J. Clin. Investig. 102:1690–703

186. Clayton PT, Leonard JV, Lawson AM, Setchell KD, Andersson S, et al. 1987. J. Clin. Investig. 79:1031–38

187. Setchell KD, Suchy FJ, Welsh MB, Zimmer-Nechemias L, Heubi J, Balistreri WF. 1988. J. Clin. Investig. 82:2148–57

188. Onishi Y, Noshiro M, Shimosato T, Okuda K. 1991. FEBS Lett. 283:215–18

189. Russell DW, Wilson JD. 1994. Annu. Rev. Biochem. 63:25–61

190. Ferdinandusse S, Denis S, Clayton PT, Graham A, Rees JE, et al. 2000. Nat. Genet. 24:188–91

191. Haslewood GAD. 1967. J. Lipid Res. 8:535–50

192. Wanders RJA, Jakobs C, Skjeldal OH. 2000. See Ref. 59, II:3303–21

192a. Setchell KDR, Heubi JE, Bove KE, O'Connell NC, Brewsbaugh T, et al. 2003. Gastroenterology 124:217–32

193. Watkins PA, Chen WW, Harris CJ, Hoefler G, Hoefler S, et al. 1989. J. Clin. Investig. 83:771–77

194. Suzuki Y, Jiang LL, Souri M, Miyazawa S, Fukuda S, et al. 1997. Am. J. Hum. Genet. 61:1153–62

195. Une M, Konishi M, Suzuki Y, Akashi S, Yoshii M, et al. 1997. J. Biochem. 122:655–58

196. Dieuaide-Noubhani M, Novikov D, Baumgart E, Vanhooren JC, Fransen M, et al. 1996. Eur. J. Biochem. 240:660–66

197. van Grunsven EG, van Berkel E, Ijlst L, Vreken P, de Klerk JBC, et al. 1998. Proc. Natl. Acad. Sci. USA 95:2128–33

198. van Grunsven EG, van Berkel E, Mooijer PAW, Watkins PA, Moser HW, et al. 1999. Am. J. Hum. Genet. 64:99–107

199. Nelson DR, Koymans L, Kamataki T, Stegeman JJ, Feyereisen R, et al. 1996. Pharmacogenetics 6:1–42

CONTENTS

WITH THE HELP OF GIANTS, Irwin Fridovich 1

THE ROTARY MOTOR OF BACTERIAL FLAGELLA, Howard C. Berg 19

ALIPHATIC EPOXIDE CARBOXYLATION, Scott A. Ensign, Jeffrey R. Allen 55

FUNCTION AND STRUCTURE OF COMPLEX II OF THE RESPIRATORY CHAIN, Gary Cecchini 77

PROTEIN DISULFIDE BOND FORMATION IN PROKARYOTES, Hiroshi Kadokura, Federico Katzen, Jon Beckwith 111

THE ENZYMES, REGULATION, AND GENETICS OF BILE ACID SYNTHESIS, David W. Russell 137

PROTEIN-LIPID INTERPLAY IN FUSION AND FISSION OF BIOLOGICAL MEMBRANES, Leonid V. Chernomordik, Michael M. Kozlov 175

THE MANY FACES OF VITAMIN B12: CATALYSIS BY COBALAMIN-DEPENDENT ENZYMES, Ruma Banerjee, Stephen W. Ragsdale 209

SEMISYNTHESIS OF PROTEINS BY EXPRESSED PROTEIN LIGATION, Tom W. Muir 249

MECHANISMS OF ALTERNATIVE PRE-MESSENGER RNA SPLICING, Douglas L. Black 291

COVALENT TRAPPING OF PROTEIN-DNA COMPLEXES, Gregory L. Verdine, Derek P.G. Norman 337

TEMPORAL AND SPATIAL REGULATION IN PROKARYOTIC CELL CYCLE PROGRESSION AND DEVELOPMENT, Kathleen R. Ryan, Lucy Shapiro 367

SIGNALS FOR SORTING OF TRANSMEMBRANE PROTEINS TO ENDOSOMES AND LYSOSOMES, Juan S. Bonifacino, Linton M. Traub 395

THE RNA POLYMERASE II CORE PROMOTER, Stephen T. Smale, James T. Kadonaga 449

THE ESTABLISHMENT, INHERITANCE, AND FUNCTION OF SILENCED CHROMATIN IN SACCHAROMYCES CEREVISIAE, Laura N. Rusche, Ann L. Kirchmaier, Jasper Rine 481

CHALLENGES IN ENZYME MECHANISM AND ENERGETICS, Daniel A. Kraut, Kate S. Carroll, Daniel Herschlag 517

THE DYNAMICS OF CHROMOSOME ORGANIZATION AND GENE REGULATION, David L. Spector 573

TRK RECEPTORS: ROLES IN NEURONAL SIGNAL TRANSDUCTION, Eric J. Huang, Louis F. Reichardt 609

A GENETIC APPROACH TO MAMMALIAN GLYCAN FUNCTION,  
John B. Lowe, Jamey D. Marth  
643  

THE RNA POLYMERASE II ELONGATION COMPLEX, Ali  
Shilatifard, Ronald C. Conaway, Joan Weliky Conaway  
693  

DYNAMICS OF CELL SURFACE MOLECULES DURING T CELL  
RECOGNITION, Mark M. Davis, Michelle Krogsgaard, Johannes B.  
Huppa, Cenk Sumen, Marco A. Purbhoo, Darrell J. Irvine, Lawren C.  
Wu, Lauren Ehrlich  
717  

BIOLOGY OF THE P 21-ACTIVATED KINASES, Gary M. Bokoch  
743  

Proteomics, Heng Zhu, Metin Bilgin, Michael Snyder  
783  

THE STRUCTURAL BASIS OF LARGE RIBOSOMAL SUBUNIT  
FUNCTION, Peter B. Moore, Thomas A. Steitz  
813
