
# Reviews

**Biosynthesis of Antibiotics**

## Vancomycin Assembly: Nature's Way

### Brian K. Hubbard* and Christopher T. Walsh*

#### Keywords:
antibiotics · biosynthesis · glycopeptides · natural products

---

![Diagram of Vancomycin Assembly](attachment://vancomycin_assembly_diagram.png)

---

### Angewandte Chemie

© 2003 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim  
1433-7851/03/4207-0730 $ 20.00+.50/0  
Angew. Chem. Int. Ed. **2003**, 42, No. 7

Biosynthesis of Antibiotics

**A**ntibiotics are precious resources in the fight to combat bacterial infections caused by pathogenic organisms. Vancomycin is one of the antibiotics of last resort in the treatment of life-threatening infections by gram-positive bacteria. The rules by which nature assembles the glycopeptide (vancomycin) and lipoglycopeptide (teicoplanin) antibiotics are becoming elucidated and verified: first amino acids are synthesized, then joined together and cross-linked. This knowledge opens up approaches for reprogramming strategies at the level of altered monomers, swapped assembly lines, and different post-assembly tailoring enzymes.

### 1. Introduction

Antibiotics are precious resources in the fight to combat bacterial infections caused by pathogenic organisms. Vancomycin, the subject of this review, is one of the antibiotics of last resort in the treatment of life-threatening infections by gram-positive bacteria. Table 1 lists a selection of pathogenic gram-positive and gram-negative bacteria and the syndromes or disease states in which these bacteria are common causative agents.[1] These include several bacteria associated with different outbreaks of diarrheal diseases such as Clostridium difficile, Escherichia coli, Salmonella typhimurium, and the Vibrio cholerae which causes the massive watery diarrhea that is part of the life-threatening syndrome of cholera. In addition to cholera, two other pathogens of long historical standing in human health are the Yersinia pestis and Mycobacterium tuberculosis strains that cause plague and tuberculosis, respectively.

Many surgical patients are treated prophylactically with antibiotics to reduce the incidence of bacterial infections in the surgical wounds: Staphylococci and Enterococci are the most common bacteria that cause post-surgical infections in US hospitals. Bacteria differ in their degrees of virulence and pathogenicity:[2] Staphylococcus aureus and Streptococcus pneumoniae can be highly virulent and have been described as professional pathogens, while Enterococcus faecalis and Pseudomonas aeruginosa are often characterized as opportunistic pathogens which will grow up when other bacteria have been eradicated and when the host defenses are low, for example, in immunosuppressed states.[3]

All these bacterial pathogens have targets in common (see below), which are acted on by the major classes of antibacterial drugs. Antibiotics with a major niche in human therapeutics fall into two categories of origin, natural products and derivatives thereof, and totally synthetic drugs. The synthetic classes include three examples in current therapeutic use: the two agents in bactrim, sulfamethoxazole, and trimethoprim, which inhibit two enzymatic steps in bacterial biosynthesis of folate coenzymes;[4] the fluoroquinolone antibiotics which inhibit DNA topoisomerases;[5] and the recently introduced oxazolidinone linezolid which targets protein synthesis.[6] All the other major classes of antibiotics, the β-lactams, the glycopeptides, the macrolides, the tetracyclines, and aminoglycosides, are natural products or semisynthetic derivatives of natural products.

---

### From the Contents

1. **Introduction**                                                                                       731
2. **Targets for Antibacterial Drugs**                                                                   731
3. **Resistance Development**                                                                            733
4. **Mechanism of Action**                                                                               733
5. **Self-Protection in Organisms**                                                                      735
6. **Molecular Basis of the VanA, VanB, and VanC Phenotypes**                                             736
7. **Synthetic Strategies**                                                                              738
8. **Biosynthetic Gene Clusters**                                                                        739
9. **Enzymatic Formation of the Nonproteinogenic Amino Acids**                                            742
10. **The Nonribosomal Peptide Synthetase Assembly Line**                                                 750
11. **Post Assembly Line Tailoring of the Heptapeptide**                                                  752
12. **Glycosylation: The Thymidinediphospho Sugar Donors**                                                 753
13. **Glycosylation: Peptide Antibiotic Glycosyl Transferases**                                           758
14. **Approaches to Treatment of Vancomycin-Resistant Bacteria**                                          760
15. **Conclusions**                                                                                      762

---

### 2. Targets for Antibacterial Drugs

Aside from the bactrim combination of two antifolate drugs, almost all the major antibiotics act against three primary processes in the pathogenic bacteria: cell-wall biosynthesis, protein biosynthesis, and DNA replication and repair (Figure 1).[7,8] Antibiotics directed against cell walls include the β-lactam group of penicillins and cephalosporins as well as the glycopeptide antibiotic class, of which two

[*] Dr. B. K. Hubbard, Prof. Dr. C. T. Walsh  
Department of Biological Chemistry and Molecular Pharmacology  
Harvard Medical School  
Boston, MA 02115 (USA)  
Fax: (+1) 617-432-0438  
E-mail: bhubbard@mpi.com  
christopher_walsh@hms.harvard.edu

Angew. Chem. Int. Ed. 2003, 42, 730–765 © 2003 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim 1433-7851/03/4207-0731 $ 20.00+.50/0

# Reviews

**Table 1:** Pathogenic bacteria and the syndromes or disease states in which they are causative agents.

| Gram-positive Bacteria | Infection                     | Gram-negative Bacteria | Infection                          |
|------------------------|-------------------------------|------------------------|-------------------------------------|
| Staphylococcus         | postsurgical infections       | Escherichia coli      | urinary tract infections, diarrhea |
| Streptococcus          | pneumonia                    | Salmonella typhimurium | diarrhea                           |
| Mycobacterium          | tuberculosis                 | Pseudomonas aeruginosa | opportunistic pathogen during cystic fibrosis |
| Enterococcus           | postsurgical infections      | Vibrio cholera        | cholera                            |
| Clostridium difficile  | diarrhea                     | Yersinia pestis       | plague                             |

their action by blocking the enzyme DNA gyrase and a cognate topoisomerase, TopoIV.[12,13]

## 2.1. Targets for β-Lactam and Glycopeptide Antibiotics

Since this review focuses on vancomycin and teicoplanin action to inhibit bacterial peptidoglycan cross-linking in bacterial cell wall biosynthesis, a brief summary of the structure and function of the peptidoglycan layer in bacterial metabolism and physiology is in order. As shown in Figure 2 both gram-negative bacteria (*E. coli*, *S. typhimurium*, *P. aeruginosa*) and gram-positive bacteria (*Staph. aureus*, *S. pneumoniae*, *Enterococcus faecalis*) have a peptidoglycan layer external to the cytoplasmic membrane. The gram-negative bacteria also have an intact outer membrane that acts as a permeability barrier to drugs such as vancomycin. Thus, vancomycin and teicoplanin cannot penetrate to the peptidoglycan (PG) layer and do not work against gram-negative bacterial infections, only against gram-positive bacteria.

The PG layer, in which the glycan strands are covalently cross-linked through peptide strands (Figure 3a), provides the mechanical strength to the PG network that allows the bacterial cell to withstand changes in osmotic pressure that would otherwise burst the cells. Most of the cross-links are introduced by transpeptidase enzymes, which catalyze the attack of the ε-amino group of lysine at position 3 of a peptide chain onto the carbonyl group of D-Ala₄ in an adjacent chain. The new lys-D-ala isopeptide cross bridge is formed as the D-Ala₄-D-Ala₅ bond cleaves, which result in a net transpeptidation. There are multiple forms of transpeptidases and many of them are the target enzymes for β-lactam antibiotics which mimic the N-acyl-D-Ala-D-Ala peptide substrates.[14,15] The transpeptidase starts a catalytic cycle with a penicillin by attacking the β-lactam and creating a more stable acyl enzyme analogue of the normal PG-D-Ala₄-enzyme.[16,17] The slowly hydrolyzing penicilloyl enzyme intermediates effectively inactivate the transpeptidases by tying up their catalytic sites and thereby blocking introduction of the PG isopeptide cross-

![Figure 1. Bacterial targets of antibiotics.](image.png)

Brian K. Hubbard, born in 1972, majored in chemistry at the College of William and Mary and completed his Ph.D. in chemistry with John A. Gerlt at the University of Illinois, Urbana-Champaign. He completed his post-doctoral research with Christopher T. Walsh in 2001. From there he moved to Millennium Pharmaceuticals Inc. in Cambridge MA. His research interests are enzyme mechanisms and novel enzyme classes. A recent focus has been on the role of enzymes as drug targets for the treatment of metabolic diseases.

Christopher T. Walsh, born in 1944, majored in biology at Harvard and completed his PhD in biochemistry in the lab of Fritz Lipmann at Rockefeller Institute of Medical Research. He spent 15 years (1972–1987) in the MIT faculty, and since 1987 has been at the Harvard Medical School. He has served as Chair of the Department of Chemistry at MIT (1982–1987) and of the Department of Biology, Chemistry, and Molecular Pharmacology at Harvard Medical School (1987–1995). His research interests are in enzyme mechanisms, the mechanism of inhibitors, and the biosynthesis of nonribosomal peptide antibiotics.

Biosynthesis of Antibiotics

Table 2: Antibiotic targets and resistance mechanisms.

| Antibiotic           | Target                                                                 | Resistance mechanism                                      |
|----------------------|------------------------------------------------------------------------|-----------------------------------------------------------|
| cell wall            |                                                                      |                                                           |
| β-lactams            | transpeptidase/transglycosylases (PBPs)                               | β-lactamases, PBP mutants                                |
| vancomycin           | d-Ala-d-Ala termini of peptidoglycan and lipid II                      | reprogramming of d-Ala-d-Ala to d-Ala-d-Lac or d-Ala-d-Ser |
| teicoplanin          | d-Ala-d-Ala termini of peptidoglycan and lipid II                      | reprogramming of d-Ala-d-Ala to d-Ala-d-Lac or d-Ala-d-Ser |
| protein synthesis    |                                                                      |                                                           |
| erythromycins        | peptidyl transferase/ribosome                                        | rRNA methylation/efflux                                  |
| tetracyclines        | peptidyl transferase                                                  | drug efflux                                               |
| aminoglycosides      | peptidyl transferase                                                  | drug modification                                         |
| oxazolidinones       | peptidyl transferase                                                  | unknown                                                   |
| DNA replication/repair |                                                                      |                                                           |
| fluoroquinolones     | DNA gyrase                                                            | gyrase mutations                                          |

---

Figure 2. Cell walls of gram-negative and gram-positive bacteria.

3. Resistance Development

The introduction of a new antibiotic molecule into widespread human use carries with it an inevitable progression to select for resistant bacteria and to have diminished utility, which ensures the need for new cycles of antibiotics discovery, design, and synthesis. The development of resistance accrues from two facets of bacterial populations: the large number of bacteria in an infection and their mutation frequency. There may be \(10^9\)–\(10^{10}\) bacteria in a fulminant infection in an animal or a person. A typical rate of spontaneous mutation that is observed is about one mutant cell in \(10^7\). At these frequencies there would be about a thousand mutants in a bacterial population of \(10^{10}\). If each of these mutants had occurred randomly they might be distributed one per gene in each of a thousand genes in the bacterial population. When the bacteria are now exposed to an antibiotic, most will die off. However, if one or more mutants in a particular gene alter the gene product to confer resistance, they will give a selective advantage for the mutant to survive and then grow up in the space vacated by their dying (sensitive) neighbors and dominate the bacterial population. Bacterial resistance to antibiotics is a matter of when, not if, and is almost independent of the structure and type of antibiotic. The resistance could be conferred by the mutation of a single gene or multiple mutations may have to accumulate before significant growth advantage and resistance accrues. In Section 5 it will be shown that five genes had to be collected to reprogram the PG biosynthetic pathway before significant resistance to vancomycin arose in Enterococcal strains in clinical isolates.

4. Mechanism of Action of Glycopeptide Antibiotics

The focus of this review is on the glycopeptide class of antibiotics, natural products produced by the Actinoplanes order of bacteria. Following the discovery of vancomycin in a soil sample collected in Borneo in the 1950s, dozens of related glycopeptide antibiotics have been isolated from soil bacteria both from *Amycolaptosis* and *Streptomyces* strains. The two main subclasses are exemplified by the antibiotics vancomy-

Reviews

B. K. Hubbard and C. T. Walsh

a)

![Diagram](https://i.imgur.com/1234567.png)

b)

![Chemical Reaction Diagram](https://i.imgur.com/8901234.png)

Figure 3. a) Peptidoglycan biosynthesis. b) Mode of inhibition by penicillins.

cin and teicoplanin (Scheme 1), each of which has a hepta-peptide moiety that has undergone extensive cross-linking of the aryl side chains of the amino acid residues to impart a rigid dome-shaped architecture to the peptide scaffold. Vancomycin and teicoplanin have the configuration D-D-L-D-D-L-L in the heptapeptide scaffold produced by the action of non-ribosomal peptide synthetase, and this configuration is presumed to be crucial for conformations that permit side-chain cyclization. Vancomycin has nonproteinogenic amino acids at residues 4, 5 (4-hydroxyphenylglycine (Hpg)), and 7 (3,5-dihydroxyphenylglycine (DHpg)) while teicoplanin has an additional Hpg group at residue 1 and a DHpg group at residue 3. Vancomycin has three oxidative cross-links, in the aryl side chains 2–4, 4–6, and 5–7, while teicoplanin, with the additional aryl side chains at residues 1 and 3, has an additional cross-link such that all the side chains are connected.

These are glycopeptide antibiotics. A disaccharide is attached to the phenolic oxygen atom of 4-OH-PheGly₄ in vancomycin, while there are sugars on residues 4, 6, and 7 in teicoplanin. The disaccharide in vancomycin is a vancosaminyl-1,2-glucosyl moiety where the L-vancosamine is a 2,3,6-trideoxy-3-amino-3-methyl-L-hexose, made just for this antibiotic by the producer cell. Sugars related to vancosamine can be found in vancomycin analogues, including 4-oxo-L-vancosamine in balhimycin and 4-epi-L-vancosamine in chloroeremomycin (Scheme 2). A second L-epivancosamine is found on the β-hydroxy substituent of β-OH-Tyr₆ in chloroeremomycin. For a list of additional sugars found in vancomycin family members, see the recent review by Nicolaou et al. and Table 8 (Section 12).[25] In teicoplanin there is an N-acyl-D-glucosamine at residues 4 and 6, with a long acyl chain on the D-glucose attached to residue 4 and a more common GlcNAc sugar at residue 6, as well as a mannose group attached to residue 7. There are teicoplanin family members without any sugar groups such as A47934, from Streptomyces toyocaensis with a sulfate group on residue 3 (Scheme 2).

Vancomycin, teicoplanin, and other members of these glycopeptide antibiotics with the rigidified cross-linked heptapeptide scaffolds recognize the N-acyl-D-Ala₄-D-Ala₅ terminal D,D-dipeptide of un-cross-linked PG strands and in lipid II molecules to form high affinity complexes through five hydrogen bonds (Figure 4a) from the undersurface of the glycopeptide to the amide and carbonyl groups of the PG terminus. A space-filling model shows the complementary surfaces lead to a tight fit of the antibiotic and the N₂,₆-bis(acyl)-L-Lys-L-Ala-D-Ala ligand. Such a model is shown in Figure 4c where the methyl side chain of D-Ala₅ is packed tightly against the antibiotic cavity. The D-Ala₄-D-Ala₅ amide linkage is sequestered in the complex with the glycopeptide and occluded from recognition by the cross-linking transpeptidases.

Biosynthesis of Antibiotics

a)

![Chemical Structure](chemical_structure_a.png)

N-Acyl-D-Ala-D-Ala  
N-Acyl-D-Ala-D-Ser  
N-Acyl-D-Ala-D-Lac  

b)

![Chemical Structure](chemical_structure_b.png)

Cell  
membrane  

Figure 4. a) Hydrogen bonding in the interaction of vancomycin with N-Acyl-D-Ala, N-Acyl-D-Ala-D-Lac, and N-Acyl-D-Ala-D-Ser. b) Hydrogen bonding in the interaction of vancomycin with the cell-wall intermediate. c) Space-filling model of vancomycin bound to N₂,₆-bis(acyl)-L-Lys-D-Ala-D-Ala. Vancomycin is shown in blue and the peptide shown in orange and red.

### 5. Self-Protection in Glycopeptide-Producer Organisms

Most bacteria that produce antibiotics have strategies for self-protection and immunity from the action of these chemical weapons. In the nonribosomal peptide antibiotic and polyketide antibiotics where the biosynthetic genes are clustered (see Section 7), there are also immunity-conferring genes. This co-localization of antibiotic production and antibiotic resistance genes enables coordinate regulation. When the antibiotic biosynthetic genes are turned on, so are the resistance genes, and it has been argued that antibiotic production and resistance genes co-evolved.[26] Typical resistance-gene products include transmembrane proteins that serve as efflux pumps, to pump out the nascent antibiotic molecules and keep the intracellular concentration in the

produced by the D-Ala-D-Ala ligase enzyme. The net result is that the concentration of the normal cell-wall precursor D-Ala-D-Ala falls while the new metabolite D-Ala-D-lactate rises. The MurF enzyme, which normally uses D-Ala-D-Ala as a substrate, now cannot find any and substitutes D-Ala-D-lactate in its condensation with UDP-muramyl tripeptide to make the UDP-muramyl-D-Ala-D-isoGlu-L-Lys-D-Ala-D-lactate, the depsipeptide analogue of the normal UDP-muramyl pentapeptide.[31] This analogue is glycosylated, transferred to the C55-lipid-PP carrier, translocated to the outer face of the cytoplasmic membrane, and used for PG elongation (Scheme 3). The transpeptidases use the tetrapeptidyl-D-Lac chain for cross-linking, thus disconnecting the D-Ala₄-D-Lac₅ ester bond as the isopeptide cross-link is produced.

The net effect of expression of the three resistance enzymes is to reprogram the cytoplasmic steps of PG assembly to produce D-Ala-D-Lac and carry it through in place of the D-Ala-D-Ala. As noted in the next section the affinity of vancomycin for the PG-D-Ala-D-Lac termini is 1000 fold lower than for the D-Ala-D-Ala termini. The producing bacteria have reprogrammed the biosynthetic pathway of the cell wall to become immune to vancomycin and teicoplanin.

## 6. Molecular Basis of the VanA, VanB, and VanC Phenotypes in Vancomycin-Resistant Enterococci

As the prevalence of antibiotic resistance spread during the 1980s, especially with the outbreak of methicillin-resistant Staphylococcus aureus (MRSA), vancomycin became a more important drug and came into wider use for the treatment of life-threatening gram-positive bacterial infections. It is now front-line therapy for endocarditis, for many bacteremias that occur in cancer patients undergoing cycles of chemotherapy, and became the drug of last resort for the treatment of MRSA infections in hospital wards.[32] With wider use of vancomycin in hospital wards, vancomycin-resistant gram-positive bacterial pathogens emerged, most notably vancomycin-resistant Enterococci (VRE). VRE have been categorized as opportunistic pathogens that are selected for when other bacteria die off.[33] The incidence of VRE rose alarmingly from 0.4% in hospital clinical isolates in 1987 to 16% just six years later,[34] and carried a high mortality rate. About 95% of VRE are Enterococcus faecalis strains and about 5% are Enterococcus faecium strains. Two main clinical phenotypes have been described as VanA and VanB (Table 3), which are distinguished largely by differential sensitivity to teicoplanin (resistant in VanA, sensitive in VanB).[3,35] A third minor clinical phenotype is categorized as VanC, and more recently variants of the phenotypes A and B have been detected.[36,37]

The VanA and VanB phenotypes of VRE have the same underlying molecular logic and it is the same as that in the glycopeptide antibiotic producer organisms, namely, a reprogramming of the PG termini from D-Ala-D-Ala to D-Ala-D-lactate. A five-gene cassette is required for this inducible enzymatic reprogramming of the PG: *vanR*, *vanS*, *vanH*, *vanA*, *vanX*, where *vanH* encodes the D-specific pyruvate reductase, *vanA* the D-Ala-D-lactate ligase and *vanX* the D-

Biosynthesis of Antibiotics

**Scheme 2. Structures of chloroeremomycin, balhimycin, A47934, complestatin, and orienticin C.**

- Chloroeremomycin
- Balhimycin
- A47934
- Complestatin
- Orienticin C

**Scheme 3. Biosynthesis and incorporation of the D-Ala-D-Lac terminus of the peptidoglycan.**

Ala-D-Ala dipeptidase. This sequence runs along the pathway shown in Figure 5 to produce D-Ala-D-lactate and destroy the normal D-Ala-D-Ala dipeptide selectively.[31,38] The binding of vancomycin to N-acyl-D-Ala-D-Lac has been quantitated to be 1000 fold lower in affinity than N-Acyl-D-Ala-D-Ala, and accounts fully for the three orders of magnitude higher minimal inhibitory concentrations (MICs; Table 3) seen in VRE clinical isolates of the VanA and VanB phenotypes.[39] Figure 4a shows a schematic representation of complexes of vancomycin with N-Acyl-D-Ala₄-D-Ala₅ and N-acyl-D-Ala₄-D-Lac₅ in which the one fewer hydrogen bond in the former complex is indicated.[31,35] It is likely there are also ground-state electrostatic repulsions between the vancomycin carbonyl oxygen atom and the ester oxygen atom in the D-Ala-D-Lac bond that contribute to the 1000 fold drop in affinity.

The vancomycin-resistant phenotype in VanA and VanB VRE can be induced by antibiotics, and is mediated by the other two proteins, VanS and VanR, encoded in the five-gene resistance cassette. The VanS and VanR pair form a two-component regulatory system, in which VanS is transmembrane sensor protein kinase and VanR a DNA-binding response regulator (Figure 5).[40] The presence of both vancomycin and teicoplanin at the outer surface of the enterococcal cell in the VanA phenotype, but only vancomycin in the VRE VanB phenotype, induces the VanS transmembrane protein to autophosphorylate itself in its cytoplasmic histidine kinase domain, perhaps by induced dimerization of VanS by an external ligand to bring the cytoplasmic domains together for transphosphorylation.[41] The phosphohistidyl VanS kinase domain can then transfer the phosphate

group to the VanR protein and phosphorylate a conserved aspartyl-β-COOH side chain in the N-terminal “receiver” domain of VanR. The phospho-VanR has a lower binding affinity to its target DNA sequences than VanR, which causes the P-VanR to fall off the promoter regions of the DNA in front of the *vanHAX* genes, thus relieving transcriptional repression.[42] The expression of VanHAX enzymes now starts the peptidoglycan reprogramming that leads to replacement of the D-Ala-D-Ala PG termini by D-Ala-D-Lac.

The continuing sensitivity of VanB-type of VRE to teicoplanin is entirely the result of its failure to be recognized by the VanS<sub>B</sub> sensor kinase and failure to turn on the reprogramming trio of enzymes (VanH<sub>B</sub>, VanA<sub>B</sub>, VanX<sub>B</sub>).[43,44] This effect could be demonstrated by mutants that gained resistance to teicoplanin, all of which mapped to amino acid changes in VanS<sub>B</sub> that now transmitted the signal to VanR<sub>B</sub>. These findings are consistent with the glycopeptides, vancomycin (VanS<sub>A</sub>, VanS<sub>B</sub>) and teicoplanin (VanS<sub>A</sub>), being the direct ligands for the external domains of the sensor kinases.

As a generalization of the phenomena that reprogramming the peptidoglycan D-Ala-D-Ala termini is the molecular determinant of vancomycin resistance, *Enterococcus gallinarum* which cause the VanC phenotype produce a D-Ala-D-Ser ligase constitutively.[45,46] Analysis of PG termini show the D-Ala-D-Ser termini with the larger CH<sub>2</sub>OH group of D-Ser<sub>5</sub> cause an order of magnitude drop in vancomycin binding affinity and an order of magnitude rise in MIC values (Table 3) relative to that found for termini containing the CH<sub>3</sub> group of D-Ala<sub>5</sub> (Figure 4a). Remarkably, one can even grow *E. faecalis* strains that require vancomycin to survive.[47] These have inactivating mutations in the chromosomal D-Ala-D-Ala ligase and only if the vancomycin-inducible D-Ala-D-lactate ligase (VanA) is expressed can this bacterium make peptidoglycan and survive.

---

### 7. Biosynthetic and Synthetic Chemical Strategies

The synthesis and assembly of vancomycin is a formidable challenge both to producer microorganisms and to synthetic chemists. While the main focus of this review is analysis of nature's route to the glycopeptide antibiotics, and the synthetic approaches have been reviewed previously by Nicolaou et al.,[25] it is instructive to compare briefly the successful strategies planned and executed by three synthetic chemistry groups (Evans et al.,[48–51] Nicolaou et al.,[52–56] and Boger et al.) to nature's way.[57] The major difficulty is the introduction of the three cross-links (two aryl ether and one C–C link) that stitch together the five aryl side chains in the vancomycin heptapeptide and all seven of the aryl side chains (three aryl ether and one C–C link) in the teicoplanin subclass.

The first synthesis of a vancomycin-type aglycone was that of the orienticin aglycone (Scheme 2),[50] which differed from the vancomycin aglycone by the absence of chloro substituents on β-hydroxy-Tyr2 and -Tyr6. One starting distinction between the biosynthetic assembly line and the chemical synthesis routes is that the *Amycolaptosis orientalis* bacteria use unprotected amino acid monomers while the chemists build in protective groups (Scheme 4). Nature needs five amino acids, two of which (L-Hpg at residues 4 and 5 and D-Hpg at residue 7) are nonproteinogenic and are biosynthesized just for this purpose. The protected monomers used by the research groups of Evans, Nicolaou, and Boger are listed in Scheme 4 as AA-1-A to AA-7-A (Evans), AA-2-B to AA-7-B (Nicolaou), and AA-3-C and AA-7-C (Boger). The protections at AA-1-A, -3-A, and -7-A are unexceptional, while the other four protection schemes were devised to allow the 2-4, 4-6, and 5-7 cross-links to be introduced. Analogously, the protection manifolds of AA-4-B were devised to enable the double cross-linking (to residue 2 and to residue 6) during the course of the synthesis.

---

### Table 3: Types of resistance mechanisms and their sensitivity to vancomycin and teicoplanin.

| Characteristic | VanA | VanB | VanC | VanD | VanE* |
|----------------|------|------|------|------|-------|
| genetic characteristics | acquired (c.g., Tn1546) | acquired (gene Tn1547) | intrinsic (chromosomally encoded characteristic of the species) | acquired | acquired |
| terminus of peptidoglycan precursor | D-Ala-D-Lac | D-Ala-D-Lac | D-Ala-D-Ser | D-Ala-D-Ser | D-Ala-D-Ser |
| minimal inhibitory concentration (µg mL<sup>-1</sup>) |  |  |  |  |  |
| vancomycin | 64 to >1000 | 4 to >1000 | 2 to 32 | 16 to 64 | 16 |
| teicoplanin | 16 to 512 | 0.5 to >32 | 0.5 to 1 | 2 to 4 | 0.5 |
| ligase gene | vanA | vanB | vanC-1 and vanC-2/vanC-3 | vanD | vanE |
| bacteria found to have these genes for resistance in nature | *Enterococcus faecium*, *E. faecalis*, *E. durans*, *E. mundtii*, *E. avium*, *E. gallinarum*, *E. casseliflavus*, *Bacillus circulans*, *Streptococcus gallolyticus*, *corynebacteria*, *arcanobacteria*, *lactococci*, *oerskovia* | *E. faecalis*, *E. faecium*, *Streptococcus bovis* | *E. gallinarum* and *E. casseliflavus/E. favesccus* | *E. faecium* | *E. faecalis* |
| bacteria to which vancomycin resistance has been transferred from enterococci in the laboratory setting | *S. sanguis*, *S. pyogenes*, *listeria*, and *Staphylococcus aureus* | *E. faecalis*, *E. faecium*, including transfer with ampicillin |  |  |  |

Angew. Chem. Int. Ed. 2003, 42, 730–765

Biosynthesis of Antibiotics

**Protected Evans Monomers**

![Chemical Structures](https://i.imgur.com/1234567.png)

**Protected Nicolaou Monomers**

![Chemical Structures](https://i.imgur.com/8901234.png)

**Protected Boger Monomers**

![Chemical Structures](https://i.imgur.com/5678901.png)

**Free amino acid substrates for biosynthetic assembly line**

![Chemical Structures](https://i.imgur.com/2345678.png)

**Scheme 4.** Monomers used in the chemical and biosynthetic synthesis of vancomycin.

---

### 7.1. Timing of Monomer Assembly and Cross-Linking

A retrosynthetic analysis of the three chemical routes to glycopeptide assembly show that the approach of Nicolaou et al. (Scheme 5) differed somewhat from that of Evans et al. and also from that of Boger et al. (Scheme 6). To simplify the overall logic, the protecting groups, which are crucial for the actual implementation of the syntheses, are omitted in these schemes. All three syntheses built the 5–7 C–C cross-link first, then the 4–6, and finally the 2–4 aryl ether linkages to complete the aglycone scaffold. Nicolaou et al. built the 5–7 biaryl cross-link as the first connection then created the 5–6 and 4–5 peptide bonds. The 4–6 aryl ether cross-link was then introduced followed by construction of the 6–7 amide bond to give the macrobicyclic, doubly cross-linked tetrapeptide containing residues 4–7. The tripeptide 1–3 was assembled and coupled to make the 3–4 peptide bond, thereby generating the heptapeptide that lacked only the 2–4 aryl ether cross-link. Its introduction, followed by side-chain deprotections (not shown), completed the synthetic assembly logic to make the three rings of the fully cross-linked heptapeptide.

The routes used by the research groups of Evans and then Boger first assembled the 5–7 tripeptide and then introduced the 5–7 cross-link. Addition of residue 4 through formation of a 4–5 amide bond generated the mono-cross-linked tetrapeptide that was then subjected to 4–6 aryl ether cross-linking. The 1–3 tripeptide unit was then added to produce the equivalent heptapeptide scaffold in which the 2–4 aryl ether cross-link was introduced. All three routes are highly convergent with 28 steps constituting the longest linear synthetic sequence of steps. All three routes put the sugars

on after completion of the aglycone scaffold. These are monumental feats of synthetic chemistry. However, the routes are not practical for facile production, averaging 95 steps from readily available starting materials.

A comparable retrosynthetic dissection of nature’s logic for vancomycin assembly is shown in Scheme 7. A notable difference, besides the lack of any of the functional group protections noted above, is the assembly of the full linear heptapeptide before any aryl side chain cross-links are introduced. This creates complex problems of conformation control for regiospecific and stereospecific closure of the three rings for the cross-linking enzymes. It is known from gene-knockout studies that the 4–6 aryl ether cross-link is formed first, but the relative timing of the 2–4 and 5–7 cross-links is not yet deciphered.[59] We will discuss the enzymatic machinery for nonproteinogenic amino acid monomers in Section 9 and enzymatic cross-linking in Section 11. The natural route from the start to the fully glycosylated vancomycin involves about 35 steps.

---

### 8. Biosynthetic Gene Clusters for the Peptide Scaffold and the Tailoring Enzymes

The availability of the DNA sequence for antibiotic biosynthetic gene clusters illuminates the molecular logic of assembly and allows predictions of the functions of the different open reading frames (orfs) by homology to proteins of known functions and also by location of one orf relative to others within a cluster. Sequence data for the vancomycin group antibiotics has begun to appear. To date, the full cluster

Angew. Chem. Int. Ed. **2003**, *42*, 730–765

for chloroeremomycin, a vancomycin class member, and for complestatin (Scheme 2), a nonglycosylated molecule, have been reported.[60,61] A partial sequence of the balhimycin cluster has been published, mapping data for the teicoplanin cluster has appeared,[62,30] and the sequence of the A47934 cluster has been reported.[145]

### 8.1. Chloroeremomycin and Balhimycin

Some 40 contiguous genes from the chloroeremomycin-producing (Cep) *Amycolatopsis orientalis* were sequenced and partially annotated in the 1998 report of van Waginngen et al.[60] Currently it appears that 30 of those Cep orfs can be clearly assigned to genes in the glycopeptide biosynthetic pathway and they are listed in Table 4 as orfs 1–30 in their order of appearance (Figure 6). The protein names are also tabulated, together with predicted or known functions and cosubstrates and cofactors for enzyme activity. Analysis of the start and stop codons of those 30 genes indicates there are at least 19 mRNA transcripts, that is, several of the genes are transcribed together and there are overlaps of some stop and start codons, which is typical of translational coupling (as noted in Table 4). Of the 30 gene products, at least 23 have now been purified to substantial purity and their catalytic function established, as will be discussed in the next sections.

Five distinct amino acids are incorporated into the heptapeptide skeleton which is identical for chloroeremomycin, vancomycin, and balhimycin (Scheme 4). Two of these are not found in proteins (Hpg and Dpg), while a third, 3-chloro-β-hydroxytyrosine, is a nonproteinogenic variant of tyrosine. It is anticipated that enzymes devoted to the de novo biosynthesis of Hpg and 3,5-dihydroxyphenylglycine would be found in the cluster. The four orfs 1, 17, 21, and 22 constitute the four-enzyme pathway to Hpg, while the four orfs 27–30 lead to Dpg. The conversion of the common amino acid L-tyrosine to the 3-chloro-β-hydroxy-Tyr found at residues 2 and 6 of the heptapeptide scaffold is proposed to involve the four orfs 10, 18, 19, and 20. The detailed function of these twelve orfs is described in the next sections.

Three Orfs, CepA, CepB, and CepC constitute the nonribosomal peptide synthetase assembly line to produce the heptapeptide. An additional three Orfs, 7, 8, and 9, are heme proteins involved in oxidative cross-linking of the heptapeptide. Eight of the remaining twelve Orfs are involved in producing and attaching the sugar moieties of the glycopeptide. Orfs 14, 23, 24, 25, and 26 produce dTDP-L-epivancosamine from dTDP-D-glucose and orfs 12–14 are three glycosyltransferases that install the one D-glucose and two L-epivancosamine sugars onto the heptapeptide scaffold. Of the remaining four Orfs, Orf 16 is the N-methyltransferase that N-methylates the Leu₁ residue.[63] Orf 2 is predicted to be a transporter that is presumed to be the export pump for secretion of chloroeremomycin. Orf 15 has homologies to epimerases, while Orf 6 is the sole gene product with no ascribed function to date.

The partial sequence of the balhimycin cluster has so far revealed seven complete orfs (Table 5) with an almost identical organization to the orfs in the Cep cluster (Table 4).[62] The balhimycin orfs are homologues to Cep orf 6, three hemeproteins are analogous to Cep 7–9, a putative halogenase, three glycosyl transferases, and a partial sequence of a Cep 14 homologue.

#### Figure 6. Genes in the chloroeremomycin biosynthetic cluster.
![Diagram](https://i.imgur.com/yourimageurl.png)

### 8.2. Complestatin

The natural product complestatin (Scheme 2) from *Streptomyces lavendulae* is reported to block protein-protein interactions both in the complement cascade and in gp120-CD4 binding in the HIV life cycle.[64–67] The molecule clearly has similarities to the peptide scaffold of vancomycin, but is not glycosylated. The complestatin biosynthetic gene cluster has now been sequenced (Table 6 and Figure 7).[61] Complestatin is an acyl hexapeptide with all seven residues bearing aromatic side chains. The cross-linking of the electron-rich aryl side chains of residues 2-4-6 mirrors the 2-4-6 pattern found in the vancomycin and teicoplanin classes. Four of the six residues in complestatin are nonproteinogenic and derive from Hpg. The first residue, a p-hydroxy-3,5-dichlorobenzoyl formate, could arise by oxidation of p-hydroxy-3,5-dichlorophenylglycine, which is found at residues 3 and 5. Complestatin also contains the amino acid tryptophan, which is unique compared to the rest of the vancomycin family members.

Of the 16 genes in the complestatin cluster the four orfs encoding Hpg synthesis are found (orfs 13–16) as homologues to the corresponding cep genes. There are four protein subunits for nonribosomal protein synthesis, orfs 3–6, which like the CepA–C subunits have seven recognizable modules for activating the seven monomers. There are two hemeproteins, most likely the cross-linking oxidases orfs 10, 11, and a ferredoxin, orf 12, for electron transport into those hemeproteins. Of the remaining four genes, two appear to be involved in transport, one is a transcriptional regulator, and again one has an as yet unassigned function.

Biosynthesis of Antibiotics

![Chemical Structure 1](chemical_structure_1.png)

$\xrightarrow{}$

![Chemical Structure 2](chemical_structure_2.png)

$\downarrow$

![Chemical Structure 3](chemical_structure_3.png)
heptapeptide (4-6, 5-7 x-link)

$\rightleftharpoons$

![Chemical Structure 4](chemical_structure_4.png)
2-4 x-link

$\downarrow$

![Chemical Structure 5](chemical_structure_5.png)

$\xrightarrow{}$

![Chemical Structure 6](chemical_structure_6.png)

$\downarrow$

![Chemical Structure 7](chemical_structure_7.png)
AA-1

$\downarrow$

![Chemical Structure 8](chemical_structure_8.png)
4-6 x-link

$\downarrow$

![Chemical Structure 9](chemical_structure_9.png)
4-7 tetrapeptide
(4-6, 5-7 x-link)

$\downarrow$

![Chemical Structure 10](chemical_structure_10.png)
AA-4

$\downarrow$

![Chemical Structure 11](chemical_structure_11.png)

$\downarrow$

![Chemical Structure 12](chemical_structure_12.png)
AA-6

$\xrightarrow{5-7 \text{ x-link}}$

![Chemical Structure 13](chemical_structure_13.png)
AA-7

$\downarrow$

![Chemical Structure 14](chemical_structure_14.png)
AA-5

---

**Scheme 5.** Retrosynthetic analysis of the chemical synthesis of vancomycin used by Nicolaou et al.

Angew. Chem. Int. Ed. **2003**, *42*, 730–765

Reviews

B. K. Hubbard and C. T. Walsh

![Chemical Structures](chemical_structures.png)

Scheme 6. Retrosynthetic analysis of the chemical synthesis of vancomycin used by Boger et al. and Evans et al.

While the sequences of teicoplanin class glycopeptide antibiotics have begun to emerge, it is highly likely that the above two examples reveal the common molecular logic for synthesis of dedicated amino acid and amino/deoxy sugar monomers;[30] this is, assembly of the nonribosomal peptide scaffold and the halogenation, oxidation, and glycosylation tailoring enzyme strategies for this large class of natural product antibiotics.

### 9. Enzymatic Formation of the Nonproteinogenic Amino Acid Monomers

#### 9.1. 4-Hydroxyphenylglycine

Every member of the vancomycin and teicoplanin family of glycopeptide antibiotics contains the nonproteinogenic amino acid 4-hydroxyphenylglycine. Each oxidative cross-link in vancomycin and teicoplanin involves one Hpg group, which makes the presence of this amino acid essential to the overall structure of the antibiotic. An understanding of the biosyn-

Biosynthesis of Antibiotics

![Chemical Structure 1](chemical_structure_1.png)

$\longrightarrow$

![Chemical Structure 2](chemical_structure_2.png)

$\longleftrightarrow$

![Chemical Structure 3](chemical_structure_3.png)

$\downarrow$

![Chemical Structure 4](chemical_structure_4.png)

$\longrightarrow$

![Chemical Structures AA-1 to AA-7](chemical_structures_aa1_to_aa7.png)

**Scheme 7.** Retrosynthetic analysis of the biosynthesis of vancomycin.

thesis of this unusual amino acid is essential to understanding how the glycopeptides are assembled.

Early structural studies in vancomycin family members showed that all of the Hpg groups in the mature antibiotics were in the D configuration.[68] Feeding studies also showed that radiolabeled racemic Hpg may serve as monomers for incorporation into the final glycopeptide.[69–71] Feeding experiments with ${}^{13}$C-labeled tyrosine demonstrated that the aromatic ring of Hpg was derived from the aromatic ring of tyrosine and the α-carbon atom of tyrosine forms the carboxylic acid carbon atom of Hpg (Scheme 8).[69,71–73] From these experiments, the linear conversion of tyrosine into 4-hydroxyphenylglycine through 4-hydroxybenzoyl formate

Angew. Chem. Int. Ed. **2003**, *42*, 730–765

Table 4: Genes in the chloroeremomycin biosynthetic cluster.

| Orf no. | Accession number | Metabolic pathway | Protein name | Predicted function | Cofactors or cosubstrates | Relative start codon | Relative stop codon | No. base pairs | No. amino acids | Transcript | C[a] | P[b] | A[c] |
|---------|------------------|-------------------|--------------|---------------------|---------------------------|----------------------|-------------------|-----------------|----------------|------------|-------|-------|-------|
| 1       | CAA11792.1       | Hpg               | PD           | prephenate dehydrogenase | 1                         | 1104                 | 1104              | 367            | 1          | +     | -     | -     |
| 2       | CAA11793.1       | transport protein | CepM         | ABC transporter      | ATP                       | 1308                 | 3260              | 650            | 2          | -     | -     | -     |
| 3       | CAA11794.1       | heptapeptide assembly | CepA        | NRPS                | CoA, ATP                  | 3257                 | 12733             | 9477           | 3158       | 2     | +     | -     |
| 4       | CAA11795.1       | heptapeptide assembly | CepB        | NRPS                | CoA, ATP                  | 12908                | 25141             | 12234          | 4077       | 3     | +     | -     |
| 5       | CAA11796.1       | heptapeptide assembly | CepC        | NRPS                | CoA, ATP                  | 25161                | 30743             | 5583           | 1860       | 4     | +     | -     |
| 6       | CAA11799.1       | unknown function   | CepD         |                     |                           | 30740                | 30949             | 210            | 69         | 4     | -     | -     |
| 7       | CAA11797.1       | oxidative cross-linking | CepE       | oxidation           | NADPH                     | 31044                | 32246             | 1203           | 400        | 5     | +     | -     |
| 8       | CAA11798.1       | oxidative cross-linking | CepF       | oxidation           | NADPH                     | 32266                | 33486             | 1221           | 406        | 6     | +     | -     |
| 9       | CAA11800.1       | oxidative cross-linking | CepG       | oxidation           | NADPH                     | 33525                | 34892             | 1368           | 455        | 7     | +     | -     |
| 10      | CAA11780.1       | halogenation      | CepH         | halogenation        | NADPH                     | 34954                | 36429             | 1476           | 491        | 8     | +     | -     |
| 11      | CAA11774.1       | glycosylation     | GtfA         | attach sugar        |                           | 36426                | 37664             | 1239           | 412        | 8     | +     | +     |
| 12      | CAA11775.1       | glycosylation     | GtfB         | attach sugar        |                           | 37699                | 38922             | 1224           | 407        | 9     | +     | +     |
| 13      | CAA11776.1       | glycosylation     | GtfC         | attach sugar        |                           | 39002                | 40300             | 1299           | 432        | 10    | +     | +     |
| 14      | CAA11777.1       | TDP-epivancosamine | EvaC        | methylation         | SAM                       | 40373                | 41599             | 1227           | 408        | 11    | +     | +     |
| 15      | CAA11778.1       | epimerization     | CepI         | epimerase           | NADP                      | 41601                | 42434             | 834            | 277        | 12    | +     | -     |
| 16      | CAA11779.1       | N-methylation     | MtfA         | methylation         | SAM                       | 42431                | 43309             | 879            | 292        | 12    | +     | +     |
| 17      | CAA11790.1       | Hpg               | HpgT         | aminotransferase    | PLP                       | 44594                | 43278             | 1317           | 438        | 13    | +     | +     |
| 18      | CAA11784.1       | β-hydroxytyrosine | CepJ         | thioesterase        |                           | 44659                | 45642             | 984            | 327        | 14    | +     | +     |
| 19      | CAA11773.1       | β-hydroxytyrosine | CepK         | NRPS                | CoA, ATP                  | 45639                | 47381             | 1743           | 580        | 14    | +     | -     |
| 20      | CAA11772.1       | β-hydroxytyrosine | CepL         | hydroxylation       | NADPH                     | 47397                | 48590             | 1194           | 397        | 15    | +     | -     |
| 21      | CAA11761.1       | Hpg               | HmaS         | dioxygenation       | Fe                        | 48766                | 49908             | 1143           | 380        | 16    | +     | +     |
| 22      | CAA11762.1       | Hpg               | Hmo          | oxidase             | FAD                       | 49905                | 50978             | 1074           | 357        | 16    | +     | +     |
| 23      | CAA11763.1       | TDP-epivancosamine | EvaA        | oxidase             | FMN                       | 50978                | 52393             | 1416           | 471        | 17    | +     | +     |
| 24      | CAA11764.1       | TDP-epivancosamine | EvaE        | reductase           | NADPH                     | 52395                | 53372             | 978            | 325        | 18    | +     | +     |
| 25      | CAA11782.1       | TDP-epivancosamine | EvaB        | aminotransferase    | PLP                       | 53369                | 54478             | 1110           | 369        | 18    | +     | +     |
| 26      | CAA11781.1       | TDP-epivancosamine | EvaD        | epimerase           |                           | 54475                | 55119             | 645            | 214        | 18    | +     | +     |
| 27      | CAA11765.1       | L-Dpg              | DpgA         | PKS Type III        |                           | 55312                | 56430             | 1119           | 372        | 19    | +     | -     |
| 28      | CAA11766.1       | L-Dpg              | DpgB         | isomerase           |                           | 56427                | 57230             | 804            | 267        | 19    | +     | -     |
| 29      | CAA1178.5        | L-Dpg              | DpgC         | thioesterase        |                           | 57227                | 58525             | 1299           | 432        | 19    | +     | +     |
| 30      | CAA11767.1       | L-Dpg              | DpgD         | dehydration         |                           | 58522                | 59325             | 804            | 267        | 19    | +     | +     |

[a] C = clone. [b] P = purified. [c] A = Analyzed. Hpg = 4-hydroxyphenylglycine; ATP = adenosinetriphosphate; NRPS = nonribosomal peptide synthetase; CoA = coenzyme A; NADP = nicotinamide adeninediphosphate (oxidized); NADPH = nicotinamide adeninediphosphate (reduced); SAM = S-adenosyl-L-methionine; PLP = pyridoxal 5'-phosphate; FAD = flavin-adenine-dinucleotide; FMN = flavin mononucleotide; L-Dpg = L-3,5-dihydroxyphenylglycine; PKS = polyketide synthetase.

Biosynthesis of Antibiotics

Table 5: Genes in the partial balhimycin biosynthetic cluster.

| Orf no. | Cep homologue (orf no.) | Metabolic pathway | Protein name | Predicted function | Cofactors or cosubstrates | Relative start codon | Relative stop codon | No. base pairs | No. amino acids | Transcript | C[a] | P[b] | A[c] |
|---------|------------------------|--------------------|--------------|--------------------|--------------------------|---------------------|-------------------|----------------|----------------|------------|-------|-------|-------|
| 1       | 2                      | regulation         | ComG         |                    |                          | 1                   | 1047              | 1047           | 348          | 1      | –     | –     | –     |
| 2       | 2                      | transport          | ComI         | ABC transporter    | ATP                      | 1472                | 3598              | 2127           | 708          | 2      | –     | –     | –     |
| 3       | 3                      | acylhexapeptide assembly | ComA         | NRPS               | CoA, ATP                 | 3645                | 9995              | 6351           | 2116         | 3      | +     | +     | –     |
| 4       | 3                      | acylhexapeptide assembly | ComB         | NRPS               | CoA, ATP                 | 9992                | 14587             | 4596           | 1531         | 3      | +     | +     | –     |
| 5       | 4                      | acylhexapeptide assembly | ComC         | NRPS               | CoA, ATP                 | 14686               | 29382             | 14697          | 4898         | 4      | –     | –     | –     |
| 6       | 5                      | acylhexapeptide assembly | ComD         | NRPS               | CoA, ATP                 | 29420               | 35914             | 6495           | 2164         | 5      | +     | +     | –     |
| 7       | 6                      | unknown function   | ComE         |                    |                          | 35919               | 36140             | 222            | 73           | 6      | –     | –     | –     |
| 8       | 10                     | regulation         | ComF         | ion antiporter     |                          | 36179               | 37456             | 1278           | 425          | 7      | –     | –     | –     |
| 9       | 8,9                    | halogenation       | ComH         | halogenation       | NADPH                    | 37485               | 38978             | 1494           | 497          | 8      | +     | +     | –     |
| 10      | 8,9                    | oxidative cross-linking | ComJ         | oxidation          | NADPH                    | 38996               | 40189             | 1194           | 397          | 9      | +     | +     | –     |
| 11      | 7                      | oxidative cross-linking | ComK         | oxidation          | NADPH                    | 40203               | 41474             | 1272           | 423          | 10     | +     | +     | –     |
| 12      |                        | oxidative cross-linking | ComL         | ferredoxin         | NADPH                    | 41519               | 41737             | 219            | 72           | 11     | +     | +     | –     |
| 13      | 22                     | Hpg                 | Hmo          | oxidation          | FAD                      | 42933               | 41815             | 1119           | 372          | 12     | +     | +     | –     |
| 14      | 21                     | Hpg                 | HmaS         | dioxygenation      | Fe                       | 43936               | 42920             | 1017           | 338          | 13     | –     | –     | –     |
| 15      | 17                     | Hpg                 | HpgT         | aminotransferase   | PLP                      | 44174               | 45520             | 1347           | 448          | 14     | +     | –     | +     |
| 16      | 1                      | Hpg                 | PD           | prephenate dehydrogenase |                          | 45517               | 46665             | 1149           | 382          | 15     | –     | –     | –     |

[a] C = clone. [b] P = purified. [c] A = Analyzed.

---

Table 6: Genes in the complestatin biosynthetic cluster.

| Orf no. | Cep homologue (orf no.) | Metabolic pathway | Protein name | Predicted function | Cofactors or cosubstrates | Relative start codon | Relative stop codon | No. base pairs | No. amino acids | Transcript | C[a] | P[b] | A[c] |
|---------|------------------------|--------------------|--------------|--------------------|--------------------------|---------------------|-------------------|----------------|----------------|------------|-------|-------|-------|
| 1       | 7                      | oxidative cross-linking | OxyA         | oxidation          | NADPH                    | 224                 | 1399              | 1176           | 391          | 1      | –     | –     | –     |
| 2       | 8                      | oxidative cross-linking | OxyB         | oxidation          | NADPH                    | 1449                | 2645              | 1197           | 398          | 2      | –     | –     | –     |
| 3       | 9                      | oxidative cross-linking | OxyC         | oxidation          | NADPH                    | 2795                | 4015              | 1221           | 406          | 3      | –     | –     | –     |
| 4       | 10                     | halogenation       | BhaA         | halogenation       | NADPH                    | 4076                | 5551              | 1476           | 491          | 4      | –     | –     | –     |
| 5       | 11                     | glycosylation      | BgftA        | attach sugar       |                          | 5634                | 6824              | 1191           | 396          | 5      | –     | –     | –     |
| 6       | 12                     | glycosylation      | BgftB        | attach sugar       |                          | 6889                | 8118              | 1230           | 409          | 6      | –     | –     | –     |
| 7       | 13                     | glycosylation      | BgftC        | attach sugar       |                          | 8266                | 9495              | 1230           | 409          | 7      | –     | –     | –     |
| 8       | 14                     | TDP-ketovancosamine | OrfX'        | methylation        | SAM                      | 9529                | –                 | –              | –            | 8      | –     | –     | –     |

[a] C = clone. [b] P = purified. [c] A = Analyzed.

B. K. Hubbard and C. T. Walsh

1 → 2 → Com A → Com B  
Cep C  
Cep D  
8 → 9 → 10 → 11 → 12  
13 → 14 → 15 → 16  

NRPS  
Orf 3 - ComA  
Orf 4 - ComB  
Orf 5 - ComC  
Orf 6 - ComD  
Oxidative Cross-linking  
Orf 10 - ComJ  
Orf 11 - ComK  
Orf 12 - ComL  
Halogenation  
Orf 9 - ComH  

HPG Biosynthesis  
Orf 13 - Hmo  
Orf 14 - HmaS  
Orf 15 - HpgT  
Orf 16 - PD  
Other Genes  
Orf 1 - ComG  
Orf 2 - ComI  
Orf 7 - ComE  
Orf 8 - ComF  

**Figure 7.** Genes in the complestatin biosynthetic cluster.

The aminotransferase homologue in the chloroeremomycin cluster (orf 17) is a pyridoxal phosphate dependent enzyme that converts 4-hydroxybenzoylformate into L-4-hydroxyphenylglycine, with L-tyrosine as the amino donor.[74] The aminotransferase had been designated L-hydroxyphenylglycine transaminase (HpgT). The use of L-tyrosine by HpgT allows for the conversion of prephenate and L-tyrosine into L-4-hydroxyphenylglycine by the cycle shown in Scheme 12. This cycle is catalytic with respect to prephenate and stoichiometric with respect to L-tyrosine.

The genes that encode for prephenate dehydrogenase (PD), HmaS, Hmo, and HpgT are present in both the calcium-dependent antibiotic (CDA) and complestatin gene clusters. Both of these compounds contain Hpg (Scheme 13). The HpgT enzyme from the complestatin gene cluster has been characterized and shown to utilize both L-Hpg and 3,5-dichloro-L-4-hydroxyphenylglycine in the reverse direction.[61] It is presumed that these four enzymes constitute the pathway by which hydroxyphenylglycine is synthesized for all of the glycopeptide antibiotics as well in other compounds (for example, ramoplanin, nocardicin) that contain Hpg.

**9.2. 3,5-Dihydroxyphenylglycine**

Vancomycin and teicoplanin contain the nonproteinogenic amino acid 3,5-dihydroxyphenylglycine at residue 7 and at residues 3 and 7, respectively. Initial feeding experiments showed that all of the carbon atoms in 3,5-dihydroxyphenylglycine in avoparcin, vancomycin, aradacin, and ristocetin are derived from acetate and led to the hypothesis that 3,5-dihydroxyphenylglycine was derived from a polyketide synthetase reaction.[69–71,76] However, the method by which the proposed tetraketide was converted into the final amino acid was unclear.

There are four orfs (orf 27, 28, 29, and 30) in the chloroeremomycin cluster that are hypothesized to be involved in the biosynthesis of 3,5-dihydroxyphenylglycine. The protein encoded by gene orf 27 is homologous to chalcone synthase, a member of the Type III polyketide synthase family, and Orfs 28–30 are members of the crotonase superfamily. The mechanism for chalcone synthase, an

Angew. Chem. Int. Ed. 2003, 42, 730–765

Biosynthesis of Antibiotics

![Chemical Reaction Diagram](https://i.imgur.com/chemical_reaction.png)

**Scheme 11.** Comparison of the enzymatic mechanisms of HppD and HmaS.

![Biosynthesis of L-Hpg](https://i.imgur.com/biosynthesis_l_hpg.png)

**Scheme 12.** Biosynthesis of L-Hpg.

Expression of DpgA–D in *E. coli* has allowed purification and initial investigation of the mechanism.[79] Both DpgB and DpgD show enoyl CoA hydratase activity and are likely to function physiologically in the dehydratase direction. When DpgB and D are added to incubations containing DpgA and malonyl CoA, the basal rate of formation of 3,5-dihydroxyphenylacetyl CoA is accelerated more than 30-fold. DpgC is sufficient to convert this intermediate acyl CoA into 3,5-dihydroxyphenylglyoxalate, which is then transaminated by the HpgT enzyme (see Section 9.1) to produce the 3,5-dihydroxyphenylglycine.[78,79] The DpgC reaction involves an O₂-dependent four-electron oxidation of the α-CH₂ group of the phenylacetyl CoA skeleton to the C=O group of phenylglyoxylal CoA and subsequent thioester hydrolysis to yield the phenylglyoxalate product.[79]

enzyme in plant flavonoid production, serves as precedent for Orf 27. Orfs 28–30 are members of the crotonase superfamily, one of the most diverse enzyme superfamilies, which catalyzes over 15 different reactions.[77] These reactions can consist of hydrations/dehydrations, isomerizations, carbon-bond forming/breaking reactions, and thioesterifications.

The proposal for the biosynthesis of 3,5-dihydroxyphenylglycine is shown in Scheme 14. DpgA (Orf 27) converts four malonyl CoA units into the linear polyketide as well as directing decarboxylation and regiospecific cyclization.[78]

---

### 9.3. β-Hydroxytyrosine

Residues 2 and 6 in vancomycin and teicoplanin are β-hydroxytyrosine groups. The 2R,3R-3-OH diastereomer is found at position 2 while the 2S,3R-3-OH isomer is found at residue 6. As will be noted in Section 10, the CepA subunit of the nonribosomal peptide synthetase (NRPS) assembly line contains a predicted epimerase (E) domain in module 2, which suggests that a 2S,3R-3-OH Tyr-S-T₂ acyl enzyme intermediate is epimerized before chain elongation (T = thio-

lation domain). In turn this predicts that the A domains in both modules 2 and 6 will activate the L-β-OH-tyrosine (2S,3R) and focuses attention on how and when the β-oxygenation occurs.

Three orfs, Cep 18, 19, and 20, that are contiguous in the Cep biosynthetic cluster suggest there is a pathway from L-Tyr to L-β-OH-Tyr. Orf 19 is a freestanding adenylation-thiolation (A-T) di-domain subunit, predicted to activate L-Tyr as Tyr-AMP and then install it as the covalent thioester intermediate of the L-Tyr-S-T domain (Scheme 15). Orf 20 is

**Scheme 13.** Presence of Hpg in complestatin and a calcium-dependent antibiotic (chloroeremomycin).

**Scheme 14.** Proposal for the biosynthesis of L-3,5-dihydroxyphenylglycine (L-Dpg).

**Scheme 15.** Proposal for the biosynthesis of S,R-β-hydroxy-L-tyrosine in chloroeremomycin.

Biosynthesis of Antibiotics

predicted by informatic analysis to be a hemeprotein, and that has been validated by expression and purification from *E. coli* (Hubbard and Walsh, unpublished results). Orf 20 is the likely β-hydroxylase, which acts on the tyrosyl thioester of Orf 19 to yield the tethered β-OH-Tyr-S-Orf19 product. Orf 20 is predicted to have thioesterase activity and release the free β-OH-Tyr for use by the CepA and CepB subunits in heptapeptide assembly.

Support for this assignment of Cep 18–20 function is the recent demonstration that two homologues of Orf 19 and 20, NovH and NovI in the novobiocin biosynthetic cluster,[80,81] indeed carry out the first two steps shown in Scheme 16 to generate the 2S-3R-3-OH-Tyr-S-NovI covalent intermediate on the way to formation of the aminocoumarin ring.[81] There are also three Orfs, NikP1, NikQ, and NikP2, that are homologous to Cep 18–20 in the biosynthetic cluster for the antifungal compound nikkomycin Z[82,83] (Scheme 17). These three nikkomycin proteins have been purified by heterologous expression in *E. coli* and proven to act as predicted above. NikP1 makes a covalent His-S-enzyme on its T domain, the hemeprotein hydroxylase NikQ converts it into β-OH-His-S-NikP1, and NikP2 is a thioesterase generating the free β-hydroxyhistidine.[83]

Thus, Cep 18–20 are a three-enzyme cassette for sequestering some of the L-Tyr pool, regio- and stereospecifically hydroxylating it with O₂, and then releasing the free L-β-OH-Tyr for use by the NRPS assembly line.

### 9.4. Chlorination of β-Hydroxytyrosines

Each of the two β-hydroxytyrosines at residues 2 and 6 in vancomycin and teicoplanin are chlorinated at position 3 of the aromatic ring, adjacent to the phenol OH group. The cross-linked heptapeptide scaffold of vancomycin is a single atropisomer at both the 2–4 and 4–6 aryl ether linkages. Little is known about the mechanism or timing of the biosynthetic chlorination in this pathway or in the complestatin biosynthesis where 3,5-dichloro-4-hydroxyphenylglycine groups are present in the final cross-linked antibiotic. Both the Cep cluster (Orf 10 or CepH) and the Com cluster (Orf 9, or ComH) have proteins thought to be halogenases, by homology to other non-heme halogenases,[84–86] and most convincingly to a homologue in chlortetracycline biosynthesis.[87] Purified CepH and ComH are yellow, because of a stoichiometric content of bound flavin coenzyme; neither contains any detectable iron (M. Burkart and C. Walsh, unpublished results). If these indeed are catalytic components of an aromatic ring halogenase, then the mechanism advanced by van Pee and co-workers,[88] (Scheme 18) is plausible, with the flavoenzymes acting as epoxidases of the electron-rich phenolic rings of Tyr or Hpg side chains to yield, transiently, the epoxyhexadiene that could be opened by chloride ions and then dehydrated to the 3-chlorotyrosine.

Alternatively the FADH₂ form of Orf 10 (FAD = flavin adenine-dinucleotide) could yield a flavin hydroperoxide and then a flavin hypochloride (Fl-OCl) species that could be a direct chlorinating agent. It is unclear when chlorination occurs, whether on the free amino acid (for example, L-β-OH-

Angew. Chem. Int. Ed. **2003**, 42, 730–765

a) CepH (FAD) + NADH + H⁺ → CepH (FADH₂) + NAD⁺

b) Cep FADH₂, O₂ [FI-OOH]

Scheme 18. Possible mechanisms of ComH. a) Direct epoxidation and ring opening with a nucleophilic chloride ion. b) Generation of an electrophilic chloride ion for electrophilic aromatic substitution.

### 10. The Nonribosomal Peptide Synthetase Assembly Line

Once the nonproteinogenic amino acid monomers needed for the nonribosomal heptapeptide are generated to enable vancomycin biosynthesis to proceed, the nonribosomal peptide synthetase (NRPS) assembly line comes into play. The Cep gene cluster encodes three contiguous, large open reading frames (orfs 3–5) that are the three subunits CepA, CepB, and CepC (Figure 8) that constitute the heptapeptide synthetase. At a presumed stoichiometry of 1:1:1, CepA (336 kDa), CepB (436 kDa), and CepC (200 kDa) comprise almost a megadalton (972 kDa) assembly line (Figure 8). The organizational logic of this NRPS assembly line appears to follow the predictions for thio-templated nonribosomal peptide synthetase logic.[89–91] There are seven amino acids to be activated; one by each of seven adenylation (A) domains, whose placement in the assembly line determines the sequence of the heptapeptide released at the end. All the A domains select a specific amino acid, activate it as the aminoacyl-AMP, and then transfer the aminoacyl moiety to a paired peptidyl carrier protein domain, also known as the thiolation (T) domain because of the post-translationally introduced phosphopantetheinyl arm that ends in a cysteine thiol moiety which acts as the nucleophile on the

**Scheme 19. Phosphopantethylation of the thiol domain.**

aminoacyl-AMP to produce the aminoacyl-S-T domain (Scheme 19). Thus the seven paired A-T domains activate amino acids and install them as covalent aminoacyl-S-enzyme intermediates (Figure 9) In the CepA/CepB/CepC assembly line the seven modules are distributed 3/3/1, with Leu₁, β-OH-Tyr₂, and Asn₃ to be activated on CepA, HPG₄ and HPG₅ installed at modules 4 and 5, and β-OH-Tyr₆ at module 6. CepC has the seventh module, which should load DHPG₇ as the thioester.

Six peptide bonds have to be made to form a heptapeptide and there are six condensation (C) domains, two in CepA, three in CepB, and one in CepC. Each time a C domain makes a peptide bond, the growing chain is translocated directionally from the most upstream T domain to the next

**Figure 8. NRPS assembly line in chloroeremomycin.**

Biosynthesis of Antibiotics

![Chemical Structures](chemical_structures.png)

Figure 9. NRPS assembly line in chloroeremomycin with activated amino acids loaded on the thiol domains.

downstream T domain. For example, two chain elongation steps, catalyzed by the C domains in modules 2 and 3, respectively, will generate a tripeptidyl-S-T₃ at the C-terminus of CepA. The next peptide bond-forming step involves transfer of the peptidyl chain *trans* to Hpg-S-T₄ across the interface between the CepA and CepB subunits. Peptidyl chain growth on such NRPS assembly lines involves a cascade of elongating peptidyl-S-enzyme intermediates sequestered along the way from adventitious hydrolysis by water.

The vancomycin and teicoplanin heptapeptides have the configuration D-D-L-D-D-L-L. In general, only L-amino acids are available in substantial concentrations, even in bacterial cells, unless there are dedicated amino acid racemases, as for alanine at the start of peptidoglycan assembly.[92,93] In particular, the Hpg group produced by Orfs 21, 22, and 17 even yields the L isomer of this nonproteinogenic amino acid.[74] The common solution for the generation of D-amino acid residues in nonribosomal peptide antibiotics is the in situ epimerization on the NRPS assembly line by specialized epimerase (E) domains.[94,95] E domains are found in modules where an L-aminoacyl-S-T acyl enzyme domain is epimerized to a L-aminoacyl-S-T domain (Figure 9). This could occur on the free aminoacyl-S-T intermediates before C-domain-mediated peptide bond formation, or after reaction at the C domain, in which case the peptidyl-S-T domain must be epimerized before chain transfer to the next downstream module (Scheme 20). Inspection of the CepA/CepB/CepC assembly line reveals three predicted E domains in modules 2, 4, and 5 in the correct regiospecific placement to account for formation of D-β-OH-Tyr₂, D-Hpg₄, and D-Hpg₅

during chain elongation. However, there is no E domain associated with module 1,[60] which raises the question of how this D configuration is generated. Analysis of the loading of the CepA module 1 heterologously expressed in *E. coli* indicated both L-Leu and D-Leu were activated and installed as the Leu-S-T₁ acyl enzyme species.[96] If this route is operant in the producer *A. orientalis*, it mandates a Leu epimerase, which has not yet been identified.

Once the full-length D-D-L-D-D-L-L-heptapeptidyl chain has arrived at the last thiol way station (T₇), the covalent heptapeptidyl thioester needs to be disconnected for the enzymatic assembly line to proceed through subsequent catalytic cycles. Almost all NRPS assembly lines have a specialized thioesterase (TE) catalytic domain as their most C terminal domain; a TE is indeed the last of the 24 domains in the Cep synthetase, at the C terminus of CepC. All TE domains have serine residues in the active site that act as nucleophiles to transfer the acyl chain and make in this case the heptapeptidyl-O-Ser-TE acyl enzyme (Scheme 21). TE domains can have specific cosubstrates for deacylation, such that macrocyclization can occur with a nucleophilic group internal to the acyl chain, but here intermolecular hydrolysis is the physiologic fate of the acyl-O-TE intermediate, thus releasing the D-D-L-D-D-L-L-heptapeptide acid as the sole product of the assembly line.[97]

Two additional features of this particular NRPS assembly line for the vancomycin/chloroeremomycin heptapeptide scaffold are worth comment. First, is the unresolved question at present of the timing of both β-hydroxylation and ring chlorination of Tyr₂ and Tyr₆. Until modules 2 and 6 are

B. K. Hubbard and C. T. Walsh

Table 7: Specificity of the condensation domains in chloroeremomycin.

| Condensation module | Peptidyl donor specificity | Aminoacyl acceptor specificity |
|---------------------|---------------------------|-------------------------------|
| $C_1$              | D                         | D                             |
| $C_2$              | D                         | L                             |
| $C_3$              | L                         | D                             |
| $C_4$              | D                         | D                             |
| $C_5$              | D                         | L                             |
| $C_6$              | L                         | L                             |

bonds, then the $C_1$ peptide synthase domain is D/D specific for donor and acceptor, $C_4$ is L/D specific, and $C_5$ is D/D specific (Table 7). If the E domains in modules 2, 4, and 5 work after the C domain condensation steps then $C_2$ will be D/L specific, $C_6$ will be L/L specific, and $C_5$ will be D/L specific. For any combinatorial biosynthetic schemes where libraries of C domains might be made, it is essential to know their stereo preferences for both donor and acceptor. Whatever the timing of the epimerization domains, the $C_2$ and $C_5$ domains must be D/L specific, while $C_6$ is L/L specific. Indeed if $C_1$ and $C_4$ would be D/D specific these would be intriguing peptide synthase components in an enzyme library.

### 11. Post Assembly Line Tailoring of the Heptapeptide

#### 11.1. Cross-Linking of Residues 2-4-6 and 5-7 in Vancomycin

Once the full-length, chlorinated acyclic heptapeptide has been produced by the CepA, CepB, and CepC assembly line it becomes a substrate for side-chain cross-linking and rigidification. It is not yet known whether one or more of the cross-links occurs at the heptapeptidyl-S-CepC acyl enzyme stage or whether chain release precedes recognition by the cross-linking heme proteins. Both the Cep cluster and the partially sequenced balhimycin cluster,[62] which make the identical heptapeptide scaffold, have three contiguous orfs, for example, Cep 7, Cep 8, and Cep 9, which are predicted to be heme proteins by homology to cytochrome P450 enzymes.[60,62] Likewise, sequencing of the A47934 cluster from *S. toyocaensis* reveals that four P450 Orfs are predicted for a teicoplanin-type of heptapeptide scaffold that has a fourth cross-link at residues 1 and 3.[145] Purification of Cep 7, Cep 8, and Cep 9 in *E. coli* expression systems yields fully loaded heme proteins (Hubbard and Walsh, unpublished results). Although in vitro reconstitution of the cross-linking of the aryl side chains at 2–4, 4–6, and 5–7 have not yet been reported, a knockout of the balhimycin orf 7 equivalent (oxyA, which corresponds to cepE) led to isolation of a partially cyclized aglycone, with the 4–6 cross-link intact (Scheme 22). This result suggests that the aryl ether between the β-OH-3-Cl-Tyr₆ and the 4-OH-PheGly₄ is constructed first and that the three hemeprotein oxidases may act regiospecifically and introduce particular cross-links in a given order. This conclusion was validated by subsequent observation that a knockout of oxyC from the balhimycin cluster (equivalent to cepG) gave a glycopeptide with two of the three cross-links intact (2–4 and 4–6), which indicates that the 5–7 C–C link is

expressed (heterologously?) and their A domains assayed, it cannot be established whether Tyr is the preferred substrate or whether it is β-OH-Tyr or 3-chloro-Tyr. We commented on the probable hydroxylation of Tyr by Orfs 18, 19, 20 in Section 9.3, and suggested that 2S-3R-3-OH-Tyr is the substrate for the A₂ and A₆ domains.

Second is the issue of stereospecificity of the C domains as peptide synthetases, but this depends on the timing of the E₂, E₄, and E₅ domains. If the species epimerized are the free amino acyl S-T intermediates (2S-3R-3-OH-Tyr-S-T₂, 2S-HPG-T₄, and 2S-HPG-S-T₅) before formation of peptide

Biosynthesis of Antibiotics

cyclization. The chlorine moieties on the Tyr₂ and Tyr₆ rings may also affect cyclization, for example, by tuning the redox potential to facilitate one-electron pathways in aryl ether formation.

Teicoplanin has one additional cross-link, which is enabled by the substitution of Leu₁ and Asn₃ in vancomycin by 4-OH-phegly₁ and 3,5-(OH)₂-PheGly₃ in the teicoplanin precursor. A fourth heme protein oxidase has indeed been found in the A47934 cluster.[145] There are two hemeproteins in the Com cluster for introducing the two cross-links into the complestatin scaffold (Table 6 and Figure 7).

### 11.2. N-Methylation of D-Leu₁

The N-methylation of D-Leu₁ is catalyzed by Orf 16 (MtfA), which has been purified and shown to have N-methyltransferase activity against the aglycone and linear forms (albeit with low specific activities).[63] When mutants in the balhimycin biosynthetic pathway accumulate linear or partially cross-linked precursors they are not yet N-methylated, which suggests this may be a late-stage event, perhaps occurring on the fully cross-linked peptide scaffold, just before glycosylation. The function of N-methylation is not well understood.

### 12. Glycosylation: The Thymidinediphospho Sugar Donors

The vancomycin and teicoplanin subfamilies are glycopeptides and the glycosylations occur at the last stages of antibiotic maturation, on the cross-linked peptide scaffolds. The placement and identity of the sugars decorating the aglycone scaffold are distinct. In the vancomycin/chloroeremomycin/balhimycin group, there is a glucosyl moiety attached to the phenolic oxygen atom derived from 4-OH-PheGly₄ and an L-aminodeoxyhexose of the vancosamine series is then attached in a 1,2-linkage. The 4-substituent of the vancosamine sugar varies, from 4-oxo in balhimycin X to 4-OH (axial) in vancomycin and 4-OH (equatorial) in chloroeremomycin, which suggests there is a common pathway for biosynthesis (Scheme 25). There are also 3-hydroxy and 3-amino epimeric variants in other glycopeptide members.[98–101] A second L-epivancosamine residue is attached at the benzylic oxygen atom of β-OH-Tyr₆ in chloroeremomycin. There are no comparable amino deoxy sugars in the teicoplanin group, but rather more conventional sugars of primary metabolism, 2-N-acetyl-D-glucose (GlcNAc) and D-mannose (Scheme 1). The GlcNAc introduced at the comparable phenolic oxygen atom (4-OH-PheGly₄) of the teicoplanin aglycone has the short-chain acetyl replaced by a branched C10 fatty acyl group.

Deoxy sugars and/or amino deoxy sugars are common components of other natural product antibiotics, including the noviose sugar in coumarin inhibitors of DNA gyrase,[80] the macrolide antibiotics (for example, D-desosamine and L-mycarose), and sugars in enediyne and bleomycin classes of antitumor antibiotics (Scheme 26).[102,103] The mixture of

Angew. Chem. Int. Ed. 2003, 42, 730–765

![Reviews](#)

B. K. Hubbard and C. T. Walsh

![Chemical Structure 1](#)  
$\xrightarrow{\text{CepF}}$  
![Chemical Structure 2](#)  

$\downarrow$  
CepE  

![Chemical Structure 3](#)  
$\xleftarrow{\text{CepG}}$  
![Chemical Structure 4](#)  

**Scheme 23.** Potential coupling route of the chloroeremomycin heptapeptide by CepE, CepF, and CepG.

---

![Mechanism Diagram 1](#)  

**C-O-C Linkage**  
![Mechanism Diagram 2](#)  

**C-C Linkage**  
![Mechanism Diagram 3](#)  

**Scheme 24.** Potential mechanisms of the heme-catalyzed oxidative couplings performed by CepE, CepF, and CepG.

754  
Angew. Chem. Int. Ed. 2003, 42, 730–765

Biosynthesis of Antibiotics

![Vancosamine](https://i.imgur.com/1234567.png)

![Epivancosamine](https://i.imgur.com/8901234.png)

Vancomycin

Balhimycin

Chloroeremomycin

**Scheme 25.** Biosynthesis of vancosamine and epivancosamine from 4-oxovancosamine and incorporation into vancomycin, chloroeremomycin, and balhimycin.

Novobiocin

5-O-desosaminyl-Tylactone

Bleomycin

Kedarcidin chromophore

**Scheme 26.** Structural formulas of novobiocin, 5-O-desosaminyl-tylactone, bleomycin, and the kedarcidin chromophore.

Angew. Chem. Int. Ed. **2003**, *42*, 730–765

deoxy centers, methyl groups, amino groups, and residual hydroxy substituents impart a mixed hydrophobic–hydrophilic character to the deoxy sugar components of antibiotics that may be crucial for recognition. Williams and co-workers have characterized back-to-back dimerization equilibria with a wide range of \( K_d \) values for the glycopeptides (Figure 10). [104, 105] These values are dependent on and mediated by hydrogen-bonding interactions through the deoxy sugars, and the authors have argued that these correlate with antibiotic potency. The sugars certainly play a role in conferring water solubility to the hydrophobic aglycones, both in the glycopeptides and in the macrolide antibiotics. The aglycone in the erythromycin biosynthesis has negligible antibiotic activity. Addition of mycarose generates the set of interactions required for binding at the 50S ribosome (Scheme 27) and produces antibiotic activity. It is also likely

that glycosylation may be part of the mechanism by which antibiotic-producing bacteria engage in self-protection as extra glycosylation occurs during oleandomycin biosynthesis, thus leading to inactive macrolide, which is activated only after export and extracellular glycosidase action.[106] Partially cyclized peptide intermediates accumulating in mutants of the balhimycin producer are adventitiously glycosylated, perhaps for self-protection and/or to facilitate export.[59]

The general strategy for the biosynthesis of deoxy and amino/deoxy sugar components of antibiotics is now well understood.[107–109] First, all the modified hexoses are built as nucleoside diphospho sugars (NDP sugars) with the pyrophosphate linkage of the NDP attached to C₁ of the hexose, thus activating it for attack by some cosubstrate nucleophile in subsequent glycosyl transferase catalysis. The NDP deoxy sugars are UDP or thymidinediphospho (TDP) sugars in the Streptomyces and Actinoplanes antibiotic producers. All the enzymatic manipulations to generate TDP-deoxy sugars dedicated to antibiotic maturation start from the readily available, activated sugar TDP-glucose. Scheme 28 lists the enzymatic transformations for conversion of TDP-glucose into TDP-L-β-epivancosamine, and indicates the adjustment of the redox state and/or substituent and configuration at each carbon site (C₂–C₆) to emphasize the exquisite tailoring of the TDP-sugar donor. The other general feature of the biosynthetic pathways to antibiotic sugars is that the encoding genes, for example, for the transformations of Scheme 28, are clustered with the other glycopeptide biosynthetic genes (Figure 6 and Table 4). In the Cep gene cluster, the five open reading frames, orf 23, 25, 14, 26, and 24, act sequentially in that order to convert TDP-4-keto-6-deoxyglucose into TDP-L-epivancosamine (Scheme 29).[110, 111] The clustering of the biosynthetic NDP-deoxy sugar genes in antibiotic operons enables coordinate regulation and just-in-time biosynthesis of these key monomer units during antibiotic production.

Scheme 29 delineates the specific enzymatic steps that have been elucidated for the five committed ORFs of Figure 6, by purification of each of the five enzymes, now known as EvaA, EvaB, EvaC, EvaD, and EvaE (Eva = 4-L-epivancosamine). The enzyme catalyzing the first step, that is, conversion of TDP-glucose to the 4-keto compound, followed

**Scheme 27. Attachment of thymidinediphosphate sugars (TDP-sugars) in the biosynthesis of erythromycin.**

**Figure 10.** Back-to-back dimerization of vancomycin, shown as a space-filling model (bound to N₂₆-bis-acyl-L-Lys-D-Ala-D-Ala and the hydrogen-bonding network).

Biosynthesis of Antibiotics

![Chemical Structure 1](chemical_structure_1.png)

| Position | Change          | Type           | Gene   |
|----------|-----------------|----------------|--------|
| Conformation | ring-flip      | ${}^{4} \mathrm{C}_{1} \rightarrow{ }^{1} \mathrm{C}_{4}$ | EvaD   |
| C-2       | deoxygenation   | $-\mathrm{OH} \rightarrow-\mathrm{H}$ | EvaA   |
| C-3       | methylation    | $-\mathrm{H} \rightarrow-\mathrm{Me}$ | EvaC   |
| C-3       | amination      | $-\mathrm{OH} \rightarrow \mathrm{NH}_{2}$ | EvaB   |
| C-4       | epimerization  | $\mathrm{S} \rightarrow \mathrm{R}$ | EvaE   |
| C-5       | epimerization  | $\mathrm{R} \rightarrow \mathrm{S}$ |        |
| C-6       | deoxygenation  | $-\mathrm{OH} \rightarrow \mathrm{H}$ | RfbB   |

**Scheme 28.** Transformations in the biosynthesis of TDP-L-epivancosamine from TDP-glucose.

![Chemical Reaction Scheme 1](chemical_reaction_scheme_1.png)

**Scheme 29.** Biosynthesis of TDP-L-epivancosamine and its attachment in the assembly of chloroeremycin.

by dehydration to the 4-keto-6-ene and reduction of the conjugated olefin to the 6-methyl sugar to produce the TDP-4-keto-6-deoxyglucose substrate for EvaA, is not encoded in the Cep cluster. This is a well-known and well-studied enzymatic transformation. It is likely that it initiates several pathways and is a more general housekeeping enzyme for sugar metabolism in bacterial cells, and so is not part of the specialized, dedicated antibiotic gene cluster.[112, 108]

EvaA acts to create the 3,4-diketo sugar as part of the strategy to deoxygenate C2, by abstraction of the C3-H,

acidification by its adjacency to the 4-keto group, and elimination of the C2-OH group. The resulting 2,3-ene is the enol form of the 3,4-diketo-6-deoxy sugar nucleotide. The 3-keto group is reductively aminated by EvaB, a pyridoxal phosphate-dependent aminotransferase, which acts with regio- and stereoselectivity to introduce the amino group in an equatorial arrangement in position 3. EvaC is an S-adenosylmethionine-dependent C-methyl transferase, which introduces the methyl group with inversion of stereochemistry when attacked by the carbanion of the C3 amino sugar. Now, C3 has

Angew. Chem. Int. Ed. 2003, 42, 730–765

Table 8: Some sugars found in the vancomycin family.

| Sugar             | Structure                                                                 | Antibiotic                                                                                          |
|-------------------|--------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
| L-olivose         | HO—O—OH                                                                  | orienticin B                                                                                       |
| L-rhodinose       | HO—O—OH                                                                  | UK69542                                                                                            |
| L-acosamine       | HO—O—OH                                                                  | actinoidin A, MM47767, MM55256                                                                     |
| L-actinosamine    | MeO—O—OH                                                                 | actinoidin, MM47767, MM55256                                                                       |
| L-ristosamin      | HO—NH₂—OH                                                                | actaplanin, avoparcin, chloropolysporin A–C, galacardin, symnonicin A–C, helvecardin, ristocetin A, B |
| L-vancosamine     | NH₂—O—OH                                                                 | A42867, A51568 B, dechlorovancomycin, demethylvancomycin, M43, vancomycin CDP-I, vancomycin |
| 3-epi-L-vancosamine | HO—NH₂—OH                                                              | A35512 B                                                                                           |
| L-eremonsamine    | HO—NH₂—OH                                                                | chloroorienticin, dechloroeremomycin, eremomycin, MM47761, MM49721, orenticin                      |
| 4-oxovacosamine   | O—NH₂—OH                                                                 | A83850, balhimycin, balhimycin V, dechlorobalhimycin V, deglucobalhimycin, demethylbalhimycin, methylbalhimycin |

been both aminated and C-methylated and its configuration fixed. At this point EvaD, a member of the TDP-sugar 3- and 5-epimerase family, acts. Only C5 bears an acidic hydrogen atom so it can be enolized by EvaD and equilibrated to the L stereochemistry at C5, thus creating the TDP-L-β-4-oxo-vancosamine. This is presumed to be the 4-oxovancosamine donor in balhimycin biosynthesis (Scheme 2). Finally, EvaE catalyzes the stereospecific reduction of the 4-oxo group (by hydride addition from NADPH) to yield the equatorial 4-OH group of TDP-epivancosamine, thus completing the transformation of TDP-glucose (Scheme 29). In TDP-L-β-vancosamine biogenesis, only the last step will be distinct; a homologue of EvaE that introduces a hydride ion from the opposite face of the 4-keto group yields the 4-OH group in an axial orientation.

Comparable enzymes that process NDP sugars (C₂, C₃, and C₆ deoxygenation, C₃ amination and methylation, and epimerization of the C₃ and C₅ hydroxy configuration) are likely for other sugars found in the vancomycin family (Table 8), and are well-precedented in the TDP-D-desosamine, TDP-L-mycarose, and TDP-daunosamine biosynthetic pathways.[113–115]

### 13. Glycosylation: Peptide Antibiotic Glycosyl Transferases

#### 13.1. GtfA, GtfB, and GtfC in Cep; GtfD and GtfE in Van Clusters

As shown in Figure 6 and Table 4 there are three contiguous genes (encoding orfs 11–13) in the Cep gene cluster that have homologies to glycosyl transferases. After heterologous expression and purification to homogeneity the three proteins have been proved to be the glycosyl trans-

Biosynthesis of Antibiotics

![Chemical Structure 1](chemical_structure_1.png)

UDP-Glucose  
GtfB  

![Chemical Structure 2](chemical_structure_2.png)

UDP-Epivancosamine  
GtfC  

![Chemical Structure 3](chemical_structure_3.png)

**Scheme 31.** Order of addition of uridinediphosphate glucose (UDP-glucose) and UDP-L-epivancosamine by GtfB and GtfC, respectively.

transferases, GtfD and GtfE, which are homologues of GtfB and GtfC, respectively, and they utilize TDP-glucose and TDP-L-β-vancosamine, respectively,[117,60] to create the comparable vancosaminyl-1,2-glucose disaccharide moiety at 4-OH-Phegly₄ of the cross-linked aglycone scaffold. The X-ray structure of GtfB has been determined (Figure 11): it reveals a two-lobed enzyme with only a single strand connecting the two domains. GtfB has architectural similarity to the MurG transglycosylase in peptidoglycan biosynthesis and to a phage DNA glucosyl transferase.[118–120] GtfA, GtfC, GtfD, GtfE, and other antibiotic glycosyl transferases show sequence homology to GtfB, thus making it likely they will all be part of the same architectural super family.[62]

The corresponding two or three Gtfs in the teicoplanin pathway (one enzyme could transfer a GlcNAc to both residue 4 and 6) have not yet been characterized. Since the TDP sugars are more prosaic, it will be of interest to see how homologous the teicoplanin Gtfs will be to the above Gtfs in the vancomycin subfamily. The timing of acylation of the glucosamine in teicoplanin by the 10-methylundecanoyl group is not known, nor has the acyl transferase been identified. A possible precedent is the two-step enzymatic conversion of GlcNAc residues during lipid A biosynthesis to long-chain 3-OH acyl derivatives, where a zinc-dependent deacetylase generates the free amino group, which is then reacylated by an enzyme using a 3-OH-myristoyl-S-acyl carrier protein (CP) substrate.[121] In the teicoplanin case, a 10-methylundecanoyl-S-ACP could be an acyl thioester intermediate rerouted from normal fatty acid biosynthesis in the teicoplanin producer.

![Crystal Structure of GtfB](crystal_structure_gtfb.png)

**Figure 11.** Crystal structure of GtfB.

The glycol transferases in clusters of other types of antibiotics have been assigned largely by homology and knockout or complementation studies. In the maturation of

Angew. Chem. Int. Ed. 2003, 42, 730–765

the erythronolide aglycone the L-mycarosyl unit is added first  
to the C3-OH group by EryBV and then the D-desosamine by  
Ery CIII (Scheme 27). A third sugar is present at the C6-OH  
in the polyketide macrolide megalomycin.[122] Analogous Gtfs  
have been detected in, for example, the novobiocin, the  
bleomycin, and the daunomycin clusters.[80,103,123]

### 14. Approaches to Treatment of Vancomycin-Resistant Bacteria

#### 14.1. Inhibitors of VanS, VanR, VanH, VanA, and VanX

With the molecular principle of vancomycin resistance in  
the VanA and VanB clinical phenotypes of VRE revealed,  
there is a pressing need for new drugs that work against VRE  
and an opportunity for rational design efforts to reverse the  
phenotype. One approach is to produce inhibitors to any of  
the five proteins, the two component regulatory pair VanS  
and VanR, and the three enzymes VanH, VanA, and  
VanX.[40,41] X-ray structures (Figure 12) of the VanA D-Ala-

**Figure 12. Crystal structure of VanA bound to a phosphate inhibitor.**

D-Ala ligase with a phosphanylphosphate analogue of the D-  
D-dipeptide substrate bound and of the VanX D-D-dipeptidase  
with a phosphinate inhibitor bound have been reported to aid  
ligand design.[124,125] While the phosphinate is a potent  
inhibitor of VanX and a substrate analogue of VanA (which  
generates the tightly bound phosphanylphosphate), bacterial  
cells are impermeable to the anionic phosphinate so there is  
no activity against intact bacteria. A mechanism-based  
inactivator of VanX has also been described that reveals a  
reactive group on cleavage of the peptide bond by the  
peptidase (Scheme 32).[126] No inhibitors of response regula-  
tors have yet been reported, but compounds described as  
inhibitors of other two-component sensor systems have been  
described.[127,128] However, it is likely those were acting as  
membrane perturbants and/or nonspecific surface agents that  
disrupt the transmembrane sensor. Given that the intra-  
cellular domain of VanS and other sensor proteins of bacterial  
two-component regulatory systems are histidine autokinases,  
it should be possible to build on the increasing number of  
protein kinase inhibitors that compete against ATP to find  
leads against this feature of VanS.[129]

**Scheme 32. Inactivation of VanX by a mechanistic-based inhibitor.**

#### 14.2. Antibiotics Active Against VRE

Two recently approved antibiotics have shown some  
efficacy against VRE during clinical development. The first  
is synercid, a combination of synergizing pristinamycins I and  
II (Scheme 33). This compound belongs to a class of anti-  
biotics also known historically as virginiamycins and strep-  
togramins. The pristinamycins are produced by Streptomyces  
pristinaespiralis, with pristinamycin I (trade name dalfoprus-  
tin) and pristinamycin II (trade name quinopristin) as semi-  
synthetic variants modified to improve the water solubility.  
Pristinamycin II is a nonribosomally generated hexapeptide,

**Scheme 33. Structure of pristinamycin I and pristinamycin II.**

while pristinamycin I is a hybrid of a polyketide synthase assembly line and three NRPS modules.[130–131] Both pristinamycin components act on the same site of the 50S ribosome subunit as erythromycin and block protein biosynthesis.[132]

The second antibiotic active against VRE is linezolid. Linezolid is the first example of a new class of synthetic antibacterial compounds, the oxazolidinones, that also act at the ribosome to block protein biosynthesis.[133]

### 14.3. Semisynthetic Glycopeptides Active Against VRE

When the clinical isolates of VRE strain began to appear, they were used to screen for new antibacterial agents that would be active against both sensitive and resistant strains of enterococci. Semisynthetic derivatives of vancomycins and chloroeremomycins with lipids attached to the amino group of the vancosamine or epivancosamine were active against VRE (Scheme 34). One representative compound, an *N*-chlorobiphenyl derivative of chloroeremomycin known as LY333328, recovered about 80 fold of the 1000 fold loss of activity against VRE and has proceeded into clinical trials.[134,135] This and related *N*-aryl and *N*-alkyl substituents are introduced by reductive alkylation of the 3-amino groups of vancosamine and epivancosamine. They are reminiscent of the *N*-acyl chain in teicoplanin and may act to preconcentrate the lipoglycopeptides at the outer surface of the cytoplasmic membrane in the vicinity of both the lipid II substrate and the transpeptidase and transglycosylase enzymes that catalyze the last stages of peptidoglycan assembly. Moving the lipophilic chlorobiphenyl group around on the disaccharyl tail of vancomycin-group antibiotics maintained the gain in activity towards VRE. For example, when a 6-aminoglucosyl analogue of the vancomycin disaccharide unit was constructed synthetically from the aglycone and *N*-arylated, this regioisomer had good VRE activity,[136] which suggests it is the lipid anchor rather than a specific structure on the lipodisaccharide that produces activity.

No enhanced affinity was observed when LY333328 and analogues are measured in binding assays against *N*-acyl-*d*-Ala-*d*-Lac, which suggests that the regain of activity in VRE may proceed at least in part by a distinct mechanism. This was further supported by retention of VRE activity in chlorobiphenyl analogues in which the binding site for *N*-acyl-*d*-Ala-*d*-Ala was destroyed by removal of *N*-methyl-*d*-Leu1.[136] Kahne and co-workers suggested membrane-localized transglycosylases (Figure 13) may be the targets for these impaired lipoglycopeptides.[136,137] If this proposal turns out to be generally valid, it offers new targets and new structure–activity relationships to reverse clinical VRE phenotypes in life-threatening infections.

### 14.4. Combinatorial Biosynthesis of Glycopeptide Antibiotics

A potential complement to synthetic efforts to modify the natural glycopeptide skeletons to generate new useful properties in glycopeptide antibiotics would be a combinatorial biosynthesis approach. This would build on the information

Angew. Chem. Int. Ed. **2003**, 42, 730–765

Figure 13. Proposed membrane-localized transglycosylases as targets for impaired lipoglycopeptides.

producing actinoplanes may not be very user friendly for such transformations, but *Streptomyces toyocaensis*, the producer of A47934, would be more tractable.[29]

A second stage for reprogramming the biosynthetic pathway would be at the heptapeptide NRPS assembly line. Two kinds of approaches are now feasible. One is to alter the selectivity of any of the adenylation domains by mutation of a small number of side chains that control the selection and activation of amino acids.[138,139] For example, one could reprogram module 1 and/or 3 to select an amino acid other than Leu1 or Asn3. These are the two side chains not involved in cross-linking. Whether one would need to supply a D-amino acid at residue 1 remains to be determined. The VanC-VRE variants elaborate a serine racemase and the encoding *vanT* gene could be introduced into such a mutant producer to encode Ser at position 1 or 3.[140] A variant of this approach would be to inactivate the E domain in modules 2, 3, or 4 by mutation of a highly conserved histidine residue.[94] For example, would an E5 mutant still make the full-length heptapeptide chain, now D-D-L-D-L-L, and would that still be cross-linked and glycosylated and retain antibiotic activity? A second approach is to swap domains or modules to alter identity and placement of any of the seven amino acids. Linne and Marahiel have begun to elaborate rules for both domain and module swaps in the tyrocidine synthetase assembly line, and those could be applied in the Cep, Van, and Teicoplanin NRPS assembly lines.[141] For example, would a replacement of the DHpg-activating module 3 in teicoplanin synthetase or A47934 synthetase by Hpg lead to a full-length teicoplanin analogue?

A third stage of intervention for reprogramming would be in the tailoring enzyme steps after the NRPS assembly line, and includes N-methylation, β-hydroxylation of tyrosine, or the cross-linking of P450 hemeprotein genes.

A fourth stage for reprogramming is to swap or replace enzymes in the TDP-deoxy sugar pathway. It is likely that the oxovancosamine sugar in the balhimycins is such a natural

experiment where the final reductase is defective or absent. Would TDP-daunosamine or TDP-mycarose be made in the glycopeptide producers and utilized by the Gtfs? There is now good precedent in the polyketide macrolide pathways for reprogramming of the deoxy sugar biosynthesis and utilization by the Gtfs to produce final products with altered glycosyl moieties.[142–144] This strategy for changing the glycosyl moieties depends on Gtfs such as GtfA–E being sufficiently promiscuous to use alternate TDP-deoxyhexoses and for GtfB/C or D/E to act in series on the altered substrates and intermediates. We have shown this is possible by using GtfB and GtfC to make epivancomycin, and more intriguingly to utilize the teicoplanin aglycone and introduce the epivancosaminyl-1,2-glucosyl chain,[116] which has never been seen on this scaffold in nature (Scheme 35). In this connection, it may be possible to construct chimeras of such antibiotic-modifying Gtfs if they all have the two-lobed structure of GtfB (Figure 11) where the top lobe recognizes the TDP-sugar and the bottom lobe the aglycone. Combinatorial chimerization of such Gtfs might generate libraries of Gtfs able to mix and match new aglycones with TDP-deoxy and amino sugars for efficient glycosylation.

### 15. Conclusions

Bacterial genomics has now revealed dozens of gene clusters for nonribosomal peptide antibiotics, polyketide, and hybrid peptide/polyketide antibiotics, and revealed the logic used by nature for templated assembly. The strategy can now be deciphered from bioinformatics-based predictions and validated by experiment, even for antibiotics as complex as chloroeremomycin and vancomycin.

The clustering of all or almost all of the genes needed to encode a dedicated metabolic pathway to a secondary metabolite allows coordinate regulation of gene transcription, protein production, and construction of small molecules to

Biosynthesis of Antibiotics

cascade of covalently tethered aminoacyl- and peptidyl-S-
enzyme intermediates.

The third phase, post assembly line, is the tailoring of the
released heptapeptide. Oxidations and glycosylations are
typical and N-methylations are not uncommon. All three
types of transformations occur here, through seven dedicated
enzymes. The most striking architectural and chemical feature
of the vancomycin/teicoplanin classes of antibiotics are the
side-chain phenolic cross-links. The inventory of cross-linking
heme protein oxidases is now apparent, although the timing,
specificity, and mechanisms remain to be worked out.

The rules by which nature assembles the glycopeptide
(vancomycin) and lipoglycopeptide (teicoplanin) antibiotics
are becoming elucidated and verified. This situation opens up
approaches for reprogramming strategies, at the level of
altered monomers, swapped assembly lines, and different
batteries of post assembly tailoring enzymes.

Received: July 16, 2001 [A486]

[1] E. M. Scholar, W. B. Pratt, *The Antimicrobial Drugs*, 2nd ed., Oxford University Press, New York, 2000, 7–10.
[2] *Antimicrobial Chemotherapy*, 4th ed., (Ed.: H. Greenwood), Oxford University Press, New York, 2000, 223–387.
[3] H. S. Gold, R. C. Moellering, Jr., *N. Engl. J. Med.* 1996, 335, 1445–1453.
[4] A. J. Salter, *Rev. Infect. Dis.* 1982, 4, 196–236.
[5] E. M. Scholar, W. B. Pratt, *The Antimicrobial Drugs*, 2nd ed., Oxford University Press, New York, 2000, chapt. 7.
[6] P. Kloss, L. Xiong, D. L. Shinabarger, A. S. Mankin, *J. Mol. Biol.* 1999, 294, 93–101.
[7] H. C. Neu, *Science* 1992, 257, 1064–1073.
[8] C. Walsh, *Nature* 2000, 406, 775–781.
[9] N. Ban, P. Nissen, J. Hansen, P. B. Moore, T. A. Steitz, *Science* 2000, 289, 905–920.
[10] D. E. Brodersen, W. M. Clemons, Jr., A. P. Carter, R. J. Morgan-Warren, B. T. Wimberly, V. Ramakrishnan, *Cell* 2000, 103, 1143–1154.
[11] P. Nissen, J. Hansen, N. Ban, P. B. Moore, T. A. Steitz, *Science* 2000, 289, 920–930.
[12] A. Maxwell, *Trends Microbiol.* 1997, 5, 102–109.
[13] L. Ferrero, B. Cameron, J. Crouzet, *Antimicrob. Agents Chemother.* 1995, 39, 1554–1558.
[14] B. G. Spratt, *Science* 1994, 264, 388–393.
[15] B. G. Spratt, K. D. Cromie, *Rev. Infect. Dis.* 1988, 10, 699–711.
[16] J. M. Ghuyesen, *Annu. Rev. Microbiol.* 1991, 45, 37–67.
[17] J. R. Knox, P. C. Moews, J. M. Frere, *Chem. Biol.* 1996, 3, 937–947.
[18] D. H. Williams, *Nat. Prod. Rep.* 1996, 13, 469–477.
[19] D. H. Williams, B. Bardsley, *Angew. Chem.* 1999, 111, 1264–1286; *Angew. Chem. Int. Ed.* 1999, 38, 1172–1193.
[20] D. H. Williams, M. P. Williamson, D. W. Butcher, S. J. Hammond, *J. Am. Chem. Soc.* 1983, 105, 1332–1339.
[21] J. C. Barna, D. H. Williams, *Annu. Rev. Microbiol.* 1984, 38, 339–357.
[22] *Bacterial Cell Wall* (Eds.: J.-M. Ghuyesen, R. Hakenbeck), Elsevier, Amsterdam, 1994.
[23] K. K. Wong, D. L. Pompliano, *Adv. Exp. Med. Biol.* 1998, 456, 197–217.
[24] *Glycopeptide Antibiotics* (Ed.: R. Nagarajan), Marcel Dekker, New York, 1994.
[25] K. C. Nicolaou, C. N. C. Boddy, S. Brase, N. Winssinger, *Angew Chem.* 1999, 111, 2230–2287; *Angew Chem. Int. Ed.* 1999, 38, 2096–2152.
[26] J. Davies, *Science* 1994, 264, 375–382.

Scheme 35. Novel functions of GtfB and GtfC to add UDP-glucose and
UDP-L-epivancosamine to the teicoplanin aglycone.

provide the dedicated building blocks. The 29 genes in the
chloroeremomycin cluster segregate into three functional
phases, typical for these classes of bioactive natural products.

The first phase, pre-assembly line, is enzymatic creation of
a pool of dedicated small-molecule monomers, in a “just in
time” inventory control, to serve as substrates for the
assembly line process. In the vancomycin class of glycopep-
tide antibiotics this means creation of three nonproteinogenic
amino acids, of the five amino acids in the heptapeptide
scaffold. Some 12 of the 29 genes are assignable to production
of these three amino acids. The other dedicated monomer is
the activated L-amino deoxy sugar nucleotide TDP-L-β-
epivancosamine, and five more genes generate this specific
substrate.

Phase II is the assembly line process. The NRPS assembly
line for these glycoheptapeptide antibiotics has 24 domains
distributed over three protein subunits. The identity and
placement of each domain controls monomer selection and
activation in chain elongation, chain elongation involving four
of the seven D residues, and then hydrolytic chain termina-
tion. Growth of the peptide chain is directional, from the N-
to the C-termini of the assembly line, by an elongation

[27] I. T. Paulsen, M. H. Brown, R. A. Skurray, *Microbiol. Rev.* 1996, **60**, 575–608.

[28] M. Putman, H. W. van Veen, W. N. Konings, *Microbiol. Mol. Biol. Rev.* 2000, **64**, 672–693.

[29] C. G. Marshall, I. A. Lessard, I. Park, G. D. Wright, *Antimicrob. Agents Chemother.* 1998, **42**, 2215–2220.

[30] M. Sosio, A. Bianchi, E. Bossi, S. Donadio, *Antonie Van Leeuwenhoek* 2000, **78**, 379–384.

[31] C. T. Walsh, S. L. Fisher, I. S. Park, M. Prahalad, Z. Wu, *Chem. Biol.* 1996, **3**, 21–28.

[32] M. N. Swartz, *Proc. Natl. Acad. Sci. USA* 1994, **91**, 2420–2427.

[33] B. E. Murray, *N. Engl. J. Med.* 2000, **342**, 710–721.

[34] D. Hoeffler, U. Zimmermann, *Lancet* 1997, **350**, 739.

[35] M. Arthur, P. Courvalin, *Antimicrob. Agents Chemother.* 1993, **37**, 1563–1571.

[36] S. Dutka-Malen, C. Molinas, M. Arthur, P. Courvalin, *Gene* 1992, **112**, 53–58.

[37] M. Fines, B. Perichon, P. Reynolds, D. F. Sahm, P. Courvalin, *Antimicrob. Agents Chemother.* 1999, **43**, 2161–2164.

[38] V. L. Healy, I. A. Lessard, D. I. Roper, J. R. Knox, C. T. Walsh, *Chem. Biol.* 2000, **7**, R109–R119.

[39] T. D. Bugg, G. D. Wright, S. Dutka-Malen, M. Arthur, P. Courvalin, *Biochemistry* 1991, **30**, 10408–10415.

[40] M. Arthur, C. Molinas, P. Courvalin, *J. Bacteriol.* 1992, **174**, 2582–2591.

[41] T. R. Holman, Z. Wu, B. L. Wanner, C. T. Walsh, *Biochemistry* 1994, **33**, 4625–4631.

[42] A. Haldimann, S. L. Fisher, L. L. Daniels, C. T. Walsh, B. L. Wanner, *J. Bacteriol.* 1997, **179**, 5903–5913.

[43] M. Arthur, F. Depardieu, P. Reynolds, P. Courvalin, *Mol. Microbiol.* 1996, **21**, 33–44.

[44] M. Baptista, F. Depardieu, P. Courvalin, M. Arthur, *Antimi- crob. Agents Chemother.* 1996, **40**, 2291–2295.

[45] I. S. Park, C. H. Lin, C. T. Walsh, *Proc. Natl. Acad. Sci. USA* 1997, **94**, 10040–10044.

[46] F. Navarro, P. Courvalin, *Antimicrob. Agents Chemother.* 1994, **38**, 1788–1793.

[47] N. Mani, P. Sancheti, Z. D. Jiang, C. McNaney, M. DeCenzo, B. Knight, M. Stankis, M. Kuranda, D. M. Rothstein, P. Sanchet, B. Knighti, *J. Antibiot.* 1998, **51**, 471–479.

[48] D. A. Evans, C. J. Dinsmore, A. M. Ratz, D. A. Evrard, J. C. Barrow, *J. Am. Chem. Soc.* 1997, **119**, 3417–3418.

[49] D. A. Evans, C. J. Dinsmore, A. M. Ratz, *Tetrahedron Lett.* 1997, **38**, 3189–3192.

[50] D. A. Evans, J. C. Barrow, P. S. Watson, A. M. Ratz, C. J. Dinsmore, D. A. Evrard, K. M. DeVries, *J. Am. Chem. Soc.* 1997, **119**, 3419–3420.

[51] D. A. Evans, M. R. Wood, B. W. Trotter, T. I. Richardson, J. C. Barrow, J. L. Katz, *Angew. Chem.* 1998, **110**, 2868–2872; *Angew. Chem. Int. Ed.* 1998, **37**, 2704–2708.

[52] K. C. Nicolaou, S. Natarajan, H. Li, N. F. Jain, R. Hughes, M. E. Solomon, J. M. Ramanjulu, C. N. C. Boddy, M. Takayanagi, *Angew. Chem.* 1998, **110**, 2872–2878; *Angew. Chem. Int. Ed.* 1998, **37**, 2708–2714.

[53] K. C. Nicolaou, N. F. Jain, S. Natarajan, R. Hughes, M. E. Solomon, H. Li, J. M. Ramanjulu, M. Takayanagi, A. E. Koumbis, T. Bando, *Angew. Chem.* 1998, **110**, 2879–2881; *Angew. Chem. Int. Ed.* 1998, **37**, 2714–2717.

[54] K. C. Nicolaou, M. Takayanagi, N. F. Jain, S. Natarajan, A. E. Koumbis, T. Bando, J. M. Ramanjulu, *Angew. Chem.* 1998, **110**, 2881–2883; *Angew. Chem. Int. Ed.* 1998, **37**, 2717–2719.

[55] K. C. Nicolaou, H. J. Mitchell, N. F. Jain, N. Winssinger, R. Hughes, T. Bando, *Angew. Chem.* 1999, **111**, 253–255; *Angew. Chem. Int. Ed.* 1999, **38**, 240–244.

[56] K. C. Nicolaou, H. J. Mitchell, N. F. Jain, T. Bando, R. Hughes, N. Winssinger, S. Natarajan, A. E. Koumbis, *Chem. Eur. J.* 1999, **5**, 2648–2667.

[57] D. L. Boger, S. Miyazaki, S. H. Kim, J. H. Wu, O. Loiseleur, S. L. Castle, *J. Am. Chem. Soc.* 1999, **121**, 3226–3227.

[58] D. Bischoff, S. Pelzer, A. Holtzel, G. J. Nicholson, S. Stockert, W. Wohlleben, G. Jung, R. D. Süßmuth, *Angew. Chem.* 2001, **113**, 1736–1739; *Angew. Chem. Int. Ed.* 2001, **40**, 1693–1696.

[59] D. Bischoff, S. Pelzer, B. Bister, G. J. Nicholson, S. Stockert, W. Wohlleben, G. Jung, R. D. Süßmuth, *Angew. Chem.* 2001, **113**, 4824–4827; *Angew. Chem. Int. Ed.* 2001, **40**, 4688–4691.

[60] A. M. van Wageningen, P. N. Kirkpatrick, D. H. Williams, B. R. Harris, J. K. Kershaw, N. J. Lennard, M. Jones, S. J. Jones, P. J. Solenberg, *Chem. Biol.* 1998, **5**, 155–162.

[61] H.-T. Chiu, B. K. Hubbard, A. N. Shah, J. Eide, R. A. Freden- burg, C. T. Walsh, C. Khosla, *Proc. Natl. Acad. Sci. USA* 2001, **98**, 8548–8553.

[62] S. Pelzer, R. Süßmuth, D. Heckmann, J. Recktenwald, P. Huber, G. Jung, W. Wohlleben, *Antimicrob. Agents Chemother.* 1999, **43**, 1565–1573.

[63] D. P. O'Brien, P. N. Kirkpatrick, S. W. O'Brien, T. Staroske, T. I. Richardson, D. A. Evans, A. Hopkinson, J. B. Spencer, D. H. Williams, *Chem. Commun.* 2000, 103–104.

[64] H. Seto, T. Fujioka, K. Furihata, I. Kaneko, S. Takahashi, *Tetrahedron Lett.* 1989, **30**, 4987–4990.

[65] K. Matsuzaki, H. Ikeda, T. Ogino, A. Matsumoto, H. B. Woodruff, H. Tanaka, S. Omura, *J. Antibiot.* 1994, **47**, 1173–1174.

[66] K. Matsuzaki, T. Ogino, T. Sunazuka, H. Tanaka, S. Omura, *J. Antibiot.* 1997, **50**, 66–69.

[67] K. Tachikawa, K. Hasumi, A. Endo, *Thromb. Haemostasis* 1997, **77**, 137–142.

[68] C. M. Harris, T. M. Harris, *J. Am. Chem. Soc.* 1982, **104**, 4293–4295.

[69] S. J. Hammond, M. P. Williamson, D. H. Williams, L. D. Boeck, G. G. Marconi, *J. Chem. Soc. Chem. Commun.* 1982, 344–346.

[70] S. J. Hammond, D. H. Williams, R. V. Nielsen, *J. Chem. Soc. Chem. Commun.* 1983, 116–117.

[71] M. J. Zmijewski, Jr., B. Briggs, R. Logan, L. D. Boeck, Anti- microb. Agents Chemother.* 1987, **31**, 1497–1501.

[72] S. K. Chung, P. Taylor, Y. K. Oh, C. DeBrosse, P. W. Jeffs, *J. Antibiot.* 1986, **39**, 642–651.

[73] T. I. Nicas, R. D. G. Cooper in *Biotechnology of Antibiotics* (Ed.: W. R. Strohl), Marcel Dekker, New York, 1997, pp. 363–392.

[74] B. K. Hubbard, M. G. Thomas, C. T. Walsh, *Chem. Biol.* 2000, **7**, 931–942.

[75] O. W. Chorba, D. H. Williams, J. B. Spencer, *J. Am. Chem. Soc.* 2000, **122**, 5389–5390.

[76] W. J. McGahren, J. H. Martin, G. O. Morton, R. T. Hargreaves, R. A. Leese, F. M. Lovell, G. A. Ellestad, E. O'Brien, J. S. E. Holker, *J. Am. Chem. Soc.* 1980, **102**, 1671–1684.

[77] H. M. Holden, M. M. Benning, T. Haller, J. A. Gerlt, *Acc. Chem. Res.* 2001, **34**, 145–157.

[78] V. Pfeifer, G. J. Nicholson, J. Ries, J. Recktenwald, A. B. Schäfer, R. M. Shawky, J. Schröder, W. Wohlleben, S. Pelzer, *J. Biol. Chem.* 2001, **276**, 38370–38377.

[79] H. Chen, C. C. Tseng, B. K. Hubbard, C. T. Walsh, *Proc. Natl. Acad. Sci USA*, 2001, **98**, 14901–14906.

[80] M. Steffensky, A. Muhlenweg, Z. X. Wang, S. M. Li, L. Heide, *Antimicrob. Agents Chemother.* 2000, **44**, 1214–1222.

[81] H. Chen, C. T. Walsh, *Chem. Biol.* 2001, **8**, 301–312.

[82] B. Lauer, R. Russwurm, C. Bormann, *Eur. J. Biochem.* 2000, **267**, 1698–1706; B. Lauer, R. Russwurm, W. Schwarz, A. Kalmanczhelyi, C. Bruntner, A. Rosemeier, C. Bormann, *Mol. Gen. Genet.* 2001, **264**, 662–673.

[83] H.-W. Chen, B. K. Hubbard, S. O. O'Connor, C. T. Walsh, unpublished results.

[84] S. Kirner, S. Krauss, G. Sury, S. T. Lam, J. M. Ligon, K. H. van Pee, *Microbiology* 1996, **142**, 2129–2135.

[85] P. E. Hammer, D. S. Hill, S. T. Lam, K.-H. van Pee, J. M. Ligon, Appl. Environ. Microbiol. 1997, 63, 2147–2154.

[86] K. Hohaus, A. Altmann, W. Burd, I. Fischer, P. E. Hammer, D. S. Hill, J. M. Ligon, K.-H. van Pee., Angew. Chem. 1997, 109, 2102–2104; Angew. Chem. Int. Ed. Engl. 1997, 36, 2012–2013.

[87] T. Dairi, T. Nakano, K. Aisaka, R. Katsumata, M. Hasegawa, Biosci. Biotechnol. Biochem. 1995, 59, 1099–1106.

[88] S. Keller, T. Wage, K. Hohaus, M. Holzer, E. Eichhorn, K.-H. van Pee, Angew. Chem. 2000, 112, 2380–2382; Angew. Chem. Int. Ed. 2000, 39, 2300–2302.

[89] M. A. Marahiel, T. Stachelhaus, H. D. Mootz, Chem. Rev. 1997, 97, 2651–2673.

[90] H. von Doehren, U. Keller, J. Vater, R. Zocher, Chem. Rev. 1997, 97, 2675–2705.

[91] T. Stein, J. Vater, V. Krüft, A. Otto, B. Wittmann-Liebold, P. Franke, M. Panico, R. McDowell, H. R. Morris, J. Biol. Chem. 1996, 271, 15428–15435.

[92] C. T. Walsh, J. Biol. Chem. 1989, 264, 2393–2396.

[93] K. Hoffmann, E. Schneider-Scherzer, H. Kleinkauf, R. Zocher, J. Biol. Chem. 1994, 269, 12710–12714.

[94] T. Stachelhaus, C. T. Walsh, Biochemistry 2000, 39, 5775–5787.

[95] L. Luo, C. T. Walsh, Biochemistry 2001, 40, 5329–5337.

[96] J. W. Trauger, C. T. Walsh, Proc. Natl. Acad. Sci. USA 2000, 97, 3112–3117.

[97] J. W. Trauger, R. M. Kohli, H. D. Mootz, M. A. Marahiel, C. T. Walsh, Nature 2000, 407, 215–218.

[98] A. Klemer, H. Wilbers, Liebigs Ann. Chem. 1987, 815–823.

[99] G. Fronza, C. Fuganti, P. Grasseli, G. Pedrocchi-Fantoni, J. Carbohydr. Chem. 1983, 2, 5073–5076.

[100] G. Fronza, C. Fuganti, P. Grasseli, G. Pedrocchi-Fantoni, J. Carbohydr. Chem. 1983, 2, 5073–5076.

[101] R. Greven, P. Jutten, H.-D. Scharf, Carbohydr. Res. 1995, 275, 83–93.

[102] J. Thorson, B. Shen, R. Whitman, H.-w. Liu, Y. Li, J. Ahlert, Bioorg. Chem. 1999, 27, 172–188.

[103] L. Du, C. Sanchez, M. Chen, D. J. Edwards, B. Shen, Chem. Biol. 2000, 7, 623–642.

[104] J. P. Mackay, U. Gerhard, D. A. Beauregard, R. A. Maplestone, D. H. Williams, J. Am. Chem. Soc. 1994, 116, 4573–4580.

[105] U. Gerhard, J. P. Mackay, R. A. Maplestone, D. H. Williams, J. Am. Chem. Soc. 1993, 115, 232–237.

[106] L. M. Quiros, I. Aguirrezabalaga, C. Olano, C. Mendez, J. A. Salas, Mol. Microbiol. 1998, 28, 1177–1185.

[107] R. Kornfeld, S. Kornfeld, Annu. Rev. Biochem. 1985, 54, 631–664.

[108] H. W. Liu, J. S. Thorson, Annu. Rev. Microbiol. 1994, 48, 223–256.

[109] A. Kirschning, A. F.-W. Bechtold, J. Rohr, Top. Curr. Chem. 1997, 188, 1–84.

[110] H. Chen, M. G. Thomas, B. K. Hubbard, H. C. Losey, C. T. Walsh, M. D. Burkart, Proc. Natl. Acad. Sci. USA 2000, 97, 11942–11947.

[111] P. N. Kirkpatrick, W. Scaife, T. M. Hallis, H.-w. Liu, J. B. Spencer, D. H. Williams, Chem. Commun. 2000, 1565–1566.

[112] A. D. Hegeman, J. W. Gross, P. A. Frey, Biochemistry 2001, 40, 6598–6610.

[113] S. Gaisser, G. A. Bohm, J. Cortes, P. F. Leadlay, Mol. Gen. Genet. 1997, 256, 239–251.

[114] S. Gaisser, G. A. Bohm, M. Doumith, M. C. Raynal, N. Dhillon, J. Cortes, P. F. Leadlay, Mol. Gen. Genet. 1998, 258, 78–88.

[115] L. Katz, S. Donadio, Genetics and Biochemistry of Antibiotic Production, Butterworth-Heinemann, Boston, 1995, pp. 385–420.

[116] H. C. Losey, M. W. Peczuh, Z. Chen, U. S. Eggert, S. D. Dong, I. Pelczer, D. Kahne, C. T. Walsh, Biochemistry 2001, 40, 4745–4755.

[117] P. J. Solenberg, P. Matsushima, D. R. Stack, S. C. Wilkie, R. C. Thompson, R. H. Baltz, Chem. Biol. 1997, 4, 195–202.

[118] A. M. Mulichak, H. C. Losey, C. T. Walsh, R. M. Garvito, Structure 2001, 9, 547–557.

[119] S. Ha, D. Walker, Y. Shi, S. Walker, Protein Sci. 2000, 9, 1045–1052.

[120] A. Vrielink, W. Ruger, H. P. Driessen, P. S. Freemont, EMBO J. 1994, 13, 3413–3422.

[121] J. E. Jackman, C. R. Raetz, C. A. Fierke, Biochemistry 1999, 38, 1902–1911.

[122] Y. Volchegursky, Z. Hu, L. Katz, R. McDaniel, Mol. Microbiol. 2000, 37, 752–762.

[123] C. R. Hutchinson, Chem. Rev. 1997, 97, 2525–2535.

[124] D. I. Roper, T. Huyton, A. Vagin, G. Dodson, Proc. Natl. Acad. Sci. USA 2000, 97, 8921–8925.

[125] D. E. Bussiere, S. D. Pratt, L. Katz, J. M. Severin, T. Holzman, C. H. Park, Mol. Cell 1998, 2, 75–84.

[126] R. Araoz, E. Anhalt, L. Rene, M. A. Badet-Denisot, P. Courvalin, B. Badet, Biochemistry 2000, 39, 15971–15979.

[127] J. F. Barrett, J. A. Hoch, Antimicrob. Agents Chemother. 1998, 42, 1529–1536.

[128] K. Stephenson, Y. Yamaguchi, J. A. Hoch, J. Biol. Chem. 2000, 275, 38900–38904.

[129] G. McMahon, L. Sun, C. Liang, C. Tang, Curr. Opin. Drug Discov. Dev. 1998, 1, 131–146.

[130] N. Bamas-Jacques, S. Lorenzon, F. Massey, G. deSwetschin, I. DeCourty, G. Sezonov, J. L. Permodet, A. Friedmann, M. Vuilhorgne, M. Couder, D. Thabaut, J. F. Desnotes, Xth International Symposium on Biology of Actinomycetes. Abstract 2P11., Beijing, 1997.

[131] L. Du, C. Sanchez, B. Shen, Metab. Eng. 2001, 3, 78–95.

[132] C. Cocitto, Microbiol. Rev. 1979, 43, 145–198.

[133] P. Kloss, L. Xiong, D. L. Shinabarger, A. S. Mankin, J. Mol. Biol. 1999, 294, 93–101.

[134] F. Biavasco, C. Vignaroli, R. Lupidi, E. Manso, B. Facinelli, P. E. Varaldo, Antimicrob. Agents Chemother. 1997, 41, 2165–2172.

[135] E. Hershberger, J. R. Aeschlimann, T. Moldovan, M. J. Rybak, Antimicrob. Agents Chemother. 1999, 43, 717–721.

[136] R. Kerns, S. D. Dong, S. Fukuzawa, J. Carbeck, J. Kohler, L. Silver, D. Kahne, J. Am. Chem. Soc. 2000, 122, 12608–12609.

[137] R. C. Goldman, E. R. Baizman, C. B. Longley, A. A. Branstrom, FEMS Microbiol. Lett. 2000, 183, 209–214.

[138] T. Stachelhaus, H. D. Mootz, M. A. Marahiel, Chem. Biol. 1999, 6, 493–505.

[139] G. L. Challis, J. Ravel, C. A. Townsend, Chem. Biol. 2000, 7, 211–224.

[140] C. A. Arias, M. Martin-Martinez, T. L. Blundell, M. Arthur, P. Courvalin, P. E. Reynolds, Mol. Microbiol. 1999, 31, 1653–1664.

[141] U. Linne, M. A. Marahiel, Biochemistry 2000, 39, 10439–10447.

[142] S. Gaisser, J. Reather, G. Wirtz, L. Kellenberger, J. Staunton, P. F. Leadlay, Mol. Microbiol. 2000, 36, 391–401.

[143] H. Yamase, L. Zhao, H.-w. Liu, J. Am. Chem. Soc. 2000, 122, 12397–12398.

[144] L. Zhao, J. Ahlant, Y. Zue, J. S. Thorson, D. H. Sherman, H.-w. Liu, J. Am. Chem. Soc. 1999, 121, 9881–9882.

[145] J. Pootoolal, M. G. Thomas, C. G. Marshall, J. M. Neu, B. K. Hubbard, C. T. Walsh, G. D. Wright, Proc. Natl. Acad. Sci. 2002, 99, 8962–8967.
