
VOL. 55 NO. 12, DEC. 2002 THE JOURNAL OF ANTIBIOTICS pp. 1063 — 1071

**Cloning of the Staurosporine Biosynthetic Gene Cluster from Streptomyces sp. TP-A0274 and Its Heterologous Expression in Streptomyces lividans**

HIROYASU ONAKA*, SHIN-ICHI TANIGUCHI, YASUHIRO IGARASHI and TAMOTSU FURUMAI

Biotechnology Research Center, Toyama Prefectural University,  
Kosugi, Toyama 939-0398, Japan  

(Received for publication July 15, 2002)

Staurosporine is a representative member of indolocarbazole antibiotics. The entire staurosporine biosynthetic and regulatory gene cluster spanning 20-kb was cloned from *Streptomyces* sp. TP-A0274 and sequenced. The gene cluster consists of 14 ORFs and the amino acid sequence homology search revealed that it contains three genes, *staO*, *staD*, and *staP*, coding for the enzymes involved in the indolocarbazole aglycone biosynthesis, two genes, *staG* and *staN*, for the bond formation between the aglycone and deoxysugar, eight genes, *staA*, *staB*, *staE*, *staJ*, *staI*, *staK*, *staMA*, and *staMB*, for the deoxysugar biosynthesis and one gene, *staR* is a transcriptional regulator. Heterologous gene expression of a 38-kb fragment containing a complete set of the biosynthetic genes for staurosporine cloned into pTOYAMAcos confirmed its role in staurosporine biosynthesis. Moreover, the distribution of the gene for chromopyrrolic acid synthase, the key enzyme for the biosynthesis of indolocarbazole aglycone, in actinomycetes was investigated, and *rebD* homologs were shown to exist only in the strains producing indolocarbazole antibiotics.

---

Streptomyces produce a wide variety of biologically active secondary metabolites including antibiotics. Staurosporine (Fig. 1) is the most well-known member of the indolocarbazole alkaloid and acts as a strong protein kinase inhibitor¹. Most indolocarbazole compounds are inhibitors for protein kinases or DNA topoisomerases, and thus expected to be useful as antitumor drugs. A series of other indolocarbazole compounds like rebeccamycin², K-252a³ and UCN-01⁴ were isolated from actinomycetes. We are interested in isolating the biosynthetic genes for indolocarbazoles to study their organization and regulation, and to construct recombinant streptomycetes to produce novel indolocarbazole antibiotics.

The biosynthetic gene cluster of rebeccamycin from *Lechevalieria aerocolonigenes* ATCC 39243 was recently cloned and characterized by two groups⁵,⁶. Ten ORFs were identified in the cluster over 17-kb⁵ and eight of them were characterized on the basis of the gene disruption experiments and the structural analysis of the products from the disrupted mutants⁶. According to our result, rebeccamycin is biosynthesized through 11,11′-dichlorochromopyrrolic acid (CCA) [3,4-di-(3-(8-chloroindoly))-pyrrole-2,5-dicarboxylic acid] from two molecules of tryptophan. In the biosynthesis of other indolocarbazoles, chromopyrrolic acid is presumed to be an intermediate instead of CCA. CGA might be directly biosynthesized by *rebD* which encodes a putative chromopyrrolic acid synthase. The deduced amino acid sequence of *rebD* has no significant similarity to the GenBank entries except for that of *vioB* which encodes a putative multifunctional enzyme that catalyzes tryptophan decarboxylation and condensation reaction in violacein biosynthesis. Although the basic biosynthetic pathway for the indolocarbazole aglycone was revealed in the analysis of the rebeccamycin biosynthesis, modified pathways for the individual indolocarbazole compounds remain unidentified. In case of rebeccamycin and staurosporine biosynthesis, the oxidation state at the C-7 position of aglycone and the pattern of connectivity between the deoxysugar and aglycone are different. In this study, we cloned and characterized the staurosporine biosynthetic gene cluster from *Streptomyces* sp. TP-A0274.

* Corresponding author: onaka@pu-toyama.ac.jp

Fig. 1. Chemical structure of staurosporine, rebeccamycin, and K-252a.

Staurosporine  
from *Streptomyces sp.* TP-A0274  

Rebeccamycin  
from *Lechevalieria aerocolonigenes* ATCC 39243  

K252a  
from *Nocardiopsis sp.* NRRL 15532  

Table 1. Bacterial strains in this study.

| designation                          | relevant characteristics               | source of reference |
|--------------------------------------|----------------------------------------|---------------------|
| *Escherichia coli* DH5α              | for gene cloning                      | 7                   |
| *Escherichia coli* XL-1 Blue MR      | for cosmid library construction        | 7                   |
| *Escherichia coli* S17-1             | conjugational host                    | 9                   |
| *Streptomyces sp.* TP-A0274*         | staurosporine producer                | this study          |
| *Streptomyces lividans* TK23         | heterologous host                     | 11                  |
| *Lechevalieria aerocolonigenes* ATCC 39243* | rebeccamycin producer            | ATCC                |
| *Nocardiopsis sp.* NRRL 15532*       | K-252a producer                       | NRRL                |
| *Streptomyces coelicolor* M130       | actinorhodin producer                 | 11                  |
| *Streptomyces griseus* IFO 13350     | streptomycin producer                 | IFO                 |
| *Streptomyces avermitilis* IFO 14893 | avermectin producer                   | IFO                 |
| *Streptomyces acidiscabies* JCM 7913 | the cause of potato scab              | JCM                 |
| *Streptomyces scabies* JCM 7914      | the cause of potato scab              | JCM                 |
| *Streptomyces spinicoumarensis* IFO 13857 | coumermycin producer           | IFO                 |
| *Streptomyces sp.* TP-A0584          | goadsporine producer                  | 8                   |

*: indolocarbazole-producing strains

Furthermore, we demonstrated that *rebD* homologs exist only in the indolocarbazole-producing strains and *rebD* gene can be used as a probe to screen such strains.

Materials and Methods

Bacterial Strains, Plasmids, and Growth Condition

Bacterial strains used in this study are summarized in Table 1. ATCC strain 39243 was obtained from American Type Culture Collection, Rockville, MD. IFO strains were obtained from Institute of Fermentation, Osaka, Japan. NRRL strain 15532 was obtained from Northern Utilization Research and Development division, U.S. Department of Agriculture, Peoria, Illinois. *Streptomyces sp.* TP-A0274 was used as a source of DNA in the construction of the genomic libraries. *E. coli* DH-5α served as the host for plasmid subcloning in pUC19 and the derivatives. *E. coli* XL1-Blue MR was used for the construction of pTOYAMAcos cosmid library. *E. coli* S17-1 was used for transconjugation from *E. coli* to *S. lividans*. Growth conditions and manipulations of *E. coli* were as described

VOL. 55 NO. 12                                                                 THE JOURNAL OF ANTIBIOTICS                                                                 1065

by SAMBROOK and RUSSELL<sup>7</sup>. The production medium for S. lividans and strain TP-A0274 was Medium G134<sup>6</sup>. The seed culture for S. lividans and strain TP-A0274 was V-22 medium<sup>8</sup>. The Bennett's glucose agar<sup>8</sup>, nutrient broth agar and mannitol soya flour agar were used for conjugational transformation<sup>9</sup>. The pTOYAMAcos is a newly constructed E. coli-actinomycetes shuttle cosmid vector which is based on the pKU402 cosmid vector<sup>10</sup>.

### General Recombinant DNA Techniques
Restriction enzymes, T4 DNA ligase, and Taq polymerase were purchased from New England Biolabs. DNA fragments were labelled with [α-<sup>32</sup>P]-dCTP (6000 Ci mmol<sup>-1</sup>; Amersham Pharmacia Biotech) and the BcaBEST labelling kit (Takara Shuzo Co., Ltd.). PCR was carried out with PTC-200 DNA Engine (MJ Research, MA, USA). DNA manipulations in E. coli were as described by SAMBROOK and RUSSELL<sup>7</sup>, and those in Streptomyces were as described by KIESER et al.<sup>11</sup>.

#### DNA Sequencing of 16S rDNA
The 16S rDNA fragment was amplified from strain TP-A0274 genomic DNA by using the primers 16S-1 and 16S-2. These PCR primers were prepared on the basis of the consensus sequences of eubacterial 16S rDNA, and the sequences are 16S-1: 5′-GAGAAGCTTAGAGTTTGATCCTGGCTCAG-3′, and 16S-2: 5′-GAGGAATTCCACGGCTACCTTGTTCGACT-3′, in which EcoRI or HindIII sites (underline indicated) were created for facilitation of the cloning. PCR was done with PTC-200 DNA Engine, and the products were directly used as sequencing templates. Automatic DNA sequencing was carried out with BigDye terminator cycle sequencing ready reaction kit (Applied Biosystems) and primers (16S-1, 16S-2, 16S-4: 5′-GCCTACGAGCTCTTTACGCC-3′, 515F: 5′-GTGCCAGCAGCCGCGGT-3′, 785F: 5′-GGATTAGATACCCTGGTAGTC-3′, 1099F: 5′-GCAACGAGCGCAACCC-3′ and 1115R: 5′-AGGGTTGCGCTCGTTG-3′), and analyzed on an ABI PRISM 310 DNA sequencer (Applied Biosystems).

#### Nucleotide Sequence Data Accession Number
The nucleotide sequences determined in the present work are deposited in the DDBJ/EMBL/GenBank database under accession no. AB088119 and AB088069.

### Results

#### The Distribution of the rebD Homolog in Actinomycetes Strains
We initially investigated the distribution of the rebD homolog gene in actinomycetes strains by the PCR method. Chromopyrrolic acid synthase is the key enzyme in the indolocarbazole aglycone synthesis. Chromopyrrolic acid-related compounds have been isolated from rebeccamycin and violacein-producing strains but there is no report on the isolation of chromopyrrolic acid derivatives from other actinomycetes. Therefore the existence of chromopyrrolic acid synthase is expected to be a useful indicator of indolocarbazole-producing strains. Deduced amino acid sequences of the genes encoding chromopyrrolic acid synthase in the biosynthesis of rebeccamycin (rebD) and violacein (vioB) have 36.6% identity over the entire sequences. The two PCR primers, rebBN1 and rebBC1, were synthesized using the highly conserved regions corresponding to amino acids from 611 to 618 and from 780 to 786 in RebD amino acid sequence. We tested indolocarbazole-producing strains, Streptomyces sp. TP-A0274, Nocardopsis sp. NRRL 15532 and L. aerocolonigenes ATCC 39243 and eight strains which do

#### The rebD Homologues Distribution in Actinomycetes Strains
The DNA fragments were amplified by using the primers rebBN1 and rebBC1. The sequences of primers are rebBN1: 5′-GAAGAATTCGTSATGCTSCAGTACCTSTA-3′, and rebBC1: 5′-CGAAAGCTTSAGGAASAGGTGGTGCTCSCC-3′, in which EcoRI or HindIII sites (underline indicated) were created for facilitation of cloning. PCR was carried out at 98°C for 20 seconds, 60°C for 30 seconds, 72°C for 1 minute in a total of 30 cycles.

#### Analysis of Metabolites
S. lividans harboring pTYMCsta or strain TP-A0274 were inoculated into a 500-ml K-1 flask containing 100 ml of V-22 medium. After incubation at 30°C for 2-days on a rotary shaker at 200 rpm, 5-ml aliquots of the seed culture were transferred into a 500-ml K-1 flask containing 100 ml of Medium G134. The fermentation was carried out at 30°C for 7-days on the same rotary shaker. The culture broth was extracted with 100 ml of *n*-butanol. After the evaporation of *n*-butanol, the resultant was dissolved in DMSO for HPLC analysis. LC-MS spectra were obtained on an API 165 (Applied Biosystems). HPLC analysis was performed on a HP1090 system with a diode array detector (Hewlett Packard) using a C18 Rainin microsorb column (3 μm, 100×4.6 mm, i.d.; Rainin Instrument Co., MA, USA). Acetonitrile-0.15% KH<sub>2</sub>PO<sub>4</sub> (pH 3.5) was served as the elution buffer (The gradient diagram is shown in Fig. 4A). Temperature was 40°C, flow rate was 1.2 ml/minute; and detection was at 254 nm.

not produce indolocarbazoles. The strain TP-A0274 was isolated by us as a staurosporine-producing strain from a soil sample collected in Kosugi-machi, Toyama, Japan. Strain TP-A0274 was identified on the basis of its 16S rDNA sequence. The GenBank database showed that the 16S rDNA sequence of the strain TP-A0274 is identical with that of Streptomyces padanus ATCC 25646 (accession No. AF455813). The result indicated that the strain belongs to the genus Streptomyces. As shown in Fig. 2A, a 0.5-kb fragment was amplified only when chromosomal DNAs from indolocarbazole producers were used as templates (Fig. 2A, lane 1, 2 and 3). On the other hand, any DNA fragments with the size of about 0.5-kb were not amplified with chromosomal DNAs from indolocarbazole non-producers. In the latter case, however, DNA fragments with different sizes were amplified (Fig. 2A, lane 6 and 10). These fragments were cloned and their nucleotide sequences were analyzed to examine whether these DNA fragment carried a rebD homolog. No rebD-related genes

were found on these fragments, suggesting that the rebD homolog exists only in the indolocarbazole-producing strains and that genes on the amplified fragments are the rebD homolog of the staurosporine and K-252a biosynthetic genes. In fact, DNA sequencing of the region covering the 0.5-kb fragment amplified with the chromosomal DNAs of the strain TP-A0274 and the strain NRRL 15532 showed that they encoded proteins homologous with the RebD (Fig. 2B). Since the rebD gene homologs are distributed only in the indolocarbazole producers, the rebD gene is a useful marker of the indolocarbazole-producing strains.

Cloning of a rebD Homolog from Streptomyces sp. TP-A0274

The cloning procedure of staurosporine biosynthetic gene cluster was described in our previous paper<sup>6</sup>. To clone a complete staurosporine biosynthetic gene cluster, a

Fig. 2. Amplification of DNAs by PCR with the primers designed from rebD and chromosomal DNAs of various actinomycetes strains (A).

A

![image](https://i.imgur.com/your_image_url.png)

B


rebD AFSIPTHGAGQELVSRGDWTEPQLRLMCGDGGETTDGGVRGSLLGVAREEMIHFLVVNNVMAVGEPFHVPDLDFGTINDTLMVPLDFSLEA  
staD AYSVPTHGAGAEYVRRGLWTPDQLRLACGDGGETLDEGIRSMLLTIAREEMIHFLLVNNILMAVGEPFHAPRIDFGTVNQRLAVPLDFALER  
K252 GYTVPDHGAGGREYVRRGDWTEPQLALACGDGGETLAGGIRGTLIRVAREEMIHFLLANNILMAIGEPFHLPHLDFGTLNAELPIALDFSLEA  
.....* ***** * ** ***** * ********** *.*.*****.*.*********** *.*.****.****** * .******.***

rebD LGLGSVQRFIQIEQPEGLTGAVRLG-----------------DLPVPVREAEDFH-YASLSELYGDIREGLQRVPGLFLVERGRGG  
staD LGPGSVERFVQIERPEDLVDEVRRG-----------------DAPAPPAAYDERHPYASLSELYADIREGLESIPGLFLVDKGRRGG  
K252 LDVASVARFVRIEEPDGLVTVPVRLGTRTSTGTSTSADADADADGDGGEPGYASISELYGDIRRALTTIPGLFLTEPGRDA  
*. .*** ***..*** *.** ** *  
*********.**** .*.*.******.******

The used chromosomal DNAs were prepared from L. aerocolonigenes ATCC 39243 (control; lane 1), Streptomyces sp. TP-A0274 (lane 2), Nocardopsis sp. NRRL 15532 (lane 3), S. coelicolor A3 (2) M130 (lane 4), S. lividans TK23 (lane 5), S. griseus IFO 13350 (lane 6), S. avermitilis IFO 14893 (lane 7), S. acidiscabies JCM 7913 (lane 8), S. scabiei JCM 7914 (lane 9), S. spinicoumarensis MCRL 1136 (lane 10), Streptomyces sp. TP-A0584 (lane 11). Sequence alignment of three amplified fragments from L. aerocolonigenes (rebD), Streptomyces sp. TP-A0274 (staD) and Nocardopsis sp. NRRL 15532 (K252) (B). The lower line reports the level of homology: asterisk, conserved residue; full stop, consistent homology.

cosmid library of genomic DNA of strain TP-A0274, which was constructed with Sau3AI partially digested genomic DNA and a cosmid, pTOYAMAcos, was screened by colony hybridization with the 0.4-kb rebD homolog fragment as a probe. Several positive clones were obtained and one of them was designated to pTYMCsta. pTOYAMAcos has int gene derived from φC31 actinophage, and integrated in actinomycetes chromosomal DNA after transconjugation.

Heterologous Production of Staurosporine in S. lividans

We attempted the cloning of the staurosporine biosynthetic genes assuming that they form a cluster including regulatory gene(s) and that the gene expression and production of staurosporine can be carried out in S. lividans as a surrogate host that does not produce indolocarbazole compounds. S. lividans TK23 was used for transformation with pTYMCsta by using 20 μg/ml thiostrepton for selection. The transformants were cultured

Fig. 3. Restriction map and genetic organization of cloned DNA fragments from Streptomyces sp. TP-A0274.

![Diagram](https://i.imgur.com/yourimageurl.png)

- regulatory gene
- indolocarbazole aglycone synthesis
- deoxysugar synthesis
- coupling of aglycone and deoxysugar

B: BamHI restriction enzyme site.

Fig. 4. HPLC of products of S.lividans TK23 harboring pTYMCsta (A) and S. lividans TK23 (B).

![Graphs](https://i.imgur.com/yourimageurl2.png)

The arrowed peaks correspond to staurosporine. HPLC conditions and sample preparation were described in the text. The elution was done with a linear gradient as indicated on the right-hand scale in A. Staurosporine production by Streptomyces sp. TP-A0274 (□) and S. lividans transformed with pTYMCsta (○) (C). The quantity of staurosporine was estimated by the peak area of HPLC. Fermentations were carried out in triplicate flasks and the average value was indicated in the graph.

for 7-days in liquid production medium and the
fermentation broth was extracted with *n*-butanol. HPLC
analysis of this *n*-butanol extract revealed a production of a
new compound in culture broth (Fig. 4A) compared with
control TK23 strain (Fig. 4B). The new peak at 9.0 minutes
exhibited a UV-visible spectrum characteristic to
staurosporine and showed the molecular ion [M+H]<sup>+</sup> at
*m/z* 467.4 on LC/MS analysis. The authentic staurosporine
showed the same retention time and molecular mass (data
not shown). Therefore we concluded that pTYMCsta
includes the entire set of staurosporine biosynthetic genes.
The productivity of staurosporine by strain TP-A0274 and
*S. lividans* harboring pTYMCsta was compared (Fig. 4C).

The amount of staurosporine produced in *S. lividans* was
2.6 mg/liter (9-days) whereas that in strain TP-A0274 was
10.5 mg/liter (9-days).

DNA Sequencing of pTYMCsta and
Identification of ORFs

Analysis of ~22-kb of the DNA sequence from
pTYMCsta indicated that all of the staurosporine
biosynthetic genes are located in a single operon containing
14 ORFs (Fig. 3). The cluster showed striking similarity to
the rebeccamycin biosynthetic gene cluster. Six of the
identified ORFs (*staR*, *staG*, *staO*, *staD*, *staP* and *staMA*)

Table 2. Deduced genes and their proposed functions in the *sta* cluster.

| gene   | amino acids | homologous gene                              | % identity of products | origin (biosynthetic gene cluster)               | proposed function |
|--------|-------------|-----------------------------------------------|------------------------|--------------------------------------------------|--------------------|
| *staR* | 943         | *rebR*: luxR type transcriptional regulator    | 35                     | *Lechevalieria aerocolonigenes* (rebeccamycin)   | R                  |
| *staB* | 334         | *aviE*: dTDP-glucose-4,6-dehydratase          | 70                     | *Streptomyces viridochromogenes* (avilamycin)     | B                  |
| *staA* | 350         | *strD*: glucose-1-phosphate thymidyltransferase | 69                     | *Streptomyces griseus* (streptomycin)            | B                  |
| *staN* | 394         | *doxA*: cytochrome P450                        | 27                     | *Streptomyces peucetius* (daunorubicin)          | C                  |
| *staG* | 446         | *rebG*: N-glycosyltransferase                  | 56                     | *L. aerocolonigenes*                               | C                  |
| *staO* | 504         | *rebO*: L-amino acid oxidase                   | 53                     | *L. aerocolonigenes*                               | A                  |
| *staD* | 1096        | *rebD*: CCA synthetase                        | 55                     | *L. aerocolonigenes*                               | A                  |
| *staP* | 417         | *rebP*: cytochrome P450                       | 52                     | *L. aerocolonigenes*                               | A                  |
| *staMA*| 276         | *rebM*: O-methyltransferase                   | 36                     | *L. aerocolonigenes*                               | B                  |
| *staJ* | 472         | *urdS*: 2,3-dehydratase                       | 68                     | *Streptomyces fradiae* (urdamycin)                | B                  |
| *staK* | 328         | *jadV*: 4-ketoreductase                       | 47                     | *Streptomyces venezuelae* (jadomycin B)           | B                  |
| *staI* | 369         | *orf10*: aminotransferase                     | 81                     | *Amycolatopsis mediterranei* (balhimycin)         | B                  |
| *staE* | 206         | *orf11*: 3,5-epimerase                        | 71                     | *A. mediterranei* (balhimycin)                    | B                  |
| *staMB*| 280         | *mitM*: O-methyltransferase                   | 53                     | *Streptomyces lavendulae* (mitomycin)             | B                  |

Proposed function, R: pathway specific regulator, A: indolocarbazole aglycone synthesis.
B: deoxysugar biosynthesis, C: coupling of aglycone with deoxysugar

were found to have, on average, 47.8% identity to the corresponding ORFs of the rebeccamycin cluster at the amino acid sequence level. The remaining eight ORFs were identified to have no corresponding genes in the rebeccamycin biosynthetic gene cluster. Homology search based upon their deduced amino acid sequences revealed that seven of them (staA, staB, staJ, staK, stal, staE and staMB) are responsible for the biosynthesis of the deoxysugar moiety, but the function of a remaining ORF, staN, was unidentified at this point. Table 2 summarizes the homologies found between the genes from strain TP-A0274 and the GenBank entries, and proposed functions in the staurosporine biosynthesis.

### Discussion

A proposed biosynthetic pathway for the aglycone and deoxysugar moiety of staurosporin is shown in Fig. 5A and 5B. The biosynthesis of aglycone has been discussed in our previous paper⁶). The biosynthesis of the deoxysugar, 2,3,6-trideoxy-3-amino-3,4-N,O-dimethylaldohexose 8, is first proposed in this paper. Tentative assignment of each ORF to a specific function in the deoxysugar biosynthesis was made by comparing with the deoxysugar biosynthetic genes responsible for the biosynthesis of secondary metabolites from actinomycetes¹²), in particular, megalomicin¹³) and erythromycin¹⁴). The staA and staB are supposed to code for D-glucose-1-phosphate thymidytransferase and dTDP-glucose-4,6-dehydratase, respectively. These enzymes mediate the conversion of D-glucose-1-phosphate 1 to dTDP-4-keto-6-deoxy-D-glucose 3, the key intermediate for deoxysugars in various secondary metabolites of actinomycetes. The next steps seem to be catalyzed by StaE and StaJ coding for a 3,5-epimerase and 2,3-dehydratase, respectively. The formation of the 3,4-diketosugar 5 from 3 has been proposed in the biosynthesis of L-daunosamine¹⁵). In the biosynthesis of 8, four further genes (staMA, staK, stal, staMB) are involved. Sequence similarities between the deduced enzymes and proteins in databases suggested that StaK is a 4-ketoreductase, Stal functions as an aminotransferase and both of StaMA and StaMB are methyltransferases. It seems that StaK and Stal catalyse the reduction at C-4 and the amination at C-3 of 5 but the order of the reactions has not been identified yet. StaMA and StaMB are probably responsible for the N- and O-methylation at C-3 and C-5 but it was not concluded which gene product catalyzes N- or O-methylation by amino acid

Fig. 5. Proposed staurosporine biosynthetic pathway.

**A**

![Diagram A]

**B**

![Diagram B]

The aglycone biosynthetic pathway (A). The deoxysugar biosynthetic pathway (B).

sequence homology search.

In the structure of staurosporine, the aglycone and deoxysugar are connected by two C–N bonds. Of these the linkage between the nitrogen at N-13 of aglycone and the anomic position of deoxysugar is probably formed by StaG, a putative *N*-glycosyltransferase. The bond formation between the nitrogen at N-12 of aglycone and the carbon at C-5′ of deoxysugar is proposed to be mediated by StaN in consideration of the following results provided by the sequence analysis. In the biosynthetic cluster, *staN* is the only one which was not assigned to any functions in staurosporine biosynthesis. *staN* is translationally coupled to *staG*, indicating that *staG* and *staN* function together in the biosynthesis. The deduced gene product of *staN* shows 27% similarity to cytochrome P450 hydroxylase in daunorubicin biosynthetic cluster<sup>18)</sup>. The involvement of an enzyme catalyzing an oxidation reaction is reasonable because hydrogens at N-12 of aglycone and at C-5′ of deoxysugar must be eliminated from the molecule to form the C–N bond between N-12 and C-5′.

The present study also shows that the amino acid sequence of chromopyrrolic acid synthase is utilized as a PCR primer for the effective cloning of indolocarbazole biosynthetic gene clusters. With this method, indolocarbazole biosynthetic gene clusters for unknown indolocarbazoles can be cloned from screening sources such as soil-derived bacteria. In the near future the combinatorial biosynthesis of indolocarbazoles as well as polyketide and non-ribosomal peptide antibiotics will be started and the products will be supplied for pharmaceutical screenings.

Acknowledgements

We thank Dr. H. IKEDA at Kitasato University for providing pKU402 plasmid. We also thank Mr. Y. IKEDA at Toyama Prefectural University for obtaining of MS spectra. This work was supported in part by a Grant-in-Aid for Scientific Research to H.O. from the Japan Society for the Promotion of Science.

References

1) OMURA, S.; Y. SASAKI, Y. IWAI & H. TAKESHIMA: Staurosporine, a potentially important gift from a microorganism. J. Antibiotics 48: 535~548, 1995
2) NETTLETON, D. E.; T. W. DOYLE, B. KRISHNAN, G. K. MATSUMOTO & J. CLARDY: Isolation and structure of rebeccamycin—a new antitumor antibiotic from *Nocardia aerocoligenes*. Tetrahedron lett. 26: 4011~4014, 1985
3) KASE, H.; K. IWAHASHI & Y. MATSUDA: K-252a, a potent inhibitor of protein kinase C from microbial origin. J.
Antibiotics 39: 1059~1065, 1986
4) AKINAGA, S.; K. SUGIYAMA & T. AKIYAMA: UCN-01 (7-hydroxystaurosporine) and other indolocarbazole compounds: a new generation of anti-cancer agents for the new century? Anticancer Drug Des. 15: 43~52, 2000
5) SANCHEZ, C.; I. A. BUTOVICH, A. F. BRANA, J. ROHR, C. MENDEZ & J. A. SALAS: The biosynthetic gene cluster for the antitumor rebeccamycin: characterization and generation of indolocarbazole derivatives. Chem. Biol. 9: 519~531, 2002
6) ONAKA, H.; S. TANIGUCHI, Y. IGARASHI & T. FURUMAI: Characterization of the biosynthetic gene cluster of rebeccamycin from *Lechevalieria aerocolonigenes* ATCC 39243. Biosci. Biotechnol. Biochem. 67(1), 2003, in press
7) SAMBROOK, J. & D. W. RUSSELL: Molecular cloning, a laboratory manual, third edition. Cold spring harbor laboratory press, NY, USA, 2001
8) ONAKA, H.; H. TABATA, Y. IGARASHI, Y. SATO & T. FURUMAI: Goadsporin, a chemical substance which promotes secondary metabolism and morphogenesis in streptomycetes. I. Purification and characterization. J. Antibiotics 54: 1036~1044, 2001
9) MAZODIER, P.; R. PETTER & C. THOMPSON: Intergeneric conjugation between *Escherichia coli* and *Streptomyces* species. J. Bacteriol. 171: 3583~3585, 1989
10) PANG, C. H.; M. SHIIYAMA, H. IKEDA, H. TANAKA & S. OMURA: Cosmid vector for cloning and analysis of *Streptomyces* DNA. Actinomycetol. 8: 21~25, 1994
11) KIESER, T.; M. J. BIBB, M. J. BUTTNER, K. F. CHATER & D. A. HOPWOOD: *Practical Streptomyces genetics.* (The John Innes Foundation, Norwich, United Kingdom), 2000
12) TREFZER, A.; J. A. SALAS & A. BECHTHOLD: Genes and enzymes involved in deoxysugar biosynthesis in bacteria. Nat. Prod. Rep. 16: 283~299, 1999
13) VOLCHEGURSKY, Y.; Z. HU, L. KATZ & R. MCDANIEL: Biosynthesis of the anti-parasitic agent megalomicin: transformation of erythromycin to megalomicin in *Saccharopolyspora erythraea*. Mol. Microbiol. 37: 752~762, 2000
14) SUMMERS, R. G.; S. CONADIO, M. J. STAVER, E. WENDT-PIENKOWSKI, C. R. HUTCHINSON & L. KATZ: Sequencing and mutagenesis of genes from the erythromycin biosynthetic gene cluster of *Saccharopolyspora erythraea* that are involved in L-mycarose and D-desosamine production. Microbiol. 143: 3251~3262, 1997
15) OTTEN, S. L.; M. A. GALLO, K. MADDURI, X. LIU & C. R. HUTCHINSON: Cloning and characterization of the *Streptomyces peucetius dnmZUV* genes encoding three enzymes required for biosynthesis of the daunorubicin precursor thymidine diphospho-L-daunosamine. J. Bacteriol. 179: 4446~4450, 1997
16) WESTRICH, L.; S. DOMANN, B. FAUST, D. BEDFORD, D. A. HOPWOOD & A. BECHTHOLD: Cloning and characterization of a gene cluster from *Streptomyces cyanogenus* S136 probably involved in landomycin biosynthesis. FEMS Microbiol. Lett. 170: 381~387, 1999
17) WEIDNER, S.; M. KITTELMANN, K. GOEKE, O. GHISALBA & H. ZAHNER: 3′-Demethoxy-3′-hydroxystaurosporine-

VOL. 55 NO. 12                                                                 THE JOURNAL OF ANTIBIOTICS                                                                 1071

O-methyltransferase from *Streptomyces longisporoflavus*  
catalyzing the last step in the biosynthesis of  
staurosporine. J. Antibiotics 51: 679~682, 1998  

18) LOMOVSKAYA, N.; S. L. OTTEN, Y. DOI-KATAYAMA, L.  
    FONSTEIN, X. C. LIU, T. TAKATSU, A. INVENTI-SOLARI, S.  

FILIPPINI, F. TORTI, A. L. COLOMBO & C. R. HUTCHINSON:  
Doxorubicin overproduction in *Streptomyces peucetius*:  
cloning and characterization of the *dnrU* ketoreductase  
and *dnrV* genes and the *doxA* cytochrome P-450  
hydroxylase gene. J. Bacteriol. 181: 305~318, 1999
