
This article was downloaded by: [York University Libraries]
On: 18 November 2014, At: 04:06
Publisher: Taylor&Francis
Informa Ltd Registered in England and Wales Registered Number: 1072954 Registered office: Mortimer
House, 37-41 Mortimer Street, London W1T 3JH, UK

---

**Bioscience, Biotechnology, and Biochemistry**

Publication details, including instructions for authors and subscription information:
[http://www.tandfonline.com/loi/tbbb20](http://www.tandfonline.com/loi/tbbb20)

**Characterization of the Biosynthetic Gene Cluster of Rebeccamycin from Lechevalieria aerocolonigenes ATCC 39243**

Hiroyasu ONAKA${}^{a}$, Shin-ichi TANIGUCHI${}^{a}$, Yasuhiro IGARASHI${}^{a}$ & Tamotsu FURUMAI${}^{a}$

${}^{a}$ Biotechnology Research Center, Toyama Prefectural University Kosugi, Toyama 939-0398, Japan

Published online: 22 May 2014.

---

To cite this article: Hiroyasu ONAKA, Shin-ichi TANIGUCHI, Yasuhiro IGARASHI & Tamotsu FURUMAI (2003) Characterization of the Biosynthetic Gene Cluster of Rebeccamycin from Lechevalieria aerocolonigenes ATCC 39243, Bioscience, Biotechnology, and Biochemistry, 67:1, 127-138, DOI: [10.1271/bbb.67.127](10.1271/bbb.67.127)

To link to this article: [http://dx.doi.org/10.1271/bbb.67.127](http://dx.doi.org/10.1271/bbb.67.127)

---

PLEASE SCROLL DOWN FOR ARTICLE

---

Taylor & Francis makes every effort to ensure the accuracy of all the information (the “Content”) contained in the publications on our platform. However, Taylor & Francis, our agents, and our licensors make no representations or warranties whatsoever as to the accuracy, completeness, or suitability for any purpose of the Content. Any opinions and views expressed in this publication are the opinions and views of the authors, and are not the views of or endorsed by Taylor & Francis. The accuracy of the Content should not be relied upon and should be independently verified with primary sources of information. Taylor and Francis shall not be liable for any losses, actions, claims, proceedings, demands, costs, expenses, damages, and other liabilities whatsoever or howsoever caused arising directly or indirectly in connection with, in relation to or arising out of the use of the Content.

This article may be used for research, teaching, and private study purposes. Any substantial or systematic reproduction, redistribution, reselling, loan, sub-licensing, systematic supply, or distribution in any form to anyone is expressly forbidden. Terms & Conditions of access and use can be found at [http://www.tandfonline.com/page/terms-and-conditions](http://www.tandfonline.com/page/terms-and-conditions)

Biosci. Biotechnol. Biochem., 67 (1), 127–138, 2003

# Characterization of the Biosynthetic Gene Cluster of Rebeccamycin from *Lechevalieria aerocolonigenes* ATCC 39243

Hiroyasu ONAKA,$^{\dagger}$ Shin-ichi TANIGUCHI, Yasuhiro IGARASHI, and Tamotsu FURUMAI

Biotechnology Research Center, Toyama Prefectural University, Kosugi, Toyama 939-0398, Japan

Received July 29, 2002; Accepted August 19, 2002

The biosynthetic gene cluster for rebeccamycin, an indolocarbazole antibiotic, from *Lechevalieria aerocolonigenes* ATCC 39243 has 11 ORFs. To clarify their functions, mutants with *rebG*, *rebD*, *rebC*, *rebP*, *rebM*, *rebR*, *rebH*, *rebT*, or *orfD2* disrupted were constructed, and the gene products were examined. *rebP* disruptants produced 11,11′-dichlorochromopyrrolic acid, found to be a biosynthetic intermediate by a bioconversion experiment. Other genes encoded *N*-glycosyltransferase (*rebG*), monooxygenase (*rebC*), methyltransferase (*rebM*), a transcriptional activator (*rebR*), and halogenase (*rebH*). *rebT* disruptants produced rebeccamycin as much as the wild strain, so *rebT* was probably not involved in rebeccamycin production. Biosynthetic genes of staurosporine, another indolocarbazole antibiotic, were cloned from *Streptomyces* sp. TP-A0274. *staO*, *staD*, and *staP* were similar to *rebO*, *rebD*, and *rebP*, respectively, all of which are responsible for indolocarbazole biosynthesis. But a *rebC* homolog, encoding a putative enzyme oxidizing the C-7 site of pyrrole rings, was not found in the staurosporine biosynthetic gene cluster. These results suggest that indolocarbazole is constructed by oxidative decarboxylation of chromopyrrolic acid (11,11′-dichlorochromopyrrolic acid in rebeccamycin) generated from two molecules of tryptophan by coupling and that the oxidation state at the C-7 position depends on the additional

enzyme(s) encoded by the biosynthetic genes.

Key words: rebeccamycin; indolocarbazole; biosynthetic pathway; *Lechevalieria aerocolonigenes* ATCC 39243; chromopyrrolic acid

Rebeccamycin (Fig. 1) is an indolocarbazole alkaloid produced by *Lechevalieria aerocolonigenes* ATCC 39243.* It inhibits DNA topoisomerase I with an IC₅₀ of 1.75 μM in vitro and inhibits the growth of human lung adenocarcinoma cells with a GI₅₀ of 6.0 μg/ml in vivo.¹ Indolocarbazole compounds from natural sources have attracted attention because of they are inhibitors of protein kinases or DNA topoisomerases. Staurosporine (Fig. 1), the strongest inhibitor of phospholipid-Ca²⁺-dependent protein kinase (protein kinase C), with an IC₅₀ of 2.7 nM, is frequently used to study cellular processes.² Protein kinases and DNA topoisomerases are important in cell proliferation, so specific inhibitors are promising candidates for antitumor drugs. In fact, UCN-01,³ BMY-27557,⁴ and J-109384⁵ are indolocarbazoles now in clinical trials for cancer chemotherapy.

Most indolocarbazole compounds are isolated from actinomycetes: rebeccamycin is from *Lechevalieria aerocolonigenes* ATCC 39243, staurosporine

![Chemical Structures](chemical_structures.png)

1: Rebeccamycin  
2: Staurosporine  
3: Violacein  

Fig. 1. Chemical Structures of Indolocarbazole Compounds in This Study, and of Violacein.

$^\dagger$ To whom correspondence should be addressed. Tel: +81-766-56-7500 ext. 522; Fax: +81-766-56-2498; E-mail: onaka@pu-toyama.ac.jp

Abbreviations: CCA, 11,11′-dichlorochromopyrrolic acid; ORF, open reading frame

* The genus name for strain ATCC 39243 was changed from *Saccharothrix* to *Lechevalieria* in 2001.²⁷

is from *L. aerocolonigenes* subsp. *staurosporeus* and *Streptomyces* sp. strain TP-A0274, K252a is from *Nocardiopsis* sp. strain K252<sup>6</sup> and J-104303 is from *Streptomyces mobaraensis* BA 13793.<sup>7</sup> Biosynthetic studies done with isotope-labeled precursors indicated that the indolocarbazole structure is derived from two molecules of tryptophan, probably via indolepyruvic acid as an intermediate.<sup>8–10</sup> Ohuchi et al. reported cloning of the *ngt* gene encoding an N-glycosyltransferase from *L. aerocolonigenes* ATCC 39243.<sup>5</sup> The Ngt protein converted an indolocarbazole, J-104303, to its N-glucoside, indicating that *ngt* is responsible for N-glycosylation in rebeccamycin biosynthesis. Very recently, in a first report about indolocarbazole antibiotics, Sanchez et al. cloned the rebeccamycin biosynthetic genes, using an internal fragment of *ngt* as a probe.<sup>11</sup> Nucleotide sequence analysis of a 25.6-kb fragment of the *L. aerocolonigenes* ATCC 39243 genome found 18 open reading frames (ORFs). On the basis of sequence analysis and database searches, they proposed four genes for indolocarbazole biosynthesis (*rebO*, *rebD*, *rebC*, and *rebP*), two genes for halogenation (*rebH* and *rebF*), glycosylation (*rebG*, the name was changed from *ngt*), and sugar methylation (*rebM*), as well as a regulatory gene (*rebR*) and two resistance and secretion genes (*rebU* and *rebT*). Heterologous gene expression of the set *rebO*, *rebD*, *rebC*, and *rebP* for indolocarbazole biosynthesis resulted in production of deschlororebeccamycin aglycone. We have been independently investigating the biosynthetic genes for rebeccamycin,<sup>12</sup> and discovered its biosynthetic pathway with the identification of several key biosynthetic intermediates. We here characterize the biosynthetic gene cluster for rebeccamycin using a newly developed host-vector system for the genus *Lechevalieria* and gene-disruption experiments. The structures of the accumulated products of each *reb* disruptant were identified and an overall biosynthetic pathway of rebeccamycin is proposed.

colonigenes was medium G134.<sup>1</sup> The seed culture for *L. aerocolonigenes* was in V-22 medium.<sup>17</sup> Bennett's glucose agar,<sup>17</sup> nutrient broth agar, and mannitol soya flour agar<sup>18</sup> were used for conjugational transformation.

**General recombinant DNA techniques.** Restriction enzymes, T4 DNA ligase, and Taq polymerase were purchased from New England Biolabs. DNA fragments were labelled with [α-<sup>32</sup>P]-dCTP (6000 Ci mmol<sup>-1</sup>; Amersham) and the BcaBEST labelling kit (Takara Shuzo Co., Ltd.). PCR was carried out with the PTC-200 DNA Engine (MJ Research, MA, USA). DNA manipulations were as described by Sambrook and Russell.<sup>16</sup>

**Cloning of the rebeccamycin biosynthetic genes.** The two PCR primers, 5′-TCGGAATTCATGG-GGGCACGAGTGCTG-3′ (NGT1) and 5′-AGGA-AGCTTGAACGGGCCGACGAACCT-3′ (NGT2), were synthesized on the basis of the *ngt* DNA sequence recorded as accession No. AB023953 in GenBank. Underlining letters indicate EcoRI and HindIII restriction enzyme sites for cloning to vectors. A 731-bp DNA fragment was amplified by PCR with each of the PCR primers and *L. aerocolonigenes* chromosomal DNA. The amplified products were radiolabelled by the random priming method and put through colony hybridization. For the construction of a genomic library, chromosomal DNA of *L. aerocolonigenes* was prepared as described by Kieser et al.<sup>18</sup> and partially digested with Sau3AI, and DNA fragments of >30 kb were purified by gel electrophoresis. The purified DNA fragments were ligated with pKU402 and packaged into λ phage. A hybridized cosmid, pREBC5, was digested with BamHI and the hybridizing 8-kb fragment was subcloned into pUC19 to give pREB1. 3.4-kb BamHI, 2.9-kb BamHI, 2.7-kb SphI, 2-kb PstI, and 2.3-kb XhoI fragments, which were upstream from of the 8-kb fragment, also were cloned into pUC19 to give pREB5, pREB7, pREB9, pREB10, and pREB11, (see Fig. 2(A)).

**Materials and Methods**

**Bacterial strains, plasmids, and growth condition.** *L. aerocolonigenes* ATCC 39243 was obtained from the American Type Culture Collection and used as the source of DNA in the construction of the genomic libraries. *Streptomyces* sp. strain TP-A0274 was newly isolated from soil collected in Kosugi-machi, Toyama, Japan. *Escherichia coli* DH-5α served as the host for plasmid subcloning in pUC19 and pK18mob.<sup>13</sup> *E. coli* XL1-Blue MR was used for cloning of the pK402<sup>14</sup> and the pTOYAMAcos cosmid (Onaka, in preparation). *E. coli* S17-1 was used for transconjugation.<sup>15</sup> Growth conditions and manipulations of *E. coli* were as described by Sambrook and Russell.<sup>16</sup> The production medium for *L. aerocolonigenes* was medium G134.<sup>1</sup> The seed culture for *L. aerocolonigenes* was in V-22 medium.<sup>17</sup> Bennett's glucose agar,<sup>17</sup> nutrient broth agar, and mannitol soya flour agar<sup>18</sup> were used for conjugational transformation.

**Construction of plasmids for gene inactivation.** (i) pDrebG. pDrebG was constructed by the cloning of a 776-bp *PmaCI-SphI* fragment prepared from pREB1, which contained an internal region of the *rebG* gene, into the *SmaI-SphI* sites of pK18mob. (ii) pDrebD, pDrebP, pDrebM, pDrebH, pDrebT, and pDorfD2. A 543-bp fragment was amplified by PCR with primers designed on the basis of the internal region of the *rebD* and pREB1 as the template. The fragment was inserted into pK18mob to give pDrebD. Five plasmids, pDrebP, pDrebM, pDrebH, pDrebT, and pDorfD2, which contained internal regions of *rebP*, *rebM*, *rebH*, *rebT*, and *orfD2* as shown in Fig. 2(A), also were constructed by the

Biosynthetic Pathway of Rebeccamycin

A
10 kb
B B B B BB
| | | || |
Probe pREB1 S N NB
pDorfD2 pDrebG pDrebD ΔrebC pDrebM
reb orfD2 rebG rebO rebD rebC rebP rebM
B
10 kb
B BBB B B B B B
| | | || | | |
S BE S 1 kb B S S B
pSTA3 pSTA1 pSTA2
sta staG staO staD staP orf1 orf2
N-Glycosyltransferase Monoxygenase Flavin reductase Regulator
L-Amino acid oxidase Cytochrome P450 Na+/H+ antiporter Halogenase
Trp condensation enzyme Methyltransferase Function unknown

Fig. 2. Restriction and Gene Organization Map of Cloned DNA Fragments from *L. aerocolonigenes* (A) and *Streptomyces sp.* Strain TP-A0274 (B).

Filled black bars indicate cloning regions of the plasmids used for single crossover disruption. Open bars indicate deleted regions of the in-frame deletion. B: *BamHI*, E: *EcoRI*, N: *NcoI*, P: *PstI*, S: *SphI*, X: *XhoI*.

same method as that for pDrebD with pREB5 (pDrebP, pDrebM), pREB1 (pDorfD2), pREB10 (pDrebH), or pREB11 (pDrebT) as the templates for PCR. The primer sequences are as follows: rebD-N; 5′-GAAGAATTCGTGATGCTGCAGTACCTGTA-3′, rebD-C; 5′-CGAAAGCTTCAGGAACAGGGTG-GTGCTCGCC-3′, rebP-N; 5′-CCGAAGCTTCACGGCGCTGAGATCGTTG-3′, rebP-C; 5′-GTC GAATCCGCCCAGCGAGTGACCATCTGCAC-3′, rebM-N; 5′-ACGAAGCTTTCCGGCTTGGCG-AACATCGGAC-3′, rebM-C; 5′-CGGGAATTCA CGTCGGCTGCGGGATCGGGC-3′, rebH-N; 5′- AGCGAATCCCGCTGTCGCACTACTGGTTCG-3′, rebH-C; 5′-GTCAAAGCTTCTCCTGCATGC-CGTCCGCCAG-3′, rebT-N; 5′-CGGGAATTC CCTGTTCCGGTGCCGCCTCCCT-3′, rebT-C; 5′- CCGAAGCTTTCTCCGGCGAGTCCCCGGCGACGG-3′, orfD2-N; 5′-CACGAATTCCGAGGAGGTTCC GCGAGAAGGTGGG-3′, orfD2-C; 5′-GAAAAAG CTTGAAGCCGTGGGATAGCGGTGGTG-3′, in which *EcoRI* or *HindIII* sites (underlining letters indicated) were created for facilitation of the cloning.

(iii) pDrebC and pDrebR. For in-frame deletion of the rebC, a 3.0-kb *SphI* fragment was prepared from pREB1 and cloned into the *SphI* site of pUC19 to generate pREB1A. The plasmid was digested with *NcoI* and the resulting rebC-truncated large fragment was purified and self-ligated to give pREB1B. A 2.9-kb *BamHI-EcoRI* fragment was prepared from

pREB5 and subcloned into the *BamHI-EcoRI* site of pREB1B to construct pREB1C. A 5-kb *EcoRI-Hin-dIII* fragment was prepared from pREB1C and cloned into the *EcoRI-HindIII* site of pK18mob to construct pDrebC. For in-frame deletion of rebR, a 2.2-kb *SphI* fragment was prepared from pREB7 and cloned into the *SphI* site of pK18mob to generate pREB7A. A 2.3-kb *PstI* fragment (one of the *PstI* sites is on a multiple cloning site of pUC19) was prepared from pREB5 and subcloned into the *PstI* site of pREB7A to construct pDrebR. Both of the deletion regions are shown in Fig. 2(A).

Gene disruption and transformation in *Leche-valeria aerocolonigenes*. For disruption of the chromosomal genes, we used the method of insertional inactivation via a single crossover with the non-replicating *E. coli* plasmid, pK18mob, derivatives (pDrebG, pDrebD, pDrebP, pDrebM, pDrebH, pDrebT, and pDorfD2) or via a double crossover with pDrebC and pDrebR. rebG and rebC double disruptants were constructed from the rebC-truncated mutant strain to use the method via a single crossover with pDrebG. The method of transconjugation is modified from the method for *Streptomyces* species as described by Mazodier et al. After transconjugation, 3 ml of 0.5% nutrient broth agar (Difco) with 200 μg/ml kanamycin was overlaid on a mannitol soya flour agar plate for transformant selection

and the selected colonies were transferred to a Bennett's glucose agar plate containing 5 μg/ml nalidixic acid and 20 μg/ml kanamycin. The chromosomal DNAs were prepared from the candidate transformants and the restriction map of the chromosomal DNA was checked by Southern hybridization. For disruption of the chromosomal genes with pDrebC and pDrebR, first crossover events were done with the same method as for single crossover disruption. The pDrebC- or pDrebR-integrated strains were cultivated in a 500-ml K-1 flask containing 100 ml of V-22 medium with no antibiotics. After two rounds of cultivation, we selected colonies that were kanamycin-sensitive to obtain in-frame deletion mutants. The genomic structure of the strains obtained was checked by Southern hybridization.

HPLC of the products from gene-disrupted mutants. HPLC was done on a HP 1090 system with a diode array detector (Hewlett Packard) and a C18 Rainin Microsorb column (3 μm, 100 × 4.6 mm, i.d.; Rainin Instrument Co., MA, USA). Acetonitrile-0.15% KH₂PO₄ was the elution buffer. The temperature was 40°C, the flow rate was 1.2 ml/min, the solvent was acetonitrile-0.15% KH₂PO₄ (pH 3.5), and detection was at 254 nm. (the gradient diagram is in Fig. 4(A)) Each of the disrupted mutants was used to inoculate a 500-ml K-1 flask containing 100 ml of V-22 medium. After incubation at 30°C for 2 days on a rotary shaker at 200 rpm, 5-ml portions of the seed culture were transferred into 500-ml K-1 flasks containing 100 ml of medium G134. Fermentation was at 30°C for 11 days on the same rotary shaker. The culture broth was centrifuged to collect a mycelial cake, which was extracted with 100 ml of acetone. After the acetone was evaporated off, the resultant solution was extracted with 100 ml of ethyl acetate three times, concentrated under reduced pressure, and dissolved in Me₂SO for HPLC. The supernatant of the culture broth also was checked for production of rebeccamycin or its related compounds.

Purification of rebeccamycin intermediates. To purify the rebeccamycin aglycon (5), *L. aerocolonigenes* rebG::km' was used to inoculate 500-ml K-1 flasks containing 100 ml of V-22 medium. After cultivation at 30°C for 2 days on a rotary shaker, 5-ml portions of the seed culture were transferred into fifty 500-ml K-1 flasks, each containing 100 ml of medium G134. The culture broth (5 liters) was centrifuged at 5000 × g for 10 min. The supernatant was discarded and the mycelial cake was extracted with 5 liters of acetone. After the evaporation, the resultant aqueous solution was extracted with an equal volume of ethyl acetate two times and concentrated under reduced pressure to give 1.2 g of crude extract. The residue was dissolved in 2.5 ml of methanol and put on an ODS-AM reverse-phase silica gel column

(4.6 i.d. × 20 cm; YMC) equilibrated with 20% aqueous methanol. The column was eluted with 50% aqueous methanol and then 100% methanol. The rebeccamycin aglycon eluted in the 100% methanol fraction. The fractions containing the rebeccamycin aglycon were pooled, and evaporated under reduced pressure to give pure rebeccamycin aglycon (61.2 mg). For purification of 4 (11,11'-dichlorochromopyrrolic acid), 10 (4'-O-demethylrebeccamycin), 8 (7-deoxo-7-hydroxyrebeccamycin aglycon), 9 (7-deoxorebeccamycin aglycon), and 11 (1,11-dichloro-4'-O-demethylrebeccamycin), strain rebD::km' (4), strain rebM::km' (10), strain ΔrebC+rebG::km' (8, 9), or strain rebH::km' (11) was used and the purification method was the same as for rebeccamycin aglycon except that the materials eluted in the 50% methanol fraction from the ODS-AM column. Further HPLC was done of the products from strain ΔrebC+rebG::km' (8, 9) and strain rebH::km' (11) to obtain purity high enough for NMR. HPLC purification was done on a Shimadzu HPLC system (SPD-M10A) with a diode array detector (LC10-AT, Shimadzu) and an XTerra RP18 column (19 i.d. × 300 mm; Waters). Partially purified samples containing the intermediates were dissolved in methanol and put on an HPLC column. The temperature was 30°C, flow rate 15 ml/min, solvent 50% acetonitrile-0.15% KH₂PO₄ (pH 3.5). Each intermediate was fractionated on the basis of the information from the diode array detector, with detection at 254 nm.

Analysis of metabolites. ¹H and ¹³C NMR spectra were measured at 400 and 100 MHz in Me₂SO-*d*₆ on a JNM-LA 400 apparatus (JEOL). LC-MS spectra were obtained on an API 165 machine (Applied Biosystems). UV-visible spectra were taken on a HP 1090 system. NMR data for compounds 4, 5, 8, 9, 10, and 11 are shown in Tables 1 to 3. MS data are as follows. Compound 4: ESI-MS [M+H]⁺: *m/z* = 454.2, [M-H]⁻: *m/z* 452.2. Compound 5: ESI-MS [M-H]⁻: *m/z* 392.0. Compound 8: ESI-MS [M+H]⁺: *m/z* 396.2. Compound 9: ESI-MS [M+H]⁺: *m/z* 380.2. Compound 10: ESI-MS [M+H]⁺: *m/z* 556.0, [M-H]⁻: *m/z* 554.0. Compound 11: ESI-MS [M+H]⁺: *m/z* 488.6, [M-H]⁻: *m/z* 486.0.

Bioconversion experiments. Two hundred μg of a putative purified intermediate was added to a tube containing 10 ml of medium G134 and already cultured *L. aerocolonigenes* rebH::km' was used to inoculate the same tube. After cultivation at 30°C for 5 days and 7 days on a rotary shaker, 5-ml portions were extracted. HPLC conditions and sample preparation were as described in the section on HPLC of the products from gene-disrupted mutants. The rebeccamycin production was identified with the retention time.

tion time by HPLC and from the UV-visible spectrum.

**Cloning and sequencing of the staurosporine biosynthetic genes.** The *staD* gene fragment was amplified from *Streptomyces* sp. strain TP-A0274 genomic DNA with the primers rebBN and rebBC, designed on the basis of the *rebD* and *vioB* sequence, and was used as a colony hybridization probe. The sequences of primers were (rebBN) 5′-GCCAA-GCTTATCTGGCAGATGTGCGACCCG-3′ and (rebBC) 5′-CAGGAATTCGATCATCTCCTCGC-GGGCGA-3′, in which *EcoRI* or *HindIII* sites (underlining letters indicated) were created for facilitation of the cloning. PCR was done at 98°C for 20 sec, 60°C for 30 sec, and 72°C for 1 min, for 30 cycles. A 426-bp amplified fragment was radiolabeled by the random priming method and used for hybridization. For the construction of a genomic library, chromosomal DNA of *Streptomyces* sp. strain TP-A0274 was partially digested with Sau3AI, and DNA fragments of >30 kb were purified by gel electrophoresis. The resulting DNA fragments were ligated with BamHI-digested pTOYAMAcos, which is an actinomycetes—*E. coli* shuttle cosmid vector, and packaged into λ phage. Five hybridized cosmids were isolated. A selected cosmid was designated pTYMCsta and digested with BamHI, and the hybridizing 5.6-kb fragment was subcloned into pUC19 to give pSTA1. Two-kilobase and 2.3-kb BamHI fragments, which were in the upstream and downstream regions of the 5.6-kb fragment, respectively, also were cloned into pUC19 to give pSTA2 and pSTA3, as shown in Fig. 2(B). These fragments were sequenced with an ABI PRISM cycle sequencing kit and analyzed on an ABI PRISM 310 automated sequencer (Applied Biosystems). The templates for DNA sequencing were prepared with the Locus Pocus transposon insertion system kit (Novagen). The DNA and protein sequence similarity searches were done with the BLAST server at the National Center for Biotechnology Information (Bethesda, MD).

**Nucleotide sequence accession number.** The nucleotide sequence for the rebeccamycin biosynthetic genes has been deposited in the DDBJ database under DDBJ Accession Number AB071405 and that for the staurosporine biosynthetic genes has been deposited under DDBJ Accession Number AB071406.

---

**Results and Discussion**

**Construction of disrupted mutants for biosynthesis**

The conventional transformation protocol involving protoplasting and regeneration of mycelia usually

used for *Streptomyces* strains could not be used for *L. aerocolonigenes* ATCC 39243. We therefore developed a protocol for conjugational transfer from *E. coli* to strain ATCC 39243 by modifying the *Streptomyces* method.¹⁵ A plasmid, pK18mob, carrying the ColE1 replicon and an origin essential for conjugal transfer was used. Chromosomal DNAs were prepared from transformants that grew well on the selection plates, and chromosomal recombination events were monitored by Southern hybridization (Figs. 3(C) and 3(D)). Except for *rebC* and *rebR* disruptants, insertional inactivation via a single crossover was confirmed (Fig. 3(A)). *rebC* and *rebR* were disrupted by in-frame deletion (Fig. 3(B)) because insertional inactivation of *rebC* or *rebR* probably would have a polar effect on the transcription of genes downstream form *rebC* or *rebR* (Fig. 2(A)). Double disruptants of *rebC* and *rebG* were constructed from an in-frame *rebC* deletion mutant by a single crossover in the *rebG* region. Finally, three mutants were independently isolated and further analyzed for each of the following: *rebG*, *rebD*, *rebC*, *rebC+rebG*, *rebP*, *rebR*, *rebH*, *rebT*, and *orfD2* disruptants. Two mutants were isolated from *rebM* disruptants and analyzed.

**Characterization of rebeccamycin biosynthetic genes**

The *rebG*, *rebD*, *rebC*, *rebP*, *rebM*, *rebR*, and *rebH* genes were confirmed to constitute the rebeccamycin biosynthetic gene cluster by gene disruption experiments. Although the cloning and expression of *rebG* was previously described,⁵ it was not proved there that *rebG* is involved in rebeccamycin biosynthesis. We constructed a *rebG* mutant by gene disruption and found that it produced the rebeccamycin aglycon but not rebeccamycin (Fig. 4(B)). The *rebG* product therefore was the N-glycosyltransferase responsible for rebeccamycin biosynthesis.

*rebD* encoded a protein of 1,013 amino acid residue. A BLAST search indicated 36.6% identity over the entire sequence of RebD to VioB, which is involved in violacein (Fig. 1) biosynthesis in *Chromobacterium violaceum* and catalyzes a condensation reaction of 5-hydroxytryptophan and indolepyruvic acid to generate the bisindole intermediate for violacein.¹⁹ The *rebD*-disrupted mutant did not produce compounds related to indolocarbazoles (Fig. 4(C)); in the same way, the *vioB* disruptant does not produce any violacein-related metabolites.¹⁹ *rebO*, *rebD*, *rebC*, and *rebP* seemed to be translationally coupled, suggesting that at least these four genes might be transcribed polycistrionically. Strain *rebD::km'* might have polar effects on the expression of *rebC* and *rebP*, but we could ignore such effects in this experiment because *rebC* and *rebP* products were involved in later biosynthetic steps than the *rebD* product, as described in the following paragraph

H. ONAKA et al.

**Fig. 3. Inactivation of the reb Cluster Genes by Gene Disruption Via Homologous Recombination.**

(A) Schematic presentation of rebG gene disruption via single crossover. The 0.78-kb PmaCI-SphI DNA fragment used as the probe is indicated as a thin black bar. X, XhoI; kmr, kanamycin resistance gene (aphII). rebD, rebP, rebC + rebG, rebM, rebH, rebT, and orfD2 disruptants were prepared by a similar procedure.

(B) Schematic presentation of rebC gene disruption with an in-frame deletion. N, NcoI.

(C) Southern blotting of L. aerocolonigenes wild type (W) and a disruptant of rebG, rebD, rebP, rebC + rebG, rebM, rebH, rebT, or orfD2. The chromosomal DNAs were digested with XhoI (rebG and rebH), SalI (rebC + rebG and rebD), BamHI (orfD2, rebP, rebT, and rebM). The 0.78-kb internal rebG fragment from pDrebG, 0.54-kb rebD fragment from pDrebD, 0.61-kb rebP fragment from pDrebP, 0.42-kb rebM fragment from pDrebM, 0.42-kb rebH fragment from pDrebH, 0.92-kb rebT fragment from pDrebT, and 0.57-kb orfD2 fragment from pDorfD2 were used as 32P-labelled probes.

(D) Southern blotting of a disruptant of rebC or rebR. The chromosomal DNAs were digested with SacI (rebC) and PvuII (rebR). The 0.54-kb rebD fragment from pDrebD and 2.2-kb SphI fragment from pREB7 were used as rebC and rebR probes, respectively. Lanes 1 and 2 indicate pDrebC- and pDrebR-integrated strains.

about results of the analysis of rebP and rebC disruptants. Considering the structural similarities of violacein and rebeccamycin, both of which are derived from two molecules of tryptophan, this observation led us to propose that RebD is involved in the construction of the bisindole structure by catalyzing a coupling reaction between 7-chlorotryptophan and 7-chloroindole-3-pyruvic acid.

The rebP gene product is quite similar to a cytochrome-P-450-like enzyme. The major product that the rebP disruptant accumulated was 11,11′-dichlorochromopyrrolic acid (CCA, 4) (Fig. 4(D)). The structure of CCA was identified from NMR and MS spectra. A 13C NMR spectrum showed 11 sp² carbons, suggesting that CCA was a symmetrical molecule and that all of the carbons in tryptophan were retained in CCA. 1H–13C long-range couplings observed in the HMBC spectrum confirmed the chloroindole moiety (Fig. 5(A)). The presence of a pyrrole ring was not directly confirmed by HMBC, but the structure of CCA was deduced to be the chlorinated analog of chromopyrrolic acid [20] by the LC-MS, which gave a molecular ion [M + H]+ at m/z 454.2, and by comparison of NMR results with those reported for chromopyrrolic acid. Chromopyrrolic acid [3,4-di-(3-indolyl)-pyrrole-2,5-dicarboxylic acid] is a secondary metabolite found in C. violaceum, a producer of violacein. [20] The close similarity of vioB and rebD suggested that CCA was biosynthesized directly by RebD. Conversion of CCA to indolocarba-

Biosynthetic Pathway of Rebeccamycin

HPLC                                      UV                                    Structure

A  
OD<sub>254</sub>  
Wild type  

B  
rebG::km<sup>r</sup>  

C  
rebD::km<sup>r</sup>  

D  
rebP::km<sup>r</sup>  

E  
ΔrebC  

F  
ΔrebC + rebG::km<sup>r</sup>  

G  
rebM::km<sup>r</sup>  

H  
rebH::km<sup>r</sup>  

I  
ΔrebR  

J  
OrfD2::km<sup>r</sup>  

K  
rebT::km<sup>r</sup>  

Retention time (min)

Fig. 4. HPLC Chromatograph and UV-Vis Spectrum of Products of Wild Type and Gene Disruptant Mutants.

HPLC conditions and sample preparation were described in the text. The elution was done with a linear gradient as indicated on the right-hand scale in A.

---

1: Rebeccamycin  
5: Rebeccamycin aglycon  
No products  
4: 11,11'-Dichlorochromopyrrollic acid  
6: 7-Deoxy-7-hydroxyrebeccamycin  
7: 7-Deoxy-7-hydroxy-4'-demethylrebeccamycin  
8: 7-Deoxy-7-hydroxyrebeccamycin aglycon  
9: 7-Deoxorebeccamycin aglycon  
10: 4'-Demethylrebeccamycin  
11: 1,11-Dechloro-4'-demethylrebeccamycin  
No products  
1: Rebeccamycin  
1: Rebeccamycin  

---

zole through oxidative decarboxylation is generally thought to be catalyzed by RebP. In a check of our proposal, a bioconversion experiment was done (Fig. 6). CCA was added to the culture broth of the rebH disruptant and the products were analyzed by HPLC after 5 and 7 days. The peak of CCA (9.9 min) disappeared and the production of rebeccamycin (13.3 min) was detected after 7 days (Fig. 6(B)). Because the rebH disruptant was unable to produce chlorinated indolocarbazoles, including rebeccamycin, we concluded that CCA was converted to rebeccamycin as a biosynthetic intermediate.

The rebC gene product was similar to a flavin binding site containing monooxygenase. The rebC-trun-

H.ONAKA et al.

Table 1. ${}^{1}$H NMR Data for Compounds 5, 8, 9, 10, and 11

| Compound | 5 | 8 | 9 | 10 | 11 |
|----------|---|---|---|----|----|
| Position | $\delta$ | $\delta$ | $\delta$ | $\delta$ | $\delta$ |
| 1        | —   | —   | —   | —    | 7.68(1H, d, 8.1) |
| 2        | 7.50(2H, d, 7.6) | 7.56(1H, d, 8.1) | 7.54(1H, d, 7.6) | 7.66(1H, d, 7.8) | 7.55(1H, t, 8.3) |
| 3        | 7.36(2H, t, 8.0) | 7.28(1H, t, 7.8) | 7.26(1H, t, 7.8) | 7.36(1H, t, 7.8) | 7.35(1H, t, 7.6) |
| 4        | 8.85(2H, d, 8.1) | 9.09(1H, d, 7.8) | 9.16(1H, d, 8.0) | 8.97(1H, d, 8.1) | 9.15(1H, d, 8.1) |
| 6-NH     | 11.09(1H, s) | 8.87(1H, s) | 8.61(1H, s) | 11.29(1H, s) | 11.11(1H, s) |
| 7        | —   | 6.41(1H, d, 10.0) | 4.96(2H, s) | —    | —    |
| 7-OH     | —   | 6.52(1H, d, 10.2) | —   | —    | —    |
| 8        | —   | 8.32(1H, d, 8.0) | 8.02(1H, d, 7.8) | 9.18(1H, dd, 1.2, 7.8) | 9.15(1H, d, 8.1) |
| 9        | —   | 7.31(1H, t, 7.8) | 7.32(1H, t, 7.8) | 7.37(1H, t, 7.8) | 7.36(1H, t, 7.8) |
| 10       | —   | 7.58(1H, d, 8.1) | 7.58(1H, d, 7.8) | 7.63(1H, d, 7.6) | 7.57(1H, t, 8.8) |
| 11       | —   | —   | —   | —    | 7.97(1H, d, 8.8) |
| 12-NH    | 11.86(2H, br.s) | 11.77(1H, s) | 11.76(1H, s) | —    | —    |
| 13-NH    | —   | 11.63(1H, s) | 11.58(1H, s) | 10.67(1H, s) | 11.66(1H, s) |
| 1'       | —   | —   | —   | 6.90(1H, d, 9.2) | 6.27(1H, d, 9.0) |
| 2'       | —   | —   | —   | 3.75(1H, dt, 2.7, 7.8) | 3.54(1H, m) |
| 3'       | —   | —   | —   | 3.36(1H, br.t) | 3.56(1H, m) |
| 4'       | —   | —   | —   | 3.68(1H, t, 9.3) | 3.98(1H, m) |
| 5'       | —   | —   | —   | 3.58, 3.75(1H, m) | 3.97(1H, m) |
| 6'       | —   | —   | —   | 3.97(2H, m) | 3.82, 4.08(2H, m) |
| 2'-OH    | —   | —   | —   | 4.92(1H, d, 5.8) | 4.92(1H, d, 5.2) |
| 3'-OH    | —   | —   | —   | 5.22(1H, d, 4.9) | 5.15(1H, d, 4.6) |
| 4'-OH    | —   | —   | —   | 5.39(1H, d, 5.4) | 5.41(1H, d, 4.2) |
| 6'-OH    | —   | —   | —   | 5.12(1H, br.t) | 6.01(1H, t, 4.0) |

The Me$_2$SO-$d_6$ signal (2.49ppm) was used as a reference. Integral multiplicities, and coupling constants (Hz) are in parentheses.

---

Table 2. ${}^{13}$C NMR Data for Compounds 5, 8, 9, 10, and 11

| Compound | 5      | 8      | 9      | 10     | 11     |
|----------|--------|--------|--------|--------|--------|
| Position | $\delta$ | $\delta$ | $\delta$ | $\delta$ | $\delta$ |
| 1        | 115.65 | 115.27 | 115.21 | 116.02 | 112.12 |
| 2        | 126.05 | 124.68* | 124.46* | 126.92 | 126.94 |
| 3        | 121.39 | 120.34 | 120.16 | 122.08 | 120.36 |
| 4        | 123.24 | 124.05 | 124.24 | 123.53 | 124.44 |
| 4a       | 123.15 | 124.26 | 124.36 | 123.17 | 121.43 |
| 4b       | 115.74 | 115.52 | 116.04 | 117.46 | 116.92 |
| 4c       | 120.84 | 119.16 | 119.88 | 120.59 | 121.09* |
| 5        | 170.81 | 170.35 | 171.84 | 170.31 | 170.99** |
| 7        | —      | 78.49  | 45.25  | 170.50 | 171.06** |
| 7a       | —      | 136.21 | 133.79 | 122.62 | 119.41 |
| 7b       | —      | 115.52 | 114.75 | 119.37 | 118.42 |
| 7c       | —      | 123.88 | 124.09 | 125.13 | 121.05* |
| 8        | —      | 122.04 | 120.33 | 124.10 | 124.44 |
| 9        | —      | 120.93 | 121.14 | 122.57 | 120.61 |
| 10       | —      | 124.70* | 124.52* | 130.00 | 126.76 |
| 11       | —      | 115.63 | 115.70 | 116.24 | 111.80 |
| 11a      | —      | 136.21 | 135.94 | 137.72 | 142.12 |
| 12a      | —      | 128.02 | 127.72 | 129.65 | 128.32 |
| 12b      | —      | 126.15 | 125.35 | 129.77 | 129.69 |
| 13a      | 136.96 | 136.10 | 135.80 | 137.00 | 140.77 |
| 1'       | —      | —      | —      | 84.57  | 84.44  |
| 2'       | —      | —      | —      | 71.98  | 73.04  |
| 3'       | —      | —      | —      | 77.73  | 76.59  |
| 4'       | —      | —      | —      | 69.92  | 67.51  |
| 5'       | —      | —      | —      | 82.07  | 78.55  |
| 6'       | —      | —      | —      | 60.55  | 58.27  |

The Me$_2$SO-$d_6$ signal (39.5ppm) was used as a reference.* and **, interchangeable.

---

Table 3. ${}^{1}$H NMR and ${}^{13}$C NMR Data for CCA

| 11, 11'-Dichlorochromopyrrolic acid |
|-------------------------------------|
| Position | ${}^{13}$C | ${}^{1}$H |
| 1-NH     | —         | 11.67(1H, s) |
| 2        | 123.24    | —           |
| 3        | 123.50    | —           |
| 4-NH     | —         | 11.13(2H, d, 1.9) |
| 5        | 126.25    | 7.14(2H, d, 1.7) |
| 6        | 109.37    | —           |
| 7        | 129.64    | —           |
| 8        | 118.43    | 7.06(2H, d, 8.1) |
| 9        | 119.23    | 6.76(2H, t, 7.6) |
| 10       | 119.89    | 7.00(2H, d, 7.6) |
| 11       | 115.48    | —           |
| 12       | 132.22    | —           |
| 13       | 161.47    | —           |
| 13-COOH  | —         | 12.39(2H, br.s) |

The Me$_2$SO-$d_6$ signals (2.49ppm for ${}^{1}$H; 39.5ppm for ${}^{13}$C) were used as references. Integral, multiplicities, and coupling constants (Hz) are in parentheses.

Biosynthetic Pathway of Rebeccamycin

A  
1,11-Dichlorochromopyrrolic acid (CCA, 4)

B  
Rebeccamycin aglycon (5)

C  
7-Deoxo-7-hydroxyrebeccamycin aglycon (8)

D  
7-Deoxorebeccamycin aglycon (9)

E  
4′-O-Demethylrebeccamycin (10)

F  
1,11-Dechloro-4′-O-demethylrebeccamycin (11)

Fig. 5. HMBC Correlations with Metabolites from Gene-disrupted Mutants.

A  

B  

Retention time (min)

Fig. 6. HPLC of Products of a rebH-Disruptant Grown in Medium with CCA.

The whole broth of 5-days (A) and 7-days (B) cultures were prepared as HPLC samples. The peaks of 9.9 and 13 min correspond to CCA and rebeccamycin, respectively.

LC-MS, the peaks at 7.7 (6) and 9 (7) min showed the molecular ion [M + H]⁺ at m/z 572 and 558, two units higher than that of rebeccamycin (m/z 570) and 4′-O-demethylrebeccamycin (m/z 556), respectively; NMR results showed that each peak consisted of a mixture of 12-N- and 13-N-glycosides not separable on HPLC (NMR results not shown). Taken together, with the function of RebC assumed from the sequence similarity, the structures of 6 and 7 were deduced to be 7-deoxo-7-hydroxyrebeccamycin and its 4′-O-demethyl derivative, although it is not clear whether the glycosylation site was at the 12- or 13-nitrogen. The rebG disruption experiment showed that N-glycosylation happened after indolocarbazole formation. We concluded that these two glycosylated products are shunt products. We then constructed rebG and rebC double disruptants to eliminate the glycosylated products, and analyzed the fermented products. Three peaks were detected in HPLC of the culture broth (Fig. 4(F), peaks 5, 8, and 9), and they were identified as rebeccamycin aglycon (5), 7-deoxo-7-hydroxyrebeccamycin aglycon (8), and 7-deoxorebeccamycin aglycon (9), respectively, by NMR. NMR assignments and HMBC results for 5, 8, and 9 are summarized in Tables 1 and 2 and in Figs. 5(B)-5(D). NMR results for these compounds showed similarity to reported values.²¹,²² Bioconversion experiments with 8 or 9 as an added sample were done to confirm which was the real intermediate, but neither compound was converted to rebeccamycin. Presumably, in the rebC disruptants, the other oxidase enzymes recognize the real intermediate of rebC disruptants as a substrate and produce 8 and 9. The RebC products were not confirmed in these disruption experiments, although, considering that rebeccamycin aglycon was identified as one of the products, we suppose that RebP took part in the conversion from CCA to rebeccamycin aglycone and that RebC controlled the oxidation state at the C-7 site.

The rebM product was similar to methyltransferases from various organisms. The rebM disruptant accumulated compound 10 (Fig. 4(G)),

which was identified as 4′-O-demethylrebeccamy-
cin²³ by NMR (Tables 1 and 2 and Fig. 5(E)) and LC-
MS, which gave a [M+H]⁺ ion at m/z 556.0. There-
fore, rebM encoded the methyltransferase responsi-
ble for methylation at the 4-hydroxy group of the
glucose moiety. Additionally, 4′-O-demethylrebec-
camycin was converted to rebeccamycin by an rebH
disruptant (not shown), which confirmed that the
methylation was the last step of the biosynthesis.

The rebH gene product was similar to the trypto-
phan halogenase from *Pseudomonas chlororaphis*,
which enzyme catalyzes the chlorination reaction
from L-tryptophan to 7-chloro-L-tryptophan in
pyrrolnitrin biosynthesis.²⁴ The rebH disruptant
produced an indolocarbazole compound, 11,
corresponding to the peak at 7.9 min on HPLC
(Fig. 4(H)). The mass spectrum showed [M+H]⁺ at
m/z 487 and NMR confirmed compound 11 to
be 1,11-dechloro-4′-O-demethylrebeccamycin. The
NMR results for 11 were in accordance with those
reported earlier.¹¹ We concluded that RebH is
responsible for chlorination of the indole moiety.
Taking into account the similarity to the tryptophan
halogenase from *P. chlororaphis*, chlorination
probably takes place with tryptophan at the begin-
ning of the biosynthetic cascade. The production of
1,11-dechloro-4′-O-demethylrebeccamycin might be
allowed by substrate recognition by RebO, RebD,
RebC, RebP, and RebG.

### Regulatory and other genes

A database search found that RebR was similar to
an *luxR* family transcriptional regulator. In our
experiment, the rebR-truncated mutant did not
produce compounds related to rebeccamycin
(Fig. 4(I)). Sanchez et al. reported that rebR is essen-
tial for production in the heterologous expression of
indolocarbazole biosynthesis genes.¹¹ These findings
indicate that RebR is a pathway-specific activator for
rebeccamycin production.

rebT was at the end of the rebeccamycin biosyn-
thetic gene cluster and was similar to transmembrane
efflux proteins, which are candidates for rebeccamy-
cin resistance or secretion. Disruption of rebT did
not decrease the production level: the wild type
produced rebeccamycin at the concentration of
52.5 ± 7 mg/l in 9 days of culture, and the rebT
disruptants produced it at the concentration of
52.6 ± 1 mg/l. Sanchez et al. reported that rebT in-
troduced into *Streptomyces albus* confers resistance
to rebeccamycin.¹¹ RebT might protect cells from the
influx of an excess of rebeccamycin. We concluded
that rebT is not involved in the rebeccamycin produc-
tion in *L. aerocolonigenes* ATCC 39243.

orfD2 was immediately downstream of rebG with
the reverse orientation on the chromosomal DNA.
The orfD2-disrupted mutants produced as much
rebeccamycin as the wild type strain did (Fig. 4(J)),

indicating that orfD2 was not related to rebeccamy-
cin biosynthesis and that the rebG gene was at the
end of the rebeccamycin biosynthetic cluster.

### Similarities and differences in indolocarbazole
biosynthetic genes between rebeccamycin and
staurosporine

The staurosporine-producing strain *Streptomyces*
sp. strain TP-A0274 was newly isolated from a soil
sample collected in Toyama, Japan, and the stauro-
sporine biosynthetic gene cluster was cloned on the
basis of information from the rebD nucleotide se-
quence. The genomic library of strain TP-A0274 was
constructed and the biosynthesis gene was screened
for with colony hybridization. The selected cosmid,
pTYMCsta, was confirmed to include the entire
staurosporine biosynthetic genes by heterologous
expression in *Streptomyces lividans* TK23.²⁵ The gene
organization map of rebD homolog loci is shown in
Fig. 2(B). A comparison between the amino acid
sequences encoded by *staG* and *rebG* from *L. aer-
ocolonigenes* found 55.8% identity, suggesting that
*staG* encoded an N-glycosyltransferase. Upstream
from *staG*, there were three ORFs, *staO*, *staD*, and
*staP*, and the deduced amino acid sequences were
similar to *rebO* (53.2% identity, 80.3% similarity),
*rebD* (55.2% identity, 77.4% similarity), and *rebP*
(51.8% identity, 79.8% similarity), respectively, sug-
gesting that these ORFs encoded indolocarbazole
biosynthesis genes. There was no gene similar to *rebC*
in the staurosporine biosynthetic genes in strain TP-
A0274. The differences in the structures of staurospo-
rine and rebeccamycin aglycon were the presence or
absence of a carbonyl group at C-7. *rebC*-disrupted
mutants produced 7-deoxo-rebeccamycin, so RebC
seemed to catalyze the oxidation at the C-7-position
to the carbonyl group.

### Proposed overall biosynthetic pathway of rebecc-
amycin

The most striking finding in this study was the
identification of CCA as a biosynthetic intermediate:
decarboxylation occurs after the coupling of two
tryptophan-molecules derivative. On the basis of our
results, the biosynthetic pathway for rebeccamycin
shown in Fig. 7 is proposed. The initial step is the
chlorination of tryptophan by the putative halo-
genase RebH. The produced 7-chlorotryptophan (13)
is oxidatively deaminated by RebO to give 7-chlo-
roindole-3-pyruvic acid (14). The *rebO* gene product
has 25.3% identity in its 473 amino acid sequence to
the L-amino acid oxidase from *Crotalus adamanteus*
(eastern diamondback rattlesnake),²⁶ which is a com-
ponent of snake poison and catalyzes an oxidation
reaction of L-tryptophan to indole-3-pyruvic acid.
Therefore RebO may be responsible for the deamina-
tion of 7-chlorotryptophan. 7-Chloroindole-3-
pyruvic acid (14) is then used in the coupling reaction

Biosynthetic Pathway of Rebeccamycin

12: Tryptophan  
![Structure 12](https://i.imgur.com/structure12.png)

13: 7-Chlorotryptophan  
![Structure 13](https://i.imgur.com/structure13.png)

14: 7-Chloroindole-3-pyruvic acid  
![Structure 14](https://i.imgur.com/structure14.png)

4: CCA  
![Structure 4](https://i.imgur.com/structure4.png)

5: Rebeccamycin aglycon  
![Structure 5](https://i.imgur.com/structure5.png)

10: 4'-O-Demethylrebeccamycin  
![Structure 10](https://i.imgur.com/structure10.png)

1: Rebeccamycin  
![Structure 1](https://i.imgur.com/structure1.png)

Fig. 7. Proposed Overall Biosynthetic Pathway of Rebeccamycin.

with 7-chlorotryptophan catalyzed by RebD to afford CCA (4). This dicarboxylic bisindole compound 4 is transformed to the rebeccamycin aglycon (5) via a putative oxidative reaction catalyzed by RebP, and oxidation at the 7-position to the carbonyl by RebC follows. Finally, the rebeccamycin aglycon is glycosylated with D-glucose by RebG (10), and the methylation of the hydroxyl group of the glucose moiety by RebM ends the biosynthesis of rebeccamycin.

Compared with the rebeccamycin biosynthetic pathway, staurosporin biosynthesis in strain TP-A0274 is proposed to be as follows. Tryptophan would be oxidatively deaminated by StaO to 3-indole pyruvic acid, and it would be used for the coupling reaction with tryptophan catalyzed by StaD to produce chromopyrrolic acid. As the last step of the aglycon biosynthesis, StaP would convert chromopyrrolic acid to staurosporine aglycon.

### Acknowledgments

We thank Dr. H. Ikeda at Kitasato University for providing the plasmids and *E. coli* strains, and Dr. T. Dairi at Toyama Prefectural University for valuable comments. We also thank Y. Ikeda at Toyama Prefectural University for obtaining the MS spectra and Nikka Fats & Oils Co., Ltd., for the gift of linseed meal in medium G134. This work was supported in part by a Grant-in-Aid for Scientific Research to H.O. from the Japan Society for the Promotion of Science.

### References

1) Bush, J. A., Long, B. H., Catino, J. J., Bradner, W. T., and Tomita, K., Production and biological activity of rebeccamycin, a novel antitumor agent. *J. Antibiot.*, 40, 668–678 (1987).

2) Omura, S., Sasaki, Y., Iwai, Y., and Takeshima, H., Staurosporine, a potentially important gift from a microorganism. *J. Antibiot.*, 48, 535–548 (1995).

3) Akinaga, S., Sugiyama, K., and Akiyama, T., UCN-01 (7-hydroxystaurosporine) and other indolocarbazole compounds: a new generation of anticancer agents for the new century? *Anticancer Drug Des.*, 15, 43–52 (2000).

4) Kaneko, T., Wong, H., Utzig, J., Schurig, J., and Doyle, T., Water-soluble derivatives of rebeccamycin. *J. Antibiot.*, 43, 125–127 (1990).

5) Ohuchi, T., Ikeda-Araki, A., Watanabe-Sakamoto, A., Kojiri, K., Nagashima, M., Okanishi, M., and Suda, H., Cloning and expression of a gene encoding N-glycosyltransferase (*ngt*) from *Saccarothrix aerocolonigenes* ATCC 39243. *J. Antibiot.*, 53, 393–403 (2000).

6) Kase, H., Iawahashi, K., and Matsuda, Y., K-252a, a potent inhibitor of protein kinase C from microbial origin. *J. Antibiot.*, 39, 1059–1065 (1986).

7) Kojiri, K., Kondo, H., Yoshinari, T., Arakawa, H., Nakajima, S., Satoh, F., Kawamura, K., Okura, A., Suda, H., and Okanishi, M., A new antitumor substance BE-13793C, produced by a streptomycete: taxonomy, fermentation, isolation, structure determination and biological activity. *J. Antibiot.*, 44, 723–728 (1991).

8) Pearce, C. J., Doyle, T. W., Forenza, S., Lam, K. S., and Schroeder, D. R., The biosynthetic origins of rebeccamycin. *J. Nat. Prod.*, 51, 937–940 (1988).

9) Yang, S. W., Lin, L. J., Cordell, G. A., Wang, P., and Corley, D. G., O- and N-methylation in the biosynthesis of staurosporine. *J. Nat. Prod.*, 62,

10) Lam, K. S., Forenza, S., Doyle, T. W., and Pearce, C. J., Identification of indolepyruvic acid as an intermediate of rebeccamycin biosynthesis. *J. Ind. Microbiol.*, 6, 291–294 (1990).

11) Sanchez, C., Butovich, A. I., Brana, F. A., Rohr, J., Mendez, C., and Salas, A. J., The biosynthetic gene cluster for the antitumor rebeccamycin: characterization and generation of indolocarbazole derivatives. *Chem. Biol.*, 9, 519–531 (2002).

12) Onaka, H., Taniguchi, S., Igarashi, Y., and Furumai, T., The analysis of rebeccamycin biosynthetic pathway mediated by the biosynthetic gene disruption experiments. *Nippon Nougeikagaku Kaishi Rinzi Zokan* (in Japanese), 75, 359 (2001).

13) Schafer, A., Tauch, A., Jager, W., Kalinowski, J., Thierbach, G., and Puhler, A., Small mobilizable multi-purpose cloning vectors derived from the *Escherichia coli* plasmids pK18 and pK19: selection of defined deletions in the chromosome of *Corynebacterium glutamicum*. *Gene*, 145, 69–73 (1994).

14) Pang, C. H., Shiiyama, M., Ikeda, H., Tanaka, H., and Omura, S., Cosmid vector for cloning and analysis of *Streptomyces* DNA. *Actinomycetol.*, 8, 21–25 (1994).

15) Mazodier, P., Petter, R., and Thompson, C., Intergeneric conjugation between *Escherichia coli* and *Streptomyces* species. *J. Bacteriol.*, 171, 3583–3585 (1989).

16) Sambrook, J., and Russell, D. W., Molecular cloning, a laboratory manual, third edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (2001).

17) Onaka, H., Tabata, H., Igarashi, Y., Sato, Y., and Furumai, T., Goadsporin, a chemical substance which promotes secondary metabolism and morphogenesis in streptomycetes. I. Purification and characterization. *J. Antibiot.*, 54, 1036–1044 (2001).

18) Kieser, T., Bibb, M. J., Buttner, M. J., Chater, K. F., and Hopwood, D. A., Practical Streptomyces genetics, John Innes Foundation, Norwich, UK (2000).

19) August, P. R., Grossman, T. H., Minor, C., Draper, M. P., MacNeil, I. A., Pemberton, J. M., Call, K. M., Holt, D., and Osburne, M. S., Sequence analysis and functional characterization of the violacein biosynthetic pathway from *Chromobacterium violaceum*. *J. Mol. Microbiol. Biotechnol.*, 2, 513–519 (2000).

20) Hoshino, T., Kojima, Y., Hayashi, T., Uchiyama, T., and Kaneko, K., A new metabolite of tryptophan, chromopyrrolic acid, produced by *Chromobacterium violaceum*. *Biosci. Biotechnol. Biochem.*, 57, 775–781 (1993).

21) Fabre, S., Prudhomme, M., and Rapp, M., Preparation of synthons for the synthesis of protein kinase C inhibitors from rebeccamycin. *Bioorg. Med. Chem.*, 2, 449–452 (1992).

22) Fabre, S., Prudhomme, M., Sancelme, M., and Rapp, M., Indolocarbazole protein kinase C inhibitors from rebeccamycin. *Bioorg. Med. Chem.*, 2, 73–77 (1994).

23) Lam, K. S., McDonald, L. A., Mattei, J., Forenza, S., and Matson, J. A., U.S. Patent 830969 (1994).

24) Hammer, P. E., Burd, W., Hill, D. S., Ligon, J. M., and van Pee, K., Conservation of the pyrrolnitrin biosynthetic gene cluster among six pyrrolnitrin-producing strains. *FEMS Microbiol. Lett.*, 180, 39–44 (1999).

25) Onaka, H., Taniguchi, S., Igarashi, Y., and Furumai, T., Cloning of the staurosporine biosynthetic gene cluster from *Streptomyces* sp. TP-A0274 and its heterologous expression in *Streptomyces lividans*. *J. Antibiot.*, 55, 1063–1071 (2002).

26) Raibekas, A. A., and Massey, V., Primary structure of the snake venom L-amino acid oxidase shows high homology with the mouse B cell interleukin 4-induced Fig. 1 protein. *Biochem. Biophys. Res. Commun.*, 248, 476–478 (1998).

27) Labeda, D. P., Hatano, K., Kroppenstedt, R. M., and Tamura, T., Revival of the genus *Lentzea* and proposal for *Lechevalieria* gen. nov. *Int. J. Syst. Evol. Microbiol.*, 51, 1045–1050 (2001).
