
# Bacterial Metabolism of Naphthalene: Construction and Use of Recombinant Bacteria To Study Ring Cleavage of 1,2-Dihydroxynaphthalene and Subsequent Reactions†

**RICHARD W. EATON* AND PETER J. CHAPMAN**

*Environmental Research Laboratory, U.S. Environmental Protection Agency, Gulf Breeze, Florida 32561*

Received 13 July 1992/Accepted 23 September 1992

The reactions involved in the bacterial metabolism of naphthalene to salicylate have been reinvestigated by using recombinant bacteria carrying genes cloned from plasmid NAH7. When intact cells of *Pseudomonas aeruginosa* PAO1 carrying DNA fragments encoding the first three enzymes of the pathway were incubated with naphthalene, they formed products of the dioxygenase-catalyzed ring cleavage of 1,2-dihydroxynaphthalene. These products were separated by chromatography on Sephadex G-25 and were identified by <sup>1</sup>H and <sup>13</sup>C nuclear magnetic resonance spectroscopy and gas chromatography-mass spectrometry as 2-hydroxychromene-2-carboxylate (HCCA) and trans-*o*-hydroxybenzylidenepyruvate (tHBPA). HCCA was detected as the first reaction product in these incubation mixtures by its characteristic UV spectrum, which slowly changed to a spectrum indicative of an equilibrium mixture of HCCA and tHBPA. Isomerization of either purified product occurred slowly and spontaneously to give an equilibrium mixture of essentially the same composition. tHBPA is also formed from HCCA by the action of an isomerase enzyme encoded by plasmid NAH7. The gene encoding this enzyme, *nahD*, was cloned on a 1.95-kb *KpnI-BglII* fragment. Extracts of *Escherichia coli* JM109 carrying this fragment catalyzed the rapid equilibration of HCCA and tHBPA. Metabolism of tHBPA to salicylaldehyde by hydration and aldol cleavage is catalyzed by a single enzyme encoded by a 1-kb *MluI-StuI* restriction fragment. A mechanism for the hydratase-aldolase-catalyzed reaction is proposed. The salicylaldehyde dehydrogenase gene, *nahF*, was cloned on a 2.75-kb *BamHI* fragment which also carries the naphthalene dihydrodiol dehydrogenase gene, *nahB*. On the basis of the identification of the enzymes encoded by various clones, the gene order for the *nah* operon was shown to be *p, A, B, F, C, E, D*.

---

Naphthalene is the simplest fused polycyclic aromatic hydrocarbon. Information obtained from studies of bacterial degradation of naphthalene has been valuable for understanding and predicting the pathways used in the metabolism of the structurally more complex polycyclic aromatic hydrocarbons and related heterocyclic aromatic compounds. However, much about the bacterial metabolism of naphthalene has remained unclear, in particular the steps in the pathway by which 1,2-dihydroxynaphthalene (Fig. 1, compound III) is metabolized to salicylate.

In the currently accepted naphthalene metabolic pathway (Fig. 1) (3, 42, 53), 1,2-dihydroxynaphthalene is cleaved by a dioxygenase (Fig. 1, enzyme C) to an unstable ring cleavage product (Fig. 1, compound V), which spontaneously recycles to 2-hydroxychromene-2-carboxylate (HCCA) (compound VII). This compound is subsequently converted by means of an isomerase (enzyme D) to cis-*o*-hydroxybenzylidenepyruvate (cHBPA) (compound VII), which is cleaved by an aldolase (enzyme E), yielding salicylaldehyde and pyruvate. An NAD<sup>+</sup>-requiring aldehyde dehydrogenase then transforms salicylaldehyde to salicylate. This pathway for the metabolism of 1,2-dihydroxynaphthalene is based primarily on the work of Barnsley (3) and differs from the pathway proposed earlier by Davies and Evans (7), in which the unstable ring cleavage product (compound V) is rearranged to give cHBPA (compound VIII), which is then metabolized in two sequential steps (hydration followed by aldol cleavage to salicylaldehyde). Direct evidence for the hydratase (enzyme O) or intermediate IX was not obtained; their possible involvement was proposed on the basis of the structural requirements of substrates undergoing aldolase cleavage (a hydroxyl group located β to a carbonyl). The Davies-Evans pathway does not include HCCA as an intermediate.

Davies and Evans (7) and Barnsley (3) reached these different conclusions despite taking similar experimental approaches. These workers incubated cell extracts with low concentrations of 1,2-dihydroxynaphthalene for a short time and then rapidly took steps to purify and stabilize the product. Davies and Evans made the perchlorate derivative, which they crystallized, while Barnsley separated the product from cell extracts by chromatography on Sephadex G-25 followed by freeze-drying. Both Davies and Evans (7) and Barnsley (3) obtained HCCA. Barnsley identified this compound as the initial product of ring cleavage, but Davies and Evans recognized it as the hemiacetal (actually the hemiketal) of cHBPA and considered it to be an artifact of their isolation procedure. Both Davies and Evans (7) and Barnsley (3) demonstrated that HCCA spontaneously isomerizes in water at a neutral pH to give an equilibrium mixture with cHBPA and that the equilibrium is 100% in favor of cHBPA at pH 11. The cHBPA formed could be metabolized by extracts of induced cells to salicylaldehyde and pyruvate. Barnsley also partially purified from cell extracts an enzyme (isomerase) (Fig. 1, enzyme D) that transformed HCCA to cHBPA. However, Davies and Evans did not describe such an enzyme.

Both of these studies and a subsequent study by Patel and Barnsley (42) using purified 1,2-dihydroxynaphthalene dioxygenase were hampered by their use of the unstable sub-

* Corresponding author.
† Contribution 795 from the Environmental Research Laboratory, U.S. Environmental Protection Agency, Gulf Breeze, Fla.

#### Figure 1: Pathways for the metabolism of naphthalene to salicylate

![Chemical Reaction Diagram](attachment:chemical-reaction-diagram.png)

**FIG. 1.** Pathways for the metabolism of naphthalene to salicylate. See references 3, 5, 7, 17, 30, 31, 42, 52, and 53. Chemical designations:
- I, naphthalene;
- II, *cis*-1,2-dihydroxy-1,2-dihydronaphthalene (naphthalene dihydrodiol);
- III, 1,2-dihydroxynaphthalene;
- IV, 1,2-naphthoquinone;
- V, 2-hydroxy-4-(2′-oxo-3,5-cyclohexadienyl)-buta-2,4-dienoate;
- VI, tHBPA;
- VII, HCCA;
- VIII, cHBPA;
- IX, 2-oxo-4-hydroxy-4-(2′-hydroxyphenyl)-butyrate;
- X, salicylaldehyde;
- XI, salicylate.

Enzyme designations:
- A, naphthalene-1,2-dioxygenase;
- B, 1,2-dihydronaphthalene dehydrogenase;
- C, 1,2-dihydroxynaphthalene dioxygenase;
- D, 2-hydroxychromene-2-carboxylate isomerase;
- E, *cis*-o-hydroxybenzylidenepyruvate aldolase;
- O, hydratase;
- F, salicylaldehyde dehydrogenase.

---

### Text Content

strate 1,2-dihydroxynaphthalene, which is rapidly and spontaneously oxidized in water to 1,2-naphthoquinone (Fig. 1, compound IV). The short incubation times required resulted in the use of large amounts of enzyme but yielded products in quantities that were insufficient for rigorous identification. To overcome this problem, we took a genetic approach; genes encoding the metabolism of naphthalene to salicylate were cloned from the naphthalene catabolic plasmid NAH7. Subclones that carry genes encoding the first three enzymes of the metabolic pathway were obtained; bacteria carrying these genes converted the stable substrate, naphthalene, to ring cleavage products which could be accumulated in large quantities and identified. Other subclones encode the subsequent enzymes of the pathway. The use of these clones to reexamine the pathway for bacterial metabolism of 1,2-dihydroxynaphthalene to salicylate and to establish a revised gene order is described in this paper.

#### MATERIALS AND METHODS

##### Bacterial plasmids and strains
The bacterial plasmids used in this study are listed in Table 1. *Pseudomonas putida* PpG1064 (*trpB615 nah7*) (52) was the source of plasmid NAH7. *Pseudomonas aeruginosa* PAO1 (48) was chosen to be the host for recombinant plasmids in several experiments because it is relatively sensitive to chloramphenicol and allowed adequate expression of the cloned genes. *Escherichia coli* JM109 \[recA endA1 gyrA96 thi hsdR17 supE44 relA1 Δ(lac-proAB) (F' traD36 proAB lacI<sup>q</sup>ZΔM15)\] (51) was routinely used as a host in cloning experiments. *E. coli*

AA102F' lac/low \[recA pro thi supE endA hsdR<sub>K</sub> Tet<sup>r</sup> Δ1(gal-chlD-pgl-att λ) (F' traD36 proAB lacI<sup>F</sup>ZΔM15)\] (1, 2, 20) was the host in deletion mutagenesis experiments and was obtained with pAA-PZ718 from Gold Biotechnology, Inc., St. Louis, Mo. Plasmids pUCBM20 and pUCBM21 were obtained from Boehringer-Mannheim Corp., Indianapolis, Ind., and plasmids pMMB207 and pMMB277 were obtained from M. Bagdasarian.

##### Media
Luria-Bertani (LB) medium (8) was used for the cultivation of bacteria except where indicated otherwise. The minimal medium was R medium (14) containing trace elements (23) and 20 mM sodium lactate. The media were solidified with 1.5% Bacto Agar (Difco Laboratories, Detroit, Mich.).

##### Chemicals and enzymes
Naphthalene, salicylaldehyde, α-naphthol, β-naphthol, 1,2-naphthoquinone, 1,2-dihydroxynaphthalene, and trans-*o*-hydroxycinnamic acid were obtained from Aldrich Chemical Co., Milwaukee, Wis. Potassium trans-benzylidenepyruvate and trans-*o*-methoxybenzylidenepyruvic acid were synthesized by the condensation of pyruvic acid with benzaldehyde (46) and *o*-anisaldehyde (47), respectively; their identities and trans stereochemistry were confirmed by proton and <sup>13</sup>C nuclear magnetic resonance (NMR) spectroscopy. trans-4-(3-Hydroxy-2-thianaphthyl)-2-oxo-but-3-enoate and trans-4-(3-hydroxy-2-benzofuranyl)-2-oxobut-3-enoate were produced by biotransformation of dibenzothiophene and dibenzofuran, respectively, by *P. aeruginosa* PAO1(pRE654) (11), while trans-4-(3-hydroxy-2-thienyl)-2-oxobut-3-enoate was a product of biotransformation of benzothiophene by *P. putida* RE204 (13, 15).

---

### Notes
- The chemical reaction diagram has been described in the text above.
- References are cited within the text as numbers in parentheses (e.g., (52)).
- The figure and its description are included at the beginning for clarity.### TABLE 1. Plasmids

| Plasmid(s) | Description | Source or reference(s) |
|------------|-------------|-----------------------|
| NAH7       | Encodes naphthalene degradation, 83 kb | 53            |
| pBR322     | Ap<sup>r</sup> Tc<sup>r</sup> | 4             |
| pUC19      | Ap<sup>r</sup>, multiple cloning site in *lacZα lacI<sup>+</sup>* | 51            |
| pMMB207    | Cm<sup>r</sup> Mob<sup>+</sup>, derived from RSF1010, *lacI<sup>q</sup>*, multiple cloning site downstream from P<sub>tac</sub><sup>a</sup> | 39            |
| pMMB277    | Cm<sup>r</sup> Mob<sup>+</sup>, derived from RSF1010, *lacI<sup>q</sup>*, multiple cloning site in *lacZα* downstream from P<sub>tac</sub> | 39            |
| pRK2013    | Ap<sup>r</sup> Km<sup>r</sup>, RK2 origin, helper plasmid for mobilization | 9             |
| pAA-PZ718  | pAA-PZ1 with Tc<sup>r</sup> gene deleted and replaced with *lacZα* carrying the mp18 multiple cloning site, Gal<sup>s</sup> Ap<sup>r</sup> Cm<sup>r</sup> | 1, 2, 20, 43 |
| pUCBM20    | Derived from pUC18 with the following additional cloning sites inserted into *lacZα*: NcoI, EcoRV, ApaI, SacII, XmaIII, NotI, BfrI, MluI | 53            |
| pUCBM21    | Identical to pUCBM20 but with the multiple cloning site reversed | 53            |
| pLV59      | Encodes EcoRI restriction endonuclease and temperature-sensitive EcoRI methylase, Cm<sup>r</sup>, positive selection cloning vector | 41            |
| pRE621     | 11-kb EcoRI-HindIII fragment and 3-kb HindIII fragment from NAH7 inserted into pBR322, Ap<sup>r</sup> Ind<sup>+</sup><sup>b</sup> | This study   |
| pRE630     | 11-kb EcoRI-HindIII fragment from pRE621 inserted into pMMB277, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE644     | 11-kb EcoRI-HindIII fragment from pRE621 inserted into pAA-PZ718, Ap<sup>r</sup> Cm<sup>r</sup> Ind<sup>+</sup><sup>c</sup> | This study   |
| pRE654     | 10-kb EcoRI-ClaI fragment from pRE630 inserted into pBR322, Ap<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE657     | 10-kb EcoRI-HindIII fragment from pRE654 inserted into pMMB277, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE668     | 7.35-kb EcoRI-BglII fragment from pRE621 inserted into pMMB277, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE669     | 5.25-kb EcoRI-BamHI fragment and 2.75-kb BamHI fragment (map coordinates 0 to 8.0) from pRE621 inserted into pMMB277, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE670     | 4.9-kb EcoRI-SalI fragment and 4.85-kb SalI fragment (map coordinates 0 to 9.75) from pRE621 inserted into pMMB277, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE672     | 2.75-kb BamHI fragment (map coordinates 5.25 to 8.0) from pRE654 inserted into pMMB277, Cm<sup>r</sup> Ind<sup>−</sup> | This study   |
| pRE674 to pRE679 | Deletion mutants generated from pRE644, Ap<sup>r</sup> Ind<sup>+</sup><sup>c</sup> | This study   |
| pRE681     | 6.25-kb EcoRI-NheI fragment from pRE621 inserted into EcoRI-XbaI-digested pMMB207, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE694     | pRE630 digested with ClaI, ends filled and religated, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE695     | pRE630 digested with NheI, ends filled and religated, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE696     | 10.7-kb EcoRI-StuI fragment from pRE621 inserted into EcoRI-SmaI-digested pMMB277, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE697     | EcoRI-digested pRE674 inserted into pMMB277, Cm<sup>r</sup> Ap<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE698     | EcoRI-digested pRE675 inserted into pMMB277, Cm<sup>r</sup> Ap<sup>r</sup> Ind<sup>+</sup> | This study   |
| pRE699     | 3-kb BamHI-HindIII fragment (map coordinates 8.0 to 11.0) from pRE621 inserted into pMMB277, Cm<sup>r</sup> Ind<sup>−</sup> | This study   |
| pRE701     | 1-kb MluI-HindIII fragment (map coordinates 9.7 to 10.7) from pRE696 inserted into pUCBM20, Ap<sup>r</sup> Ind<sup>−</sup> | This study   |
| pRE712     | 6.5-kb MluI fragment (map coordinates 9.7 to 16.2) from NAH7 inserted into pUCBM20, Ap<sup>r</sup> Ind<sup>−</sup> | This study   |
| pRE714     | 4.7-kb BglII fragment (map coordinates 7.35 to 12.05) from NAH7 inserted into pLV59, Cm<sup>r</sup> Ind<sup>−</sup> | This study   |
| pRE718     | 1.95-kb KpnI-BglII fragment (map coordinates 10.1 to 12.05) from pRE714 inserted into KpnI-BamHI-digested pUCBM21, Ap<sup>r</sup> Ind<sup>−</sup> | This study   |
| pRE720     | 1.35-kb StuI-BglII fragment (map coordinates 10.7 to 12.05) from pRE714 inserted into EcoRV-BamHI-digested pUCBM21, Ap<sup>r</sup> Ind<sup>−</sup> | This study   |
| pRE731     | 10-kb EcoRI-ClaI fragment (map coordinates 0 to 10) from pRE654 and 2.05-kb ClaI-BglII fragment (map coordinates 10 to 12.05) from pRE712 inserted into EcoRI-BamHI-digested pMMB277, Cm<sup>r</sup> Ind<sup>+</sup> | This study   |

#### Footnotes:
- <sup>a</sup> See reference 22 for a description of RSF1010.
- <sup>b</sup> Ind<sup>+</sup>, encodes the ability to convert indole to indigo.
- <sup>c</sup> See Fig. 3.

---

### Restriction Endonucleases and T4 DNA Ligase
Restriction endonucleases and T4 DNA ligase were obtained from New England BioLabs, Beverly, Mass., Bethesda Research Laboratories, Gaithersburg, Md., and Boehringer-Mannheim Corp. and were used according to the manufacturers' instructions.

Proteins were determined by using the BCA protein assay reagent (Pierce, Rockford, Ill.) and bovine serum albumin as the standard.

#### Preparation, Analysis, and Cloning of DNA
Plasmid NAH7 DNA was isolated from *P. putida* PpG1064 by the method of Hansen and Olsen (26) followed by centrifugation to equilibrium through a cesium chloride-ethidium bromide gradient. Plasmid DNA was isolated from *E. coli* by the method of Holmes and Quigley (29) when recombinants were screened and by the procedure of Clewell and Helinski (6) for large-scale preparations.

Restriction enzyme-digested DNA was analyzed by electrophoresis through 0.7% agarose gels in Tris-borate-EDTA buffer (8). Gels were stained with ethidium bromide, visualized on a UV transilluminator, and photographed through a Kodak no. 22 Wratten gelatin filter.

Ligated DNA was transformed into *E. coli* recipients (36). Recombinant bacteria carrying DNA fragments encoding naphthalene-1,2-dioxygenase were identified by spreading transformants onto LB agar supplemented with antibiotics (100 μg of ampicillin per ml or 30 μg of chloramphenicol per ml) and 1 mM indole. Bacteria carrying the cloned naphthalene dioxygenase genes produce indigo from indole (18). Such bacteria formed dark blue to olive green colonies on indole-containing media and were readily identified. DNA fragments not encoding naphthalene dioxygenase were cloned as insertions into the *lacZα* gene of the vector (pUCBM20, pUCBM21, or pMMB277), and *E. coli* JM109 carrying these fragments appeared white on media containing 0.25 mM isopropyl-β-D-thiogalactopyranoside (IPTG) and 0.02% 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside.

Strain JM109 colonies carrying pRE712 and pRE714 were further characterized.
identified by colony hybridization (12, 21), using the gel-purified (10), ${}^{32}$P-labelled 1-kb *MluI-StuI* fragment from pRE701 as the probe, as previously described (12).

Recombinant plasmids were introduced into *P. aeruginosa* PAO1 from strain JM109 on LB agar in triparental matings involving these strains and strain JM109 carrying the mobilizing plasmid, pRK2013, on LB agar. Transconjugants were selected on R medium agar supplemented with lactate, 90 μg of chloramphenicol per ml, and 1 mM indole.

Cultivation of organisms. Organisms were often prepared on a large scale for transformation of naphthalene or preparation of cell extracts. In general, an overnight culture grown on LB medium supplemented with the appropriate antibiotic was used to inoculate 50 volumes of the same medium in an Erlenmeyer flask (typically, 500 ml in a 2-liter flask). After incubation with shaking at 200 rpm for 2 h at 37°C, IPTG was added to a concentration of 1 mM, and incubation was continued for 3 to 4 h. Cells were harvested by centrifugation and were resuspended in minimal medium lacking an added carbon source. Cell suspensions were then either chilled on ice for the preparation of extracts or washed by a second centrifugation before incubation with naphthalene.

Preparation of cell extracts. Cell extracts were prepared by passing cell suspensions twice through a French pressure cell at 14,000 to 20,000 lb/in². Particulates were removed by centrifugation at 47,800 × g for 40 min.

Transformation of naphthalene by *P. aeruginosa* PAO1 carrying recombinant plasmids. Washed cells of strain PAO1 carrying recombinant plasmids were normally resuspended in a volume of R medium that was one-fourth the volume of LB medium in which they were grown. After finely ground naphthalene was added, the cells were incubated at 30 or 37°C with shaking at 200 rpm for 20 to 40 h. Following incubation, the cells and residual naphthalene were removed by centrifugation.

Characterization of naphthalene transformation products. Proton NMR and ${}^{13}$C NMR spectra were obtained with a General Electric model QE Plus spectrometer at 300 and 75 MHz, respectively, by J. Gurst. Assignment of chemical shifts was aided by automated chemical shift correlated spectroscopy and attached proton test analyses.

Mass spectra of trimethylsilyl (TMS) derivatives were obtained by W. Gilliam, who used a Hewlett-Packard model 5988A mass spectrometer coupled to an HP model 5890 gas chromatograph. Derivatized compounds were injected at 240°C onto a type HP-S capillary column (cross-linked 5% phenylmethylsilicone; 25 m by 0.2 mm; film thickness, 0.11 μm) operated with helium as the carrier gas at a flow rate of 0.37 m/s by using a temperature program that increased the oven temperature from 50 to 250°C over a period of 21.5 min.

UV-visible spectra were recorded with a Perkin-Elmer Lambda 6 spectrophotometer and were prepared for publication by using CorelDRAW! (Corel Corp., Ottawa, Ontario, Canada).

Salicylic acid was identified by high-pressure liquid chromatography, using a 5-μm Hypersil ODS (Hewlett-Packard) reverse-phase column (100 by 2.1 mm) and CH₃CN–50 mM phosphate (pH 3.5) as the solvent in a gradient in which the CH₃CN concentration increased from 10 to 80% at a rate of 3%/min. Authentic salicylic acid, which was detected by using a diode array detector, eluted at 2.4 min and had absorbance maxima at 230 and 296 nm. The high-pressure liquid chromatography analysis was carried out by B. Blattman and M. Downey.

1,2-Dihydroxy-1,2-dihydronaphthalene (naphthalene dihydrodiol) was identified by its acid-catalyzed dehydration to a mixture of α- and β-naphthols. This process was followed spectrophotometrically by the rapid change in the spectrum of the dihydrodiol ($\lambda_{\text{max}}$, 262 nm) to a spectrum that closely resembles the spectrum of its major dehydration product, α-naphthol ($\lambda_{\text{max}}$, 287 nm) following addition of HCl (final concentration, 0.12 N). When chromatographed on Silica Gel 60 F-254 thin-layer plates (EM Laboratories, Elmsford, N.Y.) with CH₂Cl₂ as the solvent, the dihydrodiol had an $R_f$ of 0.03, while the acid-treated dihydrodiol produced two spots, a major one migrating with the same $R_f$ as α-naphthol (0.45) and a minor spot with the same mobility as β-naphthol ($R_f$ 0.34). The dihydrodiol was a substrate for an NAD⁺-linked dehydrogenase found in extracts of certain clones. Naphthalene dihydrodiol dehydrogenase activity was followed spectrophotometrically by observing the change in the spectrum of the dihydrodiol ($\lambda_{\text{max}}$, 262 nm) in reaction mixtures containing NAD⁺ and cell extracts (through 1,2-dihydroxynaphthalene) to the spectrum of 1,2-naphthoquinone ($\lambda_{\text{max}}$, 251 nm).

---

**RESULTS**

Cloning naphthalene catabolic genes. Plasmid NAH7 from *P. putida* G7 carries genes that encode the complete degradation of naphthalene. These genes are grouped in two operons (52, 53). Most of the operon encoding the metabolism of naphthalene to salicylate was initially cloned by digesting NAH7 with EcoRI and HindIII and ligating it to the similarly digested plasmid cloning vector pBR322. The ligation mixture was transformed into *E. coli* JM109, and transformants were selected on LB agar plates containing 100 μg of ampicillin per ml and 1 mM indole. In this way an indigo-producing colony of strain JM109 was identified that carried both an 11-kb EcoRI-HindIII fragment and a 3-kb HindIII fragment from NAH7 inserted into pBR322 (designated pRE621). The 11-kb EcoRI-HindIII fragment was subsequently subcloned in broad-host-range plasmid vector pMMB277, yielding pRE630. A restriction enzyme cleavage map of pRE630 was then generated (Fig. 2). The *nahC* gene, which encodes 1,2-dihydroxynaphthalene dioxygenase, had been sequenced previously (27), so that its exact location, extending from 2.15 to 3.05 kb to the left of the HindIII site, was known. The general locations of *nahA* and *nahB* were also known (52, 53), as well as that of the gene encoding the enzyme that acts on the ring cleavage product; the latter gene was located to the right of *nahC*. *nahF* was also thought to be located to the right of *nahC* (53). Fragments cloned in this and subsequent experiments were oriented so that transcription initiated at a vector-specified promoter would continue from left to right. Several subclones (pRE696, pRE630, and pRE670) and deletions (pRE697 and pRE698) (see below) that eliminated DNA between *nahC* and the HindIII site were obtained, with the expectation that these subclones and deletions would leave only the genes that encode the first three pathway enzymes. Three additional cloned fragments extended from EcoRI to the left of *nahC* and included *nahA* (pRE669, pRE671, and pRE681). Frame-shift mutations were introduced into pRE630 by digesting it with NheI (pRE695) or ClaI (pRE694), filling in with deoxy-nucleoside triphosphates, and religating the blunt ends. This resulted in the loss of the restriction site and the addition of 4 bp (pRE695) or 2 bp (pRE694). Smaller DNA fragments not carrying *nahA* were inserted into cloning sites in *lacZa* in pMMB277 (pRE672 and pRE699), pUCBM20 (pRE710 and pRE714), and pUCBM21 (pRE718 and pRE720). Strain JM109 carrying these recombinant plasmids produced white

#### Image Description and Text Extraction

The image contains a restriction map of a 12.05-kb DNA segment extending from an EcoRI site on the left to a BglII site on the right. The map includes the locations of genes encoding pathway enzymes, indicated at the top. Below the map, DNA fragments that have been subcloned or deletions obtained are shown, with plasmid names indicated on the right.

---

### Restriction Map Details

- **Genes**: nah genes
- **Restriction Sites**:
  - A: MluI, XhoI, KpnI, PstI, MluI, SalI, BamHI, SmaI, XhoI, NheI
  - B: SacI, BgII, KpnI, SacI, PstI, SacI, BamHI
  - C: MluI, SalI, ClaI, KpnI, StuI, HindIII, HindIII, XhoI, BgII
- **Plasmids**:
  - pRE695 (ΔNheI)
  - pRE694 (ΔClaI)
  - pRE630
  - pRE696
  - pRE657
  - pRE670
  - pRE698
  - pRE697
  - pRE669
  - pRE668
  - pRE681
  - pRE672
  - pRE699
  - pRE701
  - pRE714
  - pRE718
  - pRE720
  - pRE731

---

### Figure Caption

**FIG. 2.** Plasmid NAH7 DNA encoding the metabolism of naphthalene to salicylate: restriction map of a 12.05-kb DNA segment extending from an EcoRI site on the left to a BglII site on the right. The locations of the genes encoding pathway enzymes (see legend to Fig. 1) are indicated at the top. `nahA` includes three or more genes that encode the components of naphthalene-1,2-dioxygenase (16, 17, 24, 25, 35, 52, 53). Below the map, DNA fragments that have been subcloned or deletions that have been obtained are shown. Plasmid names are indicated on the right.

---

### Text Below the Figure

#### Colonies on LB Agar

Colonies on LB agar containing the appropriate antibiotic, IPTG, and 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside were identified by hybridization to the 1-kb MluI-StuI fragment from pRE701 (map coordinates 9.7 to 10.7). Plasmids pRE718 and pRE720 were constructed by subcloning from pRE714 into pUCBM21. Genes `nahABFCED` were cloned together on a single plasmid (pRE731) by ligating a 10-kb EcoRI-ClaI fragment from PRE654 with a 2.05-kb ClaI-BglII fragment from pRE712 to pMMB277.

#### Isolation of Deletion Mutant Plasmids

The isomerase gene, `nahD`, was expected to be located directly to the right of `nahC` (52), while the SalI site at map coordinate 9.75 is 900 bp to the right of `nahC`. To eliminate `nahD` from clones intended to have only `nahA`, `nahB`, and `nahC`, deletions that eliminated most of the DNA between `nahC` and the SalI site were isolated. The 11-kb EcoRI-HindIII fragment from pRE630 was inserted into deletion vector pAA-PZ718, yielding pRE644 (Fig. 3). Deletion mutants were isolated according to the supplier's instructions (1, 2, 20, 43). The galactose operon is deleted from host strain E. coli AA102F' lac/low, which also harbors a plasmid that reduces the copy number of pAA-pZ718 to one or two copies per cell. When this strain carrying pAA-PZ718 derivatives, such as pRE644, is grown on media containing galactose, it accumulates a lethal product, UDP-galactose. Only bacteria made resistant to galactose by IS1-L-induced deletion of `galK` form colonies on MacConkey agar containing galactose and ampicillin. Some of these colonies carried deletions extending past the `galKTE'` genes and into the fragment cloned in pRE644. In plasmids pRE674 to pRE679, the deletion endpoints are located between the HindIII site and the `nahC` gene; these endpoints are indicated on the map (Fig. 3). Two of these plasmids, pRE674 and pRE675, were subsequently inserted whole into pMMB277 as EcoRI fragments, yielding pRE697 and pRE698, respectively (Fig. 2).

#### Naphthalene Transformations

Naphthalene transformations by *P. aeruginosa* PAO1 carrying recombinant plasmids. When washed cells of *P. aeruginosa* PAO1 carrying pRE630 or a slightly smaller plasmid, pRE696, were incubated for 16 h with naphthalene, they transformed this compound into a product which was extracted from acidified supernatants by using diethyl ether and subsequently identified as salicylic acid. This initially suggested that the 11-kb HindIII-EcoRI and 10.7-kb EcoRI-StuI fragments encode all of the enzymes required for the transformation of naphthalene to salicylate. As shown below, this fragment actually lacks the `nahD` gene, which encodes an isomerase catalyzing the isomerization of HCCA to trans-o-hydroxybenzylidenepyruvate (tHBPA). This isomerization occurs spontaneously, although at a slower rate than the enzyme-catalyzed reaction, so that although

---

### Restriction Map of Plasmid NAH7 DNA

**Figure 2.** Plasmid NAH7 DNA encoding the metabolism of naphthalene to salicylate: restriction map of a 12.05-kb DNA segment extending from an EcoRI site on the left to a BglII site on the right. The locations of the genes encoding pathway enzymes (see legend to Fig. 1) are indicated at the top. `nahA` includes three or more genes that encode the components of naphthalene-1,2-dioxygenase (16, 17, 24, 25, 35, 52, 53). Below the map, DNA fragments that have been subcloned or deletions that have been obtained are shown. Plasmid names are indicated on the right.

#### Restriction Sites and Plasmids

- **Genes**: nah genes
- **Restriction Sites**:
  - A: MluI, XhoI, KpnI, PstI, MluI, SalI, BamHI, SmaI, XhoI, NheI
  - B: SacI, BgII, KpnI, SacI, PstI, SacI, BamHI
  - C: MluI, SalI, ClaI, KpnI, StuI, HindIII, HindIII, XhoI, BgII
- **Plasmids**:
  - pRE695 (ΔNheI)
  - pRE694 (ΔClaI)
  - pRE630
  - pRE696
  - pRE657
  - pRE670
  - pRE698
  - pRE697
  - pRE669
  - pRE668
  - pRE681
  - pRE672
  - pRE699
  - pRE701
  - pRE714
  - pRE718
  - pRE720
  - pRE731

#### Colonies on LB Agar

Colonies on LB agar containing the appropriate antibiotic, IPTG, and 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside were identified by hybridization to the 1-kb MluI-StuI fragment from pRE701 (map coordinates 9.7 to 10.7). Plasmids pRE718 and pRE720 were constructed by subcloning from pRE714 into pUCBM21. Genes `nahABFCED` were cloned together on a single plasmid (pRE731) by ligating a 10-kb EcoRI-ClaI fragment from PRE654 with a 2.05-kb ClaI-BglII fragment from pRE712 to pMMB277.

#### Isolation of Deletion Mutant Plasmids

The isomerase gene, `nahD`, was expected to be located directly to the right of `nahC` (52), while the SalI site at map coordinate 9.75 is 900 bp to the right of `nahC`. To eliminate `nahD` from clones intended to have only `nahA`, `nahB`, and `nahC`, deletions that eliminated most of the DNA between `nahC` and the SalI site were isolated. The 11-kb EcoRI-HindIII fragment from pRE630 was inserted into deletion vector pAA-PZ718, yielding pRE644 (Fig. 3). Deletion mutants were isolated according to the supplier's instructions (1, 2, 20, 43). The galactose operon is deleted from host strain E. coli AA102F' lac/low, which also harbors a plasmid that reduces the copy number of pAA-pZ718 to one or two copies per cell. When this strain carrying pAA-PZ718 derivatives, such as pRE644, is grown on media containing galactose, it accumulates a lethal product, UDP-galactose. Only bacteria made resistant to galactose by IS1-L-induced deletion of `galK` form colonies on MacConkey agar containing galactose and ampicillin. Some of these colonies carried deletions extending past the `galKTE'` genes and into the fragment cloned in pRE644. In plasmids pRE674 to pRE679, the deletion endpoints are located between the HindIII site and the `nahC` gene; these endpoints are indicated on the map (Fig. 3). Two of these plasmids, pRE674 and pRE675, were subsequently inserted whole into pMMB277 as EcoRI fragments, yielding pRE697 and pRE698, respectively (Fig. 2).

#### Naphthalene Transformations

Naphthalene transformations by *P. aeruginosa* PAO1 carrying recombinant plasmids. When washed cells of *P. aeruginosa* PAO1 carrying pRE630 or a slightly smaller plasmid, pRE696, were incubated for 16 h with naphthalene, they transformed this compound into a product which was extracted from acidified supernatants by using diethyl ether and subsequently identified as salicylic acid. This initially suggested that the 11-kb HindIII-EcoRI and 10.7-kb EcoRI-StuI fragments encode all of the enzymes required for the transformation of naphthalene to salicylate. As shown below, this fragment actually lacks the `nahD` gene, which encodes an isomerase catalyzing the isomerization of HCCA to trans-o-hydroxybenzylidenepyruvate (tHBPA). This isomerization occurs spontaneously, although at a slower rate than the enzyme-catalyzed reaction, so that although

---

**FIG. 3.** 11-kb EcoRI-HindIII fragment (see Fig. 2) inserted in deletion vector pAA-PZ718 to give pRE644. The pAA-PZ718 portion of the plasmid is indicated by the light line; the NAH7 portion is indicated by the heavy line (see text).

HCCA accumulates transiently, it is eventually converted to salicylate.

Strain PAO1 carrying pRE681 or pRE695 accumulated 1,2-dihydroxy-1,2-dihydronaphthalene (Fig. 1, compound II) when it was incubated with naphthalene. This dihydrodiol served as a substrate for the NAD⁺-requiring dihydrodiol dehydrogenase in cell extracts of strain PAO1 carrying pRE630 or pRE668 or strain JM109 carrying pRE672 (data not shown).

Plasmids pRE657, pRE670, pRE697, pRE698, and pRE694 all carry DNA segments that begin at the EcoRI site and extend to the right of `nahC` but do not include `nahE` or `nahD`. When strain PAO1 cells carrying any of these plasmids were incubated with naphthalene, they all gave rise to essentially identical spectra (Fig. 4). The spectrum recorded after 20 min of incubation of strain PAO1(pRE654) with naphthalene (Fig. 4A) is similar to the spectrum reported for HCCA (3, 7), with a λ_max at 256 nm and a smaller peak at 296 nm. With prolonged incubation, the spectrum (Fig. 4B) changed; a new peak appeared at 340 nm and there was an increase in the A₂₉₆ relative to the A₂₅₆. These changes result from the formation of an equilibrium mixture of HCCA and tHBPA, as shown below.

The products of this transformation were prepared on a large scale by inoculating eight 500-ml volumes of LB medium containing chloramphenicol (90 μg/ml) with 10-ml volumes of overnight cultures of strain PAO1(pRE654) grown on the same medium. After 2 h of incubation at 37°C, 1 mM IPTG was added, and the cultures were incubated for an additional 4 h, after which they were harvested, washed with R medium (14), and resuspended in 1 liter of the same medium to which 1 g of finely ground naphthalene was added. The cells were incubated with naphthalene for 40 h and then removed by centrifugation, and the supernatant was freeze-dried.

As much of the dried residue as possible was redissolved

---

### Figures

#### FIG. 3.
- **Description:** Circular map of the plasmid pRE644 (21 kb).
- **Key Features:**
  - Origin of replication.
  - Various restriction sites: PstI, ApR, IS1L, CmR, IS1R', K, T, E', Sacl, HindIII, Stul, Clal, Sall, nahC.
  - Deletion endpoints marked at positions 674, 675, 677, 678, 679.
  - Light line indicates the pAA-PZ718 portion.
  - Heavy line indicates the NAH7 portion.

#### FIG. 4.
- **Title:** Transformation of naphthalene by strain PAO1 (pRE657).
- **Description:**
  - Washed, IPTG-induced cells of strain PAO1(pRE657) were resuspended in minimal medium to which solid, powdered naphthalene was added, and then the preparations were incubated at 37°C.
  - Samples were taken at various times, cells were removed by centrifugation, and the spectra of supernatants, diluted 1/20 in 50 mM potassium-sodium phosphate buffer (pH 7), were recorded.
  - **Panel A:**
    - Spectrum a, time zero; spectrum b, 10 min; spectrum c, 20 min; spectrum d, 30 min.
  - **Panel B:**
    - Spectrum a, time zero; spectrum d, 30 min; spectrum e, 60 min; spectrum f, 120 min; spectrum g, 180 min; spectrum h, 240 min.

---

### Additional Notes
- The figures show spectral data over wavelengths ranging from 200 to 400 nanometers.
- The chemical structures of compounds involved in the transformation are also depicted.

#### Figure 5: Sephadex G-25 Chromatographic Separation

![Chromatogram](https://i.imgur.com/1234567.png)

**Figure 5.** Sephadex G-25 chromatographic separation of the products of naphthalene transformation by strain PAO1(pRE657). Transformation products in 20 ml of water were loaded onto a Sephadex G-25 column (87 by 5 cm; 1,707 ml) and eluted with water at \(4^\circ \mathrm{C}\). Fractions (13.7 ml) were collected and diluted in 50 mM potassium-sodium phosphate buffer (pH 7), and their UV-visible spectra were recorded. Fraction 125 is 1 column volume.

---

### Figure 6: \({}^1\mathrm{H}\) and \({}^{13}\mathrm{C}\) NMR Data for HCCA and tHBPA

| Chemical Shift (ppm); Coupling Constant, \(J\) (Hz). |
|-----------------------------------------------------|
| **2-hydroxychromene-2-carboxylate**                  |
| ![Structure 1](https://i.imgur.com/ABCDEFG.png)     |
| **trans-o-hydroxybenzylidenepyruvate**               |
| ![Structure 2](https://i.imgur.com/HIJKLMN.png)     |

| \({}^1\mathrm{H}\) | 2-hydroxychromene-2-carboxylate | trans-o-hydroxybenzylidenepyruvate |
|-------------------|----------------------------------|------------------------------------|
| \(a\)             | \(6.75\) (d), \(J = 8.26\)      | \(6.94\) (d), \(J = 7.5\)         |
| \(b\)             | \(7.13\) (t), \(J = 6.15, 5.17\) | \(7.19\) (t), \(J = 7.7, 1.5\)    |
| \(c\)             | \(6.85\) (t), \(J = 7.8, 0.8, 7.1, 0.4\) | \(6.79\) (t), \(J = 7.6, 7.2\)   |
| \(d\)             | \(7.11\) (d), \(J = 7.3\)       | \(7.53\) (d), \(J = 7.8, 1.5\)    |
| \(e\)             | \(6.57\) (d), \(J = 9.8\)      | \(7.74\) (d), \(J = 16.5\)       |
| \(f\)             | \(5.66\) (d), \(J = 9.8\)      | \(6.72\) (d), \(J = 16.5\)       |

| \({}^{13}\mathrm{C}\) | 2-hydroxychromene-2-carboxylate | trans-o-hydroxybenzylidenepyruvate |
|----------------------|----------------------------------|------------------------------------|
| \(1\)                | \(123.09\)                      | \(121.76\)                       |
| \(2\)                | \(152.09\)                      | \(157.45\)                       |
| \(3\)                | \(115.91\)                      | \(116.61\)                       |
| \(4\)                | \(125.79\) or \(126.47\)        | \(131.55\)                       |
| \(5\)                | \(120.36\) or \(120.58\)        | \(119.27\)                       |
| \(6\)                | \(125.79\) or \(126.47\)        | \(128.27\)                       |
| \(7\)                | \(128.69\)                      | \(139.79\)                       |
| \(8\)                | \(120.36\) or \(120.58\)        | \(124.33\)                       |
| \(9\)                | \(94.34\)                       | \(198.1\)                        |
| \(10\)               | \(170.9\)                       | \(169.88\)                       |

**Figure 6.** \({}^1\mathrm{H}\) and \({}^{13}\mathrm{C}\) NMR data for HCCA and tHBPA in \(d_6\)-dimethyl sulfoxide. d, doublet; t, triplet.

---

### Text Content

Retarded to varying extents as a result of this affinity (44). Two compounds were separated (Fig. 5), and the pooled peak fractions (fractions 113 to 118 and 130 to 149) were freeze-dried. Both of these products isomerized in a neutral aqueous solution at room temperature to the same equilibrium mixture, which produced a spectrum similar to spectrum h in Fig. 4B; this spontaneous isomerization was 90% complete in about 5 h beginning with either chemical. However, both products were stable if they were kept as dried solids under desiccation conditions.

Proton and \({}^{13}\mathrm{C}\) NMR spectra of freshly prepared solutions of both products in \(d_6\)-dimethyl sulfoxide were obtained. These spectra are consistent with the proposed structures (Fig. 6). The coupling constants (\(J\)) of vinylic protons \(e\) and \(f\) and the chemical shifts of carbon 9 in each compound require special note. For HCCA, \(J_{ef}\) is \(9.8 \, \mathrm{Hz}\), which is typical of cis-oriented olefinic hydrogens, while the trans arrangement in tHBPA is evidenced by a \(J_{ef}\) of \(16.5 \, \mathrm{Hz}\). The chemical shift of the hemiketal carbon of HCCA is \(94.34 \, \mathrm{ppm}\), while the chemical shift of the carbonyl carbon of tHBPA is \(198.1 \, \mathrm{ppm}\) (45).

Mass spectra were obtained after gas chromatography of di(TMS) derivatives (see Materials and Methods). The tHBPA derivative emerged at 20.03 min and had major ions at \(m/z\) 336 (M), 321 (M – CH₃), and 219 (M – TMS – CO₂). A small peak was also evident at 16.65 min, but its mass spectrum was not determined. HCCA gave three chromatographic peaks, one at 16.65 min (major ions at \(m/z\) 321 [M – CH₃] and 219 [M – TMS – CO₂]), one at 18.44 min (major ions at \(m/z\) 336 [M], 321 [M – CH₃], and 219 [M – TMS – CO₂]), and one at 20.01 min (major ions at \(m/z\) 336 [M] and 219 [M – TMS – CO₂]). These three chromatographic peaks are attributed to the di(TMS) derivatives of all three possible isomers, cHBPA and tHBPA (18.44 and 20.01, min, respectively), as well as the hemiketal, HCCA (16.65 min).

HCCA isomerase. Barnsley (3) partially purified an HCCA-metabolizing enzyme from extracts of *Pseudomonas* sp. strain NCIB 9816. The pH optimum for this enzyme was 10, and its \(K_m\) for HCCA was \(0.2 \, \mathrm{mM}\). The enzyme was assayed in mixtures containing \(100 \, \mathrm{mM}\) sodium pyrophosphate (pH 10), \(0.5 \, \mathrm{mM}\) glutathione, \(0.05 \, \mathrm{mM}\) HCCA, and cell extracts, and the appearance of the product, described as cHBPA but according to our data actually tHBPA, was measured at \(420 \, \mathrm{nm}\) by using an extinction coefficient of \(10,500 \, \mathrm{M}^{-1} \, \mathrm{cm}^{-1}\). This naphthalene-inducible isomerase activity was abolished in several Tn5 mutants isolated by Yen and Gunsalus (52), who mapped the isomerase gene, *nahD*, between *nahC* and *nahE* on plasmid NAH7. On the basis of the data of these authors, extracts of strain PAO1 or of strain JM109 carrying pRE630 or pRE699 in which the DNA segment between *nahC* and *nahE* was located should have contained HCCA isomerase, but none was detected by using the assay system of Barnsley. It seemed likely that an isomerase required for the metabolism of naphthalene to salicylate would be encoded by the same operon as *nahA* through *nahF* and that a promising place to look for *nahD* would be on an adjacent DNA segment. Such a DNA segment was cloned from NAH7 by using the *MluI-StuI* fragment (coordinates 9.7 to 10.7) as a probe to identify clones containing DNA from the right side of the operon. pRE714, a *BglII* fragment inserted into pLV59, and pRE712, an *MluI* fragment inserted into pUCBM21, were obtained in this experiment. pRE714 was subcloned to give pRE718 and pRE720.

Extracts of *E. coli* JM109 carrying pRE718 catalyzed the conversion of tHBPA (data not shown) or HCCA (Fig. 7) to an equilibrium mixture that contained \(55\%\) HCCA and \(45\%\) tHBPA at pH 7. As Barnsley indicated (3), enzyme-catalyzed isomerization of HCCA to tHBPA proceeds rapidly and completely at pH 10 in \(100 \, \mathrm{mM}\) sodium pyrophosphate buffer. Because the reaction occurs entirely in one direction at pH 10, this is a more useful pH than pH 7 for carrying out

---

#### Figures

**FIG. 7.** Conversion of HCCA to an equilibrium mixture with tHBPA by cell extracts of *E. coli* JM109(pRE718) at 30°C. Sample and reference cuvettes contained 50 mM potassium-sodium phosphate buffer (pH 7) in 1-ml volumes. The sample cuvette also contained 75 nmol of HCCA. Spectra were recorded before the addition of 5 μl of extract containing 120 μg of protein to both cuvettes and thereafter at 0.17, 5, 10, 15, 20, 25, 30, and 35 min. For HCCA, ε<sub>255</sub> is 7,790 M<sup>-1</sup> cm<sup>-1</sup>, ε<sub>296</sub> is 2,550 M<sup>-1</sup> cm<sup>-1</sup>, and ε<sub>340</sub> is 280 M<sup>-1</sup> cm<sup>-1</sup>, while for tHBPA, ε<sub>255</sub> is 3,510 M<sup>-1</sup> cm<sup>-1</sup>, ε<sub>296</sub> is 12,080 M<sup>-1</sup> cm<sup>-1</sup>, and ε<sub>340</sub> is 8,530 M<sup>-1</sup> cm<sup>-1</sup>.

**FIG. 8.** Conversion of tHBPA to salicylaldehyde and pyruvate by cell extracts of *E. coli* JM109(pRE701) at 30°C. The sample and reference cuvettes contained 50 mM potassium-sodium phosphate buffer (pH 7.0) in 1-ml volumes. The sample cuvette also contained 75 nmol of tHBPA. Spectra were recorded before the addition of 2.5 μl of extract containing 3.2 μg of protein to both cuvettes and after 0.17, 3, 6, 9, 12, 15, 18, and 21 min. For salicylaldehyde, ε<sub>255</sub> is 9,740 M<sup>-1</sup> cm<sup>-1</sup>, ε<sub>296</sub> is 1,450 M<sup>-1</sup> cm<sup>-1</sup>, ε<sub>323</sub> is 2,520 M<sup>-1</sup> cm<sup>-1</sup>, and ε<sub>340</sub> is 1,890 M<sup>-1</sup> cm<sup>-1</sup>.

#### Text

routine isomerase assays. The extracts used for the experiments shown in Fig. 7 metabolized HCCA to tHBPA at a rate of 33 nmol min<sup>-1</sup> mg of protein<sup>-1</sup> when the preparations were assayed at pH 10. When glutathione (0.5 mM) was included in the assay mixture, the rate increased to 212 nmol min<sup>-1</sup> mg of protein<sup>-1</sup>; glutathione seems to stabilize the enzyme, which loses activity rapidly in the absence of this compound. Glutathione was left out of the reaction shown in Fig. 7 because it also appears to react with tHBPA and this alters the spectrum. No isomerase activity was detected in extracts of strain JM109(pRE720); this indicates that the gene encoding this enzyme begins to the left of the *StuI* site at map coordinate 10.7.

Other researchers (3, 7, 34, 52) have treated HCCA with NaOH to produce what was assumed to be cHBPA. This compound was then used as an assay substrate for the next enzyme of the pathway. In order to determine whether this product is actually the *cis* isomer or the *trans* isomer, HCCA

was dissolved in D<sub>2</sub>O and then NaOH was added. The <sup>13</sup>C and <sup>1</sup>H NMR spectra were recorded immediately and were recorded again over the next 2 h. The NMR spectra of tHBPA in D<sub>2</sub>O containing NaOH were also obtained (Table 2). The NMR spectra not only indicate from the *trans* coupling constant of 16.3 Hz for protons *e* and *f* that tHBPA, not cHBPA, is the product of treatment of HCCA with alkali but also indicate that these two protons are not displaced during this isomerization since they would have been replaced in the product by deuterium from the solvent.

tHBPA hydratase-aldolase. Extracts of *E. coli* JM109 (pRE701) transform tHBPA (λ<sub>max</sub>, 296 and 340 nm) to salicylaldehyde (λ<sub>max</sub>, 255 nm) and pyruvate (Fig. 8). The rate of spectral change corresponds to a specific activity of 1.55 μmol min<sup>-1</sup> mg of protein<sup>-1</sup>. This activity does not require added cofactors and is not sensitive to EDTA (5 mM). The isosbestic points at 267, 242, and 226 nm indicate that the spectra are due solely to two components, substrate

#### Table

**TABLE 2. Selected chemical shifts and coupling constants in the NMR spectra of HCCA and tHBPA in D<sub>2</sub>O containing NaOH**

| Resonance due to: | HCCA | tHBPA |
|-------------------|------|-------|
|                   | Initial chemical shift (ppm) | Initial coupling constant (Hz) | Final chemical shift (ppm)<sup>b</sup> | Final coupling constant (Hz)<sup>b</sup> | Chemical shift (ppm) | Coupling constant (Hz) |
| <sup>1</sup>H<sub>e</sub> | 6.83 | 8.2 | 8.10 | 16.3 | 8.08 | 16.3 |
| <sup>1</sup>H<sub>f</sub> | 5.95 | 8.2 | 6.81 | 16.3 | 6.80 | 16.3 |
| <sup>13</sup>C<sub>9</sub> | 94.11 | — | 198.21 | — | 198.29 | — |

<sup>a</sup> For designations of selected carbon and hydrogen atoms see Fig. 6.  
<sup>b</sup> Final chemical shifts and final coupling constants were determined after incubation for 2 h.

#### Figure 9: Structures of chemicals tested as substrates for the hydratase-aldolase present in extracts of strain JM109(pRE701).

**Hydratase-Aldolase Substrates:**
- Compound VI:
  \[
  \text{Structure: } \ce{C6H5CH(OH)COO^-}
  \]
- Compound XII:
  \[
  \text{Structure: } \ce{C6H4(OH)C(O)COO^-}
  \]
- Compound XIII:
  \[
  \text{Structure: } \ce{C6H4(S)CH(OH)COO^-}
  \]
- Compound XIV:
  \[
  \text{Structure: } \ce{C6H4(S)(OH)C(O)COO^-}
  \]

**Not Hydratase-Aldolase Substrates:**
- Compound XV:
  \[
  \text{Structure: } \ce{C6H5CH(COO^-)}
  \]
- Compound XVI:
  \[
  \text{Structure: } \ce{C6H4(OCH3)C(O)COO^-}
  \]
- Compound XVII:
  \[
  \text{Structure: } \ce{C6H5(OH)COO^-}
  \]

---

### Text Content

and product, and that no significant concentration of any UV-absorbing intermediate is present. Plasmid pRE701 carries a 1-kb *MluI-StuI* fragment that is only large enough to encode proteins having a total molecular weight of 33,000. Therefore, it is likely that tHBPA is metabolized to salicylaldehyde in a one-step reaction by a single hydratase-aldolase.

Several analogs of tHBPA were tested as substrates for extracts containing the hydratase-aldolase (Fig. 9). Two analogs that lack the phenolic hydroxyl group, trans-benzylidenepyruvate (Fig. 9, compound XV) and trans-*o*-methoxybenzylidenepyruvate (compound XVI), and one analog that lacks the keto group, trans-*o*-hydroxycinnamate (compound XVII), were not attacked by these extracts. However, trans-4-(3-hydroxy-2-thianaphthyl)-2-oxo-but-3-enoate (compound XIV), trans-4-(3-hydroxy-2-benzofuranyl)-2-oxobut-3-enoate (compound XII), and trans-4-(3-hydroxy-2-thienyl)-2-oxobut-3-enoate (compound XIII), all of which carry a trans-α,β-unsaturated keto acid side chain and an adjacent hydroxyl group, were metabolized by the hydratase-aldolase (Fig. 9); this suggests that these groups are important in the hydratase-aldolase reaction mechanism.

Aldolases are normally reversible enzymes, and it was expected that the hydratase-aldolase might catalyze the reaction of salicylaldehyde and pyruvate to produce tHBPA. This reaction was observed. However, because the equilibrium in this reaction favors cleavage, an excess of substrates is required in order to direct the reaction in the reverse direction (11). The hydratase-aldolase catalyzes the condensation of pyruvate with several other aromatic aldehydes, including benzaldehyde, to give trans-benzylidenepyruvate (Fig. 10). The identification of the product as trans-benzylidenepyruvate was confirmed by comparing the product of a large-scale reaction with the synthetic compound by ^13C and ^1H NMR spectroscopy (11). Since benzylidenepyruvate is not a substrate for the enzyme, the reaction is irreversible and can be carried out to completion by using relatively low concentrations of the substrates, benzaldehyde and pyruvate.

Transformation of salicylaldehyde to salicylate by cell extracts of strain JM109(pRE672). Cell extracts of strain JM109(pRE672) converted salicylaldehyde to salicylate (λ_max, 296 nm) in the presence of a catalytic quantity of NAD⁺ (Fig. 11). It is not necessary to provide an equimolar quantity of NAD⁺ because this compound is regenerated by an NADH oxidase which is also present in these extracts. Salicylaldehyde dehydrogenase is also present in extracts of strain PAO1 carrying pRE630 or pRE695 but not in extracts of strain PAO1 carrying pRE699 or pRE668.

---

### Figure 10: Aldol condensation of benzaldehyde and pyruvate catalyzed by cell extracts of strain JM109(pRE701) at 30°C.

The sample and reference cuvettes contained 50 mM potassium-sodium phosphate buffer (pH 7) and 10 μmol of sodium pyruvate in 1-ml volumes. The sample cuvette also contained 100 nmol of benzaldehyde. Spectra were recorded before the addition of 4 μl of extract containing 80 μg of protein and thereafter at 0.17, 10, 20, 30, 40, 50, 60, and 70 min.

---

### Discussion

Recombinant bacteria that catalyze selected reactions of the naphthalene catabolic pathway have allowed a reassessment of the reactions from 1,2-dihydroxynaphthalene to salicylate. Pathway intermediates were produced and rigorously identified. A pathway is proposed on the basis of these results (Fig. 12). 1,2-Dihydroxynaphthalene (Fig. 12, compound III) is cleaved by a dioxygenase which inserts a molecule of oxygen between the angular carbon and carbon 1 to give the unstable compound 2-hydroxy-4-(2′-oxocyclohexa-3,5-dienyl)-buta-2,4-dienoate (compound V), which rearomatizes, yielding another unstable intermediate, cHBPA (compound VIII). This compound is spontaneously converted to its hemiketal, HCCA (compound VII), the first detectable ring cleavage product. HCCA isomerizes to an equilibrium mixture containing 45% tHBPA (compound VI) both spontaneously and more rapidly in a reaction catalyzed by an isomerase (enzyme D). The tHBPA formed from HCCA is metabolized by a hydratase-aldolase (enzyme E) to salicylaldehyde (compound X) and pyruvate. An NAD⁺-requiring dehydrogenase (enzyme F) then catalyzes the oxidation of salicylaldehyde to salicylate (compound XI).

Misidentification of tHBPA as the *cis* isomer in previous work (3, 7, 34, 52) has been a factor which has complicated a complete understanding of the naphthalene pathway.
isomer clarifies this paradox; HCCA is formed from cHBPA and metabolized to tHBPA.

The relationship of cHBPA to HCCA is analogous to the relationship of the open-chain form of D-glucose to its cyclic hemiacetal forms, α- and β-D-glucose. The open-chain form of glucose accounts for less than 0.5% of an aqueous equilibrium mixture with the hemiacetals and is an intermediate in the mutarotation between the anomeric α and β forms (40). Similarly, cHBPA appears to exist almost entirely as its hemiketal, HCCA, in aqueous solutions. Barnsley (3) noted that although the hemiketal carbon of HCCA is chiral, HCCA has no optical activity. This further supports the conclusion that HCCA is formed by the non-enzymecatalyzed nucleophilic attack of the hydroxyl oxygen of cHBPA from either side of the carbonyl carbon to yield a racemic product.

Ring cleavage products formed as intermediates in the metabolism of fused-ring polycyclic aromatic hydrocarbons and aromatic heterocyclic compounds have the same structural constraints as cHBPA; the aromatic hydroxyl group and the keto group of the cis-oriented side chain of cHPBA are in close proximity, and this facilitates hemiketal formation. Therefore, it is not surprising that hemiketals analogous to HCCA have been identified as possible intermediates in the bacterial metabolism of anthracene (19), phenanthrene (19), dibenzothiophene (33), and pyrene (50). Similar structural constraints may be responsible for the formation of 2-pyrone-4,6-dicarboxylate in the bacterial metabolism of protocatechuate (14, 37, 38) and gallate (32) and the formation of 3,4-dimethyl-2-pyrone-6-carboxylate during the metabolism of riboflavin (28).

Although HCCA isomerizes spontaneously to form tHBPA, the rate of this reaction is evidently insufficient for efficient catabolism since cells synthesize an isomerase for this purpose. Recombinant bacteria that lack the isomerase gene, such as strain PAO1(pRE630), are able to metabolize naphthalene to salicylate but only with the transient accumulation of HCCA. In contrast, a strain carrying the complete 12.05-kb EcoRI-BglII segment in pMMB277, strain PAO1(pRE731), metabolizes naphthalene to salicylate without the accumulation of intermediates (11).

It was suggested previously (3) that o-hydroxybenzylidenepyruvate is metabolized to salicylaldehyde and pyruvate in a single step by an aldolase. However, this compound

---

**FIG. 11. Conversion of salicylaldehyde to salicylate by cell extracts of *E. coli* JM109(pRE672) at 30°C.** The sample and reference cuvettes contained 50 mM potassium-sodium phosphate buffer (pH 7.0) and 10 nmol of NAD⁺ in 1-ml volumes. The sample cuvette also contained 75 nmol of salicylaldehyde. Spectra were recorded before the addition of 10 μl of extract containing 31 mg of protein to both cuvettes and after 0.17, 2.5, 5, 7.5, 10, 12.5, 15, 17.5, 20, 22.5, and 25 min.

Barnsley (3) identified the first product of ring cleavage of 1,2-dihydroxynaphthalene as HCCA and showed that this compound isomerized spontaneously to form an equilibrium mixture with a compound that he identified as cHBPA. He reasoned that if cHBPA was the ring cleavage product, it should have appeared initially, either alone or in an equilibrium mixture with HCCA, but not after HCCA. However, HCCA is the hemiketal derivative of cHBPA and can only have been formed from cHBPA. Identification of the o-hydroxybenzylidenepyruvate that is formed from and is in equilibrium with HCCA as the trans isomer and not the cis

---

![Chemical Reaction Diagram](https://i.imgur.com/chemical_diagram.png)

**FIG. 12. Proposed pathway for the plasmid NAH7-encoded metabolism of 1,2-dihydroxynaphthalene to salicylate.** Chemical designations: III, 1,2-dihydroxynaphthalene; V, 2-hydroxy-4-(2′-oxo-3,5-cyclohexadienyl)-buta-2,4-dienoate; VIII, cHBPA; VII, HCCA; VI, tHBPA; X, salicylaldehyde; XI, salicylate. Enzyme designations: C, 1,2-dihydroxynaphthalene dioxygenase; D, 2-hydroxychromene-2-carboxylate isomerase; E, trans-o-hydroxybenzylidenepyruvate hydratase-aldolase; F, salicylaldehyde dehydrogenase.

#### Figure 13: Possible Mechanism for the Reaction of tHBPA Catalyzed by the pRE701-Encoded Hydratase-Aldolase

**Chemical Structures and Reactions:**


Vla                          Vlb                          Vlc
: B-Enz                      H:B-Enz                     H+
     OH                       H                           H
     |                        |                           |
     C=O                      C=O                         C=O
     |                        |                           |
COO-                          COO-                        COO-



X + H3C-COO-                 XVIII                       IX
OH                            OH                          OH
CHO                           |                           |
                              C=O                         C=O
                              |                           |
                              H                           H
                              |                           |
                              H+                          H+


**Legend:**
- Compound XVIII is pyruvate.
- For other compound designations, see the legend to Fig. 1.

---

### Text Content

**Figure 13.** Possible mechanism for the reaction of tHBPA catalyzed by the pRE701-encoded hydratase-aldolase. Compound XVIII is pyruvate. For other compound designations see the legend to Fig. 1.

The reaction requires hydration to become an aldolase substrate so that either a separate hydratase is required to prepare it for aldol cleavage or an unusual enzyme that catalyzes both hydration and aldol cleavage is involved. From our work, it is clear that a single enzyme must be involved. A 1-kb DNA fragment cloned from plasmid NAH7 is only large enough to encode an enzyme that has a maximum molecular weight of 33,000 and catalyzes the conversion of tHBPA to salicylaldehyde. A possible catalytic mechanism is suggested by the requirement for an ionizable hydroxyl group and a keto group in the substrate (Fig. 13). The reaction could be initiated by the abstraction of the phenolic proton in tHBPA (Fig. 13, compound VIa) to give the carbanion (compound VIb). Hydration presumably occurs with the sequential addition of a proton and a hydroxyl group to yield the intermediate, 2-oxo-4-hydroxy-4-(2′-hydroxyphenyl)-butyrate (compound IX). This is an aldolase type of substrate with a hydroxyl group β to an enolizable carbonyl. Chemicals with structures like compound IX are unstable (40), and compound IX would be expected to dehydrate instantly to form the thermodynamically more stable compound tHBPA, in which the trans double bond is conjugated with both the keto group and the aromatic ring. Consequently, it is likely that compound IX is not a freely dissociable intermediate but is formed and degraded at the active site of a single enzyme. Spectrophotometric data show no evidence for any UV-absorbing species other than tHBPA and salicylaldehyde during the course of the reaction. Spontaneous dehydration of aldolase-type substrates also accounts for the irreversible formation of trans-benzylidenepyruvate from benzaldehyde and pyruvate in the presence of the hydratase-aldolase (as in the chemical synthesis of this compound). Apparently the hydratase half-reaction is limited to substrates possessing an adjacent ionizable hydroxyl, whereas the aldolase half-reaction can handle a variety of aromatic aldehydes whether they possess this substituent or not. Once formed, an aldol product can spontaneously lose water to give a trans-benzylidenepyruvate, which, if it lacks a 2′-hydroxyl group, is accumulated irreversibly.

Studies of the early reactions of naphthalene metabolism are complicated by a number of spontaneous reactions, including oxidation of 1,2-dihydroxynaphthalene to 1,2-naphthoquinone, cyclization of the ring fission product to HCCA, isomerization of HCCA to tHBPA, and dehydration of 2-oxo-4-hydroxy-4-(2′-hydroxyphenyl)-butyrate. Similar if not greater complications can be anticipated as the degradative reactions of structurally more complex aromatic hydrocarbons are studied.

The combined genetic-biochemical approach taken in this study has proven to be valuable in overcoming some of these problems. Recombinant bacteria constructed in this study that carry genes encoding the first three pathway enzymes allowed the ring cleavage reaction to be studied by using the stable precursor, naphthalene, instead of the unstable substrate, 1,2-dihydroxynaphthalene. While the spontaneous isomerization of the ring cleavage product necessitated the development of a novel method to separate the two isomers, HCCA and tHBPA, the study of the degradation of these two compounds was facilitated by subcloning the genes encoding the enzymes responsible for their degradation. By separating these enzymes from other pathway enzymes, it became possible to study these reversible reactions from either direction.

The gene order `nahABFCED` differs significantly from the gene orders proposed by Yen and Gunsalus (`nahABCDEF`) and Schell (`nahABCDF`) but is compatible with the data of these workers. Schell (49) was not able to demonstrate salicylaldehyde dehydrogenase activity in bacteria carrying the EcoRI-SacI fragment (map coordinates 0 to 7.2 [Fig. 2]) or the EcoRI-BglII fragment (map coordinates 0 to 7.35 [Fig. 2]), although this activity was present in bacteria carrying a larger EcoRI fragment (map coordinates 0 to 25). As a result, Schell concluded that `nahF`, which encodes salicylaldehyde dehydrogenase, is located to the right of `nahC`. However, it is likely that both the SacI and BglII sites are within `nahF`, which was shown in this study to be located within the BamHI fragment cloned in pRE672 and to the left of `nahC`. Yen and Gunsalus (52) reported that `nahF` was expressed in a strain that had Tn5 inserted into `nahD`. This observation is consistent with a location of `nahF` upstream (to the left in Fig. 2) of `nahD`.

---

### Figure 13: Possible Mechanism for the Reaction of tHBPA Catalyzed by the pRE701-Encoded Hydratase-Aldolase

**Chemical Structures and Reactions:**


Vla                          Vlb                          Vlc
: B-Enz                      H:B-Enz                     H+
     OH                       H                           H
     |                        |                           |
     C=O                      C=O                         C=O
     |                        |                           |
COO-                          COO-                        COO-



X + H3C-COO-                 XVIII                       IX
OH                            OH                          OH
CHO                           |                           |
                              C=O                         C=O
                              |                           |
                              H                           H
                              |                           |
                              H+                          H+


**Legend:**
- Compound XVIII is pyruvate.
- For other compound designations, see the legend to Fig. 1.

---

### Text Content

**Figure 13.** Possible mechanism for the reaction of tHBPA catalyzed by the pRE701-encoded hydratase-aldolase. Compound XVIII is pyruvate. For other compound designations see the legend to Fig. 1.

The reaction requires hydration to become an aldolase substrate so that either a separate hydratase is required to prepare it for aldol cleavage or an unusual enzyme that catalyzes both hydration and aldol cleavage is involved. From our work, it is clear that a single enzyme must be involved. A 1-kb DNA fragment cloned from plasmid NAH7 is only large enough to encode an enzyme that has a maximum molecular weight of 33,000 and catalyzes the conversion of tHBPA to salicylaldehyde. A possible catalytic mechanism is suggested by the requirement for an ionizable hydroxyl group and a keto group in the substrate (Fig. 13). The reaction could be initiated by the abstraction of the phenolic proton in tHBPA (Fig. 13, compound VIa) to give the carbanion (compound VIb). Hydration presumably occurs with the sequential addition of a proton and a hydroxyl group to yield the intermediate, 2-oxo-4-hydroxy-4-(2′-hydroxyphenyl)-butyrate (compound IX). This is an aldolase type of substrate with a hydroxyl group β to an enolizable carbonyl. Chemicals with structures like compound IX are unstable (40), and compound IX would be expected to dehydrate instantly to form the thermodynamically more stable compound tHBPA, in which the trans double bond is conjugated with both the keto group and the aromatic ring. Consequently, it is likely that compound IX is not a freely dissociable intermediate but is formed and degraded at the active site of a single enzyme. Spectrophotometric data show no evidence for any UV-absorbing species other than tHBPA and salicylaldehyde during the course of the reaction. Spontaneous dehydration of aldolase-type substrates also accounts for the irreversible formation of trans-benzylidenepyruvate from benzaldehyde and pyruvate in the presence of the hydratase-aldolase (as in the chemical synthesis of this compound). Apparently the hydratase half-reaction is limited to substrates possessing an adjacent ionizable hydroxyl, whereas the aldolase half-reaction can handle a variety of aromatic aldehydes whether they possess this substituent or not. Once formed, an aldol product can spontaneously lose water to give a trans-benzylidenepyruvate, which, if it lacks a 2′-hydroxyl group, is accumulated irreversibly.

Studies of the early reactions of naphthalene metabolism are complicated by a number of spontaneous reactions, including oxidation of 1,2-dihydroxynaphthalene to 1,2-naphthoquinone, cyclization of the ring fission product to HCCA, isomerization of HCCA to tHBPA, and dehydration of 2-oxo-4-hydroxy-4-(2′-hydroxyphenyl)-butyrate. Similar if not greater complications can be anticipated as the degradative reactions of structurally more complex aromatic hydrocarbons are studied.

The combined genetic-biochemical approach taken in this study has proven to be valuable in overcoming some of these problems. Recombinant bacteria constructed in this study that carry genes encoding the first three pathway enzymes allowed the ring cleavage reaction to be studied by using the stable precursor, naphthalene, instead of the unstable substrate, 1,2-dihydroxynaphthalene. While the spontaneous isomerization of the ring cleavage product necessitated the development of a novel method to separate the two isomers, HCCA and tHBPA, the study of the degradation of these two compounds was facilitated by subcloning the genes encoding the enzymes responsible for their degradation. By separating these enzymes from other pathway enzymes, it became possible to study these reversible reactions from either direction.

The gene order `nahABFCED` differs significantly from the gene orders proposed by Yen and Gunsalus (`nahABCDEF`) and Schell (`nahABCDF`) but is compatible with the data of these workers. Schell (49) was not able to demonstrate salicylaldehyde dehydrogenase activity in bacteria carrying the EcoRI-SacI fragment (map coordinates 0 to 7.2 [Fig. 2]) or the EcoRI-BglII fragment (map coordinates 0 to 7.35 [Fig. 2]), although this activity was present in bacteria carrying a larger EcoRI fragment (map coordinates 0 to 25). As a result, Schell concluded that `nahF`, which encodes salicylaldehyde dehydrogenase, is located to the right of `nahC`. However, it is likely that both the SacI and BglII sites are within `nahF`, which was shown in this study to be located within the BamHI fragment cloned in pRE672 and to the left of `nahC`. Yen and Gunsalus (52) reported that `nahF` was expressed in a strain that had Tn5 inserted into `nahD`. This observation is consistent with a location of `nahF` upstream (to the left in Fig. 2) of `nahD`.

# ACKNOWLEDGMENTS

We thank M. Bagdasarian of the Michigan Biotechnology Institute, Lansing, for his generous gifts of pMMB207 and pMMB277, W. Gilliam of our laboratory for the gas chromatography-mass spectrometry analyses, B. Blattman and M. Downey of Technical Resources, Inc., Gulf Breeze, Fla., for the high-pressure liquid chromatography analyses, and J. Gurst of the Department of Chemistry, University of West Florida, Pensacola, for the NMR analyses.

Partial support for the purchase of the NMR spectrometer at the University of West Florida was provided by grant USE-9050802 from the National Science Foundation.

---

## REFERENCES

1. Ahmed, A. 1985. A rapid procedure for DNA sequencing using transposon-promoted deletions in *Escherichia coli*. Gene 39: 305–310.
2. Ahmed, A. 1987. Use of transposon-promoted deletions in DNA sequence analysis. Methods Enzymol. 155:177–204.
3. Barnsley, E. A. 1976. Naphthalene metabolism by pseudomonads: the oxidation of 1,2-dihydroxynaphthalene to 2-hydroxychromene-2-carboxylic acid and the formation of 2′-hydroxybenzalpyruvate. Biochem. Biophys. Res. Commun. 72:1116–1121.
4. Bolivar, F., R. L. Rodriguez, P. J. Greene, M. C. Betlach, H. L. Heyneker, H. W. Boyer, J. H. Crossa, and S. Falkow. 1977. Construction and characterization of new cloning vehicles. II. A multipurpose cloning system. Gene 2:95–113.
5. Catterall, F. A., K. Murray, and P. A. Williams. 1971. The configuration of 1,2-dihydroxy-1,2-dihydronaphthalene formed in the bacterial metabolism of naphthalene. Biochim. Biophys. Acta 237:361–364.
6. Clewell, D. B., and D. R. Helinski. 1969. Supercoiled circular DNA-protein complex in *Escherichia coli*: purification and induced conversion to an open circular DNA form. Proc. Natl. Acad. Sci. USA 62:1159–1166.
7. Davies, J. L., and W. C. Evans. 1964. Oxidative metabolism of naphthalene by soil pseudomonads. The ring-fission mechanism. Biochem. J. 91:251–261.
8. Davis, R. W., D. Botstein, and J. R. Roth. 1980. Advanced bacterial genetics. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.
9. Ditta, G., S. Stanfield, D. Corbin, and D. Helinski. 1980. Broad host range DNA cloning system for gram-negative bacteria: construction of a gene bank of *Rhizobium meliloti*. Proc. Natl. Acad. Sci. USA 77:7347–7351.
10. Dretzen, G., M. Bellard, P. Sassone-Corsi, and P. Chambon. 1981. A reliable method for the recovery of DNA fragments from agarose and acrylamide gels. Anal. Biochem. 112:295–298.
11. Eaton, R. W. Unpublished data.
12. Eaton, R. W., and J. S. Karns. 1991. Cloning and analysis of s-triazine catabolic genes from *Pseudomonas* sp. strain NRRL B-12227. J. Bacteriol. 173:1215–1222.
13. Eaton, R. W., and J. D. Nitterauer. 1992. Abstr. Ann. Meet. Am. Soc. Microbiol. 1992, K 44, p. 263.
14. Eaton, R. W., and D. W. Ribbons. 1982. Metabolism of dibutylyphthalate and phthalate by *Micrococcus* sp. strain 12B. J. Bacteriol. 151:48–57.
15. Eaton, R. W., and K. N. Timmis. 1986. Characterization of a plasmid-specified pathway for catabolism of isopropylbenzene in *Pseudomonas putida* RE204. J. Bacteriol. 168:123–131.
16. Ensley, B. D., and D. T. Gibson. 1983. Naphthalene dioxygenase: purification and properties of a terminal oxygenase component. J. Bacteriol. 155:505–511.
17. Ensley, B. D., D. T. Gibson, and A. L. Laborde. 1982. Oxidation of naphthalene by a multicomponent enzyme system from *Pseudomonas* sp. strain NCIB 9816. J. Bacteriol. 149:948–954.
18. Ensley, B. D., B. J. Ratzkin, T. D. Osslund, M. J. Simon, L. P. Wackett, and D. T. Gibson. 1983. Expression of naphthalene oxidation genes in *Escherichia coli* results in the biosynthesis of indigo. Science 222:167–169.
19. Evans, W. C., H. N. Fernley, and E. Griffiths. 1965. Oxidative metabolism of phenanthrene and anthracene by soil pseudomonads. The ring fission mechanism. Biochem. J. 95: 819–831.
20. Gold, P. 1990. Sequelap II instruction manual. Gold Biotechnology, Inc., St. Louis, Mo.
21. Grunstein, M., and D. Hogness. 1975. Colony hybridization: a method for the isolation of cloned DNAs that contain a specific gene. Proc. Natl. Acad. Sci. USA 72:3961–3965.
22. Guerry, P., J. Van Embden, and S. Falkow. 1974. Molecular nature of two nonconjugative plasmids carrying drug resistance genes. J. Bacteriol. 117:619–630.
23. Guiard, B. M., and E. E. Snell. 1981. Biochemical factors in growth, p. 79–111. In P. Gerhardt, R. G. E. Murray, R. N. Costilow, E. W. Nester, W. A. Wood, N. R. Krieg, and G. B. Phillips (ed.), Manual of methods for general bacteriology. American Society for Microbiology, Washington, D.C.
24. Haigler, B. E., and D. T. Gibson. 1990. Purification and properties of NADH-ferredoxin_NAP reductase, a component of naphthalene dioxygenase from *Pseudomonas* sp. strain NCIB 9816. J. Bacteriol. 172:457–464.
25. Haigler, B. E., and D. T. Gibson. 1990. Purification and properties of ferredoxin_NAP, a component of naphthalene dioxygenase from *Pseudomonas* sp. strain NCIB 9816. J. Bacteriol. 172:465–468.
26. Hansen, J. H., and R. H. Olsen. 1978. Isolation of large bacterial plasmids and characterization of the P2 incompatibility group plasmids pMG1 and pMG5. J. Bacteriol. 135:227–238.
27. Harayama, S., and M. Rekik. 1989. Bacterial aromatic ring-cleavage enzymes are classified into two different gene families. J. Biol. Chem. 264:15328–15333.
28. Harkness, D. R., L. Tsai, and E. R. Stadtman. 1964. Bacterial degradation of riboflavin. V. Stoichiometry of riboflavin degradation to oxamide and other products, oxidation of C^14^-labeled intermediates and isolation of the pseudomonad effecting these transformations. Arch. Biochem. Biophys. 108:323–333.
29. Holmes, D. S., and M. Quigley. 1981. A rapid boiling method for the preparation of bacterial plasmids. Anal. Biochem. 114:193–199.
30. Jeffrey, A. M., H. Y. C. Yeh, D. M. Jerina, T. R. Patel, J. F. Davey, and D. T. Gibson. 1975. Initial reactions in the oxidation of naphthalene from *Pseudomonas putida*. Biochemistry 14: 575–584.
31. Jerina, D. M., J. W. Daly, A. M. Jeffrey, and D. T. Gibson. 1971. cis-1,2-Dihydroxy-1,2-dihydronaphthalene: a bacterial metabolite from naphthalene. Arch. Biochem. Biophys. 142:394–396.
32. Kersten, P. J., S. Dagley, J. W. Whittaker, D. M. Arciero, and J. D. Lipscomb. 1982. 2-Pyrone-4,6-dicarboxylic acid, a catabolite of gallic acids in *Pseudomonas* species. J. Bacteriol. 152:1154–1162.
33. Kodama, K., K. Umehara, K. Shimizu, S. Nakatani, Y. Minoda, and K. Yamada. 1973. Identification of microbial products from dibenzothiophene and its proposed oxidation pathway. Agric. Biol. Chem. 37:45–50.
34. Kuhm, A. E., A. Stolz, K.-L. Ngai, and H.-J. Knackmuss. 1991. Purification and characterization of a 1,2-dihydroxynaphthalene dioxygenase from a bacterium that degrades naphthalenesulfonic acids. J. Bacteriol. 173:3795–3802.
35. Kurkela, S., H. Lehtväläiho, E. T. Palva, and T. H. Teeri. 1988. Cloning, nucleotide sequence and characterization of genes encoding naphthalene dioxygenase of *Pseudomonas putida* strain NCIB 9816. Gene 73:355–362.
36. Leaderberg, E. M., and S. N. Cohen. 1974. Transformation of *Salmonella typhimurium* by plasmid deoxyribonucleic acid. J. Bacteriol. 119:1072–1074.
37. Maruyama, K. 1979. Isolation and identification of the reaction product of α-hydroxy-γ-carboxymuconic ε-semialdehyde dehydrogenase. J. Biochem. 86:1671–1677.
38. Maruyama, K. 1983. Purification and properties of 2-pyrone-4,6-dicarboxylate hydrolase. J. Biochem. 93:557–565.
39. Morales, V. M., A. Bäckman, and M. Bagdasarian. 1991. A series of wide-host-range low-copy-number vectors that allow direct screening for recombinants. Gene 97:39–47.
40. Morrison, R. T., and R. N. Boyd. 1966. Organic chemistry, 2nd ed. Allyn and Bacon, Inc., Boston.
41. O’Connor, C. D., and G. O. Humphries. 1982. Expression of the

EcoRI restriction-modification system and the construction of positive-selection cloning vectors. Gene 20:219–229.

42. Patel, T. R., and E. A. Barnsley. 1980. Naphthalene metabolism by pseudomonads: purification and properties of 1,2-dihydroxynaphthalene oxygenase. J. Bacteriol. 143:668–673.

43. Peng, Z.-G., and R. Wu. 1986. A simple and rapid nucleotide sequencing strategy and its application in analyzing a rice histone 3 gene. Gene 45:247–252.

44. Pharmacia Fine Chemicals. 1977. Sephadex. Gel filtration in theory and practice. Pharmacia Fine Chemicals, Uppsala, Sweden.

45. Pretsch, E., J. Seibl, W. Simon, and T. Clerc. 1989. Tables of spectral data for structure determination of organic compounds. Springer-Verlag, New York.

46. Reimer, M. 1931. Benzalpyruvic acid dibromide. J. Am. Chem. Soc. 53:3147–3149.

47. Reimer, M., and M. Howard. 1928. Addition reactions of unsaturated alpha-ketonic acids. II. J. Am. Chem. Soc. 50:

2506–2512.

48. Royle, P. L., H. Matsumoto, and B. W. Holloway. 1981. Genetic circularity of the *Pseudomonas aeruginosa* PAO chromosome. J. Bacteriol. 145:145–155.

49. Schell, M. 1983. Cloning and expression in *Escherichia coli* of the naphthalene degradation genes from plasmid NAH7. J. Bacteriol. 153:822–829.

50. Walter, U., M. Beyer, J. Klein, and H.-J. Rehm. 1991. Degradation of pyrene by *Rhodococcus* sp. UW1. Appl. Microbiol. Biotechnol. 34:671–676.

51. Yanisch-Perron, C., J. Vieira, and J. Messing. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119.

52. Yen, K.-M., and I. C. Gunsalus. 1982. Plasmid gene organization: naphthalene/salicylate oxidation. Proc. Natl. Acad. Sci. USA 79:874–878.

53. Yen, K.-M., and C. M. Serdar. 1988. Genetics of naphthalene catabolism in pseudomonads. Crit. Rev. Microbiol. 15:247–268.
