
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2004 by The American Society for Biochemistry and Molecular Biology, Inc.

Crystal Structure and Mechanistic Implications of  
$N^{2}$-(2-Carboxyethyl)arginine Synthase, the First Enzyme in the  
Clavulanic Acid Biosynthesis Pathway*

Received for publication, October 1, 2003, and in revised form, November 6, 2003  
Published, JBC Papers in Press, November 17, 2003, DOI 10.1074/jbc.M310803200

Matthew E. C. Caines‡, Jonathan M. Elkins‡, Kirsty S. Hewitson, and Christopher J. Schofield§  
From the Department of Chemistry, University of Oxford, Chemistry Research Laboratory,  
Oxford OX1 3TA, United Kingdom

The initial step in the biosynthesis of the clinically important $\beta$-lactamase inhibitor clavulanic acid involves condensation of two primary metabolites, D-glyceraldehyde 3-phosphate and L-arginine, to give $N^{2}$-(2-carboxyethyl)arginine, a $\beta$-amino acid. This unusual N-C bond forming reaction is catalyzed by the thiamin diphosphate (ThP$_{2}$)-dependent enzyme $N^{2}$-(2-carboxyethyl)arginine synthase. Here we report the crystal structure of $N^{2}$-(2-carboxyethyl)arginine synthase, complexed with ThP$_{2}$ and Mg$^{2+}$, to 2.35-Å resolution. The structure was solved in two space groups, $P2_{1}2_{1}2_{1}$ and $P2_{1}2_{1}2$. In both, the enzyme is observed in a tetrameric form, composed of a dimer of two more tightly associated dimers, consistent with both mass spectrometric and gel filtration chromatography studies. Both ThP$_{2}$ and Mg$^{2+}$ cofactors are present at the active site, with ThP$_{2}$ in a “V” conformation as in related enzymes. A sulfate anion is observed in the active site of the enzyme in a location proposed as a binding site for the phosphate group of the D-glyceraldehyde 3-phosphate substrate. The mechanistic implications of the active site arrangement are discussed, including the potential role of the aminopyrimidine ring of the ThP$_{2}$. The structure will form a basis for future mechanistic and structural studies, as well as engineering aimed at production of alternative $\beta$-amino acids.

An important mechanism of bacterial resistance to $\beta$-lactam antibiotics involves $\beta$-lactamase-catalyzed hydrolysis of the $\beta$-lactam ring, resulting in biologically inactive products (1). Clavulanic acid (Fig. 1), one of many $\beta$-lactams produced by Streptomyces clavuligerus, is the most widely used inhibitor of the serine $\beta$-lactamases, and is clinically administered in combination with the penicillin amoxycillin (2, 3). Clavulanic acid is itself a bicyclic $\beta$-lactam, and reacts with the $\beta$-lactamases to form stable acyl-enzyme complexes (4, 5). A commercially viable synthesis of clavulanic acid is rendered difficult by the high liability and density of functionalization around the bicyclic core (3, 6). Consequently clavulanic acid is produced by fermentation and there is considerable interest in its biosynthetic pathway.

A total of 18 genes from the genome of *S. clavuligerus* have thus far been implicated in clavulanic acid biosynthesis (7–10). Of these, functions for open reading frames (*orfs*)$^{1}$ 2–6 and *orf9* have been assigned (Fig. 1) (11–19). Furthermore, the enzymes $\beta$-lactam synthetase, proclavaminic amidino hydrolase, and clavaminic acid synthase, products of *orfs*3, -4, and -5, respectively, have been structurally characterized (20–22).

$N^{2}$-(2-Carboxyethyl)arginine synthase (CEAS) is the enzyme product of the first gene in the clavulanic acid cluster, *orf2*. It catalyzes the first step in the clavulanic acid pathway, the condensation of L-arginine and a three-carbon precursor to give $N^{2}$-(2-carboxyethyl)arginine (Fig. 1). On the basis of turnover experiments with a variety of potential three carbon precursors the best substrate identified was D-glyceraldehyde 3-P (12). Sequence analysis indicated the presence of a conserved thiamin diphosphate (ThP$_{2}$) binding fold, and indeed, CEAS catalysis has been shown to be dependent on ThP$_{2}$ (12).

ThP$_{2}$-dependent enzymes generally employ the cofactor both to effect nucleophilic attack at a carbonyl group and to stabilize the carbanion subsequently formed by proton transfer or decarboxylation. The reactions catalyzed by ThP$_{2}$ enzymes may be subclassified as either non-oxidative, such as those of yeast pyruvate decarboxylase (PDC) (23) and benzoylformate decarboxylase (24), or oxidative, such as those of pyruvate oxidase (25, 26). More recently, several ThP$_{2}$ enzymes that catalyze atypical reactions have been identified, including CEAS and 2-hydroxyphytanoylcoenzyme A lyase, an enzyme involved in the degradation of phytanic acid that is derived from the side chain of chlorophyll (27).

In addition to its key role in clavam biosynthesis, CEAS is of mechanistic interest due to the unprecedented manner in which a ThP$_{2}$-dependent enzyme is used in a N-C bond forming reaction, proposed to proceed via an $\alpha,\beta$-unsaturated acyl-ThP$_{2}$ intermediate. Here we report the crystal structure of CEAS complexed with ThP$_{2}$ and Mg$^{2+}$, and discuss the mechanistic implications arising.

---

* This work was supported by the Biotechnology and Biological Sciences Research Council, the European Union, and GlaxoSmithKline Ltd. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The atomic coordinates and structure factors (codes 1UPA, 1UPB, and 1UPC) have been deposited in the Protein Data Bank, Research Collaboratory for Structural Bioinformatics, Rutgers University, New Brunswick, NJ (http://www.rcsb.org/).
* Both authors contributed equally to this work.
* To whom correspondence should be addressed: Department of Chemistry, University of Oxford, Chemistry Research Laboratory, Mansfield Road, Oxford OX1 3TA, United Kingdom. Tel.: 44-1865-275625; Fax: 44-1865-275674; E-mail: christopher.schofield@chem.ox.ac.uk.

---

EXPERIMENTAL PROCEDURES

Protein Expression and Purification—The *ceas* gene was PCR amplified and subcloned directly as an NdeI/BamHI fragment into the pET24a(+) vector (Novagen). Primers used were: forward, 5′-acgct

---

1 The abbreviations used are: *orf*, open reading frame; CEAS, $N^{2}$-(2-carboxyethyl)arginine synthase; ThP$_{2}$, thiamin diphosphate; PDC, yeast pyruvate decarboxylase; ESI-MS, electrospray ionization mass spectrometry; ZmPDC, *Zymomonas mobilis* pyruvate decarboxylase.

This paper is available on line at http://www.jbc.org 5685

5686 Crystal Structure of Carboxyethylarginine Synthase

![Chemical Structures](chemical_structures.png)

FIG. 1. Biosynthetic pathway leading to clavulanic acid. BLS, β-lactam synthetase; PAH, proclavaminic amidino hydrolase; CAS, clavaminic synthase; CAD, clavaldehyde dehydrogenase; 2-OG, 2-oxoglutarate.

catatgtcccgtgta-3′; reverse, 5′-ggacatgagcggttccccgactccgtcagtacc-3′. Following amplification, the integrity of *ceas* was confirmed by DNA sequencing. The *ceas/pET24a(+)* construct was transformed into *Escherichia coli* BL21(DE3) and grown at 37 °C in 2TY medium containing 30 μg/ml kanamycin. Protein production was induced at an $A_{600}$ of 0.8 by addition of isopropyl-β-D-thiogalactopyranoside to a final concentration of 1.0 mM, following reduction of the temperature to 30 °C. CEAS was purified by sequential use of Q-Sepharose and phenyl-Resource chromatography. Size exclusion chromatography (Superdex S200) was used as a final step to yield CEAS of >95% purity by SDS-PAGE analysis. Electrospray ionization mass spectrometry (ESI-MS) revealed that the mass of the isolated CEAS was consistent with loss of N-terminal methionine from the predicted amino acid sequence (observed, 60,776 Da; calculated without N-terminal methionine, 60,776 Da). N-terminal amino acid sequence analysis (observed, SRVSTAPSGK) confirmed the identity of the protein and the loss of the N-terminal methionine. The activity of purified CEAS was confirmed by high performance liquid chromatography assay and comparison with known synthetic standards.

**Selenomethionine-substituted Protein Expression**—Selenomethionine-substituted CEAS was produced using a metabolic inhibition protocol and LeMaster media supplemented with 50 mg/liter L-selenomethionine. Selenomethionine incorporation was >95% by ESI-MS.

**Native Molecular Weight Determination**—The “native” molecular weight of CEAS was determined by size exclusion chromatography (Superdex 200 HR). Calibration was carried out using cytochrome c (12 kDa), chymotrypsin (25 kDa), ovalbumin (43 kDa), bovine serum albumin (67 kDa), aldolase (158 kDa), and blue dextran (2000 kDa) (Gel Filtration Calibration kit, Amersham Biosciences). An elution volume parameter ($K_{\text{av}}$) was calculated for each of the calibration proteins and a calibration curve was constructed. By calculating $K_{\text{av}}$ for CEAS the native molecular weight was established.

**Crystallization**—Crystallization conditions were initially sought by the vapor diffusion method, using commercially available crystallization screens. Hanging drops of 2 μl of protein (10 mg/ml), containing a 3-fold excess of thiamin diphosphate, and 2 μl of well solution were suspended at 17 °C over a well solution containing 1.6 M ammonium sulfate and 0.1 M HEPES, pH 7.4. Under these conditions, crystals of two different morphologies grew over a period of 2 weeks. The crystals belonged to two different space groups, however, the shape of the crystals was not a determinant of the space group. The majority of the crystals belonged to space group $P2_12_12_1$, with unit cell dimensions $a = 117.7 \, \AA$, $b = 127.3 \, \AA$, $c = 196.8 \, \AA$. The remainder of the crystals belonged to space group $P2_12_12$, with unit cell dimensions $a = 123.1 \, \AA$, $b = 187.2 \, \AA$, $c = 198.6 \, \AA$.

**Crystallographic Data Collection and Structure Solution**—Crystals were cryocooled by plunging into liquid nitrogen and x-ray data were collected at 100 K using a nitrogen stream. Cryoprotection was accomplished by sequential transfer into solutions of 1.8 M ammonium sulfate, containing 10% (v/v) glycerol and 25% glycerol. Data from selenomethionine-substituted crystals was collected on beamline BM14 at the European Synchrotron Radiation Facility, Grenoble, France, using a MarCCD detector. All other data were collected on beamlines 14.1 and 14.2 of the Synchrotron Radiation Source, Daresbury, UK. The data were processed using MOSFLM (28) and the CCP4 suite of programs (29).

The selenomethionine-substituted crystal used for the structure solution belonged to the $P2_12_12_1$ space group. Using SHELXD (30), 40 selenium positions were determined, corresponding to four molecules per asymmetric unit, excluding the N-terminal methionine that was not present by N-terminal sequencing and ESI-MS. Phasing and density modification were performed using CNS (31). The phases had a figure of merit of 0.38 over the resolution range to 2.35 Å, which rose to 0.94 following density modification. Data collection, phasing, and refinement statistics are shown in Table I.

**Refinement**—An initial model was built using the program O (32). Refinement of this model against the selenomethionine-substituted data (remote wavelength) using a strict non-crystallographic symmetry relationship was performed using CNS. One cycle of simulated annealing followed by grouped $B$-factor refinement brought the $R_{\text{free}}$ to 30.5%. 5% of the reflections were excluded for calculation of $R_{\text{free}}$. Further rounds of refinement using REFMAC5 (33), including individual $B$-factor refinement and the addition of solvent molecules and ThP$_2$ cofactor, brought the conventional $R$-factor to 15.7% and the $R_{\text{free}}$ to 19.8%. Non-crystallographic symmetry restraints were used throughout refinement. The current model contains residues 12–572 for all four molecules in the asymmetric unit, excluding residues 182–184 for which the electron density was unclear. In total the final model contains

Crystal Structure of Carboxyethylarginine Synthase

**TABLE I**
Data collection and refinement statistics

| Dataset | Se λ1 | Se λ2 | Native | P2₁2₁2 |
|---------|-------|-------|--------|---------|
| **Data collection** |  |  |  |  |
| Space group | $P2_{1}2_{1}2_{1}$ | $P2_{1}2_{1}2_{1}$ | $P2_{1}2_{1}2_{1}$ | $P2_{1}2_{1}2$ |
| Wavelength (Å) | 0.9788 | 0.8856 | 1.488 | 0.978 |
| Unit cell ($a$, $b$, $c$ (Å)) | 117.7, 127.3, 196.8 | 117.8, 127.4, 197.0 | 117.7, 127.3, 197.1 | 123.1, 187.2, 198.6 |
| Resolution range$^{a}$ (Å) | 35.38–2.60 (2.74–2.60) | 32.44–2.35 (2.48–2.35) | 38.35–2.35 (2.48–2.35) | 35.05–2.45 (2.58–2.45) |
| No. of unique observations$^{a}$ | 87,762 (12,377) | 119,393 (16,804) | 114,144 (11,387) | 168,019 (24,074) |
| No. of total observations$^{a}$ | 745,323 (98,790) | 690,214 (119,393) | 420,132 (23,614) | 683,385 (92,570) |
| Completeness$^{a}$ (%) | 96.2 (94.1) | 96.7 (94.2) | 92.3 (63.7) | 99.8 (98.9) |
| Multiplicity$^{a}$ | 8.5 (8.0) | 5.8 (5.0) | 3.7 (2.1) | 4.1 (3.8) |
| $R_{\text{merge}}$$^{a}$ (%) | 8.8 (16.0) | 8.1 (18.9) | 5.4 (11.5) | 8.3 (34.7) |
| $\langle I/\sigma(I)\rangle$$^{a}$ | 6.8 (4.2) | 7.6 (3.8) | 10.6 (5.8) | 8.7 (2.2) |
| Overall phasing power (centric/acentric) | (1.7/1.5) | (1.1/1.0) |  |  |

| **Refinement** |  |  |  |
|-----------------|-------|--------|---------|
| $R$ factor (%) | 15.7 | 15.7 | 17.6 |
| $R_{\text{free}}$ (%) | 19.8 | 19.5 | 21.6 |
| Rmsd bond length (Å) (angle (°)) | 0.012 (1.34) | 0.012 (1.34) | 0.014 (1.47) |
| $B$-factors$^{b}$ (Å²) | 21.0, 22.7, 24.0 | 17.5, 18.7, 18.2 | 30.9, 32.0, 25.4 |
| PDB ID number | 1UPA | 1UPB | 1UPC |

$a$ Numbers in parentheses represent the value in the highest resolution shell.

$b$ $B$-factors in order for main chain, side chain, and solvent atoms.

16,616 non-hydrogen protein atoms out of a theoretical maximum of 17,144. According to Procheck (34) there were no Ramachandran outliers.

The 2.35-Å native dataset was solved by rigid body refinement using REFMAC5, further rounds of refinement brought the $R$-factor to 15.7% and the $R_{\text{free}}$ to 19.5%. The $P2_{1}2_{1}2$ crystal form was solved by molecular replacement with the 2.35-Å selenomethionine structure using CNS. There were six molecules per asymmetric unit. Refinement of this data was performed using REFMAC5.

Figures were prepared with Molscript (35), Bobscript (36), and Raster3D (37).

---

**RESULTS AND DISCUSSION**

**Crystallization—**Diffraction quality CEAS crystals were obtained in the presence of ThP₂, using ammonium sulfate as precipitant, whereas poor quality crystals that did not diffract x-rays were obtained in the absence of ThP₂ under a variety of conditions. The useful crystals grew in two morphologies, rods and many-sided polyhedra. The majority of the crystals were of the polyhedral form. The crystals belonged to two different space groups, $P2_{1}2_{1}2_{1}$ and $P2_{1}2_{1}2$, with differing cell dimensions. Both space groups were observed for the polyhedral-shaped crystals although as yet only the $P2_{1}2_{1}2_{1}$ space group has been observed for the rod-shaped crystals.

**Structure Solution—**The structure of CEAS has been determined to 2.35-Å resolution in the $P2_{1}2_{1}2_{1}$ crystal form using the multiple anomalous dispersion method and the $P2_{1}2_{1}2$ form was subsequently solved to 2.45-Å resolution by molecular replacement. In both crystal forms a similar tetrameric arrangement of CEAS molecules was observed (Fig. 2). In the $P2_{1}2_{1}2_{1}$ form the tetramer was comprised of the four molecules in the asymmetric unit, whereas the $P2_{1}2_{1}2$ form contained one tetramer in the asymmetric unit and one dimer that formed a tetramer by association about a crystallographic 2-fold rotation axis.

**Monomer Structure—**Each CEAS subunit consists of three domains, labeled α, β, and γ as in the structure of PDC (Fig. 3) (38). The overall structure and fold of CEAS is similar to that of PDC (Fig. 4) and is reflected in a root mean square deviation of 1.95 Å over 372 Ca atoms (66% of CEAS residues). The α domain (residues 12–193) is the most similar to its equivalent in PDC (root mean square deviation 1.50 Å over 150 Ca atoms, 83% of CEAS residues in this domain), whereas the β domain (194–376) is the least similar (root mean square deviation 1.63 Å over 109 Ca atoms, 59.9% of CEAS residues). The most substantial difference between the α domains is the complete absence in CEAS of the second α helix of PDC. The lower

**Oligomerization—**The observation of a tetramer in the crys-

5688 Crystal Structure of Carboxyethylarginine Synthase

FIG. 3. Structure of the CEAS monomer.

![Structure Image](image_url)

Alpha domain  
Gamma domain  

Beta domain  

N

FIG. 4. Comparison of the structures of CEAS (left) and yeast pyruvate decarboxylase (right).

![Comparison Image](image_url)

tal form is in agreement with gel filtration chromatographic analysis, which showed CEAS to exist in two oligomeric solution states. The calculated approximate native molecular masses of 240 and 130 kDa correspond to tetrameric and dimeric forms of CEAS (60,776 Da after N-terminal methionine loss). Relative peak areas suggest that, under the conditions of analysis, these forms exist in approximately equal quantities (Fig. 5).

The tetramer consists of two tightly bound dimers (root mean square deviation 0.19 Å over all Cα atoms), which are more loosely bound together to form a tetramer. The tetramer has a size of ~115 Å on the longest dimension and 60 Å on the shortest. Monomer-monomer contacts occur exclusively between α and γ domains, whereas dimer-dimer contacts to form the tetramer mainly feature the β domain. Internally, each tightly bound dimer buries a surface area of ~7400 Å², whereas each monomer buries a surface area of 1950 Å² with the closest monomer of the other dimer and 460 Å² with the further monomer. Altogether, the dimer-dimer interaction buries a surface area of ~4600 Å². Each dimer is stabilized by a large number of hydrogen bonds. By contrast, there are very few hydrogen bonds at the dimer-dimer interface, although there are a considerable number of crystallographic water molecules located in the sizeable cavity between the dimers, at the center of the tetramer. Whereas these water molecules may provide substantial stabilization, because of the greatly differing surface areas of interaction the tetramer is best described as a dimer of dimers.

It is not yet known for CEAS whether large conformational changes occur in the tetramer on substrate binding, as in yeast PDC, where an allosteric substrate activation process takes place involving an approximate 30° movement of the dimers with respect to each other (39). This rotation is asymmetrical with respect to the tetramer and results in a separation of one pair of β domains, whereas the other pair remain associated much as in the starting tetramer. The assembly of the CEAS tetramer is most similar to that of PDC from *Zymomonas mobilis* (ZmPDC) (40), which does not behave in such a manner.

Crystal Structure of Carboxyethylarginine Synthase 5689

FIG. 5. Native gel filtration chromatogram showing the solution oligomerization state of CEAS.

and in which the tetramer is symmetrically arranged. By comparison to yeast PDC the dimers of the ZmPDC tetramer are rotated ~30° with respect to each other, but about the central point of the tetramer. In contrast to yeast PDC, this tetrameric arrangement is much more closely associated, hindering movement, and for ZmPDC has been ascribed as the activated conformation. Based on this comparison it seems likely that CEAS would not undergo tetrameric rearrangement although further investigation will be needed to confirm this supposition.

ThP₂ Binding Site—The ThP₂ binding region is situated across two subunits of the closely associated dimer, as with related enzymes such as PDC. The γ domain of one subunit and the α domain of the other subunit contribute to binding, with two ThP₂ binding regions per dimer. ThP₂ binding is facilitated by a combination of hydrogen bonding and formation of a complex with a single Mg²⁺ ion. The Mg²⁺ ion is coordinated octahedrally by the side chains of Asp-463 and Asn-490, the backbone carbonyl of Thr-492, two diphosphate oxygens of ThP₂ and a water molecule, also bound to the carbonyl oxygen of Val-488 (Fig. 6). This environment is similar to the characteristic ThP₂/Mg²⁺ binding motif identified as a conserved sequence in all ThP₂ enzyme structures (41). This sequence is found entirely in the γ domain and contributes to the binding of the diphosphate region of the ThP₂. Further hydrogen bonding of the diphosphate oxygens occurs directly to the backbone nitrogens of Phe-413, Leu-495, and Gly-465, all from the γ domain. The aminopyrimidine “end” of the ThP₂ is associated with non-conserved regions of both the same γ domain, and the α domain of the other monomer in the dimer. It is anchored via hydrogen bonds between N-4′ and the carbonyl oxygen of Ser-43, and between N-1′ and the side chain of Glu-57 of the α domain of the other monomer.

The ThP₂ cofactor is observed in a “V” conformation (Fig. 6), bent around the hydrophobic side chain of Phe-438. Structural information on other ThP₂-dependent enzymes indicates that a single hydrophobic residue is always found in a similar position, possibly aiding distortion of ThP₂ into this high potential energy state not normally witnessed outside a protein environment (23). The type of residue carrying out this function is specific to an individual enzyme, and evidence from mutation studies on yeast PDC suggests that it may have an important effect on mechanism (42).

Potential Glyceraldehyde-3-P and Arginine Binding Sites—In all the refined structures, a sulfate anion could be identified in the electron density at a distance of ~6 Å from the nucleophilic C-2 carbon of ThP₂. It is likely that this sulfate occupies the binding position of the phosphate of glyceraldehyde-3-P. The sulfate is bound at one oxygen by the side chains of Arg-414 and His-415, at another by Tyr-271, and at a third by His-120 from the other monomer in the dimer (Fig. 7). The fourth sulfate oxygen is not bound directly to the protein and is located at a distance of 4.8 Å from the nucleophilic C-2 of ThP₂. This distance is ideal for a fully extended four-bond link between these two atoms as might be expected from the reaction intermediate formed by ThP₂ and glyceraldehyde-3-P (Fig. 8).

A similar phosphate binding site has been identified in the crystal structure of transketolase complexed with erythrose 4-phosphate (43). The phosphate is anchored in a pocket of two arginine side chains, a histidine and a serine. The phosphate is bound further from the C-2 of ThP₂ than in CEAS, at a distance of ~9 Å, which fits with the larger size of the substrate over the glyceraldehyde-3-P of CEAS. The structure of yeast acetoxyacid synthase also reveals a phosphate bound close to the C-2 of ThP₂ (44). However, the phosphate ligands differ and the authors propose this as a binding site for the carboxyl group of the pyruvate substrate.

The active site cavity surrounding the proposed glyceraldehyde-3-P binding position is largely hydrophobic, containing the side chains of Ile-410 and Leu-495. The exceptions in this region are Gln-121 and Ser-436. It is also possible that, in the presence of substrate, the side chain of Arg-36 would move into a position capable of participating in the reaction. Overall, however, the CEAS active site is remarkable through its apparent lack of polar residues positioned to assist catalysis. By comparison, some of the active site residues with acidic/basic side chains proposed in a functional role in PDC are replaced in CEAS with neutral amino acids, e.g. His-115 by Gln-121, and Glu-477 by Ile-496. It is therefore difficult to assign distinct mechanistic functions to particular residues in the CEAS active site and it appears that CEAS must rely heavily on ThP₂ for general acid/base catalysis.

Analysis of the external region of the active site channel reveals a number of residues that may facilitate arginine binding. Furthest from the ThP₂, Tyr-298 and Asp-113 could potentially bind the positively charged guanidino group. Nearer the proposed phosphate binding site, Arg-303 and Asp-301 may participate in arginine binding.

Mechanistic Discussion—In their assignment of the function

5690 Crystal Structure of Carboxyethylarginine Synthase

FIG. 6. The ThP₂ binding site. ThP₂ is shown as a ball-and-stick representation in yellow, surrounded by its ligands in the active site of CEAS. An mF₀ – DF₀ (33) difference electron density map, calculated after random model perturbation and refinement with ThP₂ omitted to remove model bias, is shown contoured around the ThP₂ at 3.0 σ in red. A 2mF₀ – DF₀ electron density map is shown contoured around the CEAS residues at 1.0 σ in blue.

FIG. 7. The potential glyceraldehyde-3-P binding site. The sulfate molecule proposed to occupy the glyceraldehyde-3-P phosphate binding position is shown as a ball-and-stick representation in yellow, as is the ThP₂ molecule. Protein residues that bind the sulfate molecule are shown, surrounded by a 2mF₀ – DF₀ (33) electron density map contoured at 1.0 σ in blue. The sulfate molecule and ThP₂ are surrounded by an mF₀ – DF₀ difference electron density map, calculated after random model perturbation and refinement with these atoms omitted to remove model bias, contoured at 3.0 σ in red.

of CEAS, Khaleeli *et al.* (12) proposed an outline mechanism for the condensation of D-glyceraldehyde-3-P with L-arginine, involving initial nucleophilic attack of the ThP₂ ylide on the aldehyde of D-glyceraldehyde-3-P. Subsequent elimination and tautomerization then allows loss of the hydroxyl function at the C-2 position. Prior labeling studies imply that addition of arginine to glyceraldehyde-3-P occurs with retention of configuration at C-2 (45, 46), suggesting an elimination-addition process during formation of the C-N bond. In the final step, the acyl-ThP₂ intermediate must be hydrolyzed to release the N²-(2-carboxyethyl)arginine product.

The absence of residues capable of general acid/base catalysis at the active site of CEAS suggests that the 4′ amino group of the aminopyrimidine ring of ThP₂ plays a key role. The structure shows that the suggested binding/polarizing site for the phosphate of D-glyceraldehyde-3-P could position the aldehyde of this substrate close to the nucleophilic carbon of ThP₂. Considerable work on the early stages of ThP₂ reaction mechanisms has shown that formation of the “activated” ThP₂, by deprotonation at C-2, occurs via an intramolecular proton transfer to the 4′ amino nitrogen of the aminopyrimidine ring (38, 47). The stabilized negative charge at C-2 can then effect a nucleophilic attack on a substrate carbonyl group.

Following such a nucleophilic attack by ThP₂ at C-1 of glyceraldehyde-3-P (Fig. 8), proton transfer from C to O, via the amino group of ThP₂, may occur to give a C-1 anion. Studies on deamino ThP₂ analogues have indicated that the 4′ amino group of ThP₂ then stabilizes the negatively charged oxygen formed at the α-carbon position (48–50), possibly via partial protonation. The proposed role for the 4′-amino group is supported by freeze-quench crystallographic studies on ThP₂-substrate adducts with transketolase (51). Although the geometry

Crystal Structure of Carboxyethylarginine Synthase

![Chemical Reaction Diagram](attachment:chemical-reaction-diagram.png)

FIG. 8. Proposed outline biosynthetic mechanism of CEAS. Note that the stage at which L-arginine binds and the details of proton transfers and eliminations are uncertain.

of the adduct does not seem to satisfactorily allow for such stabilization, during transition state formation, the change in hybridization from $sp^2$ to $sp^3$ allows for a more favorable interaction between the oxygen and 4′ amino group.

After formation of the enamine/carbanion, two distinct reaction pathways appear to occur in the mechanisms of most ThP₂-dependent enzymes: either the enamine/carbanion is protonated, as in PDC, or it is not, such as in transketolase, where it reacts with another substrate molecule. This fundamental difference in mechanism is reliant on the presence of a proton donor close to the enamine/carbanion position. In PDC, the Glu-477 residue has been shown to fulfill this function, via a bridging water molecule (52). In benzoylformate decarboxylase, His-281 has been shown to perform the same role, possibly with the aid of Ser-26 (53). There appears to be no such proton donor present in CEAS where an isoleucine occupies an equivalent position to Glu-477 of PDC. It therefore seems reasonable that with CEAS the carbanion subsequently reacts to eliminate the C-2 hydroxyl group to give an enol/enolate.

The three positively charged residues binding the glyceraldehyde-3-P phosphate are bound to no other negatively charged atom, and the consequent polarization of glyceraldehyde-3-P would promote phosphate elimination. This elimination would give an α,β-unsaturated acyl-ThP₂ species, which could act as a Michael acceptor for L-arginine. Possible binding sites for arginine could place the nucleophilic amino group in a suitable position for such an addition. Finally, hydrolysis of the acyl-ThP₂ complex would leave the β-amino acid product, N²-(2-carboxyethyl)arginine. Although details of the steps, including proton transfers are uncertain, it seems likely that the aminopyrimidine ring of ThP₂ plays a key role in some of the steps, possibly including elimination of both water and phosphate, and addition of arginine.

Acknowledgments—We thank Dr. Martin Walsh of beamline BM14 of the ESRF, Grenoble, France, and the staff of beamlines 14.1 and 14.2 of the SRS, Daresbury, UK, for expert assistance. We also thank the staff at the Oxford Protein Production Facility for help with high throughput crystallization.

**REFERENCES**

1. Matagne, A., Dubus, A., Galleni, M., and Frère, J.-M. (1999) Nat. Prod. Rep. **16**, 1–18
2. Jensen, S. E., and Paradkar, A. S. (1999) Antonie Leeuwenhoek Int. J. Gen. Mol. Microbiol. **75**, 125–133
3. Baggaley, K. H., Brown, A. G., and Schofield, C. J. (1997) Nat. Prod. Rep. **14**, 309–333
4. Brown, R. P., Aplin, R. T., and Schofield, C. J. (1996) Biochemistry **35**, 12421–12432
5. Knowles, J. R. (1985) Acc. Chem. Res. **18**, 97–104
6. Bentley, P. H., Brooks, G., Gilpin, M. L., and Hunt, E. (1979) Tetrahedron Lett. **20**, 1889–1890
7. Mellado, E., Lorenzana, L. M., Rodriguez-Saiz, M., Diez, B., Liras, P., and Barredo, J. L. (2002) Microbiol. SGM **148**, 1427–1438
8. Li, R. F., Khaleeli, N., and Townsend, C. A. (2000) J. Bacteriol. **182**, 4087–4095
9. Hodgson, J. E., Fosberry, A. P., Rawlinson, N. S., Ross, H. N., Neal, R. J., Arnell, J. C., Earl, A. J., and Lawlor, E. J. (1995) Gene (Amst.) **166**, 49–55
10. Jensen, S. E., Alexander, D. C., Paradkar, A. S., and Aidoo, K. A. (1993) in Industrial Microorganisms: Basic and Applied Molecular Genetics (Baltz, R. H., Hegeman, G. D., and Skatrud, P. L., eds) pp. 169–176, American Society for Microbiology, Washington, DC
11. Kershaw, N. J., McNaughton, H. J., Hewitson, K. S., Hernandez, H., Griffin, J., Hughes, C., Greaves, P., Barton, B., Robinson, C. V., and Schofield, C. J. (2002) Eur. J. Biochem. **269**, 2052–2059
12. Khaleeli, N., Li, R., and Townsend, C. A. (1999) J. Am. Chem. Soc. **121**, 9223–9224
13. McNaughton, H. J., Thirkettle, J. E., Zhang, Z., Schofield, C. J., and Jensen, S. E. (1998) J. Chem. Soc. Chem. Commun. **21**, 2325–2326
14. Bachmann, B. O., Li, R., and Townsend, C. A. (1998) Proc. Natl. Acad. Sci. U. S. A. **95**, 9082–9086
15. Nicholson, N. H., Baggaley, K. H., Cassels, R., Davison, M., Elson, S. W., Fulston, M., Tyler, J. W., and Woronicki, S. R. (1994) J. Chem. Soc. Chem. Commun. **11**, 1281–1282
16. Baldwin, J. E., Adlington, R. M., Bryans, J. S., Bringhen, A. O., Coates, J. B., Crouch, N. P., Lloyd, M. D., Schofield, C. J., Elson, S. W., Baggaley, K. H., Cassels, R., and Nicholson, N. (1991) Tetrahedron **47**, 4089–4100
17. Wu, T.-K., Busby, R. W., Houston, T. A., McIlwaine, D. B., Egan, L. A., and Townsend, C. A. (1995) J. Bacteriol. **177**, 3714–3720
18. Elson, S. W., Baggaley, K. H., Davidson, M., Fulston, M., Nicholson, N. H., and Risbridger, G. D. (1993) J. Chem. Soc. Chem. Commun. **15**, 1212–1214
19. Aidoo, K. A., Wong, A., Alexander, D. C., Rittamer, R. A., and Jensen, S. (1994) Gene (Amst.) **147**, 41–46
20. Miller, M. T., Bachmann, B. O., Townsend, C. A., and Rosenzweig, A. C. (2001) Nat. Struct. Biol. **8**, 684–689
21. Elkins, J. M., Clifton, I. J., Hernandez, H., Doan, L. X., Robinson, C. V., Schofield, C. J., and Hewitson, K. S. (2002) Biochem. J. **366**, 423–434
22. Zhang, Z., Ren, J., Stammers, D. K., Baldwin, J. E., Harlos, K., and Schofield, C. J. (2000) Nat. Struct. Biol. **7**, 127–133
23. Jordan, F. (2003) Nat. Prod. Rep. **20**, 184–201
24. Hasson, M. S., Muscate, A., McLeish, M. J., Polovnikova, L. S., Gerlt, J. A., Kenyon, G. L., Petsko, G. A., and Ringe, D. (1998) Biochemistry **37**, 9918–9930
25. Muller, Y. A., and Schulz, G. E. (1993) Science **259**, 965–967
26. Bertagnolli, B. L., and Hager, L. P. (1991) J. Biol. Chem. **266**, 10168–10173
27. Foulon, V., Antonenkov, V. D., Croes, K., Waelkens, E., Mannaerts, G. P., Van Veldhoven, P. P., and Casteels, M. (1999) Proc. Natl. Acad. Sci. U. S. A. **96**, 10039–10044
28. Leslie, A. G. (1999) Acta Crystallogr. Sect. D Biol. Crystallogr. **55**, 1696–1702
29. Collaborative Computational Project Number 4 (1994) Acta Crystallogr. Sect. D Biol. Crystallogr. **50**, 760–763
30. Schneider, T. R., and Sheldrick, G. M. (2002) Acta Crystallogr. Sect. D Biol. Crystallogr. **54**, 1772–1779
31. Brünger, A. T., Adams, P. D., Clore, G. M., DeLano, W. L., Gros, P., Grosse-Kunstleve, R. W., Jiang, J. S., Kuszewski, J., Nilges, M., Pannu, N. S., Read, R. J., Rice, L. M., Simonson, T., and Warren, G. L. (1998) Acta Crystallogr. Sect. D Biol. Crystallogr. **54**, 905–921
32. Jones, T. A., Zou, J. Y., Cowan, S. W., and Kjeldgaard, M. (1991) Acta Crystallogr. Sect. A **47**, 110–119
33. Murshudov, G. N., Vagin, A. A., Lebedev, A., Wilson, K. S., and Dodson, E. J. (1999) Acta Crystallogr. Sect. D Biol. Crystallogr. **55**, 247–255
34. Laskowski, R. A., MacArthur, M. W., Moss, D. S., and Thornton, J. M. (1993) J. Appl. Crystallogr. **26**, 283–291
35. Kraulis, P. J. (1991) J. Appl. Crystallogr. **24**, 946–950
36. Esnouf, R. M. (1997) J. Mol. Graph. Model. **15**, 132–134
37. Merritt, E. A., and Bacon, D. J. (1997) Methods Enzymol. **277**, 505–524
38. Arjunan, P., Umland, T., Dyda, F., Swaminathan, S., Furey, W., Sax, M., Farrenkopf, B., Gao, Y., Zhang, D., and Jordan, F. (1996) J. Mol. Biol. **256**, 590–600
39. Lu, G. G., Dobritzsch, D., Baumann, S., Schneider, G., and König, S. (2000) Eur. J. Biochem. **267**, 861–868
40. Dobritzsch, D., König, S., Schneider, G., and Lu, G. (1998) J. Biol. Chem. **273**, 20196–20204
41. Hawkins, C. F., Borges, A., and Perham, R. N. (1989) FEBS Lett. **255**, 77–82
42. Guo, F. S., Zhang, D. Q., Kahyaoglu, A., Farid, R. S., and Jordan, F. (1998) Biochemistry **37**, 13379–13391
43. Nilsson, U., Meshalkina, L., Lindqvist, Y., and Schneider, G. (1997) J. Biol. Chem. **272**, 1864–1869
44. Pang, S. S., Duggleby, R. G., and Guddat, L. W. (2002) J. Mol. Biol. **317**, 249–262
45. McIlwaine, D. B., and Townsend, C. A. (1993) J. Chem. Soc. Chem. Commun. **17**, 1346–1347
46. Basak, A., Salowe, S. P., and Townsend, C. A. (1990) J. Am. Chem. Soc. **112**, 1654–1656
47. Kern, D., Kern, G., Neef, H., Tittmann, K., Killenberg-Jabs, M., Wikner, C., Schneider, G., and Hubner, G. (1997) Science **275**, 67–70
48. Schellenberger, A. (1998) Biochim. Biophys. Acta Protein Struct. **1385**, 177–186
49. Schellenberger, A. (1990) Chem. Ber. **123**, 1489–1494
50. Golbik, R., Neef, H., Hubner, G., König, S., Seliger, B., Meshalkina, L., Kochetov, G. A., and Schellenberger, A. (1991) Bioorg. Chem. **19**, 10–17
51. Fiedler, E., Thorell, S., Sandalova, T., Golbik, R., König, S., and Schneider, G. (2002) Proc. Natl. Acad. Sci. U. S. A. **99**, 591–595
52. Lobell, M., and Crout, D. H. G. (1996) J. Am. Chem. Soc. **118**, 1867–1873
53. Polovnikova, E. S., McLeish, M. J., Sergienko, E. A., Burgner, J. T., Anderson, N. L., Bera, A. K., Jordan, F., Kenyon, G. L., and Hasson, M. S. (2003) Biochemistry **42**, 1820–1830

Enzyme Catalysis and Regulation: Crystal Structure and Mechanistic Implications of $N^{2}$-(2-Carboxyethyl)arginine Synthase, the First Enzyme in the Clavulanic Acid Biosynthesis Pathway

Matthew E. C. Caines, Jonathan M. Elkins, Kirsty S. Hewitson and Christopher J. Schofield

J. Biol. Chem. 2004, 279:5685-5692. doi: 10.1074/jbc.M310803200 originally published online November 17, 2003

---

Access the most updated version of this article at doi: [10.1074/jbc.M310803200](http://dx.doi.org/10.1074/jbc.M310803200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- When this article is cited
- When a correction for this article is posted

[Click here](https://www.jbc.org/site/alerts) to choose from all of JBC's e-mail alerts

This article cites 52 references, 10 of which can be accessed free at [http://www.jbc.org/content/279/7/5685.full.html#ref-list-1](http://www.jbc.org/content/279/7/5685.full.html#ref-list-1)
