
THE MECHANISMS OF IMMUNE DIVERSIFICATION AND THEIR DISORDERS

**Jean-Pierre de Villartay, Alain Fischer and Anne Durandy**

Three molecular mechanisms contribute to the diversity of the immune repertoire of B and T cells: V(D)J recombination generates the primary repertoire in both cases, whereas class-switch recombination (CSR) and somatic hypermutation (SHM) improve the quality of the B-cell response after antigen triggering. These three mechanisms involve marked DNA damage and modification, which require a fully competent cellular DNA-repair machinery. Defects in V(D)J recombination, CSR or SHM reactions lead to immune deficiencies, the study of which has allowed the identification of genes that are central to these processes. The inability to properly manage DNA damage/modification during V(D)J recombination can also promote the development of cancer, as shown by the emergence of B-cell lymphomas in patients with a partial Artemis defect.

---

SOMATIC HYPERMUTATION  
Mutations occurring with high frequency in the V region of immunoglobulin genes, followed by positive selection of B cells.

GERMINAL CENTRES  
Lymphoid formations in secondary lymphoid organs that are formed by B-cell proliferation after antigen stimulation. CSR and SHM occur inside the germinal centres.

D éveloppement Normal et Pathologique du Système Immunitaire (INSERM U429), H ôpital Necker Enfants Malades, 149 rue de Sèvres, 75015 Paris, France.  
Correspondence to J.-P. d. V.  
e-mail: devillar@necker.fr  
doi:10.1038/nri1247

B and T cells recognize and respond to foreign antigens through specialized polymorphic membrane receptors: the B-cell receptor (BCR) and the T-cell receptor (TCR), respectively. The chromosomal organization of the genes that encode these molecules is highly conserved between the receptor-chain loci (the example of the immunoglobulin heavy-chain (IgH) locus is shown in FIG. 1), as well as between species. The variable antigen-recognition domains of these receptors are encoded by many scattered gene segments of three types — variable (V), diversity (D) and joining (J) — which are somatically rearranged, in appropriate cell lineages, before their expression¹. So, V(D)J recombination generates the diversity of B- and T-cell primary immune repertoires²⁻⁴. In the case of B cells, two additional mechanisms, which are triggered after antigen recognition, further optimize the antibody response⁵. Class-switch recombination (CSR) allows a previously rearranged IgH variable domain to be expressed in association with a different constant (C) region, leading to the production of different isotypes (IgG, IgA or IgE) — which mediate antigen elimination by different routes — without changing antibody specificity. The variable domains of immunoglobulins can also increase their affinity for antigen through the accumulation of SOMATIC HYPERMUTATIONS (SHMs) within the V gene segment. These two B-cell specific antibody-maturation processes take place after antigenic stimulation, in the GERMINAL CENTRES of peripheral lymphoid organs, whereas V(D)J recombination occurs in the bone marrow (B cells) and thymus (T cells). Primary immunodeficiencies in humans are a 'gold mine' of natural models for the study of immune system physiology⁶. Eighty-five genetically defined disorders have been identified within the past 10 years and these are classified according to the cellular compartment affected. In this review, we concentrate on two classes of immunodeficiency (FIG. 2): a subset of severe combined immunodeficiencies that are characterized by an absence of T and B cells (T-B-severe combined immunodeficiency, T-B-SCID), and the hyper IgM (HIGM) syndromes, as they both arise from defects in one of the three pathways of antigen-receptor diversification. These diseases are good examples of the power of molecular medicine in identifying and understanding key events that govern the development of the immune system, and also provide information on the importance of these pathways in generating efficient defence mechanisms

Bone marrow

![Diagram](#)

V(D)J recombination

Excised  
intervening DNA

Germlinal centre

CSR  
SHM

Excised  
intervening DNA  

New isotypes (such as IgG and IgA)  

Mutated  
V gene  

High-affinity antibody  

Figure 1 | Organization of the human immunoglobulin IgH locus and its somatic modifications to generate a primary and secondary B-cell repertoire. In its germline configuration, the variable domain of immunoglobulin heavy chain (IgH) is composed of clusters of gene segments encoding variable (V, orange boxes), diversity (D, green boxes) and joining (J, light blue boxes) exons. V(D)J recombination leads to the formation of a VDJ-rearranged productive gene unit that is formed by the physical link between these three exons. The intervening DNA is excised as circles. During class-switch recombination (CSR), a new rearrangement process leads to the association of the VDJ unit to another constant (C) region exon. The intervening DNA is also excluded from the chromosome as a circle. High-affinity antibodies are obtained through the accumulation of somatic hypermutations (SHMs) in the V genes (red box).

against various microorganisms. V(D)J recombination, CSR and SHM all proceed through DNA breaks. As DNA damage is a highly mutagenic event, when misrepaired it can be anticipated that specific defects in DNA-repair factors could participate in the onset of lymphoid malignancies in humans, as has been shown in several animal models over the past few years.

**V(D)J recombination**

Numerous excellent reviews have been published on V(D)J recombination recently²⁻⁴, so we only briefly discuss the molecular mechanisms of this process before focusing more on what has been learnt from the analysis of human conditions in which V(D)J recombination is defective. V(D)J rearrangements are site specific and occur through the recognition of RECOMBINATION SIGNAL SEQUENCES (RSSs) that flank all V, D and J gene segments. Schematically, the V(D)J recombination reaction can be divided into three steps (FIG. 3). The reaction is initiated by the two lymphoid-specific factors recombinase activating gene 1 (RAG1) and RAG2, which recognize and bind to RSSs that flank the gene segments to be rearranged, and introduce a DNA double-strand break at the junction between the heptamer and the coding sequence, leaving hairpin-sealed coding ends on the chromosome and phosphorylated, blunt signal ends that are excised from the chromosome. The resolution of DNA damage is carried out by the general DNA-repair machinery of the cell, specifically the non-homologous end-joining (NHEJ) pathway⁷. Briefly, the DNA double-strand break is first recognized by the DNA-dependent protein kinase (DNA-PK) complex,

formed by the Ku70–Ku80 heterodimer, which binds to DNA ends, and the DNA-PK catalytic subunit (DNA-PKcs), which belongs to the phosphatidylinositol 3-kinase (PI3K) family of kinases and requires DNA association for enzyme activity. DNA-PK probably activates Artemis for hairpin opening and DNA processing at coding ends before ligation (see later). During the final phase of V(D)J recombination, the lymphoid-specific terminal deoxynucleotidyl transferase (TDT) enzyme further increases joining diversity by adding non-templated nucleotides to processed coding ends. Finally, the XRCC4-DNA-ligase IV complex catalyses the ligation step.

**RAG1/RAG2 deficiencies** V(D)J recombination is not only the primary immune diversification mechanism, but it also constitutes a main checkpoint in the development of both B and T cells. This was first shown in RAG1/RAG2 targeted inactivation experiments in mice, which resulted in animals that were devoid of both lymphoid populations⁸,⁹. Subsequently, the same phenotype was identified in a subset of patients with SCID that lack T and B cells but have normal numbers of natural killer (NK) cells, and these patients were shown to have mutations in either the RAG1 or RAG2 genes¹⁰,¹¹. Analysis of RAG2 mutations in some of these patients showed that they were clustered in the same region, the proposed β-propeller structure of the amino-terminal moiety of RAG2 (REFS 12,13), which is the putative region of interaction with RAG1. Although both RAG1 and RAG2 proteins show a high degree of sequence conservation through evolution,

REVIEW S

| Bone marrow | Thymus | Periphery |
| --- | --- | --- |
| HSC | Pro-T cell | Mature T cell |
|  | γδ |  |
| CLP | Pre-T cell | αβ CD4⁺ |
|  | αβ | αβ CD8⁺ |
|  |  | NK |
|  | Pro-NK |  |
|  | Pro-B cell | Immature B cell |
|  |  | Mature B cell |
|  |  | IgM⁺ |
|  |  | IgG⁺, IgA⁺, for example |

V(D)J recombination deficiency

Mutations in:
- Rag1
- Rag2
- Artemis (RS-SCID)
- Other genes

CSR/SHM deficiency

Mutations in:
- CD40/CD40L
- AID
- UNG
- Other genes

Figure 2 | Blockade of lymphoid development in humans with severe combined immunodeficiency (SCID). The various lineages of the immune system arise from a common lymphoid progenitor (CLP), which differentiates from a haematopoietic stem cell (HSC) in the bone marrow. T cells further mature in the thymus, whereas B cells develop in the bone marrow. V(D)J - recombination deficiency (vertical, blunt-ended red arrows) caused by defects in recombinase-activating gene 1 (RAG1), RAG2 and Artemis leads to a T-B-SCID phenotype that is characterized by an arrest of B- and T-cell maturation at the stage of pro-B and pre-T cells, respectively, whereas natural killer (NK)-cell maturation is not affected. In these patients, the arrest of B-cell maturation occurs at the transition from cytoplasmic IgM⁻ pre-B-I cells to cytoplasmic IgM⁺ pre-B-II cells³⁴,¹²⁸. Mature B cells mature further in the germinal centres of peripheral lymphoid organs after antigen triggering. They switch their immunoglobulin isotype and acquire somatic mutations in the variable (V) genes. Defects in these two events (horizontal, blunt-ended red arrow), caused by mutations in the genes encoding CD40, CD40 ligand (CD40L), activation-induced cytidine deaminase (AID), uracil N-glycosylase (UNG) and possibly others, is a characteristic of hyper-IgM (HIGM) syndrome. CSR, class-switch recombination; SHM, somatic hypermutation; RS-SCID, radiosensitive severe combined immunodeficiency.

initial experiments had documented that, in both cases, a minimal region (known as the core) of each factor was necessary and sufficient for V(D)J recombination on extrachromosomal (plasmid) substrates in vivo, as well as DNA cleavage in vitro, leaving the role of the remaining portion of the protein (known as the dispensable region) an open issue¹⁴. It has been known, however, for some time that residues that are important for the cell-cycle-regulated stability of RAG2 are present in its dispensable region¹⁵. Moreover, the dispensable region seems to have important functions in endogenous immunoglobulin and TCR-gene rearrangements, as mutations in this region also result in T-B-SCID in humans¹⁶ (J.-P.d.V., unpublished observations). The issue of the function of the dispensable region was recently revisited by the development of two new Rag2-deficient mice¹⁷,¹⁸, in which the wild-type alleles were replaced by core-only gene segments by homologous recombination. The resulting Rag2-knock-in mice differ from the classical

Rag2-knockout mice and the T-B-SCID patients mentioned earlier in that the block in the development of B and T cells is only partial (one-third of the normal number of B cells are present and almost wild-type levels for T cells). A defect in the rearrangement of some immunoglobulin and TCR genes was nevertheless documented in these mice, which mainly affected V to DJ rearrangements at the IgH and TCR-β loci. This result confirmed previous studies showing that a core-RAG2 construct could not fully complement the defect in V(D)J recombination of the endogenous IgH locus in RAG2-deficient Abelson pre-B-cell lines¹⁹,²⁰. The carboxy-terminal region of RAG2 contains important signals for protein degradation and nuclear localization that might be crucial for its activity in vivo²¹,²². Another interesting possibility comes from the observation that the dispensable region of RAG2 has homology with plant homeodomain motifs, which are known to be regions of interaction with chromatin associated proteins¹²,²³. Such interactions might prove
to be important for the rearrangement of endogenous loci, as opposed to plasmid substrates. These observations indicate that although the core RAG2 suffices for the initial phases of V(D)J recombination in vitro, the carboxy-terminal region is required for its activity in more complex environments, such as in endogenous immunoglobulin and TCR loci.

HYPMORPHIC MUTATIONS, localized in the core regions of RAG1 and RAG2 were identified in patients with Omenn syndrome²⁴ — a disease that is characterized by the absence of circulating B cells but the presence of T cells in blood and in tissues, causing skin lesions that are similar to those found in GRAFT-VERSUS-HOST DISEASE⁶. Circulating and infiltrating T cells are poorly diversified, express activation markers and are skewed towards the T helper 2-type phenotype, which, together, indicates that a limited number of T-cell clones with autoreactivity emerge and proliferate in this otherwise compromised immune environment. So, Omenn syndrome provides the first evidence that mutations in factors of the V(D)J-recombinase machinery that preserve residual enzyme activity can result in clinical features that are distinct from T-B-SCID. However, the same RAG1/RAG2 mutations can cause either Omenn syndrome or T-B-SCID, suggesting that a second, as-yet-unidentified factor might be required for the development of Omenn syndrome²⁵.

**Artemis deficiency.** About half of T-B-SCID cases cannot be accounted for by mutations in either the RAG1 or RAG2 genes, although these cases have the same clinical and biological features. The immune deficiency in these patients is accompanied by increased cellular sensitivity to ionizing radiation (and is, therefore, known as radiosensitive, RS-SCID), which reflects a defect in DNA double-strand-break repair similar to that observed in SCID mice, which is caused by a mutation in the gene that encodes DNA-PKcs²⁶,²⁷. This defect is associated with an abortive V(D)J-recombination process, specifically affecting the formation of coding joints, although signal joints are spared²⁸. However, in contrast to SCID mice, the Ku70–Ku80–DNA-PKcs complex is present and functional in patients with RS-SCID²⁹. The putative affected gene was localized on the short arm of human chromosome 10 (REFS 30,31). Systematic in silico annotation of the large genomic sequences that cover this region led to the identification of a gene encoding an as-yet-unidentified protein, Artemis, which has low sequence identity with DNA-repair factors in yeast (PSO2) and mice (SNM1). RS-SCID was found to be caused by mutations in the gene encoding this ubiquitously expressed nuclear protein³¹⁻³⁵. Artemis is not essential for viability, as many patients with RS-SCID have complete loss-of-function alleles caused by homozygous genomic deletion of the first 4 exons³² and

### Figure 3 | V(D)J recombination.
**a** | During the initial phase of V(D)J recombination, the lymphoid-specific recombinase-activating gene 1 (RAG1)/RAG2 factors, together with ubiquitous DNA architectural proteins (high mobility group, HMG, proteins), recognize and bind to recombination signal sequences (RSSs) that flank all variable (V) and joining (J) segments and introduce a DNA double-strand break at the border of the RSS. On the chromosome, coding ends are left as hairpin-sealed structures, whereas signal ends, which are excised from the chromosome, are blunt and 5′ phosphorylated. The subsequent steps are taken care of by the DNA-repair machinery of the non-homologous end-joining (NHEJ) apparatus. **b** | The DNA-double-strand break is first identified by the DNA-dependent protein kinase (DNA-PK) complex (formed by the Ku70–Ku80 heterodimer and the DNA-PK catalytic subunit, DNA-PKcs). Before re-ligation, the hairpins at the coding ends are first opened, presumably by the Artemis-DNA-PKcs complex. **c** | The XRCC4–DNA-ligase IV complex carries out the ligation step. The terminal deoxynucleotidyl transferase (TDT) further increases the diversity of the coding joint by adding non-templated nucleotides (N). The initiation phase is abrogated in patients with T-B-severe combined immunodeficiency (T-B-SCID) that have RAG1 or RAG2 mutations. Mutations in Artemis (radiosensitive (RS)-SCID) or DNA-PKcs (SCID mice) both lead to an arrest at the second phase, such that hairpin structures at coding ends accumulate, precluding the formation of the coding joints. Signal-joint formation, which does not depend on a previous hairpin-opening phase, is not affected in these two conditions.

**HYPOMORPHIC MUTATIONS**
Mutations are considered hypomorphic when they do not result in a complete loss of function. Study of these mutations is sometimes the only way to link a disease to a particular gene defect when the complete loss of function is embryonic lethal (for example, DNA ligase IV).

**GRAFT-VERSUS-HOST DISEASE (GVHD).** The immune reaction that results from injection of allogenic T cells into an immunodeficient animal or human that is incapable of mediating graft rejection. One of the characteristics of GVHD is infiltration of tissues (skin, gut) by activated, reactive T cells.
Artemis-knockout mice have a similar phenotype to patients with RS-SCID with the noticeable exception of the presence of low numbers of CD4+ T cells in the periphery of some mice, which indicates the possible existence of alternative hairpin-cleavage activities that can compensate, to some extent, for the Artemis defect in mice³⁶. Two protein complexes are good candidates and were indeed proposed to be V(D)J hairpin openers. The RAG1-RAG2 complex itself, which remains on the broken DNA ends after the initial cut, has hairpin-opening activity *in vitro*⁴¹⁻⁴⁵. Another complex of the NHEJ apparatus formed by the NBS1, MRE11 and RAD50 proteins (the NMR complex) has been proposed as a second candidate, given that MRE11 has been shown to have hairpin-opening activity *in vitro*⁴⁶. Moreover, the NMR complex has been visualized at the site of broken DNA in the TCR-α locus in cells undergoing V(D)J recombination⁴⁷, indicating a possible role in the DNA-repair process. However, patients with NIJMEGEN BREAKAGE SYNDROME (NBS), caused by mutation of the *NBS1* gene, do not have any obvious defect in V(D)J recombination, which indicates that the NMR complex has a minimal role, if any, in this process⁴⁸. The same is true for patients with Ataxia-like disease (ATLD), which is caused by mutation of *MRE11*. Further studies are now required to establish whether these alternative hairpin-opening activities are uncovered only in the context of Artemis deficiency or if they are part of the physiological reaction. In addition, the role of Artemis in DNA double-strand-break repair, outside of V(D)J recombination, remains to be defined precisely, although it might have a function in DNA-end processing of some sort.

**DNA-ligase IV deficiency.** DNA-ligase IV, as well as XRCC4, is absolutely required for the V(D)J-recombination reaction as determined by gene targeting in mice⁴⁹⁻⁵¹. Moreover, the embryonic lethality of mice that are deficient in DNA ligase IV and XRCC4 shows that these two factors are essential for viability. This could explain why T-B-SCID in humans has never been associated with a complete loss-of-function of either of these two genes. Nevertheless, hypomorphic mutations in DNA ligase IV in humans cause immunodeficiency, developmental delay, chromosome instability and microcephaly — a phenotype that resembles NBS⁵². One puzzling observation, however, is that the immunodeficiency in these patients does not seem to be caused by a specific defect in the V(D)J-recombination process as expected. Indeed, signal-, but not coding-joint formation is affected in V(D)J-recombination assays in fibroblasts from these patients⁵³, and V(D)J junctions at TCR-α and TCR-β loci in T cells from these patients are normal (J.-P.d.V., unpublished observations). This indicates that DNA ligase IV (and probably the other NHEJ factors) is also required during the development of the immune system, at stages beyond the V(D)J recombination step, possibly to repair DNA damage that might occur during lymphocyte proliferation.

COMPLEMENTARITY-DETERMINING REGION (CDR). The hypervariable regions of an antibody molecule, consisting of three loops from the heavy chain and three from the light chain, that together form the antigen-binding site.

**A new V(D)J recombinase deficiency.** So far, nine proteins (including TDT) have been unequivocally identified as part of the V(D)J recombinase. Not surprisingly given the high complexity of this reaction, a recent report has indicated the existence of at least one other important factor<sup>54</sup>. This stemmed from the observation of a new human type of RS-SCID that is characterized by a low number of B and T cells. This condition is associated with a defect in V(D)J recombination, but there is no abnormality in the nine known components of the V(D)J recombinase. In contrast to previously studied patients with RS-SCID, both coding and signal joints are impaired, indicating that the defective factor, which has yet to be discovered, directly participates in the DNA-rejoining phase.

CSR and SHM

In germinal centres, in secondary lymphoid organs and after activation through the BCR and CD40, the two main events of antibody maturation occur (FIG. 5).

CSR involves recombination between two different switch (S) regions that are located upstream from each C region of IgH, with deletion of the intervening DNA.

Replacement of the Cμ region by a C region of another class of immunoglobulin (Cy1–4, Cα1–2 or Ce) results in the production of different isotypes (IgG1–4, IgA1–2 or IgE) with the same V region, and therefore, the same specificity and affinity for antigen. The nature of the produced isotype determines its activity (half-life, ability to bind Fc receptors or to activate complement) and the location to which it is delivered (such as IgA in the mucosa)<sup>55–58</sup>.

SHM introduces mutations in the V region and its flanking regions with high frequency (1 × 10⁻³ bases per generation). These mutations, which are essentially missense mutations and more rarely deletions or insertions, occur in the COMPLEMENTARITY-DETERMINING REGIONS (CDRs) and target specifically the Arg-Gly-Tyr-Trp motifs (Arg→Gly or Ala, Tyr→Thr or Cys, Trp→Ala or Thr). Normally, SHM is eventually followed by the positive selection of B cells that express a BCR with high affinity for antigen, whereas B cells that express a BCR with low affinity are deleted by apoptosis. This selection process occurs in close contact with follicular dendritic cells. SHM, therefore, results in the generation of efficient antibodies<sup>59–62</sup>.

CSR and SHM occur simultaneously, in germinal centres, but neither is a prerequisite for the other, as IgM can be mutated and IgG or IgA can be unmutated<sup>63,64</sup>. They share three main features. Both are triggered by the BCR/CD40 activation pathway in germinal centres. Both require transcription of the targeted (S or V) DNA. Transcription of S regions in CSR is induced by cytokines that are secreted by activated T cells. Each cytokine specifically activates the intron promoter that is located upstream from each S region, inducing the expression of germ-line transcripts after splicing of the intervening S regions<sup>57</sup>. During SHM, transcription is also required as the intronic enhancer and the promoter are required, and the mutation frequency is correlated with transcription efficiency<sup>65–67</sup>. DNA alterations are required for both processes. They are produced by activation-induced cytidine deaminase (see later).

The DNA-repair mechanism differs for CSR and SHM. In CSR, the repair machinery that joins the two broken S regions has been shown to involve the NHEJ enzymes, which act on DNA double-strand breaks, and the Ku proteins<sup>68</sup>. However, DNA-PK is dispensable for CSR to IgG1 (REF. 58) and normal CSR has been reported in SCID mice despite the lack of DNA-PK activity<sup>69</sup>, indicating that other DNA-repair factors could be required. In SHM, NHEJ enzymes are known to be dispensable<sup>70</sup> and DNA repair is probably carried out by the mismatch-repair enzymes mutS homologue 2 (MSH2) and mutL homologues 1 (MLH1) and post-meiotic segregation increased 2 (PMS2)<sup>71–74</sup> and/or error-prone polymerases ι, η, ζ and μ<sup>75–78</sup>.

**Hyper-IgM syndrome and CD40-activation defect.**

HIGM syndrome is a primary immunodeficiency, characterized by normal or increased serum IgM levels and low or absent IgG, IgA and IgE serum levels, indicating a

Figure 5 | Schematic representation of a germinal centre (GC) in which CSR and SHM of GC founder B cells (CD38⁺IgM⁺IgD⁺) occur in cooperation with activated CD4⁺ T cells. Both processes require the CD40 ligand (CD40L)-CD40 interaction and activation-induced cytidine deaminase (AID) activity. The selection process that occurs after somatic hypermutation (SHM) is achieved in close contact with follicular dendritic cells (FDCs). The stages in this process that are affected in the different molecular-defined hyper-IgM (HIGM) syndromes are shown. CD40L and CD40 deficiencies result in HIGM1 and HIGM3 syndromes, respectively. AID deficiency is responsible for HIGM2 syndrome. AR, autosomal recessive; X, X-linked.

HIGM2 and activation-induced cytidine deaminase.

Mutations in the gene encoding AID are responsible for human HIGM2 syndrome<sup>84</sup> and Aid-knockout mice have similar phenotypes<sup>91</sup> — markedly impaired CSR and defective SHM — providing evidence for the essential role of AID in both antibody-maturation processes<sup>84,91</sup>. AID is only expressed by germinal centre B cells<sup>92</sup> and by B cells undergoing *in vitro* CSR or SHM<sup>92–94</sup>. AID expression alone was sufficient to induce SHM and CSR on artificial substrates in fibroblasts<sup>95,96</sup>, which showed that AID is the only B-cell-specific factor that is required for CSR and SHM. Another pathway of B-cell maturation that occurs in chickens (immunoglobulin gene conversion) also depends on AID<sup>97</sup>. There is some evidence that CSR and SHM, although they can occur independently, could have, in part, a common pathway (FIG. 6). Both processes require transcription of the targeted DNA<sup>98,99</sup>. Sterile germ-line transcripts, which are induced by interleukins, are detected in AID-deficient B cells in mice<sup>91</sup> and in humans<sup>84</sup>, which indicates that AID has a role that is downstream of this transcription step. SHM and CSR are also both associated with DNA nicks<sup>100,101</sup>, which can subsequently lead to DNA double-strand breaks in the case of CSR<sup>102</sup>. However, the occurrence of SHM-induced DNA double-strand breaks in the V region remains controversial<sup>78,102–104</sup>. CSR has been shown to require the histone H2AX and the NBS1 proteins that are associated with DNA double-strand-break repair<sup>105</sup> and the NHEJ pathway, which is known to be involved in DNA double-strand-break repair<sup>68,106,107</sup>, although its role in CSR-DNA repair is now debated<sup>69</sup>. Petersen *et al.*<sup>105</sup> have shown that the formation of nuclear foci that have been associated with DNA double-strand-break repair does not occur properly in Aid-deficient mice<sup>105</sup>, indicating that AID is either involved in the formation of DNA breaks or in a post-cleavage event that is required for the formation of repair foci. Together, these data strongly indicate that AID acts downstream from DNA-target transcription and upstream from the DNA alterations that occur in CSR and SHM.

The mechanism of action of AID remains controversial. Because of the sequence similarity of AID with the RNA-editing protein APOBEC1 (REF. 92), Honjo’s group<sup>5,108</sup> proposed that AID could modify a messenger RNA encoding a common factor that is required for CSR and SHM, probably an endonuclease, although the possibility that AID could modify an RNA encoding a co-factor that is required for this activity cannot be excluded. The recently described requirement for *de novo* protein synthesis for CSR agrees with the RNA-editing hypothesis<sup>109</sup>. Alternatively, AID could directly edit DNA as shown in *E. coli* experiments<sup>110</sup>. In cellular assays, AID catalyses the deamination of deoxycytidine (dC) on single-stranded DNA<sup>111</sup>. Transcription enhances the AID-induced cytidine deamination of single-stranded DNA and targets its activity to the non-template strand<sup>112–114</sup>. However, APOBEC1 — a well-known RNA-editing enzyme — has the same effect on DNA in the same experimental model<sup>110,115</sup>. So far, no definitive proof has been provided for either DNA- or RNA-editing activity for AID in physiological conditions.

NEMO or AID genes. The molecular defect, although not yet defined, occurs downstream of AID activity, as CSR-induced DNA double-strand breaks normally occur in the S regions¹¹⁹. This observation indicates that HIGM4 syndrome might be caused by abnormal CSR-specific DNA repair, although a defect in switched B-cell survival cannot be excluded definitively.

### Lymphomas

DNA damage (modification) is a common feature of the three immune diversification mechanisms discussed here and represents a cost for the organism to be better protected from attack by foreign pathogens. As such, defects in these three reactions are mainly associated with immune deficiencies⁶. However, faulty DNA-processing mechanisms that result in genome instability are also general hallmarks of tumour development¹²⁰ and dysregulation of the V(D)J recombinase is already known to have an important role in leukaemogenesis¹²¹.

#### Figure 7 | NHEJ factors are genome caretakers.

Non-homologous end-joining (NHEJ)-defective allele crossed onto a Trp53⁻/⁻ background leads to the development of pro-B-cell lymphomas. These models recapitulate the pathway to neoplasia that was proposed by Kinzler and Vogelstein, which states that cancer development requires mutations in both alleles of both gatekeeper and caretaker genes. p53 is the gatekeeper and NHEJ factors are genome caretakers.

Another mystery is how AID specifically recognizes the immunoglobulin loci. This specific targeting presumably depends on co-factors, which are most probably widely expressed⁹⁵,⁹⁶. It has also been shown that Sµ and downstream S regions are specific targets for DNA cleavage. S regions contain short stretches of inverted repeats, which, after transcription, can allow the formation of secondary structures on single-stranded DNA⁵,¹¹⁶. The last step in the process, DNA repair, seems to be different for SHM and CSR. NHEJ enzymes (DNA-PK and the Ku70–Ku80 recombination complex) have been suggested to have a role in CSR¹⁰⁶,¹⁰⁷, although DNA-PK seems to be dispensable for CSR to IgG1 (REF. 58) and recent data have shown that SCID mice can undergo CSR despite lack of DNA-PK activity⁶⁹. It is possible that as-yet-unknown factors specifically participate in the last step of CSR¹¹⁷.

#### Uracil N-glycosylase

The recent description of a partial CSR defect and a skewed pattern of SHM in uracil N-glycosylase (Ung)-deficient mice¹¹⁸ strongly supports the hypothesis of a direct action of AID on DNA. According to the model proposed by Neuberger and colleagues¹¹⁰, AID could deaminate dC on one strand of DNA into deoxyuridine (dU) residues. After deglycosylation and removal of the misintegrated dU residues by UNG, apyrimidic endonuclease (APE) leads to the formation of DNA nicks, which is a necessary step for both CSR and SHM (FIG.6). The occurrence of DNA double-strand breaks required for CSR is not defined. AID might deaminate dC on the template strand in the region of transcription¹¹¹. Another unknown mechanism could also be involved. In UNG deficiency, CSR would be impaired as a consequence of the lack of glycosylation and the removal of dU residues. This observation opens the interesting possibility of an analogous gene defect in some of the unresolved HIGM syndromes in humans.

#### HIGM4 syndrome and other factors

HIGM4 syndrome is phenotypically similar to HIGM2 and is characterized by defective CSR, but normal SHM¹¹⁹. Patients with HIGM4 syndrome have normal expression of CD40L and CD40, and do not have mutations in
thought to be the catalytic region of the protein (see earlier). In this context, few B and T cells emerge. Interestingly, two out of four patients developed aggressive Epstein-Barr-virus-associated B-cell lymphomas that were similar to those found in the NHEJ/p53-deficient mouse models discussed earlier, further supporting the notion that Artemis is a tumour-suppressor factor. The same conclusions are drawn from the analysis of Artemis-knockout mice, in the context of a cell-cycle-checkpoint defect (such as p53 deficiency) (F. Alt, personal communication). Therefore, to establish whether or not Artemis deficiency could be associated with the onset of a broader range of tumours is an important issue. The careful follow-up of bone-marrow transplanted RS-SCID patients, as well as the survey of Artemis-knockout mice, should help to resolve this issue in the future.

### Conclusion

The study of natural mutants of the immune system has been helpful in the past in identifying new genes that are involved in the various aspects of the development of the immune system, as well as the processes of an immune response. As mentioned in this review, we are far from understanding all of the molecular and biochemical mechanisms that are involved. The development of new strategies to create new animal models by using high-throughput mutagenesis<sup>127</sup>, together with the survey of as-yet-unidentified immune deficiencies in humans, will undoubtedly shed new light on the subject in the near future. One important issue has emerged in the past several years, regarding the implication of the V(D)J/SHM/CSR factors in the development of haematological malignancies. Here, again, the careful analysis of these disorders in humans and the generation of new specific animal models will help us to understand of the broad spectrum of diseases, from immune deficiency to immune malignancies, for which defects in these factors are responsible.

#### Note added in proof

Recently, UNG deficiency has been shown in humans to be responsible for a new HIGM syndrome that is characterized by a marked defect in CSR and the same skewed pattern of SHM that is observed in Ung-deficient mice<sup>129</sup>.

---

1. Tonegawa, S. Somatic generation of antibody diversity. *Nature* **302**, 575–581 (1983).
2. Bassing, C. H., Swat, W. & Alt, F. W. The mechanism and regulation of chromosomal V(D)J recombination. *Cell* **109**, S45–S55 (2002).
3. Gellert, M. V(D)J recombination: RAG proteins, repair factors, and regulation. *Annu. Rev. Biochem.* **71**, 101–132 (2002).
4. Brandt, V. L. & Roth, D. B. A recombinase diversified: new functions of the RAG proteins. *Curr. Opin. Immunol.* **14**, 224–229 (2002).
5. Honjo, T., Kinoshita, K. & Muramatsu, M. Molecular mechanism of class switch recombination: linkage with somatic hypermutation. *Annu. Rev. Immunol.* **20**, 165–196 (2002).
6. Fischer, A. Primary immunodeficiency diseases: an experimental model for molecular medicine. *Lancet* **357**, 1863–1869 (2001).
7. Haber, J. E. Partners and pathways repairing a double-strand break. *Trends Genet.* **16**, 259–264 (2000).
8. Mombaerts, P. *et al.* RAG-1 deficient mice have no mature B and T lymphocytes. *Cell* **68**, 869–877 (1992).
9. Shinkai, Y. *et al.* RAG-2 deficient mice lack mature lymphocytes owing to inability to initiate V(D)J rearrangement. *Cell* **68**, 855–867 (1992).
10. Schwarz, K. *et al.* RAG mutations in human B cell-negative SCID. *Science* **274**, 97–99 (1996).
11. de Villartay, J. P. V(D)J recombination and DNA repair: lessons from human immune deficiencies and other animal models. *Curr. Opin. Allergy Clin. Immunol.* **2**, 473–479 (2002).
12. Callebaut, I. & Mornon, J. P. The V(D)J recombination activating protein RAG2 consists of a six-bladed propeller and a PHD fingerlike domain, as revealed by sequence analysis. *Cell. Mol. Life Sci.* **54**, 880–891 (1998).
13. Corneo, B. *et al.* 3D clustering of human RAG2 gene mutations in severe combined immune deficiency (SCID). *J. Biol. Chem.* **275**, 12672–12675 (2000).
14. Fugmann, S. D., Lee, A. I., Shockett, P. E., Villey, I. J. & Schatz, D. G. The RAG proteins and V(D)J recombination: complexes, ends, and transposition. *Annu. Rev. Immunol.* **18**, 495–527 (2000).
15. Li, Z., Dordai, D. I., Lee, J. & Desiderio, S. A conserved degradation signal regulates RAG-2 accumulation during cell division and links V(D)J recombination to the cell cycle. *Immunity* **5**, 575–589 (1996).
16. Villa, A. *et al.* V(D)J recombination defects in lymphocytes due to RAG mutations: severe immunodeficiency with a spectrum of clinical presentations. *Blood* **97**, 81–88 (2001).
17. Liang, H. E. *et al.* The ‘dispensable’ portion of RAG2 is necessary for efficient V-to-DJ rearrangement during B and T cell development. *Immunity* **17**, 639–651 (2002).

18. Akamatsu, Y. *et al.* Deletion of the RAG2 C terminus leads to impaired lymphoid development in mice. *Proc. Natl Acad. Sci. USA* **100**, 1209–1214 (2003).
References 17 and 18 describe the development of knock-in mice that express only the core of recombinase-activating gene 2 (Rag2), showing the absolute requirement of full length Rag2 for the proper rearrangement of endogenous immunoglobulin and T-cell receptor (TCR) loci *in vivo*.
19. Roman, C. A., Cherry, S. R. & Baltimore, D. Complementation of V(D)J recombination deficiency in RAG-1<sup>-/-</sup> B cells reveals a requirement for novel elements in the N-terminus of RAG-1. *Immunity* **7**, 13–24 (1997).
20. Kirch, S. A., Rathbun, G. A. & Oettinger, M. A. Dual role of RAG2 in V(D)J recombination: catalysis and regulation of ordered Ig gene assembly. *EMBO J.* **17**, 4881–4886 (1998).
21. Corneo, B., Benmerah, A. & de Villartay, J. P. A short peptide at the C-terminus is responsible for the nuclear localization of RAG2. *Eur. J. Immunol.* **32**, 2068–2073 (2002).
22. Ross, A. E., Vuica, M. & Desiderio, S. Overlapping signals for protein degradation and nuclear localization define a role for intrinsic RAG-2 nuclear uptake in dividing cells. *Mol. Cell Biol.* **23**, 5308–5319 (2003).
23. Aasland, R., Gibson, T. J. & Stewart, A. F. The PHD finger: implications for chromatin-mediated transcriptional regulation. *Trends Biochem. Sci.* **20**, 56–59 (1995).
24. Villa, A., Sobacchi, C. & Vezzoni, P. Recombination activating gene and its defects. *Curr. Opin. Allergy Clin. Immunol.* **1**, 491–495 (2001).
25. Corneo, B. *et al.* Identical mutations in RAG1 and RAG2 genes leading to defective V(D)J recombinase activity can cause either T-B-severe combined immune deficiency or Omenn syndrome. *Blood* **97**, 2772–2776 (2001).
26. Cavazzana-Calvo, M. *et al.* Increased radiosensitivity of granulocyte macrophage colony-forming units and skin fibroblasts in human autosomal recessive severe combined immunodeficiency. *J. Clin. Invest.* **91**, 1214–1218 (1993).
27. Bosma, M. J. & Carroll, A. M. The SCID mouse mutant: definition, characterization, and potential uses. *Annu. Rev. Immunol.* **9**, 323–350 (1991).
28. Nicolas, N. *et al.* A human SCID condition with increased sensitivity to ionizing radiations and impaired V(D)J rearrangements defines a new DNA recombination/repair deficiency. *J. Exp. Med.* **188**, 627–634 (1998).
29. Nicolas, N. *et al.* Lack of detectable defect in DNA double-strand break repair and DNA-dependent protein kinase activity in radiosensitive human severe combined immunodeficiency fibroblasts. *Eur. J. Immunol.* **26**, 1118–1122 (1996).

30. Li, L. *et al.* The gene for severe combined immunodeficiency disease in Athabascan-speaking native americans is located on chromosome 10p. *Am. J. Hum. Genet.* **62**, 136–144 (1998).
31. Moshous, D. *et al.* A new gene involved in DNA double-strand break repair and V(D)J recombination is located on human chromosome 10p. *Hum. Mol. Genet.* **9**, 583–588 (2000).
32. Moshous, D. *et al.* Artemis, a novel DNA double-strand break repair/V(D)J recombination protein, is mutated in human severe combined immune deficiency. *Cell* **105**, 177–186 (2001). This reference describes the cloning of the human Artemis gene from the analysis of patients with radiosensitive-severe combined immunodeficiency (RS-SCID).
33. Li, L. *et al.* A founder mutation in Artemis, an SNM1-like protein, causes SCID in Athabascan-speaking native Americans. *J. Immunol.* **168**, 6323–6329 (2002).
34. Noordzij, J. G. *et al.* Radiosensitive SCID patients with Artemis gene mutations show a complete B-cell differentiation arrest at the pre-B-cell receptor checkpoint in bone marrow. *Blood* **101**, 1446–1452 (2003).
35. Kobayashi, N. *et al.* Novel Artemis gene mutations of radiosensitive severe combined immunodeficiency in Japanese families. *Hum. Genet.* **112**, 348–352 (2003).
36. Rooney, S. *et al.* Leaky SCID phenotype associated with defective V(D)J coding end processing in Artemis-deficient mice. *Mol. Cell* **10**, 1379–1390 (2002). This paper provides a description of Artemis-knockout mice.
37. Aravind, L. An evolutionary classification of the metallo-β-lactamase fold. *In Silico Biology* **1**, 69–91 (1999).
38. Callebaut, I., Moshous, D., Mornon, J. P. & De Villartay, J. P. Metallo-β-lactamase fold within nucleic acids processing enzymes: the β-CASP family. *Nucl. Acids Res.* **30**, 3592–3601 (2002).
39. Roth, D. B., Menetski, J. P., Nakajima, P. B., Bosma, M. J. & Gellert, M. V(D)J recombination: broken DNA molecules with covalently sealed (hairpin) coding ends in SCID mouse thymocytes. *Cell* **70**, 983–991 (1992).
40. Ma, Y., Pannicke, U., Schwarz, K. & Lieber, M. R. Hairpin opening and overhang processing by an Artemis/DNA-dependent protein kinase complex in nonhomologous end joining and V(D)J recombination. *Cell* **108**, 781–794 (2002). The first evidence that Artemis, when complexed to the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs), is capable of opening RAG1/RAG2-generated hairpin structures.
41. Hiom, K. & Gellert, M. Assembly of a 12/23 paired signal complex: a critical control point in V(D)J recombination. *Mol. Cell* **1**, 1000–1019 (1998).
42. Qiu, J. X., Kale, S. B., Yarnell Schultz, H. & Roth, D. B. Separation-of-function mutants reveal critical roles for RAG2 in both the cleavage and joining steps of V(D)J recombination. *Mol. Cell* **7**, 77–87 (2001).

43. Yarnell Schultz, H., Landree, M. A., Qiu, J. X., Kale, S. B. & Roth, D. B. Joining-deficient RAG1 mutants block V(D)J recombination *in vivo* and hairpin opening *in vitro*. *Mol. Cell* **7**, 65–75 (2001).

44. Shockett, P. E. & Schatz, D. G. DNA hairpin opening mediated by the RAG1 and RAG2 proteins. *Mol. Cell Biol.* **19**, 4159–4166 (1999).

45. Besmer, E. *et al.* Hairpin coding end opening is mediated by RAG1 and RAG2 proteins. *Mol. Cell* **2**, 817–828 (1998).

46. Paull, T. T. & Gellert, M. Nbs1 potentiates ATP-driven DNA unwinding and endonuclease cleavage by the Mre11/Rad50 complex. *Genes Dev.* **13**, 1276–1288 (1999).

47. Chen, H. T. *et al.* Response to RAG-mediated V(D)J cleavage by NBS1 and γ-H2AX. *Science* **290**, 1962–1965 (2000).

48. Harfst, E., Cooper, S., Neubauer, S., Distel, L. & Grawunder, U. Normal V(D)J recombination in cells from patients with Nijmegen breakage syndrome. *Mol. Immunol.* **37**, 915–929 (2000).

49. Barnes, D. E., Stamp, G., Rosewell, I., Denzel, A. & Lindahl, T. Targeted disruption of the gene encoding DNA ligase IV leads to lethality in embryonic mice. *Curr. Biol.* **8**, 1395–1398 (1998).

50. Gao, Y. *et al.* A critical role for DNA end-joining proteins in both lymphogenesis and neurogenesis. *Cell* **95**, 891–902 (1998).

51. Frank, K. M. *et al.* Late embryonic lethality and impaired V(D)J recombination in mice lacking DNA ligase IV. *Nature* **396**, 173–177 (1998).

52. O’Driscoll, M. *et al.* DNA ligase IV mutations identified in patients exhibiting developmental delay and immunodeficiency. *Mol. Cell* **8**, 1175–1185 (2001). A description of DNA ligase IV deficiency in humans.

53. Riballo, E. *et al.* Cellular and biochemical impact of a mutation in DNA ligase IV conferring clinical radiosensitivity. *J. Biol. Chem.* **276**, 31124–31132 (2001).

54. Dai, Y. *et al.* Nonhomologous end joining and V(D)J recombination require an additional factor. *Proc. Natl Acad. Sci. USA* **100**, 2462–2467 (2003). This report indicates the existence of additional unknown factors in V(D)J recombination/DNA repair.

55. Iwasato, T., Shimizu, A., Honjo, T. & Yamagishi, H. Circular DNA is excised by immunoglobulin class switch recombination. *Cell* **62**, 143–9 (1990).

56. Matsuoka, M., Yoshida, K., Maeda, T., Usuda, S. & Sakano, H. Switch circular DNA formed in cytokine-treated mouse splenocytes: evidence for intramolecular DNA deletion in immunoglobulin class switching. *Cell* **62**, 135–142 (1990).

57. Kinoshita, K. & Honjo, T. Unique and unprecedented recombination mechanisms in class switching. *Curr. Opin. Immunol.* **12**, 195–198 (2000).

58. Manis, J. P., Dudley, D., Kaylor, L. & Alt, F. W. IgH class switch recombination to IgG1 in DNA-PKcs-deficient B cells. *Immunity* **16**, 607–617 (2002).

59. Rajewsky, K. Clonal selection and learning in the antibody system. *Nature* **381**, 751–758 (1996).

60. Frazer, J. K. *et al.* Identification and cloning of genes expressed by human tonsillar B lymphocyte subsets. *Ann. NY Acad. Sci.* **815**, 316–318 (1997).

61. Storb, U. *et al.* Somatic hypermutation of immunoglobulin genes is linked to transcription. *Curr. Top. Microbiol. Immunol.* **229**, 11–19 (1998).

62. Jacobs, H. & Bross, L. Towards an understanding of somatic hypermutation. *Curr. Opin. Immunol.* **13**, 208–218 (2001).

63. Kaartinen, M., Griffiths, G. M., Markham, A. F. & Milstein, C. mRNA sequences define an unusually restricted IgG response to 2-phenyloxazolone and its early diversification. *Nature* **304**, 320–324 (1983).

64. Liu, Y. J. *et al.* Within germinal centers, isotype switching of immunoglobulin genes occurs after the onset of somatic mutation. *Immunity* **4**, 241–250 (1996).

65. Bachl, J., Carlson, C., Gray-Schopfer, V., Dessing, M. & Olsson, C. Increased transcription levels induce higher mutation rates in a hypermutating cell line. *J. Immunol.* **166**, 5051–5057 (2001).

66. Betz, A. G. *et al.* Elements regulating somatic hypermutation of an immunoglobulin κ gene: critical role for the intron enhancer/matrix attachment region. *Cell* **77**, 239–248 (1994).

67. Fukita, Y., Jacobs, H. & Rajewsky, K. Somatic hypermutation in the heavy chain locus correlates with transcription. *Immunity* **9**, 105–114 (1998).

68. Manis, J. P. *et al.* Class switching in B cells lacking 3′ immunoglobulin heavy chain enhancers. *J. Exp. Med.* **188**, 1421–1431 (1998).

69. Bosma, G. C. *et al.* DNA-dependent protein kinase activity is not required for immunoglobulin class switching. *J. Exp. Med.* **196**, 1483–1495 (2002).

70. Bemark, M. *et al.* Somatic hypermutation in the absence of DNA-dependent protein kinase catalytic subunit (DNA-PKcs) or recombination-activating gene (RAG)1 activity. *J. Exp. Med.* **192**, 1509–1514 (2000).

71. Kenter, A. L. The liaison of isotype class switch and mismatch repair: an illegitimate affair. *J. Exp. Med.* **190**, 307–310 (1999).

72. Casalho, M., Wong, J., Steinberg, C. & Wabl, M. Mismatch repair co-opted by hypermutation. *Science* **279**, 1207–1210 (1998).

73. Schrader, C. E., Edelmann, W., Kucherlapati, R. & Stavnezer, J. Reduced isotype switching in splenic B cells from mice deficient in mismatch repair enzymes. *J. Exp. Med.* **190**, 323–330 (1999).

74. Evans, E. & Alani, E. Roles for mismatch repair factors in regulating genetic recombination. *Mol. Cell Biol.* **20**, 7839–7844 (2000).

75. Dominguez, O. *et al.* DNA polymerase μ (Polμ), homologous to TdT, could act as a DNA mutator in eukaryotic cells. *EMBO J.* **19**, 1731–1742 (2000).

76. Zan, H. *et al.* The translesion DNA polymerase ζ plays a major role in Ig and BCL-6 somatic hypermutation. *Immunity* **14**, 643–653 (2001).

77. Zeng, X. *et al.* DNA polymerase η is an A-T mutator in somatic hypermutation of immunoglobulin variable genes. *Nature Immunol.* **2**, 537–541 (2001).

78. Faili, A. *et al.* Induction of somatic hypermutation in immunoglobulin genes is dependent on DNA polymerase ι. *Nature* **419**, 944–947 (2002).

79. Korthauer, U. *et al.* Defective expression of T-cell CD40 ligand causes X-linked immunodeficiency with hyper-IgM. *Nature* **361**, 539–541 (1993).

80. DiSanto, J. P., Bonnefoy, J. Y., Gauchat, J. F., Fischer, A. & de Saint Basile, G. CD40 ligand mutations in X-linked immunodeficiency with hyper-IgM. *Nature* **361**, 541–543 (1993).

81. Aruffo, A. *et al.* The CD40 ligand, gp39, is defective in activated T cells from patients with X-linked hyper-IgM syndrome. *Cell* **72**, 291–300 (1993).

82. Allen, R. C. *et al.* CD40 ligand gene defects responsible for X-linked hyper-IgM syndrome. *Science* **259**, 990–993 (1993).

83. Ferrari, S. *et al.* Mutations of CD40 gene cause an autosomal recessive form of immunodeficiency with hyper IgM. *Proc. Natl Acad. Sci. USA* **98**, 12614–12619 (2001).

84. Revy, P. *et al.* Activation-induced cytidine deaminase (AID) deficiency causes the autosomal recessive form of the hyper-IgM syndrome (HIGM2). *Cell* **102**, 565–575 (2000). This paper and reference 91 show, for the first time, the role of activation-induced cytidine deaminase (AID) in class-switch recombination (CSR) and somatic hypermutation (SHM) in humans and mice, linking these two processes of antibody maturation.

85. Notarangelo, L. D., Duse, M. & Ugazio, A. G. Immunodeficiency with hyper-IgM (HIM). *Immunodefic. Rev.* **3**, 101–121 (1992).

86. Zonana, J. *et al.* A novel X-linked disorder of immune deficiency and hypohidrotic ectodermal dysplasia is allelic to incontinentia pigmenti and due to mutations in IKK-γ (NEMO). *Am. J. Hum. Genet.* **67**, 1555–1562 (2000).

87. Doffinger, R. *et al.* X-linked anhidrotic ectodermal dysplasia with immunodeficiency is caused by impaired NF-κB signaling. *Nature Genet.* **27**, 277–285 (2001).

88. Jain, A. *et al.* Specific missense mutations in NEMO result in hyper-IgM syndrome with hypohidrotic ectodermal dysplasia. *Nature Immunol.* **2**, 223–228 (2001).

89. Durandy, A. *et al.* Abnormal CD40-mediated activation pathway in B lymphocytes from patients with hyper-IgM syndrome and normal CD40 ligand expression. *J. Immunol.* **158**, 2576–2584 (1997).

90. Minegishi, Y. *et al.* Mutations in activation-induced cytidine deaminase in patients with hyper IgM syndrome. *Clin. Immunol.* **97**, 203–210 (2000).

91. Muramatsu, M. *et al.* Class switch recombination and hypermutation require activation-induced cytidine deaminase (AID), a potential RNA editing enzyme. *Cell* **102**, 553–563 (2000).

92. Muramatsu, M. *et al.* Specific expression of activation-induced cytidine deaminase (AID), a novel member of the RNA-editing deaminase family in germinal center B cells. *J. Biol. Chem.* **274**, 18470–18476 (1999).

93. Faili, A. *et al.* AID-dependent somatic hypermutation occurs as a DNA single-strand event in the BL2 cell line. *Nature Immunol.* **3**, 815–821 (2002).

94. Nagaoka, H., Muramatsu, M., Yamamura, N., Kinoshita, K. & Honjo, T. Activation-induced deaminase (AID)-directed hypermutation in the immunoglobulin Sµ region:

95. Yoshikawa, K. *et al.* AID enzyme-induced hypermutation in an actively transcribed gene in fibroblasts. *Science* **296**, 2033–2036 (2002).

96. Okazaki, I. M., Kinoshita, K., Muramatsu, M., Yoshikawa, K. & Honjo, T. The AID enzyme induces class switch recombination in fibroblasts. *Nature* **416**, 340–345 (2002).

97. Arakawa, H., Hauschild, J. & Buerstedde, J. M. Requirement of the activation-induced deaminase (AID) gene for immunoglobulin gene conversion. *Science* **295**, 1301–1306 (2002).

98. Hein, K. *et al.* Processing of switch transcripts is required for targeting of antibody class switch recombination. *J. Exp. Med.* **188**, 2369–2374 (1998).

99. Wiesendanger, M., Scharff, M. D. & Edelmann, W. Somatic hypermutation, transcription, and DNA mismatch repair. *Cell* **94**, 415–418 (1998).

100. Kong, Q. & Maizels, N. DNA breaks in hypermutating immunoglobulin genes: evidence for a break-and-repair pathway of somatic hypermutation. *Genetics* **158**, 369–378 (2001).

101. Sale, J. E. & Neuberger, M. S. TdT-accessible breaks are scattered over the immunoglobulin V domain in a constitutively hypermutating B cell line. *Immunity* **9**, 859–869 (1998).

102. Wuerffel, R. A., Du, J., Thompson, R. J. & Kenter, A. L. IgSγ3 DNA-specific double strand breaks are induced in mitogen-activated B cells and are implicated in switch recombination. *J. Immunol.* **159**, 4139–4144 (1997).

103. Bross, L. *et al.* DNA double-strand breaks in immunoglobulin genes undergoing somatic hypermutation. *Immunity* **13**, 589–597 (2000).

104. Papavasiliou, F. N. & Schatz, D. G. Cell-cycle-regulated DNA double-stranded breaks in somatic hypermutation of immunoglobulin genes. *Nature* **408**, 216–221 (2000).

105. Petersen, S. *et al.* AID is required to initiate Nbs1/γ-H2AX focus formation and mutations at sites of class switching. *Nature* **414**, 660–665 (2001).

106. Rolink, A., Me heavy chain class switching. *Immunity* **5**, 319–330 (1996).

107. Casellas, R. *et al.* Ku80 is required for immunoglobulin isotype switching. *EMBO J.* **17**, 2404–2411 (1998).

108. Chen, X., Kinoshita, K. & Honjo, T. Variable deletion and duplication at recombination junction ends: implication for staggered double-strand cleavage in class-switch recombination. *Proc. Natl Acad. Sci. USA* **98**, 13860–13865 (2001).

109. Doi, T., Kinoshita, K., Ikegawa, M., Muramatsu, M. & Honjo, T. De novo protein synthesis is required for the activation-induced cytidine deaminase function in class-switch recombination. *Proc. Natl Acad. Sci. USA* **100**, 2634–2638 (2003).

110. Petersen-Mahrt, S. K., Harris, R. S. & Neuberger, M. S. AID mutates *E. coli* suggesting a DNA deamination mechanism for antibody diversification. *Nature* **418**, 99–104 (2002).

111. Bransteitter, R., Pham, P., Scharff, M. D. & Goodman, M. F. Activation-induced cytidine deaminase deaminates deoxycytidine on single-stranded DNA but requires the action of RNase. *Proc. Natl Acad. Sci. USA* **100**, 4102–4107 (2003). This paper shows the action of AID on single-stranded DNA.

112. Chaudhuri, J. *et al.* Transcription-targeted DNA deamination by the AID antibody diversification enzyme. *Nature* **422**, 726–730 (2003).

113. Ramiro, A. R., Stavropoulos, P., Jankovic, M. & Nussenzweig, M. C. Transcription enhances AID-mediated cytidine deamination by exposing single-stranded DNA on the nontemplate strand. *Nature Immunol.* **4**, 452–456 (2003).

114. Shinkura, R. *et al.* The influence of transcriptional orientation on endogenous switch region function. *Nature Immunol.* **4**, 435–441 (2003). References 112–114 describe the activity of AID on the non-transcribed single-stranded DNA.

115. Harris, R. S., Petersen-Mahrt, S. K. & Neuberger, M. S. RNA editing enzyme APOBEC1 and some of its homologs can act as DNA mutators. *Mol. Cell* **10**, 1247–1253 (2002).

116. Yu, K., Chedin, F., Hsieh, C. L., Wilson, T. E. & Lieber, M. R. R-loops at immunoglobulin class switch regions in the chromosomes of stimulated B cells. *Nature Immunol.* **4**, 442–451 (2003). This paper indicates that the formation of RNA-DNA hybrids during transcription leads to R-loops and that cytosine residues on single-stranded DNA become targets for AID.

117. Fugmann, S. D. & Schatz, D. G. RNA AIDS DNA. *Nature Immunol.* **4**, 429–430 (2003).

REVIEW S

118. Rada, C. *et al.* Immunoglobulin isotype switching is inhibited and somatic hypermutation perturbed in UNG-deficient mice. *Curr. Biol.* 12, 1748–1755 (2002). This description of the immunological phenotype of uracil N-glycosylase (Ung)-deficient mice supports the hypothesis of AID as a DNA-editing enzyme.

119. Imai, K. *et al.* Hyper-IgM syndrome type 4 with a B-lymphocyte intrinsic selective deficiency in immunoglobulin class switch recombination. *J. Clin. Immunol.* 112, 136–142 (2003).

120. Kinzler, K. W. & Vogelstein, B. Cancer-susceptibility genes. Gatekeepers and caretakers. *Nature* 386, 761–763 (1997).

121. Vanasse, G. J., Concannon, P. & Willerford, D. M. Regulated genomic instability and neoplasia in the lymphoid lineage. *Blood* 94, 3997–4010 (1999).

122. Ferguson, D. O. & Alt, F. W. DNA double strand break repair and chromosomal translocation: lessons from animal models. *Oncogene* 20, 5572–5579 (2001). A review of the mouse models that show the ‘genome caretaker’ role of non-homologous end-joining (NHEJ) factors.

123. Difilippantonio, M. J. *et al.* Evidence for replicative repair of DNA double-strand breaks leading to oncogenic translocation and gene amplification. *J. Exp. Med.* 196, 469–480 (2002).

124. Zhu, C. *et al.* Unrepaired DNA breaks in p53-deficient cells lead to oncogenic gene amplification subsequent to translocation. *Cell* 109, 811–821 (2002).

125. Rooney, S. *et al.* Defective DNA repair and increased genomic instability in Artemis-deficient murine cells. *J. Exp. Med.* 197, 553–565 (2003).

126. Moshous, D. *et al.* Partial T and B lymphocyte immunodeficiency and predisposition to lymphoma in patients with hypomorphic mutations in Artemis. *J. Clin. Invest.* 111, 381–387 (2003). The authors show that partial Artemis deficiency in humans can cause B-cell lymphoma, indicating a genome caretaker role for Artemis.

127. Nelms, K. A. & Goodnow, C. C. Genome-wide ENU mutagenesis to reveal immune regulators. *Immunity* 15, 409–418 (2001).

128. Noordzij, J. G. *et al.* The immunophenotypic and immunogenotypic B-cell differentiation arrest in bone marrow of RAG-deficient SCID patients corresponds to residual recombination activities of mutated RAG proteins. *Blood* 100, 2145–2152 (2002).

129. Imai, K. *et al.* Human uracil-DNA glycosylase deficiency associated with profoundly impaired immunoglobulin class-switch recombination. *Nature Immunol.* 4, 1023–1028 (2003).


Acknowledgements  
We thank F. Alt for his permission to refer to unpublished data. This work was supported by institutional grants from Institut National de la Santé et de la Recherche Médicale (INSERM) and from Ministère de la Recherche et de la Technologie as well as grants from Commissariat à l’Energie Atomique, Association de Recherche sur le Cancer and the Louis J eantet Fundation.


Competing interests statement  
The authors declare that they have no competing financial interests.


Online links  

DATABASES  
The following terms in this article are linked online to:  
LocusLink: http://www.ncbi.nlm.nih.gov/LocusLink/  
AID | APOBEC1 | Artemis | CD40 | CD40L | HIGM4 | Ku70 | Ku80 | MRE11 | NBS1 | RAD50 | RAG1 | RAG2  
OMIM: http://www.ncbi.nlm.nih.gov/Omim/  
NBS  

Access to this interactive links box is free online.
