
Metabolism and Bioenergetics:

CYP2U1, a Novel Human Thymus- and Brain-specific Cytochrome P450, Catalyzes ω- and (ω-1)-Hydroxylation of Fatty Acids

Samuel S. Chuang, Christian Helvig, Mohammed Taimi, Heather A. Ramshaw, Andrew H. Collop, Ma'an Amad, Jay A. White, Martin Petkovich, Glenville Jones and Bozena Korczak

J. Biol. Chem. 2004, 279:6305-6314.

doi: 10.1074/jbc.M311830200 originally published online December 3, 2003

---

Access the most updated version of this article at doi: [10.1074/jbc.M311830200](http://dx.doi.org/10.1074/jbc.M311830200)

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 41 references, 16 of which can be accessed free at [http://www.jbc.org/content/279/8/6305.full.html#ref-list-1](http://www.jbc.org/content/279/8/6305.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2004 by The American Society for Biochemistry and Molecular Biology, Inc.

CYP2U1, a Novel Human Thymus- and Brain-specific Cytochrome P450, Catalyzes ω- and (ω-1)-Hydroxylation of Fatty Acids*

Received for publication, October 28, 2003, and in revised form, December 2, 2003
Published, JBC Papers in Press, December 3, 2003, DOI 10.1074/jbc.M311830200

Samuel S. Chuang‡§, Christian Helvig‡§, Mohammed Taimi‡, Heather A. Ramshaw‡,
Andrew H. Collop‡, Ma’an Amad‡, Jay A. White‡, Martin Petkovich¶, Glenville Jones¶,
and Bozena Korczak***

From ‡Cytochroma, Inc., Markham, Ontario L3R 8E4 and the ¶Department of Biochemistry, Queen’s University,
Ontario K7L 3N6, Canada

Long chain fatty acids have recently emerged as critical signaling molecules in neuronal, cardiovascular, and renal processes, yet little is presently known about the precise mechanisms controlling their tissue distribution and bioactivation. We have identified a novel cytochrome P450, CYP2U1, which may play an important role in modulating the arachidonic acid signaling pathway. Northern blot and real-time PCR analysis demonstrated that CYP2U1 transcripts were most abundant in the thymus and the brain (cerebellum), indicating a specific physiological role for CYP2U1 in these tissues. Recombinant human CYP2U1 protein, expressed in baculovirus-infected Sf9 insect cells, was found to metabolize arachidonic acid exclusively to two region-specific products as determined by liquid chromatography-mass spectrometry. These metabolites were identified as 19- and 20-hydroxy-modified arachidonic acids by liquid chromatography-tandem mass spectrometry analysis. In addition to ω/ω-1 hydroxylation of arachidonic acid, CYP2U1 protein also catalyzed the hydroxylation of structurally related long chain fatty acid (docosahexaenoic acid) but not fatty acids such as lauric acid or linoleic acid. This is the first report of the cloning and functional expression of a new human member of P450 family 2, CYP2U1, which metabolizes long chain fatty acids. Based on the ability of CYP2U1 to generate bioactive eicosanoid derivatives, we postulate that CYP2U1 plays an important physiological role in fatty acid signaling processes in both cerebellum and thymus.

strates into locally active hormones, vitamins, and signaling molecules including derivatives of arachidonic acid (AA) known as eicosanoids (1). This diverse group of substrates is composed of prostaglandins, thromboxanes, lipoxins, leukotrienes, as well as epoxy- and hydroxy-modified arachidonic acids (EETs and HETEs, respectively) and is involved in regulation of local blood flow (2), activity of smooth muscle cells (3, 4), secretion of cytokines (5), cell proliferation (6), and cell migration (7) and aggregation (8). The activities of AA derivatives are implicated in a number of physiological processes including inflammation, anaphylaxis, and hypertension (9–11). The identification and characterization of the enzymatic processes involved in generating and metabolizing these important signaling molecules are critical for understanding the role of these molecules in health and disease.

Arachidonic acid is an abundant component of cell membrane phospholipids and is released by phospholipase A₂ in response to extracellular signals (12). All mammalian cells except erythrocytes convert AA into bioactive eicosanoids (13) using some or all of the following three enzymatic pathways: 1) the cyclooxygenase pathway that leads to the synthesis of numerous prostaglandins, prostacyclins, and thromboxanes (14, 15); 2) the lipoxygenase pathway that results in accumulation of hydroperoxides (HPETEs) and leukotrienes (14, 15); 3) the AA monooxygenase pathway that creates EETs and HETEs and consists of P450s possessing epoxygenase, lipoxygenase-like, or ω/ω-1 hydroxylase activity (16). The first two enzymatic cascades involve multiple P450s responsible for the synthesis of secondary eicosanoids. Cytochrome P450 epoxygenases synthesize four regio-epoxy isomers 5, 6-EET, 8, 9-EET, 11,12-EET and 14,15-EET (16), which can be further converted by epoxide hydrolases to corresponding dihydroxyeicosatrienoic acids (17). Lipoxygenase-like P450s create 5-HETE, 8-HETE, 9-HETE, 11-HETE, 12-HETE, or 15-HETE (16). ω/ω-1-Hydroxylase converts AA into 20-HETE, 19-HETE, 18-HETE, 17-HETE, or 16-HETE (18). All of the above products can be modified further into additional eicosanoids.

Arachidonic acid monooxygenase activity was first characterized in microsomal fractions obtained from kidney and liver (18). More recently, numerous tissue-specific AA-metabolizing P450s have been identified in human and animal tissues. For example, human hepatic CYP1A2 (19, 20) creates 14,15-EET, 11,12-EET, 8,9-EET, 7-HETE, 10-HETE, 13-HETE, and 19-HETE, whereas hepatic 2C19 (20, 21) synthesizes 8,9-EET, 14,15-EET, 19-HETE, and 20-HETE. Other P450s convert AA into a more discrete set of products. Human CYP2J2 activity results in the accumulation of all four EETs (22), rat CYP4A1 creates predominantly 20-HETE (23) and murine CYP2J9,

Members of the cytochrome P450 (P450)¹ family of enzymes play key roles in tissue-specific conversion of natural sub-

* This work was supported in part by Cytochroma, Inc. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) AY343323.
§ Both authors contributed equally to this paper.
|| Supported by the National Cancer Institute of Canada, the Canadian Institute of Health Research, and Cytochroma, Inc.
** To whom correspondence should be addressed: Cytochroma, Inc., 330 Cochrane Dr., Markham, Ontario L3R 8E4, Canada. Tel.: 905-479-5306; Fax: 905-479-1287; E-mail: bozena@cytochroma.com.
¹ The abbreviations used are: P450, cytochrome P450; AA, arachidonic acid; DHA, docosahexaenoic acid; DTA, docosatraenoic acid; EET, epoxy-modified arachidonic acid; EPA, eicosapentaenoic acid; EST, expressed sequence tag; ETA, eicosatrienoic acid; h*b*, human cytochrome b₅; HETE, hydroxy-modified arachidonic acid; hOR, human cytochrome P450-NADPH reductase; HPETE, hydroperoxide; HPLC, high performance liquid chromatography; LC, liquid chromatography; MS, mass spectrometry; 17-ODYA, 17-octodecynoic acid.

6306

**Identification and Characterization of CYP2U1**

ctcaagcaagggaacccgaggccgccggcccggacc ATG TCG TCT CCG GGG CCG TCG CAG CCG GCC GAG GAC CCG  
M S S P S Q P P A E D P  
CCC TGG CCC GCG CGC CTC CTG CGT GCG CCT CTG GGG CTG CTG CGG CTG GGC CCC AGC GGG GGC CTG CTG  
P W P A R L L R A P L G L L R D P S G G A L L  
CTA TGC GGC CTC GTA GCG CTG GTG GGC TGG AGC TGG CTG CGG AGG CGC CGG GGC GGC ATC CCG CCC GGG  
L C G L V A L L G W S W L R R R A R G I P P G  
CCC ACG CCC TGG CCT CTG GTG GGC AAC TTC GGT CAC GTG CTG CTG CCT CCC TTC CTC CGG CGG AGC TGG  
P T P W P L V G N F G H V L L D P S R R R A R G I P P G  
CTG AGC AGC AGG ACC AGG GCC GCA GGG ATT GAT CCC TCG GTC ATA GGC CCG CAG GTG CTC GCT CAC CTA  
L S S R T R A A G I D P S V I G P Q V L L A H L  
GCC CGC GTG TAC GGC AGC ATC TTC AGC TTT ATC GGC CAC TAC CTG GTG GTG GTC CTC AGC GAC TTC CAC  
A R V Y G S I F S F I G H Y L V V L S D F H  
AGC GTG CGC GAG GCG CTG GTG CAG CAG GCC GAG GTC TTC AGC GAC CGC CCG CGG GTG CCG CTC ATC TCC ATC  
S V R E A L V Q Q A E V F S D R P V L I S I  
GTG ACC AAG GAG AAG GGG GTT GTG TTT GCA CAT TAT GGT CCC GTC TGG AGA CAA CAA AGG AAG TTC TCT CAT  
V T K E K G V V F A H Y G P V W R Q Q R K F S H  
TCA ACT CTT CGT CAT TTT GGG TTG GGA AAA CTT AGC TTG GAG CCC AAG ATT ATT GAG GAG TTC AAA TAT GTG  
S T L R H F G L G K L S L E P K I I E E F K Y V  
AAA GCA GAA ATG CAA AAG CAC GGA GAA GAC CCC TTC TGT CCT TTC TCC ATC ATC AGC AAT GCC GTC TCT AAC  
K A E M Q K H G E D P F C P F S I I S N A V S N  
ATC ATT TGC TCC TTG TGC TTT GGC CAG CGC TTT GAT TAC ACT AAT AGT GAG TTC AAG AAA ATG CTT GGT TTT  
I I C S L C F G Q R F D Y T N S E F K K M L G F  
ATG TCA CGA GGC CTA GAA ATC TGT CTG AAC AGT CAA GTC CTC CTG GTC AAC ATA TGC CCT TGG CTT TAT TAC  
M S R G L E I C A T G C A C A T G C C T T G G A T  
CTT CCC TTT GGA CCA TTT AAG GAA TTA AGA CAA ATT GAA AAG GAT ATA ACC AGT TTC CTT AAA AAA ATC ATC  
L P F G P F K E L R Q I E K D I T S F K K I I  
AAA GAC CAT CAA GAG TCT CTG GAT AGA GAG AAC CCT CAG GAC TTC ATA GAC ATG TAC CTT CTC CAC ATG GAA  
K D H Q E S L D R E N P Q D F I D M Y L L H M E  
GAG GAG AGG AAA AAT AAT AGT AAC AGC AGT TTT GAT GAA GAG TAC TTA TTT TAT ATC ATT GGG GAT CTC TTT  
E E R K N N S N S F D E Y L F Y I I G D L F  
ATT GCT GGG ACT GAT ACC ACA ACT AAC TCT TTG CTC TGG TGC CTG CTG TAT ATG TCG CTG AAC CCC GAT GTA  
I A G T D T T N S L L W C L L Y M S L N P D V  
CAA GAA AAG GTT CAT GAA GAA ATT GAA AGA GTC ATT GGC GCC AAC CGA GCT CCT TCC CTC ACA GAC AAG GCC  
Q E K V H E E R V I G A N R A P S L T D K A  
CAG ATG CCC TAC ACA GGC ACC ATC ATG GAA GTG CAG AGG CTA ACT GTG GTG GTG CCG CTT GCC ATT CCT  
Q M P Y T E A G C A G G A C T A C T G T G T G C C G T T T G C C A T T C C T  
CAT ATG ACC TCA GAG AAC ACA GTG CTC CTC CAA GGG TAT ACC ATT CCT AAA GGC ACA TTG ATC TTA CCC AAC CTG  
H M T S E N T V L Q G Y T I P K G T L I L P N L  
TGG TCA GTA CAT AGA GAC CCA GCC ATT TGG GAG AAA CCg GAG GAT TTC TAC CCT AAT CGA TTT CTG GAT GAC  
W S V H R D P A I W E K P E D F Y P N R F L D D  
CAA GGA CAA CTA ATT AAA AAA GAA ACC TTT ATT CCT TTT GGG ATA GGG AAG CGG GTG TGT ATG GGA GAA CAA  
Q G Q L I K K E T F I P F G I G K R V C M G E Q  
CTG GCA AAG ATG GAA TTA TTC CTA ATG TTT GTG AGC CTA ATG CAG AGT TTC GCA TTT GCT TTA CCT GAG GAT  
L A K M E L F L M F V S L M Q S F A L P E D  
TCT AAG AAG CCC CTC CTG ACT GGA AGA TTT GGT CTA ACT TTA GCC CCA CAT CCA TTT AAT ATA ACT ATT TCA  
S K K P L L R G E F G T C A T A C C A T A T T A A T A C T  

AGG AGA TGA agagcatctccaagaagagatggtaaaaagatatataaatatatccttctaagcagattcttcctactgcaaaggcagtgaat  
R R  

ccagcaactcagtggatccaagctgggctcagagggtagagcacactgggaggtttcatctttggaggattcctcagcgaggatactt  
cagccattttagtaatgcaggtctgtgatTTTTGGGATAGAAAAAAGTACCTATGAAACGGGATATCTGGATTTTACTTTGCAGTGGCCTTCCACCg  
atggccaaactcttctcatTTTTCTTAGTGCCtCAGACATCCCCATATGTTAAAAATGAGAGATAAAAACTTGGGCTTCTCTACCTCTCAGCACtAATGAT  
GGTCAAATGcCTTTAcATCTTTTTCTGATATCTCTAAAAATGCTGTTAAGTTCTGGGAGAAGAActTCAGGAGAAGAAGATCTATCAGCTGGCTTTTAAAG  
acctatgacaaacatgaaaagtggtgttcagcgcctggaatgctttgtcagagatggggtgtggattaggttatatactgggggagaacttttctcagcacag  
attctatgccagcttctttggcttgtttctgtcactatctttttgttttatgtttttacttttttgtagatgtgggatgaagtgactctgt  
cgtgtatattgagggAAAAAAGAAAAAttataaTTTTAAAAACCTTTTGTAGGATTATTATCTAAATTTTATATGTCTAActtctactacaactacagga  
acagtgagccttgctacttcttttagtagcttctttggcagaattccttttctactgagtatttgcaaagatgcagctcacctttttacttaaggcct  
gaatggtgagcatgggattttgactgggactcatcaggaaggattctgctttcaaactatactgaacattcctgtgcctagcgtccctgccc  
agggccaatgcatctgatccttgaatatatactctaaagaattcactctcgtttttattaagagaactaaattgttagatgggtccctctg  
gaaaagcagtttttcagcagggggttaaccccttcagagggagttttgaaatgtgtggtgatgattttggttatcataaatgatgggggtgctactg  
gccttctgctgcccattgggaccaggatgctaaatgtcaaggtagtcctatacagtgaagaattgtgcctgctcaagatgccaggattccccagtgag  
aacatgctctaaggaatgaccacccctttctcccatgctgtatgtttctgttaattcttacataaattctattaagagggggcatactgttagtg  
gaacctttcctcatgctgggctctacacttaatccttttacttgtatgttttctgtaatttcttacataaattctattaagaggggcatactgttagtg  
atgaagctgaggccttatagtaggtaaggcacaAAAGTTAAAAAGTAACATCactgggTTTCCAACCTACTGGTCTGTGACTAAAGAACACTTTCAg  
aaccacttcttgattctgcccacacttgatccccataacaggctaccccttgcatgctggagtttgtgtgtgtctgtcatcccaggctgagc  
tccttgagggtgaggatgTTTGTGCTTTTGCCCTCACAGTGCCttgTTTGTGTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTGTTTTGTG
Identification and Characterization of CYP2U1

CYP2U1 (4p15+33)

Chromosome 4

CYP4f2  
CYP4f3  
CYP4f11  
CYP4f12  
CYP4f8  
CYP4f22  
CYP4a11  
CYP4a22  
CYP4b1  
CYP4x1  
CYP4z1  
CYP4v2  
CYP46  
CYP26B1  
CYP26C1  
CYP26A1  
CYP3a4  
CYP3a7  
CYP3a5  
CYP3a43  
CYP5a1  
CYP11b1  
CYP11b2  
CYP11a1  
CYP24  
CYP27b1  
CYP27c1  
CYP27A1  
CYP7a1  
CYP7b1  
CYP8a1  
CYP8b1  
CYP39a1  
CYP51  
CYP19  
CYP2c19  
CYP2c9  
CYP2c18  
CYP2c8  
CYP2e1  
CYP2f1  
CYP2a6  
CYP2a7  
CYP2a13  
CYP2b6  
CYP2s1  
CYP2w1  
CYP2j2  
CYP2r1  
CYP2U1  
CYP2d6  
CYP1a1  
CYP1a2  
CYP1b1  
CYP17  
CYP21a2  
CYP20  

FIG. 2. Phylogenetic tree. Phylogenetic tree of the 57 human P450s was compiled using Clustal method with PAM250 residue weight table. The arrow indicates the position of CYP2U1. A band on chromosome 4 shows the position of the CYP2U1 gene.

EXPERIMENTAL PROCEDURES

Materials—[α-32P]dATP was purchased from PerkinElmer Life Sciences. 20-Hydroxy-AA was purchased from Cayman Chemical (Ann Arbor). [1-14C]Arachidonic, docosahexaenoic, linolenic, eicosapentaenoic (EPA), docosatraenoic (DTA), and eicosatrienoic (ETA) acids and all other chemicals were purchased from Sigma unless specified.

Full-length cDNA Cloning and Sequencing—Human EST, genomic and High Throughput Genomic Sequence data bases (NCBI, Bethesda, MD) were searched using TBLASTN, BLASTN, and BLASTX algorithms. EST clone AI216236 and genomic clone AC000016 were identified and purchased from Research Genetics (Birmingham, AL). Custom oligonucleotide synthesis and DNA sequencing were performed by Cortec (Kingston, ON, Canada). A human thymus 5′-STRETCH Plus cDNA library (Clontech) was screened according to the manufacturer's instructions. A [α-32P]dATP-radiolabeled probe was prepared by random priming using as template a purified 380-bp fragment of EST clone AI216236 corresponding to nucleotides 1400–1779 (accession no. AY343323). One clone was identified, and based on the sequence data this clone lacked the 5′- and 3′-ends. To obtain the full-length cDNA sequence of CYP2U1, the 5′- and 3′-ends were amplified using the SMART RACE cDNA amplification kit (as per the manufacturer’s instructions, Clontech) and thymus cDNA as template. CYP2U1 specific primers 5′-GTT GCT TGG ATT CAC TGT CCT T-3′, 5′-GGT TTC TCC CAA ATG GCT GGG TCT CT-3′, and 5′-CCA CAG TTA GCC TCT GCA CTT CC-3′ were synthesized and used with the primers provided in the amplification kit. Unique PCR products were then cloned into a pTAdv vector (Clontech) and sequenced. The full-length clone of CYP2U1 (accession no. AY343323) was assembled from the various fragments using a homologous recombination kit (Pangene, Fremont CA) as per the manufacturer’s instructions.

and 100 μM, respectively) just prior to infection with recombinant baculovirus. Seventy-two hours later, transfected cells were harvested, and crude microsomal preparations were prepared as described by Zeldin *et al.* (26). Total protein concentration was estimated by the bicinchoninic acid method. Microsomal preparations were stored in small aliquots in liquid nitrogen. The presence of a functional CYP2U1 protein in the microsomal preparation was confirmed by reduced carbon monoxide difference spectroscopic analysis.

**Immunoblot Analysis of CYP2U1 Expression—** Polyclonal rabbit antibodies were raised against keyhole limpet hemocyanin-conjugated oligopeptide IKDHQESLDREN PQD corresponding to amino acids 302–316 of the CYP2U1 protein. Peptide synthesis and immunizations were conducted by Research Genetics (Huntsville, AL). Sf9 insect cells and CYP2U1-transfected Sf9 cells were lysed and analyzed in 10% SDS-polyacrylamide gels (under reducing conditions). Western blots were performed according to the manufacturer’s chemiluminescence detection system instructions (Amersham Biosciences) and hybridized with rabbit anti-CYP2U1 polyclonal antibody or anti-His<sub>6</sub> monoclonal antibody (Invitrogen).

**Metabolism of Fatty Acids and Product Characterization—** Sf9 cell microsomal preparations (0.1–1 mg of protein/ml final concentration) were diluted in assay buffer (0.1 M potassium phosphate, pH 7.4, 5 mM MgCl<sub>2</sub>, 1 mM EDTA). When necessary, exogenous substrate (up to 100 μM final concentration) was added in ethanol solution. The mixture was preincubated at 37 °C for 5 min in a water bath under constant mixing. The reactions were started by the addition of NADPH-regenerating mix (1 mM NADPH, 1 unit/ml glucose-6-phosphate dehydrogenase, 2 mM glucose 6-phosphate, final concentration) and carried out for 5–60 min at 37 °C. Aliquots were taken and stopped with acetic acid. An internal standard (100 ng of myristic acid) was added to each reaction, and the samples were extracted twice with acidified ethyl acetate. The metabolites were analyzed by LC-MS. Inhibition studies were conducted as described above in the presence of 0.1–50 μM ketoconazole or 17-octodecynoic acid (17-ODYA).

**HPLC and MS Analysis—** LC-MS was performed using a Micromass Quattro Ultima triple-stage quadrupole mass spectrometer (Manchester, UK) connected to a Waters Alliance 2969 HPLC system equipped with a photodiode array UV (200–400 nm) detector (Milford, MA). The analytical column was a Zorbax C18 Eclipse XDB 150 × 2.1 mm (inner diameter) (5-μm particle size) (Agilent Technologies, Palo Alto, CA) and maintained at 25 °C. Fast separation was achieved using a 30-min HPLC linear gradient at a flow rate of 0.2 ml/min. The mobile phase consisted of water (solvent A), acetonitrile (solvent B), and 10% glacial acetic acid (solvent C). The initial mobile phase conditions were set at 39.5% A, 59.5% B, and 1% C. Solvent B was then increased to 99% with a constant flow of 1% C. The mobile phase was then held isocratically for 10 min at 99% B and 1% C before returning to the initial conditions. For better separation of metabolites, the following HPLC linear gradient at a flow rate of 0.2 ml/min was used. Initial mobile phase conditions were set at 54% A, 45% B, and 1% C for 1 min. Solvent B was increased to 55% B with a constant flow of 1% C at 30 min. The mobile phase was then held isocratically for 10 min at a composition of 44% A, 55% B, and 1% C. Solvent B was increased to 99% in 5 min while maintaining C at 1%. The mobile phase was held isocratically for 15 min before returning to the initial conditions over 5 min and equilibrating for 5 min.

The mass spectrometer was operated in negative electrospray mode with a –2.86-kV capillary voltage and –16-V cone voltage. Nitrogen was used as a drying gas. Solvent flow rate was fixed at 200 μl/min with a cone gas flow of 112 liters/h, desolvation gas flow of 778 liters/h, a source temperature of 149 °C, and a desolvation temperature of 200 °C. For tandem MS experiments, argon was used as a collision gas at a pressure of 2.3 × 10<sup>−3</sup> torr with collision energy of 22 V. The quadrupole mass resolution settings were fixed at 15.0, and the multiplier voltage was 650 V. The ion source and the mass spectrometer parameters were optimized by infusion of 10 μl/min AA standard solution into the HPLC eluent (75% acetonitrile, 24% water, and 1% of 10% glacial acetic acid) flowing at 200 μl/min. The mass spectrometer was operated in a full mass scan (*m/z* 150–500) and product ion scan mode. Metabolites of AA were characterized by performing product ion scans of the molecular ion.

**Competition Assay—** Microsomes isolated from CYP2U1-transfected Sf9 cells were incubated as described above with 2.5 μM [1-<sup>14</sup>C]AA and 0–25 μM unlabeled fatty acids. In these experiments, AA metabolism was measured for each concentration of EPA, DHA, DTA, ETA, and linolenic acid and plotted on a graph to determine whether these compounds compete with AA for the substrate binding site.

Identification and Characterization of CYP2U1

A.  
B.  
C.

FIG. 4. Detection of recombinant CYP2U1 protein. A, cell lysates from Sf9 cells transfected with empty vector (U), pVL-CYP2U1-His₆ vector (1), and pVL-CYP2U1 vector (2) were analyzed by Western blot using anti-His₆ monoclonal antibody. B, Western blot analysis using rabbit anti-CYP2U1 polyclonal antibody reveals CYP2U1 as a ~55-kDa protein under reducing conditions. Positions of the molecular mass markers (in kDa) are indicated. C, carbon monoxide/reduced spectrum of microsomes containing recombinant CYP2U1 protein. A base line was recorded after the addition of an excess of sodium dithionite to both cuvettes. The difference spectrum was obtained after bubbling carbone monoxide for 45 s in the sample cuvette.

FIG. 5. Metabolism of AA by CYP2U1. Microsomal preparations from Sf9 cells transfected with empty vector (A) or pVL-CYP2U1 vector (B) were incubated at 37 °C with AA as described under “Experimental Procedures.” After 30 min, the reaction products were extracted and resolved by LC-MS. 1 and 2 denote two products formed with both [M-H]⁻ equal to 319.

RESULTS

**Molecular Cloning of CYP2U1 cDNA**—In an effort to discover novel human P450s, we searched the human EST data base using TBLASTN with a consensus sequence for the P450 heme binding domain PF(G/S)XGX(A/R/H)XCXG. One of the identified sequences (accession no. AI216236) closely matched a partial genomic sequence (accession no. AC00016) that was selected in a similar search of the human genomic data base. To identify which tissue RNA should be used for the full-length cloning of this novel human P450, we screened a human RNA dot blot containing poly(A)⁺ RNA samples from a variety of tissues and cell lines using a 381-bp probe from the EST clone AI216236. A strong hybridization signal was detected on dot blots loaded with mRNA from adult and fetal thymus tissue (data not shown).

Subsequently, we screened a human thymus cDNA library that resulted in an assembly of full-length cDNA (3,544 nucleotides) encoding 544 amino acids (Fig. 1). The predicted amino acid sequence contains many of the hallmark features of cytochrome P450s (27): heme binding domain **PFGIGKRVCMGA** (underlined are the amino acids from position 482 to 492 which are conserved in the heme binding domain (with the axial Cys ligand in bold to the heme)); PERF domain (PNRF position 464 to 467) which forms an ERR triad (28), using Glu and Arg from the K-helix domain and Arg from the PERF domain; ETLR domain (EVQR) conserved sequence was also found 50 amino acids upstream the PERF domain. The CYP2U1 designation for this gene was approved by the Cytochrome P450 Nomenclature Committee (Dr. David Nelson Web Page: www.dnelson.utmem.edu/CytochromeP450.html).

CYP2U1 gene has five exons (Fig. 1) and therefore differs from most of the CYP2 family members, which have nine exons. In the CYP2U1 gene, the position of introns 1, 2, and 4 is conserved with other members of the CYP2 family; however, the third intron position is different. The alignment analysis of CYP2U1 amino acid sequence reveals that it has the highest percentages of identity to fish AA epoxygenases, CYP2N1 and CYP2N2, with 39.6 and 38.4% identity, respectively (29).

6310

Identification and Characterization of CYP2U1

A.

FIG. 6. CYP2U1 metabolizes AA to 19- and 20-HETE. Product ion spectrum of the two oxygenated derivatives identified as peak 1 (A) and peak 2 (B) (as shown in Fig. 5). Drawings represent predicted 19- and 20-HETE fragmentation scenarios, respectively.

B.

phylogenetic comparison of amino acid sequence with other human CYPs showed that CYP2U1 is most closely related to CYP2J2 (30), an AA epoxygenase gene, and CYP2R1, a microsomal vitamin D 25-hydroxylase (31) with 36 and 37.1% amino acid sequence identity, respectively (Fig. 2).

Tissue Distribution of CYP2U1 mRNA—The expression of CYP2U1 mRNA was determined in the human by real-time PCR (Fig. 3). These experiments revealed that CYP2U1 mRNA was most abundant in the thymus and cerebellum and present at lower levels in numerous other tissues. The quantitative analysis showed that CYP2U1 expression in thymus is ~3-fold higher than in the cerebellum and about 200 magnitudes higher compared with other tissues.

Recombinant Expression of CYP2U1 in Sf9 Insect Cells—To characterize the substrate and enzymatic activity of CYP2U1, we used baculovirus to express CYP2U1 protein. A recombinant CYP2U1-His₆ protein was coexpressed with hOR and hb₅ in Sf9 insect cells. The protein expression was verified by Western blot and spectral analysis. The anti-His₆ antibody cross-reacted with a ~55–60-kDa protein present in the microsomal fraction of infected Sf9 cells (Fig. 4A). In addition, anti-CYP2U1 antibody confirmed the presence of CYP2U1 protein (Fig. 4B). Spectral analysis using microsomal fraction from Sf9 cells infected with CYP2U1 baculovirus showed a characteristic carbon monoxide/reduced 450 nm absorption peak (Fig. 4C). From these data, we estimated the presence of 0.5 nmol of recombinant CYP2U1 protein per mg of total protein in microsomal fractions.

Metabolism of Arachidonic Acid by CYP2U1—Sequence similarities between CYP2U1 and others CYP2 family members prompted us to study the potential role of CYP2U1 in fatty acid metabolism. Microsomal fractions containing recombinant

FIG. 7. Metabolism of DHA by CYP2U1. Microsomal preparations from Sf9 cells transfected with empty vector (- - - -) or pVL-CYP2U1 vector (——) were incubated at 37 °C with DHA as described under “Experimental Procedures.” After 30 min, the reaction products were extracted and resolved by LC-MS. Peak A and Peak B denote two products formed.

CYP2U1 protein were incubated in the presence of 10 μM exogenous AA and analyzed by LC-MS at different time intervals. Arachidonic acid was metabolized by CYP2U1 in a time-dependent manner and with an apparent \(K_m\) value of 2.7 μM. LC-MS chromatograms revealed the presence of two more polar, water-soluble metabolites of AA as indicated in Fig. 5 as peak 1 and peak 2. These compounds were identified based on MS data. The negative electrospray mode mass spectrum of the two products is shown in Fig. 6. The [M-H]⁻ ion at m/z 319 corresponds to a single carboxylate anion of oxygenated derivatives of AA (HETE). The product ion spectrum of the two oxygenated derivatives is shown in Fig. 6, A and B. For peak 1, the product ion spectrum yielded fragment ions at m/z 301, 275, 259, 231, 203, 177, 135, 121, and 107. The fragment at m/z 231

Identification and Characterization of CYP2U1

A.
Linolenic acid
(m/z 277)
Relative intensity
100
50
m/z 293
Time (min)

B.
EPA (m/z 301)
Relative intensity
100
50
m/z 317
Time (min)

C.
ETA (m/z 305)
Relative intensity
100
50
m/z 326
Time (min)

FIG. 8. Metabolism of linolenic acid, EPA, and ETA by CYP2U1. Microsomal preparations from Sf9 cells transfected with empty vector (- - - -) or pVL-CYP2U1 vector (——) were incubated at 37 °C with linolenic acid (A), EPA (B), or ETA (C) as described under “Experimental Procedures.” After 30 min, the reaction products were extracted and resolved by LC-MS. The mass spectrometer was operated in a full mass scan (m/z 150–500). To assess metabolism, the following ions (m/z) were extracted: A, 277 and 293; B, 301 and 317; and C, 305 and 326. These ions correspond to the original substrate and oxygenated metabolites.

probably arises from the combined loss of CO₂ and CH₂OH (see inset of Fig. 6A). The product ion spectrum of peak 1 yields characteristic fragment ions at m/z 275 and 231, which were reported previously by Bylund and coauthors (21) to be characteristic fragments for 19-HETE.

The product ion spectrum of the second metabolite (peak 2) yielded fragment ions at m/z 301, 289, 275, 257, 245, 109, 107, and 97, with two characteristic signals at m/z 289 and 245 (21). The fragment at m/z 245 might be the result of the combined loss of CO₂ and CH₂O (see inset, Fig. 6B). The retention time and product ion spectrum of this metabolite (peak 2) were consistent with the formation of 20-HETE. Moreover, the use of a reference confirmed the identity of peak 2 as 20-HETE (data not shown). From these observations, it was confirmed that CYP2U1 converts AA into two bioactive derivatives, 19- and 20-HETE.

CYP2U1 Substrate Specificity—To assess the substrate binding properties of different fatty acids to CYP2U1, competition experiments using 2.5 μM [1-¹⁴C]AA and different concentrations (up to 25 μM) of unlabeled fatty acids were conducted. In these experiments, we measured a concentration-dependent decrease in AA metabolism when linolenic acid, EPA, DHA, and ETA were added to the incubation, confirming their ability to compete with AA for the binding site of CYP2U1 (data not shown).

TABLE I
CYP2U1 substrates specificity

Sf9 cell microsomal preparations containing CYP2U1 were incubated in the absence or presence of exogenous substrate at 37 °C for 1 h in a water-bath under constant mixing. After extraction with acidified ethyl acetate, the metabolites were analyzed by LC-MS. The abbreviations are: –, not metabolized by CYP2U1; +, metabolized by CYP2U1; ND, not determined.

| Substrate               | In presence of exogenous substrate | In absence of exogenous substrate |
|-------------------------|------------------------------------|------------------------------------|
| Lauric acid (C12:0)     | –                                  | –                                  |
| Palmitic acid (C16:0)   | ND                                 | +                                  |
| Palmitoleic acid (C16:1)| ND                                 | +                                  |
| Stearic acid (C18:0)    | ND                                 | +                                  |
| Vaccenic acid (C18:1)   | ND                                 | +                                  |
| Linoleic acid (C18:2)   | –                                  | –                                  |
| Linolenic acid (C18:3)  | +                                  | ND                                 |
| ETA (C20:3)             | +                                  | ND                                 |
| AA (C20:4)              | +                                  | +                                  |
| EPA (C20:5)             | +                                  | +                                  |
| DHA (C22:6)             | +                                  | +                                  |

To characterize further the substrate specificity of CYP2U1, incubations of fatty acids with different chain lengths or degrees of saturation were conducted. HPLC analysis demonstrated that linolenic acid, EPA, ETA, and DHA are substrates of recombinant CYP2U1 (Figs. 7 and 8 and Table I). In the case

6312

**Identification and Characterization of CYP2U1**

---

**A.**
![Graph A](#)
- **Relative intensity**
- **Time (min)**

**B.**
![Graph B](#)
- **Relative intensity**
- **Time (min)**

**C.**
![Graph C](#)
- **Relative intensity**
- **Time (min)**

**D.**
![Graph D](#)
- **Relative intensity**
- **Time (min)**

**E.**
![Graph E](#)
- **Relative intensity**
- **Time (min)**

**F.**
![Graph F](#)
- **Relative intensity**
- **Time (min)**

**G.**
![Graph G](#)
- **Relative intensity**
- **Time (min)**

---

**FIG. 9. Endogenous metabolism of fatty acids by CYP2U1.** *Sf9* cells transfected with empty vector (- - - -) or pVL-CYP2U1 vector (——) were incubated at 37 °C as described under “Experimental Procedures.” After 30 min, the reaction products were extracted and resolved by LC-MS. The mass spectrometer was operated in a full mass scan (*m/z* 150–500). To assess metabolism of, the following ions (*m/z*) were extracted: *A*, 255 and 271; *B*, 253 and 269; *C*, 283 and 299; *D*, 281 and 297; *E*, 301 and 317; *F*, 303 and 319; and *G*, 327 and 343 for palmitic acid, palmitoleic acid, stearic acid, vaccenic acid, EPA, AA, and DHA, respectively. These ions correspond to the original substrate and oxygenated metabolites.

of DHA, Fig. 7 shows the LC-MS chromatogram of more polar metabolites generated by CYP2U1. The two metabolites have a retention time in our HPLC conditions of 26.9 and 37 min, for peak A and B, respectively. In contrast to the mass to charge ratio of the deprotonated DHA ([M-H]⁻ = 327), the [M-H]⁻ for these two metabolites is equal to 343, which could correspond to the addition of one molecular oxygen ([M-H]⁻ + 16). Based on this observation, we postulate that the metabolites formed correspond to a ω- and (ω-1)-hydroxylated DHA or to an epoxy-DHA such as 19,20-epoxydocosapentaenoic acid. Because of the absence of available reference compounds, the identification of metabolites formed could not be established. Moreover, CYP2U1 did not show any metabolism for lauric acid (C12:0) or linoleic acid (C18:2) even after the addition of high concentration (up to 100 μM) of exogenous substrate (data not shown).

We also investigated whether endogenous fatty acids present in insect cell membranes could be metabolized by CYP2U1. To this end, we performed LC-MS analysis using microsomes ex-

pressing recombinant CYP2U1 or empty vector. LC-MS analysis showed that CYP2U1 was able to metabolize saturated (palmitic and stearic acids) or unsaturated (palmitoleic, vaccenic, eicosapentaenoic, arachidonic, and docosahexaenoic acids) long chain fatty acids (C16–C22) to more polar water-soluble products (Fig. 9 and Table I). For most of these fatty acids, two metabolites were formed after incubation with recombinant CYP2U1. For each of the metabolites formed, the *m/z* ([M-H]⁻) (Fig. 9) was consistent with the insertion of a molecular oxygen leading to the formation of a hydroxy- or epoxy-fatty acid.

Inhibition Studies—Based on the fact that ketoconazole is known to be a nonspecific P450 inhibitor and 17-ODYA a specific P450 ω/(ω-1)-hydroxylase inhibitor, we measured the ability of these two compounds to inhibit CYP2U1 metabolism. We used incubations containing different concentrations of ketoconazole or 17-ODYA (0.05–50 μM) in the presence of microsomal fractions from CYP2U1/hOR/hb₅-transfected Sf9 cells, NADPH, NADPH-regenerating system, and 2 μM AA. By plotting the percentage of AA metabolism inhibition against concentrations of ketoconazole or 17-ODYA, IC₅₀ values (concentration required for 50% inhibition) of 15 and 5 μM were determined, respectively.

### DISCUSSION

This work describes the enzymatic activities of CYP2U1, the first of this novel class of cytochrome P450s to be characterized. This study also suggests that CYP2U1 plays critical roles in brain and thymus function. We have shown that CYP2U1 metabolizes AA to two bioactive metabolites 19- and 20-HETE and demonstrated that CYP2U1 was able to metabolize saturated or unsaturated long chain fatty acids (C16–C22) to metabolites that potentially could be involved in lipid signaling pathways. Furthermore, the specific expression of this gene was clearly established in thymus as well as cerebellum.

CYP2U1 protein shares 39.6 and 38.4% amino acid sequence identity with two fish AA epoxygenases, CYP2N1 and CYP2N2 (29), respectively. Surprisingly, the CYP2U1 gene is localized on chromosome 4 at position 4Q25, whereas other CYP2 family members are clustered on chromosome 10. This might suggest that no duplication event has occurred recently for this gene unlike the majority of other CYP2 family members. Also, the CYP2U1 gene is well conserved across species, sharing 52.6, 78.9, and 88.9% amino acid sequence identity with fish (*Takifugu rubripes*), mouse (*Mus musculus*), and rat (*Rattus norvegicus*) Cyp2U1, respectively.

We have established that CYP2U1 can metabolize saturated or unsaturated long chain fatty acids (Table I). The metabolism of short chain fatty acid was also investigated, but our results show that lauric acid (C12) was not a substrate of CYP2U1 (Table I), demonstrating the preference of this P450 for longer fatty acids (C16–C22). A broad substrate specificity toward different fatty acid chain lengths has been shown for several P450s in mammals (32), plants (33) and microorganisms (34). However, the CYP2U1 apparent tight substrate selectivity toward long chain fatty acid appears to be uncommon.

Previously, in human, the ω- and/or (ω-1)-hydroxylation of AA has only been shown for members of the CYP4 family, namely CYP4Fs and CYP4A11. Our data show that CYP2U1 has the ability to transform AA into two polar metabolites, 19- and 20-HETE, characterized previously as bioactive compounds. In others species (e.g. mouse), members of the CYP2 family have also been characterized to hydroxylate AA at the 19 and 20 positions.

Important physiological functions, especially in the regulation of ion channels, have been described for 19- and 20-HETE. In kidney and brain for example, 20-HETE has been characterized as a powerful vasoconstrictor (35, 36). In rat brain, Kehl and coauthors (37) have shown that 20-HETE contributes to the acute fall in cerebral blood flow after subarachnoid hemorrhage. Moreover, 20-HETE, as well as 19-HETE, has been shown to regulate calcium-dependent K⁺ channels in vascular smooth muscle (38). 19-HETE can inhibit the activity of recombinant P/Q-type Ca²⁺ channels that are known to be expressed preferentially in cerebellar Purkinje cells and are involved in triggering neurotransmitter release (38). A highly expressed P450 in murine cerebellar Purkinje cells, Cyp2J9, has been shown to metabolize exclusively AA to 19-HETE (24). Through the formation of 19- and 20-HETE CYP2U1 could also play a role in controlling ion transport in specific organs/tissues and consequently be involved in the regulation of blood pressure in these organs.

We have demonstrated that CYP2U1 can also act on fatty acids such as DHA or EPA which are associated with decreased triglyceride levels and decreased risk of developing coronary heart disease. So far, no data have been previously published on the identification of a human cytochrome P450 involved in the metabolism of DHA or EPA. In rabbit, a member of the CYP4 family, CYP4A7, was shown to catalyze the ω-hydroxylation of DHA (39). DHA is the most unsaturated fatty acid normally occurring in mammals and fish and has been found in high concentrations in brain, retina, and testis. DHA and EPA have been shown to be the major n-3 fatty acids found in fish oils. DHA signaling through retinoid X receptors has also been proposed in mouse brain (40). Interestingly, adult rat brain was shown to express high P450-dependent ω-hydroxylation activity of DHA (41).

Considering the expression of CYP2U1 in cerebellum and its ability to generate 19- and 20-HETE, we can postulate that it might play a role in the regulation of ion channels or neurotransmitters in defined neuronal cells or be involved in the modulation of cerebellar blood flow. The fact that CYP2U1 metabolizes DHA to possibly 22- and 21-hydroxy-DHA and that it is expressed in the cerebellum suggest that CYP2U1 is a good candidate for the ω-hydroxylation of DHA in the brain.

We have established that CYP2U1 is highly expressed in the thymus, suggesting a physiological role in regulating and/or development of immune function. AA-derived eicosanoids such as prostaglandin E₂ are known as proinflammatory mediators and regulators of immune cell function of cells of the immune system. Also, an enriched diet of long chain n-3 polyunsaturated fatty acids such as DHA and EPA have been shown to reduce proliferation of lymphocyte stimulated with T- or B-cell mitogens (42) and to lower the proportion of spleen and thymic lymphocytes bearing the interleukin-2 receptors (CD25) after ConA stimulation (42, 43). CYP2U1 may play a significant role in modulating signal transduction and/or gene expression within inflammatory and immune cells through AA, DHA, and/or EPA metabolism in thymus.

#### Acknowledgments—We thank Laura Hampel, Jan Wisniewski, Sumsullah Khan, and Wayne A. Stangle for providing technical assistance. We thank Samir Tabash for helpful comments during the preparation of this manuscript.

#### REFERENCES

1. Capdevila, J. H., Falck, J. R., and Harris, R. C. (2000) *J. Lipid Res.* **41**, 163–181
2. Capdevila, J. H., and Falck, J. R. (2001) *Biochem. Biophys. Res. Commun.* **285**, 571–576
3. Fang, X., Weintraub, N. L., Stoll, L. L., and Spector, A. A. (1999) *Hypertension* **34**, 1242–1246
4. Fleming, I. (2001) *Circ. Res.* **26**, 753–762
5. Planaguma, A., Titos, E., Lopez-Parra, M., Gaya, J., Pueyo, G., Arroyo, V., and Claria, J. (2002) *FASEB J.* **16**, 1937–1939
6. Nie, D., Che, M., Grignon, D., Tang, K., and Honn, K. V. (2001) *Cancer Metastasis Rev.* **20**, 195–206
7. Glenn, H. L., and Jacobson, B. S. (2003) *Cell Motil. Cytoskeleton* **55**, 265–277
8. Stockton, R. A., and Jacobson, B. S. (2001) *Mol. Biol. Cell* **12**, 1937–1956
9. Seeds, M. C., and Bass, D. A. (1999) *Clin. Rev. Allergy Immunol.* **17**, 5–26

10. Parnes, S. M. (2002) *Expert Opin. Pharmacother.* **3**, 33–38
11. Moreno, C., Maier, K. G., Hoagland, K. M., Yu, M., and Roman, R. J. (2001) *Am. J. Hypertens.* **14**, 90S–97S
12. Bazan, N. G., Tu, B., and Rodriguez de Turco, E. B. (2002) *Prog. Brain Res.* **135**, 175–185
13. Funk, C. D. (2001) *Science* **294**, 1871–1875
14. Imig, J. D. (2000) *Am. J. Physiol.* **279**, F965–F981
15. Ziboh, V. A., Cho, Y., Mani, I., and Xi, S. (2002) *Arch. Pharm. Res.* **25**, 747–758
16. Capdevila, J. H., Harris, R. C., and Falck, J. R. (2002) *Cell Mol. Life Sci.* **59**, 780–789
17. Fang, X., Kaduce, T. L., Weintraub, N. L., Harmon, S., Teesch, L. M., Morisseau, C., Thompson, D. A., Hammock, B. D., and Spector, A. A. (2001) *J. Biol. Chem.* **276**, 14867–14874
18. Oliw, E. H. (1994) *Prog. Lipid Res.* **33**, 329–354
19. Rifkind, A. B., Lee, C., Chang, T. K., and Waxman, D. J. (1995) *Arch. Biochem. Biophys.* **320**, 380–389
20. Bylund, J., Kunz, T., Valmsen, K., and Oliw, E. H. (1998) *J. Pharmacol. Exp. Ther.* **284**, 51–60
21. Bylund, J., Ericsson, J., and Oliw, E. H. (1998) *Anal. Biochem.* **265**, 55–68
22. Wu, S., Moomaw, C. R., Tomer, K. B., Falck, J. R., and Zeldin, D. C. (1996) *J. Biol. Chem.* **271**, 3460–3468
23. Aoyama, T., Hardwick, J. P., Imaoka, S., Funae, Y., Gelboin, H. V., and Gonzalez, F. J. (1990) *J. Lipid Res.* **31**, 1477–1482
24. Qu, W., Bradbury, J. A., Tsao, C. C., Maronpot, R., Harry, G. J., Parker, C. E., Davis, L. S., Breyer, M. D., Waalkes, M. P., Falck, J. R., Chen, J., Rosenberg, R. L., and Zeldin, D. C. (2001) *J. Biol. Chem.* **276**, 25467–25479
25. Higgins, D. G., and Sharp, P. M. (1989) *Comput. Appl. Biosci.* **5**, 151–153
26. Zeldin, D. C., DuBois, R. N., Falck, J. R., and Capdevila, J. H. (1995) *Arch. Biochem. Biophys.* **322**, 76–86
27. Werck-Reichhart, D., and Feyereisen, R. (2000) *Genome. Biol.* **1**, REVIEWS3003

28. Zheng, Y. M., Fisher, M. B., Yokotani, N., Fujii-Kuriyama, Y., and Rettie, A. E. (1998) *Biochemistry* **37**, 12847–12851
29. Oleksiak, M. F., Wu, S., Parker, C., Karchner, S. I., Stegeman, J. J., and Zeldin, D. C. (2000) *J. Biol. Chem.* **275**, 2312–2321
30. Zeldin, D. C. (2001) *J. Biol. Chem.* **276**, 36059–36062
31. Cheng, J. B., Motola, D. L., Mangelsdorf, D. J., and Russell, D. W. (2003) *J. Biol. Chem.* **278**, 38084–38093
32. Koike, K., Kusunose, E., Nishikawa, Y., Ichihara, K., Inagaki, S., Takagi, H., Kikuta, Y., and Kusunose, M. (1997) *Biochem. Biophys. Res. Commun.* **232**, 643–647
33. Benveniste, I., Tijet, N., Adas, F., Philipps, G., Salaun, J. P., and Durst, F. (1998) *Biochem. Biophys. Res. Commun.* **243**, 688–693
34. Boddupalli, S. S., Estabrook, R. W., and Peterson, J. A. (1990) *J. Biol. Chem.* **265**, 4233–4239
35. Roman, R. J., Maier, K. G., Sun, C. W., Harder, D. R., and Alonso-Galicia, M. (2000) *Clin. Exp. Pharmacol. Physiol.* **27**, 855–865
36. Kroetz, D. L., and Zeldin, D. C. (2002) *Curr. Opin. Lipidol.* **13**, 273–283
37. Kehl, F., Cambj-Sapunar, L., Maier, K. G., Miyata, N., Kametani, S., Okamoto, H., Hudetz, A. G., Schulte, M. L., Zagorac, D., Harder, D. R., and Roman, R. J. (2002) *Am. J. Physiol.* **282**, H1556–H1565
38. Escalante, B., Sessa, W. C., Falck, J. R., Yadagiri, P., and Schwartzman, M. L. (1990) *J. Cardiovasc. Pharmacol.* **16**, 438–443
39. Sawamura, A., Kusunose, E., Satouchi, K., and Kusunose, M. (1993) *Biochim. Biophys. Acta* **168**, 30–36
40. de Urquiza, A. M., Liu, S., Sjoberg, M., Zetterstrom, R. H., Griffiths, W., Sjovall, J., and Perlmann, T. (2000) *Science* **290**, 2140–2144
41. Yamane, M., and Abe, A. (1995) *Neurosci. Lett.* **200**, 203–206
42. McMurray, D. N., Jolly, C. A., and Chapkin, R. S. (2000) *J. Infect. Dis.* **182** (Suppl. 1) S103–S107
43. Yaqoob, P., Newsholme, E. A., and Calder, P. C. (1994) *Immunology* **82**, 603–610
