
(R,S)-Reticuline 7-O-methyltransferase and (R,S)-norcoclaurine 6-O-methyltransferase of Papaver somniferum – cDNA cloning and characterization of methyl transfer enzymes of alkaloid biosynthesis in opium poppy

Anan Ounaroon${}^{1,\dagger}$, Gabriele Decker${}^{2}$, Jürgen Schmidt${}^{1}$, Friedrich Lottspeich${}^{2}$ and Toni M. Kutchan${}^{1,*}$

${}^{1}$Leibniz-Institut für Pflanzenbiochemie, Weinberg 3, D-06120 Halle/Saale, Germany, and  
${}^{2}$Max-Planck-Institut für Biochemie, Am Klopferspitz 18a, D-82152 Martinsried, Germany

Received 20 June 2003; revised 9 September 2003; accepted 10 September 2003.  
*For correspondence (fax +49 345 5582 1209; e-mail kutch@ipb-halle.de).  
†Current address: Faculty of Pharmaceutical Sciences, Naresuan University, Phitsanulok 6500, Thailand.

---

### Summary

S-Adenosyl-L-methionine:(R,S)-reticuline 7-O-methyltransferase converts reticuline to laudanine in tetrahydrobenzylisoquinoline biosynthesis in the opium poppy *Papaver somniferum*. This enzyme activity has not yet been detected in plants. A proteomic analysis of *P. somniferum* latex identified a gel spot that contained a protein(s) whose partial amino acid sequences were homologous to those of plant O-methyltransferases. cDNA was amplified from *P. somniferum* RNA by reverse transcription PCR using primers based on these internal amino acid sequences. Recombinant protein was then expressed in *Spodoptera frugiperda* Sf9 cells in a baculovirus expression vector. Steady-state kinetic measurements with one heterologously expressed enzyme and mass spectrometric analysis of the enzymatic products suggested that this unusual enzyme is capable of carrying through sequential O-methylations on the isoquinoline and on the benzyl moiety of several substrates. The tetrahydrobenzylisoquinolines (R)-reticuline (4.2 sec${}^{-1}$ mM${}^{-1}$), (S)-reticuline (4.5 sec${}^{-1}$ mM${}^{-1}$), (R)-protosinomenine (1.7 sec${}^{-1}$ mM${}^{-1}$), and (R,S)-isoorientaline (1.4 sec${}^{-1}$ mM${}^{-1}$) as well as guaiacol (5.9 sec${}^{-1}$ mM${}^{-1}$) and isovanillic acid (1.2 sec${}^{-1}$ mM${}^{-1}$) are O-methylated by the enzyme with the ratio $k_{\text{cat}}/K_{m}$ shown in parentheses. A *P. somniferum* cDNA encoding (R,S)-norcoclaurine 6-O-methyltransferase was similarly isolated and characterized. This enzyme was less permissive, methylating only (R,S)-norcoclaurine (7.4 sec${}^{-1}$ mM${}^{-1}$), (R)-norprotosinomenine (4.1 sec${}^{-1}$ mM${}^{-1}$), (S)-norprotosinomenine (4.0 sec${}^{-1}$ mM${}^{-1}$) and (R,S)-isoorientaline (1.0 sec${}^{-1}$ mM${}^{-1}$). A phylogenetic comparison of the amino acid sequences of these O-methyltransferases to those from 28 other plant species suggests that these enzymes group more closely to isoquinoline biosynthetic O-methyltransferases from *Coptis japonica* than to those from *Thalictrum tuberosum* that can O-methylate both alkaloid and phenylpropanoid substrates.

Keywords: (R,S)-reticuline 7-O-methyltransferase, norcoclaurine 6-O-methyltransferase, alkaloid biosynthesis, opium poppy, *Papaver somniferum*.

---

### Introduction

S-Adenosylmethionine (AdoMet)-dependent O-methyltransferases involved in plant natural product biosynthesis yield methyl ether derivatives of hydroxylated polycyclic aromatic low-molecular weight compounds. Regiospecific oxygen methylation significantly contributes to the vast metabolic diversity of plant secondary metabolism. O-methyltransferase-encoding genes are tentatively identified and annotated based upon sequence similarity to those of other proteins (Ibrahim et al., 1998; Joshi and Chiang, 1998; Schröder et al.,

biosynthesis from *Thalictrum tuberosum* (Frick and Kutchan, 1999) and from *Coptis japonica* (Morishige et al., 2000). This can also occur within one species, as for caffeic acid 3-*O*-methyltransferase from *Nicotiana tabacum* (Maury et al., 1999). Because of the broad substrate specificities of some *O*-methyl transfer enzymes, prediction of the *in vivo* role of an *O*-methyltransferase is not trivial. This is further exemplified in sweet basil (Gang et al., 2002). To this end, determination of the steady-state kinetic parameters for available substrates can aid in predicting the specific function of an enzyme.

The opium poppy *Papaver somniferum* produces more than 80 tetrahydrobenzylisoquinoline-derived alkaloids. The most renowned of these are the narcotic analgesic phenanthrene alkaloids codeine and morphine. Other important alkaloids from this plant are the antitussive phthalidisoquinoline noscapine, the vasodilator papaverine, and the antimicrobial benzo[c]phenanthridine sanguinarine. A central biosynthetic pathway leads from two molecules of L-tyrosine to (*S*)-reticuline (Figure 1) (reviewed by Kutchan, 1998). The pathway then bifurcates as the (*S*)-reticuline molecule is regio- and stereospecifically transformed into committed isoquinoline subclass intermediates. Two classes of enzyme effectuate this diversification:

oxidoreductases and *O*-methyltransferases. In the specific pathway that leads to morphine (*S*)-reticuline is oxidized by (*S*)-reticuline oxidase to form the dehydrotetrolinium ion, which is then stereospecifically reduced to (*R*)-reticuline. To enter the sanguinarine pathway, the *N*-methyl group of (*S*)-reticuline is oxidatively cyclized by the berberine bridge enzyme to the bridge carbon (C-8) of (*S*)-scoulerine. The biosynthetic pathways that lead to papaverine and noscapine are not yet well understood at the enzyme level.

Requisite to metabolic engineering of commercial varieties of *P. somniferum* is the understanding of the alkaloid biosynthetic pathways at the molecular genetic level. To date, we have isolated a cDNA that encodes the cytochrome P-450-dependent monooxygenase (*S*)-*N*-methylcoclaurine 3′-hydroxylase (Huang and Kutchan, 2000; Pauli and Kutchan, 1998) and the corresponding cytochrome P-450 reductase (Rosco et al., 1997). This enzyme is common to the biosynthetic pathways of all the *P. somniferum* alkaloids. Specific to the sanguinarine pathway is the cDNA encoding the berberine bridge enzyme (Dittrich and Kutchan, 1991; Facchini et al., 1996; Huang and Kutchan, 2000). Finally, specific to morphine biosynthesis are the cDNAs for salutaridinol 7-*O*-acetyltransferase (Grothe et al., 2001) that results in the formation of the five-ring

**Figure 1. Schematic biosynthetic pathway leading from (*S*)-norcoclaurine to (*S*)-scoulerine, (*R*)-reticuline, and laudanine in *P. somniferum*.** The pathway from (*S*)-norcoclaurine to (*S*)-reticuline is central to the isoquinoline alkaloids accumulated in opium poppy. (*S*)-Reticuline is a branch point intermediate that is subsequently oxidized at C-1–N to lead into the morphinan pathway, or at N-CH₃ to proceed on to (*S*)-scoulerine-derived alkaloids such as the benzophenanthridines. In addition, reticuline can be 7-*O*-methylated to laudanine.

810 Anan Ounaroon et al.

system of the morphinans and for codeinone reductase, the penultimate enzyme of the morphine pathway that reduces codeinone to codeine (Unterlinner *et al.*, 1999).

In this work, we report on the isolation and characterization of cDNAs encoding two *O*-methyltransferases of tetrahydrobenzylisoquinoline alkaloid biosynthesis in *P. somniferum*: (*R*,*S*)-reticuline 7-*O*-methyltransferase and (*R*,*S*)-norcoclaurine 6-*O*-methyltransferase.

Results

Amino acid sequence analysis of a putative *O*-methyltransferase and isolation of the corresponding cDNA

Latex was harvested from field-grown *P. somniferum* by incising capsules 3–6 days after flower petal fall. The exuded latex was immediately added to ice-cold potassium phosphate buffer containing 20 mM sodium ascorbate and 500 mM mannitol, pH 7.2. The latex buffer ratio was approximately 1 : 1. Particulates were removed by centrifugation (Antoun and Roberts, 1975a; Roberts *et al.*, 1983) prior to two-dimensional polyacrylamide gel electrophoretic resolution of the proteins in the 1000 g supernatant according to Decker *et al.* (2000) (Figure 2). Internal amino acid microsequencing of spots in the size range expected for plant methyltransferase monomers (approximately 40 kDa) yielded five peptides from a gel spot that were homologous to *O*-methyltransferases. The amino acid sequences of these five peptides are as follows:

OMT-Pep 1 RTEAE  
OMT-Pep 2 VIIVDcVLRPDGNDL  
OMT-Pep 3 VGGDMFVDIPEADAV  
OMT-Pep 4 ILLNNAGFPRYNVIRTPAFPIIEA  
OMT-Pep 5 DGFSGIAGSLVDVGG  

Degenerated oligodeoxynucleotide PCR primers were derived from OMT-Pep 1 and OMT-Pep 5. PCR amplification of *P. somniferum* cDNA prepared from stem poly(A)+ RNA yielded a DNA band of the expected size (approximately 400 bp) upon analysis by agarose gel electrophoresis. Subcloning of the PCR product into pGEM-T Easy followed by nucleotide sequence determination of 230 randomly chosen samples identified only two independent *O*-methyltransferase-encoding partial cDNA clones denoted PSOMT1 and PSOMT2. Each *O*-methyltransferase partial sequence was used to design specific oligodeoxynucleotide primers for rapid amplification of cDNA ends (RACE)-PCR, by which cDNAs containing the entire open-reading frames for both *O*-methyltransferases were generated. The details of these experiments are provided in the Experimental procedures section.

Figure 2. Two-dimensional gel electrophoretic pattern of the cytosolic fraction proteins of latex collected from *P. somniferum* capsules. Fifty micrograms of protein was loaded per gel and visualized by silver staining. The arrow points to the position of the *O*-methyltransferase described in this work. Protein spots from Coomassie Brilliant Blue-stained gels were excised, the proteins were digested *in situ* with endoproteinase Lys-C, and the peptides were resolved and sequenced according to Decker *et al.* (2000).

Figure 3. Amino acid sequence comparison of PSOMT1 and PSOMT2 from *P. somniferum*. The shaded motifs are conserved regions motif A, J, K, B, C, and L indicative of plant methyltransferases according to Joshi and Chiang (1998). These sequence signatures were based upon plant methyltransferase amino acid sequence comparisons and were not functionally defined. They are mainly used to indicate whether unidentified proteins may be O-methyltransferases.

| PSOMT1                                                                 | PSOMT2                                                                 |
|-----------------------------------------------------------------------|-----------------------------------------------------------------------|
| MDTAEERLKGQAEIWEHMFAFVDSMA-LKCAVELGIPDIINSHGRPVTIS                      | METVSKIDQQNQAKIWKQIYGFAESLVLKCAVQLEIAETLHNNVKPMSLS                      |
| EIVDSLKTNTPSSSPNIDYLTRIMRLLVHKRLFTSELHQESNQLLYNLTR                      | ELASKLPVAQPVNEDRLFRI---MRYLVHMELFKIDATTQYSLAPPAKY                      |
| SSKWLLKDSKFNLSPVLVWETNPILLKPWQYLGKCAQEKKSPFERAHGCE                      | LLRGWEKSMVDSILCINDKDFLAPWHHLGDGLTGNCDAFEKALGKSIWVY                      |
| IWDLALADPKFNNFLNGAMQCSTTTIINEMLLEYKDGFSGIAGSLVDVGG                       | MSVNEPEKNQLFNAAMACDTRLVTSALANECKSIFSDGIST----LVDVGG                     |
| GTGSIIAEIVKAHPHIQGINFDLPHVVATAAEFPGVKHVGGDMEVDIPEA                       | GTGTAVKAISKAFPDIKCTIYDLPHVIADSPEIPNITKISGDMFKSIPSA                     |
| DAVIMKWILLHDWSDEDCTIILKNCYRAIRKKKNGKVIIVDCVLRPDGNDL                      | DAIFMKCILLHDWNDDECIQILKRCKEAL--PKGGKVIIVDVVIDMDSTHP                   |
| FDKMGLIFDVLMMAHTTAGKERTEAEWKILLNNAGFPRYNVIRTPAFPCI                       | YAKIRLTLDDLDMMLNT-GGKERTKEEWKTLFDAAGFASHKVTOISAVQSV                    |
| IEAFPE                                                                  | IEAYPY                                                                  |

**Sequence analyses of O-methyltransferases**

Translation of the complete nucleotide sequences of PSOMT1 and PSOMT2 yielded polypeptides of 356 and 347 amino acids, respectively. Amino acid sequence alignment carried out using the program from Heidelberg Unix Sequence Analysis Resources demonstrated 36% identity of the two proteins. Amino acid sequences of O-methyl transfer enzymes contain consensus sequences putatively involved in catalysis. Conserved motifs A, B, C, J, K, and L proposed by Joshi and Chiang (1998) are shown for PSOMT1 and PSOMT2 as shaded regions in Figure 3.

A phylogenetic diagram of 44 putative and defined O-methyltransferase amino acid sequences from 29 plant species was constructed using the Phylogeny Inference Package program (PHYLIP Version 3.57c) (Figure 4). Among these 44 sequences, PSOMT1 showed the closest relationship to a caffeic acid/caffeoyl CoA 3-O-methyltransferase from loblolly pine *Pinus taeda* (Li et al., 1997) and to a putative caffeic acid 3-O-methyltransferase from Monterey pine *Pinus radiata*. In contrast, PSOMT2 grouped together with norcoclaurine 6-O-methyltransferase from *C. japonica* (Morishige et al., 2000). The next most closely related sequence was 3'-hydroxy-N-methylcoclaurine 4'-O-methyltransferase, also from *C. japonica* (Morishige et al., 2000). These new *P. somniferum* O-methyl transfer enzymes group more closely to isoquinoline biosynthetic O-methyltransferases from *C. japonica* than to those identified from *T. tuberosum* (Frick and Kutchan, 1999). The percent amino acid sequence identity among these enzymes of isoquinoline alkaloid biosynthesis is listed in Table 1. The results of the phylogenetic analysis formed the basis for the enzyme assays that were later carried out with heterologously expressed cDNAs as reported below.

**Gene expression analyses**

RNA gel blot analysis suggests that PSOMT1 is expressed predominantly in bud and stem and, to a much lesser degree, in leaf of *P. somniferum* (Figure 5). In contrast, PSOMT2 transcript is detectable in bud, stem, leaf, and root and, to a lesser degree, in capsule (Figure 5). The distribution of PSOMT2 transcript parallels the distribution of transcript of several other genes of tetrahydrobenzylisoquinoline biosynthesis in *P. somniferum*. *Cyp80b1* that encodes the cytochrome P-450-dependent monooxygenase (S)-N-methylcoclaurine 3'-hydroxylase (Huang and Kutchan, 2000; Pauli and Kutchan, 1998) common to the biosynthetic pathways of all the *P. somniferum* alkaloids, *salAT* that encodes salutaridinol 7-O-acetyltransferase (Grothe et al., 2001) and *cor1* that encodes codeinone reductase (Unterlinner et al., 1999), both specific to morphine biosynthesis, are all expressed in bud, capsule, leaf, root, and stem. This gene transcript distribution of PSOMT2, taken together with the results of the phylogenetic analysis is congruent with PSOMT2 encoding norcoclaurine 6-O-methyltransferase of (S)-reticuline biosynthesis (Frick and Kutchan, 1999; Morishige et al., 2000).

The transcript distribution and phylogenetic analysis of PSOMT1 suggest that the gene product may be involved in tetrahydrobenzylisoquinoline alkaloid formation, but possibly not directly in the morphine biosynthetic pathway because of the absence of transcript in root.

**Purification and functional characterization of recombinant enzymes**

The PSOMT1 and PSOMT2 cDNAs were each constructed to express the recombinant proteins with six histidine

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), 36, 808–819

812 Anan Ounaroon et al.

![Figure 4](#fig4)

Figure 4. Phylogenetic tree of plant methyltransferases of defined and of unknown function.

Forty-four amino acid sequences of proteins of plant origin were compared to generate a tree that indicates the phylogenetic relationship between PSOMT1, PSOMT2, and other putative and defined *O*-methyltransferases. PSOMT1 grouped most closely to caffeic acid 3-*O*-methyltransferases from pine (PtaCafOMT and PraCafOMT) (Li *et al.*, 1997), while PSOMT2 was most similar to (*R*,*S*)-norcoclaurine 6-*O*-methyltransferase of (*S*)-reticuline biosynthesis from *C. japonica* (*Cja6OMT*) (Morishige *et al.*, 2000). Two additional (*R*,*S*)-norcoclaurine 6-*O*-methyltransferases from *T. tuberosum* (*Ttu6OMT*) (Frick and Kutchan, 1999) are clearly more related to caffeic acid *O*-methyltransferases from a variety of plant species than to either PSOMT1 or PSOMT2. Highlighted in red are the *O*-methyltransferases involved in isoquinoline alkaloid biosynthesis. CbrlEMT, *Clarkia breweri*, AAC01533; CbrCafOMT, *C.* *breweri*, AAB71141; TtuCafOMT4, *T.* *tuberosum*, AAD29844; Ttu6OMT1, *T.* *tuberosum*, AAD29841; Ttu6OMT2, *T.* *tuberosum*, AAD29842; TtuCafOMT3, *T.* *tuberosum*, AAD29843; TtuOMT5, *T.* *tuberosum*, AAD29845; PsoCOMT, *P.* *somniferum*, AY268895; Geclli2OMT, *Glycyrrhiza echinata*, BAA13683; MsatlliOMT, *Medicago sativa*, AAB48059; PtaCafOMT, *P.* *taeda*, AAC49708; PraCafOMT, *P.* *radiata*, AAD24001; PSOMT1, *P.* *somniferum*, AY268893; PSOMT2, *P.* *somniferum*, AY268894; Cja6OMT, *C.* *japonica*, BAB08004; Cja4'OMT, *C.* *japonica*, BAB08005; TaeOMT, *Triticum aestivum*, AAD10485; ZmaOMT, *Zea mays*, P47917; HvuF7OMT, *Hordeum vulgare*, S52015; RhyOrcOMT, *Rosa hybrida*, AAM23004; RhyOrcOMT2, *R.* *hybrida*, AAM23005; PduOMT, *P.* *dulcis*, CAA11131; ParOMT, *P.* *armeniaca*, AAB71213; PpyOMT, *P.* *pyrifolia*, BAA86059; ObaCVOMT, *Ocimum basilicum*, AF435007; ObaEOMT, *O.* *basilicum*, AF435008; Msa7IOMT, *M.* *sativa*, T09254; MsaOMT, *M.* *sativa*, T09299; PsaHMOMT, *Pisum sativum*, T06786; AthCOMT, *Arabidopsis thaliana*, T04963; CroCafOMT, *C.* *roseus*, AAK20170; ObaCafOMT, *O.* *basilicum*, AAD38189; ZelCafOMT, *Zinnia elegans*, AAA86718; NtaCOMT, *Nicotiana tabacum*, S36403; CanCOMT, *Capsicum annuum*, T12259; PtoCafOMT, *Populus tomentosa*, AAF63200; PtrCafOMT, *P.* *tremuloides*, Q00763; PbaCafOMT, *P.* *balsamifera*, CAA01820; PduCafOMT, *P.* *dulcis*, CAA58218; MsaCafOMT, *M.* *sativa*, AAB46623; AthCafOMT, *A.* *thaliana*, AAB96879; CamCafOMT, *Chrysosplenium americanum*, AAA86982; EglCafOMT, *Eucalyptus globules*, AAD50440; Egu-CafOMT, *Eucalyptus gunnii*, CAA52814.

residues elongating the amino terminus. The proteins were then purified from *Spodoptera frugiperda* Sf9 cell culture medium in one step by cobalt affinity chromatography to yield electrophoretically homogeneous proteins. PSOMT1 and PSOMT2 each have relative molecular masses of 43 kDa, as determined by SDS-PAGE. This compares with the calculated molecular masses of 39 841 and 38 510 based on the translation of the nucleotide sequences. The native relative molecular masses were determined by gel filtration on a calibrated Sephacryl 200 column

(Amersham Biosciences, Freiburg, Germany). PSOMT1 and PSOMT2 are each homodimers with an *M*<sub>r</sub> of 85 and 80 kDa, respectively. This is consistent with that observed for norcoclaurine 6-*O*-methyltransferases of (*S*)-reticuline biosynthesis in *T.* *tuberosum* (Frick and Kutchan, 1999).

Radioassay of pure, recombinant *O*-methyltransferases using [methyl-<sup>3</sup>H]-AdoMet together with each of 40 different substrates demonstrated that PSOMT1 and PSOMT2 are relatively substrate specific (Table 2). PSOMT1 methylates the simple catechols guaiacol and isovanillic acid as

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), 36, 808–819

Opium poppy benzylisoquinoline 6-OMT and 7-OMT 813

Table 1 Percent amino acid sequence identity of isoquinoline alkaloid biosynthetic O-methyltransferases from *P. somniferum*, *T. tuberosum*, and *C. japonica*

| PSOMT1 | Ttu6OMT1 | Ttu6OMT2 | Cja6OMT | Cja4′OMT |
|--------|----------|----------|---------|----------|
| PSOMT2 | 36       | 29       | 63      | 52       |
| PSOMT1 |          | 32       | 36      | 32       |
| Ttu6OMT1 |         |          | 94      | 31       |
| Ttu6OMT2 |         |          | 30      | 32       |
| Cja6OMT |          |          |         | 50       |

PSOMT1, *P. somniferum* (*R,S*)-reticuline 7-O-methyltransferase; PSOMT2, *P. somniferum* (*R,S*)-norcoclaurine 6-O-methyltransferase; Ttu6OMT1, *T. tuberosum* (*R,S*)-norcoclaurine 6-O-methyltransferase 1; Ttu6OMT2, *T. tuberosum* (*R,S*)-norcoclaurine 6-O-methyltransferase 2; Cja6OMT, *C. japonica* (*R,S*)-norcoclaurine 6-O-methyltransferase; Cja4′OMT, *C. japonica* (*S*)-3′hydroxy-*N*-methylcoclaurine 4′-O-methyltransferase.

pH 7.5. The temperature optima for PSOMT1 with various substrates are: guaiacol, (*R*)-reticuline, and (*S*)-reticuline (37°C); (*R*)-protosinomenine (39°C); and (*R,S*)-isoorientaline and isovanillic acid (37–41°C). PSOMT2 optimally methylated all substrates at 37–41°C.

The kinetic parameters determined for methylation of each substrate of PSOMT1 and PSOMT2 are shown in Table 3. As designated by the ratio $k_{cat}/K_m$, PSOMT1 methylates (*R*)-reticuline and (*S*)-reticuline with equal efficiency. Both substrates occur in *P. somniferum*, but only (*R*)-reticuline is specific to morphine biosynthesis. The high $k_{cat}/K_m$ ratio for guaiacol (135% of those values determined for reticuline) does not correlate with *in vivo* significance, as this simple catechol has not been reported to occur in *P. somniferum*. Likewise, (*R*)-protosinomenine, (*R,S*)-isoorientaline and isovanillic acid do not occur in this plant. The highest $k_{cat}/K_m$ ratio for PSOMT2 was obtained with (*R,S*)-norcoclaurine as substrate. The next best substrates are (*R*)-and (*S*)-norprotosinomenine with values equal to 55% of that obtained for norcoclaurine. However, norprotosinomenines do not naturally occur in *P. somniferum*.

Structure elucidation of enzymic products

Initial enzyme activity measurements were carried out using a radioassay. Many of the substrates tested contained more than one site of potential methylation. As the radioassay is only a facile measure of whether methylation had likely occurred, but does not indicate the position of methyl transfer, each positive assay was repeated with unlabeled substrate and the enzymic product was subjected to HPLC-MS analysis. Tetrahydrobenzylisoquinolines readily cleave at low ionization energies into the corresponding isoquinoline and benzyl ions. This enables identification of methylation at either moiety. The structures of the 10 substrates that were methylated by either PSOMT1 or PSOMT2 are shown in Figure 6. Each alkaloidal substrate was monitored for purity by HLPC-MS, and the fragmentation pattern was

Figure 5. RNA gel blot analysis of PSOMT1 and PSOMT2.

Top panel: PSOMT1 is expressed predominantly in bud and stem and, to a much lesser degree, in leaf of *P. somniferum*. Middle panel: PSOMT2 is expressed in bud, stem, leaf, and root and, to a lesser degree, in capsule. These results were obtained after blotting a *P. somniferum* RNA gel and hybridizing to ${}^{32}$P-labeled full-length PSOMT1 or PSOMT2. Radioactivity was visualized by phosphorimaging. The bottom panel is a photograph of ethidium bromide-visualized RNA in the gel prior to blotting. This served as an RNA loading control.

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), 36, 808–819

Table 2 Substrate specificities of PSOMT1 and PSOMT2

| Substrate                     | PSOMT1 | PSOMT2 |
|-------------------------------|--------|--------|
| Phenolics                     |        |        |
| 1 Catechol                    | 0      | 0      |
| 2 Protocatechuic acid         | 0      | 0      |
| 3 Dopamine                    | 0      | 0      |
| 4 Caffeic acid                | 0      | 0      |
| 5 Guaiacol                    | 242    | 0      |
| 6 Isovanillic acid            | 40     | 0      |
| 7 Vanillic acid               | 0      | 0      |
| Isoquinoline alkaloids        |        |        |
| 8 (R,S)-Norcoclaurine         | 0      | 100<sup>a</sup> |
| 9 (S)-Coclaurine              | 0      | 0      |
| 10 (R,S)-Isococlaurine        | 0      | 0      |
| 11 (R,S)-4'-O-methylcoclaurine | 0      | 0      |
| 12 (R,S)-Nororientaline       | 0      | 0      |
| 13 (R)-Norprotosinomenine     | 0      | 26     |
| 14 (S)-Norprotosinomenine     | 0      | 26     |
| 15 (R)-Norreticuline          | 0      | 0      |
| 16 (S)-Norreticuline          | 0      | 0      |
| 17 (R)-7-Dehydroxy-norreticuline | 0      | 0      |
| 18 (S)-7-Dehydroxy-norreticuline | 0      | 0      |
| 19 (R,S)-N-Methylcoclaurine   | 0      | 0      |
| 20 (R,S)-6-O-Methylaudanosoline | 0      | 0      |
| 21 (S)-4'-O-Methylaudanosoline | 0      | 0      |
| 22 (R)-Reticuline             | 100<sup>a</sup> | 0      |
| 23 (S)-Reticuline             | 100<sup>a</sup> | 0      |
| 24 (R,S)-Orientaline          | 48     | 0      |
| 25 (R)-Protosinomenine        | 52     | 0      |
| 26 (R,S)-Isoorientaline       | 46     | 47     |
| 27 (R,S)-Laudanidine          | 0      | 0      |
| 28 (R,S)-Codamine             | 0      | 0      |
| 29 (S)-Scoulerine             | 0      | 0      |
| 30 (S)-Coreximine             | 0      | 0      |
| 31 Salutaridine               | 0      | 0      |
| 32 Codeine                    | 0      | 0      |
| 33 Morphine                   | 0      | 0      |
| Flavonoids                    |        |        |
| 34 Quercetin                  | 0      | 0      |
| 35 Quercetin-3-methylether    | 0      | 0      |
| 36 Quercetin-7-methylether    | 0      | 0      |
| 37 Luteolin                   | 0      | 0      |
| 38 Morin                      | 0      | 0      |
| 39 Cyanidin                   | 0      | 0      |
| Coumarin                     |        |        |
| 40 Esculetin                  | 0      | 0      |

<sup>a</sup> 100% Activity of PSOMT1 and PSOMT2 is 1.5 and 2.0 pmol sec<sup>-1</sup> mg<sup>-1</sup> total protein, respectively. Assay conditions are given in the Experimental procedures.

206 (isoquinoline + CH<sub>2</sub>) and m/z 137 (unmodified benzyl). Likewise, (R,S)-norcoclaurine ([M+H]<sup>+</sup> m/z 272) has the major fragment ions m/z 161 (isoquinoline) and m/z 107 (benzyl). The methylation of (R,S)-norcoclaurine by PSOMT2 results in a product of [M+H]<sup>+</sup> m/z 286 (methylated (R,S)-norcoclaurine) with fragment ions at m/z 175 (isoquinoline + CH<sub>2</sub>) and m/z 107 (unmodified benzyl).

Surprising results were obtained when the PSOMT1 methylation products of (R,S)-orientaline and (R,S)-iso-orientaline were analyzed by HPLC-MS. The fragment ions obtained for the methylation products of orientaline are shown in Figure 7. Methylation of the 7-hydroxyl group resulted in the main enzymic product 7-O-methylorientaline. Approximately 1% of the product produced is the double methylated 7,4'-O-dimethylorientaline (laudanosine) and the monomethylated 4'-O-methylorientaline.

Discussion

The identification of a new O-methyltransferase presented herein is the result of a first attempt to use proteome analysis to identify latex proteins in *P. somniferum* (Decker et al., 2000). Latex collected from capsules was resolved into a cytosolic and a vesicular fraction by centrifugation, and the cytosolic proteins were then resolved by two-dimensional polyacrylamide gel electrophoresis. From internal amino acid sequence determination of these gel spots, one set of peptides showed homology to plant O-methyltransferases. Using RT-PCR followed by RACE-PCR, two cDNAs PSOMT1 and PSOMT2 encoding complete open-reading frames were isolated. One of these cDNAs, PSOMT1, encoded the sequenced protein.

A sequence comparison of the translations of PSOMT1 and PSOMT2 with those sequences available in the GenBank/European Molecular Biology Laboratory (EMBL) databases revealed that PSOMT1 showed the highest amino acid sequence relationship to a caffeic acid/caffeoyl CoA 3-O-methyltransferase from loblolly pine *P. taeda* (Li et al., 1997) and to a putative caffeic acid 3-O-methyltransferase from Monterey pine *P. radiata*. PSOMT2 was likely functionally equivalent to (R,S)-norcoclaurine 6-O-methyltransferase from *C. japonica* (Morishige et al., 2000). Using amino acid sequence comparison to predict the in vivo function of plant O-methyltransferases is not trivial because of the broad substrate specificities that can be found for closely related enzymes (Frick and Kutchan, 1999). To overcome the uncertainties associated with phylogenetic comparison, PSOMT1 and PSOMT2 were each introduced into a baculovirus expression vector and the corresponding proteins PSOMT1 and PSOMT2 were produced in S. frugiperda Sf9 cell culture. Forty compounds were tested as potential substrates for the two enzymes. Most of these substances were tetrahydrobenzylisoquinoline alkaloids, but simple catechols and a few

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), 36, 808–819

Table 3 Kinetic parameters of PSOMT1 and PSOMT2 for various substrates and co-substrate (AdoMet)

| Enzyme | Substrate                | $K_{\mathrm{m}}$ AdoMet ($\mu$M) | $K_{\mathrm{m}}$ Substrate ($\mu$M) | $V_{\max }$ Substrate (pmol sec$^{-1}$) | $k_{\mathrm{cat}}$ Substrate (sec$^{-1}$) | $k_{\mathrm{cat}} / K_{\mathrm{m}}$ Substrate (sec$^{-1}$ mM$^{-1}$) |
|--------|--------------------------|---------------------------------|------------------------------------|-----------------------------------------|-----------------------------------------|------------------------------------------------------------|
| PSOMT1 | Guaiacol                 | 310                             | 17                                 | 6                                       | 0.1                                     | 5.9                                                          |
|        | $(S)$-Reticuline         | 360                             | 16                                 | 4                                       | 0.07                                    | 4.5                                                          |
|        | $(R)$-Reticuline         | 310                             | 17                                 | 4                                       | 0.07                                    | 4.2                                                          |
|        | $(R)$-Protosinomenine    | 320                             | 16                                 | 2                                       | 0.03                                    | 1.7                                                          |
|        | $(R, S)$-Isoorientaline  | 260                             | 17                                 | 1                                       | 0.02                                    | 1.4                                                          |
|        | Isovanillic acid         | 150                             | 14                                 | 1                                       | 0.02                                    | 1.2                                                          |
| PSOMT2 | $(R, S)$-Norcoclaurine   | 100                             | 10                                 | 5                                       | 0.08                                    | 7.4                                                          |
|        | $(R)$-Norprotosinomenine | 200                             | 5                                  | 1                                       | 0.02                                    | 4.1                                                          |
|        | $(S)$-Norprotosinomenine | 260                             | 5                                  | 1                                       | 0.02                                    | 4.0                                                          |
|        | $(R, S)$-Isoorientaline  | 280                             | 29                                 | 2                                       | 0.03                                    | 1.0                                                          |

Assay conditions are given in the Experimental procedures.

common phenylpropanoid-derived compounds were also included: PSOMT1: O-methylated guaiacol, isovanillic acid, $(R)$-reticuline, $(S)$-reticuline, $(R, S)$-orientaline, $(R)$-protosinomenine, and $(R, S)$-isoorientaline; PSOMT2: O-methylated $(R, S)$-norcoclaurine, $(R)$-norprotosinomenine, $(S)$-norprotosinomenine, and $(R, S)$-isoorientaline.

The broad substrate specificities of plant O-methyltransferases can make the assignment of an in vivo role to these enzymes quite challenging. A comparison of the $k_{\mathrm{cat}} / K_{\mathrm{m}}$ ratio for the various substrates suggested that the in vivo substrates for PSOMT1 are likely $(R)$-reticuline and $(S)$-reticuline. Guaiacol demonstrated the highest $k_{\mathrm{cat}} / K_{\mathrm{m}}$ ratio, but this catechol has not been reported to accumulate in

Figure 6. Chemical structures of the substrates methylated by either PSOMT1 or PSOMT2.

Guaiacol  
![Guaiacol Structure](https://i.imgur.com/structure1.png)

Isovanillic acid  
![Isovanillic Acid Structure](https://i.imgur.com/structure2.png)

Substrate  
![Substrate Structure](https://i.imgur.com/structure3.png)

| Substrate                     | $R_1$ | $R_2$ | $R_3$ | $R_4$ | $R_5$ |
|-------------------------------|-------|-------|-------|-------|-------|
| $(R, S)$-Norcoclaurine        | H     | OH    | OH    | H     | OH    |
| $(R)$-Norprotosinomenine      | H     | OH    | OCH₃  | OH    | OCH₃  |
| $(S)$-Norprotosinomenine      | H     | OH    | OCH₃  | OH    | OCH₃  |
| $(R)$-Reticuline              | CH₃   | OCH₃  | OH    | OH    | OCH₃  |
| $(S)$-Reticuline              | CH₃   | OCH₃  | OH    | OH    | OCH₃  |
| $(R, S)$-Orientaline          | CH₃   | OCH₃  | OH    | OCH₃  | OH    |
| $(R)$-Protosinomenine         | CH₃   | OH    | OCH₃  | OH    | OCH₃  |
| $(R, S)$-Isoorientaline       | CH₃   | OH    | OCH₃  | OCH₃  | OH    |

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), 36, 808–819

The methylating capacity of PSOMT1 was more promiscuous than that of PSOMT2. PSOMT1 O-methylation of guaiacol, isovanillic acid, (R)-reticuline (S)-reticuline, (R,S)-orientaline, (R)-protosinomenine, and (R,S)-isoorientaline resulted in a more complicated product profile. HPLC-MS analysis indicated that (R)-reticuline, (S)-reticuline, and (R,S)-orientaline, each of which has a C-6 methoxy group and a C-7 hydroxy moiety, were O-methylated at C-7. In contrast, (R)-protosinomenine and (R,S)-isoorientaline each has a free hydroxyl group at C-6 and is methoxylated at C-7. These molecules were O-methylated by PSOMT1 at C-6. The ratio of $k_{cat}/K_m$ for C-7 O-methylation compared to C-6 O-methylation was 3.8 : 1, suggesting that C-7 O-methylation is preferred. Multiple products were detected when either (R,S)-orientaline or (R,S)-isoorientaline was used as substrate. In addition to methylation of the isoquinoline half of the tetrahydrobenzylisoquinolines, the benzyl moiety was also methylated. (R,S)-orientaline and (R,S)-isoorientaline differ from the other tetrahydrobenzylisoquinoline substrates in that the benzyl ring is 3′-methoxylated and 4′-hydroxylated. Reticuline and the protosinomenines are 4′-methoxylated and 3′-hydroxylated. The free 4′-hydroxy group of (R,S)-orientaline and (R,S)-isoorientaline is methylated by PSOMT1. 4′-O-methylation appears to occur independent of both hydroxyl groups of the isoquinoline nucleus being methylated, as three products can be identified by HPLC-MS, representing monomethylation at the isoquinoline moiety, monomethylation at the benzyl moiety, and double methylation.

ready fragmentation of these types of molecules into two halves: an isoquinoline moiety and a benzyl moiety. Methylation of either portion of the molecule can be readily identified: PSOMT2: O-methylated (R,S)-norcoclaurine, (R)-norprotosinomenine, (S)-norprotosinomenine, and (R,S)-isoorientaline on the isoquinoline moiety. In the case of (R,S)-norcoclaurine, both C-6 and C-7 are hydroxylated. (R)-norprotosinomenine, (S)-norprotosinomenine, and (R,S)-isoorientaline all have a free hydroxyl group at C-6, but C-7 is methoxylated. This indicates that the position of O-methylation of these molecules is at C-6. Dimethylation of (R,S)-norcoclaurine would have been readily detected by mass spectroscopic analysis, but was not observed. (Based upon the phylogenetic analysis and the structures of the methylated alkaloidal products, it can be concluded that PSOMT2 encodes the tetrahydroisoquinoline biosynthetic enzyme (R,S)-norcoclaurine 6-O-methyltransferase.) In *P. somniferum*, this enzyme participates in the early steps of (S)-reticuline biosynthesis, which intermediate leads to numerous alkaloids of the morphinan, benzo[c]-phenanthridine, papaverine, and phthalideisoquinoline types that are accumulated in this plant. The distribution of PSOMT2 transcript in bud, stem, leaf, root, and capsule is consistent with this role because these are all major sites of accumulation of one or the other of these alkaloid classes (i.e. morphinans in latex and benzo[c]phenanthridines in root).

The main enzymatic reaction product formed by PSOMT1 (approximately 99%) results from monomethylation of the isoquinoline group. Based upon these combined kinetic and mass spectroscopic results, it is concluded that PSOMT1 encodes (R,S)-reticuline 7-O-methyltransferase, a new enzyme of tetrahydrobenzylisoquinoline alkaloid biosynthesis in *P. somniferum*. The product of this reaction, 7-O-methylreticuline (laudanine), is a natural product that has been reported to occur in opium (Small and Lutz, 1932), and this occurrence has been confirmed for the variety of *P. somniferum* used herein (A. J. Fist, personal communication). The distribution of PSOMT1 transcript predominantly in bud and stem correlates with latex as the site of laudanine accumulation. Early literature reports of the alkaloid methylating capacity of *P. somniferum* latex (Antoun and Roberts, 1975b) are also consistent with the results presented herein, which suggest that at least

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), **36**, 808–819
part of opium poppy alkaloid biosynthesis can occur in laticifers.

Enzymatic O-methylation of tetrahydrobenzylisoquinolines has been reported to be catalyzed by catechol O-methyltransferase (COMT) isolated from rat liver as part of a program investigating the nature and biosynthetic origin of mammalian alkaloids (Sekine et al., 1990). In that particular report, COMT O-methylated norcoclaurine at the 6-hydroxy- and 7-hydroxy positions in a ratio of 8 : 2. This low specificity compares to that of norcoclaurine 6-O-methyltransferase characterized from *T. tuberosum*, which methylated tetrahydrobenzylisoquinolines that contained a catechol and, to a lesser degree, a guaiacol moiety (Frick and Kutchan, 1999). The *P. somniferum* 7-O- and 6-O-methyltransferases characterized herein appear to methylate with higher regiospecificity. From an evolutionary point of view, the *T. tuberosum* O-methyltransferases suggest that the alkaloid biosynthetic enzyme norcoclaurine 6-O-methyltransferase arose from caffeic acid 3-O-methyltransferase from the phenylpropanoid pathway, as two of four *T. tuberosum* caffeic acid 3-O-methyltransferases could also O-methylate norcoclaurine. These particular enzymes group very closely upon phylogenetic comparison to caffeic acid 3-O-methyltransferases from many different species. The *P. somniferum* norcoclaurine 6-O-methyltransferase however groups more tightly to two other reticuline biosynthetic enzymes from *C. japonica*, all of which do not methylate caffeic acid. This type of specialization may suggest that *P. somniferum* and *C. japonica* norcoclaurine 6-O-methyltransferases have evolved further than those from *T. tuberosum* and have lost the ability to participate in phenylpropanoid biosynthesis. The evolutionary argument that part of alkaloid biosynthesis has been derived from phenylpropanoid biosynthesis should be further advanced by amino acid sequence comparison of the *P. somniferum* and *C. japonica* caffeic acid 3-O-methyltransferases to the corresponding norcoclaurine 6-O-methyltransferases.

### Experimental procedures

#### Plant material

Papaver somniferum seedlings were routinely grown aseptically on Gamborg B5 medium (Gamborg et al., 1968) containing 0.8% agar in a growth chamber at 22°C and 60% relative humidity under cycles of 16-h light/8-h dark with a light intensity of 85 μmol m⁻² sec⁻¹ μA⁻¹. Differentiated *P. somniferum* plants were grown either outdoors in Saxony-Anhalt or in a greenhouse at 24°C, 18-h light and 50% humidity.

#### Generation of partial cDNAs from P. somniferum

Partial cDNAs encoding O-methyltransferases from *P. somniferum* were produced by PCR using cDNA generated by reverse transcription of mRNA isolated from floral stem. DNA amplification using either thermus aquaticus (Taq) or pyrococcus furiosus (Pfu) polymerase was performed under the following conditions: 3 min at 94°C; 35 cycles of 94°C for 30 sec, 50°C for 30 sec, and 72°C for 1 min. At the end of 35 cycles, the reaction mixtures were incubated for an additional 7 min at 72°C prior to cooling to 4°C. The amplified DNA was resolved by agarose gel electrophoresis, and the bands of approximately correct size (400 bp) were isolated and subcloned into pGEM-T Easy (Promega, Mannheim, Germany) prior to nucleotide sequence determination. The specific sequences of the oligodeoxynucleotide primers used were: OMT-Pep 5 sense primer, 5′-GCI GGI A/T C/G I C/T TI GTI GAC/T GTI GGI GG-3′; OMT-Pep 1 antisense primer, 5′-C/T TC IGC C/T TC IGT ICG/T C/T TC CTT-3′.

#### Generation of full-length cDNAs

The sequence information requisite to the generation of a full-length cDNA was derived from the nucleotide sequence of the partial cDNA produced as described in the Results section. The complete nucleotide sequence was generated in two steps using one O-methyltransferase-specific PCR primer (*PSOMT1*: 5′-AGT CAT TTC CAT CTG GTC GCA ACA-3′ for 5′-RACE and 5′-ATG GAT ACT GCA GAA GAA AGG TTG-3′ for 3′-RACE; *PSOMT2*: 5′-ATA AGG GTA AGC CTC AAT TAC AGA TTG-3′ for 5′-RACE and 5′-GCT GCA GTG AAA GCC ATA ATC T-3′ for 3′-RACE) and one RACE-specific primer as specified by the manufacturer. The 5′- and 3′-RACE-PCR experiments were carried out using a SMART™ cDNA amplification kit (Clontech, Heidelberg, Germany). RACE-PCR was performed using the following PCR cycle: 3 min at 94°C; 25 cycles of 94°C for 30 sec, 68°C for 30 sec, and 72°C for 3 min. At the end of 25 cycles, the reaction mixtures were incubated for an additional 7 min at 72°C prior to cooling to 4°C. The amplified DNA was resolved by agarose gel electrophoresis, and the bands of the expected size (*PSOMT1*: 990 bp for 5′-RACE and 1177 bp for 3′-RACE; *PSOMT2*: 1124 bp for 5′-RACE and 671 bp for 3′-RACE) were isolated and subcloned into pGEM-T Easy prior to sequencing.

The full-length clone was generated in one piece using the primers *PSOMT1*: 5′-TAT CGG ATC CAT GGA TAC TGC AGA A-3′ and 5′-TTA GGC GGC CGC TTA TTC TGG AAA GGC-3′ or *PSOMT2*: 5′-TAT CGG ATC CAT GGA AAC AGT AAG C-3′ and 5′-TTA GGC GGC CGC TTA ATA AGG GTA AGC-3′ for PCR with *P. somniferum* floral stem cDNA as template. The final primers used for cDNA amplification contained recognition sites for the restriction endonucleases *BamH*I and *Not*I, appropriate for subcloning into pFastBac HTa (Life Technologies, Karlsruhe, Germany) for functional expression. DNA amplification was performed under the following conditions: 3 min at 94°C; 35 cycles of 94°C for 30 sec, 60°C for 30 sec, and 72°C for 2 min. At the end of 35 cycles, the reaction mixtures were incubated for an additional 7 min at 72°C prior to cooling to 4°C. The amplified DNA was resolved by agarose gel electrophoresis, and the band of approximately correct size (*PSOMT1*: 1068 bp; *PSOMT2*: 1041 bp) was isolated and subcloned into pCR4-TOPO (Invitrogen, Karlsruhe, Germany) prior to nucleotide sequence determination.

#### Heterologous expression and enzyme purification

The full-length cDNA generated by RT-PCR was ligated into pFastBac HTa that had been digested with restriction endonucleases *BamH*I and *Not*I. The recombinant plasmid was transposed into baculovirus DNA in the *Escherichia coli* strain DH10BAC (Life Technologies) and then transfected into *S. frugiperda* Sf9 cells according to the manufacturer’s instructions. The insect cells were

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), **36**, 808–819
propagated, and the recombinant virus was amplified according to Kutchan *et al.* (1994) and Pauli and Kutchan (1998). INSECT-XPRESS serum-free medium (Bio Whittaker, Verviers, Belgium) was used in the enzyme expression experiments.

After infection of 20-ml-suspension-grown insect cells had proceeded for 3–4 days at 28°C and 130 r.p.m., the cells were removed by centrifugation under sterile conditions at 900 g for 5 min at 4°C. All subsequent steps were performed at 4°C. The pellet was discarded, and the following were added to the medium: 0.73 g NaCl, 2.5 ml of glycerol, and 50 μl of β-mercaptoethanol. The pH was adjusted to 7.0 with 1.0 M NaOH. The His-tagged O-methyltransferase was then purified by affinity chromatography using a cobalt resin (Talon, Clontech, Heidelberg, Germany) according to the manufacturer’s instructions.

### Enzyme assay and product identification

The O-methylation reactions catalyzed by the two O-methyltransferases were assayed at least two times in duplicate according to Rüffer *et al.* (1983a,b) as follows. Substrate (25 nmol), [methyl-³H]-AdoMet (20 000 d.p.m., 0.4 fmol), AdoMet (10 nmol), Tris-HCl buffer, pH 8.0 (10 μmol), ascorbate (5 μmol), and 5–10 μg of enzyme were incubated in a total volume of 150 μl at 35°C for 5–60 min. The enzymatic reaction was terminated by addition of 200 μl of 1.0 M NaHCO₃ buffer, pH 9.5. This mixture was then extracted with 400 μl of ethylacetate. The organic phase (300 μl) was added to 3 ml of high-flash point liquid scintillation cocktail (Packard, Frankfurt, Germany), and the radioactivity was quantified with a Beckman LS6000TA liquid scintillation counter. For *K*~m~ determinations, substrate concentration was varied from 0 to 400 μM.

The identity of the enzymic reaction products was ascertained by HPLC-MS using a Finnigan MAT TSQ 7000 (electrospray voltage, 4.5 kV; capillary temperature, 220°C; carrier gas, N₂) coupled to a Micro-Tech Ultra-Plus Micro-LC equipped with an Ultrasep RP18 column; 5 μm; 1 × 10 mm (solvent system (A): 99.8% (v/v) H₂O, 0.2% HOAc; (B): 99.8% CH₃CN (v/v), 0.2% HOAc; gradient: 0–15 min, 10–90% B; 15–25 min, 90% B; flow: 70 μl min⁻¹). The collision-induced dissociation (CID; collision energy, –25 eV; collision gas, argon; collision pressure, 1.8 × 10⁻³ Torr) mass spectra for the tetrahydrobenzylisoquinoline alkaloids were recorded.

### General methods

Total RNA was isolated, and RNA gels were run and blotted as described previously by Pauli and Kutchan (1998). Genomic DNA was isolated, and DNA gels were run and blotted according to Bracher and Kutchan (1992). cDNA clones were labeled by PCR labeling with [α-³²P]dATP. Hybridized RNA on RNA gel blots and DNA on DNA gel blots were visualized with a STORM phosphor imager (Amersham Biosciences). The entire nucleotide sequence on both DNA strands of the full-length clone was determined by dideoxy cycle sequencing using internal DNA sequences for the design of deoxyoligonucleotides as sequencing primers. Saturation curves and double reciprocal plots were constructed with the FIG. P program Version 2.7 (Biosoft, Cambridge, UK). The influence of pH on enzyme activity was monitored in sodium citrate (pH 4–6), sodium phosphate (pH 6–7.0), Tris-HCl (pH 7.0–9), and glycine/NaOH (pH 9–10.5) buffered solutions.

### Acknowledgements

We thank Dr Jörg Ziegler for the phylogenetic comparison. This work was supported in part by the Deutschen Akademischen

#### References

Antoun, M.D. and Roberts, M.F. (1975a) Some enzymes of general metabolism in the latex of *Papaver somniferum*. Phytochemistry, 14, 909–914.

Antoun, M.D. and Roberts, M.F. (1975b) Enzymic studies with *Papaver somniferum*. Part 5. The occurrence of methyltransferase enzymes in poppy latex. Planta Med. 28, 6–11.

Bracher, D. and Kutchan, T.M. (1992) Strictosidine synthase from *Rauvolfia serpentina*. Analysis of a gene involved in indole alkaloid biosynthesis. Arch. Biochem. Biophys. 294, 717–723.

Cacace, S., Schröder, G., Wehinger, E., Strack, D., Schmidt, J. and Schröder, J. (2003) A flavonol O-methyltransferase from *Catharanthus roseus* performing two sequential methylations. Phytochemistry, 62, 127–137.

Decker, G., Wanner, G., Zenk, M.H. and Lottspeich, F. (2000) Characterization of the proteins in latex of the opium poppy (*Papaver somniferum*) using two-dimensional gel electrophoresis and microsequencing. Electrophoresis, 21, 3500–3516.

Dittrich, H. and Kutchan, T.M. (1991) Molecular cloning, expression and induction of berberine bridge enzyme, an enzyme essential to the formation of benzophenanthridine alkaloids in the response of plants to pathogenic attack. Proc. Natl. Acad. Sci. USA, 88, 9969–9973.

Facchini, P.J., Penzes, C., Johnson, A.G. and Bull. D. (1996) Molecular characterization of berberine bridge enzyme genes from opium poppy. Plant Physiol. 112, 1669–1677.

Frick, S. and Kutchan, T.M. (1999) Molecular cloning and functional expression of O-methyltransferases common to isoquinoline alkaloid and phenylpropanoid biosynthesis. Plant J. 17, 329–339.

Gamborg, O.L., Miller, R.A. and Ojina, K. (1968) Nutrient requirements of suspension cultures of soybean root cells. Exp. Cell Res. 50, 151–158.

Gang, D.R., Lavid, N., Zubieta, C., Chen, F., Beuerle, T., Lewinsohn, E., Noel, J.P. and Pichersky, E. (2002) Characterization of phenylpropene O-methyltransferases from sweet basil: facile change of a substrate specificity and convergent evolution within a plant O-methyltransferase family. Plant Cell, 14, 505–519.

Ghosal, S., Majumdar, S.K. and Chakraborti, A. (1971) Erythrina alkaloids. Part III. Occurrence of N-norprotosinomenine and other alkaloids in *Erythrina lithosperma* (Leguminosae). Aust. J. Chem. 24, 2733–2735.

Grothe, T., Lenz, R. and Kutchan, T.M. (2001) Molecular characterization of the salutaridinol 7-O-acetyltransferase involved in morphine biosynthesis in opium poppy *Papaver somniferum*. J. Biol. Chem. 276, 30717–30723.

Huang, F.-C. and Kutchan, T.M. (2000) Distribution of morphinan and benzophenanthridine alkaloid gene transcript accumulation in the opium poppy *Papaver somniferum*. Phytochemistry, 53, 555–564.

Ibrahim, R.K., Bruneau, A. and Bantignies, B. (1998) Plant O-methyltransferases: molecular analyses, common signature and classification. Plant Mol. Biol. 36, 1–10.

Joshi, C.P. and Chiang, V.L. (1998) Conserved sequence motifs in plant S-adenosyl-L-methionine-dependent methyltransferases. Plant Mol. Biol. 37, 663–674.

Kutchan, T.M. (1998) Molecular genetics of plant alkaloid biosynthesis. In *The Alkaloids*, Vol. 50 (Cordell, G., ed.). San Diego: Academic Press, pp. 257–316.

Kutchan, T.M., Bock, A. and Dittrich, H. (1994) Heterologous expression of strictosidine synthase and berberine bridge enzyme in insect cell culture. Phytochemistry, 35, 353–360.

Lavid, N., Wang, J., Shalit, M. *et al.* (2002) O-Methyltransferases involved in the biosynthesis of volatile phenolic derivatives in rose petals. Plant Physiol. 129, 1899–1907.

Li, L., Popko, J.L., Zhang, X.-H., Osakabe, K., Tsai, C.J., Joshi, C.P. and Chiang, V.L. (1997) A novel multifunctional O-methyltransferase implicated in a dual methylation pathway associated with lignin biosynthesis in loblolly pine. Proc. Natl. Acad. Sci. USA, 94, 5461–5466.

Maury, S., Geoffroy, P. and Legrand, M. (1999) Tobacco O-methyltransferases involved in phenylpropanoid metabolism. The different caffeoyl-coenzyme A/5-hydroxyferuloyl-coenzyme A 3/5-O-methyltransferase and caffeic acid/5-hydroxyferulic acid 3/5-O-methyltransferase classes have distinct substrate specificities and expression patterns. Plant Physiol. 121, 215–223.

Morishige, T., Tsujita, T., Yamada, Y. and Sato, F. (2000) Molecular characterization of the S-adenosyl-L-methionine: 3′-hydroxy-N-methylcoclaurine 4′-O-methyltransferase involved in isoquinoline alkaloid biosynthesis in *Coptis japonica*. J. Biol. Chem. 275, 23398–23405.

Pauli, H.H. and Kutchan, T.M. (1998) Molecular cloning and functional heterologous expression of two alleles encoding (S)-N-methylcoclaurine 3′-hydroxylase (CYP80B1), a new methyl jasmonate-inducible cytochrome P-450 dependent monooxygenase of benzylisoquinoline alkaloid biosynthesis. Plant J. 13, 793–801.


Roberts, M.F., McCarthy, D., Kutchan, T.M. and Coscia, C.J. (1983) Localization of enzymes and alkaloidal metabolites in *Papaver* latex. Arch. Biochem. Biophys. 222, 599–609.

Rosco, A., Pauli, H.H., Priesner, W. and Kutchan, T.M. (1997) Cloning and heterologous expression of cytochrome P450 reductases from the Papaveraceae. Arch. Biochem. Biophys. 348, 369–377.

Rüffer, M., Nagakura, N. and Zenk, M.H. (1983a) Partial purification and properties of S-adenosylmethionine:(R,S)-norlaudanosoline 6-O-methyltransferase from *Argemone platyceras* cell cultures. Planta Med. 49, 131–137.

Rüffer, M., Nagakura, N. and Zenk, M.H. (1983b) A highly specific O-methyltransferase for nororientaline synthesis isolated from *Argemone platyceras* cell cultures. Planta Med. 49, 196–198.

Schröder, G., Wehinger, E. and Schröder, J. (2002) Predicting the substrates of cloned plant O-methyltransferases. Phytochemistry, 59, 1–8.

Sekine, Y., Creveling, C., Bell, M. and Brossi, A. (1990) Mammalian alkaloids: O-methylation of (±)-norcoclaurine-1-carboxylic acid and related isoquinolines including (S)- and (R)-norcoclaurine with ¹⁴C-labeled S-adenosyl-L-methionine in presence of mammalian catechol O-methyltransferase. Helv. Chim. Acta, 73, 426–432.

Small, L.F. and Lutz, R.E. (1932) Chemistry of the Opium Alkaloids. Supplement No. 103, Public Health Reports of the United States Treasury Department. Washington: Oxford University Press, p. 34.

Unterlinner, B., Lenz, R. and Kutchan, T.M. (1999) Molecular cloning and functional expression of codeinone reductase – the penultimate enzyme in morphine biosynthesis in the opium poppy *Papaver somniferum*. Plant J. 18, 465–475.


The nucleotide sequences reported in this paper have been submitted to the GenBank™/EBI Data Bank with Accession numbers AY268893 (PSOMT1) and AY268894 (PSOMT2).
