
Planta (2004) 218: 1026–1035  
DOI 10.1007/s00425-003-1184-3  

**ORIGINAL ARTICLE**

Susanne Textor · Stefan Bartram · Jürgen Kroymann  
Kimberly L. Falk · Alastair Hick · John A. Pickett  
Jonathan Gershenzon  

**Biosynthesis of methionine-derived glucosinolates in *Arabidopsis thaliana*: recombinant expression and characterization of methylthioalkylmalate synthase, the condensing enzyme of the chain-elongation cycle**

Received: 10 July 2003 / Accepted: 10 November 2003 / Published online: 22 January 2004  
© Springer-Verlag 2004  

**Abstract** The major class of glucosinolates in *Arabidopsis thaliana* (L.) Heynh. are biosynthesized from methionine involving a three-step chain-elongation cycle. Each passage through the cycle results in the net addition of a single methylene group, with up to six cycles of elongation occurring in *A. thaliana*. The first reaction of the cycle is catalyzed by a methylthioalkylmalate synthase (MAMS), which condenses a ω-methylthio-2-oxoalkanoic acid with acetyl-CoA. Here we have demonstrated that *MAM1*, one of two similar genes in the *A. thaliana* ecotype Columbia, encodes a MAMS catalyzing the condensing reactions of the first two elongation cycles but not those of further cycles. The Columbia ecotype is dominated by compounds that have undergone only two elongation cycles. The *A. thaliana* MAM1 protein exhibits basic sequence similarity to other previously described enzymes catalyzing the condensation of 2-oxo acids and acetyl-CoA, such as isopropylmalate synthase (EC 2.3.3.13), an enzyme of leucine biosynthesis, and homocitrate synthase (EC 2.3.3.14). It also shares similar properties with them, including the catalytic requirements for a divalent metal ion and an adenine nucleotide. However, the MAM1 protein does not show activity with the substrates of any of these other enzymes, and was chromatographically separable from isopropylmalate synthase in extracts of *A. thaliana*. Thus, MAM1 is exclusively an enzyme of secondary metabolism, distinct from primary metabolic enzymes catalyzing similar reactions.

**Keywords** Arabidopsis · Brassicaceae · Condensing enzyme · Glucosinolate biosynthesis · Isopropylmalate synthase · Methionine

**Abbreviations** IPMS Isopropylmalate synthase · MAM Methylthioalkylmalate · MAMS Methylthioalkylmalate synthase

---

**Introduction**

Glucosinolates are sulfur-containing plant secondary metabolites derived from various amino acids. They occur almost exclusively in members of the Brassicales (Halkier 1999; Fahey et al. 2001), an order that includes cabbage, broccoli, mustard, rape, and *Arabidopsis*. Glucosinolates have long been of interest for their influence on the taste and flavor of brassicaceous vegetables, the quality of rapeseed oil and the value of the rapeseed meal remaining after oil extraction for animal feed. Recently, certain glucosinolates have been identified as potent cancer-prevention agents (Hecht 2000), and have also been studied extensively for their role in the resistance of rape and other plants to insect and fungal pests (Rask et al. 2000). In order to manipulate glucosinolate levels in plants for increasing the anticancer potential of vegetables intended for human consumption and for improving crop resistance to pests, we have been studying the biosynthesis of these substances in the model species, *Arabidopsis thaliana*.

The majority of glucosinolates in *A. thaliana* and many other species of the Brassicaceae are formed from methionine that has been modified by the sequential addition of one to six additional methylene groups to its side chain (Hogge et al. 1988; Reichelt et al. 2002). Although nearly all of the enzymes and genes involved in the conversion of chain-elongated methionine derivatives into glucosinolates have been characterized (Wittstock and Halkier 2002), much less is known about

S. Textor · S. Bartram · J. Kroymann · K. L. Falk  
J. Gershenzon (✉)  
Max-Planck Institute for Chemical Ecology,  
Winzerlaer Straße 10, Beutenberg Campus,  
07745 Jena, Germany  
E-mail: gershenzon@ice.mpg.de  
Fax: +49-3641-571302  

A. Hick · J. A. Pickett  
Rothamsted Research, Harpenden, AL5 2JQ, UK
the chain-elongation sequence itself (Fig. 1). Early feeding experiments with radiolabeled amino acids and acetate (Chisholm and Wetter 1964; Lee and Serif 1968, 1970) suggested an outline for the pathway. This was confirmed by more detailed in vivo studies with stable-isotope-labeled tracers (Graser et al. 2000) and by isolation of a key intermediate (Chapple et al. 1988). First, methionine (1) is deaminated to the corresponding 2-oxo acid, 4-methylthio-2-oxobutanoic acid (2), which then condenses with acetyl-CoA to form the malate derivative, 2-(2′-methylthioethyl)malate (3) (Fig. 2a). Following isomerization and oxidation-decarboxylation, the product is 5-methylthio-2-oxopentanoic acid (5), a 2-oxo acid with an additional methylene group. This chain-elongated 2-oxo acid can undergo additional cycles of condensation, isomerization and oxidative decarboxylation resulting in further elongation. Up to six cycles of elongation are known to occur in *A. thaliana*. Or, 5 can be transaminated to form the corresponding amino acid, homomethionine (6), which then enters the main pathway of glucosinolate biosynthesis (Fig. 1). Homomethionine forms C₃ glucosinolates, so called because they contain three methylene groups in the side chain, two from the original methionine skeleton and one from chain elongation. Similar 2-oxo acid-based chain-elongation sequences involving condensation, isomerization and oxidative decarboxylation occur in the TCA cycle (oxaloacetate–citrate–isocitrate–2-oxoglutarate), in leucine biosynthesis [2-oxoisovalerate (13)–2-isopropylmalate (14)–3-isopropylmalate (15)–2-oxoisocaproate (16)] (Fig. 2b), as well as elsewhere in plant secondary metabolism, such as in the formation of the acyl groups of sucrose esters (Kroumova and Wagner 2003).

In an effort to understand what controls chain elongation in aliphatic glucosinolate biosynthesis, we

Methionine  
Chain elongation  
cycle  

Methionine homologues with 1–6  
additional methylene groups  

Cytochrome P450:  
CYP79 family  

Methionine aldoximes  

Cytochrome P450: CYP83  

C-S lyase  

Thiohydroximates  

Glucosyltransferase  

Aliphatic desulfoglucosinolates  

Sulfotransferase  

Aliphatic glucosinolates  

Fig. 1 The chain-elongation cycle is the first stage of aliphatic glucosinolate biosynthesis in *Arabidopsis thaliana*

focused on the initial step of the elongation cycle: the condensation of acetyl-CoA with a 2-oxo acid derived from methionine or a higher-order methionine homologue to form a methylthioalkylmalate (MAM) product. The reaction of 4-methylthio-2-oxobutanoic acid (2), the 2-oxo acid formed from methionine, with acetyl-CoA was first demonstrated in purified chloroplast extracts of *Eruca sativa* and several other species producing chain-elongated, methionine-derived glucosinolates, and shown to be a soluble, divalent metal ion-requiring activity (Falk et al. 2004).

To study the protein catalyzing this reaction in more detail, we sought to isolate the corresponding gene from the *A. thaliana* genome. Four different gene sequences were found with high similarity to sequences encoding the enzyme catalyzing the condensing reaction of chain elongation in leucine biosynthesis, 2-isopropylmalate synthase (Kroymann et al. 2001). Two of these sequences (At1g18500, At1g74040) had 60–70% amino acid similarity to other plant 2-isopropylmalate synthases and were proposed to participate in leucine biosynthesis in *A. thaliana*. However, the other two sequences (At5g23010, At5g23020) had less than 50% similarity to other plant 2-isopropylmalate synthases and were located on chromosome V at a locus known to control the length of glucosinolate side chains in *A. thaliana* (Magrath et al. 1994; de Quiros et al. 2000). We implicated one of these genes (At5g23010) in the condensing reaction of methionine elongation in glucosinolate biosynthesis by three lines of evidence: (i) fine-scale mapping in a cross of two ecotypes (Columbia × Landsberg *erecta*) differing in the chain length of their dominant glucosinolates (Kroymann et al. 2001), (ii) isolation of two mutants with missense mutations in this gene that had altered chain length profiles of glucosinolates compared to wild-type *A. thaliana*, and (iii) heterologous expression of this gene in *Escherichia coli* which gave extracts capable of condensing 2 with acetyl-CoA to give 3, the MAM derivative expected for the first chain-elongation cycle of methionine (Kroymann et al. 2001). Thus, this gene was named methylthioalkylmalate synthase 1 (*MAM1*).

To learn more about the properties of the MAM1 protein, we have now characterized the recombinant enzyme in more detail, as well as examined methylthioalkylmalate synthase (MAMS) activity in in-vitro extracts from *A. thaliana*. In particular, we investigated whether this protein is capable of catalyzing the condensation reactions of subsequent chain-elongation cycles in *A. thaliana*, which produce intermediates that lead to the formation of longer-chain glucosinolates.

---

### Materials and methods

#### Chemicals

For the enzyme assays, \[^{14}C\]acetyl-CoA was purchased from Amersham Biosciences. The native 2-oxo acid co-substrates were obtained from the following sources: compound 2,Sigma;
a

![Diagram](attachment:diagram_a.png)

b

![Diagram](attachment:diagram_b.png)

and washing with 60 ml buffer, a linear gradient from 1 to 0 M ammonium sulfate was run in this buffer at 3 ml min⁻¹ and 4- or 8- ml fractions collected. Size determination was carried out on a Superdex 200 column equilibrated with 50 mM Tris (pH 8.0), 100 mM NaCl, 1 mM MgCl₂ and 1 mM DTT. The marker proteins used were: blue dextran (2,000 kDa), β-amylase (200 kDa), alcohol dehydrogenase (150 kDa), bovine serum albumin (66 kDa), carbonic anhydrase (29 kDa) and cytochrome c (12.4 kDa). A portion of the active Phenylsepharose fractions, concentrated to 1–2 ml, was loaded on this column and 1-ml fractions collected at a flow rate of 1 ml min⁻¹.

The enriched chloroplast fraction was obtained as described by the following protocol. Tissue of *A. thaliana* incubated in the dark for at least 3 days was ground in a mixer in a buffer containing 50 mM Mes (pH 6.7), 0.33 M sorbitol, 2 mM EDTA, 1 mM MgCl₂, 1 mM MnCl₂, 0.5 mM KH₂PO₄, 2 mM NaNO₃ and 5 mM DTT. The resulting brei was filtered through eight layers of cheesecloth. After two centrifugation steps (200 g, 3 min; 6000 g, 30 s), the pellet was suspended in a wash buffer identical to the extraction buffer except that the pH was 7.6 and the buffer lacked DTT. The washed pellet was then suspended in a small volume of lysis buffer consisting of 50 mM Tris (pH 7.6), 1 mM NaCl, 2 mM MgCl₂, 10% glycerol, 1 mM ascorbate, 1 mM DTT, 1% (w/v) BSA, 1% (w/v) PVP-40, 1% (w/v) Amberlite XAD resin, leupeptin (5 μg ml⁻¹), antipain (5 μg ml⁻¹) and 1 mM each of the following protease inhibitors: benzamidine, benzamide, PMSF and γ-aminocaproate. The suspension was sonicated twice with a Sonoplus HD2070 sonicator (Bandelin, Berlin) at 60% power for 5 min to burst the chloroplasts. The resulting extract was centrifuged at 80,000 g for 30 min and the supernatant subjected to gel filtration chromatography on a Superdex 200 column (Amersham), which was equilibrated with 50 mM Tris (pH 8.0), 1 mM NaCl, 2 mM MgCl₂ and 1 mM DTT. Separation was performed under isocratic conditions at a flow rate of 1 ml min⁻¹.

Expression in *E. coli*

A *MAM1* cDNA in which the first 246 nucleotides (representing an apparent N-terminal signal sequence) were truncated (Kroymann et al. 2001) was cloned into a pET-28a expression vector (Novagen, Madison, WI, USA). The construct was expressed in *E. coli* strain BL21(DE3) (F⁻ *ompT rB⁻ mB⁻*; Studier et al. 1990) grown in M9 medium with acid-hydrolyzed casein to an OD₆₀₀ of 0.6 and then induced with 1 mM isopropyl β-D-thiogalactopyranoside (IPTG) for 2 h. Cells were harvested by centrifugation at 6,500 g. After resuspension in 50 mM Tris (pH 8.0) with 1 mM DTT, cells were sonicated twice with a microprobe at 55% full power for a 5 min-20% cycle. Cell debris was precipitated by centrifugation at 8,500 g and the supernatant used for the enzyme assays. A typical preparation contained 5 mg ml⁻¹ of soluble protein.

Plants

Seeds of *Arabidopsis thaliana* (L.) Heynh., ecotype Columbia, were obtained from Lehle Seeds (Round Rock, TX, USA) and ecotype Landsberg *erecta* and mutant TU1 were obtained from the *Arabidopsis* Biological Resource Center (Columbus, Ohio, USA) and the Nottingham *Arabidopsis* Stock Center (Nottingham, UK). Seeds were sown densely in ordinary potting soil mixed with vermiculite (3:1) and raised in a controlled growth chamber with a diurnal cycle of 10 h light at 22°C and 14 h dark at 15°C. Illumination was from a mixture of Gro-Lux and Cool White lamps at 150 μmol photons m⁻² s⁻¹.

Enzyme preparations

All protein solutions were handled at 4°C. Leaves from 3- to 5-week-old plants were harvested, weighed, frozen in liquid nitrogen and ground in a mortar. The resulting powder was resuspended in extraction buffer [50 mM Tris (pH 8.0), 10% glycerol, 10 mM Na₂S₂O₅, 2 mM MgCl₂, 1 mM ascorbate, 1% polyvinylpyrrolidone-40 (PVP-40), 1% polyvinylpolypyrrolidone (PVPP), 1% Amberlite XAD resin, 1 mM benzamidine, 1 mM phenylmethylsulfonyl fluoride (PMSF), 1 mM γ-aminocaproate and 1 mM dithiothreitol (DTT)] at a ratio of 1 g leaf powder to 2 ml buffer, and filtered through six layers of cheesecloth. After the addition of 30% ammonium sulfate and stirring for 30 min, the precipitated protein was removed by centrifugation at 26,000 g. After a second centrifugation step at 90,100 g, the supernatant was loaded onto a Phenylsepharose column (Pharmacia) equilibrated with a buffer of 50 mM Tris (pH 8.0), 1 mM MgCl₂ and 1 mM DTT. After loading
employed a flow-through radioactivity monitor (Packard Radiomatic 500TR) with a 0.5-ml flow cell and Ultima-Flo AP scintillation fluid (Packard) in a ratio of 4:1 to column eluent. The counting efficiency for ${}^{14}$C was 55–60%. Under these conditions, the reaction product, 2-(2′-methylthioethyl)malic acid (3), eluted at approximately 36.5 min and 2-(3′-methylthiopropyl)malic acid (7) at approximately 43 min. For the products of the assays using non-sulfur co-substrates, 2-butymlmalic acid, the product of 2-oxohexanoic acid, eluted at approximately 45 min and 2-pentylmalic acid, the product of 2-oxoheptanoic acid, eluted at approximately 57 min. All assays were carried out in the linear range with respect to time and protein concentration and were repeated at least twice. For determination of kinetic parameters, each substrate was tested at various concentrations in four to six independent experiments.

### Assay product identification

The assay product was identified with an LC–MS system using a Quattro II (Micromass, Altrincham, UK) tandem quadrupole mass spectrometer equipped with an electrospray interface (capillary, 2.5 kV; sample cone, 12 V; desolvation temperature, $375^{\circ}\mathrm{C}$). The HPLC conditions were as described above except that the solvent was 0.03% (aqueous) $\mathrm{CF}_{3} \mathrm{COOH}$ and the flow rate $0.2 \mathrm{~ml} \min ^{-1}$. The MS–MS daughter-ion spectra were recorded by fragmentation of the protonated parent ion, $\left[\mathrm{M}+\mathrm{H}\right]^{+}$. For 3, this ion was found at $m / z 209$, and for 7 at $m / z 223$. Argon was used as the collision gas at $1.5 \times 10^{-3} \mathrm{mbar}$, and a collision energy of $7 \mathrm{eV}$ was employed to achieve fragmentation. The scanned mass range was $m / z 50$ to $m / z 230$ and the scan time $1 \mathrm{~s}$.

### Purification of MAM1 for raising antibodies

The recombinant MAM1 protein had both N- and C-terminal His Tags, and so could be purified by affinity chromatography under denaturing conditions using a pH gradient on Ni-NTA resin (Qiagen) following the manufacturer’s protocol. MAM1-containing fractions were identified by SDS–PAGE, pooled and re-chromatographed on a gel filtration column (Superdex 200) in $50 \mathrm{mM}$ Tris (pH 7.3), $100 \mathrm{mM} \mathrm{NaCl}$ and $6 \mathrm{M}$ urea. The purity of the MAM1-containing fractions was checked by SDS–PAGE. Rabbit antibodies against MAM1 were raised by Davids Biotechnologie (Regensburg).

### Western blotting

An SDS-gel with 10% acrylamide in the separation gel was blotted using Mini-Trans-Blot (BioRad, Munich) onto a polyvinylidenedifluoride (PVDF) membrane (Immobilon-P, $0.45 \mu \mathrm{m}$; Millipore, Eschborn, Germany) for $1 \mathrm{~h}$ at $100 \mathrm{~V}$ (transfer buffer: $25 \mathrm{mM}$ Tris, $192 \mathrm{mM}$ glycine, $20 \%$ methanol). The membrane was blocked with $2 \% \mathrm{BSA}$ in TBST [10 mM Tris–HCl (pH 8.0), $150 \mathrm{mM} \mathrm{NaCl}$, $0.05 \%$ Tween 20] for $30 \mathrm{~min}$ before adding the antibody serum in a $1: 10,000$ dilution and incubation at room temperature for a further $30 \mathrm{~min}$. After three wash steps with TBST ($5 \mathrm{~min}$), the secondary antibody (anti-IgG–AP conjugate, Sigma, $1: 10,000$) was added in TBST plus $2 \% \mathrm{BSA}$ and incubated for $30 \mathrm{~min}$. After three additional washes with TBST ($5 \mathrm{~min}$ each), detection was carried out with $400 \mu \mathrm{l}$ NBT–BCIP solution (Sigma).

---

### Results and discussion

#### Methylthioalkylmalate synthase (MAMS) activity in *Arabidopsis thaliana* extracts

We searched for the condensing enzyme of methionine chain elongation in crude extracts of *A. thaliana* leaves

incubated with ${ }^{[14]} \mathrm{C}]$ acetyl-CoA and 4-methylthio-2-oxobutanoic acid (2). The latter is the keto acid produced from methionine and the substrate for the first round of elongation (Fig. 2a). An acetyl-CoA regeneration system was included because of the high levels of acetyl-CoA thioesterase activity in this extract, and the reaction products were analyzed by HPLC with a radiodetector to maximize the sensitivity of the assay. After numerous attempts, no product was detected. In contrast, the activity of the condensing enzyme of leucine biosynthesis, 2-isopropylmalate synthase (IPMS, Fig. 2b), was readily observed in these extracts when incubations were performed with ${ }^{[14]} \mathrm{C}]$ acetyl-CoA and 2-oxoisovalerate.

Methylthioalkylmalate synthase (MAMS) activity was detected only after preliminary separation using hydrophobic interaction chromatography. MAMS and IPMS activities were cleanly separated on this matrix, with MAMS eluting between 500 and $850 \mathrm{mM}$ ammonium sulfate and IPMS between 0 and $250 \mathrm{mM}$ ammonium sulfate (Fig. 3). Neither set of fractions was able to carry out the activity of the other. Thus, *A. thaliana* possesses distinct enzyme activities for MAMS and IPMS. Size determination via gel filtration showed a single peak for MAMS at $52 \pm 8 \mathrm{kDa}$, while IPMS activity smeared over the range $50-200 \mathrm{kDa}$ with a major peak at $95-120 \mathrm{kDa}$. From the *A. thaliana* gene sequences, the masses of MAMS and IPMS proteins are $55$ and $69 \mathrm{kDa}$, respectively, including potential plastid targeting sequences, suggesting that native MAMS is a monomer and IPMS is a dimer. Further characterization was carried out with preparations of the recombinant enzyme.

![Graph](https://i.imgur.com/yourimageurl.png)

**Fig. 3** Separation of methylthioalkylmalate synthase (MAMS) and 2-isopropylmalate synthase (IPMS) activities in crude extracts of *A. thaliana* leaves using hydrophobic interaction chromatography on Phenylsepharose. The column was equilibrated with $1 \mathrm{M}$ ammonium sulfate [in $50 \mathrm{mM}$ Tris (pH 8.0), with $1 \mathrm{mM}$ DTT and $1 \mathrm{mM} \mathrm{MgCl}_{2}$] and eluted with a linear gradient from 1 to $0 \mathrm{M}$ ammonium sulfate. Flow rate was $3 \mathrm{ml} \min ^{-1}$ and fraction size was $4$ or $8 \mathrm{ml}$Given the presence of potential organellar targeting sequences in both MAMS and IPMS proteins (Kroymann et al. 2001), an attempt was made to determine the subcellular location of these activities. IPMS from spinach (Hagelstein and Schultz 1993) and MAMS from *Eruca sativa* (K. Falk, personnel communication) had previously been shown to be localized in the chloroplasts. An enriched chloroplast extract of *A. thaliana* ecotype Columbia was tested for both activities, but only IPMS activity was found. MAMS was not detected even after the extract was purified via gel filtration. Competition for the acetyl-CoA substrate is unlikely to be responsible for the failure to observe MAMS, since acetyl-CoA thioesterase activity was low in the crude chloroplast extract, and, after gel filtration, thioesterase activity eluted at a much earlier point in the gradient than would have been expected for MAMS activity. Our ability to detect IPMS but not MAMS in *A. thaliana* chloroplast extracts is puzzling and at variance with the presence of a predicted plastid targeting sequence in the *MAM1* gene (Kroymann et al. 2001), and with previous results from our laboratory on other species (K. Falk, C. Vogel, S. Bartram, S. Textor, J. Gershenzon, unpublished results).

### Characterization of the recombinant methylthioalkylmalate synthase protein

The methylthioalkylmalate synthase 1 (*MAM1*) gene from *A. thaliana*, ecotype Columbia, was overexpressed in *Escherichia coli* without its putative plastid targeting sequence (Kroymann et al. 2001). *E. coli* strain BL21(DE3), harboring the expression construct under the control of an inducible T7 promoter, was grown on minimal medium at 37°C. When cells were in the early logarithmic growth phase, they were induced by the addition of 1 mM IPTG for 2 h. Recombinant MAM1 was readily detectable by SDS-PAGE as a band of approximately 50 kDa (calculated size 51.2 kDa) in extracts of induced but not non-induced cells. Crude extracts were used for characterization since extracts of *E. coli* cells lacking the expression construct showed no background MAMS activity.

MAM1 had a pH optimum of 8.5 with half maxima at 7.4 and 9.7. Its optimum temperature was 32°C, and it was almost completely inactive at temperatures above 45°C. DTT (1 mM) was required for activity and iodacetamide was an effective inhibitor (100% inhibition at 1 mM), suggesting that a sulfhydryl or histidine residue is involved in catalysis. The necessity of using DTT made it impossible to gauge the progress of the reaction by detecting the CoASH liberated from acetyl-CoA with Ellman's reagent, 5,5′-dithiobis-(2-nitrobenzoate) (Sreeramulu 1966). Hence, catalysis was measured by detecting the product by HPLC, as in crude extracts.

MAM1 activity also required a divalent metal ion. Of the divalent ions tested, Mn²⁺ at 4 mM gave optimal activity and was used routinely in the standard assay, although Co²⁺, Fe²⁺ and Ni²⁺ supported nearly equivalent rates of reaction also at 4 mM. Mg²⁺ and Ca²⁺ gave only about half of the activity of Mn²⁺, while Cu²⁺ and Zn²⁺ did not support catalysis at all. No activity was detected in the presence of 200 μM EDTA.

The addition of ATP or ADP to the MAM1 reaction increased the rate of reaction over 10-fold, with AMP being much less active (Table 1). However, these adenine nucleotides appear to play no direct role in the reaction because the non-hydrolyzable analogs, AMP-PNP (5′-adenylyl imidodiphosphate), ADP-β-S [adenosine 5′-O-(2-thiodiphosphate)] and ATP-γ-S [adenosine 5′-O-(3-thiotriphosphate)] had similar effects. ATP and ADP may serve as allosteric effectors that regulate MAM1 activity according to the energy status of the cell, a possibility that deserves further investigation. If the active adenine nucleotide effector species is actually a complex with Mg²⁺ or another divalent cation, this may partially explain the requirement of MAM1 for divalent cations.

MAM1 belongs to a large family of enzymes, all of which catalyze condensation reactions between an acyl-CoA ester and a 2-oxo acid (Table 2). Within this group, it shows highest sequence similarity to homocitrate synthase, 2-isopropylmalate synthase (IPMS) and a recently described citramalate synthase from methanogenic bacteria (Wang et al. 1991; Howell et al. 1999) (PROSITE database, see Falquet et al. 2002). The properties of MAM1 are generally similar to those of these related enzymes. For example, homocitrate synthases have also been reported to require ATP and Mg²⁺ (Gray and Bhattacharjee 1976; Masurekar and Demain 1974). 2-Isopropylmalate synthases require a divalent cation, such as Mn²⁺ (Hagelstein and Schultz 1993) or Zn²⁺ (Roeder and Kohlhaw 1980).

For comparison with MAM1, the properties of *A. thaliana* IPMS were studied using crude leaf extracts that had been partially purified by hydrophobic interaction chromatography to remove MAMS (Fig. 3). *A. thaliana* IPMS also displayed a requirement for a divalent metal ion. However, the relative effects of individual cations were somewhat different from those

| Nucleotide at 20 mM | Relative activity (%) |
|---------------------|-----------------------|
| ATP                 | 100<sup>b</sup>       |
| ADP                 | 89                    |
| AMP                 | 25                    |
| No nucleotide       | 7.5                   |
| AMP-PNP<sup>a</sup> | 85                    |
| ADP-β-S<sup>a</sup> | 74                    |
| ATP-γ-S<sup>a</sup> | 87                    |

<sup>a</sup> Abbreviations for non-hydrolyzable analogues: AMP-PNP, 5′-adenylyl imidodiphosphate; ADP-β-S, adenosine 5′-O-(2-thiodiphosphate); ATP-γ-S, adenosine 5′-O-(3-thiotriphosphate).

<sup>b</sup> Activity with ATP was 0.6 pkat (mg protein)<sup>-1</sup>
Table 2 Substrates of MAM1 and related enzymes

| Enzyme                          | EC number   | Acyl-CoA ester co-substrate | 2-Oxo acid co-substrate       |
|----------------------------------|-------------|-----------------------------|-------------------------------|
| Citramalate synthase             | –           | Acetyl-CoA                  | Pyruvic acid                  |
| Citrate synthase                 | 2.3.3.1     | Acetyl-CoA                  | Oxaloacetic acid              |
| Decylcitrate synthase            | 2.3.3.2     | Lauroyl-CoA                 | Oxaloacetic acid              |
| 3-Ethylmalate synthase           | 2.3.3.7     | Butanoyl-CoA                | Glyoxylic acid                |
| Homocitrate synthase             | 2.3.3.14    | Acetyl-CoA                  | 2-Oxoglutaric acid            |
| 2-Isopropylmalate synthase       | 2.3.3.13    | Acetyl-CoA                  | 2-Oxoisovaleric acid          |
| Malate synthase                  | 2.3.3.9     | Acetyl-CoA                  | Glyoxylic acid                |
| 2-Methylcitrate synthase         | 2.3.3.5     | Propionyl-CoA               | Oxaloacetic acid              |
| Methylthioalkylmalate synthase 1 (MAM1) | –           | Acetyl-CoA                  | Various ω-methylthio-2-oxoalkanoates |
| 3-Propylmalate synthase          | 2.3.3.12    | Pentanoyl-CoA               | Glyoxylic acid                |

for MAMS. For example, Mg²⁺ at 4 mM gave twice the activity of Mn²⁺ at the same concentration, quite opposite to the pattern seen with MAMS. IPMS also differed in the lack of a requirement for an adenine nucleotide. IPMS from spinach has been reported to be feedback-inhibited by L-leucine, the end product of the pathway, with complete inhibition at 20 μM (Hagelstein and Schultz 1993). However, the A. thaliana IPMS showed no inhibition with 20 μM L-leucine at pH 8.5, and only 10–20% inhibition with 200 μM L-leucine. The feedback inhibition of yeast IPMS has been described as pH dependent (Ulm et al. 1972), but this phenomenon was not investigated for A. thaliana.

Substrate specificity of the recombinant MAM1

The substrate specificity of MAM1 was tested with a variety of 2-oxo acids. Potentially, the enzyme might use all six of the 2-oxo-ω-methylthioalkanoic acids, including 2, 5, 9 and 13, involved in methionine chain elongation in the biosynthesis of aliphatic glucosinolates in A. thaliana. Since not all of these substrates are commercially available, we tested analogs in which the sulfur atom is replaced by an additional methylene group and generated one substrate in a coupled assay by deamination of its amino acid precursor with L-amino acid oxidase. Among the native 2-oxo-ω-methylthioalkanoates, only two compounds, 2 and 5, served as substrates for MAM1 (Table 3). Reaction products were identified as 3 and 7, respectively, by LC-MS. Compound 3 had been identified previously by co-chromatography with a synthetic standard (Kroymann et al. 2001). The mass spectrum of 7 was similar except that the major fragment ions were 14 mass units larger than the major fragment ions of 3 (Fig. 4).

Kinetic studies revealed that 5-methylthio-2-oxopentanoate (5) had a similar velocity but a significantly lower Kₘ than 2 (Table 3), suggesting that the enzyme carries out the condensation reaction of the second cycle more efficiently than that of the first cycle. The non-sulfur 2-oxo acids tested followed the same pattern as that of the native substrates, with the analogs of 2 and 5 (2-oxohexanoic acid and 2-oxoheptanoic acid,

Table 3 Kinetic parameters of various 2-oxo acids with recombinant MAM1

| Substrate                        | Chemical structure | Kₘ (mM) | Relative velocity (%) |
|----------------------------------|---------------------|---------|-----------------------|
| 2-Oxopentanoic acid              |                     | –       | –                     |
| 4-Methylthio-2-oxobutanoic acid (2) |                     | 3.0     | 86                    |
| 2-Oxohexanoic acid               |                     | 11.2    | 17                    |
| 5-Methylthio-2-oxopentanoic acid (5) |                     | 0.64    | 100ᵇ                   |
| 2-Oxoheptanoic acid              |                     | 7.5     | 14                    |
| 6-Methylthio-2-oxohexanoic acid (coupled assay) (9) |         | –       | –                     |
| 2-Oxoctanoic acid                |                     | –       | –                     |
| Glyoxylic acid                   |                     | –       | –                     |
| Pyruvic acid                     |                     | –       | –                     |
| 2-Oxoisovaleric acid             |                     | –       | –                     |
| β-Phenylpyruvic acid             |                     | –       | –                     |
| Oxaloacetic acid                 |                     | –       | –                     |
| 2-Oxoglutaric acid               |                     | –       | –                     |
| 2-Oxadipic acid                  |                     | –       | –                     |

ᵃ No activity detected  
ᵇ Activity with 5-methylthio-2-oxopentanoic acid was 110 pkat/mg protein of the crude bacterial extract

respectively) supporting the reaction. However, the next shorter compound in the series, 2-oxopentanoic acid, and the next longer compound, 2-oxooctanoic acid, were both inactive. Kinetic data indicated that reaction with the non-sulfur 2-oxo acids was significantly less efficient than with the native substrates, suggesting that the sulfur atom may play a role in binding to the active site. Other 2-oxo acids, the substrates of enzymes catalyzing related condensation reactions with acetyl-CoA, were also tested, but none supported catalysis. The Kₘ for the second substrate, acetyl-CoA, was measured as 245 μM.

Fig. 4a, b Mass spectra of methylthioalkylmalate derivatives that are products of the condensation reactions of acetyl-CoA and methylthioalkanoic acids catalyzed by MAM1. Depicted are collision-induced dissociation-product ion spectra of [M+H]⁺ of a compound 3, the product of the condensation of 2 and acetyl-CoA, and b compound 7, the product of the condensation of 5 and acetyl-CoA

Taken together, these results indicate that recombinant MAM1 from the Columbia ecotype is capable of catalyzing the condensation reactions of only the first two rounds of chain elongation in the biosynthesis of methionine-derived glucosinolates (2 → 3, 5 → 7). The species *A. thaliana* actually produces glucosinolates arising from as many as six rounds of chain elongation. Yet, the ecotype Columbia, consistent with the abilities of MAM1, produces principally glucosinolates that arise from two rounds of elongation. (The major Columbia glucosinolates are referred to as C₄ glucosinolates because they contain four methylene groups in the side chain, two from the original methionine skeleton and two from side chain elongation.) A recent report describes the cloning of a MAM-type gene from *Brassica oleracea* whose sequence and intron/exon arrangement are more similar to *MAM1* than to the other *MAM*-like gene on chromosome V of *A. thaliana* (Li and Quiros 2002). This gene segregates closely with the presence of C₄ glucosinolates in various broccoli and cauliflower varieties, suggesting it could also have the ability to catalyze the condensation reactions of just the first two rounds of elongation. However, the situation in *Eruca sativa*, another member of the Brassicaceae that accumulates chiefly C₄ glucosinolates, appears to be quite different. In this species, a MAMS activity was purified that catalyzes the condensation reaction of the first round of elongation, but not the second (Falk et al. 2004).

In the present study, the heterologously expressed *A. thaliana* MAM1 was unable to use 2-oxoisovalerate, the substrate of IPMS, which catalyzes the condensation reaction involved in leucine biosynthesis. Consistent with this finding, IPMS activity was readily separated from MAMS activity in crude extracts of *A. thaliana*. However, a recent report describes the ability of both *MAM1* and the other *MAM*-like gene on chromosome V to complement an *E. coli* leucine auxotroph lacking IPMS activity (Junk and Mourad 2002). Perhaps under certain conditions MAM1 is capable of low levels of 2-oxoisovalerate utilization.

MAMS activity in other lines of *A. thaliana*

A number of *A. thaliana* ecotypes and mutants have been characterized with differences in the chain-lengths of their major aliphatic glucosinolates. For example, while the glucosinolates of ecotype Columbia are dominated by C₄ compounds (Fig. 2a), the aliphatic glucosinolates of the Landsberg *erecta* ecotype are dominated by C₃ compounds (side chains with three methylene groups, requiring only one round of elongation; Magrath et al. 1994; Kliebenstein et al. 2001). Fine-scale mapping analysis in the Columbia and Landsberg *erecta* cross had previously identified *MAM1* as the gene responsible for the C₃ versus C₄ variation (Kroymann et al. 2001). A mutant line of ecotype Columbia (TU1) has also been characterized in which *MAM1* has a missense mutation resulting in the near complete disappearance of C₄ glucosinolates (Haughn et al. 1991; Kroymann et al. 2001). To determine the effects of these genetic changes on MAMS enzyme activity, we investigated crude extracts prepared from leaves of these plants that had been subjected to hydrophobic interaction chromatography. Fractions from extracts of Landsberg *erecta* and TU1 all showed MAMS activity in the same portion of the gradient as in Columbia. And, in all cases this activity was capable of carrying out the condensation reactions of the first two rounds of chain elongation producing 3 and 7 (data not shown), just as in Columbia.

It was surprising to find that both an ecotype (Landsberg *erecta*) and a mutant line (TU1) that did not accumulate C₄ glucosinolates were nevertheless capable of performing the condensing reactions of the chain-elongation cycle leading ultimately to C₄ glucosinolates, especially since these plants were shown to have altered *MAM1* genes (Kroymann et al. 2001). Perhaps other enzymes present in the plant, such as IPMS or proteins participating in chain-elongation cycles forming longer-chain glucosinolates, catalyzed the formation of 3 and 7 in our crude extracts.

To test these possibilities, we attempted to identify the proteins responsible for MAMS activity in the

extract of Landsberg *erecta*. Antibodies were raised to recombinant MAM1 that had been purified by SDS-PAGE. The anti-MAM1 serum reacted with proteins of approximately 50 and 70 kDa (Fig. 5), neither of which were detected by the pre-immune serum. These bands appeared to represent MAMS and IPMS, respectively, since the mature processed forms of MAMS and IPMS should have masses of 51 and 66 kDa, respectively, based on their gene sequence lengths and length of their predicted N-terminal targeting sequences. While not as specific as we had hoped, the anti-MAM1 serum was nevertheless useful for distinguishing between MAMS and IPMS proteins in plant extracts because of their size difference. Western blot analysis of the partially purified MAMS fractions from the Landsberg *erecta* ecotype showed that these contained an immunologically recognized protein of the size of MAMS, but showed no evidence of IPMS. Thus, IPMS probably does not carry out the condensation reactions of methionine chain elongation in *A. thaliana*, either in Columbia or in other ecotypes. The protein(s) in Landsberg *erecta* extracts that catalyzes the condensation reaction (5 → 7) leading to C4 glucosinolates is probably a MAMS protein that normally lacks access to 5 *in planta* or produces 7 that is channeled to higher-order intermediates of chain elongation, so that no C4 glucosinolates are formed. We are currently employing biochemical and genetic approaches to study the MAMS proteins of Landsberg *erecta*, an *A. thaliana* ecotype that contains three MAM genes rather than two as in Columbia (Kroymann et al. 2003).

---

Table 4 Summary of properties of recombinant MAM1 from *A. thaliana* ecotype Columbia expressed in *Escherichia coli*

| Molecular mass | 51.2 kDa |
| --- | --- |
| Optimum temperature | 32°C |
| pH Optimum | 8.5 |
| Half maxima | 7.4, 9.7 |
| Thiol reagent requirement | DTT, 1 mM |
| Divalent metal ion requirement | Mn²⁺, 4 mM (Co²⁺, Fe²⁺, Ni²⁺ nearly equivalent) |
| Adenine nucleotide requirement | ATP, 20 mM |
| Kₘ, Acetyl-CoA | 245 μM |
| Kₘ, 4-Methylthio-2-oxobutanoic acid (2, first round of chain elongation) | 3.0 mM |
| Kₘ, 5-Methylthio-2-oxopentanoic acid (5, second round of chain elongation) | 0.64 mM |

---

### Conclusions

Enzymes known as methylthioalkylmalate synthases (MAMS) catalyze the first reaction of the elongation cycle of methionine-derived glucosinolates, condensing acetyl-CoA and a 2-oxo methylthioalkanoic acid to form a malate derivative. In this contribution, we have shown that *MAM1*, one of two *MAM*-type genes in *A. thaliana* ecotype Columbia encodes a protein whose properties (Table 4) are similar to those of other enzymes catalyzing the condensation of acetyl-CoA and 2-oxo acids, such as homocitrate synthase (which makes a ligand used in nitrogen fixation) and isopropylmalate synthase (IPMS), which participates in leucine biosynthesis. However, when heterologously expressed in *E. coli*, the MAM1 protein cannot catalyze any of these other reactions. In fact, MAMS and IPMS activity are clearly separable chromatographically in *A. thaliana* extracts. The heterologously expressed MAM1 protein is capable of catalyzing only the condensation reactions of the first two elongation cycles of methionine-derived glucosinolates, sufficient for the elongation of C4 glucosinolates, the major glucosinolate type in the ecotype Columbia. Additional as yet uncharacterized enzymes must carry out the condensation reactions of the later elongation cycles forming C5-C8 glucosinolates. These activities could have been responsible for the condensation reactions observed in extracts of *A. thaliana* lines mutated or otherwise altered in *MAM1* that do not accumulate C4 glucosinolates.

---

### References

Chapple CCS, Decicco C, Ellis BE (1988) Biosynthesis of 2-(2'-methylthio)ethylmalate in *Brassica carinata*. Phytochemistry 27:3461–3463  
Chisholm MD, Wetter LR (1964) Biosynthesis of mustard oil glucosides: IV. The administration of methionine-C¹⁴ and related compounds to horseradish. Can J Biochem 42:1033–1040  
Dawson GW, Hick AJ, Bennett RN, Donald A, Pickett JA, Wallsgrove RM (1993) Synthesis of glucosinolate precursors

and investigations into the biosynthesis of phenylalkyl- and methylthioalkylglucosinolates. J Biol Chem 268:27154–27159

de Quiros HC, Magrath R, McCallum D, Kroymann J, Schnabelrauch D, Mitchell-Olds T, Mithen R (2000) α-Keto acid elongation and glucosinolate biosynthesis in *Arabidopsis thaliana*. Theor Appl Genet 101:429–437

Fahey JW, Zalcmann AT, Talalay P (2001) The chemical diversity and distribution of glucosinolates and isothiocyanates among plants. Phytochemistry 56:5–51

Falk KL, Vogel C, Textor S, Bartram S, Hick A, Pickett JA, Gershenzon J (2004) Glucosinolate biosynthesis: demonstration and characterization of the condensing enzyme of the chain elongation cycle in *Eruca sativa*. Phytochemistry (in press)

Falquet L, Pagni M, Bucher P, Hulo N, Sigrist CJ, Hofmann K, Bairoch A (2002) The PROSITE database, its status in 2002. Nucleic Acids Res 30:235–238

Graser G, Schneider B, Oldham NJ, Gershenzon J (2000) The methionine chain elongation pathway in the biosynthesis of glucosinolates in *Eruca sativa* (Brassicaceae). Arch Biochem Biophys 378:411–419

Gray GS, Bhattacharjee JK (1976) Biosynthesis of lysine in *Saccharomyces cerevisiae*: properties and spectrophotometric determination of homocitrate synthase activity. Can J Microbiol 22:1664–1667

Hagelstein P, Schultz G (1993) Leucine synthesis in spinach chloroplasts: partial characterization of 2-isopropylmalate synthase. Biol Chem Hoppe-Seyler 374:1105–1108

Halkier BA (1999) Glucosinolates. In: Ikan R (ed) Naturally occurring glycosides: chemistry, distribution and biological properties, Wiley, Chichester, UK, pp 193–223

Haughn GW, Davin L, Giblin M, Underhill EW (1991) Biochemical genetics of plant secondary metabolites in *Arabidopsis thaliana*—the glucosinolates. Plant Physiol 97:217–226

Hecht SS (2000) Inhibition of carcinogenesis by isothiocyanates. Drug Metab Rev 32:395–411

Hogge LR, Reed DW, Underhill EW, Haughn GW (1988) HPLC separation of glucosinolates from leaves and seeds of *Arabidopsis thaliana* and their identification using thermospray liquid chromatography-mass spectrometry. J Chromatogr Sci 26:551–556

Howell DM, Xu H, White RH (1999) (R)-Citramalate synthase in methanogenic archaea. J Bacteriol 181:331–333

Junk DJ, Mourad GS (2002) Isolation and expression analysis of the isopropylmalate synthase gene family of *Arabidopsis thaliana*. J Exp Bot 53:2453–2454

Kliebenstein DJ, Kroymann J, Brown P, Figuth A, Pedersen D, Gershenzon J, Mitchell-Olds T (2001) Genetic control of natural variation in *Arabidopsis* glucosinolate accumulation. Plant Physiol 126:811–825

Kroumova AB, Wagner GJ (2003) Different elongation pathways in the biosynthesis of acyl groups of trichome exudate sugar esters from various solanaceous plants. Planta 216:1013–1021

Kroymann J, Textor S, Tokuhisa JG, Falk KL, Bartram S, Gershenzon J, Mitchell-Olds T (2001) A gene controlling variation in *Arabidopsis* glucosinolate composition is part of the methionine chain elongation pathway. Plant Physiol 127:1077–1088

Kroymann J, Donnerhacke S, Schnabelrauch D, Mitchell-Olds T (2003) Evolutionary dynamics of an *Arabidopsis* insect resistance quantitative trait locus. Proc Natl Acad Sci USA (in press)

Lee CJ, Serif GS (1968) 2-Amino-6-(methylthio)caproic acid, a methionine homolog and precursor of progoitrin. Biochim Biophys Acta 165:569–571

Lee CJ, Serif GS (1970) Precursor role of \[^{14}C\], \[^{15}N\]-2-amino-6-(methylthio)caproic acid in progoitrin biosynthesis. Biochemistry 9:2068–2071

Li GY, Quiros CF (2002) Genetic analysis, expression and molecular characterization of *BoGSL-ELONG*, a major gene involved in the aliphatic glucosinolate pathway of *Brassica* species. Genetics 162:1937–1943

Magrath R, Bano F, Morgner M, Parkin I, Sharpe A, Lister C, Dean C, Turner J, Lydiate D, Mithen R (1994) Genetics of aliphatic glucosinolates. I. Side chain elongation in *Brassica napus* and *Arabidopsis thaliana*. Heredity 72:290–299

Masurekar PS, Demain AL (1974) Insensitivity of homocitrate synthase in extracts of *Penicillium chrysogenum* to feedback inhibition by lysine. Appl Microbiol 28:265–270

Rask L, Andreasson E, Ekbom B, Eriksson S, Pontoppidan B, Meijer J (2000) Myrosinase: gene family evolution and herbivore defense in Brassicaceae. Plant Mol Biol 42:93–113

Reichelt M, Brown PD, Schneider B, Oldham NJ, Stauber E, Tokuhisa J, Kliebenstein DJ, Mitchell-Olds T, Gershenzon J (2002) Benzoic acid glucosinolate esters and other glucosinolates from *Arabidopsis thaliana*. Phytochemistry 59:663–671

Roeder PR, Kohlhaw GB (1980) α-Isopropylmalate synthase from yeast. A zinc metalloenzyme. Biochim Biophys Acta 613:482–487

Srere PA (1966) Citrate-condensing enzyme-oxaloacetate binary complex. Studies of its physical and chemical properties. J Biol Chem 241:2157–2165

Studier FW, Rosenberg AH, Dunn JJ, Dubendorf JW (1990) Use of T7 RNA polymerase to direct expression of cloned genes. Methods Enzymol 185:60–89

Ulm EH, Bohme R, Kohlhaw G (1972) α-Isopropylmalate synthase from yeast: purification, kinetic studies, and effect of ligands on stability. J Bacteriol 110:1118–1126

Wang SZ, Dean DR, Chen JS, Johnson JL (1991) The N-terminal and C-terminal portions of NifV are encoded by two different genes in *Clostridium pasteurianum*. J Bacteriol 173:3041–3046

Wittstock U, Halkier BA (2002) Glucosinolate research in the *Arabidopsis* era. Trends Plant Sci 7:263–270
