
Eur. J. Biochem. 271, 932–940 (2004) © FEBS 2004 doi:10.1111/j.1432-1033.2004.03995.x

# Furanocoumarin biosynthesis in *Ammi majus* L.

## Cloning of bergaptol O-methyltransferase

### Marc Hehmann${}^{1,*}$, Richard Lukačin${}^{1,*}$, Halina Ekiert${}^{2}$ and Ulrich Matern${}^{1}$

${}^{1}$Institut für Pharmazeutische Biologie, Philipps-Universität Marburg, Germany; ${}^{2}$Department of Pharmaceutical Botany, Collegium Medicum, Jagiellonian University, Kraków, Poland

Plants belonging to the Apiaceae or Rutaceae accumulate methoxylated psoralens, such as bergapten or xanthotoxin, as the final products of their furanocoumarin biosynthesis, and the rate of accumulation depends on environmental and other cues. Distinct $O$-methyltransferase activities had been reported to methylate bergaptol to bergapten and xanthotoxin to xanthotoxin, from induced cell cultures of *Ruta graveolens*, *Petroselinum crispum* and *Ammi majus*. Bergaptol 5-$O$-methyltransferase (BMT) cDNA was cloned from dark-grown *Ammi majus* L. cells treated with a crude fungal elicitor. The translated polypeptide of 38.7 kDa, composed of 354 amino acids, revealed considerable sequence similarity to heterologous caffeic acid 3-$O$-methyltransferases (COMTs). For homologous comparison, COMT was cloned from *A. majus* plants and shown to share 64% identity and about 79% similarity with the BMT sequence at the polypeptide level. Functional expression of both enzymes in *Escherichia coli* revealed that the BMT activity in the bacterial extracts was labile and rapidly lost on purification, whereas the COMT activity remained stable. Furthermore,

the recombinant AmBMT, which was most active in potassium phosphate buffer of pH 8 at 42 °C, showed narrow substrate specificity for bergaptol ($K_{\text{m}}$ SAM 6.5 μM; $K_{\text{m}}$ Bergaptol 2.8 μM) when assayed with a variety of substrates, including xanthotoxin, while the AmCOMT accepted 5-hydroxyferulic acid, esculetin and other substrates. Dark-grown *A. majus* cells expressed significant BMT activity which nevertheless increased sevenfold within 8 h upon the addition of elicitor and reached a transient maximum at 8–11 h, whereas the COMT activity was rather low and did not respond to the elicitation. Complementary Northern blotting revealed that the BMT transcript abundance increased to a maximum at 7 h, while only a weak constitutive signal was observed for the COMT transcript. The AmBMT sequence thus represents a novel database accession specific for the biosynthesis of psoralens.

**Keywords:** *Ammi majus* L.; Apiaceae; furanocoumarin biosynthesis; bergaptol $O$-methyltransferase; caffeate $O$-methyltransferase.

Cell suspension cultures of the Apiaceae, in particular *Ammi majus* L. [1,2] and *Petroselinum crispum* [3–6], have served in numerous model studies on the induced plant disease resistance response. Upon treatment with fungal elicitor, these cells produce linear furanocoumarins (psoralens) besides lignin-like compounds for reinforcement of their cell walls [7–9]. Various crude cell wall elicitors, particularly Pmg (from *Phythophthora sojae*, formerly *Phythophthora megasperma f.* sp. *glycinea*), have been used in the past, and at least in case of the induction of parsley cells the eliciting principle

has been identified as a peptide [10]. Furanocoumarins are potentially toxic compounds which probably function as phytoalexins in the response to fungal infection [3], but their accumulation can also be triggered by other means, e.g. wounding of plants or exposure to acid fog [11,12]. A considerable proportion of psoralens may be recovered from the surface of plants [13], and most of the psoralens elicited in cell cultures also accumulated in the culture fluid. Psoralens are capable of intercalating with DNA, and the methoxylated psoralens bergapten and xanthotoxin are the most relevant natural furanocoumarins in terms of their therapeutic potential. These psoralens exhibit photosensitizing and antiproliferative activities [14] and were evaluated as photosensitizing drugs in oral psoralen plus UVA irradiation (PUVA) therapy of psoriasis and vitiligo [15,16].

*A. majus* cells also uniquely produce derivatives of 7-$O$-prenylumbelliferone under conditions of elicitation (Fig. 1) [17], and the induction of coumarin biosynthesis in these cell cultures provided the basis for extensive *in vitro* investigations. Both the umbelliferone 7-$O$-prenyltransferase activity and a 6-$C$-prenyltransferase activity forming demethyluberosin en route to the psoralens (Fig. 1) were found associated with the microsomal fraction [18]. Such a 6-$C$-prenyltransferase activity had been reported initially as a Mn${}^{2+}$-dependent enzyme from *Ruta graveolens* and assigned to the plastidic membranes [19]. Individual

Correspondence to U. Matern, Institut für Pharmazeutische Biologie, Philipps-Universität Marburg, Deutschhausstrasse 17A, D-35037 Marburg, Germany.
Fax: + 49 6421 282 6678, Tel.: + 49 6421 282 2461,
E-mail: matern@staff.uni-marburg.de

Abbreviations: SAM, S-adenosyl-L-methionine; BMT, bergaptol 5-$O$-methyltransferase; XMT, xanthotoxin 8-$O$-methyltransferase; OMT, $O$-methyltransferase; COMT, caffeic acid 3-$O$-methyltransferase; PUVA, psoralen plus UVA irradiation; RACE, rapid amplification of cDNA ends; RLM-RACE, RNA ligase-mediated rapid amplification of cDNA ends.

*These authors contributed equally to the work described.

(Received 3 November 2003, revised 17 December 2003, accepted 15 January 2004)

© FEBS 2004

Bergaptol O-methyltransferase in *Ammi majus* (Eur. J. Biochem. 271) 933

![Chemical Structures](https://i.imgur.com/yourimage.png)

Fig. 1. Schematic outline of linear furanocoumarin biosynthesis. The sequence of hydroxylations and O-methylations of psoralen leading to isopimpinellin has not been established.

cytochrome P450 monooxygenases sequentially convert demethylsuberosin to (+)-marmesin and psoralen, and these activities were also demonstrated in the microsomal fraction of elicited *A. majus* cells [1,2]. Moreover, the conversion of (+)-marmesin to psoralen (Fig. 1) was proven to proceed by syn-elimination releasing acetone [20]. The subsequent hydroxylation of psoralen in the 5- or 8-position yields bergaptol and xanthotoxin, respectively (Fig. 1). Both of these hydroxylations to yield 5,8-dihydroxypsoralen are required for the formation of isopimpinellin, which commonly accumulates as a minor byproduct upon induction, but the order of hydroxylations and O-methylations remains unresolved [19]. However, psoralen 5-monooxygenase activity forming bergaptol was demonstrated in vitro with microsomes from elicited *A. majus* cells [2]. The biosynthesis of bergaptol from umbelliferone is thus entirely catalyzed through membrane-bound enzymes, involving one prenyltransferase and three P450 monooxygenases, and is preceded by the formation of umbelliferone from 4-coumaric acid, which was also proposed to depend on a P450 monooxygenase. The 5- or 8-hydroxylated furanocoumarins (bergaptol or xanthotoxin) are further processed by O-methylation to bergapten and xanthotoxin, and the corresponding O-methyltransferases (OMTs) were identified as distinct entities and purified by affinity chromatography from *Ruta graveolens* [21] and later also from *Petroselinum crispum* [22]. S-Adenosyl-L-methionine–bergaptol 5-O-methyltransferase (SAM–BMT) and S-adenosyl-L-methionine–xanthotoxin 8-O-methyltransferase (SAM–XMT) (Fig. 1) are also expressed in dark-grown *A. majus* cells, and in all instances these methyltransferases are soluble and inducible enzymes.

We report here the cloning and functional characterization of BMT from elicitor-treated *Ammi majus* L. cells as a major step towards a molecular understanding of psoralen biosynthesis. For comparison, the closely related S-adenosyl-L-methionine–caffeate 3-O-methyltransferase (SAM–COMT) was also cloned from *A. majus* plants, and the differential regulation of these methyltransferases was examined upon elicitation of the cell cultures.

## Materials and methods

### Ammi majus cell cultures and induction

Cell suspension cultures of *A. majus* L. (40 mL B5⁺ medium in 250 mL flasks) were initiated and grown continuously in the dark as described elsewhere [1,2]. Pmg

934 M. Hehmann *et al.* (Eur. J. Biochem. 271)

elicitator was suspended in distilled water (5 mg·mL⁻¹), the suspension was heated to boiling point and added to 6-day-old cell cultures (1 mL per 40 mL culture). The cells were harvested 4 h later and immediately frozen in liquid nitrogen and stored at –70 °C until use.

### Chemicals

Biochemicals were purchased from Roth (Karlsruhe, Germany), vectors and *Escherichia coli* host strains from Invitrogen (Karlsruhe, Germany) or Qiagen (Hilden, Germany). Restriction enzymes and DNA modifying enzymes were from MBI-Fermentas (St. Leon-Rot, Germany), Promega (Mannheim, Germany) or Stratagene (Heidelberg, Germany). Bergaptol was bought from Extrasynthese (Genay, France), caffeic acid from Roth (Karlsruhe, Germany), and [methyl-¹⁴C]S-adenosyl-L-methionine was purchased from Hartmann Analytic (Braunschweig, Germany).

### RNA isolation, PCR cloning and heterologous expression

Total RNA was isolated from *Pmg* elicitor-induced cells following the protocol of Giuliano *et al.* [23]. The time of elicitor-induction was chosen from previous induction experiments in which the time course of furanocoumarin-specific enzyme activities in *A. majus* had been monitored [1,2]. Alternatively, the RNA was isolated from the stems and leaves of 4–6 week-old *A. majus* plants. cDNA fragments were generated by RT-PCR amplification using degenerate oligonucleotide primers [24] which had been designed according to conserved amino acid sequences of plant OMTs [25,26]. The cDNA fragments were cloned, sequenced, and full length clones were generated by the rapid amplification of cDNA ends (RACE) and RNA ligase-mediated rapid amplification of cDNA ends (RLM-RACE) techniques, respectively, using gene-specific primers. Cloning of the PCR products was performed by TOPO TA Cloning (Invitrogen, Karlsruhe, Germany). Briefly, the protein coding regions of the putative BMT and COMT were amplified with 5′-primers providing an *NcoI* site directly before the start codon and 3′-primers inserting a *BamHI* site after the stop codon before they were cloned into the pCR2.1-TOPO vector. An internal *NcoI* site contained in the ORF of the COMT was deleted by using QuikChange® Multi Site-Directed Mutagenesis Kit as described by the manufacturer (Stratagene, Heidelberg, Germany) without altering the amino acid sequence. The mutation was verified by DNA sequencing [27], and the BMT- and COMT-coding DNA clones were subsequently isolated by digestion with *NcoI* and *BamHI*. The cDNAs were subcloned into pQE60 vector (Qiagen, Hilden, Germany) for functional expression in *E. coli* strain M15 (Qiagen) harboring the plasmid pRep4 and employed for BMT and COMT activity assays, respectively. The expression was induced by the addition of 1.0 mm isopropyl thiogalactoside [28]. Following the induction, the cells were harvested by centrifugation [29], disrupted by ultrasonication, the crude extract was cleared by centrifugation (30 000 g, 4 °C, 10 min), and enzyme assays were carried out with the supernatants.

### Sequence analysis

The cDNAs amplified by RT-PCR were sequenced by the dideoxy nucleotide chain termination technique [27]. The cDNA sequences were subjected to BLAST searches (advanced WU-Blast2; EMBL) and alignments with CLUSTALW algorithm (EMBL).

### Purification procedure

The crude extract was fractionated by ammonium sulfate precipitation from 0 to 45%, 45–60% and 60–80% saturation. The 60–80% fraction (BMT) and the 45–60% fraction (COMT) were dissolved in 200 mM potassium phosphate buffer of pH 8.0 and 200 mM Tris/HCl buffer of pH 7.5, respectively. The extracts were desalted by size exclusion chromatography through PD-10 columns (Amersham, Freiburg, Germany) and on Fractogel EMD BioSEC (S) (Merck, Darmstadt, Germany). The purification was monitored by SDS/PAGE [30] and enzyme activity assays of individual fractions.

### Enzyme assays and other analytical methods

BMT activity was routinely measured at 42 °C in 200 mM potassium phosphate buffer pH 8.0 in the presence of sodium ascorbate (20 mM), magnesium chloride (1.5 mM), bergaptol (250 μM) and the recombinant bacterial enzyme extract. The reaction was started by the addition of S-adenosyl-L-[methyl-¹⁴C]methionine (40 μM), and the product was identified as bergapten by silica thin-layer chromatography employing trichloromethane/ethylacetate (2 : 1, v/v; *R*~F~ bergaptol 0.52, *R*~F~ bergapten 0.77), trichloromethane/methanol (95 : 5, v/v; *R*~F~ bergaptol 0.26, *R*~F~ bergapten 0.78), *n*-hexane/ethylacetate/methanol (5 : 5 : 1, v/v/v; *R*~F~ bergaptol 0.67, *R*~F~ bergapten 0.78) or toluene/ethylacetate (3 : 2, v/v; *R*~F~ bergaptol 0.34, *R*~F~ bergapten 0.68) as the solvent systems. The COMT activity assay was carried out at 32 °C in 200 mM potassium phosphate buffer pH 7.5 containing sodium ascorbate (20 mM), magnesium chloride (1.5 mM), caffeic acid (250 μM) and S-adenosyl-L-[methyl-¹⁴C]methionine (40 μM) in addition to the crude enzyme protein. The reactions were stopped by the addition of 1 M HCL (30 μL) and extracted with 400 μL ethyl acetate. Aliquots of the organic phase (200 μL) were mixed with 5 mL scintillation cocktail (Roth, Karlsruhe, Germany) and measured in a liquid scintillation counter (1214 Rackbeta; PerkinElmer, Wellesley, MA, USA). Incubations with boiled enzyme (5 min at 100 °C), or mixtures lacking bergaptol and caffeic acid, were run for control and served for background corrections.

Linear conditions for kinetic assays were established by adjusting the amount of enzyme protein (BMT: 0.15–3.0 μg per assay; COMT: 0.5–10.25 μg per assay). The BMT assays were usually conducted for 20 min, using 1.5 μg desalted protein and 4.0 nmol S-adenosyl-L-[methyl-¹⁴C]methionine and 25.0 nmol bergaptol per 100 μL incubation, which secured linear conversion rates for about 60 min. The COMT assays were carried out accordingly using 5 μg desalted protein. Protein was determined according Lowry *et al.* [31] and the data were extrapolated from Lineweaver–Burk plots.

© FEBS 2004                                                                                     Bergaptol *O*-methyltransferase in *Ammi majus* (Eur. J. Biochem. 271) 935

### Northern blotting

Following the addition of *Pmg* elicitor to the *A.* *majus* cell suspensions, total RNA was isolated from the cells every 0.5 h up to 8 h and used for Northern blot analysis (RNA dot blot). The RNA (4 μg) was denatured in 0.5× Mops buffer pH 6.0, containing 50% (v/v) formamide and 2.2 M formaldehyde, and transferred to a Hybond-N⁺ nylon membrane (Amersham Biosciences, Freiburg) [32] in an I-SRc 96-Dot Blot Minifold (Schleicher and Schüll, Dassel, Germany). The full size *A.* *majus* BMT cDNA (1062 bp) and COMT cDNA (1095 bp) were ^32^P-labeled using a Rediprime™II-random prime labeling system (Amersham Biosciences, Freiburg), and used as probes. The blots were blocked and then hybridized with 25 ng of one of these two labeled probes. Hybridization was carried out overnight at 68 °C in 2× Denhardt’s solution in the presence of 0.5% (w/v) SDS and 100 μg·mL⁻¹ salmon sperm DNA (Sigma, Deisenhofen, Germany). After stringent washings for 20 min at room temperature in 2× NaCl/Cit followed by 15 min at 68 °C in 2× NaCl/Cit the membranes were exposed to a Bio Imaging Analyzer FLA-2000 (Fujifilm).

the correspondingly low COMT activity in cultured *Petroselinum crispum* cells [36]. In *Petroselinum* as well as in *Ruta* cells, the COMT activity had been clearly distinguished from the BMT and XMT activities [22,34].

### cDNA cloning and functional expression

The total RNA from *A.* *majus* cells that had been elicited for 4 h was used as a template for RT-PCR amplifications, with degenerate oligonucleotide primers designed for the cloning of COMT-related enzymes from other plant sources [24–26]. These experiments generated two different fragments of 215 bp which were cloned into the pCR2.1-TOPO vector and extended to the full size cDNAs of 1062 and 1074 bp, respectively, by RACE and RLM-RACE (GenBank accession nos. AY443006 and AY443008). Preliminary sequence alignments had already revealed a close similarity of the cDNAs with those of other plant OMTs. Therefore, the inserts were subcloned into an expression vector for the expression in *E.* *coli*, and the recombinant polypeptides were extracted from the induced transformants.

---

**Results**

#### Induction of *O*-methyltransferase activities

Elicitor-induction studies had been carried out previously with *A.* *majus* cell cultures, and the coumarin-specific enzyme activities (dimethylallyldiphosphate: umbelliferone 6-*C*- and 7-*O*-dimethylallyltransferases) commonly reached a first maximum at 12 h of induction [18]. This time course suggested maximal transcript abundances within the first 6 h of elicitation and corresponded to the patterns reported for the elicitor induction of phenylalanine ammonia lyase and 4-coumarate:CoA ligase activities in *Petroselinum crispum* cell cultures [33]. However, a different induction profile was reported by Hauffe *et al.* [22] for the BMT activity in *P.* *crispum* cells with a maximum beyond 25 h. Therefore, preliminary assays of BMT and XMT activities were conducted with crude extracts of dark-grown *A.* *majus* cells and under the conditions described for *Ruta graveolens* [34]. These assays revealed that both *O*-methyltransferase activities were expressed already in the controls (0.4 μkat·kg⁻¹ BMT and 66 nkat·kg⁻¹ XMT, on average), but increased considerably in response to treatment of the cell suspensions with *Pmg* elicitor. The activities were measured every three hours over the time period from 2 to 23 h following the addition of elicitor, and shown to increase sevenfold within 8 h to reach a transient maximum at about 8–11 h. (H. Ekiert and R. Lukačin, unpublished data). Thus, the cells at 4 h after the addition of elicitor were considered to contain the highest BMT and XMT transcript abundances.

The activity of COMT in *A.* *majus* was examined for comparison, and only low levels (2.5 μkat·kg⁻¹ on average) were observed in crude extracts of suspension-cultured cells, which hardly changed upon elicitation and might be due to related OMTs with specificity towards catechols [26,35]. However, higher activity (4.2 μkat·kg⁻¹) was determined in extracts of leaf and stem tissues. The moderate rate of COMT expression in the suspension cells is reminiscent of

Fig. 2. Thin-layer cochromatography of the labeled product from BMT assays with authentic bergaptol and bergapten on silica $F_{254}$ plates developed with *n*-hexane/ethylacetate/methanol (5 : 5 : 1, v/v/v). Reference furanocoumarins separated in the absence (lane 1) and in the presence of the enzymatic product (lane 3) were spotted by their quenching under irradiation at 254 nm, and the enzymatic product (lanes 2 and 3) was detected by autoradiography using a Bioimager (inverse presentation). S, start line; F, solvent front.

Fig. 3. Induction of BMT transcript abundance in *Ammi majus* cell cultures. Total RNA (4 μg per dot) isolated from the cells at different time intervals following the addition of elicitor (lanes 1+, 2+) or from controls (lanes 3−, 4−) treated with water (1 mL·40 mL⁻¹ culture) was employed for Northern dot blot hybridization using labeled *AmBMT* cDNA as a probe.

in 70 mM Tris/HCl buffer pH 7.5, containing 10 mM EDTA. The enzyme activity of the crude supernatants was determined with a variety of potential substrates, and the 1062 bp transformant was found to methylate bergaptol to bergapten with narrow substrate specificity. The identity of the enzymatic product was firmly established by thin-layer chromatography with authentic bergapten in four solvent systems (Fig. 2), and hence the transformant encoded a BMT, designated *AmBMT*. The functionality of the 1074 bp clone encoding a COMT-like protein, however, has so far not been assigned.

Because the RT-PCR from cell culture RNA failed to amplify a full-size COMT sequence, we turned back to *A. majus* plants and used the RNA from leaf and stem tissues as a template. These experiments yielded a cDNA of 1095 bp (GenBank accession no. AY443007), which was expressed in *E. coli* and shown to encode a COMT that converts caffeate to ferulate.

Both labeled cDNAs were used as probes for Northern blotting experiments employing the total RNA of *A. majus* cells at various time points following the addition of the *Pmg* elicitor. The abundance of BMT transcripts, which were hardly detectable in control cells, increased significantly to a transient maximum at 7 h (Fig. 3). However, a very weak hybridization signal was recorded for the COMT transcripts that hardly changes in intensity over the time of the experiment and corresponded with the low constitutive level of enzyme activity in the cells.

### Characterization of enzymes

The BMT activity (≈ 2.5 μkat·kg⁻¹) in the desalted bacterial extracts was very labile and could not be purified extensively, whereas the COMT activity (1.6 μkat·kg⁻¹) remained stable upon storage. The enzyme extracts were therefore subjected only to ammonium sulfate fractionation (45–60% saturation for COMT; 60–80% saturation for BMT) and subsequent desalting through PD-10 columns, enhancing the apparent specific activities to 20.2 and 10.3 μkat·kg⁻¹, respectively, for BMT and COMT. The rates of enzyme activity were compared in various buffers in pH range 2.0–10.0 and at temperatures ranging from 20 to 50 °C. Significant activity of the recombinant BMT was observed between 38 °C and 44 °C and from pH 6.5–9.0, and the optimum was recorded at 42 °C in potassium phosphate buffer pH 8.0, whereas the optimal COMT activity was observed at 32 °C in potassium phosphate buffer pH 7.0. These conditions were routinely chosen for all further assays. This activity profile of the recombinant BMT was fully compatible with the data measured for BMT extracted from *A. majus* cells (H. Ekiert and R. Lukačin, unpublished data), and the pH dependency corresponded to that of the BMTs from *P. crispum* [22] or *R. graveolens* [34]. The effect of a number of metal ions (Co²⁺, Cu²⁺, Fe²⁺, Fe³⁺, Mg²⁺, Mn²⁺, Ni²⁺, Zn²⁺) at 1.5 mM or 0.1 mM concentration was also examined. Significant inhibition of the BMT activity was observed in the presence of Cu²⁺ (100%

Table 1. Substrate specificities of *A. majus* OMTs. Substrates were used at 10 mM concentration in the assays. Neither of the OMTs accepted umbelliferone, psoralen, xanthotoxol, sinapate, 4-coumarate, 2-coumarate, 3-coumarate, catechol, kaempferol, quercetin, dihydrokaempferol, apigenin or naringenin to a significant extent (< 1%) as a substrate. The relative activity values relate to caffeate as the standard substrate.

| Substrate                      | COMT          | BMT           |
|--------------------------------|---------------|---------------|
|                                | Rel. activity (%) | $K_{\text{m}}$ | Rel. activity (%) | $K_{\text{m}}$ |
| Bergaptol                     | < 1           |               | 100             | 2.8 μMᵇ         |
| Caffeate                      | 100           | 122.0 μMᵃ     | < 1             |                 |
| 5-Hydroxyferulate             | 177           | 29.0 μM       | < 1             |                 |
| Caffeic acid methyl ester     | 207           | 42.0 μM       | < 1             |                 |
| Caffeoyl coenzyme A            | 13.5          | 219.0 μM      | < 1             |                 |
| 3-(3,4-dihydroxyphenyl)propionate | 10.4        | 2.2 mM        | < 1             |                 |
| Esculetin                     | 27            | 133.0 μM      | < 1             |                 |
| Daphnetin                     | 8             | 103.0 μM      | < 1             |                 |

ᵃ $K_{\text{m}} \text{SAM} = 2 \, \mu \text{M}$.  
ᵇ $K_{\text{m}} \text{SAM} = 6.5 \, \mu \text{M}$.

© FEBS 2004                                                                                     Bergaptol O-methyltransferase in *Ammi majus* (Eur. J. Biochem. 271) 937

| BMT-Am | ---MAEMKTS | PSQ----DEE | AGVVAMQLAT | STVLPMILKS | AIELDLLNTI | AKAGPGNYLS 53 |
| COMT-Am | MNTTTELIPP | TFQVNDEEEE | ACMFAMQLAS | ASVLPMVLKS | AIELNLLESI | AKAGPGAYVS 60 |
| COMT cons | MGSTN--ET | M T EEE | AFLFAMQLAS | ASVLPMVLK | AIELDLLEIM | AK GPGAF S |

| BMT-Am | PSDLASKLLL | SNPDAPVMLA | RILRVLATYK | VLGCKR---- | -GEVEWLYCW | TPVCKYLSNN 108 |
| COMT-Am | PSQLAAALPS | SQDTPVMLD | RILRLLASYS | VLNCKLRDLP | DARVERLYGL | APVCKFLTKN 120 |
| COMT cons | PS LAAQLPT | KNP APVMLD | RMLRLLASYS | LNCTLRLLP | DG VERLY L | APVCKFLTKN |

| BMT-Am | EDGASIAPIL | LVHQDKVTIK | SWYHLTDAVR | DGGTAFNKAH | DMSIFEYASQ | DPQFNKAFNR 168 |
| COMT-Am | SDGVSMAPLL | LMNQDKILME | SWYHLKDAVL | DGGIPFNKAY | GMTAFEYHGK | DPRFNKVFNQ 180 |
| COMT cons | EDGVSIAPLL | LMNQDKVLME | SWYHLKDAVL | DGGIPFNKAY | GMTAFEYHGT | DPRFNKVFN |

| region I | region II |
| BMT-Am | SMRGHSTITM | KKILETYKGF | EGLKSIVDVG | GGTGATLNMI | ISKYPTIKGI | NFDLPHVVGD 228 |
| COMT-Am | GMSNHSTITM | KKILQTYDGF | GGLKTVVVDG | GGTGATLNMI | ISKYPNLKGI | NFDLPHVVED 240 |
| COMT cons | GMSDHSTITM | KKILETYKGF | EGL S_VDG | GGTGAT NMI | VSKYPSIKGI | NFDLPHVIED |

| region III | region IV |
| BMT-Am | APSLPGVEHV | GGNMFASVPK | GDAIFLKWIF | HSWGDEECLK | ILKKCHQALG | DNKKVIVAEF 288 |
| COMT-Am | APSYAGVEHV | GGDMFVSVPK | GDAIFMKWIC | HDWSDAHCLA | FLKNCYKALP | KDGKVILAEC 300 |
| COMT cons | APSYPGVEHV | GGDMFVSVPK | GDAIFMKWIC | HDWSDEHCLK | FLKNCYEALP | NGKVILAEC |

| motif 2 | region V |
| BMT-Am | ILPEDPGGSD | SATKSAVHLD | AIMLAYVPGG | KERTEKEFES | LAKRAGFKSF | TKVCCAFNTW 348 |
| COMT-Am | ILPEAPD-SK | LTTKNVILID | VIMLAHNPGG | KERTEKEFEA | FGKQAGFKSF | NKACCAYNTW 359 |
| COMT cons | ILPEAPD-TS | LATKNVVHID | VIMLAHNPGG | KERTEKEFEA | LAKGAGFQGF | RKACCAYNTW |

| BMT-Am | IMEFSK 354 |
| COMT-Am | VIEYYK 365 |
| COMT cons | VMEFLKK |

Fig. 4. Alignment of AmBMT and AmCOMT polypeptides from *Ammi majus*. Hyphens were inserted for maximal alignment. The consensus sequence (COMTcons) derived from the COMT polypeptides of *Ocimum basilicum*, *Catharanthus roseus*, *Capsicum annuum*, *Capsicum chinense*, *Prunus dulcis* and *Rosa chinensis* is given in the bottom line. Identical amino acid residues are denoted by asterisks, and dots mark conservative exchanges. The amino acids are numbered in the right margin. Highly conserved regions I–V proposed as a signature of S-adenosyl-L-methionine-dependent O-methyltransferases [26,39,40] are underlined, and the motifs 1 and 2 considered to govern the substrate specificity [44,47] are in bold.

and 51%) and Ni²⁺ (47% and 16%) as well as Co²⁺ (21% and 10%). COMT activity was completely inhibited at either of the Cu²⁺ concentrations but less by Ni²⁺ (91.5% and 47%), Mn²⁺ (83% and 7.5%), or Co²⁺, Fe³⁺ and Zn²⁺ (23–30% and 0–5%). Fe²⁺ and Mg²⁺ did not affect the turnover rates. Similar results were reported for heterologous OMTs [37,38].

A variety of potential substrates, including xanthotoxol (Table 1), was employed to determine the substrate specificities of the recombinantly expressed *A. majus* BMT and COMT. However, only bergaptol was accepted as a substrate by the BMT. Kinetic assays revealed the affinities to S-adenosyl-L-methionine and bergaptol at *K*ₘ = 6.5 and 2.8 μM, respectively. The COMT was much less selective and showed the highest affinity to 5-hydroxyferulate (*K*ₘ = 29 μM) followed by caffeic acid methyl ester, caffeate, esculetin, caffeoyl-CoA, 3-(3,4-dihydroxyphenyl)propionate (dihydrocaffeate) or daphnetin (Table 1).

Relationship of sequences

S-Adenosyl-L-methionine-dependent O-methyltransferases are characterized by a common signature of five highly

938 M. Hehmann *et al.* (Eur. J. Biochem. 271)

conserved regions [39–41], and a corresponding consensus sequence was assigned from plant OMTs [26]. These elements were also recognized in both the BMT and COMT sequences from *A. majus* (Fig. 4; regions I–V) showing 94.5% and 97% identity with the consensus sequence. In case of rat liver catechol OMT regions I and IV were shown by X-ray diffraction to be involved in S-adenosyl-L-methionine and metal binding [42], and the other three regions are likely to serve the same purpose. Generally, five different structural folds have been reported to bind SAM during catalysis [43], subclassifying the OMTs, but most plant OMTs, including COMT and the homologous BMT, belong to class I. In contrast to BMTs, COMTs occur ubiquitously in plants and have been cloned from many different sources. Based on the COMTs of *Ocimum basilicum*, *Catharanthus roseus*, *Capsicum annuum*, *Capsicum chinense*, *Prunus dulcis* and *Rosa chinensis*, which share a sequence identity of about 50% and 72.3% similarity, a consensus sequence was derived also for the full size polypeptides (Fig. 4). The *AmCOMT* sequence was fully compatible with this consensus sequence showing 57% identity at 80% similarity. However, in the light of such a relationship it is particularly notable that the alignment of the *AmBMT* polypeptide with the *AmCOMT* sequence revealed 64% identity and 78.4% similarity.

### Discussion

The activities of XMT and BMT were described previously from *Ruta graveolens* [21] and *Petroselinum crispum* cells [22]. In case of dark-grown *Petroselinum* cells both activities were induced upon elicitor treatment to transient maxima at 30–35 h with *PcXMT* reaching a three- to fourfold higher value than *PcBMT* [22], whereas the constitutive BMT activity of irradiated *Ruta* cells far exceeded that of XMT activity [34]. The native enzymes purified from induced parsley cells were reported as stable homodimers of 67 kDa (XMT) and 73 kDa (BMT) being most active in potassium phosphate buffer of pH 7.5–8.0 (XMT) or of 8.0–8.5 (BMT). The native enzymes from *Ruta* appeared to be larger (85 kDa for BMT and 110 kDa for XMT [34]), and their stability differed greatly in desalted crude extracts. XMT activity was lost rapidly while the BMT activity decreased at only a moderate rate [34]. In contrast, BMT from *A. majus* was found to be a rather labile enzyme in crude extract from plant cells or after recombinant expression, which did not enable the extensive purification. Furthermore, XMT activity was induced to a negligible extent in elicited *A. majus* cells as compared to BMT. Nevertheless, the expression of *AmBMT* in *E. coli* yielded highly active extracts that revealed a molecular mass corresponding to that of bovine serum albumin (67 ± 5 kDa) on size exclusion chromatography calibrated with alcohol dehydrogenase, bovine serum albumin, ovalbumin, chymotrypsinogen A and ribonuclease A. Apparent *K*<sub>m</sub> values were determined at 2.8 μM (bergaptol) and 6.5 μM (SAM) which is in accordance with the values reported for native *PcBMT* (*K*<sub>m</sub><sub> bergaptol</sub> 4.0 μM; *K*<sub>m</sub><sub>SAM</sub> 3.1 μM) [22].

The cloning of *AmBMT* revealed a molecular mass of 38.7 kDa for the translated polypeptide, strongly suggesting a homodimer composition for the native *Ammi* enzyme as was shown previously for other OMTs [44–46]. The high degree of homology with heterologous COMTs prompted us to clone the *AmCOMT* also, which was achieved using seedlings. The alignment for these two OMT polypeptides revealed a surprisingly high degree of homology (Fig. 4). The similarity to annotated COMTs had been proposed also for the *PcBMT* [26], but, unfortunately, the relevant sequence data have not been published and cannot be compared. *AmBMT* and *AmCOMT* are regulated differently upon elicitation of *A. majus* cells, because the low level of COMT transcript abundance and activity of the cells hardly changed over the time of the experiments, whereas the *AmBMT* transcript was transiently induced to a maximum at 7 h. More importantly, despite the homology (Fig. 4) the substrate specificities of the recombinant OMTs differed greatly. The *AmBMT* exclusively methylated bergaptol to bergapten (Fig. 1), whereas the *AmCOMT* accepted several substrates apart from caffeate, with a preference for 5-hydroxyferulate (Table 1).

It is thus obvious that small sequence elements, in addition to the five highly conserved regions required for SAM binding [39,41], strongly affect the substrate specificity of OMTs. *AmBMT* is a typical member of the COMT family of enzymes which differ from the recently crystallized small-molecule methyl ester OMTs [45]. In the case of two crystallized OMTs (chalcone OMT, daidzein 7-OMT) from *Medicago sativa*, two such regions of 14 (motif I) and 11 amino acids (motif II) were identified and proposed to control the specificity [44]. From these and corresponding sequence elements in COMTs and flavonoid OMTs a consensus sequence was established (Fig. 4) and used to predict the substrate specificity of a novel OMT [47]. In summary, the studies suggested that substitutions of two to three amino acids in motifs I or II may provide a basis for the OMT classification, although a reliable prediction was not possible. This is reminiscent of the few amino acid substitutions reported for *Clarkia* OMTs to discriminate caffeate and (iso)eugenol substrates [48]. On comparison of *AmCOMT* and *AmBMT* only subtle differences in the motifs I and II were noticed due to five substitutions each (Fig. 4), most of which were conservative exchanges. However, the mutation of a single residue at other locations might considerably shift the specificity of OMTs for related substrates as has been shown for phenylpropene OMTs from sweet basil [46]. Future point mutations will reveal the relevance of these substitutions.

### Acknowledgements

Financial support by the Deutsche Forschungsgemeinschaft and Fonds der Chemischen Industrie is gratefully acknowledged. The authors are indebted to S. Specker and S. Schreiner for fruitful discussions, to B. Rohde for chromatography and to A. Batschauer and O. Panajotow (Pflanzenphysiologie und Photobiologie, Fachbereich Biologie, Universität Marburg) for their part in the subcloning experiments.

### References

1. Hamerski, D. & Matern, U. (1988) Elicitor-induced biosynthesis of psoralens in *Ammi majus* L. suspensions cultures. Microsomal conversion of demethylsuberosin into (+)-marmesin and psoralen. *Eur. J. Biochem.* **171**, 369–375.

© FEBS 2004

Bergaptol O-methyltransferase in *Ammi majus* (Eur. J. Biochem. 271) 939

2. Hamerski, D. & Matern, U. (1988) Biosynthesis of psoralens. Psoralen 5-monooxygenase activity from elicitor-treated *Ammi majus* cells. FEBS Lett. **239**, 263–265.

3. Tietjen, K.G., Hunkler, D. & Matern, U. (1983) Differential response of cultured parsley cells to elicitors from two non-pathogenic strains of fungi. 1. Identification of induced products as coumarin derivatives. Eur. J. Biochem. **131**, 401–407.

4. Wendorff, H. & Matern, U. (1986) Differential response of cultured parsley cells to elicitors from two non-pathogenic strains of fungi. Eur. J. Biochem. **161**, 391–398.

5. Hahlbrock, K. & Scheel, D. (1989) Physiology and molecular biology of phenylpropanoid metabolism. Annu. Rev. Plant Physiol. Mol. Biol. **40**, 347–369.

6. Hahlbrock, K., Scheel, D., Logemann, E., Nürnberger, T., Parniske, M., Reinold, S., Sacks, W.R. & Schmelzer, E. (1995) Oligopeptide elicitor-mediated defense gene activation in cultured parsley cells. Proc. Natl Acad. Sci. USA **92**, 4150–4157.

7. Matern, U. (1991) Coumarins and other phenylpropanoid compounds in the defense response of plant cells. Planta Med. **57**, 15–20.

8. Matern, U. & Grimmig, B. (1994) Natural phenols as stress metabolites. Natural phenols in plant resistance. Acta Hortic. **381**, 448–462.

9. Matern, U., Grimmig, B. & Kneisel, R.E. (1995) Plant cell wall reinforcement in the disease resistance response: molecular composition and regulation. Can. J. Bot. **73**, 511–517.

10. Nürnberger, T., Nennstiel, D., Jabs, T., Sacks, W.R., Hahlbrock, K. & Scheel, D. (1994) High affinity binding of a fungal oligopeptide elicitor to parsley plasma membranes triggers multiple defense responses. Cell **78**, 449–460.

11. Zangerl, A.R. & Berenbaum, M.R. (1990) Furanocoumarin induction in wild parsnip: evidence for an induced defense against herbivores. Ecology **71**, 1933–1940.

12. Dercks, W., Trumble, J. & Winter, C. (1990) Impact of atmospheric pollution alters linear furanocoumarin content in celery. J. Chem. Ecol. **16**, 443–454.

13. Zobel, A.M. & Brown, S.A. (1990) Seasonal changes of furanocoumarin concentrations in leaves of *Heracleum lanatum*. J. Chem. Ecol. **16**, 1623–1634.

14. Pathak, M.A., Parrish, J.A. & Fitzpatrick, T.B. (1981) Psoralens in photochemotherapy of skin diseases. Farmaco **36**, 479–491.

15. Honigsmann, H., Jaschke, E., Gschnait, F., Brenner, W., Fritsch, P. & Wolff, K. (1979) 5-Methoxypsoralen (bergapten) in photochemotherapy of psoriasis. Br. J. Dermatol. **101**, 369–378.

16. Hann, S.K., Cho, M.Y., Im, S. & Park, Y.K. (1991) Treatment of vitiligo with oral 5-methoxypsoralen. J. Dermatol. **18**, 324–329.

17. Hamerski, D., Beier, R.C., Kneisel, R.E., Matern, U. & Himelspach, K. (1990) Accumulation of coumarins in elicitor-treated cell suspension cultures of *Ammi majus*. Phytochemistry **29**, 1137–1142.

18. Hamerski, D., Schmitt, D. & Matern, U. (1990) Induction of two prenyltransferases for the accumulation of coumarin phytoalexins in elicitor-treated *Ammi majus* cell suspension cultures. Phytochemistry **29**, 1131–1135.

19. Murray, R.D.H., Méndez, J. & Brown, S.A. (1982) *The Natural Coumarins: Occurrence, Chemistry and Biochemistry*. Wiley, New York.

20. Stanjek, V., Miksch, M., Lüer, P., Matern, U. & Boland, W. (1999) Biosynthesis of psoralen: mechanism of a cytochrome P450 catalyzed oxidative bond cleavage. Angew. Chem. Int. Ed. **38**, 400–402.

21. Sharma, S.K., Garrett, J.M. & Brown, S.A. (1979) Separation of the S-adenosylmethionine: 5- and 8-hydroxyfuranocoumarin O-methyltransferases of *Ruta graveolens* L. by general ligand affinity chromatography. Z. Naturforsch. **34c**, 387–391.

22. Hauffe, K.D., Hahlbrock, K. & Scheel, D. (1986) Elicitor-stimulated furanocoumarin biosynthesis in cultured parsley cells: S-adenosyl-L-methionine: bergaptol and S-adenosyl-L-methionine: xanthotoxol O-methyltransferases. Z. Naturforsch. **41c**, 228–239.

23. Giuliano, G., Bartley, G.E. & Scolino, P.A. (1993) Regulation of carotenoid biosynthesis during tomato development. Plant Cell **5**, 379–387.

24. Frick, S. & Kutchan, T.M. (1999) Molecular cloning and functional expression of O-methyltransferase common to isoquinoline alkaloid and phenylpropanoid biosynthesis. Plant J. **17**, 329–339.

25. Dumas, B., van Doorsselaere, J., Gielen, J., Legrand, M., Fritig, B., van Montagu, M. & Inzé, D. (1992) Nucleotide sequence of a complementary DNA encoding O-methyltransferase from poplar. Plant Physiol. **98**, 796–797.

26. Ibrahim, R.K., Bruneau, A. & Bantignies, B. (1998) Plant O-methyltransferases: molecular analysis, common signature and classification. Plant. Mol. Biol. **36**, 1–10.

27. Sanger, F., Nicklen, S. & Coulson, A.R. (1977) DNA sequencing with chain-terminating inhibitors. Proc. Natl Acad. Sci. USA **74**, 5463–5467.

28. Junghanns, K.T., Kneuel, R.E., Baumert, A., Maier, W., Gröger, D. & Matern, U. (1995) Molecular cloning and heterologous expression of acridone synthase from elicited *Ruta graveolens* L. cell suspension cultures. Plant Mol. Biol. **27**, 681–692.

29. Lukačin, R. & Britsch, L. (1997) Identification of strictly conserved histidine and arginine residues as part of the active site in *Petunia hybrida* flavanone 3β-hydroxylase. Eur. J. Biochem. **249**, 748–757.

30. Laemmli, U.K. (1970) Cleavage of structural proteins during assembly of the head of bacteriophage T4. Nature **227**, 680–685.

31. Lowry, O.H., Rosebrough, N.J., Farr, A.L. & Randall, R.J. (1951) Protein measurements with the Folin phenol reagent. J. Biol. Chem. **193**, 265–275.

32. Sambrook, J. & Russel, D.W. (2001) *Molecular Cloning: A Laboratory Manual*. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

33. Hahlbrock, K., Lamb, C.J., Purwin, C., Ebel, J., Fautz, E. & Schäfer, E. (1981) Rapid response of suspension-cultured parsley cells to the elicitor from *Phytophthora megasperma* var. *sojae*. Plant Physiol. **67**, 768–773.

34. Thompson, H.J., Sharma, S.K. & Brown, S.A. (1978) O-Methyltransferases of furanocoumarin biosynthesis. Arch. Biochem. Biophys. **188**, 272–281.

35. Maury, S., Geoffroy, P. & Legrand, M. (1999) Tobacco O-methyltransferases involved in phenylpropanoid metabolism. The different caffeoyl-coenzyme A/5-hydroxyferuloyl-Coenzyme A 3/5-O-methyltransferase and caffeic acid/5-hydroxyferulic acid 3/5-O-methyltransferase classes have distinct substrate specificities and expression patterns. Plant Physiol. **121**, 215–223.

36. Ebel, J., Hahlbrock, K. & Grisebach, H. (1972) Purification and properties of an o-dihydricphenol meta-O-methyltransferase from cell suspension culture of parsley and its relation to flavonoid biosynthesis. Biochim. Biophys. Acta **268**, 313–326.

37. Sato, F., Tsujita, T., Katagiri, Y., Yoshida, S. & Yamada, Y. (1994) Purification and characterization of S-adenosyl-L-methionine: norcoclaurine 6-O-methyltransferase from cultured *Coptis japonica* cells. Eur. J. Biochem. **225**, 125–131.

38. Morishige, T., Tsujita, T., Yamada, Y. & Sato, F. (2000) Molecular characterization of the S-adenosyl-L-methionine: 3′-hydroxy-N-methylcoclaurine 4′-O-methyltransferase involved in isoquinoline alkaloid biosynthesis in *Coptis japonica*. J. Biol. Chem. **275**, 23398–23405.

39. Joshi, C.P. & Chiang, V.L. (1998) Conserved sequence motifs in plant S-adenosyl-L-methionine-dependent methyltransferases. Plant Mol. Biol. **37**, 663–674.

940 M. Hehmann *et al.* (Eur. J. Biochem. 271)

40. Schluckebier, G., O’Gara, M., Saenger, W. & Cheng, X. (1995) Universal catalytic domain structure of AdoMet-dependent methyltransferases. *J. Mol. Biol.* **247**, 16–20.

41. Ibrahim, R.K. (1997) Plant *O*-methyltransferase signatures. *Trends Plant Sci.* **2**, 249–250.

42. Vidgren, J., Svensson, L.A. & Liljas, A. (1994) Crystal structure of catechol *O*-methyltransferase. *Nature* **368**, 354–358.

43. Schubert, H.L., Blumenthal, R.M. & Cheng, X. (2003) Many paths to methyltransfer: a chronicle of convergence. *Trends Biochem. Sci.* **28**, 329–335.

44. Zubieta, C., He, X.-Z., Dixon, R.A. & Noel, J.P. (2001) Structures of two natural product methyltransferases reveal the basis for substrate specificity in plant *O*-methyltransferases. *Nat. Struct. Biol.* **8**, 271–279.

45. Zubieta, C., Ross, J.R., Koscheski, P., Yang, Y., Pichersky, E. & Noel, J.P. (2003) Structural basis for substrate recognition in the

salicylic acid carboxyl methyltransferase family. *Plant Cell* **15**, 1704–1716.

46. Gang, D.R., Lavid, N., Zubieta, C., Chen, F., Beuerle, T., Lewinsohn, E., Noel, J.P. & Pichersky, E. (2002) Characterization of phenylpropene *O*-methyltransferases from sweet basil: facile change of substrate specificity and convergent evolution within a plant *O*-methyltransferase family. *Plant Cell* **14**, 505–519.

47. Schröder, G., Wehinger, E. & Schröder, J. (2002) Predicting the substrates of cloned plant *O*-methyltransferases. *Phytochemistry* **59**, 1–8.

48. Wang, J. & Pichersky, E. (1999) Identification of specific residues involved in substrate discrimination in two plant *O*-methyltransferases. *Arch. Biochem. Biophys.* **368**, 172–180.
