
REVIEW

NONSENSE-MEDIATED mRNA DECAY: SPLICING, TRANSLATION AND mRNP DYNAMICS

Lynne E. Maquat

Studies of nonsense-mediated mRNA decay in mammalian cells have proffered unforeseen insights into changes in mRNA-protein interactions throughout the lifetime of an mRNA. Remarkably, mRNA acquires a complex of proteins at each exon-exon junction during pre-mRNA splicing that influences the subsequent steps of mRNA translation and nonsense-mediated mRNA decay. Complex-loaded mRNA is thought to undergo a pioneer round of translation when still bound by cap-binding proteins CBP80 and CBP20 and poly(A)-binding protein 2. The acquisition and loss of mRNA-associated proteins accompanies the transition from the pioneer round to subsequent rounds of translation, and from translational competence to substrate for nonsense-mediated mRNA decay.

PREMATURE TERMINATION CODON (PTC). A UAA, UAG or UGA codon that is located within an mRNA upstream of the normal site of translation termination. The PTC directs the premature termination of translation.

Nonsense-mediated mRNA decay (NMD), which is also known as mRNA surveillance, is one of several post-transcriptional mechanisms that are used by eukaryotic cells to control the quality of mRNA function. By eliminating abnormal transcripts that prematurely terminate translation, NMD prevents the production of truncated proteins that could function in dominant-negative or other deleterious mechanisms²⁻¹¹. NMD in mammalian cells was initially discovered in studies of β°-thalassemias that were due to PREMATURE TERMINATION CODONS (PTCs)¹²,¹³. Nevertheless, it is likely that the driving evolutionary force for NMD was not the degradation of disease-associated transcripts, but the degradation of PTC-containing transcripts that are a consequence of routine abnormalities in gene expression. For example, the advantages that are afforded by proteome diversity, which arises from the alternative splicing of more than 55% of human pre-mRNAs¹⁴, are undoubtedly accompanied by disadvantages in the form of inefficient or inaccurate intron removal during splicing. Such errors can generate a PTC that is either intron-derived or exonic, either of which might be due to a shift in the translational reading frame. NMD also

targets naturally occurring substrates, which include certain alternatively spliced mRNAs¹⁵ and some, but not all, SELENOPROTEIN mRNAs¹⁶,¹⁷. In fact, there is growing evidence that the mechanistic coupling of alternative splicing and NMD provides an often-used means of regulating gene expression (REF. 18; see also BOX 1).

The importance of NMD is evident from the finding that mouse embryos that are inactive in NMD resorb shortly after implantation¹⁹. Furthermore, blastocysts that are isolated 3.5 days post-coitum and that are inactive in NMD undergo apoptosis in culture after a brief growth period¹⁹. The lack of viability of NMD-deficient mammalian cells probably reflects a failure to regulate natural substrates properly and to eliminate transcripts that are generated in error.

In this review, I present the current conceived mechanism of NMD in mammalian cells, with the goal of describing how pre-mRNA splicing and subsequent metabolic events are vital for generating NMD-susceptible mRNA RIBONUCLEOPROTEIN PARTICLES (mRNPs).

The pioneer round of translation and splicing It is well established that NMD requires translation for NONSENSE CODON RECOGNITION. For example, NMD is

Department of Biochemistry and Biophysics,
School of Medicine and Dentistry,
601 Elmwood Avenue,
Box 712, University of Rochester, Rochester,
New York 14642, USA.
e-mail: lynne_maquat@urmc.rochester.edu
doi:10.1038/nrm1310

NATURE REVIEWS | MOLECULAR CELL BIOLOGY | VOLUME 5 | FEBRUARY 2004 | 89

©2004 Nature Publishing Group

REVIEW S

Box 1 | Functions of nonsense-mediated mRNA decay

Nonsense-mediated mRNA decay (NMD) is an mRNA quality-control mechanism that degrades abnormal mRNAs that arise as a consequence of routine mistakes in gene expression, or are produced from mutated (often disease-associated) genes. By so doing, NMD eliminates the production of the encoded truncated protein, which could function to the detriment of cells. NMD also degrades natural substrates, such as certain alternatively spliced mRNAs, certain selenoprotein mRNAs, or mRNAs that have a splicing-generated exon-exon junction that is located more than 50–55 nucleotides downstream of the translation termination codon. This undoubtedly means that NMD also functions to regulate normal gene expression.

inhibited by the antibiotics anisomycin, cycloheximide, emetine, pactamycin or puromycin, each of which binds to and inactivates translationally active ribosomes [20-22]. NMD is also inhibited by: infection with the polio virus [22], which cleaves and inactivates the eukaryotic initiation factors eIF4GI and eIF4GII, and poly (A)-binding protein (PABP) [23,24]; a secondary structure in the 5′ untranslated region that either directly or through bound protein blocks the scanning of 40S ribosomal subunits to the translation initiation codon [25,26]; and suppressor transfer RNAs, which direct the incorporation of an amino acid at PTCs and, in so doing, restore translation termination to the normally used codon [25,27].

The finding that translation reinitiation downstream of, and in frame with, a PTC also inhibits NMD [28] indicates that PTC recognition by translating ribosomes is not sufficient for NMD, and that NMD depends on events that are subsequent to translation termination and are precluded by reinitiation. These events involve the EXON JUNCTION COMPLEX (EJC) of proteins that is deposited as a consequence of pre-mRNA splicing (that is, intron removal) upstream of splicing-generated exon-exon junctions (see below).

SELENOPROTEIN mRNA  
An mRNA that has one or more UGA codons and that, together with a *cis*-residing selenocysteine insertion element, competes with the process of translation termination to direct the incorporation of the amino acid selenocysteine into the growing polypeptide chain.

mRNA RIBONUCLEOPROTEIN PARTICLE (mRNP). The composite of mRNA and associated proteins. mRNPs can affect mRNA localization, mRNA translation or mRNA half-life.

NONSENSE CODON RECOGNITION  
The process by which UAA, UAG or UGA codons direct translation termination, which is mediated by eukaryotic release factors eRF1 and eRF3.

EXON JUNCTION COMPLEX (EJC). A complex of proteins that is deposited as a consequence of pre-mRNA splicing ~20–24 nucleotides upstream of splicing-generated exon-exon junctions of newly synthesized mRNA.

mammalian-cell mRNAs that have been studied, it is not universal. mRNAs from Saccharomyces cerevisiae, Schizosaccharomyces pombe, Caenorhabditis elegans, Drosophila melanogaster and plants seem to have a different type of element that defines a nonsense codon as premature [162].

**Exceptions to the rule.** As with all rules, there are exceptions. One exception is provided by the T-cell receptor (TCR)-β transcript, which is susceptible to NMD even when the distance between a PTC within the penultimate exon and the downstream exon-exon junction is less than 50–55 nt [30]. For reasons that are unknown, the PTCs that have been tested manifest a polar effect — those PTCs that are situated more 5′ elicit NMD more efficiently than those that are situated more 3′ — and the same was found for PTCs within the second exon [30,40]. At least in *S. cerevisiae*, 5′ PTCs elicit NMD more efficiently than 3′ PTCs by more efficiently triggering the decapping activity [41]. This might also apply to TCR-β mRNA as NMD in mammalian cells also involves decapping (see below).

An explanation for what causes some TCR-β PTCs to break the ‘distance-relative-to-a-downstream-exon-exon-junction’ rule has yet to be found. However, it might be that some TCR-β mRNA sequences, other than a splicing-generated exon-exon junction, can function under certain circumstances to elicit NMD. For example, so-called fail-safe sequences within triosephosphate isomerase (TPI) and β-globin mRNAs are not exon—exon junctions, but seem to function similarly in eliciting NMD [29,31]. Deletion of the 3′-most intron from either the TPI or β-globin gene has surprisingly little effect on the efficiency with which a PTC within the penultimate exon elicits NMD. Notably, however, the deletion of all introns precludes NMD [29,31].

Another exception to the rule is exemplified by PTCs within β-globin exon 1 that unexpectedly fail to elicit NMD, even though intron 1 is more than 50–55 nt downstream [42]. As a PTC at position 1 (that is, the codon that is immediately downstream of the AUG initiation codon) of TPI mRNA does elicit NMD (once the AUG codon at position 14 is mutated so that it does not function as an efficient site of translation reinitiation), early PTCs can inherently elicit NMD and, in fact, peptide-bond formation is not required for NMD [28]. It is possible that early PTCs within β-globin mRNA fail to elicit NMD because there is a specific sequence that confers immunity to NMD and that resides downstream of the PTC — this has been reported for the general control nonrepressible (GCN4) and yeast AP-1 (YAP1) mRNAs of *S. cerevisiae* [43]. Another case of a PTC within β-globin mRNA that unexplainably fails to elicit NMD was generated within exon 2 by a single nucleotide change in intron 1. The nucleotide change induced aberrant splicing and, in so doing, a shift in the translational reading frame [44].

Notably, the distances between a PTC and a downstream exon-exon junction that are >5 kilobases (kb) can support NMD [38,45,46] (Y. Ishigaki and L.E.M., unpublished observations). The largest distance that was

90 | FEBRUARY 2004 | VOLUME 5
www.nature.com/reviews/molcellbio
©2004 Nature Publishing Group

MAGOH and, depending on experimental conditions, the acute myeloid leukaemia-associated protein (DEK)⁴⁹⁻⁶¹. In vitro, REF/ALY seems to interact with pre-mRNA before spliceosome assembly, whereas RNPS1, SRm160, Y14, UAP56 and MAGOH associate with splicing intermediates⁶⁰. On exon ligation, the binding of RNPS1, UAP56 and SRm160 to spliced product is relatively unstable compared with REF/ALY, Y14 and MAGOH, which indicates that the latter three proteins comprise the EJC core⁶⁰.

Co-immunoprecipitation studies of RNP that is formed in cultured mammalian cells from endogenous cellular proteins have shown that further components of the EJC are the NMD factors **UPF2**, **UPF3** (which is also known as UPF3A) and **UPF3X** (which is also known as UPF3B), as well as **NXF1/TAP** (nuclear export factor 1/tyrosine-kinase-interacting protein-associated protein)³⁶,⁶². However, DEK was not detected³⁶, which is in contrast to some studies of pre-mRNA splicing in which nuclear extracts were used⁶⁰. DEK was also not detected as an EJC component when expressed in HeLa cells as an epitope-tagged protein⁶³, which supports the idea that DEK is not a bona fide component of EJCs *in vivo*. SRm160, REF/ALY, Y14, RNPS1, UPF3 or UPF3X, UPF2 and NXF1/TAP (UAP56 and MAGOH were not assayed) were detected as components of the EJC in both nuclear and cytoplasmic fractions of monkey kidney COS cells, indicating that the EJC and associated UPF NMD factors are exported in association with at least some mRNAs to the cytoplasm³⁶ (see below). There are data that indicate that Y14 is removed by translating ribosomes⁶⁴; this might also hold true for other EJC proteins.

SRm160 and RNPS1 function as co-activators of splicing⁵⁰,⁵¹, whereas REF/ALY and Y14 seem to recruit the mRNA-export heterodimer NXF1/TAP–p15 (REFS 57,65–69). REF/ALY also interacts with UAP56, a putative helicase and splicing factor that functions in mRNA export, at least in cultured *D. melanogaster* cells and isolated *Xenopus laevis* oocytes⁵⁸,⁵⁹,⁷⁰. Y14 and MAGOH form a heterodimer that is required for NMD⁷¹⁻⁷³, as well as for the proper localization of oskar mRNA during *D. melanogaster* development²⁰⁷,⁷⁵. Nevertheless, despite the appealing idea that EJCs facilitate the export of newly synthesized nuclear mRNA to the cytoplasm⁶⁰,⁷⁶, other data indicate that this is not the case⁶⁹,⁷⁷⁻⁷⁹. It is now thought that EJCs might have only a minor role in export compared with *cis*-residing exonic sequences that can also recruit NXF1/TAP⁸⁰,⁸¹, which possibly reflects what seems to be a loose (that is, RNase-sensitive) association of NXF1/TAP with the EJC³⁶. In addition to its role in recruiting NMD factors, the EJC has been reported to upregulate the level of gene expression by enhancing RNA 3′-end formation and mRNA translation⁷⁹.

Studies with mammalian cells indicate that the EJC component Y14 and, to a lesser extent, RNPS1 provide a demonstrable connection between intron position within pre-mRNA and NMD. First, tethering Y14 or, to a lesser extent, RNPS1 to the 3′ untranslated region of β-globin mRNA recapitulates the NMD function of a

Figure 1 | NMD and the ‘position-of-an-exon-exon-junction’ rule. Only the 3′-most exon-exon junction within a generic mammalian mRNA is shown. Mammalian mRNAs generally have an average of 7–8 splicing-generated exon-exon junctions¹⁵⁹,¹⁶⁰. A premature termination codon (PTC) that is located in the region indicated in blue, which is followed by an exon-exon junction more than 50–55 nucleotides (nt) downstream, elicits nonsense-mediated mRNA decay (NMD), whereas a PTC that is located in the region indicated in green fails to elicit NMD. The normal termination codon (Ter) usually resides within the 3′-most exon³⁷.

examined typifies apolipoprotein B (apoB) mRNA. apoB transcripts undergo C-TO-U EDITING, which results in the conversion of a CAA glutamine codon to a UAA PTC in exon 26, so that the resulting protein (apoB48) is only 48% of the full-length protein (apoB100) that derives from unedited mRNA. Remarkably, edited *apoB* mRNA is immune to NMD, not because of the unusually large distance between the PTC and downstream exon-exon junction, but because of a complex that consists of the dimer that is formed by apoB mRNA editing catalytic polypeptide1 (APOBEC1) and the RNA-binding protein APOBEC1 complementation factor (ACF). APOBEC1-ACF binds to a so-called mooring sequence that is 6 nt downstream of the editing site and functions as a CYTIDINE DEAMINASE⁴⁶. Notably, the specific positioning of the editing-generated PTC relative to bound APOBEC1-ACF, rather than the process of editing *per se*, is crucial for immunity to NMD⁴⁶. Consistent with a requirement for specific positioning, PTCs that are either upstream or downstream of the edited codon fail to confer immunity to NMD⁴⁷,⁴⁸.

C-TO-U EDITING  
A post-transcriptional process that involves the deamination of a cytidine (C) nucleotide to a uridine (U) nucleotide within pre-mRNA that, in the case of apolipoprotein B transcripts, converts a glutamine codon (CAA) to a termination codon (UAA).

APOBEC1-ACF  
Apolipoprotein B mRNA editing catalytic polypeptide 1 (APOBEC1) in complex with the RNA-binding protein APOBEC1 complementation factor (ACF). APOBEC1-ACF is required for the C-to-U editing of apolipoprotein B transcripts.

CYTIDINE DEAMINASES  
A family of enzymes, one member of which is the 27-kDa apolipoprotein B mRNA editing catalytic polypeptide 1 (APOBEC1), that catalyse the C-to-U editing of apolipoprotein pre-mRNA.

The role of the exon junction complex. An explanation for how the introns in a pre-mRNA (that is, exon-exon junctions within mRNA) function in NMD comes from the findings that pre-mRNA splicing deposits the EJC ~20–24 nt upstream of presumably every exon-exon junction in spliced mRNA, and that EJCs go on to recruit up-frameshift (UPF) proteins, which are required for NMD (see below). Each EJC that assembles during pre-mRNA splicing in HeLa-cell nuclear extracts protects only 8–10 nt of spliced product from RNase digestion⁴⁹. Co-immunoprecipitation studies of RNase-resistant mRNPs or mass spectrometric and western blotting analyses of purified mRNPs indicate that the EJC that is generated by splicing *in vitro* is ~335 kDa and consists minimally of **RNPS1** (RNA-binding protein prevalent during S phase), **SRm160** (serine/arginine-related matrix protein of 160 kDa), **REF** (RNA and export factor; also known as ALY), **Y14** (so-called because 14 cDNAs were obtained in a yeast two-hybrid screen using a portion of RanBP5 as bait), **UAP56** (U2 snRNP auxiliary factor of 65 kDa associated protein of 56 kDa), the mago nashi homologue

Box 2 | The pioneer round of translation

The first round of translation involves mRNA that is bound by the cap-binding protein (CBP) heterodimer CBP80–CBP20 at the 5′ cap, an exon junction complex (EJC) that is upstream of each splicing-generated exon–exon junction (which recruits nonsense-mediated mRNA decay (NMD) factors UPF2 and UPF3 or UPF3X) and the poly(A)-binding protein PABP2 at the 3′ poly(A) tail<sup>36,62</sup> (FIG. 2). At present it is not known whether one ribosome or more than one ribosome translates mRNA during the pioneer round. The pioneer translation initiation complex is ultimately remodelled to the steady-state initiation complex, which involves mRNA bound by eukaryotic initiation factor eIF4E at the 5′ cap and PABP1 at the 3′ poly(A) tail<sup>90</sup>. Notably, EJCs and associated NMD factors are removed by the time eIF4E binds the cap<sup>36,62</sup>, which is consistent with the finding that NMD is restricted to the pioneer round of translation.

---

as well as ribosomal protein 10 and 80S ribosomes, as would be expected if it were translated<sup>36,62</sup> (S.-Y. Chiu and L.E.M., unpublished observations; A. Ranganathan, M. Coldwell, L. McKendrick, S. Morley and L.E.M., unpublished observations).

### Nucleus-associated and cytoplasmic NMD

Notably, most mammalian mRNAs that have been studied so far are subject to nucleus-associated NMD<sup>26,30–32,34</sup>, whereas a minority are subject to cytoplasmic NMD<sup>16</sup>. The nuclear export of CBP80–CBP20, the EJC, UPF2 and UPF3 or UPF3X in association with at least some mRNAs is consistent with data that show that the NMD of some mRNAs takes place as a consequence of translation by cytoplasmic ribosomes. This is certainly the case for mRNAs that are subject to cytoplasmic NMD and, possibly, is also true for mRNAs that are subject to nucleus-associated NMD (see below). As yet, the cellular site of nucleus-associated NMD remains enigmatic.

In theory, mRNAs that are subject to nucleus-associated NMD could be degraded in the nucleoplasm, which indicates that they might be either translated by nuclear ribosomes or subject to feedback after being translated in the cytoplasm<sup>92,93</sup>. Alternatively, nucleus-associated NMD could involve translation by cytoplasmic ribosomes and decay that take place during or immediately after transport across the nuclear pore complex but before release from nuclei into the cytoplasm<sup>92,94</sup>. There are reports that argue for<sup>95–98</sup> and against<sup>99–104</sup> the possibility of nuclear translation, and only time will tell which is true. Notably, there is evidence that an mRNA can, in fact, be simultaneously exported from nuclei and translated by cytoplasmic ribosomes. Indeed, this has been shown for the 35–40-kb BALBIANI RING mRNA of the insect *Chironomus tentans*, which is sufficiently large to be readily visualized using electron microscopy<sup>105</sup>.

### Factors and modifiers of NMD

The three mammalian UPF proteins are named after their orthologues in *S. cerevisiae* (BOX 3). In addition to UPF2 and UPF3 or UPF3X, which are known to be required for NMD<sup>101,106</sup> (Y. K. Kim, X. Li and L.E.M., unpublished observations), UPF1, which is an RNA-dependent ATPase and 5′-to-3′ helicase, is also needed for NMD<sup>106–109</sup>. Unlike the other UPF proteins, UPF1 is not detected in association with mammalian-cell mRNPs<sup>62</sup>, which is consistent with the finding that an anti-UPF1 antibody does not immunoprecipitate spliced mRNA generated from pre-mRNA that has been injected into the nuclei of *X. laevis* oocytes<sup>68</sup>.

In most cell types that have been examined, UPF1 is primarily cytoplasmic but also shuttles to and from nuclei, UPF2 is cytoplasmic and is concentrated on the cytoplasmic side of the nuclear envelope, and UPF3 and UPF3X are primarily nuclear but shuttle to and from the cytoplasm<sup>82,106,110–113</sup>. Whereas UPF1 and UPF2 are phosphoproteins<sup>109,112</sup>, the phosphorylation status of either UPF3 or UPF3X remains to be characterized. UPF1 phosphorylation by SMG1, a

---

BALBIANI RING mRNA  
A 35–40-kilobase mRNA in the insect *Chironomus tentans* that, as shown by electron microscopy studies, is exported from nuclei to the cytoplasm 5′-end first, and becomes associated with cytoplasmic ribosomes before nuclear export is complete.

Box 3 | Identification of Upf and SMG NMD factors

Factors that are involved in nonsense-mediated mRNA decay (NMD) were initially described in genetic studies of *Saccharomyces cerevisiae* and *Caenorhabditis elegans*.

**Yeast**

Factors that are known as Upf (for up-frame shift) were obtained in a strain containing the *his4-38* allele, which results in a +1 frameshift within the *HIS4* transcript that generates a premature termination codon (PTC) and, consequently, elicits NMD ${}^{144,145}$. The +1 frameshift is attributable to a single nucleotide insertion. *upf* alleles were selected in a strain that contained not only the *his4-38* allele but also the *SUF1-1* gene, which encodes a transfer-RNA frameshift suppressor that decodes the four-nucleotide codon. By so doing, the suppressor allows the synthesis of the full-length HIS4 protein and a His${}^+$ phenotype at $30^\circ$C. Upf mutations were obtained by selecting for a His${}^+$ phenotype at $37^\circ$C that was the result of abrogating NMD, which increased the abundance of *his4-38* mRNA, to compensate for the temperature-sensitive decreased suppression.

**Worms**

Factors that are known as SMG (for suppressor with morphogenic effect on genitalia) were isolated as extragenic suppressors of *unc-54(r293)* ${}^{146}$. The *unc-54* allele encodes a mutated myosin heavy chain B (MHC B) that is expressed in body-wall muscles. *r293* denotes the type of mutation: a 256-base-pair deletion within the allele that encompasses the 3′ cleavage/polyadenylation site as well as most of the 3′ untranslated region. The motility of *unc-54(r293)* mutants indicates that they express a small, but measurable, amount of functional MHC B. Mutations in six loci (*smg1–6*) were found to be allele-specific recessive suppressors of *unc-54(r293)* as well as other alleles, so that they caused abnormal morphogenesis of the male bursa and the hermaphrodite vulva. Suppression could be assayed by an increase in motility that correlated with an increase in the amount of *unc-54(r293)* mRNA that was due to an abrogation of NMD ${}^{147}$. As *unc-54(r293)* mRNA is mutated downstream of the normally used translation termination codon so as to acquire new sequences within the 3′ untranslated region, it is likely that these new sequences include a *cis*-acting determinant that functions together with the normal termination codon to elicit NMD. These data indicate that pre-mRNA splicing is not crucial for NMD in *C. elegans*, as is also true for NMD in *S. cerevisiae*. Notably, whereas all three characterized *S. cerevisiae* Upf factors have SMG counterparts in *C. elegans* (Upf1 is orthologous to SMG2, Upf2 to SMG3, and Upf3 to SMG4), SMG1, SMG5, SMG6 and SMG7 have no Upf counterpart. A modified screen was used to identify SMG7, which led to the conclusion that SMG7 is required for NMD only at elevated temperatures ${}^{148}$.

**Mammals**

Factors in mammalian cells (named after their *S. cerevisiae* or, if non-existent, *C. elegans* counterparts) were identified using *in silico* approaches. When necessary, these methods relied on related but uncharacterized sequences from other species to identify potentially functional amino acids ${}^{112,118}$. Mammalian cells have counterparts to all SMG factors and, therefore, all Upf factors.

phosphatidylinositol-kinase-related protein kinase that was named after its orthologue in *C. elegans* ${}^{114}$ (BOX 3), is vital for NMD ${}^{115,116}$. Likewise, UPF1 dephosphorylation by SMG6 and, probably, **SMG5** and **SMG7**, which are also named after their *C. elegans* orthologues ${}^{114,117}$, is also required for NMD ${}^{112,118}$ (*SMG5, SMG6* and *SMG7* were initially known as *SMG5/7B, SMG5/7A* and *SMG5/7C*, respectively, because the *C. elegans* SMG5, SMG6 and SMG7 sequences were not available at the time ${}^{112}$). As SMG6 interacts with protein phosphatase PP2A ${}^{112}$, as does SMG5 in *C. elegans* ${}^{117}$, it is likely that all three mammalian SMG proteins direct PP2A to dephosphorylate UPF1. UPF1 is primarily hypophosphorylated when isolated from whole cells and primarily hyperphosphorylated when isolated in association with polysomes ${}^{109}$.

**NMD factors and translation.** Although studies in *S. cerevisiae* indicate that, first, Upf1 interacts with EUKARYOTIC RELEASE FACTOR (eRF). eRF1 and eRF3 function in translation termination at the A site of the 80S ribosome: eRF1 recognizes all three termination codons, and eRF3 functions as a ribosome-dependent GTPase that helps eRF1 to release the newly synthesized polypeptide. EUKARYOTIC RELEASE FACTORS eRF1 and eRF3, and that, second, Upf2 and Upf3 interact with eRF3 (REFS 119, 120), it is unclear at present whether Upf protein function in translation termination reflects their function in termination accuracy or efficiency ${}^{121}$. Notably, there is no evidence that mammalian UPF1 functions in any aspect of termination: an association with eRFs has not been found ${}^{122}$ (Y.K. Kim and L.E.M., unpublished observations), and downregulation of either UPF1 or UPF2 does not increase UAA, UAG or UGA suppression, at least in the codon context that was assayed ${}^{106}$. Therefore, the idea that has been put forward for *S. cerevisiae* that, first, Upf1 functions in association with ribosomes in translation termination and then, second, switches to a mode in which it functions in NMD — in mammals this would involve binding to EJC-bound UPF2 — lacks direct proof.

**A model for NMD.** At present, considering the available data, the link between translation and decay during NMD is best explained as follows (FIG. 2). UPF3 or UPF3X join the EJC in the nucleus, and UPF2 and UPF1 associate with the EJC either before or during export to the cytoplasm (that is, before or as soon as mRNA has access to the cytoplasm) in the case of nucleus-associated NMD, or after export to the cytoplasm in the case of cytoplasmic NMD. The association of UPF1 with an EJC must somehow be triggered by translation termination, provided that termination occurs sufficiently upstream of an EJC. It is reasonable to assume that UPF1 is the last of the UPF proteins to join the EJC because, in addition to not being detected

as a stable component of mRNPs (in contrast to UPF2,  
UPF3 and UPF3X ${}^{36,62}$), a helicase-deficient UPF1 protein impairs NMD in trans when UPF2, UPF3 or  
UPF3X, but not UPF1, is tethered downstream of a termination codon ${}^{82}$. So, UPF1 probably functions in  

NMD as part of a post-termination complex. Considering that there are $\sim 4 \times 10^6$ UPF1 molecules per  
HeLa cell (that is, about one copy for every three ribosomes), which is around tenfold more abundant than  
either UPF2 or the sum of UPF3 and UPF3X ${}^{123}$, the  

---

**Figure 2 | Pre-mRNA splicing, the pioneer round of translation and steady-state translation.** Pre-mRNA, which consists of exons (pink boxes) and introns (black lines between boxes), is bound by the cap-binding proteins CBP80 and CBP20 at the 5′ cap and, after 3′-end formation, poly(A)-binding protein PABP2 at the 3′ poly(A) tail ${}^{36}$. Pre-mRNA processing generates spliced mRNA that is likewise bound by CBP80, CBP20 and PABP2, as well as an exon junction complex (EJ C) of proteins ~20–24 nucleotides (nt) upstream of each exon-exon junction. This EJ C consists minimally of RNPS1, SRm160 and UAP56, as well as Y14, REF/ALY and NXF1/TAP-p15 (REFS 36,60). The EJ C acquires further proteins, including UPF2 and UPF3 or UPF3X, which function in nonsense-mediated mRNA decay (NMD) ${}^{36,83,161}$. UPF3 or UPF3X, which is mostly nuclear but shuttles to the cytoplasm, is thought to recruit UPF2, which concentrates along the cytoplasmic side of the nuclear envelope ${}^{82,111}$. The resulting messenger ribonucleoprotein particle (mRNP) constitutes the pioneer translation initiation complex ${}^{36,62}$. This complex is thought to undergo a pioneer round of translation either in association with nuclei, in the case of mRNAs that are subject to nucleus-associated NMD (a), or in the cytoplasm, in the case of mRNAs that are subject to cytoplasmic NMD (b). If the mRNP lacks a premature termination codon (PTC) or has a PTC that fails to elicit NMD, then the mRNP is remodelled to the steady-state translation initiation complex. During remodelling, the EJ C and associated UPF proteins are removed, CBP80 and CBP20 are replaced by eukaryotic initiation factor eIF4E, and PABP2 is replaced by PABP1 (REF. 36). Whether translation is required for all steps of mRNP remodelling is unclear. So far, translation has been reported to remove Y14 (REF. 64).

EXOSOME  
A complex of at least 11 3′-to-5′  
exonucleases that functions in  
nuclei and the cytoplasm in  
several different RNA-processing  
and RNA-degradation pathways.

DEADENYLASE  
An enzyme that functions to  
remove the 3′ poly(A) tail from  
RNA in a 3′-to-5′ direction.

recruitment of UPF1 is not likely to limit NMD. Rather,  
there is probably not enough UPF3 and UPF3X to bind  
to every EJC<sup>4</sup>, which might explain, at least in part, why  
NMD is less than 100% efficient: for example, only  
75–90% of PTC-containing β-globin or TPI mRNA is  
generally subject to NMD<sup>26,31,32</sup>. Nevertheless, there are  
sequences within the coding region of TCR-β mRNA  
that, by an unknown mechanism, enhance the efficiency  
with which downstream PTCs elicit NMD so that NMD  
is almost 100% efficient<sup>124</sup>.

**NMD factors and other functions.** Notably, factors that  
are involved in NMD function in other pathways,  
although how they do so is not yet understood. For  
example, SMG1 seems to function not only in NMD,  
which is viewed as a type of RNA surveillance, but also  
in genome surveillance. SMG1 phosphorylates not only  
UPF1 but also the cell-cycle checkpoint protein p53  
(K.B. Brumbaugh et al., unpublished observations).  
Moreover, ionizing or UV B radiation upregulates p53  
and UPF1 phosphorylation, and cells deficient in SMG1  
are not only hypersensitive to radiation but also accumulate in G2/M and undergo apoptotic death, which  
indicates that they develop spontaneous DNA damage  
(K.B. Brumbaugh et al., unpublished observations).  
Interestingly, UPF1 has been identified as the δ helicase  
that co-purifies with DNA polymerase δ in fetal bovine  
thymus<sup>125</sup>. This helicase functions 5′ to 3′ and has a strong  
preference for a fork-like DNA substrate<sup>126</sup>. Consistent

![Diagram](attachment://diagram.png)

Figure 3 | **NMD degrades mRNAs from both ends.** Nonsense-mediated mRNA decay (NMD)  
is initiated during a pioneer round of translation when a premature termination codon (PTC)  
resides more than 50–55 nucleotides (nt) upstream of an exon-exon junction that is bound by an  
exon junction complex (EJC), and EJC-associated UPF2 and UPF3/UPF3X proteins as well as  
UPF1, which does not stably associate with mRNA<sup>62</sup>, trigger NMD. The triggering mechanism  
remains to be resolved, but it presumably involves one or more components of the translation  
termination complex. Data indicate that NMD initiates from the mRNA 5′ end, where it involves  
decapping followed by 5′-to-3′ exonucleolytic decay, and from the mRNA 3′ end, where it  
involves deadenylation followed by 3′-to-5′ exonucleolytic decay<sup>84</sup>. At present it is not possible to  
determine the relative efficiencies of 5′-to-3′ and 3′-to-5′ decay during NMD.

with a role in DNA synthesis and RNA metabolism,  
UPF1 has two putative zinc-finger motifs and one putative zinc-knuckle motif that are often found in proteins  
that function as both DNA- and RNA-binding proteins. However, the possibility that UPF1 functions in DNA  
synthesis will have to be reconciled with its primarily  
cytoplasmic location in most mammalian cells.

In addition, studies in *C. elegans*<sup>127</sup> have shown that  
orthologues of UPF1, SMG5 and SMG6 might be  
required for the persistence of RNAi. SMG6 (which is  
also known as EST1A) possibly provides another example of this, as it associates with telomerase and, when  
overexpressed, promotes chromosomal end-to-end  
fusion within telomeric DNA<sup>128</sup>. However, SMG6 might  
not modulate telomere structure directly. In support of  
this possibility, downregulating NMD in *S. cerevisiae*  
upregulates the steady-state level of mRNAs that  
encode telomere components and regulators, including  
Est1 (REF. 129).

NMD involves mRNA decay from both ends  
In *S. cerevisiae*, NMD has long been known to trigger  
rapid deadenylation-independent decapping followed  
by 5′-to-3′ exonucleolytic decay<sup>130-132</sup>. More recently,  
a role for deadenylation followed by EXOSOME-mediated  
3′-to-5′ decay has become evident<sup>131,133</sup>. Data now indicate that NMD in mammalian cells also involves decapping followed by 5′-to-3′ decay, as well as deadenylation  
followed by 3′-to-5′ decay<sup>84</sup> (FIG. 3; see also BOX 4).

First, downregulating the decapping protein DCP2,  
the RNase-D-like exosome component PM/SCL100, or  
the cytoplasmic DEADENYLASE poly(A) ribonuclease PARN  
abrogates nucleus-associated NMD of β-globin mRNA  
and cytoplasmic NMD of glutathione peroxidase 1  
mRNA<sup>84</sup>. Abrogation was shown as an increase in the  
level of steady-state PTC-containing mRNA, as well as a  
decrease in the decay rate of newly synthesized PTC-  
containing mRNA that was made during a transcriptional pulse (the downregulation of PARN was not  
analysed). Downregulating the 5′-to-3′ exonuclease  
XRN1 was also shown to abrogate the NMD of TCR-β  
mRNA (J.T. Mendell and H.C. Dietz, personal communication).

Second, NMD factors UPF1, UPF2 and UPF3X co-  
immunopurify not only with DCP2, PM/SCL100 and  
PARN, but also with RAT1 and XRN1 (which are putative and proven 5′-to-3′ exonucleases, respectively), and  
the ribosomal RNA-processing RRP4 and RRP41 components of the exosome<sup>84</sup>. In addition, UPF1 co-  
immunopurifies with DCP1 and DCP2 (REF. 134), and  
two-hybrid analysis using all or part of UPF2 as bait  
revealed interactions with DCP2, XRN2, exosomal  
components PM/SCL100, RRP43 and mRNA transport  
regulator MTR3, and the decapping activator LSM1  
(B. Lehner and C.M. Sanderson, personal communication). A homologue of *S. cerevisiae* Ccr4 protein might  
also function as a deadenylase in NMD, because this  
homologue co-purifies with UPF1 (E. Wagner and J.  
Lykke-Andersen, personal communication). All of  
these data indicate that NMD degrades mRNA from  
both 5′ and 3′ ends, possibly but not assuredly on the

Box 4 | mRNA degradative activities in mammalian cells

**Decapping**

Decapping is mediated by decapping protein DCP2 (REFS, 134, 136, 149), which binds RNA as a prerequisite for cap recognition<sup>150</sup>. The intrinsic decapping activity of DCP2 is augmented by DCP1 (REF. 151) and several SM-LIKE LSM PROTEINS, all of which colocalize in distinct cytoplasmic foci with the 5′-to-3′ exonuclease XRN1 (REFS 135, 136). Whether or not decapping and decay of the mRNA body occur within these foci remains to be shown.

**5′-to-3′ exonuclease activity**

XRN1 is a proven 5′-to-3′ exonuclease that localizes to the cytoplasm<sup>152</sup>. RAT 1/XRN2 is only thought to be a 5′-to-3′ exonuclease on the basis of its similarity to the yeast orthologue. It localizes to the nuclear fraction in biochemical fractionations and can be detected around the nuclear envelope using indirect immunofluorescence<sup>84,153</sup>.

**Deadenylation**

A main cytoplasmic deadenylase in mammalian cells seems to be poly(A) ribonuclease<sup>154,155</sup> (PARN), and PARN is known to function in NMD<sup>84</sup>. PARN is one of five mammalian homologues to yeast Caf1/Pop2 protein (E. Wagner and J. Lykke-Andersen, personal communication). In addition, five homologues to yeast Ccr4 protein have been identified, one of which might function in nonsense-mediated mRNA decay (NMD) as it co-purifies with UPF1 (E. Wagner and J. Lykke-Andersen, personal communication).

**3′-to-5′ exonuclease activity**

The exosome consists minimally of: six RNase-PH-DOMAIN components, PM/SC L75, MTR3, RRP41, RRP42, RRP43 and RRP46; three S1 and KH RNA-binding components, RRP4, RRP40 and CSL4; the RNASE D-like components PM/SC L100; the putative helicase KIAA0053; and a protein that is phosphorylated in the M phase of the cell cycle<sup>156,157</sup>. Although it is possible that different nucleases of the same type might manifest substrate specificities, it is clear that at least some can target the same mRNA for NMD (F. Lejeune, X. Li and L.E.M., unpublished data).

**PMR1-like activity**

Polysomal ribonuclease 1 (PMR1) is a polysome-associated mRNA endonuclease that was originally found to mediate the oestrogen-stimulated generalized degradation of most serum protein mRNAs in Xenopus at AYUGA elements<sup>158</sup>. It turns out that premature termination codon (PTC)-containing β-globin mRNA in erythroid cells is degraded by a PMR1-like activity<sup>141,142</sup> that might supplement or supersede the usual NMD pathway, which involves decapping and 5′-to-3′ exonucleolytic decay as well as deadenylation and 3′-to-5′ exonucleolytic decay<sup>84</sup>.

---

whether or not it is produced in a transcriptional pulse, is restricted to the nuclear fraction and involves decay from both mRNA ends<sup>26,32,84</sup>.

Interestingly, there is a notable distinction between the decay of PTC-harbouring β-globin mRNA in non-erythroid cells and in erythroid cells. The erythroid tissues of mice that are transgenic for one of several β°-thalassemic β-globin alleles, or mouse erythroleukaemic cells that are stably transfected with one of several β°-thalassemic β-globin alleles, produce readily detectable β-globin mRNA decay intermediates that are polyadenylated but lack regions of the 5′ end<sup>138–141</sup>. Data indicate that the 5′ ends of the decay intermediates are first generated endonucleolytically, preferentially at UG dinucleotides, by a polysome-associated activity that is similar to *X. laevis* PMR1 (REFS 141, 142), and are subsequently capped<sup>139</sup>. Capping presumably confers resistance to 5′-to-3′ decay. As endonucleolytic cleavage generally typifies mRNA-specific pathways, and as NMD is not an mRNA-specific pathway, the UG-site-selective PMR1-like activity that generates detectable β-globin mRNA decay intermediates in erythroid cells, although dependent on the recognition of a PTC during translation<sup>141,142</sup>, might be superimposed on or, possibly, supersede the general pathway of NMD (BOX 4).

**Implications for the future**

Studies of NMD in mammalian cells have uncovered remarkable links between pre-mRNA splicing and the pioneer round of translation — processes that typify every intron-containing transcript, whether a PTC is present or not. Just as pre-mRNA splicing influences the structure of mRNPs, so does the pioneer round of translation. Future studies will undoubtedly identify more completely the factors that constitute the pioneer initiation complex, and how and when these factors are removed relative to steady-state translation. There is no doubt that our knowledge of the pioneer round of translation and NMD at present is far from complete. For example, unpublished data from the laboratories of G. Dreyfuss, E. Izaurralde, M. Moore, R. Reed and N. Sonenberg show that eIF4AIII is a component of the EJC and is required for NMD (N. Sonenberg, personal communication). eIF4AIII has been shown to have RNA-dependent ATPase activity and ATP-dependent RNA helicase activity, as does the eIF4AI isoform<sup>143</sup>. However, eIF4AIII fails to substitute for eIF4AI in an *in vitro*-reconstituted 40S ribosome binding assay, which indicates that it does not function in bulk cytoplasmic translation<sup>143</sup>. It will be important to determine whether eIF4AIII is required for the pioneer round of translation or the PTC-dependent decay process that follows. In future studies, it will also be important to quantify the translational efficiency of CBP80-bound mRNA relative to eIF4E-bound mRNA and, of course, to determine if the pioneer round of translation in the case of nucleus-associated NMD occurs within nuclei or during mRNA export from nuclei to the cytoplasm.

Once a PTC is recognized by the translational machinery, NMD will be triggered by a splicing-generated

same molecule, by recruiting decapping and 5′-to-3′ exonuclease activities, as well as deadenylating and 3′-to-5′ exonuclease activities.

Whether NMD takes place in the cytoplasmic foci, which were shown in HeLa cells to consist of DCP1, DCP2, XRN1 and the decapping activators LSM1–7 (REFS 135, 136), remains to be determined. However, it is reasonable to think that NMD involves the loss and acquisition of mRNP proteins that allow mRNA susceptibility to decay and, possibly, catalyse mRNA decay *per se*. Notably, the failure to detect partially or completely deadenylated decay intermediates indicates that either 3′-to-5′ decay is minor relative to 5′-to-3′ decay, or the rate of deadenylation and subsequent decay of the mRNA body are fast enough to preclude the accumulation of decay intermediates<sup>84</sup>. By contrast, cytoplasmic deadenylated intermediates were reported as a necessary first step in the NMD of β-globin mRNA that is produced in a transcriptional pulse<sup>137</sup>. Although this is consistent with the involvement of PARN, it is unclear at present how this report differs from previous work that shows that the NMD of β-globin mRNA in non-erythroid cells,

---

SM-LIKE LSM PROTEIN  
A subunit of a heptameric complex that functions in RNA metabolism. LSM2–8 functions in pre-mRNA splicing in nuclei, and LSM1–7 functions in mRNA decay in the cytoplasm.

PH DOMAIN  
A protein domain that is characteristic of the RNase PH family of bacterial phosphate-dependent ribonucleases.

S1 DOMAIN  
An RNA-binding domain that is characteristic of the small ribosomal subunit protein S1.

KH DOMAIN  
An RNA-binding domain that typifies hnRNP K (hnRNP K homology).

RNASE D DOMAIN  
A protein domain that is  
characteristic of bacterial  
RNase D.

exon-exon junction that is situated more than 50–55 nt  
downstream, provided that it is associated with an EJC  
as well as UPF2 and UPF3 or UPF3X proteins. mRNPs  
are thought to undergo further structural rearrangements before or during decay, which might direct their  
localization to decay foci, and it will be important to  

understand these transitions. Future studies will also  
undoubtedly reveal how UPF proteins function to  
recruit decapping, deadenylating and exonucleolytic  
activities that are already known to be involved in NMD  
and, possibly, other degradative activities that have yet  
to be tested for a role in NMD.

1. Maquat, L. E. & Carmichael, G. G. Quality control of mRNA  
   function. *Cell* **104**, 173–176 (2001).  
2. Maquat, L. E. When cells stop making sense: effects of  
   nonsense codons on RNA metabolism in vertebrate cells.  
   *RNA* **1**, 453–465 (1995).  
3. Maquat, L. E. in *Translational Control of Gene Expression*  
   (eds. Sonenberg, N., Hershey, J. W. B. & Mathews, M. B.)  
   849–868 (Cold Spring Harbor Press, New York, 2000).  
4. Arraiano, C. M. & Maquat, L. E. Post-transcriptional control  
   of gene expression: effectors of mRNA decay. *Mol.*  
   *Microbiol.* **49**, 267–276 (2003).  
5. Peltz, S. W. & Jacobson, A. in *Translational Control of Gene*  
   *Expression* (eds. Sonenberg, N., Hershey, J. W. B. &  
   Mathews, M. B.) 827–847 (Cold Spring Harbor Press,  
   New York, 2000).  
6. Li, S. & Wilkinson, M. F. Nonsense surveillance in  
   lymphocytes? *Immunity* **8**, 135–141 (1998).  
7. Frischmeyer, P. A. & Dietz, H. C. Nonsense-mediated mRNA  
   decay in health and disease. *Hum. Mol. Genet.* **8**,  
   1893–1900 (1999).  
8. Hentze, M. W. & Kulokiz, A. E. A perfect message: RNA  
   surveillance and nonsense-mediated decay. *Cell* **96**,  
   307–310 (1999).  
9. Hilleren, P. & Parker, R. Mechanisms of mRNA surveillance in  
   eukaryotes. *Annu. Rev. Genet.* **33**, 229–260 (1999).  
10. Wagner, E. & Lykke-Andersen, J. mRNA surveillance: the  
    perfect persist. *J. Cell. Sci.* **115**, 3033–3038 (2002).  
11. Culbertson, M. R. & Leeds, P. F. Looking at mRNA decay  
    pathways through the window of molecular evolution.  
    *Curr. Opin. Genet. Dev.* **13**, 207–214 (2003).  
12. Maquat, L. E., Kinniburgh, A. J., Rachmilewitz, E. A. &  
    Ross, J. Unstable β-globin mRNA in mRNA-deficient  
    β°-thalassemia. *Cell* **27**, 543–553 (1981).  
13. Kinniburgh, A. J., Maquat, L. E., Schedl, T., Rachmilewitz, E.  
    & Ross, J. mRNA-deficient β-thalassemia results from a  
    single nucleotide deletion. *Nucleic Acids Res.* **10**,  
    5421–5427 (1982).  
14. Kan, Z., Rouchka, E. C., Gish, W. R. & States, D. J. Gene  
    structure prediction and alternative splicing analysis using  
    genomically aligned ESTs. *Genome Res.* **11**, 889–900  
    (2001).  
15. Morrison, M., Harris, K. S. & Roth, M. B. *smg* mutants affect  
    the expression of alternatively spliced SR protein mRNAs in  
    *Caenorhabditis elegans*. *Proc. Natl Acad. Sci. USA* **94**,  
    9782–9785 (1997).  
16. Moriarty, P. M., Reddy, C. C. & Maquat, L. E. Selenium  
    deficiency reduces the abundance of mRNA for  
    Se-dependent glutathione peroxidase 1 by a UGA-dependent  
    mechanism likely to be nonsense codon-mediated decay of  
    cytoplasmic mRNA. *Mol. Cell. Biol.* **18**, 2932–2939 (1998).  
    Describes a natural target for cytoplasmic NMD.  
17. Sun, X. *et al.* Nonsense-mediated decay of mRNA for the  
    selenoprotein phospholipid hydroperoxide glutathione  
    peroxidase is detectable in cultured cells but masked or  
    inhibited in rat tissues. *Mol. Biol. Cell* **12**, 1009–1017  
    (2001).  
18. Lewis, B. P., Green, R. E. & Brenner, S. E. Evidence for the  
    widespread coupling of alternative splicing and nonsense-  
    mediated mRNA decay in humans. *Proc. Natl Acad. Sci.*  
    *USA* **100**, 189–192 (2003).  
    Provides evidence for widespread use of NMD as a  
    means of regulating gene expression.  
19. Medghalchi, S. M. *et al.* Rent1, a trans-effector of nonsense-  
    mediated mRNA decay, is essential for mammalian  
    embryonic viability. *Hum. Mol. Genet.* **10**, 99–105 (2001).  
20. Qian, L. *et al.* T cell receptor-β mRNA splicing: regulation of  
    unusual splicing intermediates. *Mol. Cell. Biol.* **13**,  
    1686–1696 (1993).  
21. Menon, K. P. & Neufeld, E. F. Evidence for degradation of  
    mRNA encoding α-L-iduronidase in Hurler fibroblasts with  
    premature termination alleles. *Cell. Mol. Biol.* **40**, 999–1005  
    (1994).  
22. Carter, M. S. *et al.* A regulatory mechanism that detects  
    premature nonsense codons in T-cell receptor transcripts  
    in vivo is reversed by protein synthesis inhibitors *in vitro*.  
    *J. Biol. Chem.* **270**, 28995–29003 (1995).  

23. Gradi, A., Svitkin, Y. V., Imataka, H. & Sonenberg, N.  
    Proteolysis of human eukaryotic translation initiation factor  
    eIF4GI, but not eIF4GII, coincides with the shutoff of host  
    protein synthesis after poliovirus infection. *Proc. Natl Acad.*  
    *Sci. USA* **95**, 11089–11094 (1998).  
24. Kuyumcu-Martinez, N. M., Joachims, M. & Lloyd, R. E.  
    Efficient cleavage of ribosome-associated poly(A)-binding  
    protein by enterovirus 3C protease. *J. Virol.* **76**, 2062–2074  
    (2002).  
25. Belgrader, P., Cheng, J. & Maquat, L. E. Evidence to  
    implicate translation by ribosomes in the mechanism by  
    which nonsense codons reduce the nuclear level of human  
    triosephosphate isomerase mRNA. *Proc. Natl Acad. Sci.*  
    *USA* **90**, 482–486 (1993).  
26. Thermann, R. *et al.* Binary specification of nonsense codons  
    by splicing and cytoplasmic translation. *EMBO J.* **17**,  
    3484–3494 (1998).  
27. Li, S., Leonard, D. & Wilkinson, M. F. T cell receptor (TCR)  
    mini-gene mRNA expression regulated by nonsense  
    codons: a nuclear-associated translation-like mechanism.  
    *J. Exp. Med.* **185**, 985–992 (1997).  
28. Zhang, J. & Maquat, L. E. Evidence that translation  
    reinitiation abrogates nonsense-mediated mRNA decay in  
    mammalian cells. *EMBO J.* **16**, 826–833 (1997).  
29. Cheng, J., Belgrader, P., Zhou, X. & Maquat, L. E. Introns are  
    cis effectors of the nonsense-codon-mediated reduction in  
    nuclear mRNA abundance. *Mol. Cell. Biol.* **14**, 6317–6325  
    (1994).  
30. Carter, M. S., Li, S. & Wilkinson, M. F. A splicing-dependent  
    regulatory mechanism that detects translation signals.  
    *EMBO J.* **15**, 5965–5975 (1996).  
31. Zhang, J., Sun, X., Qian, Y., LaDuca, J. P. & Maquat, L. E.  
    At least one intron is required for the nonsense-mediated  
    decay of triosephosphate isomerase mRNA: a possible link  
    between nuclear splicing and cytoplasmic translation.  
    *Mol. Cell. Biol.* **18**, 5272–5283 (1998).  
32. Zhang, J., Sun, X., Qian, Y. & Maquat, L. E. Intron function in  
    the nonsense-mediated decay of β-globin mRNA:  
    indications that pre-mRNA splicing in the nucleus can  
    influence mRNA translation in the cytoplasm. *RNA* **4**,  
    801–815 (1998).  
33. Sun, X., Moriarty, P. M. & Maquat, L. E. Nonsense-mediated  
    decay of glutathione peroxidase 1 mRNA in the cytoplasm  
    depends on intron position. *EMBO J.* **19**, 4734–4744 (2000).  
34. Cheng, J. & Maquat, L. E. Nonsense codons can reduce  
    the abundance of nuclear mRNA without affecting the  
    abundance of pre-mRNA or the half-life of cytoplasmic  
    mRNA. *Mol. Cell. Biol.* **13**, 1892–1902 (1993).  
35. Belgrader, P., Cheng, J., Zhou, X., Stephenson, L. S. &  
    Maquat, L. E. Mammalian nonsense codons can be cis  
    effectors of nuclear mRNA half-life. *Mol. Cell. Biol.* **14**,  
    8219–8228 (1994).  
    Shows that nucleus-associated NMD targets newly  
    synthesized mRNA.  
36. Lejeune, F., Ishigaki, Y., Li, X. & Maquat, L. E. The exon  
    junction complex is detected on CBP80-bound but  
    not eIF4E-bound mRNA in mammalian cells: dynamics  
    of mRNP remodeling. *EMBO J.* **21**, 3536–3545 (2002).  
    Characterizes the exon junction complex that is  
    formed *in vivo*.  
37. Nagy, E. & Maquat, L. E. A rule for termination-codon  
    position within intron-containing genes: when nonsense  
    affects RNA abundance. *Trends Biochem. Sci.* **23**, 198–199  
    (1998).  
    Establishes a rule for which PTCs elicit NMD.  
38. Maquat, L. E. & Li, X. Mammalian heat shock p70 and  
    histone H4 transcripts, which derive from naturally intronless  
    genes, are immune to nonsense-mediated decay. *RNA* **7**,  
    445–456 (2001).  
39. Brocke, K. S., Neu-Yilik, G., Gehring, N. H., Hentze, M. W. &  
    Kulokiz, A. E. The human intronless melanocortin 4-receptor  
    gene is NMD insensitive. *Hum. Mol. Genet.* **11**, 331–335  
    (2002).  
40. Wang, J., Gudikote, J. P., Olivas, O. R. & Wilkinson, M. F.  
    Boundary-independent polar nonsense-mediated decay.  
    *EMBO Rep.* **3**, 274–279 (2002).  

41. Cao, D. & Parker, R. Computational modeling and  
    experimental analysis of nonsense-mediated decay in yeast.  
    *Cell* **113**, 533–545 (2003).  
42. Romao, L. *et al.* Nonsense mutations in the human β-globin  
    gene lead to unexpected levels of cytoplasmic mRNA  
    accumulation. *Blood* **96**, 2895–2901 (2000).  
43. Ruiz-Echevarria, M. J. & Peltz, S. W. The RNA binding  
    protein Pub1 modulates the stability of transcripts  
    containing upstream open reading frames. *Cell* **101**,  
    741–751 (2000).  
44. Danckwardt, S. *et al.* Abnormally spliced β-globin mRNAs: a  
    single point mutation generates transcripts sensitive and  
    insensitive to nonsense-mediated mRNA decay. *Blood* **99**,  
    1811–1816 (2002).  
45. Neu-Yilik, G. *et al.* Splicing and 3′end formation in the  
    definition of nonsense-mediated decay-competent human  
    β-globin mRNPs. *EMBO J.* **20**, 532–540 (2001).  
46. Chester, A. *et al.* The apolipoprotein B mRNA editing  
    complex performs a multifunctional cycle and suppresses  
    nonsense-mediated decay. *EMBO J.* **22**, 3971–3982  
    (2003).  
    Describes the mechanism by which edited apoB  
    mRNA is immune to NMD.  
47. Homanics, G. E. *et al.* Targeted modification of the  
    apolipoprotein B gene results in hypobetalipoproteinemia  
    and developmental abnormalities in mice. *Proc. Natl Acad.*  
    *Sci. USA* **90**, 2389–2393 (1993).  
48. Kim, E., Ambroziak, P., Veniant, M. M., Hamilton, R. L. &  
    Young, S. G. A gene-targeted mouse model for familial  
    hypobetalipoproteinemia. Low levels of apolipoprotein B  
    mRNA in association with a nonsense mutation in exon 26  
    of the apolipoprotein B gene. *J. Biol. Chem.* **273**,  
    33977–33984 (1998).  
49. Le Hir, H., Izaurralde, E., Maquat, L. E. & Moore, M. J. The  
    spliceosome deposits multiple proteins 20–24 nucleotides  
    upstream of mRNA exon-exon junctions. *EMBO J.* **19**,  
    6860–6869 (2000).  
50. Blencowe, B. J., Issner, R., Nickerson, J. A. & Sharp, P. A.  
    A coactivator of pre-mRNA splicing. *Genes Dev.* **12**,  
    996–1009 (1998).  
51. Mayeda, A. *et al.* Purification and characterization of human  
    RNPS1: a general activator of pre-mRNA splicing. *EMBO J.*  
    **18**, 4560–4570 (1999).  
52. Kataoka, N. *et al.* Pre-mRNA splicing imprints mRNA in the  
    nucleus with a novel RNA-binding protein that persists in the  
    cytoplasm. *Mol. Cell* **6**, 673–682 (2000).  
53. Kataoka, N., Diem, M. D., Kim, V. N., Yong, J. & Dreyfuss, G.  
    Magoh, a human homolog of *Drosophila mago nashi*  
    protein, is a component of the splicing-dependent  
    exon-exon junction complex. *EMBO J.* **20**, 6424–6433  
    (2001).  
54. Le Hir, H., Moore, M. J. & Maquat, L. E. Pre-mRNA splicing  
    alters mRNP composition: evidence for stable association of  
    proteins at exon-exon junctions. *Genes Dev.* **14**,  
    1098–1108 (2000).  
55. Le Hir, H., Gatfield, D., Braun, I. C., Forler, D. & Izaurralde, E.  
    The protein Mago provides a link between splicing  
    and mRNA localization. *EMBO Rep.* **2**, 1119–1124  
    (2001).  
    Provides an initial characterization of components of  
    the exon junction complex formed *in vitro*.  
56. McGarvey, T. *et al.* The acute myeloid leukemia-associated  
    protein, DEK, forms a splicing-dependent interaction with  
    exon-product complexes. *J. Cell Biol.* **150**, 309–320  
    (2000).  
57. Zhou, Z. *et al.* The protein Aly links pre-messenger-RNA  
    splicing to nuclear export in metazoans. *Nature* **407**,  
    401–405 (2000).  
58. Gatfield, D. *et al.* The DExH/D box protein HEL/UAP56 is  
    essential for mRNA nuclear export in *Drosophila*. *Curr. Biol.*  
    **11**, 1716–1721 (2001).  
59. Luo, M. L. *et al.* Pre-mRNA splicing and mRNA export linked  
    by direct interactions between UAP56 and Aly. *Nature* **413**,  
    644–647 (2001).

NATURE REVIEWS | MOLECULAR CELL BIOLOGY

VOLUME 5 | FEBRUARY 2004 | 97

©2004 Nature Publishing Group

REVIEW S

60. Reichert, V. L., Le Hir, H., Jurica, M. S. & Moore, M. J. 5′ exon interactions within the human spliceosome establish a framework for exon junction complex structure and assembly. *Genes Dev.* **16**, 2778–2791 (2002).

61. Kim, V. N. *et al.* The Y14 protein communicates to the cytoplasm the position of exon-exon junctions. *EMBO J.* **20**, 2062–2068 (2001).

62. Ishigaki, Y., Li, X., Serin, G. & Maquat, L. E. Evidence for a pioneer round of mRNA translation: mRNAs subject to nonsense-mediated decay in mammalian cells are bound by CBP80 and CBP20. *Cell* **106**, 607–617 (2001). Shows that NMD targets CBP80-bound mRNA during a pioneer round of translation.

63. Lykke-Andersen, J., Shu, M. D. & Steitz, J. A. Communication of the position of exon-exon junctions to the mRNA surveillance machinery by the protein RNPS1. *Science* **293**, 1836–1839 (2001).

64. Dostie, J. & Dreyfuss, G. Translation is required to remove Y14 from mRNAs in the cytoplasm. *Curr. Biol.* **12**, 1060–1067 (2002). Provides evidence that translating ribosomes remove a component of the exon junction complex.

65. Luo, M. J. & Reed, R. Splicing is required for rapid and efficient mRNA export in metazoans. *Proc. Natl Acad. Sci. USA* **96**, 14937–14942 (1999).

66. Katahira, J. *et al.* The Mex67p-mediated nuclear mRNA export pathway is conserved from yeast to human. *EMBO J.* **18**, 2593–2609 (1999).

67. Bach, A. *et al.* The C-terminal domain of TAP interacts with the nuclear pore complex and promotes export of specific CTE-bearing RNA substrates. *RNA* **6**, 136–158 (2000).

68. Le Hir, H., Gatfield, D., Izaurralde, E. & Moore, M. J. The exon-exon junction complex provides a binding platform for factors involved in mRNA export and nonsense-mediated mRNA decay. *EMBO J.* **20**, 4987–4997 (2001).

69. Rodrigues, J. P. *et al.* REF proteins mediate the export of spliced and unspliced mRNAs from the nucleus. *Proc. Natl Acad. Sci. USA* **98**, 1030–1035 (2001).

70. Strasser, K. & Hurt, E. Yra1p, a conserved nuclear RNA-binding protein, interacts directly with Mex67p and is required for mRNA export. *EMBO J.* **19**, 410–420 (2000).

71. Lau, C. K., Diem, M. D., Dreyfuss, G. & Van Duyne, G. D. Structure of the y14-magoH core of the exon junction complex. *Curr. Biol.* **13**, 933–941 (2003).

72. Shi, H. & Xu, R. M. Crystal structure of the *Drosophila* Mago nashi-Y14 complex. *Genes Dev.* **17**, 971–976 (2003).

73. Fribourg, S., Gatfield, D., Izaurralde, E. & Conti, E. A novel mode of RBD-protein recognition in the Y14-Mago complex. *Nature Struct. Biol.* **10**, 433–439 (2003).

74. Hachet, O. & Ephrussi, A. *Drosophila* Y14 shuttles to the posterior of the oocyte and is required for oskar mRNA transport. *Curr. Biol.* **11**, 1666–1674 (2001).

75. Mohr, S. E., Dillon, S. T. & Boswell, R. E. The RNA-binding protein Tsunagi interacts with Mago Nashi to establish polarity and localize oskar mRNA during *Drosophila* oogenesis. *Genes Dev.* **15**, 2886–2899 (2001).

76. Reed, R. & Hurt, E. A conserved mRNA export machinery coupled to pre-mRNA splicing. *Cell* **108**, 523–531 (2002).

77. Gatfield, D. & Izaurralde, E. REF1/Aly and the additional exon junction complex proteins are dispensable for nuclear mRNA export. *J. Cell Biol.* **159**, 579–588 (2002).

78. Nott, A., Meislin, S. H. & Moore, M. J. A quantitative analysis of intron effects on mammalian gene expression. *RNA* **9**, 607–617 (2003).

79. Wiegand, H. L., Lu, S. & Cullen, B. R. Exon junction complexes mediate the enhancing effect of splicing on mRNA expression. *Proc. Natl Acad. Sci. USA* **100**, 11327–11332 (2003).

80. Huang, Y. & Steitz, J. A. Splicing factors SRp20 and 9G8 promote the nucleocytoplasmic export of mRNA. *Mol. Cell* **7**, 899–905 (2001).

81. Huang, Y., Gattoni, R., Stevenin, J. & Steitz, J. A. SR splicing factors serve as adapter proteins for TAP-dependent mRNA export. *Mol. Cell* **11**, 837–843 (2003).

82. Lykke-Andersen, J., Shu, M. D. & Steitz, J. A. Human Upf proteins target an mRNA for nonsense-mediated decay when bound downstream of a termination codon. *Cell* **103**, 1121–1131 (2000). Demonstrates that tethering any one of the UPF proteins downstream of a normal termination codon is sufficient to elicit NMD.

83. Gehring, N. H., Neu-Yilik, G., Schell, T., Hentze, M. W. & Kulokiz, A. E. Y14 and hUpf3b form an NMD-activating complex. *Mol. Cell* **11**, 939–949 (2003). Illustrates a functional difference between UPF3 (UPF3A) and UPF3X (UPF3B) as well as the importance of Y14 to NMD.

84. Lejeune, F., Li, X. & Maquat, L. E. Nonsense-mediated mRNA decay in mammalian cells involves decapping,

deadenylating, and exonucleolytic activities. *Mol. Cell* **12**, 675–687 (2003).

Provides an initial characterization of the enzymology of NMD in mammalian cells.

85. Izaurralde, E. *et al.* A nuclear cap binding protein complex involved in pre-mRNA splicing. *Cell* **78**, 657–668 (1994).

86. Lewis, J. D. & Izaurralde, E. The role of the cap structure in RNA processing and nuclear export. *Eur. J. Biochem.* **247**, 461–469 (1997).

87. Visa, N., Izaurralde, E., Ferreira, J., Daneholt, B. & Mattaj, I. W. A nuclear cap-binding complex binds Balbiani ring pre-mRNA cotranscriptionally and accompanies the ribonucleoprotein particle during nuclear export. *J. Cell Biol.* **133**, 5–14 (1996).

88. Shen, E. C., Stage-Zimmermann, T., Chui, P. & Silver, P. A. The yeast mRNA-binding protein Npl3p interacts with the cap-binding complex. *J. Biol. Chem.* **275**, 23718–23724 (2000).

89. Lejbkowicz, F. *et al.* A fraction of the mRNA 5′ cap-binding protein, eukaryotic initiation factor 4E, localizes to the nucleus. *Proc. Natl Acad. Sci. USA* **89**, 9612–9616 (1992).

90. Gingras, A. C., Raught, B. & Sonenberg, N. eIF4 initiation factors: effectors of mRNA recruitment to ribosomes and regulators of translation. *Annu. Rev. Biochem.* **68**, 913–963 (1999).

91. Dostie, J., Lejbkowicz, F. & Sonenberg, N. Nuclear eukaryotic initiation factor 4E (eIF4E) colocalizes with splicing factors in speckles. *J. Cell Biol.* **148**, 239–247 (2000).

92. Maquat, L. E. NAS ty effects on fibrillin pre-mRNA splicing: another case of ESE does it, but proposals for translation-dependent splice site choice live on. *Genes Dev.* **16**, 1743–1753 (2002).

93. Wang, J., Chang, Y. F., Hamilton, J. I. & Wilkinson, M. F. Nonsense-associated altered splicing: a frame-dependent response distinct from nonsense-mediated decay. *Mol. Cell* **10**, 951–957 (2002).

94. Dahlberg, J. E., Lund, E. & Goodwin, E. B. Nuclear translation: what is the evidence? *RNA* **9**, 1–8 (2003). Evaluates the possibility of translation within nuclei, which was re-established with the discovery of NMD.

95. Iborra, F. J., Jackson, D. A. & Cook, P. R. Coupled transcription and translation within nuclei of mammalian cells. *Science* **293**, 1139–1142 (2001).

96. Muhlemann, O. *et al.* Precursor RNAs harboring nonsense codons accumulate near the site of transcription. *Mol. Cell* **8**, 33–43 (2001).

97. Buhler, M., Wilkinson, M. F. & Muhlemann, O. Intranuclear degradation of nonsense codon-containing mRNA. *EMBO Rep.* **3**, 646–651 (2002).

98. Brogna, S., Sato, T. A. & Rosbash, M. Ribosome components are associated with sites of transcription. *Mol. Cell* **10**, 93–104 (2002).

99. Bohnsack, M. T. *et al.* Exp5 exports eEF1A via tRNA from nuclei and synergizes with other transport pathways to confine translation to the cytoplasm. *EMBO J.* **21**, 6205–6215 (2002).

100. Calado, A., Treichel, N., Muller, E. C., Otto, A. & Kutay, U. Exportin-5-mediated nuclear export of eukaryotic elongation factor 1A and tRNA. *EMBO J.* **21**, 6216–6224 (2002).

101. Wang, J., Hamilton, J. I., Carter, M. S., Li, S. & Wilkinson, M. F. Alternatively spliced TCR mRNA induced by disruption of reading frame. *Science* **297**, 108–110 (2002).

102. Nathanson, L., Xia, T. & Deutscher, M. P. Nuclear protein synthesis: a re-evaluation. *RNA* **9**, 9–13 (2003).

103. Cosson, B. & Philippe, M. Looking for nuclear translation using Xenopus oocytes. *Biol. Cell* **95**, 321–325 (2003).

104. Trotta, C. R., Lund, E., Kahan, L., Johnson, A. W. & Dahlberg, J. E. Coordinated nuclear export of 60S ribosomal subunits and NMD3 in vertebrates. *EMBO J.* **22**, 2841–2851 (2003).

105. Visa, N. *et al.* A pre-mRNA-binding protein accompanies the RNA from the gene through the nuclear pores and into polysomes. *Cell* **84**, 253–264 (1996).

106. Mendell, J. T., Ap Rhys, C. M. & Dietz, H. C. Separable roles for rent1/hUpf1 in altered splicing and decay of nonsense transcripts. *Science* **298**, 419–422 (2002).

107. Sun, X., Perlick, H. A., Dietz, H. C. & Maquat, L. E. A mutated human homologue to yeast Upf1 protein has a dominant-negative effect on the decay of nonsense-containing mRNAs in mammalian cells. *Proc. Natl Acad. Sci. USA* **95**, 10009–10014 (1998).

108. Bhattacharya, A. *et al.* Characterization of the biochemical properties of the human Upf1 gene product that is involved in nonsense-mediated mRNA decay. *RNA* **6**, 1226–1235 (2000).

109. Pal, M., Ishigaki, Y., Nagy, E. & Maquat, L. E. Evidence that phosphorylation of human Upf1 protein varies with intracellular location and is mediated by a wortmannin-sensitive and rapamycin-sensitive PI3-kinase-related kinase signaling pathway. *RNA* **7**, 5–15 (2001).

110. Applequist, S. E., Selg, M., Raman, C. & Jack, H. M. Cloning and characterization of HUPF1, a human homolog of the *Saccharomyces cerevisiae* nonsense mRNA-reducing UPF1 protein. *Nucleic Acids Res.* **25**, 814–821 (1997).

111. Serin, G., Gersappe, A., Black, J. D., Aronoff, R. & Maquat, L. E. Identification and characterization of human orthologues to *Saccharomyces cerevisiae* Upf2 protein and Upf3 protein (*Caenorhabditis elegans* SMG-4). *Mol. Cell. Biol.* **21**, 209–223 (2001).

112. Chiu, S.-Y., Serin, G., Ohara, O. and Maquat, L. E. Characterization of human Smg5/7a: a protein with similarities to *C. elegans* SMG5 and SMG7 that functions in the dephosphorylation of Upf1. *RNA* **9**, 77–87 (2003).

113. Mendell, J. T., Medghalchi, S. M., Lake, R. G., Noensie, E. N. & Dietz, H. C. Novel Upf2p orthologues suggest a functional link between translation initiation and nonsense surveillance complexes. *Mol. Cell. Biol.* **20**, 8944–8957 (2000).

114. Page, M. F., Carr, B., Anders, K. R., Grimson, A. & Anderson, P. SMG-2 is a phosphorylated protein required for mRNA surveillance in *Caenorhabditis elegans* and related to Upf1p of yeast. *Mol. Cell. Biol.* **19**, 5943–5951 (1999). Characterizes the function of SMG in NMD.

115. Denning, G., Jameson, L., Maquat, L. E., Thompson, E. A. & Fields, A. P. Cloning of a novel phosphatidylinositol kinase-related kinase: characterization of the human SMG-1 RNA surveillance protein. *J. Biol. Chem.* **276**, 22709–22714 (2001).

116. Yamashita, A., Ohnishi, T., Kashima, I., Taya, Y. & Ohno, S. Human SMG-1, a novel phosphatidylinositol 3-kinase-related protein kinase, associates with components of the mRNA surveillance complex and is involved in the regulation of nonsense-mediated mRNA decay. *Genes Dev.* **15**, 2215–2228 (2001).

117. Anders, K. R., Grimson, A. & Anderson, P. SMG-5, required for *C. elegans* nonsense-mediated mRNA decay, associates with SMG-2 and protein phosphatase 2A. *EMBO J.* **22**, 641–650 (2003).

118. Gatfield, D., Unterholzner, L., Ciccarelli, F. D., Bork, P. & Izaurralde, E. Nonsense-mediated mRNA decay in *Drosophila*: at the intersection of the yeast and mammalian pathways. *EMBO J.* **22**, 3960–3970 (2003).

119. Czaplinski, K. *et al.* The surveillance complex interacts with the translation release factors to enhance termination and degrade aberrant mRNAs. *Genes Dev.* **12**, 1665–1677 (1998).

120. Wang, W., Czaplinski, K., Rao, Y. & Peltz, S. W. The role of Upf proteins in modulating the translation read-through of nonsense-containing transcripts. *EMBO J.* **20**, 880–890 (2001).

121. Bidou, L. *et al.* Nonsense-mediated decay mutants do not affect programmed-1 frameshifting. *RNA* **6**, 952–961 (2000).

122. Schell, T. *et al.* Complexes between the nonsense-mediated mRNA decay pathway factor human upf1 (up-framsht protein 1) and essential nonsense-mediated mRNA decay factors in HeLa cells. *Biochem. J.* **373**, 775–783 (2003).

123. Maquat, L. E. & Serin, G. Nonsense-mediated mRNA decay: insights into mechanism from the cellular abundance of human Upf1, Upf2, Upf3, and Upf3X proteins. *Cold Spring Harb. Symp. Quant. Biol.* **66**, 313–320 (2001).

124. Gudikote, J. P. & Wilkinson, M. F. T-cell receptor sequences that elicit strong down-regulation of premature termination codon-bearing transcripts. *EMBO J.* **21**, 125–134 (2002).

125. Carastro, L. M. *et al.* Identification of δ-helicase as the bovine homolog of HUPF1: demonstration of an interaction with the third subunit of DNA polymerase δ. *Nucleic Acids Res.* **30**, 2232–2243 (2002).

126. Li, X., Tan, C. K., So, A. G. & Downey, K. M. Purification and characterization of δ-helicase from fetal calf thymus. *Biochemistry* **31**, 3507–3513 (1992).

127. Domeier, M. E. *et al.* A link between RNA interference and nonsense-mediated decay in *Caenorhabditis elegans*. *Science* **289**, 1928–1931 (2000).

128. Reichenbach, P. *et al.* A human homolog of yeast est1 associates with telomerase and uncaps chromosome ends when overexpressed. *Curr. Biol.* **13**, 568–574 (2003).

129. Dahlseid, J. N. *et al.* mRNAs encoding telomerase components and regulators are controlled by UPF genes in *Saccharomyces cerevisiae*. *Eukaryot. Cell* **2**, 134–142 (2003).

130. Hagan, K. W., Ruiz-Echevarria, M. J., Quan, Y. & Peltz, S. W. Characterization of *cis*-acting sequences and decay intermediates involved in nonsense-mediated mRNA turnover. *Mol. Cell. Biol.* **15**, 809–823 (1995).

131. Muhlrad, D. & Parker, R. Premature translational termination triggers mRNA decapping. *Nature* **370**, 578–581 (1994).

132. Muhlrad, D. & Parker, R. Aberrant mRNAs with extended 3′ UTRs are substrates for rapid degradation by mRNA surveillance. *RNA* **5**, 1299–1307 (1999).

133. Mitchell, P. & Tollervey, D. An NMD pathway in yeast involving accelerated deadenylation and exosome-mediated 3′→5′ degradation. *Mol. Cell* **11**, 1405–1413 (2003).

134. Lykke-Andersen, J. Identification of a human decapping complex associated with hUpf proteins in nonsense-mediated decay. *Mol. Cell. Biol.* **22**, 8114–8121 (2002).

135. Ingelfinger, D., Arndt-Jovin, D. J., Lührmann, R. & Achsel, T. The human LSM1–7 proteins colocalize with the mRNA-degrading enzymes Dcp1/2 and Xrn1 in distinct cytoplasmic foci. *RNA* **8**, 1489–1501 (2002).

136. Van Dijk, E. *et al.* Human Dcp2: a catalytically active mRNA decapping enzyme located in specific cytoplasmic structures. *EMBO J.* **21**, 6915–6924 (2002).

137. Chen, C. Y. & Shyu, A. B. Rapid deadenylation triggered by a nonsense codon precedes decay of the RNA body in a mammalian cytoplasmic nonsense-mediated decay pathway. *Mol. Cell. Biol.* **23**, 4805–4813 (2003).

138. Lim, S., Mullins, J. J., Chen, C. M., Gross, K. W. & Maquat, L. E. Novel metabolism of several β°-thalassemic β-globin mRNAs in the erythroid tissues of transgenic mice. *EMBO J.* **8**, 2613–2619 (1989).

139. Lim, S. K. & Maquat, L. E. Human β-globin mRNAs that harbor a nonsense codon are degraded in murine erythroid tissues to intermediates lacking regions of exon I or exons I and II that have a cap-like structure at the 5′ termini. *EMBO J.* **11**, 3271–3278 (1992).

140. Lim, S. K., Sigmund, C. D., Gross, K. W. & Maquat, L. E. Nonsense codons in human β-globin mRNA result in the production of mRNA degradation products. *Mol. Cell. Biol.* **12**, 1149–1161 (1992).

141. Stevens, A. *et al.* β-globin mRNA decay in erythroid cells: UG site-preferred endonucleolytic cleavage that is augmented by a premature termination codon. *Proc. Natl Acad. Sci. USA* **99**, 12741–12746 (2002). Characterizes an endonuclease that degrades β-globin mRNA with a PTC in erythroid cells.

142. Bremer, K. A., Stevens, A. & Schoenberg, D. R. An endonuclease activity similar to Xenopus PMR1 catalyzes the degradation of normal and nonsense-containing human β-globin mRNA in erythroid cells. *RNA* **9**, 1157–1167 (2003).

143. Li, Q. *et al.* Eukaryotic translation initiation factor 4AⅢ (eIF4AⅢ) is functionally distinct from eIF4AI and eIF4AII. *Mol. Cell. Biol.* **19**, 7336–7346 (1999).

144. Leeds, P., Peltz, S. W., Jacobson, A. & Culbertson, M. R. The product of the yeast *UPF1* gene is required for rapid turnover of mRNAs containing a premature translational termination codon. *Genes Dev.* **5**, 2303–2314 (1991). Provides initial characterization of a factor that is required for NMD.

145. Leeds, P., Wood, J. M., Lee, B. S. & Culbertson, M. R. Gene products that promote mRNA turnover in *Saccharomyces cerevisiae*. *Mol. Cell. Biol.* **12**, 2165–2177 (1992).

146. Hodgkin, J., Papp, A., Pulak, R., Ambros, V. & Anderson, P. A new kind of informational suppression in the nematode *Caenorhabditis elegans*. *Genetics* **133**, 301–313 (1989).

147. Pulak, R. & Anderson, P. mRNA surveillance by the *Caenorhabditis elegans smg* genes. *Genes Dev.* **7**, 1885–1897 (1993).

148. Cali, B. M., Kuchma, S. L., Latham, J. & Anderson, P. *smg-7* is required for mRNA surveillance in *Caenorhabditis elegans*. *Genetics* **151**, 605–616 (1999).

149. Wang, Z., Jiao, X., Carr-Schmid, A. & Kiledjian, M. From the cover: the hDcp2 protein is a mammalian mRNA decapping enzyme. *Proc. Natl Acad. Sci. USA* **99**, 12663–12668 (2002).

150. Piccirillo, C., Khanna, R. & Kiledjian, M. Functional characterization of the mammalian mRNA decapping enzyme hDcp2. *RNA* **9**, 1138–1147 (2003).

151. Decker, C. J. & Parker, R. mRNA decay enzymes: decappers conserved between yeast and mammals. *Proc. Natl Acad. Sci. USA* **99**, 12512–12514 (2002).

152. Bashkirov, V. I., Scherthan, H., Solinger, J. A., Buerstedde, J. M. & Heyer, W. D. A mouse cytoplasmic exoribonuclease (mXRN1p) with preference for G4 tetraplex substrates. *J. Cell Biol.* **136**, 761–773 (1997).

153. Zhang, M. *et al.* Cloning and mapping of the *XRN2* gene to human chromosome 20p11.1-p11.2. *Genomics* **59**, 252–254 (1999).

154. Dehlin, E., Wormington, M., Korner, C. G. & Wahle, E. Cap-dependent deadenylation of mRNA. *EMBO J.* **19**, 1079–1086 (2000).

155. Korner, C. G. & Wahle, E. Poly(A) tail shortening by a mammalian poly(A)-specific 3′-exoribonuclease. *J. Biol. Chem.* **272**, 10448–10456 (1997).

156. Allmang, C. *et al.* The yeast exosome and human PM-Scl are related complexes of 3′→5′ exonucleases. *Genes Dev.* **13**, 2148–2158 (1999).

157. Chen, C. Y. *et al.* AU binding proteins recruit the exosome to degrade ARE-containing mRNAs. *Cell* **107**, 451–464 (2001).

158. Hanson, M. N. & Schoenberg, D. R. Identification of *in vivo* mRNA decay intermediates corresponding to sites of *in vitro* cleavage by polysomal ribonuclease 1. *J. Biol. Chem.* **276**, 12331–12337 (2001).

159. Lander, E. S. *et al.* Initial sequencing and analysis of the human genome. *Nature* **409**, 860–921 (2001).

160. Venter, J. C. *et al.* The sequence of the human genome. *Science* **291**, 1304–1351 (2001).

161. Kim, V. N., Kataoka, N. & Dreyfuss, G. Role of the nonsense-mediated decay factor hUpf3 in the splicing-dependent exon-exon junction complex. *Science* **293**, 1832–1836 (2001).

162. Maquat, L. E. Nonsense-mediated mRNA decay: a comparative analysis of different species. *Curr. Genomics* (in the press).

**Acknowledgements**

I thank B. Lehner, J. Lykke-Andersen, J. Mendell and N. Sonenberg for communicating unpublished data, F. Lejeune for generating figures, and members of the Maquat laboratory for their comments on the manuscript. This work was supported by Public Health Service Grants from the National Institutes of Health.

**Competing interests statement**

The author declares that she has no competing financial interests.

**Online links**

**DATABASES**

The following terms in this article are linked online to:

Swiss-Prot: http://ca.expasy.org/sprot/

CBP20 | CBP80 | DCP2 | eIF4E | MAGOH | NXF1/TAP | PABP2 | PARN | PM/SCL100 | REF | RNPS1 | SMG1 | SMG5 | SMG7 | SRm160 | UPF1 | UPF2 | UPF3 | UPF3X | Y14

**FURTHER INFORMATION**

Lynne E. Maquat's laboratory: http://dbb.urmc.rochester.edu/labs/maquat/maquat_lab.htm

Access to this interactive links box is free online.
