
P1: JQX
Journal of Bioenergetics and Biomembranes (JOBB) pp1123-jobb-478870 February 27, 2004 17:12 Style file version June 22, 2002

**Journal of Bioenergetics and Biomembranes, Vol. 36, No. 1, February 2004 (© 2004)**

# The Membrane-Bound Electron Transport System of *Methanosarcina* Species

## Uwe Deppenmeier¹

---

Members of the genus *Methanosarcina* are strictly anaerobic archaea that derive their metabolic energy from the conversion of a restricted number of substrates to methane. H₂ + CO₂ are converted to CH₄ via the CO₂-reducing pathway, while methanol and methylamines are metabolized by the methylotrophic pathway. Two novel electron transport systems are involved in the process of methanogenesis. Both systems are able to use a heterodisulfide as electron acceptor and either H₂ or F₄₂₀H₂ as electron acceptors and generate a proton-motive force by redox potential-driven H⁺-translocation. The H₂:heterodisulfide oxidoreductase is composed of an F₄₂₀-nonreducing hydrogenase and the heterodisulfide reductase. The latter protein is also part of the F₄₂₀H₂:heterodisulfide oxidoreductase system. The second component of this system is referred to as F₄₂₀H₂ dehydrogenase. The archaeal protein is a homologue of complex I of the respiratory chain from bacteria and mitochondria. This review focuses on the biochemical and genetic characteristics of the three energy-transducing enzymes and on the mechanisms of ion translocation.

### KEY WORDS: Archaea; methane; methanogenesis; respiratory chain; energy transduction; proton gradient; complex I; F₄₂₀H₂ dehydrogenase; heterodisulfide reductase; hydrogenase.

---

## INTRODUCTION

Methanogenic organisms belong to the domain of the Archaea and are widespread in anoxic environments such as fresh water sediments of lakes and rivers, swamps, tundra areas, rice fields, and anaerobic digesters of sewage plants. Other habitats are the intestinal tracts of ruminants and termites (Garcia et al., 2000). The process of methanogenesis is fundamental for the global carbon cycle because it represents the terminal step in the anaerobic breakdown of organic matter in freshwater sediments (Conrad, 1996). CH₄ and CO₂ are the major products of methanogenesis. The gases are released from anaerobic environments and can reenter the global carbon cycle. Since large amounts of CH₄ escape into the atmosphere, the process is also of great interest for the global ecology because methane is one of the most important greenhouse gases and contributes to a high degree to global warming effects (Khalil and Rasmussen, 1994; Reay, 2003). On the other hand,

---

¹ Department of Biological Sciences, University of Wisconsin–Milwaukee, PO Box 413, Milwaukee, Wisconsin 53201; e-mail: udeppen@uwm.edu.

it is important to note that methanogens create a combustible gas that can be used as energy sources for domestic or industrial use. In addition, there is one more relevant aspect of methanogenesis. It is thought that biological methane production is involved or even responsible for the formation of the so-called methane hydrates (Marchesi et al., 2001). These ice-like structures are found in ocean floor sediments at water depths greater than about 500 m. Gas hydrates are solids, composed of rigid cages of water molecules that trap CH₄ molecules (Kvenvolden, 1999) and represent a potentially enormous natural gas resource (Wood et al., 2002).

The formation of methane from H₂ + CO₂, methylated C₁-compounds (methanol, methylamines, methylthiols) or acetate is the characteristic feature of members of the genus *Methanosarcina*. Very recently, the genome of two species, *Ms. acetivorans* and *Ms. mazei*, have been published (Deppenmeier et al., 2002; Galagan et al., 2002). Furthermore, raw sequence data from *Ms. barkeri* are available (http://genome.ornl.gov/microbial/mbar). The data will certainly lead to new insights into the metabolism and the cellular functions of this interesting group of microorganisms.

As described above the process of methanogenesis is of major ecological importance but also the metabolic pathways of methane formation are of great interest because they are rather unique and involve a number of unusual enzymes and coenzymes (Deppenmeier, 2002a; Shima et al., 2002). This review describes novel membrane-bound enzymes generating primary proton gradients that are involved in methane formation.

## BIOCHEMISTRY OF METHANOGENESIS

The pathways of methane formation from the above-mentioned substrates are summarized in Fig. 1. Methanogenesis from H₂ + CO₂ proceeds via the carbon dioxide-reducing pathway. The first intermediate is formyl-methanofuran that is formed by the reduction of CO₂ bound to the C₁-carrier methanofuran (MFR) (Deppenmeier, 2003). The endergonic reaction is catalyzed by a formyl-MFR dehydrogenase and is driven by an electrochemical ion gradient (for details see Hedderich R., this issue). The formyl group is then transferred to tetrahydromethanopterin (H₄MPT) and the resulting formyl-H₄MPT is stepwise reduced to methyl-H₄MPT (Shima et al., 2002). The enzymes catalyzing these reactions are dependent on reduced coenzyme F₄₂₀, which is the central electron carrier in the cytoplasm of methanogens (∆E° = -360 mV). The cofactor is a deazaflavin derivative that accepts or donates hydride ions (Walsh, 1986). A special F₄₂₀-reducing hydrogenase is able to reduce the cofactor using molecular hydrogen as reductant (Sorgenfrei et al., 1997). In the next reaction of this pathway the methyl group of methyl-H₄MPT is transferred to HS-CoM by the methyl-H₄MPT:HS-CoM methyltransferase. The exergonic reaction is coupled to the formation of an electrochemical sodium ion gradient (for review see Gottschalk and Thauer, 2001). The methyltransfer reaction leads to the formation of methyl-S-CoM (methyl-mercaptoethanesulfonate) that is reductively cleaved by the methyl-CoM reductase. Two electrons required for the reduction are derived from another unique methanogenic cofactor, HS-CoB (7-mercaptoheptanoyl-O-phospho-L-threonine) (Noll et al., 1986). Thus, the reaction results in the formation of CH₄ and a heterodisulfide (CoM-S-S-CoB) from HS-CoM and HS-CoB. In a second reaction CoM-S-S-CoB is reduced by the heterodisulfide reductase (see below). The reducing equivalents are derived from H₂ and are provided by a membrane-bound electron transport system that is referred to as H₂:heterodisulfide oxidoreductase (Ide et al., 1999).

Methylated substrates such as methanol are degraded by the methylotrophic pathway of methanogenesis and are converted to CH₄ and CO₂ (Fig. 1):

\[
4 \mathrm{CH}_{3} \mathrm{OH} \longrightarrow 3 \mathrm{CH}_{4} + 1 \mathrm{CO}_{2} + 2 \mathrm{H}_{2} \mathrm{O}
\]

\[
(\Delta G_{0}^{\prime}=-106 \mathrm{~kJ} / \mathrm{mol} \mathrm{CH}_{4})
\]

The series of reactions involve soluble methyltransferase systems that catalyze the transfer of the methyl-group to HS-CoM (Ding et al., 2002). In the oxidative branch of the pathway (Fig. 1), one out of four methyl groups is oxidized to CO₂ by the reversed CO₂-reduction route as described above (Deppenmeier, 2002a). (i) The methyl moiety of methyl-S-CoM is transferred to H₄MPT in an endergonic reaction that is driven by an electrochemical sodium ion gradient (Gottschalk and Thauer, 2001). (ii) The resulting intermediate, methyl-H₄MPT, is stepwise oxidized to formyl-H₄MPT. Reducing equivalents derived from these reactions are used for F₄₂₀ reduction. (iii) After transfer of the formyl group to MFR the formyl-MFR

Fig. 1. Pathway of methanogenesis from H₂ + CO₂ and methanol. Reactions catalyzed by membrane-bound enzyme complexes are boxed. Abbreviations: F₄₂₀H₂, reduced form of coenzyme F₄₂₀; Fd_red, reduced form of ferredoxin; MFR, methanofuran; H₄MPT, tetrahydromethanopterin; HS-CoM, coenzyme M; HS-CoB, coenzyme B; MPH2, reduced form of methanophenazine. Enzymes: 1. Ech hydrogenase; 2. formylmethanofuran dehydrogenase; 3. formyl-MFR:H₄MPT formyl-transferase; 4. methenyl-H₄MPT cyclohydrolase; 5. methylene-H₄MPT dehydrogenase; 6. F₄₂₀-reducing hydrogenase; 7. methylene-H₄MPT reductase; 8. methyl-H₄MPT:HS-CoM methyltransferase; 9. methyl-CoM reductase; 10. soluble methyltransferases; 11. F₄₂₀-nonreducing hydrogenase; 12. Heterodisulfide reductase; 13. F₄₂₀H₂ dehydrogenase.

The Membrane-Bound Electron Transport System of *Methanosarcina* Species

dehydrogenase catalyzes the oxidation of CHO-MFR to CO₂ and MFR (see Hedderich R., this issue). In the reductive branch of the pathway three out of four methyl groups are transferred to HS-CoM and are reduced to CH₄ (Fig. 1). As aforementioned HS-CoB functions as electron donor forming CoM-S-S-CoB. Finally, a membrane-bound electron transfer system (F₄₂₀H₂:heterodisulfide oxidoreductase) is responsible for the reduction of the heterodisulfide (Bäumer *et al.*, 2000).

In the course of methanogenesis from acetate the carboxyl group is phosphorylated and subsequently the acetyl group is transferred to coenzyme A (Ferry, 1997). The key enzyme in the so-called aceticlastic pathway is the acetyl-CoA synthase/CO dehydrogenase (Grahame, 2003; Seravalli *et al.*, 2003). This protein cleaves the C—C- and C—S-bonds in acetyl-CoA and oxidizes CO to CO₂. The methyl-moiety of acetyl-CoA is transferred to H₄MPT, and electrons derived from CO-oxidation are used for ferredoxin reduction. The methyl-group of CH₃-H₄MPT is transferred to HS-CoM and finally, CoM-S-S-CoB and CH₄ are formed from methyl-CoM and HS-CoB. The remaining intermediates CoM-S-S-CoB and reduced ferredoxin (Fd_red) are further metabolized by a third membrane-bound electron transport system (Fd_red:heterodisulfide oxidoreductase) (for further details the reader is referred Hedderich R., this issue).

MEMBRANE-BOUND ELECTRON TRANSPORT AND ENERGY CONSERVATION DURING GROWTH ON H₂ + CO₂ AND METHYLATED SUBSTRATES

*Methanosarcina* species such as *Ms. mazei* and *Ms. barkeri* possess two novel membrane-bound electron transport systems both of which are able to use the heterodisulfide as electron acceptor and either H₂ or F₄₂₀H₂ as electron acceptors (Eqs. (2) and (3)) (Deppenmeier *et al.*, 1999).

H₂ + CoM-S-S-CoB → HS-CoM + HS-CoB  
(ΔG′₀ = −40 kJ/mol)   (2)

F₄₂₀H₂ + CoM-S-S-CoB → HS-CoM + HS-CoB + F₄₂₀  
(ΔG′₀ = −30.9 kJ/mol)   (3)

The electron transport chains are referred to as F₄₂₀H₂: heterodisulfide oxidoreductase and H₂:heterodisulfide oxidoreductase, respectively (Fig. 1). It has been shown that electron transport as catalyzed by the systems is accompanied by proton translocation across the cytoplasmic membrane. The electrochemical proton gradient thereby generated is used for ATP synthesis from ADP + Pᵢ catalyzed by an A₁A₀-type ATP synthase (see Müller V., this issue). Each anaerobic respiratory chain yields stoichiometries of 4 mol of translocated protons per mol of reduced heterodisulfide, respectively (Deppenmeier, 2002b). The addition of the protonophore SF6847 or of the ATP synthase inhibitor DCCD resulted in effects that resemble the phenomenon of respiratory control as described for mitochondria. Further investigations showed that the H₂:heterodisulfide oxidoreductase system is composed of a membrane-bound hydrogenase (later on referred to as F₄₂₀-nonreducing hydrogenase) and the heterodisulfide reductase (Fig. 3) (Deppenmeier *et al.*, 1999). The latter protein is also part of the F₄₂₀H₂: heterodisulfide oxidoreductase. The second component of the F₄₂₀-dependent system was named F₄₂₀H₂ dehydrogenase (Fig. 2). An interesting question concerned the nature of the electron carriers that mediate electron

![Figure 2](#fig2)

Fig. 2. Tentative model of complex I from *E. coli* (A) and the F₄₂₀H₂ dehydrogenase from *Methanosarcina mazei* (B).

transfer between the above-mentioned proteins. Membranes of methanogenic archaea do not contain typical quinone components such as ubiquinone or menaquinone. However, a unique hydrophobic cofactor has been isolated recently from the cytoplasmic membrane of *Methanosarcina* species that was referred to as methanophenazine (Fig. 2). The component represents a 2-hydroxyphenazine derivative that is connected via an ether bridge to a pentaprenyl side chain (Abken *et al.*, 1998).

Using 2-hydroxyphenazine as a water-soluble analogue of methanophenazine the catalytic activity of the above-mentioned enzymes could be analyzed in more detail. It became evident that 2-OH-phenazine is reduced by molecular hydrogen as catalyzed by the F₄₂₀-nonreducing hydrogenase. Furthermore, the membrane-bound heterodisulfide reductase is able to use dihydro-2-OH-phenazine as electron donor for the reduction of CoM-S-S-CoB (Bäumer *et al.*, 2000; Ide *et al.*, 1999). Both reactions are coupled to the transfer of two protons across the cytoplasmic membrane. Hence, the H₂:heterodisulfide oxidoreductase system contains two proton-translocating redox-loops that are catalyzed by the F₄₂₀-nonreducing hydrogenase and the heterodisulfide reductase, respectively.

H₂ + OH-phenazine + 2H⁺<sub>in</sub>
→ dihydro-OH-phenazine + 2H⁺<sub>out</sub>  (4)

CoM-S-S-CoB + dihydro-OH-phenazine + 2H⁺<sub>in</sub>
→ HS-CoM + HS-CoB + OH-phenazine + 2H⁺<sub>out</sub>  (5)

2-OH-phenazine also mediates electron transfer within the F₄₂₀H₂:heterodisulfide oxidoreductase system (Bäumer *et al.*, 1998). Comprehensive analysis revealed that reducing equivalents are transferred from F₄₂₀H₂ to 2-OH-phenazine by the membrane-bound F₄₂₀H₂ dehydrogenase. Once again, the reaction is coupled to proton transfer across the cytoplasmic membrane exhibiting a stoichiometry of about two protons translocated per two electrons transferred.

F₄₂₀H₂ + OH-phenazine + 2H⁺<sub>in</sub>
→ F₄₂₀ + dihydro-OH-phenazine + 2H⁺<sub>out</sub>  (6)

The second reaction of this electron transport system is again catalyzed by the heterodisulfide reductase that uses dihydro-2-OH-phenazine as electron donor for CoM-S-S-CoB reduction. Just as in the H₂-dependent system, both partial reactions of the F₄₂₀H₂: heterodisulfide oxidoreductase system are coupled to the translocation of protons. Thus, the H⁺/2e⁻ stoichiometries of both electron transport chains add up to four and support the

value of 4H⁺/2e⁻ in the overall electron transport from F₄₂₀H₂ and from H₂ to the heterodisulfide, respectively (Deppenmeier, 2002a). In agreement to these findings is the fact that 3–4 H⁺ were transferred by whole cell preparations of *Ms. barkeri* when methane formation from methanol + H₂ was analyzed (Blaut *et al.*, 1987).

As described above, first experiments on the biological function of phenazines in the cytoplasmic membrane of *Ms. mazei* were performed using the model compound 2-OH-phenazine and its reduced form. It was demonstrated that all key enzymes react with the artificial electron carrier. After completion of the total synthesis of methanophenazine similar tests could be performed with the natural electron carrier (Beifuss *et al.*, 2000). The results clearly indicated that methanophenazine serves as an electron acceptor to both the membrane-bound hydrogenase and the F₄₂₀H₂ dehydrogenase if H₂ and F₄₂₀ were added, respectively. In addition, the heterodisulfide reductase uses the reduced form of methanophenazine as an electron donor for the heterodisulfide reduction. Thus, the cofactor functions as membrane integral electron carrier connecting protein complexes of the respiratory chain of *Ms. mazei* (Deppenmeier, 2002b).

### CHARACTERISTICS OF PROTON-TRANSLOCATING ENZYMES IN METHANOGENS
#### F₄₂₀H₂ DEHYDROGENASE

Reduced coenzyme F₄₂₀ is formed during methanogenesis from methylated C₁-compounds and can be produced by an F₄₂₀H₂-dependent hydrogenase in the presence of molecular hydrogen. The F₄₂₀H₂ dehydrogenase is responsible for the reoxidation of the cofactor and is the initial enzyme of the F₄₂₀H₂:heterodisulfide oxidoreductase system. As described above, the protein catalyzes the F₄₂₀H₂-dependent reduction of phenazine derivatives (Eq. (6)), thereby transferring two protons across the cytoplasmic membrane (Bäumer *et al.*, 2000). Hence, it represents a novel proton-translocating enzyme and is therefore of interest with respect to subunit composition, cofactor content, and reaction mechanisms.

First attempts to purify the F₄₂₀H₂ dehydrogenase from *Ms. mazei* revealed the presence of five different polypeptides with molecular masses of 40, 37, 22, 20, and 16 kDa. Furthermore, the protein contained nonheme iron, acid-labile sulfur and evidence was presented that FAD functions as electron carrier within the enzyme (Abken and Deppenmeier, 1997). Similar enzymes were isolated from *Methanobulus tindarius* and the sulfate-reducing archaeon *Archaeoglobus fulgidus* (Haase *et al.*,

1992; Kunow *et al.*, 1994). Furthermore, one of the genes (*ffdB*) encoding the F₄₂₀H₂ dehydrogenase from the former organism has been sequenced (Westenberg *et al.*, 1999). However, a detailed insight into the organization of the genes encoding the F₄₂₀H₂ dehydrogenase from methanogenic archaea had to await the genome-sequencing project of *Ms. mazei* (Deppenmeier *et al.*, 2002). In summary, it became evident that the subunits of the enzyme are encoded by the *fpo* operon that comprises 12 genes (*fpo* A, -B, -C, -D, -H, -I, -J, -K, -L, -M, -N, -O) (Bäumer *et al.*, 2000). Furthermore, the results clearly showed that the subunits with molecular masses of 40, 22, 20, and 16 kDa of the purified enzyme are encoded by the genes *fpo* D, -B, -C, and -I, respectively. The gene (*fpoF*) encoding the 37 kDa subunit is not part of the operon and is located at a different site on the chromosome. The corresponding polypeptide is homologous to the β subunit of F₄₂₀-reducing hydrogenases and to subunits of the F₄₂₀H₂ dehydrogenase from *Methanobolus tindarius* (FfdB) (Westenberg *et al.*, 1999) and *Archaeoglobus fulgidus* (FqoF). Recently, FqoF from *Archaeoglobus fulgidus* (Brüggemann *et al.*, 2000) and FpoF from *Ms. mazei* (unpublished results) were overproduced and purified to homogeneity. The subunits contained nonheme iron, acid-labile sulfur, and FAD and are able to oxidize F₄₂₀H₂ when the artificial electron acceptor methylviologen was added.

DNA sequence analysis and amino acid sequence alignments have revealed that all subunits of the F₄₂₀H₂ dehydrogenase (with the exception of FpoF and FpoO) show high similarities to subunits of proton-translocating NADH:quinone oxidoreductases from prokaryotes and eukaryotes (from here on referred to as complex I).

The enzyme from *E. coli* is a suitable model to describe the composition of bacterial complex I (David *et al.*, 2002; Leif *et al.*, 1995) (Fig. 2). Therefore, the genes and subunits of complex I are named according to the protein of *E. coli* throughout the text (see Yagi and Matsuno-Yagi, 2003, for the nomenclature of complex I subunits from other organisms). Purified complex I from *E. coli* easily disintegrates into a so-called NADH dehydrogenase fragment, a connecting fragment, and a membrane fragment by changing the pH and the detergent (Leif *et al.*, 1995). The soluble NADH dehydrogenase fragment comprises the subunits NuoE, -F, and -G and harbors the flavin mononucleotide and the EPR detectable FeS clusters N1a, N1b, N3, and N4 (Bungert *et al.*, 1999; Ohnishi, 1998). The amphipathic connecting fragment consists of NuoB, -CD, and -I and forms three FeS clusters (Duarte *et al.*, 2002; Rasmussen *et al.*, 2001).

The membrane fragment is composed of the hydrophobic subunits NuoA, -H, and -J to -N (Friedrich and Scheide, 2000). This part of the protein is probably involved in quinone binding and in H⁺-translocation (Fig. 2).

Several lines of evidence revealed that the proton pumping NADH:ubiquinone oxidoreductase has evolved from preexisting proteins (Böttcher *et al.*, 2002; Friedrich and Scheide, 2000). It is built from a peripheral NADH-oxidizing subcomplex (subunits NuoEFG) that is homologous to soluble NAD-dependent hydrogenases (Pilkington *et al.*, 1991). All subunits of the connecting fragment (NuoB, C, D, I) together with the hydrophobic subunits NuoH and -L of the membrane fragment constitute the so-called hydrogenase module, which is related to a family of membrane-bound multisubunit [NiFe] hydrogenases (see Hedderich R., this issue; Terstegen and Hedderich, 1999). This part of the enzyme is involved in electron transfer to ubiquinone and proton translocation (Fig. 2). It is important to note that the hydrogenase-module does not possess a measurable hydrogenase activity. One of the reasons for this finding is that the four cysteine residues coordinating the nickel atom in NiFe hydrogenases are not found in complex I. Thus, the bimetallic center necessary for H₂ cleavage is not present in the NADH dehydrogenase.

The remaining subunits of the membrane fragment, namely NuoA, -J, -K, -M, and -N probably participate in H⁺-transfer and quinone-binding (Gong *et al.*, 2003). Together the subunits build the so-called transporter module containing homologues of H⁺/Na⁺ (or K⁺) antiporter subunits (Friedrich and Weiss, 1997; Ito *et al.*, 2001; Mathiesen and Hagerhall, 2002).

Hydropathy plots revealed that the deduced subunits from *fpo* A, -J, -K, -M, and -N of the F₄₂₀H₂ dehydrogenase are membrane-integral components comprising more than 30 transmembrane helices. This part of the F₄₂₀H₂ dehydrogenase complex shows high similarities to the transporter module of bacterial NADH dehydrogenases (NuoA, -J, -K, -M, -N) (Friedrich and Scheide, 2000) and to the mitochondrially encoded complex I subunits from eukarya (Nd 2, 3, 4, 4L, and 6) (Carroll *et al.*, 2003). Thus, Fpo A, -J, -K, -M, and -N build the transporter module of the F₄₂₀H₂ dehydrogenase (Fig. 2).

Homologues of the second module of complex I are also found in the F₄₂₀H₂ dehydrogenase. In the methanogenic enzyme the so-called hydrogenase module is constituted of the hydrophilic subunits FpoB, -C, -D, -I and the hydrophobic polypeptides FpoH and -L (Fig. 2). It has been proposed that the hydrogenase module couples the transfer of electrons from F₄₂₀H₂ to methanophenazine with the translocation of protons across the membrane (Deppenmeier, 2002b).

Keeping in mind these findings, some important conclusions can be drawn about the relationship of complex I and F₄₂₀H₂ dehydrogenases. On one hand, the subunit composition of the hydrogenase- and the transporter

module of the enzymes are identical and the corresponding subunits are highly homologous. On the other hand it is to mention that quinones are not produced by methanogenic archaea (Hughes and Tove, 1982). Thus, quinone binding sites and semiquinone radicals found in complex I cannot exist in the methanogenic enzyme. In summary, the electron transport pathway through the membrane-integral module of the F₄₂₀H₂ dehydrogenase must be different compared to complex I and must involve the electron carrier methanophenazine instead of quinones.

EPR analysis of complex I showed that three distinct iron-sulfur clusters are present in the hydrogenase module: (i) Cluster N2 is characterized by a high and pH-dependent \( E^{\circ} \) value that is sensitive to the electron chemical proton gradient (Hellwig et al., 2000; Yano and Ohnishi, 2001). Since it shows mutual magnetic interacts with semiquinone species, the cluster might be directly involved in quinone reduction and proton translocation (see below) (Yano et al., 2000). Several experimental data and homology analysis indicate that the NuoB subunits bear iron-sulfur cluster N2 (Duarte et al., 2002; Flemming et al., 2003). (ii) Clusters N6a and N6b are tetranuclear FeS cluster located on NuoI (Friedrich et al., 2000). Electrochemical titration revealed a pH-independent midpoint potential of -270 mV. All iron-sulfur signatures of NuoB and NuoI are conserved in corresponding subunits of the F₄₂₀H₂ dehydrogenase from *Ms. mazei*. It is, therefore, reasonable to assume that FeS-clusters comparable to N2 and N6a/b are present in subunits FpoB and FpoI of the archaeal protein. However, it is to mention that there are other theories about the localization of the FeS clusters in complex I and the ion-translocating activity of the enzyme. Because of the limited space the reader is referred to Chevallet et al. (2003), Albracht et al. (2003), and Steuber (in press) for further reading.

Besides the electron transfer function, the hydrogenase module of complex I might also be involved in quinone reduction (Dupuis et al., 2001; Fisher and Rich, 2000). Strong evidence has been presented that the binding site of the polar head of the electron accepting quinone is located in NuoD close to subunit NuoB (Prieur et al., 2001), which are the counterparts of FpoB and FpoD in the F₄₂₀H₂ dehydrogenase (Fig. 2). Inhibitor studies using different mutants indicated that the C-terminal domain of NuoD might be essential for the binding process (Darrouzet et al., 1998). It is known that many inhibitors targeting complex I such as rotenone, fenaquine, capsaicin, and annonine I bind at, or close to, the quinone binding site(s) (Espositi, 1998). Interestingly, all these inhibitors of complex I have no effect on the activity of the F₄₂₀H₂ dehydrogenase (unpublished results). The most simple explanation for

the noninhibitory effect on the archaeal enzyme is that the methanophenazine binding site must be enlarged to allow the entrance of a phenazine derivative that is composed of three aromatic rings. Hence, ubiquinone analogous inhibitors do not fit into this cavity and do not lead to inhibition of electron transfer within the F₄₂₀H₂ dehydrogenase (Deppenmeier, 2002b). From this point of view, it is not a surprise that the corresponding amino acid sequence of the C-terminus of FpoD is different from NuoD. Small amino acids such as Gly367 might provide the space necessary for the binding of methanophenazine. In NuoD-like subunits large aliphatic amino acids (Val, Ile, or Phe) or Pro are found in this position. In summary, the C-terminal domain of FpoD is obviously modified and adapted for the reduction of methanophenazine (Fig. 2).

As outlined above, all subunits of the F₄₂₀H₂ dehydrogenase reveal high similarities to subunits of complex I. There are only two exceptions which concern the input device FpoF (see above) and subunit FpoO that is encoded by the last gene of the *fpo* operon (Bäumer et al., 2000). FpoO has no counterpart in complex I and is predicted to be a hydrophilic subunit that contains a motif for the binding of one [2Fe-2S] cluster. Very recently the protein has been overproduced and purified (unpublished results). EPR studies and protein analysis clearly showed that a binuclear FeS center is present in the subunit indicating that it might be involved in electron transfer within the F₄₂₀H₂ dehydrogenase. Our current working hypothesis is that FpoO participates in the reduction of methanophenazine. If this hypothesis is correct, FpoO would represent a special equipment needed for the reduction of the methanogenic cofactor.

In contrast to the aforementioned subcomplexes, the reduced cofactor oxidizing devices from F₄₂₀H₂ dehydrogenases and from NADH dehydrogenases are not homologous. The corresponding module of the bacterial and eukaryotic enzymes is made from three different subunits (NuoE, -F, -G; see above). As mentioned before the oxidation of reduced cofactor F₄₂₀ is catalyzed by subunit FpoF of the F₄₂₀H₂ dehydrogenase from *Ms. mazei*. However, there are also parallels because both modules catalyze the oxidation of obligate hydride donors (F₄₂₀H₂ in case of the F₄₂₀H₂ dehydrogenase and NADH in case of complex I) and contain flavins and FeS clusters (Braun et al., 1998; Brüggemann et al., 2000).

The similarities of the hydrogenase modules and the transporter modules of the dehydrogenases allow to speculate about reaction mechanisms of the methanogenic protein and about structure/function relationships of its subunits (Fig. 2): (i) As shown above FpoF is the input device and oxidizes reduced coenzyme F₄₂₀. The electron

The Membrane-Bound Electron Transport System of Methanosarcina Species

MEMBRANE-BOUND HYDROGENASES

Three types of [NiFe] hydrogenases have been described in *Methanosarcina* species (F₄₂₀-reducing hydrogenase, F₄₂₀-nonreducing hydrogenase, and Ech hydrogenase) but only two of them seem to be involved in membrane-bound electron transfer and energy conservation (Deppenmeier, 2002a; Thauer, 1998). Generally, the core of these enzymes is composed of a small electron transfer subunit and a large catalytic subunit harboring a bimetallic nickel–iron center. The small subunit contains 2–3 FeS clusters and is responsible for electron transport from the catalytic center (Fontecilla-Camps and Ragsdale, 1997; Vignais *et al.*, 2001; Volbeda *et al.*, 2002) to the electron acceptor subunit.

1. **Ech hydrogenase**: This novel type of hydrogenase was discovered in acetate-grown cells of *Ms. barkeri* (Künkel *et al.*, 1998) and has been purified to homogeneity (Meuer *et al.*, 1999). After sequencing of the corresponding genes (*echABCDEF*) it became evident that the subunits show homologies to multisubunit hydrogenases from bacteria and to subunits of complex I (Friedrich and Scheide, 2000). Because of these homologies it was proposed that the enzyme functions as a proton pump (Hedderich *et al.*, 1998). The purified Ech hydrogenase catalyzes the H₂-dependent reduction of a 2[4Fe-4S] ferredoxin from *Ms. barkeri* and is also able to perform the reverse reaction, namely hydrogen formation from reduced ferredoxin (Kurkin *et al.*, 2002). It is thought that the enzyme plays an essential role in providing the cell with reduced ferredoxin for the first step of methanogenesis from H₂/CO₂ and for the synthesis of pyruvate in an anabolic pathway (Meuer *et al.*, 2002). The enzyme is described in detail by Hedderich in this issue and the reader is referred to this article for further reading.

2. **F₄₂₀-nonreducing hydrogenase**: In contrast to the soluble F₄₂₀-reducing hydrogenase this enzyme is not able to interact with the central electron carrier F₄₂₀ (Fig. 1). The membrane-bound F₄₂₀-nonreducing hydrogenase has been isolated from *Ms. mazei* and *Ms. barkeri* (Deppenmeier *et al.*, 1992; Kemner and Zeikus, 1994). The purified protein is composed of two subunits with molecular masses of 60.3 kDa and 39.9 kDa and contains acid-labile sulfur, nonheme iron and about 1 mol Ni/mol enzyme. The structural genes of the F₄₂₀-nonreducing hydrogenases from *Ms. mazei*

transfer from the two-electron donor to iron–sulfur clusters present in the FpoF requires a $2e^{-}/1e^{-}$ switch, which is managed by FAD. (ii) Further electron transport within the enzyme is catalyzed by the membrane-associated fragment composed of subunits FpoB, -C, -D, and -I. The signatures for the formation of iron–sulfur clusters found in NuoB and NuoI are conserved in FpoB and FpoI, indicating that metal-clusters similar to N2 and N6 are formed by these subunits of the F₄₂₀H₂ dehydrogenase. Hence, electron transport within the amphipatic part of the methanogenic enzyme is probably managed by cluster N6a/b that accepts electrons from subunit FpoF and donates them to cluster N2. (iii) Finally, the binuclear Fe-S center of FpoO takes up the electrons and reduces methanophenazine that is bound at the interface between the connecting module and membrane-integral module. In analogy to the quinone-binding site of complex I (Dupuis *et al.*, 2001; Kerscher *et al.*, 2001; Schuler and Casida, 2001) the phenazine binding site might be constituted of two parts. A hydrophobic pocket formed by the FpoH subunit that encloses the “isoprenyl-tail” and a polar head borne by the FpoD subunit that builds the binding site of the polar head of the electron acceptor.

Analysis of the proton-translocating activity of the F₄₂₀H₂ dehydrogenase revealed that the enzyme is only able to pump two protons in the course of the reaction cycle (Bäumer *et al.*, 2000). Thus, the energy transducing efficiency is only half of the one of complex I (Fig. 2). These findings are in accordance with thermodynamic considerations. Taking into account that the midpoint potential of methanophenazine is in the range of –200 mV (Beifuss, personal communication) the change of free energy ($\Delta G^{\circ'}$) coupled to the F₄₂₀H₂-dependent methanophenazine reduction is only –30.9 kJ/mol. Thus, the Gibbs free energy available from the reaction is just sufficient to drive the transfer of about two protons/$2e^{-}$.

In summary, the F₄₂₀H₂ dehydrogenase and complex I have some important characteristics in common (Bäumer *et al.*, 2000; Yagi and Matsuno-Yagi, 2003): (i) The flavin and iron–sulfur containing proteins are the initial enzymes of membrane-bound electron transport systems and reveal a complex subunit composition. (ii) The amino acid sequences of subunits forming the hydrogenase- and the transport module are highly homologous. (iii) The electron donors F₄₂₀H₂ and NADH are both reversible hydride donors with comparable midpoint potentials. (iv) The enzymes take advantage of small hydrophobic nonproteinous electron acceptors namely quinones (complex I) and methanophenazine (F₄₂₀H₂ dehydrogenase), respectively. (v) Both enzymes are characterized by their redox-driven proton-translocating activity.

Cys, His, and Glu residues (Garcin *et al.*, 1998; Volbeda *et al.*, 1995). (iii) It is tempting to speculate that iron-sulfur clusters in the small subunit accept the electrons and transfer them to the respective electron acceptor to the cytochrome $b$ subunit (Bernhard *et al.*, 1996; Brodersen *et al.*, 1999; Drosset *al.*, 1992). (iv) To complete the reaction cycle the membrane-integral cytochrome $b$ subunit accepts two protons from the cytoplasm for the reduction of methanophenazine (Beifuss *et al.*, 2000). Thus, the overall reaction would lead to the production of two scalar protons (Fig. 3) (Ide *et al.*, 1999).

HETERODISULFIDE REDUCTASE

As mentioned above the heterodisulfide reductase catalyzes the two-electron reduction of CoM-S-S-CoB (Hedderich *et al.*, 1990) which is the final step in the electron transport chain of methanogens (Hedderich *et al.*, 1998; Fig. 3). The enzymes of methylotrophic methanogens consist of two subunits (HdrD, HdrE) (Küinkel *et al.*, 1997, Simianu *et al.*, 1998). The membrane-integral subunit HdrE represents a $b$-type cytochrome (Heiden *et al.*, 1994) and contains two heme molecules with midpoint potentials of $-180$ and $-23 \, \text{mV}$ (Simianu *et al.*, 1998). The large subunit HdrD contains the active site for disulfide reduction and comprises two [4Fe-4S] clusters with midpoint potentials of $-100$ and $-400 \, \text{mV}$. EPR studies of the enzyme from *Ms. barkeri* led to the discovery of a paramagnetic species generated by an unusual FeS cluster that is connected to the sulfur of HS-CoM (Madadi-Kahkesh *et al.*, 2001). Taking advantage of this observation, Hedderich and coworkers proposed a possible catalytic mechanism for the heterodisulfide reductase (Duin *et al.*, 2003). According to their hypothesis the methanophenazine-dependent reduction of the heterodisulfide involves the one-electron reduction of the active site [4Fe2-4S]$^{2+}$ cluster to the corresponding [4Fe-4S]$^{1+}$, which immediately reacts to cleave CoM-S-CoB via a nucleophilic substitution reaction. Thus, HS-CoB and a [4Fe-4S]$^{3+}$ cluster with CoM-S$^-$ attached to one iron of the cluster are formed. Also the pathway of electron transfer from reduced phenazines to the heterodisulfide has been studied using the purified heterodisulfide reductase from *Ms. thermophila* (Murakami *et al.*, 2001). Stopped flow experiments showed that the low-potential heme participates in reduction of CoM-S-S-CoB. Furthermore, spectroscopic and kinetic studies with the inhibitor diphenylene iodonium indicated that only a high potential [4Fe-4S] cluster is involved in

electron transfer to CoM-S-S-CoB. In terms of energy conservation it is to speculate that proton translocation is based on the production of scalar protons as in case of the F₄₂₀-nonreducing hydrogenase (Deppenmeier, 2002a) (Fig. 3). According to this hypothesis reducing equivalents from reduced phenazine derivatives are transferred to the 1-electron accepting prosthetic groups of the enzyme (heme b) and protons are released at the outer phase of the cytoplasmic membrane (Abken et al., 1998; Brodersen et al., 1999) (Fig. 3). Then, the electrons enter the reactive center where CoM-S-S-CoB is reduced. The protons necessary for this reaction are derived from the cytoplasm. Thus, two protons are translocated, which contribute to the electrochemical proton gradient (Deppenmeier, 2002b).

### REFERENCES

- Abken, H. J., and Deppenmeier, U. (1997). *FEMS Lett.* **154**, 231–237.
- Abken, H. J., Tietze, M., Brodersen, J., Bäumer, S., Beifuss, U., and Deppenmeier, U. (1998). *J. Bacteriol.* **180**, 2027–2032.
- Albracht, S. P. J., van der Linden, E., and Faber, B. W. (2003). *Biochim. Biophys. Acta* **1557**, 41–49.
- Bäumer, S., Ide, T., Jacobi, C., Johann, A., Gottschalk, G., and Deppenmeier U. (2000). *J. Biol. Chem.* **275**, 17968–17973.
- Bäumer, S., Murakami, E., Brodersen, J., Gottschalk, G., Ragsdale, S. W., and Deppenmeier, U. (1998). *FEBS Lett.* **428**, 295–298.
- Beifuss, U., Tietze, M., Bäumer, S., and Deppenmeier U. (2000). *Angew. Chem. Int. Ed.* **39**, 2470–2473.
- Bernhard, M., Schwartz, E., Rietdorf, J., and Friedrich, B. (1996). *J. Bacteriol.* **178**, 4522–4529.
- Blaut, M., Müller, V., and Gottschalk G. (1987). *FEBS Lett.* **215**, 53–57.
- Blokesch, M., Paschos, A., Theodoratou, E., Bauer, A., Hube, M., Huth, S., and Bock, A. (2002). *Biochem. Soc. Trans.* **30**, 674–680.
- Böttcher, B., Scheide, D., Hesterberg, M., Nagel-Steger, L., and Friedrich, T. (2002). *J. Biol. Chem.* **277**, 17970–17977.
- Braun, M., Bungert, S., and Friedrich, T. (1998). *Biochemistry* **37**, 1861–1867.
- Brodersen, J., Bäumer, S., Abken, H. J., Gottschalk, G., and Deppenmeier, U. (1999). *Eur. J. Biochem.* **259**, 218–224.
- Brüggemann, H., Falinski, F., and Deppenmeier, U. (2000). *Eur. J. Biochem.* **267**, 5810–5814.
- Bungert, S., Krafft, B., Schlesinger, R., and Friedrich, T. (1999). *FEBS Lett.* **460**, 207–211.
- Carroll, J., Fearnley, I. M., Shannon, R. J., Hirst, J., and Walker, J. E. (2003). *Mol. Cell. Proteomics* **2**, 117–126.
- Chevallet, M., Dupuis, A., Issartel, J. P., Lunardi, J. L., van Belzen, R., and Albracht, S. P. J. (2003). *Biochim. Biophys. Acta* **1557**, 51–66.
- Conrad, R. (1996). *Microbiol. Rev.* **60**, 609–640.
- David, P., Baumann, M., Wikstrom, M., and Finel, M. (2002). *Biochim. Biophys. Acta* **1553**, 268–278.
- Darrouzet, E., Issartel, J.-P., Lunardi, J., and Dupuis, A. (1998). *FEBS Lett.* **431**, 34–38.
- Deppenmeier, U. (2002a). *Prog. Nucleic Acid Res. Mol. Biol.* **71**, 223–283.
- Deppenmeier, U. (2002b). *Cell. Mol. Life Sci.* **59**, 1513–1533.
- Deppenmeier, U., Blaut, M., Lentes, S., Herzberg, C., and Gottschalk, G. (1995). *Eur. J. Biochem.* **227**, 261–269.
- Deppenmeier, U., Blaut, M., Schmidt, B., and Gottschalk, G. (1992). *Arch. Microbiol.* **157**, 505–511.

- Deppenmeier, U., Johann, A., Hartsch, T., Merkl, R., Schmitz, R. A., Martinez-Arias, R., Henne, A., Wiezer, A., Bäumer, S., Jacobi, C., Brüggemann, H., Lienard, T., Christmann, A., Bomeke, M., Steckel, S., Bhattacharyya, A., Lykidis, A., Overbeek, R., Klenk, H. P., Gunsalus, R. P., Fritz, H. J., Gottschalk, G. (2002). *J. Mol. Microbiol. Biotechnol.* **4**, 453–461.
- Deppenmeier, U., Lienard, T., and Gottschalk, G. (1999). *FEBS Lett.* **457**, 291–297.
- De Poorter, L. M. I., Geerts, W. G., Theuvenet, A. P. R., and Keltjens, J. T. (2003). *Eur. J. Biochem.* **270**, 66–75.
- Ding, Y. H. R., Zhang, S. P., Tomb, J. F., and Ferry, J. G. (2002). *FEMS Microbiol. Lett.* **215**, 127–132.
- Dross, F., Geisler, V., Lengler, R., Theis, F., Krafft, T., Fahrenholz, F., Kojro, E., Duchene, A., Tripiers, D., Juvenal, K., and Kröger A. (1992). *Eur. J. Biochem.* **206**, 93–102.
- Duarte, M., Populo, H., Videira, A., Friedrich, T., and Schulte, U. (2002). *Biochem. J.* **364**, 833–839.
- Dubini, A., Pye, R. L., Jack, R. L., Palmer, T., and Sargent, F. (2002). *Int. J. Hydrogen Energy* **27**, 1413–1420.
- Duin, E. C., Bauer, C., Jaun, B., and Hedderich, R. (2003). *FEBS Lett.* **538**, 81–84.
- Dupuis, A., Prieur, I., and Lunardi, J. (2001). *J. Bioenerg. Biomembr.* **33**, 159–168.
- Eismann, K., Mlejnek, K., Zipprich, D., Hoppert, M., Gerberding, H., and Mayer, F. (1995). *J. Bacteriol.* **177**, 6309–6312.
- Espositi, M. D. (1998). *Biochim. Biophys. Acta* **1364**, 222–235.
- Ferry, J. G. (1997). *Biofactors* **6**, 25–35.
- Fisher, N., and Rich, P. R. (2000). *J. Mol. Biol.* **296**, 1153–1162.
- Flemming, D., Hellwig, P., and Friedrich, T. (2003). *J. Biol. Chem.* **278**, 3055–3062.
- Fontecilla-Camps, J. C., and Ragsdale, S. W. (1997). *Adv. Inorg. Chem.* **47**, 283–333.
- Friedrich, T., Brors, B., Hellwig, P., Kintscher, L., Rasmussen, T., Scheide, D., Schulte, U., Mantele, W., and Weiss, H. (2000). *Biochim. Biophys. Acta* **1459**, 305–309.
- Friedrich, T., and Scheide, D. (2000). *FEBS Lett.* **479**, 1–5.
- Friedrich, T., and Weiss, H. (1997). *J. Theor. Biol.* **187**, 529–541.
- Galagan, J. E., Nusbaum, C., Roy, A., Endrizzi, M. G., MacDonald, P., FitzHugh, W., Calvo, S., Engels, R., Smirnov, S., Atwood, D., Brown, A., Allen, N., Naylor, J., Stange-Thomann, N., DeArellano, K., Johnson, R., Linton, L., McEvans, P., McKernan, K., Talamas, J., Tirrell, A., Ye, W., Zimmer, A., Barber, R. D., Cann, I., Graham, D. E., Grahame, D. A., Guss, A. M., Hedderich, R., Ingram-Smith, C., Kuettner, H. C., Krzycki, J. A., Leigh, J. A., Li,
Hellwig, P., Scheide, D., Bungert, S., Mäntele, W., and Friedrich, T. (2000). *Biochemistry* **39**, 10884–10891.

Hughes, P. E., and Tove, S. B. (1982). *J. Bacteriol.* **151**, 1397–1402.

Ide, T., Bäumer, S., and Deppenmeier, U. (1999). *J. Bacteriol.* **181**, 4076–4080.

Ito, M., Guffanti, A. A., and Krulwich, T. A. (2001). *FEBS Lett.* **496**, 117–120.

Kamlage, B., and Blaut, M. (1992). *J. Bacteriol.* **175**, 3043–3050.

Kemner, J. M., and Zeikus, J. G. (1994). *Arch. Microbiol.* **161**, 47–54.

Kerscher, S., Kashani-Poor, N., Zwicker, K., Zickermann, V., and Brandt, U. (2001). *J. Bioenerg. Biomembr.* **33**, 187–196.

Khalil, M. A. K., and Rasmussen, R. A. (1994). *Chemosphere* **29**, 833–842.

Künkel, A., Vaupel, M., Heim, S., Thauer, R. K., and Hedderich, R. (1997). *Eur. J. Biochem.* **244**, 226–234.

Künkel, A., Vorholt, J. A., Thauer, R. K., and Hedderich, R. (1998). *Eur. J. Biochem.* **244**, 226–234.

Kunow, K., Linder, D., Stetter, K. O., and Thauer, R. K. (1994). *Eur. J. Biochem.* **223**, 503–511.

Kurkin, S., Meuer, J., Koch, J., Hedderich, R., and Albracht, S. P. (2002). *Eur. J. Biochem.* **269**, 6101–6111.

Kvenvolden, K. (1999). *Proc. Natl. Acad. Sci. U.S.A.* **96**, 3420–3426.

Leif, H., Sled, V. D., Ohnishi, T., Weiss, H., and Friedrich, T. (1995). *Eur. J. Biochem.* **230**, 538–548.

Madadi-Kahkesh, S., Duin, E. C., Heim, S., Albracht, S. P. J., Johnson, M. K., and Hedderich, R. (2001). *Eur. J. Biochem.* **268**, 2566–2577.

Marchesi, J. R., Weightman, A. J., Cragg, B. A., Parkes, R. J., and Fry, J. C. (2001). *FEMS Microbiol. Ecol.* **34**, 221–228.

Mathiesen, C., and Hagerhall, C. (2002). *Biochim. Biophys. Acta* **1556**, 121–132.

Meuer, J., Bartoschek, S., Koch, J., Künkel, A., and Hedderich, R. (1999). *Eur. J. Biochem.* **265**, 325–335.

Meuer, J., Kuettner, H. C., Zhang, J. K., Hedderich, R., and Metcalf, W. W. (2002). *Proc. Natl. Acad. Sci. U.S.A.* **99**, 5632–5637.

Murakami, E., Deppenmeier, U., and Ragsdale, S. W. (2001). *J. Biol. Chem.* **276**, 2432–2439.

Noll, K. M., Donnelly, M. I., and Wolfe, R. S. (1986). *J. Biol. Chem.* **262**, 513–515.

Ohnishi, T. (1998). *Biochim. Biophys. Acta* **1364**, 186–206.

Pilkington, S. J., Skehel, J. M., Gennis, R. B., and Walker, J. E. (1991). *Biochemistry* **30**, 2166–2175.

Prieur, I., Lunardi, J., and Dupuis, A. (2001). *Biochim. Biophys. Acta* **1504**, 173–178.

Rasmussen, T., Scheide, D., Brors, B., Kintscher, L., Weiss, H., and Friedrich, T. (2001). *Biochemistry* **40**, 6124–6131.

Reay, D. S. (2003). *Biologist (London)* **50**, 15–19.

Schuler, F., and Casida, J. E. (2001). *Biochim. Biophys. Acta* **1506**, 79–87.

Seravalli, J., Gu, W., Tam, A., Strauss, E., Begley, T. P., Cramer, S. P., and Ragsdale, S. W. (2003). *Proc. Natl. Acad. Sci. U.S.A.* **100**, 3689–3694.

Shima, S., Warkentin, E., Thauer, R. K., and Ermler, U. (2002). *J. Biosci. Bioeng.* **93**, 519–530.

Simianu, M., Murakami, E., Brewer, J. M., and Ragsdale, S. W. (1998). *Biochemistry* **37**, 10027–10039.

Sorgenfrei, O., Müller, S., Pfeiffer, M., Sniezko, I., and Klein, A. (1997). *Arch. Microbiol.* **167**, 189–195.

Steuber, J. (in press). *J. Biol. Chem.* **278**, 26817–26822.

Terstegen, A., and Hedderich, R. (1999). *Eur. J. Biochem.* **264**, 930–943.

Thauer, R. K. (1998). *Microbiology* **144**, 2377–2406.

Vignais, P. M., Billoud, B., and Meyer, J. (2001). *FEMS Microbiol. Rev.* **25**, 455–501.

Volbeda, A., Charon, M. H., Piras, C., Hatchikian, E. C., Frey, M., and Fontecilla-Camps J. C. (1995). *Nature* **373**, 580–587.

Volbeda, A., Montet, Y., Vernede, X., Hatchikian, E. C., and Fontecilla-Camps, J. C. (2002). *Int. J. Hydrogen Energy.* **27**, 1449–1461

Walsh, C. (1986). *Acc. Chem. Res.* **19**, 216–221.

Westenberg, D. J., Braune, A., Ruppert, C., Müller, V., Herzberg, C., Gottschalk G., and Blaut, M. (1999). *FEMS Microbiol. Lett.* **170**, 389–398.

Wood, W. T., Gettrust, J. F., Chapman, N. R., Spence, G. D., and Hyndman, R. D. (2002). *Nature* **420**, 656–660.

Wu, L. F., Chanel, A., and Rodrigue, A. (2000). *Arch. Microbiol.* **173**, 319–324.

Yagi, T., and Matsuno-Yagi, A. (2003). *Biochemistry* **42**, 2266–2274.

Yano T., Magnitsky, S., and Ohnishi, T. (2000). *Biochim. Biophys. Acta* **1459**, 299–304.

Yano, T., and Ohnishi, T. (2001). *J. Bioenerg. Biomembr.* **33**, 213–222.
