
Microbiology (2004), 150, 1859–1867

**Rhodococcus opacus expresses the xsc gene to utilize taurine as a carbon source or as a nitrogen source but not as a sulfur source**

Karin Denger, Jürgen Ruff, David Schleheck and Alasdair M. Cook

Department of Biological Sciences, The University, D-78457 Konstanz, Germany

The Gram-positive bacteria *Rhodococcus opacus* ISO-5 and *Rhodococcus sp.* RHA 1 utilized taurine (2-aminoethanesulfonate) as the sole source of carbon or of nitrogen or of sulfur for growth. Different gene clusters and enzymes were active under these different metabolic situations. Under carbon- or nitrogen-limited conditions three enzymes were induced, though to different levels: taurine-pyruvate aminotransferase (Tpa), alanine dehydrogenase (Ald) and sulfoacetaldehyde acetyltransferase (Xsc). The specific activities of these enzymes in *R. opacus* ISO-5 were sufficient to explain the growth rates under the different conditions. These three enzymes were purified and characterized, and the nature of each reaction was confirmed. Analyses of the genome of *Rhodococcus sp.* RHA 1 revealed a gene cluster, *tauR-ald-tpa*, putatively encoding regulation and oxidation of taurine, located 20 kbp from the *xsc* gene and separate from two candidate phosphotransacetylase (*pta*) genes, as well as many candidate ABC transporters (*tauBC*). PCR primers allowed the amplification and sequencing of the *tauR-ald-tpa* gene cluster and the *xsc* gene in *R. opacus* ISO-5. The N-terminal sequences of the three tested proteins matched the derived amino acid sequences of the corresponding genes. The sequences of the four genes found in each *Rhodococcus* strain shared high degrees of identity (>95% identical positions). RT-PCR studies proved transcription of the *xsc* gene when taurine was the source of carbon or of nitrogen. Under sulfur-limited conditions no *xsc* mRNA was generated and no Xsc was detected. Taurine dioxygenase (TauD), the enzyme catalysing the anticipated desulfonative reaction when taurine sulfur is assimilated, was presumed to be present because oxygen-dependent taurine disappearance was demonstrated with taurine-grown cells only. A putative *tauD* gene (with three other candidates) was detected in strain ISO-5. Regulation of the different forms of metabolism of taurine remains to be elucidated.

---

**INTRODUCTION**

Taurine (2-aminoethanesulfonate) is a ‘phylogenetically ancient compound’ (Huxtable, 1992), which has long been known as a sole source of sulfur for growth of aerobic microorganisms (Huxtable, 1992; Uria-Nickelsen *et al.*, 1993). The compound also serves as a source of carbon and energy for many aerobic bacteria (Cook & Denger, 2002), and Chien *et al.* (1999) considered it to be a source of carbon, energy, nitrogen and sulfur for growth of *Rhodococcus* spp., though the nutritional elements were not all tested individually. Seitz *et al.* (1993) isolated bacteria which could apparently utilize taurine as a source of carbon and sulfur,

but which mutated to lose the ability to utilize it as a source of carbon: this indicates that different enzymes are involved in the physiologically different processes of carbon dissimilation, on the one hand, and assimilation of sulfonate-sulfur on the other (cf. Cook & Denger, 2002; Kertesz, 2000). Kertesz (2000) reviewed the nature of the desulfonative oxygenase (taurine dioxygenase [EC 1.14.11.17], TauD) (Elkins *et al.*, 2002) (Fig. 1B) involved in assimilation of sulfur from taurine in aerobic bacteria, and the global regulation of its expression. Cook & Denger (2002) reviewed the desulfonation of sulfoacetaldehyde involved in the dissimilation of taurine (sulfoacetaldehyde acetyltransferase [EC 2.3.3.15], Xsc) (Fig. 1A), and postulated regulated gene clusters involved in taurine catabolism in different bacteria (see also Brüggemann *et al.*, 2004; Ruff *et al.*, 2003).

*Abbreviations*: Ald, alanine dehydrogenase; PLP, pyridoxal 5′-phosphate; Pta, phosphate acetyltransferase; TauD, taurine dioxygenase; TauR, taurine regulator; Tpa, taurine-pyruvate aminotransferase; ThDP, thiamin diphosphate; Xsc, sulfoacetaldehyde acetyltransferase.

The GenBank accession numbers for the sequences reported in this paper are: *xsc*<sub>ISO-5</sub>, AY498611; *tauR-ald-tpa*<sub>ISO-5</sub>, AY498612.

*Rhodococcus opacus* ISO-5 was isolated for its ability to desulfonate many aromatic compounds (Schleheck *et al.*, 2003). Following on the work of Chien *et al.* (1999), the organism was found to utilize taurine as a sole source of

K. Denger and others

Taurine

Sulfo-
acetaldehyde

Acetyl phosphate

Acetyl-CoA

$\begin{aligned}
& \text { Tpa } \\
& \text { PLP } \\
& \text { Pyruvate } \\
& \text { Alanine } \\
& \text { NADH } \\
& \text { NH}_{4}^{+} \\
& \text { Ald } \\
& \text { H_{2}O }
\end{aligned}$

$\begin{aligned}
& \text { Pta } \\
& \text { ThDP } \\
& \text { Xsc } \\
& \text { HSO}_{3}^{-}
\end{aligned}$

$\begin{aligned}
& \text { Krebs } \\
& \text { Glyoxylate } \\
& \text { cycle } \\
& \text { cycle }
\end{aligned}$

$\begin{aligned}
& \text { Export } \\
& \text { HSO}_{3}^{-} \longrightarrow \text { SO}_{4}^{2-}
\end{aligned}$

Membrane

Export

NH$_{4}^{+}$

B

$\begin{aligned}
& \text { 2-Oxoglutarate } \\
& \text { Succinate + CO}_{2} \\
& \text { O}_{2} \\
& \text { TauD } \\
& \text { HSO}_{3}^{-} \longrightarrow \text { cellular organic sulfur }
\end{aligned}$

Fig. 1. Pathway of dissimilation of taurine in *R. opacus* ISO-5 with putative homeostatic mechanisms to detoxify sulfite and maintain the osmolarity in the cell (A), and dioxygenolytic desulfonation of taurine to aminoacetaldehyde for the assimilation of taurine sulfur (B). Different transport systems are inferred for the high fluxes involved in supplying carbon and nitrogen from taurine for growth (A), compared with the low flux and high affinity required to supply sulfur from taurine (B). Abbreviations not used in the Introduction: PLP, pyridoxal 5′-phosphate; ThDP, thiamin diphosphate.

carbon (Ruff et al., 2003) via an Xsc encoded by a gene in xsc subgroup 1 (Cook & Denger, 2002), or as a sole source of sulfur (Schleheck et al., 2003). It was also observed that Rhodococcus sp. strain RHA1 contained potential xsc and tauD genes (W. W. Mohn, unpublished data). Further analysis of the genome sequence showed the presence of candidates for genes encoding a regulator (TauR) (Brüggemann et al., 2004), taurine-pyruvate aminotransferase (Tpa) [EC 2.6.1.77] and alanine dehydrogenase (Ald) [EC 1.4.1.1], which suggested that the dissimilatory pathway in Fig. 1(A) was operative in Rhodococcus spp. This dissimilatory pathway is widespread (Cook & Denger, 2002), but analyses of the first genome sequences, which were found to contain putative xsc genes, reintroduced the alternative pathway via taurine dehydrogenase [EC 1.4.99.2, probably EC 1.4.2.-, TauXY] (Brüggemann et al., 2004; Kondo et al., 1973; Ruff et al., 2003).

We now report that *R. opacus* ISO-5 and Rhodococcus sp. strain RHA1, which were isolated on different continents for reasons independent of taurine metabolism, supplied two independent, but almost identical, pictures of the utilization of the individual nutritional macroelements of

taurine, with the advantage that the genome of one of these organisms is being sequenced. Taurine was dissimilated via Tpa, which had not been observed previously in a sequenced organism. The dissimilatory genes, and those encoding assimilative desulfonation, seem not to be clustered in either organism. Further, the high level of identity between corresponding genes in the two organisms was not matched in the DNA sequences surrounding those genes. The use of two organisms thus allowed us to confirm the generality of the phenomena under study.

### METHODS

#### Material. Commercial chemicals were of the highest purity available, and they were purchased from Fluka, Roth, Merck, Serva or Sigma. The preparation of sulfoacetaldehyde, as the bisulfite addition complex, was described elsewhere (Denger et al., 2001).

#### Organisms, growth, harvesting of cells and preparation of cell-free extracts. *Rhodococcus opacus* ISO-5 was isolated in this laboratory (Schleheck et al., 2003) and deposited with the German Culture Collection DSMZ, Braunschweig, Germany, under the accession number DSM 44600. *Rhodococcus* sp. strain RHA1 was a kind gift of W. W. Mohn, University of British Columbia, Vancouver, Canada.
Cells under carbon-limited conditions were grown aerobically at 30 °C in a phosphate-buffered mineral-salts medium (Thurnheer *et al.*, 1986) with 10–20 mM taurine or acetate, or 3 mM benzoate, *p*-aminobenzenesulfonate, 4-sulfobenzoate, 4-phenolsulfonate, 4-toluenesulfonate, benzenesulfonate, salicylate, 3-phenylpropionate, 4-phenylbutyrate or phenylacetate as the sole added source of carbon. Nitrogen-limited cells were grown aerobically at 30 °C in the same medium but with the omission of ammonium chloride and the addition of 5 mM glucose or 20 mM acetate as carbon source; taurine or ammonium ion (2 mM) served as sole added source of nitrogen. Sulfur-limited Tris-buffered salts medium (Laue *et al.*, 1996) with glucose or acetate as carbon source was used for sulfur-limited conditions. In this medium 100 μM taurine or sodium sulfate, or 30 μM 4-toluene-, benzene-, anthrachinone-2-, *p*-nitrobenzene-, *p*-aminobenzene-, *p*-chlorobenzene-, naphthalene-2-, phenyl-, octylbenzene-, 1,2-ethanedi-, methane-, ethane- or heptanesulfonate, or orange II, acid reds 26, 27, 88 or 112, azophloxine, Congo red, sodium dodecyl sulfate or 2-hydroxy-5-nitrophenylsulfate was the sole added source of sulfur.

Precultures and cultures for the determination of the substrate range (3 ml) were grown in 30 ml screw-cap tubes in a roller. Growth experiments were done on the 100 ml scale in 1 litre Erlenmeyer flasks on a shaker. Samples were taken at intervals to measure optical density at 580 nm, to assay protein (an OD~580~ of 1 corresponded to 186 mg protein l^−1^), and to determine the concentrations of taurine or sulfate. Similar cultures were used to generate small amounts of cells for cell-suspension experiments or enzyme assays. For enzyme purification, 1 litre cultures in 5 litre Erlenmeyer flasks were harvested at the end of the exponential growth phase by centrifugation (15 000 g, 20 min, 4 °C), washed in 50 mM potassium phosphate buffer, pH 7·5 (containing 2·5 mM MgCl~2~), and stored frozen. The same buffer served as extraction buffer. Cell-free extracts free of nucleic acids were generated after up to eight passages through a chilled French press cell at 138 MPa (Junker *et al.*, 1994).

**Cell suspension experiments.** Dense cell suspensions (OD~580~ about 20), pregrown with either sulfate or taurine as a sole source of sulfur, were incubated at 37 °C in 10 mM potassium phosphate buffer including 100 μM taurine. Samples were taken at intervals and subjected to derivatization. Anoxic controls were set up after repeatedly degassing, and gassing with nitrogen. Samples were taken with syringes through butyl rubber septa.

**Enzyme assays.** Taurine-pyruvate aminotransferase (Tpa) was assayed discontinuously as the disappearance of taurine concomitant with the formation of alanine; substrate and product were derivatized with dinitrofluorobenzene and separated and quantified by HPLC (Denger *et al.*, 1997). The reaction mixture at 37 °C contained (in a final volume of 1·0 ml): 50 μmol Tris/HCl buffer, pH 9·0, 0·1 μmol pyridoxal 5′-phosphate (PLP), 5 μmol taurine, 10 μmol pyruvate, and 0·05–0·5 mg protein, with which the reaction was started. The reverse reaction was assayed by following the disappearance of sulfoacetaldehyde after derivatization (Denger *et al.*, 2001) in a reaction mixture with the same buffer with PLP but with 2 μmol sulfoacetaldehyde and 10 μmol alanine.

Alanine dehydrogenase (Ald) was routinely measured photometrically as the reduction of NAD (Laue & Cook, 2000b) during oxidative deamination. Occasionally the reaction was confirmed by the assay of reductive amination of pyruvate following alanine formation or NADH oxidation (Laue & Cook, 2000b).

Sulfoacetaldehyde acetyltransferase (Xsc) was assayed by GC as the thiamin diphosphate (ThDP)- and phosphate-dependent release of acetate after acidification to hydrolyse the acetyl phosphate formed (Ruff *et al.*, 2003). The routine assay was occasionally augmented by the

colorimetric determination of acetyl phosphate, by the disappearance of substrate, or by the formation of sulfite (Ruff *et al.*, 2003).

Phosphate acetyltransferase (Pta) was assayed photometrically as the HS-CoA-dependent formation of acetyl-CoA (Bergmeyer *et al.*, 1983).

**Purification of Tpa, Ald and Xsc.** Particulate matter was removed from crude extract by ultracentrifugation (200 000 g, 30 min, 4 °C), and the soluble fraction (the supernatant fluid) was diluted 1:2·5 with distilled water to generate the correct buffer concentration for chromatography. This generated a precipitate, which was removed by centrifugation (10 000 g, 3 min, room temperature). The clear supernatant fluid was loaded on to a Mono Q anion-exchange column and protein was eluted as described elsewhere (Ruff *et al.*, 2003). Representative fractions were assayed for Tpa, Ald and Xsc.

Pooled fractions with Xsc activity were concentrated, desalted with equilibrated PD 10 columns (Pharmacia) according to the manufacturer’s instructions and subjected to a second separation on Mono Q equilibrated with 20 mM potassium phosphate buffer of higher pH (pH 6·9).

**Analytical methods.** Absorbance was measured in a Uvikon 922 spectrophotometer (Kontron). Sulfate was quantified as turbidity in the presence of BaCl~2~ (Sörbo, 1987). Ammonium ion was assayed colorimetrically by the Berthelot reaction (Gesellschaft-Deutscher Chemiker, 1996). Acetyl phosphate was determined chemically as iron(III) acetyl hydroxamate (Racker, 1962; Stadtman, 1957). Reversed-phase HPLC was used to quantify taurine (Laue *et al.*, 1997) or sulfoacetaldehyde (Cunningham *et al.*, 1998) after derivatization. Acetate was quantified by GC (Laue *et al.*, 1997). Protein was assayed by protein-dye binding (Bradford, 1976) or with whole cells by a Lowry-type method (Cook & Hütter, 1981). SDS-PAGE and staining were done by standard methods (Laemmli, 1970) to monitor protein purification and to estimate molecular masses under protein-denaturing conditions. The sequence of the N-terminal amino acids was determined by Edman degradation under contract by the sequencing facility of the Fachhochschule Bingen, Germany (http://zpa.fh-bingen.de/). Values of apparent *K*~m~ were derived by hyperbolic curve-fitting as cited elsewhere (Ruff *et al.*, 2003): the relevant ranges of concentration of substrates were 0–50 mM taurine with Tpa, 0–25 mM alanine for Ald and 0–20 mM sulfoacetaldehyde for Xsc.

**Amplification, nucleotide sequencing and sequence analysis of genes.** These were done as described elsewhere (Brüggemann *et al.*, 2004); PCR primers are given in Table 1. The protein sequences initially used in BLAST searches of the genome of *Rhodococcus* sp. strain RHA1 were TauR, TauABC, TauXY, Xsc and Pta from *Burkholderia* sp. strain LB400 (Brüggemann *et al.*, 2004; Ruff *et al.*, 2003), and Tpa (AF269146) and Ald (AF269148) from *Bilophila wadsworthia*.

**RT-PCR.** Cells for the preparation of total RNA were grown in the required selective medium and harvested in the mid-exponential growth phase (100–140 mg protein l^−1^); the bacterial culture was stabilized by addition of RNAProtect Bacteria Reagent (Qiagen) prior to harvesting. Total RNA from these cells (0·1 g wet weight) was prepared with the RNeasy Mini Kit and the RNase-free DNase set (Qiagen) following the protocol of the manufacturer, but with the following modifications: (i) resuspended cells in lysis buffer (manufacturer’s instructions) were disrupted in a bead-mill homogenizer (Fastprep FP120; Savant Instruments) at a setting of 6·5 m s^−1^ for 45 s, and the lysate was centrifuged (12 000 g, 5 min, 4 °C) to remove particulate matter; (ii) ethanol was added to the supernatant fluid (manufacturer’s instructions), and this solution was vortexed (5 s) to yield clumps of precipitated (chromosomal)
K.Denger and others

Table 1. PCR primers and anticipated product lengths with selected primer pairs

Analyses with the BLAST algorithm allowed a provisional annotation of genes in the genome of Rhodococcus sp. strain RHA1. Gene-specific PCR primers were selected or derived as appropriate. The PCR primers for the different xsc subgroups were derived from sequence alignments.

| Primer       | Sequence                              | Fragment length with given primer | Source of information           |
|--------------|---------------------------------------|------------------------------------|----------------------------------|
| Ald1         | CCCGCAGCGAGGGTGCGAGGTA                | 729 with Tpal                      | RHA1 genome sequence            |
| Tpal         | GGCGATGTGCSTYGCCGGCCGT                |                                    | RHA1 genome sequence            |
| Xscsg1-f     | GCCCGGGCATCAGCAACTGCCG                | 1289 with Xscsg1-r                 | This paper                      |
| Xscsg1-r     | GGCGGTTGTAGAAGTCCACCTG                |                                    | This paper                      |
| Xscsg3-f     | TGGGCTACAACGGYT CGAA                  | 726 with Xscsg3-r                 | Brüggemann et al. (2004)        |
| Xscsg3-r     | GCSCCCCCACTGGTAGTTGCG                 |                                    | Brüggemann et al. (2004)        |
| XscRT-f      | GATCATCCAGRTCGAIGCCGA                 | 860 with XscRT-r                  | RHA1 genome sequence            |
| XscRT-r      | TCAGACGTAGTCCTTGTACTTGTCGAGCA          |                                    | RHA1 genome sequence            |
| TauDuniv-f   | CAAGGTSCTGKTYYSTCG                    | 420 with TauDuniv-r               | RHA1 genome sequence            |
| TauDuniv-r   | YICGSACCACIGGRTGYTC                   |                                    | RHA1 genome sequence            |
| Taucand1-f   | GACGGCAACCGCGAGTTCGG                  | 118 with TauDuniv-r               | RHA1 genome sequence            |
| Taucand2-f   | CATCCC GGGCTCGTG CAGATC               | 331 with TauDuniv-r               | RHA1 genome sequence            |
| Taucand3-f   | CGATGTCACCGCGCTGATCAAT                | 262 with TauDuniv-r               | RHA1 genome sequence            |
| Taucand4-f   | TGCCAACCTGCCGGAATCGTTG                | 155 with TauDuniv-r               | RHA1 genome sequence            |

DNA, which were discarded prior to loading this solution on to the RNeasy-column (manufacturer's instructions). The First Strand cDNA Synthesis Kit (MBI Fermentas) was used for reverse transcription of RNA (1·0 μg) according to the manufacturer's instructions, and with the use of primer XscRT-r (Table 1). RNA was quantified photometrically (A260). cDNA from these reverse transcriptions was used as template (2 μl of reaction mixture) for PCR reactions (20 μl final volume) to amplify an 860 bp fragment of xsc (primer pair XscRT-f and XscRT-r; Table 1) under standard PCR conditions as described previously (Tralau et al., 2003). Controls without reverse transcriptase were used to detect any contamination of undigested DNA in the RNA preparations.

---

**RESULTS**

Carbon-limited growth with taurine initiated by transamination and desulfonation

R. opacus ISO-5 utilized taurine (or acetate) as sole source of carbon and energy for aerobic growth with a molar growth yield that showed mass balance for carbon (Table 2) and a specific growth rate (μ = 0·12 h⁻¹) which allow the specific utilization rate for taurine to be calculated as 3·1 mkat (kg protein)⁻¹. The sulfonate-sulfur was

---

Table 2. Growth of R. opacus ISO-5 in different media, with data on the presence of Tpa, Ald and Xsc detected as enzyme activity and by SDS-PAGE, and data on the transcript of the xsc gene

The acetate-salts medium (control experiment) contained ammonium ion as nitrogen source and sulfate ion as sulfur source.

| Acetate-salts medium | Growth medium containing taurine as sole source of: |
|----------------------|---------------------------------------------------|
|                     | Carbon                                      | Nitrogen                                  | Sulfur                                   |
| Molar growth yield*  | 6·0                                         | 5·4                                       | 2600                                     |
| Activity of Tpa [mkat(kg prot.)⁻¹] | ND                                        | 2·4                                       | ND                                       |
| Induced 50 kDa protein (Tpa)† | ND                                        | ++                                        | +                                        | ND                                       |
| Activity of Ald [mkat(kg prot.)⁻¹] | ND                                        | 10·2                                      | 2·3                                      | ND                                       |
| Induced 41 kDa protein (Ald)† | ND                                        | ++                                        | +                                        | ND                                       |
| Activity of Xsc [mkat(kg prot.)⁻¹] | ND                                        | 3·0                                       | 1·5                                      | ND                                       |
| Induced 64 kDa protein (Xsc)† | ND                                        | ++                                        | +                                        | ND                                       |
| Detected xsc mRNA†   | ND                                        | ++                                        | +                                        | ND                                       |

ND, Not detected.

*Mean molar growth yields are about 6 g protein (mol C)⁻¹, 50 g protein (mol N)⁻¹ and 3000 g protein (mol S)⁻¹ (Cook, 1987).

†Intensity of bands observed by SDS-PAGE or RT-PCR: ++, strong signals; +, weak signals.
recovered as sulfate, which was released concomitantly with growth, and ammonium ion was recovered in the growth medium (not shown). Traces of transient sulfite were also detected in the medium (not shown). *Rhodococcus* sp. strain RHA1 behaved similarly (not shown), which shows that the growth we observed previously (Cook & Denger, 2002; Ruff *et al.*, 2003) reflected overall mass balances.

The BLAST algorithm was used to make comparisons of genes encoding components of pathways of taurine metabolism (see Introduction and Fig. 1) with preliminary sequence data from the genome of strain RHA1. A single putative *xsc* gene was found. It was not contiguous (about 20 kbp distant from the *tauR-tpa-ald* quasi-cluster; see below) with genes known or presumed to be relevant to the regulation, transport or metabolism of taurine. PCR primers derived from the sequence of the *xsc* gene in strain RHA1 (*xsc*~*RHA1*~) allowed a fragment of DNA from strain ISO-5 to be amplified and sequenced. The sequence of the *xsc*~*ISO-5*~ gene shared 97% identical positions with *xsc*~*RHA1*~. This *xsc* gene allowed amplification of the anticipated product from the consensus PCR primers for *xsc*-subgroup 1 (Table 1) but not from primers for subgroup 3 (Table 1). There is negligible identity between the sequences of subgroup 1 and the few known members of subgroup 2, which is found in low G+C content Gram-positive bacteria, so the latter subgroup is also not represented. Further analyses of the genome in strain RHA1 indicated the presence of homologues of (i) a gene (*tauR*) postulated to encode regulation of taurine metabolism, (ii) three possible genes (*tpa*) encoding taurine-pyruvate amino-transferase, (iii) two possible genes (*ald*) encoding alanine dehydrogenases, (iv) two possible genes (*pta*) encoding phosphate acetyltransferases, and (v) many possible ATP-binding-cassette transporters (*tauBC*); the latter were not further examined. There was one apparent cluster, potentially *tauR-ald-tpa*, which was sequenced in strain ISO-5 as indicated above for the *xsc* gene. Homologues of the genes (*tauXY*) presumed to encode taurine dehydrogenase (Brüggemann *et al.*, 2004) were either absent (*tauX*), or

shared a low level of identity (*tauY*; 28% identical positions).

Strain ISO-5 was grown with acetate or taurine as sole source of carbon, and cell extracts were prepared. Inducible formation of Tpa [to 2·4 mkat (kg protein)^−1^], Ald [to 10·2 mkat (kg protein)^−1^] and Xsc [to 3·0 mkat (kg protein)^−1^] (Table 2) was detected in taurine-grown cells; Pta was apparently constitutive [0·1 mkat (kg protein)^−1^] and was not examined further. No taurine dehydrogenase was detected, as predicted from the absence of *tauXY* (see above). Proteins in these extracts were separated by SDS-PAGE, which showed the strong induction of three proteins (molecular masses of 50, 41 and 64 kDa) in the taurine-grown cells (Fig. 2); similar data were obtained with strain RHA1 (not shown). Enzyme purification from strain ISO-5 (see below) showed these proteins to be Tpa (50 kDa), Ald (41 kDa) and Xsc (64 kDa) (Fig. 2); the molecular masses corresponded to those derived from the gene sequences (see below).

A single Tpa eluted (at about 140 mM Na₂SO₄ in the gradient) at >95% purity from the anion-exchange column (Fig. 2); the specific activity of this material [15·0 mkat (kg protein)^−1^] represented about a sixfold purification. The identity of the enzyme, suggested by the routine assay by HPLC in which taurine disappeared and alanine was formed, was confirmed by following the reverse reaction in which sulfoacetaldehyde disappeared. The apparent *K*~m~ value for taurine was calculated to be 42±2 mM. The N-terminal heptapeptide was determined to be MDVTELR. This was identical with the sequence deduced from *tpa*~*ISO-5*~, assuming that the GUG codon (nominally encoding valine) encodes fMet (Kozak, 1999) under these conditions. This then allowed *tpa*~*RHA1*~ to be identified and distinguished from the other candidate genes to encode Tpa proteins (N-terminal sequences MTTVTNS or MSMASAL) of strain RHA1. Gene *tpa*~*ISO-5*~ was 1371 bp long.

A single Ald eluted (at about 130 mM Na₂SO₄ in the gradient) at >99% purity from the anion-exchange

|  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 |
|  |  |  |  |  |  |  | kDa |
|  |  |  |  |  |  |  | 97·4 |
|  |  |  |  |  |  |  | 66·2 |
|  |  |  |  |  |  |  | 45·0 |
|  |  |  |  |  |  |  | 31·0 |
|  |  |  |  |  |  |  | 21·5 |
|  |  |  |  |  |  |  | 14·0 |

Fig. 2. Electropherogram of protein fractions from extracts of *R.* *opacus* ISO-5 grown under different conditions. Lanes 1–4, crude extracts (30 µg) grown with: lane 1, acetate as carbon source; lane 2, taurine as sulfur source; lane 3, taurine as nitrogen source; lane 4, taurine as carbon source. Lanes 5–7, protein fractions after anion-exchange chromatography (5 µg): lane 5, Xsc; lane 6, Tpa; lane 7, Ald. Lane 8, low-molecular-mass standards.
K. Denger and others

column (Fig. 2); the specific activity of this material [47·8 mkat (kg protein)⁻¹] represented about a fivefold purification. The identity of the enzyme, indicated photometrically by the alanine-dependent reduction of NAD⁺ to NADH, was confirmed by the reverse reaction in which ammonium ion, pyruvate and NADH were essential to form alanine, which was quantified, after derivatization, by HPLC. The apparent \( K_m \) value for alanine was determined to be 3·9 ± 0·5 mM. The N-terminal amino acid sequence was determined to be MRIGIPK, which was identical to that derived from the nucleotide sequence of \( ald_{\text{ISO-5}} \) and to the sequence data of one RHA1 gene, which we annotated \( ald \). The other candidate gene in strain RHA1 was derived to contain a different N-terminus (MKIGIPR); this allele presumably encodes an Ald for a different pathway.

A single Xsc eluted (at about 160 mM Na₂SO₄ in the gradient) at about 90% purity from the anion-exchange column; the specific activity of this material [30 mkat (kg protein)⁻¹] represented about a tenfold purification. The purification was improved (Fig. 2, lane 5) by subsequent anion-exchange chromatography under slightly different conditions (not shown). The enzyme was identified by observing that sulfoacetaldehyde, phosphate and ThDP were essential to generate sulfite and acetyl phosphate. The apparent \( K_m \) value for sulfoacetaldehyde was calculated to be 2·6 ± 0·6 mM, typical for the subgroup 1 enzymes. The N-terminal amino acid sequence was determined to be XXXXNGADR. Allowing for cleavage of the initiatory methionine, an identical pentapeptide at the correct position was derived from the nucleotide sequence of the putative \( xsc_{\text{ISO-5}} \) and from the sequence data of strain RHA1. The complete sequence of \( xsc \) of strain ISO-5 (1809 bp) is amongst the longest of the 21 known \( xsc \) sequences; only the gene from *Achromobacter xylosoxidans* NCIMB 10751 is longer (1812 bp).

RT-PCR analysis showed that mRNA was transcribed from the \( xsc \) gene during growth with taurine as a carbon source (Fig. 3). The gene was not transcribed when acetate was the carbon source (Table 2). It was also not transcribed when either sulfate or taurine was the sole source of sulfur (Table 2).

Several separate loci are required for taurine dissimilation. The tauR-ald-tpa cluster involves divergent transcription of the ald and tpa genes, which are distant from the unknown genes encoding transport, from \( xsc \) and from pta. This situation differs from that in several α-Proteobacteria, where the genes are clustered (Brüggemann et al., 2004).

---

**Nitrogen-limited growth with taurine also initiated by transamination and desulfonation**

*R. opacus* ISO-5 utilized taurine as sole source of nitrogen for aerobic growth with a molar growth yield that showed mass balance for nitrogen (Table 2) and a specific growth rate (\( \mu = 0 \cdot 1 \) h⁻¹) which allow the specific utilization rate for taurine to be calculated as 0·5 mkat (kg protein)⁻¹. The

---

Fig. 3. RT-PCR of the mRNA transcribed from the \( xsc \) gene during growth of *R. opacus* ISO-5 with taurine as carbon source (C-limited) and nitrogen source (N-limited). Key: +, PCR reactions with cDNA as template from reverse-transcribed total RNA; –, negative controls from PCR with total RNA as template; P, positive control from PCR with DNA from strain ISO-5 as template.

sulfonate sulfur was recovered as sulfate, which was released concomitantly with growth (not shown). Extracts of cells utilizing taurine as the sole source of nitrogen contained inducible Tpa [0·7 mkat (kg protein)⁻¹], Ald [2·3 mkat (kg protein)⁻¹] and Xsc [1·5 mkat (kg protein)⁻¹, Table 2], and these denatured enzymes were visible in SDS-PAGE (Fig. 2). Analysis by RT-PCR showed that mRNA was transcribed from the \( xsc \) gene (Fig. 3). This indicated that the same degradative pathway in its entirety was used in dissimilation of taurine carbon and in the assimilation of taurine nitrogen. *Rhodococcus* sp. strain RHA1 behaved similarly (not shown).

---

**Sulfur-limited growth with taurine independent of Xsc**

Strain ISO-5 utilized taurine as the sole source of sulfur for aerobic growth (\( \mu = 0 \cdot 11 \) h⁻¹) with the standard molar growth yield (Table 2); this represents a specific degradation rate of 10 μkat (kg protein)⁻¹. Similar data were obtained for *Rhodococcus* sp. strain RHA1 (not shown). No Xsc was detected in these cells, no induction of Xsc was detected by SDS-PAGE (Fig. 2) and no mRNA from the \( xsc \) gene was detected (Table 2). So the utilization of taurine was independent of Xsc, as predicted in the Introduction.

The predicted desulfonation reaction under these metabolic conditions is catalysed by taurine dioxygenase, TauD (see Introduction). Cells of strain ISO-5 grown with limiting sulfate as the sulfur source did not catalyse disappearance of taurine (Fig. 4). In contrast, cells grown with taurine as the sole sulfur source displayed oxygen-dependent taurine disappearance (Fig. 4) with a specific activity of 9·3 μkat (kg protein)⁻¹. This was interpreted as preliminary evidence for the presence of taurine dioxygenase, TauD. The genome of strain RHA1 was found to contain a major candidate for a \( tauD \) gene (no gaps) with three minor candidates (with gaps). In addition, a putative \( atsK \) gene, also encoding a

Taurine metabolism in *Rhodococcus opacus*

![Graph](https://i.imgur.com/1234567.png)

Fig. 4. Inducible, oxygen-dependent disappearance of taurine catalysed by cell suspensions of *R. opacus* ISO-5. Cells were cultured with sulfate or taurine as the sole source of sulfur for growth. The taurine-grown cells are presumed to express TauD. △, Sulfate-grown cells incubated under an air atmosphere; ○, taurine-grown cells incubated under nitrogen; ■, taurine-grown cells incubated under an air atmosphere.

identical positions in the corresponding genes in strains ISO-5 and RHA1. In addition, the four candidate *tauD* genes were detected in strain ISO-5 using the primers developed for strain RHA1, so high levels of identity are obviously present here, too. We then wondered whether we could detect any significant differences between the organisms. One difference was that PCR primers from the 3′ end of the *tauR*<sub>RHA1</sub> gene or further downstream gave no amplification product with any potential reverse primer from the 5′ end of the gene. Analyses with the BLAST algorithm indicated potential *ssuD*, *asfAB*, *asaA* and *atsK* genes on the RHA1 genome, and the corresponding sulfur substrates (see Methods), alkylsulfonates (*ssu* genes), arylsulfonates (*ssu* and *asf* genes), aryl sulfate ester (*asaA* gene) and alkyl sulfate ester (*atsK* gene) supported growth of the organism. Strain ISO-5 grew with alkylsulfonates, arylsulfonates and alkyl sulfate ester, but one aryl sulfate ester (2-hydroxy-5-nitrophenylsulfate) did not support growth, so one difference in the range of sulfur sources was evident. Neither organism grew with five tested arylsulfonates as sources of carbon. Each organism could utilize glucose, benzoate, 3-(phenyl)propionate and 4-(phenyl)butyrate, but they differed in that only strain RHA1 could utilize phenylacetate.

2-oxoglutarate-coupled dioxygenase, was detected. The candidates for the *tauD* gene shared two common segments of sequence that allowed a PCR primer pair (*tauD*<sub>univ-f</sub> and *tauD*<sub>univ-r</sub>) (Table 1) to be generated, which should amplify a fragment from each of these four genes in strain RHA1, and from the *tauD* gene from *Pseudomonas putida*, *Pseudomonas aeruginosa* or *E. coli*. The amplification yielded two bands, one with the expected size range (420 nt) and one (530 nt) which was generated with one primer alone (*tauD*<sub>univ-r</sub>). We developed specific forward primers for each candidate gene (*tauD*<sub>cand1-f</sub> to *tauD*<sub>cand4-f</sub>) (Table 1), but we had to use *tauD*<sub>univ-r</sub> as the reverse primer in each case. The 420 nt band was used as a template, and the amplification products we obtained (118, 155, 262 and 331 nt; not shown) showed that all four major and minor candidates for the *tauD* gene were present in strain ISO-5 (see Table 1). RT-PCR experiments to detect mRNA transcribed from a *tauD*-candidate during growth of strain ISO-5 with taurine as sulfur source were set up with primer *tauD*<sub>univ-r</sub> for reverse transcription, and with the primer pair *tauD*<sub>univ-f</sub> plus *tauD*<sub>univ-r</sub> for subsequent PCR. This yielded neither a specific 420 nt fragment nor a non-specific 530 nt fragment (not shown). We suspected that this particular mRNA is transcribed only in low levels (corresponding to the low specific activity of TauD, see above), and we desisted from further RT-PCR experiments.

It thus seems likely that TauD is involved in the desulfo- nation of taurine under sulfur-limiting conditions, but it is currently impracticable to identify which of the candidate genes encodes this TauD.

### Differences between *R. opacus* ISO-5 and *Rhodococcus* sp. strain RHA1

We have sequenced five genes in strain ISO-5 (*tauR*, *ald*, *tpa*, *xsc* and putative *atsK*) and we observed about 95 %

---

DISCUSSION

The dissimilation of taurine via Tpa is widespread in Gram-positive and Gram-negative bacteria, and the pathway in aerobes is now known to end at acetyl-CoA, where it enters the Krebs and glyoxylate cycles (Fig. 1) (Cook & Denger, 2002). Activity of each of the four metabolic enzymes involved, Tpa, Ald, Xsc and Pta, can be assayed in *R. opacus* ISO-5 and *Rhodococcus* sp. strain RHA1. Whereas Tpa, Ald and Xsc are obviously induced to specific activities [2–10 mkat (kg protein)<sup>−1</sup>] which can essentially explain the growth rate [3·1 mkat (kg protein)<sup>−1</sup>], we wonder whether we may have suboptimal conditions for the assay of the apparently constitutive Pta.

We have confirmed directly that taurine nitrogen is a sole source of nitrogen for growth of two *Rhodococcus* spp. (compare Chien et al., 1999) in the presence of excess carbon source. The organisms use the same degradative pathway [Tpa, Ald, Xsc (Table 2), and presumably Pta], because sulfate is released, and involve the only *xsc* gene (Fig. 3), so presumably the same regulator (putative TauR) is involved. TauR obviously controls the level of expression economically, because different levels of expression are found appropriate to the different fluxes that are required [3·0 and 1·5 mkat (kg protein)<sup>−1</sup>].

Another aspect of the dissimilation of taurine that requires regulation is the excretion of sulfite and sulfate on the one hand, and ammonium ion on the other (Fig. 1). This control will require modulation when the cells utilize taurine as a sole source of nitrogen, where the same regulator, TauR, presumably in cooperation with other factors, will presumably prevent the export of ammonium

http://mic.sgmjournals.org

ion, while still excreting sulfate. Aspects of nitrogen control are discussed elsewhere (e.g. Magasanik, 1996).

Tpa [EC 2.6.1.77] has been purified only once before (Laue & Cook, 2000a), but a gene whose product encodes the Tpa involved in the anaerobic assimilation of taurine-sulfur in *Rhodobacter capsulatus* has been identified (Masepohl *et al.*, 2001). These three gene products, Tpa<sub>Bw</sub>, Tpa<sub>Rc</sub> and Tpa<sub>ISO-5</sub> (with the almost identical Tpa<sub>RHA1</sub>), belong to COG1061, according to the Domain Search Tool of the NCBI BLAST server, but the levels of identity amongst them is low: Tpa<sub>RHA1</sub>, Tpa<sub>Bw</sub> and Tpa<sub>Rc</sub> share 99%, 32% and 30% identical positions, respectively, with Tpa<sub>ISO-5</sub>, while Tpa<sub>Bw</sub> and Tpa<sub>Rc</sub> share 57% identical positions. This poor correlation of sequence with function suggests that, of the three *tpa*-like genes in strain RHA1, only that annotated *tpa* actually encodes a Tpa. Heterogeneity in aminotransferases is, apparently, not unusual (Mehta & Christen, 2000). The initial codon of the *tpa* gene, GUG, obviously encodes fMet. This codon is frequently used in strain RHA1 (W. W. Mohn, personal communication) so *Rhodococcus* sp. strain RHA1 presumably uses this initial codon much more commonly than the 10% reported for prokaryotes in a review of initiation of translation (Kozak, 1999).

There was only one candidate *xsc* gene in strain RHA1, though candidate genes for other ThDP-coupled enzymes, e.g. *ilvB*, were detected (see Ruff *et al.*, 2003). As with Tpa and Ald, the N-terminal amino acid sequence of Xsc obtained with microsequencing confirmed the tentative identification of the corresponding gene. The *xsc*<sub>RHA1</sub> gene (and *xsc*<sub>ISO-5</sub>) shared 78–87% identical positions with the other nine genes in subgroup 1 of *xsc*, and the appropriate portion of the gene was amplified with the consensus primers for subgroup 1 (Table 1): these data confirm and expand the unpublished data in our earlier attribution (Ruff *et al.*, 2003). Up till now, we had the impression that subgroup 1 of *xsc* was restricted to β-Proteobacteria. Obviously the subgroup is also found in Gram-positive organisms, which have a similar G+C content to that in the β-Proteobacteria.

Just as there was only one candidate for the *xsc* gene in strain RHA1, there was just one candidate for the putative regulator gene, *tauR* (dendrogram in Brüggemann *et al.*, 2004) and the level of identical positions in the corresponding section of incomplete *tauR*<sub>ISO-5</sub> was 95%. If *TauR*<sub>ISO-5</sub> is closely related to homologues, the pattern of regulation of genes is somewhat different. In the α-Proteobacteria, the regulated genes are located in a cluster that could be an operon (in *Sinorhizobium meliloti*), whereas the cluster in the β-Proteobacteria (e.g. *Burkholderia fungorum* LB 400) is a less structured entity (see Brüggemann *et al.*, 2004). In strains RHA1 and ISO-5 there is a different start of transcription for *tpa, ald, xsc* and transport, while *pta* may be differently regulated. As this is the first transamination pathway for taurine dissimilation to be explored on a sequenced genome, it

is possible that this non-contiguous pattern is normal. However, preliminary annotation of the genome of *Rhodococcus* sp. strain RHA1 indicates that most pathways are encoded on clustered genes (W. W. Mohn, unpublished data).

*R. opacus* ISO-5 was isolated for its ability to desulfonate quantitatively a multitude of complex sulfonated aromatic compounds (Schleheck *et al.*, 2003) as sulfur sources for growth, and *Rhodococcus* sp. strain RHA1 can also desulfonate many aromatic sulfonates (listed in Methods). Genes corresponding to the known desulfonation complex of desulfonation enzymes (*Ssu* and *Ats*) are present in strain RHA1. Several other desulfonation or sulfoesterase enzymes must be present in both organisms, and the corresponding genes can be detected in the RHA1 genome. These enzymes were initially found in Gram-negative bacteria, each encoded in a cluster which also encodes an ABC transport system (Kertesz, 2000, 2001; Kertesz & Kahnert, 2001). Many ABC transporter systems can be detected in the RHA1 genome, sometimes with a potential A gene (periplasmic binding protein), and often without. In no case is a potential ABC transporter encoded adjacent to genes encoding enzymes to release sulfite of sulfate from a source of sulfur. This situation resembles that seen above for the dissimilation of taurine.

There is mass balance for the assimilation of taurine-sulfur (Table 2) and the specific degradation rate of taurine calculated under these conditions [10 μkat (kg protein)<sup>−1</sup>] can be observed in whole cells (Fig. 4). *Xsc* is not involved in this assimilation (see Table 2), and we interpret the enzymic and PCR data to indicate the presence of TauD. Independent sets of metabolism for quantitative dissimilation of taurine-carbon or the quantitative assimilation of taurine-sulfur are thus confirmed, which is scarcely surprising given the need for a high turnover of taurine at high concentrations [0·5–3 mkat (kg protein)<sup>−1</sup> at 2–10 mM taurine] or a low turnover at low concentrations [10 μkat (kg protein)<sup>−1</sup> at 30 μM]. When taurine is serving as a source of sulfur under anoxic conditions, however, TauD (taurine dioxygenase) cannot be functional, and a *tpa* gene with an *xsc* gene have been shown in genetic experiments to be involved in desulfonation in *Rhodobacter capsulatus* (Masepohl *et al.*, 2001), while another group has found a Tpa under corresponding conditions in *Clostridium pasteurianum* (Chien *et al.*, 1997), so presumably an Xsc is involved in the desulfonation. It remains to be seen whether the kinetic properties of these desulfonative enzymes involve a higher affinity for the sulfonate than that (about 5 mM) found in the subgroup 1 enzymes examined to date (Ruff *et al.*, 2003).

## ACKNOWLEDGEMENTS

We are grateful to participants in an advanced practical class and an exchange programme for data: Sarah Meier and Katharina Styp von Rekowski for work on Tpa, Ald and Xsc; Lihi Khazoom and Maren Steinberg for sequence data; Astrid Magenau and Sonja Weinitschke

for RT-PCR data. Andreas Brune kindly introduced us to the bead-
mill. Sequence data from the genome of *Rhodococcus* sp. strain RHA1
were kindly made available by the Microbial Envirogenomics
team of Genome Canada, mediated by W. W. Mohn, University of
British Columbia, before the data became available online (http://
www.bcgsc.bc.ca/cgi-bin/rhodococcus/blast_rha1.pl). The research
was funded by the University of Konstanz.

---

**REFERENCES**

Bergmeyer, H. U., Graßl, M. & Walter, E.-M. (1983). Phospho-
transacetylase. In Methods of Enzymatic Analysis, pp. 295–296. Edited
by H. U. Bergmeyer. Weinheim: Verlag Chemie.

Bradford, M. (1976). A rapid and sensitive method for the quantita-
tion of microgram quantities of protein utilizing the principle of
protein-dye binding. Anal Biochem 72, 248–254.

Brüggemann, C., Denger, K., Cook, A. M. & Ruff, J. (2004). Enzymes
and genes of taurine and isethionate dissimilation in *Paracoccus*
denitrificans. Microbiology 150, 805–816.

Chien, C.-C., Leadbetter, E. R. & Godchaux, W., III (1997). Taurine-
sulfur assimilation and taurine-pyruvate aminotransferase activity in
anaerobic bacteria. Appl Environ Microbiol 63, 3021–3024.

Chien, C.-C., Leadbetter, E. R. & Godchaux, W., III (1999). *Rhodococcus*
spp. utilize taurine (2-aminoethanesulfonate) as sole
source of carbon, energy, nitrogen and sulfur for aerobic respiratory
growth. FEMS Microbiol Lett 176, 333–337.

Cook, A. M. (1987). Biodegradation of s-triazine xenobiotics. FEMS
Microbiol Rev 46, 93–116.

Cook, A. M. & Hütter, R. (1981). s-Triazines as nitrogen sources for
bacteria. J Agric Food Chem 29, 1135–1143.

Cook, A. M. & Denger, K. (2002). Dissimilation of the C₂ sulfonates.
Arch Microbiol 179, 1–6.

Cunningham, C., Tipton, K. F. & Dixon, H. B. F. (1998). Conversion
of taurine into N-chlorotaurine (taurine chloramine) and sulfo-
acetaldehyde in response to oxidative stress. Biochem J 330, 939–945.

Denger, K., Laue, H. & Cook, A. M. (1997). Anaerobic taurine
oxidation: a novel reaction by a nitrate-reducing *Alcaligenes* sp.
Microbiology 143, 1919–1924.

Denger, K., Ruff, J., Rein, U. & Cook, A. M. (2001). Sulfoacetaldehyde
sulfo-lyase [EC 4.4.1.12] from *Desulfonispora thiosulfatigenes*: purifi-
cation, properties and primary structure. Biochem J 357, 581–586.

Elkins, J. M., Ryle, M. J., Clifton, I. J., Dunning Hotopp, J. C.,
Lloyd, J. S., Burzlaff, N. I., Baldwin, J. E., Hausinger, R. P. & Roach,
P. L. (2002). X-ray crystal structure of *Escherichia coli* taurine/α-
ketoglutarate dioxygenase complexed to ferrous iron and substrates.
Biochemistry 41, 5185–5192.

Gesellschaft-Deutscher-Chemiker (1996). German
Standard
Methods for the Laboratory Examination of Water, Waste Water
and Sludge. Weinheim: VCH.

Huxtable, R. J. (1992). Physiological actions of taurine. Physiol Rev
72, 101–163.

Junker, F., Leisinger, T. & Cook, A. M. (1994). 3-Sulphocatechol 2,3-
dioxygenase and other dioxygenases (EC 1.13.11.2 and EC 1.14.12.–)
in the degradative pathways of 2-aminobenzenesulphonic, benzene-
sulphonic and 4-toluenesulphonic acids in *Alcaligenes* sp. strain O-1.
Microbiology 140, 1713–1722.

Kertesz, M. A. (2000). Riding the sulfur cycle – metabolism of
sulfonates and sulfate esters in Gram-negative bacteria. FEMS
Microbiol Rev 24, 135–175.

Kertesz, M. A. (2001). Bacterial transporters for sulfate and
organosulfur compounds. Res Microbiol 152, 279–290.

Kertesz, M. A. & Kahnert, A. (2001). Organoschwefel-Metabolismus
in Gram-negativen Bakterien. Biospektrum 7, 325–329.

Kondo, H., Kagotani, K., Oshima, M. & Ishimoto, M. (1973). Purification and some properties of taurine dehydrogenase from a
bacterium. J Biochem 73, 1269–1278.

Kozak, M. (1999). Initiation of translation in prokaryotes and
eukaryotes. Gene 234, 187–208.

Laemmli, U. K. (1970). Cleavage of structural proteins during the
assembly of the head of bacteriophage T4. Nature London 227,
680–685.

Laue, H. & Cook, A. M. (2000a). Biochemical and molecular charac-
terization of taurine : pyruvate aminotransferase from the anaerobe
*Bilophila wadsworthia*. Eur J Biochem 267, 6841–6848.

Laue, H. & Cook, A. M. (2000b). Purification, properties and primary structure of alanine dehydrogenase involved in
taurine metabolism in *Bilophila wadsworthia*. Arch Microbiol 174,
162–167.

Laue, H., Field, J. A. & Cook, A. M. (1996). Bacterial desulfonation of the ethanesulfonate metabolite of the chloroacetanilide herbicide
metazachlor. Environ Sci Technol 30, 1129–1132.

Laue, H., Denger, K. & Cook, A. M. (1997). Taurine reduction in
anaerobic respiration of *Bilophila wadsworthia* RZATAU. Appl
Environ Microbiol 63, 2016–2021.

Magasanik, B. (1996). Regulation of nitrogen utilization. In *Escherichia coli* and *Salmonella*: Cellular and Molecular Biology,
pp. 1344–1356. Edited by F. C. Neidhardt and others. Washington,
DC: American Society for Microbiology.

Masepohl, B., Führer, F. & Klipp, W. (2001). Genetic analysis of a
*Rhodobacter capsulatus* gene region involved in utilization of taurine
as a sulfur source. FEMS Microbiol Lett 205, 105–111.

Mehta, P. K. & Christen, P. (2000). The molecular evolution of
pyridoxal-5′-phosphate-dependent enzymes. Adv Enzymol Relat
Areas Mol Biol 74, 129–184.

Racker, E. (1962). Fructose-6-phosphate phosphoketolase from
*Acetobacter xylinum*. Methods Enzymol 5, 276–280.

Ruff, J., Denger, K. & Cook, A. M. (2003). Sulfoacetaldehyde
acetyltransferase yields acetyl phosphate: purification from *Alcali-
genes defragrans* and gene clusters in taurine degradation. Biochem J
369, 275–285.

Schleheck, D., Lechner, M., Schönenberger, R., Suter, M. J.-F. &
Cook, A. M. (2003). Desulfonation and degradation of sulfodiphenyl-
ethercarboxylates from linear alkylphenyletherdisulfonate surfac-
tants. Appl Environ Microbiol 69, 938–944.

Seitz, A. P., Leadbetter, E. R. & Godchaux, W. III (1993). Utiliza-
tion of sulfonates as sole sulfur source by soil bacteria including
*Comamonas acidovorans*. Arch Microbiol 159, 440–444.

Sörbo, B. (1987). Sulfate: turbidimetric and nephelometric methods.
Methods Enzymol 143, 3–6.

Stadtman, E. R. (1957). Preparation and assay of acetyl phosphate.
Methods Enzymol 3, 228–231.

Turnheer, T., Köhler, T., Cook, A. M. & Leisinger, T. (1986). Orthanilic acid and analogues as carbon sources for bacteria:
growth physiology and enzymic desulphonation. J Gen Microbiol
132, 1215–1220.

Tralau, T., Cook, A. M. & Ruff, J. (2003). An additional regulator,
TsaQ, is involved with TsaR in regulation of transport during the
degradation of p-toluenesulfonate in *Comamonas testosteronei* T-2.
Arch Microbiol 180, 319–326.

Uria-Nickelsen, M. R., Leadbetter, E. R. & Godchaux, W., III (1993).
Sulfonate-sulfur assimilation by yeasts resembles that of bacteria.
FEMS Microbiol Lett 114, 73–78.
