
JOURNAL OF BACTERIOLOGY, Aug. 2004, p. 5400–5409  
0021-9193/04/$08.00+0 DOI: 10.1128/JB.186.16.5400–5409.2004  
Copyright © 2004, American Society for Microbiology. All Rights Reserved.

**Isoleucine Biosynthesis in *Leptospira interrogans* Serotype lai Strain 56601 Proceeds via a Threonine-Independent Pathway†‡**

Hai Xu,$^{1,2}$ Yuzhen Zhang,$^{2}$ Xiaokui Guo,$^{3}$ Shuangxi Ren,$^{4}$ Andreas A. Staempfli,$^{5}$ Juishen Chiao,$^{1}$ Weihong Jiang,$^{1}$ and Guoping Zhao$^{1,4*}$

Laboratory of Microbial Molecular Physiology, Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai 200032,$^{1}$ State Key Laboratory of Microbial Technology, Shandong University, Jinan, Shandong 250100,$^{2}$ Department of Microbiology and Parasitology, Shanghai Second Medical University, Shanghai 200025,$^{3}$ and Chinese National Human Genome Center at Shanghai, Shanghai 201203,$^{4}$ China, and F. Hoffmann-La Roche Ltd., CH-4070 Basel, Switzerland$^{5}$

Received 2 February 2004/Accepted 6 May 2004

Three *leuA*-like protein-coding sequences were identified in *Leptospira interrogans*. One of these, the *cimA* gene, was shown to encode citramalate synthase (EC 4.1.3.-). The other two encoded α-isopropylmalate synthase (EC 4.1.3.12). Expressed in *Escherichia coli*, the citramalate synthase was purified and characterized. Although its activity was relatively low, it was strictly specific for pyruvate as the keto acid substrate. Unlike the citramalate synthase of the thermophile *Methanococcus jannaschii*, the *L. interrogans* enzyme is temperature sensitive but exhibits a much lower $K_{m}$ (0.04 mM) for pyruvate. The reaction product was characterized as (*R*)-citramalate, and the proposed β-methyl-D-malate pathway was further confirmed by demonstrating that citraconate was the substrate for the following reaction. This alternative pathway for isoleucine biosynthesis from pyruvate was analyzed both in vitro by assays of leptospiral isopropylmalate isomerase (EC 4.2.1.33) and β-isopropylmalate dehydrogenase (EC 1.1.1.85) in *E. coli* extracts bearing the corresponding clones and in vivo by complementation of *E. coli ilvA*, *leuC/D*, and *leuB* mutants. Thus, the existence of a leucine-like pathway for isoleucine biosynthesis in *L. interrogans* under physiological conditions was unequivocally proven. Significant variations in either the enzymatic activities or mRNA levels of the *cimA* and *leuA* genes were detected in *L. interrogans* grown on minimal medium supplemented with different levels of the corresponding amino acids or in cells grown on serum-containing rich medium. The similarity of this metabolic pathway in leptospires and archaea is consistent with the evolutionarily primitive status of the eubacterial spirochetes.

The leptospires are a physiologically unique genus of spirochetes that include the saprophyte *Leptospira biflexa* and the pathogen *Leptospira interrogans*. The latter organism is the etiologic agent of leptospirosis, a worldwide waterborne zoonosis that constitutes a health threat not only in tropical and subtropical countries but also in cities where sanitation is substandard and where rats serve as reservoirs (16).

Spirochetes are an evolutionarily primitive species of bacteria (8, 42). As suggested by whole-genome sequencing, the metabolism of *L. interrogans* differs extensively from that of other well-studied bacteria and from that of two other obligatory parasitic spirochetes, *Treponema pallidum* (18) and *Borrelia burgdorferi* (17).

The biosynthesis of isoleucine is a case in point. In most microorganisms, isoleucine is synthesized from aspartate via threonine (38). L-Threonine dehydratase (deaminase, *ilvA*; EC 4.2.1.16) is the key enzyme in this pathway. However, alternative routes to isoleucine from precursors other than threonine have been reported. Some anaerobes can assimilate 2-methylbutyrate into isoleucine (33). In some pseudorevertants of *ilvA* mutants, α-ketobutyrate can arise from precursors other than threonine (3, 6, 9, 11, 13, 14, 15, 30, 39, 41). Of the latter, the most commonly observed was a route from pyruvate and acetyl coenzyme A (acetyl-CoA) via citramalate (3, 9, 39, 41). This “pyruvate pathway” was initially proposed for the genus *Leptospira* (9, 41) because isotope-labeling experiments indicated that in some leptospiral strains, α-ketobutyrate was derived from pyruvate rather than threonine. Only a limited number of leptospires possess catabolic threonine dehydratase (41). Later, a similar observation was made with a thermophilic archaeon, *Methanobacterium thermoautotrophicum*, suggesting that isoleucine biosynthesis involves pyruvate as a precursor (14). Recently, (*R*)-citramalate synthase (EC 4.1.3.-) activity was demonstrated in the thermophilic archaeon *Methanococcus jannaschii* (21).

We were unable to identify an *ilvA* gene in *L. interrogans* during genome annotation but found three putative *leuA* protein-coding sequences (CDSs LA0469, LA2202, and LA2350) having LeuA domains associated with isopropylmalate, homocitrate, and citramalate synthases (see supplemental material) (32). Here, we demonstrate that LA2350 encodes a citramalate synthase (CimA) while LA0469 encodes an α-isopropylmalate synthase (EC 4.1.3.12) (LeuA). In addition, we show that the α-isopropylmalate isomerase (EC 4.2.1.33)

* Corresponding author. Mailing address: Laboratory of Microbial Molecular Physiology, Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, 300 Fenglin Rd., Shanghai 200032, China. Phone: 86-21-64042090. Fax: 86-21-64042385. E-mail: gpzhao@sibs.ac.cn.
† Supplemental material for this article may be found at http://jb.asm.org/.
‡ Dedicated to late professor emeritus H. E. Umbarger (Purdue University, Lafayette, Ind.) in recognition of his long-time education, care, and help to G.P.Z. while he was studying at Purdue University during the 1980s and for his lifetime contributions to the study of branched-chain amino acid biosynthesis in bacteria.

Half of the cells were used to prepare extracts for enzyme assays, while the remainder was used to prepare total RNA for the real-time reverse transcription (RT)-PCR analysis. *E. coli* was routinely grown in LB medium (35) at 37°C, except where indicated otherwise. The solid medium used contained 1.0% agar. Antibiotics, when required, were used at the following concentrations: ampicillin, 100 μg/ml; chloramphenicol, 35 μg/ml; kanamycin, 50 μg/ml. Minimal medium M9 (35) supplemented with the appropriate amino acids (histidine, methionine, and arginine) and thiamine (each at 200 μg/ml) was used to cultivate *E. coli* mutants. In complementation experiments, amino acids (isoleucine, valine, and leucine) and α-ketobutyrate were added to the minimal medium individually at 200 μg/ml.

General molecular biology techniques. Bacterial genomic DNA was prepared by a small-scale method (1). Restriction endonucleases, *Taq* DNA polymerases, EZ Spin Column DNA Gel Extraction Kit, DNA Ligation Kit Ver.2 (from TaKaRa, Sangon, and Promega Corp.) were used in accordance with the manufacturers' protocols. Standard techniques were used for agarose gel electrophoresis, small-scale plasmid isolation, and bacterial transformation (26). The DNA sequencing and analysis methods used have been described previously (32).

Cloning and heterologous expression of *L. interrogans* genes. The DNA of genes of interest was synthesized by PCR with genomic DNA of strain 56601 as the template. The design of the following primers was based on the genomic sequences of individual genes with the incorporation of NdeI/HindIII, NdeI/EcoRI, or NheI/HindIII restriction sites (underlined): LA2202, 5′-CATATG GTTTCGTTTAATGATTG-3′ and 5′-AAGCTTTGAGTTTCTTGAACGA CCCA-3′; LA2350, 5′-CATATGGGACGTCTCAAAAGGTATC-3′ and 5′-A AGCTTATGCCGGTTGTGAACATATT-3′; LA0469, 5′-GCTAGCGAGAAA ACAATGAAACAAGA-3′ and 5′-AAGCTTACACAAATAACTTGACTGGA G-3′; LA2095/LA2096, 5′-CATATGAATTCGATGAAGACAAATGT-3′ and 5′-AAGCTTCTAGAATTATATATGGCGTA-3′; LA2152, 5′-CATATGAGA ATGAAGAATGTAGCAGT-3′ and 5′-GAATTCCTACACCTGCTTTTCATAT TT-3′.

The resulting DNA fragments were inserted into pBluescript II KS that had been linearized with EcoRV. NdeI/HindIII, NdeI/EcoRI, or NheI/HindIII. Fragments harboring the desired genes were further cloned into pET28b and expressed in *E. coli* BL21(DE3) as previously described (27). Protein expression was confirmed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) (12.5% polyacrylamide).

Plasmid constructions for complementation studies. The XbaI/HindIII fragments harboring *L. interrogans* gene LA2202, LA2350, LA2096, or LA2152 with ribosomal binding sites (probably from *L. interrogans* for LA2096 but from the *E. coli* vector for the others) were generated from the corresponding expression plasmids (pEX_2202, pEX_2350, pEX_2095&2096, and pEX_2152). These fragments were inserted into psu2718 (25) to generate intermediate-copy-number expression plasmids psu_2202, psu_2350, psu_2095, and psu_2152. Because there are XbaI sites within the LA0469 and LA2095 genes, the following primers were designed on the basis of the sequences of individual genes with additional KpnI/HindIII or SacI/HindIII cleavage sites (underlined): psu_0469, 5′-GGTA CCGTTTAACCTTTAAGAAGGAGA-3′ and 5′-AAGCTTACACAAATAACTT GACTGGAG-3′; psu_2095, 5′-GGTACCGTTTAACCTTTAAGAAGGAGA-3′ and 5′-AAGCTTCCCTTTTTATTTCCAGTTTCG-3′; psu2095&2096, 5′-GAGC TCGTTTAACCTTTAAGAAGGAGA-3′ and 5′-AAGCTTGAATTATATATGGC GTATAACC-3′.

First, PCRs with pEX_0469 or pEX_2095&2096 as the template were used to generate proper fragments for insertion into EcoRV-cleaved pBluescript II KS. From these newly generated plasmids, KpnI/HindIII and/or SacI/HindIII fragments harboring the LA0469, LA2095, and LA2095&2096 genes were generated and introduced into psu2718 to construct plasmids psu_0469, psu_2095, and psu_2095&2096.

Enzyme purification and activity assays. For crude enzyme preparation, extracts of *E. coli* cells bearing *L. interrogans* genes were prepared by sonication of the cell pellets (50 to 100 mg, wet weight) suspended in 500 μl of extraction buffers. The sonication conditions were three bursts at a 50% duty cycle, output 6, with a 30-s pause between bursts (Ultrasonic Homogenizer 4710 series; Cole-Parmer Instrument, Chicago, Ill.). In assays of citramalate synthase and α-isopropylmalate synthase, the extraction buffer was 0.1 M TES [N-tris(hydroxymethyl)methyl-2-aminoethanesulfonic acid], pH 7.5. For assays of α-isopropylmalate isomerase and β-isopropylmalate dehydrogenase, potassium phosphate buffer (pH 7.0, 300 μM) was used.

The CimA protein was purified in accordance with the recommendations of *QIAexpressionist* (The *QIAexpressionist*: a handbook for high-level expression and purification of 6xHis-tagged proteins, 5th edition, QIAGEN GmbH), with modifications. Specifically, crude extracts were prepared by sonication of 290 mg (wet

(LeuC/D) and β-isopropylmalate dehydrogenase (EC 1.1.1.85) (LeuB) of *L. interrogans* are functional in the biosynthesis of both leucine and isoleucine. Thus, *L. interrogans* serotype lai strain 56601 uses a pyruvate pathway for isoleucine biosynthesis (Fig. 1).

**FIG. 1.** Schematic illustration of the leucine-like pyruvate pathway of isoleucine biosynthesis in *L. interrogans* and the corresponding pathway of leucine biosynthesis.

**MATERIALS AND METHODS**

Bacterial strains, plasmids, and culture conditions. The bacterial strains and plasmids used in this work are listed in Table 1. *L. interrogans* (serogroup Icterohaemorrhagiae, serovar lai, type strain 56601) was grown in EMJH or Korthof (37) medium at 28°C. Only mid-log-phase cultures were used for gene expression or enzyme analysis experiments. Cells used in gene expression analysis were grown to a density of 10^8^/ml in 100 ml of EMJH medium supplemented with 0.5, 2.0, or 5.0 mM isoleucine or leucine. After around two generations (about 40 h), the cells were harvested by centrifugation at 8,000 × g for 5 min.

TABLE 1. Bacterial strains and plasmids used in this study

| Bacterial strain or plasmid | Relevant genotype or phenotype | Source and/or reference |
|----------------------------|---------------------------------|--------------------------|
| **Bacterial strains**       |                                 |                          |
| $L$. interrogans 56601      | Serogroup Icterohaemorrhagiae; serovar lai type strain | ICDC$^a$ |
| $E$. coli DH5$\alpha$       | $F^{-} \phi 80 \Delta lacZ \Delta M15 (lacZYA-argF) U169 deoR recA1 endA1 hsdR17 (r_{K}^{-} m_{K}^{-})$<br>$supE44 \lambda^{-} thi-1 gyr96 relA1$ | GIBCO |
| $E$. coli BL21(DE3)         | $F^{'} ompT r_{B}^{-} m_{B}^{-} (\lambda DE3)$ | Novagen |
| $E$. coli AB1255            | $F^{'} fhuA2 lacY1 or lacZ4 tsx-5 glnV44(AS) gal-6 LAM-hisG1(Fs) rpsL8 or rpsL9 or$<br>$rpsL17 malT1 (lamR) xylA mtlA2 ilvA201 metB1 argI11(del) thi-1$ | CGSC$^{b}$; 40 |
| $E$. coli CV512             | $F^{+} leuA317$ | CGSC; 36 |
| $E$. coli CV516             | $F^{+} leuB61$ | CGSC; 36 |
| $E$. coli CV522             | $F^{+} leuC222$ | CGSC; 36 |
| $E$. coli CV524             | $F^{+} leuD141$ | CGSC; 36 |
| **$E$. coli plasmids**      |                                 |                          |
| pBluescript II KS           | pUC18-derived cloning vector | Stratagene |
| pET28(b)                    | Expression vector carrying N-terminal T7 His tag sequence; Kan$^{r}$ | Novagen |
| pLA0469                     | pBluescript II KS harboring 1.424-kb LA0469 fragment | This study |
| pLA2202                     | pBluescript II KS harboring 1.744-kb LA2202 fragment | This study |
| pLA2350                     | pBluescript II KS harboring 1.631-kb LA2202 fragment | This study |
| pLA2095&2096                | pBluescript II KS harboring 2.22-kb LA2095&2096 fragment | This study |
| pLA2152                     | pBluescript II KS harboring 1.56-kb LA2152 fragment | This study |
| pEX_0469                    | pET28b carrying LA0469 gene | This study |
| pEX_2202                    | pET28b carrying LA2202 gene | This study |
| pEX_2350                    | pET28b carrying LA2350 gene | This study |
| pEX_2095&2096               | pET28b carrying leuC/D, LA2095&2096 genes | This study |
| pEX_2152                    | pET28b carrying leuB, LA2152 gene | This study |
| pCleuA                      | pET28b carrying $E$. coli leuA gene | This study |
| psu2718                     | HaeII fragment of pUC18 containing lacZ$\alpha$ and MCS, $^{c}$P15A replicon with chloramphenicol acetyl transferase gene (cat) | 25 |
| pco0469                     | pBluescript II KS harboring LA0469 | This study |
| psu_0469                    | psu2718 carrying LA0469 | This study |
| psu_2202                    | psu2718 carrying LA2202 | This study |
| psu_2350                    | psu2718 carrying LA2350 | This study |
| pco2095                     | pBluescript II KS harboring LA2095 | This study |
| pco2095&2096                | pBluescript II KS harboring LA2095&2096 | This study |
| psu_2095                    | psu2718 carrying LA2095 | This study |
| psu_2096                    | psu2718 carrying LA2096 | This study |
| psu_2095&2096               | psu2718 carrying LA2095&2096 | This study |
| psu2152                     | psu2718 carrying LA2152 | This study |

$a$ ICDC, National Institute for Communicable Disease Control and Prevention, Chinese Center for Disease Control and Prevention.

$b$ CGSC, $E$. coli Genetic Stock Center, Yale University.

$c$ MCS, multiple cloning site.

Almost identical conditions were used for the GC-MS experiments. All experiments were carried out on a Hewlett Packard 5989 engine B equipped with an HP 5890 gas chromatograph. The injector temperature was set to 200°C. The split was set at 1:15. The temperature gradient was 60 to 220°C at 3°C/min. The carrier gas was nitrogen. The interface was set to 250°C. The source temperature was 220°C. The mass spectrometer was operated in electron impact mode at 70 eV. The mass range was 33 to 700 atomic mass units in 1.5 s.

RNA extraction and RT-PCR analysis. The Trizol method (Gibco BRL) was used for cell lysis and total RNA extraction. The extracted RNA, dissolved in 30 μl of RNase-free water, was treated with RQ1 RNase-free DNase (Promega Corp.). Treated RNA (2 μl) was processed with the Reverse Transcription System (20 μl; Promega Corp.). After incubation for 10 min at room temperature, reactions were carried out at 42°C for 25 min, followed by enzyme inactivation at 95°C for 5 min. Real-time PCR was performed in accordance with the manufacturer's recommendations on the ABI Prism 7000 Sequence Detection System (Applied Biosystems) with the SYBR Green PCR Core Reagents (Applied Biosystems) in a 50-μl reaction mixture. The amount of template cDNA used was 10 to 15 ng per assay. For negative controls, the template cDNA was replaced with an RNA preparation without RT. The reaction was initiated by 2 min of incubation at 50°C, followed by AmpliTaq Gold activation for 10 min at 95°C. Amplification was achieved by 40 cycles of 15 s at 95°C and 1 min at 55°C. The cycle threshold parameters were normalized to that of 16S rRNA, which served as an internal standard. The following primers were used in real-time RT-PCR experiments: RT_2350, 5′-AGTCTTCAAGCGATTCGTGC-3′ and 5′-GTGGTTACGA GTGCTTCCAA-3′; RT_0469, 5′-CTTCGATTGCACCGGATAAC-3′ and 5′-AA CCGTATCGCCTTCCAAGA-3′; RT_2202, 5′-GGTAATTGCAGCAGGAGCT A-3′ and 5′-CTCCTCTTACGTTGCTGAGT-3′; RT_16S rRNA, 5′-CCTCAGTA ACGAACCTAACG-3′ and 5′-TCACTCTTGCGAGCATAGTC-3′.

Nucleotide sequence accession numbers. The *L. interrogans* genomic sequences reported here have been submitted to the GenBank database and assigned accession no. AE010300 for CI, the large chromosome of 4,332,241 bp, and AE010301 for CII, the small chromosome of 358,943 bp.

Fluka, Buchs, Switzerland). After sonication, the contents were transferred to a 2-ml sample vial and 500 μl of diazomethane was added (prepared in house, 98% pure). The reaction mixture was held at room temperature for 1 h and occasionally stirred on a Vortex mixer. After centrifugation, 1 μl of the supernatant was injected onto a BGB-176 column for either GC-MS for identification of the citramalate diester or GC FID for quantification of the two isomers.

Standards for GC-MS were (*R*)-, (*S*)-, and (*RS*)-citramalate purchased from Aldrich and Sigma (catalog no. 32,914-2, 33,199-6, and C0634, respectively). Standards were processed by the procedure described above.

All GC FID analyses were performed on an Agilent 6890 gas chromatograph (Agilent Schweiz AG, Mettmenstetten, Switzerland) with a BGB-176 chiral column (30 m by 0.25 mm [inside diameter], 0.25-μm-thick film; BGB Analytics, Boeckten, Switzerland). The injector temperature was set to 200°C. The injection mode was splitless. The temperature gradient was 60 to 200°C at 3°C/min. The detector was held at 220°C. Hydrogen was used as the carrier gas at a pressure of 60 kPa.

FIG. 2. SDS-PAGE analysis of purified CimA (LA2350) protein. CimA was purified as described in Materials and Methods. The amount of purified protein loaded was 4 μg. The molecular size markers were purchased from Promega Corp.

RESULTS

The genome of *L. interrogans* has three CDSs highly similar to *leuA* but none for *ilvA*. In direct enzyme assays, we failed to detect any threonine deaminase activity in *L. interrogans* strain 56601 (Table 2). However, one of the three *leuA*-like CDSs (LA2350) shows substantial homology to the *cimA* genes of archaea, e.g., those of *Methanopyrus kandleri* (32% identity, 51% similarity), *M. thermoautotrophicum* (33% identity, 50% similarity), and *M. jannaschii* (32% identity, 52% similarity) (see supplemental material). All of these genes have a pyruvate carboxylase domain (PycA), and the one from *M. jannaschii*

TABLE 2. Enzymatic activities of *L. interrogans* genes expressed in *E. coli* relevant to the leucine biosynthesis-like pathways

| $L$. interrogans genes expressed in $E$. coli$^{a}$ | Citramalate synthase | $\alpha$-Isopropylmalate synthase | Citraconate hydration | $\alpha$-Ketobutyrate formation | Threonine deaminase | $\alpha$-Isopropylmalate isomerase | $\alpha$-Isopropylmalate dehydrogenase |
| --- | --- | --- | --- | --- | --- | --- | --- |
| LA0469 ($leuA2^{+}$) | 29.4 ± 2.0 | 136.6 ± 1.6 | NT$^{c}$ | NT | NT | NT | NT |
| LA2202 ($leuA1^{+}$) | <0.01 | 2.34 ± 0.19 | NT | NT | NT | NT | NT |
| LA2350 ($cimA^{+}$) | 174.3 ± 6.5 | <0.01 | NT | NT | NT | NT | NT |
| LA2095&2096 ($leuC^{+}D^{+}$) | NT | NT | 97.5 ± 0.5 | NT | NT | 42.7 ± 1.2 | NT |
| LA2152 ($leuB^{+}$) | NT | NT | NT | 366 ± 14 | NT | NT | 132.8 ± 2.1 |
| Control strains$^{d}$ |  |  |  |  |  |  |  |
| $E$. coli CV 512 ($leuA$) | 28.52 ± 0.68 | 117.6 ± 2.0 | NT | NT | NT | NT | NT |
| $E$. coli K-12 | 7.89 ± 0.32 | 32.88 ± 2.01 | NT | NT | 44.0 ± 4.2 | NT | NT |
| $L$. interrogans (E) | 14.53 ± 0.35 | 10.95 ± 0.27 | NT | NT | <0.01 | NT | NT |
| $L$. interrogans (K) | 6.94 ± 0.48 | 7.36 ± 0.34 | NT | NT | <0.01 | NT | NT |

$a$ Plasmid construction is described in Materials and Methods.

$b$ Methods for bacterial culture and enzyme assay are described in Materials and Methods.

$c$ NT, not tested.

$d$ $E$. coli strains were grown in minimal medium, while $L$. interrogans strains were grown in either EMJH medium (E) or Korthof medium (K).

TABLE 3. Purification of citramalate synthase from *E. coli* BL21(DE3) carrying pEX_2350

| Purification step | Vol (ml) | Total protein (mg) | Total activity (nmol/min) | Sp act (nmol/min/mg of protein) | Yield (%) | Fold purification |
|------------------|----------|--------------------|---------------------------|---------------------------------|-----------|-------------------|
| Crude extract    | 4.00     | 11.3               | 1,996                     | 176.6                          | 100       | 1.0               |
| Six-His tagged   | 0.50     | 0.2                | 505.5                     | 2,527.5                        | 25.3      | 14.3              |

(MJ1392) was experimentally proved a citramalate synthase (21; see supplemental material). On the other hand, both CDSs LA2202 and LA0469 are more similar to LeuA than to CimA, with the former bearing even higher homology than the latter.

LA2350 encodes a citramalate synthase without detectable α-isopropylmalate synthase activity. The three aforementioned *L. interrogans* genes were cloned and expressed in *E. coli*. Activity assays confirmed the differences in catalytic specificity predicted by bioinformatics (Table 2). Because the protein encoded by LA2350 has high citramalate synthase activity but no detectable α-isopropylmalate synthase activity, this gene was designated *cimA*. Extracts of *E. coli* expressing the LA2202 protein had very low α-isopropylmalate synthase activity but no citramalate synthase activity (Table 2). It was designated *leuA1*. LA0469 was designated *leuA2* on the basis of its high α-isopropylmalate synthase activity and relatively low citramalate synthase activity (Table 2).

The *L. interrogans* LA2350 protein, expressed in *E. coli*, was purified to 92% homogeneity, as determined by SDS-PAGE (Fig. 2; Table 3). The specific activity of the purified citramalate synthase was 2.53 μmol/min/mg of protein under standard assay conditions (see Materials and Methods); thus, the $k_{cat}$ of the enzyme was 2.41 s⁻¹ (Table 4). The $K_m$ values for proteins expressed in *E. coli* were as follows: CimA (LA2350), 0.043 mM for pyruvate; LeuA2 (LA0469), 0.709 mM for pyruvate and 0.108 mM for α-ketoisovalerate (Table 4).

The optimal reaction temperature and the thermostability of the *L. interrogans* citramalate synthase were measured (Fig. 3). Unlike its counterpart from *M. jannaschii*, the *L. interrogans* CimA protein lost most of its enzymatic activity at 50°C. Its optimal reaction temperature was very close to the transition point of the thermostability curve. Thermostability is thus likely to be the main factor determining the optimal reaction temperature. Purified citramalate synthase was less thermostable than the crude enzyme (Fig. 3).

The substrate specificity of the partially purified *L. interrogans* citramalate synthase (CimA) was tested (Table 5). The activity toward pyruvate was high and very specific, with no

---

**FIG. 3. Optimal reaction temperature (■) and thermostability (▲) of *L. interrogans* citramalate synthase. The experiments were carried out as described in Materials and Methods, with cell extract containing either the highly expressed (A) or the purified (B) enzyme.**

---

TABLE 4. Kinetic parameters of the *L. interrogans* LeuA-like proteins expressed in *E. coli*

| Enzyme or expression plasmid                  | $K_m (k_{cat})$ for $^b$ |
|-----------------------------------------------|-------------------------|
|                                               | Pyruvate                | α-Ketoisovalerate           |
| pEX_2350 $^a$                                 | 0.04 (2.41)             | Not detected               |
| pEX_0469 $^d$                                 | 0.71                    | 0.12                       |
| α-Isopropylmalate synthase from $^c$          | 10.0                    | 0.06 $^c$                  |
| *S. enterica* serovar Typhimurium $^c$        |                         |                             |

$a$ CimA was purified from *E. coli* carrying the plasmid. Only citramalate activity was observed (see Materials and Methods).

$b$ The $K_m$ (in millimolar units) of *L. interrogans* citramalate synthase was determined as described in Materials and Methods with crude extracts of *E. coli* expressing *L. interrogans* CimA (pEX_2350). The $k_{cat}$ (per second) of *L. interrogans* citramalate synthase was derived from the specific activity of the purified enzyme (Table 3) with a calculated molecular mass of 57,316.34 Da.

$c$ Apparent $k_{m}$

$d$ The LeuA2 protein was expressed in *E. coli*. Crude extracts were assayed for both citramalate synthase and α-isopropylmalate synthase (see Materials and Methods).

$e$ The $K_m$ for citramalate activity was determined in crude extracts of *S. enterica* serovar Typhimurium. The apparent $K_m$ for α-isopropylmalate synthase was previously reported (23).

VOL. 186, 2004

ISOLEUCINE BIOSYNTHESIS IN *L. INTERROGANS* 5405

TABLE 5. Substrate specificity of purified *L. interrogans* citramalate synthase$^a$

| Substrate           | Product formed$^b$ | Activity compared with that of pyruvate (%) |
|---------------------|---------------------|---------------------------------------------|
| Pyruvate            | 0.455              | 100                                         |
| α-Ketoisovalerate   | 0                   | 0                                           |
| α-Ketobutyrate      | 0.003              | 0.66                                        |
| α-Ketoisocaproate   | 0                   | 0                                           |
| α-Ketoglutarate     | 0                   | 0                                           |

$^a$ The concentration of each substrate was 1 mM in the final assay mixture. All of the assay conditions are described in Materials and Methods.

$^b$ Values are optical densities at 412 nm.

detectable activities toward any of the other keto acid substrates.

The citramalate generated by *L. interrogans* CimA is of $R$ chirality. The identity of citramalate was proven by GC-MS of its dimethyl ester (20). The product was found to be $(R)$-citramalate (Fig. 4). This result not only is consistent with the previously reported data for citramalate synthase from archaea (21) but also indicates that of the two proposed pyruvate pathways for isoleucine biosynthesis (9), the leucine-like pathway is more likely to be used by *L. interrogans* than the other ($\beta$-methylaspartate) pathway. This experiment also showed that imidazole inhibited citramalate synthase and altered the $R/S$ ratio of the products (Fig. 4).

The LeuC/LeuD and LeuB enzymes function in isoleucine biosynthesis in *L. interrogans*. To complete the leucine-like pathway for isoleucine biosynthesis in *L. interrogans*, both the heterodimeric enzyme, LeuC/LeuD, catalyzing the interconversion of $\alpha$-isopropylmalate and $\beta$-isopropylmalate via dimethylcitraconate, and $\beta$-isopropylmalate dehydrogenase, LeuB, catalyzing the NAD${}^{+}$-dependent conversion of $\beta$-isopropylmalate to $\alpha$-ketoisocaproate, could catalyze the conversion of

![Figure 4](#fig4)

FIG. 4. Chirality of the citramalate produced by LA2350 (CimA) reaction. (a) Peaks of $(R)$-citramalate and $(S)$-citramalate as products of the condensation reaction catalyzed by the crude enzyme. Standards were added to the samples (top), highlighting the elution positions of the corresponding $R$ and $S$ isomers. Most of the products were $(R)$-citramalate. (b) Purified enzyme with TES buffer (imidazole was removed from the sample buffer) was used for the reaction. $(R)$-citramalate is the predominant product. The peak area of $(R)$-citramalate was 183,099.1 µV · s (98.57%), while the peak area of $(S)$-citramalate was 2,647.0 µV · s (1.43%). (c) Purified enzyme with imidazole was used for the reaction. The production of $(R)$-citramalate was decreased. The peak area of $(R)$-citramalate was 40,327.0 µV · s (83%), while the peak area of $(S)$-citramalate was 7,933.6 µV · s (16.44%). These two reactions were carried out under the same conditions, including the same concentrations of enzymes and substrates, except for the presence or absence of imidazole in the reaction buffer.

TABLE 6. Complementation of *E. coli* isoleucine and leucine auxotrophs with cloned *L. interrogans* genes

| Clone(s) of *L. interrogans* gene(s) | Complementation of *E. coli* mutation: |
|-------------------------------------|----------------------------------------|
|                                   | **ilvA** | **leuA** | **leuB** | **leuC** | **leuD** |
| psu_0469 (leuA2)                   | –        | +        | –        | –        | –        |
| psu_2202 (leuA1)                   | –        | –        | –        | –        | –        |
| psu_2350 (cimA)                    | +        | –        | –        | –        | –        |
| psu_2095 (leuC)                    | –        | –        | –        | –        | –        |
| psu_2096 (leuD)                    | –        | –        | –        | –        | –        |
| psu_2095&2096 (leuCD)              | –        | –        | –        | +        | +        |
| psu_2152 (leuB)                    | –        | –        | +        | –        | –        |
| psu2718 (vector)                   | –        | –        | –        | –        | –        |

(R)-citramalate to α-ketobutyrate, in parallel with their functions in leucine biosynthesis. The *L. interrogans* leuC (LA2095) and leuD (LA2096) genes were cloned and expressed in *E. coli*. The activity of the product of these gene was assayed (Table 2). Because mesaconate, a stereoisomer of citraconate, failed to serve as a substrate, the β-methylaspartate pathway is unlikely to be used for isoleucine biosynthesis in *L. interrogans*.

Mixed extracts of *E. coli* expressing the *L. interrogans* LeuC/LeuD (pEX_2095&2096) and LeuB (pEX_2152) enzymes catalyzed the formation of α-ketobutyrate from citraconate or α-ketoisocaproate from dimethylcitraconate (Table 2). Thus, there exists a set of genes of *L. interrogans* that are functional in isoleucine biosynthesis via a leucine-like pathway.

Both citramalate synthase and α-isopropylmalate synthase activities were detected in extracts of *E. coli* K-12. However, the ratio of these two activities was the same as that of either the LA0469 clone or an *E. coli* leuA clone expressed in *E. coli* BL21(DE3) (Table 2).

Complementation analysis of the leucine-like pathway for isoleucine biosynthesis. Because there is no system of genetic manipulation available for *L. interrogans*, mutants of *E. coli* with lesions in ilvA and other genes (e.g., leuA, leuB, and leuC/leuD) were used for complementation analysis. The results (Table 6 and 7) indicated that expression of the *L. interrogans* cimA gene (LA2350) was able to relieve the isoleucine requirement of an *E. coli* ilvA mutant (AB1255), but not the leucine requirement of the *E. coli* leuA mutant (CV512). This result not only confirms the reaction specificity of the *L. interrogans* CimA protein but also indicates that the *E. coli* LeuC/LeuD and LeuB proteins are functional in the subsequent reactions of the pathway leading to α-ketobutyrate formation. This is the first direct in vivo evidence for the operation of a threonine-independent pathway of isoleucine biosynthesis under physiological conditions.

LA0469 is a bona fide leuA gene since it only complements the leucine requirement of leuA tester strain CV512. It failed to relieve the isoleucine requirement of an *E. coli* ilvA mutant, although it has low citramalate synthase activity. The leucine complementation effect of LA2202 is marginal, consistent with its low catalytic activity in vitro (Table 2).

Together, LA2095 and LA2096 can complement a leuC mutant (CV522) or a leuD mutant (CV524). However, neither of the genes alone is functional in this regard. Finally, LA2152 was able to complement a leuB *E. coli* mutant (CV516).

The expression of cimA and leuA in *L. interrogans* is transcriptionally regulated by isoleucine and leucine, respectively. Both citramalate synthase and isopropylmalate synthase were readily detected in *L. interrogans* grown on synthetic minimal (EMJH) medium. However, these activities were significantly reduced in cells grown on serum-containing Korthof medium (Table 2). When 5 mM isoleucine or leucine was present in minimal medium, neither CimA nor LeuA activity was detected. These results were supported by Western blot analysis with CimA-specific antibodies (data not shown).

The results of a real-time PCR assay with total RNAs extracted from *L. interrogans* cells grown in EMJH medium supplemented with isoleucine or leucine, as well as cells grown on Korthof medium, indicated that the expression of cimA (LA2350) was repressed by isoleucine but not by leucine (Fig. 5). On the other hand, the expression of the leuA genes (LA0469 and LA2202) was repressed not only significantly by leucine but also moderately by isoleucine. Thus, leucine and isoleucine are capable of crossing the *L. interrogans* cell membrane and exerting their regulatory effects on the transcription of the corresponding biosynthetic genes.

DISCUSSION

Charon et al. (9) first proposed the pyruvate pathway for isoleucine biosynthesis in *L. interrogans* on the basis of their studies with ^14^C-labeled carbon dioxide and other tracers. They suggested that isoleucine might be synthesized from α-ketobutyrate via citramalate, formed in turn from pyruvate and acetyl-CoA, which was very similar to that of leucine biosynthesis. This report not only fully supports this hypothesis but is also the first description of the citramalate pathway leading to de novo isoleucine synthesis under normal conditions of bacterial physiology.

TABLE 7. Complementation of an *E. coli* ilvA mutant AB1255 by leuA-like genes from *L. interrogans* in different media

| Medium for complementation$^a$ | Complementation of *E. coli* mutant strain and cloned *L. interrogans* gene(s): |
|-------------------------------|---------------------------------------------------------------|
|                               | AB1255/psu_2350 (ilvA/cimA$^+$) | AB1255/psu_2202 (ilvA/leuA1$^+$) | AB1255/psu_0469 (ilvA/leuA2$^+$) | AB1255 (ilvA) |
| M9$^+$                        | +                                                             | –                                                             | –                                                             | –                                                             |
| M9$^+$, isoleucine            | +                                                             | +                                                             | +                                                             | +                                                             |
| M9$^+$, α-ketobutyrate        | +                                                             | +                                                             | +                                                             | +                                                             |
| M9$^+$, α-ketobutyrate, valine | –                                                             | –                                                             | –                                                             | –                                                             |
| M9$^+$, valine                | –                                                             | –                                                             | –                                                             | –                                                             |
| M9$^+$, valine, isoleucine    | +                                                             | +                                                             | +                                                             | +                                                             |

$^a$ M9$^+$, M9 medium supplemented with His, Met, Arg, and thiamine as the minimal medium described in Materials and Methods.

VOL. 186, 2004                                                                 ISOLEUCINE BIOSYNTHESIS IN *L.* INTERROGANS 5407

![Graph](https://i.imgur.com/yourimageurl.png)

FIG. 5. Expression levels of *cimA* (LA2350, diamond), *leuA1* (LA2202, triangle), and *leuA2* (LA0469, square) in *L.* interrogans revealed by RT-PCR. *L.* interrogans was grown in different media as described in Materials and Methods. Solid lines connecting filled data points represent the conditions under which isoleucine was added to EMJH medium. Dashed lines connecting filled data points represent the leucine-supplemented EMJH medium. The single open unconnected data points represent Korthof medium. All data points represent average measurements from three independently conducted assays with error bars indicated. ct, cycle threshold.

Early suggestions regarding alternative pathways for the formation of α-ketobutyrate were mainly based on reversion studies of an *E. coli ilvA* mutant (30). Later reversion experiments with enteric bacteria (22) and yeasts (34) offered further support for the involvement of the enzymes of leucine biosynthesis in an alternative isoleucine pathway, although the physiological significance of these observations was unknown. In the 1980s, radioactive labeling studies revealed that isoleucine was probably synthesized from pyruvate and acetyl-CoA via citramalate in *M.* thermoautotrophicum, where the conventional pathway involving threonine was missing (14). Howell et al. (21) cloned the *cimA* gene of *M.* jannaschii and expressed citramalate synthase in *E. coli*. The characterization of this enzyme further supported a role for CimA in a novel pyruvate pathway for isoleucine biosynthesis. However, direct evidence for a complete alternative pathway under normal conditions of bacterial physiology was not provided.

The recently available genomic sequence of *L.* interrogans serotype lai confirmed that this strain lacks an *ilvA* gene but contains three CDSs that are homologous to *leuA* (32). In this report, we have characterized these CDSs and confirmed that LA2350 encodes citramalate synthase while the other two (LA0469 and LA2202) are *leuA* genes.

Unlike the LeuA enzymes, CimA (LA2350) exhibits strict substrate specificity catalyzing the condensation of acetyl-CoA and pyruvate to form an isomer of citramalate. However, it failed to utilize α-ketoisovalerate, α-ketobutyrate, α-ketoisocaproate, or α-ketoglutarate (Table 5). This may reflect the structure of the enzyme and the molecular size of the substrate. Pyruvate is the smallest of the substrates tested. Therefore, if

CimA has a tight binding pocket for pyruvate (supported by the low *Km* value of 43 μM), other substrates larger than pyruvate are likely to be excluded. On the other hand, because the LeuA enzymes are able to bind α-ketoisovalerate (*Km* = 108 μM for LA0469), pyruvate may also be accommodated, leading to detectable citramalate synthase activity (*Km* = 709 μM for LA0469). To further clarify this issue, we studied the *leuA* gene product from *E. coli* K-12. This protein also utilizes both substrates with a ratio basically the same as that of LA0469. Previously, it was reported that the α-isopropylmalate synthase of *Salmonella enterica* serovar Typhimurium had an affinity for pyruvate, but with a *Km* value about twofold higher than that of its physiological substrate (23). Citramalate was reported to be formed from pyruvate by a *Serratia marcescens* mutant having a form of α-isopropylmalate synthase that was insensitive to feedback inhibition (22). A similar phenomenon was observed in a respiration-deficient yeast mutant (34). However, under normal physiological conditions, the use of pyruvate for isoleucine biosynthesis does not occur because of the very low affinity of α-isopropylmalate synthase toward pyruvate. Moreover, all of these microorganisms are different from *L.* interrogans by virtue of having a complete pathway from threonine to isoleucine. It is likely, therefore, that LA0469 is primarily responsible for leucine biosynthesis in *L.* interrogans. Since the expression of LA2202 (*leuA1*) in *E. coli* was very low, it was not further characterized.

The citramalate synthase CDS MJ1392 of *M.* jannaschii was the first to be characterized and, as expected, was thermostable. However, the *L.* interrogans CimA enzyme lost most of its activity at 50°C or above. At 37°C, the specific activity of the

purified *L. interrogans* citramalate synthase was low (2.53 μmol/min/mg of protein) but was similar to that of the thermotable *M. jannaschii* enzyme measured at 50°C (2.87 μmol/ min/mg), as well as other, similar, enzymes. In addition, because the \( K_m \) value for pyruvate of the *L. interrogans* enzyme (43 μM) is almost 20 times less than that of the *M. jannaschii* enzyme (0.85 mM), LA2350 should be appropriate for the metabolic needs of this slow-growing bacterium.

Charon et al. (9) proposed that the condensation of acetyl-CoA with pyruvate might yield either (*S*)- or (*R*)-citramalate, and two possible pathways were postulated. (*S*)-citramalate might be fed into the β-methylaspartate pathway of α-ketobutyrate biosynthesis via mesaconate. Alternatively, (*R*)-citramalate might eventually yield α-ketobutyrate via citraconate—a leucine-like pathway. Our experiments indicated that the majority of the citramalate was of *R* chirality (Fig. 4), thus supporting the leucine-like pathway. It is interesting that high concentrations of imidazole may significantly alter both the chirality of the product and the enzyme activity.

We were able to demonstrate that two leucine biosynthetic enzymes encoded by *L. interrogans* genes, isopropylmalate isomerase (LA2095/LA2096, i.e., *leuC/leuD*) and β-isopropylmalate dehydrogenase (LA2152, i.e., *leuB*), catalyzed the conversion of (*R*)-citramalate to α-ketobutyrate. Mesoacetate, an intermediate of the proposed β-methylaspartate pathway, was not a substrate. Thus, the in vitro enzymatic data support the operation in *L. interrogans* of an alternative pathway of isoleucine biosynthesis, the leucine-like pyruvate pathway.

These conclusions were further supported by in vivo complementation experiments conducted with *E. coli*. The rationale for these experiments was as follows. First, except for CimA, the enzymes of leucine biosynthesis and isoleucine biosynthesis via the pyruvate pathway are shared. Second, the enzymes of leucine biosynthesis in *L. interrogans* have the same function as their *E. coli* counterparts, as confirmed by previous work (12). We first confirmed that the *cimA* gene of *L. interrogans* was able to reverse the isoleucine requirement of an *E. coli ilvA* mutant, AB1255. Because AB1255 requires either α-ketobutyrate or isoleucine for growth, the complementation is certainly isoleucine specific. The growth of *cimA* transformants on minimal medium was inhibited by valine but not by isoleucine (Table 7). Similarly, *ilvA* mutants were able to grow on α-ketobutyrate-supplemented minimal medium but not on medium containing both α-ketobutyrate and valine. The first step of valine biosynthesis in *E. coli* is the condensation of pyruvate with active acetaldehyde derived from pyruvate to yield α-acetolactate. This reaction is strongly inhibited by valine and because the same enzyme catalyzes the conversion of α-ketobutyrate to α-aceto-hydroxybutyrate in isoleucine biosynthesis, valine blocks the biosynthesis of isoleucine (4, 28). These observations confirmed that the metabolic intermediate supplied during *cimA* complementation is α-ketobutyrate. Although the *L. interrogans leuA2* gene (LA0469) has traces of citramalate synthase activity in vitro, this gene only reversed the leucine requirement of an *E. coli leuA* mutant but not the isoleucine auxotrophy of an *ilvA* mutant.

Complementation studies of the three genes encoding the two leucine biosynthetic enzymes responsible for the succeeding steps, isopropylmalate isomerase (*leuC/leuD*, LA2095/ LA2096) and isopropylmalate dehydrogenase (*leuB*, LA2152),

clearly indicate that these genes function in vivo in isoleucine biosynthesis via the threonine-independent pathway. LeuC and LeuD are the large and small subunits of isopropylmalate isomerase, respectively. In most microorganisms (including *L. interrogans*), these two genes are cotranscribed (19). Thus, the initial complementation experiment was performed with the cistronic *leuCD* construct. However, when these two genes were tested individually, no complementation was observed. This suggests that the LeuC subunit of *L. interrogans* cannot form a functional enzyme with the *E. coli* LeuD subunit and vice versa.

When the leucine-like pyruvate pathway was originally proposed, Charon et al. (9) and Westfall et al. (41) indicated that feedback inhibition of the pyruvate pathway was occurring in *L. interrogans*. They were puzzled to notice that the pyruvate pathway was not controlled by leucine (41). In other words, isoleucine synthesis and leucine synthesis appear to be independently regulated despite the fact that the same enzyme system is used. This study experimentally proved, at three different levels (Table 2 and Fig. 5), that the expression of either the *cimA* or the *leuA* gene was regulated by the corresponding amino acid end products. In particular, the regulation of *cimA* expression is very strict; i.e., it only responds to isoleucine but not to leucine. On the other hand, the regulation of *leuA* expression was relatively less stringent, i.e., isoleucine moderately regulated *leuA* expression. This regulatory mechanism, together with other amino acid biosynthetic pathways in *L. interrogans*, remains to be clarified. In this connection, as shown in previous publications (2, 31), this study points to the presence of specific amino acid transporters in *L. interrogans*, despite the fact that the annotation of the genome failed to identify any of these systems.

Finally, except for *S. marcescens* (22), *cimA* genes were mainly thought to be present in the genomes of archaea. Among bacteria, spirochetes are evolutionarily primitive. Significant portions of the genomes of *L. interrogans* are more similar to those of either archaea or eukaryotes than to those of bacteria (32). It has been known that the biosynthesis of methionine in *L. interrogans* corresponds to the pathways of both bacteria and yeast (31). The leucine-like pyruvate pathway for isoleucine biosynthesis might be rare in bacteria but common in archaea (43).

---

**ACKNOWLEDGMENTS**

This work was supported by the National Natural Science Foundation of China, the State Key Laboratory of Microbial Technology of Shandong University, and the Chinese National High Technology Development Program (863).

We thank R. L. Somerville for helpful comments on our research strategy and for manuscript preparation. We thank Junsong Han and Yaozhou Shi of Shanghai Biochip Center for help in carrying out the real-time PCR experiments. We thank Robert H. White for suggestions about the preparation of methyl esters from the reaction mixture and Sophie Brogly and Christine Tournoux for determination of citramalate chirality. We appreciate the assistance of Hai-Gang Zhu in preparing the drawings, as well as Bao-Yu Hu, Pin He, Yang Yang, and Xiang-Yan Zhang in bacterial culture preparation. We are grateful to Mary Berlyn and the *E. coli* Genetic Stock Center for the generous gift of the *E. coli ilvA, leuA, leuB, leuC,* and *leuD* mutants. We particularly thank the members of the Laboratory of Microbial Molecular Physiology, IPPE, SIBS, CAS, for strong support and encouragement.

VOL. 186, 2004

ISOLEUCINE BIOSYNTHESIS IN *L.* INTERROGANS 5409

REFERENCES

1. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. A. Smith, and K. Struhl. 1991. Molecular cloning: a laboratory manual. Greene and Wiley Inter-science, New York, N.Y.

2. Bauby, H., I. Saint Girons, and M. Picardeau. 2003. Construction and complementation of the first auxotrophic mutant in the spirochaete *Leptospira meyeri*. Microbiology 149(Pt. 3):689–693.

3. Belly, R. T., S. Greenfield, and G. W. Claus. 1970. β-Methylaspartate metabolism and isoleucine biosynthesis in *Acetobacter suboxydans*. Bacteriol. Proc., p. 141.

4. Blatt, J. M., W. J. Pledger, and H. E. Umbarger. 1972. Isoleucine and valine metabolism in *Escherichia coli*. XX. Multiple forms of acetohydroxy acid synthetase. Biochem. Biophys. Res. Commun. 48:444–450.

5. Bradford, M. M. 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. 72:248–254.

6. Buchanan, B. B. 1969. Role of ferredoxin in the synthesis of α-ketobutyrate from propionyl coenzyme A and carbon dioxide by enzymes from photosynthetic and non-photosynthetic bacteria. J. Biol. Chem. 244:4218–4223.

7. Burns, R. O., H. E. Umbarger, and S. R. Gross. 1963. The biosynthesis of leucine. III. The conversion of α-hydroxy-β-carboxyisocaproate to α-ketoisocaproate. Biochemistry 2:1053–1058.

8. Charon, N. W., and S. F. Goldstein. 2002. Genetics of motility and chemotaxis of a fascinating group of bacteria: the spirochetes. Annu. Rev. Genet. 36:47–73.

9. Charon, N. W., R. C. Johnson, and D Peterson. 1974. Amino acid biosynthesis in the spirochete *Leptospira*: evidence for a novel pathway of isoleucine biosynthesis. J. Bacteriol. 117:203–211.

10. Cho-Chung, Y. S., and H. E. Umbarger. 1971. Isopropylmalate isomerase (yeast). Methods Enzymol. 17A:782–785.

11. Cohen, P. P., and H. J. Sallach. 1961. Nitrogen metabolism of amino acids, p. 1–66. In D. M. Greenberg (ed.), Metabolic pathways, vol. II. Academic Press, Inc., New York, N.Y.

12. Ding, M., and D. B. Yelton. 1993. Cloning and analysis of the *leuB* gene of *Leptospira interrogans* serovar *pomona*. J. Gen. Microbiol. 139(Pt. 5):1093–1103.

13. Eikmanns, B., R. Jaenchen, and R. K. Thauer. 1983. Propionate assimilation by methanogenic bacteria. Arch. Microbiol. 136:106–110.

14. Eikmanns, B., D. Linder, and R. K. Thauer. 1983. Unusual pathway of isoleucine biosynthesis in *Methanobacterium thermoautotrophicum*. Arch. Microbiol. 136:111–113.

15. Ekiel, I., Smith, I. C. P., and G. D. Sprott. 1984. Biosynthesis of isoleucine in methanogenic bacteria: a ¹³C NMR study. Biochemistry 23:1683–1687.

16. Faine, S., B. Adler, C. Bolin, and P. Perolat. 1999. *Leptospira* and leptospirosis. Medisci, Melbourne, Australia.

17. Fraser, C. M., S. Casjens, W. M. Huang, G. G. Sutton, R. Clayton, R. Lathigra, O. White, K. A. Ketchum, R. Dodson, E. K. Hickey, M. Gwinn, B. Dougherty, J. F. Tomb, R. D. Fleischmann, D. Richardson, J. Peterson, A. R. Kerlavage, J. Quackenbush, S. Salzberg, M. Hanson, R. van Vugt, N. Palmer, M. D. Adams, J. Gocayne, J. Weidman, T. Utterback, L. Watthey, L. Mcdonal, P. Artiach, C. Bowman, S. Garland, C. Fujii, M. D. Cotton, K. Horst, K. Roberts, B. Hatch, H. O. Smith, and J. C. Venter. 1997. Genomic sequence of a Lyme disease spirochaete, *Borrelia burgdorferi*. Nature 390:580–586.

18. Fraser, C. M., S. J. Norris, G. M. Weinstock, O. White, G. G. Sutton, R. Dodson, M. Gwinn, E. K. Hickey, R. Clayton, K. A., Ketchum, E. Sodergren, J. M. Hardham, M. P. McLeod, S. Salzberg, J. Peterson, H. Khalak, D. Richardson, J. K. Howell, M. Chidambaram, T. Utterback, L. McDonald, P. Artiach, C. Bowman, M. D. Cotton, C. Fujii, S. Garland, B. Hatch, K. Horst, K. Roberts, M. Sandusky, J. Weidman, H. O. Smith, and J. C. Venter. 1998. Complete genome sequence of *Treponema pallidum*, the syphilis spirochete. Science 281:375–388.

19. Fultz, P. N., and J. Kemper. 1981. Wild-type isopropylmalate isomerase in *Salmonella typhimurium* is composed of two different subunits. J. Bacteriol. 148:210–219.

20. Howell, D. M., K. Harich, H. Xu, and R. H. White. 1998. The α-keto acid chain elongation reactions involved in the biosynthesis of coenzyme B (7-mercaptopheptanoylthreonine phosphate) in methanogenic Archaea. Biochemistry 37:10108–10117.

21. Howell, D. M., H. Xu, and R. H. White. 1999. (*R*)-Citramalate synthase in methanogenic archaea. J. Bacteriol. 181:331–333.

22. Kisumi, M., S. Komatsubara, and I. Chibata. 1977. Pathway for isoleucine formation from pyruvate by leucine biosynthetic enzymes in leucine-accumulating isoleucine revertants of *Serratia marcescens*. J. Biochem. 82:95–103.

23. Kohlhaw, G. B., T. R. Leary, and H. E. Umbarger. 1969. α-Isopropylmalate synthase from *Salmonella typhimurium*: purification and properties J. Biol. Chem. 244:2218–2225.

24. Lawther, R. P., and G. W. Hatfield. 1978. A site of action for tRNA mediated regulation of the *ilvOEDA* operon of *Escherichia coli* K12. Mol. Gen. Genet. 167:227–234.

25. Martinez, E., B. Bartolome, and F. de la Cruz. 1988. pACYC184-derived cloning vectors containing the multiple cloning site and *lacZα* reporter gene of pUC8/9 and pUC18/19 plasmids. Gene 68:159–162.

26. Miller, J. H. 1992. A laboratory manual for *Escherichia coli* and related bacteria. Cold Spring Harbor Laboratory Press, Plainview, N.Y.

27. Motorin, Y., and H. Grosjean. 1999. Multisite-specific tRNA: m5C-methyltransferase (Trm4) in yeast *Saccharomyces cerevisiae*: identification of the gene and substrate specificity of the enzyme. RNA 5:1105–1118.

28. O'Neill, J. P., and M. Freundlich. 1972. Two forms of biosynthetic aceto-hydroxy acid synthetase in *Salmonella typhimurium*. Biochem. Biophys. Res. Commun. 48:437–443.

29. Parsons, S. J., and R. O. Burns. 1971. Methods Enzymol. 17A:793–799.

30. Phillips, A. T., J. I. Nuss, J. Moosic, and C. Foshay. 1972. Alternate pathway for isoleucine biosynthesis in *Escherichia coli*. J. Bacteriol. 109:714–719.

31. Picardeau, M., H. Bauby, and I. Saint Girons. 2003. Genetic evidence for the existence of two pathways for the biosynthesis of methionine in the *Leptospira* spp. FEMS. Microbiol. Lett. 225:257–262.

32. Ren, S. X., G. Fu, X. G. Jiang, R. Zeng, Y. G. Miao, H. Xu, Y. X. Zhang, H. Xiong, G. Lu, L. F. Lu, H. Q. Jiang, J. Jia, Y. F. Tu, J. X. Jiang, W. Y. Gu, Y. Q. Zhang, Z. Cai, H. H. Sheng, H. F. Yin, Y. Zhang, G. F. Zhu, M. Wan, H. L. Huang, Z. Qian, S. Y. Wang, W. Ma, Z. J. Yao, Y. Shen, B. Q., Qiang, Q. C. Xia, X. K. Guo, A. Danchin, I. Saint Girons, R. L. Somerville, Y. M. Wen, M. H. Shi, Z. Chen, J. G. Xu, and G. P. Zhao. 2003. Unique physiological and pathogenic features of *Leptospira interrogans* revealed by whole genome sequencing. Nature 422:888–893.

33. Robinson, I. M., and M. J. Allison. 1969. Isoleucine biosynthesis from 2-methylbutyric acid by anaerobic bacteria from the rumen. J. Bacteriol. 97:1220–1226.

34. Sai, T., K. Aida, and T. Uemura. 1969. Studies on (−)-citramalic acid formation by respiration-deficient yeast mutants. V. Purification and some properties of citramalate condensing enzyme. J. Gen. Appl. Microbiol. 15:345–363.

35. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

36. Somers, J. M. 1973. Genetic fine structure of the leucine operon of *Escherichia coli* K-12. J. Bacteriol. 113:1268–1272.

37. Te Brugge, L., and H. Louw. 1985. Addition of rabbit serum to EMJH medium improves isolation of *Leptospira interrogans* serovar hardjo. Onderstepoort J. Vet. Res. 52:53–54.

38. Umbarger, H. E. 1978. Amino acid biosynthesis and its regulation. Annu. Rev. Biochem. 47:533–606.

39. Vollbrecht, D. 1974. Three pathways of isoleucine biosynthesis in mutant strains of *Saccharomyces cerevisiae*. Biochim. Biophys. Acta 362:382–389.

40. Wechsler, J. A. 1969. Antipolarity in the *ilv* operon of *Escherichia coli* K-12. J. Bacteriol. 98:1179–1194.

41. Westfall, H. N., N. W. Charon, and D. E. Peterson. 1983. Multiple pathways for isoleucine biosynthesis in spirochete *Leptospira*. J. Bacteriol. 154:846–853.

42. Wolf, Y., I. Rogozin, N. Grishin, R. Tatusov, and E. V. Koonin. 2001. Genome trees constructed using five different approaches suggest new major bacterial clades. BMC. Evol. Biol. 1:8.

43. Xie, G., C. Forst, C. Bonner, and R. A. Jenen. 2002. Significance of two distinct types of tryptophan synthase beta chain in bacteria, archaea and higher plants. Genome Biol. 3:research 0004.1–0004.13.
