
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2005 by The American Society for Biochemistry and Molecular Biology, Inc.

**A Novel Putrescine Utilization Pathway Involves γ-Glutamylated Intermediates of Escherichia coli K-12**

Received for publication, September 28, 2004, and in revised form, December 6, 2004
Published, JBC Papers in Press, December 8, 2004, DOI 10.1074/jbc.M411114200

Shin Kurihara‡, Shinpei Oda‡, Kenji Kato‡, Hyeon Guk Kim§, Takashi Koyanagi‡, Hidehiko Kumagai¶, and Hideyuki Suzuki∥

*From the Division of Integrated Life Science, Graduate School of Biostudies, Kyoto University, Oiwake-cho, Kitashirakawa, Sakyo-ku, Kyoto 606-8502, Japan*

---

A novel bacterial putrescine utilization pathway was discovered. Seven genes, the functions of whose products were not known, are involved in this novel pathway. Five of them encode enzymes that catabolize putrescine; one encodes a putrescine importer, and the other encodes a transcriptional regulator. This novel pathway involves six sequential steps as follows: 1) import of putrescine; 2) ATP-dependent γ-glutamylation of putrescine; 3) oxidation of γ-glutamylputrescine; 4) dehydrogenation of γ-glutamyl-γ-aminobutyraldehyde; 5) hydrolysis of the γ-glutamyl linkage of γ-glutamyl-γ-aminobutyrate; and 6) transamination of γ-aminobutyrate to form the final product of this pathway, succinate semialdehyde, which is the precursor of succinate.

γ-Glutamyl compounds are widely found in both prokaryotic and eukaryotic cells. For example, glutathione (γ-glutamylcysteinylglycine) is a very important antioxidant (1) for living cells. We have studied bacterial γ-glutamyltranspeptidase (GGT)¹ (2, 3) that catalyzes the hydrolysis of the γ-glutamyl linkage of glutathione (4). During the course of our study on *Escherichia coli* GGT, we unexpectedly found that the γ-glutamyl linkage of γ-glutamyl-p-nitroanilide (γ-GpNA) was cleaved by the cell-free extract of a GGT-deficient strain. The enzyme that is responsible for this reaction was purified, and the relevant gene, *ycjL*, was identified. Further study of *ycjL* and neighboring genes (*ycj* gene cluster) revealed that the products of the *ycj* gene cluster constitute a novel putrescine-utilizing pathway in which γ-Glu-Put is involved.

---

*This work was supported in part by Grant-in-aid for Scientific Research 15580061 from the Ministry of Education, Culture, Sports, Science, and Technology of Japan. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.*

‡ Supported by the 21st Century COE Program of the Ministry of Education, Culture, Sports, Science, and Technology of Japan.

§ Supported by the BK21 Program of Graduate School of Biostudies, Kyoto University and Graduate School of Biotechnology, Korea University.

¶ Present address: Research Institute of Agricultural Resources, Ishikawa Agricultural College, Nonoichi-cho, Ishikawa-gun, Ishikawa 921-8836, Japan.

∥ To whom correspondence should be addressed: Division of Integrated Life Science, Graduate School of Biostudies, Kyoto University, Oiwake-cho, Kitashirakawa, Sakyo-ku, Kyoto 606-8502, Japan. Tel.: 81-75-753-6278; Fax: 81-75-753-6275; E-mail: hideyuki@lif.kyoto-u.ac.jp.

¹ The abbreviations used are: GGT, γ-glutamyltranspeptidase; γ-Glu-Put, γ-glutamylputrescine; γ-Glu-GABA, γ-glutamyl-γ-aminobutyrate; GABA, γ-aminobutyrate; γ-GpNA, γ-glutamyl-p-nitroanilide; IPA, isopropylamine; HPLC, high pressure liquid chromatography; MS, mass spectrometry; GabT, GABA aminotransferase; GabD, succinic semialdehyde dehydrogenase.

Putrescine, the substrate of this pathway, is one of polyamines that are found in a wide range of organisms from bacteria to plants and animals especially when these cells are proliferating or in stressful conditions (5). Also in *E. coli*, putrescine plays important roles in cell proliferation and normal cell growth (5). Putrescine is synthesized from ornithine by ornithine decarboxylase (SpeC) or from arginine by the sequential actions of arginine decarboxylase (SpeA) and agmatinase (SpeB) in *E. coli* cells (6). Putrescine is converted to spermidine, another polyamine, by the addition of an aminopropyl group derived from decarboxylated S-adenosylmethionine through the activity of spermidine synthase (SpeE) (7).

Putrescine has been reported to be converted to γ-aminobutyraldehyde by putrescine transaminase and then to be oxidized to γ-aminobutyrate (GABA) by aminobutyraldehyde dehydrogenase (8). However, neither of the genes encoding this pathway has been identified yet. GABA is converted to succinate semialdehyde by GABA aminotransferase (GabT) and then is oxidized to succinate by succinate semialdehyde dehydrogenase (GabD) (9). Recently, it was reported that the ΔgabT strain grew in minimal medium containing putrescine as a sole nitrogen source but not in minimal medium containing GABA instead of putrescine (10). This unexpected growth in the presence of putrescine likely resulted from the specific induction of a gab-independent enzyme by putrescine. It was reported that an excess amount of putrescine is exported by PotE, putrescine ornithine antiporter (11), and that putrescine outside of the cell is imported by PotFGHI, the ABC transporter (12), according to the environment in which *E. coli* lives.

In this paper we report on a completely new catabolic pathway for extracellular putrescine. This pathway involves the γ-glutamylation of putrescine and plays an important physiological role to utilize putrescine in nutrient starvation. All the enzymes, a transporter, and a regulator that constitute this pathway are encoded by the *ycj* gene cluster.²

---

### MATERIALS AND METHODS

#### Media and Reagents—Strains were grown in M9-tryptone (M9 minimal medium (13) except that 1% Bacto-tryptone was used instead of 0.2% glucose) with high aeration (140 rpm) at 37 °C in the determination of enzyme activities, accumulation of amine or amino acids in cells, or overexpression of proteins. To determine the putrescine utilization of various strains, W-Put plates (W salts minimal medium (14) containing 0.4% glucose as a sole carbon source and 0.2% putrescine as a sole nitrogen source (10)) were used. [1,4-¹⁴C]Putrescine dihydrochloride (107 mCi/mmol) was purchased from Amersham Biosciences.

#### Bacterial Strains and Plasmids—P1 transduction and DNA manipulation and transformation were performed by the standard methods (13, 15). A DNA fragment of Kohara clone 257 (16) containing the *ycj* gene cluster was cloned onto high copy number plasmids. The bacterial

---

² The relationship between gene designations and their products is summarized in Table II.

TABLE I  
Strains and plasmids used in this study  

| Strain or plasmid | Characteristic or sequence | Source or Ref. |
|-------------------|---------------------------|----------------|
| Strains          |                           |                |
| KJ107            | SH639 ΔgabT::FRT          | This study     |
| KJ109            | SH639 ΔgoaG::kan⁺ ΔgabT::FRT | This study     |
| SH639            | F⁻ Δggt-2                 | 2              |
| SH703            | F⁻ Δggt-2 zhg::Tn10       | 30             |
| SK111            | SH703 ΔycjLC::kan⁺;       | This study     |
| SK112            | pBR-ycjL/SK111            | This study     |
| SK121            | pBR322-ycjLC/SK111        | This study     |
| SK187            | SH639 ΔgoaG::kan⁺         | This study     |
| SK189            | SH639 Δ(ordL-goaG)::kan⁺  | This study     |
| SK247            | SH639 ΔycjLC::FRT Δ(aldH-ordL-goaG)::FRT | This study     |
| SK263            | pACYC 184/SK247           | This study     |
| SK279            | pACYC-aldH-ordL/SK247     | This study     |
| SK283            | pACYC-ordL/SK247          | This study     |
| SK285            | pACYC-aldH/SK247          | This study     |
| SK303            | pBelo-ycjLC-aldH-ordL-goaG/SK247 | This study     |
| SK306            | pBeloBAC 11/SK247         | This study     |
| SO23             | SH639 ΔycjJ::tet⁺         | This study     |
| SO24             | pBR-ycjJ/SO23             | This study     |
| SO58             | SH639 ΔycjKLC::cat⁺FRT Δ(aldH-ordL-goaG)::kan⁺ | This study     |
| SO60             | pmal-c2X/SO58             | This study     |
| SO61             | pmal-c2X-ycjK/SO58        | This study     |
| SO66             | pQE-80L-ordL/SK247        | This study     |
| SO67             | pQE-80L/SK247             | This study     |
| Plasmids         |                           |                |
| pACYC 184        | p15A replicon cat⁺tet⁺    | New England Biolabs |
| pACYC-aldH       | p15A replicon cat⁺aldH⁺   | This study     |
| pACYC-aldH-ordL  | p15A replicon cat⁺aldH⁺-ordL⁺ | This study     |
| pACYC-aldH-ordL-goaG | p15A replicon cat⁺aldH⁺-ordL⁺-goaG⁺ | This study     |
| pACYC-ordL       | p15A replicon cat⁺ordL⁺; after aldH-ordL were cloned, the DNA sequence encodes aldH were looped out | This study     |
| pBeloBAC 11      | Mini-F replicon cat⁺      | New England Biolabs |
| pBelo-ycjLC-aldH-ordL-goaG | Mini-F replicon cat⁺ycjL⁺ycjC⁺aldH⁺ordL⁺goaG⁺ | This study     |
| pBR322           | ColE1 replicon rop⁺bla⁺tet⁺ | New England Biolabs |
| pBR-ycjJ         | ColE1 replicon rop⁺bla⁺tet⁺ycjJ⁺ | This study     |
| pBR-ycjL         | ColE1 replicon rop⁺bla⁺tet⁺ycjL⁺ | This study     |
| pBR-ycjLC        | ColE1 replicon rop⁺bla⁺ycjL⁺C⁺ | This study     |
| pmal-c2X         | ColE1 replicon f1IGrop⁺bla⁺lacIq Φ(malE-'lacZα) (Hyb) (tacp); (malE without signal peptide) | New England Biolabs |
| pmal-c2X-ycjK    | ColE1 replicon f1IGrop⁺bla⁺lacIq Φ(malE-ycjK) (Hyb) (tacp); (malE without signal peptide) | This study     |
| pQE-80L          | ColE1 replicon bla⁺lacIq T5p-(His)₆ | Qiagen        |
| pQE-80L-ordL     | ColE1 replicon bla⁺lacIq T5p-(His)₆-ordL⁺ | This study     |

strains and plasmids used in this study are summarized in Table I. Genes were disrupted according to the method of Datsenko and Wanner (17) or as described previously (18).

Analysis of γ-Glutamyl Compounds, Amino Acids, and Amines—The γ-glutamyl compounds, amino acids, and amines in the samples were measured by using an HPLC system (model LC-9A; Shimadzu, Kyoto, Japan) equipped with a Shim-pack Amino-Na column (Shimadzu), with gradient elution at 60°C at a flow rate of 0.6 ml/min. The gradient of the mobile phase was formed with buffer A (66.6 mM tri-sodium citrate dehydrate, 1% perchloric acid, 7% ethanol (pH 3.2)) and buffer B (200 mM tri-sodium citrate dehydrate, 200 mM boric acid, 0.12 N NaOH (pH 10)). The concentration of buffer B was kept at 0% until 9 min. It was linearly increased to 7% from 9 to 13 min, to 8% from 13 to 17.2 min, and then to 11%. o-Phthalaldehyde was used as the detection reagent (19) as described previously (20), and the fluorescence was detected with a fluorescence detector (model RF-535; Shimadzu) as the absorbance at 450 nm, with excitation at 348 nm. Standard compounds were purchased from Nacalai Tesque, Kyoto, Japan, and Sigma, except for γ-Glu-Put and γ-Glu-GABA, which were synthesized as described below. In our HPLC system, γ-Glu-GABA, glutamate, GABA, and γ-Glu-Put was eluted at 8, 11, 26, and 29 min, respectively. Before measuring amino acids from the cell-free extract, the extract was previously treated with trichloroacetic acid (final concentration, 10%) to precipitate its proteins.

Assays for Enzymes—The γ-GpNA hydrolysis activity of YcjL and GGT was assayed as described previously (2). γ-Glu-Put synthetase activity was determined by measuring the γ-Glu-Put synthesized from glutamate and putrescine. The reaction mixture containing 50 mM sodium glutamate, 50 mM putrescine dihydrochloride, 15 mM ATP, 60 mM MgCl₂, and 200 mM imidazole chloride buffer (pH 7.0) was incubated at 37°C. γ-Glu-Put oxidase activity was assayed by measuring the decrease in γ-Glu-Put by HPLC. The reaction mixture containing 1 mM γ-Glu-Put and 10 mM potassium phosphate buffer (pH 7.0) was incubated at 37°C. γ-Glu-GABA hydrolase activity was determined by measuring the glutamate released from γ-Glu-GABA. The reaction mixture containing various concentrations of γ-Glu-GABA and 50 mM imidazole-HCl buffer (pH 8.7) was incubated at 37°C. GABA aminotransferase activity was determined by measuring the glutamate formed from GABA and α-ketoglutarate. The reaction mixture containing 10 mM GABA, 10 mM α-ketoglutarate monosodium salt, and 100 mM potassium phosphate buffer (pH 7.0) was incubated at 37°C. Adding trichloroacetic acid (final concentration, 10%) terminated the reactions, and the filtrates were analyzed with HPLC.

Purification of Proteins—Following the γ-GpNA hydrolysis activity, YcjL was purified from the cell-free extract of SH639 (Δggt-2) by ammonium sulfate precipitation (30–50%) and column chromatographies using DEAE-Cellulofine A-500m (Chisso, Tokyo, Japan), phenyl-Sepharose CL-4B (Amersham Biosciences), Cellulofine GC-700m (Chisso), and MiniQ (Amersham Biosciences). During the purification, the protein was basically dissolved in buffer C (50 mM Tris-HCl (pH 8.7) and 5 mM β-mercaptoethanol). After the ammonium sulfate precipitation, the enzyme was dissolved in buffer C and dialyzed against the same buffer. The dialyzed enzyme solution was applied to DEAE-Cellulofine A-500m previously equilibrated with buffer C. Without washing the column, YcjL was eluted with 0.2 M NaCl in buffer C from the column. In the purification step using phenyl-Sepharose CL4B, the enzyme solution was dialyzed against buffer D (0.8 M ammonium sulfate in buffer C) and applied to the column equilibrated with buffer D. The column was washed with buffer D, and YcjL was eluted at the end of a linear gradient formed between buffer D (0.8 M ammonium sulfate in buffer C) and

4604

**Novel Putrescine Utilization Pathway of E. coli**

![Diagram](https://i.imgur.com/yourimageurl.png)

FIG. 1. Organization of the $ycj$ and $aldH-ordL-goaG$ loci. The relative location and putative transcriptional orientation of genes at these loci are represented by arrows. $ycjJ_p$, $ycjK_p$, $ycjLC_p$, and $(aldH-ordL-goaG)_p$ represent the locations of putative promoters.

buffer E (40% ethylene glycol in buffer C). In the purification step using Cellulofine GC-700, the enzyme solution was concentrated by the ammonium sulfate precipitation and applied to the column previously equilibrated with buffer C. The active fraction was applied to the MiniQ column previously equilibrated with buffer C. YcjL was eluted with a linear gradient formed between buffer C and 0.5 M NaCl in buffer C. YcjL was eluted when the concentration of NaCl was 0.15 M. MalE-YcjK was overexpressed after induction by isopropyl-$\beta$-D-thiogalactopyranoside and purified from the cell-free extract by ammonium sulfate precipitation (30–50%) and an amylose column (New England Biolabs) chromatography according to the protocol recommended by the manufacturer.

**N-terminal Amino Acid Sequence—**The N-terminal amino acid sequence of YcjL was determined by Edman degradation as described previously (21).

**Nuclear Magnetic Resonance Spectrometry and Mass Spectrometry—**Purified $\gamma$-Glu-Put and $\gamma$-Glu-GABA was dissolved in D₂O and then analyzed with a Bruker 500 MHz spectrometer. The mass of the purified $\gamma$-Glu-Put was measured by fast atom bombardment (FAB⁺)-mass spectrometry (model JEOLJMS600; JEOL, Tokyo, Japan) using glycrol as a matrix. Purified $\gamma$-Glu-Put was fragmented by using N₂ as a collision gas (at 5 millitorrs) at a collision energy (RO2 to RO1) of 10–11.5 eV, and then the masses of fragments were measured by ion spray mass spectrometry (model API-3000; PE Sciex).

**Synthesis and Purification of $\gamma$-Glu-Put and $\gamma$-Glu-GABA—**$\gamma$-Glu-Put was originally purified by cation exchange chromatography using Dowex 50W-X8 (Dow Chemical) from the cell-free extract of SK247. After the structure was identified by NMR, MS, and MS/MS spectroscopic analysis, $\gamma$-Glu-Put was enzymatically synthesized from putrescine and glutamate using MalE-YcjK and purified by Dowex 50W-X8. The purity was authenticated by HPLC. $\gamma$-Glu-GABA was purified by anion exchange chromatography using Dowex 1-X8 (Dow Chemical) from the cell-free extract of SK279. The structure was identified by NMR. The purity was checked by HPLC.

**Transport Assay—**The transport assay was performed as described previously (22, 23) using [$^{14}$C]putrescine dihydrochloride, except that M9-tryptone medium was used instead of M63 medium.

---

### RESULTS

#### A Novel $\gamma$-GpNA Hydrolysis Activity of GGT-deficient Strain of E. coli—$\gamma$-GpNA is an artificial substrate widely used to measure GGT activity (24). We found a novel $\gamma$-GpNA hydrolysis activity in the cytosol of E. coli strain SH639 ($\Delta ggt-2$) grown at high aeration (60 ml of culture in a 300-ml Erlenmeyer flask shaken at 140 rpm). To identify the protein responsible for this activity, the relevant protein was purified, and its N-terminal amino acid sequence was determined by the Edman degradation method. The amino acid sequence obtained, MENIMNNPVIG, was compared with sequences in a data base, Colibri (genolist.pasteur.fr/Colibri/), and the relevant gene was found to be $ycjL$ whose function was not known. Strain SK111 ($\Delta ycjLC$) completely lost the $\gamma$-GpNA hydrolysis activity, and SK112 (pBR-$ycjL$/SK111) overproduced the protein responsible for the activity. Purified YcjL hydrolyzed glutamine to glutamate and theanine ($\gamma$-glutamylethylamide) to glutamate and ethylamine. These results suggested that YcjL is a novel enzyme that catalyzes the hydrolysis of the $\gamma$-glutamyl linkage.

#### Hypothesis for Physiological Roles of $ycjL$ and Genes Around $ycjL$—It was not easy to determine the physiological role of YcjL because the $\Delta ycjL$ mutant could grow on M9-glucose minimal medium as well as the wild-type strain. Several genes

A

$\begin{aligned}
& \text{CH}_3 \\
& \text{H}_2\text{N-CH-CH}_3 \\
& \downarrow \\
& \text{IpuG (Importer)} \\
& \text{CH}_3 \\
& \text{H}_2\text{N-CH-CH}_3 \\
& \downarrow \\
& \text{ATP} \\
& \text{ADP} \\
& \text{IpuC} \\
& \text{CH}_3 \\
& \gamma-\text{Glu-CO-NH-CH-CH}_3 \\
& \text{NAD(P)H + O}_2 \\
& \text{NAD(P)}^+ + \text{H}_2\text{O} \\
& \text{IpuD (A, B, E)} \\
& \text{CH}_3 \\
& \gamma-\text{Glu-CO-NH-CH-CH}_2-\text{OH} \\
& \text{H}_2\text{O} \\
& \text{IpuF} \\
& \text{CH}_3 \\
& \text{H}_2\text{N-CH-CH}_2-\text{OH} \\
& \text{NAD(P)}^+ \\
& \text{NAD(P)H} \\
& \text{Alcohol DH} \\
& \text{CH}_3 \\
& \text{H}_2\text{N-CH-CHO} \\
& \text{NAD(P)}^+ + \text{H}_2\text{O} \\
& \text{NAD(P)H + H}^+ \\
& \text{IpuI, H} \\
& \text{CH}_3 \\
& \text{H}_2\text{N-CH-COOH} \\
& \text{Alanine}
\end{aligned}$

$\begin{aligned}
& \text{IPA} \\
& \text{Glu} \\
& \text{Glu-IPA} \\
& \text{γ-Glu-alaninol} \\
& \text{Alaninol} \\
& \text{2-Aminopropionaldehyde}
\end{aligned}$

B

$\begin{aligned}
& \text{H}_2\text{N-CH}_2-\text{CH}_2-\text{CH}_2-\text{NH}_2 \\
& \downarrow \\
& \text{YcjJ (Importer)} \\
& \text{Glu} + \text{H}_2\text{N-(CH}_2)_4-\text{NH}_2 \\
& \text{ATP} \\
& \text{ADP} \\
& \text{YcjK} \\
& \gamma-\text{Glu-CO-NH-(CH}_2)_4-\text{NH}_2 \\
& \text{H}_2\text{O} + \text{O}_2 \\
& \text{H}_2\text{O}_2 + \text{NH}_3 \\
& \text{OrdL} \\
& \gamma-\text{Glu-CO-NH-(CH}_2)_3-\text{CHO} \\
& \text{NAD(P)}^+ + \text{H}_2\text{O} \\
& \text{NAD(P)H + H}^+ \\
& \text{AldH} \\
& \gamma-\text{Glu-CO-NH-(CH}_2)_3-\text{COOH} \\
& \text{H}_2\text{O} \\
& \text{YcjL} \\
& \text{Glu} + \text{H}_2\text{N-(CH}_2)_3-\text{COOH} \\
& \alpha-\text{KG} \\
& \text{Glu} \\
& \text{GoaG} \\
& \text{OHC-(CH}_2)_2-\text{COOH} \\
& \text{NAD(P)}^+ + \text{H}_2\text{O} \\
& \text{NAD(P)H + H}^+ \\
& \text{(GabD)} \\
& \text{HOOC-(CH}_2)_2-\text{COOH} \\
& \text{to TCA cycle}
\end{aligned}$

$\begin{aligned}
& \text{Put} \\
& \gamma-\text{Glu-Put} \\
& \gamma-\text{Glu-γ-aminobutyraldehyde} \\
& \gamma-\text{Glu-GABA} \\
& \text{GABA} \\
& \text{Succinate semialdehyde} \\
& \text{Succinate}
\end{aligned}$

$<\text{Pseudomonas sp.}>$

$<\text{E. coli}>$

FIG. 2. A, postulated pathway for the degradation of isopropylamine by Pseudomonas sp. (25). B, postulated pathway for the degradation of putrescine by E. coli. Abbreviations: IPA, isopropylamine; PUT, putrescine; γ-Glu-Put, γ-glutamyl-putrescine; γ-Glu-GABA, γ-glutamyl-γ-aminobutyrate; GABA, γ-aminobutyrate.

catalyzes the reaction of γ-Glu-Put synthesis in vivo, SO61 (pmal-c2X-ycjK/ΔycjKLC Δ(aldH-ordL-goaG); ycjJ⁺ ycjK⁺ ycjL⁻ ycjC⁻ aldH⁻ ordL⁻ goaG⁻) and SO60 (pmal-c2X/ΔycjKLC Δ(aldH-ordL-goaG); ycjJ⁺ ycjK⁻ ycjL⁻ ycjC⁻ aldH⁻ ordL⁻ goaG⁻) were grown in M9-tryptone, and the cell-free extract was subjected to HPLC. The strain with the plasmid encoding MalE-YcjK exhibited γ-Glu-Put accumulation, whereas the strain with the empty vector did not. To show γ-Glu-Put synthesis by YcjK in vitro, MalE-YcjK was purified and mixed with putrescine, glutamate, ATP, and Mg²⁺, and the mixture was incubated at 37°C. The synthesis of γ-Glu-Put was confirmed by HPLC and was not observed without ATP and Mg²⁺. These results strongly indicated that YcjK is γ-Glu-Put synthetase, and the products of the ycj gene cluster catabolize putrescine.

Putrescine is reported to be degraded after acetylation in mammalian cells (27). Referring to that report (27), we hypothesized that putrescine is degraded in E. coli after γ-glutamylation instead of acetylation in mammalian cells. The entire novel putrescine utilization pathway of E. coli proposed here is summarized in Fig. 2B.

OrdL Is γ-Glu-put Oxidase and AldH Is γ-Glu-γ-aminobutyraldehyde Dehydrogenase—After the YcjK reaction, γ-Glu-Put was expected to be oxidized to γ-Glu-γ-aminobutyraldehyde by OrdL and then to γ-Glu-γ-aminobutyrate (γ-Glu-GABA) by AldH (Fig. 2B). To show these reactions in vivo, four strains, SK263 (pACYC184/ΔycjLC Δ(aldH-ordL-goaG); ycjJ⁺ ycjK⁺ ycjL⁻ ycjC⁻ aldH⁻ ordL⁻ goaG⁻), SK285 (pACYC-aldH/ΔycjLC Δ(aldH-ordL-goaG); ycjJ⁺ ycjK⁺ aldH⁺ ycjL⁻ ycjC⁻ ordL⁻ goaG⁻), SK283 (pACYC-ordL/ΔycjLC Δ(aldH-ordL-

4606 Novel Putrescine Utilization Pathway of E. coli

$\gamma$-Glu-Put $\gamma$-Glu-Put $\gamma$-Glu-GABA

A B C D

FIG. 3. Detection of amines or amino acids in E. coli cells by HPLC. A, SK263 ($ycjJ^{+} ycjK^{+}$). B, SK285 ($ycjJ^{+} ycjK^{+} aldH^{+}$). C, SK283 ($ycjJ^{+} ycjK^{+} ordL^{+}$). D, SK279 ($ycjJ^{+} ycjK^{+} aldH^{+} ordL^{+}$). $\gamma$-Glutamyl-$\gamma$-aminobutyraldehyde could not be detected with our system. The numbers below the charts indicate the elution time in min. Abbreviations: $\gamma$-Glu-Put, $\gamma$-glutamyl-putrescine; $\gamma$-Glu-GABA, $\gamma$-glutamyl-$\gamma$-aminobutyrate.

pression and that the deletion of $goaG$, the product of which is located downstream of YcjL in the metabolic pathway, did not influence $ycjL$ expression. YcjK catalyzes the $\gamma$-glutamylation of putrescine to form the $\gamma$-glutamyl linkage of $\gamma$-Glu-Put, whereas YcjL catalyzes the hydrolysis of the $\gamma$-glutamyl linkage of $\gamma$-Glu-GABA. If $\gamma$-Glu-Put is a better substrate for YcjL than $\gamma$-Glu-GABA, there is no good reason for the $\gamma$-glutamate putrescine consuming ATP. To determine which is the better substrate, $K_m$ and $k_{cat}$ values of YcjL for $\gamma$-Glu-Put and $\gamma$-Glu-GABA were determined. The $K_m$ for $\gamma$-Glu-Put was 18.5 mM and that for $\gamma$-Glu-GABA was 2.93 mM. The $k_{cat}$ for $\gamma$-Glu-Put was 434 min$^{-1}$ and that for $\gamma$-Glu-GABA was 2490 min$^{-1}$. The $k_{cat}/K_m$ for $\gamma$-Glu-Put was 23.5 and that for $\gamma$-Glu-GABA was 850, i.e., the $k_{cat}/K_m$ for $\gamma$-Glu-GABA was 36 times higher than that for $\gamma$-Glu-Put. The results showed that $\gamma$-Glu-GABA rather than $\gamma$-Glu-Put is the natural substrate of YcjL. The YcjL activity measured as $\gamma$-GpNA hydrolysis activity was 16 times higher when E. coli was grown in the W-Put medium rather than in the W salts minimal medium containing 0.2% NH$_4$Cl instead of putrescine. These results indicated that YcjL is $\gamma$-Glu-GABA hydrolase.

GoaG Is the Second GABA Aminotransferase—Previously, putrescine was reported to be oxidized directly to GABA (8) and be degraded to succinate by GabT and GabD (9). It was reported in another study on the $gabDTPC$ operon that encodes the GABA degradation pathway in E. coli that the strain $\Delta gabT$ did not grow in W salts minimal medium containing 0.2% GABA as a sole nitrogen source, whereas this strain grew in W-Put medium (10). Moreover, the cell-free extract of $\Delta gabT$ grown in W-Put medium exhibited significant GABA aminotransferase activity. These results suggested that besides GabT, there is another GABA aminotransferase that is induced by putrescine. We suspected that this second aminotransferase was encoded by $goaG$ in the $ycj$ gene cluster. The cell-free extract of KJ107 ($\Delta gabT$), SK187 ($\Delta goaG$), and KJ109 ($\Delta gabT \Delta goaG$) grown in M9-tryptone supplemented with 5 mM putrescine exhibited 68, 35, and 1.3% GABA aminotransferase activity, respectively, compared with SH639 ($gabT^{+} goaG^{+}$). These results indicate that both GabT and GoaG are important in the catabolism of GABA. In the cell-free extracts of the $\Delta goaG$ strain, accumulation of GABA was observed. These results agreed well with the putrescine-catabolizing pathway postulated in Fig. 2B.

YcjJ Is a Novel Putrescine Importer—Downstream of $ycjK$, there is a gene, $ycjJ$, whose product is suggested to be an amine transporter from its amino acid sequence. From the results described so far, the substrate of YcjJ is expected to be putrescine.

[14C]Putrescine uptake

Time (min)

FIG. 4. Putrescine uptake by the $ycjJ$ mutants. Strains SH639 (wild type, triangles), SO23 ($\Delta ycjJ$, diamonds), and SO24 (pBR-$ycjJ/\Delta ycjJ$, squares) are shown. After the absorbance at 600 nm had reached 0.6–0.7, the cells were harvested and suspended in M9-tryptone containing 60 $\mu$g of chloramphenicol/ml. Cell suspensions were incubated in the presence of 10 $\mu$M (107 mCi/mmol) labeled putrescine, and samples were withdrawn at the time points indicated. Each value is the average of triplicate determinations.

To show that putrescine is imported by YcjJ, a transporter assay using [14C] putrescine was performed. The putrescine transport activity of SO23 ($\Delta ycjJ$) was reduced compared with that of SH639 ($ycjJ^{+}$), whereas SO24 (pBR-$ycjJ$/SO23) imported putrescine over 24 times more effectively than SH639 ($ycjJ^{+}$) (Fig. 4). The result indicated that YcjJ is a novel putrescine importer. In SO23 ($\Delta ycjJ$), putrescine import activity was very much diminished, although the strain is $potF^{+}G^{+}H^{+}I^{+}$ (12). The result indicates that under our growth conditions for E. coli, the putrescine importer mainly expressed is YcjJ. A more detailed analysis of YcjJ will be published elsewhere.

Disruption and Complementation of $ycj$ Gene Cluster Influence the Utilization of Putrescine in E. coli—To show that the products of the $ycj$ gene cluster play an important role in the catabolism of putrescine, various parts of the $ycj$ gene cluster were disrupted or complemented, and the mutants and transformants were grown on W-Put medium. Whereas SO23 ($\Delta ycjJ$) and SO25 (pBR322/SO23) were not able to grow on the W-Put plates, SH639 ($ycjJ^{+}$) and SO24 (pBR-$ycjJ$/SO23) were (Fig. 5A). These results suggested that YcjJ is the main putrescine importer when E. coli grows on medium containing putrescine as a sole nitrogen source. Whereas SK247 ($\Delta(ycjLC-aldH-ordL-goaG)$) and SK306 (pBeloBAC11/SK247) could not grow on the W-Put plates, SH639 ($ycjL^{+}C^{+} aldH^{+} ordL^{+} goaG^{+}$) and SK303 (pBelo-$ycjLC$-aldH-ordL-$goaG$/SK247) could (Fig. 5B). These results suggested that the catabolic pathway shown in Fig. 2B is the main one when E. coli grows in the medium containing putrescine as the sole source of nitrogen. KJ109 ($\Delta gabT \Delta goaG$) did not grow on W-Put plates, whereas KJ107 ($\Delta gabT$), SK187 ($\Delta goaG$), and SH639 ($goaG^{+} gabT^{+}$) did (Fig. 5C). The result suggested that both GoaG and GabT are important in the catabolism of GABA. The two enzymes can substitute for each other in the catabolic pathway. This is consistent with the results from the assay of the GABA aminotransferase activity of SK187 ($\Delta goaG$), KJ107 ($\Delta gabT$), and KJ109 ($\Delta goaG \Delta gabT$).

Novel Putrescine Utilization Pathway of E. coli

$\Delta y c j J /$  
$\Delta y c j J$  

$\Delta y c j J$  

$p /$  
$\Delta y c j J$  

$W T$

$p-(y c j L C-a l d H-$  
$o r d L-g o a G) /$  
$\Delta(y c j L C-a l d H-$  
$o r d L-g o a G)$  

$\Delta(y c j L C-a l d H-$  
$o r d L-g o a G)$  

$p /$  
$\Delta(y c j L C-a l d H-$  
$o r d L-g o a G)$  

$W T$

$\Delta g a b T$  

$\Delta g o a G$  

$\Delta g a b T$  

$\Delta g o a G$  

$W T$

A B C

FIG. 5. Physiological role of $y c j$ gene products in E. coli cells grown in minimal medium containing putrescine as a sole nitrogen source. A, strains relevant to $y c j J$ were streaked on plates of W salts minimal medium containing 0.2% putrescine as a sole nitrogen source (W-Put plates). The vector used was pBR322. "p" in the figure indicates the plasmid vector. B, strains relevant to the $(y c j L C-a l d H-o r d L-g o a G)$ gene cluster were streaked on W-Put plates. The vector used was pBeloBAC11. C, strains relevant to $g o a G$ and $g a b T$ were streaked on W-Put plates.

TABLE II  
The function of $y c j$ gene cluster and their new gene names proposed  

| Previous gene names | Function | New gene names |
|---------------------|----------|----------------|
| $y c j J$           | Putrescine importer | $p u u P$ |
| $y c j K$           | $\gamma$-Glu-Put synthetase | $p u u A$ |
| $o r d L$           | $\gamma$-Glu-Put oxidase | $p u u B$ |
| $a l d H$           | $\gamma$-Glu-$\gamma$-aminobutyraldehyde dehydrogenase | $p u u C$ |
| $y c j L$           | $\gamma$-Glu-GABA hydrolase | $p u u D$ |
| $g o a G$           | GABA aminotransferase | $p u u E$ |
| $y c j C$           | Transcriptional regulator | $p u u R$ |

DISCUSSION

The Significance of $\gamma$-Glutamylation of Putrescine—In the novel pathway we proposed, putrescine is first $\gamma$-glutamylated and then catabolized (Fig. 2B), but previously, putrescine had been thought to be degraded directly to $\gamma$-aminobutyraldehyde without any modification and then to GABA (8). However, $\gamma$-aminobutyraldehyde is unstable, and its amino group and aldehyde group form cyclic $\Delta^{1}$-pyrroline nonenzymatically (8). $\gamma$-Glutamylation of putrescine may facilitate its catabolism by masking the amino group in advance and by preventing cyclization.

Regulation of the $y c j$ Gene Cluster—Nac is a $\sigma^{70}$-dependent transcriptional regulator under nitrogen-limited conditions (28). All of the members of the $y c j$ gene cluster are expected to be transcribed from $\sigma^{70}$-dependent promoters, and many results obtained so far were under nitrogen-limited conditions. Therefore, Nac was expected to control the expression of the $y c j$ gene cluster; however, there is no Nac-binding site upstream of any of these genes. Furthermore, in a previous study using microarrays (29), no member of the $y c j$ gene cluster was recognized as a Nac-controlled gene. However, the addition of putrescine to the medium as the sole nitrogen source (W-put medium) induced the expression of $y c j L$ (data not shown). This result suggested that a new system or protein (for example, YcjC) involved in the response to the nitrogen starvation controls the expression of the $y c j$ gene cluster. The addition of glucose to the M9-tryptone medium repressed the expression of $y c j L$ (data not shown). However, there is no CRP-binding site upstream of any of these genes. Furthermore, the expression of $y c j L$ was also repressed by the addition of glycerol and succinate to the M9-tryptone medium. These results suggested that the regulation of the expression of $y c j L$ is independent of CRP. YcjL was highly expressed in the early stationary phase (data not shown) when the nutrient in the medium is drying up. These results suggested that the products of the $y c j$ gene cluster are expressed when E. coli grows under conditions of starvation, and the products of the $y c j$ gene cluster catabolize putrescine both as the carbon source and as the nitrogen source. We could not perform the experiment to detect the growth of E. coli on medium containing putrescine as a sole carbon source because wild-type E. coli could not grow on that medium.

In summary, the expression of $y c j L$ (and maybe the other members of the $y c j$ gene cluster) is repressed in the presence of $y c j C$ and by the addition of another carbon or nitrogen source more easily utilized than putrescine, and is induced by nutrient starvation and by the addition of putrescine to the medium. A more detailed analysis of the regulation of the $y c j$ gene cluster will be published elsewhere.

Proposed New Gene Name for the $y c j$ Gene Cluster—We propose the name $p u u$ after putrescine utilization pathway for the $y c j$ gene cluster. $y c j J$, $y c j K$, $o r d L$, $a l d H$, $y c j L$, $g o a G$, and $y c j C$ would be renamed $p u u P$, $p u u A$, $p u u B$, $p u u C$, $p u u D$, $p u u E$, and $p u u R$, respectively (Table II).

Acknowledgments—We thank Dr. Michihiko Kataoka and Dr. Sakayu Shimizu, Graduate School of Agriculture, Kyoto University, for protein sequencing of PuuD. We are grateful to Dr. Kazuhiro Irie,

Graduate School of Agriculture, Kyoto University, for NMR and mass spectrometry analyses of γ-Glu-Put and γ-Glu-GABA. We are also indebted to Dr. Tatsuo Kurihara and Dr. Nobuyoshi Esaki, Institute for Chemical Research, Kyoto University, for MS/MS spectrometry analysis of γ-Glu-Put.

**REFERENCES**

1. Mannervik, B., Carlberg, I., and Larson, K. (1989) in *Coenzymes and Cofactors* (Dolphin, D., Rozanne P., and Avramovic, O., eds) Vol. 3, pp. 475–516, Wiley-Interscience, New York
2. Suzuki, H., Kumagai, H., and Tochikura, T. (1986) *J. Bacteriol.* **168**, 1325–1331
3. Minami, H., Suzuki, H., and Kumagai, H. (2003) *Enzyme Microb. Technol.* **32**, 431–438
4. Suzuki, H., Hashimoto, W., and Kumagai, H. (1993) *J. Bacteriol.* **175**, 6038–6040
5. Tabor, C. W., and Tabor, H. (1984) *Annu. Rev. Biochem.* **53**, 749–790
6. Boyle, S. M., Markham, G. D., Hafner, E. W., Wright, J. M., Tabor, H., and Tabor, C. W. (1984) *Gene (Amst.)* **30**, 129–136
7. Tabor, C. W., Tabor, H., and Xie, Q. W. (1986) *Proc. Natl. Acad. Sci. U. S. A.* **83**, 6040–6044
8. Shaibe, E., Metzer, E., and Halpern, Y. S. (1985) *J. Bacteriol.* **163**, 933–937
9. Metzer, E., Levitz, R., and Halpern, Y. S. (1979) *J. Bacteriol.* **137**, 1111–1118
10. Schneider, B. L., Ruback, S., Kiupakis, A. K., Kasbarian, H., Pybus, C., and Reitzer, L. (2002) *J. Bacteriol.* **184**, 6976–6986
11. Kashiwagi, K., Miyamoto, S., Suzuki, F., Kobayashi, H., and Igarashi, K. (1992) *Proc. Natl. Acad. Sci. U. S. A.* **89**, 4529–4533
12. Pistocchi, R., Kashiwagi, K., Miyamoto, S., Nukui, E., Sadakata, Y., Kobayashi, H., and Igarashi, K. (1993) *J. Biol. Chem.* **268**, 146–152
13. Miller, J. H. (1992) *A Short Course in Bacterial Genetics. A Laboratory Manual*

and Handbook for Escherichia coli and Related Bacteria, pp. 263–274 and 437, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY

14. Bueno, R., Pahel, G., and Magasanik, B. (1985) *J. Bacteriol.* **164**, 816–822
15. Sambrook, J., and Russell, D. W. (2001) *Molecular Cloning: A Laboratory Manual*, 3rd Ed., pp. 1.1–2.110, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY
16. Kohara, Y., Akiyama, K., and Isono, K. (1987) *Cell* **50**, 495–508
17. Datsenko, K. A., and Wanner, B. L. (2000) *Proc. Natl. Acad. Sci. U. S. A.* **97**, 6640–6645
18. Russell, C. B., Thaler, D. S., and Dahlquist, F. W. (1989) *J. Bacteriol.* **171**, 2609–2613
19. Benson, J. R., and Hare, P. E. (1975) *Proc. Natl. Acad. Sci. U. S. A.* **72**, 619–622
20. Suzuki, H., Izuka, S., Minami, H., Miyakawa, N., Ishihara, S., and Kumagai, H. (2003) *Appl. Environ. Microbiol.* **69**, 6399–6404
21. Suzuki, H., and Kumagai, H. (2002) *J. Biol. Chem.* **277**, 43536–43543
22. Koyanagi, T., Katayama, T., Suzuki, H., and Kumagai, H. (2004) *J. Bacteriol.* **186**, 343–350
23. Katayama, T., Suzuki, H., Koyanagi, T., and Kumagai, H. (2002) *J. Bacteriol.* **184**, 3135–3141
24. Orlowski, M. M., and Meister, A. (1963) *Biochim. Biophys. Acta* **73**, 679–681
25. de Azevedo, W. S. I., van der Ploeg, J. R., Maire, T., Lebreton, A., Kiener, A., and Leisinger, T. (2002) *Appl. Environ. Microbiol.* **68**, 2368–2375
26. Jovanovic, G., and Model, P. (1997) *J. Bacteriol.* **179**, 3095–3102
27. Seiler, N., and Al-Therib, M. J. (1974) *Biochem. J.* **144**, 29–35
28. Muse, W. B., and Bender, R. A. (1998) *J. Bacteriol.* **180**, 1166–1173
29. Zimmer, D. P., Soupene, E., Lee, H. L., Wendisch, V. F., Khodursky, A. B., Peter, B. J., Bender, R. A., and Kustu, S. (2000) *Proc. Natl. Acad. Sci. U. S. A.* **97**, 14674–14679
30. Suzuki, H., Kumagai, H., and Tochikura, T. (1987) *J. Bacteriol.* **169**, 3926–3931

Metabolism and Bioenergetics:
A Novel Putrescine Utilization Pathway
Involves γ-Glutamylated Intermediates of
Escherichia coli K-12

Shin Kurihara, Shinpei Oda, Kenji Kato,
Hyeon Guk Kim, Takashi Koyanagi, Hidehiko
Kumagai and Hideyuki Suzuki
J. Biol. Chem. 2005, 280:4602-4608.
doi: 10.1074/jbc.M411114200 originally published online December 8, 2004

Access the most updated version of this article at doi: 10.1074/jbc.M411114200

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 27 references, 22 of which can be accessed free at
http://www.jbc.org/content/280/6/4602.full.html#ref-list-1
