
Curr Genet (2005) 47:156–162  
DOI 10.1007/s00294-004-0557-x  

---

**RESEARCH ARTICLE**

Yuko Kawasaki · Mari Onozuka · Tomoko Mizote  
Kazuto Nosaka  

**Biosynthesis of hydroxymethylpyrimidine pyrophosphate in *Saccharomyces cerevisiae***

Received: 13 July 2004 / Revised: 22 November 2004 / Accepted: 25 November 2004 / Published online: 22 December 2004  
© Springer-Verlag 2004  

**Abstract** Two redundant genes, *THI20* and *THI21*, of *Saccharomyces cerevisiae* encode a 2-methyl-4-amino-5-hydroxymethylpyrimidine monophosphate (HMP-P) kinase required for thiamin biosynthesis. Using functional complementation analysis with an *Escherichia coli* mutant strain and a defined biochemical system containing partially purified proteins for the reconstitution of thiamin monophosphate synthesis, we demonstrate that both Thi20p and Thi21p proteins also have HMP kinase activity. Although each isoform independently can synthesize HMP pyrophosphate (HMP-PP) from HMP, there is a marked difference in efficiency between the two proteins. The *thi20* deletion strain grows at the same rate as the parental strain in minimal medium without thiamin, but its ability to synthesize HMP-PP from HMP is significantly decreased. We discuss the possibility that HMP is not involved in the pathway of de novo thiamin synthesis in *S. cerevisiae*.

**Keywords** Hydroxymethylpyrimidine kinase · Thiamin biosynthesis · Bifunctional enzyme · *Saccharomyces cerevisiae*

---

**Introduction**

Thiamin biosynthesis in microorganisms involves the independent formation of two-ring structures, 2-methyl-4-amino-5-hydroxymethylpyrimidine (hydroxymethylpyrimidine; HMP) and 4-methyl-5-β-hydroxyethylthiazole (hydroxyethylthiazole; Th), and their subsequent condensation to form thiamin monophosphate (TMP; Fig. 1; for a review, see Begley 1996). The overall reaction from HMP and Th is catalyzed by four enzymes, HMP kinase (EC 2.7.1.49), HMP monophosphate kinase (HMP-P kinase; EC 2.7.4.7), Th kinase (EC 2.7.1.50) and TMP pyrophosphorylase (TMP-PPase; EC 2.5.1.3). TMP is then converted to thiamin pyrophosphate, a coenzyme form. In almost all prokaryotic cells thiamin pyrophosphate is directly synthesized by phosphorylation of TMP, whereas in eukaryotic cells and some bacterial cells this final step occurs in two stages: hydrolysis to free thiamin and its pyrophosphorylation by thiamin pyrophosphokinase (EC 2.7.6.2). We previously characterized some of the enzyme activities in Fig. 1 and identified the corresponding genes of *Saccharomyces cerevisiae* (Kawasaki et al. 1990; Nosaka et al. 1993, 1994). The *THI6* gene encodes a bifunctional enzyme consisting of a TMP-PPase domain and a Th kinase domain; and the *THI80* gene encodes a thiamin pyrophosphokinase. Expression of these structural genes is repressed by thiamin in growth medium and induced in the absence of thiamin. We also demonstrated that *THI2* and *THI3* gene products are required for expression of the structural genes in the thiamin regulatory system (Nishimura et al. 1992a, b), whereas thiamin pyrophosphate serves as a corepressor (Nishimura et al. 1991).

---

**Communicated by S. Hohmann**

Y. Kawasaki  
Department of Food Microbiology,  
Doshisha Women's College,  
Kamigyo-ku, Kyoto 602-0893, Japan  

M. Onozuka  
Department of Biochemistry,  
Kyoto Prefectural University of Medicine,  
Kamigyo-ku, Kyoto 602-8566, Japan  

T. Mizote  
Department of Food and Nutrition,  
Yamaguchi Prefectural University, Sakurabatake,  
Yamaguchi 753-8502, Japan  

K. Nosaka (✉)  
Department of Chemistry, Kyoto Prefectural University of Medicine,  
Kita-ku, Kyoto 603-8334, Japan  
E-mail: knosaka@koto.kpu-m.ac.jp  
Tel.: +81-75-4657660  
Fax: +81-75-4657651

thiamin biosynthetic pathway. In this paper, we describe the successful reconstitution of TMP biosynthesis from Th and HMP in a defined biochemical system containing Thi6p with Thi20p or Thi21p; and we demonstrate that Thi20p has a greater ability than Thi21p to synthesize HMP-PP from HMP.

---

### Materials and methods

#### Organisms and media

An *E. coli* strain, DH5α, was used to amplify plasmids and express the recombinant proteins. Another *E. coli* strain, SN372L-3-9-10 (*thiC thiD pdxB20 pdxL2 argG6 metB1 his-1 leu-6 trp-31 rpsL104; Mizote et al. 1999), was used for functional complementation analysis. A yeast strain, YPH500 (*MATa/ura3-52 his3-Δ200 leu2-Δ1 trp1-Δ63 ade2-101 lys2-801*), was used as the parental strain for gene disruption. Single deletions of *THI20* and *THI21* were made by a PCR-directed integration method (Baudin et al. 1993). The primers used in this procedure (Table 1) contain two distinct regions, one that allows homologous recombination at the target locus and a second part that permits the PCR amplification of a selectable marker (*HIS3*). A double-deletion strain was constructed successively by conventional insertion of a *TRP1* gene within the *THI21* coding region. All constructed strains were confirmed to be correctly disrupted by restriction mapping of PCR-isolated fragments from the genomic DNAs. The standard media and growth conditions for the bacterial and yeast cells were as described by Nosaka et al. (1993, 1994).

#### Plasmids and chemicals

The intact coding sequences of *THI6*, *THI20*, *THI21*, *THI22* and their N-terminal fragments (1-302) were PCR-amplified from yeast genomic DNA by the primers listed in Table 1. Bacterial expression vectors coding for the yeast recombinant proteins with a myc epitope and a six-histidine tag at the C-terminus were made by inserting the PCR products into a TA-cloning site in pTrcHis2-TOPO, using TOPO TA cloning kits (Invitrogen, Calif., USA). To express the proteins with a V5 epitope at the C-terminus from the *GAL1* promoter in yeast cells, the amplified fragments were digested with *HindIII* and *Xhol* (these sites were designed into the primers) and subcloned into a multicopy 2 μ plasmid pYES2/NT B (Invitrogen). All the plasmids were verified to be constructed correctly by sequencing.

HMP-P was synthesized as reported by Kawasaki et al. (1990). HMP was a kind gift from the late S. Yurugi (Takeda Pharmaceutical Industries, Osaka, Japan) and Th was from Sigma–Aldrich (Mo., USA). All other chemicals were of analytical grade.

Table 1 Oligonucleotides used in this study. Nucleotide positions are given relative to the translation initiation codon of the targeted gene. The sequences designed for HIS3 gene amplification are CTCTTGGGCCCTCCTTAG (HIS3F) and TCGTTCAGAATGA-CACG (HIS3R). Nucleotides in italics indicate the restriction site for HindIII or XhoI.

| Gene | Nucleotide position | Sequence |
|------|---------------------|----------|
| **Oligonucleotides used to construct deletion strain** |  |  |
| THI20 | -50 to -1 | ACTAGACAGATTAATAAAATAATATAAG GAAAACGTCTCGCATTTTACT-(HIS3F) |
|  | 1,705-1,656 | TTCATTATATGTAGGTATATCTTTTATCTATT GTGCATAAGCCATCTTATC-(HIS3R) |
| THI21 | -50 to -1 | ATCTGATCACGTTAGGAAATATACATAAGG AAAGCAACTGTCTTGCCCACA-(HIS3F) |
|  | 1,700-1,651 | TACGTATAACTTTTGTATATTTTGTAAAAA TCGATGTGTCTATTCATTC-(HIS3R) |
| **Oligonucleotides used to clone into the bacterial expression vector** |  |  |
| THI20 | 1-21 | ATGACCTATTCTACAGTTAGC |
|  | 906-889 | CTCCAATGGAATTTCCAC |
|  | 1,653-1,637 | CTCGTATTCTAGCGCAG |
| THI21 | 1-24 | ATGACCTATTCTACAGTTAATATT |
|  | 906-887 | TTCTAATGGAATTTTCGATGG |
|  | 1,653-1,636 | TTCATATTCTAGAGCGGGC |
| THI22 | 1-20 | ATGGTTATCATCCTCCTAGG |
|  | 1,716-1,670 | TTCGTATTCCATGGCGG |
| THI6 | 1-22 | ATGGTATTTTACTAAGGAAGAAG |
|  | 1,620-1,599 | TTTGAATTTCTTCAAAGAAGCG |
| **Oligonucleotides used to clone into the yeast expression vector** |  |  |
| THI20 | 1-21 | CCAAGCTTATGACCTATTCTACAGTTAGC |
|  | 1,653-1,637 | CGCTCGAGGCTCGTATTCTAGCGCAG |
| THI21 | 1-24 | CCAAGCTTATGACCTATTCTACAGTTAATATT |
|  | 1,653-1,636 | CGCTCGAGTTCATATTCTAGAGCGGGC |
| THI22 | 1-22 | CCAAGCTTATGGTTATCATCCTCCTAGGTC |
|  | 1,716-1,699 | CGCTCGAGTTTCGTATTCCATGGCGGGC |

Expression and partial purification of recombinant proteins

The E.coli strain DH5α bearing the pTrcHis 2-TOPO plasmid was grown overnight at 37°C in 1 ml of LB medium containing 50 μg/ml ampicillin, pelleted and grown for 2 h in 10 ml of fresh medium. The cells were further shaken at 28°C for 2 h with 0.1 mM isopropyl-β-D-thiogalactopyranoside to induce the recombinant protein. The cells were then pelleted, resuspended in 0.5 ml of lysis buffer (50 mM Na₂HPO₄, pH 8.0, 300 mM NaCl) containing 1 mM phenylmethylsulfonyl fluoride and 20 μg/ml pepstatin A and sonicated four times in ice-cold water, using a Bioruptor (Cosmo Bio, Tokyo, Japan) at 200 W for 30 s each time, with intervals of 120 s. Cell debris was removed by centrifugation at 14,000 g for 20 min and the supernatant was served as a crude extract. The histidine-tagged proteins were purified from the crude extracts by magnetic nickel-agarose beads (MagExtractor-His tag; Toyobo, Osaka, Japan), according to the manufacturer's instructions.

contained 50 mM Tris-HCl, pH 7.5, 10 μM HMP or HMP-P, 10 μM Th, 10 mM ATP, 20 mM MgCl₂, with a combination of partially purified Thi6p and one (2 μg) of Thi20p, Thi21p, Thi20p(1-302) or Thi21p(1-302) in a final volume of 1.0 ml. After incubation at 37°C for the indicated time, the reaction was stopped by 0.5 ml of 10% metaphosphoric acid and centrifuged at 2,000 g for 10 min to remove denatured protein. After the pH of the supernatant was adjusted to 4.5 with 4 M sodium acetate, TMP was hydrolyzed to free thiamin by incubation for 1 h at 45°C with 0.1 ml of 1% diastase (Wako Pure Chemicals, Osaka, Japan). Thiamin was determined fluorometrically by the thiochrome method (Kawasaki et al., 1990). When the HMP-PP synthesis ability of the deletion strains was determined, the yeast crude extracts were prepared as described by Kawasaki et al. (1990) and 2 mg of each were added to the reaction mixture instead of the recombinant proteins.

Western blotting analysis

The yeast NKCl1 (Δthi20 Δthi21) cells harboring pYES2/NTB derivatives were grown in 25 ml of minimal medium with 20 nM thiamin and 2% raffinose to an optical density at 540 nm (OD₅₄₀) of 0.8. Then, they were pelleted and grown for 16 h in 100 ml of the same medium, but with 2% galactose instead of raffinose. The

Enzyme assay

The ability to synthesize HMP-PP from HMP or HMP-P was assessed by coupling HMP-PP and Th-P in a defined biochemical system. The reaction mixture
yeast crude extracts were first separated on 10% SDS-PAGE gels and then transferred to a polyvinylidene difluoride membrane. The recombinant Thi proteins were visualized using an anti-V5 antibody (Invitrogen), a biotinylated anti-mouse IgG (Vector) and an amplified alkaline phosphatase Immun-Blot assay kit (Bio-Rad, Calif., USA).

---

### Results

In bacterial cells, HMP kinase is involved in the salvage synthesis of HMP-P from HMP in the thiamin biosynthesis pathway (Nakayama 1990); and the activity is generated by not only *thiD* protein but also pyridoxal kinase (*pdxL*; Fig. 1; Mizote and Nakayama 1989). In contrast, *thiC* is identified as the gene involved in the de novo synthesis of HMP-P; and the *thiC* mutation leads to a requirement for exogenous HMP or thiamin (Vander Horn et al. 1993). The *E. coli* mutant strain SN372L-3-9-10 carries *thiC*, *thiD* and *pdxL* mutations simultaneously and therefore requires thiamin for growth (Mizote et al. 1999). To investigate the possibility that yeast Thi20p and Thi21p have HMP kinase activity in addition to HMP-P kinase activity, we performed heterologous functional complementation of the *THI20* and *THI21* genes in SN372L-3-9-10. The intact coding sequences of *THI20* and *THI21* were cloned into pTrcHis2-TOPO and used to transform the strain. Each of the bacterial cells was inoculated into a liquid minimal medium lacking thiamin, but containing HMP. After culturing the cells through two rounds of 10²-fold dilution in the same liquid medium, each 50 μl of the culture was spotted onto the same agar medium. As shown in Fig. 2, the triple mutant SN372L-3-9-10 transformed with the vector alone did not grow on the medium lacking thiamin but containing HMP, whereas both the pTrcHis2-THI20 vector and the pTrcHis2-THI21 vector restored the thiamin prototrophy of the mutant. When HMP was withdrawn from the medium, none of the cells bearing either of these two vectors grew (data not shown). This result indicated that expression of either *THI20* or *THI21* alone from the vector is sufficient to synthesize HMP-PP from HMP in these bacterial cells.

Given the fact that the *THI20* and *THI21* genes are able to restore thiamin prototrophy of SN372L-3-9-10, the biosynthesis of HMP-PP from HMP by Thi20p and Thi21p was examined by its coupling with Th-P in a defined biochemical system containing Th, Thi6p, a bifunctional TMP-PPase and Th kinase (Fig. 1). Thi20p, Thi21p and Thi6p were separately overexpressed with a histidine-tag at the C-terminus in *E. coli* and partially purified. Since the N-terminal parts of Thi20p and Thi21p are reported to be sufficient to express HMP-P kinase (Llorente et al. 1999), we also prepared Thi20p(1-302) and Thi21p(1-302). These proteins were expressed as soluble proteins with the expected size in the bacterial cells; and about 50 μg each of the partially purified protein was obtained from 10 ml of culture (Fig. 3). The reaction mixture contained HMP or HMP-P, Th, ATP, MgCl₂, Thi6p and one of Thi20p, Thi21p, Thi20p(1-302) or Thi21p(1-302); and the synthesized TMP was determined as described in the Materials and methods. As shown in Fig. 4, 43% and 11% conversion by Thi20p and Thi21p, respectively, were detected (based on 10 μM HMP-P, after 2 h of incubation). We also observed 26% conversion after 2 h from 10 μM HMP by Thi20p. In contrast, only 1% conversion was observed from the same concentration of HMP when Thi21p was supplied instead of Thi20p. The TMP synthesis was not detected when Thi20p or Thi21p was omitted

![Figure 2](https://via.placeholder.com/500x500.png?text=Fig.+2+Heterologous+functional+complementation+of+the+E.+coli+SN372L-3-9-10+strain.)

**Fig. 2** Heterologous functional complementation of the *E. coli* SN372L-3-9-10 strain. The cells transformed with vector alone or with the recombinant plasmid were cultured and spotted onto agarose plates containing 1 nM HMP with or without 1 nM thiamin, as described in the text. Plates were incubated at 37°C for 2 days

![Figure 3](https://via.placeholder.com/500x500.png?text=Fig.+3+SDS-PAGE+analysis+of+the+histidine-tagged+proteins+used)

**Fig. 3** SDS-PAGE analysis of the histidine-tagged proteins used for the reconstitution of TMP synthesis. Crude extracts (20 μg of protein) from pTrcHis2-THI20 transformant grown in uninduced (lane 2) and induced (lane 3) cultures and purified fractions (4 μg of protein) of histidine-tagged Thi20p (lane 4), Thi21p (lane 5), Thi20p(1-302) (lane 6), Thi21p(1-302) (lane 7) and Thi6p (lane 8) were analyzed using 10% SDS-PAGE, followed by Coomassie blue staining. Lane 1 shows molecular standard proteins (Life Technologies)
However, Thi20p was more efficient than Thi21p in HMP-PP synthesis catalysis, in particular when HMP was used as the substrate. An unexpected result was that Thi20p(1-302) and Thi21p(1-302) failed to synthesize TMP from HMP-P in the defined system, although the C-terminal part of both proteins appears not to be required for HMP-P kinase activity (Llorente et al. 1999). There are a number of possibilities to explain this apparent inconsistency, including a loss of proper conformation by the replacement of the C-terminal domain with the histidine tag. We assume that the N-terminal domains of Thi20p and Thi21p are responsible for HMP-PP synthesis from HMP under physiological conditions.

Although each isoform of Thi20p and Thi21p can independently synthesize HMP-PP from HMP and they are functionally redundant, there is a marked difference in the catalytic efficiency of the two proteins. We therefore tested the growth of single- and double-deletion strains for THI20 and THI21 genes in a synthetic medium with and without 20 nM thiamin (Fig. 5) and subsequently investigated their HMP-PP synthesis ability from HMP using their crude extracts (Table 2). The

Table 2 TMP synthesis activity from HMP and Th. Activity was determined as described in the Materials and methods, using the crude extract from yeast cells grown in a minimal medium containing 20 nM thiamin, which does not cause thiamin repression. The NKC11 strain transformed with pYES2/NTB or its derivatives was cultured in medium containing 2% galactose as described in the Materials and methods. Each value is the mean from two experiments

| Strain           | Plasmid         | Activity (pmol mg⁻¹ min⁻¹) |
|------------------|-----------------|-----------------------------|
| YPH 500 (wild)   | None            | 19.0                        |
| NKC 9 (∆thi20)   | None            | 4.7                         |
| NKC 10 (∆thi21)  | None            | 20.7                        |
| NKC 11 (∆thi20 ∆thi21) | None          | Not detected                |
|                  | pYES2/NTB       | Not detected                |
|                  | pYES2/NTB-THI20 | 192.5                       |
|                  | pYES2/NTB-THI21 | 15.8                        |
|                  | pYES2/NTB-THI22 | Not detected                |

Fig. 4 Reconstitution of TMP synthesis from HMP (open symbols) or HMP-P (closed symbols) with Th by recombinant proteins. The reaction mixture and the assay procedure are described in the Materials and methods. The amounts of TMP in the reaction mixture (1 ml) are shown. Each point is the mean from two experiments. Symbols indicate the following combination of proteins: diamonds Thi20p and Thi6p, circles Thi21p and Thi6p, squares Thi20p(1-302) and Thi6p, triangles Thi21p(1-302) and Thi6p, inverted triangles Thi20p, Thi21p or Thi6p alone

from the reaction mixture. From these results, we conclude that both Thi20p and Thi21p have a HMP kinase activity together with a HMP-P kinase activity.

Fig. 5 Growth of single- or double-deletion strains for THI20 and THI21 genes. After yeast cells were grown in a liquid rich medium to the stationary phase and subsequently cultured through two rounds of 50-fold dilution in a synthetic medium without thiamin, the cell suspensions were washed and diluted to OD₅₄₀ = 0.1 in fresh medium without thiamin (a) or with 20 nM thiamin (b). At the indicated time-points, the OD₅₄₀ of the culture was determined. Values are means from two experiments. Symbols indicate the following strains: diamonds YPH 500 (parental strain), circles NKC9 (∆thi20), squares NKC10 (∆thi21), triangles NKC11 (∆thi20 ∆thi21)

A thiamin (–)
![Graph A](https://via.placeholder.com/300x200)

B thiamin (+)
![Graph B](https://via.placeholder.com/300x200)
deletion strain for both genes (NKC11) showed no TMP synthesis activity, which is in accordance with the observation of thiamin auxotrophy in the strain. Llorente et al. (1999) demonstrated that a deficiency in HMP-PP synthesis by simultaneous disruption of the THI20 and THI21 genes is responsible for thiamin auxotrophy. While the TMP synthesis activity from HMP and Th decreased significantly in the thi20 deletion strain (NKC9), the thi21 deletion strain (NKC10) expressed almost the same activity as the parental strain (YPH500). However, no detectable difference in the growth curves of YPH500, NKC9, and NKC10 was observed, regardless of whether thiamin was present or absent in the medium. Likewise, the addition of HMP to the medium did not give rise to any difference in their growth (data not shown). These results suggest that an adequate level of HMP-PP was synthesized, even in the thi20 deletion strain (NKC9).

All genetic complementation attempts using bacterial or yeast cells to show a HMP-P kinase activity from the third member of the THI20 gene family, THI22, have failed (Llorente et al. 1999). In addition, we could detect no TMP synthesis activity from Th with HMP (Table 2) or HMP-P in the crude extracts of the deletion strain for both THI20 and THI21 genes (NKC11); and in these cells significant expression of THI22 mRNA was verified by RT-PCR analysis (data not shown). It was interesting to examine the activity of Thi22p in the defined biochemical system, as performed with Thi20p and Thi21p. However, although Thi22p with a myc epitope and a six-histidine tag at the C-terminus was detected in the crude soluble extracts with certainty by Western analysis using an anti-myc antibody (data not shown), the recombinant protein could not bind to magnetic nickel-agarose beads. The histidine tag fused to Thi22p, but not the tag fused to Thi20p or Thi21p, seemed to be inaccessible to the beads for some unexplained reason. We next tried to produce Thi20p, Thi21p, and Thi22p separately, with a V5 epitope at the C-terminus, in yeast NKC11 cells. Employing the multiple-copy 2 μ plasmid and the yeast GAL1 promoter, we found by Western analysis using an anti-V5 antibody that these recombinant proteins were overexpressed (Fig. 6). In accordance with the fact that Thi22p possesses a putative signal peptide of 22 aa at its N-terminus, the recombinant Thi22p was detected in both truncated and untruncated forms. Nevertheless, TMP synthesis activity was not detected in the crude extracts (Table 2). In contrast, overexpressing the THI20 gene exerted a marked TMP synthesis activity. From these results, we supposed that Thi22p itself has no ability to synthesize HMP-PP from HMP-P or HMP. This suggestion is compatible with the article by Llorente et al. (1999), in which the cleavage of the N-terminal signal sequence from Thi22p does not complement the thiamin auxotrophy of the deletion strain for both THI20 and THI21 genes. An understanding of the physiological function of Thi22p awaits more detailed analysis.

---

Fig. 6 The recombinant proteins expressed in yeast NKC11 (∆thi20 ∆thi21) cells. Crude extracts (1 mg of protein) from pYES2/NT (lane 2), pYES2/NT-THI20 (lane 3), pYES2/NT-THI21 (lane 4), and pYES2/NT-THI22 (lane 5) transformants were analyzed by Western blotting, as described in the Materials and methods. Lane 1 shows biotinylated molecular standard proteins (Bio-Rad). The arrow indicates the recombinant Thi proteins and the asterisk marks the position of the untruncated Thi22p

---

### Discussion

We demonstrate in this study that both Thi20p and Thi21p proteins have a HMP kinase activity in addition to a HMP-P kinase activity. Biochemical analysis using recombinant proteins or crude extracts from the deletion strains indicates that Thi20p has a higher ability than Thi21p to synthesize HMP-PP from HMP. Moreover, microarray-based expression analysis reveals that the THI20 gene is about 2.5-fold more strongly expressed than THI21 when thiamin is withdrawn from the medium (K. Nosaka, unpublished data). It appears likely from these facts that the HMP-PP synthesis from HMP in yeast cells is mainly catalyzed by Thi20p. However, the thi20-deletion strain grew at the same rate as the parental strain and the thi21-deletion strain in minimal medium without thiamin. It is noted that the difference in the efficiency of HMP-PP synthesis from HMP-P between Thi20p and Thi21p is not as significant as that from HMP in the defined biochemical system. We therefore assume that, as in *E. coli*, HMP is not an obligatory intermediate in the pathway of de novo thiamin synthesis and that HMP kinase is involved in the salvage synthesis of HMP-P from HMP in *Saccharomyces cerevisiae*. However, the possibility that the residual HMP kinase activity in the thi20-deletion strain is sufficient to supply HMP-P cannot be ruled out.

It is interesting that TMP synthesis from Th and HMP in *S. cerevisiae* consists of two bifunctional enzymes,
Thi6p and Thi20p (or Thi21p). The products of the first enzymatic reaction of both enzymes could be effectively utilized for the second reaction without diffusion. However, Thi6p is the fusion of a TMP-PPase and a Th kinase domain, whereas the HMP kinase and HMP-P kinase activities of Thi20p and Thi21p seem to be derived from a single domain. We therefore expect that the C-terminal parts of Thi20p and Thi21p also catalyze the HMP-P synthesis reaction, probably from a de novo precursor. These sequences are similar to the *S. cerevisiae* PET18 gene product over its full length (about 20% amino acid identity), whose function is unknown but whose expression is induced in the absence of thiamin. Since a disruption in the *PET18* gene does not confer thiamin auxotrophy (K. Nosaka, unpublished data), any of three fusion proteins might compensate for the function of Pet18p. It has been demonstrated that histidine and pyridoxine are the advanced precursors of the pyrimidine moiety of thiamin under aerobic conditions in yeast (Tanaka et al. 2000; Tazuya et al. 1989, 1993). While none of the enzymatic activities have yet been identified, three kinds of gene families (*SNO2/SNO3, SNZ2/SNZ3, THI5/THI11/THI12/THI13*) are reported to be involved in the pathway of the pyrimidine moiety synthesis via pyridoxine (Wightman and Meacock 2003; Rodriguez-Navarro et al. 2002). Wightman and Meacock (2003) noticed that any of the enzymes required for HMP-PP synthesis in the yeast thiamin synthesis pathway are encoded by multigene families (*SNO2, SNZ2, THI5, THI20*). A combination of genetic, enzymological and chemical approaches is needed to clarify the function of these amplified enzymes, including Pet18p, and to establish the intriguing intermediates of the synthetic pathway to HMP-P.

Acknowledgements This work was supported in part by Grant-in-Aid for Exploratory Research 16650187 from the Japan Society for the Promotion of Science and by Individual Research Grants of the Doshisha Women's College of Liberal Arts.

---

**References**

Baudin A, Ozier-Kalogeropoulos O, Denouel A, Lacroute F, Cullin C (1993) A simple and efficient method for direct gene deletion in *Saccharomyces cerevisiae*. Nucleic Acids Res 21:3329–3330

Begley TP (1996) The biosynthesis and degradation of thiamin (vitamin B₁). Nat Prod Rep 13:177–185

Cheng G, Bennett EM, Begley TP, Ealick SE (2002) Crystal structure of 4-amino-5-hydroxymethyl-2-methylpyrimidine phosphate kinase from *Salmonella typhimurium* at 2.3 Å resolution. Structure 10:225–235

Kawasaki Y, Nosaka K, Kaneko Y, Nishimura H, Iwashima A (1990) Regulation of thiamine biosynthesis in *Saccharomyces cerevisiae*. J Bacteriol 172:6145–6147

Kawasaki Y, Onozuka M, Nishimura H, Nosaka K (1999) Analyses of novel genes involved in yeast thiamin biosynthesis (in Japanese). Vitamins 73:263

Llorente B, Fairhead C, Dujon B (1999) Genetic redundancy and gene fusion in the genome of the baker’s yeast *Saccharomyces cerevisiae*: functional characterization of a three-member gene family involved in the thiamine biosynthetic pathway. Mol Microbiol 32:1140–1152

Mizote T, Nakayama H (1989) Purification and properties of hydroxymethylpyrimidine kinase from *Escherichia coli*. Biochim Biophys Acta 991:109–113

Mizote T, Tsuda M, Smith DDS, Nakayama H, Nakazawa T (1999) Cloning and characterization of the *thiD/J* gene of *Escherichia coli* encoding a thiamine-synthesizing bifunctional enzyme, hydroxymethylpyrimidine kinase/phosphomethylpyrimidine kinase. Microbiology 145:495–501

Nakayama H (1990) Genetic analyses of thiamin pyrophosphate biosynthesis in *Escherichia coli* (in Japanese). Vitamins 64:619–632

Nishimura H, Kawasaki Y, Nosaka K, Kaneko Y, Iwashima A (1991) A constitutive thiamine metabolism mutation, *thi80*, causing reduced thiamine pyrophosphokinase activity in *Saccharomyces cerevisiae*. J Bacteriol 173:2716–2719

Nishimura H, Kawasaki Y, Kaneko Y, Nosaka K, Iwashima A (1992a) Cloning and characterization of a positive regulatory gene, *THI2(PHO6)*, of thiamin biosynthesis in *Saccharomyces cerevisiae*. FEBS Lett 297:155–158

Nishimura H, Kawasaki Y, Kaneko Y, Nosaka K, Iwashima A (1992b) A positive regulatory gene, *THI3*, is required for thiamine metabolism in *Saccharomyces cerevisiae*. J Bacteriol 174:4701–4706

Nosaka K, Kaneko Y, Nishimura H, Iwashima A (1993) Isolation and characterization of a thiamin pyrophosphokinase gene, *THI80*, from *Saccharomyces cerevisiae*. J Biol Chem 268:17440–17447

Nosaka K, Nishimura H, Kawasaki Y, Tsujihara T, Iwashima A (1994) Isolation and characterization of the *THI6* gene encoding a bifunctional thiamin-phosphate pyrophosphorylase/hydroxyethylthiazole kinase from *Saccharomyces cerevisiae*. J Biol Chem 269:30510–30516

Petersen LA, Downs DM (1997) Identification and characterization of an operon in *Salmonella typhimurium* involved in thiamine biosynthesis. J Bacteriol 179:4894–4900

Reddick JJ, Kinsland C, Nicewonger R, Christian T, Downs DM, Winkler ME, Begley TP (1998) Overexpression, purification and characterization of two pyrimidine kinases involved in the biosynthesis of thiamine: 4-amino-5-hydroxymethyl-2-methylpyrimidine kinase and 4-amino-5-hydroxymethyl-2-methylpyrimidine phosphate kinase. Tetrahedron 54:15983–15991

Rodriguez-Navarro S, Llorente B, Rodriguez-Manzaneque MT, Ramne A, Uber G, Marchesan D, Dujon B, Herrero E, Sunnerhagen P, Perez-Ortin JE (2002) Functional analysis of yeast gene families involved in metabolism of vitamins B1 and B6. Yeast 19:1261–1276

Tanaka K, Tazuya K, Yamada K, Kumaoka H (2000) Biosynthesis of thiamin under anaerobic conditions in *Saccharomyces cerevisiae*. Biol Pharm Bull 23:108–111

Tazuya K, Yamada K, Kumaoka H (1989) Incorporation of histidine into the pyrimidine moiety of thiamin in *Saccharomyces cerevisiae*. Biochim Biophys Acta 990:73–79

Tazuya K, Yamada K, Kumaoka H (1993) Pyridoxine is a precursor of the pyrimidine moiety of thiamin in *Saccharomyces cerevisiae*. Biochem Mol Biol Int 30:893–899

Vander Horn PB, Backstrom AD, Stewart V, Begley TP (1993) Structural genes for thiamine biosynthetic enzymes (*thiCEFGH*) in *Escherichia coli* K-12. J Bacteriol 175:982–992

Wightman R, Meacock PA (2003) The *THI5* gene family of *Saccharomyces cerevisiae*: distribution of homologues among the hemiascomycetes and functional redundancy in the aerobic biosynthesis of thiamin from pyridoxine. Microbiology 149:1447–1460
