
Molecular Cell, Vol. 17, 121–135, January 7, 2005, Copyright ©2005 by Elsevier Inc.        DOI 10.1016/j.molcel.2004.11.041

# The Structure of the Human Adenovirus 2 Penton

Chloe Zubieta,$^{1}$ Guy Schoehn,$^{1,2,3}$ Jadwiga Chroboczek,$^{2}$ and Stephen Cusack$^{1,*}$

$^{1}$European Molecular Biology Laboratory  
Grenoble Outstation, BP 181  
38042 Grenoble Cedex 9  
France  

$^{2}$Institut de Biologie Structurale J.P. Ebel  
38027 Grenoble Cedex 1  
France  

$^{3}$Laboratoire de Virologie Moléculaire  
EA 2939  
Université Joseph Fourier  
Grenoble Cedex 9  
France  

## Summary

The adenovirus penton, a noncovalent complex of the pentameric penton base and trimeric fiber proteins, comprises the vertices of the adenovirus capsid and contains all necessary components for viral attachment and internalization. The 3.3 Å resolution crystal structure of human adenovirus 2 (hAd2) penton base shows that the monomer has a basal jellyroll domain and a distal irregular domain formed by two long insertions, a similar topology to the adenovirus hexon. The Arg-Gly-Asp (RGD) motif, required for interactions with cellular integrins, occurs on a flexible surface loop. The complex of penton base with bound N-terminal fiber peptide, determined at 3.5 Å resolution, shows that the universal fiber motif FNPVYPY binds at the interface of adjacent penton base monomers and results in a localized structural rearrangement in the insertion domain of the penton base. These results give insight into the structure and assembly of the adenovirus capsid and will be of use for gene-therapy applications.

## Introduction

Human adenoviruses are nonenveloped DNA viruses that can cause respiratory, ocular, and enteric disease. The approximately fifty known human serotypes are divided amongst six subgroups (A–F) based on such factors as crossimmunoreactivity, agglutination, nucleic acid homology, and genetic organization (Bailey and Mautner, 1994). Adenoviruses have also been isolated from many other mammals, birds, reptiles, amphibians, and fish (Benko and Harrach, 2003). All adenoviruses exhibit the same overall capsid architecture. The T = 25 icosahedral capsid is approximately 1000 Å in diameter and consists primarily of three different polypeptides, hexon, penton base, and fiber (van Oostrum and Burnett, 1985). Each facet of the capsid contains 12 copies of the trimeric hexon protein, a major viral antigen. A noncovalent complex of two proteins, the pentameric penton base (polypeptide III) and the externally projecting trimeric fiber protein (polypeptide IV), forms the 12 vertices of the particle. Additionally, a number of proteins located on the internal or external surface of the capsid, such as IIIa, VI, VIII, and IX, have been implicated in cementing the hexon-hexon and hexon-penton interactions, (Furcinitti et al., 1989; Stewart et al., 1991; Stewart et al., 1993).

The capsid proteins fiber and penton base are the major players in adenovirus cell entry. Initial attachment of the virus occurs through the binding of the distal C-terminal knob domain of the fiber to primary cellular receptors. Coxsackie and adenovirus receptor (CAR), an integral membrane protein now known to be part of the tight junction in epithelial cells (Cohen et al., 2001; Walters et al., 2002), is the primary receptor for adenovirus subgroups A and C–F (Bergelson et al., 1997; Tomko et al., 1997; Roelvink et al., 1998). Recently, CD46 has been identified as the receptor for a number of subgroup B serotypes such as Ad3, Ad11, and Ad33, as well as Ad37 of subgroup D (Gaggar et al., 2003; Segerman et al., 2003; Sirena et al., 2004; Wu et al., 2004). Additionally, in Ad37, sialic acid may also play a role as a receptor (Burmeister et al., 2004).

After initial cell binding, α<sub>v</sub>β<sub>3</sub> or α<sub>v</sub>β<sub>5</sub> integrins serve as coreceptors by binding to a highly conserved RGD motif in the penton base, an interaction that is essential for virus internalization by receptor-mediated endocytosis (Wickham et al., 1993). Clustering of integrins induced by this interaction activates signaling pathways that promote rapid internalization into clathrin-coated pits and endosomes (Li et al., 1998). Integrin binding is accompanied by fiber shedding from the capsid (Greber et al., 1993; Nakano et al., 2000). Both penton base and fiber, as well as adenovirus protease, have been implicated in the escape of the virus from the endosome as the pH lowers, although the details of this process are unclear (Seth et al., 1985; Blumenthal et al., 1986; Nemerow and Stewart, 1999; Medina-Kauwe et al., 2001; Medina-Kauwe, 2003). After internalization, partially disassembled capsids are transported along microtubules to the perinuclear envelope where further disintegration of the capsid finally allows import of the viral DNA into the nucleus accompanied by protein VI. The critical importance of fiber and penton base in cell infection is shown by the fact that dodecahedral particles comprising only these two viral proteins efficiently enter cells and accumulate at the periphery of the nucleus, presumably by using the same pathway as the complete virus (Fender et al., 1997). Unlike all other human adenoviruses, subgroup F enteric adenoviruses Ad40 and Ad41 neither have an RGD motif nor use integrins for cell infection (Albinsson and Kidd, 1999).

Structural studies of intact adenoviruses, individual capsid proteins, and complexes with receptors by X-ray crystallography and electron microscopy (EM) have provided a detailed understanding of many aspects of adenovirus capsid structure, assembly, and virus-host cell interactions. Cryo-electron microscopy (cryo-EM) three-dimensional (3 D) reconstructions of the entire adenovi-


Molecular Cell
122

| Ad2 penton base | MQR AAMYE---------EGPPPSYESVVSAAAPVAAALGSPFDA PLDPFVPPRYLRPTGGRNSIRYSELA PLFDTTRRVYLVDNKSTDV | 79 |
| --- | --- | --- |
| Ad3 penton base | MRRRAVLGGAVVYPEGPPPSYESVMQQQ--AAMI----QPPLEAPFVPPRYLAPTEGRNSIRYSDVSPLYDTTKLYLV DNKSA DI | 79 |
| Ad7 penton base | MRRRAVLGGAMVYPEGPPPSYESVMQQQ--AAMI----QPPLEAPFVPPRYLAPTEGRNSIRYSELSPLYDTTKLYLV DNKSA DI | 79 |
| Ad11 penton base | -MRRVVLGGAVVYPEGPPPSYESVMQQQATAVM----QSPLEAPFVPPRYLAPTEGRNSIRYSELA PQYDTT RLYLV DNKSA DI | 80 |
| Ad41 penton base | MRAVGVPVVMAYAEGPPPSYESVMGSADSPATL--------EALYVPPRYLG PTEGRNSIRYSELA PLYDTTRRVYLVDNKSA DI | 77 |
| Ad12 penton base | -MRAVELQTVAFPETPPPSYETVMAAA-----------------PPYVPPRYLG PTEGRNSIRYSELSPLYDTTRVYLVDNKSSDI | 69 |
| Ad17 penton base | -MRR-------AVVSSSPPPSYESVMAQA---------------TLEVPFVPPRYMAPTEGRNSIRYSELA PLYDTTRRVYLVDNKSA DI | 66 |
| Ad25 penton base | MMRR--------AYPEGPPPSYESVMQAMA AAM----QPPLEAPYVPPRYLAPTEGRNSIRYSELA PLYDTT RLYLV DNKSA DI | 74 |
| Ad37 penton base | -MRR-------AVVSSSPPPSYESVMAQA---------------TLEVPFVPPRYMAPTEGRNSIRYSELA PLYDTTRRVYLVDNKSA DI | 66 |
| * | ********* ** ********** * ** **** ********** * |  |

---

| C | α2 | B2 | D | B3 | B4 |  |
| --- | --- | --- | --- | --- | --- | --- |
| ASLNYQNDHSNFLTTVIQNNDYSPGEASTQTINLDDRSHWGGDLKTILHTNMPNVNEFMFTNKFKARVMVSRSLTK--------------DKQVE LKYEWVEFTLPEGN | 174 |
| ASLNYQNDHSNFLTTVVQNNDFTPTEASTQTINFDERSRWGGQLKTIMHTNMPNVNEYMF SNKFKARVMVSRKAPEGVT VNDTYDHKEDILKYEWFEFTLPEGN | 183 |
| ASLNYQNDHSNFLTTVVQNNDFTPTEASTQTINFDERSRWGGQLKTIMHTNMPNVNEYMF SNKFKARVMVSRKAPEGVIV NDTYDHKEDILKYEWFEFTLPEGN | 183 |
| ASLNYQNDHSNFLTTVVQNNDFTPTEASTQTINFDERSRWGGQLKTIMHTNMPNVNEYMF SNKFKARVMVSRKPPDGA AVGDTYDHKQDILKYEWFEFTLPEGN | 184 |
| ASLNYQNDHSNFQTTVVQNNDFTPTEASTQTINFDERSRWGGDLKTILRTNMPNINEFMS TNKFKARLMVEKK--------------NKETGLP RYEWFEFTLPEGN | 170 |
| ASLNYQNDHSNFLTTVVQNNDYSPIEAGQTINFDERSRWGGDLKTILHTNMPNVNDFMFT TKF KARVMVARKTNN--------------EGQTILE YEWAEFVLPEGN | 164 |
| ASLNYQNDHSNFLTTVVQNNDFTPAAEASTQTINFDERSRWGGDLKTILHTNMPNVNEYM FTSKFKARVMVARKHPQGV EATD---LSKDILEYEWEFTLPEGN | 167 |
| ASLNYQNDHSNFLTTVVQNNDFTPTEASTQTINFDERSRWGGQLKTIMHTNMPNVNEFMY SNKFKARVMVSRKTPNGVT VTEDYDGSQDELKYEWVEFELPEGN | 178 |
| ASLNYQNDHSNFLTTVVQNNDFTPAAEASTQTINFDERSRWGGDLKTILHTNMPNVNEYM FTSKFKARVMVARKKAEGAD AND---RSKDILEYQWFEFTLPEGN | 167 |
| ************** *** ***** * ** ******* * ****** * ** ******* ** * * ** ******* * |  |

---

| α3 | B5 | B6 | B7 | α4 | B8 |  |
| --- | --- | --- | --- | --- | --- | --- |
| YSETMTIDLMNNAIVEHYLKVRQNGVLESDIGVKFDTRNFLRGFDPVTGLVMPGVYTNEAFHPDIILLPGCGVDFTHSRLSNLLGIRKR QPFQEGFRITYDDL | 278 |
| FSATMTIDLMNNAIIDNYLEIGRQNGVLESDIGVKFDTRNFLGWDPETKLIMPGVYTYEAFHPDIVLLPGCGVDFTESRLSNLLGIRKR HPFQEGFKIMYEDL | 287 |
| FSATMTIDLMNNAIIDNYLEIGRQNGVLESDIGVKFDTRNFLGWDPETKLIMPGVYTYEAFHPDIVLLPGCGVDFTESRLSNLLGIRKR HPFQEGFKIMYEDL | 287 |
| FSVTMTIDLMNNAIIDNYLKVRQNGVLESDIGVKFDTRNFKLGWDPETKLIMPGVYTYEAFHPDIVLLPGCGVDFTESRLSNLLGIRKK QPFQEGFKILYEDL | 288 |
| YSETMTIDLMNNAIVDNYLEVGRQNGVLESDIGVKFDTRNFLGWDPVTKLVMPGVYTNEAFHPDIVLLPGCGVDFTQSRLSNLLGIRKRL PFQEGFQIMYEDL | 274 |
| YSETMTIDLMNNAIIEHYLRVGRQHGVLESDIGVKFDTRNFLGWDPETQLVTPGVYTNEAFHPDIVLLPGCGVDFTESRLSNILGIRKRQ PFQEGFVIMYHL | 268 |
| FSETMTIDLMNNAILENYLQVGRQNGVLESDIGVKFDSRNFKLGWDPVTKLVMPGVYTYEAFHPDVVLLPGCGVDFTESRLSNLLGIRKK QPFQEGFRIMYEDL | 271 |
| FSVTMTIDLMNNAIIDNYLAVGRQNGVLESDIGVKFDTRNFLGWDPVTELVMPGVYTNEAFHPDIVLLPGCGVDFTESRLSNLLGIRKR QPFQEGFKIMYEDL | 282 |
| FSETMTIDLMNNAILENYLQVGRQNGVLESDIGVKFDSRNFKLGWDPVTKLVMPGVYTYEAFHPDVVLLPGCGVDFTESRLSNLLGIRKK QPFQEGFRIMYEDL | 271 |
| ************** ** ************** * ****** * ********* ********* * ********* * * |  |

---

| α5 |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| EGGINIPALLDVDAYQASLKDDTEQGGDGAGGGNNSGSGAEENSNA AAAAMQPVEDM--------------NDHAIRGD TFA TRAE EKRAEAEAAAAAAPAAQPEVEKPQ | 374 |
| EGGINIPALLDVTAYEESKKDTT----------------------TE TTTLAVAEE TSE--------------DDDITRG DTYITE-KQKREAAA AEVKK------------------ | 348 |
| EGGINIPALLDVTAYEESKKDTT----------------------TE TTTLAVAEE TSE--------------DDNITRG DTYITE-KQKREAAA AEVKK------------------ | 348 |
| EGGINIPALLDVDAYENSKKEQK----------------------AK IEAATAAAE AKANIVASDSTRVANAGEVR GD NFAPT PVPTAESLLADVSEG T--------------D | 365 |
| EGGINIPALLDVAKEYASIQKAKEEGKE------------------IG DDTFATRP | 311 |
| EGGINIPALLDVKKYENSLQDN----------------------TV RGDNFI ALN | 301 |
| EGGINIPALLDVPKYLESKKLE----------------------EA LE NA AKAN--GPA--------------RGD SSVSR-----EVEKAAEK | 321 |
| EGGINIPALLDVDAYEKSKEDAA----------------------AE ATA AVATA STEV--------------RGD NFASA-AAVAAAAEAETES | 338 |
| VGGNIPALLNVKEYLKDKEEAG----------------------KA DANTI KAQNDA VP--------------RGD NYASA-----AEAKAAGK | 323 |
| *********** * * |  |

---

| α6 | E | F | B9 |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| KKPVIKPLTEDSKRSYNLISNDSTFTQYRSWYLAYNYGDPQTGIRSWTLLCTPDVTCGSEQVWSLPDMMQDPVTFRSTSQISNFPVVGAELL PVHSKSFYND | 478 |
| -ELKIOPLEKDSKRSYNVLE-DKINTAYRSWYLSNYGNPEKGIRSWTLLTTSDVTCGAEQVYWSLPDMMQDPVTFRSTRQVNNYPVVGAEL MPVFSKSFYNE | 451 |
| -ELKIOPLEKDSKRSYNVLE-DKINTAYRSWYLSNYGNPEKGIRSWTLLTTSDVTCGAEQVYWSLPDMMQDPVTFRSTRQVNNYPVVGAEL MPVFSKSFYNE | 451 |
| VKLTIQPVEKDSKNRSYNVLE-DKINTAYRSWYLSNYGDPEKGVRSWTLLTTSDVTCGAEQVYWSLPDMMKDPVTFRSTRQVSNYPVVGE VELLPVHAKSFYNE | 468 |
| QDLVIEPVAKDSKNRSYNLLPNQNTAYRSWFLAYNYGDPKKGVQSWTLLTTADVTCGSQQVYWSLPDMMQDPVTFRPSTQVSNYPVVGEV ELLPVHAKSFYNE | 415 |
| KAARIEPVETDPKGRSYNLLP-DKKNTKYRSWYLAYNYGDPEKGVRSWTLLTTPDVTCGGEQVYWSLPDMLMQDPVTFRSSTQQVSNYPV VGAELLPVYSKSFFNE | 404 |
| -ELVIEPIKQDDTKRSYNLIE-GTMDTLYRSWYLSYT YRD PENGVQSWTLLTTPDVTCGAEQVYWSLPDLMQDPVTFRSTQQVSNYPVVGAELLPVFRAKSFYND | 424 |
| -KIVIQPVEKDSKNRSYNVLP-DKINTAYRSWYLAYNYGDPEKGVRSWTLLTTSDVTCGVEQVYWSLPDMMQDPVTFRSSRQVSNYPVVA AELLPVHAKSFYNE | 441 |
| -EIELKAILKDDSDRSYNVIE-GTTDTLYRSWYLSYTTGIPRRGCSRGRCSPPRNHGCAEQVYWSLPDLMQDPVTFRSTQQVSNYPVVGA ELMPFRAKSFYND | 426 |
| ***** * ****** * * ** ********** * ** ********** * ** ******** * |  |

---

| α7 | B10 | G | B11 | H | I |  |
| --- | --- | --- | --- | --- | --- | --- |
| QAVYSQILIRQFTSLTHVFNRFPENQILARPPAPTITTVSENVPALTDHGTLPLRNSIGGVQRVTIDARRRTC PYVYKALGIVSPRVLSSRTF | 571 |
| QAVYSQQLRQATSLTHVFNRFPENQILIRPPAPTITTVSENVPALTDHGTLPLRNSIRGVQRVTVTDARRRTC PYVYKALGIVAPRVLSSRTF | 544 |
| QAVYSQQLRQATSLTHVFNRFPENQILIRPPAPTITTVSENVPALTDHGTLPLRNSIRGVQRVTVTDARRRTC PYVYKALGIVAPRVLSSRTF | 544 |
| QAVYSQQLRQSTSLTHVFNRFPENQILIRPPAPTITTVSENVPALTDHGTLPLRSSIRGVQRVTIDARRRTC PYVHKALGIVAPKVLS SRTF | 561 |
| QAVYSQILRQSTALTHVFNRFPENQILVRPPAPTITTVSENVPALTDHGTLPLRSSISGVQRVTIDARRRTC PYVYKALGIVSPRVLSSRTF | 508 |
| QAVYSQILRQSTALTRVFNRFPENQILVRPPAATITTVSENVPALTDHGTLPLRSSISGVQRVTVTDARRRTC PYVYKALGIVAPRVLSSRTF | 497 |
| LAVYSQILRSYTSILTHVFNRFPDNIILCRPPAPTITTVSENVPALTDHGTLPLRSSIRGVQRVTVTDARRRTC PYVYKALGIVAPRVLSSRTF | 517 |
| QAVYSQQLRAFTSLTHVFNRFPENQILVRPPAPTITTVSENVPALTDHGTLPLRSSIRGVQRVTVTDARRRTC PYVYKALGIVAPRVLSSRTF | 534 |
| LAVYSQILRSYTSILTHVFNRFPDNIILCRPPAPTITTVSENVPALTDHGTLPLRSSIRGVQRVTVTDARRRTC PYVYKALGIVAPRVLSSRTF | 519 |
| ***** * ** ********* ***** ************ ************ ************ ************ ************ * ** ********* * |  |

---

Figure 1. Sequence Alignment of Penton Base Proteins from Representative Adenovirus Serotypes and Subgroup
Subgroup C Ad2, accession number PO3276; subgroup B Ad3, S41389; subgroup B Ad7, AAR89958; subgroup B Ad11, AAP49205; subgroup F Ad41, AAF14179; subgroup A Ad12, P36716; subgroup D Ad17, NP_049379; subgroup E Ad25, NP_478405; and subgroup D Ad37, CAC82544.

Structure of the Adenovirus Penton

rus have enabled location of the major and some of the minor capsid proteins (Stewart et al., 1991; Stewart et al., 1993). Atomic-resolution structural data are available for the Ad5 hexon (Rux and Burnett, 2000); the fiber head domains of Ad5 (Xia et al., 1994), Ad12 (Bewley et al., 1999), Ad3 (Durmort et al., 2001), Ad37 (Burmeister et al., 2004); and the Ad2 fiber head and part of its shaft (van Raaij et al., 1999). The only major capsid protein without a high-resolution structure has hitherto been the penton base protein. Negative stain electron microscope images of the adenovirus penton show that the penton base pentamer forms a bucket-shaped structure from the center of which the fiber shaft emerges (Ruigrok et al., 1990). More detailed information on the penton base is available from cryo-EM, firstly on dodecahedral particles composed of 12 Ad3 penton bases or complete pentons (Schoehn et al., 1996), and secondly in the context of the complete virus, including a complex of Ad12 and a soluble extracellular $\alpha_{v}\beta_{5}$ integrin construct (Chiu et al., 1999).

The penton base sequence is exceptionally highly conserved even between human and other animal adenoviruses (typically 70% homology between any two penton bases, Figure 1). It has a peptide chain length of between 470 and 570 residues with the difference almost exclusively being due to a hypervariable loop region of variable length in the middle of the chain, which in most cases contains the RGD motif (Figure 1). Cryo-EM structures of the Ad2, Ad3, and Ad12 penton base show five external protrusions that have been identified by RGD specific monoclonal antibody binding as the flexible, hypervariable RGD loop (Schoehn et al., 1996, Stewart et al., 1997). Up to five integrins can bind to the penton base leading to a ring-like structure that may be important for coreceptor clustering and endocytosis (Chiu et al., 1999).

Given the importance of the penton base in many aspects of adenovirus biology, we set out to determine the atomic structure of the protein. Here, we report the 3.3 Å crystal structure of human Ad2 penton base (Ad2pb) in a dodecahedral form, which provides the last remaining major piece of the adenovirus capsid puzzle. We also describe the 3.5 Å crystal structure of the penton base in complex with the interacting N-terminal fiber peptide. This gives insight into the symmetry mismatch implicated in the assembly of the penton from pentameric penton base and trimeric fiber. Finally, this structural information will aid in studies of other less well-understood roles of the penton base, such as in endocytosis, and will potentially allow rational engineering of the penton base for applications in gene therapy.

Results and Discussion

Structure Determination
A full-length construct of the 571 residue Ad2pb polypeptide was expressed in the baculovirus system and purified. Because this protein was easily degraded, N-terminal sequencing was used to identify proteolytic products. An N-terminal truncation (49-TGGR...) was identified as a stable species and used for all subsequent structural analysis.

All constructs of adenovirus 2 penton base (Ad2pb) formed isolated pentamers in solution as shown by negative stain electron microscopy (Figure 2A). Three crystal forms of Ad2pb were obtained but only those in spacegroup C2, diffracting to 3.3 Å resolution (Table 1), could be reproducibly grown. The unit cell dimensions, $a = 436.0 \, \AA$, $b = 300.2 \, \AA$, $c = 420.6 \, \AA$, and $\beta = 104.4^\circ$, suggested a large assembly in the unit cell. Indeed, it was unexpectedly found that under crystallization conditions, the pentamers had associated into regular dodecahedral particles, as deduced from the characteristic icosahedral symmetry observed in the self-rotation function and EM of dissolved crystals and Ad2pb after preincubation in crystallization buffer (Figure 2A). Formation of dodecahedra of pentons has been observed in vivo in infected cells (Norrby, 1964) and with recombinant protein (Fender et al., 1997) for other adenovirus serotypes, notably Ad3, but not previously for Ad2pb. The Ad2pb/fiber peptide complex was formed by coin cubation with a 21 amino acid peptide corresponding to the N terminus of the Ad2 fiber protein and crystallized in similar conditions with a similar unit cell (Table 1). A cryo-EM model of the Ad3 dodecahedron (Schoehn et al., 1996), extended to 15 Å resolution, was used to solve the Ad2pb dodecahedron structure by molecular replacement. Subsequent phase extension with 60-fold noncrystallographic symmetry averaging (one dodecahedron per asymmetric unit) yielded a readily interpretable experimental electron density map at 3.3 Å resolution.

Overall Structure
The Ad2pb monomer folds into two domains, a lower jellyroll-motif domain facing the capsid interior (40% of the residues) and an upper insertion domain facing the solvent exposed exterior (60% of the residues) (Figure 3A). The monomer is elongated and measures approximately 100 Å from the base of the jellyroll domain to the top of the insertion domain (excluding the disordered RGD loop). The jellyroll motif, common to many viral capsid proteins, consists of two, four-stranded antiparallel $\beta$ sheets, which pack against each other forming a $\beta$-barrel (Chelvanayagam et al., 1992). The $\beta$ strands of the jellyroll are conventionally labeled CHEF and BIDG (Richardson, 1981), with adjacent strands B and I being formed by the opposite extremities of the polypeptide (Figures 3A and 6). Thus, residues 49 and 569 are within ~12 Å of each other at the base of the protein. The location of the N-terminal 49 residues absent from the structure is of considerable importance because this region contains two highly conserved xPPxY motifs, 11-PPPSY and 41-VPPRY, in Ad2pb. The first of these

---

The hypervariable RGD loop region (yellow), the variable loop (light green), residues involved in dodecahedron formation (blue), fiber peptide binding residues (pink), and residues implicated in detergent molecule binding (purple) are highlighted. Secondary structure elements are denoted above the sequence by blue arrows ($\beta$ strands) and pink cylinders ($\alpha$ helices). Stars below the sequences denote completely conserved residues. The RGD sequence is highlighted in red.

Molecular Cell
124

A

B

C

Figure 2. Adenovirus 2 Penton Base Dodecahedron

(A) Electron micrographs of adenovirus 2 penton base (Ad2pb). The scale bar corresponds to 150 nm. Left, Ad2pb –49 construct after incubation for 1 hr in crystallization solution showing formation of dodecahedra. Right, Ad2pb –49 construct in a buffer of 25 mM Tris, (pH 7.5), and 100 mM NaCl showing the pentameric form.

(B) Stereo view of a space-filling model of the dodecahedron looking down a 3-fold axis with each pentamer colored separately. The contact region between pentamers across the 2-fold axis is visible around the small triangular hole at the 3-fold axis.

(C) Stereo view of the residues involved in dodecamerization. The surface is semitransparent with residues involved in the dodecahedron interface depicted as stick representations. The pentamer in red donates loops colored bright green (452–455), orange (58–61), and red (99–102), and the pentamer in blue donates 2-fold related loops colored in cyan (452–455), pink (58–61), and blue (99–102).

has been implicated in interactions with host cell WW domain-containing ubiquitin ligases such as WWP1, WWP2, Nedd4, and AIP4 (Galinier et al., 2002). In the free penton structure, the N-terminal region is presumably exposed to the solvent. In the intact virus, based on the observed position of residue 49, it seems most likely that the N-terminal residues would be located in the viral interior, perhaps making contact with neighboring hexons or other internal proteins. In the recent X-ray structure of the enveloped bacteriophage PRD1 (Abre-

Structure of the Adenovirus Penton
125

Table 1. Data Collection, Averaging, and Refinement Statistics

| Crystal Contents | Ad2 Penton Base Dodecahedron | Ad2 Penton Base Dodecahedron Cocrystallized with Fiber Peptide |
|------------------|------------------------------|-------------------------------------------------------------|
| Space group      | C2                           | C2                                                          |
| Cell dimensions (Å) | a = 436.0, b = 300.2, c = 420.6, β = 104.4° | a = 438.4, = 299.8, c = 419.6, β = 103.2° |
| Observations     |                              |                                                             |
| Total measured reflections | 657,395                  | 761,440                                                    |
| Unique reflections | 447,000                    | 354,521                                                    |
| Resolution (Å)   | 20–3.3                      | 30–3.5                                                     |
| Completeness<sup>a</sup> | 0.57 (0.22)              | 0.54 (0.29)                                                |
| R<sub>merge</sub><sup>a,b</sup> | 0.21 (0.47)            | 0.19 (0.45)                                                |
| Average measurement redundancy | 1.5                   | 2.1                                                        |
| NCS redundancy   | 60-fold                     | 60-fold                                                    |
| Initial NCS averaging statistics<sup>c</sup> |                          |                                                             |
| Correlation<sup>d</sup>/Rfac<sup>e</sup> | 0.87/0.25                 | 0.68/0.37                                                 |
| Rigid-body refinement, R<sub>cryst</sub> change<sup>e</sup> | 0.32→0.29                  | 0.41→0.32                                                 |
| Final NCS averaging statistics<sup>f</sup> |                          |                                                             |
| Correlation<sup>d</sup>/Rfac<sup>e</sup> | 0.90/0.22                 | 0.91/0.20                                                 |
| Model refinement R<sub>cryst</sub>/R<sub>free</sub> (%)<sup>g</sup> | 31.1/31.1                  | 30.5/30.3                                                 |
| Model geometry    |                              |                                                             |
| RMSD bond lengths (Å) | 0.012                     | 0.014                                                      |
| RMSD bond angles (°) | 1.89                      | 1.83                                                       |
| Ramachandran plot |                              |                                                             |
| Favored + additional regions | 95.1%                  | 93.9%                                                      |

<sup>a</sup> Values in parentheses refer to the highest resolution shell.

<sup>b</sup> R<sub>merge</sub> = Σ<sub>h</sub>Σ<sub>i</sub>[I<sub>hi</sub> - ⟨I<sub>h</sub>⟩]/Σ<sub>h</sub>(I<sub>h</sub>), where I<sub>hi</sub> is the *i*th observation of a symmetry equivalent of reflection *h*.

<sup>c</sup> Averaging with RAVE to high-resolution limit with strict icosahedral symmetry.

<sup>d</sup> Correlation between the densities of all NCS related points. Rfac = Σ|F<sub>obs</sub> - F<sub>map</sub>|/ΣF<sub>obs</sub> where F<sub>map</sub> is Fourier coefficient of back-transformed averaged map.

<sup>e</sup> Rigid-body refinement of 60 individual monomers with CNS over resolution range 20–5 Å.

<sup>f</sup> Averaging with RAVE to high-resolution limit with relaxed icosahedral symmetry as defined by rigid body refinement<sup>e</sup>.

<sup>g</sup> Strict NCS was applied with matrices found by rigid-body refinement<sup>e</sup>. R<sub>cryst</sub> = Σ|F<sub>obs</sub> - F<sub>calc</sub>|/ΣF<sub>obs</sub>, where summation is over data used in the refinement. R<sub>free</sub> was calculated by using 2.7% of the observed reflections excluded from refinement. Excluded data were selected from thin shells to minimize correlation with noncrystallographic symmetry related reflections.

scia et al., 2004), which has striking structural similarities to adenovirus (Benson et al., 1999), the N-terminal 13 residues of the protein P31, equivalent to the penton base, interact with the peripentonal P3 proteins (equivalent to hexon).

The second, distal domain of Ad2pb has an ordered but irregular fold formed by two long insertions, one comprising residues 129–436 between jellyroll strands D and E and the other formed by residues 468–521 between strands F and G (Figure 3A). The first insertion contains the hypervariable RGD loop region that exhibits great variability in sequence and length amongst adenovirus serotypes. The RGD loop of Ad2pb consists of ~80 residues and is glycine and alanine rich (Figure 1). It is clearly flexible because no electron density is seen for residues 298–375 in NCS averaged maps. At a unique crystal contact, however, about 15 residues of the RGD loop are visible in one monomer in an unaveraged map comprising an α helix followed by a turn and β strand. However, a sequence assignment cannot be made with certainty, as the density is not continuously connected to the bulk of the protein. Although this region has reasonably clear electron density, the position relative to the rest of the penton base is most likely an artifact of the crystal packing. On the other hand, consistent with EM studies, this observation shows that although the long RGD loop is flexible (Stewart et al., 1997, Chiu et al., 1999), it may possess at least some structured regions.

A second surface loop variable between serotypes, and hence denoted the variable loop, also occurs in the first insertion (Figures 1, 3A, and 3C). In Ad2pb, this loop comprises residues 142–169 and forms a flexible β ribbon projecting into the solvent. In other serotypes the variable loop can be up to ten residues longer (Figure 1), for instance in the Ad3 penton base where it is prominent in the cryo-EM structure (Schoehn et al., 1996). The function of this variable loop is unknown, but it is potentially a site where additional sequences could be engineered into the structure for gene therapy applications (see Conclusions).

Pentamerization

Ad2pb forms an intimate pentamer that buries approximately 26% of the total surface area of each monomer (Figure 2C). The stability gained in pentamer formation is due mainly to the burying of hydrophobic surface. Only two salt bridges are formed between adjacent monomers upon pentamerization, Arg420-Glu172* and Arg197-Asp220* (asterisk indicates the second subunit). A ring of jellyroll domains forms the compact base of the pentamer, and the insertion domains create a wider, α helical-lined top, giving an overall shape resembling a tulip (but with five “petals” not six). Because the insertion domains are tilted relative to the jellyroll domain, the monomers appear to wrap around each other with a right-handed twist. Interestingly, in contrast to the penton base, the hexon is wider at the base and narrower at the top, suggesting a complementarity of shape between the two proteins that is most likely important for capsid assembly and stability.

Molecular Cell
126

A
RGD loop
373
Variable
loop
Zwittergent 3-12
C
N
298
298
373
Variable
loop
RGD loop
~90°
rotation
C
N

B

C
RGD
loop
Variable
loop
Figure 3. Structure of the Adenovirus 2 Penton Base
(A) Left, tricolor ribbon representation of the monomer with the jellyroll domain (green) and the insertion domain (blue, residues 129–434; red, residues 466–519). The termini are labeled N and C. The variable loop and the RGD loop are labeled with disordered residues 298–373 of the RGD loop depicted by a dotted line. Putative zwittergent 3-12 molecules are drawn as stick representations. Right, rainbow ribbon representation colored from N terminus (blue) to C terminus (red) viewed after rotation of approximately 90° from previous image.
(B) Stereo view of the putative zwittergent 3-12 binding site on the penton base monomer. The detergent molecules are colored green. Interacting residues from the protein are cyan.
(C) The pentamer, the functional unit of the penton base protein, shown as a surface representation (left) and ribbon diagram (right), with each monomer colored uniquely. The variable loop, RGD loop and N- and C termini are marked.

Structure of the Adenovirus Penton
127

A pore occurs along the 5-fold axis of the pentamer with diameter 15–20 Å at the base and tapering to 5–10 Å at the top of the insertion domain. The top, narrow part of the pore is predominantly hydrophobic, apart from Asp478, and lined by bulky residues Phe489, Leu485, Tyr482, Tyr476, and Phe475 coming from helix α7 and the preceding strand β9 of each monomer of the insertion domain. Notably, fiber binding modifies this region (see below). The five side chains of Arg-473 point into the central pore forming a tight constriction at the bottom of the insertion domain. Beneath this, the lower, wider section of the pore formed by His471 from helix α8 and residues from β strands I, G, and D from the jellyroll domain is much more hydrophilic with residues Asp65, Asp122, Asp253, Lys124, Arg564, and Glu518 and two sets of salt bridges formed by Asp122-Arg564 and Lys124-Glu525 exposed to the solvent-filled pore. The possibilities of inserting the fiber protein into the pore or exposing to the surface the N terminus of the penton base through the pore are excluded, as it is too narrow for a polypeptide chain to be threaded all the way along.

Dodecahedron Formation
The Ad2 dodecahedron is a highly regular particle of diameter 280 Å with an internal spherical cavity of diameter about 80 Å (Figures 2A and 2B). The particle is formed by interactions between three loop regions in the jellyroll domain, which, however, do not form an extensive interface. Residues 99–102 (NDYS) and 58–61 (SELA) from one monomer and 452–455 (RSTS) from an adjacent monomer in the same pentamer (bold type indicates residue conservation, Figure 1) interact with the corresponding loops from a neighboring pentamer in the dodecahedron about the icosahedral 2-fold axis (Figure 2C).

Under normal conditions, Ad2pb does not spontaneously form dodecahedra, unlike the penton base of Ad3. As monitored by negative stain EM, Ad2pb dodecahedra formed only upon addition of the crystallization buffer, which contains both 1.5 M ammonium sulfate and 10% dioxane (Figure 2A). Thus, changes in the solvent environment are sufficient to favor association of pentamers into dodecahedra. Because there are no residues in the three loop areas that are exclusive to those penton bases that spontaneously form dodecahedra (Figure 1), it is difficult to identify the key residues except by further mutational analysis.

Detergent Binding
Ad2pb and the Ad2pb/fiber peptide complex were crystallized in the presence of zwittergent 3-12 (3-dodecyl-dimethyl-ammonio-propane-1-sulphonate), a zwitterionic detergent. This detergent additive was required for reproducible growth of the high resolution C2 crystal form. Two unexplained strong peaks in the averaged map for Ad2pb were assigned to the sulphonate and ammonio head group of two putative detergent molecules with electron density extending to two or three carbons of the aliphatic tail (Figures 3A and 3B). The detergent molecules bind to solvent-exposed basic patches on the insertion domain. The sulphonate head group of one putative detergent molecule is positioned in a hydrophilic pocket formed by Arg272, Arg146, Thr274, and the nitrogen of Trp165. The sulphonate of the second molecule forms a salt bridge with Lys263, and the ammonio group is positioned within hydrogen bonding distance of Asp277 (Figure 3B).

It has recently been found that the Ad3 penton base dodecahedra, but not Ad3 virions, selectively bind heparin and heparan sulfate with high affinity and that this interaction at the cell surface can promote internalization (Vives et al., 2004). The binding of detergent molecules in the Ad2pb structures could mimic this interaction due to the resemblance of the sulphonate-containing moiety of zwittergent 3-12 to the sulfate groups in heparin and heparan. Obstruction of these sites due to the presence of neighboring hexons might explain the inability of the Ad3 virus itself to use heparan sulfates as receptors (Vives et al., 2004).

These observations are interesting in the light of the little understood but critical activity of the penton base in pH-dependent lipid binding and endocytosis (Medina-Kauwe, 2003). Mutating the conserved Trp165, to which we find one detergent molecule binding, to histidine resulted in penton bases that were severely deficient in cell attachment and somewhat in internalization, consistent with the idea that Trp165 may be involved in lipid binding (Karayan et al., 1997). Clearly, further studies of this putative lipid binding site, including the effect of pH, are required.

Fiber Binding
The penton complex is formed by the noncovalent binding of the N-terminal extremity of the fiber polypeptide with the penton base top surface (Devaux et al., 1987). The penton is very stable and can be dissociated only by treatment with chaotropic agents at elevated temperature (Boudin and Boulanger, 1982). There has been much speculation about the nature and significance of the symmetry mismatch between the trimeric fiber and the pentameric base, which we can now largely resolve.

A peptide corresponding to the 21 N-terminal residues of adenovirus 2 fiber protein (1-MKRARPSEDTFNPVY PYDTEC) was cocrystallized with Ad2pb. The 3.5 Å averaged electron density map gave unambiguous density for a central 11 residue portion of the fiber tail peptide, but not the preceding basic nuclear location signal (Figure 4). The observed residues, 10-TFNPVYPYDTE, include the FNPVYPY motif extremely highly conserved amongst fiber proteins in diverse adenovirus serotypes (Figure 4D). The fiber peptide lies in a groove on the top surface of the protein formed between two adjacent monomers and resulting from a conformational change in helix α7 of the insertion domain. Fiber peptide binding requires this helix to kink by almost 45° between residues Gln482 and Ile484, and additionally, the loop region C-terminal to helix α7 is rearranged (Figure 5). Structural changes in the penton base are restricted to a 23-residue stretch from residues 482–505, indicating that peptide binding does not have global repercussions.

It has been previously proposed that the peptide 253-SRLSNLLG is the fiber binding site (Hong and Boulanger, 1995). The crystal structure shows that this peptide forms a turn and short, largely buried helix α4 (Figure 1) that is at least 25 Å from the peptide and is therefore not part of the fiber binding site. In reality, the peptide binding site is multipartite and distributed over adjacent

Molecular Cell
128

A

B

C

D
Ad2    M-KRARP---------SEDTFNPVYPYDTE-TGPPTPVPFLTPPFVSPNGFQES
Ad3    MAKRAR---------LSTSFNPVYPYEDES-SSQHPFINPGFISPDGFTQS
Ad7    MTKRVR---------LSDSFNPVYPYEDES-TSQHPFINPGFISPNGFTQS
Ad11   MTKRVR---------LSDSFNPVYPYEDES-TSQHPFINPGFISPNGFTQS
Ad12   M-KRSRTQYAEETEENDDFNPVYPFDPDF-TSDVPFVTPPFTSSNGLQEK
Ad17   MSKRLR---------VEDDFNPVYPYGYAR-NQNIPFLTPPFVSSDGFKNF
Ad25   MSKKRVR--------VDDDFDPVYPYDADN-APTVPFINPPFVSSDGFQEK
Ad37   MSKRLR---------VEDDFNPVYPYGYAR-NQNIPFLTPPFVSSDGFKNF
Ad41(s) M-KRTR---------IEDDFNPVYPYDTFS-TPSIPYVAPPFVSSDGLQEK
Ad41(l) M-KRAR---------LEDDFNPVYPYEHYN-PLDIPFITPPFASSNGLQEK

monomers. It is formed by helices $\alpha 7$ and $\alpha 3$ and strand $\beta 5$ at the interface between adjacent monomers of the pentamer (Figure 4B). The monomers in turn are connected in this region by a salt bridge between Asp220 (monomer B) and Arg197 (monomer A), which is also present in the native structure. Hydrophobic residues Leu193, Phe499, and the aliphatic portion of Arg498 contributed by monomer A and residues Met227, Pro228, and Tyr292 from monomer B form part of the peptide binding surface. Although the binding groove is relatively hydrophobic, a number of putative hydrogen bonds are also formed between the peptide and the penton base. The side chain amine of Asn12(fiber) forms a hydrogen bond with the carboxylate of Glu203. Additionally, the $\eta 1$-nitrogen of Arg197 is within 3 Å of the main chain carbonyl of Asn12(fiber). The main chain carbonyl of Val14(fiber) forms a hydrogen bond with the $\eta 1$-nitrogen Arg496, and a salt bridge occurs between the carboxylate of Asp18(fiber) and the side chain amino group of Lys387. The $\delta$-1 nitrogen of His494 hydrogen bonds to the side chain hydroxyl of Tyr15(fiber). Tyr17-(fiber) is stacked between Tyr15(fiber) and Tyr292, donated from the adjacent monomer B (Figure 4B). All cited interactions are to the universally conserved FNPVYPY motif of the fiber tail. In addition, all cited residues on the penton base with which the peptide interacts are also very highly conserved with the exception of Lys387 (Figure 1). Thus, this is likely to be a universal mode of association for adenovirus penton base and fiber.

How does the trimeric fiber fit into a pentameric penton base? The clear electron density for the fiber peptide after 60-fold NCS averaging implies a conserved and stable binding conformation for the fiber peptide in the penton base monomer, with each fiber binding site equivalent and one penton being capable of binding five peptides. Comparison of adjacent monomers in the bound and unbound conformations shows that they are sterically incompatible because Thr488 of one monomer with bound peptide clashes with Gln482 and Val479 of an adjacent monomer without peptide. Thus, the conformational switch upon fiber binding is likely to be cooperative around the penton base. This would allow the three independently flexible fiber tails to be accommodated in three binding sites, leaving two sites unsatisfied (Figure 5C). However, all five sites would likely adopt a conformation more closely resembling a fiber peptide bound state. The peptide lies almost horizontal and radial on the top surface of the penton base with its N terminus external and the C-terminal part central (Figure 4A). This arrangement allows the three flexible tails to come together in the center to form a trimeric structure probably involving the following conserved proline-rich regions (27-PFLTPPF) prior to the first repeating shaft motif. This

is consistent with observations from EM showing the fiber shaft emerging rigidly from the center of the penton base (Chroboczek et al., 1995; Wu et al., 2003). There are two possible arrangements; either the three fiber tails occupy three successive sites $72^\circ$ apart, or two are in neighboring sites and one is separated by $144^\circ$ as in Figure 5C. The latter solution is most likely because it approximates more closely to 3-fold symmetry. The tightness of the trimeric fiber binding is probably due to the avidity effect of the three binding sites and the fact that the conformational switch leads to a more stable conformation of the penton base, notably of helix $\alpha 7$ and the following residues, which are poorly ordered in the unliganded state.

There are possibly additional fiber interactions with the penton base not seen in our structure due to the shortness of the tail peptide used. The presence of the intact fiber in the penton base could alter and/or stabilize the conformation of the RGD loop possibly through direct interactions as suggested by comparative EM reconstructions of Ad3 penton base and penton dodecahedra, where a conformational change in the penton base upon fiber binding was reported (Schoehn et al., 1996).

In the spirit of poetic license, we note that nature exhibits a trimeric/pentameric symmetry mismatch phenomenon in the beautiful flowers of the Passiflora family, which have five stamens and three stigma. An example is shown in Figure 5C that appears to us to be a plausible representation of how the three fiber tails might bind at the interface of the five penton base subunits, leaving two sites vacant.

It remains unclear how some avian adenoviruses are capable of binding two fiber proteins, one short and one long, per penton base. Sequence analysis and EM suggest that the short avian fiber is structurally similar to mammalian adenovirus fibers and interacts normally with the penton base, whereas the long fiber is clearly abnormal both in sequence (with a long polyglycine sequence immediately following the penton base binding motif) and in disposition (it appears to come out horizontally from the penton base) (Hess et al., 1995). Assuming the short fiber binds in three binding sites, these observations could be explained by two of the long fiber tails interacting with the remaining two binding sites (which must both be on the same half of the penton base) and the polyglycine stretch, providing sufficient flexibility to avoid steric clashes between the fibers.

Structural Comparisons with Other Proteins Based on the DALI algorithm (Holm and Sander, 1993), the Ad2pb monomer is structurally most closely related to proteins such as Drosophila nucleoplasm-like pro-

---

**Figure 4. Fiber Peptide Interactions with the Penton Base**

(A) Stereo diagram of the fiber peptide (red) bound to the penton base subunit interface viewed looking down the 5-fold axis.

(B) Stereo view of the peptide binding site. The ribbon diagrams of the monomers are colored blue and green and the bound peptide is depicted as sticks with carbons colored white. Residues forming the peptide binding pocket are colored yellow for monomer A and orange for monomer B. Putative hydrogen bonding interactions are denoted by dashed yellow lines.

(C) Stereo view of the experimental averaged electron density of the bound peptide.

(D) Sequence alignment of the N-terminal portion of representative fiber proteins from serotypes Ad2, accession number P03275; Ad3, P04501; Ad7, AAR89975; Ad11, NP_852715; Ad12, DAA00575; Ad17, DAA00599; Ad25, NP_478420; Ad37, AAB71734; Ad41 short fiber, P16883; and Ad41 long fiber, P14267. Conserved residues (yellow) and the peptide sequence used for structural analysis (red) are highlighted.

Molecular Cell
130

A

Peptide

B

C

Figure 5. Structural Rearrangement of Ad2 Penton Base upon Fiber Peptide Binding

(A) The pentamer without (left) and with (right) bound peptide shown as surface models. Colors are rainbow from N terminus (blue) to C terminus (red, not visible). At right, the peptide is colored dark blue.

(B) Least squares structural alignment of Ad2pb and Ad2pb/fiber peptide complex showing conformational switch. The stereo ribbon diagram of the monomer is colored green for the jellyroll domain, blue for the first insertion, and red (uncomplexed) or yellow (complexed with peptide) for the second insertion domain. The carbon, oxygen, and nitrogen atoms of the fiber peptide are in cyan, red, and blue, respectively.

(C) A passion fruit flower (Passiflora Purple Haze) that exhibits a natural occurrence of a 5-fold, 3-fold symmetry mismatch (see text). This photograph is reproduced with permission of Ulrich Katz, Botanical Garden Bochum, Germany.

Structure of the Adenovirus Penton
131

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 6. Structurally Related Viral Capsid Proteins

Comparison of Ad2pb (left), the P1 domain of adenovirus 2 hexon (middle), and PRD1 P31 protein (equivalent to the penton base) (right). Each comprises a basic jellyroll domain with strands labeled conventionally B to I (top). Whereas P31 has no loop extensions, both hexon and penton base have complex and different elaborations arising from the DE loop (blue) and FG loop (red). The P2 domain of hexon, the PRD1 P3 protein (equivalent to hexon), and Vp54 from *Paramecium bursaria* have similar topologies (cf. Figure 3 in Nandhagopal et al. [2002] and Figure 4 in Benson et al. [1999]).

tein (dNLP) (Namboodiri et al., 2003), a pentameric histone chaperone protein, or to viral capsid proteins from the picornavirus superfamily such as bean pod mottle virus (BPMV) middle component (Chen et al., 1989) and tobacco ring spot virus (Chandrasekar and Johnson 1998), each of which contain an 8-stranded antiparallel β barrel jellyroll domain. The penton base is also topologically related, but less structurally related, to other viral capsid proteins that contain two jellyroll motifs and form trimers such as the hexon (Rux et al., 2003), the coat protein P3 of the bacteriophage PRD1 (Benson et al., 1999, 2002), and Vp54 from *Paramecium bursaria Chlorella* virus (PBCV) (Nandhagopal et al., 2002). It is very striking that the large distal insertions into the jellyroll domains of the penton base, hexon, P3, and Vp54, all occur between the DE and FG strands, although the insertions themselves are structurally diverse in length and fold (Benson et al., 1999; Nandhagopal et al., 2002) (Figure 6).

Very recently, the X-ray structure of the complete enveloped bacteriophage PRD1, whose protein capsid resembles that of adenovirus, has been published (Abrescia et al., 2004). The 5-fold vertices of PRD1 are plugged by a pentameric assembly of the protein P31, which is equivalent to the adenovirus penton base. However, the P31 polypeptide is much smaller (126 residues) than the penton base and, while conserving the basic β-barrel with jellyroll topology, has no loop elaborations whatsoever except for short N- and C-terminal extensions (Figure 6). Thus, both PRD1 major capsid proteins, P3 and P31, while preserving the same basic topology, are less elaborate if not minimal versions of their adenovirus counterparts, hexon and penton base. The exact structure of the PRD1 5-fold vertex is still obscure. Associated with P31 are the trimeric P5 and receptor binding P2 proteins that form a flexible spike. It will be interesting to see whether the trimeric P5 protein binds in a similar way to the interface between the P31 subunits.

Conclusions

Recombinant adenoviruses are potentially an important vehicle for gene delivery for therapeutic applications. Despite extensive development, their utility has been restricted by promiscuous tropism and host inflammatory immune response (see for example, Nemerow [2000]; Young and Mautner [2001]; Amalfitano and Parks [2002]; and Einfeld and Roelvink [2002]). Structural studies of adenoviral capsid proteins are critical in being able to overcome these limitations. Most frequently, retargeting of adenovirus vectors has been achieved through alteration of the knob domain of the fiber protein, the site of primary receptor binding. Insertion of targeting sequences into the penton base has also been successfully used in the design of adenoviral vectors exhibiting nonnative tropism (Einfeld et al., 1999). Tropism can be successfully modulated in this manner, whereas the immune response to the vector is not eliminated. The immune system responds to the adenovirus vector by first producing antifiber antibodies recognizing the trimeric form of the C-terminal knob domain (Gahery-Segard et al., 1998). One potential method of improving adenovirus vectors for gene therapy application is therefore to use fiberless adenoviruses (Von Seggern et al., 1999; Li et al., 2000). However, a serious limitation of the fiberless vectors is the reduced stability of the capsid (Von Seggern et al., 1999) perhaps due to a nonoptimal conformation of the penton base in the absence of the fiber. Based on the results presented here, the conformational change in the penton base upon fiber binding can be to a large extent duplicated with short peptides and possibly small molecule mimics of the fiber protein N-terminal region. Thus, the stability of the fiberless vector could be improved for instance by engineering into the RGD loop the fiber tail motif. Furthermore, the extensive structural information now available on the three major capsid proteins allows the design and addition of amino acid sequences shown to interact with cellular receptors into surface loop regions of these proteins. Such a region in the penton base could be the second variable loop (residues 150–160), which forms a surface accessible protuberance (Figures 2C and 5A). Vectors based on such constructs would have the benefit of nonnative tropism and potentially less antigenic response.

Finally, due to the very high conservation of the interaction, a compound that would block the fiber tail binding to the penton base could be a general inhibitor of adenoviral assembly and multiplication. Such a compound might be useful as an agent for treating diverse adenoviral infections, which can still be the cause of serious epidemics especially for children and immunocompromised patients (Kolavic-Gray et al., 2002; Kim et al., 2003; Kojjaoghlanian et al., 2003). By using the crystal structure of the peptide-penton base complex, lead compounds of this kind could be evolved into pentavalent ligands with very tight avidity based binding. Such an approach has been explored for inhibitors of pentameric cholera and other toxins (Zhang et al., 2002).

Experimental Procedures

Protein Expression and Purification

A full-length construct of hAd2 penton base was a gift from Pierre Boulanger (Karayan et al., 1994). For baculovirus expression, the Bac-to-Bac system and Htb expression vector were from Invitrogen, and Sf21 and HighFive insect cells were from Gibco. Zwittergent 3-12 was obtained from Hampton. The hAd2 fiber N-terminal peptide (MKRARPSEDTFNPVYPYDTEC) was purchased from DiverDrugs. The C-terminal cysteine was added to the native sequence for possible heavy atom derivatization.

Purification of the full-length construct revealed closely spaced bands that by N-terminal sequencing corresponded to truncations of –49 and –30 residues. New –30 and –49 constructs for expression in the baculovirus system were obtained with forward oligomers 5′-TATTTTCAGGGCGCCATGGGATCCCCCTTCGATGCTCCC-3′ (–30 construct) and 5′-TATTTTCAGGGCGCCATGGGAACCGGGGGAGAAACAGC-3′ (–49 construct) and a reverse oligomer, 5′-GTAGGCCTTTGAATTCCTCAAAAAGTGCGGCTCGAT-3′ with NcoI (forward) and EcoR1 (reverse) cleavage sites for cloning. Virus amplification was performed in a monolayer of Sf21 cells and protein expression was done in HighFive cells in shaker flasks (135 rpm, 27°C, 3 days) with a multiplicity of infection between 1 and 5. Cells were collected at 1000 rpm for 5 min and stored at –80°C. Cells were resuspended in 25 mM Tris, (pH 7.5) containing 100 mM NaCl (buffer A) plus protease inhibitors (Roche) and lysed by sonication. The cell debris was pelleted at 100,000 × g for 45 min and the supernatant collected. Ammonium sulfate (30%) was added to the supernatant and the precipitant collected. The precipitant was resuspended in buffer A and the ammonium sulfate removed by dialysis against the same buffer. The solution was then applied to a MONO Q column (Pharmacia) by using a linear gradient from 100 mM to 1 M NaCl, in 25 mM Tris (pH 7.5). The Ad2pb was eluted at approximately 220 mM NaCl. The fractions of interest were concentrated to 6–9 mg/ml and stored at –80°C. Due to higher expression levels and greater proteolytic stability, only the –49 construct was used in extensive crystallization trials.

EM

Purified samples of Ad2pb (full-length and –49 construct) at 6 mg/ml were diluted to ∼0.1 mg/ml in crystallization buffer (1.6 M ammonium sulfate, 10% dioxane, 0.1 M MES, and [pH 6.5]) and incubated at room temperature for at least 1 hr to allow formation of dodecahedral particles. Protein samples were similarly diluted in a 1.6 M ammonium sulfate solution buffered with 0.1 M MES (pH 6.5) or a 10% dioxane solution buffered with 0.1 M MES (pH 6.5) and incubated at room temperature. Samples were then applied to the clean side of carbon on mica (carbon/mica interface) and negatively stained with 1% sodium silico-tungstate (pH 7.0). Micrographs were taken under low-dose conditions with a JEOL 1200 EX II microscope at 100 kV and a nominal magnification of 40,000×.

Crystallization

Crystals were grown in hanging drops against 1.5–1.6 M ammonium sulfate, 10% dioxane, and 0.1 M MES (pH 6.5). 1 μl of the crystallization solution was added to 1 μl of the protein solution at concentration 6–8 mg/ml (approximately 0.1 mM) in buffer A. Approximately 0.3 μl of a 40 mM zwittergent 3-12 stock solution was added directly to the hanging drop as an additive. Crystals of Ad2pb grew over 4–8 weeks at 15°C, reaching dimensions of 100 × 75 × 30 μm³. Prior to flash freezing, crystals were soaked for ∼5 min in 20% sucrose, 1.8 M ammonium sulfate, 10% dioxane, and 0.1 M MES (pH 6.5) as a stabilizing and cryoprotecting solution. For the crystallization of the Ad2 penton base/fiber peptide complex, a 3-fold molar excess of peptide was added to the Ad2pb protein and incubated on ice for 30 min prior to addition of crystallization buffer and zwittergent 3-12. The average crystal size was ∼75 × 75 × 10 μm³ for the Ad2pb fiber peptide complex.

Structure Determination

Crystallographic data for Ad2pb were collected on European Synchrotron Radiation Facility (ESRF) beamlines ID14-EH1, EH2, and EH4 in 0.5° frames. Due to radiation sensitivity, data from ten crystals were merged to create a more complete dataset. For the Ad2pb/fiber peptide complex, data were collected on the ESRF microfocus beamline ID13 with the EMBL microdiffractometer (Perrakis et al., 1999). Radiation damage was readily apparent after a few exposures, which limited the amount of usable data per exposed position

to 2° (four 0.5° frames). One large crystal (100 × 75 × 20 µm³) provided 104 frames (52°) of usable data to a maximum resolution of 3.5 Å, the combination of a 10 µm beam and the precision of the microdiffractometer permitting exposure of multiple volumes. A smaller crystal (75 × 75 × 10 µm³) gave an additional 12 frames (6°) of data to 3.5 Å. All data were integrated with MOSFLM (Leslie 1992) and scaled with SCALA (CCP4, 1994).

MOLREP (Vagin and Teplyakov, 2000) was used to place the cryo-EM electron density of the Ad3 dodecahedron (Schoehn et al., 1996) in the unit cell. Subsequent 60-fold NCS-averaging and gradual phase extension from 20 Å with RAVE (Kleywegt and Jones, 1999) allowed the generation of high quality electron density maps (Table 1). Model building was performed in O (Jones et al., 1991) and refinements carried out with CNS (Brünger et al., 1998). Only one monomer was built, and all refinements used strict 60-fold icosahedral NCS constraints. Unaveraged maps calculated with CNS from the model were used to locate deviations from perfect icosahedral symmetry at crystal contacts. In the case of the Ad2pb-fiber peptide complex, the native Ad2pb dodecahedron structure was used for molecular replacement and phase extension recommended from low resolution. However, maintaining strict 60-fold icosahedral NCS restraints gave poor final averaging statistics and poor density for the peptide. By using rigid-body refinement in CNS to relax the icosahedral symmetry for each of the 60 monomers and then repeating the averaging with newly defined NCS matrices vastly improved the map and allowed the peptide and interacting region of the penton base to be built unambiguously.

According to PROCHECK (Laskowski et al., 1993) (Table 1), only six (Tyr84, Phe137, Asp156, Glu167, Glu172, and Ser176) and four (Ser58, Asp65, Asp156, and Ser176) residues for the Ad2pb and Ad2pb/fiber peptide complex structures, respectively, were in disallowed regions of the Ramachandran plot. These generally lie in poorly ordered regions. Residues 568–571 (native C terminus) and residues 298–335 were disordered and not modeled in the Ad2pb structure. Similarly, residues 49–51, 297–373, and 568–571 were not modeled in the Ad2pb/fiber peptide structure. Figures were generated with PyMOL (DeLano 2002).

### Acknowledgments

The authors wish to thank the European Molecular Biology Laboratory and the European Synchrotron Radiation Facility (ESRF) staff, in particular Christian Riekel, for access to ESRF beamlines; Manfred Burghammer and David Flot for assistance in data collection on ID13; John Johnson and colleagues (Scripps Research Institute) for discussions about icosahedral symmetry and self-rotation functions; Fasseli Coulibaly and Felix Rey (Gif-sur-Yvette) for help with NCS averaging in RAVE; Pierre Boulanger (Lyon) for the Ad2 penton base construct; James Conway and Elisabeth Hewat (IBS) for electron microscopy; Catherine Mazza and Evelyne Gout for help with baculovirus expression; Wim Burmeister, Pascal Fender, Ulrike Kapp, and Catherine Stehlin for initiating the penton base crystallography project; Max Nanao for help with figures; and Rob Ruigrok for frequent discussions. We thank Ulrich Katz (Botanical Garden Bochum, Germany) for Figure 5C.

Received: August 25, 2004  
Revised: November 10, 2004  
Accepted: November 24, 2004  
Published online: December 16, 2004  

### References

Abrescia, N.G., Cockburn, J.J., Grimes, J.M., Sutton, G.C., Diprose, J.M., Butcher, S.J., Fuller, S.D., San Martin, C., Burnett, R.M., Stuart, D.I., et al. (2004). Insights into assembly from structural analysis of bacteriophage PRD1. Nature 432, 68–74.

Albinsson, B., and Kidd, A.H. (1999). Adenovirus type 41 lacks an RGD alpha(v)-integrin binding motif on the penton base and undergoes delayed uptake in A549 cells. Virus Res. 64, 125–136.

Amalfitano, A., and Parks, R.J. (2002). Separating fact from fiction: assessing the potential of modified adenovirus vectors for use in human gene therapy. Curr. Gene Ther. 2, 111–133.

Bailey, A., and Mautner, V. (1994). Phylogenetic relationships among adenovirus serotypes. Virology 205, 438–452.

Benko, M., and Harrach, B. (2003). Molecular evolution of adenoviruses. Curr. Top. Microbiol. Immunol. 272, 3–35.

Benson, S.D., Bamford, J.K., Bamford, D.H., and Burnett, R.M. (1999). Viral evolution revealed by bacteriophage PRD1 and human adenovirus coat protein structures. Cell 98, 825–833.

Benson, S.D., Bamford, J.K., Bamford, D.H., and Burnett, R.M. (2002). The X-ray crystal structure of P3, the major coat protein of the lipid-containing bacteriophage PRD1, at 1.65 Å resolution. Acta Crystallogr. D Biol. Crystallogr. 58, 39–59.

Bergelson, J.M., Cunningham, J.A., Droguett, G., Kurt-Jones, E.A., Krithivas, A., Hong, J.S., Horwitz, M.S., Crowell, R.L., and Finberg, R.W. (1997). Isolation of a common receptor for Coxsackie B viruses and adenoviruses 2 and 5. Science 275, 1320–1323.

Bewley, M.C., Springer, K., Zhang, Y.B., Freimuth, P., and Flanagan, J.M. (1999). Structural analysis of the mechanism of adenovirus binding to its human cellular receptor, CAR. Science 286, 1579–1583.

Blumenthal, R., Seth, P., Willingham, M.C., and Pastan, I. (1986). pH-dependent lysis of liposomes by adenovirus. Biochemistry 25, 2231–2237.

Boudin, M.L., and Boulanger, P. (1982). Assembly of adenovirus penton base and fiber. Virology 116, 589–604.

Brünger, A.T., Adams, P.D., Clore, G.M., DeLano, W.L., Gros, P., Grosse-Kunstleve, R.W., Jiang, J.S., Kuszewski, J., Nilges, M., Pannu, N.S., et al. (1998). Crystallography & NMR system: a new software suite for macromolecular structure determination. Acta Crystallogr. D Biol. Crystallogr. 54, 905–921.

Burmeister, W.P., Guilligay, D., Cusack, S., Wadell, G., and Arnberg, N. (2004). Crystal structure of species D adenovirus fiber knobs and their sialic acid binding sites. J. Virol. 78, 7727–7736.

CCP4 (Collaborative Computational Project, Number 4) (1994). The CCP4 suite: programs for protein crystallography. Acta Crystallogr. D Biol. Crystallogr. 50, 760–763.

Chandrasekar, V., and Johnson, J.E. (1998). The structure of tobacco ringspot virus: a link in the evolution of icosahedral capsids in the picornavirus superfamily. Structure 6, 157–171.

Chelvanayagam, G., Heringa, J., and Argos, P. (1992). Anatomy and evolution of proteins displaying the viral capsid jellyroll topology. J. Mol. Biol. 228, 220–242.

Chen, Z.G., Stauffacher, C., Li, Y., Schmidt, T., Bomu, W., Kamer, G., Shanks, M., Lomonossoff, G., and Johnson, J.E. (1989). Protein-RNA interactions in an icosahedral virus at 3.0 Å resolution. Science 245, 154–159.

Chiu, C.Y., Mathias, P., Nemerow, G.R., and Stewart, P.L. (1999). Structure of adenovirus complexed with its internalization receptor, alphavbeta5 integrin. J. Virol. 73, 6759–6768.

Chroboczek, J., Ruigrok, R.W., and Cusack, S. (1995). Adenovirus fiber. Curr. Top. Microbiol. Immunol. 199, 163–200.

Cohen, C.J., Shieh, J.T., Pickles, R.J., Okegawa, T., Hsieh, J.T., and Bergelson, J.M. (2001). The coxsackievirus and adenovirus receptor is a transmembrane component of the tight junction. Proc. Natl. Acad. Sci. USA 98, 15191–15196.

DeLano, W.L. (2002). The PyMOL Molecular Graphics System. (San Carlos, CA: DeLano Scientific).

Devaux, C., Caillet-Boudin, M.L., Jacrot, B., and Boulanger, P. (1987). Crystallization, enzymatic cleavage, and the polarity of the adenovirus type 2 fiber. Virology 161, 121–128.

Durmort, C., Stehlin, C., Schoehn, G., Mitraki, A., Drouet, E., Cusack, S., and Burmeister, W.P. (2001). Structure of the fiber head of Ad3, a non-CAR-binding serotype of adenovirus. Virology 285, 302–312.

Einfeld, D.A., and Roelvink, P.W. (2002). Advances towards targetable adenovirus vectors for gene therapy. Curr. Opin. Mol. Ther. 4, 444–451.

Einfeld, D.A., Brough, D.E., Roelvink, P.W., Kovesdi, I., and Wickham, T.J. (1999). Construction of a pseudoreceptor that mediates transduction by adenoviruses expressing a ligand in fiber or penton base. J. Virol. 73, 9130–9136.

Fender, P., Ruigrok, R.W., Gout, E., Buffet, S., and Chroboczek, J. (1997). Adenovirus dodecahedron, a new vector for human gene transfer. Nat. Biotechnol. 15, 52–56.

Furcinitti, P.S., van Oostrum, J., and Burnett, R.M. (1989). Adenovirus polypeptide IX revealed as capsid cement by difference images from electron microscopy and crystallography. EMBO J. 8, 3563–3570.

Gaggar, A., Shayakhmetov, D.M., and Lieber, A. (2003). CD46 is a cellular receptor for group B adenoviruses. Nat. Med. 9, 1408–1412. Published online October 19, 2003. 10.1038/nm952.

Gahery-Segard, H., Farace, F., Godfrin, D., Gaston, J., Lengagne, R., Tursz, T., Boulanger, P., and Guillet, J.G. (1998). Immune response to recombinant capsid proteins of adenovirus in humans: antifiber and anti-penton base antibodies have a synergistic effect on neutralizing activity. J. Virol. 72, 2388–2397.

Galinier, R., Gout, E., Lortat-Jacob, H., Wood, J., and Chroboczek, J. (2002). Adenovirus protein involved in virus internalization recruits ubiquitin-protein ligases. Biochemistry 41, 14299–14305.

Greber, U.F., Willetts, M., Webster, P., and Helenius, A. (1993). Stepwise dismantling of adenovirus 2 during entry into cells. Cell 75, 477–486.

Hess, M., Cuzange, A., Ruigrok, R.W., Chroboczek, J., and Jacrot, B. (1995). The avian adenovirus penton: two fibres and one base. J. Mol. Biol. 252, 379–385.

Holm, L., and Sander, C. (1993). Protein structure comparison by alignment of distance matrices. J. Mol. Biol. 233, 123–138.

Hong, S.S., and Boulanger, P. (1995). Protein ligands of the human adenovirus type 2 outer capsid identified by biopanning of a phage-displayed peptide library on separate domains of wild-type and mutant penton capsomers. EMBO J. 14, 4714–4727.

Jones, T.A., Zou, J.Y., Cowan, S.W., and Kjeldgaard (1991). Improved methods for building protein models in electron density maps and the location of errors in these models. Acta Crystallogr. A 47, 110–119.

Karayan, L., Gay, B., Gerfaux, J., and Boulanger, P.A. (1994). Oligomerization of recombinant penton base of adenovirus type 2 and its assembly with fiber in baculovirus-infected cells. Virology 202, 782–795.

Karayan, L., Hong, S.S., Gay, B., Tournier, J., d’Angeac, A.D., and Boulanger, P. (1997). Structural and functional determinants in adenovirus type 2 penton base recombinant protein. J. Virol. 71, 8678–8689.

Kim, Y.J., Hong, J.Y., Lee, H.J., Shin, S.H., Kim, Y.K., Inada, T., Hashido, M., and Piedra, P.A. (2003). Genome type analysis of adenovirus types 3 and 7 isolated during successive outbreaks of lower respiratory tract infections in children. J. Clin. Microbiol. 41, 4594–4599.

Kleywegt, G.J., and Jones, T.A. (1999). Software for handling macromolecular envelopes. Acta Crystallogr. D Biol. Crystallogr. 55, 941–944.

Kojaghlian, T., Flomenberg, P., and Horwitz, M.S. (2003). The impact of adenovirus infection on the immunocompromised host. Rev. Med. Virol. 13, 155–171.

Kolavic-Gray, S.A., Binn, L.N., Sanchez, J.L., Cersovsky, S.B., Polayk, C.S., Mitchell-Raymund, F., Asher, L.V., Vaughn, D.W., Feigner, B.H., and Innis, B.L. (2002). Large epidemic of adenovirus type 4 infection among military trainees: epidemiological, clinical, and laboratory studies. Clin Infect Dis 35, 808–818. Published online September 10, 2002.

Laskowski, R.A., MacArthur, M.W., Moss, D.S., and Thornton, J.M. (1993). PROCHECK: a program to check the stereochemical quality of protein structures. J. Appl. Crystallogr. 26, 283–291.

Leslie, A.G.W. (1992). Recent changes to the MOSFLM package for processing film and image plate data. Joint CCP4 + ESF-EAMCB Newsletter on Protein Crystallography 26.

Li, E., Stupack, D., Bokoch, G.M., and Nemerow, G.R. (1998). Adenovirus endocytosis requires actin cytoskeleton reorganization mediated by Rho family GTPases. J. Virol. 72, 8806–8812.

Li, E., Brown, S.L., Von Seggern, D.J., Brown, G.B., and Nemerow,

G.R. (2000). Signaling antibodies complexed with adenovirus circumvent CAR and integrin interactions and improve gene delivery. Gene Ther. 7, 1593–1599.

Medina-Kauwe, L.K. (2003). Endocytosis of adenovirus and adenovirus capsid proteins. Adv. Drug Deliv. Rev. 55, 1485–1496.

Medina-Kauwe, L.K., Maguire, M., Kasahara, N., and Kedes, L. (2001). Nonviral gene delivery to human breast cancer cells by targeted Ad5 penton proteins. Gene Ther. 8, 1753–1761.

Nakano, M.Y., Boucke, K., Suomalainen, M., Stidwill, R.P., and Greber, U.F. (2000). The first step of adenovirus type 2 disassembly occurs at the cell surface, independently of endocytosis and escape to the cytosol. J. Virol. 74, 7085–7095.

Namboodiri, V.M., Dutta, S., Akey, I.V., Head, J.F., and Akey, C.W. (2003). The crystal structure of Drosophila NLP-core provides insight into pentamer formation and histone binding. Structure (Camb.) 11, 175–186.

Nandhagopal, N., Simpson, A.A., Gurnon, J.R., Yan, X., Baker, T.S., Graves, M.V., Van Etten, J.L., and Rossmann, M.G. (2002). The structure and evolution of the major capsid protein of a large, lipid-containing DNA virus. Proc. Natl. Acad. Sci. USA 99, 14758–14763. Published online October 31, 2002. 10.1073/pnas.232580699

Nemerow, G.R. (2000). Adenoviral vectors—new insights. Trends Microbiol. 8, 391–394.

Nemerow, G.R., and Stewart, P.L. (1999). Role of alpha(v) integrins in adenovirus cell entry and gene delivery. Microbiol. Mol. Biol. Rev. 63, 725–734.

Norby, E. (1964). The relationship between the soluble antigens and the virion of adenovirus type 3. I. Morphological characteristics. Virology 1, 236–248.

Perrakis, A., Cipriani, F., Castagna, J.C., Claustre, L., Burghammer, M., Riekel, C., and Cusack, S. (1999). Protein microcrystals and the design of a microdiffractometer: current experience and plans at EMBL and ESRF/ID13. Acta Crystallogr. D Biol. Crystallogr. 55, 1765–1770.

Richardson, J.S. (1981). The anatomy and taxonomy of protein structure. Adv. Protein Chem. 34, 167–339.

Roelvink, P.W., Lizunova, A., Lee, J.G., Li, Y., Bergelson, J.M., Finberg, R.W., Brough, D.E., Kovesdi, I., and Wickham, T.J. (1998). The coxsackievirus-adenovirus receptor protein can function as a cellular attachment protein for adenovirus serotypes from subgroups A, C, D, E, and F. J. Virol. 72, 7909–7915.

Ruigrok, R.W., Barge, A., Albiges-Rizo, C., and Dayan, S. (1990). Structure of adenovirus fibre. II. Morphology of single fibres. J. Mol. Biol. 215, 589–596.

Rux, J.J., and Burnett, R.M. (2000). Type-specific epitope locations revealed by X-ray crystallographic study of adenovirus type 5 hexon. Mol. Ther. 1, 18–30.

Rux, J.J., Kuser, P.R., and Burnett, R.M. (2003). Structural and phylogenetic analysis of adenovirus hexons by use of high-resolution x-ray crystallographic, molecular modeling, and sequence-based methods. J. Virol. 77, 9553–9566.

Schoehn, G., Fender, P., Chroboczek, J., and Hewat, E.A. (1996). Adenovirus 3 penton dodecahedron exhibits structural changes of the base on fibre binding. EMBO J. 15, 6841–6846.

Segerman, A., Atkinson, J.P., Marttila, M., Dennerquist, V., Wadell, G., and Arnborg, N. (2003). Adenovirus type 11 uses CD46 as a cellular receptor. J. Virol. 77, 9183–9191.

Seth, P., Willingham, M.C., and Pastan, I. (1985). Binding of adenovirus and its external proteins to Triton X-114. Dependence on pH. J. Biol. Chem. 260, 14431–14434.

Sirena, D., Lilienfeld, B., Eisenhut, M., Kalin, S., Boucke, K., Beerli, R.R., Vogt, L., Ruedl, C., Bachmann, M.F., Greber, U.F., and Hemmi, S. (2004). The human membrane cofactor CD46 is a receptor for species B adenovirus serotype 3. J. Virol. 78, 4454–4462.

Stewart, P.L., Burnett, R.M., Cyrklaff, M., and Fuller, S.D. (1991). Image reconstruction reveals the complex molecular organization of adenovirus. Cell 67, 145–154.

Stewart, P.L., Fuller, S.D., and Burnett, R.M. (1993). Difference im-

aging of adenovirus: bridging the resolution gap between X-ray crystallography and electron microscopy. EMBO J. 12, 2589–2599.

Stewart, P.L., Chiu, C.Y., Huang, S., Muir, T., Zhao, Y., Chait, B., Mathias, P., and Nemerow, G.R. (1997). Cryo-EM visualization of an exposed RGD epitope on adenovirus that escapes antibody neutralization. EMBO J. 16, 1189–1198.

Tomko, R.P., Xu, R., and Philipson, L. (1997). HCAR and MCAR: the human and mouse cellular receptors for subgroup C adenoviruses and group B coxsackieviruses. Proc. Natl. Acad. Sci. USA 94, 3352–3356.

Vagin, A., and Teplyakov, A. (2000). An approach to multi-copy search in molecular replacement. Acta Crystallogr. D Biol. Crystallogr. 56, 1622–1624.

van Oostrum, J., and Burnett, R.M. (1985). Molecular composition of the adenovirus type 2 virion. J. Virol. 56, 439–448.

van Raaij, M.J., Mitraki, A., Lavigne, G., and Cusack, S. (1999). A triple beta-spiral in the adenovirus fibre shaft reveals a new structural motif for a fibrous protein. Nature 401, 935–938.

Vives, R.R., Lortat-Jacob, H., Chroboczek, J., and Fender, P. (2004). Heparan sulfate proteoglycan mediates the selective attachment and internalization of serotype 3 human adenovirus dodecahedron. Virology 321, 332–340.

Von Seggern, D.J., Chiu, C.Y., Fleck, S.K., Stewart, P.L., and Nemerow, G.R. (1999). A helper-independent adenovirus vector with E1, E3, and fiber deleted: structure and infectivity of fiberless particles. J. Virol. 73, 1601–1608.

Walters, R.W., Freimuth, P., Moninger, T.O., Ganske, I., Zabner, J., and Welsh, M.J. (2002). Adenovirus fiber disrupts CAR-mediated intercellular adhesion allowing virus escape. Cell 110, 789–799.

Wickham, T.J., Mathias, P., Cheresh, D.A., and Nemerow, G.R. (1993). Integrins alpha v beta 3 and alpha v beta 5 promote adenovirus internalization but not virus attachment. Cell 73, 309–319.

Wu, E., Pache, L., Von Seggern, D.J., Mullen, T.M., Mikyas, Y., Stewart, P.L., and Nemerow, G.R. (2003). Flexibility of the adenovirus fiber is required for efficient receptor interaction. J. Virol. 77, 7225–7235.

Wu, E., Trauger, S.A., Pache, L., Mullen, T.M., von Seggern, D.J., Siuzdak, G., and Nemerow, G.R. (2004). Membrane cofactor protein is a receptor for adenoviruses associated with epidemic keratoconjunctivitis. J. Virol. 78, 3897–3905.

Xia, D., Henry, L.J., Gerard, R.D., and Deisenhofer, J. (1994). Crystal structure of the receptor-binding domain of adenovirus type 5 fiber protein at 1.7 Å resolution. Structure 2, 1259–1270.

Young, L.S., and Mautner, V. (2001). The promise and potential hazards of adenovirus gene therapy. Gut 48, 733–736.

Zhang, Z., Merritt, E.A., Ahn, M., Roach, C., Hou, Z., Verlinde, C.L., Hol, W.G., and Fan, E. (2002). Solution and crystallographic studies of branched multivalent ligands that inhibit the receptor-binding of cholera toxin. J. Am. Chem. Soc. 124, 12991–12998.

Accession Numbers

The native penton base structure and complex with fiber peptide have PDB codes 1X9P and 1X9T, respectively.
