
Environmental Microbiology (2005) 7(2), 179–190

doi:10.1111/j.1462-2920.2004.00681.x

# Proposed involvement of a soluble methane monooxygenase homologue in the cyclohexane-dependent growth of a new Brachymonas species

Patricia C. Brzostowicz,$^{1}$ Dana M. Walters,$^{1}$ Raymond E. Jackson,$^{1}$ Kimberly H. Halsey,$^{2}$ Hao Ni$^{1}$ and Pierre E. Rouvière$^{1*}$

$^{1}$E. I. Dupont de Nemours and Company, Central Research and Development, Wilmington, DE 19880-0328, USA.

$^{2}$Department of Botany and Plant Pathology, Oregon State University, Corvallis, OR 97331, USA.

## Summary

High-throughput mRNA differential display (DD) was used to identify genes induced by cyclohexane in *Brachymonas petroleovorans* CHX, a recently isolated β-proteobacterium that grows on cyclohexane. Two metabolic gene clusters were identified multiple times in independent reverse transcription polymerase chain reactions (RT-PCR) in the course of this DD experiment. These clusters encode genes believed to be required for cyclohexane metabolism. One gene cluster (8 kb) encodes the subunits of a multicomponent hydroxylase related to the soluble butane of *Pseudomonas butanovora* and methane monooxygenases (sMMO) of methanotrophs. We propose that this butane monooxygenase homologue carries out the oxidation of cyclohexane into cyclohexanol during growth. A second gene cluster (11 kb) contains almost all the genes required for the oxidation of cyclohexanol to adipic acid. Real-time PCR experiments confirmed that genes from both clusters are induced by cyclohexane. The role of the Baeyer–Villiger cyclohexanone monooxygenase of the second cluster was confirmed by heterologous expression in *Escherichia coli*.

## Introduction

There is an increasing demand in the chemical industry for the design of ‘green’ processes, biotechnology routes that bypass the use of environmentally harsh chemical processes in favour of fermentation or biocatalysis-based routes with a smaller environmental footprint. Adipic acid (1,6-dicarboxy-hexane), a monomer of Nylon, is made chemically from cyclohexane. This high-temperature process involves high-pressure air and nitric acid oxidation steps and uses heavy metal catalysts (Weissermel and Arpe, 1993). We have been looking for a bioconversion route to the production of adipic acid and other long-chain α,ω-dicarboxylic acids, starting from relatively cheap cyclic alkanes. The proposed route to adipic acid starts with the hydroxylation of cyclohexane into cyclohexanol (Stirling *et al.*, 1977; Trower *et al.*, 1985a) followed by the oxidation of cyclohexanol into adipic acid through cyclohexanone, caprolactone, 6-hydroxycaproate and 6-oxo-caproate (Fig. 1). A bacterium capable of growing on cyclohexane should possess both the genes for the oxidation of cyclohexane to cyclohexanol as well as the genes for the oxidation of cyclohexanol to adipic acid (Donoghue and Trudgill, 1975). Bacteria growing on cyclic alcohols and ketones are easy to isolate, and the genes responsible for the degradation of these molecules have now been characterized in many organisms (Cho *et al.*, 1991; Iwaki *et al.*, 1999; 2002; Brzostowicz *et al.*, 2000; 2002; 2003; Cheng *et al.*, 2000; Kostichka *et al.*, 2001). In contrast, bacteria capable of growing on cyclic alkanes are difficult to isolate (De Klerk and Van der Linden, 1974; Walker and Colwell, 1976; Perry, 1984; Solano-Serena *et al.*, 2000a,b), and only a few bacteria that use cyclohexane as a carbon source have been described (Stirling *et al.*, 1977; Anderson *et al.*, 1980; Trower *et al.*, 1985a; Kästner *et al.*, 1994; Lang, 1996; van der Werf *et al.*, 2001). Until now, nothing was known about their relevant metabolic genes. We have followed the idea that specialized industrial wastewater bioreactors constitute a source of novel microorganisms for biotechnology (Bramucci and Nagarajan, 2000), and we recently isolated from petroleum refinery wastewater *Brachymonas petroleovorans* CHX, a new β-proteobacterium of the Comamonadaceae family (Rouvière and Chen, 2003). This strain grows on cyclohexane and a variety of cyclic alkanes and their substituted derivatives.

The goal of this work was to identify genes involved in the oxidation of cyclohexane. Because this strain is rather

Received 10 November, 2003; revised 10 May, 2004; accepted 17 May, 2004. *For correspondence.* E-mail pierre.e.rouviere@usa.dupont.com; Tel. (+1) 302 695 1782; Fax (+1) 302 695 1829.

© 2005 Society for Applied Microbiology and Blackwell Publishing Ltd

Cyclohexane
![Cyclohexane](https://i.imgur.com/1234567.png)

Cyclohexanol
![Cyclohexanol](https://i.imgur.com/8901234.png)

NAD(P)H NAD(P)+

Cyclohexanol
![Cyclohexanol](https://i.imgur.com/5678901.png)
Cyclohexanone
![Cyclohexanone](https://i.imgur.com/2345678.png)
ε-Caprolactone
![ε-Caprolactone](https://i.imgur.com/9012345.png)
6-Hydroxy-
hexanoate
![6-Hydroxy-hexanoate](https://i.imgur.com/6789012.png)
6-Oxohexanoate
![6-Oxohexanoate](https://i.imgur.com/3456789.png)
Adipic acid
![Adipic acid](https://i.imgur.com/0123456.png)

ChnA
ChnB
ChnC
ChnD
ChnE

Fig. 1. Degradation pathway of cyclohexane. The nomenclature for the enzymes responsible of the oxidation of cyclohexanol and cyclohexanone is that describe for *Acinetobacter* (Iwaki et al., 1999; Cheng et al., 2000). No genes or enzymes involved in the oxidation of cyclohexane for growth have been described before this work.

difficult to grow, it would be very difficult to isolate the corresponding enzymes from cell extracts and subsequently identify their genes by a reverse genetic approach. Similarly, because of the lack of genetic tools in this new isolate and its poor growth on plates, a direct genetic approach was not possible. We thus used the mRNA-based technique of differential display (DD) (Liang and Pardee, 1992) to search for the genes involved in the oxidation of cyclohexane.

Differential display and its variant, random arbitrarily primed polymerase chain reaction (RAP-PCR) (Welsh et al., 1992), involve the random, but reproducible amplification of DNA fragments from the total RNA of the cell, which constitutes a ‘sampling’ of the RNA population. By comparing the fragments obtained from the RNA of ‘control’ and challenged cells, one can identify the sequence of genes induced under specific physiological conditions. In our approach to DD, we performed many independent reverse transcription (RT)-PCRs, each initiated by a different arbitrary primer, leading to a high density of sampling of the mRNA population. While DD is reported to generate many false positives, we have observed that genes and operons sampled multiple times correspond to pathways truly induced (Brzostowicz et al., 2000; 2003; Walters et al., 2001). We have found this approach particularly useful for the elucidation of metabolic pathways.

Using this technique, we have identified two gene clusters likely to be involved in the oxidation of cyclohexane in *Brachymonas petroleovorans* CHX.

### Results

#### Induction of cyclohexane degradation genes in *Brachymonas petroleovorans* CHX

Differential display (DD) experiments aim to detect differences in the mRNA populations of organisms under two physiological states. It is essential to characterize precisely the conditions of induction of the genes to be studied. Growth of *B. petroleovorans* CHX on minimum cyclohexane medium is very poor and, in initial experiments, we were not able to detect an increase in respiration upon exposure of the cells to cyclohexane. In R2A medium, a moderately rich liquid medium, strain CHX grew more quickly under cyclohexane vapours, and induction of the cyclohexane degradation pathway could be demonstrated. As shown in Fig. 2, CHX cells grown in R2A medium and exposed to cyclohexane for 3 h increased their oxygen consumption upon addition of cyclohexane. Control cells grown in the liquid R2A media alone did not increase their rate of O₂ consumption upon addition of cyclohexane but did when acetate was added, indicating their viability (Fig. 2). Cells from the same culture.

Cyclohexane-induced culture

Control culture

Dissolved O₂ (%)

0    200   400   600   800   1000  1200

Time (s)

Fig. 2. Induction of cyclohexane oxidation. Oxygen consumption of cultures grown on acetate or cyclohexane was measured before and after addition of acetate or cyclohexane (indicated by arrows). The cyclohexane-exposed culture (top), but not the control culture (bottom), can oxidize cyclohexane indicating the inducibility of this pathway.

tures were used to compare the mRNA profile and to search for genes involved in the oxidation of cyclohexane.

Identification of cyclohexane-induced gene sequences by mRNA differential display

Total RNA was extracted from both the control and the cyclohexane-induced cultures and used as the template for 240 RT-PCRs, each using a single unique primer. The products of these reactions are DNA fragments amplified at arbitrary positions on the RNA molecules. These positions are determined by the approximate random match of the 3' end sequence of each primer with sequence on the RNA present in the total RNA population (Welsh et al., 1992). Each RT-PCR resulted in at least 20 DNA bands so that the complete experiment of 240 RT-PCRs generated ~5000 bands from each RNA template. The band patterns were very similar between the reactions using RNA from control and from cyclohexane-induced cells, indicating that the majority of the genes were expressed in similar fashion under the two physiological conditions (data not shown). Bands present in only one of the reactions are candidates for genes that are differentially expressed.

The 54 most clearly differentially amplified fragments (i.e. amplified from the RNA of induced cells but not from that of the control cells) were excised from the gel, reamplified and cloned. Our experience is that the repeated and independent sampling of individual genes or of functionally related genes is a strong indication that they are indeed induced under the physiological conditions studied. In contrast, false positives, i.e. fragments of protein-coding genes not induced but differentially amplified by RT-PCR, are never sampled multiple times (Brzostowicz et al., 2000; 2003; Walters et al., 2001; D. M. Walters and P. E. Rouvière, unpublished).

In this DD analysis, four contigs were assembled from the overlapping sequence of different RT-PCR fragments amplified in independent RT-PCRs. The genes in two of these clusters are likely to be involved in the oxidation of cyclohexane. The largest contig (1 kb) was assembled from the sequences of three differentially amplified DNA bands in three independent RT-PCRs. The corresponding gene is a homologue of the cyclohexanone monooxygenase gene of Acinetobacter sp. Sampling of a cyclohexanone monooxygenase gene indicated a successful induction of the genes for degradation of cyclohexane as cyclohexanone monooxygenase mediates the oxidation of cyclohexanone, a downstream intermediate in the pathway for biological degradation of cyclohexane (Stirling et al., 1977; Trower et al., 1985a) (Fig. 3). Cyclohexanone itself had not been added to the induction culture.

The other recognizable metabolic operon likely to have been sampled more than once includes the genes encoding homologues of two subunits of the butane monooxygenase (BMO) from 'Pseudomonas' butanovora and the soluble methane monooxygenase of methanotrophic bacteria (sMMO). The gene encoding a homologue for the α-subunit of the butane hydroxylase was sampled twice and the gene encoding the β-subunit once. While the substrate specificity of the butane monooxygenase of P. butanovora has not yet been reported, the sMMOs have long been known to have a broad substrate specificity and to oxidize many alkanes, alkenes, aromatics, heterocyclic and cycloalkanes including cyclohexane to their respective alcohols (Colby et al., 1977; Grosse et al., 1999). This gene cluster as well as the gene cluster carrying the cyclohexanone oxidation gene was studied further.

Two other contigs (0.6 and 0.9 kb) were assembled from the sequence of four and three RT-PCR fragments respectively. Both contigs encode fragments of a homologue of a putative surface protein of Rhodospirillum

© 2004 Society for Applied Microbiology and Blackwell Publishing Ltd, Environmental Microbiology, 7, 179–190

182 P. C. Brzostowicz et al.

Cyclohexane / "Butane" monooxygenase gene cluster

![Diagram of Cyclohexane / "Butane" monooxygenase gene cluster]

Fig. 3. Organization of the gene clusters proposed to be involved in cyclohexane oxidation. Functional assignment of the genes are presented in Tables 1 and 2. Genes proposed to be involved in cyclohexane oxidation are filled in grey. Solid black bars above the genes represent gene fragments identified in the differential display experiment. Top: gene cluster encoding a butane monooxygenase homologue. Genes were named *bmo* according to the closely related butane monooxygenase of 'Pseudomonas' butanovora, a bacterium that also hydroxylates cyclohexane to cyclohexanol. Bottom: genes encoding cyclohexanone oxidation. Genes were named according to the Acinetobacter nomenclature (Iwaki et al., 1999; Cheng et al., 2000). The gene encoding the cyclohexanol dehydrogenase was not found on the DNA fragment sequenced and possibly lies upstream of *orf1*.

rubrum (ZP00013646). Its induction probably corresponds to the modification of the cell envelope and the formation of aggregates in response to exposure of the cells to cyclohexane and other solvents (Rouvière and Chen, 2003). Finally, the gene encoding a large putative electron transfer protein with a 4Fe-4S and transmembrane domains was sampled in two RT-PCRs. The sequences of these two fragments do not overlap. This ferredoxin (homologous to ZP00021876.1) may be involved in providing electrons to the cyclohexane monooxygenase system but was not investigated further at this point.

Gene sequences for putative cyclohexane oxidation enzymes

The sequence of a 8.7 kb DNA fragment carrying the DD fragment homologous to the butane monooxygenase gene was determined (AY438629) (Fig. 4). This gene cluster contained open reading frames (ORFs) homologous to five of the *P. butanovora* butane monooxygenase genes (Table 1) that encode the structural genes of the α-chain (*bmoX*), β-chain (*bmoY*) and γ-chain (*bmoZ*) of the α₂β₂γ₂ hydroxylase component as well as component B (*bmoB*), a protein regulating the activity of the hydroxylase (Brazeau and Lipscomb, 2003), and component D (*bmoD*), a protein hypothetically involved in the assembly of the hydroxylase and the hydroxylase reductase (*bmoC*). The *Brachymonas bmoX*, *bmoY*, *bmoB*, *bmoZ*, *bmoD* and *bmoC* genes were found to be organized similarly to that of the butane monooxygenase genes in *P. butanovora* (Sluis et al., 2002) and that of the methane monooxygenases of *Methylcystis*, *Methylosinus*, *Methylococcus* and *Methylomonas* (Staintorpe et al., 1990; McDonald et al., 1997; Shigematsu et al., 1999) (Fig. 3). In contrast to the other gene clusters encoding members of the sMMO family such as that of *Methylosinus trichosporium* MmoR, the *Brachymonas* BMO cluster does not start with a transcription factor but with a transposase, which may indicate a recent origin of this gene cluster in *Brachymonas*.

Sequence conservation with the *P. butanovora* Bmo proteins was high (Table 1). It was maximal for the α-chain of the hydroxylase BmoX (90% amino acid identity, 95% amino acid similarity, 84% nucleotide identity) and decreased with BmoB, BmoY, BmoZ, BmoC and BmoD, as is the case in the rest of the sMMO family (Shigematsu)

![Phylogenetic tree showing sequence relationships]

Fig. 4. Sequence relationship between the predicted *Brachymonas* BMO large subunit and the large subunit of related multicomponent monooxygenases: *Methylcystis* sp. M sMMO (MmoX, AAC45289.1), *Methylosinus trichosporium* sMMO (MmoX, S15207), *Methylomonas* sp. KSWIII sMMO (MmoX, BAA84757), *Methylococcus capsulatus* (Bath) sMMO (MmoX, P22869); ‘*Pseudomonas*’ *butanovora* BMO (BmoX, AAM19727.1) and *Pseudomonas* C600 phenol oxidase (DmpN, 118694) (Nordlund et al., 1990). The scale bar represents 10% of amino acid difference. The relationship between the *Brachymonas* BmoY, BmoZ, BmoD and BmoC with their respective homologues is similar to the one presented here.

Table 1. Sequence similarity of ORFs of the butane monooxygenase homologue gene cluster.

| ORF   | Size (aa) | Top BLASTX hit                                                                                   | Accession no. | %ID | %Sim | E-value |
|-------|-----------|-------------------------------------------------------------------------------------------------|---------------|-----|------|---------|
| orf1  | 322       | ABC-type branched-chain amino acid transport system [Thermobifida fusca]                          | ZP00057121    | 33  | 42   | 0.006   |
| orf2  | 323       | Putative transposase ORF 4 [Delftia acidovorans]                                                  | BAB85586.1    | 80  | 89   | 1e-154  |
| bmoX  | 536       | Butane monooxygenase hydroxylase BMOH alpha subunit [Pseudomonas butanovora]                       | AAM19727.1    | 90  | 95   | 0.0     |
|       |           | Soluble methane monooxygenase protein A alpha subunit [Methylocystis sp.M]                        | AAC45289.1    | 63  | 78   | 0.0     |
| bmoY  | 391       | Butane monooxygenase hydroxylase BMOH beta subunit [Pseudomonas butanovora]                        | AAM19728.1    | 75  | 86   | 0.0     |
|       |           | Soluble methane monooxygenase protein A beta subunit [Methylocystis sp.M]                         | AAC45290.1    | 45  | 61   | 9e-95   |
| bmoB  | 138       | Butane monooxygenase regulatory protein [Pseudomonas butanovora]                                  | AAM19729.1    | 79  | 92   | 3e-61   |
|       |           | Soluble methane monooxygenase regulatory protein B [Methylococcus capsulatus]                     | M90050        | 51  | 74   | 2e-37   |
| bmoZ  | 168       | Butane monooxygenase hydroxylase BMOH gamma subunit [Pseudomonas butanovora]                       | AAM19730.1    | 67  | 80   | 2e-62   |
|       |           | Soluble methane monooxygenase hydroxylase component gamma subunit [Methylococcus capsulatus]       | AAF04157.2    | 37  | 58   | 6e-25   |
| bmoD  | 88        | Butane monooxygenase BMOD assembly protein [Pseudomonas butanovora]                                | AAM19731.1    | 57  | 77   | 5e-24   |
|       |           | Methane monooxygenase component D [Methylococcus capsulatus]                                      | P22867        | 30  | 53   | 0.040   |
| bmoC  | 364       | Butane monooxygenase reductase BMOc [Pseudomonas butanovora]                                       | AAM19732.1    | 61  | 74   | e-128   |
|       |           | Methane monooxygenase reductase component [Methylococcus capsulatus]                              | JQ701         | 38  | 58   | 3e-71   |

The genes encoding the putative cyclohexane hydroxylase and accessory proteins were named *bmo* and follow the X, Y, B, Z, D and C nomenclature of the methane and butane monooxygenase genes.

et al., 1999; Sluis et al., 2002). A neighbour-joining tree representing the phylogenetic relationship between the Brachymonas BmoX and its relatives is shown in Fig. 4. The topology of the trees derived from sequence comparisons for Bmo polypeptides is identical (data not shown).

The only structural difference between the *P. butanovora* and Brachymonas BMO clusters is in the intergenic region between *bmoZ* and *bmoD*. In Brachymonas, this region extends over 310 bp and does not show any ORF. In contrast, the *bmoZ/bmoD* intergenic region of *P. butanovora* encodes a small conserved hypothetical protein found in the genome sequences of *Pseudomonas fluorescens* (ZP 000887264.1) and *Mesorhizobium loti* (NC 002678.1).

Gene sequences for cyclohexanone oxidation enzymes

Cosmids carrying the cyclohexanone monooxygenase and the butane monooxygenase putative genes were isolated, and the sequence of the genes needed for oxidation of cyclohexanone to adipic acid was obtained from plasmid subclones. The assembled gene cluster (Fig. 3, accession number AY437859) contained ORFs coding for homologues of a cyclohexanone monooxygenase, a caprolactone hydrolase, a 6-hydroxycaproate dehydrogenase (Zn-dependent long-chain dehydrogenase family)

and 6-oxohexanoate dehydrogenase (aldehyde dehydrogenase) from *Acinetobacter*, *Brevibacterium*, *Arthrobacter* and *Rhodococcus* (Table 2). These ORFs were named according to the nomenclature established in *Acinetobacter* (Iwaki et al., 1999; Cheng et al., 2000). For two of the genes, *chnC* (caprolactone hydrolase) and *chnB* (cyclohexanone monooxygenase), the closest homologue was a gene demonstrated to be involved in cyclohexanone oxidation (Table 2). The gene of the Zn-dependent long-chain dehydrogenase homologue to the hydroxyproate dehydrogenase (*chnD*) (Cho et al., 1991; Iwaki et al., 1999; Cheng et al., 2000; Kostichka et al., 2001; Brzostowicz et al., 2003) had also been sampled in a single reaction in the differential display experiment (Fig. 3). The gene for a cyclohexanol dehydrogenase, a Zn-independent short-chain dehydrogenase in other organisms, was not present in the DNA fragment sequenced. It may be present upstream of the CoA-ligase gene or somewhere else on the chromosome.

An ORF encoding a large sigma 54-dependent transcription factor is divergently transcribed from the gene of the cyclohexanone monooxygenase (*chnB*). It is similar to the transcription factor located upstream of the cyclohexanone monooxygenase gene in *Brevibacterium HCU* (Brzostowicz et al., 2002) as well as in *Rhodococcus Phi2* (Brzostowicz et al., 2003) and was named *chnR*. No

Table 2. Sequence similarity of ORFs of the cyclohexanone oxidation gene cluster.

| ORF            | Size (aa) | Proposed function/top BLASTX hit                                                                 | Accession no. | %ID | %Sim | E-value |
|----------------|-----------|---------------------------------------------------------------------------------------------|---------------|------|-------|---------|
| orf1 (partial sequence) | -     | Fatty acid-CoA ligase [Oceanobacillus iheyensis]                                              | BAC 14626.1   | 53   | 73    | 1e-52   |
| chnE           | 484       | Aldehyde dehydrogenase [Bacillus cereus ATCC 14579]                                               | NP833288.1    | 47   | 62    | e-122   |
| chnD           | 369       | 5-exo-hydroxycamphor dehydrogenase [Pseudomonas putida]                                          | S34613        | 36   | 52    | 2e-50   |
|                |           | 6-Hydroxycaproate dehydrogenase ChnD [Acinetobacter sp. SE19]                                   | AAG 10028     | 29   | 43    | 3e-21   |
| chnC           | 302       | 6-Hexanolactone hydrolase [Xanthobacter flavus]                                                  | CAD 10800.1   | 44   | 60    | 7e-67   |
|                |           | 6-Hexanolactone hydrolase ChnC [Acinetobacter sp. NCIMB 9871]                                    | BAB 61745.1   | 41   | 62    | 2e-65   |
| chnB           | 537       | Cyclohexanone monooxygenase ChnB [Rhodococcus sp. Phi1]                                         | AAN 37494.1   | 62   | 77    | 0.0     |
| chnR           | 667       | Sigma 54-dependent transcription factor [Ralstonia metallidurans]                                | ZP00022513.1  | 42   | 60    | e-107   |
|                |           | Sigma 54-dependent transcription factor ChnR2 [Brevibacterium epidermidis HCU]                   | AAK 73166     | 24   | 38    | 4e-28   |
| orf7           | 528       | N-acetylmuramoyl-L-alanine amidase [Ralstonia solanacearum]                                       | CAD 16246.1   | 53   | 66    | 2e-62   |

genes can be identified in the large 770 bp intergenic region between chnB and chnR. This region contains a large 20 bp inverted repeat (CCTAGGAGGCCTGTTCG GACTT-129 bp-AAGTCCTACAGGCTCCTAGG) that is also found downstream of the AlkS regulator of the alkane oxidase of Pseudomonas putida Gpo1 OCT plasmid (AJ245436) and is possibly involved in the regulation of the chn genes. This long inverted repeat has no homology with the short inverted repeat recognized by AlkS. In strain CHX, the genes predicted to carry out the metabolic sequence of cyclohexanone to adipic acid oxidation are not arranged in a unidirectional fashion (Fig. 3). The esterase and monooxygenase lie head to head encoded by opposite strands. The order and the orientation of the genes in cyclohexanone oxidation gene clusters add to the variety of structural organization seen in other organisms (Cheng et al., 2002; Brzostowicz et al., 2003).

The biochemical function of the Baeyer-Villiger cyclohexanone monooxygenase was confirmed by expressing the chnB gene in Escherichia coli as described previously for other cyclohexanone monooxygenases (Brzostowicz et al., 2003). The purified enzyme was active with cyclohexanone as well as with a wide range of cyclic ketones. The best substrates were C5-C8 cyclic ketones and their 2-methyl derivatives, which were also among the best growth substrates for growth of B. petroleovorans CHX (Rouvière and Chen, 2003). This new Baeyer-Villiger monooxygenase also has a higher relative activity with larger substrates than the Acinetobacter enzymes such as tridecanone and cyclo-octanone (data not shown). The specific activity with cyclohexanone was 0.4 μmol of substrate min⁻¹ mg⁻¹ enzyme. This value is about one-tenth of that of other monooxygenase and probably reflects improper folding when heterologously expressed in E. coli

Both gene clusters are induced by cyclohexane

To confirm that both the putative cyclohexanone gene cluster and the cyclohexane monooxygenase (butane monooxygenase homologue) were indeed induced by cyclohexane, we measured the relative amount of mRNA present in control and cyclohexane-induced cells using real-time RT-PCR. We also performed this analysis with cells exposed to cyclohexanone as we had observed previously that cyclohexanone oxidation was induced by cyclohexanone (data not shown). We monitored the expression of one gene per cluster, the cyclohexanone monooxygenase gene chnB and the hydroxylase gene bmoX. By the time we performed these experiments, we had further characterized the growth conditions of B. petroleovorans by showing that growth is strongly stimulated using a rich medium, although it still requires an additional carbon source such as acetate, cyclohexane or cyclohexanone (Rouvière and Chen, 2003). The results of the real-time RT-PCR using RNA from these three cultures are presented in Table 3. The reproducibility of the real-time RT-PCR method was demonstrated by the very low standard deviation of Ct between triplicates (usually 0.1%, at most 0.5%). The 16S RNA used as an internal standard with relatively constant expression indeed showed less than 15% variation between different conditions of induction. Expression of the bmoX gene for the butane monooxygenase homologue, the putative cyclohexane monooxygenase, was indeed induced approximately.

Table 3. Quantification of induction of genes proposed to be involved in the oxidation of cyclohexane by real-time RT-PCR.

| Gene | Medium | chnB | bmoX | 16S RNA |
|------|--------|------|------|---------|
|      |        | Acetate | c-hexanone | c-hexane | Acetate | c-hexanone | c-hexane | Acetate | c-hexanone | c-hexane |
| Ct<sup>a</sup> |      | 33.754 | 27.054 | 28.506 | 26.052 | 24.325 | 22.314 | 17.049 | 17.075 | 16.846 |
| Standard deviation (SD) |      | 0.175 | 0.022 | 0.072 | 0.029 | 0.029 | 0.029 | 0.021 | 0.014 | 0.013 |
| Ct acetate-Ct |      | – | 6.7 | 5.248 | – | 1.727 | 3.738 | – | –0.026 | 0.203 |
| (Ct acetate-Ct)<sup>b</sup> |      | – | 104 | 38 | – | 3.3 | 13.3 | – | 0.98 | 1.15 |
| (Ct acetate-Ct) normalized to 16S<sup>c</sup> |      | – | 106 | 33 | – | 3.4 | 11.6 | – | – | – |

a. Ct, number of cycles when the fluorescence intensity generated in the RT-PCR reaches 10 standard deviations above the fluorescence background. Values represent the mean of triplicate RT-PCR amplifications.

b. Difference between the Ct values for the induction conditions of cyclohexanone or cyclohexane versus acetate.

c. Relative increase in mRNA between induction conditions of cyclohexanone or cyclohexane versus acetate. This number corresponds to the relative induction of the genes studied.

Discussion

In a search for a ‘green’ process to produce adipic acid, we have looked for cyclohexane-degrading bacteria that grow on cyclohexane. In the past, many groups have reported the failure to isolate such bacteria, and only a few strains have been isolated. These bacteria have been typed to several families known to be particularly proficient at degradation of xenobiotic compounds: high G+C Gram-positives (Nocardia) and Proteobacteria (Xanthobacter, Pseudomonas). The genes for the hydroxylation of cyclohexane are not known for any of these organisms. We recently isolated B. petroleovorans CHX, a β-proteobacterium that grows on cyclohexane (Rouvière and Chen, 2003) and, using mRNA differential display, we looked for the genes induced by cyclohexane.

In this DD analysis of cyclohexane oxidation, we have not reached the high multiplicity of sampling obtained in other experiments to identify the genes involved in picric acid degradation (Walters et al., 2001) or itaconate degradation (D. M. Walters and P. E. Rouvière, in preparation). This is probably due to the fact that, at the time we performed the DD analysis, we had not yet identified a medium that supported strong growth of the pure strain CHX. To generate biomass, cells had to be grown for 10 days on R2A plates before being resuspended in medium and exposed to cyclohexane under essentially non-growing conditions where mRNA levels are low. Nevertheless, despite the less than optimum conditions for the induction of the oxidation of cyclohexane, this DD experiment was successful, demonstrating again the utility of this technique for the genetic elucidation of physiological pathways. In this work, two gene clusters, out of at least three that were sampled multiple times, stood out in our analysis because they encode proteins very likely to be directly involved in the degradation of cyclohexane. These gene clusters were partially sequenced, and their induction by cyclohexane was confirmed by real-time RT-PCR.

The first gene cluster includes four of the five genes for the oxidation of cyclohexanone to adipic acid. The gene for the oxidation of cyclohexanol, which encodes a Zn-independent short-chain dehydrogenase in other organisms, was not present among the genes sequenced. The gene possibly lies upstream of the operon (Fig. 3) or, alternatively, it may be part of another metabolic operon as multiple short-chain dehydrogenases are often present in genomes and have broad range specificity. Expression of one of these genes, the cyclohexanone monooxygenase chnB gene, was induced 30-fold upon exposure of the cells to cyclohexane. The sampling of the cluster involved in the oxidation of the cyclohexanone is a confirmation that DD analysis worked.

The cyclohexane monooxygenase was cloned, expressed in E. coli and its activity with cyclohexanone was confirmed. This enzyme also oxidizes a variety of cyclohexanone derivatives, some with high enantiomeric specificity (P. C. Brzostowicz and P. E. Rouvière unpublished results; Kyte et al., 2004). Baeyer–Villiger monooxygenases are of great interest for biocatalysis because they carry out reactions on a broad range of substrates relevant for the synthesis of pharmaceutical intermediates and usually do so with a high enantiomeric specificity (Mihovilovic et al., 2001; Wang et al., 2003; Kyte et al., 2004). The new enzyme from Brachymonas CHX adds to

this set of enzymes and certainly increases their repertoire and applications.

The second gene cluster to stand out in our DD analysis (Fig. 3) encodes proteins closely related to the recently identified butane monooxygenase enzymatic complex from *P. butanovora* (Sluis et al., 2002) as well as the soluble monooxygenase of methanotrophic bacteria (Stainthorpe et al., 1990; Cardy et al., 1991; McDonald et al., 1997; Shigematsu et al., 1999). This hydroxylase system, which is induced 12-fold by cyclohexane, is a very strong candidate to carry out cyclohexane oxidation. The related soluble methane monooxygenases have long been known to oxidize many hydrocarbons and, in particular, to hydroxylate cyclohexane into cyclohexanol (Colby et al., 1977). Cell suspensions of methane-oxidizing bacteria were shown to oxidize cyclohexane into cyclohexanol (Imai et al., 1986). Cell suspensions of *P. butanovora* have also been shown to oxidize cyclohexane (K. Hasley and D. Arp, unpublished). It is thus expected that the *Brachymonas* butane monooxygenase will also carry out this reaction. The ability of *Brachymonas* to grow on cyclic alkanes may reflect the co-existence in the same organism of both a cyclic alkane hydroxylase and the enzymes responsible for the oxidation of cyclohexanone. Soluble methane monooxygenase and the butane monooxygenase of *P. butanovora* can also hydroxylate substituted cyclic alkanes, linear alkanes and toluene (Colby et al., 1977). The activity of the butane monooxygenase is possibly involved in the utilization of these growth substrates for *Brachymonas* CHX (Rouvière and Chen, 2003). This work provides the basis for the functional confirmation and characterization of the butane monooxygenase gene cluster in *B. petroleovorans* CHX.

The *Brachymonas* hydroxylase gene cluster is the second member of the Bmo branch within the sMMO family. Both BMOs are from β-proteobacteria, *P. butanovora* actually being a relative of *Thauera* (87% identity of 16S rDNA sequence with *Brachymonas*). The acquisition of the cyclohexane oxidation genes by the *Brachymonas* genome may be a relatively recent event. Unlike the *bmo* gene cluster of *P. butanovora* and the *mmo* genes of methanotrophs, no transcription regulator is found upstream of the *bmo* genes, but instead the transposase gene. The significant expression of the *bmo* operon in the absence of cyclohexane, as measured by real time RT-PCR of the *bmoX* gene, possibly reflects the fact that this set of genes has not completely evolved into a metabolic system dedicated to the degradation of cyclic alkanes.

The butane monooxygenase cluster is the only hydroxylating system that was identified in this DD analysis. Specifically, we did not encounter sequences encoding a cytochrome P450. We were initially expecting to identify this class of genes because a three-component cytochrome P450 hydroxylase was identified in a *Xanthobacter*

sp. as the cyclohexane hydroxylase (Trickett et al., 1991; Trower et al., 1985b; Warburton et al., 1990). This enzyme can oxidize numerous hydrocarbons including toluene, cyclohexane, cyclopentane and 1- and 3-methyl-cyclohexane to their corresponding alcohols (Trickett et al., 1991). This *Xanthobacter* sp. is the only cyclohexane-degrading organism in which cyclohexane hydroxylase has been purified and characterized. Activity measurement and spectral characterization in *Nocardia* (Stirling et al., 1977) and *Xanthobacter* (van der Werf et al., 2001) also suggest the involvement of a P450 hydroxylase. The genes for these enzymatic systems have not been identified.

The DD approach we have used in this work to study the degradation of cyclohexane in *B. petroleovorans* strain CHX has yielded the genes likely to code for almost the entire degradation pathway of cyclohexane to adipic acid. Further biochemical and physiological characterization of the soluble BMO and its regulation may shed light on why bacteria growing on cyclic alkanes are so rare in nature while the individual enzymatic systems capable of oxidizing cyclohexane to cyclohexanol and cyclohexanol into adipic acid are widespread. The identification of the cyclohexane oxidation genes will also allow the engineering of strains capable of converting cyclohexane to adipic acid that have better and more robust growth than *Brachymonas* CHX.

## Experimental procedures

### Induction of cyclohexane degradation genes

For the DD experiment, cells of *B. petroleovorans* CHX (Rouvière and Chen, 2003) were grown as a lawn on R2A plates (Difco) and, after 10 days ,were scraped, resuspended in 25 ml of R2A liquid medium and incubated at 30°C for 16 h. The culture was then diluted twofold with fresh R2A liquid broth [per litre: 0.5 g of yeast extract (Difco), 0.5 g of protease peptone (Difco), 0.5 g of casamino acids (Difco), 0.5 g of dextrose (Difco), 0.5 g of insoluble wheat starch, 0.3 g of sodium pyruvate, 0.3 g of potassium phosphate and 0.05 g of magnesium sulphate]. After 15 min of incubation at 30°C, the culture was split into two screw-cap Erlenmeyer flasks equipped with a Teflon-lined septum. One of the flasks received 5 μl of cyclohexane to allow for the induction of cyclohexane degradation genes. Both flasks were incubated with agitation for 3 h. At that point, the cultures were chilled on ice. Ten millilitres of each culture was immediately used to harvest total RNA, and 10 ml was kept on ice to test for induction of cyclohexane utilization by respirometry. The absorption of the cultures was 0.8 at 600 nm at the time of harvest.

For the respirometry experiment, cells of each culture described above were washed twice by centrifugation and resuspension in 10 ml of S12 mineral media [50 mM KHPO₄ (pH 7.0), 10 mM (NH₄)SO₄, 2 mM MgCl₂, 0.7 mM CaCl₂, 50 μM MnCl₂, 1 μM FeCl₃, 1 μM ZnCl₃, 1.72 μM CuSO₄, 2.53 μM CoCl₂, 2.42 μM Na₂MoO₂ and 0.0001% FeSO₄] (Brzostowicz et al., 2000). Five millilitres of each was placed in a water-jacketed respirometry cell equipped with an oxy-

gen electrode (Yellow Springs Instruments) at 30°C. After establishing the baseline for respiration (oxygen consumption) of each cell suspension, cyclohexane and/or acetate was added, and the rate of O₂ consumption was monitored further.

For the real-time RT-PCR experiments, cells were grown at 30°C for 2 days in 200 ml of S12 mineral medium complemented by 2.5 g l⁻¹ each yeast extract, casamino acid and proteose peptone (5× YECAPP) to stimulate growth and 10 mM acetate as a carbon substrate. Cells were washed by centrifugation in S12–1× YECAPP medium and resuspended in three Erlenmeyer flasks equipped with Teflon-lined screw-caps containing 20 ml of S12–1× YECAPP with acetate (10 mM), cyclohexanone (10 μl) or cyclohexane (10 μl). Cells were incubated at 30°C for 8 h.

### Isolation of total cellular RNA

Ten millilitres of the cyclohexane-induced and control cultures was harvested at 9400 g in a centrifuge chilled to –4°C. The supernatant was removed, and cells were resuspended in 2.5 ml of the RLT buffer from the RNeasy RNA purification kit (Qiagen). Cells were disrupted by adding 1.25 ml of culture to each of two tubes containing 200 μl of 0.5 mm zirconia beads (Biospec Products) and homogenizing at 2400 beats min⁻¹ for 4 min in a Bead Beater (Biospec). After cell disruption, the RNA was purified from cell extracts using the Qiagen RNeasy miniprep kit. RNA was treated with RNase-free DNase I (Ambion) for 1 h at 37°C. Polymerase chain reaction was used to test for residual DNA contamination with two primers used for differential display and described below. For real-time RT-PCR, total RNA was quantified spectrophotometrically by measuring the A₂₆₀.

### Generation of RT-PCR DNA fragments from arbitrarily reverse-transcribed total RNA

High-density sampling differential display analysis was performed as described earlier (Walters *et al.*, 2001; Brzostowicz *et al.*, 2003). A set of 240 primers with the sequence CGGAGCAGATCGAWXYZ, where WXYZ represent all the combinations of the three bases A, G and C minus the three primers forming the strongest primer dimers, was used in separate RT-PCRs with RNA from either cyclohexane-induced or control cells. The SuperScript one-step RT-PCR system (Life Technologies Gibco BRL) reaction mixture was used with 2–5 ng of total RNA, a 25 μl total reaction volume and the following temperature programme: 4°C (2 min), 5 min ramp to 37°C (1 h), followed by 95°C incubation (3 min), one cycle with 94°C (1 min), 40°C (5 min) and 72°C (5 min), 40 cycles with 94°C (1 min), 60°C (1 min) and 72°C (1 min), followed by an incubation at 70°C (5 min) and 4°C until separated by electrophoresis. RT-PCR DNA fragments generated with the same arbitrary primer from the RNA of control cells and of induced cells were analysed side by side by electrophoresis using precast polyacrylamide gels (Excel gels; Amersham Biosciences) and visualized by silver staining using an automated gel stainer (Amersham Biosciences).

### Reamplification of differentially amplified DNA fragments

A 25 μl volume of an elution buffer (10 mg ml⁻¹ NaCN, 20 mM Tris-HCl, pH 8.0, 50 mM KCl and 0.05% NP40) was incubated with an excised gel band of a differentially amplified fragment at 95°C for 20 min. Reamplification of this DNA fragment was achieved in a PCR using 5 μl of the elution mixture in a 25 μl reaction using the primer from which the fragment was originally generated. The temperature programme for reamplification was: 94°C (5 min), 20 cycles of 94°C (1 min), 55°C (1 min) and 72°C (1 min), followed by 72°C (7 min). The reamplification products were cloned directly into the pCR4-TOPO vector (Invitrogen) and were sequenced using an ABI model 377 with ABI BigDye terminator sequencing chemistry (Perceptive Biosystems). Eight clones were submitted for sequencing for each reamplified band, and all the resulting sequences were assembled in contigs using the SEQUENCHER software program (Gene Codes). Forty-two contigs were assembled ranging from 0.2 to 1.2 kb in size. The nucleotide sequence of the cloned fragments was compared with the non-redundant GenBank database using the BLASTX program (NCBI). Three contigs assembled from the sequence of 10 bands differentially amplified (18% of the bands) corresponded to 16S and 23S rRNA genes and were removed from our analysis.

### Cloning and sequencing genes needed for cyclohexane oxidation

A cosmid library was constructed from DNA isolated from *B. petroleovorans* CHX using the pWEB cosmid cloning kit (Epicentre Technologies). Cosmids were screened by PCR using primers designed against differentially displayed fragments homologous to the cyclohexanone monooxygenase and the butane monooxygenase α-subunit. Recombinant *E. coli* carrying the cosmid clones were used as the template with 1 μl of cell culture added to a 25 μl PCR mixture. Cosmids from recombinant *E. coli* that produced the correct product size in PCRs were isolated, digested partially with Sau3A1, and 10–15 kb fragments from this partial digest were subcloned into pSU19 (Martinez *et al.*, 1988), a pACYC184 derivative blue/white screening vector. These plasmid subclones were isolated using Qiagen Turbo96 miniprep kits and rescreened by PCR as described previously (Walters *et al.*, 2001). Subclones carrying the correct sequence fragment were subjected to in vitro transposition using the GPS-1 genome priming system kit (New England Biolabs). A number of these ‘transposed’ plasmids were sequenced from each end of the transposon. Sequence assembly was performed with the SEQUENCHER program (Gene Codes). Sequence analyses were performed with the BLASTX, BLASTP and CLUSTAL W programs. The sequences of these gene clusters were deposited in GenBank under the accession numbers AY438629 and AY437859.

### Quantification of RNA by real-time RT-PCR

To confirm that the metabolic operons identified by repeated sampling of the mRNA are indeed induced by cyclohexane, the abundance of the mRNA of the putative cyclohexane monooxygenase alpha subunit (*bmoX*) and of the cyclohex-

188 P. C. Brzostowicz et al.

anone monooxygenase (chnB) were quantified by real-time RT-PCR on an ABI TaqMan (Applied Biosystems).

For the chnB, bmoX and 16S RNAs, a forward and a reverse PCR primer were designed as well as a dual labelled probe containing a 5′ reporter dye (FAM, 6-carboxyfluorescein) and a 3′ quencher dye (TAMRA, 6-carboxy-N,N,N′,N′-tetramethylrhodamine). 16S rRNA quantification was used to normalize the data for differences in the amount of total RNA added to each reaction. The sequences of the primers are as follows: chnB forward primer, 5′-TCGTCGAAAGAATACTGATAGACATGA-3′; chnB reverse primer, 5′-CCTGGTATTGGAATCGCTATCC-3′; chnB probe, 5′-FAM-TGTGCGTGTCGGACAAGGCTCC-Tamra-3′; bmoX forward primer, 5′-AGCTCGATCAACCTGCAACTC-3′; bmoX reverse primer, 5′-GCGACGATCAGCGGATTG-3′, bmoX probe, 5′-FAM-TCGGCGAGGCCCTGCTTCA-TAMRA-3′, 16S rDNA forward, 5′-CCGCTCCCCATGGGTGTGA-3′, 16S rDNA reverse, 5′-TCAGCATGTCGCGGTGAAT-3′; 16S rDNA probe, 5′-FAM-CGGTTGTGTACAAGCCCGGGAACG TAMRA-3′. Primers and dual labelled probes were designed using PRIMER EXPRESS version 1.0 software (Applied Biosystems).

The probes were designed to anneal to the sequence between the two PCR primers. During the extension phase of the PCR, the 5′ to 3′ exonuclease activity of the Taq polymerase degrades the probe annealed to the PCR fragment being copied and liberates the reporter dye that can now be detected by its fluorescence. The increasing fluorescence, proportional to the amount of PCR fragment synthesized, is monitored in real time as the PCR proceeds. The point at which the measured fluorescence reaches a value of 10 standard deviations above background fluorescence is recorded as the cycle threshold (Ct) for each reaction. The ratio of the amounts of the specific mRNA present in two different samples is determined by the formula \(2^{-\Delta Ct}\). The three fragments were reverse transcribed and amplified with similar efficiency as demonstrated by the parallel plots of Ct as a function of the Log(RNA template) for each primer pair (data not shown). The ratio of the mRNA of each gene can be normalized further to that of the 16S RNA that was chosen for its constant expression under the conditions studied (Table 3).

Real-time RT-PCR was performed on an ABI 7900 sequence detection system instrument (PE Applied Biosystems), according to the manufacturer’s instructions. A one-step RT-PCR reaction (20 μl) was run using the following reagents: 10 μl of ABI TaqMan one-step RT-PCR master mix, 2 μl of each forward and reverse primers (10 μM), 0.05 μl of TaqMan probe (100 μM) and 4.5 μl of RNase-free water. One microlitre containing 0.04 ng of total RNA extracted from cells grown in S12 medium containing acetate, cyclohexanone or cyclohexane was added per reaction. The thermal cycling conditions used were: reverse transcription 30 min at 48°C, 10 min at 95°C for AmpliTaq gold activation followed by 40 cycles of 15 s at 95°C + 30 s at 60°C. A control lacking RNA template and a control without reverse transcriptase were run with each primer set. All reactions were run in triplicate.

### Expression of the Baeyer–Villiger monooxygenase

The chnB gene was amplified by PCR and cloned in the expression vector pTrcHis2-Topo (Invitrogen) to produce a C-terminal hexahistidine protein fusion. The enzyme was expressed in E. coli and purified on a Ni agarose column as described previously (Brzostowicz et al., 2003). Its activity was assayed by monitoring spectrophotometrically the oxidation of its co-substrate NADPH (Brzostowicz et al., 2003).

#### Acknowledgements

We thank D. Arp for facilitating K. Halsey’s preliminary experiments with *P. butanovora*.

#### References

Anderson, M.S., Hall, R.A., and Griffin, M. (1980) Microbial metabolism of alicyclic hydrocarbons: cyclohexane catabolism by a pure strain of *Pseudomonas* sp. *J Gen Microbiol* **120**: 89–94.

Bramucci, M.G., and Nagarajan, V. (2000) Industrial wastewater bioreactors: sources of novel microorganisms for biotechnology. *Trends Biotechnol* **18**: 501–505.

Brazeau, B.J., and Lipscomb, J.D. (2003) Key amino acid residues in the regulation of soluble methane monooxygenase catalysis by component B. *Biochemistry* **42**: 5618–5631.

Brzostowicz, P.C., Gibson, K.L., Thomas, S.M., Blasko, M.S., and Rouvière, P.E. (2000) Simultaneous identification of two cyclohexanone oxidation genes from an environmental *Brevibacterium* isolate using mRNA differential display. *J Bacteriol* **182**: 4241–4248.

Brzostowicz, P.C., Blasko, M.S., and Rouvière, P.E. (2002) Identification of two gene clusters involved in cyclohexanone oxidation in *Brevibacterium epidermidis* strain HCU. *Appl Microbiol Biotechnol* **58**: 781–789.

Brzostowicz, P.C., Walters, D.M., Thomas, S.M., Nagarajan, V., and Rouvière, P.E. (2003) mRNA differential display in a microbial enrichment culture: simultaneous identification of three cyclohexanone monooxygenases from three species. *Appl Environ Microbiol* **69**: 334–342.

Cardy, D.L., Laidler, V., Salmond, G.P., and Murrell, J.C. (1991) Molecular analysis of the methane monooxygenase (MMO) gene cluster of *Methylosinus trichosporium* OB3b. *Mol Microbiol* **5**: 335–342.

Cheng, Q., Thomas, S.M., Kostichka, K., Valentine, J.R., and Nagarajan, V. (2000) Genetic analysis of a gene cluster for cyclohexanol oxidation in *Acinetobacter* sp. Strain SE19 by in vitro transposition. *J Bacteriol* **182**: 4744–4751.

Cheng, Q., Thomas, S.M., and Rouviere, P. (2002) Biological conversion of cyclic alkanes and cyclic alcohols into dicarboxylic acids: biochemical and molecular basis. *Appl Microbiol Biotechnol* **58**: 704–711.

Cho, T., Takahashi, Y., and Yamamoto, S. (1991) Manufacture of adipic acid by biotechnology. *Bio Industry* **8**: 671–678.

Colby, J., Stirling, D.I., and Dalton, H. (1977) The soluble methane mono-oxygenase of *Methylococcus capsulatus* (Bath). Its ability to oxygenate n-alkanes, n-alkenes, ethers, and alicyclic, aromatic and heterocyclic compounds. *Biochem J* **165**: 395–402.

De Klerk, H., and Van der Linden, A.C. (1974) Bacterial degradation of cyclohexane. Participation of cooxidation

reaction. Antonie van Leeuwenhoek J Microbiol Serol 40: 7–15.

Donoghue, N.A., and Trudgill, P.W. (1975) The metabolism of cyclohexanol by Acinetobacter NCIB 9871. Eur J Biochem 60: 1–7.

Grosse, S., Laramee, L., Wendlandt, K.D., McDonald, I.R., Miguez, C.B., and Kleber, H.P. (1999) Purification and characterization of the soluble methane monooxygenase of the type II methanotrophic bacterium Methylocystis sp. strain WI 14. Appl Environ Microbiol 65: 3929–3935.

Imai, T., Takigawa, H., Nakagawa, S., Shen, G., Kodama, T., and Minoda, Y. (1986) Microbial oxidation of hydrocarbons and related compounds by whole cell suspensions of the methane-oxidizing bacterium H-2. Appl Environ Microbiol 52: 1403–1406.

Iwaki, H., Hasegawa, Y., Teraoka, M., Tokuyama, T., Bergeron, H., and Lau, P.C. (1999) Identification of a transcriptional activator (ChnR) and a 6-oxohexanoate dehydrogenase (ChnE) in the cyclohexanol catabolic pathway in Acinetobacter sp. strain NCIMB 9871 and localization of the genes that encode them. Appl Environ Microbiol 65: 5158–5162.

Iwaki, H., Hasegawa, Y., Wang, S., Kayser, M.M., and Lau, P.C. (2002) Cloning and characterization of a gene cluster involved in cyclopentanol metabolism in Comamonas sp. strain NCIMB 9872 and biotransformations effected by Escherichia coli-expressed cyclopentanone 1,2-monooxygenase. Appl Environ Microbiol 68: 5671–5684.

Kästner, M., Breuer-Jammali, M., and Mahro, B. (1994) Enumeration and characterization of the soil microflora from hydrocarbon-contaminated soil sites able to mineralize polycyclic aromatic hydrocarbons (PAH). Appl Microbiotechnol 41: 267–273.

Kostichka, K., Thomas, S.M., Gibson, K.J., Nagarajan, V., and Cheng, Q. (2001) Cloning and characterization of a gene cluster for cyclododecanone oxidation in Rhodococcus ruber SC1. J Bacteriol 183: 6478–6486.

Kyte, B.G., Rouvière, P., Cheng, Q., and Stewart, J.D. (2004) Assessing the substrate selectivities and enantioselectivities of eight novel Baeyer–Villiger monooxygenases toward alkyl-substituted cyclohexanones. J Org Chem 69: 12–17.

Lang, E. (1996) Diversity of bacterial capabilities in utilizing alkylated benzenes and other aromatic compounds. Lett Appl Microbiol 23: 257–260.

Liang, P., and Pardee, A.B. (1992) Differential display of eukaryotic messenger RNA by means of the polymerase chain reaction. Science 257: 967–971.

McDonald, I.R., Uchiyama, H., Kambe, S., Yagi, O., and Murrell, J.C. (1997) The soluble methane monooxygenase gene cluster of the trichloroethylene-degrading methanotroph Methylocystis sp. strain M. Appl Environ Microbiol 63: 1898–1904.

Martinez, E., Bartolome, B., and de la Cruz, F. (1988) pACYC184-derived cloning vectors containing the multiple cloning site and lacZ alpha reporter gene of pUC8/9 and pUC18/19 plasmids. Gene 68: 159–162.

Mihovilovic, M.D., Chen, G., Wang, S., Kyte, B., Rochon, F., Kayser, M.M., and Stewart, J.D. (2001) Asymmetric Baeyer–Villiger oxidations of 4-mono- and 4,4-disubstituted cyclohexanones by whole cells of engineered Escherichia coli. J Org Chem 66: 733–738.


Nordlund, I., Powlowski, J., and Shingler, V. (1990) Complete nucleotide sequence and polypeptide analysis of multicomponent phenol hydroxylase from Pseudomonas sp. strain CF600. J Bacteriol 172: 6826–6833.

Perry, J.J. (1984) Microbial metabolism of cyclic alkanes. In Petroleum Microbiology. Atlas, R.M. (ed.). New York, USA: Macmillan, pp. 61–67.

Rouvière, P.E., and Chen, M.W. (2003) Isolation of Brachymonas petroleovorans CHX, a novel cyclohexane-degrading β-proteobacterium. FEMS Microbiol Lett 227: 101–106.

Sayavedra-Soto, L.A., Byrd, C.M., and Arp, D.J. (2001) Induction of butane consumption in Pseudomonas butanovora. Arch Microbiol 176: 114–120.

Shigematsu, T., Hanada, S., Eguchi, M., Kamagata, Y., Kanagawa, T., and Kurane, R. (1999) Soluble methane monooxygenase gene clusters from trichloroethylene-degrading Methylomonas sp. strains and detection of methanotrophs during in situ bioremediation. Appl Environ Microbiol 65: 5198–5206.

Sluis, M.K., Sayavedra-Soto, L.A., and Arp, D.J. (2002) Molecular analysis of the soluble butane monooxygenase from ‘Pseudomonas butanovora’. Microbiology 148: 3617–3629.

Solano-Serena, F., Marchal, R., Casaregola, S., Vasnier, C., Lebeault, J.M., and Vandecasteele, J.P. (2000a) A Mycobacterium strain with extended capacities for degradation of gasoline hydrocarbons. Appl Environ Microbiol 66: 2392–2399.

Solano-Serena, F., Marchal, R., Lebeault, J.M., and Vandecasteele, J.P. (2000b) Selection of microbial populations degrading recalcitrant hydrocarbons of gasoline by monitoring of culture-headspace composition. Lett Appl Microbiol 30: 19–22.

Stainthorpe, A.C., Lees, V., Salmond, G.P., Dalton, H., and Murrell, J.C. (1990) The methane monooxygenase gene cluster of Methylococcus capsulatus (Bath). Gene 91: 27–34.

Stirling, L.A., Watkinson, R.J., and Higgins, I.J. (1977) Microbial metabolism of alicyclic hydrocarbons: isolation and properties of a cyclohexane-degrading bacterium. J Gen Microbiol 99: 119–125.

Trickett, J.M., Hammonds, E.J., Worrall, T.L., Trower, M.K., and Griffin, M. (1991) Characterization of cyclohexane hydroxylase; a three-component enzyme system from a cyclohexane-grown Xanthobacter sp. FEMS Microbiol Lett 82: 329–333.

Trower, M.K., Buckland, R.M., Higgins, R., and Griffin, M. (1985a) Isolation and characterization of a cyclohexane-metabolizing Xanthobacter sp. Appl Environ Microbiol 49: 1282–1289.

Trower, M.K., Buckland, R.M., and Griffin, M. (1985b) Characterization of a cyclohexane hydroxylase and a cyclohexanone monooxygenase from a cyclohexane-degrading Xanthobacter sp. Biochem Soc Trans 13: 463–464.

Walker, J.D., and Colwell, R.R. (1976) Measuring the potential activity of hydrocarbon-degrading bacteria. Appl Environ Microbiol 31: 189–197.

Walters, D.M., Russ, R., Knackmuss, H., and Rouvière, P.E. (2001) High-density sampling of a bacterial operon using mRNA differential display. Gene 273: 305–315.

190 P. C. Brzostowicz et al.

Wang, S., Kayser, M.M., Iwaki, H., and Lau, P.C.K. (2003) Monooxygenase-catalyzed Baeyer-Villiger oxidations: CHMO versus CPMO. J Mol Catal B: Enzymatic 22: 211–218.

Warburton, E.J., Magor, A.M., Trower, M.K., and Griffin, M. (1990) Characterization of cyclohexane hydroxylase; involvement of a cytochrome P-450 system from a cyclohexane grown Xanthobacter sp. FEMS Microbiol Lett 66: 5–9.

Weissermel, K., and Arpe, H.-J. (1993) Industrial Organic Chemistry. Weinheim: Verlagsgesellschaft, pp. 238–240.

Welsh, J., Chada, K., Dalal, S.S., Cheng, R., Ralph, D., and McClelland, M. (1992) Arbitrarily primed PCR fingerprinting of RNA. Nucleic Acids Res 20: 4965–4970.

van der Werf, M.J., Keijzer, P.M., and van der Schaft, P.H. (2001) Xanthobacter sp. C20 contains a novel bioconversion pathway for limonene. J Biotechnol 84: 133–143.
