
Molecular Microbiology (2005) 56(4), 971–989

doi:10.1111/j.1365-2958.2005.04602.x

# Biosynthesis of the red antibiotic, prodigiosin, in Serratia: identification of a novel 2-methyl-3-n-amyl-pyrrole (MAP) assembly pathway, definition of the terminal condensing enzyme, and implications for undecylprodigiosin biosynthesis in Streptomyces

**Neil R. Williamson,**¹ **Henrik T. Simonsen,**²  
**Raef A. A. Ahmed,**² **Gabrielle Goldet,**² **Holly Slater,**¹  
**Louise Woodley,**¹ **Finian J. Leeper**² **and**  
**George P. C. Salmond**¹*

Departments of ¹Biochemistry and ²Chemistry, University of Cambridge, Cambridge, UK.

## Summary

The biosynthetic pathway of the red-pigmented antibiotic, prodigiosin, produced by *Serratia* sp. is known to involve separate pathways for the production of the monopyrrole, 2-methyl-3-n-amyl-pyrrole (MAP) and the bipyrrrole, 4-methoxy-2,2′-bipyrrrole-5-carbaldehyde (MBC) which are then coupled in the final condensation step. We have previously reported the cloning, sequencing and heterologous expression of the *pig* cluster responsible for prodigiosin biosynthesis in two *Serratia* sp. In this article we report the creation of in-frame deletions or insertions in every biosynthetic gene in the cluster from *Serratia* sp. ATCC 39006. The biosynthetic intermediates accumulating in each mutant have been analysed by LC-MS, cross-feeding and genetic complementation studies. Based on these results we assign specific roles in the biosynthesis of MBC to the following Pig proteins: PigI, PigG, PigA, PigJ, PigH, PigM, PigF and PigN. We report a novel pathway for the biosynthesis of MAP, involving PigD, PigE and PigB. We also report a new chemical synthesis of MAP and one of its precursors, 3-acetyloctanal. Finally, we identify the condensing enzyme as PigC. We reassess the existing literature and discuss the significance of the results for the biosynthesis of undecylprodigiosin by the Red cluster in *Streptomyces coelicolor* A3(2).

---

## Introduction

Gram-negative bacteria of the genus *Serratia* are opportunistic human, plant and insect pathogens and are members of the Enterobacteriaceae. Some strains of *Serratia marcescens* cause nosocomial infections that are clinically problematic because multidrug resistance is widespread within the species (Traub, 2000; Stock *et al.*, 2003).

Some *Serratia* strains produce prodigiosin, which is a red linear tripyrrole antibiotic and a member of the prodiginines (Fig. 1). Other members of the prodiginines include undecylprodigiosin and a cyclic derivative, butyl-meta-cycloheptylprodiginine produced by the red cluster of the actinomycete *Streptomyces coelicolor* A3(2) (Tsao *et al.*, 1985; Cerdeño *et al.*, 2001).

Prodigiosin is typical secondary metabolite only appearing in the later stages of bacterial growth (Williams *et al.*, 1971). The production of prodigiosin has been shown to be influenced by numerous environmental factors including: inorganic phosphate availability, media composition, temperature and pH (Williams *et al.*, 1971; Williams and Quadri, 1980; Sole *et al.*, 1994; Slater *et al.*, 2003). The biosynthesis of prodigiosin in *Serratia* sp. ATCC 39006 is controlled by a complex regulatory network of both N-acyl-L-homoserine lactone quorum-sensing-dependent and -independent pathways (Thomson *et al.*, 1997; 2000; Slater *et al.*, 2003; Fineran *et al.*, 2005). Prodigiosin has no defined role in the physiology of producing strains. A recent study has suggested prodigiosin may be an important factor for the trypanolytic activity of a *S.* marcescens strain (Azambuja *et al.*, 2004). Other suggested roles for prodigiosin have included a role in metabolic overflow from primary metabolism (Bu’Lock, 1961; Hood *et al.*, 1992).

Prodigiosin is of potential clinical interest because it is reported to have anti-fungal, anti-bacterial, anti-protozoal/anti-malarial, immunosuppressive and anti-cancer activities (Williams and Quadri, 1980; Tsuji *et al.*, 1990; 1992; Demain, 1995; Mortellaro *et al.*, 1999; D’Alessio *et al.*, 2000; Montaner *et al.*, 2000). D’Alessio *et al.* (2000).

---

Accepted 2 February, 2005. *For Correspondence.* E-mail gpcs@mole.bio.cam.ac.uk; Tel. (+44) 1223 333 650; Fax (+44) 1223 766 108.

© 2005 Blackwell Publishing Ltd

972 N. R. Williamson et al.

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

Fig. 1. Structures of some members of the prodiginines and key prodigiosin biosynthesis intermediates. Prodigiosin and undecylprodigiosin are formed by the condensation of 4-methoxy-2,2′-bipyrrrole-5-carbaldehyde (MBC) and either 2-methyl-3-n-amyl-pyrrole (MAP) or 2-undecylpyrrole respectively. Also shown are the structures of butyl-meta-cycloheptylprodigiosine, a cyclic derivative of undecylprodigiosin, and two analogues of MAP: 2,4-dimethyl-3-ethylpyrrole (DEP) and 2,4-dimethylpyrrole (DMP).

patented a prodigine compound 4-benzylxyo-5-[(5-undecyl-2H-pyrrol-2-ylidene)methyl]-2,2′-bi-1H-pyrrole PNU-156804 as it exhibits promising immunosuppressive activity and reduced toxicity. Cámpos et al. (2003) have shown that prodigiosin also induces apoptosis of human primary cancer cells in B-cell chronic lymphocytic leukaemia samples. The different modes of action of prodigiosin on human cells have recently been illustrated (Pérez-Tomás et al., 2003).

A bifurcated pathway has been proposed for the biosynthesis of prodigiosin culminating in the enzymic condensation of the terminal products of the two pathways, 4-methoxy-2,2′-bipyrrrole-5-carbaldehyde (MBC) and 2-methyl-3-n-amyl-pyrrole (MAP) (Harris et al., 2004). Although the condensation reaction can occur spontaneously at low pH, in bacteria this reaction occurs by a temperature-sensitive enzymatic reaction (Rapoport and Holden, 1962; Williams et al., 1965; Fürstner, 2003). The precursors for prodigiosin were shown to be acetate, serine, alanine, methionine and proline (Williams, 1973).

A detailed pathway for the biosynthesis of undecylprodigiosin in Str. coelicolor A3(2) has been proposed and roles for many of the genes of the red cluster suggested, based mainly on predicted functions of the genes (Cerdeño et al., 2001). The mechanism of proline incorporation into one of the pyrrole rings of MBC was then confirmed biochemically (Cerdeño et al., 2001; Thomas et al., 2002). The monopyrrole precursor (2-undecylpyrrole) of undecylprodigiosin is different from the monopyrrole (MAP) of prodigiosin although the MBC moiety is common to both.

Recently we have reported the cloning, sequencing and heterologous expression of the pig cluster responsible for the biosynthesis of prodigiosin in two separate Serratia species, one of them being strain ATCC 39006 (Harris et al., 2004). Initially it was thought that Serratia sp. ATCC

39006 was a *S. marcescens* strain (Thomson et al., 1997) but it is now clear that this strain is atypical and does not fall into the *marcescens* classification. We now refer to this strain by the ATCC nomenclature, strain 39006 (Thomson et al., 1997; Harris et al., 2004). The *Serratia* 39006 *pig* cluster consists of 15 biosynthesis genes (*pigA–O*) transcribed as a single polycistronic mRNA (Slater et al., 2003; Harris et al., 2004). We have previously reported that inactivation of *pigO* did not affect the production of prodigiosin and thus it seems that PigO is not directly involved in the biosynthesis of prodigiosin (Harris et al., 2004). Furthermore, we tentatively proposed a prodigiosin biosynthetic pathway for strain 39006 of *Serratia*, based on homology with the *red* genes and their proposed roles. In this article we report a detailed investigation of the biosynthetic pathway of prodigiosin. In-frame deletion or insertion mutations were created in every biosynthetic gene and for each mutant the accumulating biosynthetic intermediates were analysed by LC-MS, biochemical cross-feeding and genetic complementation. We determine a role for each Pig enzyme and propose a new pathway for the biosynthesis of prodigiosin. We also describe the identification of the condensing enzyme. Finally, based on our new pathway and a reassessment of prior literature in the field, we discuss the implications of this study for the biosynthesis of other prodiginines, in particular undecylprodigiosin from the Gram-positive *Str. coelicolor* A3(2).

Results

LC-MS of wild-type *Serratia* 39006 and previously described *pig* mutants

To confirm that *Serratia* 39006 produced prodigiosin, pigment was extracted from an 18 h culture of the strain

© 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **56**, 971–989

The biosynthetic pathway of prodigiosin in Serratia 973

hts133
%
hts273
%
hts256
%
hts237
%

Prodigiosin
Norprodigiosin
Norprodigiosin
HBC
MBC
MAP

1: Scan ES+
TIC
3.56e8
39006
1: Scan ES+
TIC
3.84e7
Sma OF
1: Scan ES+
TIC
2.40e8
pigDΔ
1: Scan ES+
TIC
6.63e7
pigH-H57

Time

Fig. 2. LC-MS chromatograms of wild-type Serratia 39006 and the mutants Sma OF, NW13 and NW8. Prodigiosin, norprodigiosin, HBC, MBC and MAP are all seen as clearly identified peaks. The number given shows the predominant $m/z$ signal (M + H⁺ ion) in each peak.

grown at 30°C in Luria–Bertani (LB) medium supplemented with 0.25 M sorbitol. LB medium supplemented with 0.25 M sorbitol has been shown to cause hyperpigmentation in wild-type Serratia 39006 (L. Everson, pers. comm.). This approach was used for the LC-MS studies of all mutants. Extracts from both the cell pellet and the supernatant of the culture of wild-type Serratia 39006 contained a major compound identical to authentic prodigiosin by both LC-MS (Fig. 2) and MS/MS. High resolution mass spectrometry (HRMS) gave $m/z$ 324.2071 confirming the molecular formula is that of prodigiosin. This confirms, for the first time, that the red pigment produced by Serratia 39006 is indeed prodigiosin.

Previously published pig mutants were also tested in our LC-MS and HRMS systems. We found the S. marcescens OF mutant (Sma OF) to produce norprodigiosin and the S. marcescens WF (Sma WF) produces MAP as reported in the literature (Fig. 2) (Morrison, 1966). For the mutant S. marcescens 933 (Sma 933) we could only detect very low levels of 4-hydroxy-2,2′-bipyrrole-5-methanol (HBM) and not the previously described MBC. This probably results from additional mutations that could have occurred subsequent to the original mutation and which have affected this particular strains ability to produce MBC. The extracts of Sma OF (norprodigiosin) and Sma WF (MAP), pure prodigiosin and chemically synthesized MAP were used as standards throughout the work.

Marker exchange mutagenesis of the pigA–pigN genes

In-frame mutagenesis of all the pigA–pigN genes was performed using either in-frame linker mutagenesis or deletion of the putative biosynthetic genes. In vitro EZ::TN in-frame linker mutagenesis was carried out on pTON245 and pNRW25 and the mutagenized plasmids were used to transform into electrocompetent Escherichia coli DH5α and kanamycin resistance transformants selected. The kanamycin-resistant plasmids were purified and location of the insertion determined by restriction digestion and sequencing. Once a plasmid had been selected the kanamycin marker was removed by restriction digestion with NotI and the plasmid was then self-ligated, which resulted in an in-frame 57 bp insertion. This approach yielded 57 bp (19 codon) in-frame insertions in the following pig genes: pigF, pigH (×3), pigI, pigJ, pigL and pigM (Table 1). These mutated alleles were cloned into pKNG101 with at least 500 bp of homology flanking either side of the target gene to be inactivated, and the resulting plasmids (Table 1) were used to marker exchange Serratia 39006. Marker exchange colonies were initially screened by colony polymerase chain reaction (PCR) with primers outside the region of recombination and the corresponding PCR products were digested with NotI to confirm presence of the introduced NotI restriction site.

© 2005 Blackwell Publishing Ltd, Molecular Microbiology, 56, 971–989

Table 1. Bacterial strains and plasmids used in this study.

| Strain/plasmid | Genotype/phenotype | Source or reference |
|-----------------|--------------------|---------------------|
| Escherichia coli |  |  |
| CC118λpir | araD, Δ(ara, leu), ΔlacZ74, phoA20, galK, thi-1, rspE, rpoB, argE, recA1, λpir | Herrero et al. (1990) |
| DH5α | F⁻, φ80ΔlacZM15, Δ(lacZYA-argF)U169, endA1, recA1, hsdR17 (r<sub>k</sub>⁻m<sub>k</sub>⁻), deoR, thi-1, supE44, λ⁻gyrA96, relA1 | GibcoBRL, Life Technologies |
| ER2566 | F⁻ λ⁻ fhuA2 [lon] ompT lacZ::T7 gene1 gal sulA11Δ(mcrC-mrr)114::IS10R(mcr-73::miniTn10–TetS)2 R(zgb-210::Tn10)(TetS), endA1 [dcm] | NEB |
| HH26 | Mobilizing strain for conjugal transfer | Kaniga et al. (1991) |
| Serratia |  |  |
| ATCC 39006 | Wild type (Car⁺, Pig⁺) | Bycroft et al. (1987) |
| S. marcescens 933 | MBC-producing strain<sup>a</sup> | Morrison (1966) |
| S. marcescens OF | Norprodigiosin-producing strain | Morrison (1966) |
| S. marcescens WF | MAP-producing strain | Morrison (1966) |
| Mutants derived from ATCC 39006 |  |  |
| NW3 | In-frame pigH::57 bp (14699) | This study |
| NW4 | In-frame pigH::57 bp (14837) | This study |
| NW5 | In-frame pigF::57 bp (13117) | This study |
| NW6 | In-frame pigEΔ (10321–12865, 848 aa Δ) | This study |
| NW8 | In-frame pigH::57 bp (15366) | This study |
| NW9 | In-frame pigL::57 bp (16592) | This study |
| NW10 | In-frame pigM::57 bp (21151) | This study |
| NW11 | In-frame pigL::57 bp (20835) | This study |
| NW13 | In-frame pigDΔ (7895–9446, 517 aa Δ) | This study |
| NW14 | In-frame pigBΔ (3270–4773, 501 aa Δ) | This study |
| NW16 | In-frame pigJ::57 bp (18430) | This study |
| NW18 | In-frame pigKΔ (19992–20193, 67 aa Δ) | This study |
| NW19 | In-frame pigCΔ (4910–7508, 877 aa Δ) | This study |
| NW20 | In-frame pigMΔ (20965–21980, 338 aa Δ) | This study |
| NW21 | In-frame pigGΔ (13963–14149, 62 aa Δ) | This study |
| NW22 | In-frame pigN::57 bp (22196) | This study |
| NW23 | In-frame pigAΔ (1747–2830, 361 aa Δ) | This study |
| TS08 | pigN::miniTn5/lacZ1, Km<sup>R</sup> | This study |
| Plasmids |  |  |
| pBluescriptII-KS+ | ColE1 replicon, Ap<sup>R</sup> | Stratagene |
| pGEX-2T | GST fusion expression vector, Ap<sup>R</sup> | Amersham biosciences |
| pKNG101 | Sm<sup>R</sup>, sacBR, mobRK2, oriR6K | Kaniga et al. (1991) |
| pUC18 | ColE1 replicon, Ap<sup>R</sup> | Yanisch-Perron et al. (1985) |
| pUC19 | ColE1 replicon, Ap<sup>R</sup> | Yanisch-Perron et al. (1985) |
| pKM24 | pigH::EZ::TN derivative of pTON245 (14699) Km<sup>R</sup>, Tc<sup>R</sup> | This study |
| pKM47 | pigF::EZ::TN derivative of pTON245 (13117) Km<sup>R</sup>, Tc<sup>R</sup> | This study |
| pKM56 | pigH::EZ::TN derivative of pTON245 (14837) Km<sup>R</sup>, Tc<sup>R</sup> | This study |
| pKM57 | pigH::EZ::TN derivative of pTON245 (15366) Km<sup>R</sup>, Tc<sup>R</sup> | This study |
| pKM18 | pigN::EZ::TN derivative of pTON245 (22196) Km<sup>R</sup>, Tc<sup>R</sup> | This study |
| pLKM1 | pigM::EZ::TN derivative of pNRW25 (21151) Km<sup>R</sup>, Ap<sup>R</sup> | This study |
| pLKM2 | pigJ::EZ::TN derivative of pNRW25 (18430) Km<sup>R</sup>, Ap<sup>R</sup> | This study |
| pLKM28 | pigL::EZ::TN derivative of pNRW25 (16592) Km<sup>R</sup>, Ap<sup>R</sup> | This study |
| pLKM38 | pigL::EZ::TN derivative of pNRW25 (20835) Km<sup>R</sup>, Ap<sup>R</sup> | This study |
| pLKM<sup>S</sup>1 | pigM::57 bp (21,151), Km<sup>S</sup>, Ap<sup>R</sup> derivative of pLKM1 | This study |
| pLKM<sup>S</sup>2 | pigJ::57 bp in-frame insertion (18,430), Km<sup>S</sup>, Ap<sup>R</sup> derivative of pLKM2 | This study |
| pLKM<sup>S</sup>28 | pigL::57 bp (bp), Km<sup>S</sup>, Ap<sup>R</sup> derivative of pLKM28 | This study |
| pLKM<sup>S</sup>38 | pigL::57 bp (20835), Km<sup>S</sup>, Ap<sup>R</sup> derivative of pLKM38 | This study |
| pNRW16 | pigH::57 bp (14699), Km<sup>S</sup>, Tc<sup>R</sup> derivative of pKM24 | This study |
| pNRW17 | SalI/HindIII 5′ and 3′HindIII/BamHI pigE ligated into pbluescriptSK | This study |
| pNRW18 | pigH::57 bp (14837), Km<sup>S</sup>, Tc<sup>R</sup> derivative of pKM56 | This study |
| pNRW21 | BamHI/SalI pigEΔ from pNRW17 ligated into pKNG101 | This study |
| pNRW22 | pigF::57 bp (13117), Km<sup>S</sup>, Tc<sup>R</sup> derivative of pKM47 | This study |
| pNRW24 | pigH::57 bp (15366), Km<sup>S</sup>, Tc<sup>R</sup> derivative of pKM57 | This study |
| pNRW25 | 8.4 kb EcoRI fragment of pTON245 (′pigH–pigO′) ligated into EcoRI-digested pUC18 | This study |
| pNRW27 | 12.7 kb EcoRI/PstI fragment of pTON245 (5′pigA-pigE′) ligated into EcoRI/PstI-digested pUC19 | This study |
| pNRW33 | 3.5 kb Scal/BamHI fragment of pNRW27 (′pigA-pigC′) ligated into Small/BamHI-digested pUC19 | This study |
| pNRW34 | 1.5 kb Tthl digestion of pNRW33, klenow blunt end filled and self-ligated, pigBΔ (3270–4773), Ap<sup>R</sup> | This study |

The biosynthetic pathway of prodigiosin in Serratia 975

Table 1. cont.

| Strain/plasmid | Genotype/phenotype | Source or reference |
|----------------|--------------------|---------------------|
| pNRW35         | 8.4 kb HindIII fragment of pNRW27 ('pigB–pigE') ligated into HindIII-digested pBluescript SK | This study |
| pNRW36         | NruI/Ndel-digested pNRW35 Klenow filled and self-ligated, pigDΔ (7895–9446), Ap<sup>R</sup> | This study |
| pNRW40         | 2.5 kb BamHI fragment of pNRW36 ligated into BamHI-digested pKNG101, pigDΔ, Ap<sup>R</sup> | This study |
| pNRW41         | 0.85 kb Hincll–EcoRV fragment of pNRW34 ligated into Smal-digested pKNG101 | This study |
| pNRW42         | 1.9 kb SalI fragment of pNRW16 containing pigH::57 bp ligated into pKNG101 | This study |
| pNRW43         | 1.9 kb SalI fragment of pNRW18 containing pigH::57 bp ligated into pKNG101 | This study |
| pNRW44         | 1.0 kb BamHI fragment of pNRW22 containing pigF::57 bp ligated into pKNG101 | This study |
| pNRW45         | 6.2 kb BamHI fragment of pNRW24 (pigH57::57 bp) ligated into BamHI-digested and dephosphorylated pKNG101 | This study |
| pNRW46         | 4.7 kb BamHI–SalI fragment of pNRW containing pigN::57 bp ligated into pKNG101 | This study |
| pNRW47         | 3.2 kb SalI fragment of pLKM<sup>S</sup>28 containing pigI::57 bp ligated into pKNG101 | This study |
| pNRW48         | 2.0 kb BamHI–Hpal fragment of pLKM<sup>S</sup>2 containing pigJ::57 bp ligated into pKNG101 | This study |
| pNRW49         | 3.6 kb BamHI fragment of pLKM<sup>S</sup>38 containing pigL::57 bp ligated into pKNG101 | This study |
| pNRW50         | 4.7 kb SalI fragment of pLKM<sup>S</sup>1 containing pigM::57 bp ligated into pKNG101 | This study |
| pNRW51         | 1.5 kb SalI–Spel fragment containing pigKΔ ligated into pKNG101 | This study |
| pNRW52         | 1.5 kb SalI–Spel fragment containing pigMΔ ligated into pKNG101 | This study |
| pNRW53         | 2.0 kb SalI–Apal fragment containing pigGΔ ligated into pKNG101 | This study |
| pNRW54         | 1.1 kb SalI–Spel fragment containing pigCΔ ligated into pKNG101 | This study |
| pNRW55         | 2.6 kb EcoRI fragment containing pigC ligated into pGEX-2T | This study |
| pNRW58         | 1.5 kb SalI–Spel fragment containing pigAΔ ligated into pKNG101 | This study |
| pTON245        | pHCP19 vector, containing the Pig cluster (24 kb) from Serratia 39006 | Harris et al. (2004) |
| pNJ5000        | Mobilizing plasmid used in marker exchange, Tc<sup>R</sup> | Grinter (1983) |

a. Sma 933 has previously been stated to produce MBC; however, we could only detect low levels of HBM in this particular strain. The position of all insertions/deletions is shown in the brackets and is based on Serratia 39006 pig cluster numbering (Accession No. AJ833001).

The second approach to pig gene inactivation was to create in-frame deletions by the use of suitable in-frame restriction sites (pigBΔ and pigDΔ). Where there was no suitable restriction sites available, in-frame deletions were generated by PCR amplification of 5′ and 3′ regions flanking the target gene containing appropriate restriction sites. The digested flanking regions were ligated together such that they could only ligate into the vector in one orientation and the resulting introduced internal restriction site was in frame with the target gene. When designing primers for this approach particular care was taken to avoid affecting upstream or downstream neighbouring genes. In-frame deletions were generated in this manner for pigA, pigC, pigE, pigG, pigK and pigM genes. In each case the mutated allele was cloned in pKNG101 and used to marker exchange the chromosome of Serratia 39006. The genotype and in-frame nature of every mutant was confirmed by sequencing the region of recombination. Each biosynthetic mutation was complemented by the introduction of a functional copy of the corresponding mutated gene by a wild-type version supplied in trans on a plasmid (N. R. Williamson, H. T. Simonsen, F. J. Leeper and

G. P. C. Salmond, unpublished results). In most cases the complemented mutant was confirmed to produce prodigiosin as shown by LC-MS. This result confirms rigorously the in-frame nature of each mutation and excludes the possibility of secondary mutation(s) elsewhere in the genome affecting the observed phenotype.

Analysis of accumulating intermediates in each pig mutant

The phenotype of the 501 aa pigBΔ mutant (NW14) was white on peptone glycerol media (PGM) and was shown to produce 4-hydroxy-2,2′-bipyrrole-5-carbaldehyde (HBC), MBC and a compound with MW 154.1 (M + H⁺ ion) by LC-MS. HRMS gave m/z 154.1598, corresponding to a molecular formula of C₁₀H₂₀NO, the same as MAP plus two hydrogen atoms (Table 2). To confirm the nature of the compound an extract of NW14 was reduced with NaBH₃CN, a reducing agent specific for imine groups (C=N). This resulted in the disappearance of the LC-MS peak of mass 154.1 and the appearance of a new peak of mass 156.2. This confirmed the presence of an imine in the unknown compound. We therefore conclude that the

© 2005 Blackwell Publishing Ltd, Molecular Microbiology, 56, 971–989

Table 2. Analysis of the Serratia pig mutants.

| Genotype          | Strain   | Phenotype as seen on PGM | Compounds determined by LC-MS                          | Cross-feeds knockoutmutants of the following genes                                                                 | Cross-fed by knockout mutants of the following genes, MAP$^b$ and 3-acetyloctanal$^b$                                                                 |
|-------------------|----------|-------------------------|------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
| $pigA\Delta$      | NW23     | White                   | ND                                                   | D, E                                                                                                              | B, C, D, F, K, M, Sma OF                                                                                             |
| $pigB\Delta$      | NW14     | White                   | H$_2$MAP, HBC, MBC                                   | D, E, G, H, I, J, M, Sma WF                                                                                      | C, F, G, H, I, J, K, L, M, N, Sma WF, Sma OF, MAP                                                                   |
| $pigC\Delta$      | NW19     | White                   | MAP, HBC, HBM, MBC                                   | B, D, E, F, G, H, I, J, L, M, Sma WF                                                                             |                                                                                                                       |
| $pigD\Delta$      | NW13     | White                   | HBC, HBM, MBC                                        | F, G, H, J, M, Sma WF                                                                                            | B, C, F, G, H, J, K, M, Sma WF, Sma OF, MAP, 3-acetyloctanal                                                         |
| $pigE\Delta$      | NW6      | White                   | HBC, HBM, MBC                                        |                                                                                                                  | B, C, F, G, H, I, J, K, M, N, Sma OF, MAP                                                                           |
| $pigF::57$ bp     | NW5      | Orange                  | NPG                                                  | B, D, E, G, H, I, J, M, Sma WF                                                                                   | C                                                                                                                     |
| $pigG\Delta$      | NW21     | White                   | MAP                                                  | B, D, E                                                                                                          | B, C, D, F, K, M, Sma OF                                                                                             |
| $pigH-H24$        | NW3      | Light pink              | PG                                                   | B, D, E                                                                                                          | B, C, D, F, K, M, Sma OF                                                                                             |
| $pigH-H56$        | NW4      | Pink                    | NPG, PG 1:40                                         | B, D, E                                                                                                          | B, C, D, F, K, M, Sma OF                                                                                             |
| $pigH-H57$        | NW8      | White                   | MAP                                                  | B, D, E                                                                                                          | B, C, D, F, K, M, Sma OF                                                                                             |
| $pigI::57$ bp     | NW9      | White                   | MAP                                                  | B, D, E                                                                                                          | B, C, D, F, K, M, Sma OF                                                                                             |
| $pigJ2::57$ bp    | NW16     | White                   | MAP                                                  | B, D, E                                                                                                          | B, C, D, E, K, M, Sma OF                                                                                             |
| $pigK\Delta$      | NW18     | Red                     | NPG, PG – 1:10                                       | B, D, E, G, H, I, J, SmaWF                                                                                       |                                                                                                                       |
| $pigL::57$ bp     | NW11     | Light pink              | NPG, PG – 3:2$^a$                                    | B, D, E                                                                                                          | C, Sma OF                                                                                                             |
| $pigM::57$ bp     | NW10     | Pink                    | HBC, HBM, MBC, NPG, PG                               | B, D, E, G, H, I, J                                                                                              | B, C, D, F                                                                                                            |
| $pigM\Delta$      | NW20     | White                   | MAP, HBM                                             | B, D, E, G, H, I, J                                                                                              | B, C, D, F                                                                                                            |
| $pigN::57$ bp     | NW22     | Orange                  | NPG, PG – 15:1                                       | B, E                                                                                                             |                                                                                                                       |
| $pigN::miniTn5lacZ1$ | TSO8    | Orange                  | NPG, PG – 2:3                                        | B, E                                                                                                             | B, C, D, F, K, M                                                                                                      |
|                   | Sma WF   | White                   | MAP                                                  | B, D, E                                                                                                          |                                                                                                                       |
|                   | Sma 933  | White                   | HBM$^a$                                              |                                                                                                                  |                                                                                                                       |
|                   | Sma OF   | Orange                  | NPG                                                  | B, D, E, G, H, I, J, L, M, Sma WF                                                                                |                                                                                                                       |
| Wild type         | Serratia 39006 | Red                   | NPG, PG 1:40                                         | None                                                                                                             | None                                                                                                                  |

a. The overall amount produced by this mutant is at least 10 times lower than the production of the wild type of Serratia 39006.
b. Synthetically made MAP and 3-acetyloctanal. See Experimental procedures for the new synthesis pathway.

The different strains were analysed by cross-feeding, LC-MS and the feeding of certain intermediates. The following compounds were detected by LC-MS: prodigiosin (PG), norprodigiosin (NPG), HBC, HBM, MBC, MAP and H$_2$MAP. ND, not determined.

Fig. 3. A. The biosynthetic pathway of prodigiosin. The dashed arrows indicate that HBC and MAP can condense to form norprodigiosin.
B. The proposed biosynthetic pathway of undecylprodigiosin catalysed by enzymes encoded by the Red cluster. Each colour represents a different initial precursor. Incorporation into either prodigiosin or undecylprodigiosin and their precursors is shown by the same colour.

© 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **56**, 971–989

ACP domain in PigH is necessary for the biosynthesis of prodigiosin to proceed. The NW8 strain could not be complemented biochemically by the addition of MAP and therefore PigH must act in the biosynthesis of MBC.

The *pigM*::57 bp mutant (NW10) was white on LBA and pink when grown on PGM. LC-MS analysis revealed the presence of prodigiosin. Because of the apparently leaky nature of this in-frame insertion, an in-frame deletion of *pigM* was also generated. The *pigMΔ* mutant (NW20) was white on PGM and produced HBM and MAP when grown in LB sorbitol. This result is consistent with the view that *pigM*::57 bp is a leaky mutation and the accumulation of MAP and HBM in the *pigMΔ* mutant indicates PigM facilitates the conversion of HBM to HBC.

Colonies of the final mutant strain NW18 (*pigKΔ*) appeared very similar to wild-type Serratia 39006 on PGM and the strain was shown to produce predominantly prodigiosin.

---

Fig. 4. Cross-feeding mutants on PGM agar.
A. *pigBΔ* mutant (NW14) on the left cross-feeding *pigDΔ* mutant (NW13) on the right.
B. Reciprocal cross-feeding between *pigI*::57 bp mutant (MBC pathway) on the left and *pigDΔ* (MAP pathway) on the right.
C. *pigCΔ* mutant (centre) cross-feeding *pigBΔ* mutant on the left and *pigF*::57 bp mutant on the right.
D. *E. coli* DH5α pNRW55 (centre) expressing a GST–PigC fusion protein condensing MAP and MBC excreted by Sma WF and the *pigBΔ* mutant (NW14) to form prodigiosin.

which catalyses the transfer of a methyl group from SAM to the hydroxyl of HBC to give MBC. The other two norprodigiosin producing strains TS08 and NW22 (*pigN*::57 bp) carried insertions in *pigN*. The two *pigN* insertion mutants produce norprodigiosin and prodigiosin in strikingly different ratios, 15:1 and 2:3 for NW22 and TS08 respectively (Table 2).

The NW21 (*pigGΔ*), NW9 (*pigI*::57 bp) and NW16 (*pigJ2*::57 bp) strains were white, could not be complemented by the exogenous addition of MAP, and produce MAP as shown by LC-MS. The NW23 (*pigAΔ*) strain was white and could not be complemented by the exogenous addition of MAP. These results indicated that PigA, PigG, PigI and PigJ act in the biosynthesis of MBC.

Each of the three *pigH*::57 bp mutants has a distinct phenotype when grown on PGM. Only strain NW8 (with a 19 codon insertion in the aminotransferase domain) was white and it produced MAP and no prodigiosin. The NW3 strain contains an insertion close to the second predicted ACP domain of *pigH* (conserved residue, Ser-139) and NW4 contains an insertion which is located between the two ACP domains and the aminotransferase domain. Strain NW3 was light pink and produced less prodigiosin than the wild type whereas strain NW4 is not significantly different from the wild type. This suggests that only one

Assigning each Pig protein to a particular step in the biosynthesis of prodigiosin

In conjunction with the LC-MS analysis cross-feeding studies were performed to confirm the order of each step in the biosynthesis of prodigiosin. All cross-feeding studies were performed on PGM agar incubated at 30°C for 2 days. PGM was used in the early studies of the prodigiosin biosynthetic pathway because of enhanced production of prodigiosin and its precursors (Morrison, 1966). Table 2 summarizes the results of the cross-feeding experiments. Cross-feeding between different biosynthetic mutants could occur in two ways. First, an intermediate that accumulates in a mutant blocked late in either pathway could cross-feed a mutant that was blocked earlier in the same pathway. Second, a mutant blocked in the biosynthesis of MAP or MBC would accumulate the intermediate of the unaffected pathway and thus be capable of cross-feeding a mutant defective in the opposing pathway.

Cross-feeding analysis of *pigBΔ*, *pigDΔ* and *pigEΔ* mutants all blocked in the biosynthesis of MAP showed that the *pigBΔ* strain could cross-feed both *pigDΔ* and *pigEΔ* mutants (Fig. 4). This result shows that PigB must act later in the MAP pathway than PigD and PigE and thus PigB is the terminal enzyme in this pathway. *pigDΔ* and *pigEΔ* strains cannot cross-feed each other. The *pigDΔ* mutant does cross-feed mutants blocked in the biosynthesis of MBC, whereas the *pigEΔ* strain does not.

All mutants blocked in the biosynthesis of MBC, with the exception of those defective in *pigN* can cross-feed mutants blocked in the biosynthesis of MAP. The *pigBΔ* and *pigDΔ* mutants can reciprocally cross-feed all MBC mutants, with the exception of the *pigF*::57 bp mutant and the mutants with insertions in *pigN*. The lack of cross-
feeding activity by *pigN* mutants may result from the inability of these mutants to allow intermediates to diffuse (or be transported) in or out of the cell. Also, the majority of free MAP and HBC is probably converted into norprodigiosin in these mutants. PigF is clearly involved in the terminal step of the biosynthesis of MBC because the corresponding mutant can cross-feed all other MBC biosynthesis mutants. The *pigMΔ* strain can cross-feed mutants defective in *pigA*, *pigG*, *pigH*, *pigI* or *pigJ*, but not *pigF*, and therefore PigM is the penultimate enzyme in the biosynthesis of MBC. Mutants with the following genes inactivated can only cross-feed mutants blocked in the biosynthesis of MAP: *pigA*, *pigG*, *pigH*, *pigI* and *pigJ*. Cross-feeding may not occur in these cases if the accumulating intermediate is unstable, unable to diffuse or be transported, or if shunted into an alternative pathway by the bacterial cell. As has been mentioned earlier, the Red homologues of *pigI*, *pigG* and *pigA* have been shown to be involved in the first steps of MBC biosynthesis (incorporation of proline). Another explanation for the lack of cross-feeding of intermediates from *pigI*, *pigG*, *pigA*, *pigJ* and *pigH* is that the intermediates are tethered to either PigG or PigH and therefore cannot diffuse out of the cell.

Finally, although LC-MS analysis showed that the *pigKΔ* strain produced predominantly prodigiosin it was capable of cross-feeding *pigAΔ*, *pigBΔ*, *pigDΔ*, *pigEΔ*, *pigGΔ*, *pigH::57 bp*, *pigI::57 bp* and *pigJ::57 bp* mutants whereas the wild-type strain cannot. Crucially the *pigKΔ* strain cannot cross-feed the *pigMΔ* mutant. These results suggest that the deletion of *pigK* may affect the flux of intermediates through the MBC pathway before the involvement of PigF. This indicates that PigK is not directly involved in the biosynthesis of an intermediate but may have an ancillary role late in the biosynthesis of MBC.

To confirm that cross-feeding of various mutants actually reinstates prodigiosin production, the *pigH::57 bp* mutant (NW8) and *pigDΔ* mutant (NW13) were used as starter cultures to co-inoculate 200 ml of LB containing 0.25 M sorbitol. This mixed culture was grown at 30°C for 18 h and allowed the reciprocal co-synthesis of prodigiosin which was confirmed by LC-MS. We repeated this co-synthesis experiment with a mixed culture of NW14 (*pigBΔ*) and NW13 (*pigDΔ*) and again prodigiosin biosynthesis was observed by LC-MS analysis. In this latter experiment only the *pigDΔ* mutant would have produced prodigiosin.

### LC-MS of one mutant fed with MAP and the same mutant fed with DEP

By feeding DEP after 18 h of growth to a *pigDΔ* mutant it was shown that DEP can be incorporated, giving 2,4-dimethyl-3-ethyl-prodiginine as a product. This was confirmed by both LC-MS and HRMS data. The same

---

The biosynthetic pathway of prodigiosin in Serratia 979

experiment was also performed with MAP and this, as expected, gave prodigiosin production.

### Heterologous expression of MAP in E. coli

To confirm the involvement of PigB–PigE in the biosynthesis of MAP a 12.7 kb EcoRI/PstI fragment of pTON245 containing the upstream region of the *pig* cluster and the *pigA–pigE* genes was cloned into pUC19. This ligation created pNRW27 and the genes were expressed from the constitutive *lac* promoter on pUC19. *E. coli* DH5α and *E. coli* ER2566 transformed with pNRW27 were streaked next to *pigBΔ* (MAP⁻, MBC⁺) on PGM and incubated at 30°C for 2 days. After 2 days both *E. coli* strains containing pNRW27 were pigmented red, whereas *E. coli* pUC19 was not. The red pigment extracted from cells scraped off the PGM agar exhibited the characteristic adsorption spectrum of prodigiosin in acidified ethanol (N. R. Williamson, H. T. Simonsen, F. J. Leeper and G. P. C. Salmond, unpublished). Thus, heterologous expression of *pigA–pigE* in *E. coli* (DH5α or ER2566) is sufficient for the biosynthesis of MAP and condensation with MBC to form prodigiosin. We have already assigned PigA to the biosynthesis of MBC, and PigC as the condensing enzyme. We therefore conclude that just three enzymes, PigB, PigD and PigE, are sufficient for the biosynthesis of MAP. Furthermore, as the pNRW27 construct encodes a slightly truncated PigE protein with a 32 amino acid residue C-terminal deletion, our results show that this deletant retains catalytic activity.

### Discussion

Our results assign a function to all the products of the genes in the *pig* cluster with the exception of PigK. Although the *pigKΔ* mutant produces prodigiosin at levels not significantly different from those of wild-type Serratia 39006, this strain is capable of cross-feeding certain *pig* mutants, whereas the wild type cannot. These observations suggest that, although PigK may not be directly involved in biosynthesis it may have some role mediating flux of the intermediates through the pathway. We therefore speculate that PigK may have a role as a molecular chaperone assisting the folding of one or more of the Pig enzymes involved in the later stages of MBC biosynthesis. We cannot yet assign a specific biochemical function to PigN, although this enzyme clearly acts in the MBC pathway. As seen in Fig. 3A, this study has revealed a new pathway for the biosynthesis of MAP. This pathway clearly differs from the proposed biosynthesis of the corresponding monopyrrole in undecylprodigiosin (2-undecylpyrrole) in both the chemistry and proteins required for biosynthesis (Cerdeño *et al.*, 2001). Our studies have also been enabled by our new chemical synthesis route to MAP and

© 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **56**, 971–989
980 N. R. Williamson et al.

3-acetyloctanal, and allowed production of the biosynthetic intermediates in the MAP pathway.

# Biosynthesis of MBC

The results described in this article show that the biosynthesis of MBC in *Serratia* sp. 39006 follows the same general chemical pathway as previously predicted (Cerdeño *et al.*, 2001; Harris *et al.*, 2004). However, our new data have encouraged a reassessment of these predictions and so some of the enzymes catalysing the conversion of these intermediates have now changed from those predicted (Harris *et al.*, 2004).

The results in this study confirm the involvement of PigI, PigG and PigA in the early steps of MBC biosynthesis, because the corresponding mutants can be cross-fed by strains NW20 (*pigMΔ*) and NW5 (*pigF*:57 bp). We therefore propose that PigI, PigG and PigA act in the manner we have suggested previously (Fig. 3A) and as *in vitro*, for their respective Red homologues (RedO, RedM and RedW) (Cerdeño *et al.*, 2001; Thomas *et al.*, 2002; Harris *et al.*, 2004). In support of this hypothesis, this mechanism of L-proline incorporation and a similar set of proteins are common to many pyrrole-containing compounds, including undecylprodigiosin, pyoluteorin, coumermycin A1, novobiocin and chlorobiocin (Cerdeño *et al.*, 2001; Thomas *et al.*, 2002; Xu *et al.*, 2002). Although Thomas *et al.* (2002) clearly defined the function of RedO and RedM, there was doubt as to whether RedW performed one or both oxidations to convert the prolyl group to a pyrrole. PigA, the *pig* cluster homologue of RedW, shows homology with NikD, which is involved in the biosynthesis of the nikkomycins, and Bruckner *et al.* (2004) have shown that NikD is responsible for two successive 2-electron oxidation steps of tetrahydropicolinate to give picolinate (Bruckner *et al.*, 2004). We propose that PigA also catalyses two successive 2-electron oxidation steps using bound FAD and that the same kind of chemistry is performed by RedW, which has been purified as a monomer, and the other enzymes involved in 4-electron oxidations of pyrrolidine or tetrahydropyridine rings using FAD as electron acceptor (Thomas *et al.*, 2002).

The chemistry of the next step in the biosynthesis of MBC, polyketide-type chain extension of the prolyl-ACP starter unit, remains the same as previously predicted but is now shown to involve PigJ and PigH. We had previously been unable to assign an enzyme to this ketosynthase reaction and had assigned PigJ and PigH to the biosynthesis of MAP based on their sequence identity with RedX and RedN (Cerdeño *et al.*, 2001; Harris *et al.*, 2004). It was concluded in our earlier paper that the two aminotransferases PigE and PigH might each act in the opposite pathway to the one initially predicted. Mutations in both *pigJ* and *pigH* abolish MBC biosynthesis and cannot be complemented by the exogenous addition of MAP. Either of these mutations are cross-fed by NW6 (*pigF*:57 bp) and NW20 (*pigMΔ*) mutants and thus the respective enzyme defects must occur early in MBC biosynthesis.

PigJ, like its homologue RedX, is predicted to contain two keto synthase-like domains. The C-terminal domain has the essential active-site cysteine (Cys-525) but the N-terminal domain has the cysteine replaced by an aspartate (Asp-197) and is presumed to be catalytically silent. This is common in polyketide syntheses, although often the two domains are separate proteins, as in actinorhodin biosynthesis. The catalytically silent domain or protein is often called the chain-length factor (CLF) as it is thought to regulate how many cycles of chain extension occur (Tang *et al.*, 2003; Keatinge-Clay *et al.*, 2004). However, another hypothesis is that it is involved in the decarboxylation of malonyl units to provide the acetyl starter units (Bisang *et al.*, 1999). In the biosynthesis of MBC the pyrrole-2-carboxyl unit attached to PigG is the starter unit not acetate, so any decarboxylase activity would not be relevant. Classical polyketide syntheses also require an acyl carrier protein (ACP) and there are two predicted ACP domains near the N-terminus of PigH. Therefore, the reaction sequence envisioned is: (i) transfer of the pyrrole-2-carboxyl unit from PigG to the active-site cysteine of PigJ, (ii) transfer of a malonyl group from malonyl CoA to the phosphopantetheinyl side-chain of one of the ACP domains of PigH and (iii) decarboxylative attack of the malonyl unit on the pyrrole-2-carboxyl thioester giving a pyrrolyl-β-ketothioester attached to PigH.

Polyketide synthases often terminate with a thioesterase domain that hydrolyses or cyclizes the final thioester intermediate to release it from the enzyme. In PigH there is no thioesterase domain but, instead, the C-terminal domain exhibits sequence identity with pyridoxal phosphate (PLP)-dependent enzymes, particularly 8-amino-7-oxononanoate synthase. This enzyme catalyses the decarboxylative condensation of L-alanine with pimeloyl CoA as part of biotin biosynthesis (Webster *et al.*, 2000). As it has been shown previously that serine is incorporated into the section of prodigiosin that derives from MBC (Wasserman *et al.*, 1960; Harris *et al.*, 2004), we propose that the C-terminal domain of PigH performs a similar condensation of serine with the pyrrolyl-β-ketothioester unit, releasing it from its ACP. The resulting amino diketone would then cyclize, dehydrate and tautomerize (possibly spontaneously) to give HBM.

The *pigL* insertion mutant also shows the characteristics of a strain deficient in the early stages of MBC biosynthesis. It cross-feeds all mutants defective in the MAP pathway and is cross-fed by Sma OF which produces HBC. Sequence homology indicates that PigL is a 4′-phosphopantetheinyl transferase, and this is fully consistent.

tent with our observations, as MBC biosynthesis requires phosphopantetheinylation of the ACP PigG and, presumably, the two ACP domains of PigH. It is not yet known whether PigL catalyses one, two or all three of these phosphopantetheinylations. The biosynthesis of MAP, on the other hand, does not involve any phosphopantetheinylated proteins.

The oxidation of the alcohol group of HBM to an aldehyde has been shown to involve PigM. The *pigMΔ* (NW20) mutant is cross-fed by the *pigF* insertion strain and *pigCΔ* mutants and cross-feeds the *pigH::57 bp* (NW8) strain. Therefore, PigM must act between PigH and PigF in the biosynthesis of MBC. LC-MS analysis of the *pigMΔ* mutant identified only HBM and MAP to be present and this was confirmed by HRMS. In BLASTP and PFAM searches for PigM the only extensive strong sequence similarity is to RedV but there is homology of a short sequence of PigM (amino acids 8–37) to bacterial nitroreductases. These enzymes contain FMN (flavin mononucleotide) and use it to reduce nitroaromatics (e.g. the antibiotic metronidazole). The oxidized FMN is then reduced by NAD(P)H. The *E. coli* nitroreductase has been crystallized (PDB code 1ICU) and the region of homology with PigM is a loop that contacts both the substrate and the ribitol side-chain and benzene ring of FMN. Thus, PigM is an oxidoreductase that catalyses the conversion of HBM to HBC and is most probably FMN dependent. This oxidation of the hydroxyl group to an aldehyde has been proposed to be catalysed by RedK in *Str. coelicolor* A3(2) but there is no homologue of RedK in either Pig cluster (Cerdeño et al., 2001; Harris et al., 2004). RedV is the homologue of PigM so it is seems likely that RedV will catalyse the same reaction in *Str. coelicolor* A3(2), although this remains to be determined experimentally.

The final step of the MBC pathway, methylation of the hydroxy group of HBC, involves the two enzymes PigF and PigN. Norprodigiosin is a shunt product derived from condensation of HBC and MAP. Thus, the accumulation of norprodigiosin in the NW5 (*pigF::57 bp*) strain is consistent with PigF (homologue of RedI) being a SAM-dependent methyl transferase as has been suggested by sequence analysis (Harris et al., 2004). The specific function of PigN is not known. However, the N-terminal region is clearly important for the function of PigN, as a 19 aa insert in this domain results in a higher norprodigiosin:prodigiosin ratio than found for strain TSO8, which has an insertion in the predicted transmembrane domain.

## Biosynthesis of MAP

Our results demonstrate a new pathway for the biosynthesis of MAP not previously considered by ourselves or other workers (Cerdeño et al., 2001; Harris et al., 2004). We had predicted that PigJ, PigH, PigC and PigN were

The biosynthetic pathway of prodigiosin in Serratia 981 involved in the biosynthesis of the monopyrrole but cross-feeding and LC-MS analysis of the corresponding null mutants have proved that none of them are involved in this pathway. We also demonstrated that the heterologous expression of *pig* genes A–E in *E. coli* is sufficient for the synthesis of MAP. Of these five genes the product of *pigA* is involved in the initial steps of MBC biosynthesis and the gene product of *pigC* is the condensing enzyme (discussed later). Interestingly Dauenhauer et al. (1984) in an attempt to isolate prodigiosin biosynthesis genes identified a clone that formed prodigiosin when supplied with MBC but not MAP, although no sequence data were reported for this clone. That result suggested that the genes responsible for MAP synthesis and the gene encoding the condensing enzyme are closely linked as we have demonstrated. No clone was found that could synthesize prodigiosin when supplied with MAP but no MBC, which is consistent with the more fragmented distribution of the MBC biosynthetic genes.

The first steps in the MAP pathway are thought to be performed by enzymes of fatty acid biosynthesis to give 2-octenal. Alternatively 2-octenal may be derived by autoxidation of unsaturated fatty acids (Spiteller et al., 2001). PigD is believed to act early in the biosynthesis of MAP because the *pigDΔ* (NW13) strain was cross-fed by the *pigBΔ* (NW14) strain and PigB is the terminal enzyme in this pathway. Sequence analysis shows that PigD contains a thiamine pyrophosphate (TPP) requiring domain with homology to acetolactate synthase. This enzyme catalyses the decarboxylation of pyruvate and attack of the resulting two-carbon fragment on a second pyruvate molecule. We propose that PigD also decarboxylates pyruvate but that the two-carbon fragment then adds to C-3 of 2-octenal giving 3-acetyloctanal (Fig. 3A). This proposal is supported by our observation that synthetic 3-acetyloctanal restores pigment production to the *pigDΔ* mutant but not the *pigBΔ* mutant (Table 2). Interestingly, Goldschmidt and Williams (1968) showed that the addition of thiamine, the precursor of TPP, to cultures of *S. marcescens* in the early stages of growth induces monopyrrole formation. Therefore, it seems plausible that the stimulatory effect of thiamine results from the TPP requirement of PigD.

The derivation of C-2 of MAP and its attached methyl group from pyruvate instead of from alanine is fully consistent with the previously reported incorporation of ^14^C- and ^13^C-labelled alanine, as alanine can readily be converted to pyruvate by a cellular transamination reaction (Wasserman et al., 1973).

PigE, which the cross-feeding experiments show is the only other enzyme before Pig B in the MAP pathway, has clear homology with aminotransferases. We propose in Fig. 3A that it performs a transamination on the aldehyde group of 3-acetyloctanal and the resulting aminoketone

982 N. R. Williamson et al.

would cyclize spontaneously to give the cyclic imine H₂MAP. However, it is possible that the transamination occurs on the keto group instead, in which case an isomeric H₂MAP would be formed with the double bond on the other side of the nitrogen atom. Aminotransferases require an amino compound, normally an amino acid, as amino group donor. At this stage it is not known which amine or amino acid is responsible.

The precursor accumulated by the pigBΔ strain was found to be H₂MAP by LC-MS, HRMS and reaction with NaBH₃CN (Fig. 3A). This unexpected discovery was what initially lead to the proposals described above. PigB is responsible for a 2-electron oxidation of H₂MAP to MAP and the majority of the protein (amino acids 157–655) aligns well with FAD-dependent amine oxidases, so we assume that FAD is the electron acceptor. The N-terminal 146 amino acids of PigB are homologous with RedS and are predicted to consist of three transmembrane-spanning regions (Harris *et al.*, 2004). Thus, both enzymes (PigN and PigB) involved in the terminal steps of the MBC and MAP pathways are predicted to be anchored in the membrane. The physiological significance of this location for both enzymes is unclear. However, it is known that prodigiosin is very hydrophobic and has been reported to be membrane associated and so it may make sense that the final steps of the biosynthesis occur where this toxic molecule is located (Williams and Quadri, 1980; Kobayashi and Ichikawa, 1989).

otherwise fairly unreactive aldehyde towards attack by C-5 of MAP. After this initial formation of the C–C bond the loss of the proton from what had been C-5 of MAP and elimination of phosphate should be facile.

In this study, the condensing reaction and, in this report the enzyme itself has been shown to be able to couple substrates with different side chains. For example, DEP, an analogue of MAP, and HBC, which lacks the O-methyl group on ring B of MBC, can both be incorporated by the condensing enzyme to form analogues of prodigiosin. These data suggest a degree of relaxed substrate specificity and so it would be of interest to determine how relaxed is this substrate utilization with an array of analogues of MAP and MBC. Recently much research has been focussed on chemically synthesizing MBC and MAP derivatives (Fürstner, 2003). A wild-type, or mutated, PigC capable of condensing these synthetic compounds could be of great biotechnological interest to facilitate the development of new analogues of prodigiosin with different side-chains and hopefully enhanced activity or reduced toxicity. A crystal structure of PigC with either MAP or MBC bound would help identify key residues within the active site and this would facilitate targeted mutagenesis studies.

Implications of the new Serratia pathway for the biosynthesis of undecylprodigiosin in Streptomyces coelicolor A3(2)

All of the genes identified here as being involved in the biosynthesis of MBC have close homologues in the red cluster whereas none of the genes involved in the biosynthesis of MAP do. This very strongly suggests that the same chemical steps are involved in MBC synthesis in the two organisms and are catalysed by the homologous gene products. In their paper on the red cluster, Cerdeño *et al.* (2001) did propose the same chemical steps as we do here but assigned genes to some of the steps which were not the ones homologous to the pig genes belonging to this pathway. We have therefore taken the opportunity to reassign the red genes as follows (Fig. 3B). RedO, RedM and RedW are involved in formation of the pyrrolyl-2-carboxyl unit as previously proposed and experimentally demonstrated (Cerdeño *et al.*, 2001; Thomas *et al.*, 2002). This unit is then transferred to the ketosynthase RedX (not RedL), which, with the ACP and PLP-dependent domains of RedN, completes the synthesis of HBM. Oxidation by RedV (not RedK) followed by methylation by RedI (as proposed) assisted by RedF then gives MBC.

This reassignment of several of the genes previously assigned to the biosynthesis of 2-undecylpyrrole, and release of several of the genes previously assigned to MBC biosynthesis, allows the following putative assignments for 2-undecylpyrrole biosynthesis. The RedQ doce-

Identification of PigC as the condensing enzyme

The final biosynthetic step is the condensation of the terminal products of the two parallel pathways, MAP and MBC, to form prodigiosin. The data presented here clearly identify PigC as the condensing enzyme because the pigCΔ strain can cross-feed all other mutants (except those defective in pigN), cannot be cross-fed by any other strain, and contains both MBC and MAP in its extract. We have also demonstrated the catalytic ability of a GST-PigC fusion protein to couple MBC and MAP in E. coli.

PigC and its Red homologue (RedH) have N- and C-terminal domains which share sequence similarity with the ATP-binding and phosphoryl transfer domains of a pyruvate phosphate dikinase (Herzberg *et al.*, 1996; Cerdeño *et al.*, 2001; Harris *et al.*, 2004). Both proteins contain a third, central domain which shows low similarity to other proteins, and presumably contains the core catalytic region that binds MAP and MBC. PigC, and presumably RedH, represent the first members of a novel class of pyrrole-condensing enzymes. The mechanism that we envisage for this condensation reaction is that a phosphoryl group from ATP is transferred via His-840 in the C-terminal phosphoryl transfer domain to the oxygen atom of the aldehyde group of MBC. This would activate this

© 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **56**, 971–989

canoyl thioester is probably formed by enzymes of fatty acid biosynthesis along with the ketosynthases RedP and RedR, as proposed (Cerdeño *et al.*, 2001). The dodecanoyl group is then transferred to the ACP domain of RedL and thence to the active-site cysteine of the ketosynthase domain of RedL (not RedX). Reaction with a malonyl group attached to the second ACP domain of RedL (not RedN) would then give a β-ketomyristoyl thioester. The C-terminal PLP domain of RedL would then perform the decarboxylative condensation of glycine with the β-keto thioester, to give, after cyclization, 4-keto-2-undecyl-4,5-dihydropyrrole. Interestingly PigE (which we show in this article functions in the MAP pathway) has 35% identity with the PLP-dependant domain of RedL (Cerdeño *et al.*, 2001; Harris *et al.*, 2004). Reduction by RedK (not RedF and RedH) and dehydration would then give the required 2-undecylpyrrole. The final condensation of 2-undecylpyrrole with MBC would be catalysed by RedH, the homologue of PigC.

At first sight these new assignments would seem to be inconsistent with some of the cross-feeding results reported by Cerdeño *et al.* (2001). However, on re-examination of the earlier data, we believe that those results may have been misinterpreted and in fact are fully consistent with our new assignment, more so than their original assignment. Cerdeño *et al.* (2001) showed that both a redXWY deletion mutant and a redN deletion mutant are cross-fed by previously isolated class C and class E red mutants of Str. coelicolor A3(2) and by Sma OF but not Sma WF or Sma 933. They stated that all the strains that do cross-feed their deletion mutants are blocked in MBC biosynthesis and therefore excrete monopyrrole (MAP or 2-undecylpyrrole) and they concluded that both redX and redN are involved in 2-undecylpyrrole biosynthesis. However, it had previously been reported (and confirmed in this study) that Sma WF excretes MAP. The lack of cross-feeding by Sma WF therefore suggests that their deletion mutants are blocked in MBC biosynthesis. Of the other mutants used, class C and class E red mutants and Sma OF are known to be blocked in the later stages of MBC biosynthesis and are able to cross-feed mutants blocked at an earlier stage (Morrison, 1966; Feitelson and Hopwood, 1983; Feitelson *et al.*, 1985; Tsao *et al.*, 1985). Therefore, the cross-feeding results are consistent with the positioning of redX and redN (as well as redW) in the early part of MBC biosynthesis, before the first protein-free intermediate, which is HBM. Finally, Sma 933 is a strain that was reported to produce MBC but, in our hands, it produces almost no intermediate metabolites, presumably because the strain has acquired a further mutation(s) and this could explain the negative results with this strain. One further piece of evidence in favour of the notion that the redXWY and redN deletion mutants and the class C red mutants are blocked in the same pathway and not

The biosynthetic pathway of prodigiosin in Serratia 983

different pathways is that if they were blocked in different pathways one would see reciprocal cross-feeding, whereas in fact no pigmentation of the class C red mutants was observed where they were close to the redXWY or redN deletion mutants.

Our reassignment of the red genes also removes another problem that was faced by Cerdeño *et al.* (2001), which was how to explain the production of MBC by the redXWY deletion mutant when redW was known to be involved in the oxidation of the prolyl unit in the early stages of MBC synthesis. To explain this, the existence of a second, alternative pathway for the formation of pyrrole-2-carboxyl units was invoked. In our reinterpretation the redXWY deletion mutant is blocked in two of the early stages of MBC production, redX and redW, and does not produce MBC at all, unless cross-fed by a mutant blocked in the later stages of MBC biosynthesis. The third deleted gene, redY, is the homologue of pigK and we have shown that, in Serratia, deletion of pigK makes no noticeable difference to prodigiosin production.

### Conclusion

The results reported here have led to the elucidation of all the steps for the production of prodigiosin in Serratia 39006 and the unambiguous assignment of the gene products responsible for each of those steps (Table 3). The biosynthesis of MBC is essentially the same in Serratia as in Str. coelicolor A3(2), but the monopyrrole moieties are made by quite different pathways. The similarity between the genes for MBC biosynthesis in Serratia and some of the red genes in Str. coelicolor A3(2) has necessitated a reassignment of those genes and a full assignment, which is consistent between the two organisms as well as with all the published cross-feeding results, is given here.

The nature of the majority of the free intermediates in the pathway detected by cross-feeding studies has been confirmed by LC-MS, HRMS and in some cases by chemical synthesis and incorporation. Work is now in progress to confirm the proposed enzymatic steps in our new pathway.

### Experimental procedures

#### Bacterial strains, plasmids and culture conditions

The bacterial strains and plasmids used in this study are shown in Table 1. *E. coli* strains were cultured at 37°C and Serratia strains were cultured at 30°C in LB medium. Cross-feeding analysis of the Serratia 39006 pig mutants was performed on PGM (1% glycerol, 0.5% peptone, 1.6% agar). Antibiotic final concentrations were as follows, unless otherwise stated: 100 μg ml⁻¹ ampicillin (Ap), 100 μg ml⁻¹ streptomycin (Sm) and 10 μg ml⁻¹ tetracycline (Tc).

© 2005 Blackwell Publishing Ltd, *Molecular Microbiology*, **56**, 971–989

984 N. R. Williamson et al.

Table 3. Previously predicted and newly assigned functions of the different proteins of the Serratia 39006 Pig cluster. The respective Red homologue of each Pig enzyme and the biosynthetic pathway that it is involved in are also shown.

| Protein | Red homologue | Putative function from sequence homology | Pathway | Function |
|---------|----------------|------------------------------------------|----------|----------|
| PigA    | RedW           | L-prolyl-PCP dehydrogenase               | MBC      | L-prolyl-PCP dehydrogenase |
| PigB    | RedS           | Amine oxidase                           | MAP      | H₂MAP oxidase/dehydrogenase |
| PigC    | RedH           | Kinase                                  | Terminal step | Condensing enzyme |
| PigD    | PdorfL<sup>a</sup> | No assigned function                   | MAP      | 2-Acetyloctanal synthase |
| PigE    | PdorfM<sup>a</sup> | Aminotransferase                       | MAP      | 2-Acetyloctanal aminotransferase |
| PigF    | RedI           | O-methyl transferase                    | MBC      | HBC O-methyl transferase |
| PigG    | RedO           | Peptidyl carrier protein (PCP)           | MBC      | Peptidyl carrier protein (PCP) |
| PigH    | RedN           | Aminotransferase                        | MBC      | HBM synthase (seryl transferase) |
| PigI    | RedM           | L-prolyl-AMP ligase                     | MBC      | L-prolyl-AMP ligase |
| PigJ    | RedX           | β-Ketoacyl synthase                     | MBC      | Pyrrolyl-β-ketoacyl ACP synthase |
| PigK    | RedY           | No assigned function                    | None     | Unknown |
| PigL    | RedU           | 4′-Phosphopantetheinyl transferase      | MBC      | 4′-Phosphopantetheinyl transferase |
| PigM    | RedV           | No assigned function                    | MBC      | HBM oxidase/dehydrogenase |
| PigN    | RedF           | Oxidoreductase                          | MBC      | Unknown |
| PigO    | PAB0452<sup>b</sup> | No assigned function                   | None     | Unknown |

a. PdtorfL and PdtorfM are from *Pseudomonas stutzeri* (not from the Red cluster).

b. PAB0452 is a VirR-related protein from *Pyrococcus abyssi* (not from the Red cluster) (Harris et al., 2004).

DNA manipulations

All DNA manipulations were performed as described in Sambrook et al. (1989). Genomic DNA from *Serratia* strains was extracted with the DNAeasy Tissue kit (Qiagen). DNA fragments were isolated from agarose gels using a gel extraction kit (Qiagen). Isolation of cosmids and plasmids was performed with the Qiagen Miniprep kit. Oligonucleotide primers were purchased from Sigma-Genosys. Double-stranded sequencing was performed at the DNA Sequencing facility, Department of Biochemistry, University of Cambridge using an ABI 3700 DNA analyser. All PCR reactions were performed using Expand high fidelity Taq polymerase (Roche).

Construction of in-frame deletion marker exchange constructs

In-frame deletions were generated either by the use of restriction sites introduced by PCR amplification or by native in-frame restriction sites. Primers used in this study are listed in Table 4. The *pigBΔ* construct was made by digesting pNRW33 with *Tthl* and removal of a 1.504 kb internal fragment of *pigB*. The 5′ T overhang was filled in with klenow enzyme (NEB) and the plasmid backbone was self-ligated with T4 DNA ligase (Invitrogen) to create plasmid pNRW34. The *pigBΔ* allele was cloned into the Smal site of pKNG101 on a 0.85 kb *HincII/EcoRV* fragment to create the marker exchange construct, pNRW41. To construct the *pigDΔ* allele, pNRW35 was digested with *NruI/Ndel* and the plasmid backbone was blunt-ended with klenow and self-ligated with T4 DNA ligase to form pNRW36. pNRW36 was digested with *BamHI* and a 2.5 kb fragment was ligated into the *BamHI* site of pKNG101 to create the *pigDΔ* marker exchange construct pNRW40. The 5′ and 3′ flanking regions of *pigM* were amplified with primers NW54-NW51R and NW52-NW53 and digested with *SalI-BamHI* and *BamHI-Spel* and ligated in a three-way ligation into the marker exchange vector pKNG101 to create pNRW52. The 5′ and 3′ flanking regions of *pigK* were amplified with primers NW60-NW63 and NW62-NW61 and digested with *SalI-HindIII* and *HindIII-Spel* and ligated in a three-way ligation with pKNG101 to create pNRW51. The 5′ and 3′ flanking regions of *pigC* were amplified with primers NW49-Cdelr and Cdelr-NW50 and digested with *SalI-BamHI* and *BamHI-Spel* and ligated in a three-way ligation with pKNG101 to create pNRW54. The 5′ and 3′ flanking regions of *pigA* were amplified with primers NW68-NW69 and NW70-71 and digested with *SalI-BamHI* and *BamHI-Spel* and ligated in a three-way ligation with pKNG101 to create pNRW58. The *pigGΔ* marker exchange plasmid pNRW53 was created in a similar manner. The 5′ flanking region was PCR amplified with NW20-NW72 and the 3′ region was amplified with NW73-NW74 and the subsequent PCR products were digested with *SalI-HindIII* and *HindIII-Apal*. In this case the *SalI* site in the 5′ PCR product was already present.

Construction of in-frame linker mutants

In vitro transposon insertion reactions were carried out as described in the manufacturer’s instructions (Epicentre). Briefly, equimolar amounts of the EZ::TN <NotI/KAN-3> and 0.2 μg of target DNA (pNRW25 or pTON245) plus 1 μl of EZ::TN transposase were incubated at 37°C for 2 h in 1× EZ::TN reaction buffer. The reaction was stopped by the addition of 1 μl of 10× stop solution and heat inactivation at 70°C for 10 min. Electrocompetent *E. coli* DH5α cells were transformed with 1 μl of this reaction mixture and kanamycin (Km)-resistant colonies were selected. The location of the transposon insertion was identified using PCR with KAN3FP2 in combination with *pig* cluster-specific primers and confirmed by sequencing of purified plasmids with the transposon specific primers. The Km cassette was removed by *NotI* restriction digestion and the plasmid was resolved by agarose gel electrophoresis and ligated to itself using T4 DNA ligase (Invitrogen). The resulting plasmid was Km<sup>s</sup>, and the mutant allele carrying the 57 bp in-frame insertion was confirmed by sequencing and then cloned into the marker exchange vector pKNG101 (Table 1).

The biosynthetic pathway of prodigiosin in Serratia 985

Table 4. A list of oligonucleotide PCR primers used this study.

| Primer | Sequence 5′–3′ | Target |
|--------|-----------------|--------|
| CdelF  | GATGGATCCACGGTCATTATCCAACAC | 7499–7525 (BamHI) |
| CdelR  | ACCGGATCCTTCAACAACCAGTGCGTTG | 4886–4912 (BamHI) |
| NW20   | GACGCTCATATGACATTAACC | 12875–12895 (NdeI) |
| NW21   | ATTTACCTCGAGACATTGTGA | 13919–13939 |
| NW37   | GATAAGCTTTAAACTTCATGATATAG | 10305–10329 (HindIII) |
| NW38   | TCAAAGCTTTTAGAGTAAGGACGCTG | 12856–12881 (HindIII) |
| NW39   | GGCTGCATAAGGATTCGGTC | 7695–7714 |
| NW49   | GAAAGTCGACAGCAATAAAGATCCGTTG | 4413–4439 (SalI) |
| NW50   | TGCACTAGTCAACATTTCCCCAACGAGG | 8142–8168 (SpeI) |
| NW51R  | TGCAGATCCAATGGCGTTTTGCAATG | 20949–20974 (BamHI) |
| NW52   | CTGGGATCCGGGGGCTGATGTAACAAA | 21972–21997 (BamHI) |
| NW53   | AGAACTAGTGGATAAAGCACTTCACCTTC | 22717–22745 (SpeI) |
| NW54   | GGTGTCGACGTTTGATCAACTGGCTGAG | 20183–20210 (SalI) |
| NW60   | ATTGTCGACGGATCTGGTGCTGTGCAG | 19288–19314 (SalI) |
| NW61   | CCAACTAGTCCGTGTCAGTCATCAATG | 20855–20881 (SpeI) |
| NW62   | GTGAAGCTTTTTGATCAACTGGCTGAG | 20184–20210 (HindIII) |
| NW63   | TGCAAGCTTCTGAGTAATATCTTTGAG | 19974–20000 (HindIII) |
| NW68   | TTCGTCGACTGAGTAATTACTCTTTGTGTC | 991–1019 (SalI) |
| NW69   | CTCGGATCCATCTGGACTGACTTTTGACAG | 1726–1755 (BamHI) |
| NW70   | ATCGGATCCGAGATTATGGCCCCTAAACTC | 2821–2850 (BamHI) |
| NW71   | CAAACTAGTTTCAGTAAGCGCAGCATATTC | 3515–3543 (SpeI) |
| NW72   | ATCAAGCTTCTGAGTAGCAATATGGT | 13946–13971 (HindIII) |
| NW73   | CTGAAGCTTCGGCTGAAGGCGCATCCG | 14140–14166 (HindIII) |
| NW74   | CAGCGGGCCCACGACAGCCGGTATTTGGTG | 15134–15162 (Apal) |
| NW81   | GCCGAATTCACCGTCATGGTTTGCAGCTC | 7532–7558 (EcoRI) |
| NW82   | GGAATTCTGAATCAACCACCTGGTTG | 4874–4898 (EcoRI) |

The location of the site at which the primer binds is shown (Serratia 39006 *pig* cluster numbering; Accession No. AJ833001) and the restriction site is underlined and the restriction enzyme is shown in the brackets.

and was not introduced by modification of the primer. The *pigEΔ* mutation was first constructed in pbluescriptIIKS+. The 5′ and 3′ flanking regions were PCR amplified with NW39-NW37 and NW38-NW21 and digested with BamHI-HindIII and HindIII-SalI respectively. Again the SalI site in this case was native and not introduced by incorporation into the primer. The 5′ and 3′ flanking regions were ligated together using equimolar amounts of each PCR product and of the vector using T4 DNA ligase (Invitrogen). All three-way ligations were performed in this manner. Marker exchange with pNRW21, pNRW40-pNRW54 and pNRW58 (Table 1) was carried out by a similar protocol to Kaniga et al. (1991).

Cloning of *pigC* into pGEX-2T expression vector

The *pigC* gene was amplified by PCR using primers NW81 and NW82 (Table 4) and cloned into the EcoRI site of pGEX-2T, forming construct pNRW55 (correct orientation of insert determined by restriction digestion).

Cross-feeding analysis

All cross-feeding studies were performed on PGM plates unless otherwise stated. Each Serratia 39006 *pig* mutant was streaked next to every other *pig* mutant as exemplified in Fig. 4. Cross-feeding was observed after 2 days of growth at 30°C.

Feeding of MAP and DEP to Serratia 39006 *pig* mutants

Serratia 39006 *pig* mutants were incubated at 30°C for 2 days

on PGM agar. Ten microlitres of either, DEP, a solution of MAP dissolved in DMSO or DMSO were added to a sterile paper disc and this was then placed next to the bacterial colonies. The plates were then sealed with tape and incubated overnight at 30°C.

DEP or MAP was added to a 17-h-old culture of NW13 (*pigDΔ*) grown at 30°C, 300 r.p.m and the culture was grown for a further 2 h and pigment was extracted as described in the section LC-MS studies.

Co-synthesis studies

Five millilitres of overnight cultures of two strains exhibiting positive cross-feeding were used to inoculate 200 ml of LB containing 0.25 M sorbitol. This mixed culture was grown at 30°C with shaking (300 r.p.m.) for 18 h. Pigment was then extracted as described in the section LC-MS studies.

LC-MS studies

Sample preparation. Two hundred millilitres of media were inoculated from a starting culture and incubated at 30°C for 16–18 h. Two different broths were used as media, minimal medium and LB with 0.25 M sorbitol.

After 16–18 h the medium was separated from the cells by centrifugation. The medium was extracted twice with 100 ml of acetone-ether 4:1, and the cell pellet was extracted once with 100 ml of acetone-ether 4:1. The two extracts were concentrated under reduced pressure to 5 ml, and a 300 μl sample was taken from each extract. The extracts were then concentrated to dryness and redissolved in acetone-ether

© 2005 Blackwell Publishing Ltd, Molecular Microbiology, 56, 971–989

4:1 to a concentration of 1 mg ml⁻¹. From each redissolved extract a 300 μl sample was taken, making a total of four samples from each culture.

**LC and MS methods.** HPLC was performed with a Hichrom Hypersil H5MOS column (C8, 5 μm, 4.6 × 250 mm) with gradient elution at a flow rate of 1.0 ml min⁻¹. Solvent A was H₂O/MeOH 95:5 with 0.02% TFA, and solvent B was H₂O/MeOH 5:95 with 0.02% TFA. The gradient programme was as follows: 0 min, 30% B; 12 min, 85% B; 30 min, 100% B; 39 min, 100% B; 40 min, 30% B. There was a 5 min post-run period before the next injection. Ten microlitres of each sample were injected. The diode array detector scanned between 200 and 600 nm, and a PC acquired the data with MassLynx v 2.3 and OpenLynx LC Control v 2.3 as software. The flow was split after the UV in a 95:5 ratio, with 5% entering the mass spectrometer. A reference sample of prodigiosin was obtained from the National Cancer Institute (USA) and MAP was synthesized as described below.

The MS was performed on a Micromass Platform LC instrument using positive ion electrospray ionization and the following settings: capillary voltage 3.4 V, cone voltage 30 V, source temperature 100°C. A PC with MassLynx v 2.3 and OpenLynx LC Control v 2.3 as software acquired data.

High resolution mass spectrometry (HRMS) was performed on a Bruker Bio-Apex II FT-ICR instrument and MS/MS studies on a Micromass Q-TOF instrument, both with electrospray ionization.

(2.5 ml, 28.9 mmol) was added dropwise over 2 min and the mixture was heated at reflux for 2 h, then filtered and evaporated under reduced pressure. The residue was partitioned between dichloromethane (50 ml) and water (80 ml), and the organic layer was dried (MgSO₄) and evaporated under reduced pressure. Purification of the residue by flash chromatography on silica eluting with hexane–ethyl acetate (8:1) gave ethyl 2-acetyl-2-pentylpent-4-enoate (4.7 g, 87%) as an oil (Found: *M*⁺, 240.17373. C₁₄H₂₄O₃ requires *M*, 240.1725); *R*ₐ 0.65 (EtOAc-hexane 1:5); ν_max(CH₂Cl₂)/cm⁻¹ 2957, 2872, 2360, 1740 and 1712; δ_H (400 MHz, CDCl₃) 0.82 (3 H, t, *J* 6.9, CH₂CH₂CH₃), 1.15 (3 H, t, *J* 7.1, CH₂CH₃), 1.21–1.27 (6 H, m, CH₂CH₂CH₂CH₃), 1.75 (1 H, ddd, *J* 5.1, 11.3 and 14.2, CHₐHₐC), 1.84 (1 H, ddd, *J* 5.1, 11.3 and 14.2, CHₐHₐC), 2.08 (3 H, s, CH₃CO), 2.53 (1 H, dd, *J* 7.5 and 14.5, CHₐHₐCH=CH₂), 2.58 (1 H, dd, *J* 7.5 and 14.5, CHₐHₐCH=CH₂), 4.15 (2 H, q, *J* 7.1, CH₂CH₃), 5.03 (1 H, d, *J* 17.1, *cis*-HC=CHₐHₐ), 5.07 (1 H, d, *J* 17.1, *trans*-HC=CHₐHₐ), 5.55 (1 H, ddt, *J* 17.1, 10.1 and 7.5, CH=CH₂); δ_C (100 MHz, CDCl₃) 13.86 (CO₂CH₂CH₃), 14.04 (CH₃CH₂CH₂), 22.27 (CH₃CH₂CH₂), 23.19 (CCH₂CH₂), 26.69 (CH₃C=O), 31.22 (CCH₂), 32.01 (CH₂CH=), 35.77 (CH₃CH₂CH₂), 61.15 (CH₃CH₂O), 63.35 (CCH₂), 118.60 (CH₂=CH), 132.55 (CH₂=CH), 172.04 (CO₂Et), 204.74 (CH₃CO).

**3-Pentylhex-5-en-2-one.** A solution of ethyl 2-acetyl-2-pentylpent-4-enoate (5.5 g, 22.91 mmol) in 5% aqueous sodium hydroxide (50 ml) and methanol (10 ml) was heated at 80°C for 24 h and then cooled to 0°C, acidified to pH 5–6 with sulphuric acid and extracted with dichloromethane (4 × 80 ml). The combined organic layers were dried (MgSO₄) and evaporated under reduced pressure. The residue was purified by flash chromatography on silica eluting with EtOAc–hexane (1:5) to give 3-pentylhex-5-en-2-one (2.8 g, 72%) as an oil (Found: *M*⁺, 168.15102. C₁₁H₂₀O requires *M*, 168.1514); *R*ₐ 0.55 (EtOAc-hexane 2:5); δ_H (400 MHz, CDCl₃) 0.85 (3 H, t, *J* 6.9, CH₂CH₂CH₃), 1.19–1.27 (6 H, m, CH₂CH₂CH₂CH₃), 1.38–1.5 (1 H, m, CHₐHₐCH), 1.53–1.65 (1 H, m, CHₐHₐCH), 2.09 (3 H, s, CH₃CO), 2.17 (1 H, ddd, *J* 6.3, 7.6 and 14, CHₐHₐCH=CH₂), 2.32 (1 H, ddd, *J* 6.3, 7.6 and 14, CHₐHₐCH=CH₂), 2.53 (1 H, tt, *J* 6.3 and 7.6, CH₂CH), 4.97 (1 H, d, *J* 9.9, *cis*-HC=CHₐHₐ), 5.03 (1 H, d, *J* 17.1, *trans*-HC=CHₐHₐ), 5.70 (1 H, ddt, *J* 17.1, 10.1 and 7.1, CH=CH₂); δ_C (400 MHz, CDCl₃) 13.94 (CH₃CH₂), 22.42 (CH₃CH₂), 26.86 (CHCH₂), 29.10 (CH₃C=O), 31.11 (CHCH₂CH₂), 31.83 (CH₂CH=), 35.71 (CH₃CH₂CH₂), 52.70 (CHCH₂), 116.61 (CH=CH₂), 135.62 (CH=CH₂), 212.16 (CH₃C=O).

**Synthesis of 3-acetyloctanal and 2-methyl-3-n-amylpyrrole (MAP)**

**Ethyl 2-acetylheptanoate.** Dry ethanol (25 ml) was added slowly and carefully to sodium metal (800 mg, 34.7 mmol) under argon. After the initial vigorous reaction the mixture was heated to 60°C until the sodium had reacted completely. Freshly distilled ethyl acetoacetate (4 g, 30.76 mmol) was then added over 10 min followed by the iodopentane (7.28 g, 36.76 mmol). The solution was heated at reflux for 8 h, then cooled and evaporated to dryness. The residue was partitioned between dichloromethane and water and the combined organic layers were dried (MgSO₄) and evaporated under reduced pressure. The residue was purified by flash chromatography on silica eluting with ethyl acetate–hexane (1:4) to give ethyl 2-acetylheptanoate (4.5 g, 73%) as an oil. (Found: *M*⁺, 200.14151. C₁₁H₂₀O₃ requires *M*, 200.1412); *R*ₐ 0.64 (EtOAc–hexane 1:6); ν_max(CH₂Cl₂)/cm⁻¹ 2957, 2860, 1740 and 1717; δ_H (400 MHz, CDCl₃) 0.84 (3 H, t, *J* 6.7, CH₂CH₂CH₃), 1.18–1.27 (9 H, m, CH₂CH₃ and CH₂CH₂CH₂CH₃), 1.70–1.85 (2 H, m, CH₂CH), 2.17 (3 H, s, CH₃CO), 3.35 (1 H, t, *J* 7.3, CHCO₂), 4.15 (2 H, q, *J* 7.2, CH₂CH₃); δ_C (100 MHz, CDCl₃) 13.82 (CH₂CH₃), 14.00 (CH₃CH₂CH₂), 22.26 (CH₃CH₂CH₂), 25.85 (CH₂CH), 26.96 (CH₂CH₂CH), 28.07 (CH₃CO), 31.40 (CH₃CH₂CH₂), 59.86 (CH), 61.14 (CH₂CH₃), 169.86 (CO₂Et), 203.24 (CH₃C=O).

**Ethyl 2-acetyl-2-pentylpent-4-enoate.** Ethyl 2-acetylheptanoate (4.5 g, 22.5 mmol) was added dropwise at 50°C to a solution of sodium ethoxide (prepared as above, 650 mg of sodium in 20 ml of dry ethanol). After 4 min, allyl bromide

3-Acetyloctanal. A stirred solution of 3-pentylhex-5-en-2-one (2 g, 11.90 mmol) in dichloromethane (120 ml) was cooled to –78°C and a stream of ozone in oxygen was bubbled through for 1.5 h, until TLC indicated all starting material had been consumed. Then dimethyl sulphide (4 ml) was added and the solution was warmed to room temperature and stirred for 48 h. Further aliquots of dimethyl sulphide (4 ml) were added every 24 h until TLC showed complete breakdown of the ozonide. The solution was then evaporated under reduced pressure. The resulting residue was purified by flash chromatography on silica eluting with ethyl acetate–hexane (4:1) to give 3-acetyloctanal (950 mg, 47%) as an oil (Found: *MN*a⁺,

193.1199. C₁₀H₁₈O₂ requires MNa, 193.1204); Rf 0.49 (EtOAc-hexane 4:1); νmax (liquid) 2926, 2857 and 1711 cm⁻¹; δH (400 MHz, CDCl₃) 0.85 (3 H, t, J 6.9, CH₂CH₂CH₃), 1.22 (6 H, m, CH₂CH₂CH₂), 1.37 (1 H, m, CHₐHₐCH), 1.60 (1 H, m, CHₐHₐCH), 2.20 (3 H, s, CH₃CO), 2.45 (1 H, dd, J 3.7 and 18.1, CHₐHₐCHO), 2.92 (1 H, dd, J 9.5 and 18.1, CHₐHₐ-CHO), 3.02 (1 H, m, CH₂CH), 9.70 (1 H, s, CHO); δC (125 MHz, CDCl₃) 13.70 (CH₃CH₂), 22.14 (CH₃CH₂), 26.35 (CHCH₂CH₂), 29.11 (CH₃CO), 30.91 (CHCH₂), 31.48 (CH₃CH₂CH₂), 44.59 (CH₂CHO), 45.62 (CH), 200.49 (CHO), 210.47 (CH₃CO); m/z (EI): 168 (M⁺, 60%), 125 (M⁺-CH₃CO, 10%) and 98 (M⁺-CH₃CO and CH₂=CH, 100%).

2-Methyl-3-pentylpyrrole (MAP). A solution of 3-acetyloctanal (800 mg, 4.70 mmol) in ethanol (90 ml) was bubbled with ammonia gas for 10 min at –20°C. After stirring for 2 h, the solution was evaporated under reduced pressure and the residue was distilled by bulb-to-bulb distillation, to give 2-methyl-3-pentylpyrrole (MAP) (500 mg, 70%) as an oil (Found: M⁺, 151.13676 C₁₀H₁₇N requires M, 151.1361); Rf 0.55 (EtOAc-hexane 4:1); νmax (liquid)/cm⁻¹ 3385, 2955, 2852, 1636; δH (400 MHz, CDCl₃) 0.90 (3 H, t, J 7, CH₃CH₂), 1.20–1.35 (4 H, m, CH₃CH₂CH₂), 1.53 (2 H, m, CCH₂CH₂), 2.17 (3 H, s, pyrrole-CH₃), 2.37 (2 H, t, J 7.5, CCH₂), 6.01 (1 H, t, J 2.7, CH=CHN), 6.58 (1 H, t, J 2.7, CH=CHN), 7.69 (1 H, br s, NH); δC (100 MHz, CDCl₃) 11.00 (pyrrole-CH₃), 14.13 (CH₃CH₂), 22.65 (CH₃CH₂), 25.91 (CCH₂), 31.06 (CH₃CH₂CH₂), 31.82 (CCH₂CH₂), 108.84 (CH=CHN), 114.84 (CH=CHN), 119.76 (CCH₂), 123.17 (NCCH₃); m/z (EI): 151 (M⁺, 72%) and 94 (M⁺-CH₃CH₂CH₂CH₂, 97%).

Reduction of H₂MAP to 2-methyl-3-n-amyl-pyrrolidine

An extract of NW14 (pigBΔ) (20 mg), obtained as described under sample preparation, was dissolved in diethyl ether and NaBH₃CN (25 mg) and concentrated HCl (10 μl) were added. The mixture was stirred for 1 h and then concentrated under reduced pressure, redissolved in ether and washed with water (3 × 10 ml). Samples of the ether fraction were analysed by LC-MS.

Acknowledgements

We thank Martin Welch and members of the Salmond group for helpful discussions. We would also like to thank Keith Chater of the John Innes Centre, UK for kindly supplying the Sma 933, Sma OF and Sma WF strains. Finally we would like to thank the National Cancer Institute (USA) for the gift of a sample of prodigiosin. This study was supported generously by the BBSRC.

References

Azambuja, P., Feder, D., and Garcia, E.S. (2004) Isolation of *Serratia marcescens* in the midgut of *Rhodnius prolixus*: impact on the establishment of the parasite *Trypanosoma cruzi* in the vector. *Exp Parasitol* **107**: 89–96.

Bisang, C., Long, P.F., Cortes, J., Westcott, J., Crosby, J., Matharu, A.L., et al. (1999) A chain initiation factor com-

The biosynthetic pathway of prodigiosin in Serratia 987 mon to both modular and aromatic polyketide synthases. *Nature* **401**: 502–505.

Bruckner, R.C., Zhao, G., Venci, D., and Jorns, M.S. (2004) Nikkomycin biosynthesis: formation of a 4-electron oxidation product during turnover of NikD with its physiological substrate. *Biochemistry* **43**: 9160–9167.

Bu’Lock, J.D. (1961) Intermediary metabolism and antibiotic synthesis. *Annu Rev Microbiol* **3**: 293–342.

Bycroft, B.W., Maslen, C., Box, S.J., Brown, A., and Tyler, J.W. (1987) The isolation and characterisation of (3R,5R)- and (3S,5R)-carbapenem-3-carboxylic acid from *Serratia* and *Erwinia* species and their putative biosynthetic role. *Chemical Soc Chem Commun* **21**: 1623–1625.

Cámpas, C., Dalmau, M., Montaner, B., Barragan, M., Bellosillo, B., Colomer, D., et al. (2003) Prodigiosin induces apoptosis of B and T cells from B-cell chronic lymphocytic leukemia. *Leukemia* **17**: 746–750.

Cerdeño, A.M., Bibb, M.J., and Challis, G.L. (2001) Analysis of the prodigiosine biosynthesis gene cluster of *Streptomyces coelicolor* A3(2): new mechanisms for chain initiation and termination in modular multienzymes. *Chem Biol* **8**: 817–829.

D’Alessio, R., Bargiotti, A., Carlini, O., Colotta, F., Ferrari, M., Gnocchi, P., et al. (2000) Synthesis and immunosuppressive activity of novel prodigiosin derivatives. *J Med Chem* **43**: 2557–2565.

Dauenhauer, S.A., Hull, R.A., and Williams, R.P. (1984) Cloning and expression in *Escherichia coli* of *Serratia marcescens* genes encoding prodigiosin biosynthesis. *J Bacteriol* **158**: 1128–1132.

Demain, A.L. (1995) Why do microorganisms produce antimicrobials? In *Fifty Years of Antimicrobials: Past Perspectives and Future Trends*. Hunter, P.A., Darby, G.K., and Russell, N.J. (eds). Cambridge: Society for General Microbiology, Symposium 53, pp. 205–228.

Feitelson, J.S., and Hopwood, D.A. (1983) Cloning of a *Streptomyces* gene for an O-methyltransferase involved in antibiotic biosynthesis. *Mol Gen Genet* **190**: 394–398.

Feitelson, J.S., Malpartida, F., and Hopwood, D.A. (1985) Genetic and biochemical characterization of the red gene cluster of *Streptomyces coelicolor* A3(2). *J Gen Microbiol* **131** (Pt 9): 2431–2441.

Fineran, P.C., Slater, H., Everson, L., Hughes, K., and Salmond, G.P. (2005) Biosynthesis of tripyrrole and β-lactam secondary metabolites in *Serratia*: integration of quorum sensing with multiple new regulatory components in the control of prodigiosin and carbapenem antibiotic production. *Mol Microbiol* (in press).

Fürstner, A. (2003) Chemistry and biology of roseophilin and the prodigiosin alkaloids: a survey of the last 2500 years. *Angew Chem Int Ed Engl* **42**: 3582–3603.

Goldschmidt, M.C., and Williams, R.P. (1968) Thiamine-induced formation of the monopyrrole moiety of prodigiosin. *J Bacteriol* **96**: 609–616.

Grinter, N.J. (1983) A broad-host-range cloning vector transposable to various replicons. *Gene* **21**: 133–143.

Harris, A.K.P., Williamson, N.R., Slater, H., Cox, A., Abbasi, S., Foulds, I., et al. (2004) The *Serratia* gene cluster encoding biosynthesis of the red antibiotic, prodigiosin, shows species and strain dependent genome context variation. *Microbiology* **150**: 3547–3560.

Herrero, M., De lorenzo, V., and Timmis, K.N. (1990) Transposon vectors containing non-antibiotic resistance selection markers for cloning and stable chromosomal insertion of foreign genes in Gram-negative bacteria. J Bacteriol 172: 6557–6567.

Herzberg, O., Chen, C.C., Kapadia, G., McGuire, M., Carroll, L.J., Noh, S.J., and Dunaway-Mariano, D. (1996) Swiveling-domain mechanism for enzymatic phosphotransfer between remote reaction sites. Proc Natl Acad Sci USA 93: 2652–2657.

Hood, D.W., Heidstra, R., Swoboda, U.K., and Hodgson, D.A. (1992) Molecular genetic analysis of proline and tryptophan biosynthesis in Streptomyces coelicolor A3(2): interaction between primary and secondary metabolism – a review. Gene 115: 5–12.

Kaniga, K., Delor, I., and Cornelis, G.R. (1991) A wide-host-range suicide vector for improving reverse genetics in Gram-negative bacteria: inactivation of the *blaA* gene of Yersinia enterocolitica. Gene 109: 137–141.

Keatinge-Clay, A.T., Maltby, D.A., Medzihradszky, K.F., Khosla, C., and Stroud, R.M. (2004) An antibiotic factory caught in action. Nat Struct Mol Biol 11: 888–893.

Kobayashi, N., and Ichikawa, Y. (1989) A protein associated with prodigiosin formation in Serratia marcescens. Microbiol Immunol 33: 257–263.

Montaner, B., Navarro, S., Pique, M., Vilaseca, M., Martinell, M., Giralt, E., et al. (2000) Prodigiosin from the supernatant of Serratia marcescens induces apoptosis in haematopoietic cancer cell lines. Br J Pharmacol 131: 585–593.

Morrison, D.A. (1966) Prodigiosin synthesis in mutants of Serratia marcescens. J Bacteriol 91: 1599–1604.

Mortellaro, A., Songia, S., Gnocchi, P., Ferrari, M., Fornasier, C., D’Alessio, R., et al. (1999) New immunosuppressive drug PNU156804 blocks IL-2-dependent proliferation and NF-kappa B and AP-1 activation. J Immunol 162: 7102–7109.

Pérez-Tomás, R., Montaner, B., Llagostera, E., and Soto-Cerrato, V. (2003) The prodigiosins, proapoptotic drugs with anticancer properties. Biochem Pharmacol 66: 1447–1452.

Rapoport, H., and Holden, K.G. (1962) The synthesis of prodigiosin. J Am Chem Soc 84: 635–642.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) Molecular Cloning, a Laboratory Manual. New York: Cold Spring Harbour Laboratory Press.

Slater, H., Crow, M., Everson, L., and Salmond, G.P. (2003) Phosphate availability regulates biosynthesis of two antibiotics, prodigiosin and carbapenem, in Serratia via both quorum-sensing-dependent and -independent pathways. Mol Microbiol 47: 303–320.

Sole, M., Rius, N., Francia, A., and Loren, J.G. (1994) The effect of pH on prodigiosin production by non-proliferating cells of Serratia marcescens. Lett Appl Microbiol 19: 341–344.

Spiteller, P., Kern, W., Reiner, J., and Spiteller, G. (2001) Aldehydic lipid peroxidation products derived from linoleic acid. Biochim Biophys Acta 1531: 188–208.

Stock, I., Grueger, T., and Wiedemann, B. (2003) Natural antibiotic susceptibility of strains of Serratia marcescens and the S. liquefaciens complex: S. liquefaciens sensu stricto, S. proteamaculans and S. grimesii. Int J Antimicrob Agents 22: 35–47.

Tang, Y., Tsai, S.C., and Khosla, C. (2003) Polyketide chain length control by chain length factor. J Am Chem Soc 125: 12708–12709.

Thomas, M.G., Burkart, M.D., and Walsh, C.T. (2002) Conversion of 1-proline to pyrrolyl-2-carboxyl-S-PCP during undecylprodigiosin and pyoluteorin biosynthesis. Chem Biol 9: 171–184.

Thomson, N.R., Cox, A., Bycroft, B.W., Stewart, G.S., Williams, P., and Salmond, G.P. (1997) The rap and hor proteins of Erwinia, Serratia and Yersinia: a novel subgroup in a growing superfamily of proteins regulating diverse physiological processes in bacterial pathogens. Mol Microbiol 26: 531–544.

Thomson, N.R., Crow, M.A., McGowan, S.J., Cox, A., and Salmond, G.P. (2000) Biosynthesis of carbapenem antibiotic and prodigiosin pigment in Serratia is under quorum sensing control. Mol Microbiol 36: 539–556.

Traub, W.H. (2000) Antibiotic susceptibility of Serratia marcescens and Serratia liquefaciens. Chemotherapy 46: 315–321.

Tsao, S.W., Rudd, B.A., He, X.G., Chang, C.J., and Floss, H.G. (1985) Identification of a red pigment from Streptomyces coelicolor A3(2) as a mixture of prodigiosin derivatives. J Antibiot (Tokyo) 38: 128–131.

Tsuji, R.F., Yamamoto, M., Nakamura, A., Kataoka, T., Magae, J., Nagai, K., and Yamasaki, M. (1990) Selective immunosuppression of prodigiosin 25-C and FK506 in the murine immune system. J Antibiot (Tokyo) 43: 1293–1301.

Tsuji, R.F., Magae, J., Yamashita, M., Nagai, K., and Yamasaki, M. (1992) Immunomodulating properties of prodigiosin 25-C, an antibiotic which preferentially suppresses induction of cytotoxic T cells. J Antibiot (Tokyo) 45: 1295–1302.

Wasserman, H.H., Mc, K.J., and Santer, U.V. (1960) Studies related to the biosynthesis of prodigiosin in Serratia marcescens. Biochem Biophys Res Commun 3: 146–149.

Wasserman, H.H., Skles, R.J., Peverada, P., Shaw, C.K., Cushley, R.J., and Lipsky, C.R. (1973) Biosynthesis of prodigiosin. Incorporation patterns of C-labeled alanine, proline, glycine, and serine elucidated by fourier transform nuclear magnetic resonance. J Am Chem Soc 95: 6874–6875.

Webster, S.P., Alexeev, D., Campopiano, D.J., Watt, R.M., Alexeeva, M., Sawyer, L., and Baxter, R.L. (2000) Mechanism of 8-amino-7-oxononanoate synthase: spectroscopic, kinetic, and crystallographic studies. Biochemistry 39: 516–528.

Williams, R.P. (1973) Biosynthesis of prodigiosin, a secondary metabolite of Serratia marcescens. Appl Microbiol 25: 396–402.

Williams, R.P., and Quadri, S.M. (1980) The pigments of Serratia. In The Genus Serratia. Von Graevenitz, A., and Rubin, S.J. (eds). Boca Raton, FL: CRC Press, pp. 31–75.

Williams, R.P., Goldschmidt, M.E., and Gott, C.L. (1965) Inhibition by temperature of the terminal step in biosynthesis of prodigiosin. Biochem Biophys Res Commun 19: 177–181.

Williams, R.P., Gott, C.L., Qadri, S.M., and Scott, R.H. (1971) Influence of temperature of incubation and type of growth medium on pigmentation in *Serratia marcescens*. *J Bacteriol* 106: 438–443.

Xu, H., Wang, Z.X., Schmidt, J., Heide, L., and Li, S.M. (2002) Genetic analysis of the biosynthesis of the pyrrole

The biosynthetic pathway of prodigiosin in Serratia 989

and carbamoyl moieties of coumermycin A1 and novobiocin. *Mol Genet Genomics* 268: 387–396.

Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and Puc19 vectors. *Gene* 33: 103–119.
